<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-02-17 09:27:13 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>28</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>66</td>
          <td>134</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>105</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>314</td>
          <td>134</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>269</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cf05d6f649a55a948cbad9eac9ac4b6467ec" target='_blank'>
              Modern biology of extrachromosomal DNA: A decade-long voyage of discovery
              </a>
            </td>
          <td>
            Qing-Lin Yang, Yipeng Xie, Kailiang Qiao, Jun Yi Stanley Lim, Sihan Wu
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>3</td>
          <td>1</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a prevalent driver of cancer, whose random segregation promotes aggressive tumors. Acentric ecDNAs attach to chromosomes during mitosis for segregation. However, the molecular mechanism governing ecDNA-chromosome mitotic interactions remains poorly understood. This study shows that ecDNAs attach to histone 3 lysine 27 acetylation (H3K27ac)-marked chromatin during mitosis. H3K27ac depletion resulted in ecDNA detachment from mitotic chromosomes. Diverse bromodomain proteins, which are known readers of H3K27ac, stabilize ecDNAs’ mitotic interaction, exhibiting context-dependent and mutually complementary roles. Furthermore, disruptions of the Mediator complex and RNA polymerase II transcription activity both dissociate ecDNAs from mitotic chromosomes, suggesting that the transcription machinery mediates ecDNA segregation. Mis-segregated ecDNAs were expelled into the cytosol and degraded, leading to diminished oncogene expression and a reversal of therapy resistance. Our research provides new insights into the interplay between RNA transcription and acentric ecDNA inheritance in cancer, offering a novel avenue for disrupting ecDNA-driven oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b99e11fc3712d503e0a3c800fc35c4668a35ad3" target='_blank'>
              Transcription Machinery Anchors ecDNAs to Mitotic Chromosomes for Segregation
              </a>
            </td>
          <td>
            Yipeng Xie, Jun Yi Stanley Lim, Wenyue Liu, C. Gilbreath, Yoon Jung Kim, Sihan Wu
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="In mammalian cells, gene copy number is tightly controlled to maintain gene expression and genome stability. However, a common molecular feature across cancer types is oncogene amplification, which promotes cancer progression by drastically increasing the copy number and expression of tumor-promoting genes. For example, in tyrosine kinase inhibitor (TKI)-resistant lung adenocarcinoma (LUAD), oncogene amplification occurs in over 40% of patients’ tumors. Despite the prevalence of oncogene amplification in TKI-resistant tumors, the mechanisms facilitating oncogene amplification are not fully understood. Here, we find that LUADs exhibit a unique chromatin signature demarcated by strong CTCF and cohesin deposition in drug-naïve tumors, which correlates with the boundaries of oncogene amplicons in TKI-resistant LUAD cells. We identified a global chromatin priming effect during the acquisition of TKI resistance, marked by a dynamic increase of H3K27Ac, cohesin loading, and inter-TAD interactions, which occurs before the onset of oncogene amplification. Furthermore, we have found that the METTL7A protein, which was previously reported to localize to the endoplasmic reticulum and inner nuclear membrane, has a novel chromatin regulatory function by binding to amplified loci and regulating cohesin recruitment and inter-TAD interactions. Surprisingly, we discovered that METTL7A remodels the chromatin landscape prior to large-scale copy number gains. Furthermore, while METTL7A depletion has little effect on the chromatin structure and proliferation of drug-naïve cells, METTL7A depletion prevents the formation and maintenance of TKI resistant-clones, highlighting the specific role of METTL7A as cells are becoming resistant. In summary, we discovered an unexpected mechanism required for the acquisition of TKI resistance regulated by a largely uncharacterized factor, METTL7A. This discovery sheds light into the maintenance of oncogene copy number and paves the way to the development of new therapeutics for preventing TKI resistance in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864e77e3ffbc341dd750e7bc0c5fd24e1c53b031" target='_blank'>
              Epigenetic priming promotes acquisition of tyrosine kinase inhibitor resistance and oncogene amplification in human lung cancer
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G. Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B. Jensen, Ning Sun, Nelli Khudaverdyan, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, Gang Greg Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Xiao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87febc9c07e2846c41020472ac578cf946a0ca8e" target='_blank'>
              Breakage fusion bridge cycles drive high oncogene number with moderate intratumoural heterogeneity
              </a>
            </td>
          <td>
            Siavash Raeisi Dehkordi, I. Wong, Jing Ni, J. Luebeck, Kaiyuan Zhu, Gino Prasad, Lena Krockenberger, Guanghui Xu, Biswanath Chowdhury, Utkrisht Rajkumar, Ann Caplin, Daniel Muliaditan, Aditi Gnanasekar, C. Coruh, Q. Jin, Kristen Turner, Shu Xian Teo, A. W. Pang, Ludmil B. Alexandrov, Christelle En Lin Chua, F. Furnari, J. Maciejowski, Thomas G Paulson, Julie A. Law, Howard Y. Chang, Feng Yue, Ramanuj DasGupta, Jean J. Zhao, P. Mischel, V. Bafna
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are large, acentric, circular DNA molecules that occur pervasively across many human cancers. EcDNA can drive tumor formation and evolution, contribute to drug resistance, and associate with poor patient survival outcomes. Beyond mediating high copy numbers, the circular topology and dynamic conformational changes of ecDNA disrupt topological domains and rewire regulatory networks, thereby conferring an important role in the transcriptional regulation of oncogenes. Here, we develop ec3D, a computational method for reconstructing the three-dimensional structures of ecDNA and analyzing significant interactions from high-throughput chromatin capture (Hi-C) data. Given a candidate ecDNA sequence and the corresponding whole-genome Hi-C as input, ec3D reconstructs the spatial structure of ecDNA by maximizing the Poisson likelihood of observed interactions. Ec3D’s performance was validated using both simulated ecDNA structures with varying conformations, and Hi-C data from previously-characterized cancer cell lines. Our reconstructions reveal that ecDNAs occupy spherical configurations and mediate unique long-range interactions involved in gene regulation. Through algorithmic innovations, ec3D can resolve complex ecDNA structures with duplicated copies of large genomic segments, identify multi-way interactions, distinguish between interactions arising from direct spatial proximity and secondary interactions resulting from alternative folding patterns or intermolecular (trans) contacts of ecDNA molecules. Our findings provide insights into how the spatial organization of ecDNA may influence gene regulation and contribute to increased oncogene expression. Code availability https://github.com/AmpliconSuite/ec3D">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aebb6e832c80fde421190f885801636b3438e629" target='_blank'>
              Reconstructing the three-dimensional architecture of extrachro-mosomal DNA with ec3D
              </a>
            </td>
          <td>
            Biswanath Chowdhury, Kaiyuan Zhu, Chaohui Li, J. Luebeck, Owen S. Chapman, Katerina Kraft, Shu Zhang, Lukas Chavez, A. Henssen, P. Mischel, Howard Y. Chang, V. Bafna
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c6d858496cf48cc3c25b9e46942a18e35bc4d" target='_blank'>
              A compendium of Amplification-Related Gain Of Sensitivity genes in human cancer
              </a>
            </td>
          <td>
            Verónica Rendo, Michael Schubert, Nicholas Khuu, Maria F Suarez Peredo Rodriguez, Declan Whyte, Xiao Ling, A. van den Brink, Kaimeng Huang, Michelle L Swift, Yizhou He, Johanna Zerbib, Ross Smith, J. Raaijmakers, P. Bandopadhayay, Lillian M. Guenther, Justin H Hwang, A. Iniguez, Susan Moody, J-H. Seo, Elizabeth H Stover, Levi A. Garraway, William C. Hahn, K. Stegmaier, René H. Medema, Dipanjan Chowdhury, Maria Colomé-Tatché, Uri Ben-David, R. Beroukhim, F. Foijer
          </td>
          <td>2025-01-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e5d3752411db0d4a1f1f0e0c0f053217196ee" target='_blank'>
              Unveiling the mysteries of extrachromosomal circular DNA: from generation to clinical relevance in human cancers and health
              </a>
            </td>
          <td>
            Zilong Wang, Jiaying Yu, Wenli Zhu, Xiaoning Hong, Zhen Xu, Shuang Mao, Lei Huang, Peng Han, Chunxiao He, Changze Song, Xi Xiang
          </td>
          <td>2024-12-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Dysregulation of enhancer-promoter communication in the context of the three-dimensional (3D) nucleus is increasingly recognized as a potential driver of oncogenic programs. Here, we profiled the 3D enhancer-promoter networks of primary patient-derived glioblastoma stem cells (GSCs) in comparison with neuronal stem cells (NSCs) to identify potential central nodes and vulnerabilities in the regulatory logic of this devastating cancer. Specifically, we focused on hyperconnected 3D regulatory hubs and demonstrated that hub-interacting genes exhibit high and coordinated expression at the single-cell level and strong association with oncogenic programs that distinguish IDH-wt glioblastoma patients from low-grade glioma. Epigenetic silencing of a recurrent 3D enhancer hub—with an uncharacterized role in glioblastoma—was sufficient to cause concordant downregulation of multiple hub-connected genes along with significant shifts in transcriptional states and reduced clonogenicity. By integrating published datasets from other cancer types, we also identified both universal and cancer type-specific 3D regulatory hubs which enrich for varying oncogenic programs and nominate specific factors associated with worse outcomes. Genetic alterations, such as focal duplications, could explain only a small fraction of the detected hyperconnected hubs and their increased activity. Overall, our study provides computational and experimental support for the potential central role of 3D regulatory hubs in controlling oncogenic programs and properties. HIGHLIGHTS - 3D regulatory “hubs” in glioblastoma enrich for highly coregulated genes at a single-cell level and expand oncogenic regulatory networks. - Targeted perturbation of a highly recurrent 3D regulatory hub in GSCs results in altered transcriptional states and cellular properties. - 3D regulatory hubs across cancer types associate with tumor-specific and universal oncogenic programs and worse outcomes. - The majority of hyperconnected hubs do not overlap with structural variants, suggesting epigenetic mechanisms. eTOC Here we profile the 3D enhancer connectomes of primary patient-derived human glioblastoma stem cells (GSCs), identify hyperconnected 3D regulatory “hubs”, and examine the impact of 3D hub perturbation on the transcriptional program and oncogenic properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef46a4bf29419cbc8b345c6cf8f3bc9f4287626" target='_blank'>
              Three-dimensional regulatory hubs support oncogenic programs in glioblastoma
              </a>
            </td>
          <td>
            Sarah L. Breves, Dafne Campigli Di Giammartino, James Nicholson, S. Cirigliano, Syed Raza Mahmood, UkJin Lee, Alexander Martínez-Fundichely, Johannes Jungverdorben, R. Singhania, Sandy Rajkumar, Raphael Kirou, L. Studer, Ekta Khurana, A. Polyzos, Howard A. Fine, E. Apostolou
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Genome organization recapitulates function, yet ciliates like Oxytricha trifallax possess highly-specialized germline genomes, which are largely transcriptionally silent. During post-zygotic development, Oxytricha’s germline undergoes large-scale genome editing, rearranging precursor genome elements into a transcriptionally-active genome with thousands of gene-sized nanochromosomes. Transgenerationally-inherited RNAs, derived from the parental somatic genome, program the retention and reordering of germline fragments. Retained and eliminated DNA must be distinguished and processed separately, but the role of chromatin organization in this process is unknown. We developed tools for studying Oxytricha nuclei and apply them to map the 3D organization of precursor and developmental states using Hi-C. We find that the precursor conformation primes the germline for development, while a massive spatial reorganization during development differentiates retained from eliminated regions before DNA rearrangement. Further experiments suggest a role for RNA-DNA interactions and chromatin remodeling in this process, implying a critical role for 3D architecture in programmed genome rearrangement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/381a65f6ebad009521fa755048f58360bf57403c" target='_blank'>
              Widespread 3D genome reorganization precedes programmed DNA rearrangement in Oxytricha trifallax
              </a>
            </td>
          <td>
            D. Villano, Manasa Prahlad, Ankush Singhal, K. Sanbonmatsu, L. Landweber
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Lampbrush chromosomes are giant meiotic bivalents in growing oocyte nuclei that have served as a classic model system for studying chromatin organization and RNA synthesis for over a century. Despite their importance, the molecular mechanisms underlying lampbrush chromosomes formation and their distinctive chromomere-loop architecture have remained poorly understood. Here, we provide the first comprehensive genomic, cytological, and biophysical analysis of lampbrush chromosome organization by integrating single-cell Hi-C, RNA-seq, NOMe-seq, FISH mapping, and chromatin simulations. Single-nucleus Hi-C analysis revealed CTCF-independent contact domains with stable boundaries defined by transcription units in a convergent orientation. Сontact domains identified through Hi-C analysis correspond to insulated chromomeres in lampbrush chromosomes. Small transcriptionally inactive contact domains surrounded by transcription units in the diverged orientation form ‘chromatin knots’, which are often detached from the chromosome axis. Transcription loops frequently manifest as a ‘cross’ pattern with reduced contacts within chromatin domains. Integrative analysis of the whole-genome data uncovers the mechanisms underlying lampbrush chromosome structure, revealing how hypertranscription modulates chromatin stiffness and repositions SMC complexes to establish the distinctive chromomere-loop organisation. Biophysical modeling through polymer simulation reproduces key features of lampbrush chromosomes, including transcription loop formation, chromomere compaction, and insulation patterns. These findings offer a unifying framework for understanding the remarkable chromatin architecture of lampbrush chromosomes and their transcription-dependent organization. Highlights - First integration of single-cell Hi-C, RNA-seq, NOMe-seq and microscopy methods uncovers molecular mechanisms underlying lampbrush chromosome architecture. - Hi-C reveals contact patterns corresponding to lampbrush chromomeres and transcription loops, validated through BAC-based FISH mapping. - Lampbrush chromosomes are segmented into contact domains formed via a CTCF-independent mechanism, with boundaries coinciding with convergently oriented gene pairs. - Hypertranscription shapes lampbrush chromosome through multiple mechanisms, increasing stiffness and decreasing compaction of transcribed units, generating outward pressure, pushing transcription loops away from the chromosome axis, and repositioning SMC complexes to form transcription-dependent domains with stable boundaries. - Polymer simulations including these mechanisms replicate key features of LBC organization, supporting the proposed model of lampbrush chromosomes formation. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46e642c22825b99c48d9f70a1c1f9236c18b4d5e" target='_blank'>
              The 3D genomics of lampbrush chromosomes highlights the role of active transcription in chromatin organization
              </a>
            </td>
          <td>
            T. Lagunov, M. Gridina, A. Nurislamov, T. Kulikova, A. Maslova, V. Konstantinov, A. Popov, A. Krasikova, V. Fishman
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Originating from, but independent of, linear chromosomes, extrachromosomal DNA (ecDNA) exists in a more active state of transcription and autonomous replication. It plays a crucial role in the development of malignancies and therapy resistance. Since its discovery in eukaryotic cells more than half a century ago, the biological characteristics and functions of ecDNA have remained unclear due to limitations in detection methods. However, recent advancements in research tools have transformed ecDNA research. It is believed that ecDNA exhibits greater activity in the abnormal amplification of oncogenes, thereby driving cancer progression through their overexpression. Notably, compared to linear DNA, ecDNA can also function as a genomic element with regulatory roles, including both trans- and cis-acting functions. Its critical roles in tumorigenesis, evolution, progression, and drug resistance in malignant tumors are increasingly recognized. This review provides a comprehensive summary of the evolutionary context of ecDNA and highlights significant progress in understanding its biological functions and potential applications as a therapeutic target in malignant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf3896d81e701493b4d97e132a04a0cb6710489" target='_blank'>
              Insights into Extrachromosomal DNA in Cancer: Biogenesis, Methodologies, Functions, and Therapeutic Potential.
              </a>
            </td>
          <td>
            Xudong Mao, Guocheng Rao, Gonghui Li, Shihan Chen
          </td>
          <td>2025-02-13</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="All cells possess mechanisms to maintain and replicate their genomes, whose integrity and transmission are constantly challenged by DNA damage and replication impediments. In eukaryotes, the protein kinase Ataxia-Telangiectasia and Rad3-related (ATR), a member of the phosphatidylinositol 3-kinase-like family acts as a master regulator of the eukaryotic response to DNA injuries, ensuring DNA replication completion and genome stability. Here we aimed to investigate the functional relevance of the ATR homolog in the DNA metabolism of Leishmania major, a protozoan parasite with a remarkably plastic genome. CRISPR/cas9 genome editing was used to generate a Myc-tagged ATR cell line (mycATR), and a Myc-tagged C-terminal knockout of ATR (mycATRΔC-/-). We show that the nuclear localisation of ATR depends upon its C-terminus. Moreover, its deletion results in single-stranded DNA accumulation, impaired cell cycle control, increased levels of DNA damage, and delayed DNA replication restart after replication stress. In addition, we show that ATR plays a key role in maintaining L. major’s unusual DNA replication program, where larger chromosomes duplicate later than smaller chromosomes. Our data reveals loss of the ATR C-terminus promotes the accumulation of replication signal around replicative stress fragile sites, which are enriched in larger chromosomes. Finally, we show that these alterations to the DNA replication program promote chromosome instability. In summary, our work shows that ATR acts to moderate DNA replication timing thus limiting the plasticity of the Leishmania genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3499060635df2f56eb484fb5207c0367f3df1797" target='_blank'>
              ATR, a DNA damage kinase, modulates DNA replication timing in Leishmania major
              </a>
            </td>
          <td>
            Gabriel L. A. da Silva, J. Damasceno, J. A. Black, R. McCulloch, L. Tosi
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a6c622797b8d89bba676a8b9a3206b6fe5c1df9" target='_blank'>
              A constitutive heterochromatic region shapes genome organization and impacts gene expression in Neurospora crassa
              </a>
            </td>
          <td>
            Andrew T. Reckard, Abhishek Pandeya, Jacob M. Voris, Carlos G. Gonzalez Cruz, Oluwatosin Oluwadare, Andrew D. Klocko
          </td>
          <td>2024-12-20</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72bc6fce345e0628d21c6c9477e460943646693b" target='_blank'>
              Spatial deconvolution from bulk DNA methylation profiles determines intratumoral epigenetic heterogeneity
              </a>
            </td>
          <td>
            Binbin Liu, Yumo Xie, Yu Zhang, Guannan Tang, Jinxin Lin, Ze Yuan, Xiaoxia Liu, Xiaolin Wang, Meijin Huang, Yanxin Luo, Huichuan Yu
          </td>
          <td>2025-01-23</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Centromeres are chromosomal loci that ensure proper chromosome segregation by providing a platform for kinetochore assembly and spindle force transduction during cell division. Human centromeres are defined primarily by a unique chromatin domain featuring the histone H3 variant, Centromere Protein A (CENP-A), that marks a single active centromere locus per chromosome. CENP-A chromatin typically occupies a small subregion of low DNA methylation within multi-megabase arrays of hypermethylated alpha-satellite repeats and constitutive pericentric heterochromatin. However, the mechanisms defining and maintaining precise centromere position and domain size, and the role of the underlying alpha satellite DNA sequence, are poorly characterised. Using an experimentally-induced neocentromere in RPE1 cells, we discovered that the SUV39H1 and H2 methyltransferases tri-methylate H3K9 at neocentromere boundaries to maintain CENP-A domain size independent of DNA methylation or satellite sequences. Furthermore, we found that the CENP-A domain at canonical alpha-satellite-based centromeres is characterized by local depletion of H3K9me3-mediated heterochromatin, coinciding with the DNA methylation dip region. We identified the SETDB1 methyltransferase as key to maintaining H3K9me3 within flanking active higher order alpha satellite arrays while SUV39s and SUZ12 contribute to globally heterochromatinize both alpha satellites and neighbouring repeats. Loss of this heterochromatin boundary results in the progressive expansion of the primary CENP-A domain, erosion of DNA methylation, and the nucleation of new centromeres across alpha satellite sequences. Our study identifies the functional specialization of different H3K9 methyltransferases across centromeric and pericentric domains, crucial for maintaining centromere domain size and number.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d21ac02ba5e831d24deadbae2a16d8fc0cb0af4d" target='_blank'>
              Heterochromatin boundaries maintain centromere position, size and number
              </a>
            </td>
          <td>
            Ben L. Carty, Danilo Dubocanin, Marina Murillo-Pineda, Marie Dumont, Emilia Volpe, Pawel Mikulski, Julia Humes, Oliver Whittingham, D. Fachinetti, Simona Giunta, Nicolas Altemose, L. Jansen
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Endogenous retroviral (ERV) RNA is highly expressed in cancer, although the molecular causes and consequences remain unknown. We found that ZC3H18 (Z18), a component of multiple nuclear RNA surveillance complexes, has recurrent truncating mutations in cancer. We show that Z18trunc mutations are oncogenic and that Z18 plays an evolutionarily conserved role in nuclear RNA surveillance of ERV RNA. In zebrafish, Z18trunc expedited melanoma onset and promoted a specific accumulation of ERV RNA. Z18 mutant human cell lines from the Cancer Cell Line Encyclopedia also expressed higher levels of ERV RNA. In engineered human melanoma cells, Z18trunc enhanced ERV RNA accumulation more than loss of one Z18 copy, indicating dominant negative activity. Z18trunc directly bound and stabilized ERV RNA. Notably, expression of ERV RNA was sufficient to expedite oncogenesis in a zebrafish model, which is the first evidence of which we are aware that ERV transcripts can play a functional role in cancer. Our work illuminates a mechanism for elevated ERV transcripts in cancer and supports that aberrant RNA accumulation is broadly oncogenic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85414cac93dcb00a3b2cbff32569ab0497071b3c" target='_blank'>
              Recurrent oncogenic ZC3H18 mutations stabilize endogenous retroviral RNA
              </a>
            </td>
          <td>
            Tanzina Tanu, Anna M. Cox, Jennifer A. Karlow, Priyanka Sharma, Xueyang He, Constance Wu, Swathy Babu, Jared Brown, Kevin M. Brown, Stephen J. Chanock, David Liu, Tongwu Zhang, Kathleen H. Burns, P. Boutz, Megan L. Insco
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06ca40e33657af743c749690c1200f7526a655d0" target='_blank'>
              NKX6.3 modulation of mitotic dynamics and genomic stability in gastric carcinogenesis
              </a>
            </td>
          <td>
            J. H. Yoon, J. Kim, J. Eun, H. Ashktorab, Duane T. Smoot, S. Nam, Won Sang Park
          </td>
          <td>2025-01-20</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Prostate cancer is characterized by profound heterogeneity in its clinical trajectory. While genomic heterogeneity has been well-characterized, epigenomic heterogeneity remains less understood. To fill this gap, we compiled 2,149 multi-ancestric prostate methylomes spanning normal tissue through localized disease of varying grades to poly-metastatic disease, most with multi-omic DNA and/or RNA characterization. We identify four subtypes that varied by stage, grade and mutational subtype. We identify extensive interplay between DNA ploidy and DNA methylation, with transcriptional consequences that vary across driver-genes. Each major prostate cancer driver gene mutation triggers a specific epigenetic dysregulation, and we define a set of 14 reusable models that accurately predict clinico-molecular features of a prostate cancer from DNA methylation data. Specific epigenetic features predict disease aggression, including metastasis, with epigenomic and genomic features synergizing to optimize predictions. These results define a complex interplay between tumour genetics and epigenetics that converges to modify gene-expression programs, and subsequent clinical presentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c74606bb02b8c93f71db20ba35bf801348005a77" target='_blank'>
              The Landscape of Prostate Tumour Methylation
              </a>
            </td>
          <td>
            J. Arbet, Yu-Jia Shiah, Takafumi N. Yamaguchi, Rupert Hugh-White, Adriana Wiggins, Jieun Oh, Tsumugi Gebo, Adrien Foucal, R. Lesurf, Chol-Hee Jung, Rachel M. A. Dang, Raag Agrawal, J. Livingstone, A. Salcedo, C. Yao, S. M. G. Espiritu, Kathleen E. Houlahan, Fouad Yousif, Lawrence E Heisler, M. Fraser, A. Kishan, Alejandro Berlin, Melvin LK Chua, T. H. van der Kwast, Christopher M Hovens, Rob Bristow, P. Boutros
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="
 Nearly half of the human genome is composed of transposable elements (TEs), which were once considered “junk” DNA due in part to their repetitive and non-coding characteristics. However, TEs are now emerging as important regulators of gene activity, especially when they escape typical epigenetic silencing mechanisms like DNA methylation and histone modifications. When activated, TEs can contribute sequences that are exploited as disruptors of genomic stability or novel gene regulatory elements, making them important factors in both development and disease, including cancer. In cancer, these previously silenced elements can act as alternative promoters of oncogenes, leading to the production of TE-chimeric transcripts in a process called “onco-exaptation”. While these oncoexaptation events are reported to be widespread across many cancers, only a few TE-chimeric transcripts have been validated as active contributors to cancer progression. Additionally, it remains unclear whether these TE-chimeric transcripts are merely a consequence of the inherent genomic instability in cancer or whether they can initiate cancer. To address these gaps, we designed a cDNA overexpression library incorporating our previously identified pan-cancer TE-chimeric transcript candidates. This library is used in early passage (EP) BJ fibroblast, a cell line known for its transformation potential in the well-established Weinberg transformation model, which allows us to systematically investigate these onco-exaptation events during cancer progression. Our initial analyses indicate that most TE-chimeric oncogenes are not expressed in EP fibroblast, but are activated in subsequent transformation stages following hTERT, SV40 and HRas induction, underscoring the model viability. We are currently using this model to distinguish TE-chimeric transcripts that act as drivers (TE-chimeric oncogenes) from those requiring previous vulnerabilities, such as telomere inactivation by hTERT or p53 and Rb inactivation by SV40. Ultimately, this work will identify novel biomarkers and therapeutic targets, enhance insights into gene regulation, and provide opportunities for early detection and intervention. These findings have the potential to improve cancer treatment and patient outcomes.
 Citation Format: Wesley N Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang. The functional role of transposable element-oncogene chimeric transcript during cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f7a8875ef14376de973322ef301e4c4ddf4aab" target='_blank'>
              Abstract B005: The functional role of transposable element-oncogene chimeric transcript during cancer progression
              </a>
            </td>
          <td>
            Wesley N. Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bd6b6473526b044dda123e9fa9d72ac0fc045ca" target='_blank'>
              A dual role of Cohesin in DNA DSB repair
              </a>
            </td>
          <td>
            Michael Fedkenheuer, Yafang Shang, Seolkyoung Jung, Kevin Fedkenheuer, Solji Park, Davide Mazza, Robin Sebastian, Hiroyuki Nagashima, D. Zong, Hua Tan, S. Jaiswal, Haiqing Fu, Anthony Cruz, Supriya V Vartak, Jan Wisniewski, Vittorio Sartorelli, John J O'Shea, L. Elnitski, A. Nussenzweig, M. Aladjem, Fei Meng, Rafael Casellas
          </td>
          <td>2025-01-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfacdbb6d5b3609b2de9ba21e1d307534c0157c2" target='_blank'>
              SETD2 loss-of-function uniquely sensitizes cells to epigenetic targeting of NSD1-directed H3K36 methylation
              </a>
            </td>
          <td>
            Ryan T Wagner, R. Hlady, Xiaoyu Pan, Liguo Wang, Sungho Kim, Xia Zhao, Louis Y El Khoury, Shafiq Shaikh, Jian Zhong, Jeong-Heon Lee, J. Grembecka, Tomasz Cierpicki, Thai H Ho, Keith D. Robertson
          </td>
          <td>2025-02-05</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7990aaecf7325cfad3b1c506171580acc4c1663d" target='_blank'>
              Characterization of the genomic landscape of canine diffuse large B-cell lymphoma reveals recurrent H3K27M mutations linked to progression-free survival
              </a>
            </td>
          <td>
            Anna Darlene van der Heiden, Raphaela Pensch, Sophie Agger, Heather L. Gardner, William Hendricks, V. Zismann, Shukmei Wong, N. Briones, Bryce Turner, Karin Forsberg-Nilsson, Cheryl A London, Kerstin Lindblad-Toh, M. Arendt
          </td>
          <td>2025-02-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Ultraviolet (UV)-induced DNA mutations produce genetic drivers of cutaneous melanoma initiation and numerous neoantigens that can trigger anti-tumor immune responses in the host. Consequently, melanoma cells must rapidly evolve to evade immune detection by simultaneously modulating cell-autonomous epigenetic mechanisms and tumor-microenvironment interactions. Angiogenesis has been implicated in this process; although an increase of vasculature initiates the immune response in normal tissue, solid tumors manage to somehow enhance blood flow while preventing immune cell infiltration. By comparing the expression of transcription factors (TFs) across early-stage melanoma, naevi, and other cancer types, we found the homeodomain-containing TF HOXD13 drives a melanoblast-like developmental program, which is upregulated in melanoma and strongly correlated with angiogenesis and immune cell exclusion. Using transcriptomics, 3D chromatin profiling, and in vivo models, we demonstrate that HOXD13 upregulation promotes tumor growth in vivo by concomitantly enhancing angiogenesis and suppressing T-cell infiltration. HOXD13 orchestrates 3D chromatin contacts between distal enhancers and promoters, simultaneously activating VEGFA, SEMA3A, and CD73. VEGFA and SEMA3A remodel the tumor vasculature and CD73 elevates extracellular levels of adenosine, a vasodilator and immune suppressor that binds adenosine receptors (AdR) on endothelial and T cells. In line with these findings, HOXD13-induced growth advantage in vivo was significantly reversed by the concomitant administration of VEGFR and AdR inhibitors. By revealing a dual pro-angiogenic and immunosuppressive HOXD13-CD73/VEGF gene regulatory axis, we identify a subset of patients who might benefit from combinations of AdR and VEGFR inhibitors which are both currently being tested in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65e09ae66ed07e79880f3fb8e2a1de2aa1f2113" target='_blank'>
              A targetable developmental program co-regulates angiogenesis and immune evasion
              </a>
            </td>
          <td>
            P. Berico, Amanda Flores Yanke, Fatemeh Van Rajabpour, Catherine Do, Irving Simonin Wilmer, Theodore Sakellaropoulos, Ines Delclaux, Robert Stagnitta, Estefania Vázquez-Cruz, Iman Osman, Jane A Skok, C. D. Robles-Espinoza, Amanda W. Lund, Markus Schober, Eva Hernando
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Two decades after the initial report on increased micronuclei in human chronic liver disease (CLD) and hepatocellular carcinoma (HCC), their role in HCC development is still poorly understood. Here we show that micronuclei in hepatocytes trigger hepatic immune response and promote HCC development via an increased level of extrachromosomal circular DNA (eccDNA). Livers from a CLD model (Mcl1Δhep mice) show increased micronuclei in parallel to eccDNA. Circular sequencing confirms higher eccDNA levels in micronuclei compared to primary nuclei. We developed nuclei-segregated DNA fiber (NuSeF) assay to show that micronuclei are more susceptible to replication stress, showing increased replication fork slowing. By comparing different murine liver disease models, high eccDNA is correlated with increased tumor incidence. EccDNA is a strong immunostimulant and promotes a crosstalk between hepatocytes and immune cells through the cGAS-STING pathway. Deletion of Sting1 in Mcl1Δhep mice reduces immune cell chemotaxis, as well as tumor incidence. Our findings suggest that eccDNA from micronuclei mediates inflammation-driven liver carcinogenesis in CLD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/563897a8a27ca1930e83755c188ad3a2e992e96a" target='_blank'>
              Extrachromosomal circular DNA promotes inflammation and hepatocellular carcinoma development
              </a>
            </td>
          <td>
            Lap Kwan Chan, Juanjuan Shan, Elias Rodriguez-Fos, Marc E. Healy, Peter Leary, Rossella Parrotta, Nina Desboeufs, Gabriel Semere, Nadine Wittstruck, A. Henssen, Achim Weber
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Histones scaffold genomic DNA and regulate access to the transcriptional machinery. However, naturally occurring histone variants can alter histone-DNA interactions, DNA and histone modifications, and the chromatin interactome. Hence, alterations in histone variant deposition can disrupt chromatin, and are increasingly recognized as a way to trigger various disease, including cancer. While significant attention has been placed on the biochemical and functional roles of H2A, H3, and H4 histone variants, the variants of H2B remain largely understudied. Here, we show that H2B variants are dysregulated in breast cancer and that certain variants are associated with specific breast cancer subtypes. HIST1H2BO overexpression (in particular) is more common in Asian, African American/Black, and young female populations and is associated with a worse prognosis. In vitro studies show that H2B1O compacts nucleosome structure. Incorporating H2B1O into chromatin activates pro-inflammatory and oncogenic pathways, induces the epithelial-to-mesenchymal transition (EMT), and generates resistance to first-line chemotherapeutic agents. Thus, H2B1O acts much like an onco-histone, with H2B variant expression being a prognostic biomarker for breast cancer and a potential new target for drug therapies to enhance treatment efficacy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afd8a4ca936d0e08dadf10708fe50ded7fd3756c" target='_blank'>
              Incorporating histone H2B variants into chromatin modifies chromatin accessibility to induce epithelial-to-mesenchymal transition in breast cancer
              </a>
            </td>
          <td>
            Hejer Dhahri, Kin H. Lau, Wesley N. Saintilnord, Elisson Lopes, Hannah Damico, Flavio R. Palma, Daniёl P. Melters, Darrell P Chandler, Yamini Dalal, Jonathan D. Licht, Marcello G. Bonini, Yvonne N Fondufe-Mittendorf
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Ductal carcinoma in situ (DCIS) harbors genomic copy number changes reflecting early events in advancing invasive breast cancer, and changes in specific cell states related to cancer initiation and progression. DNA methylation is a covalent modification to cytosines with cell type specificity, is stable in archival samples, and provides most of the specificity in multiple noninvasive pan-cancer screens. However, the creation of single-cell DNA methylomes is technically challenging and analysis is immature, leaving cell type and state level interpretation understudied. This has led to a gap in knowledge: it is not understood how the methylome reacts to gene dosage changes brought on by early copy number events, and it is not known how long these potential methylation changes persist. To this end, we have created and optimized a high throughput assay to measure these methylation changes at single-cell level, pairing our analysis with a multimodal view of these early copy number events. Here we describe three major steps in our work on elucidating methylome response to early breast cancer events: 1) We have generated methylome libraries for breast cancer cell lines with known large scale copy number alterations and validated our copy number calling in methylomes with those of gold-standard single-cell whole genome libraries. 2) We have generated thousands of single cell methylomes on normal breast samples attained from reduction surgeries paired with our groups’ work on the Human Breast Cell Atlas project. We were able to identify all major cell types of the human breast using methylation data for the first time. We used this data set to generate high-quality pseudobulk cell type specific methylomes. 3) We have profiled 5 DCIS and 1 synchronous invasive ductal carcinoma sample with subclonal copy number events, paired with single-cell transcriptomics. Through this we identified subclones in our methylome libraries and validated these through inferred copy number changes in our single-cell transcriptome libraries. We have uncovered thousands of hypomethylated regions enriched for specific transcription factor motifs, and widespread methylation changes across DCIS subclones. Notably many differential methylation changes occur outside of the genomic boundaries of copy number effects, suggesting distal changes by transcription factor gene dosage, chromatin perturbation, or tumor microenvironment interactions. These data will serve as a basis for future analyses into long-term follow through of archival samples and expand our understanding of this understudied epigenomic mark and its importance in early cancer evolution.
 Citation Format: Ryan M Mulqueen, Mariam Mosaad, Shanshan Bai, Jiahong Li, Emi Sei, Bora Lim, Savitri Krishnamurthy, Alastair Thompson, Nicholas Navin. Copy number analysis of single-cell methylomes reveal distinct subclone methylation in early breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fa4f976dba878ba71cab253948818a9b0e43066" target='_blank'>
              Abstract A006: Copy number analysis of single-cell methylomes reveal distinct subclone methylation in early breast cancer
              </a>
            </td>
          <td>
            Ryan M Mulqueen, Mariam Mosaad, S. Bai, Jiahong Li, Emi Sei, Bora Lim, Savitri Krishnamurthy, Alastair Thompson, Nicholas Navin
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="An aggressive subtype of acute myeloid leukemia (AML) is caused by enhancer hijacking resulting in MECOM overexpression. Several chromosomal rearrangements can lead to this: the most common (inv(3)/t(3;3)) results in a hijacked GATA2 enhancer, and there are several atypical MECOM rearrangements involving enhancers from other hematopoietic genes. The set of enhancers which can be hijacked by MECOM can also be hijacked by BCL11B. Enhancer deregulation is also a driver of oncogenesis in a range of other malignancies. The mechanisms of enhancer deregulation observed in other cancer types, including TAD boundary disruptions and the creation of de novo (super-) enhancers, may explain overexpression of MECOM or other oncogenes in AML without enhancer hijacking upon translocation. Gaining mechanistic insight in both enhancer deregulation and super-enhancer activity is critical to pave the way for new treatments for AML and other cancers that are the result of enhancer deregulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc88804fb71dad9d4f7cceb8ac265d5c25968818" target='_blank'>
              Mechanisms of enhancer-driven oncogene activation.
              </a>
            </td>
          <td>
            Joyce Vriend, Ruud Delwel, Dorien Pastoors
          </td>
          <td>2025-01-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a prevalent male cancer with high survival rates, except in advanced or metastatic stages, for which effective treatments are lacking. Metastatic PCa involves complex mechanisms including loss of tumor suppressor genes and DNA repair molecules, which impacts therapy responses. We have reanalyzed data from a CRISPR/Cas9 genome wide screening previously performed to identify essential regulators of invasive abilities of the metastatic cell line DU145 identifying SYCP3 as a regulator of metastatic invasion. Subsequent analyses of tumor samples demonstrated that SYCP3 expression is frequently upregulated in PCa tumors from patients in advanced stages. Furthermore, SYCP3 genetic depletion significantly reduced the invasive and migratory abilities of DU145 cells and increased their adhesion capacity. Additionally, and due to the implication of SYCP3 on DNA repair processes, we have analyzed the role of SYCP3 on the cellular response to radiotherapy (RT) and found that its depletion induced RT resistance, suggesting a role for SYCP3 in DNA damage response and genomic instability. All these data support a role for SYCP3 in PCa metastasis and provides opportunities for personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016d79d4fcdaa6ad45de27d2da94398a7826f930" target='_blank'>
              CRISPR screening reveals SYCP3 as a key driver of metastasis in prostate cancer
              </a>
            </td>
          <td>
            Maria Rodrigo-Faus, Inés del Monte-García, Marina Hermosilla-Trespaderne, Alicia Gordo-Vega, Natalia Vidal, Javier Puente, Melchor Saiz-Pardo, Ángel M. Cuesta, Hui-Qi Qu, H. Hakonarson, Almudena Porras, Daniel Sanchez-Parcerisa, Paloma Bragado, Á. Gutierrez-Uzquiza
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are four-stranded alternative secondary structures formed by guanine-rich nucleic acids and are prevalent across the human genome. G4s are enzymatically resolved using specialized helicases. Previous in vitro studies showed that DEAH-box Helicase 36 (DHX36/G4R1/RHAU), has the highest specificity and affinity for G4 structures. Here, by mapping genome-wide DNA double-strand breaks (DSBs), we demonstrate that knockout (KO) of DHX36 helicase increases DSB enrichment at G4 sites and that the presence of the G4 motif is a significant mediator of genome instability at regulatory regions. The loss of DHX36 corresponds with the significant upregulation of NF-κB transcriptional programs, culminating in the production and secretion of proinflammatory cytokines. Loss of DHX36 expression results in an increase in the innate immune signaling stimulator of interferon response cGAMP interactor 1 (STING1) expression and activation of genes involved in immune response pathways. Importantly, higher levels of DHX36 mRNA expression in human B-cell acute lymphoblastic leukemia correlate with improved overall survival relative to lower expression of DHX36, highlighting its critical role in preserving genome integrity at a cellular level and in the context of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f6a2067e514e41d31ed3af27971fbc66a915c36" target='_blank'>
              Loss of DHX36/G4R1, a G4 resolvase, drives genome instability and regulates innate immune gene expression in cancer cells
              </a>
            </td>
          <td>
            Anna R Bartosik, Pei-Chi Hou, James P. Vaughn, P. J. Smaldino, A. Ratan, Marty W. Mayo, Wang Yuh-Hwa
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA Damage Response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically re-programmed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of Pancreatic Ductal Adenocarcinomas (PDAC), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells (HPNE). Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced mRNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a Homologous Recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR-deficiency (HRD) phenotypes including sensitivity to inhibitors of Trans-Lesion Synthesis (TLS) and Poly-ADP Ribose Polymerase (PARP). Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13-depletion. Taken together our results expose TRIP13 as an attractive new and therapeutically-tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89fa9b68b58350edbc77be5948633226c0fa98f1" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="Abstract Sequencing of human patient tumors has identified recurrent missense mutations in genes encoding core histones. We report that mutations that convert histone H3 amino acid 50 from a glutamate to a lysine (H3E50K) support an oncogenic phenotype. Expression of H3E50K is sufficient to transform human cells as evidenced by an increase in cell migration and invasion, and an increase in proliferation and clonogenicity. H3E50K also increases the invasive phenotype in the context of co-occurring BRAF mutations, which are present in patient tumors characterized by H3E50K. H3E50 lies on the globular domain surface in a region that contacts H4 within the nucleosome. We find that H3E50K selectively increases chromatin accessibility and perturbs proximal H3 post-translational modifications including H3K27me3; together these changes to chromatin dynamics dysregulate gene expression to support the epithelial-to-mesenchymal transition. Functional studies using Saccharomyces cerevisiae reveal that, while yeast cells that express H3E50K as the sole copy of histone H3 show sensitivity to cellular stressors, including caffeine, H3E50K cells display some genetic interactions that are distinct from the characterized H3K36M oncohistone yeast model. Taken together, these data suggest that additional H3 mutations have the potential to support oncogenic activity and function through distinct mechanisms that dysregulate gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ab101c8206ab8a7a20f301f0b33f3fd7807cac2" target='_blank'>
              Histone H3E50K remodels chromatin to confer oncogenic activity and support an EMT phenotype
              </a>
            </td>
          <td>
            K. Sad, Dorelle V. Fawwal, Celina Y Jones, Emily J Hill, Katie T. Skinner, Miranda Adams, Severin Lustenberger, Richard S Lee, Sandhya V Lohano, Satvik R Elayavalli, Jonathan Farhi, Christina C Mehta, Laramie D. Lemon, Milo B Fasken, Andrew L Hong, Steven A. Sloan, Anita H Corbett, Jennifer Spangle
          </td>
          <td>2025-01-15</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/560c8effcee249e562abe69a29bf3bcf7c2bff96" target='_blank'>
              Unravelling single-cell DNA replication timing dynamics using machine learning reveals heterogeneity in cancer progression
              </a>
            </td>
          <td>
            Joseph M. Josephides, Chun-Long Chen
          </td>
          <td>2025-02-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Kinase-related gene fusion and point mutations play pivotal roles as drivers in cancer, necessitating optimized, targeted therapy against these alterations. The efficacy of molecularly targeted therapeutics varies depending on the specific alteration, with great success reported for such therapeutics in the treatment of cancer with kinase fusion proteins. However, the involvement of actionable alterations in solid tumors, especially regarding kinase fusions, remains unclear. Therefore, in this study, we aimed to compare the number of actionable alterations in patients with tyrosine or serine/threonine kinase domain fusions, mutations, and copy number alterations (CNAs). We analyzed 613 patients with 40 solid cancer types who visited our division between June 2020 and April 2024. Furthermore, to detect alterations involving multiple-fusion calling, we performed comprehensive genomic sequencing using FoundationOne® companion diagnostic (F1CDx) and FoundationOne® Liquid companion diagnostic (F1LCDx). Patient characteristics and genomic profiles were analyzed to assess the frequency and distribution of actionable alterations across different cancer types. Notably, 44 of the 613 patients had fusions involving kinases, transcriptional regulators, or tumor suppressors. F1CDx and F1LCDx detected 13 cases with kinase-domain fusions. We identified 117 patients with kinase-domain mutations and 58 with kinase-domain CNAs. The number of actionable alterations in patients with kinase-domain fusion, mutation, or CNA (median [interquartile range; IQR]) was 2 (1–3), 5 (3–7), and 6 (4–8), respectively. Patients with kinase fusion had significantly fewer actionable alterations than those with kinase-domain mutations and CNAs. However, those with fusion involving tumor suppressors tended to have more actionable alterations (median [IQR]; 4 [2–9]). Cancers with kinase fusions exhibited fewer actionable alterations than those with kinase mutations and CNAs. These findings underscore the importance of detecting kinase alterations and indicate the pivotal role of kinase fusions as strong drivers of cancer development, highlighting their potential as prime targets for molecular therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5249feb71c9f79a3c5d96e6debe6951cd92ec6" target='_blank'>
              Comparison of actionable alterations in cancers with kinase fusion, mutation, and copy number alteration
              </a>
            </td>
          <td>
            S. Suzuki, T. Akahane, A. Tanimoto, M. Higashi, I. Kitazono, M. Kirishima, M. Nishigaki, T. Ikeda, S. Kanemitsu, J. Nakazawa, E. Akahane, H. Nishihara, Kimiharu Uozumi, M. Yoshimitsu, K. Ishitsuka, S.-i. Ueno
          </td>
          <td>2025-01-23</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0977541bd5b827628ce81b0938c0df52c0a295" target='_blank'>
              Proteogenomic analysis reveals adaptive strategies for alleviating the consequences of aneuploidy in cancer.
              </a>
            </td>
          <td>
            Jan-Eric Bökenkamp, Kristina Keuper, Stefan Redel, Karen Barthel, Leah Johnson, Amelie Becker, A. Wieland, M. Räschle, Zuzana Storchová
          </td>
          <td>2025-02-10</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Mutations in the TP53 gene and chromosomal instability (CIN) are two of the most common alterations in cancer. CIN, marked by changes in chromosome numbers and structure, drives tumor development, but is poorly tolerated in healthy cells, where developmental and tissue homeostasis mechanisms typically eliminate cells with chromosomal abnormalities. Mechanisms that allow cancer cells to acquire and adapt to CIN remain largely unknown. Tumor suppressor protein p53, often referred to as the “guardian of the genome”, plays a critical role in maintaining genomic stability. In cancer, CIN strongly correlates with TP53 mutations, and recent studies suggest that p53 prevents the propagation of cells with abnormal karyotypes arising from mitotic errors. Furthermore, p53 dysfunction is frequent in cells that underwent whole-genome doubling (WGD), a process that facilitates CIN onset, promotes aneuploidy tolerance, and is associated with poor patient prognosis across multiple cancer types. This review summarizes current insights into p53’s role in protecting cells from chromosome copy number alterations and discusses the implications of its dysfunction for the adaption and propagation of cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee772438cd2bfb17d973a17539b1ffdae477941" target='_blank'>
              The Role of p53 Mutations in Early and Late Response to Mitotic Aberrations
              </a>
            </td>
          <td>
            Anna Hertel, Zuzana Storchová
          </td>
          <td>2025-02-08</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Deletion of the long q arm of chromosome 22 (22qDEL) is the most frequently identified recurrent somatic copy number alteration observed in papillary thyroid carcinoma (PTC). Since its role in PTC is not fully understood, we conducted a pooled analysis of genomic characteristics and clinical correlates in 1094 primary tumors from four published PTC genomic studies. The majority of PTC cases with 22qDEL exhibited arm-level loss of heterozygosity (86%); nearly all PTC cases with 22qDEL had losses in 22q12 and 13, which together constitute 70% of the q arm. Our analysis confirmed that 22qDEL occurs more frequently with RAS point mutations (50.4%), particularly HRAS (70.3%), compared with other PTC drivers (9.3%), supporting the conclusion that 22qDEL is unlikely to be a solitary driver of PTC but possibly an important co-factor in carcinogenesis, particularly in PTCs with RAS driver mutations. Differential RNA expression analyses revealed downregulation of most genes located on chromosome 22 in cases with 22qDEL compared to those without 22qDEL. Many differentially expressed genes are drawn from immune response and regulation pathways. These findings highlight the value of further investigations into the contributions of 22qDEL events to PTC, perhaps mediated through immune perturbations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e0fd29e42888016677b1946cca20fdf0013b09" target='_blank'>
              Somatic copy number deletion of chromosome 22q in papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Olivia W Lee, D. Karyadi, Stephen W Hartley, Weyin Zhou, M. Machiela, Shahriar A. Zamani, L. Zurnadzhy, John N Weinstein, Young Joo Park, Jeong-Sun Seo, Gerry A Thomas, Tetiana I Bogdanova, M. Tronko, Lindsay M. Morton, S. Chanock
          </td>
          <td>2025-01-01</td>
          <td>European Thyroid Journal</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Genome-wide hypertranscription is common in human cancer and predicts poor prognosis. To understand how hypertranscription might drive cancer, we applied our formalin-fixed paraffin-embedded (FFPE)-cleavage under targeted accessible chromatin method for mapping RNA polymerase II (RNAPII) genome-wide in FFPE sections. We demonstrate global RNAPII elevations in mouse gliomas and assorted human tumors in small clinical samples and discover regional elevations corresponding to de novo HER2 amplifications punctuated by likely selective sweeps. RNAPII occupancy at S-phase-dependent histone genes correlated with WHO grade in meningiomas, accurately predicted rapid recurrence, and corresponded to whole-arm chromosome losses. Elevated RNAPII at histone genes in meningiomas and diverse breast cancers is consistent with histone production being rate-limiting for S-phase progression and histone gene hypertranscription driving overproliferation and aneuploidy in cancer, with general implications for precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d46087a5dee58a70e59310b431eba431b7c66c3" target='_blank'>
              RNA polymerase II at histone genes predicts outcome in human cancer.
              </a>
            </td>
          <td>
            S. Henikoff, Ye Zheng, Ronald M. Paranal, Yiling Xu, Jacob E. Greene, J. Henikoff, Zachary R. Russell, Frank Szulzewsky, H. N. Thirimanne, S. Kugel, Eric C. Holland, K. Ahmad
          </td>
          <td>2025-01-02</td>
          <td>Science</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="This study delves into the intriguing world of extrachromosomal DNA (ecDNA) in breast cancer, uncovering its pivotal role in cancer's aggressiveness and genetic variability. ecDNA, a form of circular DNA found outside chromosomes, is known to play a significant role in cancer progression by increasing oncogene expression. Focusing on two contrasting cell lines, MDA-MB-231 (triple-negative) and MCF-7 (Luminal-A), we utilized advanced microscopy and fluorescence techniques to detect and characterize ecDNA. Our findings reveal a stark difference: MDA-MB-231 cells, known for their high metastatic potential, exhibit a striking abundance of ecDNA, manifested as double minutes and single form with intense fluorescence signals. In contrast, the less aggressive MCF-7 cells harbor significantly fewer ecDNA. This disparity highlights the potential of ecDNA as a key player in cancer progression and a promising target for novel therapies. This research sheds light on the unseen genetic forces driving breast cancer and opens the door to new strategies in cancer treatment. Further research is necessary to understand the mechanisms of ecDNA formation and its role in different breast cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fb924ca3420181045f4a1ed603d86cd06599e30" target='_blank'>
              Extrachromosomal DNA in Breast Cancer Cell Lines: Detection and Characterization.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, Fadhillah Fadhillah, Retno Lestari, Fadilah Fadilah, A. Bowolaksono, A. Dwiranti
          </td>
          <td>2024-12-29</td>
          <td>Microscopy research and technique</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Many tumors evolve through cellular mutation and selection, where subpopulations of cells (subclones) with shared ancestry compete for dominance. Introduction of next generation sequencing enables subclone identification using small somatic. However, there are several advantages to marking subclones with structural variants: they have greater functional impact, play a crucial role in late-stage tumors, and provide a more complete view of genomic instability driving tumor evolution. Here, we present SVCFit, a scalable method to estimate the cellular prevalence of somatic deletions, duplications and inversions. We demonstrate that cellular prevalence estimation can be improved by incorporating distinct read patterns for each structural variant type. Additionally, this improvement is achieved without prior knowledge of tumor purity, which is often inaccurate. Using a combination of simulated data and patient-derived metastatic samples with known mixture proportions, we show that our algorithm achieves significantly greater accuracy than state-of-the-art in estimating the structural variants cellular prevalence (p<0.05). The speed of SVCFit estimation from cost-effective bulk whole-genome sequencing (WGS) makes it well-suited for analyzing large cohorts of sequenced tumor samples, enhancing the accessibility of SV-based clonal reconstruction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77e71e2d20d9e52879b8c2d8cc5014dd16b7a335" target='_blank'>
              SVCFit: Inferring structural variant cellular fraction in tumors
              </a>
            </td>
          <td>
            Yunzhou Liu, Jiaying Lai, Laura D. Wood, Rachel Karchin
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Immunotherapy has been a promising treatment for various cancer types. This therapy heavily relies on the efficiency of the targeted neoantigens for T cell recognition and downstream immune responses. Neoantigens derived from tumor-specific nonsynonymous somatic mutations have been a great resource and manifested remarkable therapeutic effects, yet with high patient specificity and high dependency on mutation rate. Here, we propose a novel strategy for discovering neoantigens derived from epigenetic mutations, rather than genetic mutations, from transposable elements (TEs) sequences. TEs make up 50% of the human genome but have been long considered to be “junk DNA” and therefore overlooked in medical genomics. However, recent works from our lab have shown that, despite being epigenetically silenced in somatic cells, certain TEs contribute significantly to the evolution of regulatory networks and shape the cancer transcriptome landscape, including functioning as promoters and producing novel, tumor-specific TE-derived transcripts (TSTETs) that can be translated into proteins. Given the plasticity of epigenome, neoantigens resulted from epigenetic mutations could be pan-cancer with high recurrence. Thus, we hypothesize that cryptic promoters embedded in TEs can be re-activated due to epigenetic dysregulation in cancer, leading to the generation of highly recurrent, pan-cancer, TE-derived, tumor-specific protein products that could be employed as neoantigens for immuno-therapy. In this study, we developed a computational pipeline termed TE Promoter Finder 3 (TEProf3) to precisely identify TE-derived promoters and transcripts genome-wide from transcriptomic data, including data from both long-read and short-read sequencing. We applied TEProf3 to analyze 12,015 tumor samples encompassing 33 cancer types from The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) datasets. Our analysis revealed 14,417 of TSTETs and hundreds of tumor-specific TE-derived antigens (TSTEAs) from the tumor transcriptomic and mass spectrometry data, respectively. Notably, most TSTETs exhibited high recurrence rates, with over 10% recurrence in the majority and some exceeding 90% recurrence. Each tumor sample displayed a median number of 19 TSTETs. Certain cancer types showed a comparable landscape of tumor-specific TE-derived promoters, potentially due to similarities in epigenetic mutations and cell types of origin. The number of TSTETs correlated with tumor progression in some cancer types, suggesting a link between progressive dysregulation of TE-derived promoters and tumorigenesis. We further assessed the efficiency of antigen presentation in cancer cell lines as a proof of concept. We successfully identified T cell receptors sequences from CD8+ T cells that exhibited strong binding affinity for TSTEAs and demonstrated cytotoxicity effects on the cancer cell lines. In summary, this work represents one of the first comprehensive catalogs of TSTEAs, providing a valuable resource for future immunotherapy strategies.
 Citation Format: Yonghao Liang, Xiaoyun Xing, Ting Wang. Transposable elements activation produces pan-cancer tumor-specific neoantigens [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The ribosomal genes (rDNA genes) encode 47S rRNA which accounts for up to 80% of all cellular RNA. At any given time, no more than 50% of rDNA genes are actively transcribed, and the other half is silent by forming heterochromatin structures through DNA methylation. In cancer cells, upregulation of ribosome biogenesis has been recognized as a hallmark feature, thus, the reduced methylation of rDNA promoter has been thought to support conformational changes of chromatin accessibility and the subsequent increase in rDNA transcription. However, an increase in the heterochromatin state through rDNA hypermethylation can be a protective mechanism teetering on the brink of a threshold where cancer cells rarely successfully proliferate. Hence, clarifying hypo- or hypermethylation of rDNA will unravel its additional cellular functions, including organization of genome architecture and regulation of gene expression, in response to growth signaling, cellular stressors, and carcinogenesis. Using the bisulfite-based quantitative real-time methylation-specific PCR (qMSP) method after ensuring unbiased amplification and complete bisulfite conversion of the minuscule DNA amount of 1 ng, we established that the rDNA promoter was significantly hypermethylated in 107 breast, 65 lung, and 135 colon tumour tissue samples (46.81%, 51.02% and 96.60%, respectively) as compared with their corresponding adjacent normal samples (26.84%, 38.26% and 77.52%, respectively; p < 0.0001). An excessive DNA input of 1 μg resulted in double-stranded rDNA remaining unconverted even after bisulfite conversion, hence the dramatic drop in the single-stranded DNA that strictly required for bisulfite conversion, and leading to an underestimation of rDNA promoter methylation, in other words, a faulty hypomethylation status of the rDNA promoter. Our results are in line with the hypothesis that an increase in rDNA methylation is a natural pathway protecting rDNA repeats that are extremely sensitive to DNA damage in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e41a5d7f178f6e61d7cd45d8e11b2f922f3fa334" target='_blank'>
              Hypermethylation at 45S rDNA promoter in cancers
              </a>
            </td>
          <td>
            Trang Thi Quynh Tran, Trang Hien Do, Tung The Pham, Phương Thi Thu Luu, Oanh Minh Pham, Uyen Quynh Nguyen, L. Vuong, Quang Ngoc Nguyen, Tuan Van Mai, Son Van Ho, Than Thi Nguyen, Lan Thi Thuong Vo
          </td>
          <td>2025-01-07</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Squamous cancers are the most frequently diagnosed solid tumours world-wide. The closely related transcriptional co-regulators YAP and TAZ(WWTR1) have emerged as important drivers of tumour initiation and progression in various types of squamous cell carcinoma. How YAP and TAZ execute their oncogenic functions in squamous cancers is still not fully understood. Here, we report that in addition to controlling transcriptional programmes that determine the balance between proliferation and terminal differentiation, YAP also engages with RIF1, a key regulator of DNA replication timing and protector of stalled replication forks under replication stress. YAP modulates gene expression of RIF1 via TEAD transcription factors and also stabilizes RIF1 protein on the chromatin. YAP-RIF1 complex formation increases in response to exogenous replication stress, and treatment with hydroxyurea to induce replication fork stalling acts synergistically with YAP depletion to impair cell proliferation. Our results thus demonstrate that YAP’s oncogenic functions in squamous cell carcinoma involve both transcriptional and non-transcriptional mechanisms, and that RIF1 is key for squamous cell carcinoma cells to survive with their high levels of endogenous replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3985e13774d5b1194193c3f3fcbde897bc3a759f" target='_blank'>
              YAP engages RIF1 to dampen replication stress in squamous cell carcinoma
              </a>
            </td>
          <td>
            Jodie Bojko, Benjamin Flynn, Bertram Aschenbrenner, Alexander Howard, Emily Lay, Emma Bailey, Natalia Krajic, Sandra Catalan, Kelli Gallacher, Elodie Sins, Jun Wang, Ute Jungwirth, B. Lichtenberger, Gernot Walko
          </td>
          <td>2025-01-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="HPV integration (HPVint) is associated with carcinogenesis and tumor progression in HPV-associated cancers, including head and neck squamous cell carcinomas (HNSCC). While its impact on human DNA has been well characterized, its relationship with clinical outcomes remains unconfirmed. Here we investigate the consequences of HPVint both with respect to human and HPV characteristics by analyzing 261 HPV-associated HNSCC bulk and single-cell RNA-seq samples from five cohorts, and DNA HPVint events from 102 HPV+ participants in two of the cohorts. By leveraging this large meta-cohort, we first reveal an oncogenic network based on the recurrent HPV integration locations in HNSCC. We then classify HPVint-positive (HPVint(+)) participants by HPV RNA features, specifically based on spliced HPV-human fusion transcripts and ratios of HPV gene transcripts, showing that subsets of participants have worse clinical outcomes. Our analyses, focused mainly on RNA instead of DNA, expand our understanding of the carcinogenic mechanisms of HPVint, partially addressing the conflicting findings of whether HPVint is associated with aggressive phenotypes and worse clinical consequences, and provide potential biomarkers to advance precision oncology in HPV-associated HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0657cc50cebd2411c16a35477f64b6cd706ce1d3" target='_blank'>
              HPV integration in head and neck cancer: downstream splicing events and expression ratios linked with poor outcomes
              </a>
            </td>
          <td>
            Shiting Li, Shaomiao Xia, Maria Lawas, Aishani Kulshreshtha, Bailey F Garb, AA Chamila Perera, Chen Li, Tingting Qin, Joshua D. Welch, N. D’Silva, Laura S. Rozek, Maureen A. Sartor
          </td>
          <td>2025-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Mitotic fidelity depends on proper chromosome alignment at the spindle equator, a process known as chromosome congression, driven by well-established protein networks. Whereas RNA-binding proteins and noncoding RNAs have been implicated in cell division, their interplay during this process remains unknown. Here, we discover that RSRC2, an arginine/serine-rich RNA-binding protein, plays an essential role in cell division by interacting with the long non-coding RNA C1QTNF1-AS1. The loss of either RSRC2 or C1QTNF1-AS1 results in defects in chromosome congression and mitotic progression. We show that RSRC2 interacts with distinct sets of proteins involved in splicing and centrosome biogenesis, contributing to the fidelity of cell division through two different mechanisms: one linked to the splicing of mitotic regulators and the other by localising to mitotic centrosomes for which the interaction with the C1QTNF1-AS1 RNA is required. Our study uncovers RSRC2 as a new regulator of cell division and illustrates how RNA-protein complexes promote error-free mitosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb0d898e39c9e3e90e4923dfd61d6149838f2b92" target='_blank'>
              The RNA-binding protein RSRC2 promotes mitotic fidelity by interacting with the lncRNA C1QTNF1-AS1
              </a>
            </td>
          <td>
            Kaliya Georgieva, Alice O Coomer, Parnia Babaei, Giulia Guiducci, Martin Dodel, Eleni Maniati, Anisha Thind, S. Krishnamurthy, Anna Nawrocka, Sam Wallis, Alena Shkumatava, Jun Wang, F. Mardakheh, Lovorka Stojic
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 DNA methylation alterations are observed across multiple cancers, occurring early during tumor initiation. However, evolution and drivers of these changes remain poorly understood and difficult to assess at the clonal level. Consequently, we developed a method for profiling genome-wide DNA methylation from low DNA amounts and applied it to the organoid model of BRAFV600E oncogene-induced colon cancer (CC) tumorigenesis. Single organoid clones, derived from single stem cells, were profiled every month for 5 months to map nucleotide-level dynamic heterogeneity in methylation changes. Uninduced control organoids exhibit a pattern of epigenetic drift leading towards clonal convergence not until 5 months. Whereas BRAFV600E-induced organoids exhibit a step-wise divergence throughout the different time points, likely explained by progressive clonal takeovers. The methylation changes in BRAFV600E induced organoids are explained by the age-in-culture associated methylation changes; especially BRAFV600E did not induce epigenetic changes, as previously thought. The CpG sites affected in the control and BRAFV600E comparison were not shared but mapped to the same genomic elements, indicating common drivers of methylation changes regardless of oncogene. Investigation of promoter CpG-island methylation gains revealed that they are decoupled from replication. External validation in other pre-cancerous and CC datasets, and experimental cell line data sets, show that methylation gains at regulatory elements are a function of “biological aging” and independent of replication or cell division cycles. Our studies provide new insights into the early epigenetic changes involved in BRAFV600E driven carcinogenesis rooted in the age-dependent drivers of epigenetic alterations.
 Citation Format: Sara-Jayne Thursby, Zhicheng Jin, Nibedita Patel, Yong Tao, Yuba Bhandari, Daniel Petkovich, Shangzhi Lu, Lijing Yang, Tza-Huei Wang, Thomas R. Pisanic II, Stephen B Baylin, Hariharan Easwaran. Promoter methylation gains in aging and cancer are independent of replication [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B004.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/082f1b7fdd5a838908f885cbac828f34e7906f35" target='_blank'>
              Abstract B004: Promoter methylation gains in aging and cancer are independent of replication
              </a>
            </td>
          <td>
            Sara-Jayne Thursby, Zhicheng Jin, Nibedita Patel, Yong Tao, Yuba Bhandari, Daniel Petkovich, Shangzhi Lu, Lijing Yang, Tza-Huei Wang, T. R. Pisanic, Stephen B. Baylin, H. Easwaran
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Abstract Background The causes of breast cancer are poorly understood. A potential risk factor is Epstein-Barr virus (EBV), a lifelong infection nearly everyone acquires. EBV-transformed human mammary cells accelerate breast cancer when transplanted into immunosuppressed mice, but the virus can disappear as malignant cells reproduce. If this model applies to human breast cancers, then they should have genome damage characteristic of EBV infection. Objective This study tests the hypothesis that EBV infection predisposes one to breast cancer by causing permanent genome damage that compromises cancer safeguards. Methods Publicly available genome data from approximately 2100 breast cancers and 25 ovarian cancers were compared to cancers with proven associations to EBV, including 70 nasopharyngeal cancers, 90 Burkitt lymphomas, 88 diffuse large B-cell lymphomas, and 34 gastric cancers. Calculation algorithms to make these comparisons were developed. Results Chromosome breakpoints in breast and ovarian cancer clustered around breakpoints in EBV-associated cancers. Breakpoint distributions in breast and EBV-associated cancers on some chromosomes were not confidently distinguished (P>.05), but differed from controls unrelated to EBV infection. Viral breakpoint clusters occurred in high-risk, sporadic, and other breast cancer subgroups. Breakpoint clusters disrupted gene functions essential for cancer protection, which remain compromised even if EBV infection disappears. As CRISPR (clustered regularly interspaced short palindromic repeats)–like reminders of past infection during evolution, EBV genome fragments were found regularly interspaced between Piwi-interacting RNA (piRNA) genes on chromosome 6. Both breast and EBV-associated cancers had inactivated genes that guard piRNA defenses and the major histocompatibility complex (MHC) locus. Breast and EBV-associated cancer breakpoints and other variations converged around the highly polymorphic MHC. Not everyone develops cancer because MHC differences produce differing responses to EBV infection. Chromosome shattering and mutation hot spots in breast cancers preferentially occurred at incorporated viral sequences. On chromosome 17, breast cancer breakpoints that clustered around those in EBV-mediated cancers were linked to estrogen effects. Other breast cancer breaks affected sites where EBV inhibits JAK-STAT and SWI-SNF signaling pathways. A characteristic EBV-cancer gene deletion that shifts metabolism to favor tumors was also found in breast cancers. These changes push breast cancer into metastasis and then favor survival of metastatic cells. Conclusions EBV infection predisposes one to breast cancer and metastasis, even if the virus disappears. Identifying this pathogenic viral damage may improve screening, treatment, and prevention. Immunizing children against EBV may protect against breast, ovarian, other cancers, and potentially even chronic unexplained diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31f55b81eb64a7ed0044ee061bd1cada36b4674e" target='_blank'>
              Identifying Safeguards Disabled by Epstein-Barr Virus Infections in Genomes From Patients With Breast Cancer: Chromosomal Bioinformatics Analysis
              </a>
            </td>
          <td>
            B. Friedenson
          </td>
          <td>2023-07-10</td>
          <td>JMIRx Med</td>
          <td>3</td>
          <td>1</td>
        </tr>

        <tr id="Cohesin folds genomes into chromatin loops, whose roles are under debate. We report that double strand breaks (DSB) induce de novo formation of chromatin loops, with the break positioned at the loop base. These loops form only in S/G2 phases and occur during repair via homologous recombination (HR), concomitant with DNA end resection and RAD51 assembly. RAD51 showed two-tiered accumulation around DSBs, with a broad (~Mb) domain arising from the homology search. This domain is regulated by cohesin unloader, is constrained by TAD boundaries, and it overlaps with chromatin regions reeled through the break-anchored loop, suggesting that loop extrusion regulates the homology search. Indeed, depletion of NIPBL results in reduced HR, and this effect is more pronounced when the HR donor is far (~100 kb) from the break. Our data indicates that loop-extruding cohesin promotes the mammalian homology search by facilitating break-chromatin interactions within the damaged TAD. One-Sentence Summary High spatiotemporal resolution analysis of double strand beak repair in 3D genome revealed the role of cohesin-driven loop extrusion in the homology search.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e040dd0709d7779dc334b56e7eb34dab3396724" target='_blank'>
              Cohesin drives chromatin scanning during the RAD51-mediated homology search
              </a>
            </td>
          <td>
            Alberto Marin-Gonzalez, Adam Rybczynski, Namrata Nilavar, Daniel Nguyen, Violetta Karwacki-Neisius, Andrew G. Li, Roger S. Zou, Franklin J. Avilés-Vázquez, Masato T. Kanemaki, Ralph Scully, T. Ha
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13edecae9b7bef7cd629efeb486e0288bd13530b" target='_blank'>
              Tumour hypoxia in driving genomic instability and tumour evolution.
              </a>
            </td>
          <td>
            Alexandru Suvac, Jack Ashton, Robert G Bristow
          </td>
          <td>2025-01-28</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Recently revised brain tumor classification suggested a glioma treatment strategy that takes into consideration molecular variants in IDH1 and TP53 marker genes. While pathogenic variants of IDH1 and TP53 can be accompanied by chromosomal instability (CIN), the impact of IDH1 and TP53 mutations on genome stability remains unstudied. Elevated CIN might provide therapeutic targets, based on synergistic effects of chemotherapy with CIN-inducing drugs. Using an assay based on human artificial chromosomes, we investigated the impact of common glioma missense mutations in IDH1 and TP53 on chromosome transmission and demonstrated that IDH1R132H and TP53R248Q variants elevate CIN. We next found enhanced CIN levels and the sensitivity of IDH1R132H/WT and TP53R248Q/R248Q genotypes, introduced into U87 MG glioma cells by CRISPR/Cas9, to different drugs, including conventional temozolomide. It was found that U87 MG cells carrying IDH1R132H/WT exhibit dramatic sensitivity to paclitaxel, which was independently confirmed on cell cultures derived from patients with naturally occurring IDH1R132H/WT. Overall, our results suggest that the development of CIN-enhancing therapy for glioma tumors with the IDH1R132H/WT genotype could be advantageous for adjuvant treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641df7ee46092e60a05501256dd4211c20f30f9f" target='_blank'>
              Therapy enhancing chromosome instability may be advantageous for IDH1R132H/WT gliomas
              </a>
            </td>
          <td>
            Nikolay V Goncharov, Ivan Baklanov, Valeriia S Gulaia, Anastasiia P Shuliak, Daria V Lanskikh, Valeriia Zhmenia, M.E. Shmelev, N. Shved, Jing Wu, Mikhail Liskovykh, Vladimir Larionov, N. Kouprina, Vadim Kumeiko
          </td>
          <td>2025-02-13</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0491183160fe7bbdae2e8ae2e68e3e5bf8212367" target='_blank'>
              Intratumor heterogeneity of HPV integration in HPV-associated head and neck cancer
              </a>
            </td>
          <td>
            Noa Sasa, T. Kishikawa, Masashi Mori, R. Ito, Yumie Mizoro, Masami Suzuki, Hirotaka Eguchi, Hidenori Tanaka, T. Fukusumi, Motoyuki Suzuki, Y. Takenaka, K. Nimura, Yukinori Okada, H. Inohara
          </td>
          <td>2025-01-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Breast cancer is the most prevalent neoplasm in women. ER+ (Luminal subtype), representing over 70% of breast tumors, is a genetically diverse group. Structural and Numerical-Chromosomal instability initiates tumor development and is recognized as the primary driver of genetic alteration in luminal breast tumors. Genomic instability refers to the increased tendency of cancer cells to accumulate genomic alterations during cell proliferation. The cell cycle check-point response to constant and stable genomic alterations in tumor cells drives this process. The impact of CNV patterns and aneuploidies in cell cycle and proliferation perturbation has recently been highlighted by scientists in Luminal breast tumors. The impact of chromosomal instability on cancer therapy and prognosis is not a new concept. Still, the degree of emerging genomic instability leads to prognosis alteration following cell cycle deregulation by chromosomal instability could be predicted by CNVs-based reclassification of breast tumors. In this review, we try to explain the effect of CIN in the cell cycle that ended with genomic instability and altered prognosis and the impact of CIN in decision-making for a therapy strategy for patients with luminal breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bef9c6cdff104c948a88f734e3a20c0747386c" target='_blank'>
              A new insight into the impact of copy number variations on cell cycle deregulation of luminal-type breast cancer
              </a>
            </td>
          <td>
            Amir Mahdi Khamaneh, Davoud Jafari-Gharabaghlou, Khalil Ansarin, Pouya Pazooki, Zahra Akbarpour, B. Naghili, Nosratollah Zarghami
          </td>
          <td>2025-02-12</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="X-chromosome inactivation (XCI) is initiated during early mammalian embryogenesis by a long non-coding RNA (lncRNA) XIST, which coats one of the two X-chromosomes and facilitates epigenetic transcriptional silencing. A second, evolutionarily recent primate-specific lncRNA XACT was proposed to antagonize XIST’s ability to induce XCI. XACT expression is restricted to pluripotent states and early embryonic stages and coats the active X-chromosome in both females and males. Here, we report a novel XACT transcript expressed in normal and cancerous somatic cells from both the inactive (Xi) and active (Xa) X chromosomes. It coexists with XIST on the Xi without affecting XIST expression or (re)activating X-linked genes inactivated by XCI. During hematopoietic stem cell (HSC) differentiation, XACT is primarily expressed in myeloid progenitors in both sexes. XACT expression is activated in HSCs and peaks in megakaryocyte-erythrocyte progenitor cells (MEPs) before rapidly declining as the MEPs differentiate into megakaryocytes or erythrocytes. By combining CRISPR-based XACT perturbation with epigenomic and transcriptional studies, we revealed the key role of XACT in the self-renewal and differentiation of erythroid progenitors into erythrocytes, by recruiting cis-regulatory proteins and regulating transcription through ETS and AP-1 transcription factors. Furthermore, XACT is expressed in a subset of acute myeloid leukemia (AML) patients, with high levels found in erythroid-megakaryocytic blast cells; suggesting XACT’s potential as a marker for AML subtypes. Thus, we identified a novel XACT transcript with a unique expression profile and important roles in normal and cancer cells, revealing XACT lncRNA functions beyond its previously proposed role in XCI during embryogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7754eb09126d1d587b2765e70f76183499d8bcbf" target='_blank'>
              A Novel XACT lncRNA Transcript with Functions Transcending X-Chromosome Inactivation
              </a>
            </td>
          <td>
            Peifen Zhang, Paula Haro Angles, Hong Zhang, Tingyue Li, Roger Mulet-Lazaro, W. V. van Ijcken, S. Philipsen, Ruud Delwel, J. Gribnau, D. Huylebroeck, Frank Grosveld, C. Rougeulle, Eskeatnaf Mulugeta
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="DNA double strand breaks (DSBs) are widely considered the most cytotoxic DNA lesions occurring in cells because they physically disrupt the connectivity of the DNA double helix. Homologous recombination (HR) is a high-fidelity DSB repair pathway that copies the sequence spanning the DNA break from a homologous template, most commonly the sister chromatid. How both DNA ends, and the sister chromatid are held in close proximity during HR is unknown. Here we demonstrate that the PST repeat region of MDC1 is a mutlivalent nucleosome binding domain, sufficient to tether chromatin in multiple contexts. In mitotic cells the affinity of the PST repeats for chromatin is downregulated by phosphorylation to prevent chromosome missegregation, while still contributing to DNA break tethering by the MDC1-TOPBP1-CIP2A complex. In interphase, the PST repeat region is critical for RAD51 focus formation but not the recruitment of 53BP1 to DNA breaks, consistent with a chromatin tethering function. In total, this work demonstrates that the PST repeat region of MDC1 is a multivalent chromatin binding domain with tunable affinity that contributes to DNA break tethering during HR and in mitosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dbf6e7c3f0e969876e0503353c0c93165dcec37" target='_blank'>
              The PST repeat region of MDC1 is a tunable multivalent chromatin tethering domain
              </a>
            </td>
          <td>
            Joshua R. Heyza, Maria Mikhova, Gloria I Perez, David G. Broadbent, Jens C. Schmidt
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Homologous recombination maintains genome stability by repairing double strand breaks and protecting replication fork stability. Defects in homologous recombination results in cancer predisposition but can be exploited due to increased sensitivity to certain chemotherapeutics such as PARP inhibitors. The NEK8 kinase has roles in the replication response and homologous recombination. NEK8 is overexpressed in breast cancer, but the impact of NEK8 overexpression on homologous recombination has not been determined. Here, we demonstrate NEK8 overexpression inhibits RAD51 focus formation resulting in a defect in homologous recombination and degradation of stalled replication forks. Importantly, NEK8 overexpression sensitizes cells to the PARP inhibitor, Olaparib. Together, our results suggest NEK8 overexpressing tumors may be recombination-deficient and respond to chemotherapeutics that target defects in recombination such as Olaparib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8173c08c1d4d30ea1a0d792e303a9db46e92a70" target='_blank'>
              Overexpression of the NEK8 kinase inhibits homologous recombination
              </a>
            </td>
          <td>
            Joshua L. Turner, Georgia Moore, Tyler J. McCraw, Jennifer M. Mason
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: The human FOXM1 transcription factor controls cell cycle progression and genome stability, and it has been correlated to the onset and progression of many tumor types. Methods: In our study, we collected all recent sequence and quantitative transcriptomics data about FOXM1, testing its presence across vertebrate evolution and its upregulation in cancer, both in bulk tissue contexts (by comparing the TCGA tumor dataset and the GTEx normal tissue dataset) and in single-cell contexts. Results: FOXM1 is significantly and consistently upregulated in all tested tumor types, as well as in tumor cells within a cancer microenvironment. Its upregulation reverberates in the upregulation of its target genes and can be used as a biomarker for poor cancer outcome in at least four tumor types. Conclusions: Despite its lack of cancer-related mutations and amplifications, the recurring upregulation of FOXM1 in all tumors puts a focusing lens on this gene as a candidate pan-cancer master regulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e19cd5cddb455ecb73607992ac0679d96d8dc34" target='_blank'>
              Pan-Cancer Upregulation of the FOXM1 Transcription Factor
              </a>
            </td>
          <td>
            Daniele Pozzobon, Arianna Bellezza, F. Giorgi
          </td>
          <td>2025-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="The SNF2 family chromatin remodeler HELLS has emerged as an important regulator of cell proliferation, genome stability, and several cancer pathways. Significant upregulation of HELLS has been reported in 33 human cancer types. While HELLS has been implicated in DNA damage response, its function in DNA repair is poorly understood. Here we report a new regulatory link between HELLS and single-strand break (SSB) repair in cellular responses to DNA alkylation damage. We found that loss of HELLS impairs SSB repair, and selectively sensitizes cells to DNA alkylating agents and PARP inhibitors (PARPi). Furthermore, we found that HELLS is co-expressed with PARP1 in cancer cells, and its loss is synthetic lethal with homologous recombination deficiency (HRD). This work unveils new functions of HELLS in modulating SSB repair and responses to clinically relevant DNA alkylation damage, thus offering new insights into the potential therapeutic value of targeting HELLS in cancer. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd4d063ed0793e8f2d3e58d28ea55c516edbff0" target='_blank'>
              Chromatin regulator HELLS mediates SSB repair and responses to DNA alkylation damage
              </a>
            </td>
          <td>
            Joyous T Joseph, Christine M Wright, E. Peixoto, Asad Khan, Yong Li, Jason S Romero Neidigk, Olivia Decker, Azait Imtiaz, Brianna A Jones, Yangfeng Zhang, Etsuko Shiabata, S. Gradilone, Zachary A. Lewis, Rafael Contreras-Galindo, Arko Sen, A. Dutta, Wioletta Czaja
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Embryonal Tumors with Multilayered Rosettes (ETMRs) are relatively rare but aggressive embryonal brain tumors that mainly affect children under the age of 4 and which have a poor outcome. Around 90% of ETMRs harbor amplification of the C19MC miRNA cluster, in most cases fused to the TTHY1 gene. The second most recurrent aberration, present in about half of the C19MC negative cases, are bi-allelic mutations in DICER1. ETMRs are also characterized by overall lower DNA methylation levels and high LIN28A gene expression as compared with other embryonal brain tumors like medulloblastomas (MB) and atypical teratoid rhabdoid tumors (ATRT). However, a more detailed description of the epigenetic landscape marked by DNA methylation and histone modifications, and how this affects the ETMR transcriptome is still lacking. To decode the epi-genome, we have collected whole genome bisulfite sequencing (WGBS) data, ChIP sequencing (ChIP-seq) data of various (active and repressive) histone marks and epigenetic modifiers (EZH2, BRD4, CTCF), and RNA sequencing (RNA-seq) data from a series of ETMR patient tumors, and similar data from a series of MBs and ATRTs covering all molecular subtypes for comparison reasons. Using all these data and performing bioinformatic analyses, we have depicted chromatin states and DNA-methylation-defined genomic segments of ETMR for the first time. ETMR genomes are generally hypomethylated, with wider distributions of un-methylated regions and DNA methylation valleys (DMVs) on its genome compared with MB and ATRT. By integrating these data with all other omics data, we found that BRD4-regulated overexpression of TET demethylases may cause global hypomethylation in ETMRs. Polycomb and TETs are recruited at DMVs to ensure DMVs are longer in ETMRs than MBs and ATRTs. Furthermore, genes involved in embryo development and cell proliferation are enriched in entity-shared DMVs and are highly expressed in ETMRs and other embryonal brain tumors. We also depicted that TET family members, BRD4, and Polycomb complex (EZH2) are potential therapeutic targets for ETMR patients. In conclusion, we have partially described the epi-genome and epigenetic regulations of ETMR, and are currently working on more detailed regulation axes and performing drug test experiments for these potential therapeutic targets to understand better the underlying biology of these tumors and how this may lead to alternative and more effective therapies.
 Citation Format: Shanzheng Wang, Sander Lambo, Monika Mauermann, Robert J Autry, Natalie Jäger, Stefan M Pfister, Marcel Kool. Epigenetic landscape and gene regulations in embryonal brain tumors with multilayered rosettes (ETMR) [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B028.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0e28f4d695edfbb9dadc1b324a1b929e51f3786" target='_blank'>
              Abstract B028: Epigenetic landscape and gene regulations in embryonal brain tumors with multilayered rosettes (ETMR)
              </a>
            </td>
          <td>
            Shanzheng Wang, S. Lambo, Monika Mauermann, Robert J Autry, N. Jäger, Stefan M. Pfister, M. Kool
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Solid tumors often have an abundance of collagen-I that stiffens the tissue, and they are invariably driven by mutations that include chromosome losses and gains. These observations are linked here by showing that 3D matrix stiffness induces heritable changes to a cell’s DNA. We use live-cell chromosome reporters (ChReporters) and hydrogels of tunable stiffness to show mitotic compression, micronuclei counts, ChReporter losses and heterogeneity all increase as functions of stiffness. Increased mistakes occur despite suppressed cell division in stiff matrix and minimal size variation between spheroids. Colonies of ChReporter-negative cells within cancer spheroids align with Luria-Delbruck’s seminal theory for heritable mutations, which predicts inter-spheroid variances that exceed Poisson statistics. Suppression of the contractility motor Myosin-II also increases chromosome loss in 3D but not 2D and does not affect spheroid growth – thus clarifying Myosin-II’s putative role as a tumor suppressor. Consistent with experiments, pan-cancer analyses of clinical data associates chromosome losses and gains with collagen-I levels and genetic variation. Stiff extracellular matrix thus drives mechano-evolution of solid tumors as a Darwin-Lamarck process with heterogeneity that complicates therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21422305f6e2cd46c2231ab78870bc6be52cefd" target='_blank'>
              Matrix stiffness induces heritable changes in chromosome numbers, consistent with solid tumor heterogeneity
              </a>
            </td>
          <td>
            Alişya A. Anlaş, Markus T. Sprenger, Mai Wang, Nicholas M. Ontko, Steven H. Phan, D.E. Discher
          </td>
          <td>2025-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genomic and epigenomic instability are defining features of cancer, driving tumor progression, heterogeneity, and therapeutic resistance. Central to this process are epigenetic echoes, persistent and dynamic modifications in DNA methylation, histone modifications, non-coding RNA regulation, and chromatin remodeling that mirror underlying genomic chaos and actively influence cancer cell behavior. This review delves into the complex relationship between genomic instability and these epigenetic echoes, illustrating how they collectively shape the cancer genome, affect DNA repair mechanisms, and contribute to tumor evolution. However, the dynamic, context-dependent nature of epigenetic changes presents scientific and ethical challenges, particularly concerning privacy and clinical applicability. Focusing on lung cancer, we examine how specific epigenetic patterns function as biomarkers for distinguishing cancer subtypes and monitoring disease progression and relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3b401f96a57f38873c48e3928dd0d7e44a2cbef" target='_blank'>
              Epigenomic Echoes—Decoding Genomic and Epigenetic Instability to Distinguish Lung Cancer Types and Predict Relapse
              </a>
            </td>
          <td>
            Alexandra A. Baumann, Zholdas Buribayev, Olaf Wolkenhauer, Amankeldi A. Salybekov, M. Wolfien
          </td>
          <td>2025-02-05</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="In most cancers, including endometrial cancer, tumor suppressor genes harboring inactivating mutations have been systematically cataloged. However, locus-specific epigenetic alterations contributing to cancer initiation and progression remain only partly described, creating knowledge gaps about functionally significant tumor suppressors and underlying mechanisms associated with their inactivation. Here, we show that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event not associated with promoter hypermethylation. PAX2 is expressed throughout normal endometrial glands; however, microscopic clones with PAX2 protein loss arise spontaneously in adulthood and progress to endometrial precancers and cancers. Here we show that PAX2 protein loss occurs via transcriptional silencing in 80% of endometrial cancers. Mechanistically, transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that silencing is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive features (H3K27me3). The spread of the H3K27me3 signal is restrained by cohesin loops, preventing transcriptional dysregulation of neighboring genes. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. Genetically engineered mouse models together with organoid and human cell line studies established Pax2 as an in vivo endometrial tumor suppressor that cooperates with Pten to produce lethal cancers. Our discovery of a specific and recurring epigenetic alteration that transcriptionally silences PAX2 to initiate most endometrial cancers opens new lines of investigation into the origins of endometrial cancer with diverse implications for its diagnosis and treatment. One Sentence Summary We demonstrate that PAX2 is silenced by a highly recurrent epigenetic mechanism to initiate most endometrial cancers, establishing a novel mechanism of tumor suppressor inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90c481e61973d015ee3841a58286f99ac21dc81f" target='_blank'>
              PAX2 is Transcriptionally Silenced by a Distinct Mechanism of Epigenetic Reprogramming to Initiate Endometrial Carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Efficient DNA lesion repair is crucial for cell survival, especially within actively transcribed DNA regions that contain essential genetic information. Additionally, DNA breaks in regions of active transcription are prone to generating insertions and deletions, which are hallmark features of cancer genomes. Cockayne syndrome protein B (CSB) is the sole ATP-dependent chromatin remodeler that is essential for coupling DNA repair pathways with transcription, leading to more efficient DNA repair in regions of active transcription. CSB is best known for its essential function in transcription-coupled nucleotide excision repair (TC-NER), a process that rapidly removes helix-distorting DNA lesions that stall RNA polymerase II, such as those created by chemotherapeutic platinum compounds and UV irradiation. In addition to NER, CSB has also been reported to couple homologous recombination to transcription. Most recently, CSB has also been shown to couple single-strand DNA break repair to transcription. In this review, we will discuss the overlapping and distinct mechanisms by which CSB couples these different DNA repair pathways to transcription. We will also discuss how these CSB functions may account for Cockayne syndrome and the emerging roles of CSB as an innovative target for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59205832acc879122f4c23f208dc50f2ad1f678e" target='_blank'>
              ATP-Dependent Chromatin Remodeler CSB Couples DNA Repair Pathways to Transcription with Implications for Cockayne Syndrome and Cancer Therapy
              </a>
            </td>
          <td>
            Rabeya Bilkis, Robert J. Lake, Hua-Ying Fan
          </td>
          <td>2025-02-07</td>
          <td>Cells</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Despite the development of HER2-targeting drugs such as trastuzumab and T-DXd, treatment resistance is a substantial challenge, often leading to relapse and distant metastasis. Tumor heterogeneity in HER2-positive breast cancer drives the evolution of resistant clones following therapeutic stress. However, the targetable drivers of anti-HER2 treatment resistance are not thoroughly identified. This study aims to use neoadjuvant-targeted therapy cohorts and a patient-derived organoid in vitro treatment model to uncover the potential targetable drivers of anti-HER2 treatment resistance. it is found that LINC01235 significantly enhances DNA replication licensing and chromosomal instability, fostering clonal expansion and evolution, and ultimately increasing resistance to therapeutic interventions. LINC01235 regulates global H3K27ac, H3K9ac, and H3K36me3 modifications, promotes H2A.Z expression in regulatory regions, and increases the accessibility of DNA licensing factors to their promoter regions. XRCC5 is identified as a key component for maintaining genomic stability, crucial for LINC01235's role in replication licensing. Furthermore, therapeutic strategies targeting LINC01235, including the use of antisense oligonucleotides or ATR inhibitors, which showed promise in overcoming treatment resistance are explored. These findings underscore the pivotal role of LINC01235 in driving resistance mechanisms and highlight novel avenues for targeted therapies to improve the outcomes of patients with HER2-positive breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fde95930ed4b5d2e1c4571f86786e616e7f2cf6" target='_blank'>
              LINC01235 Promotes Clonal Evolution through DNA Replication Licensing-Induced Chromosomal Instability in Breast Cancer.
              </a>
            </td>
          <td>
            Qi Zhang, Xuliren Wang, Zhibo Shao, Yi Zhang, Li-Ying Zhang, Ming Chen, Xujie Zhou, Han Zhu, Yue Zhou, Xinya Lu, Pei Li, W. Chi, Lun Li, Zhibo Shao, Shenglin Huang, J. Xue, Y. Chi, Jiong Wu, B. Xiu
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The impact of cancer driving mutations in regulating immunosurveillance throughout tumor development remains poorly understood. To better understand the contribution of tumor genotype to immunosurveillance, we generated and validated lentiviral vectors that create an epi-allelic series of increasingly immunogenic neoantigens. This vector system is compatible with autochthonous Cre-regulated cancer models, CRISPR/Cas9-mediated somatic genome editing, and tumor barcoding. Here, we show that in the context of KRAS-driven lung cancer and strong neoantigen expression, tumor suppressor genotype dictates the degree of immune cell recruitment, positive selection of tumors with neoantigen silencing, and tumor outgrowth. By quantifying the impact of 11 commonly inactivated tumor suppressor genes on tumor growth across neoantigenic contexts, we show that the growth promoting effects of tumor suppressor gene inactivation correlate with increasing sensitivity to immunosurveillance. Importantly, specific genotypes dramatically increase or decrease sensitivity to immunosurveillance independently of their growth promoting effects. We propose a model of immunoediting in which tumor suppressor gene inactivation works in tandem with neoantigen expression to shape tumor immunosurveillance and immunoediting such that the same neoantigens uniquely modulate tumor immunoediting depending on the genetic context. One Sentence Summary Here we uncover an under-appreciated role for tumor suppressor gene inactivation in shaping immunoediting upon neoantigen expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a2622766d9e8241bb1972eff5c441baddd14069" target='_blank'>
              Tumor suppressor genotype influences the extent and mode of immunosurveillance in lung cancer
              </a>
            </td>
          <td>
            Keren M. Adler, Haiqing Xu, Amy C. Gladstein, Valerie M. Irizarry-Negron, Maggie R. Robertson, Katherine R. Doerig, D. Petrov, M. Winslow, D. Feldser
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Structural maintenance of chromosome-1A (SMC1A) is overexpressed in various malignancies including triple-negative breast cancer (TNBC). As a core component of the cohesin complex, SMC1A was initially recognized for its involvement in chromosomal cohesion and DNA-repair pathways. However, recent studies have unveiled its pivotal role in epithelial–mesenchymal transition (EMT), metastasis, and chemo- and radio-resistance in cancer cells. In hepatocellular carcinoma, aberrant phosphorylation of SMC1A has been associated with enhanced cell proliferation and migration. Despite these insights, the precise role of SMC1A phosphorylation in breast cancer remains largely unexplored. This study represents the first investigation to test the phosphorylation status and subcellular localization of SMC1A (p-SMC1A) in breast cancer and normal breast tissues. Immunohistochemical (IHC) staining was conducted using previously validated phospho-SMC1A antibodies on a histological section and tissue microarray (TMA) comprising samples from primary, invasive, and metastatic breast cancer and normal breast tissues. Our results revealed that p-SMC1A staining intensity was lower in normal breast tissues compared to invasive or metastatic breast cancer tissues (p < 0.001). Approximately 40% of breast cancer tissue exhibited cytoplasmic/membranous localization of p-SMC1A, whereas nuclear expression was observed in normal breast tissues. Moreover, elevated phosphorylation levels were significantly associated with higher tumor grade and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4ac66fe9226183dafb37845efa13c23cb0dc9fe" target='_blank'>
              Association of Structural Maintenance of Chromosome-1A Phosphorylation with Progression of Breast Cancer
              </a>
            </td>
          <td>
            Sushma Yadav, C. Kowolik, Daniel Schmolze, Yuan Yuan, Min Lin, Arthur D. Riggs, David A. Horne
          </td>
          <td>2025-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Plants host a range of DNA elements capable of self-replication. These molecules, usually associated to the activity of transposable elements or viruses, are found integrated in the genome or in the form of extrachromosomal DNA. The activity of these elements can impact genome plasticity by a variety of mechanisms, including the generation of structural variants, the shuffling of regulatory or coding DNA sequences across the genome, and DNA endoduplication. This plasticity can dynamically alter gene expression and genome stability, ultimately affecting plant development or the response to environmental changes. While the activation of these elements is often considered deleterious to the genome, their role in creating variation is important in adaptation and evolution. Moreover, the mechanisms by which mobile DNA proliferate have been exploited for plant engineering, or contributed to understand how desirable traits can be generated in crops. In this review, we discuss the origins and the roles of mobile DNA element activity on genome plasticity and plant biology, as well as their potential function and current application in plant biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b762e6df52387f3493832fc69614173777627231" target='_blank'>
              The role of mobile DNA elements in the dynamic of plants genome plasticity.
              </a>
            </td>
          <td>
            Robyn Emmerson, M. Catoni
          </td>
          <td>2025-01-10</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9043774f4d71a3f413b248306070648bf9c170e4" target='_blank'>
              Multi-omics approaches reveal that diffuse midline gliomas present altered DNA replication and are susceptible to replication stress therapy
              </a>
            </td>
          <td>
            Anastasia E. Hains, Kashish Chetal, Tsunetoshi Nakatani, Joana G. Marques, Andreas Ettinger, C. B. Junior, Adriana Gonzalez-Sandoval, Renjitha Pillai, Mariella G. Filbin, M. Torres-Padilla, Ruslan I. Sadreyev, Capucine Van Rechem
          </td>
          <td>2024-12-20</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Genes involved in the regulation of chromatin structure are frequently disrupted in cancer, contributing to an aberrant transcriptome and phenotypic plasticity. Yet, therapeutics targeting mutant forms of chromatin-modifying enzymes have yielded only modest clinical utility, underscoring the difficulty of targeting the epigenomic underpinnings of aberrant gene regulatory networks. Here, we sought to identify novel epigenetic vulnerabilities in diffuse large B-cell lymphoma (DLBCL). Through phenotypic screens and biochemical analysis, we demonstrated that inhibition of the H3K9 demethylases KDM4A and KDM4C elicits potent, subtype-agnostic cytotoxicity by antagonizing transcriptional networks associated with B-cell identity and epigenetically rewiring heterochromatin. KDM4 demethylases associated with the KRAB zinc finger ZNF587, and their enzymatic inhibition led to DNA replication stress and DNA damage-induced cGAS-STING activation. Broad surveys of transcriptional data from patients also revealed KDM4 family dysregulation in several other cancer types. To explore this potential therapeutic avenue, we performed high-throughput small molecule screens with H3K9me3 nucleosome substrates and identified novel KDM4 demethylase inhibitors. AI-guided protein-ligand binding predictions suggested diverse modes of action for various small molecule hits. Our findings underscore the relevance of targeting fundamental transcriptional and epigenetic mechanisms for anti-cancer therapy. HIGHLIGHTS Phenotypic screens identified JIB-04 as a potent anti-cancer agent for multiple subtypes of diffuse large B-cell lymphoma JIB-04 binds and inhibits KDM4 demethylases resulting in epigenomic rewiring of heterochromatin KDM4 demethylases cooperate with KRAB zinc fingers to limit DNA replication stress, and KDM4 inhibition instigates DNA-damage and cGAS-STING activation in several human cancers High-throughput small molecule screens with semi-synthetic nucleosome substrates and AI-guided molecular docking simulations identify novel KDM4 inhibitors">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502a11d348f0facd56363bda5866813863b5fcc9" target='_blank'>
              Heterochromatin fidelity is a therapeutic vulnerability in lymphoma and other human cancers
              </a>
            </td>
          <td>
            M. Najia, Deepak K. Jha, Cheng Zhang, Benoit Laurent, Caroline Kubaczka, Arianna Markel, Christopher Li, Vivian M Morris, Allison Tompkins, Luca Hensch, Yue Qin, B. Chapuy, Yu-Chung Huang, Michael A. Morse, M. Marunde, Anup Vaidya, Zachary B. Gillespie, S. A. Howard, Trista E. North, Daniel Dominguez, Michael-Christopher Keogh, Thorsten M Schlaeger, Yang Shi, Hu Li, Margaret M. Shipp, P. Blainey, George Q. Daley
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Whole genome duplication, or polyploidy, has been implicated in driving genome instability and tumorigenesis. Recent studies suggest that polyploidy in tumors promotes cancer genome evolution, progression, and chemoresistance resulting in worse prognosis of survival. The mechanisms by which whole genome duplications confer genome instability are not yet fully understood. In this study, we use Schizosaccharomyces pombe (fission yeast) diploids to investigate how whole genome duplication affects genome maintenance and response to stress. We find that S. pombe diploids are sensitive to replication stress and DNA damage, exhibit high levels of loss of heterozygosity, and become dependent on a group of ploidy-specific lethal genes for viability. These findings are observed in other eukaryotic models suggesting conserved consequences of polyploidy. We further investigate ploidy-specific lethal genes by depleting them using an auxin-inducible degron system to elucidate the mechanisms of genome maintenance in diploids. Overall, this work provides new insights on how whole genome duplications lead to genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ff76f6b675fe3b53c1d6072d58b13f856a3bdc6" target='_blank'>
              Diploidy confers genomic instability in Schizosaccharomyces pombe
              </a>
            </td>
          <td>
            Joshua M. Park, Daniel F. Pinski, S. Forsburg
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Chromosome segregation is a fundamental process in all life forms and requires coordination with genome organization, replication and cell division. The mechanism that mediates chromosome segregation in archaea remains enigmatic, despite the centre-stage role assumed by these organisms in the discourse about the origin of eukaryotes. Previously, we identified two proteins, SegA and SegB, which form a minimalist chromosome partition machine in Sulfolobales. Here we uncover patterns and mechanisms that the SegAB system employs to link chromosome organization to genome segregation. Deletion of the genes causes growth and chromosome partition defects. SegB binds to multiple sites scattered across the chromosome, but mainly localised close to the segAB locus in most archaeal genera. The sites are predominantly intragenic and enriched in one of the two compartments into which the chromosome folds. We show that SegB coalesces into multiple foci through the nucleoid, exhibiting a biased localisation towards the cell periphery, which hints at potential tethers to the cell membrane. The short-range DNA compaction activity of SegB and long-range looping of distant sites point to a significant role for SegB in chromosome condensation that in turn enables genome segregation. Collectively, our data put forward SegAB as important players in bridging chromosome organization to genome segregation in archaea.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bea5d93484886454e63c62bbf7266af68724c725" target='_blank'>
              Coupling chromosome organization to genome segregation in Archaea
              </a>
            </td>
          <td>
            Azhar F. Kabli, Irene W. Ng, Nicholas Read, Parul Pal, Julia Reimann, N. Tran, Sonja-Verena Albers, Tung B. K. Le, D. Barillà
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="REV7, also known as MAD2B, MAD2L2, and FANCV, is a HORMA-domain family protein crucial to multiple genome stability pathways. REV7’s canonical role is as a member of polymerase ζ, a specialized translesion synthesis polymerase essential for DNA damage tolerance. REV7 also ensures accurate cell cycle progression and prevents premature mitotic progression by sequestering an anaphase-promoting complex/cyclosome activator. Additionally, REV7 supports genome integrity by directing double-strand break repair pathway choice as part of the recently characterized mammalian shieldin complex. Given that genome instability is a hallmark of cancer, it is unsurprising that REV7, with its numerous genome maintenance roles, is implicated in multiple malignancies, including ovarian cancer, glioma, breast cancer, malignant melanoma, and small-cell lung cancer. Moreover, high REV7 expression is associated with poor prognoses and treatment resistance in these and other cancers. Promisingly, early studies indicate that REV7 suppression enhances sensitivity to chemotherapeutics, including cisplatin. This review aims to provide a comprehensive overview of REV7’s myriad roles in genome maintenance and other functions as well as offer an updated summary of its connections to cancer and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ab376b5318f868977c071b0b61bc16785632df" target='_blank'>
              REV7: a small but mighty regulator of genome maintenance and cancer development
              </a>
            </td>
          <td>
            Lara Maggs, Mitch McVey
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gene expression is coordinated by a multitude of transcription factors (TFs), whose binding to the genome is directed through multiple interconnected epigenetic signals, including chromatin accessibility and histone modifications. These complex networks have been shown to be disrupted during aging, disease, and cancer. However, profiling these networks across diverse cell types and states has been limited due to the technical constraints of existing methods for mapping DNA:Protein interactions in single cells. As a result, a critical gap remains in understanding where TFs or other chromatin remodelers bind to DNA and how these interactions are perturbed in pathological contexts. To address this challenge, we developed a transformative single-cell immuno-tethering DNA:Protein mapping technology. By coupling a species-specific antibody-binding nanobody to a cytosine base editing enzyme, this approach enables profiling of even weak or transient factor binding to DNA, a task that was previously unachievable in single cells. Thus, our Docking & Deamination followed by sequencing (D&D-seq) technique induces cytosine-to-uracil edits in genomic regions bound by the target protein, offering a novel means to capture DNA:Protein interactions with unprecedented resolution. Importantly, this technique can be seamlessly incorporated into common single-cell multiomics workflows, enabling multimodal analysis of gene regulation in single cells. We tested the ability of D&D-seq to record TF binding both in bulk and at the single-cell level by profiling CTCF and GATA family members, obtaining high specificity and efficiency, with clear identification of TF footprint and signal retention in the targeted cell subpopulations. Furthermore, the deamination reaction showed minimal off-target activity, with high concordance to bulk ChIP-seq reference data. Applied to primary human peripheral blood mononuclear cells (PBMCs), D&D-seq successfully identified CTCF binding sites and enabled integration with advanced machine-learning algorithms for predicting 3D chromatin structure. Furthermore, we integrated D&D-seq with single-cell genotyping to assess the impact of IDH2 mutations on CTCF binding in a human clonal hematopoiesis sample, uncovering altered binding and chromatin co-accessibility patterns in mutant cells. Altogether, D&D-seq represents an important technological advance enabling the direct mapping of TF or chromatin remodeler binding to the DNA in primary human samples, opening new avenues for understanding chromatin and transcriptional regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c35b9d0d36121406dcb7489958fdbc0f064e833" target='_blank'>
              Single-cell mapping of regulatory DNA:Protein interactions
              </a>
            </td>
          <td>
            Wei-Yu Chi, Sang-Ho Yoon, Levan Mekerishvili, S. Ganesan, Catherine Potenski, Franco Izzo, Dan A Landau, Ivan Raimondi
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Approximately 97% of the human genome comprises non-coding sequences, with nearly half originating from transposable elements. Among these, retrotransposons represent a critical subclass that replicates via a "copy-and-paste" mechanism and significantly influences the regulation of host genomes. In both normal and pathological contexts, retrotransposons contribute a vast reservoir of regulatory elements that can modulate the expression of genes. If left unchecked, retrotransposons can substantially affect host transcriptional programs and genomic integrity. Therefore, various mechanisms, including epigenetic modifications, are employed to mitigate their potentially deleterious effects. In diseases such as cancers, the epigenome is often significantly reprogrammed, which can lead to retrotransposon dysregulation. Drawing insights from recent studies conducted in human and murine cells, this review examines how retrotransposons expand the complexity of mammalian genomes, describes the impact of their epigenetic dysregulation in cancer development, and highlights the potential of targeting these sequences for therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acf5c3f4d65d70d88355c0196d8616e76468f57c" target='_blank'>
              Epigenetic dysregulation of retrotransposons in cancer.
              </a>
            </td>
          <td>
            Danny Leung, Kwok Yu Liu
          </td>
          <td>2025-02-13</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer is characterized by genetic and epigenetic deregulations, leading to aberrant expression of tissue-specific genes that are normally silent in healthy breast tissue. Our previous work identified the embryonic stem cell-specific gene DNMT3B, a de novo DNA methyltransferase, as aberrantly activated in breast cancer, correlating with aggressive tumour behaviour and high relapse risk, regardless of molecular subtype. Through integrative bioinformatic analyses of DNA methylation and transcriptomic data, we identified 154 genes downregulated via DNMT3B-driven promoter hypermethylation, many of which are associated with high relapse risk. Notably, the tumour suppressor gene GATA3 emerged as a primary target of functional inactivation through either loss-of-function mutations or DNMT3B-controlled hypermethylation, in a mutually exclusive manner. Both mechanisms of GATA3 inactivation were associated with similar molecular signatures linked to tumour progression, increased malignancy, and poorer prognosis. However, distinct differences were observed, with immune- and inflammation-related genes enriched in GATA3 hypermethylation cases but depleted in mutation-driven silencing. Additionally, our analysis uncovered other potential tumour suppressor genes epigenetically repressed in aggressive breast cancers. These findings underscore a broader role of GATA3 inactivation beyond genetic alterations and suggest therapeutic opportunities to target epigenetically silenced tumour suppressors in aggressive breast tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c122f003440fc5017c72299634c64a7c4caded88" target='_blank'>
              Discovery of Epigenetically Silenced Tumour Suppressor Genes in Aggressive Breast Cancer Through a Computational Approach
              </a>
            </td>
          <td>
            Anne-Laure Vitte, Florent Chuffart, Emmanuelle Jacquet, Eleni Nika, Mireille Mousseau, Ina Jung, S. Tabone-Eglinger, Thomas Bachelot, Sophie Rousseaux, Saadi Khochbin, Ekaterina Bourova-Flin
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25b8ef7036ff1fb3a03cc5984eb57541db6e9f2c" target='_blank'>
              Functional screen identifies RBM42 as a mediator of oncogenic mRNA translation specificity.
              </a>
            </td>
          <td>
            Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Nimmy Mohan, Kaylee Yiakis, Isabelle Liu, Kwun Wah Wen, Grace E. Kim, S. Miglani, Alma L. Burlingame, Hani Goodarzi, Rushika M. Perera, Davide Ruggero
          </td>
          <td>2025-02-04</td>
          <td>Nature cell biology</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="Background Because transposable elements (TEs) can cause heritable genetic changes, past work on TE mobility in Arabidopsis thaliana has mostly focused on new TE insertions in the germline of hypomethylated plants. It is, however, well-known that TEs can also be active in the soma, although the high-confidence detection of somatic events has been challenging. Here, we leveraged the high accuracy of PacBio HiFi long reads to evaluate the somatic mobility of TEs in individuals of an A. thaliana non-reference strain lacking activity of METHYLTRANSFERASE1 (MET1), a major component of the DNA methylation maintenance machinery. Most somatically mobile families coincided with those found in germline studies of hypomethylated genotypes, although the exact TE copies differed. We also discovered mobile elements that had been missed by standard TE annotation methods. Somatic TE activity was variable among individual plants, but also within TE families. Finally, our approach pointed to the possible involvement of alternative transposition as a cause for somatic hypermutability in a region that contains two closely spaced VANDAL21 elements. We conclude that long-read sequencing can reveal widespread TE transposition in the soma of A. thaliana hypomethylated mutants. Assessing somatic instead of germline mobilization is a fast and reliable method to investigate different aspects of TE mobility at the single plant level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62607e49f83191e3ad9371253c4b7877fe10686a" target='_blank'>
              Long-read detection of transposable element mobilization in the soma of hypomethylated Arabidopsis thaliana individuals
              </a>
            </td>
          <td>
            Andrea Movilli, Svitlana Sushko, Fernando A. Rabanal, Detlef Weigel
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="DNA methylation at gene promoters is generally considered to be associated with transcriptional repression. However, lack of a clear picture of where promoter methylation is most important for transcriptional regulation has hindered our understanding of this relationship and resulted in the use of a wide variety of arbitrary promoter definitions. We demonstrate here that the use of different promoter definitions can lead to contradictory results between studies of promoter methylation. In response, we have developed Methodical, a computational method that combines RNA-seq and whole genome bisulfite sequencing (WGBS) data to identify genomic regions where DNA methylation is highly correlated with transcriptional activity. We refer to these regions as transcript-proximal methylation-associated regulatory sites (TMRs). We applied Methodical to one normal prostate tissue data set, one prostate tumour dataset and one prostate metastases dataset and characterized the identified TMRs. We show that the region just downstream of the TSS, particularly within the first exon and intron, is the most common location for TMRs and that TMRs are enriched for particular genomic features, chromatin states and transcription factor binding sites. Finally, we demonstrate that the methylation of TMRs is generally strongly correlated with transcription in diverse cancer types and that TMRs are highly subject to altered DNA methylation in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/229e0f93b401077a9ccbe06fb3844f9ba3a44d0d" target='_blank'>
              Systematic identification of regions where DNA methylation is correlated with transcription refines regulatory logic in normal and tumour tissues
              </a>
            </td>
          <td>
            Richard Heery, Martin H Schaefer
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mammalian genomes contain millions of regulatory elements that control the complex patterns of gene expression. Previously, The ENCODE consortium mapped biochemical signals across many cell types and tissues and integrated these data to develop a Registry of 0.9 million human and 300 thousand mouse candidate cis-Regulatory Elements (cCREs) annotated with potential functions1. We have expanded the Registry to include 2.35 million human and 927 thousand mouse cCREs, leveraging new ENCODE datasets and enhanced computational methods. This expanded Registry covers hundreds of unique cell and tissue types, providing a comprehensive understanding of gene regulation. Functional characterization data from assays like STARR-seq, MPRA, CRISPR perturbation, and transgenic mouse assays now cover over 90% of human cCREs, revealing complex regulatory functions. We identified thousands of novel silencer cCREs and demonstrated their dual enhancer/silencer roles in different cellular contexts. Integrating the Registry with other ENCODE annotations facilitates genetic variation interpretation and trait-associated gene identification, exemplified by discovering KLF1 as a novel causal gene for red blood cell traits. This expanded Registry is a valuable resource for studying the regulatory genome and its impact on health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99941d31f540b8743cd6f7ea4d2eb9891068be3e" target='_blank'>
              An Expanded Registry of Candidate cis-Regulatory Elements for Studying Transcriptional Regulation
              </a>
            </td>
          <td>
            Jill E. Moore, H. Pratt, K. Fan, Nishigandha Phalke, Jonathan Fisher, Shaimae I. Elhajjajy, Gregory Andrews, Mingshi Gao, Nicole Shedd, Yu Fu, Matthew C Lacadie, Jair Meza, Mohit Ganna, Eva Choudhury, Ross Swofford, Nina P. Farrell, Anusri Pampari, Vivekanandan Ramalingam, Fairlie Reese, Beatrice Borsari, Michelle Yu, Eve Wattenberg, M. Ruiz-Romero, Milad Razavi-Mohseni, Jinrui Xu, Timur Galeev, Michael A. Beer, R. Guigó, Mark B. Gerstein, Jesse M. Engreitz, Mats Ljungman, Timothy E. Reddy, M. Snyder, Charles B. Epstein, Elizabeth Gaskell, Bradley E. Bernstein, D. Dickel, Axel Visel, Len A. Pennacchio, A. Mortazavi, A. Kundaje, Zhiping Weng
          </td>
          <td>2024-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>124</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5181ddbe51c7dee5ac944fbf5ce920c81fec81b8" target='_blank'>
              Enhancer regulatory networks globally connect non-coding breast cancer loci to cancer genes
              </a>
            </td>
          <td>
            Yihan Wang, Daniel A Armendariz, Lei Wang, Huan Zhao, Shiqi Xie, Gary C. Hon
          </td>
          <td>2025-01-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Numerical abnormalities in chromosomal states, referred to as aneuploidy, is commonly observed in many cancer cells. Although numerous internal and external factors induce aneuploidy, the primary cause of aneuploidy in humans remains unclear. DNA damage is identified as a potential cause of aneuploidy by inducing chromosome segregation errors. However, a direct relationship between DNA damage and aneuploidy remains poorly understood. A major reason for this is the extremely low frequency of aneuploidy in cultured cells, making quantitative analyses challenging. In this study, we investigated the relationship between DNA damage and aneuploidy in cell lines containing minichromosomes. These chromosomes are more prone to loss than normal chromosomes, with the rate of loss substantially increased following exposure to various DNA-damaging agents. To determine whether damaged chromosomes were subjected to direct loss or whether chromosome loss occurred as an indirect consequence of a prolonged G2 phase or other factors, we used the CRISPR-Cas9 system to introduce a single DNA double-strand break (DSB) on a minichromosome. The rate of minichromosome loss increased by approximately seven-fold compared with that of the control. Furthermore, the loss rate was significantly elevated in the absence of KU70, a key factor in non-homologous end joining, and upon inhibition of ataxia telangiectasia mutated (ATM), a DNA damage checkpoint protein. Finally, two closely spaced nicks, believed to generate a 5’-overhang, were also shown to induce minichromosome loss. These findings indicated that a single DSB or two closely spaced nicks can cause aneuploidy if improperly repaired in vertebrates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/657f1468476733a77f4e32e0f0402d6f84ad1e95" target='_blank'>
              Quantitative analysis of the frequency of chromosome loss following DSB induction
              </a>
            </td>
          <td>
            Seiya Matsuno, Ryo Ishida, Ryotaro Kawasumi, Kouji Hirota, Takuya Abe
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Oncogenic gene fusions are key drivers of cancer, yet most remain untargetable by current therapies. Here, we establish CRISPR-PspCas13b as a personalizable platform for systematic silencing of various fusion transcripts. We reveal that recognition and cleavage of the breakpoint sequence by PspCas13b disrupts the fusion transcript, resulting in unexpected RNA nicking and ligation near the cleavage site, which generates out-of-frame, translation-incompetent transcripts. This approach efficiently degrades canonical and drug-resistant BCR::ABL1 mutants (e.g., T315I), a primary cause of resistance to tyrosine kinase inhibitors (TKIs) and relapse in chronic myeloid leukemia (CML). Silencing T315I BCR::ABL1 mRNA in drug-resistant CML cells triggers extensive transcriptomic, proteomic, and phosphoproteomic remodelling, causing erythroid differentiation and apoptosis. Beyond BCR-ABL1 mutants, personalized design of PspCas13b effectively silences other undruggable fusions, including RUNX1::RUNX1T1 and EWSR1::FLI1, key drivers in acute myeloid leukemia and in Ewing sarcoma, respectively. Collectively, this study establishes a framework for systematic, precise, and personalizable targeting of otherwise undruggable or drug-resistant oncogenic transcripts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d993b12a12216338adf29eefc996f4f14d40a3" target='_blank'>
              Systematic silencing of oncogenic fusion transcripts with ultra-precise CRISPR-Cas13b
              </a>
            </td>
          <td>
            Wenxin Hu, Honglin Chen, J. Casan, Carolyn Shembrey, Steven He, Lauren M Brown, Timothy P. Hughes, Deborah L. White, Antoine de Weck, I. Voskoboinik, Joseph A Trapani, Paul G Ekert, Teresa Sadras, Mohamed Fareh
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Immune escape is a critical hallmark of cancer progression and underlies resistance to multiple immunotherapies. However, it remains unclear when the genetic events associated with immune escape occur during cancer development. Here, we integrate functional genomics studies of immunomodulatory genes with a tumor evolution reconstruction approach to infer the evolution of immune escape across 38 cancer types from the Pan-Cancer Analysis of Whole Genomes dataset. Different cancers favor mutations in different immunomodulatory pathways. For example, the antigen presentation machinery is highly mutated in colorectal adenocarcinoma, lung squamous cell carcinoma, and chromophobe renal cell carcinoma, and the protein methylation pathway is highly mutated in bladder transitional cell carcinoma and lung adenocarcinoma. We also observe different timing patterns in multiple immunomodulatory pathways. For instance, mutations impacting genes involved in cellular amino acid metabolism were more likely to happen late in pancreatic adenocarcinoma. Mutations in the glucocorticoid receptor regulatory network pathway tended to occur early, while mutations in the TNF pathways were more likely to occur late in B-cell non-Hodgkin lymphoma. Mutations in the NOD1/2 signaling pathway and DNA binding transcription factor activity tended to happen late in breast adenocarcinoma and ovarian adenocarcinoma. Together, these results delineate the evolutionary trajectories of immune escape in different cancer types and highlight opportunities for improved immunotherapy of cancer. Significance Despite its critical role in cancer progression, the evolution of immune escape is poorly understood. We integrate functional genomics and tumor evolution reconstruction and infer immune escape trajectories across cancer types. Our results have important implications for developing biomarkers for immunoprevention and treatment strategies for immune escape of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a491b543872787c05b0f657359563c86a9093f6" target='_blank'>
              Evolutionary trajectories of immune escape across cancers
              </a>
            </td>
          <td>
            Wenjie Chen, Toby M. Baker, Zhihui Zhang, Huw A. Ogilvie, P. Van Loo, S. Gu
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="High fidelity of replication is important to preserve genomic integrity and ensure healthy progeny. Perturbations of replication, also known as replication stress, is frequently observed in cancer cells and is considered a cancer cell-specific trait. Although replication stress drives genomic instability and tumor progression, it also generates a targetable cancer-specific vulnerability. In order to identify potential therapeutic targets in cancer cells that experience replication stress, we performed a genome wide genetic screen in human HAP1 cells challenged with low doses of replication stress-inducing drugs. We identified a large set of genes that specifically hamper cell survival in the context of replication stress. In addition to well-known players in the replication stress response and DNA repair, such as RNASEH1, BRIP1, and MDC1, we identified several genes with no prior described role in DNA replication, damage tolerance or repair. We validated that the loss of GIGYF2, HNRNPA2B1, and SUMO2 renders cells more vulnerable to replication stress. For GIGYF2 and SUMO2, we could implicate a role in homologous recombination. Taken together, our replication stress screen identified several known as well as some novel factors that protect against the toxic implications of replication stress. These factors could entail potential therapeutic targets for cancer cells experiencing replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddea308b1694526a4e6fc1d9af3b1fa983ba1e3c" target='_blank'>
              Identification of critical factors of the replication stress response in human cells
              </a>
            </td>
          <td>
            L. M. Janssen, Empar Baltasar Perez, C. Vaarting, Abdelghani Mazouzi, M. Raaben, T. Brummelkamp, René H. Medema, J. Raaijmakers
          </td>
          <td>2025-01-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Significance Mutations in the metabolic enzyme isocitrate dehydrogenase (IDH1) are frequent and considered early events in a subset of adult gliomas. The mutant IDH1 enzyme changes functionally, favoring the production of the oncometabolite 2-hydroxyglutarate, and alters metabolic and epigenetic pathways. However, a comprehensive understanding of the epigenetic landscape in IDH1mut cells and how it contributes to tumorigenesis is lacking. This study presents a high-dimensional single-cell analysis of chromatin alterations driven by mutant-IDH1. Our analysis reveals extensive remodeling of chromatin patterns, highlighting a rapid decrease in histone acetylation marks and increased epigenetic heterogeneity. These changes in histone acetylation, affecting enhancer patterns, are strongly associated with the metabolic abnormalities of IDH1mut cells, underscoring the interplay between chromatin and metabolism dysregulation in glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1395bf17f297913f9bbe2a4e1db052179fa9f3f" target='_blank'>
              Oncogenic IDH1mut drives robust loss of histone acetylation and increases chromatin heterogeneity
              </a>
            </td>
          <td>
            N. Furth, Niv Cohen, Avishay Spitzer, T. Salame, B. Dassa, Tevie Mehlman, Alexander Brandis, Arieh Moussaieff, D. Friedmann-Morvinski, Maria G Castro, Jerome Fortin, M. Suvà, I. Tirosh, A. Erez, Guy Ron, E. Shema
          </td>
          <td>2024-12-30</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Background The understanding of genetic and epigenetic regulation of gene expression in endothelial and vascular smooth muscle cells remains fragmented with limited experimental validation. Methods Chromatin conformation (Micro-C), DNA methylation (RRBS), chromatin accessibility (ATAC-seq), and transcriptome profiles (RNA-seq) were mapped in human induced pluripotent stem cell (hiPSC)-derived, isogenic endothelial and vascular smooth muscle cells (iECs and iVSMCs). CTCF and RAD21 were depleted to assess the functional relevance of chromatin architecture, and genome editing was used to evaluate the allelic effect of a blood pressure-associated single nucleotide polymorphism (SNP). Results Significant correlations were identified between gene expression levels and chromatin interactions, chromatin accessibility, and DNA methylation in iECs and iVSMCs, with chromatin interactions showing the strongest association. Chromatin contact regions displayed distinct epigenetic landscapes depending on the types of regulatory element interactions involved. Perturbation of CTCF and RAD21 revealed their differential regulatory effects, particularly on the expression of genes overlapping chromatin contacts, with RAD21 exhibiting a broader regulatory impact. SNPs associated with several vascular traits were enriched in chromatin loops or accessible regions in iECs or iVSMCs. Precise genome editing demonstrated allele-dependent effects of SNP rs9833313 on the expression of SHOX2 located 247.4 kbp away but within the same chromatin loops as the SNP. Conclusion This study provides an extensive epigenetic landscape of vascular cells that may drive novel research on the role of genetic and epigenetic mechanisms of vascular function and disease as demonstrated by our targeted experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cca08a65eb2683798ce82f8cb05c4cd45763cf1" target='_blank'>
              Global and genetic regulation of gene expression in human endothelial and vascular smooth muscle cells
              </a>
            </td>
          <td>
            Pengyuan Liu, Atrayee Ray, Yong Liu, Manoj K. Mishra, Qiongzi Qiu, Rajan Pandey, Bhavika Therani, Jing Huang, Jiayi Ren, C. Stelloh, Xiaowen Bai, Andrew S Greene, Allen W Cowley, Aron M. Geurts, Sridhar Rao, Mingyu Liang
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="In most eukaryotic cells, euchromatin is localized in the nuclear interior, whereas heterochromatin is enriched at the nuclear envelope (NE). This conventional chromatin organization is established by heterochromatin tethering to the NE, however its importance for cellular homeostasis is largely unexplored. Peripheral heterochromatin localization relies on redundant NE-tethering systems. One tether is constituted by the lamin B receptor (LBR) in mammals, but the enigmatic nature of the other tethers has hampered functional analyses. Here we demonstrate that the downregulation of abundant, ubiquitous NE proteins can induce the global detachment of heterochromatin from the NE. Among these factors, we identify LBR and LAP2 as major players in bulk heterochromatin attachment to the NE in pluripotent and differentiated mammalian cells. Their loss leads to repositioning of heterochromatin to the nuclear interior, changes in chromatin accessibility, deregulation of gene expression including activation of antiviral innate immunity, and defects in cell fate determination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fde3d7b0c4771c759bd6fcaac94c205c62bfd4c" target='_blank'>
              LBR and LAP2 mediate heterochromatin tethering to the nuclear periphery to preserve genome homeostasis
              </a>
            </td>
          <td>
            Renard Lewis, Virginia Sinigiani, K. Koos, Cristiana Bersaglieri, C. Ashiono, Raffaella Santoro, C. Ciaudo, Peter Horvath, Puneet Sharma, Ulrike Kutay
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="PURPOSE
Previous research has shown, that ABC transporters gene expression level can predict the efficacy of therapy. However, other mechanisms of gene activity are rarely considered, especially in non-small cell lung cancer (NSCLC). Thus, the purpose of the work was to assess chromosomal aberrations of all 49 ABC transporters genes and the expression levels of some ABC genes, as well as their correlation with survival.


MATERIALS AND METHODS
The surgical material of 104 patients with NSCLC was used in this study. Treatment included surgery and 3 courses of adjuvant chemotherapy with "platinum doublets". DNA and RNA were isolated from the samples, followed by microarray analysis to assess the expression and chromosomal aberrations (deletions and amplifications) of ABC genes.


RESULTS
Metastatic-free survival (MFS) was higher with ABCC1, ABCC2, and ABCG1 hypoexpression at a statistically significant level (p=0.01). The presence of deletion in ABCB1 correlates with 100% MFS (p=0.001). The survival rates with ABCG1 amplification are not higher than 45% (p<0.0001). ABCA11 deletion is associated with a low MFS rate (38%) versus 91% with normal copy number (p=0.006). ABCB9 analysis showed opposite results, with survival rates of 55% and 91% in the presence of amplification and normal copy number, respectively (p=0.006). ABCC subfamily genes showed a similar result in the presence of amplification, where ABCC3 and ABCC10 account for 64% and 60% survival, respectively (p=0.005, p=0.01).


CONCLUSION
Thus, not only expression but also chromosomal aberrations were found to be associated with patient survival. These findings could be a potential marker of metastatic-free survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1581dfdb2ff097ba8f6291fda2517229cc004d6d" target='_blank'>
              Influence of DNA Copy Number Aberrations in ABC Transporter Family Genes on the Survival of Patients with Primary Operatable Non-Small Cell Lung Cancer.
              </a>
            </td>
          <td>
            Matvey Mikhailovich Tsyganov, M. Ibragimova, E. Rodionov, Anastavia Alekseevna Frolova, Irina Aleksandrovna Tsydenova, E. A. Lutzkaya, S. Miller
          </td>
          <td>2025-01-31</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Metastatic cancer, a major cause of mortality, has been understudied compared to primary tumors, leaving gaps in our understanding of how cancer genes adapt between these states. We analyzed the association between mutations and copy-number alterations in 25,000 tumor samples from both primary and metastatic cancers. Our findings show that cancer genes display distinct interaction strengths across these states, with 27.45% of genes, including ARID1A, FBXW7, and SMARCA4, shifting between one-hit and two-hit drivers. Interaction strengths varied by cancer state and treatment conditions, revealing seven state-specific interactions. We also identified 38 primary-specific and 21 metastatic-specific high-order interactions, enriched in cancer hallmarks, indicating unique tumor progression mechanisms. These findings highlight dynamic tumor progression mechanisms and underscore the importance of considering cancer state in research and treatment strategies for precise therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/968f72ea3f3d388ad8546b594ad1735637aaf86d" target='_blank'>
              Mapping cancer gene dynamics through state-specific interactions
              </a>
            </td>
          <td>
            Adrián Maqueda-Real, Laia Ollé-Monràs, Solip Park
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09816baa7e13a8a81394d1d42faa90a68058ed21" target='_blank'>
              Stochastic demethylation and redundant epigenetic suppressive mechanisms generate highly heterogeneous responses to pharmacological DNA methyltransferase inhibition
              </a>
            </td>
          <td>
            Mie K. Jakobsen, S. Traynor, A. Y. Nielsen, Christina Dahl, Mette Staehr, S. T. Jakobsen, M. S. Madsen, Rasmus Siersbaek, Mikkel G Terp, Josefine B. Jensen, C. B. Pedersen, Anup Shrestha, J. R. Brewer, P. H. Duijf, Odd L. Gammelgaard, Henrik J Ditzel, Alexei F Kirkin, Per Guldberg, Morten F. Gjerstorff
          </td>
          <td>2025-01-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Unscheduled whole genome duplication (WGD), also described as unscheduled or non-physiological polyploidy, can lead to genetic instability and is commonly observed in human cancers. WGD generates DNA damage due to scaling defects between replication factors and DNA content. As a result, DNA damage repair mechanisms are thought to be critical for ensuring cell viability and proliferation under these conditions. In this study, we explored the role of homologous recombination and Holliday junction resolution in non-physiological polyploidy in vivo. Using Drosophila genetics and high-resolution imaging, we identified a key and surprising role for Gen/Gen1 nuclease. Our findings revealed that loss-of-function and overexpression of Gen have opposing effects, delaying or accelerating the proliferation of polyploid cells, respectively. These changes ultimately impact cell proliferation, nuclear asynchrony and mitotic DNA damage levels. Surprisingly, our findings show that this effect is unrelated with the expected Gen’s function in DNA damage repair. Instead, Gen seems to influence polyploid DNA replication rates. This work identifies a novel function for Gen nuclease and provides new insights into the cellular and molecular requirements of non-physiological polyploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbd8ff52437b49af76b524b72bb72040b51dd497" target='_blank'>
              A novel DNA repair-independent role for Gen nuclease in promoting unscheduled polyploidy cell proliferation
              </a>
            </td>
          <td>
            Manon Budzyk, Anthony Simon, Anne-Sophie Macé, R. Basto
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="RAD18 is a conserved eukaryotic E3 ubiquitin ligase that promotes genome stability through multiple pathways. One of these is gap-filling DNA synthesis at active replication forks and in post-replicative DNA. RAD18 also regulates homologous recombination (HR) repair of DNA breaks; however, the current literature describing the contribution of RAD18 to HR in mammalian systems has not reached a consensus. To investigate this, we examined three independent RAD18-null human cell lines. Our analyses found that loss of RAD18 in HCT116, but neither hTERT RPE-1 nor DLD1 cell lines, resulted in elevated sister chromatid exchange, gene conversion, and gene targeting, i.e., HCT116 mutants were hyper-recombinogenic (hyper-rec). Interestingly, these phenotypes were linked to RAD18’s role in PCNA K164 ubiquitination, as HCT116 PCNAK164R/+ mutants were also hyper-rec, consistent with previous studies in rad18−/− and pcnaK164R avian DT40 cells. Importantly, the knockdown of UBC9 to prevent PCNA K164 SUMOylation did not affect hyper-recombination, strengthening the link between increased recombination and RAD18-catalyzed PCNA K164 ubiquitination, but not K164 SUMOylation. We propose that the hierarchy of post-replicative repair and HR, intrinsic to each cell type, dictates whether RAD18 is required for suppression of hyper-recombination and that this function is linked to PCNA K164 ubiquitination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ade4d531e6f454e305c916fc9c53a0c286d620e" target='_blank'>
              Cell Type Specific Suppression of Hyper-Recombination by Human RAD18 Is Linked to Proliferating Cell Nuclear Antigen K164 Ubiquitination
              </a>
            </td>
          <td>
            Colette B. Rogers, W. Leung, Ryan M. Baxley, Rachel E. Kram, Liangjun Wang, Joseph P. Buytendorp, Khoi Le, D. Largaespada, Eric A. Hendrickson, A. Bielinsky
          </td>
          <td>2025-01-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="The study explores the role of KIF18A, a gene linked to whole-genome doubling (WGD) and chromosomal instability (CIN), in the metastatic progression of hepatocellular carcinoma (HCC). We found that KIF18A is a critical link between CIN and anoikis resistance, key factors in HCC metastasis. Through the bulk-seq analysis, we identified the pressures of anoikis and CIN faced by tumor cells during HCC metastasis and identified hub genes central to this process. Our results reveal that E2F activation during HCC progression leads to the transcription of KIF18A, promoting the survival and metastasis of CIN tumors. Overexpression of KIF18A leads to longer tumor life, more micronuclei, and increased survival and metastasis through non-canonical NF-kB activation. Deletion of KIF18A stabilizes the nuclear membrane of the micronucleus, silences cGAS-STING and PI3K-AKT pathways, and inhibits classical NF-kB. The study’s limitations include the need for further animal studies to validate the findings and explore the transient activation of the cGAS-STING pathway. Additionally, future research could focus on the potential therapeutic implications of targeting KIF18A in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae22aedbaa5a2bdc1f487c6a0f986379d3bb08c" target='_blank'>
              KIF18A in HCC Metastasis: Dual Role in Anoikis Resistance and Chromosome Instability
              </a>
            </td>
          <td>
            Changhao Kan, Weiyu Bai, Xinru Yang, Hayam Hamdy, Xianghui Yang, Xin Liu, Yan Lv, Junling Shen, Jianwei Sun
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chromosomes, as the fundamental unit of genetic material located within the cell nucleus, have undergone extensive and complex changes throughout the evolutionary history of eukaryotes. Many of these patterns and mechanisms of change share commonalities across various diseases, including cancer. For a long time, biologists were limited to research methods with relatively low resolution, such as fluorescence in situ hybridization (FISH). However, the rapid advancement of high-throughput sequencing technologies is revolutionizing our understanding of chromosomal variations across different species, among individuals of the same species, and even at the cellular level within a single individual. In this review, we focus on the chromosomal evolution in vertebrates, and provide an overview of the role of chromosom rearrangements in speciation, the molecular mechanisms of chromosomal rearrangements, the evolutionary patterns from ancestral chromosomes to extant chromosomes, and the significance of sex chromosomes as a general paradigm for studying chromosomal evolution. Finally, we discuss the new opportunities and challenges that synthetic biology brings to the field of chromosomal evolution research, with the aim of offering new insights and references for understanding and studying vertebrate chromosomal evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bbc3a9efead12e9b20f611725318c5fc08d7716" target='_blank'>
              The evolution of sequences and spatial conformation in vertebrate chromosomes.
              </a>
            </td>
          <td>
            Jing Liu, Qi Zhou
          </td>
          <td>2025-02-01</td>
          <td>Yi chuan = Hereditas</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Identifying the genes capable of driving tumorigenesis in different tissues is one of the central goals of cancer genomics. Computational methods that exploit different signals of positive selection in the pattern of mutations observed in genes across tumors have been developed to this end. One such signal of positive selection is clustering of mutations in areas of the three-dimensional (3D) structure of the protein above the expectation under neutrality. Methods that exploit this signal have been hindered by the paucity in proteins with experimentally solved 3D structures covering their entire sequence. Here, we present Oncodrive3D, a computational method that by exploiting AlphaFold 2 structural models extends the identification of proteins with significant mutational 3D clusters to the entire human proteome. Oncodrive3D shows sensitivity and specificity on par with state-of-the-art cancer driver gene identification methods exploiting mutational clustering, and clearly outperforms them in computational efficiency. We demonstrate, through several examples, how significant mutational 3D clusters identified by Oncodrive3D in different known or potential cancer driver genes can reveal details about the mechanism of tumorigenesis in different cancer types and in clonal hematopoiesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b62083d3dd11f079792a9e41da3af07b399c20f" target='_blank'>
              Oncodrive3D: Fast and accurate detection of structural clusters of somatic mutations under positive selection
              </a>
            </td>
          <td>
            Stefano Pellegrini, Olivia Dove Estrella, Ferran Muiños, Núria López-Bigas, A. González-Pérez
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="SETD6 is a non-histone lysine methylatransferase, previously shown to participate in several housekeeping signaling pathways such as the NFkB pathway, Wnt signaling pathway, mitosis and more. In the current study we show evidence that SETD6 methylation is involved in the regulation of cytokinesis - the final process that divides cell contents into two daughter cells. SETD6 depleted HeLa cells presented high levels of chromatin bridges and actin patches, which are commonly observed following chromosomal segregation errors. In a proteomic screen we identified Aurora-B as a novel SETD6 substrate. Aurora-B kinase is an essential regulator of cytokinesis, known to actively delay cytokinesis as a response to the presence of chromatin in the midzone. We found that SETD6 binds and methylates Aurora-B on two adjacent lysine residues. Upon replication stress, Aurora-B methylation by SETD6 increases but is abolished when the two lysine methylation targets are substituted. In addition, replication stress led to a high tendency of SETD6 depleted cells to multinucleate, a major chromosomal-instability (CIN) phenotype. We detected a significant reduction in the Aurora-B kinase activity during cytokinesis in SETD6 knockout cells upon replication stress, which could be the mechanism underlying the accumulation of CIN phenotypes in these cells. CIN is a hallmark of cancer and is associated with tumor cell malignancy. Our findings suggest that Aurora-B methylation by SETD6 carries meaningful implications on tumorigenic cellular pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30079fd8588b1378d0602ce8bccaf091eb97d109" target='_blank'>
              AuroraB-kinase methylation by SETD6 regulates cytokinesis and protects cells from chromosomal instability
              </a>
            </td>
          <td>
            Michal Feldman, Anand Chopra, Dikla Nachmias, K. Biggar, Daniel Sevilla, Natalie Elia, D. Levy
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Tumor protein 53 (TP53) is a tumor repressor gene that is highly responsible for activating cell cycle arrest in the event of DNA damage, stopping the replication of damaged cells, which may cause cancer. However, when mutated, TP53 is an oncogene, with mutated variants of it being seen in over half of all cancers diagnosed including the aforementioned four. With groundbreaking CRISPR-Cas9 gene editing technology coming to the forefront of biotechnological research, the applications to cancer research and gene therapy have been increasing. This review explores CRISPR-Cas9 and its uses for treatment and medication investigation in various cancers, including lung, prostate, breast, and bone cancer through the lens of the TP53 gene. Due to TP53s commonality in cancer, CRISPR-Cas9 gene editing technology can be used to its maximum potential in this context as a treatment or investigative tool, with it yielding positive results in reverting tumor growth, as well as great versatility when it comes to screening related genes that inhibit or encourage tumor growth which can be targeted by drugs, or even the screening of the drugs effectiveness themselves. Thus, CRISPR-Cas9s applications in TP53-related cancer treatment and chemotherapeutic fields prove this technology to be successful.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da087d15d0c931988cddd598ffd040ef96f3cb9b" target='_blank'>
              Using CRISPR/Cas9 Technology to Target TP53 Related Cancers
              </a>
            </td>
          <td>
            Man Ho Damien Hsieh
          </td>
          <td>2025-01-09</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer’s epigenetic landscape, a labyrinthine tapestry of molecular modifications, has long captivated researchers with its profound influence on gene expression and cellular fate. This review discusses the intricate mechanisms underlying cancer epigenetics, unraveling the complex interplay between DNA methylation, histone modifications, chromatin remodeling, and non-coding RNAs. We navigate through the tumultuous seas of epigenetic dysregulation, exploring how these processes conspire to silence tumor suppressors and unleash oncogenic potential. The narrative pivots to cutting-edge technologies, revolutionizing our ability to decode the epigenome. From the granular insights of single-cell epigenomics to the holistic view offered by multi-omics approaches, we examine how these tools are reshaping our understanding of tumor heterogeneity and evolution. The review also highlights emerging techniques, such as spatial epigenomics and long-read sequencing, which promise to unveil the hidden dimensions of epigenetic regulation. Finally, we probed the transformative potential of CRISPR-based epigenome editing and computational analysis to transmute raw data into biological insights. This study seeks to synthesize a comprehensive yet nuanced understanding of the contemporary landscape and future directions of cancer epigenetic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf231d1889782a99759e5b17e3a47e9d23349015" target='_blank'>
              Mechanisms and technologies in cancer epigenetics
              </a>
            </td>
          <td>
            Z. Sherif, O. Ogunwobi, H. Ressom
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="TnpB is an evolutionarily diverse family of RNA-guided endonucleases associated with prokaryotic transposons. Due to their small size and putative evolutionary relationship to Cas12s, TnpB holds significant potential for genome editing and mechanistic exploration. However, most TnpBs lack robust gene-editing activity, and unbiased profiling of mutational effects on editing activity has not been experimentally explored. Here, we mapped comprehensive sequence-function landscapes of a TnpB ribonucleoprotein and discovered many activating mutations in both the protein and RNA. Single-position changes in the RNA outperform existing variants, highlighting the utility of systematic RNA scaffold mutagenesis. Leveraging the mutational landscape of the TnpB protein, we identified enhanced protein variants from a combinatorial library of activating mutations, which increased editing levels in yeast and human cells relative to wild-type TnpB. In total, this study highlights unknown elements critical for regulation of endonuclease activity in both the TnpB protein and the RNA, and reveals a surprising amount of latent activity accessible through mutation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2e2c5976d3fbd7664491b03e4240c4d3937effa" target='_blank'>
              Discovery of widespread activating mutations in a compact RNA-guided endonuclease
              </a>
            </td>
          <td>
            B. W. Thornton, Rachel F. Weissman, Ryan V. Tran, Brenda T. Duong, Jorge E. Rodriguez, Cynthia I. Terrace, Evan D. Groover, Jung-Un Park, Julia Tartaglia, Jennifer A. Doudna, David F. Savage
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 The lysine methyltransferase KMT2A/MLL1 plays a crucial role in normal development and hematopoiesis. Rearrangement of KMT2A results in the formation of a KMT2A fusion protein with aberrant activity, which is a well-characterized driver of leukemogenesis and is therapeutically targetable. In contrast to the widely studied KMT2A translocation, the molecular consequences of other KMT2A alterations, including KMT2A amplification, remain unknown. KMT2A amplification is detected in highly aggressive and treatment-resistant myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). It strongly associates with TP53 mutations and complex karyotypes. Unlike KMT2A rearrangement, KMT2A amplification is reported mostly in patients older than 65 and treated with alkylating agents. Here, we conducted a comprehensive molecular analysis of KMT2A-amplified MDS/AML by long-read nanopore sequencing. We have revealed the unique epigenetic landscape, including DNA methylation, histone modification, and chromatin accessibility, associated with KMT2A amplification. We also explored the potential of targeting KMT2A or KMT2A-medicated pathways to block leukemogenesis. These results provide a foundation for targeting the KMT2A complex in KMT2A-amplified disease, representing a promising first step toward treating MDS/AML associated with KMT2A amplification.
 Citation Format: Jayme Ogino, Yang Liu, Wentao Yang, Madina Sukhanova, Sheng Li, Yali Dou. Epigenetic dynamics in KMT2A-amplified leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/245971e55eeef33c3dd5df76ff7015895c3cd83a" target='_blank'>
              Abstract A008: Epigenetic dynamics in KMT2A-amplified leukemia
              </a>
            </td>
          <td>
            Jayme Ogino, Yang Liu, Wentao Yang, M. Sukhanova, Sheng Li, Yali Dou
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Histones are key epigenetic factors for regulating the accessibility and compaction of eukaryotic genomes, affecting the replication, repair, and expression of DNA. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of over 12,000 patients. Overall, histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using a combination of genomic, structural, and biophysical analyses, we found several predominant modes of action, where cancer missense mutations in histones affected acidic patches and protein binding interfaces in a cancer-specific manner and targeted interaction sites with specific DNA repair proteins. Consistent with this finding, we observed a high tumour mutational burden in patients with histone mutations affecting interactions of DNA repair proteins. We also identified potential cancer driver mutations in several histone genes, including histone H4-a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00566c578efc8bbf332002783a58fe37b3053b70" target='_blank'>
              Cancer Histone Mutations Impact Binding and DNA Repair Processes, Leading to Increased Mutagenesis
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J. Aristizabal, Anna R Panchenko
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chemotherapy often kills a large fraction of cancer cells but leaves behind a small population of drug- tolerant persister cells. These persister cells survive drug treatments through reversible, non-genetic mechanisms and cause tumour recurrence upon cessation of therapy. Here, we report a drug tolerance mechanism regulated by the germ-cell-specific H3K4 methyltransferase PRDM9. Through histone proteomic, transcriptomic, lipidomic, and ChIP-sequencing studies combined with CRISPR knockout and phenotypic drug screen, we identified that chemotherapy-induced PRDM9 upregulation promotes metabolic rewiring in glioblastoma stem cells, leading to chemotherapy tolerance. Mechanistically, PRDM9-dependent H3K4me3 at cholesterol biosynthesis genes enhances cholesterol biosynthesis, which persister cells rely on to maintain homeostasis under chemotherapy- induced oxidative stress and lipid peroxidation. PRDM9 inhibition, combined with chemotherapy, resulted in strong anti-cancer efficacy in preclinical glioblastoma models, significantly enhancing the magnitude and duration of the antitumor response by eliminating persisters. These findings demonstrate a previously unknown role of PRDM9 in promoting metabolic reprogramming that enables the survival of drug-tolerant persister cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44ca423a805e163b74f1186c734661fa71cbd4de" target='_blank'>
              Histone methyltransferase PRDM9 promotes survival of drug-tolerant persister cells in glioblastoma
              </a>
            </td>
          <td>
            George L. Joun, Emma G. Kempe, Brianna Chen, Jayden Sterling, Ramzi H. Abbassi, W. Daniel du Preez, A. Recasens, Teleri Clark, Tian Y. Du, Jason K.K. Low, Hani Kim, Pengyi Yang, Jasmine Khor, Monira Hoque, Dinesh C. Indurthi, M. Kuchibhotla, Ranjith Palanisamy, William T Jorgensen, Andrew P. Montgomery, Jennifer R. Baker, S. Higginbottom, E. Tomaskovic-Crook, J. M. Crook, Lipin Loo, Bryan W. Day, G. G. Neely, Ernesto Guccione, Terrance G. Johns, Michael Kassiou, A. S. Don, L. Munoz
          </td>
          <td>2025-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="This paper provides a critical analysis of the molecular mechanisms presently used to explain transcriptional strand asymmetries of single base substitution (SBS) signatures observed in cancer genomes curated at the Catalogue of Somatic Mutations in Cancer (COSMIC) database (Wellcome Trust Sanger Institute). The analysis is based on a deaminase-driven reverse transcriptase (DRT) mutagenesis model of cancer oncogenesis involving both the cytosine (AID/APOBEC) and adenosine (ADAR) mutagenic deaminases. In this analysis we apply what is known, or can reasonably be inferred, of the immunoglobulin somatic hypermutation (Ig SHM) mechanism to the analysis of the transcriptional stand asymmetries of the COSMIC SBS signatures that are observed in cancer genomes. The underlying assumption is that somatic mutations arising in cancer genomes are driven by dysregulated off-target Ig SHM-like mutagenic processes at non-Ig loci. It is reasoned that most SBS signatures whether of “unknown etiology” or assigned-molecular causation, can be readily understood in terms of the DRT-paradigm. These include the major age-related “clock-like” SBS5 signature observed in all cancer genomes sequenced and many other common subset signatures including SBS1, SBS3, SBS2/13, SBS6, SBS12, SBS16, SBS17a/17b, SBS19, SBS21, as well as signatures clearly arising from exogenous causation. We conclude that the DRT-model provides a plausible molecular framework that augments our current understanding of immunogenetic mechanisms driving oncogenesis. It accommodates both what is known about AID/APOBEC and ADAR somatic mutation strand asymmetries and provides a fully integrated understanding into the molecular origins of common COSMIC SBS signatures. The DRT-paradigm thus provides scientists and clinicians with additional molecular insights into the causal links between deaminase-associated genomic signatures and oncogenic processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a875e39ec433bc73f24db2e1195da36a00350df" target='_blank'>
              Deaminase-Driven Reverse Transcription Mutagenesis in Oncogenesis: Critical Analysis of Transcriptional Strand Asymmetries of Single Base Substitution Signatures
              </a>
            </td>
          <td>
            Edward J. Steele, R. Lindley
          </td>
          <td>2025-01-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Single-stranded DNA secondary structures such as G-quadruplexes (G4s) can potentially disrupt transcription, replication, and repair. Using bio-informatic analysis, here we show that BMI1 is enriched at putative G4s flanked by heterochromatin domains, and that BMI1 knockdown in human dermal fibroblasts (HDFs) resulted in heterochromatin relaxation and G4 induction, followed by replication stress and genomic instability. In these cells, G4s co-localized with large 53BP1 and PCNA foci resembling replication catastrophes. Inhibiting transcription partly attenuated DNA damage, suggesting rescue of transcription-replication collisions at difficult-to-replicate sequences. In BMI1 knockdown or pyridostatin-exposed HDFs, the Werner helicase accumulated and co-localized with G4s. Acute WRN knockdown also resulted in G4 induction. In HDFs from Werner and Hutchinson-Gilford progeria syndromes, loss of heterochromatin and nuclear envelope anomalies were associated with G4 induction and DNA damage. Nuclear envelope anomalies were also prominent following BMI1 knockdown. These findings suggests that heterochromatin-mediated repression of G4s attenuates replication stress and genomic instability, and that this mechanism is shared across distinct progeroid models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/731920cfa53eda1eb3b00a50ee2eec0629da79d1" target='_blank'>
              BMI1-mediated heterochromatinization represses G-quadruplex DNA formation to maintain genomic stability during replication
              </a>
            </td>
          <td>
            Roy Hanna, Gilbert Bernier
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genome functions are regulated by the means of chromatin and 3D chromosome organization. Dynamic condensation and relaxation of chromosomes during cell cycle is largely controlled by the Condensin complexes. We mapped mutants in Condensin II subunits SMC2A, CAP-D3 and CAP-H2 as hypersensitive to DNA-protein crosslink (DPC) inducer zebularine. This suggested that Condensin II complex is required for resistance to genotoxic stress in Arabidopsis and prompted us to explore the underlying phenotypes. We show that the Condensin II role in resistance to zebularine is independent of the DNA damage response signaling by SOG1 and the homology-directed repair pathway. Furthermore, we found that Arabidopsis Condensin II mutants have incompletely condensed mitotic chromosomes and show abnormal chromatin connections during anaphase. Based on the structural and temporal analyses, we propose that the connections represent catenated chromatids. The chromosome connections were more frequent when the Condensin II mutants were treated by zebularine or TOP2 crosslinker and inhibitor ICRF-187. Altogether, we demonstrate that the proper large-scale chromatin organization by Condensin II is important for resistance to DNA damage response-inducing agents in Arabidopsis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6084d605a281f1ce54dec4a816d64fd93e825187" target='_blank'>
              Condensin II mediates efficient chromatid resolution and resistance to genotoxic stress in Arabidopsis
              </a>
            </td>
          <td>
            Jovanka Vladejić, Klára Procházková, E. Tomaštíková, Zynke de Cock, Jana Zwyrtková, Kateřina Kaduchová, Markéta Pavlíková, A. Pečinka
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047a24a952e93dd146dc76725bcf8e221c664c94" target='_blank'>
              Targeting chromosomally unstable tumors with a selective KIF18A inhibitor
              </a>
            </td>
          <td>
            Aaron F Phillips, Rumin Zhang, Mia Jaffe, Ryan Schulz, Marysol Chu Carty, Akanksha Verma, Tamar Feinberg, Michael D Arensman, Alan Chiu, Reka Letso, Nazario Bosco, Katelyn A. Queen, Allison R Racela, Jason K. Stumpff, Celia Andreu-Agullo, Sarah E. Bettigole, R. Depetris, Scott Drutman, Shinsan M Su, Derek A Cogan, Christina H Eng
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The interplay between genetics and cancer has been a focal point of research for decades, leading to profound understandings into the molecular mechanisms driving tumorigenesis. In this comprehensive review article, we explore the genetic basis of cancer, encompassing the diverse array of alterations that underline oncogenic transformation. From oncogenes to tumor suppressor genes, and from point mutations to chromosomal rearrangements, we delve into the molecular hallmarks of cancer and their implications for diagnosis, treatment, and prevention. Drawing on recent advancements in genomic technologies, we discuss the role of next-generation sequencing, single-cell sequencing, and computational modeling in unraveling the complexity of cancer genetics. Furthermore, we examine the clinical implications of genetic predisposition to cancer, highlighting the importance of genetic testing and counselling in cancer risk assessment and management. Through an exploration of tumor heterogeneity, clonal evolution, and therapeutic resistance, we underscore the challenges and opportunities in precision oncology. Finally, we discuss future directions in cancer genetics research, including precision prevention strategies and ethical considerations.Keywords: Cancer Genetics; Oncogenes; Tumor Suppressor Genes; Genetic Alterations; Precision Oncology; Clonal Evolution   ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b6286c0dc712fbb40ada9cedccccb4c8abdd42a" target='_blank'>
              Understanding the Role of Genetics in Tumour and Cancer Biology
              </a>
            </td>
          <td>
            A. Asiri, Amer Al Ali, Mohammed H. Abu-Alghayth
          </td>
          <td>2025-01-29</td>
          <td>Advancements in Life Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Human colorectal cancer (CRC) most often arises as a result of spontaneous genetic driver mutations such as APC, BRAF, KRAS, SMAD4, and TP53. These are among the most common drivers of CRC and are thought to contribute directly to cancer initiation and growth. CRC tumors harboring BRAFV600E mutation exhibit a CpG Island Methylator Phenotype (CIMP). Other drivers are also more weakly associated with distinct DNA methylation profiles. It is not clear whether these DNA methylation profiles are a result of differences in the cells-of-origin for these cancers, with pre-existing methylation patterns or whether there is selective pressure for various DNA methylation alterations that cooperate with each driver. To address this question, we have modeled this in mice by inducing colon tumors in a controlled cell population at the same age, with either BRAFF-V600E, KRASLSLG12D, Smad4, or Trp53 drivers in combination with Apc heterozygosity. We have profiled DNA methylation in tumors arising in these mice to determine the interaction between different driver mutations and the tumor methylome. Using a multiple linear regression model, we have identified DNA methylation alterations that are shared across all tumor samples, as well as changes that are specific to each of the different genetic drivers. The DNA methylation profiles accurately recapitulate well-established findings, including high levels of hypermethylation in KRASLSLG12D and BRAF F-V600E tumors reminiscent of CpG Island Methylator Phenotype (CIMP), universal tumor hypermethylation at binding sites for transcription factors involved in intestinal epithelial differentiation, and hypomethylation at AP1 complex component binding sites. Driver-specific histology findings broadly align with those observed in human cancers, such as sessile serrated morphology in BRAFF-V600E mutated tumors. The tumor models with different driver mutations also show variation in hypermethylation of polycomb repressive complex target sites and replicative history. We are using the differences in the DNA methylation profiles between the driver tumor models in conjunction with RNA-seq analysis and conservation of the alterations in human CRC to identify epigenetic events cooperative with the drivers and functionally relevant to tumorigenesis.
 Citation Format: Manpreet Kalkat, Toshinori Hinoue, Liang Kang, Peter W. Laird, Mary Olesnavich. Inducible mouse models of cancer driver mutations as a powerful tool for in vivo and epigenetic studies of colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c32f4b048765e136fdf9884436d3047629c3c3d9" target='_blank'>
              Abstract B025: Inducible mouse models of cancer driver mutations as a powerful tool for in vivo and epigenetic studies of colorectal cancer
              </a>
            </td>
          <td>
            Manpreet Kalkat, T. Hinoue, Liang Kang, Peter W. Laird, Mary Olesnavich
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract High hyperdiploid (HeH) B‐cell acute lymphoblastic leukemia (B‐ALL) is the most prevalent subtype of childhood ALL. This leukemia is characterized by trisomies and tetrasomies of specific chromosomes and additional point mutations. Here, we used single‐cell targeted DNA and antibody sequencing to determine the clonal evolution of HeH B‐ALL during development and chemotherapy treatment. Chromosomal copy number changes were mostly stable over all the leukemia cells, while mutations were typically subclonal. Within all 13 cases, at least one RAS mutant (KRAS or NRAS) subclone was detected (range: 1 to 4 subclones with RAS mutations), indicating the importance of RAS signaling in HeH B‐ALL development. NSD2 mutations were detected in 4 out of 13 cases and always in a subclone with RAS signaling mutations. Single‐cell DNA sequencing detected residual leukemia cells during chemotherapy treatment, and analysis of chromosomal copy number changes aided in the accurate detection of these cells. Our single‐cell data demonstrate that chromosomal changes are acquired prior to additional mutations and that RAS signaling mutations are present in all HeH cases, often as subclonal mutations. This single‐cell multi‐omics study enabled us to extensively characterize the genetic and surface protein heterogeneity in patients with HeH B‐ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5be0cc4c5cf6db382e785e540535d41eade046" target='_blank'>
              Single‐cell DNA and surface protein characterization of high hyperdiploid acute lymphoblastic leukemia at diagnosis and during treatment
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, O. Gielen, Jochen Lamote, Barbara Dewaele, M. Tajdar, Johan Maertens, Jolien de Bie, K. De Keersmaecker, N. Boeckx, Lucienne Michaux, A. Uyttebroeck, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2025-02-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Selection of a suitable chromatin environment during retroviral integration is a tightly regulated and multilayered process that involves interplay between viral and host factors. However, whether intrinsic chromatin dynamics during mitosis modulate retroviral genome invasion is currently poorly described. Direct interaction between the spumaretrovirus prototype foamy virus (PFV) Gag protein and cellular chromatin has been described as a major determinant for integration site selection. A previous Gag chromatin-binding site (CBS)–nucleosome co-crystal structure revealed an interaction with the histone H2A-H2B acidic patch via a highly conserved arginine anchor residue. Yet, the molecular mechanisms regulating Gag-chromatin capture during PFV infection remain obscure. Here, we investigated the kinetics of Gag-chromatin interactions during mitosis and proviral integration of PFV-infected synchronized cells. Using Gag CBS variant viruses, we showed that alteration of Gag affinity for nucleosome binding induced untimely chromatin tethering during mitosis, decreased infectivity and redistributed viral integration sites to markers associated with late replication timing of host chromosomes. Mutant Gag proteins were moreover defective in their ability to displace the histone H4 tail from the nucleosome acidic patch of highly condensed mitotic chromatin. These data indicate that the mitotic chromatin landscape during Gag–nucleosome interactions hosts PFV integration site selection determinants and that spumaretroviruses evolved high-affinity chromatin binding to overcome early mitosis chromatin condensation for optimal viral DNA tethering, integration and infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15c034dba5335395c865d3f3e018bafaae4ee93b" target='_blank'>
              Timed chromatin invasion during mitosis governs prototype foamy virus integration site selection and infectivity
              </a>
            </td>
          <td>
            F. Lagadec, P. K. Singh, C. Calmels, D. Lapaillerie, D. Lindemann, V. Parissi, P. Cherepanov, A. N. Engelman, P. Lesbats
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The xenogeneic silencer protein Lsr2 from Mycobacterium tuberculosis plays a critical role in its survival and pathogenesis. Lsr2 is a nucleoid-associated protein that interacts with DNA in vivo and regulates many genes. Purified Lsr2 forms nucleoprotein filaments with DNA molecules leading to highly compacted DNA conformations. However, the physical mechanism underlying Lsr2-mediated DNA compaction, resulting in gene regulation, remains elusive. We employed a combination of biochemical assay, single-molecule imaging, and molecular dynamics simulations to investigate the governing principles of Lsr2-mediated DNA compaction. We show that, while Lsr2 alone undergoes phase separation, addition of DNA substantially lowers the required concentration for its phase separation. Strikingly, our single-molecule and simulation data establish that Lsr2 forms condensates with long stretches of AT-rich DNA, providing strong evidence for sequence-dependent co-condensation. This observation is contrary to the classical view of sequence-dependent binding of individual protein molecules to DNA, our findings rather suggest that protein-DNA co-condensates ‘sense’ the average binding energy landscape. We present a physical model for Lsr2-mediated DNA compaction and gene regulation, describing a novel mechanism for NAP-mediated genome organization in bacteria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d84d86b264f5c1ad8de5c81e2ae317173c400dd" target='_blank'>
              Sequence-dependent co-condensation of Lsr2 with DNA elucidates the mechanism of genome compaction in Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Thejas Satheesh, R. Singh, Prakshi Gaur, Sneha Shahu, Saminathan Ramakrishnan, Mansi Srivastava, Shreyasi Neogi, Sandeep Choubey, Mahipal Ganji
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Random mutations followed by natural selection is one of the key mechanisms during natural evolution that results in the generation of new functions. This process is typically slow in nature because of low mutational frequency. In contrast, the B cells of mammalian immune systems have evolved somatic hypermutation (SHM) mechanisms that introduce mutations at the immunoglobulin genomic loci at a significantly higher frequency than the rest of the genome. SHM allows B cells to rapidly evolve new antibody sequences without compromising their fitness as a consequence of genome-wide mutations. In this work, we developed a continuous directed evolution platform in human B cell lines (CODE-HB) that recruits and repurposes the SHM mechanisms to rapidly evolve reporter proteins like the Green Fluorescent Protein. This approach uses a stable, non-immunoglobulin locus within the genomes of human B cell lines. To comprehensively characterize the mutational profile and breadth of this strategy, we performed single-molecule sequencing experiments. We illustrate the utility of the platform by rapidly evolving antibody fragments in a continuous manner by B cell surface display targeting avian subtypes of influenza hemagglutinin (e.g., H5) from emerging strains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a837142332a2e1fd6fe0933cdb79622e5382b338" target='_blank'>
              Virus-free continuous directed evolution in human cells using somatic hypermutation
              </a>
            </td>
          <td>
            Stanley Bram, Jordan Quenneville, Graeme Lindsey, Sarah Leach, Jenna J. Guthmiller, Angad P Mehta
          </td>
          <td>2024-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9ab65a9ee1a176544b4d48a367f9b18440b017c" target='_blank'>
              R-loops acted on by RNase H1 influence DNA replication timing and genome stability in Leishmania
              </a>
            </td>
          <td>
            J. Damasceno, E. Briggs, M. Krasiļņikova, Catarina A Marques, C. Lapsley, R. McCulloch
          </td>
          <td>2025-02-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Transposable elements (TEs) comprise the majority of the human genome. In most somatic tissues, TEs are silenced by DNA methylation and histone modifications. Tumors exhibit changes in DNA methylation including global loss of methylation at regions that are silenced for genome stability, like TEs, and gain of methylation at the promoter regions of tumor suppressor genes. We showed that low doses of DNMT inhibitors (DNMTis) activate type I interferon signaling by reducing methylation and increasing expression of TEs that activate dsRNA sensors. DNMTis increase TEs in murine models of cancer, activating interferon signaling and recruiting CD8+ T cells to kill cancer cells and sensitize to immune checkpoint blockade therapy. We performed a genome-wide analysis of the response to DNMTi in a panel of ovarian cancer cell lines. Alu elements, which directly bind to the dsRNA sensor MDA5, are the most significantly demethylated TEs after DNMTi treatment. Our RNA-sequencing analyses indicated that A-to-I RNA editing of TEs by the RNA editing enzyme ADAR1 was increased after DNMTi treatment. This A-to-I editing makes TE RNA less immunogenic and inhibits the immune response to DNMTi. We find that combining ADAR1 knockdown and DNMTi treatment significantly increases type I interferon signaling and cancer cell secretion of downstream cytokines, including CCL5 and CXCL10. Further, this combination increased recruitment and activation of host immune cells and significantly improved survival in the aggressive ID8 Trp53-/- murine model of ovarian cancer. CITE-Seq (cellular indexing of transcriptomes and epitopes, a type of single-cell RNA sequencing that utilizes antibody tagging of each cell type before sequencing) demonstrated significantly higher infiltration of T and B cells in the tumor microenvironment. To understand the immune cell types causing the DNMTi anti-tumor response, we used antibodies to deplete CD8 T cells and NK cells in the ID8 Trp53-/- model with ADAR1 knockdown and DNMTi treatment. In addition, we performed these experiments in the C57BL/6-Tg(IghelMD4)4Ccg/J mice, which only produce antibodies against “Hel” antigen and have no other B cell responses. Of these lymphocyte populations, depleting T cells had the biggest effect and we observed specific killing of tumor cells by T cells isolated from the mice. We performed immunopeptidomics on cancer cell lines with and without DNMTi and showed that DNMTi upregulate TE peptide presentation by MHC molecules on the surface of tumor cells. Using a novel immuno-engineering approach, we are able to generate antigen-specific T cells targeting TEs and other antigens in murine and human cancer cell line models. These T cells specifically recognize (produce IFNgamma and TNFalpha) and kill the target tumor cells. As DNMTi have also been shown to demethylate genes crucial for T cell function, including IFNgamma and TCF7, we are currently combining these T cells with DNMTis for a novel cell therapy against cancer.
 Citation Format: Katherine B Chiappinelli. Epigenetic regulation of transposable elements to reverse cancer immune evasion [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr PR008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d21853f30868268b4b8dd99d7c90c4354a570de" target='_blank'>
              Abstract PR008: Epigenetic regulation of transposable elements to reverse cancer immune evasion
              </a>
            </td>
          <td>
            Katherine B. Chiappinelli
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="DNA viruses at once elicit and commandeer host pathways, including DNA repair pathways for virus replication. Despite encoding its own DNA polymerase and processivity factor, human cytomegalovirus (HCMV) recruits the cellular processivity factor, proliferating cell nuclear antigen (PCNA) and specialized host DNA polymerases involved in translesion synthesis (TLS) to replication compartments (RCs) where viral DNA (vDNA) is synthesized. While the recruitment of TLS polymerases is important for viral genome stability, the role of PCNA is poorly understood. PCNA function in DNA repair is regulated by monoubiquitination (mUb) or SUMOylation of PCNA at lysine 164 (K164). We find that mUb-PCNA increases over the course of infection, and modification of K164 is required for PCNA-mediated restriction of virus replication. mUb-PCNA plays important known roles in recruiting TLS polymerases to DNA, which we have shown are important for viral genome integrity and diversity, represented by novel junctions and single nucleotide variants (SNVs), respectively. We find that PCNA drives SNVs on vDNA similar to Y-family TLS polymerases, but that this did not require modification at K164. Unlike TLS polymerases, PCNA was dispensable for preventing large scale rearrangements on vDNA. These striking results suggest separable PCNA-dependent and - independent functions of TLS polymerases on vDNA. By extension, these results imply roles for TLS polymerase beyond their canonical function in TLS in host biology. These findings highlight PCNA as a complex restriction factor for HCMV infection, likely with multiple distinct roles, and provides new insights into the PCNA-mediated regulation of DNA synthesis and repair in viral infection. IMPORTANCE Genome synthesis is a critical step of virus life cycles and a major target of antiviral drugs. Human cytomegalovirus, like other herpesviruses, encodes machinery sufficient for viral DNA synthesis and relies on host factors for efficient replication. We have shown that host DNA repair factors play important roles in HCMV replication, but our understanding of this is incomplete. Building on previous findings that specialized host DNA polymerases contribute to HCMV genome integrity and diversity, we sought to determine the importance of PCNA, the central polymerase regulator. PCNA associates with nascent viral DNA and restricts HCMV replication. While PCNA is dispensable for genome integrity, it contributes to genome diversity. Our findings suggest that host polymerases function on viral genomes by separable PCNA-dependent and - independent mechanisms. Through revealing complex roles for PCNA in HCMV replication, this study expands the repertoire of host DNA synthesis and repair proteins hijacked by this ubiquitous herpesvirus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afe97c65ecc13c9b2841234ccb055156970288c9" target='_blank'>
              Complex roles for proliferating cell nuclear antigen in restricting human cytomegalovirus replication
              </a>
            </td>
          <td>
            P. Longmire, Olivia Daigle, S. Zeltzer, Matias Lee, Marek Svoboda, Marco Padilla-Rodriguez, Carly Bobak, Giovanni Bosco, F. Goodrum
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Genome maintenance is essential for the integrity of the genetic blueprint, of which only a small fraction is transcribed in higher eukaryotes. DNA lesions occurring in the transcribed genome trigger transcription pausing and transcription-coupled DNA repair. There are two major transcription-coupled DNA repair pathways. The transcription-coupled nucleotide excision repair (TC-NER) pathway has been well studied for decades, while the transcription-coupled homologous recombination repair (TC-HR) pathway has recently gained attention. Importantly, recent studies have uncovered crucial roles of RNA transcripts in TC-HR, opening exciting directions for future research. Transcription also plays pivotal roles in regulating the stability of highly specialized genomic structures such as telomeres, centromeres, and fragile sites. Despite their positive function in genome maintenance, transcription and RNA transcripts can also be the sources of genomic instability, especially when colliding with DNA replication and forming unscheduled pathological RNA:DNA hybrids (R-loops), respectively. Pathological R-loops can result from transcriptional stress, which may be induced by transcription dysregulation. Future investigation into the interplay between transcription and DNA repair will reveal novel molecular bases for genome maintenance and transcriptional stress-associated genomic instability, providing therapeutic targets for human disease intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd6afacedaa734f271cf97ef11d6135943f79a8" target='_blank'>
              Transcription as a double-edged sword in genome maintenance.
              </a>
            </td>
          <td>
            Ouyang Jian
          </td>
          <td>2024-12-20</td>
          <td>FEBS letters</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a heterogeneous and aggressive brain tumour that is invariably fatal despite maximal treatment. Genetic or transcriptomic ‘biomarkers’ could be used to stratify patients for treatments, however, pairing biomarkers with appropriate therapeutic ‘targets’ is challenging. Consequently, therapeutics have not yet been optimised for specific GBM patient subsets. Here we integrate genome-wide CRISPR/Cas9 knockout screening and genetic-annotation data for 60 distinct patient-derived, IDHwildtype, adult GBM cell lines, quantifying the essentiality of 15,145 genes. We describe a novel method using Targeted Learning, to estimate the effect size of GBM-relevant biomarkers on context-dependent gene essentiality (GBM-CoDE). We derive multiple target-biomarker pair hypotheses, which we release in an accessible platform to accelerate translation to biomarker-stratified clinical trials. Two of these (WWTR1 with EGFR mutation/amplification, and VRK1 with VRK2 expression suppression) have been validated in GBM, implying that our additional novel findings may be valid. Our method is readily translatable to other cancers of unmet need.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1cf5308b3e1f097593cf8ac72787c4818c5257d" target='_blank'>
              Systematic identification of context-dependent gene-essentiality in Glioblastoma: The GBM-CoDE platform
              </a>
            </td>
          <td>
            Mitchell T Foster, Ailith Ewing, M. Frame, Paul M Brennan, A. Khamseh, S. Beentjes, Neil O. Carragher, Colin A. Semple
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Whole-genome duplications (WGD) and transposable element (TE) activity can act synergistically in genome evolution. WGDs can increase TE activity directly through cellular stress or indirectly by relaxing selection against TE insertions in functionally redundant, duplicated regions. Because TEs can function as, or evolve into, TE-derived cis-regulatory elements (TE-CREs), bursts of TE activity following WGD are therefore likely to impact evolution of gene regulation. Yet, the role of TEs in genome regulatory evolution after WGDs is not well understood. Here we used Atlantic salmon as a model system to explore how TE activity after the salmonid WGD ~100MYA shaped CRE evolution. We identified 55,080 putative TE-CREs using chromatin accessibility data from liver and brain. Retroelements were both the dominant source of TE-CREs and had higher regulatory activity in MPRA experiments compared to DNA elements. A minority of TE-subfamilies (16%) accounted for 46% of TE-CREs, but these "CRE-superspreaders" were mostly active prior to the WGD. Analysis of individual TE insertions, however, revealed enrichment of TE-CREs originating from WGD-associated TE activity, particularly for the DTT (Tc1-Mariner) DNA elements. Furthermore, coexpression analyses supported the presence of TE-driven gene regulatory network evolution, including DTT elements active at the time of WGD. In conclusion, our study supports a scenario where TE activity has been important in genome regulatory evolution, either through relaxed selective constraints, or strong selection to recalibrate optimal gene expression phenotypes, during a transient period following genome doubling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08496aae0238cce493a4d3baa60b2ce62c4ee152" target='_blank'>
              The role of transposon activity in shaping cis-regulatory element evolution after whole genome duplication.
              </a>
            </td>
          <td>
            Oystein Monsen, L. Grønvold, Alex Datsomor, T. Harvey, James Kijas, Alexander Suh, T. R. Hvidsten, S. Sandve
          </td>
          <td>2025-02-12</td>
          <td>Genome research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Kaposi Sarcoma (KS) is a frequently aggressive malignancy caused by Kaposi sarcoma herpesvirus (KSHV/HHV-8). People with immunodeficiencies, including HIV, are at increased risk for developing KS, but our understanding of the contributions of the cellular genome to KS pathogenesis remains limited. To determine if there are cellular genetic alterations in KS that might provide biological or therapeutic insights, we performed whole exome sequencing on 78 KS tumors and matched normal control skin from 59 adults with KS (46 with HIV-associated KS and 13 with HIV-negative KS) receiving treatment at the Uganda Cancer Institute in Kampala, Uganda. We found a very low mutational burden in all but one specimen (median=11 mutations), which is the lowest number of mutations among all 33 tumor types in The Cancer Genome Atlas (TCGA). No recurrent mutations were seen and the most commonly affected oncogenic pathway was RTK/RAS. Mutational signatures included defective DNA mismatch repair and smoking. There was no evidence suggesting that multiple tumors from the same patient originated from the same original clone. The number of genome copy alterations per genome were higher in tumors from those without HIV infection and in tumors from participants with advanced stage disease, suggesting that lesions that take longer to develop may accumulate more alterations, although the number of alterations remain low compared to other cancers. Implications: Our findings indicate that the pathogenesis of KS differs from other malignancies, and that the primary driver of carcinogenesis is KSHV viral infection and expression of viral oncogenes, rather than clonal oncogenic transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a654923c04481c427a68db628b9b8ba07960d87" target='_blank'>
              Exome sequencing reveals a sparse genomic landscape in Kaposi sarcoma.
              </a>
            </td>
          <td>
            Warren Phipps, B. Bhinder, Andrea Towlerton, Peter Mooka, J. Kafeero, Matthew Fitzgibbon, O. Elemento, Ethel Cesarman
          </td>
          <td>2025-01-30</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Short tandem repeats (STRs) have been reported to influence gene expression across various human tissues. While STR variations are enriched in colorectal (CRC), stomach (STAD) and endometrial (UCEC) cancers, particularly in microsatellite instable (MSI) tumors, their functional effects and regulatory mechanisms on gene expression remain poorly understood across these cancer types. Results Here, we leverage whole-exome sequencing and gene expression data to identify STRs for which repeat lengths are associated with the expression of nearby genes (eSTRs) in CRC, STAD and UCEC tumors. Our analyses reveal that tumor STR profiles effectively capture both MSI phenotype and population structure. While most eSTRs are cancer-specific, shared eSTRs across multiple cancers exhibit consistent effects on gene expression. Notably, coding-region eSTRs identified in all three cancer types show positive correlations with nearby gene expression. We further validate the functional effects of eSTRs by demonstrating associations between somatic eSTR mutations and gene expression changes during the transition from normal to tumor tissues, suggesting their potential roles in tumorigenesis. Combined with DNA methylation data, we perform the first quantitative analysis of the interplay between STR variations and DNA methylation in tumors. We identify eSTRs where repeat lengths are associated with methylation levels of nearby CpG sites (meSTRs) and show that over 70% of eSTRs are significantly linked to local DNA methylation. Importantly, the effects of meSTRs on DNA methylation remain consistent across cancer types. Conclusions Overall, our findings enhance the understanding of how functional STR variations influence gene expression and DNA methylation. Our study highlights shared regulatory mechanisms of STRs across multiple cancers, offering a foundation for future research into their broader implications in tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a625ce20d0d6f7f915320a2751a41cca4a94608" target='_blank'>
              Multicancer analyses of short tandem repeat variations reveal shared gene regulatory mechanisms
              </a>
            </td>
          <td>
            Feifei Xia, M. Verbiest, Oxana Lundström, Tugce Bilgin Sonay, M. Baudis, Maria Anisimova
          </td>
          <td>2025-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Genetic mutations are well known to influence tumorigenesis, tumor progression, treatment response and relapse, but the role of epigenetic variation in cancer progression is still largely unexplored. The lack of epigenetic understanding in cancer evolution is in part due to the limited availability of methods to examine such a heterogeneous disease. However, in the last decade the development of several single-cell methods to profile diverse chromatin features (chromatin accessibility, histone modifications, DNA methylation, etc.) has propelled the study of cancer epigenomics. In this review, we detail the current landscape of single-omic and multi-omic single-cell methods with a particular focus on the examination of histone modifications. Furthermore, we provide recommendations on both the application of these methods to cancer research and how to perform initial computational analyses. Together, this review serves as a referential framework for incorporating single-cell methods as an important tool for tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e561846c42367eca2d20c4639ff6a8849a7d2d" target='_blank'>
              A hitchhiker's guide to single-cell epigenomics: Methods and applications for cancer research.
              </a>
            </td>
          <td>
            Marta Moreno-Gonzalez, Isabel Sierra, Jop Kind
          </td>
          <td>2024-12-22</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42786039d5d90ecc6efc329111435b0ff4acfda0" target='_blank'>
              Homologous recombination deficiency (HRD) diagnostics: underlying mechanisms and new perspectives.
              </a>
            </td>
          <td>
            Andrey Kechin, M. Koryukov, Regina Mikheeva, M. Filipenko
          </td>
          <td>2024-12-26</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d06e293f47afb8c1e77d741db124642957b624" target='_blank'>
              Characterization of RNF144B and PPP2R2A identified by a novel approach using TCGA data in ovarian cancer
              </a>
            </td>
          <td>
            P. Manasa, S. Krishnapriya, C. Sidhanth, S. Vasudevan, Kanchan Murhekar, T. S. Ganesan
          </td>
          <td>2025-02-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 During solid tumor development, gene-poor and lamina-associated genomic regions undergo progressive loss of DNA methylation, whereas promoter CpG islands become DNA hypermethylated. We report evidence suggesting that the recruitment of DNMT3A by histone modifications contributes to this cancer-associated DNA methylome reprogramming. The genomic targeting of DNMT3A is regulated by the competition between NSD1-mediated histone H3K36 methylation and polycomb-mediated H2AK119 ubiquitination, via DNMT3A's PWWP domain and N-terminal ubiquitin-interacting region, respectively. Mutations in NSD1, DNMT3A reader domains or polycomb complexes perturb DNMT3A recruitment and can recapitulate patterns of DNA methylation observed in patient tumors. Lastly, tumors exhibiting progressive hypomethylation, such as those harboring NSD1 mutations, are paradoxically immune-cold despite the increased expression of retrotransposons. These results suggest that DNA hypomethylation and aberrant retrotransposon activation are significant bottlenecks to tumor development and may represent promising therapeutic targets.
 Citation Format: Chao Lu. Interplay between histone modifications and DNA methylation in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr PR004.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8283805e8d954027bac5a7fd108f9580c6417e4d" target='_blank'>
              Abstract PR004: Interplay between histone modifications and DNA methylation in cancer
              </a>
            </td>
          <td>
            Chao Lu
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The human fungal pathobiont Candida albicans displays extensive genomic plasticity, including large-scale chromosomal changes such as aneuploidy. Chromosome trisomy appears frequently in natural and laboratory strains of C. albicans. Trisomy of specific chromosomes has been linked to large phenotypic effects, such as increased murine gut colonization by strains trisomic for chromosome 7 (Chr7). However, studies of whole-chromosome aneuploidy are generally limited to the SC5314 genome reference strain, making it unclear whether the imparted phenotypes are conserved across C. albicans genetic backgrounds. Here, we report the presence of a Chr7 trisomy in the “commensal-like” oral candidiasis strain, 529L, and dissect the contribution of Chr7 trisomy to colonization and virulence in 529L and SC5314. These experiments show that strain background and homolog identity (i.e., AAB vs ABB) interact with Chr7 trisomy to alter commensal and virulence phenotypes in multiple host niches. In vitro filamentation was the only phenotype altered by Chr7 trisomy in similar ways across the two strain backgrounds. Oral colonization of mice was increased by the presence of a Chr7 trisomy in 529L but not SC5314; conversely, virulence during systemic infection was reduced by Chr7 trisomy in SC5314 but not 529L. Strikingly, the AAB Chr7 trisomy in the SC5314 background rendered this strain avirulent in murine systemic infection. Increased dosage of NRG1 failed to reproduce most of the Chr7 trisomy phenotypes. Our results demonstrate that aneuploidy interacts with background genetic variation to produce complex phenotypic patterns that deviate from our current understanding in the genome reference strain. AUTHOR SUMMARY Candida albicans is a clinically important fungus of humans that is also part of the microbiota that typically colonizes our bodies. Changes in the number of copies of chromosome 7 (Chr7) in the genome of the reference strain of C. albicans enhances host colonization. We identified a third copy of Chr7 in a strain classified as a good colonizer of humans that doesn’t cause disease, mimicking the increased colonization of the reference strain. Here, we demonstrate that the number of copies of Chr7 alters phenotypes differently between these strains, and three copies of Chr7 does not universally produce a strain that is only a better commensal strain. Additionally, which copy of Chr7 in this diploid species exists as the third copy influences commensal and pathogen phenotypes. Therefore, trisomic changes of the same chromosome in different C. albicans strains have substantially different effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2332e9447e18d3247be54ad40b31742ed672d8fa" target='_blank'>
              Strain background interacts with chromosome 7 aneuploidy to determine commensal and virulence phenotypes in Candida albicans
              </a>
            </td>
          <td>
            Abhishek Mishra, N. Solis, Siobhan M Dietz, Audra L Crouch, S. Filler, Matthew Z. Anderson
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Mitosis is a critical phase of the cell cycle and a vulnerable point where cancer cells can be effectively disrupted, leading to cell death and inhibition of tumor growth. However, challenges such as drug resistance remain significant in clinical applications. During mitosis, mRNA translation is generally downregulated, while non-canonical translation of specific transcripts proceeds. Here, we demonstrate that mitotic cancer cells redistribute ribosomes toward the 5’ untranslated region (5’ UTR) and the start of the coding sequence (CDS), enhancing the translation of thousands of upstream open reading frames (uORFs) and upstream overlapping open reading frames (uoORFs). This mitotic induction of uORF/uoORF enriches the presentation of immunopeptides at the surface of cancer cells following treatment with mitotic inhibitors. Functional assays indicate the potential of such neoepitopes to provoke cancer-cell killing by T cells. Altogether, our findings highlight the therapeutic potential of targeting uORF/uoORF-derived neoepitopes in combination with mitotic inhibitors to enhance immune recognition and tumor cell elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e58b9a8310537bb25bde77518c59ca5cd45e675f" target='_blank'>
              Upstream open reading frame translation enhances immunogenic peptide presentation in mitotically arrested cancer cells
              </a>
            </td>
          <td>
            Alexander Kowar, Jonas P. Becker, Rossella Del Pizzo, Zhiwei Tang, Julien Champagne, Pierré-René Körner, Jasmine Montenegro Navarro, Fiona Megan Tilghman, Hanan Sakeer, Angelika B. Riemer, R. Agami, Fabricio Loayza-Puch
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="The genome is dynamically reorganized, partitioned, and divided during mitosis. Despite their role in organizing interphase chromatin, transcription factors were largely believed to be mitotic spectators evicted from chromatin during mitosis, only able to reestablish their position on DNA upon entry into G1. However, a panoply of evidence now contradicts this early belief. Numerous transcription factors are now known to remain active during mitosis to achieve diverse purposes, including chromosome condensation, regulation of the centromere/kinetochore function, and control of centrosome homeostasis. Inactivation of transcription factors during mitosis results in chromosome segregation errors, key features of cancer. Moreover, active transcription and the production of centromere-derived transcripts during mitosis are also known to play key roles in maintaining chromosomal stability. Finally, many transcription factors are associated with chromosomal instability through poorly defined mechanisms. Herein, we will review the emerging roles of transcription factors and transcription during mitosis with a focus on their role in promoting the faithful segregation of sister chromatids.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b9bda3d766e3ac7d2259564d99c0e6e216b7356" target='_blank'>
              Emerging Roles for Transcription Factors During Mitosis
              </a>
            </td>
          <td>
            Samuel Flashner, Jane Azizkhan-Clifford
          </td>
          <td>2025-02-12</td>
          <td>Cells</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA mutations and methylation often contribute to disease development in a synergistic manner. While duplex sequencing is the most accurate method for detecting DNA mutations, it typically lacks the ability to simultaneously assess methylation or requires many reads. Here, we developed Methyl-CODEC to enable simultaneous methylation sequencing and duplex sequencing using single read pairs. To achieve this, Methyl-CODEC links an enzymatically deaminated sense strand to the reverse complement of the antisense strand, which is protected from conversion by using conversion-resistant dCTPs in the strand linking step of CODEC. Methyl-CODEC shows high concordance with standard enzymatic or bisulfite based whole genome methylation sequencing, while also uniquely preserving the original DNA sequence. This improves genetic sequencing accuracy, enables better alignment for next-generation sequencing (NGS), and distinguishes C>T mutations from unmethylated Cs. It also identifies rare mutations including those producing methylated Cs, which are enriched in CpG contexts. Methyl-CODEC opens new horizons for enhanced detection of biomarkers in cancer and molecular medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9e8bc3a0b30b9eaac4e88cab2287bf68732738e" target='_blank'>
              Methyl-CODEC enables simultaneous methylation and duplex sequencing
              </a>
            </td>
          <td>
            Ruolin Liu, Farzaneh Darbeheshti, Laurel Walsh, Rachel Li, Jin-hyung Bae, Hayet Radia Zeggar, Azeet Narayan, Kan Xiong, G. M. Makrigiorgos, V. Adalsteinsson
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Prostate cancer is a prevalent tumor type that, despite being highly curable, progresses to metastatic disease in a fraction of patients, thus accounting for more than 350.000 annual deaths worldwide. In turn, uncovering the molecular insights of metastatic disease is instrumental to improve the survival rate of prostate cancer patients. By means of gene expression metanalysis in multiple prostate cancer patient cohorts, we identified a set of genes that are differentially expressed in aggressive prostate cancer. Transcription factor 19 (TCF19) stood out as an unprecedented epithelial gene upregulated in metastatic disease, with prognostic potential and associated with the activity of androgen receptor. By combining computational and empiric approaches, our data revealed that TCF19 is required for full metastatic capacity and its depletion influences core cancer-related processes, such as vascular permeability, supporting the role of this gene in the dissemination of prostate tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ecdf804c61fcb68669b4e697cde8e1d82f6075a" target='_blank'>
              Transcription factor 19 is an androgen responsive gene that modulates vessel homeostasis and sustains metastatic prostate cancer
              </a>
            </td>
          <td>
            Amaia Ercilla, Jana R. Crespo, Saioa Garcia-Longarte, Marta Fidalgo, Natalia Martín-Martín, Onintza Carlevaris, I. Astobiza, Sonia Fernández-Ruiz, M. Guiu, Laura Bárcena, Isabel Mendizabal, A. Aransay, M. Graupera, R. Gomis, Arkaitz Carracedo
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="High-resolution conformation capture assays such as HiChIP are commonly used for profiling chromatin loops among cis-regulatory elements including enhancers and promoters. Detection of differential loops between two conditions (e.g., same cell type different genotypes or before/after perturbations) help contextualize condition-specific activities of genes in connection with such cis-regulatory elements. Existing differential loop callers for HiChIP data employ count-based models that are designed with gene expression data in mind and, hence, do not account for the distance decay of contact counts from HiChIP data. These approaches are not ideal for detection of differential long-range (>400Kb) loops, a limitation that persists even with the use of implicit or explicit corrections for this distance effect. We have implemented DiffHiChIP, the first comprehensive framework to call differential loops from HiChIP and similar 3C protocols. DiffHiChIP supports both DESeq2 and edgeR using either complete contact map or a subset of contacts (filtered) for background estimation, incorporates edgeR with generalized linear model (GLM) using either quasi-likelihood F-test or likelihood ratio test, and implements independent hypothesis weighting (IHW) as well as a distance stratification technique for modeling distance decay of contacts in estimating their statistical significance. Our results on 5 different datasets, each with two conditions or cell types, suggest that edgeR GLM-based models with IHW correction capture differential interactions, including long-range, that are supported by published Hi-C data and reference studies. Given the increasing trend of generating and utilizing HiChIP data for modeling chromatin regulation, DiffHiChIP promises to have broad impact and utility in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5b3ae5c6ae828629c6c5b98d37e7980de17f06f" target='_blank'>
              DiffHiChIP: Identifying differential chromatin contacts from HiChIP data
              </a>
            </td>
          <td>
            Sourya Bhattacharyya, Daniela Salgado Figueroa, Katia Georgopoulos, F. Ay
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Cancer cells display high levels of oncogene-induced replication stress (RS) and rely on DNA damage checkpoint for viability. This feature is exploited by cancer therapies to either increase RS to unbearable levels or inhibit checkpoint kinases involved in the DNA damage response. Thus far, treatments that combine these two strategies have shown promise but also have severe adverse effects. To identify novel, better-tolerated anticancer combinations, we screened a collection of plant extracts and found two natural compounds from the plant, Psoralea corylifolia, that synergistically inhibit cancer cell proliferation. Bakuchiol inhibited DNA replication and activated the checkpoint kinase CHK1 by targeting DNA polymerases. Isobavachalcone interfered with DNA double-strand break repair by inhibiting the checkpoint kinase CHK2 and DNA end resection. The combination of bakuchiol and isobavachalcone synergistically inhibited cancer cell proliferation in vitro. Importantly, it also prevented tumor development in xenografted NOD/SCID mice. The synergistic effect of inhibiting DNA replication and CHK2 signaling identifies a vulnerability of cancer cells that might be exploited by using clinically approved inhibitors in novel combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/042cde9aa3dc3d91bb528dfff54349708df45e8a" target='_blank'>
              Synergistic effect of inhibiting CHK2 and DNA replication on cancer cell growth
              </a>
            </td>
          <td>
            Flavie Coquel, Sing-Zong Ho, K. Tsai, Chun-Yen Yang, Antoine Aze, Julie Devin, Ting-Hsiang Chang, Marie Kong-Hap, Audrey Bioteau, J. Moreaux, Domenico Maiorano, Philippe Pourquier, Wen-Chin Yang, Yea-Lih Lin, Philippe Pasero
          </td>
          <td>2025-01-31</td>
          <td>eLife</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="X Chromosome Inactivation (XCI) equalizes X-linked gene expression between sexes. B cells exhibit dynamic XCI, with Xist RNA/heterochromatic marks absent on the inactive X (Xi) in naive B cells but returning following mitogenic stimulation. The impact of dynamic XCI on Xi structure and maintenance was previously unknown. Here, we find dosage compensation of the Xi with state-specific XCI escape genes in naive and in vitro activated B cells. Allele-specific OligoPaints indicate similar Xi and Xa territories in B cells that are less compact than in fibroblasts. Allele-specific Hi-C reveals a lack of TAD-like structures on the Xi of naive B cells, and stimulation-induced alterations in TAD-like boundary strength independent of gene expression. Notably, Xist deletion in B cells changes TAD boundaries and large-scale Xi compaction. Altogether, our results uncover B cell-specific Xi plasticity which could underlie sex-biased biological mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf439585ea0599081bd08309bf4216f355afcfa4" target='_blank'>
              B cell stimulation changes the structure and higher-order organization of the inactive X chromosome
              </a>
            </td>
          <td>
            Isabel Sierra, Natalie E. Toothacre, Robin H. van der Weide, Claudia D. Lovell, Son C. Nguyen, R. Jordan Barnett, Ashley L. Cook, Han-Seul Ryu, Sarah Pyfrom, Harrison Wang, D. Beiting, Jennifer E. Philips-Cremins, Eric F. Joyce, Montserrat C. Anguera
          </td>
          <td>2025-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="H3K4me3 is a fundamental and highly conserved chromatin mark across eukaryotes, playing a central role in many genome-related processes, including transcription, maintenance of cell identity, DNA damage repair, and meiotic recombination. However, identifying the causal function of H3K4me3 in these diverse pathways remains a challenge, and we lack the tools to manipulate it for agricultural benefit. Here we use the CRISPR-based SunTag system to direct H3K4me3 methyltransferases in the model plant, Arabidopsis thaliana. Targeting of SunTag-SDG2 activates the expression of the endogenous reporter gene, FWA. We show that SunTag-SDG2 can be employed to increase pathogen resistance by targeting the H3K4me3-dependent disease resistance gene, SNC1. Meiotic crossover recombination rates impose a limit on the speed with which new traits can be transferred to elite crop varieties. We demonstrate that targeting of SunTag-SDG2 to low recombining centromeric regions can significantly stimulate crossover formation. Finally, we reveal that the effect is not specific to SDG2 and is likely dependent on the H3K4me3 mark itself, as the orthogonal mammalian-derived H3K4me3 methyltransferase, PRDM9, produces a similar effect on gene expression with reduced off-target potential. Overall, our study supports an instructive role for H3K4me3 in transcription and meiotic recombination and opens the door to precise modulation of important agricultural traits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7c9ac7e7e4f2e661d415bc49c2d94c638f0dbf9" target='_blank'>
              CRISPR targeting of H3K4me3 activates gene expression and unlocks centromeric crossover recombination in Arabidopsis
              </a>
            </td>
          <td>
            Jenia Binenbaum, V. Adamkova, Hannah Fryer, Linhao Xu, Nicola Gorringe, Piotr Włodzimierz, Robin Burns, Ashot Papikian, Steve Jacobsen, Ian R Henderson, C. Harris
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Transposable elements (TEs), once considered as "junk" DNA, are now recognized for their epigenetic and regulatory influence, especially in cancer. Cryptic regulatory elements within TEs can be epigenetically reactivated in cancer, contributing to oncogenesis and altering gene expression patterns. This reactivation results in TE-derived transcripts, which have the potential to be translated and form neoantigens that can be presented by the human leukocyte antigens (HLA) complex and recognized by T cells, ultimately impacting the tumor immune response. In this study, we analyzed RNA-seq data from melanoma patients undergoing PD1/PDL1 immunotherapy. By identifying tumor specific TE-derived transcripts and utilizing in silico translation, we detected TE-derived neoantigens in patients with differing responses to immunotherapy. Our findings indicate that the number of TE-derived neoantigens is correlated with patients' response to immunotherapy, with a higher number of neoantigens observed in patients who responded to treatment compared to those who did not. These results imply that TE-derived neoantigens may serve as potential biomarkers for predicting the prognosis of PD1/PDL1 immunotherapy treatment. Incorporating TE-derived neoantigen profiles into clinical decision-making could enhance patient selection for immunotherapy, thereby improving treatment outcomes.
 Citation Format: Yixin Wu, Ting Wang, Yonghao Liang. Epigenetic reactivation of transposable elements as prognostic markers for PD1/PDL1 immunotherapy in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B007.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c61593260331e8272d84d80f0c9cc19c5bdc07d" target='_blank'>
              Abstract B007: Epigenetic reactivation of transposable elements as prognostic markers for PD1/PDL1 immunotherapy in cancer
              </a>
            </td>
          <td>
            Yixin Wu, Ting Wang, Yonghao Liang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chromosomes and their constituent compartments are core elements of the cell division machinery. In mammalian oocytes, defects in the structure or composition of chromosomes are a leading cause of aberrant or failed meiosis, and by extension, can cause miscarriage and infertility. The underlying mechanisms are poorly understood, but are critical for development of novel diagnostics and treatments. The chromosome periphery, the least understood chromosome compartment, has recently emerged as an essential component of mitotic chromosomes and somatic cell division. However, in female meiosis it remains completely unexplored. This study provides the first comprehensive survey of a meiotic chromosome periphery compartment in both human and mouse oocytes. Using a combination of time-lapse microscopy, super-resolution imaging and 3DCLEM we show that removing the chromosome periphery, via Ki67 depletion, has substantial negative impact on chromosome structure, spatial organization and positional awareness, with many oocytes stalling and arresting in meiosis I. Importantly, we also reveal key differences between the mitotic and oocyte meiotic chromosome periphery compartments, most remarkably in the retention of Ki67 in the oocyte through anaphase I, where unwanted chromosomes are stripped of Ki67 before being ejected from the oocyte. This work presents the discovery of an exciting new pathway operating during female meiosis and a provides a platform for future work exploring the meiotic chromosome periphery for therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4d4e852fd562f1b6718584414157f534cabd865" target='_blank'>
              The Chromosome Periphery is an Essential Compartment of Oocyte Chromosomes
              </a>
            </td>
          <td>
            Eva L Simpson, B. Wetherall, A. Byrska, Liam P. Cheeseman, Tania Mendonca, Xiaomeng Xing, Alison J. Beckett, H. Maiato, Alexandra Sarginson, Ian A. Prior, Geraldine M Hartshorne, A. McAinsh, S. Madgwick, Daniel G. Booth
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/053c90f2e024c025545d0a836542f7f28e9a61c7" target='_blank'>
              KLF5 loss sensitizes cells to ATR inhibition and is synthetic lethal with ARID1A deficiency
              </a>
            </td>
          <td>
            Samah W. Awwad, Colm Doyle, Josie Coulthard, Aldo S. Bader, Nadia Gueorguieva, Simon Lam, Vipul Gupta, R. Belotserkovskaya, Tuan-Anh Tran, Shankar Balasubramanian, S. P. Jackson
          </td>
          <td>2025-01-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c108f9b6c451482458b3d22e6cbf9ba4232a44ff" target='_blank'>
              Repeated ionizing radiation exposure induces TRIP13 expression, conferring radioresistance in lung cancer cells
              </a>
            </td>
          <td>
            Wenqing Liu, Qijing Lei, A. V. van Pelt, G. Hamer
          </td>
          <td>2025-01-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87b3f437af02a37eaf52895dfc3343570b14c40b" target='_blank'>
              The role of Box A of HMGB1 in producing γH2AX associated DNA breaks in lung cancer
              </a>
            </td>
          <td>
            Sirapat Settayanon, P. Chanvorachote, Apiwat Mutirangura
          </td>
          <td>2025-01-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The link between the “stress phenotype”-a well-established hallmark of cancer-and its role in tumor progression and intratumor heterogeneity remains poorly defined. The integrated stress response (ISR) is a key adaptive pathway that enables tumor survival under oncogenic stress. While ISR has been implicated in promoting tumor growth, its precise role in driving tumor evolution and heterogeneity has not been elucidated. In this study, using a genetically engineered mouse models, we demonstrate that ISR activation—indicated by elevated levels of phosphorylated eIF2 (p-eIF2) and ATF4—is essential for the emergence of dedifferentiated, therapy-resistant cell states. ISR, through the coordinated actions of ATF4 and MYC, facilitates the development of tumor cell populations characterized by high plasticity, stemness, and an epithelial-mesenchymal transition (EMT)-prone phenotype. This process is driven by ISR-mediated expression of genes that maintain mitochondrial integrity and function, critical for sustaining tumor progression. Importantly, genetic, or pharmacological inhibition of the p-eIF2–ATF4 signaling axis leads to mitochondrial dysfunction and significantly impairs tumor growth in mouse models of lung adenocarcinoma (LUAD). Moreover, ISR-driven dedifferentiation is associated with poor prognosis and therapy resistance in advanced human LUAD, underscoring ISR inhibition as a promising therapeutic strategy to disrupt tumor evolution and counteract disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984621d182e32177c7e7be128d21ab0e1df4a135" target='_blank'>
              Lineage plasticity of the integrated stress response is a hallmark of cancer evolution
              </a>
            </td>
          <td>
            Shiqi Diao, Jia Yi Zou, Shuo Wang, N. Ghaddar, , Hyungdong Kim, Nicolas Poulain, C. Koumenis, M. Hatzoglou, Peter Walter, Nahum Sonenberg, John Le Quesne, Tuomas Tammela, A. Koromilas
          </td>
          <td>2025-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Basal-like breast cancer (BLBC) is characterized by an aggressive clinical course, high genomic instability, and limited therapeutic options. The Inhibitor of Differentiation 4 (ID4) protein has been identified as a critical regulator of BLBC, where its overexpression correlates with poor prognosis. However, the mechanistic contributions of ID4 to BLBC tumorigenesis remain incompletely understood. In this study, we employed an integrative approach combining CRISPR-Cas9-mediated ID4 knockout, small-molecule inhibition, in vivo tumor modeling, and in silico transcriptional analyses to investigate the functional role of ID4 in BLBC. CRISPR-Cas9-mediated knockout of ID4 in MDA-MB-231 cells resulted in significant reductions in proliferation, colony formation, and Ki67 expression, indicating a loss of aggressive phenotypic traits. In vivo xenograft studies further revealed that ID4-silenced cells exhibited markedly delayed tumor formation and a significant reduction in metastatic potential compared to controls. Kaplan-Meier survival analysis of basal-like tumors from The Cancer Genome Atlas (TCGA) dataset demonstrated that patients with low ID4 expression had improved relapse-free survival. Gene set enrichment analysis (GSEA) of BLBC tumors stratified by ID4 expression revealed a shift toward luminal-like transcriptional programs in the ID4-low subgroup, including increased estrogen response and inflammatory signaling pathways. Furthermore, transcription factor activity analysis identified the activation of MYC, JUN, and STAT in ID4-low tumors, suggesting a transition toward a more differentiated phenotype. Finally, pharmacological inhibition of ID4 using the small-molecule degrader AGX51 effectively reduced proliferation in TNBC cells, highlighting ID4 as a potential therapeutic target. Together, these findings establish ID4 as a key driver of BLBC aggressiveness and suggest that its inhibition may represent a viable therapeutic strategy. This study provides compelling evidence supporting the development of ID4-targeted therapies for TNBC patients, with the potential to improve clinical outcomes in this challenging disease subset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/147bfd57152033ce1c79fd9d00f386b5bffcdc1b" target='_blank'>
              Exploring ID4 as a Driver of Aggression and a Therapeutic Target in Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            C. Toro, S. Real, S. Laurito, MT Branham
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c7a49a64fc95173d062b597a609f1c34ffa6ed" target='_blank'>
              PIF/harbinger transposon-derived protein promotes 7SL expression to enhance pathogen resistance.
              </a>
            </td>
          <td>
            S. Geng, Xing Lv, Tianjun Xu
          </td>
          <td>2025-01-30</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be789fb687d3f78efa3b5343344eaabbca303d35" target='_blank'>
              Chromatin licensing and DNA replication factor 1 as a potential prognostic and diagnostic biomarker for gastric cancer
              </a>
            </td>
          <td>
            Sijia Wang, Min Cui, Na Zhou, Meichen Tong, Shuhua Wu, Hong Li, Zhang Cao
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Breast cancers of the IntClust-2 type, characterized by amplification of a small portion of chromosome 11, have a median survival of only five years. Several cancer-relevant genes occupy this portion of chromosome 11, and it is thought that overexpression of a combination of driver genes in this region is responsible for the poor outcome of women in this group. In this study we used a gene editing method to knock out, one by one, each of 198 genes that are located within the amplified region of chromosome 11 and determined how much each of these genes contributed to the survival of breast cancer cells. In addition to well-known drivers such as CCND1 and PAK1, we identified two different genes (SERPINH1 and P4HA3), that encode proteins involved in collagen synthesis and organization. Using both in vitro and in vivo functional analyses, we determined that P4HA3 and/or SERPINH1 provide a critical driver function on IntClust-2 basic processes, such as viability, proliferation, and migration. Inhibiting these enzymes via genetic or pharmacologic means reduced collagen synthesis and impeded oncogenic signaling transduction in cell culture models, and a small-molecule inhibitor of P4HA3 was effective in treating 11q13 tumor growth in an animal model. As collagen has a well-known association with tissue stiffness and aggressive forms of breast cancer, we believe that the two genes we identified provide an opportunity for a new therapeutic strategy in IntClust-2 breast cancers. Implications: Breast cancers with 11q13/14 chromosomal amplifications may be vulnerable to inhibitors of collagen synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd01b0833b0dda4d17087283c1b41d82a5f1780" target='_blank'>
              Identifying and targeting key driver genes for collagen production within the 11q13/14 breast cancer amplicon.
              </a>
            </td>
          <td>
            D. Araiza-Olivera, T. Prudnikova, Cristina Uribe-Alvarez, Kathy Q Cai, J. Franco-Barraza, Jesus M Dones, Ronald T. Raines, J. Chernoff
          </td>
          <td>2025-01-21</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Renowned as a highly invasive and lethal tumor derived from neural stem cells in the central nervous system, glioblastoma (GBM) exhibits substantial histopathological variation and genomic complexity, which drive its rapid progression and therapeutic resistance. Alterations in mitochondrial DNA (mtDNA) copy number (CN) serve a crucial role in GBM development and progression, affecting various aspects of tumor biology, including energy production, oxidative stress regulation and cellular adaptability. Fluctuations in mtDNA levels, whether elevated or diminished, can impair mitochondrial function, potentially disrupting oxidative phosphorylation and amplifying reactive oxygen species generation, thereby fueling tumor growth and influencing treatment responses. Understanding the mechanisms of mtDNA-CN variations, and their interplay with genetic and environmental elements in the tumor microenvironment, is essential for advancing diagnostic and therapeutic strategies. Targeting mtDNA alterations could strengthen treatment efficacy, mitigate resistance and ultimately enhance the prognosis of patients with this aggressive brain tumor. The present review summarizes the existing literature on mtDNA alterations, specifically emphasizing variations in mtDNA-CN and their association with GBM by surveying articles published between 1996 and 2024, sourced from databases such as Scopus, PubMed and Google Scholar. In addition, the review provides a brief overview of mitochondrial genome architecture, knowledge regarding the regulation of mtDNA integrity and CN, and how mitochondria significantly impact GBM tumorigenesis. This review further presents information on therapeutic approaches for restoring mtDNA-CN that contribute to optimized mitochondrial function and improved health outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/608231a30fdc2050cf19ab35c688a4cbcbde93d3" target='_blank'>
              Mitochondrial DNA copy number alterations: Key players in the complexity of glioblastoma (Review)
              </a>
            </td>
          <td>
            Abdul Aziz Mohamed Yusoff, Siti Zulaikha Nashwa Mohd Khair, Siti Muslihah Abd Radzak
          </td>
          <td>2025-01-24</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Since the discovery of the Musashi (MSI) protein, its ability to affect the mitosis of Drosophila progenitor cells has garnered significant interest among scientists. In the following 20 years, it has lived up to expectations. A substantial body of evidence has demonstrated that it is closely related to the development, metastasis, migration, and drug resistance of malignant tumors. In recent years, research on the MSI protein has advanced, and many novel viewpoints and drug resistance attempts have been derived; for example, tumor protein p53 mutations and MSI-binding proteins lead to resistance to protein arginine N-methyltransferase 5-targeted therapy in lymphoma patients. Moreover, the high expression of MSI2 in pancreatic cancer might suppress its development and progression. As a significant member of the MSI family, MSI2 is closely associated with multiple malignant tumors, including hematological disorders, common abdominal tumors, and other tumor types (e.g., glioblastoma, breast cancer). MSI2 is highly expressed in the majority of tumors and is related to a poor disease prognosis. However, its specific expression levels and regulatory mechanisms may differ based on the tumor type. This review summarizes the research progress related to MSI2 in recent years, including its occurrence, migration mechanism, and drug resistance, as well as the prospect of developing tumor immunosuppressants and biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dd0dc9ec8f8cc8ab893bc32f47ba299917d963" target='_blank'>
              Update on the Progress of Musashi-2 in Malignant Tumors.
              </a>
            </td>
          <td>
            Yiting Niu, Tao Zhou, Yanjun Li
          </td>
          <td>2025-01-17</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Traditional drug discovery efforts primarily target rapid, reversible protein-mediated adaptations to counteract cancer cell resistance. However, cancer cells also utilize DNA-based strategies, often perceived as slow, irreversible changes like point mutations or drug-resistant clone selection. Extrachromosomal DNA (ecDNA), in contrast, represents a rapid, reversible, and predictable DNA alteration critical for cancer’s adaptive response.In this study, we developed a novel post-processing pipeline for automated detection and quantification of ecDNA in metaphase Fluorescence in situ Hybridization (FISH) images, leveraging the Microscopy Image Analyzer (MIA) tool. This pipeline is tailored to monitor ecDNA dynamics during drug treatment.Our approach effectively quantified ecDNA changes, providing a robust framework for analyzing the adaptive responses of cancer cells under therapeutic pressure.The pipeline not only serves as a valuable resource for automating ecDNA detection in metaphase FISH images but also highlights the role of ecDNA in facilitating swift and reversible adaptation to epigenetic remodeling agents such as JQ1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed078aa348654cdad2d2c2164c322729a574c316" target='_blank'>
              Leveraging AI to automate detection and quantification of extrachromosomal DNA to decode drug responses
              </a>
            </td>
          <td>
            Kohen Goble, Aarav Mehta, Damien Guilbaud, Jacob Fessler, Jingting Chen, William Nenad, Christina Ford, Oliver Cope, Darby Cheng, William Dennis, Nithya Gurumurthy, Yue Wang, Kriti Shukla, Elizabeth Brunk
          </td>
          <td>2025-02-03</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction The altered expression of genes encoding histone acetyltransferases (HATs) and histone deacetylases (HDACs) has been implicated in the tumorigenesis and progression of various solid tumors. However, systematic characterization of the transcriptomic landscape and clinical relevance of HATs and HDACs in pan-cancer contexts remains lacking. Methods Transcriptome and clinical data of 9,483 patients across 31 tumor types from The Cancer Genome Atlas were collected for systematic pan-cancer analysis. Additional glioma-specific datasets (Chinese Glioma Genome Atlas, GlioVis, GSE43378, and GSE182109) were also collected to validate the transcriptional characteristics of HATs and HDACs in gliomas. Consensus clustering analysis was applied to identify distinct expression patterns of HATs and HDACs. Results Based on the transcriptomic data of 25 genes encoding 9 HATs and 16 HDACs, we identified five major subtypes across 31 cancer types (AC-I to AC-V). Notably, the AC-V subtype comprised over 95% of glioma patients, suggesting glioma patients exhibited distinct expression patterns of histone acetylation-modifying enzymes compared to patients with other solid tumors. Therefore, we re-conducted the consensus clustering analysis specifically within the context of gliomas and identified five subtypes, denoted “AC-GI” to “AC-GV”, which were characterized by differences in HATs/HDACs expression patterns, biological and immune status, genetic alterations, and clinical outcomes. The AC-GII patients exhibited the best prognosis and were sensitive to temozolomide, while AC-GV patients had the poorest prognosis and the lowest sensitivity to temozolomide among all subtypes. Moreover, based on the Connectivity Map database analysis and experimental verification, we identified several pan-HDAC inhibitors that could serve as sensitizers for temozolomide therapy in AC-GV patients, such as panobinostat and scriptaid. Considering the distinctive clinical characteristics of patients with AC-GII and AC-GV, we constructed the “ACG score” model capable of effectively recognizing patients with these subtypes and predicting patient prognosis. Conclusion Herein, we established novel biologically and clinically relevant molecular classifications for pan-solid tumors and gliomas based on transcriptional expression profiles of HATs and HDACs. Moreover, the ACG score model, calculated by the transcriptional expression of 29 genes, was not only an independent prognostic factor for glioma patients, but can also provide valuable references for promoting more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9506518bda05def7ccbdb74a1bb450d215aa14aa" target='_blank'>
              Pan-cancer analysis of the transcriptional expression of histone acetylation enzymes in solid tumors defines a new classification scheme for gliomas
              </a>
            </td>
          <td>
            Junhao Zhang, Lingbo Li, Aiwei Tang, Chucheng Wang, Yupeng Wang, Yongqi Hu, Guangting He, Wangjun Liao, Rui Zhou
          </td>
          <td>2025-01-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Primary cilia have been considered as tumor-suppressing organelles in cholangiocarcinoma (CCA), though the mechanisms behind their protective role are not fully understood. This study investigates how the loss of primary cilia affects DNA damage response (DDR) and DNA repair processes in CCA. Human cholangiocyte cell lines were used to examine the colocalization of DNA repair proteins at the cilia and assess the impact of experimental deciliation on DNA repair pathways. Deciliation was induced using shRNA knockdown or CRISPR knockout of IFT20, IFT88, and KIF3A, followed by exposure to genotoxic agents such as cisplatin, methyl methanesulfonate (MMS), and irradiation. Cell survival, cell cycle progression, and apoptosis rates were evaluated, while DNA damage was assessed using comet assays and γH2AX quantification. An in vivo liver-specific knockout model of IFT88 was generated using the Cre/Lox recombination system. Results showed that RAD51 localized at the cilia base, while ATR, PARP1, CHK1 and CHK2 were found within the cilia. Deciliated cells displayed dysregulation in critical DDR pathways. These cells also showed reduced survival and increased S-phase arrest after genotoxic challenges. Enhanced DNA damage was observed via increased γH2AX signals and comet assay results. An increase in γH2AX expression was observed in our in vivo model, indicating elevated DNA damage. Additionally, key DDR proteins, such as ATM, p53, and p21, were downregulated in deciliated cells after irradiation. This study underscores the crucial role of primary cilia in regulating DNA repair, suggesting that targeting cilia-related mechanisms could present a novel therapeutic approach for CCA. Key Points The results presented delineate the following critical observations: DNA Damage Response (DDR) and DNA repair proteins localized to primary cilia structures Deciliated cells exhibited dysregulated DDR pathways, reduced survival, and S-phase arrest under genotoxic stress Elevated DNA damage was evident in deciliated cells, with increased γH2AX signals and comet assay results In vivo IFT88 knockout mice also showed increased γH2AX expression, indicating heightened DNA damage Key DDR proteins (ATM, p53, p21) were downregulated in deciliated cells after irradiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec01dcc215e6343c000fcd771392ef5c400186d" target='_blank'>
              Cholangiocytes’ Primary Cilia Regulate DNA Damage Response and Repair
              </a>
            </td>
          <td>
            E. Peixoto, Kishor Pant, Seth Richard, J. Abrahante, Wioletta Czaja, S. Gradilone
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e1743fa19de0544fdab9fff89543caabd4c7a18" target='_blank'>
              D-2-hydroxyglutarate impairs DNA repair through epigenetic reprogramming
              </a>
            </td>
          <td>
            Fengchao Lang, Karambir Kaur, Haiqing Fu, Javeria Zaheer, Diego Luis Ribeiro, M. Aladjem, Chunzhang Yang
          </td>
          <td>2025-02-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e667edb95ec839a056e02fa834f44d25b6553f6" target='_blank'>
              Cohesin positions the epigenetic reader Phf2 within the genome
              </a>
            </td>
          <td>
            W. Tang, Lorenzo Costantino, Roman R. Stocsits, G. Wutz, Rene Ladurner, Otto Hudecz, K. Mechtler, Jan-Michael Peters
          </td>
          <td>2025-01-02</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c600bc41b490f1030b42efdd32168810e5ead4" target='_blank'>
              The dynamics of loss of heterozygosity events in genomes
              </a>
            </td>
          <td>
            Abhishek Dutta, J. Schacherer
          </td>
          <td>2025-01-02</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Among the many features of cancer cells, they commonly display aberrant accumulation of RNA-DNA hybrids known as R-loops. R-loops are formed in healthy cells during many cellular processes, including transcription and DNA replication, and are crucial for proper maintenance of telomeres and centromeres, as well as for adequate mitochondrial function. However, aberrant accumulation of R-loops can lead to increases in DNA damage and inflammation. R-loop accumulation has been documented in different types of cancer, including BRCA1-mutant breast cancer, diffuse large B cell lymphoma, and myelodysplastic syndrome (MDS). Indeed, accumulation of R-loops has been proposed as a unifying mechanism underlying MDS and other myeloid malignancies. Ten-eleven translocation enzymes (TET1, TET2, TET3) are 5-methylcytosine dioxygenases that undergo frequent mutational or functional inactivation in human cancers. We have shown that mice with profound, acute TET deficiency induced by deletion of all Tet genes (hereafter referred to as Tet iTKO mice) show rapid onset of an uncontrolled malignant myeloid expansion. Expanded myeloid cells from Tet iTKO mice show increased levels of R-loops, together with increased DNA damage and inflammation, compared to control myeloid cells. Using Tet iTKO hematopoietic stem and progenitor cells (HSPC), we performed an in vivo CRISPR knockout screen of genes encoding RNA and DNA sensors, as well as R-loop regulators, to find how loss of genes related to nucleic acid sensing and innate immune response would affect the proliferation of TET-deficient myeloid cells. We observed that loss of enzymes that prevent the accumulation of R-loops impaired the uncontrolled malignant myeloid expansion of Tet iTKO cells. CRISPR-mediated depletion of the gene encoding RNase H1, an enzyme that specifically degrades the RNA strand in R-loops, further increased R-loop levels and DNA damage, and decreased proliferation and in vivo expansion of TET-deficient myeloid cancer cells, compared to Tet iTKO cells treated with non-targeting sgRNAs. Moreover, the accumulation of DNA breaks in Tet iTKO cells upon Rnaseh1 loss was not observed in Tet wild-type cells similarly depleted for RNase H1, suggesting a role for TET enzymes and potentially TET-oxidized 5-methylcytosines in DNA repair, which will be discussed in this presentation. Overall, these findings indicate that further increasing R-loops in TET-deficient cancer cells blocks their proliferation, and suggest that RNase H depletion or inhibition could be a new therapeutic strategy for cancer treatment in cancers that display augmented levels of R-loops, independent of the presence of TET mutations.
 Citation Format: Isaac F López-Moyado, Otokani Ortega de la Tejera, Lot M Hernández-Espinosa, Arthur X Cheng, Roberta Nowak, Zhen Dong, Caitlin Brown, André Nussenzweig, Anjana Rao. Forced accumulation of R-loops through inhibiton of RNase H impairs the proliferation of TET-deficient cancer cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr PR005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c95c688477a3d43d5e5b015493b97f465e9711d" target='_blank'>
              Abstract PR005: Forced accumulation of R-loops through inhibiton of RNase H impairs the proliferation of TET-deficient cancer cells
              </a>
            </td>
          <td>
            Isaac F. López-Moyado, Otokani Ortega de la Tejera, Lot M Hernández-Espinosa, A. X. Cheng, Roberta Nowak, Zhen Dong, Caitlin Brown, André Nussenzweig, Anjana Rao
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 For the last thirty years an increasing amount of evidence has shown the relevance of epigenetics in cell biology and tissue physiology, being DNA methylation aberrations in cancer the flag-ship for the recognition of its disturbance in human diseases. From the candidate gene approaches, new powerful technologies such as comprehensive DNA methylation microarrays and whole genome bisulfite sequencing has recently emerged that have reinforced the notion of epigenetic disruption in the crossroad of many sickness. From the poster-boy case of MGMT hypermethylation in the prediction of alkylating drug response to the personalized treatment of leukemia with small molecules targeted to fusion proteins involving histone modifiers, the field has walked a long path. The current talk will focus in the epigenetic profiling, basically at the level of DNA methylation and histone modifications, that is starting to provide clinical value in the diagnosis, prognosis and prediction of response to drug therapies. For cancer, we have already a wide view of the undergoing DNA methylation events that expand beyond classical promoter CpG islands of tumor suppressor genes and we have a growing list of mutated chromatin remodeler genes that contributes to the tumorigenesis process. It is time to apply this knowledge in practical clinical situations like the diagnosis of cancers of unknown primary, the screening of malignancies in high-risk populations or a biomarker selection of the patients that should receive treatment with anticancer drugs, including immunotherapy. Beyond cancer, DNA methylation is starting to be recognized as playing a major role in infectious diseases, and in this regard, the present lecture will also address the epigenomic component of COVID-19. Most important, Epigenomics can be added to Genomics, Transcriptomics and Proteomics plus Bioinformatics to create a Multomics view of human diseases.
 Citation Format: Manel Esteller. DNA methylation in cancer: From knowledge to applications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr IA007">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/855f061aa62959677f73a3c7e17c250ce96d93c4" target='_blank'>
              Abstract IA007: DNA methylation in cancer: From knowledge to applications
              </a>
            </td>
          <td>
            M. Esteller
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The Chromosome-Centric Human Proteome Project (C-HPP) is an international initiative. It aims to create a protein list expressed in human cells by each chromosomal and mitochondrial DNA to enhance our understanding of disease mechanisms, akin to the gene list generated by the Human Genome Project. Transmembrane protein 160 (TMEM160) is a member of the transmembrane proteins (TMEM) family. TMEM proteins have been implicated in cancer-related processes, including cell proliferation, migration, epithelial-mesenchymal transition, metastasis, and resistance to chemotherapy and radiotherapy. This study aimed to investigate the role of TMEM160 in non-small cell lung cancer and cervical cancer using cell lines, clinical samples, and xenograft studies. Our findings demonstrated that TMEM160 knockdown decreased the proliferation of lung and cervical cancer cell lines. We observed that TMEM160 is localized in the nucleus and cytoplasm and dynamic localization during mitosis of cancer cells and discovered a novel interaction between TMEM160 and nuclear proteins such as NUP50. Furthermore, the TMEM160 interactome was enriched in processes associated with apical junctions, xenobiotic metabolism, glycolysis, epithelial-mesenchymal transition, reactive oxygen species, UV response DNA, the P53 pathway, and the mitotic spindle. This study provides an initial understanding of the function of TMEM160 in lung and cervical cancer progression and clarifies the need to continue investigating the participation of TMEM160 in these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e13c463e1e4f1ddbb09b31ff1b7171d0bdfe61c" target='_blank'>
              TMEM160 Promotes Tumor Growth in Lung Adenocarcinoma and Cervical Adenocarcinoma Cell Lines
              </a>
            </td>
          <td>
            Gloria Angelina Herrera-Quiterio, H. A. Valencia-González, K. G. de la Cruz-López, D. L. Fernández-Coto, Jeovanis Gil, Gyorgy Marko-Varga, Josué Morales-Gálvez, Nilda C. Sánchez, Rubén Rodríguez-Bautista, A. Avilés-Salas, Oscar Arrieta, Alejandro García-Carrancá, S. Encarnación-Guevara
          </td>
          <td>2025-01-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241144354fceff41346aae0d566101a01ccaa823" target='_blank'>
              Advances in cancer genomics and precision oncology.
              </a>
            </td>
          <td>
            Yonjong Heo, Woo-Jin Kim, Yong-Joon Cho, Jae-Won Jung, Nam-Soo Kim, Ik-Young Choi
          </td>
          <td>2025-01-23</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mutations in the NPM1 gene represent the most common (>30% of patients) genetic alteration in Acute Myeloid Leukaemia (AML) and results in the mis-localisation of the mutated NPM1 protein from a predominantly nucleolar localisation to a predominantly cytoplasmic distribution. Numerous studies of NPM1 mutated AML have focussed on the aberrant cytoplasmic localisation of the mutated protein but efforts to reverse this mis-localisation therapeutically have so far resulted in limited clinical benefit. More recently, attention has shifted towards the nucleus with studies showing that mutant NPM1 binds to specific chromatin regions, where it directly regulates oncogenic gene expression. Here, we use high resolution imaging to demonstrate that Nucleophosmin (NPM1) is critical for maintaining normal nucleoli architecture and specifically the integrity of the nucleoli rim. We report for the first time that NPM1 mutated cell lines and primary samples have aberrant nucleoli architecture and demonstrate that the abnormal nucleoli phenotype is reversible. We also report the novel finding that NPM1 mutated protein forms distinct aggregates in NPM1 mutated cells and characterise these for the first time. This work reveals how nucleolar organisation contributes to the molecular mechanisms underpinning NPM1 driven AML and reveals unexpected novel vulnerabilities to be exploited for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b14a9c2136580651e7ba63ac9928b96ae556480" target='_blank'>
              Nucleophosmin mutations lead to abnormal, but reversible, nucleoli architecture and aggregate formation – Implications for NPM1-targeting therapies in AML
              </a>
            </td>
          <td>
            Martin Grundy, Kellie Lucken, Xiaomeng Xing, Eva L Simpson, Ahmed Bayyoomi, Alison J. Beckett, Ian A. Prior, Daniel G. Booth, C. Seedhouse
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Simple Summary This review presents an overview of recent findings on various epigenetic and post-transcriptional modifications of the epidermal growth factor receptor (EGFR), which activates the oncogenes RAS and MYC. Alterations in EGFR, RAS, and MYC play a crucial role in dysregulated processes that contribute to oncogenesis. Focused, targeted research aimed at understanding the distinct effects of these oncogenes and their carcinogenic modifications has facilitated the successful treatment of various malignancies using inhibitors of EGFR tyrosine kinase activity and other therapeutic agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e6048dcc607aa9e108befb3b09588362de65007" target='_blank'>
              Dynamic Multilevel Regulation of EGFR, KRAS, and MYC Oncogenes: Driving Cancer Cell Proliferation Through (Epi)Genetic and Post-Transcriptional/Translational Pathways
              </a>
            </td>
          <td>
            M. Šereš, Katarina Spacayova, Z. Sulová, Jana Špaldová, Albert Breier, L. Pavlikova
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="LINE-1 retrotransposons, comprising 17% of the genome, drive cancer instability through hypomethylation. The DIAMOND assay, targeting LINE-1 hypomethylation with bisulfite sequencing of cell free DNA, achieved AUCs of 88% to 100% across six cancer types, surpassing mutation-based diagnostics and suggesting utility in early cancer detection and management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57cf07128fb65aaad9c55c65c3a93161bd10c47" target='_blank'>
              LINE-1 Retrotransposons as Cell-free DNA Biomarkers for Multi-Cancer Early Detection.
              </a>
            </td>
          <td>
            J. Szymanski, Pradeep S Chauhan, A. Chaudhuri
          </td>
          <td>2025-01-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb8360ddf104ac6e4b2843cdfd9b7edf6a857c0d" target='_blank'>
              Dysregulation of transposable elements and PIWI-interacting RNAs in myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Z. Krejcík, D. Kundrát, Jirí Kléma, A. Hruštincová, I. Trsova, M. Belickova, Jaroslav Čermák, A. Jonášová, Jiri Dostal, M. Dostalova Merkerova
          </td>
          <td>2025-01-23</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="HOXB13 X285K is a frameshift mutant due to the deletion of one nucleotide, c.853delT, within the HOXB13 stop codon, resulting in an extension of HOXB13 protein by 96 amino acids on the C terminus. It was found primarily in men of West African ancestry and associated with early-onset prostate cancer (PCa) and more advanced stage. Whether and how X285K contributes to PCa progression remains largely unknown. Here, we established isogenic 22Rv1 cell lines with heterozygous wildtype/X285K HOXB13, which recapitulates the genotypes of X285K in PCa patients. In addition, using the unique C-terminal region as an antigen, we developed an antibody that specifically recognizes the HOXB13 X285K protein. Lastly, we demonstrated that X285K retained the function of HOXB13 in suppressing target genes, such as FASN and PSA. In summary, our data suggest that the isogenic cell lines and the antibody generated in this study will be useful for a comprehensive investigation of X285K function in PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eed5e8e9d2022eb0eb9f170bbcfe9edb5266f2f" target='_blank'>
              Delineating the roles of HOXB13 X285K variant in prostate cancer
              </a>
            </td>
          <td>
            Xiaodong Lu, Samantha Ye, Mingyang Liu, Jindan Yu
          </td>
          <td>2025-01-14</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The autonomous and active Long-Interspersed Element-1 (LINE-1, L1) and the non-autonomous Alu retrotransposon elements, contributing to 30% of the human genome, are the most abundant repeated sequences. With more than 90% of their sequences being methylated in normal cells, these elements undeniably contribute to the global DNA methylation level and constitute a major part of circulating-cell-free DNA (cfDNA). So far, the hypomethylation status of LINE-1 and Alu in cellular and extracellular DNA has long been considered a prevailing hallmark of ageing-related diseases and cancer. This study demonstrated that errors in LINE-1 and Alu methylation level measurements were caused by an excessive input quantity of genomic DNA used for bisulfite conversion. Using the minuscule DNA amount of 0.5 ng, much less than what has been used and recommended so far (500 ng-2 μg) or 1 μL of cfDNA extracted from 1 mL of blood, we revealed hypermethylation of LINE-1 and Alu in 407 tumour samples of primary breast, colon and lung cancers when compared with the corresponding pair-matched adjacent normal tissue samples (P < 0.05–0.001), and in cfDNA from 296 samples of lung cancers as compared with 477 samples from healthy controls (P < 0.0001). More importantly, LINE-1 hypermethylation in cfDNA is associated with healthy ageing. Our results have not only contributed to the standardized bisulfite-based protocols for DNA methylation assays, particularly in applications on repeated sequences but also provided another perspective for other repetitive sequences whose epigenetic properties may have crucial impacts on genome architecture and human health.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6af9de458a2650aad970d8197a4d94f0fb4283e9" target='_blank'>
              An appropriate DNA input for bisulfite conversion reveals LINE-1 and Alu hypermethylation in tissues and circulating cell-free DNA from cancers
              </a>
            </td>
          <td>
            Trang Thi Quynh Tran, Tung The Pham, Than Thi Nguyen, Trang Hien Do, Phương Thi Thu Luu, Uyen Quynh Nguyen, L. Vuong, Quang Ngoc Nguyen, Son Van Ho, Hang Viet Dao, Tong Van Hoang, Lan Thi Thuong Vo
          </td>
          <td>2024-12-30</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebd3b0816d9ce9321d656adc7fa4e3bc0da80c79" target='_blank'>
              Patterns and variations of copy number alterations in acute myeloid leukemia: insights from the LeukAtlas database.
              </a>
            </td>
          <td>
            Yanxun Su, Zhenxian Han, Yutong Ji, Anqi Liu, D. Zou, Lina Yan, Dan Liu, Zhang Zhang, Qian-Fei Wang
          </td>
          <td>2025-02-02</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ef97a03979c9e266e81720f51438d4d5cd7b23" target='_blank'>
              BRCA1 is involved in sustaining rapid antler growth possibly via balancing of the p53/endoplasmic reticulum stress signaling pathway
              </a>
            </td>
          <td>
            Qianqian Guo, Zhen Wang, Jiping Li, Chao Ma, Junjun Zheng, H. Ba, Guokun Zhang, Chunyi Li
          </td>
          <td>2025-01-24</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Historically considered downstream effects of tumorigenesis—arising from changes in DNA content or chromatin organization—nuclear alterations have long been seen as mere prognostic markers within a genome‐centric model of cancer. However, recent findings have placed the nuclear envelope (NE) at the forefront of tumor progression, highlighting its active role in mediating cellular responses to mechanical forces. Despite significant progress, the precise interplay between NE components and cancer progression remains under debate. In this review, we provide a comprehensive and up‐to‐date overview of how changes in NE composition affect nuclear mechanics and facilitate malignant transformation, grounded in the latest molecular and functional studies. We also review recent research that uses advanced technologies, including artificial intelligence, to predict malignancy risk and treatment outcomes by analyzing nuclear morphology. Finally, we discuss how progress in understanding nuclear mechanics has paved the way for mechanotherapy—a promising cancer treatment approach that exploits the mechanical differences between cancerous and healthy cells. Shifting the perspective on NE alterations from mere diagnostic markers to potential therapeutic targets, this review calls for further investigation into the evolving role of the NE in cancer, highlighting the potential for innovative strategies to transform conventional cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ec969f050943713ac6f56187d24a844cc59674" target='_blank'>
              At the nucleus of cancer: how the nuclear envelope controls tumor progression
              </a>
            </td>
          <td>
            F. Paganelli, Alessandro Poli, Serena Truocchio, Alberto M. Martelli, Carla Palumbo, Giovanna Lattanzi, Francesca Chiarini
          </td>
          <td>2025-01-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Mutations in the exonuclease domains of the replicative nuclear DNA polymerases POLD1 and POLE are associated with increased cancer incidence, elevated tumor mutation burden (TMB), and enhanced response to immune checkpoint blockade (ICB). Although ICB is approved for treatment of several cancers, not all tumors with elevated TMB respond, highlighting the need for a better understanding of how TMB affects tumor biology and subsequently immunotherapy response. To address this, we generated mice with germline and conditional mutations in the exonuclease domains of Pold1 and Pole. Engineered mice with Pold1 and Pole mutator alleles presented with spontaneous cancers, primarily lymphomas, lung cancer, and intestinal tumors, while Pold1 mutant mice also developed tail skin carcinomas. These cancers had highly variable tissue-type dependent increased TMB with mutational signatures associated with POLD1 and POLE mutations found in human cancers. The Pold1 mutant tail tumors displayed increased TMB, however, only a subset of established tumors responded to ICB. Similarly, introducing the mutator alleles into mice with lung cancer driven by mutant Kras and Trp53 deletion did not improve survival, whereas passaging these tumor cells in vitro without immune editing and subsequently implanting them into immune-competent mice caused tumor rejection in vivo. These results demonstrated the efficiency by which cells with antigenic mutations are eliminated in vivo. Finally, ICB treatment of mutator mice earlier, before observable tumors had developed delayed cancer onset, improved survival, and selected for tumors without aneuploidy, suggesting the potential of ICB in high-risk individuals for cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/274ef0221dce61440b76214e33f127197aac5688" target='_blank'>
              Immune Checkpoint Blockade Delays Cancer Development and Extends Survival in DNA Polymerase Mutator Syndromes.
              </a>
            </td>
          <td>
            Akshada Sawant, Fuqian Shi, E. Cararo Lopes, Zhixian Hu, Somer Abdelfattah, Jennele Baul, Jesse R. Powers, Christian S. Hinrichs, Joshua D. Rabinowitz, Chang S. Chan, Edmund C. Lattime, Shridar Ganesan, Eileen White
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ae5f6bb3e804b17358c04ea87f466f1ee8583c6" target='_blank'>
              Shedding light on DNA methylation and its clinical implications: the impact of long-read-based nanopore technology
              </a>
            </td>
          <td>
            Alexandra Chera, Mircea Stancu-Cretu, N. Zabet, Octavian Bucur
          </td>
          <td>2024-12-30</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e1421e272a6e4ff8131686f1fc164e53532b276" target='_blank'>
              Altered chromatin landscape and 3D interactions associated with primary constitutional MLH1 epimutations
              </a>
            </td>
          <td>
            Paula Climent-Cantó, Marc Subirana-Granés, M. Ramos-Rodríguez, Estela Dámaso, F. Marín, Covadonga Vara, Beatriz Pérez-González, Helena Raurell, Elisabet Munté, José Luis Soto, Ángel Alonso, GiWon Shin, Hanlee Ji, Megan Hitchins, Gabriel Capellá, Lorenzo Pasquali, M. Pineda
          </td>
          <td>2024-12-31</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca8eb632e8dcc4d371aa3266d6271acffef83ed2" target='_blank'>
              The integrated stress response drives MET oncogene overexpression in cancers.
              </a>
            </td>
          <td>
            M. Cerqua, Marco Foiani, C. Boccaccio, Paolo M Comoglio, D. Altintas
          </td>
          <td>2025-01-07</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Epigenetic modifications, which reversibly regulate gene expression without altering the DNA sequence, are increasingly described in the literature as essential elements in the processes leading to cancer development. SETDB1 regulates histone 3 (H3) K9 di- and trimethylation, promoting heterochromatin formation, and plays a key role in gene silencing. Epigenetic deregulation of SETDB1 expression appears to be involved in different cancers types, particularly in aggressive, relapsing or treatment-resistant subtypes. Despite advances in research, the full range of mechanisms through which this protein acts remains unclear; however, it is evident that SETDB1 has a pivotal role, particularly in the mesenchymal stem cells differentiation, tumor evasion and treatment resistance. Its role in genetically complex sarcomas, such as osteosarcoma, has not been fully explored, although recent Omics analyses suggest its presence and amplification in osteosarcoma. Given its involvement in osteoblastogenesis and adipogenesis, we discuss the potential of SETDB1 as a key target for new therapeutic strategies in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b378d2be6324f27ab95439dedf132da8d385c12" target='_blank'>
              SETDB1 amplification in osteosarcomas: Insights from its role in healthy tissues and other cancer types
              </a>
            </td>
          <td>
            Elodie Verdier, Nathalie Gaspar, Maria Eugénia Marques da Costa, A. Marchais
          </td>
          <td>2025-02-12</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa423d61a8720aa08805a6d061e12902556221c5" target='_blank'>
              Comprehensive landscape and oncogenic role of extrachromosomal circular DNA in malignant biliary strictures
              </a>
            </td>
          <td>
            Zhuo Cheng, Xuanmei Luo, Wenzheng Liu, Xiaofang Lu, Hong Chang, Yingchun Wang, Wei Zheng, Xiue Yan, Yonghui Huang
          </td>
          <td>2025-02-07</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Background Hepatocellular carcinoma (HCC) is one of the most common and highly lethal cancers worldwide. RIO kinase 1 (RIOK1), a protein kinase/ATPase that plays a key role in regulating translation and ribosome assembly, is associated with a variety of malignant tumors. However, the role of RIOK1 in HCC remains largely unknown. Methods Changes in RIOK1 expression in HCC and patient prognosis were evaluated using HCC tissues and public databases. The functional role of RIOK1 in HCC was analyzed by RTCA assay, clonogenic assay, and flow cytometry in vitro, and by mouse tumor xenograft model in vivo. Potential mechanism studies were performed using multi‐omics analysis, public database screening, and qRT‐PCR assay. Results In this study, we found that RIOK1 was elevated in HCC tissues and correlated with poor prognosis. Functional assays demonstrated that RIOK1 knockdown suppressed HCC cell proliferation, survival, and tumor growth in vivo, while RIOK1 overexpression enhanced these oncogenic phenotypes. Meanwhile, RIOK1 knockdown affected cell cycle progression and the expression of cyclin A2 and cyclin B1. Furthermore, integrated transcriptomic and proteomic analysis revealed that RIOK1 may promote HCC cell proliferation by affecting the cell cycle and DNA repair pathways. Moreover, we identified five potential effectors regulated by RIOK1: PMS1, SPDL1, RAD18, BARD1, and SMARCA5, which were highly expressed in HCC tissues and negatively correlated with the overall survival of HCC patients. Conclusion Our findings suggest that RIOK1 is a novel oncogenic driver that may serve as a potential diagnostic and therapeutic target for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cb5ec9c22d3623d1b6437e3e54cca76d51046d9" target='_blank'>
              RIOK1: A Novel Oncogenic Driver in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Chunyan Ruan, Tianyu Shang, Sijia Zhang, Wenhong Ru, Yuefeng Yang, Yi Shen
          </td>
          <td>2025-01-25</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The human genome is like a museum of ancient retroviral infections. It contains a large number of endogenous retroviruses (ERVs) that bear witness to past integration events. About 5,000 of them are so-called long terminal repeat 12 (LTR12) elements. Compared with 20,000 human genes, this is a remarkable number. Although LTR12 elements can act as promoters or enhancers of cellular genes, the function of most of these retroviral elements has remained unclear. In our mini-review, we show that different LTR12 elements share many similarities, including common transcription factor binding sites. Furthermore, we summarize novel insights into the epigenetic mechanisms governing their silencing and activation. Specific examples of genes and pathways that are regulated by LTR12 loci are used to illustrate the regulatory network built by these repetitive elements. A particular focus is on their role in the regulation of antiviral immune responses, tumor cell proliferation, and senescence. Finally, we describe how a targeted activation of this fascinating ERV family could be used for diagnostic or therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4041a47b44612c0a5daa01efba716fa89ab732e8" target='_blank'>
              Viral influencers: deciphering the role of endogenous retroviral LTR12 repeats in cellular gene expression.
              </a>
            </td>
          <td>
            Veronika Krchlikova, Yueshuang Lu, D. Sauter
          </td>
          <td>2025-01-31</td>
          <td>Journal of virology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e33b574995e584f7a063a425a60100a1abcb82e" target='_blank'>
              Identification of a novel TOP2B::AFF2 fusion gene in B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Tao Liu, Yang Wang, Xi-Zhou An, Jiaqi Liu, Yuqin Wu, Yan Xiang, Yong-Jie Zhang, Lan Huang, Jia-Cheng Li, Yu-Zhuo-Pu Li, Jie Yu
          </td>
          <td>2025-01-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor cell networks formed by tumor microtubes (TMs) may play a key role in the development of therapy resistance in glioblastoma (GB). TM-mediated detoxification from radiation-induced reactive oxygen species (ROS) may infer radioresistance. We hypothesize that high linear energy transfer (LET) radiation, which describes the amount of energy deposited by radiation per unit length, interacts directly with the DNA backbone to induce complex lesions and thus might be less dependent on TM-mediated resistance mechanisms. Therefore, we sought to systematically investigate the impact of LET-induced complex DNA damage on TM formation and GB survival. To this end, the formation of TMs, radiation-induced nuclear DNA damage repair foci (RIF), and GB survival were correlated with a gradual increase in LET using a dose series of clinical protons (low), helium (intermediate), and carbon (high) ion beams. Consistent with conventional photon/X-rays, low-LET proton irradiation promoted TM formation in a dose- dependent manner. In contrast, an anti-correlation between LET and TM induction was found, i.e., a decreased network connectivity with gradual increase of LET and formation of complex DNA damage. Consequently, LET increase correlated with reduced cell survival, with the most pronounced cell killing observed after high-LET carbon irradiation. Moreover, the inverse correlation between LET and TM density was further confirmed for a broad range of LET modulated within the carbon ion irradiation spectrum. This is the first report on the relevance of LET as a novel mean to overcome TM network-mediated radioresistance in GB, with ramifications for the clinical translation of high-LET particle radiotherapy to further improve outcome in this still devastating disease. Highlights Tumor microtubes (TM) density correlates with glioblastoma cell viability after X-rays irradiation. Induction of DNA damage and its complexity depends on linear energy transfer (LET) and it correlates with decreased TM density. High-LET radiotherapy with carbon ions significantly reduces TM density and GB survival dependency from TM formation. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c77d1ef913ee8a8c60eddee5504f119ad2317c6" target='_blank'>
              High LET Radiation: A Novel Strategy to Overcome Tumor Microtubes - Mediated Radioresistance in Glioblastoma
              </a>
            </td>
          <td>
            I. Dokic, Federica Ciamarone, D. Hoffmann, J. Bojcevski, D. Krunic, T. Tessonnier, F. Winkler, Jürgen Debus, V. Venkataramani, A. Mairani, W. Wick, Amir Abdollahi
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="
 DNA double strand breaks (DSBs) result in activation of several key DNA damage response (DDR) kinases including ATM, ATR, and DNA-PK. These protein kinases not only promote DNA damage-induced checkpoint control, but also facilitate DSB repair in humans. Thus, these DDR kinases have become promising drug targets for cancer therapy. However, the benefits of targeting DDR kinases remain to be realized, in part due to the lack of predictive biomarkers. By undertaking CRISPR screens with inhibitors targeting key DDR kinases, we obtained a global and unbiased view of genetic interactions with DDR inhibition. Additionally, we compared the synergistic effects of combining different DDR inhibitors and found that ATM and PARP inhibitors showed remarkable synergy, which depends on the dominant negative function of ATM inhibition. Moreover, to provide comprehensive and unbiased perspective of DDR signaling pathways, we performed 30 fluorescence-activated cell sorting–based genome-wide CRISPR screens with antibodies recognizing distinct endogenous DNA damage–signaling proteins to identify new regulators involved in DNA damage response (DDR). We discovered that proteasome-mediated processing is an early and prerequisite event for cells to trigger camptothecin- and etoposide-induced DDR signaling. Furthermore, we identified PRMT1 and PRMT5 as new modulators that regulate ATM protein level. Additionally, we discovered that GNB1L is a master regulator of DDR signaling via its role as a co-chaperone for PIKK proteins. Collectively, these screens offer a rich resource for further investigation of DDR, which may provide insight into strategies of targeting these DDR pathways to improve therapeutic outcomes. More recently, we have adopted dTAG technology for the investigation of many essential DDR and replication proteins, which provide mechanistic insights into the roles of these essential proteins in genome maintenance. Additionally, we have expanded FACS-based screens for the studies of other cancer-associated pathways and explored key biological processes critical for cancer progression in vivo. These studies will facilitate the development of better therapies for cancer patients.
 Citation Format: Junjie Chen. Targeting DNA damage responsive pathways in cancer therapy. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr A009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b542469fcebc2361e69bec6d43fd73238a87bdc" target='_blank'>
              Abstract A009: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Fungal genome sizes exhibit more than a 100-fold variation, largely driven by the expansion of repetitive sequences such as transposable elements (TEs). Genomic defenses have evolved to counteract TE proliferation through silencing mechanisms operating at the epigenetic or transcript level. In fungi, repeat-induced point mutation (RIP) targets TEs by recognizing repetitive sequences and inducing mutagenesis. However, the prevalence of RIP across the fungal kingdom and the fidelity of the canonical C-to-T mutation signatures remain unclear. In this study, we address these gaps by tracking shifts in genome architecture across the fungal kingdom. We show that the loss of RIP components is associated with an 80% (∼30 Mb) increase in genome size in ascomycetes (class of Leotiomycetes). To track the impact of genome defenses, we designed a quantitative screen for RIP-like mutation signatures. The phylum of ascomycetes was unique in showing enrichment in mutation signatures in non-coding and repetitive sequences, consistent with a phylogenetically restricted occurrence of RIP-like genome defense systems. Then, we gene functions associated in a phylogenetic context with RIP-like mutation signatures. We identified a zinc-finger protein as the strongest candidate underpinning a novel mechanism of genome defenses. Our findings reveal the multifaceted drivers of genome defense systems and their close ties to genome size evolution in fungi, highlighting how proximate molecular mechanisms can shape genome evolution on deep phylogenetic scales.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6148de5b7f0cf13898d5f28034ac9ab633d52a0" target='_blank'>
              A systematic screen for genetic factors underpinning transposon defense systems across the fungal kingdom
              </a>
            </td>
          <td>
            Thomas Badet, D. Croll
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/352956df3f50dd3a8802a00f3ce2fb6c110258ba" target='_blank'>
              SMC translocation is unaffected by an excess of nucleoid associated proteins in vivo
              </a>
            </td>
          <td>
            Zhongqing Ren, Lindsey E. Way, Xindan Wang
          </td>
          <td>2025-01-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87208750323d073805f3062a30f76b4849f02905" target='_blank'>
              Unveiling cell-type-specific microRNA networks through alternative polyadenylation in glioblastoma
              </a>
            </td>
          <td>
            Mert Cihan, Greta Schmauck, Maximilian Sprang, Miguel Andrade
          </td>
          <td>2025-01-21</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2e7583ed7379bcbc35a51a5c2875a8894ab63e6" target='_blank'>
              Global characterization of extrachromosomal circular DNAs in four body fluids.
              </a>
            </td>
          <td>
            Lanrui Jiang, Mengyuan Song, Feng Song, Shuangshuang Wang, Yuxiang Zhou, Ze Wang, Chaoran Sun, Hewen Yao, Zhirui Zhang, Xindi Wang, Miao Liao, Yanyun Wang, Haibo Luo
          </td>
          <td>2025-02-13</td>
          <td>International journal of legal medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a significant global health challenge, underscoring the need for innovative therapeutic strategies. Oncogenic miRNAs (oncomiRs) play a significant biological role in the initiation and progression of colorectal cancer. Inspired by the cooperative mechanisms of plant nanovirus, which employ multiple circular single-stranded DNA (CssDNA) genomes, it is hypothesized that the development and delivery of CssDNA to target oncomiRs would achieve therapeutic benefits in CRC. In this study, a multi-omics approach is utilized to identify key tumor suppressor genes (TSGs) and their related oncomiRs implicated in CRC, followed by the development of CssDNA, each of which is loaded with multiple miRNA binding sites targeting one oncomiR. When transfected into the cells, these CssDNA can effectively target and sequester the corresponding oncomiRs to restore the expression of TSGs, leading to a marked reduction in CRC development both in vitro and in vivo. The findings highlight the therapeutic potential of nanovirus-inspired CssDNA in modulating the miRNA-mediated regulatory network in CRC. This study lays the groundwork for the development of non-coding DNA-based therapies with broad implications for the treatment of colorectal cancer and potentially other malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a389d3e4c09149bfce4cb4ae8bf6fdc4a619a01" target='_blank'>
              Circular Single-Stranded DNA-Based Artificial Nanoviruses Mitigate Colorectal Cancer Development.
              </a>
            </td>
          <td>
            Jinghao Wang, Pengfei Zhang, Yonglian Huang, Guang Hu, Kexuan Zou, Songtao Zhou, Dandan Shao, Jianming Wang, Jie Song
          </td>
          <td>2025-01-21</td>
          <td>Small methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Gene methylation in cells is an important factor in tumorigenesis, and radiotherapy can change DNA methylation in cells. In this study, complete genome methylation sequencing (BS-Seq) technology was used to analyze the genome-wide methylation of patients with cervical cancer before and after radiotherapy. Methods Three pairs of cervical squamous cell carcinoma samples were collected from patients before and after radiotherapy in July 2020. Genome-wide DNA methylation profiles were generated using WGBS. Bioinformatics analysis was conducted to identify differential methylation regions (DMRs) and their associated genes and pathways. The study focused on the methylation changes of LHX2, LHX5, and LHX9 genes, assessing their expression levels using qRT-PCR and correlating these changes with cervical cancer stages. Results MCG was the main way of genomic DNA methylation in the three patients. The DNA methylation level and methylation density on each chromosome varied greatly. As revealed by comparison of methylation before and after radiation in the three patients, 1287, 1261 and 789 differential methylation genes were identified, respectively. 3) Combined with clinical treatment, methylation level difference and correlation enrichment analysis, it was found that LHX2, LHX5 and LHX9 were closely related to the occurrence and development of cervical cancer. After 5-Aza-DC and radiotherapy, the methylation of the CpG islands in LHX2, LHX5 and LHX9 genes in these patients was decreased (p < 0.01), and the mRNA and protein expression levels were relatively increased (p < 0.01). Conclusion In our present work, genome-wide DNA methylation maps of cervical cancer tissues before and after radiotherapy were successfully constructed. We found that LHX5 and LHX9 genes are closely related to cervical cancer. LHX5 and LHX9 have a negative effect on cervical cancer. The migration ability of LHX9 silenced cells was significantly enhanced after irradiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fea2269d56f3d3d8896922414dbb511f2559dfa8" target='_blank'>
              Whole-Genome Methylation Sequencing Analysis and Functional Verification of LIM-Homeobox Family Genes in Cervical Cancer
              </a>
            </td>
          <td>
            Rong Yu, Qin Yu, Jie Shi, Xue Meng, Zhiyuan Deng, Jing Suo, Hao Yang
          </td>
          <td>2025-01-01</td>
          <td>International Journal of General Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Cancer cells arise from multiple complementary genomic and epigenomic abnormalities that deregulate the pathways controlling cell proliferation, cell survival, and tissue homeostasis. Epigenetic modifications of histones, DNA, chromatin structure, and non-coding RNA expression are emerging as critical determinants of gene expression and essential drivers of tumor cell heterogeneity and neoplastic cell phenotypes. Multipotent stem-like cells (also referred to as cancer stem cells, CSCs) are critical determinants of tumor propagation, therapeutic resistance, and recurrence following treatment. However, our understanding of how these multidimensional epigenetic modifications cooperatively drive cancer cell stemness and how to efficaciously target them is currently inadequate to positively impact patient outcomes. Abstract Background/Objectives: CSCs are critical drivers of the tumor and stem cell phenotypes of glioblastoma (GBM) cells. Chromatin modifications play a fundamental role in driving a GBM CSC phenotype. The goal of this study is to further our understanding of how stem cell-driving events control changes in chromatin architecture that contribute to the tumor-propagating phenotype of GBM. Methods: We utilized computational analyses to identify a subset of clinically relevant genes that were predicted to be repressed in a Polycomb repressive complex 2 (PRC2)-dependent manner in GBM upon induction of stem cell-driving events. These associations were validated in patient-derived GBM neurosphere models using state-of-the-art molecular techniques to express, silence, and measure microRNA (miRNA) and gene expression changes. Advanced Poly(β-amino ester) nanoparticle formulations (PBAEs) were used to deliver miRNAs in vivo to orthotopic human GBM tumor models. Results: We show that glioma stem cell (GSC) formation and tumor propagation involve the crosstalk between multiple epigenetic mechanisms, resulting in the repression of the miRNAs that regulate PRC2 function and histone H3 lysine 27 tri-methylation (H3K27me3). We also identified miR-217-5p as an EZH2 regulator repressed in GSCs and showed that miR-217-5p reconstitution using advanced nanoparticle formulations re-activates the PRC2-repressed genes, inhibits GSC formation, impairs tumor growth, and enhances the effects of ionizing radiation in an orthotopic model of GBM. Conclusions: These findings suggest that inhibiting PRC2 function by targeting EZH2 with miR-217-5p advanced nanoparticle formulations could have a therapeutic benefit in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a650a5c81b8e5180cf08ed0736baa5ed09de3eb0" target='_blank'>
              miR-217-5p NanomiRs Inhibit Glioblastoma Growth and Enhance Effects of Ionizing Radiation via EZH2 Inhibition and Epigenetic Reprogramming
              </a>
            </td>
          <td>
            Jack Korleski, Sweta Sudhir, Yuan Rui, Christopher Caputo, Sophie Sall, Amanda L Johnson, Harmon Khela, Tanmaya Madhvacharyula, Anisha Rasamsetty, Yunqing Li, B. Lal, Weiqiang Zhou, Karen Smith-Connor, Stephany Tzeng, Jordan Green, J. Laterra, H. Lopez-Bertoni
          </td>
          <td>2024-12-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/125e72b0b5ffa707da90fa52c6a8e151fa428862" target='_blank'>
              TRACERx analysis identifies a role for FAT1 in regulating chromosomal instability and whole-genome doubling via Hippo signalling
              </a>
            </td>
          <td>
            Wei-Ting Lu, L. Zalmas, Chris Bailey, James R. M. Black, C. Martínez-Ruiz, O. Pich, F. Gimeno-Valiente, Ieva Usaite, Alastair Magness, K. Thol, Thomas A. Webber, Ming Jiang, R. Saunders, Yun-Hsin Liu, D. Biswas, Esther O. Ige, Birgit Aerne, E. Grönroos, Subramanian Venkatesan, Georgia Stavrou, Takahiro Karasaki, M. Al Bakir, M. Renshaw, Hang Xu, Deborah Schneider-Luftman, Natasha Sharma, Laura Tovini, M. Jamal-Hanjani, S. McClelland, K. Litchfield, Nicolai J. Birkbak, Michael Howell, N. Tapon, K. Fugger, N. Mcgranahan, J. Bartek, N. Kanu, C. Swanton
          </td>
          <td>2024-12-30</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a614193ffb0a35548300d5126509bf2d1bf067aa" target='_blank'>
              Defining ortholog-specific UHRF1 inhibition by STELLA for cancer therapy
              </a>
            </td>
          <td>
            Wenjing Bai, Jinxin Xu, Wenbin Gu, Danyang Wang, Ying Cui, Weidong Rong, Xiaoan Du, Xiaoxia Li, Cuicui Xia, Qingqing Gan, Guantao He, Huahui Guo, Jinfeng Deng, Yuqiong Wu, R. Yen, Srinivasan Yegnasubramanian, S. Rothbart, Cheng Luo, Linping Wu, Jinsong Liu, Stephen B Baylin, Xiangqian Kong
          </td>
          <td>2025-01-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae1a74726e395311f7dc9708215816307d41962f" target='_blank'>
              Asymmetric distribution of G-quadruplex forming sequences in genomes of retroviruses
              </a>
            </td>
          <td>
            Filip Kledus, Michaela Dobrovolná, J. Mergny, Václav Brázda
          </td>
          <td>2025-01-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/272731c989a5ab1beb52c9be0f816a9c8eb7f104" target='_blank'>
              Homologous recombination promotes non-immunogenic mitotic cell death upon DNA damage
              </a>
            </td>
          <td>
            R. Szmyd, S. Casolin, Lucy French, A.G. Manjon, Melanie Walter, Léa Cavalli, Christopher B. Nelson, Scott G Page, Andrew Dhawan, Eric Hau, Hilda A. Pickett, Harriet Gee, Anthony J. Cesare
          </td>
          <td>2025-01-13</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acca60e9e5684fac9b42ec113ffc78d1e571a20d" target='_blank'>
              Pan-inhibition of super-enhancer-driven oncogenic transcription by next-generation synthetic ecteinascidins yields potent anti-cancer activity.
              </a>
            </td>
          <td>
            Max Cigrang, Julian Obid, Maguelone Nogaret, Léane Seno, Tao Ye, G. Davidson, Philippe Catez, P. Berico, Clara Capelli, Clara Marechal, Amélie Zachayus, Clémence Elly, Marie Jose Guillen Navarro, Marta Martínez Diez, Gema Santamaría Nuñez, Tsai-Kun Li, E. Compe, Pablo Aviles, Irwin Davidson, J. Egly, Carmen Cuevas, F. Coin
          </td>
          <td>2025-01-08</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="DNA G-quadruplexes (G4s) are non-B form secondary DNA structures that are highly conserved across evolution. G4 structures occupy key regulatory sites in the mammalian genomes and are implicated in several cellular processes. However, the mechanisms by which G4s contribute to distinct facets of genome function are not well understood. Here, we conduct a proteomics screen with G4s of diverse topologies to uncover novel G4 binding activities in genomic regulators of nucleosome remodeling, paraspeckle assembly, RNA splicing, transcriptional elongation, and 3D genome organization. Among the most prominent hits, we identify the genomic architectural protein, CTCF, as one of the strongest G4 binders. Building on this discovery, we perform extensive biochemical validation of CTCF-G4 interaction and identify CTCF mutants, with pronounced affinity for G4s over its consensus DNA motif. By implementing well-established approaches and developing new G4 mapping tools, we define a comprehensive catalog of genomic G4s and investigate their association with CTCF binding. Using genetic reconstitution of mouse embryonic stem cells with a G4-specific CTCF mutant, we ascribe genomic functions to CTCF-G4 interactions in regulation of CTCF occupancy, chromatin looping and gene expression. Interestingly, our studies reveal a subset of G4-linked chromatin loop anchors that form “persistent loops” which are retained even upon CTCF depletion. Collectively, our work establishes the G4 binding activity of CTCF and provides new insights into the functional significance of G4 structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d70927bc9ed14c4ce7fc52ee5d90b72979e548f3" target='_blank'>
              CTCF directly binds G-quadruplex structures to regulate genome topology and gene expression
              </a>
            </td>
          <td>
            Daniela Samaniego-Castruita, Isabella Han, Roxroy C Morgan, Samantha Carpenter, Bryce Williams, Abhijit Chakraborty, Ishwar Radhakrishnan, F. Ay, Samuel A. Myers, Anjana Rao, Vipul Shukla
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Abstract Evidence suggests that increases in ploidy have occurred frequently in the evolutionary history of organisms and can serve adaptive functions to specialized somatic cells in multicellular organisms. However, the sudden multiplication of all chromosome content may present physiological challenges to the cells in which it occurs. Experimental studies have associated increases in ploidy with reduced cell survival and proliferation. To understand the physiological challenges that suddenly increased chromosome content imposes on cells, we used S. cerevisiae to ask how much chromosomal DNA cells may contain and what determines this limit. We generated polyploid cells using 2 distinct methods causing cells to undergo endoreplication and identified the maximum ploidy of these cells, 32–64C. We found that physical determinants that alleviate or exacerbate cell surface stress increase and decrease the limit to ploidy, respectively. We also used these cells to investigate gene expression changes associated with increased ploidy and identified the repression of genes involved in ergosterol biosynthesis. We propose that ploidy is inherently limited by the impacts of growth in size, which accompany whole-genome duplication, to cell surface integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4c9419ff84380a674e570b7e466126c557aa8fb" target='_blank'>
              Cell integrity limits ploidy in budding yeast
              </a>
            </td>
          <td>
            Juliet Barker, Andrew W. Murray, Stephen P Bell
          </td>
          <td>2025-01-13</td>
          <td>G3: Genes | Genomes | Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/824026b9cf3feecceded2241b45f5f7f89022d53" target='_blank'>
              A CRISPR-Cas9 screen reveals genetic determinants of the cellular response to decitabine.
              </a>
            </td>
          <td>
            Pinqi Zhang, Zhuqiang Zhang, Yiyi Wang, Wenlong Du, Xingrui Song, Weiyi Lai, Hailin Wang, Bing Zhu, Jun Xiong
          </td>
          <td>2025-02-10</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Epigenetic therapies that induce global DNA hypomethylation activate transcriptionally repressed transposable elements (TEs). The activation of a class of TEs known as human endogenous retroviruses (HERVs) can trigger an innate antiviral response in cells called “viral mimicry”. Viral mimicry induced by aberrantly expressed HERVs stimulates interferon signaling genes resulting in an adaptive immune response that can ultimately lead to immunogenic cell death. This heightened immune activity can enhance the efficacy of immune checkpoint inhibitor therapies. Identifying cancers that are susceptible to a viral mimicry response or treatment regimens that induce aberrant transcription of TEs has not yet been fully explored. Recent computational tools enable the measurement of TE signal from short-read RNA-seq data and several large-scale reprocessing pipelines have uniformly processed RNA-seq data of cancer cell lines treated with epigenetic and cytotoxic drugs. However, these large-scale pipelines omit the quantification of TEs. To address this gap and fully utilize publicly available data, we have developed a computational pipeline and database of TE expression for more than 7,000 samples from 220 cancer cell lines treated with over 700 cytotoxic and epigenetic drugs, yielding over 2,100 differential expression experiments. To illustrate the usefulness of this database, we ranked drugs by %TE activation and found that the top three targets were DNMT (Decitabine), CDK9 (MC180295), and Topoisomerase II (Doxorubicin). We enable the exploration of this dataset by implementing several meta-analysis methods and visualizations made available as an interactive web application. This database and web application can serve as a valuable resource for the cancer and epigenetics community by enabling hypothesis generation and providing insights into how specific drugs may induce TE expression.
 Citation Format: Gennaro H Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre Issa. TEDEdb: Transposable element differential expression database [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B022.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5443bedc86457ac044103f87e00c65e563c2339" target='_blank'>
              Abstract B022: TEDEdb: Transposable element differential expression database
              </a>
            </td>
          <td>
            G. Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre J Issa
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer's inherent ability to evolve presents significant challenges for its categorization and treatment. Cancer evolution is driven by genetic, epigenetic, and phenotypic diversity influenced by microenvironment changes. Aging plays a crucial role by altering the microenvironment and inducing substantial genetic and epigenetic heterogeneity within an individual's somatic cells even before cancer initiation.This review highlights the clinical significance of epigenetic mechanisms in cancer evolution, focusing on hematopoietic and solid tumors. The review aims to explore opportunities for integrating evolutionary principles and data science into cancer research.The review synthesizes recent advancements in omics technologies, single‐cell sequencing, and genetic barcoding to elucidate epigenetic mechanisms and aging's role in cancer evolution.Epigenetic mechanisms' high plasticity generates heritable phenotypic diversity, driving malignant evolution toward poor prognosis. Advances in single‐cell sequencing and genetic barcoding enable the precise detection and tracking of biomarkers, allowing early, personalized interventions. Incorporating data science into cancer research has the potential to map, predict, and prevent cancer evolution effectively.Understanding cancer evolution through novel technologies and data analysis offers a proactive approach to cancer prevention and treatment. By predicting key evolutionary events and leveraging personalized strategies, patient outcomes can be improved, and healthcare burdens reduced, marking a transformative shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc8601f31e88365526c6e3fa1b322df2e0a2b70" target='_blank'>
              Deciphering Aging, Genetic, and Epigenetic Heterogeneity in Cancer Evolution: Toward Personalized Precision Preventative Medicine
              </a>
            </td>
          <td>
            Lamis Naddaf, Sheng Li
          </td>
          <td>2025-01-28</td>
          <td>Aging and Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab8b68e7b5a2f581513237acff3c4e88418f66b" target='_blank'>
              Haploinsufficient phenotypes promote selection of PTEN and ARID1A-deficient clones in human colon.
              </a>
            </td>
          <td>
            Nefeli Skoufou-Papoutsaki, Sam Adler, Shenay Mehmed, Claire Tume, Cora Olpe, Edward Morrissey, Richard Kemp, Anne-Claire Girard, Elisa B. Moutin, C. Chilamakuri, Jodi L Miller, Cecilia Lindskog, Fabian Werle, Kate Marks, Francesca Perrone, Matthias Zilbauer, D. Tourigny, Doug Winton
          </td>
          <td>2025-02-07</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Bladder cancer (BC) is a malignant tumor characterized by a high incidence of urinary system diseases. The complex pathogenesis of BC has long been a focal point in medical research. With the robust development of epigenetics, the crucial role of epigenetic modifications in the occurrence and progression of BC has been elucidated. These modifications not only affect gene expression but also impact critical biological behaviors of tumor cells, including proliferation, differentiation, apoptosis, invasion, and metastasis. Notably, DNA methylation, an important epigenetic regulatory mechanism, often manifests as global hypomethylation or hypermethylation of specific gene promoter regions in BC. Alterations in this methylation pattern can lead to increased genomic instability, which profoundly influences the expression of proto-oncogenes and tumor suppressor genes. MiRNAs, as noncoding small RNAs, participate in various biological processes of BC by regulating target genes. Consequently, this work aims to explore the interaction mechanisms between DNA methylation and miRNAs in the occurrence and development of BC. Research has demonstrated that DNA methylation not only directly influences the expression of miRNA genes but also indirectly affects the maturation and functionality of miRNAs by modulating the methylation status of miRNA promoter regions. Simultaneously, miRNAs can regulate DNA methylation levels by targeting key enzymes such as DNA methyltransferases (DNMTs), thereby establishing a complex feedback regulatory network. A deeper understanding of the crosstalk mechanisms between DNA methylation and miRNAs in BC will contribute to elucidating the complexity and dynamics of epigenetic modifications in this disease, and may provide new molecular targets and strategies for the early diagnosis, treatment, and prognostic evaluation of BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a4c5da3f83b53fe6663c6f857d2ab94191b45d2" target='_blank'>
              Epigenetic modifications in bladder cancer: crosstalk between DNA methylation and miRNAs
              </a>
            </td>
          <td>
            Wei Li, Peiyue Luo, Qi Chen, Le Cheng, Lifeng Gan, Fangtao Zhang, Haidong Zhong, Liying Zheng, Biao Qian
          </td>
          <td>2025-02-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5202cd39039a85ffc734b688a9ee572ce5b91b15" target='_blank'>
              Inhibition of GSK3β is synthetic lethal with FHIT loss in lung cancer by blocking homologous recombination repair
              </a>
            </td>
          <td>
            Shishi Tao, Yue Pu, E. Yang, Guowen Ren, Changxiang Shi, Li-Jie Chen, Liang Chen, J. Shim
          </td>
          <td>2025-01-06</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Tumor suppressor gene TP53 is ubiquitously mutated in ovarian cancer precursor lesions that undergo pervasive accumulation of DNA damage. Prior literature provides evidence that the expression of mutant p53 protein in epithelial cells is associated with increased survival in response to DNA-damaging treatments. Hence, identification and understanding of the mechanisms that, in response to DNA damage accumulation, support survival of ovarian cancer precursor cells carrying p53 mutations might provide important information about the evolution of the disease. Here we used a combination of OC precursor cell models, biochemistry, microscopy, and flow cytometry to provide evidence that the taurine transporter, the SLC6A6 molecule, contributes to cell protection from DNA-damaging (cisplatin) treatment. We found that expression of mutant p53R175H in OC precursor cells, the fallopian tube non-ciliated epithelial (FNE) cells, induced resistance to the DNA-damaging agent cisplatin. Most importantly, shRNA-mediated targeting of SLC6A6 transcript re-sensitized FNE cells expressing mutant p53R175H to cisplatin treatment. Our studies are consistent with the model that the loss of SLC6A6 alters mechanisms involved in the regulation of cell survival in response to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/258ec54986c78fd95aedc9a7b2b989006fafe5e2" target='_blank'>
              Mutant p53R175H-Associated Protection of Ovarian Cancer Precursor Cells from Cisplatin Requires Expression of SLC6A6 Taurine Transporter
              </a>
            </td>
          <td>
            Teagan Polotaye, Daniel Centeno, Marcin Iwanicki
          </td>
          <td>2024-08-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfc051f8dccd4e965261720e0c363a2fb399619e" target='_blank'>
              Aberrant expression of GTPase-activating protein ARAP1 triggers circular dorsal ruffles associated with malignancy in hepatocellular carcinoma Hep3B cells
              </a>
            </td>
          <td>
            Xiaowei Sun, Yanan Li, Yuxin He, Longjiao Cheng, Li Wang, Jinzi Wei, Jianan Chen, Linxuan Du, Zhongyang Shen, Yan Xie, Adam C. Midgley, Wentao Jiang, Sei Yoshida
          </td>
          <td>2025-02-11</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b2824e967c6a4d06b3abc17af2f0703099dbde0" target='_blank'>
              Uncovering the whole genome silencers of human cells via Ss-STARR-seq
              </a>
            </td>
          <td>
            Xiusheng Zhu, Lei Huang, Chao Wang, Guoli Li, Biao Deng, Dashuai Kong, Xiaoxiao Wang, Rongrong Chang, Yi Gu, Qiuhan Wen, Siyuan Kong, Yuwen Liu, Yubo Zhang
          </td>
          <td>2025-01-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Achieving targeted hypermutation of specific genomic sequences without affecting other regions remains a key challenge in continuous evolution. To address this, we evolved a T7 RNA polymerase (RNAP) mutant that synthesizes single-stranded DNA (ssDNA) instead of RNA in vivo, while still exclusively recognizing the T7 promoter. By increasing the error rate of the T7 RNAP mutant, it generates mutated ssDNA that recombines with homologous sequences in the genome, leading to targeted genomic hypermutation. This approach, termed T7 RNAP mutant-assisted continuous evolution (T7ACE), functions effectively in both typical prokaryotic and eukaryotic microorganisms (Escherichia coli and Saccharomyces cerevisiae), achieving targeted hypermutations at rates 2800- and 1200-fold higher than the genomic mutation rates, respectively. Using T7ACE, we successfully evolved an eight-fold increase in tigecycline resistance within 7 days and doubled the efficiency of a xylose utilization pathway in 10 days, demonstrating the efficiency and broad applicability of this single-component tool for continuous evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8bff78f49477a4ea5070aa46ecc36f6c1f66021" target='_blank'>
              An evolved, orthogonal ssDNA generator for targeted hypermutation of multiple genomic loci
              </a>
            </td>
          <td>
            Weiran Chu, Rongzhen Tian, Yaxin Guo, Yaokang Wu, Fabian B H Rehm, Long Liu, Jianghua Li, Guocheng Du, Jian Chen, Yanfeng Liu
          </td>
          <td>2025-01-24</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c26d06bb50d508e1ca0550e77b81ac840e53150" target='_blank'>
              PARP4 deficiency enhances sensitivity to ATM inhibitor by impairing DNA damage repair in melanoma
              </a>
            </td>
          <td>
            Yuehua Li, Yu Liu, Jingjing Ma, Yuqi Yang, Q. Yue, Guannan Zhu, Weinan Guo, Tianwen Gao, Qiong Shi, Chunying Li
          </td>
          <td>2025-01-30</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Identifying genes involved in cancer is crucial for understanding the underlying molecular mechanisms of the disease and developing effective treatment strategies. Differential expression analysis (DEA) is the predominant method used to identify cancer-related genes. This approach involves comparing gene expression levels between different samples, such as cancerous and non-cancerous tissues, to identify genes that are significantly upregulated or downregulated in cancer. DEA is based on the commonly believed assumption that genes upregulated in cancerous tissues have the potential to function as oncogenes. Their expression levels often correlate with cancer advancement and unfavorable prognosis, whereas downregulated genes display the opposite correlation. However, contrary to the prevailing belief, our analysis utilizing The Cancer Genome Atlas (TCGA) databases revealed that the upregulated or downregulated genes in cancer do not always align with cancer progression or prognosis. These findings emphasize the need for alternative approaches for identifying cancer-related genes that may be more accurate and effective. To address this need, we compared the effectiveness of machine learning (ML) methods with that of traditional DEA in the identification of cancer-related genes. ML algorithms have the advantage of being able to analyze large-scale genomic data and identify complex patterns that may go unnoticed by traditional methods. Our results demonstrated that ML methods significantly outperformed DEA in the screening of cancer-related genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6167fa4f27157ad0473050eb8b2f5d919ef194a8" target='_blank'>
              Challenging Conventional Perceptions of Oncogenes and Tumor Suppressor Genes: A Comprehensive Analysis of Gene Expression Patterns in Cancer.
              </a>
            </td>
          <td>
            Mingyuan Zou, Li Qiu, Wentao Wu, Hui Liu, Han Xiao, Jun Liu
          </td>
          <td>2025-02-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer (BC) is one of the most prevalent forms of cancer globally, and has recently become the leading cause of cancer-related mortality in women. BC is a heterogeneous disease comprising various histopathological and molecular subtypes with differing levels of malignancy, and each patient has an individual prognosis. Etiology and pathogenesis are complex and involve a considerable number of genetic alterations and dozens of alterations in non-coding RNA expression. Non-coding RNAs are part of an abundant family of single-stranded RNA molecules acting as key regulators in DNA replication, mRNA processing and translation, cell differentiation, growth, and overall genomic stability. In the context of breast cancer, non-coding RNAs are involved in cell cycle control and tumor cell migration and invasion, as well as treatment resistance. Alterations in non-coding RNA expression may contribute to the development and progression of breast cancer, making them promising biomarkers and targets for novel therapeutic approaches. Currently, the use of non-coding RNAs has not yet been applied to routine practice; however, their potential has been very well studied. The present review is a literature overview of current knowledge and its objective is to delineate the function of diverse classes of non-coding RNAs in breast cancer, with a particular emphasis on their potential utility as diagnostic and prognostic markers or as therapeutic targets and tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0045975d00155c439ba80ba1b588114b3af277" target='_blank'>
              Non-Coding RNAs in Breast Cancer: Diagnostic and Therapeutic Implications
              </a>
            </td>
          <td>
            Roman Beňačka, Daniela Szabóová, Z. Guľašová, Z. Hertelyová
          </td>
          <td>2024-12-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="DNA damage is a common event in cells, resulting from both internal and external factors. The maintenance of genomic integrity is vital for cellular function and physiological processes. The inadequate repair of DNA damage results in the genomic instability, which has been associated with the development and progression of various human diseases. Accumulation of DNA damage can lead to multiple diseases, such as neurodegenerative disorders, cancers, immune deficiencies, infertility and aging. This comprehensive review delves the impact of alterations in DNA damage response genes (DDR) and tries to elucidate how and to what extent the same traits modulate diverse major human diseases, such as cancer, neurodegenerative diseases, and immunological disorders. DDR is apparently the trait connecting important complex disorders in humans. However, the pathogenesis of the above disorders and diseases are different and leading to the divergent consequences. It is important to discover the switch(es) that direct further the pathogenic process either to proliferative, or degenerative diseases. Our understanding the influence of DNA damage on diverse human disorders may enable a development of the strategies to prevent, diagnose, and treat these diseases. In our article, we analysed publicly available GWAS summary statistics from the NHGRI-EBI GWAS Catalog and identified 12,009 single nucleotide polymorphisms (SNPs) associated with cancer. Among these, 119 SNPs were found in DDR pathways, exhibiting significant p-values. Additionally, we identified 44 SNPs linked to various cancer types and neurodegenerative diseases (NDDs), including four located in DDR-related genes: ATM, CUX2, and WNT3. Furthermore, 402 SNPs were associated with both cancer and immunological disorders, with two found in DDR gene RAD51B. This highlights the versatility of the DDR pathway in multifactorial diseases. However, the specific mechanisms that regulate DDR to initiate distinct pathogenic processes remain to be elucidated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9bdcdd5d823de10dac2aaf9306d09b471c64a04" target='_blank'>
              Defective DNA Repair: A Putative Nexus Linking Immunological Diseases, Neurodegenerative Disorders, and Cancer.
              </a>
            </td>
          <td>
            Safaa Andarawi, L. Vodickova, Anusha Uttarilli, Petr Hanak, P. Vodicka
          </td>
          <td>2025-02-12</td>
          <td>Mutagenesis</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bb4f23b9e1892130e632d422e0888edddfeef1" target='_blank'>
              Molecular heterogeneity and MYC dysregulation in triple-negative breast cancer: genomic advances and therapeutic implications.
              </a>
            </td>
          <td>
            Priya, Arun Kumar, Dhruv Kumar
          </td>
          <td>2025-01-01</td>
          <td>3 Biotech</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chromosome aberrations (CAs) play an important role in tumor initiation, followed by metastasis. Short-term peripheral blood lymphocyte (PBL) cultures
using RPMI-1640 media with 5 % FBS were used to evaluate the frequency of structural and numerical chromosome aberrations in breast cancer patients.
Highest frequency of aneuploidy (11.53%) were observed followed by trisomy -21(8.93%) in karyotypes. De-novo mutation involving 12q21 with loss
of 16.38 Mbp DNA fragment is the most relevant finding in breast cancer (BC) patients and has not been reported earlier. Secondly, role of trisomy-21
(8.93%) might have increase genetic susceptibility of disease because of “giant satellites”, are the active sites of rRNA sub unit of 18S and 28S. These active
sites might have increase gene-expression of truncated protein during tumor Significant increase in the frequency of aneuploidy with increased number
of non-homologous chromosome was the striking feature due to increase of non-disjunction event followed by unequal crossing-over and synapse
formation by known environmental factors like arsenic. However, the present study is small, but interesting to explore the etiopathology of BC patients
associated chromosome instability. However, further study is required to confirm these changes, whether these mutations are either familial (inherited)
or spontaneous in nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f684d4b189095eeab5e425ab3b1be62a314e19" target='_blank'>
              Genomic Instability with Non- Homologues Chromosomes in Aneuploid Cell Increase Risk Factor for the Development of Breast Cancer Patients
              </a>
            </td>
          <td>
            A. K. Saxena
          </td>
          <td>2025-01-31</td>
          <td>Journal of Medicine and Healthcare</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8fd6c1c580a3737a18f3485f8040f1c52d76c" target='_blank'>
              TRIM28-dependent developmental heterogeneity determines cancer susceptibility through distinct epigenetic states.
              </a>
            </td>
          <td>
            Ilaria Panzeri, Luca Fagnocchi, S. Apostle, Megan Tompkins, Emily Wolfrum, Z. Madaj, G. Hostetter, Yanqing Liu, Kristen Schaefer, Chih-Hsiang Yang, Alexis Bergsma, Anne Drougard, Erez Dror, Darrell P Chandler, Daniel Schramek, Timothy J. Triche, J. Pospisilik
          </td>
          <td>2025-01-24</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="While the genetic paradigm of cancer etiology has proven powerful, it remains incomplete as evidenced by the widening spectrum of non-cancer cell-autonomous “hallmarks” of cancer. Studies have demonstrated the commonplace presence of high oncogenic mutational burdens in homeostatically-stable epithelia. Hence, the presence of driver mutations alone does not result in cancer. Here, we reveal additional forces governing the eco-evolutionary dynamics of carcinogenesis. Using a UV-driven mouse model of cutaneous squamous cell carcinoma, we tested our central hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of these goliaths as evidenced by the presence of areas of unusually high local densities of cells (termed “micro-lumps”) with higher mutational burdens (evolutionary driver), and 3) emergence of macroscopic lesions. We tracked these tissue level ecological and evolutionary drivers of cancer initiation via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25,085 clade measurements from 14 mice over 3 months, and 14,525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little between UV and non-UV exposure (cc. 65,000 µm3), our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3), almost exclusively within UV-exposed skin. Goliath clades emerged by month 2, increased dramatically in number by months 3-4, and plateaued between months 5-6. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes. Unexpectedly, targeted DNA sequencing revealed mutations with very low variant allele frequencies within clades, but substantial differences among clades, suggesting that positive selection for these mutations is superfluous to the development of goliaths early in carcinogenesis. scRNAseq analysis of both bulk skin and sorted clades revealed epidermal de-differentiation and immune suppression as early events. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions began to emerge between months 6 and 7, only in UV-exposed skin. To confirm that goliath clades are orders of magnitude more likely to spawn lesions, we randomly selected 21 goliath clades at months 6-7, prior to the time of the emergence of detectable lesions, and followed them over time. Remarkably, 2 of these developed into macroscopic lesions. Our adaptation of the Drake equation estimated the probability of this to be <10-6. Taken together, our results support the presence of three phases of cancer initiation, the earliest of which presage the acquisition of driver mutations and also explains why cancers are relatively rare in relation to the degree of somatic mosaicism present in UV-exposed skin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75272d3f8095e90a23d2fb0d732b9102d07a0066" target='_blank'>
              Goliath clades and in vivo tracking of clonal dynamics show three phases of UV-induced skin carcinogenesis
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A. Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef930bd56fde447e683b8e74f0950f751b9d7434" target='_blank'>
              Reassessing the roles of oxidative DNA base lesion 8-oxoGua and repair enzyme OGG1 in tumorigenesis
              </a>
            </td>
          <td>
            Jing Wang, Chunshuang Li, Jinling Han, Yaoyao Xue, Xu Zheng, Ruoxi Wang, Z. Radák, Y. Nakabeppu, Istvan Boldogh, Xueqing Ba
          </td>
          <td>2025-01-01</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="IDH-mutant low-grade gliomas (LGGs) are slow-growing brain tumors that frequently progress to aggressive high-grade gliomas that have dismal outcomes. In a recent study, Wu and colleagues provide critical insights into the mechanisms underlying malignant progression by analyzing single-cell gene expression and chromatin accessibility across different tumor grades. Their findings support a two-phase model: in early stages, tumors are primarily driven by oligodendrocyte precursor-like cells and epigenetic alterations that silence tumor suppressors like CDKN2A and activate oncogenes such as PDGFRA. As the disease advances, the tumors become sustained by more proliferative neural precursor-like cells, where genetic alterations, including PDGFRA, MYCN, and CDK4 amplifications and CDKN2A/B deletion, drive tumor progression. The study further highlights a dynamic regulation of interferon (IFN) signaling during progression. In low-grade IDH-mutant gliomas, IFN responses are suppressed through epigenetic hypermethylation, which can be reversed with DNMT1 inhibitors or IDH inhibitors, leading to reactivation of the IFN pathway. In contrast, higher-grade gliomas evade IFN signaling through genetic deletions of IFN gene clusters. These findings emphasize a broader epigenetic-to-genetic shift in oncogenic regulation that drives glioma progression, provides a valuable framework for understanding the transition from indolent tumors to lethal malignancies, and has implications for therapy and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598e5f4259516cb6e86cc401bbe6fe663aa489ab" target='_blank'>
              Switching Drivers: Epigenetic Rewiring to Genetic Progression in Glioma.
              </a>
            </td>
          <td>
            K. Drucker, Robert B. Jenkins, Daniel Schramek
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Background Mitosis maintains a genome's genetic information in daughter cells by accurately segregating chromosomes. However, chromosome aberrations are common during early mammalian embryogenesis. Chromosomal abnormalities during the early stages of embryogenesis result in the formation of mosaic embryos, wherein cells with normal genomes coexist with cells exhibiting abnormal genomes. The precise frequency and etiology of such abnormalities remain unclear. It is postulated that these aberrations contribute to the etiology of a number of conditions, including infertility and congenital diseases such as Down's syndrome. Methods This review synthesizes current literature and data to elucidate the causes and implications of chromosome aberrations in early mammalian embryos. It places particular emphasis on identifying patterns of mosaicism and investigating the underlying mechanisms responsible for these abnormalities. Main Findings The underlying causes of chromosome abnormalities in early embryos were examined in the context of DNA replication and embryonic development. Conclusion A deeper understanding of chromosome abnormalities in early embryos could help develop new infertility treatments and advance research on cancers caused by these abnormalities. This article reviews current knowledge and gaps in understanding chromosome segregation abnormalities during embryogenesis and future directions in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0411fdbb93ab5a617be60af6f02c7f92304ed6b" target='_blank'>
              Chromosome segregation errors during early embryonic development
              </a>
            </td>
          <td>
            Hirohisa Kyogoku
          </td>
          <td>2025-01-01</td>
          <td>Reproductive Medicine and Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="p27Kip1 is a key cell cycle gatekeeper governing the timing of Cyclin-dependent kinase (CDK) activation/inactivation and, consequently, cell proliferation. Structurally, the protein is largely unfolded, a feature that strongly increases its plasticity and interactors and enhances the number of regulated cellular processes. p27Kip1, like other intrinsically unstructured proteins, is post-translationally modified on several residues. These modifications affect its cellular localization and address p27Kip1 for specific interactions/functions. Several germline or somatic CDKN1B (the p27Kip1 encoding gene) mutations have been demonstrated to be associated with multiple endocrine neoplasia type 4 (MEN4), hairy cell leukemia, small-intestine neuroendocrine tumors, and breast and prostate cancers. Here, we analyzed the effect of four CDKN1B missense and nonsense mutations found in patients affected by MEN4 or cancers, namely, c.349C>T, p.P117S; c.397C>A, p.P133T; c.487C>T, p.Q163*; and c.511G>T, p.E171*. By transfecting breast cancer cell lines, we observed increased growth and cell motility for all the investigated mutants compared to wild-type p27Kip1 transfected cells. Furthermore, we discovered that the mutant forms exhibited altered phosphorylation on key residues and different localization or degradation mechanisms in comparison to the wild-type protein and suggested a possible region as crucial for the lysosome-dependent degradation of the protein. Finally, the loss of p27Kip1 ability in blocking cell proliferation was in part explained through the different binding efficiency that mutant p27Kip1 forms exhibited with Cyclin/Cyclin-dependent Kinase complexes (or proteins involved indirectly in that binding) with respect to the WT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b858b52e6f58495fd391a4dfcc6f1900f00849e" target='_blank'>
              p27Kip1 and Tumors: Characterization of CDKN1B Variants Identified in MEN4 and Breast Cancer
              </a>
            </td>
          <td>
            Debora Bencivenga, E. Stampone, Jahanzaib Azhar, Daniela Parente, Waqar Ali, Vitale Del Vecchio, Fulvio Della Ragione, A. Borriello
          </td>
          <td>2025-01-26</td>
          <td>Cells</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56da4ec3ca3ec43d5746b51ede7497a19adee094" target='_blank'>
              Regulatory non-coding somatic mutations as drivers of neuroblastoma.
              </a>
            </td>
          <td>
            Annalaura Montella, Matilde Tirelli, V. A. Lasorsa, Vincenzo Aievola, Vincenza Cerbone, Rosa Manganiello, Achille Iolascon, Mario Capasso
          </td>
          <td>2025-01-23</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="DNA 5-methylcytosine (5mC) modification plays a pivotal role in many biological processes, yet 5mC information and pattern hidden behind remains to be explored. Here, we develop Methylation Language Model based on Quintuple Bidirectional Transformer (MethylQUEEN), a novel pre-trained DNA methylation foundation model capable of sensing methylation states and covering the genome-wide methylation landscape. Through tailored methylation-prone pre-training, MethylQUEEN effectively captured epigenetics information hidden within the DNA sequences: it accurately traces DNA’s tissue-of-origin, and successfully recovers the expression profile through methylation states. Integrative analysis on MethylQUEEN’s attention scores also enables us to reveal the unique methylation status of a tissue for precise disease detection, and identifying key regulatory 5mC sites for disease intervention. As a result, MethylQUEEN signifies a new paradigm in methylation analysis for various biological problems. Besides, our study demonstrates the effectiveness of directly integrating methylation information into pre-training, offering new perspectives and methodologies for a range of methylation-related biological processes. It serves as an initial exploration for the development of more comprehensive epigenomic models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/423eb56b00c3f4af40dbdd570aa3032ee701aef5" target='_blank'>
              MethylQUEEN: A Methylation Encoded DNA Foundation Model
              </a>
            </td>
          <td>
            Mingyang Li, Ruichu Gu, Shiyu Fan, Yu Fan, Bo He, Jinmin Yang, Yuting Chen, Mengling Xin, Han Wen, Chengqi Yi
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Rationale: Leukemic evolution occurs through gradual clonal selection at early and late stages of disease development. The early stage of this evolutionary process is called clonal hematopoiesis (CH) and is typified by expansion, and increased self-renewal capacity, of non-malignant, somatically mutated hematopoietic stem and progenitor cells (HSPCs). CH-mutant genes result in either genomic hypomethylation through loss of DNMT3A, or hypermethylation either by genetic loss of TET2 or its’ inhibition by the oncometabolite 2-HG generated by mutant IDH1/IDH2. Leukemic transformation is often associated with subsequent mutagenesis in a broader range of genes including mitogenic signaling genes that confer increased proliferative capacity (FLT3, NRAS). The stepwise acquisition of mutations generates a polyclonal “ecosystem” of clones with increasing clonal diversity indicative of disease progression and poor prognosis. The goal of this study is to 1) identify genetic evolutionary trends associated with distinct CH-related mutations in DNMT3A and TET2 and 2) implement innovative single cell DNA sequencing protocols to identify genotype-selective DNA methylation alterations at single cell resolution. Approach: We compiled patients diagnosed with de novo AML at the University of Pennsylvania that had multiple genomic profiling studies, a cohort comprising 182 patients. All patients were profiled at diagnosis, with n=119 at first remission and n=76 at first relapse. 84.6% of patients received induction chemotherapy with combination anthracycline and nucleoside analog therapy. To investigate the genomic changes in AML disease course, we calculated the change in variant allele frequencies (VAFs) for all identified variants, evaluating matched samples from individual patients across time. We infer phylogenetic relationships from matched genomic profiles across timepoints in individual patients using CALDER. Finally, we performed single cell DNA methylation sequencing on 3 patients (n=4000 cells) using the Single Cell Methylation Kit from Scale Biosciences. This is a combinatorial indexing-based whole genome bisulfite sequencing based approach. Results: Comparing VAFs between diagnosis and relapse for DNMT3Amut vs TET2mut AML, we observed that FLT3 and NPM1 variants were similarly split between increased and decreased VAFs at relapse vs diagnosis. NRAS mutations were both gained and lost in DNMT3Amut samples between diagnosis and relapse, but no NRAS mutations were identified at all in TET2mut samples. When inferring phylogeny associations, we found DNMT3Amut samples were more likely to undergo subclonal swapping from diagnosis to relapse, whereas TET2mut samples relapsed with a more stable mutation profile. Analyses are currently underway for single cell DNA methylation data. Conclusions: Our analyses suggest that DNMT3A and TET2 have distinct sets of co-occurring genetic alterations at diagnosis and relapse. Early CH-associated mutations preceding transformation can shape the evolutionary trajectories of AML throughout disease development.
 Citation Format: Nisargbhai Shah, Ryan Cho, Priya Velu, Selina Luger, Martin Carroll, Jennifer Morrissette, Michael Bowman, Robert Bowman. Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A021.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da48ac3143bd55d667bb07036faa9f708e803fd9" target='_blank'>
              Abstract A021: Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia
              </a>
            </td>
          <td>
            Nisargbhai Shah, Ryan D Chow, Priya Velu, Selina M Luger, Martin P Carroll, Jennifer J D Morrissette, Michael Bowman, Robert L Bowman
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Stable inheritance of DNA N6-methyladenine (6mA) is crucial for its biological functions in eukaryotes. Here, we identify two distinct methyltransferase (MTase) complexes, both sharing the catalytic subunit AMT1, but featuring AMT6 and AMT7 as their unique components, respectively. While the two complexes are jointly responsible for 6mA maintenance methylation, they exhibit distinct enzymology, DNA/chromatin affinity, genomic distribution, and knockout phenotypes. AMT7 complex, featuring high MTase activity and processivity, is connected to transcription-associated epigenetic marks, including H2A.Z and H3K4me3, and is required for the bulk of maintenance methylation. In contrast, AMT6 complex, with reduced activity and processivity, is recruited by PCNA to initiate maintenance methylation immediately after DNA replication. These two complexes coordinate in maintenance methylation. By integrating signals from both replication and transcription, this mechanism ensures the faithful and efficient transmission of 6mA as an epigenetic mark in eukaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2893bf0a4630bdeb041736c2ee7ebc2881aa61a" target='_blank'>
              Dual modes of DNA N6-methyladenine maintenance by distinct methyltransferase complexes.
              </a>
            </td>
          <td>
            Yuanyuan Wang, Bei Nan, Fei Ye, Zhe Zhang, Wentao Yang, Bo Pan, Fan Wei, Lili Duan, Haicheng Li, Junhua Niu, Aili Ju, Yongqiang Liu, Dantong Wang, Wenxin Zhang, Yifan Liu, Shan Gao
          </td>
          <td>2025-01-15</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 In colon cancer, the BRAFV600E mutation is strongly associated with the CpG island methylator phenotype (CIMP). Here, we characterized the contribution of BRAFV600E to maintenance of aberrant DNA methylation. A reverse CRISPR gene editing approach was applied to revert the V600E mutation in BRAF back to wildtype (E600V) in organoids derived from a late-stage tumour. DNA methylation analyses identified 5187 differentially methylated CpGs within CpG islands, predominantly hypermethylated (82%) in BRAFV600E organoids, including genes associated with CIMP, as well as Polycomb Repressor Complex 2 (PRC2) target genes. RNA sequencing showed that expression of those genes was concordantly repressed. Furthermore, BRAFV600Einduced high expression of PRC2 core components (EZH2, SUZ12, EED), caused PRC2-induced H3K27 trimethylation in promoter regions, and maintained a PRC2-associated embryonic phenotype. This phenotype was lost following mutation correction or pharmacological inhibition of DNA methylation. These findings show that BRAFV600E maintains aberrant DNA and histone methylation patterns in advanced colon cancer, likely preserving the transformed phenotype by silencing of PRC2 target genes. Epigenetic therapies may have value in the treatment of BRAFV600E-mutant colon cancer.
 Citation Format: Layla El Bouazzaoui, Jeroen M Bugter, Emre Küçükköse, André Verheem, Jasmin B Post, Nicola Fenderico, Inne H.M Borel Rinkes, Hugo J.G Snippert, Madelon M Maurice, Onno Kranenburg. BRAFV600E is essential for maintenance of the CpG island methylator phenotype and DNA methylation of PRC2 target genes in colon cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aeb4732d6e0e4b7c91f83caa7be8cc3275c4570" target='_blank'>
              Abstract B003: BRAFV600E is essential for maintenance of the CpG island methylator phenotype and DNA methylation of PRC2 target genes in colon cancer
              </a>
            </td>
          <td>
            Layla El Bouazzaoui, J. Bugter, Emre Küçükköse, A. Verheem, J. Post, N. Fenderico, Inne H. M. Borel Rinkes, Hugo J. Snippert, M. Maurice, O. Kranenburg
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="BACKGROUND
α thalassemia/mental retardation syndrome X-linked (ATRX) serves as a part of the sucrose nonfermenting 2 (SNF2) chromatin-remodeling complex. In interphase, ATRX localizes to pericentromeric heterochromatin, contributing to DNA double-strand break repair, DNA replication, and telomere maintenance. During mitosis, most ATRX proteins are removed from chromosomal arms, leaving a pool near the centromere region in mammalian cells, which is critical for accurate chromosome congression and sister chromatid cohesion protection. However, the function and localization mechanisms of ATRX at mitotic centromeres remain largely unresolved.


METHODS
The clustered regularly interspaced short palindromic repeats with CRISPR-associated protein 9 (CRISPR-Cas9) system and overexpression approaches were employed alongside immunofluorescence to investigate the mechanism of ATRX localization at the centromere. To study the binding mechanism between ATRX and heterochromatin protein 1 (HP1), both full-length and truncated mutants of hemagglutinin (HA)-ATRX were generated for co-immunoprecipitation and glutathione S-transferase (GST)-pull assays. Wild-type ATRX and HP1 binding-deficient mutants were created to investigate the role of ATRX binding to HP1 during mitosis, with the Z-Leu-Leu-Leu-al (MG132) maintenance assay, cohesion function assay, and kinetochore distance measurement.


RESULTS AND CONCLUSIONS
Our research demonstrated that HP1α, HP1β, and HP1γ facilitate the positioning of ATRX within the mitotic centromere area through their interaction with the first two [P/L]-X-V-X-[M/L/V] (PxVxL)motifs at the N-terminus of ATRX. ATRX deficiency causes aberrant mitosis and decreased centromeric cohesion. Furthermore, reducing Wapl activity can bypass the need for ATRX to protect centromeric cohesion. These results provide insights into the mechanism of ATRX's centromeric localization and its critical function in preserving centromeric cohesion by reducing Wapl activity in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa2508ee19573d906286f1694bdd429742f1bd10" target='_blank'>
              HP1 Promotes the Centromeric Localization of ATRX and Protects Cohesion by Interfering Wapl Activity in Mitosis.
              </a>
            </td>
          <td>
            Erchen Zhang, Lei Peng, Kejia Yuan, Zexian Ding, Qi Yi
          </td>
          <td>2025-01-21</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Oncogenic gamma herpesviruses, including Epstein–Barr Virus (EBV) and Kaposi’s Sarcoma-associated Herpesvirus (KSHV), are opportunistic cancer-causing viruses and induces oncogenesis through complex mechanisms, which involves manipulation of cellular physiology as well as epigenetic and epitranscriptomic reprogramming. In this review, we describe the intricate processes by which these viruses interact with the epigenetic machinery, leading to alterations in DNA methylation, histone modifications, and the involvement of non-coding RNAs. The key viral proteins such as EBNA1 and LMP1 encoded by EBV; LANA and vGPCR encoded by KSHV; play pivotal roles in these modifications by interacting with host factors, and dysregulating signaling pathways. The resultant reprogramming can lead to activation of oncogenes, silencing of tumor suppressor genes, and evasion of the immune response, which ultimately contributes to the oncogenic potential of these viruses. Furthermore, in this review, we explore current therapeutic strategies targeting these epigenetic alterations and discuss future directions for research and treatment. Through this comprehensive examination of the epigenetic and epitranscriptomic reprogramming mechanisms employed by oncogenic gamma herpesviruses, we aim to provide valuable insights into potential avenues for novel therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d106bf10134bf9b8889820e2db155164b9012c0" target='_blank'>
              Epigenetic and epitranscriptomic regulation during oncogenic γ-herpesvirus infection
              </a>
            </td>
          <td>
            Rajnish Kumar Singh, Ramakrishna Vangala, Atharva S. Torne, Dipayan Bose, E. Robertson
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Pancreatic cancer (PC), known as the "king of cancers," is characterized by an exceptionally low five-year survival rate, posing a formidable challenge to global public health. N6-methyladenosine (m6A) methylation is prevalent across various stages of eukaryotic RNA expression, including splicing, maturation, stability, translation, and localization, and represents a pivotal mechanism of epigenetic regulation. m6A methylation influences tumor initiation and progression by modulating post-transcriptional processes, playing a critical role in sustaining cancer cell stemness, promoting cell proliferation, and mediating drug resistance. Extensive research underscores the substantial contribution of m6A modifications to PC development. However, the multiplicity of m6A regulators and their intricate mechanisms of action complicate the landscape. This review aims to deepen the understanding of m6A's role in PC by delineating its involvement in four key areas of tumorigenesis: the hypoxic tumor microenvironment, metabolic reprogramming, immune microenvironment, and resistance mechanisms. Additionally, the review addresses the emerging frontier of m6A interactions with non-coding RNAs (ncRNAs), offering insights into the potential therapeutic and prognostic applications of m6A in the treatment and prognosis prediction of PC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/323b08ca5467f445ec0f42c55b9835ad0ac4ba47" target='_blank'>
              The Mechanism and Latest Progress of m6A Methylation in the Progression of Pancreatic Cancer
              </a>
            </td>
          <td>
            Ze-Hao Liu, Peng Ma, Ying He, Yuejiao Zhang, Zuo Mou, Ting Fang, Wei Wang, Kaihuan Yu
          </td>
          <td>2025-01-13</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="DNA methylation is an epigenetic mechanism of gene silencing crucial to the regulation of gene expression in normal physiological events such as differentiation and X-inactivation. However, aberrant silencing of regulatory genes can contribute to oncogenic transformation, further perpetuated due to the heritability of these changes down the cell line. Though aberrant DNA methylation is implicated across cancer types, epigenetic therapy with DNA methyltransferase 1 (DNMT1) depleting drugs is only approved in the treatment of hematological cancers. This limitation is due to the drugs’ high toxicity, a byproduct of their mechanism as nucleoside analogs. Identifying less toxic nucleoside analogs or developing non-nucleoside analogs is necessary to expand the application of epigenetic therapy. This review examines the existing nucleoside and non-nucleoside DNMT1-inhibitors at various stages of preclinical and clinical development, and overviews prospective applications of epigenetic therapy in solid tumors as monotherapy and combined therapy. The list of drugs reported in this review is non-comprehensive as it excludes primary research on drugs last tested prior to the FDA approval of azacitidine and drugs not tested or inviable in human cells. This is to limit the survey to studies that intend to improve upon the pharmacological profile of the approved drugs. Of particular importance are the recently developed DNMT1-inhibitor (DNMT1-i) GSK analogs and the advancements in protein modeling that have elucidated their mechanism to the greatest precision yet. Not yet discussed in length in secondary literature, these findings provide a clearer model for the development of more specific DNMT-is. Furthermore, evidence showing enhanced efficacy of DNMT1 inhibitors when combined with DNA damaging agents identifies epigenetic combination therapy as a pertinent focus of future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f220480bdc96c0e4bd612e7cc2d97304ce081345" target='_blank'>
              Nucleoside and Non-Nucleoside DNA Methyltransferase 1 Inhibitors in Epigenetic and Combination Therapies in Cancer: A Scoping Review
              </a>
            </td>
          <td>
            Nila Mohan
          </td>
          <td>2025-01-31</td>
          <td>Undergraduate Research in Natural and Clinical Science and Technology (URNCST) Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Objectives: The highly polymorphic Major Histocompatibility Complex (MHC) genomic region, located on the short arm of chromosome 6, is implicated genetically in Parkinson’s disease (PD), a progressive neurodegenerative disorder with motor and non-motor symptoms. Previously, we reported significant associations between SINE-VNTR-Alu (SVA) expression quantitative trait loci (eQTLs) and Human Leucocyte Antigen (HLA) class I genotypes in PD. In this study, we aimed to evaluate SVA associations and their regulatory effects on transposable element (TE) transcription in the MHC class I region. Methods: Transcriptome data from the peripheral blood cells of 1530 individuals in the Parkinson’s Progression Markers Initiative (PPMI) cohort were reanalyzed for TE and gene expression using publicly available bioinformatics tools, including Salmon and Matrix-eQTL. Results: Four structurally polymorphic SVAs regulated the transcription of 18 distinct clusters of 235 TE loci, comprising LINEs (33%), SINEs (19%), LTRs (35%), and ancient transposon DNA elements (12%) located near HLA genes. The transcribed TEs were predominantly short, with an average length of 445 nucleotides. The regulatory effects of these SVAs varied significantly in terms of TE types, numbers, and transcriptional activation or repression. The SVA-regulated TE RNAs in blood cells appear to function as enhancer-like elements, differentially influencing the expression of HLA class I genes, non-HLA genes, and noncoding RNAs. Conclusions: These findings highlight the roles of SVAs and their associated TEs in the complex regulatory networks governing coding and noncoding gene expression in the MHC class I region, with potential implications for immune function and disease susceptibility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a7af8d419a6f31034c910704dd0806ace1362d2" target='_blank'>
              The Transcription of Transposable Elements Differentially Regulated by SVAs in the Major Histocompatibility Complex Class I Region of a Parkinson’s Progression Markers Initiative Cohort
              </a>
            </td>
          <td>
            Jerzy K. Kulski, A. Pfaff, S. Kõks
          </td>
          <td>2025-01-06</td>
          <td>Journal of Molecular Pathology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae55557a79f951a485decd6834444eb8baa9c9b2" target='_blank'>
              Recent updates of centromere proteins in hepatocellular carcinoma: a review
              </a>
            </td>
          <td>
            Zhongyuan Yang, Wenjiao Chen, Yunhui Liu, Yuxin Niu
          </td>
          <td>2025-02-06</td>
          <td>Infectious Agents and Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52a090853d43190aed49536f43b138c297c3e652" target='_blank'>
              Telomerase in cancer- ongoing quest and future discoveries.
              </a>
            </td>
          <td>
            Apurwa Mishra, Trupti N. Patel
          </td>
          <td>2025-01-25</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA topology is a direct consequence of the double helical nature of DNA and is defined by how the two complementary DNA strands are intertwined. Virtually every reaction involving DNA is influenced by DNA topology or has topological effects. It is therefore of fundamental importance to understand how this phenomenon is controlled in living cells. DNA topoisomerases are the key actors dedicated to the regulation of DNA topology in cells from all domains of life. While significant progress has been made in the last two decades in understanding how these enzymes operate in vivo in Bacteria and Eukaryotes, studies in Archaea have been lagging behind. This review article aims to summarize what is currently known about DNA topology regulation by DNA topoisomerases in main archaeal model organisms. These model archaea exhibit markedly different lifestyles, genome organization and topoisomerase content, thus highlighting the diversity and the complexity of DNA topology regulation mechanisms and their evolution in this domain of life. The recent development of functional genomic assays supported by next-generation sequencing now allows to delve deeper into this timely and exciting, yet still understudied topic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a4860c875d7d4484e2675f291d10098c1fde66" target='_blank'>
              Regulation of DNA Topology in Archaea: State of the Art and Perspectives.
              </a>
            </td>
          <td>
            Paul Villain, Tamara Basta
          </td>
          <td>2024-12-22</td>
          <td>Molecular microbiology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Variant histone H3.3 is thought to be critical for survival of many cells, since it is deposited in expressed genes, a feature different from core histones. For example, H3.3 deletion leads to embryonic lethality in mice. However, requirement of H3.3 in later stage of development has remained unclear. The aim of this work was to elucidate the role of H3.3 for development of myeloid lineage, important for innate immunity. We conditionally knocked out (cKO) the H3.3 genes in myeloid progenitor cells differentiating into bone marrow derived macrophages (BMDMs). Progenitor cells lacking H3.3 were defective in replication, suffered from extensive DNA damage, and underwent apoptosis. Surviving H3.3cKO cells expressed many interferon stimulated genes (ISGs) throughout differentiation. Further, H3.3cKO BMDMs possessed chromatin accessible sites, and histone posttranslational modifications consistent with the gene expression profiles, Accordingly, H3.3cKO BMDMs retained general nucleosomal structure genome wide. In summary, H3.3 is required for proliferation of myeloid progenitor cells, but is in large part dispensable for differentiation of BMDMs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee58f7554d51c36d858c992137d0b639e970461a" target='_blank'>
              Histone H3.3 ensures cell proliferation and genomic stability during myeloid cell development
              </a>
            </td>
          <td>
            Sakshi Chauhan, A. Dey, Todd S Macfarlan, Keiko Ozato
          </td>
          <td>2024-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c98077b8e7b73940125f0da65abe02bf159f00a4" target='_blank'>
              Multi-omics analyses reveal biological and clinical insights in recurrent stage I non-small cell lung cancer
              </a>
            </td>
          <td>
            Chengdi Wang, Jingwei Li, Jingyao Chen, Zhoufeng Wang, Guonian Zhu, Lujia Song, Jiayang Wu, Changshu Li, Rong Qiu, Xuelan Chen, Li Zhang, Weimin Li
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Recent studies have demonstrated that the production of bidirectional enhancer-derived transcripts (eRNAs) is a characteristic of an active Cis-regulatory element (CRE). Higher levels of eRNA synthesis correlate with the activation of histone modifications, a potentially valuable tool for deciphering the complexity of the gene regulatory network. To understand the changes of CREs during gonadal development in mice, we collected gonadal WT1-positive cells from the piggyBac-Wt1-mCherry-2A-EGFP (PBWt1-RG) reporter strain at E13.5, E16.5, and P0 in both sexes and conducted Cap Analysis of Gene Expression analysis (CAGE) which is capable to capture transcriptional starting site (TSS). We compared the levels of intergenic bidirectional RNA, i.e, potentially eRNA, according to sex at each stage (testis somatic cells vs ovary somatic cells at E13.5, E16.5, and P0) and stage in each sex (E13.5 vs E16.5, E16.5 vs P0, and E13.5 vs P0 in testis somatic cells or ovary somatic cells). Intergenic RNAs with significant changes (|Log2FC| > 1, p < 0.05) were selected. The TSS profile of intergenic RNA changed more profoundly in testis somatic cells than in ovary somatic cells, suggesting embryonic testicular development is driven by larger changes in a transcriptional regulatory network than ovarian development. Based on the profiles of the predicted transcription factors (TFs) that would bind to the active CREs during gonadal development, the NR4A, EGR, and TCF3 families would be novel TFs to play pivotal roles in gonadal development. Identifying active CREs using eRNAs would provide a means to comprehensively understand the transcriptional regulatory system, leading to valuable insights into the gonadal development of male and female individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8877e8a45402ec1d0fc04d7b02b6aa2118971232" target='_blank'>
              Analysis of Functional Cis-regulatory Elements Reveals Novel Transcriptional Regulatory Mechanisms in Gonadal Development.
              </a>
            </td>
          <td>
            S. Kirino, Ryuichi Nakagawa, Maki Gau, K. Takasawa, Yasuhiro Murakawa, H. Kawaji, Y. Hayashizaki, Tomohiro Morio, K. Kashimada
          </td>
          <td>2025-01-20</td>
          <td>Sexual development : genetics, molecular biology, evolution, endocrinology, embryology, and pathology of sex determination and differentiation</td>
          <td>0</td>
          <td>119</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/692b233441e797a73e14defb069a03a55d2ca324" target='_blank'>
              DNA methylation profiling at base-pair resolution reveals unique epigenetic features of early-onset colorectal cancer in underrepresented populations
              </a>
            </td>
          <td>
            J. S. Li, Karen Riggins, Li Yang, Chaorong Chen, Patricia Castro, Wedad Alfarkh, Neda Zarrin-Khameh, Michael E. Scheurer, Chad J. Creighton, Benjamin Musher, Wei Li, Lanlan Shen
          </td>
          <td>2025-01-22</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee69beabd8a047c43345e066d4bca544bc872682" target='_blank'>
              Satellite DNA shapes dictate pericentromere packaging in female meiosis.
              </a>
            </td>
          <td>
            Damian Dudka, Jennine M. Dawicki-McKenna, Xueqi Sun, Keagan Beeravolu, Takashi Akera, Micheal A. Lampson, Ben E. Black
          </td>
          <td>2025-01-08</td>
          <td>Nature</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="
 High-grade serous carcinoma (HGSC) remains the deadliest gynecological cancer, mostly due to the widespread metastases in abdominal cavities already at diagnosis. To improve therapy success, understanding metastasis routes and their molecular characteristics is crucial. Herein, we used >400 samples subjected to whole-genome bisulfite (WGBS) and RNA sequencing from 125 patients with HGSC, enrolled to prospective, longitudinal, multi-region, observational DECIDER trial (NCT04846933). These real-world epigenetics data allowed, for the first time, to delineate metastatic routes in HGSC. Our results show that cancer cells released from the site-of-origin sites seed to both mid-abdomen and upper abdomen in parallel. The dominant mechanism responsible for almost half of the spread was direct surface-to-surface release of ovarian tumor cells into the peritoneal cavity. The major source of cancer cells in the pelvic area was ovary, from which cancer disseminated to all intra-abdominal organs, including omentum. Tumor cells from pelvic bowel displayed strong tropism for virtually all metastatic deposits, including such distant metastases, as liver, vagina, and lungs, which suggested hematogenous spread. Ascites play a prominent role in fueling colonization, accounting for additional 20% of direct spread. The seeding via ascitic fluid remained localized within the abdominal area, however, ascites colonized primarily para-aortic and inguinal lymph nodes. Lymph node metastases were noted in one-third of possible spreading modes. HGSC dissemination via inguinal lymph nodes represented a rare, but aggressive trajectory, as these tumors, originated directly from tube were prone to invade all distant locations. Comparison of the paired samples before and after chemotherapy revealed that most sensitive to neoadjuvant chemotherapy (NACT) tumors are adnexal and omental tumors. Peritoneum and mesentery showed prominent resistance to treatment, as well as lymph nodes. Thus, patients with metastases in inguinal lymph nodes and bowel are at the highest risk of developing widespread intra-abdominal and distant metastases. Furthermore, the standard-of-care platinum/taxane therapy, along with maintenance relapse therapies are ineffective for eradication of these tumors. Ascites displayed an intriguing case: being virtually non-responding to NACT, epigenetic profiles at relapse exhibited a large change. This implies that epigenetic processes in ascites evolve along with the cancer progression, driving chemoresistance. Taken together, using DNA methylation changes from >400 samples we revealed metastasis routes in HGSC, as well as tissue-specific chemotherapy treatment response. Dissection of the key regulatory pathways of metastatic routes helps identify patients at the highest risk of distant metastases and suggests new targets for therapies.
 Citation Format: Alexandra Lahtinen, Giovanni Marchi, Kari Lavikka, Daria Afenteva, Veli-Matti Isoviita, Susanna Holmström, Yilin Li, Taru A Muranen, Jaana Oikkonen, Antti Häkkinen, Anni Virtanen, Johanna Hynninen, Sampsa Hautaniemi. DNA methylation reveals metastatic colonization routes in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr PR003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f704aa07c09b6c48c3c1f8d354bbb48ab6359478" target='_blank'>
              Abstract PR003: DNA methylation reveals metastatic colonization routes in ovarian cancer
              </a>
            </td>
          <td>
            A. Lahtinen, G. Marchi, K. Lavikka, D. Afenteva, Veli-Matti Isoviita, Susanna Holmström, Yilin Li, T. Muranen, J. Oikkonen, Antti Häkkinen, A. Virtanen, J. Hynninen, S. Hautaniemi
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7e408d8468ed99bd88ba4923953c9ea7ed47d" target='_blank'>
              Roles of human papillomavirus in cancers: oncogenic mechanisms and clinical use
              </a>
            </td>
          <td>
            Yu Zhang, Ke Qiu, Jian-jun Ren, Yu Zhao, Ping Cheng
          </td>
          <td>2025-01-24</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b99a1f2cb92c3a6d34aa29596c928af0010bece5" target='_blank'>
              Single-cell sequencing reveals the mechanisms of multiple myeloma progression: clarity or confusion?
              </a>
            </td>
          <td>
            Yunhui Xiang, G. Sun, Lvbo Tian, Pinpin Xiang, Chunbao Xie
          </td>
          <td>2025-02-07</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The three most prevalent malignant skin tumors are basal cell carcinoma (BCC), squamous cell carcinoma (SCC), and cutaneous melanoma (CM). While BCC and SCC generally exhibit low levels of malignancy, some tumors can become invasive and metastasize. In contrast, CM is the most malignant and lethal form of skin cancer among these three. In recent years, the increasing incidence of skin tumors has garnered significant attention within the medical community. The advent of single-cell RNA sequencing (scRNA-seq) technology has revolutionized the analysis of the tumor microenvironment (TME) in skin tumors, offering novel insights for clinical research. In this review, we provide a concise introduction to scRNA-seq technology and its application in delineating the heterogeneity of the TME in skin tumors, elucidating the mechanisms of tumor progression and pathogenicity, and uncovering new therapeutic targets. Our review offers a comprehensive overview for researchers, offering insights that may advance the understanding of the skin TME in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c806552e713787b83ea0933cd31a52d109ce5cf9" target='_blank'>
              Advances and applications of single-cell RNA sequencing in deciphering the tumor microenvironment of malignant skin tumors
              </a>
            </td>
          <td>
            Yiting Feng, Lanlan Liu, Chunlan Hu, Jie Sun, Mingzhu Yin, Xin Li
          </td>
          <td>2025-01-10</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39dc1a99977067c119e44f09575781707edcc62" target='_blank'>
              Interplay of aurora kinase a functional residues and Epstein-barr Nuclear Antigen 1 in Epstein-barr virus associated Gastric cancer using AGS cells
              </a>
            </td>
          <td>
            N. Varshney, Siddharth Singh, M. Kandpal, Vaishali Saini, Erle S. Roberston, Hem Chandra Jha
          </td>
          <td>2025-01-24</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT This study explored the genomic alterations in Yarrowia lipolytica, a key yeast in industrial biotechnology, under both spontaneous and mutagen-induced conditions. Our findings reveal that spontaneous mutations occur at a rate of approximately 4 × 10−10 events per base pair per cell division, primarily manifesting as single-nucleotide variations (SNVs) and small insertions and deletions (InDels). Notably, C-to-T/G-to-A transitions and C-to-A/G-to-T transversions dominate the spontaneous SNVs, while 1 bp deletions, likely resulting from template slippage, are the most frequent InDels. Furthermore, chromosomal aneuploidy and rearrangements occur, albeit at a lower frequency. Exposure to ultraviolet (UV) light, methylmethane sulfonate (MMS), and Zeocin significantly enhances the rates of SNVs and alters their mutational spectra in distinct patterns. Notably, Zeocin-induced SNVs are predominantly T-to-A and T-to-G substitutions, often occurring within the 5′-TGT*−3′ motif (* denotes the mutated base). Additionally, Zeocin exhibits a higher potency in stimulating InDels compared to UV and MMS. Translesion DNA synthesis is implicated as the primary mechanism behind most Zeocin-induced SNVs and some InDels, whereas non-homologous end joining serves as the main pathway for Zeocin-mediated InDels. Intriguingly, the study identifies the gene YALI1_E21053g, encoding a protein kinase, as negatively associated with Zeocin resistance. Overall, our results not only deepened our knowledge about the genome evolution in Y. lipolytica but also provided reference to develop innovative strategies to harness its genetic potential. IMPORTANCE Yarrowia lipolytica exhibits high environmental stress tolerance and lipid metabolism capabilities, making it a microorganism with significant industrial application potential. In this study, we investigated the genomic variation and evolutionary patterns of this yeast under both spontaneous and induced mutation conditions. Our results reveal distinctive mutation spectra induced by different mutagenic conditions and elucidate the underlying genetic mechanisms. We further highlight the roles of non-homologous end joining and translesion synthesis pathways in Zeocin-induced mutations, demonstrating that such treatments can rapidly confer drug resistance to the cells. Overall, our research enhances the understanding of how yeast genomes evolve under various conditions and provides guidance for developing more effective mutagenesis and breeding techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b4727fb2831092d46e2f4802e9e2fadf0e55a7" target='_blank'>
              Patterns of spontaneous and induced genomic alterations in Yarrowia lipolytica
              </a>
            </td>
          <td>
            Yuan-Ru Xiong, Yuan-Chun Fang, Min He, Keke Li, Lei Qi, Yang Sui, Ke Zhang, Xuechang Wu, Liang Meng, Ou Li, Dao-Qiong Zheng
          </td>
          <td>2024-12-23</td>
          <td>Applied and Environmental Microbiology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef7d2fbf8a9f9c82b98c7984b3782b85218b8319" target='_blank'>
              Crypt density and recruited enhancers underlie intestinal tumour initiation.
              </a>
            </td>
          <td>
            Liam T. Gaynor, Harshabad Singh, Guodong Tie, Krithika Badarinath, Shariq Madha, Andrew Mancini, Swarnabh Bhattacharya, Mikio Hoshino, Frederic J. de Sauvage, Kazutaka Murata, Unmesh Jadhav, Ramesh A. Shivdasani
          </td>
          <td>2025-01-08</td>
          <td>Nature</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background: In eukaryotes with a double-stranded linear DNA genome, the loss of terminal DNA during replication is inevitable due to an end-replication problem; here, telomeres serve as a buffer against DNA loss. Thus, the activation of the telomere maintenance mechanism (TMM) is a prerequisite for malignant transformation. Methods: We compared neurofibroma (NF, benign) and malignant peripheral nerve sheath tumors (MPNSTs) occurring in the same patient with type 1 neurofibromatosis, where each NF–MPNST pair shared the same genetic background and differentiation lineage; this minimizes the genetic bias and contrasts only those changes that are related to malignant transformation. A total of 20 NF–MPNST pairs from 20 NF1 patients were analyzed. Whole-transcriptome sequencing (WTS) was conducted to profile the transcriptional relationship, and whole-genome sequencing (WGS) was performed to measure the telomere length. Results: We identified 22 differentially expressed genes (DEGs) during the malignant transformation of MPNSTs. Among them, NELL2 activated PAX7, which sequentially activated RAD52, the recombinase of RAD52-dependent alternative lengthening of telomeres (ALT). RAD52 elongated MPNSTs–telomeres (p = 0.017). Otherwise, neither NELL2 nor PAX7 affected telomere length (p = 0.647 and p = 0.354, respectively). RAD52 increased MPNSTs–telomeres length, independently of NELL2 and PAX7 in multiple analyses (p = 0.021). The group with increased telomere length during the malignant transformation showed inferior overall survival (OS) (HR = 3.809, p = 0.038) to the group without increased telomere length. Accordingly, the group with increased PAX7 showed inferior OS (HR = 4.896, p = 0.046) and metastasis-free survival (MFS) (HR = 9.129, p = 0.007) in comparison to the group without increased PAX7; the group with increased RAD52 showed inferior MFS (HR = 8.669, p = 0.011) in comparison to the group without increased RAD52. Conclusions: We suggest that the NELL2-PAX7 transcriptional cascade activates RAD52-dependent ALT to increase telomere length during the malignant transformation of MPNSTs, resulting in a poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec682f73e8b37948b58b7e542b7ea83e47ad44c3" target='_blank'>
              NELL2-PAX7 Transcriptional Cascade Suggests Activation Mechanism for RAD52-Dependent Alternative Lengthening of Telomeres During Malignant Transformation of Malignant Peripheral Nerve Sheath Tumors: Elongation of Telomeres and Poor Survival
              </a>
            </td>
          <td>
            Jungwoo Lee, Eunji Choi, Hyoju Kim, Young-Joon Kim, Seung Hyun Kim
          </td>
          <td>2025-01-23</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 DNA methylation (DNAm) reflects regions of the genome that are transcriptionally repressed or active depending on the methylation level. Multitudes of DNAm “clocks” have been created to predict biological age and health outcomes. Deviations in biological age from chronological age reflect an alteration in the normal biological aging process. Clonal hematopoiesis of indeterminate potential (CHIP) arises in hematopoietic stem cells that have accumulated somatic mutations in key driver genes which provide a selective advantage for clonal expansion. Subjects with CHIP mutations are more prone to negative clinical outcomes such as coronary heart disease. While DNAm and CHIP have been studied for the past decade, the associations between the two are only now starting to become apparent. In this study we propose an improved measurement of aging based on the Jenson-Shannon distance (JSD) and associate this measurement with CHIP and adverse clinical outcomes. Bisulfite amplicon sequencing of 20 CpG-rich targets that change in methylation with age was performed on an initial cohort of 155 individuals from the National Institute of Neurological Disorders and Stroke (NINDS). JSD was used to construct a DNA methylation chaos clock (DMC) on these targets. DMC provided better predictions of chronological age than DNAm alone (r=0.9; p<0.0001). We validated our findings in 286 individuals from Cooper University Hospital for Management of Trauma where the accuracy of the DMC was confirmed in correlations of predicted vs actual age (r=0.86; p<0.0001). We next examined the association between DMC age acceleration and clinical outcomes. It was found that in individuals over the age of 50, DMC acceleration was associated with smoking and disease incidence (e.g. congestive heart failure and chronic obstructive pulmonary disease) independent of chronological age. The incidence of CHIP mutations in three key driver genes (DNMT3A, TET2, and ASXL1) was assayed in a subset of 345 individuals from both cohorts. Individuals with CHIP mutations in key driver genes had greater deviations in their predicted age than individuals without CHIP mutations (p<0.001). This study demonstrates that the DMC assay is a candidate for improvements in our measurements of biological age and it reflects important changes in an individual’s biological aging process.
 Citation Format: Anthony J Pompetti, Woonbok Chung, Jaroslav Jelinek, Tanya Egodage, Jean-Pierre Issa. DNA methylation variability in aging and Clonal Hematopoiesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A016.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aadd86e514d184ff56ec546d04b1399f1bbf484d" target='_blank'>
              Abstract A016: DNA methylation variability in aging and Clonal Hematopoiesis
              </a>
            </td>
          <td>
            Anthony J Pompetti, W. Chung, Jaroslav Jelinek, Tanya Egodage, Jean-Pierre J Issa
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9029d65e0d9d3415e021015d3185275320222a73" target='_blank'>
              G-quadruplex stabilization provokes DNA breaks in human PKD1, revealing a second hit mechanism for ADPKD
              </a>
            </td>
          <td>
            Agata M Parsons, Seth Byrne, Jesse Kooistra, John Dewey, Aaron L Zebolsky, Gloria Alvarado, Gerrit J Bouma, Gregory B Vanden Heuvel, Erik D Larson
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Repairing DNA double-strand breaks is crucial for maintaining genome integrity, which occurs primarily through homologous recombination (HR) in Saccharomyces cerevisiae. Nucleosomes, composed of DNA wrapped around a histone octamer, present a natural barrier to end resection to initiate HR, but the impact on the downstream HR steps of homology search, DNA strand invasion, and repair synthesis remain to be determined. Displacement loops (D-loops) play a pivotal role in HR, yet the influence of chromatin dynamics on D-loop metabolism remains unclear. Using the physical D-loop capture and D-loop extension (DLE) assays to track HR intermediates, we employed genetic analysis to reveal that H2B mono-ubiquitylation (H2Bubi) affects multiple steps during HR repair. We infer that H2Bubi modulates chromatin structure, not only promoting histone degradation for nascent D-loop formation but also stabilizing extended D-loops through nucleosome assembly. Furthermore, H2Bubi regulates DNA resection via Rad9 recruitment to suppress a feedback control mechanism that dampens D-loop formation and DLE at hyper-resected ends. Through physical and genetic assays to determine repair outcomes, we demonstrate that H2Bubi plays a crucial role in preventing break-induced replication and thus promoting genomic stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de8118b6033b5fe8f381d1e71c77a4e0bae2aca0" target='_blank'>
              Multifaceted roles of H2B mono-ubiquitylation in D-loop metabolism during homologous recombination repair
              </a>
            </td>
          <td>
            Shih-Hsun Hung, Yuan Liang, W. Heyer
          </td>
          <td>2025-02-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="
 239


 Background:
 Inactivating genomic alteration (GA) of the TP53 tumor suppressor gene leads to inactivation of p53 protein and is currently the most frequently associated GA across all cancers. Colorectal cancer is a common and lethal subtype of cancer, in the top 5 for incidence, prevalence and cancer-related deaths worldwide. TP53 is the second most frequent GA identified in clinically advanced CRC. It has been postulated that GA in TP53 not only affects the molecular biology of malignant cells but also impacts the tumor microenvironment in CRC. This GA is currently untargetable by approved drugs however, recently a novel approach designed to revert to wild type functioning of TP53 inactivated by Y220C base substitution has gained significant clinical interest.
 Methods:
 67,301 cases of clinically advanced CRC underwent hybrid capture based comprehensive genomic profiling to assess all classes of GA. Cases were sequenced to a mean coverage depth of 650X with microsatellite instability (MSI) status and tumor mutation burden (TMB) determined from the sequencing data and PD-L1 expression measured by immunohistochemistry using the DAKO tumor proportional staining system (low positive set at 1-49% staining and high positive set at ≥50% staining).
 Results:
 422 (0.6%) of the CRC featured the
 TP53
 Y220C GA (Y220C+). When compared with the CRC that were
 TP53
 Y220C negative (Y220C-), the Y220C+ cases were of similar age, gender and number of GA per tumor. The Y220C- cases has significantly higher frequencies of MSI status (6.1% vs 2.2%; p=.0007) and TMB levels greater >10 mutations/Mb (8.7% vs 4.7%; p=.004). The Y220C+ featured lower frequencies of GA in
 PIK3CA
 (12.9% vs 18.6%; p=.002),
 BRAF
 (8.8% vs 10.1%; NS) and
 PTEN
 (7.4% vs 8.3%, NS) and a higher frequency of
 ERBB2
 amplification (5.0% vs 2.7%; p=.004). GA in
 APC
 ,
 KRAS
 ,
 NRAS
 ,
 BRCA2
 ,
 EGFR
 and
 MET
 were similar in both Y220C+ and Y220C- CRC. PD-L1 low expression was similarly infrequent (12.1% to 16.7% range) in both groups.
 Conclusions:
 The potentially targetable
 TP53
 Y220C short variant mutation is uncommon in clinically advanced CRC but is associated with unique genomic and biomarker characteristics. Further research might elucidate subpopulations or CRC subtypes with a greater frequency of this mutation who might benefit from a novel targeted therapy.






 CRC
 TP53
 Y220C+(422 Cases)


 CRC
 TP53
 Y220C-(66,879 Cases)

 P Value




 Gender
 41.2% F/ 58.8% M
 44.4% F/55.6% M
 NS


 Median Age (range)
 62 (23-89+)
 61 (8-89+)
 NS


 GA/tumor
 6.1
 6.5
 NS



 TP53
 non-Y220C GA

 16.1%
 75.7%
 <.0001



 PIK3CA

 12.9%
 18.6%
 .002



 ERBB2

 6.2% (5.0% amp)
 5.1% (2.7% amp)
 .004 (amp)


 MSI High
 2.2%
 6.1%
 .0007


 TMB > 10 mut/Mb
 4.7%
 8.7%
 .004


 PD-L1 Low (1-49%TPS)/ PD-L1 High (>50% TPS)
 16.7% (132 cases)/2.3%
 12.1% (20,759 cases)/1.4%
 NS




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99ba4427e7070f06aa4eaa87971edb3bcc24e315" target='_blank'>
              TP53
 Y220C mutations in colorectal carcinoma (CRC): A genomic landscape study.
              </a>
            </td>
          <td>
            Deevyashali Parekh, J. Ross, Nimisha Srivastava, Tamara Jamaspishvili, A. Basnet
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Immune checkpoint inhibitor (ICI) therapy is a promising anti-tumor therapeutic strategy; however, its efficacy in solid tumors is limited. Chromosome missegregation is common in various solid tumors; however, its role in tumor progression remains poorly understood, and its correlation with ICI is yet to be explored. Here, it is found that increased chromosome missegregation promotes tumor immune microenvironment, and eventually immunotherapeutic efficacy, by triggering pyroptosis. yin yang 2 (YY2) is identified as a mitotic checkpoint regulator, which promotes chromosome missegregation by upregulating BUB1B transcription. Increased chromosome missegregation promoted the formation of micronuclei and release of double-stranded DNA (dsDNA) into the cytosol, triggering an AIM2-mediated cytosolic dsDNA response. The subsequent pyroptosis strengthened the tumor immune microenvironment, thereby enhancing immunoinfiltration and cytotoxicity of CD8+ T cells, while preventing their exhaustion. Finally, through in vitro and in vivo experiments, it is demonstrated that combining YY2 overexpression-induced chromosome missegregation/cytosolic dsDNA response and PD-1 inhibitor significantly enhanced the efficacy of ICI immunotherapy in microsatellite instable and microsatellite stable colorectal cancer cells. Together, these findings provide new insights on the role of chromosome missegregation in triggering cytosolic dsDNA response-mediated pyroptosis and modulating the tumor immune microenvironment, suggesting a novel strategy for improving ICI therapeutic efficacy in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/210fffb6c8ca89fde7413dc4cb1951b37eabd3d0" target='_blank'>
              Chromosome Missegregation Triggers Tumor Cell Pyroptosis and Enhances Anti-Tumor Immunotherapy in Colorectal Cancer.
              </a>
            </td>
          <td>
            Wei Duan, R. Hosea, Lingxian Wang, Cao Ruan, Fuqiang Zhao, Jingyi Liu, Hezhao Zhao, Makoto Miyagishi, Shourong Wu, Vivi Kasim
          </td>
          <td>2025-02-04</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most aggressive form of primary brain tumor. The infiltrative nature of GBM makes complete surgical resection impossible. The selective forces that govern gliomagenesis are strong, shaping the composition of tumor cells during the initial progression to malignancy with late consequences for invasiveness and therapy response. Here, we developed a mathematical model that incorporates ploidy level and the nature of the brain tissue microenvironment to simulate the growth and invasion of GBM and used the model to make inferences about GBM initiation and response to standard-of-care treatment. The spatial distribution of resource access in the brain was approximated through integration of in silico modeling, multi-omics data, and image analysis of primary and recurrent GBM. The in silico results suggested that high ploidy cells transition faster from oxidative phosphorylation to glycolysis than low ploidy cells because they are more sensitive to hypoxia. Between surgeries, simulated tumors with different ploidy compositions progressed at different rates; however, whether higher ploidy predicted fast recurrence was a function of the brain microenvironment. Historical data supported the dependence on available resources in the brain, as shown by a significant correlation between the median oxygen levels in human tissues and the median ploidy of cancers that arise in the respective tissues. Taken together, these findings suggest that availability of metabolic substrates in the brain drives different cell fate decisions for cells with different ploidy, thereby modulating both gliomagenesis and GBM recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e110685415835cdd67555abc6bde1dc3e70baed" target='_blank'>
              Interactions Between Ploidy and Resource Availability Shape Clonal Evolution in Glioblastoma.
              </a>
            </td>
          <td>
            Z. Nowicka, Frederika Rentzeperis, Vural Tagal, J. Teer, Didem Ilter, Richard J Beck, Jackson P Cole, Ana M Forero Pinto, Joanne D Tejero, Elisa Scanu, Thomas Veith, William Dominguez-Viqueira, Konstantin Maksin, Francisco Carrillo-Perez, Olivier Gevaert, Xiaonan Xu, F. Karreth, Mahmoud A Abdalah, Giada Fiandaca, Stefano Pasetto, Sandhya Prabhakaran, Andrew Schultz, Awino M. E. Ojwang', J. Barnholtz-Sloan, Joaquim M Farinhas, Ana P Gomes, Parag Katira, Noemi Andor
          </td>
          <td>2025-02-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="In eukaryotes, chromatin is compacted within nuclei under the principle of compartmentalization. On top of that, condensin II establishes eukaryotic chromosome territories, while cohesin organizes the vertebrate genome by extruding chromatin loops and forming topologically associating domains (TADs). Thus far, the formation and roles of these chromatin structures in plants remain poorly understood. This study integrates Hi-C data from diverse plant species, demonstrating that nuclear DNA content influences large-scale chromosome conformation and affects the finer details of compartmental patterns. These contrasting compartmental patterns are distinguished by gene-to-gene loops and validated through cytological observations. Additionally, a novel chromatin domain type associated with tandem duplicate gene clusters is identified. These domains are independent of H3K27me3-mediated chromatin compartmentalization and exhibit evolutionary conservation across species. Gene pairs within TAD-like domains are younger and show higher levels of coexpression. These domains potentially promote the formation of tandem duplicates, a property appears unique to the Actinidia family. Overall, this study reveals functional chromatin domains in plants and provides evidence for the role of three-dimensional chromatin architecture in gene regulation and genome evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f1f811c2a347e9ec96c4d8e9fe6d632b6cffed1" target='_blank'>
              Chromatin Topological Domains Associate With the Rapid Formation of Tandem Duplicates in Plants.
              </a>
            </td>
          <td>
            Ni Ma, Xiaopeng Li, Dong Ci, Haiyue Zeng, Congxiao Zhang, Xiaodong Xie, Caihong Zhong, Xing Wang Deng, Dawei Li, Hang He
          </td>
          <td>2024-12-27</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c103a4b54230f99fc3c742a3396a3568699c7f8" target='_blank'>
              H3K56 acetylation regulates chromatin maturation following DNA replication
              </a>
            </td>
          <td>
            Shoufu Duan, I. Nodelman, Hui Zhou, T. Tsukiyama, G. Bowman, Zhiguo Zhang
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Radiotherapy is an integral component in the treatment of many types of cancer, with approximately half of cancer patients receiving radiotherapy. Systemic therapy applies pressure that can select for resistant tumor subpopulations, underscoring the importance of understanding how radiation impacts tumor evolution to improve treatment outcomes. We integrated temporal genomic profiling of 120 spatially distinct tumor regions from 20 patients with undifferentiated pleomorphic sarcomas (UPS), longitudinal circulating tumor DNA (ctDNA) analysis, and evolutionary biology computational pipelines to study UPS evolution during tumorigenesis and in response to radiotherapy. Most unirradiated UPS displayed initial linear evolution followed by subsequent branching evolution with distinct mutational processes during early and late development. Metrics of genetic divergence between regions provided evidence of strong selection pressures during UPS development that further increased during radiotherapy. Subclone abundance changed following radiotherapy with subclone contraction tied to alterations in calcium signaling, and inhibiting calcium transporters radiosensitized sarcoma cells. Finally, ctDNA analysis accurately measured subclone abundance and enabled non-invasive monitoring of subclonal changes. These results demonstrate that radiation exerts selective pressures on UPS and suggest that targeting radioresistant subclonal populations could improve outcomes after radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bdc94aca0e727c5cc3864519a9f72d72e8fbb00" target='_blank'>
              Evolutionary Pressures Shape Undifferentiated Pleomorphic Sarcoma Development and Radiotherapy Response.
              </a>
            </td>
          <td>
            Erik Blomain, Shaghayegh Soudi, Ziwei Wang, Anish Somani, Ajay Subramanian, Serey C L Nouth, E. Oladipo, Chistin New, Deborah E Kenney, Neda Nemat-Gorgani, Thomas Kindler, R. Avedian, Robert J Steffner, D. Mohler, S. Hiniker, Alexander L. Chin, A. Kalbasi, M. Binkley, Marius Fried, Matthias M Gaida, Matt van de Rijn, E. J. Moding
          </td>
          <td>2025-01-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a highly heterogeneous gastrointestinal malignancy. Despite significant advances in molecular targeted therapies for CRC in recent years, the increase in the overall survival rates for CRC patients remains limited. Therefore, there is an urgent need to explore novel drug targets. Herein, we show that heparin binding growth factor (HDGF) is highly expressed in CRC, and that its overexpression is associated with a poor disease-free interval. Additionally, we reveal that HDGF knockout reduces proliferation, migration, and invasion, while enhancing apoptosis in CRC cells, thereby validating HDGF as a potential therapeutic target for CRC. Mechanistically, we found that HDGF modulates DNA damage response and, by recruiting C-terminal binding protein-interacting protein (CtIP), it facilitates homologous recombination repair to influence CRC drug sensitivity. Furthermore, we propose that HDGF may serve as a recognition protein for H3K36me3, participating in the repair of damaged transcriptionally active genes, thus maintaining genomic stability in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f394455983264a89c22ede32239ab6721086ed77" target='_blank'>
              HDGF Knockout Suppresses Colorectal Cancer Progression and Drug Resistance by Modulating the DNA Damage Response
              </a>
            </td>
          <td>
            Riya Su, Qin Wang, Qun Hu, Wendurige, Kexin Li, Changshan Wang, Liang Tao
          </td>
          <td>2025-02-14</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 RNaseH2 plays a critical role in removing mis-bounded RNA from the DNA double strand helix1. Absence of RNaseH2 results in spontaneous double-stranded DNA breaks and cytosolic release of dsDNA2. We have previously shown that epithelial deficiency of RNaseH2b (H2bΔIEC) leads to a p53-dependent stem cell suppression and additional deficiency of p53 results in spontaneous intestinal carcinogenesis3. Due to the central role of cGAS/STING in sensing dsDNA, we aim to investigate the role of epithelial STING in a mouse model of chronic epithelial DNA damage and tumor prevention in this study.



 DNA damage was assessed using comet assay in ModeK cell line with cGAS and STING knock out. Protein kinase activity were assessed by Pamgene Kinase Assay. Intestinal organoids were generated from H2bΔIEC and H2bΔIEC/Tmem173-/-mice. Colony forming assay and EdU staining were performed in all genotypes to assess the impact of STING on epithelial stem cell function and proliferation were assessed by CellTiter Glo assay.
 RNA sequencing was performed in intestinal organoids. Spontaneous carcinogenesis was assessed in 1-year aged mice.



 Loss of RNaseH2b results in activation of cGAS/STING pathway and subsequent upregulation of IFN-I signature. Deficiency of cGAS/STING lead to impaired IFN-I independent DNA damage repair which were accessed by WB and comet assay. During the DNA damage response, STING but not cGAS mediates the P53 pathway and the P21-CDK axis. Organoid EdU stanning displayed restored proliferation in H2bΔIEC/Tmem173-/- compared to H2bΔIEC. RNA sequencing from H2bΔIEC/Tmem173-/-organoid further confirmed decreased G2/M DNA damage checkpoint and P53 signature compared to H2bΔIEC. In vivo, we show that H2bΔIEC/Tmem173-/- mice have downregulated DNA damage repair and developed spontaneous small bowel adenocarcinoma after 53 weeks. Tumor derived from H2bΔIEC/Tmem173-/- mice show sensitivity to CDK inhibitor consistent with in vitro validation.



 We provide evidence that non-canonical role of epithelial STING mediates DNA damage response and guides P21-CDK1 axis in preventing tumor genesis. Our research provided evidence understand the role of STING in DNA damage response and intestine carcinogenesis.



 1.Reijns, Martin A.M., et al., Enzymatic Removal of Ribonucleotides from DNA Is Essential for Mammalian Genome Integrity and Development. Cell, 2012. 149(5): p. 1008-1022.
 2.Bartsch, K., et al., Absence of RNase H2 triggers generation of immunogenic micronuclei removed by autophagy. Hum Mol Genet, 2017. 26(20): p. 3960-3972.
 3.Aden, K., et al., Epithelial RNase H2 Maintains Genome Integrity and Prevents Intestinal Tumorigenesis in Mice. Gastroenterology, 2019. 156(1): p. 145-159 e19.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0539a26d1a5ba9dc2728ef562232f66e1b1c39" target='_blank'>
              P0212 Non-canonical STING signaling in intestinal epithelial cells guides DNA damage-driven stem cell function and protects from intestinal carcinogenesis
              </a>
            </td>
          <td>
            H. Xiang, C. Woitaske-Proske, J. Bernardes, G. R. Martini, F. Wottawa, J. Bornhaeuser, N. Kakavand, Y. Guang, Q. Wu, J. Kugler, L. Welz, B. Konukiewitz, C. Peifer, S. Schreiber, M. Tschurtschenthaler, P. Rosenstiel, K. Aden
          </td>
          <td>2025-01-01</td>
          <td>Journal of Crohn's and Colitis</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma (GBM) tumors exhibit extensive genomic, epigenomic, and transcriptional diversity, with significant intratumoral heterogeneity, complicating standard treatment approaches involving radiation (RT) and the DNA-alkylating agent temozolomide (TMZ). In this study, we employed an integrative multi-omics approach, including targeted proteomics, transcriptomics, genomics, and DNA methylation profiling, to investigate the response of a representative panel of GBM patient-derived cancer stem cells (CSCs) to astrocytic differentiation and RT and TMZ treatments. Differentiated CSC progenies retained the expression of key stemness genes and survival pathways, while activating the BMP-Smad signaling pathway and upregulating extracellular matrix components. This was associated with increased resistance to TMZ, though not to RT, across all models. We identified TP53 status as a critical determinant of transcriptional response to both RT and TMZ, which was also modulated by the differentiation state and treatment modality in wildtype (wt) p53 GBM cells. Both mutant and wt p53 models exhibited significant activation of the DNA-damage associated interferon (IFN) response in CSCs and differentiated cells, implicating this pathway in the GBM response to therapy. We observed that activation of NF-κB was positively correlated with the levels of O-6-methylguanine-DNA methyltransferase (MGMT) protein, a direct DNA repair enzyme leading to TMZ resistance, regardless of MGMT promoter methylation status, further supporting the clinical potential for inhibition of NF-kB signaling in GBM treatment. Our integrative analysis of the impact of GBM cell developmental states, in the context of genomic and molecular diversity of patient-derived models, provides valuable insights for pre-clinical studies aimed at optimizing treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a242e45dbcf0e04c69a3eb415e835a6082edb4c" target='_blank'>
              Impact of developmental state, p53 status, and interferon signaling on glioblastoma cell response to radiation and temozolomide treatment
              </a>
            </td>
          <td>
            A. Berezovsky, O. Nuga, Indrani Datta, Kimberly Bergman, Thais S Sabedot, Katherine Gurdziel, S. Irtenkauf, L. Hasselbach, Yuling Meng, Claudius Mueller, Emanuel F. Petricoin, Stephen Brown, N. Purandare, Sidhesh Aras, Tom Mikkelsen, Laila M Poisson, H. Noushmehr, Douglas Ruden, A. deCarvalho
          </td>
          <td>2025-02-07</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Background: Tubulin alpha 1b (TUBA1B) is a key microtubule protein essential for maintaining cellular structure and function. This protein contributes significantly to cytoskeletal formation and is implicated in various diseases. Despite its fundamental roles, TUBA1B’s impact on tumor prognosis and the tumor immune microenvironment across cancer types remains inadequately understood.Methods To elucidate TUBA1B’s role in cancer prognosis and immune response, we conducted a comprehensive analysis, integrating data from established databases such as The Cancer Genome Atlas, Genotype Tissue Expression, Cancer Cell Lineage Encyclopedia, Human Protein Atlas, Kaplan-Meier Plotter, cBioPortal, TIMER, and ImmuCellAI, along with a large-scale clinical study and immunotherapy cohort. We also conducted in vitro functional assays to assess TUBA1B’s functional role in tumor cells, allowing for a detailed examination of its relationship with cancer prognosis and immune modulation.Results: Our findings indicate that TUBA1B expression is dysregulated across multiple cancers, correlating strongly with poor survival outcomes and advanced pathological stages. Functional enrichment analyses further revealed that TUBA1B regulates key cell cycle processes, driving tumor proliferation, migration, and invasion. It also influences immune functions within both the innate and adaptive immune systems, affecting immune-related signaling pathways. These insights underscore TUBA1B’s multifaceted role in cancer progression and immune response.Conclusion: This study highlights TUBA1B’s potential as a human oncogene with substantial roles in tumorigenesis and immune regulation. Elevated TUBA1B levels are associated with an immunosuppressive tumor microenvironment, impacting cancer progression and treatment outcomes. Targeting TUBA1B may offer promising therapeutic avenues for enhancing cancer treatment, offering new perspectives for innovative anti-tumor strategies with high clinical impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4260f12a89430b00b11790e7c615000c70db0a6e" target='_blank'>
              TUBA1B as a novel prognostic biomarker correlated with immunosuppressive tumor microenvironment and immunotherapy response
              </a>
            </td>
          <td>
            Juntao Qi, Mingming Zhou, Na Yang, Huiyun Ma, Min He, Gujie Wu, Chang Ge, Liuyin Jin, Lin Cheng, Wei Liao, Hefei Ren, Caiyun Lei
          </td>
          <td>2025-02-04</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="In the M-phase, the nuclear membrane is broken down, nucleosomes are condensed as mitotic chromosomes, and transcription factors are generally known to be dislocated from their recognition sequences and dispersed to the cytoplasm. However, some transcription factors have recently been reported to remain on mitotic chromosomes and facilitate the rapid re-activation of the target genes in early G1-phase. Paired-like homeobox 2B (PHOX2B) is a transcription factor exhibiting chromosomal localization during M-phase. PHOX2B mutations are associated with congenital central hypoventilation syndrome, Hirschsprung disease, and neuroblastoma. In this study, we investigated PHOX2B chromosomal localization during M-phase through immunostaining and fluorescence recovery after photobleaching analysis to determine whether the chromosomal localization of disease-associated PHOX2B mutants is altered during M-phase. Missense mutations in the homeodomain and the frameshift mutation in the C-terminal domain disrupted the chromosomal localization of PHOX2B in M-phase, leading to its dispersion in the cell. Furthermore, a PHOX2B mutant with polyalanine expansion showed a line-shaped localization to the restricted region of mitotic chromosomes. Our findings suggest an association between the disease-associated mutations and defective chromosomal localization of transcription factors during M-phase. Further investigations of PHOX2B chromosomal localization during M-phase could reveal pathogenic mechanisms of such diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc280a650e26ad4fbfbcabfa8901f734f1fa074d" target='_blank'>
              Chromosomal localization of PHOX2B during M-phase is disrupted in disease-associated mutants.
              </a>
            </td>
          <td>
            Yuki Sato, Shin-ichiro Hayashi, Souichi Oe, T. Koike, Y. Nakano, Ryohei Seki-Omura, Hikaru Iwashita, Y. Hirahara, Masaaki Kitada
          </td>
          <td>2025-02-11</td>
          <td>Development, growth & differentiation</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a819a83329e7d0a414c5918852d0411278218a5" target='_blank'>
              High CDC20 levels increase sensitivity of cancer cells to MPS1 inhibitors.
              </a>
            </td>
          <td>
            Siqi Zheng, Linoy Raz, Lin Zhou, Yael Cohen-Sharir, Ruifang Tian, M. Ippolito, Sara Gianotti, Ron Saad, René Wardenaar, Mathilde Broekhuis, Maria F Suarez Peredo Rodriguez, Soraya Wobben, A. van den Brink, Petra Bakker, S. Santaguida, F. Foijer, Uri Ben-David
          </td>
          <td>2025-01-21</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Cytosine (DNA) methylation plays important roles in silencing transposable elements, plant development, genomic imprinting, stress responses, and maintenance of genome stability. To better understand the functions of this epigenetic modification, several tools have been developed to manipulate DNA methylation levels. These include mutants of DNA methylation writers and readers, targeted manipulation of locus-specific methylation, and the use of chemical inhibitors. Here, we summarize the effects of commonly used cytidine analog chemical inhibitors represented by zebularine, 5-azacytidine, and their related compounds on plants. These analogs are incorporated into the chromosomal DNA, where they block the activity of the replicative CG DNA methyltransferase 1 (MET1). This leads to manifold alterations in plant epigenome, modified developmental programs, or suppression of hybridization barriers. We also highlight the DNA-damaging effects of cytidine analogs, particularly the formation of stable DNA-protein crosslinks between DNA and MET1. This sheds new light on specific phenotypes observed upon cytidine analog treatments. In conclusion, cytidine analogs remain a vital tool for plant genome research and have the potential to open new promising avenues for applications in plant biotechnology and breeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/347eb5947bdb94f55760bce2a406c75b6df28aaf" target='_blank'>
              Cytidine analogs in plant epigenetic research and beyond.
              </a>
            </td>
          <td>
            Eva Dvořák Tomaštíková, A. Pečinka
          </td>
          <td>2024-12-28</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background: Epigenetic dysregulation is a common feature of cancer. Promoter demethylation of tumor-promoting genes and global DNA hypomethylation may trigger tumor progression. Epigenetic changes are unstable; thus, research has focused on detecting remedies that target epigenetic regulators. Previous studies have suggested that concordantly targeting hypomethylation and hypermethylation is beneficial for suppressing both the oncogenic and pro-metastatic functions of cancer cells. Therefore, we aimed to investigate the effect of a combination of S-adenosylmethionine (SAM) and the demethylating agent decitabine on prostate cancer cells. Materials and Methods: Prostate cancer cells (PC-3) were treated with SAM, decitabine, or a combination of both. Proliferation, migration, invasion, and methylation assays were also performed. A transcriptome study was conducted to detect different gene clusters between the treatment groups, followed by analyses using the Kyoto Encyclopedia of Genes and Genomes pathway and ingenuity pathway analysis. Finally, to gain information on differential gene expression, promoter methylation studies were performed. Results: Groups treated with decitabine, SAM, or their combination showed reduced proliferative capacity. The decitabine-treated group showed a marginal increase in cell migration and invasion, whereas the SAM-treated and combination treatment groups showed reduced invasion and migration potential. Methylation assays demonstrated the restoration of decitabine-induced demethylation in prostate cancer samples, whereas the transcriptome study revealed the upregulation of different gene clusters between the treatment groups. Methylation studies confirmed that SAM could restore the decitabine-induced demethylation of proto-oncogenes, but it did not induce the re-methylation of tumor-suppressor genes. Conclusions: Combination treatment with SAM and decitabine had an additive effect and did not nullify each other.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60ece44a8b2382c13d866bb1b93e63772b3f622f" target='_blank'>
              The Simultaneous Treatment of PC-3 Cells with the DNA-Demethylating Agent Decitabine and S-Adenosylmethionine Leads to Synergistic Anticancer Effects
              </a>
            </td>
          <td>
            Thomas Schmidt, Carsten Sticht
          </td>
          <td>2024-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c43b2e6ebcea9dd9d569a152511cd9284977827" target='_blank'>
              Establishment and characterization of ovarian clear cell carcinoma patient-derived xenografts
              </a>
            </td>
          <td>
            Joseph J. Caumanns, Shang Li, G. Meersma, E. Duiker, A. van der Zee, G. Wisman, Steven de Jong
          </td>
          <td>2025-02-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="
 Aging is a process of systemic deterioration and the most significant risk factor for cancers. Clonal hematopoiesis (CH) commonly occurs with aging and links to higher mortality, leukemia risk, and cardiovascular diseases. Age-related CH involves the abnormal clonal expansion of hematopoietic stem cells (HSCs) bearing somatic mutations in genes frequently mutated in leukemia, including genes encoding epigenetic regulators, e.g., the DNA demethylase TET2. While such mutations are known to alter the HSC epigenome, the mechanisms through which these mutations drive HSC self-renewal and myeloid transformation preferentially in old age remain elusive. In this study, we hypothesize that aging and CH mutations cooperatively reshape the HSC transcriptomic landscape, enhancing HSC competitive advantage that facilitates clonal expansion. Using single-cell multi-omic analyses and flow cytometry, we demonstrated, for the first time, that Tet2 deficiency mitigates detrimental HSC aging processes at the transcriptomic, epigenomic, and cellular levels, thereby preserving HSC function and promoting age-dependent expansion. Age-related gene expression changes, including increased Eya4, Lars2, and Mical2 and decreased Cdk6, CD53, and Sox4, were mitigated in Tet2-deficient HSCs. These gene expression changes were confirmed by single-nucleus ATAC-seq. Chromatin accessibility analysis also revealed Tet2 deficiency predominantly mitigates epigenetic changes associated with HSC aging. Specifically, age-related increases in chromatin accessibility of binding motifs of transcription factors (TFs), such as E2f4, Ppard, Mycn, and Max, and decreases in binding motifs of chromatin accessibility for Gabpa, Runx3, Etv4, and Elf1, were also mitigated in Tet2-deficient HSCs. Furthermore, DNA methylation status of these TF binding motifs is linked to the regulation for the targets’ gene expression, and CTCF insulator binding during HSC aging are also mitigated by Tet2 deficiency. Together, Tet2 deficiency stabilizes TF-target gene pairs critical for maintaining HSC self-renewal during aging. In conclusion, our findings reveal that Tet2 deficiency significantly contributes to the mitigation of HSC aging and hijacks the HSC expansion strategy during HSC aging via epigenetic reprogramming, which contributes to age-related clonal hematopoiesis. By elucidating the transcriptomic and epigenomic alterations in Tet2-deficient HSCs, our study provides novel insights into how age and somatic mutations interact to promote the pathogenesis of age-related hematological diseases. These discoveries enhance our understanding of how aging and somatic mutations interact, offering insights into potential biomarkers and therapeutic targets for age-associated hematological disorders.
 Citation Format: Shilpita Karmakar, Efthymios Motakis, Yang Liu, Kaustav Sengupta, Lamis Naddaf, Travis Roeder, Brandon Vu, Kristina Mujica, Ming Xu, Eric Pietrias, James DeGregori, Jennifer Trowbridge, Hideyuki Oguro, Sheng Li. Tet2 deficiency mitigates epigenetic aging in clonal hematopoiesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A014.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b4766c6ad595902085d51a487e2edbf0d4dfa6d" target='_blank'>
              Abstract A014: Tet2 deficiency mitigates epigenetic aging in clonal hematopoiesis
              </a>
            </td>
          <td>
            Shilpita Karmakar, Efthymios Motakis, Yang Liu, Kaustav Sengupta, Lamis Naddaf, Travis Roeder, Brandon Vu, Kristina D. Mujica, Ming Xu, Eric Pietrias, James DeGregori, Jennifer J. Trowbridge, Hideyuki Oguro, Sheng Li
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Graphical abstract The chromosomal status of Day 5 blastocysts was determined based on trophectoderm biopsy for all 24 chromosomes by array comparative genomic hybridization (array-CGH). The blastocoel fluid (BF) was extruded and amplified, and DNA concentration was read by a Qubit fluorometer. Apoptotic gene expression (TNFRSF10B, CASP2, BAX and CASP3) was analyzed by real-time quantitative PCR (RT-qPCR). Our findings suggest BF-DNA may be released in the blastocoel cavity of aneuploidy blastocysts under the influence of the apoptosis mechanism, and it appears that quantifying BF-DNA has the potential to aid in the selection of viable embryos. Abstract It is believed that aneuploid embryos release cell-free DNA (cfDNA) into the blastocyst cavity during the self-correction process through the apoptotic mechanism. This study aimed to develop less invasive methods for predicting ploidy status by investigating how ploidy status affects blastocoel fluid DNA (BF-DNA) levels and apoptotic gene expression as indicators of embryo viability. Human blastocysts were classified into three groups; survivable embryo (SE), fatal single and double aneuploidy (FSDA) and multiple aneuploidy (MA) using array comparative genomic hybridization (array-CGH) by trophectoderm biopsy. Following BF aspiration and whole genome amplification, BF-DNA level was quantified. Apoptotic activity was assessed by measuring the genes TNFRSF10B, CASP2, BAX and CASP3 using real-time quantitative PCR. Day-5 intracytoplasmic sperm injection blastocysts were scored according to the Gardner and Schoolcraft system. BF-DNA levels were significantly higher in the MA vs SE group (P = 0.01), while these were not statistically significant differences between the MA and FSDA groups or between the FSDA and SE groups, P = 0.17 and P = 0.38, respectively. TNFRSF10B, CASP2 and CASP3 were overexpressed in the MA and FSDA groups compared to SE, while BAX was downregulated. We found a significant correlation between the amount of BF-DNA and apoptosis marker genes. No significant correlation was found between embryo morphology score and BF-DNA level, BF volume or apoptosis marker gene expression levels. We observed that the correlation between apoptotic activity and BF-DNA levels is influenced by the embryo’s ploidy status. These findings suggest that BF-DNA level evaluation can be applicable in selecting viable embryos for transfer. Lay summary Preimplantation genetic testing helps doctors choose healthy human embryos for transfer during fertility treatments, but it can be expensive, invasive and time-consuming. Recently, scientists have found a less invasive way to study embryos by looking at DNA in a fluid inside the embryo’s cavity. This fluid may give us clues about how embryos try to fix problems with their chromosomes through a natural process of cell death. Our study shows that the amount of DNA in this fluid and the activity of some genes are influenced by chromosomal problems in the embryo. We also discovered that just looking at an embryo under a microscope is not enough to evaluate its genetic health. Our findings suggest that measuring the DNA in this fluid may be a promising approach for picking the best embryos for transfer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ea8bae39e64106e3ac6f98389c15425d00eec3c" target='_blank'>
              Correlation among blastocoel fluid DNA level, apoptotic genes expression and preimplantation aneuploidy
              </a>
            </td>
          <td>
            F. Khajehoseini, Zahra Noormohammadi, P. Eftekhari-Yazdi, H. Gourabi, R. Pazhoomand, S. Hosseinishenatal, M. Bazrgar
          </td>
          <td>2025-01-01</td>
          <td>Reproduction & Fertility</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d98b4e56fd873ffbe8593e5588ab4259e177e82" target='_blank'>
              CircTTC13 promotes sorafenib resistance in hepatocellular carcinoma through the inhibition of ferroptosis by targeting the miR-513a-5p/SLC7A11 axis
              </a>
            </td>
          <td>
            Ying Zhang, Ruiwei Yao, Mingyi Li, Chong-Kai Fang, Kun-Liang Feng, Xiuru Chen, Ji-Nan Wang, Rui Luo, Hanqian Shi, Xinqiu Chen, Xilin Zhao, Hanlin Huang, Shuwei Liu, Bing Yin, Chong Zhong
          </td>
          <td>2025-01-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="FHOD1 is a crucial regulator of cellular actin dynamics, and growing evidence suggests its involvement in tumorigenesis. Nevertheless, the precise function of FHOD1 in colorectal cancer (CRC) is still not well-defined. FHOD1 expression was analyzed using TIMER 2.0, and its prognostic value was assessed using the Kaplan-Meier Plotter. Functional analysis was performed via LinkedOmics, and its role in immune infiltration was investigated using the TISCH2 database and GSVA package. Drug sensitivity related to FHOD1 was evaluated with R software. Additionally, the CCK-8 assay, colony formation assay, wound-healing assay, and Transwell migration assay were used to evaluate the impact of FHOD1 on the proliferation and migration of colorectal cancer cells. Our study proved that FHOD1 expression was substantially higher in CRC tissues than in normal tissues, correlating with poorer patient prognosis. Functional analysis indicated that FHOD1 was involved in immune-related processes and the tumor microenvironment, particularly affecting numerous types of immune cells, such as natural killer cells and T cells. FHOD1 expression was positively associated with sensitivity to multiple chemotherapeutic agents. Finally, knockdown of FHOD1 in HCT116 and RKO NL cell lines impaired cell proliferation and migration, highlighting its potential as a target for treatment in managing CRC. In conclusion, these findings underscore the importance of FHOD1 in CRC progression and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f18a6ecb626755975b3df2b432fa11bf48956c18" target='_blank'>
              FHOD1 is a promising biomarker for diagnosis, prognosis, and immunotherapy in colorectal cancer
              </a>
            </td>
          <td>
            Yanan Li, Hua Zuo, Zhihong Liu, Ziyu Wu, Zeyang Zhou, Jianbo Deng, Xiaodong Yang
          </td>
          <td>2025-02-07</td>
          <td>Molecular &amp; Cellular Biomechanics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Motivation Allele-specific methylation (ASM) refers to differential DNA methylation patterns between two alleles at a given locus. This phenomenon is often driven by genetic variants, such as single nucleotide variants (SNVs), which influence methylation in cis by affecting transcription factor or methylation regulator binding, leading to allele-specific differences. Understanding ASM is critical for elucidating gene regulation, as it impacts gene expression and contributes to normal biological variation as well as disease processes, including cancer [1] and autoimmune disorders [2],[3]. Another key driver of ASM is genomic imprinting, an epigenetic mechanism in which gene expression is regulated in a parent-of-origin-specific manner. Imprinted regions, marked during gametogenesis and maintained through cell divisions, are essential for growth, development, and metabolism. Dysregulation of imprinting is associated with developmental and metabolic disorders, such as Prader-Willi and Angelman syndromes, and certain cancers. Detecting ASM across the genome remains challenging due to its tissue- and cell-specific nature and the technical difficulty of phasing reads to assign methylation patterns to specific alleles. Current ASM detection pipelines (e.g., [4])) often require phasing via genetic variants, a computationally intensive process that is limited in regions with low heterozygosity. Results To address these limitations, we developed asms (Allele-Specific Methylation Scanner), a tool designed to detect ASM directly from methylation data without the need for prior phasing. asms offers a faster and complementary approach to uncovering the regulatory effects of ASM, particularly in genomic regions where genetic variants or imprinting play a critical role. For demonstration purposes we leverage the fact that reads generated by Oxford Nanopore (ONT) technology measure sequence and methylation status at once, but the same software can be used with other sequencing technologies. asms can check thousands of loci in a short time. The initial list of loci to examine can be given by the user, or generated by asms through a genomic scan. The asms cluster subcommand separates the reads based on methylation. If phasing results are available, asms can use them to verify whether distinct alleles correspond to distinct base modifications patterns. We benchmark our software using publicly available Ashkenazi trio data [5]. Implementation and availability asms is implemented in rust and python. The software is available at https://github.com/ecmra/asms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c7dfc00aae1ea08fcc56a90d449bd2018337af0" target='_blank'>
              ASMS: finding allele specific methylation in human genomes without phasing
              </a>
            </td>
          <td>
            E. Raineri, Miguel Ángel Esteve Marco, Anna Esteve Codina
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Circular RNA (CircRNA)s, a newly discovered type of noncoding RNAs, have been found to play a role in controlling the development and aggressiveness of tumors. Abnormal control of circRNA has been observed in various types of human cancers, including bladder cancer, hepatocellular carcinoma (HCC), breast cancer, and gastric cancer (GC). CircRNAs possess binding sites for microRNAs (miRNAs) and function as miRNA sponges in posttranscriptional regulation. This mechanism has been documented to influence the course of cancer. Significantly, among these putative circRNAs, circular RNA protein tyrosine kinase 2 (circPTK2) exhibited increased expression and displayed a substantial association with adverse clinical characteristics and a negative prognosis. The production of these transcripts occurs via a back-splicing mechanism. The enclosed conformation of circRNAs shields them from destruction and enhances their potential as biomarkers. Gaining insight into the molecular mechanisms involved in these processes would aid in the development of treatment approaches and the discovery of new tumor markers. This article provides a comprehensive assessment of the latest research on the biosynthesis and features of circRNAs. It examines the role of circPTK2 in the diagnosis, treatment, and prognosis evaluation of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdd1e91ef7563824a8ebebdc925d90d66cc38471" target='_blank'>
              CircPTK2 as a Valuable Biomarker and Treatment Target in Cancer.
              </a>
            </td>
          <td>
            Chengqiu Yan, Yu Du, Lihong Cui, Han Bao, Hui Li
          </td>
          <td>2025-01-30</td>
          <td>Journal of biochemical and molecular toxicology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="While macroautophagy (autophagy) contributes to maintaining chromosomal stability via multiple pathways, including regulating chromatin ubiquitination and cytoplasmic DNA fragment degradation, the impacts of microautophagy and chaperone-mediated autophagy (CMA) on maintaining chromosomal stability are not known. The TTC28 (tetratricopeptide repeat domain 28) gene is frequently mutated and downregulated in human cancers. The molecular mass of the TTC28 protein is 271 kDa, which makes its functional study very difficult. Recently, we reported that TTC28 plays a key role in maintaining chromosomal stability, probably through regulating mitosis and cytokinesis, and that TTC28 downregulation may contribute to the high chromosomal instability (CIN) of cancer cells, according to the results of serial experiments and bioinformatics analyses. Notably, our findings demonstrate that TTC28 is a substrate of CMA and microautophagy and that these pathways also play a role in maintaining chromosomal stability in a TTC28-dependent manner. These findings demonstrate that CMA- and microautophagy-mediated degradation is a master regulator of the ability of TTC28 to maintain genome stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92539e94d331171c7415fced3b2e09352d075f34" target='_blank'>
              Chaperone-mediated autophagy contributes to chromosomal stability by controlling TTC28 degradation.
              </a>
            </td>
          <td>
            Ge Zhang, Wei Tian, Dajun Deng
          </td>
          <td>2025-02-02</td>
          <td>Autophagy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba51e0ae284150abf200e92c2ae4133d35b65e6d" target='_blank'>
              Heterozygous mutation in BRCA2 induces accelerated age-dependent decline in sperm quality with male subfertility in rats
              </a>
            </td>
          <td>
            Yashiro Motooka, Hideaki Tanaka, Yu-ichi Maeda, Misako Katabuchi, Tomoji Mashimo, Shinya Toyokuni
          </td>
          <td>2025-01-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Viral mimicry refers to an active antiviral response triggered by the activation of endogenous retroviruses (ERVs), usually manifested by the formation of double-stranded RNA (dsRNA) and activation of the cellular interferon response, which activates the immune system and produces anti-tumor effects. Epigenetic studies have shown that epigenetic modifications (e.g. DNA methylation, histone modifications, etc.) play a crucial role in tumorigenesis, progression, and treatment resistance. Particularly, alterations in DNA methylation may be closely associated with the suppression of ERVs expression, and treatment by demethylation may restore ERVs activity and thus strengthen the tumor immune response. Therefore, we propose that viral mimicry can induce immune responses in the tumor microenvironment by activating the expression of ERVs, and that epigenetic alterations may play a key regulatory role in this process. In this paper, we review the intersection of viral mimicry, epigenetics and tumor immunotherapy, and explore the possible interactions and synergistic effects among the three, aiming to provide a new theoretical basis and potential strategies for cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/916b8820f91cde5f4b984b3eba25d2fd40a973b1" target='_blank'>
              Exploring viral mimicry combined with epigenetics and tumor immunity: new perspectives in cancer therapy
              </a>
            </td>
          <td>
            Ruirui Wang, Xin Dong, Xiongjian Zhang, Jinzhuang Liao, Wei Cui, Wei Li
          </td>
          <td>2025-01-06</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e63083f2ebe39f75bcbba803ff0b3dd9f4e01b8c" target='_blank'>
              The role of exosomal non-coding RNAs in the breast cancer tumor microenvironment.
              </a>
            </td>
          <td>
            Mohamed J Saadh, O. Q. B. Allela, Radhwan Abdul Kareem, S. Ballal, Mamata Chahar, Suman Saini, G. V. S. Prasad, Hayder Naji Sameer, Atheer Khdyair Hamad, Zainab H. Athab, Mohaned Adil
          </td>
          <td>2025-02-01</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="MCM2 belongs to the microchromosome maintenance [MCM] family and plays an essential role in initiating DNA replication as well as maintaining normal cellular cycle functions. Recent research indicates that there is the abnormal expression of MCM2 in various cancers, such as breast, cervical, ovarian, lung, hepatocellular carcinoma, nephroblastoma, prostate, and pancreatic cancers, where it shows a strong link to tumorigenesis, growth, invasion, migration, and adverse prognosis. Thus, MCM2 could serve as a significant biomarker for the early identification, diagnosis, and prognostic evaluation of multiple cancers. In addition, targeting MCM2 expression may open new possibilities for a full range of cancer treatments. In this paper, the protein structure, physiological function, and carcinogenic mechanism of MCM2 were reviewed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8b9305ad5bdf4a40999bdb1d8deb688207b86c2" target='_blank'>
              Utilization of DNA Replication factor MCM2 by Cancer.
              </a>
            </td>
          <td>
            Yuwei Yuan, Tian Zeng, Anbo Gao, Yang Guan, Qun-feng Zhang, Yukun Li, Hui Tan, Juan Zou
          </td>
          <td>2025-02-13</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Chronic hepatitis B virus (HBV) infection can significantly increase the incidence of cirrhosis and liver cancer, and there is no curative treatment. The persistence of HBV covalently closed circular DNA (cccDNA) is the major obstacle of antiviral treatments. cccDNA is formed through repairing viral partially double-stranded relaxed circular DNA (rcDNA) by varies host factors. However, the detailed mechanisms are not well characterized. To dissect the biogenesis of cccDNA, we took advantage of an in vitro rcDNA repair system to precipitate host factors interacting with rcDNA and identified co-precipitated proteins by mass spectrometry. Results revealed the MRE11–RAD50–NBS1 (MRN) complex as a potential factor. Transiently or stably knockdown of MRE11, RAD50 or NBS1 in hepatocytes before HBV infection significantly decreased viral markers, including cccDNA, while reconstitution reversed the effect. Chromatin immunoprecipitation assay further validated the interaction of MRN complex and HBV DNA. However, MRN knockdown after HBV infection showed no effect on viral replication, which indicated that MRN complex inhibited the formation of cccDNA without affecting its stability or transcriptional activity. Interestingly, Mirin, a MRN complex inhibitor which can inhibit the exonuclease activity of MRE11 and MRN-dependent activation of ATM, but not ATM kinase inhibitor KU55933, could decrease cccDNA level. Likewise, the MRE11 endonuclease activity inhibitor PFM01 treatment decreased cccDNA. MRE11 nuclease assays indicated that rcDNA is a substrate of MRE11. Furthermore, the inhibition of ATR-CHK1 pathway, which is known to be involved in cccDNA formation, impaired the effect of MRN complex on cccDNA. Similarly, inhibition of MRE11 endonuclease activity mitigated the effect of ATR-CHK1 pathway on cccDNA. These findings indicate that MRN complex cooperates with ATR-CHK1 pathway to regulate the formation of HBV cccDNA. In summary, we identified host factors, specifically the MRN complex, regulating cccDNA formation during HBV infection. These findings provide insights into how HBV hijacks host enzymes to establish chronic infection and reveal new therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54b77d859792051cb6318950a8aecf132b1dd9b2" target='_blank'>
              Hepatitis B virus hijacks MRE11–RAD50–NBS1 complex to form its minichromosome
              </a>
            </td>
          <td>
            Kaitao Zhao, Jingjing Wang, Zichen Wang, Mengfei Wang, Chen Li, Zaichao Xu, Qiong Zhan, Fangteng Guo, Xiaoming Cheng, Yuchen Xia
          </td>
          <td>2025-01-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id=". The development of cancer involves an intri‑ cate mechanism, including oncogene activation and tumor suppressor gene inactivation, often driven by epigenetic altera‑ tions. Notably, these epigenetic modifications, such as DNA hypermethylation, are dynamic and reversible, and emerging research suggests that dietary factors can influence them. Recent findings have examined the intricate associations between dietary compounds and cancer development, with a particular focus on the mechanisms through which natural components affect epigenetic alterations. Natural compounds have gained significant attention for their potential in chemo‑ prevention and treatment due to their capacity to modulate DNA methylation patterns, particularly at promoter CpG islands. Chemotherapy and radiation therapy are effective cancer treatments and are often associated with severe adverse effects. As a result, there is increasing interest in exploring natural substances as alternative therapeutic options. The present review focuses on natural compounds that can reverse hypermethylation, providing a potentially safer approach for targeted cancer treatment. These bioactive agents, abundant in natural compounds, exhibit the potential to combat cancer by inhibiting metastasis, inducing cell cycle arrest and reversing DNA hypermethylation. The present review aimed to provide further in‑depth mechanistic insight into the mechanisms through which natural compounds, such as genistein, resvera‑ trol, quercetin and capsaicin modulate DNA hypermethylation in various types of cancer, including breast, cervical, prostate and neuroblastoma. By combining in silico , in vitro and in vivo approaches, it uniquely integrates computational and experimental evidence to demonstrate compound‑specific reversal of epigenetic marks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eaa3ae9852c9314b773d0fad7daf6e3a6798eb1" target='_blank'>
              Natural compounds as epigenetic modulators: Reversing DNA hypermethylation in cancer (Review)
              </a>
            </td>
          <td>
            Farheen Farheen, Kantimahanti Shreya, R. Mathur, A. Jha
          </td>
          <td>2024-12-30</td>
          <td>World Academy of Sciences Journal</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="INhibitor of Growth (ING1-5) proteins are epigenetic readers that target histone acetyltransferase (HAT) or histone deacetylase (HDAC) complexes to the H3K4Me3 mark of active transcription. ING5 targets Moz/Morf and HBO1 HAT complexes that alter acetylation of H3 and H4 core histones, affecting gene expression. Previous experiments in vitro indicated that ING5 functions to maintain stem cell character in normal and in cancer stem cells. Here we find that CRISPR/Cas9 ING5 knockout (KO) mice are sub-fertile but show no decrease in lifespan or ability to heal wounds despite indications of depleted stem cell pools in several tissues. ING5 KO mouse embryo fibroblasts accumulate in G2 of the cell cycle, have high levels of abnormal nuclei and show high basal levels of the γH2AX indicator of DNA damage. KO animals also develop severe dermatitis at a 5-fold higher rate that wild-type littermates. Consistent with ING5 serving a tumor suppressive role, ING5 KO mice developed germinal centre diffuse large B-cell lymphomas at a rate 6-fold higher than control mice at 18 months of age. These data suggest that ING5 functions in vivo to maintain stem cell character in multiple organs, that reduction of stem cell populations is not limiting for murine lifespan and that like a subset of other ING family members, ING5 functions as a tumor suppressor in hematopoietic cells in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7856dbb73e87aa51549dc70eedb5371d011208e6" target='_blank'>
              Knockout of the ING5 epigenetic regulator confirms roles in stem cell maintenance and tumor suppression in vivo
              </a>
            </td>
          <td>
            Buthaina Al Shueili, Arthur Dantas, Etienne Mahe, Tak-Ho Chu, Yang Yang, Elodie Labit, E. Kutluberk, Nicolas Lasaleta, Anand Masson, Hiba Omairi, Kenichi Ito, Roman J. Krawetz, R. Midha, Gregory Cairncross, Karl Riabowol
          </td>
          <td>2025-01-09</td>
          <td>PLOS ONE</td>
          <td>1</td>
          <td>54</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes led to a better understanding of tumorigenesis, and prompted the development of molecularly targeted therapy. Over the past 30 years, many new drugs, which are primarily aimed at activated oncogenic proteins in signal transduction pathways involved in cell proliferation and survival, have been introduced in the clinic. Despite its rational design, the overall efficacy of targeted therapy has been modest. Recently, the noncoding RNAs (ncRNAs) have emerged as key regulators of important cellular processes in addition to the known regulatory proteins. It now appears that dual epigenetic regulatory systems exist in higher eukaryotic cells: a ncRNA network that governs essential cell functions, like cell fate decision and maintenance of homeostasis, and a protein-based system that presides over core physiological processes, like cell division and genomic maintenance. Modifications of the ncRNA network due to altered ncRNAs can cause the cell to shift towards to neoplastic phenotype; this is cancer initiation. Mutations in the well-known cancer driver genes provide the incipient cancer cell with a selective growth advantage and fuel its consequent clonal expansion. Because of the crucial role of the altered ncRNAs in tumorigenesis, targeting them may be a reasonable therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4bbdeb17c896767d39cab33f2cc2532e16bb98" target='_blank'>
              Implications of noncoding RNAs for cancer therapy: Are we aiming at the right targets?
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-01-16</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eec91fe3f66acbc6c5440f4e1bf95d9c7af5ce45" target='_blank'>
              Predicting CTCF cell type active binding sites in human genome
              </a>
            </td>
          <td>
            Lu Chai, Jie Gao, Zihan Li, Hao Sun, Junjie Liu, Yong Wang, Lirong Zhang
          </td>
          <td>2024-12-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction FAM162A is a mitochondrial protein evolutionarily conserved across taxa and ubiquitously expressed in various tissues. It is known for its role in hypoxia-induced apoptosis. However, paradoxically, FAM162A is overexpressed in cancer, where its pro-apoptotic function seems overridden, suggesting an alternative role associated with mitochondrial function and cell survival. Additionally, its precise localization and topology remain controversial. Objectives To assess the role of FAM162A in mitochondrial structure, dynamics, and bioenergetics and its impact on cell viability, while establishing its precise localization, orientation, and topology. Additionally, to generate a transgenic Drosophila model overexpressing human FAM162A to evaluate its effects on organismal survival under normal and stress conditions. Methods Localization, orientation, and topology were determined by protease protection assays in COS7 cells. Loss-and gain-of-function experiments were performed to assess mitochondrial function and turnover by confocal microscopy, immunoblots and Seahorse technology. A transgenic Drosophila model overexpressing human FAM162A was generated to evaluate organismal survival under normal and stress conditions. Results FAM162A is essential for maintaining mitochondrial ultrastructure and bioenergetics, thereby influencing cell viability and stress resistance. Localization studies revealed that FAM162A resides predominantly in the inner mitochondrial membrane, particularly within the cristae, where it modulates the fusion protein OPA1. Transgenic Drosophila overexpressing human FAM162A exhibited increased lifespan and locomotor activity under both normal and heat stress conditions. Conclusion FAM162A emerges as a crucial player in maintaining mitochondrial integrity and bioenergetics. Its functional role, potentially mediated through interaction with OPA1, impacts mitochondrial health, stress resistance, cellular viability, and organismal longevity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6268fc10cfb81c121840b39972df18df93dfb23" target='_blank'>
              FAM162A is Crucial for Mitochondrial Structure, Dynamics, and Bioenergetics, Driving Cellular Protection and Longevity
              </a>
            </td>
          <td>
            Andrea Matamoros, Juan Pablo Soffia, Marcelo Muñoz, Michael Maturana, Alvaro Gonzalez-Ibañez, Gabriela Gomez-Lillo, Cesar Astorga, L. Ruiz, Ramón Jorquera, Álvaro A. Elorza
          </td>
          <td>2024-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Breast cancer is the most common malignancy affecting women, manifesting as a heterogeneous disease with diverse molecular characteristics and clinical presentations. Recent studies have elucidated the role of epigenetic modifications in the pathogenesis of breast cancer, including drug resistance and efflux characteristics, offering potential new diagnostic and prognostic markers, treatment efficacy predictors, and therapeutic agents. Key modifications include DNA cytosine methylation and the covalent modification of histone proteins. Unlike genetic mutations, reprogramming the epigenetic landscape of the cancer epigenome is a promising targeted therapy for the treatment and reversal of drug resistance. Epidrugs, which target DNA methylation and histone modifications, can provide novel options for the treatment of breast cancer by reversing the acquired resistance to treatment. Currently, the most promising approach involves combination therapies consisting of epidrugs with immune checkpoint inhibitors. This review examines the aberrant epigenetic regulation of breast cancer initiation and progression, focusing on modifications related to estrogen signaling, drug resistance, cancer progression, and the epithelial–mesenchymal transition (EMT). It examines existing epigenetic drugs for treating breast cancer, including agents that modify DNA, inhibitors of histone acetyltransferases, histone deacetylases, histone methyltransferases, and histone demethyltransferases. It also delves into ongoing studies on combining epidrugs with other therapies and addresses the upcoming obstacles in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b1d287b99832f39586fe0fd89b9b4c10e300e3c" target='_blank'>
              Cellular Epigenetic Targets and Epidrugs in Breast Cancer Therapy: Mechanisms, Challenges, and Future Perspectives
              </a>
            </td>
          <td>
            Ibrahim S. Alalhareth, Saleh M. Alyami, Ali H. Alshareef, Ahmed O. Ajeibi, M. A. Al Munjem, Ahmad A. Elfifi, Meshal M. Alsharif, S. A. Alzahrani, Mohammed A. Alqaad, Marwa B. Bakir, Basel A. Abdel-Wahab
          </td>
          <td>2025-02-03</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Increased telomerase activity has been considered as a conspicuous sign of human cancers. The catalytic cores of telomerase involve a reverse transcriptase and the human telomerase RNA (hTR). However, current detection of telomerase is largely limited to its activity at the tissue and single-cell levels. To reveal the precise distribution of subcellular hTR and telomerase activity, here a modular engineered DNA nanodevice (DNA-ND) is designed capable of imaging hTR and telomerase activity in cytoplasm and nucleus, enabling colocalization analysis. DNA-ND is a modular DNA complex comprising hTR and telomerase activity detection modules, which respectively sense intercellular hTR and telomerase activity via target-sensitive allosteric transition of DNA switches, actuating orthogonal activation of fluorescence signals to achieve in situ co-imaging of hTR and telomerase activity. By integrating DNA-ND with specific localized signals, the DNA-ND based precise profiling of colocalization of hTR and telomerase activity in different cell lines as well as their dynamic changes during pharmacological interventions is demonstrated. Notably, the results suggest that the locations of hTR and telomerase activity are not exactly overlapped, indicating the influence of intracellular environment on the binding of hTR to telomerase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5a34a17a7f6af1948fe0851efc3da900886ee2f" target='_blank'>
              A Modular Engineered DNA Nanodevice for Precise Profiling of Telomerase RNA Location and Activity.
              </a>
            </td>
          <td>
            Shiyi Zhang, Jian Lv, Ze-Rui Zhou, Peter X Geng, Dawei Li, Ruocan Qian, H. Ju
          </td>
          <td>2024-12-27</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cell division cycle-associated (CDCA) genes are dysregulated in carcinomas. Our study aims to identify similarities and differences of the clinical roles of CDCAs in breast cancer (BRCA) and to explore their potential mechanisms. In GEPIA, compared to normal tissues, expressions of CDCAs were higher in BRCA and sub-types. In addition, CDCAs were significantly positively related to stages and predicted worse survival in BRCA. In CancerSEA, expression levels of most CDCAs were strongly positively related to cell cycle, DNA damage, DNA repair, and proliferation. In TIMER, CDCAs were linked with immune infiltration levels of BRCA, including Dendritic cell, B cell and so on, and were positively related to most of the common markers of immune cells, especially CD38 of B cell and IL12RB2 of Th1. In GeneMANIA, there were complex interactions and co-expression relationships between CDCAs and cell division-associated genes. In addition, CDCA1, CDCA3, CDCA5, CDCA6 and CDCA8 had a high proportion of amplification in BRCA, and CDCA1, CDCA2, CDCA5, CDCA7 and CDCA8 had high levels of body DNA methylation. Among 11 transcription factors possibly combining promoters of all CDCAs, FOXP3 and YY1 were significantly higher in BRCA in comparison to normal tissues, and both had a positive relationship with all CDCAs in GEPIA and IHC. In addition, silencing FOXP3 or YY1 decreased levels of CDCAs in MDA-MB-231. In summary, CDCAs have various similarities in clinical functions, functional states, immune infiltration, and mechanisms, and they may become novel potential biomarkers for BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0573bd3b9dd89f3992773c13c07581660383688" target='_blank'>
              Roles of the CDCA gene family in breast carcinoma
              </a>
            </td>
          <td>
            Wei Ding, Wei Han, Chun-Tao Shi, Li-qian Yao, Zhi-wei Liang, Ming-hui Zhou, Hao-nan Wang
          </td>
          <td>2025-01-01</td>
          <td>Science Progress</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer therapies have attempted to target the transcription factor p53, a gene also described as the “guardian of the genome,” for decades. However, the approach has faced numerous barriers to clinical efficacy due to several factors: mutations in p53 occur in almost half of all human cancers, mutations are cancer-specific, and the associated genomic changes grant mutant p53 with oncogenic potential unique from that of wild-type p53. A host of new therapeutic agents have emerged that work to target mutant p53. These agents can broadly be classified into six categories: the viral approach, direct modifiers of the p53 pathway, epigenetic modifiers of the p53 pathway, synthetic lethal agents, structural reactivators, and immune activating vaccines. Even these strategies have been met with limited success. Bypassing p53 entirely may be the next avenue in cancer therapeutics to kill tumor cells regardless of p53’s mutation pattern.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25b1fb68a093354533852b3fe76935ba3e2890db" target='_blank'>
              Side-stepping the guardian of the genome: current cancer therapeutics targeting mutant p53
              </a>
            </td>
          <td>
            I. Taritsa, Eric T. Fossel
          </td>
          <td>2025-01-29</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genomic rearrangements play an important role in shaping genetic diversity, as they enable the generation of novel structural variations through specific genome manipulation tools. These variations contribute to phenotypic differences among individuals within a population, thereby serving as the foundation for natural selection and driving evolutionary processes. In recent years, Synthetic Chromosome Rearrangement and Modification by LoxP-mediated Evolution (SCRaMbLE) has emerged as a promising tool for studying genomic rearrangements. SCRaMbLE utilizes site-specific recombination mediated by loxPsym sites to induce targeted chromosomal rearrangements in yeast cells. In this review, we provide a comprehensive overview of recent advancements in optimization strategies of the SCRaMbLE system and discuss influential factors that affect its performance based on recent research findings. We demonstrate how the SCRaMbLE system can be employed for pathway engineering, phenotype improvement, genome minimization, and dissection of genotype-to-phenotype relationships. We highlight both the advantages and challenges associated with SCRaMbLE and envision its potential applications beyond yeast genetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0314619e822b8ab25ecabe0ad0985b3e4d4d9a40" target='_blank'>
              SCRaMbLE: a versatile tool for genome manipulation.
              </a>
            </td>
          <td>
            Li Cheng, Jielin Li, Yaojun Chen, Junbiao Dai
          </td>
          <td>2025-02-12</td>
          <td>Physiology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/937fb72f5c07d87edcc2493e093c8ba719c5c0a0" target='_blank'>
              TET2 cascade: a novel regulator of chromatin structure and leukaemogenesis
              </a>
            </td>
          <td>
            Wolfram C. M. Dempke, K. Fenchel
          </td>
          <td>2025-01-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Saccharomyces cerevisiae meiosis-specific Hop1, a structural constituent of the synaptonemal complex, also facilitates the formation of programmed DNA double-strand breaks and the pairing of homologous chromosomes. Here, we reveal a serendipitous discovery that Hop1 possesses robust DNA-independent ATPase activity, although it lacks recognizable sequence motifs required for ATP binding and hydrolysis. By leveraging molecular docking combined with molecular dynamics simulations and biochemical assays, we identified an ensemble of five amino acid residues in Hop1 that could potentially participate in ATP-binding and hydrolysis. Consistent with this premise, we found that Hop1 binds to ATP and that substitution of amino acid residues in the putative ATP-binding site significantly impaired its ATPase activity, suggesting that this activity is intrinsic to Hop1. Notably, K65A and N67Q substitutions in the Hop1 N-terminal HORMA domain synergistically abolished its ATPase activity, noticeably impaired its DNA-binding affinity and reduced its association with meiotic chromosomes, while enhancing the frequency of meiotic crossovers (COs). Overall, our study establishes Hop1 as a DNA-independent ATPase and reveals a potential biological function for its ATPase activity in the regulation of meiotic CO frequency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2297e5fc2e40c9dac9d79a6d0d22ef4ffd4fc1ea" target='_blank'>
              The ATPase activity of yeast chromosome axis protein Hop1 affects the frequency of meiotic crossovers
              </a>
            </td>
          <td>
            Kshitiza M Dhyani, Suman Dash, Sameer Joshi, Aditi Garg, D. Pal, K. T. Nishant, K. Muniyappa
          </td>
          <td>2024-12-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Mismatch repair (MMR) is a crucial DNA repair pathway that maintains genomic integrity by correcting replication errors and various forms of DNA damage. MMR deficiency (MMRd) leads to increased mutation rates, microsatellite instability, and contributes to tumorigenesis in multiple cancer types. Using a CRISPR-Cas9-mediated knockout assay in human isogenic cell lines, we characterised mutational profiles in MMR-deficient cells. Our findings revealed expected increases in mutation burden and the emergence of known MMR-associated mutational signatures. Notably, we identified a previously unconnected process, SBS57, and linked it to germline single-nucleotide polymorphisms and MMR-driven indels in MMRd cells, establishing its association with tensor signature TS27. Comparative analyses of in vitro MMRd profiles and in vivo tumour data uncovered key differences in mutational signatures, highlighting the biological context dependence of MMR-associated mutations. Furthermore, we provide direct experimental evidence that MMR plays a role in repairing 5-methylcytosine deamination, a repair process previously inferred from tumour sequencing data. These findings offer novel insights into MMR deficiency, shedding light on previously uncharacterised mutational mechanisms and their implications in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a293399722ef8f7c62c0c36dd67ff9d94d28e7" target='_blank'>
              Dissecting DNA-mismatch-repair-driven mutational processes in human cells
              </a>
            </td>
          <td>
            Michel Owusu, Jörg Menche, Joanna Loizou, Donate Weghorn
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background/Objectives: Lung cancer ranks as the leading cause of cancer-related deaths globally and is highly associated with cisplatin resistance due to both intrinsic and extrinsic mechanisms. Proliferating Cell Nuclear Antigen (PCNA) plays a critical role in molecular processes, such as DNA replication and repair, chromatin structure maintenance, and cell cycle progression. PCNA is known as a molecular marker for proliferation and an excellent inhibition target to shut down highly proliferative cells. One of the mechanisms of cisplatin resistance is the increase in DNA repair, and studies have reported an association between PCNA, lung cancer, and cisplatin treatment. The present study aimed to characterize the absence of PCNA in A549 lung adenocarcinoma cells. Methods: Employing a CRISPR/Cas9 gene-editing approach, we generated a monoclonal cell culture, termed PKO (PCNA knockout). Results: PKO cells exhibited a residual PCNA expression, significantly decreased clonogenic potential and ubiquitylation at K164 residue. IC50 assay suggested that PKO cells could not acquire cisplatin resistance when compared to PX. After cisplatin treatment, PKO cells presented impaired ubiquitylation and did not have increased STAT3 phosphorylation (Tyr705), a previously characterized mechanism of cisplatin resistance. Conclusions: We suggest that PCNA participates in cisplatin resistance in A549, partially by DNA damage tolerance through failure on PCNA monoubiquitylation, and its inhibition may be an approach to circumvent cisplatin resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4e12fe22d095e0fcb48503db24d12eb54e726a" target='_blank'>
              Partial Proliferating Cell Nuclear Antigen Functional Knockout Impairs Cisplatin Resistance and Clonogenic Potential in Lung Adenocarcinoma Cells
              </a>
            </td>
          <td>
            A. P. Morelli, Nathalia Quintero-Ruiz, M. Mancini, I. Pavan, Isabelle Lima Flores, Luiz Guilherme Salvino Silva, M. B. Severino, R. M. N. Bezerra, F. Simabuco
          </td>
          <td>2025-02-02</td>
          <td>DNA</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The DNA Damage Tolerance pathway (DDT) is one of the major mechanisms for resolving replication fork blocks. A central factor for DDT is the fork-associated clamp, PCNA, which can undergo to mono- or polyubiquitination, leading to error-free or error-prone modes of repair, respectively. The Rad5 protein in the yeast Saccharomyces cerevisiae plays important roles in both pathways: promoting the error-free mode by PCNA polyubiquitination and interacting with specialized DNA polymerases involved in the error-prone pathway. Rad5 also associates with telomeres, the repetitive DNA regions present at the ends of chromosomes. Telomeric DNA, tightly bound by tandem proteins arrays, poses unique challenges to replication fork progression. Here, I review the current understanding of the link between Rad5 and telomeres and present evidence that Rad5 is associated with yeast telomeres, with notable enrichment during telomere replication. This data highlights a connection between telomeres and a key DDT factor in unperturbed wild-type cells, raising intriguing possibilities about the functional interplay between telomere replication and DNA damage tolerance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55178fe5efcefe7e662ad3c10eadc34e3ab5e75a" target='_blank'>
              Rad5 and Telomere Replication
              </a>
            </td>
          <td>
            Stefano Mattarocci
          </td>
          <td>2025-01-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Personalized precision medicine (PPPM) in cancer immunology and oncology is a rapidly advancing field with significant potential. Gliomas, known for their poor prognosis, rank among the most lethal brain tumors. Despite advancements, there remains a critical need for precise, individualized treatment strategies.We conducted a comprehensive analysis of RNA-seq and microarray data from the TCGA and GEO databases, supplemented by single-cell RNA sequencing (scRNA-seq) data from glioma patients. By integrating single-cell sequencing analysis with foundational experiments, we investigated the molecular variations and cellular interactions within neural glioma cell subpopulations during tumor progression.Our single-cell sequencing analysis revealed distinct gene expression patterns across glioma cell subpopulations. Notably, differentiation trajectory analysis identified NUSAP1 as a key marker for the terminal subpopulation. We found that elevated NUSAP1 expression correlated with poor prognosis, prompting further investigation of its functional role through both cellular and animal studies.NUSAP1-based risk models hold potential as predictive and therapeutic tools for personalized glioma treatment. In-depth exploration of NUSAP1’s mechanisms in glioblastoma could enhance our understanding of its response to immunotherapy, suggesting that targeting NUSAP1 may offer therapeutic benefits for glioma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5625f39e4872fec67e5df1a449a03f506dae3b79" target='_blank'>
              Single-cell sequencing elucidates the mechanism of NUSAP1 in glioma and its diagnostic and prognostic significance
              </a>
            </td>
          <td>
            Meng-Yu Zhao, Zhao-Lei Shen, Hongzhen Dai, Wan-Yan Xu, Li-Na Wang, Yu- Gu, Jie-Hui Zhao, Tian-Hang Yu, Cun-Zhi Wang, Jia-feng Xu, Guan-Jun Chen, Dong-Hui Chen, Wen-Ming Hong, Fan Zhang
          </td>
          <td>2025-02-05</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f33f540ba07c7d118037674f121aaa35d22c9a7" target='_blank'>
              Prolonged persistence of mutagenic DNA lesions in somatic cells.
              </a>
            </td>
          <td>
            M. Spencer Chapman, E. Mitchell, Kenichi Yoshida, N. Williams, Margarete A. Fabre, A. Ranzoni, Philip S. Robinson, Lori D Kregar, Matthias Wilk, Steffen Boettcher, Krishnaa Mahbubani, Kourosh Saeb Parsy, K. Gowers, Sam M Janes, Stanley W K Ng, Matt Hoare, Anthony R Green, G. Vassiliou, A. Cvejic, M. Manz, E. Laurenti, I. Martincorena, Michael R Stratton, J. Nangalia, Tim H. H. Coorens, P. J. Campbell
          </td>
          <td>2025-01-15</td>
          <td>Nature</td>
          <td>1</td>
          <td>88</td>
        </tr>

        <tr id="Abstract Recent studies have confirmed that certain circRNAs encode proteins that are integral to various biological functions. In this study, we present CICADA, an algorithm specifically designed to assess the protein-coding potential and coding products of circRNAs at high throughput, which enables the identification of previously unknown circRNA-encoded proteins. By harnessing the potential of this algorithm, we identified a variety of functional, protein-coding circRNAs in esophageal squamous cell carcinoma and established circRNA translation profiles for diverse types of cancer. This advancement innovatively explores the hidden proteome within the genome, poised to catalyze discoveries in biomarkers and therapies for cancers and complex diseases. CICADA is accessible as a Python module (https://github.com/SunLab-biotool/CICADA).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87d9e64c4a92f4e68d7e0fc75f0c4526c12b1fa0" target='_blank'>
              CICADA: a circRNA effort toward the ghost proteome
              </a>
            </td>
          <td>
            Linyuan Fan, Xinyuan Zhou, Mian Li, Anwei Gao, Haojie Yu, Hui Tian, Liandi Liao, Liyan Xu, Liang Sun
          </td>
          <td>2024-12-23</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Pituitary neuroendocrine tumors (PitNETs) are unusual among neoplasms in that sporadic tumors are not typically associated with genetic mutations. Instead, epigenetics, a non-mutational process by which gene expression is modified via a variety of mechanisms, may be a driving factor in PitNET growth and behavior. DNA methylation is one of the most well understood forms of epigenetic modification. Research on DNA methylation profiles of PitNETs has identified a large number of genes silenced or upregulated by DNA methylation, particularly when methylated at CpG islands of gene promoter regions. Global patterns of DNA methylation may provide valuable insight into origins of pituitary adenoma subtypes, assist with PitNet diagnostics and have been found to correspond to the current WHO classification of PitNETs based on transcription factor lineage. Analysis of differentially methylated regions of individual genes may have prognostic value as well as guide research toward nonsurgical therapeutic strategies. Pituitary epigenetics and DNA methylation analysis are rapidly growing areas of interest with the potential to shape the future of pituitary tumor diagnostics and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c91afb11897061f0dd790a1feb3aca845da9bfbb" target='_blank'>
              Current understanding of the role of DNA methylation in pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            Racheal Peterson, David J Cote, G. Zada
          </td>
          <td>2025-01-02</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction Ovarian cancer (OC) is the sixth most common malignancy in women and the poor 5-year survival emphasises the need for novel therapies. NK cells play an important role in the control of malignant disease but the nature of tumour-infiltrating and peripheral NK cells in OC remains unclear. Methods Using flow cytometric analysis, we studied the phenotype and function of NK cells in blood, primary tumour and metastatic tissue in 80 women with OC. The cell type contexture of metastatic OC tissue was explored utilising scRNAseq analysis, with a focus on portraying an immunogenic tumour microenvironment and determining the characteristics of a dysfunctional NK cell population. Results The proportion of peripheral NK cells was markedly elevated with a highly activated profile and increased cytotoxicity. In contrast, NK cell numbers in primary tumour and metastasis were substantially reduced, with downregulation of activatory receptors together with elevated PD-1 expression. scRNA-Seq identified 5 NK cell subpopulations along with increased exhausted and immature NK cells within tumour tissue compared to normal tissue. These features were attenuated following chemotherapy where higher levels of activated and cytotoxic NK cells associated with improved disease-free survival. Correlation of NK cell phenotype with clinical outcomes revealed high levels of DNAM-1 expression on tissue-localised and peripheral NK cells to be associated with reduced survival. Expression of PVR, the DNAM-1 ligand, was significantly increased on tumours and DNAM-1 mediated NK cell lysis of primary tumour tissue was observed in vitro. Discussion These findings reveal profound modulation of the tumour tissue and systemic profile of NK cells which likely contributes to the high rates of local progression and metastasis seen with OC. Immunotherapeutic approaches that overcome local immune suppression and enhance DNAM-1-targeted lysis of OC offer the potential to improve disease control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ec3263f862b7dea4d3ddaf31f9b8ccd7a6ea974" target='_blank'>
              The emergence of DNAM-1 as the facilitator of NK cell-mediated killing in ovarian cancer
              </a>
            </td>
          <td>
            R. Pounds, W. Croft, Hayden Pearce, Tasnia Hossain, Kavita Singh, J. Bálega, D. Jeevan, S. Sundar, S. Kehoe, J. Yap, Paul Moss, J. Zuo
          </td>
          <td>2025-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ac025963bbd7b574699ac753639c82b08e0f94a" target='_blank'>
              DNA methylation of ACADS promotes immunogenic cell death in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Ze Qian, Yifan Jiang, Yacong Wang, Yu Li, Lin Zhang, Xiaofeng Xu, Diyu Chen
          </td>
          <td>2025-01-12</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Transcriptional control of transgene expression can be linked to dynamic changes in cellular states if this is accompanied by differential expression of transcription factors (TFs). Synthetic promoters (SPs) designed to respond to the desired TFs can provide this regulation with compactness, specificity, and orthogonality. T cells display differentially expressed TFs according to the functional state. In solid tumors, the highly immunosuppressive TME and the chronic exposure to antigens lead to a progression of T cells from a functional to a dysfunctional state known as exhaustion (Tex), in which their power against cancer cells is strongly compromised. Importantly, this transition is accompanied by a marked increase of several TFs, among other factors, that drive targeted genetic programs. Strategies to detect and mitigate Tex are extremely needed. Here, we design SPs that respond to TFs differentially expressed in activated and exhausted T cells to enable new classifiers of the functional/dysfunctional states. We developed a library of over 80 SPs responsive to 7 TFs. The SPs showed broad strength of activation of reporter genes or immunomodulatory molecules in HEK293 and Jurkat T cell lines. Moreover, using a transfer learning strategy we show SPs strength predictability. By combining SPs responding to different TFs, we created Boolean logic gates and implemented a feed-forward design that was previously shown to reduce noise in the OFF-state. Finally, as proof of principle, we demonstrate the dynamic activation of the NR4A2-responsive SP according to the T cell state in primary human CD8+ T cells. Collectively we present a sensing platform that provides a versatile tool to study and monitor the dynamic changes occurring in T cells. In perspective the biosensors coupled to therapeutic genes can be used to reprogram the TME and reinvigorate the T cell anti-tumoral functionality, preventing or reverting the exhausted phenotype. Graphical Abstract Created with BioRender.com.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/396338870509bf01505cfe58f9f60b72b3d35505" target='_blank'>
              Design of novel synthetic promoters to tune gene expression in T cells
              </a>
            </td>
          <td>
            Giuliano Bonfá, Giovanna Martino, Assunta Sellitto, A. Rinaldi, F. Tedeschi, Fabio Caliendo, Loris Melchiorri, Daniela Perna, Freddie Starkey, Evangelos-Marios Nikolados, Filippo Menolascina, V. Siciliano
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024', '2025'],
    y: [3, 3, 9],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>