<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-04-21 09:27:44 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>31</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>67</td>
          <td>136</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>108</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>323</td>
          <td>136</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>283</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f03a05980baac4dbe599f109d605f3d1cea815" target='_blank'>
              EcDNA-borne PVT1 fusion stabilizes oncogenic mRNAs
              </a>
            </td>
          <td>
            Hyerim Yi, Shu Zhang, Jason Swinderman, Yanbo Wang, Vishnupriya Kanakaveti, King L. Hung, I. Wong, Suhas Srinivasan, Ellis J Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G Jones, J. Luebeck, Yanding Zhao, J. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Frances M. Liang, D. Felsher, Luke A. Gilbert, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a critical driver of cancer progression, contributing to tumour growth, evolution, and therapeutic resistance through oncogene amplification. Despite its significance, the replication of ecDNA remains poorly understood. In this study, we investigated the replication dynamics of ecDNA using high-resolution replication timing analysis (Repli-seq) and DNAscent, a method for measuring origin firing and replication fork movement based on ultra-long read Oxford Nanopore Sequencing, that we applied to both bulk DNA and to ecDNA isolated with FACS-based Isolation of Native ecDNA (FINE), a new method for isolating intact, chromatinised ecDNA without DNA or protein digestion. We demonstrate that ecDNA in the COLO 320DM colorectal cancer cell line exhibits largely asynchronous replication throughout the S phase, contrasting with the conserved replication timing of the corresponding normal linear chromosomal DNA in RPE-1 cells and the chromosomally reintegrated ecDNA in COLO 320HSR, which forms a homogeneously staining region. Replication origins on ecDNA are redistributed, and replication forks exhibit reduced velocity and increased stalling, particularly near the c-MYC oncogene. Under replication stress induced by hydroxyurea treatment, ecDNA replication is further compromised, leading to altered origin activation, reduced fork velocity and eventual ecDNA depletion from cells. Our findings reveal fundamental differences in the replication dynamics of ecDNA, providing insights that could inform the development of therapies targeting ecDNA-associated oncogene amplification in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc7a9c2ef16ae19f8f5b97c5088f5ea3ae1f2bd" target='_blank'>
              ecDNA replication is disorganised and vulnerable to replication stress
              </a>
            </td>
          <td>
            Jedrzej J. Jaworski, PL Pfuderer, Pawel Czyz, G. Petris, MA Boemo, Julian E. Sale
          </td>
          <td>2025-03-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db1a49566751259c29d9900a0683040a703d19a" target='_blank'>
              Selective depletion of cancer cells with extrachromosomal DNA via lentiviral infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D. Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) are commonly produced within the nucleus to drive genome dynamics and heterogeneity, enabling cancer cell evolution and adaptation. However, the mechanisms underlying ecDNA biogenesis remain poorly understood. Here using genome-wide CRISPR screening in human cells, we identified the BRCA1-A and the LIG4 complexes mediate ecDNA production. Following DNA fragmentation, the upstream BRCA1-A complex protects DNA ends from excessive resection, promoting end-joining for circularization. Conversely, the MRN complex, which mediates end resection and thus antagonizes the BRCA1-A complex, suppresses ecDNA formation. Downstream, LIG4 conservatively catalyzes ecDNA production in Drosophila and mammals, with patient tumor ecDNA harboring junctions marked by LIG4 activity. Notably, disrupting LIG4 or BRCA1-A in cancer cells impairs ecDNA-mediated adaptation, hindering resistance to both chemotherapy and targeted therapies. Together, our study reveals the roles of the LIG4 and BRCA1-A complexes in ecDNA biogenesis, and uncovers new therapeutic targets to block ecDNA-mediated adaptation for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65472598115e4d60921b909d76b9f2af808705a0" target='_blank'>
              BRCA1-A and LIG4 complexes mediate ecDNA biogenesis and cancer drug resistance
              </a>
            </td>
          <td>
            Oliver W. Chung, Shun Yao, Fu Yang, Ling Wang, Christian G Cerda-Smith, Haley M Hutchinson, Kris C. Wood, Weijia Su, Mustafa Khasraw, Lee Zou, Dale A. Ramsden, Z. Zhang
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Aneuploidy is observed as gains or losses of whole chromosomes or chromosome arms and is a common hallmark of cancer. Whereas models for the generation of aneuploidy in cancer invoke mitotic chromosome segregation errors, whole-arm losses might occur simply as a result of centromere breakage. We recently showed that elevated RNA Polymerase II (RNAPII) level over S-phase-dependent histone genes predicts rapid recurrence of human meningioma and is correlated with total whole-arm losses relative to gains. To explain this imbalance in arm losses over gains, we have proposed that histone overexpression at S-phase competes with the histone H3 variant CENP-A, resulting in centromere breaks and whole-arm losses. To test whether centromere breaks alone can drive aneuploidy, we ask whether total whole-arm aneuploids can predict outcome across different cancer types in large RNA and whole-genome sequencing databanks. We find that total whole-arm losses generally predict outcome, suggesting that centromere breakage is a major initiating factor leading to aneuploidy and the resulting changes in the selective landscape that drive most cancers. We also present evidence that centromere breakage alone is sufficient to account for whole-arm losses and gains, contrary to mitotic spindle error models for generation of aneuploidy. Our results suggest that therapeutic intervention targeting histone overexpression has the potential of reducing aneuploidy and slowing cancer progression. Significance Statement Gain or loss of whole chromosome arms following centromere breaks is frequent in cancer, but whether or not there is a common initiating event is unknown. Here we show that the total number of whole-arm losses predicts patient outcomes across cancer types, suggesting a causal relationship. This general excess of losses over gains is not predicted by mitotic error models of aneuploidy but rather suggests that centromere breaks themselves initiate whole-arm aneuploidies. Insofar as aneuploidy reshapes the selective landscapes that drive most cancers, our results have potential clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6556f74989cd6bd7e813ef9e417ef3b7531711e1" target='_blank'>
              Total whole-arm chromosome losses predict malignancy in human cancer
              </a>
            </td>
          <td>
            Ye Zheng, K. Ahmad, S. Henikoff
          </td>
          <td>2025-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Intratumour heterogeneity and phenotypic plasticity drive tumour progression and therapy resistance1,2. Oncogene dosage variation contributes to cell-state transitions and phenotypic heterogeneity3, thereby providing a substrate for somatic evolution. Nonetheless, the genetic mechanisms underlying phenotypic heterogeneity are still poorly understood. Here we show that extrachromosomal DNA (ecDNA) is a major source of high-level focal amplification in key oncogenes and a major contributor of MYC heterogeneity in pancreatic ductal adenocarcinoma (PDAC). We demonstrate that ecDNAs drive varying levels of MYC dosage, depending on their regulatory landscape, enabling cancer cells to rapidly and reversibly adapt to microenvironmental changes. In the absence of selective pressure, a high ecDNA copy number imposes a substantial fitness cost on PDAC cells. We also show that MYC dosage affects cell morphology and dependence of cancer cells on stromal niche factors. Our work provides a detailed analysis of ecDNAs in PDAC and describes a new genetic mechanism driving MYC heterogeneity in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db22625a85ea34b5a5adadae1847c161e783d66" target='_blank'>
              MYC ecDNA promotes intratumour heterogeneity and plasticity in PDAC
              </a>
            </td>
          <td>
            Elena Fiorini, A. Malinova, Daniel Schreyer, Davide Pasini, Michele Bevere, Giorgia Alessio, Diego Rosa, Sabrina D'Agosto, Luca Azzolin, Salvatore Milite, S. Andreani, F. Lupo, L. Veghini, Sonia Grimaldi, S. Pedron, Monica Castellucci, C. Nourse, R. Salvia, G. Malleo, A. Ruzzenente, A. Guglielmi, M. Milella, R. Lawlor, C. Luchini, A. Agostini, Carmine Carbone, Christian Pilarsky, A. Sottoriva, Aldo Scarpa, D. Tuveson, Peter Bailey, V. Corbo
          </td>
          <td>2025-03-12</td>
          <td>Nature</td>
          <td>1</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edc027322f110679344e9de5725b902edc43f13e" target='_blank'>
              Deregulated enhancer-promoter communication in cancer through altered nuclear architecture.
              </a>
            </td>
          <td>
            Isabelle Seufert, Claire Vargas, Sina Jasmin Wille, K. Rippe
          </td>
          <td>2025-04-12</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="MYC-driven (MYC+) cancers are aggressive and often fatal. MYC dysregulation is a key event in these cancers, but overexpression of MYC alone is not always enough to cause cancer. Plasmocytoma Variant Translocation 1 (PVT1), a long non-coding RNA (lncRNA) adjacent to MYC on chromosome 8 is a rearrangement hotspot in many MYC+ cancers. In addition to being co-amplified with MYC, the genomic rearrangement at PVT1 involves translocation, which has had obscure functional consequences. We report that translocation at the PVT1 locus cause asymmetric enrichment of 5’-PVT1 and loss of 3’-PVT1. Despite being classified as a non-coding RNA, the retained 5’ region of PVT1 generates a circular RNA (CircPVT1) that codes for the novel peptide we call Firefox (FFX). FFX augments AKT signaling and synergistically activates MYC and mTORC1 in these cells. Further, the 3’ end of PVT1, which is lost during the translocation, codes for a tumor-suppressing micropeptide we named as Honeybadger (HNB). We demonstrate that HNB interacts with KRAS and disrupts the activation of KRAS effectors. Loss of HNB leads to activation of RAS/MAPK signaling pathway, and enhances MYC stability by promoting phosphorylation of MYC at Ser62. These findings identify PVT1 as a critical node that synchronizes MYC, AKT, and RAS-MAPK activities in cancer. Our study thus identifies a key mechanism by which rearrangements at the PVT1 locus activate additional oncogenic pathways that synergize with MYC to exacerbate the aggressiveness of MYC+ cancers. This newfound understanding explains the poor prognosis associated with MYC+ cancers and offers potential therapeutic targets that could be leveraged in treatment strategies for these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e076955c4c71d3d95422db7f1fcfc733c5d61e3" target='_blank'>
              Synergistic RAS-MAPK and AKT Activation in MYC-Driven Tumors via Adjacent PVT1 Rearrangements
              </a>
            </td>
          <td>
            Ashutosh Tiwari, Utkarsha Paithane, Jordan Friedlein, Kojiro Tashiro, Olivier Saulnier, Karina Barbosa, Quang Trinh, Bryan Hall, Shrawantee Saha, Aditi Soni, Takuma Nakashima, Andrey Bobkov, Lynn Fujimoto, Rabi Murad, Svetlana Maurya, Mayank Saraswat, Shahab Sarmashghi, Joshua T. Lange, Sihan Wu, M. Masihi, Srija Ghosh, Gazal Hemmati, Owen S. Chapman, Liam D Hendrikse, Brian James, J. Luebeck, Tanja Eisemann, Theophilos Tzaridis, Deepak Rohila, Robyn Leary, Jyotika Varshney, Badrinath R. Konety, S. Dehm, Yasuhiko Kawakami, R. Beroukhim, D. Largaespada, Lincoln Stein, Lukas Chavez, Hiromichi Suzuki, William A. Weiss, Jianhua Zhao, Aniruddha J. Deshpande, R. Wechsler-Reya, Michael D Taylor, Anindya Bagchi
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="Understanding lung cancer evolution can identify tools for intercepting its growth. In a landscape analysis of 1024 lung adenocarcinomas (LUAD) with deep whole-genome sequencing integrated with multiomic data, we identified 542 LUAD that displayed diverse clonal architecture. In this group, we observed an interplay between mobile elements, endogenous and exogenous mutational processes, distinct driver genes, and epidemiological features. Our results revealed divergent evolutionary trajectories based on tobacco smoking exposure, ancestry, and sex. LUAD from smokers showed an abundance of tobacco-related C:G>A:T driver mutations in KRAS plus short subclonal diversification. LUAD in never smokers showed early occurrence of copy number alterations and EGFR mutations associated with SBS5 and SBS40a mutational signatures. Tumors harboring EGFR mutations exhibited long latency, particularly in females of European-ancestry (EU_N). In EU_N, EGFR mutations preceded the occurrence of other driver genes, including TP53 and RBM10. Tumors from Asian never smokers showed a short clonal evolution and presented with heterogeneous repetitive patterns for the inferred mutational order. Importantly, we found that the mutational signature ID2 is a marker of a previously unrecognized mechanism for LUAD evolution. Tumors with ID2 showed short latency and high L1 retrotransposon activity linked to L1 promoter demethylation. These tumors exhibited an aggressive phenotype, characterized by increased genomic instability, elevated hypoxia scores, low burden of neoantigens, propensity to develop metastasis, and poor overall survival. Re-activated L1 retrotransposition-induced mutagenesis can contribute to the origin of the mutational signature ID2, including through the regulation of the transcriptional factor ZNF695, a member of the KZFP family. The complex nature of LUAD evolution creates both challenges and opportunities for screening and treatment plans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4fa308a5c703341cd700c368469f4a3b71892c1" target='_blank'>
              Deciphering lung adenocarcinoma evolution and the role of LINE-1 retrotransposition
              </a>
            </td>
          <td>
            Tongwu Zhang, Wei Zhao, Christopher Wirth, M. Díaz-Gay, Jinhu Yin, M. Cecati, F. Marchegiani, P. Hoang, Charles Leduc, M. Baine, William D Travis, L. Sholl, Philippe Joubert, Jian Sang, John McElderry, Alyssa Klein, Azhar Khandekar, Caleb Hartman, J. Rosenbaum, Frank Colon-Matos, Mona Miraftab, Monjoy Saha, Olivia W Lee, Kristine Jones, N. E. Caporaso, Maria Pik Wong, Kin Chung Leung, Chao (Agnes) Hsiung, Chih-Yi Chen, E. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, D. Mates, Sasa Milosavljevic, Milan Savic, Y. Bossé, Bonnie E. Gould Rothberg, David C. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, Robert J Homer, S. Yang, A. Pesatori, D. Consonni, Lixing Yang, Bin Zhu, Jianxin Shi, Kevin Brown, Nathaniel Rothman, Stephen J. Chanock, Ludmil B. Alexandrov, Jiyeon Choi, Maurizio Cardelli, Qing Lan, Martin A. Nowak, David C Wedge, Marion Landi
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains an incurable disease primarily due to the emergence of drug resistance, and the underlying mechanisms remain unclear. Extrachromosomal circular DNAs (eccDNAs) are prevalent in cancer genomes of both coding and non-coding regions. However, the role of non-coding eccDNA regions that serve as enhancers has been largely overlooked. Here, genome-wide profiling of serum eccDNAs from donors and MM patients who responded well or poorly to bortezomib-lenalidomide-dexamethasone (VRd) therapy is characterized. A high copy number of eccDNA ANKRD28 (eccANKRD28) predicts poor therapy response and prognosis but enhanced transcriptional activity. Established VRd-resistant MM cell lines exhibit a higher abundance of eccANKRD28, and CRISPR/Cas9-mediated elevation of eccANKRD28 desensitizes bortezomib and lenalidomide treatment both in vitro and in vivo. Integrated multi-omics analysis (H3K27ac ChIP-seq, scRNA-seq, scATAC-seq, CUT&Tag, et al.) identifies eccANKRD28 as an active enhancer involved in drug resistance driven by the key transcription factor, POU class 2 homeobox 2 (POU2F2). POU2F2 interacts with sequence-specific eccANKRD28 as well as RUNX1 and RUNX2 motifs to form the protein complex, which activates the promoter of oncogenes, including IRF4, JUNB, IKZF3, RUNX3, and BCL2. This study elucidates the potential transcriptional network of enhancer eccANKRD28 in MM drug resistance from a previously unrecognized epigenetic perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea6e610cf350e732841fb77da78765020fd07704" target='_blank'>
              Enhancer Extrachromosomal Circular DNA ANKRD28 Elicits Drug Resistance via POU2F2-Mediated Transcriptional Network in Multiple Myeloma.
              </a>
            </td>
          <td>
            Binzhen Chen, Jia Liu, Yaoxin Zhang, Changming Shi, Di Zhu, Guoqiang Zhang, Fei Xiao, L. Zhong, Minyue Zhang, Lai Guan Ng, Honghui Huang, Tingting Lu, Jian Hou
          </td>
          <td>2025-04-01</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="As a well-known cancer risk region, the 8q24 locus is frequently amplified in a variety of solid tumors. Here we identify a pseudogene-derived oncogenic lncRNA, ZNF252P, which is upregulated in a variety of cancer types by copy number gain as well as c-Myc-mediated transcriptional activation. Mechanistically, ZNF252P binds and drives “phase separation” of HNRNPK and ILF3 protein in the nucleus and cytoplasm, respectively, to transcriptionally and posttranscriptionally activate c-Myc, thus forming a c-Myc/ZNF252P/c-Myc positive feedback loop. These findings expand the understanding of the relationship between genomic instability in the 8q24 region and tumorigenesis and clarify a regulatory mechanism involved in transcription and posttranscription from the perspective of RNA-mediated nuclear and cytoplasmic protein phase separation, which sheds light on the dialogue with the driver oncogene c-Myc. The pivotal regulatory axis of ZNF252P/c-Myc has potential as a promising biomarker and therapeutic target in cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7394db232e99965882ed6981c1decbce7cc434ab" target='_blank'>
              8q24 derived ZNF252P promotes tumorigenesis by driving phase separation to activate c-Myc mediated feedback loop
              </a>
            </td>
          <td>
            Tianyu Qu, Chang Zhang, Xiyi Lu, Jiali Dai, Xuezhi He, Wei Li, Liang Han, Dandan Yin, Erbao Zhang
          </td>
          <td>2025-02-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ad9d941478590c32e5cf0763f3872cb1fd2c287" target='_blank'>
              Satellite DNA Amplification in Advanced Prostate Cancer Is Largely Independent From Euchromatic and Oncogene Amplicons.
              </a>
            </td>
          <td>
            Anja Weise, A. Ornellas, Gilda Alves, Constanze Pentzold, Jenny Holler, Melanie Wolter, Elena Jamali, Bernhard Theis, Thomas Liehr
          </td>
          <td>2025-03-17</td>
          <td>The journal of histochemistry and cytochemistry : official journal of the Histochemistry Society</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) has been found in most types of human cancers, and ecDNA incorporating viral genomes has recently been described, specifically in human papillomavirus (HPV)-mediated oropharyngeal cancer (OPC). However, the molecular mechanisms of human-viral hybrid ecDNA (hybrid ecDNA) for carcinogenesis remains elusive. We characterize the epigenetic status of hybrid ecDNA using HPVOPC cell lines and patient-derived tumor xenografts, identifying HPV oncogenes E6/E7 in hybrid ecDNA are flanked by previously unrecognized somatic DNA enhancers and HPV L1 enhancers, with strong cis-interactions. Targeting of these enhancers by clustered regularly interspaced short palindromic repeats interference or hybrid ecDNA by bromodomain and extra-terminal inhibitor reduces E6/E7 expression, and significantly inhibites in vitro and/or in vivo growth only in ecDNA(+) models. HPV DNA in hybrid ecDNA structures are associated with previously unrecognized somatic and HPV enhancers in hybrid ecDNA that drive HPV ongogene expression and carcinogenesis, and can be targeted with ecDNA disrupting therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76981a5b85fcdfc01f05694a03b59cd41140b0ab" target='_blank'>
              Inhibition of human-HPV hybrid ecDNA enhancers reduces oncogene expression and tumor growth in oropharyngeal cancer
              </a>
            </td>
          <td>
            Takuya Nakagawa, J. Luebeck, Kaiyuan Zhu, Joshua T. Lange, Roman Sasik, Chad Phillips, S. Sadat, Sara Javadzadeh, Qian Yang, Abdula Monther, Santiago Fassardi, Allen Wang, Kersi Pestonjamasp, Brin Rosenthal, Kathleen M Fisch, P. Mischel, V. Bafna, Joseph A Califano
          </td>
          <td>2025-03-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Chromosome territories (CTs) are intricately organized and regulated within the nucleus. Despite remarkable advances in our understanding of genome packaging and gene expression, the interplay among CTs, pairing of parental homologous chromosomes, and genome function during development remains elusive. Here, we employ an Oligopaints-based high-resolution imaging approach to examine variable CT organization in single nuclei during the developmental process of zygotic genome activation. We reveal large-scale chromosome changes with extensive homolog pairing at the whole-chromosome level that decreases locally due to spatial variability in chromosome conformations. In the absence of one homolog copy, the dynamics of CT compaction and RNA polymerase II recruitment are supported by transcriptional changes in haploid embryos. Finally, global inhibition of transcription results in decreased CT opening and no significant impact on CT pairing levels. These findings enhance our understanding of parental genome folding and regulation, which may inform strategies for chromosome-based diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/461f9f862c0bc9f9b6c4530eae22c13287308545" target='_blank'>
              The functional organization of chromosome territories in single nuclei during zygotic genome activation
              </a>
            </td>
          <td>
            Akshada Shankar Ganesh, Taylor M. Orban, Romir Raj, Peter I. Fatzinger, Anna Johnson, Sean M. Riccard, Akhmed Zhanaidarov, Mayu Inaba, Jelena Erceg
          </td>
          <td>2025-04-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cancer development is characterized by chromosomal instability, manifesting in frequent occurrences of different genomic alteration mechanisms ranging in extent and impact. Mathematical modeling can help evaluate the role of each mutational process during tumor progression, however existing frameworks can only capture certain aspects of chromosomal instability (CIN). We present CINner, a mathematical framework for modeling genomic diversity and selection during tumor evolution. The main advantage of CINner is its flexibility to incorporate many genomic events that directly impact cellular fitness, from driver gene mutations to copy number alterations (CNAs), including focal amplifications and deletions, missegregations and whole-genome duplication (WGD). We apply CINner to find chromosome-arm selection parameters that drive tumorigenesis in the absence of WGD in chromosomally stable cancer types from the Pan-Cancer Analysis of Whole Genomes (PCAWG, n=718). We found that the selection parameters predict WGD prevalence among different chromosomally unstable tumors, hinting that the selective advantage of WGD cells hinges on their tolerance for aneuploidy and escape from nullisomy. Analysis of inference results using CINner across cancer types in The Cancer Genome Atlas (n=8207) further reveals that the inferred selection parameters reflect the bias between tumor suppressor genes and oncogenes on specific genomic regions. Direct application of CINner to model the WGD proportion and fraction of genome altered (FGA) in PCAWG uncovers the increase in CNA probabilities associated with WGD in each cancer type. CINner can also be utilized to study chromosomally stable cancer types, by applying a selection model based on driver gene mutations and focal amplifications or deletions (chronic lymphocytic leukemia in PCAWG, n=95). Finally, we used CINner to analyze the impact of CNA probabilities, chromosome selection parameters, tumor growth dynamics and population size on cancer fitness and heterogeneity. We expect that CINner will provide a powerful modeling tool for the oncology community to quantify the impact of newly uncovered genomic alteration mechanisms on shaping tumor progression and adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a271cd04acf8979b5f4cf725c9c1e047db0d2529" target='_blank'>
              CINner: Modeling and simulation of chromosomal instability in cancer at single-cell resolution
              </a>
            </td>
          <td>
            K. Dinh, I. Vázquez-García, Andrew Chan, Rhea Malhotra, A. Weiner, Andrew McPherson, Simon Tavaré
          </td>
          <td>2025-04-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b492f14ec0b02275c3aedf5d1fc7e017eb0ec151" target='_blank'>
              Tracking Replicating HPV Genomes in Proliferating Keratinocytes
              </a>
            </td>
          <td>
            Jonathan R. Shin, Ivan Avilov, Mario Schelhaas, Franck Gallardo, Alison A. McBride
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Recent research reveals that eukaryotic genomes form circular DNA from all parts of their genome, some large enough to carry whole genes. In organisms like yeast and in human cancers, it is often observed that extrachromosomal circular DNA (eccDNA) benefits the individual cell by providing resources for rapid cellular growth. However, our comprehension of eccDNA remains incomplete, primarily due to their transient nature. Early studies suggest they arise when DNA breaks and is subsequently repaired incorrectly. In this review, we provide an overview of the evidence for molecular mechanisms that lead to eccDNA formation in human cancers and yeast, focusing on nonhomologous end joining, alternative end joining, and homologous recombination repair pathways. Furthermore, we present hypotheses in the form of molecular eccDNA formation models and consider cellular conditions which may affect eccDNA generation. Finally, we discuss the framework for future experimental evidence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd036a4aeb41767279a21c8043d9cb8d21781a98" target='_blank'>
              Molecular mechanisms of extrachromosomal circular DNA formation
              </a>
            </td>
          <td>
            Rasmus A B Eugen-Olsen, J. M. Hariprakash, V. Oestergaard, Birgitte Regenberg
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Kaposi’s sarcoma-associated herpesvirus (KSHV) genome contains a terminal repeats (TR) sequence. Previous studies demonstrated that KSHV TR functions as a gene enhancer for inducible lytic gene promoters. Gene enhancers anchor bromodomain-containing protein 4 (BRD4) at specific genomic region, where BRD4 interacts flexibly with transcription-related proteins through its intrinsically disordered domain and exerts transcription regulatory function. Here, we generated recombinant KSHV with reduced TR copy numbers and studied BRD4 recruitment and its contributions to the inducible promoter activation. Reducing the TR copy numbers from 21 (TR21) to 5 (TR5) strongly attenuated viral gene expression during de novo infection and impaired reactivation. The EF1α promoter encoded in the KSHV BAC backbone also showed reduced promoter activity, suggesting a global attenuation of transcription activity within TR5 latent episomes. Isolation of reactivating cells confirmed that the reduced inducible gene transcription from TR-shortened DNA template and is mediated by decreased efficacies of BRD4 recruitment to viral gene promoters. Separating the reactivating iSLK cell population from non-responders showed that reactivatable iSLK cells harbored larger LANA nuclear bodies (NBs) compared to non-responders. The cells with larger LANA NBs, either due to prior transcription activation or TR copy number, supported KSHV reactivation more efficiently than those with smaller LANA NBs. With auxin-inducible LANA degradation, we confirmed that LANA is responsible for BRD4 occupancies on latent chromatin. Finally, with purified fluorescence-tagged proteins, we demonstrated that BRD4 is required for LANA to form liquid-liquid phase-separated dots. The inclusion of TR DNA fragments further facilitated the formation of larger BRD4-containing LLPS in the presence of LANA, similar to the “cellular enhancer dot” formed by transcription factor-DNA bindings. These results suggest that LANA binding to TR establishes an enhancer domain for infected KSHV episomes. The strength of this enhancer, regulated by TR length or transcription memory, determines the outcome of KSHV replication. Importance Gene enhancers are genomic domains that regulate frequency and duration of transcription burst at gene promoters, with BRD4 playing a critical role in their enhancer functions. KSHV latent mini-chromosome also contains an enhancer domain made with multiple copies of 801 bp identical repeat DNA fragments, terminal repeats. Here, we utilized manipulable mini-scale chromatins with convenient inducible KSHV reactivation to systematically examine the association between enhancer strength and the outcome of inducible promoter activation. This study illustrated the amount of BRD4 recruitment at the enhancer associated with frequencies of BRD4 distribution to the inducible promoters during KSHV reactivation and, therefore, KSHV lytic replication. Recruitment of BRD4 to the TR is specifically regulated by KSHV latent protein, LANA. KSHV evolves clever enhancer elements designed to be regulated by the KSHV own latent protein, LANA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41ac4dd8639934c73f419b670737fe670abf6ad3" target='_blank'>
              Studies on Gene Enhancer with KSHV mini-chromatin
              </a>
            </td>
          <td>
            Tomoki Inagaki, Ashish Kumar, Kang-Hsin Wang, Somayeh Komaki, Jonna M. Espera, Christopher S. A. Bautista, Ken-ichi Nakajima, Chie Izumiya, Y. Izumiya
          </td>
          <td>2025-03-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Chromosomal rearrangements, such as translocations, deletions, and inversions, underlie numerous genetic diseases and cancers, yet precise engineering of these rearrangements remains challenging. Here, we present a CRISPR-based homologous recombination-mediated rearrangement (HRMR) strategy that leverages homologous donor templates to align and repair broken chromosome ends. HRMR improves efficiency by approximately 80-fold compared to non-homologous end joining, achieving over 95% homologous recombination. Validated across multiple loci and cell lines, HRMR enables efficient and accurate chromosomal rearrangements. Live-cell imaging reveals that homologous donors mediate chromosome end proximity, enhancing rearrangement efficiency. Thus, HRMR provides a powerful tool for disease modeling, chromosomal biology, and therapeutic applications. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03523-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7170c44494d1fc7e8ae358052e6bacf3d4c1c009" target='_blank'>
              Recruitment and rejoining of remote double-strand DNA breaks for enhanced and precise chromosome editing
              </a>
            </td>
          <td>
            Mingyao Wang, Pengchong Fu, Ziheng Chen, Xiangnan Wang, Hanhui Ma, Xuedi Zhang, Guanjun Gao
          </td>
          <td>2025-03-11</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdd747492529d370113dee3945f74bf5ec74e07f" target='_blank'>
              High levels of DNA replication initiation factors indicate ATRi sensitivity via excessive origin firing
              </a>
            </td>
          <td>
            A. Lumeau, PL Pfuderer, S. De Angelis, JA Scarth, MA Guscott, N. Shaikh, FB Copley, H. Gerdes, Emilie L Alard, PR Cutillas, F. Mardakheh, MA Boemo, JV Forment, SE McClelland
          </td>
          <td>2025-02-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9124f513a579068358157bc6335b2150cb3dde2" target='_blank'>
              Targeting processive transcription for Myc-driven circuitry in medulloblastoma
              </a>
            </td>
          <td>
            Lays Martin Sobral, Faye Walker, Krishnakumar Madhavan, Elizabeth Janko, Sahiti Donthula, Ilango Balakrishnan, Dong Wang, Angela M Pierce, Mary M. Haag, Billie J. Carstens, Natalie J Serkova, Nicholas K Foreman, S. Venkataraman, Bethany L. Veo, Rajeev Vibhakar, Nathan A Dahl
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3ffdab6d59ce2f03de66a13a4efd90a8a827d6" target='_blank'>
              Chromoplexy: A Pathway to Genomic Complexity and Cancer Development
              </a>
            </td>
          <td>
            Franck Pellestor, Benjamin Ganne, Jean Baptiste Gaillard, V. Gatinois
          </td>
          <td>2025-04-18</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db41c8c1cf9654af3ad975bf6a786aaf0b60bc69" target='_blank'>
              Somatic repetitive element insertions Define New Biomarkers in Pan-cancer genome
              </a>
            </td>
          <td>
            Lin Xia, Kailing Tu, Huan Wang, Xuyan Liu, Yahui Liu, Qilin Zhang, Ziting Feng, Chunyan Yu, Maolin Li, Yan Huang, Jing Wang, Ranlei Wei, Tianfu Zeng, Xuenan Pi, Chen Ling, Qianwen Zhang, Yixin Qiao, Shengzhong Hou, Zhuoyuan Zhang, Ailin Wei, Haotian Liao, Wenhao Guo, Jifeng Liu, Zhong Wu, Zhoufeng Wang, Ang Li, Weimin Li, Dan Xie
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/632a6b07fed31db9bd0bc2462cc1bb896521dfbb" target='_blank'>
              Timing and clustering co-occurring genome amplifications in cancers
              </a>
            </td>
          <td>
            Sara Cocomello, Giovanni Santacatterina, R. Bergamin, Alice Antonello, G. Caravagna
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) presents a promising target for cancer therapy; however, its spatial-temporal diversity and influence on tumor evolution and the immune microenvironment remain largely unclear. We apply computational methods to analyze ecDNA from whole-genome sequencing data of 595 urothelial carcinoma (UC) patients. We demonstrate that ecDNA drives clonal evolution through structural rearrangements during malignant transformation and recurrence of UC. This supports a model wherein tumors evolve via the selective expansion of ecDNA-bearing cells. Through multi-regional sampling of tumors, we demonstrate that ecDNA contributes to the evolution of multifocality and increased intratumoral heterogeneity. EcDNA is present in 36% of UC tumors and correlates with an immunosuppressive phenotype and poor prognosis. Single-cell RNA sequencing analyses reveal that ecDNA+ malignant cells exhibit diminished expression of major histocompatibility complex class I molecules, enabling them to evade T-cell immunity. Finally, we show that sequencing of urinary sediment-derived DNA has excellent specificity in detecting ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27eaa871f09220399d901baadcfefa6c53b148cd" target='_blank'>
              Spatial-Temporal Diversity of Extrachromosomal DNA Shapes Urothelial Carcinoma Evolution and Tumor-Immune Microenvironment.
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, Huiying Tao, Yanfen Zhu, Xiaolong Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez-Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xi Xiang, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, J. Luebeck, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-03-11</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Traditionally, the evolutionary perspective of cancer has been understood as gradual alterations in passenger/driver genes that lead to branching phylogeny. However, in cases of prostate adenocarcinoma and kidney renal cell carcinoma, macroevolutionary landmarks like chromoplexy and chromothripsis are frequently observed. Unfortunately, short-read sequencing techniques often miss these significant macroevolutionary changes, which involve multiple translocations and deletions at the chromosomal level. To resolve such genomic dark matters, we provided high-fidelity long-read sequencing data (78–92 Gb of ~Q30 reads) of six genitourinary tumour cell lines (one benign kidney tumour and two kidney and three prostate cancers). Based on these data, we obtained 12 high-quality, partially phased genome assemblies (Contig N50 1.85–29.01 Mb; longest contig 2.02–171.62 Mb), graph-based pan-genome variant sets (11.57 M variants including 60 K structural variants), and 5-methylcytosine sites (14.68%–27.05% of the CpG sites). We also identified several severe chromosome aberration events, which would result from chromosome break and fusion events. Our cancer genome assemblies will provide unprecedented resolution to understand cancer genome instability and chromosomal aberration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39aabbe27003a6111c9e819cce1949d58b416548" target='_blank'>
              Genome assembly resources of genitourinary cancers for chromosomal aberration at the single nucleotide level
              </a>
            </td>
          <td>
            Hyunho Han, H. Lee, Min Gyu Kim, Yoo Sub Shin, Jin Soo Chung, Jun Kim
          </td>
          <td>2025-04-01</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, the effects of non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, on tetraploid (4N) cancer cell physiology are less understood. Here, we show that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. We find that small size is associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD. Statement of Significance We report that WGD generates tetraploid cancer cells that vary in size, with larger cells displaying high chromosomal instability and smaller cells exhibiting high fitness and tumorigenicity. Furthermore, WGD status and cancer cell nuclear size in human tumors correlated with patient survival, demonstrating the clinical relevance of this association.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49f68f3dcdb4dd414fdeb5e81fa097ba1603de66" target='_blank'>
              Cell and nuclear size are associated with chromosomal instability and tumorigenicity in cancer cells that undergo whole genome doubling
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney Huth, Daniella McCausland, Ron Saad, Nazia Bano, Tran Chau, Megan Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, Uri Ben-David, Daniela Cimini
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Structural variants (SVs) are increasingly recognized as important contributors to oncogenesis through their effects on 3D genome folding. Recent advances in whole-genome sequencing have enabled large-scale profiling of SVs across diverse tumors, yet experimental characterization of their individual impact on genome folding remains infeasible. Here, we leveraged a convolutional neural network, Akita, to predict disruptions in genome folding caused by somatic SVs identified in 61 tumor types from the Children’s Brain Tumor Network dataset. Our analysis reveals significant variability in SV-induced disruptions across tumor types, with the most disruptive SVs coming from lymphomas and sarcomas, metastatic tumors, and germline cell tumors. Dimensionality reduction of disruption scores identified five recurrently disrupted regions enriched for high-impact SVs across multiple tumors. Some of these regions are highly disrupted despite not being highly mutated, and harbor tumor-associated genes and transcriptional regulators. To further interpret the functional relevance of high-scoring SVs, we integrated epigenetic data and developed a modified Activity-by-Contact scoring approach to prioritize SVs with disrupted genome contacts at active enhancers. This method highlighted highly disruptive SVs near key oncogenes, as well as novel candidate loci potentially implicated in tumorigenesis. These findings highlight the utility of machine learning for identifying novel SVs, loci, and genetic mechanisms contributing to pediatric cancers. This framework provides a foundation for future studies linking SV-driven regulatory changes to cancer pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22e24a79ed206cbada1bc6b51069582d272e00ef" target='_blank'>
              Machine learning-predicted chromatin organization landscape across pediatric tumors
              </a>
            </td>
          <td>
            Ketrin Gjoni, Shu Zhang, Rachel E. Yan, Bo Zhang, Daniel Miller, Adam Resnick, Nadia Dahmane, K. S. Pollard
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8033a50256e64b94ba359a06046d4ac24e218fe" target='_blank'>
              Targeting SUV4-20H epigenetic enzymes as therapeutic strategy for enhancing topoisomerase II poisoning in prostate cancer
              </a>
            </td>
          <td>
            Fatima Alhourani, Marine Tauziet, Julie Patouillard, Benoit Miotto, Véronique Baldin, Aurélie Gennetier, Simon George, Xavier Miahle, Cyril Ribeyre, Philippe Pourquier, Florence M. Cammas, A. Torro, Diego Tosi, Céline Gongora, Mona Dergham, Raghida Abou-Merhi, Eric Julien
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Somatic mobilization of LINE-1 (L1) has been implicated in cancer etiology. We analyzed a recent TCGA data release comprised of nearly 5000 pan-cancer paired tumor-normal whole-genome sequencing (WGS) samples and ~9000 tumor RNA samples. We developed TotalReCall an improved algorithm and pipeline for detection of L1 retrotransposition (RT), finding high correlation between L1 expression and “RT burden” per sample. Furthermore, we mathematically model the dual regulatory roles of p53, where mutations in TP53 disrupt regulation of both L1 expression and retrotransposition. We found those with Li-Fraumeni Syndrome (LFS) heritable TP53 pathogenic and likely pathogenic variants bear similarly high L1 activity compared to matched cancers from patients without LFS, suggesting this population be considered in attempts to target L1 therapeutically. Due to improved sensitivity, we detect over 10 genes beyond TP53 whose mutations correlate with L1, including ATRX, suggesting other, potentially targetable, mechanisms underlying L1 regulation in cancer remain to be discovered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c16a8e22a8e26a1832c9dc60f1f0a97f3c5163" target='_blank'>
              Pan-cancer multi-omic model of LINE-1 activity reveals locus heterogeneity of retrotransposition efficiency
              </a>
            </td>
          <td>
            Alexander Solovyov, Julie M. Behr, D. Hoyos, Eric Banks, Alexander W Drong, Bryan Thornlow, Jimmy Z Zhong, Enrique Garcia-Rivera, Wilson McKerrow, Chong Chu, Cédric Arisdakessian, Dennis M Zaller, J. Kamihara, Liyang Diao, M. Fromer, Benjamin D. Greenbaum
          </td>
          <td>2025-02-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) has potential in tumor diagnosis, particularly for improving diagnostic accuracy and early cancer detection; however, many challenges remain in its application to clinical practice. Methods We conducted a Circle‐Seq analysis on clinical samples at different stages of colorectal cancer progression to examine the dynamic changes of eccDNA during the progression of colorectal cancer. We used breakpoint-specific PCR to verify candidate eccDNAs identified by Circle‐Seq. The results were further validated using the AOM/DSS-induced colorectal cancer model. Results There was an increase in the abundance of eccDNA with the progression of colorectal cancer. The genes associated with these eccDNA molecules were primarily related to signaling pathways involved in tumor development and metastasis. Our analysis also revealed that eccDNA abundance positively correlates with gene expression, and eccDNA derived from specific genes has potential value for the early diagnosis of tumors. Conclusions This study revealed a connection between eccDNA and colorectal cancer progression and highlights the clinical potential of eccDNA for the early diagnosis of colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01164-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3999071e24587252e2469121996c435197d05b" target='_blank'>
              Extrachromosomal circular DNA as a novel biomarker for the progression of colorectal cancer
              </a>
            </td>
          <td>
            Quanpeng Qiu, Yi Ding, Xiaolong Guo, Jing Han, Jiaqi Zhang, Yaping Liu, Junjun She, Yinnan Chen
          </td>
          <td>2025-04-01</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Identification of new therapeutic targets in hepatocellular carcinoma (HCC) remains critical. Chromatin regulating complexes are frequently mutated or aberrantly expressed in HCC, suggesting dysregulation of chromatin environments is a key feature driving liver cancer. To investigate whether the altered chromatin state in HCC cells could be targeted, we designed and utilized an epigenome-focused CRISPR library that targets genes involved in chromatin regulation. This focused approach allowed us to test multiple HCC cell lines in both 2D and 3D growth conditions, which revealed striking differences in the essentiality of genes involved in ubiquitination and multiple chromatin regulators vital for HCC cell survival in 2D but whose loss promoted growth in 3D. We found the core subunits of the menin-MLL1 complex among the strongest essential genes for HCC survival in all screens and thoroughly characterized the mechanism through which the menin-MLL1 complex promotes HCC cell growth. Inhibition of the menin-MLL1 interaction led to global changes in occupancy of the complex with concomitant decreases in H3K4me3 and expression of genes involved in PI3K/AKT/mTOR signaling pathway. Menin inhibition affected chromatin accessibility in HCC cells, revealing that increased chromatin accessibility at sites not bound by menin-MLL1 was associated with the recruitment of the pioneer transcription factor complex NF-Y. A CRISPR/Cas9 screen of chromatin regulators in the presence of menin inhibitor SNDX-5613 revealed a significantly increased cell death when combined with NFYB knockout. Together these data show that menin-MLL1 is necessary for HCC cell survival and cooperates with NF-Y to regulate oncogenic gene transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cb9b8fe99339ff834a043435b391f2d743bc86f" target='_blank'>
              Menin-MLL1 complex cooperates with NF-Y to promote HCC survival
              </a>
            </td>
          <td>
            Margarita Dzama-Karels, Peyton C. Kuhlers, Mallory Sokolowski, Jacqueline A. Brinkman, John P. Morris, Jesse R. Raab
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd1c02bf99dc94856478bd198e131107c59bd8c" target='_blank'>
              KDM5 demethylases suppress R-loop-mediated “viral mimicry” and DNA damage in breast cancer cells
              </a>
            </td>
          <td>
            Lena Lau, Kurt W Henderson, Ahu Turkoz, Sara Linker, Dörte Schlesinger, Brad Townsley, Brian Egan, Shoba Ragunathan, Robert Rollins, Xianju Bi, Zhijian J. Chen, Oleg Brodsky, Clifford Restaino, Murali Gururajan, Kristen Jensen-Pergakes, A. Malarstig, Chames Kermi, Paul C Moore, Marie Classon
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Significance Chemotherapeutics effectively kill cancer cells but can also induce mutations in surviving cells, leading to therapy resistance and secondary malignancies. We analyzed the DNA rearrangements caused by doxorubicin in murine HER2+ breast cancer organoids, focusing on genomic structural variants (SVs). Using single-cell sequencing, we revealed extensive therapy-induced SVs and genomic instability in a subset of posttreatment cells. Additionally, by advancing computational methods to analyze nucleosome occupancy (NO) in different cell types of the mammary lineage, we link SVs to gene expression changes that could potentially lead to breast cancer progression. This research underscores the need for strategies to minimize drug-induced genomic damage to improve treatment efficacy and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95047a549d97c358dede6e2ed75d74511cbd0c06" target='_blank'>
              Structural variant and nucleosome occupancy dynamics postchemotherapy in a HER2+ breast cancer organoid model
              </a>
            </td>
          <td>
            Maja Starostecka, Hyobin Jeong, Patrick Hasenfeld, Eva Benito-Garagorri, Tania Christiansen, Catherine Stober Brasseur, Maise Gomes Queiroz, Marta Garcia Montero, Martin Jechlinger, J. Korbel
          </td>
          <td>2025-02-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="In eukaryotic cells, gene-distal regulatory elements (REs) facilitate long-range gene regulation, ensuring cell type-specific transcriptional programs. This mechanism is frequently disrupted in cancer, often driven by transcription factors (TFs) that serve as targets for cancer therapy. However, targeting these TFs can lead to acquired resistance mechanisms that are not fully understood. We demonstrate that mesothelioma cancer cells, dependent on the oncogenic driver TF family TEAD, develop resistance to a pan-TEAD inhibitor and revert to an evolutionarily ancient, promoter-centric gene regulatory mechanism to recover gene expression following TEAD inhibition. Base-pair-resolution 3D chromatin conformation mapping reveals that RE-promoter interactions are disrupted in resistant cells, despite epigenetic and transcriptomic recovery. Mechanistically, in resistant cells, TF complexes, including resistance-specific FOSL1 and KLF4, preferentially bind and enhance promoter activity to recover gene expression, rendering distal REs dispensable. Our findings highlight promoter elements and promoter-specific TFs as potential therapeutic targets using a model of drug-resistant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fe445b4115593745bb7783131bb805e9f85aeb" target='_blank'>
              Promoter-centric gene regulation in drug-resistant cancer
              </a>
            </td>
          <td>
            Vasumathi Kameswaran, Sayantanee Paul, Daniel Le, Jonathan Hoover, Luke Y. Zhao, Alissa D. Guarnaccia, Thijs J. Hagenbeek, Jessica M. Lund, Ana Xavier-Magalhães, Minyi Shi, Julia Lau, Marco De Simone, Yuxin Liang, Anwesha Dey, Z. Modrusan, Bence Daniel
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="N6-methyladenosine (m6A), the most abundant internal RNA modification in humans, regulates most aspects of RNA processing. Prostate cancer is characterized by widespread transcriptomic dysregulation; therefore, we characterized the m6A landscape of 162 localized prostate tumors with matched DNA, RNA and protein profiling. m6A abundance varied dramatically across tumors, with global patterns emerging via complex germline–somatic cooperative regulation. Individual germline polymorphisms regulated m6A abundance, cooperating with somatic mutation of cancer driver genes and m6A regulators. The resulting complex patterns were associated with prognostic clinical features and established the biomarker potential of global and locus-specific m6A patterns. Tumor hypoxia dysregulates m6A profiles, bridging prior genomic and proteomic observations. Specific m6A sites, such as those in VCAN, drive disease aggression, associating with poor outcomes, tumor growth and metastasis. m6A dysregulation is thus associated with key events in the natural history of prostate cancer: germline risk, microenvironmental dysregulation, somatic mutation and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19695af6dbb1a44b8e5db5cde53d9277c7377300" target='_blank'>
              The landscape of N6-methyladenosine in localized primary prostate cancer
              </a>
            </td>
          <td>
            Xin Xu, Helen Zhu, Rupert Hugh-White, J. Livingstone, Stefan E. Eng, Nicole Zeltser, Yujuan Wang, Kinga Pajdzik, Sujun Chen, Kathleen E. Houlahan, Wenqin Luo, Shun Liu, Xi Xu, Minzhi Sheng, Wangyuan Guo, J. Arbet, Yuxi Song, Miranda Wang, Yong Zeng, Shiyan Wang, Guanghui Zhu, Tingxiao Gao, Wei Chen, Xinpei Ci, Wenjie Xu, Kexin Xu, M. Orain, V. Picard, H. Hovington, A. Bergeron, L. Lacombe, Bernard Têtu, Yves Fradet, Mathieu Lupien, Gong-Hong Wei, Marianne Koritzinsky, Robert G Bristow, Neil Fleshner, Xue Wu, Yang Shao, Chuan He, Alejandro Berlin, T. H. van der Kwast, Hon Leong, P. Boutros, H. H. He
          </td>
          <td>2025-03-24</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Chromosomal rearrangements involving additional material in individuals with phenotypic alterations usually result in partial trisomy, often accompanied by partial monosomy. To characterize chromosomal rearrangements and analyze genomic characteristics in the breakpoint regions in 31 patients with additional material on an autosomal chromosome. Different tests were performed to characterize these patients, including karyotyping, chromosomal microarray analysis (CMA), and fluorescent in situ hybridization (FISH). In silico analyses evaluated A/B chromosomal compartments, segmental duplications, and repetitive elements at breakpoints. The 31 rearrangements resulted in 47 copy number variations (CNVs) and a range of structural aberrations were identified, including six tandem duplications, 19 derivative chromosomes, two intrachromosomal rearrangements, one recombinant, two dicentric chromosomes, and one triplication. A deleted segment was associated with the duplication in 16 of the 19 patients with derivative chromosomes from translocation. Among the trios whose chromosome rearrangement origin could be investigated, 54,5% were de novo, 31,9% were maternally inherited, and 13,6% were paternally inherited from balanced translocations or inversion. Breakpoint analysis revealed that 22 were in the A compartment (euchromatin), 25 were in the B compartment (heterochromatin), and five were in an undefined compartment. Additionally, 14 patients had breakpoints in regions of segmental duplications and repeat elements. Our study found that a deletion accompanied by additional genetic material was present in 51.6% of the patients, uncovering the underlying genetic imbalances. Statistical analyses revealed a positive correlation between chromosome size and the occurrence of CNVs in the rearrangements. Furthermore, no preference was observed for breakpoints occurring in compartments A and B, repetitive elements, or segmental duplications. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-97077-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4680fc424cbc00a4561d2d03671779c2ded30f6" target='_blank'>
              Cytogenomic characterization of karyotypes with additional autosomal material
              </a>
            </td>
          <td>
            Priscila Soares Rodrigues, Bruna Burssed, F. Bellucco, Débora Cristina Batista Rosolen, Chong Ae Kim, M. Melaragno
          </td>
          <td>2025-04-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Allele-specific DNA methylation (ASM) provides critical insights into the complex genetic and epigenetic mechanisms regulating gene transcription. Emerging evidence suggests that ASM is particularly enriched in gene enhancer regions, and recent studies have demonstrated that ASM is increased in cancer tissues compared with normal tissues. Despite the increasing recognition of ASM as a potential biomarker in tumorigenesis, systematic resources dedicated to identifying and annotating ASMs in cancer contexts remain limited. In this study, we developed CanASM (https://bioinfor.nefu.edu.cn/CanASM/), the first comprehensive database specifically designed to identify and annotate ASM in cancer. In CanASM, ASM sites identified from bisulfite sequencing data across 31 cancer types and their matched normal tissue samples are cataloged. Importantly, CanASM includes extensive regulatory annotations for ASMs, including associated genes, cis-regulatory elements and transcription factor binding colocalizations, transcription factor affinity changes, etc. Users can query and explore ASMs using various parameters, such as single-nucleotide variations (SNVs), chromosomal coordinates, and gene names. The current version of CanASM includes 5,003,877 unique SNV–CpG pairs, including 3,056,776 index SNVs, of which 2,634,406 are single-nucleotide polymorphisms (SNPs), and 4,157,508 CpGs. With an intuitive interface for browsing, querying, analyzing, and downloading, CanASM serves as a valuable resource for researchers investigating cancer-associated genetic variations and epigenetic regulation in cancer. Author summary Identifying target genes and regulatory mechanisms of cancer-associated genetic variants remains a major challenge in the post-GWAS era. Existing databases like Pancan-meQTL and ASMdb have contributed to uncovering the role of genetic variants in cancer by linking DNA variations to methylation. However, they lack comprehensive regulatory annotations for such sites and rely solely on genomic proximity for target gene prediction, overlooking distal interactions. Our analysis using 3D interaction data reveals that over 94.8% of ASMs exhibit distal regulatory relationships, emphasizing their critical role in cancer epigenetics. Furthermore, these databases cover a limited number of cancer types (Pancan-meQTL: 23, ASMdb: 8). To address these limitations, we developed CanASM, a database that provides ASMs identified from BS-seq data for 31 human cancers, along with multidimensional regulatory annotations for ASM sites. This includes target genes, cis-regulatory elements (enhancers, ATAC-seq regions, insulators, 3D interactions, CpG islands), TFBSs, and GWAS cancer trait associations. Additionally, CanASM offers three interactive tools for analyzing TF binding affinity changes, predicting ASM target genes through 3D interactions, and identifying cell marker variations. As a comprehensive resource for SNV-driven epigenetic regulation in cancer, CanASM is helpful for in-depth research in cancer genetics and epigenetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0525da13a9172e31ed829ad891d330567e188ba3" target='_blank'>
              CanASM: A comprehensive database for genome-wide allele-specific DNA methylation identification and annotation in cancer
              </a>
            </td>
          <td>
            Jianmei Zhao, Zeyu zhao, Hongfei Li, Haojie Yu, Hao Lin, Hanqi Chen, Xuecang Li, Di Liu, Yiming Wang, Guohua Wang
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c3773603d3c9014e1c679ae73200f35f924d76" target='_blank'>
              Single-cell SNP-DNA sequencing precisely maps genotoxic events in CRISPR-edited primary cells
              </a>
            </td>
          <td>
            J. Boutin, S. Fayet, V. Marin, C. Berges, M. Riandière, Jérôme Toutain, I. Lamrissi‐Garcia, C. Thibault, David Cappellen, S. Dabernat, Arthur Poulet, Maëla Francillette, Nathalie Droin, F. Moreau-Gaudry, A. Bedel
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9feff4a41380afc1edc6d3d9d9371441107ea7a0" target='_blank'>
              VAV2 exists in extrachromosomal circular DNA and contributes Enzalutamide resistance of prostate cancer via stabilization of AR/ARv7
              </a>
            </td>
          <td>
            Qinghua Wang, Xin'an Wang, Hanchu Ye, Yicong Yao, Haopeng Li, Xin Qin, Yan Zhao, Wei Jiang, Mingming Xu, Tong Zi, Xilei Li, Xi Chen, Juan Zhou, Gang Wu
          </td>
          <td>2025-04-09</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genomic instability is a hallmark of cancer, driving oncogenic mutations that enhance tumor aggressiveness and drug resistance. MYC, a master transcription factor that is deregulated in nearly all human tumors, paradoxically induces replication stress and associated DNA damage while also increasing expression of DNA repair factors and mediating resistance to DNA-damaging therapies. Emerging evidence supports a non-transcriptional role for MYC in preserving genomic integrity at sites of active transcription and protecting stalled replication forks under stress. Understanding how MYC’s genotoxic and genoprotective functions diverge may reveal new therapeutic strategies for MYC-driven cancers. Here, we identify a non-canonical role of MYC in DNA damage response (DDR) through its direct association with DNA breaks. We show that phosphorylation at serine 62 (pS62-MYC) is crucial for the efficient recruitment of MYC to damage sites, its interaction with repair factors BRCA1 and RAD51, and effective DNA repair to support cell survival under stress. Mass spectrometry analysis with MYC-BioID2 during replication stress reveals a shift in MYC’s interactome, maintaining DDR associations while losing transcriptional regulators. These findings establish pS62-MYC as a key regulator of genomic stability and a potential therapeutic target in cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73224953c4cbe478fbdd30504471b72d2f0b5d71" target='_blank'>
              MYC Serine 62 phosphorylation promotes its binding to DNA double strand breaks to facilitate repair and cell survival under genotoxic stress
              </a>
            </td>
          <td>
            Gabriel M. Cohn, Colin J. Daniel, Jenny Eng, Xiao-Xin Sun, C. Pelz, Koei Chin, Alexander Smith, Charles D. Lopez, Jonathan R. Brody, M. Dai, Rosalie C. Sears
          </td>
          <td>2025-03-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="ABSTRACT The non‐coding genome, constituting 98% of human DNA, remains largely unexplored, yet holds potential for identifying new biomarkers and therapeutic targets in acute lymphoblastic leukemia (ALL). In this study, we conducted a systematic analysis of recurrent somatic non‐coding single nucleotide variants (SNVs) in pediatric B‐cell precursor (BCP) ALL. We leveraged whole genome sequencing (WGS) data from 345 pediatric BCP ALL cases, representing all major genetic subtypes and identified 346 mutational hotspots that harbored somatic SNVs in at least three cases. Through the integration of paired RNA sequencing along with published ChIP‐seq and ATAC‐seq data, we found 128 non‐coding hotspots associated with differentially expressed genes nearby, which were enriched for cis‐regulatory elements, demonstrating the effectiveness of multi‐omics integration in distinguishing pathogenic mutations from passengers. We identified one mutational hotspot that was associated with increased expression of the leukemia‐associated gene NRAS in three primary ALLs. Micro‐C analysis in the leukemia cell line demonstrated interactions between the hotspot region and NRAS regulatory elements. Dual luciferase assays indicated that the mutations disrupted regulatory interactions and CRISPR‐mediated deletion of the region significantly upregulated NRAS, confirming the hypothesized regulatory link. Altogether, we provide new insights into the functional roles of non‐coding mutations in leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6089d599cb2b181049865efd98f322d1bb52759" target='_blank'>
              Discovery of Cis‐Regulatory Mechanisms via Non‐Coding Mutations in Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Efe Aydın, Eleanor L Woodward, Gladys Telliam Dushime, Rebeqa Gunnarsson, H. Lilljebjörn, L. Moura-Castro, T. Fioretos, B. Johansson, K. Paulsson, Minjun Yang
          </td>
          <td>2025-03-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a22caf0363586789d3ef13130aa7bdb69bf876e9" target='_blank'>
              Nuclear and genome dynamics underlying DNA double-strand break repair.
              </a>
            </td>
          <td>
            I. Chiolo, Matthias Altmeyer, G. Legube, Karim Mekhail
          </td>
          <td>2025-03-17</td>
          <td>Nature reviews. Molecular cell biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Regulator of telomere length 1 (RTEL1) helicase facilitates telomere replication by disassembling DNA secondary structures, such as G-quadruplexes and telomeric loops (t-loops), at the ends of the chromosomes. The recruitment of RTEL1 to telomeres occurs during the S-phase of the cell cycle, but the dynamics of the process has not been studied. Here, we utilized CRISPR genome editing and single-molecule imaging to monitor RTEL1 movement within human cell nuclei. RTEL1 utilizes rapid three-dimensional diffusion to search for telomeres and other nuclear targets. Only 5% of the chromatin-bound RTEL1 is associated with telomeres at any time in the S-phase, but the telomere-bound RTEL1 has much more extended associations. This binding is enhanced by the interaction between RTEL1 and the telomeric protein TRF2 but is largely independent of RTEL1 ATPase activity. The absence of RTEL1 catalytic activity leads to severe defects in cell proliferation, slow progression out of S-phase, and chromosome end-to-end fusion events. We propose that the rapid diffusion of RTEL1 allows this low-abundance protein to explore the nucleus, bind TRF2, and be recruited to telomeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d521287870dd985b41001fde2641677325213069" target='_blank'>
              Rapid dynamics allow the low-abundance RTEL1 helicase to promote telomere replication
              </a>
            </td>
          <td>
            Guanhui Wu, Erin Taylor, Daniel T. Youmans, Nausica Arnoult, Thomas R. Cech
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d383f031fb716745aefe8cfd14a2a089d4db51c" target='_blank'>
              Quantifying replication stress in cancer without proliferation confounding
              </a>
            </td>
          <td>
            Philipp Jungk, Maik Kschischo
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA damage response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically reprogrammed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of pancreatic ductal adenocarcinomas (PDACs), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells [human pancreatic nestin-expressing (HPNE) cells]. Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced messenger RNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a homologous recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR deficiency phenotypes, including sensitivity to inhibitors of translesion synthesis and poly-ADP ribose polymerase. Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13 depletion. Taken together, our results expose TRIP13 as an attractive new and therapeutically tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c52be837d7841e9eb90ef56e65d2eb6df322017a" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-03-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="Malignant pleural mesothelioma (MPM) is a rare yet highly aggressive malignancy with a severe prognosis. Compounded by the lack of effective treatment modalities, MPM remains a formidable health challenge. Therefore, the identification of actionable liabilities is critical for advancing precision medicine to combat this lethal disease. Here, we exploit an unbiased genome-wide CRISPR screen, integrating and cross-comparing three MPM cell lines with nonmalignant mesothelial cells, to selectively map the gene targets whose depletion indicates a common dependency in MPM cells. This systematic approach unveils a cohort of verifiable genes, among which BUB1, a mitotic checkpoint serine/threonine kinase, emerges as a high-confidence hit in cancer cells. Cellular and molecular studies demonstrate that genetic depletion or pharmacological inhibition of BUB1 profoundly impairs MPM cell survival and growth while inducing G2/M cell cycle arrest, cellular senescence, and apoptosis, and attenuating functional hallmarks of aggressive cancer cells. Transcriptomic profiling of BUB1-depleted cells discloses differential gene expression signatures congruent with cell fate phenotypes, including the reprogramming of mitotic network genes. Mechanistically, BUB1 is indispensable for the proper localization of essential mitotic regulators MAD1, MAD2, and Shugoshin (SGO1), thereby ensuring the functionality of the spindle assembly checkpoint (SAC). Furthermore, BUB1 ablation leads to cytokinesis failure and multinucleation, a phenotype characterized by the downregulation of CDC20, Cyclin A, and Cyclin B, and a reciprocal upregulation of the cyclin-dependent kinase inhibitor p21. Clinically, MPM tumors exhibit elevated levels of BUB1, and high BUB1 expression is associated with shorter patient survival. Our novel findings accentuate comparative CRISPR screens as a powerful platform to explore tumor cell-selective gene essentiality and propose BUB1 kinase as a potential marker and druggable vulnerability with therapeutic implications for MPM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/147c280cc9a37e6a39ce87d2c82cecf0b12eda26" target='_blank'>
              Genome-wide CRISPR screen identifies BUB1 kinase as a druggable vulnerability in malignant pleural mesothelioma
              </a>
            </td>
          <td>
            E. Cakiroglu, Sude Eris, Ozden Oz, G. Karakülah, Serif Senturk
          </td>
          <td>2025-04-03</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471fa22c73f7c1a8083ff473dd937339ee492d0c" target='_blank'>
              Abstract A022: Decoding aneuploidy: Identifying drivers and therapeutic targets in recurrent breast cancer copy number alterations
              </a>
            </td>
          <td>
            Khalid N. Al-Zahrani, Ellen Langille, A. Obersterescu, Christopher Lowden, Katie Teng, Lauren Caldwell, David Cook, Miguel Pérez-Castro, Cynthia H Chiu, Alec Bahcheli, R. Tsai, Jacob Berman, Kin Chan, Linkang Zhang, K.W. Annie Bang, Michael Parsons, Adele Lopes, Jocelyn Nurtanto, E. I. Temel, Iosifina Fotiadou, Julien Dessapt, Hartland Jackson, Sean E Egan, J. Reimand, Jeffrey Wrana, Daniel Schramek
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Glioblastoma (GBM) is one of the most malignant primary brain tumors, and factors governing its progression are not fully characterized. Recent research suggests that the long non-coding RNA (lncRNA) HOTAIR and super-enhancers (SEs) contribute significantly to GBM progression. Here, we performed TCGA data analysis revealing that high HOTAIR expression in GBM is associated with poor prognosis. Conversely, HOTAIR knock-down (KD) decreased proliferation, colony formation, and invasion of GBM cells. Furthermore, RNA-seq analysis identified DEGs in GBM cells related to cell growth and adhesion. Using an integrated approach, we also identify MEST as a HOTAIR-associated SE target gene. Intriguingly, MEST suppression in GBM cells phenocopied HOTAIR KD, as evidenced by reduced cell proliferation and invasion, whereas MEST overexpression counteracted effects of HOTAIR depletion. Moreover, 3 C technique-based PCR confirmed reduced interaction between HOTAIR-associated SEs and target genes after HOTAIR KD. This study reveals a novel regulatory mechanism governing GBM, offering promising directions for clinical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0725c8dbcb7813eec6ac9a1c7efd33d858328bd0" target='_blank'>
              A HOTAIR-associated super-enhancer orchestrates glioblastoma malignancy via MEST
              </a>
            </td>
          <td>
            Peng Li, Yang Yang, Chunpeng Luan, Wenbin Wang, Yuan Jiang, Zhenhao Zhao, Bo Wang, Yuting Zhao, Yunlong Bai, Man Liu, Zhongfang Zhao, Lei Zhang, Yuyang Qian, Jiandang Shi
          </td>
          <td>2025-04-07</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract The Philadelphia chromosome, the translocation between BCR and ABL genes, is seen in 95% of chronic myeloid leukemia (CML) patients. Although discovered >60 years ago, the molecular mechanism of BCR fragility is unclear. Here, we have identified several G4 DNA motifs at the BCR fragile region of CML patients. Various lines of experimentation revealed that the breakpoint regions could fold into multiple intramolecular G-quadruplex structures. The sodium bisulfite modification assay revealed single strandedness in the fragile region when present on a plasmid and human genome. Circular dichroism spectroscopy revealed the parallel G4 DNA formation, leading to polymerase arrest at the BCR breakpoints. Intracellular recombination assay revealed that DNA breakage at the BCR fragile region could join with the break generated by ISceI endonuclease. Finally, purified AID could bind and deaminate cytosines when present on single-stranded DNA generated due to G4 DNA, both in vitro and inside the cells. Therefore, our results suggest that AID binds to G4 DNA present at the BCR fragile region, resulting in the deamination of cytosines to uracil and induction of DNA breaks in one of the DNA strands, which can later get converted into a double-strand break, leading to t(9;22) chromosomal translocation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad133901fa4a2888984a520b619646b9998e103" target='_blank'>
              Formation of multiple G-quadruplexes contributes toward BCR fragility associated with chronic myelogenous leukemia
              </a>
            </td>
          <td>
            Shivangi Sharma, Elizabeth Thomas, Sumedha Dahal, Sayak Das, Shefali Kothari, U. Roy, Nitu Kumari, V. Gopalakrishnan, S. Raghavan
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Large eukaryotic genomes are packaged into the restricted area of the nucleus to protect the genetic code and provide a dedicated environment to read, copy and repair DNA. The physical organisation of the genome into chromatin loops and self-interacting domains provides the basic structural units of genome architecture. These structural arrangements are complex, multi-layered, and highly dynamic and influence how different regions of the genome interact. The role of chromatin structures during transcription via enhancer-promoter interactions is well established. Less understood is how nuclear architecture influences the plethora of chromatin transactions during DNA replication and repair. In this review, we discuss how genome architecture is regulated during the cell cycle to influence the positioning of replication origins and the coordination of DNA double strand break repair. The role of genome architecture in these cellular processes highlights its critical involvement in preserving genome integrity and cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7d6e724c4765bfa8612f020a665aed0f98394cb" target='_blank'>
              Roles for the 3D genome in the cell cycle, DNA replication, and double strand break repair
              </a>
            </td>
          <td>
            Katherine A Giles, P. Taberlay, Anthony J. Cesare, Mathew J. K. Jones
          </td>
          <td>2025-02-27</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Replicative stress (RS) is emerging as a promising therapeutic target in oncology, yet full exploitation of its potential requires a detailed understanding of the mechanisms and genes involved. Here, we investigated the RNA helicase Senataxin (SETX), an enzyme that resolves RNA-DNA hybrids and R-loops, to address its role in preventing RS by oncogenic Myc. Upon Myc activation, silencing of SETX led to selective engagement of the DNA damage response (DDR) and robust cytotoxicity. Pharmacological dissection of the upstream kinases regulating the DDR uncovered a protective role of the ATR pathway, that once inhibited, boosted SETX driven-DDR. While SETX loss did not lead to a genome-wide increase of R-loops, mechanistic analyses revealed enhanced R-loops localized at DDR-foci and newly replicated genomic loci, compatible with a selective role of SETX in resolving RNA-DNA hybrids to alleviate Myc-induced RS. Genome-wide mapping of DNA double-strand breaks confirmed that SETX silencing exacerbated DNA damage at transcription-replication conflict (TRC) regions at early replicated sites. We propose that SETX prevents Myc-induced TRCs by resolving transcription-associated R-loops that encounter the replisome. The identification of SETX as a genetic liability of oncogenic Myc opens up new therapeutic options against aggressive Myc-driven tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7b454878504977a6303670c09e4ebf64ca85ff3" target='_blank'>
              Senataxin prevents replicative stress induced by the Myc oncogene
              </a>
            </td>
          <td>
            Silvia Sberna, Marco Filipuzzi, Nicola Bianchi, Ottavio Croci, Federica Fardella, C. Soriani, S. Rohban, Sara Carnevali, Alessandra Alberta Albertini, Nicola Crosetto, S. Rodighiero, Arianna Chiesa, Laura Curti, S. Campaner
          </td>
          <td>2025-03-19</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/330a5593d326f97de03e4da3203d107ef7927e48" target='_blank'>
              Extending differential gene expression testing to handle genome aneuploidy in cancer
              </a>
            </td>
          <td>
            Katsiaryna Davydzenka, G. Caravagna, Guido Sanguinetti
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung cancer exhibits altered metabolism, influencing its response to radiation. To investigate the metabolic regulation of radiation response, we conducted a comprehensive, metabolic-wide CRISPR-Cas9 loss-of-function screen using radiation as selection pressure in human non–small cell lung cancer. Lipoylation emerged as a key metabolic target for radiosensitization, with lipoyltransferase 1 (LIPT1) identified as a top hit. LIPT1 covalently conjugates mitochondrial 2-ketoacid dehydrogenases with lipoic acid, facilitating enzymatic functions involved in the tricarboxylic acid cycle. Inhibiting lipoylation, either through genetic LIPT1 knockout or a lipoylation inhibitor (CPI-613), enhanced tumor control by radiation. Mechanistically, lipoylation inhibition increased 2-hydroxyglutarate, leading to H3K9 trimethylation, disrupting TIP60 recruitment and ataxia telangiectasia mutated (ATM)–mediated DNA damage repair signaling, impairing homologous recombination repair. In summary, our findings reveal a critical role of LIPT1 in regulating DNA damage and chromosome stability and may suggest a means to enhance therapeutic outcomes with DNA-damaging agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524c8836655ab7f9cea8b4528bffe4e53726a560" target='_blank'>
              Lipoylation inhibition enhances radiation control of lung cancer by suppressing homologous recombination DNA damage repair
              </a>
            </td>
          <td>
            Jui-Chung Chiang, Zeng Shang, Tracy Rosales, Ling Cai, Wei-Min Chen, Feng Cai, Hieu S Vu, John D Minna, Min Ni, Anthony J. Davis, Robert D Timmerman, Ralph J DeBerardinis, Yuanyuan Zhang
          </td>
          <td>2025-03-12</td>
          <td>Science Advances</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1afe205564d682ee54296311b108e6d8a61e8b1" target='_blank'>
              Spatial mapping of DNA synthesis reveals dynamics and geometry of human replication nanostructures
              </a>
            </td>
          <td>
            Michael Hawgood, Bruno Urién González, Ana Agostinho, Praghadhesh Thiagarajan, Yiqiu Yang, Xue Zhang, Giovanni Giglio, Gemma Quijada, Matilde Fonseca, J. Bartek, Hans Blom, B. Lemmens
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="YTHDC1, a key protein in the m6A-related regulatory network within cells, is involved in multiple cellular processes such as chromatin-related regulation, RNA splicing, and nuclear export. Understanding its role in colorectal cancer (CRC) development and DNA damage repair is critical for the advancement of treatment strategies. Our study found that YTHDC1 was highly expressed in high-malignancy CRC tissues compared with low-malignancy ones. Upon silencing YTHDC1, we observed a pronounced suppression of the proliferation of CRC cell lines, accompanied by a substantial increase in cell apoptosis. Furthermore, we identified RAD51 as a crucial downstream target of YTHDC1. Knocking down YTHDC1 led to a notable decrease in RAD51 protein levels, and silencing RAD51 also inhibited cancer cell proliferation. Interestingly, RNA-sequencing data indicated that the YTHDC1 deletion did not affect RAD51 transcription. However, Western blot revealed that this deletion increased the ubiquitination of RAD51, likely due to the upregulated E3 ligase UBE3A. Ubiquitination experiments subsequently confirmed that RAD51 is indeed one of the substrates of UBE3A. In summary, our study provides novel insights into how YTHDC1 modulates the expression of RAD51 through post-translational modifications. These findings offer valuable information that may potentially contribute to the development of more effective therapeutic strategies for CRC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-92925-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64cf4d3cb02cc31b95af37c1defc2a6fa9b119aa" target='_blank'>
              YTHDC1 negatively regulates UBE3A to influence RAD51 ubiquitination and inhibit apoptosis in colorectal cancer cells
              </a>
            </td>
          <td>
            Mingwei Gao, Yueguang Wu, Li Zhang, Yan Zhou, Huijuan Liu, Weimin Zhang, Shubin Wang, Yongping Cui
          </td>
          <td>2025-03-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="A string of nucleosomes, where genomic DNA is wrapped around histones, is organized in the cell as chromatin, ranging from euchromatin to heterochromatin, with distinct genome functions. Understanding physical differences between euchromatin and heterochromatin is crucial, yet specific labeling methods in living cells remain limited. Here, we have developed replication-dependent histone (Repli-Histo) labeling to mark nucleosomes in euchromatin and heterochromatin based on DNA replication timing. Using this approach, we investigated local nucleosome motion in the four known chromatin classes, from euchromatin to heterochromatin, of living human and mouse cells. The more euchromatic (earlier-replicated) and more heterochromatic (later-replicated) regions exhibit greater and lesser nucleosome motions, respectively. Notably, the motion profile in each chromatin class persists throughout interphase. Genome chromatin is essentially replicated from regions with greater nucleosome motions, although the replication timing is perturbed. Our findings, combined with computational modeling, suggest that earlier-replicated regions have more accessibility, and local chromatin motion can be a major determinant of genome-wide replication timing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d2b9973e20e3b9a1e91f7d1257dd38b93d96cad" target='_blank'>
              Replication-dependent histone labeling dissects the physical properties of euchromatin/heterochromatin in living human cells
              </a>
            </td>
          <td>
            Katsuhiko Minami, Kako Nakazato, Satoru Ide, Kazunari Kaizu, Koichi Higashi, Sachiko Tamura, Atsushi Toyoda, Koichi Takahashi, Ken Kurokawa, K. Maeshima
          </td>
          <td>2025-03-28</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="The apicoplast organelle of Plasmodium falciparum is essential for parasite’s replication, however the details of its biogenesis, inheritance and regulation throughout the cell cycle are unknown. Here, we report the development of a dynamic imaging platform coupled with an analytical pipeline that enables us to follow and measure subcellular structures throughout the 48-hour cell cycle of live parasites. We reveal a predetermined sequence of four discrete morphological steps in organelle development, which are tightly correlated with nuclear replication. We show that one of these steps, which we term the Crown morphology, is required for nucleus-apicoplast attachment. During Crown, apicoplast is stretched over multiple nuclei, fastened by centriolar tubulin. A complementary molecular approach was used to discover the basic ploidy of apicoplast and mitochondrial genomes, their replication rates and association with nuclear DNA replication. We inhibited nuclear DNA replication and found that it completely blocks apicoplast biogenesis in its most initial stages, demonstrating dependency on S-phase initiation. Conversely, specific inhibition of apicoplast genome replication resulted in an almost-undisturbed organelle development and division. However, it affected the Crown step, preventing association to tubulin-nuclear structures, leading to failure in accurate organelle sorting into daughter cells. Collectively, these experiments reveal a central cellular pathway linking apicoplast development to the parasite’s cell cycle, and a second independent organellar mechanism responsible for segregation into daughter cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6bfca923aa81dca1cabb89b41f451971dd56d89" target='_blank'>
              Organelle Development and Inheritance are Driven by Independent Nuclear and Organellar Mechanisms in Malaria Parasites
              </a>
            </td>
          <td>
            Shahar Michal, Qasem Alia, Freedman Eshkar, Maron Yariv, Tissawak Amanda, Florentin Anat
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344c45deee3ee4d6c328b84a6e73872831228859" target='_blank'>
              Abstract B019: Cyclin E1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Chunyang Bao, Jin Zhou, A. Bass
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BCL6 is a master transcriptional regulator of germinal center (GC) B cells. BCL6 is frequently translocated at the major translocation cluster (MTC) within intron 1 of the BCL6 locus, a hotspot commonly rearranged in diffuse large B cell lymphomas (DLBCLs). BCL6 amplifications are associated with therapeutic resistance and poor survival outcomes in hematological and solid cancers. However the mechanisms suppressing genome instability at the BCL6-MTC preventing BCL6 rearragements remain unclear. Here, transcriptome analysis and genome-wide mapping of histone H3 lysine 4 trimethylation (H3K4me3) in hydroxyurea (HU)-treated Raji cells (a Burkitt’s lymphoma model) revealed the induced expression of MBD1, encoding the DNA CpG methylation-binding protein. Functional studies using shRNA silencing and ectopic overexpression demonstrated that MBD1 suppresses BCL6 transcription whose promoter harbours conserved CpG methylation sites, suggesting a DNA methylation-dependent regulation of BCL6 trasncription by MBD1. Conversely, BCL6 repressed MBD1 expression by binding to its promoter. MBD1-depleted Raji cells exhibited increased genomic instability at the BCL6-MTC upon HU treatment, heightened sensitivity to DNA replication inhibitors (HU, gemcitabine, and etoposide), and reduced tumorigenicity in xenograft mouse models. We propose that MBD1 prevents genomic instability at the BCL6-MTC to suppress DLBCL formation. Moreover, MBD1 promotes genomic stability and cell viability during DNA replication stress. MBD1 thus represents a potential therapeutic target for cancers exhibiting resistance to chemotherapies targeting DNA replication. Key Points MBD1 suppresses BCL6 expression under the DNA replication stress MBD1 suppresses genomic instability at BCL6 translocation hotspots MBD1 depletion is associated with reduced tumorigenicity in mouse xenograft and sensitivity to DNA replication inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57727c3c919ef41a91774411963507ee9349496c" target='_blank'>
              DNA replication stress-induced transcriptome of Human Burkitt’s lymphoma identifies MBD1 as a novel suppressor of BCL6 rearrangements in germinal center derived B-lymphomagenesis
              </a>
            </td>
          <td>
            S. Gothwal, Kyoko Oichai, Jacqueline H Barlow
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/391f121bccfd400a13a99c3af96bbd051ffc5650" target='_blank'>
              Cohesin mutations and chromatin changes in cancer.
              </a>
            </td>
          <td>
            Ariel D Swett, Zuzana Tothova
          </td>
          <td>2025-03-10</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9573a097c9d79cc31651e0409f2f3e1d71e62921" target='_blank'>
              Replication coordination marks the domestication of large extrachromosomal replicons in bacteria
              </a>
            </td>
          <td>
            Jakub Czarnecki, Morgan Lamberioux, O. Skovgaard, Amaury Bignaud, Najwa Taib, Théophile Niault, P. Bourhy, Julia Bos, Elvira Krakowska, D. Bartosik, Romain Koszul, M. Marbouty, Didier Mazel, Marie-Eve Val
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Genome-wide human genetic studies have identified inherited cis-regulatory loci variants that predispose to cancers. However, the mechanisms by which these germline variants influence cancer progression, particularly through gene expression and proteostasis control, remain unclear. By analyzing genomic data from a gastric cancer (GC) case-control study (2,117 individuals), focusing on the ubiquitin-specific protease (USP) family, we identify the SNP rs72856331 (G>A) in the promoter region of the proto-oncogene USP47 as a putative susceptibility allele for GC. Mechanistically, the risk allele G is associated with enhanced USP47 expression, mediated by altered recruitment of the transcription factor GLI3 and changes in the epigenetic status at promoter. CRISPR/Cas9-mediated single-nucleotide conversion into risk allele G results in increased GLI3 binding and subsequent USP47 upregulation. The depletion of GLI3 results in a reduction of cancer-related phenotypes, similar to those observed following USP47 knockdown. Furthermore, we identify Snai1 as a deubiquitination target of USP47, explaining USP47-dependent activation of the epithelial-mesenchymal transition pathway and tumor progression. Our findings identify an important genetic predisposition that implicates the perturbation of transcription and proteostasis programs in GC, offering insights into prevention and therapeutic strategies for genetically stratified patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1084b39e6f7bd79c603ae5a0dd8fa214a3f635" target='_blank'>
              An inherited predisposition allele promotes gastric cancer via enhancing deubiquitination-mediated activation of epithelial-to-mesenchymal transition signaling
              </a>
            </td>
          <td>
            Bolin Tao, Zhenning Wang, Xuanyi Wang, Aixia Song, Jiaxian Liu, Jianan Wang, Qin Zhang, Zhaolin Chen, Zixian Wang, Wenjie Xu, Menghong Sun, Yanong Wang, Ping Zhang, Tao Xu, Gong-Hong Wei, F. X. Chen, Mengyun Wang
          </td>
          <td>2025-02-25</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2be5d83a44f2113bcc7ca92950bf19d6878d25c9" target='_blank'>
              Functional landscape of circular RNAs in human cancer cells
              </a>
            </td>
          <td>
            Peter Hyunwuk Her, Tiantian Li, Ziwei Huang, Xin Xu, Weining Yang, Moliang Chen, Mona Teng, Sujun Chen, Yong Zeng, Stanley Liu, Benjamin Haibe-Kains, Fraser Soares, Jie Ming, H. H. He
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Oncogenic gene fusions occur across a broad range of cancers and are a defining feature of some cancer types. Cancers driven by gene fusion products tend to respond well to targeted therapies, where available; thus, detection of potentially targetable oncogenic fusions is necessary to select optimal treatment. Detection methods include non-sequencing methods, such as fluorescence in situ hybridization and immunohistochemistry, and sequencing methods, such as DNA- and RNA-based next-generation sequencing (NGS). While NGS is an efficient way to analyze multiple genes of interest at once, economic and technical factors may preclude its use in routine care globally, despite several guideline recommendations. The aim of this review is to present a summary of oncogenic gene fusions, with a focus on fusions that affect tyrosine kinase signaling, and to highlight the importance of testing for oncogenic fusions. We present an overview of the identification of oncogenic gene fusions and therapies approved for the treatment of cancers harboring gene fusions, and summarize data regarding treating fusion-positive cancers with no current targeted therapies and clinical studies of fusion-positive cancers. Although treatment options may be limited for patients with rare alterations, healthcare professionals should identify patients most likely to benefit from oncogenic gene fusion testing and initiate the appropriate targeted therapy to achieve optimal treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a05e8c035d2b5c9d05b3addb1c0208226f39c0" target='_blank'>
              Oncogenic gene fusions in cancer: from biology to therapy
              </a>
            </td>
          <td>
            Stephen V Liu, Misako Nagasaka, Judith Atz, Flavio Solca, Leonhard Müllauer
          </td>
          <td>2025-04-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Ovarian cancer remains one of the most lethal gynecological malignancies owing to its high recurrence rate and chemotherapeutic resistance. MYC is a well-known proto-oncogene that is frequently amplified in ovarian cancer and has been implicated in drug resistance. Previously, we established a new promoter–reporter system combined with a CRISPR activation library to identify unknown MYC regulators, and M1AP was identified as a novel MYC regulator. However, considering the insufficient explanation for the absence of guide RNA (gRNA) of MYC, this present study explored methods to prevent the gRNA of MYC itself from binding. This study first modified the promoter–reporter vector to improve its quality, then conducted CRISPR screening and analyzed candidate genes as MYC promoter regulators using next-generation sequencing in OVSAHO ovarian cancer cells. Eighty-six genes had ≥ 1000 reads, and Pearson’s correlation coefficient analysis was performed on the cBioPortal of the Cancer Genomics database. Fourteen genes were identified as candidate MYC regulators with positive and significant correlations with MYC. Seven genes, including CYP4v2, ASPH, ANP32D, PCED1A, ABI1, FUZ, and HOOK2, demonstrated significantly higher luciferase activity than the control genes. Four genes, including ABI1, PCED1A, HOOK2, and CYP4v2, activated the MYC promoter, which showed over twofold higher activity than the control when overexpressed using a vector. In conclusion, four genes that activate MYC promoters were identified in an ovarian cancer cell line using the CRISPR library system with a modified promoter–reporter tool. These results will prove helpful in the development of novel treatment strategies for ovarian cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01644-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9220d2085ff0b29849fe6d91dbca5cdaecc789b" target='_blank'>
              Modified screening of MYC promotor region elements using the CRISPR library in ovarian cancer
              </a>
            </td>
          <td>
            Akiko Yamamoto, Yosuke Tanaka, S. Ishibashi, Masumi Ikeda, Keisuke Sugita, Masanori Ono, Hirotaka Nishi, Morito Kurata
          </td>
          <td>2025-04-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c67dd9e9f49939f230935043557be5d3e8724d9" target='_blank'>
              Distinct satellite DNA composition between core and germline restricted chromosomes in Bradysia (Sciara) coprophila
              </a>
            </td>
          <td>
            Anne Kerrebrock, Jullien M. Flynn, Robert B. Baird, Christina N. Hodson, Laura Ross, Yukiko M. Yamashita
          </td>
          <td>2025-04-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96f5ae7db10ebf2238c71827c45c782f6d159f1b" target='_blank'>
              New insights into chromosomal regions 15p11.2 to 15q11.2 by studying submicroscopic variations using molecular cytogenetics.
              </a>
            </td>
          <td>
            Maria N Correia, Stefanie Kankel, I. Carreira, Joana B. Melo, Thomas Liehr
          </td>
          <td>2025-04-17</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1df029231423cb536b06a558936a00ba5d9246d" target='_blank'>
              PRAME Epitopes are T-Cell Immunovulnerabilities in BRD4::NUTM1 Initiated NUT Carcinoma
              </a>
            </td>
          <td>
            Jeffrey L. Jensen, Sara K. Peterson, Shawn Yu, Tomoaki Kinjo, Brandon A. Price, Maria Sambade, Steven Vesko, Johnathan D. DeBetta, Julie Geyer, Kwang P Nickel, Randall J. Kimple, R. Kotecha, Ian J. Davis, Jeremy R. Wang, Christopher A. French, Brian Kuhlman, Alex Rubinsteyn, Jared Weiss, Benjamin G. Vincent
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Genomic instability and inflammation are distinct hallmarks of aging, but the connection between them is poorly understood. Here we report a mechanism directly linking genomic instability and inflammation in senescent cells through a mitochondria-regulated molecular circuit involving p53 and cytoplasmic chromatin fragments (CCF) that are enriched for DNA damage signaling marker γH2A.X. We show that p53 suppresses CCF accumulation and its downstream inflammatory phenotype. p53 activation suppresses CCF formation linked to enhanced DNA repair and genome integrity. Activation of p53 in aged mice by pharmacological inhibition of MDM2 reverses transcriptomic signatures of aging and age-associated accumulation of monocytes and macrophages in liver. Mitochondrial ablation in senescent cells suppresses CCF formation and activates p53 in an ATM-dependent manner, suggesting that mitochondria-dependent formation of γH2A.X + CCF dampens nuclear DNA damage signaling and p53 activity. These data provide evidence for a mitochondria-regulated p53 signaling circuit in senescent cells that controls DNA repair, genome integrity, and senescence- and age-associated inflammation, with relevance to therapeutic targeting of age-associated disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4941baca9720335671879d8e4ca5f2fe5568326" target='_blank'>
              p53 enhances DNA repair and suppresses cytoplasmic chromatin fragments and inflammation in senescent cells
              </a>
            </td>
          <td>
            Karl N. Miller, Brightany Li, Hannah R. Pierce-Hoffman, Shreeya Patel, Xue Lei, Adarsh Rajesh, M. G. Teneche, Aaron Havas, Armin Gandhi, Carolina Cano Macip, Jun Lyu, Stella G. Victorelli, Seung-Hwa Woo, A. Lagnado, Michael A LaPorta, Tianhui Liu, Nirmalya Dasgupta, Sha Li, Andrew Davis, Anatoly Korotkov, Erik Hultenius, Zichen Gao, Yoav Altman, Rebecca A Porritt, Guillermina Garcia, C. Mogler, A. Seluanov, Vera Gorbunova, S. Kaech, Xiao Tian, Zhixun Dou, Chongyi Chen, João F. Passos, P. Adams
          </td>
          <td>2025-03-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2792ac282d27759d5d74152b7ef668d21c563cc" target='_blank'>
              CDK2 Inhibition Produces a Persistent Population of Polyploid Cancer Cells.
              </a>
            </td>
          <td>
            Liliya Tyutyunyk‐Massey, Zibo Chen, Xiuxia Liu, Masanori Kawakami, Adam Harned, Yeap Ng, Brian Luke, Samuel C Okpechi, Blessing Ogunlade, Y. Alfaro-Mora, Roberto Weigert, Kedar Narayan, Xi Liu, Ethan Dmitrovsky
          </td>
          <td>2025-04-15</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a02c68e4480c29b3b195dc3be77303fbc346b69" target='_blank'>
              Enhanced FLI1 accessibility mediates STAG2-mutant leukemogenesis
              </a>
            </td>
          <td>
            Jane J. Xu, Viviana Scoca, Yi Chen, Y. Zhan, Alexander Fisher, Eno-obong Udoh, Sebastian Fernando, Besmira Alija, John Pantazi, Varun S Sudunagunta, Edna M. Stewart, A. M. D. Galang, Mike Williams, Govind Bhagat, C. Gebhard, Valeria Visconte, Sarah Ondrejka, Ruud Delwel, Ming Hu, R. Koche, A. Viny
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Abstract TAQing technologies are based on the restriction enzyme-induced DNA double-strand break (DSB) formation in living cells, which results in large-scale genomic rearrangements and phenotypic alterations. Originally, the TAQing system requires heat treatments to activate restriction enzymes, which sometimes leads to cell toxicity or stress responses. Here, we developed a blue-light-controlled MagTAQing system, which induces DSBs exclusively upon blue-light exposure by assembling the split restriction enzymes via Magnet modules. Application of MagTAQing to mitotic budding yeast cells successfully triggered various genomic rearrangements upon blue-light exposure. Since this technology enables the conditional induction of genomic rearrangements in specific cells or tissues, we employed MagTAQing on meiotic yeast cells lacking the recombinase Spo11 to induce artificial DSBs. Consequently, Spo11-independent meiotic DSBs resulted in aneuploidies and nonallelic homologous recombinations between repetitive sequences such as ribosomal DNA and retrotransposons. These results suggest a pivotal role of Spo11-induced recombination in preventing chromosomal abnormality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72c560ade8948ca0b121c00661a4ad61330184b0" target='_blank'>
              Light-controlled Spo11-less meiotic DNA breaks by MagTAQing lead to chromosomal aberrations
              </a>
            </td>
          <td>
            Hideyuki Yone, Yuri Kawashima, Hayato Hirai, Arisa H. Oda, Moritoshi Sato, Hiromitsu Kono, Kunihiro Ohta
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Aim: As intrinsic resistance - often driven by concurrent genomic alterations in tumor suppressor genes or oncogenes - remains a major challenge in oncology, this work aimed to comprehensively analyze BRAF somatic alterations across cancer types and identify new potential therapeutic strategies to overcome drug resistance. Methods: We conducted an extensive analysis of genomics, transcriptomics, and clinical data retrieved from public repositories, including cBioPortal. Our comprehensive analysis examined BRAF alterations [point mutations, structural variants (SVs) and copy number alteration] in more than 217,000 tumor samples across 120 distinct tumor types from primary and metastatic sites in both adult and pediatric cohorts, focusing on mutual exclusivity and co-occurrence of mutations in other oncogenes or tumor suppressors. The work also explores the association of BRAF somatic alterations with survival, clinical and molecular features. Results: Analysis of mutation frequencies across cancer types revealed that BRAFV600E represents approximately 90% of all BRAF alterations. While melanoma and thyroid carcinoma show the highest prevalence of BRAF mutations, followed by colorectal and non-small cell lung cancer in terms of absolute number of patients harboring BRAF mutations worldwide, notably high mutation frequencies were identified in rare malignancies, including hairy-cell leukemia, ganglioglioma, and serous borderline ovarian tumors. The comprehensive analysis of genomic profiling data across these tumors uncovered distinct patterns of co-occurring and mutually exclusive alterations in oncogenes and tumor suppressor genes, illuminating resistance mechanisms and suggesting novel therapeutic combinations. Conclusion: Comprehensive genomic profiling is critical for optimizing targeted therapy and overcoming drug resistance in BRAF-mutated cancers. The identification of co-occurring alterations provides opportunities for rational combination therapies, emphasizing the importance of detailed mutation profiling in developing effective treatment strategies across diverse cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7fa7f528d2835416489339d23ab2a839922e3b0" target='_blank'>
              Unraveling BRAF alterations: molecular insights to circumvent therapeutic resistance across cancer types
              </a>
            </td>
          <td>
            Caterina Perfetto, Marianna Aprile, Simona Cataldi, Elisa Giovannetti, Valerio Costa
          </td>
          <td>2025-03-24</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Aberrant epigenetic remodeling events occurred in head and neck squamous cell carcinoma (HNSCC) contribute to tumor stemness and chemotherapy resistance, yet little is known. In this study, we identified that ubiquitin-specific peptidase 10 (USP10) is up-regulated in HNSCC tissues, and high USP10 is associated with poor prognosis of patients. Functionally, USP10 serving as an oncogene potentiates the proliferation and metastasis of HNSCC cells in vitro and in vivo. Mechanistically, USP10 physically interacts with, deubiquitinate, and stabilizes BAZ1A proteins. In addition, BAZ1A complexes with SOX2 to drive the enhancer-promoter interaction and facilitate the recruitment of BRD4, thereby activating the expressions of cancer stem cells (CSCs)-related signature. Therefore, we found that USP10 relied on BAZ1A to enhance HNSCC stemness, progression, and chemotherapy resistance. The pharmacology research implicated that BAZ1A-IN-1, one specific BAZ1A inhibitor, could effectively inhibit HNSCC stemness, distal metastasis, and cisplatin resistance. Together, our study revealed a novel USP10/BAZ1A/stemness axis and one significant therapeutic target for USP10-driven HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0327056e6475da98691bd633cb8eedaa75bc211a" target='_blank'>
              USP10 stabilizes BAZ1A to drive tumor stemness via an epigenetic mechanism in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Yanni Shi, Jiawei Ding, Xiao Ling, Danfeng Xu, Yan Shen, Xingjun Qin
          </td>
          <td>2025-04-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Endemic Burkitt lymphoma (eBL) is one of the most prevalent cancer in children in sub-Saharan Africa, and while prior studies have found that Epstein-Barr virus (EBV) type and variation may alter the tumor driver genes necessary for tumor survival, the precise relationship between EBV variation and EBV-associated tumorigenesis remains unclear due to lack of scalable, cost-effective, viral whole-genome sequencing from tumor samples. This study introduces a rapid and cost-effective method of enriching, sequencing, and assembling accurate EBV genomes in BL tumor cell lines through a combination of selective whole genome amplification (sWGA) and subsequent 2-tube multiplex polymerase chain reaction along with long-read sequencing with a portable sequencer. The method was optimized across a range of parameters to yield a high percentage of EBV reads and sufficient coverage across the EBV genome except for large repeat regions. After optimization, we applied our method to sequence 18 cell lines and 3 patient tumors from fine needle biopsies and assembled them with median coverages of 99.62 and 99.68%, respectively. The assemblies showed high concordance (99.61% similarity) to available Illumina-based assemblies. The improved method and assembly pipeline will allow for better understanding of EBV variation in relation to BL and is applicable more broadly for translational research studies, especially useful for laboratories in Africa where eBL is most widespread. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-94737-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4545234794b307a82810b0886fef390a78786e91" target='_blank'>
              Comparison of nanopore with illumina whole genome assemblies of the Epstein-Barr virus in Burkitt lymphoma
              </a>
            </td>
          <td>
            Isaac E. Kim, A. Fola, Enrique Puig, T. K. Maina, Sin Ting Hui, Hongyu Ma, Kaleb Zuckerman, Eddy O. Agwati, A. Leonetti, Rebecca Crudale, Micah A. Luftig, A. M. Moormann, Cliff I. Oduor, Jeffrey A. Bailey
          </td>
          <td>2025-03-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e8c850ec330dd74c6ce3b2b64ef9b0d2af03ef5" target='_blank'>
              Chemical inhibition of RPA gap protection sensitizes BRCA1-deficient cancers to PARP inhibition
              </a>
            </td>
          <td>
            P. VanderVere-Carozza, Matthew R. Jordan, Joy E. Garrett, Karen E. Pollok, Hilary D. Hinshaw, Xierzhatijiang Sulaiman, Sheng Liu, J. Wan, K. Pawelczak, J. Turchi
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Endemic Burkitt lymphoma (eBL) is one of the most prevalent cancer in children in sub-Saharan Africa, and while prior studies have found that Epstein-Barr virus (EBV) type and variation may alter the tumor driver genes necessary for tumor survival, the precise relationship between EBV variation and EBV-associated tumorigenesis remains unclear due to lack of scalable, cost-effective, viral whole-genome sequencing from tumor samples. This study introduces a rapid and cost-effective method of enriching, sequencing, and assembling accurate EBV genomes in BL tumor cell lines through a combination of selective whole genome amplification (sWGA) and subsequent 2-tube multiplex polymerase chain reaction along with long-read sequencing with a portable sequencer. The method was optimized across a range of parameters to yield a high percentage of EBV reads and sufficient coverage across the EBV genome except for large repeat regions. After optimization, we applied our method to sequence 18 cell lines and 3 patient tumors from fine needle biopsies and assembled them with median coverages of 99.62 and 99.68%, respectively. The assemblies showed high concordance (99.61% similarity) to available Illumina-based assemblies, and differences were consistent with improved accuracy. The improved method and assembly pipeline will allow for better understanding of EBV variation in relation to BL and is applicable more broadly for translational research studies, especially useful for laboratories in Africa where eBL is most widespread.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc231a1e340bf069a402e88df346d750d87f574" target='_blank'>
              Comparison of Nanopore with Illumina Whole Genome Assemblies of the Epstein-Barr Virus in Burkitt Lymphoma
              </a>
            </td>
          <td>
            Isaac E. Kim, A. Fola, Enrique Puig, T. K. Maina, Sin Ting Hui, Hongyu Ma, Kaleb Zuckerman, Eddy O. Agwati, A. Leonetti, Rebecca Crudale, Micah A. Luftig, A. Moormann, Cliff I. Oduor, Jeffrey A. Bailey
          </td>
          <td>2025-02-25</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The study examines Opisthorchis viverrini (OV)-related cholangiocarcinoma (CCA), a serious malignancy common in Southeast Asia. Through multi-regional whole-exome sequencing of 52 tumor samples and 13 adjacent tissues from 13 patients, significant intratumoral heterogeneity (ITH) and inter-patient heterogeneity are shown. Chronic liver fluke infection induces a distinct mutational landscape, with 48–90% of mutations concentrated in each region of the tumor. The average mutation burden is 95 non-synonymous mutations per area, exceeding previous CCA investigations. Critical driver mutations in TP53, SMAD4, and other genes underscore their significance in pathogenesis. Mutational markers elucidate mechanisms including spontaneous deamination and impaired DNA repair. Unique mutation patterns distinguish OV-associated CCA from other variants. Chromosomal instability in patient K110 signifies aggressive tumor behavior and unfavorable prognosis. Targetable mutations such as ERBB2 underscore the possibility for personalized therapeutics. These findings underscore the necessity for personalized strategies for treatment that target both trunk and branch mutations in endemic areas. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-95142-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b48c940bd80f998f283a697250c46b3404991cb4" target='_blank'>
              Whole exome sequencing of multi-regions reveals tumor heterogeneity in Opisthorchis viverrini-associated cholangiocarcinoma
              </a>
            </td>
          <td>
            Sirinya Sitthirak, Arporn Wangwiwatsin, A. Jusakul, N. Namwat, P. Klanrit, Hasaya Dokduang, P. Sa-Ngiamwibool, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Bin Tean Teh, Luke Boulter, Yoshinori Murakami, W. Loilome
          </td>
          <td>2025-03-29</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a2a1bc086a943a7e73ef25816e4af3f165ae6d1" target='_blank'>
              PARP inhibitors affect the replisome and replication fork progression during a single S phase
              </a>
            </td>
          <td>
            Lina-Marie Briu, C. Maric, Nicolas Valentin, Nicolas Panara, Guillaume Chevreux, Giuseppe Baldacci, Jean-Charles Cadoret
          </td>
          <td>2025-02-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="ABSTRACT Human endogenous retroviruses (HERVs) are proviral relics of infections that affected primates' germ line. Many HERV elements retain a residual capacity to encode transcripts and proteins that have been occasionally domesticated for the host physiology. In addition, HERV transcriptional modulation is of great interest to clarify the etiology of complex disorders such as cancer, even if a few studies assessed the specific HERV loci modulated in tumor tissues. In the present work, we used a transcriptomic approach to investigate the specific expression of ~3300 HERV loci in paired tumor and normal tissues of 7 colorectal cancer (CRC) patients. A total of 102 HERVs were significantly modulated in CRC, with a general tendency towards downregulation. Of note, among the 42 upregulated HERVs 23 belonged to the HERV‐H group, that is the most investigated in CRC. De novo transcriptome reconstruction and qPCR validation allowed to identify a transcript from a HERV‐H locus on chromosome Xp22.3 with high specific expression in CRC samples, potentially encoding for a partial Pol protein. These results provide a detailed description of HERV transcriptional variations in CRC and its interindividual variability, identifying a HERV‐H transcript that deserves further investigation for its possible impact on tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8d57225f528a23471e9e5e65e4532aff3132bd2" target='_blank'>
              HERV Modulation in Colorectal Carcinoma Patients: A Snapshot of Endogenous Retroviral Transcriptome
              </a>
            </td>
          <td>
            Nicole Grandi, Ching-Hsuan Liu, Saili Chabukswar, Daniele Carta, Yun Yen, Liang-Tzung Lin, E. Tramontano
          </td>
          <td>2025-02-24</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer cells harbor cell-intrinsic programs capable of sculpting immunogenic landscapes, yet molecular brakes that enforce immune evasion remain poorly defined. Here, we identify C9ORF50, a previously uncharacterized intrinsically disordered protein, as a liquid-liquid phase separation-driven regulator of RNA processing and actionable immunotherapy target. Genetic ablation of C9ORF50 induces selective intron retention in spliceosome component transcripts, leading to cytoplasmic accumulation of immunogenic double-stranded RNA (dsRNA). This dsRNA activates type I interferon pathway that in turn orchestrates a chemokine secretory program critical for T cell recruitment. The resultant enhancement of tumor immunogenicity and reprogramming of tumor immune microenvironment drives robust antitumor immunity, effectively converting immunologically “cold” tumors to “hot” phenotypes. Therapeutic targeting of C9ORF50 via RNA interference achieved robust tumor suppression in syngeneic models, suggesting its translational relevance. Notably, C9ORF50 exhibits tumor-predominant expression, with minimal baseline levels detected in healthy tissues or immune cell populations, underscoring its potential safety as a therapeutic target. Our study not only identifies C9ORF50 as a novel LLPS-dependent regulator of RNA splicing but also establishes it as a promising and potentially safe therapeutic target for cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7433cc0492d9a830d01b0f8ac6ca2662ebe2b369" target='_blank'>
              C9orf50 is a Targetable Spliceosome Regulator for Cancer Immunotherapy
              </a>
            </td>
          <td>
            Tong Shao, Chuanyang Liu, Jingyu Kuang, Sisi Xie, Ying Qu, Yingying Li, Fangzhou Liu, Lulu Zhang, Ming Li, Tao Hou, Yanhua Qi, Jingyang Wang, Sujuan Zhang, Yu Liu, Jiali Liu, Yanming Hu, Shaowei Zhang, Lingyun Zhu, Jianzhong Shao, Aifu Lin, Guangchuan Wang, Lv-yun Zhu
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6879a0f12b521e3cdca31a407d6e24ccb72cd9a1" target='_blank'>
              AZD7648 (DNA-PKcs inhibitor): a two-edged sword for editing genomes.
              </a>
            </td>
          <td>
            Muhammad Waseem Sajjad, Ifrah Imran, Fatima Muzamil, R. Naqvi, Imran Amin
          </td>
          <td>2025-03-01</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract During the sexual cycle, programmed genome rearrangement in Paramecium tetraurelia involves the non-homologous end joining (NHEJ) DNA repair pathway to eliminate specific germinal internal eliminated sequences (IESs) from the newly developing somatic nucleus. Besides the core NHEJ factors Ku70/80 and Xrcc4/Lig4, additional enzymes are required to process the 4-base 5′-protruding ends generated following DNA cleavage at IES boundaries, prior to their ligation. Here, we report that PolXa,b,c,d, four P. tetraurelia distant orthologs of the human Polλ DNA polymerase, are involved in the repair of IES excision junctions. During rearrangements, PolX-depleted cells accumulate genome-wide errors, such as unrepaired double-strand breaks, one-nucleotide deletions, and IES retention. Although all PolX paralogs can process DNA ends, two of them (PolXa&b) are induced during rearrangements and have evolved a specific linker sequence downstream of their BRCT domain, which provides them with tight nuclear anchoring properties. We show that PolXa accumulates in nuclear foci together with other NHEJ actors and the Dicer-like enzyme Dcl5, which is involved in the biogenesis of IES-specific small RNAs. We propose that these ‘DNA repair foci’ correspond to the sites where IES concatemers, a by-product of IES excision, are ligated together to produce the precursors of these small RNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/669134220bd2c578be627533f4346df3e849e6a8" target='_blank'>
              The linker region of a development-specific DNA polymerase X ensures efficient repair of programmed DNA double-strand breaks in Parameciumtetraurelia
              </a>
            </td>
          <td>
            Baptiste Verron, O. Arnaiz, Coralie Zangarelli, Nathalie Mathy, M. Bétermier, J. Bischerour
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Fibroblast growth factor 1-4 (FGFR1-4) are well-known oncogenic drivers in many cancer types. Here, we studied the role of FGFRs in uterine leiomyoma (UL) that is a benign neoplasm arising from the myometrium and the most common tumour in women. Although ULs can be classified to molecular subtypes based on genetic drivers, potential secondary drivers are not well characterised. Methods We performed mutation analysis of RNA-sequencing data of ULs, followed by screening of FGFR alterations in our Finnish (n = 2677) and Swedish (n = 372) UL collections, utilising Sanger-, next-generation and Nanopore sequencing and SNP array data. The role of FGFR genes in UL predisposition was examined by GWAS. Results We identified FGFR activation in a subset of ULs on both genetic and epigenetic levels. In addition to single-nucleotide mutations in FGFR1/2, we detected an FGFR2-ERC1 fusion gene, FGFR1 gains and hypomethylation of regulatory regions of FGFR2/3. FGFR alterations were enriched in molecularly similar HMGA2, HMGA1 and PLAG1 UL subtypes. We also unveil a UL predisposing variant upstream of FGFR4 associated with increased expression in both normal myometrium and ULs. Conclusions Our results establish the role of FGFR signalling in the genesis of UL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee3a4ef449036c9f0b9b5728bbd660366041a80a" target='_blank'>
              Activation of FGFR genes by genetic and epigenetic alterations in uterine leiomyomas
              </a>
            </td>
          <td>
            Vilja Jokinen, Aurora Taira, Åsa Kolterud, Isa Ahlgren, Kimmo Palin, Riku Katainen, Maritta Räisänen, Eevi Kaasinen, Sini Ilves, A. Raitila, Helena Kopp Kallner, Emma Siili, Ralf Bützow, O. Heikinheimo, A. Pasanen, A. Karhu, Niko Välimäki, L. Aaltonen
          </td>
          <td>2025-02-27</td>
          <td>BJC Reports</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e62989c3597687f0bc3f412a31ec866e11a879ff" target='_blank'>
              Characterization of extrachromosomal circular DNA associated with genomic repeat sequences in breast cancer.
              </a>
            </td>
          <td>
            Wenxiang Lu, Lingsong Yao, Ying Wang, Fuyu Li, Bingbo Zhou, Wenlong Ming, Yali Jiang, Xiaoan Liu, Yun Liu, Xiao Sun, Yan Wang, Yunfei Bai
          </td>
          <td>2025-03-26</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Intratumoral heterogeneity is the main cause of tumor treatment failure, varying across disease sites (spatial heterogeneity) and polyclonal properties of tumors that evolve over time (temporal heterogeneity). As our understanding of intratumoral heterogeneity, the formation of which is mainly related to the genomic instability, epigenetic modifications, plastic gene expression, and different microenvironments, plays a substantial role in drug-resistant as far as tumor metastasis and recurrence. Understanding the role of intratumoral heterogeneity, it becomes clear that a single therapeutic agent or regimen may only be effective for subsets of cells with certain features, but not for others. This necessitates a shift from our current, unchanging treatment approach to one that is tailored against the killing patterns of cancer cells in different clones. In this review, we discuss recent evidence concerning global perturbations of intratumoral heterogeneity, associations of specific intratumoral heterogeneity in lung cancer, the underlying mechanisms of intratumoral heterogeneity potentially leading to formation, and how it drives drug resistance. Our findings highlight the most up-to-date progress in intratumoral heterogeneity and its role in mediating tumor drug resistance, which could support the development of future treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d2f0eb0a60c64efe2bd230e8933a5f80d2d405" target='_blank'>
              Intratumoral heterogeneity and drug resistance in cancer
              </a>
            </td>
          <td>
            Yue-Chun Fu, Shao-Bo Liang, Min Luo, Xue-Ping Wang
          </td>
          <td>2025-03-18</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, Gabriela García, Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bac213739380b0d30a929d69bfe478e51785426" target='_blank'>
              HP1-enhanced chromatin compaction stabilizes a synthetic metabolic circuit in yeast
              </a>
            </td>
          <td>
            Lidice González, Sergio A. García Echauri, C. Jeronimo, Christian Poitras, Melis Gencel, Adrian Serohijos, Kerry Bloom, François Robert, J. Avalos, S. Michnick
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Primary mouse renal proximal tubule epithelial cells (moRPTECs) were immortalized by lentivirus transduction to create hTERT or SV40LgT (LgT) cell lines. Prior work showed a more pronounced injury and repair response in LgT versus hTERT cells after chemical challenge. We hypothesized that unique genomic changes occurred after immortalization, altering critical genes and pathways. RNA-seq profiling and whole-genome sequencing (WGS) of parent, hTERT, and LgT cells showed that 92.5% of the annotated transcripts were shared, suggesting a conserved proximal tubule expression pattern. However, the cell lines exhibited unique transcriptomic and genomic profiles different from the parent cells. Three transcript classes were quite relevant for chemical challenge response—Cyps, ion channels, and metabolic transporters—each important for renal function. A pathway analysis of the hTERT cells suggested alterations in intermediary and energy metabolism. LgT cells exhibited pathway activation in cell cycle and DNA repair that was consistent with replication stress. Genomic karyotyping by combining WGS and RNA-seq data showed increased gene copy numbers in chromosome 5 for LgT cells, while hTERT cells displayed gene copy losses in chromosomes 4 and 9. These data suggest that the exaggerated transcriptional responses of LgT cells versus hTERT cells result from differences in gene copy numbers, replication stress, and the unique selection processes underlying LgT or hTERT immortalization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5fc2423810ee9671b81c235e2952419eb5bbc3d" target='_blank'>
              hTERT and SV40LgT Renal Cell Lines Adjust Their Transcriptional Responses After Copy Number Changes from the Parent Proximal Tubule Cells
              </a>
            </td>
          <td>
            B. A. Merrick, Ashley M Brooks, Julie F. Foley, N. Martin, Rick D. Fannin, Wesley Gladwell, Kevin E. Gerrish
          </td>
          <td>2025-04-11</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2eed7b2fdc5b17bdbf958e634178449e843736cc" target='_blank'>
              Genome-Wide CRISPR Screening Identifies Cellular Factors Controlling Nonviral Genome Editing Efficiency
              </a>
            </td>
          <td>
            Shivani Saxena, Meha Kabra, Amr Abdeen, Divya Sinha, Min Zhu, Ruosen Xie, Giovanni Hanstad, Maria A. Fernandez Zepeda, David M. Gamm, B. Pattnaik, Shaoqin Gong, Krishanu Saha
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279bd9fa8c8b49a99cb9a4bb9596137f5ef0a74d" target='_blank'>
              Sc-TUSV-Ext: Single-Cell Clonal Lineage Inference from Single Nucleotide Variants, Copy Number Alterations, and Structural Variants.
              </a>
            </td>
          <td>
            N. Bristy, Xuecong Fu, Russell Schwartz
          </td>
          <td>2025-03-06</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chromosomal instability (CIN) is a hallmark of cancer that drives metastasis, immune evasion and treatment resistance. CIN may result from chromosome mis-segregation errors and excessive chromatin is frequently packaged in micronuclei (MN), which can be enumerated to quantify CIN. The assessment of CIN remains a predominantly manual and time-consuming task. Here, we present micronuclAI, a pipeline for automated and reliable quantification of MN of varying size and morphology in cells stained only for DNA. micronuclAI can achieve close to human-level performance on various human and murine cancer cell line datasets. The pipeline achieved a Pearson’s correlation of 0.9278 on images obtained at 10X magnification. We tested the approach in otherwise isogenic cell lines in which we genetically dialed up or down CIN rates, and on several publicly available image datasets where we achieved a Pearson’s correlation of 0.9620. Given the increasing interest in developing therapies for CIN-driven cancers, this method provides an important, scalable, and rapid approach to quantifying CIN on images that are routinely obtained for research purposes. We release a GUI-implementation for easy access and utilization of the pipeline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ad501d60562e0f7baf32d920b40de21a7a01759" target='_blank'>
              micronuclAI enables automated quantification of micronuclei for assessment of chromosomal instability
              </a>
            </td>
          <td>
            Miguel A. Ibarra-Arellano, Lindsay A. Caprio, Aroj Hada, Niklas Stotzem, Luke L Cai, Shivem B Shah, Zachary H. Walsh, Johannes C. Melms, Florian Wünneman, K. Bestak, Ibrahim Mansaray, Benjamin Izar, Denis Schapiro
          </td>
          <td>2025-03-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Tumors exhibit highly heterogeneous genomic and transcriptomic landscapes. Characterization of this intratumor heterogeneity (ITH) and its implications for the cancer therapies has been one of the focal points of computational cancer biology in the past years. However, while ITH at the level of genomic changes, such as copy number aberrations, has been well characterized, the downstream impact of ITH on the gene regulatory landscape remains underexplored. Disruptions in gene regulatory programs can lead to emergence of more aggressive or drugresistant tumor clones. In order to better understand whether the evidence for ITH at the gene regulatory level can be inferred from scRNA-seq data we have conducted a series of experiments that explore the levels of gene regulatory network (GRN) discrepancies between the clonal sub-populations of tumor cells. In particular, we quantify the extent to which GRNs inferred from different tumor clones diverge from each other in three settings: triple negative breast cancer, colorectal cancer, and a longitudinal dataset from a patient undergoing chemotherapy for breast cancer. Our analyses indicate that a substantial (15-25%) proportion of GRN edges can be attributed to clone-specific activity. Furthermore, we observe differential network patterns across outlier transcription factors implicated in tumorogenesis and tumor progression. Our findings suggest that differential network patterns between clonal subpopulations can be reliably identified, and that such patterns offer a complementary insight that is not captured by differential gene analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86413a2b4bf81ee584f265b2f474b3de585e8771" target='_blank'>
              Intratumor Heterogeneity Through the Lens of Gene Regulatory Networks
              </a>
            </td>
          <td>
            Narein Rao, Nicolae Sapoval, Hamim Zafar, L. Nakhleh
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e688482cb691231e9728515c865331942ee770" target='_blank'>
              Integrating gene expression, mutation and copy number data to identify driver genes of recurrent chromosome-arm losses
              </a>
            </td>
          <td>
            Ron Saad, Ron Shamir, Uri Ben-David
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e93a4683d0794daa101a332796e1ced942dd7fd5" target='_blank'>
              A Novel De Novo Exceptional Complex Chromosomal Rearrangement Involving 5 Chromosomes Resulting in Neurodevelopmental Delay and Dysmorphism
              </a>
            </td>
          <td>
            Sabri Aynacı, Sinem Kocagil, C. Yarar, Efsun Tosumoğlu, Ebru Erzurumluoğlu Gökalp, Mehmet Burak Mutlu, S. Artan
          </td>
          <td>2025-03-25</td>
          <td>Molecular Syndromology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Despite exhaustive research efforts, integrated genetic and epigenetic mechanisms contributing to tobacco-induced initiation and progression of lung cancers have yet to be fully elucidated. In particular, limited information is available regarding dysregulation of noncoding RNAs during pulmonary carcinogenesis. Methods We examined correlations and interactions of long noncoding (lnc) RNAs and protein-coding genes in normal respiratory epithelial cells (NREC) and pulmonary tumor cells following exposure to cigarette smoke condensate (CSC) using gene expression arrays, qRT-PCR, western blot, growth assays, transwell assays, and murine xenograft models, as well as methylated DNA immunoprecipitation, RNA cross-link immunoprecipitation, and quantitative chromatin immunoprecipitation techniques with bioinformatics analyses. Results Among diverse alterations of lncRNA and coding gene expression profiles in NREC exposed to CSC, we observed upregulation of lncRNA ZFAS1 and repression of an adjacent protein-coding gene, ZNFX1, and confirmed these findings in primary lung cancers. Phenotypic experiments indicated that ZFAS1 is an oncogene, whereas ZNFX1 functions as a tumor suppressor in lung cancer cells. Mechanistically, CSC induces ZFAS1 expression via SP1 and NFĸB-associated activation of an enhancer linked to ZFAS1. Subsequently, ZFAS1 interacts with DNA methyltransferases and polycomb group proteins to silence ZNFX1. Mithramycin and methysticin repress ZFAS1 and upregulate ZNFX1 in lung cancer cells in vitro and in vivo. Conclusion These studies reveal a novel feedforward lncRNA circuit contributing to pulmonary carcinogenesis and suggest that pharmacologic targeting of SP1 and/or NFĸB may be useful strategies for restoring ZNFX1 expression for lung tumor therapy. Supplementary Information The online version contains supplementary material available at 10.1186/s11658-025-00705-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/567616f0aebcc58b7e08486a13708671c61a57ab" target='_blank'>
              Repression of ZNFX1 by LncRNA ZFAS1 mediates tobacco-induced pulmonary carcinogenesis
              </a>
            </td>
          <td>
            Sichuan Xi, J. Shan, Xinwei Wu, Haitao Wang, Mary R. Zhang, Shakirat O Oyetunji, Hong Xu, Zuoxiang Xiao, Tuana Tolunay, Shamus R Carr, Chuong D. Hoang, D.S. Schrump
          </td>
          <td>2025-04-10</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Despite the striking conservation of animal chromosomes, their repetitive element complements are vastly diverse. Only recently, high quality chromosome-level genome assemblies enabled identification of repeat compositions along a broad range of animal chromosomes. Here, utilizing the chromosome-level genome assembly of Hydractinia symbiolongicarpus, a colonial hydrozoan cnidarian, we describe an accumulation of a single 372 bp repeat unit in the subtelomeric regions. Based on the sequence divergence, its partial affinity with the Helitron group can be detected. This sequence is associated with a repeated minisatellite unit of about 150 bp. Together, they account for 26.1% of the genome (126 Mb of the 483 Mb). This could explain the genome size increase observed in H. symbiolongicarpus compared with other cnidarians, yet distinguishes this expansion from other large cnidarian genomes, such as Hydra vulgaris, where such localized propagation is absent. Additionally, we identify a derivative of an IS3EU-like DNA element accumulated at the putative centromeric regions. Our analysis further reveals that Helitrons generally comprise a large proportion of H. symbiolongicarpus (11.8%). We investigated Helitron presence and distributions across several cnidarian genomes. We find that in Nematostella vectensis, an anthozoan cnidarian, Helitron-like sequences were similarly accumulated at the subtelomeric regions. All these findings suggest that Helitron derivatives are prone to forming chromosomal extensions in cnidarians through local amplification in subtelomeric regions, driving variable genome expansions within the clade. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00355-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91fb42c253af0c01d931cd52e6e5278687b51b25" target='_blank'>
              Subtelomeric repeat expansion in Hydractinia symbiolongicarpus chromosomes
              </a>
            </td>
          <td>
            Tetsuo Kon, Koto Kon-Nanjo, Oleg Simakov
          </td>
          <td>2025-03-25</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="During mitosis, pairing of homologous chromosomes can be detrimental and has been correlated with gene misregulation, chromosomal aberrations, and various pathological diseases. We previously demonstrated that homologous chromosomes are spatially segregated, or antipaired, in neonatal human endothelial cells at metaphase/anaphase, which may help prevent abnormal recombination. However, it is unclear if this antipairing persists in adult endothelial cells. To test whether the antipairing, or one homolog per nuclear hemisphere motif, is conserved in adult endothelial cells, we examined human aortic endothelial cells at metaphase. Using ImmunoFISH and high-resolution confocal microscopy to visualize the chromosomes and centrosomes, we found that small homologous chromosomes 13, 15, 17, 19, 21, 22, and the sex chromosomes, XY, exhibit a loss of spatial segregation in human adult aortic endothelial cells. In contrast, fewer adult endothelial cells showed a loss of segregation for the larger chromosomes 1, 4, and XX, suggesting a gradual decline in the fidelity of spatial segregation of homologous chromosomes. Notably, we observed a higher frequency of abnormal pairing in both small and large chromosomes in adult aortic endothelial cells as compared to neonatal umbilical vein endothelial cells. These findings suggest that mechanisms governing chromosome antipairing may decline with aortic endothelial cell age, leading to increased susceptibility to abnormal pairing and cardiovascular disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30619766655f6da41d03dd46682fe70a005b2ede" target='_blank'>
              Higher frequency of homologous chromosome pairing in human adult aortic endothelial cells
              </a>
            </td>
          <td>
            Jemery Morales, Gabriel Quintero Plancarte, Lisa Hua
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ee29e34be0729357213c5d8d79a8e1845ecaf78" target='_blank'>
              Multidimensional characterization of cellular ecosystems in Hodgkin lymphoma
              </a>
            </td>
          <td>
            Tomohiro Aoki, Gerben Duns, Shinya Rai, A. Jiang, Andrew Lytle, Yifan Yin, Makoto Kishida, Michael Li, Denise Smorra, Laura K. Hilton, S. Alig, Mohammad Shahrokh Esfahani, C. Sarkozy, S. Hung, K. Milne, A. Telenius, Luke O'Brien, Celia Strong, Talia Goodyear, Chantal Di Vito, Cassandra Luksik, Glenn Edin, Laura Gonzalez, Juan Patino Rangel, Michael Hong, Shaocheng Wu, Eric Lee, Katsuyoshi Takata, T. Miyata‐Takata, M. Boyle, S. Ben-Neriah, Andrew P. Weng, Alexander M Xu, Akil Merchant, Andrew Roth, Michael Crump, John Kuruvilla, A. Prica, R. Kridel, D. Huntsman, Brad H Nelson, P. Farinha, Ryan D Morin, A. Alizadeh, K. Savage, David W. Scott, Christian Steidl
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38eeccd7f5dc5d4b304bd2c0df97ce58a72f338a" target='_blank'>
              RAmbler resolves complex repeats in human Chromosomes 8, 19, and X.
              </a>
            </td>
          <td>
            Sakshar Chakravarty, Glennis Logsdon, Stefano Lonardi
          </td>
          <td>2025-03-04</td>
          <td>Genome research</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="DNA packaged into cancer cell-derived EV is not well appreciated. Here, we uncovered signatures of EV DNA secreted by pancreatic cancer cells. The cancer cells and non-cancer counterparts exhibit distinct low vs. high molecular weight (LMW vs. HMW) EV DNA fragments distribution, respectively. Genome sequencing and Single Nucleotide Variants analysis revealed that 95% of reads and 94% of SNVs map to noncoding regions of the genome. Given that ~1% of the human genome represents coding regions, the 5% mapping rate to coding regions suggests a non-random enrichment of certain coding regions and mutations. The LMW DNA fragments not only set cancer cells apart, but also harbor cancer specific enrichment of unique coding regions, the top nine being FAM135B, COL22A1, TSNARE1, KCNK9, ZFAT, JRK, MROH5, GSDMD, and MIR3667HG. Additionally, the cancer cells’ LMW DNA fragments exhibit dense centromeric mapping more strikingly on chromosomes 3, 7, 9, 10, 11, 13, 17, and 20. Mutational profiling turned up close to 200 mutations specific for the cancer cells. Altogether, our analyses suggest that centromeric regions might hold clues to EV DNA content from pancreatic cancer, the molecular, mutational signatures thereof, and rationalizes the need for a new approach to DNA biomarker research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe40161d2aa729850b5719de103597d5d50c55e2" target='_blank'>
              EV DNA from pancreatic cancer patient-derived cells harbors molecular, coding, non-coding signatures and mutational hotspots
              </a>
            </td>
          <td>
            Appolinaire A Olou, Wesley A Tom, Gary D. Krzyzanowski, Chao Jiang, Dinesh S Chandel, Nirmalee Fernando, Adrian W. Draney, Joel Destino, Danny R Welch, M. R. Fernando
          </td>
          <td>2025-03-05</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Antisense oligonucleotides (ASOs) and small interfering RNA (siRNA) therapeutics highlight the power of oligonucleotides in silencing disease-causing messenger RNAs (mRNAs). Another promising class of gene-silencing oligonucleotides is RNA-cleaving nucleic acid enzymes, which offer the potential for allele-specific RNA inhibition with greater precision than ASOs and siRNAs. Herein, we chemically evolved the nucleolytic DNA enzyme (DNAzyme) 10–23, by incorporating the modifications that are essential to the success of ASO drugs, including 2′-fluoro, 2′-O-methyl, and 2′-O-methoxyethyl RNA analogues, and backbone phosphorothioate, to enhance catalytic efficiency by promoting RNA substrate binding and preventing dimerization of 10–23. These ASO-like DNAzymes cleaved structured RNA targets in long transcripts, showed prolonged intracellular stability, and downregulated mRNA and protein levels of both exogenously transfected eGFP and endogenously elevated oncogenic c-MYC. In colon cancer HCT116 cells, the downregulation of oncogenic c-MYC RNA resulted in cell cycle arrest, reduced proliferation, and increased apoptosis. RACE (rapid amplification of cDNA ends) polymerase chain reaction and Sanger sequencing confirmed precise, site-specific mRNA transcript cleavage with minimal RNase H activation in cells. By merging ASO structural and pharmacokinetic advantages with DNAzyme catalytic versatility, these ASO-like 10–23 variants offer a promising new class of potent gene-silencing agents, representing a significant step toward therapeutic DNAzyme development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0676a1fc6fbd741743702722cb6c5dbd58ba89e" target='_blank'>
              Chemical evolution of ASO-like DNAzymes for effective and extended gene silencing in cells
              </a>
            </td>
          <td>
            Yingyu Liu, Sheyu Zhang, Meiqi Zhang, Xin Liu, Yashu Wu, Qin Wu, John C Chaput, Yajun Wang
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The human APOBEC superfamily consists of eleven cytidine deaminase enzymes. Among them, APOBEC3 enzymes play a dual role in antiviral immunity and cancer development. APOBEC3 enzymes, including APOBEC3A (A3A) and APOBEC3B (A3B), induce mutations in viral DNA, effectively inhibiting viral replication but also promoting somatic mutations in the host genome, contributing to cancer development. A3A and A3B are linked to mutational signatures in over 50% of human cancers, with A3A being a potent mutagen. A3B, one of the first APOBEC3 enzymes linked to carcinogenesis, plays a significant role in HPV-associated cancers by driving somatic mutagenesis and tumor progression. The A3A_B deletion polymorphism results in a hybrid A3A_B gene, leading to increased A3A expression and enhanced mutagenic potential. Such polymorphism has been linked to an elevated risk of certain cancers, particularly in populations where it is more prevalent. This review explores the molecular mechanisms of APOBEC3 proteins, highlighting their dual roles in antiviral defense and tumorigenesis. We also discuss the clinical implications of genetic variants, such as the A3A_B polymorphism, mainly in HPV infection and associated cancers, providing a comprehensive understanding of their contributions to both viral restriction and cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2370c8c875291e3fc4adec0ad421696e86fb060" target='_blank'>
              APOBEC3 Proteins: From Antiviral Immunity to Oncogenic Drivers in HPV-Positive Cancers
              </a>
            </td>
          <td>
            Eliza Pizarro Castilha, Rosalba Biondo, K. P. Trugilo, Giulia Mariane Fortunato, Tim R. Fenton, Karen Brajão de Oliveira
          </td>
          <td>2025-03-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Chromosomal instability (CIN), characterized by pervasive copy number alterations (CNAs), significantly contributes to cancer progression and therapeutic resistance. CNAs drive intratumoral genetic heterogeneity, creating distinct subpopulations whose interactions shape tumor evolution through frequency-dependent selection. Here, we introduce, ECO-K (Ecological-Karyotypes), an inverse game theory framework that infers subpopulation interactions from longitudinal single-cell whole genome sequencing data. Applying this approach to serially-passaged, triple-negative breast cancer (TNBC) cell lines and patient-derived xenografts (PDXs), we systematically identified frequency-dependent selection dynamics governed by karyotypic diversity. Notably, in one PDX lineage, we found consistent evidence of karyotypically defined subpopulations acting as interaction hubs, associated specifically with chromosome 1 loss and chromosome 14p gain. Our framework provides testable predictions of intratumoral ecological dynamics, highlighting opportunities to strategically target key subpopulations to disrupt tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dffff279b4407d3734303c6d5c0f841102d26d8b" target='_blank'>
              Inverse Game Theory Characterizes Frequency-Dependent Selection Driven by Karyotypic Diversity in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Thomas Veith, Richard Beck, Joel S. Brown, Noemi Andor
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT Replication timing during S-phase impacts mutation rates in yeast and human cancers; however, the exact mechanism involved remains unclear. Here, we analyze the impact of replication timing on UV mutagenesis in Saccharomyces cerevisiae. Our analysis indicates that UV mutations are enriched in early-replicating regions of the genome in wild-type cells, but in cells deficient in global genomic-nucleotide excision repair (GG-NER), mutations are enriched in late-replicating regions. Analysis of UV damage maps revealed that cyclobutane pyrimidine dimers are enriched in late-replicating regions, but this enrichment is almost entirely due to repetitive ribosomal DNA. Complex mutations typically associated with TLS activity are also elevated in late-replicating regions in GG-NER deficient cells. We propose that UV mutagenesis is higher in early-replicating regions in repair-competent cells because there is less time to repair the lesion prior to undergoing replication. However, in the absence of GG-NER, increased TLS activity promotes UV mutagenesis in late-replicating regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36033ab9a3a3a64a0722f7e9267a3d283b2c3ca3" target='_blank'>
              Interplay of replication timing, DNA repair, and translesion synthesis in UV mutagenesis in yeast
              </a>
            </td>
          <td>
            Allysa Sewell, John J. Wyrick
          </td>
          <td>2025-03-13</td>
          <td>Nucleus</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0be267d81c51ec120632413ac620f868fe9f5634" target='_blank'>
              Rapid delivery of RNase H1 enables replication fork progression under stress independently of R-loop resolution
              </a>
            </td>
          <td>
            Amandine Basille, Yea-Lih Lin, Connor Kirke, D. Cluet, Yara Bawadi, Laetitia Vachez, Virgile Dervieux, Laura Guiguettaz, Pascal Bernard, Stephan Hamperl, Philippe Pasero, Emiliano P Ricci, Vincent Vanoosthuyse
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e024d7323084416b4b6e84dca254aca71e425ab" target='_blank'>
              Learning the cellular origins of cancer using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Bruno Giotti, Wooseung Lee, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="X-chromosome inactivation (XCI) is a fundamental mechanism in placental mammals that compensates for gene dosage differences between sexes. Using methylation levels of genes under XCI, we establish defective levels of XCI as a new source of interindividual variation among cancer types in females, characterized by a significant and consistent lowering of XIST expression and enrichment of differentially expressed genes under XCI. We show that defective XCI is an additive factor to the cancer risk of XCI escape deregulation in women. Defective XCI of more than 10% has an attributable risk of 40% among 12 different cancers from The Cancer Genome Atlas. Validations between independent studies of breast cancer samples show that defective XCI increases triple-negative subtype frequency, decreases survival rates, and is reduced by chemotherapy treatment. Mechanistically, it is associated with somatic mutations at TP53 and top MYC gains. In independent studies, defective XCI is detectable in blood and increases with aging, menopause, and cancer diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/303ef18264231a4b1a297a874c3c158036c417de" target='_blank'>
              Defective X-chromosome inactivation and cancer risk in women
              </a>
            </td>
          <td>
            A. Cáceres, L. Pérez-Jurado, Albert Alegret-García, Varun B. Dwaraka, Ryan Smith, Juan R. González
          </td>
          <td>2025-02-22</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Chromosomal alterations are frequent events in lung cancer progression. Although gains and losses of chromosomal position have been reported, the association between copy number alteration and lung cancer patient survival has not been extensively investigated. In this study, we performed a meta-analysis of public cBioPortal datasets spanning 25 lung cancer studies to identify putative cancer driver genes with copy number alterations associated with overall patient survival. Ten copy-number altered genes enriched in deceased lung cancer patients were identified. Seven of these putative driver genes were located in the 7p11.2 chromosomal location, and two were in the 9p21.3 cytoband. Among these genes, the mitochondrial ribosomal protein S17 (MRPS17) amplification was significantly associated with a lower patient survival rate (P = 1.47e-7). To investigate the functional role of MRPS17, small interfering RNA-mediated knockdown was performed in two non-small cell lung cancer cell lines, A549 and NCI-H460. MRPS17 knockdown significantly reduced cell proliferation, migration, invasion, and anchorage-independent growth in both cell lines. Furthermore, knockdown of MRPS17 decreased the activation of the phosphatidylinositol 3-kinase/protein kinase B signaling pathway, suggesting its role in driving lung cancer progression through this critical oncogenic pathway. Our findings highlight MRPS17 as a potential cancer therapy target and a prognostic biomarker that may improve the survival rates of lung cancer patients. Future studies should explore its inhibition as a therapeutic strategy as well as elucidate its molecular mechanisms in cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f1c0b37271452f619e3684f3550b14101ddebda" target='_blank'>
              Mitochondrial Ribosomal Protein S17 Silencing Inhibits Proliferation and Invasiveness of Lung Cancer Cells
              </a>
            </td>
          <td>
            Woo Rin Lee, Kook Sun Ha
          </td>
          <td>2025-03-30</td>
          <td>Journal of Cancer Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Over 80% of children with acute lymphoblastic leukemia (pALL) can be cured by treating them with multiple chemotherapeutic agents administered over several years, whereas pALL is incurable with 1-3 medications, suggesting significant variation in drug susceptibility across clonal populations. While bulk sequencing studies indicate that pALL cells contain relatively few genetic variants compared to other cancers, the true extent of genetic diversity at the single-cell level remains unknown. Here, we used three complementary approaches to investigate pALL genetic heterogeneity: error-corrected bulk sequencing, single-cell exome sequencing, and primary template-directed amplification (PTA)-enabled single-cell genome sequencing. We discovered that some ETV6-RUNX1 samples harbor multiple independent ras clones and that individual pALL cells harbor substantially more mutations (mean 3,553 per cell) than detected in bulk samples (mean 965 mutations), with variant signatures suggesting both early and late APOBEC-driven mutagenesis in ETV6-RUNX1 patients. Using PTA-based phylogenetic analysis of over 150 single-cell genomes from four pALL patients, we identified heritable phenotypes associated with specific genetic alterations, including some low-frequency clones that are preferentially selected for during chemotherapy treatment. Our findings reveal previously undetected genetic diversity in pALL and suggest that pre-existing mutations influence treatment response, with implications for future therapeutic strategies. This study provides a high-resolution framework for understanding cancer clonal evolution during treatment, yielding important new insights for developing more effective therapeutic approaches for pALL. Key Points Increasing the accuracy and resolution of sequencing from bulk to error-corrected bulk to single clone to single-cell sequencing reveals increasing levels of genetic diversity in pALL, including the presence of clone-specific driver and treatment resistance-associated mutations. Using PTA-enabled scWGS, single pALL cells contain several fold more mutations per cell than detected in bulk sequencing, which, when multiplied across billions of cells, reveal hidden population-scale genetic complexity. Some phenotypes in pALL are heritable, including treatment resistance, where we identify pre-existing rare clones with relapse-associated mutations that preferentially survive four weeks of standard treatment in patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cfebdd69571f854dae1eb84acf1f1884a093ad9" target='_blank'>
              Measuring Longitudinal Genome-wide Clonal Evolution of Pediatric Acute Lymphoblastic Leukemia at Single-Cell Resolution
              </a>
            </td>
          <td>
            Yakun Pang, Tamara Prieto, V. Gonzalez-Pena, Athena Aragon, Yuntao Xia, Sheng Kao, Sri Rajagopalan, John Zinno, Jean Quentin, Julien Laval, Dennis Yuan, Nathaniel D. Omans, David Klein, Matthew MacKay, I. De Vlaminck, John Easton, William Evans, D. Landau, Charles Gawad
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b20395b5511b0ca628f7388834f767639654823" target='_blank'>
              Conserved and Unique Features of Terminal Telomeric Sequences in ALT-Positive Cancer Cells
              </a>
            </td>
          <td>
            Benura Azeroglu, Wei Wu, R. Pavani, Ranjodh Sandhu, Tadahiko Matsumoto, A. Nussenzweig, E. L. Denchi
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="This study explores the multifaceted role of p21 in mediating cellular responses to DNA-damaging agents, with a focus on doxorubicin treatment in HCT116 colon carcinoma cells. We investigated how different doses of doxorubicin affect cells with varied p21 and p53 statuses, revealing distinct roles for p21 depending on the drug dosage. At high doses (HD), p21 is more critical than p53 in mediating apoptosis, whereas at low doses (LD), p21 is essential for preventing mitotic defects and multinucleation. Notably, reintroducing p21 or pharmacologically inhibiting CDK1 reduced multinucleation. The absence of p21 upon LD doxorubicin exposure led to aberrant chromosome segregation, persistent DNA damage response (DDR) activation, and increased non-homologous end-joining (NHEJ) activity, resulting in unrepaired DNA accumulation and multinucleation. Additionally, mitotic defects in p21-deficient cells were associated with mislocalization of key mitotic regulators, Aurora B and mitotic kinesin-like protein 1 (MKLP1), exacerbating defective mitosis. In summary, p21 functions as a dual regulator in response to DNA damage, promoting apoptosis at HD and preventing mitotic failure at LD. These insights have significant implications for cancer therapy, highlighting the potential of targeting the p21 to enhance treatment efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62798e3e091c52a2538fbed6da27e2c926098463" target='_blank'>
              Dual role of p21 in regulating apoptosis and mitotic integrity in response to doxorubicin in colon cancer cells
              </a>
            </td>
          <td>
            Heeyeon Kim, Haein Kim, Eunjung Jang, Young-Woo Eom, Gyesoon Yoon, Kyeong Sook Choi, Eunhee Kim
          </td>
          <td>2025-04-02</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68496fd9e9a9f934f03f0dd67116b1c071c33e67" target='_blank'>
              Diploid genome assembly of human fibroblast cell lines enables clone specific variant calling, improved read mapping and accurate phasing
              </a>
            </td>
          <td>
            T. R. Ranallo-Benavidez, Yue Hao, Emilia Volpe, Maryam Jehangir, Noelle Fukushima, Ogechukwu Mbegbu, Rebecca Reiman, Jessica Molnar, Danyael Murphy, Dorothy Marie Paredes, Shukmei Wong, Kara Karaniuk, Stephanie Buchholtz, Jonathan Keats, M. Machiela, M. Kolmogorov, B. Paten, Simona Giunta, Floris P Barthel
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/567b69c3e9b7e6f973b80fe210ce55ed16d03194" target='_blank'>
              Ancestral origin and structural characteristics of non-syntenic homologous chromosomes in abalones
              </a>
            </td>
          <td>
            S. Hirase, Takashi Makino, Takeshi Takeuchi, Mitsutaka Kadota, Shigehiro Kuraku, Kiyoshi Kikuchi
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Apurinic/apyrimidinic (AP) sites and single-strand breaks arising from base excision repair (BER) during the misincorporation of damaged nucleobases may hinder replication fork stability in homologous recombination–deficient (HRD) cells. At templated AP sites, cross-links between the DNA and 5-hydroxymethylcytosine binding, embryonic stem cell–specific (HMCES) regulate replication fork speed, avoiding cytotoxic double-strand breaks. While the role of HMCES at the template DNA strand is well studied, its effects on nascent DNA are not. We provide evidence that HMCES–DNA-protein cross-links (DPCs) are detrimental to the BER-mediated removal of 5-hydroxymethyl-2′-deoxycytidine (5hmdC)–derived 5-hydroxymethyl-2′-deoxyuridine from replication forks. HRD cells have heightened HMCES-DPCs, which increase further upon 5hmdC exposure, suggesting that HMCES binds both spontaneous and 5hmdC-induced AP sites. HMCES depletion substantially suppresses 5hmdC-mediated replication fork defects, chromosomal aberrations, and cell death in HRD cells. This reveals that HMCES-DPCs are a source of BER-initiated single-stranded DNA gaps, which indicates that endogenous DPCs contribute to genomic instability in HRD tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c622156d932ecc5d7d331e4371a1b29a1f41c3ef" target='_blank'>
              HMCES corrupts replication fork stability during base excision repair in homologous recombination–deficient cells
              </a>
            </td>
          <td>
            María José Peña-Gómez, Yaiza Rodríguez-Martín, Marta del Rio Oliva, Yodhara Wijesekara Hanthi, Sara Berrada, Raimundo Freire, Jean-Yves Masson, José Carlos Reyes, Vincenzo Costanzo, Iván V. Rosado
          </td>
          <td>2025-03-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancers that arise from germline mutations of breast cancer associated gene 1 (BRCA1), which is a crucial player in homologous recombination (HR) DNA repair, are vulnerable to DNA-damaging agents such as platinum and PARP inhibitors (PARPis). Increasing evidence suggests that BRCA1 is an essential driver of all phases of the cell cycle, thereby maintaining orderly steps during cell cycle progression. Specifically, loss of BRCA1 activity causes the S-phase, G2/M, spindle checkpoints, and centrosome duplication to be dysregulated, thereby blocking cell proliferation and inducing apoptosis. In vertebrates, loss of HR genes such as BRCA1 and/or BRCA2 is lethal, since HR is a prerequisite for genome integrity. Thus, cancer cells utilize alternative DNA repair pathways such as non-homologous end joining (NHEJ) to cope with the loss of BRCA1 function. In this review, we attempt to update and discuss how these novel components are crucial for regulating DNA damage repair (DDR) in BRCA1-deficient cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ca96851a56316ea96ea1fc73475ff478d721c03" target='_blank'>
              Overview of Roles of Novel Components in the Regulation of DNA Damage Repair in BRCA1-Deficient Cancers: An Update
              </a>
            </td>
          <td>
            Nhat Nguyen, Dominic Arris, M. T. Tran
          </td>
          <td>2025-04-01</td>
          <td>DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT ALK fusions, such as the classic EML4‐ALK, are known drivers of lung cancer and effective therapeutic targets. However, variant ALK fusions, including intergenic fusions like LOC388942‐ALK (LA), have been detected in increasing numbers of patients, with their roles in tumorigenesis and ALK inhibitor resistance remaining unclear. Using CRISPR/Cas9, we generated the LA fusion in A549 and H441 cells, confirming elevated ALK expression via qRT‐PCR and immunohistochemistry (IHC) staining. Functional analyses showed that LA significantly promoted tumor growth in vitro and in vivo while conferring increased resistance to alectinib. RNA‐seq revealed upregulation of the FOS pathway in LA tumors, identifying FOS as a potential therapeutic target. Subsequently, we demonstrated that FOS disruption and inhibition sensitized LA tumors to treatment. RNA‐seq profiling demonstrated that FOS depletion in LOC388942‐ALK tumor significantly downregulated multiple oncogenic pathways related to cell cycle progression, DNA replication fidelity, and extracellular matrix remodeling, suggesting a pivotal role of FOS in maintaining tumor growth. These findings establish LOC388942‐ALK as a novel oncogenic driver in lung cancer, highlighting its role in tumor growth and ALK inhibitor resistance. Targeting FOS may provide a promising therapeutic strategy for tumors harboring this intergenic fusion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/459d6322ba2d77e523434dfb3f269f42f6deaabb" target='_blank'>
              Identifying the Intergenic ALK Fusion LOC388942‐ALK as a Driver of Non–Small Cell Lung Cancer
              </a>
            </td>
          <td>
            X. Zhai, Manli Wang, Qi Zhang, Donglin Li, Yanmou Wu, Zuoyu Liang, Jiewei Liu, Weiya Wang, Yu Liu, Guowei Che, Qinghua Zhou, Chong Chen
          </td>
          <td>2025-03-27</td>
          <td>MedComm</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059cc807139dc64898d45632991b55c44cdfb1d6" target='_blank'>
              Abstract A108: Sub-clonal copy number variations between on-treatment lesions in immunotherapy resistant melanoma patients
              </a>
            </td>
          <td>
            Muyi Yang, Sólrún Kolbeinsdóttir, Katarina Hammarlund, V. Grozman, Martin Enge, Suzanne Egyhazi-Brage, H. Helgadottir
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background The reactivation of developmental silenced γ-globin genes (HBG1/2) has shown promise as a therapeutic strategy for improving symptoms of β-hemoglobinopathies. Currently, the focus of therapeutic targets is primarily on the major fetal hemoglobin suppressors, such as BCL11A and ZBTB7A and of their binding sites on the proximal HBG promoter. However, the role of the distal HBG promoter in regulating gene expression remains to be explored. Methods We used CRISPR/Cas9 system to edit the distal HBG promoter. In vitro and in vivo assays, as well as engrafted NCG-Kit-V831M mice, were used for functional validation and mechanistic studies. Results We discovered an insertion of nucleotide A (insA) between − 1368 and − 1369 bp upstream of the TSS in HBG2 resulting in remarkable increase in γ-globin expression in HUDEP-2 cells. We also observed elevated γ-globin expression in human CD34+ erythroid progenitor cells from healthy individuals and those with β-thalassemia when introducing insA mutation. Similarly, engrafted NCG-Kit-V831M mice showed increased γ-globin expression. Importantly, neither did insA have any off-target effects nor did it affect the maturation of erythroid cells. Furthermore, we found that the insA mutation created a binding site for the transcription activator FOXO3, which was activated by AMPK. Additionally, introducing insA specifically demethylated the − 162 CpG site on HBG promoter by reducing the enrichment of DNA methyltransferase 3 A (DNMT3A). At the same time, it activated histone modifications and RNA polymerase II (Pol II) in both distal and proximal HBG promoter and might inhibit the binding of BCL11A and ZBTB7A on -115 and − 200 sites on the HBG promoter respectively. In addition, combination of insA and the − 115 or -200 editing targets resulted in an amplify effect in reactivating γ-globin genes expression. Conclusions Overall, we presented the preclinical data to support the role of insA on regulating γ-globin expression using CD34+ HSPC cells derived from healthy donors or patients with β-thalassemia, and subsequently engrafted mice. Our study suggests that introducing insA mutation leads to significantly boosted fetal globin levels and uncovers new safe therapeutic target or strategy for β-hemoglobinopathies. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00626-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e602d03d7e5d834d0278abcf5359fc05d4a105ed" target='_blank'>
              A one-base therapeutic insertion in the HBG2 distal promoter reactivates γ-globin expression
              </a>
            </td>
          <td>
            Xiuqin Bao, Yuanyi Gao, Xiaoyi Chen, Zhongju Wang, Xiaoqin Feng, Liren Wang, Jing Du, Yuhua Ye, Feijing Chen, Li Du, Aihua Yin, Xiangmin Xu
          </td>
          <td>2025-03-28</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Introduction Chromosomal structural variations (SVs) play an important role in the formation of human cancers, including leukemias. However, many complex SVs cannot be identified by conventional tools, including karyotyping, fluorescence in situ hybridization, microarrays, and multiplex ligation-dependent probe amplification (MLPA). Methods Optical genome mapping (OGM) and whole genome sequencing (WGS) were employed to analyze five leukemia samples with SVs detected by karyotyping, MLPA, and RNA sequencing (RNA-seq). OGM was performed using the Saphyr chip on a Bionano Saphyr system. Copy number variation and rare variant assembly analyses were performed with Bionano software v3.7. WGS was analyzed by the Manta program for SVs. Results The leukemia samples had an average of 477 insertions, 457 deletions, and 32 inversions, which were significantly greater than those of the normal blood samples (p = 0.016, 0.028, and 0.028, respectively). In Case 1, OGM detected a sequential translocation between chromosomes 5, 8, 12, and 21 and ETV6::RUNX1 and BCAT1::BAALC gene fusions. Case 2 had two pathogenic SVs and a BCR::ABL1 fusion. Case 3 had one pathogenic SV and an IGH::DUSP22 fusion. Case 4 had two pathogenic SVs and a CBFB::MYH11 fusion. Case 5 had an STIL::TAL1 fusion. All breakpoint sequences were defined by WGS. An IGH::DUX4 fusion previously found by RNA-seq in Case 3 was not confirmed because DUX4, which has multiple pseudogenes, was refractory to OGM and WGS analyses. Conclusion OGM is a fundamental tool that complements G-banding analysis in identifying complex SVs in leukemia samples, and WGS effectively closes the gaps in OGM mapping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/492c9544f8deb4fc6edd5a4309b0af1076bee9ac" target='_blank'>
              Optical genome mapping with whole genome sequencing identifies complex chromosomal structural variations in acute leukemia
              </a>
            </td>
          <td>
            Meng-Ju Melody Tsai, H. Kao, Hsiao-Huei Chen, Chih-Hsiang Yu, Y. Chien, W. Hwu, Pui-Yan Kwok, Ni‐Chung Lee, Yun-Li Yang
          </td>
          <td>2025-04-02</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1324a05392de89320b697d282b9ec5e482ee4e56" target='_blank'>
              γ-tubulin mediates DNA double-strand break repair.
              </a>
            </td>
          <td>
            Abhishikt David Solomon, Odjo G. Gouttia, Ling Wang, Songli Zhu, Feifei Wang, Yanqui Li, Mohammadjavad Paydar, T. Bessho, Benjamin H Kwok, A. Peng
          </td>
          <td>2025-03-15</td>
          <td>Journal of cell science</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Background Oxaliplatin-based chemotherapy is the first-line treatment for colorectal cancer (CRC). However, oxaliplatin resistance remains a major challenge contributing to treatment failure and poor prognosis. An increased capacity for DNA damage repair is a key mechanism underlying oxaliplatin resistance. Although XPA binding protein 2 (XAB2) is implicated in various DNA damage repair mechanisms, its specific role in mediating oxaliplatin resistance remains unclear. Methods XAB2 was identified through analysis of public datasets. Western blot analysis and immunohistochemistry were performed to evaluate XAB2 expression, while survival analysis was performed to assess its clinical significance in CRC. Functional experiments were then conducted to assess the impact of XAB2 on proliferation, DNA damage repair, and oxaliplatin resistance in CRC. RNA sequencing (RNA-seq) and Chromatin immunoprecipitation-sequencing (ChIP-seq) were used to identify XAB2 target genes. Co-immunoprecipitation (Co-IP) and mass spectrometry were used to identify the proteins interacting with XAB2. Dual-luciferase reporter assays, ChIP-qPCR, Co-IP, ubiquitination site mass spectrometry, and ubiquitin assays were used to analyse the interactions and potential mechanisms involving XAB2, Annexin A2 (ANXA2), and ubiquitin-specific protease 10 (USP10). Results XAB2 was found to be expressed in CRC and was associated with poor prognosis in patients with CRC. XAB2 promoted CRC cell proliferation and enhanced oxaliplatin resistance by promoting DNA damage repair. Mechanistically, CRC cells treated with oxaliplatin exhibited increased USP10 nuclear expression. USP10 bound to XAB2 and deubiquitinated XAB2 K48-linked polyubiquitination at K593, thereby stabilising XAB2 by reducing its degradation via the ubiquitin-proteasome pathway. XAB2 upregulates ANXA2 expression at the transcriptional level by binding to the ANXA2 promoter, thereby promoting DNA damage repair, mitigating oxaliplatin-induced DNA damage, and enhancing oxaliplatin resistance. Conclusions In summary, this study demonstrates that the USP10/XAB2/ANXA2 axis promotes proliferation, DNA damage repair, and oxaliplatin resistance in CRC. These findings uncover a novel mechanism of oxaliplatin resistance in CRC and suggest potential therapeutic targets for improving the efficacy of oxaliplatin in CRC treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03357-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99a0a723b30808def36cd247f913003a04e424eb" target='_blank'>
              USP10/XAB2/ANXA2 axis promotes DNA damage repair to enhance chemoresistance to oxaliplatin in colorectal cancer
              </a>
            </td>
          <td>
            Xingwu Liu, Shaoming Zhang, Yue An, Boyang Xu, Guanyu Yan, Mingjun Sun
          </td>
          <td>2025-03-11</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c305dc46bcd16adb030553a33a9a8b7556cef2ef" target='_blank'>
              Trypanosoma cruzi Has 32 Chromosomes: A Telomere-to-Telomere Assembly Defines Its Karyotype
              </a>
            </td>
          <td>
            G. Greif, F. Díaz-Viraqué, M. L. Chiribao, Carlos E. Sanz-Rodriguez, C. Robello
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f63c297abc293dc24349467634379d43c83a0e0" target='_blank'>
              Elimusertib, a Novel ATR Inhibitor, Induces Anti-Tumor Effects through Replication Catastrophe in Breast Cancers.
              </a>
            </td>
          <td>
            Mudong Kim, A. Min, Sohyeon Kim, Seongyeong Kim, Yu-Jin Kim, S. Ham, Miso Lee, Eunice Yoojin Lee, Jinyong Kim, Dae-Won Lee, Kyung-Hun Lee, Seock-Ah Im
          </td>
          <td>2025-04-07</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Introduction Pluripotent stem cells (PSC) are known to recurrently acquire genetic aberrations including chromosomal aneuploidy during long-term culture of which the consequences remain largely unknown. A recent study reported that xenografting of aneuploid mouse embryonic stem cells (mESC) gives rise to teratomas containing undifferentiated elements with metastatic capacity. This contrasts with benign teratomas composed of fully differentiated tissues from the three embryonic germ layers obtained by xenografting diploid mESCs. The highlighted study thus proposes that general aneuploidy may drive metastasis of PSC-derived teratomas, thus directly correlating it with malignant potential of PSCs. Main The aneuploid mESC lines used in the aforementioned study showed trisomies of chromosomes 6, 8, 11, 15 or combinations thereof. Interestingly, several of these trisomies, specifically in chromosomes 6, 8, and 11, constitute well-known chromosomal aberrations recurrently acquired in mESCs upon (long-term) in vitro culture. This is suggested to be driven by numerous (proto)oncogenes in these regions. Notably, recurrent chromosomal aberrations are also observed in human PSCs (hPSCs), mostly in chromosome 1, 12, 17, 20 and X, and have similarly been associated with the acquisition of a malignant phenotype in vitro, posing a risk to their potential clinical application. There is synteny between the chromosomes recurrently affected by such aberrations in mouse and human PSCs, namely mouse chromosome 6 and human chromosome 12 as well as mouse and human chromosome 1, suggesting a chromosome-specific phenomenon that may transcend the species barrier. Additionally, in the field of cancer, malignant (but not benign) human germ cell tumors (hGCTs), pathologically similar to the tumors derived from the aforementioned aneuploid mouse PSCs, are characterized by supernumerical copies of chromosome 12, highlighting the additional clinical relevance of these findings and the aneuploid mESC models for GCT research and treatment. Conclusions We suggest that chromosome-specific trisomies, rather than general aneuploidy, might drive teratoma metastasis upon mESC xenografting in vivo. We also observed indications of synteny between the recurrent chromosomal aberrations among human and mouse PSCs, suggesting potentially common intraspecies selection mechanisms. We finally reiterate the similarities observed between the PSC and GCT field related to chromosomal aberrations and malignancy, highlighting the relevance of these experimental models in both research fields.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9961bd4e5533b7648a3a993149a92ce0795a5ee" target='_blank'>
              Chromosome-specific aberrations, rather than general aneuploidy, may drive mouse embryonic stem cell-derived teratoma metastasis
              </a>
            </td>
          <td>
            Joaquin Montilla-Rojo, Sanne Hillenius, T. F. Eleveld, Daniela C F Salvatori, Leendert H J Looijenga
          </td>
          <td>2025-04-15</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Kaposi’s sarcoma-associated herpesvirus (KSHV, HHV-8) is associated with several human malignancies. During latency, the viral genomes reside in the nucleus of infected cells as large non-integrated plasmids, known as episomes. To ensure episome maintenance, the latency protein LANA tethers the viral episomes to the cell chromosomes during cell division. Directional recruitment of protein complexes is critical for the proper function of many nuclear processes. To test for recruitment directionality between LANA and cellular proteins, we directed LANA via catalytically inactive Cas9 (dCas9) to a repeat sequence to obtain easily detectable dots. Then, the recruitment of nuclear proteins to these dots can be evaluated. We termed this assay CRISPR-PITA for Protein Interaction and Telomere Recruitment Assay. Using this protein recruitment assay, we found that LANA recruits its known interactors ORC2 and SIN3A. Interestingly, LANA was unable to recruit MeCP2, but MeCP2 recruited LANA. Both LANA and histone deacetylase 1 (HDAC1) interact with the transcriptional-repression domain (TRD) and the methyl-CpG-binding domain (MBD) of MeCP2. Similar to LANA, HDAC1 was unable to recruit MeCP2. While heterochromatin protein 1 (HP1), which interacts with the N-terminal of MeCP2, can recruit MeCP2. We propose that available interacting domains force this recruitment directionality. We hypothesized that the tandem repeats in the SunTag may force MeCP2 dimerization and mimic the form of DNA-bound MeCP2. Indeed, providing only the tandem epitopes of SunTag allows LANA to recruit MeCP2 in infected cells. Therefore, CRISPR-PITA revealed the rules of unidirectional recruitment and allowed us to break this directionality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30b4f351e09d3750518b9d66bb9ef783bc4bf147" target='_blank'>
              Unidirectional recruitment between MeCP2 and KSHV-encoded LANA revealed by CRISPR/Cas9 recruitment assay
              </a>
            </td>
          <td>
            Ido Lavi, Supriya Bhattacharya, Ankita Awase, Ola Orgil, Nir Avital, Guy Journo, Vyacheslav Gurevich, Meir Shamay
          </td>
          <td>2025-03-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Diplonemids are highly diverse and abundant marine plankton with significant ecological importance. However, little is known about their biology, even in the model diplonemid Paradiplonema papillatum whose genome sequence is available. Examining the subcellular localization of proteins using fluorescence microscopy is a powerful approach to infer their putative function. Here we report a plasmid-based method that enables YFP-tagging of a gene at the endogenous locus. By examining the localization of proteins whose homologs are involved in chromosome segregation in other eukaryotes, we discovered several interesting features in mitotically dividing P. papillatum cells. Cohesin is enriched on condensed interphase chromatin. During mitosis, chromosomes organize into two rings (termed metaphase rings herein) that surround the elongating nucleolus. Homologs of chromosomal passenger complex components (INCENP, two Aurora kinases, and KIN-A), a CLK1 kinase, spindle checkpoint protein Mad1, and microtubule regulator XMAP215 localize in between the two metaphase rings, suggesting that kinetochores may assemble in between them. We also found that a homolog of the meiotic chromosome axis protein SYCP2L1 is enriched in between metaphase rings during mitosis. These features have some resemblance to the bivalent bridge (also known as the modified synaptonemal complex), which is thought to mediate the linkage between homologous chromosomes using axis components in Bombyx mori female meiotic metaphase I. By representing the first molecular characterization of mitotic mechanisms in P. papillatum and raising a number of questions, this study forms the foundation for dissecting mitotic mechanisms in diplonemids.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3aa4cebfac9f1c833d3b7035412536cdcb8f91c" target='_blank'>
              Discovery of unique mitotic mechanisms in Paradiplonema papillatum
              </a>
            </td>
          <td>
            Bungo Akiyoshi, Drahomíra Faktorová, Julius Lukeš
          </td>
          <td>2025-03-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Intratumor transcriptional heterogeneity (ITTH) presents a major challenge in cancer treatment, particularly due to limited understanding of the diverse malignant cell populations and their relationship to therapy resistance. While single-cell sequencing has provided valuable insights into tumor composition, its high cost and technical complexity limits its use for large-scale tumor screening. In contrast, several databases collecting bulk RNA sequencing (RNA-seq) data from multiple samples across various cancer types are available and could be used to profile ITTH. Several deconvolution approaches have been developed to infer cellular composition from such data. However, most of these methods rely on predefined markers or single cell reference datasets, limiting the performance of such methods by the quality of used reference data. Although unsupervised approaches do not face such limitations, existing methods have not been specifically adapted to characterize malignant cell states, and focus instead on general cell types. To address these gaps, we introduce CDState, an unsupervised method for inferring malignant cell subpopulations from bulk RNA-seq data. CDState utilizes a Nonnegative Matrix Factorization model improved with sum-to-one constraints and a cosine similarity-based optimization to deconvolve bulk gene expression into distinct cell state-specific profiles, and estimate the abundance of each state across tumor samples. We validate CDState using bulkified single-cell RNA-seq data from five cancer types, showing that it outperforms existing unsupervised deconvolution methods in both cell state proportions and gene expression estimation. Applying CDState to 33 cancer types from TCGA, we identified recurrent malignant cell programs, with epithelial-mesenchymal transition as the main driver of tumor transcriptional heterogeneity. We further link the identified malignant states to patient clinical features, revealing states associated with worse patient prognosis. We show that ITTH is linked with patient survival and clinical features, as well as is associated with varied responses to therapy. Finally, we identified potential genetic drivers of ITTH and malignant cell states, including TP53, KRAS, and PIK3CA, known oncogenic genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f16cea30bc9f4619225da6166ac6508ace6b994e" target='_blank'>
              CDState: an unsupervised approach to predict malignant cell heterogeneity in tumor bulk RNA-sequencing data
              </a>
            </td>
          <td>
            Agnieszka Kraft, Josephine Yates, Florian Barkmann, Valentina Boeva
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Bacterial immune systems exhibit remarkable diversity and modularity, as a consequence of the continuous selective pressures imposed by phage predation. Despite recent mechanistic advances, the evolutionary origins of many antiphage immune systems remain elusive, especially for those that encode homologs of the Structural Maintenance of Chromosomes (SMC) superfamily, which are essential for chromosome maintenance and DNA repair across domains of life. Here, we elucidate the structural basis and evolutionary emergence of Lamassu, a bacterial immune system family featuring diverse effectors but a core conserved SMC-like sensor. Using cryo-EM, we determined structures of the Vibrio cholerae Lamassu complex in both apo- and dsDNA-bound states, revealing unexpected stoichiometry and topological architectures. We further demonstrate how Lamassu specifically senses dsDNA in vitro and phage replication origins in vivo, thereby triggering the formation of LmuA tetramers that activate the Cap4 nuclease domain. Our findings reveal that Lamassu evolved via exaptation of the bacterial Rad50-Mre11 DNA repair system to form a compact, modular sensor for viral replication, exemplifying how cellular machinery can be co-opted for novel immune functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b6dc8b0acb760637ea2ad121e97471a24629edc" target='_blank'>
              Structural basis for Lamassu-based antiviral immunity and its evolution from DNA repair machinery
              </a>
            </td>
          <td>
            Matthieu Haudiquet, Arpita Chakravarti, Zhiying Zhang, Josephine L. Ramirez, Alba Herrero del Valle, P. D. Olinares, Rachel Lavenir, Massilia Aït Ahmed, M. J. de la Cruz, Brian T. Chait, S. H. Sternberg, A. Bernheim, Dinshaw J Patel
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88c349203243a820b9b9bc0e351f4f372030b26b" target='_blank'>
              DNA2 and MSH2 activity collectively mediate chemically stabilized G4 for efficient telomere replication
              </a>
            </td>
          <td>
            Anthony Fernandez, Tingting Zhou, Steven Esworthy, Changxian Shen, Helen Liu, Jessica D. Hess, Hang Yuan, Nian Liu, Guojun Shi, Mian Zhou, Settapong T. Kosiyatrakul, Vikas Gaur, Joshua Sommers, Winfried Edelman, Guo-Min Li, Robert Michael Brosh, Weihang Chai, Marietta Y. W. T. Lee, Dong Zhang, Carl Schildkraut, Li Zheng, Binghui Shen
          </td>
          <td>2025-04-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background Loss of chromosome Y (LOY) has recently been proposed to be associated with cancer aggressiveness, altered T-cell function, and poor prognosis in bladder carcinomas. Methods Chromosome Y was analyzed using fluorescence in-situ hybridization on a tissue microarray containing 2,071 urothelial carcinomas of the urinary bladder from male patients, including 487 patients who had undergone cystectomy for muscle-invasive disease and for whom follow-up data were available. Data on tumor microenvironment were obtained from a previous study. Results LOY was found in 26.0% of 1,704 analyzable cancers. In non-invasive cancers, LOY frequency was comparable in pTa G2 (22.8%) and pTa G3 (24.1%, p = 0.8036) carcinomas and slightly increased from pTa to pT2 - 4 carcinomas (23.1% for pTa and 27.2% for pT2 - 4) but these differences were not significant (p = 0.0794). In muscle-invasive cancers, LOY frequency slightly increased from pT2 (25.5%) to pT4 cancers (33.0%), but this association was not significant (p = 0.1814). Among pT2 - 4 cancers, LOY was associated with venous invasion (p = 0.0010) but unrelated to pT, pN, and L-status, as well as to overall, recurrence-free, and cancer-specific survival. Muscle-invasive urothelial carcinomas with and without LOY did not show significant differences in the number of CD8 positive lymphocytes, fraction of CD8 positive intraepithelial lymphocytes, number of macrophages and dendritic cells, and fraction of T helper and T regulatory cells. Conclusion The lack of a clear association of LOY with histopathological parameters of cancer aggressiveness, patient prognosis, and parameters describing the tumor microenvironment strongly argues against the driving role of LOY in bladder cancer progression and cancer-associated immune reactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbc1bca8e295978679529dedbaa6c18114de784c" target='_blank'>
              Loss of chromosome Y is unrelated to the composition of the tumor microenvironment and patient prognosis in muscle-invasive urothelial bladder cancers
              </a>
            </td>
          <td>
            H. Plage, V. Ahlburg, S. Hofbauer, K. Furlano, Sarah Weinberger, Annika Fendler, Florian Rossner, S. Schallenberg, S. Elezkurtaj, N. Blessin, M. Lennartz, A. Marx, H. Samtleben, Margit Fisch, M. Rink, M. Słojewski, K. Kaczmarek, T. Ecke, S. Koch, N. Adamini, R. Simon, G. Sauter, H. Zecha, Joachim Weischenfeldt, T. Klatte, S. Minner, D. Horst, T. Schlomm, M. Kluth
          </td>
          <td>2025-04-14</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Aberrant alternative splicing, prevalent in cancer, impacts various cancer hallmarks involving proliferation, angiogenesis, and invasion. Splicing disruption often results from somatic point mutations rewiring functional pathways to support cancer cell survival. We introduce iSoMAs (iSoform expression and somatic Mutation Association), an efficient computational pipeline leveraging principal component analysis technique, to explore how somatic mutations influence transcriptome-wide gene expression at the isoform level. Applying iSoMAs to 33 cancer types comprising 9,738 tumor samples in The Cancer Genome Atlas, we identified 908 somatically mutated genes significantly associated with altered isoform expression across three or more cancer types. Mutations linked to differential isoform expression occurred through both cis- and trans-acting mechanisms, involving well-known oncogenes/suppressor genes, RNA binding protein and splicing factor genes. With wet-lab experiments, we verified direct association between TP53 mutations and differential isoform expression in cell cycle genes. Additional iSoMAs genes have been validated in the literature with independent cohorts and/or methods. Despite the complexity of cancer, iSoMAs attains computational efficiency via dimension reduction strategy and reveals critical associations between regulatory factors and transcriptional landscapes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d1e96c7e45926632d80fe6cd693fccd8a255dd7" target='_blank'>
              iSoMAs: Finding isoform expression and somatic mutation associations in human cancers.
              </a>
            </td>
          <td>
            Hua Tan, Valer Gotea, S. Jaiswal, Nancy E Seidel, David O. Holland, Kevin Fedkenheuer, Abdel Elkahloun, Sara R. Bang-Christensen, L. Elnitski
          </td>
          <td>2025-03-07</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Over the past few years, a revolution has occurred in cytogenetics, driven by the emergence and spread of methods for obtaining high-quality chromosome-level genome assemblies. In fact, this has led to a new tool for studying chromosomes and chromosomal rearrangements, and this tool is thousands of times more powerful than light microscopy. This tool has revolutionized the cytogenetics of many groups of insects for which previously karyotype information, if available at all, was limited to the chromosome number. Even more impressive are the achievements of the genomic approach for studying the general patterns of chromosome organization and evolution in insects. Thus, it has been shown that rapid transformations of chromosomal numbers, which are often found in the order Lepidoptera, are most often carried out in the most parsimonious way, as a result of simple fusions and fissions of chromosomes. It has been established that these fusions and fissions are not random and occur independently in different phylogenetic lineages due to the reuse of the same ancestral chromosomal breakpoints. It has been shown that the tendency for chromosome fissions is correlated with the presence in chromosomes of the so-called interstitial telomeres, i.e. telomere-like structures located not at the ends of chromosomes, but inside them. It has been revealed that, in most insects, telomeric DNA is not just a set of short repeats, but a very long sequence consisting of (TTAGG)n (or other telomeric motifs), regularly and specifically interrupted by retrotransposons, and the telomeric motifs are diverse in terms of their length and nucleotide composition. The number of high-quality chromosome-level genome assemblies available for insects in the GenBank database is growing exponentially and now exceeds a thousand species. Therefore, the exceptional prospects for using genomic data for karyotype analysis are beyond doubt.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f62c2572875ed669953e118ed44bec4832a0b82" target='_blank'>
              Cytogenetics of insects in the era of chromosome-level genome assemblies
              </a>
            </td>
          <td>
            V. A. Lukhtanov, E. Pazhenkova
          </td>
          <td>2025-04-10</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba2b2e41c178b6dbd66cbfc9ba10d448bf6866f2" target='_blank'>
              Spatial multi-omics reveals cell-type-specific nuclear compartments.
              </a>
            </td>
          <td>
            Yodai Takei, Yujing Yang, Jonathan White, Isabel N. Goronzy, Jina Yun, Meera Prasad, Lincoln J Ombelets, Simon Schindler, Prashant Bhat, Mitchell Guttman, L. Cai
          </td>
          <td>2025-04-09</td>
          <td>Nature</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ac23a1b2da77b696744fe7bf1dd3285d3f3c2cb" target='_blank'>
              Targeting the undruggable MYC in cancer: the rationale of using XPO1 inhibitors.
              </a>
            </td>
          <td>
            L. Benetatos, E. Hatzimichael, Eleni Kapsali
          </td>
          <td>2025-04-11</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d56e1d2945b307c87e3e5dbfa94e2c69fe76398" target='_blank'>
              Multi-Omic Profiling Reveals Epigenetic Drivers of Immunotherapy Resistance in Multiple Myeloma
              </a>
            </td>
          <td>
            L. Haertle, Natalia Buenache Cuenda, , Isabel Espejo Díaz, Paula Lazaro del Campo, J. M. Rosa-Rosa, Rafael Alberto, Alonso Fernandez, , J. Sánchez-Pina, A. García-Ortiz, A. Valeri, S. Barrio, Fátima Al-Shahrour, Florian Bassermann, J. Martínez-López, Di Domenico, T. Domenico
          </td>
          <td>2025-04-19</td>
          <td>None</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbddfb38eb3b46fe2a2daba20fe0e51c20e3f4a" target='_blank'>
              βTrCP is involved in the localization of the MRN complex on chromatin to enhance DNA damage repair
              </a>
            </td>
          <td>
            Alejandro Belmonte-Fernández, Joaquín Herrero-Ruíz, Carmen Sáez, M. Japón, M. Mora-Santos, F. Romero
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b66fc40e62ecd1d0cde61c3ba23696dc636af88" target='_blank'>
              Longitudinal Profiling of Circulating Tumor DNA Reveals the Evolutionary Dynamics of Metastatic Prostate Cancer During Serial Therapy.
              </a>
            </td>
          <td>
            Yuehui Zhao, Naveen Ramesh, Ping Xu, Emi Sei, Min Hu, S. Bai, Patricia Troncoso, Ana M Aparicio, Christopher J. Logothetis, Paul G Corn, Nicholas E Navin, A. Zurita
          </td>
          <td>2025-02-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer deaths; nonetheless, how tumor cells adapt to vastly different organ contexts is largely unknown. To investigate this question, we generated a transcriptomic atlas of primary tumor and diverse metastatic samples from a patient with pancreatic ductal adenocarcinoma who underwent rapid autopsy. Unsupervised archetype analysis identified both shared and site-specific gene programs, including lipid metabolism and gastrointestinal programs prevalent in peritoneum and stomach wall lesions, respectively. We developed a probabilistic approach for inferring clonal phylogeny from single-cell and matched whole-exome data. Distantly related genetic clones in the peritoneum express the lipid metabolism program, likely due to signaling by the adipocyte-rich peritoneum environment, and cells in most clones express multiple programs, suggesting that transcriptomic plasticity is a prevalent feature of metastatic cells. These deeply annotated analyses using a patient-centric platform provide a model for investigating metastatic mechanisms and plasticity in advanced cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57b77f53eedac93ceda8af4b898f6071abded0e4" target='_blank'>
              Transcriptomic plasticity is a hallmark of metastatic pancreatic cancer
              </a>
            </td>
          <td>
            Alejandro Jiménez-Sánchez, S. Persad, Akimasa Hayashi, Shigeaki Umeda, Roshan Sharma, Yubin Xie, A. Mehta, Wungki Park, I. Masilionis, T. Chu, Feiyang Zhu, Jungeui Hong, R. Chaligné, Eileen M. O’Reilly, L. Mazutis, T. Nawy, I. Pe’er, Christine A. Iacobuzio-Donahue, D. Pe’er
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77cf64c5137ce4675e3a801e6fb302671875782c" target='_blank'>
              Using Cancer Profiles to Identify Synthetic Lethal Therapeutic Targets and Predictive Biomarkers in Cancer Gene Dependency Data
              </a>
            </td>
          <td>
            Laurence H. Pearl, Frances M. G. Pearl
          </td>
          <td>2025-04-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="To counteract the damaging effects of DNA interstrand crosslinks (ICLs), cells have evolved various specialized ICL repair pathways. However, how ICL repair impacts genetic integrity remains incompletely understood. Here, we determined the mutagenic consequences of psoralen ICL repair in the animal model C. elegans and identify two mutagenic repair mechanisms: (i) translesion synthesis through POLH and REV1/3-mediated bypass, leading to single nucleotide polymorphisms (SNVs), and (ii) end joining via POLQ or HELQ action resulting in deletions. While we found no role for the Fanconi anemia genes FANCD2 and FANCI, disruption of TRAIP, which triggers unloading of the CMG helicase at sites of blocked replication, led to a strikingly altered repair profile, suggesting a role for DNA replication in the etiology of ICL-induced deletions. TRAIP deficiency did not affect SNV formation; instead, we found these SNVs to depend on the functionality of the Fanconi anemia-associated nuclease FAN1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c8e44dc0dd119d75905686eba6c875d7b3f783" target='_blank'>
              FAN1-mediated translesion synthesis and POLQ/HELQ-mediated end joining generate interstrand crosslink-induced mutations
              </a>
            </td>
          <td>
            Jip Verschuren, R. van Schendel, Ivo van Bostelen, Alex E E Verkennis, P. Knipscheer, M. Tijsterman
          </td>
          <td>2025-03-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Dysregulated chromatin remodelers have emerged as critical disease targets. However, owing to the pleiotropic functions of chromatin remodelers, the underlying mechanisms of their effects on cancer have been difficult to elucidate. Here, we investigated the helicase lymphoid-specific (HELLS) oncogenic mechanism by identifying a new direct transcriptional target. Using loss or gain experiments, we identified Mitochondrial elongation factor 1 (MIEF1) as a critical target of the HELLS molecular network in liver cancer. Liver cancer patients with a poor prognosis exhibited upregulated expression of MIEF1, and MIEF1 knockdown led to the loss of tumor capabilities, indicating MIEF1 as an oncogene in liver cancer. Suppressing the HELLS-MIEF1 axis caused mitochondrial hyperfusion, energy deprivation, and further resulting senescence. HELLS knockdown globally increased histone 3 lysine 9 trimethylation (H3K9me3), especially in genomic hotspots with upregulation of SUV39H1 and further augmented DNA methylation. This stabilized genome and hyperfused mitochondria led to reduced levels of reactive oxygen species (ROS) and DNA damage. Finally, tumor cells became famished and calm. We further validated the functions of the HELLS-MIEF1 axis by MIEF1 overexpression and mitochondrial fusion drug. Our study has important implications for medical science by highlighting the crosstalk between epigenetics and metabolism through nuclear chromatin remodeler HELLS and mitochondrial protein MIEF1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4586dfc11f20741829226d56857ec88a7260838a" target='_blank'>
              HELLS controls mitochondrial dynamics and genome stability in liver cancer by collusion with MIEF1
              </a>
            </td>
          <td>
            Sung Kyung Choi, Jihye Park, Sang Yun Ha, Myoung Jun Kim, Seor I Ahn, Jeongah Kim, Woong Sun, Yeong Min Park, Suk Woo Nam, Jeung-Whan Han, Keunsoo Kang, J. You
          </td>
          <td>2025-04-02</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Variants of uncertain significance (VUS) limit the actionability of genetic testing. A prominent example is MUTYH, a base excision repair factor associated with polyposis and colorectal cancer, which has a pathogenic variant carrier rate approaching 1 in 50 individuals in some populations. To systematically interrogate variant function in MUTYH, we coupled deep mutational scanning with a DNA repair reporter containing its lesion substrate, 8OG:A. Our variant-to-function map covers >97% of all possible MUTYH point variants (n=10,941) and achieves 100% accuracy classifying the pathogenicity of known clinical variants (n=247). Leveraging a large clinical registry, we observe significant associations with colorectal polyps and cancer, with more severely impaired missense variants conferring greater risk. We recapitulate known functional differences between pathogenic founder alleles, and highlight sites of complete missense intolerance, including residues that intercalate DNA and coordinate essential Zn2+ or Fe-S clusters. This map provides a resource to resolve the 1,032 existing missense VUS and 90 variants with conflicting interpretations in MUTYH, and demonstrates a scalable strategy to interrogate other clinically relevant DNA repair factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/123ddb79d358a7742980d1fc41247ef4a3c39e8c" target='_blank'>
              Saturation mapping of MUTYH variant effects using DNA repair reporters
              </a>
            </td>
          <td>
            Shelby L. Hemker, Ashley Marsh, Felicia Hernandez, Elena Glick, Grace Clark, Alyssa Bashir, Krystal Jiang, Jacob O. Kitzman
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a86ef1aaa24e6b5fb3e115c100f92a87dac07208" target='_blank'>
              EPIraction - an atlas of candidate enhancer-gene interactions in human tissues and cell lines
              </a>
            </td>
          <td>
            Ramil N. Nurtdinov, R. Guigó
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>124</td>
        </tr>

        <tr id="RNA alternative splicing is a fundamental cellular process implicated in cancer development. Kaposi’s sarcoma-associated herpesvirus (KSHV), the etiological agent of multiple human malignancies, including Kaposi’s sarcoma (KS), remains a significant concern, particularly in AIDS patients. A CRISPR-Cas9 screening of matched primary rat mesenchymal stem cells (MM) and KSHV-transformed MM cells (KMM) identified key splicing factors involved in KSHV-induced cellular transformation. To elucidate the mechanisms by which KSHV-driven splicing reprogramming mediates cellular transformation, we performed transcriptomic sequencing, identifying 131 differential alternative splicing transcripts, with exon skipping as the predominant event. Notably, these transcripts were enriched in vascular permeability, multiple metabolic pathways and ERK1/2 signaling cascades, which play key roles in KSHV-induced oncogenesis. Further analyses of cells infected with KSHV mutants lacking latent genes including vFLIP, vCyclin and viral miRNAs, as well as cells overexpressing LANA, revealed their involvement in alternative splicing regulation. Among the identified splicing factors, FAM50A, a component of the spliceosome complex C, was found to be crucial for KSHV-mediated transformation. FAM50A knockout resulted in distinct splicing profiles in both MM and KMM cells, and significantly inhibited KSHV-driven proliferation, cellular transformation and tumorigenesis. Mechanistically, FAM50A knockout altered SHP2 splicing, promoting an isoform with enhanced enzymatic activity that led to reduced STAT3 Y705 phosphorylation in KMM cells. These findings reveal a novel paradigm in which KSHV hijacks host splicing machinery, specifically FAM50A-mediated SHP2 splicing, to sustain STAT3 activation and drive oncogenic transformation. Importance Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc1dd5dbd2f3302db90bfa70c7f9ae17b8e6ba3f" target='_blank'>
              KSHV Reprograms Host RNA Splicing via FAM50A to Activate STAT3 and Drive Oncogenic Cellular Transformation
              </a>
            </td>
          <td>
            Shenyu Sun, Karla Paniagua, Ling Ding, Xian Wang, Yufei Huang, Mario A Flores, Shou‐Jiang Gao
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Epidermal growth factor receptor variant III (EGFRvIII) mutation is the most prevalent genetic change in glioblastoma. Abnormal DNA damage repair caused by EGFRvIII, which leads to temozolomide (TMZ) resistance, is a major cause of reduced postoperative survival in glioblastoma patients. This study aims to uncover the molecular mechanism of TMZ resistance in EGFRvIII-mutant glioblastoma.


METHODS
We constructed a CRISPR/Cas9 library to identify synthetic lethal genes for EGFRvIII-bearing cells. Abnormal epigenetic regulation of the RAD51-associated protein 1 (RAD51AP1) promoter was assessed via chromatin immunoprecipitation sequencing (ChIP-seq) and chromatin immunoprecipitation polymerase chain reaction (ChIP-PCR) analyses. In vitro and in vivo experiments were carried out to investigate the role of the RAD51AP1 gene in TMZ resistance in EGFRvIII-bearing glioblastoma.


RESULTS
The CRISPR/Cas9 library identified RAD51AP1, a synthetic lethal gene for EGFRvIII-bearing cells exposed to TMZ. ChIP-seq and ChIP-PCR analyses revealed that acetylated histone H3 lysine 27 (H3K27ac) and SRY-box transcription factor 9 (SOX9) together induced RAD51AP1 transcription in EGFRvIII cells. High expression levels of RAD51AP1, promoted formation of the RAD51-UAF1 complex to activate homologous recombination and inhibit TMZ-induced DNA damage.


CONCLUSION
The results of this study suggest that aberrant RAD51AP1 expression is a crucial mechanism by which EGFRvIII-mutant glioblastoma resists TMZ chemotherapy, laying the groundwork for future personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b5d96766fa4eddfdcc647bdcfb29efad8ef9899" target='_blank'>
              Epigenetically elevated RAD51AP1 regulates the RAD51-UAF1 complex contributing to temozolomide resistance in EGFRvIII glioblastoma.
              </a>
            </td>
          <td>
            Yuanhao Chang, Jingchen Yang, Zhuo Li, Xiaoxue Zhu, Tao Jiang, Qixue Wang, Mingchen Yu, Bo Han, Xing Liu
          </td>
          <td>2025-04-10</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The oncogenic mechanisms by which TFE3 fusion proteins drive translocation renal cell carcinoma (tRCC) are poorly characterized. Here, we integrated loss and gain of function experiments with multi-omics analyses in tRCC cell lines and patient tumors. High nuclear accumulation of NONO-TFE3 or PRCC-TFE3 fusion proteins promotes their broad binding across the genome at H3K27ac-marked active chromatin, engaging a core set of M/E-box-containing regulatory elements to activate specific gene expression programs as well as promiscuous binding to active promoters to stimulate mRNA synthesis. Within the core program, TFE3 fusions directly regulate genes involved in ferroptosis resistance and oxidative phosphorylation metabolism (OxPhos). Consequently, human tRCC tumors display high OxPhos scores that persist during their epithelial to mesenchymal transition (EMT). We further show that tRCC tumor aggressiveness is related to their EMT and their associated enrichment in myofibroblast cancer-associated fibroblasts (myCAFs) that are both hallmarks of poor prognostic outcomes. We define tRCC as a novel metabolic subtype of renal cancer and provide unique insights into how broad genomic binding of TFE3 fusion proteins regulates OxPhos and ferroptosis resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37ad1b4cad678e3afe7bfda16fec292de46cc9e9" target='_blank'>
              TFE3 fusions drive oxidative metabolism and ferroptosis resistance in translocation renal cell carcinoma.
              </a>
            </td>
          <td>
            Alexandra Helleux, G. Davidson, A. Lallement, Fatima Al Hourani, Alexandre Haller, I. Michel, A. Fadloun, C. Thibault-Carpentier, Xiaoping Su, Véronique Lindner, T. Tricard, Hervé Lang, N. Tannir, Irwin Davidson, G. Malouf
          </td>
          <td>2025-03-27</td>
          <td>EMBO molecular medicine</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8baa6a9cc6acf1a8fc2f2802c0c1373dc7878226" target='_blank'>
              Steroid-Modulated Transcription Synergistically Forms DNA Double-Strand Breaks With Topoisomerase II Inhibitor.
              </a>
            </td>
          <td>
            Ying Zhao, Tetsuro Hisayoshi, Doudou Zhang, Saaya Suzuki, Takashi Watanabe, Atsuo Kobayashi, Qianqian Guo, Y. Momozawa, Takashi Shimokawa, Shunsuke Kato, Yoshio Miki, Shigeaki Sunada
          </td>
          <td>2025-04-15</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/033ba9cf67722b7d9aad683bbbe23065d0c5012b" target='_blank'>
              Molecular architecture of heterochromatin at the nuclear periphery of primary human cells
              </a>
            </td>
          <td>
            Jan Philipp Kreysing, Sergio Cruz-León, Johannes Betz, Carlotta Penzo, Tomáš Majtner, Markus Schreiber, Beata Turoňová, M. Lusic, Gerhard Hummer, M. Beck
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Garra rufa, commonly known as doctor fish, is a small freshwater cyprinid fish recognized for its high-temperature tolerance. G. rufa is primarily known for its role in ichthyotherapy, in which it helps remove keratinized skin from human hands and feet. However, in recent years, its high-temperature tolerance has attracted attention owing to its potential use as a model for human disease research, including infectious diseases and cancer xenograft models. Despite this interest, the genomic basis underlying high-temperature tolerance remains largely unexplored, primarily because of the limited availability of genomic resources, which hinders its development as an experimental model. In the present study, a high-quality chromosome-level genome assembly of G. rufa was generated using a combination of PacBio HiFi long-read sequencing and Hi-C technology. We generated a chromosome-level genome assembly with 25 chromosomes, 1.38 Gb in total length, and scaffold N50 of 49.3 Mb. Approximately 59% of the genome comprises repetitive elements, with DNA and LTR elements being particularly abundant. In total, 27,352 protein-coding genes were annotated, of which 26,900 genes (98.3%) were functionally annotated. Benchmarking Universal Single-Copy Orthologs (BUSCO) benchmark for genome assembly and gene annotation demonstrated 94.5% and 94.7% of complete BUSCOs, respectively. We identified two heat shock transcription factor (HSF) and 239 heat shock protein (HSP)-related genes. Heat shock elements, which are HSF-binding motifs, were present within 3 kb upstream of 944 genes, with statistically significant enrichment of HSP-related genes in this set. Furthermore, molecular phylogenetic analysis and whole-genome comparisons revealed that G. rufa is evolutionarily closely related to species in the Labeoninae subfamily of the Cyprinidae family. This chromosome-level reference genome provides a valuable resource for future research aimed at elucidating the molecular mechanisms underlying high-temperature tolerance in G. rufa and establishing it as a model organism for human biomedical studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ba084e3b18995a64de9db1867a75cd406732e5f" target='_blank'>
              Chromosome-level genome assembly of the doctor fish (Garra rufa)
              </a>
            </td>
          <td>
            Tetsuo Kon, Koto Kon-Nanjo, Kiki Syaputri Handayani, Liqing Zang, Fahrurrozi, Oleg Simakov, Victor David, Nico Gultom, Yasuhito Shimada
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95cf76b81497b2613567d4d58faa4859bccba0a9" target='_blank'>
              Attenuation of mosaic aneuploidy and erroneous first mitotic division of human embryos
              </a>
            </td>
          <td>
            Cerys E. Currie, A. Byrska, D. Taylor, M. Erent, Daniela Bakalova, Xuhui Sun, Constandina Koki, Nigel J. Burroughs, Richard. A. Anderson, A. Marston, Geraldine M. Hartshorne, A. McAinsh
          </td>
          <td>2025-04-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/255726480b1fba8b96c7a797db393035b828db59" target='_blank'>
              Abstract B022: Decipher actionable tumor suppressor defects for personalized cancer medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35ff625dec261e7f2e3744cf46ad928d6659bf49" target='_blank'>
              Organisation and dynamics of individual DNA segments in topologically complex genomes
              </a>
            </td>
          <td>
            Saminathan Ramakrishnan, Auro Varat Patnaik, Guglielmo Grillo, L. Tubiana, D. Michieletto
          </td>
          <td>2025-04-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Nucleoporin 98 (NUP98) fusion oncogenes are known to promote aggressive pediatric leukemia by disrupting chromatin structure and modulating the expression of homeobox (HOX) genes, yet the precise molecular events are unclear. In this issue of the JCI, K. Hamamoto et al. explore the mechanistic underpinnings of NUP98 fusion–driven pediatric leukemia, with a focus on aberrant activation of the Hoxb-associated long, noncoding RNA (lncRNA) HoxBlinc. The authors provide compelling evidence that HoxBlinc plays a central role in the oncogenic transformation associated with NUP98 fusion protein. The study underscores a CTCF-independent role of HoxBlinc in the regulation of topologically associated domains (TADs) and chromatin accessibility, which has not been fully appreciated in previous research on the NUP98 fusion oncogenes. The discovery of HoxBlinc lncRNA as a downstream regulator of NUP98 fusion oncoproteins offers a potential target for therapeutic intervention in pediatric leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5beef2985107dd5f326a5747e1e6f58922f40b5d" target='_blank'>
              HoxBlinc: a key driver of chromatin dynamics in NUP98 fusion–driven leukemia
              </a>
            </td>
          <td>
            Jian Xu, Wei Du
          </td>
          <td>2025-04-01</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Coupling single-cell transcriptomics with tumor-initiating cell enrichment identified IFN response gene expression not previously reported in bulk RNA-sequencing–derived signatures and proposed IFN/STAT1 signaling as a candidate therapeutic target in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b49032c731c7f43ad3396d95549c8d805e477fd3" target='_blank'>
              Lineage Tracing and Single-Cell RNA Sequencing Reveal a Common Transcriptional State in Breast Cancer Tumor–Initiating Cells Characterized by IFN/STAT1 Activity
              </a>
            </td>
          <td>
            Eric P Souto, Ping Gong, John D. Landua, R. Srinivasan, Abhinaya Ganesan, L. Dobrolecki, S. C. Purdy, Xingxin Pan, Michael Zeosky, Anna Chung, , H. Ford, Michael T. Lewis, Connie Cepko, Dr. Kenneth L. Scott, Dr. Michael Davidson
          </td>
          <td>2025-02-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background Circular RNAs (circRNAs), characterized by their covalently closed-loop structures, constitute a distinct class of non-coding RNAs. They play pivotal regulatory roles within cells and are intricately associated with the progression of malignant tumors. However, their roles and the underlying mechanisms in nasopharyngeal carcinoma (NPC) progression have yet to be fully uncovered and comprehensively understood. Methods Employing RNA sequencing technology, high-abundance circular RNAs in NPC were identified. Expression analysis of circCLASP2 in NPC tissues was conducted using quantitative real-time polymerase chain reaction (qRT-PCR) and in situ hybridization experiments. Through in vitro and in vivo functional assays, the influence of circCLASP2 on the proliferation and metastasis of NPC was investigated. LC–MS/MS technology analyzed the binding partners of circCLASP2, its differentially regulated targets, and the associated proteins of PCMT1. Interactions among circCLASP2, DHX9 protein, and PCMT1 mRNA were elucidated through RNA immunoprecipitation and RNA pull-down techniques. The effects of circCLASP2 and DHX9 on RNA G-quadruplex (rG4) structures and PCMT1 mRNA translation were explored through immunofluorescence (IF), ribosomal gradient separation, and dual-luciferase reporter assays. Immunoprecipitation (IP) revealed the downstream effector of the circCLASP2-DHX9-PCMT1 regulatory axis and Phalloidin staining confirmed its ultimate effect on the cytoskeleton. PDS treatment was applied for interventions in NPC, demonstrating potential therapeutic avenues. Results Our research revealed that circCLASP2, a novel circRNA that has not been reported in tumors, is upregulated in NPC and fosters cell proliferation and metastasis both in vitro and in vivo. Mechanistically, circCLASP2 acts as a molecular scaffold, facilitating the approximation of DHX9 to PCMT1 mRNA. DHX9 unwinds the inhibitory rG4 structure near the translation initiation site on PCMT1 mRNA, increasing PCMT1 expression. PCMT1 binds to and upregulates cytoskeleton-associated proteins, modulating cytoskeleton strength and dynamics and ultimately driving NPC cell proliferation and metastasis. In both in vitro and in vivo experiments, PDS significantly inhibits NPC growth and metastasis, showcasing promising therapeutic potential. Conclusions Our investigation pinpointed a circular RNA, circCLASP2, which is upregulated in NPC and augments cytoskeletal functions via the DHX9-PCMT1 axis, contributing to the malignancy progression of NPC. This pathway holds promise as a potential therapeutic target for NPC. Furthermore, these molecules could also serve as biomarkers for adjunct diagnosis and prognosis assessment in NPC. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02272-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36698e88538b613fd8a284be88d3845947a6a8b4" target='_blank'>
              Circular RNA circCLASP2 promotes nasopharyngeal carcinoma progression through binding to DHX9 to enhance PCMT1 translation
              </a>
            </td>
          <td>
            M. Peng, Shanshan Zhang, Pan Wu, Xiang-sheng Hou, Dan Wang, Junshang Ge, Hongke Qu, Chunmei Fan, Yujuan Zhou, Bo Xiang, Q. Liao, Ming Zhou, Ming Tan, Guiyuan Li, Wei Xiong, Pan Chen, Zhaoyang Zeng, Zhaojian Gong
          </td>
          <td>2025-03-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Background Adenosine A2B receptor (ADORA2B), a G protein-coupled receptor, is implicated in tumor progression and immune regulation in various cancers. However, its specific role in head and neck squamous cell carcinoma (HNSC) remains largely unexplored. This study aims to elucidate the expression profile, prognostic value, immune modulatory role, and therapeutic potential of ADORA2B in HNSC. Methods Comprehensive bioinformatics analyses were performed using TCGA and GEO datasets to evaluate ADORA2B expression, clinical correlations, and prognostic significance in HNSC. Weighted gene co-expression network analysis (WGCNA) and functional enrichment analyses were conducted to explore ADORA2B-associated pathways. Immune infiltration was assessed via ESTIMATE and single-sample gene set enrichment analysis (ssGSEA). Immune checkpoint blockade (ICB) therapy sensitivity and drug sensitivity were analyzed using the IMvigor210 and NCI-60 databases, respectively. In vitro experiments, including siRNA-mediated ADORA2B knockdown, CCK-8 assays, colony formation, and wound healing assays, were performed to validate the oncogenic role of ADORA2B. Results ADORA2B was significantly overexpressed in HNSC tumor tissues compared to adjacent normal tissues, and its expression correlated with advanced clinical stage as well as poor overall survival (OS) and progression-free survival (PFS). Functional enrichment analyses revealed significant downregulation of immune-related pathways in high ADORA2B expression groups. High ADORA2B expression was associated with a more immunosuppressive tumor microenvironment (TME), characterized by lower immune and stromal scores and reduced immune cell infiltration. Immunotherapy response analysis demonstrated that patients with high ADORA2B expression exhibited poorer outcomes following ICB therapy. Drug sensitivity analysis identified several agents, including Ixazomib citrate, Masitinib, and others, as potential therapeutic candidates for high ADORA2B expression patients. In vitro experiments confirmed that ADORA2B knockdown significantly inhibited HNSC cell proliferation, colony formation, and migration, underscoring its critical role in tumor progression. Conclusion ADORA2B is a key oncogenic driver in HNSC, contributing to tumor proliferation, migration, and an immunosuppressive TME. Its high expression is associated with poor prognosis and reduced immunotherapy efficacy. Targeting ADORA2B may enhance therapeutic outcomes and overcome treatment resistance, highlighting its potential as a diagnostic, prognostic, and therapeutic biomarker. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14102-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eb0e3b2654c463b55d8e1da0f65c5af989bc597" target='_blank'>
              ADORA2B promotes proliferation and migration in head and neck squamous cell carcinoma and is associated with immune infiltration
              </a>
            </td>
          <td>
            Pian Li, Ke-ling Pang, Shuang-Jing Chen, Dong Yang, Ai-tao Nai, , Zhe Fang, Qiao Yang, Man-Bo Cai, Jun-Yan He
          </td>
          <td>2025-04-12</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da95ac3ec1b6b3cbe7baf6551a5457106bc739f1" target='_blank'>
              DNA damage response in cancer: New treatment strategies
              </a>
            </td>
          <td>
            Hang-Hao Zhang
          </td>
          <td>2025-03-31</td>
          <td>New Cell</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a03b89d9e1c30e4ab6bb1a8b31d9bee97c5524d" target='_blank'>
              Replisomes restrict SMC-mediated DNA-loop extrusion in vivo
              </a>
            </td>
          <td>
            Qinchan Liao, Hugo B. Brandão, Zhongqing Ren, Xindan Wang
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f6d59a1daee3d02c062b71b39f0a877f021a8f8" target='_blank'>
              Splice-switching ASOs targeting an Alu-derived exon in the AURKA 5’UTR collapse an SRSF1-AURKA-MYC oncogenic circuit in pancreatic cancer
              </a>
            </td>
          <td>
            Alexander J. Kral, Lu Jia, GeunYoung Sim, Ledong Wan, A. Krainer
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="The histone chaperone Holliday Junction Recognition Protein (HJURP) has been associated with multiple types of cancers, but its role in GC is not yet fully understood. Considering its functions in centromere stability and DNA repair, investigating HJURP’s role in GC may offer novel therapeutic perspectives.HJURP expression was examined in a dataset comprising TCGA-STAD samples and an internal group of GC patients, utilizing RNA sequencing and Western blot techniques. Functional experiments were carried out on the AGS and HGC-27 GC cell lines. The expression levels of HJURP, MYC, and Topoisomerase II alpha (TOP2A) were assessed via quantitative real-time PCR and Western blot. Proliferation rates of the cells were determined through EdU, CCK-8, and colony formation assays.Compared to adjacent normal tissues, HJURP expression was notably increased in GC tissues, a finding consistent across both the TCGA-STAD database and our internal patient group. Silencing HJURP markedly reduced GC cell growth and chemoresistance. Mechanistically, HJURP enhanced MYC stability, which in turn promoted TOP2A transcription. Rescue experiments confirmed that overexpression of TOP2A alters proliferation and chemoresistance of GC cells with HJURP knockdown, indicating the dependency of this axis on MYC activity.Our study demonstrates that HJURP is critical for promoting GC proliferation and chemoresistance through the regulation of the MYC/TOP2A transcriptional network. Targeting HJURP might offer a novel therapeutic avenue for GC, necessitating further exploration of its clinical potential. This work underscores the value of investigating histone chaperones as potential targets in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feb8f81bdc3f0ec8326d390d83a122cdb0440ebe" target='_blank'>
              HJURP modulates cell proliferation and chemoresistance via the MYC/TOP2A transcriptional axis in gastric cancer
              </a>
            </td>
          <td>
            Xu Li, Xiwen Li, Yanlin Ren, Ling Wang, Zehao Mao, Shikun Gao, Peng Ma, Junjie Chen
          </td>
          <td>2025-04-11</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="N4-acetylcytidine (ac4C) is deposited on diverse RNAs by N-acetyltransferase 10 (NAT10), a protein with high biological relevance for aging and cancer. We performed a comprehensive survey of ac4C using metabolic labeling, sodium cyanoborohydride chemical treatment coupled to next-generation sequencing (NGS), and ac4C antibody–based cell and molecular biology techniques. Our analysis shows that NAT10-dependent ac4C-acetylation is robust in rRNA and specific tRNAs but low/spurious in mRNA. It also revealed an inflammatory signature and mutagenesis at transcriptionally active sites in NAT10-KO cells. This finding led us to explore the role of NAT10 in R-loops, which were recently linked to APOBEC3B-mediated mutagenesis. Our analysis showed that R-loops are ac4C-acetylated in a NAT10-dependent manner. Furthermore, NAT10 restrains the levels of R-loops at a subset of differentially expressed genes in a catalytic activity–dependent manner. Together with cellular biology data showing ac4C-modified RNA in endosomal structures, we propose that increased levels of ac4C-unmodified RNAs, likely derived from R-loops, in endosomal structures induce inflammatory responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87dd80c89b9eb6c1d31263cc293acb2bcf5cb94c" target='_blank'>
              NAT10 and N4-acetylcytidine restrain R-loop levels and related inflammatory responses
              </a>
            </td>
          <td>
            Turja K Debnath, Nathan S. Abell, Yi-Ru Li, Sravan K. Devanathan, Enrique Navedo, Blerta Xhemalçe
          </td>
          <td>2025-03-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Most molecular classifications of cancer are based on bulk-tissue profiles that measure an average over many distinct cell-types. As such, cancer subtypes inferred from transcriptomic or epigenetic data are strongly influenced by cell-type composition and do not necessarily reflect subtypes defined by cell-type specific cancer-associated alterations, which could lead to suboptimal cancer classifications. Methods To address this problem, we here propose the novel concept of cell-type specific combinatorial clustering (CELTYC), which aims to group cancer samples by the molecular alterations they display in specific cell-types. We illustrate this concept in the context of DNA methylation data of liver and kidney cancer, deriving in each case novel cancer subtypes and assessing their prognostic relevance against current state-of-the-art prognostic models. Results In both liver and kidney cancer, we reveal improved cell-type specific prognostic models, not discoverable using standard methods. In the case of kidney cancer, we show how combinatorial indexing of epithelial and immune-cell clusters define improved prognostic models driven by synergy of high mitotic age and altered cytokine signaling. We validate the improved prognostic models in independent datasets, and identify underlying cytokine-immune-cell signatures driving poor outcome. Conclusions In summary, cell-type specific combinatorial clustering is a valuable strategy to help dissect and improve current prognostic classifications of cancer in terms of the underlying cell-type specific epigenetic and transcriptomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a494e730aa784cad617baf97d8efcc5bc5c5345" target='_blank'>
              Cell-type-specific subtyping of epigenomes improves prognostic stratification of cancer
              </a>
            </td>
          <td>
            Qi Luo, Andrew E. Teschendorff
          </td>
          <td>2024-12-03</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Bleomycin (Bleo) is a glycopeptide with potent antitumor activity that induces DNA double-strand breaks (DSBs) through free radical generation, similar to ionizing radiation (IR). Therefore, Bleo is considered a radiomimetic drug. However, differences in DNA repair mechanisms between IR- and Bleo-induced DNA damage have not been fully elucidated. Therefore, in the present study, we examined a panel of repair-deficient human TK6 cell lines to elucidate the relative contributions of individual repair factors. Results Our comprehensive profiling indicated that both non-homologous end joining (NHEJ) and homologous recombination (HR) contributed to DSB repair induced by X-rays and Bleo. Furthermore, tyrosyl-DNA phosphodiesterase (TDP)-related repair was a significant factor for cellular sensitivity to Bleo treatment. TDP1−/−/TDP2−/− cells exhibited greater sensitivity to Bleo than TDP1−/− or TDP2−/− cells, but not to X-rays. In addition, we determined whether TDP2 is involved in the repair of Bleo-induced DSBs using a neutral comet assay. In TDP1-deficient cells, knockout of TDP2 resulted in a significant delay in the repair kinetics of DSBs induced by Bleo, but not by X-rays. Conclusions The contribution of the TDP-related pathway to DSB repair significantly differed between IR and radiomimetic drugs. The discovery of this novel TDP2-dependent repair of DSBs resulting from radiomimetic drug exposure indicates that TDP1 and TDP2 inhibition in combination with radiomimetic drugs represents a strategy for cancer treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s41021-025-00329-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b896deb7b39c22219dcb649c924ec1a29423e547" target='_blank'>
              Role of TDP2 in the repair of DNA damage induced by the radiomimetic drug Bleomycin
              </a>
            </td>
          <td>
            N. Shimizu, Kazuki Izawa, Mubasshir Washif, Ryosuke Morozumi, Kouji Hirota, Masataka Tsuda
          </td>
          <td>2025-03-28</td>
          <td>Genes and Environment</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0444f2eee8d5a4470978d8e6c693faf1ee9e687e" target='_blank'>
              A Phenotype-Driven Multi-Omic Atlas of Glioblastoma Invasion
              </a>
            </td>
          <td>
            C. Krona, Anders Sundström, Emil Rosén, Soumi Kundu, H. Mangukiya, H. Babačić, Irem Uppman, Madeleine Skeppås, Ida Larsson, Ludmila Elfineh, Xiaofang Cao, Rashmi Ramachandra, Ramy Elgendy, Katrin Rut Halldorsdottir, Zankruti Dave, Milena Doroszko, K. Olausson, Johan Wikström, M. Pernemalm, Sven Nelander
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d25cb925af9bcdbf8c78878896d2879472b19c22" target='_blank'>
              Use of Droplet Digital PCR for Consistent Detection of TMPRSS2:ERG Gene Fusion Transcripts Initiated In Vitro
              </a>
            </td>
          <td>
            Megan H. Check, Sarah E. Ernst, K. Sfanos
          </td>
          <td>2025-03-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Natural plasmids are common in prokaryotes, but few have been documented in eukaryotes. The natural 2µ plasmid present in the yeast Saccharomyces cerevisiae is one of these best-characterized exceptions. This highly stable genetic element has coexisted with its host for millions of years, faithfully segregating at each cell division through a mechanism that remains unclear. Using proximity ligation methods (such as Hi-C, Micro-C) to map the contacts between 2µ plasmid and yeast chromosomes under dozens of different biological conditions, we found that the plasmid is tethered preferentially to regions with low transcriptional activity, often corresponding to long, inactive genes. These contacts do not depend on common chromosome-structuring factors, such as members of the structural maintenance of chromosome complexes (SMC) but depend on a nucleosome-encoded signal associated with RNA Pol II depletion. They appear stable throughout the cell cycle and can be established within minutes. This chromosome hitchhiking strategy may extend beyond the 2µ plasmid/S. cerevisiae pair, as suggested by the binding pattern of the natural eukaryotic plasmid Ddp5 along silent chromosome regions of the amoeba Dictyostelium discoideum.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe60d701689f8f3be285cb82707065b4218a7a99" target='_blank'>
              Parasitic plasmids are anchored to inactive regions of eukaryotic chromosomes through a nucleosome signal
              </a>
            </td>
          <td>
            Fabien Girard, Antoine Even, Agnès Thierry, Myriam Ruault, Léa Meneu, Pauline Larrous, Mickael Garnier, Sandrine Adiba, A. Taddei, Romain Koszul, A. Cournac
          </td>
          <td>2025-02-27</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e847c35fd0feaa6983cfa58c60c7da74934800f2" target='_blank'>
              Moss BRCA2 lacking canonical DNA binding domain promotes efficient homologous recombination and binds to DNA
              </a>
            </td>
          <td>
            Alice Chanteau, Suliane Quilleré, Arthur Crouset, Sreejith Allipra, Ulysse Tuquoi, P. Perroud, Simona Miron, P. Dupaigne, Sophie Zinn-Justin, Fabien Nogué, Rajeev Kumar
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="While DNA:RNA hybrids contribute to multiple genomic transactions, their unscheduled formation is a recognized source of DNA lesions. Here, through a suite of systematic screens, we rather observed that a wide range of yeast mutant situations primarily triggering DNA damage actually leads to hybrid accumulation. Focusing on Okazaki fragment processing, we establish that genic hybrids can actually form as a consequence of replication-born discontinuities such as unprocessed flaps or unligated Okazaki fragments. Strikingly, such “post-lesion” DNA:RNA hybrids neither detectably contribute to genetic instability, nor disturb gene expression, as opposed to “pre-lesion” hybrids formed upon defective mRNA biogenesis, e.g., in THO complex mutants. Post-lesion hybrids similarly arise in distinct genomic instability situations, triggered by pharmacological or genetic manipulation of DNA-dependent processes, both in yeast and human cells. Altogether, our data establish that the accumulation of transcription-born DNA:RNA hybrids can occur as a consequence of various types of natural or pathological DNA lesions, yet do not necessarily aggravate their genotoxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b67011b6e27fed1e3b5196a0fb25f35fae9832be" target='_blank'>
              DNA lesions can frequently precede DNA:RNA hybrid accumulation
              </a>
            </td>
          <td>
            Raphaël M Mangione, Steven Pierce, Myriam Zheng, Robert M Martin, Coralie Goncalves, Arun Kumar, S. Scaglione, Cristiana de Sousa Morgado, Arianna Penzo, Astrid Lancrey, Robert J. D. Reid, Ophélie Lautier, P. Gaillard, Peter C Stirling, Sérgio F. de Almeida, Rodney Rothstein, Benoît Palancade
          </td>
          <td>2025-03-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background The plasma cell malignancy multiple myeloma (MM) remains incurable due to the inevitable development of drug resistance (DR). Epigenetic modifiers are frequently mutated or deregulated in MM patients, contributing to MM progression and relapse. Overexpression of the de novo DNA methyltransferase 3B (DNMT3B) in MM has been reported, correlating with poor prognosis. However, its exact role in MM cell biology and relapse remains elusive. Methods To evaluate the basal expression and prognostic value of DNMT3B mRNA in terms of overall survival the publicly available gene expression profiling datasets GSE2658, GSE9782, GSE4581, E-MTAB-372, E-TABM-1088 and E-TABM-937 were used. Both the DNMT3B selective inhibitor Nanaomycin A and genetic knockdown using a doxycycline inducible shRNA against DNMT3B were used to target DNMT3B. Viability and apoptosis were assessed using respectively a CellTiter-Glo assay and AnnexinV/7AAD stainings. Cell proliferation was measured by BrdU incorporation and cell cycle analysis, while the clonogenic capacity was evaluated by a colony formation assay. Finally, RNA-seq was performed upon genetic knockdown. Results Here, we show that DNMT3B is significantly increased in the relapsed setting and high DNMT3B levels are strongly correlating with disease progression and high-risk disease, irrespective of the treatment. Targeting DNMT3B using either genetic inhibition or the selective inhibitor Nanaomycin A strongly impaired MM cell growth, survival and clonogenicity. Moreover, Nanaomycin A reduced viability of primary MM cells from newly diagnosed and relapsed patients. Mechanistic studies revealed that DNMT3B inhibition mainly affects cell cycle and stemness-related transcriptional programs. Notably, DNMT3B depletion affected the stability of the master cell cycle regulator MYC, thereby reducing c-MYC levels and cell viability both in parental and c-MYC overexpressing cells. Finally, Nanaomycin A (re)sensitized MM cells to bortezomib, melphalan and anti-CD38 monoclonal antibodies (daratumumab, isatuximab). Conclusion Collectively, our findings uncover DNMT3B as a targetable vulnerability in high-risk patients with high DNMT3B/MYC levels. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03382-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f198d3928a28f81e501543dbd3eebc6f8caa4a8" target='_blank'>
              The de novo DNA methyltransferase 3B is a novel epigenetic regulator of MYC in multiple myeloma, representing a promising therapeutic target to counter relapse
              </a>
            </td>
          <td>
            Catharina Muylaert, L. V. Van Hemelrijck, A. Van der Vreken, Robbe Heestermans, Hatice Satilmis, Emma Verheye, E. Alaterre, Catharina Olsen, Nathan De Beule, K. De Veirman, E. Menu, K. Vanderkerken, Jérôme Moreaux, E. De Bruyne
          </td>
          <td>2025-04-17</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Background The incidence of colorectal cancer (CRC) in young adults (age of diagnosis < 50 years old) has been rapidly increasing. Although ∼20% of early-onset (EO) CRC cases are due to germline mutations, the etiology of the majority of EOCRC cases remains poorly understood. Non-genetic factors such as environmental exposure and lifestyle changes are likely to have a direct link to the increased incidence of sporadic EOCRC. We hypothesize that such factors may be observable as differences in the EOCRC epigenome, microbiome and immunome. We sought to address this by comparing differences in DNA methylation from the cohort of colorectal cancer patients in The Cancer Genome Atlas (TCGA). Further, we carefully identified intra-tumoral microbes from TCGA and two other datasets and then related the microbes to EOCRC status and deconvolved immune cell abundances. We found that DNA methylation (DNAm) age acceleration by12 years when compared with average-onset CRC (AOCRC) patients. Differentially methylated sites associated with genes are related to CREB signaling in neurons, G protein coupled receptor signaling, phagosome formation and S100 family signaling. These differences were validated in the gene expression from TCGA and a second, larger real-world dataset from the Oncology Research Information Exchange Network (ORIEN). However, no consistent differences were observed in the intra-tumor microbes between EOCRC and AOCRC. Interestingly, the most abundant microbes interacted with the immune systems differently between the EOCRC and AOCRC tumors, characterized by more, larger, positive correlations in EOCRC. These data suggest epigenetic modulation and accelerated aging may play a key role in the development of EOCRC. SIGNIFICANCE We investigated whether environmentally driven factors contribute to early-onset colorectal cancer (EOCRC). We observed accelerated epigenetic aging in EOCRC and epigenetic changes associated with chronic inflammation. Tumor immune cell abundances correlated more strongly with microbes in EOCRC than average-onset CRC. These data suggest a dysregulation of immune response in EOCRC, driving chronic inflammation and tissue aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8859cb309a43420dfdd0f2da144b37f5d23d1c73" target='_blank'>
              Epigenetic Modulation, Intra-tumoral Microbiome and Immunity in Early Onset Colorectal Cancer
              </a>
            </td>
          <td>
            Ning Jin, R. Hoyd, Ayse Selen Yilmaz, Jian Zhu, YunZhou Liu, M. J. Singh, Dennis Grencewicz, Xiaokui Mo, Matthew Kalady, Daniel Rosenberg, C. Dravillas, Eric A Singer, J. Carpten, Carlos HF Chan, Michelle L Churchman, N. Denko, Frances Di Clemente, Rebecca D Dodd, Islam Eljilany, Naomi Fei, S. Hardikar, A. Ikeguchi, A. Ma, Qin Ma, Martin D McCarter, A. Osman, Gregory Riedlinger, Lary A. Robinson, Bryan P. Schneider, Ahmad A Tarhini, G. Tinoco, Jane C Figueiredo, Yousef Zakharia, C. Ulrich, Aik Choon Tan, Daniel J. Spakowicz
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Simple Summary Cancer arises as an abnormality in genomic control, either due to the activation of oncogenes or loss of functioning tumor-suppressor genes. Oncogenes result in an uncontrolled increase in cell proliferation; tumor-suppressor genes act as a brake, leading the cell cycle to stop and promote cell death when necessary. This review seeks to tease out the complex interactions between these genes, throwing light on key signaling pathways such as MAPK/ERK, PI3K/AKT/mTOR, and Wnt/β-catenin, all of which play a role in the progression of the tumor. We discuss major genetic theories, especially Knudson’s two-hit hypothesis, and how modern technologies like next-generation sequencing have advanced our understanding of cancer genetics. We also underscore the most current therapeutic approaches involving oncogenes and the reactivation of tumor-suppressor gene function, with challenges including drug resistance and heterogeneity of the tumor. In this regard, understanding the balance between oncogenes and tumor-suppressor genes is critical in developing effective and personalized cancer treatment, thereby bettering patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b78e398fe4233b0d93ca9c44f0387aa4d32d4ab" target='_blank'>
              Exploring the Genetic Orchestra of Cancer: The Interplay Between Oncogenes and Tumor-Suppressor Genes
              </a>
            </td>
          <td>
            Sajal Raj Singh, Rakesh Bhaskar, Shampa Ghosh, Bhuvaneshwar Yarlagadda, Krishna Kumar Singh, Prashant Verma, Sonali Sengupta, M. Mladenov, N. Hadzi-Petrushev, R. Stojchevski, J. K. Sinha, Dimiter B Avtanski
          </td>
          <td>2025-03-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54bb9c5a7aaba38fdef52e5c2b089ee886ff2b49" target='_blank'>
              Seasonal brain regeneration and chromosome instability are linked to selection on DNA repair in Sorex araneus
              </a>
            </td>
          <td>
            William R. Thomas, Tanya M. Lama, Diana Moreno Santillán, Marta Farré, Cecilia Baldoni, Linelle Abueg, Jennifer R. Balacco, O. Fedrigo, G. Formenti, Nivesh Jain, J. Mountcastle, Tatiana Tilley, Alan Tracey, David A. Ray, D. Dechmann, Dominik von Elverfeldt, John Nieland, Angelique P. Corthals, Erich D. Jarvis, Liliana M. Dávalos
          </td>
          <td>2025-04-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Eukaryotic somatic cells undergo continuous telomere shortening because of end-replication problems. Approximately 10%~15% of human cancers rely on alternative lengthening of telomeres (ALT) to overcome telomere shortening. ALT cells are characterized by persistent telomere DNA replication stress and rely on recombination-based DNA repair pathways for telomere elongation. The Bloom syndrome (BLM) helicase is a member of the RecQ family, which has been implicated as a key regulator of the ALT mechanism as it is required for either telomere length maintenance or telomere clustering in ALT-associated promyelocytic leukemia bodies (APBs). Here, we summarize recent evidence detailing the role of BLM in the activation and maintenance of ALT. We propose that the role of BLM-dependent recombination and its interacting proteins remains a crucial question for future research in dissecting the molecular mechanisms of ALT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecfa6b604141f203a1f4a0229bf2941af19cc8c" target='_blank'>
              Multiple functions of the ALT favorite helicase, BLM
              </a>
            </td>
          <td>
            Shun Chang, Jiang Tan, Ren Bao, Yanduo Zhang, Jinkai Tong, Tongxin Jia, Jing Liu, Juhua Dan, Shuting Jia
          </td>
          <td>2025-03-01</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff150fb08f88fc51a25fd22d924b89d76aa03ba" target='_blank'>
              Cell Confluency Affects p53 Dynamics in Response to DNA Damage.
              </a>
            </td>
          <td>
            Alina Simerzin, Emily E Ackerman, Kotaro Fujimaki, Rainer H. Kohler, Yoshiko Iwamoto, Mathias S. Heltberg, A. Jambhekar, Ralph Weissleder, G. Lahav
          </td>
          <td>2025-04-09</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Transposable elements are abundant in host genomes but are generally considered to be confined to the cell in which they are expressed, with the notable exception of endogenous retroviruses. Here, we identify a group of LTR retrotransposons that infect the germline from somatic cells within the Drosophila ovary, despite lacking the fusogenic Envelope protein typically required for retroviral entry. Instead, these elements encode a short transmembrane protein, sORF2, with structural features reminiscent of viral cell-cell fusogens. Through genetics, imaging, and electron microscopy, we show that sORF2 localizes to invasive somatic protrusions, enabling the direct transfer of retrotransposon capsids into the oocyte. Remarkably, sORF2-like proteins are widespread among insect retrotransposons and also occur in piscine nackednaviruses and avian picornaviruses. These findings reveal a noncanonical, Envelope-independent transmission mechanism shared by retrotransposons and non-enveloped viruses, offering important insights into host-pathogen evolution and soma-germline interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81230deb71ae111c0bc4f1081eb1e2d5be4c7d01" target='_blank'>
              Direct cell-to-cell transmission of retrotransposons
              </a>
            </td>
          <td>
            Maya Voichek, Andreas Bernhard, Maria Novatchkova, Dominik Handler, Paul Möseneder, Baptiste Rafanel, P. Duchek, Kirsten A. Senti, Julius Brennecke
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tumor dissemination is a key event in tumor progression. During this event, a main role is played by circulating tumor cells (CTCs), immune cells, and their interaction. How the immune system supports the survival and proliferation of CTCs is not fully elucidated. In this study we established an in-vitro co-culture system consisting of immune cells and CTCs from the same patient, which increased the success rate in the establishment of CTC-derived long-term cell cultures. In this system, we characterized the immune cells of successful co-cultures and the signals they exchange with cancer cells, including cytokines and extracellular vesicle (EV) content. Using this protocol, we stabilized four CTC-derived cell lines from patients with metastatic gastroesophageal cancer, which were cultured for over a year and characterized from a genetic and molecular point of view. The four cell lines harbor shared chromosomal aberrations including the amplification at 8q24.21 containing MYC and deletion 9p21.3 containing CDKN2A/B and the IFN type I cluster. The transcriptomic profile of CTC cell lines is distinct from primary tumors, and we detected the activation of E2F, G2M and MYC pathways and the downregulation of interferon response pathway. Each cell line shows a degree of invasiveness in zebrafish in-vivo, and the most invasive ones share the same mutation in RAB14 gene. In addition, the four cell lines secrete cell-line specific EVs containing microRNAs that target YAP, BRG1-AKT1, TCF8-HDAC pathways. Overall, we highlight how the immune system plays a key role in the proliferation of CTCs through EV signaling, and how CTC cell line genomic and transcriptomic alterations make these cells less visible from the immune system and likely responsible for the survival advantage in sites distant from the microenvironment of origin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37fac3e4fbae2c7ba5a9e39e780e9dfb2e2cf33f" target='_blank'>
              Gastroesophageal circulating tumor cell crosstalk with peripheral immune system guides CTC survival and proliferation
              </a>
            </td>
          <td>
            T. Rossi, Martina Valgiusti, Maurizio Puccetti, Giacomo Miserocchi, Michele Zanoni, D. Angeli, Chiara Arienti, Ilaria Pace, C. Bassi, Ivan Vannini, M. Melloni, E. Bandini, Milena Urbini, Massimo Negrini, Massimiliano Bonafé, Manuela Ferracin, Giulia Gallerani
          </td>
          <td>2025-03-29</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Glioblastoma (GBM) is the most prevalent malignant brain tumor with poor prognosis. Although chromatin intratumoral heterogeneity is a characteristic feature of GBM, most current studies are conducted at a single tumor site. To investigate the GBM-specific 3D genome organization and its heterogeneity, we conducted Hi-C experiments in 21 GBM samples from nine patients, along with three normal brain samples. We identified genome subcompartmentalization and chromatin interactions specific to GBM, as well as extensive intertumoral and intratumoral heterogeneity at these levels. We identified copy number variants (CNVs) and structural variations (SVs) and demonstrated how they disrupted 3D genome structures. SVs could not only induce enhancer hijacking but also cause the loss of enhancers to the same gene, both of which contributed to gene dysregulation. Our findings provide insights into the GBM-specific 3D genome organization and the intratumoral heterogeneity of this organization and open avenues for understanding this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c40cfec55257420ce8df49630fbce7033db9c8" target='_blank'>
              Spatial 3D genome organization reveals intratumor heterogeneity in primary glioblastoma samples
              </a>
            </td>
          <td>
            Qixuan Wang, Juan Wang, Radhika Mathur, Mark Youngblood, Q. Jin, Ye Hou, L. Stasiak, Yu Luan, Hengqiang Zhao, Stephanie Hilz, Chibo Hong, Susan M Chang, Janine M Lupo, Joanna J Phillips, Joseph F Costello, Feng Yue
          </td>
          <td>2025-03-12</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Extracellular vesicles (EVs) produced in the tumor microenvironment in response to chemotherapy promote chemotherapy-resistant phenotypes. However, the role of EVs proteins induced by gastric cancer (GC) cell chemotherapy in regulating chemotherapy resistance remains unclear. Methods Immunohistochemistry was used to verify the relationship between brain-specific angiogenesis inhibitor 1-associated protein-2-like protein 2 (BAIAP2L2) expression and chemotherapy resistance in advanced GC. The relationship between BAIAP2L2 and chemotherapy resistance was verified using a subcutaneous tumor model in nude mice. Transmission electron microscopy, nanoparticle tracking analysis, and western blotting were performed to detect purified EVs. Tandem mass tag (TMT) analysis was used to detect differential labels. The interaction between YTH domain-containing family protein1 (YTHDF1) and BAIAP2L2 in GC cells was confirmed by RIP-qPCR analysis using a YTHDF1-specific antibody. Results We found that BAIAP2L2 was associated with chemotherapy resistance to GC in clinical samples and was increased in chemotherapy-resistant GC cells. Mechanistically, BAIAP2L2 promotes the transfer of chemotherapy resistance from resistant GC cells to sensitive cells through EVs proteins, such as ANXA4. Furthermore, ANXA4 promoted platinum-based chemical resistance in GC by mediating autophagy. Interestingly, YTHDF1 facilitates the translation of BAIAP2L2 and ANXA4 through m6A modifications. Conclusions Our findings reveal the key role of BAIAP2L2 as a potential prognostic marker and therapeutic target for chemotherapy resistance in GC. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06340-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da28068ee3d99a5bdc693522fa3844cc3c9f7028" target='_blank'>
              N6-methyladenosine RNA modified BAIAP2L2 facilitates extracellular vesicles-mediated chemoresistance transmission in gastric cancer
              </a>
            </td>
          <td>
            Yuhan Liao, Xinhua Chen, Hao Xu, Yunfei Zhi, Xinghua Zhuo, Jiang Yu, Liang Zhao
          </td>
          <td>2025-03-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Somatic alterations, like mutations and copy number changes, driver oncogenesis and cancer progression. Their inhibition has been exploited in the clinic, with several targeted therapies approved for patients with specific mutations or amplifications. However, the response rate of these treatments remains low. The causes are several, ranging from clonal heterogeneity to off target binding. For this reason, CRISPR assays have been developed to study the exact effect of a gene’s deletion. Still, the results from them are puzzling with the same alterations responding different to knockout even in the same cellular context. For this reason, we have developed SAEG, a novel deep learning architecture for somatic alterations in cancer. Our architecture is able to model mutations and copy number alterations and protein-protein interactions to predict if a cell will be susceptible to a gene knockout. SAEG outperforms other models and we show that it learns patterns that can be traced back to the biochemical and biological properties of genes and amino acids. Code Availability https://github.com/Luisiglm/SAEG Contact luis.iglesiarmatinez@ucd.ie">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caba93fa83f15f4694f814ad4d5d269b25293aad" target='_blank'>
              SAEG: A Novel Deep Learning Architecture for Somatic Alterations
              </a>
            </td>
          <td>
            Luis F. Iglesias-Martinez, Dan Yang Wester, Walter Kolch
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0992e4a25dabd2c0752a1440000c36e4af2e792" target='_blank'>
              A Chk1-Sp1-CD59 axis of the DNA damage response impedes rituximab-mediated complement-dependent cytotoxicity
              </a>
            </td>
          <td>
            Allison S. Y. Chan, P. Jaynes, A. Anbuselvan, C. Z. Ong, M. Hoppe, Wai Khang Yong, Vartika Khanchandani, Jie Min Lee, Nurulhuda Mustafa, M. I. Azaman, Phuong M. Hoang, Guo Hong, W. Chng, M. Cragg, Dennis Kappei, C. Tripodo, Anand D. Jeyasekharan
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37c650c9b7245da58253a96046c65fd7d628305a" target='_blank'>
              Understanding DNA and PARP in Cancer: Tackling Inhibitor Resistance
              </a>
            </td>
          <td>
            Hardha Balachandran, Gowramma Byran, V. Karri, Senthil Kumar Murugesan, K. Rajagopal
          </td>
          <td>2025-03-13</td>
          <td>Drug Metabolism and Bioanalysis Letters</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Simple Summary Our research aims to test the suitability of ddPCR for the detection of tumor-related genomic imbalances (CNVs) in ovarian cancer (OC) in the peripheral blood system. In principle, in an artificial CNV simulation, we can show that CNVs can be detected even in larger dilutions in diploid DNA using ddPCR. Furthermore, we identified ovarian cancer CNVs by using aCGH and used this information to produce suitable ddPCR tests. These ddPCR tests are based on the principle of two differently colored comparative DNA loci that mark both very common genome imbalances in OC and regions that hardly indicate genome imbalances in OC. We were able to generate six ddPCR tests that indicate OC by identifying cell-free tumor DNA with a sensitivity of 0.8 and a specificity of 0.8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a29b17cdaeb68fcde9e2c47b4a309aeadaf32f5" target='_blank'>
              Detection of Genomic Copy Number Variations in Ovarian Cancer in the Peripheral Blood System
              </a>
            </td>
          <td>
            Lena Wahl, Ulyana Hliabtsova, Xueqian Qian, Anastasia Klopf, N. Hedemann, Inken Flörkemeier, C. Rogmans, Michael Kalab, Astrid Dempfle, Nicolai Maass, D. Bauerschlag, M. V. van Mackelenbergh, J. P. Weimer, Huy Duc Le
          </td>
          <td>2025-02-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13d3f3fcfa9e530a07a432c07ef3c3f60b8209c5" target='_blank'>
              The conserved SEN1 DNA:RNA helicase has multiple functions during yeast meiosis
              </a>
            </td>
          <td>
            Robert Gaglione, Jonathan Caradonna, A. MacQueen, Ed Luk, N. M. Hollingsworth
          </td>
          <td>2025-04-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Homologous recombination (HR) is a major pathway to repair DNA double-strand breaks. Hereditary breast and ovarian cancer syndrome (HBOC) is caused by germline pathogenic variants of HR-related genes, such as BRCA1 and BRCA2 (BRCA1/2). Cancer cells with HR deficiency are sensitive to poly(ADP-ribose) polymerase (PARP) inhibitors. Therefore, accurate evaluation of HR activity is helpful to diagnose HBOC and predict the effects of PARP inhibitors. The direct-repeat GFP (DR-GFP) assay has been utilized to evaluate cellular HR activity. However, evaluation by the DR-GFP assay tends to be qualitative and requires the establishment of stable cell lines. Therefore, we developed an assay to quantitatively measure HR activity called Assay for Site-Specific HR Activity (ASHRA), which can be performed by transiently transfecting two plasmids. In ASHRA, we use Cas9 endonuclease to create DNA double-strand breaks at specific sites in the genome, enabling the targeting of any endogenous loci. Quantification of HR products by real-time PCR using genomic DNA allows HR activity evaluated at the DNA level. Thus, ASHRA is an easy and quantitative method to evaluate HR activity at any genomic locus in various samples. Here, we present the protocols for adherent cells, suspension cells, and tumor tissues. Key features • This assay quantitatively evaluates homologous recombination (HR) activity. • This assay can measure HR activity in adherent cells, suspension cells, and tumor tissues. • This real-time PCR-based assay does not require a flow cytometer or next-generation sequencer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b6041469126e683215dc13399e4746d11c09375" target='_blank'>
              Assay for Site-Specific Homologous Recombination Activity in Adherent Cells, Suspension Cells, and Tumor Tissues
              </a>
            </td>
          <td>
            Yuki Yoshino, Shin Kikuta, N. Chiba
          </td>
          <td>2025-04-05</td>
          <td>Bio-protocol</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The Gram-negative anaerobe Fusobacterium nucleatum is an oral oncobacterium that promotes colorectal cancer (CRC) development with the amyloid-forming cell surface adhesin FadA integral to CRC tumorigenesis. We describe here molecular genetic studies uncovering a novel mode of metabolic regulation of FadA-mediated tumor formation by a highly conserved respiratory enzyme known as the Rnf complex. First, we show that genetic disruption of Rnf, via rnfC deletion, significantly reduces the level of fadA transcript, accompanied by a near-complete abolishment of the precursor form of FadA (pFadA), reduced assembly of FadA at the mature cell pole, and severe defects in the osmotic stress-induced formation of FadA amyloids. We show further that the Rnf complex regulates three response regulators (CarR, ArlR, and S1), which modulate the expression of pFadA, without affecting fadA transcript. Consistent with our hypothesis that these response regulators control factors that process FadA, deletion of rnfC, carR, arlR, or s1 each impairs expression of the signal peptidase gene lepB, and FadA production is nearly abolished by CRISPR-induced depletion of lepB. Importantly, while rnfC deletion does not affect the ability of the mutant cells to adhere to CRC cells, rnfC deficiency significantly diminishes the fusobacterial invasion of CRC cells and formation of spheroid tumors in vitro. Evidently, the Rnf complex modulates the expression of the FadA adhesin and tumorigenesis through a gene regulatory network consisting of multiple response regulators, each controlling a signal peptidase that is critical for the post-translational processing of FadA and surface assembly of FadA amyloids. IMPORTANCE The Rhodobacter nitrogen-fixation (Rnf) complex of Fusobacterium nucleatum plays an important role in the pathophysiology of this oral pathobiont, since genetic disruption of this conserved respiratory enzyme negatively impacts a wide range of metabolic pathways, as well as bacterial virulence in mice. Nonetheless, how Rnf deficiency weakens the virulence potential of F. nucleatum is not well understood. Here, we show that genetic disruption of the Rnf complex reduces surface assembly of adhesin FadA and FadA-mediated amyloid formation, via regulation of signal peptidase LepB by multiple response regulators. As FadA is critical in the carcinogenesis of colorectal cancer (CRC), the ability to invade CRC cells and promote spheroid tumor growth is strongly diminished in an Rnf-deficient mutant. Thus, this work uncovers a molecular linkage between the Rnf complex and LepB-regulated processing of FadA – likely via metabolic signaling – that maintains the virulence potential of this oncobacterium in various cellular niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbe80828055ab4b08cc5ea46b200a6e60a644da8" target='_blank'>
              Inactivation of the Fusobacterium nucleatum Rnf complex reduces FadA-mediated amyloid formation and tumor development
              </a>
            </td>
          <td>
            Timmie A. Britton, Ju Huck Lee, Chungyu Chang, A. Bhat, Yi-Wei Chen, Rusul Mohammed Ali, Chenggang Wu, A. Das, Hung Ton-That
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba5821f61b692c344ab6ba3f9c1abecff3caf90b" target='_blank'>
              Multisite long-read sequencing reveals the early contribution of somatic structural variations to HBV-related hepatocellular carcinoma tumorigenesis.
              </a>
            </td>
          <td>
            Tianfu Zeng, Haotian Liao, Lin Xia, Siyao You, Yan Huang, Jiaxun Zhang, Yahui Liu, Xuyan Liu, Dan Xie
          </td>
          <td>2025-03-04</td>
          <td>Genome research</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d4b38200a1231a3f3c1164315cc7d10135fa36f" target='_blank'>
              Redirecting resistance evolution in BRAFV600 melanoma by inhibition of the peroxiredoxin-thioredoxin system
              </a>
            </td>
          <td>
            Stefanie Egetemaier, H. Chauvistré, R. Váraljai, Yichao Hua, Smiths S. Lueong, Samira Makhzami, Nalini Srinivas, Jan Forster, V. Ullrich, Simone Stupia, Valéria Schroeder, Sarah Scharfenberg, A. Hoewner, B. Shannan, J. Siveke, M. Baietti, Eleonora Leucci, J. Marine, A. Paschen, Bjoern Scheffler, D. R. Engel, Lisa M. Becker, Felix Nensa, Johannes Koester, Barbara Grünwald, S. Poepsel, Sabrina Ninck, F. Kaschani, Dirk Schadendorf, Juergen C. Becker, Alpaslan Tasdogan, Florian Rambow, Alexander Roesch
          </td>
          <td>2025-03-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43a623593b9a20fd048f7dbc4cfe070fa57a8f07" target='_blank'>
              PRMT activity promotes global 3’ UTR shortening in proliferating cells
              </a>
            </td>
          <td>
            Llywelyn Griffith, Charlotte Capitanchik, Shaun Moore, Anca Farcas, Mags Gwynne, Martina Pedna, D. M. Jones, A. Chakrabarti, Dimitris Lagos, Jelena Urosevic, James T. Lynch, J. Ule
          </td>
          <td>2025-03-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Topoisomerase III-beta (Top3b) reduces nucleic acid torsional stress and intertwining generated during RNA and DNA metabolism while protecting the genome from pathological R-loops, which otherwise result in DNA breakage and genome instability. By studying Top3b knockout mice (Top3b-KO), we find that the loss of Top3b accelerates the development of spontaneous lymphoid tumors arising in spleens and lymph nodes, the organs with prominent Top3b expression. Aging Top3b-KO mice also display splenomegaly and systemic immune alterations including neutrophilia and lymphopenia suggestive of chronic inflammation. At the molecular level, Top3b deficiency causes genome-wide R-loop accumulation in splenocytes as measured by CUT&Tag sequencing. Increased R-loops is associated with genomic DNA breaks and activation of immune signaling pathways including the IL-6 signaling, interleukin-7 signaling and cGAS-STING. Moreover, knocking-out Top3b promotes the rapid development of syngeneic EL4 T-cell lymphomas. In conclusion, our work implies that, in addition to its role in preserving the nervous system, Top3b protects from tumorigenesis and immune dysregulations. Significance Topoisomerase III-beta (TOP3B) is the only dual RNA-DNA topoisomerase. Its inactivation has previously been shown to be critical for neurodevelopmental disorders in humans and R-loop suppression in cell line models. Here, we demonstrate that loss of Top3b accelerates spontaneous lymphomagenesis and promotes syngeneic lymphoma growth in association with innate immune defects and R-loop accumulation in Top3b-KO mice. These findings provide evidence for a previously unappreciated role of Top3b as a tumor suppressor gene and immune regulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/670788860a0acb20f381f0101f5e0bb53a8bd0df" target='_blank'>
              Topoisomerase IIIβ protects from tumorigenesis and immune dysregulation
              </a>
            </td>
          <td>
            Rasel Al Mahmud, S. Baechler, A. Dhall, Sourav Saha, Hongliang Zhang, Shuling Zhang, Min-Jung Lee, Nahoko Sato, Shraddha Rastogi, Suresh Kumar, Muhammad S. Alam, L. Saha, Beverly A. Mock, Valentina M Factor, Y. Pommier
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96bb67ca8a1fde17024b6bb4d2cb9ffe3c6efe6e" target='_blank'>
              Activation of DCTN1-RET Fusion Through Coiled-coil Domain as a Potential Target for RET Inhibitors.
              </a>
            </td>
          <td>
            Kohei Hayashi, K. Ishida, Masanori Kato, Shuichi Ohkubo, Yoshihiro Uto
          </td>
          <td>2025-03-28</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Metastasis is a leading cause of colorectal cancer (CRC)-related mortality, yet its molecular mechanisms remain poorly understood. Long noncoding RNAs (lncRNAs) have emerged as critical regulators of CRC metastasis, but their specific roles are not fully elucidated. This study identifies and characterizes a novel lncRNA LINC02167 as a critical regulator of CRC metastasis. Methods LINC02167 expression was analyzed in CRC tissues via real-time quantitative polymerase chain reaction and fluorescence in situ hybridization. Functional assays evaluated its role in CRC cell migration, invasion, and metastasis in vitro and in vivo. Mechanistic exploration involves a combination of techniques, including RNA sequencing, mass spectrometry, RNA pull-down, RNA immunoprecipitation, chromatin immunoprecipitation, luciferase reporter assays, RNA stability assays, and bioinformatics analysis, to uncover the molecular interactions and pathways regulated by LINC02167. Results LINC02167 is markedly upregulated in CRC tissues and strongly correlates with advanced clinical features and poor prognosis. Functional analyses reveal that LINC02167 enhances CRC cell migration and invasion in vitro and promotes metastasis in vivo. Mechanistically, LINC02167 serves as a molecular scaffold, forming a complex with YBX1 and ILF3 to facilitate YBX1 binding to NSUN2-mediated m5C modification sites on KSR1 mRNA, thereby stabilizing KSR1 mRNA and activating the ERK/MAPK signaling pathway to drive CRC metastasis. Additionally, MYC-driven transcriptional activation leads to the upregulation of LINC02167 in CRC. Conclusions This study uncovers a novel mechanism through which LINC02167 promotes the ERK/MAPK pathway and CRC metastasis via m5C modification, underscoring its potential as a promising therapeutic target for metastatic CRC treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03368-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c87a1229126a6672350e24e97d2e6f03e765a6b" target='_blank'>
              LINC02167 stabilizes KSR1 mRNA in an m5C-dependent manner to regulate the ERK/MAPK signaling pathway and promotes colorectal cancer metastasis
              </a>
            </td>
          <td>
            Junwen Qi, Tao Jiang, Bowen Liu, Qihang Hu, Junnan Chen, Ning Ma, Yixin Xu, Hu Song, Jun Song
          </td>
          <td>2025-04-15</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214e281d859cf2be94a12cc54dca351d068768d3" target='_blank'>
              Genome-wide CRISPR activation screen identifies ARL11 as a sensitivity determinant of PARP inhibitor therapy.
              </a>
            </td>
          <td>
            Teng Zhang, Yuan Zhang, Xuxiang Wang, Haitian Hu, Christopher G. Lin, Yaru Xu, Hanqiu Zheng
          </td>
          <td>2025-03-23</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Anti-PD-1/PD-L1 immune checkpoint blockade (ICB) therapy has revolutionized clinical cancer treatment, while abnormal PD-L1 or HLA-I expression in patients can significantly impact the therapeutic efficacy. Somatic mutations in cancer cells that modulate these critical regulators are closely associated with tumor progression and ICB response. However, a systematic interpretation of cancer immune-related mutations is still lacking. Here, we harnessed the ABEmax system to establish a large-scale sgRNA library encompassing approximately 820,000 sgRNAs that target all feasible serine/threonine/tyrosine residues across the human genome, which systematically unveiled thousands of novel mutations that decrease or augment PD-L1 or HLA-I expression. Beyond residues associated with phosphorylation events, our screens also identified functional mutations that affect mRNA or protein stability, DNA binding capacity, protein-protein interactions, and enzymatic catalytic activity, leading to either gene inactivation or activation. Notably, we uncovered certain mutations that concurrently modulate PD-L1 and HLA-I expression, represented by the clinically relevant mutation SETD2_Y1666. We demonstrated that this mutation induces consistent phenotypic effects across multiple cancer cell lines and enhances the efficacy of immunotherapy in different tumor models. Our findings provide an unprecedented resource of functional residues that regulate cancer immunosurveillance, offering valuable guidance for clinical diagnosis, ICB therapy, and the development of innovative drugs for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0151a36f604dedfea97d97f1dfa3c6ae2a57eb97" target='_blank'>
              Massively parallel interrogation of human functional variants modulating cancer immunosurveillance
              </a>
            </td>
          <td>
            Ying Liu, Yongshuo Liu, Xuran Niu, Ang Chen, Yizhou Li, Ying Yu, Binrui Mo, Zhiheng Liu, Tao Xu, Jie Cheng, Zeguang Wu, Wensheng Wei
          </td>
          <td>2025-03-19</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d1e49024118358c3779c136165092b1f37168ca" target='_blank'>
              Single-cell parallel analysis of DNA damage and transcriptome reveals selective genome vulnerability.
              </a>
            </td>
          <td>
            Dongsheng Bai, Zhenkun Cao, Nivedita Attada, Jinghui Song, Chenxu Zhu
          </td>
          <td>2025-03-24</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Long interspersed element 1 (LINE-1) retrotransposons are repetitive sequences that can move within the genome by an autonomous mechanism. To limit their mutagenic potential, benign cells restrict LINE-1 expression through molecular mechanisms such as DNA methylation and histone modification, but these mechanisms are usually impaired in cancer. Clear cell ovarian carcinoma (CCOC) represents 5–10% of ovarian cancers and is thought to arise from endometriosis. Women with advanced CCOC face poor prognoses, highlighting the importance of understanding early disease pathogenesis. In our study, 33 of 40 cases (over 82%) of CCOC tumors express ORF1p, a LINE-1-encoded protein. We found that LINE-1 de-repression is an early event in CCOC, as ORF1p is enhanced during the transition from typical to atypical endometriosis and persists in invasive cancer. Finally, using single-molecule array (Simoa) assays, we detected ORF1p in patient blood, suggesting it as a potential minimally invasive biomarker for this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e1b48d57468e7ee31e0d2b20955ec88056a5d42" target='_blank'>
              LINE-1 ORF1p expression occurs in clear cell ovarian carcinoma precursors and is a candidate blood biomarker
              </a>
            </td>
          <td>
            Pamela R. de Santiago, Sho Sato, Stephanie J. Zhang, Meaghan Dougher, Kyle M. Devins, A. Bilecz, Sagar Rayamajhi, Gabriel Mingo, Hannah S. Rendulich, Yi Feng, Connie Wu, Martin S Taylor, Yelena Zhuravlev, Euihye Jung, Dalia K Omran, Tian-Li Wang, I. Shih, Lauren E. Schwartz, Sarah Kim, Mark Aloysuis Morgan, J. Tanyi, Kathleen H. Burns, Ernst Lengyel, C. Parra‐Herran, Andrew K Godwin, David R. Walt, Ronny Drapkin
          </td>
          <td>2025-03-06</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73299a1254c80e9af5d25fe381a681e9866a35bc" target='_blank'>
              Studying the cellular efficacy and tolerability of using CRISPR-gRNA ribonucleoprotein (RNP) complex for in-vitro knockdown of TRIB2 in acute myeloid leukaemia (AML) cells as preliminary clinical evaluations.
              </a>
            </td>
          <td>
            Josephine You Pheng Tan, Ramesh Thevendran, Yek Song Quek, Solayappan Maheswaran
          </td>
          <td>2025-04-09</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: KMT2A rearrangements occur in ~10% of acute myeloid leukemia (AML) cases and are critical for classification, risk stratification, and use of targeted therapy. However, insertions involving the KMT2A gene can evade detection using chromosomal analysis and/or fluorescence in situ hybridization (FISH). Methods: We present a case of a 22-year-old woman with acute monoblastic leukemia harboring a cryptic KMT2A::AFDN fusion identified by RNA sequencing. Initial FISH showed a 3′ KMT2A deletion, while conventional karyotyping and the automated bioinformatic pipeline for optical genome mapping (OGM) did not identify the canonical translocation. Results: To resolve these discrepancies, metaphase KMT2A FISH (break-apart fusion probe) was performed to assess whether KMT2A was translocated to another chromosome. However, the results did not support this possibility. As the fusion signal remained on the normal chromosome 11, with the 5′ KMT2A signal localized to the derivative chromosome 11. A subsequent manual review of the OGM data revealed a cryptic ~300 kb insertion of AFDN into the 3′ region of KMT2A, reconciling the discrepancies between chromosomal analysis, FISH, and RNA fusion results. Conclusions: This case highlights the importance of integrating multiple testing modalities with expert review when there is a discrepancy. Our findings emphasize the need for a comprehensive approach to genomic assessment to enhance diagnostic accuracy and guide therapeutic decision-making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f9b71c31c191c38bbe63bd8eb6331d6c11375f1" target='_blank'>
              Cryptic KMT2A::AFDN Fusion Due to AFDN Insertion into KMT2A in a Patient with Acute Monoblastic Leukemia
              </a>
            </td>
          <td>
            Qing Wei, Gokce A. Toruner, B. Thakral, Keyur P. Patel, N. Pemmaraju, Sa A. Wang, R. Kanagal-Shamanna, Guilin Tang, G. Issa, S. Loghavi, L. J. Medeiros, Courtney D. DiNardo
          </td>
          <td>2025-03-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="To maintain cell fitness, deleterious genetic alterations are buffered by compensatory changes in additional genes. In cancer, buffering processes could be targeted by synthetic lethality. However, despite the large-scale identification of synthetic lethal effects in preclinical models, evidence that these operate clinically is limited. This impedes the application of synthetic lethal approaches. By integrating molecular profiling data from >9,000 cancers with synthetic lethal screens, we show that transcriptomic buffering of tumor suppressor gene (TSG) loss by hyperexpression of synthetic lethal partners is a common phenomenon, extending to multiple TSGs and histotypes. Transcriptomic buffering is also notable in cancers that phenocopy TSG loss, such as BRCAness cancers, where expression of BRCA1/2 synthetic lethal genes correlates with clinical outcome. Synthetic lethal genes that exhibit transcriptomic buffering also represent more robust synthetic lethal effects. These observations have implications for understanding how tumor cells tolerate TSG loss, in part explain transcriptomic architectures in cancer and provide insight into target selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a52e19b56a0e26f2474ec866322bdccd6f6854" target='_blank'>
              The transcriptomic architecture of common cancers reflects synthetic lethal interactions
              </a>
            </td>
          <td>
            S. Haider, R. Brough, Santiago Madera, Jacopo Iacovacci, Aditi Gulati, Andrew J. Wicks, J. Alexander, S. Pettitt, Andrew N J Tutt, C. J. Lord
          </td>
          <td>2025-03-03</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="R-loops are branched nucleic acid structures formed during transcription. The nascent RNA reanneals with the template DNA and displaces the non-template DNA as single stranded. The displaced DNA is now more vulnerable to damage and thus R-loops contribute to genomic instability. From this perspective, R-loops control gene expression and are linked to several pathologies such as cancer and neurological disorders. Despite the importance of R-loops and their widespread on the human genome, it is unknown how chromatin structure and supercoiling impacts R-loop formation. DNA supercoiling refers to the twist in DNA helicity that impacts DNA conformation and torsion. DNA supercoiling is tightly balanced in the cell and regulates transcription, replication, protein recognition and can also contribute to DNA strain and instability yet its role in R-loop biogenesis is poorly understood. To address this gap, we used in vitro transcription (IVT) to reconstitute R-loops on plasmid DNA with distinct topologies containing the mouse antisense Igf2r RNA (Airn) gene, a hotspot for R-loop formation. We used restriction enzyme digestion before and after IVT to relax plasmid DNA (remove supercoiling) and assess the contribution of supercoiling on R-loop formation and stability. R-loops were visualized by agarose gel electrophoresis and revealed that linearization of plasmid DNA reduces their formation. Moreover, linearization of a plasmid containing an R-loop leads to a dramatic reduction in the R-loop species suggesting that supercoiling is required for optimal R-loop stability. Collectively, these observations indicate the requirement for supercoiling for efficient R-loop formation and that linear oligonucleotide fragments are a poor model for structural and mechanistic studies of R-loops.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28de8c482c36f39cce80c1e020d9eeeec361cb36" target='_blank'>
              DNA Supercoiling is Required for Efficient Formation of R-loops
              </a>
            </td>
          <td>
            Taylor White, Charles Bou-Nader
          </td>
          <td>2025-03-01</td>
          <td>Structural Dynamics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background The occurrence and development of colorectal cancer (CRC) is an incredibly long process that involves continuous changes in the tumor microenvironment. These constant changes may ultimately result in genetic alterations and changes in the metabolic processes of some symbiotic bacteria as a way to adapt to the changing environment. Patients with CRC exhibit an altered abundance of Bacteroides fragilis (B. fragilis) as indicated by several studies. To better understand the genomic characteristics and virulence spectrum of B. fragilis strains in tumor tissues, B. fragilis strains were isolated from tumor and paracancerous tissues of CRC patients. Methods The isolates were identified using 16 S rRNA sequencing, morphological analysis, physiological and biochemical characterization and PCR, and they were then subjected to whole genome sequencing (WGS) analysis. Results A strain of B. fragilis enterotoxin (BFT) bft1-producing ZY0302 and a non-enterotoxin-producing B. fragilis ZY0804 were isolated from cancerous and paraneoplastic tissues, respectively. Analysis based on the core and nonessential genes showed that the genomic profiles of the isolates, ZY0302 and ZY0804, differed from those of B. fragilis from other tissue sources. This core and the co-evolution of non-essential genes may be the result of their adaptation to fluctuations in the tumor microenvironment and enhancing their survival. In addition, the ZY0302 and ZY0804 genomes underwent extensive horizontal gene transfer and varying degrees of genomic rearrangements, inversions, insertions, and deletion events, which may favor the enhancement of bacteria’s ability to adapt to environmental changes. For instance, the virulence factors, such as the capsular biosynthesis gene clusters and components of the type IV secretion system, acquired through horizontal gene transfer, may facilitated B. fragilis in evading immune responses and managing oxidative stress. Moreover, our analysis revealed that multiple virulence factors identified in the isolates were mainly involved in bacterial adhesion and colonization, oxidative stress, iron acquisition, and immune evasion. This observation is worth noting given that enzymes such as neuraminidase, lipase, hemolysin, protease, and phosphatase, along with genes responsible for LPS biosynthesis, which are recognized for their association with the virulence of B. fragilis, were prevalent among the isolates. Conclusions In summary, it is our assertion that the alterations observed in both core and nonessential genes of B. fragilis, which have been isolated from tissues of colorectal cancer patients, along with significant instances of horizontal gene transfer to the genome, are likely intended to enhance adaptation to the evolving conditions of the tumor microenvironment. This study may provide new insights into the interaction between B. fragilis and the CRC microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11421-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c824eb8f15816e1643c36807ead38caf17a553" target='_blank'>
              Genomic alterations in Bacteroides fragilis favor adaptation in colorectal cancer microenvironment
              </a>
            </td>
          <td>
            Hao Yang, Yu Gan, Shenghai Jiang, Xianchang Zhu, Yang Xia, Dengmei Gong, Xianrang Xie, Yao Gong, Yi Zhang, Qian Lei, Maijian Wang, Jida Li
          </td>
          <td>2025-03-18</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041f509af47278c9ea1863c279f50106d1d6e43b" target='_blank'>
              The 12th 3R + 3C International Symposium: A Meeting for Research Into DNA Replication, Repair, and Recombination, as Well as Chromatin, Chromosomes, and the Cell Cycle.
              </a>
            </td>
          <td>
            Tsutomu Katayama, Masatoshi Fujita, Tatsuro S Takahashi
          </td>
          <td>2025-04-10</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Objectives: Breast cancer is the most prevalent cancer in adult women. Currently, new therapies and protein determinations with prognostic value are under development. Fascin (encoded by the FSCN1 gene) is an actin-binding protein that is critical for the development of cytoplasmic projections that are essential for tumor invasion. DNA topoisomerase 2-alpha (TOP2A) is a nuclear protein crucial for ATP-dependent breakage, passage, and rejoining of double-stranded DNA and cell division. Both proteins are associated with higher proliferation rates and worse prognosis in breast cancer and together can provide comprehensive information on prognosis and treatment response. Methods: We simultaneously assessed fascin expression and TOP2A/CEP17 DNA copy number ratios in various histological and molecular subtypes. Additionally, these markers were analyzed along with previously established diagnostic markers and other relevant clinical data. Results: Our series included 265 patients, four of whom were male, and all of which were diagnosed with breast carcinoma. Of the 265 patients initially included, sufficient material for analysis was available for 175 cases, as some samples were excluded because of insufficient tissue quantity, poor preservation, or lack of hybridization in certain assays. Immunohistochemical (IHC) expression of fascin, both in its aggregated form and by category, showed no association with the TOP2A gene alteration ratio. Fascin expression was significantly associated with histological subtype (p < 0.001), molecular subtype (p < 0.001), hormone receptor (HR) (p < 0.001), BCL2 (p = 0.003), Ki67 (p = 0.002), and histological grade (p < 0.001). TOP2A was significantly associated with molecular subtype (p = 0.041), Ki67 (p = 0.048), and histological grade (p = 0.033). In our study, molecular subtype (p = 0.037) emerged as an independent variable for the complete histological response to neoadjuvant treatment. Multivariate analysis linked pathological stage (p = 0.002) and estrogen receptor (ER) expression (p = 0.004) to overall survival (OS) and disease-free survival (DFS). Conclusions: No statistical relationship was evident between fascin expression (IHC) and the TOP2A copy ratio. The results of this study suggested that the mechanisms of increased cell proliferation associated with alterations in fascin and TOP2A are independent.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebbe80a72237e1c2774d5c95c120ee2f1e9208a6" target='_blank'>
              Expression of Fascin and DNA Topoisomerase 2-Alpha in Breast Carcinoma: Correlation with Histological Subtypes and Other Prognostic Markers
              </a>
            </td>
          <td>
            Alberto Sánchez-Espinosa, José García-Rodríguez, Virginia Alonso-Aguirre, Jesús Acosta‐Ortega, Pablo Conesa-Zamora, J. García‐Solano, G. Luengo-Gil
          </td>
          <td>2025-03-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background/Objectives: Ultraviolet B (UV-B) is a significant risk factor for skin damage, as it induces cyclobutane pyrimidine dimers (CPD), which suppress DNA replication and transcription. Photolyase (PHR) is a blue light-dependent enzyme that repairs DNA damage caused by UV irradiation. While it is absent in human, it plays a crucial role in repairing CPD in other organisms. Acerola (Malpighia emarginata DC), a fruit with high antioxidant content, is widely consumed for health benefits. This study aimed to identify a novel PHR in acerola and evaluate its photorepair activity. Methods: Using RNA-seq data, we cloned the full-length sequence of the acerola PHR gene and constructed an expression vector. A stable transfected HEK293 cell line (HEK293/acPHR) was established. CPD repair activity was analyzed under blue light in these cells, as well as in normal human dermal fibroblasts (NHDFs) supplemented with extracellular vesicles (EVs) from HEK293/acPHR cells and extracellular vesicle-like nanoparticles derived from acerola extract. Results: Blue light-dependent CPD reduction was observed in HEK293/acPHR cells compared to control cells following UV-B irradiation. Additionally, CPD repair activity was demonstrated in NHDFs and HEK293 cells treated with EVs from HEK293/acPHR cells and nanoparticles from acerola extract. Conclusions: Acerola-derived PHR exhibits the potential to repair UV-induced DNA damage in human cells. Furthermore, EV-mediated delivery of PHR provides a promising avenue for extending photorepair capabilities to other cells. These findings highlight the potential applications of acerola PHR in the prevention and treatment of UV-induced skin damage and related conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb666706d3fe84c4a2f13c15b7b58624bee3eba8" target='_blank'>
              Acerola-Derived Photorepair System for Eliminating Ultraviolet-Induced Pyrimidine Dimers in Human Cells
              </a>
            </td>
          <td>
            Mamoru Yanagimachi, Tomohiro Umezu, M. Takanashi, Yoshiki Murakami, Takahiro Ochiya, Masahiko Kuroda
          </td>
          <td>2025-02-25</td>
          <td>Nutrients</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55117bc2de6fcbb4b98eef78862245221661654d" target='_blank'>
              Diverse evolutionary trajectories of mitocoding DNA in mammalian and avian nuclear genomes.
              </a>
            </td>
          <td>
            Yu-Chi Chen, David Lj Vendrami, Maximilian L. Huber, Luisa E. Y. Handel, Christopher R. Cooney, J. Hoffman, T. I. Gossmann
          </td>
          <td>2025-03-31</td>
          <td>Genome research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a92d215150e22b00d817410c07ac0e4856717a42" target='_blank'>
              Multidimensional OMICs reveal ARID1A orchestrated control of DNA damage, splicing, and cell cycle in normal-like and malignant urothelial cells.
              </a>
            </td>
          <td>
            Rebecca M Schlösser, Florian Krumbach, Eyleen Corrales, G. Andrieux, Christian Preisinger, Franziska Liss, A. Golzmann, M. Boerries, Kerstin Becker, Ruth Knüchel, Stefan Garczyk, Bernhard Lüscher
          </td>
          <td>2025-04-01</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccd321a5ebdb5267d14caeaf74b23869909c920a" target='_blank'>
              Restoration of tumor suppressor protein with enhanced activity using engineered tRNAs to induce tumor regression
              </a>
            </td>
          <td>
            Bangyan Kong, Zhetao Zheng, Zeyuan Mi, Zhiyang Dou, Yuelin Yang, Yuxuan Shen, Lian-Hua Dong, Jingru Lv, Ying Li, Qing Liang, Songlin Tian, Jiayu Wang, Hua Qiao, Qing Xia
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background. Mutations in homologous recombination repair (HRR) genes (BRCA2, ATM, CHEK2, NBN, etc.) are found in 20–25 % of patients with metastatic prostate cancer (PC) and are an indication for prescription of PARP inhibitors. Sensitivity to these drugs results from homologous recombination deficiency (HRD) in the tumor. It is currently not fully elucidated which HRR genes besides BRCA1/2 cause HRD. The aim of the study was to evaluate the presence of homologous recombination deficiency in pc associated with mutations in different HRR genes. Material and methods. Paired tumor and normal DNA samples from 272 pc patients were examined using the HiSNP Ultra Panel v1.0 NGS panel (Nanodigmbio). Tumor copy number variation profiles were used to obtain the HRD score, which was determined as the unweighted sum of three characteristics: the number of chromosomal regions with loss of heterozygosity (LOH), large scale state transitions (LST), and telomeric allelic imbalances (TAI). HRD scores in different pc categories were compared using the Mann–Whitney test. Results. The studied case series included 58 pcs with pathogenic/ likely pathogenic mutations in at least one of 34 HRR genes, and 214 pcs without HRR mutations. The median HRD scores in the groups with BRCA2, ATM, CHEK2, other HRR mutations and without HRR mutations were 41, 22.5, 7.5, 7.5, 14 and 9, respectively. The HRD score was significantly higher in BRCA2-associated tumors than in other PC categories (p<0.01 for all comparisons), except in cases with atm mutations, where the difference did not reach formal significance (p=0.051). Homologous recombination deficiency, defined as HRD score ≥25, was observed in 19/58 (32.8 %) PCs with HRR mutations and 40/214 (18.7 %) tumors without HRR mutations (p=0.03). In tumors without HRR mutations, the HRD score was significantly higher in the presence of somatic TP53 mutations (p<0.0001). Conclusion. In contrast to BRCA2-associated PCs, most tumors with mutations in the CHEK2, NBN, BLM, FANCM, BRCA1 genes are not characterized by homologous recombination deficiency. in the case of ATM gene lesions, approximately half of pcs have a high HRD score. Testing for HRD score allows identification of a significant proportion (5–19 %, depending on the threshold chosen) of tumors with HRD among pcs that do not contain mutations in HRR genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/475a2bac32f3b3fba84f14504ff2c301042d61c8" target='_blank'>
              Analysis of homologous recombination deficiency in prostate cancer
              </a>
            </td>
          <td>
            A. Iyevleva, S. Aleksakhina, A. Sokolenko, E. A. Otradnova, A. S. Nikitina, K. A. Kashko, M. V. Syomina, N. S. Morozova, A. D. Shestakova, E. Kuligina, E. Imyanitov
          </td>
          <td>2025-03-18</td>
          <td>Siberian journal of oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6af5f4741557d24f0a2d8105bb65cd5398d4c085" target='_blank'>
              Alectinib enhances response to RBM39 degradation via SRPK1 inhibition
              </a>
            </td>
          <td>
            Yurui Ma, E. Poon, Chenchen Jin, Barbara Martins Da Costa, Yuewei Xu, Sadiya Quazi, Nikolaos Zourdoumis, Chiharu Wickremesinghe, L. Chesler, Hector C. Keun, Anke Nijhuis
          </td>
          <td>2025-04-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="DNA–RNA hybrids triggered by double-strand breaks (DSBs) are crucial intermediates during DSB repair, and their timely resolution requires numbers of RNA helicases, including DEAD box 1 (DDX1). However, how these helicases are recruited to DSB-induced hybrids in time remains largely unclear. Here, we revealed that squamous cell carcinoma antigen recognized by T cells 3 (SART3) promotes DDX1 binding to DNA–RNA hybrids at DSBs for optimal homologous recombination (HR) repair. SART3 itself associates with DNA–RNA hybrids and PAR chains and accumulates at DSBs in both PARylation- and DNA–RNA hybrids-dependent fashion. SART3 also associates with DDX1 and is necessary for DDX1 enrichment at DSBs. The defective SART3-DDX1 association observed in cells expressing the cancer-associated variant SART3-R836W impairs not only the accumulation of DDX1, but also hybrid removal and HR efficiency. Moreover, SART3 promotes DNA end resection through enhancing USP15-BARD1 association and BRCA1-BARD1 retention. Together, our study reveals an role of SART3 in DSB repair, rendering SART3 a promising target for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0676fb55661a5af6ba07cbca79e578a0d145a7" target='_blank'>
              SART3 promotes homologous recombination repair by stimulating DNA-RNA hybrids removal and DNA end resection
              </a>
            </td>
          <td>
            Hui Fu, Min Huang, Honglin Wu, Hui Zheng, Yifei Gong, Lingyu Xing, Juanjuan Gong, Ruiyuan An, Qian Li, Xinyu Jie, Xiaolu Ma, Tie-Shan Tang, Caixia Guo
          </td>
          <td>2025-03-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Preimplantation genetic testing for aneuploidy and for chromosomal structural rearrangement (PGT-A/-SR) can improve clinical pregnancy rates and live birth rates, and shorten the time to pregnancy. The large-scale statistics on their efficacy and accuracy across different centres, as well as the frequency of abnormalities for each chromosome, will provide a valuable supplement to previous research. Methods Patients who had PGT-A or -SR procedures at five reproductive centres from 2018 to 2022 were recruited based on PGT-A/-SR indications. ChromInst and next-generation sequencing (NGS)-based PGT technology were utilised to detect copy number variations in embryos. Sequencing data metrics such as median absolute pairwise difference (MAPD) and detection success rate were analysed to evaluate the robustness of ChromInst. To assess ChromInst’s accuracy, the chromosomal results from amniocentesis, abortions, and neonatal blood was as the gold standard for negative PGT results; the fluorescence in situ hybridisation (FISH), which was performed on embryos that identified as aneuploid through PGT was as the gold standard for positive PGT results. The frequency of abnormalities in each chromosome was also explored in aneuploid embryos. Results A total of 5,730 embryos were tested from 1,015 patients in the study, 391 of whom had PGT-A and 624 of whom had PGT-SR. 99.5% (5,699/5,730) of the embryos had an NGS sequencing MAPD value < 0.25, and 99.3% (5,689/5,730) of the embryos achieved successful PGT-A/-SR detection. Compared with the gold standard, the concordance of negative PGT-A/-SR results was 99.8% (506/507), and that of positive results was 99.8% (1,123/1,125). The euploidy rate in the PGT-A population was 45.9% (981/2,135). The proportion of euploid + balanced embryos was highest among couples with non-polymorphic inversions (44.6%, 152/341), followed by those with Robertsonian translocations (39.0%, 293/752), and lowest among those with reciprocal translocations (22.5%, 483/2,143). Chromosomes 16, 22, and 15 had the highest frequency of autosomal trisomies among the embryos from PGT-A patients, while chromosomes 16, 22, and 21 had the highest frequency of monosomies. High-frequency chromosomes with de novo chromosomal abnormalities for trisomies and monosomies were similar in the PGT-SR patients to those in the PGT-A patients. Conclusions ChromInst-based PGT-A/-SR could accommodate operational variations among different clinical centres, ensuring accurate results through robust and stable detection performance. Prior to PGT-A/-SR, more trustworthy data could be provided to support the genetic counselling. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06242-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c27907d7fcef7fbc30355e3fd09a1e1e55996153" target='_blank'>
              ChromInst: a multicentre evaluation of robustness in aneuploidy and structural rearrangement testing
              </a>
            </td>
          <td>
            W. Niu, Shanjun Dai, Linli Hu, Yao He, Xiqian Zhang, Xia Xue, Li Wu, Haixia Jin, Dun Liu, Keya Tong, Senlin Shi, L. Tian, Yifan Zhou, Guidong Yao, Sijia Lu, Ya-xin Yao, Dunmei Zhao, Lei Jin, H. Bai, Fenghua Liu, Dongyun Liu, Yingpu Sun
          </td>
          <td>2025-02-26</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f7430000b0cae106ff398286a8c011c0d35dbd7" target='_blank'>
              Loss-of-Function genetic Screen Unveils Synergistic Efficacy of PARG Inhibition with Combined 5-Fluorouracil and Irinotecan Treatment in Colorectal Cancer
              </a>
            </td>
          <td>
            Cristina Queralt, Cristina Moreta-Moraleda, Marta Costa, Ferran Grau-Leal, J. Diesch, Carla Vendrell-Ayats, Eva Musulén, C. Bugés, José Luis Manzano, Sara Cabrero, Johannes Zuber, M. Buschbeck, Sonia Forcales, E. Martínez-Balibrea
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Many antibiotics indirectly generate reactive oxygen species (ROS) that can damage bacterial genomes. Oxidised nucleobases become genotoxic when they are targeted for repair through excision, generating a single-strand discontinuity that can be converted to a double-strand break (DSB) by an oncoming replication fork. Because the genomic location of nucleobase oxidation is stochastic, investigating the fate of DNA replication machinery (replisome) at single-strand discontinuities has been limited. Here we have addressed this issue by expressing Cas9 nickases in Bacillus subtilis to create site specific single-strand discontinuities in a bacterial chromosome. We find that nicks in either leading or lagging strand arrest bacterial replication fork progression and generate a DSB that requires repair using homologous recombination to allow replication restart. These discoveries provoke reassessment of the fundamental mechanism of bacterial homologous recombination and provide insights to the development of alternative antimicrobials by identifying a specific pathway that can potentiate ROS-dependent bacterial killing. HIGHLIGHTS Single-strand discontinuities inactivate the bacterial replisome Nicks on the leading or lagging strand template differentially affect fate of the helicase Double-strand breaks are repaired via homologous recombination and elicit the replication restart pathway Resection often generates a DNA strand with a 5′-end that must be processed to ensure productive helicase reloading">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f733169863ada127f78f4c42d1d0de7e45edaadc" target='_blank'>
              Rescuing bacterial genome replication: essential functions to repair a double-strand break and restart DNA synthesis
              </a>
            </td>
          <td>
            Charles Winterhalter, Stepan Fenyk, H. Murray
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Recently, de novo variants in an 18 nucleotide region in the centre of RNU4-2 were shown to cause ReNU syndrome, a syndromic neurodevelopmental disorder (NDD) that is predicted to affect tens of thousands of individuals worldwide. RNU4-2 is a non-protein-coding gene that is transcribed into the U4 small nuclear RNA (snRNA) component of the major spliceosome. ReNU syndrome variants disrupt spliceosome function and alter 5' splice site selection. Here, we performed saturation genome editing (SGE) of RNU4-2 to identify the functional and clinical impact of variants across the entire gene. The resulting SGE function scores, derived from variants' effects on cell fitness, discriminate ReNU syndrome variants from those observed in the population and dramatically outperform in silico variant effect prediction. Using these data, we redefine the ReNU syndrome critical region at single nucleotide resolution, resolve variant pathogenicity for variants of uncertain significance, and show that SGE function scores delineate variants by phenotypic severity. Further, we identify variants impacting function in regions of RNU4-2 that are critical for interactions with other spliceosome components. We show that these variants cause a novel recessive NDD that is clinically distinct from ReNU syndrome. Together, this work defines the landscape of variant function across RNU4-2, providing critical insights for both diagnosis and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be67df893954c2a33da2a88fca579e4f84ef0e0b" target='_blank'>
              Saturation genome editing of RNU4-2 reveals distinct dominant and recessive neurodevelopmental disorders
              </a>
            </td>
          <td>
            J. De Jonghe, H. C. Kim, A. Adedeji, E. Leitao, R. Dawes, Y. Chen, A. Blakes, C. Simons, R. Rius, J. Alvi, F. Amblard, C. Austin-Tse, S. Baer, E. Balton, P. Blanc, D. Calame, C. Coutton, C. Cunningham, N. Dargie, K. M. Dipple, H. Du, S. El Chehadeh, I. Glass, J. G. Gleeson, O. Grunewald, P. Gueguen, R. Harbuz, M.-L. Jacquemont, R. Leventer, P. Marijon, O. Messaoud, T. Sultan, C. Thauvin, C. Vincent-Delorme, E. Yilmaz Gulec, J. Thévenon, R. Mendez, D. MacArthur, C. Depienne, C. Nava, N. Whiffin, G. M. Findlay
          </td>
          <td>2025-04-11</td>
          <td>None</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Purpose Small cell carcinoma of the ovary, hypercalcemic type (SCCOHT) is a rare and lethal cancer lacking effective treatment. Its genomic mutations and tumor microenvironment need further exploration. Methods We performed whole-exome sequencing or gene panel test to explore the SMARCA4 mutation spectrum in SCCOHT (15 samples). Single-cell RNA sequencing was conducted on one primary lesion with matched normal ovarian tissue and one recurrent lesion to investigate the intra-tumoral heterogeneity and immune microenvironment. Multiplex immunofluorescence staining validated T cell infiltration and PD-1 expression. Results 13/15 (86.7%) patients harbored SMARCA4 mutations. The loss of heterozygosity (LOH) occurred in 10/15 (66.7%) patients. Cancer cells and immune cells were observed in SCCOHT tumors. Cancer cells were further divided into seven subtypes and one from recurrent lesion exhibited the highest stemness accompanied by high expression of genes related to cell mitosis (AURKB, CHEK2, CCNB1, WEE1), DNA repair (BRCA1, RAD51) and epigenetic (EZH2, DNMT1). Immune cells mainly included macrophages and T cells. Lipid-associated tumor-associated macrophages (TAMs) was mainly in primary lesion while inflammatory cytokine-enriched TAMs in recurrent lesion. CD4+/ CD8+ T cell infiltration was observed in SCCOHT tumor and a certain proportion of T cells expressed PD-1. Conclusions SCCOHT exhibits universal SMARCA4 LOH and significant intra-tumoral heterogeneity, suggesting potential therapeutic targets, including CHEK2, CCNB1, and WEE1. Exhausted T cells and distinct TAM subsets infiltrate tumors. Targeting macrophage polarization or cytokine signaling may also be promising. These findings provide insights for developing novel therapies to improve outcomes in SCCOHT. Clinical trial number Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01649-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdba27a2b17d1b242308fa85ab562553932c973d" target='_blank'>
              Single-cell RNA sequencing reveals the intra-tumoral heterogeneity and immune microenvironment of small cell carcinoma of the ovary, hypercalcemic type
              </a>
            </td>
          <td>
            Yijun Gao, Kewei Zheng, Haowen Tan, Mingyi Kang, Bingjian Lu, Ling Chen, Jing Xu, Chong Lu, Ranran Chai, Congjian Xu, Yu Kang
          </td>
          <td>2025-04-09</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adaf9f6208a7ba956fdb23f77caee472cbc415b2" target='_blank'>
              Dual Disruption of DNA Repair by a Novel CHK2 Inhibitor, ART-446, and Olaparib is a Promising Strategy for Triple-Negative Breast Cancer Therapy.
              </a>
            </td>
          <td>
            Hong-Jun Kang, Y. Kang, Ha-Young Lee, Sojung Ha, Jong Oh Kim, Woo-Young Kim, Taegon Baik
          </td>
          <td>2025-04-08</td>
          <td>Biomolecules & therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0818d31f654dc2a6f66d1620f983e96d2fa6b573" target='_blank'>
              Genomic and fragmentomic landscapes of cell-free DNA for early cancer detection.
              </a>
            </td>
          <td>
            D. C. Bruhm, Nicholas A. Vulpescu, Zachariah H. Foda, J. Phallen, Robert B Scharpf, V. Velculescu
          </td>
          <td>2025-03-04</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="The hierarchical organization of the eukaryotic genome is crucial for nuclear activities and cellular development. Genetic aberrations can disrupt this 3D genomic architecture, potentially driving oncogenesis. However, current research often lacks a comprehensive perspective, focusing on specific mutation types and singular 3D structural levels. Here, pathological changes from chromosomes to nucleotides are systematically cataloged, including 10 789 interchromosomal translocations (ICTs), 18 863 structural variants (SVs), and 162 769 single nucleotide polymorphisms (SNPs). The multilayered analysis reveals that fewer than 10% of ICTs disrupt territories via potent 3D interactions, and only a minimal fraction of SVs disrupt compartments or intersect topologically associated domain structures, yet these events significantly influence gene expression. Pathogenic SNPs typically show reduced interactions within the 3D genomic space. To investigate the effects of variants in the context of 3D organization, a two-phase scoring algorithm, 3DFunc, is developed to evaluate the pathogenicity of variant-gene pairs in cancer. Using 3DFunc, IGHV3-23's critical role in chronic lymphocytic leukemia is identified and it is found that three pathological SNPs (rs6605578, rs7814783, rs2738144) interact with DEFA3. Additionally, 3DGAtlas is introduced, which provides a highly accessible 3D genome atlas and a valuable resource for exploring the pathological effects of genetic mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daa65f99b6475aca863361e2ed2a67fe83692271" target='_blank'>
              A 3D Genome Atlas of Genetic Variants and Their Pathological Effects in Cancer.
              </a>
            </td>
          <td>
            Li Tang, M. C. Hill, Mingxing He, Junhao Chen, Zirui Wang, Patrick T. Ellinor, Min Li
          </td>
          <td>2025-03-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eed4652faa332d57f89c15c04ea896c21df38766" target='_blank'>
              Individualized pseudogenes networks for survival prognosis in B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Arturo Kenzuke Nakamura-García, M. Kuijjer, J. Espinal-Enríquez
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Uniparental disomy is a type of chromosomal variation leading to in which both homologous chromosomes or chromosomal regions are inherited from one parent. Such variations have been detected for all chromosomes. The frequency of uniparental disomies is estimated at 1 case per 2000 births. The causes of uniparental disomies include errors during meiosis, postzygotic errors, Robertsonian and reciprocal translocations. Clinical manifestations are associated with loss of heterozygosity for pathogenic genetic variants and defects in genomic imprinting.Currently, the diagnosis of uniparental disomy is performed using methods such as microsatellite analysis, chromosomal microarray analysis, methyl-sensitive PCR, methyl-specific amplification of a probe dependent on multiplex ligation and next-generation sequencing (NGS). The methods used nowadays separately do not allow for a definitive diagnosis of uniparental disomy. A combination of NGS method that simultaneously assesses the DNA methylation status and regions of loss of heterozygosity, in particular those based on fragmentation of genomic DNA by methyl-dependent restriction enzymes, with classical approaches such as methyl-sensitive PCR and microsatellite testing will enable rapid and accurate diagnosis of uniparental disomies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26da94ff26872e12435f3b4e40259e455c4432b4" target='_blank'>
              Whole-genome sequencing for the identification of uniparental disomy
              </a>
            </td>
          <td>
            P. A. Suchko, A. Danilova, E. V. Pinkovskaya, O. Glotov, L. Danilov
          </td>
          <td>2025-03-05</td>
          <td>Rossiyskiy Vestnik Perinatologii i Pediatrii (Russian Bulletin of Perinatology and Pediatrics)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The landscapes of somatic mutation in normal cells inform on the processes of mutation and selection operative throughout life, permitting insight into normal ageing and the earliest stages of cancer development. Here, by whole-genome sequencing of 238 microdissections from 30 individuals, including 18 with gastric cancer, we elucidate the developmental trajectories of normal and malignant gastric epithelium. We find that gastric glands are units of monoclonal cell populations which accrue ∼28 somatic single nucleotide variants per year, predominantly attributable to endogenous mutational processes. In individuals with gastric cancer, glands often show elevated mutation burdens due to acceleration of mutational processes linked to proliferation and oxidative damage. These hypermutant glands were primarily detected in the gastric antrum and were mostly associated with chronic inflammation and intestinal metaplasia, known cancer risk factors. Unusually for normal cells, gastric epithelial cells often carry recurrent trisomies of specific chromosomes, which are highly enriched in a subset of individuals. Surveying approximately 8,000 gastric glands by targeted sequencing, we found somatic driver mutations in a distinctive repertoire of known cancer genes, including ARID1A, CTNNB1, KDM6A and ARID1B. Their prevalence increases with age to occupy approximately 5% of the gastric epithelial lining by age 60 years. Our findings provide insights into the intrinsic and extrinsic influences on somatic evolution in the gastric epithelium, in healthy, precancerous and malignant states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a80de4630550f795baa857c6ee76f35822aa4bc" target='_blank'>
              The somatic mutation landscape of normal gastric epithelium
              </a>
            </td>
          <td>
            Tim H. H. Coorens, G. Collord, Hyungchul Jung, Yichen Wang, L. Moore, Y. Hooks, Krishnaa Mahbubani, K. Saeb-Parsy, P. J. Campbell, I. Martincorena, Suet Yi Leung, Michael R Stratton
          </td>
          <td>2024-03-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>88</td>
        </tr>

        <tr id="The genomes of multi-replicon bacteria are composed of a primary replicon (the chromosome) and secondary replicons (chromids). Currently, there is a lack of understanding of the mutation features and evolutionary patterns of these different replicons. Specifically, in the genus Pseudoalteromonas, the chromids of multi-replicon species exhibit both unidirectional and bidirectional replication. Here, we investigated the similarities and differences between chromosomes and chromids in sequence composition and gene synteny of Pseudoalteromonas species by comparative genomic analysis, as well as the spontaneous mutation features of different replicons by mutation accumulation (MA) experiments combined with whole-genome sequencing strategy (MA-WGS). MA-WGS analysis revealed that there was no significant difference between chromids and chromosomes in the mutation rate or mutation spectrum of P. sp. LC0214 (where the chromid is unidirectional in replication) and P. sp. JCM12884T (where the chromid is bidirectional in replication). In addition, the context-dependence and variation pattern of the base-pair substitutions (BPSs) rates of the entire replicons exhibited differences that may be caused by the different replication directions of the chromids. The results of this study provide a new theoretical foundation for an in-depth understanding of the origin and evolution of chromids in multi-replicon bacterial species and facilitate further exploration of the complex mechanisms of bacterial diversity.IMPORTANCEDe novo mutations are a critical driving force in species evolution. Currently, there is a lack of sufficient research on the influence of replicon types on the occurrence of genomic mutations in bacteria. Moreover, the scarcity in systematic analysis and comparison of spontaneous mutation features between different replicons results in the limited information on the evolutionary dynamics of multi-replicon species. The diversity of replication direction in the multi-replicon species of the genus Pseudoalteromonas provides a unique opportunity for studying the impact of replication direction on the patterns of mutation. In addition to the composition characteristics between chromosomes and chromids, the spontaneous mutation rates in the context-dependence and variation pattern of the base-pair substitutions (BPSs) across different replicons within Pseudoalteromonas species revealed in this study provide valuable insights into the evolutionary dynamics of bacterial secondary replicons.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8323ad0a022a4612f59827ebdd0205de2a451a58" target='_blank'>
              Mutational features of chromids and chromosomes in Pseudoalteromonas provide new insights into the evolution of secondary replicons.
              </a>
            </td>
          <td>
            Wanyue Jiang, Jiaoyi Pan, Tongtong Lin, Yaohai Wang, Yanze Wang, Rongxiao Zhang, Xiaoming Zhou, Yu Zhang
          </td>
          <td>2025-03-25</td>
          <td>Microbiology spectrum</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34833006929f8c53f4dd25f1eecd3e7760e96187" target='_blank'>
              Biochemical Mechanisms of Genetic Recombination and DNA Repair.
              </a>
            </td>
          <td>
            Vivek B. Raina, Aidan M. Jessop, Eric C Greene
          </td>
          <td>2025-03-28</td>
          <td>Annual review of biochemistry</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Stem cells display asymmetric histone inheritance, while nonstem progenitor cells exhibit symmetric patterns in the Drosophila male germ line. Here, we report that components involved in lagging strand synthesis, DNA polymerases α and δ, have substantially reduced levels in stem cells compared to progenitor cells, and this promotes local asymmetry of parental histone incorporation at the replication fork. Compromising Polα genetically induces the local replication-coupled histone incorporation pattern in progenitor cells to resemble that in stem cells, seen by both nuclear localization patterns and chromatin fibers. This is recapitulated using a Polα inhibitor in a concentration-dependent manner. The local old versus new histone asymmetry is comparable between stem cells and progenitor cells at both S phase and M phase. Together, these results indicate that developmentally programmed expression of key DNA replication components is important to shape stem cell chromatin. Furthermore, manipulating one crucial DNA replication component can induce replication-coupled histone dynamics in nonstem cells to resemble those in stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8c0df555eddd954cf53afad22d00f47394127d" target='_blank'>
              Increased levels of lagging strand polymerase α in an adult stem cell lineage affect replication-coupled histone incorporation
              </a>
            </td>
          <td>
            Brendon E. M. Davis, Jonathan C. Snedeker, R. Ranjan, Matthew Wooten, Savannah Sáde Barton, Joshua Blundon, Xin Chen
          </td>
          <td>2025-02-28</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Cancer is a disease of multicellularity, observed across the tree of life. In principle, animals with larger body sizes and longer lifespans should be at increased risk of developing cancer. However, there is no strong relationship between these traits and cancer across mammals. Previous studies have proposed that increased copy number of cancer-related genes may enhance the robustness of cancer suppression pathways in long-lived mammals, but these studies have not extended beyond known cancer-related genes. In this study, we conducted a phylogenetic generalized least squares analysis to test for associations between copy number of all protein-coding genes and longevity, body size, and cancer prevalence across 94 species of mammals. In addition to investigating the copy number of individual genes, we tested sets of related genes for a relationship between the aggregated gene copy number of the set and these traits. We did not find strong evidence to support the hypothesis that adaptive changes in gene copy number contribute to the lack of correlation between cancer prevalence and body size or lifespan. However, we found several biological processes where aggregate copy number was associated with malignancy rate. The strongest association was for the gene set relating to transforming growth factor beta, a cytokine that plays a role in cancer progression. Overall, this study provides a comprehensive evaluation of the role of gene copy number in adaptation to body size and lifespan and sheds light on the contribution of gene copy number to variation in cancer prevalence across mammals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12a5893a996e91a392fac375709075fef182cddc" target='_blank'>
              Variable Gene Copy Number in Cancer-Related Pathways Is Associated With Cancer Prevalence Across Mammals
              </a>
            </td>
          <td>
            Sophie Matthews, Vahid Nikoonejad Fard, Marc Tollis, Cathal Seoighe
          </td>
          <td>2025-03-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cancer is a rising issue worldwide, and numerous studies have focused on understanding the underlying reasons for its occurrence and finding proper ways to defeat it. By applying technological advances, researchers are continuously uncovering and updating treatments in cancer therapy. Their vast functions in the regulation of cell growth and proliferation and their significant role in the progression of diseases, including cancer. This review provides a comprehensive analysis of ncRNAs in breast cancer, focusing on long non-coding RNAs such as HOTAIR, MALAT1, and NEAT1, as well as microRNAs such as miR-21, miR-221/222, and miR-155. These ncRNAs are pivotal in regulating cell proliferation, metastasis, drug resistance, and apoptosis. Additionally, we discuss experimental approaches that are useful for studying them and highlight the advantages and challenges of each method. We then explain the results of these clinical trials and offer insights for future studies by discussing major existing gaps. On the basis of an extensive number of studies, this review provides valuable insights into the potential of ncRNAs in cancer therapy. Key findings show that even though the functions of ncRNAs are vast and undeniable in cancer, there are still complications associated with their therapeutic use. Moreover, there is an absence of sufficient experiments regarding their application in mouse models, which is an area to work on. By emphasizing the crucial role of ncRNAs, this review underscores the need for innovative approaches and further studies to explore their potential in cancer therapy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2d581bb8c9e5870e5b44e989456c39919368fe9" target='_blank'>
              Non-coding RNAs, a double-edged sword in breast cancer prognosis
              </a>
            </td>
          <td>
            Maryam Solaimani, Sahar Hosseinzadeh, M. Abasi
          </td>
          <td>2025-04-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76a9d7d928f4b0e1d14db6867d082f27a7a018ac" target='_blank'>
              A Novel Approach to Linking Histology Images with DNA Methylation
              </a>
            </td>
          <td>
            Manahil Raza, Muhammad Dawood, Talha Qaiser, Nasir M. Rajpoot
          </td>
          <td>2025-04-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Epigenetic dysregulation is prevalent in human cancers, affecting gene expression and metabolic patterns to meet the demands of malignant evolution and abnormal epigenetic processes, and resulting in a protumor immune microenvironment. Tumors require a steady supply of methionine for maintaining epigenetic flexibility, which is the only exogenous precursor of methyl donor S‐adenosylmethionine for methylation, crucial for their resistance to therapies and survival in a nutrient‐deficient microenvironment. Thus, tumor cells upregulate the Lat4 transporter to compete and deprive methionine in the microenvironment, sustaining their malignant phenotypes and also impairing immune cell functions. Addressing this methionine addiction is the key to overcoming drug resistance and improving immune response. Despite the challenge of lacking specific Lat4 inhibitors, an oxaliplatin prodrug crosslinked fluorinated polycation/anti‐Lat4 small interfering RNA complex nanoregulator (AS‐F‐NP) has been designed and developed here. This nanoregulator restricted the greedy methionine uptake of tumor cells by knocking down Lat4, which in turn inhibited the malignant evolution of the tumor while restoring the viability and function of tumor‐infiltrating immune cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e01c550a22a629f238ced405d904af38f03862" target='_blank'>
              A Methionine Allocation Nanoregulator for the Suppression of Cancer Stem Cells and Support to the Immune Cells by Epigenetic Regulation
              </a>
            </td>
          <td>
            Boyu Su, Qinjun Chen, Xuwen Li, Mingzhu Fang, Yu Wang, Haolin Song, Haoyu You, Zhen-yong Zhou, Yuxing Wu, Zhenhao Zhao, Yun Chen, Hongrui Fan, Chufeng Li, Chen Jiang, Tao Sun
          </td>
          <td>2025-02-22</td>
          <td>Advanced Science</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47bc580b247bccc66bfbe3ed13daf3da5e31eb79" target='_blank'>
              Eukaryotes evade information storage-replication rate trade-off with endosymbiont assistance leading to larger genomes
              </a>
            </td>
          <td>
            Parthasarathi Sahu, Sashikanta Barik, Koushik Ghosh, Hemachander Subramanian
          </td>
          <td>2025-02-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background The tumor suppressor protein, p53, which is mutated in half of human tumors, plays a critical role in cellular responses to DNA damage and maintenance of genome stability. Therefore, increasing our understanding of the p53 pathway is essential for improving cancer treatment and diagnosis. Methods This study, which aimed to identify genes and pathways that mediate resistance to p53 upregulation, used genome-wide CRISPR-Cas9 loss-of-function screening done with Nutlin-3a, which inhibits p53-MDM2 interaction, resulting in p53 accumulation and apoptotic cell death. We used bioinformatics analysis for the identification of genes and pathways that are involved in the p53 pathway and cell survival assays to validate specific genes. In addition, we used RNA-seq to identify differentially expressed p53 target genes in gene knockout (KO) cell lines. Results Our screen revealed three significantly enriched pathways: The heparan sulfate glycosaminoglycan biosynthesis, diphthamide biosynthesis and Hippo pathway. Notably, TRIP12 was significantly enriched in our screen. We found that TRIP12 is required for the p53-dependent transcription of several pro-apoptotic genes. Conclusion Our study has identified two novel pathways that play a role in p53-mediated growth restriction. Moreover, we have highlighted the interaction between the Hippo and the p53 pathways. Interestingly, we have shown that TRIP12 plays an important function in the p53 pathway by selectively affecting its role as a transcription factor. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01141-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f2f128a395ad66b24bd497a265456014e21a60" target='_blank'>
              Genome-wide screening in human embryonic stem cells identifies genes and pathways involved in the p53 pathway
              </a>
            </td>
          <td>
            Amir Haddad, T. Golan‐lev, N. Benvenisty, Michal Goldberg
          </td>
          <td>2025-03-13</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Background CD44, a widely recognized cancer stem cell marker, displayed a vital participation in the cancer immune invasion and may related with the response to the immunotherapy. However, the role of CD44 in cancer immunology is not well defined. Therefore, we intended to explore its prognostic value and potential immunological functions across 33 human cancer types. Methods Based on the data of patients from The Cancer Genome Atlas (TCGA), Sangerbox was used to analyze the correlations between CD44 expression and tumor-infiltrated immune cells, immune checkpoints, neoantigens, microsatellite instability (MSI), and tumor mutational burden (TMB) in human cancers. A mouse model xenografted with shRNA-CD44 MC38 was established. Results The elevated CD44 was associated with tumor stage and prognosis in several different cancers. GSEA results showed that upregulated CD44 involved in cancer stem cell associated process, antigen processing and presentation, and immune cells proliferation and activation. CD44 plays an essential role in the tumor immune regulation and immune checkpoints inhibitor response. The correlation of CD44 gene expression and infiltration levels of immune cells varied across different cancer types. Notably, the upregulation of CD44 expression is positively correlated with regulatory CD4 T cells, macrophages M1 and M2 in several analyzed cancers. Furthermore, we verified the effect of CD44 on tumor growth and immune microenvironment in mouse xenografted with shRNA-CD44 MC38. Moreover, DNA methylation existed in CD44 expression and associated with dysfunctional T-cell phenotypes via different mechanisms, thus resulting in tissue-dependent prognoses. Conclusion CD44 is both a cancer stem cell marker and a potential prognostic and immunological biomarker in various malignant tumors. Moreover, CD44 could be a novel target for immune-based therapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03748-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e36514de7406397ff66824fce066565eed39db70" target='_blank'>
              CD44 on cancer stem cell is a potential immunological and prognostic pan-cancer biomarker
              </a>
            </td>
          <td>
            Ya-Juan Zhu, , Shan-Shan Yang, Yang Du, Zhuo-Yuan Zhang, Ji-Yan Liu
          </td>
          <td>2025-04-08</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a502514bcaada21614887b3c2bdc65fdc9de6a2" target='_blank'>
              Regulation of nucleotide excision repair by wild-type HRAS signaling in head and neck cancer.
              </a>
            </td>
          <td>
            L. Hoxhallari, Konstantinos Katsikis, Antigoni Makri, Marialena Pouliou, Z. Kanaki, G. Vatsellas, Christina Sonou, D. Telios, Evangelos I Giotakis, Aristeidis Giotakis, Periklis Makrythanasis, M. Agelopoulos, Amanda Psyrri, T. Rampias
          </td>
          <td>2025-04-12</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Birt-Hogg-Dubé syndrome (BHD) patients are uniquely susceptible to all renal tumor subtypes. However, the underlying mechanism of carcinogenesis is unclear. To study cancer development in BHD, we used human proximal kidney (HK2) cells and found that long-term folliculin (FLCN) knockdown was required to increase the tumorigenic potential of these cells, as evidenced by the formation of larger spheroids under nonadherent conditions. Transcriptomic and proteomic analyses revealed links between the FLCN, cell cycle control and DNA damage response (DDR) machinery. In addition, HK2 cells lacking FLCN had an altered transcriptome profile and enriched cell cycle control genes. G1/S cell cycle checkpoint signaling was compromised by increased protein levels of cyclin D1 (CCND1) and hyperphosphorylation of retinoblastoma 1 (RB1). A FLCN interactome screen revealed that FLCN binds to DNA-dependent protein kinase (DNA-PK). This novel interaction was reversed in an irradiation-responsive manner. Knockdown of FLCN in HK2 cells caused a marked increase in γH2AX and RB1 phosphorylation. The levels of both CCND1 and phosphorylated RB1 remained high during DNA damage, which was associated with defective cell cycle control caused by FLCN knockdown. Furthermore, Flcn-knockdown C. elegans were defective in cell cycle arrest caused by DNA damage. This work revealed that long-term FLCN loss and associated cell cycle defects in BHD patients could contribute to their increased risk of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce72e1df530d7f8cfbe56a7a8e356852f9f7441" target='_blank'>
              Characterizing the tumor suppressor activity of FLCN in Birt-Hogg-Dubé syndrome cell models through transcriptomic and proteomic analysis.
              </a>
            </td>
          <td>
            Rachel-Ann Jones, E. Dunlop, Jesse D Champion, Peter F Doubleday, T. Claessens, Zahra Jalali, Sara Seifan, Iain Perry, Peter J. Giles, Oliver Harrison, Barry J. Coull, A. Houweling, A. Pause, Bryan A Ballif, Andrew R. Tee
          </td>
          <td>2025-03-25</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f37dc921f83b90daf61b10b8456e0c63c032e710" target='_blank'>
              [Pathogenic large duplication in TP53 as a hereditary predisposing factor in breast cancer].
              </a>
            </td>
          <td>
            Viktória Kovács, H. Butz, János Papp, T. Pócza, Kornél Vince Grolmusz, Petra Nagy, Attila Patócs, A. Bozsik
          </td>
          <td>2025-03-21</td>
          <td>Magyar onkologia</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8876255590dea7ef4a31a07f23a3590537b155f1" target='_blank'>
              Immunogenomic cancer evolution: A framework to understand cancer immunosuppression.
              </a>
            </td>
          <td>
            S. Kumagai, Yusaku Momoi, H. Nishikawa
          </td>
          <td>2025-03-28</td>
          <td>Science immunology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Oscillatory p53 expression occurs in individual cells responding to DNA breaks. While the majority of cells exhibit the same qualitative response, quantitative features of the oscillations (e.g., amplitude or period) can be highly variable between cells, generating heterogeneity in downstream cell fate responses. Since heterogeneity can be detrimental to therapies based on DNA damage, methods to induce synchronization of p53 oscillations across cells in a population have the potential to generate more predictable responses to DNA-damaging treatments. Using mathematical modeling and time-lapse microscopy, we demonstrated that p53 oscillations can be synchronized through the phenomenon of phase resetting. Surprisingly, p53 oscillations were synchronized over a wider range of damage-induction frequencies than predicted computationally. Recapitulating the range of synchronizing frequencies required, non-intuitively, a less robust oscillator. We showed that p53 phase resetting altered the expression of downstream targets responsible for cell fate depending on target mRNA stability. This study demonstrates that p53 oscillations can be phase reset and highlights the potential of driving p53 dynamics to reduce cellular variability and synchronize cell fate responses to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/372ff4f36befbd056cd96855dd2d6817d3ff38d6" target='_blank'>
              Pulsed stimuli enable p53 phase resetting to synchronize single cells and modulate cell fate
              </a>
            </td>
          <td>
            Harish Venkatachalapathy, Samuel Dallon, Zhilin Yang, Samira M. Azarin, Casim A. Sarkar, Eric Batchelor
          </td>
          <td>2025-03-03</td>
          <td>Molecular Systems Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Single-stranded DNA (ssDNA) gaps impact genome stability and PARP inhibitor (PARPi) sensitivity, especially in BRCA1/2-deficient tumors. Using single-molecule DNA fiber analysis, electron microscopy, and biochemical methods, we found that MRN, CtIP, EXO1, and DNA2-WRN/BLM resect ssDNA gaps through a mechanism different from their actions at DNA ends. MRN resects ssDNA gaps in the 3'-to-5' direction using its pCtIP-stimulated exonuclease activity. Unlike at DNA ends, MRN does not use its endonucleolytic activity to cleave the 5'-terminated strand flanking the gap or the ssDNA. EXO1 and DNA2-WRN/BLM specifically resect the 5' end of the gap independent of MRN-CtIP. This resection process alters ssDNA gap repair kinetics in BRCA1-proficient and -deficient cells. In BRCA1-deficient cells treated with PARPis, excessive resection results in larger ssDNA gaps, hindering their repair and leading to DNA breaks in subsequent cell cycle stages due to ssDNA gaps colliding with DNA replication forks. These findings broaden our understanding of the role of human nucleases in DNA metabolism and have significant implications for defining the mechanisms driving PARPi sensitivity in BRCA-deficient tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbd5bfadd87ec60e42f11dd8435521e451769fad" target='_blank'>
              MRN-CtIP, EXO1, and DNA2-WRN/BLM act bidirectionally to process DNA gaps in PARPi-treated cells without strand cleavage.
              </a>
            </td>
          <td>
            Isabelle M Seppa, Ilaria Ceppi, Mithila Tennakoon, Giordano Reginato, Jessica Jackson, Celia D Rouault, Sumedha Agashe, Vladislav O Sviderskiy, Mangsi Limbu, Erica Lantelme, A. Meroni, Stefan Braunshier, Damiano Borrello, Priyanka Verma, Petr Cejka, A. Vindigni
          </td>
          <td>2025-03-24</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb7435e798afc5c276c5bcf5f266a5cdfa975999" target='_blank'>
              Abstract B008: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eda66ec4c5f5122019392e54f6e9f77707173c94" target='_blank'>
              BRCA1 is a molecular correlate of cell proliferation in human brain development and in Group 3 and 4 medulloblastoma
              </a>
            </td>
          <td>
            Ian Cheong, Xinghan Sun, Leo Lau, Nishka Kishore, Andrea Senff‐Ribeiro, A. Díaz-Navarro, Hana Hajari, Ellen Mak, Rhea Ahluwalia, Luca Bianchini, Jaskirat Singh Sandhu, Michael D. Taylor, Lena M. Kutscher, Lincoln D. Stein, Shraddha Pai
          </td>
          <td>2025-04-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Variant calling is hindered in segmental duplications by sequence homology. We developed Paraphase, a HiFi-based informatics method that resolves highly similar genes by phasing all haplotypes of paralogous genes together. We applied Paraphase to 160 long (>10 kb) segmental duplication regions across the human genome with high (>99%) sequence similarity, encoding 316 genes. Analysis across five ancestral populations revealed highly variable copy numbers of these regions. We identified 23 paralog groups with exceptionally low within-group diversity, where extensive gene conversion and unequal crossing over contribute to highly similar gene copies. Furthermore, our analysis of 36 trios identified 7 de novo SNVs and 4 de novo gene conversion events, 2 of which are non-allelic. Finally, we summarized extensive genetic diversity in 9 medically relevant genes previously considered challenging to genotype. Paraphase provides a framework for resolving gene paralogs, enabling accurate testing in medically relevant genes and population-wide studies of previously inaccessible genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/927dd3f13d710a29fb90f0b9b93d268e6b2b6131" target='_blank'>
              Genome-wide profiling of highly similar paralogous genes using HiFi sequencing
              </a>
            </td>
          <td>
            Xiao Chen, Daniel Baker, E. Dolzhenko, Joseph M Devaney, Jessica Noya, April S Berlyoung, Rhonda Brandon, Kathleen S Hruska, Lucas Lochovsky, Paul Kruszka, Scott Newman, Emily G Farrow, I. Thiffault, T. Pastinen, Dalia Kasperaviciute, Christian Gilissen, L. Vissers, A. Hoischen, Seth I. Berger, E. Vilain, Emmanuèle C. Délot, Michael Eberle
          </td>
          <td>2025-03-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/591c39d8ca29bfec5f8e19848bdc01b09dbd47c9" target='_blank'>
              Lake Malawi cichlid pangenome graph reveals extensive structural variation driven by transposable elements.
              </a>
            </td>
          <td>
            Fu Xiang Quah, M. V. Almeida, Moritz Blumer, Chengwei Ulrika Yuan, Bettina Fischer, Kirsten See, Ben Jackson, R. Zatha, B. Rusuwa, George F. Turner, M. E. Santos, H. Svardal, M. Hemberg, R. Durbin, E. Miska
          </td>
          <td>2025-04-10</td>
          <td>Genome research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1178b1ac2d65f9262b22b808cb25c2ac38fa27ab" target='_blank'>
              Mutation of SMARCA4 Induces Cancer Cell-Intrinsic Defects in the Enhancer Landscape and Resistance to Immunotherapy.
              </a>
            </td>
          <td>
            Yawen Wang, Ismail M. Meraz, Md Qudratullah, Sasi Kotagiri, Yanyan Han, Yuanxin Xi, Jing Wang, Kadir C Akdemir, Jack A Roth, Yonathan Lissanu
          </td>
          <td>2025-03-13</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f56b53109651f083205e0fc320c14b9f0267e185" target='_blank'>
              Mining metagenomes and metatranscriptomes unveils viruses associated with cutaneous squamous cell carcinoma in hematopoietic stem cell transplant recipients.
              </a>
            </td>
          <td>
            Jana K Dickter, Yuqi Zhao, Vishwas Parekh, Huiyan Ma, Badri G Modi, Wai-Yee Li, S. Armenian, Xiwei Wu, Farah R Abdulla
          </td>
          <td>2025-03-31</td>
          <td>Microbiology spectrum</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Abstract Background The HIV‐1 reservoir in CD4+ T cells (HRCD4) pose a major challenge to curing HIV, with many of its mechanisms still unclear. HIV‐1 DNA integration and immune responses may alter the host's epigenetic landscape, potentially silencing HIV‐1 replication. Methods This study used bisulphite capture DNA methylation sequencing in CD4+ T cells from the blood of 427 virally suppressed women with HIV to identify differentially methylated sites and regions associated with HRCD4. Results The average total HRCD4 size was 1409 copies per million cells, with most proviruses defective and only a small proportion intact. The study identified 245 differentially methylated CpG sites and 85 regions linked to HRCD4 size, with 52% of significant sites in intronic regions. Genes associated with HRCD4 were involved in viral replication, HIV‐1 latency and cell growth and apoptosis. HRCD4 size was inversely related to DNA methylation of interferon signalling genes and positively associated with methylation at known HIV‐1 integration sites. HRCD4‐associated genes were enriched on the pathways related to immune defence, transcription repression and host–virus interactions. Conclusions These findings suggest that HIV‐1 reservoir is linked to aberrant DNA methylation in CD4+ T cells, offering new insights into epigenetic mechanisms of HIV‐1 latency and potential molecular targets for eradication strategies. Key points Study involved 427 women with HIV. Identified 245 aberrant DNA methylation sites and 85 methylation regions in CD4+ T cells linked to the HIV‐1 reservoir. Highlighted genes are involved in viral replication, immune defence, and host genome integration. Findings suggest potential molecular targets for eradication strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae7ef3531f4c3263677a74b3760489606fbd9df" target='_blank'>
              Aberrant DNA methylation of genes regulating CD4+ T cell HIV‐1 reservoir in women with HIV
              </a>
            </td>
          <td>
            Ke Xu, Xinyu Zhang, Kesava Asam, B. C. Quach, Grier P Page, Deborah Konkle-Parker, Claudia Martinez, Cecile D Lahiri, Elizabeth F Topper, Mardge H. Cohen, S. Kassaye, Jack Dehovitz, M. Kuniholm, N. Archin, Amir Valizadeh, Phyllis C Tien, Vincent C Marconi, Dana B. Hancock, E. O. Johnson, Bradley E Aouizerat
          </td>
          <td>2025-03-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Skin cancer is the most common malignancy worldwide, comprising three major types: melanoma, basal cell carcinoma (BCC), and squamous cell carcinoma (SCC). A critical goal in advancing our understanding of skin cancer is to identify the shared and distinct biological mechanisms that drive the risk profiles for each skin cancer type. In this study, we integrated tissue-specific chromatin conformation and expression quantitative trait loci data (spatial eQTL) to construct gene regulatory networks (GRNs) for melanocytes, sun-exposed skin, not sun-exposed skin, and blood. These GRNs, which capture spatially regulated gene expressions, were used as instrumental variables along with GWAS summary statistics for melanoma, BCC and SCC, to infer causal relationships between specific gene expression changes and each type of skin cancer. These Mendelian randomization analyses identified 82, 62, and 125 causal genes for melanoma, BCC, and SCC, respectively, with many of these genes not evident from the GWAS data alone. Our analyses revealed distinct mechanisms for each skin cancer type: telomere maintenance and nevus pathways in melanoma, inherited immune traits in BCC, and p53 dysfunction in SCC. Notably, apoptosis and pigmentation emerged as shared biological processes across skin cancers. These findings provide new insights into the genetic drivers of three distinct skin cancers, highlighting new gene targets which can be used to increase diagnostic precision, as well as potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d1a0a87d4f015e13185fa2728ac9fd439bf3488" target='_blank'>
              Mendelian randomizations with spatial gene networks reveal shared and distinct drivers of risk in major skin cancer types
              </a>
            </td>
          <td>
            Michael Pudjihartono, J. O’Sullivan, W. Schierding
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e35698f05d68e24632b02180e50c1bfb7426f2b2" target='_blank'>
              The establishment of the 3D genome structure during zygotic genome activation.
              </a>
            </td>
          <td>
            Anastasiia L Sivkina, O. Iarovaia, S. V. Razin, S. Ulianov
          </td>
          <td>2025-03-03</td>
          <td>Annals of the New York Academy of Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f8c8e278af4ce203bace432e47ab666e9ddfe2" target='_blank'>
              Small Genomes, Big Disruptions: Parvoviruses and the DNA Damage Response
              </a>
            </td>
          <td>
            Rhiannon R. Abrahams, Kinjal Majumder
          </td>
          <td>2025-03-29</td>
          <td>Viruses</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a32f7726bad7de5bd6ab6b676111c1867233e6" target='_blank'>
              Precancerous Cells Initiate Glioblastoma Evolution and Contribute to Intratumoral Heterogeneity.
              </a>
            </td>
          <td>
            Hyun Jung Kim, K. Kim, Do Hyeon Cha, Jihwan Yoo, E. Kim, Jong Hee Chang, Seok-Gu Kang, Jung Won Park, Ja Hye Kim, Yeonhee Lee, Eunha Lim, Yiseul Kim, Myeong-Heui Kim, Xue Li, Joo Ho Lee, J. H. Lee
          </td>
          <td>2025-04-16</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Mutator phenotypes are short-lived due to the rapid accumulation of deleterious mutations. Yet, recent observations reveal that certain fungi can undergo prolonged accelerated evolution after losing DNA repair genes. Here, we surveyed 1,154 yeast genomes representing nearly all known yeast species of the subphylum Saccharomycotina to examine the relationship between reduced DNA repair repertoires and elevated evolutionary rates. We identified three distantly related lineages—encompassing 12% of species—with substantially reduced sets of DNA repair genes and the highest evolutionary rates in the entire subphylum. Two of these “faster-evolving lineages” (FELs)—a subclade within the order Pichiales and the Wickerhamiella/Starmerella (W/S) clade (order Dipodascales)—are described here for the first time, while the third corresponds to a previously documented Hanseniaspora FEL. Examination of DNA repair gene repertoires revealed a set of genes predominantly absent in these three FELs, suggesting a potential role in the observed acceleration of evolutionary rates. Genomic signatures in the W/S clade are consistent with a substantial mutational burden, including pronounced A|T bias and signatures of endogenous DNA damage. The W/S clade appears to mitigate UV-induced damage through horizontal acquisition of a bacterial photolyase gene, underscoring how gene loss may be offset by nonvertical evolution. These findings highlight how the loss of DNA repair genes gave rise to hypermutators that persist across macroevolutionary timescales, with horizontal gene transfer as an avenue for partial functional compensation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e78c4fc2b6cc810f590c3dd41520772da92ce0ea" target='_blank'>
              Stable hypermutators revealed by the genomic landscape of DNA repair genes among yeast species
              </a>
            </td>
          <td>
            Carla Gonçalves, Jacob L. Steenwyk, D. Rinker, D. A. Opulente, A. Labella, Marie-Claire Harrison, John F. Wolters, Xiaofan Zhou, Xing‐Xing Shen, S. Covo, M. Groenewald, C. T. Hittinger, A. Rokas
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="All cancer cells must adopt a telomere maintenance mechanism to achieve replicative immortality. Most human cancer cells utilize the enzyme telomerase to maintain telomeres. Alternative splicing of TERT regulates the amount and function of telomerase, however many alternative splicing isoforms of TERT have unknown functions. Single molecule long read RNA/cDNA sequencing of TERT revealed 45 TERT mRNA variants including 13 known and 32 novel variants. Among the variants, TERT Delta 2–4, which lacks exons 2–4 but retains the original open reading frame, was selected for further study. Induced pluripotent stem cells and cancer cells express higher levels of TERT Delta 2–4 compared to primary human bronchial epithelial cells. Overexpression of TERT Delta 2–4 enhanced clonogenicity and resistance to cisplatin-induced cell death. Knockdown of endogenous TERT Delta 2–4 in Calu-6 cells reduced clonogenicity and resistance to cisplatin. Our results suggest that TERT Delta 2–4 enhances cancer cells’ resistance to cell death. RNA sequencing following knockdown of Delta 2–4 TERT indicates that translation is downregulated and that mitochondrial related proteins are upregulated compared to controls. Overall, our data indicate that TERT produces many isoforms that influence the function of TERT and the abundance and activity of telomerase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31160a8616dac1a93a1309bf2063539c61bdfcd2" target='_blank'>
              Discovery and characterization of a novel telomerase alternative splicing isoform that protects lung cancer cells from chemotherapy induced cell death
              </a>
            </td>
          <td>
            Jeongjin J. Kim, Alexander Ahn, Jeffrey Y Ying, Andrew T. Ludlow
          </td>
          <td>2025-02-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/735584e14c88c91d3e142e0b8a6d20e3a98ec51e" target='_blank'>
              C-terminal fusion partner activity contributes to the oncogenic functions of YAP1::TFE3
              </a>
            </td>
          <td>
            Patrick J Cimino, Dylan J Keiser, Abigail G Parrish, Eric C. Holland, Frank Szulzewsky
          </td>
          <td>2025-04-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaaa908df3e4b28207f7ab8e87d00d4a0e587cfc" target='_blank'>
              Integration Analysis of Circle-Sequencing and Transcriptome Reveals Extrachromosomal Circular DNA Is Involved in the Regulation of Vascular Cambium Annual Cycle in Chinese Pine.
              </a>
            </td>
          <td>
            Yushuang Song, Keji Yu, Yayu Guo, Yuan Cao, Changwen Xu, Lingyu Ma, Zhun Zhang, Yiqun Liu, Jinhuan Yin, Zijian Hu, Shiya Shen, Huimin Xu, Xi Zhang, Shihui Niu, Yaning Cui, Jinxing Lin
          </td>
          <td>2025-04-09</td>
          <td>Plant, cell & environment</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Therapy resistance in breast cancer is increasingly attributed to polyploid giant cancer cells (PGCCs), which arise through whole genome doubling and exhibit heightened resilience to standard treatments. Characterized by enlarged nuclei and increased DNA content, these cells tend to be dormant under therapeutic stress, driving disease relapse. Despite their critical role in resistance, strategies to effectively target PGCCs are limited, largely due to the lack of high-throughput methods for assessing their viability. Traditional assays lack the sensitivity needed to detect PGCC-specific elimination, prompting the development of novel approaches. To address this challenge, we developed a high-throughput single-cell morphological analysis workflow designed to differentiate compounds that selectively inhibit non-PGCCs, PGCCs, or both. Using this method, we screened a library of 2726 FDA Phase 1-approved drugs, identifying promising anti-PGCC candidates, including proteasome inhibitors, FOXM1, CHK, and macrocyclic lactones. Notably, RNA-Seq analysis of cells treated with the macrocyclic lactone Pyronaridine revealed AXL inhibition as a potential strategy for targeting PGCCs. Although our single-cell morphological analysis pipeline is powerful, empirical testing of all existing compounds is impractical and inefficient. To overcome this limitation, we trained a machine learning model to predict anti-PGCC efficacy in silico, integrating chemical fingerprints and compound descriptions from prior publications and databases. The model demonstrated a high correlation with experimental outcomes and predicted efficacious compounds in an expanded library of over 6,000 drugs. Among the top-ranked predictions, we experimentally validated five compounds as potent PGCC inhibitors using cell lines and patient-derived models. These findings underscore the synergistic potential of integrating high-throughput empirical screening with machine learning-based virtual screening to accelerate the discovery of novel therapies, particularly for targeting therapy-resistant PGCCs in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0482d6424427dcdf25109ef995747da0a47de141" target='_blank'>
              High-Throughput Empirical and Virtual Screening To Discover Novel Inhibitors of Polyploid Giant Cancer Cells in Breast Cancer
              </a>
            </td>
          <td>
            Yushu Ma, Chien-Hung Shih, Jinxiong Cheng, Hsiao-Chun Chen, Li-Ju Wang, Yanhao Tan, Yuan Zhang, Daniel D Brown, S. Oesterreich, Adrian V Lee, Yu-Chiao Chiu, Yu-Chih Chen
          </td>
          <td>2025-03-04</td>
          <td>Analytical Chemistry</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c98c6553903c4874ca18db08ae52cca732c52fe8" target='_blank'>
              Ivosidenib Confers BRCAness Phenotype and Synthetic Lethality to Poly (ADP-Ribose) Polymerase Inhibition in BRCA1/2-Proficient Cancer Cells
              </a>
            </td>
          <td>
            Danyang Zhou, Wei Liu, Yanyan Zhang, Chong Li
          </td>
          <td>2025-04-14</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="FOXP3+ natural regulatory T cells (nTregs) promote resolution of inflammation and repair of epithelial damage following viral pneumonia-induced lung injury, thus representing a cellular therapy for patients with acute respiratory distress syndrome (ARDS). Whether in vitro induced Tregs (iTregs), which can be rapidly generated in substantial numbers from conventional T cells, also promote lung recovery is unknown. nTregs require specific DNA methylation patterns maintained by the epigenetic regulator, ubiquitin-like with PHD and RING finger domains 1 (UHRF1). Here, we tested whether iTregs promote recovery following viral pneumonia and whether iTregs require UHRF1 for their pro-recovery function. We found that adoptive transfer of iTregs to mice with influenza virus pneumonia promotes lung recovery and that loss of UHRF1-mediated maintenance DNA methylation in iTregs leads to reduced engraftment and a delayed repair response. Transcriptional and DNA methylation profiling of adoptively transferred UHRF1-deficient iTregs that had trafficked to influenza-injured lungs demonstrated transcriptional instability with gain of effector T cell lineage-defining transcription factors. Strategies to promote the stability of iTregs could be leveraged to further augment their pro-recovery function during viral pneumonia and other causes of ARDS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8259d3d12170176131ead01b0d7705b88e45403" target='_blank'>
              Maintenance DNA methylation is required for induced regulatory T cell reparative function following viral pneumonia
              </a>
            </td>
          <td>
            A. Joudi, J. Gurkan, Qianli Liu, Manuel A. Torres Acosta, K. Helmin, Luisa Morales-Nebreda, N. Mambetsariev, Carla P. Reyes Flores, H. Abdala-Valencia, Elizabeth M. Steinert, S. Weinberg, Benjamin D. Singer
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Objective This study examines the cellular heterogeneity of epithelial cells within pancreatic ductal adenocarcinoma (PDAC) and their contributions to tumor progression, metastasis, and immunosuppressive interactions using single-cell RNA sequencing. Methods Single-cell RNA-sequencing data from two datasets (GSE154778 and GSE158356) were integrated using the Harmony algorithm, followed by quality control, clustering, and differential gene expression analysis. Distinct subpopulations of epithelial cells were identified, and their gene expression profiles were analyzed. To assess the malignancy of these subpopulations, single-cell copy number variation (CNV) analysis and trajectory analysis were conducted. Additionally, intercellular communication was examined using the CellChat platform. Results The analysis revealed pronounced heterogeneity among PDAC epithelial cells, with specific subpopulations exhibiting distinct roles in tumor proliferation, extracellular matrix remodeling, and metastatic dissemination. Subpopulations 4 and 6 were characterized by increased CNV levels and a more malignant phenotype, suggesting an enhanced capacity for metastasis. Single-cell trajectory analysis, along with CellChat, mapped the temporal evolution of epithelial cells, identifying key regulatory genes such as DCBLD2 and JUN. A prognostic model incorporating five key genes, including KLF6, was developed and demonstrated strong predictive accuracy for patient outcomes. Notably, KLF6 emerged as a critical prognostic marker associated with immune modulation, particularly through interactions with M2 macrophages. Conclusion The study highlights the pronounced heterogeneity of epithelial cells in PDAC and their distinct contributions to tumor progression, metastasis, and immune modulation. Through single-cell transcriptomic and CNV analyses, we identified epithelial subpopulations with varying malignant potentials and distinct interactions with the tumor microenvironment. Among these, KLF6 emerged as a key regulator associated with immune modulation and metastasis. Our findings emphasize the significance of epithelial cell heterogeneity in shaping pancreatic cancer progression. These insights provide a foundation for future investigations into novel prognostic markers and therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12957-025-03793-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c02e096347df4014f9175b24f19fbd3cae4aa9d0" target='_blank'>
              Deciphering the heterogeneity of epithelial cells in pancreatic ductal adenocarcinoma: implications for metastasis and immune evasion
              </a>
            </td>
          <td>
            Jie Liu, Hui Li, Xiuyun Lin, Jiani Xiong, Guangfeng Wu, Lingyan Ding, Bin Lin
          </td>
          <td>2025-04-16</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b63e3c72612fc0a154d330c02ca702342a62c1e" target='_blank'>
              Novel Approach to Overcome Osimertinib Resistance Using Bromodomain and Extra-Terminal Domain Inhibitors.
              </a>
            </td>
          <td>
            Yosuke Miyashita, Ken Tajima, Kenta Izumi, N. Matsumoto, D. Hayakawa, Ikuko Nakamura, Isana Katayama, Adityo Wibowo, Hironari Matsuda, Wira Winardi, B. R. Amien, Y. Mitsuishi, Fumiyuki Takahashi, Kohta Nakamura, K. Uchibori, N. Yanagitani, Takuo Hayashi, K. Takamochi, Kenji Suzuki, Ryohei Katayama, Kazuhisa Takahashi
          </td>
          <td>2025-03-04</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="eccDNA is a driver of many cancers and a potential intermediate in other age-related disorders. However, little is known about the mechanisms underlying eccDNA formation in healthy tissue and how aging affects these processes. Here, we present an atlas of eccDNA across seven tissues of male mice spanning four ages. EccDNA correlates with open chromatin characterized by signatures of H3K27ac and H3K4me1. Additionally, the mutational load of eccDNA on genes correlates with tissue-specific transcription and increases logarithmically as a function of transcript level. Still, a population of intron-dense genes with many splice forms remains sheltered from eccDNA formation. We also find that the total number of eccDNA molecules does not increase as mice age, unlike other types of mutations. Our data reveal a link between eccDNA formation and transcript level that may drive gene architecture in mammals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1bc42a12825a4c11c2ddcf3582debb617c1482e" target='_blank'>
              EccDNA atlas in male mice reveals features protecting genes against transcription-induced eccDNA formation
              </a>
            </td>
          <td>
            Xue Liang, Gerard Arrey, Yating Qin, Lucía Álvarez-González, J. M. Hariprakash, Jie Ma, Sylvester Holt, Peng Han, Yonglun Luo, Hanbo Li, Aurora Ruiz-Herrera, Henriette Pilegaard, Birgitte Regenberg
          </td>
          <td>2025-02-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Recent research has shifted the focus from the genetic code of DNA to its structural variations, which significantly impact cancer, genetic diseases, and gene regulation. Structural changes, such as the transition from B-DNA to A-DNA, influence DNA stability and flexibility and are affected by factors like DNA methylation and sugar puckering. This study is the first to investigate the relationship between DNA conformational changes and lifespan in two rodent species. The analysis focused on long-lived Nannospalax xanthodon and shorter-lived Rattus rattus, utilizing infrared spectroscopy and principal component analysis (PCA) to examine liver DNA. Results indicated that transition from B-form to A- and Z-forms were more prevalent in N. xanthodon than in R. rattus. However, the dominant DNA conformations in both species are in B-form. Additionally, N-type sugar puckers (C3-endo conformation), associated with these DNA forms, were more prominent in N. xanthodon. In contrast, S-type sugar puckers (C2-endo conformation), characteristic of B-DNA, were found at lower levels in N. xanthodon. Furthermore, the variations in methylation-specific structural modifications of nucleobases were quantitatively assessed among these species. The study proposes a significant connection between the long lifespan of N. xanthodon, which live underground, and their unique DNA structure, offering insights into how different DNA forms, as well as the conformations of their backbone and sugar-base components, may affect longevity, highlighting potential research avenues regarding the biomolecular aspects of aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90e194ada8d97ff0f6ac7375a7dc3c14a580e955" target='_blank'>
              Structural features of DNA and their potential contribution to blind mole rat (Nannospalax xanthodon) longevity
              </a>
            </td>
          <td>
            Gamzenur Sönmez, Tuba Yağcı Gurbanov
          </td>
          <td>2025-03-25</td>
          <td>Biogerontology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) poses a formidable clinical challenge due to multidrug resistance (MDR) caused by tumor heterogeneity. To elucidate the intricate mechanisms underlying HGSOC heterogeneity, we conducted a comprehensive analysis of five single-cell transcriptomes and eight spatial transcriptomes derived from eight HGSOC patients. This study provides a comprehensive view of tumor heterogeneity across the spectrum of gene expression, copy number variation (CNV), and single-cell profiles. Our CNV analysis revealed intratumor heterogeneity by identifying distinct tumor clones, illuminating their evolutionary trajectories and spatial relationships. We further explored the homogeneity and heterogeneity of CNV across tumors to pinpoint the origin of heterogeneity. At the cellular level, single-cell RNA sequencing (scRNA seq) analysis identified three meta-programs that delineate the functional profile of tumor cells. The communication networks between tumor cell clusters exhibited unique patterns associated with the meta-programs governing these clusters. Notably, the ligand-receptor pair MDK - NCL emerged as a highly enriched interaction in tumor cell communication. To probe the functional significance of this interaction, we induced NCL overexpression in the SOVK3 cell line and observed enhanced tumor cell proliferation. These findings indicate that the MDK - NCL interaction plays a crucial role in promoting HGSOC tumor growth and may represent a promising therapeutic target. In conclusion, this study comprehensively unravels the multifaceted nature of HGSOC heterogeneity, providing potential therapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/375bc93ef6f0dd0954afbd4a3ef7ab6a5339dc00" target='_blank'>
              Deciphering ovarian cancer heterogeneity through spatial transcriptomics, single-cell profiling, and copy number variations
              </a>
            </td>
          <td>
            Songyun Li, Zhuo Wang, Hsien-da Huang
          </td>
          <td>2025-03-04</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff0e4ce978bcbd93656c4cbcf8bd876d6363d193" target='_blank'>
              Analysis of the role of cellular autosomal genetic banding position subtyping on clinical prognosis and immune infiltration in colorectal cancer
              </a>
            </td>
          <td>
            Jiansong Wang, Xin Mao
          </td>
          <td>2025-03-27</td>
          <td>Advances in Engineering Technology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9782e1bf6046bb6e08da1b33f9dcb44ab84e474" target='_blank'>
              A pivotal contribution of proteostasis failure and mitochondrial dysfunction to chromosomal instability-induced microcephaly
              </a>
            </td>
          <td>
            Amanda González-Blanco, Adrián Acuña-Higaki, David Boettger, Marco Milán
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The ribosomal DNA (rDNA) plays a vital role in regulating protein synthesis by ribosome biogenesis, essential for maintaining cellular growth, metabolism, and more. Cancer cells show a high dependence on ribosome biogenesis and exhibit elevated rDNA transcriptional activity. CX-5461, also known as Pidnarulex, is a First-in-Class anticancer drug that has received 'Fast Track Designation' approval from the FDA. Initially reported to inhibit Pol I-driven rDNA transcription, CX-5461 was recently identified as a G-quadruplex structure (G4) stabilizer and is currently completed or undergoing multiple Phase I clinical trials in patients with breast and ovarian cancers harboring BRCA1/2, PALB2, or other DNA repair deficiencies. Additionally, preclinical studies have confirmed that CX-5461 demonstrates promising therapeutic effects against multifarious non-cancer diseases, including viral infections, and autoimmune diseases. This review summarizes the mechanisms of CX-5461, including its transcriptional inhibition of rDNA, binding to G4, and toxicity towards topoisomerase, along with its research status and therapeutic effects across various diseases. Lastly, this review highlights the targeted therapy strategy of CX-5461 based on nanomedicine delivery, particularly the drug delivery utilizing the nucleic acid aptamer AS1411, which contains a G4 motif to specifically target the highly expressed nucleolin on the surface of tumor cell membranes; It also anticipates the strategy of coupling CX-5461 with peptide nucleic acids and locked nucleic acids to achieve dual targeting, thereby realizing individualized G4-targeting by CX-5461. This review aims to provide a general overview of the progress of CX-5461 in recent years and suggest potential strategies for disease treatment involving ribosomal RNA synthesis, G4, and topoisomerase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38252b99654ce0eee22dc4caf1a279f3bd484a1a" target='_blank'>
              The G-quadruplex ligand CX-5461: an innovative candidate for disease treatment
              </a>
            </td>
          <td>
            Hong-Xia Li, , Jing Fei, Man Guo, Chen Zeng, Pi-Jun Yan, Yong Xu, Gang Qin, Fang-Yuan Teng
          </td>
          <td>2025-04-18</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma (GBM) is a common and highly lethal type of primary brain tumor in adults. Therapeutic failure is partly attributed to a fraction of Glioblastoma Stem Cells (GSCs) that show high levels of heterogeneity and plasticity. GSCs exist in a transcriptional gradient between two states: Developmental (Dev) and Injury Response (IR) in which IR-GSCs exhibit more invasive behaviors. While previous studies have identified fitness genes in GSCs, the genes required to establish and maintain the Dev and IR states remain poorly defined. To identify the regulators of the IR GSC state, we performed a phenotypic genome-wide CRISPR-Cas9 knockout (KO) screen in patient-derived GSCs based on cell surface expression of the IR marker CD44. Notably, we found that perturbations of the histone acetyltransferase EP300 in IR GSCs led to decreased CD44 cell surface expression, significant downregulation of gene expression signatures associated with the IR state, and to decreased self-renewal and invasion. Furthermore, genetic targeting of Ep300 in a mouse GBM model delayed tumor initiation and/or progression. Collectively, our results establish EP300 as a regulator of the IR state in GSCs and provide a mechanistic basis for its therapeutic targeting in GBM. Significance A genome-wide phenotypic CRISPR-Cas9 screen in a patient-derived Glioblastoma Stem Cell line identified the genes required to maintain the Injury-Response cellular state, with a focus on the histone acetyl transferase gene EP300. This study suggests how therapeutic targeting of cellular state could reduce the aggressiveness of GBM tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72194daa3f94b0757cee54e56b558726a1fc4778" target='_blank'>
              Identification of Injury-Response State Determinants in Glioblastoma Stem Cells
              </a>
            </td>
          <td>
            Fatemeh Molaei, G. Macleod, Shahan Haider, Aleczandria S. Tiffany, Frank M Oteng, Jacob M. Berman, Arianna Skirzynska, Molly S. Shoichet, Daniel Schramek, Peter B. Dirks, Stéphane Angers
          </td>
          <td>2025-04-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78eb760dac971fc0c9d9c921e3630cb4941aa8c4" target='_blank'>
              Tumor-colonizing Pseudoalteromonas elyakovii metabolically reprograms the tumor microenvironment and promotes breast ductal carcinoma.
              </a>
            </td>
          <td>
            Shuyan Liu, Youpeng Pan, Chaopeng Zheng, Qinghui Zheng, Yaoqiang Du, Yajuan Zheng, Hongchao Tang, Xiaozhen Liu, Jiancheng Mou, Xin Zeng, Zhuotao Yang, Wenjuan Gui, Yuning Tang, Mingxing Xu, Zhihao Ye, Haotian Su, Qiuran Xu, Xuli Meng
          </td>
          <td>2025-04-07</td>
          <td>mBio</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Background Clinical evidence has established anti-PD-1 antibody as a transformative treatment modality for relapsed/refractory peripheral T-cell lymphoma (r/r PTCL), yet reveals a therapeutic plateau with drug resistance observed in 60% of r/r PTCL. The biological determinants underlying this resistance—particularly the complex interplay between tumor-intrinsic characteristics (including tumor mutation burden and oncogenic mutations) and immune microenvironment features (notably PD-L1 expression)—remain insufficiently illustrated. Therefore, we systematically depicted the comprehensive mutation profile of r/r PTCL patients and correlated them with the efficacy and prognosis of anti-PD-1 therapy. Methods Here, we enrolled a cohort of 109 patients with r/r PTCL and performed targeted next-generation sequencing of 440 cancer-associated genes. Clinical information was collected and correlated with genetic mutations. We constructed JAK3 mutant models using Jurkat and BA/F3 cell lines. We performed single-cell transcriptomics, western blotting, and flow cytometry to elucidate the molecular mechanism. Additionally, we built a JAK3-mutant syngeneic mouse model to demonstrate in vivo antitumor efficacy of Tofacitinib and anti-PD-1 therapy. Results We identified and validated that PD-L1 was a predictor for the efficacy of anti-PD-1 therapy in PTCL patients. The subset of PTCL patients (13.5%) characterized by enrichment of the APOBEC-related mutation signature had worse overall survival (p=0.031) compared with non-APOBEC-enriched samples. JAK3 and EZH2 mutations were associated with lower PD-L1 expression (p<0.05), and JAK3 mutations were independently correlated with shorter progression-free survival (HR=6.07, p=0.0144). Among all types of JAK3 mutations, single-cell transcriptomics, western blotting, and flow cytometry revealed that JAK3 p.A573V and p.M511I mutations led to decreased PD-L1 expression in Jurkat and BA/F3 cell lines through inactivation of STAT3. Compared with JAK3 wild-type syngeneic mouse models, JAK3 p.A573V and p.M511I mutant mice were more sensitive to Tofacitinib but not anti-PD-1 antibody. Conclusions In conclusion, we found that JAK3 mutations, especially JAK3 p.A573V and JAK3 p.M511I mutations, lead to poor prognosis of anti-PD-1 therapy through the STAT3/PD-L1 pathway. Tofacitinib is more suitable than anti-PD-1 antibody for JAK3 mutant PTCL patients. Trial registration number NCT03502629.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/833d9e02b6293d47526d23308ad227c791c9d787" target='_blank'>
              JAK3A573V and JAK3M511I mutations in peripheral T-cell lymphoma mediating resistance to anti-PD-1 therapy through the STAT3/PD-L1 pathway
              </a>
            </td>
          <td>
            N. Lou, Mengwei Yang, Zucheng Xie, Ruyun Gao, Lei Zhang, Le Tang, Jiarui Yao, Xiaohong Han, Yuankai Shi
          </td>
          <td>2025-04-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="ABSTRACT Introduction Gastric cancer (GC) remains among the top five global health problems. Therefore, comprehending the tumor energetic behavior is critical to understanding its progression. This study aimed to investigate mitochondrial DNA (mtDNA) alterations in GC cancer cell lines in an animal model. Material and Methods Four mitochondrial genes (COI, ATP8, ND1, and ND3) were analyzed in GC (AGP01, ACP02, ACP03, and PG100) and control (Walker 256 carcinosarcoma) cell lines inoculated in Sapajus apella , exposed and not exposed to N‐methyl‐N‐nitrosourea. Results Two synonymous alterations were identified in ND1. In ND3, a non‐synonymous alteration (A10398G ➔ Thr114Ala) may decrease the respiratory chain Complex I efficiency, enhancing cellular reactive oxygen species and contributing to mtDNA damage. As alterations in ND1 and ND3 were observed in highly aggressive cell lines, our results suggest these genes may play crucial roles in energetic efficiency and gastric carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c47f24a7df1aebc32a5ada6b198ff00992e9443d" target='_blank'>
              Mitochondrial Complex I Molecular Alterations in Sapajus apella as a Human Gastric Carcinogenesis Model After MNU Exposure
              </a>
            </td>
          <td>
            Bárbara dos Santos Dias, Symara Rodrigues Antunes, Danilo do Rosário Pinheiro, R. Burbano, Bárbara do Nascimento Borges
          </td>
          <td>2025-04-01</td>
          <td>Journal of Medical Primatology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Maintaining genomic integrity and faithful transmission of genetic information is essential for the survival and proliferation of cells and organisms. DNA damage, which threatens the integrity of the genome, is rapidly sensed and repaired by mechanisms collectively known as the DNA damage response. The RNA demethylase FTO has been implicated in this process; however, the underlying mechanism by which FTO regulates DNA repair remains unclear. Here, we use an unbiased quantitative proteomic approach to identify the proximal interactome of endogenous FTO protein. Our results demonstrate a direct interaction with the DNA damage sensor protein PARP1, which dissociates upon ultraviolet stimulation. FTO inhibits PARP1 catalytic activity and controls its clustering in the nucleolus. Loss of FTO enhances PARP1 enzymatic activity and the rate of PARP1 recruitment to DNA damage sites, accelerating DNA repair and promoting cell survival. Interestingly, FTO regulates PARP1 function and DNA damage response independent of its catalytic activity. We conclude that FTO is an endogenous negative regulator of PARP1 and the DNA damage response in cells beyond its role as an RNA demethylase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0cd587329b1ccdece8791396b9b3d5b2f506b3e" target='_blank'>
              FTO suppresses DNA repair by inhibiting PARP1
              </a>
            </td>
          <td>
            Tianyi Zhu, Jing Zhi Anson Tan, Lingrui Zhang, He Huang, Sooraj S Das, F. Cheng, P. Padmanabhan, Mathew J K Jones, Mihwa Lee, Albert Lee, Jocelyn Widagdo, Victor Anggono
          </td>
          <td>2025-03-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="ABSTRACT The transcription factor p53 is the most frequently impaired tumor suppressor in human cancers. In response to various stress stimuli, p53 activates transcription of genes that mediate its tumor-suppressive functions. Distinctive characteristics of p53 outlined here enable a well-defined program of genes involved in cell cycle arrest, apoptosis, senescence, differentiation, metabolism, autophagy, DNA repair, anti-viral response, and anti-metastatic functions, as well as facilitating autoregulation within the p53 network. This versatile, anti-cancer network governed chiefly by a single protein represents an immense opportunity for targeted cancer treatment, since about half of human tumors retain unmutated p53. During the last two decades, numerous compounds have been developed to block the interaction of p53 with the main negative regulator MDM2. However, small molecule inhibitors of MDM2 only induce a therapeutically desirable apoptotic response in a limited number of cancer types. Moreover, clinical trials of the MDM2 inhibitors as monotherapies have not met expectations and have revealed hematological toxicity as a characteristic adverse effect across this drug class. Currently, combination treatments are the leading strategy for enhancing efficacy and reducing adverse effects of MDM2 inhibitors. This review summarizes efforts to identify and test therapeutics that work synergistically with MDM2 inhibitors. Two main types of drugs have emerged among compounds used in the following combination treatments: first, modulators of the p53-regulated transcriptome (including chromatin modifiers), translatome, and proteome, and second, drugs targeting the downstream pathways such as apoptosis, cell cycle arrest, DNA repair, metabolic stress response, immune response, ferroptosis, and growth factor signaling. Here, we review the current literature in this field, while also highlighting overarching principles that could guide target selection in future combination treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3895fe8ca5319763195e6c375cf525ba1915bfa" target='_blank'>
              Harnessing p53 for targeted cancer therapy: new advances and future directions
              </a>
            </td>
          <td>
            Z. Andrysik, Joaquin M Espinosa
          </td>
          <td>2025-03-03</td>
          <td>Transcription</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="For nearly three decades, interspecies somatic cell nuclear transfer (iSCNT) has been explored as a potential tool for cloning, regenerative medicine, and wildlife conservation. However, developmental inefficiencies remain a major challenge, largely due to persistent barriers in nucleocytoplasmic transport, mitonuclear communication, and epigenome crosstalk. This review synthesized peer-reviewed English articles from PubMed, Web of Science, and Scopus, spanning nearly three decades, using relevant keywords to explore the molecular mechanisms underlying iSCNT inefficiencies and potential improvement strategies. We highlight recent findings deepening the understanding of interspecies barriers in iSCNT, emphasizing their interconnected complexities, including the following: (1) nucleocytoplasmic incompatibility may disrupt nuclear pore complex (NPC) assembly and maturation, impairing the nuclear transport of essential transcription factors (TFs), embryonic genome activation (EGA), and nuclear reprogramming; (2) mitonuclear incompatibility could lead to nuclear and mitochondrial DNA (nDNA-mtDNA) mismatches, affecting electron transport chain (ETC) assembly, oxidative phosphorylation, and energy metabolism; (3) these interrelated incompatibilities can further influence epigenetic regulation, potentially leading to incomplete epigenetic reprogramming in iSCNT embryos. Addressing these challenges requires a multifaceted, species-specific approach that balances multiple incompatibilities rather than isolating a single factor. Gaining insight into the molecular interactions between the donor nucleus and recipient cytoplast, coupled with optimizing strategies tailored to specific pairings, could significantly enhance iSCNT efficiency, ultimately transforming experimental breakthroughs into real-world applications in reproductive biotechnology, regenerative medicine, and species conservation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2095defd86819f42b7aa13e7b28c1855e90cdf2a" target='_blank'>
              The Complexities of Interspecies Somatic Cell Nuclear Transfer: From Biological and Molecular Insights to Future Perspectives
              </a>
            </td>
          <td>
            Peachanika Pankammoon, Marvin Bryan Segundo Salinas, C. Thitaram, A. Sathanawongs
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/996e0da402cc341f0ff28812c9362a9660b3a867" target='_blank'>
              Abstract A026: Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens
              </a>
            </td>
          <td>
            Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole T. Roybal, B. Shy, Alexander Marson
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4421024c849016285df42c8f390ac35abcedd51e" target='_blank'>
              Pan-cancer spatial characterization of key immune biomarkers in the tumor microenvironment
              </a>
            </td>
          <td>
            James R. Lindsay, Jennifer Altreuter, J. Alessi, Jason L. Weirather, Anita Giobbie-Hurder, Ian D Dryg, Katharina V. Hoebel, B. Sharma, K. Felt, F. S. Hodi, N. Lindeman, L. Sholl, E. Cerami, Jonathan A. Nowak, Mark M Awad, Scott J Rodig, William E. Lotter
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Inflammation is a multifaceted defense mechanism of the immune system against infection. Chronic inflammation is intricately linked to all stages of tumorigenesis and is therefore associated with an elevated risk of developing serious cancers. Epigenetic mechanisms have the capacity to trigger inflammation as well as facilitate tumor development and transformation within an inflammatory context. They achieve this by dynamically modulating the expression of both pro—inflammatory and anti—inflammatory cytokines, which in turn sustains chronic inflammation. The aberrant epigenetic landscape reconfigures the transcriptional programs of inflammatory and oncogenic genes. This reconfiguration is pivotal in dictating the biological functions of both tumor cells and immune cells. Aberrant histone H3 lysine 27 site (H3K27) methylation has been shown to be involved in biological behaviors such as inflammation development, tumor progression, and immune response. The establishment and maintenance of this repressive epigenetic mark is dependent on the involvement of the responsible histone modifying enzymes enhancer of zeste homologue 2 (EZH2), jumonji domain containing 3 (JMJD3) and ubiquitously transcribed tetratricopeptide repeat gene X (UTX) as well as multiple cofactors. In addition, specific pharmacological agents have been shown to modulate H3K27 methylation levels, thereby modulating inflammation and carcinogenesis. This review comprehensively summarises the current characteristics and clinical significance of epigenetic regulation of H3K27 methylation in the context of inflammatory response and tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a333da9e2131a9a75deaa130f4cc548713bf46a8" target='_blank'>
              Regulation of histone H3K27 methylation in inflammation and cancer
              </a>
            </td>
          <td>
            Jing Ma, Yalin Zhang, Jingyuan Li, Y. Dang, Dan Hu
          </td>
          <td>2025-03-05</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5e0309eee74a6c6b459d5c9b0da675c6d37451c" target='_blank'>
              EXO1 as a potential biomarker for prognosis, immune infiltration, and immunotherapy in pan-cancer analysis.
              </a>
            </td>
          <td>
            Jikang Wang, Fubo Wang, Haoxuan Huang
          </td>
          <td>2025-04-11</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Gastric cancer is a highly aggressive malignancy characterized by a complex tumor microenvironment (TME). Cancer-associated fibroblasts (CAFs), which are a key component of the TME, exhibit significant heterogeneity and play crucial roles in tumor progression. Therefore, a comprehensive understanding of CAFs is essential for developing novel therapeutic strategies for gastric cancer. Methods This study investigates the characteristics and functional information of CAF subtypes and explores the intercellular communication between CAFs and malignant epithelial cells (ECs) in gastric cancer by analyzing single-cell sequencing data from 24 gastric cancer samples. CellChat was employed to map intercellular communication, and Seurat was used to integrate single-cell sequencing data with spatial transcriptome data to reconstruct a comprehensive single-cell spatial map. The spatial relationship between apCAFs and cancer cells was analyzed using multicolor immunohistochemistry. Results Cells were categorized into nine distinct categories, revealing a positive correlation between the proportions of epithelial cells (ECs) and fibroblasts. Furthermore, six fibroblast subpopulations were identified: inflammatory (iCAFs), pericytes, matrix (mCAFs), antigen-presenting (apCAFs), smooth muscle cells (SMCs), and proliferative CAFs (pCAFs). Each of these subpopulations was linked to various biological processes and immune responses. Malignant ECs exhibited heightened intercellular communication, particularly with CAF subpopulations, through specific ligand-receptor interactions. High-density regions of CAF subpopulations displayed spatial exclusivity, with pericytes serving as a source for iCAFs, mCAFs, and apCAFs. Notably, malignant ECs and apCAFs showed increased interactions, with certain ligand-receptor pairs potentially impacting the prognosis of gastric cancer. Multiplex immunohistochemistry (mIHC) confirmed the close spatial proximity of apCAFs to cancer cells in gastric cancer. Conclusion Our study provided a comprehensive characterization of CAF heterogeneity in gastric cancer and revealed the intricate intercellular networks within the TME. The identified CAF subpopulations and their interactions with malignant cells could serve as potential therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06376-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24163c4c446c34923a67d83b5c03139aa6a3cb4" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics reveal the heterogeneity and intercellular communication of cancer-associated fibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Xijie Zhang, Bo Ren, Bo Liu, Rui Wang, Sen Li, Yuzhou Zhao, Wence Zhou
          </td>
          <td>2025-03-18</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6e0f5dce3feb58cd819120465c413a2814ac1e2" target='_blank'>
              Path2Omics: Enhanced transcriptomic and methylation prediction accuracy from tumor histopathology
              </a>
            </td>
          <td>
            Danh-Tai Hoang, E. Shulman, S. R. Dhruba, N. Nair, R. K. Barman, H. Lalchungnunga, Omkar Singh, MacLean P Nasrallah, Eric A Stone, Kenneth D Aldape, E. Ruppin
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Colorectal cancer (CRC) is responsible for over 900,000 annual deaths worldwide. Emerging evidence supports pro‐carcinogenic bacteria in the colonic microbiome are at least promotional in CRC development and may be causal. We previously showed toxigenic C. difficile from human CRC‐associated bacterial biofilms accelerates tumorigenesis in Apc Min/+ mice, both in specific pathogen‐free mice and in gnotobiotic mice colonized with a defined consortium of bacteria. To further understand host–microbe interactions during colonic tumorigenesis, we combined single‐cell RNA‐sequencing (scRNA‐seq), spatial transcriptomics, and immunofluorescence to define the molecular spatial organization of colonic dysplasia in our consortium model with or without C. difficile. Our data show a striking bipartite regulation of Deleted in Malignant Brain Tumors 1 (DMBT1) in the inflamed versus dysplastic colon. From scRNA‐seq, differential gene expression analysis of normal absorptive colonocytes at 2 weeks postinoculation showed DMBT1 upregulated by C. difficile compared to colonocytes from mice without C. difficile exposure. In contrast, our spatial transcriptomic analysis showed DMBT1 dramatically downregulated in dysplastic foci compared with normal‐adjacent tissue. We further integrated our datasets to generate custom colonic dysplasia scores and ligand‐receptor mapping. Validation with immunofluorescence showed DMBT1 protein downregulated in dysplastic foci from three mouse models of colonic tumorigenesis and in adenomatous dysplasia from human samples. Finally, we used mouse and human organoids to implicate WNT signaling in the downregulation of DMBT1 mRNA and protein. Together, our data reveal cell type‐specific regulation of DMBT1, a potential mechanistic link between bacteria and colonic tumorigenesis. Published 2025. This article is a U.S. Government work and is in the public domain in the USA. The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7def78e585a3b57bba506b3445109c092c108f8" target='_blank'>
              Multiomic spatial atlas shows deleted in malignant brain tumors 1 (DMBT1) glycoprotein is lost in colonic dysplasia
              </a>
            </td>
          <td>
            Emily H Green, Subhag R. Kotrannavar, Megan E. Rutherford, Hannah M. Lunnemann, Harsimran Kaur, Cody N. Heiser, Hua Ding, A. Simmons, Xiao Liu, Borden Lacy, Kay Washington, M. Shrubsole, Qi Liu, Ken S. Lau, Cynthia L Sears, Robert J. Coffey, Julia L. Drewes, Nicholas O Markham
          </td>
          <td>2025-03-03</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2daedc5d36f880d0e2fc581751bb206d1ca1d20" target='_blank'>
              Recurrent spontaneous miscarriages from sperm after ABVD chemotherapy in a patient with Hodgkin's lymphoma: sperm DNA and methylation profiling.
              </a>
            </td>
          <td>
            G. Lecuyer, A. Rolland, A. Neyroud, Bertrand Evrard, Nathan Alary, Clemence Genthon, Nathalie Dejucq-Rainsford, Célia Ravel, Jessika Moreau, N. Moinard, Mohamed Hadi Mohamed Abdelhamid, Christophe Klopp, Louis Bujan, Frédéric Chalmel
          </td>
          <td>2025-04-15</td>
          <td>Asian journal of andrology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ffb5861f3091086b8e6d39be2fc47951f44015b" target='_blank'>
              Cohesins: Crossroad Between Cornelia de Lange Spectrum and Cancer Predisposition.
              </a>
            </td>
          <td>
            Laura Rigotti, Stefano Rebellato, Antonella Lettieri, Silvia Castiglioni, M. Mariani, Simona Totaro, C. Saitta, C. Gervasini, G. Fazio, Valentina Massa, Giovanni Cazzaniga, Angelo Selicorni
          </td>
          <td>2025-04-05</td>
          <td>American journal of medical genetics. Part A</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ce363baca9236ffeeafdea9d54bda339177ab7e" target='_blank'>
              Structural Basis for DNA Replication and Uracil Repair in Phage A-Family DNA Polymerases
              </a>
            </td>
          <td>
            Sophia Missoury, Soizick Lucas-Staat, R. Sieskind, Marc Delarue
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Dysbiosis of intratumoral fungal and bacterial communities is associated with poor prognosis in various cancers. However, the mycobiome characteristics in laryngeal squamous cell carcinoma (LSCC) and its correlation with recurrence have not yet been investigated. The mycobiome in 80 LSCC samples was characterized using internal transcribed spacer sequencing, encompassing both tumor tissues and their matched para-cancerous tissues. The intratumoral bacteriome was further identified using 16S rRNA sequencing. These two microbial communities were analyzed using bioinformatics and statistical methods to determine its potential correlation with LSCC recurrence. The fungal alpha diversity in tumors was higher compared with that in para-cancerous tissues (P < 0.001). A significant difference in the overall fungal community patterns between tumor tissues and para-cancerous tissues was observed based on Bray-Curtis dissimilarity (P < 0.001). The presence of Alloprevotella, Porphyromonas, Candida, and Fusarium in tumors exhibited a correlation with alcohol consumption. The relative abundance of Penicillium, Exophiala, and Aspergillus in the mycobiome, as well as that of Alloprevotella, Porphyromonas, and Peptostreptococcus in the bacteriome significantly increased the risk of LSCC recurrence (P < 0.05). These six microorganisms can combine to form a microbial complex, which may independently contribute to recurrence risk in patients with LSCC when enriched within the tumor (hazard ratio = 6.844, P < 0.01). Intratumoral fungi and bacteria can be valuable indicators for assessing recurrence in patients with LSCC, indicating their potential as valuable targets for therapeutic intervention. IMPORTANCE Our results revealed that dysbiosis of intratumoral microbiota, including increased fungal community diversity and overgrowth of several fungal or bacterial organisms, is substantially linked to the recurrence of LSCC. Drinking habits might alter the laryngeal microbiota to influence the recurrence of LSCC. We also explored a method to potentially predict the recurrence of LSCC from a novel perspective. These findings could offer insights into the etiology of LSCC and pave way to prevent and treat LSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfc011a9eb62b9ac278bb5cc6ef5e19d7054ea20" target='_blank'>
              Complex of intratumoral mycobiome and bacteriome predicts the recurrence of laryngeal squamous cell carcinoma
              </a>
            </td>
          <td>
            Xinhui Mao, Hui-ying Huang, Limin Zhao, Feiran Li, Zhenwei Wang, Xiaohui Yuan, Hui-Ching Lau, Chi-Yao Hsueh, Ming Zhang
          </td>
          <td>2025-02-21</td>
          <td>Applied and Environmental Microbiology</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="Transposon is the main component of the eukaryotic genome, and more and more plant genome data show that transposons are diverse in regulating genome structure, variation, function and evolution, with different transposition mechanisms in the genome. Hybridization and polyploidy play an important role in promoting plant speciation and evolution, and recent studies have shown that polyploidy is usually accompanied by the expansion of transposons, which affect the genome size and structure of polyploid plants. Transposons can insert into genes and intergenic regions, resulting in great differences in the overall genome structure of closely related plant species, and it can also capture gene segments in the genome to increase the copy number of genes. In addition, transposons influence the epigenetic modification state of the genome and regulate the expression of the gene, while plant phenotype, biological and abiotic stress response are also regulated by transposons. Overall, transposons play an important role in the plant genome, especially polyploid plant genome, adaptation and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ba6bc45c010534e8127dc7401c27891c7b84e09" target='_blank'>
              Understanding the Regulation Activities of Transposons in Driving the Variation and Evolution of Polyploid Plant Genome
              </a>
            </td>
          <td>
            Yafang Xiao, Jianbo Wang
          </td>
          <td>2025-04-08</td>
          <td>Plants</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42b078825f058bd38a50c194336988df8e0a2a93" target='_blank'>
              Enhancer hijacking discovery in acute myeloid leukemia by pyjacker identifies MNX1 activation via deletion 7q.
              </a>
            </td>
          <td>
            Etienne Sollier, Anna Riedel, U. Toprak, Justyna A. Wierzbinska, D. Weichenhan, J. P. Schmid, M. Hakobyan, Aurore Touzart, E. Jahn, B. Vick, Fiona Brown-Burke, Katherine Kelly, Simge Kelekçi, Anastasija Pejkovska, Ashish Goyal, Marion Bähr, Kersten Breuer, Mei-Ju May Chen, Maria Llamazares-Prada, M. Hartmann, M. Schönung, Nadia Correia, A. Trumpp, Yomn Abdullah, Ursula Klingmüller, S. S. Mughal, B. Brors, Frank Westermann, Elias Ulrich, Robert J. Autry, Matthias Schlesner, S. Vosberg, Tobias Herold, P. Greif, Dietmar Pfeifer, Michael Lübbert, Thomas Fischer, F. Heidel, Gebhard Gebhard, W. Walter, Torsten Haferlach, Ann-Kathrin Eisfeld, Krzysztof Mrózek, D. Nicolet, L. Bullinger, L. Smeenk, Claudia A. J. Erpelinck-Verschueren, Roger Mulet-Lazaro, Ruud Delwel, Aurélie Ernst, Michael Scherer, P. Lutsik, I. Jeremias, Konstanze Döhner, H. Döhner, D. Lipka, C. Plass
          </td>
          <td>2025-03-31</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/187ef3faa0e6c95091fda1daef8c22d574a2979f" target='_blank'>
              Training your genome
              </a>
            </td>
          <td>
            Alan Herbert
          </td>
          <td>2025-04-08</td>
          <td>Open Access Government</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b52e6b9621db88b32d01f7e93fc6430a88ce3be" target='_blank'>
              Comprehensive interrogation of synthetic lethality in the DNA damage response.
              </a>
            </td>
          <td>
            John Fielden, Sebastian M. Siegner, Danielle N Gallagher, Markus S Schröder, Maria Rosaria Dello Stritto, Simon Lam, L. Kobel, Moritz F. Schlapansky, S. P. Jackson, Petr Cejka, Marco Jost, Jacob E. Corn
          </td>
          <td>2025-04-09</td>
          <td>Nature</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e453c601838cb49a7f75622a8cb6e3e57e33c8f" target='_blank'>
              Integrative analysis of m6A-SNPs and single-cell RNA sequencing reveals key drivers of endocrine combined with CDK4/6 inhibitor therapy resistance in ER+ breast cancer
              </a>
            </td>
          <td>
            Ruijie Ming, Han Zhang, Huan Wu, Fangbiao Zhan, Xiaoping Huang, Huawen Liu
          </td>
          <td>2025-04-15</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genomic defects caused by truncating mutations or deletions in the Retinoblastoma tumor suppressor gene (RB1) are frequently observed in many cancer types leading to dysregulation of the RB pathway. Here, we propose an integrative proteogenomic approach that predicts cancers with dysregulation in the RB pathway. A subset of these cancers, which we term as “RBness,” lack RB1 genomic defects and yet phenocopy the transcriptional profile of RB1-defective cancers. We report RBness as a pan-cancer phenomenon, associated with patient outcome and chemotherapy response in multiple cancer types, and predictive of CDK4/6 inhibitor response in estrogen-positive breast cancer. Using RNA interference and a CRISPR-Cas9 screen in isogenic models, we find that RBness cancers also phenocopy synthetic lethal vulnerabilities of cells with RB1 genomic defects. In summary, our findings suggest that dysregulation of the RB pathway in cancers lacking RB1 genomic defects provides a molecular rationale for how these cancers could be treated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b769a127908e214629b60c835bc205b258e11d" target='_blank'>
              Proteogenomic discovery of RB1-defective phenocopy in cancer predicts disease outcome, response to treatment, and therapeutic targets
              </a>
            </td>
          <td>
            Jacopo Iacovacci, R. Brough, Fatemeh Ahmadi Moughari, J. Alexander, H. Kemp, Andrew N J Tutt, R. Natrajan, C. J. Lord, Syed Haider
          </td>
          <td>2025-03-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Cancer remains a significant global health challenge, claiming numerous lives annually and posing complex hurdles in management. The disease's multifaceted nature, combined with disparities in healthcare infrastructure and socioeconomic factors worldwide, complicates effective treatment. With alarming mortality rates, there is an urgent need to enhance prevention and early detection strategies. Early detection not only enables less invasive treatments but also improves the chances of successful recovery, highlighting the importance of proactive approaches. Lung cancer ranks highest in mortality rates, followed by breast, prostate, and colon cancers. Cancer is often seen as both a "genetic disease," involving gene mutations, and an "environmental disease," influenced by various environmental factors affecting gene expression. Metabolic alterations are a hallmark of cancer, and disruptions in cellular stability and function significantly contribute to its progression. Non-coding RNAs (ncRNAs) play crucial regulatory roles across cancers, with diverse structures and functions. They act as oncogenes or tumor suppressors depending on the cancer type, intricately modulating genetic and epigenetic processes. Recent studies challenge the perception of ncRNAs' insignificance, revealing their substantial impact on regulatory mechanisms. Indeed, ncRNAs intricately connect genetic networks, influencing vital protein effectors that dictate cellular responses and fate. Consequently, dysregulated ncRNAs play a pivotal role in disease pathogenesis, acting as either oncogenic drivers or tumor suppressors through aberrant expression in various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e52de2b0fb9cae18b101db41cf959bce17a1986" target='_blank'>
              The role of non-coding RNAs (ncRNAs) and their potential connection with cancer
              </a>
            </td>
          <td>
            Safura Absalan, Hamidreza Vaziri
          </td>
          <td>2025-03-22</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdb7d8e81c3f345d4ada707194f030cd48f14acb" target='_blank'>
              The malignant signature gene of cancer-associated fibroblasts serves as a potential prognostic biomarker for colon adenocarcinoma patients
              </a>
            </td>
          <td>
            Hao Zhang, Zirui Zhuang, Li Hong, Ruipeng Wang, Jinjing Xu, Youyuan Tang
          </td>
          <td>2025-04-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [1, 11],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>