<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-06-30 09:31:39 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>35</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>72</td>
          <td>138</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>114</td>
          <td>145</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>339</td>
          <td>138</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>297</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="The Extrachromosomal DNA (ecDNA) in cancer biology has emerged as a critical driver, enhancing the knowledge about tumorigenesis, progression and treatment. The current review article explores the multifaceted role of Extrachromosomal DNA, focusing on its ability to amplify oncogenes, enhance tumor heterogenicity, and support cancer cell survival and understanding therapeutics pressure. Moreover, the unique transcriptional and replication dynamics of ecDNA with linking the uneven segregation during cell division, create a significant environment for the understanding of genetic diversity and adaptive evolution in tumors. In clinics the ecDNA holds significant potential as a biomarker for cancer, diagnosis, treatment, therapeutic monitoring and progenies. The higher level of ecDNA correlates with poor clinical outcomes and resistance to therapies, emphasizing its prognostic value. The current therapeutic techniques targeting ecDNA specific vulnerabilities, such as disrupting replication stress pathways or silencing the transcriptional activity, represents promising ways for cancer treatment This review also highlights the potential of circulating ecDNA in liquid biopsies as a non-invasive diagnostic and monitoring tool. Despite these advances, challenges such as standardizing detection methods and understanding ecDNA diverse roles across cancer types remain. Addressing these gaps will unlock new opportunities in precision oncology, making ecDNA a cornerstone of future cancer diagnostics and therapeutics.
Bangladesh J Medicine 2025; 36(2): 82-91">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc548fd944fe0c719676cbeb264f28cf2ce64996" target='_blank'>
              The Multifaceted Role of Extrachromosomal DNA in Cancer: Functions and Clinical Implications
              </a>
            </td>
          <td>
            Shafee Ur Rehman
          </td>
          <td>2025-05-05</td>
          <td>Bangladesh Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has emerged as a dynamic and versatile genomic element with key roles in physiological regulation and disease pathology. This review synthesizes current knowledge on eccDNA, covering its classification, biogenesis, detection methods, biological functions, and clinical implications. Once considered rare anomalies, eccDNAs are now recognized as major drivers of oncogene amplification, genomic plasticity, and therapeutic resistance, particularly in cancer. EccDNA subtypes such as microDNA, double minutes, and ecDNA are defined by their structural, genomic, and pathological features. EccDNAs originate through diverse mechanisms including DNA repair, chromothripsis, breakage fusion bridge cycles, and apoptosis, occurring in both normal and stressed cells. Advances in long-read and single-cell sequencing, CRISPR-based synthesis, and computational tools have improved detection and functional analysis. Functionally, eccDNAs contribute to transcriptional amplification, activate immune responses through cGAS-STING signaling, and facilitate intercellular communication. They are found across a range of tissues and disease states—including cancer, cardiovascular, neurological, autoimmune, and metabolic disorders—and serve as both biomarkers and regulatory elements. We introduce the concept of the stress selection theory, which proposes eccDNA as an adaptive reservoir that enhances cellular fitness in response to environmental and therapeutic pressures. Despite growing insights, challenges remain in understanding tissue-specific roles, achieving clinical translation, and standardizing methodologies. Emerging tools in multi-omics, spatial biology, and artificial intelligence are expected to drive future breakthroughs in precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d31cb2df8f26047d209cf38790b651f7df3921fd" target='_blank'>
              Extrachromosomal circular DNA drives dynamic genome plasticity: emerging roles in disease progression and clinical potential
              </a>
            </td>
          <td>
            Bin Shi, Ping Yang, Huaijin Qiao, Daojiang Yu, Shuyu Zhang
          </td>
          <td>2025-05-25</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d556d25908485193c4e548584bab10bfa70daa1" target='_blank'>
              Oncogenic drivers shape the tumor microenvironment in human gliomas
              </a>
            </td>
          <td>
            Bo Zhao, Chae Yun Cho, Lingqun Ye, Tula Keal, Thomas Mitchell, Inés Martín-Barrio, C. A. Faraj, Mesut Unal, Jessica Alsing, Nora M Lawson, Ellie Rahm Kim, Francisco Chavez, Kristy Mendoza Rangel, Sanjay K Singh, Joseph H. McCarthy, D.M. Ashley, W. Yung, Vinay K. Puduvalli, Guy Nir, Jeffrey S. Weinberg, Linghua Wang, Keith L. Ligon, G. Genovese, Peter Van Loo, P. Futreal, Jason T. Huse, Jesse R. Dixon, Frederick F. Lang, Kadir C. Akdemir
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="The role of extrachromosomal DNA (ecDNA) in lung cancer, particularly in subjects who never smoked (LCINS), remains unclear. Examination of 1,216 whole-genome-sequenced lung cancers identified ecDNA in 18.9% of patients. Enrichment of MDM2 and other oncogenes’ amplification via ecDNA possibly drives a LCINS subset. Tumors harboring ecDNA or other focal amplifications showed similarly poor survival. A strong association with whole-genome doubling suggests most ecDNA reflects genomic instability in treatment-naïve lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20bf29ecb0d2ae5c2ac96f25fe4450672918b4ba" target='_blank'>
              Examining the Role of Extrachromosomal DNA in 1,216 Lung Cancers
              </a>
            </td>
          <td>
            Azhar Khandekar, P. Hoang, J. Luebeck, M. Díaz-Gay, Wei Zhao, John P. McElderry, Caleb Hartman, Mona Miraftab, Olivia W. Lee, Kara M. Barnao, E. Bergstrom, Yang Yang, Martin A. Nowak, Nathaniel Rothman, Robert J Homer, S. Yang, Qing Lan, David C Wedge, Lixing Yang, Stephen J Chanock, Tongwu Zhang, Ludmil B. Alexandrov, Marion Landi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Chromothripsis, the chaotic shattering and repair of chromosomes, is common in cancer. Whether chromothripsis generates actionable therapeutic targets remains an open question. In a cohort of 64 patients in blast phase of a myeloproliferative neoplasm (BP-MPN), we describe recurrent amplification of a region of chromosome 21q (‘chr. 21amp’) in 25%, driven by chromothripsis in a third of these cases. We report that chr. 21amp BP-MPN has a particularly aggressive and treatment-resistant phenotype. DYRK1A, a serine threonine kinase, is the only gene in the 2.7-megabase minimally amplified region that showed both increased expression and chromatin accessibility compared with non-chr. 21amp BP-MPN controls. DYRK1A is a central node at the nexus of multiple cellular functions critical for BP-MPN development and is essential for BP-MPN cell proliferation in vitro and in vivo, and represents a druggable axis. Collectively, these findings define chr. 21amp as a prognostic biomarker in BP-MPN, and link chromothripsis to a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4397c5f80ce07fe2ffd98add28275e15180327b" target='_blank'>
              Chromothripsis-associated chromosome 21 amplification orchestrates transformation to blast-phase MPN through targetable overexpression of DYRK1A
              </a>
            </td>
          <td>
            Charlotte K Brierley, B. Yip, G. Orlando, J. Wen, S. Wen, H. Goyal, M. Levine, G. M. Jakobsdottir, Avraam Tapinos, A. Cornish, A. Rodriguez-Romera, A. Rodriguez-Meira, M. Bashton, A. Hamblin, SA Clark, Joseph C Hamley, Olivia Fox, M. Giurgiu, Jennifer O'Sullivan, L. Murphy, A. Adamo, Aude-Anais Olijnik, A. Cotton, Emily Hendrix, S. Narina, S. Pruett-Miller, A. Enshaei, Claire N Harrison, Mark Drummond, S. Knapper, A. Tefferi, I. Antony-Debré, James O. J. Davies, A. Henssen, S. Thongjuea, David C Wedge, S. Constantinescu, E. Papaemmanuil, B. Psaila, John D. Crispino, Adam J Mead
          </td>
          <td>2025-06-09</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Chromoanagenesis is a catastrophic genomic phenomenon involving sudden, extensive rearrangements within one or a few cell cycles. In osteosarcoma, the most prevalent malignant bone tumor in children and adolescents, these events dramatically alter the genomic landscape, frequently disrupting key tumor suppressor genes like TP53 and RB1, amplifying oncogene expression, and propelling tumor progression and evolution. This review elucidates how key chromoanagenic mechanisms, such as chromothripsis and chromoanasynthesis, arise from replication stress and impaired DNA repair pathways, ultimately contributing to genomic instability in osteosarcoma. Chromothripsis features prominently in osteosarcoma, occurring in up to 62% of tumor regions and driving intratumoral heterogeneity through persistent genomic crises. Next-generation sequencing, optical genome mapping, and emerging technologies like single-cell sequencing empower researchers to detect and characterize these complex structural variants, demonstrating how a single catastrophic event can profoundly influence osteosarcoma progression over time. While targeted therapies for osteosarcoma have proven elusive, innovative strategies harnessing comprehensive genomic profiling and patient-derived preclinical models hold promise for uncovering tumor-specific vulnerabilities tied to chromoanagenesis. Ultimately, unraveling how these rapid, large-scale rearrangements fuel osteosarcoma’s aggressive nature will not only refine disease classification and prognosis but also pave the way for novel therapeutic approaches to enhance patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d502c33306d89faa688fa75a0e5f2710c0c03efc" target='_blank'>
              Chromoanagenesis in Osteosarcoma
              </a>
            </td>
          <td>
            Guozhuang Li, Nan Wu, Jennifer Ghabrial, Victoria Stinnett, Melanie Klausner, Laura A Morsberger, Patty Long, Ezra G. Baraban, John M. Gross, Ying S. Zou
          </td>
          <td>2025-06-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Extrachromosomal circular DNAs (eccDNAs) have gained attention as key players in cancer heterogeneity, potentially associated with elevated oncogene copy numbers in many cancers. While the presence of eccDNA in both normal and cancer cells is confirmed, its influence on gene-level alterations in cancer cells remains largely unexplored. This study delves into the genomic profiles of eccDNA in cholangiocarcinoma (CCA), an aggressive biliary tract cancer with extensive heterogeneity and diverse molecular alterations, using a modified long-read CircleSeq method. We reveal distinct eccDNA characteristics in CCA compared to non-tumor cells, focusing on genic components and chromosomal origins. Analysing read depth differences in oncogene-containing eccDNA; we identified potential eccDNA candidates that may be relevant for CCA biology. Subsequent bioinformatics analysis was performed using the established CReSIL tool, revealing distinct patterns of these oncogenes, particularly genes in the RAS/BRAF pathway, suggesting a potential functional role. These findings highlight the remarkable heterogeneity and diverse origins of eccDNA in CCA. This study establishes the first profiling of eccDNA in cholangiocarcinoma and paves the way for further investigation of its potential contribution to oncogene amplification and disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb865d79e02176dea33b880a5f277801ea0b0e0" target='_blank'>
              Characterization and identification of extrachromosomal circular DNA in cholangiocarcinoma
              </a>
            </td>
          <td>
            Zar Zar Win, Hasaya Dokduang, Siriyakorn Kulwong, W. Loilome, N. Namwat, Jutarop Phetcharaburanin, T. Wongsurawat, Piroon Jenjaroenpun, Poramate Klanrit, Arporn Wangwiwatsin
          </td>
          <td>2025-05-05</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Testicular germ cell tumors (TGCTs) are the most common malignancy in young men, exhibit a unique developmental origin and exceptional chemosensitivity. However, the molecular distinctions between TGCT subtypes remain poorly understood. Here we present a comprehensive genomic analysis of 252 treatment-naive primary TGCTs, integrating deep whole-genome sequencing with matched transcriptomic and epigenomic data. We identify new driver genes and uncover defining features of TGCTs, including pervasive chromosome X amplification with subtype-specific X chromosome inactivation, and a germ cell-like transcriptional program. Although previously reported, whole genome doubling (WGD) in TGCTs is further characterized here as ubiquitous, developmentally early, and associated with age at onset. Seminomas are enriched for early driver mutations, secondary WGD events, sustained XIST expression and replication stress-associated indel mutational signatures, while non-seminomas show greater structural complexity, subclonal diversity, relatively earlier-onset WGD, extended tumor latency, and telomere elongation. Moreover, we identify a mutational signature, SBS87, that is exceptionally rare across cancers with exception of thiopurine-treated leukemia, but strikingly prevalent in TGCT, especially non-seminomas. SBS87 is linked to extended tumor latency and telomere elongation, implicating possible environmental or endogenous processes that mimic thiopurine-induced DNA damage in TGCT pathogenesis. Collectively, our findings define TGCTs as molecularly distinct tumors shaped by early genomic instability and highlight SBS87 as a novel mutational footprint with potential etiologic and clinical relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/797d0b2769352db6f37defdb43fa1da2c4040fb1" target='_blank'>
              A Thiopurine-like Mutagenic Process Defines TGCT Subtypes
              </a>
            </td>
          <td>
            Kevin Brown, Jun Zhong, Adriana Morales Miranda, Mengyan Zhang, Joycelyn Williams, Jacob Williams, Haoyu Zhang, Cheng Liang, Wenbo Li, Bin Zhu, Stephen J Chanock, K. Nathanson, Tongwu Zhang
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6086eff81e816ca3205d5bed8a0f1a0d7841fa7" target='_blank'>
              Oncogene aberrations drive medulloblastoma progression, not initiation.
              </a>
            </td>
          <td>
            K. Okonechnikov, P. Joshi, Verena Körber, Anne Rademacher, Michele Bortolomeazzi, Jan-Philipp Mallm, Jan Vaillant, P. B. da Silva, Britta Statz, Mari Sepp, Ioannis Sarropoulos, Tetsuya Yamada, A. Wittmann, Kathrin Schramm, Mirjam Blattner-Johnson, Petra Fiesel, Barbara Jones, N. Jäger, T. Milde, K. Pajtler, Cornelis M. van Tilburg, Olaf Witt, Konrad Bochennek, Katharina Johanna Weber, Lisa Nonnenmacher, Christian Reimann, David R. Ghasemi, U. Schüller, M. Mynarek, S. Rutkowski, David T. W. Jones, A. Korshunov, K. Rippe, Frank Westermann, S. Thongjuea, Thomas Höfer, Henrick Kaessmann, Lena M. Kutscher, Stefan M. Pfister
          </td>
          <td>2025-05-07</td>
          <td>Nature</td>
          <td>1</td>
          <td>112</td>
        </tr>

        <tr id="Maintaining genome integrity is crucial for the proper functioning and development of organisms. One intriguing aspect of genome integrity is the formation and function of neocentromeres at non-centromeric sites. CENP-A, a centromere-specific protein, is essential for centromere identification and function. However, in many cancers, CENP-A is often found to be ectopically misplaced when overexpressed. Moreover, CENP-A deposition at the centromere depends on the transcription of centromeric non-coding RNAs. Consequently, ectopic CENP-A is found at transcriptionally active and frequent breakpoint regions. To further explore ectopic CENP-A localization, we previously engineered a stable ectopic CENP-A site on a naïve chromosome by overexpressing a non-centromeric oncogenic lncRNA, PCAT2, which was capable of recruiting CENP-A to its transcription site. In this work, we tracked cells carrying this stable transgene to understand the longevity of the induced ectopic CENP-A site at the chromosome that harbors it. Our findings revealed that the induced epigenetic memory was eventually lost due to the suppression of the transgene through competing epigenetic silencing mechanisms. This epigenetic restoration naturally reversed the ectopic CENP-A level to its previous levels at the engineered site. These data suggest that cells may have evolved failsafe mechanisms to prevent neocentromere formation at ectopic sites by suppressing transcription, unless otherwise favored by selection involving multiple components.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/812046bef63516ecf0a2f8e99e68c96c27339a88" target='_blank'>
              Oncogenic lncRNA transgene transcription modulates epigenetic memory at a naïve chromosomal locus
              </a>
            </td>
          <td>
            Sweta Sikder, Songjoon Baek, Yamini Dalal, Ganesan Arunkumar
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Acquired genetic alterations drive resistance to endocrine and targeted therapies in metastatic breast cancer; however, the underlying processes engendering these alterations are largely uncharacterized. To identify the underlying mutational processes, we utilized a clinically annotated cohort of 3,880 patient samples with tumor-normal sequencing. Mutational signatures associated with apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) enzymes were prevalent and enriched in post-treatment hormone receptor-positive cancers. These signatures correlated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor therapy in hormone receptor-positive metastatic breast cancer. Whole-genome sequencing of breast cancer models and paired primary-metastatic samples demonstrated that active APOBEC3 mutagenesis promoted therapy resistance through characteristic alterations such as RB1 loss. Evidence of APOBEC3 activity in pretreatment samples illustrated its pervasive role in breast cancer evolution. These studies reveal APOBEC3 mutagenesis to be a frequent mediator of therapy resistance in breast cancer and highlight its potential as a biomarker and target for overcoming resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65dfedea9c7ed384b972c98e352341985ea58fa" target='_blank'>
              APOBEC3 mutagenesis drives therapy resistance in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Hong Shao, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Yanjun Chen, Isabella Vegas, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, G. Curigliano, B. Weigelt, N. Riaz, Simon N. Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-05-16</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) is among the most frequently diagnosed cancer types, and affected patients frequently experience poor prognostic outcomes and high mortality rates. Many genomic studies of ESCC have been performed in recent years, yet the mutational mechanisms driving ESCC and their clinical implications remain incompletely understood. In this study, paired tumor and normal tissue samples from 22 patients with ESCC were used for whole genome sequencing-based analyses of genome-wide mutational events. These comprehensive analyses enabled the detection and characterization of various mutation subtypes in ESCC including somatic single-nucleotide variants, small insertions and deletions, copy number variations, structural variations, and circular extrachromosomal DNA. Of identified genes harboring non-silent mutations, TP53, NOTCH1, CSMD3, EP300, and FAM135B were the most frequently mutated genes in this study and they were annotated in the COSMIC Cancer Gene Census. With the exception of aging-related signatures, an APOBEC-associated mutational signature was the dominant mutational feature detected in ESCC samples, suggesting that APOBEC-mediated cytidine deamination is likely a major driver of mutations in this cancer type. Notably, our study also detected circular extrachromosomal DNA (ecDNA) events in these ESCC patient samples. The oncogenes COX6C, PVT1, and MMP12 as well as the oncogenic long non-coding RNA AZIN1-AS1 which were detected in ecDNA regions in these analyses may be associated with worse disease-free survival in ESCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b55745d3238cc107d72d837356b7050823d1fc40" target='_blank'>
              Whole genome sequencing revealed esophageal squamous cell carcinoma related biomarkers
              </a>
            </td>
          <td>
            Mingjun Li, Lei Li, Xizi Wang, Yanwei Zhao, Peina Du, Wei Wang, , Yadong Wang, Yanxing Sheng, Mingliang Gu, Xiaodong Jia
          </td>
          <td>2025-06-26</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background The incidence and mortality of endometrial cancer (EC) is on the rise. Eighty-five percent of ECs depend on estrogen receptor alpha (ERα) for proliferation, but little is known about its transcriptional regulation in these tumors. Results We generate epigenomics, transcriptomics, and Hi-C datastreams in healthy and tumor endometrial tissues, identifying robust ERα reprogramming and profound alterations in 3D genome organization that lead to a gain of tumor-specific enhancer activity during EC development. Integration with endometrial cancer risk single-nucleotide polymorphisms and whole-genome sequencing data from primary tumors and metastatic samples reveals a striking enrichment of risk variants and non-coding somatic mutations at tumor-enriched ERα sites. Through machine learning-based predictions and interaction proteomics analyses, we identify an enhancer mutation which alters 3D genome conformation, impairing recruitment of the transcriptional repressor EHMT2/G9a/KMT1C, thereby alleviating transcriptional repression of ESR1 in EC. Conclusions In summary, we identify a complex genomic-epigenomic interplay in EC development and progression, altering 3D genome organization to enhance expression of the critical driver ERα. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03596-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a79c6ec91a0f963e937bbd37603ce55880af9ede" target='_blank'>
              Endometrial tumorigenesis involves epigenetic plasticity demarcating non-coding somatic mutations and 3D-genome alterations
              </a>
            </td>
          <td>
            Sebastian Gregoricchio, Aleksandar Kojic, M. Hoogstraat, K. Schuurman, Suzan Stelloo, T. Severson, Tracy A. O’Mara, M. Droog, Abhishek A. Singh, Dylan M Glubb, L. Wessels, Michiel Vermeulen, Flora E. van Leeuwen, W. Zwart
          </td>
          <td>2025-05-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 3090


 Background:
 Recurrent cytogenetic abnormalities represent candidate therapeutic targets for children with neuroblastoma (NB).
 MYCN
 oncogene amplification is associated with significantly worse survival rates for children with NB and remains one of the primary predictors of patient prognosis.
 MYCN
 amplifications in NB can be found both within the linear genome and on circular extrachromosomal DNA (ecDNA), and therapeutic targeting of the mechanisms underlying
 MYCN
 amplification represents a novel and promising strategy in NB. However, the molecular features and clinical and biological significance of these amplifications in NB tumors are not sufficiently understood.
 Methods:
 Whole genome and RNA sequencing data were analyzed for NB cell lines and NCI TARGET NB samples using AmpliconSuite software for ecDNA identification and characterization. GISTIC was used for identification of recurrently amplified regions. Gene expression levels were determined using StringTie, and gene clustering heatmaps were generated using FeatureCounts software. For differential gene expression analyses, samples were divided into ecDNA
 +
 and ecDNA
 -
 , and genes contained on ecDNA were compared to the same regions on linear DNA across samples using DESeq2. Associations between ecDNA quantity, content, and patient survival were performed using multivariate Cox regression survival analysis. Associations of gene expression with patient survival were performed using the R2 Platform. The efficacy of targeting ecDNA-associated gene products was assessed using live cell imaging and cell viability assays.
 Results:
 WGS analysis confirmed 7/20 NB patient tumors from the TARGET database to be ecDNA amplified with 1-5 independent ecDNA elements and
 MYCN
 gene expression correlated with the ecDNA copy number. ecDNAs in
 MYCN
 -amplified neuroblastoma cell lines contained distinct gene combinations and possessed unique structures. MYCN overexpression in NB cells has been shown to be associated with replication stress (RS), and tumor cells containing ecDNA are hyper-reliant on the DNA damage response (DDR) kinase CHK1 to manage heightened replication stress. Expression of the
 CHK1
 gene was associated with neuroblastoma patient outcomes and neuroblastoma was most significantly associated with
 CHK1
 RNA dependency. We further validated CHK1i as a promising therapeutic strategy in
 MYCN
 amplified NB, as CHK1 inhibition with the novel inhibitor BBI-2779 was most effective against ecDNA+,
 MYCN
 -amplified neuroblastoma cell lines.
 Conclusions:
 Our results emphasize the critical role of ecDNA in NB. We identify a synthetic lethality axis shaped by ecDNA
 MYCN
 amplification and CHK1 dependence. We further demonstrate the feasibility of targeting this vulnerability through CHK1 inhibition, thus offering new avenues for treatment in
 MYCN
 amplified tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e222ec29cad608a8fd4761ccb837028a1803c8a" target='_blank'>
              Effect of extrachromosomal DNA (ecDNA) on
 MYCN
 amplified neuroblastoma and patient outcomes.
              </a>
            </td>
          <td>
            Mihika Sonalkar, Carla Sampaio, Ting Yang, Taelor Getz, Catherine Shaw, Vicky Pham, Joshua Lange, Sudhir Chowdhry, Christian A Hassig, L. Alexandrov, Peter E. Zage
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Despite the advances in single cell whole genome sequencing, the ability to detect structural variations from mitotic crossovers remains challenging, while sister chromatid exchanges remain undetectable. Here we describe sci-L3-Strand-seq, a combinatorial indexing method with linear amplification for DNA template strand sequencing that cost-effectively scales to millions of single cells. We provide a computational framework to fully leverage the throughput, as well as the relatively sparse but multifaceted genotype information within each cell that includes strandedness, digital counting of copy numbers, and haplotype-aware chromosome segmentation, to systematically distinguish seven possible types of mitotic crossover outcomes and resulting genome instability. We showcase the power of sci-L3-Strand-seq by quantifying the rates of error-free and mutational crossovers in thousands of cells, enabling us to explore enrichment patterns of genomic and epigenomic features. The throughput of sci-L3-Strand-seq also gave us the ability to measure subtle phenotypes, opening the door for future large mutational screens. Furthermore, mapping clonal lineages provided insights into the temporal order of certain genome instability events, showcasing the potential to dissect cancer evolution. Altogether, we show the wide applicability of sci-L3-Strand-seq to the study of DNA repair, structural variations, and genome instability in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/867f71c31d23d231530c8f668476d966ca7500c8" target='_blank'>
              High-throughput mapping of spontaneous mitotic crossover and genome instability events with sci-L3-Strand-seq
              </a>
            </td>
          <td>
            Peter Chovanec, Trevor Ridgley, Yi Yin
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is recognized for its aggressiveness, yet the mechanism underlying metastasis remains unclear. Here, we report that CREPT/RPRD1B, which exhibits somatic gene copy-number amplifications and elevated expression, correlates with poor patient survival and drives TNBC metastasis. We demonstrate that CREPT alters three-dimensional genome structures in topologically-associating domain (TAD) status and chromatin loops via occupying promoters and enhancers. Specifically, CREPT mediates 1082 co-operational chromatin loops configured by enhancer-promoter and promoter-termination loops, which are validated by HiChIP analyses and visualized by Tn5-FISH experiments. These loops orchestrate RNAPII loading and recycling to enhance the metastatic gene expression. Disruption of these co-operational loops using CRISPR-dCas9 suppresses TNBC metastasis in vivo. Furthermore, depletion of CREPT using an AAV-based shRNA blocks TNBC metastasis in both preventative and therapeutic mouse models. We propose that targeting CREPT to disrupt the co-operational chromatin loop structures represents a promising therapeutic strategy for metastatic TNBC. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02361-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d99f51f69c1d7213a6e3e1569c1bdb0463d7e42a" target='_blank'>
              CREPT is required for the metastasis of triple-negative breast cancer through a co-operational-chromatin loop-based gene regulation
              </a>
            </td>
          <td>
            Jianghua Li, Lu Xu, Jiayu Wang, Xuning Wang, Yuting Lin, Alex Yutian Zou, Fangli Ren, Yinyin Wang, Jun Li, Zhijie Chang
          </td>
          <td>2025-06-10</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Cancer cells undergo widespread changes in epigenetic patterns that mediate cancer compromised gene expression programs during cancer progression. However, the alterations in higher-order genome organization in which these changes occur and their functional implications are less well understood. To explore how chromatin structure and epigenetic parameters of genome architecture changes during cancer progression at a fine scale and genome-wide, we generated high-resolution Micro-C contact maps in non-malignant, pre-cancerous, and metastatic MCF10 breast cancer epithelial cells. Results We profiled progression-associated reorganization of chromatin compartments, topologically associated domains (TADs), and chromatin loops, and also identified invariable chromatin features. We find large-scale compartmental shifts occur predominantly in early stages of cancer development, with more fine-scale structural changes in TADs and looping accumulating during the later transition to metastasis. We related these structural features to changes in gene expression, histone marks, and potential enhancers and found a large portion of diYerentially expressed genes physically connected to distal regulatory elements. While changes in chromatin loops were relatively rare during progression, diYerential loops were enriched for progression-associated genes, including those involved in proliferation, angiogenesis, and diYerentiation. Changes in either enhancer-promoter contacts or distal enhancer activity were accompanied by diYerential gene regulation, suggesting that changes in chromatin contacts are not necessary but can be suYicient for gene regulation. Conclusions Together, our results demonstrate a functionally relevant connection between gene regulation and genome remodeling at many key genes during cancer progression. KEY FINDINGS ‒ The cancer genome is reorganized throughout cancer progression at the level of compartments, chromatin domains, and loops ‒ Compartmental shifts occur in early stages of cancer development, with more fine-scale structural changes accumulating during metastasis ‒ Chromatin domain boundaries are weakened during cancer progression ‒ Many progression-regulated genes exhibit changes in distal enhancer histone modifications that are bridged by stable chromatin loops ‒ Changes in enhancer activity or subtle changes in chromatin contacts can rewire enhancer-promoter connections to facilitate changes in gene expression ‒ Prominent changes in chromatin loops occur at a small subset of diYerentially regulated genes during progression">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f24eeb9036338ab76b344cd3bcd26b76e9bfec03" target='_blank'>
              Genome reorganization and its functional impact during breast cancer progression
              </a>
            </td>
          <td>
            K. S. Reed, Andrew J. Fritz, Haley Greenyer, Kerstin Heselmeyer-Haddad, Seth Frietze, Janet Stein, Gary S. Stein, Tom Misteli
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Loss-of-function mutations in genome maintenance genes fuel tumorigenesis through increased genomic instability. A subset of these tumor suppressors are challenging to identify due to context dependency, including functional interactions with other genes and pathways. Here, we searched for potential causal genes that impact tumor development and/or progression in breast cancer through functional-genetic screening of candidate genes. MYH4, encoding a class II myosin, emerged as a top hit impacting genomic stability. We show that MYH4 suppresses DNA replication stress by promoting replication licensing and replication fork progression. Moreover, we observed a strong synergistic relationship among class II myosins in suppressing replication-associated DNA damage. Genomic analysis of Pan-Cancer Analysis of Whole Genomes project breast cancer samples revealed frequent concomitant loss of TP53 with MYH4 and class II myosins on chromosome 17p. Notably, Myh4 disruption accelerated mouse mammary tumorigenesis in a Trp53-deficient background. In conclusion, our results suggest an unanticipated function of MYH4 in p53-mediated tumor suppression that can explain their combined loss in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64806f05c07b8d24d35ea0d2ce3f2fd4c89aa895" target='_blank'>
              The class II myosin MYH4 safeguards genome integrity and suppresses tumor progression
              </a>
            </td>
          <td>
            J. Thatte, Ana Moisés da Silva, Judit Börcsök, Thorkell Gudjónsson, Jan Benada, Xin Li, Muthiah Bose, H. van der Gulden, Ji-Ying Song, Renée Menezes, Elena Martín-Doncel, Luis Toledo, Valdemaras Petrosius, Cord Brakebusch, J. Jonkers, F. C. Nielsen, Maria Rossing, C. S. Sørensen
          </td>
          <td>2025-06-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Transcriptional activation of otherwise repressed endogenous retroelements (RTEs) is a hallmark of cancer, shaping tumour progression and immunogenicity by multifaceted, yet incompletely understood mechanisms. Methods We used an extended pan-cancer transcriptome assembly to identify potential effects of RTEs on the genes near or within which they have integrated. These were subsequently verified in test cases by further analysis of transcriptional profiles in cancer patient data, and by in vitro studies involving restoration of gene activity, and proliferation and migration assays in cancer cell lines. Results We report that cancer-specific transcriptional activation of RTEs causes frequent reduction or loss of gene function. Exonisation and alternative splicing of RTEs creates non-functional RNA and protein isoforms and derepressed RTE promoter activity initiates antisense transcription, both at the expense of the canonical isoforms. Contrary to theoretical expectation, transcriptionally activated RTEs affect genes with established tumour-promoting function, including the common essential RNGTT and the lung cancer-promoting CHRNA5 genes. Furthermore, the disruptive effect of RTE activation on adjacent tumour-promoting genes is associated with slower disease progression in clinical data, whereas experimental restoration of gene activity enhances tumour cell in vitro growth and invasiveness Conclusions These findings underscore the gene-disruptive potential of seemingly innocuous germline RTE integrations, unleashed only by their transcriptional utilisation in cancer. They further suggest that such metastable RTE integrations are co-opted as sensors of the epigenetic and transcriptional changes occurring during cellular transformation and as executors that disrupt the function of tumour-promoting genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90139c8e3ae8545f259aeac6842fc6fa1b255360" target='_blank'>
              Retroelement co-option disrupts the cancer transcriptional programme
              </a>
            </td>
          <td>
            Jane Loong, Rachael Thompson, Callum Hall, Laura Doglio, Judith Pape, G. Kassiotis
          </td>
          <td>2025-02-27</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23f7337991d50d13c4c6add597a9dcc86990c583" target='_blank'>
              Persistent chromatin alterations and gene expression reprogramming follow widespread DNA damage in glioblastoma
              </a>
            </td>
          <td>
            A. Modrek, Ken Chandradoss, Catherine Do, R. Ezhilarasan, Theodore Sakellaropoulos, Jerome Karp, Yingwen Ding, Ze-Yan Zhang, Melanie Graciani, Giulia Cova, Jennifer E. Phillips-Cremins, Erik P. Sulman, Jane A. Skok
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041a1ec7061bd7c816f107766afc5488bd6dd34f" target='_blank'>
              G-quadruplex stabilization induces DNA breaks in pericentromeric repetitive DNA sequences in B lymphocytes
              </a>
            </td>
          <td>
            Irina Waisertreiger, Kalkidan Ayele, Mehad Hilal Elshaikh, Jacqueline H. Barlow
          </td>
          <td>2025-05-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transposable elements (TEs) constitute over half of the human genome and have played a profound role in genome evolution. While most TEs have lost the ability to transpose, many retain functional elements that serve as drivers of genome innovation, including the emergence of novel genes and regulatory elements. Recent advances in experimental and bioinformatic methods have provided new insights into their roles in human biology, both in health and disease. In this review, we discuss the multifaceted roles of TEs in haematopoiesis, highlighting their contributions to both normal and pathological contexts. TEs influence gene regulation by reshaping gene-regulatory networks, modulating transcriptional activity, and creating novel regulatory elements. These activities play key roles in maintaining normal haematopoietic processes and supporting cellular regeneration. However, in haematological malignancies, TE reactivation can disrupt genomic integrity, induce structural variations, and dysregulate transcriptional programmes, thereby driving oncogenesis. By examining the impact of TE activity on genome regulation and variation, we highlight their pivotal roles in both normal haematopoietic processes and haematological cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae81192284f686304fb88058cdc41885e0ae02d1" target='_blank'>
              Transposable elements as genome regulators in normal and malignant haematopoiesis
              </a>
            </td>
          <td>
            D. Prokopov, Hale Tunbak, Eve Leddy, Bryce Drylie, Francesco Camera, Özgen Deniz
          </td>
          <td>2025-05-06</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Oncogene-induced replicative stress (RS) drives tumor progression by disrupting genome stability, primarily through transcription-replication conflicts (TRCs), which promote R-loop accumulation and trigger the DNA damage response (DDR). In this study, we investigate the role of chromatin regulators in exacerbating TRCs and R-loop accumulation in cancer. We find that in breast cancer patients, the simultaneous upregulation of MYC and the H2A.Z-specific chaperone ANP32E correlates with increased genomic instability. Genome-wide analyses reveal that ANP32E-driven H2A.Z turnover alters RNA polymerase II processivity, leading to the accumulation of long R-loops at TRC sites. Furthermore, we show that ANP32E overexpression enhances TRC formation and activates an ATR-dependent DDR, predisposing cancer cells to R-loop-mediated genomic fragility. By exploiting the vulnerability of ANP32E-expressing cancer cells to ATR inhibitors, we find that tumors relied on this DDR pathway, whose inhibition halts their pro-metastatic capacity. These findings identify ANP32E as a key driver of TRC-induced genomic instability, indicating ATR inhibition as a potential therapeutic strategy for ANP32E-overexpressing tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a5a4a477e7eaceca00bef62d4e70f4afdb446c" target='_blank'>
              ANP32E drives vulnerability to ATR inhibitors by inducing R-loops-dependent transcription replication conflicts in triple negative breast cancer
              </a>
            </td>
          <td>
            Sara Lago, V. Poli, Lisa Fol, Mattia Botteon, Federica Busi, A. Turdo, Miriam Gaggianesi, Yari Ciani, Giacomo D’Amato, Luca Fagnocchi, A. Fasciani, F. Demichelis, Matilde Todaro, A. Zippo
          </td>
          <td>2025-05-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Reactivation of transposable elements (TEs) in somatic tissues, particularly of LINE-1, is associated with disease by causing gene mutations and DNA damage. Previous work has shown that the PIWI pathway is crucial for TE suppression in the germline. However, the status and function of this pathway has not been well characterized in differentiated somatic cells and there is lack of consensus on the role of the pathway in somatic tumorigenesis. To shed light on this conundrum, we examined the PIWI pathway in colon cancer through a combination of bioinformatic analyses and cell-based assays. Shifted Weighted Annotation Network (SWAN) analysis revealed that the pathway experiences significant allelic losses in colon cancer and that PIWIL2, the main catalytic component of the pathway responsible for TE silencing, experiences the highest percent deletions. PIWIL2 is downregulated in colon tumors of advanced stage, nodal metastasis, and in certain subtypes, correlating with poor survival, while it is even downregulated in ulcerative colitis, an inflammatory bowel disease that predisposes to colon cancer. Knockout studies in colon epithelial Caco2 cells show that PIWIL2 depletion leads to increased anchorage-independent growth, increased LINE-1 levels and activity, and in DNA damage, altogether highlighting a tumor-suppressing role of PIWIL2 in the colon. SUMMARY STATEMENT This study investigates the PIWI-piRNA pathway in colon cancer using a nuanced bioinformatic and cell-based approach, linking the downregulation of PIWIL2 to disease progression, transposable element activation and DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6553455e0dd4bcf44406169f133238204316d7c1" target='_blank'>
              PIWIL2 downregulation in colon cancer promotes transposon activity and pro-tumorigenic phenotypes
              </a>
            </td>
          <td>
            Alyssa Risner, J. Nair-Menon, Abhinav Cheedipudi, Joe R Delaney, Vamsi K Gangaraju, A. Kourtidis
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57de502f1073df10f45cbf431033ea87108c552c" target='_blank'>
              ARID1A stabilises non-homologous end joining factors at DNA breaks induced by the G4 ligand pyridostatin
              </a>
            </td>
          <td>
            Susana Simões-Sousa, Noa Amin, Karen A. Lane, Alison Harrod, Malin Pedersen, Mercedes Pardo, Christine Rasetti-Escargueil, Z. Kozik, Katheryn A. G. Begg, Alison Ribeiro, Susana Banerjee, Manuel Stucki, Kevin J. Harrington, Jyoti S. Choudhary, Jessica A. Downs
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Focal gene amplification serves as an oncogenic driver during tumorigenesis and is a hallmark of many forms of cancer. Oncogene amplifications promote genomic instability, which is integral to cancer cell survival and evolution. However, focal gene amplification potentially affords an opportunity for therapeutic exploitation. As a proof-of-concept, we leverage CRISPR-Cas9 nickase to selectively promote cancer cell death in MYCN-amplified neuroblastoma in a gene amplification-dependent manner. Our analysis demonstrates that CRISPR-Cas9 nickase can generate a lethal number of highly toxic, replication-dependent double-strand breaks in cells harboring amplified loci. Furthermore, we demonstrate that Cas9 nickase—mediated toxicity can be modulated in combination with small molecule inhibitors targeting key regulators of the DNA-damage response or cell death pathways. Importantly, our findings in MYCN-amplified neuroblastoma translate to other cancer types with distinct oncogene amplifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426fa411ebfe0306ef5fb3727f439d058ec49944" target='_blank'>
              Selective targeting of genome amplifications and repeat elements by CRISPR-Cas9 nickases to promote cancer cell death
              </a>
            </td>
          <td>
            Matthew B Hanlon, Jason M Shohet, S. Wolfe
          </td>
          <td>2025-06-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Chromosomal instability (CIN) is a hallmark of cancer, closely associated with tumor evolution, metastasis, immune evasion, and resistance to treatment. CIN is driven by persistent chromosome missegregation, resulting in abnormal chromosomal copy numbers and promoting tumor progression through mitotic errors, faulty chromosome segregation, and the formation of micronuclei or chromosome bridges. Previous studies have demonstrated that p62 localizes to micronuclei, where it interacts with mitochondria, enhancing ROS-mediated cysteine oxidation and promoting p62 homo-oligomerization. This disrupts repair by recruiting CHMP7 and other ESCRT-III components, impairing their function in restoring the micronuclear envelope. Notably, we observed a time-dependent increase in nuclear p62 levels following radiation therapy, which correlated with enhanced chromatin fragmentation and chromothripsis. While p62 inhibits homologous recombination repair and promotes non-homologous end joining, we hypothesize that radiation-induced nuclear p62 accumulation impairs nuclear envelope resealing by inhibiting ESCRT-III components, thereby contributing to chromosomal fragmentation and chromothripsis. These chromosomal alterations may play a significant role in tumor evolution and treatment resistance. Our findings suggest that irradiation-induced p62 accumulation is crucial for chromothripsis and may affect nuclear repair processes, impacting chromosomal stability. Future research will focus on elucidating how p62 contributes to these abnormalities and their implications for tumor progression and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a93ad5e20684542464f1d1902962e0915d7957" target='_blank'>
              Irradiation-induced increase in nuclear p62 levels contributes to chromosomal fragmentation and chromothripsis.
              </a>
            </td>
          <td>
            Xiaojuan Yang, Ying Zhou, Buzhe Zhang, Xue Tao, Weili Qi, Hong Wu
          </td>
          <td>2025-06-05</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Oncogene activation is a key driver of cancer development, inducing aberrant cellular proliferation and DNA replication stress. This in turn, leads to DNA damage-which accumulates in specific genomic regions-contributing to genomic instability in cancer. However, the interplay between oncogene-induced DNA damage and chromatin organization is still poorly understood. In this study, we introduce a QUantitative ANalysis of DNA cOunterstains (QUANDO) to investigate the subnuclear localization of DNA damage in single-cell nuclei of U937-PR9 cells, an in vitro model of acute promyelocytic leukemia (APL). Using advanced imaging techniques, including DNA intensity analysis and colocalization by image cross-correlation spectroscopy (ICCS), we map DNA damage foci and correlate them with chromatin regions of different density. QUANDO is applied to dual-color confocal images of the DNA damage marker γ-H2AX and the DNA counterstain DAPI, allowing single-cell measurements of foci distribution within areas of low or high DNA density. We find that spontaneous DNA damage and DNA damage induced by the activation of PML-RARα oncogene predominantly localize in euchromatic regions. Conversely, when DNA damage is induced by the radiomimetic agent neocarzinostatin (NCS), the foci appear more evenly distributed in euchromatic and heterochromatic regions. These findings underscore the complex interplay between oncogene activation and chromatin organization, revealing how disruptions in DNA damage distribution can contribute to genomic instability and offering new insights for targeting DNA repair mechanisms in cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524273685b3a7b2a132bed81549714ac0e395f5e" target='_blank'>
              Location of oncogene-induced DNA damage sites revealed by quantitative analysis of a DNA counterstain.
              </a>
            </td>
          <td>
            Greta Paternò, Silvia Scalisi, G. Dellino, M. Faretta, P. Pelicci, Alberto Diaspro, Luca Lanzanò
          </td>
          <td>2025-05-07</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Purpose To investigate the causal effects of gene expression levels associated with mitochondrial DNA copy number (mtDNA-CN) on cancer survival outcomes. Methods We performed two-sample mendelian randomization (MR) analyses to evaluate the causal relationship between mtDNA-CN in peripheral blood and overall survival period of different cancer types. Additionally, colocalization analyses were performed to identify genes whose expression correlates with mtDNA-CN. Furthermore, we investigated how expression levels of these genes impact survival period in different cancers. Results Higher mtDNA-CN in peripheral blood is significantly associated with reduced survival period in gastric cancer (HR = 72.97, 95% CI 68.86 to 77.08, p = 0.046). Colocalization analysis identified six genes, PNP, CDK10, C7orf73, NLRP2, MYO15A, and OSGEP, that regulate the mtDNA-CN in stomach tissue. Among them, higher expressions of MYO15A, OSGEP, and CDK10 are significantly associated with shorter gastric cancer survival. Notably, the impact of OSGEP expression on survival was particularly significant in the ERBB2 amplified subgroup (p = 0.025). Enrichment analysis suggests that the risk genes impact overall survival in gastric cancer prognosis by modulating mitochondrial function and DNA copy number. Conclusion Higher mtDNA-CN in peripheral is correlated with the shorter overall survival in gastric cancer patient. Higher expression of MYO15A, OSGEP, and CDK10 is associated with shorter survival in gastric cancer patients, potentially by regulating mtDNA-CN, suggesting potential targets for future therapeutic interventions of gastric cancer. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02825-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee200b9d06dd59ef8cf215af498983da37e799cc" target='_blank'>
              Dynamics of mitochondrial DNA copy number regulation in relation to gastric cancer survival
              </a>
            </td>
          <td>
            Qizhou Yang, Tian Zhao, Yuying Han, Zhengguo Ji, Haihong Ye
          </td>
          <td>2025-06-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Retrotransposon mobilization in germline cells enables the rewriting of genetic information to drive genome innovation, species evolution, and adaptation through the generation of de novo mutations. However, uncontrolled mobilization can cause DNA breaks and genome instability, often leading to sterility. How germ cells balance retrotransposon-induced genome innovation with the need for genomic integrity remains poorly understood. Here, we used Drosophila spermatogenesis as a model to investigate retrotransposon mobilization dynamics. Although many retrotransposon families are transcriptionally active, we found that the LTR-retrotransposon nomad completes the full mobilization cascade—including mRNA export, protein translation, and reverse transcription—to produce double-stranded DNA (dsDNA) the most efficiently. Strikingly, despite successfully generating dsDNA, nomad rarely achieves genomic reintegration. Instead, its newly synthesized DNA predominantly forms extrachromosomal circular DNA (ecDNA). These findings suggest that ecDNA formation acts as a protective mechanism to sequester retrotransposon-derived DNA and prevent widespread genomic integration during spermatogenesis, thereby preserving genome stability while allowing limited retrotransposon activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e357c0e1274581f338acdf17e4b08fe74b5c828" target='_blank'>
              Safeguarding spermatogenesis from retrotransposon insertions by forming ecDNA
              </a>
            </td>
          <td>
            Lauren Tracy, Z. Zhang
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chromosomal 11q13.3 amplification is the most common gene copy-number variation event in head and neck squamous cell carcinoma (HNSCC) that corresponds with poor prognosis. Although cyclin D1, a G1/S phase cell-cycle regulatory protein at this locus, is considered as a key driver of malignant progression, further exploration is needed to develop more effective targets for cases with this amplification. Using CRISPR-based gene knockout screening of genes located in chr11q13.3, we found that loss of the gene encoding the Fas-associated death domain (FADD) protein, a well-recognized adapter to caspase-8 that induces cell apoptosis, significantly reduced cancer cell proliferation. FADD expression was elevated in chr11q13.3-amplified tumors and correlated with poor prognosis. RNA sequencing, mass spectrometry, and proteomics analyses revealed a direct relationship between FADD and the DNA helicase MCM5 in the S phase. FADD and cyclin D1 acted at different stages of the cell cycle to synergistically induce proliferation, and caspase-8 deficiency was required for the oncogenic activity of FADD. In a patient-derived xenograft model with chr11q13.3 amplification, combined administration of the DNA helicase complex inhibitor and CDK4/6 inhibitor effectively curtailed tumor growth. Overall, this study identified a nonclassic oncogenic role for FADD in mediating tumor progression in HNSCC and provided a feasible treatment option for patients with chr11q13.3 amplification. Significance: FADD promotes progression of tumors with chr11q13.3 amplification by binding to the DNA helicase complex, which can be targeted in combination with cyclin D1 as a viable therapeutic strategy for HNSCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/659d7605f636160fd781d9f5d4b484dd97ec74c4" target='_blank'>
              FADD Functions as an Oncogene in Chr11q13.3-Amplified Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yang Zheng, Yinan Chen, Xiaoyan Meng, Li Zhang, Yanni Ma, Rong Zhou, Shuiting Fu, Heng Chen, Xinyang Xuanyuan, Ruixin Jiang, Pengcong Hou, Xiaomeng Song, Yanqiu Wang, Jingjing Sun, Wuchang Zhang, Jiang Li, Zhonglong Liu, Zhiyuan Zhang, Hanlin Zeng, Yue He
          </td>
          <td>2025-02-27</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9cc09a30e02fe7681c9c9c4146bae9713fa5e17" target='_blank'>
              Integrative CRISPR Screens and RNA-Omics Discover an Essential Role for PUF60-3’ Splice Site Interactions in Cancer Progression
              </a>
            </td>
          <td>
            Alexandra T. Tankka, J. Einstein, Cathy Zhou, Vivian N. Pham, Yuhan Zhang, Jack T. Naritomi, Grady G. Nguyen, Orel Mizrahi, Mark Perelis, Joseph Sarsam, Frederick E. Tan, D. Kaufman, Corina E. Antal, Gene W Yeo
          </td>
          <td>2025-05-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The primary epigenetic alteration in the development of prostate cancer (PC) is aberrant methylation of the promoter region of tumor suppressor genes. Maintaining DNA methylation activity requires ubiquitin-like PHD and RING finger domain-containing protein 1 (UHRF1). Through its SET and RING-associated (SRA) domain, UHRF1 identifies methylated cytosine-phosphate-guanine sequences and binds DNA methyltransferase 1 (DNMT1) directly to replicated DNA foci. Nevertheless, the modality used by UHRF1 to regulate gene networks in PC cells precisely and efficiently is unknown. Research has indicated that epigenetic modification is significantly influenced by the phase separation of epigenetic regulators. Thus, the purpose of this work was to find out if UHRF1 phase separation is a mechanism for PC cells to inherit DNA methylation in a stable manner. We evaluated the correlation between UHRF1 expression level and PC development, as well as used PC cell lines to study the mechanism of UHRF1 phase separation driving the progression of PC. The results of this study demonstrated that UHRF1 formed phase-separated droplets by its SRA motif and intrinsically disordered region 3. Furthermore, UHRF1 formed nuclear condensates at the promoter region of multiple cancer-related genes and recruited DNMT1 to allow epigenetic gene silencing that led to malignant proliferation of PC cells. These results revealed that phase separation of UHRF1 is crucial for PC pathogenesis. Supplementary Information The online version contains supplementary material available at 10.1186/s12964-025-02309-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15af2891867d172d0d1de48706256f4948adb838" target='_blank'>
              UHRF1 phase separation mediates stable inheritance of DNA methylation and promotes the proliferation of prostate cancer cells
              </a>
            </td>
          <td>
            Zhean Li, Yihan Wang, Ziqi Yu, Yujia Luo, Yixiang Zhao, Erfei Xue, Junfeng Zhan, Huimin Lu, Fei Sun
          </td>
          <td>2025-06-20</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27a802d056910b5b0c0dd0ae061c28d9e6b15a90" target='_blank'>
              Lamin A/C Deficiency Drives Genomic Instability and Poor Survival in Small-Cell Lung Cancer through Increased R-loop Accumulation
              </a>
            </td>
          <td>
            Christopher W. Schultz, Sourav Saha, A. Dhall, Yang Zhang, Parth Desai, L. Pongor, D. Scheiblin, Valentin Magidson, Yilun Sun, Christophe E Redon, Suresh Kumar, Manan Krishnamurthy, Henrique B. Dias, V. Aksenova, Elizabeth Giordano, N. Takahashi, Michael Nirula, Mohit Arora, Chiori Tabe, M. Thomas, Rajesh Kumar, Yasuhiro Arakawa, Ukhyun Jo, Beverly A. Teicher, M. Aladjem, Stephen Lockett, M. Dasso, Y. Pommier, A. Sharma, Anish Thomas
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="
 e18058


 Background:
 HNCs are a diverse group of malignancies characterized by genetic, epigenetic, and proteomic alterations that drive tumor progression, treatment resistance, and poor outcomes.
 Methods:
 Molecular profiling of 273 HNC samples was conducted using a semiconductor-based next-generation sequencing (NGS) platform at Datar Cancer Genetics. A subset of 106 cases underwent targeted transcriptomic analysis of 20,802 genes. Additionally, PD-L1 (22C3) IHC, microsatellite instability (MSI), and tumor mutational burden (TMB) were analyzed in selected samples.
 Results:
 Transcriptomic analysis highlighted significant alterations in critical cellular pathways, including cell cycle control, apoptosis, DNA repair mechanisms, and transcription regulation. Among the most dysregulated genes were
 DLGAP5
 ,
 CASP14
 ,
 BCL2
 , and
 JUN
 . Copy number profiling in a subset (N=74) revealed significant chromosomal instability, with 34% of cases showing losses on specific chromosome arms and 9% exhibiting gains, notably in regions such as
 4p, 19p
 and
 1p.
 Mutations were predominantly observed in tumor suppressor genes, with a lower frequency in oncogenes. Frequently altered genes include
 TP53
 (63%),
 CDKN2A
 (19%),
 PIK3CA
 (12%),
 TERT
 (15%),
 HRAS
 (6%) and
 EGFR
 (1%).
 HRAS
 mutations offer potential for targeted therapy with tipifarnib, a farnesyltransferase inhibitor.
 EGFR
 alterations, though rare, have proven challenging to effectively target with
 EGFR
 -specific therapies such as cetuximab or tyrosine kinase inhibitors, underscoring the critical need for innovative strategies to exploit this target. Amplifications were observed in
 MYC
 (9%),
 CCND1
 (8%),
 FGF19
 (6%),
 FGF3
 (4%) and
 FGF4
 (4%). Targetable amplifications were seen in
 EGFR
 (6%),
 ERBB2
 (2%) and
 MET
 (1%), offering potential opportunities for personalized therapies.
 MET
 amplifications are also linked to aggressive tumor behavior. Fusion was a relatively less common event, observed in 9.5% cases, including one case each of targetable
 BRAF-MRPS33
 and
 FGFR1-PLAG1
 fusion. High TMB (≥10 muts/mb) was seen in 21% cases (Median 12, range 0-28). None of the tumors showed MSI-high status (n=63). Higher PD-L1 positivity was observed in TMB high samples (76% vs 59%), with PD-L1 22-C3 CPS ≥1 in 66% (50/ 76) and PD-L1 28-8 TPS ≥1 in 46% (36/ 78).
 Conclusions:
 This study underscores the genetic heterogeneity of HNCs, pointing to deregulated pathways involved in cell cycle control, apoptosis, and DNA repair as promising targets for future therapeutic development and personalized treatment strategies. Also this study points towards a subset of patients who may derive a greater benefit with immunotherapy, based on TMB/PDL1.


 ESCAT TIER distribution of the molecular alterations detected in head and neck cohort (N=273).




 TIERs
 Total Cases
 % occurrence




 IC
 6
 2%


 IIA
 21
 8%


 IIIA
 85
 31%


 IIIA
 1
 0%


 IIIB
 57
 21%


 IVA
 180
 66%


 IVB
 71
 26%


 X
 142
 52%




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a204052c6edd84c3f9c00c6d7bada097fe56677e" target='_blank'>
              Molecular profiling of head and neck cancers (HNCs): Genetic heterogeneity, immune landscape, and therapeutic targets.
              </a>
            </td>
          <td>
            R. K. Choudhary, S. Limaye, Padman Vamadevan, Vivek Agarwala, Vikas Goswami, H. P, Vijay Anand Reddy, Niyati Krunal Shah, Darshit Kalpeshkumar Shah, Shivam Shingla, Dhanashri Ahire, Vinayak Rao, R. Gosavi, Priyanka Desale, R. Datar, D. Patil
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Emerging evidence suggests that dysregulated RNA-binding proteins (RBPs) are associated with a wide variety of cancers. However, the exact roles and pathways of RBPs in the tumorigenesis of hepatocellular carcinoma (HCC), the most common subtype of liver cancer, remain largely unknown. Here, we systematically searched for altered RBP candidates in HCC through multi-omics data integrative analyses and identified that GPATCH4 gene is amplified in >70% HCC patients and its high expression predicts poor prognosis. We mapped the in vivo RNA binding sites of GPATCH4 by iCLIP-seq and characterized that GPATCH4 primarily bound ribosomal RNA (rRNAs). GPATCH4 promoted HCC cell proliferation and transformation both in vitro and in vivo through increasing rRNA transcription and global protein synthesis. GPATCH4 is mainly localized in the nucleolus and helps to unwind RNA loops formed at the rDNA through interacting with DDX21 via its C-terminal intrinsically disordered region. Removal of accumulated R-loops induced by GPATCH4 depletion rescued decreased rRNA transcription and cell proliferation. Taken together, we characterized the understudied GPATCH4 as an RBP with oncogenic function in HCC and revealed a new mechanism by which GPATCH4 functions as a regulator of nucleolar R-loops to control rRNA transcription through interacting with DDX21.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c76262c7c41bf119badd0461b097958f0ba08c39" target='_blank'>
              GPATCH4 functions as a regulator of nucleolar R-loops in hepatocellular carcinoma cells
              </a>
            </td>
          <td>
            Yi-Ming Zhao, Yan Jiang, Jin-Zhu Wang, Shang Cao, Hong Zhu, Wei-Kang Wang, Jian-Cheng Yu, Jiaquan Liu, Jingyi Hui
          </td>
          <td>2025-05-22</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 e15108


 Background:
 Children with high-risk and relapsed neuroblastoma (NB) need improved therapies, and recurrent cytogenetic abnormalities, such as
 MYCN
 oncogene amplification, represent candidate therapeutic targets.
 MYCN
 amplification and increased
 MYCN
 expression drive deregulated hyper-transcription that leads to development and growth of NB tumors, and
 MYCN
 amplifications, which can be found both within the linear genome (HSR) and on circular extrachromosomal DNA (ecDNA), are associated with significantly worse survival rates for children with NB.
 MYCN
 overexpression has been linked to an increase in replication stress (RS), and RS and subsequent genome instability are important drivers of tumor initiation and progression. Flap Endonuclease 1 (FEN1), a non-essential DNA replication enzyme, was identified as a synthetic lethal target in
 BRCA1
 /
 2
 -deficient cancers via induction of RS. Recent success of targeting RS-elevated cancers with replication enzyme inhibitors opens a new avenue to target
 MYCN
 -amplified NB.
 Methods:
 Associations of gene expression with patient survival and prognostic features were performed on available neuroblastoma tumor databases using the R2 Genomics Analysis and Visualization Platform. The efficacy of FEN1 inhibition was assessed using live cell imaging and cell viability assays, comparing results in
 MYCN
 -amplified to -nonamplified NB cells and in NB cells with inducible
 MYCN
 expression and repression. Mechanisms of cell death and impacts on replication stress in cells treated with FEN1 inhibitors were evaluated by Western blots.
 Results:
 We have found that
 FEN1
 expression levels are associated wtih NB patient outcomes and with features of high-risk disease, including tumor stage and
 MYCN
 amplification. FEN1 inhibition was effective against NB cells and was significantly more effective in
 MYCN
 -amplified NB cells causing reduced cell growth and viability. Increased
 MYCN
 expression also led to increased sensitivity to FEN1 inhibition, and reduced MYCN expression reduced sensitivity. FEN1 inhibition led to the induction of apoptosis but not necroptosis in NB cells and responses to FEN1 inhibition were associated with markers of replication stress, including activation of the ATR-CHK1 and ATM-CHK2 pathways.
 Conclusions:
 We have discovered that
 MYCN
 -amplified NB cells are hypersensitive to FEN1 inhibition, suggesting that FEN1 inhibition may be a promising therapeutic strategy for children with high-risk and relapsed neuroblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa405fa9ce38d24eb7641c652f857fc594ac4869" target='_blank'>
              Mechanisms of efficacy of endonuclease inhibition in neuroblastoma.
              </a>
            </td>
          <td>
            Carla Sampaio, Eric Wu, Madison Cinelli, Erica Steen, Andrew Shiau, Richard Kolodner, Jean Yin Jen Wang, Peter E. Zage
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumors (ATRTs) are aggressive central nervous system tumors mainly affecting young children. Extensive molecular characterization based on gene expression and DNA methylation patterns has solidly established three major ATRT subtypes (MYC, SHH and TYR), which show distinct clinical features, setting the basis for more effective, targeted treatment regimens. Transcriptional activity of transposable elements (TEs), like LINE1s and LTRs, is tightly linked with human cancers as a direct consequence of lifting epigenetic repression over TEs. The sole recurrent biallelic loss-of-function mutation in SMARCB1 in ATRTs, a core component of the SWI/SNF chromatin remodeling complex, raises the question of how TE transcription contributes to ATRT development. Here, we comprehensively investigate the transcriptional profiles of 1.9M LINE1 and LTR elements across ATRT subtypes in primary human samples. We find TE transcription profiles are unique, allowing sample stratification into ATRT subtypes. The TE activity signature in ATRT-MYC subtype is unique, setting these tumors apart from SHH and TYR ATRTs. More specifically, ATRT-MYC shows broadly reduced transcript levels of LINE1 and ERVL-MaLR subfamilies. ATRT-MYC is also unique in having significantly less LTR and LINE1 loci with bidirectional promoter activity. Furthermore, we identify 849 differentially transcribed TEs in primary samples, which are predictive towards established ATRT-SHH and-MYC cell line models. In summary, including TE transcription profiles into the molecular characterization of ATRTs might reveal new tumor vulnerabilities leading to novel therapeutic interventions, such as immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88b1746e8b34806ab31721a10a88339f9bbc0904" target='_blank'>
              Expression of LTR and LINE1 transposable elements defines atypical teratoid/rhabdoid tumor subtypes
              </a>
            </td>
          <td>
            M.V. Hamann, Shweta Godbole, Maisha Adiba, Sabrina M. Leddy, Jelena Navolić, Ghazaleh Tabatabai, Daniel J Merk, Ulrike C. Lange, J. Neumann
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71c01f9fcd4978a181214c5cc7b38fe087c3cef1" target='_blank'>
              ZCCHC4 Promotes Translation of Replication-dependent Histone mRNAs by Recruiting Cytoplasmic eIF3 complex
              </a>
            </td>
          <td>
            Ruiqi Wang, Xiaoyan Shi, Yangyi Zhang, Yuci Wang, Yanlan Cao, Rui Feng, Chen Chen, Yingchun Zhang, Hao Chen, Honghui Ma
          </td>
          <td>2025-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d8533a84bf23dfa93f9b981af3fd2fcfd03e25b" target='_blank'>
              AURKA inhibition amplifies DNA replication stress to foster WEE1 kinase dependency and synergistic antitumor effects with WEE1 inhibition in cancers
              </a>
            </td>
          <td>
            Jong Woo Lee, Jackie Shi, Julian Barrantes, Pratima Chaurasia, Sundong Kim, S. Cruz-Gomez, Cindy Yang, Jianlei Gu, Hongyu Zhao, Dejian Zhao, Benjamin Schiff, Ansley Roche, Elizabeth B. Perry, Jeffrey P. Townsend, Gary M. Kupfer, Katerina Politi, E. Golemis, Barbara Burtness
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Genome instability (GIN) is a cell pathology linked to cancer promotion and tumor evolution. Transcription is an essential cellular process but also a potential source of DNA damage and GIN. Transcription-replication conflicts (TRCs) are a predominant source of GIN, and defective TRC resolution may seriously compromise genome integrity. Importantly, chromatin dynamics helps orchestrate the response to TRCs to preserve genome integrity. Multiple epigenetic deficiencies have been shown to cause transcription-induced replication stress, resulting in DNA breaks and mutations. Consistently, chromatin alterations are frequent in cancer and correlate with increased mutation burden at TRC sites in tumors. Here, we review our current knowledge of TRC processing, the consequences of its dysfunction, and its relevance in cancer. We focus on the interplay between the DNA damage response (DDR) and chromatin dynamics and discuss the clinical potential of targeting TRCs as anticancer strategies and drugging the associated epigenetic signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/633badfaea4225294a91ff97893c1fea15ce4095" target='_blank'>
              Transcription-Replication Conflicts: Unlocking New Frontiers in Cancer.
              </a>
            </td>
          <td>
            Aleix Bayona-Feliú, A. Aguilera
          </td>
          <td>2025-06-09</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Ploidy determination across the genome has been challenging for low-pass-WGS tumor-only samples. We present BACDAC, a method that calculates tumor ploidy down to 1.2X effective tumor coverage. Allele fraction patterns displayed in the Constellation Plot verify tumor ploidy and reveal subclonal populations. BACDAC outputs a metric, 2N+LOH, that when combined with ploidy better distinguishes near-diploid from high-ploidy tumors. Validated using TCGA, BACDAC had good agreement with other methods and 88% agreement with experimental methods. Discrepancies occur mainly when BACDAC predicts diploidy with subclones rather than high-ploidy. Applied to 653 low-pass-WGS samples spanning 12 cancer subtypes, BACDAC calls 40% as high-ploidy. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03599-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4a5801328a2687dfdef502362bf6b982a0712bd" target='_blank'>
              Tumor ploidy determination in low-pass whole genome sequencing and allelic copy number visualization using the Constellation Plot
              </a>
            </td>
          <td>
            Sarah H. Johnson, J. Smadbeck, Roman M. Ženka, Michael T. Barrett, Athanasios Gaitatzes, Arnav Solanki, Angela B. Florio, M. Borad, John C. Cheville, G. Vasmatzis
          </td>
          <td>2025-05-20</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Osimertinib, a third-generation EGFR tyrosine kinase inhibitor (EGFR-TKI), has dramatically transformed the treatment landscape for patients with EGFR-mutant NSCLC. However, the long-term success of this therapy is often compromised by the onset of acquired resistance, with non-genetic mechanisms increasingly recognized as pivotal contributors. Here, we exploit a multi-omics approach to profile genome-wide chromatin accessibility and transcriptional landscapes between drug sensitive and resistant EGFR-mutant cells. Our findings reveal a robust concordance between epigenetic regulome and transcriptomic changes that characterize the osimertinib resistant state. Through CRISPR-based functional genomics screen targeting epigenetic regulators and transcription factors, we uncover a critical regulatory network featuring key members of the NuRD and PRC2 complexes that mediate resistance. Most critically, our screen identifies FOSL1 and JUN, two subunits of the AP-1 transcription factor within this network, as the most significant hits. Mechanistically, we demonstrate that cis-regulatory elements exhibiting altered chromatin accessibility in the resistant state are enriched for cognate AP-1 binding motifs, enabling AP-1 to orchestrate a gene expression program that underpins the druggable MEK/ERK signaling axis, potentially enhancing cell viability and fitness of resistant cells. Importantly, genetic depletion or pharmacological inhibition of AP-1 reinstates cellular and molecular sensitivity, and reverts resistance-associated phenotypes, such as epithelial-to-mesenchymal transition, upon anti-EGFR rechallenge by repressing AKT and ERK signaling. These findings provide novel insights into the epigenetic and transcriptional control of osimertinib resistance in EGFR-mutant NSCLC, highlighting AP-1 as a targetable vulnerability of resistance-related hallmarks and offering a promising avenue for developing resistance reversal strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b98b05c64d117a31d26404e164fc0fbed4525adc" target='_blank'>
              Integrative multi-omics identifies AP-1 transcription factor as a targetable mediator of acquired osimertinib resistance in non-small cell lung cancer
              </a>
            </td>
          <td>
            B. Dayanc, Sude Eris, Nazife Ege Gulfirat, Gulden Ozden-Yilmaz, E. Cakiroglu, Ozlem Silan Coskun Deniz, G. Karakülah, Serap Erkek-Ozhan, Serif Senturk
          </td>
          <td>2025-05-25</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Mapping the spatial organization of DNA-level somatic copy number changes in tumors can provide insight to understanding higher-level molecular and cellular processes that drive pathogenesis. We describe an integrated framework of spatial transcriptomics, tumor/normal DNA sequencing, and bulk RNA sequencing to identify shared and distinct characteristics of an initial cohort of eleven gliomas of varied pathology and a replication cohort of six high-grade glioblastomas. We identify focally amplified extrachromosomal DNA (ecDNA) in four of the eleven initial gliomas, with subclonal tumor heterogeneity in two EGFR-amplified grade IV glioblastomas. In a TP53-mutated glioblastoma, we detect a subclone with EGFR amplification on ecDNA coupled to chromosome 17 loss of heterozygosity. To validate subclonal somatic aneuploidy and copy number alterations associated with ecDNA double minutes, we examine the replication cohort, identifying MDM2/MDM4 ecDNA subclones in two glioblastomas. The spatial heterogeneity of EGFR and p53 inactivation underscores the role of ecDNA in enabling rapid oncogene amplification and enhancing tumor adaptability under selective pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48c8db15868df338dc48b4c0bf5130f5e0f200b" target='_blank'>
              Resolving spatial subclonal genomic heterogeneity of loss of heterozygosity and extrachromosomal DNA in gliomas
              </a>
            </td>
          <td>
            Michelle G. Webb, Frances Chow, Carmel G. McCullough, Bohan Zhang, John J. Y. Lee, Rania Bassiouni, Norman E. Garrett, Kyle Hurth, J. Carpten, Gabriel Zada, David W. Craig
          </td>
          <td>2025-06-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f6786fefc42e4d77082eb7a0b7de5ec1cad7e96" target='_blank'>
              MITF maintains genome stability in non-melanocytic cell lineages and suppresses Hippo pathway signaling
              </a>
            </td>
          <td>
            Drifa H. Gudmundsdottir, Adrián López García de Lomana, Thejus B. Venkatesh, Kritika Kirty, S. Sigurdsson, Linda Vidarsdottir, R. Dilshat, Erla Sveinbjornsdottir, Snædís Ragnarsdóttir, Daniel H Magnusson, Maria R. Bustos, E. Steingrímsson, Thorkell Gudjónsson, Stefan Sigurdsson
          </td>
          <td>2025-06-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Somatic cells can transform into tumors due to mutations, and the tumors further evolve towards increased aggressiveness and therapy resistance. We develop DiffInvex, a framework for identifying changes in selection acting on individual genes in somatic genomes, drawing on an empirical mutation rate baseline derived from non-coding DNA that accounts for shifts in neutral mutagenesis during cancer evolution. We apply DiffInvex to >11,000 somatic whole-genome sequences from ~30 cancer types or healthy tissues, identifying genes where point mutations are under conditional positive or negative selection during exposure to specific chemotherapeutics, suggesting drug resistance mechanisms occurring via point mutation. DiffInvex identifies 11 genes exhibiting treatment-associated selection for different classes of chemotherapies, linking selected mutations in PIK3CA, APC, MAP2K4, SMAD4, STK11 and MAP3K1 with drug exposure. Various gene-chemotherapy associations are further supported by differential functional impact of mutations pre- versus post-therapy, and are also replicated in independent studies. In addition to nominating drug resistance genes, we contrast the genomes of healthy versus cancerous cells of matched human tissues. We identify noncancerous expansion-specific drivers, including NOTCH1 and ARID1A. DiffInvex can also be applied to diverse analyses in cancer evolution to identify changes in driver gene repertoires across time or space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c7adc2b8c2faa562f62002c5e5407d79d8555" target='_blank'>
              DiffInvex identifies evolutionary shifts in driver gene repertoires during tumorigenesis and chemotherapy
              </a>
            </td>
          <td>
            Ahmed Khalil, F. Supek
          </td>
          <td>2025-05-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="DNA double strand breaks (DSBs) are the effective lesion of cancer radiotherapy and induce gene editing. 53BP1 accumulates at DSBs and is implicated in end joining (EJ) repair, but its influence on DSB repair is distinct from canonical non-homologous end joining (C-NHEJ). We sought to define the genetic interplay of 53BP1 with C-NHEJ, focusing on the DNA-PKcs kinase. We examined Cas9 DSBs, which largely generates blunt DSBs, since blunt DSB EJ is dependent on C-NHEJ. Loss of 53BP1 does not affect blunt DSB EJ, but causes a reduction in such repair in DNA-PKcs deficient cells. In contrast, disrupting 53BP1 and DNA-PKcs, both alone and together, has similar effects on the type of deletion mutation (increase in microhomology deletions). We found similar effects on EJ with RIF1 loss, which is a downstream effector of 53BP1. Thus, 53BP1/RIF1 appear to play a backup role for DNA-PKcs during blunt DSB EJ, but function in the same pathway to suppress microhomology deletions. In contrast, 53BP1 and DNA-PKcs function independently to suppress homology-directed repair. Finally, DNA-PKcs kinase inhibition causes marked radiosensitivity, which is not additive with loss of 53BP1 and RIF1. Altogether, 53BP1/RIF1 and DNA-PKcs show distinct genetic interactions with diverse DSB repair outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbb1f8550e47aed05c0a51b4b294e0030758aa9f" target='_blank'>
              53BP1/RIF1 and DNA-PKcs show distinct genetic interactions with diverse chromosomal break repair outcomes
              </a>
            </td>
          <td>
            Kaela Makins, Metztli Cisneros-Aguirre, F. Lopezcolorado, Jeremy M. Stark
          </td>
          <td>2025-05-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a45fc2e4de0ca1a04af5876a2240acffe2c6ff19" target='_blank'>
              Role of PLK4 inhibition in cancer therapy.
              </a>
            </td>
          <td>
            Kishore Banik, Thomas J Hayman
          </td>
          <td>2025-06-13</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Objectives: Aneuploidy is near-ubiquitous in cancer and can decrease chemotherapy efficacy while also sensitizing cells to other drugs. Methods: To systematically identify treatment strategies that target aneuploid cancers, data were integrated from The Cancer Genome Atlas (TCGA; 10,967 samples, 16,948 aneuploidy events) and the Broad Institute’s Profiling Relative Inhibition Simultaneously in Mixtures (PRISM) screen of 578 cancer cell lines and 4518 compounds. Results: Our analyses uncovered 37,720 significant positive and negative associations linking specific aneuploidies and treatments with patient prognosis or cell viability. Within TCGA data, 22 treatments correlated with improved 5-year survival for specific aneuploid cancers, whereas 46 were linked to worse outcomes. A complementary analysis of PRISM identified 17,946 compound–aneuploidy associations and 16,189 mechanism of action (MOA)–aneuploidy associations. Pathway-altering compounds that selectively reduce viability in cells with aneuploidy profiles were discovered, including an unexpectedly prominent number of glucocorticoid receptor agonists. Conclusions: This integrated dataset provides a resource for designing therapeutic decision hypotheses, identifying drug-repurposing opportunities, and informing future studies aimed at targeting aneuploidy-induced vulnerabilities in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/811f8e93329a18a966c2c968f4a1e0431dca5e53" target='_blank'>
              Half the Chromosome It Used to Be: Identifying Cancer Treatments Targeting Aneuploid Losses
              </a>
            </td>
          <td>
            Andrew O. Disharoon, Joe R. Delaney
          </td>
          <td>2025-06-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The tumorigenesis of small intestinal neuroendocrine tumors (siNETs) is not understood and comprehensive genomic and transcriptomic data sets are limited. Therefore, we performed whole genome and transcriptome analysis of 39 well differentiated siNET samples. Our genomic data revealed a lack of recurrent driver mutations and demonstrated that multifocal siNETs from individual patients can arise genetically independently. We detected germline mutations in Fanconi anemia DNA repair pathway (FANC) genes, involved in homologous recombination (HR) DNA repair, in 9% of patients and found mutational signatures of defective HR DNA repair in late-stage tumor evolution. Furthermore, transcriptomic analysis revealed low expression of the transcriptional repressor REST. Summarizing, we identify a novel common transcriptomic signature of siNETs and demonstrate that genomic alterations alone do not explain initial tumor formation, while impaired DNA repair likely contributes to tumor evolution and represents a potential pharmaceutical target in a subset of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44f7cb964c906b47270fb2a184506e3a4a4d940" target='_blank'>
              Small intestinal neuroendocrine tumors lack early genomic drivers, acquire DNA repair defects and harbor hallmarks of low REST expression
              </a>
            </td>
          <td>
            Felix Bolduan, Niklas Müller-Bötticher, Olivia Debnath, I. Eichhorn, Yvonne Giesecke, Alexandra Wetzel, S. Sahay, Tomasz Zemojtel, Marten Jaeger, Ute Ungethuem, Christoph Roderburg, C. A. Kunze, Annika Lehmann, David Horst, Frank Tacke, Roland Eils, Bertram Wiedenmann, Michael Sigal, N. Ishaque
          </td>
          <td>2025-05-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c62dd3d2788ffe055d52a584d9b884866f23c266" target='_blank'>
              Delayed indel formation after Cas9 cleavage promotes mosaicism and aneuploidy in human embryos
              </a>
            </td>
          <td>
            Alex Robles, Julie Sung, Stepan Jerabek, Jishnu Talukdar, Diego Marin, Jia Xu, Nathan Treff, Dieter Egli
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Segmental duplications (SDs) are blocks of genomic DNA with high sequence homology that are hotspots for chromosomal rearrangements, coinciding with copy-number and single-nucleotide variations in the population. SDs could represent unstable genomic regions that are susceptible to somatic alterations in human cancers. Here, we aimed to elucidate the genomic locations of SDs in relation to cancer-related genes and their propensity for somatic alterations in cancer. The analysis showed that tumor suppressor genes (TSGs) were less associated with SDs compared to non-cancer genes in nearly all mammalian species. TSGs with SDs were larger in size in humans but only modestly conserved among mammals. In humans, the proportion of non-cancer genes with SDs decreased as the gene age increased. However, for TSGs, a loss of association with SDs was apparent among young genes. Pan-cancer analysis revealed that TSGs with SDs were more prone to deletions and structural variations independently of gene size. Re-analysis of publicly available experimental data further revealed that genes with SDs tended to replicate late and were more likely to undergo the error-prone mitotic DNA synthesis upon replication stress. In conclusion, the loss of SDs from TSGs during mammalian evolution protects against tumor formation by reducing somatic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7dee62f17cffda68dd501cc4ece25c8e474528a" target='_blank'>
              Tumor Suppressor Genes with Segmental Duplications are Prone to Somatic Deletions and Structural Variations.
              </a>
            </td>
          <td>
            Ziheng Huang, Lin Zhang, Huarong Chen, Xiaodong Liu, Likai Tan, Dan Huang, Yingzhi Liu, Yushan Wang, Xinyi Zhang, A. S. Cheng, Maggie Haitian Wang, Wei Kang, K. To, Jun Yu, Ho Ko, Le Yu, Sunny H Wong, Tony Gin, M. T. Chan, Xiansong Wang, W. K. K. Wu
          </td>
          <td>2025-05-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 8523


 Background:
 Driver-negative lung adenocarcinoma (DN-LUAD) without actionable genomic alterations (AGAs) has a poor prognosis. A deeper understanding of its molecular background, particularly across the entire genome, is crucial for improving risk assessment and therapeutic development.
 Methods:
 We performed deep whole-genome sequencing (WGS; tumor depth: 100–120x, normal depth: 30x, paired-end reads: 150 bp) and RNA sequencing using fresh-frozen tissues from four institutions. DN-LUAD was defined as LUAD without known actionable mutations or gene fusions. We analyzed somatic mutations, copy number alterations (CNAs), and structural variations (SVs). Tumors were classified as whole-genome doubling (WGD) if more than half of the autosomal tumor genome showed at least two copies in the major copy numbers of somatic cells. HRDetect score ≥0.7 was used to determine the presence of homologous recombination DNA repair deficiency (HRD) (Davies H, et al. Nat Med. 2017). Gene Set Enrichment Analysis (GSEA) was used for pathway analysis.
 Results:
 Among the 745 patients (pts), 517 were classified as having DN-LUAD. WGS identified AGAs undetected by whole-exome sequencing (WES) in 33 (4.4%) pts (
 EGFR
 13,
 KRAS
 7,
 BRAF
 5,
 ERBB2
 4,
 MET
 4, and
 HRAS
 1). The frequently observed genomic alterations in DN-LUAD are shown in the Table. WGD was observed in 62.1% (n = 321) of DN-LUADs and was significantly associated with higher TMB, CNA, and SV burden. Tumor suppressor gene (TSG) mutations in
 TP53
 ,
 STK11
 , and
 KEAP1
 , as well as
 CDKN2A
 copy number loss and SVs, were significantly more frequent in WGD pts. HRD was identified in 19 DN-LUAD pts (3.7%), with a significantly higher frequency in the WGD group (WGD vs. without WGD; 4.98% vs. 1.5%, p = 0.016). GSEA showed significant (q < 0.0001) upregulation of cell cycle pathways (E2F targets, G2M checkpoint, and MYC targets) and downregulation of immune pathways (allograft rejection and interferon-gamma response) in DN-LUADs with WGD.
 Conclusions:
 This largest-ever WGS study identified AGAs undetectable by WES and uncovered a subgroup of DN-LUAD characterized by increased genomic instability driven by multiple TSG alterations associated with WGD, which was also more likely to exhibit HRD. WGD was associated with the upregulation of cell cycle pathways and the downregulation of immune pathways. These findings highlight the critical role of WGS in elucidating the pathogenesis of DN-LUAD.





 AllN = 517
 WGDn = 321
 Without WGDn = 196

 p





 Median TMB (Mutations/Mb)
 6.6
 10.2
 3.0
 <0.0001


 Median No. of CNAs
 101
 117
 77
 <0.0001


 Median No. of SVs
 185
 244
 103
 <0.0001



 TP53

 381 (74)
 277 (86)
 104 (53)
 <0.0001



 SMARCA4

 59 (11)
 43 (13)
 16 (8)
 0.094



 STK11

 56 (11)
 42 (13)
 14 (7)
 0.049



 CDKN2A

 44 (9)
 33 (10)
 11 (6)
 0.092



 KEAP1

 40 (8)
 34 (11)
 6 (3)
 0.003



 CDKN2A
 loss

 106 (21)
 78 (24)
 28 (14)
 0.009



 MET
 amplification

 44 (9)
 34 (11)
 10 (5)
 0.045



 CDKN2A

 135 (26)
 95 (30)
 40 (20)
 0.028



 FHIT

 125 (24)
 99 (31)
 26 (13)
 <0.0001




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16b5bbd4f2e96da01654006c7177d2c691759a6f" target='_blank'>
              Whole-genome and transcriptome landscape of actionable driver-negative lung adenocarcinoma.
              </a>
            </td>
          <td>
            Masahiro Torasawa, K. Shiraishi, Akifumi Mochizuki, Shingo Matsumoto, Junko Hamamoto, Hirokazu Matsushita, Issei Imoto, Hiroyuki Yasuda, Kazuhisa Takahashi, Yuichi Shiraishi, Masahiro Tsuboi, Koichi Goto, Shun-ichi Watanabe, R. Hamamoto, T. Kohno
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 e17121


 Background:
 Chromosome arm 8q is frequently amplified in prostate cancer, harboring genes that drive tumor progression, therapy resistance, and poor outcomes. Key loci include
 MYC
 , which regulates cell proliferation;
 AR
 , central to prostate cancer biology and resistance; and
 RAD21
 , essential for chromosomal stability. While the prognostic implications of 8q gains have been explored, their role in driving racial disparities in prostate cancer outcomes remains underexplored. This study investigates chromosome 8q amplification as a prognostic biomarker, examining its differential prevalence and impact across racial groups to elucidate its contribution to disparities in disease progression and patient survival.
 Methods:
 Using cBioPortal's clinical and genomic data from Memorial Sloan Kettering Cancer Center, this work examined two datasets from 2021 and 2024. Comprehensive genomic profiling included copy number alterations, mutational co-occurrence of important 8q genes, and mutational analysis. Multivariate Cox regression and Kaplan-Meier survival analysis were conducted to evaluate the impact of 8q amplifications on overall survival (OS) and disease outcomes, stratified by racial demographic groups and disease stages.
 Results:
 In this cohort of 1,577 prostate cancer patients, 652 (41%) exhibited chromosome 8q amplification, which correlated with aggressive tumor phenotypes. The most common amplifications were observed in PTEN (21%), FOXA1 (17%), AR (16%), and MYC (7%),, with notable co-occurrences observed between MYC-RAD21, MYC-NBN, and PRDM14-ELOC (
 P
 = <0.001), linking these alterations to genomic instability. Kaplan-Meier analysis showed patients with 8q gain had significantly inferior overall survival (HR = 2.31, 95% CI: 1.99 - 2.69,
 P
 < 0.001, mOS gain vs neutral: 35mo vs 55mo). Black patients exhibited a higher frequency of 8q gain (10% vs. 6.1%,
 P
 = 0.003) and distinct oncogenic profiles, including MYC (12%), RAD21 (7%), and LYN (4%) alterations. While Black patients experienced inferior overall outcomes, chromosome 8q amplification was a consistent predictor of poor prognosis across all racial groups.
 Conclusions:
 This study establishes chromosome 8q amplification as a robust prognostic biomarker in prostate cancer. The integration of clinical and genomic data highlights the potential of 8q amplification to guide personalized therapeutic strategies. These findings underscore the value of incorporating 8q analysis into clinical risk stratification frameworks to enhance treatment decision-making and address racial disparities in prostate cancer outcomes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/844ee49cccb708b6dfbb629567a7cde15db25ac6" target='_blank'>
              Prognostic impact and racial disparities of chromosome 8q gains in prostate cancer: Insights from integrated molecular profiling.
              </a>
            </td>
          <td>
            Chidiebube J. Ugwu, G. Laliotis, U. C. Okere, Nneoma Ubah, Elvis Obomanu, Colton Jones, Muluken Megiso, Nzubechukwu J. Ugochukwu, Gabor Varadi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Background: Genomic structural variation (SV) is a recognized property of cancer cells, contributing to genomic instability and oncogenesis. SV breakpoints and rearrangement patterns are often tumor-specific and can reflect underlying tumor biology. The landscape and implications of SV in breast cancer has been incompletely characterized. Furthermore, the utility of using SVs for prospective circulating tumor DNA (ctDNA) detection and monitoring in early breast cancer (EBC) has not been evaluated. Methods: SV was evaluated through whole genome sequencing (WGS) analysis in two independent breast cancer datasets: (1) The 100,000 genomes project [n=3044 patients; Genomics England, GEL], made possible through access to data and findings in the National Genomic Research Library via the Genomics England Research Environment and (2) a cohort of patients with EBC treated with neoadjuvant chemotherapy [ctDNA evaluation in early breast cancer (TRACER; n=210 patients) Princess Margaret Cancer Centre, Canada]. SV burden and type was evaluated in the GEL dataset using MANTA and in TRACER using an in-house pipeline (SV size cut-off of >50 bp). Survival correlates within the GEL dataset were evaluated using Cox proportional hazard models with cases censored after 6 years. The performance of an SV-based ctDNA assay was evaluated in BT-474 (cell line, contrived samples) and in FFPE tumors (TRACER, clinical samples) using shallow depth (∼15X) WGS followed by SV detection (up to 16) via proprietary multiplex digital PCR. Longitudinal ctDNA detection was performed in the TRACER cohort. Results: SV was common across all breast cancer subtypes (GEL data; median SV burden: 108, range: 4-1448; median aggregate copy number of the top 16 SV: 57, range: 16-160). A higher proportion of inversions were seen in HER2-positive tumors, while deletions were common in TNBC. In a multivariate model (including clinical stage, subtype, and tumor mutational burden), there was a significant association between higher SV count and worse overall survival (OS) in ER+/HER2- breast cancer (HR: 2.31, p=0.0207). Patients with ER+/HER2- breast cancer who experienced a clinical recurrence had higher SV copy number in the top 16 variants than those who did not (p<0.0001). In silico analyses demonstrated that personalized SV-based ctDNA panels (fingerprints) could be successfully designed for 97.1% of GEL cases. To assess the characteristics of an SV-based ctDNA assay, a limit of detection (LoD) study was performed with BT-474 using contrived cfDNA (70 ng). The LoD95 was estimated at 0.00052% tumor fraction (5 PPM) with variants detected as low as 1 in 10 million (0.00001% or 0.1 PPM in 31% of cases). A specificity of 100% was seen in 134 healthy donors using 24 different fingerprints (assessment of 1600 SVs). In an initial cohort of 55 patients (TRACER; ER+/HER2-:19, HER2+:23, TNBC:13), shallow WGS and fingerprint design were successful in all patients. The median number of SVs was 336 (range: 73-1345) with the SV type distributed in a similar fashion (more inversions in HER2+ tumors, deletions in TNBC). A trend towards higher SV copy number based on tumor-only WGS was also seen in those with ER+/HER2- disease who experienced subsequent recurrence (TRACER; SV in no recurrence vs. recurrence: 54.3 vs. 134.1, p=0.099). Shallow WGS and SV-based ctDNA assay design is underway for additional patients in the TRACER cohort. Conclusion: SVs are prevalent in breast cancer and associated with prognosis. Personalized SV-based panels permitted ultrasensitive ctDNA detection with high sensitivity and specificity in contrived samples, supporting assay feasibility for early breast cancer. Further analysis of the prognostic impact of SV-burden and type as well as on-treatment and adjuvant ctDNA detection in patients with EBC (TRACER) using SV tracking will be presented at the meeting.
 Citation Format: Mitchell Elliott, Karen Howarth, Sasha Main, Jesús Fuentes Antrás, Philippe Echelard, Aaron Dou, Eitan Amir, Michelle B. Nadler, Elizabeth Shah, Celeste Yu, Scott Bratman, Taylor Bird, June Roh, Elza C. de Bruin, Christopher Rushton, Sofia Birkeälv, Miguel Alcaide, Lucia Oton, Sergii Gladchuk, Yilun Chen, Anthony George, Girish Putcha, Samuel Woodhouse, Philippe L. Bedard, Lillian L. Siu, Hal K. Berman, David W. Cescon. Somatic Structural Variation in Breast Cancer and its Application in Longitudinal Analysis of Circulating Tumor DNA in Early Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS9-05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8337c1a084ec3751caa33b2040b1336acad4d2d" target='_blank'>
              Abstract PS9-05: Somatic Structural Variation in Breast Cancer and its Application in Longitudinal Analysis of Circulating Tumor DNA in Early Breast Cancer
              </a>
            </td>
          <td>
            M. Elliott, Karen Howarth, S. Main, J. F. Antrás, Philippe Echelard, Aaron Dou, Eitan Amir, M.B. Nadler, Elizabeth Shah, Celeste Yu, Scott V. Bratman, Taylor Bird, June Roh, Elza C de Bruin, Christopher Rushton, Sofia Birkeälv, M. Alcaide, L. Oton, Sergii Gladchuk, Yilun Chen, A. George, Girish Putcha, Samuel Woodhouse, P. Bedard, Lillian L. Siu, H. Berman, D. Cescon
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="During cancer development, mutations promote gene expression changes that cause transformation. Leukemia is frequently associated with aberrant HOXA expression driven by translocations in nucleoporin genes or KMT2A, and mutations in NPM1. How disparate mutations converge on this regulatory pathway is not understood. Here we demonstrate that mutant NPM1 (NPM1c) forms nuclear condensates in multiple human cell lines, mouse models, and primary patient samples. We show NPM1c phase separation is necessary and sufficient to coordinate the recruitment of NUP98 and KMT2A to condensates. Through extensive mutagenesis and pharmacological destabilization of phase separation, we find that NPM1c condensates are necessary for regulating gene expression, promoting in vivo expansion, and maintaining the undifferentiated leukemic state. Finally, we show that nucleoporin and KMT2A fusion proteins form condensates that are biophysically indistinguishable from NPM1c condensates. Together, these data define a new condensate underlying leukemias that we term coordinating bodies (C-bodies), and propose C-bodies as a therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0742ebc47874d0a681b5ec4cb75157b2b57840b5" target='_blank'>
              Nuclear Phase Separation Drives NPM1-mutant Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Gandhar K. Datar, Elmira Khabusheva, Archish Anand, Joshua Beale, Marwa Sadek, Chun-Wei Chen, Evdokiia Potolitsyna, Nayara Alcantara-Contessoto, Guangyuan Liu, Josephine De La Fuente, Christina Dollinger, A. Guzman, Alejandra G Martell, Katharina Wohlan, A. Maiti, N. Short, S. S. Yi, V. Andresen, Bjørn T. Gjertsen, B. Falini, R. E. Rau, L. Brunetti, Nidhi Sahni, M. Goodell, Joshua A. Riback
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b48b921317e55f0a7a7e7ce1f3c412ed182dc30b" target='_blank'>
              DNA2 and FANCM function in two distinctive pathways in disrupting TERRA R-loops and suppressing replication stress at ALT telomeres
              </a>
            </td>
          <td>
            Ashwin Ragupathi, Heba Z. Abid, Yun Chen, Rongwei Zhao, Settapong T. Kosiyatrakul, Derin I. Yetil, Julia Neiswander, Rachel Feltman, Shannon Thomas, Benjamin Yusupov, Manrose Singh, Li Zheng, Binghui Shen, Huaiying Zhang, H. C. Chu, Carl L. Schildkraut, Ming Xiao, Dong Zhang
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Diffuse gliomas are the commonest malignant primary brain tumour in adults. Herein, we present analysis of the genomic landscape of adult glioma, by whole genome sequencing of 403 tumours (256 glioblastoma, 89 astrocytoma, 58 oligodendroglioma; 338 primary, 65 recurrence). We identify an extended catalogue of recurrent coding and non-coding genetic mutations that represents a source for future studies and provides a high-resolution map of structural variants, copy number changes and global genome features including telomere length, mutational signatures and extrachromosomal DNA. Finally, we relate these to clinical outcome. As well as identifying drug targets for treatment of glioma our findings offer the prospect of improving treatment allocation with established targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba058669a48e25adcfe1a09c6668631baae8f882" target='_blank'>
              Genomic landscape of diffuse glioma revealed by whole genome sequencing
              </a>
            </td>
          <td>
            B. Kinnersley, Josephine Jung, A. Cornish, D. Chubb, Ross Laxton, A. Frangou, Andreas J. Gruber, Amit Sud, G. Caravagna, A. Sottoriva, David C Wedge, Thomas C. Booth, S. Al-Sarraj, Sam Lawrence, Erminia Albanese, Giulio Anichini, David Baxter, Alexandros Boukas, Yasir A. Chowdhury, Pietro D'Urso, Robert Corns, Andrew Dapaah, E. Edlmann, Fay Greenway, Paul L Grundy, C. Hill, Michael D. Jenkinson, Sandhya Trichinopoly Krishna, Stuart Smith, S. Manivannan, Andrew J Martin, Samir Matloob, Soumya Mukherjee, Kevin O'Neill, P. Plaha, Jonathan Pollock, Stephen J. Price, Ola Rominiyi, Bobby Sachdev, Fozia Saeed, Saurabh Sinha, Lewis Thorne, I. Ughratdar, Peter Whitfield, A. S. Youshani, H. Bulbeck, Prabhu Arumugam, R. Houlston, K. Ashkan
          </td>
          <td>2025-05-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Background CRISPR interference (CRISPRi), the repurposing of the RNA-guided endonuclease dCas9 as a programmable transcriptional repressor, allows highly specific repression (knockdown) of gene expression. CRISPRi platforms can often have incomplete knockdown, performance variability across cell lines and gene targets, and inconsistencies dependent on the guide RNA sequence employed. Results Here, we explore the combination of novel repressor domains with strong Krüppel-associated box (KRAB) repressors, screening > 100 bipartite and tripartite fusion proteins for their ability to reduce gene expression as CRISPRi effectors. We show that these novel repressor fusions have reduced dependence on guide RNA sequences, better slow cell growth when used to knock down expression of essential genes, and function in either fusion or scaffold modalities. Furthermore, we isolate and characterize a particularly effective CRISPRi platform, dCas9-ZIM3(KRAB)-MeCP2(t), which shows improved gene repression of endogenous targets both at the transcript and protein level across several cell lines and when deployed in genome-wide screens. Conclusions We posit that these novel repressor fusions can enhance the reproducibility and utility of CRISPRi in mammalian cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03640-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/541b174ba857f7a16f7cb6e09012bb05165641ba" target='_blank'>
              Engineering novel CRISPRi repressors for highly efficient mammalian gene regulation
              </a>
            </td>
          <td>
            Andrew J Kristof, Krithika Karunakaran, Christopher Allen, Paula Mizote, Sophie Briggs, Zixin Jian, Patrick Nash, J. Blazeck
          </td>
          <td>2025-06-12</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="TEA domain transcription factor 1 (TEAD1), a Hippo pathway transcription factor important in cellular homeostasis and development, is increasingly implicated in cancer biology. Here, we reveal a novel role for TEAD1 in organizing nuclear condensates, independent of active transcription. Using high-resolution imaging, ChIP-seq, RNA-seq and proximity-based proteomics, we demonstrate that in patient-derived renal cell carcinoma cells, TEAD1 forms micron-sized foci by binding to the heterochromatic pericentromeric regions using its DNA-binding domain. These TEAD1 foci do not mediate transcription but instead serve as depots for excess TEAD1. This contrasts with TEAD1 organization in other genomic regions of both RCC and normal kidney cells, where TEAD1 associates with markers of active transcription. Our findings provide a mechanistic framework for TEAD1’s dual regulatory roles, offering new insights into its contribution to transcriptional dysregulation and tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baf7a61d4683b21760f0a534c0754f40cc6df07c" target='_blank'>
              TEAD1 condensates are transcriptionally inactive storage sites on the pericentromeric heterochromatin
              </a>
            </td>
          <td>
            Jindayi Liang, Yiran Wang, Justin Demmerle, Britney Jiayu He, Christopher J. Ricketts, W. Marston Linehan, Chongzhi Zang, Danfeng Cai
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8869d55f4afe1d8deeba75e4afa7c23281aaeaa0" target='_blank'>
              Landscape and mutational dynamics of G-quadruplexes in the complete human genome and in haplotypes of diverse ancestry
              </a>
            </td>
          <td>
            Nikol Chantzi, Shi Wei. Liew, Aurell Wijaya, Candace Chan, Ioannis Mouratidis, Emilyane de Oliveira Santana Amaral, Yasin Uzun, M. Hemberg, Karen M. Vasquez, C. Kwok, Ilias Georgakopoulos-Soares
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Endometrial cancer (EC) is the most common gynecological malignancy. Although prognosis is favorable for patients with an early-stage disease, those with recurrent or more advanced disease have low response rates to chemotherapy and poor clinical outcomes. Previously, we have shown that DNA repair gene (NEIL3) is required for retaining replication fork integrity during replication stress. Here, we examined whether the overexpression of NEIL3 in endometrial cancer associated with altered genomic instability, tumor immunogenicity and anti-tumor immunity in endometrial tumor. In this study, we show that endometrial cancer patients with tumors that a have high NEIL3 expression associated with worse overall survival (OS) outcomes in patients. In addition, tumor with high NEIL3 expression is associated with high number of mutation and chromosomal instability. Furthermore, NEIL3 expression in EC tumors positively correlated with mutation of DNA polymerase eta (POLE) and TP53 as well as high expression of replicative polymerases genes (POLE, POLD1 and POLA1). In contrast, tumor with high NEIL3 expression exhibit low tumor immunogenicity and poor anti-tumor immune cell infiltration. Our findings may have important clinical implications for utilizing NEIL3 as a potential prognostic biomarker to stratify EC patients and as a target to enhance immunotherapy response in endometrial cancer. However, our NEIL3 overexpression associated observation still requires further experimental-based scientific validation studies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-00393-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/229a73859c8660361d382a29f27f2f93cf0da384" target='_blank'>
              DNA glycosylase (NEIL3) overexpression associated with low tumor immune infiltration and poor overall patient survival in endometrial cancer
              </a>
            </td>
          <td>
            Cristofer Barry, Aashirwad Shahi, Dawit Kidane
          </td>
          <td>2025-05-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Medulloblastoma is the most common malignant brain tumor of childhood. The highest-risk tumors are driven by recurrent Myc amplifications (Myc-MB) and experience poorer outcomes despite intensive multimodal therapy. The Myc transcription factor defines core regulatory circuitry for these tumors and acts to broadly amplify downstream pro-survival transcriptional programs. Therapeutic targeting of Myc directly has proven elusive, but inhibiting transcriptional cofactors may present an indirect means of drugging the oncogenic transcriptional circuitry sustaining Myc-MB.


METHODS
Independent CRISPR-Cas9 screens were pooled to identify conserved dependencies in Myc-MB. We performed chromatin conformation capture (Hi-C) from primary patient Myc-MB samples to map enhancer-promoter interactions. We then treated in vitro and xenograft models with CDK9/7 inhibitors to evaluate effect on Myc-driven programs and tumor growth.


RESULTS
Eight CRISPR-Cas9 screens performed across three independent labs identify CDK9 as a conserved dependency in Myc-MB. Myc-MB cells are susceptible to CDK9 inhibition, which is synergistic with concurrent inhibition of CDK7. Inhibition of transcriptional CDKs disrupts enhancer-promoter activity in Myc-MB and downregulates Myc-driven transcriptional programs, exerting potent anti-tumor effect.


CONCLUSIONS
Our findings identify CDK9 inhibition as a translationally promising strategy for the treatment of Myc-MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51ba77a6e26484b146e2f28eb319b79477cb1029" target='_blank'>
              Targeting processive transcription for Myc-driven circuitry in medulloblastoma.
              </a>
            </td>
          <td>
            L. M. Sobral, Faye Walker, Krishnakumar Madhavan, Elizabeth Janko, Sahiti Donthula, Etienne P. Danis, Pradeep Bompada, Ilango Balakrishnan, Dong Wang, Angela M Pierce, Mary M. Haag, Billie J. Carstens, Natalie J Serkova, Nicholas K Foreman, S. Venkataraman, Bethany L. Veo, Rajeev Vibhakar, Nathan A Dahl
          </td>
          <td>2025-05-15</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c746b03fe346aca4a7ae7979467caf2a477ca" target='_blank'>
              Molecular Phenotypic Plasticity Informs Possible Adaptive Change of Triple-Negative Breast Cancer Cells In Vivo
              </a>
            </td>
          <td>
            Md. Iftehimul, P. Muganda, Robert H. Newman, C. J. Rorie, Scott H. Harrison, Muhammad T. Hossain, Misty D. Thomas, Joseph L. Graves, Dipongkor Saha
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Chromosome arm-level aneuploidies (CAAs) are a common consequence of genomic instability during the evolution of solid tumor and often associated with metastasis and therapy resistance. In breast cancer, chromosome arms 1q and 8q are the frequent amplified regions for CAAs. This study aims to investigate whether critical genes amplified in genomic regions with CAAs contribute to the pathophysiology in triple-negative breast cancer (TNBC). We established multiple long-term culturable TNBC patient-derived cells (PDCs) using spheroid culture technique, which is favorable for the enrichment of cell population with cancer stemness. We performed a genome-wide chromatin immunoprecipitation study for histone H3K27ac and super-enhancer analysis in these TNBC PDCs and cell lines. Super-enhancer analysis revealed that 8q is a substantial chromosomal region with a high density of super-enhancers. Among genes in the vicinity of 8q super-enhancers, we identified that MYB proto-oncogene like 1 (MYBL1) is a transcription factor with high abundance in TNBC PDCs as well as in basal-like BT549 cells. In TCGA breast cancer database, 8q gain is a common genomic feature in MYBL1 gene-amplified breast cancer tissues and MYBL1 expression is substantially correlated with 8q-related genes and proliferation-related genes like MKI67 and BUB1. In TNBC PDCs and cell lines, we showed that MYBL1-specific siRNAs significantly repressed TNBC cell proliferation and migration. RNA sequencing analysis in TNBC cells revealed that MYBL1 silencing substantially downregulated genes involved in the transcription machinery such as nuclear envelope, chromatin dynamics, and DNA replication. In immunohistochemical analysis of clinical TNBC tissues from a Japanese cohort, we showed that a positive MYBL1 immunoreactivity (IR) was significantly associated with shorter disease-free survival (DFS). Among candidate MYBL1 targets, we demonstrated that NCAPH, a subunit of condensin I complex, was a prognostic factor for patients with TNBC based on the immunohistochemistry of clinical TNBC tissues. Notably, patients with double IR positivity for MYBL1 and NCAPH showed shorter DFS than those with MYBL1 or NCAPH positivity alone. MYBL1 has been characterized as an essential transcriptional regulator in male meiosis and in female mammary gland development. Our findings indicate that MYBL1 plays an essential role in the pathophysiology for advanced breast cancers such as TNBC, and MYBL1 and its downstream genes can be potential diagnostic and therapeutic targets for the disease.
 Citation Format: Akihiro Fujimoto, Kazuhiro Ikeda, Keiichi Kinowaki, Hidetaka Kawabata, Akihiko Osaki, Satoshi Inoue, Kuniko Horie. Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-02-08.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bc0fd91427900f720610a3a8305e7e955f0eb39" target='_blank'>
              Abstract P2-02-08: Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer
              </a>
            </td>
          <td>
            Akihiro Fujimoto, K. Ikeda, Keiichi Kinowaki, H. Kawabata, Akihiko Osaki, Satoshi Inoue, K. Horie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d7251ffb7f87367d6e05c4f72a28b57671f1431" target='_blank'>
              Senescence-Associated Chromatin Rewiring Promotes Inflammation and Transposable Element Activation
              </a>
            </td>
          <td>
            A. Dalgarno, Shane A. Evans, M. Kelsey, Thomas A. Nunez, Azucena Rocha, Kelly Clark, J. Sedivy, Nicola Neretti
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 APOBEC3 (A3) enzymes play a pivotal role in mutagenesis across various cancer types. These enzymes, which are single-stranded DNA cytosine deaminases, convert cytosine to uracil (C-to-U) as part of innate antiviral immune responses. Aberrant expression of A3 enzymes in several cancers leads to DNA damage, mutagenesis, and genomic instability. In breast cancer, in particular, APOBEC3B (A3B) is the major endogenous mutator that contributes to tumor evolution and resistance to treatment (1–4). DNA C-to-U deamination events result in characteristic C-to-T transitions and C-to-G transversions, and genomic uracils can also be processed into single- or double-stranded DNA breaks and larger chromosomal aberrations. Consequently, tumors with elevated A3B levels endure chronic genotoxic stress and may exhibit increased sensitivity to inhibitors targeting specific DNA repair pathways. Previous research from our laboratory identified DNA uracil glycosylase 2 (UNG2), a critical initiator of the base excision repair (BER) pathway, as a synthetic lethal partner with A3B (5). Genetic disruption of UNG2, combined with high A3B expression, led to cell death (5). This finding suggests that other DNA repair proteins could potentially serve as synthetic lethal partners with A3B under conditions of elevated A3B-induced DNA damage. To test this hypothesis, we used pharmacological and genetic inhibition of DNA damage repair proteins in isogenic cell line models for A3B. Specifically, we quantified cancer cell viability following treatment with various commercially available DNA damage response inhibitors, in the presence or absence of A3B. Additionally, we performed CRISPR screens using a guide RNA library targeting 237 DNA damage repair and response genes in a doxycycline-inducible TREX-293-A3Bi-eGFP cell line. Cells expressing or lacking A3B were harvested for DNA extraction and sequencing at different time points. By comparing guide RNA abundance between doxycycline-treated cells and untreated cells, we identified dropout guides that disrupt genes, creating potential synthetic lethal combinations with A3B. The results from both pharmacological and genetic inhibition approaches will be presented.
 Selected References:
 1. Burns, M. B. et al. APOBEC3B is an enzymatic source of mutation in breast cancer. Nature 494, 366–370 (2013).
 2. Law, E. K. et al. The DNA cytosine deaminase APOBEC3B promotes tamoxifen resistance in ER-positive breast cancer. Sci Adv 2, e1601737 (2016).
 3. Bertucci, F. et al. Genomic characterization of metastatic breast cancers. Nature 569, 560–564 (2019).
 4. Venkatesan, S. et al. Induction of APOBEC3 exacerbates DNA replication stress and chromosomal instability in early breast and lung cancer evolution. Cancer Discov (2021) doi:10.1158/2159-8290.CD-20-0725.
 5. Serebrenik, A. A. et al. The deaminase APOBEC3B triggers the death of cells lacking uracil DNA glycosylase. Proc Natl Acad Sci USA 116, 22158–22163 (2019).
 Citation Format: Bojana Stefanovska, Benjamin Troness, Kevin Lin, Chad Myers, Reuben S. Harris. APOBEC3B And DNA Damage Repair As Synthetic Lethal Pairs [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86e021a93ee84b96a73fc8c1738f94674803feaf" target='_blank'>
              Abstract P5-06-26: APOBEC3B And DNA Damage Repair As Synthetic Lethal Pairs
              </a>
            </td>
          <td>
            B. Stefanovska, Benjamin Troness, Kevin Lin, Chad Myers, R. Harris
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Abstract Mutation, deletion, or silencing of genes encoding cellular metabolism factors occurs frequently in human malignancies. Neomorphic mutations in isocitrate dehydrogenases 1 and 2 (IDH1/2) promoting the production of R-2-hydroxyglutarate (R-2HG) instead of α-ketoglutarate (αKG) are recurrent in human brain cancers and constitute an early event in low-grade gliomagenesis. Due to its structural similarity with αKG, R-2HG acts as an inhibitor of αKG-dependent enzymes. These include the JUMONJI family of lysine demethylases, among which KDM4A is particularly sensitive to R-2HG-mediated inhibition. However, the precise molecular mechanism through which inhibition of αKG-dependent enzymes by R-2HG promotes gliomagenesis remains poorly understood. Here, we show that treatment with R-2HG induces cellular senescence in a p53-dependent manner. Furthermore, expression of mutated IDH1R132H or exposure to R-2HG, which leads to KDM4A inhibition, causes telomeric dysfunction. We demonstrate that KDM4A localizes to telomeric repeats and regulates abundance of H3K9(me3) at telomeres. We show that R-2HG caused reduced replication fork progression, and that depletion of SMARCAL1, a helicase involved in replication fork reversal, rescues telomeric defects caused by R-2HG or KDM4A depletion. These results establish a model whereby IDH1/2 mutations cause R-2HG-mediated inhibition of KDM4A, leading to telomeric DNA replication defects, telomere dysfunction, and associated genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/728014013df3b457cbd616bc46c4a758e940b3f8" target='_blank'>
              R-2-hydroxyglutarate-mediated inhibition of KDM4A compromises telomere integrity
              </a>
            </td>
          <td>
            Florence Couteau, Laurence M Gagné, Karine Boulay, Philippe Rousseau, Mélissa Carbonneau, Mary McQuaid, Jyoti Sharma, C. Sawchyn, Erlinda Fernandez, Dagmar Glatz, Rana Rizk, Marie-Eve Lalonde, Yosra Mehrjoo, Tsz Wai Chu, G. Moquin-Beaudry, Christian Beauséjour, Mikhail Sergeev, Santiago Costantino, D. Avizonis, I. Topisirovic, Nada Jabado, Hugo Wurtele, Chantal Autexier, Frédérick A. Mallette
          </td>
          <td>2025-06-06</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="The human genome is widely transcribed, with part of these transcribed regions producing stably expressed protein-coding or non-coding RNAs. Long intergenic non-coding RNAs (lincRNAs) are significantly differentially expressed in various cell lines and tissues. However, the influence of their transcription events remains unclear. In this study, we constructed a human genomic interaction network and found frequent interactions between lincRNA genes and protein-coding genes that are highly related to the occupancy of RNA polymerase II on the lincRNA gene. Interestingly, in the human genome interaction networks, the degree of lincRNA genes was significantly higher than that of protein-coding genes. The promoter regions of the protein-coding genes interacting with the lincRNA genes are enriched with R-loop structures, indicating that lincRNA may influence the target genes through R-loop structures. These promoters were enriched in more transcription factor binding sites. Furthermore, the whole network and sub-network could be utilized to explore potential biomarkers of leukemia. We found that zinc finger protein 668 (ZNF668), eosinophil granule ontogeny transcript (EGOT), and glutamate metabotropic receptor 7 (GRM7) could serve as novel biomarkers for acute myeloid leukemia (LMAL). Pasireotide acetate (CAS No. 396091-76-2) represents a potential drug for LMAL patients. These results suggested that potential biomarkers and corresponding drugs for cancer could be identified based on lincRNA–promoter network/sub-network topological parameters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0fd4f873b33226dc62ac979d88627e1f00373ee" target='_blank'>
              Genome-Wide Detection of Leukemia Biomarkers from lincRNA–Protein-Coding Gene Interaction Networks in the Three-Dimensional Chromatin Structure
              </a>
            </td>
          <td>
            Yue Hou, Wei Ning, Muren Huhe, Chuanjun Shu
          </td>
          <td>2025-05-22</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4968170702089395233b0c88bfdd298c3048f328" target='_blank'>
              Role of Chromosomal Microarray and RNA Fusion Analysis in Detecting KMT2A Partial Tandem Duplication with High Mortality in Myeloid Neoplasms: Retrospective Study
              </a>
            </td>
          <td>
            MD¹ Bhaumik Shah, MD¹ Jianming Pei, MD¹ Roniya Francis, MD² Peter Abdelmessieh, MD¹ Mariusz Wasik, MD¹ Nicholas Mackrides, MD¹ Reza Nejati
          </td>
          <td>2025-05-28</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Aim: The over-activation of oncogenes is a critical genetic event in the development and progression of solid malignancies. Gene amplification and specific mutations represent the prominent mechanisms that convert proto-oncogenes into their active, oncogenic forms. C-myc oncogene (gene locus: 8q24.21) regulates crucial cell and tissue functions, whereas its deregulation is implicated in carcinogenesis. Our research aimed to investigate the impact C-myc numerical imbalances on a series of laryngeal squamous cell carcinomas (LSCCs) characterized by different clinic-pathological features. Materials and Methods: We analyzed 55 LSCC archival tissue sections by implementing a dual-color gene/chromosome fluorescent in situ hybridization assay (C-myc/Chr 8 FISH). Results: According to the extracted C-myc gene/Chr 8 signal ratios, we identified 18 cases with numerical imbalances. The majority of them were gene amplifications (pure n=15, amplification and polysomy n=1), whereas the last two (n=2) cases presented multiple gene copies as a result of Chr 8 polysomy. C-myc amplification was correlated with advanced stage. Stage III/IV cases demonstrated the highest levels of gene amplification (p=0.024). Conclusion: C-myc amplification, combined or not with Chr 8 polysomy, is a relatively frequent event in LSCC. This numerical imbalance is associated with an aggressive phenotype, and it seems to be a significant key element among other oncogenes and suppressor genes that form specific genomic signatures in the corresponding patients with LSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf169522cd061e72ca9d71b7495c4317a28f61a5" target='_blank'>
              C-myc Oncogene Numerical Imbalances Analysis in Laryngeal Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Sotirios Papouliakos, V. Papanikolaou, A. Chrysovergis, E. Tsiambas, D. Roukas, Sofianiki Mastronikoli, G. Tsouvelas, G. Agrogiannis, Nikolaos Kavantzas, Vasileios Ragos, Andreas C. Lazaris, E. Kyrodimos
          </td>
          <td>2025-05-27</td>
          <td>AntiCancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="A characteristic of triple-negative breast cancer (TNBC) is the epigenetic regulation of tumor suppressor genes, leading to TNBC heterogeneity and treatment resistance in patients. TNBC exhibits high methylation rates, leading to the silencing of numerous tumor suppressor genes. DNA methyltransferase inhibitors (DNMTis) have shown limited clinical efficacy in TNBC treatment. This study aims to uncover a target that could be used to reverse the epigenetic silencing of tumor suppressor genes in TNBC. The Western blot analysis demonstrated that ROR1 knockdown, an oncofetal gene, reduced DNMT3A and DNMT3B protein expression in the TNBC cell lines MDA-MB-231 and HCC1806, as well as a non-malignant breast cell line, MCF10A. The reduced representation bisulfite sequencing (RRBS) analysis identified differential methylation of CREB3L1 when ROR1 is knocked down in TNBC cell lines. CREB3L1 is a transcription factor that plays tumor-suppressive roles in TNBC and is commonly epigenetically silenced in patients. This study shows that ROR1 requires pSTAT3 activation to upregulate DNMT3A and DNMT3B expression to induce CREB3L1 epigenetic silencing in TNBC. ROR1 knockdown resulted in the re-expression of CREB3L1 in TNBC cells. The data provide evidence that ROR1 inhibition, in combination with DNMTis, could enhance patient outcomes as a therapeutic approach for TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac855cfb53ff2a45c0c45fe36069b3895f3153ea" target='_blank'>
              Uncovering a Novel Role of ROR1 in the Epigenetic Regulation of Tumor Suppressor Gene CREB3L1 in Triple-Negative Breast Cancer Cells
              </a>
            </td>
          <td>
            Victoria L. Reed, Eric Lalu, Leena Yoon, Norman Fultang, B. Peethambaran
          </td>
          <td>2025-05-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2b1a766c9476f757f76140a52e67514cf195c7f" target='_blank'>
              Longitudinal and multisite sampling reveals mutational and copy number evolution in tumors during metastatic dissemination.
              </a>
            </td>
          <td>
            K. Zhao, J. Vos, S. Lam, Lillian A Boe, Daniel Muldoon, Catherine Y Han, Cristina Valero, Mark Lee, C. Fitzgerald, A. Lee, Manu Prasad, Swati Jain, Xinzhu Deng, Timothy A Chan, Michael F. Berger, C. Bandlamudi, Xi Kathy Zhou, Luc G. T. Morris
          </td>
          <td>2025-06-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Aneuploidy is defined as the loss or gain of a whole chromosome or its region. Even at early stages of development, it usually leads to fatal consequences, including developmental defects/abnormalities and death. For a long time, it was believed that the disruption of gene balance results in pronounced effects at both the cellular and organismal levels, adversely affecting organism formation. It has been shown that the gene imbalance resulting from aneuploidy leads to proteotoxic and metabolic stress within the cell, reduced cell proliferation, genomic instability, oxidative stress, etc. However, some organisms have exhibited tolerance to aneuploidies, which may even confer adaptive advantages, such as antibiotic resistance in pathogenic fungal strains. A significant factor likely lies in the complexity of the tissue and organ organization of specific species. Polyploid organisms are generally more tolerant of aneuploidy, particularly those that have recently undergone whole-genome duplication. This review places special emphasis on the examination of sex chromosome aneuploidies in humans. In addition to primary effects, or cis effects (changes in the quantity of the transcripts of genes located on the aneuploid chromosome), aneuploidy can induce secondary or trans effects (changes in the expression levels of genes located on other chromosomes). The results of recent studies have prompted a reevaluation of the impact of aneuploidy on the structural-functional organization of the genome, transcriptome, and proteome of both the cell and the entire organism. Despite the fact that, in the cases of aneuploidy, the expression levels for most genes correlate with their altered copy numbers in the cell, there have been instances of dosage compensation, where the transcript levels of genes located on the aneuploid chromosome remained unchanged. The review presents findings from recent studies focused on compensatory mechanisms of dosage compensation that modify gene product quantities at post-transcriptional and post-translational levels, alleviating the negative effects of aneuploidy on cellular homeostasis. It also discusses the influence of extrachromosomal elements on the spatial organization of the genome and the changes in gene expression patterns resulting from their presence. Additionally, the review specifically examines cases of segmental aneuploidy and changes in copy number variants (CNVs) in the genome. Not only the implications of their composition are considered, but also their localization within the chromosome and in various compartments of the interphase nucleus. Addressing these questions could significantly contribute to enhancing cytogenomic diagnostics and establishing a necessary database for accurate interpretation of identified cases of segmental aneuploidy and CNVs in the genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/271a442ac782b355db581186dfcccbd9f3d50dd8" target='_blank'>
              From cytogenetics to proteogenomics: new horizons in the study of aneuploidies
              </a>
            </td>
          <td>
            K. Zadesenets, N. B. Rubtsov
          </td>
          <td>2025-06-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Targeted inhibition of DNA damage response proteins has received significant clinical attention owing to the success of PARP inhibitors. Due to the loss of the G1/S checkpoint, cancer cells are reliant on the G2/M checkpoint to cope with elevated DNA replication stress. We previously demonstrated a single induction of 8-oxo-guanine at telomeres in cancer cells was sufficient to induce replication stress, but was well tolerated at the cellular level. Here, we found inhibition of ATR, Chk1, or Wee1 after induction of telomere oxidative stress significantly induced genome instability and reduced cell viability. This occurred at doses markedly less than those required to increase instability in non-cancer cells. We determined the mechanism of this instability is due to cells progressing through S-phase with telomere damage and exiting G2-phase prematurely, prolonging their mitosis. This study demonstrates targeted oxidative base damage at telomeres can enhance the therapeutic efficacy of ATR inhibition in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/520fb2116ae4feadf96591801b8f58200ddc5cec" target='_blank'>
              Oxidative Base Damage to Telomeres Sensitizes Cancer Cells to ATR Inhibition
              </a>
            </td>
          <td>
            Alex Garbouchian, Natalia Cestari Moreno, A. Dey, Patricia Opresko, Ryan Barnes
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Exacerbating the DNA replication problems of cancer cells serves as a viable therapeutic approach. Nevertheless, the cytotoxicity of cancer drugs is often hampered by therapy-induced senescence, leading to unfavorable patient outcomes. Here, we employ acute replisome dysfunction in combination with Ataxia telangiectasia and Rad3-related (ATR) inhibition as a strategy to divert senescent cells toward death by triggering DNA replication catastrophe, a form of irreversible replication fork collapse caused by excessive single-stranded DNA (ssDNA) accumulation. RNA-sequencing revealed a distinct set of p53-responsive genes responsible for death. We identify c-Jun N-terminal kinase (JNK) to be essential for augmenting p53-dependent apoptotic programs and inducing pan-nuclear distribution of γH2AX, together constituting a feed-forward loop to drive cell death. Activation of DNA-PKcs initiates the signaling cascade of replication catastrophe, including CHK1-dependent JNK activation, which relies on MRE11 and PARP1 to expand and recognize ssDNA gaps, defining replication-associated gaps as an underlying basis for replication catastrophe. Our study elucidates the dynamic regulation of proximal and distal effectors along the DNA-PKcs/JNK/p53 axis that govern the cell fate decision between senescence and death. We propose that key determinants of replication catastrophe signaling are targetable vulnerabilities that can be exploited to limit senescent cell populations and increase the efficacy of anti-cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c9be1cc99c822e2708983101c9fab63e3ac941" target='_blank'>
              The DNA-PKcs/JNK/p53 pathway underlies changes in cell fate decision toward death during DNA replication catastrophe
              </a>
            </td>
          <td>
            Jinal A. Patel, Julie Rageul, Natalie Lo, Auntara Nandi, Camryn Zezelic, Cynthia T Lee, Arafat Khan, Hyungjin Kim
          </td>
          <td>2025-06-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Enhancers, as distal cis-regulatory elements in the genome, have a pivotal influence on orchestrating precise gene expression. Enhancer RNAs (eRNAs), transcribed from active enhancer regions, are increasingly recognized as key regulators of transcription. N6-methyladenosine (m6A), the most plentiful internal modification in eukaryotic mRNAs, has garnered significant research interest in recent years. With advancements in high-throughput sequencing technologies, it has been established that m6A modifications are also present on eRNAs. An accumulative body of evidence demonstrates that aberrant enhancers, eRNAs, and m6A modifications are intimately connected with carcinoma onset, progression, invasion, metastasis, treatment response, drug resistance, and prognosis. However, the underlying molecular mechanisms governing m6A modification of eRNAs in cancer remain elusive. Here, we review and synthesize current understanding of the regulatory roles of enhancers, eRNAs, and m6A modifications in cancer. Furthermore, we investigate the possible roles of eRNAs m6A modification in tumorigenesis based on existing literature, offering novel perspectives and directions for future research on epigenetic regulatory mechanisms in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6966deaca0f1174a84788609926a02bf29bca084" target='_blank'>
              Biological roles of enhancer RNA m6A modification and its implications in cancer
              </a>
            </td>
          <td>
            Yangyang Han, Jingqi Sun, Minghui Yao, L. Miao, Mengjia Li
          </td>
          <td>2025-05-30</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41db77b7599b92a63937c7f3610506611aa97e8f" target='_blank'>
              A survey of human cancer-germline genes : linking X chromosome localization, DNA methylation and sex-biased expression in early embryos
              </a>
            </td>
          <td>
            A. Loriot, Julie Devis, Laurent Gatto, C. De Smet
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Ewing sarcoma (EwS) is a group of bone and soft tissue cancers in children and young adults. The EWS-FLI1 protein, the oncogenic driver in EwS, is a fusion of the FET protein, EWSR1, at its amino terminus, with the carboxy terminus of the transcription factor, FLI1. Since EwS cells have pronounced sensitivity to radiation and chemotherapy-induced DNA damage, the role of EWS-FLI1 in DNA repair is likely. Here, we demonstrate that EWS-FLI1 causes a defect in microhomology-mediated end-joining (MMEJ) repair. EWSR1 is a splicing factor that promotes the faithful splicing of the POLQ pre-mRNA, required for the expression of POLΘ, a critical protein in the MMEJ pathway. Expression of EWS-FLI1, or loss of EWSR1, causes exon 25 skipping of the POLQ transcript, decreased POLΘ protein expression, impaired MMEJ, and cellular sensitivity to inhibitors of NHEJ repair and HR repair, through the mechanism of synthetic lethality. Knockdown of EWS-FLI1 expression restores POLΘ mitotic foci and increases MMEJ activity. Based on this mechanism, inhibitors of NHEJ or HR may provide a targeted therapy for patients with Ewing sarcoma. Highlights - Ewing sarcoma tumors have a deficiency in POLθ expression and a corresponding loss of MMEJ activity - EWSR1 is a splicing factor that interacts with other splicing factors such as FUBP1 and KHSRP/FUBP2 to accurately splice the POLQ mRNA. - The EWS-FLI1 fusion oncoprotein, or loss of EWSR1, causes a splicing defect, leading to exon25 skipping of the POLQ pre-mRNA and loss of POLθ expression - The MMEJ deficiency of EwS cells results in cellular sensitivity to inhibitors of Homologous Recombination or Non-Homologous End-Joining - Exon 25 skipping of POLQ mRNA is a predictive biomarker for HR inhibitors in human cancers">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f1eaf5b79d22ae4a1a31716b8447290969ad6a6" target='_blank'>
              Disruption of Microhomology-mediated End-joining in Ewing Sarcoma
              </a>
            </td>
          <td>
            S. Asada, Guangli Zhu, J. Abeykoon, Yutaro Tanaka, Huy Nguyen, Y. Hirohashi, Divya R Iyer, N. Ashton, Sirisha Mukkavalli, Lige Jiang, Miles Del Busso, J. Thangaiah, Steven I. Robinson, K. Parmar, E. V. Van Allen, R. Gillani, Geoffrey I. Shapiro, Alan D. D’Andrea
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e285847d1a624eba0863eb2e7a290a733cac7918" target='_blank'>
              Phenotypic and Functional Characterization of Oncohistone Mutations in Breast Cancers
              </a>
            </td>
          <td>
            Andrea D. Edwards, Yangyang Dai, Siddharth Singh, Micah Thornton, T. Nandu, Ralf Kittler, C. Camacho, Dan Huang, W. Kraus
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Background: Large-scale genomic profiling has cataloged the prevalence of single base substitution (SBS) mutational signatures associated with the activity of Apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) cytidine deaminases in breast cancer (BC). These mutational signatures are enriched in metastatic BC (mBC) compared to early tumors indicating an association with poor prognosis and a potential function in therapy resistance and disease progression. We sought out to investigate whether APOBEC3 mutational signatures can serve as biomarkers for poor treatment outcomes and if APOBEC3 mutagenesis-driven genomic instability can induce therapy resistance in mBC. Methods: We analyzed SBS mutational signatures in 3,880 BC samples with paired tumor-normal sequencing by the MSK-IMPACT assay using the SigMA algorithm. We utilized the detailed clinical annotation to assess the clinical characteristics of APOBEC3-dominant tumors including survival analyses on endocrine and targeted therapies. We generated cellular models of BC to investigate the molecular drivers of APOBEC3 mutagenesis and its function in promoting therapeutic resistance. We performed whole genome sequencing (WGS) of BC models and paired primary/metastatic patient samples to identify broader genomic alterations mediated by APOBEC3 activity. Results: Building on published results, we found that APOBEC3 mutational signatures were highly prevalent in all subtypes of BC and enriched in metastatic hormone receptor-positive (HR+) and triple-negative breast cancers (TNBC) compared to unmatched primary tumors (p < 0.0001 for HR+/HER2-, p < 0.01 for HR+/HER2+ and TNBC). APOBEC3 mutational signatures were independently associated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor combination therapy in patients with HR+ mBC (HR 1.5, 95% CI 1.2 - 1.8, p < 0.001). Expression of APOBEC3A (A3A) and APOBEC3B (A3B) enzymes generated APOBEC3-associated alterations including single nucleotide variants, copy number alterations (CNAs), and clustered mutations in a deamination-dependent manner, and promoted resistance acquisition of estrogen receptor-positive (ER+) BC cells to agents including an ER degrader and CDK4/6 inhibitors. In HER2+ cells, endogenous A3A-driven APOBEC3 activity was also necessary for faster resistance development against anti-HER2 therapies. WGS analyses of resistant models identified CNA events such as loss of heterozygosity in chromosome 13 exclusively in APOBEC3-positive cells. Upon exposure to the CDK4/6 inhibitor abemaciclib, these cells acquired APOBEC3-context truncating mutations in RB1 tumor suppressor gene, a well-characterized mechanism of resistance. Detailed analyses of WGS of five paired patient samples also highlighted acquired resistance-linked alterations such as PIK3CA E54XK mutations in APOBEC3-dominant tumors, which were corroborated in paired pre/post-treatment samples in our clinical cohort. The acquisition of APOBEC3-context alterations in APOBEC3-dominant samples highlights the causality of APOBEC3 mutagenesis in driving resistance-promoting changes. Lastly, comparison of mutational signatures in the paired cohort demonstrated pre-existence of APOBEC3 signatures in 75% of pre-treatment samples that became APOBEC3-dominant post-treatment illustrating that APOBEC3 mutagenesis can be active during early stages of BC. Conclusions: Our work reveals that APOBEC3 mutational signatures predict poor treatment outcomes of HR+ mBC. We demonstrate that APOBEC3 mutagenesis, primarily through the enzymatic activity of A3A and A3B, drives resistance to endocrine and targeted therapies by causing APOBEC3-context resistance-associated changes. We further show that the presence of APOBEC3 mutagenesis can be detected before therapy exposure and may therefore represent a valuable biomarker and therapeutic target.
 Citation Format: Avantika Gupta, Andrea Gazzo, Pier Selenica, Anton Safonov, Fresia Pareja, Edaise M. da Silva, David N. Brown, Yingjie Zhu, Juber Patel, Juan Blanco-Heredia, Bojana Stefanovska, Michael A. Carpenter, Xin Pei, Denise Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, Britta Weigelt, Nadeem Riaz, Simon N. Powell, Pedram Razavi, Reuben S. Harris, Jorge S. Reis-Filho, Antonio Marra, Sarat Chandarlapaty. APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr GS2-11.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b57625ce2cff9cbb61ff4f9d159b42b1e5eb578" target='_blank'>
              Abstract GS2-11: APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, B. Weigelt, N. Riaz, Simon Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="RNA–DNA interactions are fundamental to cellular physiology, playing critical roles in genome integrity, gene expression, and stress responses. This review highlights the diverse structures of RNA–DNA hybrids, including R-loops, RNA–DNA triplexes, and RNA–DNA hybrid G-quadruplexes (hG4s) and their relevance in physiology. R-loops are formed during transcription and replication, which regulate gene expression and chromatin dynamics but can also threaten genome stability. RNA–DNA triplexes, often formed by long noncoding RNAs (lncRNAs) such as FENDRR and MEG3, recruit chromatin modifiers like Polycomb repressive complex 2 to modulate gene expression, influencing organogenesis and cell specification. hG4s, formed by guanine-rich sequences in RNA and DNA, regulate transcription termination and telomere stability. Through this, hG4s can affect gene suppression and replication regulation. RNA–DNA hybrids are tightly regulated by helicases, RNase H enzymes, and topoisomerases, with altered regulation linked to genomic instability and disease. This review discusses the complexity of RNA–DNA interactions and their recently identified contributions to cellular physiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e76333f352e42183b0f9c7b2686b45523c8684f6" target='_blank'>
              The relevance of RNA–DNA interactions as regulators of physiological functions
              </a>
            </td>
          <td>
            Julia Stötzel, T. Warwick, Praveenya Tirunagari, Ralf Peter Brandes, M. Leisegang
          </td>
          <td>2025-05-21</td>
          <td>Pflugers Archiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Inter-chromosomal interactions play a crucial role in 3D genome organization, yet the organizational principles and functional significances remain elusive. In general, lncRNA loci and transcripts are frequently associated with transcriptional programs modulated by long-range chromatin interactions. Here, we identified a novel lncRNA named Gm26793, which is abundantly distributed in the primitive streak and mesodermal cells of embryonic day 7.5 mouse gastrula. Through genetic ablation of Gm26793, we observed a preferential responsiveness to primitive endoderm lineage during stem cell differentiation, as well as enhanced occurrence of transient and degenerative state cells in early mouse embryos when the cell fate segregates between epiblast and primitive endoderm. Mechanistically, we revealed that the genomic locus of Gm26793, rather than the lncRNA transcript or adjacent gene, governs the cell fate preference towards primitive endoderm. Concretely, Gm26793 locus (Chromosome 7) forms an inter-chromosomal molecular lock with Cubn (Chromosome 2) via CTCF, restraining the expression of Cubn and maintaining a natural epigenetic landscape, thus ensuring the proper lineage specification in vitro and in vivo. Overall, our study provides a clear paradigm that inter-chromosomal interaction collaborates with architectural factors to stabilize nuclear conformation and guarantee faithful gene expression during stem cell differentiation and mammalian embryogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5737d88dd5a339188134898f4ba7c9851ba9807" target='_blank'>
              Genomic locus of lncRNA-Gm26793 forms an inter-chromosomal interaction with Cubn to ensure proper stem cell differentiation in vitro and in vivo
              </a>
            </td>
          <td>
            Zhiwen Liu, Xin Wan, Jiehui Chen, Yongjian Ma, Yonggao Fu, Yingying Chen, Mingzhu Wen, Yun Yang, Yun Qian, Yong Zhang, Dahai Zhu, Jinsong Li, Naihe Jing, Xianfa Yang
          </td>
          <td>2025-06-03</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Every generation, the human genome is shuffled during meiosis and a single fertilized egg gives rise to all of the cells of the body1. Meiotic errors leading to chromosomal abnormalities are known causes of pregnancy loss2,3, but genetic aetiologies of euploid pregnancy loss remain largely unexplained4. Here we characterize sequence diversity in early pregnancy loss through whole-genome sequencing of 1,007 fetal samples and 934 parental samples from 467 trios affected by pregnancy loss (fetus, mother and father). Sequenced parental genomes enabled us to determine both the parental and meiotic origins of chromosomal abnormalities, detected in half of our set. It further enabled us to assess de novo mutations on both homologous chromosomes from parents transmitting extra chromosomes, and date them, revealing that 6.6% of maternal mutations occurred before sister chromatid formation in fetal oocytes. We find a similar number of de novo mutations in the trios affected by pregnancy loss as in 9,651 adult trios, but three times the number of pathogenic small (<50 bp) sequence variant genotypes in the loss cases compared with adults. Overall, our findings indicate that around 1 in 136 pregnancies is lost due to a pathogenic small sequence variant genotype in the fetus. Our results highlight the vast sequence diversity that is lost in early pregnancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa3651219511c4ef3423241fcef77876aaae5715" target='_blank'>
              Sequence diversity lost in early pregnancy
              </a>
            </td>
          <td>
            G. Arnadottir, Hákon Jónsson, T. Hartwig, J. Gruhn, P. L. Møller, Arnaldur Gylfason, D. Westergaard, Andrew Chi-ho Chan, A. Oddsson, L. Stefánsdóttir, L. Roux, V. Steinthorsdottir, Kristjan H Swerford Moore, S. Olafsson, Pall I. Olason, H. Eggertsson, G. Halldorsson, G. Walters, H. Stefánsson, S. A. Gudjonsson, G. Pálsson, B. Jensson, Run Fridriksdottir, Jesper Friis Petersen, , H. Nielsen, S. Bliddal, K. Banasik, M. Krog, A. Kolte, N. la Cour Freiesleben, S. Ostrowski, Erik Sørensen, Mille Løhr, Markus J. Herregård, E. Hoffmann, J. Gruhn, Andrew Chi-ho Chan, U. Þorsteinsdóttir, Kári Stefánsson, Hákon Jónsson, O. Magnusson, L. Ambye, Lone Schmidt, P. Kamstrup, M. Nyegaard, Ellen Christine Leth Løkkegaard, H. E. Bredkjær, H. W. Hvidman, Agnar Helgason, G. Norddahl, P. D. Rohde, J. Saemundsdottir, O. Magnusson, B. Halldórsson, D. Gudbjartsson, P. Sulem, U. Thorsteinsdóttir, E. Hoffmann, H. Nielsen, Kári Stefánsson
          </td>
          <td>2025-05-21</td>
          <td>Nature</td>
          <td>2</td>
          <td>184</td>
        </tr>

        <tr id="For chromosome abnormalities (CA), such as Down syndrome (DS), the influence of genomic variations to chromosome conformation and gene transcription remains elusive. Based on the complete genomic sequence from the parents of the DS trisomy patient, we systematically delineated an atlas of parental-specific haploid single nucleotide polymorphism (SNP), copy number variation (CNV), three-dimensional (3D) genome, and RNA expression profiles of the diencephalon in the DS patient. The integrated haploid multi-omics analysis demonstrated that one-dimensional genomic variations including SNPs and CNVs in the DS patient are highly correlated with the alterations of the 3D genome and the subsequent gene transcription. The correlation relationship remains valid in haploid-levels. Moreover, we revealed the 3D genome alteration associated mis-regulation of DS-related genes, which facilitates to understanding the pathogenesis of CA. Together, our study contributes to decipher the coding from one-dimensional genomic variations to 3D genomic architecture and the subsequent gene transcription in healthy and diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d9de44c4aa76967640e3414bc7acd3b2037c35" target='_blank'>
              Deciphering Haploid Chromosome Conformation Alternation in Down Syndrome by Multiple Haploid Omics Analysis.
              </a>
            </td>
          <td>
            Chengchao Wu, Tianshu Zhou, Wenfu Ke, Wei Xiong, Zhihui Zhang, Siheng Zhang, Jinyue Wang, Lulu Deng, Keji Yan, Man Wang, Shenglong He, Qi Gong, Chao Ma, Xiaping Chen, Yan Li, He Long, Chong Guo, Gang Cao, Zhijun Zhang
          </td>
          <td>2025-06-12</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e2397ca3adf8845c9715ac4636c6b9da727b026" target='_blank'>
              Epigenetic dynamics and molecular mechanisms in oncogenesis, tumor progression, and therapy resistance.
              </a>
            </td>
          <td>
            Mohammed Kaleem, Lubna Azmi, N. Shahzad, Murtada Taha, Shiv Kumar, Md. Ali Mujtaba, Abdulaziz Ali H Hazazi, A. Kayali
          </td>
          <td>2025-05-13</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Studies on Hippo pathway regulation of tumorigenesis largely center on YAP and TAZ, the transcriptional co-regulators of TEADs. Here, we present an oncogenic mechanism involving VGLL and TEAD fusions that is Hippo pathway-related but YAP/TAZ-independent. We characterize two recurrent fusions, VGLL2-NCOA2 and TEAD1-NCOA2, recently identified in human spindle cell rhabdomyosarcoma. We demonstrate that in contrast to VGLL2 and TEAD1 the fusion proteins are potent activators of TEAD-dependent transcription, and the function of these fusion proteins does not require YAP/TAZ. Furthermore, we identify that VGLL2 and TEAD1 fusions engage specific epigenetic regulation by recruiting histone acetyltransferase EP300 to control TEAD-mediated transcriptional and epigenetic landscapes. We show that small-molecule EP300 inhibition can suppress fusion protein-induced oncogenic transformation both in vitro and in vivo in mouse models. Overall, our study reveals a molecular basis for VGLL involvement in cancer and provides a framework for targeting tumors carrying VGLL, TEAD, or NCOA translocations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a702abde9d999a5456ad7e5b1831d671902648" target='_blank'>
              VGLL2 and TEAD1 fusion proteins identified in human sarcoma drive YAP/TAZ-independent tumorigenesis by engaging EP300
              </a>
            </td>
          <td>
            Susu Guo, Xiaodi Hu, J. Cotton, Lifang Ma, Qi Li, Jiangtao Cui, Yongjie Wang, Ritesh P Thakare, Zhipeng Tao, Tony Ip, Xu Wu, Jiayi Wang, Junhao Mao
          </td>
          <td>2025-05-08</td>
          <td>eLife</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Intratumor heterogeneity arises from ongoing somatic evolution complicating cancer diagnosis, prognosis, and treatment. Here we present TEATIME (estimating evolutionary events through single-timepoint sequencing), a novel computational framework that models tumors as mixtures of two competing cell populations: an ancestral clone with baseline fitness and a derived subclone with elevated fitness. Using cross-sectional bulk sequencing data, TEATIME estimates mutation rates, timing of subclone emergence, relative fitness, and number of generations of growth. To quantify intratumor fitness asymmetries, we introduce a novel metric—fitness diversity—which captures the imbalance between competing cell populations and serves as a measure of functional intratumor heterogeneity. Applying TEATIME to 33 tumor types from The Cancer Genome Atlas, we revealed divergent as well as convergent evolutionary patterns. Notably, we found that immune-hot microenvironments constraint subclonal expansion and limit fitness diversity. Moreover, we detected temporal dependencies in mutation acquisition, where early driver mutations in ancestral clones epistatically shape the fitness landscape, predisposing specific subclones to selective advantages. These findings underscore the importance of intratumor competition and tumor-microenvironment interactions in shaping evolutionary trajectories, driving intratumor heterogeneity. Lastly, we demonstrate that TEATIME-derived evolutionary parameters and fitness diversity offer novel prognostic insights across multiple cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4e5a68da211456c515bccdc663dee67f4b48423" target='_blank'>
              Competing Subclones and Fitness Diversity Shape Tumor Evolution Across Cancer Types
              </a>
            </td>
          <td>
            Haiming Chen, Jingmin Shu, Rekha Mudappathi, Elaine Li, Panwen Wang, Leif Bergsagel, Ping Yang, Zhifu Sun, Logan Zhao, Changxin Shi, Jeffrey P. Townsend, Carlo Maley, Li Liu
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cddde94c859320bea53863c22a0c0f005dca0fc" target='_blank'>
              Comprehensive analysis of FAM83B in pan-cancer and preliminary exploration in esophageal squamous cell carcinoma.
              </a>
            </td>
          <td>
            Wei Guo, Xixi Zhao, Xinran Huang, Ruijuan Zhang, Yuchen Wang, Xinyu He, Xiangyun Ma, Yu Hao, Shangyi Geng, S. Pan, Hongbing Ma
          </td>
          <td>2025-05-26</td>
          <td>Journal of molecular histology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4ed51bfd7d0538e3c3fdaf6bc3f9553670a3712" target='_blank'>
              Cancer-Associated USP28 Missense Mutations Disrupt 53BP1 Interaction and p53 Stabilization
              </a>
            </td>
          <td>
            Hazrat Belal, Esther Feng Ying Ng, Midori Ohta, Franz Meitinger
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 e16580


 Background:
 Non-muscle invasive bladder cancer (NMIBC) presents a significant clinical challenge due to its high recurrence rate. Recently, a deeper understanding of tumor epigenetics has provided new opportunities for cancer detection and treatment. While significant efforts have been made to understand NMIBC biology, the molecular and epigenetic mechanisms driving its progression remain elusive.
 Methods:
 We used multiomics analysis including expression, DNA methylation, and cell-free DNA chromatin immunoprecipitation sequencing (cfChIP-seq) assays to uncover the molecular and epigenetic landscape of NMIBC from human tissue samples and urine.
 Results:
 High-throughput sequencing revealed more than 6,000 differentially methylated regions (DMRs) within regulatory elements, many of which were linked to cancer-related pathways and exhibited distinct signatures in cancer tissues compared to healthy controls. Next, we distinguished high and low-grade tumors based on unique DNA methylation profiles. These patterns correlated with transcription level of genes associated with cell cycle regulators, consistent with the higher proliferative capacity of high-grade tumors. Moreover, Among the DMRs, we also identified genes involved in T cell regulation. Supporting this, we demonstrated reduced CD8+ T cells infiltration in high-grade tumors, further highlighting immune dysregulation in the more aggressive disease. We extended our investigation to urine samples, performing cfChIP-seq on samples from healthy donors and NMIBC patients. By using the H3K4me3 promoter histone mark on cfDNA we could detect cell origin. Utilizing this assay, we were able to identify changes in the urine of NMIBC patients, owing to cf-nucleosome contributions from both the tumor and immune cells. Additionally, we stratified the cancer cohort by tumor grade and found enrichment in the number of differentially marked genes in high-grade tumors.
 Conclusions:
 Our detection of unique epigenetic signatures offer opportunities for biomarker development, enabling early detection through non-invasive urine based diagnostic approach. This could grade tumor more accurately and tailor personalized patient care and monitoring.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38dd958dabb53d9b68b4717906734c6068947417" target='_blank'>
              Elucidating the epigenetic landscape of non-muscle invasive bladder cancer.
              </a>
            </td>
          <td>
            T. Falick Michaeli, Batia Azria, M. Lotem, Nir Friedman
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef02803fbaf5cb1abbd79351dc6a4ca1c68ba79" target='_blank'>
              Constitutional copy number amplifications: rare or under-evaluated? Revisiting a 25-year-old cold case.
              </a>
            </td>
          <td>
            Eliana Salvo, Romano Tenconi, R. Giorda, Sara Bertuzzo, Luca Cesana, Roberta Murru, S. Giglio, Mana M Mehrjouy, Niels Tommerup, Orsetta Zuffardi, M. Bonaglia
          </td>
          <td>2025-06-04</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="The human genome encodes 19 adenosine and cytidine deaminase genes, classified as A-to-I versus C-to-U editors. A-to-I editors have been widely identified as a promising therapeutic target in various cancers. Conversely, the investigation into C-to-U editors is relatively limited. This study evaluated RNA-editing genes in prostate cancer (PCa). Notably, the APOBEC3 genes are clustered in terms of their chromosomal locations, and their transcriptional changes exhibit significant positive correlations in both primary PCa and castration-resistant prostate cancer (CRPC). One member of this family, APOBEC3C, is demonstrated here as an androgen receptor (AR)-repressed gene. Consistently, APOBEC3 loci are epigenetically inhibited in PCa progression, with APOBEC3C level lower in PSA-high patients. APOBEC3C-low PCa cohorts exhibit increased resistance to Abiraterone and Enzalutamide. Clinicopathological profiling further confirmed APOBEC3C downregulation along PCa progression to advanced phases (grade IV/V, stage III-IV, and pathological stage T3-4), underscoring its prognostic value. Additionally, APOBEC3C expression inversely correlates with PCa relapse and mortality, and low APOBEC3C levels are linked to unfavorable survival. Notably, integrated analyses identified APOBEC3C as the sole RNA-editing gene with significance in both differential expression and PCa prognosis, and APOBEC3C had the best diagnostic performance among 19 genes. Our efforts provide a foundation for further RNA editors research in PCa diagnosis and therapy, and grant APOBEC3C as a candidate tumor suppressor. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-00169-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d0fae2048158500534064188ceb1148d343ad87" target='_blank'>
              Characterization of RNA editing gene APOBEC3C as a candidate tumor suppressor in prostate cancer
              </a>
            </td>
          <td>
            Li-Yang Wang, Ji Shi, Mo-Fei Wang, Yi-Meng Liu, Hong-Shan Guo, Jin-Cheng Wang, Shu Jiang, Jia-Qian Liang, Xinghua Liao, Shao-Yong Chen
          </td>
          <td>2025-05-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Fluoroquinolones induce double-strand breaks in bacterial DNA, triggering the SOS response, a major DNA damage response that ensures the expression of repair proteins but also promotes the emergence and spread of antibiotic resistance. Fluoroquinolone resistance, particularly in Escherichia coli, is a growing global health concern. Understanding bacterial responses to these antibiotics is critical for developing preventive strategies and novel treatments to combat resistance development. This study investigates DNA morphology in E. coli following exposure to ciprofloxacin (CIP), a fluoroquinolone antibiotic. We show that CIP induces a stepwise DNA reorganization, culminating in a highly dense nucleoid structure at midcell-a process we term DNA supercompaction. This phenomenon occurred also with other genotoxic agents. Live-cell imaging revealed that RecN, a structural maintenance of chromosomes (SMC)-like protein, is required for DNA supercompaction, and that RecN's dynamics and activity in this response depend on RecA. Additionally, RecN and RecA frequently colocalized at nucleoid-associated positions. We suggest that RecN and RecA play active roles in DNA supercompaction following severe DNA damage, that their interplay is part of a prompt universal survival response to DNA double-strand breaks in E. coli, and that the extent of the compaction response depends on the DNA damage severity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/506e392d68ea028b658f17b6f901be5aeef9ee4e" target='_blank'>
              RecN and RecA orchestrate an ordered DNA supercompaction response following ciprofloxacin-induced DNA damage in Escherichia coli.
              </a>
            </td>
          <td>
            Krister Vikedal, S. B. Ræder, Ida Mathilde Marstein Riisnæs, Magnar Bjørås, J. Booth, Kirsten Skarstad, Emily Helgesen
          </td>
          <td>2025-05-22</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Tumors are characterized by global changes in epigenetic modifications such as DNA methylation and histone modifications that are functionally linked to tumor progression. Accordingly, several drugs targeting the epigenome have been proposed for cancer therapy, notably, histone deacetylase inhibitors (HDACi) such as vorinostat and DNA methyltransferase inhibitors (DNMTi) such as zebularine. However, a fundamental challenge with such approaches is the lack of genomic specificity, i.e., the transcriptional changes at different genomic loci can be highly variable, thus making it difficult to predict the consequences on the global transcriptome and drug response. For instance, treatment with DNMTi may upregulate the expression of not only a tumor suppressor but also an oncogene, leading to unintended adverse effect. Methods Given the pre-treatment transcriptome and epigenomic profile of a sample, we assessed the extent of predictability of locus-specific changes in gene expression upon treatment with HDACi using machine learning. Results We found that in two cell lines (HCT116 treated with Largazole at eight doses and RH4 treated with Entinostat at 1µM) where the appropriate data (pre-treatment transcriptome and epigenome as well as post-treatment transcriptome) is available, our model distinguished the post-treatment up versus downregulated genes with high accuracy (up to ROC of 0.89). Furthermore, a model trained on one cell line is applicable to another cell line suggesting generalizability of the model. Conclusions Here we present a first assessment of the predictability of genome-wide transcriptomic changes upon treatment with HDACi. Lack of appropriate omics data from clinical trials of epigenetic drugs currently hampers the assessment of applicability of our approach in clinical setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a8c2bc7236dcb12e89963dabb5f55fdfd61138" target='_blank'>
              Predicting gene expression changes upon epigenomic drug treatment
              </a>
            </td>
          <td>
            P. Agrawal, Vishaka Gopalan, M. A. Rumi, Sridhar Hannenhalli
          </td>
          <td>2025-05-02</td>
          <td>F1000Research</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy with extremely poor prognosis. SCLC cells exhibit high plasticity and can progress from neuroendocrine (NE) to non-NE phenotypes. This dynamic evolution promotes treatment resistance and relapses, representing a challenge for targeted therapies in this elusive disease. Here we identify the transcription factor ONECUT2 (OC2) as a driver of plasticity in SCLC, leading to non-NE transcriptional states. OC2 is highly expressed in SCLC tumors compared to normal lung tissue and its expression is associated with heightened clinical stage and lymph node metastasis. We show that OC2 is a repressor of ASCL1, the NE master regulator transcription factor. In addition, OC2 upregulates non-NE programs through activation of c-MYC and Notch signaling. We also demonstrate that OC2 is required for growth and survival of SCLC cells and that it can be targeted with a small molecule inhibitor that acts synergistically with the standard combination of cisplatin and etoposide, providing a novel therapeutic strategy for OC2 active SCLC tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01267-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736f50c299b3fe9c13bfbd3d67e9601f1567eb2" target='_blank'>
              ONECUT2 reprograms neuroendocrine fate and is an actionable therapeutic target in small cell lung cancer
              </a>
            </td>
          <td>
            Mirian Gutiérrez, Irene Zamora, Raquel Iriarte, M. Pajares, Qian Yang, Chen Qian, N. Otegui, J. Fernández-Irigoyen, Enrique Santamaria, N. Alcala, A. Sexton-Oates, L. Fernandez-Cuesta, Miguel Barajas, Alfonso Calvo, Luis M. Montuenga, Beatrice S. Knudsen, Sungyong You, Michael R. Freeman, Ignacio J. Encío, Mirja Rotinen
          </td>
          <td>2025-06-11</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background CTCF, a highly studied transcription factor, is essential for chromatin interaction maintenance. Several independent studies report that CTCF interacts with RNAs in vitro and in cells. Yet continuous debates about the authenticity of the RNA-binding affinity of CTCF and its biological role remain in large part due to limited research techniques available, such as CLIP-seq. Result Here, we investigate RNA’s role in CTCF’s transcription factor function through its chromatin occupancy. To systematically explore whether RNAs affect CTCF’s ability to bind DNA, we perturb CTCF-RNA interactions by three independent approaches and examine CTCF genome occupancy by ChIP-seq. Although RNase A and triptolide treatment each affect a certain number of CTCF-binding peaks, few peaks overlap between treatment groups indicating the effect of RNA in regulating CTCF’s DNA binding affinity is variable between loci. In addition, limited transcriptional or chromatin accessibility changes occur between cells expressing wild-type CTCF or CTCF lacking the RNA binding region. Conclusion Our data provide a complementary approach and in silico evidence to consider the significance of RNA affecting CTCF’s DNA binding affinity globally. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03582-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59a181f3e3ab1f0c8b8efc73cd9bde644736cf84" target='_blank'>
              Deciphering the role of RNA in regulating CTCF’s DNA binding affinity in leukemia cells
              </a>
            </td>
          <td>
            Judith Hyle, Wenjie Qi, M. Djekidel, Wojciech Rosikiewicz, Beisi Xu, Chunliang Li
          </td>
          <td>2025-05-12</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Guanine-rich DNA sequences have a propensity to form G-quadruplex structures. These structures play several important biological roles and are potential targets for anticancer drugs. However, no G-quadruplex-stabilizing agent has yet been approved for clinical use. Given that G-quadruplex stabilization is quite promising as a mechanism for novel anticancer therapies, it is crucial to elucidate its effects on healthy human cells. In our study, we modeled a potential human treatment using G4 -stabilizing agents and analyzed their effects on genome integrity, transcriptomic changes, and mitochondrial function focusing on non-cancerous cells to predict potential side effects of such treatments. We found that G-quadruplex stabilization does not compromise genome integrity. However, it can induce persistent alterations in the transcriptomic profile of human cells, including genes encoded on the mitochondrial genome. Notably, certain G-quadruplex-stabilizing agents triggered mitophagy in both human cells and Drosophila melanogaster. In summary, our findings indicate that while G-quadruplex stabilization does not cause genome instability, it may pose potential risks due to its long-term effects on transcription and its ability to induce mitophagy. Therefore, we recommend that all potential drug candidates be thoroughly evaluated for their ability to induce mitophagy and to promote cancer formation in animal models prior to clinical trials. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-04791-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/277b0abab64e6f6a40e0606dd3b857caf1865195" target='_blank'>
              The effect of prolonged G-quadruplex stabilization on the functions of human cells
              </a>
            </td>
          <td>
            Nargis Karatayeva, Lili Hegedűs, Arindam Bhattacharjee, Eszter Németh, Ádám Póti, Lorinc Pongor, Gábor Juhász, David Szuts, Peter Burkovics
          </td>
          <td>2025-06-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5519679bc38714d714355e4393f194de04706e50" target='_blank'>
              Actionable biological programs to enhance EGFR-targeted therapy response unveiled by single-cell lineage tracing in clinically relevant lung cancer models
              </a>
            </td>
          <td>
            Beatrice Gini, W. Tamaki, Johnny Yu, Dora Barbosa, Wei Wu, Yashar K Pourmoghadam, Paul Allegakoen, Donghwa Kim, S. Miglani, Sarah Elmes, V. Olivas, Hani Goodarzi, T. Bivona
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Heterogeneity in cancer gene expression is typically linked to genetic and epigenetic alterations, yet post-transcriptional regulation likely influences these patterns as well. However, the quantitative contribution of post-transcriptional mechanisms to cancer transcriptome dynamics remains unclear. Here, we systematically measured mRNA dynamics across diverse breast cancer models, revealing that mRNA stability significantly shapes gene expression variability. To decipher the regulatory grammar underlying these dynamics, we developed GreyHound, an interpretable multimodal deep-learning framework integrating RNA sequence features and RNA-binding protein (RBP) expression. GreyHound identified an extensive network of RBPs and their regulons underlying variations in mRNA stability. Among these, we uncovered a metastasis-suppressive regulatory axis centered on the RNA-binding protein RBMS3 and its post-transcriptional control of the redox regulator TXNIP. Functional and molecular analyses revealed that RBMS3 depletion resulted in targeted transcript destabilization, which was associated with poor clinical outcomes and enhanced metastatic potential in xenograft models. Using in vivo epistasis studies, we confirmed that RBMS3 regulation of TXNIP mRNA stability drives this metastasis-suppressive program. These findings position the RBMS3-TXNIP regulatory axis as a key post-transcriptional mechanism in breast cancer and illustrate how interpretable models of RNA dynamics can uncover hidden regulatory programs in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a157de4d34fa592e866b9d4ddb110508f65adde6" target='_blank'>
              Integrative analysis of mRNA stability regulation uncovers a metastasis-suppressive program in breast cancer
              </a>
            </td>
          <td>
            Heather Karner, Tabea Mittmann, Vicky W. Chen, Ashir A. Borah, Andreas Langen, Hassan Yousefi, L. Fish, Balyn W. Zaro, A. Navickas, Hani Goodarzi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 11561


 Background:
 Liposarcoma (LPS) is characterized by unstable genomes and high occurrence of gene fusions. Hotspots of both recurrent and non-recurrent gene fusions can provide information about structural alterations in certain LPS subtypes. For instance, myxoid (M) LPS expresses the oncogenic
 FUS-DDIT3
 protein and is known to be fusion-driven. Moreover, disrupted locus 12q13-15 is an important feature of well-dedifferentiated (WD) and dedifferentiated (DD) LPS, and is the site of copy number alterations (CNAs) and gene fusions. Here, we used RNA sequencing to uncover hotspots of intra- and transchromosomal gene fusions in LPS patient samples, identifying potentially clinically relevant events in certain chromosomal regions.
 Methods:
 The BostonGene internal LPS cohort (n=150) was analyzed by bulk whole-transcriptome sequencing, using STAR-fusion for sequence calling. Quality control was performed using FastQC, FastQ Screen, RSeQC, and MultiQC. Tumor purity was assessed via pathological and bioinformatics examination with a threshold of 20%. One sample Poisson rate test was used to evaluate statistical significance of gene fusion hotspots.
 Results:
 We identified 4,080 gene fusions among four LPS subtypes (DDLPS, WDLPS, MLPS, and PLPS - pleomorphic LPS). Of those, 2,302 (56.4%) were intrachromosomal and 1,778 (43.6%) were transchromosomal. Over half of these fusions (1,263/2,302 intrachromosomal fusions, 54.9%; 1,047/1,778 transchromosomal fusions, 58,9%) were detected on chromosome 12. Most identified fusions occurred in the 12q13-15 region (q-value < 0.001), with q15 being especially prevalent in transchromosomal fusions (q-value < 0.001). The most prevalent recurrent fusion across our LPS cohort was
 FUS-DDIT3
 (N=21, MLPS samples). Another notable recurrent fusion was
 TRIO-TERT
 (N=4, DDLPS). We also identified extended gene fusion hotspots in regions containing important oncogenes such as
 MDM2
 and
 FRS2
 in LPS subtypes (DDLPS, WDLPS, PLPS) that are not considered fusion-driven (Table). These findings suggest an oncogenic role of such fusions in these LPS subtypes, along with known CNAs like
 MDM2
 amplification.
 Conclusions:
 Our comprehensive transcriptomic analysis of gene fusions in LPS samples uncovered both new and established hotspots of chromosomal rearrangements. Identification of such hotspots improves our understanding of LPS oncogenesis and thus can enhance the diagnostic accuracy and discovery of new biomarkers.


 Hotspots of gene fusions in liposarcoma.




 Hotspot cytoband
 Diagnosis
 q-value
 Important genes




 1q23.3
 DDLPS
 < 0.001

 ATF6



 1q24.3
 DDLPS, WDLPS
 < 0.001

 DNM3



 12q13.3
 MLPS
 < 0.001

 DDIT3



 12q14.1
 DDLPS, WDLPS
 < 0.001

 CDK4



 12q14.3
 DDLPS, WDLPS
 < 0.001

 HMGA2, YEATS4



 12q15
 DDLPS, WDLPS, PLPS
 < 0.001

 MDM2, FRS2, CPM



 16p11.2
 MLPS
 < 0.001

 FUS





">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df3d585e6b1b252046eb1085a1fdfa70c3a17980" target='_blank'>
              Detecting hotspots of intra- and transchromosomal fusions in liposarcomas by RNA sequencing.
              </a>
            </td>
          <td>
            Dmitrii Grachev, Danil Ivanov, Oleg Baranov, Vladimir Kushnarev, Melissa Clemons, Sheila T. Yong, N. Kotlov, Konstantin Chernyshov, A. Bagaev, N. Fowler, A. Conley, G.M. Cote, Sant P Chawla
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 10615


 Background:
 ATM
 is a tumor suppressor gene involved in DNA repair and telomere maintenance.
 ATM
 biallelic pathogenic or likely pathogenic germline variants (gPV) are associated with

 ataxia-telangiectasia syndrome. Monoallelic ATM gPV are associated with increased cancer risk; however, their contribution to carcinogenesis has not been elucidated. We sought to characterize the genomic landscape of gPV
 ATM
 -associated cancers in a pan-cancer cohort.
 Methods:
 ATM
 alterations (germline and somatic) were identified in patients (Pts) with a solid tumor diagnosis sequenced with MSK-IMPACT, an FDA-approved, tumor-normal paired targeted NGS. Genetic-imputed ancestry clinicopathologic characteristics and FACETS estimated allele-specific copy number profiles were evaluated. Samples with purity <0.2 or with undetermined copy number profiles were excluded. HRD score was evaluated using FACETS profiles. Both somatic and germline sequencing data were analyzed within protocol NCT01775072.
 Results:
 Among 40,136 Pts with cancer who underwent germline testing, 1.1% (n=442) harbored an
 ATM
 gPV, inclusive of 2 Pts with biallelic
 ATM
 gPVs. The most frequent mutations were: R2547_S2549del (n=17), K2756* (n=14) and V1268* (n=11). Among these, R2547_S2549del and K2756* were identified exclusively in Pts with European ancestry, while E343Ifs*2 (n= 9/10) and c.1065+1G>T (n= 6/7) were most prevalent in Pts with Ashkenazi Jewish (AJ) ancestry. Concomitant gPVs in other genes were observed in 10% (n=47), with the most frequent ones being
 APC
 (I1307K),
 MUTYH
 ,
 BRCA2, CHEK2
 , and
 BRCA1
 . 63,270 out of 86,039 tumor samples had usable FACETS profiles.
 ATM
 somatic allele-specific information was available for 67% (297/442) of Pts. 62% (197/317) of tumor samples from Pts with monoallelic
 ATM
 gPVs exhibited somatic biallelic (Bi)
 ATM
 inactivation, while 38% (120/317) retained monoallelic (Mono) ATM status. The underlying mechanism of Bi-ATM inactivation was loss of heterozygosity in 75% (149/197) and additional somatic
 ATM
 mutations in 25% (48/197). Samples from Pts with tumors known to be associated with ATM gPVs (Breast, Pancreatic and Prostate cancers) had significant enrichment in Bi-ATM inactivation compared to tumors without strong association with ATM gPVs (88% Vs 58%; p<0.01). Although samples with Bi-
 ATM
 had higher overall HRD-scores compared to Mono-
 ATM
 (median = 34 Vs 20; p<0.01), no significant enrichment in HRD-High phenotype was seen in Bi-
 ATM
 (23 Vs 18; p=0.49).
 Conclusions:
 Evaluation of a pan-cancer Pt population with
 ATM
 gPVs demonstrated a high prevalence of biallelic somatic inactivation. Most Pts with
 ATM
 gPVs who had malignancies implicated in
 ATM
 -associated cancer risk, had biallelic somatic inactivation in their tumors suggestive of their contribution to tumorigenesis. While biallelic ATM inactivation is associated with higher genomic instability, no enrichment in HRD phenotype was observed.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/171b6148e94d64a55d3944e70772bbb94d153ffa" target='_blank'>
              Clinicopathologic and allele-specific analysis of germline
 ATM
 alterations in a pan-cancer cohort.
              </a>
            </td>
          <td>
            Matteo Repetto, Daniel Muldoon, P. Selenica, Kanika Arora, Ying L. Liu, M. Carlo, Lauren Banaszak, Alicia J Latham, C. Bandlamudi, E. O'Reilly, M. E. Robson, B. Rousseau, D. Mandelker, A. Drilon, Britta Weigelt, K. Offit, Michael F. Berger, Y. Murciano-Goroff, Z. Stadler, Mohammad Abbass
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="Nucleoporin 98 (NUP98) fusion oncoproteins are strong drivers of pediatric acute myeloid leukemia (AML) with poor prognosis. Here we show that NUP98 fusion-expressing AML harbors an epigenetic signature that is characterized by increased accessibility of hematopoietic stem cell genes and enrichment of activating histone marks. We employ an AML model for ligand-induced degradation of the NUP98::KDM5A fusion oncoprotein to identify epigenetic programs and transcriptional targets that are directly regulated by NUP98::KDM5A through CUT&Tag and nascent RNA-seq. Orthogonal genome-wide CRISPR/Cas9 screening identifies 12 direct NUP98::KDM5A target genes, which are essential for AML cell growth. Among these, we validate cyclin-dependent kinase 12 (CDK12) as a druggable vulnerability in NUP98::KDM5A-expressing AML. In line with its role in the transcription of DNA damage repair genes, small-molecule-mediated CDK12 inactivation causes increased DNA damage, leading to AML cell death. Altogether, we show that NUP98::KDM5A directly regulates a core set of essential target genes and reveal CDK12 as an actionable vulnerability in AML with oncogenic NUP98 fusions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fac2ff38eb74ff54c2c90e5931c96b450175dda5" target='_blank'>
              Transcriptional and epigenetic rewiring by the NUP98::KDM5A fusion oncoprotein directly activates CDK12
              </a>
            </td>
          <td>
            Selina Troester, Thomas Eder, Nadja Wukowits, M. Piontek, P. Fernández-Pernas, Johannes Schmoellerl, B. Haladik, G. Manhart, M. Allram, Margarita Maurer-Granofszky, Nastassja Scheidegger, K. Nebral, Giulio Superti-Furga, Roland Meisel, B. Bornhauser, P. Valent, Michael N Dworzak, Johannes Zuber, Kaan Boztug, Florian Grebien
          </td>
          <td>2025-05-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ecdff2c617c821690715d7aa483fadd831dee7c" target='_blank'>
              The Splicing Factor PTBP1 interacts with RUNX1 and is Required for Leukemia Cell Survival
              </a>
            </td>
          <td>
            Arjun Dhir, Alexander Ethell, Riley Watkins, Calvin Lam, Kevin Tur-Rodriguez, Jim Persinger, Kasidy K. Dobish, Sipra Panda, Shannon M. Buckley, Samantha A Swenson, Sandipan Brahma, M. J. Rowley, R. K. Hyde
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="C1QBP (Complement Component 1 Q Subcomponent-Binding Protein) plays a critical role in maintaining cellular metabolism, but its function in radiation-induced damage remains unclear. In this study, we generated C1QBP-deficient Huh-7 hepatocellular carcinoma (HCC) cells using CRISPR/Cas9 technology and observed that C1QBP deficiency significantly enhanced radiation-induced damage, as indicated by reduced cell proliferation, impaired colony formation, and increased γ-H2AX foci, a marker of DNA double-strand breaks. Additionally, C1QBP deficiency resulted in elevated phosphorylation of key DNA damage response (DDR) molecules, ATM and CHK2, and caused pronounced S phase cell cycle arrest. Mechanistic investigations revealed that C1QBP modulates NF-κB nuclear activity via the AMPK signaling pathway. The loss of C1QBP reduced NF-κB nuclear translocation, further exacerbating radiation-induced damage. Reintroducing C1QBP alleviated DNA damage, enhanced cell proliferation, and improved survival following radiation exposure. These findings highlight the critical role of C1QBP in modulating HCC cells radiosensitivity and underscore its potential as a therapeutic target to enhance radiotherapy outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0fd333aa569fa02c5f14e774f4c85d5637baf1e" target='_blank'>
              C1QBP Modulates DNA Damage Response and Radiosensitivity in Hepatocellular Carcinoma by Regulating NF-κB Activity
              </a>
            </td>
          <td>
            Haitao Zhou, Yanjin Wu, Jiahui Meng, Xiaotong Zhao, Yujia Hou, Qin Wang, Yang Liu
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c37e8efeea3f80c12980365d094cd561d8a7d326" target='_blank'>
              LINE-1 ORF1p is a shared and immunogenic antigen in cancer
              </a>
            </td>
          <td>
            Wilson McKerrow, Megan Snyder, Chong Chu, Deena Kelly, Heike Keilhack, Liyang Diao
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The conceptional shifts on genome organization and hereditary variability occurred during transition from classical mendelian to current mobile or dynamic genetics. The main changed premises of this transition are firstly presented in detail. Mendelian genetics mainly conceived genome as the set of chromosomes with of all genes. Now genome semantics is changed. It comprises entire hereditary constitution of the cell, including both structural and dynamic aspects of coding, storage and transfer of species-specific information. There are three kinds of heritable changes: mutations, variations and epigenetic alterations. It is reasonable to discriminate in the genome two subsystems: Obligate genetic elements (OGE) and Facultative genetic elements (FGE). FGEs comprise various kinds of repeated DNA, mobile elements, amplicons, inserted viral and foreign DNA, B-chromosomes and cytobionts. FGEs are predominant genome content of many plants. The number and cell topography of FGEs are different in different cells/tissues and most eukaryote individuals. Changes in the structure or order of OGEs correspond to classical mutations. Various changes in FGEs it is reasonable to call variations. Facultative elements and their variations are the first genomic reaction on biotic and environmental challenges. Together with epigenetic alterations they implement the operational genomic memory. Three template genome processes Replication, Transcription, Translation and three basic genetic processes – Repair, Recombination and Segregation are capable to facultative expression according to principle: the unity of the whole and freedom of the parts. This is the essence of the presented generalized concept of the genome organization and hereditary variations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08181d3ed9f40145f41c1bd8a5154dfef8c50437" target='_blank'>
              Beyond Mendel and Morgan to the dynamic genome
              </a>
            </td>
          <td>
            M. D. Golubovsky
          </td>
          <td>2025-05-11</td>
          <td>Vavilovia</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2222f4aa3745d3827861310375ad2ef6308f3da" target='_blank'>
              Mapping genetic modifiers of epimutation rates reveals a punctuated-equilibrium model of CG methylome evolution
              </a>
            </td>
          <td>
            Zhilin Zhang, Wilma Wanney, Yangyang Xu, J. Zicola, Angela M. Hancock, Robert J. Schmitz, F. Johannes
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b94228db7f216ff8d9ace96b3b303dbe02cf6e0" target='_blank'>
              Circular RNA rising star: new findings of circPRMT5 in cancer.
              </a>
            </td>
          <td>
            DuanZhong Chen, Caiyan Yin, Chao Li, Jun He, Peng Wu
          </td>
          <td>2025-06-18</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Oncogenic viruses typically manipulate host cellular mechanisms to drive tumorigenesis. They exploit pioneering transcription factors to modify gene expression, enabling uncontrolled proliferation. These viruses alter chromatin accessibility and induce chromatin remodelling, disrupting DNA repair and promoting viral genome integration. Additionally, epigenetic reprogramming through mechanisms like DNA methylation and histone modifications silences tumor suppressor genes and activates oncogenes. Understanding these mechanisms is critical for identifying more improved therapeutic targets, improving diagnostics, and predicting disease progression. Advances in this field can guide the development of innovative treatments and early detection tools. This comprehensive review synthesizes existing knowledge on the contributions of oncogenic viruses such as hepatitis B virus (HBV), hepatitis C virus (HCV), human papillomavirus (HPV), and human T-cell leukaemia virus type 1 (HTLV-1), Epstein–Barr virus (EBV), human herpesvirus 8 (HHV-8), and Merkel cell polyomavirus (MCV) to cancer development, highlighting their therapeutic relevance and driving forward research in viral oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36b6df879c24dbe78b1949e0a2207cf7cabf08ee" target='_blank'>
              The role of pioneering transcription factors, chromatin accessibility and epigenetic reprogramming in oncogenic viruses
              </a>
            </td>
          <td>
            M. Kgatle, S. Mbambara, Leon Khoza, Olalekan Fadebi, T. Mashamba-Thompson, Mike Sathekge
          </td>
          <td>2025-06-16</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Historically, DNA sequence mutability has been considered relatively uniform and low in tumors with chromosomal instability (CIN), based on the assumption that high mutability would be detrimental in karyotypically aberrant contexts. Recent in silico analyses have challenged this view, suggesting some heterogeneity in mutation rates across CIN tumors; however, these predictions lack experimental validation. It also remains unclear how the intertumor variability of mutation rates compares to intratumor diversification and evolves along disease progression, whether mutation rates are functionally relevant in CIN cancers, and which mutational processes shape mutational accrual during CIN tumor onset and evolution. To address these gaps, we performed mutation accumulation experiments using clonal populations of patient-derived tumoroids from seven CIN, microsatellite-stable colorectal cancers (CRCs), and one microsatellite-unstable CRC. Each tumor exhibited a distinctive mutation rate footprint that was conserved among different clones from the same ancestor. In contrast, mutation rates diverged markedly across different tumors, with variations in magnitude within microsatellite-stable tumors as prominent as those distinguishing them from microsatellite-unstable tumors. New mutations reflected mutational processes associated with defective DNA replication and repair, which were not detected in normal tissues. Last, both mutation accumulation assays and high-depth whole-exome sequencing of subclonal variants showed higher mutation rates in metastatic lesions compared with matched primary tumors, suggesting positive selection for cells with increasing mutability during cancer dissemination. By providing an empirical assessment of mutation rates in human cancer, our data delineate heterogeneity, heritability, and progression-associated evolvability of DNA mutational instability as hallmarks of microsatellite-stable CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c57088e09b71e7ad2e34a58e69bb450d1b43d18" target='_blank'>
              Heterogeneity and evolution of DNA mutation rates in microsatellite stable colorectal cancer.
              </a>
            </td>
          <td>
            Elena Grassi, V. Vurchio, George D. Cresswell, Irene Catalano, B. Lupo, F. Sassi, Francesco Galimi, Sofia Borgato, Martina Ferri, Marco Viviani, Simone Pompei, Gianvito Urgese, Bingjie Chen, E. Zanella, F. Cottino, Mariangela Russo, G. Mauri, F. Pietrantonio, M. G. Zampino, L. Lazzari, S. Marsoni, A. Bardelli, M. Lagomarsino, A. Sottoriva, L. Trusolino, Andrea Bertotti
          </td>
          <td>2025-05-21</td>
          <td>Science translational medicine</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Abstract Ageing is a gradual biological process marked by a decline in physiological function, increasing susceptibility to disease, and mortality. Transposable elements (TEs) are repetitive DNA sequences capable of moving within the genome and thus potentially inducing mutations and disrupting normal cellular functions. Their mobile nature contributes to genomic variation, as transposition events can alter gene expression, chromosome structure, and the epigenetic landscape. To mitigate TE-induced damage, cells rely on epigenetic mechanisms, such as DNA methylation, histone modifications, and small RNAs, to repress TE activity. However, these silencing mechanisms become less effective with age, leading to increased TE activation. This review explores the dual role of TEs as both a cause and consequence of ageing, suggesting a complex relationship between TEs and the ageing process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca3b56d174d42a8f2b0813e61df3277c11215871" target='_blank'>
              Exploring the Relationship of Transposable Elements and Ageing: Causes and Consequences
              </a>
            </td>
          <td>
            Miriam Merenciano, Anaïs Larue, Chloé Garambois, William Vilas Boas Nunes, Cristina Vieira
          </td>
          <td>2025-05-15</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is an age-related expansion of white blood cell (WBC) progenitors linked to risk of hematological malignancy. Patients with cancer have increased CH prevalence compared to healthy populations, but the characteristics and relevance of CH in advanced urological cancers are unknown. We interrogated CH and circulating tumor DNA (ctDNA) in 299 patients with metastatic urothelial or renal cell carcinoma using error-corrected targeted sequencing of matched WBC DNA and plasma cell-free DNA (cfDNA). 73% of patients carried CH variants at ≥0.25% allele frequency, with 13% exhibiting large CH populations marked by variants ≥10%. CH presence, clone size, and genotype did not impact patient survival. However, CH variants frequently affected solid cancer driver genes and were not individually discriminable from ctDNA variants based on cfDNA features including fragment length. In contrast, matched WBC DNA sequencing to ≥25% of cfDNA depth sufficiently resolved CH from ctDNA variants. Serial profiling revealed ctDNA and CH temporal dynamics including treatment-related expansion of PPM1D-mutated CH clones following platinum chemotherapy. Our data reveal the molecular landscape of CH in urological cancers and suggest that CH interferes in clinical ctDNA genotyping. We urge test providers to comprehensively filter CH from ctDNA results using matched WBC sequencing and propose a cost-effective framework for its integration into existing plasma-only assays.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d83b52d733bf76b0d395be6ba59d1da7d2e76a09" target='_blank'>
              Clonal hematopoiesis in metastatic urothelial and renal cell carcinoma
              </a>
            </td>
          <td>
            Aslı D Munzur, J. Bacon, Francine Fishbein, C. Herberts, G. Donnellan, C. Bernales, Karan Parekh, G. Vandekerkhove, David C Müller, Yi Jou Ruby Liao, Maria Stephenson, Lucia Nappi, D. Khalaf, C. Maurice-Dror, Kim N Chi, B. Eigl, Christian Kollmannsberger, Maryam Soleimani, A. Wyatt
          </td>
          <td>2025-06-13</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Simple Summary A comprehensive study analyzed circulating cell-free DNA (cfDNA) from 30 oral squamous cell carcinoma (OSCC) patients, 199 public OSCC, and 192 normal samples to explore its potential in early-stage diagnostics and prognosis. The findings suggest that the cfDNA concentration correlates with the tumor stage, malignancy, and survival prognosis in OSCC patients. Deep genome sequencing of cfDNA revealed several genomic alterations: 1. Copy number variation analysis identified amplifications and deletions at loci 1q, 2q, 3p, 3q, and 8q22. Notably, amplifications of TP53 and PIK3CA, along with other genes related to keratinization, were identified in relapsed OSCC cases. Single nucleotide variations were observed in genes highly mutated in OSCC. 2. A novel fusion gene, TRMO-TRNT1, was detected in seven high-grade tumor samples. The parental genes, TRMO and TRNT1, are involved in tRNA modification and DNA repair, respectively. 3. Integrations of human papillomavirus, simian virus, and enterovirus were found in the OSCC samples, potentially indicating viral involvement in OSCC pathogenesis. These results underscore the utility of cfDNA as a non-invasive biomarker for OSCC, offering insights into tumor genomics and aiding in early detection and prognosis assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c52e96cadb3f868e649f16559709defe911b5c" target='_blank'>
              Applications for Circulating Cell-Free DNA in Oral Squamous Cell Carcinoma: A Non-Invasive Approach for Detecting Structural Variants, Fusions, and Oncoviruses
              </a>
            </td>
          <td>
            M. Bhattacharya, D. Yaniv, D. P. D’Souza, Eyal Yosefof, Sharon Tzelnick, R. Detroja, Tal Wax, A. Levy-Barda, G. Baum, A. Mizrachi, G. Bachar, Milana Frenkel Morgenstern
          </td>
          <td>2025-06-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Background NDC80, a crucial component of the kinetochore, plays a pivotal role in regulating cell mitosis. Recent studies reported that NDC80 regulates the proliferation of cancer cells and may be related to poor prognosis in cancer. However, the biological function and mechanism of NDC80 in esophageal squamous cell carcinoma (ESCC) have not yet been elucidated. In this study, we analyzed the expression and mutation of NDC80 in ESCC tissue and assessed its impact on ESCC cells in vitro. Methods A total of 52 ESCC tissues samples were collected and NDC80 gene status was analyzed by whole exome sequence. shRNA was used to knockdown NDC80 expression in two ESCC cell lines TE1 and ECA109 by targeting silence NDC80 gene. Cellular proliferation, apoptosis, migration, and invasion were examined. Results Copy number variants (CNVs) of NDC80 were detected in 40.4% (21/52) cancer tissues. Genes with local CNVs values greater than 0.3 were classified as “gain”. 23.1% (12/52) of cases showed CNVs gain and 58.3% (7/12) of which with genotype of AAB. Compared with patients of other NDC80 CNVs status, CNVs gain patients had relatively shorter overall survival, however with no statistic difference (19.9 ± 0.8 months vs. 30.9 ± 5.1 months, p = 0.124). Single nucleotide polymorphism mutation of NDC80 was detected in only 5.8% (3/52) cancer tissues including two previous reported mutation rs2271754 and rs6506019, and one novel SNP of C/A-T/G in 2,577,492, GRCh38.p13 intron variant in one case. The knockdown of NDC80 in vitro assays significantly suppressed cellular proliferation and induced increased apoptosis in both TE1 and ECA109 cells. Furthermore, the migration and invasion ability of both TE1 and ECA109 cells were also attenuated upon knockdown of NDC80. Conclusion CNVs were identified as the predominant genetic variation of NDC80 gene in ESCC patients. In vitro assays suggested that NDC80 was not only involved into proliferation and apoptosis but also migration and invasion of ESCC cells. Supplementary Information The online version contains supplementary material available at 10.1186/s12876-025-04048-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16d73cd77e8485bddfd9d633b8f7422ebd5f99a7" target='_blank'>
              Expression of kinetochore component NDC80 promotes esophageal squamous cell carcinoma cells proliferation and migration
              </a>
            </td>
          <td>
            Youwu Shi, Jing Sun, Feng Du, Zhi-wei Sun, Ying Yang, Jing Yu, Yanjie Xiao, Xiaoyu Sun, Wen Zhang, Hongkun Zheng, Xiaodong Zhang, Jun Jia
          </td>
          <td>2025-06-05</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ef9a4a9c46c279f0d7bcc3c37222b01d2f23045" target='_blank'>
              Minimal activation of the p53 DNA damage response by a modular cytosine base editor enables effective multiplexed gene knockout in induced pluripotent stem cells
              </a>
            </td>
          <td>
            R. Blassberg, Olga Mielczarek, J. Stombaugh, H. Okkenhaug, Alice Abreu Torres, Paul Russell, John Lambourne, Immacolata Porreca
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 The apolipoprotein B mRNA editing enzyme, catalytic polypeptide- like 3 (APOBEC3 or A3) family of cytosine deaminases has been implicated in some of the most prevalent mutational signatures in cancer. A3-associated mutational signatures have been identified in more than 70% of cancer types and around 50% of all cancer genomes, with prominence in breast cancer as well as other cancer types. Many tumors are hypermutated by C-to-T/G mutations induced by A3 within TCW (W:T,A) motifs, accounting for many driver mutations in genes such as PIK3CA, ERBB2, and PPP2R1A. The genome and transcriptome of tumor tissue biopsies contain a wealth of information about these A3-associated signatures. However, quantification of A3 expression changes in tumor cells is confounded by the ubiquitous expression of these enzymes in infiltrating immune cells. To overcome this we will utilize single-cell RNA-seq analysis across a wide range of publicly available breast cancer datasets in order to follow the relative changes of A3 expression within individual cell types. Results from this analysis will give our lab insights into to changes in A3 expression in tumor cells which will be helpful in unveiling the complex relationships among breast cancer cells, stromal elements, and immune cells, facilitating the development of personalized therapies and prognostic markers, while also shedding light on breast cancer heterogeneity and potential vulnerabilities.
 Citation Format: Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi. Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-06-27.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4bd3dd6ddb39a7b246a7fddf688cff30066edc6" target='_blank'>
              Abstract P2-06-27: Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data
              </a>
            </td>
          <td>
            Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Mammalian cells have numerous DNA repair pathways to repair lesions generated by replication errors, metabolism, and exogenous agents. Cells can sense and respond to DNA damage within seconds, suggesting that there is a highly effective sensor of lesions although the mechanistic details are unclear. The DNA damage response in mammalian cells results in a localized transient de-condensation of chromatin, loss of linker histones and the recruitment of DNA repair proteins such as PARP1 and chromatin remodelers. Results Here we investigated the interactions between poly(ADP-ribose) polymerase-1 (PARP1), the linker histone H1.0 and linker histone chaperone Prothymosin α (PTMA). Using H1.0 tagged with a photoconvertible fluorescent protein, we observed a significant increase in the initial rate of exit of H1.0 from regions of chromatin containing microirradiation-induced DNA lesions. Surprisingly, this was also seen in Parp1−/− cells but not in stable cell lines with homozygous null mutations in the PTMA gene (Ptma−/−). The recruitment of PARP1 to damaged DNA was inhibited by overexpression of a mutant of H1.0 with a tighter chromatin-binding affinity or by reduced expression of PTMA. Relative to the wild type, Ptma−/− cell lines displayed increased sensitivity to DNA-damaging agents. Conclusion We suggest that DNA damage alters the interaction of H1.0 with the nucleosome to allow the chaperone PTMA to bind and promote release of linker histones thereby initiating the local chromatin de-condensation necessary for the efficient recruitment of repair proteins such as PARP1. In this context linker histones may serve as in situ “sensors” of DNA damage. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00599-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76b434110b02c9c576f6e6173ea01f3a42ac50b9" target='_blank'>
              The linker histone chaperone Prothymosin α (PTMA) is essential for efficient DNA damage repair and the recruitment of PARP1
              </a>
            </td>
          <td>
            Ciara A McKnight, Mary E Graichen, Eric M George, David T. Brown
          </td>
          <td>2025-06-05</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00047d76a26a1c71865a20d3d10212996573c803" target='_blank'>
              A recurrent pathogenic BRCA2 truncating variant reveals a role for BRCA2-PCAF complex in modulating NF-κB-driven transcription
              </a>
            </td>
          <td>
            Anna Minello, Jesus Gomez-Escudero, S. C. Sridhara, Charlotte Martin, Elodie Girard, Juan C. Cañas, Yasin Memari, Maria R. Bustos, A. Galarreta, Virginie Boucherit, E. Imyanitov, S. K. Bodvarsdottir, Sylvain Baulande, A. Vincent-Salomon, Nicolas Servant, Matthias Altmeyer, Stefan Sigurdsson, D. Stoppa-Lyonnet, S. Nik-Zainal, A. Carreira
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3593951b80dada0fabedb9d7b8680116321df128" target='_blank'>
              Development of Paclitaxel Resistance in Triple-Negative Breast Cancer Is Associated with Extensive DNA Methylation Changes That Are Partially Reversed by Decitabine
              </a>
            </td>
          <td>
            Lenka Trnkova, M. Burikova, A. Soltysova, A. Ficek, Jana Plava, Andrea Cumova, Lucia Rojikova, Kristina Jakic, Eva Sedlackova, Boris Tichý, V. Bystrý, F. Busato, Yimin Shen, M. Matúšková, L. Kucerova, J. Tost, S. Miklikova, M. Cihova, V. Buocikova, B. Smolková
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Epitranscriptomics has emerged as a rapidly evolving field that focused on studying post-transcriptional RNA modifications and their role in spatiotemporal regulation of gene expression. N6-methyladenosine (m6A) and 5-methylcytosine (m5C) represent the most extensively studied modifications on mRNAs. These reversible modifications, mediated by ‘writer,’ ‘eraser,’ and ‘reader’ proteins, dynamically fine-tune mRNA stability, splicing, and translation. Growing evidence links their dysregulation to pathological states, including cancer progression and metastasis, where their aberrant deposition on oncogenes or tumor suppressors alters cellular signaling and therapeutic responses. In the current study, we present a detailed analysis of the m5C epitranscriptomic landscape across distinct breast cancer molecular subtypes. Using CRISPR/Cas9, we confirm NSUN2 as a key m5C writer in human mRNAs. NSUN2 loss was validated by targeted sequencing and Western blotting. Furthermore, we demonstrate the regulatory effects of NSUN2 on its canonical mRNA targets, revealing its critical role in maintaining proper gene expression networks. Our findings strongly suggest that additional m5C writers contribute to m5C methylation machinery. Additionally, we assessed the functional impact of NSUN2 depletion on mRNAs harboring m5C sites using mRNA stability assays. Furthermore, our analysis revealed distinct m5C methylation patterns among breast cancer subtypes, highlighting unique m5C signatures associated with the disease. Notably, we identified specific hypomethylated and hypermethylated m5C sites in each breast cancer cell line, representing a universal m5C methylation signature for breast cancer. Our study constitutes the first comprehensive m5C epitranscriptomic atlas in human breast cancer and paves the way for future research aimed at developing targeted therapeutic interventions that leverage the m5C methylation landscape. Supplementary Information The online version contains supplementary material available at 10.1007/s10142-025-01648-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c7ab96a995706a16195a85c4f63f01b36d8d6b" target='_blank'>
              Exploring the m5C epitranscriptome of mRNAs in breast cancer cells through genome engineering and long-read sequencing approaches
              </a>
            </td>
          <td>
            Konstantina Athanasopoulou, P. Adamopoulos, P. Tsiakanikas, Andreas Scorilas
          </td>
          <td>2025-06-25</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0631bffe60efc9e321a6ea4a06deb1f550391138" target='_blank'>
              Clonal effects of the Ras oncogene revealed by somatic mutagenesis in a Drosophila cancer model
              </a>
            </td>
          <td>
            Takuya Akiyama, M. Gibson
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea81a914a4a5b1e0240796b0fe52e36431943de7" target='_blank'>
              Genetic barcoding uncovers the clonal makeup of solid and liquid biopsies and their ability to capture intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            A. Serrano, Tom S. Weber, J. Berthelet, Sarah Ftouni, Farrah El-Saafin, Verena C. Wimmer, K. Rogers, Elgene Lim, Emmanuelle Charaffe-Jauffret, Christophe Ginestier, David Williams, Frédéric Hollande, B. Yeo, Sarah-Jane Dawson, S. Naik, D. Merino
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61956d7589478355803ed0351b120e6a0e356110" target='_blank'>
              SLFN11 Enhances Cisplatin Sensitivity in Pediatric Cancer via Activation of Stress-Response and Suppression of Survival Pathways
              </a>
            </td>
          <td>
            A. Jabeen, Dina Awartani, S. Sherif, Eiman I. Ahmed, R. Alanany, Ayman Saleh, W. Hendrickx, C. Raynaud
          </td>
          <td>2025-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Retroviral and retrotransposon invasion pose a constant threat to genome integrity and have driven the evolution of host defense pathways able to counter these attacks. The human silencing hub (HUSH complex) is an epigenetic transcriptional repressor complex that recognizes and silences newly integrated retroelements through the establishment of ectopic heterochromatin and chromatin compaction. HUSH provides a genome-wide immunosurveillance system whose challenging task is to detect and silence any newly integrated retroelements, and it thus plays a key role in host defense. To distinguish self from nonself genomic DNA, HUSH recognizes long single-exon (intronless) DNA, the essential hallmark of reverse transcription. Retroelements, being RNA derived, lack classical, noncoding cellular introns, so a long, intronless sequence of DNA is the abnormal molecular pattern that allows HUSH to distinguish invading retroelements from intron-containing host genes. As a newly identified component of the innate immune system, HUSH protects the genome from the reverse flow of genetic information from RNA to DNA, revealing an unanticipated mechanism of postintegration genome immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8b1d5426e615e5a445a45b06d1bcad8bd22f0c5" target='_blank'>
              Silencing by the HUSH Epigenetic Transcriptional Repressor Complex.
              </a>
            </td>
          <td>
            Paul J. Lehner
          </td>
          <td>2025-06-01</td>
          <td>Annual review of biochemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eca7843e2f4903653396c94ffa8fe3fdaa395232" target='_blank'>
              ZNF280A links DNA double-strand break repair to human 22q11.2 distal deletion syndrome.
              </a>
            </td>
          <td>
            Thomas L Clarke, Hyo Min Cho, Ilaria Ceppi, Boya Gao, Tribhuwan Yadav, Giorgia G Silveira, Ruben Boon, Barbara Martinez-Pastor, Nana Yaa A Amoh, Belen Machin, Tiziano Bernasocchi, Dua Ashfaq, Josefina Mendez, Zeeba Kamaliyan, José Del Río Pantoja, Giuliana Sardi Rogines, Blaine T Crowley, D. McGinn, V. Giunta, Oanh Tran, E. Zackai, Li Lan, Lee Zou, B. S. Emanuel, Donna M. McDonald-McGinn, Petr Cejka, Raul Mostoslavsky
          </td>
          <td>2025-06-01</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Double‐strand breaks (DSBs) are universally acknowledged as the most detrimental type of DNA damage, and their effective repair primarily depends on the non‐homologous end joining (NHEJ) pathway. Such DSBs, which require NHEJ for resolution, can arise from intrinsic and extrinsic DNA‐damaging factors or emerge naturally during essential biological processes like V(D)J recombination and antibody class switch recombination.Failure to properly repair DSBs may lead to genomic instability, disruption of cellular functions, and immunodeficiency, thereby promoting the development of hematologic malignancies. Conversely, overexpression of NHEJ‐related genes can enhance resistance to DNA‐damaging therapies in these cancers. Analyzing mutations in key classical NHEJ (cNHEJ) components and understanding their mechanisms could provide valuable biomarkers for predicting therapeutic outcomes and guiding treatment decisions. Consequently, defects in cNHEJ may offer insights into the development of novel drugs targeting DNA repair pathways.We focus on genetic changes and alterations in gene regulation, while also providing an overview of cNHEJ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37d192621ee63d84a5e302bfd454c006794e79d5" target='_blank'>
              Current advances in the role of classical non‐homologous end joining in hematologic malignancies
              </a>
            </td>
          <td>
            Pengcheng Liu, Zi-zhen Xu
          </td>
          <td>2025-05-29</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mitochondria play critical roles in energy production and cellular metabolism. Despite the Warburg effect, mitochondria are crucial for the survival and proliferation of cancer cells. Heat Shock Factor 1 (HSF1), a key transcription factor in the cellular heat shock response, promotes malignancy and metastasis when aberrantly activated. To understand the multifaceted roles of HSF1 in cancer, we performed a genome-wide CRISPR screen to identify epistatic interactors of HSF1 in cancer cell proliferation. The verified interactors of HSF1 include those involved in DNA replication and repair, transcriptional and post-transcriptional gene expression, and mitochondrial functions. Specifically, we found that HSF1 promotes cell proliferation, mitochondrial biogenesis, respiration, and ATP production in a manner dependent on TIMM17A, a subunit of the inner membrane translocase. HSF1 upregulates the steady-state level of the short-lived TIMM17A protein via its direct target genes, HSPD1 and HSPE1, which encode subunits of the mitochondrial chaperonin complex and are responsible for protein refolding once imported into the matrix. The HSF1- HSPD1/HSPE1-TIMM17A axis remodels the mitochondrial proteome to promote mitochondrial translation and energy production, thereby supporting robust cell proliferation. Our work reveals a mechanism by which mitochondria adjust protein uptake according to the folding capacity in the matrix by altering TIM complex composition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/494bb0a10315606aa65eb17cf96c1f4f6db57aeb" target='_blank'>
              HSF1 remodels mitochondrial biogenesis and function in cancer cells via TIMM17A
              </a>
            </td>
          <td>
            Ngoc Gia Tuong Nguyen, Hem Sapkota, Yoko Shibata, Aleksandra Fesiuk, Matthew Antalek, Vibhavari Sail, Daniel J. Ansel, David R. Amici, F. Peelor, Holly Van Remmen, Benjamin F. Miller, Marc L. Mendillo, Richard I. Morimoto, Jian Li
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a94b93bbdecb6b043f4125dc44dd985694b019b6" target='_blank'>
              ZNF865 (BLST) Regulates Human Cell Senescence and DNA Damage
              </a>
            </td>
          <td>
            Christian Lewis, H. Levis, Jonah Holbrook, Jacob T. Polaski, Timothy D. Jacobsen, S. Gullbrand, B. Diekman, J. Iatridis, Jason Gertz, Brandon Lawrence, Robert D Bowles
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Chromodomain Helicase DNA-binding protein 1-Like (CHD1L) is a chromatin remodeling enzyme increasingly recognized as an oncogenic factor promoting tumor progression and metastatic potential by orchestrating transcriptional programs that drive epithelial-mesenchymal transition (EMT), cytoskeletal remodeling, and metastatic dissemination. In parallel, CHD1L has emerged as a master regulator of tumor cell survival by regulating DNA damage response and repair and enforcing G1 cell cycle progression. Furthermore, CHD1L plays a key role in immune evasion pathways by regulating signaling cascades and by suppressing both apoptotic and non-apoptotic cell death. In particular, CHD1L is a key suppressor of PARthanatos, a caspase-independent mechanism triggered by poly(ADP-ribose) (PAR) polymer fragmentation and apoptosis-inducing factor (AIF) activation. By regulating SPOCK1, MDM2, and TCTP, CHD1L further supports survival under cellular stress. Its overexpression correlates with metastasis, therapy resistance, and poor prognosis across many solid tumors. This review covers CHD1L’s structure, oncogenic functions, and developmental origins, and highlights emerging therapeutic strategies that target CHD1L as a druggable vulnerability in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6296c3dbcbe7b1d1919c8debad6aee4011adbed2" target='_blank'>
              CHD1L in cancer and beyond: structure, oncogenic functions, and therapeutic potential
              </a>
            </td>
          <td>
            Sophia Clune, Paul Awolade, Hector Esquer, Qiong Zhou, Daniel V. LaBarbera
          </td>
          <td>2025-05-30</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="ABSTRACT RNA alternative splicing is a fundamental cellular process implicated in cancer development. Kaposi’s sarcoma-associated herpesvirus (KSHV), the etiological agent of multiple human malignancies, including Kaposi’s sarcoma (KS), remains a significant concern, particularly in AIDS patients. A CRISPR-Cas9 screening of matched primary rat mesenchymal stem cells (MM) and KSHV-transformed MM cells (KMM) identified key splicing factors involved in KSHV-induced cellular transformation. To elucidate the mechanisms by which KSHV-driven splicing reprogramming mediates cellular transformation, we performed transcriptomic sequencing, identifying 131 differentially alternative spliced transcripts, with exon skipping as the predominant event. Notably, these transcripts were enriched in vascular permeability, multiple metabolic pathways, and ERK1/2 signaling cascades, which play key roles in KSHV-induced oncogenesis. Further analyses of cells infected with KSHV mutants lacking latent genes, including vFLIP, vCyclin, and viral miRNAs, as well as cells overexpressing LANA, revealed their involvement in alternative splicing regulation. Among the identified splicing factors, FAM50A, a component of the spliceosome complex C, was found to be crucial for KSHV-mediated transformation. FAM50A knockout resulted in distinct splicing profiles in both MM and KMM cells and significantly inhibited KSHV-driven proliferation, cellular transformation, and tumorigenesis. Consistently, FAM50A knockdown suppressed the proliferation of PEL cells. Mechanistically, FAM50A knockout altered SHP2 splicing, promoting an isoform with enhanced enzymatic activity that led to reduced STAT3 Y705 phosphorylation in KMM cells. These findings reveal a novel paradigm in which KSHV hijacks host splicing machinery, specifically FAM50A-mediated SHP2 splicing, to sustain STAT3 activation and drive oncogenic transformation. IMPORTANCE Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies. Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33b22cecd1973d138799718526caad5582306174" target='_blank'>
              KSHV reprograms host RNA splicing via FAM50A to activate STAT3 and drive oncogenic cellular transformation
              </a>
            </td>
          <td>
            Shenyu Sun, Ling Ding, Karla Paniagua, Xian Wang, Yufei Huang, Mario A. Flores, Shou‐Jiang Gao
          </td>
          <td>2025-06-12</td>
          <td>mBio</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The nucleolus is a membraneless organelle and an excellent stress sensor. Any changes in its architecture or composition lead to nucleolar stress, resulting in cell cycle arrest and interruption of ribosomal activity, critical factors in aging and cancer. In this study, we identified and described the pivotal role of the RNA-binding protein HNRNPK in ribosome and nucleolar dynamics. We developed an in vitro model of endogenous HNRNPK overexpression and an in vivo mouse model of ubiquitous HNRNPK overexpression. These models showed disruptions in translation as the HNRNPK overexpression caused alterations in the nucleolar structure, resulting in p53-dependent nucleolar stress, cell cycle arrest, senescence, and bone marrow failure phenotype, similar to what is observed in patients with ribosomopathies. Together, our findings identify HNRNPK as a master regulator of ribosome biogenesis and nucleolar homeostasis through p53, providing what we believe to be a new perspective on the orchestration of nucleolar integrity, ribosome function and cellular senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f5c155c3ee2cccb1bcb1a6518d0d3272385f671" target='_blank'>
              The tumor suppressor HNRNPK induces p53-dependent nucleolar stress to drive ribosomopathies
              </a>
            </td>
          <td>
            P. Aguilar-Garrido, María Velasco-Estévez, M. A. Navarro-Aguadero, Á. Otero-Sobrino, Marta Ibáñez-Navarro, Miguel Ángel Marugal, María Hernández-Sánchez, P. Malaney, Ashley G Rodriguez, Oscar Benitez, Xiaorui Zhang, M. Aitken, Alejandra Ortiz-Ruiz, Diego Megias, M. Pérez-Martínez, Gadea Mata, J. Gómez, M. Lafarga, Orlando Domínguez, Osvaldo Graña-Castro, Eduardo Caleiras, P. Ximénez-Embún, Marta Isasa, P. J. de Andrés, S. Rodríguez-Perales, R. Torres-Ruiz, E. Revilla, R. García-Martín, Daniel Azorín, Josune Zubicaray, Julián Sevilla, O. Sirozh, V. Lafarga, Joaquín Martínez-López, S. Post, M. Gallardo
          </td>
          <td>2025-05-08</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9270b9d8496ccc10f9651565b149fb129450105b" target='_blank'>
              ATM and p53 in aging and cancer: a double-edged sword in genomic integrity.
              </a>
            </td>
          <td>
            Surya Nath Pandey, Muhammad Afzal, Jyoti Uikey, Subbulakshmi Ganesan, Swati Mishra, Pooja Bansal, Imran Kazmi, S. Alzarea, Waleed Hassan Almalk, Kavita Goyal, Gaurav Gupta, Mohit Rana
          </td>
          <td>2025-05-05</td>
          <td>Biogerontology</td>
          <td>1</td>
          <td>41</td>
        </tr>

        <tr id="Background Cervical cancer progresses through distinct precancerous stages, making early screening and intervention crucial for prevention. However, conventional screening modalities, such as cytology and HPV testing, face challenges related to sensitivity, specificity, and resource dependency. Circular RNAs (circRNAs), owing to their high stability and tissue-specific expression, have emerged as promising biomarkers, though their role in cervical carcinogenesis remains underexplored. In particular, the clinical utility of circRNAs for optimizing cervical cancer screening and early diagnosis has yet to be established. This study aimed to investigate the dynamic expression profiles of circRNAs across various stages of cervical cancer progression and identify potential biomarkers to enhance early detection. Methods CircRNA sequencing was performed on cervical tissues spanning normal cervical epithelium (NCE), high-grade squamous intraepithelial lesions (HSIL), and cervical squamous cell carcinoma (CSCC). Functional assays, including cell viability, colony formation, and apoptosis, were performed to assess the oncogenic potential of circPOLD1 and its interaction with YBX1 in cervical cancer cells. BaseScope and immunohistochemistry (IHC) were applied to tissue microarrays for clincial validation and ROC curve analysis evaluated the diagnostic performance of circPOLD1 in serum as a liquid biopsy marker. Results CircRNA profiling revealed a progressive increase in circPOLD1 expression from NCE to HSIL and CSCC. Mechanistically, circPOLD1 functioned as an oncogene by binding to and phosphorylating YBX1, activating the AKT/mTOR/HIF-1α pathway to enhance glycolysis-driven tumorigenesis. BaseScope and IHC confirmed the stage-specific elevation of circPOLD1 and YBX1 in cervical lesions. The circPOLD1/YBX1 multi-marker panel demonstrated superior diagnostic performance, achieving an AUC of 0.951 for LSIL+ and 0.817 for HSIL+ detection. Furthermore, serum circPOLD1 levels exhibited a progressive increase across disease stages, underscoring its potential as a non-invasive biomarker. Conclusion circPOLD1 and YBX1 synergistically drive cervical carcinogenesis and exhibit stage-specific expression patterns. Their combined detection significantly enhanced the accuracy for cervical cancer screening and dynamic monitoring. The successful application of BaseScope and IHC highlights the immediate translational potential of these biomarkers, paving the way for refined risk stratification, improved therapeutic targeting, and reduced cervical cancer burden through early intervention. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06494-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f23f6664f03d6924fc8f8933932ffe6615430fec" target='_blank'>
              The biomarker potential of circPOLD1 and its binding protein YBX1 in cervical carcinogenesis
              </a>
            </td>
          <td>
            Lu Zhao, Xin Chen, Yanan Zhang, Y. Cen, Tingjia Zhu, Lingfang Wang, L. Xia, Yang Li, Xiaodong Cheng, Xing Xie, Weiguo Lu, Junfen Xu
          </td>
          <td>2025-05-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe12fe3c0e52aeffbd01092c11945a919867298" target='_blank'>
              Multimodal single-cell analyses reveal distinct fusion-regulated transcriptional programs in Ewing sarcoma
              </a>
            </td>
          <td>
            Olivia G. Waltner, April A. Apfelbaum, Emma D. Wrenn, Shruti S Bhise, Sami B. Kanaan, R. Gladden, Mark A. Mendoza, R. Volden, Z. Kronenberg, Anand G. Patel, Michael A Dyer, J. Sarthy, Elizabeth R. Lawlor, Scott N. Furlan
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background: DNA replication and sister chromatid cohesion1 (DSCC1) is a component of the alternative replication factor C complex (RFC), which plays an important role in sister chromatid cohesion and regulates the cell cycle. DSCC1 is reported to have increased expression in tumor progression, but the underlying mechanisms of DSCC1 in tumor immunity remain obscure. Methods: Data were collected from The Cancer Genome Atlas (TCGA) and Genotype-Tissue Expression (GTEx) project to obtain DSCC1 expression and clinicopathologically relevant data. Differential expression of DSCC1 and its association with prognosis, tumor microenvironment, immune infiltration, immune regulation, and genomic stability in various cancers were analyzed using R software. Results: The results of pan-cancer analysis showed that DSCC1 expression was elevated in 22 tumors. In 15 tumors, high expression of DSCC1 was observed to correlate with dismal overall survival. Furthermore, a correlation between DSCC1 expression and immune checkpoints was determined. In addition, DSCC1 expression was correlated with tumor-infiltrating immune cells, especially in Thymoma (THYM). Finally, gene set enrichment analysis (GSEA) demonstrated that DSCC1 is critically involved in tumor proliferation, immunity, and metabolism. Conclusions: High DSCC1 expression is found in many common tumors and is associated with poor prognosis. The gene is an essential factor in sister chromatid cohesion and may contribute to tumor progression by affecting the tumor immune microenvironment and genomic stability. DSCC1 has the potential to be a prognostic marker, and therapies targeting it may benefit patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175f7489907d723129eee83d91ea9ec6a461cb27" target='_blank'>
              Gene DSCC1 is a Potential Biomarker in Pan-cancer
              </a>
            </td>
          <td>
            Jialing Jiang, Xiaolong Yan, Dan Huang, Jieyu Chen, Xiaodan Hao
          </td>
          <td>2025-05-29</td>
          <td>Journal of Contemporary Medical Practice</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdb5b694d2a7c09771777d4a88a36c6a1c17392" target='_blank'>
              SP1 amplifies the effects of cellular proliferation via the Wnt/β-catenin pathway in colon development and cancer
              </a>
            </td>
          <td>
            Ankita Sharma, G. Dsilva, Saurabh J. Pradhan, Sudiksha Mishra, Kirankumar Santhakumar, Sanjeev Galande
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Non-replicative herpes simplex virus type 1 (nrHSV-1) vectors are promising delivery vehicles for gene therapy due to their large DNA payload capacity and ability to infect a broad range of cell types. However, the genomic deletions made to generate such nrHSV-1 vectors can result in undersized genomes that trigger genomic instability—including rearrangements and size extensions—compromising their therapeutic potential. This study investigates the stabilization of undersized nrHSV-1 vectors through the insertion of stuffer DNA segments. We assess genomic stability, productivity, toxicity, and transgene expression in vitro and in vivo. Our findings demonstrate that nrHSV-1 can accommodate variations in genome size up to 5–6% and highlight the importance of maintaining a genome size close to that of the wild-type HSV-1 for enhanced genomic stability and sustained transgene expression without adverse effects. This strategy offers a promising approach for optimizing nrHSV-1 vectors for clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63984927847424424cc3892babc095b69c28460a" target='_blank'>
              Genomic Size Is Critical to Guarantee the Genomic Stability of Non-Replicative HSV1 Vectors
              </a>
            </td>
          <td>
            Justine Basset, Alexandra Seraffin, Julien Ratelade, Yohann Dickx, Tomasz Benedyk, Grzegorz Sarek, Teddy Jégu, Alberto L. Epstein
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="HPV integration disrupts host genomic structure and expression, but whether these alterations promote cancer development remains unclear. Multiple genomic analyses of oropharyngeal cancers identified several host fusion genes, including recurrent FGFR3-TACC3 fusions, expressed from rearranged genomic loci adjacent to HPV integration sites. Evolutionary modeling implicated integration of virus concatemers into the host genome as a common initiating event in fusion formation. Co-expression of HPV16 E6/E7 and FGFR3-TACC3, but neither alone, was sufficient for tumor development in both xenograft and syngeneic mouse models and led to unique transcriptional programs implicated in carcinogenesis. FGFR3-TACC3 expression decreased the ubiquitination and degradation of E6 and E7, thereby increasing oncoprotein abundance. We conclude that expression of HPV16 oncoproteins and host gene fusions generated from HPV integration sites can be sufficient for cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87641a322ac1dbfc7184a0c61b91e897e328415f" target='_blank'>
              Human papillomavirus integration induces oncogenic host gene fusions in oropharyngeal cancers.
              </a>
            </td>
          <td>
            N. Khan, Keiko Akagi, Shiming Jiang, J. D. Dunn, Bo Jiang, Weihong Xiao, Madison P O'Hara, Li Shen, Qi Wang, Vakul Mohanty, Jing Wang, Sara Goodwin, Jamie L Hutchins, Kevin R. Coombes, Jagannadha K Sastry, D. Symer, M. Gillison
          </td>
          <td>2025-05-14</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Given their rarity and diversity, a fundamental understanding of the genomic underpinnings for many sarcoma subtypes is still lacking. To better define the molecular landscape of this group of diseases, we perform matched whole exome sequencing and RNA sequencing on a cohort of 1340 sarcoma tumor specimens. We identify recurrent somatic mutations and observe an increased mutational burden in metastatic vs. primary samples (p < 0.001). We observe frequent copy number alterations including whole genome doubling, with this feature being more common in metastatic tumors (p = 0.026). Estimation of immune cell abundances followed by hierarchical clustering identifies five immune subtypes ranging from low to high and we observe inferior overall survival in immune deplete clusters compared to immune enriched (p < 0.01). Interestingly, GIST predominantly form a distinct “immune intermediate” cluster that is marked by a specific enrichment for NK cells (FDR < 0.01).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09dedff6b3501b2376796d9aaef10264491ab2ba" target='_blank'>
              Genomic, transcriptomic, and immunogenomic landscape of over 1300 sarcomas of diverse histology subtypes
              </a>
            </td>
          <td>
            Alex C. Soupir, Oscar E. Ospina, Oliver Hampton, Michelle L. Churchman, Michael D. Radmacher, D. Hedges, David M. McKean, Phaedra Agius, Saman Zeeshan, Nathan D. Seligson, Raphael Pollock, D. Liebner, James L. Chen, G. Tinoco, B. Salhia, Martin D McCarter, B. Wilky, Benjamin J. Miller, M. Cavnar, John S. Groundland, Bryan P. Schneider, Gregory Riedlinger, Stephen B. Edge, C. Moskaluk, Kenneth Cardona, A. R. Naqash, Ricardo J. Gonzalez, J. Mullinax, David M Joyce, O. Binitie, G. Douglas Letson, A. Naghavi, M. Druta, Damon R Reed, E. Siegel, Jamie K Teer, Brooke L. Fridley, Andrew S Brohl
          </td>
          <td>2025-05-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Background: While recent advances have improved outcomes for Triple Negative Breast Cancer (TNBC), it continues to have a poor prognosis. Long non-coding RNAs are among a recent class of epigenetic regulators that function in the nucleus to support the stability of cells and maintain the fidelity of chromatin interactions. Our laboratory discovered the long noncoding RNA MANCR (LINC00704) as being upregulated in human breast cancer. Furthermore, our work demonstrated that MANCR is enriched in TCGA breast cancer patient samples that are not estrogen or progesterone receptor positive. We also demonstrated that the 10-year survival in this TCGA analysis is substantially worse in patients with high MANCR expression. More recently, we have found MANCR to be aberrantly expressed in TNBC cells, and these cells are highly dependent on MANCR to retain their tumorigenic characteristics. Methods: Functional in vitro studies in MDA-MB-231 TNBC cells used short antisense nucleic acids (GapmerRs) to knockdown MANCR. For in vivo studies, TNBC cells were injected into the mammary fat pad of mice, allowing tumor formation, and subsequent treatment with 2 nmol/g of a negative control or MANCR targeting GapmeR. We also identified genome interaction sites at single nucleotide resolution by chromatin isolation by RNA purification sequencing (ChIRP-seq) to determine the mechanism of MANCR activity in TNBC cells. Results: We now demonstrate that MANCR knockdown promotes DNA damage and decreases cell proliferation, migration, anchorage-independent colony formation, transwell invasion, and cellular survival. Additionally, in vivo targeting of MANCR drastically inhibited tumor growth over time and the end-point tumor mass. The MANCR GapmeR treatment also inhibited the ability of TNBC cells to circulate and disseminate to distant organs in vivo. After performing ChIRP-seq in the MDA-MB-231 cells, we identified 1206 genome-wide binding sites that exhibit MANCR interactions, of which 48% are intergenic and 52% are in genic regions. Furthermore, many MANCR ChIRP peaks were found to overlap with fragile sites in the genome, indicating MANCR provides stability to these sites. Conclusions: These data suggest that targeting MANCR has therapeutic potential for patients with “MANCR-high” TNBC tumors by disrupting genome stability. Indeed, our in vivo studies demonstrate that “MANCR-high” TNBC tumors require MANCR to rapidly grow and promote disease progression. Significantly, many of the MANCR-chromatin interactions identified were found in intergenic regions and overlap with fragile sites within the genome. Collectively, these data strongly indicate that MANCR stabilizes the TNBC genome, and disrupting genome stability by targeting MANCR has therapeutic potential.
 Citation Format: Janine S. A. Warren, Bodhisattwa Banerjee, Jonathan A. R. Gordon, Prachi N. Ghule, Janet L. Stein, Gary S. Stein, Jane B. Lian, Peter A. Kaufman. Mitotically-associated long noncoding RNA (MANCR): A novel long noncoding RNA that promotes genomic stability and cellular proliferation in Triple Negative Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-07-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbb40dcaca0c24edab0b1fb3286bc3d79cdad166" target='_blank'>
              Abstract P1-07-17: Mitotically-associated long noncoding RNA (MANCR): A novel long noncoding RNA that promotes genomic stability and cellular proliferation in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Janine Warren, Bodhisattwa Banerjee, J. Gordon, Prachi N Ghule, Jane L. Stein, G. S. Stein, J. Lian, Peter A. Kaufman
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Plant nuclear genomes contain thousands of genes of mitochondrial and plastid origin as the result of endosymbiotic gene transfer (EGT). EGT is a still-ongoing process, but the molecular mechanisms determining its frequency remain largely unknown. Here we demonstrate that nuclear double-strand break (DSB) repair is a strong suppressor of EGT. Through large-scale genetic screens in tobacco plants, we found that EGT from plastids to the nucleus occurs more frequently in somatic cells when individual DSB repair pathways are inactive. This effect is explained by the expected increase in the number and residence time of DSBs available as integration sites for organellar DNA. We also show that impaired DSB repair causes EGT to increase 5- to 20-fold in the male gametophyte. Together, our data (1) uncover DSB levels as a key determinant of EGT frequency, (2) reveal the strong mutagenic potential of organellar DNA and (3) suggest that changes in DNA repair capacity can impact EGT across evolutionary timescales.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7706b7ef66d1fdcd1d30d7b318635c18156686f3" target='_blank'>
              Suppression of plastid-to-nucleus gene transfer by DNA double-strand break repair
              </a>
            </td>
          <td>
            Enrique Gonzalez-Duran, Xenia Kroop, Anne Schadach, Ralph Bock
          </td>
          <td>2025-05-16</td>
          <td>Nature Plants</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Background Ewing sarcomas (ESs) are aggressive paediatric tumours of bone and soft tissues afflicting children and adolescents. Despite current therapies having improved the 5-year survival rate to 70% in patients with localized disease, 25% of patients relapse and most have metastasis at diagnosis. Resistance to chemotherapy, together with the high propensity to metastasize, remain the main causes of treatment failure. Thus, identifying novel targets for alternative therapeutic approaches is urgently needed. Methods Biochemical and functional analyses were carried out to elucidate the mechanism of regulation of YAP1 expression by pncRNA_YAP1-1 in ES cells. Results Here, we identified a novel promoter-associated noncoding RNA, pncRNA_YAP1-1, transcribed from the YAP1 promoter in ES cells. We found that pncRNA_YAP1-1 level exerts antitumour effects on ES by destabilizing YAP1 protein. The molecular mechanism relies on the interaction of pncRNA_YAP1-1 with the RNA binding protein FUS, which stabilizes the transcript. Furthermore, pncRNA_YAP1-1 binding to TEAD impairs its interaction with YAP1, thus determining YAP1 translocation into the cytoplasm, its phosphorylation and degradation. Conclusions Overall, our findings reveal a novel layer of regulation of YAP1 protein expression by pncRNA_YAP1-1 in Ewing sarcoma. Considering the role of YAP1 in therapy response and cell propensity to metastasize, our results indicate pncRNA_YAP1-1 as an actionable target that could be exploited to enhance chemotherapy efficacy in Ewing sarcoma. Significance PncRNA_YAP1-1 counteracts the YAP1 oncogenic transcriptional program in Ewing sarcoma cells by interfering with YAP1-TEAD interaction and impairing YAP1 protein stability. These findings uncover a novel treatment option for Ewing sarcoma. Graphical abstract Supplementary Information The online version contains supplementary material available at 10.1186/s11658-025-00736-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/945aa0f8515a96b425b539864e7b5d32769ff4ff" target='_blank'>
              YAP1 promoter-associated noncoding RNA affects Ewing sarcoma cell tumorigenicity by regulating YAP1 expression
              </a>
            </td>
          <td>
            L. Chellini, Arianna Del Verme, Veronica Riccioni, M. Paronetto
          </td>
          <td>2025-05-25</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 11529


 Background:
 Epigenetic dysregulation is a key mechanism in cancer biology, driving both immune evasion and the generation of potentially druggable neoantigen targets. Transposable elements (TEs) are proviral sequences mostly found in epigenetically repressed genomic regions. In cancer, TEs are one potential source of neoantigens. The tumor suppressor gene
 ATRX
 is an epigenetic regulator that is lost in up to 30% of soft tissue sarcomas. Loss of function of this heterochromatin modulator leads to widespread alterations to transcription patterns. Our prior work in sarcoma, along with studies in other tumors, showed that ATRX loss leads to the de-repression of TEs. We hypothesize that this mechanism leads to TE neoantigen presentation on MHC class I (MHC-I) molecules.
 Methods:
 Leveraging the gEVE database of proviral sequences with open reading frames, we examined the differential expression of TEs using RNA sequencing of multiple models comparing ATRX lost and ATRX wild-type tumors. Analysis included our previously published data from human undifferentiated pleomorphic sarcoma cell lines (2 clones, 3 replicates per group; GEO: GSE240030) and a publicly available dataset from a mouse model of soft tissue sarcoma (n = 3 per group; GEO: GSE167537). Additionally, we analyzed analogous data from a mouse model of glioblastoma multiforme (GBM), another malignancy with frequent ATRX loss (n = 3 per group; GEO: GSE178113). RNA-seq data was analyzed using Salmon for transcript-to-genome pseudo-alignment, DESeq2 for differential TE expression analysis, and arcasHLA for sample-level MHC-I subtype identification. Differentially overexpressed TEs in ATRX mutant tumors were analyzed for predicted neoantigens using the NetMHCpan4.1 package.
 Results:
 Across multiple models, ATRX loss of function alterations were associated with both activation and repression of TEs. Within two sarcoma contexts, upregulated TEs were found to have protein-coding sequences with the potential to lead to MHC-I neoantigen presentation. Within a human undifferentiated pleomorphic sarcoma cell line with ATRX loss, there were 7 upregulated TEs shared between the clones, each predicted to contain 12 – 50 MHC-I binding epitopes. Within a mouse soft tissue sarcoma model, 7 TEs were upregulated with ATRX loss, each with 5 – 28 MHC-I binding epitopes. Since ATRX loss alters TE expression in other malignancies, parallel data from a mouse GBM model was additionally analyzed. In this model, 308 TEs were found to be upregulated with ATRX loss, and all were predicted to generate neoantigens. ERVs and LINE1 elements were recurrently identified, making up 97 – 100% of the activated TEs.
 Conclusions:
 Transposable element transcription altered by ATRX loss in sarcoma and GBM creates a predicted source of MHC-I-binding neoantigens. These neoantigens come from TEs of the ERV and LINE1 families.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d406b838be4f31bb8e68fe40b2f943691a36673" target='_blank'>
              Predicting transposable element-derived neoantigens in ATRX-mutated sarcoma.
              </a>
            </td>
          <td>
            Simon Cao, Ziyang Wang, B. Nacev
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Androgen receptor (AR), a member of the nuclear receptor superfamily controls prostate epithelial cell plasticity by repressing a panel of genes involved in epithelial-mesenchymal transition (EMT), including the human CDH2 gene encoding N-cadherin. At the opposite, pathological AR variants such as AR-V7 associated with prostate tumor progression upregulate those EMT genes. Here, focusing on the human CDH2 gene, we show that this duality between AR and AR-V7 relies on a potential human accelerated region present in the intron 1. This fastest-evolving region of the human genome is actually a variable number tandem repeat (VNTR) comprising 24 repetitions of a DNA sequence that englobes binding sites for steroid hormone receptors, recombination signal binding protein for immunoglobulin kappa j region (RBPJ) an effector of the Notch pathway, and zinc finger e-box binding homeobox 1 (ZEB1). Genomic DNA sequencing, multiple sequence alignment, data mining, as well as protein-DNA interaction and gene expression analyses indicate that this VNTR constitutes a potential transcriptional hub for different transcription factors to control human CDH2 expression. Also, our data suggest that prostate tumor cells may unlock an up to now unknown molecular mechanism associated with a fine-tuned control of human CDH2 gene expression. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1007/s00018-025-05725-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b161c5324c0a97c9c54e04e62611e3f6b1ceb501" target='_blank'>
              Human-specific genomic evolution of a regulatory network enables fine-tuning of N-cadherin gene expression
              </a>
            </td>
          <td>
            É. Erdmann, Savera Agolli, Simon Fix, F. Cottard, Christine Keyser, V. Zvénigorosky, Angéla Gonzalez, Zakary Haili, Bruno Kieffer, J. Céraline
          </td>
          <td>2025-05-09</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Human papillomavirus associated head and neck cancer (HPV + HNC) is rising globally, emphasizing the need for improved therapeutic and screening strategies. To test novel therapies and study HPV-related disease progression, it is vital to develop relevant preclinical models. However, many fail to address critical concerns, including generating a representative immune microenvironment and adequately modeling HPV-driven malignant transformation. Recent multi-omics studies reveal the significance of HPV integration location in HPV-related carcinogenesis and highlight the necessity of targeted treatment methods. Thus, we have developed a murine model of HPV16 + HNC modifying the published CRISPR-based Somatic Oncogene kNock-In for Cancer Modeling (CRISPR-SONIC) system for precise integration of HPV oncogenes. We showed that CRISPR-SONIC knock-in of KrasG12D, HPV16 E6 and E7, and a luciferase reporter at the murine β-actin 3’-UTR locus could induce spontaneous buccal tumors with sarcomatous morphology under transient or selective immunosuppression. Both preventative and therapeutic pNGVL4a-CRT/E7(detox) DNA vaccination could induce HPV16 E7-specific immune response and reduce tumor growth. Furthermore, CRISPR-SONIC knock-in of HPV16 E6 and E7 with co-delivery of HNC-relevant oncogenes AKT and c-Myc produced tumors in NSG mice capturing the characteristic carcinomic morphology of HPV + HNC. Overall, our model offers a robust platform for evaluating new therapies and exploring HPV-related carcinogenesis. Supplementary Information The online version contains supplementary material available at 10.1186/s13578-025-01427-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c396727d3082a56902b37a32cd0299a9e91f92a0" target='_blank'>
              Generation of a spontaneous murine HPV + oral cancer model with site-specific oncogene insertion using CRISPR-SONIC
              </a>
            </td>
          <td>
            Julia Tao, Jason Murray, Hsin-Fang Tu, Darrell Fan, Y. Tsai, Ming-Hung Hu, Annie A Wu, Deyin Xing, Chien-Fu Hung, T-C Wu
          </td>
          <td>2025-06-18</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) contributes to 3% of all childhood cancers with roughly 400-500 cases diagnosed each year in the United States. The World Health Organization classifies rhabdomyosarcoma into four histological subtypes which include alveolar, embryonal, spindle-cell and pleomorphic. The primary genetic drivers in a subset of alveolar and spindle-cell histological subtypes are gene fusions. This review explores the fusion oncogenes identified in RMS such as PAX-and NCOA2-based fusions, along with discussing studies defining fusion oncogene biology and tumorigenic mechanisms. Focus areas include data around transformation events and progression along with dysregulated biological processes. Furthermore, we summarize model systems, ranging from cell to animal models, that have been implemented to study fusion oncogenes identified in RMS. With the constant identification of novel fusion oncogenes, this review also emphasizes the need for genetically characterizing RMS tumors and rapidly developing new model systems. These models are critical to study fusion oncogene activity and to delineate key regulatory players and potential therapeutic targets that suppress tumorigenesis. The identification of RMS fusion oncogenes and integration with animal and cell culture models will help identify conserved molecular targets, optimize therapeutic approaches, and ultimately improve clinical outcomes for children with RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b23d84102b5431b0bc6bd415779fd94bf54425c" target='_blank'>
              Fusion oncogenes in rhabdomyosarcoma: model systems, mechanisms of tumorigenesis, and therapeutic implications
              </a>
            </td>
          <td>
            Chinmay S. Sankhe, Lisa Hall, Genevieve C. Kendall
          </td>
          <td>2025-06-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 e15100


 Background:
 Intratumoral heterogeneity, driven by phenotypic plasticity, contributes to disease progression and therapy failures. However, a thorough understanding of its underlying mechanisms is still lacking. Breast cancer exemplifies this with its histological and molecular diversity; estrogen receptor-positive (ER+), progesterone receptor-positive (PR+), and HER2- luminal A subtypes, such as the MCF7 cell line, are not the exception. While single-cell RNA sequencing (scRNA-seq) and three-dimensional multicellular tumor spheroids (MCTS) represent initial approaches to understanding heterogeneity, integrating these tools with computational strategies offers new paths to uncover the regulatory groundwork of tumor behavior. When modeled via Boolean frameworks, transcriptional regulatory networks (TRNs) provide a dynamic platform for studying gene interactions, identifying attractors (stable cellular states corresponding to distinct phenotypes), and simulating cellular transitions. This study represents the first step in constructing such a framework for ER+ breast cancer to advance precision medicine and improve therapeutic strategies.
 Methods:
 Regulatory elements relevant to the MCF7 cell line were identified through an extensive review of NCBI literature. Interactions were strictly curated based on evidence quality (ranging from robust experimental validation to predictive hypotheses), and evaluated for type and significance, such as feedback loops, activation, coactivation, and inhibition. Logical operators (AND, OR, NOT) were used to translate regulatory connections into Boolean functions to simulate binary gene expression dynamics. The final TRN was visualized using Cytoscape (v3.10.3) with a hierarchical layout.
 Results:
 The TRN consists of 54 nodes and 81 interactions, representing the regulatory landscape of ER+, PR+, and HER2- breast cancer. Boolean equations offer a framework for simulating the evolution of binary states to attractors, enabling the observation of how perturbations, such as gene overexpression or silencing, influence heterogeneity. While simulation and attractor analyses are ongoing, the network provides a foundation for elucidating the regulatory mechanisms underlying breast cancer plasticity.
 Conclusions:
 This study highlights the potential of integrating experimental data, TRN reconstruction, and Boolean modeling to understand intratumoral heterogeneity. Our rigorously curated model establishes a foundational framework for precision oncology and the integration of experimental and computational methods.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10947cef519fbed46d1a42e2676b4ea1fd778e16" target='_blank'>
              Boolean network modeling: The first step toward a new understanding of breast cancer heterogeneity.
              </a>
            </td>
          <td>
            Lia Medina Montalvo, Thelma Escobedo-Tapia, Osbaldo Resendis-Antonio
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="AIMS & METHODS
DNA methylation and genomic instability are critical drivers of cancer initiation and malignant progression. However, the roles of methylation aberrations and genomic instability in malignant progression have not been thoroughly investigated in intrahepatic cholangiocarcinoma (ICC). To address this, we identified differentially methylated regions (DMRs) and somatic copy number alterations (SCNAs) from 341 ICC samples across various stages.


RESULTS
Our findings revealed that stages IAIB, II, IIIA, and IIIB exhibited comparable methylation changes, whereas stage IV ICC showed a pronounced accumulation of stage-specific methylation alterations. Leveraging these findings, we developed a classification model that effectively distinguished stage IV ICC from earlier stages with high accuracy using 15 DMRs. Furthermore, stage IV ICC exhibited slightly higher genomic instability, including an elevated aneuploidy score and a greater proportion of focal amplifications. We also observed a positive correlation between SCNA burden and DNA methylation entropy in the promoter, gene body, and CpG island regions, with the gene body of MDM2 serving as a notable example.


CONCLUSIONS
These findings highlight the potential of DNA methylation as a biomarker for metastasis diagnosis and the interplay between local genomic instability and aberrant methylation, emphasizing their synergistic roles in driving the evolutionary trajectory of ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d995c84a0cfeeb8c871fd5bd4ce5a040e83eca26" target='_blank'>
              Methylation aberrations and genomic instability synergistically drive the evolution of intrahepatic cholangiocarcinoma.
              </a>
            </td>
          <td>
            Guanghao Li, Youhuang Bai, Feng Tao, Tingting Hu, Ting Wang, Yong Zeng, Deqiang Sun
          </td>
          <td>2025-06-17</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Advances in cancer biology have been achieved by the identification of oncogenes and tumor suppressor genes through the remarkable progression of next‐generation sequencing. New techniques, such as single‐cell analysis, help uncover cancer progression and heterogeneity. Reverse genetic screenings, including methods like random mutagenesis via retroviruses, transposons, RNA interference, and CRISPR, are useful for exploring gene functions and their roles in cancer. Especially in random mutagenesis, CRISPR screening and its modifications have recently emerged as powerful tools due to their comprehensiveness and simplicity in inducing genetic mutations. Initially, CRISPR screening focused on analyzing biological phenotypes in a cell population. It has since evolved to incorporate advanced techniques, such as combining single‐cell and spatial analyses. These developments enable the investigation of cell–cell and spatial interactions, which more closely mimic In Vivo microenvironments, offering deeper insights into complex biological processes. These approaches allow for the identification of essential genes involved in cancer survival, drug resistance, and tumorigenesis. Together, these technologies are advancing cancer research and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8d93feca0cdb57db69fce7d9acc441dc2942da5" target='_blank'>
              Identification of Target Genes Using Innovative Screening Systems
              </a>
            </td>
          <td>
            Keisuke Sugita, Morita Kurata
          </td>
          <td>2025-05-05</td>
          <td>Pathology International</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The DNA damage response comprises a set of imperfect pathways that maintain cell survival following exposure to DNA damaging agents. Cancers frequently exhibit DNA repair pathway alterations that contribute to their intrinsic genome instability. This, in part, facilitates a therapeutic window for many chemotherapeutic agents whose mechanisms of action often converge at the generation of a double-strand DNA break. The development of therapy resistance occurs through countless molecular mechanisms that promote tolerance to DNA damage, often by preventing break formation or increasing repair capacity. This review broadly discusses the DNA damaging mechanisms of action for different classes of chemotherapeutics, how avoidance and repair of double-strand breaks can promote resistance, and strategic directions for counteracting therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77a69284f3f1f8221d2572829a8005590b029476" target='_blank'>
              DNA repair and the contribution to chemotherapy resistance
              </a>
            </td>
          <td>
            K. Nešić, Phoebe Parker, Elizabeth M. Swisher, John J Krais
          </td>
          <td>2025-05-26</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background To maintain homeostasis in the mature human body, certain differentiated cells adopted high plasticity to refine their cellular functions. However, mechanisms that supported cellular plasticity still remained elusive. Here, through comprehensive transcriptomic and epigenetic studies of highly plastic vascular smooth muscle cells (SMCs), we aimed to decipher the chromatin basis that could mediate cellular plasticity. Results In vascular smooth muscle cells, actively transcribed and highly adjustable genes tended to be associated with a continuously accessible region downstream of transcription start site (CAR-downTSS). This CAR-downTSS was located beyond the classic RNA polymerase II paused region, accessible at mono-nucleosome level and incorporated with histone variant H2A.Z. Depletion of H2A.Z reduced active histone modifications within CAR-downTSS, impaired RNA polymerase II transpassing when cells were stimulated, and consequently inhibited the ability of CAR-downTSS-associated genes to adjust their expression. Further in vitro and in vivo studies verified that this CAR-downTSS could be dynamically re-deployed onto different genes in vascular SMCs, whereas it was deployed in smaller quantities and remained quantitatively stable on genome within the quiescent cardiomyocytes. Conclusions Vascular SMCs dynamically deployed H2A.Z-positive nucleosomes extending continuously downstream transcription start sites on different genes to support their transcriptional adjustability, which served as an important mechanism mediating cellular plasticity. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11679-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4fec2cc361ec5fe548feef27516eeb74be0950f" target='_blank'>
              Dynamic deployment of H2A.Z positive nucleosome mediated transcriptomic plasticity within vascular smooth muscle cell
              </a>
            </td>
          <td>
            Chao Yu, Jiaxin Huang, Yan Wang, Jia Song, Wei-lei Shen
          </td>
          <td>2025-05-19</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Recent advancements in genomic technologies have become critical tools for deciphering the genetic complexities of cancer tissues, enabling precision medicine strategies aimed at improving patient clinical outcomes. Here we performed a comprehensive analysis of clinically annotated whole genome and whole transcriptome sequences from 1,364 breast cancer cases. Our investigation provides the most detailed genomic landscapes of breast cancer to date, which allowed us to comprehensively correlate genomic changes with clinical characteristics. Our findings include, but not limited to, (1) whole-genome-based homologous recombination deficiency profiles effectively predict responses to adjuvant chemotherapy and first-line CDK4/6 inhibitor treatment, (2) focal ERBB2 amplifications, often arising via extrachromosomal DNA mechanisms, are strongly associated with a positive response to neoadjuvant chemotherapy. Additionally, our study identified (3) recurrent copy number amplifications linked to patient survival, and (4) whole-genome-based intratumoral heterogeneity profiles that can predict patient survival as well as response to anti-HER2 therapies. Our work underscores the power of highly annotated whole-genome and transcriptome sequencing of a population-scale cancer cohorts for uncovering clinically relevant genomic mutations and novel molecular targets, thereby advancing precision oncology strategies for breast cancer.
 Citation Format: Ryul Kim,JongHan Yu, Joonoh Lim, Brian Baek-Lok Oh, Seok Jin Nam, Seok Won Kim, Jeong Eon Lee, Byung Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, Pil Sun Baek, Soo Yeon Bae, Chang Ik Yoon, Young Joo Lee, Dooreh Kim, Kabsoo Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, Erin Connolly-Strong, Sangmoon Lee, Bo Rahm Lee, Yuna Lee, Ki Jong Yi, Young Oh Kwon, In Hwan Chun, Junggil Park, Jihye Kim, Chahyun Choi, Jong Yeon Shin, Hyunjung Lee, Minji Kim, Hansol Park, Ilecheon Jeong, Boram Yi, Won-Chul Lee, Jeong Seok Lee, Woo Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Yeon Hee Park. Whole-genome landscapes of 1,364 breast cancers with clinical outcomes [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-01-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea155495e72da0b7ab53fc77458b87559af5c5e0" target='_blank'>
              Abstract P1-01-06: Whole-genome landscapes of 1,364 breast cancers with clinical outcomes
              </a>
            </td>
          <td>
            Ryul Kim, J. Yu, Joonoh Lim, Brian Baek-Lok Oh, S. Nam, S. Kim, Jeong-Eon Lee, Byung-Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, Pil Sun Baek, Soo Yeon Bae, C.I. Yoon, Young Joo Lee, D. Kim, K. Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, E. Connolly-Strong, Sangmoon Lee, Bo-Rahm Lee, Yuna Lee, Kijong Yi, Youngoh Kwon, In Hwan Chun, J. Park, Jihye Kim, Chahyun Choi, Jong-Yeon Shin, Hyunjun Lee, Minji Kim, H. Park, Ilecheon Jeong, Boram Yi, W. Lee, Jeongmin Lee, Woo Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Y. Park
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The relationship between TP53 and transposable elements (TEs) has been obscure. Given the important role of TEs in oncogenesis, a comprehensive profiling of TE expression dynamics under the regulation of TP53 provides valuable resources for more clarity in TP53's roles in cancer. In this study, we characterized the TE transcriptomic landscape using long-read RNA-seq and short-read RNA-seq in three cancer cell lines varying only in TP53 genetic status. To identify transcripts that use TEs as potential promoters, we developed a computational pipeline, TEProf3, and identified in total 1942 transcripts with high confidence. Among these TE-derived transcripts, 239 are activated by TP53 and 221 are repressed by TP53. These TP53-responsive TE-derived transcripts are mainly driven by members of the ERV and LINE families. Following knockdown of wild-type (WT) TP53 expression, rescuing WT TP53 expression allows for partial recovery of the TE expression profile observed in the context of chronic TP53 expression. TP53 mutations R175H and R273H manifest their oncogenic characteristic partially through activating TE promoters in a cell type–specific manner. Lastly, we identified important sequence motifs that help govern the interactions between TEs and TP53, where TP53 activates TEs with TP53 binding motifs through direct binding and represses TEs indirectly via other pathways. Overall, we present a comprehensive profiling of the impact of TP53 on the activity of TE-derived promoters in isogenic cancer cell lines and provide a high-confidence TE expression atlas of TE promoters that are direct and indirect targets of TP53.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f53c846a8196b5df2291595170d1ce035cd5084b" target='_blank'>
              Charting the regulatory landscape of TP53 on transposable elements in cancer
              </a>
            </td>
          <td>
            Xuan Qu, Yonghao Liang, Colin McCornack, Xiaoyun Xing, Heather Schmidt, Chad Tomlinson, C. Fronick, E. Belter, Juan F. Macias-Velasco, Ting Wang
          </td>
          <td>2025-05-13</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="The tumor suppressor gene NKX3.1 and the LPL gene are located in close proximity on chromosome 8, and their deletion has been reported in multiple studies. However, the significance of LPL loss may be misinterpreted due to its co-deletion with NKX3.1, a well-established event in prostate carcinogenesis. This study investigates whether LPL deletion represents a biologically relevant event or occurs merely as a bystander to NKX3.1 loss. We analyzed 28 formalin-fixed paraffin-embedded prostate cancer samples with confirmed LPL deletion and 28 without. Immunohistochemical staining was performed, and previously published whole-genome sequencing data from 103 prostate cancer patients were reanalyzed. Deletion of the 8p21.3 region was associated with higher Gleason grade groups. While NKX3.1 expression was significantly reduced in prostate cancer compared to benign prostatic hyperplasia, LPL protein expression showed no significant difference between cancerous and benign tissue, nor was it affected by the 8p21.3 deletion status. Copy number analysis confirmed the co-deletion of NKX3.1 and LPL in 54 patients. Notably, NKX3.1 loss without accompanying LPL deletion was observed in eight additional cases. These findings suggest that LPL deletion is a passenger event secondary to NKX3.1 loss and underscore the importance of cautious interpretation of cytogenetic findings involving the LPL locus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74796121b67d50bf9eb77e02037db96e6cc25807" target='_blank'>
              Deletions of LPL and NKX3.1 in Prostate Cancer Progression: Game Changers or By-Standers in Tumor Evolution
              </a>
            </td>
          <td>
            Tereza Vodičková, Mária Wozniaková, Vladimír Židlík, Jana Žmolíková, Jana Dvořáčková, Adéla Kondé, Jana Schwarzerová, Michal Grepl, J. Bouchal
          </td>
          <td>2025-05-24</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a013cf13f270b70a82c0a8355212488d6a1c6ef" target='_blank'>
              Microhomology-Mediated End Joining as a Novel Mechanism Underlying Androgen Insensitivity Syndrome
              </a>
            </td>
          <td>
            J. M. Marques, R. M. Ramos, N. D. R. D' Alessandre, G. D. A. Guardia, A. F. Afonso, B. L. Braga, M. F. D. A. Funari, M. Y. Nishi, P. Asprino, S. Domenice, P. A. F. Galante, B. B. Mendonca, R. L. Batista
          </td>
          <td>2025-06-26</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is a deadly disease known for its genetic heterogeneity. LTR12C is an endogenous retrovirus-derived regulator of pro-apoptotic genes and is normally silenced by epigenetic regulation. In this study, we found that the treatment of two glioblastoma cell lines, T98-G and U87-MG, with DNA methyltransferase (DNMT) and histone deacetylase (HDAC) inhibitors activated LTR12C expression. Combined treatment with these epigenetic drugs exerted a synergistic action on the LTR12C activation in both cell lines, while treatment with each drug as a single agent had a far weaker effect. A strong induction of the expression of the TP63 gene was seen in both cell lines, with the pro-apoptotic isoform GTA-p63 accounting for most of this increase. Coherently, downstream targets of p63, such as p21 and PUMA, were also induced by the combined treatment. Furthermore, we observed a significant reduction in the GBM cell growth and viability following the dual DNMT/HDAC inhibition. These findings reveal that the reactivation of LTR12C expression has the potential to modulate survival pathways in glioblastoma and provide information regarding possible epigenetic mechanisms that can be used to treat this deadly disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b329500c4414f15e1eee2caec6a0bfaed4d1e7f7" target='_blank'>
              Functional Characterization of LTR12C as Regulators of Germ-Cell-Associated TA-p63 in U87-MG and T98-G In Vitro Models
              </a>
            </td>
          <td>
            Lucia Meola, Sohum Rajesh Shetty, A. Peschiaroli, Claudio Sette, Camilla Bernardini
          </td>
          <td>2025-06-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Genomic instability is a hallmark of cancer, enabling the generation of mutations and gross chromosomal rearrangements to drive neoplastic cell transformation and oncogenesis. The BRCA1-BARD1 protein complex acts to eliminate highly toxic DNA double-strand breaks, to ensure the faithful propagation of our genetic blueprint and to suppress cancer development. BRCA1 is a well-described tumor suppressor protein associated with hereditary breast and ovarian cancers as well as sporadic breast cancers, with loss or mutation of BRCA1 leading to triple negative breast cancer and poor patient prognosis. The BRCA1-BARD1 complex promotes homologous recombination (HR), which is the major pathway for the accurate repair of double-strand breaks. However, there is little information regarding the intricate roles fulfilled by BRCA1-BARD1 in this process, or how loss of specific BRCA1-BARD1 functions leads to tumorigenesis. BRCA1 has been previously reported to physically interact with both DNA and RAD51, key factors in HR, but the contributions of the interaction attributes to DNA damage repair remain unknown. Here, we delineate major sites of DNA and RAD51 binding in BRCA1 and use a combination of biochemical and NMR methods to identify the specific residues mediating interactions with these ligands. This has allowed us to develop mutations to ablate BRCA1’s ability to interact with these substrates without affecting BRCA1-BARD1’s interaction with other key DNA repair substrates. Using these BRCA1 mutants impaired for either DNA or RAD51 binding, we have interrogated the contributions of these interaction attributes to BRCA1-BARD1’s function by comparing the activity of WT to mutant BRCA1-BARD1 in biochemical assays to reconstitute various steps of HR. We have found that both DNA and RAD51 binding are indispensable for BRCA1-BARD1’s ability to promote RAD51-mediated D-loop formation, thus helping to delineate the mechanism by which BRCA1 promotes HR. Our studies provide the foundation to determine the functional consequences of cancer mutations in BRCA1-BARD1 and for the development of therapeutic strategies to target HR-deficient tumors.
 Citation Format: Angela Jasper, Hoang Dinh, Cody M. Rogers, Sameer Salunkhe, Hardeep Kaur, Antoine Baudin, David S. Libich, Patrick Sung. The Mechanistic Role of BRCA1 DNA and RAD51 Binding in DNA Double-Strand Break Repair [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19be3dd24b8cbf64d48385ebe0e058a4d636d0fd" target='_blank'>
              Abstract P5-06-06: The Mechanistic Role of BRCA1 DNA and RAD51 Binding in DNA Double-Strand Break Repair
              </a>
            </td>
          <td>
            Angela M Jasper, Hoang Dinh, Cody M Rogers, Sameer Salunkhe, Hardeep Kaur, Antoine Baudin, David S. Libich, Patrick Sung
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Triggering cancer cell death by inducing DNA damage is the primary aim of radiation therapy; however, normal cells are also damaged. Here, we showed that delivery of only four synthetic guide RNAs (sgRNAs) with Cas9 endonuclease efficiently induced simultaneous DNA double-strand breaks, resulting in efficient cell death in a cell type-specific manner. Off-target effects of Cas9 endonuclease were prevented by using Cas9-nickase to induce DNA single-strand breaks and blocking their repair with poly ADP-ribose polymerase (PARP) inhibitors. When recombinant Cas9-nickase protein and multiple sgRNAs were delivered with PARP inhibitors into cultured cells, in vivo xenografts, and patient-derived cancer organoids via lipid nanoparticles, cancer cells were unable to tolerate the induced DNA damage, even in the presence of a functional BRCA2 gene. This approach has the potential to expand the use of PARP inhibitors with verified safety and thus is a potentially powerful tool for personalized genome-based anti-cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fde0090503581f24fa582fe8e81b3d566923d8b" target='_blank'>
              Combining Multiplexed CRISPR/Cas9-Nickase and PARP Inhibitors Efficiently and Precisely Targets Cancer Cells.
              </a>
            </td>
          <td>
            Soyoung Lee, Kyunghwan Kim, Hye-Jin Jeong, Subin Choi, Himchan Cheng, Dayoung Kim, Soomin Heo, Jinhee Mun, Minjong Kim, Eunjin Lee, Yoon Ji Choi, Seon-Gyeong Lee, Eunjin Lee, Yewon Jang, Kayeong Lim, Heon Seok Kim, Euihwan Jeong, Seung-Jae Myung, Deok-Beom Jung, Chang Sik Yu, In Ho Song, M. Corces, Joo H. Kang, Kyungjae Myung, Taejoon Kwon, Tae-Eun Park, Jinmyoung Joo, Seung Woo Cho
          </td>
          <td>2025-05-06</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Human exonuclease 1 (EXO1), a member of the structure-specific nuclease family, plays a critical role in maintaining genome stability by processing DNA double-strand breaks (DSBs), nicks, and replication intermediates during DNA replication and repair. As its exonuclease activity is essential for homologous recombination (HR) and replication fork processing, EXO1 has emerged as a compelling therapeutic target, especially in cancers marked by heightened DNA damage and replication stress. Through high-throughput screening of 45,000 compounds, we identified seven distinct chemical scaffolds that demonstrated effective and selective inhibition of EXO1. Representative compounds from two of the most potent scaffolds, C200 and F684, underwent a comprehensive docking analysis and subsequent site-directed mutagenesis studies to evaluate their binding mechanisms. Biochemical assays further validated their potent and selective inhibition of the EXO1 nuclease activity. Tumor cell profiling experiments revealed that these inhibitors exploit synthetic lethality in BRCA1-deficient cells, emphasizing their specificity and therapeutic potential for targeting genetically HR-deficient (HRD) cancers driven by deleterious mutations in HR genes like BRCA1/2. Mechanistically, EXO1 inhibition suppressed DNA end resection, stimulated the accumulation of DNA double-strand breaks, and triggered S-phase PARylation, effectively disrupting DNA repair pathways that are essential for cancer cell survival. These findings establish EXO1 inhibitors as promising candidates for the treatment of HRD cancers and lay the groundwork for the further optimization and development of these compounds as targeted therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53bbdf0d10f263cc553f41d2d920cc6308756913" target='_blank'>
              Discovery and Characterization of Small Molecule Inhibitors Targeting Exonuclease 1 for Homologous Recombination-Deficient Cancer Therapy
              </a>
            </td>
          <td>
            Yixing Wang, Jessica D Hess, Chen Wang, Lingzi Ma, Megan Luo, J. Jossart, John J. Perry, David Kwon, Zhe Wang, Xinyu Pei, Changxian Shen, Yingying Wang, Mian Zhou, Holly Yin, David Horne, A. Nussenzweig, Li Zheng, Binghui Shen
          </td>
          <td>2025-05-16</td>
          <td>ACS Chemical Biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background PERP may have the potential to function as an oncogene. However, the precise function, prognostic value, and predictive significance remain shrouded in ambiguity. Methods We conducted an in-depth analysis using pan-cancer RNA sequencing data and various online web tools to investigate the correlation between PERP and crucial clinical outcomes such as prognosis, tumor microenvironment, and tumor metabolism. In addition, we explored the tumor-promoting role of PERP and its potential mechanisms through models such as immunofluorescence staining, flow cytometry, cell proliferation assays, wound healing assays, cell migration assays, mass spectrometry analysis and isotope tracing. Further in vivo models confirmed the functional consistency of PERP across pan-cancer. Finally, we analyzed the potential of PERP as a predictive factor for immunotherapy sensitivity in a clinical cohort. Results PERP exhibits elevated expression in the majority of cancer types and impedes immune cell infiltration as well as immune checkpoint reactivity in pan-cancer. We confirmed that PERP can promote tumor progression by tumor cell proliferation, scratch and transwell experiments. Meanwhile, the absence of PERP restricts the flux of 13C6-glucose into glycolysis and the tricarboxylic acid (TCA) cycle. Importantly, the deficiency of PERP enhances the in vivo anti-tumor efficacy of PD1 monoclonal antibodies. In addition, low PERP expression is highly correlated with the response of head and neck squamous cell carcinoma (HNSCC) patients to immunotherapy. Conclusions PERP represents a promising predictive/diagnostic biomarker and therapeutic target for HNSCC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02360-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/165cbe4ddcc5a029683b55fb68914aec7383f0fa" target='_blank'>
              Targeting PERP promotes anti-tumor immunity in HNSCC by regulating tumor immune microenvironment and metabolic homeostasis
              </a>
            </td>
          <td>
            Xueying Wang, Yuxi Tian, Xiaohong Wu, Yewen Zhu, Huihong Chen, Zeyao Wang, Zihan Liu, Jiaqi Tan, Zhaoyu Pan, Jiaoyan Cao, Zhenjiang Li, Xin Zhang, Zhongjie Shi, Junchen Wang, Tong Liu
          </td>
          <td>2025-06-07</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52f4e542b3c6d086c581d428766c82b0185e38fd" target='_blank'>
              Intratumoral heterogeneity in microsatellite instability status at single cell resolution
              </a>
            </td>
          <td>
            Harrison Anthony, C. Seoighe
          </td>
          <td>2025-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Pediatric brain tumors, the most devastating cancers affecting children, are believed to originate from neural stem/progenitor cells in developing brain. In precise timing and specific regions during the brain development, chromatin deregulation plays crucial roles in redirecting normal neuronal differentiation pathways toward tumorigenesis. Indeed, epigenomic abnormalities are thought to be more important for brain tumor formation especially in children than adults, as pediatric brain tumors generally exhibit fewer genetic mutations compared to adult brain tumors. Given the small number of mutations, targeting such limited alterations involved in cancer epigenomes is expected to be more effective in pediatric brain tumors. The mechanisms of cancer epigenomes include mutation or dysregulation of chromatin remodelers, histone modifiers, histones themselves, and DNA methylation enzymes. Furthermore, genomic rearrangements and/or higher-order chromatin topology also contribute to these epigenomic mechanisms. These mechanisms are commonly observed in various types of pediatric brain tumors. However, alterations in chromatin regulatory factors differ across tumor types, reflecting the unique epigenetic landscapes shaped by their tumor origins. Accordingly, clarifying their functional similarities and differences across tumor types could offer valuable insights for finding new therapeutic strategies. Thus, this review article focuses on elucidating how pediatric brain tumors arise from epigenomic deregulation and what epigenetic molecules or mechanisms could serve as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a334571a5c8d012851148672d620a490ae61353" target='_blank'>
              Epigenetic modifications and their roles in pediatric brain tumor formation: emerging insights from chromatin dysregulation
              </a>
            </td>
          <td>
            Kento Kawata, Owen S. Chapman, Satoshi Narumi, Daisuke Kawauchi
          </td>
          <td>2025-06-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="HSP110 is a ubiquitous chaperone contributing to proteostasis. It has a disaggregation activity and can refold denatured proteins. It can regulate fundamental signaling pathways involved in oncogenesis, such as Wnt/β-catenin, NF-κB and STAT3 signaling pathways. In gastric and colorectal cancer, HSP110 has been detected in the nucleus, and nuclear expression has been associated with the resistance of cells to 5-FU chemotherapy. Nuclear translocation of HSP110 is promoted by the exposure of cells to DNA-damaging agents. In a previous work, we demonstrated that nuclear HSP110 participates in the NHEJ DNA repair pathway by facilitating the recruitment of DNA-PKcs to Ku70/80 heterodimers at the site of DNA double-strand breaks. In the present work, analysis of HSP110s’ nuclear interactome revealed an enrichment of components from SWI/SNF chromatin remodeling complexes. We demonstrate that HSP110 is strongly associated with chromatin in temozolomide- and oxaliplatin-treated cells and directly interacts with the core subunit SMARCC2, thereby facilitating the assembly of SWI/SNF complexes. This work expands upon the role of HSP110, which regulates not only proteostasis but also the assembly of critical nuclear macromolecular complexes involved in the adaptive stress response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c88c3e640399a73a02e0ec0eeda0b57f77e5458" target='_blank'>
              HSP110 Regulates the Assembly of the SWI/SNF Complex
              </a>
            </td>
          <td>
            Océane Pointeau, Manon Paccagnini, Natalia Borges-Bonan, L. Biziorek, Sebastien Z. Causse, Carmen Garrido, L. Dubrez
          </td>
          <td>2025-06-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d46f5896df3074e9079fd05c2ab487e0d0fe604" target='_blank'>
              Context-Dependent Regulation of Microhomology-Mediated End Joining in Normal Tissues: Insights into Tissue-Specific Activation of DNA Repair Pathways
              </a>
            </td>
          <td>
            Diksha Rathore
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most common malignancies worldwide, and its complex pathogenesis and significant tumor cell heterogeneity remain major challenges. With the rapid development of single-cell sequencing technology, we can now delve deeper into the cellular composition and dynamic changes within the tumor microenvironment, revealing cellular interactions and their potential roles in tumorigenesis. Method In this study, we systematically analyzed comprehensive single-cell RNA sequencing data from 25 colorectal cancer and 10 adjacent normal tissue samples. We explored the characteristics and biological significance of tumor cell subpopulations, performed quality control, dimensionality reduction, and cell type identification, and further investigated epithelial cell copy number variations, cell communication, and pseudotime analysis. Subsequently, Boruta feature selection algorithm was combined to identify prognosis related genes. The expression patterns, clinical significance and biological effects of PRSS22 were validated in vitro. Results Our analysis found an epithelial cell subcluster with high expression of PRSS22 exhibited high proliferation and migration abilities, and it was also associated with the dysregulated immune microenvironment. After further experimental verification, we proved the high expression patterns and clinical significance of PRSS22. Downregulation of PRSS22 in CRC cells resulted in a reduction of proliferation, migration and invasion. Conclusion Our study has identified a cell subcluster that is closely linked to progression, immune dysregulation and prognosis in CRC, and we have also identified PRSS22 as its hub gene that has great potential to become a new immunotherapeutic targets target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0d396c713a83490019a7c5d54cf76e345198f9" target='_blank'>
              Single-cell transcriptomics in colorectal cancer uncover the potential of metastasis and immune dysregulation of a cell cluster overexpressed PRSS22
              </a>
            </td>
          <td>
            Chengyuan Xu, Ziheng Zhou, Dongfei Zhu, Qingyun Zhang, Shoubin Zhong, Zhenhua Li
          </td>
          <td>2025-05-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d1093733a8ade229255066e064b631c0ae1860e" target='_blank'>
              ImmunoFusion: A Unified Platform for Investigating RNA-seq-Derived Gene Fusions in Cancer and Immunotherapy
              </a>
            </td>
          <td>
            YanKun Zhao, Shixiang Wang, Shensuo Li, Minjun Chen, Su-Han Jin, U. Gaipl, Hu Ma, Jian-Guo Zhou
          </td>
          <td>2025-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1683b65b8227a142ec0662e619dfd830c76c770" target='_blank'>
              Major effect mutations drive DNA methylation variation after colonization of a novel habitat
              </a>
            </td>
          <td>
            Johan Zicola, Emmanuel Tergemina, Mehmet Göktay, Célia Neto, Robert J. Schmitz, Angela M. Hancock
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Medulloblastoma comprises four principal molecular disease groups and their component subgroups, each with distinct molecular and clinical features. Group assignment is currently achieved diagnostically using Illumina DNA methylation microarray. Whole-genome sequencing (WGS) capacity is rapidly expanding in the clinical setting and the development of platform-independent, sequence-based assays of molecular group offers significant potential. Specifically, whole-genome bisulfite sequencing (WGBS) enables assessment of genome-wide methylation status at single-base resolution, however its routine application has been limited by high DNA input requirements, cost, and a lack of pipelines tailored to more rapidly-acquired and cost-effective low-depth (< 10x) sequencing data. We utilised WGBS data for 69 medulloblastomas, comprising 35 in-house low-depth (~ 10x) and 34 publicly available high-depth (~ 30x) samples, alongside cerebellar controls (n = 8), all with matched DNA methylation microarray data. We assessed quality (QC) and imputation approaches using low-pass WGBS data, assessed inter-platform correlation and identified molecular groups and subgroups by directly integrating matched/associated loci from WGBS sample data with the MNP classifier probeset. We further assessed and optimised reference-free aneuploidy detection using low-pass WGBS and assessed concordance with microarray-derived calls. We developed and optimised pipelines for processing, QC, and analysis of low-pass WGBS data, suitable for routine molecular subgrouping and reference-free aneuploidy assessment. We demonstrate that low-pass WGBS data can (i) be integrated into existing array-trained models with high assignment probabilities for both principal molecular groups (97% concordance) and molecular subgroups (94.2% concordance), and (ii) detect clinically relevant focal copy number changes, including SNCAIP, with greater sensitivity than microarray approaches. Low-pass WGBS performs equivalently to array-based methods at comparable cost. Finally, its ascertainment of the full methylome enables elucidation of additional biological complexity and inter-tumoural heterogeneity that has hitherto been inaccessible. These findings provide proof-of-concept for clinical adoption of low-pass WGBS, applied using standard WGS technology. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02049-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbe3d3a3f742b6178900eb1bb44957ac2eaf7cd1" target='_blank'>
              Robust molecular subgrouping and reference-free aneuploidy detection in medulloblastoma using low-depth whole genome bisulfite sequencing
              </a>
            </td>
          <td>
            Dean Thompson, Jemma Castle, M. Sill, S. Pfister, Simon Bailey, Debbie Hicks, Steven C. Clifford, E. Schwalbe
          </td>
          <td>2025-06-24</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c71b63f87cb576d755f4a88747d8d75f40eef18" target='_blank'>
              Multimodal profiling unveils a reversible basal-like breast cancer cell state resistant to AKT inhibition
              </a>
            </td>
          <td>
            Colin D. H. Ratcliffe, Hugh Sparks, G. L. Boezio, Yuriy Alexandrov, Robert P. Jenkins, A. Le Marois, Pablo Soro-Barrio, Rebecca Lee, S. Strohbuecker, Sudeep Joshi, James K. Ellis, Anne-Marie Fortier, Nils Gustafsson, Ana C. Cunha, Morag Park, J. Macrae, A. Serio, J. Briscoe, Andy Riddell, S. Barry, Chris Dunsby, E. Sahai
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genomic instability has been increasingly recognized over the past decade as a fundamental driver of cancer initiation and progression, largely owing to its association with specific genes and cellular mechanisms that offer therapeutic potential. However, a comprehensive molecular framework that captures the interconnected processes underlying this phenomenon remains elusive. In this study, we focused on polo-like kinase 1 (PLK1), a key cell cycle regulator frequently overexpressed in diverse human tumors, to reconstruct a regulatory network that consolidates pre-existing biological knowledge exclusively related to pathways involved in genome stability maintenance and cancer. The resulting model integrates nine biological processes, 1030 reactions, and 716 molecular species to form a literature-supported network in which PLK1 serves as a central regulatory node. However, rather than depicting an isolated PLK1-centric system, this network reflects a broader and more complex architecture of interrelated genomic instability mechanisms. As expected, the simulations reproduced known behaviors associated with PLK1 dysregulation, reinforcing the well-established role of the kinase in genome destabilization. Importantly, this model also enables the exploration of additional, less-characterized dynamics, including the potential involvement of genes such as kif2c, incenp, and other regulators of chromosomal segregation and DNA repair, which appear to contribute to instability events downstream of PLK1. While these findings are grounded in mechanistic simulations and require further experimental validation, gene expression and survival analyses across tumor types support their clinical relevance by linking them to poor prognosis in specific cancers. Overall, the model provides a systemic and adaptable foundation for studying PLK1-related genomic instability, enabling both the reinforcement of known mechanisms and discovery of candidate genes and circuits that may drive tumorigenesis through compromised genome integrity across distinct cancer contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c66a6d480aea1f4304a1ce04eec8ec37b7bd0e2a" target='_blank'>
              The plk1 Gene Regulatory Network Modeling Identifies Three Circuits for plk1-mediated Genomic Instability Leading to Neoplastic Transformation
              </a>
            </td>
          <td>
            Jeison F. Suescum-Holguín, Diana Carolina Clavijo-Buriticá, Edward Fabian Carrillo-Borda, M. Quimbaya
          </td>
          <td>2025-05-01</td>
          <td>Life</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34ad39aac8ea35ca3a29505be2a3f10fc20c5850" target='_blank'>
              TP53 and DNA-PK as potential biomarkers for enhanced efficacy of Olaparib in colorectal cancer.
              </a>
            </td>
          <td>
            Do Yeon Kim, Hyeseon Yun, Ji-Eun You, Y. Park, Yeaseong Ryu, Dong-In Koh, Jae-Sik Shin, Dong-Hoon Jin
          </td>
          <td>2025-05-16</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="RNA N6-methyladenosine (m6A) modification plays a crucial role in the DNA damage response, while the detailed mechanisms remain to be explored. In this study, we report the involvement of the m6A demethylase ALKBH5 in X-ray-induced DNA damage response. Depletion of ALKBH5 reduces X-ray-induced DNA damage, induces G2/M phase arrest and reduces cell apoptosis. RNA sequencing and m6A sequencing analysis reveal that ALKBH5 removes m6A modifications from its target mRNAs and suppresses their expression. A subset of mRNAs encoding cyclin dependent kinase inhibitors, such as CDKN1A and CDKN2B, show increased stability and expression upon ALKBH5 knockdown. Subsequently, the upregulation of CDKN1A and CDKN2B contributes to G2/M phase arrest to facilitate DNA repair. Our findings unveil the epigenetic regulation of cell cycle checkpoint by ALKBH5 in X-ray-induced DNA damage, offering potential targets for DNA damage-based therapy for cancers. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-01207-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81787fc89efd9197a1f437ceb995089be421e6e3" target='_blank'>
              RNA demethylase ALKBH5 regulates cell cycle progression in DNA damage response
              </a>
            </td>
          <td>
            Bo Gao, Haitao Pan, Xiaoling Zhou, Lei Yu, Yunyi Gao, Tao Zhang, Xiangwei Gao, Jingyu Hou
          </td>
          <td>2025-05-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="XPO1 is a nuclear export receptor that is essential for cell survival. Previous genomic analyses have identified recurrent XPO1 hotspot mutations in cancer. Here, we conducted a large-scale genomic analysis of 217,570 cancer patients to identify and characterize XPO1 variants from real-world patient tumors. XPO1 harbored a R749Q mutation in various solid tumors, with a clear enrichment in endometrial and colorectal cancers, and XPO1R749Q mutations significantly co-occurred with POLE mutations. Analysis of isogenic colon cancer cell lines revealed that XPO1R749Q localized more in the cytoplasm than wildtype XPO1, with enhanced export of a large number of proteins. Structural modeling of XPO1R749Q suggested an increase in RanGTP affinity, which is consistent with enhanced protein export. A compound library screen using over 200 FDA-approved anticancer drugs indicated a general trend towards chemoresistance, specifically to topoisomerase I inhibition, in XPO1R749Q mutant cells. Mechanistically, XPO1R749Q mutant cells exhibited enhanced DNA damage response via RPA phosphorylation in response to topoisomerase I inhibition. Combining XPO1 and topoisomerase I inhibitors reduced DNA damage-induced RPA phosphorylation and mediated synergistic antitumor effects in cells harboring the XPO1R749Q mutation. Finally, the combination of selinexor and irinotecan overcame chemotherapeutic resistance in xenograft mouse models, prolonging survival. These findings suggest that XPO1 alterations in cancer are selected for in POLE mutant tumors and may confer resistance to DNA-damaging chemotherapies, which have implications for patients with tumors bearing XPO1R749Q and for XPO1 inhibitor development in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e52bf0adf008e92bc55a03de218e8fe730ea2f" target='_blank'>
              XPO1R749Q Mutations Co-occur with POLE Mutations in Cancer and can be Targeted to Overcome Chemoresistance.
              </a>
            </td>
          <td>
            T. Totiger, Wannasiri Chiraphapphaiboon, Y. Baca, S. Chaudhry, R. Notti, S. Montoya, M. Chojnacka, Gabriel Gaidosh, Jumana Afaghani, Maurizio Affer, Chris Armstrong, Paul M Kavanaugh, Efe Karaca, Jenna Zabroski, Michael Lewis, Alyssa Maye, Jacob Jahn, R. Soni, Daniel Bilbao, Phil Walker, Andrew Elliott, E. Lou, Wafik S. El-Deiry, Josean Rodriguez, Hai Dang Nguyen, Justin Taylor
          </td>
          <td>2025-06-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Oncogene activation through DNA amplification or overexpression plays a critical role in cancer initiation and progression. Chromosome 17 harbors numerous cancer-associated genetic anomalies, some of which are strongly linked to poor prognosis in breast cancer patients. The FOXK2 gene, located on 17q25, encodes a transcription factor with a forkhead DNA binding domain. Integrative analyses of publicly available genomic datasets reveal that FOXK2 is frequently amplified and overexpressed in breast cancer, correlating with poor overall survival. FOXK2 knockdown via shRNA significantly inhibited the proliferation, migration, and anchorage-independent growth of breast cancer cells, causing G0/G1 cell cycle arrest. The C-terminal nuclear localization signal (NLS) is crucial for FOXK2's oncological function. Additionally, FOXK2 knockdown in MCF7 breast cancer cells delayed tumor growth in a xenograft mouse model. Inhibiting FOXK2 expression also sensitized breast cancer cells to frontline chemotherapy agents, including doxorubicin, 5-fluorouracil, and etoposide in vitro. Co-overexpression of FOXK2 and PI3KCA with oncogenic mutations (E545K or H1047R) induced the transformation of non-tumorigenic MCF-10A cells, suggesting that FOXK2 acts as an oncogene in breast cancer and contributes to PI3KCA-driven tumorigenesis. CCNE2, PDK1, and ESR1 were identified as direct transcriptional targets of FOXK2 in MCF-7 cells. Moreover, co-inhibition of FOXK2 function, either through knockdown or inhibition of its transcriptional targets (CCNE2/CDK2 by Dinaciclib or PDK1 by DCA) and PI3KCA by alpelisib, had synergistic anti-tumor effects in breast cancer cells with FOXK2 overexpression and PI3KCA oncogenic mutations. FOXK2 functional inhibition by the CDK2 inhibitor Dinaciclib enhanced the tumor-suppressive effect of alpelisib in a MCF7 breast cancer xenograft mouse model. These findings provide compelling evidence that FOXK2 plays an oncogenic role in breast tumorigenesis. The combination of PIK3CA and CCNE2/CDK2 inhibition by alpelisib and Dinaciclib offers a potential therapeutic strategy for breast cancer with FOXK2 overexpression and PIK3CA oncogenic mutation.
 Citation Format: Hong Zhang, Yang Yu, Weng-Ming Cao, Feng Cheng, Edaise da silva, Higinio Dopeso, Hui Chen, Xiaosong Wang, Chunchao Zhang. Targeting FOXK2 signaling in breast cancers with FOXK2 amplification/overexpression and PIK3CA oncogenic mutations: a promising therapeutic strategy? [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-02-24.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caaa74a368ec1f757b26670f47abf7954355f919" target='_blank'>
              Abstract P5-02-24: Targeting FOXK2 signaling in breast cancers with FOXK2 amplification/overexpression and PIK3CA oncogenic mutations: a promising therapeutic strategy?
              </a>
            </td>
          <td>
            Hong Zhang, Yang Yu, Wen-Ming Cao, Feng Cheng, E. D. da Silva, Higinio Dopeso, Hui Chen, Xiaosong Wang, Chunchao Zhang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8503e81194e8e95ebd9edbe7edc4c0fd9d960853" target='_blank'>
              Mechanisms of resistance to VHL loss-induced genetic and pharmacological vulnerabilities
              </a>
            </td>
          <td>
            Jianfeng Ge, Shoko Hirosue, Saroor A. A. Patel, L. Wesolowski, Anna Dyas, Cissy Yong, Leticia Castillon, Sanne de Haan, J. Drost, Grant D. Stewart, Anna Obenauf, D. Muñoz-Espín, Sakari Vanharanta
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f86b4088fda0a17a1413678d49be792807c01757" target='_blank'>
              Human TLS DNA polymerase: saviors or threats under replication stress?
              </a>
            </td>
          <td>
            Y. S. Rajpurohit, Mitu Lal, D. K. Sharma, Ishu Soni
          </td>
          <td>2025-05-23</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/892ce08d08aef5a794575f58122e8dce3a63ce74" target='_blank'>
              Aneuploidy alleviates the cell proliferation defect caused by mutations affecting origin licensing in Saccharomyces cerevisiae
              </a>
            </td>
          <td>
            Christophe de La Roche Saint-André
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer drug resistance is multi-factorial, driven by heritable (epi)genetic changes but also phenotypic plasticity. Here, we dissected drivers of resistance by perturbing colorectal cancer patient-derived organoids longitudinally with drugs in sequence. Combined longitudinal lineage tracking, single cell multi-omics analysis, evolutionary modelling, and machine learning revealed that different targeted drugs select for distinct subclones, supporting rationally designed drug sequences. The cellular memory of drug resistance was encoded as a heritable epigenetic configuration from which multiple transcriptional programs could run, supporting a one-to-many (epi)genotype-to-phenotype map that explains how clonal expansions and plasticity manifest together. This epigenetic landscape may ensure drug resistant subclones can exhibit distinct phenotypes in changing environments while still preserving the cellular memory encoding for their selective advantage. Chemotherapy resistance was instead entirely driven by transient phenotypic plasticity rather than stable clonal selection. Inducing further chromosomal instability before drug application changed clonal evolution but not convergent transcriptional programs. Collectively, these data show how genetic and epigenetic alterations are selected to engender a "permissive epigenome" that enables phenotypic plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab83269ee41c4604edc0996e3b22f4bf8776d7c6" target='_blank'>
              Epigenetic Heritability of Cell Plasticity Drives Cancer Drug Resistance through a One-to-Many Genotype-to-Phenotype Paradigm.
              </a>
            </td>
          <td>
            É. Oliveira, Salvatore Milite, J. Fernández-Mateos, George D. Cresswell, Erika Yara-Romero, G. Vlachogiannis, Bingjie Chen, Chela James, Lucrezia Patruno, Gianluca Ascolani, Ahmet Acar, Timon Heide, Inma Spiteri, Alex Graudenzi, G. Caravagna, Andrea Bertotti, Trevor A. Graham, Luca Magnani, Nicola Valeri, A. Sottoriva
          </td>
          <td>2025-06-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="BACKGROUND Hepatocellular carcinoma (HCC) ranks among the most prevalent and deadly malignancies, characterized by a high recurrence rate. Regulator of chromosome condensation 1 (RCC1) serves as a principal guanine nucleotide exchange factor for ras-related nuclear protein guanosine triphosphatase (GTPase) and is implicated in various cancers. However, the role of RCC1 in HCC remains unexplored. AIM To elucidate the functional significance and molecular mechanisms of RCC1 in HCC. METHODS Bioinformatics were to examine the expression levels of RCC1 in HCC and to assess its impact on the prognosis of this malignancy. The cell counting kit-8 assay and flow cytometry were utilized to evaluate the cell viability and cell cycle of HCC cells. Furthermore, quantitative reverse transcription and immunoblotting were to investigate the influence of RCC1 on cyclin associated proteins. RESULTS Bioinformatics analysis revealed that RCC1 was highly expressed in HCC and correlated with poor prognosis in HCC patients. Functional studies showed that RCC1 overexpression promoted the malignant phenotype of HCC cells, especially the proliferation of HCC cells, whereas RCC1 knockdown had the opposite effect. Mechanistically, we identiﬁed cell division cycle-associated (CDCA) 8 as a downstream target of RCC1 in HCC. RCC1 overexpression markedly increased CDCA8 levels, consequently enhancing cell proliferation and survival in HCC cells. Additionally, we discovered that RCC1 contributed to the development and progression of HCC by activating the phosphoinositide 3-kinase/protein kinase B/cyclin-dependent kinase inhibitor 1a pathway through CDCA8. CONCLUSION Our study provides profound insights into the pivotal role of RCC1 in HCC and its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6654467e3e683d7f28902cff2c98d0e0d1baee2f" target='_blank'>
              Regulator of chromosome condensation 1 promotes hepatocellular carcinoma proliferation via cell-division-cycle-associated-8 dependent phosphoinositide 3-kinase/protein kinase B signaling
              </a>
            </td>
          <td>
            Ya-Tao Wang, Yu‐Le Yong, Ze-Kun Liu, Yi-Xuan Shen, Xiang-min Yang, Zhi-Nan Chen
          </td>
          <td>2025-06-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Organotropism results from the functional versatility of metastatic cancer cells to survive and proliferate in diverse microenvironments. This adaptivity can originate in clonal variation of the spreading tumor and is often empowered by epigenetic and molecular reprogramming of cell regulatory circuits. Related to organotropic colonization of metastatic sites are environmentally-sensitive, differential responses of cancer cells to therapeutic attack. Accordingly, understanding the organotropic profile of a cancer and probing the underlying driver mechanisms are of high clinical importance. However, determining systematically the organotropism of one cancer versus the organotropism of another cancer, potentially with the granularity of comparing the same cancer type between patients or tracking the evolution of a cancer in a single patient for the purpose of personalized treatment, has remained very challenging. It requires a host organism that allows observation of the spreading pattern over relatively short experimental times. Moreover, organotropic patterns often tend to be statistically weak and superimposed by experimental variation. Thus, an assay for organotropism must give access to statistical powers that can separate ‘meaningful heterogeneity’, i.e., heterogeneity that determines organotropism, from ‘meaningless heterogeneity’, i.e., heterogeneity that causes experimental noise. Here we describe an experimental workflow that leverages the physiological properties of zebrafish larvae for an imaging-based assessment of organotropic patterns over a time-frame of 3 days. The workflow incorporates computer vision pipelines to automatically integrate the stochastic spreading behavior of a particular cancer xenograft in tens to hundreds of larvae allowing subtle trends in the colonization of particular organs to emerge above random cell depositions throughout the host organism. We validate our approach with positive control experiments comparing the spreading patterns of a metastatic sarcoma against non-transformed fibroblasts and the spreading patterns of two melanoma cell lines with previously established differences in metastatic propensity. We then show that integration of the spreading pattern of xenografts in 40 – 50 larvae is necessary and sufficient to generate a Fish Metastatic Atlas page that is representative of the organotropism of a particular oncogenotype and experimental condition. Finally, we apply the power of this assay to determine the function of the EWSR1::FLI1 fusion oncogene and its transcriptional target SOX6 as plasticity factors that enhance the adaptive capacity of metastatic Ewing sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5025875828f543583bc1d12169d95499fa95592" target='_blank'>
              Robust statistical assessment of Oncogenotype to Organotropism translation in xenografted zebrafish
              </a>
            </td>
          <td>
            D. Saucier, Xuexia Jiang, Divya Rajendran, Roshan Ravishankar, Erin Butler, A. Marchetto, R. Kurmasheva, T. G. Grünewald, James F. Amatruda, G. Danuser
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Abstract Across the tree of life, DNA damage response (DDR) proteins play a pivotal, yet dichotomous role in organismal development and evolution. Here, we present a comprehensive analysis of 432 DDR proteins encoded by 68 genomes, including that of Nucleospora cyclopteri, an intranuclear microsporidia sequenced in this study. We compared the DDR proteins encoded by these genomes to those of humans to uncover the DNA repair-ome across phylogenetically distant eukaryotes. We also performed further analyses to understand if organismal complexity and lifestyle play a role in the evolution of DDR protein length and conserved domain architecture. We observed that the genomes of extreme parasites such as Paramicrocytos, Giardia, Spironucleus, and certain microsporidian lineages encode the smallest eukaryotic repertoire of DDR proteins and that pathways involved in modulation of nucleotide pools and nucleotide excision repair are the most preserved DDR pathways in the eukaryotic genomes analysed here. We found that DDR and DNA repair proteins are consistently longer than housekeeping and metabolic proteins. This is likely due to the higher number of physical protein–protein interactions which DDR proteins are involved. We find that although DNA repair proteins are generally longer than housekeeping proteins, their functional domains occupy a relatively smaller footprint. Notably, this pattern holds true across diverse organisms and shows no dependence on either lifestyle or mitochondrial status. Finally, we observed that unicellular organisms harbour proteins that are tenfold longer than their human homologues, with the extra amino acids forming interdomain regions with a clearly novel albeit undetermined function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d2f7581b50b1d038466b2efbeb5d637e6743082" target='_blank'>
              Evolutionary Insights into the Length Variation of DNA Damage Response Proteins Across Eukaryotes
              </a>
            </td>
          <td>
            Dominic Wiredu-Boakye, Laurence Higgins, O. Gahura, A. Butenko, Guy Leonard, Mark A Freeman, Á. Kristmundsson, Karen Moore, Jamie W Harrison, Shani Mac Donald, V. Yurchenko, Bryony A P Williams, Richard Chahwan
          </td>
          <td>2025-05-19</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) affects genomic stability and has potential as a biomarker for the effectiveness of poly (ADP-ribose) polymerase (PARP) inhibitors and immune checkpoint inhibitors. However, the clinical and molecular profile of HRD in non-small cell lung cancer (NSCLC), particularly in the Chinese population, remains poorly characterized. Based on the next-generation sequencing data of 158 Chinese NSCLC patients, we analyzed the HRD scores of mutations in homologous recombination repair (HRR) genes and dissected the correlation between HRD state and programmed death-ligand 1 (PD-L1) expression. Alterations in HRR genes were observed in 8.9% of the patients, with ATM and BRCA2 being the most commonly affected genes. HRD-high (HRD-H) status was significantly associated with advanced disease stage (≥III) and lung squamous cell carcinoma (LUSC). Transcriptomic analysis revealed distinct gene expression profiles between HRD-H and HRD-low (HRD-L) subgroups, with HRD-H tumors exhibiting predominantly downregulated genes. While EGFR mutations occurred at similar frequencies across HRD status, TP53 mutations were significantly enriched in HRD-H cases. HRD-H status correlated with higher PD-L1 positivity in NSCLC overall, but not within the lung adenocarcinoma (LUAD) subgroup in our cohort. The Cancer genome atlas analysis showed higher PD-L1 protein expression in HRD-H LUAD, but not in LUSC. Kyoto Encyclopedia of Genes and Genomes analysis identified enrichment of complement and coagulation cascades, ABC transporters, and bile secretion pathways in HRD-H tumors, suggesting links to immune evasion and drug resistance. This study elucidates the genomic landscape of HRD in Chinese NSCLC patients and provides insights into its potential clinical utility for therapeutic targeting. Our findings suggest that integrated HRD scoring may guide the application of PARP inhibitors and immunotherapy in specific NSCLC patient subgroups. Further prospective clinical studies are needed to validate the predictive value of HRD scoring in NSCLC treatment and to optimize patient selection strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c403957972723c45a13a1636514a41ce8c5b8d3" target='_blank'>
              Genomic alterations of homologous recombination deficiency in Chinese NSCLC patients
              </a>
            </td>
          <td>
            Shuang Xiang, Changqiong Shen, Chun Huang, Ya-ting Yang, Jing Guo, Yi Liu, Mingzhu Yin, Song Duan
          </td>
          <td>2025-06-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="This study analyzed the genetics of classical Hodgkin lymphoma (cHL) by using circulating tumor DNA (ctDNA). Two genetic subtypes were identified, differing in genetic instability mechanisms: one subtype (64% of cases) showed a higher mutation load and a higher fraction of mutations associated with activation-induced cytidine deaminase and microsatellite instability signatures, while the other subtype (36% of cases) exhibited chromosomal instability with more somatic copy number alterations. Whole-genome duplication was more common in cHL compared to other B-cell tumors and emerged as a prognostic biomarker for patients undergoing ABVD-based therapy. Non-coding regulatory mutations, similar to those in diffuse large B-cell lymphoma, were highly prevalent in 86% of cHL. A recurrent somatic expression quantitative trait locus (seQTL) involving the BCL6 gene was found in 30% of cases. The seQTL of BCL6 aligned with accessible chromatin and increased H3K27 acetylation in cHL, disrupted PRDM1 binding, and co-occurred with BCL6 expression in cHL cells. Weak to strong expression of BCL6 was observed in 68% of cases and BCL6 expression associated with gene repression similarly in cHL and germinal center B cells. After BCL6 degradation, the core set of genes directly bound and regulated by BCL6 was derepressed in cHL and proliferation was impaired. The number and clonality of neoantigens was associated with tumor microenvironment type and response to checkpoint blockade. Finally, ctDNA analysis was suggested as a tool to distinguish ambiguous PET/CT-positive lesions post-treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc33ff26bbe2b930345064a0a4545ff56f194772" target='_blank'>
              A Comprehensive Genetic Study of Classical Hodgkin Lymphoma Using Circulating Tumor DNA.
              </a>
            </td>
          <td>
            M. Pirosa, A. Bruscaggin, L. Terzi di Bergamo, M. Salehi, F. Jauk, G. Forestieri, S. Bocchetta, D. Piffaretti, R. Moia, V. Cristaldi, M. di Trani, G. A. Galimberti, K. Pini, V. Spina, Claudia Giordano, A. Condoluci, Ilaria Romano, S. Annunziata, F. Bergesio, R. Boldorini, E. Borsatti, P. Bulian, Eleonora Calabretta, S. Chauvie, Francesco Corrado, S. Crisci, M. Cuzzocrea, R. De Filippi, Bernhard Gerber, M. Kurlapski, L. M. Larocca, Elisabetta Merlo, A. Rinaldi, M. Rodari, G. Romanowicz, G. Sacchetti, A. Stathis, G. Stussi, I. Zangrilli, A. Pinto, L. Mazzucchelli, V. Gattei, J. M. Zaucha, Armando Santoro, S. Hohaus, Franco Cavalli, A. Tzankov, Carmelo Carlo-Stella, G. Gaidano, L. Ceriani, Emanuele Zucca, Davide Rossi
          </td>
          <td>2025-05-13</td>
          <td>Blood</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61df03d16c11da50362eb27baf0281aee3eb21d5" target='_blank'>
              A novel role for UPF1 in safeguarding DNA replication in S phase and beyond
              </a>
            </td>
          <td>
            Thomas Walne, Laura Maple, Nan Li, Isabel Christie, Carl Smythe, Ruth Thompson
          </td>
          <td>2025-05-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="One of the key challenges in defeating advanced tumors is the ability of cancer cells to evade the selective pressure imposed by chemotherapy, targeted therapies, immunotherapy and cellular therapies. Both genetic and epigenetic alterations contribute to the development of resistance, allowing cancer cells to survive initially effective treatments. In this narration, we explore how genetic and epigenetic regulatory mechanisms influence the state of tumor cells and their responsiveness to different therapeutic strategies. We further propose that an altered balance between cell growth and cell death is a fundamental driver of drug resistance. Cell death programs exist in various forms, shaped by cell type, triggering factors, and microenvironmental conditions. These processes are governed by temporal and spatial constraints and appear to be more heterogeneous than previously understood. To capture the intricate interplay between death-inducing signals and survival mechanisms, we introduce the concept of Death-ision. This framework highlights the dynamic nature of cell death regulation, determining whether specific cancer cell clones evade or succumb to therapy. Building on this understanding offers promising strategies to counteract resistant clones and enhance therapeutic efficacy. For instance, combining DNMT inhibitors with immune checkpoint blockade may counteract YAP1-driven resistance or the use of transcriptional CDK inhibitors could prevent or overcome chemotherapy resistance. Death-ision aims to provide a deeper understanding of the diversity and evolution of cell death programs, not only at diagnosis but also throughout disease progression and treatment adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24390bbe04bc2b2ed371e1dae568af6c9f084554" target='_blank'>
              Death-ision: the link between cellular resilience and cancer resistance to treatments
              </a>
            </td>
          <td>
            Gustavo Baldassarre, Ivana L. de la Serna, François M Vallette
          </td>
          <td>2025-05-15</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54cf680837c223860a295c7d973297324c3a3da0" target='_blank'>
              From predictive biomarker to therapeutic target: the dual role of SLFN11 in chemotherapy sensitivity.
              </a>
            </td>
          <td>
            Yue Feng, Yingze Li, Zhenhao Zhang, Yuxuan Dai, Xingchun Gou, Kejing Lao, Running Zhang
          </td>
          <td>2025-06-18</td>
          <td>Cancer chemotherapy and pharmacology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Primary liver cancer represents a significant global health burden, with limited therapeutic options for advanced disease. Long non-coding RNAs (lncRNAs) are increasingly found to play crucial roles in hepatic biology and disease progression. Here, we identify FAM99A as a highly liver-specific lncRNA that is systematically downregulated across liver malignancies, with reduced expression correlating with poor clinical outcomes. FAM99A exhibits remarkable tissue specificity with minimal expression outside the liver, and its levels rapidly decline during primary hepatocyte dedifferentiation in culture. Through isoform analysis, we establish FAM99A-203 as the predominant transcript in normal liver tissue and observe altered isoform distribution in liver cancers. Functionally, FAM99A overexpression inhibits anchorage-independent growth in liver cancer cell lines. Transcriptomic analysis reveals that FAM99A negatively regulates translation-related pathways in both liver cancer cells and primary hepatocytes. This is corroborated by protein synthesis assays showing that FAM99A overexpression substantially reduces global translation rates. Targeted RNase H-mediated extraction coupled with mass spectrometry identifies multiple components of the translation machinery as direct FAM99A binding partners, including eukaryotic translation initiation factors and RNA helicases involved in ribosome biogenesis. Clinical data analysis demonstrates significant inverse correlations between FAM99A expression and ribosomal protein genes in liver cancer patients. Additionally, hepatitis B virus appears to downregulate FAM99A expression, potentially contributing to its oncogenic properties. Our findings establish FAM99A as a liver-specific translational regulator that exerts tumor-suppressive effects by restraining protein synthesis rates, offering insights into hepatocarcinogenesis and the potential of FAM99A as both a biomarker and agent in new therapeutic avenues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8128103d9afc143a58c63b13f6c939ef19ac6372" target='_blank'>
              The Liver-Enriched Long Non-Coding RNA FAM99A Suppresses Tumorigenesis through Negative Regulation of Protein Synthesis
              </a>
            </td>
          <td>
            Nima Sarfaraz, Ranjit Kaur, Sky Harper, Lilly Oni, Srinivas Somarowthu, Michael Bouchard
          </td>
          <td>2025-05-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Aberrant function or overactivation of exonuclease 1 (EXO1) may be associated with cancer tumor development, drug resistance, and response to immunotherapy in female-related cancers. Methods By analyzing RNA-sequencing data from The Cancer Genome Atlas database, combined with validation through quantitative polymerase chain reaction experiments, we explored the expression levels of EXO1 in breast cancer (BRCA) cell lines and assessed its multidimensional roles in various female-related cancers. Results Our experiments revealed elevated expression of EXO1 in BRCA cell lines, consistent with the RNA-sequencing data. The high expression of EXO1 is associated with poor prognosis in various female-related cancers, especially in BRCA and UCEC. It significantly correlates with clinical and pathological characteristics. In specific cancer subtypes like the basal-like subtype of BRCA, high EXO1 expression is associated with a better prognosis. Genetic mutation analysis indicates a higher frequency of EXO1 gene mutations in uterine sarcoma and BRCA. DNA methylation levels may play a role in the regulation of EXO1 gene expression in some cancers. EXO1 expression is correlated with various factors within the tumor immune microenvironment and may be associated with the sensitivity to anticancer drugs. Conclusion EXO1 exhibits multidimensional roles in female-related cancers as a prognostic biomarker and potentially influences tumor immune therapy responses and drug sensitivities. Further studies are needed to fully understand the complex mechanisms underlying these associations and to explore potential therapeutic strategies targeting EXO1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00f61f934389fb7d273be0eaa350ec5d0b17ae1" target='_blank'>
              Deciphering the multifaceted role of EXO1 in female-related cancers: implications for prognosis and therapeutic responsiveness
              </a>
            </td>
          <td>
            Cong Yu, Guoying Wu
          </td>
          <td>2025-05-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Ovarian cancer is one of the most common malignancies of the female reproductive system and is associated with poor prognosis. This study aimed to utilize single-cell RNA sequencing to investigate the heterogeneity of malignant epithelial cells in ovarian cancer, focusing on their potential functions and the implications for treatment and prognosis. Methods Single-cell RNA sequencing data were clustered using a single-cell transcriptome clustering method, and malignant epithelial cells were identified through copy number variation analysis. The interaction patterns between different malignant subpopulations and immune/stromal cells were analyzed using cell-to-cell communication analysis. A risk score (URS) model based on the UBE2C + epithelial subpopulation was then constructed through LASSO and multivariable Cox regression. High and low URS groups were compared in terms of tumor mutational burden (TMB), survival outcomes, and drug sensitivity. Finally, the role of GTF2F2 in ovarian cancer progression was validated through gene knockdown experiments in an ovarian cancer cell line (ES-2). Results Three major malignant epithelial cell subpopulations were identified (TMSB4X + Epi, TSC22D1 + Epi, and UBE2C + Epi). The UBE2C + Epi subpopulation exhibited higher stemness and greater invasive potential. The constructed URS model effectively stratified patients into high- and low-risk groups, with the high-risk group displaying a higher TMB level (p = 0.00011). Drug sensitivity predictions indicated that osimertinib, rapamycin, and dihydrorotenone might have stronger inhibitory effects in the high-risk group, whereas ERK inhibitors were more effective in the low-risk group. Functional assays demonstrated that GTF2F2 knockdown significantly suppressed ovarian cancer cell migration and invasion. Western blot analyses further showed elevated E-cadherin and reduced N-cadherin expression, suggesting that GTF2F2 may promote epithelial-mesenchymal transition (EMT). Conclusion The risk score model established in this study offers a novel framework for patient stratification and personalized therapy. Notably, the identification of the UBE2C + Epi subpopulation and key genes such as GTF2F2 highlights potential diagnostic and therapeutic targets, shedding light on the pathogenesis of ovarian cancer and paving the way for precision medicine approaches. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01686-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/519b55d3039bd908fc634a743e91a97b194c1fa0" target='_blank'>
              Single-cell RNA sequencing reveals the role of GTF2F2 in ovarian cancer oncogenesis and progression
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-05-29</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 The combination of CDK4/6 inhibitors (CDK4/6i) with endocrine therapy prolongs survival of patients with ER+/HER2- metastatic breast cancer. Nearly all patients with advanced disease treated with this combination eventually progress, highlighting a need for therapeutic interventions that prevent the emergence of drug resistance. Drug tolerant persister (DTP) cells represent a reservoir of surviving cells from which drug-resistant clones eventually emerge. We hypothesize that targeting the DTP state which emerges under selective pressure from CDK4/6i could prevent the emergence of drug resistance. In vitro treatment of MCF7 and T47D ER+/HER2- breast cancer cells with the CDK4/6i Palbociclib and estrogen deprivation for 4 weeks suppressed cell viability with 2-5% of cells surviving treatment and achieving a DTP state. Cell cycle analysis by PI staining revealed that 95% of the DTP cells were arrested in G0/G1 after 4 weeks of continuous therapy. Upon drug washout, the DTP population resumed proliferation within ∼1 week and regained sensitivity to retreatment with CDK4/6i, suggesting epigenetic reprogramming is responsible for the DTP state. Whole exome sequencing did not show enrichment of known genomic alterations associated with CDK4/6i resistance (e.g., RB1, FAT1, PTEN). Bulk RNA sequencing of MCF7 DTP cells revealed transcriptome reprogramming compared to untreated controls, which reverted upon drug washout to that of the treatment naïve state. While cell cycle-related gene signatures such as E2F targets and G2M checkpoint were downregulated, interferon response signatures were upregulated in MCF7 DTP cells relative to untreated controls. MCF7 persisters were characterized by enrichment in senescence and chemotherapy-induced stressed gene signatures, while diapause and MYC target signatures were downregulated. These same signatures were similarly enriched in ER+/HER2- tumors from patients in the NeoPalAna study treated with neoadjuvant palbociclib and the aromatase inhibitor anastrozole compared to baseline biopsies. To study evolution of drug resistance, we transduced MCF7 and T47D cells with the LARRY (Lineage and RNA recovery) barcode library to uniquely label and track each persister cells long-term, and identify programs that are causal to the emergence of drug resistant clones. We performed single cell RNA-seq on barcoded cells treated with palbociclib plus estrogen deprivation for 0 h, 72 h, and 4 weeks. Several clones that arrested in G0/G1 initially eventually expanded at 4 weeks, while other clones were eliminated. We are currently investigating which genes/pathways are upregulated in the barcoded clones enriched at the 4-week time point. We will then validate results from these single cell approach in cells surviving neoadjuvant treatment in the NeoPalAna trial. In summary, a small subset of ER+/HER2- breast cancer cells survive long-term treatment with CDK4/6i plus estrogen deprivation and exhibits reversible transcriptomic features of DTPs. Our ongoing single cell-seq analyses may reveal transcriptional programs that enable the DTP state which, in turn, could be targeted to reduce survival of drug-tolerant cells and prevent the emergence of drug resistance.
 Citation Format: Yuki Matsunaga, Hima Milan Patel, Emilija Aleksandrovic, Dhyvya R. Sudan, Khushi Ahuja, Dan Ye, Chang-Ching Lin, Lei Guo, Cynthia X. Ma, Siyuan Zhang, Carlos L. Arteaga, Ariella B. Hanker. Single cell analysis enables tracking the evolution of resistance to CDK4/6 inhibitors in ER+ breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-30.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9a0b4864d21ce42a1a87671ea1a257909c574fe" target='_blank'>
              Abstract P3-04-30: Single cell analysis enables tracking the evolution of resistance to CDK4/6 inhibitors in ER+ breast cancer
              </a>
            </td>
          <td>
            Yuki Matsunaga, H. M. Patel, Emilija Aleksandrovic, Dhyvya R. Sudan, Khushi Ahuja, D. Ye, Chang-Ching Lin, Lei Guo, Cynthia X. Ma, Siyuan Zhang, Carlos L. Arteaga, A. Hanker
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Chromosomal instability in Alzheimer’s disease (AD) neurons has been previously reported. This pilot study aimed to establish a quantitative technique for assessing X chromosome centromere signals using fluorescence in situ hybridization (FISH). Hippocampal brain tissue was collected at autopsy from sporadic AD patients and age- and gender-matched controls. FISH was utilized to detect and measure the intensity of hybridization signals for X chromosome centromeres in the interphase nuclei of hippocampal brain cells. The premature centromere division (PCD) phenomenon, marked by a close bipartite signal appearing as two separated FISH spots, was examined to see if the hybridized DNA amount in each spot matched the expected centromere DNA amount. The technique effectively distinguished between PCD+ and PCD− signals. The average PCD frequency of the X chromosome in the AD group was 7 ± 1%, compared with 3.2 ± 0.84% in the controls. This quantitative approach supports qualitative analyses of FISH centromere spots, reinforcing findings of chromosomal instability in AD. The presence of a double signal at the centromere of a single X chromosome indicates re-entered cell cycles, DNA replication, and PCD in hippocampal neurons. This technique provides a reliable method for identifying PCD + signals and contributes to understanding chromosomal instability in AD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/682335163a6ea498c4fd17acd3e21f0876fd8d01" target='_blank'>
              Assessment of X Chromosome Centromere Instability in Alzheimer’s Disease: A Quantitative FISH Approach
              </a>
            </td>
          <td>
            B. Spremo-Potparević, Petar Popović, Dijana Topalović, Andrea Pirković, George Perry, L. Živković
          </td>
          <td>2025-06-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The cellular networks that maintain genome stability encompass numerous pathways involved in all aspects of nucleic acid metabolism. Through bioinformatic analysis, we identified the Zinc Finger CCCH-Type Containing 4 protein (ZC3H4), a suppressor of noncoding RNA (ncRNA) production, as a pivotal player in this system. Experimentally, ZC3H4 deficiency led to increased DNA damage, abnormal mitosis, and cellular senescence. Biochemical analysis and super-resolution microscopy revealed that the loss of ZC3H4 increased replication stress (RS)—a major driver of genome instability—by inducing a hypertranscription state that promoted R loop formation and transcription-replication conflicts (TRCs), both of which drive RS. Further bioinformatic analysis demonstrated that ZC3H4 preferentially binds to genomic regions prone to TRCs and R loops, where it suppresses ncRNA bursts, functioning as part of the Restrictor complex. Our findings identify ZC3H4 as a crucial factor in maintaining genome integrity, strategically positioned at the critical intersection of DNA and RNA synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2eceaa09e7e61059a9c40891a86a83f0e1a6b53" target='_blank'>
              ZC3H4 safeguards genome integrity by preventing transcription-replication conflicts at noncoding RNA loci
              </a>
            </td>
          <td>
            Yann Frey, Liana J Goehring, M. Haj, G. Róna, Carel Fijen, Michele Pagano, Tony T Huang, Eli Rothenberg, Y. Ziv, Yosef Shiloh
          </td>
          <td>2025-06-18</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Simple Summary This study investigated circRNA_4083, which is a stable circular RNA which originates from the MSH3 gene of chicken. It is highly expressed in cardiac, pulmonary and renal tissues. Its biogenesis depends on flanking genomic sequences. CircRNA_4083 overexpression reduced apoptosis and enhanced cell proliferation, indicating its role in promoting cellular viability. Bioinformatics analysis indicates that circRNA_4083 interacts with RNA-binding molecules and regulates pathways that are crucial for genomic stability, possibly protecting the genome from damage. These findings underscore its importance in maintaining cellular health. By elucidating circRNA-driven mechanisms, this work advances the understanding of non-coding RNA functions in poultry biology, offering insights for developing strategies to improve disease resilience and productivity in agricultural practices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c0e289cb6c7acbb75967e45fbe212b55b3a552a" target='_blank'>
              Mechanism of circRNA_4083 Circularization and Its Role in Regulating Cell Viability
              </a>
            </td>
          <td>
            Wenhao Li, Ting Yang, Haojie Wang, Hao Bai, Guobin Chang, Lingling Qiu
          </td>
          <td>2025-05-23</td>
          <td>Animals : an Open Access Journal from MDPI</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ARID1A, a subunit of the SWI/SNF chromatin remodeling complex, has emerged as a pivotal tumor suppressor altered in a broad range of human malignancies. Its frequent inactivation across diverse cancer types has revealed pleiotropic roles that intersect multiple Hallmarks of Cancer. In this review, we integrate current knowledge on how ARID1A loss influences cellular processes including proliferative signaling, resistance to cell death, genomic instability, metabolic reprogramming, immune evasion, and more. We discuss the context-specific consequences of ARID1A deficiency, its cooperation with other oncogenic events, and its implications for therapeutic vulnerability—particularly in the realm of synthetic lethality and immune modulation. By mapping ARID1A’s functional impact onto the established hallmarks framework, we highlight its centrality in cancer biology and underscore opportunities for biomarker-driven strategies and targeted interventions. Understanding ARID1A’s multifaceted roles offers a compelling lens through which to explore chromatin dysregulation in cancer and guide translational advances.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017483c65324200e0adc9a76376f32832abb8dbc" target='_blank'>
              ARID1A and Its Impact Across the Hallmarks of Cancer
              </a>
            </td>
          <td>
            Bridger Kearns, Andralyn McKell, Isaac Steveson, Peyton Worley, Braeden Barton, Jordan Bennett, DeLaney Anderson, Jacob Harris, James Christensen, Jared Barrott
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ovarian cancer has the highest mortality rate among gynecologic tumors worldwide, with unclear underlying mechanisms of pathogenesis. RNA-binding proteins (RBPs) primarily direct post-transcriptional regulation through modulating RNA metabolism. Recent evidence demonstrates that RBPs are also implicated in transcriptional control. However, the role and mechanism of RBP-mediated transcriptional regulation in tumorigenesis remain largely unexplored. Here, we show that the RBP heterogeneous ribonucleoprotein L (hnRNPL) interacts with chromatin and regulates gene transcription by forming phase-separated condensates in ovarian cancer. hnRNPL phase separation activates PIK3CB transcription and glycolysis, thus promoting ovarian cancer progression. Notably, we observe that the PIK3CB promoter is transcribed to produce a non-coding RNA which interacts with hnRNPL and promotes hnRNPL condensation. Furthermore, hnRNPL is significantly amplified in ovarian cancer, and its high expression predicts poor prognosis for ovarian cancer patients. By using cell-derived xenograft and patient-derived organoid models, we show that hnRNPL knockdown suppresses ovarian tumorigenesis. Together, our study reveals that phase separation of the chromatin-associated RBP hnRNPL promotes PIK3CB transcription and glycolysis to facilitate tumorigenesis in ovarian cancer. The formed hnRNPL-PIK3CB-AKT axis depending on phase separation can serve as a potential therapeutic target for ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52be5a8bfdb16bc7fa79af73b8cbcefd26da39c9" target='_blank'>
              hnRNPL phase separation activates PIK3CB transcription and promotes glycolysis in ovarian cancer
              </a>
            </td>
          <td>
            Fengjiang Qin, Yuya Wang, Chenyue Yang, Yifei Ren, Qinglv Wei, Yan Tang, Jie Xu, Haocheng Wang, Fatao Luo, Qingya Luo, Xin Luo, Xiaoyi Liu, Dan Yang, Xinzhao Zuo, Yu Yang, Chunming Cheng, Jing Xu, Wei Wang, Tao Liu, Ping Yi
          </td>
          <td>2025-05-24</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The ATP-hydrolytic ectoenzyme ENPP1 has been implicated in the metastasis and recurrence in triple-negative breast cancer (TNBC), primarily by contributing to tumor cell survival and treatment resistance. However, the precise mechanisms remain unclear. In a model of local recurrence (LR), circulating tumor cells (CTC) engrafting in the post-resection tumor bed developed a radioresistant phenotype linked to an ENPP1+-gene signature which was also identified in TNBC patients, suggesting ENPP1´s role in genome integrity. Blockade of ENPP1 using a permeable ENPP1 inhibitor (AVA-NP-695) reduced radioresistance, mechanistically attributed to decreased homologous recombination (HR) resulting in persistent DNA damage, as evidenced by enhanced tail moment and sustained γH2AX formation. This impaired DNA damage repair (DDR) sensitized tumor cells to ionizing radiation (IR). Notably, several DDR inhibitors (i) (including PARPi and ATMi) showed the highest synergy score in a targeted pharmacological screening. In vivo, dual ENPP1/ATM inhibition heightened radiosensitivity, compromised tumor cell survival and enhanced STING-TBK1 signaling by preventing ENPP1-mediated cGAMP hydrolysis. This resulted in robust innate and long-lasting adaptive antitumor immune memory responses, leading to significant tumor regression. Remarkably, combined treatment post-IR reduced spontaneous metastasis and local recurrence, and induced abscopal effects that impacted distant tumor spread in orthotopic tumor models. Thus, these findings position ENPP1 as a critical link between genome integrity and immunosuppression, offering promising translational opportunities for treating local or distant dissemination in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34af6ea74bf5effdace3557bda27f7680f31a271" target='_blank'>
              Dual ENPP1/ATM depletion blunts DNA damage repair boosting radioimmune efficacy to abrogate triple-negative breast cancer
              </a>
            </td>
          <td>
            Borja Ruiz-Fernández de Córdoba, K. Valencia, C. Welch, Haritz Moreno, S. Martínez-Canarias, C. Zandueta, Eduardo Gómez, Alfonso Calvo, N. Otegui, M. Echepare, Ignacio Garzón, D. Ajona, David Lara-Astiaso, Elisabet Guruceaga, Laura Guembe, Ruben Pio, I. Melero, Silve Vicent, Fernando Pastor, R. Martinez-Monge, F. Lecanda
          </td>
          <td>2025-06-13</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b91ec8476ea04c15a90514f720adb0e97fd88bcc" target='_blank'>
              ASXL1 truncating mutations drive leukemic resistance to T cell attack
              </a>
            </td>
          <td>
            D. McCurry, Zhongqi Ge, Jaehyun Lee, Rishi Pasumarthi, Xiaohong Leng, Thomas Koehnke, O. Pasvolsky, Pranaya Raparla, Vinhkhang N. Nguyen, K. Maurer, Shuqiang Li, K. Livak, Emerie Danson, Bijal Thakkar, Elham Azizi, Robert J. Soiffer, S. Shukla, Ravi Majeti, Jerome Ritz, Catherine J. Wu, J. Molldrem, P. Bachireddy
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3149b4341060758cdf9ca55d79d80ad382bb72c" target='_blank'>
              The adenovirus E4orf1 protein initiates a feedback loop involving insulin and growth factor receptors, AKT, and NF-κB, leading to abnormal DNA content in infected cells
              </a>
            </td>
          <td>
            Madison Moore, Jiang Kong, Ahlam Akmel, Michael Thomas
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa7409bece8a61b55b1fe618c705451021becc29" target='_blank'>
              Tissue-Specific Activation of Microhomology-Mediated End Joining with Age Reflects Dynamic Rewiring of DNA Repair in Rats
              </a>
            </td>
          <td>
            Diksha Rathore
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
DNA hydroxymethylation (5hmC) is known to be altered in human prostate cancer. An animal model is required to study the functional roles of the ten eleven translocation (TET) DNA dioxygenases and the 5hmC modification in prostate cancer development and progression.


METHODS
We characterized Tet expression, global genomic 5hmC, and genome-wide 5hmC patterns, and the transcriptome, in the TRansgenic Adenocarcinoma of Mouse Prostate (TRAMP) autochthonous model of prostate cancer.


RESULTS
We observed increased mRNA and protein levels of Tet1 in TRAMP samples, as compared to normal mouse prostate. Additionally, we found minimal expression of Tet2 mRNA overall, and Tet3 mRNA levels appeared similar in both sample types. However, TRAMP tumors expressed what appeared to be the inactive form of Tet3, versus the active form expressed in normal prostates. TRAMP tumors displayed global genomic hypohydroxymethylation (i.e., loss of 5hmC), and genome-wide analysis revealed widespread hypohydroxymethylation was interspersed with regions of locus-specific hyperhydroxymethylation (i.e., increased 5hmC). The differentially hydroxymethylated regions correlated with altered gene expression, and pathway analyses indicated that these genes often participate in oncogenic pathways.


CONCLUSIONS
Tet expression and 5hmC patterns are altered in the TRAMP model and closely match what has been observed in human prostate cancer, suggesting that TRAMP is a suitable model to study the role of Tets and 5hmC in prostate cancer development and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bf2dbc75da9f830aa2222da5dc9b25efa86e6a0" target='_blank'>
              Altered Ten Eleven Translocation Methylcytosine Dioxygenase Expression and DNA Hydroxymethylation in a Mouse Model of Prostate Cancer.
              </a>
            </td>
          <td>
            Matthew Aguilera, D. Klinkebiel, Victoryn Williams, Jeffrey Rovatti, Benjamin Purizhansky, Barbara A. Foster, A. Karpf, Shannon R M Kinney
          </td>
          <td>2025-05-11</td>
          <td>The Prostate</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Cervical and other anogenital malignancies are largely caused by E6 and E7 oncogenes of high-risk human papillomaviruses (HPVs), which inhibit important tumor suppressors like p53 and pRb when they are persistently activated. The main goal of traditional treatments is to physically or chemically kill cancer cells, but they frequently only offer temporary relief, have serious side effects, and have a high risk of recurrence. Exploring the efficacy and accuracy of CRISPR-Cas9 gene editing in both inducing death in HPV-infected cancer cells and restoring the activity of tumor suppressors is our main goal. In this study, we propose a novel precision oncology strategy that targets and inhibits the detrimental effects of the E6 and E7 oncogenes using the CRISPR-Cas9 gene editing system. In order to do this, we create unique guide RNAs that target the integrated HPV DNA and reactivate p53 and pRb. Reactivation is meant to halt aberrant cell development and restart the cell’s natural dying pathways. This review discusses the potential of CRISPR/Cas9 in targeting HPV oncogenes, with a focus on studies that have demonstrated its promise in cancer treatment. Given the absence of a definitive treatment for papillomavirus infection and its subsequent association with various cancers, future clinical trials and experimental investigations appear essential to establish and evaluate the therapeutic potential of CRISPR-based approaches. This approach provides a less invasive alternative to conventional treatments and opens the door to personalized care that considers the genetic makeup of each patient’s tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/612cd6ffb8df66c436b36650924c944f8c2ee3b0" target='_blank'>
              HPV-driven cancers: a looming threat and the potential of CRISPR/Cas9 for targeted therapy
              </a>
            </td>
          <td>
            Atefeh Zamani Kermanshahi, Fatemeh Ebrahimi, Ahmad Taherpoor, Narges Eslami, H. B. Baghi
          </td>
          <td>2025-05-22</td>
          <td>Virology Journal</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="DNA damage is a key factor in many human disorders, including cancer, chronic inflammation, and early aging. Genes involved in DNA damage repair (DDR) not only maintain genomic stability but also contribute to immune responses by regulating the expression of antimicrobial peptides and ligands that activate immune receptors. Various components of the DNA damage response (DDR), including DNA damage sensors, transducer kinases, and effector proteins, are capable of activating diverse immunological signaling pathways. While DDR gene mutations are common in cancers, their effects on immune characteristics are not well understood. We investigated how mutations in DDR genes influence the expression of immune regulatory genes, including immune stimulators, inhibitors, and genes related to the major histocompatibility complex (MHC) pathway. Using gene expression data from The Cancer Genome Atlas (TCGA) and mutation data from cBioPortal, we analyzed 264 DDR-related genes and 66 immune regulatory genes. These genes were clustered and categorized using Metascape, an integrative bioinformatics tool that applies enrichment-based analysis to group functionally related genes into clusters. The clustered genes were further validated through a literature review and the GeneCards database. We scored the change in immune regulatory gene expression in response to DDR gene mutations to identify differentially expressed immune stimulators, inhibitors, and MHC-related genes. Our analysis revealed positive and negative correlations between DDR gene mutations and the expression of immune modulators. These findings could help guide future cancer treatments based on biomarkers and immunotherapy strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-99965-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cc5ec8884f344e1ff8037b8e2ddb589a5a344f4" target='_blank'>
              Pancancer analysis of DNA damage repair gene mutations and their impact on immune regulatory gene expression
              </a>
            </td>
          <td>
            Kanchana Yadav, Trishala Das, Andrew M Lynn
          </td>
          <td>2025-05-05</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background/Objectives: TIGD1 (Trigger Transposable Element Derived 1) is a recently identified oncogene with largely unexplored biological functions. Emerging evidence suggests its involvement in multiple cellular processes across cancer types. This study aimed to perform a comprehensive pan-cancer analysis of TIGD1 to evaluate its expression patterns, diagnostic utility, prognostic value, and association with immunotherapy response and drug resistance. Methods: Transcriptomic and clinical data from TCGA and GTEx were analyzed using various bioinformatic tools. Expression profiling, survival analysis, immune correlation studies, gene set enrichment, single-cell sequencing, and drug sensitivity assessments were performed. Results: TIGD1 was found to be significantly upregulated in various tumor types, with notably high expression in colon adenocarcinoma. Elevated TIGD1 expression was associated with poor prognosis in several cancers. TIGD1 levels correlated with key features of the tumor immune microenvironment, including immune checkpoint gene expression, TMB, and MSI, suggesting a role in modulating anti-tumor immunity. GSEA and single-cell analyses implicated TIGD1 in oncogenic signaling pathways. Furthermore, high TIGD1 expression was linked to resistance to several therapeutic agents, including Zoledronate, Dasatinib, and BLU-667. Conclusions: TIGD1 may serve as a promising diagnostic and prognostic biomarker, particularly in colon, gastric, liver, and lung cancers. Its strong associations with immune modulation and therapy resistance highlight its potential as a novel target for precision oncology and immunotherapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1163400219b2ecb1878714587256ba612b794cd1" target='_blank'>
              Beyond Transposons: TIGD1 as a Pan-Cancer Biomarker and Immune Modulator
              </a>
            </td>
          <td>
            Merve Gülşen Bal Albayrak, Tuğcan Korak, Gurler Akpinar, Murat Kasap
          </td>
          <td>2025-05-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The overexpressed gene MYEOV in multiple myeloma, as an oncogene, has been widely recognized for its high expression levels in various malignant tumors. MYEOV plays a significant role in multiple malignancies, particularly in diseases such as multiple myeloma, breast cancer, lung cancer, pancreatic cancer, and esophageal cancer. The presence of the open reading frame of MYEOV in humans and other primates suggests its potential protein-coding capacity, although direct evidence of functional MYEOV protein is currently lacking. The role of MYEOV in various tumors is not limited to its direct effects as an oncogene; it also includes its complex role in tumor cell signaling pathways and its ability to participate in miRNA regulatory networks as a competing endogenous RNA (ceRNA). Studies have shown that MYEOV may affect the expression of cancer-related genes through enhancer activity. This article aims to provide a comprehensive review of the role of MYEOV, its involvement in signaling pathways in tumor cells, and the latest advancements in MYEOV-targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e799fb69a5049d179cc77bd3716c162048abf1c" target='_blank'>
              The role of MYEOV gene: a review and future directions
              </a>
            </td>
          <td>
            Yidan Xi, Xuefei Liu, Yanan Ge, Mingzhe Jiang, Dong Zhao, Ershu Zhao, Fei Cai, Xinlong Wang, Jiatong Li, Tingting Li, Zhendong Zheng
          </td>
          <td>2025-06-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="R-loops are three-stranded nucleic acid structures consisting of an RNA/DNA hybrid and a displaced strand of DNA. These structures have been implicated in a variety of regulatory cellular processes. Their untimed or excess accumulation, however, can cause genomic instability and induce DNA damage. Most R-loops form co-transcriptionally when the nascent transcript reanneals to unwound DNA duplex. Changes in the rate of transcription have the potential to impact R-loop formation, and compounds that modulate R-loop formation would be useful molecular tools and therapeutic leads. Cleavage and Polyadenylation Specific Factor 3 (CPSF3) recognizes the pre-mRNA 3’ cleavage site, cleaves the transcript prior to polyadenylation, and has been linked to R-loop formation. Inhibition of CPSF3 has been found to induce transcriptional readthrough and cell proliferation defects. A previous report suggested that inhibition of CPSF3 with a small molecule causes a global increase in R-loop abundance. Here we test the impact of YT-II-100, a novel inhibitor of CPSF3. We find that addition of YT-II-100 increases global R-loop formation but does not change R-loop formation at specific genes that are normally used as positive controls for R-loop formation. We performed parallel assays using previously reported compound JTE-607 and observed similar results. Our data emphasize the need for cautious interpretation of experiments using JTE-607 and YT-II-100. There may be different mechanisms of R-loop formation depending on gene loci, with the control of R-loop formation at some genes diverging from the regulation of global R-loop formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9851ed0f70643edda979c566684bef5010741b35" target='_blank'>
              Small molecule inhibition of CPSF3 impacts R-loop distribution and abundance
              </a>
            </td>
          <td>
            Cristina R. Hofman, Victor Tse, Jiaxin Hu, David R Corey
          </td>
          <td>2025-05-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="ABSTRACT The Gram-negative anaerobe Fusobacterium nucleatum is an oral oncobacterium that promotes colorectal cancer (CRC) development with the amyloid-forming cell surface adhesin FadA integral to CRC tumorigenesis. We describe here molecular genetic studies uncovering a novel mode of metabolic regulation of FadA-mediated tumor formation by a highly conserved respiratory enzyme known as the Rnf complex. First, we show that genetic disruption of Rnf, via rnfC deletion, significantly reduces the level of fadA transcript, accompanied by a near-complete abolishment of the precursor form of FadA (pFadA), reduced assembly of FadA at the mature cell pole, and severe defects in the osmotic stress-induced formation of FadA amyloids. We show further that the Rnf complex regulates three response regulators (CarR, ArlR, and S1), which modulate the expression of pFadA, without affecting fadA transcript. Consistent with our hypothesis that these response regulators control factors that process FadA, deletion of rnfC, carR, arlR, or s1 each impairs expression of the signal peptidase gene lepB, and FadA production is nearly abolished by CRISPR-induced depletion of lepB. Importantly, while rnfC deletion does not affect the ability of the mutant cells to adhere to CRC cells, rnfC deficiency significantly diminishes the fusobacterial invasion of CRC cells and formation of spheroid tumors in vitro. Evidently, the Rnf complex modulates the expression of the FadA adhesin and tumorigenesis through a gene regulatory network consisting of multiple response regulators, each controlling a signal peptidase that is critical for the post-translational processing of FadA and surface assembly of FadA amyloids. IMPORTANCE The Rhodobacter nitrogen-fixation (Rnf) complex of Fusobacterium nucleatum plays an important role in the pathophysiology of this oral pathobiont since genetic disruption of this conserved respiratory enzyme negatively impacts a wide range of metabolic pathways, as well as bacterial virulence in mice. Nonetheless, how Rnf deficiency weakens the virulence potential of F. nucleatum is not well understood. Here, we show that genetic disruption of the Rnf complex reduces surface assembly of adhesin FadA and FadA-mediated amyloid formation, via regulation of signal peptidase LepB by multiple response regulators. As FadA is critical in the carcinogenesis of colorectal cancer (CRC), the ability to invade CRC cells and promote spheroid tumor growth is strongly diminished in an Rnf-deficient mutant. Thus, this work uncovers a molecular linkage between the Rnf complex and LepB-regulated processing of FadA—likely via metabolic signaling—that maintains the virulence potential of this oncobacterium in various cellular niches. The Rhodobacter nitrogen-fixation (Rnf) complex of Fusobacterium nucleatum plays an important role in the pathophysiology of this oral pathobiont since genetic disruption of this conserved respiratory enzyme negatively impacts a wide range of metabolic pathways, as well as bacterial virulence in mice. Nonetheless, how Rnf deficiency weakens the virulence potential of F. nucleatum is not well understood. Here, we show that genetic disruption of the Rnf complex reduces surface assembly of adhesin FadA and FadA-mediated amyloid formation, via regulation of signal peptidase LepB by multiple response regulators. As FadA is critical in the carcinogenesis of colorectal cancer (CRC), the ability to invade CRC cells and promote spheroid tumor growth is strongly diminished in an Rnf-deficient mutant. Thus, this work uncovers a molecular linkage between the Rnf complex and LepB-regulated processing of FadA—likely via metabolic signaling—that maintains the virulence potential of this oncobacterium in various cellular niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b853997bcaf224c53b9af4dc2ba5437887be127" target='_blank'>
              Inactivation of the Fusobacterium nucleatum Rnf complex reduces FadA-mediated amyloid formation and tumor development
              </a>
            </td>
          <td>
            Timmie A. Britton, Ju Huck Lee, Chungyu Chang, A. Bhat, Yi-Wei Chen, Rusul Mohammed Ali, Chenggang Wu, A. Das, Hung Ton-That
          </td>
          <td>2025-05-22</td>
          <td>mBio</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Objective. This study describes mutations of genes that stimulate and regulate cell growth, programmed cell death, DNA repair, and cell growth suppression in a boy with osteosarcoma. 
Case Report. We report a case of bone sarcoma in a 9-year-old boy with possible familial predisposition. In our patient, only a subset of tumor cells expressed the ATRX protein, which is known to control the expression of several genome regions. The function of the p53 protein, which acts as a transcription factor that regulates the DNA damage repair response, cell cycle progression, and apoptosis pathways, is lost in 40-50% of malignant cells. Retinoblastoma was positive in the predominant subset of tumor cells. Deletion is found on chromosome 9, cytoband 9p21.3, where the genes for CDKN2A and CDKN2B are located. Neoplastic cells were SATB2-positive in a substantial subset, with nu- clear staining. The SATB2 protein is a DNA-binding protein involved in transcriptional regulation and chromatin remodeling. Chromosomal losses of 8p and 19q11-q13.43 were also found. These regions contain several tumor suppressor genes, including NKX3.1, whose reduced expression correlates with 8p loss in high-grade tumors. Although there was no known cancer syn- drome in the family, the maternal grandfather had a similar tumor requiring amputation. 
Conclusion. Chromosomal instability is a hallmark of osteosarcoma and is characterized by heterogeneous and extensive genetic complexity. Various numerical and structural genomic rearrangements have been described in cancer cells. However, there is little consistent genetic change to understand the etiopathogenesis of this aggressive tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec206c4dcb70e35f0aed0eba180bbda6edcf22d" target='_blank'>
              Genetic Analysis of Osteosarcoma Cells in a 9-year-old Boy: Genes Involved in Cell Cycle Control
              </a>
            </td>
          <td>
            Dragan Jovanovic, Alwajih Tariq, Sara Dlugos
          </td>
          <td>2025-06-20</td>
          <td>Acta Medica Academica</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Summary: The term “anaplasia” was coined in 1890 to describe chromosomal changes common to primordial cancer cells, but ever since the mechanisms whereby a cell becomes anaplastic has been the subject of much speculation. Recent results based on genomic and epigenomic profiles of cancer patient samples provide a glimpse into early events that lead to aneuploidy, the original defining feature of an anaplastic cell. We propose that the anaplastic cell is one in which RNA Polymerase II hypertranscribes S-phase-dependent histone genes, and the resulting histone excess facilitates DNA replication while competing for CENP-A, causing centromere breaks that initiate whole-arm aneuploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e327392e26ad44de92c835846a2ea7101b5c4f9e" target='_blank'>
              Hansemann’s anaplastic theory of cancer after 135 years
              </a>
            </td>
          <td>
            S. Henikoff, K. Ahmad
          </td>
          <td>2025-05-02</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="N-Nitrosodimethylamine (NDMA) is present in food, water, and drugs and is considered a probable human carcinogen by the International Agency for Research on Cancer. The mechanism of action of NDMA involves the generation of carcinogenic methyl lesions such as 3-methyladenine (3MeA) on DNA bases. Alkyladenine DNA Glycosylase (AAG) removes 3MeA to initiate Base Excision Repair, leaving an intermediary lesion that is subsequently resolved by backbone cleavage, nucleotide insertion, and backbone ligation. The intermediate steps following lesion removal produce potentially toxic and mutagenic single-strand DNA breaks. Here, we explored differences between males and females regarding downstream DNA damage, toxicity, mutations and cancer arising from 3MeA in the livers of WT, Aag-/-, and Aag-overexpressing (AagTg) mice. We found that males were more susceptible to NDMA-induced mutations (WT and Aag-/-) and cancer (all genotypes). In contrast, AagTg females were more prone than males to micronucleus induction. As we showed in our prior analyses where data were pooled for males and females, Aag-/- mice were significantly more susceptible to NDMA-induced mutations and cancer, and AagTg mice displayed significantly greater toxicity. Building on these findings, our analyses of sex-related differences show that Aag deficiency and maleness are both susceptibility factors for NDMA-induced liver cancer, while Aag overexpression drives toxicity, potentially with a greater effect on females. By assessing differences between males and females, this study reveals a deeper mechanistic understanding of the underpinnings for a well-known increased risk of liver cancer in men versus women by demonstrating a higher susceptibility of male mice to both mutations and cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c2ad8f9213bd9ddfa98933a729d0b81e465af3f" target='_blank'>
              Sex and Alkyladenine DNA Glycosylase Expression are Key Susceptibility Factors for NDMA-induced Mutations, Toxicity, and Cancer
              </a>
            </td>
          <td>
            Jennifer E. Kay, J. Corrigan, Lindsay B Volk, Amanda L Armijo, Ilana S Nazari, D. Torous, Svetlana L. Avlasevich, R. G. Croy, D. Wadduwage, S. Dertinger, J. Essigmann, Leona D. Samson, B. Engelward
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d761fbf2c55cb40143e978e2de946691c992d9b3" target='_blank'>
              G34R cancer mutation alters the conformational ensemble and dynamics of the histone H3.3 tails
              </a>
            </td>
          <td>
            Harrison A. Fuchs, Yunhui Peng, Shine Ayyapan, Houfang Zhang, Ruben Rosas, Anna R. Panchenko, C. Musselman
          </td>
          <td>2025-05-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Alterations in the homologous recombination repair genes, such as BRCA1 and BRCA2, are prevalent in various cancers, presenting a unique opportunity to develop synthetic lethal strategies that target homologous recombination deficiency (HRD). Poly ADP-ribose polymerase inhibitors (PARPis) have been developed to induce synthetic lethality in tumors with HRD by inhibiting the repair of single-strand DNA breaks. Beyond the initial approach to target cancers associated with HRD, the utility of PARPis has expanded to combination therapy with immune checkpoint inhibitors, anti-angiogenic drugs, or anti-androgen drugs based on the molecular biological rationale. In the field of genitourinary (GU) cancer, PARPis, such as olaparib, rucaparib, and talazoparib, are approved by the Food and Drug Administration in metastatic prostate cancer patients with BRCA1/2 mutations, sometimes in combination with other agents (e.g., olaparib plus abiraterone acetate, or talazoparib plus enzalutamide). More recently, pivotal clinical trials have broadened the potential of PARPis to the other GU cancers, including urothelial carcinoma and renal cell carcinoma. In this review, we examine the biomarkers for the response to PARPis beyond mutations in BRCA1/2 and discuss the current state and future perspectives of PARPis in GU cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28ab480620686400ea1a47db842007f871807c33" target='_blank'>
              PARP Inhibitors in Genitourinary Cancer: A New Paradigm Beyond Prostate Cancer.
              </a>
            </td>
          <td>
            Yohei Okuda, Taigo Kato, Y. Ishizuya, Takuji Hayashi, Y. Yamamoto, K. Hatano, A. Kawashima, Junko Murai, Norio Nonomura
          </td>
          <td>2025-06-09</td>
          <td>International journal of urology : official journal of the Japanese Urological Association</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cells are the fundamental structural and functional units of living organisms and the study of these entities has remained a central focus throughout the history of biological sciences. Traditional cell research techniques, including fluorescent protein tagging and microscopy, have provided preliminary insights into the lineage history and clonal relationships between progenitor and descendant cells. However, these techniques exhibit inherent limitations in tracking the full developmental trajectory of cells and elucidating their heterogeneity, including sensitivity, stability and barcode drift. In developmental biology, nucleic acid barcode technology has introduced an innovative approach to cell lineage tracing. By assigning unique barcodes to individual cells, researchers can accurately identify and trace the origin and differentiation pathways of cells at various developmental stages, thereby illuminating the dynamic processes underlying tissue development and organogenesis. In cancer research, nucleic acid barcoding has played a pivotal role in analyzing the clonal architecture of tumor cells, exploring their heterogeneity and resistance mechanisms and enhancing our understanding of cancer evolution and inter-clonal interactions. Furthermore, nucleic acid barcodes play a crucial role in stem cell research, enabling the tracking of stem cells from diverse origins and their derived progeny. This has offered novel perspectives on the mechanisms of stem cell self-renewal and differentiation. The present review presented a comprehensive examination of the principles, applications and challenges associated with nucleic acid barcode technology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d5cf40687295751922bb2f7725d94b4cd317977" target='_blank'>
              Barcodes based on nucleic acid sequences: Applications and challenges (Review)
              </a>
            </td>
          <td>
            Ying Hong Wei, Faquan Lin
          </td>
          <td>2025-05-02</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA topoisomerases, particularly type II, are crucial for DNA processes, such as replication, transcription, and chromosome segregation, making them prime targets for cancer therapy. This review delves into the multifaceted mechanisms of action of type II topoisomerases, highlighting their essential roles beyond cancer progression. It explores recent advancements in screening and designing metallic complexes as inhibitors of topoisomerase II activity. Emphasizing the structural and functional diversity between alpha and beta isoforms, it elucidates their significance in DNA metabolism and genome integrity. Additionally, this review discusses the interplay of topoisomerase II with cellular components, underscoring its regulatory roles in gene expression. Insights into screening and design strategies for metallic complex inhibitors are provided, showcasing their therapeutic potential against cancer. Overall, this review highlights the importance of understanding topoisomerase II inhibition mechanisms and the versatility of metallic complexes in biomedical research, paving the way for novel therapeutic strategies and broader applications beyond cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a3a5eac5f306d411595288c7868532f4247f662" target='_blank'>
              Topoisomerase II Inhibition in Cancer: A Focus on Metal Complexes.
              </a>
            </td>
          <td>
            A. O. Akinyemi, Josias da Silveira Rocha, Gabriela Porto de Oliveira, Josenilton de Jesus Santos, B. Owoyemi, Fillipe Vieira Rocha
          </td>
          <td>2025-06-05</td>
          <td>Mini reviews in medicinal chemistry</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract To protect genome integrity from pervasive threats of damage and prevent diseases like cancer, cells employ an integrated network of signalling pathways called the DNA damage response. These pathways involve both protein and RNA components, which can act within the damaged cell or be transferred intercellularly to influence population-wide responses to damage. Here, we show that radioprotection can be conferred by damage-derived exosomes and is dependent on YBX1-packaged Y3-derived ysRNA. In recipient cells, ysRNAs are methylated on cytosine and bound by m5C reader, YBX1. YBX1/ysRNA localise at double strand break (DSB) sites to promote efficient DNA repair and cell survival through complex formation with PARP1. YBX1 modulates PARP1 auto-modification by facilitating ysRNA ADP-ribosylation, promoting increased PARP1 residency at DSBs. Our data highlight an unprecedented role for these under-studied species of small non-coding RNAs, identifying them as a novel substrate for PARP1 mediated ADP-ribosylation with a function in DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b129803142b95daeca0e6c5e6ddbee0e7b9c7d8" target='_blank'>
              Non-coding Y RNA fragments in a complex with YBX1 modulate PARP1 residency at DNA double strand breaks
              </a>
            </td>
          <td>
            Annabelle Shaw, Kamal Ajit, Manon Chataignier, M. Gullerová
          </td>
          <td>2025-06-06</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs), among the most deleterious forms of DNA damage, are primarily repaired by non-homologous end-joining or homologous recombination repair in mammalian cells. Although DSB repair is regulated by various posttranslational modifications, such as phosphorylation, the contribution of protein arginine methylation, catalyzed by protein arginine N-methyltransferases (PRMTs), is less understood. To explore this, we conducted a screen of human PRMTs for their recruitment to the DSB sites in living cells. Among the hits, PRMT4 (also known as coactivator-associated arginine methyltransferase 1: CARM1) was found to accumulate at the DSB sites within 1 min, with the signal dissipating by 10 min post-damage. Further analysis revealed that PRMT4 recruitment to DSB sites depended on its interaction with PARP1, in which both N- and C-terminal domains of PRMT4 were required. In addition, the catalytic activity of PRMT4 was indispensable for releasing PRMT4 from the DSB sites. Finally, the PRMT4 knockout cell line exhibited delayed DSB repair, as evidenced by the neutral comet assay. Consistently, this cell line displayed a higher residual γH2AX signal compared to the parental cells following phleomycin treatment. Collectively, our findings extend the list of PRMTs involved in maintaining genome integrity and identify PRMT4 as a novel factor promoting DSB repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec4b09c7d6c576bdaf67e9c1c84a1dda0dc8942" target='_blank'>
              PRMT4/CARM1 Is a Novel Factor Promoting DNA Double-Strand Break Repair.
              </a>
            </td>
          <td>
            Yurina Abe, Hayaki Ikegame, Yuina Tsuchiya, Ryotaro Nishi
          </td>
          <td>2025-06-02</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8569a1e64e6674714c2e763b602f6c886995d4db" target='_blank'>
              Multiple FGFR1 mutations modulate tumorigenic mechanisms in glioneuronal tumors
              </a>
            </td>
          <td>
            Jacopo Boni, Míriam Fernández-González, HyeRim Han, Carla Roca, Cassandra J. Wong, Cristina Rioja, C. Nogué, Leticia Manen-Freixa, J. Boulais, Endika Torres-Urtizberea, Antonio Gómez, M. Hasselblatt, R. Estrada-Tejedor, A. Antolin, Islam E. Elkholi, Nada Jabado, Jean-François Côté, A. Gingras, Barbara Rivera
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="
 A crucial bulwark of genome stability is the DNA repair system. The homologous recombination (HR) pathway is essential for the repair of the most harmful DNA lesions, like DNA double-strand breaks. Deficiencies in this pathway result in genomic instability and contribute to tumorigenesis. BRCA1 and BRCA2 (Breast Cancer type 1 and 2 susceptibility proteins) play critical roles in HR. Germline mutations in the BRCA1 and BRCA2 genes are responsible for nearly 50% of all familial breast and ovarian cancers. BRCA1/2-deficiencies coerce tumor cells to depend on alternative DNA repair pathways. Recent studies have demonstrated that inactivation of RAD52 protein induces lethality (synthetic lethality) in BRCA1/2-deficient cancer cells, without affecting normal cells. By exploiting this synthetic lethality relationship, targeting RAD52 with inhibitors may provide an exceptionally selective method for eliminating cancer cells. Focusing on DNA repair proteins involved in synthetically lethal relationships in cancer cells has become an important strategy in the development of specific cancer therapeutics. Our laboratory has developed small molecule inhibitors targeting RAD52. We have successfully created small molecule inhibitors of RAD52 (RAD52i) and have shown their ability to inhibit the biochemical activities RAD52 including DNA annealing and DNA strand exchange. Using the GFP-based recombination assays, we show that small molecule inhibitors of RAD52 disrupt the homologous DNA repair (HDR) and single strand annealing (SSA) repair pathways in human cells. Our compounds exhibited a notable reduction in cell viability in BRCA deficient cells. Furthermore, our small molecule RAD52i exhibited considerable efficacy in a mouse xenograft model cells markedly suppressing growth of human BRCA1-mutated cancer cells. We aim to develop small molecule RAD52i as novel cancer therapeutics.
 Citation Format: Shiva Ostadrahimi, Matthew J. Rossi, Sarah F. DiDomenico, Saiful Amin, Alexander V. Mazin. TARGETING BRCA-DEFICIENT CANCER CELLS WITH SMALL MOLECULE RAD52 INHIBITORS [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-12-25.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745e4e002b1607f75e1773159229e8749f23eb29" target='_blank'>
              Abstract P4-12-25: TARGETING BRCA-DEFICIENT CANCER CELLS WITH SMALL MOLECULE RAD52 INHIBITORS
              </a>
            </td>
          <td>
            Shiva Ostadrahimi, Matthew Rossi, Sarah DiDomenico, Saiful Amin, Alexander Mazin
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromosomal instability (CIN), a pervasive feature of cancer, promotes tumor evolution and inflammatory signaling, yet its influence on innate immune sensing remains incompletely understood. Ruptured micronuclei, a direct byproduct of CIN arising from missegregated chromosomes, expose out‐of‐context double‐stranded DNA that engages the cGAS‐STING pathway. In their recent study, Sasaki et al. show that micronuclei are also a source of immunogenic double‐stranded RNA (dsRNA), triggering MAVS‐dependent type I interferon responses independently of STING. The authors show that micronuclei undergo aberrant transcription, producing dsRNA from nonexonic, transcriptionally accessible loci, with many species localizing near interferon‐stimulated genes. This work suggests a feedforward loop in which type I interferon signaling reinforces its own activation through transcriptional dysregulation. Using MPS1 inhibition to induce acute CIN, the authors show that MAVS signaling promotes MHC Class I expression and immune cell recruitment. These findings reposition CIN as a dual trigger of innate immunity through cytoplasmic DNA and RNA sensing. Future work should define how these pathways integrate in the context of chronic CIN and evaluate strategies to target DNA and RNA sensing in immune‐edited tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/828c0bc85c3a2a64866df460ecbc5473d78feef8" target='_blank'>
              CINs of the cytoplasm: dissecting dsRNA signaling in chromosomal instability
              </a>
            </td>
          <td>
            Aglaia Skolariki, Rose L Jady-Clark, Eileen E Parkes
          </td>
          <td>2025-05-07</td>
          <td>Molecular Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epigenetics and genome science have become central to current molecular biology research. Among the key mechanisms ensuring genomic integrity is the silencing of transposable elements in germline cells, a process essential for fertility in both sexes. A pivotal component of this silencing machinery involves PIWI-interacting RNAs (piRNAs), a distinct class of small non-coding RNAs that regulate gene expression and suppress transposable elements at both the transcriptional and post-transcriptional levels. piRNAs function in concert with PIWI proteins, whose expression is critical for proper oogenesis, spermatogenesis, and early zygote development. Disruptions in piRNA or PIWI protein pathways not only impair germline function but also contribute to genome instability, unchecked cell proliferation, and aberrant epigenetic modifications, hallmarks of tumorigenesis. Emerging evidence links the dysregulation of the piRNA/PIWI axis to the development and progression of various cancers, including lung and colorectal carcinomas. This review highlights the fundamental roles of piRNAs and PIWI proteins in reproductive biology and their increasingly recognized relevance in cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e264ddb82fab7df947d4ae7f4949ecb2a0652c26" target='_blank'>
              piRNA-Mediated Maintenance of Genome Stability in Gametogenesis and Cancer
              </a>
            </td>
          <td>
            Martyna Zawalska, Maciej Tarnowski
          </td>
          <td>2025-06-20</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Synthetic gene networks hold promise for genetic diagnostics and gene therapy but face limitations due to insufficient molecular tools. Gene-encoded circular single-stranded DNA (Css DNA) has been developed as a switchable vector to enrich regulatory components beyond protein/RNA-based systems in mammalian cells. However, the previous Css DNA regulator suffered from constrained regulatory sequence flexibility, disability of multiple-input multiple-output (MIMO) signals, and lack of endogenous orthogonal regulation. Here, we address these challenges by engineering a “bridge” design into the Css DNA regulator. These bridges function as sequence-programmable switches to control gene expression, responding to endogenous molecular signals (such as ATP, APE1, and RNase H) and enabling trans-regulation within or between Css DNAs. We exploit the orthogonality of Css DNA regulator to construct the three-input three-output genetic circuits. The upgraded Css DNA–based regulatory strategy represents a versatile and powerful platform for gene regulation and provides a promising avenue for the development of synthetic gene networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8aac315b23f924e6c115573b2e0b15218e25dc8" target='_blank'>
              Upgraded circular single-stranded DNA regulators for multiple-input multiple-output gene circuits in mammalian cells
              </a>
            </td>
          <td>
            Linlin Tang, Jinghao Wang, Kaiqi Xu, Zhen Li, Jie Song
          </td>
          <td>2025-05-16</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Malignant rhabdoid tumor of the kidney (MRTK) is a rare renal tumor with poor prognosis. While germline mutations of SMARCB1 are considered to be the primary cause of MRTK, emerging evidence suggests that somatic epigenetic changes also play a vital role in the development and progression of MRTK. YTHDF1, an m6A reader protein, has been implicated in regulation of tumorigenesis by influencing RNA translation and stability in several adult cancers. However, the exploration of the role of YTHDF1 in pediatric cancer, especially MRTK, remains limited. Methods In this study, CRISPR/Cas9 was employed to knockout (KO) YTHDF1 in G401 cells. The impact of YTHDF1 on the cell growth and chemoresistance were assessed using CCK-8 assays. Western blot and qRT-PCR were used to determine the changes in ferroptosis marker gene expression. Additionally, 4D-label free quantitative proteomics was conducted to uncover alterations by YTHDF1 deletion. Results We observed that the deletion of YTHDF1 in the MRTK cell line led to a significant reduction in malignancy-associated characteristics, including decreased cell motility, invasive growth, and chemoresistance. Quantitative proteomic analysis revealed that the glutathione-related signaling pathway was notably affected by YTHDF1 KO. Specifically, YTHDF1 KO resulted in a reduction of both mRNA and protein levels of Glutathione S-Transferase Mu 2 (GSTM2), a phase II metabolizing enzyme responsible for conjugating glutathione to electrophilic compounds. The decrease in GSTM2 levels following YTHDF1 KO increased the susceptibility of MRTK cells to ferroptosis. Notably, overexpression of GSTM2 in YTHDF1 KO cells partially restored the oncogenic phenotype of MRTK cells, underscoring its role in MRTK progression. Conclusions In summary, our findings provide new insights into the molecular mechanisms driving MRTK progression, highlighting YTHDF1 and GSTM2 as potential therapeutic targets for this aggressive pediatric renal tumor. Graphical Abstract YTHDF1 is highly expressed in MRTK cells. YTHDF1 recognizes the m6A modification on GSTM2 mRNA, enhancing its stability. GSTM2 protects MRTK cells against ferroptosis independent of GPX4. Supplementary Information The online version contains supplementary material available at 10.1007/s10565-025-10049-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/248db6cfe415c7d8ca495a38efdbd0087162d544" target='_blank'>
              Loss of YTHDF1 suppresses the progression of malignant rhabdoid tumor of the kidney by regulating Glutathione S-transferase Mu 2 (GSTM2)
              </a>
            </td>
          <td>
            Qian-Wen Xiong, Yuntao Liu, Min He, Xiao-Die Shen, Manli Zhao, Shuang-Ai Liu, Gensheng Zhang, Qian Liu, Jinhu Wang, Wan-Xin Peng
          </td>
          <td>2025-06-07</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b5bfa4068fc673194fc62b48dc4a50c589763e7" target='_blank'>
              Ataxin-2 preserves oocyte genome integrity by promoting timely premeiotic DNA replication
              </a>
            </td>
          <td>
            V. L. Monteiro, Crystal Yu, Camilla Roselli, B. Bakthavachalu, Mani Ramaswami, T. Hurd
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94ef251e1dfb7904ef7b82484076e545a6c98075" target='_blank'>
              DNA remnants in red blood cells enable early detection of cancer.
              </a>
            </td>
          <td>
            Haobo Sun, Xingyun Yao, Yurong Jiao, Xiangxing Kong, Yuehua Han, Ying Li, Jianping Ge, Yanfei Cao, Hongsheng Lu, Pingli Wang, Yu Xu, Jun Li, Kefeng Ding, Xiaofei Gao
          </td>
          <td>2025-05-09</td>
          <td>Cell research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Previously, we found that an uncharacterized protein CXorf38 is significantly downregulated in human ZIP8-knockout (KO) cells. Given that ZIP8 regulates essential micronutrients linked to diseases including cancer, this study aims to characterize CXorf38 and evaluate its role in lung adenocarcinoma. Methods: iTRAQ-based proteomics was previously used to identify the abundance of proteins in ZIP8-KO cells. Cell proliferation and colony formation assays were used to examine the function of CXorf38 by overexpressing the gene in lung adenocarcinoma cell lines. Kaplan–Meier survival analysis was used to assess the prognostic value of CXorf38, while TCGA clinical database analysis was used to evaluate its expression in lung cancer tissues, particularly in smokers. Bioinformatics analyses (GO, KEGG, PPI, and ICI) were performed on CXorf38-coexpressed genes derived from patients with lung cancer. Results: CXorf38 overexpression suppressed lung cancer cell proliferation and colony formation, suggesting a tumor-suppressive role. Higher CXorf38 expression correlated with improved survival in patients with lung adenocarcinoma, but not in lung squamous cell carcinoma. Clinical data showed CXorf38 downregulation with lung cancer tissues of smokers, indicating a potential role in smoking-induced cancer progression and treatment. Functional analysis using bioinformatics linked CXorf38 to immune response regulation, suggesting involvement in the tumor immune microenvironment. Conclusions: Our study reveals for the first time that CXorf38 is a potential tumor suppressor, prognostic biomarker, and/or tumor immune regulator in lung adenocarcinoma—further research is warranted to explore its role in tumor immunity and its therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0de769da2616c4ecb2cadcb3f426fcabdecfbd13" target='_blank'>
              Chromosome X Open Reading Frame 38 (CXorf38) Is a Tumor Suppressor and Potential Prognostic Biomarker in Lung Adenocarcinoma: The First Characterization
              </a>
            </td>
          <td>
            Rui Yan, H. Tan, Na-Li Cai, Le Yu, Yan Gao, Yan-Ming Xu, Andy T. Y. Lau
          </td>
          <td>2025-06-01</td>
          <td>Proteomes</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f31d8e3aa41a4d4ed605eb1a2bff762b1dfaf76" target='_blank'>
              NUSAP1 regulates mitotic processes via KIF2C interaction and AURKA phosphorylation in primary microcephaly
              </a>
            </td>
          <td>
            Decheng Ren, Keyi Li, Zhen Liu, Yan Bi, Liangjie Liu, Lei Ji, Ke Yang, Yingying Luo, Le Luo, Yang Yan, Yang Li, Feng-ping Yang, Hua Wang, Guang He, Xiao Mao
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Women who harbor germline heterozygous mutations of BRCA1 or BRCA2 have a high risk of breast cancer. Our previous study showed that patient-derived, ostensibly normal BRCA2mut/+ luminal progenitor (LP) cells are more prone to exhibit sub-chromosomal copy number variations and associated DNA damage relative to non-carriers, potentially reflecting early breast tumorigenesis. Clinically assessable biomarkers for early pathological changes of BRCA1/2 mutation in LP cells remain unknown. Single-cell RNA sequencing (scRNAseq) of LP cells of BRCA1/2mut/+ carriers revealed enrichment of KIT expression (KIT+) and subsequent transcriptional factor activations were observed in LP cells of BRCA1/2mut/+ carriers relative to non-carriers. Moreover, pathway analysis uncovered that KIT+ BRCA-mutated LP cells were enriched in pathways involving DNA binding transcription activator activity and oxidative phosphorylation. These gene signature profiles were recapitulated in bulk RNA-seq of BRCA2 mut/+ carrier LP cells. Collectively, our preliminary data suggest such analyses may identify potential biomarkers for near-term risk prediction that reflect early pathogenesis of BRCA1/2-associated carcinogenesis. Following further validation via tissue microassay analysis and clinical studies, our findings may eventually assist patients and clinicians in decision making regarding the timing and necessity of preventive surgeries for BRCA1/2 mutation carriers.
 Citation Format: Zuen Ren, Siang Boon Koh, Kai Stewart, Nick Haradhvala, Aylin S. Dedeoglu, Isabella Vianna, Ilze Smidt, Akiko Suzuki, Taisha Joseph, Veerle Bossuyt, Esther Rheinbay, Michael Lawrence, Gad Getz, Leif W. Ellisen. Genomic and transcriptomic profiling of BRCA mutation carrier tissues reveals the landscape of early pathogenesis of BRCA1/2-associated breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82a865906cd027bf501016cc53a811825bba9923" target='_blank'>
              Abstract P3-04-06: Genomic and transcriptomic profiling of BRCA mutation carrier tissues reveals the landscape of early pathogenesis of BRCA1/2-associated breast cancer
              </a>
            </td>
          <td>
            Zuen Ren, S. Koh, Kai Stewart, Nick Haradhvala, Aylin S. Dedeoglu, Isabella Vianna, Ilze Smidt, Akiko Suzuki, Taisha Joseph, V. Bossuyt, Esther Rheinbay, Michael Lawrence, Gad Getz, Leif W Ellisen
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Intra-tumor heterogeneity impacts disease progression and therapeutic resistance but remains poorly characterized by conventional histologic, immunophenotypic, and molecular approaches. Single-cell biophysical properties distinguish functional phenotypes complementary to these approaches, providing additional insight into cellular diversity. Here we link both buoyant mass and stiffness to gene expression to identify clinically relevant phenotypes within primary mantle cell lymphoma (MCL) cells, employing MCL as a model of biological and clinical diversity in human cancer. Linked measurements reveal that buoyant mass and stiffness characterize B-cell development states from naïve to plasma cell and correlate with expression of oncogenic B-cell receptor signaling genes such as BLK and CD79A. Additionally, changes in cell buoyant mass within primary patient specimens ex vivo correlate with sensitivity to Bruton’s Tyrosine Kinase inhibitors in vivo in MCL and chronic lymphocytic leukemia, another B-cell malignancy. These findings highlight the value of biophysical properties as biomarkers of response in pursuit of future precision therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/222e5440bee40a1cfc03b8b975d31d717ee09614" target='_blank'>
              Integrating Single-Cell Biophysical and Transcriptomic Features to Resolve Functional Heterogeneity in Mantle Cell Lymphoma
              </a>
            </td>
          <td>
            Ye Zhang, Lydie Debaize, Adam Langenbucher, Jenalyn Weekes, Ioulia Vogiatzi, Teemu P. Miettinen, Mingzeng Zhang, Emily Sumpena, Huiyun Liu, Sarah M. Duquette, L. Hackett, Jeremy Zhang, Sona Baghiyan, R. Redd, M. Aryee, M. Davids, Austin I. Kim, C. Ryan, David M. Weinstock, S. Manalis, M. Murakami
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Small cell carcinoma of the ovary, hypercalcemic type (SCCOHT) is a rare, aggressive cancer affecting young women, driven by inactivating mutations in SMARCA4, a key SWI/SNF chromatin remodeling gene. To uncover its druggable vulnerabilities, we performed a compound screen and found that SCCOHT cells and tumors were sensitive to PARP inhibitors. Paradoxically, SCCOHT displayed BRCA-deficient traits despite retaining wild-type BRCA1 expression. Elevated R-loop in SCCOHT sequestered BRCA1, limiting its availability for DNA damage repair. Proximity-dependent biotin identification revealed that wild-type SMARCA4, but not its pathogenic variants, promoted RNA polymerase II (Pol II) elongation by mediating the assembly of the polymerase-associated factor 1 (PAF1) complex. Thus, SMARCA4 loss increased Pol II pausing, resulting in elevated R-loops and BRCA1 redistribution. The suppression of BRCA1 activity sensitized SMARCA4-deficient SCCOHT cells and tumors to PARP inhibitors, which was further enhanced by the addition of a CDK9 inhibitor targeting Pol II elongation. Co-targeting PARP/CDK9 also elicited synergistic effects against other undifferentiated ovarian cancer cells with SMARCA4 loss. These findings link SMARCA4 loss to perturbed Pol II elongation and compromised DNA repair by BRCA1, providing a therapeutic opportunity to target SCCOHT and other SWI/SNF-deficient ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/350e8c98aa41fa4d992b18cc3ee2eb0de4ea4277" target='_blank'>
              SMARCA4 loss increases RNA Polymerase II pausing and elevates R-loops to inhibit BRCA1-mediated repair in ovarian cancer.
              </a>
            </td>
          <td>
            Xianbing Zhu, Z. Fu, Giulio Aceto, Jonathan R St-Germain, Kexin Liu, A. Arabzadeh, Yuxuan Qi, Yibo Xue, L. Witkowski, Elise Graulich, Jutta Steinberger, Selim Misirlioglu, Nicklas Bassani, Racim Sansal, Amber Yasmeen, Geneviève Morin, Jingjie Guo, A. Monast, V. Pilon, Alica Valachová, K. Pavlakis, Lili Fu, Walter H. Gotlieb, W. McCluggage, D. Huntsman, Alexander J R Bishop, Douglas A Levine, Morag Park, Yemin Wang, B. Raught, W. Foulkes, Sidong Huang
          </td>
          <td>2025-05-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="CRISPR has become a revolutionary tool for cancer biology and therapy with unprecedented precision for genome editing. Besides cancer, CRISPR is also under research for possible use in the treatment of viral diseases, such as Bovine Viral Diarrhea Virus (BVDV), a serious livestock problem. Through enabling gene modifications to be made to individual genes, CRISPR is used in identifying oncogenic drivers, characterizing tumor suppressor networks, and designing innovative therapeutics. This review focuses on the various uses of CRISPR for cancer studies with special attention to methods like gene knockout, gene activation, and base editing that each has a great potential for correcting oncogenic mutations and restoring tumor suppressor activity. Moreover, CRISPR's capacity to regulate viral replication, as that of BVDV, signifies its dual role in oncology as well as in viral pathogenesis. Interventions based on CRISPR such as chimeric antigen receptor (CAR) T cell therapy and synthetic lethality are essentially revolutionizing cancer therapy by improving immune responses and capitalizing on the specificity that is characteristic of cancer cells. However, off-target effects, tumor heterogeneity, and ethical dilemmas remain onerous challenges for clinical application. CRISPR delivery systems, despite playing a central role in advancing cancer therapies, also have prospects in optimizing the effectiveness of treatment against viral pathogens like BVDV. Advances in delivery systems using nanoparticles and viral vectors are mitigating against these challenges and improving efficacy and specificity of CRISPR reagent in vivo. In addition to this, continued progress in new technologies such as prime editing and base editing is predicted to improve precision and efficacy of CRISPR-based therapy. Since CRISPR technology is continuously developing, CRISPR's potential to treat both cancer and viral infections, like BVDV, simultaneously will be instrumental in precision medicine. This review highlights the revolutionary potential of CRISPR to revolutionize cancer treatment paradigms and brings hope for more effective and individualized therapy. With potential developments to emergent and potentiated CRISPR tools and reagents, it is predicted that they will play a center stage role in precision oncology and promote better patient results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73d3748e9db2d0a3114958d899e7fef2d33921ff" target='_blank'>
              CRISPR in Cancer Biology and Bovine Viral Diarrhea Therapy: Precision Genome Editing at the Frontier of Oncology and Viral Pathogenesis
              </a>
            </td>
          <td>
            Jindong Gao, Mengdi Zhang, Linjia Gao, Lei Kuang, Tuniyazi. Maimaiti, Ling Zhao, Qinghua Shang, Changmin Hu
          </td>
          <td>2025-05-19</td>
          <td>Bio Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Papillary thyroid cancer (PTC) is an ever-increasing cancer type worldwide, and greatly decreases the life quality and affects survival time of patients during its development and progression, but the underlying mechanisms and key factors for PTC progression are not clear. Recent studies demonstrated the potassium channel protein SK4 participates in the progression of many cancers, while it lacks the molecular mechanism study for SK4 function. In this study, we performed functional and molecular explorations for SK4 by overexpressing its level in thyroid cancer BHT101 cells. Cellular proliferation and invasion experiments were performed to assess the influences of SK4 on cell behaviors. Further, whole transcriptome sequencing (RNA-seq) analysis helped us systematically investigated the targets of SK4, including differentially expressed genes (DEGs) and regulated alternative splicing events (RASEs), and validated several related DEGs and RASEs by RT-qPCR experiment. In thyroid cancer patients, SK4 expression was completely lost in normal tissues and significantly increased in every stage of tumor tissues compared with normal tissues, which probably results from the low DNA methylation level at its promoter region. Consistent with previous study, SK4 overexpression (SK4-OE) promoted proliferation and invasion ability of BHT10 cells compared with negative control (NC). By analyzing the RNA-seq data, we detected dozens of DEGs and found that up DEGs were enriched in negative regulation of apoptotic progress, including VTCN1, MSX1, FATE1, TEK, and PRAMEF2. More importantly, we found SK4-OE globally changed the alternative splicing (AS) pattern and identified 1,639 RASEs. The genes of RASEs were enriched in DNA damage/repair, viral process, translation, and mRNA splicing pathways, which were tightly associated with the pathogenesis and progression of cancers. The splicing regulatory genes from regulated alternative splicing genes (RASGs) could partly explain the reason of global AS dysregulation by SK4-OE in BHT101 cells. Finally, we found the expression of VTCN1, EDN1, SLC29A4, RP11-473M20.16, and CH507-513H4.4 were validated by RT-qPCR, as well as the AS pattern of TMEM116. In summary, we highlight that SK4-regulated AS pattern probably is a novel regulatory mechanism for SK4 in PTC. The identified DEGs and RASEs, as well as SK4 itself, could be used as potential therapeutic targets for PTC treatment in future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bf9b607a916a81e5a08e0bc05a7b09aa5fe042f" target='_blank'>
              SK4 potentially modulates the alternative splicing profile associated with papillary thyroid cancer development in BHT101 cells
              </a>
            </td>
          <td>
            Alimujiang Wupuer, Xuekelaiti Kuerban, Gong Ye
          </td>
          <td>2025-05-28</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer development requires an evolutionary transformation from mammalian cells fully regulated by and integrated into multicellular tissue to cancer cells that, as single cell protists, are individually subject to Darwinian selection. Through genetic and epigenetic mechanisms of inheritance, the evolving cancer phenotype must acquire independence from host controls, downregulate differentiated functions that benefit the host but not individual cells, and generate phenotypic traits that increase fitness in the context of the selection forces within the local microenvironment. Here, we investigate this evolutionary transition in breast (BRCA) and lung (LUAD, without EGFR, KRAS or BRAF driver mutations) adenocarcinomas using bulk mutation and expression data from the TCGA database. We define evolution selection for genes and molecular pathways based on 1) changes in gene expression compared to normal tissue, and 2) significantly larger or smaller observed mutation rates compared to those expected based on the gene size. We find BRCA and LUAD disable different genes and gene pathways associated with tissue-specific signaling and differentiated functions but promote common molecular pathways associated with cell cycle, cell-cell interactions, cytoskeleton, voltage gated ion channels, and microenvironmental niche construction. Thus, tissue-specific parallel evolution in early cancer development is followed by convergence to a common cancer phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ced41c3cd6dbbc3e387b68d444b592a5918ecc9" target='_blank'>
              Parallel and convergent dynamics in the evolution of primary breast and lung adenocarcinomas
              </a>
            </td>
          <td>
            Robert A. Gatenby, Jamie K Teer, Kenneth Y. Tsai, Joel S. Brown
          </td>
          <td>2025-05-21</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction The nuclear receptor TR4 binding protein, TRA16, has been implicated in lung carcinogenesis; however, its broader role across diverse human cancers remains poorly understood. Understanding TRA16’s involvement in cancer biology could uncover novel regulatory mechanisms and potential therapeutic targets. Methods We conducted a comprehensive pan-cancer analysis of TRA16 expression and function across multiple human malignancies. Gene co-expression networks, pathway enrichment, transcription factor analysis, organoid modeling, and intercellular communication profiling were employed. Tumor mutation burden (TMB) and microenvironmental features were also assessed in relation to TRA16 expression, stratified by TP53 mutation status. Results Correlation analysis identified the cell cycle as the top enriched pathway among TRA16-associated genes, with key transcription factors, including RB-E2F, MYC, and TP53, regulating genes co-expressed with TRA16. In liver cancer organoid models, TRA16 and its co-expressed genes were significantly upregulated. Intercellular communication analysis showed that TRA16-positive cells exhibited increased autocrine signaling and overall signaling activity. Importantly, patients with high TRA16 expression demonstrated elevated TMB and decreased stromal and immune features. Discussion These findings highlight TRA16 as a potential master regulator of oncogenic processes, contributing to tumor progression through coordinated regulation of cell cycle genes, intercellular signaling, and genomic instability. Our results provide new insights into TRA16’s role across cancers and support its potential as a novel oncogene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd2dccc29031753da9209432f119524033cedf5b" target='_blank'>
              Pan-cancer analysis reveals TRA16 as a master regulator of human carcinogenesis
              </a>
            </td>
          <td>
            Yanyan Zhu, Yike Gao, Xiaoqing Huang, Bowang Chen, Xinyi Wang, Ying Wu, Jian Sun, Xiaoyun Huang
          </td>
          <td>2025-05-13</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 e15180


 Background:
 Pancreatic cancer has apoor prognosis and limited therapeutic options. While
 K-RAS
 mutations are prevalent in this malignancy, a subset of patients harboring wild-type
 K-RAS
 presents distinct clinical behavior. Gene fusions have been discerned in pancreatic cancer and are promising candidates for administering targeted therapies.
 Methods:
 We conducted a retrospective analysis of 926 pancreatic cancer patients of which 165 (18%) carried a wild-type
 K-RAS
 gene, by next generation sequencing (NGS) or polymerase chain reaction (PCR) testing. Tumors were analyzed with a clinical-grade hybrid capture-based RNA-seq clinical fusionome test, evaluating gene fusions between 1104 genes or to other known or novel partners as well as gene expression transcript analysis. Single nucleotide variants/insertions deletions (SNV/Indels) were tested with a clinical-grade NGS assay targeting > 300 cancer genes.
 Results:
 Gene fusions were detected in 32% of
 K-RAS
 wild-type pancreatic cancers, with 75% involving known oncogenic, clinically actionable genes including
 FGFR
 (two
 FGFR2
 and one
 FGFR1
 fusion), two
 NTRK3
 and one each of
 RAF1, RET, ROS1
 and
 BRAF
 fusions. Deleterious SNVs were frequently found in both fusion-positive and fusion-negative
 K-RAS
 wild-type pancreatic cancers. Transcriptomic expression revealed additional potentially druggable targets, e.g.,
 ROS1
 and
 FGFR3
 overexpression in a tumor harboring an
 NTRK3
 fusion,
 BRAF
 overexpression in the context of an otherwise non-actionable
 ETV6
 fusion, and
 MET
 overexpression in a
 TES::TFEC
 -fusion bearing cancer.
 Conclusions:
 Druggable fusions are frequent in
 K-RAS
 wild-type pancreatic cancers. Moreover, RNA fusion assays as well as transcriptomic expression testing can identify candidate personalized therapies for patients with
 K-RAS
 wild-type pancreatic cancer.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/723fc53e1c67e373fda744cec67f0720b9c9b122" target='_blank'>
              The cancer fusionome and overexpressed druggable oncogenic signals in
 K-RAS
 wild-type pancreatic cancer.
              </a>
            </td>
          <td>
            Michal Krawczyk, Roisin Puentes, Nathan D Montgomery, Derek D Lyle, R. Kurzrock, F. Lopez-Diaz
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18c7e36e3b876db6b83aaa4bdaf8e2434f722c8a" target='_blank'>
              Oncogenic fusions converge on shared mechanisms in initiating astroblastoma.
              </a>
            </td>
          <td>
            Yixing Shi, Qianqian Sun, Fuchuan Jia, Xiangyu Xie, Xiangyu Zhou, Rong Guo, Yangfan Zeng, Shanshan Chen, Zhenzhen Guo, Wenli Sun, Tong Guo, Yu Xia, Wenlong Li, Li Zhang, Wei Shi, Yang Yu
          </td>
          <td>2025-05-14</td>
          <td>Nature</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 The single-stranded DNA cytosine deaminase enzyme, APOBEC3B (A3B), has been implicated as a mutational driver in multiple human cancers1,2. Breast cancer in particular shows high levels of A3B expression and positive associations with poor clinical outcomes3. Recent studies have also demonstrated that human A3B expression in mice is carcinogenic with significantly shortened life expectancies4. The only genetic factor in breast cancer thus far associated with A3B expression and APOBEC signature single base substitutions is p53 mutation5. Therefore, here we test the hypothesis in vivo that human A3B expression will show pathlogical synergy in the form of accelerated mammary tumor penetrance upon combination with p53 haploinsufficiency due to a heterozygous deletion mutation spanning exons 2-10. Custom and commercially available animals were subjected to standard breeding practices to generate experimental (A3B, p53Delta exons 2-10/+) and control groups (p53Deltanexons 2-10/+). Surprisingly, in contrast to A3B-accelerated tumor development on a wildtype genetic background, A3B had no effect on the rates of tumor development in p53 heterozygous animals (median 13.5 months). The observed genetic epistasis was not due to a lack of A3B expression or activity, as evidenced by strong IHC positivity and a clear acculumlation of APOBEC signature single base substitution mutation in tumors. We conclude that A3B-accelerated tumor development requires full p53 function, and that a haploinsufficiency in p53 enables tumor cells (or their precursors) to better tolerate DNA damage lesions induced by A3B.
 Selected References:
 1. Petljak, M. et al. Addressing the benefits of inhibiting APOBEC3-dependent mutagenesis in cancer. Nat Genet 54, 1599-1608 (2022).
 2. Butler, K. & Banday, A. R. APOBEC3-mediated mutagenesis in cancer: causes, clinical significance and therapeutic potential. J Hematol Oncol 16, 31 (2023).
 3. Roelofs, P. A. et al. Clinical implications of APOBEC3-mediated mutagenesis in breast cancer. Clin Cancer Res 29, 1658-1669 (2023).
 4. Durfee, C. et al. Human APOBEC3B promotes tumor development in vivo including signature mutations and metastases. Cell Rep Med 4, 101211 (2023).
 5. Burns, M. B. et al. APOBEC3B is an enzymatic source of mutation in breast cancer. Nature 494, 366–370 (2013).\
 Citation Format: Joshua Proehl, Cameron Durfee, Yuan Zhao, Nuri Alpay Temiz, Reuben Harris. p53 Haploinsufficiency Epistatically Masks A3B Tumor Phenotype in vivo [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-06-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8de4891dd8d8ec2c399682371385c3645385994" target='_blank'>
              Abstract P3-06-26: p53 Haploinsufficiency Epistatically Masks A3B Tumor Phenotype in vivo
              </a>
            </td>
          <td>
            Joshua Proehl, Cameron Durfee, Yuan Zhao, N. A. Temiz, R. Harris
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="
 3046


 Background:
 Prior work has shown that detection of circulating tumor DNA (ctDNA) at time of diagnosis of leiomyosarcoma (LMS) is associated with lower likelihood of objective response and patients with detectable ctDNA after two cycles of chemotherapy have worse survival. However, because ctDNA exists at much lower concentrations in plasma compared to cell-free DNA of non-tumor origin, the sensitivity of these prognostic measures to tumor signals remains unclear. With increasing evidence of ctDNA fragments being shorter than the background non-tumor cell-free DNA, we sought to test whether restricting our analysis to smaller fragment sizes would improve detection of ctDNA in LMS.
 Methods:
 Plasma was serially collected from patients with LMS undergoing chemotherapy. Cell-free DNA extracted from these samples was profiled by ultra-low-passage whole-genome sequencing (ULPWGS). Copy number alterations were identified and used to detect ctDNA using the ichorCNA algorithm before and after restricting the dataset to fragments of 90-150bp (short). We compared detectability of ctDNA via ichorCNA between analyses using all sequencing data and those using data restricted to short fragments.
 Results:
 From 28 patients, 126 plasma samples were profiled. The median fragment length of cell-free DNA was 240bp (IQR 142-349) for patients with LMS, compared to 307bp (IQR 171-465, p < 0.001) in samples collected from healthy controls. Short fragments made up 19.48% of LMS libraries at diagnosis when ctDNA levels were highest, 15.27% of all LMS samples, and 12.96% of libraries from healthy controls. While ctDNA was detectable by ULPWGS in 17% of all samples, detection increased to 40% when analyzed using only short cell-free DNA fragments (p < 0.0001). The proportion of diagnostic samples with detectable ctDNA was nominally higher when analyzed by short fragments (39% vs. 64% with 90-150bp size restriction, p = 0.1078, n = 28) and was significantly higher in samples collected after two cycles of chemotherapy (5% vs. 40% with 90-150bp size restriction, p = 0.0197, n = 20). Increases in ctDNA detectability with fragment size restriction were also observed in each of localized and metastatic subgroups (metastatic: 16% without restriction, 38% with restriction, n = 97, p = 0.0012; localized: 17% vs. 45%, n = 39, p = 0.04).
 Conclusions:
 Our results demonstrate that detection of ctDNA is improved by analyzing short fragments of cell-free DNA in samples collected from patients with LMS. These findings represent a potential to increase the sensitivity of an affordable, low-coverage liquid biopsy assay and may enhance the prognostic value of ctDNA detection in these patients. Further study of the association between ctDNA detection and outcome is needed to fully validate the impact of fragment size restricted analysis of cell-free DNA samples in patients with LMS and is currently ongoing in a prospective study.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8ceaaf3e27c10098ba0b8ae1de9b437c08db2f3" target='_blank'>
              Improved detection of circulating tumor DNA in patients with leiomyosarcoma with fragment size restriction.
              </a>
            </td>
          <td>
            N. Ruzgar, K. Klega, Cora A. Ricker, Mohammad Tanhaemami, Amanda M. Shafer, Denise K. Reinke, Brittany L. Siontis, S. Schuetze, B. Crompton
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background: Isocitrate dehydrogenase 1 and 2 (IDH1 and IDH2) are enzymes that catalyze the oxidative decarboxylation of isocitrate to alpha-ketoglutarate (α-KG), which is essential for many metabolic processes, including some steps in DNA repair. In tumors, notably in gliomas, IDH1 and IDH2 are frequently mutated. The mutation found in different cancers is functionally active, causing, instead of α-KG, the formation of 2-hydroxyglutarate (2-HG), which inhibits α-KG-dependent enzymes. Gliomas harboring mutated IDH1/2 show a better prognosis than IDH1 wild-type (wt) tumors of the same grade, which might result from the inhibition of DNA repair functions. A DNA repair enzyme dependent on α-KG is alkB homolog 2 (ALKBH2), which removes several lesions from DNA. These findings prompted us to investigate the response of glioma cells to artesunate (ART), a plant ingredient with genotoxic and anticancer activity currently used in several trials. Materials and Methods: We used isogenic glioblastoma cell lines that express IDH1 wild-type or, based on a TET-inducible system, the IDH1 mutant (mt) protein, and treated them with increasing doses of artesunate. We also treated glioblastoma cells with 2-HG, generated ALKBH2 knockout cells, and checked their sensitivity to the cytotoxic effects of artesunate. Results: We show that the cell-killing effect of ART is enhanced if the IDH1 mutant (R132H) is expressed in glioblastoma cells. Further, we show that 2-HG imitates the effect of IDH1mt as 2-HG ameliorates the cytotoxicity of ART. Finally, we demonstrate that the knockout of ALKBH2 causes the sensitization of glioblastoma cells to ART. Conclusions: The data indicate that ALKBH2 protects against the anticancer effect of ART, and the mutation of IDH1/2 commonly occurring in low-grade gliomas sensitizes to ART via an ALKBH2-dependent mechanism. The data support the use of ART in the therapy of IDH1/2-mutated cancers both in combination with chemotherapy and adjuvant treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2cc28ac7b0185e3dec2c573931d59e4e677a09d" target='_blank'>
              IDH1 Mutation Impacts DNA Repair Through ALKBH2 Rendering Glioblastoma Cells Sensitive to Artesunate
              </a>
            </td>
          <td>
            Olivier Switzeny, Stefan Pusch, M. Christmann, Bernd Kaina
          </td>
          <td>2025-06-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="
 e15067


 Background:
 Our group previously identified the oncogenic fusion
 CRTC1::MAML2
 , arising from t(11;19) translocations in mucoepidermoid carcinoma, the most common type of salivary gland malignancy. Subsequent studies have reported additional fusions of
 MAML2
 with
 CRTC3
 ,
 YAP1
 , and
 NR1D1
 . These fusions retain the C-terminal
 MAML2
 transactivating domain (TAD), which is believed to drive aberrant gene transcription. While the oncogenic roles of these
 MAML2
 fusions have been well-established in preclinical and clinical studies, we aimed to identify novel
 MAML2
 fusions across a broader range of human malignancies.
 Methods:
 DNA and RNA sequencing were performed on tumor samples submitted to Caris Life Sciences.
 MAML2
 fusions were identified from RNA transcripts and filtered to include only known pathogenic fusions or in-frame fusions containing a C-terminal
 MAML2
 TAD that were observed in ≥3 samples. Fusion burden was defined as the number of unique fusion isoforms per sample. Statistical significance was evaluated using Mann-Whitney, Fisher’s Exact, or Chi-squared tests.
 Results:
 Among 180,124 tumor samples analyzed, 509 specimens were found to harbor
 MAML2
 fusion transcripts. After filtering for non-recurrent fusions and those lacking a
 MAML2
 TAD, 143 unique fusions remained. Over half of these specimens harbored known fusions involving
 CRTC1
 (42/143),
 YAP1
 (37/143),
 CRTC3
 (7/143), and
 NR1D1
 (2/143). In addition, novel fusions with
 MTMR2
 (31/143),
 SESN3
 (11/143),
 CCDC82
 (6/143),
 FAM76B
 (4/143), and
 ATXN3
 (3/143) were identified. Compared to known fusions, the novel fusions were identified in fewer sequencing reads (8 vs 13 median junction reads, p = 0.0064) and tended to have a higher median fusion burden (6 vs 2 median fusion isoforms per sample, p < 0.0001), more frequent
 TP53
 co-mutations (80% vs 11.5%, p < 0.0001), and no clear association with tissue of origin. Novel fusions, excluding
 ATXN3::MAML2
 , were located near
 MAML2
 in the genome and appeared to result from duplication or deletion events. At least 75% of samples harboring each novel fusion also carried a concurrent pathogenic mutation, primarily involving
 TP53
 . In contrast,
 ATXN3::MAML2
 fusions, the product of which interacts with YAP1, arose via translocation, occurred in samples with a low fusion burden (2 fusion isoforms per sample), and were not associated with
 TP53
 mutations.
 Conclusions:
 Our comprehensive analysis of tumor samples identified several novel
 MAML2
 fusion partners. Most of these appear to represent passenger alterations arising from genomic instability, likely due to impaired p53 function. However,
 ATXN3::MAML2
 fusions, previously reported in a precancerous pancreatic disease case, may represent a pathogenic alteration warranting further investigation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e97cd82342c70352bfdec0ff8af6680522007f" target='_blank'>
              Novel
 MAML2
 fusions as oncogenic drivers in human malignancy.
              </a>
            </td>
          <td>
            Takefumi Komiya, K. Sweeney, Chao Hui Huang, A. Crymes, Beverly Wang, E. Antonarakis, A. Elliott, M. Oberley, M. Evans
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cervical cancer remains a significant global health concern, impacting over half a million women annually. The primary cause is a persistent infection with hr-HPV, which disrupts various cellular processes crucial for normal function. This disruption leads to genetic instability, including changes in the expression of microRNAs and their corresponding host genes, with far-reaching consequences for cellular regulation. Researchers have widely utilized high-throughput technologies to analyze gene expression in cervical cancer, aiming to identify distinct molecular signatures of microRNAs and genes through genomic analysis. However, discrepancies among studies have been noted, possibly due to variations in sample collection, technological platforms, and data processing methods such as normalization and filtering. Therefore, it is essential to synthesize findings from diverse studies to comprehensively understand the molecular mechanisms of regulatory genetic networks involved in the initiation and progression of cervical cancer. This review examined the evidence detailing the role of microRNA signatures and their target genes in cervical carcinogenesis and disease advancement. The accumulated data suggest the presence of widespread regulatory genetic networks active in both precancerous and cancerous cervical cells, potentially acting as key drivers of this malignancy. Identifying these molecular genomic signatures could open new avenues for developing therapeutic strategies for cervical cancer, particularly in the realm of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5110a729c4003c4b384cf7e2d5409b9bf5d54dee" target='_blank'>
              Regulatory Genetic Networks by microRNAs: Exploring Genomic Signatures in Cervical Cancer
              </a>
            </td>
          <td>
            C. Pérez-Plasencia, Y. C. Orbe-Orihuela, Armando Méndez-Herrera, Jessica Deas, Claudia Gómez-Cerón, H. Jiménez-Wences, J. Ortiz-Ortiz, G. Fernández-Tilapa, Aldo F. Clemente-Soto, J. R. Parra-Unda, J. S. Velarde-Félix, M. Rodríguez-Dorantes, O. Peralta-Zaragoza
          </td>
          <td>2025-06-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 2519


 Background:
 Tumor heterogeneity and low mutational burden limits the availability of effective immunotherapy targets. Aberrant RNA-splicing (neojunctions) represents an underexplored yet promising source of neoantigens. To address this, we developed a neoantigen discovery platform (SNIPP) that characterizes a novel class of clonally-expressed, splicing-derived neoantigens. Furthermore, we validated the immunogenicity of these neoantigens by identifying specific TCRs that drive CD8+ T-cell-mediated tumor killing.
 Methods:
 SNIPP identified public neojunctions by analyzing TCGA RNA-seq data, selecting neojunctions with a positive sample rate (PSR) > 10% and filtering out those found in GTEx normal tissue RNA-seq data (PSR < 1%) across 12 cancer types. To characterize intratumorally conserved neojunctions, we performed maximally-distanced multi-site biopsies (
 n
 = 535) within glioma patients (
 n
 = 56) and generated RNA-seq data for each intratumoral site. Two independent algorithms were utilized to predict peptide processing likelihood and HLA-binding affinity of splicing-derived neoantigen candidates. Neoantigen-specific TCR sequences were identified via
 in vitro
 sensitization of PBMCs and subsequent 10x V(D)J scRNA-seq. These TCRs were transduced into CD8+ T-cells, which were tested downstream for immunogenicity and cytotoxicity against glioma cell lines.
 Results:
 Our pipeline identified 789 public neojunctions, including 32 neojunctions concurrently detected in transcriptomic and proteomic glioma datasets and confidently predicted to be presented by HLA-A*02:01. IVS and subsequent 10x V(D)J scRNA-seq identified TCR clonotypes reactive against neojunctions in
 RPL22
 (
 n
 = 7) and
 GNAS
 (
 n
 = 1), with the latter exhibiting high intratumoral conservation (detected in > 90% of spatially-mapped biopsies across 17/56 patients (26.78%)). TCR-transduced CD8+ T-cells recognized and were immunogenically activated and demonstrated cytotoxicity against endogenously processed and presented neoantigens in GBM and melanoma lines. Additionally, IDH1-mutant oligodendrogliomas exhibited significantly higher neojunction expression compared to IDH1-mutant astrocytomas and IDH1wt subtypes. Differential gene expression analysis (DESeq2) revealed reduced expression of splicing factors in oligodendrogliomas, attributed to their specific co-deletion of chromosomes 1p and 19q. CRISPRi-mediated knockdown of these splicing factors (e.g. SF3A3, SNRPD2) in IDH1wt glioma cells resulted in significantly increased expression of corresponding neojunctions.
 Conclusions:
 Our study highlights a novel class of neoantigens derived from tumor-wide aberrant RNA splicing. The SNIPP platform effectively identifies public intratumorally-conserved neojunctions with strong therapeutic potential. Furthermore, elevated neojunction expression in oligodendroglioma underscores the mechanistic link between dysregulated splicing factor expression and RNA splicing abnormalities.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/476976fc09e028193d5d0a3658a6d5d31763d4d8" target='_blank'>
              Tumor-wide RNA splicing aberrations and their potential as therapeutic neoantigen targets.
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O. Stevers, Iñaki Etxeberria, T. Nejo, Maggie Colton Cove, Lee-may Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Aaron A. Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideho Okada
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is among the most prevalent malignancies worldwide. Cuproptosis, a copper-induced form of cell death, has been identified as a key process in LUAD progression; however, the molecular mechanisms underlying cuproptosis in LUAD and potential therapeutic targets remain unclear. The present study utilized The Cancer Genome Atlas database to retrieve mRNA expression profiles and clinical information of LUAD, identifying 10 candidate genes from differentially expressed genes associated with cuproptosis. Protein-protein interaction analysis indicated that CDK inhibitor 2A (CDKN2A), an upregulated gene in LUAD, may function as a hub gene. Furthermore, multiple online databases were used to analyze Spi-1 proto-oncogene (SPI1), a transcription factor upstream of CDKN2A, which was downregulated in LUAD cuproptosis. The LinkedOmics database identified the p53-mediated cuproptosis-related pathway regulated by CDKN2A. Gene expression patterns were examined through Gene Expression Profiling Interactive Analysis, the Human Protein Atlas and reverse transcription-quantitative polymerase chain reaction. Prognostic significance was assessed using the UALCAN and Kaplan-Meier plotter databases. In vitro experiments demonstrated that CDKN2A knockdown and SPI1 overexpression inhibited the proliferation and migration of the H1975 cell line. After copper-induced cuproptosis in H1975 cells, SPI1 expression was upregulated, whereas CDKN2A expression was downregulated. When H1975 cells were pretreated with tetrathiomolybdate, the upregulation of SPI1 was inhibited and the downregulation of CDKN2A was also suppressed. Cell Counting Kit-8 assays indicated that SPI1 overexpression and CDKN2A knockdown facilitated elesclomol-CuCl2-induced cuproptosis. Western blot analysis revealed an inverse association between SPI1 overexpression and CDKN2A/p53 levels. In conclusion, the present study demonstrated the role of the SPI1/CDKN2A/p53 axis in LUAD cuproptosis, providing insights into potential therapeutic targets and contributing to clinical research on treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a265e6ec17f3bc3d7df377a278752607ec192b44" target='_blank'>
              Role of the SPI1/CDKN2A/p53 signaling pathway in cuproptosis of lung adenocarcinoma cells
              </a>
            </td>
          <td>
            Jiayue An, Wei-Wei Song, Qin Wang, Boyu Tan, Xuan Fei, Ruoxi Wang, Siyan Li, Xiyu Lu, Youjie Li, Ning Xie
          </td>
          <td>2025-05-19</td>
          <td>Oncology Letters</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Objective: To undertake a literature review on the development of cervical cancer, emphasizing epidemiological data, human papillomavirus (HPV) infection in carcinogenesis, and the role of the XRCC1 gene in DNA repair, in order to provide valuable information for the medical and scientific community. The goal is to provide a valuable resource for clinical practice. Bibliographic Review: Cervical cancer ranks as the fourth most common malignancy worldwide. Despite known environmental, behavioral, and genetic risk factors, their management remains inadequate. Persistent infection with oncogenic HPV is a well-established risk factor; however, not all infected individuals develop cervical cancer, suggesting the involvement of additional mechanisms. Among these, polymorphisms in the XRCC1 gene have gained attention. XRCC1 encodes a scaffold protein essential for DNA repair coordination and cell cycle regulation. Three main XRCC1 polymorphisms—Arg194Trp, Arg280His, and Arg399Gln—are associated with impaired DNA repair, genomic instability, and increased cervical cancer susceptibility. Final Considerations: A deeper understanding of the genetic and cellular mechanisms underlying cervical cancer may facilitate the development of targeted medical interventions for women, contributing to improved quality of life reduced mortality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607876d22f013a85473f817b1c9f164dd5e62623" target='_blank'>
              Cervical cancer and XRCC1 gene polymorphisms
              </a>
            </td>
          <td>
            Andréia Michelle Alves Cunha de Alcântara, Ivan Barbosa Barros, Israel Santos da Silva, Karla Roberta Lemos Maul de Souza, Ivan Barbosa Barros, M. Maia, Paulo Roberto Eleutério de Souza
          </td>
          <td>2025-05-30</td>
          <td>Revista Eletrônica Acervo Saúde</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The CHEK2 gene serves a canonical role in the DNA damage response (DDR) pathway encoding the regulatory kinase CHK2 in the homologous recombination (HR) repair of double-strand breaks (DSB). Although CHEK2 is traditionally considered a tumor suppressor gene, recent studies suggest additional functions. Across several cohort studies, CHEK2 expression was negatively correlated with the efficacy of immune checkpoint inhibitors (ICI), which target the interaction between effector immune and tumor cells. This review explores two possible explanations for this observed phenomenon: the first relating to the canonical role of CHEK2, and the second introducing a novel role of the CHEK2 gene in immunomodulation of the tumor microenvironment (TME). DDR mutations have been implicated in increased levels of tumor mutation burden (TMB), often manifesting as neoepitope expression on the tumor cell surface recognized by effector immune cells. As a result, impaired DNA repair due to CHEK2 loss of function, either from germline deleterious variants or acquired mutations, results in the recruitment of CD8+ cytotoxic T-cells and subsequent efficacy of ICI treatment. However, functional loss of CHEK2 may be directly involved in potentiating the immune response through canonical inflammatory and anti-tumor pathways, acting through the cGAS-STING pathway. Although the exact mechanism by which CHEK2 modulates immune responses is still under investigation, combination therapy with CHEK1/2 inhibition and ICI immunotherapy has shown benefit in preclinical studies of several solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/561bae3be7aafdc8f4a911392f292dcb271c31d1" target='_blank'>
              Beyond DNA damage response: Immunomodulatory attributes of CHEK2 in solid tumors
              </a>
            </td>
          <td>
            Helen Qian, Heba Ali, Vivekanudeep Karri, Justin T. Low, David M Ashley, Amy B. Heimberger, Lucy A. Godley, A. Sonabend, C. Dmello
          </td>
          <td>2025-06-10</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 The loss of chromosome 16q is the single most common genomic event in BC, occurring in over 50% of tumors. Despite this, the literature is focused almost exclusively on the importance of focal mutations that show a maximum frequency of ∼35-40%. We have modeled 16q loss by deleted this region in mammary epithelium of genetically modified mice. This event was sufficient to induce mammary tumor formation. This finding establishes for the first time that 16q is a BC-suppressing chromosome arm. We have gone on to identify two genes on 16q that can promote tumor formation when one copy is lost (Cbfb and Ankrd11, each in cooperation with Pik3caHR). We have also identified mutations that cooperate with 16q syntenic sequence loss in spontaneous tumors from our 16q mouse model (c-Met gene amplification and p53 mutation) as well as in a Sleeping Beauty transposon-based screen (identifying mutations that enhance Ras and Rho signaling). Finally, we have used transcriptional profiling and proteomics to compare matching mammary tumors with/without 16q-syntenic sequence loss, which revealed that hemizygous deletion of this region suppressed keratinocyte-like differentiation in mammary tumors.
 Citation Format: Sean Egan, Idil Eda Temel, YeJi An, Katelyn Kozma, Amanda Loch, Wei Wang. 16q is a breast cancer suppressor arm [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e57f36d665bf1dd94705e6477b5680fa560c1a" target='_blank'>
              Abstract P3-04-17: 16q is a breast cancer suppressor arm
              </a>
            </td>
          <td>
            Sean E Egan, Idil Eda Temel, Yeji An, Katelyn J Kozma, Amanda J. Loch, Wei Wang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Genome instability is a major force driving tumorigenesis. The ssDNA-binding protein complex CTC1-STN1-TEN1 (CST) plays a pivotal role in maintaining genome stability by countering replication stress, modulating DNA damage repair, and maintaining telomere integrity. Despite its well-documented role in genome maintenance, the involvement of CST in skin cancer development has yet to be investigated. We recently found that CST localizes at stalled DNA replication sites after UV exposure and may suppress the unwanted repriming activity, suggesting a potential role of CST in suppressing genome instability caused by UV damage. In this study, we first analyzed CST expression and alterations in cutaneous melanoma database and found that the CST genes are frequently altered in cutaneous melanoma and their expression is significantly downregulated in melanoma samples compared to normal tissues. We then generated a conditional knockout (cKO) mouse model with STN1 deficiency specifically in melanocytes to investigate its role in skin cancer formation. Upon chronic exposure to UV irradiation, STN1-deficient mice exhibit no obvious difference in melanoma incidence compared to control littermates, suggesting that STN1 downregulation in mature melanocytes has no significant effect on UV-induced skin cancer development in lab mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b040e5d51366f43fd311948a74f723a17b17afb" target='_blank'>
              In vivo investigation of STN1 downregulation in melanoma formation in adult mice following UV irradiation
              </a>
            </td>
          <td>
            Sara Knowles, Fang Wang, Maarten C Bosland, S. Gaddameedhi, Weihang Chai
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 A deeper understanding of the complicated transcriptional landscape of human breast cancers are vital to identifying more efficacious treatments. Investigating genetic variations among breast cancer subtypes at a single-cell level holds the potential to enhance our understanding of cancer formation, progression, and elucidate actionable targets. In this study, we compile single-cell RNA sequencing data from patient tumors and matched lymph metastasis, reduction mammoplasties, breast cancer patient-derived xenografts (PDXs), PDX-derived organoids (PDXOs), and cell lines, forming a diverse dataset consisting of 117 samples with a total of 506,719 cells. These samples include hormone receptor-positive (HR+), human epidermal growth factor receptor 2 positive (HER2+), and triple-negative breast cancer (TNBC) subtypes, along with isogenic model pairs. Within this study, we outline the similarities and differences across these models and patient samples, and assess therapeutic drug efficacy based on subtype proportions. Within the TNBC subtype, PDX models exhibited a closer resemblance to patient samples in terms of tumor heterogeneity and cell cycle features compared to cell lines. Acquired drug resistance in TNBC PDX tumors was linked to an increase in basal-like cell proportions, as defined by SCSubtype and TNBCtype cell typing predictors. Cell-wise subtyping revealed all patient samples comprised a mix of subtypes; notably, HR+ lymph node metastases had a lower proportion of HER2-Enriched cells compared to matched primary tumors. PDXOs displayed differences in metabolic-related transcripts relative to matched PDX tumors. Correlative analyses of cytotoxic drugs on PDX cells demonstrated that therapeutic efficacy was dependent on subtype proportions. Herein we provide an extensive multi-model dataset, a dynamic approach to cell-wise sample annotation, and a thorough examination of models within human breast cancer systems. This analysis and reference resource will support informed decision-making in preclinical research and therapeutic development by establishing model limitations, subtype-specific insights, and novel targetable pathways.
 Citation Format: Julia Altman, Amy L. Olex, Emily K. Zboril, Carson J. Walker, David C. Boyd, Rachel K. Myrick, Nicole S. Hairr, Jennifer E. Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. Steven Chen, Yunshun Chen, Charles M. Perou, Brian D. Lehmann, Jane E. Visvader, J. Chuck Harrell. Single-cell transcriptional features of human breast tumors and model systems [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad1272929a89f74bc5ab4bbe73f4045f2023308" target='_blank'>
              Abstract P5-06-19: Single-cell transcriptional features of human breast tumors and model systems
              </a>
            </td>
          <td>
            Julia E Altman, Amy L. Olex, Emily K. Zboril, Carson J Walker, David C. Boyd, Rachel K. Myrick, Nicole S Hairr, Jennifer E Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. S. Chen, Yunshun Chen, C. Perou, B. Lehmann, Jane E. Visvader, J. Harrell
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="
 3577


 Background:
 HRD is linked to sensitivity to platinum-based chemotherapy and poly (ADP-ribose) polymerase inhibitors across various tumor types. However, the presence of HRD, driver events (such as pathogenic mutations in HR genes or other genomic alterations), and its clinical relevance in mCRC remain underexplored.
 Methods:
 We performed the VHIO-300 test, an ISO15189 accredited custom NGS panel profiling over 450 genes (including HR-related genes:
 ATM, BRCA1, BRCA2, BRIP1, CHEK2 or PALB2
 ) on 356 Stage IV mCRC patients (corresponding to 247 primary colorectal and 109 to metastatic samples) enrolled in the Vall d’Hebron Institute of Oncology’s Molecular Prescreening Program from June 2021 to December 2024. All samples had a tumor cellularity > 40% as per pathologist evaluation
 .
 An HR score (sHR) based on genome-wide copy number alterations (CNA) and loss of heterozygosity (LOH) patterns is generated. After cross-validation with Myriad MyChoice, sHR ≥ 56 was established based on a cohort of ovarian tumors and used to identify HRD in mCRC tumor samples.
 Results:
 HRD prevalence was 3.4% in our mCRC cohort (12/356), but, much higher in metastatic lesions, (6.4%, 7/109) than in primary samples (2.03%, 5/247) (p = 0.05). In fact, the median sHR between primary (21) vs. metastases (30) in CRC was significantly different (p < 0.01). Regarding HR gene status, HR-mutated samples were not significantly within the HRD group (p = 0.27) and only 6.5% (2/31) were HRD. Noteworthy,
 BRCA2
 exhibits a frameshift deletion in a homopolymer stretch, that is a frequent hotspot in microsatellite instable (MSI) tumors, but this event was not found to be associated with HRD. In fact, all HRD tumors (n = 12) were microsatellite stable (MSS). Other frequent alterations in mCRC were studied and
 BRAF
 mutations were found to be present in 42% of HRD tumors (p < 0.01). Inversely, HRD was rare in
 KRAS
 -mutated samples (0.7%; 1/140) and, in fact, highly correlated with non-HRD status (p = 0.02), especially the G12 mutation (p < 0.01). Interestingly, CNA profiles also revealed a strong association between the
 BCL2L1
 loci gain and HRD (p < 0.01). Clinically, HRD was not significantly associated with prognostic value nor clinical benefit to oxaliplatin-based combinations. However, the limited sample size and heterogeneous treatment lines restricted robust statistical analysis.
 Conclusions:
 This study identified a small, yet significant subset of mCRC that displays HRD. sHR and HRD rates were higher in metastatic lesions vs primary tumors, indicating HR scarring could be accumulating over time in some mCRC patients. No clear association between pathogenic HR gene mutations and HRD were found, suggesting the involvement of alternative molecular mechanisms in this process. Frequent co-occurring events, such as
 BRAF
 mutations or
 BCL2L1
 gains could be drivers in CRC HRD, and shape, eventually, new therapeutic options for these patients in the metastatic setting.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14cea3ee627505991df2a2a9cb9438dafa79fb0" target='_blank'>
              Drivers of homologous recombination deficiency (HRD) in metastatic colorectal cancer (mCRC).
              </a>
            </td>
          <td>
            Paula Romero Lozano, Javier Ros Montañá, Eduardo García-Galea, Maria Vila-Casadesús, R. Comas, Ágatha Martín, Jenifer González-Zorelle, Paolo Nuciforo, H. G. Pálmer, J. Tabernero, E. Élez, Ana Vivancos-Prellezo
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca9aae7f279fb50f5b49767837c2e747991818a1" target='_blank'>
              TET3 facilitates bladder cancer progression through targeted modulation of stemness pathways.
              </a>
            </td>
          <td>
            Zhiren Cai, Yanqi Xie, Luyao Li, Yuhang Zheng, Xianghui Zhou, Guanghua Zhou, Ganping Wang, Xin Zeng
          </td>
          <td>2025-06-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Disorders of sex development (DSDs) are characterized by discrepancies among karyotype, the gonadal phenotype and gonadal anatomy following birth. Among these, 46, XY DSD is the most complex sub-type and a major cause of birth defects associated with sexual development. However, due to the considerable heterogeneity in pathogenic genes, numerous cases remain genetically undiagnosed. In the present study, a novel gain-of-function variant was identified in the mitogen-activated protein 3 kinase 1 (MAP3K1) gene, contributing to 46, XY DSD through the induction of abnormal cell apoptosis. Genetic analysis of a pediatric male patient and his family revealed a heterozygous MAP3K1 c.4445 G>A variant, resulting in an arginine-to-glutamine substitution. The variant site is highly conserved across species and the amino acid transition induced structural changes in the MAP3K1 protein. In vitro experiments demonstrated that the MAP3K1 c.4445 G>A variant markedly increased apoptotic signaling, leading to abnormal cell apoptosis, disruption of the cell cycle and reduced cell viability. Moreover, the variant exhibited increased levels of ERK1/2 and p38 phosphorylation, indicative of a gain-of-function effect. Subsequent analysis revealed increased expression of the testis-determining gene, SOX9 and reduced expression of the ovary-determining gene, FOXL2. Notably, alterations in gene expression were associated with the MAP3K1 c.4445 G>A variant, providing a mechanistic basis for the pathogenesis of 46, XY DSD. Collectively, these findings offer novel insights into the anti-apoptotic role of MAP3K1, advancing genetic diagnosis and pre-natal screening for individuals with DSDs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2799a19a1fd170b1fea72ad9ddd73906ece00960" target='_blank'>
              A novel variant in the MAP3K1 genomic locus reveals abnormal cell apoptosis as a potential pathogenic mechanism in 46, XY disorders of sex development
              </a>
            </td>
          <td>
            Yufu Lu, Sijia Wei, Shuang Wang, Jingzhi Zhang, Yongjie Xu, Changyudong Huang, Wei Pan, Zhengrong Wang
          </td>
          <td>2025-06-05</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Human endogenous retroviruses (HERVs) comprise genetic material from germ cells that were infected with ancient retroviruses during Pan troglodytes' evolutionary divergence. These "non-coding" repetitive elements now make up approximately 8% of the human genome. Once inserted in the genome, HERVs increase copy number via retrotransposition (a 'copy-and-paste' mechanism) and are involved key functions including transcriptional control and cellular fusion during placenta formation. Aberrant re-expression of HERV-associated genes initially silenced by host epigenetic mechanisms, including DNA methylation, has been linked to many diseases including cancer. Furthermore, in addition to regulating gene expression, including of neighboring genes, reversal of silencing of HERVs may also impact diverse biological functions through the LTRs, and gag, pol, and env proteins. Although previous studies, specifically in white populations, have documented the expression of HERV-K in breast cancer, there is a paucity of data on the expression and role of HERV-R in triple-negative breast cancer (TNBC) among rural West African women from Nigeria. Breast cancer remains the most frequently diagnosed cancer among women globally, with the highest mortality rates across sub-Saharan African nations compared with other countries. Advanced-stage disease at presentation, as well as limited access to treatment options, typically contribute to the poor overall survival rate of breast cancer in African countries including Nigeria. However, the incidence and mortality rates of TNBC in women with West African ancestry are alarmingly high, with an overall survival rate of three years in Zaria, Nigeria. To better understand the molecular basis underlying these high incidence and mortality rates, as well as identify prognostic biomarkers we assessed expression patterns of HERV-R and TNBC-associated genes from Formalin-Fixed Paraffin-Embedded (FFPE) samples of breast cancer, focusing on TNBC in Nigerian women from rural West Africa. To examine gene expression in this study, we used immunohistochemistry, immunoblot, and immunofluorescence, with results showing strong expression of the HERV-R env protein in all but one TNBC patient, as well as in other basal breast cancer subtypes. Our results also strongly suggest that HERV-R expression is associated with overexpression of oncogenes, including TMPRSS2, BRAF, CCND1, BAZ1B, KTM2D, and FOXA1. We found that patients with HERV-R expression have a poor prognosis with low survival. These findings imply that activation of HERV-R may play a role in regulating the expression of genes implicated in TNBC and may serve as a prognostic biomarker in breast cancer.
 Citation Format: Faruk Mohammed, Mohammed Daniyan, Ines Hosni, Halimatu Sadiya Musa, Sani Kamarudeen Owolabi, Adoke Kasimu Umar, Yawale Iliyasu, Rebecca Garnham, Jane Carr-Wilkinson, Sani Ibrahim, Kevin Petrie. Activation of HERV-R is a biomarker of poor prognosis among rural Nigerian women with triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-09-18.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50d98da126d396eab075d52af30620900c11d04d" target='_blank'>
              Abstract P1-09-18: Activation of HERV-R is a biomarker of poor prognosis among rural Nigerian women with triple-negative breast cancer
              </a>
            </td>
          <td>
            Faruk Mohammed, Mohammed Daniyan, Ines Hosni, H. S. Musa, S. K. Owolabi, A. Umar, Y. Iliyasu, Rebecca Garnham, Jane Carr-Wilkinson, Sani Ibrahim, Kevin Petrie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Hypomethylating agents are frontline therapies for myelodysplastic neoplasms (MDS), yet clinical responses remain unpredictable. We conducted a phase 2 trial comparing injectable and oral azacitidine (AZA) administered over one or three weeks per four-week cycle, with the primary objective of investigating whether response is linked to in vivo drug incorporation or DNA hypomethylation. Our findings show that injection results in higher drug incorporation, but lower DNA demethylation per cycle, while global DNA methylation levels in mononuclear cells are comparable between responders and non-responders. However, hematopoietic stem and progenitor cells (HSPCs) from responders exhibit distinct baseline and early treatment-induced CpG methylation changes at regulatory regions linked to tissue patterning, cell migration, and myeloid differentiation. By cycle six—when clinical responses typically emerge—further differential hypomethylation in responder HSPCs suggests marrow adaptation as a driver of improved hematopoiesis. These findings indicate that intrinsic baseline and early drug-induced epigenetic differences in HSPCs may underlie the variable clinical response to AZA in MDS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddff8b2603da5617fe3335cc02a0fb84c36a91b1" target='_blank'>
              Clinical response to azacitidine in MDS is associated with distinct DNA methylation changes in HSPCs
              </a>
            </td>
          <td>
            J. Thoms, Feng Yan, Henry R Hampton, S. Davidson, Swapna Joshi, J. Saw, C. Sarowar, Xin Ying Lim, Andrea C. Nunez, P. Kakadia, G. S. Bhuyan, Xiaoheng Zou, Mary Nguyen, E. Ghodousi, Forrest C Koch, Fatemeh Vafaee, I. R. Thompson, Mohammad M Karimi, Russell Pickford, Mark J. Raftery, Sally Hough, Griselda Buckland, Michelle Bailey, Yuvaraj Ghodke, N. Absar, Lachlin S. Vaughan, Leonardo Pasalic, Chun Y Fong, M. Kenealy, D. Hiwase, Rohanna Stoddart, S. Mohammed, Linda Lee, F. Passam, S. Larsen, Kevin J Spring, Kristen K Skarratt, Patricia Rebeiro, P. Presgrave, William S. Stevenson, Silvia Ling, Campbell Tiley, Stephen J Fuller, F. Roncolato, A. Enjeti, D. Hoenemann, C. Lemech, Chris J. Jolly, Stefan K. Bohlander, David J Curtis, J. W. H. Wong, Ashwin Unnikrishnan, M. Hertzberg, Jake Olivier, Mark Polizzotto, J. Pimanda
          </td>
          <td>2025-05-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Transcription factors play a central role in cancer growth, progression, and metastasis, and contribute to intratumor phenotypic plasticity which enable drug tolerance and cancer relapse. Changes in the regulatory activities of transcription factors in cancer may not always be detected by investigating mutational signatures or differential expression of the transcription factor, as done in traditional analysis. In addition, past studies have focused on the activities of transcription factors in tumors as a whole and thus, have not fully captured the heterogeneity in gene regulation among different cell types within the tumor microenvironment. In this work, through an analysis of the transitions in regulatory network architecture and gene regulation dynamics, we identify the central transcription factors associated with lung adenocarcinoma progression. The gene NR2F1, associated with neurodevelopment and cancer dormancy, emerge as a key transcription factor in the progression of lung adenocarcinoma. We further identify transcription factors that are only active in cancer samples and uncover how changes in gene regulation dynamics influence intratumor heterogeneity. Taken together, our work elucidates the transitions in gene regulatory network during cancer progression, identifies central transcription factors in this process, and reveals the complex regulatory changes cooccurring in different cell types within the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a902ef3e0da1388bf4b1861117a26aab25af6066" target='_blank'>
              Gene regulatory network transitions reveal the central transcription factors in lung adenocarcinoma progression
              </a>
            </td>
          <td>
            Upasana Ray, Adarsh Singh, Debabrata Samanta, Riddhiman Dhar
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The IgLON family of tumor suppressor genes (TSG) impact a variety of cellular processes involved in cancer and non-cancer biology. OPCML is a member of this family and its inactivation is an important control point in oncogenesis and tumor growth. Here, we analyze RNA-Seq expression ratios in ovarian cancers from The Cancer Genome Atlas (TCGA) (189 subjects at Stage III) to identify genes that exhibit a cooperative survival impact (via Kaplan–Meier survival curves) with OPCML expression. Using enrichment analyses, we reconstruct functional pathway impacts revealing interactions of OPCML, and then validate these in independent cohorts of ovarian cancer. These results emphasize the role of OPCML’s regulation of receptor tyrosine kinase (RTK) signaling pathways (PI3K/AKT and MEK/ERK) while identifying three new potential RTK transcriptomic linkages to KIT, TEK, and ROS1 in ovarian cancer. We show that other known extracellular signaling receptor ligands are also transcriptionally linked to OPCML. Several key genes were validated in GEO datasets, including KIT and TEK. Considering the range of OPCML impacts evident in our analyses on both external membrane interactions and cytosolic signal transduction, we expand the understanding of OPCML’s broad cellular influences, demonstrating a multi-functional, pleiotropic, tumor suppressor, in keeping with prior published studies of OPCML function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95469dd0a784b5b2812df87071418bdff5bea009" target='_blank'>
              Bioinformatic RNA-Seq Functional Profiling of the Tumor Suppressor Gene OPCML in Ovarian Cancers: The Multifunctional, Pleiotropic Impacts of Having Three Ig Domains
              </a>
            </td>
          <td>
            Adam G. Marsh, Franziska Görtler, S. Hafizi, Hani Gabra
          </td>
          <td>2025-05-29</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Extensively studied over the past four decades, the TP53 gene has emerged as a pivotal watchman in cellular defense and a key factor in cancer biology. TP53 is the most frequently mutated gene in human malignancies, 50% of which carry alterations to it. Initially, the functions of p53 were thought to be restricted to cell-cycle arrest and apoptosis. With time, however, a growing number of functions have been discovered, illustrating p53's role as a master switch between any cellular stress and cellular or multicellular responses that contribute to its anti-tumor activity. Indeed, the peculiar landscape of TP53 mutations and its high heterogeneity are linked both to the structure of the protein and its ubiquitous function in regulating cellular homeostasis. Mutations in p53 are associated with poor response to therapy and shorter survival in most cancer types, and the diagnosis of p53 mutations is currently used to improve case management in some types of leukemia and lymphoma. Although TP53 has been defined as a tumor suppressor gene, overexpressed mutated p53 variants found in human tumors are defined as dominant oncogenes with a potential gain of function, which makes the gene a very attractive target for developing new cancer treatments. Beyond its role in cancer, our review also highlights TP53's significance in non-neoplastic conditions, such as bone marrow failure syndromes and certain developmental disorders, where chronic p53 activation plays a crucial role in cellular stress responses, demonstrating its broader biological importance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f422279b37d3fc7b3177be1ae5e2e34940dcc8f7" target='_blank'>
              The TP53 tumor suppressor gene: From molecular biology to clinical investigations.
              </a>
            </td>
          <td>
            Panagotis Baliakas, Thierry Soussi
          </td>
          <td>2025-06-16</td>
          <td>Journal of internal medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 e14558


 Background:
 DNA polymerases ε (
 POLE
 ) and δ1 (
 POLD1
 ) encode enzymes involved in DNA replication and proofreading. Somatic
 POLE
 /
 D1
 mutations occur in 1-3% of cancers. Loss-of-function alterations typically reside within the
 POLE
 /
 D1
 exonuclease domain and can result in high tumor mutation burden (TMB). Ultrahigh TMB (≥100 mut/Mb) is often associated with
 POLE/D1
 mutations, and results in near-universal responses to immune checkpoint inhibitors (ICI). Deciphering the contribution of
 POLE
 /
 D1
 mutations to TMB can be challenging, as these often co-occur with microsatellite instability (MSI-H) which also drives TMB.
 Methods:
 Samples were analyzed by NGS (NextSeq or NovaSeq) (Caris Life Sciences, Phx, AZ). TMB (totaled somatic mutations found per megabase pair) was evaluated in patients with
 POLE
 /
 POLD1
 /
 D2
 /
 D3
 /
 D4
 -altered cancers. MSI-H or microsatellite stable (MSS) status was determined by IHC and NGS. Cohorts were divided into pathogenic and likely pathogenic variants (i.e. exonuclease domain mutations) versus VUS alterations (i.e. variants outside the exonuclease domain). Differences in TMB between cohorts were assessed using Fisher’s exact test.
 Results:
 Pan-cancer, TMB ≥20 / ≥50 was identified in 10.9% / 3.4% of
 POLE
 VUS MSS cancers compared to 3.1% / 0.4% of MSS cancers with
 POLE
 /
 D1
 WT status (
 p
 < 0.0001). 0.94% of
 POLE
 VUS cancers had ultrahigh TMB versus 0.05% of MSS
 POLE
 /
 D1
 WT cancers (
 p
 < 0.0001). Similar rates were observed in MSS cancers with
 POLD1
 VUSs (Table). MSS cancers with pathogenic
 POLE
 variants exhibited variable rates of ultrahigh TMB depending on cancer type, ranging from 52.6% (ovarian) to 62.9% (endometrial) to 80.0% (colorectal). Notably, ultrahigh TMB occurred in 0.53%, 0.44%, and 0.44% of MSS patients with
 POLD2
 ,
 POLD3
 , and
 POLD4
 mutations (
 p
 < 0.0001), after excluding coexisting
 POLE
 /
 D1
 mutations.
 Conclusions:
 We propose that certain non-exonuclease
 POLE
 /
 D1
 mutations can result in loss-of-function and ultrahigh TMB in MSS patients. Further, tumor type appears to influence the impact of established
 POLE
 /
 D1
 driver mutations on TMB. Finally,
 POLD2
 -
 4
 mutations (not previously studied) may also contribute to genome instability and mutational load. These findings argue for deeper characterization of DNA polymerase mutations to identify patients that may benefit from ICI.






 POLE
 /
 D1
 WT + MSS[n (%)]


 POLE
 VUS Mutation + MSS[n (%)]


 POLE
 Pathogenic Mutation + MSS[n (%)]


 POLD1
 VUS Mutation + MSS[n (%)]


 POLD1
 Pathogenic Mutation + MSS[n (%)]





 Total
 139,842
 5,658
 557
 1,793
 11


 TMB ≥20
 4,378(3.1%)[ref]

 614(10.9%)
 p
 <0.0001


 529(95.0%)
 p
 <0.0001


 168(9.4%)
 p
 <0.0001


 5(45.5%)
 p
 <0.0001



 TMB ≥50
 586(0.42%)[ref]

 193(3.4%)
 p
 <0.0001


 467(83.8%)
 p
 <0.0001


 79(4.4%)
 p
 <0.0001


 2(18.2%)
 p
 =0.0009



 TMB ≥100
 73(0.05%)[ref]

 53(0.94%)
 p
 <0.0001


 343(61.6%)
 p
 <0.0001


 28(1.6%)
 p
 <0.0001


 1(9.1%)
 p
 =0.0058







 Evaluated by Fisher’s exact test comparing to respective POLE/D1 WT + MSS cohort.




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eb38b846148e42ca1249ce42b9e558a15a34574" target='_blank'>
              Potential impact of non-exonuclease
 POLE
 and
 POLD1
 mutations on high and ultrahigh TMB across cancers.
              </a>
            </td>
          <td>
            Alexander K. Tsai, S. Deshmukh, Sharon Wu, Dave S. B. Hoon, W. El-Deiry, Axel Grothey, Justin H Hwang, E. Antonarakis
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90517c7621a56cfd44f51891279305813cf3e121" target='_blank'>
              Notch3 regulates pericyte phenotypic plasticity in colorectal cancer
              </a>
            </td>
          <td>
            N. Chalkidi, Athanasia Stavropoulou, V. Arvaniti, Christina Paraskeva, Artemis Monogyiou, Maria Sakkou, Christoforos Nikolaou, V. Koliaraki
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="ABSTRACT Genomic imprinting is the parent-of-origin specific monoallelic expression of genes that result from complex epigenetic interactions. It is often achieved by monoallelic 5-methylcytosine, resulting in the formation of differentially methylated regions (DMRs). These show a bias towards oocyte-derived methylation and survive reprogramming in the pre-implantation embryo. Imprinting is widespread in the human placenta. We have recently performed whole-genome screens for novel imprinted placenta-specific germline DMRs (gDMRs) by comparing methylomes of gametes, blastocysts and various somatic tissues, including placenta. We observe that, unlike conventional imprinting, for which methylation at gDMRs is observed in all tissues, placenta-specific imprinting is associated with transient gDMRs, present only in the pre-implantation embryo and extra-embryonic lineages. To expand the list of bona fide imprinted genes subject to placenta-specific imprinting, we reinvestigated our list of candidate loci and characterized two novel imprinted genes, PIK3R1 and G0S2, both of which display polymorphic imprinting. Interrogation of placenta single-cell RNA-seq datasets, as well as cell-type methylation profiles, revealed complex cell-type specificity. We further interrogated their methylation and expression in placental samples from complicated pregnancies, but failed to identify differences between intrauterine growth restricted or pre-eclamptic samples and controls, suggesting they are not involved in these conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b750b53f10506274107b5e1972c3a2fd8cd37694" target='_blank'>
              PIK3R1 and G0S2 are human placenta-specific imprinted genes associated with germline-inherited maternal DNA methylation
              </a>
            </td>
          <td>
            Dagne Daskeviciute, Becky Sainty, Louise Chappell-Maor, Caitlin Bone, Sarah Russell, Isabel Iglesias-Platas, Philippe Arnaud, Ana Monteagudo-Sánchez, Maxim V. C. Greenberg, Keran Chen, Africa Manerao-Azua, G. Perez de Nanclares, Jon Lartey, David Monk
          </td>
          <td>2025-06-26</td>
          <td>Epigenetics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22bcb6bbbf4e6c64ab8d5a6fa755b5306451c07b" target='_blank'>
              EHMT1 mediates cellular motility in embryonal rhabdomyosarcoma by activating SOX8 expression.
              </a>
            </td>
          <td>
            Upasana Bajaj, Dipanwita Das, Jia Yu Leung, Nurul Fateha Binti Abdul Rashi, R. Taneja
          </td>
          <td>2025-06-04</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Ewing sarcoma (EwS) is an aggressive cancer of bone and soft tissue that predominantly affects children and young adults. A chromosomal translocation joins the low-complexity domain (LCD) of the RNA-binding protein EWS (EWSLCD) with the DNA-binding domain of Friend leukemia integration 1 (FLI1DBD), creating EWS::FLI1, a potent fusion oncoprotein essential for EwS development and responsible for over 85% of EwS tumors. EWS::FLI1 forms biomolecular condensates in vivo and promotes tumorigenesis through mediation of aberrant transcriptional changes and by interfering with the normal functions of nucleic acid-binding proteins like EWS through a dominant-negative mechanism. In particular, the expression of EWS::FLI1 in EwS directly interferes with the biological functions of EWS leading to alternate splicing events and defects in DNA-damage repair pathways. Though the EWSLCD is capable of phase separation, here we report a direct interaction between FLI1DBD and EWSLCD that enhances condensate formation and alters the physical properties of the condensate. This effect was conserved for three related E-twenty-six transformation-specific (ETS) DNA-binding domains (DBDs) while DNA binding blocked the interaction with EWSLCD and inhibited EWS::FLI1 condensate formation. NMR spectroscopy and mutagenesis studies confirmed that ETS DBDs transiently interact with EWSLCD via the ETS DBDs "wings." Together these results revealed that ETS DBDs, particularly FLI1DBD, enhance EWSLCD condensate formation and rigidity, supporting a model in which electrostatic and structural interactions drive condensate dynamics with implications for EWS::FLI1-mediated transcriptional regulation in EwS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c90380493c2e0c6b1757985f2684e59828098025" target='_blank'>
              Phase separation of the oncogenic fusion protein EWS::FLI1 is modulated by its DNA-binding domain.
              </a>
            </td>
          <td>
            Emily E. Selig, Erich J Sohn, Aiola P. Stoja, Alma K. Moreno-Romero, Shivani Akula, Xiaoping Xu, Alexander J R Bishop, David S. Libich
          </td>
          <td>2025-05-16</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Karyotyping, SNP arrays, fluorescence in situ hybridisation and next generation sequencing techniques have greatly improved our understanding of the genetic drivers of acute lymphoblastic leukemia (ALL). In the past years, another layer of genetic data has been added by the study of mutational signatures, patterns of somatic mutations that represent specific mutational mechanisms. Mutational signatures can give insight in tumor development, but also reveal mutagenic side-effects of treatment in relapse samples. Multiple treatment-related mutational signatures have been detected in relapsed ALL that could play a role in therapy resistance and relapse development. In fact, multiple pathogenic driver mutations have been attributed to these treatment-related mutational processes, including a recurrent TP53 mutation. Studies in childhood ALL revealed that thiopurine exposure is the most common source of therapy-related mutagenicity in ALL and presents differently when patients are DNA mismatch repair deficient. Thiopurine-induced DNA damage indicates that leukemic cells were able to survive thiopurine exposure. This could be due to metabolic defects, acquired mutations that induce thiopurine resistance during treatment or resistance to drugs synergizing with thiopurines. In this review, we discuss the types and prevalence of treatment-related mutational signatures in ALL, and explore mechanisms of thiopurine cytotoxicity and mutagenicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88f4deac881b3b7a4007820d8b07d9cc82f4007b" target='_blank'>
              Treatment-related mutagenic processes in acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Cédric G van der Ham, F. V. van Leeuwen, Roland P Kuiper
          </td>
          <td>2025-05-15</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Merkel Cell Polyomavirus (MCPyV) is recognized as the major aetiological agent of Merkel Cell Carcinoma (MCC), an aggressive skin tumor. MCPyV-mediated oncogenesis is strictly dependent on viral integration and the expression of a truncated form of the Large T Antigen (LT). Moreover, like other oncogenic DNA viruses, MCPyV may interfere with the DNA damage response (DDR) machinery, thus promoting genomic instability and tumorigenesis. Therefore, the objective of this study was to characterize MCPyV infection in 7 MCC patients and to elucidate the plausible role of the virus in the DDR pathway. MCPyV DNA was detected in 3/7 MCC patients and, as expected, viral integration and LT truncation were observed in virus-positive MCCs, along with the expression of early genes only. Over-expression of DDR genes such as ATM, ATR and their downstream kinases Chk1 and Chk2 was reported in MCPyV-positive MCCs supporting the potential role of the virus in interfering with DDR. Our findings support the established viral aetiology of MCC, and describe, for the first time, an over-expression of DDR components in MCPyV-positive MCC, laying the basis for future studies aimed at investigating the contribution of this pathway to MCPyV-mediated carcinogenesis and exploring the plausible clinical implications of host DDR factors for the treatment of MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cee01380ef93ba0651f961c5653ef76df058128" target='_blank'>
              Investigation of mRNA expression levels of DNA damage response genes in Merkel Cell Polyomavirus-positive Merkel Cell Carcinoma: a pilot study
              </a>
            </td>
          <td>
            S. Passerini, M. Fracella, A. Ferlosio, U. Moens, C. Scagnolari, Guido Antonelli, Marco Ciotti, V. Pietropaolo
          </td>
          <td>2025-05-21</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="The pituitary tumor transforming gene 1 (PTTG1), discovered in 1997 by Pei and Melmed, takes part in cellular replication, cell cycle control, DNA repair mechanisms, organogenesis, metabolism regulation, cellular transformation, and senescence. Its biological actions include protein-protein interactions, modulation of gene transcription, and, other than intracellular and autocrine mechanisms, even paracrine activities. For the reasons mentioned above, PTTG1 stands out as a multifaceted regulator of cancer biology; it is involved in genomic and chromosomal instability, local invasiveness, neo-lymphangiogenesis, and metastatic spreading. In solid neoplasms, endocrine neoplasms, although deemed rare, have experienced a significant increase in diagnostic incidence in recent years. Endocrine cancers are still a major challenge in healthcare and research since several questions remain unanswered, even though researchers have made considerable efforts to uncover their causes. Twenty-seven years have passed since PTTG1's discovery, and several works have been published. However, only the tip of the iceberg has been unveiled. Herein, we review current knowledge of PTTG1's action in endocrine cancers, such as pituitary, thyroid, testicular, adrenal, pancreatic, and ovarian.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a57eb69f3187a01c5af405bf459aed1d53f3b1f4" target='_blank'>
              Pituitary tumor transforming gene 1 and endocrine cancers: an up-to-date review through history, current insights and future perspectives.
              </a>
            </td>
          <td>
            E. Vergani, Emanuela Teveroni, Francesca Mancini, F. Di Nicuolo, S. Raia, S. Chiloiro, F. Pierconti, Antonio Bianchi, A. M. Isidori, Alfredo Pontecorvi, Domenico Milardi
          </td>
          <td>2025-06-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6116cf0972c0f5ba1621a1614ba9f7a9a3add325" target='_blank'>
              The SOX gene superfamily in oncogenesis: unraveling links to ncRNAs, key pathways, chemoresistance, and gene editing approaches.
              </a>
            </td>
          <td>
            Ammar Yasir Ahmed, Subasini Uthirapathy, Enwa Felix Oghenemaro, R. M, Rohit Kumawat, Yasser Fakri Mustafa, Muthena Kariem, A. Kadhim, Shilpa Sharma, M. R. Kumar
          </td>
          <td>2025-05-20</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="BACKGROUND AIMS
Intrahepatic cholangiocarcinomas (ICCs) are aggressive liver tumors with high heterogeneity and limited therapeutic options. Although traditionally thought to arise from biliary cells, recent findings suggest that hepatocytes may also serve as a cellular origin for ICC. However, the mechanisms underlying hepatocyte malignant transformation and ICC initiation remain poorly understood.


APPROACH AND RESULTS
We employed oncogene-driven and chemically induced ICC murine models, along with cellular models, to recapitulate the transformation of hepatocytes into ICC. Our findings demonstrate that mature hepatocytes undergo a significant hyperpolyploid state during ICC initiation. Hyperpolyploidy promotes aberrant cell division and chromosomal instability, accelerating hepatocyte transformation and ICC onset. Furthermore, we identified the chromatin remodeling factor Arid1a as a critical suppressor of hyperpolyploidy. Arid1a deficiency disrupts mitotic machinery at the centrosome, driving hyperpolyploidization and ICC tumorigenesis.


CONCLUSIONS
Hepatocytes can transform into ICC through a process involving hyperpolyploidizaiton. This study offers new insights into the pathogenesis of ICC, particularly in patients harboring frequent ARID1A mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb35330c9db09a4f3ded3d2167db23a4e30f36f2" target='_blank'>
              Arid1a deficiency promotes hepatocyte hyperpolyploidy and drives intrahepatic cholangiocarcinoma in mice.
              </a>
            </td>
          <td>
            Qi Bian, Shu Wang, Zimin Song, Fang Liu, Muqing Cao, Nan Yang, Jun Ying, Jia-Syuan Hu, Xinyuan Xiong, Huiqin Zhu, Jun Wu, Jie Yang, Xiaonan Wang, Shunli Shen, Xuxu Sun
          </td>
          <td>2025-06-09</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aim: Chemotherapy resistance, often linked to the development of resistance against genotoxic agents, is a major obstacle in cancer treatment. FoxM1, a transcription factor frequently overexpressed in malignancies such as breast cancer, is strongly associated with genotoxic therapy resistance. The aim of this study is to conduct a comparative analysis of the effects of thiostrepton (THIO), a FoxM1 inhibitor, on the DNA damage response in HUVEC cells (non-malignant) and MDA-MB-231 breast cancer cells (malignant) 
Materials and Methods: THIO's impact on cell viability were evaluated in both cell lines using the MTT assay. Oxidative DNA damage levels were measured with the 8-OHdG kit, and apoptosis was assessed using the Caspase 3 ELISA kit. The expression levels of DNA damage response genes (BRCA-1, DNAPKC, FOXM1, RAD51, MRE11 and XRCC1) were analyzed by RT-PCR. 
Results: MDA-MB-231 cells exhibited greater sensitivity to the cytotoxic effects of THIO than HUVEC cells. In HUVEC cells, THIO caused a significant increase in oxidative DNA damage, whereas no such effect was observed in MDA-MB-231 cell lines. Conversely, breast cancer cells showed a significant increase in Caspase 3 levels. RT-PCR results revealed a marked downregulation of DNA damage response genes, particularly BRCA-1, DNAPKC, MRE11, FOXM1, and XRCC1, in both cell types. 
Conclusion: THIO downregulates DNA damage response genes by inhibiting FoxM1 expression in both malignant and non-malignant cells. This effect suggests that THIO may enhance the sensitivity of breast cancer cells to therapy by disrupting DNA repair pathways, presenting potential for overcoming chemotherapy resistance. Further studies are required to validate these findings and explore THIO's therapeutic utility in breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0302ae4a15025cfb10555437aa0808dee744207" target='_blank'>
              FoxM1 inhibition by Thiostrepton downregulates DNA damage response genes, enhancing sensitivity of breast cancer cells to therapy
              </a>
            </td>
          <td>
            F. Demırtaş Korkmaz, Zekeriya Düzgün, Fadime Mutlu İçduygu
          </td>
          <td>2025-06-10</td>
          <td>Ege Tıp Dergisi</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/521330e6a0acc4c997107eda98c17326ccf4fe70" target='_blank'>
              Concurrent loss of the Y chromosome in cancer and T cells impacts outcome.
              </a>
            </td>
          <td>
            Xingyu Chen, Yiling Shen, Suhyeon Choi, Hany A Abdel-Hafiz, Mukta Basu, Lena Hoelzen, Martina Tufano, Saravana Kumar Kailasam Mani, Maryam Ranjpour, Jiani Zhu, V. K. Ramanujan, E. Koltsova, Vinicius F Calsavara, S. Knott, Dan Theodorescu
          </td>
          <td>2025-06-04</td>
          <td>Nature</td>
          <td>2</td>
          <td>6</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a diverse collection of malignant tumors that originate in the bile ducts. Mitochondria, the energy converters in eukaryotic cells, contain circular mitochondrial DNA (mtDNA) which has a greater mutation rate than nuclear DNA. Heteroplasmic variations in mtDNA may suggest an increased risk of cancer-related mortality, serving as a potential prognostic marker. In this study, we investigated the mtDNA variations of five CCA cell lines, including KKU-023, KKU-055, KKU-100, KKU213A, and KKU-452 and compared them to the non-tumor cholangiocyte MMNK-1 cell line. We used Oxford Nanopore Technologies (ONT), a long-read sequencing technology capable of synthesizing the whole mitochondrial genome, which facilitates enhanced identification of complicated rearrangements in mitogenomics. The analysis revealed a high frequency of SNVs and INDELs, particularly in the D-loop, MT-RNR2, MT-CO1, MT-ND4, and MT-ND5 genes. Significant mutations were detected in all CCA cell lines, with particularly notable non-synonymous SNVs such as m.8462T > C in KKU-023, m.9493G > A in KKU-055, m.9172C > A in KKU-100, m.15024G > C in KKU-213A, m.12994G > A in KKU-452, and m.13406G > A in MMNK-1, which demonstrated high pathogenicity scores. The presence of these mutations suggests the potential for mitochondrial dysfunction and CCA progression. Analysis of mtDNA structural variants (SV) revealed significant variability among the cell lines. We identified 208 SVs in KKU-023, 185 SVs in KKU-055, 231 SVs in KKU-100, 69 SVs in KKU-213A, 172 SVs in KKU-452, and 217 SVs in MMNK-1. These SVs included deletions, duplications, and inversions, with the highest variability observed in KKU-100 and the lowest in KKU-213A. Our results underscore the diverse mtDNA mutation landscape in CCA cell lines, highlighting the potential impact of these mutations on mitochondrial function and CCA cell line progression. Future research is required to investigate the functional impacts of these variants, their interactions with nuclear DNA in CCA, and their potential as targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92494ba8290eb1c5fd7c0d865dc1a82ce3683d1" target='_blank'>
              Mitochondrial genomic alterations in cholangiocarcinoma cell lines.
              </a>
            </td>
          <td>
            Athitaya Faipan, Sirinya Sitthirak, Arporn Wangwiwatsin, N. Namwat, Poramate Klanrit, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Luke Boulter, Hasaya Dokduang, W. Loilome
          </td>
          <td>2025-06-09</td>
          <td>PloS one</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="
 e16379


 Background:
 Pancreatic ductal adenocarcinoma (PDAC) is one of the most aggressive forms of malignancy. Although tissue genomics has been valuable in understanding driver mutations in PDAC, it is limited to impart subclonal mutations. ctDNA-based liquid biopsy represents a wider mutational profile, accounts for mutational heterogeneity, and better accessibility. Here we compared comprehensive mutational profiling (CGP) of advanced PDAC patients from tissue and blood samples to evaluate its relation to the disease stage.
 Methods:
 In a retrospective study, 145 Pts with advanced PDAC and on SOC underwent targeted hybrid-capture-based CGP using an OncoIndx CGP comprising 1080 genes on tumor tissue or plasma ctDNA. Enriched libraries were sequenced at an average depth of 5500 X on Illumina Nextseq 2000 platform. Raw sequenced reads were processed for variant calling using a proprietary iCare. Variants were called according to ClinVar, ACMG, and AMP databases.
 Results:
 Out of 145 patients sequenced, 104 showed the presence of at least one pathogenic mutation. 46 % (48/104) of samples were blood, 34 % (36/104) were tissue and 19 % (20/104) were paired. We observed 90 % concordance between Lbx and Tbx for KRAS and TP53 variants. TP53 mutation was the most frequent, occurring in 70 % pts with equivalent distribution in tissue and blood. Nearly 55 % TP53 variants were gain of function (GOF) while 45 % were loss of function. KRAS was the 2
 nd
 most frequent mutation (66 % Pts) with the codon 12 variant being the most frequent (57.7 %), and showing comparable distribution between KRAS:p.G12D (55 %) and KRAS:p.G12V (45%). Rare variants including codon 61 (KRAS:p.Q61H) and 146 (KRAS:p.A59T) were detected in 5 pts, while N and HRAS variants were combined present in 14 % pts, and were mutually exclusive with KRAS (P = 0.043). TP53 and KRAS variants showed high mutual co-occurrence (Fisher test; P = 0.0042, OR = 2.1). KRAS:p. G12D and TP53 GOF variants strongly correlated with aggressive metastasis (Speareman Rho = 0.8, P = 0.037), and were detected primarily in Lbx. Additionally, Lbx revealed the presence of HRR mutations (BRCA1, BRIP1, and RAD54L) in 9.6 % pts. 21 % Pts showed presence of amplified ERBB2, while 24 % showed either CCND1 or CCND2 amplification. When present, co-occurring KRAS, SMAD4 and TP53 mutations were associated with distant metastasis (P = 0.04).
 Conclusions:
 The genomic profile of advanced PDAC patients revealed a complex mutational landscape. Notably, ctDNA exhibited greater tumor mutational heterogeneity compared to tissue DNA samples and detected additional actionable pathogenic variants. Co-occurring KRAS and TP53 GOF variants and the presence of LOF SMAD4 variants along with ERBB2 amplification showed a strong association with distant metastasis, suggesting their prognostic value in advanced PDAC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0830e6b6d6a8ee429144bdcfeff5d658f0ba1df6" target='_blank'>
              Association of co-occurring KRAS and gain of function TP53 mutations with aggressive metastasis through comprehensive genomic profiling of pancreatic ductal adenocarcinoma.
              </a>
            </td>
          <td>
            A. Bharde, Mrunal Ratnaparkhi, A. D'Souza, K. Hariramani, Bhagwat Jadhav, Sangeeta Prajapati, Aarthi Ramesh, M. Basavalingegowda, Sandhya Iyer, Shaunak. Joshi, Richa Deshpande, P. Kendre, P. Kalaskar, Shyam Aggarwal, Bharat Bhosale, V. Maniar, Jayant J Khandare, Gowhar Shafi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Gliomas are the most common and lethal malignant primary brain tumors in adults, associated with the highest number of years of potential life lost. The latest WHO classification for central nervous system tumors highlighted the need for new biomarkers of diagnosis, prognosis, and response to therapy. The PI3K/Akt signaling pathway is clearly implicated in tumorigenesis, being one of the most frequently altered pathways in cancer. Activating PI3KCA mutations are oncogenic and can influence both prognosis and treatment response in various tumor types. In gliomas, however, studies have reported inconsistent PIK3CA mutational frequencies, ranging from 0% to 30%. Furthermore, the impact of these alterations on glioma diagnosis, prognosis, and therapy response remains unclear. Current evidence suggests that PIK3CA mutations may represent early and constitutive events in glioma development, associated with worse glioblastoma prognoses, earlier recurrences, and widespread disease. Among these, the hotspot mutation H1047R has been particularly associated with a more aggressive phenotype while also modulating the neuronal microenvironment. In this review, we examine the clinical relevance of PIK3CA mutations across different cancers, with a particular focus on their emerging role in glioma. Moreover, we also discuss the therapeutic potential and challenges of targeting PIK3CA mutations in the context of glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d6d6f1fbe9d06843c9afa611b81f785472f82b9" target='_blank'>
              PIK3CA Mutations: Are They a Relevant Target in Adult Diffuse Gliomas?
              </a>
            </td>
          <td>
            A. Tomás, M. Pojo
          </td>
          <td>2025-05-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cancer initiation occurs when a cell acquires and accumulates mutations in genes involved in the regulation of cell processes: each cell division throughout a person’s life introduces novel mutations in the cells’ DNA and under normal circumstances, the body is primed to prevent those from leading to cancer. Occasionally, a subset of those mutations escapes those safeguards and might eventually result in the emergence of the disease. To understand the dynamics of accumulation of somatic mutations, we have performed longitudinal whole genome sequencing of DNA obtained from whole blood from healthy individuals and cancer patients using Oxford Nanopore Technologies’ Long Read Sequencing. Here we show that the number of somatic single nucleotide variants detected increases with their age and that for specific mutational processes, changes can be detected within months. We computed aggregated metrics for unique participants at each timepoint across types of variants (based on single based substitution molecular signatures) and identified patterns of change both over an individual’s lifespan (age) and over the sampling period (months). This study showcases the suitability of long read sequencing of blood DNA for detecting coarse-grained differences over time and enable future development of “state of the system” personalized prevention programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95c0c1b8ce2a6d46369d034aca04dad0606d256" target='_blank'>
              Monitoring the rate and variability of somatic genomic alterations using long-read sequencing
              </a>
            </td>
          <td>
            Xingyao Chen, Hagai Ligumsky, Charlie Ambrose, Denisse Sibrian, Brian Tran, Daoud Arif, Olga Castellanos, Darren Kessner, Hanyi Luo, Mukta Ubale, Abigail Coleman, Vaidhyanathan Mahaganapathy, Thomas J Jönsson, Reva K Basho, Jerry S H Lee, Naim Matasci, David B. Agus
          </td>
          <td>2025-06-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35fd329abfeae45d934694f31530eb9db169304e" target='_blank'>
              A mouse model of PTEN Hamartoma Tumour Syndrome reveals that loss of the nuclear function of PTEN drives macrocephaly, lymphoid overgrowth, and late-onset cancer
              </a>
            </td>
          <td>
            Priyanka Tibarewal, V. Rathbone, Sarah E Conduit, G. Classen, Fiona M Black, Mohammad Amin Danesh, Georgia Constantinou, Zeinab Asgarian, Koujiro Tohyama, N. Kriplani, Virginia Alvarez Garcia, Elizabeth Foxall, Djenat Belarbi, M. Leverve, Wayne Pearce, Mahreen Adil, Zofia Varyova, Lucia Conde, Adriana Alves, G. Masson, Roger L. Williams, Adrienne M Flanagan, Javier Herrero, Katerina Stroud, M. Tischkowitz, K. Lachlan, Cheryl L Scudamore, Mark G H Scott, Nicholas R. Leslie, N. Kessaris, B. Vanhaesebroeck
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 3627


 Background:
 A subset of colorectal cancer (CRC) features amplification (amp) of the ERBB2 gene. The prognostic role and treatment in these patients are poorly understood. This study aims to characterize the clinical outcomes and molecular profiles of ERBB2-amp CRC.
 Methods:
 This is a retrospective analysis of clinical outcomes data and next-generation sequencing at MD Anderson from 2006-2025 in patients with ERBB2-amp CRC including all classes of genomic alterations (GA) in other genes. Progression-free survival (PFS) and overall survival (OS) were assessed by Kaplan-Meier, log-rank, and cox regression. Chi square goodness of fit was assessed for the proportion of left versus right sided CRC.
 Results:
 100 pts with ERBB2-amp CRC were identified. Patients (Table) who received ERBB2-directed therapy (n = 36) did not have significantly longer OS compared to those who received other systemic therapy (n = 57) (53.8 vs. 45.5 m, p = 0.35). Patients who underwent surgery (n = 64) (56.8 vs. 23.2 m, p = 0.01) or non-surgical localized therapy (n = 45) had longer OS (64.4 vs 45.5 m, p = 0.01). ERBB2-amp was more associated with left sided CRC (n = 80) than right-sided CRC (n = 19) (X
 2
 = 39.34, df = 1, p < 0.0001) but there was no significant OS difference (53.8 vs. 45.5 m, p = 0.76). Median PFS (mPFS) for all 1st (n = 93), 2nd (n = 80), and 3rd line (n = 61) systemic therapy was 8.0, 5.1, and 4.9 m, respectively. mPFS for all ERBB-2 directed therapy (n = 59) was 3.5 m. The most common concurrent co-GA were TP53 (90%) and APC (64%) and most common co-amp were CDK12 (31%) and EGFR (23%). Concurrent APC mutation was associated with improved OS (53.8 vs. 26.2 m, p = 0.003). No other co-GA or co-amp, including KRAS, showed significant survival outcome associations. TP53 co-GA was present in 90% of both left and right-sided ERBB-2 CRC and KRAS co-GA in 15% and 32%, respectively.
 Conclusions:
 There was no difference in OS in ERBB2 amp CRC in patients who received ERBB2 directed therapy, however patients had improved OS with surgery or localized therapy options and should be offered to eligible patients if possible. Co-GA with APC mutation correlated with improvement in survival outcomes.


 Baseline data, initial staging, treatments, and mutational profile for ERBB2 co-amp CRC.




 Age at Diagnosis (median)
 55 years




 Gender – no.
 M (56), F (44)


 Stage at Initial Diagnosis – no.
 I (6), II (6), III (18), IV (69)


 Location of Primary Tumor – no.
 R-sided (19), L-sided (80), Cecum (7), Ascending (8), Transverse (5), Descending (7), Sigmoid/Rectum (73)


 Surgery – no.
 Primary tumor (55), Liver metastectomy (24), Lung metastectomy (8)


 Localized Therapy – no.
 RT (36), Ablation (6), Y90 (4), Cryotherapy (1)


 ERBB2 Directed Therapy (excluded if n=1) – no.
 Trastuzumab (TRA) + Pertuzumab (20), ERBB2-directed trial drug (14), Trastuzumab deruxtecan (8), TRA (7), TRA + Tucatinib (4)


 Mutational Profile
 Co-GA: TP53 (90%), APC (64%), SMAD4 (18%), KRAS (18%)


 Co-amp: CDK12 (31%), EGFR (23%), MYC (19%), BRAF (13%)




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d0f42d0d349a777ee7b1fb85c5f8c10e3a8f7ef" target='_blank'>
              Characterizing clinical outcomes and DNA co-alterations of ERBB2-amplified colorectal cancer.
              </a>
            </td>
          <td>
            Mohamed Nuh, Monica Hsiang, Sunyoung S. Lee
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7668e6be2f122a26f394a92f23a38b2f39f2818" target='_blank'>
              Genome Editing in Gynecological Oncology: The Emerging Role of CRISPR/Cas9 in Precision Cancer Therapy.
              </a>
            </td>
          <td>
            Naina Kumar
          </td>
          <td>2025-05-20</td>
          <td>Therapeutic innovation & regulatory science</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Numerous long noncoding RNAs (lncRNAs) are generated in response to external stimuli, but the scope and functions of such activity are not known. Here, this study provides insight into how the transcription of lncRNAs is connected to DNA damage response by identifying the lncRNA ZFAS1, which is required for cell cycle arrest, transcription regulation, and DNA repair. Mechanistically, ZFAS1 facilitates dynamic changes in hyperphosphorylated forms of the large subunit of RNA polymerase II (RNAPII) around transcription initiation sites by directly targeting the regulated genes. It is shown that extensive transcription shutdown and concomitant stimulated engagement of RNAPII-Ser2P are crucial for repair and cell survival upon genotoxic stress. Finally, ZFAS1 knockout in mice dampened nucleotide excision repair (NER) and led to kidney dysplasia. Overall, the findings extend the understanding of lncRNAs in DNA damage repair (DDR) and imply a protective role of lncRNA against DDR-deficient developmental disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd71da5d91ac1bdf35ea5bcddbb6ad4cb3db5ea" target='_blank'>
              IncRNA-ZFAS1, an Emerging Gate-Keeper in DNA Damage-Dependent Transcriptional Regulation.
              </a>
            </td>
          <td>
            Jiena Liu, Qing Lu, Zixuan Fan, Jiahui Lin, Nan He, Xin Zhang, Zhaoya Han, Tingting Zhu, Zhenzhen Wu, Yingying Xu, Yuming Wang
          </td>
          <td>2025-05-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 e14570


 Background:
 Immune checkpoint blockades (ICBs) showed little efficacy in pediatric solid tumors. The reconfiguration of gene regulation in cancers is a major factor in tumor immune microenvironment (TiME) remodeling and ICB resistance. Hypothesis: A subset of pediatric tumors may exhibit an epigenetically altered TiME, potentially responsible for ICB resistance. Objective: classify the TiME of pediatric extracranial solid tumors and describe the gene regulatory networks (GRNs) of immune landscapes.
 Methods:
 We studied bulk tumor gene expression and DNA-methylation from 184 pediatric extracranial solid tumors. Similarity network fusion (SNFtool) individualized TiME phenotypes by dual-omic clustering of immune genes. The clusters were compared by differential analysis for gene expression and methylation. We studied the relationship of enhancer methylation to gene expression (ELMER package) to infer methylation reprogramming. An enrichment study of regulatory binding regions (ReMapEnrich) identified the master regulators that define the GRNs unique to each phenotype.
 Results:
 SNF clustering identified 3 phenotypes with 52 (28%), 83 (45%), 49 (27%) samples in clusters (cl) 1, 2 and 3, respectively. Cl1 exhibited low expression of immune genes (“cold” phenotype), cl2 overexpressed immune genes (“hot” phenotype), and cl3 featured global hypermethylation (epigenetically “altered” phenotype). Different tumor types were present across all phenotypes, but Ewing sarcoma was more frequent in altered, osteosarcoma and neuroblastoma in hot, and Wilms tumor in cold phenotype (
 p
 <0.05). Both hot and altered phenotypes overexpressed immune checkpoints (
 CD274
 ,
 PDCD1
 ), T-cell activator chemokines (
 CXCL9
 ,
 CXCL10
 ) and pro-inflammatory pathways, central to ICB sensitivity. However, only hot tumors overexpressed genes and pathways essential for antigen-presenting machinery and immune recognition. Methylation regulation was responsible for derepressing MHC-II genes and their regulators in hot tumors. In contrast, gene silencing in altered phenotype relied on hypermethylation/downregulation of the CIITA cofactor domain, the master control of MHC-II genes, and other immune-specific master regulators crucial for anti-tumor immunity and ICB sensitivity. The regulatory landscape of immune gene repression in the altered phenotype involved binding regions specific to the MYC/MAX network and the polycomb-group protein PCR2 (
 i.e.,
 EZH2 and SUZ12).
 Conclusions:
 We demonstrated that GRN reconfiguration participates in TiME reshaping in pediatric extracranial solid tumors. A subset of tumors with epigenetically altered immune phenotype has an immune recognition capacity repressed by MYC/MAX and PCR2 complexes. Future studies should investigate specific inhibitors to reprogram the GRN to foster immune recognition and ICB sensitivity.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b6a52d83f8f6dace4bda8bd4ea08fd3f518537c" target='_blank'>
              Dual-omic analysis to reveal gene regulatory network of immune landscape in childhood solid tumors and implications for immunotherapy.
              </a>
            </td>
          <td>
            Stéphanie Bianco, Anas Belaktib, Virgile Raufaste-cazavieille, Charles Joly-Beauparlant, Mona Patoughi, Thibault Mallevaey, Lara Herrmann, Sylvie Langlois, Thomas Sontag, Alex Richard-St-Hilaire, Noel Raynal, Vincent-Philippe Lavallée, Sonia Cellot, T. Tran, D. Sinnett, Arnaud Droit, Raoul Santiago
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="DNA damage repair (DDR) is essential for cancer cell survival and treatment resistance, making it a critical target for tumor therapy. The eukaryotic AAA+ adenosine triphosphatase valosin-containing protein (VCP), which is transported from the cytoplasm into the nucleus, plays a critical role in the DDR process. However, the nuclear translocation and molecular mechanism of VCP for DDR remain elusive. Here, we define VCP as a KPNB1 interacting protein through a combination of chemical and immunoprecipitation mass spectrometry approaches. Further biochemical studies elucidate that KPNB1 directly transports VCP into the nucleus. We also identify withaferin A (WA) as a small molecule that can retard VCP nuclear localization via covalent binding to CYS 158 of KPNB1. Further studies verify WA as an effective antitumor drug candidate via blocking VCP nuclear localization to impact on the DDR pathway in vivo. Our findings underly the unclear VCP's role in DDR in a KPNB1-dependent manner and provide an important theoretical basis for developing small-molecule inhibitors targeting this process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9a984030c7bf74f111e78a067197177c6ec90ce" target='_blank'>
              VCP's nuclear journey: Initiated by interacting with KPNB1 to repair DNA damage.
              </a>
            </td>
          <td>
            Z. Xing, Xiaoying Cai, Ting He, Peiheng Li, Jun He, Yuxuan Qiu, Na Li, Li Mi, Ruixi Li, Jingqiang Zhu, Zhihui Li, A. Su, Haoyu Ye, Wenshuang Wu
          </td>
          <td>2025-05-08</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="DNA damage is a serious threat to cellular viability, and it is implicated as the major cause of normal ageing. Hence, targeting DNA damage therapeutically may counteract age-related cellular dysfunction and disease, such as neurodegenerative conditions and cancer. Identifying novel DNA repair mechanisms therefore reveals new therapeutic interventions for multiple human diseases. In neurons, non-homologous end-joining (NHEJ) is the only mechanism available to repair double-stranded DNA breaks (DSB), which is much more error prone than other DNA repair processes. However, there are no therapeutic interventions to enhance DNA repair in diseases affecting neurons. NHEJ is also a useful target for DNA repair-based cancer therapies to selectively kill tumour cells. Protein disulphide isomerase (PDI) participates in many diseases, but its roles in these conditions remain poorly defined. PDI exhibits both chaperone and redox-dependent oxidoreductase activity, and while primarily localised in the endoplasmic reticulum it has also been detected in other cellular locations. We describe here a novel role for PDI in DSB repair following at least two types of DNA damage. PDI functions in NHEJ, and following DNA damage, it relocates to the nucleus, where it co-localises with critical DSB repair proteins at DNA damage foci. A redox-inactive mutant of PDI lacking its two active site cysteine residues was not protective, however. Hence, the redox activity of PDI mediates DNA repair, highlighting these cysteines as targets for therapeutic intervention. The therapeutic potential of PDI was also confirmed by its protective activity in a whole organism against DNA damage induced in vivo in zebrafish. Hence, harnessing the redox function of PDI has potential as a novel therapeutic target against DSB DNA damage relevant to several human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c93ace9388c019ac0f5bc12962ca74d85497c2d9" target='_blank'>
              The Redox Activity of Protein Disulphide Isomerase Functions in Non-Homologous End-Joining Repair to Prevent DNA Damage.
              </a>
            </td>
          <td>
            S. Shadfar, Fabiha Farzana, S. Saravanabavan, Ashley M Rozario, M. Vidal, Cyril J Jagaraj, S. Parakh, Esmeralda Paric, Kristy C Yuan, Mariana Brocardo, Donna R Whelan, Angela S Laird, J. Atkin
          </td>
          <td>2025-05-15</td>
          <td>Aging cell</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="DNA repair is a hierarchically organized, spatially and temporally regulated process involving numerous repair factors that respond to various types of damage. Despite decades of research, the mechanisms by which these factors are recruited to and depart from repair sites have been a subject of intrigue. Recent advancements in the field have increasingly highlighted the role of phase separation as a critical facilitator of the efficiency of DNA repair. This review emphasizes how phase separation enhances the concentration and coordination of repair factors at damage sites, optimizing repair efficiency. Understanding how dysregulation of phase separation can impair DNA repair and alter nuclear organization, potentially leading to diseases such as cancer and neurodegenerative disorders, is crucial. This manuscript provides a comprehensive understanding of the pivotal role of phase separation in DNA repair, sheds light on the current research, and suggests potential future directions for research and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c72f1d4a9e08095f98c1ef277b37623b6195f2e8" target='_blank'>
              Phase separation in DNA repair: orchestrating the cellular response to genomic stability
              </a>
            </td>
          <td>
            Juxin Deng, Zhaoyang Du, Lei Li, Min Zhu, Hongchang Zhao
          </td>
          <td>2025-05-02</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Programmable gene integration in human cells has the potential to enable mutation-agnostic treatments for loss-of-function genetic diseases and facilitate many applications in the life sciences. CRISPR-associated transposases (CASTs) catalyze RNA-guided DNA integration but thus far demonstrate minimal activity in human cells. Using phage-assisted continuous evolution (PACE), we generated CAST variants with >200-fold average improved integration activity. The evolved CAST system (evoCAST) achieves ~10 to 30% integration efficiencies of kilobase-size DNA cargoes in human cells across 14 tested genomic target sites, including safe harbor loci, sites used for immunotherapy, and genes implicated in loss-of-function diseases, with undetected indels and low levels of off-target integration. Collectively, our findings establish a platform for the laboratory evolution of CASTs and advance a versatile system for programmable gene integration in living systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc7a114e9413b1138dda6d46ec6c355a9e8561f" target='_blank'>
              Programmable gene insertion in human cells with a laboratory-evolved CRISPR-associated transposase.
              </a>
            </td>
          <td>
            Isaac P. Witte, George D. Lampe, Simon Eitzinger, Shannon M Miller, Kiara N Berríos, A. McElroy, Rebeca T. King, Olivia G Stringham, Diego R. Gelsinger, P. Vo, Albert T. Chen, Jakub Tolar, Mark J. Osborn, S. H. Sternberg, David R. Liu
          </td>
          <td>2025-05-15</td>
          <td>Science</td>
          <td>2</td>
          <td>34</td>
        </tr>

        <tr id="Since its advent about ten years ago, the CRISPR-Cas9 system has been frequently used in biomedical applications. It has advanced various fields, and CRISPR-Cas9-based therapeutics have shown promising results in the treatment of specific hematological diseases. Furthermore, CRISPR gene editing technologies have revolutionized cancer research by enabling a broad range of genetic perturbations, including genetic knockouts and precise single nucleotide changes. This perspective focuses on the state-of-the-art methodology of CRISPR knock-ins to engineer immune cells. Since this technique relies on homology-directed repair (HDR) of double-strand breaks (DSBs) induced by the Cas9 enzyme, it can be used to introduce specific mutations into the target genome. Therefore, this methodology offers a valuable opportunity to functionally study specific mutations and to uncover their impacts not only on overall cell functions but also on the mechanisms behind cancer-related alterations in common signaling pathways. This article highlights CRISPR knock-in strategies, protocols, and applications in cancer and immune research, with a focus on diffuse large B cell lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d15e1b343be2ebd40cfad2b5fa562b6f9cb4962e" target='_blank'>
              Strategies for CRISPR-based knock-ins in primary human B cells and lymphoma cell lines
              </a>
            </td>
          <td>
            Sophie Lund, Chun Gong, Xin Yu, Louis M. Staudt, Daniel J Hodson, Sebastian Scheich
          </td>
          <td>2025-05-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third most prevalent malignancy and the second leading cause of cancer-related mortality worldwide. The pathogenesis of CRC primarily stems from the gradual accumulation of genetic mutations, which drive oncogene (e.g., KRAS) activation and tumor suppressor gene (e.g., TP53) inactivation. Loss of genetic stability facilitates the conversion of proto-oncogenes into active oncogenes and the functional impairment of tumor suppressors, collectively propelling CRC progression. The tumor suppressor protein p53, a transcription factor, induces cell cycle arrest, apoptosis, and DNA damage repair under cellular stress, and prevents cancer development by regulating various cellular responses. However, in CRC pathogenesis, TP53 mutations (detected in ~ 74% of cases) subvert these protective mechanisms through dual mechanisms: (i) dominant-negative suppression of wild-type p53 (wt-p53) function, and (ii) acquisition of neomorphic pro-tumorigenic activities, termed gain-of-function (GOF) mutations. New evidence from laboratory and clinical trials shows that some new therapeutic strategies have the potential to treat CRC by reactivating and restoring p53 function, depleting p53 mutants, or targeting p53 with immunotherapy. In this review, we summarize the function of p53 and characterize its mutation in CRC, emphasizing the influence of p53 mutation in the pathogenesis of CRC. In addition, we also describe the current therapeutic strategies for targeting p53 mutants in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/366c1fd84b915fa1e7d9335c6be5c8ce5ddaee30" target='_blank'>
              p53 in colorectal cancer: from a master player to a privileged therapy target
              </a>
            </td>
          <td>
            Sicheng Yan, Fuyuan Zhan, Yuxuan He, Yuehong Zhu, Zhihong Ma
          </td>
          <td>2025-06-19</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 e15149


 Background:
 Cells possess a complex DNA damage response (DDR) system to prevent detrimental mutations from accumulating. The genes encoding various DDR components are frequently mutated in cancer cells and provide potential sites for therapeutic intervention. DNA damaging agents, with one exception, are repaired by the global genome nucleotide excision repair (GG-NER) pathway. Our novel class of drugs, Illudins, are the exception as Illudin-induced DNA damage is not recognized by GG-NER and can only be repaired by the transcription-coupled nucleotide excision repair (TC-NER) pathway. Studies utilizing isogeneic cell lines completely deficient in specific TC-NER proteins (e.g. homozygous ERCC6 knockout) indicate these cells are extremely sensitive to Illudins (up to 30-fold). In contrast, cells deficient in GG-NER activity are not sensitive to our drugs. Moreover, non-replicating or normal cells are minimally affected by Illudins, as evidenced by the lack of systemic toxicity in humans treated with one of our drugs. In TC-NER deficient cancer cells, however, our drug ITX-0121 blocks the TC-NER polymerase complex, which disengages from DNA, and initiates the irreversible process of apoptosis or cellular suicide. Many cancers exhibit deficiencies in DDR mechanisms and cannot properly repair DNA damage.
 Methods:
 Analysis of the GENIE Database indicates that the overall incidence of TC-NER deficient cancers in solid tumors is ~10%, and includes all major histological classifications (breast, ovarian, endometrial, prostate, renal, bladder, bone, colorectal, liver, lung, skin, pancreatic, thyroid, etc.) or ~175,000 new patients annually. These cancers, however, possess monoallelic mutations, as opposed to biallelic, and thus have some residual TC-NER functionality. To determine the relative sensitivity of cancers with monoallelic alterations in TC-NER genes, we generated colorectal HCT-116 cells with a monoallelic deletion in a single TC-NER gene. For example, we are generating daughter cell lines with small monoallelic deletions in ERCC2, ERCC3, ERCC4, ERCC6, and ELOF1 genes.
 Results:
 We are comparing sensitivity of these daughter lines to our lead drug ITX-0121, as well as to other DNA damaging agents such as cisplatin. Preliminary results indicate a monoallelic alteration in a TC-NER gene can confer > 6-fold sensitivity to ITX-0121, which is clinically relevant based on PK results. Based on these PK results, we believe the maximum tolerated dose (MTD) of ITX-0121 in patients to be > 1.2 mg/kg; yet only 0.4 mg/kg will be sufficient for clinical efficacy against cancers that are phenotypically TC-NER deficient.
 Conclusions:
 Thus, despite the administration of a cytotoxic agent, TC-NER deficient cancer patients can be treated at a non-toxic dose; if toxicity occurs, it should be limited to transient thrombocytopenia, as indicated by prior studies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15a4f45d8c212b489e848805a2ff3ec30d91ef9b" target='_blank'>
              ITX-0121: A therapeutic strategy for transcription-coupled repair-deficient cancers.
              </a>
            </td>
          <td>
            M. Kelner, Raymond T. Suhandynata, Venkata R. Kotamraju
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="CHROMR is a primate-specific long noncoding RNA with emerging roles in homeostasis and pathophysiology. Elevated blood levels of CHROMR have been observed in patients with cardiovascular disease and several cancers, where it is correlated with poor clinical outcomes. Like many lncRNAs, CHROMR accumulates in both the nucleus and the cytoplasm, and it assumes distinct functions in each of these cellular compartments. In the nucleus, CHROMR sequesters a transcriptional repressor complex to activate interferon-stimulated gene expression and antiviral immunity. In the cytoplasm, CHROMR competitively inhibits microRNAs involved in cholesterol efflux and cell cycle regulation, thereby impacting gene pathways involved in reverse cholesterol transport, HDL biogenesis, and tumor growth. In this review, we detail the multifaceted functions of CHROMR in cholesterol metabolism, innate immunity, and cancer progression. We also explore the potential molecular mechanisms that govern its expression and dynamic subcellular localization, which may be key to its functional versatility. Advancing our understanding of the regulatory networks and cellular environments that shape CHROMR activity will be critical for assessing its promise as a therapeutic target and diagnostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e434fd00ac8710c07c05ad122a8af40ef9af54ab" target='_blank'>
              The Multifaceted Roles of CHROMR in Innate Immunity, Cancer, and Cholesterol Homeostasis
              </a>
            </td>
          <td>
            Emma R. Blaustein, Coen van Solingen
          </td>
          <td>2025-06-01</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha (PIK3CA) plays a critical role in cell growth and survival. PIK3CA somatic mutations are linked to the PIK3CA-related overgrowth spectrum (PROS) syndrome. Cells from those patients appeared to be associated with a moderate but significant radiosensitivity. Mutations or amplifications in this gene are common in breast, colorectal, and lung cancers. Alterations in the PIK3CA gene, including amplification and mutation, are common in cancer, but their influence on radiotherapy is not yet fully understood. This report reveals a potential association between PIK3CA gene modifications and radiosensitivity (p < 0.05) deducted from 8 established colorectal cancer cell lines (HT-29, DLD-1, HCT-116, SW480, HCT-15, Colo-320, and LoVo). Meanwhile gene amplification (> 2) seems to be linked to increased radiation sensitivity, mutations appear to be associated with increased radioresistance in colorectal cancer cells. Leveraging this relationship, PIK3CA amplification and mutations could act as biomarker to pinpoint patients who might benefit from more personalized radiotherapy regimens. However, this association is preliminary and hypothesis-generating in view of the limited number of cases. Further studies are needed to confirm this conclusion. By uncovering the distinct mechanistic effects of these PIK3CA alterations on radiosensitivity phenotype in both normal and cancerous cells, researchers can lay the groundwork for tailored radiotherapy strategies in colorectal cancer. This insight could enhance treatment effectiveness while reducing side effects, ultimately leading to improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d2f05a3aac70c0b974b05ec72c5f8dff7113c5" target='_blank'>
              Effects of PIK3CA Gene Modifications on Radiosensitivity in Colorectal Cancer Cells.
              </a>
            </td>
          <td>
            Ghazi Alsbeih, K. Al-Hadyan
          </td>
          <td>2025-06-01</td>
          <td>Asian Pacific journal of cancer prevention : APJCP</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Aneuploidy, characterized by the gain or loss of chromosomes, plays a critical role in both cancer and congenital aneuploidy syndromes. For any aneuploidy, we can distinguish between its general effects and its chromosome-specific effects. General effects refer to the common cellular stresses induced by aneuploidy, such as impaired proliferation, proteotoxic stress, and altered metabolism, which occur regardless of the specific chromosome involved and profoundly impact cellular and organismal functions. These generalized stresses often hinder cell fitness but can also, under certain conditions, contribute to cancer progression. In contrast, chromosome-specific effects arise from the altered dosage of particular genes on the gained or lost chromosome. These effects vary depending on the chromosome involved and can provide specific fitness effects in cancer cells or distinct developmental phenotypes in congenital aneuploidies like Down syndrome. Understanding the interplay between these two levels of effects is crucial for deciphering the outcomes of aneuploidy. This review synthesizes current knowledge and discusses future directions for unraveling the hallmarks of aneuploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72635ce4b3e9d86105cc5d392c530bed90825c86" target='_blank'>
              The Hallmarks of Aneuploidy in Cancer and Congenital Syndromes.
              </a>
            </td>
          <td>
            Pan Cheng, Karan Singh, Roger H. Reeves, Teresa Davoli
          </td>
          <td>2025-05-07</td>
          <td>Annual review of genomics and human genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Cholangiocarcinoma (CCA) is a highly heterogeneous malignancy, primarily comprising intrahepatic (iCCA) and extrahepatic (eCCA) subtypes. Reconciling the variability between iCCAs and eCCAs in clinical trials remains a challenge, largely due to the inadequate understanding of their shared and subtype-specific cellular heterogeneity. We aim to address this issue using single-cell and spatially resolved transcriptomic approaches. Methods We performed comprehensive single-cell RNA sequencing (scRNA-seq) by profiling 109,071 single cells from 28 samples, including chronic biliary inflammatory conditions (n = 7) and CCAs from different anatomical sites (n = 21). Findings were validated using external multi-omics datasets, tissue microarray cohort, spatial RNA in situ sequencing, CCA patient-derived organoids (PDOs), and mouse models. Results iCCAs and eCCAs exhibited distinct tumor ecosystems, with notable differences in cellular composition, diversity, and abundance across various cell types. Non-malignant epithelial cells displayed divergent precancer hallmarks from different biliary sites, with inflammatory extrahepatic bile ducts exhibiting early hijacking of the gastrointestinal metaplastic process. We identified seven meta-programs within cancer cells, mapped into four major subtypes. This subtyping was validated using external CCA cohorts and PDO models, distinguishing patients based on clinical outcomes and drug vulnerabilities. Specifically, iCCAs were associated with a senescent program, while eCCAs were enriched in an IFN-responsive program linked to adverse clinical outcomes and increased drug resistance. We identified a basal-like LY6D+ cancer cell subpopulation specific to eCCAs, which displayed significant stemness, drug resistance, and IFN-responsive features. This subpopulation was closely associated with an interferon-stimulated gene 15 (ISG15)-enriched mesenchymal and immune microenvironment. Functional assays demonstrated that ISG15 stimulation significantly boosted stemness, basal-like features, and drug resistance in eCCA cells, highlighting its pivotal role in sustaining the LY6D+ progenitor niches. Conclusion We present a comprehensive single-cell landscape of CCAs, uncovering the molecular heterogeneity between iCCA and eCCA subtypes. Transcriptomic subtyping of CCA cancer cells offers implications for clinical stratification and functional precision oncology. We identify basal-like epithelial progenitors and characterize their associated ISG15-enriched microenvironment in eCCAs. These findings hold significant promise for the development of novel prognostic biomarkers, therapeutic targets, and treatment strategies for CCAs. Supplementary Information The online version contains supplementary material available at 10.1186/s13045-025-01716-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fd09acd4eb4bccd6f0a7c5305db1cb0247e6ead" target='_blank'>
              Deciphering cholangiocarcinoma heterogeneity and specific progenitor cell niche of extrahepatic cholangiocarcinoma at single-cell resolution
              </a>
            </td>
          <td>
            Chunliang Liu, Xiang Wang, Erdong Liu, Yali Zong, Wenlong Yu, Youhai Jiang, Jianan Chen, Mingye Gu, Zhengyuan Meng, Jingfeng Li, Yang Liu, Yongjie Zhang, Jing Tang, Hongyang Wang, Jing Fu
          </td>
          <td>2025-06-23</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 e15034


 Background:
 Endometrial cancer (EC) illustrates diverse nucleotide substitution and chromosomal re-arrangement signatures like other cancers. Namely, four molecular types are identified, replicative polymerases, microsatellite stable, copy number high/serous-like, and microsatellite instability characterized by mutations in mismatch repair genes. With advances in molecular techniques, early prediction of disease aggressiveness can be revealed using comprehension in ECs. Although few cell proliferative genes have been documented, a complete molecular landscaping is essential for effective treatment planning and decision-making. We report, the association between clinical disease status and corresponding molecular findings to establish the role of the latter in the early prediction of disease aggression in endometrial cancer.
 Methods:
 Retrospectively, we investigated the association of genomic mutational profiles of 40 endometrial cancer patients and their clinical disease status from PET scans and HPE reports. Next Generation Sequencing (NGS) test was performed using a comprehensive genomic panel (CGP) OncoIndx assay.
 Results:
 Amongst 40 patients, 65% (n = 26/40) of patients had progressive and metastatic disease from PET scans and HPE reports indicative of disease aggressiveness. Almost 80% (n = 20/26) of the aggressive disease cohort were detected to have a minimum of one genomic alteration. Their mutational profiles showed a diverse spectrum of pathway-related aberrations from cell proliferation, progression, double-strand DNA (dsDNA) repair, mismatch repair (MMR), and tumor suppression constituting a total of 83 genomic alterations detected in the whole cohort. Further, the cell proliferation and dsDNA repair pathways exhibited the highest mutational load of 57.8% (n = 48/83) and 14.5% (n = 12/83) respectively in the cohort. Likewise, tumor suppressor genes constituted the lowest frequency at 6% (n = 5/83).In addition, five patients were detected with high MSI scores and one patient with an intermediate MSI score of which two MSI-high patients were also found to harbor MMR gene mutations (MSH3:p.K383Rfs*32, p.D403H, p.E456* and p.E512*, MSH6:p.F1088Sfs*2).
 Conclusions:
 Using CGP, we observed mutational frequencies of cell proliferation and dsDNA repair pathways that act as crucial markers for disease aggression in EC. We also noted that high MSI scores were not always concordant with mutations in the classical MMR genes and vice versa. This also indicates the individual significance of estimating MSI scores in these patients for clinical decision-making. Since the identified landscape of mutations well correlated with the disease states categorized from their clinical findings, they could potentially transform into molecular attributes for early prediction of progression and metastasis aiding treatment planning and decision making.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8df9baa418d967a1f642449b698f5fd35ecbbbc" target='_blank'>
              Mutational spectrum of cell proliferation genes as early predictive markers for aggressive disease in endometrial cancers.
              </a>
            </td>
          <td>
            Aarthi Ramesh, Rohit Ranade, Sandhya Iyer, Nidhi Patel, A. D'Souza, Sangeeta Prajapati, Bhagwat Jadhav, A. Vasudevan, A. Bharde, Jayant J Khandare, Gowhar Shafi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [13],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>