<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-10-20 09:30:34 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>40</td>
          <td>105</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>83</td>
          <td>139</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>115</td>
          <td>146</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>366</td>
          <td>139</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>322</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="In tumors, extrachromosomal DNA (ecDNA) is an important driver of oncogene expression, genomic instability, the evolution of drug resistance, and poor patient prognosis. ecDNA is present in various tumors but is rarely found in normal cells. Here, we provide a detailed review of the structure, genetics, occurrence, outcomes, and functions of ecDNA, offering further reference for research on ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854e01c4800ec9b678143d63d0ad28cdee5f8010" target='_blank'>
              The role of extrachromosomal DNA in tumorigenesis and progression
              </a>
            </td>
          <td>
            Xiaoyang Ma, Xiaolin Yu, Chuan Wu, Lixing Song
          </td>
          <td>2025-09-15</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Extrachromosomal circular DNA (eccDNA) is an emerging class of genetic material that exists outside of the chromosomal genome. These circular DNA molecules are gaining increasing attention as important biomarkers in various cancers because of their roles in gene amplification, genetic heterogeneity, and drug resistance. In this review, we explore in depth the impacts of eccDNAs on cancer biology, their potential to predict treatment sensitivity and resistance, and their involvement in the development of new anticancer therapies. eccDNAs can be used as biomarkers for various tumor types to help diagnose and predict prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1f8c896f37f0d86135daabcae627eb40e4e88c" target='_blank'>
              Extrachromosomal Circular DNA as a Cancer Biomarker: From Diagnosis to Treatment
              </a>
            </td>
          <td>
            Hexin Li, Jiahui Cai, Gang Zhao, Lihui Zou
          </td>
          <td>2025-09-08</td>
          <td>Cancer Innovation</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6f51d8b7476391e7b48e8b1303fc52862802717" target='_blank'>
              Comparative Analysis of CHEK1, Extrachromosomal DNA Dynamics and 1p/19q Status in Lower Grade and High Grade Glioma
              </a>
            </td>
          <td>
            MD Steven Lehrer, MD JD MS Peter Rheinstein
          </td>
          <td>2025-10-05</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Extrachromosomal circular DNA (eccDNA) refers to small, circular DNA molecules that originate from chromosomal sequences and are prevalent across nearly all eukaryotic organisms. In humans, eccDNAs are widely distributed in normal tissues, cancerous tissues, and body fluids, where they play important roles in tumorigenesis and are often associated with poor clinical outcomes. Given their biological and clinical significance, a well-integrated and high-quality database is essential for advancing eccDNA-related research. To address this need, we developed eccDNABase, a comprehensive and curated resource for browsing, searching, and analyzing eccDNAs across multiple species. The database systematically catalogs eccDNA–disease associations from diverse tissues and organisms. Currently, eccDNABase contains 1,875,452 eccDNA–disease associations, encompassing 8,398 ecDNA entries across nine species, 63 diseases, and healthy individuals. Each entry provides detailed information, including eccDNA ID, type, chromosomal localization, species, tissue or cell line source, disease name and Disease Ontology ID, overlap length and percentage with genes, oncogene overlap, detection method, and links to literature and source databases. Given its extensive and curated datasets, eccDNABase serves as a valuable resource for both basic and translational research, offering deeper insights into the role of eccDNA in health and disease. The database is publicly accessible at http://cgga.org.cn/eccDNABase/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf15dcf8d0c3a1e284faf711bb5268a1015b46d0" target='_blank'>
              eccDNABase: A Comprehensive and High-Quality Database for Extrachromosomal Circular DNA
              </a>
            </td>
          <td>
            Jin-Hao Zhang, Chang-Lin Yang, Chang-Yuan Ren, Ji Shi, Han-Xiao Zhou, Ying Zhang, Zhi-Yan Sun, Tao Jiang, Zheng Zhao
          </td>
          <td>2025-09-13</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/127ba85daf25e695241e6cc43a8f791c28684c9d" target='_blank'>
              DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, E. Larose Cadieux, Nana E. Mensah, Xiaohong Liu, Kezhong Chen, O. Chervova, Takahiro Karasaki, C.E. Weeden, Corentin Richard, Siqi Lai, Carlos Martínez-Ruiz, E. Lim, A. Frankell, T. Watkins, Georgia Stavrou, Ieva Usaite, Wei-Ting Lu, Daniele Marinelli, S. Saghafinia, G. Wilson, P. Dhami, H. Vaikkinen, Jonathan Steif, S. Veeriah, R. Hynds, Martin Hirst, C. Hiley, A. Feber, Özgen Deniz, M. Jamal-Hanjani, N. Mcgranahan, Stephan Beck, J. Demeulemeester, M. Tanić, C. Swanton, P. Van Loo, N. Kanu
          </td>
          <td>2025-09-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Abstract Background Extrachromosomal circular DNA (eccDNA) has emerged as a critical driver of oncogenesis, yet its functional roles in high‐grade serous ovarian cancer (HGSOC) remain poorly characterized. This highlights the need for comprehensive investigations into the abundance, biogenesis, and functional implications of eccDNA in HGSOC. Methods To characterize eccDNA in HGSOC, we performed comprehensive Circle‐seq analysis to assess eccDNA abundance and genomic annotation in HGSOC tissues compared to normal ovarian tissue. For mechanistic validation of eccDNA biogenesis pathways, targeted knockdown experiments of microhomology‐mediated end‐joining (MMEJ) dependent on LIG3 and POLQ were conducted. Functional characterization of HGSOC‐specific eccDNA‐harboring precursor microRNAs (eccMIRs) included in vitro assays using HGSOC cells and in vivo tumor growth experiments. Results Circle‐seq analysis revealed a 13‐fold increase in eccDNA abundance in HGSOC compared to normal ovarian tissue, with significant enrichment in promoter and coding regions. The MMEJ pathway was identified as the predominant pathway for eccDNA biogenesis in HGSOC, supported by characteristic microhomologies at junction sites and validation via LIG3 and POLQ knockdown experiments. Notably, HGSOC‐specific eccDNA frequently contained functional eccMIRs (eccMIR3661, eccMIR618, and eccMIR2277), which generate oncogenic miRNAs. These miRNAs promote tumor progression by downregulating tumor suppressor genes and activating key oncogenic pathways. Functional assays confirmed that these eccMIRs significantly enhanced HGSOC cell proliferation, migration, and invasion in vitro and promoted tumor growth in vivo. Conclusions These results underscore eccDNA as an oncogenic driver in HGSOC through non‐coding RNA‐mediated regulatory mechanisms, revealing novel therapeutic opportunities for targeting eccDNA biogenesis in this aggressive malignancy. Key points This study revealed a 13‐fold increase of eccDNA in HGSOC compared to normal tissues, with significant enrichment in promoter and coding regions. eccDNA‐derived miRNAs (eccMIRs) were shown to enhance cancer cell proliferation, invasion, and tumor growth through the expression of oncogenic miRNA sequences. The study highlights the importance of the MMEJ pathway in eccDNA generation and proposes that targeting eccDNA biogenesis in this aggressive malignancy presents a novel therapeutic opportunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc21093a2da7863b736808acd2b5cbd885658bf0" target='_blank'>
              Extrachromosomal circular DNA expressing miRNA promotes ovarian cancer progression
              </a>
            </td>
          <td>
            Ning Wu, Ling Wei, Qiyu Liu, T. He, Cuiyu Huang, Yunpeng Jiang, Kailong Li, Hongyan Guo, Fengbiao Mao, Xiaolu Zhao
          </td>
          <td>2025-09-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT Background This article presents an extensive review of the advancements in research concerning extrachromosomal circular DNA (ecDNA) within the context of breast cancer. As a distinct form of DNA, ecDNA is critically involved in the initiation, progression, diagnosis, and treatment of breast cancer. Methods The article provides a comprehensive analysis of the mechanisms underlying the formation of ecDNA, highlighting factors such as aberrant DNA damage repair and chromosomal rearrangements. It further examines the biological roles of ecDNA in augmenting oncogene expression, fostering tumor heterogeneity, and facilitating immune evasion. Results Epidemiological studies indicate significant variability in the distribution of ecDNA across different breast cancer subtypes, with a notable association with invasive subtypes, such as triple‐negative breast cancer. The presence of ecDNA is linked to poor prognosis and treatment resistance in patients. Current detection technologies for ecDNA include molecular biology and imaging techniques, and its detectability in plasma underscores its potential as a biomarker for liquid biopsy. Conclusions The development of ecDNA‐targeted therapies and their integration with immunotherapy strategies offers promising new avenues for breast cancer treatment. Despite challenges such as incomplete elucidation of mechanisms, standardization of detection methods, and ethical considerations, ecDNA remains a valuable biomarker and therapeutic target in breast cancer. Future research is anticipated to advance its clinical transformation and application in individualized treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3be1ca65e1813757b54ee00006ec2026ac1e51f0" target='_blank'>
              Research Advances of Extrachromosomal Circular DNA in Breast Cancer
              </a>
            </td>
          <td>
            Xiaodan Zhu, Yiyang Qian, Quan Tang, Jiahui Li, Yuqing Geng, Shixia Gong, Chunhui Jin
          </td>
          <td>2025-10-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA abnormalities characterized by cytogenetic imaging at the single cell resolution, i.e. karyotyping, have long served as cancer diagnostic and prognostic biomarkers. To enable in-silico karyotyping using unpaired whole-genome sequencing data, we developed Seq2Karyotype (S2K), a tool that fits karyotype models with clonality estimation based on read-depth and allelic imbalance in a bulk sample and supports visualization-guided refinement. Analysis on 19 adult and pediatric cancer cell lines revealed unexpected intratumoral heterogeneity involving multiple copy number variation (CNV) states including whole-genome duplication, which were validated by imaging and single-cell omics profiling. Analyses on patient samples showed high concordance with clinical cytogenetic reports for acute myeloid leukemia, and revealed evolutionary trajectories from multi-region metastatic neuroblastomas implicating reversion. These findings highlight extensive and dynamic intratumoral heterogeneity contributed by CNV in both cell line models and patient samples, which may inform future research on tumor evolution under selective pressure such as drug exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0572920d9d208e3e04940659c463f83c4e0642eb" target='_blank'>
              Seq2Karyotype (S2K): A Method for in-silico Karyotyping Using Single-Sample Whole-Genome Sequencing Data
              </a>
            </td>
          <td>
            Limeng Pu, Karol A. Szlachta, Virginia Valentine, Meiling Jin, Sivaraman Natarajan, Robert Greenhalgh, Nadezhda V. Terekhanova, Jian Wang, Daniel K. Putnam, Li Dong, Lingxiang Jiang, Soumya Tumbath, Xiumei Huang, Xiaotu Ma, Thomas Look, Marcin W. Włodarski, Lu Wang, Steven Burden, J. Easton, Xiang Chen, Jinghui Zhang
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Abstract Cholangiocarcinoma (CCA) is a lethal cancer of the bile duct and is a major health concern in several parts of the world, including northeastern Thailand, where CCA incidence is the highest due to the endemic liver fluke Opisthorchis viverrini. Multiple studies have characterised genomic alterations in CCA tumours, and specific chromosomal alterations can predict prognosis. However, it is not known whether chromosomal instability (CIN), ongoing genomic alteration characteristic of most cancer types, is present in CCA tumours. In this study we leveraged a panel of cancer cell lines derived from fluke‐positive CCA patients, as well as a matched normal cholangiocyte line as a control, to characterise CIN in CCA. We observed elevated rates of chromosome segregation errors compared to normal cells, although overall CIN rates were lower than those for highly genomically unstable cancers, such as colorectal or ovarian cancer. Chromosome segregation errors in CCA cell lines were potentially driven by elevated DNA replication stress and centrosome duplication. Single‐cell genome sequencing and karyotyping of the cell lines showed extensive structural and numerical chromosomal aberrations, as well as copy number alterations that were heterogeneous between individual cells, supporting the presence of ongoing CIN in these cell line models. Low‐pass whole‐genome sequencing of 33 CCA tumour samples with matched normal tissue from northeastern Thailand, a liver fluke‐endemic region showed increased whole and subchromosomal level alterations, with a higher extent of genomic alterations in intrahepatic tumours compared to extrahepatic. Eight tumours carried focal amplifications and/or deletions involving known cancer genes, as well as potential chromosomal instability‐associated genes, including CCNE1 amplifications and a rare amplification of BRCA1. This study provides increased understanding of the rate and potential mechanisms of CIN in CCA that may inform new therapeutic strategies that synergise with specific ongoing CIN mechanisms. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4433abbfa4bf430e31ec8b07fb91e4f3d1fdbb22" target='_blank'>
              Chromosomal instability and genomic alterations in cholangiocarcinoma from Northeastern Thailand
              </a>
            </td>
          <td>
            Raksawan Deenonpoe, MA Guscott, Sasithorn Watcharadetwittaya, Isabel L McNeill, D. Moralli, N. Shaikh, SE McClelland
          </td>
          <td>2025-09-15</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 Structural rearrangements are increasingly understood to signiﬁcantly alter the prognosis of a wide range of CNS tumours. For example, co-deletion of CDKN2A/B is a hallmark of high-grade astrocytoma, oligoden- droglioma and meningioma. Despite their clinical importance, detection of intra-chromosomal rearrangements is challenging using short-read technologies and copy number variations (CNV) are commonly inferred from methylation array. Such testing has limited resolution to detect small, intra-chromosomal CNV calls with high conﬁdence. Nanopore technology is an emerging technique in CNS diagnostics, enabling targeted long-read sequencing alongside methylome classiﬁcation. However, current implementations of adaptive targeting are restricted to an a priori deﬁned set of key driver oncogenes. In contrast, genomic breakpoints underpinning SVs are unique to individual patients and are thus challenging to sequence at high coverages.



 Here, we present BATMAN, a development of the ROBIN diagnostic tool that enables real-time enrichment of patient-speciﬁc genomic breakpoints. BATMAN infers genomic breakpoint loci during sequencing and dynam- ically adapts the target panel in real-time.



 We present the performance of BATMAN on CNS tumours harbouring a variety of CNVs. We demonstrate the potential to generate long-reads spanning deletion of CDKN2A/B in high-grade astrocytoma, oligodendroglioma, and glioblastoma. Furthermore, we demonstrate the ability of BATMAN to resolve complex rearrangements such as chromothripsis events and accurately infer genome-wide copy number variation load, an emerging prognostic biomarker.



 BATMAN enables the sequencing of long reads spanning breakpoints at high coverages, enabling the robust detection of clinically relevant SVs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/099a421839bacfdf0fe835b8745c6eaa1153ef57" target='_blank'>
              BATMAN: BREAKPOINT ADAPTIVE TARGETING ALONGSIDE METHYLATION ANALYSIS ON NANOPORE
              </a>
            </td>
          <td>
            Simon Deacon, T. Murray, J. Trotman, P. Tarpey, Stuart Smith, S. Paine, M. Loose
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d313ac53c992b0d08bed8668f1fbbbe338a42f" target='_blank'>
              Clinicopathological and genomic features of extrachromosomal DNA in gastric cancer.
              </a>
            </td>
          <td>
            Yukio Hokazono, Mihoko Saito-Adachi, Natsuko Hama, Yasushi Totoki, Hiromi Nakamura, Y. Arai, Shinichi Yachida, A. Fukagawa, Hirofumi Rokutan, T. Ushiku, Tatsuhiro Shibata
          </td>
          <td>2025-10-18</td>
          <td>Gastric cancer : official journal of the International Gastric Cancer Association and the Japanese Gastric Cancer Association</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract While the coding genome of pancreatic cancer has been characterized in detail, features of the noncoding genome remain relatively unexplored. We used whole-genome sequencing to study the coding and noncoding genomes (promoters, enhancers, and noncoding–nonenhancer regions) in two unique patient cohorts. We find that treated cancers have a significantly higher mutational burden than untreated cancers in all four genomic regions. However, the relative proportion of mutations in each region is preserved despite treatment-induced genetic bottlenecks. Compared with other noncoding regions, enhancers have a lower number of mutations/Mb. Enhancers also have a distinct mutational signature with enrichment of SBS39. Enhancer sequences were segregated into conserved and nonconserved regions based on the overlap of predicted orthologous regions in the human and mouse genomes and the conserved regions screened for recurrent somatic mutations. We find that recurrent somatic mutations in conserved enhancer regions largely correspond to those associated with known transcription factors with a role in pancreatic development and cancer, including KLF5 and TP63. Transcriptional expression based on RNA sequencing data of cancers with enhancer mutations showed significantly different levels of expression, most often a loss of expression, compared with cancers without enhancer mutations, suggesting a functional effect. These findings expand our knowledge of the noncoding genome in pancreatic cancer and point to an unexplored role of conserved enhancer mutations for pancreatic cancer as well. Significance: Recurrent somatic mutations in enhancer regions that control pancreatic lineage genes represent a previously unrecognized source of genetic alteration in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edbe76ca8bf694ac9c6f9820ca8a2fe03fe38724" target='_blank'>
              The Noncoding Mutational Landscape of Pancreatic Cancer Reveals Recurrent Somatic Mutations in Enhancer Regions
              </a>
            </td>
          <td>
            Akimasa Hayashi, Yu-Jui Ho, Alvin P. Makohon-Moore, Amanda Zucker, Jungeui Hong, Shigeaki Umeda, Elias-Ramzey Karnoub, Jinlong Huang, Priscilla Baez, R. Kappagantula, Jerry P Melchor, Wungki Park, Eileen M. O’Reilly, Nicholas D Socci, Shinya Oki, Christine A. Iacobuzio-Donahue
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Human papillomavirus (HPV) integration is recognized as a hallmark event in cervical carcinogenesis. However, it does not represent a routine phase of the viral life cycle but rather a stochastic occurrence, often constituting a dead-end pathway for the virus. High-risk human papillomavirus (hr-HPV) exhibits a greater propensity for integration. The progression from initial infection to genomic integration constitutes a dynamic multi-step oncogenic process in the development of cervical cancer (CC). This process involves viral entry, immune evasion, persistent infection, and ultimately integration. This article innovatively provides a comprehensive overview of this multi-stage mechanism: HPV, via the L1/L2 proteins, mediates internalization and establishes infection. Subsequently, under the influence of factors such as the host’s genetic background, vaginal microbiota imbalance, and immune evasion, the host’s DNA damage response (DDR) pathways are activated. Viral DNA integrates into host genome vulnerable sites (e.g., 3q28 and 8q24) through microhomology-mediated end joining (MMEJ) or other alternative pathways. Following integration, the expression of viral oncogenes persists, triggering host genomic rearrangements, aberrant epigenetic modifications, and immune microenvironment remodeling, all of which collectively drive cervical cancer progression. The study further reveals the clinical potential of HPV integration as a highly specific molecular biomarker, offering new perspectives for precision screening and targeted therapy. This dynamic model deepens our understanding of the HPV carcinogenic mechanism and provides a theoretical basis for intervention strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2f3540f21b81b3f4af2b24e5ab200e71a6ae9d" target='_blank'>
              From Viral Infection to Genome Reshaping: The Triggering Role of HPV Integration in Cervical Cancer
              </a>
            </td>
          <td>
            Junlan Li, Shuang Li
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Histone acetylation is widely assumed to directly instruct gene activation. Among acetylated residues, H4K16ac is one of the most abundant modifications, conserved across all eukaryotes. Despite its established role in X-chromosome hyperactivation in Drosophila, its function in mammalian cells has remained elusive. Here, we show that in human somatic cells, H4K16ac does not substantially affect gene expression, but instead controls the spatiotemporal program of genome replication. By combining a meta-analysis of public datasets and perturbation experiments designed to minimize confounding effects, we found that H4K16ac is neither associated with nor required for transcriptional activity. Rather, H4K16ac depletion resulted in premature replication of heterochromatic regions and widespread alterations in replication timing across the genome. These defects were driven by the aberrant activation of cryptic replication origins at long terminal repeats—repetitive elements typically marked by H4K16ac and whose sequence context resembles that of canonical origins in euchromatic regions. Our findings reveal an unexpected role for one of the most prevalent chromatin modifications and uncover a new regulatory mechanism that safeguards genome replication fidelity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58d31c25c7ef175d7c8a34dda15d005ef97496d8" target='_blank'>
              Mammalian H4K16ac regulates the spatiotemporal order of genome replication rather than gene expression
              </a>
            </td>
          <td>
            Marta Milan, Valeria Runfola, Manthan Patel, Lucia Falbo, R. Noberini, C. Soriani, S. Rodighiero, Tiziana Bonaldi, Vincenzo Costanzo, Madapura M. Pradeepa, Paola Scaffidi
          </td>
          <td>2025-09-23</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Tumor evolution is driven by various mutational processes, ranging from single-nucleotide variants (SNVs) to large structural variants (SVs) to dynamic shifts in DNA methylation. Current short-read sequencing methods struggle to accurately capture the full spectrum of these genomic and epigenomic alterations due to inherent technical limitations. To overcome that, here we introduce an approach for long-read sequencing of single-cell derived subclones, and use it to profile 23 subclones of a mouse melanoma cell line, characterized with distinct growth phenotypes and treatment responses. We develop a computational framework for harmonization and joint analysis of different variant types in the evolutionary context. Uniquely, our framework enables detection of recurrent amplifications of putative driver genes, generated by independent SVs across different lineages, suggesting parallel evolution. In addition, our approach revealed gradual and lineage-specific methylation changes associated with aggressive clonal phenotypes. We also show our set of phylogeny-constrained variant calls along with openly released sequencing data can be a valuable resource for the development of new computational methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e007da3f913b3e2ad1860533d9c4d9c4f0893dfe" target='_blank'>
              Long-read sequencing of single cell-derived melanoma subclones reveals divergent and parallel genomic and epigenomic evolutionary trajectories
              </a>
            </td>
          <td>
            Yuelin Liu, Anton Goretsky, A. Keskus, S. Malikić, Tanveer Ahmad, Farid E. Michael Gertz, Rashidi Mehrabadi, Michael Kelly, Maria Hernandez, Charlie Seibert, J. M. Caravaca, Kayla Kline, Yongmei Zhao, Ying Wu, Biraj Shrestha, Bao Tran, Arindam Ghosh, Xiwen Cui, Antonella Sassano, Lakshay Malik, Breeana Baker, Cornelis Blauwendraat, Kimberley J. Billingsley, Eva Pérez-Guijarro, Glenn Merlino, Erin K. Molloy, S. C. Sahinalp, Chi-Ping Day, M. Kolmogorov, Contents
          </td>
          <td>2025-09-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07f59455ffad8232534ff09a38ff5659981429a5" target='_blank'>
              The formation and propagation of human Robertsonian chromosomes.
              </a>
            </td>
          <td>
            L. G. de Lima, A. Guarracino, S. Koren, T. Potapova, S. McKinney, A. Rhie, Steven J. Solar, Chris Seidel, Brandon Fagen, B. Walenz, G. Bouffard, Shelise Y. Brooks, Michael Peterson, Kate Hall, Juyun Crawford, Alice C. Young, Brandon D. Pickett, Erik Garrison, A. Phillippy, J. Gerton
          </td>
          <td>2025-09-24</td>
          <td>Nature</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Recent evidence highlights the role of extrachromosomal circular DNAs (eccDNAs) in cancers. However, reports regarding its role in hepatocellular carcinoma (HCC) are infrequent. The abundance of eccDNAs from five HCC/adjacent tissue pairs was explored using Circle-Sequencing. eccDNA PAFAH1B3 was selected as one of the objects. The effect of eccDNA PAFAH1B3 on HCC progression was determined using EdU, Transwell, and apoptosis assays. Additionally, the expressions of eccDNA PAFAH1B3, mRNA PAFAH1B3, and epithelial–mesenchymal transition (EMT)-related markers were determined using RT-PCR and WB. A xenograft tumor model was established to explore the function of PAFAH1B3 in vivo, and EMT-related markers were detected using RT-PCR and IHC analyses. The abundance of eccDNA PAFAH1B3 was significantly increased in HCC cell lines after transfection with eccDNA PAFAH1B3, and promoted the proliferation, migration, and invasion of liver cells while inhibiting apoptosis. The levels of mRNA PAFAH1B3 were also upregulated. Furthermore, intratumoral injection of PAFAH1B3 inhibitor suppressed tumor growth, and PAFAH1B3 knockdown increased and decreased the levels of the E-cadherin and N-cadherin, respectively. Our study findings reveal that eccDNA PAFAH1B3 may promote the occurrence and development of HCC by enhancing the expression of PAFAH1B3 and regulating EMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/173a3d83ec58d1fbf7f0f7dcd22ef9f85da2bb94" target='_blank'>
              PAFAH1B3 Exists in Linear Chromosomal and Extrachromosomal Circular DNA and Promotes HCC Progression via EMT
              </a>
            </td>
          <td>
            Dandan Li, Huishan Sun, Yingjie Wang, Yicong Yin, Ying Zhu, Xia Qian, Shanshan Wang, Longhao Zhang, Haitao Zhao, Ling Qiu
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Extrachromosomally amplified oncogenes are shown to drive evolution in a large subset of IDH wild-type glioblastomas, and the spatial patterns of extrachromosomal DNA contain information on a tumor’s evolutionary history.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a02add441c27bf4e8e365b8edcace16e00d2cda" target='_blank'>
              Extrachromosomal DNA–Driven Oncogene Spatial Heterogeneity and Evolution in Glioblastoma
              </a>
            </td>
          <td>
            I. Noorani, Magnus Haughey, J. Luebeck, Andrew Rowan, E. Grönroos, Francesco Terenzi, I. Wong, Davide Pradella, Marta Lisi, Jeanette Kittel, Natasha Sharma, Chris Bailey, C. Weeden, Donald M Bell, Eric Joo, Vittorio Barbè, Matthew G. Jones, King L. Hung, E. Nye, M. Green, Lucy Meader, Emma Norton, Mark Fabian, N. Kanu, M. Jamal-Hanjani, T. Santarius, Andrea Ventura, James A R Nicoll, D. Boche, Howard Y. Chang, V. Bafna, Weini Huang, P. Mischel, C. Swanton, Benjamin Werner
          </td>
          <td>2025-09-08</td>
          <td>Cancer Discovery</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="OncoDB was initially developed to advance cancer research by integrating RNA expression profiles, DNA methylation patterns, clinical annotations, and oncoviral signatures derived from the Cancer Genome Atlas (TCGA) and Genotype-Tissue Expression datasets. We now present OncoDB 2.0, an extensively expanded platform that offers a more comprehensive and integrated view of cancer omics. The updated version includes an atlas of somatic mutations discovered from combined DNA and RNA sequencing, enabling in-depth investigation of mutation patterns across tumor types and their association with clinical features. Furthermore, we have integrated proteomic data from the Clinical Proteomic Tumor Analysis Consortium (CPTAC) and chromatin accessibility data from TCGA, offering new dimensions for oncogene regulation studies. OncoDB 2.0 also introduces advanced multi-omics analysis modules that facilitate the combined exploration of RNA expression, DNA methylation, and somatic mutations, allowing researchers to examine complex cross-omic relationships with greater depth and flexibility. Together, these enhancements make OncoDB 2.0 a robust and invaluable tool for the cancer research community. OncoDB 2.0 is freely available at https://oncodb.org.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2597afaecc32114e7746cc65fb5c9507bfe36bf0" target='_blank'>
              OncoDB 2.0: a comprehensive platform for integrated pan-cancer omics analysis.
              </a>
            </td>
          <td>
            Minsu Cho, Gongyu Tang, Charles S Rogers, Makayla Dove, Xinyi Liu, Yuanxiang Li, Xiaowei Wang
          </td>
          <td>2025-09-23</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The interplay between DNA methylation and super-enhancer activity is emerging as a key focus in cancer research. Super-enhancers, a specialized class of enhancers, strongly activate transcription of their target genes due to their dense clustering with essential transcription factors (TFs) and cofactors. In cancer, especially, these super-enhancers control key oncogenic drivers and often display abnormal DNA methylation patterns that can repress or overexpress target genes in both solid and blood cancers. Furthermore, DNA methylation in the super-enhancer region has been found to influence their regulatory capacity. Although enhancers are typically characterized by low DNA methylation, dysregulated methylation at super-enhancers is seen in most malignancies, affecting TF and chromatin regulator recruitment. Hypomethylation at these sites often accompanies oncogene hyperactivation, while hypermethylation can repress tumor suppressor mechanisms. Recent research highlights DNA methylation as a promising source of cancer biomarkers. This review examines the intricate relationship between DNA methylation and super-enhancer activity in cancer, concentrating on how methylation regulates super-enhancers, modulates oncogene expression, promotes oncogenesis, and serves as a target for novel oncology therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80856f95cde56634ccf673f640abf635b4018a59" target='_blank'>
              Super-enhancer DNA methylation in cancer: the mechanism of action and therapeutic directions
              </a>
            </td>
          <td>
            Yue Bi, Kang Zou, Ke Wang
          </td>
          <td>2025-10-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mouse heterochromatin is characterized by A/T-rich, 234 bp DNA repeat arrays, called major satellite repeats (MSR). We investigated MSR expression in response to a variety of stress conditions by using small molecule compounds. We identified the isoflavone genistein to selectively stimulate MSR transcription, but not that of other DNA repeat elements. Genistein is a natural compound that is frequently used in dietary supplements and has been associated with reducing cancer risk. A 24 h exposure of mouse embryonic fibroblasts (MEF) to genistein results in a more than 100-fold induction of MSR transcripts. This up-regulation depends on the activity of RNA polymerase II and requires a cycling G1 cell population. Blocking the cell cycle at the G2/M stage significantly attenuates genistein-mediated stimulation of MSR transcription. Mechanistically, DNA topoisomerase poisons phenocopy the genistein-dependent up-regulation of MSR expression. Together, these data suggest that MSR transcriptional response is guided by an altered topology of the underlying A/T-rich MSR DNA repeat arrays and reveal a novel function for genistein that may contribute to the anticancer properties of this natural compound. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00623-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64fed7546750b813d379740d82aa6947375846a7" target='_blank'>
              The isoflavone genistein selectively stimulates major satellite repeat transcription in mouse heterochromatin
              </a>
            </td>
          <td>
            Thomas Fuhrmann, N. Shukeir, Reagan W. Ching, Galina Erikson, Yuan Dou, Zoe Sawitzki, Megumi Onishi-Seebacher, Carmen Galán, Thomas Jenuwein
          </td>
          <td>2025-08-25</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80c7c66e7f054de7552ccd4ec8d0186447c75c2a" target='_blank'>
              Single-cell Genomic Copy Number Evolution Reveals Frequent Loss of the Y chromosome in Esophageal Adenocarcinoma.
              </a>
            </td>
          <td>
            Shiwei Yin, Na Zhong, Swati Agrawal, Bingru Feng, Sneha Pradhan, Nathan K. Jobalia, Maryam Vaziripour, Sampath K Poreddy, L. Kresty, Erin A Gillaspie, Nicholas E Dietz, W. Al-Refaie, K. Nandipati, Yusi Fu, Jun Xia
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb14185df59742e1c30589cf1835a68e0538e9b9" target='_blank'>
              Tracking clonal evolution during treatment in ovarian cancer using cell-free DNA.
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Duaa H Al-Rawi, Gryte Satas, Hannah Lees, J. Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, Neeman Mohibullah, Yulia Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitry Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2025-10-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/179d18feca7d057af41f495937f48c4c8af27399" target='_blank'>
              A novel class of non-coding variants driving DNA double-strand breaks is associated with complex genetic diseases
              </a>
            </td>
          <td>
            S. Auber, S. Collins, A. Buyan, A. Aiusheeva, Anne-Laure Finoux, F. Saur, S. Cohen, V. Rocher, C. Arnould, M. Verbanck, I. V. Kulakovskiy, G. Legube, R. Mourad
          </td>
          <td>2025-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Somatic human cells contain a diploid genome consisting of 23 pairs of chromosomes. The maintenance of this diploid state is essential across all layers of biological organization, ranging from the physiology of individual cells to the proper regulation of tissue homeostasis and organismal development. Most cancer cells, however, harbor an aneuploid genome with an abnormal number of chromosomes, including whole and/or partial chromosome gains and losses. These alterations arise as a consequence of mitotic chromosome segregation errors and/or ongoing chromosomal instability (CIN). While aneuploidy usually imposes a fitness cost to nontransformed cells, certain recurrent aneuploidies confer adaptive advantages that are subjected to positive selection throughout tumorigenesis. In this review, we discuss how aneuploidy impacts cellular physiology, fitness, and adaptability in the context of cancer development. We also examine how the aneuploid state and CIN can create vulnerabilities that may be exploited for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb17fd1a7e9ab93107d809a69e7ad9c212a84ff9" target='_blank'>
              Impact of Chromosomal Instability and Aneuploidy in Cancer Development
              </a>
            </td>
          <td>
            Amanda K. Mennie, Brittiny Dhital, Peter Ly
          </td>
          <td>2025-09-30</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Ionizing radiation can lead to breaks in the double-stranded structure of DNA. Although cells have mechanisms that can repair such breaks, misrejoining events can occur that lead to mutations in an individual’s genome. When mutations affect certain genes, such as oncogenes and tumor-suppressor genes, cancer can result. Although ionizing radiation is a known cause of cancer, it has been difficult to determine how much of an excess cancer risk results from low-dose radiation exposure. Radiation signatures that distinguish radiation-induced cancers from spontaneous cancers are thus useful for assessing cancer risks after exposure to low-dose radiation. Recent advances in high-throughput sequencing technologies have enabled the comprehensive profiling of mutations in individual cancer cells. Analyses of radiation-associated cancer genomes have identified an increase in small deletions and chromosome rearrangements, including large deletions, inversions, and translocations, which are likely to be formed by the misrejoining of DNA double-strand breaks. Among these, chromosome rearrangements leading to cancer-related mutational events are potentially useful as biomarkers and as therapeutic targets for treating radiation-induced cancer. This review summarizes research on radiation-induced mutational signatures found in animal and human cancers, consolidates current knowledge, and discusses the future perspectives in this research field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a33c28820d79dcb3eabee840cf5431bbdeaec073" target='_blank'>
              Mutational Signatures in Radiation-Induced Cancer: A Review of Experimental Animal and Human Studies
              </a>
            </td>
          <td>
            K. Daino, C. Tsuruoka, Atsuko Ishikawa, S. Kakinuma, T. Imaoka
          </td>
          <td>2025-08-29</td>
          <td>Biology</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="The distinctive three-dimensional (3D) chromatin architecture adopted by chromosomes during meiosis is essential for fertility, yet the functional implications of the fine-scale spatial organization remain poorly understood. Here, we investigate the impact of RAD21L deletion, a meiosis-specific cohesin subunit, on the 3D genome architecture and gene expression in the male germ line. Using fluorescence-activated cell sorting, high-throughput chromosome conformation capture, and single-cell RNA sequencing, we demonstrate that RAD21L deficiency impairs meiotic chromatin organization and inter/intrachromosomal interactions. This conspicuous 3D genome reorganization also disrupts bouquet formation, resulting in increased telomeric interactions between heterologous chromosomes in primary spermatocytes. Genome reorganization was accompanied by detectable transcriptional dysregulation in spermatogonia and primary spermatocytes, mainly affecting sex chromosomes. Collectively, our findings establish the cohesin RAD21L as a germline-specific critical regulator of genome-wide 3D genome reorganization during spermatogenesis and a modulator of the male germline transcriptional landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2272faa3ad286c58e292f1de1607d234286a48c" target='_blank'>
              Meiotic cohesin RAD21L shapes 3D genome structure and transcription in the male germline
              </a>
            </td>
          <td>
            Laia Marín-Gual, Covadonga Vara, Raquel Sainz-Urruela, Yasmina Cuartero, Lucía Álvarez-González, Natalia Felipe-Medina, Francisca Garcia, E. Llano, M. Martí-Renom, A. Pendás, A. Ruiz-Herrera
          </td>
          <td>2025-10-01</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Significance This study highlights a specific source of endogenous DNA replication stress, G-quadruplexes (G4s), that can increase genome instability in BRCA2-deficient cerebellar granule cell progenitors (GCPs), leading to medulloblastoma. BRCA2 is necessary for maintenance of replication speed at G4s in GCPs, and its absence allows these structures to become hotspots of increased mutagenesis. The G4-unwinding PIF1 helicase was found to be upregulated in tumor cells as a potential way to cope with G4-related replication stress. Targeting PIF1 may represent a therapeutic strategy for BRCA2-deficient medulloblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba0cd9b13650ad33751efc7afb07cdd2a585f9d5" target='_blank'>
              G-quadruplexes as a source of vulnerability in BRCA2-deficient granule cell progenitors and medulloblastoma
              </a>
            </td>
          <td>
            Danielle L. Keahi, Mathijs A. Sanders, M. Paul, A. L. Webster, Yin Fang, Tom Wiley, Samer Shalaby, Thomas S. Carroll, S. Chandrasekharappa, C. Sandoval-Garcia, Margaret L MacMillan, John E. Wagner, M. Hatten, A. Smogorzewska
          </td>
          <td>2025-08-25</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Background BRD9 is involved in multiple physiological and pathological pathways, yet its functional role and molecular mechanisms in gastric cancer (GC) remain largely unexplored. Addressing this knowledge gap is critical given the persistent global mortality burden of GC and the limited efficacy of current therapeutic strategies. Methods BRD9 expression in GC patients was systematically analyzed using immunohistochemical (IHC) assays and transcriptomic datasets. Comprehensive functional validation, employing cellular and murine tumor models, elucidated BRD9’s role in GC progression. Molecular pathways underlying BRD9-mediated gastric carcinogenesis were delineated through integrated approaches, including RNA sequencing, co-immunoprecipitation (co-IP), subcellular fractionation, and luciferase reporter assays. Results BRD9 was significantly overexpressed in GC and associated with poor patient prognosis. Functionally, BRD9 promoted GC cell proliferation and enhanced DNA damage repair capacity. Mechanistically, elevated BRD9 expression inhibited p53 nuclear translocation via direct binding, subsequently activating the E2F transcription factor family. Notably, we identified that E2F1 directly bound to and transactivated the BRD9 promoter, establishing a positive feedback loop that sustains BRD9 expression. Additionally, BRD9 knockdown sensitized GC cells to cisplatin and oxaliplatin treatment. Conclusions These findings highlight the critical role of BRD9 in GC progression and its therapeutic potential. The BRD9-p53-E2F1 axis acts as a crucial regulator of GC cell proliferation and DNA damage response. Targeting BRD9 pharmacologically could be a novel therapeutic approach to enhance chemotherapy efficacy and improve treatment outcomes in GC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02449-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dadff4be7ebdf814a21760a8db20acee07e5dfa" target='_blank'>
              BRD9-p53-E2F1 circuit orchestrates cell growth and DNA damage repair in gastric cancer
              </a>
            </td>
          <td>
            Qingqing Zhou, Qi Wang, Yantao Duan, Chi Zhang, Tengfei Liu, Hengrui Liu, Mindi Zhao, Zhihuang Hu
          </td>
          <td>2025-10-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinomas (HGSOC) have one of the highest burdens of focal copy number alterations amongst cancers studied by The Cancer Genome Atlas. These focal copy number alterations are driven by chromosomal instability and rearrangements. Our current understanding of structural variants (SVs) in HGSOC come largely from a combination of macroscale karyotyping and base-level short-read Illumina sequencing. However, short-read sequencing is inherently hindered in accurate SV calling due to limited information of exact breakpoints captured using this technology. With the development of long-read sequencing, including Oxford Nanopore Technologies (ONT) sequencing used here, SVs are resolved with higher accuracy. Additionally, ONT sequencing allows better haplotype phasing and information about 5-methylcytosine (5mC; gene silencing) and 5-hydroxymethylcytosine (5hmC; gene activation). Here we sought to examine SVs and allele-specific methylation (ASM) in HGSOC cell lines, organoids, and tumors. Our cell line data, from 7 HGSOC cell lines, sequenced the genome to an average coverage of ∼35x. Using the epi2me human variation pipeline (ONT), supplemented by additional long-read SV caller cuteSV and filtering with MAVIS, we identified Insertions, Deletions, Translocations, Duplications, and Inversions. Because our samples lacked germline calls, we leveraged coverage information from the gnomAD database across the Ten-Thousand Genomes Project to remove regions of frequent variation in the population, which greatly filtered the number of insertions and deletions. A strong peak of 300bp was observed in insertions and deletions, which was nearly entirely repetitive elements (SINE, LINE, LTR, and simple repeats), making up over 50% of these SVs. MAVIS annotates SVs predicted to make non-synonymous changes to protein sequences; of 95 non-synonymous SVs across 7 cell lines, 57 were examined by PCR, with successful validation of 85.9%. Long-read sequencing was also haplotype phased, and DSS was used to determine regions of differential methylation between haplotypes. A mean of 828 promoters with ASM were observed, with about 40% shared between at least two cell lines. While 91 different tumor suppressor genes (TSG) had promoters with ASM were identified (mean 25 TSG per cell line), none were confirmed to have a mutation to the expressed haplotype. We are continuing to confirm whether ASM leads to bias in allelic expression of RNA, as well as expanding our study to examine long-read sequencing in organoids and tumor datasets. This work adds granularity to existing data on chromosomal rearrangements in HGSOC. Future directions of this work will examine how translocations between chromosomes influence novel enhancer-gene interactions.



 Jessica D. Lang, Ryan Moreno, Kristin M. Adams, Lauren Kersten. Long-read sequencing of high-grade serous ovarian carcinoma genomes: Insights into structural variants and allele-specific DNA methylation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6303026df4b6b8b5a1d48c24d59b28bf2267c15" target='_blank'>
              Abstract B010: Long-read sequencing of high-grade serous ovarian carcinoma genomes: Insights into structural variants and allele-specific DNA methylation
              </a>
            </td>
          <td>
            Jessica D. Lang, Ryan Moreno, Kristin M Adams, Lauren Kersten
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer (BC) remains one of the most prevalent cancers worldwide and a significant cause of cancer-related mortality among women. Despite significant advancements in understanding the genetic foundations of BC, numerous research initiatives have historically focused on protein-coding genes, which constitute merely about 2% of the human genome. This focus has produced significant insights into oncogenes such as HER2, TP53, and BRCA1, along with tumor suppressor genes. Nonetheless, it has led to the non-coding portions of the genome garnering relatively less focus. Recent studies illuminate the crucial significance of non-coding DNA in cancer biology, highlighting its regulatory roles and influence on tumor formation, metastasis, and treatment resistance. This review examines the importance of non-coding DNA in BC. It provides an in-depth analysis of essential non-coding regions, their functions in gene regulation and chromatin structure, and their implications for various BC subtypes. Examining these facets, we seek to reveal the potential of non-coding DNA as a viable source of novel diagnostic markers and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966867774ecd7f0803dece977fed36246e346fe6" target='_blank'>
              The Role of Non-Coding Regions in Breast Cancer: From Gene Regulation to Therapeutic Implications
              </a>
            </td>
          <td>
            Hussein Sabit, Sarah Sobhy, Shaimaa Abdel-Ghany, Al-Hassan Soliman Wadan, Olubukola Ayodele, Yasser Albrahim, Hirendra Banerjee, Ahmed Elhashash, B. Arneth
          </td>
          <td>2025-09-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is a primary cause of therapeutic failure, driving tumor progression. Within tumors, diverse cell states coexist, maintained by a specific chromatin landscape that influences various cell functions, including cancer stemness. Among factors that induce chromatin changes affecting cell state fitness, DNA damage and its repair have emerged as significant contributors. This perspective examines recent advances that elucidate the interplay between DNA repair, epigenome, and cell plasticity. We discuss how epigenome affects DNA repair and, conversely, how DNA repair-induced chromatin changes influence cell plasticity. Finally, we discuss emerging concepts and highlight the therapeutic implications of these interconnected mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0841716149f708372bcf7e59a0f69e9d199da1" target='_blank'>
              The interplay of DNA damage, epigenetics and tumour heterogeneity in driving cancer cell fitness
              </a>
            </td>
          <td>
            Celia D Rouault, E. Charafe-Jauffret, C. Ginestier
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Retroelements in the human genome are silenced via multiple mechanisms, including DNA methylation, to prevent their potential mutagenic effect. Retroelement activity, demonstrated by their expression and somatic retrotransposition events, was shown to be deregulated in multiple tumors but not yet in leukemia. We hypothesized that treatment with hypomethylating agents, commonly used in myelodysplastic syndromes and acute myeloid leukemia, could lead to increased retroelement activity and somatic retrotranspositions, thus contributing to disease progression. To address this hypothesis, we induced the expression of ORF1p protein with hypomethylating agents in DAMI and HL‐60 myeloid cell lines. To study whether long‐term hypomethylating agent therapy induces somatic retrotranspositions, we analyzed (i) both cell lines treated for 4 weeks, and (ii) sequential samples from 17 patients with myelodysplastic syndrome treated with hypomethylating agents. Using a sensitive next‐generation sequencing (NGS)‐based method, no retroelement events were identified. To conclude, we show that although hypomethylating agents induce the expression of LINE‐1‐encoded proteins in myeloid cell lines, de novo somatic retrotransposition events do not arise during the long‐term exposure to these agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6777d30e90cc8e94bcf1ee3f59b0243f01da267f" target='_blank'>
              Hypomethylating agents increase L1 retroelement expression without inducing novel insertions in myeloid malignancies
              </a>
            </td>
          <td>
            S. Pavlova, H. Svozilová, Marcela Krzyžánková, Radim Sonnek, Anastasiya Volakhava, Anastasia Smirnova, Tatiana Grigoreva, Zuzana Jašková, Hana Synáčková, Dennis Wahl, Michaela Bilčíková, L. Červinek, Š. Pospíšilová, Ilgar Mamedov, K. Plevová
          </td>
          <td>2025-09-04</td>
          <td>Molecular Oncology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="


 Liquid biopsy analyzing cell free tumor DNA (ctDNA) from Cerebrospinal Fluid (CSF) is a feasible and relatively low-risk alternative to a tissue biopsy for diagnosis of Central Nervous System (CNS) tumors. CNS liquid biopsy is challenging due to molecular and clinical diversity of primary CNS tumors, the multitude of type of metastatic tumors and molecular drivers, and the lack of assays that distinguish tumors from non-neoplastic diseases. An integrated epigenetic and genomic analysis could provide simultaneous DNA methylation classification with mutational, copy number and gene fusion analysis. Current methylation techniques lead to DNA loss due to bisulfite conversion destruction. Enzymatic 5-base chemistries enable simultaneous high throughput genomic analysis and DNA methylation classification without DNA loss.



 We analyzed ctDNA from 60 CSF samples from patients with primary and metastatic CNS tumors. Library preparation was performed using TET-assisted pyridine borane sequencing (TAPS) chemistry (Watchmakers Genomics). Libraries were sequenced on the NOVASeq 2000 with 500 mil reads/sample. Fastq files were processed using NF-CaseReports pipeline, and coverage bias corrected using fragCounter and rPCA denoising. Methylation signals were extracted using Rastair, followed by integration with the Infinium MethylationEPIC v2.0 array probe set, facilitating downstream clustering analyses and clinical DNA methylation Classifier. Copy number analysis was performed with CBS, and structural variant, using GRIDSS.



 TAPS provides comprehensive genomic landscape analysis including mutation, genome-wide and focal copy number analysis, and gene fusions and DNA methylation classifier in the same assay. For example, we detected 1p19q losses in oligodendroglioma (N=3), trisomy of chromosome 7 (N=7) and loss of chromosome 10 (N=8), as well as high level amplifications of EGFR, MET in GBM (N=11) and ERBB2 (Her2) in metastatic breast cancer. Mutational analysis clearly classified IDH mutant (N=15) and wild-type (N=25) gliomas, based on detection of IDH1/2, ATRX, TP53 and TERT promoter mutations, amplifications, and CDKN2A/B, PTEN loss. Structural variant analysis detected EGFRvIII (N=3) and MET fusions (N=1) in GBM. Furthermore, TAPS enabled targeted analysis of the MGMT promoter concordant with methylation array. In pediatric tumors we detected MN1: : CXXC5 fusion in a case of astroblastoma and isochromosome 17 in Group 4 medulloblastoma, notably a tumor lacking somatic mutations. In 13 cases (21%), we did not detect any informative mutations or epigenetic changes, likely due to low level of CSF ctDNA.



 Integrated genomic-epigenetic ctDNA analysis using TAPS chemistry is highly sensitive liquid biopsy approach that provides simultaneous genetic and epigenetic analysis from CSF ctDNA and requires minimum ctDNA input.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c953510ee5c4758248718340692da6ec1d03a9d8" target='_blank'>
              P05.12.B ULTRASENSITIVE INTEGRATED GENOMIC-EPIGENETIC CTDNA CNS TUMOR ASSAY USING TET-ASSISTED PYRIDINE BORANE SEQUENCING (TAPS)
              </a>
            </td>
          <td>
            M. Snuderl, C. Fang, E. Freitag, A. Oviedo, C. Schroff, K. Hadi, A. Deshpande, J. Rafailov, H. Weiss, N. Dhasmana, D. Orringer, K. Wrzeszczynski, M. Imielinski, A. Miller
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 The genomic landscape of ovarian high grade serous carcinoma (HGSC) is typified by extensive genomic instability and heterogeneity, near-ubiquitous TP53 mutations, copy number aberrations, and defects in homologous recombination (HR) repair in approximately 50% of patients. Due to different mechanisms underlying therapy resistance and a lack of predictive biomarkers, matching best treatment modalities to patients with HGSC is challenging. Assessing genomic instability scores to indicate HR-deficiency (HRD) status is used clinically to guide optimal maintenance regimens including PARP-inhibitor treatment. Recently, we demonstrated the degree of phenotypic and genomic heterogeneity in advanced, high tumor burden HGSC patients. Here, we aim to show that the spatial inter-tumoral heterogeneity in HGSC expands to variability in whole genome duplication (WGD) states and timing, and in HRD-score derivation. Systematic mapping of tumor dissemination patterns was performed in chemotherapy-naïve advanced HGSC patients (n=22) following maximal-effort cytoreduction. Multi-site biopsies from 4-15 different anatomical sites were collected, and paired relapse samples were collected where possible (n=10). Germline and Tumor DNA was extracted (5 tumors per case, plus all relapse samples) and underwent whole genome sequencing (WGS, ∼30x depth). We observed spatial and temporal genomic heterogeneity for common HGSC features including mutations, gene amplifications (e.g. CCNE1 and MYC), and copy number and mutational signatures. Somatic, pathogenic loss-of-function TP53 mutations were observed in all tumors in the cohort. We observed discordance in HRDsum classification measures, with 4/22 patients presenting with mixed-HRD scores, i.e. both HRD-positive and HRD-negative scores in their tumors. In each case, the HRDsum scores had a mean value close to the threshold of 42 (range 35.8-45.8). HRD-negative or mixed-HRD status patients had poorer progression free (p=0.0026) and overall survival (p=0.0015) than HRD-positive patients. Cases with mixed states for WGD (no WGD, one WGD or second sub-clonal WGD) were observed in 41% of the cohort with variable timings (early vs late) within disseminated tumors across the cohort, in patients with HRD-negative and mixed HRD-status. The significant inter-tumoral genomic diversity observed in our analysis in measures of HRD and WGD status across the various tumor sites in our cohort of advanced HGSC patients highlights the challenges facing clinicians to derive accurate clinically actionable or prognostic information from a single tumor biopsy alone to direct patient treatment.



 Paula Cunnea, Nikki Burdett, Marc Lorentzen, Ahwan Pandey, James Clark, Katherine Nixon, Jennifer Ploski, Shirya Varghese, David Bowtell, Jonathan Krell, Elizabeth Christie2, Christina Fotopoulou. Evaluating spatial inter-tumor heterogeneity in whole genome duplication states and homologous recombination deficiency status in advanced ovarian high grade serous carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28933fb2fad0d299521abc6704fd16fc40c4a93c" target='_blank'>
              Abstract B004: Evaluating spatial inter-tumor heterogeneity in whole genome duplication states and homologous recombination deficiency status in advanced ovarian high grade serous carcinoma
              </a>
            </td>
          <td>
            P. Cunnea, Nikki L Burdett, Marc Lorentzen, A. Pandey, James Clark, K. Nixon, J. Ploski, Shirya Varghese, D. Bowtell, J. Krell, Elizabeth L. Christie, C. Fotopoulou
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a leading cause of cancer-related mortality, necessitating the development of novel therapeutic strategies. In the present study, we identified TP53-regulating kinase (TP53RK) as a critical regulator of CRC cell survival and proliferation using a custom clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 library screen targeting serine/threonine kinase-related genes. TP53RK was significantly overexpressed in CRC tissues and was correlated with copy number amplification. Functional validation revealed that TP53RK depletion induced DNA replication stress, apoptosis, and cell cycle arrest, independent from p53 status. Mechanistically, TP53RK stabilized cell division cycle 7 (CDC7), a key kinase regulating DNA replication origin activation, ensuring robust minichromosome maintenance complex protein (MCM) phosphorylation and replication fork progression. Disruption of the TP53RK-CDC7 axis led to reduced MCM2 enrichment at replication origins and impaired DNA replication dynamics. Moreover, TP53RK overexpression sensitized cells to DNA replication stress (aphidicolin) and CDC7 inhibition (XL413), highlighting its potential as a therapeutic strategy. These findings establish TP53RK as a pivotal regulator of DNA replication fidelity and genomic stability, thereby providing a promising therapeutic target for CRC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-21082-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a87356ada79d745ccc5da3c1105d2fc2922b6dd0" target='_blank'>
              Identifying TP53RK as a key regulator of colorectal cancer survival and a potential therapeutic target
              </a>
            </td>
          <td>
            Younghee Choi, Seowoo Park, Seojin Yang, Eun-ju Kim, Youngwon Cho, Hye-Ju Yang, Eugene C Yi, Sang-Hyun Song, Tae-You Kim
          </td>
          <td>2025-10-16</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Tumor suppressor NF1 is recurrently mutated in glioblastoma, leading to aberrant activation of Ras/rapidly accelerated fibrosarcoma (RAF)/MEK signaling. However, how tumor heterogeneity shapes the molecular landscape and efficacy of targeted therapies remains unclear. Here, we combined bulk and single-cell genomics of human somatic NF1-mutant, isocitrate dehydrogenase (IDH) wild-type glioblastomas with functional studies in cell lines and mouse intracranial tumor models to identify mechanisms of tumor heterogeneity underlying clinical outcome and MEK inhibitor response. Targeted DNA sequencing identified CDKN2A/B homozygous deletion as a poor prognostic marker in somatic NF1-mutant, but not NF1 wild-type, glioblastoma. Single-nucleus RNA sequencing of human patient NF1-mutant glioblastomas demonstrated that mesenchymal-like (MES-like) tumor cells were enriched for MEK activation signatures. Single-cell RNA-sequencing of mouse intracranial glioblastomas treated with the MEK inhibitor selumetinib identified distinct responses among tumor subpopulations. MEK inhibition selectively depleted MES-like cells, and selumetinib-resistant MES-like cells upregulated Ras signaling while resistant non-MES cells expressed markers of glial differentiation. Finally, genome-wide CRISPR interference screens validated Ras/RAF/MEK signaling as a key mediator of selumetinib response. Repression of the RAF regulator SHOC2 sensitized glioblastomas to selumetinib in vitro and in vivo, suggesting a synergistic treatment strategy. Taken together, these results highlighted the heterogeneity of NF1-mutant glioblastomas and informed future combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9b553a3b7f2afede17ca71f5d4515ccec597b2" target='_blank'>
              Tumor heterogeneity underlies clinical outcome and MEK inhibitor response in somatic NF1-mutant glioblastoma
              </a>
            </td>
          <td>
            S. Pan, K. Mirchia, E. Payne, S. J. Liu, N. Al-Adli, Zain Peeran, Poojan D. Shukla, Jacob S. Young, Rohit Gupta, Jasper Wu, J. Pak, T. Ozawa, Brian Na, Alyssa T Reddy, Steve E Braunstein, Joanna J. Phillips, Susan Chang, D. Solomon, A. Perry, D. R. Raleigh, Mitchel S Berger, Adam Abate, H. Vasudevan
          </td>
          <td>2025-09-23</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) has been suggested as a key factor of tumor heterogeneity at the genomic and transcriptional levels, via unequal segregation and ecDNA hub formation, respectively. In this issue of Cancer Discovery, two studies further expand our understanding of the roles of ecDNA in enhancing tumor heterogeneity at the phenotypic and spatiotemporal levels, as well as in providing a strong selective advantage during tumor evolution. See related article by Montuori et al., p. 2054 See related article by Noorani et al., p. 2078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cba883c44a904170e7fcec05ae3700dab4a7298" target='_blank'>
              Extrachromosomal DNA: A Trusted Path to Tumor Heterogeneity.
              </a>
            </td>
          <td>
            Emmanuel E Korsah, Noah A Dusseau, Eunhee Yi
          </td>
          <td>2025-10-06</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Human papillomaviruses (HPVs) maintain their genomes as minichromosomes in the nuclei of infected keratinocytes. This study investigates the association of HPV31 genomes with host chromatin using both HiC and 4C-seq chromosome conformation capture techniques. We show that HPV31 genomes preferentially associate with transcriptionally active A compartments of host chromatin, regions of open chromatin defined by ATAC-seq, and super-enhancers defined by Brd4 and H3K27ac ChIP-seq. The viral genome association sites were also highly correlated with genomic loci previously identified as common HPV integration sites in cervical cancers. Recent studies have shown that transcriptionally active sites are prone to dsDNA breaks, and we find a strong correlation among dsBREAK datasets with transcriptionally active and open regions of host chromatin and the HPV31 genome association sites defined in our study. These findings suggest that HPV genomes associate with cellular transcriptional epicenters to maintain active viral gene expression during persistent infection, but also indicate that the susceptibility of these regions to dsDNA breaks could explain their propensity for viral DNA integration in HPV-associated cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38ecc91def6b800389ed078be4c219004519589f" target='_blank'>
              Human papillomavirus genomes associate with active host chromatin during persistent viral infection
              </a>
            </td>
          <td>
            Alix Warburton, T. E. Markowitz, JJ L. Miranda, Kinjal Majumder, Alison A. McBride
          </td>
          <td>2025-09-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Comprehensive epigenomic studies in multiple myeloma (MM) that unravel the connections between major epigenetic regulators, their intertwined collaboration and the potential of combinatorial targeting remain limited. Utilizing ChIP-seq, ATAC-seq, RNA-seq, and DNA methylation (DNAme) data, we generated whole-genome chromatin annotations from normal plasma cells and MM patients, revealing epigenomic re-configuration affecting downstream genes involved in tumour growth and survival. Primary MM samples showed global DNA hypomethylation but site-specific hypermethylation was observed at transcription start sites, promoters, and enhancers. Moreover, increased deposition of H3K27me3 was observed in clinically relevant functional chromatin clusters. Combined EZH2 and DNMTs inhibition resulted in extensive epigenomic alterations activating apoptosis and cell cycle genes, leading to increased G2/M arrest and apoptosis in MM cell lines. Our findings provide novel insights into the role of epigenetic gene silencing in MM tumorigenesis and the interplay between the Polycomb repressive complex 2 and DNAme. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17093-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1dfe4ebbad2d35b5cf5e512e8911342a3522d08" target='_blank'>
              Combinatorial DNMTs and EZH2 inhibition reprograms the H3K27me3 and DNAme-mediated onco-epigenome to suppress multiple myeloma proliferation
              </a>
            </td>
          <td>
            Alba Atienza Párraga, P. Nylund, K. Diamanti, Berta Garrido-Zabala, Stefania Iliana Tziola, Louella Vasquez, P. Pyl, D. Raykova, Aron T. Skaftason, Anqi Ma, Jian Jin, J. Martín-Subero, Fredrik Öberg, Elke De Bruyne, Jan Komorowski, Helena Jernberg Wiklund, A. Kalushkova
          </td>
          <td>2025-08-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Oncomicrobes are estimated to cause 15% of cancers worldwide. When cancer whole-genome sequencing (WGS) data are collected, the microbes present are also sequenced, allowing the investigation of potential etiological and clinical associations. Interrogating the microbial community for 8908 patients encompassing 22 cancer types from the Genomics England WGS dataset revealed that only colorectal tumors exhibited unmistakably distinct microbial communities that can reliably be used to distinguish anatomical site [positive predictive value (PPV) = 0.95]. This pattern was validated in two independent datasets. Potential clinical relevance uncovered by our analyses included accurate detection of alphapapillomaviruses [human papillomavirus (HPV)] in oral cancers, when compared with current clinical standards, and the detection of rare, highly pathogenic viruses such as human T-lymphotropic virus-1. Biomarker investigations demonstrated statistically significant associations (P < 0.05) between a subset of anaerobic bacteria and survival in certain subtypes of sarcoma. Our results contradict previous claims that each cancer type has a distinct microbiological signature but highlight the potential value of microbial analysis for certain cancers as WGS of tumor samples becomes common in the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82e28dcd99723ccc30fe571b2d59df15e8ee150f" target='_blank'>
              The landscape of microbial associations in human cancer.
              </a>
            </td>
          <td>
            Abraham Gihawi, H.M. Wood, Jeremy Clark, Justin O'Grady, Rosalind A. Eeles, David C Wedge, G. M. Jakobsdottir, G. Magiorkinis, A. Schache, Liam Masterson, Matt Lechner, Tim R. Fenton, Terence M Jones, Adrienne M Flanagan, Solange De Noon, Alex Rubinsteyn, Rachel Hurst, Colin S Cooper, Daniel S. Brewer
          </td>
          <td>2025-09-03</td>
          <td>Science translational medicine</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="Primary prostate cancer presents with multifocal lesions and unpredictable clinical behavior, posing significant challenges for effective prognosis. To address this, we investigate the epigenomic landscape of prostate tumor biopsies from 25 treatment-naïve male patients by analyzing chromatin compartmentalization patterns. Our analysis reveals two distinct molecular subtypes: one with a Low Degree of Decompartmentalization (LDD) and another with a High Degree of Decompartmentalization (HDD). Here we show that the HDD subgroup exhibits extensive chromatin reorganization associated with diminished oncogenic potential. This subtype shows repression of molecular pathways involved in extracellular matrix remodeling and cellular plasticity. From this distinction, we derive an 18-gene transcriptional signature capable of differentiating HDD from LDD cases. Importantly, we validate the prognostic relevance of this signature in multiple independent cohorts totaling over 900 patients. Our findings suggest that epigenetic-derived signature at the time of diagnostic biopsy can offer a powerful tool for risk stratification in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/696cb38bd331720bed277dcfa9f4701c78f389c7" target='_blank'>
              Chromatin remodeling restrains oncogenic functions in prostate cancer
              </a>
            </td>
          <td>
            V. Rosti, Giovanni Lembo, C. Petrini, Francesca Gorini, Roberto Quadri, C. Cordiglieri, Margherita Mutarelli, Elisa Salviato, Elisabetta Casari, Emanuele Di Patrizio Soldateschi, Emanuele Montanari, G. Albo, Francesco Ripa, Alessandra Fasciani, Mariacristina Crosti, E. De Lorenzis, M. Maggioni, V. Vaira, Maria Vivo, Francesco Ferrari, Chiara Lanzuolo
          </td>
          <td>2025-10-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Chromatin, the spatial organizer of genomic DNA, is hierarchically folded into higher-order structures to facilitate DNA compaction, enabling genome surveillance. Understanding the organization and function of the three-dimensional (3D) genome is critical to profile chromatin accessibility and functional interactions that govern gene regulation across multiple biological processes, including aging and one of its hallmarks, cellular senescence. Cellular senescence constitutes a defensive stress response to various intrinsic and extrinsic stimuli, preserving cellular and organismal homeostasis through a generally irreversible cell cycle arrest. In this review article we discuss epigenetic alterations occurring to DNA and chromatin that drive and fuel the onset of this complex phenomenon. As such, we describe major large-scale chromatin events, including the formation of higher-order chromatin structures and the 3D spatial alterations of the genome that occur during senescence. We also discuss global heterochromatin loss, deficiencies in nuclear lamins, the depletion of core histones and their modifications, as well as the epigenetic regulation of the senescence-associated secretory phenotype (SASP), all of which serve key roles in the epigenome of senescent cells. To clearly demonstrate the significance of epigenetic modifications, data from a computational meta-analysis are presented, aiming to further underpin key epigenetic mechanisms occurring in senescent cells. Last, we highlight promising epigenetic modulators implemented in therapeutic strategies for senescent cell detection and elimination, possibly leading to significant clinical advances against various age-related diseases as well as the delay and prevention of the aging onset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fd36c31f599c330b53d9bff8eb9f2a73d497006" target='_blank'>
              Interrogating the regulatory epigenome of cellular senescence
              </a>
            </td>
          <td>
            Dimitris-Foivos Thanos, Orestis A. Ntintas, Emmanouil Athanasiadis, A. Papaspyropoulos, R. Petty, V. Gorgoulis
          </td>
          <td>2025-08-31</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee3a629089847ef194ea5479a4ede2bc91a09e2d" target='_blank'>
              Detection and monitoring of translocation renal cell carcinoma via plasma cell-free epigenomic profiling.
              </a>
            </td>
          <td>
            S. Garinet, Karl Semaan, Jiao Li, Ananthan Sadagopan, J. Canniff, Noa Phillips, K. Klega, Medha Panday, H. Savignano, Matthew P Davidsohn, Kevin Lyons, Alessandro Medda, Prateek Khanna, M. Achom, Prathyusha Konda, Brad Fortunato, Rashad Nawfal, Razane El Hajj Chehade, Ze Zhang, Jillian O’Toole, J. Horst, Dory A Freeman, Rachel Trowbridge, C. Chau, William D. Figg, Jacob E. Berchuck, Brian D Crompton, J-H. Seo, T. Choueiri, M. Freedman, S. Baca, Srinivas R Viswanathan
          </td>
          <td>2025-09-04</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background/Aim Cell lines serve as valuable in vitro models to study altered cellular signaling pathways, to identify mutations in key oncogenic genes, and to test potential antitumor drugs. The Jurkat cell line, for example, has provided important information about various signaling pathways in lymphoblastic leukemia, establishing most of what is currently known about T-cell receptor (TCR) signaling. However, many aspects of the genome modification of this cell line have not yet been analyzed. To identify genes of potential biological and clinical relevance in acute T-lymphoblastic leukemia (T-ALL), we performed an array comparative genomic hybridization (aCGH) approach on the widely used Jurkat cell line and examined the association of the detected copy number alterations (CNAs) with cancer hallmarks and T-ALL pathogenesis. Materials and Methods Cells were harvested by using trypsin/EDTA from culture flasks to extract genomic DNA. aCGH experiments were performed on an Agilent microarray platform using the SurePrint G3 Cancer CGH + SNP Microarray 4×180 K. Functional enrichment analysis of all CNAs was performed with the R package g:Profiler2. The association of these alterations with key cancer hallmarks was analyzed using the Cancer Hallmarks web-tool. Results Our analysis revealed several novel CNAs, including losses at 5p15.2, 6q27, 10q22.2, 14q11.2, 18q11.2-q12.1, and Xp22.33, as well as gains at 2p11.2, 7p21.2, 7q21.2 and 18p11.32. Genes within these regions were associated with important oncogenic pathways, including ‘sustained proliferative signaling’, ‘tumor suppressor evasion’, and ‘angiogenesis promotion’. Conclusion These findings suggest that Jurkat cells may serve as a valuable model for identifying new targets for cancer research. Further studies are required to confirm the phenotypic implications of these variants, which may open new avenues for exploring the functional impact of these alterations and their potential role in the development of therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7da88c0ba8ab646537d572f0801ee5d18aa14a5e" target='_blank'>
              High-Resolution aCGH Analysis of the Jurkat Cell Line: Copy Number Alterations and Their Association With Cancer Hallmarks
              </a>
            </td>
          <td>
            Layssa Mota DA COSTA, Wallax Augusto SILVA FERREIRA, C. B. Machado, Carolina Koury NASSAR AMORIM, Ana Virgínia VAN DEN BERG, M. O. de Moraes Filho, M. E. D. de Moraes, Lucas Eduardo Botelho de Souza, A. Khayat, E. H. C. de Oliveira, C. A. Moreira-Nunes
          </td>
          <td>2025-08-29</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Over 95% of ependymomas that arise in the cortex are driven by a gene fusion involving the zinc finger translocation-associated (ZFTA) protein. Here, using super-resolution and lattice light-sheet microscopy, we demonstrate that the most frequent fusion variant, ZFTA–RELA (ZR), forms dynamic nuclear condensates that are required for oncogene expression and tumorigenesis. Mutagenesis studies of ZR reveal a key intrinsically disordered region (IDR) in RELA that governs condensate formation. Condensate-modulating IDR mutations introduced into ZR impaired its genomic occupancy at oncogenic loci and inhibited the recruitment of transcriptional effector proteins, such as MED1, BRD4 and RNA polymerase II. Using nuclear magnetic resonance spectroscopy, we examined the DNA-binding residues of the critical zinc finger (ZF1) found in ZR and characterized their significance for condensate formation, genomic binding and oncogene activation. We generated synthetic ZFTA fusion proteins where IDRs from known condensate-forming proteins were grafted into ZR. Synthetic ZFTA fusion oncoproteins utilizing IDRs from EWS and FUS restored condensate formation, oncogene transcription and tumour initiation in mice. These findings provide key insights into the oncogenic mechanism of ZR and the importance of IDR acquisition in fusion oncoproteins in brain cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d5d4e3905b6133ceff9f0311d9fcea6d57257e" target='_blank'>
              Synthetic ZFTA fusions pinpoint disordered protein domain acquisition as a mechanism of brain tumorigenesis
              </a>
            </td>
          <td>
            A. Arabzade, H. K. Shirnekhi, S. Varadharajan, S. M. Ippagunta, A. Phillips, N. Laboe, D. W. Baggett, Wahiduzzaman, M. Jo, T. Zheng, R. Pathak, D. Gee, D. Bhimsaria, H. Wu, X. Gao, J. Liu, E. Emanus, A. Bland, A. Kardian, A. Hancock, B. Holcomb, T. Wright, T. Bugbee, H. Sun, M. Zhai, E. Caesar, M. Park, S. Tripathi, A. Shirinifard, K. Lowe, A. Khalighifar, R. A. Petersen, S. King, D. Stabley, A. Pitre, G. Campbell, C-G Park, W. T. Freyaldenhoven, B. Chandra, Y. Xia, E. Bonten, A. Achari, S. Kandikonda, A. Carisey, S. Pounds, J. Xu, D. W. Ellison, B. Deneen, K. Bertrand, R. Kriwacki, S. C. Mack
          </td>
          <td>2025-08-27</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cee7ec76a900e6d0548a03ab74a2803cf82d9035" target='_blank'>
              Genomic Divergence Between Matched Primary and Metastatic Tumors Across Cancer Types: A Pan-Cancer Analysis of 5,692 Samples
              </a>
            </td>
          <td>
            Y. Ergun
          </td>
          <td>2025-08-26</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Lymphoma is a highly heterogeneous malignancy, demanding accurate and precise diagnosis to guide the selection of the appropriate treatment for optimal outcome. Copy number aberration (CNA) has been suggested to play an important role in the occurrence and development of lymphoma and thus can be explored as biomarker to improve disease management. It is believed that CNAs in variable forms and complexities can be triggered by both exogenous (eg viral infection and ionizing radiation) and endogenous factors (eg genetic predisposition and evolutionary forces). However, conventional cytogenetic methods have limitations to detect all types of CNAs with accuracy and adequate details. The emergence of new technologies, including fluorescence in situ hybridization (FISH), chromosome microarray analysis (CMA), and especially next-generation sequencing (NGS) has made significant progress in the identification and characterization of CNAs or CNA-related genomic aberrations. Accumulating data addressing molecular insights and clinical implications have provided us more theoretical and experimental support for its clinical translation. Currently, while only limited number of CNAs or CNA-related genomic variation, such as deletion/amplification of DNA segments, have been documented in major guidelines or consensus for their clinical potential in lymphoma, more CNAs remain to be further characterized and/or discovered for their clinical relevance. Taking together, with available and upcoming evidence, CNA should play an important role as a diagnostic and prognostic biomarker while integrated with the current settings in lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56be82adf67a41f68e1acf08975fb72e997dedc3" target='_blank'>
              Clinical Potential of Copy Number Aberration as a Diagnostic and Prognostic Biomarker in Lymphoma
              </a>
            </td>
          <td>
            Xudong Zhang, Zailin Yang, Susu Yan, Minning Zhan, Shichun Tu, Weihong Ren, Yao Liu, Zun‐M. Zhu
          </td>
          <td>2025-06-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Human positive coactivator 4 (PC4) is a highly abundant non-histone chromatin protein involved in diverse cellular processes, including transcription regulation, genome organization, autophagy, B-cell differentiation, neurogenesis, DNA repair, etc. Most PC4 is phosphorylated in cells, which interacts with core histones and the linker histone H1 to confer the compact heterochromatin state of the genome. Knocking down PC4 at both cellular and organismal levels leads to significant chromatin decondensation, altered epigenetic landscape, enhanced autophagy, and increased DNA damage susceptibility. Here, we report that besides p300, PC4 also gets acetylated by DNA repair, facilitating lysine acetyltransferase KAT5 (Tip60) at a specific lysine residue (PC4K80) when the cells are subjected to DNA damage. The vulnerability of DNA in PC4 devoid cells was substantially reduced by reintroducing wild-type PC4 to the cells but not the mutant PC4 (PC4K80R), defective in KAT5-mediated acetylation. High-resolution microscopy techniques, including transmission electron microscopy and atomic force microscopy, are employed to visualize chromatin structural changes in response to DNA damage and repair in a Tip60-mediated PC4 acetylation-dependent manner. Presumably, KAT5-mediated acetylation of PC4 at K80 residue facilitates access to the damaged DNA by altering chromatin structures at damage sites, thus promoting DNA repair. This process could be highly significant both in cancer and in aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c83d5690dbc70212ddc740cb45975fd69abb4f3" target='_blank'>
              KAT5-mediated acetylation of PC4 facilitates DNA repair by promoting chromatin reorganization
              </a>
            </td>
          <td>
            Aa yushi A gr a w al, eta Sikder, Siddharth Singh, Rakesh Kumar, Nikhil Pallaprolu, Kalyan Mitra, Sourav Mondal, Rupa Mukhopadhy, anta Sarkar, Ramaling am Per aman, Ra vic, Tapas K. Kundu
          </td>
          <td>2025-09-23</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Alterations in 3D chromatin conformation may disrupt the interplay between promoters and distal enhancers. How gene regulatory circuits are reshaped during ionizing radiation-induced tumorigenesis remains unclear, and little is known about the mediators that drive these processes. To decipher the chromatin alterations in radiation-induced lung cancer, we performed ATAC-seq, RNA-seq and Hi-C analyses of human bronchial epithelial cells and corresponding radiation-induced malignantly transformed cell lines. We found that this malignant transformation is accompanied by chromatin switching from the inactive B compartment to the active A compartment, an increased number of TADs and gained ATAC-seq peaks that mediate new distal chromatin contacts. We identified tumour protein 63 (TP63) as a mediator of new chromatin-accessible sites that anchor tumour-specific chromatin contacts in radiation-induced tumour cells. A TP63-mediated accessible chromatin site anchors a tumour-specific TAD boundary and multiple tumour-specific chromatin loops, which might underlie MYC oncogene activation during malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b8217e99ba9977673505cae49c7863f218434b" target='_blank'>
              TP63 mediates the generation of tumour-specific chromatin loops that underlie MYC activation in radiation-induced tumorigenesis
              </a>
            </td>
          <td>
            Qiaowei Liu, Shuai Jiang, Xiang Xu, Kang Xu, Yawen Luo, Zongyuan Yu, Meida Xiang, Zhouyang Xu, Lijie Wang, Sujie Zhang, H. Tao, Xuhui Yang, Chongbo Jiang, Xiaoguang Qi, Hao Li, Yiming Gao, Yao Li, Shihui Fu, Pingkun Zhou, Xiaochen Bo, Hebing Chen, Xiaohua Chen, Yi Hu
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="IDH-mutant gliomas represent a subtype of diffuse gliomas that primarily affect patients in early to mid-adolescence. These tumors are classified into three distinct CNS WHO grades of malignancy. Accurate grading is essential for selecting an appropriate treatment maximizing anti-tumor efficacy while minimizing adverse effects. However, grading of oligodendrogliomas with 1p/19q codeletion currently relies on qualitative tumor characteristics that may be influenced by observer subjectivity, sampling bias, and tumor heterogeneity. This study aimed to explore DNA methylation-based tumor deconvolution into latent methylation components (LMCs) to evaluate their potential as objective grading tools in a cohort of 137 IDH-mutant gliomas. LMCs were analyzed in relation to malignancy markers, cellular composition, and underlying methylation signatures of the chromatin landscape. Glioma subtypes were associated with distinct LMCs. Two LMCs correlated with higher cellular density and advanced epigenetic age as well as with microvascular proliferation, necrosis, and epigenetically defined high-grade astrocytoma. The epigenetic patterns defining high-grade astrocytoma or oligodendroglioma, respectively, were similar. Higher-grade oligodendrogliomas, identified by LMC-based grading, were associated with more copy number alterations. Among patients of an external cohort who died during the assessment period, higher LMC1 proportions were associated with poorer overall survival. Therefore, LMCs hold the potential to support IDH-mutant glioma grading by incorporating objective epigenetic markers. Supplementary Information The online version contains supplementary material available at 10.1007/s00401-025-02926-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d16c74139be26cefe8b9fdf9ca1fbad5b85b836" target='_blank'>
              DNA methylation analysis reveals an epigenetic signature distinctive of high-grade oligodendroglioma
              </a>
            </td>
          <td>
            Katharina J. Weber, M. Dettki, M. Münzberg, P. Zeiner, Marie-Thérèse Forster, Eike Steidl, I. Divé, Patrick N Harter, Michael Scherer
          </td>
          <td>2025-08-23</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the predominant subtype of ovarian cancer and is characterized by a high rate of relapse after platinum-based chemotherapy. Herein, we present a comprehensive analysis of 111 Korean HGSOC samples using next-generation sequencing technology to elucidate their transcriptomic landscapes. Our investigation revealed the existence of four distinct transcriptional subtypes of ovarian cancer: immunoreactive, mesenchymal, proliferative, and differentiated, which is comparable to those of TCGA HGSOC transcriptional subgroups. Each subtype exhibited unique correlation networks and their immune cell composition was computationally determined. Notably, the immunoreactive cluster displayed the highest immune score, even in the context of pan solid-cancer types, accompanied by heightened expression of CD4+ and CD8+ T cells (P < 0.05), along with notable associations with neutrophil degranulation and antigen presentation pathways (FDR < 0.01). Conversely, the differentiated cluster demonstrated immunodepleted characteristics, featuring an elevated proportion of overexpressed cancer-germline antigens. We also identified several cancer-germline HGSOC antigens that could be further investigated as potential targets for immunological intervention in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffdfd9df4e6bf6e48b7483936ad622c3c2f2e59a" target='_blank'>
              Integrative transcriptomic analysis of Korean high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Hyondeog Kim, Youngwook Kim, Juyeong Park, Dong-Min Shin, Ji Hyun Kim, Madhawa Gunathilake, So Young Kwon, Junyoung Shin, Chong Woo Yoo, S. Park, Myong Cheol Lim, Jeongseon Kim
          </td>
          <td>2025-09-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="ABSTRACT Liquid biopsy represents a transformative approach in oncology, enabling noninvasive disease detection and monitoring through epigenetic signals in circulating tumor DNA (ctDNA), nucleosomes, and noncoding RNAs. Tumor initiation is driven by epigenetic modifications, including DNA methylation, histone alterations, and dysregulated noncoding RNAs, which disrupt gene regulation, cell cycle control, DNA repair, and metastatic processes. This review systematically examines recent evidence on DNA methylation, histone marks (e.g., H3K27me3, H3K18ac), and noncoding RNAs (miRNAs, lncRNAs) as biomarkers for early cancer detection, prognosis, and therapeutic response. Particular focus is placed on aberrant DNA methylation (e.g., hypermethylation of CDKN2A, RASSF1A) and altered histone modifications (e.g., EZH2‐mediated silencing) as indicators of tumor heterogeneity and evolution. Stable and specific in biofluids, noncoding RNAs such as oncogenic miR‐21, tumor‐suppressive miR‐34a, and metastasis‐associated MALAT1/HOTAIR further enhance clinical applicability. Recent detection methods, including bisulfite sequencing, ChIP‐seq, and RNA‐seq, have advanced biomarker profiling, though challenges remain in standardization and low‐abundance detection. With over 12 active clinical studies validating their utility, integration of epigenetic markers with AI and multiomics holds promise for individualized, dynamically guided oncology care. Future innovations, such as chromatin accessibility analysis and cfDNA fragmentation profiling, may further refine diagnostic precision and therapeutic monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77f80572fe0354408f2985b22c2727d7355f5dac" target='_blank'>
              Transforming Cancer Diagnostics: The Emergence of Liquid Biopsy and Epigenetic Markers
              </a>
            </td>
          <td>
            Debalina Saha, Pritam Kanjilal, Mandeep Kaur, Soumya V. Menon, Ayash Ashraf, M. Kumar, T. Alqahtani, Shikha Atteri, D. Uti, Bikram Dhara
          </td>
          <td>2025-09-01</td>
          <td>MedComm</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b332eb177c3231afc269f28608f60924be826cc" target='_blank'>
              Pan-cancer analysis of RNA expression signatures associated with cancer tissue architecture.
              </a>
            </td>
          <td>
            Megumi Mogi, Katsushige Kawase, Hiroyoshi Y Tanaka, Suguru Miyata, Satoko Takemoto, Hideki Ikeda, Eri Katayama, Yuki Nakamura, Ryuta Kojima, Takao Morinaga, Shihori Tanabe, S. Yonekura, M. Kano, Toyoyuki Hanazawa, Masahito Kawazu
          </td>
          <td>2025-10-17</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Cancer genomes frequently carry apolipoprotein B mRNA editing catalytic polypeptide-like (APOBEC)-associated DNA mutations, suggesting APOBEC enzymes as innate mutagens during cancer initiation and evolution. However, the pure mutagenic impacts of the specific enzymes among this family remain unclear in human normal cell lineages. Here, we investigate the comparative mutagenic activities of APOBEC3A and APOBEC3B, through whole-genome sequencing of human normal gastric organoid lines carrying doxycycline-inducible APOBEC expression cassettes. Our findings demonstrate that transcriptional upregulation of APOBEC3A leads to the acquisition of a massive number of genomic mutations in just a few cell cycles. In contrast, despite clear deaminase activity and DNA damage, APOBEC3B upregulation does not generate a significant increase in mutations in the gastric epithelium. APOBEC3B-associated mutagenesis remains minimal even in the context of TP53 inactivation. Further analysis of the mutational landscape following APOBEC3A upregulation reveals a detailed spectrum of APOBEC3A-associated mutations, including indels, primarily 1 bp deletions, clustered mutations, and evidence of selective pressures acting on cells carrying the mutations. Our observations provide a clear foundation for understanding the mutational impact of APOBEC enzymes in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ca9c435f5c5a8910902a03b815334d801814d13" target='_blank'>
              APOBEC3A drives deaminase mutagenesis in human gastric epithelium
              </a>
            </td>
          <td>
            Yohan An, Ji-Hyun Lee, Joonoh Lim, J. Youk, Seongyeol Park, Ji-Hyung Park, Kijong Yi, Taewoo Kim, Chang Hyun Nam, Won Hee Lee, Soo A Oh, Yoo Jin Bae, Thomas M. Klompstra, Haeun Lee, Jinju Han, Junehwak Lee, Jung Woo Park, Jie-Hyun Kim, Hyunki Kim, Hugo Snippert, B. Koo, Young Seok Ju
          </td>
          <td>2025-08-26</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background/Objectives: Short inverted repeats (SIRs) are abundant DNA motifs capable of forming secondary structures, such as hairpins and cruciforms, that can induce genome instability. However, their mutational consequences in cancer, particularly in osteosarcoma (OS), remain largely unexplored. Methods: In this study, we systematically identified over 5.2 million SIRs in the human genome and analyzed their mutational patterns across six common cancer types. Results: We found that increased small insertion and deletion (INDEL) density within SIR spacer regions represents a consistent feature across cancers, whereas elevated single nucleotide variant (SNV) and structural breakpoint density is cancer-type specific. Integrating whole-genome sequencing data from 13 OS patients, we found that both SNVs and INDELs are significantly enriched within SIR spacer regions in OS. Notably, genomic regions with higher SIR density tend to accumulate more somatic mutations, suggesting a link between SIR abundance and local genome instability. SIR-associated mutations frequently occur in oncogenes and tumor suppressor genes, including TP53, NFATC2, MECOM, LRP1B, RB1, CNTNAP2, and PTPRD, as well as in long non-coding RNAs. Mutational signature analysis further suggests that defective DNA mismatch repair and homologous recombination may act in concert with SIR-induced DNA structural instability to drive OS development. Conclusions: Our findings highlight SIRs as mutational hotspots and potential drivers of osteosarcoma pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3071c0367ebe9199d637aacf953a87bc0377296" target='_blank'>
              Short Inverted Repeats as Mutational Hotspots and Putative Drivers of Genome Instability in Osteosarcoma
              </a>
            </td>
          <td>
            Minghua Li, Chun Liang
          </td>
          <td>2025-10-14</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Introduction The epigenetic silencing of O6-methylguanine DNA methyltransferase (MGMT) is associated with reduced DNA repair capacity, carcinogenesis and increased sensitivity to alkylating chemotherapy. However, the biological role and clinical significance of MGMT overexpression in cancer remains poorly understood. Methods Using multiplexed quantitative immunofluorescence we measured the localized levels of MGMT protein, γH2AX and CD8+ T cells in multiple retrospective colorectal cancer (CRC) cohorts. Genomic and transcriptomic features of selected cases were also studied with whole exome DNA sequencing and genome-wide methylation analysis. MGMT-methylated human CRC cells SW620 were transfected with an MGMT-containing plasmid and co-cultured with allogeneic peripheral blood mononuclear cells. Results A subset of CRCs showed MGMT protein upregulation associated with lower γH2AX, reduced CD8+ tumor infiltrating lymphocytes (TILs), mismatch repair proficient (pMMR) status and shorter survival. CD8+ TILs were more distant from MGMT-expressing cells than MGMT-negative cells and the MGMT promoter methylation status did not highly correlate with MGMT protein levels in CRC. In genomic/transcriptomic analysis, high MGMT expression was associated with a lower nonsynonymous somatic mutational burden, higher transition-to-transversion mutation ratio, increased deleterious TP53 variants and distinct transcriptomic profiles. The exogenous expression of MGMT in SW620 CRC cells reduced the number of spontaneous nonsynonymous mutations, reproduced mutational features of MGMT-high CRC and limited the in vitro T-cell-mediated killing of malignant cells induced by proinflammatory cytokines in tumor/immune cell co-cultures. Conclusions MGMT overexpression identifies a previously undescribed subset of CRCs with distinct biological and clinical properties including reduced mutagenesis, adaptive immune evasion, predominantly pMMR phenotype and aggressive clinical course. Direct, quantitative assessment of MGMT protein expression using spatially resolved analysis is more reliable than inference of MGMT expression by promoter methylation status in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43cc0eaa491cd34a4c5b928fccd34ad143431db3" target='_blank'>
              High MGMT expression identifies aggressive colorectal cancer with distinct genomic features and immune evasion properties
              </a>
            </td>
          <td>
            J. Zhang, B. K. Rajendran, Shruti S Desai, Joanna Gibson, Jassim DiPalermo, Patricia LoRusso, Yong Kong, Hongyu Zhao, Michael Cecchini, K. Schalper
          </td>
          <td>2025-09-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is characterized by rapid progression, therapeutic resistance, and a profoundly immunosuppressive tumor microenvironment. Emerging evidence suggests that endoplasmic reticulum (ER)-associated macromolecules play critical roles in tumor adaptation. In this study, we performed a multi-omics investigation of orosomucoid-like protein 2 (ORMDL2), a conserved ER membrane protein involved in sphingolipid biosynthesis and ER stress regulation, and uncovered its regulatory functions in GBM progression. Transcriptomic analyses across The Cancer Genome Atlas (TCGA), and Chinese Glioma Genome Atlas (CGGA) revealed elevated ORMDL2 expression in GBM tissues which causes poor prognosis. The MetaCore pathway and Gene Set Enrichment Analysis (GSEA) identified ORMDL2's involvement in antigen presentation via a major histocompatibility complex I (MHC class I), unfolded protein response (UPR), and mitochondrial apoptotic signaling. Single-cell RNA-sequencing data and the Human Protein Atlas showed ORMDL2 enrichment in tumor stromal cells. Pharmacogenomic correlation via the Genomics in Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) database suggested that ORMDL2 expression was associated with resistance to DNA damage response inhibitors such as etoposide, doxorubicin, talazoparib, and might interact with sphingolipid-targeting compounds. Collectively, our findings establish ORMDL2 as a multi-functional macromolecular regulator of immune suppression and therapeutic resistance in GBM, providing new mechanistic insights and potential targets for translational medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5a5f2b3cb520c8099ca5e5eaf8e443e9c0dbc4" target='_blank'>
              Targeting ORMDL2 in glioblastoma through integrated landscape of high-throughput sequencing and pharmacogenomic analysis
              </a>
            </td>
          <td>
            Fitria Sari Wulandari, Chih-Yang Wang, Sachin Kumar, Ngoc Phung Ly, Juan Lorell Ngadio, Dahlak Daniel Solomon, Do Thi Minh Xuan, Gabriela Natasha Sutandi, Hung-Yun Lin, Hui-Ru Lin, Yung-Kuo Lee, Hao-Chien Yang, Ming-Cheng Tsai, Ching-Chung Ko
          </td>
          <td>2025-09-27</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains a devastating disease with a median survival of 15 months. Its molecular definition illustrates underlying unstable genomes with copy number variation (chr 7+/10-), epigenetic modification in DNA repair enzymes (MGMT methylation), and changes to telomere maintenance genes (TERT or ATRX), all recognised as key factors in gliomagenesis and patient outcomes. Telomeres protect chromosome-ends from aberrant DNA damage responses (DDR), and telomere erosion has been shown to drive such genome instability and clonal evolution in several cancers. As more innovative platform trial designs emerge, identifying novel markers to better stratify patients and predict response to targeted therapies is a priority. We conducted single molecule telomere length (TL) analysis and long-read fusion sequencing in GBM patients and related this to clinical outcome.



 146 adults with molecular GBMs had tumour TL assessed with high-throughput single telomere length analysis. Assays for extrachromosomal c-circles and TERT promoter mutation status were performed, to assess underlying telomere maintenance mechanisms. Uni- and multi-variable analysis was conducted to assess tumour TL impact on OS and PFS. Single-molecule PCR was used to identify telomere fusions which were further characterised with long-read nanopore sequencing, including in patients with paired samples from index surgery and at recurrence.



 GBMs displayed shortened telomeres (median TL 4.0kb), unrelated to age at diagnosis or MGMT methylation status. Short TL was negatively predictive for OS on multivariable analysis (HR 4.81, p=0.009). This impact was more pronounced in patients with gross total resection at index surgery (HR 21.55, p=0.0074). Long read Fusion-Seq demonstrated widespread telomere fusions in GBMs and revealed microhomology, telomeric deletions and insertions of genomic loci, including from known protein-coding genes with driver mutations, long non-coding RNA, canonical telomeric repeats, and subtelomeric regions. This mutational spectrum suggests utilisation of specific DNA repair pathways in mediating telomere fusion.



 This study provides mechanistic evidence of dysfunctional telomeres in GBMs, with high resolution TL analysis together with MGMT methylation status providing powerful independent prognostic information. The presence of short telomeres in patients with unmethylated MGMT defines a subgroup of GBM patients that may benefit from alternative treatment regimens. The presence of telomere fusions offers direct evidence for significant genomic instability, potentially favouring targeted DDR inhibitors for patients exhibiting this signature. HB is supported by a WCAT fellowship and Brain Research UK. Work in the Baird Lab is supported by Cancer Research UK and US NIH. Thanks go to BRAIN UK and all our patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5786e68cc90683f81be8d14b8359cb26641b2fd7" target='_blank'>
              P05.18.B TELOMERE DYSFUNCTION AND FUSIONS IN GLIOBLASTOMA - DEFINING PROGNOSIS AND GENOME INSTABILITY
              </a>
            </td>
          <td>
            H. Bhatt, K. Norris, K. Cleal, M. Hodgson, C. Herbert, W. Gray, K. Kurian, D. Baird
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Numerous cellular processes rely on biomolecular condensates formed through liquid–liquid phase separation (LLPS). Recently, it has become evident that somatic mutations can interfere with or over-activate the formation of phase-separated condensates. Results Here, we set out to systematically study the connection between cancer and biological condensation, specifically mapping the extent to which LLPS is affected in cancer and understanding the molecular pathomechanisms and therapeutic consequences of mutations affecting LLPS scaffolds. We identify both known and novel combinations of molecular functions that are specific to oncogenic fusion proteins and thus have a high potential for driving tumorigenesis. Protein regions driving condensate formation show an increased association with DNA- or chromatin-binding domains of transcription regulators within oncogenic fusion proteins, indicating a common molecular mechanism underlying several soft tissue sarcomas and hematologic malignancies where phase-separation-prone oncogenic fusion proteins form abnormal condensates along the DNA and thereby dysregulate gene expression programs. Conclusions We find that proteins initiating LLPS are frequently implicated in somatic cancers, even surpassing their involvement in neurodegeneration. Our data shows that cancer-driving LLPS scaffolds tend to be potent oncogenes, giving rise to dominant phenotypes and lacking targeting options by current FDA-approved drugs. Finding the currently missing drugs to shut down oncogenic fusion proteins, to disrupt the condensation enabled by them, and to offset their downstream effects could provide cancer drugs widely applicable to diverse cancer incidences previously defying standard treatments. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03787-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54d550a595352429b7ad7fdab681cd564170f566" target='_blank'>
              Phase-separating fusion proteins drive cancer by upsetting transcription regulation
              </a>
            </td>
          <td>
            Nazanin Farahi, Tamas Lazar, Peter Tompa, Bálint Mészáros, Rita Pancsa
          </td>
          <td>2025-10-02</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49580bd801398c666e64f8508dfb8bf04f7a7c7d" target='_blank'>
              Epidrugs in cancer: mechanisms, applications, and future direction.
              </a>
            </td>
          <td>
            Roshni Bibi, Lakshmi Varshetha, Rakshit Kamal Lahary, Jahnavi Namburi, Fardeen Ahmed Laskar, Koustav Sarkar
          </td>
          <td>2025-10-08</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Deciphering the pre-malignant cell of origin (COO) of different cancers is critical for understanding tumor development and improving diagnostic and therapeutic strategies in oncology. Prior work demonstrates that somatic mutations preferentially accumulate in closed chromatin regions of a cancer’s COO. Leveraging this information, we combine 3,669 whole genome sequencing patient samples, 559 single-cell chromatin accessibility cellular profiles, and machine learning to predict the COO of 37 cancer subtypes with high robustness and accuracy, confirming both the known anatomical and cellular origins of numerous cancers, often at cell subset resolution. Importantly, our data-driven approach predicts a basal COO for most small cell lung cancers and a neuroendocrine COO for rare atypical cases. Our study also highlights distinct cellular trajectories during cancer development of different histological subtypes and uncovers an intermediate metaplastic state during tumorigenesis for multiple gastrointestinal cancers, which have important implications for cancer prevention, early detection, and treatment stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba12099e8d95054e5c9ddd1d07125ef4b9a79476" target='_blank'>
              Learning the cellular origins across cancers using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Wooseung Lee, Bruno Giotti, Akhil Kumar, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Overexpression of the proto-oncogene MYC occurs in >70% of cancers and is especially prevalent in breast cancer. Myc partners with transcription factor Max to bind to the E-box DNA response element. By patterning our frankenproteins on the basic region/helix-loop-helix/leucine zipper motif of Max, we designed MEF and MEF/C93 to bind to the E-box. In bacterial one-hybrid assays and quantitative electrophoretic mobility shift assay, both proteins bound specifically to the E-box with high sequence-specificity and affinity (Kd = 8 nM) rivaling native transcription factors. Quantitative PCR revealed that MEF and MEF/C93 selectively downregulated Myc target genes in Myc-dependent MDA-MB-231 breast cancer cells, but not in Myc-independent MCF-7 cells. Fluorescence colocalization demonstrated transport into cell nuclei. Our proteins displayed IC50 values of 1-2 μM in cell viability assays in MDA-MB-231, compared with ∼25 μM in MCF-7. These results demonstrate the specificity of targeting Myc-dependent cancer cells and mark significant progress toward protein-based therapies aimed at Myc-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8cc39bbdc2499f24c52dd2589dd7f515be4ffe6" target='_blank'>
              Designer Frankenproteins That Halt the Proliferation of Myc-Driven Cancer Cells.
              </a>
            </td>
          <td>
            Maryam Ali, Raneem Akel, Francine He, Micheline Piquette-Miller, Jumi A Shin
          </td>
          <td>2025-09-03</td>
          <td>Journal of medicinal chemistry</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="


 Hypoxia is a hallmark of the glioblastoma (GB) and a critical driver of tumor progression, cellular plasticity, and resistance to standard therapy. While its role in metabolic reprogramming is established, the mechanisms by which hypoxia shapes tumor evolution at the genomic and epigenetic levels remain poorly defined.



 To investigate the impact of hypoxia on glioblastoma evolution, we employed a multimodal approach that integrated multi-omic spatial transcriptomics and whole genome DNA methylation profiling, to explore spatial-hypoxia depended genomic alterations. We developed a hypoxia ex-vivo culturing system using human GB tissue slices analyzed by RNA-seq whole genome sequencing and whole genome methylation profiling. Further, an immunofluorescence imaging based machine learning method was established to evaluate cell cycle arrest and lineage transitions within an intact tissue architecture.



 Spatial analysis of patient-derived samples revealed that hypoxia-enriched niches are closely associated with chromosomal instability, particularly marked by the published CX2 signature—defined by short, oscillatory single-copy alterations indicative of homologous recombination defects. In ex vivo tumor slices, hypoxia induced a distinct transcriptional program characterized by activation of the unfolded protein response, metabolic reprogramming, and G0-G1 cell cycle arrest, as confirmed by immunofluorescence image analysis and consistent with prior spatial transcriptomics data. Simultaneously, hypoxia promoted epigenetic remodeling at the promoters of key neurodevelopmental genes, including WNT8B and HMGB4, pointing toward a priming of cells into an early neuronal progenitor-like state and increased chromatin accessibility of genes regulating the G0-G1 to G1-S transition. Notably, whole-genome profiling of hypoxia-treated slices identified copy number gains converging in MAPK signaling genes of BRAF/RAF fusions—alterations absent in conventional glioblastoma cell lines. Collectively, these findings highlight a tissue-specific mechanism by which hypoxia drives oncogenic signaling and neuronal lineage plasticity.



 Our results define a hypoxia-driven evolutionary program in GB, characterized by concomitant BRAF/MAPK pathway activation and neuronal epigenetic reprogramming. These adaptations promote the emergence of slow-cycling, therapy-resistant subclones poised for recurrence. Targeting the hypoxia-MAPK-lineage axis may present a promising therapeutic strategy to overcome resistance and delay GB relapse.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af2ad53cd3cc41596d1b40ea865e837da4df51f1" target='_blank'>
              P02.19.B BRAF/MAPK ACTIVATION AND NEURONAL LINEAGE PRIMING DEFINE A HYPOXIA-DRIVEN EVOLUTIONARY PROGRAM IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            G. Villa, N. Conrad, G. Kastner, J. Kückelhaus, J. Benotmane, Y. Yabo, J. Zhang, R. Sankowski, O. Schnell, D. Delev, N. Neidert, D. Heiland
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Repetitive DNA sequences, as transposable elements (TEs) and satellite DNA (satDNA) spread and diversify within host genomes, impacting genome biology in numerous ways. In the first part of this review, we emphasize the evolutionary importance of satDNAs and TEs, providing a short summary of their roles and the mechanisms by which they influence the structure and function of genomes. We also discuss the broad, complex, and extensive relationships between TEs and satDNAs. Following that, we bring together different mechanisms on the generation of satDNA from TE, as it has been demonstrated that almost any part of any type of TE can undergo tandemization and produce novel satDNAs. Importantly, we here present a hypothesis that would explain the existence of particular types of monomers, namely composite satDNA monomers which display multiple subsequent stretches of similarity to various TEs, for which the explanation was lacking so far. We propose that even highly shuffled and degraded TE remnants residing in heterochromatin "TE graveyards" can give rise to new satDNA sequence monomers, transforming these genomic loci into DNA "recycling yards." Furthermore, we emphasize important evolutionary questions regarding the causes, mechanisms, and frequency of these occurrences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56f67b41471ebec0a1b45c2936430e5e0e7ad266" target='_blank'>
              Satellite DNAs rising from the transposon graveyards.
              </a>
            </td>
          <td>
            Eva Šatović-Vukšić, Patrik Majcen, Miroslav Plohl
          </td>
          <td>2025-09-30</td>
          <td>DNA research : an international journal for rapid publication of reports on genes and genomes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract The ongoing arms race between prokaryotes and mobile genetic elements (MGEs) gives rise to a myriad of host-defense systems that detect and degrade invading nucleic acids. However, it is yet unclear whether changing the mechanical properties of nucleic acids can impact plasmid/phage invasion. Here, we demonstrate that H-NS, an abundant nucleoid-associated protein (NAP), limits plasmid transfer by directly binding to and oligomerizing along with transforming double-stranded DNA. The constitutive defense function of H-NS can be complemented by convergently evolved NAPs from different phyla. H-NS proteins form intramolecular bridges between DNA duplexes within incoming plasmids. Different from other defense systems that exist at low levels prior to detecting MGEs, major NAPs exist in high abundance, which may moonlight as a constant plasmid surveillance agent. Our study implies that mechanical modification of nucleic acids may be an underexplored mechanism for prokaryotic immunity, which could help balance the advantages and disadvantages of MGEs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4992180cb487a041111c4f065e8e80d198a5eb5b" target='_blank'>
              Abundant bacterial nucleoid-associated protein H-NS limits plasmid transfer through mechanical modification of DNA
              </a>
            </td>
          <td>
            Mingyue Fei, Mengdie Fang, Qi Zhou, Ziyan Chen, Mengxin Gong, Fabai Wu, Changfu Tian, Dongchang Sun
          </td>
          <td>2025-09-23</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Lower-grade glioma (LGG) is a common primary brain tumor with potential progression to glioblastoma. Its development is closely linked to DNA damage repair. DNA polymerase delta subunit 3 (POLD3), a subunit of the DNA polymerase δ complex, participates in DNA replication and repair. POLD3 may act as a therapeutic target and prognostic biomarker in LGG. Methods We collected primary data from The Cancer Genome Atlas (TCGA), the Chinese Glioma Genome Atlas (CGGA), the Tumor Immune Estimation Resource (TIMER), and Gene Set Enrichment Analysis (GSEA). The CGGA dataset included messenger RNA (mRNA) sequencing and clinical data from 403 LGG cases; its microarray subset contained 142 additional cases. The TCGA dataset provided clinical data for 503 LGG cases. Employing bioinformatics tools, we investigated the correlation of POLD3 expression with the clinical characteristics, pathological features, survival outcomes, and tumor immune microenvironment in LGG. Subsequent functional validation of POLD3’s impact in LGG was performed using both cellular and animal models. Results Integrative analysis combining bioinformatics and experimental approaches revealed significant POLD3 overexpression in LGG. Elevated POLD3 levels exhibited significant associations with distinct clinicopathological features, altered immune cell infiltration profiles, and adverse clinical outcomes. Functionally, POLD3 was demonstrated to drive LGG cell proliferation, invasion, and modulate the tumor microenvironment. Conclusion This study establishes POLD3 as a critical molecular determinant in LGG pathogenesis. Our findings position POLD3 as a promising multifaceted target, with significant implications for advancing LGG diagnostics, therapeutic development, immune modulation strategies, and prognostic assessment, thereby providing a fresh conceptual framework for understanding and managing this tumor. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14934-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b766711209d5df5ec31c2b94f1204021afe9d8" target='_blank'>
              POLD3 knockdown effects on low-grade glioma: insights from bioinformatics and experimental validation
              </a>
            </td>
          <td>
            Zhihui Yang, Zhonghao Dou, Hao Wang, Zhaojiang Hu, Yanbo Gao, Yilei Bai, Junfeng Liu, Rongjun Qian
          </td>
          <td>2025-10-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Aberrant expression of long noncoding RNAs (lncRNAs) serves as a critical driver in carcinogenesis and tumor evolution across various human cancers. Among these lncRNAs, small nucleolar RNA host gene 15 (SNHG15) has emerged as a key regulator in multiple cellular processes. Its dysregulation is frequently implicated in disease pathogenesis, particularly cancer. In this investigation, we delineated the functional landscape and mechanistic basis of SNHG15 in lung carcinoma. Cell cycle synchronization experiments demonstrated phase-dependent SNHG15 expression, showing maximal transcript abundance during G2/M transition. SNHG15 knockdown substantially curbed tumor growth in vitro (A549 cell line) and in vivo (subcutaneous PDX models). Notably, SNHG15 orchestrated genomic surveillance mechanisms through tripartite regulation: (1) cell cycle checkpoint coordination, (2) transcriptional machinery reactivation, and (3) nucleotide excision repair. Mechanistically, proteomic profiling coupled with RNA immunoprecipitation identified CREB5 as a direct binding partner mediating SNHG15-dependent survival signaling under genotoxic stress. These results position the SNHG15-CREB5 axis as an important player governing genomic fidelity in lung adenocarcinoma, suggesting druggable targets for precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1f937a74141f7d8c6f5eb7bb745d4f086d4cf3" target='_blank'>
              The role of lncRNA SNHG15 in UV-induced DNA damage repair
              </a>
            </td>
          <td>
            Xin Zhang, Jiahui Lin, Zixuan Fan, Sai Zhang, Meiqi Yang, Tingting Zhu, Qing Lu, Yuming Wang
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Polymerase theta (POLQ) is essential for theta-mediated end joining (TMEJ), an error-prone DNA repair pathway that is hyperactive in homologous recombination-deficient (HRD) cancers such as high-grade serous ovarian cancer (HGSOC). Inhibiting POLQ is a promising therapeutic strategy, but the molecular determinants of sensitivity and downstream mechanisms remain incompletely defined.



 We assessed the growth-inhibitory activity of two structurally distinct POLQ inhibitors, novobiocin and ART558, across 18 well-characterized HGSOC cell lines. Genomic profiling data were integrated with drug response values (GR50s) to identify predictive biomarkers, including mutational status, HRD scores, and copy number signatures (CX1–CX17). Mechanistic assays included POLQ ATPase activity, cell cycle analysis (FUCCI), immunofluorescence for γH2AX and POLQ, apoptosis markers and DNA fiber assays.



 POLQ inhibitor sensitivity varied independently from olaparib or carboplatin response, indicating a distinct mode of action. Novobiocin sensitivity significantly correlated with CX3 and CX5 copy number signatures, which reflect impaired homologous recombination and replication stress. In contrast, resistant lines were enriched for CX14 and CX6, signatures linked to mitotic errors and telomere shortening. Sensitive cell lines showed increased DNA damage (γH2AX foci), POLQ accumulation on chromatin, and apoptosis, particularly in BRCA1/2-mutant backgrounds. Cell cycle profiling revealed impaired G1/S progression and S-phase depletion. DNA fiber assays demonstrated replication fork degradation upon POLQ inhibition, consistent with POLQ’s role in fork protection.



 We identify genomic features, specifically CX3 and CX5 copy number signatures as candidate biomarkers of POLQ inhibitor sensitivity in HGSOC. Mechanistically, POLQ inhibition induces replication stress, persistent DNA damage, and apoptosis in HR-deficient settings. These findings support the clinical development of POLQ inhibitors and rational patient stratification based on genome instability signatures.



 Aleksandra Kruglikov. POLQ inhibitors induce replication stress and apoptosis in high-grade serous ovarian cancer: Functional and genomic biomarkers of response [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe824b1195032b879fa6dbdb3ed12cc08d24058" target='_blank'>
              Abstract A037: POLQ inhibitors induce replication stress and apoptosis in high-grade serous ovarian cancer: Functional and genomic biomarkers of response
              </a>
            </td>
          <td>
            Aleksandra Kruglikov
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Berberine (BBR) is an isoquinoline alkaloid with a variety of biological activities, including anti‐microbial and anti‐tumoral activities. However, the cellular targets of BBR and the roles of BBR in the radiosensitivity of breast cancer cells are not well defined. In this study, we investigated the effects of BBR on the radiosensitivity of BT549 triple‐negative breast cancer cells. Through RNA‐seq results, we found that BBR significantly down‐regulated the expression of DNA ligases. The results of western blot and comet assay confirmed that BBR attenuated DNA ligase III (LIGIII) expression and caused DNA damage in a dose‐dependent manner. The results of electron microscopy showed that BBR decreased mitochondrial copies and induced mitochondrial autophagy. Clonal formation analysis showed that BBR sensitised breast cancer cells to irradiation. The genetic complementation and interference assays showed that the effect of BBR on the radiosensitivity of BT549 breast cancer cells is dependent on the expression of LIGIII. These results contribute to the understanding of the effects of BBR on cellular DNA repair and the clinical application of BBR in breast cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d7148d132ad7e87d8b8ae13f9a1fb9fcaf3cd90" target='_blank'>
              Berberine Sensitises Breast Cancer Cells to Radiation via the Attenuation of DNA Ligase III
              </a>
            </td>
          <td>
            Yuxin Sun, Cong Li, Hang Yin, Kunyan Li, Ying Wang, Shuailong Zhang, Yi Zhao, Jing Wang, Weifeng Mao
          </td>
          <td>2025-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 In this talk I will discuss new insights into the nature of tumor evolution in high grade serous ovarian cancer (HGSOC). I will focus on two key areas that have been informed by recent advances in single cell whole genome sequencing. First, I will discuss the nature of how HGSOCs acquire genome doubled states in their evolutionary histories. I will show that genome doubling is an ongoing process throughout evolution with important consequences for phenotypic states and immunosuppression. Second, I will discuss recent work to track evolution from diagnosis to relapse using structural variants and cell-free DNA technology. I will show how clonal populations either contract or persist over a multi-year intervals in patients with relapsed disease and highlight genomic structural features linked to the evolution of drug resistance.



 Sohrab Shah. Frontiers of tumor evolution in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c181ca2707243d8dc30253d7814569567a3fe09e" target='_blank'>
              Abstract IA024: Frontiers of tumor evolution in ovarian cancer
              </a>
            </td>
          <td>
            Sohrab P. Shah
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background The dysregulation of the DNA damage response (DDR) plays a vital role in cancer development and inadequate response to anti-cancer therapy. We investigated the involvement of DNA polymerase theta (Polθ), the principal component of the microhomology-mediated end joining (MMEJ) pathway, in the pathogenesis of multiple myeloma (MM). Methods We explored the expression levels of Polθ and assessed its prognostic relevance in a cohort of 124 MM patients at our institution. We demonstrated the impacts of Polθ inhibition on genomic stability and cell viability in MM using shRNA and a pharmacological inhibitor. To elucidate the role of Polθ-mediated MMEJ in repairing melphalan-induced DNA damage, we employed Western blot and EJ2-GFP reporter assay to evaluate the activation of the MMEJ pathway following melphalan exposure. We analyzed the synergistic cytotoxicity between the Polθ inhibitor and melphalan in MM, both in vitro and in vivo. The impacts of combination therapy on genomic damage and mitotic catastrophe were examined by comet assay, Western blot, and immunofluorescence. Results We revealed that Polθ was markedly upregulated in MM. The enhanced expression of Polθ was found to be significantly associated with advanced disease stages and adverse clinical outcomes. Polθ inhibition promoted genomic instability and impaired MM growth, underscoring the polymerase's function in sustaining myeloma survival under ongoing genomic stress. Furthermore, we observed a further increase in Polθ expression and MMEJ repair efficiency following melphalan treatment. Polθ inhibitors sensitized MM cells to melphalan by increasing unrepaired DNA damage and inducing mitotic catastrophe, both in vitro and in vivo. Conclusions Collectively, our results confirm Polθ as a promising prognostic biomarker and highlight its potential to serve as a synthetic lethal target that can be used in combination with melphalan for the treatment of MM. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07065-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a916ab34fef94b64abac58ae251e6ce253205a64" target='_blank'>
              Polymerase theta inhibition impairs tumor growth and amplifies melphalan-induced DNA damage in multiple myeloma
              </a>
            </td>
          <td>
            Qun Li, Chi Ma, Liping Zuo, Aoshuang Xu, Bo Zhang, F. Zhao, Lifei Liu, Xin Zhao, Yu Hu, Chunyan Sun
          </td>
          <td>2025-10-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Breast cancer (BCa) is a major contributor to female mortality worldwide. Treatment resistance and tumor heterogeneity contribute to the lack of effective therapeutic targets, posing a significant challenge in BCa management. CENPI, a core centromere protein involved in chromosome segregation, has emerging evidence implicating it in oncogenesis across diverse malignancies. However, its functional and molecular mechanisms in BCa remain unclear. Methods We analyzed CENPI expression and its clinical significance by using the BCa dataset from the Cancer Genome Atlas (TCGA) and immunohistochemical staining of 3 human BCa tissue samples. Cellular functional assays and mice xenograft models were utilized to assess the effects of CENPI on BCa growth. RNA sequencing combined with bioinformatics analysis was conducted to elucidate the molecular mechanisms underlying CENPI function, with further validation through Western blotting, immunofluorescence, and TOP/FOP flash assays. Results CENPI was aberrantly overexpressed in BCa, with elevated expression levels strongly associated with disease progression and poor prognosis. Functional assays demonstrated that CENPI significantly promoted breast carcinogenesis in both cellular and animal models. Mechanistically, CENPI increased BCa progression and malignant phenotypes by modulating the Wnt/β-catenin axis. Conclusions CENPI is a critical oncogene in BCa, driving tumorigenesis and disease progression via the Wnt/β-catenin axis, which represents a promising biomarker and therapeutic target for BCa. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-04001-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33eef316f873dd58287f9f4d71241b7970f53004" target='_blank'>
              Centromere protein I facilitates breast cancer tumorigenesis and disease progression through modulation of Wnt/β-Catenin signaling
              </a>
            </td>
          <td>
            Chaoshen Wu, Yibing Zhou, Yuxiao Mu, Yijie Cheng, Wenqian Xu, Mengna Huang, Jingyuan Xu, Ming Li, Jianfeng Gu, Xuli Meng, Da Qian
          </td>
          <td>2025-10-10</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 There is global epigenetic dysregulation during transformation from normal to cancerous cells. One of the many consequences of this global epigenetic dysregulation is the reactivation of transposable elements (TEs). TEs comprise roughly half of the human genome and are typically repressed through epigenetic mechanisms, including DNA methylation and histone modifications. Endogenous retroviruses (ERVs) are a class of TEs that contain long terminal repeats (LTRs) with putative gene regulatory motifs. LTRs can be co-opted to be pro-tumorigenic by becoming non-canonical promoters for oncogenic genes. While several examples of TE-derived promoters have been reported in cancer, the potential significance of TE-derived enhancers remains largely unexplored. Here, we utilized publicly available multi-omics datasets (bulk and single-cell chromatin profiling and RNA-sequencing) of high-grade serous carcinoma (HGSC) and fallopian tube (FT) tissues to determine if there are tumor-specific, TE-derived enhancers that have gene regulatory function. Once putative enhancers were identified, we used Clustered Regularly Interspaced Short Palindromic Repeats interference (CRISPRi) to functionally validate a putative TE-derived enhancer and gene regulatory axis. We discovered that on the bulk tissue level, several TE families, notably LTR2B, LTR18A, and LTR18B, are enriched at open chromatin sites and sites marked by H3K27Ac (enhancer mark) in HGSC cells compared to FT cells, suggesting the presence of HGSC-specific, TE-derived enhancers. On the single-cell level, we discovered that TE transcripts from the same families are more highly expressed in tumor cells compared to stromal and immune cells in the tumor microenvironment, suggesting that the increased TE-derived enhancer activity observed on the bulk level is specific to tumor cells. Lastly, we discovered that silencing a specific LTR18A locus upstream of the TCDD-Inducible Poly ADP Ribose Polymerase (TIPARP) gene was sufficient to repress TIPARP gene expression suggesting that these putative TE-derived enhancers have a gene regulatory function. TIPARP expression has been shown to be correlated with HGSC, and our studies show a novel mechanism by which TIPARP may be regulated in HGSC. Determining the regulatory role of these TE-derived enhancers will provide novel insights into not only HGSC, but also other cancer-related transcriptomes.



 Lily Nguyen, Abigail Jeong, Atma Ivancevic, Benjamin Bitler, Edward B. Chuong. Transposable element derived enhancers in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e45139333a87abd3b3e296653a1c558db2322706" target='_blank'>
              Abstract B012: Transposable element derived enhancers in ovarian cancer
              </a>
            </td>
          <td>
            Lily L. Nguyen, Abigail D Jeong, A. Ivancevic, Benjamin G. Bitler, Edward B. Chuong
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28b36c19d01be738ee1237dc98c176d6789c1e47" target='_blank'>
              Chromatin-associated circRNA ciCRLF3(2) regulates cell differentiation blockage via activating non-homologous end joining-based DNA repair.
              </a>
            </td>
          <td>
            Ke-Jia Pu, Xiao-Tong Chen, Shun-Xin Zhu, Yan An, Xin-Yi Feng, Heng-Jing Huang, Cheng-Lin Zhou, Mei-Ying Ye, Yun-Chun Wei, Yi-Xuan Ma, Chen Fang, Nan Zhang, Dan Wang, Bin-Rong Han, Jun-Yi Lian, Tian-Qi Chen, Yumeng Sun, Yue-Qin Chen, Wentao Wang
          </td>
          <td>2025-09-04</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Repeated oncogenic mutations and polyclonal proliferation are evident in cancers. However, little is known about the polyclonal principles governing the systemic cancerous lineage during immunotherapy. Here, we examine a unique autopsy case of metastatic urothelial carcinoma that exhibits different treatment responses to anti-PD-1 therapy at each tumor site. By performing in-depth analyses of different multiregional bulk tumor masses, we reveal that subsets of subclones acquire potential driver mutations under treatment selection pressure. Spatial transcriptomics analysis reveals that subclones resistant to immunotherapy form distinct immunosuppressive environments consistent with their habitats. Furthermore, different cancer hallmarks are identified in each of the subclones that expand under immunotherapy at single-cell level; for example, one subclone is more proliferative, and another is more stem-cell-like. In summary, this study provides an overall picture of the polyclonal competition and changes in the immune microenvironment that are related to resistance to immunotherapy in patients with malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f357f9c7e57b9e95e8ee48a6378f3d690f3275bd" target='_blank'>
              Clonal diversity shapes the tumour microenvironment leading to distinct immunotherapy responses in metastatic urothelial carcinoma
              </a>
            </td>
          <td>
            Takashi Kamatani, K. Umeda, T. Iwasawa, Fuyuki Miya, K. Matsumoto, S. Mikami, Kensuke Hara, Masayuki Shimoda, Yutaka Suzuki, Jo Nishino, Mamoru Kato, Kazuhiro Kakimi, Nobuyuki Tanaka, M. Oya, T. Tsunoda
          </td>
          <td>2025-08-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is one of the most prevalent adult leukemias, derived from mature B-cells and exhibiting a highly heterogeneous disease course. Standard cytogenetic analysis of CLL includes FISH and karyotyping. However, conventional chromosome analysis of cancer specimens is often constrained by low chromosomal resolution, and FISH analysis is limited by the number of probes that can be applied. This study highlights the application of optical genome mapping (OGM), a high-resolution cytogenomic tool that visualizes ultra-long, sequence-labeled DNA molecules, to uncover the structural complexity of the cancer genome and assess the clinical relevance of chromothripsis in CLL. Comprehensive cytogenetic analysis was conducted on a 43-year-old male diagnosed with chronic lymphocytic leukemia. Karyotyping revealed a complex rearrangement: 46,XY,der(3)t(3;13)(p2?3;q14.3),der(4)t(?3;4)(p23;p16),add(11)(p13),del(13)(q14)[12]/46,sl,del(11)(q2?2.2q23.3)[6]. FISH analysis further identified the loss of ATM and a partial deletion of the D13S319 locus. OGM analysis performed on bone marrow revealed a complex genotype including chromothripsis of chromosome 13, and structural rearrangements involving chromosomes 3, 4, and 11. Additionally, multiple intrachromosomal translocations and interstitial microdeletions of chromosome 13 were identified. The resolution of these aberrations has been significantly enhanced with examples including: ogm[GRCh38] t(3;13)(p26.3;q33.1)(2,706,645~2,721,113;103,142,901~103,154,241][VAF0.45], ogm[GRCh38] t(4;13)(p15.31;q32.1)(20,869,721~20,907,265;96,617,837~96,630,317)[VAF0.42],. In conclusion, OGM revealed the intricate structural alterations of the cancer genome. The high resolution provided by OGM could facilitate the discovery of oncogenic mechanisms, novel fusion genes, prognostic markers, and potential therapeutic targets. OGM serves as a powerful tool for revisiting CLL disease classification by offering deeper insights into complex genomic rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4656aef9b3e1a0de1b2f5258ea921c03f0e3137" target='_blank'>
              Case Report: Unraveling complex genomic alterations in a case of chronic lymphocytic leukemia using optical genome mapping
              </a>
            </td>
          <td>
            Leila Youssefian, Trilochan Sahoo, Jia-Chi Wang
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background DNA methylation is an epigenetic mechanism involved in gene regulation and cellular differentiation. Accurate and comprehensive assessment of DNA methylation patterns is thus essential for understanding their role in various biological processes and disease mechanisms. Bisulfite sequencing has long been the default method for analyzing methylation marks due to its single-base resolution, but the associated DNA degradation poses a concern. Although several methods have been proposed to circumvent this issue, there is no clear consensus on which method might be better suited for specific study designs. Results We conducted a comparative evaluation of four DNA methylation detection approaches: whole-genome bisulfite sequencing (WGBS), Illumina methylation microarray (EPIC), enzymatic methyl-sequencing (EM-seq) and third-generation sequencing by Oxford Nanopore Technologies (ONT). DNA methylation profiles were assessed across three human genome samples derived from tissue, cell line, and whole blood. We systematically compared these methods in terms of resolution, genomic coverage, methylation calling accuracy, cost, time, and practical implementation. EM-seq showed the highest concordance with WGBS, indicating strong reliability due to their similar sequencing chemistry. ONT sequencing, while showing lower agreement with WGBS and EM-seq, captured certain loci uniquely and enabled methylation detection in challenging genomic regions. Despite a substantial overlap in CpG detection among methods, each method identified unique CpG sites, emphasizing their complementary nature. Conclusions Our findings underscore the strengths and limitations of current DNA methylation detection methods. EM-seq and ONT emerge as robust alternatives to WGBS and EPIC, offering unique advantages: EM-seq delivers consistent and uniform coverage, while ONT excels in long-range methylation profiling and access to challenging genomic regions. These insights provide practical guidance for method selection based on specific experimental goals. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00616-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1115f5c6b6dae937448bce4077f0951e9ac581f3" target='_blank'>
              Comparison of current methods for genome-wide DNA methylation profiling
              </a>
            </td>
          <td>
            Ana Regina de Abreu, J. Ibrahim, Vasileios Lemonidis, Ligia Mateiu, Guy Van Camp, K. Op de Beeck
          </td>
          <td>2025-08-25</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3493648729eb60ce21f81ce27852f8a58bffd93" target='_blank'>
              Influence of Epigenetic Regulators Expression on Gastric Cancer Prognosis
              </a>
            </td>
          <td>
            Lucas Brabo Rotella, A. Khayat, Monique Feitoza, Victoria Pereira Costa, Diego Pereira, Fabiano Cordeiro Moreira, S. Demachki, Samir Mansour Moraes Casseb, G. Ishak, W. Barra, R. Burbano, P. P. D. Assumpção
          </td>
          <td>2025-09-15</td>
          <td>None</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Subtelomeres are imperfect repeats adjacent to telomeres that are repressed by heterochromatin. Although essential for genome integrity, their repetitive nature has thwarted dissection of local heterochromatin assembly and maintenance mechanisms. Here, we engineered Schizosaccharomyces pombe strains carrying fluorescent reporters at a single subtelomere. We find that subtelomeric heterochromatin is organized into discrete subdomains that nucleate at telomere-proximal and cryptic internal sites. Telomere-proximal regions depend on canonical shelterin or RNA interference nucleation pathways, while telomere-distal regions require nucleosome remodelers, histone chaperones, and boundary-associated factors. Using multi-generational live imaging and targeted perturbations, we show that subtelomeric subdomains display position-specific, clonally variable silencing across a spectrum of robust to fragile epigenetic states. This clonal variegation is also induced by naturally occurring subtelomeric structural variants. These findings demonstrate that subtelomeric heterochromatin maintenance is not uniform but rather governed by local chromatin context and architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1892593b4d09d954d617fd50a949c2f67aa1bce" target='_blank'>
              Discrete Subdomains Establish Epigenetic Diversity in Subtelomeric Heterochromatin
              </a>
            </td>
          <td>
            Agnisrota Mazumder, John Cooper, Can Goksal, J. S. Khanduja, Richard I. Joh, Johannes J. Groos, Rosario Y.J. Brockhausen, Junko Kanoh, Mohammad R. Motamedi, Ilya J. Finkelstein, Bassem Al-Sady, Sigurd Braun
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract Histones are key epigenetic factors that regulate the accessibility and compaction of eukaryotic genomes, affecting DNA replication and repair, and gene expression. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of about 12 000 patients. Histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using genomic, structural, and biophysical analyses, we found several predominant modes of action by which histone mutations may alter function. Namely, cancer missense mutations primarily affected histone acidic patch residues and protein-binding interfaces in a cancer-specific manner and targeted interaction interfaces with specific DNA repair proteins. Consistent with this finding, we observed a high tumor mutational burden in patients with histone mutations affecting interactions with proteins involved in maintaining genome integrity. We identified potential cancer driver mutations in several histone genes, including mutations on histone H4—a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8347f132028127afcb5dcde491fcf3c72af1a050" target='_blank'>
              Cancer histone mutations impact protein binding and DNA repair with possible links to genomic instability
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J Aristizabal, Anna R. Panchenko
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Hypoxia-inducible factor (HIF) is a master regulator of cancer cell adaptation to tumor hypoxia and is involved in cancer progression. Single-cell (sc) differences in the HIF response allow for tumor evolution and cause therapy resistance. These sc-differences are usually ascribed to tumor microenvironmental differences and/or clonal (epi)genetic variability. However, the sc-heterogeneity of the HIF response in otherwise identical cells cultured under defined in vitro conditions has not yet been addressed. Therefore, we analyzed the sc-response to hypoxia in nonclonal cell lines and multiple clonal derivatives, including HIF-1α or HIF-2α knockouts. While HIF-1α and HIF-1 target mRNA sc-heterogeneity was slightly higher than global transcription or specific housekeeping messenger RNAs (mRNAs), HIF-2α and especially HIF-2 target mRNA sc-heterogeneity was extraordinary, and remained in independent clones following HIFα knockouts. Unexpectedly, neither HIF-2α mRNA nor nuclear protein levels correlated with target mRNA levels. Unsupervised but not supervised HIF target gene dimensionality reduction revealed the initial sample composition after scRNA-seq, demonstrating that, owing to sc-heterogeneity, individual HIF target genes are not sufficient to unequivocally identify hypoxic cancer cells. In conclusion, the pronounced intrinsic sc-heterogeneity of the HIF response represents a hitherto unrecognized feature of cancer cells that impairs clinical HIF pathway-dependent cancer cell identification and targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b54dd117e53122de89768b0fc8a01ce303d40e2" target='_blank'>
              Inherent single-cell heterogeneity of the transcriptional response to hypoxia in cancer cells
              </a>
            </td>
          <td>
            Małgorzata Wilk, Thomas Knöpfel, Stana M Burger, S. N. Khodo, Roland H. Wenger
          </td>
          <td>2025-08-28</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The circular, double-stranded DNA genomes of Human papillomaviruses (HPV) exist in a nucleosomal state throughout the infectious cycle and rely on host histone epigenetic modifications and chromatin assembly processes to promote various phases of the viral life cycle. Here, we show that the histone H3.3 chaperone HIRA and its associated complex members are recruited to HPV replication factories during the late phase of the HPV life cycle. HIRA is also recruited to HPV replication factories generated by amplification of a replicon with a minimal origin and expression of the viral replication proteins E1 and E2, demonstrating that the E1 and E2 proteins are sufficient for HIRA recruitment. Downregulation of HIRA expression reduces HPV31 DNA amplification and viral transcription in differentiated keratinocytes. Histone H3.3 that is highly phosphorylated on serine residue 31 is also enriched at sites of HPV replication and this modification links the DNA damage response to chromatin that supports rapid gene activation. We propose that deposition of histone H3.3 generates viral minichromosomes that are highly primed to support the late stages of the HPV life cycle.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a5b34a817df65cd52a7a01714509e40fb97f7ea" target='_blank'>
              Chromatin assembly by the histone chaperone HIRA facilitates Human Papillomavirus replication
              </a>
            </td>
          <td>
            Ashley N. Della Fera, Dan Chen, Alison A. McBride
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Mutations in numerous genes contribute to human cancer, with different oncogenic lesions prevalent in different cancer types. However, the malignant phenotype is simple, characterized by unrestricted cell growth, invasion, and often metastasis. One possible hypothesis explaining this dichotomy is that cancer genes regulate common targets, which then function as master regulators of essential cancer phenotypes. To identify mechanisms that drive the most fundamental feature shared by all tumors-unrestricted cell proliferation-we used a multiomic approach, which identified translation and ribosome biogenesis as common targets of major oncogenic pathways across cancer types. Proteomic analysis of tumors and functional studies of cell cultures established nucleolar and coiled-body phosphoprotein 1 as a key node, whose convergent regulation, both transcriptionally and posttranslationally, is critical for tumor cell proliferation. Our results indicate that lineage-specific oncogenic pathways regulate the same set of targets for growth control, revealing key downstream nodes that could be targeted for therapy or chemoprevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e15820af1f4a4174736d844aaa4b18b01cd6fd" target='_blank'>
              Diverse oncogenes use common mechanisms to drive growth of major forms of human cancer.
              </a>
            </td>
          <td>
            O. Kauko, Mikko Turunen, Päivi Pihlajamaa, Antti Häkkinen, R. Queiroz, M. Pääkkönen, S. Ventelä, Massimilliano Gaetani, Susanna L. Lundström, Antonio Murgia, Biswajyoti Sahu, J. Routila, Gong-Hong Wei, H. Irjala, J.L. Griffin, Kathryn S. Lilley, Teemu Kivioja, S. Hautaniemi, J. Taipale
          </td>
          <td>2025-08-22</td>
          <td>Science advances</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="ABSTRACT Herpes simplex virus type 1 (HSV-1) is a double-stranded DNA virus that infects most of the human population. We previously found that cellular topoisomerase I (TOP1) associates with the HSV-1 genome throughout infection. TOP1 relieves topological stress on DNA to enable and regulate transcription, DNA replication, and DNA repair. We hypothesized that TOP1 contributes to these key viral processes because HSV-1 does not encode a topoisomerase. We found that TOP1 inhibition with camptothecin (CPT) or β-Lapachone results in a significant reduction in viral yield. The effects of CPT are reversible and occur in a dose-dependent manner, independent of strain and cell type. CPT treatment results in a decrease in viral gene expression and DNA replication, with replication defects observed at all concentrations tested. Defects in viral gene expression are not restricted to a specific gene class. However, the timing and extent of the observed defects are dose dependent. We also demonstrate that if CPT is added to cells after the onset of viral DNA replication, subsequent rounds of replication are inhibited, indicating that the replication defect observed is not simply the result of prior steps in the infectious cycle being blocked. Together, these results demonstrate that CPT is a potent inhibitor of HSV-1 infection, supporting the hypothesis that TOP1 is active on HSV-1 genomes during infection. IMPORTANCE HSV-1 is a common human pathogen. It utilizes both viral and cellular factors to facilitate infection. We previously characterized the proteins that associate with HSV-1 DNA throughout infection. One of these proteins, which has been largely unexplored in the context of HSV-1 infection, is TOP1. TOP1 functions to relieve topological stress to regulate transcription, DNA replication, and other processes that involve DNA cleavage and unwinding. Our findings support a role for TOP1 in HSV-1 DNA replication and gene expression and highlight the potential to target TOP1 activity or interactions for antiviral therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cec9b98a8bb668803e7c5faf9951486f2e65bf8c" target='_blank'>
              Camptothecin, a topoisomerase I inhibitor, impedes productive herpes simplex virus type 1 infection
              </a>
            </td>
          <td>
            Joseph R Heath, Alison E. Lloyd, Raegen M. Kulinski, Daniel P Fromuth, Jill A. Dembowski
          </td>
          <td>2025-08-28</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is one of the most common cancers among malignant tumors, and extensive evidence suggests that the prognosis of various human cancers is associated with DNA methylation. Therefore, we analyzed potential GBM prognostic biomarkers by combining GBM methylation sequencing data and transcriptome sequencing data. Biostatistical analysis of the expression of three RNA types (long non-coding RNA, microRNA, and mRNA) from The Cancer Genome Atlas (TCGA) was performed to determine the correlation between NACAD and GBM prognosis. Additionally, we predicted and validated a competitive endogenous RNA (ceRNA) regulatory network associated with epigenetic DNA methylation modifications. The ceRNA regulatory network (DCTN1-AS1–hsa-let-7a-5p–NACAD) was found to be related to overall survival (OS) and disease-specific survival (DSS) of GBM. Furthermore, we identified NACAD as an independent prognostic factor for both GBM OS and DSS. In summary, comprehensive analysis of methylation and transcriptome data led to the identification of genes and pathways in different GBM subtypes, highlighting prognostic genes and ceRNA regulatory networks, and proposing potential candidates. Supplementary Information The online version contains supplementary material available at 10.1186/s40001-025-03020-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d242277602456fb7dfcd53ca176ff548d65ded" target='_blank'>
              Epigenetic and transcriptomic analysis revealed ceRNA regulatory network in glioblastoma multiforme
              </a>
            </td>
          <td>
            Zhenghui Li, Yanqing Li, Jenna Z Liang, Tianxiang Geng, Jing Chen, Yangyang Yuan, Yufang Sui, Wanhong Zhang, Qian Chen, Guowei Tan, Qiao Shan
          </td>
          <td>2025-10-09</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1110fcfacea74bd5588618a3f029d0c49019fbe" target='_blank'>
              Mechanisms and genomic implications of break-induced replication.
              </a>
            </td>
          <td>
            Adel Atari, Haoyang Jiang, Roger A. Greenberg
          </td>
          <td>2025-08-22</td>
          <td>Nature structural & molecular biology</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Centromeres, characterized by their unique chromatin attributes, are indispensable for safeguarding genomic stability. Due to their intricate and fragile nature, centromeres are susceptible to chromosomal rearrangements. However, the mechanisms preserving their functional integrity and supporting nucleus homeostasis following breakages remained enigmatic. In this study, we use wheat ditelosomic stocks, which arise from centromere breakage, to explore the genetic and epigenetic alterations in damaged centromeres. Our investigations unveil novel chromosome end structures marked by de novo addition of telomeres, as well as localized chromosomal shattering, including segment deletions and duplications near centromere breakpoints. We reveal that the damaged centromeres possess a remarkable capacity for self-regulation, through employing structural modifications such as expansion, contraction, and neocentromere formation to maintain their functional integrity. Centromere breakage triggers nucleosome remodeling and is accompanied by local transcription changes and chromatin reorganization, and subsequently may contribute to the stabilization of broken chromosomes. Our findings highlight the resilience and adaptability of plant chromosomes in response to centromere breakage, and provide valuable insights into the stability of centromeres, thereby offering promising prospects to manipulate centromeres for targeted chromosomal innovation and crop genetic improvement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c301bcc4744c9e967b82a2d0c48310497c9c67e9" target='_blank'>
              Adaptation of centromere to breakage through local genomic and epigenomic remodeling in wheat.
              </a>
            </td>
          <td>
            Jingwei Zhou, Yuhong Huang, Huan Ma, Yiqian Chen, Chuanye Chen, Fang Han, H. Su
          </td>
          <td>2025-09-30</td>
          <td>Genome research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Antigen receptor diversity depends on the assembly of variable (V), diverse (D), and joining (J) exons in genes encoding immunoglobulins (Igs) and T cell receptors (TCRs). During V(D)J recombination, DNA double-strand breaks (DSBs) introduced by the RAG1/2 nuclease complex are repaired by the process of nonhomologous end-joining (NHEJ). We hypothesized that functional redundancies between NHEJ and the chromatin DSB response, which depends on the kinase ATM, potentially masked the activity of additional factors that regulate V(D)J recombination. We performed targeted CRISPR-Cas9 knockout screens for genes implicated in V(D)J recombination in pro-B cells that were either untreated or treated with an ATM inhibitor. We found that loss of the RNA/DNA helicase senataxin (SETX) impaired V(D)J recombination and led to the formation of aberrant hybrid joints between coding ends and signal ends, both in vitro and in mice. The loss of SETX in a background deficient in the NHEJ factor XLF or in which ATM was inhibited led to substantial impairment of V(D)J recombination and to the presence of unsealed coding ends. SETX limited aberrant activation-induced cytidine deaminase (AID)-induced DNA end-joining between Igh-containing alleles during the process of class-switch recombination. Together, our findings reveal a previously uncharacterized role for SETX in promoting recombination fidelity during antigen receptor gene diversification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d8273b1b0c7d99410036337ebbbcd14723910dc" target='_blank'>
              Senataxin promotes recombination fidelity during antigen receptor gene diversification.
              </a>
            </td>
          <td>
            Alice Libri, Jinglong Wang, Timea Marton, Wei Yu, François Dossin, G. Balmus, B. Reina-San-Martin, Richard L. Frock, C. Lescale, L. Deriano
          </td>
          <td>2025-10-14</td>
          <td>Science signaling</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Neural tube defects (NTDs) are one of the most common congenital malformations. Folic acid deficiency in pregnant women increases the risk of developing NTDs; however, the underlying etiology and mechanisms remain elusive. In this study, the role of DNA double-strand breaks (DSBs) in 3D genome organization in NTDs with folate deficiency is reported. The NTD mouse model is burdened with abundant DSBs associated with the disruption of 3D genome organization. DSBs occurring in active genes lead to the stalling of RNA polymerase II (Pol II) and formation of R-loops in the 3D genome. The DSB ratios of the genomic regions negatively correlated with the distance from the transcription start sites of the gene. The DSB ratios of the proximal and distal enhancers are significantly higher and induce the displacement of loops with busy anchors. Furthermore, DSB-associated dysregulation of chromatin loops occurs in neural tube closure-associated genes that are abnormally expressed in human NTDs. Taken together, excessive DSB-associated 3D genome organization disruption within NTDs with folate deficiency contributes to the dysregulation of neural tube closure-associated genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8659be399a2187d7c9eb42445c50713cbee2a3dc" target='_blank'>
              Excessive DNA Double-Strand Breaks-Associated 3D Genome Reorganization Contributes to Neural Tube Defects with Folate Deficiency.
              </a>
            </td>
          <td>
            Ting Zhang, Lin Lin, Jianting Li, Caihua Li, Shengjun Liang, Xuemei Bai, Fang Wang, Yihua Bao, Dan Guo, Xiaochen Bo, Hao Li, Hebing Chen, Qiu Xie
          </td>
          <td>2025-09-18</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Repetitive elements (REs), involving both satellite DNA (satDNA) and transposable elements (TEs), significantly influence the evolutionary dynamics of the host genome. The order Orthoptera, known for its large genomes, serves as an ideal model for studying the relationship between repeat elements and genome size. We conducted a comparative repeatome analysis across ten Tetrigidae species, using low-coverage short reads in RepeatExplorer2 and dnaPipeTE. Our findings revealed that RE content constitutes between 42.82% (in Thoradonta yunnana) and 60.86% (in Saussurella cornuta) of their genomes, with major components including LINEs, LTR/Gypsy, LTR/Copia, and DNA transposons (Maverick and Helitron). We observed a strong positive correlation between repeat and TE content with genome size, suggesting genome expansion in Tetrigidae is likely driven primarily by repetitive DNA, particularly transposable elements. TE divergence analysis, calculated using RepeatMasker, indicates that species with large genomes, such as S. cornuta (2.836 Gb) and Formosatettix gonggashanensis (1.465 Gb), have experienced both ancient and recent bursts of TE activity. Satellitome analysis revealed a diverse range of satDNA families, with Macromotettixoides jiuwanshanensis having the highest number (34 families) and Phaesticus moniliantennatus showing the greatest abundance (16.76%). Additionally, a fossil-calibrated phylogeny based on 13 mitochondrial protein-coding genes (PCGs) enabled ancestral state reconstructions of genome size, repeat content, and TE composition, revealing lineage-specific patterns of genomic evolution. This study highlights the critical role of repetitive DNA, particularly TEs, in driving genomic complexity and expansion, offering key insights into the evolutionary dynamics of Tetrigidae. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17670-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b593b3af0a7ea56ed7a288134f47841e234e817" target='_blank'>
              Evolutionary dynamics of repetitive elements and genome size in Tetrigidae (Orthoptera: Caelifera)
              </a>
            </td>
          <td>
            Muhammad Salman, Ping Wang, Nian Liu, Xuanzeng Liu, Weian Deng, Yuan Huang
          </td>
          <td>2025-10-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Endometrial cancer represents a significant global health challenge, with undifferentiated and dedifferentiated endometrial carcinoma (UDEC) being a particularly aggressive subset. Methods We employed whole-exome sequencing (WES) to comprehensively characterize the molecular landscape of 29 UDECs in a single institution cohort. We analyzed driver mutations, molecular subgroups, SWI/SNF complex, DNA mismatch repair (MMR) pathways, and somatic copy number alterations (SCNAs). Results We identified 5 (17%) ultramutated tumors, 14 (48%) DNA mismatch repair (MMR)-deficient tumors, and 10 (35%) mutation-low tumors in this cohort. Mutations in the SWI/SNF family and other driver genes are common, including PTEN, ARID1A, KMT2B, ARID1B, SMARCA4, and PIK3CA. Genomic inactivation of SWI/SNF complex genes occurred in 19 of 29 (66%) of cases, highlighting the frequent chromatin remodeling dysfunction in UDEC. We observed frequent homopolymer mutations in UDECs with MMR deficiency, with RPL22K15Rfs*5 mutation found in 10 of 14 (71%) MMR-deficient tumors. Notably, ultramutated tumors demonstrated superior survival compared to the other two subtypes. Conclusions Our analysis revealed distinct architectures and actionable alterations in UDEC. The identification of molecular subgroups provides a promising framework for prognostic stratification. Collectively, our findings not only advance our understanding of UDEC biology but also illuminate potential translational applications. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-15053-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc88a4ff5294346452474f0c0aa09fa1c499cb7b" target='_blank'>
              Distinct genomic subgroups and mutational patterns in undifferentiated/dedifferentiated endometrial carcinoma
              </a>
            </td>
          <td>
            Chen-Yang Huang, Angel Chao, C. Lin, Steven G. Rozen, Angel Chao, Chen-Bin Chang, H. Tung, Ren-Chin Wu, Chyong-Huey Lai
          </td>
          <td>2025-10-09</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The ataxia telangiectasia mutated (ATM) kinase orchestrates the early stages of DNA double-strand break repair by promoting hyperphosphorylation of CtIP, a key step in the initiation of DNA end resection. However, the regulatory mechanisms controlling resection extent remain incompletely understood. Here we identify ERCC6L2 as a key regulator of DNA end resection in response to ATM inhibition. ERCC6L2 undergoes liquid–liquid phase separation via its intrinsically disordered regions, forming dynamic nuclear condensates that regulate CtIP stability. Disruption of these condensates renders CtIP susceptible to RNF138-mediated ubiquitination and degradation, thereby mitigating the heightened chromatin recruitment of CtIP induced by ATM inhibition. Intriguingly, ERCC6L2 is frequently downregulated in multiple cancer types and correlates with resistance to ATM inhibitors in both in vitro and in vivo settings. Our findings unveil the crucial role of ERCC6L2–CtIP condensates in governing the extent of DNA end resection and underscore the potential significance of ERCC6L2 as a predictive biomarker for ATM inhibitor response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a07119c6c29989e52a1007b94a87a49c0954c86a" target='_blank'>
              Phase separation of ERCC6L2–CtIP regulates the extent of DNA end resection
              </a>
            </td>
          <td>
            Yixin Yin, Jinlong Lin, Xiaoxia Cai, Runcong Nie, Jiliang Lin, Shuting Li, Z. Xiang, Yihong Ling, Yiyang Zhang, Jie Zhou, Jiewei Chen, Wenping Lin, Jinling Duan, Xueyi Zheng, Dan Xie, Mu-yan Cai
          </td>
          <td>2025-09-05</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="Simple Summary Enhancer of Zeste Homolog 2 (EZH2) is a protein widely recognized for its role in gene silencing and plays a significant role in cancer-related processes, including cell survival, proliferation, invasion, and self-renewal. As the catalytic subunit of Polycomb-Repressive Complex 2 (PRC2), canonically EZH2 mediates the trimethylation of histone H3 at lysine 27 (H3K27), leading to transcriptional repression. Dysregulated EZH2 expression is a hallmark of numerous cancers, both solid and hematologic, and is frequently associated with enhanced metastasis and poor patient prognosis. Notably, EZH2 also exhibits non-canonical functions, including gene activation, which can further promote tumor progression. Due to its significant involvement in oncogenesis and therapy resistance, EZH2 expression is being explored as a diagnostic and/or prognostic biomarker. This review summarizes the function and diverse roles of EZH2 across human cancers, highlighting its potential as both a diagnostic and/or prognostic biomarker. A deeper understanding of EZH2’s intricate regulatory network may enable the development of more effective strategies for managing EZH2-driven malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac9083fda14d6cccbdbf2f40711aac3833884d4a" target='_blank'>
              EZH2 Dysregulation and Its Oncogenic Role in Human Cancers
              </a>
            </td>
          <td>
            Shiv Verma, Nikita Goyal, Suhani Goyal, Parminder Kaur, Sanjay Gupta
          </td>
          <td>2025-09-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="B-cell acute lymphoblastic leukemia (B-ALL) remains a major clinical challenge in hematologic oncology, characterized by a continuous evolution of molecular drivers that shape its heterogeneity across the age spectrum. Pediatric B-ALL is generally associated with high cure rates, while adult forms of the disease are often more aggressive and less responsive to treatment. This review examines the age-specific genetic and epigenetic landscapes that contribute to this disparity, revealing how the nature and timing of molecular alterations point to fundamentally different leukemogenic processes. Favorable genetic aberrations, such as ETV6::RUNX1 and hyperdiploidy, are predominant in children, whereas adults more frequently present with high-risk features, including BCR::ABL1 fusions and IKZF1 deletions. Epigenetic distinctions are similarly age-dependent, involving divergent patterns of DNA methylation, histone modifications, and non-coding RNA expression. For example, pediatric B-ALL frequently harbors mutations in epigenetic regulators like SETD2 and CREBBP, while adult B-ALL is more commonly affected by alterations in TET2 and IDH1/2. These molecular differences are not only prognostic but also mechanistic, reflecting distinct developmental trajectories and vulnerabilities. Understanding these age-driven transitions is essential for improving risk stratification and developing precision therapies tailored to the unique biology of B-ALL across the lifespan.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb81ee6e89e6609c04144954df008c19fbd5a755" target='_blank'>
              Age-Driven Genetic and Epigenetic Heterogeneity in B-ALL
              </a>
            </td>
          <td>
            Yoana Veselinova, M. Esteller, Gerardo Ferrer
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Caliban, the Drosophila ortholog of human Nuclear export mediator factor (NEMF), is a recently identified regulator of the intrinsic apoptotic signaling pathway in response to DNA damage; however, the mechanism governing its expression after DNA damage remains unclear. In this study, we demonstrated that DNA damage upregulated caliban expression concomitant with p53 activation. Over-expression of p53 upregulated the mRNA and protein levels of caliban. We characterized the core region of the caliban promoter, which exhibited enhanced activity following DNA damage or p53 activation. Further analysis of the caliban promoter revealed a p53-binding site that directly interacts with p53 in response to DNA damage. Moreover, mutation of this p53-binding site or knock-down of p53 expression abolished the DNA damage-induced increase in caliban promoter activity, confirming p53’s critical role in regulating caliban expression. Taken together, our findings indicate that caliban is a direct transcriptional target of p53 in response to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdfadba5f968970331f0d881d129c4d34407e435" target='_blank'>
              Caliban is a transcriptional target of p53 in response to DNA damage
              </a>
            </td>
          <td>
            Jiaqian Cui, Haiyan Zhang, Yan Cheng, Xiaolin Bi, Dong Li
          </td>
          <td>2025-08-28</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Topoisomerase 1 (Top1) removes transcription-related helical torsions and thus plays an important role in preventing genome instability instigated by the formation of non-canonical DNA secondary structures. The genetically tractable Saccharomyces cerevisiae model proved effective in defining the critical function of Top1 to prevent recombination and chromosomal rearrangement at G4-forming genomic loci and studying the human cancer-associated Top1 mutants through the expression of analogous yeast mutants. We previously showed that cleavage-defective Top1 mutants strongly elevate the rate of recombination at G4 DNA, which involves binding to G4 DNA and interaction with the protein nucleolin (Nsr1 in yeast). Here, we further explored the mechanism of genome instability induced by the yeast Top1Y740* mutant, analogous to the human Top1W765Stop mutant conferring resistance to CPT. We show that yTop1Y740* elevates duplications as well as recombination specifically at G4-forming sequences. Interestingly, SUMOylation of yTop1Y740*, which does not affect the G4 DNA-binding or Nsr1-interaction by this mutant, is necessary for such elevated G4-specific genome instability. Many tumors with mutations at the C-terminal residues of Top1, particularly W765, have significantly high G4-associated mutations, underscoring the importance of further investigation into how SUMOylation affects the function of these Top1 mutants at G4 DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/414ccc1f67726dc580ae95a50a7d8e620a487775" target='_blank'>
              Mutations and Recombination at G4 DNA-Forming Sequences Exacerbated by CPT-Resistant Mutant Topoisomerase 1 Is Dependent on SUMOylation
              </a>
            </td>
          <td>
            Shivani Singh, Xinji Zhu, Nayun Kim
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background: Bladder cancer is rare in young women, and advanced presentations are exceptionally uncommon. We report a de-identified case of a previously healthy 31-year-old female who developed rapidly progressive stage IV bladder cancer within 12 months of completing a three-dose Moderna mRNA vaccination series (May 2021, June 2021, December 2021). Case Findings: Comprehensive multi-omic profiling was performed using PBIMA (Molecular Surveillance and Individualized Targeted Immunotherapy Peptide Editing) and REViSS (Spike-associated Transcription-al/Translational Instability Surveillance), incorporating analyses of plasma-derived circulating tumor DNA, whole-blood RNA, and urine exosome proteomics. Dysregulated gene expressions were identified across oncogenic driver genes (KRAS, ATM, MAPK1, NRAS, CHD4, PIK3CA, and SF3B1), auxiliary tumor-promoting signals (TOP1, PSIP1, and ERBB2), and broad evidence of genome instability with impaired DNA repair (ATM, MSH2). Within circulating tumor DNA, a host–vector chimeric read mapped to chr19:55,482,637–55,482,674 (GRCh38), in cytoband 19q13.42, positioned ~367 kb downstream of the canonical AAVS1 safe harbor and ~158 kb upstream of ZNF580 at the proximal edge of the zinc-finger (ZNF) gene cluster. This sequence aligned with perfect 20/20 bp identity to a segment (bases 5905–5924) within the Spike open reading frame (ORF) coding region (bases 3674–7480) of the Pfizer BNT162b2 DNA plasmid reference (GenBank accession OR134577.1), despite the patient only receiving Moderna vaccinations. This apparent paradox is best explained by shared Spike ORF sequences within the expression cassette across both vaccine platforms; because Moderna has not deposited its proprietary plasmid sequence in NCBI, BLAST defaults to Pfizer’s published reference as the nearest available match. The integration site was located outside the canonical AAVS1 “safe harbor” and within a gene-dense, recombination-prone regulatory region, raising concern for transcriptional disruption, fusion transcript formation, and oncogenic potential. The probability of a random 20-base sequence perfectly matching a predefined target is approximately 1 in a trillion, making this alignment statistically compelling and highly unlikely to be an incidental artifact. Conclusions: This sentinel case report provides the first documented evidence of genomic integration of mRNA vaccine-derived genetic material in a human subject, documenting a temporal association between COVID-19 mRNA vaccination and aggressive malignancy, reproducible multi-omic evidence of oncogenic signaling, and a non–safe harbor host–vector integration event. While causality cannot be established from a single case, the con-vergence of (i) close temporal proximity to vaccination, (ii) genomic integration of a vaccine plasmid–derived spike gene fragment, and (iii) consistent transcriptomic and proteomic instability across biospecimens represents a highly unusual and biologically plausible pattern. These findings highlight an urgent need for systematic genomic sur-veillance, orthogonal validation with long-read sequencing, and larger cohort studies to rigorously define the im-pact of synthetic mRNA vaccine platforms on genome integrity and cancer risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8848f5c106afb2dfada322a3642c497ad8b443d" target='_blank'>
              Genomic Integration and Molecular Dysregulation in Aggressive Stage IV Bladder Cancer Following COVID-19 mRNA Vaccination
              </a>
            </td>
          <td>
            J. A. Catanzaro, Nicolas Hulscher, Peter A. McCullough
          </td>
          <td>2025-10-05</td>
          <td>International Journal of Innovative Research in Medical Science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Extrachromosomal DNA (ecDNA) drives the evolution of cancer cells. Its widespread presence in tumors and strong association with poor clinical outcomes make ecDNA a promising and broadly applicable therapeutic target. Recent studies have begun to unravel the mechanisms by which ecDNA promotes tumorigenesis and maintains its presence in cancer cells. These discoveries have paved the way for developing ecDNA-targeted therapies. In this Perspective, we summarize the latest advances in our understanding of the mechanism underlying both the ecDNA-induced cancer phenotype and ecDNA maintenance. We also explore potential strategies for targeting ecDNA in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55785ec96f370b7285e3fa5a23ac44e0552ecce9" target='_blank'>
              Unfolding ecDNA as a pan-cancer therapeutic target
              </a>
            </td>
          <td>
            Yang Zhang, Ruirui Chen, Haiyun Gan
          </td>
          <td>2025-10-07</td>
          <td>Medical Review</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Human endogenous retroviruses (HERVs), particularly the HERV-K family, are increasingly recognized for their roles in cancer biology, yet the function of ERVK3-1 (HERVK_3p21.31) in liver hepatocellular carcinoma (LIHC) remains largely unexplored. Methods We analyzed transcriptomic data from the TCGA-LIHC cohort to identify differentially expressed genes (DEGs) between high and low ERVK3-1 expression groups, followed by functional enrichment analyses (GO, KEGG, GSEA), protein-protein interaction (PPI) network construction, and hub gene identification. The immunological relevance of ERVK3-1 was assessed through TIP immune cycle analysis, single-cell RNA sequencing datasets, and correlation with immune checkpoint expression. Immunotherapy responsiveness was evaluated using TIDE and TCIA databases. Results High ERVK3-1 expression was associated with enrichment in metabolic and oxidative stress-related pathways, while low expression correlated with cell cycle and DNA replication. PPI analysis revealed mitosis-related hub genes (e.g., CCNB1, CDK1). ERVK3-1 expression promoted early immune cell recruitment but was inversely correlated with later stages of the cancer immunity cycle, including immune infiltration and T-cell killing. Single-cell data showed high ERVK3-1 expression in immunosuppressive subsets, alongside positive associations with inhibitory immune checkpoints (e.g., PD-1, CTLA-4, TIM-3). High ERVK3-1 expression also correlated with greater immune evasion and reduced immunotherapy responsiveness. Conclusions ERVK3-1 plays a multifaceted role in LIHC progression, contributing to metabolic reprogramming, immune suppression, and resistance to immunotherapy. These findings highlight ERVK3-1 as a potential prognostic biomarker and therapeutic target in liver cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12985-025-02928-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/186dc1e93c9e9bba5bd5b29915ff6bacc720805a" target='_blank'>
              Human endogenous retrovirus ERVK3-1 characterizes a metabolically active and immunosuppressive subtype of liver cancer
              </a>
            </td>
          <td>
            Xiaofen Wen, Shulv Weng, Minna Chen, Danxia Lin, Wenwu Xue, De Zeng, Jiaxin Shen
          </td>
          <td>2025-10-01</td>
          <td>Virology Journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction: Long interspersed element-1 (L1) and Alu elements are reported to regulate tumor development and progression by forming chimeric transcripts with diverse sequences. While these elements have been implicated in various cancers, the landscape of chimeric transcript formation in breast cancer remains poorly characterized.
Objective: Therefore, in this study, we comprehensively investigated the profiles of L1 and Alu chimeric transcripts in breast tumors by analyzing RNA sequences from The Cancer Genome Atlas (TCGA) database.
Methods: More than 50 billion paired-end RNA sequencing reads from 606 primary breast tumor tissues and 59 matched adjacent normal tissues from the TCGA database were aligned to L1 and Alu consensus sequences with Bowtie2 software and further analyzed.
Results: Our analysis revealed significantly elevated expression levels of L1 and Alu retrotransposons in breast tumor tissues compared to adjacent non-tumor tissues. Importantly, higher expression of L1 and Alu in breast tumors was associated with poorer 10-year overall survival, suggesting their potential as prognostic biomarkers. Further exploration demonstrated that L1 and Alu were extensively expressed in chimeric form with adjacent transcripts. A significant difference in the frequency of 651 L1-mRNA and 1,525 Alu-mRNA chimeric transcripts was observed between non-tumor and tumor tissues. Similarly, 1,009 L1-long non-coding RNA (lncRNA) and 2,575 Alu-lncRNA chimeric transcripts exhibited significantly different frequencies between non-tumor and tumor tissues. Functional enrichment analysis revealed that these differentially expressed chimeric genes played crucial roles in multiple facets of tumorigenesis, including metabolism, signal transduction, immunity, cell cycle regulation, and apoptosis.
Conclusion: Our findings demonstrate that Chimeric transcript formation with L1 and Alu retrotransposons represents a widespread phenomenon in breast cancer that impacts multiple oncogenic pathways. These results significantly expand our understanding of retrotransposon involvement in breast cancer and highlight potential novel targets for diagnostic and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99e34ef62da2b088b93dd7d2cad99c0994dc0a0e" target='_blank'>
              Landscape of long interspersed element-1 and Alu chimeric transcripts in breast tumors
              </a>
            </td>
          <td>
            Jian Li, Yin Zhang, Yiting Zhou, Jianyou Liao, Yabin Guo
          </td>
          <td>2025-09-03</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 IDH-mutant gliomas are diffusely infiltrating, slow-growing brain tumors that inevitably undergo malignant progression to grade IV astrocytoma - a highly aggressive and uniformly fatal disease with dismal prognosis. Despite extensive research, the molecular mechanisms underlying this transformation remain poorly understood. While alterations in canonical driver genes such as CDKN2A, PTEN, and PDGFRA have been implicated, the broader landscape of genetic drivers involved in IDH-mutant glioma progression is largely uncharacterized. In contrast to many other malignancies, IDH-mutant gliomas acquire relatively few somatic point mutations during progression - typically in the range of 30 to 40 - suggesting that alternative mechanisms may be driving tumor evolution. Notably, these tumors exhibit widespread chromosomal instability, resulting in a heterogeneous and highly prevalent pattern of copy number alterations (CNAs). This observation raises the possibility that gene dosage changes due to CNAs play a critical role in tumor progression.



 To investigate this hypothesis, we performed integrative analysis of IDH-mutant glioma cohorts from the Mayo Clinic and TCGA datasets (~1,500 patients), identifying twelve highly recurrent CNAs significantly associated with patient outcomes. We refer to these as malignant IDH-associated CNAs (mIDH CNAs). We hypothesized that these recurrent CNAs harbor driver genes that confer selective growth advantages, thereby contributing to their recurrence and prognostic relevance. To systematically assess the functional impact of genes within these regions, we developed a novel in vivo CRISPR-based screening platform, termed CRISPR-Knock Out and Activation Linked Assay (CRISPR-KOALA). This approach utilizes stereotaxic delivery of lentiviral gRNA libraries - targeting mouse orthologs of ~1,800 genes within recurrently deleted regions and ~1,200 genes within amplified regions - into the brains of a genetically engineered mouse model of IDH-mutant glioma (rs557mut;IDHmut;P53KO;AtrxKO;Cas9GFP: Yanchus et. al Science 2022).



 Mice injected with mIDH CNA-targeting libraries developed tumors significantly more rapidly than control cohorts. Deconvolution of the CRISPR-KOALA screen identified several putative driver genes, including regulators of the NOTCH signaling pathway, lipid metabolism modulators such as THEM6, and the NMDA receptor modulator GRINA - implicating dysregulation of metabolism, immune response, and glutamatergic signaling in the malignant transformation of IDH-mutant gliomas.



 Together, these findings provide critical insights into the genetic mechanisms driving the progression of IDH-mutant gliomas and identify new candidate genes and pathways for potential therapeutic targeting.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/174bd23b8ba2dba117306ce08fc0c554d76de475" target='_blank'>
              OS09.5.A RECURRENT COPY NUMBER ALTERATIONS DRIVE IDH-MUTANT ASTROCYTOMA DEVELOPMENT AND PROGRESSION
              </a>
            </td>
          <td>
            S. Haider, K. Drucker, R. Tsai, K. Al-Zahrani, C. Chiu, T. Kollmeyer, G. Zadeh, R. Jenkins, D. Schramek
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) is the most common subtype of ovarian cancer (∼80%) and is one of the most lethal gynecological malignancies in the United States. HGSOC is characterized by universal mutations in the TP53 gene and defects in DNA repair pathways, contributing to high incidence of chromosomal instability (CIN). CIN refers to an ongoing process involving the gain or loss of parts of chromosomes, leading to chromosomal mis-segregation, aneuploidy, and copy number variations (CNV). CINhigh cancers display a greater dependency on KIF18A, an essential cytoskeletal motor protein for cell division, in contrast to normal somatic cells. Therefore, KIF18A represents a promising target for therapeutic interventions in CINhigh cancers. We investigated the use of a novel KIF18A inhibitor (KIF18Ai), ATX020, for selectively targeting CINhigh HGSOC cells. Our approach involved advanced techniques such as RNA sequencing (RNA-seq), siRNA-based gene silencing, cell cycle analysis, and immunofluorescence. We utilized DEPMAP and flow cytometric analysis to categorize ovarian cancer cell lines according to their aneuploidy score (AS) and ploidy to investigate their response to ATX020. Cell growth inhibition studies demonstrated a correlation between AS/ploidy levels and sensitivity to ATX020 in a panel of HGSOC cells including PEO4, OVCAR3, OVCAR8 and A2780. Assessments of cell viability, cell cycle analysis, phospho-histone H3 expression, clonogenic assays, and invasion assays indicated that the growth inhibition observed over 72 hours was primarily due to the toxic effects of ATX020 on tumorigenicity and mitotic arrest. Mechanistically, ATX020 exerts its toxic effects by inhibiting the plus-end movement of KIF18A on spindle fibers, which increases the inter-kinetochore (Inter-KT) distance and induces adverse effects such as chromosomal mis-segregation, aneuploidy, and double-strand DNA damage. We also observed that KIF18Ai-resistant cells (A2780, PEO4) showed inhibited KIF18A plus-end movement and increased inter-KT distance but did not exhibit the accompanying toxic effects. In summary, our findings indicate that ATX020 inhibits CINhigh HGSOC cells primarily by inducing mitotic arrest, causing DNA damage, and reducing tumorigenicity. This effect is facilitated by ATX020's ability to inhibit the plus-end directed movement of KIF18A along spindle fibers, which is detrimental to CINhigh cells that rely on KIF18A for normal mitotic divisions. Further mechanistic investigation is ongoing.



 Jayakumar R. Nair, Maureen M. Lynes, Laura Ghisolfi, Serena J. Silver, Jung-Min Lee. Investigating a novel KIF18A inhibitor ATX020 in chromosomally instable ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/256c1d3be3ec4d6c1fa4908dbe7af1b6e3ef12d5" target='_blank'>
              Abstract B024: Investigating a novel KIF18A inhibitor ATX020 in chromosomally instable ovarian cancer
              </a>
            </td>
          <td>
            Jayakumar R. Nair, Maureen M. Lynes, Laura Ghisolfi, Serena J Silver, Jung-Min Lee
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Pancreatic ductal adenocarcinoma (PDAC) is a highly heterogeneous malignancy, characterized by low tumor cellularity, a dense stromal response, and intricate cellular and molecular interactions within the tumor microenvironment (TME). Although bulk omics technologies have enhanced our understanding of the molecular landscape of PDAC, the specific contributions of non-malignant immune and stromal components to tumor progression and therapeutic response remain poorly understood. Methods: We explored genome-wide DNA methylation and transcriptomic data from the Cancer Genome Atlas Pancreatic Adenocarcinoma cohort (TCGA-PAAD) to profile the immune composition of the TME and uncover gene co-expression networks. Bioinformatic analyses included DNA methylation profiling followed by hierarchical deconvolution, epigenetic age estimation, and a weighted gene co-expression network analysis (WGCNA). Results: The unsupervised clustering of methylation profiles identified two major tumor groups, with Group 2 (n = 98) exhibiting higher tumor purity and a greater frequency of KRAS mutations compared to Group 1 (n = 87) (p < 0.0001). The hierarchical deconvolution of DNA methylation data revealed three distinct TME subtypes, termed hypo-inflamed (immune-deserted), myeloid-enriched, and lymphoid-enriched (notably T-cell predominant). These immune clusters were further supported by co-expression modules identified via WGCNA, which were enriched in immune regulatory and signaling pathways. Conclusions: This integrative epigenomic–transcriptomic analysis offers a robust framework for stratifying PDAC patients based on the tumor immune microenvironment (TIME), providing valuable insights for biomarker discovery and the development of precision immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec3422e79e1fa464a39c9ceade9afd8d95e43c98" target='_blank'>
              Deciphering the Heterogeneity of Pancreatic Cancer: DNA Methylation-Based Cell Type Deconvolution Unveils Distinct Subgroups and Immune Landscapes
              </a>
            </td>
          <td>
            Barbara Mitsuyasu Barbosa, Alexandre Todorovic Fabro, Roberto da Silva Gomes, C. Rainho
          </td>
          <td>2025-09-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="In mammals, repression of germline-specific gene expression is essential for preserving somatic cell identity and preventing disease. Germline gene silencing is often dependent on the presence of promoter 5-cytosine DNA methylation (5meC). Early mammalian development, however, is marked by a dramatic loss of 5meC levels genome-wide. Recent research has highlighted a specialized variant of the Polycomb Repressive Complex 1, PRC1.6, as a key regulator that maintains transient silencing of germline genes in this embryonic window. Eventually, PRC1.6 seems to stimulate the recruitment of de novo DNA methyltransferases (DNMTs), although the precise mechanisms remain to be fully elucidated. Evidence suggests a coordinated epigenetic relay, potentially involving direct protein interactions and shaping the local histone modification landscape. This review explores PRC1.6 as a central epigenetic hub that connects Polycomb repression, histone 3 lysine 9 (H3K9) methylation, and DNA methylation pathways. Unraveling this interplay will provide key insights into the mechanisms that maintain the critical barrier between the germline and the soma, essential not only for proper development but also for preserving somatic cell function and health throughout life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ed478b995bf1b398b0bfe5ba03fe4e97c29208" target='_blank'>
              Epigenetic relay: Polycomb-directed DNA methylation in mammalian development
              </a>
            </td>
          <td>
            Teresa Urli, Maxim V. C. Greenberg
          </td>
          <td>2025-09-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="It has become evident from decades of clinical trials that multimodal therapeutic approaches with focus on cell intrinsic and microenvironmental cues are needed to improve understanding and treat the rare, inoperable, and ultimately fatal diffuse intrinsic pontine glioma (DIPG), now categorized as a diffuse midline glioma. In this study we report the development and characterization of an in vitro system utilizing 3D Tumor Tissue Analogs (TTA), designed to replicate the intricate DIPG microenvironment. The innate ability of fluorescently labeled human brain endothelial cells, microglia, and patient-derived DIPG cell lines to self-assemble has been exploited to generate multicellular 3D TTAs that mimic tissue-like microstructures, enabling an in- depth exploration of the spatio-temporal dynamics between neoplastic and stromal cells. The 3D-TTA model recapitulates clinical patterns of DIPG growth, evidenced by resistance to chemotherapy, HDAC and proteasome inhibitors, as well as sensitization to the antibody-activated innate immune microenvironment including complement proteins and surrounding microglia. Multimodal fluorescence imaging platforms integrated with high-throughput omics revealed that alterations in tumor cell motility and growth in the 3D-TTA model compared to tumor cell only spheroids correlated with specific transcriptomic and proteomic changes. STAT3, ITGA5, LGALS1, SOD2, MVP, and CLIC1, associated with microenvironment signaling, DNA replication, and immune regulation, were identified as potential novel targets in the 3D model. The results indicate that the 3D TTA platform developed here represents a powerful tool for preclinical studies, paving the way for identification/validation of tissue specific biomarkers and novel drug targets, thus advancing disease management strategies for DIPG in children.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b07b858c6fd1cb8139cba6229ad20ebbdfab9d" target='_blank'>
              Multicellular tumor-stromal interactions recapitulate aspects of therapeutic response and human oncogenic signaling in a 3D disease model for H3K27M-altered DIPG
              </a>
            </td>
          <td>
            Meenakshi Upreti, Astgik Petrosyan, M. Thornton, A. Hovsepyan, G. E. Fernandez, David S. Koos, S. Byrum, Samuel G Mackintosh, Jacob K Al-Husseini, Tania Porras, Joseph Ha, A. Tackett, Miqin Zhang, M. Johal, Anat Erdreich-Epstein, Susan Durham, Mark D Krieger, Ashley S Margol, Brendan H. Grubbs, T. Chambers, S. Asgharzadeh, Rex A. Moats, Peter A Chiarelli
          </td>
          <td>2025-09-06</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Gene fusions are a class of important oncogenic drivers, with many matched FDA-approved targeted therapies across multiple solid tumors. However, the prevalence of fusions varies considerably by cancer type and assay. Fusion detection is technically challenging, and studies have shown that RNA-based next-generation sequencing (NGS) can improve fusion detection rates when used in conjunction with DNA-based NGS. In this study, we performed a retrospective pan-cancer analysis of 67,278 patients receiving both RNA- and DNA-NGS in 43 distinct solid-tumor cancer types, including: NSCLC (18.6%), colorectal cancer (18.2%), and breast cancer (13.1%). In this cohort, 1497 patients (2.2%) had at least one of nine fusions detected-each having an FDA-approved matched therapy in at least one indication. Three hundred and sixteen patients (21.1%) had a fusion detected (RET or NTRK1/2/3) with matched-targeted therapy approved in all cancer indications. Concurrent RNA- and DNA-NGS increased the detection of driver gene fusions by 21% compared to DNA-NGS alone. Gene fusions were observed in a range of cancers beyond their approved cancer indications: of 1,501 fusions detected, 29% (n=437) were detected outside of an FDA-approved indication. Finally, emerging fusion drivers with targets in drug development were found in an additional 218 patients, with combined RNA- and DNA-NGS increasing detection of these variants by 127%. Our findings support combined RNA-NGS and DNA-NGS to maximize detection of clinically actionable fusions with FDA-approved matched therapies, and potentially actionable fusions in non-FDA-approved indications or those matched to therapies in clinical development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/530d6e9ee73f2135b86fd5e121c7010a80e6616d" target='_blank'>
              Molecular characterization of oncogenic gene fusions in a large real-world cohort of solid tumors.
              </a>
            </td>
          <td>
            Lisa Gai, Bradley Bowles, A. Hockenberry, Brittany Mineo, Christine Chin, K. Sasser, H. Nimeiri, Kyle A Beauchamp, Rotem Ben-Shachar, J. Guinney, S. Patel, Ben Ho Park
          </td>
          <td>2025-10-15</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Purpose Microsatellite instability-high colorectal cancer is characterized by hypermutated genomes and high neoantigen loads, yet a significant proportion of patients exhibit resistance to immune checkpoint blockade. This study aims to investigate tumor cell functional heterogeneity and its role in immune evasion. Patients and Methods We integrated single-cell RNA sequencing, spatial transcriptomics, and bulk RNA sequencing from microsatellite instability-high colorectal cancer patients. After quality control, normalization, and clustering, malignant epithelial subpopulations were identified through copy number variation analysis and non-negative matrix factorization. Functional characterization employed gene set enrichment analysis. Spatial transcriptomics clarified immune cell and tumor subpopulation localization, and survival analyses assessed prognostic implications. Results We identified a glycolysis-enriched tumor subpopulation (MP2) that co-localized with immunosuppressive niches marked by Treg accumulation, effector T-cell depletion, and FOLR2+ tumor-associated macrophages. MP2-high tumors were associated with immune checkpoint blockade resistance and poor prognosis. Mechanistically, MP2 cells secreted lactate, promoting Treg differentiation and macrophage polarization toward an immunosuppressive phenotype. Spatial transcriptomics revealed the precise organization of these lactate-rich, immune-excluded niches within tumors. Conclusion These findings establish tumor cell-intrinsic glycolysis as a key driver of immune evasion in microsatellite instability-high colorectal cancer and propose metabolic targeting as a strategy to overcome immune checkpoint blockade resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/773cdb1598a2f7975562a7cc2f1fda279ad89676" target='_blank'>
              Glycolysis-Driven Immune Evasion in Microsatellite Instability-High Colorectal Cancer: An Integrated Single-Cell and Spatial Transcriptomics Study
              </a>
            </td>
          <td>
            Chenchen Li, Peicong Cai, Hengda Zeng, Jianxia Li, Huabin Hu, Jianwei Zhang, Zehua Wu, Ge Qin, Yanhong Deng
          </td>
          <td>2025-09-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Genomic drivers in glioblastoma (GB) are associated with specific transcriptional states. Within tumour microenvironment architecture however, malignant transcriptional states are largely determined by local cues such as hypoxia or immune infiltration. To what extent genomic drivers affect the TME architecture remains unknown. To investigate the effect of different genomic drivers on TME architecture in GB, we used a replication-competent avian-like sarcoma virus system to introduce EGFRvIII expression, NF1 loss and PDGFB overexpression in a tp53-/- mouse model with Nestin promoter-dependent tumor virus A expression. These genetic alterations recapitulated known classical, mesenchymal, and proneural transcriptomic GB subtypes. We leveraged single cell RNA sequencing and combined spatial transcriptomics and proteomics data at regional (NanoString GeoMx) and single cell (10x Visium HD) level to acquire an in-depth characterization of the differences in the TME organization. Our results revealed intricate dependencies of the GB TME architecture on genetic backgrounds. At single cell level, NF1- and EGFRvIII- driven tumours overrepresented mesenchymal- and astrocyte-like states, whereas PDGFB-driven tumours showed increased neuronal progenitor states. Regardless these evident transcriptional differences, cellular neighbourhoods within the TME locally showed conserved spatial compositions across all samples, with nuanced distinctions between genomic drivers; our integration of spatial transcriptomics and proteomics data showed that EGFRvIII-driven tumours contained complex perivascular immune neighbourhoods with high clustering of plasma B and antigen presenting cells. NF1 driven tumours contained strong perivascular mesenchymal tumour cell enrichment with wound healing features. PDGFB driven tumours increased neuronal signalling with markedly decreased immune infiltration. To extrapolate our findings to human glioblastoma, we leveraged a cohort of 50 spatial transcriptomics samples with known genomic alterations in EGFR, NF1 or PDGFB. Using a graph neural network approach, we were able to reliably predict which cellular neighbourhoods represented genomic alterations. Strikingly, both in mouse and human settings, EGFR, NF1 and PDGFB alterations showed enrichment of immune, tissue reactive and neurodevelopmental organizational structures. By employing sophisticated model systems alongside high dimensional spatial interrogations, we showed how the TME organization is dependent on a complex interplay between cancer cell intrinsic features and its TME. While spatial tumour architecture largely relies on external factors, genomic drivers impose preferred TME structures across mouse and human glioblastomas that could be important in future therapy designs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd60449c2037e91ea85fb1a099b133b1ae3a448" target='_blank'>
              JS09.7.A GENETIC DRIVES PROMOTE DISTINCT MICROENVIRONMENT ORGANIZATION IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            J. Zhang, T. Joshi, D. Heiland, D. Hambardzumyan, L. van Hijfte
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c132647a53f0f45522e7759c35b22627e428420" target='_blank'>
              The value of optical genomic mapping in a patient with azoospermia and a complex chromosomal rearrangement.
              </a>
            </td>
          <td>
            A. Clément, Samira Ahmed‐Eli, Dana Jaber, L. E. Khattabi, P. Clément, François Vialard
          </td>
          <td>2025-09-10</td>
          <td>Journal of assisted reproduction and genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract The O6-methylguanine-DNA methyltransferase (MGMT) plays a significant role in the pathogenesis and progression of glioma. Numerous enhancer variants, including those within the MGMT gene region and adjacent gene regions, have been found to be associated with cancer development and progression. We investigated the significance of enhancer variants located in the intergenic spacer far from the MGMT gene in relation to glioma susceptibility and progression. We recruited 402 glioma patients and 654 controls for this investigation using Sequenom MassARRAY genotyping. We identified a significantly elevated risk of glioma among carriers with the rs11016629 TG genotype compared to those with the GG genotype (OR = 1.41, 95% CI 1.03–1.93; P = 0.034). Subgroup analyses revealed that rs11016629 was significantly associated with glioma risk in subjects with WHO grade IV tumor (OR = 1.59, 95% CI 1.07–2.38; P = 0.023) and high-grade glioma (OR = 1.57, 95% CI 1.11–2.21; P = 0.011). Patients who underwent gross total resection with TG/TT genotypes exhibited a 2.66-fold higher risk of disease progression than GG carriers (HR = 2.66, 95% CI 1.23–5.79; P = 0.014). The study demonstrates that a MGMT enhancer variant rs11016629 contributes to both glioma susceptibility and progression. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1007/s10571-025-01605-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78ffd3767daacb020b869e9275c5c5550f74b90c" target='_blank'>
              A MGMT Enhancer Variant is Associated with Glioma Susceptibility and Progression
              </a>
            </td>
          <td>
            Shu Zhang, Yiji Chen, Wenshen Xu, Danfang Yan, Meiqin Chen, Jiayi Zheng, Liming Huang
          </td>
          <td>2025-10-16</td>
          <td>Cellular and Molecular Neurobiology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hepatitis B virus (HBV) integration is a key driver of hepatocellular carcinoma (HCC) occurrence and progression; however, its oncogenic mechanisms remain incompletely understood because of limitations in detection methods and sample availability. In this study, we employed Oxford Nanopore Technologies (ONT) whole-genome sequencing and full-length transcriptome sequencing to characterize HBV integration events at the genomic and transcriptomic levels, along with their regulatory effects on structural variations (SVs) and gene expression. Functional validation was performed using dual-luciferase assays and cell-based experiments. Our findings revealed that integrated HBV sequences form long concatemers, mediating inter- and intrachromosomal recombination in the human genome. Notably, integrated HBV enhancer I (HBV-Enh I) was detected in 6 of 7 tumor tissues and was associated with aberrant gene expression. HBV integration induced oncogenic SVs, such as focal MYC amplification and NAV2 deletion, and directly modulated gene expression. Additionally, ectopic overexpression of MYOCD, driven by HBV-Enh I integration, promoted HCC cell migration and invasion. In summary, HBV integration acts as a major driver of large-scale genomic SVs and transcriptomic dysregulation, through either direct alterations in genome dosage or cis-regulatory mechanisms. HBV-Enh I is frequently integrated in HCC and might play a pivotal role in abnormal gene expression, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/428a53d8eb2aca52cd5c27a1ec4e813d0e350e58" target='_blank'>
              Long-read sequencing reveals HBV integration patterns and oncogenic impact on early-onset hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yao Wang, Dong Yu, Yue Mei, Zhida Fu, Jian Lin, Di Wu, Yuan Yang, Hongli Yan
          </td>
          <td>2025-09-16</td>
          <td>Genome research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af9a242ccc6ebd9d79ab2d3fe8d1c436d8e52055" target='_blank'>
              Enabling whole-genome DNA methylation-based classification of central nervous system tumors
              </a>
            </td>
          <td>
            Jonghoon Lee, Young Seok Ju, B.-L. Oh, S. Choi, Se Hoon Kim, Ji Won Lee, J. Phi, Y. Jung
          </td>
          <td>2025-09-21</td>
          <td>None</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Transcription factor E3 (TFE3) oncofusions are frequently detected in the Microphthalmia transcription factor (MiT) family translocation renal cell carcinoma (tRCC), a rare pediatric renal cancer with limited treatment options. The mechanisms by which TFE3 oncofusions promote tRCC malignancy remain inadequately defined. Here, we demonstrate that the RNA-binding capability conferred by TFE3 fusion partners drives the formation of TFE3 condensates. This further enables TFE3 oncofusions to co-condensate with RNA polymerase II (RNAPII) and other RNA-binding proteins, such as paraspeckle component 1 (PSPC1), ultimately driving the formation of transcriptional hubs to promote pro-oncogenic transcription. Dissolution of oncofusion condensates through nanobody-based chemogenetic manipulation effectively curtails tRCC cell growth both in vitro and in vivo, suggesting the therapeutic potential for targeting oncofusion condensation in tRCC. Collectively, our study establishes the causal role of RNA and RNA-binding proteins in facilitating oncofusion condensation to promote renal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79cdc6e21b779dea89a691271b52beca3b734a9" target='_blank'>
              RNA-mediated condensation of TFE3 oncofusions facilitates transcriptional hub formation to promote translocation renal cell carcinoma
              </a>
            </td>
          <td>
            Lei Guo, Rongjie Zhao, Yi-Tsang Lee, Junhua Huang, James Wengler, Logan Rivera, Tingting Hong, Tianlu Wang, Kunjal Rathod, Ashley Suris, Yitian Wu, Xiaoli Cai, Rui Wang, Yubin Zhou, Yun Huang
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background EME1, a critical DNA repair endonuclease, has emerged as a potential oncogene implicated in genome instability and cancer progression. However, its pan-cancer roles, prognostic significance, immune interactions, and therapeutic targeting remain underexplored. Methods We conducted a comprehensive pan-cancer analysis integrating multi-omics data from public databases, including TIMER2.0, GEPIA2, TISIDB, and cBioPortal, to evaluate EME1 expression, genetic alterations, and their association with clinical outcomes, immune infiltration, and molecular pathways. Virtual screening of 3180 FDA-approved drugs and molecular dynamics (MD) simulations were employed to identify and validate potential EME1 inhibitors. Results EME1 was significantly overexpressed in various human cancers and positively associated with advanced tumor grade and stage. High EME1 expression and mutations were linked to poor overall and disease-free survival. Immunogenomic profiling revealed strong positive correlations between EME1 and myeloid-derived suppressor cells (MDSCs), alongside a negative association with endothelial cell function, suggesting immunosuppressive roles. Machine learning models based on EME1-associated genes demonstrated high predictive accuracy for liver hepatocellular carcinoma (AUC > 0.90). Virtual screening identified eight promising drug candidates, including Everolimus and Dioscin, with strong binding affinities. MD simulations confirmed the stability of these interactions, particularly for Dioscin. Conclusion This study reveals the multifaceted oncogenic roles of EME1 in tumor progression, immune evasion, and prognosis. It proposes EME1 as a promising biomarker and therapeutic target across multiple cancer types. The identified drug candidates warrant further in vitro and in vivo validation for potential repurposing in EME1-targeted cancer therapy. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03631-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2914997c524dd90bc8f262d9d5d651e9af6b9f8" target='_blank'>
              Oncogenic EME1 promotes tumor progression and immune modulation in human cancers with therapeutic targeting potential
              </a>
            </td>
          <td>
            Muhammad Alaa Eldeen, Abdelrahman Mostafa, Farag Mamdouh, Walid K Abdulsahib, D. Alshaya, Eman Fayad, Hassan M. Otifi, Hesham M. Hassan, Mohammed A. Alshehri, A. Althobaiti, Ghadi Alsharif, Mohamed A. Soltan
          </td>
          <td>2025-10-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Tumor metastasis is a major contributor for mortality in patients with pancreatic ductal adenocarcinoma (PDAC). While circular RNAs (circRNAs) have emerged as pivotal regulators of tumor progression and metastasis, their functional roles in PDAC remain poorly understood. Through comprehensive circRNA profiling of 10 pairs of PDAC tumors and adjacent normal tissues, we identified circPRKD3 (hsa_circ_0000992) as being substantially up-regulated in malignant specimens. Functional characterization demonstrated that circPRKD3 overexpression potently enhanced cellular migration, invasion, and metastatic capacity in vitro and in vivo, whereas its knockdown produced opposite phenotypic effects. Mechanistic investigations revealed that circPRKD3 directly interacted with the oncogenic RNA-binding protein Musashi-2, protecting it from β-TRCP-mediated ubiquitination and subsequent proteasomal degradation. Clinical correlation analysis revealed a close association between elevated circPRKD3 expression and shorter survival of PDAC patients. Notably, we validated the translational potential of circPRKD3 as a liquid biopsy marker, showing that serum detection, when combined with conventional biomarkers (CA19-9, CEA, and CA125), dramatically improved diagnostic performance. These findings not only delineate a novel circRNA-mediated regulatory axis in PDAC metastasis but also identify circPRKD3 as a promising diagnostic and prognostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1441280c9bbaab596bfcee43907f7ebe16003d78" target='_blank'>
              A Circular RNA Promotes Tumor Metastasis through Stabilizing MSI2 Protein in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Zhang Li, Lanyang Gao, Yang Yang, Yue Ming, Wenrong Liu, Tingting Zhang, Zixia Ye, Fuyan Xu, Juan He, Jiao Li, Jiawei Guo, Xiaojuan Yang, Qing Zhu, Yong Peng
          </td>
          <td>2025-09-10</td>
          <td>Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Mouse embryonic stem cells (ESCs) occasionally transit into a rare two-cell-like (2C) state characterized by transient activation of endogenous retroviruses such as MERVL and expression of 2C-specific genes including the Zscan4 cluster. These 2C-like cells (2CLCs) resemble early blastomeres and display expanded developmental potential, but their unstable and sporadic nature has hindered mechanistic studies. Here, we establish stable 2CLCs (s2CLCs) that maintain persistent MERVL expression and homogeneous 2C gene activation. Live-cell imaging revealed uniform and sustained MERVL activity in s2CLCs, contrasting with the heterogeneous and transient expression observed in conventional ESCs. Transcriptome profiling demonstrated robust induction of 2C-specific regulatory networks, and embryoid body differentiation combined with machine learning uncovered increased lineage variability and expanded developmental trajectories. Epigenomic profiling further revealed unique chromatin states, distinctive super enhancer landscapes, and active enhancer marking at MERVL loci. Together, these findings demonstrate that stable maintenance of the 2C-like state is achievable in vitro, providing a powerful model to dissect ERV-driven transcriptional regulation, epigenomic remodeling, and totipotent-like developmental potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef0d205ec741e491b63d47b95a2bdcf988b0e740" target='_blank'>
              Stable Maintenance of Two-Cell-Like Cells from Embryonic Stem Cells Reveals Chromatin and Super Enhancer Regulation of MERVL Elements
              </a>
            </td>
          <td>
            Rui Geng, Benjamin L. Kidder
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA), shed by malignant tumor cells into extracellular fluid, provides valuable epigenetic information indicative of cancer status. Nipple aspirate fluid (NAF), a noninvasive liquid biopsy from at-risk women, contains nucleic acid and protein biomarkers from adjacent cancer cells, showing promise for breast cancer (BrC) detection. However, despite its potential, the application of cfDNA in NAF for BrC screening is still underexplored. Here, we report a proof-of-concept study for using cfDNA bisulfite sequencing (cfBS) to assess tumor DNA methylation signatures from NAF samples. For four healthy individuals and three BrC patients, cfBS achieved greater than 20× sequencing depth with an average coverage of 26.5× on the genome. A total of 7471 differentially methylated regions were identified, with significant hypermethylation in BrC samples compared to healthy controls. Gene set enrichment analysis indicated that the differentially methylated genes (DMGs) were significantly associated with epithelial–mesenchymal transition (EMT). By developing a novel EMT scoring metric, we found that BrC samples had more of a mesenchymal phenotype than samples from healthy individuals. CDH1, WNT2, and TRIM29 were hypermethylated near the promoter region, while COL5A2 was hypermethylated in the coding region. The DNA methylation and EMT changes were validated through The Cancer Genome Atlas Breast Invasive Carcinoma study, which confirmed that DMGs were associated with gene expression change and that our methylation-based EMT score reliably distinguished tumors from healthy controls. Our findings support the utilization of the NAF cfDNA cfBS methylation profile for noninvasive BrC screening and pave the way for enhanced early detection of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/910913725dce178fd3c9aae5a2e95c8cd7107e54" target='_blank'>
              Cell-Free DNA Bisulfite Sequencing Reveals Epithelial–Mesenchymal Transition Signatures for Breast Cancer
              </a>
            </td>
          <td>
            M. S. Jeon, Zehuan Ding, Casey Pei, Jing Li, Linglin Xie, Edward Sauter, Ke Kurt Zhang
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Understanding genetic dependencies in cancer is key to identifying novel actionable drug targets to advance precision medicine. Whole-genome CRISPR-knockout library screening methods have facilitated this goal. Pooled libraries of single guide RNAs (sgRNAs) targeting over 90% of the annotated protein coding genome are used to induce gene knockouts in pre-clinical cancer models. Novel genes of interest are identified by evaluating sgRNA dropout or enrichment following selection pressure application. This method is particularly beneficial for researching cancers where effective treatment strategies are limited. One example of a commonly chemoresistant cancer, particularly at relapse, is the low survival malignancy epithelial ovarian cancer (EOC), made up of multiple histotypes with distinct molecular profiles. CRISPR-knockout library screens in pre-clinical EOC models have demonstrated the ability to predict biomarkers of treatment response, identify targets synergistic with standard-of-care chemotherapy, and determine novel actionable targets which are synthetic lethal with cancer-associated mutations. Robust experimental design of CRISPR-knockout library screens, including the selection of strong pre-clinical cell line models, allows for meaningful conclusions to be made. We discuss essential design criteria for the use of CRISPR-knockout library screens to discover genetic dependencies in cancer and draw attention to discoveries with translational potential for EOC. Supplementary Information The online version contains supplementary material available at 10.1007/s13402-025-01102-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c4704796a9cf8557ad095d79824927f1f6b68e0" target='_blank'>
              Revealing genetic drivers of ovarian cancer and chemoresistance: insights from whole-genome CRISPR-knockout library screens
              </a>
            </td>
          <td>
            Tali S. Skipper, Kristie-Ann Dickson, Christopher E. Denes, Matthew A. Waller, Tian Y. Du, G. Neely, N. Bowden, Alen Faiz, Deborah J. Marsh
          </td>
          <td>2025-08-28</td>
          <td>Cellular Oncology (Dordrecht, Netherlands)</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a4fa771302c68d0f49b58a8ac556af478b3c9db" target='_blank'>
              DUX4-rearranged B-ALL: deciphering a biological and clinical conundrum.
              </a>
            </td>
          <td>
            Jack Bakewell, Anthony V Moorman, S. Ryan
          </td>
          <td>2025-09-12</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Our previous work demonstrated that oncogenic KRAS mutations in pancreatic cancer induce aberrant transcription-replication conflicts (TRCs). TRCs arise when DNA replication forks collide with transcription complexes due to loss of spatial and temporal coordination between these processes. TRC resolution mechanisms are and adaptive dependency to KRAS-mediated oncogenic stress. Here we identify a key pathway that regulates TRCs in PDAC and performs mechanistic studies towards clinical translation.



 TRC factors were identified through an siRNA screen against 200 DNA repair genes in Panc1 cells using a high-throughput proximity ligation assay (HiPLA). Knock-out and complementation experiments were performed using CRISPR-Cas9 systems. Cell proliferation was measured using incucyte. Replication stress and DNA damage were measured using multiple orthogonal methods. Protein-protein interactions were assessed by co-immunoprecipitation mass spectrometry and validated by western blot. Chromatin occupancy of key factors was determined using ChIP-seq.



 Through high-throughput proximity ligation assay (HiPLA) screening of 200 known DDR genes, we identified the BER pathway as a key mechanism for TRC resolution. Among these, Neil2 - a bifunctional glycosylase in the base excision repair (BER) pathway - emerged as one of the highest-scoring BER factors; and the only BER glycosylase involved—likely due to its transcription-coupled repair activity. Using CRISPR/Cas9-mediated knockdown, we found that loss of Neil2 led to elevated double-stranded DNA breaks (DSBs) and replication stress. Notably, this significantly impaired cell cycle progression. colony formation, and proliferation in pancreatic cancer cell lines, with minimal effects on normal pancreatic epithelial cells, highlighting Neil2’s selective oncogenic function. To elucidate the underlying mechanisms, we combined proteomic approaches—including co-immunoprecipitation and iPOND-mass spectrometry—with transcriptomic and chromatin profiling with ChIP-seq and RNAseq. Our data revealed that Neil2 plays a critical role in transcription termination, and that its dysregulation promotes RNA Polymerase II readthrough, likely contributing to R-loop accumulation, downstream TRCs, and subsequent genotoxicity. Intriguingly, deletion of Neil2’s DNA binding domain—which abolishes its DNA lesion binding capability—did not compromise its regulation of TRCs or support of cancer cell viability. This suggests that Neil2’s oncogenic role in pancreatic cancer may be DNA-binding independent.



 We uncovered a noncanonical and essential function of Neil2 in TRC regulation and oncogenic maintenance in pancreatic cancer, and we propose Neil2 as a promising therapeutic target for selective intervention in this disease.



 Fan Meng, Mustafa Raoof. Neil2 is a potential theratputic targert in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d45ba09d3e2c9d65e7a73127234b8bec15900c3d" target='_blank'>
              Abstract A041: Neil2 is a potential theratputic targert in pancreatic cancer
              </a>
            </td>
          <td>
            Fan Meng, Mustafa Raoof
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Four decades ago, the discovery of centromere protein-A (CENP-A) marked a pivotal breakthrough in chromosome biology, revealing the epigenetic foundation of centromere identity. CENP-A, a histone H3 variant, directs the formation of the microtubule-binding kinetochore complex, designating the chromosomal site for its assembly and underpins the accurate partitioning of genetic material during cell division. Errors in cell division can give rise to DNA instability and aneuploidy, implicated in human diseases such as cancer. Therefore, discovering the underlying pathways and mechanisms responsible for the formation, regulation and maintenance of the centromere is important to our understanding of genome stability, epigenetic inheritance, and in providing the knowledge to help generate possible treatments and therapeutics. Here, we review various molecular pathways and mechanisms implicated in maintaining centromere identity and highlight some of the key outstanding questions with a focus on the human centromere.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/481df10bbcc98403846b3169d8fe656f17c65c42" target='_blank'>
              Preserving centromere identity: right amounts of CENP-A at the right place and time
              </a>
            </td>
          <td>
            Zofia Pukało, Bethan Medina-Pritchard, M. A. Abad, A. A. Jeyaprakash
          </td>
          <td>2025-09-30</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Most of the human genome is transcribed into noncoding RNAs (ncRNAs), which although do not encode proteins, are often involved in key cellular processes. Recent technological advances have accelerated the discovery of ncRNA as well as our understanding of ncRNAs as cancer biomarkers and targets. In this “In the Limelight” special issue of FEBS Open Bio, four review articles by leading experts provide an overview of the role of ncRNAs in cancer, from their association to liver cancer to their role as oncogenes in head and neck squamous cell carcinoma (HNSCC).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39d403967dd7059c9aa239237ba085b5894f7584" target='_blank'>
              Non‐coding RNAs as key player in cancer diagnosis and treatment
              </a>
            </td>
          <td>
            Marcin Majka
          </td>
          <td>2025-09-01</td>
          <td>FEBS Open Bio</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="NUPTs are DNA sequences of plastid origin present in plant nuclear genomes to varying, though typically low, amounts. It is assumed that they are continuously formed and, due to their potentially mutagenic effect, removed at a constant turnover rate, which should result in an exponential decay of their age distributions and a negative correlation between age and size. However, these assumptions are based on analysis from a limited number of species and have never been explicitly tested. To gain insight into the mechanisms driving the origin and evolution of NUPTs, here we surveyed the plastid and nuclear genomes of 30 species representing the main angiosperm (flowering plants) lineages. By modeling the distribution of ages and sizes, examining their linear arrangement across the plastid genome, and statistically assessing spatial biases with respect to other genomic features, we showed that NUPTs are i) formed by both continuous and episodic mechanisms; ii) unevenly represented across the plastid genome; iii) consistently associated with certain classes of RNA genes, in particular rRNA, tRNA and regulatory RNA genes; iv) differentially contributing to structural genes; and v) closer than expected to different superfamilies of transposons in a species-specific manner. Our results reveal the unexpected complexity in the mechanisms driving the origin of NUPTs, which do not only involve their continuous formation but also episodic, highlight their role as a major source of non-coding RNA genes and other genomic features and provide a more complete picture of the different drivers of evolutionary change at the genome level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cddf52eeced1161dba2e2d7fa0a44d7221d0674f" target='_blank'>
              Episodic and ongoing mechanisms drive plastid-derived nuclear DNA evolution in angiosperms.
              </a>
            </td>
          <td>
            Juan Pablo Marczuk-Rojas, Ana D Maldonado, Lorenzo Carretero-Paulet
          </td>
          <td>2025-10-13</td>
          <td>Genome biology and evolution</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Lysine methyltransferase 5a (KMT5a) plays a key role in the pathogenesis of many human diseases. Here, we review the diverse impacts of KMT5a activity on human cancer development and progression. First, KMT5a is the only Mammalian enzyme that specifically induces monomethylation of histone 4 (H4) on lysine 20, thus regulating chromatin organization and, in turn, the transcription of several oncogenes and tumor suppressor genes. KMT5a, by inducing H4 methylation, also critically establishes the choice between different pathways of DNA double-strand break repair, with important consequences for genomic instability and cancer origin. Finally, KMT5a also methylates lysine residues on nonhistone proteins, and KMT5a-induced methylation of key oncogenic and tumor suppressor proteins, including TP53, strongly affects cancer cell functions. Overall, KMT5a is overexpressed in a high percentage and wide variety of human cancers and has protumorigenic activity, which makes it a target for innovative therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0d2924a6f5a75832faa0ef5f5bf3e3b48c7c2e1" target='_blank'>
              Multiple functions of the lysine methyltransferase KMT5a in cancer: potential targets for innovative therapies
              </a>
            </td>
          <td>
            R. Della Monica, Michela Buonaiuto, Mariella Cuomo, Davide Costabile, Claudio Schonauer, G. Catapano, Lorenzo Chiariotti, Roberta Visconti
          </td>
          <td>2025-09-29</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Each cell type exhibits a unique chromatin organization and transcriptome that define its identity and function. Chromatin architectural proteins like CTCF and the cohesin complex are pivotal in shaping this regulatory landscape. Understanding how they drive cell-type-specific gene expression is essential for decoding complex traits and designing targeted therapies for diseases rooted in cellular identity. Endothelial cells (ECs), which line blood vessels and maintain vascular homeostasis, are key regulators of blood pressure (BP) and possess distinct transcriptional programs and
 cis
 -regulatory elements governed by chromatin architecture.

 We hypothesized that CTCF and RAD21 (a cohesin subunit) play specialized roles in establishing the EC-specific transcriptome by modulating chromatin organization. To test this hypothesis, we depleted CTCF and RAD21 in human induced pluripotent stem cells (iPSCs) and iPSC-derived ECs (iECs), representing undifferentiated and lineage-committed states, respectively. We performed RNA sequencing to assess transcriptional changes and CUT&Tag to profile CTCF and RAD21 binding in both cell types.
 Transcriptomic analysis revealed that both factors regulate EC-specific gene expression through shared and distinct pathways. Consistent with these transcriptional changes, CTCF or RAD21 depletion in iECs led to prominent loss of binding at promoters, suggesting direct regulatory roles, whereas in iPSCs, loss occurred mainly at intronic regions, indicating broader structural functions in the undifferentiated state. We also identified a potential EC-specific association between RAD21 and the ETS transcription factor FLI1, suggesting a novel regulatory axis in vascular gene control. Supporting these distinct roles, integration with Micro-C data showed that CTCF- and RAD21-sensitive loops in iECs were largely promoter-centric, whereas in iPSCs, RAD21-sensitive loops engaged more with non-promoter regions. Notably, we identified disrupted CTCF/Rad21 binding promoter regions that harbor SNPs associated with BP and vascular traits, linking chromatin architecture to noncoding genetic risk.
 In conclusion, our study reveals distinct yet cooperative roles of CTCF and RAD21 in shaping EC-specific gene regulation. These findings provide insight into chromatin-mediated vascular identity and highlight regulatory loci with therapeutic relevance to hypertension and vascular disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34ee78ddac4277d3f74eaac1bb7823a42a27b02a" target='_blank'>
              Abstract TH192: CTCF and Cohesin Mediated Endothelial Cell-specific Chromatin Organization and Transcriptional Control at Loci Harboring Blood Pressure-Associated Genetic Variants
              </a>
            </td>
          <td>
            Atrayee Ray, Rajan Pandey, Josiah Murray, C. Stelloh, Pengyuan Liu, Yong Liu, Qiongzi Qiu, Paul Auer, Michael E. Widlansky, A. Geurts, Allen W Cowley, A. Kwitek, Andrew S. Greene, Mingyu Liang, Sridhar Rao
          </td>
          <td>2025-09-01</td>
          <td>Hypertension</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Retroelements are repetitive sequences that comprise 42% of the human genome and are strictly regulated through various epigenetic mechanisms. Examining retroelement expression on a locus-specific level in relation to cancer can uncover distinct disease signatures. Results Using over 5000 RNA-sequencing samples, we assessed retroelement transcription across 23 tissue systems, 159 cell types, 1019 cancer cell lines, and cells isolated from various stages of embryogenesis using the specialized software tool, Telescope. In healthy individuals, 11,388 retroelements were found to be actively transcribed and dynamically regulated in a tissue- and cell type-dependent manner. Using the adult human body as a reference, we observed that 94% of cancer cell lines displayed elevated transcription of at least one cancer-specific retroelement, providing a three-fold larger reservoir of cancer biomarkers (1182) than our comparable analysis of annotated protein-coding genes (338). The precise retroelements that were transcribed following tumorigenesis were influenced by the originating location, with cancers of the blood, lungs, and soft tissue displaying the highest origin-specific activation. Moreover, nearly half of the cancer-specific retroelement loci, mostly from the HERV-H family, were found to be expressed during early embryonic development. Conclusions Our data demonstrate that elevated transcription of certain tissue-specific and embryonic retroelements can be considered as a hallmark of tumorigenesis. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00376-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e92e95f88b68527f2a00c19cdab4d4246b04c" target='_blank'>
              Strict retroelement regulation is frequently lost following cancer transformation and generates a promising reservoir of cancer biomarkers
              </a>
            </td>
          <td>
            Eric Russ, Sergey Iordanskiy
          </td>
          <td>2025-10-03</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA methylation inhibitors are widely used in treating myeloid malignancies, yet their precise effects on chromatin organization and nuclear architecture remain incompletely understood. Here, the integrated molecular, cellular, and biophysical approaches to investigate the impact of azacitidine (AZA) and decitabine (DEC) on chromatin structure and nuclear mechanics in AML-007 leukemia cells are presented. Confocal microscopy revealed drug-induced alterations in nuclear morphology and actin cytoskeleton organization, with DEC inducing significant nuclear enlargement and disorganization at lower concentrations (1.0 µM) compared to AZA (5.0 µM). Chromatin condensation assays demonstrated that DEC increased chromatin accessibility in a concentration-dependent manner, while AZA produced subtler effects. Optical tweezers measurements showed both agents reduced nuclear stiffness, with DEC exerting a greater impact. Spectroscopic analysis confirmed differential drug incorporation into DNA, with higher methylation loss and structural changes observed following DEC treatment. Refractive index mapping revealed chromatin decompaction, aligning with increased accessibility and nuclear softening. These findings demonstrate that DNA hypomethylating agents exert distinct, concentration-dependent effects on nuclear organization and chromatin structure, which can be quantified through molecular and biophysical readouts. This study underscores the value of integrative methods for revealing epigenetic drug effects on chromatin architecture in leukemia cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4906405a8ab8442becabd2d89c99471c5e4596eb" target='_blank'>
              Effects of DNA Methylation Inhibitors on Molecular and Structural Changes in Chromatin Organization in Leukemia Cells.
              </a>
            </td>
          <td>
            Przemysław Sareło, A. Kaczorowska, Marlena Gąsior-Głogowska, Kinga Godkowicz, Weronika Lamperska, Sławomir Drobczyński, Ewa Zioło, Anna A Licznerska, Tomasz Wróbel, Halina Podbielska, Wojciech Kałas, M. Kopaczyńska
          </td>
          <td>2025-09-29</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Neoadjuvant chemotherapy (NAC) is a critical component of breast cancer treatment, but the molecular mechanisms underlying resistance remain poorly understood. This study aimed to identify transcriptomic changes associated with NAC resistance across four breast cancer subtypes: Luminal A, Luminal B/HER2-positive, Luminal B/HER2-negative, and Triple-Negative Breast Cancer (TNBC). RNA-seq analysis was performed on paired pre- and post-NAC breast cancer samples from 32 non-responders. Differentially expressed genes (DEGs) were identified, and functional enrichment analyses were conducted. Protein-protein interaction (PPI) networks were constructed to identify hub genes. Tumor microenvironment (TME) infiltration was estimated using deconvolution algorithms. The results revealed distinct gene expression profiles between pre- and post-NAC samples, with FOS and NR4A1 being common DEGs across all subtypes. Enriched pathways varied among subtypes, including signal transduction, estrogen biosynthesis, extracellular matrix organization, dendritic cell activation, and B cell activation. TME analysis showed increased infiltration of specific immune cell populations after NAC, including CD4 memory T cells, regulatory T cells, neutrophils, macrophages, and mast cells, varying by subtype. These findings suggest that NAC modulates gene expression, cellular activity, and TME interactions, potentially contributing to treatment resistance. Understanding the molecular determinants of NAC resistance is crucial for developing targeted therapeutic strategies and improving outcomes for breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e52a24e6d865b78f25fc1cec6097a262050bb3b1" target='_blank'>
              Molecular determinants of neoadjuvant chemotherapy resistance in breast cancer: An analysis of gene expression and tumor microenvironment
              </a>
            </td>
          <td>
            H. M. Guevara-Nieto, Carlos Orozco-Castaño, R. Parra-Medina, Jenny Nathaly Poveda-Garavito, J. Garai, Jovanny Zabaleta, Liliana Lopez-Kleine, A. Cómbita
          </td>
          <td>2025-10-14</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Replication stress (RS) poses a threat to genome stability and drives genomic rearrangements. The homologous recombination (HR) pathway repairs stalled replication forks (RFs) and prevents such instability. Through an E3 ubiquitin ligase screen aimed at identifying regulators of RAD51, we identified macrophage erythroblast attacher (MAEA), a core component of C-terminal to Lish (CTLH) E3 ubiquitin ligase complex, as a regulator of the HR pathway. Loss of MAEA impairs RAD51 recruitment at stalled RFs, leading to increased sensitivity to RS-inducing agents and excessive degradation of nascent DNA strands. Mechanistically, MAEA associates with and mediates the ubiquitylation of Ku80, enabling its removal from RF ends and facilitating the loading of RAD51. Notably, MAEA deficiency is associated with a developmental disorder involving microcephaly, craniofacial abnormalities, ocular defects, and heart malformations. Functional assays show that disease-linked MAEA variants (R34C, E349G, Y394D, and M396R) are defective in RS response. These findings establish MAEA as an essential factor in RF protection and genome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c499707b53080fc3870cbe7d66d1176114a45109" target='_blank'>
              Pathogenic variants in MAEA disrupt DNA replication fork stability and are associated with developmental abnormalities in humans
              </a>
            </td>
          <td>
            Elham Zeinali, Fatemeh Mashayekhi, Rabih Abou Farraj, Talah Hasanni, Marie-christine Caron, Yan Coulombe, A. Fitieh, J. N. M. Glover, Jean-Yves Masson, Ismail Hassan Ismail
          </td>
          <td>2025-08-29</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4c68bd6ad67db140a21e19e70ce02281cc63746" target='_blank'>
              Transcriptional Rewiring of BET Inhibitor Treated Ewing Sarcoma Cells Augments their Dependency on Focal Adhesion Kinase.
              </a>
            </td>
          <td>
            Shireen S Ganapathi, Emma D. Wrenn, Nicolas M Garcia, Neerja Katiyar, Stephanie I. Walter, Aya Miyaki, Jonah R. Valenti, Yuqi Kang, Marina Chan, Taranjit S. Gujral, Elizabeth R. Lawlor
          </td>
          <td>2025-09-06</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="



Oncohistones represent a distinct class of cancer-driving mutations affecting core histone proteins, particularly histone H3 variants. Unlike conventional oncogenes and tumor suppressors, these single amino acid substitutions exert their oncogenic effects through widespread disruption of chromatin architecture and epigenetic regulation. This narrative review examines the molecular mechanisms underlying oncohistone function, with particular focus on H3K27M mutations found in pediatric diffuse midline gliomas and H3K36M mutations characteristic of chondroblastomas. We explore how these mutations interfere with normal chromatin-modifying enzymes, including polycomb repressive complex 2 (PRC2) and SETD2 methyltransferase, leading to aberrant gene expression patterns that promote tumorigenesis. The review discusses the clinical significance of oncohistone mutations in cancer diagnosis and classification, highlighting their integration into current diagnostic frameworks. Additionally, we examine emerging therapeutic approaches targeting oncohistone-driven cancers, including epigenetic modulators, immunotherapeutic strategies, and novel synthetic lethality approaches. Finally, we address current challenges in the field, including therapeutic resistance mechanisms and the complexity of targeting global epigenetic alterations, while identifying key areas for future research to advance our understanding and treatment of oncohistone-driven malignancies. 



">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc4872fffa3cd46351e5594106479bc2c6f61544" target='_blank'>
              Oncohistones in Cancer: A Narrative Review of Molecular Mechanisms and Targeted Therapeutic Strategies
              </a>
            </td>
          <td>
            A. R. Al-Zaalan, Hussam Saadi Aziz
          </td>
          <td>2025-10-03</td>
          <td>European Journal of Medical and Health Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The chromatin is folded into three-dimensional (3D) structures, and aberrant 3D chromatin folding has been implicated in cancer. We performed ATAC-seq and TOP2A ChIP-seq to assess the potential effects of various anthracycline drugs on the chromatin architecture. We found that specific anthracycline variants selectively disrupt chromatin looping anchors by interfering with CTCF binding, suggesting an additional therapeutic mechanism of anthracycline drugs targeting the 3D genome. Hi-C experiments in K562 cells treated with anthracycline drugs revealed widespread disruption of 3D chromatin organization, including altered long-range regulation at the Myc locus. Furthermore, AML patients treated with anthracycline drugs exhibited changes in chromatin structures near possible looping anchors, which were associated with distinct clinical outcomes. Together, our findings indicate that anthracycline drugs function as potent and selective epigenomic modulators, with the capacity to further target the 3D genome to exert anticancer effects, highlighting their potential for personalized therapy in tumors with aberrant 3D chromatin architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3d9cc4974c393dd5c8e640f80106dc4e3c5a03a" target='_blank'>
              Targeting the 3D genome by anthracyclines for chemotherapeutic effects.
              </a>
            </td>
          <td>
            Minkang Tan, Shengnan Sun, Yuchen Liu, Andrea A. Perreault, D. Phanstiel, Liping Dou, Baoxu Pang
          </td>
          <td>2025-10-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="R-loop is a common chromatin feature consisting of a displaced single-stranded DNA and an RNA-DNA hybrid, and dysregulation of R-loop surveillance results in genomic and transcriptomic instability. Although the RNA moiety of most R-loops originates from linear transcripts, circular RNAs (circRNAs), outputs from back-splicing, can also hybridize with the complementary strand of a DNA duplex. However, how circRNA-associated R-loops (ciR-loops) are monitored remains elusive. Here, we identify the DEAD-box RNA helicase Brr2 as an evolutionarily-conserved ciR-loop repressor with dual roles in inhibiting circRNA generation and resolving harmful ciR-loops. Accumulation of ciR-loops caused by loss-of-function of this dual-action factor induces antisense transcription and premature transcription termination for many genes and generates significant DNA damage, which further leads to a series of defects in DNA replication, cell division and cell proliferation. We propose that functional integration of multilayered regulation by a single protein can be an efficient double protection against genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75e11f7017a151564f0e8bfd51f16ad2197c7b10" target='_blank'>
              Dual roles of DEAD-box RNA helicase Brr2 in genome stability regulation
              </a>
            </td>
          <td>
            Xiaolan Chen, Jin You, Qin Ma, Jiamei Lin, Yu Ji, Shi Li, Zhenxing Song, Rui Su, Ge Shan, Chuan Huang
          </td>
          <td>2025-10-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Single base substitution (SBS) mutations, particularly C > T and T > C, are increased owing to unrepaired DNA replication errors in mismatch repair-deficient (MMRd) cancers. Excess CpG > TpG mutations have been reported in MMRd cancers defective in mismatch detection (dMutSα), but not in mismatch correction (dMutLα). Somatic CpG > TpG mutations conventionally result from unrepaired spontaneous deamination of 5’-methylcytosine throughout the cell cycle, causing T:G mismatches and signature SBS1. It has been proposed that MutSα detects those mismatches, prior to error correction by base excision repair (BER). However, other evidence appears inconsistent with that hypothesis: for example, MutSα is specifically expressed in S/G2 phases of the cell cycle, and defects in replicative DNA polymerase proofreading specifically cause excess CpG > TpG mutations in signature SBS10b. Methods We analysed mutation spectra and COSMIC mutation signatures in whole-genome sequencing data from 1803 colorectal cancers (164 dMutLα, 20 dMutSα) and 596 endometrial cancers (103 dMutLα, 9 dMutSα) from the UK 100,000 Genomes Project. We mapped each C > T mutation to its genomic features, including normal DNA methylation state, replication timing, transcription strand, and replication strand, to investigate the mechanism(s) by which these mutations arise. Results We confirmed that dMutSα tumours specifically had higher CpG > TpG burdens than dMutLα tumours. We could fully reconstitute the observed dMutSα CpG > TpG mutation spectrum by adding CpG > TpG mutations in proportion to their SBS1 activity to the dMutLα spectrum. However, other evidence indicated that the SBS1 excess in dMutSα cancers did not come from 5’-methylcytosine deamination alone: non-CpG C > T mutations were also increased in dMutSα cancers; and, in contrast to tumours deficient in BER, CpG > TpG mutations were biased to the leading DNA replication strand, at similar levels in dMutSα and dMutLα cancers, suggesting an origin in DNA replication. Other substitution mutations usually corrected by BER were not increased in dMutSα tumours. Conclusions There is a CpG > TpG and SBS1 excess specific to dMutSα MMRd tumours, consistent with previous reports, and we find a general increase in somatic C > T mutations. Contrary to some other studies, the similar leading replication strand bias in both dMutSα and dMutLα tumours indicates that at least some of the excess CpG > TpG mutations arise via DNA replication errors, and not primarily via the replication-independent deamination of 5’-methylcytosine. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01525-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2f0be2946a1331e8205341cd8d8b7fbe7da7996" target='_blank'>
              Replication-associated mechanisms contribute to an increased CpG > TpG mutation burden in mismatch repair-deficient cancers
              </a>
            </td>
          <td>
            Joseph C Ward, Ignacio Soriano, Steve Thorn, Juan Fernández-Tajes, Kitty Sherwood, Güler Gül, Joost Scheffers, A. Frangou, Ben Kinnersley, Ioannis Kafetzopoulos, Duncan Sproul, Sara Galavotti, C. Palles, Andreas J. Gruber, David N. Church, Ian Tomlinson
          </td>
          <td>2025-08-25</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Alterations in the mitochondrial genome (mtDNA) have been shown to be key in cancer development and could be useful as biomarkers for diagnosis, prognosis, and treatment. To identify mtDNA variants associated with breast cancer, we analyzed the whole mtDNA sequence from paired tissues (tumor–peripheral blood) of women with this malignancy and from peripheral blood samples of healthy women. The mtDNA mutational landscape, heteroplasmy levels of the variants, and mitochondrial ancestry were established. Comparative analysis between cases and controls revealed significant differences in the number and location of variants, as well as in the heteroplasmy levels. Cases showed higher mutation number in MT-ND5, tRNAs, and rRNAs genes; increased proportion of missense variants; and elevated mtDNA content, than controls. Notably, a high blood mtDNA mutational burden (OR = 3.83, CI: 1.89–7.95, p = 5.3 × 10−5) and five mtDNA variants showed association with the risk of breast cancer. Furthermore, a low tumor mutational burden (HR = 7.82, CI: 1.0–63.6, p = 0.05) and the haplogroup L (HR = 12.16, CI: 2.0–72.8, p = 0.0062) were associated with decreased overall and disease-free survival, respectively. Our study adds evidence of the potential usefulness of mtDNA variants as risk and prognosis biomarkers for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b89daf629cb05af3adbfba97c1f371049ff76ce" target='_blank'>
              Mitogenomic Alterations in Breast Cancer: Identification of Potential Biomarkers of Risk and Prognosis
              </a>
            </td>
          <td>
            C. Pérez-Amado, Amellalli Bazan-Cordoba, L. Gómez-Romero, Julian Ramírez-Bello, V. Bautista-Piña, A. Tenorio-Torres, E. Ruvalcaba-Limón, Felipe Villegas-Carlos, Diana Karen Mendiola-Soto, A. Hidalgo-Miranda, Silvia Jiménez-Morales
          </td>
          <td>2025-08-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDA) is a highly aggressive disease with a 5-year survival rate of 12%. Patients are typically diagnosed at the metastatic stage due to lack of symptoms and current treatment options do not effectively control the disease. There is an urgent need to develop sensitive strategies to detect PDA at an early stage before it spreads. Characterizing circulating tumor DNA (ctDNA) in plasma is an effective approach to detect and monitor cancer. Typically, cancer genomes contain thousands of mutations, but targeted sequencing approaches only focus on pre-determined driver genes, which limits their performance in detecting ctDNA. Whole-genome sequencing (WGS) however, tracks all the tumor mutations simultaneously, and has a higher sensitivity than targeted approaches, especially in low tumor burden samples. In this study, we aim to use plasma WGS to build a liquid biopsy platform for PDA detection and characterization. To date, we have established the largest plasma WGS (20-60x) PDAC cohort which consists of over 250 baseline and longitudinal plasma samples from 205 PDA patients, with an additional 45 plasma samples from age-matched healthy individuals. Paired tumor tissue WGS and RNAseq are also available. We developed and validated a bioinformatics pipeline to detect cancer mutations in plasma and estimate plasma tumor fraction (TFx) for tumor burden. In the PDA cohort, plasma TFx increased with clinical stages (medians of 0.5% in resectable, 0.8% in locally advanced and 5% in metastatic), as did the ctDNA detection rate which was measured by whether the plasma TFx was significantly higher than controls (87%, 91% and 95%). Plasma TFx positively correlated with liver metastasis size (R=0.68, P<0.001), but not with the primary pancreatic tumor size. These results suggest that in PDA, metastases are the major site to shed ctDNA, but not primary tumors. We also found that liver metastasis size had a stronger correlation with plasma TFx than with serum CA19-9 (R=0.32, P=0.002). Baseline plasma TFx was associated with shorter overall survival (OS) in both resectable (P=0.013) and metastatic (P=0.022) cases. By analyzing the paired tumor WGS and RNAseq, we found that higher plasma TFx was associated with higher cell cycle, lower immunity and polyploid genomes in the tumors. No difference was found in plasma TFx between basal-like and classical PDA subtypes. In summary, the current study provides insights into the interplay between ctDNA dynamics, tumor biology and clinical features in PDA. Alongside with these results, we will provide a freely accessible computational platform for ctDNA genomic profiling, PDA early detection and progression monitoring using plasma.



 Yuanchang Fang, Michelle Chan-Seng-Yue, Karen Ng, Amy Zhang, Tuan Hoang, Gun Ho Jang, Sabiq Chaudhary, Eugenia Flores-Figueroa, Daniela Bevacqua, Stephanie Ramotar, Ayelet Borgida, Shawn Hutchinson, Anna Dodd, Julie Wilson, Barbara Grünwald, Robert Grant, Erica Tsang, George Zogopoulos, Jennifer Knox, Masoom Haider, Steven Gallinger, Faiyaz Notta. Plasma whole-genome sequencing to monitor pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B072.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41904b2fb7e8d9c5cced0dd565b2ced62da7f071" target='_blank'>
              Abstract B072: Plasma whole-genome sequencing to monitor pancreatic cancer
              </a>
            </td>
          <td>
            Yuanchang Fang, M. Chan-Seng-Yue, Karen Ng, Amy Zhang, Tuan Hoang, G. Jang, Sabiq Chaudhary, Eugenia Flores-Figueroa, Daniela Bevacqua, S. Ramotar, A. Borgida, S. Hutchinson, A. Dodd, Julie M. Wilson, Barbara Grünwald, Robert C. Grant, Erica S. Tsang, G. Zogopoulos, Jennifer J Knox, Masoom A. Haider, Stephen Gallinger, F. Notta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Copy number control of DNA and centrosomes is essential for accurate genetic inheritance. DNA replication and centrosome duplication have been recognized as parallel key events for cell division. Here, we discover that the DNA replication machinery directly regulates the licensing and execution processes of centrosome duplication to prevent centrosome amplification. We find that the microcephaly protein DONSON couples DNA replication initiation with Cdc6 translocation to centrosomes. The Cdc6 signal prevents the precocious occurrence of centriole disengagement, the licensing step for centrosome duplication. During DNA replication, DONSON inhibits replisome disassembly by interacting with the CMG helicase, maintaining the intrinsic S/G2 checkpoint signal that blocks centriole-to-centrosome conversion, the execution step for centrosome duplication. Disruption of these dual signals causes precocious centrosome duplication and chromosome mis-segregation, observed in DONSON patient cells. Our results reveal that the DNA replication machinery not only duplicates genetic material but also controls the system for its accurate segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895f8bba3e9e4612c1b72098d2d19a10b12b8583" target='_blank'>
              The DNA replication machinery transmits dual signals to prevent unscheduled licensing and execution of centrosome duplication
              </a>
            </td>
          <td>
            Kyohei Matsuhashi, Kei K Ito, Kaho Nagai, Akira Sanada, Koki Watanabe, Kasuga Takumi, Atsushi Toyoda, Masamitsu Fukuyama, Shohei Yamamoto, Takumi Chinen, Grant S Stewart, Shoji Hata, Daiju Kitagawa
          </td>
          <td>2025-09-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Long interspersed nuclear element-1 (LINE-1 or L1), the only autonomously active retrotransposon in humans today, constitutes a large proportion of the genome and continues to evolve the genome and impact fundamental biological processes. L1 retrotransposition critically depends on its endonuclease and reverse transcriptase subunit open reading frame 2 protein (ORF2p), which targets genomic loci and nicks DNA using an evolutionarily distinct yet not fully understood mechanism. Our structural and biochemical analyses revealed that ORF2p is a structure-dependent endonuclease. It binds a double-stranded DNA region upstream of the nicking site and recognizes a downstream forked or flap structure for efficient DNA nicking. This discovery suggests that L1 mobilization piggybacks on chromosomal processes with noncanonical DNA structure intermediates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f0ddeb532f6d95f55ced91946685205519cc0c7" target='_blank'>
              Mechanism of DNA targeting by human LINE-1.
              </a>
            </td>
          <td>
            Wenxing Jin, Cong Yu, Yan Zhang, Changchang Cao, Tianfan Xia, Ge Song, Zhaokui Cai, Yuanchao Xue, Bing Zhu, Rui-Ming Xu
          </td>
          <td>2025-10-09</td>
          <td>Science</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a59a21b3122614696fcbf37a9973f0709eb05a0" target='_blank'>
              Unravelling the genetics and epigenetics of the ageing tumour microenvironment in cancer.
              </a>
            </td>
          <td>
            H. Easwaran, A. Weeraratna
          </td>
          <td>2025-09-08</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The chromatin folding and the spatial arrangement of chromosomes in the cell play a crucial role in DNA replication and genes expression. An improper chromatin folding could lead to malfunctions and, over time, diseases. For eukaryotes, centromeres are essential for proper chromosome segregation and folding. Despite extensive research using de novo sequencing of genomes and annotation analysis, centromere locations in yeasts remain difficult to infer and are still unknown in most species. Recently, genome-wide chromosome conformation capture coupled with next-generation sequencing (Hi-C) has become one of the leading methods to investigate chromosome structures. Some recent studies have used Hi-C data to give a point estimate of each centromere, but those approaches highly rely on a good pre-localization. Here, we present a novel approach that infers in a stochastic manner the locations of all centromeres in budding yeast based on both the experimental Hi-C map and simulated contact maps.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08274ce877ee4b3550c4365a542bed53e84e596e" target='_blank'>
              Simulation-based inference of yeast centromeres
              </a>
            </td>
          <td>
            Eloise Touron, Pedro L. C. Rodrigues, Julyan Arbel, Nelle Varoquaux, Michael Arbel
          </td>
          <td>2025-08-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15f8c45e1c8b7fe16d612e988c34218a0a5811b8" target='_blank'>
              Chemotherapy-induced CA-repeat DNA fragments in breast cancer trigger antitumor immune responses.
              </a>
            </td>
          <td>
            Xiaoqian Zhang, Penghan Huang, Huiping Chen, Chaoqun Yang, Xinyu Yang, Yujie Liu, Baixing Wu, Wenfeng Zeng, P. Saw, Shuying Ye, Jiaqian Li, Jiayi Wang, Zizhen Wu, Jiawen Wang, Heliang Li, Jiandong Huang, Musheng Zeng, Mengfeng Li, Qiang Liu, Shicheng Su, Erwei Song, Jianing Chen
          </td>
          <td>2025-09-29</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Mutations reflect the net effects of myriad types of damage, replication errors, and repair mechanisms, and thus are expected to differ across cell types with distinct exposures to mutagens, division rates, and cellular programs. Yet when mutations in humans are decomposed into a set of “signatures”, one single base substitution signature, SBS5, is present across cell types and tissues, and predominates in post-mitotic neurons as well as male and female germlines [1–3]. The etiology of SBS5 is unknown. By modeling the processes by which mutations arise, we infer that SBS5 is the footprint of errors in DNA synthesis triggered by distinct types of DNA damage. Supporting this hypothesis, we find that SBS5 rates increase with signatures of endogenous and exogenous DNA damage in cancerous and non-cancerous cells and co-vary with repair rates along the genome as expected from model predictions. These analyses indicate that SBS5 captures the output of a “funnel”, through which multiple sources of damage result in a similar mutation spectrum. As we further show, SBS5 mutations arise not only from translesion synthesis but also from DNA repair, suggesting that the signature reflects the occasional, shared use of a polymerase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0289ae32bcbca4304d80ec26b288975aaa0f5e9" target='_blank'>
              Collateral mutagenesis funnels multiple sources of DNA damage into a ubiquitous mutational signature
              </a>
            </td>
          <td>
            Natanael Spisak, Marc de Manuel, Molly Przeworski
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="DNA topoisomerase IIα (TOP2A) plays a key role in resolving DNA topological stress during replication, transcription, and chromosome segregation. Yatein, a biogenic precursor of deoxypodophyllotoxin possesses antitumor effects. Herein, we explored its antitumor effect in breast cancer. Yatein significantly inhibited the proliferation of breast cancer cells by MTT, colony formation, EdU assay, and cell cycle analysis. Moreover, yatein suppressed the stemness of cancer cells, causing inhibition of migration and epithelial-mesenchymal transition. Mechanism study revealed that yatein inhibited TOP2A activity and stabilized TOP2A-DNA cleavage complexes, caused DNA damage and apoptosis, suggesting its potential in the treatment of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2d219f76fc877a179547f374e85d00ebe2c5225" target='_blank'>
              Yatein-induced DNA damage of breast cancer cells by interfering with DNA topoisomerase IIα.
              </a>
            </td>
          <td>
            Yu-Yao Liu, Ya Chen, Qi-Qi Wang, Xi Zhang, Xiao-Xia Ma, Xing-De Wu, Jia Su
          </td>
          <td>2025-08-25</td>
          <td>Journal of Asian natural products research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Post-transcriptional RNA modifications, such as N6-methyladenosine (m6A) methylation and adenosine to inosine (A-to-I) editing, are critical regulators of hematopoietic stem cell (HSC) self-renewal and differentiation, yet their precise contributions to malignant transformation are not fully elucidated. In this study, we uncovered the epitranscriptomic landscape caused by knockdown of genes from the methyltransferase (METTL)-family in hematopoietic stem and progenitor cells (HSPCs). We identified both converging and distinct roles of METTL3 and METTL14, known members of the m6A writer complex, as well as orphan gene METTL13. Notably, METTL13 was uniquely upregulated by adenosine deaminase acting on RNA 1 (ADAR1) overexpression, while other METTL genes were downregulated. Knockdown of METTL13 altered the expression of multiple genes involved in oncogenic development in HSPCs. Furthermore, METTL13 was associated with a high-risk profile in pediatric T-cell acute lymphoblastic leukemia (T-ALL), and functional studies confirmed that METTL13 is required for T-ALL cell proliferation and survival both in vitro and in vivo. Collectively, our results indicate a previously unrecognized, oncogenic role for METTL13 in pre-leukemic transformation and T-ALL pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047267bf0f7cb87eac1cee31332201b93b817d49" target='_blank'>
              METTL13 Promotes Pre-Leukemic Transformation and the Development of Pediatric Leukemia
              </a>
            </td>
          <td>
            Sabina Enlund, Chae-eun Lim, Isabella Hoang, Sonali Joshi, Amanda Ramilo Amor, Cecilia Thomsson, Indranil Sinha, Shahrzad Shirazi Fard, Anna Nilsson, Ola Hermanson, Qingfei Jiang, Frida Holm
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Fusions between protein-coding genes are common oncogenic drivers across cancers, typically pairing a proto-oncogene with a partner that does not independently drive cancer. In all therapeutically actionable fusions, the proto-oncogene is the drug target; the contributions to oncogenicity of the fusion partner have largely been ignored. KIAA1549::BRAF (K::B) is the most common fusion protein across brain tumors, and particularly in pediatric gliomas. We found that KIAA1549 is necessary for oncogenicity of K::B, in contrast to the notion that the BRAF kinase domain alone is enough for transformation. We therefore performed genome-wide CRISPR/Cas9 anti-transformation screens in isogenic neural stem cell models with K::B overexpression to identify genetic dependencies specific to this oncogenic driver. We found that the BRAF fusion partner KIAA1549 engenders a striking and specific dependency on the protein O-mannosyltransferase complex (POMT1/2). Specifically, we show that genetic silencing or pharmacologic inhibition of the POMT1/2 complex reverses K::B transformation, thereby representing a novel therapeutic target beyond the MAPK pathway. Furthermore, POMT1/2 is required to glycosylate and enable maturation of K::B, reflecting the native biology of KIAA1549 but representing a non-canonical process for BRAF. These findings represent a proof-of-concept for targeting the partners in oncogenic fusions as a potential cancer therapeutic strategy.



 Anna Borgenvik, Sean Misek, Daniel Christen, Gloria Kyrila, Alexander Zhang, Sarah Reel, Michelle Boisvert, Kelly Cai, Kevin Zhou, Elizabeth M Gonzales, Lorena Lazo de la Vega, Amy Goodale, Esteban Miglietta, Jacquelyn Jones, Seth Malinowski, Lobna Elsadek, Merve Ozdemir, Zachary Eisenbies, Joohee Lee, April Apfelbaum, Jenna Robinson, Antonio Maldera, Daniel Bondeson, Jason Kwon, Mounica Vallurupalli, Sangita Pal, Todd Golub, William Hahn, Eric Fischer, Jesse Boehm, Jörn Dengjel, Henrik Clausen, Nada Jabado, Till Milde, Beth A Cimini, Keith Ligon, Kathrine Janeway, Michael Eck, David Root, David Jones, Timothy N Phoenix, Rameen Beroukhim, Hiren Jitendra Joshi, Tilman Brummer, Adnan Halim, Pratiti Bandopadhayay. O-mannosylation represents a therapeutic opportunity in BRAF fusion protein oncogenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e2365acc2ab5bd8824942a859ad9e18262c35e6" target='_blank'>
              Abstract B019: O-mannosylation represents a therapeutic opportunity in BRAF fusion protein oncogenesis
              </a>
            </td>
          <td>
            Anna Borgenvik, Sean A. Misek, Daniel Christen, Gloria Kyrila, Alexander Zhang, Sarah Reel, Michelle Boisvert, Kelly Y Cai, Kevin N Zhou, Elizabeth M Gonzales, Lorena Lazo de la Vega, A. Goodale, Esteban Miglietta, Jacquelyn S Jones, Seth Malinowski, Lobna Elsadek, Merve Ozdemir, Zachary Eisenbies, Joohee Lee, April A Apfelbaum, Jenna Robinson, Antonio Maldera, Daniel Bondeson, Jason Kwon, Mounica Vallurupalli, Sangita Pal, Todd Golub, William Hahn, Eric S Fischer, Jesse Boehm, J. Dengjel, Henrik Clausen, Nada Jabado, T. Milde, Beth Cimini, Keith L. Ligon, Kathrine Janeway, Michael J Eck, David E. Root, David Jones, T. Phoenix, R. Beroukhim, H. Joshi, Tilman Brummer, Adnan Halim, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>151</td>
        </tr>

        <tr id="RAD50 is the central protein of the MRN complex and crucial in DNA double-strand break repair. RAD50 deficiency causes a genomic instability disorder characterized by microcephaly and stunted growth. Using lentiviral constructs, we investigated whether cancer-related RAD50 missense variants can complement the delayed damage response after exposure to the chemotherapeutic agent epirubicin and/or mitotic progression in RAD50-deficient fibroblasts. Eight missense variants, all capable of forming an MRN complex, supported the DNA damage response and mitotic features to different extents, indicating these functions are separable. Three variants showed both an impaired epirubicin response and slowed cell division in the likely pathogenic range. Assessing RAD50 missense variants with distinct functional readouts may help to further elucidate their differential roles in immunodeficiency and cancer and could improve therapeutic strategies. Impact statement RAD50 has a strong impact on DNA repair and cancer therapy. Here, we analyse RAD50 missense variants at four functional levels. Some variants showed an impaired epirubicin response and mitotic progression in the pathological range, while for others these endpoints were separable. Functional heterogeneity of RAD50 variants could contribute to clinical variability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dc8ea9e9e82f5d91c26df1ede413c6f7f658286" target='_blank'>
              RAD50 missense variants differentially affect the DNA damage response and mitotic progression.
              </a>
            </td>
          <td>
            Hanna Redeker, Swantje Kebel, Lea Völkening, Anna Vatselia, Louisa Weinhold, G. Asgedom, Axel Schambach, Detlev Schindler, T. Dörk, Kristine Bousset
          </td>
          <td>2025-10-01</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is the age-related expansion of mutated hematopoietic stem cells without other hematologic abnormalities. In patients with solid tumors, CH is associated with higher mortality and may evolve to therapy-related myeloid neoplasms; however, the mechanisms by which cancer treatments promote CH dynamics remain largely unknown. Here, we analyzed 392 serial samples from a prospective cohort of breast cancer patients and showed that cytotoxic treatments (chemotherapy {+/-} radiation) led to strong therapeutic bottlenecks, resulting in significant reductions in hematopoietic allelic populations and differential clonal selection. CH clones that were positively selected and expanded through dose-dependent therapeutic bottlenecks frequently harbored mutations in TP53, PPM1D, SRCAP, DNMT3A, and YLPM1. Patients with positively selected CH during treatment had the shortest progression-free and overall survival compared to patients with unchanging or negatively selected CH across all therapies. These findings, corroborated in independent breast cancer and pan-cancer cohorts, provide strong evidence for clinical relevance of changes in CH during cancer treatment which may help identify patients at high risk for inferior outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6a553a5184b2aa0287d71bedac04551bf6a5da6" target='_blank'>
              Clonal Hematopoiesis Dynamics and Evolutionary Fitness During Cancer Treatment Impact Clinical Outcomes
              </a>
            </td>
          <td>
            Mona Arabzadeh, Yi-Han Tang, Christelle M. Colin-Leitzinger, S. Marzban, Daniel Walgenbach, Stefania Morganti, Vaidhyanathan Mahaganapathy, Erika Harper, Mingxiang Teng, Jacob Kresovich, Iman Washington, H. Parsons, Judy E. Garber, J. West, Shridar Ganesan, Hossein Khiabanian, Nancy Gillis
          </td>
          <td>2025-09-02</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cellular organisms store heritable information in two forms, genetic and epigenetic, the latter being largely dependent on cytosine methylation (5mC). Chargaff’s Second Parity Rule (CSPR) describes the nucleotide composition of cellular genomes in terms of intra-strand DNA symmetry. However, it remains unknown whether DNA methylation patterns display intra-strand DNA symmetry. Computational analysis was conducted of the DNA methylation patterns observed in human cell lines and in tissue samples from healthy donors. Analysis of 5mC marks in mutually reverse-complementary pairs of short oligomers, containing CpG dinucleotide in the middle, revealed deviations from CSPR and methylation asymmetry that can be observed for two non-overlapping mirror groups defined by CpG methylation values. Deviations from CSPR, together with combinatorial probabilities of pattern distributions and computer simulations, highlight the non-random nature of methylation processes and enabled us to identify specific cell types as outliers. Further analysis revealed a compensatory methylation asymmetry that reduces deviations from intra-strand symmetry and implies the existence of strand-specific methylation during cell differentiation. Among six pairs of reverse-complementary tetranucleotides, four pairs with specific sequence motifs display pronounced methylation asymmetry. This mirror asymmetry may be associated with chromosome folding and the formation of a complex three-dimensional landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/387c52e696253beba52f792a823e42cb5be6a863" target='_blank'>
              Study of the Patterns of DNA Methylation in Human Cells Through the Prism of Intra-Strand DNA Symmetry
              </a>
            </td>
          <td>
            Z. Ramazanova, Aizhan Alikul, D.A. Begimbetova, S. Taipakova, Bakhyt Matkarimov, Murat K. Saparbaev
          </td>
          <td>2025-09-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Double Homeobox 4 (DUX4) is a potent transcription factor encoded by a retrogene mapped in D4Z4 repeated elements on chromosome 4q35. DUX4 has emerged as pivotal in the pathomechanisms of facioscapulohumeral muscular dystrophy (FSHD), a relatively common hereditary muscle wasting condition, although classified as a rare disease. DUX4 contributes to zygote genome activation before its expression is repressed in most somatic tissues through epigenetic mechanisms, including DNA methylation and chromatin modifications. In FSHD, inappropriate activation of DUX4 expression is driven by a complex interplay of genomic and epigenetic alterations. The ectopic presence of DUX4 in skeletal muscle cells activates genes, viral elements and pathways that are typical of very early embryonic development, disturbing cell function and ultimately contributing to muscle weakness and wasting. This review first traces the history of DUX4, from the FSHD genetic linkage studies in the early 1990s, through to identification and characterization of the DUX4 gene in 1999. We then discuss the seminal studies that showed how and why DUX4 is expressed in FSHD and the effects of this ectopic expression in muscle, notably cellular toxicity. Other pathological roles of DUX4, such as participation in cancer and viral infection, are also highlighted. Maintenance of DUX4 in the genome was explained by discovery of the function of DUX4 in zygotic genome activation to institute the totipotent cells of the embryo. Thus, we encompass the gradual transition of DUX4 over the past 25 years from being considered a pseudogene in “junk DNA” to becoming central to understanding the molecular pathogenesis of FSHD and the primary focus for FSHD therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad639b9256d2a222879537cb6ffce2084c7d6dd4" target='_blank'>
              DUX4 at 25: how it emerged from “junk DNA” to become the cause of facioscapulohumeral muscular dystrophy
              </a>
            </td>
          <td>
            Alexandra Belayew, Alberto L. Rosa, Peter S. Zammit
          </td>
          <td>2025-08-25</td>
          <td>Skeletal Muscle</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract The maintenance of genomic integrity is paramount for normal cell physiology and survival as well as avoidance of carcinogenesis. Cellular DNA is periodically subjected to a myriad of exogenous and endogenous threats and requires constant monitoring to limit genomic instability. To this end, cells possess an intricate DNA damage response and repair (DDR) module comprised of different classes of protein players. The DNA helicases, ATP-dependent enzymes that unwind the DNA double helix, are one such important class of proteins, which act as a linchpin between the recognition and resolution of DNA damage via facilitating various DNA repair processes. Dysfunction or absence of DDR helicase function is implicated in several human disorders including Bloom syndrome, Werner syndrome, Rothmund–Thomson syndrome, and Fanconi anemia. Somatic helicase mutations or dysregulation of helicase function can also contribute to cancer development, progression, and chemotherapy sensitivity, making helicases a promising target class for chemotherapeutic drug intervention. In addition, recent discoveries have identified some DDR helicases in novel synthetic lethal interactions. In this critical review, we will focus on human DNA helicases that are directly or indirectly involved in DDR with special emphasis on their mechanistic actions and clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40deeb85611db34dede498af9b0ad1f2d2a9a398" target='_blank'>
              DNA repair helicases: from mechanistic understanding to therapeutic implications
              </a>
            </td>
          <td>
            Vijay Menon, Susan E. Gueble
          </td>
          <td>2025-10-07</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND
Therapies for diffuse glioma induce DNA damage response (DDR), and strategies to exploit DDR defects are active areas of investigation. While global DNA methylation profiling effectively classifies gliomas into subtypes, the epigenetic and gene expression patterns of DDR genes, and their contribution to tumor classification and outcomes, have yet to be fully elucidated. Thus, dissecting the DDR epigenetics, gene expression, and single-cell heterogeneity may reveal key molecular characteristics, refine prognosis, and identify novel treatment strategies and resistance mechanisms.


METHODS
We characterized DDR epigenetics and gene expression of TCGA glioblastomas (GBM) and low-grade gliomas (LGG). Single-cell protein analysis by imaging mass cytometry (IMC) was performed on a separate cohort of 118 diffuse gliomas.


RESULTS
Analysis of TCGA cohorts revealed two DDR methylation groups that correlated with IDH mutation status and previously reported molecular groups. DDR transcription profiling further classified tumors into four groups. Those with high DDR transcription across pathways were linked to poor survival independent of IDH or MGMT status, and potentially improved prognostication beyond established biomarkers. Single-cell characterization of a separate cohort revealed intratumoral DDR diversity and identified proliferative tumor cells with high DDR protein expression across pathways that are associated with unfavorable grade and survival.


CONCLUSIONS
Tumor-level epigenetic and transcriptional DDR signatures alone can distinguish molecular-defined diagnosis and outcomes of gliomas beyond established biomarkers. A higher abundance of glioma cells with high DDR effector expression across pathways is associated with poor survival. Thus, clinical assessment of pan-DDR expression may inform prognosis and identify potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0b0fbeaa6fa3878bda82552131f1d944fc87086" target='_blank'>
              DNA damage response profile distinguishes poor-acting gliomas with shared methylome signatures.
              </a>
            </td>
          <td>
            Nalin Leelatian, Charu Singh, Richard Bouffard, R. Sundaram, Kirsten E Diggins, William Sullivan, Sateja Paradkar, Zeynep E. Erson Omay, B. Mobley, Susan E. Gueble, Juan C Vasquez, R. Bindra
          </td>
          <td>2025-08-27</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Acetyl-CoA carboxylase alpha (ACACA), a crucial rate-limiting enzyme governing de novo biosynthesis of fatty acids, drives oncogenic metabolic reprogramming in diverse malignancies. However, the multiomics investigation and immunological implications of ACACA across cancers remain unclear.We performed a comprehensive pan-cancer analysis of ACACA via transcriptomic, proteomic, and clinical data from The Cancer Genome Atlas (TCGA), Clinical Proteomic Tumor Analysis Consortium (CPTAC), and the Human Protein Atlas (HPA) databases. Then, single-cell RNA sequencing acquired from the Gene Expression Omnibus (GEO) database was employed to map the expression pattern of ACACA in the tumor microenvironment (TME). Subsequently, functional validation experiments were conducted in lung cancer and sarcoma cells.High ACACA expression was associated with poor survival in various cancers, particularly those exhibiting dysregulated lipid metabolism. Immune profiling revealed that elevated ACACA expression was associated with low infiltration of CD8+ T cells and activated natural killer (NK) cells. Single-cell analysis of lung adenocarcinoma revealed that ACACA was expressed predominantly within malignant cells and contributed to an immunosuppressive microenvironment through migration inhibitory factor (MIF) signaling and the extracellular matrix (ECM) remodeling pathway. Furthermore, in vitro studies demonstrated that ACACA inhibition suppresses fatty acid synthesis and tumor growth in lung cancer and sarcoma cells.Our study establishes ACACA as a key metabolic regulator that links lipid metabolism to immune evasion and drug resistance, highlighting its potential as a promising therapeutic target across cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ef9b3c72fa2e6ad7dbbe3729933ef811be46587" target='_blank'>
              Integrated single-cell and bulk RNA sequencing analysis reveals ACACA as a potential prognostic and immunotherapeutic biomarker across cancers
              </a>
            </td>
          <td>
            Haihua He, Zhen Zhang, Leifeng Chen, Fushan Gao, Yuze Wu, Lina Yi, Fei Shao, Yibo Gao, Jie He
          </td>
          <td>2025-10-15</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d20ba148e7406fb91b956b71267ccb358bcdb11" target='_blank'>
              MNase stratification reveals heterogeneous 5hmC in naive B cells.
              </a>
            </td>
          <td>
            Michael Hsu, Heng-Yi Chen, Fang-Yun Lay, Qin Ma, Ben Delatte, C. J. Lio
          </td>
          <td>2025-08-28</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is defined by the expansion of a lineage of genetically identical cells in blood. Genetic lesions that confer a fitness advantage, such as leukemogenic point mutations or mosaic chromosomal alterations (mCAs), are frequent mediators of CH. However, recent analyses of both single cell-derived colonies of hematopoietic cells and population sequencing cohorts have revealed CH frequently occurs in the absence of known driver genetic lesions. To characterize CH without known driver genetic lesions, we use 51,399 deeply sequenced whole genomes from the NHLBI TOPMed sequencing initiative to perform simultaneous germline and somatic mutation analyses among individuals without leukemogenic point mutations (LPM), which we term CH-LPMneg. We quantify CH by estimating the total mutation burden. Because estimating somatic mutation burden without a paired-tissue sample is challenging, we develop a novel statistical method, the Genomic and Epigenomic informed Mutation (GEM) rate, that uses external genomic and epigenomic data sources to distinguish artifactual signals from true somatic mutations. We perform a genome-wide association study of GEM to discover the germline determinants of CH-LPMneg. We identify seven genes associated with CH-LPMneg (TCL1A, TERT, SMC4, NRIP1, PRDM16, MSRA, SCARB1).Functional analyses of SMC4 and NRIP1 implicated altered hematopoietic stem cell self-renewal and proliferation as the primary mediator of mutation burden in blood. We then perform comprehensive multi-tissue transcriptomic analyses, finding that the expression levels of 404 genes are associated with GEM. Finally, we perform phenotypic association meta-analyses across four cohorts, finding that GEM is associated with increased white blood cell count, but is not significantly associated with incident stroke or coronary disease events. Overall, we develop GEM for quantifying mutation burden from WGS and use GEM to discover the genetic, genomic, and phenotypic correlates of CH-LPMneg.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a90296a3758d552c4d460a056b5b1554e1d3253" target='_blank'>
              Genetic determinants and genomic consequences of non-leukemogenic somatic point mutations
              </a>
            </td>
          <td>
            J. Weinstock, Sharjeel A Chaudhry, Maria Ioannou, Maria Viskadourou, Paula Reventun, Yasminka A. Jakubek, L. Liggett, C. Laurie, J. Broome, Alyna T Khan, K. Taylor, Xiuqing Guo, P. Peyser, Eric Boerwinkle, Nathalie Chami, Eimear Kenny, Ruth J. F. Loos, B. Psaty, R. Tracy, J. Brody, Jeong H Yun, Michael H Cho, Ramachandran S. Vasan, Sharon L. R. Kardia, Jennifer A Smith, L. Raffield, A. Bidulescu, Emily C O'Brien, Mariza de Andrade, Jerome I. Rotter, S. Rich, Yii-DerIda Chen, C Charles Gu, C. Hsiung, C. Kooperberg, Bernhard Haring, Rami Nassir, Rasika A Mathias, Alexander P. Reiner, Vijay G. Sankaran, Charlie Lowenstein, T. Blackwell, G. Abecasis, Albert V. Smith, Hyun M Kang, P. Natarajan, S. Jaiswal, Alexander G. Bick, Wendy S. Post, P. Scheet, Paul L. Auer, Theodoros Karantanos, Alexis Battle, M. Arvanitis
          </td>
          <td>2025-10-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>187</td>
        </tr>

        <tr id="Abstract Genes encoding the RNA-binding proteins FUS, EWSR1, and TAF15 (FET proteins) are involved in chromosomal translocations in rare sarcomas. The encoded fusion oncoproteins typically include a DNA-binding domain and function as oncogenic transcription factors. FET-rearranged sarcomas are often aggressive malignancies affecting patients of all ages. Oncogenic fusion transcription factors are challenging to target directly, and new therapies are needed. Treatment with the small molecule SP-2509 results in reversal of the transcriptional activity of the FET fusion that causes Ewing sarcoma, EWSR1::FLI1. A similar molecule, seclidemstat (SP-2577), is currently in clinical trials for FET-rearranged sarcomas (NCT03600649), but its pharmacologic activity in FET fusion–positive sarcomas has not been demonstrated. We found potent seclidemstat cytotoxicity against both FET-rearranged and other fusion-positive sarcoma cell lines in vitro, including Ewing sarcoma, desmoplastic small round cell tumor, clear cell sarcoma, myxoid liposarcoma, and fusion-positive rhabdomyosarcoma. We also define transcriptomic effects of seclidemstat with bulk RNA sequencing. Seclidemstat recapitulated much of SP-2509 transcriptional activity in Ewing sarcoma. Widespread transcriptional changes were seen in all tested cell lines after seclidemstat treatment, regardless of the fusion protein expressed. This included reversal of FET fusion transcriptional signatures for EWSR1::WT1, EWSR1::ATF1, and EWSR1::ERG. Though novel inhibitors are unlikely to display single-agent efficacy in the clinic, these data suggest that seclidemstat remains a promising new treatment strategy for patients with FET-rearranged and other fusion-positive sarcomas. Significance: In this study, we show seclidemstat has in vitro activity in multiple rare and aggressive sarcomas caused by FET fusion proteins. With 13 RNA sequencing experiments, including multiple FET-rearranged sarcoma cell lines, this dataset is a rich resource for those studying FET-rearranged sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fb78384576a9a5b2487874950835b687b754a22" target='_blank'>
              Seclidemstat (SP-2577) Induces Transcriptomic Reprogramming and Cytotoxicity in Multiple Fusion–Positive Sarcomas
              </a>
            </td>
          <td>
            G. Rask, C. Taslim, A. Bayanjargal, Rachel D. Dreher, Matthew V. Cannon, Julia Selich-Anderson, Jesse C. Crow, Aundrietta Duncan, Emily R. Theisen
          </td>
          <td>2025-08-25</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Long noncoding RNAs (lncRNAs) are emerging as important regulators of gene expression in both normal biological systems and disease. Among them, metastasis-associated lung adenocarcinoma transcript 1 (MALAT1) has attracted considerable attention due to its high nuclear expression and evolutionary conservation. It was first identified as a biomarker for lung cancer metastasis, but since then, its involvement has been reported in a wide range of cancers. MALAT1 influences several key cancer-related processes, including cell proliferation, migration, angiogenesis, apoptosis, epithelial–mesenchymal transition, and the behavior of cancer stem cells. In this review, we take a closer look at how MALAT1 functions through different mechanisms, such as regulating RNA splicing, altering chromatin states, acting as a sponge for microRNAs, or modifying protein interactions, to drive these changes. We also discuss its growing role in shaping the immune landscape of tumors, particularly its involvement in immune evasion, inflammatory signaling, and regulation of immunogenic cell death. In addition, MALAT1 contributes to resistance against therapy, rewiring of cellular metabolism, and communication between tumor and stromal cells via exosomes. While many studies describe MALAT1 as an oncogene, others suggest it may also play a suppressive role depending on the cancer type and biological context. Here, we aim to offer a comprehensive overview of MALAT1’s diverse actions and evaluate its potential as a diagnostic marker and therapeutic target in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73cb56fb6ad82b5646d4f05c255a6ee44614bc12" target='_blank'>
              MALAT1 as a molecular driver of tumor progression, immune evasion, and resistance to therapy
              </a>
            </td>
          <td>
            A. Bitaraf, Alireza Zafarani, Pardis Jahandideh, Benyamin Hakak-Zargar, A. Haghi, Golareh Asgaritarghi, S. Babashah
          </td>
          <td>2025-10-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f43327ba689a502fd2aba63a16648713112594" target='_blank'>
              Epigenomic Heterogeneity of Non-Functional Pancreatic Neuroendocrine Tumors Uncovered by Single nucleus and Spatial ATAC Profiling.
              </a>
            </td>
          <td>
            Dejiang Wang, Xiangjun Di, Fu Gao, Gaocai Li, Liyun Lin, Shujie He, Di Zhang, Junyi Yu Jin, Yuanxin Liang, Michael Cecchini, Jill Lacy, John Kunstman, Pamela Kunz, Jing Du, Yang Liu
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Gastric cancer (GC) is the fifth-leading cause of cancer-related mortality, with a 5-year survival rate less than 20%. It develops from preneoplastic lesions to adenocarcinoma, but these early genetic alterations remain poorly understood. Therefore, we aimed to identify early genetic drivers underlying the development of preneoplastic lesions and the initiation of gastric carcinogenesis. Methods We characterized preneoplastic and early gastric adenocarcinoma using 48 samples from 16 Guatemalan patients, a country with a high incidence of GC. We sequenced a panel of 127 genes to identify early genetic drivers and possible actionable targets. Results We identified extensive genetic heterogeneity, including single nucleotide and copy number variations. After comparing our data with other studies, we identified TP53 and APC as the most mutated genes in preneoplastic lesions and early GC. Our mean tumor mutational burden was higher in diffuse (0.017 mutations/Mb) and intestinal adenocarcinomas (0.015) than in chronic gastritis (0.005), and analysis of the mutational signatures revealed several processes acting at different stages of the disease. Signatures S15 (DNA mismatch repair deficiency) and S03 (homologous recombination deficiency) were more frequent in early adenocarcinoma than in chronic gastritis, intestinal metaplasia, necrosis, tubular adenoma, and atrophy. Notably, 10 of 16 patients (62.5%) had at least one actionable mutation in their preneoplastic lesions or gastric adenocarcinomas. Conclusions We show that at the preneoplastic and earliest stages, GC is genetically heterogeneous and presents key cancer-driving mutations that may participate in neoplastic transformation and progression, with 62.5% of lesions having the potential for treatment. This study expands the limited research on early GC and highlights key opportunities for precision medicine in populations with high GC incidence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f131a17bcf2decc26bf9525e923b46c1c5c1bd" target='_blank'>
              Genetic heterogeneity and key driver mutations in the preneoplastic and earliest stages of gastric cancer
              </a>
            </td>
          <td>
            Eduardo Alfonso Marroquín-Estrada, Héctor Martínez-Gregorio, C. Díaz-Velásquez, Fernando Ambriz-Barrera, Eduardo Emilio Córdoba-García, Laura Itzel Hernández-Romero, Miguel Ruiz De-La-Cruz, Armando Cáceres, Elisa Hernández, José Rodolfo Gil, Elizabeth Orellana, Daniel Penados, Carolina Richter, Marcelino Díaz, Felipe Vaca-Paniagua
          </td>
          <td>2025-10-09</td>
          <td>Gastroenterology Report</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Glioblastoma (GBM), a highly aggressive malignant brain tumor, leads to poor clinical outcomes and low long-term survival rates. Using bioinformatics approaches, we analyzed single-cell RNA sequencing (scRNA-seq) data (GSE273274) from glioma patient samples in this study. It was found that mesenchymal-like (MES-like) cells in the GBM center (GC) significantly facilitated GBM’s aggressive spread and microenvironmental reprogramming. Further analysis on MES-like cells suggested that these cells may participate in the progression from glial differentiation to invasive metastasis. Differential expression analysis, univariate Cox regression, and LASSO regression identified COL22A1 as a key gene. COL22A1 was highly expressed in GC compared to GBM surrounding tissue (GS) and exhibited potential connection to aberrant phosphatidylinositol 3-kinase (PI3K) signaling activation, indicating that COL22A1 may act as a potential regulator of GBM, promoting malignant processes such as proliferation, apoptosis resistance, migration, and invasion. In vitro experiments showed that COL22A1 overexpression (COL22A1-OE) U-87 cells exhibited enhanced proliferation, viability, migration, and invasion. Moreover, immunoblotting of BCL-2 and BAX revealed that COL22A1-OE increased resistance to apoptosis in U-87 cells. However, all these effects were reversed upon silencing of COL22A1, which suggests COL22A1 could be a promising new target for GBM therapy. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03559-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a091b05d62e57c97e8673640ca9464f145f296ba" target='_blank'>
              Malignant progression of MES-like cells mediated by COL22A1 in the spatial heterogeneity of glioblastoma
              </a>
            </td>
          <td>
            Guangyuan Chen, Zhonghua Fu, Xulin He, Zhaoli Shen
          </td>
          <td>2025-10-06</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA-binding molecules regulate gene expression, replication, repair, and transcription, making their research crucial to molecular biology. This review investigated their architectures, processes, and impacts on cancer research and therapy to determine their biological and therapeutic potential. A thorough examination of published data on DNA-binding agents, including intercalators, groove binders, and metal complexes, focused on their chemical properties, biological activity, and therapeutic significance. Doxorubicin intercalated between base pairs to inhibit replication and transcription, whereas cisplatin produced covalent cross-links with guanine bases to cause tumor cell death. Multiple derivatives of metal complexes reduced tumor development by over 70% in leukemia models as DNA probes and therapeutics. Also reviewed, Dps proteins in *Escherichia coli* showed that their non-specific DNA binding offered up to 65% oxidative stress resistance compared to control cultures, validating DNA-protein protection as a survival strategy. In Phase II studies, amsacrine caused substantial remission in acute leukemia patients, whereas doxorubicin was more effective across many cancer types but had greater cardiotoxicity concerns. Selectivity, toxicity, and resistance limit DNA-targeting medicines, although they are successful. Chemical modifications like hydrophobic tailoring and sequence-specific binding have enhanced binding affinity and therapeutic index, yet only 10% of candidate compounds get clinical approval. Recent studies show that AI-driven design has expedited screening, lowering development costs by 30% and durations by 3–5 years. These data suggest that DNA-binding medicines have great potential in cancer, but safer, more selective, and resistance-free therapy are still needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bfaa0d838d07a9aad5a057a0a95be735af4700a" target='_blank'>
              Chemistry of DNA-binding Molecules
              </a>
            </td>
          <td>
            Evon Akram, Dina A. Najeeb, Asmaa A. Jawad, N. Bedair, A. M. Hussein, Saba R. Jaafar, Ruaa H. Ali, R. F. Shaher, M. A. Hussein, Daniah M. Hamid, Reem H. Al-Tabra, Alyaa K. Abood, Salam Mohammed
          </td>
          <td>2025-08-30</td>
          <td>Al-Salam Journal for  Medical Science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The STAG2 tumor suppressor gene is commonly inactivated by mutations in a wide range of common cancer types. STAG2 encodes a component of the cohesin complex, which controls sister chromatid cohesion and 3D genome organization. In bladder cancer, STAG2 mutations are most common in the earliest low-grade lesions, suggesting that mutational inactivation of STAG2 may be an initiating event. To provide insight into the mechanisms of STAG2 tumor suppression in bladder cancer, siRNA and shRNA were used to knock down STAG2 in several different human non-neoplastic bladder cancer precursor cell lines. Gene editing was used to generate cultured human cancer cell lines that differ only in the presence or absence of bladder-cancer derived STAG2 mutations. These systems were interrogated using RNA-seq, Western blot, and qRT-PCR before and after induced differentiation. We find that inactivation of STAG2 in bladder cancer cells and in bladder epithelial precursor cells resulted in concomitant inactivation of the H3K27me3 Polycomb chromatin mark. Inactivation of STAG2 also attenuated induced differentiation of bladder epithelial precursor cells. STAG2 and other components of cohesin were upregulated during this differentiation process. This study provides new insights into the role of STAG2 in the pathogenesis of bladder cancer, demonstrating roles for STAG2 in the regulation of Polycomb-mediated epigenetic regulation and in the differentiation of bladder epithelial precursor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7c4a993de9dc00d31e9868cf193ba7252115496" target='_blank'>
              STAG2 regulates polycomb and differentiation in urothelial precursors and bladder cancer
              </a>
            </td>
          <td>
            Youngrok Park, Wanying Xu, Tianyi Yang, Victoria K. Hill, Xiaoyuan He, Lisa Sadzewicz, Luke Tallon, Jung-Sik Kim, Fulai Jin, T. Waldman
          </td>
          <td>2025-10-15</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Gastric cancer (GC) remains one of the leading causes of cancer-related mortality worldwide due to its aggressive nature and high metastatic potential. Circular RNAs (circRNAs), characterized by covalently closed loop structures and high molecular stability, have emerged as important regulatory molecules in cancer biology. However, the biological functions and molecular mechanisms of most circRNAs in GC remain largely unexplored. Methods We conducted circRNA microarray profiling on three pairs of primary GC tissues and matched adjacent normal tissues to identify differentially expressed circRNAs. The expression of circ-ANKRD36C was validated by quantitative reverse transcription polymerase chain reaction (qRT-PCR) in additional clinical specimens. Functional assays were performed to assess the effects of circ-ANKRD36C knockdown on GC cell proliferation, migration, and invasion. Dual-luciferase reporter assays provided evidence of direct interaction between circ-ANKRD36C and miR-15a-5p. Clinicopathological correlations and survival outcomes were analyzed in 84 paired GC and normal tissue samples. Results Circ-ANKRD36C was significantly upregulated in GC tissues. Knockdown of circ-ANKRD36C suppressed GC cell proliferation, migration, and invasion in vitro. Dual-luciferase assays supported its direct interaction with miR-15a-5p, potentially influencing oncogenic targets such as BMI1 and CCND1. Clinically, high circ-ANKRD36C expression was associated with shorter overall survival (OS). Multivariate Cox regression analysis suggested its independent prognostic value (hazard ratio [HR] = 2.41; 95% confidence interval [CI] = 1.22–4.77; P = 0.011). Conclusion Circ-ANKRD36C is upregulated in gastric cancer and may serve as a prognostic biomarker. Functional evidence suggests it could act as a candidate regulator, but its mechanistic role requires further validation. Supplementary Information The online version contains supplementary material available at 10.1186/s12876-025-04287-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aec96a53ffb97eb8a42c6e93ec17c0d2fe31bed9" target='_blank'>
              Circular RNA profiling and functional analysis implicate circ-ANKRD36C as a potential prognostic biomarker in gastric cancer
              </a>
            </td>
          <td>
            Rongpeng Chen, Tianxing Ji, Yuanqing Chu, Feng Xiao, Zhicheng Feng, Guoqiang Wang
          </td>
          <td>2025-09-29</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Germline mutations in the DNA repair helicase XPD can cause the diseases xeroderma pigmentosum (XP) and trichothiodystrophy (TTD). XP patients bear an increased risk of skin cancer including melanoma. This is not observed for TTD patients despite DNA repair defects. To examine whether TTD cells harbor features counteracting tumorigenesis, we developed a TTD melanoma cell model containing the XPD variant R722W. Intriguingly, TTD melanoma cells exhibited reduced proliferation and an increased signature of the melanocyte lineage factor MITF, along with a strong basal upregulation of REDD2, an inhibitor of the mTOR/S6K/4EBP1-dependent messenger RNA (mRNA) translation machinery. REDD2 levels were partially driven by MITF and contributed to reduced melanoma proliferation. In a TTD model for melanocytes—the progenitor cells of melanoma—the MITF gene signature was also increased, but here without affecting REDD2 expression. However, ribosomal protein synthesis was reduced particularly in R722W melanocytes after UV stress, indicating a compromised mRNA translation machinery. Impaired translation was also demonstrated for the TTD XPD variant A725P, but not for an XP variant. Concludingly, the impaired translation and reduced fitness observed in TTD melanocytes and melanoma cells, particularly after UV stress, offer a possible explanation why TTD patients do not develop melanomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6262e92871597f9c2f4cf51c63e9ae2bb27bcd01" target='_blank'>
              Anti-tumorigenic properties by trichothiodystrophy mutations in melanocytic cells
              </a>
            </td>
          <td>
            Rupesh Paudel, Lena F Sorger, Anita Hufnagel, Mira Pasemann, Tamsanqa Hove, André Marquardt, Susanne Kneitz, Andreas Schlosser, Caroline Kisker, Jochen Kuper, Svenja Meierjohann
          </td>
          <td>2025-08-27</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Objective: Prostate cancer, a leading cause of cancer deaths in men, involves genetic alterations and DNA damage. Oxidative stress–induced DNA damage is crucial in its development, necessitating efficient DNA repair mechanisms. Base excision repair (BER), a key pathway, addresses oxidative DNA damage with enzymes like Apurinic/Apyrimidinic Endonuclease 1 (APE1). Recent findings show that altered APE1 expression disrupts BER balance, contributing to genomic instability and cancer progression. This study examines APE1 expression in prostate cancer tissues to clarify its role in cancer development and therapeutic implications. Assessing APE1 expression in prostate cancer may elucidate its role in tumor progression, therapeutic resistance, and genomic instability, offering deeper insights into the mechanistic relevance of the BER pathway in oncogenesis. Methods: A total of 50 male patients diagnosed with prostate cancer were included in this study. Tumor and adjacent normal tissue samples were collected from consecutive male patients who underwent Tru-cut prostate biopsies due to clinical suspicion of prostate cancer, and gene expression levels were analyzed using real-time polymerase chain reaction. Results: The results showed no significant difference in APE1 expression between tumor and normal tissues (P > .05). Clinical factors like smoking, age, diabetes, hypertension, PIRADS, and Gleason scores did not significantly correlate with APE1 expression. However, a statistically significant correlation was observed between PSA levels and APE1 expression (r: 0.287, P = .04). Conclusion: The authors’ findings suggest that while no substantial changes in APE1 expression were observed in prostate cancer tissues, the enzyme’s role in DNA repair remains crucial. Abnormalities in APE1 expression and function can contribute to genomic instability and cancer progression. Future studies with larger sample sizes and standardized methodologies are necessary to better understand the potential role of APE1 as a biomarker and therapeutic target in prostate cancer.Cite this article as: Rouhi V, Habibi S, Baykal Koca S, Köseoğlu H, Güven M. Investigation of apurinic/apyrimidinic endonuclease 1 expression changes in prostate cancer. Cerrahpaşa Med J 2025, 49, 0009, doi: 10.5152/cjm.2025.25009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f5a4b0fb3242b36fc3de3889e7200ec9f3f9fad" target='_blank'>
              Investigation of Apurinic/Apyrimidinic Endonuclease 1 Expression Changes in Prostate Cancer
              </a>
            </td>
          <td>
            Vahid Rouhi, Sanaz Habibi, Sevim Baykal Koca, Hikmet Köseoğlu, Mehmet Güven
          </td>
          <td>2025-08-29</td>
          <td>Cerrahpaşa Medical Journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 The complex heterogeneity of Acute lymphoblastic leukemia (ALL) often predicts poor prognosis, high morbidity, and drug resistance. Yet, the molecular drivers that initiate aggressive ALL have yet to be fully elucidated, especially in the context of the >30 subtypes of human T- and B-ALL. Here, we used a large-scale F0 transgenic screen in zebrafish to identify synergistic combinations of 65 putative oncogenes at inducing both T- and B-cell leukemia. This approach identified a new proto-oncogene SET that collaborates with both notch1a icn and mutationally activated Interleukin 7 Receptor (IL7R) to initiate aggressive leukemia in vivo. SET is a multifunctional protein that can act as a transcriptional regulator, a histone chaperone, and an inhibitor of protein phosphatase 2A (PP2A). The high expression of SET along with mutational activation of NOTCH1 and IL7R was also found in human B- and T-ALL, supporting their oncogenic roles in leukemogenesis. Our discovery that SET is capable of independently synergizing with known drivers to initiate ALL presents a unique opportunity to explore the nature of oncogene driven ALL and define the mechanisms by which it drives transformation. We next used Non-negative Matrix Factorization to unbiasedly identify six gene programs shared across the zebrafish ALL subtypes and assessed if each was predictive of outcome in human disease. One program comprised a MYC-driven transcription signature that unexpectedly predicted good outcomes in human T-ALL patients. Additional conserved programs independently stratified patients into poor outcomes and when combined with the MYC signature profile were able to better discern disease outcomes in patients, including the aggressive ETP-like and TAL1 DP-like T-ALL. Taken together, we have uncovered new roles for SET in driving ALL initiation, uncovered unexpected high MYC expression as predictive of good outcome, and defined a new combination biomarker gene signature that predicts poor overall survival in aggressive subtypes of human T-ALL. Our zebrafish-based screening approach is a powerful tool for comparative genomic studies to identify new oncogenic drivers, vulnerability pathways, and new biomarkers of aggression in ALL.



 James R Allen, Luis Antonio Corchete Sanchez, Mohamed N Bakr, Miriam Fernández-Lajarín, Alexandra Hazelwood, Nathan Ford, Anna M Lucianò, Alexandra Veloso, Alexander D Weissman, Olivia A Strom, Esther Rheinbay, David M Langenau. Zebrafish modeling predicts clinical outcomes in human acute lymphoblastic leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B035.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79aacf42881085b0060797945bc5360d7773e354" target='_blank'>
              Abstract B035: Zebrafish modeling predicts clinical outcomes in human acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            J. R. Allen, L. A. Corchete Sanchez, M. N. Bakr, Miriam Fernández-Lajarín, Alexandra Hazelwood, Nathan Ford, Anna M Lucianò, Alexandra Veloso, Alexander D Weissman, Olivia A Strom, Esther Rheinbay, David M Langenau
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chromosomal instability (CIN) and epigenetic reprogramming are central drivers of breast cancer progression, yet the mechanisms connecting them remain elusive. We uncover a direct role of EZH2 histone methyltransferase in promoting CIN in triple-negative breast cancer (TNBC). Across breast cancers, EZH2 expression correlates with copy number alterations, and its catalytic activity is associated with increased CIN in metastasis-initiating cells. Pharmacological EZH2 inhibition suppressed CIN, revealing an unexpected vulnerability. Integrated chromatin and transcriptome profiling identified Tankyrase (TNKS), a poly(ADP-ribose) polymerase, as a direct transcriptional target of EZH2. Mechanistically, EZH2-mediated TNKS suppression disrupts CPAP (centrosomal P4.1-associated protein), driving centrosome overduplication, multipolar mitosis and exacerbated CIN. In vivo, CIN suppression is a critical mechanism underlying the anti-metastatic effects of EZH2 inhibition. These findings delineate a previously unrecognized epigenetic mechanism governing CIN and establish EZH2 inhibitors as the first therapeutic agents capable of directly suppressing CIN, underscoring the need for trials with metastasis-focused endpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5010199d8cc937113c87be8a59e3c3e7e0efc720" target='_blank'>
              Epigenetic regulation of chromosomal instability by EZH2 methyltransferase.
              </a>
            </td>
          <td>
            Yang Bai, Albert S. Agustinus, Shira Yomtoubian, Cem Meydan, Dylan R McNally, Liron Yoffe, Melissa J Hubisz, Marvel Tranquille, Sneha Pramod, Christy Hong, Magdalena L Plasilova, Aakanksha R Kapoor, Arshdeep Singh, Henry G. Withers, Lukas E. Dow, A. Laughney, B. Bhinder, O. Elemento, Ari M Melnick, S. Bakhoum, Vivek Mittal
          </td>
          <td>2025-10-02</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="


 Epigenetic dysregulation drives many pediatric tumors, including neuroblastoma, an aggressive malignancy that accounts for approximately 15% of childhood cancer deaths. Drugs targeting epigenetic regulators have emerged, with FDA approvals of tazemetostat for patients with metastatic or locally advanced epithelioid sarcoma with incomplete resection and for patients with relapsed or refractory follicular lymphoma with EZH2 mutations. More recently, the menin inhibitor revumenib was FDA approved for children and adults with relapsed/refractory KMT2A-rearranged acute myeloid leukemia. Unfortunately, these drugs are generally not curative, and resistance develops by multiple mechanisms suggesting that combination therapies are needed. Our laboratory recently identified some neuroblastomas, particularly MYCN amplified, that are dependent on the SAGA complex, a transcriptional coactivator complex that regulates gene expression by modifying histones. Knockout of TADA2B, a gene essential to the histone acetyltransferase function of SAGA, or treatment with GSK699, a selective degrader of KAT2A and KAT2B, the canonical catalytic acetyltransferases of SAGA, causes cell cycle arrest and delayed progression in xenograft models of neuroblastoma. In order to nominate combination strategies with SAGA inhibitors, we used orthogonal approaches: DepMap nearest neighbor analysis and a CRISPR sensitizer screen. Utilizing the Broad Institute’s Cancer Dependency Map (DepMap), we identified EZH2, a core member of the PRC2 complex, as a co-dependency with the SAGA complex in neuroblastoma. Furthermore, we conducted a CRISPR/Cas9 screen with a focused sgRNA library (1,350 genes) to identify additional epigenetic sensitizers and drivers of resistance in neuroblastoma cells treated with GSK699. EZH2 knockout scored as sensitizing to GSK699 while knockout of BRD2, a member of the BET family of proteins and an epigenetic reader, scored as rendering resistance to GSK699. The utilization of both in vitro and preclinical studies confirmed that pharmacological inhibition of KAT2A/B using GSK699 and EZH2 using tazemetostat induced a combinatorial inhibitory effect on neuroblastoma tumor proliferation compared to single agent treatment, with marked prolongation of survival in xenograft models with combination therapy. Transcriptional analysis of tumors treated with combination therapy showed increases in epithelial to mesenchymal transition (EMT) programs and myogenesis differentiation pathways, a surprising finding under active investigation. In conclusion, our data demonstrates that inhibition of the SAGA complex in combination with EZH2 inhibition effectively suppresses the proliferation of MYCN-amplified neuroblastoma supporting further evaluation of this combination in models of neuroblastoma, including deeper mechanistic studies.



 Melinda Soeung, Kenneth Ross, Lucy Merickel, Silvi Salhotra, Audrey Taillon, Allen Basanthakumar, Kimberly Stegmaier. Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A043.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5216fb3aee2d0a8a7d9f8e5c04d43544aa31dd4b" target='_blank'>
              Abstract A043: Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma
              </a>
            </td>
          <td>
            Melinda Soeung, Kenneth Ross, L. Merickel, S. Salhotra, A. Taillon, A. Basanthakumar, K. Stegmaier
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Whole-genome sequencing efforts, have during the past decade, unveiled the central role of genomic rearrangements—such as chromosomal inversions—in evolutionary processes, including local adaptation in a wide range of taxa. However, employment of reference genomes from distantly or even closely related species for mapping and the subsequent variant calling can lead to errors and/or biases in the datasets generated for downstream analyses. Results Here, we capitalize on the recently generated chromosome-anchored genome assemblies for Arctic cod (Arctogadus glacialis), polar cod (Boreogadus saida), and Atlantic cod (Gadus morhua) to evaluate the extent and consequences of reference bias on population sequencing datasets (approx. 15–20 × coverage) for both Arctic cod and polar cod. Our findings demonstrate that the choice of reference genome impacts the mapping statistics, including mapping depth and mapping quality, as well as core population genetic estimates, such as heterozygosity levels, nucleotide diversity (π), and cross-species genetic divergence (DXY). Furthermore, using a more distantly related reference genome can lead to inaccurate detection and characterization of chromosomal inversions, i.e., in terms of size (length) and location (position), due to inter-chromosomal reorganizations between species. Additionally, we observe that some of the verified species-specific inversions are split across multiple genomic regions when mapped against a heterospecific reference. Conclusions Inaccurate identification of chromosomal rearrangements as well as biased population genetic measures could potentially lead to erroneous interpretation of species-specific genomic diversity, impede the resolution of local adaptation, and thus, impact predictions of their genomic potential to respond to climatic and other environmental perturbations. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03761-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4fee8a8dac5e8b20e6658ae5253626aa02d7e59" target='_blank'>
              Reference genome bias in light of species-specific chromosomal reorganization and translocations
              </a>
            </td>
          <td>
            Marius F. Maurstad, S. Hoff, José Cerca, M. Ravinet, Ian R. Bradbury, K.S. Jakobsen, K. Præbel, S. Jentoft
          </td>
          <td>2025-10-15</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="CENP-C, an essential component of the kinetochore, connects centromeric chromatin to the outer kinetochore, and thereby ensures accurate chromosome segregation. Although deletion of the Mis12-binding domain (M12BD) of CENP-C does not cause developmental disorders in mice, it promotes malignant tumor progression in the two-stage DMBA/TPA-induced skin carcinogenesis model. In this study, we have demonstrated that M12BD deletion of CENP-C enhances proliferation and then promotes abnormal differentiation in DMBA/TPA-induced carcinomas in mice. To elucidate the underlying molecular mechanisms, we performed RNA sequencing and found the dysregulated expression of keratinization-related genes. Intriguingly, elevated chromosomal aneuploidy was detectable in mice with the M12BD deletion of CENP-C. Among the aneuploidies, trisomies of chromosomes 6 and 10 took place at the highest frequency. These specific chromosomal gains were accompanied by upregulation of genes involved in immune and inflammatory responses. Together, our present findings strongly suggest that M12BD of CENP-C plays a critical role in the regulation of epithelial differentiation during tumor development in mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c39fcf56ce4b3277a5d1391cbbde0aec7e6c90" target='_blank'>
              Deletion of the Mis12C-Binding Domain of CENP-C Promotes Chromosomal Aneuploidy in Cutaneous Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Megumi Saito, Kazuhiro Okumura, Yurika Tokunaga, Sora Tanaka, Keisuke Otoyama, Yoshinori Hasegawa, Masatoshi Hara, Masakazu Hashimoto, Toshihiko Fujimori, Tatsuo Fukagawa, Yuichi Wakabayashi
          </td>
          <td>2025-10-03</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Hypoxia is a hallmark of the colorectal cancer (CRC) tumor microenvironment (TME) that drives malignant progression, chemoresistance, and immune evasion. However, the cellular heterogeneity underpinning hypoxic responses in CRC and its impact on prognosis remain incompletely understood. Methods We integrated single-cell RNA sequencing data from 15 CRC samples (GSE166555 and GSE221575) to delineate hypoxic and normoxic cell populations and identify hypoxia-related genes (HRGs). Weighted gene co-expression network analysis (WGCNA) and enrichment profiling elucidated key biological processes associated with hypoxia. Cell–cell communication networks were inferred using CellChat, and transcription factor regulatory modules were reconstructed via SCENIC and GRNBoost2. A hypoxia-based prognostic signature was developed from unique H3 cluster genes using univariate Cox and Lasso regression on The Cancer Genome Atlas (TCGA; n = 606) and validated in GSE39582 (n = 579). Drug sensitivity correlations were derived from the GDSCv2 database. Finally, in vitro assays assessed the functional role of GIPC2, a model gene, in CRC cell lines. Results Single-cell profiling uncovered eight distinct hypoxic clusters, with H3 exhibiting the highest hypoxia scores and enrichment in glycoprotein metabolic and angiogenesis pathways. The eight-gene prognostic model stratified patients into high- and low-risk groups with significantly different overall survival in both TCGA (P = 0.0026) and validation cohorts (P = 0.011). Drug analysis highlighted associations of model genes with PI3K/MTOR and apoptosis pathways. GIPC2 knockdown in LS180 and HT-29 cells markedly inhibited proliferation, migration, and invasion, while inducing apoptosis and reversing EMT phenotypes. Conclusions We present a robust hypoxia-related gene signature that accurately predicts CRC patient prognosis and nominate GIPC2 as a potential biomarker and therapeutic target, offering new insights into hypoxia-driven CRC biology and personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8575947e15fbc35fb6bf68728fce7870efdbff1" target='_blank'>
              Decoding the hypoxic tumor microenvironment in colorectal cancer for prognostic modeling and therapeutic target discovery
              </a>
            </td>
          <td>
            Xiao-Cui Duan, Yan Zhou, Fan Feng, Hai-Bo Jiang, Mei-Lin Wang, Zhe Han, Hong-Fei Pang, Yu-Hang Liu, Heng-Zhe Jia, Meng- He, Hongpan Xu, Yuan-Yuan Wang
          </td>
          <td>2025-08-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) remains the most lethal gynecologic malignancy, with a 5-year survival rate below 50%. Despite initial responsiveness to platinum-based chemotherapy and the implementation of targeted therapies such as PARP inhibitors, the majority of patients relapse and develop treatment-resistant disease. Emerging evidence suggests that the spatial organization and dynamic interactions among tumor, immune, and stromal cells within the tumor microenvironment (TME) play a critical role in therapeutic response and disease progression. However, traditional transcriptomic approaches lack the spatial context necessary to fully understand these interactions. The Xenium spatial transcriptomics (ST) platform represents an innovative technology that enables high-resolution, in situ mapping of gene expression within intact tissue architecture. By preserving the spatial relationships between cell types and molecular states, ST provides the opportunity to explore how the TME evolves under therapeutic pressure and contributes to resistance. We performed spatial transcriptomic profiling on FFPE tumor sections from three matched longitudinal pairs of HGSOC samples obtained prior to treatment and upon PARPi progression. Utilizing the Xenium 5K Plex platform from 10X Genomics, we quantified the expression of 5000 genes in over 2.5 million cells. Cell type annotation was performed via reference mapping and marker gene expression. Differential expression and gene ontology enrichment analyses were conducted to identify key transcriptional changes in tumor clusters between paired naïve and progression samples. We furthermore describe spatial cellular neighborhoods and their changes upon development of resistance. Characterization of the tumors reveals great morphological heterogeneity between patients. Comparative analysis revealed distinct shifts in the cancer cell abundance, cancer cell area and spatial arrangement of the TME between naïve and progression samples. Progression samples exhibited increased expression of, for example, hypoxia-related genes, particularly in cancer cells. Spatial mapping identified distinct cellular neighborhoods, including niches with a high density of neutrophil- or macrophage-associated transcripts, which were notably more abundant in progression samples, suggesting a role in immune evasion and tumor progression. This study demonstrates the potential of spatial transcriptomics to elucidate the architectural and transcriptional changes of the ovarian cancer microenvironment at the single-cell level. While based on early findings, our results suggest the relevance of hypoxia, stromal remodeling, and niches with distinct cellular compositions in contributing to disease progression. These data support further investigation into spatially targeted therapeutic strategies and the use of ST as a tool for microenvironment-driven biomarker discovery.



 Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera. Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cf6f443298e3305bcbf89470d2e118a90600ac1" target='_blank'>
              Abstract A003: Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance
              </a>
            </td>
          <td>
            Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Strand-specific RNA sequencing is crucial for elucidating complex transcriptional regulation, yet existing methods often compromise between sensitivity, accuracy, and workflow simplicity. Here, we present directional SHERRY (d-SHERRY), a streamlined strand-specific RNA-seq method that leverages Tn5 transposase activity on RNA/DNA hybrids to eliminate second-strand cDNA synthesis while preserving strand-of-origin information. Through systematically optimizing reverse transcription and tagmentation conditions, d-SHERRY achieves over 95% strand specificity and detects more than 10,000 genes from as little as 100 pg of input RNA, outperforming commercial kits in library complexity and coverage uniformity. Its high directional precision enables accurate resolution of complex genomic regions, including overlapping antisense transcripts such as SLC4A5/MTHFD2, with over 98% specificity. With a hands-on time of just 0.5-1 h, d-SHERRY offers a rapid, sensitive, and reliable solution for strand-specific transcriptome profiling across a broad range of sample types and input amounts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c79153f544b2dec3a7118ec33fc75ce7bd9f6d6b" target='_blank'>
              Transposase Acting on an RNA/DNA Hybrid in Strand-Specific Sequencing.
              </a>
            </td>
          <td>
            Shihui Zhao, Yanyi Huang, Chunhong Zheng
          </td>
          <td>2025-10-01</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Solitary fibrous tumors (SFTs) of the central nervous system (CNS) are rare, aggressive mesenchymal neoplasms with high recurrence and metastasis rates. Here, we perform a comprehensive multi-omics analysis of 189 cases of CNS SFTs integrating 94 whole genome sequencing, 88 transcriptomics, 7 single-nucleus RNA sequencing and 3 spatial transcriptome sequencing. We find that receptor tyrosine kinase mutations are significantly more prevalent besides the widespread NAB2-STAT6 fusion and correlate with tumor grade. Transcriptomic analysis reveals four molecular subtypes—classical, neural-like, inflamed and migratory—each associated with distinct clinical and biological characteristics. Single-nucleus RNA sequencing identifies five tumor cell states, with the SFT_classical state serving as a precursor to other states influenced by hypoxia and inflammation. Notably, we identify FER kinase as a key therapeutic target, with FER inhibition significantly reducing tumor cell proliferation, migration and invasion. These findings provide important insights into CNS SFT biology and suggest potential therapeutic strategies for this challenging tumor type.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25b5476111a14e85f839f2a38bdf41cc7317dd08" target='_blank'>
              Molecular landscape, subtypes, and therapeutic vulnerabilities of central nervous system solitary fibrous tumors
              </a>
            </td>
          <td>
            Chenhui Zhao, Xueyan Hu, Xiudong Guan, Xiaojun Fu, Tingting Wang, Mengyuan Li, Xinze Liu, Jiarui Zhao, Di Wu, Fan Zhang, Jiaying Fu, Jiang Li, Tieqiang Zhang, Xiaochun Jiang, Changxiang Yan, Wang Jia, Ence Yang, Jian Chen
          </td>
          <td>2025-08-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The interplay between nuclear architecture and extracellular matrix stiffness orchestrates cell fate decisions, yet the molecular mechanisms remain poorly defined. Here, we investigate the role of Lamin A (LMNA), a nuclear structural protein whose expression correlates with tissue stiffness, in regulating cellular differentiation and fate decision. Using myoblasts and fibroblasts as models, it was observed that cells with low LMNA expression showed that higher cell deformation elevated expression of neurological genes and exhibited potential for differentiation into a neural-like fate. CUT&Tag sequencing of LMNA-knockdown cells revealed a reduction in the size of Lamin B1-associated domains, with enhanced Lamin B1 binding at muscle-related genes (Myf5 and Myf6) and diminished binding at the neural gene Nes, suggesting that changes in gene expression are associated with alterations in chromatin structure. Further analysis identified the dissolution of H3K9me2/3-labeled heterochromatin regions and their redistribution in the nucleoplasm following LMNA inhibition. Soft substrates (0.2 kPa) amplify the neural differentiation capacity in LMNA-knockout cells. Additionally, retinoic acid was shown to enhance the expression of neurologically related genes by suppressing LMNA expression. These findings reveal a novel substrate stiffness-induced mechanism by which Lamin A regulates cell fate transitions and provide a new approach for neural cell generation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880b739a355b443815a774178c98532930794235" target='_blank'>
              Decreasing Lamin A Triggers Cell Fate Transitions through Heterochromatin-Nuclear Periphery Detethering
              </a>
            </td>
          <td>
            Lijuan Sun, Yafan Xie, Zhaoyan Zuo, Jian Liu, Jiaqi Yang, Iqra Ali, Qin Peng, Juhui Qiu
          </td>
          <td>2025-08-28</td>
          <td>Biomaterials Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Intercellular communication signals in the tumor microenvironment are closely related to behaviors such as cancer cell proliferation and immune evasion. However, the specific roles of intercellular signaling pathways in intrahepatic cholangiocarcinoma (ICC) have not yet been fully characterized. In this study, we analyzed publicly available single-cell RNA sequencing (scRNA-seq) data derived from paired samples of two intrahepatic cholangiocarcinoma (ICC) tissues and two adjacent normal tissues, thoroughly examining their cellular composition. InferCNV analysis was employed to compare tumor cells and normal cells, and pseudotime analysis was used to identify the growth and differentiation trajectories of the cells. Additionally, intercellular communication analysis was conducted to elucidate the communication networks between cells. Our analysis delineated the cellular ecosystem of ICC, identifying cell subclusters with shared characteristics between ICC and normal tissues. Notably, we characterized a distinct C7-E-T subcluster that exhibited high expression of CXCR4 and BPTF, markers associated with cancer stem cells (CSCs). Further investigation revealed that the MIF intercellular signaling pathway promotes the progression of ICC by activating intracellular signals in the MYC pathway. This study highlights the dysregulation of intercellular signaling pathways within tumor clusters, which influences the onset and progression of ICC. The cancer stem cell subpopulation (CXCR4hiBPTFhiE-T) exerts a significant influence on ICC progression by secreting relevant signaling molecules via the MIF signaling pathway. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17102-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28caecb5ae60429cc1067831e07cb74217728059" target='_blank'>
              Single-cell transcriptome sequencing reveals tumor stem cells and their molecular characteristics in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Xiaowei Gu, Fuzhong Liu, Junfeng Zhang, Nigeerayi Nuermaimaiti, Wenjia Guo, Xiaogang Dong
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Somatic cell nuclear transfer (SCNT) holds promise for animal cloning but remains limited by low efficiency and phenotypic abnormalities, often attributed to incomplete nuclear reprogramming. This study presents an integrative genomic and epigenomic analysis of cloned buffaloes and their respective donors using long-read Oxford Nanopore sequencing. Our results showed a high degree of genomic similarity between clones and donors, with most variations located in non-coding regions and structural variants (SV) distributions highly correlated at the chromosomal level. Gene and protein level overlap of SV-affected loci revealed 70.9–73.3% gene-level and 69.7–72.5% protein-level similarity. Despite this genetic similarity, DNA methylation analysis identified differentially methylated regions (DMRs), particularly in intergenic and promoter regions. Clones exhibited slightly lower CpG methylation than the donors. The DMRs in donor vs. clone comparisons indicated higher hypomethylated regions than hypermethylated regions. Functional enrichment of DMR-associated genes highlighted pathways linked to mitochondrial function, oxidative phosphorylation, and reproductive processes. Although clones showed moderate genome-wide methylation correlation with donors, key differences in methylation suggest incomplete epigenetic reprogramming. Despite these epigenetic differences, all clones were phenotypically normal and healthy into adulthood. This study offers the first comprehensive SV and methylome profile of SCNT-derived buffaloes and emphasizes the role of epigenetic mechanisms in clone development and health, providing valuable insights to enhance cloning efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ed0c9146dce33964d38cc27a7e6aa2903443aa1" target='_blank'>
              Unravelling High Nuclear Genomic Similarity and Mitochondria Linked Epigenetic Divergence in SCNT Derived Buffalo Clones via Long-Read Nanopore Genome Sequencing
              </a>
            </td>
          <td>
            Meeti Punetha, Dharmendra Kumar, Satish Kumar, Bhavya Maggo, Priya Dahiya, Pradeep Kumar, Rakesh K. Sharma, Yash Pal, P. S. Yadav
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aea70db16f84a083813eb45e33b7ae4b377f28b6" target='_blank'>
              Reply to: The chemotherapeutic drug CX-5461 is a potent mutagen in cultured human cells
              </a>
            </td>
          <td>
            S. Kanwal, N. Brajanovski, J. Zhan, J. Chan, R. Rebello, O. Hofmann, S. Grimmond, L. Furic, R. Hannan, G. McArthur, G. Poortinga, S. Harrison, E. Sanij, A. Khot, S. Sandhu
          </td>
          <td>2025-10-01</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chromatin remodeling by the SWI/SNF complex is essential for hematopoietic lineage commitment and differentiation. While core subunits ARID1A and ARID1B are frequently mutated in B cell malignancies, their functions remain unclear. Recent work established ARID1A-dependent functions within germinal center (GC) B cells, but its role during early B cell development, and whether its homolog, ARID1B, contributes distinct or compensatory roles at steady state or during transformation, remain unknown. Here, we used CD19-Cre-mediated deletion initiated at the pro-B cell stage to investigate their role in B cell development in vivo. Loss of either gene partially blocked B cell differentiation, reducing immature/recirculating B cell output, and impaired germinal center formation following antigen challenge. Combined deletion further reduced peripheral B cells, shortened survival, and resulted in aggressive leukemia. Unexpectedly, the malignancy was of myeloid origin and arose from a subset of CD19-expressing multipotent progenitors (MPPs). Arid1a/Arid1b-deficient MPPs exhibited abnormal expansion, reduced colony formation, and dysregulation of stemness and lineage-priming programs, including diminished CBFA2T3 (ETO2) and Fli1 signatures. In established B cell lymphoma cells in vitro, double ARID1A/ARID1B loss modestly affected cell growth, whereas loss of ARID1A increased sensitivity to EZH2 inhibition. Transcriptomic analyses revealed alterations in cell adhesion/migration pathways, cytokine-receptor interactions and DNA repair mechanisms. Collectively, these findings reveal stage-specific and compensatory roles for ARID1A and ARID1B in B cell development, uncover a mechanism by which SWI/SNF loss in MPPs redirects transformation towards myeloid leukemia, and suggest context-dependent therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53d7d9360a90c28b3536e18cfb6b090769877ccb" target='_blank'>
              ARID1A and ARID1B preserve B cell identity, prevent myeloid transformation and reveal therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Phyo Nay Lin, Joohyung Park, Yoon-A Kang, George P. Souroullas
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are guanine-rich DNA secondary structures with known regulatory roles in eukaryotes, but their functions in bacteria remain largely unexplored. Here, we systematically analyzed potential G-quadruplexes (PG4s) across 1566 bacterial genomes. PG4s showed non-random distributions, with strong enrichment in intergenic regions, particularly near gene start sites, suggesting potential roles in transcriptional regulation. Structural analysis revealed a higher proportion of short-loop G4s in these regions, indicating enhanced stability. Comparisons with randomized genomes confirmed that the observed patterns cannot be explained by GC content or genome size alone. Ecological analysis further revealed that bacteria associated with homothermic hosts exhibit significantly lower G4 densities than free-living species, suggesting reduced reliance on G4-mediated regulation in stable environments. Together, these findings highlight G4s as conserved yet context-dependent elements of bacterial genome organization and provide new insights into their potential regulatory roles in prokaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/969b849342d5007389accc241718dd3c89890d08" target='_blank'>
              Non-Random Distribution of G-Quadruplex Structures Reveals Regulatory and Ecological Adaptations in Bacterial Genomes
              </a>
            </td>
          <td>
            Jiye Fu, Ke Xiao, Yukun He, Jing Tu
          </td>
          <td>2025-10-15</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f396f45a05f69851b7610dc8198d4d5a27e35207" target='_blank'>
              Characterization of cis-regulatory elements and functional variants in colorectal cancer using epigenomics and CRISPRi screenings.
              </a>
            </td>
          <td>
            Zequn Lu, Can Chen, Heng Zhang, Bin Li, Yizhuo Liu, Jiayi Guo, Runying Xu, Ke Shi, Qianying Ma, Ming Zhang, Yimin Cai, Jinyu Huang, Hui Geng, Linyun Fan, Caibo Ning, Yanmin Li, Shuo Chen, W. Tian, Kexin Hu, Haijie Li, Xiaojun Yang, Chaoqun Huang, Yongchang Wei, Xu Zhu, Xiangpan Li, Zhen Xiong, M. Cai, Xiaoyang Wang, Shaokai Zhang, Hongda Chen, M. Dai, Kun Chen, Mingjuan Jin, Meng Jin, Ying Zhu, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2025-08-25</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d68b17c92ffe6cab39b80d1a0d838e3cbc4b57d" target='_blank'>
              EP300 genomic redistribution following E-cadherin inhibition in MCF7 cancer cells: evidence for early epigenetic reprogramming.
              </a>
            </td>
          <td>
            H. Kaabi, Aseel Sharaireh, Hanan Aljohani
          </td>
          <td>2025-10-13</td>
          <td>Histochemistry and cell biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) is a hematological malignancy driven by diverse genetic aberrations, with the Philadelphia chromosome and its resultant BCR-ABL1 fusion gene constituting key pathogenic drivers. Atypical BCR-ABL1 fusion transcripts have distinctive structural and functional properties. Structural divergence in these variants leads to functional alterations of encoded oncoproteins, potentially influencing disease progression and therapeutic responsiveness. Conventional diagnostic modalities, including reverse transcription-PCR and fluorescence in situ hybridization, may fail to detect rare variants, necessitating complementary high-sensitivity techniques such as next-generation sequencing). Tyrosine kinase inhibitors (TKIs), including imatinib and dasatinib, remain cornerstone treatments; however, marked inter-variant heterogeneity in TKI responsiveness is observed: Patients harboring e13a3/e14a3 transcripts generally show favorable prognoses, while those with e1a3/e6a2 variants demonstrate an increased risk of relapse and/or TKI resistance, often requiring multimodal strategies combining chemotherapy or allogeneic hematopoietic stem cell transplantation. Although Chimeric Antigen Receptor)-T cell therapy has shown promise in treating (Philadelphia chromosome-positive B-cell Acute Lymphoblastic Leukemia, its application in CML, particularly in variants such as e1a3 or e6a2, is not currently recommended as a first-line treatment. Despite advances in elucidating the clinical implications of fusion gene heterogeneity in leukemogenesis, the prognostic value of atypical BCR-ABL1 isoforms requires further validation through multicenter studies with extended cohorts. This review aimed to summarize cases of atypical fusion genes in CML, with analysis of clinical characteristics, therapeutic interventions, and prognostic outcomes, to provide clinicians with enhanced reference material for improved patient management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ec3c00c917da742543a693d4c05a294ce191255" target='_blank'>
              The silent players: Atypical BCR-ABL isoforms as biomarkers and therapeutic hurdles in CML pathogenesis (Review)
              </a>
            </td>
          <td>
            Xin Zhou, Ai Li, Dexiao Kong, Yuqi Shi, Peipei Zhang, Ningning Shan
          </td>
          <td>2025-09-24</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="DNA methylation is altered in all cancers, but the mechanisms responsible for these changes are not well understood. Using data from 100 primary samples, we show that regions with intermediate methylation in normal hematopoietic cells are hotspots for stable, clonal epimutations in acute myeloid leukemia. Analysis of hematopoietic stem cell clones demonstrated that intermediate methylation at epimutation hotspots reflects random allele-specific methylation and expression at the single cell level, representing a previously unrecognized form of somatically acquired imprinting. We identified somatically imprinted regions in other tissues and show they are epimutation hotspots in other cancer types. This demonstrates that random allele-specific methylation is both a general property of normal cells, and a vulnerability that renders them susceptible to cooperating events and clonal selection during cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e7fa3ac12fa0ce8e4468a1db8da8c5facb616d2" target='_blank'>
              Intermediately Methylated Regions in Normal Cells Are Epimutation Hotspots in Cancer
              </a>
            </td>
          <td>
            Mohamed Mahgoub, Haley Abel, Nidhi Davarapalli, Heidi Struthers, E. Kotnik, Brittany Johnson, Christopher Markovic, C. Fronick, Robert Fulton, Michael P. Meers, DH Spencer
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Abstract Background Unlike other cancer types the determinants of anti-PD-1 response in clear cell RCC are not well defined; ccRCC exhibits a low mutational burden and CD8+ T cells infiltration is associated with worse prognosis. Cytomegalovirus (CMV) and human papillomavirus (HPV) infections are postulated to modulate the anti-tumor immune responses within the tumor microenvironment. We sought to investigate the clinical and immunologic impact of CMV and HPV viral transcriptomic signatures, as defined by prior studies, in patients with RCC. Methods RNA-seq data from three RCC prospetive trials were upper-quartile normalized, log2-transformed (TPM + 1), and batch-corrected using ComBat. Transcriptomic scores for CMV (75 genes) and HPV (79 genes) were computed using geometric means (71 and 77 genes were found, respectively). CD8 immunofluorescence (IF) was performed on tumor samples. Overall survival (OS) and progression-free survival (PFS) were assessed using multivariable Cox regression, adjusting for age at therapy start, gender, IMDC risk, sarcomatoid/rhabdoid features, and number of previous lines received. Results A high intratumoral CMV transcriptomic score was significantly correlated with T-effector signature (ρ  =  0.54, p = 4.8 × 10−6), IF tumor core CD8+ (ρ  =  0.43, P < .001), IF tumor margin CD8+ (ρ  =  0.41, P < .001), and myeloid signature (ρ  =  0.40, P < .001). The CMV score was significantly higher in infiltrated versus excluded/desert tumors (P = .0079). The CMV score was associated with worse OS among patients receiving immune checkpoint blockade (ICB) (HR 4.98, P = .039). In contrast, the HPV transcriptomic score showed only a modest correlation with myeloid signature (ρ  =  0.24, P = .051) and no association was found with OS nor PFS. Conclusions In metastatic RCC, a high tumor CMV transcriptomic score was associated with a paradoxical immune landscape, marked by CD8+ infiltration yet co-enriched with suppressive myeloid signatures, that predicts inferior overall survival following ICB. These viral signatures within the complex immune architecture of RCC may help understand immune responsiveness and resistance within distinct genomic and therapeutic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c3c656ee910ac2c2d9287a96ba1a9501c97839" target='_blank'>
              39Intratumoral viral transcriptomic signatures stratify immune phenotypes and clinical outcomes in renal cell carcinoma (RCC)
              </a>
            </td>
          <td>
            Mustafa Saleh, E. Saad, Pablo Barrios, M. Machaalani, Liliana Ascione, Wassim Daoud Khatoun, Jad El Masri, Razane ElHajj Chehade, Marc Eid, Rashad Nawfal, Karl Semaan, E. Yekeduz, Clara Steiner, Weiwei Bian, Maxine Sun, S. Signoretti, E. V. Van Allen, D. Braun, Alexander Gusev, Toni Choueri
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Tumor heterogeneity, a hallmark of cancer, frequently leads to treatment failure and relapse. However, the intricate communication between various cell types within the tumor microenvironment and their roles in tumor progression in vivo remain poorly understood. Here we establish a novel tumor heterogeneity model in the Drosophila larval eye disc epithelium and dissect the in vivo mechanisms by combining sophisticated genetics with single-cell RNA sequencing. We found that mutation of the tricellular junction protein M6 in cells surrounding RasV12 benign tumors promotes their malignant transformation. Mechanistically, early RasV12//M6−/− tumors secrete Pvf1, which activates the Pvr receptor on hemocytes, facilitating their recruitment to the tumor site. These tumor-associated hemocytes secrete the Spätzle (Spz) ligand to activate the Toll receptor within the RasV12 tumors. This enhanced activation of the Toll pathway synergizes with RasV12 to promote malignant transformation through the JNK-Hippo signaling cascade. In summary, our study elucidates the complex interplay between genetically distinct oncogenic cells and between tumors and hemocytes, highlighting how hemocytes exploit the ancient innate immune system to coordinate tumor heterogeneity and drive tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d320162041d03e1b05ee82602e945a36e317508" target='_blank'>
              Hemocytes facilitate interclonal cooperation-induced tumor malignancy by hijacking the innate immune system in Drosophila
              </a>
            </td>
          <td>
            Sihua Zhao, Yifan Guo, Xiaoyu Kuang, Xiaoqin Li, Chenxi Wu, Peng Lin, Qi Xie, Zongzhao Zhai, D. Kong, Xianjue Ma
          </td>
          <td>2025-08-22</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Human Papillomavirus (HPV), particularly high-risk strains such as HPV16 and HPV18, is a leading cause of cervical cancer and a significant risk factor for several other epithelial malignancies. While the oncogenic mechanisms of viral proteins E6 and E7 are well characterized, the broader effects of HPV infection on host transcriptional regulation remain less clearly defined. This study explores the hypothesis that conserved genomic motifs within the HPV genome may act as molecular decoys, sequestering human transcription factors (TFs) and thereby disrupting normal gene regulation in host cells. Such interactions could contribute to oncogenesis by altering the transcriptional landscape and promoting malignant transformation.We conducted a computational analysis of the genomes of high-risk HPV types using MEME-ChIP for de novo motif discovery, followed by Tomtom for identifying matching human TFs. Protein–protein interactions among the predicted TFs were examined using STRING, and biological pathway enrichment was performed with Enrichr. The analysis identified conserved viral motifs with the potential to interact with host transcription factors (TFs), notably those from the FOX, HOX, and NFAT families, as well as various zinc finger proteins. Among these, SMARCA1, DUX4, and CDX1 were not previously associated with HPV-driven cell transformation. Pathway enrichment analysis revealed involvement in several key biological processes, including modulation of Wnt signaling pathways, transcriptional misregulation associated with cancer, and chromatin remodeling. These findings highlight the multifaceted strategies by which HPV may influence host cellular functions and contribute to pathogenesis. In this context, the study underscores the power of in silico approaches for elucidating viral–host interactions and reveals promising therapeutic targets in computationally predicted regulatory network changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019a01c1572bf807f787812d1b2ed2b11781bb4f" target='_blank'>
              HPV as a Molecular Hacker: Computational Exploration of HPV-Driven Changes in Host Regulatory Networks
              </a>
            </td>
          <td>
            M. Chetta, Alessandra Rosati, N. Bukvic
          </td>
          <td>2025-08-27</td>
          <td>Viruses</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 DNA damaging agents remain the mainstay for high grade serous ovarian cancer (HGSOC) therapy. While many HGSOCs are sensitized to DNA damaging agents via dysregulated DNA repair, those patients harboring tumors with amplification or overexpression of CCNE1 (encoding cyclin E1) respond poorly in part due to proficiency in the homologous recombination (HR) DNA repair pathway. Here, we put forth a novel paradigm that HR proficiency of CCNE1-driven HGSOCs is promoted by altered cellular metabolism. We found that alpha-ketoglutarate (aKG) is upregulated in cyclin E1-driven HGSOC cells, and suppression of aKG sensitized these cells to DNA damaging agents both in vitro and in vivo, demonstrating that aKG drives chemoresistance in these models. aKG is required for the activity of aKG-dependent dioxygenases (aKGDD). There are >60 known aKGDD, and prior work has almost exclusively focused on aKGDD-mediated histone demethylation. Through a targeted CRISPR knockout library, we discovered that Trimethyllysine Hydroxylase Epsilon (TMLHE), the first and rate-limiting enzyme in de novo carnitine synthesis, is necessary for chemoresistance to DNA damaging agents in cyclin E1-driven cells. Unexpectedly, aKG-mediated TMLHE-dependent carnitine synthesis was required for histone acetylation, while histone methylation was affected but dispensable. These data point to a previously unrecognized role for aKG in acetylation reactions. The increase in histone acetylation via aKG-dependent carnitine synthesis promoted HR-mediated repair through site-specific histone acetylation. Additionally, analysis of HGSOC patient samples demonstrated that TMLHE positively correlates with histone acetylation, and both high tumor TMLHE or high serum acetylcarnitine are associated with worse progression free survival (PFS) in HGSOC patients treated with DNA damaging agents. Pharmacological targeting of carnitine synthesis via mildronate in vivo was sufficient to decrease tumor burden >30% in combination with cisplatin. Since mildronate is a non-toxic and extensively used drug in humans, this combination shows potential applicability of inhibition of this axis in humans. These data demonstrate for the first time that HR-proficiency in cyclin E1-driven HGSOCs is mediated through a metabolic axis that directly influences histone acetylation and highlight the translational potential of targeting this axis to induce HR deficiency and sensitize these tumors to DNA damaging agents.



 Apoorva Uboveja, Nathaniel W. Snyder, Katherine M. Aird. aKG-mediated carnitine synthesis promotes chemoresistance in CCNE1-high ovarian cancers via enhancing histone acetylation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7139762235894a9ff9c39c95032bcf77d46290e" target='_blank'>
              Abstract B031: aKG-mediated carnitine synthesis promotes chemoresistance in CCNE1-high ovarian cancers via enhancing histone acetylation
              </a>
            </td>
          <td>
            Apoorva Uboveja, Nathaniel W. Snyder, K. Aird
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Cancer continues to be a significant global health issue, influenced by genetic mutations and external factors like carcinogenic exposure, lifestyle choices, and chronic inflammation. The myelocytomatosis (MYC) oncogene family, including c-MYC, MYCN, and MYCL, is essential in the development, progression, and metastasis of various cancers such as breast, colorectal, osteosarcoma, and neuroblastoma. Beyond its well-known roles in cell growth and metabolism, MYC significantly shapes the tumor immune microenvironment (TIME) by altering immune cell dynamics, antigen presentation, and checkpoint expression. It contributes to immune evasion by upregulating checkpoints such as programmed death-ligand 1 (PD-L1) and cluster of differentiation (CD)47, suppressing antigen-presenting major histocompatibility complex (MHC) molecules, and promoting the recruitment of suppressive immune cells such as regulatory T cells (Tregs) and myeloid-derived suppressor cells (MDSCs). While direct targeting of MYC has proven challenging, recent advances in therapeutic strategies, including MYC-MYC-associated factor X (MAX) dimerization inhibitors, bromodomain and extra terminal domain (BET) and cyclin dependent kinase (CDK) inhibitors, synthetic lethality approaches, and epigenetic modulators, have shown promising results in preclinical and early clinical settings. This review discusses MYC's comprehensive impact on TIME and examines the promising therapeutic strategies of MYC inhibition in enhancing the effectiveness of immunotherapies, supported by recent preclinical and clinical findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec1348f06f8cc8d1b1f7ddaa9bea0e5801ca6bda" target='_blank'>
              The Role of MYC in Tumor Immune Microenvironment Regulation: Insights and Future Directions.
              </a>
            </td>
          <td>
            B. Nirala, Jason Yustein
          </td>
          <td>2025-09-23</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) with rearrangement of the mixed lineage leukemia gene express MLL-AF9 fusion protein, a transcription factor that impairs differentiation and drives expansion of leukemic cells. We report here that the zinc finger protein GFI1 together with the histone methyltransferase LSD1 occupies the promoter and regulates expression of the lncRNA ELDR in the MLL-r AML cell line THP-1. Forced ELDR overexpression enhanced the growth inhibition of an LSD1i/ATRA combination treatment and reduced the capacity of these cells to generate leukemia in xenografts, leading to a longer leukemia-free survival. We found that ELDR binds the clamp protein PCNA and the MCM5 helicases causing defects of DNA replication fork progression. Moreover, AML cells overexpressing ELDR showed reduced chromatin accessibility and transcription at α-satellite repeats in centromeres. In addition, ELDR RNA was detected close to MLL-AF9 at centromeres suggesting that it impedes leukemic progression preferentially of MLL-r AML by interfering with both DNA replication and centromeric transcription. Our findings reveal novel functions of the lncRNA ELDR in DNA replication and centromere biology when expressed at high levels in AML cells with MLL rearrangements. These discoveries could provide rationale for future strategies to treat MLL-r AML, which has a poor prognosis in children and adults. Delivery of the ELDR RNA could potentially be utilized as an adjunct to LSD1i/ATRA treatment or other currently used chemotherapeutic drugs to develop novel therapies for these AML subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bca6f7f7a7b4fc0402c634bb0d6e344f0b3e15b" target='_blank'>
              The lncRNA ELDR suppresses tumorigenicity of AML by interfering with DNA replication and chromatin accessibility.
              </a>
            </td>
          <td>
            K. Arman, Julie Ross, Eva-Maria Piskor, Luca Lazzari, Virginie Calderon, Gaspard Reulet, Maria Vera, Edlie St Hilaire, Hugo Wurtele, Brian T Wilhelm, T. Möröy
          </td>
          <td>2025-09-08</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a major health burden in most parts of the world, and pathogenesis of the disease is strongly associated with a complex combination between genetic mutations and epigenetic alterations. The current study will explore the DNA methylation phenomenon and how it affects the silencing of tumour suppressor genes and activate oncogenes in OSCC in a bid to explain the driving molecular process behind tumour development. We evaluated 50 samples of the OSCC tissues with 50 of the adjacent normal tissues, evaluated DNA methylation patterns with methylation-specific PCR and quantitative reverse-transcriptase PCR measured the expression of the DNA repair genes like BRCA1 and MLH1, among others. This finding showed different levels of DNA methylation of cancerous and normal tissues with hypermethylation causing the inactivation of important tumor suppressor genes and hypomethylation causing the activation of oncogenes. In addition, the downregulation of DNA repair genes was noted to be highly significant in OSCC samples indicating that genomic-instability may be related to epigenetic changes. These results demonstrate that aberrant DNA methylation is central to OSCC growth and progression, thus helping in the future use of methylation patterns that can be used as early detection, diagnosis, and prognostic biomarkers. Our findings support the idea that genetic, along with epigenetic, profiling is an issue of key importance toward comprehending OSCC biology and personalized therapeutic interventions. It is recommended that further confirmation be carried out to verify the clinical significance of such epigenetic markers in bigger cohorts, as well as testing such markers in guided therapies, which might eventually lead to better patient outcomes in the management of oral cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fd5e50c6888572f5cfc48d30ee386841f2ea57" target='_blank'>
              Epigenetic and genetic events of oral squamous cell carcinoma: perspective on DNA methylation, silencing of tumor suppressor gene, and activating oncogenes.
              </a>
            </td>
          <td>
            Zainab Nizar Jawad
          </td>
          <td>2025-10-07</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Replication stress (RS), if not effectively and timely addressed, could result in DNA damage in mitosis. However, the relationship between RS and other mitotic events, such as nuclear envelope (NE) breakdown and reassembly, remains poorly understood. Here we report that RS can lead to NE defect. Importantly, rather than de novo NE rupture, the defect per se is a result of nuclear envelope reassembly defect (NERD) during mitosis. Interestingly, NERD is associated with mitotic DNA damage, and repair of the damage by DNA polymerase theta (Polθ)-mediated end joining (TMEJ) ameliorates NERD. Genomic mapping of lamina associated domains (LADs) by cleavage under targets and tagmentation (CUT&Tag) identifies a population of replication stress-sensitive LADs (RESSLADs). Strikingly, a substantial portion of RESSLADs reside in the common fragile sites (CFSs). The loss of RESSLADs-NE interaction under RS might be attributed to the sustained phosphorylation of Lamin A/C at the sites of NERD. In addition, prominent NE defect is observed under multiple conditions of synthetic lethality. Altogether, these findings establish a link between genome instability and nuclear vulnerability under replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/939377138dd121885a6964852dac3b04dd8c9c53" target='_blank'>
              Mitotic DNA repair by TMEJ suppresses replication stress-induced nuclear envelope reassembly defect
              </a>
            </td>
          <td>
            Guojun Ye, Yide He, Yihui Zhang, Dongchen Li, Fuhai Liu, Yi Li, Qinglian Ge, Qiong Guo, Shuya Han, Chunyu Song, Weiping Chang, Haoyue Zhang, Qin Peng, Kun Sun, Wei-Ke Ji, Lin Deng
          </td>
          <td>2025-10-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Protein arginine methyltransferase 5 (PRMT5) has pleiotropic functions in human cells but also participates in orchestrating DNA double strand break (DSB) repair. It methylates the TIP60 histone acetyltransferase complex to facilitate recruitment to the DSB and chromatin remodeling. PRMT5 mutations affect DSB repair by homologous recombination and increase chromosomal instability. In this report we characterized genetic interactions between PRMT5 mutations and mutations in other components of DSB repair pathway. We used data deposited on the Catalogue of Somatic Mutations in Cancers (COSMIC). We found that PRMT5 makes negative genetic interactions with TIP60 and member of the 9-1-1 complex (RAD9, RAD1, HUS1) which is required for checkpoint activation. A comprehensive analysis of all cancer data deposited on COSMIC reveals very few samples with mutations in both PRMT5 and TIP60 or components of the 9-1-1 complex in samples where mutations in other DNA damage repair genes occur (e.g., MRN, checkpoint genes, etc). This suggests that when more factors of the DNA damage repair machinery are destabilized, the functions of TIP60 and 9-1-1 appear to become essential. Protein 3-D structure analysis shows that mutations affect protein-protein interactions that may destabilize 9-1-1 or TIP60 complex formation. These data highlight interesting interactions between the various genetic pathways governing DSB repair. It also reveals potential therapeutic targets. For example, inhibition of the 9-1-1 complex in a PRMT5 mutant may selectively kill the cell. Given that PRMT5 small molecule inhibitors are being developed or already deployed, these findings should inform potential applications of these drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50af5dcce0290d41d138fd6f03848b6fe42b6650" target='_blank'>
              PRMT5 genetic interactions with DNA double strand break repair genes
              </a>
            </td>
          <td>
            Hunter J. Bliss, Juliana Tron, Wesley Bush, Renee A. Bouley, Ruben C. Petreaca
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd5a74f2db9883066216ff43d03114dc692d9adc" target='_blank'>
              Robust methylome analysis and tumour-normal classification in TCGA-COAD: a reproducible workflow
              </a>
            </td>
          <td>
            M. Ekum, C. Heumann, E. E. E. Akarawak, K. O. Adekoya, S. I. Oke, A. S. Ogunsnaya, C. Zhang, O. J. Akintande, J. A. Akinyemi, I. Ajiboye, O. D. Amusa
          </td>
          <td>2025-10-17</td>
          <td>None</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Osteosarcoma is a genomically complex tumor characterized by widespread structural rearrangements. This complexity has limited development of therapeutic strategies informed by molecular mechanisms of oncogenesis. We hypothesized that epigenetic mechanisms could drive distinct subtypes of osteosarcoma. Through analysis of chromatin accessibility, we identified an "early osteoblast-derived" (EOD) cell state characterized by upregulation of transcription factors associated with early bone development, and a "late osteoblast-derived" state (LOD), characterized by upregulation of genes involved in late bone development. We then defined core regulatory circuitries governing the underlying gene expression programs in these two cell states. Multiomic single-cell analysis indicates that these cell states co-exist in a single tumor. Finally, using a panel of patient-derived xenograft models, we identified differential drug responses dependent on these cellular states. These findings create opportunities for developing new combination therapy strategies for osteosarcoma treatment and underscore the value of defining epigenetic subtypes in highly genomically complex cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061de65b88f0b6c7fc51a022f24db3f11873c460" target='_blank'>
              Epigenetic and transcriptional programs define osteosarcoma subtypes and establish targetable vulnerabilities.
              </a>
            </td>
          <td>
            E. López-Fuentes, Andrew S Clugston, Alex G Lee, L. Sayles, Natalie Sorensen, María V Pons Ventura, Stanley G Leung, Truc Dinh, Marcus R. Breese, E. Sweet-Cordero
          </td>
          <td>2025-10-03</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Breast cancer biology is influenced by both genetic ancestry and the environment with implications for disease outcomes. However, we lack the knowledge how non-cancerous cells within tumors are functionally altered by these factors at single-cell resolution. Here, we created a single-nucleus (sn) dataset from 33 African American, 25 Kenyan, and 24 European American women with breast cancer using combined RNA and ATAC sequencing. We successfully isolated intact, high-quality single nuclei from archival frozen breast tumor tissue using an optimized combination of enzymatic digestion and automated tissue homogenization. SnMultiome sequencing of 82 tumors was performed using the 10x Genomics platform. Following filtering, normalization, peak calling, and integration, our dataset includes a total of 292,458 nuclei. Cancerous (163,419 nuclei) and non-cancerous (129,039 nuclei) cells were distinguished based on DNA copy number. Within the microenvironment, 11 major immune, epithelial, and stromal cell types were identified. CD45+, EPCAM- cells were further analytically extracted, clustered, and manually annotated based on known marker genes, detecting 26 distinct immune subpopulations across our samples. After adjustment for potential confounders (age, body mass index, and tumor molecular subtype) in all analyses, we found that women of African descent possessed distinct, activated mesenchymal and endothelial cell subpopulations within the tumor vasculature that promote inflammation, angiogenesis, and cancer cell invasion. Women of African descent exhibited a more immune-suppressive tumor microenvironment, with elevated expression of immune checkpoint markers (PD-1, TIGIT, CTLA-4, and LAG-3), functional impairment of dendritic cells, and enrichment of regulatory and tolerogenic lymphocytic subpopulations. Further, malignant cells from African descent patients expressed a discrete set of CAR-T targets with potential implications for precision immunotherapy. Lastly, we developed three gene signatures within the microenvironment that are predictive of survival with external validation in both tumor-adjacent normal and cancerous tissues from the TCGA. Together, we uncovered molecular breast tumor characteristics of clinical significance in women of African ancestry.



 Alexandra R. Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H. Dorsey, Eytan Ruppin, Shahin Sayed, Francis Makokha, Gretchen L. Gierach, Stefan Ambs. Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C109.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b4c16fead2784b8ef235041a0b9c641f582e01" target='_blank'>
              Abstract C109: Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy
              </a>
            </td>
          <td>
            Alexandra R Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H Dorsey, E. Ruppin, Shahin Sayed, Francis W. Makokha, Gretchen L. Gierach, S. Ambs
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Introduction Ovarian Cancer remains a significant global health concern, with high mortality rates, largely due to late-stage diagnosis and limited treatment options. These extrinsic factors are driven or exacerbated by intrinsic mechanisms such as persistent activation or upregulation of Signal Transducer and Activator of Transcription 3 (STAT3). STAT3 promotes tumor growth, inhibits apoptosis, accelerates angiogenesis and metastasis, facilitates immune evasion, and contributes to chemoresistance. Consequently, STAT3 activation fosters an aggressive ovarian cancer phenotype, contributing to treatment failure, poor prognosis and low survival rates, highlighting the urgent need for novel, safe, effective and affordable STAT3-targeted therapeutic strategies. In this study, we developed a novel double-stranded DNA minicircle (mcDNA) inhibitor, designed to act as a decoy for STAT3, preventing its binding to target gene promoters. Methods Utilizing the SKOV3 ovarian cancer cell line, we evaluated the effects of our inhibitor in vitro on cell viability through MTS assay, its apoptotic and necrotic effects using flow cytometry and the expression modulation of downstream STAT3-regulated genes, assayed through RT-qPCR and Western blot analysis. Results We demonstrate that anti-STAT3 mcDNA significantly reduces the viability of SKOV3 cells at low nanomolar concentrations, while sparing the control group. The effect observed was dose-dependent. Mechanistically, anti-STAT3 mcDNA induces apoptosis and necrosis in treated cells, also revealing a certain dose-dependency, while also decreasing cell proliferation. Finally, our inhibitor significantly downregulates STAT3-dependent anti-apoptotic genes MCL1 and PIM1. Conclusion These findings suggest that anti-STAT3 mcDNA is a promising, effective and specific candidate for targeted STAT3 inhibition in SKOV3 ovarian cancer cells, warranting further validation in ovarian cancer, in vivo exploration and potential application in other types of malignancies, where STAT3 acts as an oncogenic factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/298edc0f21db1280384ca9a0eea5b54ca57a8bc1" target='_blank'>
              Breaking the cancer code: a novel DNA minicircle to disable STAT3 in ovarian cancer cells SKOV3
              </a>
            </td>
          <td>
            Adina-Gabriela Vasilescu, Andrei-Mihai Vasilescu, Livia E. Sima, Natalia Baran, S. Szedlacsek
          </td>
          <td>2025-09-15</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genomic studies have provided key insights into the molecular pathogenesis of differentiated thyroid carcinoma (DTC), including the role of genes involved in the homologous recombination (HR) related to DNA repair and genomic stability. This research aimed to investigate the genetic landscape of HR genes in thyroid pathology, associated with recurrence risk and clinical prognosis. The study involved six individuals with thyroid conditions, including two patients diagnosed with papillary thyroid carcinoma (PTC) and four individuals with benign thyroid disease. The research material consisted of tumor samples collected during surgical procedures. Protein interactions were analyzed using the STRING database (string-db.org). Homologous recombination genes were sequenced using the HRR Panel vr1.0 on the MiSeq™ Sequencing System. Bioinformatics analysis revealed a relationship between BRAF mutations and HR gene defects in PTC. Mutations in BRCA1, BRCA2, and FANCA genes, typically associated with thyroid tumors, were identified in the tissue of papillary thyroid cancer (PTC). A statistically significant correlation was found between the FANCA gene mutation (rs7195066) and the recurrent course of the PTC. The preliminary findings suggest a potential role for non-pathogenic BARD1 mutations in follicular adenoma. No significant association was found between genes involved in homologous recombination repair and the incidence of papillary thyroid carcinoma, suggesting that these genes may not play a major role in the development of this type of thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c533dacb4e803fcf956ea07926e6fa6ea3901b" target='_blank'>
              Homologous Recombination in Thyroid Tumor Samples
              </a>
            </td>
          <td>
            L. Spirina, M. M. Tsyganov, S. U. Chizhevskaya, Natalia V. Tarasenko, Veronika A. Bogdanova
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Graphical Abstract Abstract Dysfunction of several WD40 family proteins causes diverse endocrine diseases. Until recently, MEP50, a WD40 protein, was considered a gene of unknown significance because no inherited disease had been linked to its function. However, genetic inactivation of MEP50 in mouse models or somatic mutations in humans drives oncogenesis in several endocrine-related cancers, including those of the prostate, breast, and uterus. In this study, we generate new knowledge through a multi-tier integration of evolutionary genomics, sequence and structural analyses, molecular mechanic calculations, and dynamic simulations of wild-type and cancer-mutated MEP50 proteins. Indeed, we find that a conserved splicing event across evolution generates an alternative MEP50 isoform, which is smaller than the canonical MEP50 and lacks the final β-sheet of the first WD40 domain, the entirety of the second WD40 domain, and the first β-sheet of the third WD40 domain. Notably, we find that this novel, short MEP50 (s-MEP50) transcript encodes a 278 amino acid protein that retains aspects of the key regulatory and interaction sites, including those critical for androgen receptor and PRMT5 binding. Finally, we analyze the mutational landscape of MEP50 in endocrine-regulated cancers and use molecular mechanic calculations and dynamic simulations to reveal that cancer-associated mutations disrupt conserved bonding networks and induce widespread structural destabilization within the WD40 domain architecture. Thus, by combining evolutionary, structural, and biophysical approaches, we advance the understanding of MEP50 genomics, providing significant mechanistic and clinically relevant insights into endocrine-regulated tissues and their cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e4d5f2f3bba5deb45827f67cca6b9cab9291043" target='_blank'>
              Genomic features of the methylosome protein MEP50 and its implications in hormone signaling and cancer
              </a>
            </td>
          <td>
            Gareth Pollin, Young-In Chi, Raul Urrutia, G. Lomberk
          </td>
          <td>2025-09-01</td>
          <td>Endocrine Connections</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Oncogenesis, tumor progression and therapy response are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions between these two systems influence tumor evolution and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC that systematically integrates genetic cancer drivers and infiltration profiles of immune cells to find pairwise combinations of drivers and TME characteristics that jointly associate with clinical outcomes. By analyzing 8500 primary tumor samples of 26 cancer types, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell levels were significantly correlated with patient survival. Subsets of tumor samples defined by some IGXs were characterized by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of immunotherapy target genes. In luminal-A breast cancer, an IGX involving MEN1 deletion combined with reduced levels of neutrophils associated with lower progression-free survival and deregulation of immune signaling pathways, as observed in two independent cancer genomics datasets. These results showcase the ability of PACIFIC to integrate complex multi-omics datasets with clinical information, enabling the identification of clinically relevant immunogenomic interactions. Such interactions provide a rich set of hypotheses for mechanistic studies and the development of biomarkers and therapeutic targets. Implications: Co-occurrence patterns of cancer drivers and TME characteristics highlight synergistic interactions with prognostic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3413ab1802170ae0154c57fdbd8827be618414" target='_blank'>
              Cancer genomic alterations and microenvironmental features encode synergistic interactions with disease outcomes.
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P Klein, Alexander T Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C L Cheng, Jigyansa Mishra, Diogo Pellegrina, K. Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2025-09-03</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC) remains a lethal malignancy with limited therapeutic advancements, underscoring the need to identify novel oncogenic drivers. This study integrates multi-omics analyses-including ChIP-seq, bulk RNA-seq, single-cell ATAC/RNA-seq, and spatial transcriptomics-to delineate enhancer-driven transcriptional networks in 59 matched LUSC tumors and adjacent normal tissues. The analyses identify 3,447 tumor-specific oncogenic enhancers (SOEs) enriched for master transcription factors (SOX2, TP63, KLF5, GRHL2) that orchestrate malignant programs. Single-cell epigenomic profiling reveals these SOEs to be exclusively active in a cancer cell-like cluster, pinpointing PTPRZ1, a receptor tyrosine phosphatase, as a top SOE-driven candidate. Functional studies demonstrate that PTPRZ1 is essential for LUSC-tumorigenesis and tumor cells proliferation and migration in vitro and in vivo. Spatial transcriptomic analysis further implicates midkine (MDK) as the ligand activating PTPRZ1 in LUSC tumor tissue. Mechanistically, PTPRZ1 mediates MDK-induced PI3K phosphorylation that is essential for LUSC tumor growth. This work illustrates enhancer addiction as a hallmark of LUSC, identifies the PTPRZ1-MDK axis as an important targetable pathway, and establishes a paradigm for dissecting epigenetic vulnerabilities in solid tumors through multi-omics integration. These findings advance precision oncology strategies for LUSC, bridging epigenomic dysregulation to therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdd6b0f8500832ade4c7417d38653f9c0588ff5" target='_blank'>
              Enhancer Reprogramming Reveals the Tumorigenic Role of PTPRZ1 in Lung Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yong-Qiang Ning, Jie Wang, Yihui Zhang, Hongrui Li, Zenan Jiang, Xinyu Su, Xiao Yang, Xiaoxuan Wu, Bing Wei, Jianming Ying, Jian Yan, Wenbin Li
          </td>
          <td>2025-09-03</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="In the initiation and development of cancer, DNA damage plays a very crucial role. In the case of environmental perturbation, mutation events in critical control genes may occur as the integrity of genetic material is lost, either due to replication errors, oxidative stress, etc. And provided that these are proto-oncogenes or tumour suppressor genes, they can interfere with normal control of the cell cycle, programmed cell death (apoptosis) and repair of damaged DNA(Branzei & Foiani, 2023). Consequently, they can start to multiply out of control, which can result in the appearance of growth and malignancy. This abstract discusses the molecular processes that connect the occurrences of DNA damage and mutations with the out-of-control growth of cancer cells, pointing to their importance in the development of cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1cd781c4b40bc6486e36eec6ae725a99f2c36b2" target='_blank'>
              Exploring How DNA Damage And Mutations Lead To The Uncontrolled Growth Of Cancer Cells?
              </a>
            </td>
          <td>
            Yashvi Chhabra
          </td>
          <td>2025-09-24</td>
          <td>International Journal of Scientific and Research Publications</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 The mechanisms governing the progression of IDH-wildtype glioblastoma (GBM) following standard-of-care therapy remain incompletely understood. While recurrence is nearly universal, the cellular and molecular trajectories that underpin this process - particularly the dynamics of tumor and microenvironmental compartments - have yet to be fully resolved.



 To dissect the longitudinal evolution of the GBM ecosystem, we analyzed paired primary and recurrent tumor specimens from 59 patients. We employed single-nucleus RNA sequencing to profile transcriptomic states at single-cell resolution and integrated these data with bulk DNA sequencing to assess genomic alterations. This approach enabled a comprehensive interrogation of cellular heterogeneity and malignant state transitions over time.



 Across the cohort, the most consistent feature at recurrence was a decreased proportion of malignant cells, accompanied by a reciprocal expansion of non-malignant glial and neuronal cell populations within the tumor microenvironment (TME). While the dominant malignant cell state often differed between primary and recurrent samples, no state was uniquely associated with a specific disease stage. Moreover, no singular evolutionary trajectory characterized the cohort as a whole. Instead, subsets of patients exhibited enriched and partially convergent state transitions. Notably, shifts in malignant cell states were mirrored by concurrent remodeling of the TME, implicating a tightly interwoven pattern of tumor-microenvironment co-evolution.



 These findings reveal diverse and patient-specific evolutionary trajectories in IDH-wildtype GBM, shaped by both therapeutic pressure and microenvironmental context. Our study provides a reference framework for understanding longitudinal GBM dynamics and highlights the importance of ecosystem-level interactions in driving recurrence.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e738ce34652886a4547b3d5a53d1b822511571f" target='_blank'>
              OS05.2.A DECIPHERING THE LONGITUDINAL TRAJECTORIES OF GLIOBLASTOMA ECOSYSTEMS BY INTEGRATIVE SINGLE-CELL GENOMICS
              </a>
            </td>
          <td>
            A. Spitzer, K. Johnson, M. Nomura, L. Garofano, D. Nehar-Belaid, N. Galili Darnell, A. Greenwald, L. Bussema, Y. Oh, F. Varn, F. D’Angelo, S. Gritsch, K. Anderson, S. Migliozzi, L. G. Gonzalez Castro, T. Chowdhury, N. Robine, C. Reeves, J. Park, A. Lipsa, F. Hertel, A. Gołȩbiewska, S. Niclou, L. Nusrat, S. Kellett, S. Das, H. Moon, S. Paek, F. Bielle, A. Laurenge, A. D. Di Stefano, B. Mathon, A. Picca, M. Sanson, S Tanaka, N. Saito, D. M. Ashley, S. Keir, K. L. Ligon, J. Huse, W. Yung, A. Lasorella, A. Iavarone, R. Verhaak, I. Tirosh, M. Suvà
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Background/Objectives: Oral squamous cell carcinoma (OSCC) is a common and aggressive oral cancer with high recurrence and mortality rates, largely due to late diagnosis and metastasis. Epigenetic regulation, particularly aberrant DNA methylation, plays a critical role in cancer progression. Altered methylation patterns disrupt cancer-related gene regulation. Our previous study found that oral cancer patients exhibit increased synthesis of S-adenosyl-L-methionine, a key methyl donor for cytosine methylation. Therefore, the aim of this study was to explore the relationship between global DNA methylation and OSCC progression and to evaluate the impact of DNA methylation heterogeneity on oral cancer cells. Methods: Immunohistochemistry (IHC) and immunofluorescence (IF) staining were used to examine 5-methylcytosine (5-mC) expression in OSCC clinical specimens and oral cancer cells. The DNA methyltransferase inhibitor 5-Aza-dC was used to assess the effects of DNA methylation on cell function and gene expression. RNA sequencing was used to identify key differentially expressed genes affected by 5-Aza-dC treatment. Cell migration was assessed using a wound closure assay. Protein and gene expression were analyzed using Western blotting and quantitative PCR. Results: An inverse relationship was found between 5-mC levels and cancer differentiation—poorly differentiated OSCC exhibited higher 5-mC levels. Additionally, higher 5-mC staining was observed at the invasion front of oral cancer tissues. In OSCC cells, 5-mC content correlated with migration ability. Furthermore, conditioned medium from cancer-associated fibroblasts enhanced both methylation levels and migration of OSCC cells. Treatment with 5-Aza-dC significantly increased epithelial differentiation, reduced epithelial-to-mesenchymal transition and cell adhesion-related genes, and inhibited OSCC cell migration. Conclusions: The findings highlight the critical role of DNA hypermethylation in OSCC progression, particularly in regulating differentiation, migration, and EMT. The interplay between the tumor microenvironment and epigenetic modifications underscores the complexity of OSCC biology and opens avenues for innovative therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eb0ec7659aed82eb5b415d59b960ec03d7ac79b" target='_blank'>
              DNA Hypermethylation at the Invasive Front of Oral Squamous Cell Carcinoma Confers Poorly Differentiated Characteristics and Promotes Migration of Cancer Cells
              </a>
            </td>
          <td>
            Li-Po Wang, Chien-Ya Li, Yu-Hsueh Wu, Meng-Yen Chen, Yi-Ping Hsieh, Tze-Ta Huang, T. Hong, Yuh-Ling Chen
          </td>
          <td>2025-09-27</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="


 Despite aggressive treatment, glioblastoma patients suffer from an invariably poor prognosis. Whilst substan- tial progress has been made in understanding the role of DNA methylation in brain tumour pathology, methy- lation of RNA has received comparatively little attention, despite being a key driver of cellular differentiation and neurodevelopment. N6-methyladenosine (m6A) is the most abundant RNA modiﬁcation and is, crucially, reversible. The immense intra-tumoral heterogeneity and phenotypic plasticity of glioblastoma are major barri- ers to successful treatment, and dynamic m6A methylation may mediate such behaviour, by ﬂexibly inﬂuencing transcriptional networks via the regulation of mRNA metabolism.



 Nanopore sequencing technology uniquely enables direct sequencing of RNA, and therefore detection of RNA modiﬁcations within their sequence context. We present nanopore direct RNA sequencing data from four in- house glioma-derived cell lines. We validate this approach using a novel drug inhibitor of the major m6A methyl- transferase, METTL3.



 We deﬁne transcriptome-wide RNA methylome proﬁles for four cell lines and identify common candidate path- ways which exhibit methylation. We conﬁrm robust reduction in m6A marks following treatment with a novel METTL3 inhibitor. We further compare the transcriptional landscape of wild-type tumours against these treated samples, demonstrating differential expression of biological processes linked to RNA metabolism.



 We demonstrate that nanopore direct RNA sequencing can resolve m6A modiﬁcations at single-base resolu- tion, preserving m6A stoichiometry. We validate this approach with a known inhibitor of METTL3, and further identify differentially expressed pathways in response to such inhibition. We show that glioma cell lines demon- strate extensive methylation throughout the transcriptome, and exhibit a degree of inter-tumoral heterogeneity of these m6A signatures.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ad119dfd67a68d80f7976dca916d99c54e9f801" target='_blank'>
              EXPLORING THE GLIOMA M6A RNA EPITRANSCRIPTOME WITH DIRECT RNA SEQUENCING
              </a>
            </td>
          <td>
            Simon Deacon, M. Radhi, Lauryn Walker, J. Rowlinson, S. Paine, M. Loose, Stuart Smith
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Tumors acquire a set of functional capabilities, known as the Hallmarks of Cancer, to thrive and grow. While extensive research has revealed how gene and protein expression networks contribute to these traits, the relationships between tumor cell states, spatial organization of immune and stromal cells in the tumor microenvironment, and tumor genetics, remains poorly understood. To address this, we constructed a cohort of primary tumor tissues from 117 pancreatic ductal adenocarcinoma (PDAC) patients. Using Multiplexed Ion-Beam Imaging (MIBI) with a 40-antibody panel, we mapped the spatial organization of major cell types and their phenotypes. We also conducted targeted genomic and bulk proteomic analyses to explore how spatial organization patterns align with tumor genetics and proteome profiles. We identified phenotypic subtypes of tumor epithelium that shape distinct microenvironmental niches: (i) Glycolytic tumors expressing proteins such as GLUT1, LDHA, MCT1 and HIF1α reside in hypoxic niches. They are encased by dense fibrotic stroma and lacking vasculature and immune cell infiltration. (ii) Tumor and metaplastic epithelium that exhibit an oxidative proteome profile, are highly enriched for ECAD and PD-L1 expression, and are situated near pancreatic epithelium and vasculature, with significant immune infiltration. (iii) Myeloid-interacting tumors expressing CXCL5, heavily infiltrated with neutrophils and characterized by low-collagen and high FAP positive fibroblasts. Notably, neutrophils accumulate within the lumen of tumor ducts where they exhibit increased apoptosis, DNA damage, and netosis. These highly neutrophil-infiltrated tumors, associate with poor disease outcome. The genetic landscape of PDAC influences these spatial and phenotypic patterns. Patients with TGFβ/SMAD pathway mutations, demonstrate an increased proportion of basal tumor cells accompanied by reduced lymphocytic infiltration. Similarly, distinct TP53 variants shape immune-cell infiltration: patients harboring missense mutations show increased lymphocytic infiltration compared to those with truncating mutations. These findings underscore the strong link between tumor genetics, phenotypic heterogeneity and cellular architecture of the PDAC microenvironment, offering new insights into collective tumor behavior.



 Ofer Elhanani, Guy Truzman, Martin Wartenberg, Yuval Bussi, Dana Shainshein, Bar Engelberg, Ido Harlev, Adi Lilien, Inti Zlobec, Tamar Geiger, Leeat Keren. Phenotypic, Microenvironmental, and Genetic Crosstalk Shapes Pancreatic Cancer Organization [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B123.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06cb5d6211d2e33680561c9558492cf85ff16b33" target='_blank'>
              Abstract B123: Phenotypic, Microenvironmental, and Genetic Crosstalk Shapes Pancreatic Cancer Organization
              </a>
            </td>
          <td>
            O. Elhanani, Guy Truzman, Martin Wartenberg, Yuval Bussi, Dana Shainshein, Bar Engelberg, Ido Harlev, Adi Lilien, I. Zlobec, Tamar Geiger, Leeat Keren
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) is a treatment-resistant subtype that arises through lineage plasticity, allowing tumor cells to bypass androgen receptor (AR)–targeted therapies. In a recent study, Lu et al. define a transcriptional and epigenetic hierarchy that drives this transdifferentiation. The pioneer factor FOXA2 initiates enhancer remodeling and regional DNA demethylation, while the neural lineage transcription factor NKX2-1 is required to complete the NE program. Together, these factors reorganize 3D chromatin architecture and activate lineage-specific genes through enhancer–promoter looping. Crucially, the histone acetyltransferase p300/CBP is essential cofactor in this process. Pharmacologic inhibition of p300/CBP with CCS1477 suppresses NE gene expression and impairs tumor growth in NEPC models. These findings offer a mechanistic insight of lineage plasticity and highlight p300/CBP as promising therapeutic targets. The study also raises key questions about the stability and reversibility of chromatin remodeling and sets a framework for understanding enhancer-driven plasticity in other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3331955821f5eab1873b6edce226793e7651773a" target='_blank'>
              FOXA2 and NKX2-1 Orchestrate Neuroendocrine Lineage Plasticity in Prostate Cancer
              </a>
            </td>
          <td>
            Ka-wing Fong
          </td>
          <td>2025-10-01</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Background Gastric cancer (GC) is the fourth leading cause of cancer‐related death globally. Tumor profiling has revealed actionable gene alterations that guide treatment strategies and enhance survival. Among Hispanics living in Puerto Rico (PRH), GC ranks among the top 10 causes of cancer‐related death. However, the genetic mutational landscape of GC tumors from PRH remains unexplored. This study aimed to identify the most prevalent genetic alterations in GC tumors among PRH. Methods We examined tumor mutational profiles of GC from 106 PRH between 2015 and 2022 (provided by CARIS Life Sciences and the Precision Oncology Alliance). Next‐generation sequencing data were available for 85 cases, which were categorized as hypermutated (≥ 10 mutations/megabase) or non‐hypermutated (< 10 mutations/megabase). Results Among the non‐hypermutated cases, the most frequently mutated genes were TP53 (56.9%), CDH1 (29.2%), ARID1A (27.4%), and KMT2D (25.7%). Compared to TCGA, a majority non‐Hispanic cohort, PRH had significantly higher mutational frequencies in driver genes in both intestinal type (TP53, CBLB, and MYH11) and diffuse type (CDH1, ARID1A, and KMT2D) GC. Discussion Intestinal‐type GC in PR aligns with the chromosomal instability (CIN) molecular classification, showing a higher frequency of TP53 mutations than TCGA, potentially indicating more aggressive tumor biology and poorer prognosis. Diffuse‐type GC showed higher CDH1 mutations, correlating with the genomically stable (GS) classification, characterized by fewer chromosomal changes but significant genetic alterations, including those in ARID1A and KMT2D. Conclusions This study shows that the unique genetic landscape of GC tumors in this group may lead to more aggressive cases and affect treatment responses, contributing to higher mortality rates. The higher mutation rates in biomarkers related to prognosis and therapy suggest that further research might uncover additional susceptibility variants. This underscores the importance of including Hispanics in genomic studies to better understand the genetic pathways associated with the risk and progression of GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/863fb64c7ea6f314845067da6d3c3ed255479243" target='_blank'>
              Actionable Genes and Carcinogenic Pathways for Gastric Cancer in Latinos
              </a>
            </td>
          <td>
            Ingrid M. Montes-Rodríguez, Hilmaris Centeno-Girona, Sol V. Pérez-Martir, Noridza Rivera, Marcia Cruz-Correa
          </td>
          <td>2025-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer development and response to treatment are evolutionary processes1,2, but characterizing evolutionary dynamics at a clinically meaningful scale has remained challenging3. Here we develop a new methodology called EVOFLUx, based on natural DNA methylation barcodes fluctuating over time4, that quantitatively infers evolutionary dynamics using only a bulk tumour methylation profile as input. We apply EVOFLUx to 1,976 well-characterized lymphoid cancer samples spanning a broad spectrum of diseases and show that initial tumour growth rate, malignancy age and epimutation rates vary by orders of magnitude across disease types. We measure that subclonal selection occurs only infrequently within bulk samples and detect occasional examples of multiple independent primary tumours. Clinically, we observe faster initial tumour growth in more aggressive disease subtypes, and that evolutionary histories are strong independent prognostic factors in two series of chronic lymphocytic leukaemia. Using EVOFLUx for phylogenetic analyses of aggressive Richter-transformed chronic lymphocytic leukaemia samples detected that the seed of the transformed clone existed decades before presentation. Orthogonal verification of EVOFLUx inferences is provided using additional genetic data, including long-read nanopore sequencing, and clinical variables. Collectively, we show how widely available, low-cost bulk DNA methylation data precisely measure cancer evolutionary dynamics, and provides new insights into cancer biology and clinical behaviour.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57090ca5bdab1ab7fa8186d6d31b9c968e9579a" target='_blank'>
              Fluctuating DNA methylation tracks cancer evolution at clinical scale
              </a>
            </td>
          <td>
            C. Gabbutt, M. Duran-Ferrer, Heather E. Grant, D. Mallo, F. Nadeu, J. Househam, N. Villamor, Madlen Müller, Simon Heath, E. Raineri, O. Krali, J. Nordlund, T. Zenz, I. Gut, E. Campo, A. López-Guillermo, J. Fitzgibbon, C. P. Barnes, D. Shibata, J. Martín-Subero, Trevor A. Graham
          </td>
          <td>2025-09-10</td>
          <td>Nature</td>
          <td>1</td>
          <td>95</td>
        </tr>

        <tr id="


 Genetic alterations in p53 (TP53) are found in most solid malignancies, including pancreatic cancer (PANC). Mutant p53 (p53mut) contributes to cancer progression and metastasis. Existing therapeutic options for p53mut cancers are ineffective and cause toxic side effects, stressing the need for better therapies. To address these clinical problems, we developed a two-drug therapeutic strategy that selectively targets p53mut cancers. This novel strategy combines a thymidine analogue (trifluorothymidine (TFT), a component of TAS102, acting as an inducer of DNA damage), and an inhibitor of poly (ADP) ribose polymerase (PARPi) that works as an amplifier of DNA damage. Incorporation of TFT into DNA provokes post-replicative repair generating single-strand DNA break intermediates. Repair of these intermediates is assisted by PARP, while inhibition of PARP increases more lethal double-strand DNA breaks. In normal p53 wild-type (WT) cells, TAS102 and PARPi activate p53-dependent G1/S checkpoint, limiting DNA damage and promoting repair. In p53mut cells with compromised G1/S checkpoint, TAS102 and PARPi cooperate as the inducer-amplifier pair to increase lethal DNA breaks and cell death. The TAS102-PARPi strategy was validated in preclinical studies. Our first-in-human phase I study with TAS102-talazoparib regimen (NCT04511039) showed that this regimen is safe with promising efficacy. The current work investigated the mechanisms underlying the response to TAS102-PARPi regimen in p53mut cancer cells. Our data show that TAS102-PARPi induces the G2/M checkpoint mediated by ATR kinase in a p53-independent manner. ATR acts through the G2-checkpoint kinases (CHK1 and WEE1) to inactivate CDK1 and stop transition to mitosis. The transcriptome analysis showed that TAS102-PARPi treatment increased expression of Claspin and inactivated PLK1-driven mitotic pathway. Claspin is critical for activation of DNA damage and replication checkpoints by facilitating ATR-CHK1 signaling, whereas PLK1 mediates inactivation of the G2-checkpoint kinases and degradation of Claspin for normal cell cycle progression and re-entry into mitosis. Thus, the PLK1-Claspin network may play a key regulatory role in G2-checkpoint in response to TAS102-PARPi by restraining premature reentry into cell cycle of cells with unrepaired DNA. The next major question was whether TAS102-PARPi regimen can cooperate with drugs disrupting G2-checkpoint. New data show that the efficacy of TAS102-PARPi therapy is greatly enhanced by a sequential application of G2-kinase WEE1 inhibitor. Our data show that this sequential triple-drug strategy works in a synthetic-lethality manner to cause massive cell death in p53mut cancers. The triple-drug strategy greatly improved tumor control in PANC patient-derived xenografts with no signs of major toxicities to normal tissues. In summary, this study provides the scientific foundation for our novel sequential triple-drug strategy that demonstrates greater efficacy against pancreatic cancer with lower toxicity.



 Andrei Bakin, Mohammed Alruwaili, Ashley Guo, Thomas Melendy, Barbara Foster, Christos Fountzilas. Effective targeting p53 mutant pancreatic cancer by novel drug-combination strategy leveraging their DNA damage liabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a93f85c005599dc813a5323d69a7be402a3898b5" target='_blank'>
              Abstract B001: Effective targeting p53 mutant pancreatic cancer by novel drug-combination strategy leveraging their DNA damage liabilities
              </a>
            </td>
          <td>
            A. Bakin, M. Alruwaili, Ashley Guo, T. Melendy, Barbara Foster, C. Fountzilas
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/793fefef2f944b04dcdc33b9ee878b7ce1d0682e" target='_blank'>
              DNA methylation enhances cooperative disentanglement by the Hfq nucleoid-associated protein.
              </a>
            </td>
          <td>
            Frank Wien, N. C. Jones, S. V. Hoffmann, V. Arluison
          </td>
          <td>2025-10-16</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Excised signal circles (ESC) are circular DNA molecules generated during T- and B-cell maturation. Previously considered biologically inert, recent work by Gao et al. now show that ESCs can replicate and accumulate in healthy lymphocytes. Moreover, the authors link higher levels of ESCs to an increased risk of relapse in B-cell leukemia patients and propose that this phenomenon is due to the unique ability of ESCs to induce genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a90caaf42de342ff59afcf74bb20559cdc1183c5" target='_blank'>
              Not just a by-product: circular DNA molecules derived from V(D)J recombination are linked to worse prognosis in B-cell leukemia.
              </a>
            </td>
          <td>
            Davide Pradella, Andrea Ventura
          </td>
          <td>2025-09-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chromosomal fissions and fusions are common, yet the molecular mechanisms and implications in speciation remain poorly understood. Here, we confirm a fission event in one zokor species through multiple-omics and functional analyses. We traced this event to a mutation in a splicing enhancer of the DNA repair gene Aplf in the fission-bearing species, which caused exon skipping and produced a truncated protein that disrupted DNA repair. An intronic deletion in Dna2, known to facilitate neo-telomere formation when knocked out, reduced gene activity. These variants collectively drove chromosomal fission in this zokor species. The newly formed chromosome became fixed due to carrying essential genes and strong selective pressure. While geographic isolation likely initiated the divergence of this species and the sister one, the fission event and associated decline at the chromosome level in gene flow probably exacerbated the speciation process. Our work elucidates the genetic basis of chromosomal fission and underscores its role in speciation dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/057fb07b153c9671667a486fc3641c0fea2d9b85" target='_blank'>
              Aplf/Dna2 variants drive chromosomal fission and accelerate speciation in zokors
              </a>
            </td>
          <td>
            Na Wan, Qijiao Duan, Zhenyuan Cai, Zhanwu Zhu, JingOu Wang, Yonghui Tian, Wei Shen, Bowen Li, Zhuoran Kuang, Xiaolong Liang, Sanyuan Liu, Xuan An, Xiaojie Yang, Xi Liu, L. Mao, Jiaqi Chen, Yinjia Wang, Zhilong Feng, Wenwen Liu, Yueting Bu, E. Nevo, Riccardo Papa, Axel Meyer, Jianquan Liu, Kexin Li
          </td>
          <td>2025-09-05</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73c8f319f11687277fd179aa2df80ba7f003d32d" target='_blank'>
              Loss of the Y chromosome drives cancer metabolic reprogramming.
              </a>
            </td>
          <td>
            Boopathi Subramaniyan, Bipul R. Acharya, Prince Anand, Saravana Kumar Kailasam Mani, Saswat Mohapatra, Xingyu Chen, Fangyuan Qu, Yesai S Fstkchyan, Corazon Gutierrez, Hany A Abdel-Hafiz, James C. Costello, Dan Theodorescu
          </td>
          <td>2025-08-31</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The remarkable advances in cancer therapies significantly enhance the survival rates and longevity of cancer patients. Among childhood, adolescent, and young adult female cancer survivors, however, anti-cancer agents frequently cause primary ovarian insufficiency (POI), early menopause, and infertility, primarily due to the depletion of the ovarian reserve. Oocytes, the female germ cells, exhibit a notable susceptibility to DNA damage, given that they remain in meiotic arrest of prophase I for prolonged durations from months to years, which increases the risks of accumulating DNA damage overtime. To counteract this, a tightly controlled DNA damage response (DDR) signaling ensures that only oocytes with an intact genome progress to ovulation, fertilization, and next generations. Chemotherapeutic anti-cancer agents, including doxorubicin, cisplatin, cyclophosphamide, along with irradiation, elicit DNA damage via various mechanisms, including DNA crosslinking, single and double-strand DNA breaks, and oxidative stress. The genotoxic insults activate DDR in the oocytes, which detect and repair DNA damage or initiate apoptosis to eliminate impaired oocytes. Although several protein molecules such as DNA damage-sensing kinases, checkpoint kinases, p53 family transcription factors, and pro-apoptotic molecules, have been discovered, the precise mechanisms of DDR in determining the fate of oocytes, particularly how they differ from those in somatic cells and cancer cells, remain poorly understood. From an oncofertility perspective, the current review analyzes the molecular mechanisms of anti-cancer agent-induced DDR in oocytes and discuss knowledge gaps and urgent future research directions for preserving the ovarian reserve, fertility, and endocrine functions of young female cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b3c0ab5ecbb10104dc6e55fe017fd113001dfb6" target='_blank'>
              DNA damage response signaling in oocytes from an oncofertility perspective.
              </a>
            </td>
          <td>
            Yunman (Sonia) Song, Jiyang Zhang, Yingnan Bo, Qiang Zhang, So-Youn Kim, Shuo Xiao
          </td>
          <td>2025-09-12</td>
          <td>Biology of reproduction</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Intrahepatic cholangiocarcinoma (ICC) is a highly aggressive malignancy with poor prognosis and significant molecular heterogeneity. This study investigates the role of tumor-specific enhancers in metabolic reprogramming, focusing on glucose-6-phosphate dehydrogenase (G6PD) and the pentose phosphate pathway (PPP). Using native elongating transcript-cap analysis of gene expression and single-cell RNA sequencing, tumor-specific enhancers driving G6PD overexpression were identified in ICC tumor epithelial cells. Functional assays demonstrated that G6PD promotes tumor proliferation by enhancing PPP activity and maintaining redox homeostasis, which provides NADPH to counter oxidative stress. Enhancer knockdown disrupted G6PD expression and PPP activity, increasing reactive oxygen species levels and reducing the NADPH/NADP+ ratio. These metabolic changes impaired tumor cell proliferation and sensitized ICC cells to cisplatin, emphasizing the dual therapeutic potential of targeting G6PD to inhibit tumor growth and overcome chemoresistance. Survival analyses showed that high G6PD expression correlates strongly with poor overall survival in ICC patients. While previous studies have recognized the roles of G6PD and PPP in cancer metabolism, this study uniquely links enhancer-mediated regulation to these processes in ICC, offering novel insights into epigenetic mechanisms driving metabolic reprogramming. Moreover, the findings highlight tumor-specific enhancers as critical epigenetic drivers of ICC progression, with potential as therapeutic targets. Future research should explore the integration of enhancer profiling into precision medicine frameworks and the development of novel enhancer-targeting strategies. These efforts could uncover additional metabolic vulnerabilities and provide effective treatments for this highly aggressive cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34f09ce5004e00e224dd82b9d53133eeef55290e" target='_blank'>
              Epigenetic Regulation of G6PD Drives Metabolic Reprogramming in Intrahepatic Cholangiocarcinoma.
              </a>
            </td>
          <td>
            Yusuke Nakano, Miwa Tanaka, Takeharu Sakamoto, M. Hashimoto, Taro Tobo, Hideyuki Saito, Tadashi Abe, Tomohiko Ikehara, Takashi Ofuchi, Koto Kawata, Takaaki Masuda, T. Ogino, Mamoru Uemura, H. Eguchi, Y. Doki, K. Mimori
          </td>
          <td>2025-10-07</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="ABSTRACT: Gene editing exploits endogenous DNA repair pathways to introduce precise modifications into the genome. The CRISPR system was first identified in Escherichia coli (1987) and, based on Cas protein architecture, is divided into Class I (types I, III, IV) and Class II (types II, V, VI). The widely adopted CRISPR/Cas9 system comprises a guide RNA (gRNA) and the Cas9 endonuclease, which orchestrate genome editing through a tripartite mechanism: target sequence recognition via gRNA binding to a complementary DNA site adjacent to a protospacer adjacent motif (PAM), double-stranded DNA cleavage by two nuclease domains (RuvC cleaving the non‑target strand and HNH the target strand), and repair of the resulting break by cellular pathways. This approach has been applied to in vitro fertilization-derived embryos and meiotically developing oocytes to disrupt or correct genes, offering potential for eliminating heritable diseases. However, concerns remain regarding off-target effects that introduces unwanted genetic mutations, necessitating improved specificity and ethical scrutiny.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e32b11816cdfe9dbe37c9b7f4fa9f2f1d5d3ed5" target='_blank'>
              CRISPR/CAS9-Mediated Gene Editing in Human Gametes: A Review
              </a>
            </td>
          <td>
            Esha Kumari, Neha Banu, Kathrina Marbaniang, Faridha Jane R.M. Momin, B. C. Hynniewta
          </td>
          <td>2025-09-30</td>
          <td>Biosciences Biotechnology Research Asia</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f34ee7a0d8147abc850afa171aa9019409c2cc66" target='_blank'>
              DNA methylation alterations of PITX2 and PANCR are associated with left-right asymmetry of the heart and left ventricle size
              </a>
            </td>
          <td>
            Steffan Noe, Niikanoff Christiansen, K. B. Olsen, Sara Tangmose Larsen, Pernille Heimdal Holm, M. K. Palsøe, M. Kampmann, S. B. Jacobsen, Mikkel Meyer Andersen, J. Banner, N. Morling, Jacob Tfelt-Hansen, J. D. Andersen
          </td>
          <td>2025-08-28</td>
          <td>None</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Abstract Background Chromophobe renal cell carcinoma (ChRCC) is the second most common non-clear cell renal cell carcinoma (ncRCC). A series of 496 ChRCC from Memorial Sloan Kettering Cancer Center demonstrated that sarcomatoid features, which were present in about 1.2% of the cases, are strongly correlated with metastatic disease and reduced overall survival. Little is known about the biological and immunological characteristics of the sarcomatoid variant of ChRCC and how it differs from its classic, non-sarcomatoid form. Methods Eight ChRCC tumor samples containing both classic and sarcomatoid components were selected. Xenium Prime 5K assay (5000 curated genes related to oncoimmunology) was used for high-resolution spatial transcriptomic profiling. Data were analyzed using Seurat and custom scripts to annotate cell types and conduct detailed phenotypic analysis. Pathway enrichment and transcription factor regulatory networks were analyzed using the SENIC package. Cell-cell communication was inferred using CellChat, and spatial proximity of immune and tumor cells (“cellular neighborhoods”) was analyzed, which model spatial gene expression patterns and intercellular distances. Results Spatial transcriptomic analyses revealed a significantly more immune-inflamed tumor microenvironment in sarcomatoid regions compared to their classic counterparts. We identified distinct spatial immune “neighborhoods” in sarcomatoid areas that were enriched in CD8+ T cells, polarized macrophages, and dendritic cells. These neighborhoods exhibited organized proximity to tumor cells expressing immune evasion markers such as PD-L1. In contrast, classic regions were relatively immune-deserted. Analysis of transcription factor activity showed upregulation of IRF1 and STAT1-related programs in the immune infiltrates of sarcomatoid regions, consistent with inflammatory activation. Importantly, comparison of matched sarcomatoid and classic regions within the same tumors enabled us to define a “sarcomatoid-enriched gene signature” composed of immune-related, pro-metastatic, and epithelial-to-mesenchymal transition (EMT) genes. This signature was validated in bulk RNA-seq data from The Cancer Genome Atlas (TCGA), where it correlated with poor prognosis in ChRCC patients. Conclusions This study presents the first spatial transcriptomic atlas of sarcomatoid ChRCC in comparison to matched non-sarcomatoid (classic) from the same patient. Profound immune microenvironmental differences were identified between tumor compartments. The sarcomatoid variant exhibits hallmarks of an immune-inflamed but potentially immune-evasive phenotype, including enriched immune infiltration, pro-inflammatory signaling, and EMT-associated transcriptional programs. These findings highlight the biological differences between sarcomatoid and classic ChRCC and uncover specific immune pathways and spatial cell-cell interactions that may be targeted in future immunotherapeutic strategies. Our work also establishes a foundational spatial map for ChRCC that offers new insight into the disease heterogeneity and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aba1786fb42a662d7a501d0a751ee46eaa895c9" target='_blank'>
              52Spatial transcriptomic mapping reveals immune-inflamed niches in sarcomatoid chromophobe renal cell carcinoma
              </a>
            </td>
          <td>
            Yan Tang, Tiegang Han, Hadi Mansour, M. Hirsch, Katrina Collins, Elizabeth P Henske
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract The heterogeneous nature of cell populations in human tumors is a major contributor to tumor evolution, including and perhaps most importantly in response to treatment. Here, we review current knowledge on tumor heterogeneity and cell state plasticity in small cell lung cancer (SCLC), a fast growing and highly metastatic form of lung cancer which develops rapid resistance to therapy. There is a pressing need to expand treatment options for patients with SCLC, which requires a better understanding of the mechanisms by which this disease is able to rapidly grow and evolve in response to therapy. Our current understanding points to epigenetic rather than genetic factors in defining major aspects of inter- and intra-tumoral heterogeneity in SCLC. SCLC is overall considered to be a neuroendocrine (NE) cancer type but SCLC tumors harbor a wide diversity of cancer cell states, including both NE and non-neuroendocrine (non-NE) states, defined by their mutually exclusive expression of a set of transcription factors such as ASCL1, NEUROD1, and POU2F3. The immune microenvironments of SCLC tumors also contain a great deal of heterogeneity. Here, we discuss the different SCLC cell states associated with their defining transcription factors, as well as the epigenetic mechanisms regulating the ability of SCLC cells to switch from one state to another. We further discuss how the composition of SCLC tumors and the surrounding immune cells may affect the response to chemotherapy and immunotherapy. Being able to control plasticity and heterogeneity in SCLC may in the future offer unique opportunities to improve treatment efficacy in this recalcitrant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f9c75cb099eaa87426731186cf92ba20e24f46a" target='_blank'>
              Tumor Heterogeneity and Plasticity in Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Gina Duronio, Julien Sage
          </td>
          <td>2025-10-01</td>
          <td>Lung Cancer: Targets and Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

Breast cancer is the main cause of death for women, even with major improvements
in treatment. Through processes like DNA damage, estrogen metabolism, and immunological
regulation, bacterial populations have been shown to have an impact on breast cancer development
in recent studies.



This review aimed to examine and evaluate current research on the involvement of bacteria
in breast cancer progression, with an emphasis on gene action mechanisms and potential future
treatments targeting the microbiome.



A thorough literature analysis was carried out to identify pertinent research published between
1989-2024 across various databases, including PubMed, Google Scholar, Google, and Scopus.



Bacterial dysbiosis in the gut and breast tissue contributes to the progression of breast cancer
through different pathways. Double-strand breaks in DNA are linked to various bacteria, like
Escherichia coli, Staphylococcus epidermidis, Helicobacter pylori, and Fusobacterium, which contribute
to genomic instability. Breast cancers are influenced by hormones that are influenced by gut
microbiota, namely the estrobolome, which regulates estrogen levels. Bacteria also impact immune
responses by preventing anti-tumor immunity. These results suggest that restoring microbial balance
to specific bacterial taxa may open up new treatment options. Different genes may contribute
to variations, including an increase in regulatory T (Treg) cells, while FOXP3+ T cells are linked to
shorter relapse-free survival. Understanding the microbiota's role in DNA damage, hormone regulation,
and immune modulation is important.



Bacteria contribute significantly to the development of breast cancer through genelevel
processes. Probiotics, immunomodulatory techniques, and microbiome-targeted treatments are
potential future developments that could improve therapy effectiveness and reduce resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2a52b781fcea2d4ef9f890f1a8d1daa10502bc" target='_blank'>
              The Bacterial Role in the Progression of Breast Cancer through Mechanism of Gene Action: Future Prospects with Existing Studies
              </a>
            </td>
          <td>
            Yasir Nawaz, Saba Munir, Sidra Aslam, Fizza Rimal Butt, Fouzia Tanvir, Basit Nawaz, Kazam Bashir, Rikza Tahreem
          </td>
          <td>2025-09-22</td>
          <td>Current Genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Maintaining the epigenetic identity of centromeres is essential to prevent genome instability. Centromeres are epigenetically defined by the histone H3 variant CENP-A. Prior work in human centromeres has shown that CENP-A is associated with regions of hypomethylated DNA located within large arrays of hypermethylated repeats, but the functional importance of these DNA methylation (DNAme) patterns remains poorly understood. To address this, we developed tools to perturb centromeric DNAme, revealing that it causally influences CENP-A positioning. We show that rapid loss of methylation results in increased binding of centromeric proteins and alterations in centromere architecture, leading to aneuploidy and reduced cell viability. We also demonstrate that gradual centromeric DNA demethylation prompts a process of cellular adaptation. Altogether, we find that DNAme causally influences CENP-A localization and centromere function, offering mechanistic insights into pathological alterations of centromeric DNAme.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94f6767c83b49c5f93547ae2e51f853756e7c466" target='_blank'>
              DNA methylation influences human centromere positioning and function
              </a>
            </td>
          <td>
            Catalina Salinas-Luypaert, Danilo Dubocanin, Rosa Jooyoung Lee, Lorena Andrade Ruiz, Riccardo Gamba, Marine Grison, Léonid Velikovsky, Annapaola Angrisani, Andrea Scelfo, Yuan Xu, Marie Dumont, Viviana Barra, Therese Wilhelm, Guillaume Velasco, Marialucrezia Losito, René Wardenaar, C. Francastel, F. Foijer, G. Kops, K. Miga, Nicolas Altemose, D. Fachinetti
          </td>
          <td>2025-09-04</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>63</td>
        </tr>

        <tr id="Menin protein is encoded by the multiple endocrine neoplasia type 1 gene, which typically serves an oncogenic role in endocrine organs. However, in mice, menin protein is a key mediator of leukemic transformation, particularly in acute myeloid leukemia (AML), and is involved in the disease process through epigenetic regulatory mechanisms. This functional paradox may be due to the unique gene expression regulatory properties of menin, which can both activate and inhibit the expression of target genes. At the molecular level, menin protein regulates the gene transcription process by interacting with multiple protein complexes and forms a complex network with multiple signaling pathways. As the core hub of transcriptional regulation, menin protein is essential for the maintenance of cellular homeostasis and its aberrant function can lead to gene expression disorders, which contributes to the development of AML. Despite the druggability challenges of several transcriptionally regulated proteins, inhibitors of menin protein have made breakthroughs in clinical development. Particularly in AML subtypes [for example, lysine methyltransferase 2A (KMT2A) rearrangement or nucleophosmin (NPM1) mutation], menin protein inhibitors have demonstrated favorable efficacy. The present study systematically reviewed biological functions of the menin protein and its application in targeted therapy for specific AML subtypes. Notably, menin inhibitors have demonstrated potential in KMT2A/NPM1-mutant leukemia, however, the off-target effects, resistance mechanisms and a lack of biomarkers due to the extensive nature of their binding interface remains to be elucidated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18b23d4e7fbebe4140893833cf2070b2e3e6ebf" target='_blank'>
              Targeting menin in lysine methyltransferase 2A/nucleophosmin-mutated leukemia: A novel strategy from epigenetic dysregulation to clinical therapy (Review)
              </a>
            </td>
          <td>
            Junjie Bi, Hongjun Zhou
          </td>
          <td>2025-09-10</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Rolling circle amplification (RCA) has emerged as a highly versatile and robust isothermal amplification technology, offering exceptional sensitivity, specificity, and scalability for next-generation molecular diagnostics and multi-omics research. Its ability to generate long, repetitive DNA sequences with high fidelity has made it a pivotal tool in disease diagnostics, genomic analysis, and spatial transcriptome profiling. Recent advancements have expanded RCA into various formats, including solution-phase, solid-phase, hydrogel-based, and digital RCA, enhancing its analytical performance and adaptability across diverse biological applications. The integration of RCA with CRISPR-based detection, biosensors, and nanoparticle-assisted signal amplification has significantly improved its diagnostic potential, enabling ultrasensitive detection of nucleic acids, proteins, extracellular vesicles, and single cells. In addition to diagnostics, RCA has profoundly impacted genomic research, particularly in extrachromosomal circular DNA sequencing, offering insights into genome instability and cancer evolution. In spatial transcriptomics, RCA-based in situ RNA sequencing and hybridization have enabled high-resolution gene expression mapping in tissue architecture, providing invaluable insights into cellular heterogeneity and disease microenvironments. This review comprehensively explores the fundamental principles, design modifications, and advanced signal amplification strategies of RCA, with a focus on its applications in molecular diagnostics, genome analysis, and spatial transcriptome profiling. Additionally, it addresses key challenges, including reaction specificity, automation, and cost-effectiveness, and discusses future directions for integrating RCA into high-throughput molecular diagnostics, biomarker discovery, and single-cell and spatial multi-omics platforms. Through continuous innovation, RCA is poised to play a transformative role in precision medicine, next-generation sequencing, and biomedical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/576501d76c4b29ed935e06956faf76592e2b944c" target='_blank'>
              Rolling circle amplification for next-generation molecular diagnostics, genome analysis, and spatial transcriptome profiling.
              </a>
            </td>
          <td>
            Jueun Han, Yejin Song, Jihee Lee, Khulan Nyamzaya, Eunjung Kim
          </td>
          <td>2025-09-08</td>
          <td>Nanoscale</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Cancer remains a leading cause of global morbidity and mortality, underscoring the need for innovative diagnostic and prognostic biomarkers. PIWI-interacting RNAs (piRNAs) have emerged as critical regulators of gene expression and genomic stability, with increasing evidence linking their dysregulation to oncogenesis. piRNAs can modulate DNA methylation through interactions with DNA methyltransferases (DNMTs), contributing to tumor suppressor gene silencing and broader epigenetic reprogramming. However, the functional implications of piRNA-mediated methylation changes in cancer are not fully elucidated. This study investigates the circulating levels of piR-823 and piR-651, their associations with DNMT3B expression and global DNA methylation, and their correlations with clinical and molecular markers in colorectal, breast, and prostate cancers. Materials and methods A total of 300 participants, including 150 patients with histologically confirmed cancers and 150 age- and sex-matched healthy controls, were enrolled. Plasma piRNA levels were measured via qRT-PCR, while global 5-methylcytosine (5mC) and DNMT3B concentrations were quantified using ELISA. Immunohistochemistry was performed on resected tumor tissues to assess DNMT3B, Ki-67, p53, E-cadherin, vimentin, HER2, and androgen receptor (AR). Correlations were analyzed using Pearson’s test and multivariate regression models. Diagnostic performance was evaluated via Receiver Operating Characteristic (ROC) curves. Results piR-823 was significantly upregulated in colorectal and breast cancers (p < 0.001), while piR-651 was markedly downregulated in prostate cancer (p = 0.002). Cancer patients exhibited elevated levels of DNMT3B and 5mC (both p < 0.001), which correlated strongly with Ki-67 and p53 expression (r > 0.7, p < 0.001). E-cadherin expression was inversely correlated with DNMT3B and piR-823, consistent with a role in epithelial–mesenchymal transition (EMT). cfDNA levels were also significantly elevated in cancer cases (p < 0.001), suggesting potential utility as a non-invasive biomarker. Conclusion The observed associations between piRNA dysregulation, epigenetic markers, and tumor aggressiveness indicators support the potential of circulating piRNAs—particularly piR-823 and piR-651—as novel biomarkers for cancer detection and progression monitoring. These findings warrant further mechanistic studies to validate their functional roles in tumor epigenetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec55cb867b96ef67ad7cc822f9de2fa5e24bf5e8" target='_blank'>
              Epigenetic functions and biomarker potential of PIWI-interacting RNAs in oncogenesis
              </a>
            </td>
          <td>
            Kldiashvili Ekaterina, Abiatari Ivane, Kekelia Elene, Iordanishvili Saba, Metreveli Tornike, Dumbadze Eter
          </td>
          <td>2025-09-30</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="for 273 TCGA colorecatal cancer subjects in a general linear model fitting a negative binomial distribution. TCGA annotations from the Covariables Univariate analysis The general linear model compared covariates This research first identifies that CXCL10 is significantly downregulated in Black colorectal cancers in two independent datatsets. We also observe that immune related pathways and cells are significantly different between the self identified Black and White Americans from The Cancer Genome Atlas. We use a combination of RT-qPCR, bulk RNA seq anal to show that CXCL10 expression is higher in the MMRdeficient CRC subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/652519820bbd179c6d3708acf1e63564e6b14588" target='_blank'>
              The Effect of Mismatch Repair - Deficient/ Microsatellite Instability on Black Minority Colorectal Cancer Gene Expression
              </a>
            </td>
          <td>
            Dimitri F. Joseph, Joseph F. LaComb, Andrew Fu, Ricardo E. Flores, Ellen Li, Alexandra Guillaume, Bin Chen
          </td>
          <td>2025-10-01</td>
          <td>Spartan Medical Research Journal</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Ovarian cancer (OC) remains one of the deadliest gynecological malignancies. Immune checkpoint blockade (ICB) inhibitors efficacy in OC has been minimal, highlighting the need for a deeper understanding of the immune microenvironment in OC. Recent studies suggest that DNA methylation and transcription factors may influence the response to immunotherapy. This study aims to classify ovarian cancer into distinct immune subtypes by integrating DNA methylation and transcription factor data through comprehensive bioinformatics analysis. Using data from The Cancer Genome Atlas (TCGA), we identified twelve differentially methylated genes (DMGs) associated with transcription factors and categorized OC into two immune subtypes, C1 and C2.The C1 subtype exhibited higher levels of immune infiltration and better prognosis, characteristic of immune "hot" tumors, whereas the C2 subtype was associated with lower immune infiltration and poorer prognosis, indicative of immune "cold" tumors. A prognostic prediction model based on four key genes—KRT81, PAPPA2, FGF10, and FMO2—was developed using the least absolute shrinkage and selection operator (LASSO) and Cox regression analyses. This model effectively stratified the TCGA OC cohort into high- and low-risk groups and was validated by predicting patient survival outcomes. Additionally, drug sensitivity analysis revealed potential therapeutic targets for different risk groups, offering new avenues for precision treatment in ovarian cancer. Immunohistochemical tests confirmed the potential of KRT81 as a prognostic marker for ovarian cancer. Our findings enhance the understanding of the molecular characteristics of the OC immune microenvironment, propose novel biomarkers for prognosis, which may potentially improve the prognosis of OC. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02630-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01d9c66278ce901978efba119b98a7523cb64b30" target='_blank'>
              DNA methylation and transcription factor-driven immune subtypes in ovarian cancer
              </a>
            </td>
          <td>
            Jingshu Hu, Mu Su, Zhijun Qin, Jiayu Li, Hengyu Wang, Kexin Chang, Guosheng He, Yan Zhang, Xiuwei Chen
          </td>
          <td>2025-08-28</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43797c7834d20e5589904d0168347625fb0cad6e" target='_blank'>
              Unraveling cellular dynamic changes in tumor evolution induced by long-term low dose-rate radiation.
              </a>
            </td>
          <td>
            Wanshi Li, Weiwei Pei, Yiwei Wang, Jing Nie, Guang Hu, Yongsheng Zhang, Zhifei Cao, Zaozao Chen, Keyan Miao, Yuqi Chen, Likai Shi, Yingchu Dai, , Hailong Pei, C. Ye, Yasser F Ali, Wentao Hu, Wenying Yan, Guangming Zhou
          </td>
          <td>2025-09-05</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Extracellular vesicles (EVs) are promising biomarkers for cancer diagnosis and prognosis due to their ability to carry specific biomolecular cargo, including DNA. However, the clinical utility of DNA methylation-based liquid biopsies using EV-DNA remains underexplored. The low quantity and relatively long length of EV-DNA complicate whole-genome methylation profiling. To address this, we develop Tn5-assisted Enzymatic Methyl-sequencing with Post-conversion Tailing (TEMPT), a bisulfite-free whole-genome profiling method for EV-DNA. TEMPT employs single-adapter Tn5 tagmentation, enzymatic conversion of unmodified cytosines, and post-conversion tailing to generate high-depth whole-genome EV-DNA methylomes. We apply TEMPT to EV-DNA from 58 gastric cancer and polyp samples, generating methylomes from sub-nanogram inputs and identifying differentially methylated regions (DMRs) that distinguish cancer from controls. We identify potential cancer biomarkers through DMR-associated genes, highlighting the roles of EVs in cellular communication. Our findings suggest that immune cells may serve as an alternative source of EV-DNA. This approach holds significant promise for advancing EV-DNA research and its applications in early disease diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad67a59c8fc16bd26c5fe5c57db4a838137c7af1" target='_blank'>
              Whole-genome methylation profiling of extracellular vesicle DNA in gastric cancer identifies intercellular communication features
              </a>
            </td>
          <td>
            Bingqian Lin, Zhenna Jiao, Shouquan Dong, Weikai Yan, Jinting Jiang, Yanfang Du, Xiaocheng Weng, Hongling Wang, Zhiyuan Hu, Yibin Liu, Xiang Zhou
          </td>
          <td>2025-08-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cervical cancer remains the fourth most common cancer among women globally, disproportionately impacting low- and middle-income countries despite the existence of HPV vaccines. While DNA methylation has been studied extensively as a biomarker, other epigenetic mechanisms remain underexplored. This scoping review aims to report such underexplored epigenetic biomarkers linked to cervical cancer, shifting the focus beyond global nuclear DNA methylation. Literature searches were performed using Google Scholar via Publish or Perish software including studies published until January 2025. Our review focused on mitochondrial DNA, non-coding RNA, histone modifications, and repetitive elements. Mitochondrial DNA methylation has been proposed as a cervical cancer biomarker, although supporting evidence is limited. Histone modifications are more consistently reported to be involved both in cervical cancer onset and aggressiveness. Similarly, aberrant expression of lncRNAs, circRNAs, miRNAs, and piRNAs has been associated with poor prognosis. Finally, hypomethylation in repetitive elements such as LINE-1 and Alu is often observed in cervical cancer, contributing to genomic instability and tumorigenesis. Highlighting these alternative epigenetic mechanisms, our review emphasizes the importance of expanding biomarker discovery beyond the traditional nuclear DNA methylation. Understanding these mechanisms may improve early detection and personalized disease management strategies for cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a7819cfc0864d5095515e01e909ecdbee12a80f" target='_blank'>
              Epigenetic Biomarkers for Cervical Cancer Progression: A Scoping Review
              </a>
            </td>
          <td>
            Efthymios Ladoukakis, Gracia Andriamiadana, Fatema Hajizadah, Lewis G. E. James, Belinda Nedjai
          </td>
          <td>2025-09-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The t(11;22) translocation encodes the EWS::FLI1 fusion oncoprotein which is the primary driver of Ewing sarcoma. EWS::FLI1 creates unique, de novo pathogenic enhancers that drive gene expression and are a central mechanism of oncogenesis. Which chromatin regulatory proteins are critical to this mechanism is understudied. Here, we perform a comparative analysis of the function of the chromatin complexes MLL3/4 and p300/CBP in EWS::FLI1-mediated gene regulation. Using EWS::FLI1 degradation models, we define a subset of EWS::FLI1-sensitive enhancers whose activity correlates with p300/CBP function. We perturb both chromatin complexes to establish that in contrast to MLL3/4, p300/CBP is a critical regulator of EWS::FLI1-driven enhancer activity and downstream gene expression. We also show that p300/CBP small-molecule inhibition decelerates tumor growth in vivo. Our work highlights the context-dependent nature of chromatin protein activity at oncogenic enhancers and reveals p300/CBP as an important regulator of Ewing sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eb04dfc69e03ddcb4aa348ab8eef9592fa55873" target='_blank'>
              p300/CBP is an essential driver of pathogenic enhancer activity and gene expression in Ewing sarcoma
              </a>
            </td>
          <td>
            Laura C Godfrey, Brandon Regalado, Sydney R Schweber, Charlie Hatton, Daniela V. Wenge, Yanhe Wen, Meaghan Boileau, Maria Wessels, Jun Qi, Christopher J Ott, K. Stegmaier, Miguel N Rivera, Scott A Armstrong
          </td>
          <td>2025-09-01</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 CDKN2A/B homozygous deletion is among the most common genetic alterations in glioblastoma (GBM), often accompanied by losses of neighbouring tumour suppressors such as MTAP and PTPRD. These co-deletions create vulnerabilities that may be exploited for synthetic lethality-based therapies. While recent network analysis has demonstrated the potential of combination therapy to target such synthetic lethal nodes, the underlying patterns of co-deletion and their downstream molecular effects remain unclear. We conducted a multi-omics analysis to systematically assess these co-deletion patterns and their functional consequences in CDKN2A/B-deleted GBM.



 Copy number variation (CNV), whole exome sequencing (WES), mRNA expression, and reverse-phase protein array (RPPA) data from TCGA-GBM (n=63) and CPTAC (n=96) were analysed. Patients with signiﬁcant CDKN2A/B losses were identiﬁed using GISTIC2.0 and assessed for co-occurring alterations in MTAP, PTPRD, and EGFR and broader 9p21 losses using Fisher’s Exact test. Downstream mRNA and protein expression changes were evaluated with Mann-Whitney U tests.



 CDKN2A/B homozygous deletion was strongly associated with MTAP loss (TCGA-GBM: 74.6%; CPTAC: 83.0%), moderate PTPRD loss (TCGA-GBM: 55.6%; CPTAC: 30.8%), and moderate EGFR ampliﬁcation (TCGA-GBM: 55.6%; CPTAC: 60.4%). Complete 9p21 loss was common (TCGA-GBM: 54.9%; CPTAC: 35.4%) but deletion proportions varied within and across cohorts. CDKN2A/B-deleted patients had signiﬁcant reductions in MTAP and PTPRD mRNA and protein expression (p < 0.05) and increases in EGFR mRNA and protein expression (p < 0.05) com- pared to non-deleted patients.



 CDKN2A/B deletion is associated with broader 9p21 loss and functional changes in key synthetic lethal nodes, supporting its potential as a therapeutic biomarker. These results justify the development of synthetic lethality- based clinical trials targeting GBM with 9p21 loss. Future work will focus on functional validation of drug combinations and clinical trial design.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbcdf237ba9e5b742a8d73e7613b54470bb9dac1" target='_blank'>
              UNDERSTANDING PATTERNS OF CO-DELETION CDKN2A/B DELETED GBM TO SUPPORT A SYNTHETIC LETHALITY APPROACH TO TREATMENT
              </a>
            </td>
          <td>
            Tanishi Moitra, Michael Castro, Matt Williams
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35f4ca5c9c5c95098c56b14c0e5f31b3116e40f9" target='_blank'>
              Phage-encoded TelN inhibits bacterial Mre11-Rad50 nuclease to protect hairpin telomeres.
              </a>
            </td>
          <td>
            Maya Houmel, Nicolas Pellaton, Anna Anchimiuk, Stephan Gruber
          </td>
          <td>2025-10-15</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20b4e2232775eb0b9347348ac65a428b9b495195" target='_blank'>
              Quantitative Analysis of DNA Double-Strand Breaks in Genomic DNA Using Standard Curve Method.
              </a>
            </td>
          <td>
            Lihuang Guo, Hanying Dai, Jiancheng Li, Chenwei Li, Yue Huang, Keqian Xu
          </td>
          <td>2025-10-18</td>
          <td>Journal of clinical laboratory analysis</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary DNA repair mechanisms are crucial for maintanence of DNA and cell integrity. When disrupted, they can lead to increased mutation rate and contribute to the cancerogenesis. On the other hand, cancer cells that bear mutations in genes involved in DNA repair are more prone to DNA damaging insults which opens opportunities for treatments involving synthetic lethality. microRNAs are short non-coding RNAs that regulate expression by binding to gene transcripts and inducing mRNA degradation or inhibition of translation. Here, we review the miRNA-mediated dysregulation of genes involved in DNA damage response (DDR) and DNA repair pathways in ovarian cancer (OvCa), one of the deadliest female malignancy. We also discuss miRNAs, which affect response to OvCa therapy by regulating PARP1 (Poly(ADP-Ribose) Polymerase-1), a central gene of the OvCa synthetic lethality treatments. Finally, we address the limitations of miRNAs as diagnostic biomarkers and potential targets for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd4db6c3d00cf07b80c8f608dd0b112d9dfce41" target='_blank'>
              DNA Damage and Repair in Ovarian Cancer: Focus on MicroRNAs
              </a>
            </td>
          <td>
            Katarzyna D. Arczewska, A. Piekiełko-Witkowska
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Although intragenic CpG dinucleotides are conserved during evolution, they are also sensitive to methylation-dependent mechanisms. Methylation status of the TP53 introns 1, 3 and 4 have been analysed in stage III ovarian carcinoma (OC). In the present study, the methylation of exon 4 (10 CpG pairs) was analysed in advanced-stage OC to investigate TP53 methylation and compare exon and intron 4 methylation patterns. A total of 80 samples from patients with advanced-stage OC and metastatic lesions were examined, along with 80 samples derived from healthy patients who had never been diagnosed with cancer. Methylation analysis of the human A2780 ovarian cancer cell line was also performed. Exon and intron 4 were methylated in OC, corresponding metastases and paired healthy tissue. The DNA from the human A2780 ovarian cancer cell line and the normal samples from healthy subjects was also methylated. The data indicate the existence of an intragenic mechanism of regulation of TP53 activity that involves demethylation/methylation processes. This mechanism provides the ability to alter the response from cell cycle arrest to apoptosis by manipulating only the expression of long or short p53 isoforms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba65eba61b813922b785bd71652e055bf783b2a" target='_blank'>
              Exon 4 and intron 4 TP53 are both methylated in advanced-stage ovarian carcinomas
              </a>
            </td>
          <td>
            Wiktor Szewczuk, Oksana Szewczuk, Krzysztof Czajkowski, R. Gromadka, Maciej Waledziak, Marek Semczuk, Andrzej Semczuk
          </td>
          <td>2025-09-12</td>
          <td>Biomedical Reports</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Long non-coding RNAs (lncRNAs) play pivotal roles in diverse cellular processes ranging from gene regulation and chromatin remodeling to RNA stability and epigenetic modifications. Despite the identification of approximately 95,000 lncRNA genes in humans, our understanding of their structure–function relationships remains very limited. This review examines the current state of lncRNA structure determination. We briefly discuss the advantages and limitations of experimental approaches—including chemical probing methods such as SHAPE and DMS—as well as the challenges inherent to computational predictions, particularly given RNA's dynamic nature, structural heterogeneity, and the energy degeneracy of its building blocks. The review also highlights the difficulties in predicting long-range interactions, including pseudoknots, which are essential for global folding of large RNAs, and discusses how elevated, nonphysiologically Mg2⁺ concentrations used in many experiments can distort our perception of native RNA conformations. Recent advances in cryo-electron microscopy and atomic force microscopy, coupled with machine learning algorithms, offer promising strategies to capture the realistic conformational landscapes of RNAs, including lncRNAs, under near-physiological conditions. These advances have the potential to redefine our understanding of lncRNA architectures, their structural dynamics, and how they influence cellular functions, ultimately informing future directions of lncRNA research and opening new frontiers such as structure-based drug discovery and therapeutic interventions targeting lncRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9baa3ff44d8909bf0ba277649815b691743c6ced" target='_blank'>
              Challenges and opportunities in technologies and methods for lncRNA structure determination
              </a>
            </td>
          <td>
            M. F. de Souza Degenhardt, Yun-Xing Wang
          </td>
          <td>2025-10-02</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background/Objectives: Galectin-3 (Gal-3), encoded by LGALS3, is a β-galactoside-binding lectin involved in diverse tumor-associated processes, including immune modulation, cell cycle regulation, and stress adaptation. Despite its known roles in cancer biology, the full extent of its molecular functions and prognostic relevance across tumor types remains incompletely understood. This study aimed to systematically investigate the transcriptomic impact of LGALS3 deletion and assess its clinical significance in cancer. Methods: We analyzed CRISPR-Cas9 knockout transcriptomic data from the SigCom LINCS database to characterize the consensus gene signature associated with LGALS3 loss using functional enrichment analyses. Pan-cancer survival analyses were conducted using TIMER2.0. Differential Gal-3 protein levels in ductal adenocarcinoma and normal pancreatic tissues were evaluated using the Human Protein Atlas. Finally, functional analyses were performed in pancreatic ductal adenocarcinoma (PDAC). Results: LGALS3 deletion across multiple cancer cell lines led to transcriptomic changes involving mitotic progression, stress responses, and axonal guidance pathways. High LGALS3 expression was significantly associated with worse overall survival in lower-grade glioma, PDAC, uveal melanoma, and kidney renal papillary cell carcinoma. LGALS3 knockout in YAPC cells recapitulated the pan-cancer findings, linking LGALS3 to cell morphogenesis and proliferation. Conclusions: These findings identify Galectin-3 as a key regulator of oncogenic programs and a potential prognostic biomarker in PDAC and other malignancies, with implications for therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0894155b476f9fae4b8588df639cdd5d50fcb2b" target='_blank'>
              Transcriptomic and Clinical Profiling Reveals LGALS3 as a Prognostic Oncogene in Pancreatic Cancer
              </a>
            </td>
          <td>
            Grazia Scuderi, S. Mijatović, D. Maksimović‐Ivanić, M. Di Rosa, J. Muñóz-Valle, Alexis Missael Vizcaíno-Quirarte, Gian Marco Leone, K. Mangano, P. Fagone, Ferdinando Nicoletti
          </td>
          <td>2025-10-03</td>
          <td>Genes</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Detection of cancer at early stage can significantly improve the five-year survival rate of patients. Bisulfite-based methylation detection can cause DNA damage, especially in high GC content regions which was associated with the development of cancers. Loss of aberrant methylated CpG sites in cfDNA will lead to the undetectability of certain circulating tumor DNA (ctDNA), consequently may affect the cancer detection. Our study uses enzymatic method to detect whole genome abnormal methylation regions in esophageal squamous cell carcinoma (ESCC). We also provide a pretrained neural network, hybrid of BERT and CNN (BCNN), to identify ctDNA robustly. Maximum posterior probability is utilized to estimate the fraction of ESCC-derived ctDNA in plasma for predicting the risk of ESCC cancer. Our results analysis indicated that enzyme-based whole-genome methylation sequencing retained more longer cfDNA and detected more CpG sites than bisulfite-based method in both gDNA and cfDNA. Enrichment analysis of differentially methylated regions (DMR) showed that top five pathways were associated with ESCC. Compared to traditional models, our BCNN demonstrates the best performance in identifying ctDNA (AUC = 0.970). By estimating the fraction of plasma ctDNA, our BCNN exhibits high accuracy in ESCC detection even at ultralow sequencing depths (AUC = 0.946). Specifically, in the validation cohort, when the specificity is 93.75%, 7 out of 8 early-stage ESCC (TNM Stage I) were identified as positive in our preliminary results. In conclusion, our preliminary results reinforce the idea of employing BCNN as a novel strategy for ESCC early detection in clinical practice. However, to be applied in clinical, further validation with a larger sample size is necessary. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18278-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5af110454f8624475f32847f585947aa544c81e" target='_blank'>
              Improved circulating tumor DNA identification for detection of esophageal squamous cell carcinoma by enzymatic methyl sequencing and hybrid neural network
              </a>
            </td>
          <td>
            Jiangfeng Shen, Yuqi Ren, Ziyong Mao, Qiuxiang Hu, Yilong Wan, Jiangcun Du, Yanting Lai, Shengxiang Shao, Liuqing Zhang, Hao Wu, Jiaxi Li, Sheng Ju, Xin Tong, Jun Zhao, Lei Cao, Deyi Xiong, ChengCheng Xu, Jun-Chi Xu, Dong Jiang
          </td>
          <td>2025-09-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) can be effectively treated inhibiting the disease-causing BCR::ABL1 kinase by tyrosine kinase inhibitors (TKIs). Although therapy is initially tremendously successful, resistance may occur in up to 25% of CML patients. Besides aberrations in the BCR::ABL1 kinase domain, a variety of resistance mechanisms are currently discussed, among them epigenetic reprogramming. The histone-modifying enzyme lysine methyltransferase 2D (KMT2D/MLL2) belongs to the most frequently mutated genes in cancer and is also known for its association with hereditary Kabuki syndrome. However, its role in CML is widely unknown. In the present study, we analyzed the role of the KMT2D p. (Arg191Trp) variant in imatinib-resistant CML, which was recurrently acquired in imatinib resistance in vitro. SiRNA-mediated KMT2D knockdown, but also introduction of the p. (Arg191Trp) variant into treatment-naïve K-562 cells led to impaired imatinib susceptibility visible by increased cell numbers, proliferation rates and metabolic activities under imatinib exposure (p < 0.001). The effect of KMT2D p. (Arg191Trp) could be overcome by inhibiting histone demethylation with the demethylase inhibitor LSD1. In addition, rescue of KMT2D expression in imatinib-resistant cells reinstated the response to imatinib treatment. Furthermore, gene expression analysis revealed upregulation of CCNE2 in cells harboring KMT2D p. (Arg191Trp) potentially explaining increase in cell proliferation under imatinib exposure. Overall, our findings demonstrate that the loss of the tumor suppressor KMT2D promotes TKI resistance in CML. Thus, KMT2D status could serve as an additional biomarker for TKI resistance, while restoration of its expression might be a therapeutic option to overcome this resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59b4b4f7562cd8617674ef60daedf2504f6b6932" target='_blank'>
              The role of the lysine histone methylase KMT2D in chronic myeloid leukemia
              </a>
            </td>
          <td>
            Lina Schlemminger, Inga Nagel, I. Vater, Ingolf Cascorbi, Meike Kaehler
          </td>
          <td>2025-09-16</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="BACKGROUND
Colorectal cancer (CRC), a predominant cancer type globally, is one of the most common cancers worldwide. Immhibune-checkpoint initors (ICIs) have robust efficacy in the treatment of patients with metastatic DNA mismatch repair (dMMR) CRC, however, some of the patients among them demonstrate resistance to immunotherapy, the underlying molecular mechanisms remain elusive.


METHODS
Using single-cell sequencing data, we identified and annotated cell populations, systematically comparing cell proportions and gene expression changes across different groups. Gene set enrichment analysis (GSEA) was conducted to identify significantly enriched signaling pathways in T cell subpopulations. Furthermore, we assessed the impact of transcriptional dysregulation in TSTR and γδ_T cells on tumor immunotherapy responses and drug resistance. The potential role of T cell subset interactions in modulating drug resistance and sensitivity was investigated through comprehensive cell communication analysis. Lastly, clinical samples from 190 CRC patients were collected, with paired adjacent normal tissues. Immunohistochemistry (IHC) was performed using antibodies against FOS and KLRB1.


RESULTS
T cells were identified as the predominant cell population influencing immunotherapy outcomes. T cell subsets exhibited distinct functional characteristics, with notable differences in their distribution between resistant and sensitive tumor groups. Specifically, exhausted T cells (Tex), GZMK + T, TSTR, regulatory T cells (Treg), and γδ_T cells were associated with therapeutic resistance. For Tex and GZMK + T cells, resistance was correlated with activation of antigen processing and presentation pathways, whereas oxidative stress pathways were downregulated. In contrast, γδ_T cells in the sensitive group exhibited activation of protein folding pathways, which may contribute to anti-tumor immune responses. Transcriptional network dysregulation in TSTR and γδ_T cells was observed in the drug-resistant group. Cell-to-cell communication analysis showed stronger interactions among T cell subpopulations, with changes in key signaling pathways linked to treatment resistance. Additionally, downregulation of the CD69-KLRB1 signaling pathway was identified as a potential mechanism of drug resistance in CRC. Lastly, high expression of FOS was significantly associated with a worse prognosis, whereas high expression of KLRB1 predicted improved clinical outcomes, including prolonged overall survival and progression-free survival, and emerged as an independent prognostic factor for CRC patients.


CONCLUSION
This study highlights the pivotal role of T cell subsets in patients with metastatic dMMR CRC who resistance to anti-PD-1 therapy, revealing that transcriptional dysregulation and impaired cell communication networks are central mechanisms underlying drug resistance. Notably, KLRB1 has been identified as a promising biomarker for immunotherapy response in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3475e66480ee040f7a2e535d7ffc47aa52446ec" target='_blank'>
              Single-cell transcriptome analysis reveals regulatory programs associated with tumor resistance during immunotherapy in colorectal cancer.
              </a>
            </td>
          <td>
            Yan Chen, Tao Liu, Guangtao Min, Cong Wang, Dali Xi, Lei Jiang
          </td>
          <td>2025-09-10</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Meiosis is a source of genetic variation in eukaryotes. Meiosis in the eukaryotic fission yeast Schizosaccharomyces pombe leads to the formation of spores that are particularly resistant to environmental stresses. In addition to external factors, internal processes may nevertheless contribute to cellular stress and impact the genome. This study investigates the role of Pnu1 as the major meiotic nuclease in S. pombe. Transcription and cellular expression of Pnu1 are regulated upon specific phases of meiosis while its mitochondrial localization is also altered during this process. As a result, Pnu1 induces fragmentation of both genomic and mitochondrial DNA in the post meiotic phase. This sugar-non-specific endonuclease generates random double-strand breaks across the genome, an activity that appears to be mediated by direct interaction with chromatin. Given the high spore viability (∼95%) and the widespread occurrence of this phenomenon, this fragmentation appears to be physiological rather than apoptotic as observed in mammals. EndoG is the mammalian homolog of Pnu1 and is a caspases-independent apoptotic endonuclease that can allow cell survival. This study further describes the dynamics of Pnu1 action and support the conclusion that Pnu1 is a major meiotic endonuclease of S. pombe responsible for a transient post-meiotic fragmentation of cellular DNA potentially contributing to genetic variability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fdc3ebfc992761b3e2bf6cac186bc029393e2fe" target='_blank'>
              Induction of post-meiotic DNA double-strand breaks by the Pnu1 endonuclease in Schizosaccharomyces pombe.
              </a>
            </td>
          <td>
            Loïs Mourrain, Tiphanie Cavé, G. Boissonneault
          </td>
          <td>2025-09-03</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Supercoiled (Sc) circular DNA, such as plasmids, are essential in molecular biology and hold strong therapeutic potential. However, they are typically produced in Escherichia coli, resulting in bacterial methylations, unnecessary sequences, and contaminants that hinder certain applications including clinical uses. These limitations could be avoided by synthesizing plasmids entirely in vitro, but synthesizing high-purity Sc circular DNA biochemically remains a significant technical challenge. To overcome this challenge, we have developed two novel biochemical methods for in vitro synthesis of Sc circular DNA. Linear DNA with two loxP sites in the same orientation is generated by polymerase chain reaction or rolling circle amplification. Cre recombinase efficiently converts the linear DNA into relaxed circular DNA. T5 exonuclease is then used to digest unwanted linear DNA, and topoisomerases are employed to generate Sc circular DNA. Using this approach, we synthesized EGFP-FL, a 2 kb mini-circular DNA encoding essential EGFP expression elements. EGFP-FL transfected HeLa and C2C12 cells with significantly higher efficiency than its E. coli-derived counterpart. These methods enable the efficient production of Sc circular DNA from 196 bp to several kb, and in quantities from micrograms to milligrams, providing a versatile, scalable, and bacteria-free platform for basic research and therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc6c6aa3cf66d79a56fe61e937482c31c5bc93bd" target='_blank'>
              Synthesizing supercoiled circular DNA molecules in vitro
              </a>
            </td>
          <td>
            Sepideh Rezaei, Monica Moncada-Restrepo, Sophia Leng, Jeremy W. Chambers, Fenfei Leng
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="This study examines the contribution of dysregulated long non-coding RNAs (lncRNAs) to disease progression and prognosis in breast invasive carcinoma (BRCA) and cervical squamous cell carcinoma with endocervical adenocarcinoma (CESC) using data from The Cancer Genome Atlas (TCGA). Comparative analyses identified both shared and cancer-specific lncRNAs linked to tumorigenesis. Survival analysis using the Kaplan–Meier method highlighted several candidates with prognostic relevance. Notably, FAM83H-AS1 showed increased expression in advanced stages of BRCA and was linked with lymph node involvement and metastatic status. Its elevated expression was associated with poor survival outcomes in both BRCA and CESC cohorts. In contrast, lncRNAs such as FGF14-AS2, which were downregulated, demonstrated links to tumor-suppressive mechanisms and improved survival in BRCA patients. Subcellular localization analysis indicated predominant cytoplasmic presence of FAM83H-AS1, implying possible roles in post-transcriptional regulation. To support our in-silico observations, we carried out qRT-PCR experiments in breast cancer cell lines with varying degrees of aggressiveness, which confirmed the differential expression patterns of these lncRNAs. Overall, our findings highlight the clinical importance of lncRNA dysregulation in gynaecological cancers and point to their potential utility as biomarkers for prognosis and targeted therapy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-16436-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96a627efdc32c519561de180942228807a9ccfd0" target='_blank'>
              Functional characterization of dysregulated LncRNAs in BRCA and CESC highlights their role in cancer prognosis
              </a>
            </td>
          <td>
            Madhur Sharma, Nidhi Chourasia, Priyanka Priyanka, D. P. Sarkar, A. Nag, Sandeep Saxena
          </td>
          <td>2025-10-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd59fd638b49487acab5f65a27e1f99a668332d" target='_blank'>
              Mechanistic insights into MET exon 14 skipping mutations and their role in tumor progression.
              </a>
            </td>
          <td>
            Promita Ghosh, Isabella Pecora, Morag Park
          </td>
          <td>2025-09-09</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Breast cancer is a complex disease influenced not only by genes but also by the way cancer cells generate and use energy. One rare change is the loss of a gene called fumarate hydratase (FH), which normally helps cells produce energy efficiently. We examine how this change influences how tumors grow and respond to treatment. By studying thousands of breast cancer samples, we found that tumors missing FH tend to create a special environment that helps them grow and survive, especially by encouraging new blood vessel formation. We also describe a patient with this FH change who had a remarkable and long-lasting response to a therapy that blocks blood vessel growth (known as anti-VEGF treatment). These findings suggest that identifying this rare energy-related alteration may help clinicians determine which patients are most likely to benefit from this therapy. Such insights could contribute to advancing more personalized approaches to breast cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c214b499120e8d0fc3e3396e45d4d3eb6e6c018" target='_blank'>
              Revealing the Angiogenic Signature of FH-Deficient Breast Cancer: Genomic Profiling and Clinical Implications
              </a>
            </td>
          <td>
            L. A. Sinberger, Noa Keren-Khadmy, Assaf Goldberg, T. Peretz-Yablonski, A. Sonnenblick, Mali Salmon-Divon
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The initiation and progression of tumors are closely linked to aberrant regulation of multiple genes. Among these genes, sister chromatid cohesion acetyltransferase 2 (ESCO2) has emerged as an important target in cancer research. ESCO2 exhibits complex and heterogeneous expression patterns across various types of malignancies and tumor progression stages. Extensive studies have shown that ESCO2 primarily functions in chromosomal cohesion, significantly contributing to genomic stability maintenance. Dysregulated expression of ESCO2 in tumor cells is often correlated with malignant behaviors. Given its essential role in tumorigenesis, ESCO2 has increasingly been explored as a promising therapeutic target in recent years. This review article summarizes recent research progress regarding ESCO2 in malignant tumors, aiming to identify a novel target for their treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s40001-025-03164-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b1b540beced4302a863f075307c9fa3cb6930a" target='_blank'>
              The expression of establishment of cohesion 1 homolog 2 (ESCO2) in tumor cells and its research progress as a therapeutic target
              </a>
            </td>
          <td>
            Chunmei Lu, Hanchi Wu, Huiyu Wang
          </td>
          <td>2025-10-06</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="SUMMARY
The lack of targetable mutations in 50% of NSCLC cases and resistance to therapies underscore the need for alternative treatments, with epigenetic strategies potentially regulating tumor suppressor genes to inhibit growth. Our focus is to understand the STAT1-mediated epigenetic and epitranscriptomic modulations, such as R-loop formation, histone methylation/demethylation, histone acetylation and deacetylation, DNA methylation, and m6A RNA methylation, in T helper cell (TH) differentiation to evoke tumor-protective immune responses in non-small cell lung cancer (NSCLC) by knocking out (KO) and overexpressing (OE) STAT1. Peripheral blood mononuclear cells (PBMCs) were isolated from NSCLC patients and healthy controls using Ficoll-Hypaque density-gradient centrifugation. CD4+ T cells were purified with magnetic activated cell sorting (MACS). Subsequently, CRISPR/Cas9 knock-out and overexpression techniques were applied, followed by qRT-PCR to evaluate 5-mC, m6A RNA methylation, gene expression, and transcription factor enrichment. Mutations in STAT1 can cause severe immunodeficiency and malignancy, linked to genomic instability from R-loops-DNA-RNA hybrids that lead to DNA breaks through transcription-coupled nucleotide excision repair. Our study examines epigenetic regulators in CD4+ T helper cells from NSCLC patients, focusing on the effects of STAT1 depletion and overexpression on R-loop formation at key gene loci specific to T helper cells. Depletion of STAT1 increased R-loop frequencies, DNA methylation, histone deacetylation, and histone methylation, whereas its overexpression decreased them. Abnormal epitranscriptomic alterations, including m6A RNA methylation, were observed, indicating that STAT1 is crucial for T helper cell differentiation and immune responses in NSCLC, presenting promising avenues for targeted therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/deb51d643431a8993cf2a5b52181db30125d9533" target='_blank'>
              Epigenetic and Epitranscriptomic Modulation by STAT1: Unraveling T Helper Cell Differentiation in NSCLC.
              </a>
            </td>
          <td>
            Roshni Bibi, Melvin George, Koustav Sarkar
          </td>
          <td>2025-10-14</td>
          <td>Journal of immunotherapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Due to the persistently high global incidence and mortality rates of cancer, developing novel therapeutic strategies is imperative. The replication factor C (RFC) family, a critical subset of DNA replication and repair, serves multifaceted roles in tumor progression. Despite its widely recognized importance, the pleiotropic mechanisms of the RFC family lack systematic illustration, particularly regarding each member specific contributions to cancer hallmarks. In the present review, mRNA expression of each RFC family member in pan-cancer was profiled and the associations between their expression levels and tumor types evaluated. In addition, the effect of RFC expression on patients' survival across malignancies is assessed. Furthermore, the present review summarized current research on RFC family members in various malignancies with particular emphasis on the RFC-like complexes, highlighting key findings and advancements in understanding their role in tumor biology. The signaling pathways associated with RFC family members are discussed and the molecular mechanisms elucidated. Finally, the clinical importance of RFC family members including prognosis, potential inhibitors and combination treatments are also discussed. The present review aimed to provide innovative perspectives for developing combinatorial molecular targeted therapies in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a41b28d0169fef1334d7ef3750ca63ec9bbc80ab" target='_blank'>
              Multiple roles of replication factor C family in pan-cancer (Review)
              </a>
            </td>
          <td>
            Dan Luo, Bo Li, Xueping Jiang
          </td>
          <td>2025-09-25</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Small cell carcinoma of the esophagus (SCCE) is a rare and highly aggressive malignancy with limited therapeutic options and poor prognosis. The paucity of clinical specimens and lack of established experimental models have hindered a comprehensive understanding of its cellular heterogeneity and tumor microenvironment.We performed single-cell RNA sequencing on SCCE samples, and integrated them with publicly available scRNA-seq datasets from esophageal squamous cell carcinoma (ESCC), esophageal adenocarcinoma (EAC), and adjacent normal tissues (NT) from ESCC and EAC cases. An integrative transcriptomic analysis was conducted to identify cell types, infer malignant states, reconstruct differentiation trajectories, evaluate immune landscapes, and investigate fibroblast subtypes and cell–cell communication networks.SCCE tumors were characterized by a predominance of malignant epithelial cells and exhibited a profoundly immunosuppressed phenotype, with reduced immune infiltration and widespread downregulation of immune checkpoint genes. Malignant epithelial cells showed pronounced chromosomal instability and were classified into three transcriptionally distinct subtypes with divergent differentiation trajectories. The tumor microenvironment featured a complex stromal compartment, with enrichment of extracellular matrix fibroblasts (eCAFs) characterized by elevated ELF3 regulatory activity, and collagen-driven signaling predominantly mediated by inflammatory CAFs (iCAFs). SCCE also showed the most intricate cell–cell communication network among esophageal cancer subtypes.Our single-cell atlas offers a detailed view of the cellular heterogeneity and microenvironmental complexity of SCCE, highlighting its distinct tumor architecture, immune exclusion, and stromal reprogramming. These findings provide a valuable resource for understanding SCCE biology and form a basis for future mechanistic and exploratory biological investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6db8bbf656956c2fda0265e682ad202bf970eea8" target='_blank'>
              Single-cell transcriptomic analysis reveals epithelial and microenvironmental heterogeneity in small cell carcinoma of the esophagus
              </a>
            </td>
          <td>
            Xiaolei Yin, Xiaopeng Li, Lili Mi, Jiaojiao Hou, Fei Yin
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6922da00f94f2dd9a584ee514f3fa6786a061103" target='_blank'>
              Genetic landscape of DNMT3A R882H reveals mechanism of aberrant oligomerization.
              </a>
            </td>
          <td>
            Emma M. Garcia, Olivia R Lavidor, Shaunak Raval, Nicholas Z. Lue, Jessica K Liang, Maeve McFadden, Derek D Hwang, Marcanthony U Zepeda, Luke W Chen, Steven A. Carr, Malvina Papanastasiou, Brian B Liau
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Small intestinal neuroendocrine tumors (SI-NETs) represent the most frequent neoplasms of the small intestine, with loss of chromosome 18 as the most common genetic aberration. To date, no highly frequently mutated genes have been identified on chromosome 18 or elsewhere. This study aimed to explore the potential role of methyl-CpG binding domain protein 2 (MBD2), located at 18q21.2, as a tumor suppressor gene in SI-NETs. By immunohistochemistry, we found undetectable or very low levels of MBD2 expression in 64% of the analyzed SI-NETs, and overall, very low levels of mRNA and protein expression were observed. Overexpression of MBD2 in the SI-NET cell line, GOT1, reduced cell growth and induced apoptosis, though cell migration remained unaffected. In addition, MBD2 expression decreased cell proliferation and migration capacity of the neuroendocrine cells NCI-H727 but showed no effect on apoptosis. Furthermore, MBD2 expression in both cell lines was found to upregulate E-cadherin but downregulate the levels of N-cadherin and vimentin in GOT1 and NCI-H72, respectively, as determined by western blot analysis. These results suggest that MBD2 plays a role in inhibiting the epithelial-mesenchymal transition process and may function as a potential tumor suppressor gene in SI-NETs. Future studies to elucidate the underlying mechanisms are warranted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14eeef15fb15c0cced59de17e27889264700227a" target='_blank'>
              Loss of MBD2 is associated with tumor growth in small intestinal neuroendocrine tumors
              </a>
            </td>
          <td>
            Elham Barazeghi, S. Backman, Per Hellman, O. Norlén, P. Stålberg
          </td>
          <td>2025-09-01</td>
          <td>Endocrine Connections</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Rad51 is a pivotal DNA repair protein that plays a central role in homologous recombination repair. Its significance in tumorigenesis and cancer treatment resistance has garnered increasing attention. In recent years, studies have shown that Rad51 not only plays an important role in the biological characteristics of tumor cells but is also closely related to tumor immunity. The complexity of the tumor immune microenvironment makes the action mechanism of Rad51 more complex and affects the immune system's ability to recognize and clear tumors by influencing the interaction between tumor cells and immune cells. Although preliminary studies have suggested a potential role of Rad51 in tumor immunity, the underlying molecular mechanisms require further elucidation. Furthermore, the research on Rad51 as a therapeutic target is also continuously deepening, indicating its application potential in tumor treatment. In summary, this review aims to comprehensively analyze the role of Rad51 in tumor immunity and its challenges and opportunities in future tumor treatment research and clinical application in order to provide new ideas for clinical tumor immunotherapy. However, the current understanding of Rad51's immunomodulatory functions remains incomplete, and further preclinical and clinical studies are needed to validate its therapeutic potential, address potential off‐target effects, and elucidate tumor‐type‐specific mechanisms and microenvironmental interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00a9c3b142aaabf70111312c22aa00050983254" target='_blank'>
              Rad51 at the Crossroads: Bridging DNA Repair, Tumorigenesis, and Immune Regulation
              </a>
            </td>
          <td>
            Zhendong Qin, Mingjun Lu, Jingwei Guo, Chenyang Wang, Teng Ma
          </td>
          <td>2025-09-24</td>
          <td>Med Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Background Response to immune checkpoint inhibition (ICI) in sarcomas is overall low and heterogeneous. Understanding determinants of ICI outcomes may improve efficacy and patient selection. Thus, we investigated whether the expression of transposable elements (TEs), which are epigenetically silenced and can stimulate antitumor immunity, influence ICI outcomes and immune infiltrates in common sarcoma subtypes. Methods We used transcriptomic data to assign immune enhanced versus immune depleted status to 67 pretreatment and on-treatment biopsies of sarcomas from patients treated on ICI trials, along with additional cohorts from The Cancer Genome Atlas (TCGA) and an independent ICI trial (SARC028). A machine learning technique (lasso-penalized logistic regression) controlled for sarcoma subtype was used to determine if TE and epigenetic regulatory gene expression predict immune infiltrates. Correlations between top features in these models and sarcoma immune infiltrates, immune pathway expression, and clinical outcomes were explored. Results Expression of TEs and epigenetic regulators significantly predicted immune enhanced status. TE subfamilies and Ikaros family zinc finger 1 (IKZF1), a chromatin-modulating transcription factor, were significantly contributory. TE and IKZF1 expression positively correlated with tumor immune infiltrates, inflammatory pathways, and improved clinical outcomes, and increased in tumors that gained immune infiltrates during ICI treatment. TE and IKZF1 expression similarly correlated with overall survival and immune features in a TCGA cohort. In an additional cohort of patients with sarcoma treated with ICI, IKZF1 expression correlated with progression-free survival and inflammatory features. Conclusions TE and IKZF1 expression warrant further translational investigation as potential biomarkers of tumor immune infiltrates and outcomes following ICI treatment, and as therapeutic targets in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ab0102d110f6ab5f1b56088579ee9c32e847174" target='_blank'>
              High transposable element expression in sarcomas is associated with increased immune infiltrates and improved outcomes including after immunotherapy
              </a>
            </td>
          <td>
            B. Nacev, M. Bradic, H. J. Woo, A. L. Richards, C. Kelly, M. Dickson, M. Gounder, M. Keohan, P. Chi, S. Movva, Robert G. Maki, E. Slotkin, E. Rosenbaum, V. Avutu, J. Chan, L. B. Banks, T. Adamson, S. Singer, Cristina R Antonescu, W. Tap, M. T. Donoghue, S. D'Angelo
          </td>
          <td>2025-09-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Gemykibivirus, an emerging single-stranded DNA (ssDNA) virus of the recently established genus in the family of Genomoviridae, had been discovered in human blood and cerebrospinal fluid and a variety of other body fluids. However, the molecular mechanisms of gemykibivirus entrance into the host cells and its pathogenicity remain poorly understood. To investigate the host response of gemykibivirus, we used an infectious clone of gemykibivirus previously established through molecular biology techniques to rescue virus in HEK293T cells and analyzed the changes in the host transcriptome during the infection period by RNA-Seq. Our findings indicate that gemykibivirus can both express viral proteins and accomplish replication, and high-throughput transcriptome analysis identified a total 1732 significantly different genes. Functional enrichment analysis of Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways for differentially expressed genes (DEGs) showed gemykibivirus involving several important pathways, including MAPK signaling pathway, Chemical carcinogenesis-reactive oxygen species and Oxidative phosphorylation. Interestingly, mitochondrial DNA-encoded mRNAs exhibited varying levels of upregulation, suggesting that gemykibivirus may be involved in mitochondrial fission and the regulation of mitochondrial function. Subsequently, a series of experiments proved that gemykibivirus can lead an increase in mitochondrial DNA copy number, promote the release of mtDNA into the cytoplasm, enhance reactive oxygen species production and trigger other cellular antiviral responses. Overall, we lay a foundation for revealing the relationship between Gemykibivirus and human diseases through mitochondrial functional alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67fc3d43e01f79ea8442e27ecafc7aa6d559e12" target='_blank'>
              Transcriptome Analysis Reveals Gemykibivirus Infection Induces Mitochondrial DNA Release in HEK293T Cells
              </a>
            </td>
          <td>
            Runbo Yang, Hao Yan, Yifan Wang, Wenqing Yang, Jianru Qin
          </td>
          <td>2025-09-30</td>
          <td>Viruses</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aim: ERG overexpression driven by gene fusions is well-characterized event in prostate cancer, yet its impact on gene expression profiles and regulatory networks remains unclear. This study aims to identify most frequently mutated 20 genes in prostate adenocarcinoma, compare their expression levels in ERG+ and ERG- tumors, and construct gene correlation networks to uncover ERG-dependent molecular interactions.
Materials and Methods: RNA-seq and somatic mutation data from the TCGA-PRAD cohort were used. Most frequently mutated 20 genes were identified. Patients were grouped by ERG expression status. Differential expression analysis and gene co-expression network construction were performed for each subgroup.
Results: TP53, SPOP, FOXA1, KMT2D, KDM6A, and MUC17 showed higher expression in ERG+ tumors, while TTN, LRP1B, PTEN, and RYR2 were more expressed in ERG- tumors. Network analysis revealed distinct regulatory patterns between subgroups.
Conclusion: ERG status significantly shapes transcriptional and regulatory landscape of prostate cancer, offering potential targets for ERG-specific therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/715787fdd68a02367b7887119163c38e134df38d" target='_blank'>
              Integrative analysis of somatic mutations, gene expression, and co-expression networks in ERG-stratified prostate adenocarcinoma
              </a>
            </td>
          <td>
            Perçin Pazarcı
          </td>
          <td>2025-08-30</td>
          <td>Adıyaman Üniversitesi Sağlık Bilimleri Dergisi</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The heterogeneity and dynamic evolution of the tumor microenvironment (TME) are key drivers of tumor progression, immune evasion, and therapeutic resistance. Traditional transcriptomic approaches are limited by their reliance on population-averaged data and the lack of spatial context, making it challenging to fully dissect the complex interactions among tumor cells, immune infiltrates, and stromal components within the TME. In recent years, the synergistic advancement of single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (ST) has revolutionized our ability to construct high-resolution cellular atlases and delineate spatially organized functional modules within the TME. These technologies have enabled the identification of novel cell subtypes, cell-cell communication pathways, and spatial niches associated with disease progression and therapy response. This review systematically compiles recent technological breakthroughs in the integration of single-cell and spatial multi-omics, highlights key discoveries in TME research, and discusses their transformative impact on the development of precision oncology. Additionally, it addresses current technical limitations and explores emerging strategies and future directions for advancing this rapidly evolving field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90a0c36ae523e1a4fc23b5c8bc8013f692b5b483" target='_blank'>
              Refining Tumor Microenvironment Insights via Single-Cell and Spatial Transcriptomics for Precision Medicine
              </a>
            </td>
          <td>
            Xianbing Wang
          </td>
          <td>2025-09-05</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Neuroblastoma is the most common extracranial pediatric tumor and accounts for about 15% of childhood cancer deaths. It arises from precursors of the peripheral sympathetic nervous system. GD2 is a disialoganglioside that is highly expressed on the surface of neuroblastoma cells, but has limited expression in healthy tissues, making the GD2 antigen a strong candidate for targeted cancer therapy. Current integration of anti-GD2 immunotherapy has significantly improved patient outcomes, but nearly half of these patients will eventually relapse, in part due to resistance developed from antigen escape. Here, we show that GD2 expression in GD2-low neuroblastoma cells can be increased through genetic knockout or pharmacological inhibition of several proteins involved in the epigenetic regulation of transcription, including EZH2, DOT1L, and Menin. Furthermore, the combination of either knockout or inhibition of EZH2 and Menin together increased GD2 expression to a greater extent than single drug treatment. Integrated transcriptomics and epigenomics identified that combined EZH2 and Menin inhibition induced a neuronal differentiation of cells through regulation of bivalent promoters, which are characterized by the presence of both H3K27me3 and H3K4me3 histone marks. This combination increased the transcription of the ganglioside biosynthetic enzyme ST8SIA1 (GD3 synthase), which is regulated by a bivalent promoter, rendering cells sensitive to anti-GD2 therapy. Combined treatment with the EZH2 inhibitor tazemetostat and Menin inhibitor revumenib in an orthotopic neuroblastoma tumor model also increased GD2 expression more than each single agent. Increased GD2 was sustained for 30 days after discontinuing the drugs. This suggests that the combination of epigenetic therapy may be a promising mechanism to restore sensitivity to anti-GD2 immunotherapy in low GD2 expressing tumors. Our early findings identify targeting epigenetic regulation as a mechanism of increasing GD2 expression, thus providing a therapeutic strategy for overcoming resistance to current immunotherapy regimens.



 Silvi Salhotra, Nathaniel W Mabe, Susu Zhang, Gabriela Alexe, Giulia Digiovanni, Kathleen Engel, Daniel Schaefer, Guillermo N Dalton, Melinda Soeung, Robbie G Majzner, Kimberly Stegmaier. EZH2 and Menin inhibition induce neuronal differentiation and GD2 expression in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f6f63be187afaa581a6ae0634fe458d6dd2647c" target='_blank'>
              Abstract B027: EZH2 and Menin inhibition induce neuronal differentiation and GD2 expression in neuroblastoma
              </a>
            </td>
          <td>
            S. Salhotra, Nathaniel W. Mabe, Susu Zhang, G. Alexe, G. Digiovanni, Kathleen Engel, Daniel A. Schaefer, Guillermo N Dalton, Melinda Soeung, R. Majzner, K. Stegmaier
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b661e811f3dcdd8c38e021520a91d9010537f40" target='_blank'>
              Altered growth in KMT2A/ENL induced human B cells from peripheral blood
              </a>
            </td>
          <td>
            Siti Zuleha Idris, Vivek Prasad Jayamoney, A. Mohd Amin, Sethu Thakanachy Subha, Mei I Lai, Fazlina Nordin, Maha Abdullah
          </td>
          <td>2025-09-26</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hematopoietic stem cells (HSCs) sustain lifelong hematopoiesis through their capacity for self-renewal and multilineage differentiation. However, the isolation and functional characterization of HSCs remain challenging due to their cellular heterogeneity and dynamically regulated transcriptional and epigenetic landscapes. Advances in experimental and computational biology, including single-cell RNA sequencing (scRNA-seq), chromatin immunoprecipitation sequencing (ChIP-seq), network inference algorithms, and machine learning, have improved our ability to resolve transcriptional states, trace lineage trajectories, and reconstruct gene regulatory networks (GRN) at single-cell resolution. These approaches enable the discovery of novel HSC subtypes and regulatory factors, and facilitate the integration of multi-omics data to uncover epigenetic and transcriptional mechanisms that drive stem cell fate decisions. Additionally, machine learning models trained on high-throughput datasets provide predictive power for identifying novel enhancers, transcription factors, and therapeutic targets. This review underscores the synergistic role of computational tools in deciphering HSC biology and highlights their potential to improve stem cell therapies and precision treatments for hematologic disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d3c26c697c565c33ced832e7a44009a59ac4768" target='_blank'>
              Unlocking hematopoietic stem cell potential: integrative computational approaches for genomic and transcriptomic analysis
              </a>
            </td>
          <td>
            P. Raghav, Basudha Banerjee, Rajni Chadha
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="CircRNAs are a class of covalently closed circular non-coding RNAs formed through back-splicing of precursor mRNAs, lacking 5′ caps and 3′ poly(A) tails, and are widely present in eukaryotic cells. Advances in high-throughput sequencing have revealed that circRNAs regulate gene expression and participate in disease pathogenesis, often exhibiting dual oncogenic or tumor-suppressive roles in cancer. CircZFR primarily comprises two subtypes—hsa_circ_0072088 and hsa_circ_0000345—which are derived from exons of the ZFR gene located on chromosome 5q13.3. CircZFR plays a critical regulatory role in the progression of multiple malignancies. This review systematically summarizes recent research progress on circZFR across different cancer types, elucidating its molecular mechanisms in influencing tumor proliferation, invasion, metastasis, and drug resistance through acting as a miRNA sponge, modulating signaling pathways, or directly binding to proteins. Furthermore, this review integrates the interaction networks between circZFR and key regulatory factors, unveiling its multifaceted functions in remodeling the tumor microenvironment. This review emphasizes the aberrant expression patterns of circZFR and their clinical prognostic relevance in multiple cancers, including colorectal, pancreatic, thyroid, bladder, hepatocellular, breast, lung, and cervical carcinomas, discusses its potential as a novel diagnostic biomarker or therapeutic target, and aims to provide a theoretical basis for precision cancer therapy while highlighting its translational research prospects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1140548cfdf98c75b2d77d75aabe3d01573b631f" target='_blank'>
              CircZFR functions in cancer from molecular networks to precision therapy
              </a>
            </td>
          <td>
            Jinniang Nan, Min Li, Xinru Li, Yuqing Guo, Ziye Cai, Zekang Deng, Chengyi Zhan, Jiacheng Li, Shuying Liu, Ziyi Chen, Zhenjun Huang, Yihan Yang
          </td>
          <td>2025-10-14</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is highly heterogeneous, with five-year survival rates dropping from $\sim$90% in localized disease to $\sim$15% with distant metastases. Disease progression is shaped not only by tumor-intrinsic alterations but also by the reorganization of the tumor microenvironment (TME). Metabolic, compositional, and spatial changes contribute to this progression, but considered individually they lack context and often fail as therapeutic targets. Understanding their coordination could reveal processes to alter the disease course. Here, we combined multiplexed ion beam imaging (MIBI) with machine learning to profile metabolic, functional and spatial states of 522 colorectal lesions with single-cell resolution. We observed recurrent stage-specific remodeling marked by a lymphoid-to-myeloid shift, stromal-cancer cooperation, and malignant metabolic shifts. Spatial organization of epithelial, stromal, and immune compartments provided stronger stratification of disease stage than tumor-intrinsic changes or bulk immune infiltration alone. To systematically model these coordinated changes, we condensed multimodal features into 10 latent factors of TME organization. These factors tracked disease progression, were conserved across cohorts, and revealed frequent multicellular metabolic niches and distinct, non-exclusive TME trajectories. Our framework MuVIcell exposes the elements that together drive CRC progression by grouping co-occurring changes across cell types and feature classes into coordinated multicellular programs. This creates a rational basis to therapeutically target TME reorganization. Importantly, the framework is scalable and flexible, offering a resource for studying multicellular organization in other solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/884e760c0de1e86aa10844e17508931cd572069b" target='_blank'>
              Robust multicellular programs dissect the complex tumor microenvironment and track disease progression in colorectal adenocarcinomas
              </a>
            </td>
          <td>
            Loan Vulliard, Teresa Glauner, Sven Truxa, Miray Cetin, Yu-Le Wu, R. Simon, Laura Behm, J. Tanevski, Julio Saez-Rodriguez, G. Sauter, Felix J. Hartmann
          </td>
          <td>2025-10-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Metazoans suppress transposons via small RNA–mediated silencing, but ciliates physically eliminate transposons from somatic genomes. Here, we report the high-quality germline genome assembly of a marine ciliate Euplotes vannus, where ~80% are transposons and other germline-specific DNA that has to be precisely eliminated during soma development. We demonstrate that a class of soma-derived 30-nucleotide small RNAs precisely targets nontransposon DNA against elimination during this process. Small RNA–mediated targeting remains functional across varying small RNA lengths and is compatible with heterozygous sites. These small RNAs are cleaved by Dicer-like ribonuclease from long noncoding RNAs, which are bidirectionally transcribed of somatic chromosomes, initiated at subtelomeric chromosome breakage sequences (5′-TTGAA-3′). On the basis of these findings and time-course transcriptomic profiling, we propose a model elucidating the molecular mechanism of DNA elimination in Euplotes. These findings provide insights into the role of small RNAs in transmitting genetic information across generations and maintaining genome stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42400a7ce7bd33d02826f168bc703d3c0d0af547" target='_blank'>
              Soma-derived 30-nt small RNAs are coupled with chromosome breakage and precisely target nontransposon DNA against elimination in Euplotes vannus
              </a>
            </td>
          <td>
            Li-Yang Lyu, Bijia Ding, Jinyu Fu, I. Rzeszutek, E. Swart, M. Nowacki, Feng Gao
          </td>
          <td>2025-10-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="In most solid tumors, hypoxia constitutes a defining microenvironmental feature that reprograms malignant cells into a highly metastatic state by driving cellular plasticity and exacerbating chromosomal instability (CIN). However, the mechanisms by which cancer cells concurrently co-opt these elements of hypoxic adaptation to promote metastasis remains poorly understood. Here, we report that hypoxia promotes metastasis by suppressing the JmjC-containing histone lysine demethylase Kdm8. CRISPR/Cas9-mediated targeting of Kdm8 in a Kras;Trp53-driven mouse model of pancreatic ductal adenocarcinoma (PDA) robustly rewires the malignant cell transcriptomic programs, leading to a profound loss of the epithelial morphology and widespread metastatic disease. In PDA patients, a high KDM8-induced gene signature is associated with reduced metastatic burden and better survival in advanced disease. Notably, Kdm8 suppression in normoxia recapitulates key aspects of the global epigenetic and transcriptomic rewiring, mitotic spindle defects, and CIN induced by hypoxia. Moreover, disruption of Kdm8’s demethylase activity phenocopies Kdm8 loss, whereas expression of hypermorphic Kdm8 variants resistant to hypoxic suppression markedly reduces metastasis beyond the levels achieved by the wildtype protein. Through the suppression of Kdm8 demethylase function, hypoxia unleashes a potent metastatic program by simultaneously advancing cellular plasticity and CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ead3df1c0ef44aa86f0839ae810b8a1fead329" target='_blank'>
              Hypoxia-induced metastatic heterogeneity in pancreatic cancer
              </a>
            </td>
          <td>
            Pradeep Moon Gunasekaran, Qianqian Wang, Yoke-Chen Chang, Polina Guseva, Rajika Chauhan, Alexander Kley, Gene Lee, Siddharth Ghosh Roy, Yousef Masoudpoor, Arthur Roberts, Kelly Watkins Walton, Lucyann Franciosa, Shafiq Bhat, Emmanuel Zachariah, Kishan Patel, Zhongren Zhou, Wenjin Chen, J. Ni, S. Gu, Cristina Montagna, Shin-Heng Chiou
          </td>
          <td>2025-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Colorectal cancer (CRC) is the third most diagnosed cancer worldwide and the leading cause of cancer-related mortality. Despite advancements in detection and therapy, current diagnostic and treatment strategies are primarily based on evidence derived from limited cohorts, which are then applied to a diverse and genetically heterogeneous global population, often resulting in suboptimal outcomes. This one-size-fits-all model overlooks individual variability in genetic, environmental, and lifestyle factors, which have a significant impact on disease progression, therapeutic response, and overall prognosis. Disparities in incidence, progression, and treatment outcomes may arise from population-specific variations in gene expression. We investigated gene expression profiles of tumor and adjacent non-tumor tissues from people in Ibadan, Nigeria, West Africa, using transcriptomics analysis of bulk RNA sequencing data. Differentially expressed genes (DEGs) were defined by a log2 fold change (log2FC) ≥ 1.0 for upregulated genes and ≤ -1.0 for downregulated genes, with a false discovery rate (FDR) adjusted p-value < 0.05 using the Benjamini–Hochberg correction. We compared DEGs to publicly available CRC bulk RNA sequencing datasets retrieved from the NCBI-GEO database and The Cancer Genome Atlas (TCGA) database. We identified 148 upregulated and 1,769 downregulated DEGs in CRC tumor samples from the Ibadan cohort. Comparative analysis revealed substantial variation in DEG profiles across populations, with many DEGs being unique to specific datasets and no single DEG consistently shared among all datasets. Notably, individuals of African ancestry exhibited distinct DEG signatures across datasets. Among these, the tumor-associated calcium signal transducer 2 (TACSTD2) gene was the only DEG consistently upregulated across all datasets, highlighting its potential relevance in CRC among individuals of African ancestry.



 Seidu Adams, Aiden Amin, Rachel Bonacci, Udekwu Chinedum, Oluwabusola Ajagbe, Maya Wilson, Dimitri Joseph Brian Thiesen, Ellen Li, David Irabor, Akinfemi Akingboye, Olorunseun O. Ogunwobi. Tumor-associated calcium signal transducer 2 is consistently upregulated in colorectal cancer in people of African ancestry [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr A030.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/556a649592f5adc67c16f721ff85fb13fe69edcc" target='_blank'>
              Abstract A030: Tumor-associated calcium signal transducer 2 is consistently upregulated in colorectal cancer in people of African ancestry
              </a>
            </td>
          <td>
            Seidu Adams, Aiden Amin, Rachel E. Bonacci, Udekwu Chinedum, Oluwabusola Ajagbe, Maya Wilson, Dimitri F. Joseph, Brian Thiesen, Ellen Li, David Irabor, A. Akingboye, O. Ogunwobi
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0c307663da680bc20dfe3893c95285de2664ca" target='_blank'>
              Spatiotemporal analyses of the pan-cancer single-cell landscape reveal widespread profibrotic ecotypes associated with tumor immunity.
              </a>
            </td>
          <td>
            Ya Han, Lele Zhang, Dongqing Sun, Guang-xu Cao, Yuting Wang, Jiali Yue, Junjie Hu, Zhonghua Dong, Fang Li, Taiwen Li, Peng Zhang, Qiu Wu, Chenfei Wang
          </td>
          <td>2025-08-25</td>
          <td>Nature cancer</td>
          <td>2</td>
          <td>12</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly heterogeneous cancer with complex tumor–immune interactions. This heterogeneity, particularly in tumor and immune cells, complicates treatment and prognostic evaluation. Although recent studies have revealed distinct tumor cell states and immune dysfunction in HCC, the molecular basis underlying tumor aggressiveness remains poorly understood. A deeper understanding of the molecular and functional diversity of both tumor and immune cell populations, especially the identification of stem-like tumor subpopulations and immunosuppressive mechanisms, along with the development of robust prognostic biomarkers, is essential for advancing precision oncology and improving clinical outcomes.We integrated three publicly available single-cell RNA sequencing (scRNA-seq) datasets from GEO to delineate the cellular architecture of the HCC tumor microenvironment. Unsupervised clustering and dimensionality reduction were employed to identify major cell types and tumor subpopulations. Functional annotation was performed using canonical markers, Monocle, CytoTRACE, and AUCell scoring. H2AFZ was identified as a candidate oncogene and validated through in vitro knockdown experiments. The interaction between T cell subsets and tumor subpopulations were further characterized. A prognostic risk model was constructed using LASSO regression.Six major cell types were identified in HCC TME. Tumor cells were subdivided into three distinct clusters: Tumor_C0, Tumor_C1 and Tumor_C2. Tumor_C2 showed the highest stemness, pro-metastatic activity and immunogenic cell death signatures. H2AFZ was highly expressed in Tumor_C2 and associated with poor prognosis. The knockdown of H2AFZ reduced H2A.Z protein levels, inhibited proliferation, invasion, and induced apoptosis. T cell analysis revealed five subpopulations. It was found that Tumor_C2 interacts with the proliferative and exhausted T cell subpopulations, suggesting a potential functional relationship between them. The prognostic model based on tumor_C2 transcriptomic features effectively stratified patient survival across multiple cohorts, with robust AUCs and Kaplan-Meier survival distinctions.We identified a proliferative, stem-like tumor cell subpopulation (Tumor_C2) in HCC characterized by high H2AFZ expression, which drives tumor aggressiveness. T cell analysis revealed significant interactions with Tumor_C2. Moreover, a prognostic model based on Tumor_C2 features effectively stratified patient survival across multiple cohorts. Together, these findings highlight potential therapeutic targets for improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7024320dd03972a4da66c548f19e56393e35ae99" target='_blank'>
              Single-cell transcriptomics identifies an H2AFZ-driven proliferative tumor subpopulation associated with poor prognosis in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Jihui Huo, Tao Yang, Kai Lei, Zeyao Wang, Zebin Chen, Qi Zhou
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Megaphages are bacteriophages (i.e., phages) with exceptionally large genomes that have been discovered computationally across the globe. To date, all have evaded cultivation except phage G, which we examined with multiomics and artificial intelligence (AI) to resolve its 50-year cultivated history. Phage G is one of the largest phages with a size of >0.6 µm, about half the width of the host cell, and a 499 kbp, non-permuted, linear genome that has, uniquely among known phages, two pairs of ends. Phage G has >650 protein-coding open reading frames (ORFs), with >65% being hypothetical proteins, with the expansive repertoire of auxiliary metabolic genes (AMGs) acquired from its bacterial host, manipulate host sporulation (sspD, RsfA, spoK), and antiviral escape genes (e.g., anti-CBass nuclease and Anti-Pycsar protein). Our study represents a doorway into the complexity of the genomic repertoire of the only cultivated megaphage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14446808f1116f5a42848927b1c983a852439d05" target='_blank'>
              Unlocking the genomic repertoire of a cultivated megaphage
              </a>
            </td>
          <td>
            Andra Buchan, Stephanie Wiedman, Kevin C Lambirth, Madeline Bellanger-Perry, Jose L. Figueroa, E. Wright, Patil Shivprasad Suresh, Qibin Zhang, Julie A. Thomas, Philip Serwer, Richard Allen White
          </td>
          <td>2025-09-30</td>
          <td>npj Viruses</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35317e7a4cee78e23641ba79f278922203278977" target='_blank'>
              A chemical-genetic interaction between PAF1 and ENL/AF9 YEATS inhibition.
              </a>
            </td>
          <td>
            Paige A. Barta, Leopold Garnar-Wortzel, T. Bishop, Rachel E Hayward, Lauren M. Hargis, James B. Shaum, H. Kwok, Brian B Liau, Benjamin F. Cravatt, Michael A. Erb
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background The advent of poly (ADP-ribose) polymerase inhibitors (PARPi) over the past decade has significantly altered the management of epithelial ovarian cancer (EOC). We proposed that the etiology of homologous recombination deficiency (HRD) might underlie the variable responses to PARPi observed across patient populations. Methods As part of the phase 2 study of the Chinese HRD Harmonization Project, we developed a genomic methylation sequencing (GM-seq) pipeline facilitated by the TET enzyme for the simultaneous identification of methylated modifications and genetic variations in EOC tumor samples, and compared with established DNA sequencing-based HRD assays. Results Somatic mutation and HRD scores were confounded by low tumor purity in our cohort of 98 locally advanced/advanced EOC patients. In samples with tumor purity ≥ 30% (n = 45), the GM-seq pipeline showed high consistency with DNA sequencing-based HRD assay, identifying genetic variations in homologous recombination repair (HRR) genes and HRD score with 92.6% (25/27) and 97.1% (33/34) consistency respectively, in addition to conducting methylation profiling. Moreover, different underlying mechanisms of HRD were associated with varying degrees of PARPi efficacy, with BRCA1/2 LOH group having the best efficacy (median PFS, undefined), followed by BRCA1 methylation group (median PFS, 23.4 months), and those with unknown etiology of HRD having the worst efficacy (median PFS, 8.8 months, p < 0.001). Conclusion Our findings underscore the importance of considering HRD etiology when evaluating PARPi efficacy in EOC patients. The GM-seq pipeline, represents a significant advancement in HRD detection, enabling more accurate predictions of PARPi response. Supplementary Information The online version contains supplementary material available at 10.1186/s40364-025-00843-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5213220b5e2f3e35a5d8562e34edf897da4714a0" target='_blank'>
              Comprehensive assessment of homologous recombination deficiency via simultaneous methylation and mutation analysis in epithelial ovarian cancer: implications for PARP inhibitors efficacy
              </a>
            </td>
          <td>
            Lin Dong, Huanwen Wu, Ning Li, Wenbin Li, Yan Song, Yuanyuan Xiong, Huan Yin, Huan Fang, Rongrong Chen, Xin Yi, Jie Huang, Jianming Ying
          </td>
          <td>2025-10-10</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer is the second leading cause of death worldwide, with an estimated 27.5 million new cases projected by 2040. Disruptions in cell cycle control cause DNA replication errors to accumulate during cell growth, leading to genomic instability and tumor development. Proteins that regulate cell cycle progression and checkpoint mechanisms are crucial targets for cancer therapy. NIMA-related kinases (NEKs) are a family of serine/threonine kinases involved in regulating various aspects of the cell cycle and mitotic checkpoints in humans. Among these, NEK10 is the most divergent member and has been associated with both cancer and ciliopathies, a group of disorders caused by defects in cilia structure or function. Despite its biological significance and distinctive domain architecture, the structural details of NEK10 remain largely unknown. To address this gap, we employed computational modeling techniques to predict the complete structure of the NEK10 protein. Our analysis revealed a catalytic domain flanked by two coiled-coil domains, armadillo repeats (ARM repeats), an ATP binding site, two putative ubiquitin-associated (UBA) domains, and a PEST sequence known to regulate protein degradation. Furthermore, we mapped a comprehensive interactome of NEK10, uncovering previously unreported interactions with the cancer-related proteins MAP3K1 and HSPB1. MAP3K1, a serine/threonine kinase and E3 ubiquitin ligase frequently mutated in cancers, interacts with the catalytic region of NEK10. The interaction with HSPB1, a molecular chaperone associated with poor cancer prognosis, is mediated by NEK10's ARM repeats. Our findings highlight a potential connection between NEK10, ciliogenesis, and cancer, suggesting an important role in cancer development and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84bfc7864763ec8c742e96b5b16848a5dc021a94" target='_blank'>
              In Silico Analysis of Human NEK10 Reveals Novel Domain Architecture and Protein-Protein Interactions.
              </a>
            </td>
          <td>
            A. S. Eichner, Nathaniel Zimmerman, Avdar San, Shaneen Singh
          </td>
          <td>2025-10-03</td>
          <td>Proteins</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Colorectal cancer is one of the most prevalent malignant tumors worldwide, and it has one of the highest rates of mortality and morbidity. Its emergence and progression are intricately linked to various genetic and epigenetic factors influencing colonic epithelial cells. Long non-coding RNAs function as crucial regulators of the epigenetic landscape, playing significant roles in the development, progression, invasion, metastasis, and drug resistance of various tumors, particularly colorectal cancer. Epigenetic modifications, such as DNA methylation, RNA methylation, and histone modification, facilitate heritable gene expression regulation without changing the underlying DNA sequence. Recent studies have increasingly highlighted the intricate interactions between lncRNAs and these epigenetic modifications in colorectal cancer, illuminating their participation in critical processes, including cell proliferation, apoptosis, invasion, and metastasis. This review concentrates on the relationship between long non-coding RNAs and several significant epigenetic modifications that drive colorectal cancer development, offering fresh insights for future research that aims at devising more effective treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31b080b73592dba79e60d81f2a7fb3048fe78b62" target='_blank'>
              Long non-coding RNAs promote colorectal cancer development through other epigenetic modifications
              </a>
            </td>
          <td>
            Guojuan Wang, Qingqing Ma, Naicheng Zhu, Yuwei Yan, Chen Zhong, Xinlin Hong, Yujuan Lai, Xiudan Chen, Nanxin Li, Wenyan Yu
          </td>
          <td>2025-09-28</td>
          <td>Gastroenterology Report</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 In contrast to prevailing wisdom that cancer is a genetic disease that evolves via Darwinian selection, it is increasingly recognized that in addition to genetic mutations, intratumoral heterogeneity also results through non-genetic mechanisms. Furthermore, it is now evident that the malignant phenotype results from complex interactions between genetic and non-genetic mechanisms. These interactions contribute to functional changes across multiple spatiotemporal scales, from molecular to cellular to tissue level, creating a heterogeneous cancer cell population. Cancer Systems Biology is a comprehensive, up-to-date treatise that addresses these issues. The main intent of this compendium is to encourage cross-pollination of ideas from experts in multiple disciplines including basic and translational science researchers, systems/mathematical biologists, and biophysicists and clinicians interested in decoding a systems-level emergent view of cancer so that a deeper understanding of disease progression can be gained and more effective treatment strategies can be designed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a03788cb0b91aa174c4a7535dcca74621b417df9" target='_blank'>
              Cancer Systems Biology
              </a>
            </td>
          <td>
            Ravi Salgia, M. Jolly, Prakash Kulkarni, Govindan Rangarajanm
          </td>
          <td>2025-09-01</td>
          <td>None</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) involves the expansion of hematopoietic stem cells with ageacquired mutations linked to myeloid malignancy. Advances in next-generation and single-cell sequencing, along with computational modeling, have expanded our ability to detect both common and rare CH drivers, including single-nucleotide variants and mosaic chromosomal alterations, with increasing sensitivity. While sequencing methods differ in accuracy, cost, and ability to detect low-frequency variants, they have deepened our understanding of CH biology. A growing body of evidence has identified both somatic drivers, such as variants in DNMT3A, TET2, and ASXL1, and germline genetic variants that modify CH risk, highlighting the complex interplay between inherited and acquired factors. These collective discoveries are guiding the development of targeted therapies and interventions, particularly for individuals at risk of progression to myeloid neoplasms or cardiovascular disease. Additionally, CH is emerging as a clinically relevant factor in the treatment of solid tumors, where it may influence the tumour microenvironment, treatment response and the risk of therapy-related complications. Risk stratification models are facilitating earlier identification and monitoring of high-risk individuals, enabling personalized treatment decisions. The scope of CH management continues to expand, from surveillance to intervention, with ongoing trials testing preventive strategies in high-risk populations. Emerging trial frameworks emphasize risk stratification, age-appropriateness, inclusive recruitment, decentralized trial models, and the use of traditional clinical and novel endpoints. Together, these advances reflect a shift from passive observation to proactive intervention, charting a course for early detection, precision treatment, and prevention in CH care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6faa3f8b3f8e8d6ae046148a99055b41b7b424" target='_blank'>
              CHIP ahoy: charting a decade of discovery in clonal hematopoiesis.
              </a>
            </td>
          <td>
            Casey K. Wong, Alexandra McDonald, M. Buttigieg, M. Rauh
          </td>
          <td>2025-09-04</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="There is growing interest in understanding the mechanisms underlying differences in cancer incidence among species (comparative oncology). The naked mole-rat (NMR) is often referenced as "cancer-resistant" and prior studies focused on identifying mechanisms explaining this. However, efforts to assess this in vivo have been limited. Herein, we provide evidence that the NMR presents as a novel autochthonous model of lung tumor initiation, driven by an introduction of the oncogenic Eml4-Alk fusion protein using CRISPR-mediated genome editing. Whereas in mice the inversion alone is sufficient to drive tumorigenesis, the inversion alone was insufficient to drive tumorigenesis in the NMR lung and tumor development required additional losses of the tumor suppressors p53 and pRb. Our findings suggest that the proposed "resistance" of the NMR to the development of cancer may reflect that the genetic events leading to tumor initiation are likely to be comparable to those present in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a53eef9b5cb3d97ea0c88db9955706dc37ce138" target='_blank'>
              An Autochthonous Model of Lung Cancer Identifies Requirements for Cellular Transformation in the Naked Mole-Rat.
              </a>
            </td>
          <td>
            A. Shepard, Daniel K Lester, Scott Troutman, Sany Hoxha, Walid T Khaled, E. S. Smith, Thomas J. Park, Rochelle Buffenstein, Dongliang Du, Mingxiang Teng, C. Dengler-Crish, Kenneth Y Tsai, Elsa R Flores, Andrea Ventura, Joseph L. Kissil
          </td>
          <td>2025-09-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>20</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [11],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>