<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-06-09 09:30:05 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>34</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>71</td>
          <td>136</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>112</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>335</td>
          <td>136</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>294</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a common source of oncogene amplification across many types of cancer. The non-Mendelian inheritance of ecDNA contributes to heterogeneous tumour genomes that rapidly evolve to resist treatment. Here, using single-cell and live-cell imaging, single-micronucleus sequencing, and computational modelling, we demonstrate that elevated levels of ecDNA predisposes cells to micronucleation. Damage on ecDNA, commonly arising from replication stress, detaches ecDNA from the chromosomes upon which they hitchhike during cell division, thereby causing micronucleus formation in daughter cells. Clusters of oncogene-containing, CIP2A-TOPBP1-associated ecDNA molecules form, and asymmetrically segregate into daughter cell micronuclei during cell division. ecDNA chromatin remains highly active during mitosis, but upon micronucleation, it undergoes suppressive chromatin remodeling, largely ceasing oncogene transcription. These studies provide insight into the fate of damaged ecDNA during cell division.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8098c87089e0ecbf0e8dc20a062afe3e12362b0b" target='_blank'>
              Oncogene Silencing via ecDNA Micronucleation
              </a>
            </td>
          <td>
            Lotte Brückner, Robin Xu, Jun Tang, Alexander Herrmann, I. Wong, Shu Zhang, Fengyu Tu, Madison Pilon, A. Kukalev, Katharina Pardon, O. Sidorova, Joshua Atta, Qinghao Yu, Davide Pradella, M. Ilić, Marco Novais-Cruz, Sarah Kaltenbach, Denise Treue, M. Giurgiu, Sergej Herzog, Ann-Sophie Hollinger, Martina Fernandez, Finnja Becker, Varvara-Rigina Louma, R. Schmargon, J. Dörr, David Gamlin, Annika Lehmann, D. Gürgen, Matthias Richter, Frank Dubois, Fabrizio Simeoni, Betheney R. Pennycook, Alastair Hamilton, Ralph K. Lindemann, Matthias Fischer, V. Bafna, Geoffrey M. Wahl, R. Koche, Howard Y. Chang, Stamatis Papathanasiou, René Medema, B. Spanjaard, Andrea Ventura, Ana Pombo, Weini Huang, Benjamin Werner, P. Mischel, A. Henssen
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="
 Focal amplifications in the cancer genome, particularly extrachromosomal DNA (ecDNA) amplifications, are a pivotal event in cancer progression across diverse cancer contexts. These megabase-scale circular structures arise when DNA breaks off chromosomes, accumulating hundreds of copies per cell through asymmetric segregation and positive selection for oncogenic elements. While previously thought to be a late-stage consequence of genome instability, recent evidence suggests ecDNA may arise as early as pre-cancer and associate with malignant transformation (Luebeck et al., Nature 2023). However, identifying and delineating these distinct events from other modes of amplification using whole-genome sequencing (WGS) data remains challenging due to their complex profiles of copy number and structural variation. We present AmpliconSuite, a comprehensive workflow that robustly identifies focal amplifications from WGS data. At its core is the AmpliconArchitect (AA) module, which jointly analyzes structural variants and copy numbers. AA has been deployed on thousands of samples, helping elucidate the role of ecDNA in cancer. AmpliconSuite enhances reproducibility by providing a seamless end-to-end workflow: upstream, it standardizes identification of candidate focal amplification regions, and downstream, it uses AmpliconClassifier to report amplification mechanisms. The workflow has been extensively validated on a cytogenetic-backed dataset of over 150 cancer cell-line/gene probe combinations. Its easily-interpretable tabular outputs annotate gene contents, copy numbers, and summarize structural complexity of amplicons. AmpliconSuite can be run on cloud.genepattern.org, requiring only a web browser, which enables cloud-based "push button" bioinformatic analysis. To date, it has been deployed across more than 30 large-scale studies, processing over 45,000 tumor samples and three petabytes of sequencing data. We introduced novel methods to categorize ecDNA types by matching structural variation patterns to formation mechanisms. Our approach distinguishes ecDNA generated by chromothripsis from those arising from excisional models or stalled replication forks, validated against engineered ecDNA formation systems (Shoshani et al., Nature 2021, Pradella et al., Nature 2024). Complementing the tool is AmpliconRepository.org, a community-editable platform for sharing focal amplification calls. It allows researchers to share results publicly or privately and currently provides ecDNA predictions for over 4,500 tumor samples from TCGA, PCAWG, CCLE, and other datasets. AmpliconSuite advances our understanding of ecDNA biology by providing a reproducible, user-friendly tool that reveals novel insights into extrachromosomal DNA formation, breakage-fusion-bridges, and other mechanisms of oncogene amplification.


 Jens Luebeck, Edwin Huang, Ted Liefeld, Forrest Kim, Ivy T. Wong, Bhargavi Dameracharla, Gino Prasad, Rohil Ahuja, Daniel Schreyer, Kaiyuan Zhu, Rishaan Kenkre, Tushar Agashe, Devika Torvi, Soyeon Kim, Hoon Kim, Peter Bailey, Roel G. Verhaak, Viraj Deshpande, Michael Reich, Paul S. Mischel, Jill Mesirov, Vineet Bafna. AmpliconSuite enables discovery of extrachromosomal DNA in tumor genomes [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3755.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e88cd9731c06a22174c2d15ed88fd1e8e310e0fb" target='_blank'>
              Abstract 3755: AmpliconSuite enables discovery of extrachromosomal DNA in tumor genomes
              </a>
            </td>
          <td>
            J. Luebeck, Edwin Huang, T. Liefeld, Forrest Kim, I. Wong, Bhargavi Dameracharla, Gino Prasad, Rohil Ahuja, Daniel Schreyer, Kaiyuan Zhu, Rishaan Kenkre, Tushar Agashe, Devika Torvi, Soyeon Kim, Hoon Kim, Peter Bailey, R.G.W. Verhaak, Viraj Deshpande, Michael Reich, P. Mischel, Jill P. Mesirov, V. Bafna
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="The Extrachromosomal DNA (ecDNA) in cancer biology has emerged as a critical driver, enhancing the knowledge about tumorigenesis, progression and treatment. The current review article explores the multifaceted role of Extrachromosomal DNA, focusing on its ability to amplify oncogenes, enhance tumor heterogenicity, and support cancer cell survival and understanding therapeutics pressure. Moreover, the unique transcriptional and replication dynamics of ecDNA with linking the uneven segregation during cell division, create a significant environment for the understanding of genetic diversity and adaptive evolution in tumors. In clinics the ecDNA holds significant potential as a biomarker for cancer, diagnosis, treatment, therapeutic monitoring and progenies. The higher level of ecDNA correlates with poor clinical outcomes and resistance to therapies, emphasizing its prognostic value. The current therapeutic techniques targeting ecDNA specific vulnerabilities, such as disrupting replication stress pathways or silencing the transcriptional activity, represents promising ways for cancer treatment This review also highlights the potential of circulating ecDNA in liquid biopsies as a non-invasive diagnostic and monitoring tool. Despite these advances, challenges such as standardizing detection methods and understanding ecDNA diverse roles across cancer types remain. Addressing these gaps will unlock new opportunities in precision oncology, making ecDNA a cornerstone of future cancer diagnostics and therapeutics.
Bangladesh J Medicine 2025; 36(2): 82-91">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc548fd944fe0c719676cbeb264f28cf2ce64996" target='_blank'>
              The Multifaceted Role of Extrachromosomal DNA in Cancer: Functions and Clinical Implications
              </a>
            </td>
          <td>
            Shafee Ur Rehman
          </td>
          <td>2025-05-05</td>
          <td>Bangladesh Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Cisplatin resistance is a major cause of tumor recurrence and mortality in high-grade serous ovarian cancer (HGSOC). Extrachromosomal circular DNA (eccDNA) has emerged as a critical factor in tumor evolution and drug resistance. However, the specific contribution of eccDNA to cisplatin resistance in HGSOC remains unclear. Methods: We performed whole-genome sequencing, Circle-Seq, and RNA-Seq in four pairs of primary and cisplatin-resistant (cisR) HGSOC cell lines to characterize genome-wide eccDNA distribution and features. Functional enrichment analyses were subsequently conducted on differentially expressed eccDNA-related genes. Results: In the SKOV3 cisR cell line, we identified a large extrachromosomal circular DNA (ecDNA) carrying the HIF1A gene, which regulates DNA repair, drug efflux, and epithelial–mesenchymal transition, contributing to cisplatin resistance. Using Circle-Seq, we detected a total of 161,062 eccDNAs, most of which were less than 1000 bp and distributed across all chromosomes. Notably, the number of eccDNAs on chromosome 21 differed significantly between the primary and cisR cell lines. Additionally, eccDNAs were predominantly located in non-coding repetitive elements. Functional analysis of eccDNA-related differentially expressed genes revealed that, compared to primary cell lines, cisR cell lines were associated with mitotic spindle assembly, regulation of vascular permeability, and cell differentiation. eccDNA-related genes involved in these pathways include MISP, WIPF1, RHOD, KRT80, and PLVAP. Conclusions: Our findings suggest that eccDNAs, particularly ecDNA amplifications like HIF1A, contribute significantly to cisplatin resistance mechanisms in HGSOC. These insights highlight eccDNA as a potential target for overcoming therapeutic resistance and improving treatment outcomes in ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a7813e60dab9b0878e0aa308d86aa5f7fb84d46" target='_blank'>
              Characterization of Extrachromosomal Circular DNA in Primary and Cisplatin-Resistant High-Grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Youya Wang, He Li, Qinglan Li, Yi Li, Hao Wu, Yan Ge, Xingnuo Zhu, Zhiguo Zheng, Zhongsheng Sun
          </td>
          <td>2025-04-29</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d556d25908485193c4e548584bab10bfa70daa1" target='_blank'>
              Oncogenic drivers shape the tumor microenvironment in human gliomas
              </a>
            </td>
          <td>
            Bo Zhao, Chae Yun Cho, Lingqun Ye, Tula Keal, Thomas Mitchell, Inés Martín-Barrio, C. A. Faraj, Mesut Unal, Jessica Alsing, Nora M Lawson, Ellie Rahm Kim, Francisco Chavez, Kristy Mendoza Rangel, Sanjay K Singh, Joseph H. McCarthy, D.M. Ashley, W. Yung, Vinay K. Puduvalli, Guy Nir, Jeffrey S. Weinberg, Linghua Wang, Keith L. Ligon, G. Genovese, Peter Van Loo, P. Futreal, Jason T. Huse, Jesse R. Dixon, Frederick F. Lang, Kadir C. Akdemir
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="
 The amplification of oncogenes on extrachromosomal DNA (ecDNA) is common in cancer and associated with poor outcomes. Uneven inheritance of ecDNA during cell division leads to intratumoral ecDNA copy number heterogeneity. However, the relationship between shifts in ecDNA copy number and the rapid environmental adaptation of cancer cells remains unclear. In this study, we utilized imaging and single-cell multi-omics analyses to demonstrate that the transcript and protein expression of ecDNA oncogenes, along with functional heterogeneity, correlate with ecDNA copy number heterogeneity. Using fluorescence-activated cell sorting (FACS) and live-cell imaging, we found that cells with “middle” ecDNA copy numbers display the highest growth rates compared to cells with the lowest or highest copy number statuses, indicating greatest fitness is achieved by the balance of promoting cell proliferation while minimizing heightened replication stress arising from increasing copy numbers of ecDNA oncogenes. Through simulations based on random segregation and optimal copy number models, we found that ecDNA-bearing cells can rapidly adjust their copy numbers to achieve optimal growth in new environments. Notably, compared to chromosomally amplified cells, those with ecDNA exhibited superior growth following xenograft injection and treatment with single drugs targeting replication stress or oncogenes, aligning with our simulation findings. Our results reveal that functional heterogeneity resulting from uneven ecDNA inheritance enhances cancer cells' adaptability to tumor microenvironments and rapid acquisition of drug resistance, warranting that the development of combinatorial therapies that synergistically target a wide range of copy numbers is critical for effectively treating ecDNA-containing cancers.


 Shu Zhang, Aditi Gnanasekar, Ellis J. Curtis, Jun Tang, Vishnu Shankar, Ivy Tsz-Lo Wong, Xiaowei Yan, Rui Li, Lu Yang, Howard Y. Chang, Paul S. Mischel. Uneven inheritance and functional heterogeneity derived from extrachromosomal DNA enable rapid adaptation of cancer cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3884.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9123bbc8d86d0e1ae0116fdaeb8338ea509e867c" target='_blank'>
              Abstract 3884: Uneven inheritance and functional heterogeneity derived from extrachromosomal DNA enable rapid adaptation of cancer cells
              </a>
            </td>
          <td>
            Shu Zhang, Aditi Gnanasekar, Ellis J Curtis, Jun Tang, Vishnu Shankar, I. Wong, Xiaowei Yan, Rui Li, Lu Yang, Howard Y. Chang, P. Mischel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1722c16d6bca9584ea138fe4d3c40e8cbe8a4e7" target='_blank'>
              Extrachromosomal DNA Gives Cancer a New Evolutionary Pathway
              </a>
            </td>
          <td>
            Yue Wang, Oliver Cope, Jingting Chen, Aarav Mehta, Dalia Fleifel, Christina G. Ford, Poorya Benhamie, Santiago Haase, Saygin Gulec, Tim Elston, Philip M. Spanheimer, Caroline A Tomblin, Alison M Rojas, Tia Tate, Jeremy Purvis, Jeremy R Wang, Joseph M Dahl, Sam Wolff, J. Cook, Elizabeth Brunk
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Genome conformation underlies transcriptional regulation by distal enhancers, and genomic rearrangements in cancer can alter critical regulatory interactions. Here we profiled the 3D genome architecture and enhancer connectome of 69 tumor samples spanning 15 primary human cancer types from The Cancer Genome Atlas (TCGA). We discovered three archetypes of enhancer usage for over one hundred oncogenes across human cancers: static, selective gain, or dynamic rewiring. Integrative analyses revealed the enhancer landscape of non-cancer cells in the tumor microenvironment at genes related to immune escape. Deep whole genome sequencing and enhancer connectome mapping provided accurate detection and validation of diverse structural variants across cancer genomes, and revealed distinct enhancer rewiring consequences from noncoding point mutations, genomic inversions, translocations, and focal amplifications. Extrachromosomal DNA promoted more extensive enhancer rewiring of oncogenes among several types of focal amplification mechanisms. These results suggest a systematic approach to understand genome topology in cancer etiology and therapy.


 Yanding Zhao, Katie E. Yost, King L. Hung, Kaiyuan Zhu, Duo Xu, Ryan Corces, Shadi Shams, Bryan H. Louie, Shahab Sarmashghi, Laksshman Sundaram, Jens Luebeck, Stanley Clarke, Ashley S. Doane, Jeffrey M. Granja, Hani Choudhry, Marcin Imieliński, Andrew D. Cherniack, Ekta Khurana, Vineet Bafna, Ina Felau, Jean C. Zenklusen, Peter W. Laird, Christina Curtis, Cancer Genome Atlas Analysis Network, William J. Greenleaf, 1 Howard Y. Chang. Three-dimensional genome landscape of primary human cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5046.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f629203ff4e6d119bd3d8d7b5f0a9d082c7f87e7" target='_blank'>
              Abstract 5046: Three-dimensional genome landscape of primary human cancers
              </a>
            </td>
          <td>
            Yanding Zhao, Katie E. Yost, King L. Hung, Kaiyuan Zhu, Duo Xu, Ryan Corces, Shadi Shams, Brian H Louie, Shahab Sarmashghi, Laksshman Sundaram, J. Luebeck, Stanley Clarke, A. S. Doane, Jeffrey M. Granja, Hani Choudhry, Marcin Imieliński, Andrew D Cherniack, Ekta Khurana, V. Bafna, Ina Felau, J. Zenklusen, Peter W. Laird, Christina Curtis, Cancer Genome Atlas Analysis Network, W.J. Greenleaf, Howard Y. Chang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Chromoplexy is a phenomenon of complex genome rearrangement, occurring during a single cell event and characterized by the formation of chain rearrangements affecting multiple chromosomes. Unlike other genomic rearrangements such as chromothripsis, which involves a single chromosome, chromoplexy affects several chromosomes at once, creating patterns of complex, balanced translocations, and leading to the formation of fusion genes and the simultaneous disruption of several genes. Chromoplexy was first identified in prostate cancers, but it is now observed in various cancers where gene fusions take place. The precise mechanisms behind chromoplexy remain under investigation. The occurrence of these rearrangements follows multiple double-stranded breaks that appear to occur in certain regions or during particular genome configurations (open chromatin, active transcription area), and which lead to an intricate series of inter- and intra-chromosomal translocations and deletions without significant alterations in the number of copies. Although chromoplexy is considered a very early event in oncogenesis, the phenomenon can be repeated and can constitute a mechanism of clonal tumor progression. The occurrence of chromoplexy supports the equilibrium model punctuated by tumor evolution, characterized by periods of relative stability punctuated by sudden and rapid periods of radical genomic changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3ffdab6d59ce2f03de66a13a4efd90a8a827d6" target='_blank'>
              Chromoplexy: A Pathway to Genomic Complexity and Cancer Development
              </a>
            </td>
          <td>
            Franck Pellestor, Benjamin Ganne, Jean Baptiste Gaillard, V. Gatinois
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Focal amplifications of oncogenes are important cancer drivers. They can occur on chromosomes or in the context of circular extrachromosomal DNA (ecDNA). Many key features of ecDNAs were described in the 1960s to 1980s, including their “unstable” nature and their ability to confer drug resistance. With the benefit of new technologies, our understanding of ecDNAs has advanced dramatically in the last decade, both in breadth and in depth, including the remarkable discovery that ecDNAs are present in 17% of all cancers and are associated with worse patient outcomes. In this study, we present a guide to ecDNA tools and biology. Significance: Focal amplifications on ecDNAs are commonly found in cancer and are associated with poor patient outcomes and distinct biology. In this review, we provide a guide to ecDNA biology and available tools as well as our perspective on this rapidly evolving field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c8be723b80118653ca09b51446fe4768c5505b8" target='_blank'>
              A Guide to Extrachromosomal DNA: Cancer’s Dynamic Circular Genome
              </a>
            </td>
          <td>
            Natasha E Weiser, Thomas B K Watkins, Howard Y. Chang, P. Mischel
          </td>
          <td>2025-04-27</td>
          <td>Cancer Discovery</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Extrachromosomal DNA (ecDNA) has been described in multiple cancer types. ecDNAs are circular and are usually greater than 500 base pairs in size. ecDNA structures lack centromeres and thus do not follow the Mendelian mode of inheritance, segregating unequally during mitosis. This can subsequently result in high copy number gene amplifications and enhance tumor heterogeneity, promoting tumor evolution. ecDNA structures have a dynamic nature whereby they can be maintained extrachromosomally in their circular form but can also reintegrate randomly back into the genome, in the form of Homogeneously Stained Regions (HSR). Earlier studies using TCGA reported that approximately 25% of breast samples contained ecDNA. We have recently reported that over 46% of HER2+ breast cancers have ecDNA amplifications. Moreover, ecDNA have been found in both pre-cancerous lesions, such as in tissues obtained from patients with Barret’s esophagus, but also following both targeted treatment and chemotherapy. This highlights the ability of the structures to be detected at any disease stage and treatment phase although exact mechanisms of ecDNA formation and maintenance have not been described fully. In the present study we use a tumor-informed in silico approach in conjunction with in vitro readouts to elucidate these mechanisms. 2981 Whole Genome Sequencing data from the Genomics England breast cancer cohort and cell line data from the Cancer Cell Line Encyclopedia were analyzed using Amplicon Suite. In parallel, a panel of eight ERBB2 amplified breast cancer cell lines was used to detect in vitro presence of ecDNA by metaphase FISH and Optical Mapping. Genes that were found to be differentially mutated according to the presence of ERBB2 focal amplifications were silenced in vitro, then gene expression of ERBB2 and target genes was evaluated using qPCR and the effect of ERBB2 signal was assessed using FISH. In the cohort analyzed, 22.8% of breast tumors had ecDNA amplifications. Enrichment analysis was conducted to assess the presence or absence of specific mutations in ERBB2-amplified vs non-amplified tumors. Several genes were identified as being positively or negatively enriched in the cohort. Similarly, we identified 9 breast cancer cell lines containing up to 5 different species of ecDNA. In a subset of cell lines computationally predicted to harbor ERBB2 on ecDNA, ERBB2 amplification was in the form of HSR. We selected HCC1569 as our candidate model and silenced candidate epigenetic regulators. Our preliminary data suggests this leads to a measurable and significant change on ERBB2 signal. At present, we highlight the propensity of ERBB2 to be amplified on ecDNA in both breast cancer patients and breast cancer cell lines. These cell lines can serve as models for the further study of the mechanisms of maintenance of ecDNA in vitro which will shed light on some of the key unresolved questions in the field.


 Georgia Stavrou, Chris Bailey, Jeanette Kittel, Mariam Jamal-Hanjani, Charles Swanton, Nnennaya Kanu. Investigating the prevalence of ecDNA in breast cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3897.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d61674df32f0a41cdb182a7f8115a40cdbf840a" target='_blank'>
              Abstract 3897: Investigating the prevalence of ecDNA in breast cancer
              </a>
            </td>
          <td>
            Georgia Stavrou, Chris Bailey, Jeanette Kittel, M. Jamal-Hanjani, C. Swanton, N. Kanu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6350a9e7edaa4d3aae5b8855bca2a2b934a43915" target='_blank'>
              P53 deficiency triggers hypertranscription, inducing nucleotide insufficiency causing replication stress, and genomic instability
              </a>
            </td>
          <td>
            Wisam Zaatra, George Philippos, Petr Smirnov, Shira Milo, Jan Otonicar, Michelle Chan, Michal Harel, Frauke Devens, Tchelet Goldberg, A. Eliassaf, Karen Grimes, M. Sinai, Gianluca Sigismondo, Tamar Geiger, Jan O. Korbel, Jeroen Krijgsveld, O. Shalev, Batsheva Kerem, Aurélie Ernst
          </td>
          <td>2025-05-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Extrachromosomal circular DNAs (eccDNAs) have gained attention as key players in cancer heterogeneity, potentially associated with elevated oncogene copy numbers in many cancers. While the presence of eccDNA in both normal and cancer cells is confirmed, its influence on gene-level alterations in cancer cells remains largely unexplored. This study delves into the genomic profiles of eccDNA in cholangiocarcinoma (CCA), an aggressive biliary tract cancer with extensive heterogeneity and diverse molecular alterations, using a modified long-read CircleSeq method. We reveal distinct eccDNA characteristics in CCA compared to non-tumor cells, focusing on genic components and chromosomal origins. Analysing read depth differences in oncogene-containing eccDNA; we identified potential eccDNA candidates that may be relevant for CCA biology. Subsequent bioinformatics analysis was performed using the established CReSIL tool, revealing distinct patterns of these oncogenes, particularly genes in the RAS/BRAF pathway, suggesting a potential functional role. These findings highlight the remarkable heterogeneity and diverse origins of eccDNA in CCA. This study establishes the first profiling of eccDNA in cholangiocarcinoma and paves the way for further investigation of its potential contribution to oncogene amplification and disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb865d79e02176dea33b880a5f277801ea0b0e0" target='_blank'>
              Characterization and identification of extrachromosomal circular DNA in cholangiocarcinoma
              </a>
            </td>
          <td>
            Zar Zar Win, Hasaya Dokduang, Siriyakorn Kulwong, W. Loilome, N. Namwat, Jutarop Phetcharaburanin, T. Wongsurawat, Piroon Jenjaroenpun, P. Klanrit, Arporn Wangwiwatsin
          </td>
          <td>2025-05-05</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="
 Extrachromosomal DNA (ecDNA), a prevalent mechanism of gene amplification, drives oncogene overexpression and is associated with poor patient survival. Over 60% of primary glioblastoma (GBM) cases harbor ecDNA-mediated oncogene amplifications. To better understand the functional consequences of ecDNA amplification, we performed a comprehensive analysis of ecDNA distribution in tumors using longitudinally sampled multiomic data from 35 GBM patient specimens. This dataset integrates 68 whole-genome sequencing (WGS) datasets, 243, 459 transcriptomic and chromatin accessibility profiles derived from the same nuclei. We developed a single-nucleus ATAC-seq-based pipeline to classify ecDNA status at single-cell resolution. To validate the robustness of this pipeline, we utilized patient-derived neurospheres (n = 2), generating multiomic datasets and over 100 fluorescent in situ hybridization (FISH) images per cell line for structural validation. This approach enabled the identification of ecDNA-containing regions in each patient sample through WGS data and the inference of single-cell-level copy number variations. In total, we classified the ecDNA status of over 86, 345 malignant cells. Our findings demonstrated that ecDNA copy numbers remain stable during tumor progression, with no significant changes in the proportion of ecDNA-containing cells over time. Notably, we observed that oncogenes amplified on ecDNA are associated with distinct tumor cell states. By applying non-negative matrix factorization and a random forest machine learning algorithm, we extracted gene programs linked to ecDNA that are enriched for cell-cycle regulation and chromatin regulating. These results provide new insights into the regulatory landscape of ecDNA-driven glioblastoma and highlight its potential role in shaping tumor cell identity and progression.


 Dacheng Zhao, Kevin Anderson, Amit Gujar, Megan Costa, Djamel Nehar-Belaid, Frederick Varn, Elise Courtois, Paul Gabrie, Paul Robson, Hyo-Eun Moon, Anna Golebiewska, Sun Ha Paek, Simone Niclou. Integrated multiomic analysis to infer longitudinal patterns of ecDNA in IDH-wildtype integrated multiomic analysis to infer longitudinal patterns of ecDNA in IDH wild-type glioblastoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7504.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a97fcf511bf3ac6d39959b0d32a3431fd3b9263" target='_blank'>
              Abstract 7504: Integrated multiomic analysis to infer longitudinal patterns of ecDNA in IDH-wildtype integrated multiomic analysis to infer longitudinal patterns of ecDNA in IDH wild-type glioblastoma
              </a>
            </td>
          <td>
            Dacheng Zhao, K. Anderson, Amit Gujar, Megan L Costa, Djamel Nehar-Belaid, F. Varn, Elise T. Courtois, Paul Gabrie, Paul Robson, H. Moon, A. Gołȩbiewska, S. Paek, S. Niclou
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5a400876e8f32c63411441923e6bbb8ebd5d239" target='_blank'>
              Chromosome-Specific Aneuploidy Engineering via dCas9-Induced Centromeric Chromatin Relaxation
              </a>
            </td>
          <td>
            Hugang Feng, Daqi Deng, Rashmi Dahiya, Libin Wang, Jingkun Zeng, Benjy Jek Yang Tan, F. Byrne, Scott T C Shepherd, Jiahao Wang, Sarah C. Johnson, Alison Harrod, Karen A. Lane, Annika Fendler, A. Cattin, Z. Tippu, Meilun Nie, Yiming Zhao, Ruijia Wang, Wei Ai, Omar Bouricha, Taja Barber, Yuliia Dovga, Yihan Xu, Yimeng Xu, Liani G. Devito, Lyn Healy, Zhen Sun, Weiming Shen, Hongye Wang, Eugénie S. Lim, Sam O'Toole, Scott Akker, William M. Drake, Haojie Jin, Jessica A. Downs, Sarah E. McClelland, J. Diffley, Peter Ly, S. Turajlic
          </td>
          <td>2025-04-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="
 Extrachromosomal DNA (ecDNA) is prevalent across various cancer types, notably Glioblastoma Multiforme (GBM), where it facilitates the rapid acquisition of high oncogene copy numbers by cancer cells. These ecDNAs exhibit self-replication and are randomly distributed to daughter cells during mitosis, resulting in uneven segregation. This leads to variations in oncogene copy numbers among subclones, contributing to significant intratumoral heterogeneity. Traditional sequence-based analyses are insufficient for addressing spatial heterogeneity. To overcome this limitation, we propose utilizing sequential OligoSTORM imaging combined with single-molecule localization microscopy (SMLM), an innovative hybridization technique, and a computational pipeline. Our hybridization method significantly enhances imaging speed: whereas imaging approximately 40 loci with typical sequential OligoSTORM requires 11 days to obtain data from about 20 cells, this method enables imaging of 104 loci within the same time frame. Additionally, we developed a computational pipeline to complement this technique. Together, these tools enable 3D imaging of chromosomes with a high resolution of 20 kb. Our hybridization approach allows imaging at both 100-kb and 20-kb resolution steps, organized hierarchically in a Matryoshka-like structure. Our pipeline incorporates advanced computational algorithms, including iterative closest point (ICP) and space-partitioning data structures, to precisely align each 20-kb sub-step to its corresponding 100-kb step. Using this methodology, we target the EGFR gene locus with high resolution. To validate our approach, we are applying it to a non-cancerous microglial cell line (HMC3) as a control and a patient-derived glioblastoma stem-like cell (GSC) line as our GBM model. The GSC line, rich in ecDNA content with an average of 18 ecDNAs per cell, demonstrates pronounced tumor heterogeneity, including significant EGFR amplifications among subclones. Through this methodology, we can elucidate the structural organization of ecDNAs containing oncogenes and their regulatory elements with fine precision. Furthermore, our approach seamlessly integrates these structural insights with matched single-cell genomic data. This combined strategy offers a comprehensive understanding of ecDNA dynamics in cancer, shedding light on tumor heterogeneity at both spatial and genomic levels and paving the way for novel insights into oncogene-driven cancer biology.


 Mesut Unal, Jessica Alsing, Gabrielle S. Dewson, Guy Nir, Kadir C. Akdemir. Integrating single-molecule localization microscopy and computational pipelines to elucidate ecDNA dynamics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3900.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ddc0ae23b848c6a2bead56e11b48f6e63657e8f" target='_blank'>
              Abstract 3900: Integrating single-molecule localization microscopy and computational pipelines to elucidate ecDNA dynamics
              </a>
            </td>
          <td>
            Mesut Unal, Jessica Alsing, Gabrielle S. Dewson, Guy Nir, Kadir C Akdemir
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Extrachromosomal DNA (ecDNA) are circular, megabase-scale regions of the genome found outside canonical chromosomes and characterized by high copy-number and frequent oncogene amplification. They are implicated in tumorigenesis, therapeutic resistance, and shortened survival for patients across multiple cancer types. Existing tools to identify and predict ecDNA structures are limited to low throughput, high sensitivity enrichment-based assays or high throughput, low sensitivity computational methods using whole-genome sequencing data. Recent advances in long-read sequencing offer potential improvements in ecDNA detection but such data are currently limited. Evaluating the impact of ecDNA in relation to clinical characteristics requires large cohorts such as The Cancer Genome Atlas (TCGA) or the International Cancer Genome Consortium (ICGC). Hence, there is value in leveraging short-read sequencing data from these existing cohorts while improving confidence in computational predictions of ecDNA presence and structure. Here we present ecDNAInspector, a computational tool that enables systematic selection of high-confidence ecDNA predictions from sequencing data. Each ecDNA prediction is characterized using structure-based metrics: an extreme cycle size Boolean, a breakpoint mapping error Boolean, and adapted classification metrics that assess the frequency of paired ends between neighboring cycle segments with orthogonal structural variant call support. ecDNA predictions are clustered by these metrics and the user can select high-confidence groups. A subsequent filtering step enables rescue of additional ecDNA predictions based on user-defined thresholds. After selection of a final high-confidence group of ecDNA predictions, ecDNAInspector offers modules for structural comparison and analysis of ecDNA within a cohort, including inspection of structural conservation (i.e. maintenance of similar sequence) and feature comparisons across tumors. Applying ecDNAInspector to a breast cancer cohort, we nominated a subset of high-confidence ecDNA with strong orthogonal structural variant support and additional indicators of high quality. We identified both base pair and oncogene inclusion conservation within breast cancer subtype groups, and noted significant variations in structural metrics across subtype groups. Since structure often indicates origin and dictates function, these differences suggest potential biological differences in ecDNA genesis and evolution across breast cancer subtypes. Thus, ecDNAInspector provides a systematic method for selecting high-confidence ecDNA predictions for downstream structural analysis, enabling improved utilization of existing sequencing data in valuable clinical cohorts.


 Sophia J. Pribus, Kathleen E. Houlahan, Lise Mangiante, Christina Curtis. ecDNAInspector nominates high-confidence structural predictions of extrachromosomal DNA for downstream analysis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3754.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7bed6325a16795853f9398ff50f536009080006" target='_blank'>
              Abstract 3754: ecDNAInspector nominates high-confidence structural predictions of extrachromosomal DNA for downstream analysis
              </a>
            </td>
          <td>
            Sophia Pribus, Kathleen E. Houlahan, Lise Mangiante, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 We have leveraged long-read and spatial sequencing modalities to identify cancer-specific oncogenic alterations for treatment of osteosarcomas.
 Osteosarcoma is archetypal of cancers driven by aneuploidy and structural rearrangement rather than point mutations. The functional consequences of many SVs stem from their effect on multidimensional genome organization. SVs can enable enhancer hijacking, alter boundaries of topologically associated domains (TADs), and move gene loci to different regulatory compartments to affect gene expression. SVs can also lead to the formation of extrachromosomal DNA amplicons (ecDNA), which can be megabases in size and can incorporate both genes and regulatory elements to result in substantial increases in gene transcription and intratumor heterogeneity. Complex SVs may have profound implications for prognosis and treatment in OS and other cancers yet they remain poorly understood. By incorporating long-read optical genome mapping (OGM) and chromatin conformation capture sequencing (HiC), we studied SVs in osteosarcoma with the goal of elucidating their contribution to cancer development.
 We studied a unique panel of 11 OS patient-derived xenograft (PDX) cell lines which have been previously characterized regarding response to targeted therapies as well as genomically via whole genome (WGS), RNA (RNA-seq), and chromatin accessibility (ATAC-seq) sequencing. To understand the complex genomic reorganization that occurs in osteosarcoma, we integrated OGM and WGS data for all 11 cell lines to describe detailed genome-wide SVs including ecDNA formation. PDX lines assayed using OGM exhibited between 106 and 704 translocations each (versus zero detected in germline controls) and we identified dozens of potential ecDNA amplicons, including one that may explain exceptionally high expression of YAP1 and BIRC3 transcripts. Further, we use HiC to determine which ecDNA amplicons appear to be free-floating based on their genome-wide interaction scores, noting significantly greater interaction in between 33% and 67% of ecDNA detected using OGM and break point graph-based methods. We also used HiC to describe the topological environment resulting from genome-wide SVs and observed potential enhancer hijacking events affecting dozens of known oncogenes including MYC and CDK4. HiC interaction maps among six cell lines reveal genomic compartments and TAD boundaries in detail, with 4,167 unique TAD boundaries identified among six cell lines using 50kb genomic bins. Changing TAD boundaries are associated with numerous SVs and/or copy number discrepancies, and are associated with genomic regions that contain genes with dysregulated RNA transcript levels. Overall, OGM and HiC modalities add considerably to our ability to detect oncogenic SVs and the genes they affect.


 Andrew Scott Clugston. High-resolution mapping of oncogenic structural changes in osteosarcoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2775.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d2ac30d018ecdf847938d3f5d1a087883242d4" target='_blank'>
              Abstract 2775: High-resolution mapping of oncogenic structural changes in osteosarcoma
              </a>
            </td>
          <td>
            A. Clugston
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/812046bef63516ecf0a2f8e99e68c96c27339a88" target='_blank'>
              Oncogenic lncRNA transgene transcription modulates epigenetic memory at a naïve chromosomal locus
              </a>
            </td>
          <td>
            Sweta Sikder, Songjoon Baek, Yamini Dalal, Ganesan Arunkumar
          </td>
          <td>2025-05-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Tumor virus is one of the major causes of human cancers, accounting for about 10-20% of their global incidence. Viral oncogenes in the tumor viruses promote tumorigenesis by deregulating host cancer genes. For instance, E6 and E7 proteins of human papillomavirus 16 (HPV16) induce the degradation of the protein products of tumor suppressor genes (TSGs) including p53 and pRb, to promote head and neck squamous cell carcinoma (HNSCC). Therefore, it is imperative to understand how differences in the dosage of viral genes affect cancer biology. However, how the relative and absolute abundance of each viral gene affects tumorigenesis and tumor biology has remained elusive, due to the difficulties in the analysis. Unlike human genomes that exist in two copies per cell, viral genome copies can take dynamic ranges, making copy number analyses challenging. Thus, we developed an R package ELViS that analyzes viral copy number change events from targeted DNA sequencing of whole viral genomes. In the first step, it takes aligned reads in the BAM file to generate base-resolution read depth and finds baseline regions based on the shape of the data. Next, to overcome the uneven base-resolution depths of targeted sequencing data and find copy number events, our framework performs joint segmentation in two-dimensional data space using both normalized read depths and Z-scores. In the simulation dataset, ELViS could detect deletions as small as 50bp and duplications as small as 500bp with F1 score > 0.75 at 50X coverage depths. ELViS provides base-resolution copy number change calls, along with publication-ready read depth line plots and cohort-level heatmap visualization of multiple data tracks with integrative clustering results.


 Jin-Young Lee, Hyo Young Choi, Jeremiah R. Holt, Xiaobei Zhao, Katherine A. Hoadley, D. Neil Hayes. ELViS : An R package for estimating copy number levels of viral genomic segments at base-resolution [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6307.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a65e388eb0715f0681bd240054e0692300312172" target='_blank'>
              Abstract 6307: ELViS : An R package for estimating copy number levels of viral genomic segments at base-resolution
              </a>
            </td>
          <td>
            Jin-Young Lee, Hyo Young Choi, J. Holt, Xiaobei Zhao, K. Hoadley, D. N. Hayes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Tandem Duplicator Phenotypes (TDPs) are a class of genome instability profiles characterized by the presence of hundreds of somatic head-to-tail tandem duplications (TDs) evenly distributed throughout the cancer genome. Different TDPs are characterized by distinct but recurrent TD span profiles, they manifest with different frequencies across different tumor types, and driven by distinct genetic perturbations (i.e., BRCA1 loss, activation of the CCNE1/CCNA2 pathways or CDK12 loss). Occurring in over 14% of all cancers, TDs in TDP disrupt tumor suppressors and duplicate oncogene. Thus, TDP is a genomic instability profile that enhances the oncogenic burden of a tumor. To better understand the potential mechanisms of TD formation in the context of the TDPs, we performed a pancancer meta-analysis of the local genomic features associated with somatic structural variation breakpoints. Specifically, we grouped, classified and characterized the four basic types of somatic structural variation breakpoints (TDs, deletions, inversions and translocations) based on their span size, their TDP context as well as its associated genetic perturbation. Using both a discovery and a validation datasets, comprising 3,084 and 4,784 independent cancer genomes, respectively, we found that cyclin-related TD breakpoints, from small (∼10 Kb), medium (∼200-330 Kb) and large (∼1-2Mb) TDs, consistently form in regions with the highest likelihood of generating a replication fork barrier due to either secondary DNA conformations or transcription-replication collisions. On the other hand, both small and medium BRCA1-linked TD breakpoints show significantly weaker enrichment in those same regions. This suggests that the signal of BRCA1-linked TD breakpoints mapping to these regions may be diluted by a more homogenous distribution of breakpoints throughout the genome, potentially reflecting attempted replication across ssDNA breaks in the parental template followed by their conversion to single ended breaks. Interestingly, neither the medium nor the large TDs linked to CDK12 deficiency associated with local DNA features indicative of replication fork stalling, suggesting a different mechanism for CDK12-linked TD formation. Converging results were obtained in a sample-based analysis, where each genome was classified based on its TDP/genetic driver status and analyzed as a unit: cyclin-related TDP genomes are enriched for TD breakpoints mapping to regions of early replication/replication origin, active chromatin, high transcription, G4 and Rloop formation and regions prone to double strand break, when compared to non-TDP genomes that are wild type for all drivers examined. Conversely, BRCA1-TDP and CDK12-TDP genomes show no enrichment of these genomic features at their TD breakpoints. Of interest, CCNE1 amplified genomes and FBXW7 mutated genomes that do not classify as TDP have different breakpoint profiles compared to their TDP counterparts, suggesting that additional genetic factors must be at play to synergize with cyclin driven replication stress in inducing TDP.Taken together, our analyses indicates that the different drivers for TDPs favor distinct genomic features for SV formation. This work provides novel insights in the mutational processes that shape the genomic landscape of cancer cells.


 Francesca Menghi, Edison T. Liu. Pancancer meta-analysis of the genomic features associated with tandem duplication formation in cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64a8bc0b12893b6190281e1fdb8cfc040b24528b" target='_blank'>
              Abstract LB045: Pancancer meta-analysis of the genomic features associated with tandem duplication formation in cancer
              </a>
            </td>
          <td>
            F. Menghi, Edison T. Liu
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Oncogenic fusion genes and amplified oncogenes are common drivers of cancer but there are currently no therapeutic approaches available that directly target these cancer-specific genomic features. Amplified oncogenes in cancer can be found on chromosomes as homologous staining regions (HSRs) or on extrachromosomal DNA (ecDNA). Presence of oncogenes on ecDNA poses a particular challenge because they are under strong positive selection in response to anti-cancer therapies leading to treatment resistance as their copy number can rapidly change due to uncontrolled segregation during cell division. To specifically target cells with fusions or amplified oncogenes, we have developed a CRISPR-based therapeutic approach, “KLIPP, ” which is designed to target structural variants junctions (SVJs) specific to cancer genomes, with few, or no off-target effects expected in normal cell. KLIPP uses a “split enzyme” approach consisting of a dead Cas9 endonuclease (dCas9) fused to the endonuclease Fok1 where two Fok1 endonucleases need to homodimerize to become active. To “nucleate” and activate these complexes, sgRNAs are designed to bind sequences flanking cancer-specific SVJs, bringing two Fok1-dCas9 complexes together to induce double-strand breaks (DSBs). Using the KLIPP approach, we have obtained strong proof-of-concept that it can specifically target cancer-unique SVJs and induce DSBs leading to cell death in cancer cells both in culture and in vivo. We have shown this for SVJs throughout cancer genomes as well as in oncogenic fusion genes such as EWS-FLI1 in Ewing sarcoma. Furthermore, we show that we can use KLIPP to target unique SVJs on amplified oncogenes resulting in strong induction of γH2AX and loss of cell survival. We are currently investigating the efficacy of KLIPP in targeting ecDNA in cancer cells. We believe that KLIPP could be a safe and cancer-specific precision therapy for targeting cancer cells driven by fusions and amplified oncogenes.


 Huibin Yang, Radhika Suhas Hulbatte, Natalie Gratsch, Ashley Sutter, Anne Urzynicok, Mayer Cusnir, Mario Ashaka, Aadi Shiv Malhotra, Ishwarya Venkata Narayanan, Michelle Paulsen, Anna Schwendeman, Tom Wilson, Mats Ljungman. KLIPP: Precision targeting of fusions and amplified oncogenes using CRISPR [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6371.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54f93c77ea0aca0c8fd7efd0d36e03e57f00addd" target='_blank'>
              Abstract 6371: KLIPP: Precision targeting of fusions and amplified oncogenes using CRISPR
              </a>
            </td>
          <td>
            Huibin Yang, R. Hulbatte, Natalie Gratsch, Ashley Sutter, Anne Urzynicok, Mayer Cusnir, Mario Ashaka, Aadi Shiv Malhotra, Ishwarya Narayanan, Michelle Paulsen, Anna Schwendeman, Tom Wilson, Mats Ljungman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bec087600fbb41ee812fa17d65823942d9127a6b" target='_blank'>
              BTRR complex deficiency is a driver for genomic instability in Bloom syndrome
              </a>
            </td>
          <td>
            Ipek Ilgin Gönenc, Alexander Wolff, A. Busley, A. Wieland, Andrea E. Tijhuis, Christian Müller, René Wardenaar, L. Argyriou, S. Kaulfuß, M. Räschle, D. Spierings, F. Foijer, Holger Bastians, G. Yigit, Arne Zibat, Lukas Cyganek, B. Wollnik
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 3090


 Background:
 Recurrent cytogenetic abnormalities represent candidate therapeutic targets for children with neuroblastoma (NB).
 MYCN
 oncogene amplification is associated with significantly worse survival rates for children with NB and remains one of the primary predictors of patient prognosis.
 MYCN
 amplifications in NB can be found both within the linear genome and on circular extrachromosomal DNA (ecDNA), and therapeutic targeting of the mechanisms underlying
 MYCN
 amplification represents a novel and promising strategy in NB. However, the molecular features and clinical and biological significance of these amplifications in NB tumors are not sufficiently understood.
 Methods:
 Whole genome and RNA sequencing data were analyzed for NB cell lines and NCI TARGET NB samples using AmpliconSuite software for ecDNA identification and characterization. GISTIC was used for identification of recurrently amplified regions. Gene expression levels were determined using StringTie, and gene clustering heatmaps were generated using FeatureCounts software. For differential gene expression analyses, samples were divided into ecDNA
 +
 and ecDNA
 -
 , and genes contained on ecDNA were compared to the same regions on linear DNA across samples using DESeq2. Associations between ecDNA quantity, content, and patient survival were performed using multivariate Cox regression survival analysis. Associations of gene expression with patient survival were performed using the R2 Platform. The efficacy of targeting ecDNA-associated gene products was assessed using live cell imaging and cell viability assays.
 Results:
 WGS analysis confirmed 7/20 NB patient tumors from the TARGET database to be ecDNA amplified with 1-5 independent ecDNA elements and
 MYCN
 gene expression correlated with the ecDNA copy number. ecDNAs in
 MYCN
 -amplified neuroblastoma cell lines contained distinct gene combinations and possessed unique structures. MYCN overexpression in NB cells has been shown to be associated with replication stress (RS), and tumor cells containing ecDNA are hyper-reliant on the DNA damage response (DDR) kinase CHK1 to manage heightened replication stress. Expression of the
 CHK1
 gene was associated with neuroblastoma patient outcomes and neuroblastoma was most significantly associated with
 CHK1
 RNA dependency. We further validated CHK1i as a promising therapeutic strategy in
 MYCN
 amplified NB, as CHK1 inhibition with the novel inhibitor BBI-2779 was most effective against ecDNA+,
 MYCN
 -amplified neuroblastoma cell lines.
 Conclusions:
 Our results emphasize the critical role of ecDNA in NB. We identify a synthetic lethality axis shaped by ecDNA
 MYCN
 amplification and CHK1 dependence. We further demonstrate the feasibility of targeting this vulnerability through CHK1 inhibition, thus offering new avenues for treatment in
 MYCN
 amplified tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e222ec29cad608a8fd4761ccb837028a1803c8a" target='_blank'>
              Effect of extrachromosomal DNA (ecDNA) on
 MYCN
 amplified neuroblastoma and patient outcomes.
              </a>
            </td>
          <td>
            Mihika Sonalkar, Carla Sampaio, Ting Yang, Taelor Getz, Catherine Shaw, Vicky Pham, Joshua Lange, Sudhir Chowdhry, Christian A Hassig, L. Alexandrov, Peter E. Zage
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="


 Undifferentiated sarcomas (USrc) are aggressive mesenchymal tumors that do not show characteristic features and therefore remain a diagnosis of exclusion. USrc typically have a low tumor mutational burden but are genomically complex, carrying many large structural variants (SVs) and copy number aberrations (CNAs). Improving diagnostic practice by adding novel whole genome-technologies based on high molecular weight DNA could increase the sensitivity for detecting genetic alterations in USrc, allowing identification of recurrent aberrations or patterns with clinical implications.



 Snap-frozen tissue from 41 USrc in 38 patients were analyzed by both conventional karyotyping and Optical Genome Mapping (OGM). Chromothripsis was determined based on OGM data and defined as more than eight copy number segments with oscillating copy number in co-occurrence with more than eight chromosomal rearrangements within 50 Mb, or more than 20 rearrangements within 50 Mb. These rearrangements should be interleaved and approximately half of them should be inversion-type aberrations. In addition, we identify known cancer genes (COSMIC Cancer Gene Census) which are amplified/lost or show evidence of disruptive SVs. DNA methylation profiling using EPIC arrays was done on 15 USrc followed by sarcoma classification with the Deutsches Krebsforschungszentrum (DKFZ) sarcoma classifier (v12.3).



 32 out of 41 USrc karyotypes showed at least subclonal whole genome-doubling with complex patterns of SVs and CNAs, including double minutes, ring chromosomes and giant markers. OGM was performed on all USrc biopsies yielding an average coverage of 352X. 25 out of 38 patient samples presented with chromothripsis with an average of 4.3 chromosomes affected (1-13 chromosomes). 70% of amplifications (CN greater than or equal to 5) and 83% of high, focal amplifications (CN greater than or equal to 10) occurred in regions with chromothripsis. Aberrations in 590 COSMIC cancer genes were detected. The most frequently altered cancer genes were CDKN2A (42%), RB1 (34%), PTPRD (29%), MGMT (26%), and ARHGEF12 (24%). For 3 patients, biopsies from two different time points were profiled, showing a high proportion of shared aberrations. DNA methylation patterns classified 8 out of 15 analyzed samples as USrc, and the remaining 7 showed no match to any reference sarcoma group. Copy number profiles derived from OGM and DNA methylation arrays were near identical. MGMT promotor hypermethylation was observed in 5 out of 16 cases.



 Implementation of OGM improves the detection of SVs and CNAs over standard-of-care. This is demonstrated by the detection of chromothripsis regions and novel recurrent aberrations in USrc. Further investigation to confirm these findings and to identify single nucleotide variants and allele-specific copy numbers are currently ongoing using long-read HiFi sequencing (PacBio).



 Jef Baelen, Raphael Sciot, Ruben Cools, Jonas Demeulemeester, Isabelle Vanden Bempt. Deep genomic analysis of undifferentiated sarcoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7177.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6d0de577d187a1d02372f19a6bb5a20995879b2" target='_blank'>
              Abstract 7177: Deep genomic analysis of undifferentiated sarcoma
              </a>
            </td>
          <td>
            Jef Baelen, Raphael Sciot, Ruben Cools, Jonas Demeulemeester, I. V. Bempt
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="


 Whole-genome doubling (WGD), marked by the duplication of all chromosomes, is widespread in cancer with significant prognostic implications. Traditionally, WGD tumors are assumed to exist in the tetraploid state. However, the evolutionary selection pressures following WGD and their effects on aneuploidy and patient outcomes remain poorly characterized.



 We analyzed 5, 980 tumor samples from 5, 807 patients with colorectal cancer treated at Memorial Sloan Kettering between 2014 and 2024. Tumors were sequenced using the targeted MSK-IMPACT clinical sequencing assay. Arm-level copy number variation (CNV) was inferred using FACETS, an allele-specific copy number algorithm. Copy number states were defined by the total copy number to minor copy number ratio, defining the diploid state as 2:1, triploid state as 3:1, and tetraploid state as 4:2. Tumors were classified as WGD-positive (WGD+) if >50% of their autosomal genome had a major copy number ≥2; otherwise, they were classified as WGD-negative (WGD−). The fraction of genome altered (FGA) for WGD+ tumors was calculated as the proportion of the genome deviating from the triploid state. Chromosome arms were assigned selection scores based on the potency and distribution of oncogenes and tumor suppressor genes.



 Both WGD- and WGD+ tumors exhibited selective arm-level gains and losses, but these pressures were weaker in WGD+ tumors (Pearson r=0.46 vs. 0.32). WGD+ tumors exhibited an average ploidy of 2.7, with the triploid state as the predominant allelic configuration, comprising 35.8% of the genome on average compared to the tetraploid state (9.7%). The triploid state more closely mirrored the arm-level selection patterns of the diploid state in WGD- tumors than the tetraploid state (Spearman r=0.88 vs. 0.27), a relationship further confirmed by clustering analysis of arm-level CNVs in microsatellite stable (MSS) colorectal cancers (n=4899). Using FGA as a measure of overall aneuploidy, we identified that MSS WGD+ colorectal cancer patients (n=1975) in the lowest FGA quartile had improved overall survival (OS) compared to those in the highest quartile (median OS: 28.1 vs. 24.8 months, p=0.003). Prior chemotherapy exposure was associated with a significant increase in WGD prevalence in MSS colorectal cancer patients with matched samples (n=90, 49% vs. 68%, p=0.015).



 Our study identifies the triploid state as the predominant and presumably biologically optimal allelic configuration of WGD+ tumors. The FGA metric highlights the clinical relevance of the triploid state in patient stratification, while the observed link between chemotherapy and increased WGD prevalence suggests potential therapeutic implications for colorectal cancer. These findings advance our understanding of WGD-driven aneuploidy and its role in cancer evolution and treatment strategies.



 Jierui Xu, Henry Walch, Paolo Manca, Maria Perry, Christopher Fong, Justin Jee, Karl Pichotta, Efsevia Vakiani, Neil H. Segal, Adam Price, Allison L. Richards, Chai Bandlamudi, Mark Donoghue, Andrea Cercek, Rona Yaeger, Nikolaus Schultz, Walid K. Chatila. The landscape of copy number variations following WGD in 5, 980 colorectal cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1152.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b54c36ea8dc8ae6d831b9cec1f08634da1f520c" target='_blank'>
              Abstract 1152: The landscape of copy number variations following WGD in 5,980 colorectal cancers
              </a>
            </td>
          <td>
            Jierui Xu, H. Walch, Paolo Manca, Maria A Perry, Christopher Fong, Justin Jee, K. Pichotta, E. Vakiani, Neil H. Segal, Adam Price, Allison L Richards, C. Bandlamudi, M. Donoghue, A. Cercek, R. Yaeger, N. Schultz, W. Chatila
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Background Cancer cells undergo widespread changes in epigenetic patterns that mediate cancer compromised gene expression programs during cancer progression. However, the alterations in higher-order genome organization in which these changes occur and their functional implications are less well understood. To explore how chromatin structure and epigenetic parameters of genome architecture changes during cancer progression at a fine scale and genome-wide, we generated high-resolution Micro-C contact maps in non-malignant, pre-cancerous, and metastatic MCF10 breast cancer epithelial cells. Results We profiled progression-associated reorganization of chromatin compartments, topologically associated domains (TADs), and chromatin loops, and also identified invariable chromatin features. We find large-scale compartmental shifts occur predominantly in early stages of cancer development, with more fine-scale structural changes in TADs and looping accumulating during the later transition to metastasis. We related these structural features to changes in gene expression, histone marks, and potential enhancers and found a large portion of diYerentially expressed genes physically connected to distal regulatory elements. While changes in chromatin loops were relatively rare during progression, diYerential loops were enriched for progression-associated genes, including those involved in proliferation, angiogenesis, and diYerentiation. Changes in either enhancer-promoter contacts or distal enhancer activity were accompanied by diYerential gene regulation, suggesting that changes in chromatin contacts are not necessary but can be suYicient for gene regulation. Conclusions Together, our results demonstrate a functionally relevant connection between gene regulation and genome remodeling at many key genes during cancer progression. KEY FINDINGS ‒ The cancer genome is reorganized throughout cancer progression at the level of compartments, chromatin domains, and loops ‒ Compartmental shifts occur in early stages of cancer development, with more fine-scale structural changes accumulating during metastasis ‒ Chromatin domain boundaries are weakened during cancer progression ‒ Many progression-regulated genes exhibit changes in distal enhancer histone modifications that are bridged by stable chromatin loops ‒ Changes in enhancer activity or subtle changes in chromatin contacts can rewire enhancer-promoter connections to facilitate changes in gene expression ‒ Prominent changes in chromatin loops occur at a small subset of diYerentially regulated genes during progression">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f24eeb9036338ab76b344cd3bcd26b76e9bfec03" target='_blank'>
              Genome reorganization and its functional impact during breast cancer progression
              </a>
            </td>
          <td>
            K. S. Reed, Andrew J. Fritz, Haley Greenyer, Kerstin Heselmeyer-Haddad, Seth Frietze, Janet Stein, Gary S. Stein, Tom Misteli
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Extrachromosomal DNA (ecDNA) presents a promising target for cancer therapy; however, its spatial-temporal diversity and influence on tumor evolution and the immune microenvironment remain largely unclear. Here, we apply computational methods to detect and reconstruct ecDNA from whole-genome sequencing data from 595 patients prospectively recruited into the Chinese Urothelial Carcinoma Genome Atlas (CUGA) study. We demonstrate that ecDNA contributes to clonal evolution through structural rearrangements and copy number variation during malignant transformation and recurrence of urothelial carcinoma (UC). This supports a model wherein tumors evolve via the selective expansion of ecDNA-bearing cells. Through multi-regional sampling of tumors, we demonstrate ubiquitous ecDNA presence across clones in multi-focal cancers, directly linking ecDNA to the evolution of multifocality. Notably, we observe highly variable levels of ecDNA across multiple foci within the same tumor, and ecDNA contributes to increased intratumoral heterogeneity and serves as a substrate for natural selection and evolution. EcDNA is present in 36% of UC tumors and correlates with an unstable cancer genome, an immunosuppressive phenotype, and significantly reduced 5-year survival. Single-cell RNA sequencing analyses reveal that ecDNA+ malignant cells exhibit diminished expression of major histocompatibility complex class I molecules, enabling them to evade T-cell mediated immune responses. Finally, we show that sequencing of urinary sediment-derived DNA has excellent specificity in detecting ecDNA, underscoring its diagnostic potential. This study delineates the evolutionary trajectory of ecDNA during tumor progression and provides further insights into immune evasion mechanisms associated with ecDNA.


 Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, Jens Luebeck, Huiying Tao, Yanfen Zhu, Xiaolong Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xiang Xi, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, Roel G. Verhaak, Bishoy M. Faltas, Jacob B. Hansen, Sihan Wu, Paul S. Mischel, Anton G. Henssen, Vineet Bafna, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han. Spatial-temporal diversity of extrachromosomal DNA shapes urothelial carcinoma evolution and tumor-immune microenvironment [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2668.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77c67aedf7052778400f6b8258e4d803910f72b0" target='_blank'>
              Abstract 2668: Spatial-temporal diversity of extrachromosomal DNA shapes urothelial carcinoma evolution and tumor-immune microenvironment
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, J. Luebeck, Huiying Tao, Yanfen Zhu, X. Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xiang Xi, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B. Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="


 Chromosomal instability (CIN) is the predominant form of genomic instability in cancers, characterized by both numerical and structural alterations of chromosomes. CIN is crucial for the most threatening aspect of cancers: their ability to evolve, allowing them to circumvent immune detection, metastasize, develop resistance to therapies, and lead to treatment failure. At the same time, the CIN phenotype could provide valuable targets for cancer therapy. We developed a multiplex FISH (miFISH) method to study heterogeneity and ongoing CIN in cancer cells. The objective of our research was to determine whether high-grade aneuploidy develops through sequential or abrupt changes in chromosomal content, a question previously unexplored due to the absence of comprehensive methods for simultaneously detecting different forms of ongoing CIN on a single-cell level.



 Sequential hybridizations of miFISH panels allow the assessment of up to 35 probes per single cell. Each panel was designed to include the centromeric probe and gene-specific probes on the p- and q-arm for a given chromosome. This approach was applied to colorectal cancer cell lines (with and without microsatellite instability) and colorectal cancer samples. A total of 10000-25000 targets were collected from each slide. Analysis was done by counting of signals in mononucleated cells, binucleated/dividing cells and metaphase spreads.



 The simultaneous evaluation of centromeric and gene-specific sets of miFISH probes led to unprecedented insights into the patterns of distribution of chromosomal content among individual cells, and, especially, in dividing cells. We compared different forms of ongoing CIN: structural rearrangements, formation of micronuclei with whole chromosomes or chromosomal fragments, non-disjunctions of chromosomes, grossly asymmetrical cell divisions, including divisions with ploidy changes and multipolar mitoses. We observed that gradual changes in chromosomal/genomic content occur primarily due to frequent structural rearrangements, less frequent chromosome or fragment losses in micronuclei, and very rare single chromosome non-disjunctions. Of note, grossly abnormal cell divisions, characterized by massive non-disjunctions and structural rearrangements, occur with relatively high frequency. The same patterns of chromosomal heterogeneity and instability have been observed in cancer cell lines and in tumor samples.



 The novel miFISH approach applied to studies of chromosomal instability in colorectal cancers and cell lines revealed multiple paths leading to chromosomal diversification of cancer cell populations. Grossly abnormal cell divisions are likely to contribute to high-level aneuploidies and ploidy changes, and could underlie the rapid evolution of cancer under selective pressure.



 Anna Roschke, Darawalee Wangsa, Nina Gutzeit, Richard Korshkov, Tim Thoerner, Kerstin Heselmeyer-Haddad, Paul Meltzer. Novel miFISH single-cell approach gives new insights into chromosomal instability and evolution of colorectal cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3890.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3a7d37cdffe8b6b0e4ee7704d2a0b59475d492a" target='_blank'>
              Abstract 3890: Novel miFISH single-cell approach gives new insights into chromosomal instability and evolution of colorectal cancers
              </a>
            </td>
          <td>
            Anna Roschke, Darawalee Wangsa, Nina Gutzeit, Richard Korshkov, Tim Thoerner, K. Heselmeyer-Haddad, Paul S. Meltzer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal cancer characterized by extensive genomic complexity, rapid metastasis, and resistance to therapy. Our research focuses on the role of APOBEC3A (A3A), a cytidine deaminase enzyme, in promoting chromosomal instability (CIN) and intratumoral heterogeneity (ITH) as mechanisms driving PDAC progression and metastasis. Preliminary findings reveal that A3A drives CIN in a deaminase-independent manner, as demonstrated by dose-dependent increases in structural variants, micronuclei formation, and chromosomal aberrations. Furthermore, A3A expression significantly correlates with early tumor dissemination, enhanced metastatic colonization, and reduced survival, as shown in both human and murine models.
 While previous studies have highlighted A3A's molecular effects, they lack copy number signature analysis to systematically characterize CIN. Addressing this gap, our study aims to define A3A-driven copy number signatures in PDAC, providing critical insights into its role in tumor evolution and vulnerability to therapeutic interventions. Using a multi-modal approach, we integrate data from genetically engineered mouse models, human PDAC tissues, and cancer cell lines, employing bulk whole-genome sequencing (WGS) and advanced bioinformatics pipelines to uncover the full scope of A3A’s contributions to tumor evolution. We aim to demonstrate how A3A-mediated genomic instability accelerates the transition from pre-neoplastic lesions to advanced PDAC.
 Our findings will elucidate how A3A-mediated genomic instability accelerates the progression from pre-neoplastic lesions to advanced PDAC. By addressing critical gaps in the understanding of CIN and ITH, this study positions A3A as a compelling therapeutic target to mitigate genomic instability, reduce tumor heterogeneity, and ultimately improve clinical outcomes for patients with pancreatic cancer.


 Zhihui Zhang, Chunxu Gao, Sonja Wörmann, Peter Van Loo. APOBEC3A drives chromosomal instabilityand tumor evolution in pancreatic cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7501.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4473ff7fede83ad4cee5cb350abfb1fe6b9f5c2d" target='_blank'>
              Abstract 7501: APOBEC3A drives chromosomal instabilityand tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Sonja Wörmann, P. Van Loo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="
 Cancer arises from genetic and/or epigenetic disruptions. Currently, no single technology can capture the full range of variation that drives tumorigenesis with high sensitivity. Furthermore, epigenetic changes resulting from somatic variation remain poorly understood. The LinkPrep™ technology, a multi-purpose NGS assay, generates unbiased linked-read libraries through DNA proximity ligation that enable the discovery of structural variants (SVs) while simultaneously capturing 3D chromosome conformation linked to the tumor’s epigenetic state. Importantly, LinkPrep data provides uniform coverage, allowing for the detection of single nucleotide variants (SNVs), insertions/deletions (InDels), and copy number variants (CNVs) with similar performance to whole genome shotgun sequencing (WGS). Libraries are generated from a single-day workflow and are compatible with Illumina sequencers.
 In a comparison study, a fresh-frozen, clinical ovarian serous adenocarcinoma sample was profiled with LinkPrep (30X), WGS (30X), and RNA-seq (100M reads). Additionally, the same sample stored in FFPE was profiled with Dovetail-FFPE (30X), a linked-read technology designed to extract long-range genomic interactions from FFPE tissue. LinkPrep & Dovetail-FFPE both identified three translocations (tlocs) at high SV-VAF (>30%) between chromosomes 1, 3, and 12. These tlocs were supported by strong evidence (LinkPrep: 589 - 3, 282 read pairs, Dovetail-FFPE: 417 - 5, 802 read pairs) and confirmed by copy number changes. The tlocs were further validated through PCR and Sanger sequencing. By comparison, RNA-Seq detected only one fusion consistent with tloc(1;12), and WGS identified two tlocs with low specificity (ranked 140 and 374 of 404 putative tlocs). Furthermore, LinkPrep detected a subclonal event present at SV-VAF of 4.5% at tloc (2;15). LinkPrep and WGS produced concordant variant calls that showed >95% overlap with the set of known variants. Similarly, copy number profiles between LinkPrep and WGS showed >98% similarity.
 To profile the promoter-enhancer genomic interactions at high coverage (200X), LinkPrep libraries were paired with the Dovetail® Pan Promoter Enrichment Panel. This revealed SV-driven neo-interactions, with interaction strength independent of SV VAF, suggesting biological mechanisms such as enhancer hijacking are at play.
 In summary, LinkPrep outperforms WGS and RNA-seq in SV discovery with superior sensitivity and specificity, while maintaining comparable performance to WGS in the detection of SNVs, InDels, and CNVs. Additionally, LinkPrep captures epigenetic information in the same library, providing insights into epigenetic changes and drivers. Taken together, LinkPrep is a powerful tool for comprehensive profiling of the cancer genome - capturing the whole range of genetic variants and the 3D genome in a single dataset.


 Ericca Stamper, Jon Torchia, Daniel Hwang, Chu Yuan Wu, Ek Danwanichakul, Cory Padilla, J. Zachary Sanborn, Lisa Munding. LinkPrep™ - a comprehensive assay for cancer genome characterization - enabling structural variant detection and discovery of enhancer hijacking [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6625.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf2305c32fc721604cdb952ddd1c89201b94faf9" target='_blank'>
              Abstract 6625: LinkPrep™ - a comprehensive assay for cancer genome characterization - enabling structural variant detection and discovery of enhancer hijacking
              </a>
            </td>
          <td>
            Ericca Stamper, Jonathon Torchia, Daniel Hwang, Chu Yuan Wu, Ek Danwanichakul, Cory Padilla, J. Z. Sanborn, Lisa Munding
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Tumors exhibit dynamic oxygen gradients, leading to hypoxia, which is linked to poor prognosis and genomic instability. The molecular features of hypoxia and their role in lung cancer outcomes remain unclear. Hypoxia has been linked to the forma on of fragile sites, resulting in chromosomal instability. Recent research has highlighted the generation of extrachromosomal DNA (EcDNA) because breaks. Genes amplified as EcDNA exist acentric circular structures capable driving rapid and significant genomic copy number amplifications. Inherent instability oncogene amplification on EcDNA provides tumor cells with a selective advantage adapting to changing environments. Thus, hypothesize that formation hypoxic tumors provide survivein unfavorable conditions like hypoxia contributes their poor prognosis. This study aims investigate impact disease outcomes formation. Furthermore, can act pressure, we examined its effects mutational burden, positive selection mutations, mutational signatures associated with hypoxia lung TRACERx cohort. TRACERx is multicenter prospective study collects samples from various regions of primary for high-depth next-generation sequencing. Analyzed bulk RNA sequencing data establish levels examine and EcDNA detection. Our preliminary findings reveal variability within between primary tumors, indicating inter- intratumoral hypoxia heterogeneity. High status correlates poorer overall disease-free survival in LUAD, but not LUSC. Additionally, presence significantly associated necrosis. Hypoxia increased circulating tumor (ctDNA) shedding (p < 0.05) and elevated whole-genome index (wgii) scores. High-hypoxia exhibit alterations, tumor suppressor b2m, increased expression immunosuppressive chemokines (cxcl8, ccl8, cxcl10). Furthermore, extrachromosomal was detected both LUAD average" hypoxia scores were -0.006 EcDNA-positive compared -0.248 EcDNA-negative samples, highlighting prevalence tumors. In conclusion, shows existence heterogeneity of hypoxia within cohort association clinical outcomes. We highlight link hypoxia, detection, the increased expression contributing a tumor microenvironment.


 Natasha Sharma, Chris Bailey, Corentin Richard, Jeanette Kittle, Charles Swanton, Mariam Jamal-Hanjani, Nnennaya Kanu. Hypoxia-driven genomic instability and EcDNA formation in lung cancer: insights from TRACERx [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1502.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df9dfba50807f4d056e7d436de2c34ed7ddd425f" target='_blank'>
              Abstract 1502: Hypoxia-driven genomic instability and EcDNA formation in lung cancer: insights from TRACERx
              </a>
            </td>
          <td>
            Natasha Sharma, Chris Bailey, Corentin Richard, Jeanette Kittle, C. Swanton, M. Jamal-Hanjani, N. Kanu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041a1ec7061bd7c816f107766afc5488bd6dd34f" target='_blank'>
              G-quadruplex stabilization induces DNA breaks in pericentromeric repetitive DNA sequences in B lymphocytes
              </a>
            </td>
          <td>
            Irina Waisertreiger, Kalkidan Ayele, Mehad Hilal Elshaikh, Jacqueline H. Barlow
          </td>
          <td>2025-05-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transposable elements (TEs) constitute over half of the human genome and have played a profound role in genome evolution. While most TEs have lost the ability to transpose, many retain functional elements that serve as drivers of genome innovation, including the emergence of novel genes and regulatory elements. Recent advances in experimental and bioinformatic methods have provided new insights into their roles in human biology, both in health and disease. In this review, we discuss the multifaceted roles of TEs in haematopoiesis, highlighting their contributions to both normal and pathological contexts. TEs influence gene regulation by reshaping gene-regulatory networks, modulating transcriptional activity, and creating novel regulatory elements. These activities play key roles in maintaining normal haematopoietic processes and supporting cellular regeneration. However, in haematological malignancies, TE reactivation can disrupt genomic integrity, induce structural variations, and dysregulate transcriptional programmes, thereby driving oncogenesis. By examining the impact of TE activity on genome regulation and variation, we highlight their pivotal roles in both normal haematopoietic processes and haematological cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae81192284f686304fb88058cdc41885e0ae02d1" target='_blank'>
              Transposable elements as genome regulators in normal and malignant haematopoiesis
              </a>
            </td>
          <td>
            D. Prokopov, Hale Tunbak, Eve Leddy, Bryce Drylie, Francesco Camera, Özgen Deniz
          </td>
          <td>2025-05-06</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6553455e0dd4bcf44406169f133238204316d7c1" target='_blank'>
              PIWIL2 downregulation in colon cancer promotes transposon activity and pro-tumorigenic phenotypes
              </a>
            </td>
          <td>
            Alyssa Risner, J. Nair-Menon, Abhinav Cheedipudi, Joe R Delaney, Vamsi K Gangaraju, A. Kourtidis
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Numerical chromosomal instability (n-CIN) is a defining characteristic of most human cancers with serious implications of large-scale alterations of the genome - such as chromosome arm level copy number changes. These kinds of somatic alterations are frequently targeting oncogenes and tumor suppressors, enriched in cancer driver events in order to shape the transcriptome in favor of tumor progression and proliferation. While oncogenes are mostly associated with copy number gains and tumor suppressors are more frequent in chromosome arm losses, neighboring genes are also affected in both cases. This collateral damage may be able to create synthetic lethal relationships and context-specific dependencies. Cancer cells might be able to avoid partial loss and gain-related lethal effects by triggering different dose compensation mechanisms, occurring at the genomic, epigenetic and post-translational levels, which leads to deviations across copy number and expression profiles.
 To propose a metric for dose compensation potential, we analyzed copy number and gene expression profiles of the Cancer Cell Line Encyclopedia in order to characterize cancer cell lines with the ability to compensate dependencies arising from chromosomal losses and gains. Our metric consists of the sum of gene expression residuals from linear model fits, describing how much a cell line tends to under or overexpress genes compared to what we expect from copy number counts. By comparing other publicly available datasets with our results, such as perturbation transcriptomics data from the LINCS-L1000 project and the DepMap genome-wide gene essentiality measurements, we aim to identify CIN-related therapeutic vulnerabilities associated with chromosome arm losses and gains. Besides finding potential target candidates, we show that our method is capable of finding well-known pathway-specific compensation mechanisms. As an example, we show a significant correlation between the activity of the Nonsense mediated decay (NMD) pathway and our proposed compensation potential score (CPS) of cancer cell lines.
 We propose our method to be used to detect context-specific dependencies associated with gene dosage compensation, and also able to provide pathway-level evidence - both novel and previously known from the literature.


 Péter Szikora, Ivan Fekete, María Victoria Ruiz-Pérez, Dóra Kállai, Csilla Hegedűs, Bence Szalai, Daniel Veres. Characterization of chromosomal instability-related dependencies through cell-specific metric for gene dosage compensation potential [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3718.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97245337195bdd219518d6c8012e2c0d61b49d80" target='_blank'>
              Abstract 3718: Characterization of chromosomal instability-related dependencies through cell-specific metric for gene dosage compensation potential
              </a>
            </td>
          <td>
            Péter Szikora, Iván Fekete, María Victoria Ruiz-Pérez, Dóra Kállai, Csilla Hegedűs, Bence Szalai, Daniel V Veres
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aneuploidy, a hallmark of cancer, leads to widespread changes in chromosome copy number, altering the abundance of hundreds or thousands of proteins. How-ever, evidence suggests that levels of proteins encoded on affected chromosomes are often buffered toward their abundances observed in diploid cells. Despite its preval-ence, the molecular mechanisms driving this protein dosage compensation remain largely unknown. It is unclear whether all proteins are buffered to a similar degree, what factors determine buffering, and whether dosage compensation varies across different cell lines or tumor types. Moreover, its potential adaptive advantage and therapeutic relevance remain unexplored. Here, we established a novel approach to quantify protein dosage buffering in a gene copy number-dependent manner, show-ing that dosage compensation is widespread but variable in cancer cell lines and in vivo tumor samples. By developing multifactorial machine learning models, we identify mean gene dependency, protein complex participation, haploinsufficiency, and mRNA decay as key predictors of buffering. We also show that dosage com-pensation can affect oncogenic potential and that higher buffering correlates with reduced proteotoxic stress and increased drug resistance. These findings highlight protein dosage compensation as a crucial regulatory mechanism and a potential therapeutic target in aneuploid cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a63d2bd35509beaab88a25ac6ce826723195385e" target='_blank'>
              Explainable Machine Learning Identifies Factors for Dosage Compensation in Aneuploid Human Cancer Cells
              </a>
            </td>
          <td>
            Erik Marcel Heller, Karen Barthel, M. Räschle, Klaske M. Schukken, J. Sheltzer, Zuzana Storchová
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Focal gene amplification serves as an oncogenic driver during tumorigenesis and is a hallmark of many forms of cancer. Oncogene amplifications promote genomic instability, which is integral to cancer cell survival and evolution. However, focal gene amplification potentially affords an opportunity for therapeutic exploitation. As a proof-of-concept, we leverage CRISPR-Cas9 nickase to selectively promote cancer cell death in MYCN-amplified neuroblastoma in a gene amplification-dependent manner. Our analysis demonstrates that CRISPR-Cas9 nickase can generate a lethal number of highly toxic, replication-dependent double-strand breaks in cells harboring amplified loci. Furthermore, we demonstrate that Cas9 nickase—mediated toxicity can be modulated in combination with small molecule inhibitors targeting key regulators of the DNA-damage response or cell death pathways. Importantly, our findings in MYCN-amplified neuroblastoma translate to other cancer types with distinct oncogene amplifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426fa411ebfe0306ef5fb3727f439d058ec49944" target='_blank'>
              Selective targeting of genome amplifications and repeat elements by CRISPR-Cas9 nickases to promote cancer cell death
              </a>
            </td>
          <td>
            Matthew B Hanlon, Jason M Shohet, S. Wolfe
          </td>
          <td>2025-06-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Chromosomal rearrangements involving additional material in individuals with phenotypic alterations usually result in partial trisomy, often accompanied by partial monosomy. To characterize chromosomal rearrangements and analyze genomic characteristics in the breakpoint regions in 31 patients with additional material on an autosomal chromosome. Different tests were performed to characterize these patients, including karyotyping, chromosomal microarray analysis (CMA), and fluorescent in situ hybridization (FISH). In silico analyses evaluated A/B chromosomal compartments, segmental duplications, and repetitive elements at breakpoints. The 31 rearrangements resulted in 47 copy number variations (CNVs) and a range of structural aberrations were identified, including six tandem duplications, 19 derivative chromosomes, two intrachromosomal rearrangements, one recombinant, two dicentric chromosomes, and one triplication. A deleted segment was associated with the duplication in 16 of the 19 patients with derivative chromosomes from translocation. Among the trios whose chromosome rearrangement origin could be investigated, 54,5% were de novo, 31,9% were maternally inherited, and 13,6% were paternally inherited from balanced translocations or inversion. Breakpoint analysis revealed that 22 were in the A compartment (euchromatin), 25 were in the B compartment (heterochromatin), and five were in an undefined compartment. Additionally, 14 patients had breakpoints in regions of segmental duplications and repeat elements. Our study found that a deletion accompanied by additional genetic material was present in 51.6% of the patients, uncovering the underlying genetic imbalances. Statistical analyses revealed a positive correlation between chromosome size and the occurrence of CNVs in the rearrangements. Furthermore, no preference was observed for breakpoints occurring in compartments A and B, repetitive elements, or segmental duplications. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-97077-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4680fc424cbc00a4561d2d03671779c2ded30f6" target='_blank'>
              Cytogenomic characterization of karyotypes with additional autosomal material
              </a>
            </td>
          <td>
            Priscila Soares Rodrigues, Bruna Burssed, F. Bellucco, Débora Cristina Batista Rosolen, Chong Ae Kim, M. Melaragno
          </td>
          <td>2025-04-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c3773603d3c9014e1c679ae73200f35f924d76" target='_blank'>
              Single-cell SNP-DNA sequencing precisely maps genotoxic events in CRISPR-edited primary cells
              </a>
            </td>
          <td>
            J. Boutin, S. Fayet, V. Marin, C. Berges, M. Riandière, Jérôme Toutain, I. Lamrissi‐Garcia, C. Thibault, David Cappellen, S. Dabernat, Arthur Poulet, Maëla Francillette, Nathalie Droin, F. Moreau-Gaudry, A. Bedel
          </td>
          <td>2025-04-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Extrachromosomal circular DNAs (eccDNAs) are circular, double-stranded DNA molecules ubiquitously present across various organisms, playing a critical role in tumorigenesis and tumor progression. However, their precise contribution to prostate cancer (PCa) remains incompletely understood. To elucidate the function of eccDNAs in PCa, eccDNA sequencing and annotation were performed in PCa tissues and cell lines using Circle-seq. Amplified genes on eccDNAs were identified by cross-referencing annotated eccDNA-associated genes with those overexpressed in PCa based on TCGA data. Furthermore, eccDNA profiles were compared between Enzalutamide-sensitive and -resistant cell lines to investigate their role in resistance mechanisms. Notably, VAV2 was detected on both linear and circular DNA, as confirmed by PCR and Sanger sequencing. Functional analyses demonstrated that VAV2 overexpression promotes PCa proliferation and metastasis by activating the PAK1/AKT signaling pathway through PAK1 phosphorylation. Additionally, VAV2 contributes to Enzalutamide resistance by enhancing AR/ARv7 protein stability via reduced ubiquitination, mediated through the recruitment of the deubiquitinating enzyme USP48. These findings establish VAV2, identified through eccDNA sequencing, as a potential oncogene and a promising biomarker for PCa diagnosis and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9feff4a41380afc1edc6d3d9d9371441107ea7a0" target='_blank'>
              VAV2 exists in extrachromosomal circular DNA and contributes Enzalutamide resistance of prostate cancer via stabilization of AR/ARv7
              </a>
            </td>
          <td>
            Qinghua Wang, Xin’an Wang, Hanchu Ye, Yicong Yao, Haopeng Li, Xin Qin, Yan Zhao, Wei Jiang, Mingming Xu, Tong Zi, Xilei Li, Xi Chen, Juan Zhou, Gang Wu
          </td>
          <td>2025-04-09</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Chromosomal translocations are complex biological processes that require the formation of DNA double-strand breaks (DSBs) at two different loci, followed by the joining of the broken ends by DNA repair mechanisms. In B-cell lymphoma, chromosomal translocations are directed by the on- or off-target activity of activation-induced cytidine deaminase (AID) that converts cytosine to uracil in single-stranded DNA (ssDNA), facilitating DSBs when two unpaired nicks are close together. However, no such enzymes have been discovered in solid tumors. While AID, a B-cell-specific APOBEC enzyme, is responsible for somatic hypermutations and chromosomal translocations in B-cell malignancies, apolipoprotein B mRNA editing enzyme, catalytic polypeptide-like 3 (APOBEC3; A3) enzymes have been primarily studied for their mutagenic potential in solid cancers. In this study, we present detailed characterizations of A3-induced chromosomal translocations in multiple cell models. While shared features of AID- and A3-driven translocations are expected because they deaminate cytosine at similar consensus sequences which should result in similar patterns of DNA DSB hotspots, their translocation patterns are partially distinct. As expected, AID generated focal DSB clusters (FDCs) near transcription start site (TSS) regions, but AID also induced new types of FDCs in the gene body and in both TSS and gene body, as well as dispersed DSB cluster (DDC)-type translocations. Interestingly, A3s enhanced FDCs and DDCs in fibroblasts with the similar level of AID, but A3s were only able to induce FDCs in the gene body, not in the TSS region. Both AID- and A3-mediated chromosomal translocations were enriched in genomic regions associated with early-replicating constant timing regions (CTRs), A compartments, and topologically associated domain (TAD) boundaries. Moreover, we revealed an enrichment of A3-attributed mutational signatures at the vicinity of A3-induced chromosomal rearrangements and the oncogene amplification boundaries in patient samples. Finally, we found that A3 expression significantly increased the number of oncogenic tyrosine kinase (TK) fusions, such as ALK, RET, ROS1, and NTRK1 fusions that are recurrently found in non-small cell lung cancer (NSCLC). Furthermore, deletion of APOBEC3 genes significantly decreased the number of oncogenic TK fusions. This study highlights that A3 enzymes induce chromosomal translocations and the formation oncogenic TK fusions in human cancers, with their translocation patterns distinct from but partially shared with those induced by AID, suggesting A3s as promising therapeutic targets for reducing genomic instability in cancer patients.


 Taek-Chin Cheong, Qi Wang, Ahram Jang, Elif Karaca-Atabay, Jake June-Koo Lee, Roberto Chiarle. Patterns of APOBEC3-mediated chromosomal rearrangements in solid cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2847.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1b5c0f4484941083873b791a9acfc4a1fd9d211" target='_blank'>
              Abstract 2847: Patterns of APOBEC3-mediated chromosomal rearrangements in solid cancers
              </a>
            </td>
          <td>
            T. Cheong, Qi Wang, Ahram Jang, Elif Karaca-Atabay, Jake June-Koo Lee, Roberto Chiarle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Oncogene activation is a key driver of cancer development, inducing aberrant cellular proliferation and DNA replication stress. This in turn, leads to DNA damage-which accumulates in specific genomic regions-contributing to genomic instability in cancer. However, the interplay between oncogene-induced DNA damage and chromatin organization is still poorly understood. In this study, we introduce a QUantitative ANalysis of DNA cOunterstains (QUANDO) to investigate the subnuclear localization of DNA damage in single-cell nuclei of U937-PR9 cells, an in vitro model of acute promyelocytic leukemia (APL). Using advanced imaging techniques, including DNA intensity analysis and colocalization by image cross-correlation spectroscopy (ICCS), we map DNA damage foci and correlate them with chromatin regions of different density. QUANDO is applied to dual-color confocal images of the DNA damage marker γ-H2AX and the DNA counterstain DAPI, allowing single-cell measurements of foci distribution within areas of low or high DNA density. We find that spontaneous DNA damage and DNA damage induced by the activation of PML-RARα oncogene predominantly localize in euchromatic regions. Conversely, when DNA damage is induced by the radiomimetic agent neocarzinostatin (NCS), the foci appear more evenly distributed in euchromatic and heterochromatic regions. These findings underscore the complex interplay between oncogene activation and chromatin organization, revealing how disruptions in DNA damage distribution can contribute to genomic instability and offering new insights for targeting DNA repair mechanisms in cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524273685b3a7b2a132bed81549714ac0e395f5e" target='_blank'>
              Location of oncogene-induced DNA damage sites revealed by quantitative analysis of a DNA counterstain.
              </a>
            </td>
          <td>
            Greta Paternò, Silvia Scalisi, G. Dellino, M. Faretta, P. Pelicci, Alberto Diaspro, Luca Lanzanò
          </td>
          <td>2025-05-07</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 Although immunotherapy has demonstrated promise in small cell lung cancer (SCLC), certain patients encounter limited benefits, highlighting the need for identifying immunosuppressive biomarkers. Extrachromosomal circular DNA (ecDNA) promotes amplification of MYC-paralogs, which drive cross-resistance in SCLC. Here, we aim to investigate whether ecDNA-mediated MYC-paralogs amplification (ecMYC+) represents immunosuppressive features in SCLC.



 Bulk RNA sequencing (RNA-seq) data were retrieved from public database and paraffin-embedded samples (n = 40). The copy number of MYC-paralogs was investigated using real-time polymerase chain reaction. RNA-seq, flow cytometry and metaphase fluorescence in situ hybridization (FISH) were performed in SCLC cell lines. The overexpression and amplification of MYC-paralogs in SCLC clinical samples (n = 42) were identified using immunohistochemistry (IHC) and FISH. The spatial distribution of tumor immune microenvironment (TIME) was characterized by multiplex immunofluorescence and imaging mass cytometry (IMC).



 The mean copy number of ecDNAs was higher in SCLC than in the other cell lines (15.3 vs 8.4, p = 4.0×10-3), and the frequency of ecMYC+ cell lines was higher in SCLC than that in the other lineages (SCLC 22/47 vs others 15/282, p = 3.9×10 -12). In ecMYC+ SCLC, multiple immune-related pathways were down-regulated, while nucleotide metabolism processes were up-regulated. Inhibition of nucleotide metabolism by hydroxyurea and gemcitabine resulted in the reduction of ecDNA, along with the activation of antigen presenting pathways. Immunohistochemical analysis revealed that overexpression of MYC-paralogs was observed in 19 out of 42 resected SCLC samples. FISH identified 8 samples as MYC-paralogs copy number gain or amplification, and 3 samples exhibited a scattered signals distribution resembling ecDNA amplification. Through the resolution of 103, 341 cells from 24 pathological regions, we observed higher expression of MKI67, VEGFA, FAP and FOXP3 and reduced T cell infiltration in ecMYC+ samples. Moreover, ecMYC+ samples exhibited elevated cellular neighborhoods dominated by Ki67+ tumors, with reduced spatial interaction with immune cells.



 Extrachromosomal amplification of MYC-paralogs shapes suppressive tumor microenvironment, identifying potential subgroup of immunotherapy resistant patients.



 Jingwei Zhang, Yueqi Jin, Ence Yang, Xiao Li. Extrachromosomal MYC-paralogs amplification shapes suppressive tumor microenvironment in small cell lung cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5227.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c127048e0d0f262eff5e6fdada1e6625367af0dd" target='_blank'>
              Abstract 5227: Extrachromosomal MYC-paralogs amplification shapes suppressive tumor microenvironment in small cell lung cancer
              </a>
            </td>
          <td>
            Jingwei Zhang, Yueqi Jin, Ence Yang, Xiao Li
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Retrotransposon mobilization in germline cells enables the rewriting of genetic information to drive genome innovation, species evolution, and adaptation through the generation of de novo mutations. However, uncontrolled mobilization can cause DNA breaks and genome instability, often leading to sterility. How germ cells balance retrotransposon-induced genome innovation with the need for genomic integrity remains poorly understood. Here, we used Drosophila spermatogenesis as a model to investigate retrotransposon mobilization dynamics. Although many retrotransposon families are transcriptionally active, we found that the LTR-retrotransposon nomad completes the full mobilization cascade—including mRNA export, protein translation, and reverse transcription—to produce double-stranded DNA (dsDNA) the most efficiently. Strikingly, despite successfully generating dsDNA, nomad rarely achieves genomic reintegration. Instead, its newly synthesized DNA predominantly forms extrachromosomal circular DNA (ecDNA). These findings suggest that ecDNA formation acts as a protective mechanism to sequester retrotransposon-derived DNA and prevent widespread genomic integration during spermatogenesis, thereby preserving genome stability while allowing limited retrotransposon activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e357c0e1274581f338acdf17e4b08fe74b5c828" target='_blank'>
              Safeguarding spermatogenesis from retrotransposon insertions by forming ecDNA
              </a>
            </td>
          <td>
            Lauren Tracy, Z. Zhang
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 KRAS G12C inhibitors (G12Ci), including the FDA-approved Sotorasib and Adagrasib, represent a significant advancement in targeted cancer therapies. As their use in non-small cell lung cancer patients expands, addressing the pervasive challenge of drug resistance is critical. Resistance mechanisms can be innate, acquired, or both, often coexisting within complex scenarios. Recent studies have highlighted various resistance mechanisms to G12Ci, such as oncogene amplification, activation of alternative signaling pathways, additional mutations in KRAS or its associated networks, and adaptive processes like epithelial-to-mesenchymal transition. Identifying the primary drivers and predominant mechanisms of resistance is crucial for developing effective countermeasures. To tackle these challenges, we developed Oligo-CALL, an innovative cellular barcoding technology that employs antisense oligonucleotide-inducible CRISPRa for lineage labeling. This technique allows for precise clonal tracking and the retrospective isolation of clones. Using Oligo-CALL, we analyzed tumor clonal heterogeneity and traced the evolution of resistance in lung cancer cells treated with G12Ci. Treatment with Sotorasib resulted in reduced clonal diversity, indicative of strong selective pressure. Similar clonal compositions across different replicates suggest that intrinsic mechanisms significantly contribute to G12Ci resistance. Notably, about 30% of resistant clones showed increased expression of the Kras G12C allele, primarily due to genomic gains, pointing to chromosomal instability (CIN) as a key factor in resistance development. Comprehensive analyses including fluorescent karyotyping, whole-genome, and transcriptome sequencing of isolated tumor clone pairs, revealed extensive genomic complexity and increased heterogeneity under G12Ci treatment. Resistant clones exhibited specific increases in copy number variations, chromosomal structure variations, and additional single nucleotide mutations, further underscoring CIN as a driving force behind G12Ci resistance. Clones that underwent drug treatment displayed higher CIN scores compared to their treatment-naïve counterparts. Intriguingly, cells resistant to G12Ci demonstrated increased sensitivity to PARP inhibitors, suggesting that targeting DNA repair pathways could induce synthetic lethality, offering a potential strategy to overcome resistance in lung cancer cells.


 Dingcheng Gao, Yingzhuo Liu, Yi Ban. Chromosomal instability and oncogene dosage variations underlying KRAS G12C inhibition therapy resistance in lung cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 419.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb94d2d1b79c9a0abe29eed20fd570bda0af9d0" target='_blank'>
              Abstract 419: Chromosomal instability and oncogene dosage variations underlying KRAS G12C inhibition therapy resistance in lung cancer
              </a>
            </td>
          <td>
            Dingcheng Gao, Yingzhuo Liu, Yi Ban
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="PURPOSE
Immunotherapy has demonstrated promise in small cell lung cancer (SCLC), but certain patients encounter limited benefits, highlighting the need for immunosuppressive biomarkers. Extrachromosomal circular DNA (ecDNA) promotes amplification of MYC-paralogs (MYC, MYCN and MYCL), driving cross-resistance in SCLC. Here, we aim to investigate whether ecDNA-mediated MYC-paralogs amplification (ecMYC+) represents immunosuppressive features in SCLC.


EXPERIMENTAL DESIGN
Bulk RNA sequencing data were retrieved from public database and paraffin-embedded samples. The overexpression and amplification of MYC-paralogs were identified using immunohistochemistry and fluorescence in situ hybridization. Imaging mass cytometry and multiplex immunohistochemistry were used to characterize spatial distribution of tumor immune microenvironment (TIME). The copy number of MYC-paralogs was investigated using real-time polymerase chain reaction. RNA-sequencing and flow cytometry were performed in SCLC cell lines.


RESULTS
The mean copy number of ecDNAs and the frequency of ecMYC+ cell lines were higher in SCLC than that in the other lineages (SCLC 22/47 vs others 15/282). In ecMYC+ SCLC, multiple immune-related pathways were downregulated while nucleotide metabolism processes were upregulated. Inhibition of nucleotide metabolism induced ecDNA elimination, along with activated antigen presenting pathways. Highly dispersed MYC-paralogs amplifications were detected in resected treatment-naïve SCLC samples. Through the resolution of 103,341 cells from 24 pathological regions, we observed higher expression of MKI67, VEGFA, FAP and FOXP3 and reduced T cell infiltration in ecMYC+ samples. Moreover, ecMYC+ samples exhibited elevated cellular neighborhoods dominated by Ki67+ tumors, with reduced spatial interaction with immune cells.


CONCLUSIONS
Extrachromosomal amplification of MYC-paralogs shapes suppressive TIME, identifying potential subgroup of immunotherapy resistant patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/202c6e4f5a1bfae884ce6819555bba6dc3de7227" target='_blank'>
              Amplification of extrachromosomal MYC paralogs shapes immunosuppressive tumor microenvironment in small cell lung cancer.
              </a>
            </td>
          <td>
            Jingwei Zhang, Yueqi Jin, Haodong Lin, Jiaming Deng, Yan Ju, Xueyan Hu, Jianqi She, Zhijian Liang, Kongxu Dai, Mantang Qiu, Kunkun Sun, Jun Wang, Fan Yang, Jian Chen, Ence Yang, Xiao Li
          </td>
          <td>2025-04-29</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Chromosomal 11q13.3 amplification is the most common gene copy-number variation event in head and neck squamous cell carcinoma (HNSCC) that corresponds with poor prognosis. Although cyclin D1, a G1/S phase cell-cycle regulatory protein at this locus, is considered as a key driver of malignant progression, further exploration is needed to develop more effective targets for cases with this amplification. Using CRISPR-based gene knockout screening of genes located in chr11q13.3, we found that loss of the gene encoding the Fas-associated death domain (FADD) protein, a well-recognized adapter to caspase-8 that induces cell apoptosis, significantly reduced cancer cell proliferation. FADD expression was elevated in chr11q13.3-amplified tumors and correlated with poor prognosis. RNA sequencing, mass spectrometry, and proteomics analyses revealed a direct relationship between FADD and the DNA helicase MCM5 in the S phase. FADD and cyclin D1 acted at different stages of the cell cycle to synergistically induce proliferation, and caspase-8 deficiency was required for the oncogenic activity of FADD. In a patient-derived xenograft model with chr11q13.3 amplification, combined administration of the DNA helicase complex inhibitor and CDK4/6 inhibitor effectively curtailed tumor growth. Overall, this study identified a nonclassic oncogenic role for FADD in mediating tumor progression in HNSCC and provided a feasible treatment option for patients with chr11q13.3 amplification. Significance: FADD promotes progression of tumors with chr11q13.3 amplification by binding to the DNA helicase complex, which can be targeted in combination with cyclin D1 as a viable therapeutic strategy for HNSCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/659d7605f636160fd781d9f5d4b484dd97ec74c4" target='_blank'>
              FADD Functions as an Oncogene in Chr11q13.3-Amplified Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yang Zheng, Yinan Chen, Xiaoyan Meng, Li Zhang, Yanni Ma, Rong Zhou, Shuiting Fu, Heng Chen, Xinyang Xuanyuan, Ruixin Jiang, Pengcong Hou, Xiaomeng Song, Yanqiu Wang, Jingjing Sun, Wuchang Zhang, Jiang Li, Zhonglong Liu, Zhiyuan Zhang, Hanlin Zeng, Yue He
          </td>
          <td>2025-02-27</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Copy number alterations (CNAs), involving the acquisition and loss of DNA segments, are common genetic changes associated with cancer initiation and genomic instability (GI). Similar to single base substitutions, CNAs in cancer genomes often accumulate non-randomly, linked to aberrant DNA damage and repair mechanisms. Investigating higher-order CNA patterns, especially in CNA-prone malignancies like breast cancer, offers insights into GI's nature, causes, and clinical implications. While GI is historically described using predefined scores or de novo CNA signature extraction, these approaches haven't been comprehensively combined and profiled in genomically diverse diseases such as breast cancer. Fourteen pre-existing CNA scores depicting diverse GI phenomena were reintegrated within a unified computational framework. Applied to a large cohort of breast cancer patients (n = 2, 763) from TCGA and METABRIC, this framework, along with de novo CNA signature extraction, identified eight robust signatures. CNA scores and signature activities were associated with genomic features and clinical information, including subtype and patient outcomes. Breast cancer oncogenes such as ESR1, CCND1, and IKBKE were preferentially amplified by tandem duplication in a subtype-dependent manner. Additionally, two CNA signatures were characterized as harboring relatively few CNAs ("CNA quiet") within diploid and tetraploid genomes, respectively, with the latter representing an unreported subset of whole genome doubled tumors. Samples with a combination of CNA quiet genomes and low macrophage infiltration exhibited significantly better survival outcomes, independent of standard clinical risk factors. Intriguingly, several signatures coincided with the inactivation of BRCA1/2, genes critical for proficient homologous recombination (HR). These signatures showed differential activity in samples lacking BRCA1 and BRCA2, indicating a potential gene-specific CN pattern of HR deficiency. Segments assigned to another signature exhibited substantial overlap with chromothripsis-like patterns and extrachromosomal DNA (ecDNA), suggesting a common process underlying the generation of these complex GI phenomena. These findings emphasize how exploring scores and signatures can elucidate diverse mechanisms underlying CNA-driven GI in breast cancer. The relative activities of these signatures may hold notable prognostic potential, especially when integrated with other clinically relevant data modalities such as immune cell infiltration. Overall, comprehensive profiling of higher-order CNA patterns in breast cancer can not only uncover the diversity and sources of GI but also provides translational impact.


 Hannan Wong, Anna Korsakova, Andy Wu, Akila Perera, C. Pawan K. Patro, Shangying Chen, Shwetha Sundararaman, Vinay Warrier, Daniel G. Tenen, Jason J. Pitt. Copy number-based instability in breast cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2410.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5393be1b38bee2c2ba8ad230d2f832b5bf465a65" target='_blank'>
              Abstract 2410: Copy number-based instability in breast cancer
              </a>
            </td>
          <td>
            Hannan Wong, Anna Korsakova, Andy J. Wu, Akila Perera, C. P. K. Patro, Shangying Chen, Shwetha Sundararaman, Vinay Warrier, Daniel G. Tenen, Jason J. Pitt
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Advancements in drug design have resulted in the development of approved and investigational agents targeting KRAS-driven tumors. RIT1 is a GTPase protein that shares homology and oncogenic activation mechanisms with KRAS and plays a role in cell division and resistance to apoptosis. In this study, we aimed to characterize the landscape of RIT1 alterations across a pan-cancer cohort to assess its frequency and potential as an oncogenic driver.



 Tumor content, ploidy and allelic specific copy number of 86, 039 tumor-normal matched samples sequenced clinically with a targeted sequencing panel (MSK-IMPACT) were analyzed with FACETS. Low purity (<20%) and indetermined copy number samples were excluded. Presence of RIT1 amplifications (Amp), structural variants, oncogenic mutations (Mut), and concurrent OncoKb oncogenic/likely-oncogenic alterations in other genes were evaluated. Co-occurrence and mutual-exclusivity (Co-Me) analyses were performed using DISCOVER. RIT1-altered samples were evaluated for genomic complexity compared to wild-type (WT) samples. False discovery rate correction was applied when appropriate.



 After removing low-quality samples, 63, 270 cancer samples were included, and 1.32% (n=838) were found to harbor RIT1 oncogenic alterations. Most RIT1-alterations observed were Amps (735/838) followed by somatic mutations (Muts) (83/838), concomitant Mut and Amp (2/838) and structural variants (18/838). The most frequent RIT1 Mut was p.M90I (n=40), followed by p.F82C (n=17), p.A77T (n=11) and R122Q (n=9).Uterine carcinosarcomas (5.5%), hormone receptor-positive breast cancer (4.9%) and triple-negative breast cancer (4.7%) had the highest prevalence of RIT1 alterations. Restricting the analysis only to Muts, lung adenocarcinoma (LUAD) (n=15; 0.22%) and bladder cancers (n=13; 0.51%) had the highest number of oncogenic RIT1 Muts. RIT1-mutant samples had higher TMBs than RIT1-WT samples (p<0.01). This association was conserved when accounting for histology in bladder cancer and LUAD (p<0.01). RIT1 Amps were associated with higher fraction of genome altered (p<0.01) and co-occurred with amps of neighboring genes NTRK1 (p<0.01) and AKT3 (p<0.01) across most tumor types. DISCOVER assisted Co-Me analysis identified RIT1 Muts as mutually exclusive with EGFR Muts in LUAD (p<0.01). Among RIT1-mutant samples, 10/15 (66%) LUADs harbored RIT1 as the sole identifiable oncogene driver alteration, whereas only 3/13 (23%) bladder cancers were otherwise driver negative.



 RIT1 alterations appear to be rare across most tumor types. Higher TMB and rarity of RIT1 Muts and the observation of RIT1 Amps frequently co-amplified with other chromosome 1 arm q genes, suggest that these alterations might be primarily passengers. These results have implications for stakeholders interested in developing potential therapies for RIT1 as the landscape of these alterations has not been previously explored.



 Matteo Repetto, Pier Selenica, Soo-Ryum Yang, Britta Weigelt, Alexander Drilon. The pan-cancer landscape of RIT1 genetic alterations [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7145.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0db92d7678fd17061debda8f9cf53b805a41693" target='_blank'>
              Abstract 7145: The pan-cancer landscape of RIT1 genetic alterations
              </a>
            </td>
          <td>
            Matteo Repetto, P. Selenica, S. Yang, B. Weigelt, A. Drilon
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="
 Breast cancer, characterized by considerable heterogeneity, showcases genetic and epigenetic variability among individual tumors and their constituent cells. Notably, this intratumoral heterogeneity (ITH) can drive clonal evolution and therapy resistance, particularly in Human Epidermal Growth Factor Receptor 2 (HER2)-positive cancers. Despite the efficacy of HER2-targeted therapies, many patients develop resistance, resulting in poor prognosis. This study aims to elucidate the dynamic interplay between genetic alterations and epigenetic modifications during anti-HER2 therapy using advanced single-cell resolution techniques. Biopsy samples were collected at three time points from patients undergoing neoadjuvant HER2-targeted therapy as part of the TRIO-US B07 clinical trial. Utilizing methods such as low-pass whole-genome sequencing (WGS, n=47), single-nuclei WGS (snWGS, n=21), and matched single-nuclei RNA/ATAC-seq (n=22), we explored copy-number variations (CNVs), subclonal tumor dynamics, and alterations in the tumor microenvironment (TME). We also analyzed additional deep WGS (n= 205) to reconstruct the structure of HER2 amplicons. Our analysis revealed distinct CNV dynamics in HER2-positive breast cancers during treatment. We found that while broad-level CNVs remained stable, focal amplifications exhibited temporal variability, suggesting the presence of extrachromosomal DNA (ecDNA). Specifically, the structure of HER2 amplificons emerged as a pivotal determinant of aggressiveness, with higher numbers of co-amplified oncogenes correlating with non-pathological complete response (pCR) and metastasis. Single-cell resolution analyses unraveled significant subclonal diversity in a subset of patients exhibiting complex tumor architectures. Integration of transcriptomic and epigenomic profiling highlighted that pCR patients exhibited higher lymphocyte infiltration and dynamic immune shifts post-treatment. Conversely, non-pCR tumors displayed increased regulatory T cells. Furthermore, fibroblast state transitions characterized by altered extracellular matrix (ECM) gene expression suggested a potential role in modulating immune responses. Our findings provide novel insights into the genetic and epigenetic factors driving therapy resistance in HER2-positive breast cancer. We emphasize the importance of longitudinal, multimodal profiling to refine patient stratification and inform personalized treatment strategies. Overall, this study offers new avenues for understanding tumor evolution in response to therapy and highlights the need for comprehensive assessments of ITH and microenvironmental dynamics for improved clinical outcomes.


 Seongyeol Park, Eran Kotler, Zhicheng Ma, Kathleen Houlahan, Aziz Khan, Sophia Pribus, Christina Curtis. Mapping the genetic, epigenetic, and microenvironmental evolution of HER2-positive breast cancer during neoadjuvant therapy [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6355.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce63b80e10f58d9373f4a7b7bf33e6cdea133649" target='_blank'>
              Abstract 6355: Mapping the genetic, epigenetic, and microenvironmental evolution of HER2-positive breast cancer during neoadjuvant therapy
              </a>
            </td>
          <td>
            Seongyeol Park, Eran Kotler, Zhicheng Ma, Kathleen E. Houlahan, Aziz Khan, Sophia Pribus, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 The extracellular vesicle (EV) route is essential for cell-to-cell communication. Cancer cells release EVs in the extracellular space, where they can interact with cancer and non-cancer cells, activating specific signaling pathways, modulating tumor microenvironment remodeling and inducing gene expression alterations. We investigated the functional role of EVs released by cancer of unknown primary (CUP), a rare disease (1-3% of novel cancer diagnoses) that presents with metastasis of unknown or uncertain origin and no apparent primary tumor. We derived three CUP cell lines from patient’s tumor cells, CUP#55A and S and CUP#96, both characterized by FGFR2 gene amplification, either in the form of trisomy (CUP#55A), chromosomal homogeneously staining region (CUP#55S) or double minute chromosomes (CUP#96). It has been recently recognized that tumors use extrachromosomal circular DNA (eccDNA) as a way to increase oncogenic amplification, thus conferring resistance to therapy and contributing to a worse survival. We demonstrated that FGFR2 amplification in CUP cell lines is associated with ecDNA generation and that this ecDNA is loaded as cargo inside EVs and exert a functional activity in nearby cells. The full-length FGFR2 DNA was detectable inside both small and large vesicles isolated from cell culture medium and we confirmed the circular nature of a fraction of this FGFR2 ecDNA using DNAse based strategies and visualizing DNA circles in cells and in EVs with atomic force microscopy. The protein surface profile of small and large EV populations revealed the presence of cancer related molecules that mirrored the cells of origin, including EpCAM and CD44, and variable degrees of FGFR2pos EVs. Cancer (NCI-N87) and non-cancer cell exposure (THP1 and HUVEC) to CUP EVs (by direct administration and co-culture) revealed an increase of FGFR2 DNA copies at 24 hours and its functional transcription in FGFR2 mRNA after 48 hours in all exposed cells. In addition to oncogenic delivery, CUP#96 EVs induced a polarization of THP1 cells towards M2 subtype, while CUP#55S EVs exerted a proliferative effect on HUVEC. In conclusion, we identified a model of oncogene amplification and EV-mediated delivery recurrent in cancer of unknown primary, whose DNA content was demonstrated to be functionally active in recipient cells and proportional with extrachromosomal DNA generation. This mechanism could contribute to the high metastatic potential of this cancer type. (The research leading to these results has received funding from AIRC under IG 2021 - ID. 25789 project - P.I. Ferracin Manuela)


 Irene Salamon, Giulia Gallerani, Gianluca Storci, Beatrice Fontana, Salvatore Serravalle, Francesco Valle, Marco Brucale, Marco Pagano Mariano, Andrea Cavazzoni, Roberta Roncarati, Spartaco Santi, Massimiliano Bonafè, Manuela Ferracin. Extracellular vesicle delivery of functional extrachromosomal DNA in FGFR2-amplified cancer of unknown primary [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6569.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98dbff58811770a1291ecf64b12b67c6f3cebb6f" target='_blank'>
              Abstract 6569: Extracellular vesicle delivery of functional extrachromosomal DNA in FGFR2-amplified cancer of unknown primary
              </a>
            </td>
          <td>
            Irene Salamon, Giulia Gallerani, G. Storci, Beatrice Fontana, Salvatore Serravalle, Francesco Valle, M. Brucale, Marco Pagano Mariano, Andrea Cavazzoni, R. Roncarati, Spartaco Santi, Massimiliano Bonafé, M. Ferracin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 e15108


 Background:
 Children with high-risk and relapsed neuroblastoma (NB) need improved therapies, and recurrent cytogenetic abnormalities, such as
 MYCN
 oncogene amplification, represent candidate therapeutic targets.
 MYCN
 amplification and increased
 MYCN
 expression drive deregulated hyper-transcription that leads to development and growth of NB tumors, and
 MYCN
 amplifications, which can be found both within the linear genome (HSR) and on circular extrachromosomal DNA (ecDNA), are associated with significantly worse survival rates for children with NB.
 MYCN
 overexpression has been linked to an increase in replication stress (RS), and RS and subsequent genome instability are important drivers of tumor initiation and progression. Flap Endonuclease 1 (FEN1), a non-essential DNA replication enzyme, was identified as a synthetic lethal target in
 BRCA1
 /
 2
 -deficient cancers via induction of RS. Recent success of targeting RS-elevated cancers with replication enzyme inhibitors opens a new avenue to target
 MYCN
 -amplified NB.
 Methods:
 Associations of gene expression with patient survival and prognostic features were performed on available neuroblastoma tumor databases using the R2 Genomics Analysis and Visualization Platform. The efficacy of FEN1 inhibition was assessed using live cell imaging and cell viability assays, comparing results in
 MYCN
 -amplified to -nonamplified NB cells and in NB cells with inducible
 MYCN
 expression and repression. Mechanisms of cell death and impacts on replication stress in cells treated with FEN1 inhibitors were evaluated by Western blots.
 Results:
 We have found that
 FEN1
 expression levels are associated wtih NB patient outcomes and with features of high-risk disease, including tumor stage and
 MYCN
 amplification. FEN1 inhibition was effective against NB cells and was significantly more effective in
 MYCN
 -amplified NB cells causing reduced cell growth and viability. Increased
 MYCN
 expression also led to increased sensitivity to FEN1 inhibition, and reduced MYCN expression reduced sensitivity. FEN1 inhibition led to the induction of apoptosis but not necroptosis in NB cells and responses to FEN1 inhibition were associated with markers of replication stress, including activation of the ATR-CHK1 and ATM-CHK2 pathways.
 Conclusions:
 We have discovered that
 MYCN
 -amplified NB cells are hypersensitive to FEN1 inhibition, suggesting that FEN1 inhibition may be a promising therapeutic strategy for children with high-risk and relapsed neuroblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa405fa9ce38d24eb7641c652f857fc594ac4869" target='_blank'>
              Mechanisms of efficacy of endonuclease inhibition in neuroblastoma.
              </a>
            </td>
          <td>
            Carla Sampaio, Eric Wu, Madison Cinelli, Erica Steen, Andrew Shiau, Richard Kolodner, Jean Yin Jen Wang, Peter E. Zage
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Gastroesophageal adenocarcinomas (GEA) are characterized by chromosomal instability (CIN) predominantly consisting of high amplitude focal amplifications of oncogenes, including Cyclin E1. An initial pan-cancer analysis revealed that CCNE1 is one of the top genes whose expression correlates with focal CIN.



 We proposed a causal role for excess CCNE1 as a driver of CIN and gastroesophageal tumorigenesis, and investigated the therapeutic vulnerabilities of CCNE1 amplification/overexpression.



 We engineered a novel CCNE1-overexpressing mouse gastric organoid model with Trp53 deletion and studied the functional effects on tumor initiation and progression. Whole genome sequencing of tumors was performed to characterize the effects of CCNE1 overexpression on genomic instability. To identify dependencies of CCNE1-amplified GEA, we executed a CRISPRi functional genomic screen. Top genetic dependencies were translated into potential pharmacologic vulnerabilities and tested in monotherapy and combination.



 CCNE1 overexpression accelerated tumor initiation and progression of mouse gastric organoids with Trp53 deletion. These tumors demonstrated increased chromosomal complexity, mainly focal amplifications. Our functional genomic screen identified specific genetic dependencies of CCNE1-amplified GEA, including CCNE1/CDK2 as the strongest hits, and SKP2 as a novel dependency likely through its function as a p27 degrader. Pharmacologic inhibition of CDK2 was insufficient and antagonistic with concurrent inhibition of the DNA damage response via targeting the ATR/CHEK1 axis. However, staggered inhibition of these distinct vulnerabilities demonstrated efficacy.



 Our study identifies CCNE1 as a driver of CIN in GEA and builds a foundation for further exploration of therapies for CCNE1-amplified GEA characterized by genomic instability.



 Zhong Wu, Ankur Nagaraja, Shangwei Sun, Ling Lan, Jin Zhou, Adam Bass. CyclinE1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2834.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a0c88288c8c7bf7b80395b40fbfcb9b8e990950" target='_blank'>
              Abstract 2834: CyclinE1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Shangwei Sun, Ling Lan, Jin Zhou, A. Bass
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Most human cancers arise from somatic alterations, ranging from single nucleotide variations to structural variations (SVs) that can alter the genomic organization. Pathogenic SVs are identified in various cancer types and subtypes, and they play a crucial role in diagnosis and patient stratification. However, the studies on structural variations have been limited due to biological and computational challenges, including tumor heterogeneity, aneuploidy, and the diverse spectrum of SVs from simpler deletions and focal amplifications to catastrophic events shuffling large fragments from one or multiple chromosomes. Long-read sequencing provides the advantage of improved mappability and direct haplotype phasing. Yet, no tool currently exists to comprehensively analyze complex rearrangements within the cancer genome using long-read sequencing.
 Here, we present Severus, a tool for somatic SV calling and complex SV characterization using long reads. Severus first detects individual SV junctions from phased split alignments, then constructs a phased breakpoint graph to cluster junctions into complex rearrangement events.
 We first benchmarked the somatic SV calling performance using six tumor/normal cell line pairs (HCC1395, H1437, H2009, HCC1937, HCC1954, Hs578T). We sequenced all cell lines with Illumina, ONT, and PacBio HiFi. We then established a set of high-confidence calls supported by multiple technologies and tools. Severus consistently had the highest F1 scores compared to the HiFi, ONT, and Illumina methods against this high-confidence SV call set.
 We then extend our analysis to complex SVs. Severus accurately detected complex events, i.e., chromothripsis and chromoplexy, and templated insertion cycles/chains (TIC), reported for these cell lines. We then compared Severus’ performance with Jabba and Linx, two widely used tools for complex SV calling in short-read sequencing. Our comparison revealed that Severus showed higher agreement with Linx, while Jabba failed to detect most of the SV clusters identified by both Severus and Linx. Severus also outperformed the other tools in characterizing complex reciprocal translocations and TICs. Most of the junctions in complex SVs called by either of the tools but not Severus were either simple SVs with a single long-read junction or were not present in long-read sequencing. In contrast, Severus effectively resolved overlapping SVs by utilizing long-read connectivity, allowing for more accurate clustering of smaller genomic segments.
 We have also applied Severus to seventeen pediatric leukemia cases. Severus identified two chromoplexy and two cryptic translocations, which were missed by FISH and karyotype panels and were incomplete in Illumina SV calls, further validated by RNA-seq. This highlights the potential of the long-read whole genome sequencing approach for diagnosing complex cases driven by SVs.


 Ayse Keskus, Asher Bryant, Tanveer Ahmad, Anton Goretsky, Byunggil Yoo, Sergey Aganezov, Ataberk Donmez, Lisa A. Lansdon, Isabel Rodriguez, Jimin Park, Yuelin Liu, Xiwen Cui, Joshua Gardner, Brandy McNulty, Samuel Sacco, Jyoti Shetty, Yongmei Zhao, Bao Tran, Giuseppe Narzisi, Adrienne Helland, Daniel Cook, Pi-Chuan Chang, Alexey Kolesnikov, Andrew Carroll, Erin Molloy, Chengpeng Bi, Adam Walter, Margaret Gibson, Irina Pushel, Erin Guest, Tomi Pastinen, Kishwar Shafin, Karen Miga, Salem Malikic, Chi-Ping Day, Nicolas Robine, Cenk Sahinalp, Michael Dean, Midhat S. Farooqi, Benedict Paten, Mikhail Kolmogorov. Severus: A tool for detecting and characterizing complex structural variants in cancer using long-read sequencing [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2848.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0385c5e49d7e54e5c6f4ee5da5d2e85d080cf285" target='_blank'>
              Abstract 2848: Severus: A tool for detecting and characterizing complex structural variants in cancer using long-read sequencing
              </a>
            </td>
          <td>
            A. Keskus, A. Bryant, Tanveer Ahmad, Anton Goretsky, B. Yoo, Sergey Aganezov, Ataberk Donmez, L. Lansdon, Isabel Rodríguez, Jimin Park, Yuelin Liu, Xiwen Cui, J. Gardner, B. McNulty, Samuel Sacco, Jyoti Shetty, Yongmei Zhao, Bao Tran, G. Narzisi, Adrienne Helland, Daniel E. Cook, Pi-Chuan Chang, Alexey Kolesnikov, Andrew Carroll, Erin K. Molloy, Chengpeng Bi, Adam M. Walter, M. Gibson, I. Pushel, E. Guest, T. Pastinen, Kishwar Shafin, K. Miga, S. Malikić, Chi-Ping Day, N. Robine, Cenk Sahinalp, Michael Dean, M. Farooqi, B. Paten, M. Kolmogorov
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has emerged as a critical area of cancer research due to its ubiquitous presence in tumour cells and significant role in tumorigenesis, progression and drug resistance. Recent studies demonstrate that eccDNA promotes cancer progression by influencing genomic instability, amplifying oncogenes, regulating gene expression and enhancing tumour cell adaptability to adverse conditions. While the precise mechanisms underlying eccDNA formation and its biological functions remain unclear, its potential applications in cancer diagnosis, prognosis and targeted therapy are gaining increasing recognition. This review summarises the latest advancements in eccDNA research, highlighting its potential as both a biomarker and a therapeutic target. Additionally, it emphasises the translational potential of eccDNA in clinical diagnostics and personalised treatment strategies, offering new perspectives for future cancer research and innovative therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eae4df06f51107c25979631d45f7a492cc54031" target='_blank'>
              Extrachromosomal Circular DNA in Cancer: Mechanisms and Clinical Applications.
              </a>
            </td>
          <td>
            Jiajia Li, Peng Luo, Zhengrui Li, Qi Wang, Xufeng Huang, Keliang Wang, Ruo Wang, Runzhi Chen
          </td>
          <td>2025-04-29</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Low cancer incidence rates are observed in large mammals such as elephants and whales, despite their mass and long lifespan hypothetically contributing to a higher lifetime probability of acquiring oncogenic mutations. Few studies have investigated cancer-related genes in these animals. One reported that bowhead whales possess two functional copies of PCNA. Multiple studies have investigated the presence of multiple retrogene copies of TP53 in elephants. Extra copies of genes involved in DNA damage detection and repair could contribute to cancer resistance. Droplet digital PCR (ddPCR) was used to quantify gene copy number for TP53, PCNA, HER2, DLG1, and DLG2 in genomic DNA extracted from frozen skin samples of beluga, narwhal and bowhead whales (n=20 each). Results showed that all 3 whale species had more than one copy of PCNA. Agarose gel PCR showed the simultaneous presence of wild-type PCNA (possessing introns) and variable numbers of pseudogene sequences (lacking introns) within individuals for belugas, narwhals, and bowheads. ddPCR found that elephants did not have increased PCNA copy number, but agarose gel PCR showed the presence of PCNA pseudogenes. Similar patterns were also observed in rhinos, horses, cows and other ungulate species. To investigate copy number loss in cancer, ddPCR was performed on formalin-fixed, paraffin embedded normal and matched tumor tissue of 7 individual belugas from the St. Lawrence estuary, an area that was historically contaminated with industrial carcinogens. Copy number loss was not observed for any investigated tumor suppressor gene in tumor tissue compared to normal tissue. Our results show that bowhead whales are not unique in having multiple copies of PCNA. The presence of incomplete copies with unknown function may influence copy number quantification. Further study is required to understand the significance of the variable PCNA pseudogene copies. Understanding natural copy number variation in tumor suppressor genes may provide insight into risk factors and prevention methods across species.


 Morla Phan, Thompson Worden, Sarah Adamowicz, Stéphane Lair, Mauricio Seguel, Claire Jardine, Geoffrey Wood, Dirk Steinke. Cross-species investigation of gene copy number and cancer resistance [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6715.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e60c4cd21a9e7eb8974a45714112351803792a88" target='_blank'>
              Abstract 6715: Cross-species investigation of gene copy number and cancer resistance
              </a>
            </td>
          <td>
            Morla Phan, Thompson Worden, Sarah Adamowicz, Stéphane Lair, Mauricio Seguel, Claire Jardine, Geoffrey A. Wood, Dirk Steinke
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c62dd3d2788ffe055d52a584d9b884866f23c266" target='_blank'>
              Delayed indel formation after Cas9 cleavage promotes mosaicism and aneuploidy in human embryos
              </a>
            </td>
          <td>
            Alex Robles, Julie Sung, Stepan Jerabek, Jishnu Talukdar, Diego Marin, Jia Xu, Nathan Treff, Dieter Egli
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Focal gene amplifications are among the most common cancer-associated mutations but have proven challenging to engineer in primary cells and model organisms. Here we describe a general strategy to engineer large (more than 1 Mbp) focal amplifications mediated by extrachromosomal DNAs (ecDNAs) in a spatiotemporally controlled manner in cells and in mice. By coupling ecDNA formation with expression of selectable markers, we track the dynamics of ecDNA-containing cells under physiological conditions and in the presence of specific selective pressures. We also apply this approach to generate mice harbouring Cre-inducible Myc- and Mdm2-containing ecDNAs analogous to those occurring in human cancers. We show that the engineered ecDNAs spontaneously accumulate in primary cells derived from these animals, promoting their proliferation, immortalization and transformation. Finally, we demonstrate the ability of Mdm2-containing ecDNAs to promote tumour formation in an autochthonous mouse model of hepatocellular carcinoma.These findings offer insights into the role of ecDNA-mediated gene amplifications in tumorigenesis. We anticipate that this approach will be valuable for investigating further unresolved aspects of ecDNA biology and for developing new preclinical immunocompetent mouse models of human cancers harbouring specific focal gene amplifications.


 Melissa A. Yao, Davide Pradella, Minsi Zhang, Rui Gao, Katarzyna M. Gluchowska, Ylenia Cendon Florez, Tanmay Mishra, Gaspare La Rocca, Moritz Weigl, Ziqi Jiao, Hieu H. Nguyen, Marta Lisi, Mateusz M. Ozimek, Chiara Mastroleo, Kevin Chen, Felix Grimm, Jens Luebeck, Shu Zhang, Andrea Alice Zolli, Eric G. Sun, Bhargavi Dameracharla, Zhengqiao Zhao, Yuri Pritykin, Carlie Siegel, Howard Y. Chang, Paul S. Mischel, Vineet Bafna, Cristina R. Antonescu, Andrea Ventura. Engineered extrachromosomal oncogene amplifications promote tumorigenesis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2853.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e79bd95f47abcb0c8154c2b9c5f4a7bf2825f70" target='_blank'>
              Abstract 2853: Engineered extrachromosomal oncogene amplifications promote tumorigenesis
              </a>
            </td>
          <td>
            Melissa A. Yao, Davide Pradella, Minsi Zhang, Rui Gao, Katarzyna M. Gluchowska, Ylenia Cendon Florez, Tanmay Mishra, Gaspare La Rocca, Moritz Weigl, Ziqi Jiao, Hieu H. M. Nguyen, Marta Lisi, Mateusz M. Ozimek, C. Mastroleo, Kevin Chen, Felix Grimm, J. Luebeck, Shu Zhang, Andrea Alice Zolli, Eric G Sun, Bhargavi Dameracharla, Zhengqiao Zhao, Yuri Pritykin, Carlie Siegel, Howard Y. Chang, P. Mischel, V. Bafna, C. Antonescu, Andrea Ventura
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>134</td>
        </tr>

        <tr id="Segmental duplications (SDs) are blocks of genomic DNA with high sequence homology that are hotspots for chromosomal rearrangements, coinciding with copy-number and single-nucleotide variations in the population. SDs could represent unstable genomic regions that are susceptible to somatic alterations in human cancers. Here, we aimed to elucidate the genomic locations of SDs in relation to cancer-related genes and their propensity for somatic alterations in cancer. The analysis showed that tumor suppressor genes (TSGs) were less associated with SDs compared to non-cancer genes in nearly all mammalian species. TSGs with SDs were larger in size in humans but only modestly conserved among mammals. In humans, the proportion of non-cancer genes with SDs decreased as the gene age increased. However, for TSGs, a loss of association with SDs was apparent among young genes. Pan-cancer analysis revealed that TSGs with SDs were more prone to deletions and structural variations independently of gene size. Re-analysis of publicly available experimental data further revealed that genes with SDs tended to replicate late and were more likely to undergo the error-prone mitotic DNA synthesis upon replication stress. In conclusion, the loss of SDs from TSGs during mammalian evolution protects against tumor formation by reducing somatic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7dee62f17cffda68dd501cc4ece25c8e474528a" target='_blank'>
              Tumor Suppressor Genes with Segmental Duplications are Prone to Somatic Deletions and Structural Variations.
              </a>
            </td>
          <td>
            Ziheng Huang, Lin Zhang, Huarong Chen, Xiaodong Liu, Likai Tan, Dan Huang, Yingzhi Liu, Yushan Wang, Xinyi Zhang, A. S. Cheng, Maggie Haitian Wang, Wei Kang, K. To, Jun Yu, Ho Ko, Le Yu, Sunny H Wong, Tony Gin, M. T. Chan, Xiansong Wang, W. K. K. Wu
          </td>
          <td>2025-05-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 8523


 Background:
 Driver-negative lung adenocarcinoma (DN-LUAD) without actionable genomic alterations (AGAs) has a poor prognosis. A deeper understanding of its molecular background, particularly across the entire genome, is crucial for improving risk assessment and therapeutic development.
 Methods:
 We performed deep whole-genome sequencing (WGS; tumor depth: 100–120x, normal depth: 30x, paired-end reads: 150 bp) and RNA sequencing using fresh-frozen tissues from four institutions. DN-LUAD was defined as LUAD without known actionable mutations or gene fusions. We analyzed somatic mutations, copy number alterations (CNAs), and structural variations (SVs). Tumors were classified as whole-genome doubling (WGD) if more than half of the autosomal tumor genome showed at least two copies in the major copy numbers of somatic cells. HRDetect score ≥0.7 was used to determine the presence of homologous recombination DNA repair deficiency (HRD) (Davies H, et al. Nat Med. 2017). Gene Set Enrichment Analysis (GSEA) was used for pathway analysis.
 Results:
 Among the 745 patients (pts), 517 were classified as having DN-LUAD. WGS identified AGAs undetected by whole-exome sequencing (WES) in 33 (4.4%) pts (
 EGFR
 13,
 KRAS
 7,
 BRAF
 5,
 ERBB2
 4,
 MET
 4, and
 HRAS
 1). The frequently observed genomic alterations in DN-LUAD are shown in the Table. WGD was observed in 62.1% (n = 321) of DN-LUADs and was significantly associated with higher TMB, CNA, and SV burden. Tumor suppressor gene (TSG) mutations in
 TP53
 ,
 STK11
 , and
 KEAP1
 , as well as
 CDKN2A
 copy number loss and SVs, were significantly more frequent in WGD pts. HRD was identified in 19 DN-LUAD pts (3.7%), with a significantly higher frequency in the WGD group (WGD vs. without WGD; 4.98% vs. 1.5%, p = 0.016). GSEA showed significant (q < 0.0001) upregulation of cell cycle pathways (E2F targets, G2M checkpoint, and MYC targets) and downregulation of immune pathways (allograft rejection and interferon-gamma response) in DN-LUADs with WGD.
 Conclusions:
 This largest-ever WGS study identified AGAs undetectable by WES and uncovered a subgroup of DN-LUAD characterized by increased genomic instability driven by multiple TSG alterations associated with WGD, which was also more likely to exhibit HRD. WGD was associated with the upregulation of cell cycle pathways and the downregulation of immune pathways. These findings highlight the critical role of WGS in elucidating the pathogenesis of DN-LUAD.





 AllN = 517
 WGDn = 321
 Without WGDn = 196

 p





 Median TMB (Mutations/Mb)
 6.6
 10.2
 3.0
 <0.0001


 Median No. of CNAs
 101
 117
 77
 <0.0001


 Median No. of SVs
 185
 244
 103
 <0.0001



 TP53

 381 (74)
 277 (86)
 104 (53)
 <0.0001



 SMARCA4

 59 (11)
 43 (13)
 16 (8)
 0.094



 STK11

 56 (11)
 42 (13)
 14 (7)
 0.049



 CDKN2A

 44 (9)
 33 (10)
 11 (6)
 0.092



 KEAP1

 40 (8)
 34 (11)
 6 (3)
 0.003



 CDKN2A
 loss

 106 (21)
 78 (24)
 28 (14)
 0.009



 MET
 amplification

 44 (9)
 34 (11)
 10 (5)
 0.045



 CDKN2A

 135 (26)
 95 (30)
 40 (20)
 0.028



 FHIT

 125 (24)
 99 (31)
 26 (13)
 <0.0001




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16b5bbd4f2e96da01654006c7177d2c691759a6f" target='_blank'>
              Whole-genome and transcriptome landscape of actionable driver-negative lung adenocarcinoma.
              </a>
            </td>
          <td>
            Masahiro Torasawa, K. Shiraishi, Akifumi Mochizuki, Shingo Matsumoto, Junko Hamamoto, Hirokazu Matsushita, Issei Imoto, Hiroyuki Yasuda, Kazuhisa Takahashi, Yuichi Shiraishi, Masahiro Tsuboi, Koichi Goto, Shun-ichi Watanabe, R. Hamamoto, T. Kohno
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background The histone chaperone Holliday Junction Recognition Protein (HJURP) has been associated with multiple types of cancers, but its role in GC is not yet fully understood. Considering its functions in centromere stability and DNA repair, investigating HJURP’s role in GC may offer novel therapeutic perspectives. Methods HJURP expression was examined in a dataset comprising TCGA-STAD samples and an internal group of GC patients, utilizing RNA sequencing and Western blot techniques. Functional experiments were carried out on the AGS and HGC-27 GC cell lines. The expression levels of HJURP, MYC, and Topoisomerase II alpha (TOP2A) were assessed via quantitative real-time PCR and Western blot. Proliferation rates of the cells were determined through EdU, CCK-8, and colony formation assays. Results Compared to adjacent normal tissues, HJURP expression was notably increased in GC tissues, a finding consistent across both the TCGA-STAD database and our internal patient group. Silencing HJURP markedly reduced GC cell growth and chemoresistance. Mechanistically, HJURP enhanced MYC stability, which in turn promoted TOP2A transcription. Rescue experiments confirmed that overexpression of TOP2A alters proliferation and chemoresistance of GC cells with HJURP knockdown, indicating the dependency of this axis on MYC activity. Conclusion Our study demonstrates that HJURP is critical for promoting GC proliferation and chemoresistance through the regulation of the MYC/TOP2A transcriptional network. Targeting HJURP might offer a novel therapeutic avenue for GC, necessitating further exploration of its clinical potential. This work underscores the value of investigating histone chaperones as potential targets in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feb8f81bdc3f0ec8326d390d83a122cdb0440ebe" target='_blank'>
              HJURP modulates cell proliferation and chemoresistance via the MYC/TOP2A transcriptional axis in gastric cancer
              </a>
            </td>
          <td>
            Xu Li, Xiwen Li, Yanlin Ren, Ling Wang, Zehao Mao, Shikun Gao, Peng Ma, Junjie Chen
          </td>
          <td>2025-04-11</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Human papillomaviruses (HPVs), like many other viruses, are able to integrate their genomes into the host cellular genome. This integration can activate viral oncogenes or alter the function of cellular oncogenes and tumor suppressor genes, thereby increasing the likelihood of HPV-associated tumor development. In particular, HPV types 16 and 18 are responsible for over 70% of all cervical, anal, and oropharyngeal cancers worldwide, with rising incidence. Even more, high-resolution mapping of preferred integration sites using LR-Seq technologies offers deep insights into the molecular mechanisms of HPV integration. LR-Seq enables the detection of complex integration patterns, where the viral genome can be replicated and amplified into virus–host concatemers, including events within large structural variations or highly repetitive genomic regions. Furthermore, aligning LR-Seq data to the latest T2T reference genome (hs1) is necessary to provide new information about viral integration in genomic regions that were previously inaccessible, such as centromeres and other structurally complex repeat-rich loci. In this review, we provide insights into HPV genomic integration revealed by LR-Seq technologies, with a particular focus on how the use of the complete T2T reference genome enhances the detection of integration events in previously uncharacterized, repeat-rich regions of the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8bd3d524ef4cdc3c87b3f69c07be4376d8e44c3" target='_blank'>
              Novel Avenues for the Detection of Cancer-Associated Viral Genome Integrations Using Long-Read Sequencing Technologies
              </a>
            </td>
          <td>
            Larissa-Anna Bergmann, Alicja Pacholewska, Michal R. Schweiger
          </td>
          <td>2025-05-22</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Background
 Chromosomal instability (CIN) is pervasive during cancer evolution, particularly in non-small cell lung cancer (NSCLC) where it is associated with poor recurrence-free survival and correlates with the frequency of whole genome doubling events (WGD). By duplicating the complete set of chromosomes, WGD is a key event during cancer evolution and correlates with poor prognosis and targeted therapy resistance. Despite the importance of these events, genetic events responsible for the initiation and maintenance of CIN and WGD in NSCLC have not been systematically investigated.
 Methods
 TRAcking Cancer Evolution through Therapy/Rx (TRACERx) is a multicancer longitudinal study which utilizes multiregional sampling and whole exome sequencing (WES) to elucidate potential causes of tumor heterogeneity and cancer progression. TRACERx enables us to investigate genetic alterations and their relationship with WGD and CIN, and proceed to functionally characterize them.
 Results
 We identified that genetic alterations in 6 genes, including FAT1, results in homologous recombination (HR) repair deficiencies and CIN. Mutations in FAT1 are further associated with HR deficiency in patient cohorts. Using orthogonal genetic and experimental approaches, we demonstrate that FAT1 alterations are positively selected before WGD in lung cancer evolution. FAT1 ablation causes persistent replication stress, an elevated mitotic failure rate, nuclear deformation, and elevated structural CIN including chromosome translocations and radial chromosomes. Furthermore, FAT1 loss contributes to WGD, which we attribute to the dysregulation of YAP1. Co-depletion of the oncogene YAP1 partially reduces the elevated numerical CIN caused by FAT1 loss but does not relieve HR deficiencies nor structural CIN. Importantly, overexpression of constitutively active YAP15SA is sufficient to induce numerical CIN. HR deficiencies and high CIN caused by FAT1 loss synergizes with WGD to fuel tumor evolution, a prominent factor underpinning therapy resistance.
 Conclusions
 These data suggest that FAT1 loss in lung cancer attenuates HR repair and exacerbates structural and numerical CIN through two distinct downstream mechanisms, leading to increased tumor heterogeneity.


 Wei-ting Lu, Lykourgos-Panagiotis Zalmas, Chris Bailey, James R. Black, Carlos Martinez-Ruiz, Oriol Pich, Francisco Gimeno-Valiente, Ieva Usaite, Mariam Jamal-Hanjani, Nicolas Tapon, Kasper Fugger, Nicholas McGranahan, Kevin Litchfield, Nicolai J. Birkbak, Jiri Bartek, Nnennaya Kanu, Charles Swanton. FAT1 alterations contribute to chromosomal instability in cancer cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1169.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6ae5fa1f276c91179bfcbf4e5cddc5ae1bea20" target='_blank'>
              Abstract 1169: FAT1 alterations contribute to chromosomal instability in cancer cells
              </a>
            </td>
          <td>
            Wei-Ting Lu, L. Zalmas, Chris Bailey, James R. M. Black, C. Martínez-Ruiz, O. Pich, F. Gimeno-Valiente, Ieva Usaite, M. Jamal-Hanjani, N. Tapon, K. Fugger, N. Mcgranahan, K. Litchfield, Nicolai J. Birkbak, Jiri Bartek, N. Kanu, C. Swanton
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac35aaec3d88c6b3cd4be700d6af5bf25aaebdef" target='_blank'>
              Drosophila SA1 expression prevents brain tumorigenesis and PARP-mediated cell elimination
              </a>
            </td>
          <td>
            Simona Totaro, Antonella Lettieri, Silvia Castiglioni, Francesco Lavezzari, C. Gervasini, Valentina Massa, Thomas Vaccari
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Stem cells have the unique ability to self-renew and differentiate into specialized cell types. Epigenetic mechanisms, including histones and their post-translational modifications, play a crucial role in regulating programs integral to a cell’s identity, like gene expression and DNA replication. However, the transcriptional, chromatin, and replication timing profiles of adult stem cells in vivo remain poorly understood. Containing germline stem cells (GSCs) and somatic cyst stem cells (CySCs), the Drosophila testis provides an excellent in vivo model for studying adult stem cells. However, the small number of stem cells and cellular heterogeneity of this tissue have limited comprehensive genomic studies. In this study, we developed cell type-specific genomic techniques to analyze the transcriptome, histone modification patterns, and replication timing of GSC-like and CySC-like cells. Single cell RNA-sequencing validated previous findings on GSC-CySC intercellular communication and revealed high expression of chromatin regulators in GSCs. To characterize chromatin landscapes, we developed a cell-type-specific chromatin profiling assay to map H3K4me3-, H3K27me3-, and H3K9me3-enriched regions, corresponding to euchromatic, facultative heterochromatic, and constitutive heterochromatic domains, respectively. Finally, we determined cell type-specific replication timing profiles, integrating our in vivo datasets with published data using cultured cell lines. Our results reveal that GSCs display a distinct replication program compared to somatic lineages, that aligns with chromatin state differences. Collectively, our integrated transcriptomic, chromatin, and replication datasets provide a comprehensive framework for understanding genome regulation differences between these in vivo stem cell populations, demonstrating the power of multi-omics in uncovering cell type-specific regulatory features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2db91bf6625baf7a6f5b436dd4b6c6b57e16b4fc" target='_blank'>
              Cell type- and chromosome-specific chromatin landscapes and DNA replication programs of Drosophila testicular stem cells
              </a>
            </td>
          <td>
            Jennifer A. Urban, Daniel Ringwalt, John M. Urban, Wingel Xue, Ryan J Gleason, Keji Zhao, Xin Chen
          </td>
          <td>2025-04-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Despite their deleterious effects, small insertions and deletions (InDels) have received far less attention than substitutions. Here we generated isogenic CRISPR-edited human cellular models of postreplicative repair dysfunction (PRRd), including individual and combined gene edits of DNA mismatch repair (MMR) and replicative polymerases (Pol ε and Pol δ). Unique, diverse InDel mutational footprints were revealed. However, the prevailing InDel classification framework was unable to discriminate these InDel signatures from background mutagenesis and from each other. To address this, we developed an alternative InDel classification system that considers flanking sequences and informative motifs (for example, longer homopolymers), enabling unambiguous InDel classification into 89 subtypes. Through focused characterization of seven tumor types from the 100,000 Genomes Project, we uncovered 37 InDel signatures; 27 were new. In addition to unveiling previously hidden biological insights, we also developed PRRDetect—a highly specific classifier of PRRd status in tumors, with potential implications for immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63ec8a3709640e7f5608e28fbcbdbf6376d5a9c2" target='_blank'>
              A redefined InDel taxonomy provides insights into mutational signatures
              </a>
            </td>
          <td>
            G. Koh, A. S. Nanda, Giuseppe Rinaldi, Soraya Boushaki, A. Degasperi, Cherif Badja, A. Pregnall, S. Zhao, Lucia Chmelova, Daniella Black, Laura Heskin, João M. L. Dias, Jamie Young, Yasin Memari, Scott Shooter, Jan Czarnecki, Matthew A Brown, Helen R Davies, Xueqing Zou, S. Nik-Zainal
          </td>
          <td>2025-04-10</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="
 Melanoma is the most serious form of skin cancer, developed by the malignant evolution of melanocytes. Malignant melanoma incidence is increasing faster than most other cancers. While stage zero melanoma is highly treatable, survivability dramatically decreases in its advanced stages. Melanoma has shown to be one of the most heterogeneous cancers from RNA and exome analyses by The Cancer Genome Atlas and other groups. A better understanding of the key genomic and epigenomic events that characterize the diverse subclonal populations in melanoma may reveal key insights into what drives its progression and therapeutic resistance.
 In this study, we leveraged Nanopore long-read sequencing to study the evolution of the mouse B2905 melanoma cell line. Twenty-four distinct clonal sublines were derived in vitro from single cells of the cell line, and the genetically homogeneous population from each subline was sequenced using PromethION R10 flow cells. Enabled by long reads to perform haplotype phasing and accurate structural variation detection, our goal is to integrate small and structural variants to better our understanding of melanoma evolution, and build upon prior analyses of short-read sequenced sublines.
 We employed multiple SNV calling approaches, including DeepVariant and Clair, in order to provide highly accurate variants for phylogeny reconstruction using Trisicell. We performed structural variant calling with our cancer somatic structural variant (SV) caller Severus as well as copy-number alteration (CNA) analysis with our method Wakhan. Lastly, we placed SNVs, SVs, and CNAs on our reconstructed phylogeny to examine the progression of different types of variants during subline evolution.
 We identified approximately 560k unique SNVs and around 2, 400 unique SVs. The majority of SNVs (19%) are either clonal or private (73%); however, a meaningful fraction of subclonal variants were available for phylogenetic tree reconstruction. SVs are distributed across the phylogenetic tree branches similarly to SNVs. We identified loss of heterozygosity (LOH) events throughout the subline evolution as well as subclonal CNAs resulting from chromosomal translocations. We find clonal and subclonal evidence of densely clustered SNVs and SV, resembling kataegis; however, our analysis of mutational signatures did not reveal APOBEC-mediated mutations. By analyzing mutational signatures within individual branches of the phylogenetic tree, we observed relative timing of different mutational processes, such as early clonal signatures of UV damage.
 By incorporating structural variations, copy number changes, and small variant data in the phylogenetic reconstruction, our analysis offers a better characterization of the genetic landscape of subclonal evolution in melanoma.


 Anton Goretsky, Yuelin Liu, Ayse Keskus, Tanveer Ahmad, Salem Malikic, Glenn Merlino, Chi-Ping Day, Erin Molloy, S. Cenk Sahinalp, Mikhail Kolmogorov. Nanopore sequencing of single-cell derived sublines provides insights into melanoma heterogeneity and evolution [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7497.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb8f2b1ddae01e13edf72bbcb5e1a3343151981e" target='_blank'>
              Abstract 7497: Nanopore sequencing of single-cell derived sublines provides insights into melanoma heterogeneity and evolution
              </a>
            </td>
          <td>
            Anton Goretsky, Yuelin Liu, A. Keskus, Tanveer Ahmad, S. Malikić, Glenn Merlino, Chi-Ping Day, Erin K. Molloy, S. C. Sahinalp, M. Kolmogorov
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Oncogenic gene fusions occur across a broad range of cancers and are a defining feature of some cancer types. Cancers driven by gene fusion products tend to respond well to targeted therapies, where available; thus, detection of potentially targetable oncogenic fusions is necessary to select optimal treatment. Detection methods include non-sequencing methods, such as fluorescence in situ hybridization and immunohistochemistry, and sequencing methods, such as DNA- and RNA-based next-generation sequencing (NGS). While NGS is an efficient way to analyze multiple genes of interest at once, economic and technical factors may preclude its use in routine care globally, despite several guideline recommendations. The aim of this review is to present a summary of oncogenic gene fusions, with a focus on fusions that affect tyrosine kinase signaling, and to highlight the importance of testing for oncogenic fusions. We present an overview of the identification of oncogenic gene fusions and therapies approved for the treatment of cancers harboring gene fusions, and summarize data regarding treating fusion-positive cancers with no current targeted therapies and clinical studies of fusion-positive cancers. Although treatment options may be limited for patients with rare alterations, healthcare professionals should identify patients most likely to benefit from oncogenic gene fusion testing and initiate the appropriate targeted therapy to achieve optimal treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a05e8c035d2b5c9d05b3addb1c0208226f39c0" target='_blank'>
              Oncogenic gene fusions in cancer: from biology to therapy
              </a>
            </td>
          <td>
            Stephen V Liu, Misako Nagasaka, Judith Atz, Flavio Solca, Leonhard Müllauer
          </td>
          <td>2025-04-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10fbe2acee93fcc04d4cf719859f00b1fc38a341" target='_blank'>
              Megabase-scale human genome rearrangement with programmable bridge recombinases
              </a>
            </td>
          <td>
            Nicholas T. Perry, Liam J. Bartie, Dhruva Katrekar, Gabriel A. Gonzalez, Matthew G. Durrant, James J. Pai, Alison Fanton, Masahiro Hiraizumi, Chiara Ricci-Tam, Hiroshi Nishimasu, Silvana Konermann, Patrick D. Hsu
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c45c976013cae1b807eb9d7fbbf98792d114f7a" target='_blank'>
              Cell-type-specific patterns and consequences of somatic mutation in development and aging brain
              </a>
            </td>
          <td>
            Andrea J. Kriz, Shulin Mao, Diane D. Shao, Daniel A. Snellings, Rebecca E. Andersen, Guanlan Dong, Chanthia C. Ma, Hayley E. Cline, A. Huang, Eunjung Lee, Christopher A. Walsh
          </td>
          <td>2025-05-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4028f5dcc0e86189184980c13bb56279010a29ab" target='_blank'>
              Clonal lineage tracing and parallel multiomics profiling reveal transcriptional diversification induced by ARID1A deficiency
              </a>
            </td>
          <td>
            Kenichi Miyata, Liying Yang, Yuxiao Yang, Kohei Kumegawa, R. Maruyama
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7626abb52e331b0a94cdec546fd1b0c7ebf8d0ff" target='_blank'>
              Accumulation of a biparentally-inherited Neptune transposable element in natural Killifish hybrids (Fundulus diaphanus × F. heteroclitus)
              </a>
            </td>
          <td>
            Alexandre-James Roussel, Alexander Suh, Francisco J. Ruiz-Ruano, Anne-Marie Dion-Côté
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0036830d7ca0d3dd2d206e6880d31ee221a0279" target='_blank'>
              Advances in single-cell DNA sequencing enable insights into human somatic mosaicism.
              </a>
            </td>
          <td>
            Diane D. Shao, Andrea J. Kriz, Daniel A. Snellings, Zinan Zhou, Yifan Zhao, Liz Enyenihi, Christopher A. Walsh
          </td>
          <td>2025-04-25</td>
          <td>Nature reviews. Genetics</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Endometrial cancer (EC) is the most common gynecological malignancy. Although prognosis is favorable for patients with an early-stage disease, those with recurrent or more advanced disease have low response rates to chemotherapy and poor clinical outcomes. Previously, we have shown that DNA repair gene (NEIL3) is required for retaining replication fork integrity during replication stress. Here, we examined whether the overexpression of NEIL3 in endometrial cancer associated with altered genomic instability, tumor immunogenicity and anti-tumor immunity in endometrial tumor. In this study, we show that endometrial cancer patients with tumors that a have high NEIL3 expression associated with worse overall survival (OS) outcomes in patients. In addition, tumor with high NEIL3 expression is associated with high number of mutation and chromosomal instability. Furthermore, NEIL3 expression in EC tumors positively correlated with mutation of DNA polymerase eta (POLE) and TP53 as well as high expression of replicative polymerases genes (POLE, POLD1 and POLA1). In contrast, tumor with high NEIL3 expression exhibit low tumor immunogenicity and poor anti-tumor immune cell infiltration. Our findings may have important clinical implications for utilizing NEIL3 as a potential prognostic biomarker to stratify EC patients and as a target to enhance immunotherapy response in endometrial cancer. However, our NEIL3 overexpression associated observation still requires further experimental-based scientific validation studies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-00393-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/229a73859c8660361d382a29f27f2f93cf0da384" target='_blank'>
              DNA glycosylase (NEIL3) overexpression associated with low tumor immune infiltration and poor overall patient survival in endometrial cancer
              </a>
            </td>
          <td>
            Cristofer Barry, Aashirwad Shahi, Dawit Kidane
          </td>
          <td>2025-05-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

 To assess the feasibility, reliability, and limitations of using Whole Genome Bisulfite Sequencing (WGBS) for detecting somatic copy number variation (SCNV) and tumor-specific allelic imbalance (AI). Background: In cancer genomics, the use of WGBS for SCNV and AI detection enables simultaneous analysis of genetic variations and methylation landscapes, potentially providing insights into the interplay between somatic alterations and DNA methylation in tumorigenesis. However, WGBS data is inherently noisier compared to whole-genome sequencing (WGS) or microarray data due to artifacts introduced during bisulfite conversion. These challenges, including difficulties in read alignment, genotype calling, and phasing, the lack of strand complementarity, and the need to distinguish true C>T SNPs from bisulfite-induced changes, collectively complicate SCNV and AI detection. Method: We analyzed WGBS data from The Cancer Genome Atlas (TCGA), including 39 tumor samples and 8 adjacent normal tissues across 9 tumor types. SCNV detection was performed using GATK-CNV, while AI callings were conducted with BisSNP, BISCUIT, Revelio (with Unified Genotyper), and BSsnp. Confusion matrices were used to compare GATK-CNV (WGBS) and hapLOH (microarray) at the base, gene, and arm levels. F1 score, specificity, and sensitivity were calculated to evaluate their performance across different genomic scales. Additionally, AI identified from WGBS data using BisSNP, BISCUIT, Revelio (with Unified Genotyper), and BSsnp were compared to those detected from array and WGS data using hapLOH/seq. To determine detection thresholds and refine results, sequencing data were downsampled, with filters applied to exclude low-quality data and retain high-confidence SNPs and genotypes. Result: GATK-CNV exhibited variable performance across samples, with most samples showing high true positives and low false positives, indicating good sensitivity for detecting SCNVs and capturing a genome burden similar to hapLOHseq data. However, for samples with a high number of events, normalization disrupts event callings, resulting in inconsistencies in SCNV identification compared to array data. For AI detection, Revelio (with Unified Genotyper) showed the best overall performance among the tools evaluated. Downsampling analysis showed that reducing sequencing coverage impaired event detection, becoming less effective at lower coverage levels. BSsnp, BISCUIT, and BisSNP produced widespread and inconsistent variant calls, and initial filtering for high-quality SNPs showed limited improvement in precision. This study highlights the potential and challenges of WGBS as a tool for analyzing SCNV/AI alongside DNA methylation with phased data, offering insights into tumorigenesis and advancements in cancer diagnostics and treatment.



 Yiru Gu, Jian Gu, Paul Scheet, Justin Wong. Comparative analysis of somatic copy number variations and allelic imbalance in whole-genome bisulfite sequencing: challenges and insights [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5050.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a4380bb34311ba8e2b2e583f5eb6ff7d05c33e" target='_blank'>
              Abstract 5050: Comparative analysis of somatic copy number variations and allelic imbalance in whole-genome bisulfite sequencing: challenges and insights
              </a>
            </td>
          <td>
            Yiru Gu, Jian Gu, P. Scheet, Justin Wong
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 PAX5 is known to be frequently mutated in childhood B-cell precursor acute lymphoblastic leukemia (B-ALL) via several molecular mechanisms, including chimeric fusion, focal deletion, and point mutations. Among these, patients with PAX5 internal tandem duplication (PAX5-ITD) are reported to have a poor outcome. Unlike other common ITDs (e.g., FLT3), PAX5-ITD often involves gain of more than one copy of involved exons, and the exact nature of these alterations remain elusive. Here we aim to investigate the following questions: 1) what are the typical copy number gains in DNA and RNA among PAX5-ITD cases? 2) are PAX5-ITD always in-frame? 3) are these aberrations a result of multi-step or single-step events? Short-read whole genome sequencing (WGS) and RNA sequencing (RNA-seq) and PacBio ISO-seq, a state-of-the-art long-read sequencing method, are used in this study. By integrative analysis of WGS and RNA-seq data from a relapse cohort (n=584) and a non-relapse cohort (n=1055), we identified PAX5 mutation rate of ∼30% in relapsed patients versus that of ∼20% in non-relapsed patients. Among the various PAX5 mutation types, the PAX5-ITD mutation shows the greatest degree of enrichment (P=3e-6) in relapsed patients (n=21), compared with those who remain in remission (n=5). Copy number gains in PAX5-ITD cases ranged from 3 to 25 and involved four exon combinations within the gene: E5, E3-E5, E2-E5 and E1-E5 and concordant copy number gains were detected between diagnosis tumor and relapse tumor. We observed a negative correlation between the number of copy gains and the length of duplicated genomic regions. Furthermore, 11 out of the 26 PAX5-ITD cases exhibited multiple structural variants (SVs) and 1 case harbored 2 somatic SNV in ITD region. To investigate the transcriptional consequences of these SV events, we performed ISO-seq on 7 PAX5-ITD cases with available specimen. By mapping SVs and SNVs to full-length RNA transcripts, we observed complex sequences of events driving the formation of ultimate PAX5-ITD in 3 cases. Besides, copy number analysis on full-length RNA transcripts revealed variable copy number gains within tumors due to alternative splicing. Additionally, we identified neo-splice sites in 2 cases, indicating that the splicing process was influenced by the altered DNA context created by SVs. Finally, our data indicated that all PAX5-ITDs can transcribe in-frame mRNAs, suggesting activating nature of these mutations. These findings provide valuable insights into the genomic complexity of PAX5-ITD and highlight the diverse transcriptional and splicing outcomes due to these alterations.


 Yanling Liu, Bensheng Ju, Li Dong, Melanie Loyd, Yuan Feng, Heather Mulder, Pandurang Kolekar, Li Fan, Hanxia Li, Scott Foy, Geoff Neale, Patrick A. Brown, William L. Carroll, Mignon L. Loh, John Easton, Xiaotu Ma. Uncovering genomic complexity of PAX5 internal tandem duplication using short-read and long-read sequencing [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2617.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e44c370bd7521d24afccebc5dd604dc8d4b6c7b" target='_blank'>
              Abstract 2617: Uncovering genomic complexity of PAX5 internal tandem duplication using short-read and long-read sequencing
              </a>
            </td>
          <td>
            Yanling Liu, B. Ju, Li Dong, Melanie Loyd, Yuan Feng, H. Mulder, P. Kolekar, Li Fan, Hanxia Li, S. Foy, Geoffrey Neale, Patrick A. Brown, William L Carroll, M. Loh, John Easton, Xiaotu Ma
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinoma (HGSOC) is driven by chromosomal instability and patterns of genomic rearrangement have predictive and prognostic value in this disease. Nearly all next generation sequencing performed to date rely on short read sequencing, which is unable to resolve complex rearrangements or analyze repetitive centromeric or telomeric regions. Here we combine long-read sequencing (LRS) platform with the recently completed telomere-to-telomere (T2T-CHM13) reference genome to report the first comparison of whole genome variation in BRCA1-mutated and CCNE1-amplified HGSOC.



 High-molecular-weight DNA was extracted from cryopreserved, pre-treatment tumor and matched blood samples from six HGSOC patients and sequenced using Oxford Nanopore PromethION platform. Base-called reads were aligned to both the GRCh38 and T2T-CHM13 reference genomes for the analysis of SNVs, indels, allele-specific CNVs, complexed structural variants, transposable element insertions, and CpG methylation patterns.



 Somatic pathogenic TP53 mutations were identified in all tumors (P1-P6). Germline pathogenic BRCA1 mutations were present in two cases (P1, P2), while CCNE1 amplification was observed in the remaining four cases (P3-P6). Homologous recombination deficiency (HRD) scores were elevated in BRCA1-mutated tumors (73 and 84) compared to CCNE1-amplified tumors (71, 43, 53, and 29, respectively). Complete loss of heterozygosity in chromosome 17 was observed in all tumors. Chromoplexy was detected in five tumors (P1, P2, P4-P6). Focal amplifications associated with extrachromosomal DNA and breakage-fusion-bridge cycles were observed in P2 and P6. Telomere length was significantly shortened in all tumors compared to matched normal samples. Additionally, CpG methylation levels were significantly reduced in centromeric satellite repeats and inserted transposable elements in all tumors than in normal samples.



 This study establishes the first LRS-based analysis workflow for comprehensive genomic characterization of HGSOC, uncovering novel alterations in previously inaccessible genomic regions. These findings highlight the potential of LRS as a new gold standard for whole genome analysis in HGSOC, warranting further investigation for its clinical and research applications.



 Shiro Takamatsu, Jian Li, Thomas Welte, Eleonora Khlebus, Veena Vuttaradhi, Allison Brodsky, Barrett Lawson, Robert T. Hillman. Telomere-to-telomere long read whole genome sequencing provides novel insights into ovarian cancer genomics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB347.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/841194928a5faee8a089ace06821025bbd9fd89a" target='_blank'>
              Abstract LB347: Telomere-to-telomere long read whole genome sequencing provides novel insights into ovarian cancer genomics
              </a>
            </td>
          <td>
            S. Takamatsu, Jian Li, Thomas Welte, Eleonora Khlebus, Veena K Vuttaradhi, Allison L. Brodsky, B. Lawson, R. Hillman
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Retrocopies, also known as processed pseudogenes, are structural variations resulting from the duplication of protein-coding genes via the reverse transcription of their mature messenger RNA by the LINE-1 enzymatic machinery. These retrocopies can be fixed, present in all genomes of a given species, or unfixed—classified here as retroCNVs—when they are polymorphic, either germline or somatic. While fixed retrocopies have been extensively studied, the understanding of retroCNVs remains limited due to the lack of specialized bioinformatics tools for their detection and annotation in DNA sequencing data, as well as curated databases to share findings with the scientific community.



 To address this gap, we developed the sideRETRO, a robust computational tool for the identification and annotation of retroCNVs in whole-genome and exome sequencing data. The results of this tool will be made accessible through siderGrimoire, an online, curated database integrating retroCNVs from human populations and cancer samples.



 The sideRETRO pipeline, published in Bioinformatics, offers a dedicated approach to detecting non-fixed retroCNVs. Using datasets from the 1000 Genomes Project and The Cancer Genome Atlas (TCGA), siderGrimoire curates retroCNVs across human populations and tumor samples, providing detailed annotations and a user-friendly interface.



 Initial analyses reveal a comprehensive landscape of retroCNVs, uncovering their prevalence and patterns of events in diverse human populations and cancer types. This resource bridges a critical gap in the availability of retroCNV-specific data for the research community.



 The siderGrimoire database represents a significant advancement in retroCNV research, providing a valuable tool for studying the role of retrocopies in genetic variation and cancer biology. By integrating large-scale data from key genomic projects, this platform enables new insights into the genomic impacts of retroCNVs, fostering further exploration in human genetics and oncology.



 Thiago Luiz Miller, Helena Beatriz da Conceição, Pedro Alexandre Galante. siderGrimoire: a database of retroCNVs in human tumors and populations [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1070.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b3cddb316bdce4c6828dd308d722612ae6e4f52" target='_blank'>
              Abstract 1070: siderGrimoire: a database of retroCNVs in human tumors and populations
              </a>
            </td>
          <td>
            T. L. Miller, Helena Beatriz da Conceição, P. A. Galante
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Alternative lengthening of telomeres (ALT) is a telomerase-independent telomere maintenance mechanism observed in 15% of human cancers. A hallmark of ALT cancers is the presence of C-circles, circular single-stranded DNAs (ssDNAs) enriched with cytosine-rich telomere (C-rich, CCCTAA) sequences. G-circles, containing guanosine-rich telomere (G-rich, GGGTTA) ssDNAs, also exist but are much less abundant. Recent studies indicate that excessive displacement of Okazaki fragments during lagging-strand synthesis is a unique feature of ALT telomeres and responsible for generating C-circles/C-rich ssDNAs. However, the distinct characteristics of C-circles compared to G-circles remain unclear. Here, we demonstrate that co-deficiency of the DNA translocases SMARCAL1 and FANCM leads to abundant generation of G-circle/G-rich ssDNAs. These G-rich ssDNAs mainly exist in linear form, ranging in size from 500 to 3000 nucleotides, which differs significantly from the structure and size of C-circle/C-rich ssDNAs. Mechanistically, both C-rich and G-rich ssDNAs originate from BLM/POLD-mediated excessive strand displacement; however, they differ in their origins and initiation mechanisms. Specifically, C-rich ssDNAs arise from lagging daughter strands initiated by the CST complex, whereas G-rich ssDNAs originate from leading daughter strands through RAD51-dependent G-strand synthesis. Our findings propose two distinct mechanisms for generating two different extrachromosomal telomere DNAs, C-and G-circles, during ALT-mediated telomere elongation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e3561acbee649acf0e09cb6b6caaf71d6c0db30" target='_blank'>
              Distinct mechanisms underlying extrachromosomal telomere DNA generation in ALT cancers
              </a>
            </td>
          <td>
            Junyeop Lee, Eric J. Sohn, Jina Lee, Angelo Taglialatela, Alberto Ciccia, Jaewon Min
          </td>
          <td>2025-04-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1326a43bcf280179de51c8de12b664e06eaa82" target='_blank'>
              Single cell resolution of an epigenetic signature of persister tumor cells
              </a>
            </td>
          <td>
            Mihai Dumbrava, W. Ismail, Leticia Sandoval, Amelia Mazzone, Syed Mohamed Musheer Aalam, Megan L. Ritting, Xiaonan Hou, Yiwen Xie, Shariska P Harrington, Scott H. Kaufmann, Nagarajan Kannan, S. Weroha, Alexandre Gaspar-Maia
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39136831a82d4694ee219ef38ce5763db7fa3a19" target='_blank'>
              ECCFP: a consecutive full pass based bioinformatic analysis for eccDNA identification using Nanopore sequencing data
              </a>
            </td>
          <td>
            Tangxuan Zhang, Wang Li, Qingsong Zen, Jun Zhang, Biyuan Miao, Mengting Li, Juan Luo, Tianliang Liu, Shifu Chen, Shaogui Wan
          </td>
          <td>2025-05-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2b1a766c9476f757f76140a52e67514cf195c7f" target='_blank'>
              Longitudinal and multisite sampling reveals mutational and copy number evolution in tumors during metastatic dissemination.
              </a>
            </td>
          <td>
            K. Zhao, J. Vos, S. Lam, Lillian A Boe, Daniel Muldoon, Catherine Y Han, Cristina Valero, Mark Lee, C. Fitzgerald, A. Lee, Manu Prasad, Swati Jain, Xinzhu Deng, Timothy A Chan, Michael F. Berger, C. Bandlamudi, Xi Kathy Zhou, Luc G. T. Morris
          </td>
          <td>2025-06-02</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ce720a5b79eb9fb996bed883c2b0fde9438b7fa" target='_blank'>
              Genome evolution in parthenogenetic nematodes shaped by chromosome rearrangements and rare sex
              </a>
            </td>
          <td>
            George Chung, Lewis Stevens, Xiaoguang Dai, Scott Hickey, K. Kiontke, Priyesh Rughani, M. Kieninger, Joanna C. Collins, S. Juul, Fabio Piano, David H.A. Fitch, Mark Blaxter, Kristin C. Gunsalus
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Tandemly repeated DNA fragments are major components of centromeres and pericentromeric heterochromatin, which is responsible for chromosomal stability and segregation. Recent evidence suggests that transcripts from these repeats play a key role in heterochromatin maintenance, and these repeats can be highly dynamic with various copy numbers. Here, we developed a mathematical model for human satellite repeats, which tracks the silenced and desilenced repeats, lncRNA, and copy number. Our model shows that chromatin factors for silencing and RNA stability can facilitate copy gain in satellites. Also, the system can be bistable, and cells with different copy numbers, silenced repeats with a small copy number, and desilenced repeats with a large copy number may coexist. To incorporate the cooperative methylation by neighboring repeats and the local chromatin environment, we also developed a spatial model where the local chromatin environment facilitates methylation locally. This model suggests that a local domain of silenced repeats may be an important feature of copy number regulation. Our models suggest that pericentromeric repeats are highly dynamic, and small changes in chromatin regulation can lead to large changes in satellite copy numbers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bee19e6a3b205523eb44ce800caa9c0aacfabffd" target='_blank'>
              Modeling the Copy Number of HSATII Repeats in Human Pericentromere
              </a>
            </td>
          <td>
            P. Ghimire, Richard I. Joh
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) transforms a diploid cell into a tetraploid state, making it prone to genomic instability, aneuploidy, and malignancy. Tumors with evidence of past WGD events range from less than 10% in Non-Hodgkin’s Lymphoma to almost 60% in germ cell cancer. WGD confers advantages, including resistance to stressors, making it a powerful macroevolutionary force. Why, then, isn’t WGD a precursor for most cancers? In Glioblastoma (GBM), ∼86% of tumors lack evidence of WGD in their evolutionary history. We integrated a multi-modal dataset of 11 histopathology whole slide images (WSIs), 68 medical images, and whole-genome sequencing data (WGS) from six patient-matched primary and recurrent GBM. Three atlases were derived from FDG PET/MRI (glucose uptake), quantitative BOLD imaging (oxygenation), and magnetic resonance elastography (stiffness), capturing variability across brain regions and patient age. Using HATCHet, which infers clone-specific karyotypes from WGS, we quantified GBM clonal composition and its changes between surgeries. Chromosome 7 amplification and chromosome 10 loss were detected in at least one clone in all tumors. Genetic intra-tumor heterogeneity (coexistence of ≥2 clones) was found in 4/6 patients, with moderate to strong clonal changes between surgeries in all four. This data informed parameters governing how cells of variable (poly)aneuploidy respond to oxygen and glucose by dying (α_d, s_r), switching to glycolysis (O_th), or migrating (v_max). Two parameters significantly differed among patients: oxygen-dependent death constant α_d (ANOVA p = 1.7E-5) and oxygen level prompting a metabolic shift to glycolysis O_th (ANOVA p = 1.9E-4). Higher-ploidy GBMs showed elevated O_th values, and higher-ploidy clones within tumors had consistently higher O_th than coexisting lower-ploidy clones. Between surgeries, simulated tumors with distinct karyotype compositions progressed at different rates. Whether higher ploidy predicted fast recurrence was however a function of the tumor anatomic location, in line with the hypothesis that environments with limited resources are likely to exert selective pressure against WGD. Compared to intra-brain differences in resource supply, the differences observed between organs can be larger, providing an opportunity to further test this hypothesis. Indeed, the average ploidy of 20 different cancer types was positively correlated with the oxygen levels recorded in their respective organ of origin (Spearman r = 0.65, P = 0.002) and negatively correlated to the glucose uptake rates in the organ of origin (Spearman r = -0.70, P = 0.026). Overall, our findings suggest that metabolic substrate availability in the tissue micro-environment influences cell fate decisions based on karyotype, shaping GBM initiation and recurrence.


 Zuzanna Nowicka, Frederika Rentzeperis, Vural Tagal, Konstantin Maksin, Parag Katira, Noemi Andor. Interactions between ploidy and resource availability shape clonal interference in glioblastoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6238.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f63a435551ab128bd453aa26cc898c619184a9dc" target='_blank'>
              Abstract 6238: Interactions between ploidy and resource availability shape clonal interference in glioblastoma
              </a>
            </td>
          <td>
            Z. Nowicka, Frederika Rentzeperis, Vural Tagal, Konstantin Maksin, Parag Katira, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Aneuploidy, a cancer hallmark, drives chromosomal instability, drug resistance, and clinically aggressive tumors. Cyclin-dependent kinase 2 (CDK2) antagonism with independent inhibitors or CDK2 knockdown triggered anaphase catastrophe. This disrupts supernumerary centrosome clustering, causing multipolar division and apoptosis. Time-lapse fluorescence microscopy of fluorescent ubiquitination-based cell cycle indicator (FUCCI) cell cycle probes transduced into aneuploid lung cancer cells revealed distinct fates of bipolar and polyploid cells after CDK2 inhibition. Apoptosis occurred in multipolar progeny but was repressed in persistent polyploid cancer cells. RNA-Seq analyses after CDK2 inhibition of 4N versus 2N lung cancer cells were enriched for CDK1 pathway and KIF family members. The Cancer Genome Atlas (TCGA) analysis of lung cancers indicated that CDK1 and KIF family member overexpression was associated with an unfavorable survival. Intravital microscopy of transplanted lung cancer cells in mice extended findings from the in vitro to in vivo settings. CDK2 inhibition of tumor-bearing mice produced polyploid cancer cells in vivo. These cancer cells were resistant to apoptosis and proliferated despite CDK2 inhibition. In contrast, polyploid populations were rarely detected in CDK2-inhibited human alveolar epithelial cells. These findings are translationally relevant. Combined targeting of CDK2 with CDK1 or kinesin family member antagonists should eliminate polyploid cancer cells, promote apoptosis, and augment antineoplastic effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2792ac282d27759d5d74152b7ef668d21c563cc" target='_blank'>
              CDK2 inhibition produces a persistent population of polyploid cancer cells
              </a>
            </td>
          <td>
            Liliya Tyutyunyk‐Massey, Zibo Chen, Xiuxia Liu, Masanori Kawakami, Adam Harned, Yeap Ng, Brian Luke, Samuel C Okpechi, Blessing Ogunlade, Y. Alfaro-Mora, Roberto Weigert, Kedar Narayan, Xi Liu, Ethan Dmitrovsky
          </td>
          <td>2025-04-15</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e285847d1a624eba0863eb2e7a290a733cac7918" target='_blank'>
              Phenotypic and Functional Characterization of Oncohistone Mutations in Breast Cancers
              </a>
            </td>
          <td>
            Andrea D. Edwards, Yangyang Dai, Siddharth Singh, Micah Thornton, Tulip Nandu, Ralf Kittler, C. Camacho, Dan Huang, W. Kraus
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Loss-of-function mutations in genome maintenance genes fuel tumorigenesis through increased genomic instability. A subset of these tumor suppressors are challenging to identify due to context dependency, including functional interactions with other genes and pathways. Here, we searched for potential causal genes that impact tumor development and/or progression in breast cancer through functional-genetic screening of candidate genes. MYH4, encoding a class II myosin, emerged as a top hit impacting genomic stability. We show that MYH4 suppresses DNA replication stress by promoting replication licensing and replication fork progression. Moreover, we observed a strong synergistic relationship among class II myosins in suppressing replication-associated DNA damage. Genomic analysis of Pan-Cancer Analysis of Whole Genomes project breast cancer samples revealed frequent concomitant loss of TP53 with MYH4 and class II myosins on chromosome 17p. Notably, Myh4 disruption accelerated mouse mammary tumorigenesis in a Trp53-deficient background. In conclusion, our results suggest an unanticipated function of MYH4 in p53-mediated tumor suppression that can explain their combined loss in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64806f05c07b8d24d35ea0d2ce3f2fd4c89aa895" target='_blank'>
              The class II myosin MYH4 safeguards genome integrity and suppresses tumor progression
              </a>
            </td>
          <td>
            J. Thatte, Ana Moisés da Silva, Judit Börcsök, Thorkell Gudjónsson, Jan Benada, Xin Li, Muthiah Bose, H. van der Gulden, Ji-Ying Song, Renée Menezes, Elena Martín-Doncel, Luis Toledo, Valdemaras Petrosius, Cord Brakebusch, J. Jonkers, F. C. Nielsen, Maria Rossing, C. S. Sørensen
          </td>
          <td>2025-06-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="

 Patient-derived cancer models advance therapeutic development and are essential to studies of treatment resistance mechanisms. The Human Cancer Models Initiative (HCMI) has created 665 patient-derived models, including organoids, neurospheres, and conditionally reprogrammed cells, all paired with patient tumors and clinical annotations. This resource provides an opportunity to study genomic conservation and divergence during model development at scale in a large pan-cancer cohort of matched tumor-model pairs.



 We characterized the genomes of models and parent tumors, computing allele specific Copy Number Variation (CNV), Single Nucleotide Variants (SNV), small Insertions and Deletions (InDels) and Structural Variations (SV) from bulk whole genome and whole exome Sequencing. We used pyclone to study changes in clonal abundances between tumor and model, MutationTimeR to understand the timing of model specific CNVs and Whole Genome Doubling (WGD), and AmpliconArchitect to identify ecDNA.



 HCMI models exhibited high concordance with matching tumors when evaluated on SNVs, SVs, InDels and CNVs, with 69% of SNV/InDel drivers and 43% of CNV drivers conserved between tumor and model. Only 2% of investigated models (9/405) showed significantly divergent patterns of aneuploidy and presence of distinct clonal outgrowth in the model. This included two pancreatic cancer models with oncogenic KRAS point mutations that differed from those in the parent tumor. The cohort included models and tumors with ecDNA affecting known oncogenes including EGFR in GBM samples, CCNE1/KRAS in esophageal cancer and MYC in pancreatic cancer. ecDNA were the least conserved aberration type; 20% (50/254) of tumor-detected ecDNA were retained in an associated model while 35% (50/143) of model-detected ecDNA were found in their parent tumors. While the tumors and models generally showed similar WGD states, 10% of models exhibited WGD private to the model, and this model specific WGD generally occurred later in estimated mutational evolution time than models for which the tumor and model were both WGD. Surprisingly, not all models were pure for cancer cells with 19 models, enriched for melanomas, containing more than 20% diploid presumably non-tumor cells.



 HCMI models exhibited high genomic concordance with their matching tumors across all cancer types. We catalogued classes of genomic divergence representing either selection of subclones or evolution during model development. SNVs and InDels exhibited greater conservation compared to CNVs, with ecDNA showing the highest variability between the tumor and the model. Our results provide full visibility into the genomic fidelity of each HCMI model, and enable researchers to select the most relevant HCMI model for their study.



 Andrew W. McPherson, Seongmin Choi, William F. Hooper, Jennifer M. Shelton, Timothy R. Chu, Dina ElHarouni, Mushriq Al-Jazrawe, Merve Dede, Toshinori Hinoue, Sean A. Misek, Heeju Noh, Luca Zanella, Moony Tseng, Hayley E. Francies, Priya Sridevi, Rachana Agarwal, Cindy W. Kyi, Julyann Perez-Mayoral, Megan J. Stine, Eva Tonsing-Carter, James M. Clinton, The HCMI Network, Peter W. Laird, Calvin J. Kuo, Olivier Elemento, David L. Spector, Andrew D. Cherniack, Kyle Ellrott, Martin L. Ferguson, Rameen Beroukhim, Katherine A. Hoadley, Nicolas Robine, Mathew J. Garnett, David A. Tuveson, Andrea Califano, Paul T. Spellman, Keith L. Ligon, Daniela S. Gerhard, Louis Staudt, Jesse Boehm. Patterns of genomic patient-model conservation and evolution in the Human Cancer Models Initiative (HCMI) next-generation cancer model resource [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3882.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba5271ff004a44b818284ad63ea30a279c93609a" target='_blank'>
              Abstract 3882: Patterns of genomic patient-model conservation and evolution in the Human Cancer Models Initiative (HCMI) next-generation cancer model resource
              </a>
            </td>
          <td>
            Andrew McPherson, Seongmin Choi, William F Hooper, Jennifer Shelton, Timothy R. Chu, Dina Elharouni, Mushriq Al-Jazrawe, Merve Dede, T. Hinoue, Sean A. Misek, Heeju Noh, Luca Zanella, Moony Tseng, Hayley E. Francies, Priya Sridevi, Rachana Agarwal, Cindy W. Kyi, Julyann Pérez-Mayoral, Megan J. Stine, Eva Y. Tonsing-Carter, James M. Clinton, The Hcmi Network, Peter W. Laird, Calvin J. Kuo, O. Elemento, David L. Spector, Andrew D Cherniack, Kyle Ellrott, M. Ferguson, R. Beroukhim, K. Hoadley, N. Robine, Mathew J. Garnett, D. Tuveson, Andrea Califano, Paul T. Spellman, Keith L. Ligon, Daniela S. Gerhard, Louis M. Staudt, Jesse S. Boehm
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>149</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82b1e341092863e6d653d106625c09215549f517" target='_blank'>
              Homologous recombination deficiency in primary ER-positive and HER2-negative breast cancer
              </a>
            </td>
          <td>
            H. R. Davies, Daniella Black, A. Kvist, Kristín Sigurjónsdóttir, Ana Bosch Campos, Ramsay Bowden, Yasin Memari, Ziqian Chen, Giuseppe Rinaldi, Frida Rosengren, Deborah F. Nacer, S. Veerla, L. Hohmann, Nicklas Nordborg, J. Häkkinen, J. Vallon-Christersson, Åke Borg, S. Nik-Zainal, J. Staaf
          </td>
          <td>2025-06-04</td>
          <td>None</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="
 Aneuploidy, a hallmark of cancer, drives chromosomal instability and antineoplastic drug resistance. Common in hematologic and solid tumors, centrosome amplification leads to supernumerary centrosomes, causing chromosomal missegregation in dividing cancer cells. This imbalance produces progeny with abnormal chromosome counts, leading to cell death, resistance to antineoplastics, and clinically-aggressive tumors. This study examined the effects of CDK2 inhibition on aneuploid lung cancer cells and tumors. It revealed that CDK2 antagonism triggers anaphase catastrophe through centrosome clustering disruption and multipolar mitosis. Unexpectedly, this also leads to the formation of a persistent population of polyploid, apoptosis-resistant cells. Using independent CDK2 inhibitors and shRNAs, we observed that some cancer cells underwent multipolar division and subsequent cell death. Others exhibited cytokinesis failure, resulting in multinucleated, and polyploid cancer cells that continued to survive and proliferate despite continuous CDK2 inhibition. Through time-lapse imaging with transfected FUCCI cell cycle probes and RNA-seq analysis of polyploid versus diploid populations, we identified upregulated CDK1-related pathways and kinesin (KIF) family members in the polyploid cellular population. This likely contributes to the persistent polyploid population, despite CDK2 antagonism. These findings were independently validated in vivo using intravital microscopy and patient-derived xenograft (PDX) lung cancer models. Intriguingly, polyploid and multinucleated cancer cells were found to proliferate post-CDK2 inhibitor treatments. In marked contrast, a polyploid population was not found in CDK2 inhibited primary human alveolar epithelial cells, indicating that this is a pathway that is preferentially activated in aneuploid cancer cells. Analysis of gene expression in TCGA lung cancer datasets linked CDK1 and KIF family overexpression to an unfavorable patient survival, implicating this persistent polyploid population as contributing to poor survival outcomes in lung cancer. These findings are relevant to the clinical development of CDK2 inhibitors. They indicate that despite the pro-apoptotic effects of CDK2 antagonism in aneuploid lung cancer cells, a population of drug-resistant, polyploid cells persists. We propose that a combinatorial therapy targeting CDK2 alongside polo-like kinase 4 (PLK4), CDK1 or kinesin family members would further disrupt centrosome clustering, eliminate polyploid cancer cells, and enhance tumor cell apoptosis and antineoplastic effects. This dual approach would enhance the anti-tumor efficacy of CDK2 inhibition, potentially overcoming resistance mechanisms. This provides a promising strategy to eradicate persistent lung cancer cells following CDK2 inhibition.


 Liliya Tyutyunyk-Massey, Zibo Chen, Xiuxia Liu, Masanori Kawakami, Adam Harned, Yeap Ng, Brian Luke, Sameul C. Okpechi, Blessing Ogunlade, Yair Alfaro, Roberto Weigert, Kedar Narayan, Xi Liu, Ethan Dmitrovsky. CDK2 inhibition produces persistent polyploid cancer cells with a survival advantage [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5327.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0fc58e0b6388ab5fc82e889af73612c384df0b6" target='_blank'>
              Abstract 5327: CDK2 inhibition produces persistent polyploid cancer cells with a survival advantage
              </a>
            </td>
          <td>
            Liliya Tyutyunyk‐Massey, Zibo Chen, Xiuxia Liu, Masanori Kawakami, Adam Harned, Yeap Ng, Brian Luke, Sameul C. Okpechi, Blessing Ogunlade, Yair Alfaro, Roberto Weigert, Kedar Narayan, Xi Liu, Ethan Dmitrovsky
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1225ffe2d0c6c775628165cdb9bef1963448726" target='_blank'>
              A First-in-Class Therapy Targeting Transcription-Replication Conflicts Drives Synthetic Lethality in Homologous Recombination Deficient Cancers
              </a>
            </td>
          <td>
            Yi Chieh Lim, Soak-Kuan Lai, Ru Han Yang, Yong Kang Chua, Rémi Patouret, Vaishnervi Manoj, Colin Chun Xuan Soh, Benjamin Lebeau, Ning Cham, Wei Meng, Sze Keong Tey, Szu Ling Yeap, Songjing Zhang, Jessie Yong Xing Woon, Zahin Ali, Marek Mutwil, Pritpal Singh, M. Fullwood, Linda LD Zhong, Xue Ming Dong, Chiba Shunsuke, Cheng-Gee Koh, H. Li
          </td>
          <td>2025-05-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cddde94c859320bea53863c22a0c0f005dca0fc" target='_blank'>
              Comprehensive analysis of FAM83B in pan-cancer and preliminary exploration in esophageal squamous cell carcinoma.
              </a>
            </td>
          <td>
            Wei Guo, Xixi Zhao, Xinran Huang, Ruijuan Zhang, Yuchen Wang, Xinyu He, Xiangyun Ma, Yu Hao, Shangyi Geng, S. Pan, Hongbing Ma
          </td>
          <td>2025-05-26</td>
          <td>Journal of molecular histology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 e16580


 Background:
 Non-muscle invasive bladder cancer (NMIBC) presents a significant clinical challenge due to its high recurrence rate. Recently, a deeper understanding of tumor epigenetics has provided new opportunities for cancer detection and treatment. While significant efforts have been made to understand NMIBC biology, the molecular and epigenetic mechanisms driving its progression remain elusive.
 Methods:
 We used multiomics analysis including expression, DNA methylation, and cell-free DNA chromatin immunoprecipitation sequencing (cfChIP-seq) assays to uncover the molecular and epigenetic landscape of NMIBC from human tissue samples and urine.
 Results:
 High-throughput sequencing revealed more than 6,000 differentially methylated regions (DMRs) within regulatory elements, many of which were linked to cancer-related pathways and exhibited distinct signatures in cancer tissues compared to healthy controls. Next, we distinguished high and low-grade tumors based on unique DNA methylation profiles. These patterns correlated with transcription level of genes associated with cell cycle regulators, consistent with the higher proliferative capacity of high-grade tumors. Moreover, Among the DMRs, we also identified genes involved in T cell regulation. Supporting this, we demonstrated reduced CD8+ T cells infiltration in high-grade tumors, further highlighting immune dysregulation in the more aggressive disease. We extended our investigation to urine samples, performing cfChIP-seq on samples from healthy donors and NMIBC patients. By using the H3K4me3 promoter histone mark on cfDNA we could detect cell origin. Utilizing this assay, we were able to identify changes in the urine of NMIBC patients, owing to cf-nucleosome contributions from both the tumor and immune cells. Additionally, we stratified the cancer cohort by tumor grade and found enrichment in the number of differentially marked genes in high-grade tumors.
 Conclusions:
 Our detection of unique epigenetic signatures offer opportunities for biomarker development, enabling early detection through non-invasive urine based diagnostic approach. This could grade tumor more accurately and tailor personalized patient care and monitoring.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38dd958dabb53d9b68b4717906734c6068947417" target='_blank'>
              Elucidating the epigenetic landscape of non-muscle invasive bladder cancer.
              </a>
            </td>
          <td>
            T. Falick Michaeli, Batia Azria, M. Lotem, Nir Friedman
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1437313dac6bd0a59138128e06745ba119fe182" target='_blank'>
              Does chromoanagenesis play a role in the origin of B chromosomes?
              </a>
            </td>
          <td>
            Andreas Houben, Jörg Fuchs, A. Banaei-Moghaddam, Jianyong Chen, Gihwan Kim, Taoran Liu
          </td>
          <td>2025-04-19</td>
          <td>Heredity</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="

 Whole Genome Duplication (WGD) involves the replication of an organism's entire genome and is linked to increased evolutionary complexity and cancer progression. In tumors, WGD contributes to heterogeneity and therapy resistance, making its identification crucial for targeting genomic instabilities and informing treatment strategies. Current methods for detecting WGD are often expensive and time-consuming, highlighting the need for a cost-effective and efficient detection panel to improve therapeutic strategies, patient outcomes, and resource allocation in clinical settings.



 We retrospectively analyzed 4,941 tissue samples from patients spanning 14 different cancer types, including ovarian, uterine, breast, pancreatic, prostate, cervical, liver, lung, colorectal, and gastric cancers. Using a targeted sequencing panel (GENESEEQ GENAVISTA) focused on 437 cancer-related genes, we performed comprehensive mutational profiling (∼1,000X). WGD was determined by analyzing read depth, allele frequency, and employing variant calling algorithms to detect gene duplications and genomic rearrangements. Additionally, a prospective cohort of 33 cancer patients was evaluated to compare the performance of our panel with whole exome sequencing (WES, ∼200X) and whole genome sequencing (WGS, ∼30X).



 Our analysis revealed varying prevalence of WGD across cancer types, with significant rates observed in ovarian (50.93%) and breast cancers (52.32%). Pelvic cancer (60.61%) and melanoma (60.00%) also showed high WGD-positive rates. Conversely, endometrial cancer exhibited a notably lower rate of 19.41%. Pancreatic and prostate cancers showed moderate WGD frequencies of 27.46% and 21.51%, respectively, while liver and bile duct cancers had an incidence of 41.30%. In the prospective cohort, our panel demonstrated higher sensitivity compared to WES (100% vs. 84.62%) but slightly lower specificity (90% vs. 95%). The overall concordance with the gold standard WGS was 93.9%, surpassing that of WES (90.9%).



 Our study highlights the significant variability of WGD across cancer types, with a notably high prevalence in ovarian and breast cancers. This underscores the potential of WGD as a biomarker for therapeutic targeting. Our targeted sequencing panel offers an efficient and cost-effective detection method, demonstrating substantial clinical potential for informing personalized treatment strategies and improving patient outcomes.



 Hua Bao, Haimeng Tang, Rui Liu, Ningyou Li, Min Wu, Xunbiao Liu, Zhili Chang, Xue Wu. Efficient detection of whole genome duplication: a targeted sequencing approach [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2403.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b0db61bd8db3cfb8217c78cd75379d5bcca386e" target='_blank'>
              Abstract 2403: Efficient detection of whole genome duplication: a targeted sequencing approach
              </a>
            </td>
          <td>
            H. Bao, Haimeng Tang, Rui Liu, Ningyou Li, Min Wu, Xunbiao Liu, Zhili Chang, Xue Wu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Chemo-/radioresistance of malignant tumors hampers cancer control and increases patient mortality. Efficient repair of damaged DNA is critical for the maintenance of genomic integrity and fidelity of genetic information. In reverse, increased DNA repair capability in cancer cells contributes to chemo-/radioresistance of malignant tumors. DNA double-strand break (DSB) is the most serious DNA damage and is also the principal molecular basis of radiotherapy. Upon DNA damage, the Ku80 is recruited and forms a critical DNA-PK complex at the DSB sites with Ku70 and the catalytic subunit (DNA-PKcs) to initiate DNA repair. How DNA-PK is assembled and activated is not fully understood. Based on the identification of radiation-reduced Ku80 K568 crotonylation through quantitative global lysine crotonylome analysis, we reveal that Ku80 K568 is crotonylated by p300-CBP-associated factor (PCAF). Upon DNA damage, the K568cr is decrotonylated by HDAC8 (Histone deacetylase 8). Decrotonylation of K568cr empties this site for the subsequent SUMOylation of Ku80 by CBX4. The conversion of Ku80 from K568 crotonylation to SUMOylation facilitates the assembly of DNA-PK complex and autophosphorylation of DNA-PKcs S2056, consequently activating the DSB repair. Moreover, mutation disrupting the post-translational modification (PTM) of Ku80 K568 site sensitizes cancer cells to radiotherapy in tumor-bearing nude mice models. This study elucidates the conversion model between two different forms of PTMs in the regulation of DNA-PK complex assembly and DSB repair, highlighting this model’s potential in controlling chemo-/radioresistance of malignant tumors, as well as expands the atlas of therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b6e28a599caae10368a0a56703d29a0e46b8a2e" target='_blank'>
              Conversion of Ku80 K568 crotonylation to SUMOylation facilitates DNA non-homologous end joining and cancer radioresistance
              </a>
            </td>
          <td>
            Hongling Zhao, Shan-shan Gao, Yang Han, Dafei Xie, Lihui Xuan, Xin Huang, Jinhua Luo, Q. Ran, Gang Li, Hejiang Guo, Weixiang Hu, Jin Jia, Xiaochang Liu, Yuhao Liu, Jinpeng Tan, Chenjun Bai, Yongqing Gu, Teng Ma, Zhongjun Li, Hua Guan, Ruixue Huang, Ping-Kun Zhou
          </td>
          <td>2025-04-21</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Fluoroquinolones induce double-strand breaks in bacterial DNA, triggering the SOS response, a major DNA damage response that ensures the expression of repair proteins but also promotes the emergence and spread of antibiotic resistance. Fluoroquinolone resistance, particularly in Escherichia coli, is a growing global health concern. Understanding bacterial responses to these antibiotics is critical for developing preventive strategies and novel treatments to combat resistance development. This study investigates DNA morphology in E. coli following exposure to ciprofloxacin (CIP), a fluoroquinolone antibiotic. We show that CIP induces a stepwise DNA reorganization, culminating in a highly dense nucleoid structure at midcell-a process we term DNA supercompaction. This phenomenon occurred also with other genotoxic agents. Live-cell imaging revealed that RecN, a structural maintenance of chromosomes (SMC)-like protein, is required for DNA supercompaction, and that RecN's dynamics and activity in this response depend on RecA. Additionally, RecN and RecA frequently colocalized at nucleoid-associated positions. We suggest that RecN and RecA play active roles in DNA supercompaction following severe DNA damage, that their interplay is part of a prompt universal survival response to DNA double-strand breaks in E. coli, and that the extent of the compaction response depends on the DNA damage severity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/506e392d68ea028b658f17b6f901be5aeef9ee4e" target='_blank'>
              RecN and RecA orchestrate an ordered DNA supercompaction response following ciprofloxacin-induced DNA damage in Escherichia coli.
              </a>
            </td>
          <td>
            Krister Vikedal, S. B. Ræder, Ida Mathilde Marstein Riisnæs, Magnar Bjørås, J. Booth, Kirsten Skarstad, Emily Helgesen
          </td>
          <td>2025-05-22</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d4868141c39dc560c925882ee816da6d52c4deb" target='_blank'>
              Uev1A counteracts oncogenic Ras stimuli in both polyploid and diploid cells
              </a>
            </td>
          <td>
            Qi Zhang, Yunfeng Wang, Xueli Fu, Ziguang Wang, Yang Zhang, Lizhong Yan, Yuejia Wang, Muhan Yang, Dongze Song, Ruixing Zhang, Hongru Zhang, Shian Wu, Shaowei Zhao
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is the leading cause of cancer-related mortality worldwide owing to the lack of effective and early diagnostic tools and therapeutic approaches. DNAJC7, a member of the DnaJ heat shock family, is crucial in protein folding and stability; however, its specific functions and mechanisms in HCC remain unclear. Objective This study aimed to explore the role of DNAJC7 in HCC progression and evaluate its potential clinical significance as a prognostic marker. Methods Public databases (TCGA, ICGC, GEO, and TIMER) were used to assess DNAJC7 expression, correlations with clinical parameters, and related signaling pathways. Proliferation, migration, invasion, and cell cycle assays were performed to evaluate the function of DNAJC7 in HCC. Immune infiltration and associations with checkpoint proteins were analyzed using TIMER, and a Gene Set Enrichment Analysis (GSEA) was used to explore enriched pathways. Results DNAJC7 expression was higher in HCC tissues than in adjacent normal tissues and was associated with advanced malignancy and poor prognosis, including a lower overall survival, progression-free survival, and disease-free survival. DNAJC7 knockdown resulted in reduced malignant behavior of HCC cells, leading to S-phase cell cycle arrest. Increased DNAJC7 expression was associated with immune cell infiltration and the presence of immunological checkpoint molecules, including CTLA4 and PD-1. GSEA highlighted the activation of key pathways, including WNT signaling and cell cycle regulation. Conclusion DNAJC7 regulates tumor cell proliferation, migration, invasion, and immune evasion by acting as an oncogene in HCC. It can serve as a diagnostic and prognostic biomarker and potential treatment target for HCC. Supplementary Information The online version contains supplementary material available at 10.1007/s00432-025-06202-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5d2c109594fa970efe3bba40f733c0078c3d83a" target='_blank'>
              Increased expression of DNAJC7 promotes the progression of hepatocellular carcinoma by influencing the cell cycle and immune microenvironment
              </a>
            </td>
          <td>
            Jiaxing Chen, Zhizhao Yang, Yongqiang Cui, Zhilei Zhao, D. Deng, Zhihao Fu, Xiao Zhang
          </td>
          <td>2025-05-01</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Micronuclei exhibit defective proteomes rendering their chromatin vulnerable to fragmentation. This fragmentation process, known as chromothripsis, promotes tumorigenesis by catalysing the activation of oncogenes and the silencing of tumor suppressors. With this role in mind, micronuclei serve as promising targets for therapeutic intervention. This review will explore recent discoveries regarding how micronuclei form, their function in catalysing chromothripsis and how chromothripsis provides a selective advantage for cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ac4a5ea94375865c4d842a9f93ca345cb31b4e0" target='_blank'>
              Cancer cells' chamber of secrets: the link between micronuclei, chromothripsis and malignancy.
              </a>
            </td>
          <td>
            Truman McNeil, Sweta Sikder, Yamini Dalal
          </td>
          <td>2025-05-01</td>
          <td>Open biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) in plasma consists of short DNA fragments resulting from a non-random fragmentation process, with distinct “fragmentomic” characteristics that are related with their cellular origins. Despite the wealth of low-pass whole genome sequencing data from recent cancer liquid biopsy studies, somatic variants within this data have often been overlooked without detailed characterization. Here, we reported that in low-pass sequencing data, somatic variant signatures in cfDNA markedly differ between non-cancerous controls and cancer patients, indicating that tumor-associated signals are retained in these variants. Surprisingly, our investigation into cfDNA fragmentomics showed that even in controls, cfDNA molecules harboring somatic variants exhibit cancer-like traits, such as reduced size, decreased DNA methylation, and altered end motif usages and distributions in the nucleosome structure. Importantly, these somatic variants-associated fragmentomic aberrations are more pronounced in cancer patients, enabling cancer diagnosis. In a large pan-cancer cohort, we utilized artificial intelligence to integrate genomic, fragmentomic, and epigenomic features, developing diagnostic models named FreeSV (Fragmentomic and epigenomic examinations of Somatic Variants in cfDNA) and FreeSV-m. Leveraging somatic variant-associated features alone, the FreeSV model achieved area under the ROC curves (AUCs) between 0.81-0.92 across cancer types; however, when genomewide features were also included, the AUCs of FreeSV-m model substantially increased to 0.93-0.99 across cancer types, highlighting the significance of integrative genomic and fragmentomic analyses in cfDNA for cancer liquid biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f64ed6f9ef46f8ac541b4cf9a70df9227b7d0f90" target='_blank'>
              Cancer-like fragmentomic characteristics of somatic variants in cell-free DNA
              </a>
            </td>
          <td>
            Zhenyu Zhang, Yunyun An, Mengqi Yang, Yuqi Pan, Xiaoyi Liu, Fanglei Gong, Huizhen Lin, Bianbian Tang, Wanqiu Wang, Yunxia Bai, Xin Zhao, Yu Zhao, Changzheng Du, Kun Sun
          </td>
          <td>2025-05-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Despite advancements in screening and treatment, breast cancer mortality remains significant, with nearly one-third of patients worldwide succumbing to the disease. APOBECs, a large family of cytosine deaminase enzymes, function endogenously to restrict viral replication. However, their upregulation in many cancers has been linked to a distinct mutational signature characterized by cytosine-to-uracil conversions, leading to C-to-T transitions. Among these enzymes, APOBEC3B (A3B) is notably overexpressed in breast cancer. It remains uncertain, however, whether tumors with an enriched A3B mutational signature consistently exhibit elevated A3B expression across different breast cancer subtypes. Studies estimate the A3B mutational signature is present in 30-50% of breast cancer cases, with higher prevalence in triple-negative breast cancer (TNBC) and HER2-enriched subtypes. Understanding the pathways influencing A3B expression is an area of growing research interest. Using DNA and RNA sequencing data from breast cancer patients in The Cancer Genome Atlas (TCGA) and the International Cancer Genome Consortium (ICGC), we observed that over 64% of tumors with enriched A3B genomic mutational signatures exhibited above-median A3B mRNA expression compared to non-enriched tumors (p < 0.001). Pathway analyses, including KEGG, Gene Ontology (GO), and Reactome, of high versus low A3B-expressing tumors revealed associations with pathways involved in tumor drug metabolism. Notably, the tamoxifen regulatory pathway was altered in high A3B-expressing tumors, which showed reduced expression of CYP2D6 and CYP3A10. This downregulation was consistent in luminal-A-like and luminal-B-like samples. Previous studies in mouse model systems have demonstrated diminished responses to tamoxifen in cells with high A3B expression; our findings suggest a potential relationship between high A3B expression and altered tamoxifen metabolism through CYP2D6. Additionally, pathways related to uracil synthesis, involving drugs such as 5-fluorouracil and 6-mercaptopurine, were downregulated in tumors with high A3B expression. Downregulated genes included IMPDH1, NME1, TK1, and DPYS, many of which are directly associated with pyrimidine synthesis. This pattern of downregulation was consistent across luminal-A-like, luminal-B-like, and basal-like breast cancer subtypes. These findings suggest that tumors with elevated A3B expression may have a heightened ability to develop resistance to therapies such as tamoxifen. Conversely, such tumors might exhibit increased susceptibility to treatments like 5-fluorouracil. Evaluating the functional consequences of A3B expression in breast cancers could pave the way for strategies to modulate A3B activity, enhancing the efficacy of existing therapies and mitigating drug resistance.


 Joel Pardo, Nuri Temiz, Douglas Yee. Examining APOBEC3B in TCGA and ICGC breast cancer datasets reveals altered drug metabolism pathways [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2409.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17d143808abe3792603e16e2ab02c9eaad5e2640" target='_blank'>
              Abstract 2409: Examining APOBEC3B in TCGA and ICGC breast cancer datasets reveals altered drug metabolism pathways
              </a>
            </td>
          <td>
            Joel Pardo, N. Temiz, Douglas Yee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 11561


 Background:
 Liposarcoma (LPS) is characterized by unstable genomes and high occurrence of gene fusions. Hotspots of both recurrent and non-recurrent gene fusions can provide information about structural alterations in certain LPS subtypes. For instance, myxoid (M) LPS expresses the oncogenic
 FUS-DDIT3
 protein and is known to be fusion-driven. Moreover, disrupted locus 12q13-15 is an important feature of well-dedifferentiated (WD) and dedifferentiated (DD) LPS, and is the site of copy number alterations (CNAs) and gene fusions. Here, we used RNA sequencing to uncover hotspots of intra- and transchromosomal gene fusions in LPS patient samples, identifying potentially clinically relevant events in certain chromosomal regions.
 Methods:
 The BostonGene internal LPS cohort (n=150) was analyzed by bulk whole-transcriptome sequencing, using STAR-fusion for sequence calling. Quality control was performed using FastQC, FastQ Screen, RSeQC, and MultiQC. Tumor purity was assessed via pathological and bioinformatics examination with a threshold of 20%. One sample Poisson rate test was used to evaluate statistical significance of gene fusion hotspots.
 Results:
 We identified 4,080 gene fusions among four LPS subtypes (DDLPS, WDLPS, MLPS, and PLPS - pleomorphic LPS). Of those, 2,302 (56.4%) were intrachromosomal and 1,778 (43.6%) were transchromosomal. Over half of these fusions (1,263/2,302 intrachromosomal fusions, 54.9%; 1,047/1,778 transchromosomal fusions, 58,9%) were detected on chromosome 12. Most identified fusions occurred in the 12q13-15 region (q-value < 0.001), with q15 being especially prevalent in transchromosomal fusions (q-value < 0.001). The most prevalent recurrent fusion across our LPS cohort was
 FUS-DDIT3
 (N=21, MLPS samples). Another notable recurrent fusion was
 TRIO-TERT
 (N=4, DDLPS). We also identified extended gene fusion hotspots in regions containing important oncogenes such as
 MDM2
 and
 FRS2
 in LPS subtypes (DDLPS, WDLPS, PLPS) that are not considered fusion-driven (Table). These findings suggest an oncogenic role of such fusions in these LPS subtypes, along with known CNAs like
 MDM2
 amplification.
 Conclusions:
 Our comprehensive transcriptomic analysis of gene fusions in LPS samples uncovered both new and established hotspots of chromosomal rearrangements. Identification of such hotspots improves our understanding of LPS oncogenesis and thus can enhance the diagnostic accuracy and discovery of new biomarkers.


 Hotspots of gene fusions in liposarcoma.




 Hotspot cytoband
 Diagnosis
 q-value
 Important genes




 1q23.3
 DDLPS
 < 0.001

 ATF6



 1q24.3
 DDLPS, WDLPS
 < 0.001

 DNM3



 12q13.3
 MLPS
 < 0.001

 DDIT3



 12q14.1
 DDLPS, WDLPS
 < 0.001

 CDK4



 12q14.3
 DDLPS, WDLPS
 < 0.001

 HMGA2, YEATS4



 12q15
 DDLPS, WDLPS, PLPS
 < 0.001

 MDM2, FRS2, CPM



 16p11.2
 MLPS
 < 0.001

 FUS





">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df3d585e6b1b252046eb1085a1fdfa70c3a17980" target='_blank'>
              Detecting hotspots of intra- and transchromosomal fusions in liposarcomas by RNA sequencing.
              </a>
            </td>
          <td>
            Dmitrii Grachev, Danil Ivanov, Oleg Baranov, Vladimir Kushnarev, Melissa Clemons, Sheila T. Yong, N. Kotlov, Konstantin Chernyshov, A. Bagaev, N. Fowler, Anthony P Conley, G.M. Cote, Sant P Chawla
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 The FET family of intrinsically disordered proteins (FUS, EWSR1, TAF15) are frequent 5’ oncogenic transcription factor (TF) fusion partners in a diversity of sarcomas and leukemias. These TF fusion oncoproteins are often the sole oncogenic driver alteration in these cancers and due to the difficulty in pharmacologic targeting of TFs, precision medicine approaches have been lacking. Here we focus on the pediatric bone tumor Ewing sarcoma (ES) as a model for the class of FET rearranged cancers. ES patients with relapsed or metastatic disease continue to have dismal outcomes, yet decades of clinical experience and laboratory testing of ES cancer cell lines have shown ES to be among the most chemo- and radiosensitive cancers, at least initially. In this study, we address the long-standing hypothesis that ES tumors harbor an intrinsic DNA repair defect by studying the role of the ES fusion oncoprotein EWSR1-FLI1 in regulating the DNA damage response (DDR).
 Native FET family members are among the earliest factors that localize to DNA double-strand breaks (DSBs). Here we discover that EWSR1-FLI1 (and other FET) fusion oncoproteins are aberrantly recruited to DNA DSBs and interfere with native FET (EWSR1) protein function in activating the DNA damage sensor ATM. We demonstrate the functional importance and therapeutic relevance of ATM suppression in ES through increased reliance on a compensatory DDR kinase, ATR, and FET fusion oncoprotein-dependent synthetic lethality with ATR inhibitors (nanomolar range IC50’s). Lastly, we tested ATR inhibition in PDX models across the class of FET rearranged cancers as a synthetic lethal therapeutic strategy to specifically target functional ATM defects induced by FET fusion oncoproteins themselves. We demonstrate significant anti-tumor responses to ATR inhibitor monotherapy in all 5 PDX models including those derived from multiply relapsed ES and clear cell sarcoma patients. In summary, our study provides a new mechanism for how an oncogene can impair physiologic DNA repair (i.e., aberrant recruitment of an oncogenic fusion protein to sites of DNA damage) to create functional states akin to genetic DDR pathway loss, and rationale for a new targeted therapeutic strategy (i.e., ATM/ATR synthetic lethality) in ES and the broader class of undruggable FET fusion-driven cancers.


 Shruti Menon, Daniel Gracilla, Marcus Breese, Filemon Dela Cruz, Asmin Tulpule. FET fusion oncoproteins disrupt physiologic DNA repair and create a targetable opportunity for ATR inhibitor therapies [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1200.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4121bda9e26ae184324ef8e02b596fbe5bd446bc" target='_blank'>
              Abstract 1200: FET fusion oncoproteins disrupt physiologic DNA repair and create a targetable opportunity for ATR inhibitor therapies
              </a>
            </td>
          <td>
            Shruti Menon, Daniel Gracilla, Marcus R. Breese, F. D. Dela Cruz, Asmin Tulpule
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214c0603eec158f1992599e2af88a95e6ef18ce3" target='_blank'>
              Rapid gain and loss of a chromosome drives key morphology and virulence phenotypes in Histoplasma, a fungal pathogen of humans
              </a>
            </td>
          <td>
            Sarah Heater, M. Voorhies, Rosa A. Rodriguez, B. English, Anita Sil
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91eca1dbea2babed41e4311424ffb2059ec0250c" target='_blank'>
              Sequencing the gaps: dark genomic regions persist in CHM13 despite long-read advances
              </a>
            </td>
          <td>
            Mark E. Wadsworth, Madeline L. Page, Bernardo Aguzzoli Heberle, Justin B. Miller, Cody J. Steely, M. Ebbert
          </td>
          <td>2025-05-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The conceptional shifts on genome organization and hereditary variability occurred during transition from classical mendelian to current mobile or dynamic genetics. The main changed premises of this transition are firstly presented in detail. Mendelian genetics mainly conceived genome as the set of chromosomes with of all genes. Now genome semantics is changed. It comprises entire hereditary constitution of the cell, including both structural and dynamic aspects of coding, storage and transfer of species-specific information. There are three kinds of heritable changes: mutations, variations and epigenetic alterations. It is reasonable to discriminate in the genome two subsystems: Obligate genetic elements (OGE) and Facultative genetic elements (FGE). FGEs comprise various kinds of repeated DNA, mobile elements, amplicons, inserted viral and foreign DNA, B-chromosomes and cytobionts. FGEs are predominant genome content of many plants. The number and cell topography of FGEs are different in different cells/tissues and most eukaryote individuals. Changes in the structure or order of OGEs correspond to classical mutations. Various changes in FGEs it is reasonable to call variations. Facultative elements and their variations are the first genomic reaction on biotic and environmental challenges. Together with epigenetic alterations they implement the operational genomic memory. Three template genome processes Replication, Transcription, Translation and three basic genetic processes – Repair, Recombination and Segregation are capable to facultative expression according to principle: the unity of the whole and freedom of the parts. This is the essence of the presented generalized concept of the genome organization and hereditary variations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08181d3ed9f40145f41c1bd8a5154dfef8c50437" target='_blank'>
              Beyond Mendel and Morgan to the dynamic genome
              </a>
            </td>
          <td>
            M. D. Golubovsky
          </td>
          <td>2025-05-11</td>
          <td>Vavilovia</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Deciphering genomic architecture is crucial for identifying novel disease drivers and understanding cancer initiation and progression mechanisms. While short-read sequencing remains the primary tool in cancer genomics, it may miss important structural variants (SVs), especially large chromosomal alterations. A further challenge to cancer researchers is working with FFPE (formalin-fixed, paraffin-embedded) samples, the most common source of archived material, as this sample type is often at risk of nucleic acid fragmentation, DNA crosslinks, and deamination, further complicating variant discovery. We present a Dovetail-FFPE assay, an MNase-HiC-based method compatible with FFPE samples that generates linked-reads to enable improved sensitivity for SV detection. Here, we evaluate the performance of Dovetail-FFPE against a shotgun sequencing library in FFPE and FF (fresh frozen) using a clinical serous ovarian adenocarcinoma (HGSOC) sample. We generated WGS and Dovetail-FFPE libraries at 30x target coverage and utilized BWA for alignment, with Hi-C breakfinder (Dovetail-FFPE) and Manta (WGS) for SV calling. Both Dovetail-FFPE and WGS datasets were analyzed using DeepSomatic for SNV and intel detection, while CNVs were identified using Purple. Dovetail-FFPE analysis revealed four translocations (tlocs) with strong supporting evidence (1;12, 3;12, 3;4, 2;15). These tlocs, also identified in a MicroC-FF sample and validated by PCR and Sanger sequencing, corresponded with copy number changes and occurred at CNV boundaries. In contrast, WGS detected only two of these four tlocs with lower precision. Somatic mutations in coding regions showed high concordance between methods. The Dovetail-FFPE library successfully recalled all pathogenic variants from the truth set (defined as pathogenic/likely pathogenic significance by ClinVar in the FF WGS sample). Copy number analysis demonstrated strong concordance between Dovetail-FFPE and FF-WGS (89%), recalling a heterozygous deletion of chromosome 1p and gains of 1q and 12. Additionally, shotgun, Dovetail-FFPE, and MicroC-FF showed agreement in predicted purity, ploidy, and percentage of genome altered. Our findings demonstrate that Dovetail-FFPE libraries can effectively detect SVs identified in FF libraries and outperform shotgun libraries. Combined with its capability to identify SNVs, indels, and CNVs, linked-read sequencing proves to be a valuable tool for comprehensive genetic variation discovery.


 Alexander Fortuna, Jon Torchia, Mital Bhatka, Natalie Fredriksson, Daniel Hwang, Ek Danwanichakul, Cory Padilla, J. Zachary Sandborn, Lisa Munding. High-resolution characterization of FFPE-derived linked-read libraries for cancer genomics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6631.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44f748e0d579685144f938fbb6b91f060c1dc323" target='_blank'>
              Abstract 6631: High-resolution characterization of FFPE-derived linked-read libraries for cancer genomics
              </a>
            </td>
          <td>
            Alexander Fortuna, Jonathan Torchia, Mital Bhatka, Natalie Fredriksson, Daniel Hwang, Ek Danwanichakul, Cory Padilla, J. Z. Sandborn, Lisa Munding
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Prostate cancer (PCa) is a leading cause of cancer-related mortality among men worldwide. Despite progress in the understanding of tumor biology, the prognosis for advanced prostate cancer remains poor, necessitating the identification of novel diagnostic, prognostic, and therapeutic biomarkers. Methuosis, a recently identified form of programmed cell death (PCD), is characterized by cytoplasmic vacuole accumulation and subsequent cell rupture, distinct from classical apoptosis and necrosis. The key regulatory gene in Methuosis, ARF6 (ADP-ribosylation factor 6), has emerged as a potential marker for cancer diagnosis and treatment. However, its role in prostate cancer and other malignancies remains insufficiently understood. Methods In this study, we performed a comprehensive pan-cancer multi-omics analysis to investigate the role of ARF6 in Methuosis across multiple cancer types, with a specific focus on PCa as the primary context. Using data from public databases, including RNA sequencing, gene expression profiling, and clinical outcomes, we assessed the association between ARF6 expression and patient prognosis in PCa within this broader pan-cancer framework. Additionally, we employed functional enrichment analyses and survival analysis to explore the potential of ARF6 as a diagnostic and prognostic marker for prostate cancer. Immunotherapy-related gene expression signatures were also evaluated to determine the therapeutic relevance of ARF6. Results ARF6 was significantly overexpressed in PCa tissues compared to normal tissues and was associated with poor prognosis (p < 0.05), particularly in advanced and metastatic stages. Receiver operating characteristic (ROC) analysis revealed a diagnostic AUC of 0.792 for ARF6. Functional analyses indicated that ARF6 regulates pathways critical to cell migration, invasion, and drug resistance. Moreover, ARF6 expression showed a strong negative correlation with immune checkpoint markers, such as PD-L1 (r = − 0.74), suggesting its potential as an immunotherapy target. These findings underscore ARF6’s pivotal role in Methuosis and its promise as a biomarker in PCa. Conclusion ARF6 is a key regulator of Methuosis in prostate cancer, contributing to tumor progression, metastasis, and resistance to treatment. Our findings support the potential of ARF6 as a diagnostic, prognostic, and immunotherapeutic target in prostate cancer. Further experimental validation is needed to confirm these observations and to explore the therapeutic implications of targeting ARF6 in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51c475c0dde283a17762878be80bdc39f46da628" target='_blank'>
              Methuosis key gene ARF6 as a diagnostic, prognostic and immunotherapeutic marker for prostate cancer: based on a comprehensive pan-cancer multi-omics analysis
              </a>
            </td>
          <td>
            Yongjin Yang, Liangliang Qing, Chengyu You, Qingchao Li, Wenbo Xu, Zhilong Dong
          </td>
          <td>2025-05-23</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The role of the proto-oncogene MYC as a driver of lymphoma has been known since the 1980s, but the mechanisms underlying its dysregulation are not completely understood. In this issue of Blood Cancer Discovery, Iyer and colleagues employ a CRISPR interference screen targeted at open chromatin regions to unveil enhancers critical for MYC overexpression in lymphoma, including a novel regulatory element in the MYC locus that controls germinal center reentry and is recurrently amplified in diffuse large B-cell lymphomas (germinal center B-cell diffuse large B-cell lymphoma). See related article by Iyer et al., p. XX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5717acd2d0393973aea7e3766b9be2a2a3a6876f" target='_blank'>
              One for All and All for One: New Insights into Enhancers Driving MYC Dysregulation.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2025-04-16</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 The Cancer Genome Atlas (TCGA) provides comprehensive genomic and transcriptomic data from over 10,000 cancer patients, along with metadata including viral infection status and survival information. We investigated the role of viral infections in dysregulating the expression of A3 enzymes (A3A/B/C/D/F/G/H). These enzymes primarily function to restrict viral infections by introducing C-to-U mutations in viral genomes. However, certain members (notably A3A and A3B) can also mutate cellular DNA, contributing to tumor initiation and progression. Cancers such as breast, bladder, cervical, head & neck, and skin exhibit characteristic C-to-T mutations attributed to the aberrant expression/activity of A3 enzymes. Accurate quantification of A3 dysregulation in tumor cells using TCGA bulk RNAseq data is complicated by the ubiquitous expression of these enzymes in infiltrating immune cells. This highlights a broader challenge in analyzing heterogeneous tumor biopsies to uncover virus-specific dysregulated processes in cancer. To address this challenge, we employed Nonnegative Matrix Factorization (NMF) to deconvolute A3 expression profiles from TCGA bulk RNA-seq data, i.e. to separate tumor cell profiles from those of the tumor microenvironment. We validated our NMF-extracted profiles by analyzing publicly available single-cell RNA-seq datasets from approximately 3,000 cancer projects in the Gene Expression Omnibus (GEO). Additionally, to identify cell types represented by these profiles, we compared them with experimentally obtained A3 expressions from purified cell types. Our analyses identified up to four distinct A3 profiles: two associated with T cells and NK cells, one linked to dendritic cells and macrophages, and one corresponding to cancerous epithelial cells. Notably, the A3 profiles derived from our analysis of single-cell RNA-seq data closely matched those deconvoluted from TCGA bulk RNA-seq data, demonstrating that our bulk RNA-seq deconvolution effectively captures single-cell population-level profiles. We found that both tumor-specific and immune cell-specific A3 profiles correlate with viral infection; however, significant variations were observed across different viral types and tumor types. None of these profiles were correlated with the extent of A3 mutational signatures SBS2 and SBS13.In conclusion, our study demonstrates that deconvolution of bulk RNA-seq datasets can reveal single-cell population-level insights into the cellular composition of tumor biopsies. Moreover, this approach offers the resolution needed to pinpoint virus-specific dysregulated processes within the complex and heterogeneous tumor microenvironment.


 Mohadeseh Soleimanpour-Moghadam, diako Ebrahimi. Role pf viral infection in APOBEC3 dysreguation in cancer: a single-cell gene expression profiling approach [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 990.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86b0ba33764c9febe1e75cc4ff8baa0ec4e4b55" target='_blank'>
              Abstract 990: Role pf viral infection in APOBEC3 dysreguation in cancer: a single-cell gene expression profiling approach
              </a>
            </td>
          <td>
            Mohadeseh Soleimanpour-Moghadam, diako Ebrahimi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Historically, DNA sequence mutability has been considered relatively uniform and low in tumors with chromosomal instability (CIN), based on the assumption that high mutability would be detrimental in karyotypically aberrant contexts. Recent in silico analyses have challenged this view, suggesting some heterogeneity in mutation rates across CIN tumors; however, these predictions lack experimental validation. It also remains unclear how the intertumor variability of mutation rates compares to intratumor diversification and evolves along disease progression, whether mutation rates are functionally relevant in CIN cancers, and which mutational processes shape mutational accrual during CIN tumor onset and evolution. To address these gaps, we performed mutation accumulation experiments using clonal populations of patient-derived tumoroids from seven CIN, microsatellite-stable colorectal cancers (CRCs), and one microsatellite-unstable CRC. Each tumor exhibited a distinctive mutation rate footprint that was conserved among different clones from the same ancestor. In contrast, mutation rates diverged markedly across different tumors, with variations in magnitude within microsatellite-stable tumors as prominent as those distinguishing them from microsatellite-unstable tumors. New mutations reflected mutational processes associated with defective DNA replication and repair, which were not detected in normal tissues. Last, both mutation accumulation assays and high-depth whole-exome sequencing of subclonal variants showed higher mutation rates in metastatic lesions compared with matched primary tumors, suggesting positive selection for cells with increasing mutability during cancer dissemination. By providing an empirical assessment of mutation rates in human cancer, our data delineate heterogeneity, heritability, and progression-associated evolvability of DNA mutational instability as hallmarks of microsatellite-stable CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c57088e09b71e7ad2e34a58e69bb450d1b43d18" target='_blank'>
              Heterogeneity and evolution of DNA mutation rates in microsatellite stable colorectal cancer.
              </a>
            </td>
          <td>
            Elena Grassi, V. Vurchio, George D. Cresswell, Irene Catalano, B. Lupo, F. Sassi, Francesco Galimi, Sofia Borgato, Martina Ferri, Marco Viviani, Simone Pompei, Gianvito Urgese, Bingjie Chen, E. Zanella, F. Cottino, Mariangela Russo, G. Mauri, F. Pietrantonio, M. G. Zampino, L. Lazzari, S. Marsoni, A. Bardelli, M. Lagomarsino, A. Sottoriva, L. Trusolino, Andrea Bertotti
          </td>
          <td>2025-05-21</td>
          <td>Science translational medicine</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Abstract Ageing is a gradual biological process marked by a decline in physiological function, increasing susceptibility to disease, and mortality. Transposable elements (TEs) are repetitive DNA sequences capable of moving within the genome and thus potentially inducing mutations and disrupting normal cellular functions. Their mobile nature contributes to genomic variation, as transposition events can alter gene expression, chromosome structure, and the epigenetic landscape. To mitigate TE-induced damage, cells rely on epigenetic mechanisms, such as DNA methylation, histone modifications, and small RNAs, to repress TE activity. However, these silencing mechanisms become less effective with age, leading to increased TE activation. This review explores the dual role of TEs as both a cause and consequence of ageing, suggesting a complex relationship between TEs and the ageing process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca3b56d174d42a8f2b0813e61df3277c11215871" target='_blank'>
              Exploring the Relationship of Transposable Elements and Ageing: Causes and Consequences
              </a>
            </td>
          <td>
            Miriam Merenciano, Anaïs Larue, Chloé Garambois, William Vilas Boas Nunes, Cristina Vieira
          </td>
          <td>2025-05-15</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bc8a5871308fcb63b1e32d25ce00cce571108d0" target='_blank'>
              Multigenerational cell tracking of DNA replication and heritable DNA damage.
              </a>
            </td>
          <td>
            Andreas Panagopoulos, Merula Stout, Sinan Kılıç, Peter Leary, Julia Vornberger, Virginia Pasti, A. Galarreta, Aleksandra Lezaja, Kyra Kirschenbühler, Ralph Imhof, H. Rehrauer, Urs Ziegler, Matthias Altmeyer
          </td>
          <td>2025-05-21</td>
          <td>Nature</td>
          <td>1</td>
          <td>41</td>
        </tr>

        <tr id="


 Cyclin E1 (CCNE1) amplification occurs in endometrial (40%), ovarian (20%), stomach (11%) and breast (3%) cancers and is associated with poor clinical outcomes. CCNE1 amplification deregulates the G1/S checkpoint and causes premature entry into S phase. Consequently, CCNE1 amplified cells depend on the G2/M checkpoint to ensure genomic stability before mitosis. PKMYT1, a member of the WEE1 family of enzymes, phosphorylates and inhibits CDK1 to regulate the G2/M checkpoint and prevent entry into mitosis. Here, we show that PKMYT1 inhibition in CCNE1-high cancer cells results in premature entry into mitosis, catastrophic DNA damage, and cell death. PKMYT1 inhibition also synergizes with chemotherapy to increase CCNE1-high cancer cell killing. Importantly, non-tumorigenic cells are not dependent upon PKMYT1 and do not exhibit DNA damage in response to PKMYT1 inhibition.



 An isogenic pair of CCNE1-low and CCNE1-high expressing cell lines was generated using lentiviral transduction. PKMYT1 was inhibited using Compound A, a tool small molecule PKMYT1 inhibitor. Inhibition of PKMYT1 was measured by CDK1 pT14 via western blot and HTRF. Cell cycle was monitored by flow cytometry using EdU and DNA content stains. DNA damage was measured by western blot and flow cytometry for yH2AX. Viability was measured with AlamarBlue and Caspase3/7. Bliss Synergy scores were calculated and used to assess synergy after combination treatments.



 CCNE1 overexpression increases entry into S phase and dependence on the G2/M checkpoint (CDK1 pT14 and pY15 levels) without changing basal levels of DNA damage. While PKMYT1 inhibition decreases CDK1 pT14 in both CCNE1-low and CCNE1-high cells, only CCNE1-high cells exhibit decreased S phase and increased SubG1 (apoptotic) and G2/M phases. CCNE1-amplified breast and ovarian cancer cells display similar inhibition of CDK1 pT14, reduction of S phase, and increased DNA damage and cell death as a consequence of PKMYT1 inhibition. Combination treatment of PKMYT1 inhibition and chemotherapy synergistically reduced cell viability in CCNE1-amplified cancer cells. Non-tumorigenic cells are not dependent upon PKMYT1 and do not exhibit changes in the cell cycle or DNA damage after PKMYT1 inhibition. In contrast, WEE1 inhibition is detrimental to non-tumorigenic cells and induces cell cycle dysregulation and DNA damage.



 The data presented here provide a rationale for targeting PKMYT1 in cancer cells that are dependent on the G2/M checkpoint to maintain genomic stability, such as CCNE1-amplified tumors. PKMYT1 inhibition selectively kills CCNE1-amplified cancer cells while sparing non-tumorigenic cells that have intact G1/S checkpoints. Our results highlight the importance of developing selective PKMYT1 inhibitors to avoid toxicities related to WEE1 inhibition.



 Samantha S. Hodge, Natalie Hill, Kayla Walzer, Ritu Kushwaha, Yihong Guan, Jeremy Fournier, Ester Fernandez-Salas. Targeting PKMYT1 to preferentially kill tumor cells that have CCNE1 amplifications [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5418.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0eed4347ff4f53b8d985b0523c9458323314af82" target='_blank'>
              Abstract 5418: Targeting PKMYT1 to preferentially kill tumor cells that have CCNE1 amplifications
              </a>
            </td>
          <td>
            Samantha S. Hodge, Natalie Hill, Kayla Walzer, Ritu Kushwaha, Yihong Guan, Jeremy Fournier, Ester Fernandez-Salas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Homology, or relationship among characters by common descent, has been notoriously difficult to assess for many morphological features, and cell types in particular. The ontogenetic origin of morphological traits means that the only physically inherited information is encoded in the genomes. However, the complexity of the underlying gene regulatory network and often miniscule changes that can impact gene expression, make it practically impossible to postulate a clear demarcation line for what molecular signature should "define" a homologous cell type between two deeply branching animals. In this Hypothesis article, we propose the use of the recently characterized irreversible genomic states, that occur after chromosomal and sub-chromosomal mixing of genes and regulatory elements, to dissect regulatory signatures of each cell type into irreversible and reversible configurations. While many of such states will be non-functional, some may permanently impact gene expression in a given cell type. Our proposal is that such evolutionarily irreversible, and thus synapomorphic, functional genomic states can constitute a criterion for the timing of the origin of deep evolutionary cell type homologies. Our proposal thus aims to close the gap between the clearly defined homology of the individual genomic characters and their genomic states to the homology at the phenotypic level through the identification of the underlying evolutionarily irreversible and regulatory linked states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1c7265ffd0750ec9e349f309b4f00f20641d114" target='_blank'>
              The application of irreversible genomic states to define and trace ancient cell type homologies
              </a>
            </td>
          <td>
            Oleg Simakov, Günter P. Wagner
          </td>
          <td>2025-05-03</td>
          <td>EvoDevo</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b17b40b0a07f9a17dc14ef97d051fc1d5f4203b" target='_blank'>
              Mechanisms and timing of programmed DNA elimination in songbirds
              </a>
            </td>
          <td>
            D. Dedukh, L. Malinovskaya, Ondřej Kauzál, Jakub Rídl, Daria Odnoprienko, Tatyana Karamysheva, Niky Vontzou, K. Janko, Alexander Suh, Tomáš Albrecht, Anna Torgasheva, R. Reifová
          </td>
          <td>2025-05-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 We have reported genetically-engineered mouse models for colonic neoplasia by using our novel CDX2P-Cre mice, in which Cre is restrictedly expressed in colonic epithelial cells. Although colonic tumors are generated in CDX2P-Cre;Apcflox/+ (CPC;Apc) mice with highly penetrant rate, greater number and deeper invasion of colonic tumors were observed in CDX2P-Cre;Apcflox/+;Ptenflox/+ (CPC;ApcPten), possibly resulting from haploinsufficiency of Pten gene. We have also confirmed shorter lifespan in CPC;ApcPten than that in CPC;Apc (AACR Annual Meeting 2018). Here, we test if these phenotypes come from one copy loss of Pten itself or secondary genetic and/or transcriptomic alterations induced by Pten loss by using whole genome sequence (WGS) and RNA sequence from macroscopically-dissected colonic tumors (bulk RNA-seq).



 WGS for five tumors and five livers as counterpart controls in each mouse model was performed by DNBSEQ platforms. Bulk RNA-seq was performed for three tumors in each model.



 In the analysis using B-allele frequency, copy-neutral LOH occurred in chromosome 18 including Apc loci specifically in colonic neoplasia in both CPC;Apc and CPC;ApcPten mice. In contrast, LOH was not observed in chromosome 19 including Pten loci in CPC;ApcPten mice. We also confirmed remarkably decreased coverage of reads in reference mapping in Apc locus, and there was a slight decrease of coverage in Pten locus. There were no additional driver gene mutations in CPC;Apc or CPC;Apc+Pten in WGS. All these results suggested just one copy loss of Pten resulting in aggressive tumor phenotype in CPC;ApcPten mice. In RNA-seq and subsequent GSEA comparing colonic tumors in CPC;ApcPten with those in CPC;Apc revealed upregulation of some signal pathways including mTORC1. Supporting this observation, number of colon tumors, especially adenocarcinomas, were decreased by rapamycin administration to CPC;Apc+Pten. Similarly, rapamycin was more effective in tumoroids from CPC;Apc+Pten those from CPC;Apc.



 WGS reveals one copy loss of Pten locus is the only genetic alteration in CPC;Apc+Pten when compared with CPC;Apc. One copy loss of Pten is sufficient to upregulate mTORC1 pathway in a CPC;Apc+Pten mouse model, resulting in promotion of tumor formation and invasion.



 Haruki Sada, Hiroaki Niitsu, Hikaru Nakahara, Naoya Sakamoto, Hirotaka Tashiro, Hideki Ohdan, Takao Hinoi. The effect of Pten haploinsufficiency on genomic and transcriptomic profiles in combined genetically-engineered mouse models for colonic neoplasia based on Apc inactivation [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3926.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/187ffa69b09f016709afee92a536f50201d6d4b2" target='_blank'>
              Abstract 3926: The effect of Pten haploinsufficiency on genomic and transcriptomic profiles in combined genetically-engineered mouse models for colonic neoplasia based on Apc inactivation
              </a>
            </td>
          <td>
            H. Sada, Hiroaki Niitsu, Hikaru Nakahara, Naoya Sakamoto, Hirotaka Tashiro, Hideki Ohdan, T. Hinoi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="


 Metastatic bladder cancer (BLCA) is an aggressive disease complicated by the emergence of variant histological subtypes and variable responses to treatment. Understanding the heterogeneity that exists both between these histological subtypes and within individual BLCA tumors is imperative in advancing precision medicine for BLCA.



 The University of Washington/Fred Hutchinson Cancer Center metastatic BLCA rapid autopsy program is a first-of-its-kind program collecting both primary and metastatic tumor samples from advanced BLCA patients, with a focus on aggressive variant subtype tumors. To explore intra-tumoral heterogeneity of both tumor and stromal compartments, single-nucleus RNA sequencing, in combination with matching bulk DNA and RNA sequencing, was performed on 15 tumors from 9 patients. The resulting dataset totaled 39,888 cells across a variety of histological subtypes: conventional urothelial carcinoma (UC), UC with squamous differentiation (UCSD), and plasmacytoid UC (PUC).



 Our unique snRNAseq dataset, the first in either metastatic or variant subtype BLCA, was used to dissect genomic and transcriptomic heterogeneity within BLCA tumors. On a genomic level, we observe subclonal copy number variations (CNVs) in BLCA driver genes in almost all (8/9) patients. Furthermore, we observe varying levels of intra-tumoral transcriptomic heterogeneity, where metastatic survival time is positively associated with tumor heterogeneity. Despite this heterogeneity, particular pathways define the epithelial transcriptomics of variant histological subtypes, including the upregulation of cell cycle and DNA repair upregulated in PUC tumors, which is validated in both bulk RNA sequencing and immunohistochemistry of a larger UW/FH BLCA rapid autopsy cohort. Investigation of the tumor microenvironment revealed that PUC tumors may be more immune-hot than UC, with increased B- and T-cell infiltration, while UCSD tumors have immunosuppressive microenvironments enriched in tissue-associated macrophages. Interestingly, these differences across subtypes are supported by changes in inter-cellular signaling between epithelial and fibroblast cells and the immune cells.



 These findings lay the groundwork for development of precision medicines for histological subtypes in BLCA by revealing heterogeneity and immune microenvironments that underlie disease aggressiveness and therapeutic resistance.



 Samantha L. Schuster, Pushpa Itagi, Sonali Arora, Thomas Persse, Patricia C. Galipeau, John K. Lee, Robert B. Montgomery, Jonathan L. Wright, Petros Grivas, Funda Vakar-Lopez, Andrew C. Hsieh, Hung-Ming Lam, Gavin Ha. Histological subtypes of metastatic bladder cancer have distinct tumor microenvironments and high intratumoral heterogeneity [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB468.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827f74e9a451d846709a409162aebd5ef66250d5" target='_blank'>
              Abstract LB468: Histological subtypes of metastatic bladder cancer have distinct tumor microenvironments and high intratumoral heterogeneity
              </a>
            </td>
          <td>
            Samantha L. Schuster, P. Itagi, Sonali Arora, Thomas W Persse, Patricia Galipeau, John K. Lee, R. Montgomery, Jonathan L. Wright, P. Grivas, F. Vakar‐López, Andrew C. Hsieh, Hung-Ming Lam, Gavin Ha
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The characterization of somatic variation, especially in complex genomic regions, is crucial for understanding the molecular drivers of cancer progression. Accurate PacBio long-read sequencing (HiFi) enables detection of all variant classes, from simple SNVs and INDELs up to complex structural variation, tandem repeats, and changes in epigenetic signatures. Complex and repetitive regions, while fully sequenced by HiFi reads, remain bioinformatically challenging, requiring tailored solutions. Here, we describe new tools to genotype understudied repetitive regions in cancer genomes, a task that has historically posed significant challenges to short-read sequencing.Our software tool, TRGT, resolves the sequence and methylation patterns of complex repetitive regions at a single-nucleotide resolution. A recent algorithmic improvement (vclust module) enables us to type expanded hyper-polymorphic regions, termed variation clusters. Variation clusters usually result in many incorrect variant calls in computational pipelines. By analyzing a variation cluster as a single region, rather than many small variants, we can accurately measure genetic variation. To demonstrate the importance of variation clusters in cancer, we comprehensively resolved repeat regions in five cell lines and 50 clinical tumor samples (with matched healthy tissue). We identified 5,500 new variation clusters across the tumors and an average of 46,220 (out of 4 million, 1%) loci that are contracted or expanded in a tumor compared to its matched normal tissue. We find 3% of these loci can be located within genes found in the Compendium of Cancer Genes, suggesting potentially oncogenic significance. In addition, using TRGT, we were able to detect microsatellite instability (MSI) in a tumor with a compound heterozygous coding mutation in MSH2. Lastly, we further analyzed haplotypes in selected variation clusters to detect and resolve sequences in somatic mosaicism. By incorporating this new class of variants—complex repeats—into our analysis, we significantly enhanced the capability of HiFi reads to detect a comprehensive spectrum of genomic variations. The inclusion of complex repeat genotyping with PacBio HiFi reads will enable a more detailed understanding of cancer genetics and open new avenues for research and clinical applications.


 Khi Pin Chua, Egor Dolzhenko, Zev N. Kronenberg, Seiya Imoto, Seiichi Mori, Michael A. Eberle. Precise characterization of complex repeats in cancer genomes [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6288.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f99119d1610004635e935f3cbda8df6a5ed0a6" target='_blank'>
              Abstract 6288: Precise characterization of complex repeats in cancer genomes
              </a>
            </td>
          <td>
            Khi Pin Chua, E. Dolzhenko, Z. Kronenberg, Seiya Imoto, Seiichi Mori, M. Eberle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 The spatial organization of chromatin is a fundamental determinant of epigenomic landscape which is widely investigated using high-throughput chromosome conformation capture (HiC) technique. To address challenges associated with the resolution and variability of HiC technique, we applied a novel workflow, HiC-ECC (HiC Enhance, Compare, and Call). This approach not only enhances HiC data quality but also enables the identification of biologically significant chromatin interactions and genome structural features. By integrating advanced computational tools with a focus on colorectal cancer (CRC) biology, this study sheds light on the 3D genome architecture of CRC organoids and its implications for patient stratification and therapeutic targeting.



 We applied the HiC-ECC workflow to a cohort of 15 patient-derived CRC organoid models, including samples derived from primary tumors and metastatic sites in the lung and liver. The workflow involved initial data preprocessing to generate high-quality chromatin contact matrices and ensure rigorous quality control. To enhance the resolution of HiC datasets, we utilized deep learning-based methods, enabling more accurate identification of chromatin interactions. Comparative analyses were conducted to identify differential chromatin interaction patterns and conserved structural features across the cohort. Chromatin architecture was further characterized by delineating topologically associating domains (TADs) and clustering them to reveal common and sample-specific interaction networks, offering insights into the hierarchical organization of 3D genome structures.



 Our analysis revealed four distinct subtypes of CRC organoids based on their 3D genome architecture, revealing a novel approach to interrogate heterogeneity within CRC. This subtyping was driven by differential chromatin interaction patterns and structural features, including unique TADs and conserved regulatory elements. Comparative analysis highlighted subtype-specific chromatin interaction networks, suggesting distinct epigenomic landscapes that correlate with tumor progression and metastatic potential.



 This study enhances our understanding of CRC epigenomics and lays the groundwork for integrating chromatin architecture into patient stratification and precision therapy.



 Amin Nooranikhojasteh, Jakob Zerbs, Ghazaleh Tavallaee, Elias Orouji. A novel framework for deciphering 3D genome structure in colorectal cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 215.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d11e895cea0cf1e967c3e4adda2d90a1183e147" target='_blank'>
              Abstract 215: A novel framework for deciphering 3D genome structure in colorectal cancer
              </a>
            </td>
          <td>
            Amin Nooranikhojasteh, Jakob Zerbs, Ghazaleh Tavallaee, Elias Orouji
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 1049


 Background:
 Although invasive lobular carcinoma (ILC) is often classified as a separate breast cancer (BC) subtype with distinct molecular features, options for diagnosis and treatment remain similar to other BCs. Using an integrated histomolecular approach to classify 617 BC samples into either histomolecular ILC (hmILC) or histomolecular no special type (hmNST) subsets, we compared their macroscale genomic alterations to describe biological traits of the ILC BC subtype, which may lead to improved approaches in BC therapies.
 Methods:
 A total of 617 BC FFPE samples were subject to whole-exome and bulk RNA sequencing analysis. hmILC subset was defined based on CDH1 truncation/deletion or low CDH1 expression (z-score < -2.5×MAD), while all other samples were classified as hmNST. Copy number variations (CNVs) were assessed using Sequenza to detect recurrent amplifications/gains and deletions; homologous recombination deficiency (HRD) scores were calculated based on large-scale state transitions and loss-of-heterozygosity events; tumor mutational burden (TMB) scores were evaluated as percent of mutations per megabase; and mutational signatures were deconvoluted using maftools R package.
 Results:
 Genome-wide CNV analysis revealed distinct patterns in hmILC compared to hmNST. hmILC showed hallmark deletions at regions harboring
 CDH1
 (16q), while gains were observed significantly more frequently at regions harboring
 FCGR3A
 (1q) compared to hmNST. Elevated APOBEC activation signature expression was found in 32% hmILC vs. 19% hmNST samples (p = 0.002, chi-squared test). HRD-positive cases were less frequent in the hmILC subset (25%) compared to hmNST (42%) (p = 0.001). In contrast, hmILC tumors more frequently demonstrated high TMB scores compared to hmNST (10% vs. 2%, p = 0.0003). Moreover, TMB-positive hmILC samples were mostly HRD-negative.Genetic alterations in genes like
 FANCA
 ,
 FANCD2
 and
 PALB2
 were more frequently enriched in hmILC tumors compared to hmNST (p < 0.1). Furthermore, hmILC subset with high HRD scores frequently harbored additional DNA repair gene mutations (e.g.,
 BRCA2
 ) compared to the subset with low HRD scores (p = 0.02).
 Conclusions:
 This study revealed macroscale genomic alterations, such as unique CNV patterns, altered distributions of HRD- and TMB-positive cases and increased APOBEC-driven mutational processes, in the hmILC BC subset. These distinct genomic architectures highlight the need for innovative trials using inhibitors of DNA repair and related pathways for hmILC BC patients, particularly in those with high HRD-high tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/857e4886ff3022df5219469c1feb1ae7a81adbc1" target='_blank'>
              Macroscale genomic alterations in histomolecular invasive lobular carcinoma compared to other breast cancer subtypes.
              </a>
            </td>
          <td>
            J. Mouabbi, Vladimir Kushnarev, Daria F. Goncharova, Oleg Baranov, Konstantin Chernyshov, N. Kotlov, Francesca Paradiso, Michael Hensley, R. Layman, Debasish Tripathy, Paula Pohlmann, F. Meric-Bernstam
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Double‐strand breaks (DSBs) are universally acknowledged as the most detrimental type of DNA damage, and their effective repair primarily depends on the non‐homologous end joining (NHEJ) pathway. Such DSBs, which require NHEJ for resolution, can arise from intrinsic and extrinsic DNA‐damaging factors or emerge naturally during essential biological processes like V(D)J recombination and antibody class switch recombination.Failure to properly repair DSBs may lead to genomic instability, disruption of cellular functions, and immunodeficiency, thereby promoting the development of hematologic malignancies. Conversely, overexpression of NHEJ‐related genes can enhance resistance to DNA‐damaging therapies in these cancers. Analyzing mutations in key classical NHEJ (cNHEJ) components and understanding their mechanisms could provide valuable biomarkers for predicting therapeutic outcomes and guiding treatment decisions. Consequently, defects in cNHEJ may offer insights into the development of novel drugs targeting DNA repair pathways.We focus on genetic changes and alterations in gene regulation, while also providing an overview of cNHEJ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37d192621ee63d84a5e302bfd454c006794e79d5" target='_blank'>
              Current advances in the role of classical non‐homologous end joining in hematologic malignancies
              </a>
            </td>
          <td>
            Pengcheng Liu, Zi-zhen Xu
          </td>
          <td>2025-05-29</td>
          <td>Clinical and Translational Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is a devastating malignancy with a 5-year overall survival (OS) rate of approximately 12%. More than 90% of PDAC patients harbor oncogenic mutations in the Kirsten rat sarcoma viral homolog (KRAS) gene. MRTX1133 (MRTX), a novel inhibitor of KRASG12D (the most common KRAS mutation found in pancreatic and colon cancers) has shown promise as a therapeutic agent. To address reported resistance to MRTX, we adapted our anti-leukemia co-targeting strategy and evaluated a combination of MRTX and Bedaquiline (BED), an FDA-approved inhibitor of mitochondrial ATP production, in in vitro human PDAC models. The combination of MRTX and BED demonstrated enhanced cytotoxic effects by disrupting all 11 genes within the DNA helicase family (CMG complex: CDC45-MCM-GINS), which are essential for initiating DNA replication and regulating cell cycle progression. Notably, real-world data analysis from Caris Life Sciences and NCI-TCGA database revealed that low transcriptomic expression of the DNA helicase CMG complex was significantly associated with prolonged survival (e.g., low CDC45 expression and low GINS2 expression with greater than 8 months longer overall survival) in PDAC patients with KRASG12 mutations (N = 9,717; P < 0.00001). However, this combination therapy also triggered strong pro-survival nuclear reprogramming. This effect was mediated by significant genetic activation of an NFκB2-DDIT (DNA damage-induced transcript) axis, which supported tumor chromosomal integrity and DNA repair mechanisms. To overcome NFκB2-driven resistance mechanisms, we explored a triple-targeting strategy that addresses metabolic and genomic plasticity in addition to actively intercepting cell division. This approach combines MRTX1133, Bedaquiline, and the NFκB2 inhibitor SN52, offering a novel therapeutic avenue to treat aggressive pancreatic cancer and potentially improve patient outcomes. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00669-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecf14d3f8785e3a0d9d03809564ba14faf0a6c7" target='_blank'>
              Targeting DNA helicase CMG complex and NFκB2-driven drug-resistant transcriptional axis to effectively treat KRASG12D-mutated pancreatic cancer
              </a>
            </td>
          <td>
            Jeffrey Xiao, Joshua Kim, Brandon Park, David J. Baylink, Cedric Kwon, Victoria Tran, Scott Lee, Kevin Codorniz, Laren D Tan, Pamela Lobo Moreno, Amy Schill-Depew, S. Mirshahidi, David de Semir, Diana Hanna, Kiran Naqvi, H. Cao, Chien-Shing Chen, Joanne Xiu, H. Lenz, Hamid Mirshahidi, Mark E. Reeves, Yi Xu
          </td>
          <td>2025-05-26</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Viral infections have been demonstrated to influence host gene expression and drive tumorigenesis. This study aims to systematically annotate genome integration and expression of ∼3000 exogenous and human-endogenous viral species across all public DepMap cancer cell lines, representing a diverse set of cancer types and lineages. Our preliminary results from this analysis pipeline successfully recapitulated known Human Papillomavirus (HpV strains 16, 18, 30 and 53) signatures in 12 cervical, 2 head and neck, and 4 engineered prostate cancer cell lines, demonstrating sensitivity at both the strain and gene levels. Associating these with genome-wide CRISPR screening data, we identified dependencies that were more correlated with viral presence rather than their respective lineages. Additionally, we observed expression of Epstein-Barr Virus(EBV) in multiple cancer types, a majority of which were of lymphoid lineage. We also found expression of the env gene of Human Endogenous Retrovirus-K (HERV-K113) in multiple lineages, with the highest average expression in skin cancers. By analyzing whole-genome sequencing we seek to identify viral integration sites in the cell lines’ genomes, to complement viral gene expression. From this large-scale multimodal analysis of features derived from genomic and transcriptomic data combined with gene dependencies, we aim to uncover potential viral contributions to oncogenesis and biomarker or target discovery. This will provide a foundational resource for understanding the role of viruses in cancer, offering new insights into viral-driven mechanisms of disease.


 Sowmya Iyer, Moore van Tienen, Phillipp Trollman, Yao He, William R. Sellers, Catarina D. Campbell. Comprehensive quantification and annotation of viral gene integration and expression across thousands of human cancer models [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2391.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9235dd3b879fd0da050d8e59db442b8291733b6d" target='_blank'>
              Abstract 2391: Comprehensive quantification and annotation of viral gene integration and expression across thousands of human cancer models
              </a>
            </td>
          <td>
            Sowmya Iyer, Moore van Tienen, Phillipp Trollman, Yao He, William R. Sellers, Catarina D. Campbell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9270b9d8496ccc10f9651565b149fb129450105b" target='_blank'>
              ATM and p53 in aging and cancer: a double-edged sword in genomic integrity.
              </a>
            </td>
          <td>
            Surya Nath Pandey, Muhammad Afzal, Jyoti Uikey, Subbulakshmi Ganesan, Swati Mishra, Pooja Bansal, Imran Kazmi, S. Alzarea, Waleed Hassan Almalk, Kavita Goyal, Gaurav Gupta, Mohit Rana
          </td>
          <td>2025-05-05</td>
          <td>Biogerontology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Cancer drug resistance arises not only from selection of resistant clones, but also through rapid activation of adaptive transcriptional programs. One mechanism of transcriptional regulation involves N6-methyladenosine (m6A) RNA modification, which dynamically regulates mRNA processing and alternative splicing, ultimately impacting cell fate and differentiation. In prostate cancer (PC), resistance to systemic therapies such as the androgen receptor pathway inhibitor (ARPI) enzalutamide is associated with a host of well-documented androgen receptor (AR) alterations, including amplification, mutation, and alternative splicing. Given these functions, we hypothesized that m6A modifications play a role in the transition to enzalutamide resistance in PC. To test this, we used methyl-RNA-immunoprecipitation followed by sequencing (MeRIP-seq) in parallel with RNA-seq to identify gene transcripts that were both differentially methylated and differentially expressed between enzalutamide-sensitive and enzalutamide-resistant PC cells. We filtered and prioritized these genes using clinical and functional database tools, including Gene Ontology (GO) enrichment analysis and Gene Set Enrichment Analysis (GSEA), The Cancer Genome Atlas (TCGA), and the Oncology Research Information Network (ORIEN) avatar. Using this approach, we identified 487 transcripts that were both differentially methylated and differentially expressed and validated six of the top 12 candidates via targeted qPCR and MeRIP-PCR. One of these, THBS1, was found to have increased m6A level associated with decreased transcript levels in enzalutamide-resistant cells, a finding recapitulated in publicly available preclinical and clinical data. Moreover, in enzalutamide-sensitive cells, depletion of THBS1 by siRNA-knockdown induced resistance to enzalutamide. While THBS1 has previously been implicated in aggressive PC phenotypes, we now show that THBS1 downregulation directly contributes to a rapid transition to enzalutamide resistance, suggesting a novel role for this gene in PC hormonal therapy resistance. These results constitute the first comprehensive epitranscriptomic profiling of ARPI resistance and identify THBS1 as a potential driver of acute resistance in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7bc3dab4afa38c6c5372774a5530ea7583082a1" target='_blank'>
              Combined Transcriptomic and Epitranscriptomic Profiling Identifies THBS1 as A Regulator of Enzalutamide Resistance in Prostate Cancer
              </a>
            </td>
          <td>
            Emmanuelle Hodara, Lisa Swartz, Aubree Mades, Daniel Bsteh, Tong Xu, S. Rhie, A. Goldkorn
          </td>
          <td>2025-04-21</td>
          <td>Cancer heterogeneity and plasticity</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Genotoxic therapies such as ionizing radiation eliminate cancer cells by inducing extensive DNA damage but often cause normal tissue toxicity, including cutaneous injury. Extrachromosomal circular DNA (eccDNA) refers to circular DNA fragments outside the chromosomal context, with their formation and persistence linked to DNA damage repair and genomic instability. Despite growing recognition of eccDNA in oncogenesis, its role under genotoxic stress in normal tissues remains poorly understood. Here, eccDNA is profiled in irradiated rat skin using Circle-seq, identifying alterations in eccDNA number and composition. Specifically, radiation induced circle17:44148731-48208624, in which vacuolar protein sorting 41 homolog (VPS41) is the sole radiation-induced amplification gene by semiquantitative PCR and gel electrophoresis. The findings show that eccDNA or VPS41 overexpression reduces radiation-induced skin injury (RISI) in vitro and in vivo. Proteomic and interaction analyses identified metastasis suppressor kangai-1 (KAI1) as a VPS41-interacting partner. Notably, VPS41 overexpression promotes KAI1 lysosomal degradation, protecting against radiation-induced apoptotic cell death. Peptide array analysis pinpoints the VPS41-KAI1 interaction through the K263 residue, consistent with AlphaFold prediction. The findings uncover a novel mechanism in which radiation-induced eccDNA, specifically VPS41, mitigates skin injury by modulating KAI1 degradation. This study highlights the role of eccDNA in cellular defense, providing strategies to enhance tissue resilience to genotoxic stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821c77d7a2a02d4160cf107b29b6628f9d6cb1ed" target='_blank'>
              EccDNA-Driven VPS41 Amplification Alleviates Genotoxic Stress via Lysosomal KAI1 Degradation.
              </a>
            </td>
          <td>
            Bin Shi, Ping Yang, Huaijin Qiao, Jinchen He, Bin Song, Hao Bai, Fengdi Jiang, Yining Zhang, Qian Li, Tao Yan, Wenling Tu, Daojiang Yu, Shuyu Zhang
          </td>
          <td>2025-04-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Cervical cancer progresses through distinct precancerous stages, making early screening and intervention crucial for prevention. However, conventional screening modalities, such as cytology and HPV testing, face challenges related to sensitivity, specificity, and resource dependency. Circular RNAs (circRNAs), owing to their high stability and tissue-specific expression, have emerged as promising biomarkers, though their role in cervical carcinogenesis remains underexplored. In particular, the clinical utility of circRNAs for optimizing cervical cancer screening and early diagnosis has yet to be established. This study aimed to investigate the dynamic expression profiles of circRNAs across various stages of cervical cancer progression and identify potential biomarkers to enhance early detection. Methods CircRNA sequencing was performed on cervical tissues spanning normal cervical epithelium (NCE), high-grade squamous intraepithelial lesions (HSIL), and cervical squamous cell carcinoma (CSCC). Functional assays, including cell viability, colony formation, and apoptosis, were performed to assess the oncogenic potential of circPOLD1 and its interaction with YBX1 in cervical cancer cells. BaseScope and immunohistochemistry (IHC) were applied to tissue microarrays for clincial validation and ROC curve analysis evaluated the diagnostic performance of circPOLD1 in serum as a liquid biopsy marker. Results CircRNA profiling revealed a progressive increase in circPOLD1 expression from NCE to HSIL and CSCC. Mechanistically, circPOLD1 functioned as an oncogene by binding to and phosphorylating YBX1, activating the AKT/mTOR/HIF-1α pathway to enhance glycolysis-driven tumorigenesis. BaseScope and IHC confirmed the stage-specific elevation of circPOLD1 and YBX1 in cervical lesions. The circPOLD1/YBX1 multi-marker panel demonstrated superior diagnostic performance, achieving an AUC of 0.951 for LSIL+ and 0.817 for HSIL+ detection. Furthermore, serum circPOLD1 levels exhibited a progressive increase across disease stages, underscoring its potential as a non-invasive biomarker. Conclusion circPOLD1 and YBX1 synergistically drive cervical carcinogenesis and exhibit stage-specific expression patterns. Their combined detection significantly enhanced the accuracy for cervical cancer screening and dynamic monitoring. The successful application of BaseScope and IHC highlights the immediate translational potential of these biomarkers, paving the way for refined risk stratification, improved therapeutic targeting, and reduced cervical cancer burden through early intervention. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06494-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f23f6664f03d6924fc8f8933932ffe6615430fec" target='_blank'>
              The biomarker potential of circPOLD1 and its binding protein YBX1 in cervical carcinogenesis
              </a>
            </td>
          <td>
            Lu Zhao, Xin Chen, Yanan Zhang, Y. Cen, Tingjia Zhu, Lingfang Wang, L. Xia, Yang Li, Xiaodong Cheng, Xing Xie, Weiguo Lu, Junfen Xu
          </td>
          <td>2025-05-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Chromosomal instability (CIN), a pervasive feature of cancer, promotes tumor evolution and inflammatory signaling, yet its influence on innate immune sensing remains incompletely understood. Ruptured micronuclei, a direct byproduct of CIN arising from missegregated chromosomes, expose out-of-context double-stranded DNA that engages the cGAS-STING pathway. In their recent study, Sasaki et al. show that micronuclei are also a source of immunogenic double-stranded RNA (dsRNA), triggering MAVS-dependent type I interferon responses independently of STING. The authors show that micronuclei undergo aberrant transcription, producing dsRNA from nonexonic, transcriptionally accessible loci, with many species localizing near interferon-stimulated genes. This work suggests a feedforward loop in which type I interferon signaling reinforces its own activation through transcriptional dysregulation. Using MPS1 inhibition to induce acute CIN, the authors show that MAVS signaling promotes MHC Class I expression and immune cell recruitment. These findings reposition CIN as a dual trigger of innate immunity through cytoplasmic DNA and RNA sensing. Future work should define how these pathways integrate in the context of chronic CIN and evaluate strategies to target DNA and RNA sensing in immune-edited tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/828c0bc85c3a2a64866df460ecbc5473d78feef8" target='_blank'>
              CINs of the cytoplasm: dissecting dsRNA signaling in chromosomal instability.
              </a>
            </td>
          <td>
            Aglaia Skolariki, Rose L Jady-Clark, Eileen E Parkes
          </td>
          <td>2025-05-07</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: DNA replication and sister chromatid cohesion1 (DSCC1) is a component of the alternative replication factor C complex (RFC), which plays an important role in sister chromatid cohesion and regulates the cell cycle. DSCC1 is reported to have increased expression in tumor progression, but the underlying mechanisms of DSCC1 in tumor immunity remain obscure. Methods: Data were collected from The Cancer Genome Atlas (TCGA) and Genotype-Tissue Expression (GTEx) project to obtain DSCC1 expression and clinicopathologically relevant data. Differential expression of DSCC1 and its association with prognosis, tumor microenvironment, immune infiltration, immune regulation, and genomic stability in various cancers were analyzed using R software. Results: The results of pan-cancer analysis showed that DSCC1 expression was elevated in 22 tumors. In 15 tumors, high expression of DSCC1 was observed to correlate with dismal overall survival. Furthermore, a correlation between DSCC1 expression and immune checkpoints was determined. In addition, DSCC1 expression was correlated with tumor-infiltrating immune cells, especially in Thymoma (THYM). Finally, gene set enrichment analysis (GSEA) demonstrated that DSCC1 is critically involved in tumor proliferation, immunity, and metabolism. Conclusions: High DSCC1 expression is found in many common tumors and is associated with poor prognosis. The gene is an essential factor in sister chromatid cohesion and may contribute to tumor progression by affecting the tumor immune microenvironment and genomic stability. DSCC1 has the potential to be a prognostic marker, and therapies targeting it may benefit patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175f7489907d723129eee83d91ea9ec6a461cb27" target='_blank'>
              Gene DSCC1 is a Potential Biomarker in Pan-cancer
              </a>
            </td>
          <td>
            Jialing Jiang, Xiaolong Yan, Dan Huang, Jieyu Chen, Xiaodan Hao
          </td>
          <td>2025-05-29</td>
          <td>Journal of Contemporary Medical Practice</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Aneuploidy, prevalent in most solid tumors, significantly alters cellular phenotype and fitness. Despite its critical role in tumor evolution, a detailed quantitative understanding of its evolutionary dynamics remains elusive. To address this gap, we developed ALFA-K, a novel method to infer the fitness landscape of chromosome-level karyotypes. Utilizing longitudinal single-cell karyotype data from evolving cell populations, our method estimates the fitness of thousands of karyotypes proximal to the input data in karyotype space, enabling predictions of unseen karyotypes. We validated our approach using both synthetic data from an agent-based model (ABM) and empirical data from in vitro and in vivo passaged cell lines. The empirical data comprised over 40, 000 sequenced cells across five cell lines, each sequenced longitudinally at 5-10 timepoints per lineage. Analysis of the fitted landscapes yielded several key insights: 1) WGD alters the distribution of fitness effects of copy number alterations (CNAs) (K.S. test: P<10-). Specifically, karyotypes with WGD exhibit more mutations with positive selection coefficients compared to diploid karyotypes; 2) Mutations in cisplatin-treated lineages showed changes in fitness effects on average 2.3 times larger than those in untreated lineages (t-test: P<2.2×10-1); 3) The fitness impact of a CNA varies depending on the parental karyotype; 4) The rate of chromosome missegregation can dictate the predominant karyotypes in an evolving population.
 To further investigate how the tumor microenvironment shapes karyotype evolution, we are integrating these inferred fitness landscapes into an existing spatial ABM. This modeling simulates the spatially heterogeneous tumor microenvironment and its influence on karyotypic selection and adaptation. By leveraging this ABM, we aim to capture the effects of spatial constraints—such as resource availability, competition, and cellular interactions—on the fitness and distribution of karyotypes. Complementing these simulations, we are developing a bioinformatics pipeline for spatial transcriptomics data from glioblastoma clinical samples. This dataset includes samples from various regions (normal, tumor, and tumor-infiltrating) across 31 samples from 19 patients, using 10X Visium with an average of 3, 100 spots per sample. The pipeline is designed to provide detailed information about the spatial organization of karyotypic diversity, elucidating how aneuploid karyotypes are distributed and interact within the tumor microenvironment. By integrating this spatially resolved data with the ABM, we aim to generate a comprehensive framework to explore how microenvironmental pressures in glioblastoma tumors shape the evolution of karyotypic diversity and drive the emergence of therapeutic resistance in patient-specific contexts.


 Richard J. Beck, Noemi Andor. Decoding tumor evolution through fitness landscapes of aneuploid cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2486.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57a62390eeb2a72ca937d2e747b4a191022a5bb6" target='_blank'>
              Abstract 2486: Decoding tumor evolution through fitness landscapes of aneuploid cells
              </a>
            </td>
          <td>
            Richard Beck, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d82d634c4ca446f74e247158a8446b3ecb227ab" target='_blank'>
              MicroFinder: Conserved gene-set mapping and assembly ordering for manual curation of bird microchromosomes
              </a>
            </td>
          <td>
            TC Mathers, M. Paulini, Cibele G Sotero-Caio, Jonathan M. D. Wood
          </td>
          <td>2025-05-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7706b7ef66d1fdcd1d30d7b318635c18156686f3" target='_blank'>
              Suppression of plastid-to-nucleus gene transfer by DNA double-strand break repair.
              </a>
            </td>
          <td>
            Enrique Gonzalez-Duran, Xenia Kroop, Anne Schadach, Ralph Bock
          </td>
          <td>2025-05-16</td>
          <td>Nature plants</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="
 11529


 Background:
 Epigenetic dysregulation is a key mechanism in cancer biology, driving both immune evasion and the generation of potentially druggable neoantigen targets. Transposable elements (TEs) are proviral sequences mostly found in epigenetically repressed genomic regions. In cancer, TEs are one potential source of neoantigens. The tumor suppressor gene
 ATRX
 is an epigenetic regulator that is lost in up to 30% of soft tissue sarcomas. Loss of function of this heterochromatin modulator leads to widespread alterations to transcription patterns. Our prior work in sarcoma, along with studies in other tumors, showed that ATRX loss leads to the de-repression of TEs. We hypothesize that this mechanism leads to TE neoantigen presentation on MHC class I (MHC-I) molecules.
 Methods:
 Leveraging the gEVE database of proviral sequences with open reading frames, we examined the differential expression of TEs using RNA sequencing of multiple models comparing ATRX lost and ATRX wild-type tumors. Analysis included our previously published data from human undifferentiated pleomorphic sarcoma cell lines (2 clones, 3 replicates per group; GEO: GSE240030) and a publicly available dataset from a mouse model of soft tissue sarcoma (n = 3 per group; GEO: GSE167537). Additionally, we analyzed analogous data from a mouse model of glioblastoma multiforme (GBM), another malignancy with frequent ATRX loss (n = 3 per group; GEO: GSE178113). RNA-seq data was analyzed using Salmon for transcript-to-genome pseudo-alignment, DESeq2 for differential TE expression analysis, and arcasHLA for sample-level MHC-I subtype identification. Differentially overexpressed TEs in ATRX mutant tumors were analyzed for predicted neoantigens using the NetMHCpan4.1 package.
 Results:
 Across multiple models, ATRX loss of function alterations were associated with both activation and repression of TEs. Within two sarcoma contexts, upregulated TEs were found to have protein-coding sequences with the potential to lead to MHC-I neoantigen presentation. Within a human undifferentiated pleomorphic sarcoma cell line with ATRX loss, there were 7 upregulated TEs shared between the clones, each predicted to contain 12 – 50 MHC-I binding epitopes. Within a mouse soft tissue sarcoma model, 7 TEs were upregulated with ATRX loss, each with 5 – 28 MHC-I binding epitopes. Since ATRX loss alters TE expression in other malignancies, parallel data from a mouse GBM model was additionally analyzed. In this model, 308 TEs were found to be upregulated with ATRX loss, and all were predicted to generate neoantigens. ERVs and LINE1 elements were recurrently identified, making up 97 – 100% of the activated TEs.
 Conclusions:
 Transposable element transcription altered by ATRX loss in sarcoma and GBM creates a predicted source of MHC-I-binding neoantigens. These neoantigens come from TEs of the ERV and LINE1 families.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d406b838be4f31bb8e68fe40b2f943691a36673" target='_blank'>
              Predicting transposable element-derived neoantigens in ATRX-mutated sarcoma.
              </a>
            </td>
          <td>
            Simon Cao, Ziyang Wang, B. Nacev
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND
Mitochondria have their own genome (mtDNA), which in humans is a circular multi-copy genome consisting of 16 569 base pairs. Abnormalities in the mtDNA have been reported to correlate with various age-related pathophysiologies.


METHODS
Based on a total of 182 DNA samples extracted from gastric cancer tissues, we measured mtDNA copy numbers (mtDNA-CN) using real-time polymerase chain reaction (PCR) and then examined alongside sex, age, tumor stage, Laurén classification, and the overexpression of Human Epidermal Growth Factor Receptor 2 (HER2).


RESULTS
We found no sex differences in mtDNA-CN and no correlation with age, but significant differences according to tumor stage. The mtDNAcn of intestinal type by Laurén classification was significantly larger than that of diffuse type. There was no significant difference in mtDNA-CN between HER2-positive and -negative tissues. Multiple regression analyses showed that only the tumor stage was a significant variable, while Laurén classification was not.


CONCLUSION
These results indicate that mitochondrial genomic abnormalities contribute the progression of gastric cancer independently of HER2 overexpression, and may shed light on the emerging role of mtDNA-CN in situ as a possible biomarker for estimating cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d05f154cb832f6f7744c5bac28235208f5d152b7" target='_blank'>
              Mitochondrial DNA copy numbers in gastric cancer tissues: a possible biomarker for estimating cancer progression.
              </a>
            </td>
          <td>
            Hisanori Fukunaga, Mayuko Fukunaga
          </td>
          <td>2025-04-23</td>
          <td>Japanese journal of clinical oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Mitosis, the shortest and most dynamic phase of the cell cycle, entails a complex sequence of events that result in the equal distribution of genetic material to two daughter cells. It has been proposed that mitochondrial respiration increases during mitosis to ensure sufficient ATP for this highly energy-demanding process. However, beyond bioenergetics, how concomitant metabolic changes influence mitotic progression and DNA inheritance remains unclear. To visualize these metabolic changes, we introduce fluorescent-based, genetically encoded reporters into cell lines and human tumor organoids, targeting different cellular compartments to track metabolic and redox dynamics during mitosis with high temporal and subcellular resolution. We observe an increase in ATP production in mitochondria and, notably, an increase in the NAD+/NADH ratio in mitotic chromatin upon nuclear envelope breakdown. Strikingly, disrupting this redox change at mitotic entry alters the epigenetic landscape, leading to a hyperacetylated chromatin state that affects chromatin compaction and Aurora B activity. Importantly, this series of events, initiated by metabolic perturbation, induces mitotic errors that result in genetically distinct daughter cells. Altogether, these findings uncover a novel role for mitochondria in regulating genetic inheritance through NAD(H) redox control, a mechanism with the potential to underlie genomic instability in cancer, where lactate accumulation due to the Warburg effect perturbs redox homeostasis.


 Maria J. Rodriguez Colman. Multifaceted mitochondria in the regulation of chromosome inheritance [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY22-03.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb8c843c17a6c70005621aa61b21710d77ab8472" target='_blank'>
              Abstract SY22-03: Multifaceted mitochondria in the regulation of chromosome inheritance
              </a>
            </td>
          <td>
            M. J. Rodríguez Colman
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="BACKGROUND
Epidermal growth factor receptor variant III (EGFRvIII) mutation is the most prevalent genetic change in glioblastoma. Abnormal DNA damage repair caused by EGFRvIII, which leads to temozolomide (TMZ) resistance, is a major cause of reduced postoperative survival in glioblastoma patients. This study aims to uncover the molecular mechanism of TMZ resistance in EGFRvIII-mutant glioblastoma.


METHODS
We constructed a CRISPR/Cas9 library to identify synthetic lethal genes for EGFRvIII-bearing cells. Abnormal epigenetic regulation of the RAD51-associated protein 1 (RAD51AP1) promoter was assessed via chromatin immunoprecipitation sequencing (ChIP-seq) and chromatin immunoprecipitation polymerase chain reaction (ChIP-PCR) analyses. In vitro and in vivo experiments were carried out to investigate the role of the RAD51AP1 gene in TMZ resistance in EGFRvIII-bearing glioblastoma.


RESULTS
The CRISPR/Cas9 library identified RAD51AP1, a synthetic lethal gene for EGFRvIII-bearing cells exposed to TMZ. ChIP-seq and ChIP-PCR analyses revealed that acetylated histone H3 lysine 27 (H3K27ac) and SRY-box transcription factor 9 (SOX9) together induced RAD51AP1 transcription in EGFRvIII cells. High expression levels of RAD51AP1, promoted formation of the RAD51-UAF1 complex to activate homologous recombination and inhibit TMZ-induced DNA damage.


CONCLUSION
The results of this study suggest that aberrant RAD51AP1 expression is a crucial mechanism by which EGFRvIII-mutant glioblastoma resists TMZ chemotherapy, laying the groundwork for future personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b5d96766fa4eddfdcc647bdcfb29efad8ef9899" target='_blank'>
              Epigenetically elevated RAD51AP1 regulates the RAD51-UAF1 complex contributing to temozolomide resistance in EGFRvIII glioblastoma.
              </a>
            </td>
          <td>
            Yuanhao Chang, Jingchen Yang, Zhuo Li, Xiaoxue Zhu, Tao Jiang, Qixue Wang, Mingchen Yu, Bo Han, Xing Liu
          </td>
          <td>2025-04-10</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background To maintain homeostasis in the mature human body, certain differentiated cells adopted high plasticity to refine their cellular functions. However, mechanisms that supported cellular plasticity still remained elusive. Here, through comprehensive transcriptomic and epigenetic studies of highly plastic vascular smooth muscle cells (SMCs), we aimed to decipher the chromatin basis that could mediate cellular plasticity. Results In vascular smooth muscle cells, actively transcribed and highly adjustable genes tended to be associated with a continuously accessible region downstream of transcription start site (CAR-downTSS). This CAR-downTSS was located beyond the classic RNA polymerase II paused region, accessible at mono-nucleosome level and incorporated with histone variant H2A.Z. Depletion of H2A.Z reduced active histone modifications within CAR-downTSS, impaired RNA polymerase II transpassing when cells were stimulated, and consequently inhibited the ability of CAR-downTSS-associated genes to adjust their expression. Further in vitro and in vivo studies verified that this CAR-downTSS could be dynamically re-deployed onto different genes in vascular SMCs, whereas it was deployed in smaller quantities and remained quantitatively stable on genome within the quiescent cardiomyocytes. Conclusions Vascular SMCs dynamically deployed H2A.Z-positive nucleosomes extending continuously downstream transcription start sites on different genes to support their transcriptional adjustability, which served as an important mechanism mediating cellular plasticity. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11679-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4fec2cc361ec5fe548feef27516eeb74be0950f" target='_blank'>
              Dynamic deployment of H2A.Z positive nucleosome mediated transcriptomic plasticity within vascular smooth muscle cell
              </a>
            </td>
          <td>
            Chao Yu, Jiaxin Huang, Yan Wang, Jia Song, Wei-lei Shen
          </td>
          <td>2025-05-19</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Metabolites are essential substrates for epigenetic modifications. Although nuclear acetyl-CoA constitutes a small fraction of the whole cell pool, it regulates cell fate by locally providing histone acetylation substrate. Here, we combined phenotypic chemical screen and genome-wide CRISPR screen to demonstrate a nucleus-specific acetyl-CoA regulatory mechanism that can be modulated to achieve therapeutic cancer cell reprogramming. While previously thought that nucleus-localized pyruvate dehydrogenase complex (nPDC) is constitutively active, we found that nPDC is constitutively inhibited by the nuclear protein ELMSAN1 through direct interaction. Pharmacologic inhibition of the ELMSAN1-nPDC interaction derepressed nPDC activity, enhancing nuclear acetyl-CoA generation and reprogramming cancer cells to a postmitotic state with diminished cell-of-origin signatures. Reprogramming was synergistically enhanced by histone deacetylase 1/2 inhibition, resulting in inhibited tumor growth, durably suppressed tumor-initiating ability, and improved survival in multiple cancer types in vivo, including therapy-resistant sarcoma patient-derived xenografts and carcinoma cell line xenografts. Our findings highlight the potential of targeting ELMSAN1-nPDC as epigenetic cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fb01c13390c9e038310af5ef76ba07e38261aeb" target='_blank'>
              Interrupting ELMSAN1 repression of nuclear acetyl-CoA production therapeutically reprograms cancer cells
              </a>
            </td>
          <td>
            Ting Zhao, Lingli He, L. Wong, Shenglin Mei, Jun Xia, Yanxin Xu, Jonathan G. Van Vranken, Michael Mazzola, Lei Chen, Catherine Rhee, Tiancheng Fang, Tsuyoshi Fukushima, L. Sayles, Matthew Diaz, J. A. B. Gibbons, Raul Mostoslavsky, Steven P. Gygi, Zhixun Dou, David B. Sykes, R. I. Sadreyev, E. A. Sweet-Cordero, David T. Scadden
          </td>
          <td>2025-04-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 p53 and Rb co-mutations are common in several human cancers and are associated with the onset of aggressive tumors with complex karyotypes. Canonically, these tumor suppressors respond to DNA damage and cellular stress by inducing cell cycle arrest, activating pro-apoptotic and anti-proliferative signalling, and promoting cellular senescence. While these pathways are sufficient to partially explain the tumor suppressive abilities of p53 and Rb, recent studies in genetically engineered mouse models suggest additional, non-canonical means of tumor suppression, including modulation of cellular differentiation and DNA topology, and interestingly, influencing cellular response to chromosomal instability. Chromosomal instability (CIN) is a hallmark of aggressive, therapy-resistant cancers and is associated with mitotic dysfunction, faulty DNA damage responses, immune evasion, and cellular stress response pathways. While it is known that p53 and Rb maintain the integrity of the genomic landscape, the mechanisms by which they prevent CIN remain understudied. Here, we use mouse embryonic fibroblasts (MEFs) to show that p53 + Rb double knock-out (DKO) causes a dramatic increase in ploidy at very early passages, which stabilizes at tetraploid levels. These cells also harbour high levels of CIN, indicated by the presence of micronuclei, DNA double-stranded breaks, and lagging chromosomes. Additionally, we observed CIN in p5325, 26 transactivation domain (TAD) mutant MEFs that are unable to activate most canonical p53 targets yet retain the ability to suppress tumorigenesis in vivo, raising the possibility that the maintenance of chromosomal stability is dispensable to the tumor suppressive functions of p53. Using a combination of bulk and scRNA sequencing of aneuploid and diploid DKO MEFs, in future experiments we will identify p53- and Rb-dependent changes in gene expression associated with the onset and progression of CIN. Overall, the results of this study are expected to provide critical insights into the molecular mechanisms by which tumor suppressors p53 and Rb work together to repress CIN.


 Risha Banerjee, David G. Kirsch. Mechanisms promoting chromosomal instability following the loss of p53 and Rb [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2837.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8434065523456bfd26cd26f42f6a8a7a41f381b1" target='_blank'>
              Abstract 2837: Mechanisms promoting chromosomal instability following the loss of p53 and Rb
              </a>
            </td>
          <td>
            Risha Banerjee, D. Kirsch
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f19d291488e785e3492d2b652ca0281a0fb7489d" target='_blank'>
              eLaRodON: identification of large genomic rearrangements in Oxford Nanopore sequencing data
              </a>
            </td>
          <td>
            Regina Mikheeva, M. Koryukov, Pavel Leonov, Victoriya Mukhametdinova, Victoriya Borobova, Petr Laktionov, Alexander Tarkhov, M. Filipenko, Andrey Kechin
          </td>
          <td>2025-05-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Purifying selection that limits the transmission of harmful mitochondrial DNA (mtDNA) mutations has been observed in both human and animal models. Yet the precise mechanism underlying this process remains undefined. Here, we present a highly specific and efficient in situ imaging method capable of visualizing mtDNA variants that differ by only a few nucleotides at single-molecule resolution in Drosophila ovaries. Using this method, we revealed that selection primarily occurs within a narrow developmental window during germline cysts differentiation. At this stage, the proportion of the deleterious mtDNA variant decreases without a reduction in its absolute copy number. Instead, the healthier mtDNA variant replicates more frequently, thereby outcompeting the co-existing deleterious variant. These findings provide direct evidence that mtDNA selection is driven by replication competition rather than active elimination processes, shedding light on a fundamental yet previously unresolved mechanism governing mitochondrial genome transmission.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a50553c510c0eaecac058a100fd3edea07806d1" target='_blank'>
              Replication competition drives the selective mtDNA inheritance in Drosophila ovary
              </a>
            </td>
          <td>
            Cheng Zhang, Zhe Chen, Hansong Ma, Hong Xu
          </td>
          <td>2025-05-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Triggering cancer cell death by inducing DNA damage is the primary aim of radiation therapy; however, normal cells are also damaged. Here, we showed that delivery of only four synthetic guide RNAs (sgRNAs) with Cas9 endonuclease efficiently induced simultaneous DNA double-strand breaks, resulting in efficient cell death in a cell type-specific manner. Off-target effects of Cas9 endonuclease were prevented by using Cas9-nickase to induce DNA single-strand breaks and blocking their repair with poly ADP-ribose polymerase (PARP) inhibitors. When recombinant Cas9-nickase protein and multiple sgRNAs were delivered with PARP inhibitors into cultured cells, in vivo xenografts, and patient-derived cancer organoids via lipid nanoparticles, cancer cells were unable to tolerate the induced DNA damage, even in the presence of a functional BRCA2 gene. This approach has the potential to expand the use of PARP inhibitors with verified safety and thus is a potentially powerful tool for personalized genome-based anti-cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fde0090503581f24fa582fe8e81b3d566923d8b" target='_blank'>
              Combining Multiplexed CRISPR/Cas9-Nickase and PARP Inhibitors Efficiently and Precisely Targets Cancer Cells.
              </a>
            </td>
          <td>
            Soyoung Lee, Kyunghwan Kim, Hye-Jin Jeong, Subin Choi, Himchan Cheng, Dayoung Kim, Soomin Heo, Jinhee Mun, Minjong Kim, Eunjin Lee, Yoon Ji Choi, Seon-Gyeong Lee, Eunjin Lee, Yewon Jang, Kayeong Lim, Heon Seok Kim, Euihwan Jeong, Seung-Jae Myung, Deok-Beom Jung, Chang Sik Yu, In Ho Song, M. Corces, Joo H. Kang, Kyungjae Myung, Taejoon Kwon, Tae-Eun Park, Jinmyoung Joo, Seung Woo Cho
          </td>
          <td>2025-05-06</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Endogenous Retroviruses (ERVs) are remnants of ancient retroviral infections that have permanently integrated into host genomes, including humans. Human Endogenous Retroviruses (HERVs) are ‘fossil’ viral sequences, having lost their ability to generate infectious viruses due to mutations and deletions accumulated over millions of years. Although most HERVs are deemed non-functional, they constitute up to 8% of the human genome. Some HERVs have roles in gene expression regulation, and others are transcribed under specific conditions, such as in cancer. HERV-K is the most recently integrated class within our genome, with some members retaining relatively intact open reading frames (ORFs), suggesting their potential to encode proteins. In cancer, reactivation of HERV expression may impact gene expression, genomic stability, and cell signaling pathways.In this study, we investigated the expression of HERVs in breast cancer (BC) and non-tumor breast tissue using RNA sequencing (RNA-Seq) data from The Cancer Genome Atlas (TCGA). Specifically, we analyzed the expression profiles of HERVs across BC subgroups and their association with protein-coding gene expression profiles. We identified differentially expressed HERVs in BC subgroups (HR-/HER2-, HR+/HER2+, HR-/HER2+, HR+/HER2-) in comparison to non-tumor samples. Our analysis revealed 98 differentially expressed HERVs in HR-/HER2-, 51 in HR+/HER2+, 81 in HR-/HER2+, and 48 in HR+/HER2-. Distinct expression profiles across all subtypes were evident. Furthermore, we identified 45 differentially expressed protein-coding genes with transcription start sites within 5 kb of differentially expressed HERVs. This proximity suggests a potential regulatory influence of HERVs on these genes, indicating the need for further research to clarify the underlying mechanisms. In conclusion, the unique HERV expression profiles across BC subtypes suggest specific molecular characteristics and gene regulation patterns. These findings may aid in identifying new therapeutic targets and treatment strategies for breast cancer, emphasizing HERVs' importance in tumor biology.


 Nathália Da Roz D'Alessandre, Gabriela Der Agopian Guardia, Helena Beatriz da Conceição, Rafael Loch Batista, Pedro Alexandre Favoretto Galante. Characterizing HERV expression patterns across breast cancer subtypes: Implications for tumor biology and therapeutic targets [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2387.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6167c18e215886150966799c6b910032fa501470" target='_blank'>
              Abstract 2387: Characterizing HERV expression patterns across breast cancer subtypes: Implications for tumor biology and therapeutic targets
              </a>
            </td>
          <td>
            Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Helena Beatriz da Conceição, Rafael Loch Batista, P. A. Favoretto Galante
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Origin licensing is essential for DNA replication, confined to the G1 phase of the cell cycle, where human cells must license thousands of origins to ensure complete genome duplication. Insufficient licensing, or "underlicensing", leads to genome instability and increased sensitivity to chemotherapeutic agents. Altered G1 duration and dysregulated origin licensing are characteristic features of many cancers, potentially contributing to their aberrant cell proliferation. However, how origin licensing rates are controlled in cells with varying G1 lengths is poorly understood. Our research reveals that while cyclin E1 overproduction shortens G1 and causes severe underlicensing, c-MYC overproduction also shortens G1 but uniquely accelerates licensing, preventing underlicensing. *We identified a separation of function c-MYC mutant that fails to induce rapid licensing despite upregulating mRNA levels of licensing factors akin to wild-type c-MYC, indicating the involvement of non-canonical pathways beyond traditional transcriptional mechanisms*. c-MYC influence on global chromatin structure through hyperacetylation is pivotal. Our data show that c-MYC’s role in licensing acceleration depends on the recruitment of TRRAP—a transcription cofactor involved in chromatin modification—and the histone acetyltransferase GCN5. Notably, euchromatin is preferred for origin licensing in early G1. We hypothesize that heterochromatin may be rapidly licensed in c-MYC overproducing cells to compensate for potential underlicensing. We are currently profiling early versus late G1 cells overproducing c-MYC to determine the genomic locations of the licensed origins. Unraveling c-MYC downstream pathways regulating origin licensing could revolutionize therapeutic strategies by precisely controlling proliferation in MYC-driven cancers, increasing their sensitivity to DNA-damaging treatments, and paving the way for targeting non-canonical pathways in cancer therapy.


 Dalia Fleifel, Eiman Elkoumy, Arthi Adavikolanu, Sidra Qayyum, Jeanette Cook. Unraveling c-MYC role in DNA replication origin licensing through non-canonical pathways [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5408.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc37e27440b9c33809194ed3bd279c915fb98a61" target='_blank'>
              Abstract 5408: Unraveling c-MYC role in DNA replication origin licensing through non-canonical pathways
              </a>
            </td>
          <td>
            Dalia Fleifel, Eiman Elkoumy, Arthi Adavikolanu, Sidra Qayyum, J. Cook
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d46f5896df3074e9079fd05c2ab487e0d0fe604" target='_blank'>
              Context-Dependent Regulation of Microhomology-Mediated End Joining in Normal Tissues: Insights into Tissue-Specific Activation of DNA Repair Pathways
              </a>
            </td>
          <td>
            Diksha Rathore
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is one of the most common malignancies worldwide, and its complex pathogenesis and significant tumor cell heterogeneity remain major challenges. With the rapid development of single-cell sequencing technology, we can now delve deeper into the cellular composition and dynamic changes within the tumor microenvironment, revealing cellular interactions and their potential roles in tumorigenesis. Method In this study, we systematically analyzed comprehensive single-cell RNA sequencing data from 25 colorectal cancer and 10 adjacent normal tissue samples. We explored the characteristics and biological significance of tumor cell subpopulations, performed quality control, dimensionality reduction, and cell type identification, and further investigated epithelial cell copy number variations, cell communication, and pseudotime analysis. Subsequently, Boruta feature selection algorithm was combined to identify prognosis related genes. The expression patterns, clinical significance and biological effects of PRSS22 were validated in vitro. Results Our analysis found an epithelial cell subcluster with high expression of PRSS22 exhibited high proliferation and migration abilities, and it was also associated with the dysregulated immune microenvironment. After further experimental verification, we proved the high expression patterns and clinical significance of PRSS22. Downregulation of PRSS22 in CRC cells resulted in a reduction of proliferation, migration and invasion. Conclusion Our study has identified a cell subcluster that is closely linked to progression, immune dysregulation and prognosis in CRC, and we have also identified PRSS22 as its hub gene that has great potential to become a new immunotherapeutic targets target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0d396c713a83490019a7c5d54cf76e345198f9" target='_blank'>
              Single-cell transcriptomics in colorectal cancer uncover the potential of metastasis and immune dysregulation of a cell cluster overexpressed PRSS22
              </a>
            </td>
          <td>
            Chengyuan Xu, Ziheng Zhou, Dongfei Zhu, Qingyun Zhang, Shoubin Zhong, Zhenhua Li
          </td>
          <td>2025-05-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="

 Homologous recombination deficiency (HRD) is a predictive biomarker for various anti-cancer drugs such as poly(ADP-ribose) polymerase inhibitors (PARPi) and platinum-based chemotherapeutic agents. The HRD score has been estimated using genomic and molecular assays, even though due to cancer heterogeneity, HRD could vary in the tumor microenvironment. Here, we developed a model to predict HRD score by estimating copy numbers and calculating loss of heterogeneity (LOH), large-scale state transitions (LST), and telomeric-allelic imbalance (TAI) scores resulting in exhibiting both spatial coherence and heterogeneity, making them suitable for clinical biomarkers addressing tumor heterogeneity. To estimate these scores with counts from ST data, STARCH algorithm was utilized and the copy number for each gene across spots within cancer clones was obtained. HRD scores are determined by adding together LOH, TAI, and LST. We applied these analyses to Visium and VisiumHD data from breast cancer samples and visualized the spatial distribution of the HRD score within the cancer cell map in the tumor microenvironment.



 This method was applied to both Visium and Visium HD data to show the heterogeneity in the HRD score. The HRD scores effectively reflected spatial patterns and demonstrated high heterogeneity, highlighting their potential as clinical biomarkers. Furthermore, when applied to direct Visium data (Visium FF), the HRD scores predicted using only gene expression data yielded similar conclusions to those calculated considering both gene expression and allele counts. Notably, the concordance between the two approaches was higher when normal spots were not separately input. Additionally, running the analyses separately or collectively produced comparable conclusions.



 By developing a model to predict HRD scores from ST data, it becomes possible to analyze the relationship between pathophysiological mechanisms and HRD across various cancer types. This approach not only enhances our understanding of tumor biology but also enables the prediction of HRD score maps while accounting for spatial heterogeneity in predictive biomarkers. Such advancements could improve personalized treatment strategies and clinical decision-making for patients.



 Jeongbin Park, Yuchang Seong, Dongjoo Lee, Hongyoon Choi. Estimation of homologous recombination deficiency map from spatial transcriptomics [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5074.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f606536b132a87445aa5fa5b6a3753af5559e5e5" target='_blank'>
              Abstract 5074: Estimation of homologous recombination deficiency map from spatial transcriptomics
              </a>
            </td>
          <td>
            Jeongbin Park, Yuchang Seong, Dongjoo Lee, Hongyoon Choi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Medulloblastoma (MB) is the most common malignant pediatric brain tumor and is derived from neuroepithelial stem cells (NESC). Amplification of each member of the MYC family of oncogenes (MYCN, CMYC, MYCL) has been found in MB tumors and amplified-MYCN or CMYC correlates with poor prognosis. Since transcription factors such as MYC are difficult to drug with small molecules, more druggable targets need to be identified. While genetic screens can be performed on human MB cell lines, there is a lack of a proper isogenic control to distinguish targets that also negatively affect healthy non-tumor cells. Here, we show the utility of a human induced pluripotent stem cell (iPSC) based model of MB to find therapeutic vulnerabilities.



 Human iPSC derived from a healthy adult were differentiated towards NESC and transduced with empty vector (EV), MYCN, CMYC or MYCL. We analyzed how each MYC protein influences the molecular profile (RNAseq and methylation), proliferation in vitro (EdU assay), and transformation in vivo (orthotopic implantation into the hindbrain of immunocompromised mice) compared to EV in NESC. Whole genome CRISPR interference (CRISPRi) screen was performed on MYCN and CMYC NESC-derived tumors and EV NESC. Viability in response to 72 hours of inhibitor treatment was analyzed qualitatively by transmitted light images of cells and quantitatively by LIVE/DEAD stain followed by flow cytometry.



 MYCN and CMYC induced a higher percentage of NESC in S-phase (46%, 40%) compared to MYCL (35%) and EV (31%). Molecularly, MYCN and CMYC NESC had similar transcriptomic and methylation profiles compared to MYCL and EV. Gene ontology revealed the most enriched pathway in NESC with either MYCN or CMYC, but not MYCL, was mitochondria gene expression. In vivo, MYCN and CMYC were both 100% penetrant by day 98 while MYCL was only 30% penetrant and EV did not develop tumors. CRISPRi screen demonstrated MYCN and CMYC NESC-derived tumors, but not EV NESC, were dependent on subunits of Complex I, III, IV and V of the electron transport chain (ETC) in the mitochondria. In support of this, treatment with two different inhibitors of Complex I (Metformin, Mito-HNK) showed stronger toxicity against both MYC NESC-derived tumors than EV NESC. Strikingly, treatment with Mito-HNK resulted in higher cell death in CMYC-expressing MB cell lines D283 (30%), D425 (83%) and MB004 (82%) compared to healthy NESC (<10%).



 MYCN and CMYC induce molecular changes distinct from MYCL in NESC that lead to upregulation of mitochondria gene expression, faster growth and transformation. Both MYCN and CMYC driven MB tumors are more dependent on the ETC for viability than non-tumor NESC suggesting MYC tumors could be specifically targeted by disrupting their metabolism. Thus, we show the combination of human iPSC-based model of MB and CRISPR screening can uncover vulnerabilities in MB cell lines.



 Kenya Pleitez Valle, Miriam Velez, Yang Zhao, Bo Cheng, Sophia Estrada, Linwei Li, Tom O'Loughlin, Balaraman Kalyanaraman, Ming You, Luke A. Gilbert, David M. Hockenbery, David B. Shackelford, Peter J. Mullen, Miller Huang. CRISPR screen of human stem cell derived MYC-driven medulloblastoma tumors reveal the mitochondria as a vulnerability [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3941.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90996b84410758ba4f6bb325e48c6874e9494ef2" target='_blank'>
              Abstract 3941: CRISPR screen of human stem cell derived MYC-driven medulloblastoma tumors reveal the mitochondria as a vulnerability
              </a>
            </td>
          <td>
            Kenya Pleitez Valle, Miriam Velez, Yang Zhao, Bo Cheng, Sophia Estrada, Linwei Li, Tom O'Loughlin, Balaraman Kalyanaraman, Ming You, Luke A. Gilbert, David M. Hockenbery, D. Shackelford, Peter J. Mullen, Miller Huang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Genomic instability has been increasingly recognized over the past decade as a fundamental driver of cancer initiation and progression, largely owing to its association with specific genes and cellular mechanisms that offer therapeutic potential. However, a comprehensive molecular framework that captures the interconnected processes underlying this phenomenon remains elusive. In this study, we focused on polo-like kinase 1 (PLK1), a key cell cycle regulator frequently overexpressed in diverse human tumors, to reconstruct a regulatory network that consolidates pre-existing biological knowledge exclusively related to pathways involved in genome stability maintenance and cancer. The resulting model integrates nine biological processes, 1030 reactions, and 716 molecular species to form a literature-supported network in which PLK1 serves as a central regulatory node. However, rather than depicting an isolated PLK1-centric system, this network reflects a broader and more complex architecture of interrelated genomic instability mechanisms. As expected, the simulations reproduced known behaviors associated with PLK1 dysregulation, reinforcing the well-established role of the kinase in genome destabilization. Importantly, this model also enables the exploration of additional, less-characterized dynamics, including the potential involvement of genes such as kif2c, incenp, and other regulators of chromosomal segregation and DNA repair, which appear to contribute to instability events downstream of PLK1. While these findings are grounded in mechanistic simulations and require further experimental validation, gene expression and survival analyses across tumor types support their clinical relevance by linking them to poor prognosis in specific cancers. Overall, the model provides a systemic and adaptable foundation for studying PLK1-related genomic instability, enabling both the reinforcement of known mechanisms and discovery of candidate genes and circuits that may drive tumorigenesis through compromised genome integrity across distinct cancer contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c66a6d480aea1f4304a1ce04eec8ec37b7bd0e2a" target='_blank'>
              The plk1 Gene Regulatory Network Modeling Identifies Three Circuits for plk1-mediated Genomic Instability Leading to Neoplastic Transformation
              </a>
            </td>
          <td>
            Jeison F. Suescum-Holguín, Diana Carolina Clavijo-Buriticá, Edward Fabian Carrillo-Borda, M. Quimbaya
          </td>
          <td>2025-05-01</td>
          <td>Life</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34ad39aac8ea35ca3a29505be2a3f10fc20c5850" target='_blank'>
              TP53 and DNA-PK as potential biomarkers for enhanced efficacy of Olaparib in colorectal cancer.
              </a>
            </td>
          <td>
            Do Yeon Kim, Hyeseon Yun, Ji-Eun You, Y. Park, Yeaseong Ryu, Dong-In Koh, Jae-Sik Shin, Dong-Hoon Jin
          </td>
          <td>2025-05-16</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Bulk sequencing has proven inadequate in capturing the full complexity of cancer heterogeneity, shifting focus toward cellular resolution. Single-cell DNA sequencing (scDNA-seq) technologies offer an effective approach for analyzing subclonal molecular architecture of tumor with greater granularity, facilitating the identification of somatic genetic mutations (single nucleotide variants, SNVs, and short indels) and copy number variants (CNVs) at single-cell resolution. This capability holds multiple promises, such as the characterization of timing and hierarchy of molecular events driving tumor initiation and progression. However, traditional scDNA-seq methods are hindered by significant artifacts during genome amplification, causing issues like allelic dropout and imbalance that compromise variant detection accuracy. The introduction of the Primary Template-directed Amplification (PTA) addressed these limitations by providing higher genome coverage and uniformity; however, further refinement in filtering amplification-induced artifacts remains necessary for accurate variant calling.
 Here, we present CAPTATe (Characterize Artifacts in PTA Technology), a novel tool developed for improved artifact filtering and enhanced detection of SNVs, indels, and CNVs in scDNA-seq data. CAPTATe employs allele-specific CNV calling by addressing fluctuations in the read-depth ratio (RDR) through a multi-channel joint segmentation approach at the subclonal level. The tool leveraged an extension of the variational framework recently developed for single-cell RNA-seq (de Falco et al. Nature Communications 2023) and integrates both B-allele frequency (BAF) and RDR for accurate CNV detection. Additionally, SNV and indel artifact filtering are performed through a machine learning-based approach. This model accounts for allelic imbalance by evaluating the ratio between observed variant allele frequency (VAF) and expected BAF. Genetic tumor subclones can be accurately identified by clustering cells based on their genomic alteration profiles.
 The increased precision and accuracy in single-cell genomic variant identification using CAPTATe overcome the main limitations of PTA scDNA-seq profiling, enabling precise characterization of genetic tumor heterogeneity at individual cell resolution.


 Luigi Ferraro, Fulvio D'Angelo, Anna Lasorella, Antonio Iavarone, Michele Ceccarelli. CAPTATe: a novel tool for enhanced artifact filtering and precise variant detection in single-cell DNA sequencing data [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5036.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b39cc8d482393c41340901b2116da18b7ccbd397" target='_blank'>
              Abstract 5036: CAPTATe: a novel tool for enhanced artifact filtering and precise variant detection in single-cell DNA sequencing data
              </a>
            </td>
          <td>
            Luigi Ferraro, F. D’Angelo, A. Lasorella, A. Iavarone, Michele Ceccarelli
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="
 DHX9, a multi-functional RNA helicase, contributes to genome stability by unwinding nucleic acid secondary structures, including DNA/RNA hybrids (R-loops), circular RNA and G-quadruplexes. Multiple cancer types exhibit increased expression of DHX9, and selective dependance to DHX9 has been observed in molecular defined tumors that exhibit genomic instability. These tumors often have elevated replicative stress due to rapid unregulated proliferation and defective DNA damage repair (DDRD) pathways; DHX9 inhibition in these tumors further increases replicative stress and DNA damage to catastrophic levels. We previously demonstrated that DHX9 genetic loss or inhibition by the DHX9 tool compound ATX968 was efficacious in tumor types that exhibit DDRD and/or genomic instability; these include tumors with defective mismatch repair and/or high microsatellite instability (dMMR/MSI-H) or alterations in the DDR genes BRCA1 and/or BRCA2 (BRCA).
 Here we describe ATX-559, a potent, selective, orally bioavailable, small-molecule inhibitor of DHX9 helicase activity that is currently in clinical development. In preclinical studies, ATX-559 treatment leads to increased accumulation of R-loops and G-quadruplexes resulting in increased replicative stress and stalling, subsequent DNA damage, and cell death selectively in cancer models with high genomic instability. In a panel of 36 colorectal (CRC) and endometrial cancer (EC) cell lines, approximately 55% were sensitive (IC50 ≤ 1 μM) to ATX-559 treatment, with enriched antiproliferation activity (80%) in dMMR/MSI-H cells. ATX-559 sensitivity was also enriched in breast cancer cell lines, including 76% of BRCA altered triple negative breast cancer (TNBC) models.
 ATX-559 is well tolerated in vivo across multiple cell line-derived xenograft (CDX) and patient-derived xenograft (PDX) models, when dosed orally BID for periods out to 28 days or more. Robust dose-dependent tumor growth inhibition (TGI) and regression are selectively observed in CRC and EC dMMR/MSI-H and BRCA altered TNBC CDX models. TGI is correlative with plasma and tumor drug exposure, as well as with induction of the target engagement PD biomarker, circBRIP1, in both tumors and mouse PBMCs. ATX-559 was also profiled in a PDX mouse clinical trial design in vivo study of 46 models including TNBC, CRC, EC, and gastric cancers. Oral treatment of 45 mg/kg BID demonstrated enriched sensitivity in BRCA altered TNBC (75%) and dMMR/MSI-H (68%) PDX models.
 Together, these findings represent preclinical pharmacological validation of DHX9 as a selective therapeutic target in cancers that exhibit genomic instability and replication stress. ATX-559 is currently in a first-in-human, Phase 1, open-label, dose-escalation and expansion study in patients with locally advanced or metastatic solid tumors and molecularly defined cancers (NCT06625515).


 Jennifer Castro, Matthew H. Daniels, Sunaina Nayak, David Brennan, Cindy Collins, Sophie A. Shen, Monique Laidlaw, Jie Wu, Anugraha Raman, Deepali Gotur, Kevin Knockenhauer, Shihua Yao, Simina Grigoriu, Gordon J. Lockbaum, Kate Newberry, Stephen J. Blakemore, P. Ann Boriack-Sjodin, Kenneth W. Duncan, Stuart Ince, Jason A. Sager, Robert A. Copeland, Serena J. Silver. ATX-559, a first in class DHX9 inhibitor, and targeted therapeutic for molecularly defined tumors with genomic instability and replicative stress [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1758.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0c4901cd0ac19548c13b53daf9ce1ecb7db4d98" target='_blank'>
              Abstract 1758: ATX-559, a first in class DHX9 inhibitor, and targeted therapeutic for molecularly defined tumors with genomic instability and replicative stress
              </a>
            </td>
          <td>
            Jennifer Castro, Matthew H Daniels, S. Nayak, David Brennan, Cindy Collins, Sophie A. Shen, Monique Laidlaw, Jie Wu, Anugraha Raman, D. Gotur, Kevin E Knockenhauer, Shihua Yao, Simina Grigoriu, Gordon J Lockbaum, Kate Newberry, Stephen J Blakemore, P. Boriack-Sjodin, K. Duncan, Stuart Ince, Jason A. Sager, Robert A Copeland, Serena J Silver
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Chromosomal translocations are key events in cancer, driving oncogenesis by disrupting and deregulating critical genes. While specific tumor‐associated translocations are well studied, the frequencies and distributions of most remain unknown. Additionally, the role of chromosomal reshuffling in translocations has received little attention. This study presents data on the chromosomal involvement in 59 251 translocations reported in 58 tumor entities, including both benign and malignant tumors. Unlike studies focusing on tumor‐specific abnormalities identified at the chromosome band level, this study examines translocations at the chromosomal level, offering a novel perspective on their distribution. This broader approach aims to uncover patterns that do not emerge or are disregarded in studies limited to tumor‐specific aberrations. The resulting dataset provides a novel resource for deepening our understanding of the chromosomal origins of translocations in neoplasia. Comparisons of translocation frequency distributions among tumor types, when excluding the characteristic tumor‐associated translocations, revealed that the patterns of chromosomal involvement in translocations are largely unique to each tumor entity. Statistical analyses of 241 pairwise comparisons of translocation spectra within hematologic disorders, solid tumors, and between groups of hematologic malignancies and both benign and malignant solid tumors showed insignificant/very weak associations (R2 ≤ 0.3) in 98% of the comparisons. The findings hence demonstrate that different tumor types are characterized by distinct chromosomal translocation signatures, strongly suggesting that most translocations encountered in tumor cells are not merely random events. Consequently, our study highlights the potential of rare translocations to serve as indicators of disease‐specific processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2c3b03e0bf3342743907df4f51e09cd0d87a1db" target='_blank'>
              Distinct Signatures of Chromosomal Involvement in 59 251 Translocations Across 58 Tumor Types. A Novel Perspective
              </a>
            </td>
          <td>
            Felix Mitelman, Nils Mandahl
          </td>
          <td>2025-05-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Disentangling the molecular drivers of cancer progression requires a precise understanding of the somatic alterations that take place at the DNA level during tumor development. These include not only small changes like SNVs and indels, but also structural variants, changes in repetitive elements, differential methylation, as well as the haplotype context in which these changes occur. Existing short-read sequencing methods using sequencing by synthesis (SBS) chemistry lack the read length to characterize large structural variants or to span longer repetitive regions as well as to phase variants into haplotypes, while simultaneously lacking the sequencing accuracy to distinguish very low frequency somatic variants from background sequencing error or to sequence through shorter repetitive elements like microsatellites. Long-read PacBio HiFi data addresses the first set of challenges by accurately sequencing long fragments up to 25 kb, while highly accurate short-read sequencing by binding (SBB) chemistry from PacBio addresses the latter by improving raw sequencing accuracy by orders of magnitude, including in homopolymer regions which make up most microsatellites in the human genome.
 Here we apply these highly accurate long- and short-read sequencing technologies from PacBio to perform whole-genome sequencing of the newly described HG008 matched tumor-normal pair from the Genome in a Bottle (GIAB) consortium. This reference sample includes an adherent, epithelial-like pancreatic adenocarcinoma (PDAC) cell line as the tumor material, with the matched normal obtained from adjacent duodenal and pancreatic tissue. We perform whole-genome sequencing with both PacBio HiFi (35X normal, 80X tumor) and SBB (50X normal, 100X tumor), followed by alignment and tumor-normal variant calling for all variant types. These accurate long- and short-read sequencing technologies offer a more robust and comprehensive picture of somatic variation in this reference sample and contribute to developing this novel benchmark.


 Alexandra Sockell, Khi Pin Chua, Christopher Kingsley, Dan Nasko, Matthew Boitano, Young Kim, Melanie Wescott, Ian McLaughlin, Primo Baybayan, Jennifer McDaniel, Andrew Shaver, Justin M. Zook, Aaron Wenger. Comprehensive, multi-omic detection of somatic variants from the GIAB HG008 matched tumor-normal pair using highly accurate long- and short-read whole-genome sequencing [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6624.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dedc758c45f400b459a2e2b62f3f89ff408d534" target='_blank'>
              Abstract 6624: Comprehensive, multi-omic detection of somatic variants from the GIAB HG008 matched tumor-normal pair using highly accurate long- and short-read whole-genome sequencing
              </a>
            </td>
          <td>
            Alexandra Sockell, Khi Pin Chua, Christopher Kingsley, Dan Nasko, Matthew Boitano, Young Kim, Melanie Wescott, Ian J. McLaughlin, P. Baybayan, Jennifer H. McDaniel, Andrew Shaver, J. Zook, Aaron Wenger
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Structural variants (SVs) are increasingly recognized as key drivers of bacterial evolution, yet their role has not been explored thoroughly. This is due to limitations in traditional short-read sequencing and linear reference-based analyses, which can miss complex structural changes. Tuberculosis (TB), a disease caused by Mycobacterium tuberculosis (Mtb), remains a major global health concern. In this study, we harness long-read sequencing technologies and genome graph tools to construct a Mtb pangenome reference graph (PRG) from 859 high-quality, diverse, long-read assemblies. To enable accurate genotyping of SVs leveraging the PRG, we developed miniwalk, a tool that outperforms a traditional linear genome-based approach in precision for SV detection. We characterize patterns of structural variation genome-wide, revealing a virulence-associated ESX-5 deletion to be recurrent across the phylogeny, and fixed in a sub-lineage of L4. Systematic screens for additional genes that are recurrently affected by SVs implicated those related to metal homeostasis, including a copper exporter fixed in the widely distributed L1.2.1 sub-lineage. Lastly, we genotyped 41,134 isolates and found SVs putatively associated with resistance to various first and second-line drugs. These findings underscore the broader role of SVs in shaping Mtb diversity, highlighting their importance in both understanding evolution and designing strategies to combat drug-resistant TB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3817aa494e2c51a1c1335491916677d96f39a6c0" target='_blank'>
              Genome graphs reveal the importance of structural variation in Mycobacterium tuberculosis evolution and drug resistance
              </a>
            </td>
          <td>
            Aleix Canalda-Baltrons, Matthew Silcocks, Michael B. Hall, Derrick Theys, Xuling Chang, Linda Viberg, N. Sherry, L. Coin, S. J. Dunstan
          </td>
          <td>2025-05-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f86b4088fda0a17a1413678d49be792807c01757" target='_blank'>
              Human TLS DNA polymerase: saviors or threats under replication stress?
              </a>
            </td>
          <td>
            Y. S. Rajpurohit, Mitu Lal, D. K. Sharma, Ishu Soni
          </td>
          <td>2025-05-23</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5025875828f543583bc1d12169d95499fa95592" target='_blank'>
              Robust statistical assessment of Oncogenotype to Organotropism translation in xenografted zebrafish
              </a>
            </td>
          <td>
            D. Saucier, Xuexia Jiang, Divya Rajendran, Roshan Ravishankar, Erin Butler, A. Marchetto, R. Kurmasheva, T. G. Grünewald, James F. Amatruda, G. Danuser
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="This study analyzed the genetics of classical Hodgkin lymphoma (cHL) by using circulating tumor DNA (ctDNA). Two genetic subtypes were identified, differing in genetic instability mechanisms: one subtype (64% of cases) showed a higher mutation load and a higher fraction of mutations associated with activation-induced cytidine deaminase and microsatellite instability signatures, while the other subtype (36% of cases) exhibited chromosomal instability with more somatic copy number alterations. Whole-genome duplication was more common in cHL compared to other B-cell tumors and emerged as a prognostic biomarker for patients undergoing ABVD-based therapy. Non-coding regulatory mutations, similar to those in diffuse large B-cell lymphoma, were highly prevalent in 86% of cHL. A recurrent somatic expression quantitative trait locus (seQTL) involving the BCL6 gene was found in 30% of cases. The seQTL of BCL6 aligned with accessible chromatin and increased H3K27 acetylation in cHL, disrupted PRDM1 binding, and co-occurred with BCL6 expression in cHL cells. Weak to strong expression of BCL6 was observed in 68% of cases and BCL6 expression associated with gene repression similarly in cHL and germinal center B cells. After BCL6 degradation, the core set of genes directly bound and regulated by BCL6 was derepressed in cHL and proliferation was impaired. The number and clonality of neoantigens was associated with tumor microenvironment type and response to checkpoint blockade. Finally, ctDNA analysis was suggested as a tool to distinguish ambiguous PET/CT-positive lesions post-treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc33ff26bbe2b930345064a0a4545ff56f194772" target='_blank'>
              A Comprehensive Genetic Study of Classical Hodgkin Lymphoma Using Circulating Tumor DNA.
              </a>
            </td>
          <td>
            M. Pirosa, A. Bruscaggin, L. Terzi di Bergamo, M. Salehi, F. Jauk, G. Forestieri, S. Bocchetta, D. Piffaretti, R. Moia, V. Cristaldi, M. di Trani, G. A. Galimberti, K. Pini, V. Spina, Claudia Giordano, A. Condoluci, Ilaria Romano, S. Annunziata, F. Bergesio, R. Boldorini, E. Borsatti, P. Bulian, Eleonora Calabretta, S. Chauvie, Francesco Corrado, S. Crisci, M. Cuzzocrea, R. De Filippi, Bernhard Gerber, M. Kurlapski, L. M. Larocca, Elisabetta Merlo, A. Rinaldi, M. Rodari, G. Romanowicz, G. Sacchetti, A. Stathis, G. Stussi, I. Zangrilli, A. Pinto, L. Mazzucchelli, V. Gattei, J. M. Zaucha, Armando Santoro, S. Hohaus, Franco Cavalli, A. Tzankov, Carmelo Carlo-Stella, G. Gaidano, L. Ceriani, Emanuele Zucca, Davide Rossi
          </td>
          <td>2025-05-13</td>
          <td>Blood</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Advances in cancer biology have been achieved by the identification of oncogenes and tumor suppressor genes through the remarkable progression of next-generation sequencing. New techniques, such as single-cell analysis, help uncover cancer progression and heterogeneity. Reverse genetic screenings, including methods like random mutagenesis via retroviruses, transposons, RNA interference, and CRISPR, are useful for exploring gene functions and their roles in cancer. Especially in random mutagenesis, CRISPR screening and its modifications have recently emerged as powerful tools due to their comprehensiveness and simplicity in inducing genetic mutations. Initially, CRISPR screening focused on analyzing biological phenotypes in a cell population. It has since evolved to incorporate advanced techniques, such as combining single-cell and spatial analyses. These developments enable the investigation of cell-cell and spatial interactions, which more closely mimic In Vivo microenvironments, offering deeper insights into complex biological processes. These approaches allow for the identification of essential genes involved in cancer survival, drug resistance, and tumorigenesis. Together, these technologies are advancing cancer research and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8d93feca0cdb57db69fce7d9acc441dc2942da5" target='_blank'>
              Identification of Target Genes Using Innovative Screening Systems.
              </a>
            </td>
          <td>
            Keisuke Sugita, Morita Kurata
          </td>
          <td>2025-05-05</td>
          <td>Pathology international</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 c-MYC (MYC) is a central regulatory protein that is dysregulated in >50% of all human cancers and is linked to aggressive disease. Developing MYC inhibitors would revolutionize cancer treatment; however, efforts to target MYC directly using small molecular inhibitors have historically failed. A promising approach is to identify and inhibit critical MYC partner proteins to inactivate MYC and trigger cancer cell death. Inhibiting these targets therapeutically can result in synthetic lethality (MYC-SL), which can be exploited in MYC-dysregulated cancers. To identify MYC-SL targets, we performed a genome-wide CRISPR knock-out screen using an isogenic pair of non-transformed and MYC-driven breast cancer cells. In contrast to other screens, this model is both dependent on MYC and recapitulates human disease at pathological and molecular levels in vivo. Finally, these hits were cross-referenced with our MYC protein-interactome data to reveal putative MYC partner proteins that are critical for MYC activity.
 From our screen results, we performed gene set enrichment analysis to identify biological activities that may represent core functional dependencies in MYC dysregulated cancer cells. Using this approach, we identified and validated topoisomerase 1 (TOP1) as an actionable vulnerability that can be targeted with clinically approved inhibitors. Genetic and pharmacological inhibition of TOP1 in multiple orthogonal assays resulted in MYC-driven cell death. Finally, drug response to TOP1 inhibitors correlated with MYC levels and activity across panels of breast cancer cell lines and patient-derived organoids, highlighting TOP1 as a promising target for MYC-driven cancers.
 The recent accessibility of large-scale datasets detailing functional dependencies across hundreds of cancer cell lines offers an unprecedented opportunity to prioritize targets with greater translational relevance, which is a major limitation of the synthetic-lethal approach for target discovery. As a secondary analysis of our hits, we utilized DEPMAP data to classify cancer cell lines as relatively MYC-dependent or MYC-independent, enabling the in silico evaluation of each MYC-SL hit’s differential essentiality. MYC-SLs that exhibited selective essentiality in MYC-dependent cell lines were prioritized for further investigation. Results from this strategy revealed critical MYC cofactors that have been validated by us and others (e.g., CDK9), providing confidence in this approach. Excitingly, previously underexplored targets were identified with promising validation to-date. Together, this work features two successful strategies to prioritize hits from hundreds of synthetic-lethal genome-wide CRISPR screens to identify novel MYC-driven vulnerabilities in cancer.


 Peter Lin, Corey Lourenco, Jennifer Cruickshank, Luis Palomero, Jenna E. van Leeuwen, Amy H. Tong, Katherine Chan, Samah El Ghamrasni, Miquel Pujana, David W. Cescon, Jason Moffat, Linda Z. Penn. An isogenic CRISPR screen identifies novel MYC-driven vulnerabilities [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4039.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bc94f977a238b5f1df2211d9c38889ceb4f0e79" target='_blank'>
              Abstract 4039: An isogenic CRISPR screen identifies novel MYC-driven vulnerabilities
              </a>
            </td>
          <td>
            Peter Lin, Corey Lourenco, Jennifer Cruickshank, Luis Palomero, Jenna E. van Leeuwen, Amy H Y Tong, Katherine Chan, S. E. Ghamrasni, M. Pujana, D. Cescon, Jason Moffat, Linda Z. Penn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54bb9c5a7aaba38fdef52e5c2b089ee886ff2b49" target='_blank'>
              Seasonal brain regeneration and chromosome instability are linked to selection on DNA repair in Sorex araneus
              </a>
            </td>
          <td>
            William R. Thomas, Tanya M. Lama, Diana Moreno Santillán, Marta Farré, Cecilia Baldoni, Linelle Abueg, Jennifer R. Balacco, O. Fedrigo, G. Formenti, Nivesh Jain, J. Mountcastle, Tatiana Tilley, Alan Tracey, David A. Ray, D. Dechmann, Dominik von Elverfeldt, John D Nieland, Angelique P. Corthals, Erich D. Jarvis, Liliana M. Dávalos
          </td>
          <td>2025-04-17</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="Background Esophageal squamous cell carcinoma (ESCC) is a major contributor to cancer-related deaths, driven by its invasive and metastatic nature. Circular RNAs (circRNAs) are increasingly recognized as regulators of cancer progression, primarily through miRNA sponging and interactions with RNA-binding proteins. Their dysregulation has been linked to the development of in various cancers. The present study aimed to investigate the potential involvement of circSLC22A3 in the pathogenesis of ESCC. Methods CircSLC22A3 expression in ESCC tissues and cells was analyzed using transcriptome sequencing and RT-qPCR. Its circular structure was validated through Sanger sequencing, agarose gel electrophoresis, RNase R digestion, and random priming assays. Subcellular localization was determined by nucleoplasmic separation and fluorescence in situ hybridization (FISH). Clinical correlations were assessed via tissue microarrays. Functional roles of circSLC22A3 in ESCC progression were investigated through in vitro and in vivo assays. Downstream miR-19b-3p and target gene TRAK2 were screened by bioinformatics analysis and RT-qPCR, with binding confirmed via luciferase reporter assays. RNA pulldown combined with RNA immunoprecipitation (RIP) identified IGF2BP1 as a circSLC22A3-interacting protein. RNA-seq and RT-qPCR revealed ACSBG1 as a key downstream effector. IGF2BP1-mediated m6A modification of ACSBG1 was mapped by MeRIP-seq and RIP, with mRNA stability assessed via Actinomycin D assay. ACSBG1 expression and biological function in ESCC were confirmed by immunohistochemistry, RT-qPCR, and functional assays. Results Significant downregulation of circSLC22A3 was observed in both ESCC tissues and cell lines. Overexpression of circSLC22A3 significantly reduced ESCC cells’ migration and invasion capabilities. Mechanistic investigation revealed that circSLC22A3 played a pivotal role in the invasion and metastasis of esophageal cancer through distinct pathways. On one hand, circSLC22A3 functioned as a miR-19b-3p sponge to augment trafficking kinesin protein 2 (TRAK2) expression, while, on the other hand, circSLC22A3 formed a protein-RNA complex with IGF2BP1, resulting in the degradation of acyl-CoA synthetase bubblegum family member 1 (ACSBG1) mRNA through the recognition of m6A modification, thereby suppressing invasion and metastasis of ESCC. Conclusions The present study identified circSLC22A3 as a new tumor suppressor that inhibited ESCC progression through both the circSLC22A3/ miR-19b-3p/ TRAK2 and circSLC22A3/ IGF2BP1/ ACSBG1 axes. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14390-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/759ab9ec236648fb5b914b67db4cf74178231463" target='_blank'>
              CircSLC22A3 inhibits the invasion and metastasis of ESCC via the miR-19b-3p/TRAK2 axis and by reducing the stability of m6A-modified ACSBG1 mRNA
              </a>
            </td>
          <td>
            Yingjie Pan, Hang Yang, Jiayi Zhang, Ruolan Zhang, Yun Liu, Jun Bie, Qiaoling Chen, Yan Qiao, Kang Liu, Guiqin Song
          </td>
          <td>2025-05-30</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Genetic instability is a hallmark of cancer, often arising from mutations in DNA damage repair and response (DDR) proteins. It is our goal to map the repair dependencies associated with different DNA-damaging agents (DDA) and DDR inhibitors (DDRi). To this end, we developed a DDR-targeted, CRISPR knockout screening approach to assess relationships among 355 DNA repair genes and 30 DDA and DDRi in LN229 glioma cells.
 Systems-level analysis of over 11, 000 interactions revealed that DDA cluster into distinct classes. In contrast, DDRi including ATR, Pol-θ, DNA-PK, WRN, Wee1, USP1, ATM and PKMYT1 inhibitors exhibited a variety of unique dependency profiles. Pathway analysis revealed clear associations between the knockout of Fanconi anemia pathway genes and sensitivity to bifunctional alkylating agents (e.g. chlorambucil), nucleotide excision repair and monofunctional alkylating agents (e.g. illudin S), homologous recombination and PARP inhibitors (e.g. talazoparib), and non-homologous end-joining and radiomimetics (e.g. bleomycin).
 Furthermore, this platform identified differences in repair dependencies of compounds with subtle variations in chemical structure, highlighted by the imidazotetrazines temozolomide (TMZ) and KL50. In contrast to the methyl adducts deposited by TMZ, DNA modifications deposited by KL50 slowly evolve into DNA interstrand crosslinks (ICLs), circumventing mismatch repair (MMR)-driven resistance observed with TMZ. Indeed, screening revealed that resistance to TMZ, but not KL50, is driven by loss of MMR genes. Furthermore, we observed that BRIP1, a helicase known to play a key role in repair of ICLs, emerged as a unique sensitizing hit for KL50, but not TMZ, providing biological support for the formation of ICLs by KL50.
 Finally, we sought to identify the most clinically actionable chemo-genomic interactions from our screens. Bi-allelic inactivating mutations (BI) of DNA repair genes are often necessary for manifestation of a DDR deficiency phenotype. Thus, we subjected our gene list to a custom bioinformatic pipeline to quantify rates of BI in samples from The Cancer Genome Atlas (TCGA). Excluding genes listed as Level 1 biomarkers on OncoKB, we found that 4.73% of TCGA samples harbored putative BI of at least one of the 138 genes that emerged as a hit (FDR < 0.01) in our screens.
 Overall, our screens validated established chemo-genomic interactions while also identifying genetic dependencies associated with DDA of previously unknown mechanisms of action. Furthermore, we elucidated subtle differences in repair dependencies between structurally similar compounds. Finally, we identified novel genetic biomarkers with frequent BI in human tumor samples that sensitized to specific therapeutic agents upon loss, nominating chemo-genomic interactions with potential clinical relevance.


 Collin D. Heer, Spenser S. Johnson, James L. Elia, Sam Friedman, Xin Pei, Sofia R. Arbelaez, Ranjini Sundaram, Vijay Menon, Joseph Edmonds, Karlie Lucas, Nadeem Riaz, Susan E. Gueble, Ranjit S. Bindra. Systematic mapping of chemo-genomic interactions between DNA damaging agents and DNA damage repair genes with targeted CRISPR knockout screening [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2902.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b90e0ffa740ed3554dce2f43e94e83c85289e48" target='_blank'>
              Abstract 2902: Systematic mapping of chemo-genomic interactions between DNA damaging agents and DNA damage repair genes with targeted CRISPR knockout screening
              </a>
            </td>
          <td>
            Collin D. Heer, Spenser S. Johnson, James L. Elia, Sam Friedman, Xin Pei, Sofia R. Arbelaez, R. Sundaram, Vijay Menon, Joseph Edmonds, Karlie N Lucas, N. Riaz, Susan E. Gueble, R. Bindra
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="The tumor suppressor protein p53 plays a key role in the cellular response to DNA damage. In response to DNA double strand breaks (DSB), cultured cells exhibit oscillations of p53 levels, which impact gene expression and cell fate. The dynamics of p53 in-vivo have only been studied in fixed tissues or using reporters for p53's transcriptional activity. Here we established breast tumors expressing a fluorescent reporter for p53 levels and employed intravital imaging to quantify its dynamics in response to DSB in-vivo. Our findings revealed large heterogeneity among individual cells, with most cells exhibiting a single prolonged pulse. We then tested how p53 dynamics might change under high cell confluency, one factor that differs between cell culture and tissues. We revealed that highly confluent cultured breast cancer cells also show one broad p53 pulse instead of oscillations. Through mathematical modeling, sensitivity analysis and live cell imaging we identified low levels of the phosphatase Wip1, a transcriptional target and negative regulator of p53, as a key contributor to these dynamics. Since high cell confluency better reflects the microenvironment of tissues, the impact of cell confluency on p53 dynamics may have important consequences for cancerous tissues responding to DNA damage inducing therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff150fb08f88fc51a25fd22d924b89d76aa03ba" target='_blank'>
              Cell Confluency Affects p53 Dynamics in Response to DNA Damage.
              </a>
            </td>
          <td>
            Alina Simerzin, Emily E Ackerman, Kotaro Fujimaki, Rainer H. Kohler, Yoshiko Iwamoto, Mathias S. Heltberg, A. Jambhekar, Ralph Weissleder, G. Lahav
          </td>
          <td>2025-04-09</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="
 Tumors are dynamic ecosystems where interactions between cancer cells and the tumor microenvironment (TME) shape malignancy. Using spatial transcriptomics, we analyzed 63 untreated tumors from 10 cancer types to investigate the spatial distribution of 13 cancer hallmarks. Our findings reveal that hallmark activity is spatially organized, with distinct compartmentalization of hallmark phenotypes across neoplastic and TME regions. Hallmarks such as "Resisting Cell Death" and "Enabling Replicative Immortality" were enriched in cancer-dominant areas, while "Inducing Angiogenesis" and "Activating Invasion and Metastasis" were primarily localized to the TME. Genomic analysis showed that subclonal diversity influences hallmark specialization, with genetically distinct clones often occupying specific ecological niches. These insights suggest that hallmark activities are not uniformly distributed but instead reflect the interplay between genetic heterogeneity and tumor spatial organization.
 We further modeled the spatial interdependence of hallmarks between the neoplastic and TME compartments, demonstrating that hallmark activities are shaped by ecological dynamics across the tumor. For example, hallmarks such as "Sustaining Proliferative Signaling" in the TME interface strongly influenced cancer cell hallmark expression. Application of this framework to 33 bladder cancer patients from the DUTRENEO trial revealed correlations between hallmark spatial patterns and treatment response. Notably, differences in the spatial localization of "Inducing Angiogenesis" and "Resisting Cell Death" were linked to neoadjuvant chemotherapy and immunotherapy sensitivity. These findings underscore the importance of spatial hallmark distribution in determining drug efficacy and suggest that spatial hallmark signatures could serve as predictive biomarkers for personalized cancer therapy.


 Eduard Porta-Pardo, Mustafa Sibai, Sergi Cervilla, Daniela Grases, Manel Esteller. The spatial landscape of cancer hallmarks reveals patterns of tumor ecological dynamics and drug sensitivity [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB265.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89052d3ed2d82ce2288e348ab845b3ae781f2935" target='_blank'>
              Abstract LB265: The spatial landscape of cancer hallmarks reveals patterns of tumor ecological dynamics and drug sensitivity
              </a>
            </td>
          <td>
            E. Porta-Pardo, Mustafa Sibai, Sergi Cervilla, D. Grases, M. Esteller
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Aberrant function or overactivation of exonuclease 1 (EXO1) may be associated with cancer tumor development, drug resistance, and response to immunotherapy in female-related cancers. Methods By analyzing RNA-sequencing data from The Cancer Genome Atlas database, combined with validation through quantitative polymerase chain reaction experiments, we explored the expression levels of EXO1 in breast cancer (BRCA) cell lines and assessed its multidimensional roles in various female-related cancers. Results Our experiments revealed elevated expression of EXO1 in BRCA cell lines, consistent with the RNA-sequencing data. The high expression of EXO1 is associated with poor prognosis in various female-related cancers, especially in BRCA and UCEC. It significantly correlates with clinical and pathological characteristics. In specific cancer subtypes like the basal-like subtype of BRCA, high EXO1 expression is associated with a better prognosis. Genetic mutation analysis indicates a higher frequency of EXO1 gene mutations in uterine sarcoma and BRCA. DNA methylation levels may play a role in the regulation of EXO1 gene expression in some cancers. EXO1 expression is correlated with various factors within the tumor immune microenvironment and may be associated with the sensitivity to anticancer drugs. Conclusion EXO1 exhibits multidimensional roles in female-related cancers as a prognostic biomarker and potentially influences tumor immune therapy responses and drug sensitivities. Further studies are needed to fully understand the complex mechanisms underlying these associations and to explore potential therapeutic strategies targeting EXO1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00f61f934389fb7d273be0eaa350ec5d0b17ae1" target='_blank'>
              Deciphering the multifaceted role of EXO1 in female-related cancers: implications for prognosis and therapeutic responsiveness
              </a>
            </td>
          <td>
            Cong Yu, Guoying Wu
          </td>
          <td>2025-05-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Ovarian cancer is one of the most common malignancies of the female reproductive system and is associated with poor prognosis. This study aimed to utilize single-cell RNA sequencing to investigate the heterogeneity of malignant epithelial cells in ovarian cancer, focusing on their potential functions and the implications for treatment and prognosis. Methods Single-cell RNA sequencing data were clustered using a single-cell transcriptome clustering method, and malignant epithelial cells were identified through copy number variation analysis. The interaction patterns between different malignant subpopulations and immune/stromal cells were analyzed using cell-to-cell communication analysis. A risk score (URS) model based on the UBE2C + epithelial subpopulation was then constructed through LASSO and multivariable Cox regression. High and low URS groups were compared in terms of tumor mutational burden (TMB), survival outcomes, and drug sensitivity. Finally, the role of GTF2F2 in ovarian cancer progression was validated through gene knockdown experiments in an ovarian cancer cell line (ES-2). Results Three major malignant epithelial cell subpopulations were identified (TMSB4X + Epi, TSC22D1 + Epi, and UBE2C + Epi). The UBE2C + Epi subpopulation exhibited higher stemness and greater invasive potential. The constructed URS model effectively stratified patients into high- and low-risk groups, with the high-risk group displaying a higher TMB level (p = 0.00011). Drug sensitivity predictions indicated that osimertinib, rapamycin, and dihydrorotenone might have stronger inhibitory effects in the high-risk group, whereas ERK inhibitors were more effective in the low-risk group. Functional assays demonstrated that GTF2F2 knockdown significantly suppressed ovarian cancer cell migration and invasion. Western blot analyses further showed elevated E-cadherin and reduced N-cadherin expression, suggesting that GTF2F2 may promote epithelial-mesenchymal transition (EMT). Conclusion The risk score model established in this study offers a novel framework for patient stratification and personalized therapy. Notably, the identification of the UBE2C + Epi subpopulation and key genes such as GTF2F2 highlights potential diagnostic and therapeutic targets, shedding light on the pathogenesis of ovarian cancer and paving the way for precision medicine approaches. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01686-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/519b55d3039bd908fc634a743e91a97b194c1fa0" target='_blank'>
              Single-cell RNA sequencing reveals the role of GTF2F2 in ovarian cancer oncogenesis and progression
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-05-29</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 PKMYT1, a serine/threonine kinase, selectively regulates CDK1 during the G2/M phase transition to prevent mitotic entry in the presence of DNA damage, whereas WEE1 kinase regulates both CDK1 and CDK2. Inhibition of PKMYT1 forces cancer cells to prematurely enter mitosis without repairing DNA damage, thereby leading to cell death. Therefore, PKMYT1 is an attractive target for synthetic lethality in DNA damage-sensitive cancer cells, such as those with hyperactive CCNE1/CDK2 signaling and p53 mutations.
 CCNE1 amplification (Copy number ≥ 6) is observed in approximately 8% of all solid tumor patients and is particularly prevalent in ovarian cancer (31%), uterine carcinosarcoma (37%), gastric cancer (14%), and breast cancer (8%). Despite the significant patient population harboring this biomarker, no targeted therapy has been approved, emphasizing a critical unmet medical need. CCNE1 amplification drives premature S-phase entry, resulting in genome instability due to DNA replication stress.
 VRN16 is a novel PKMYT1 kinase inhibitor with high target selectivity. KINOMEScanTM showed that VRN16 has fewer off-target kinases (1% with S(10)) than lunresertib (7% with S(10)), a clinical stage PKMYT1 inhibitor. Notably, VRN16 spares WEE1, PLKs, and other off-target kinases, including BRAF. In contrast, lunresertib inhibits BRAF kinase, confirmed to cause paradoxical activation of phospho-MEK and phospho-ERK signaling in the cells, similar to the effects observed with vemurafenib, a BRAF inhibitor. Indeed, in CCNE1+ cancer cells, VRN16 demonstrates higher proliferation inhibition potency than lunresertib. Conversely, in CCNE1-normal cells, VRN16 exhibits lower cytotoxicity compared to lunresertib.
 In three CCNE1+ CDX mouse models (OVCAR3, MKN1, and HCC1569), VRN16 demonstrated superior efficacy without observed toxicity compared to lunresertib which showed adverse effects such as stomatitis, keratosis, and edema at its MTD, 10 or 20 mg/kg, BID. Additionally, in a CCNE1+ ovarian cancer PDX model, VRN16 demonstrated dose-dependent efficacy as both monotherapy and in combination therapy with gemcitabine. Although CDK2 inhibitors are also undergoing clinical trials with CCNE1+ as a major biomarker, VRN16 demonstrated superior efficacy in combination with gemcitabine in the OVCAR3 CDX, compared to the combination of BLU-222, a CDK2 inhibitor, with paclitaxel or carboplatin.
 In a 2-week repeated-dose oral toxicity study, VRN16 achieved a wide preclinical safety margin, more than 8-fold, compared to lunresertib with a limited safety margin. This wide therapeutic window highlights VRN16 as a novel therapeutic option for patients with CCNE1+ solid tumors or DNA damage sensitive tumors.


 Kwangwoo Hwang, Seohyun Jo, Ho-Soo Lee, Youngyi Lee, Donghyun Park, Songyee Han, Kyungah Seo, Jiseon Choi, Soochan Kim, Jiyeon Kim, Donghyuk Seo, Younho Lee, Yikyung Ko, Hyunjong Kwak, Ye Ji Son, Jeonghun Lee, Soyeon Je, Hong-ryul Jung, Daekwon Kim, Sunghwan Kim, Jaeyoung Ahn. VRN16: A novel PKMYT1 kinase selective inhibitor with wide therapeutic window [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB436.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/960a7979050b70a2d24fbb180b4e757ef27292f4" target='_blank'>
              Abstract LB436: VRN16: A novel PKMYT1 kinase selective inhibitor with wide therapeutic window
              </a>
            </td>
          <td>
            Kwangwoo Hwang, Seohyun Jo, Ho-Soo Lee, Youngyi Lee, Donghyun Park, Songyee Han, Kyungah Seo, Jiseon Choi, Soochan Kim, Jiyeon Kim, Dong-hyuk Seo, Younho Lee, Yikyung Ko, Hyunjong Kwak, Ye Ji Son, Jeonghun Lee, Soyeon Je, Hong-ryul Jung, Daekwon Kim, Sunghwan Kim, J. Ahn
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Pancreatic cancer (PC) is the third leading cause of cancer deaths in US with a 5-year survival rate of 13%. Most patients are diagnosed at advanced stages, and conventional treatments remain ineffective and highly toxic to these patients as both chemotherapy and radiation affect normal, healthy cells, leading to adverse side effects. We have adapted CRISPR-Cas9 as a cancer-specific killing strategy that spares normal cells. CRISPR-Cas9 is a molecular scissor that induces DNA double strand breaks (DSBs) at its targeted genomic locations. We use CRISPR-Cas9 to target the subset of somatic mutations that create protospacer adjacent motifs (PAMs). Using S. pyogenes Cas9 that recognizes a 5’-NGG-3’ PAM, we identified somatic single base substitutions (SBSs) that give rise to novel PAMs (e.g. NCG→NGG), and use the upstream sequence to design sgRNAs for cancer-targeting. We have discovered that ∼13% of SBSs in PC form novel PAMs, and these cancer-specific sgRNAs exhibited on-target activity and lack of off-target events. Notably, while examining WGS data from rapid autopsy samples, we found that 90% of PAMs were maintained among primary carcinomas and metastases. Using multi-target sgRNAs targeting 2-16 sites in the non-coding regions of human genome, we demonstrate a dose-response relationship in which >10 target sites are required to achieve >99% cell elimination. Moreover, we established that simultaneous induction of multiple DSBs by CRISPR-Cas9 causes extreme chromosomal instability that ultimately leads to cell death. We detected massive chromosomal aberrations, including dicentrics, ring chromosomes, and polyploidization. Surprisingly, CRISPR-Cas9-induced DSBs are significantly more cytotoxic in PCs than a comparable number of radiation-induced DSBs, in which the number of DSBs required to achieve equitoxic effects was ∼3 times higher for radiation than CRISPR-Cas9. Finally, we have developed a pipeline to screen for the most toxic sgRNAs in a given PC for selective killing. Using our PAM-based approach, we identified >100 Panc10.05-specific sgRNAs that have no off-target activity at 0- and 1-mismatch sites. We cloned them into lentiviral vectors and transduced in bulk, followed by NGS to discover sgRNAs that decreased in representation in the targeted cell line but not in the controls. We selected the top sgRNAs that have 100% variant allele frequency (VAF), cloned 4 of them into a multi-sgRNA expression vector, and transduced it into co-cultures consisting of two PC cell lines. Greater than 85% selective reduction of the targeted cell line was detected. Then, we selected nine top sgRNAs, transduced them as a pool into Panc10.05 cell line, and injected these cells into athymic, nude mice for tumor growth. We observed >90% tumor growth inhibition with these cancer-specific sgRNAs. Overall, our study highlights the therapeutic potential of CRISPR-Cas9 as an anti-cancer strategy and supports the genetic targeting of PCs.


 Selina Shiqing K. Teh, Akhil Kotwal, Kirsten Bowland, Alexis Bennett, Eitan Halper-Stromberg, Laura Morsberger, Sarah Wheelan, Nicholas J. Roberts, Chien-Fu Hung, Michael Goldstein, Ying S. Zou, James R. Eshleman. CRISPR-Cas9 as a novel cancer gene therapy approach against pancreatic cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 471.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27f0e3c94f86f6d8c0491e1cda5afb641525f8e" target='_blank'>
              Abstract 471: CRISPR-Cas9 as a novel cancer gene therapy approach against pancreatic cancers
              </a>
            </td>
          <td>
            S. Teh, Akhil Kotwal, Kirsten Bowland, Alexis Bennett, E. Halper-Stromberg, L. Morsberger, Sarah Wheelan, Nicholas J. Roberts, Chien-Fu Hung, M. Goldstein, Ying S. Zou, J. Eshleman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="ARID1A, a subunit of the SWI/SNF chromatin remodeling complex, has emerged as a pivotal tumor suppressor altered in a broad range of human malignancies. Its frequent inactivation across diverse cancer types has revealed pleiotropic roles that intersect multiple Hallmarks of Cancer. In this review, we integrate current knowledge on how ARID1A loss influences cellular processes including proliferative signaling, resistance to cell death, genomic instability, metabolic reprogramming, immune evasion, and more. We discuss the context-specific consequences of ARID1A deficiency, its cooperation with other oncogenic events, and its implications for therapeutic vulnerability—particularly in the realm of synthetic lethality and immune modulation. By mapping ARID1A’s functional impact onto the established hallmarks framework, we highlight its centrality in cancer biology and underscore opportunities for biomarker-driven strategies and targeted interventions. Understanding ARID1A’s multifaceted roles offers a compelling lens through which to explore chromatin dysregulation in cancer and guide translational advances.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017483c65324200e0adc9a76376f32832abb8dbc" target='_blank'>
              ARID1A and Its Impact Across the Hallmarks of Cancer
              </a>
            </td>
          <td>
            Bridger Kearns, Andralyn McKell, Isaac Steveson, Peyton Worley, Braeden Barton, Jordan Bennett, DeLaney Anderson, Jacob Harris, James Christensen, Jared Barrott
          </td>
          <td>2025-05-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Sequencing of many types of tumors shows that solid tumors are far more mutated than liquid tumors and soft tumors such as brain tumors, but it is unclear whether the differences are caused by induction of more mutations or subsequent selective pressures. Our pan-cancer analyses of clinical data shows increased chromosomal changes associate with collagen-I and further shows that genetic variance between patients also increases with the mean for a given tumor type. Using a 3D hydrogel-based model with tunable stiffness and of cancer spheroids, we discover that matrix stiffness increases mitotic aberrations including micronuclei and also increases chromosome loss as detected using live cell chromosome reporters (ChReporters). However, stiffness suppresses growth and size variation, which indicates more mistakes per division. Our approach is >10-fold more sensitive than sequencing approaches, and we quantify colonies of ChReporter-negative cells within cancer spheroids. Our results are consistent with Luria-Delbruck’s theory of heritable genetic changes which predicts inter-spheroid variance greatly exceeds Poisson statistics. Knockdown of myosin-II, a well-known mechanosensor, increases chromosomal loss and variance without affecting spheroid growth, consistent with a tumor suppressor role. Various drug strategies provide deeper insight into mechanisms, and use of clinically deployed CDK4/6-inhibitor reveals this drug on its own drives micronuclei formation and genetic variation.


 Dennis E. Discher, Markus Sprenger. Solid tumor microenvironments cause more mutations than liquid or soft tumor microenvironments -- even before selection [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2669.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffd3cedfd64f7bf2629754512f7cbfd53af9bdaa" target='_blank'>
              Abstract 2669: Solid tumor microenvironments cause more mutations than liquid or soft tumor microenvironments -- even before selection
              </a>
            </td>
          <td>
            D.E. Discher, Markus T. Sprenger
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is characterized by high heterogeneity, and its molecular features and microenvironment complexity contribute to varied patient prognoses. Anoikis, a form of programmed cell death, is considered closely linked to the development and progression of liver cancer. This study aimed to analyze the cellular heterogeneity of HCC using single-cell sequencing and to construct an anoikis-related prognostic model for predicting patient outcomes. Methods Single-cell sequencing data of HCC patients were obtained from the GEO database. The AUCell algorithm was used to quantify anoikis gene activity across cell types. Cells were grouped based on enrichment scores. Cox regression and 101 machine learning algorithms were employed to identify prognostic genes and construct a risk model, which was validated using TCGA, ICGC, and GEO datasets. Key findings were validated using qPCR on clinical samples, and functional assays using shRNA-mediated knockdown of KPNA2 were conducted in Hep-3B cell lines to assess changes in proliferation and migration. Results Single-cell analysis revealed the cellular heterogeneity of HCC, with particular enrichment of anoikis-related genes in fibroblasts, endothelial cells, and macrophages. The prognostic model, developed using 101 algorithms, demonstrated the highest performance. High-risk patients showed poor prognosis, and the ROC curve analysis showed high accuracy for model predictions. Genetic mutation analysis indicated significant gene amplifications and deletions in the high-risk group. PCR analysis of clinical samples confirmed the expression patterns of key gene KPNA2, and gene knockdown in cell experiments further validated its oncogentic role in HCC. Conclusion This study highlights the anoikis-related cellular heterogeneity in HCC and constructs a machine learning-based prognostic model with high predictive value, enabling effective risk stratification for patients. It offers further support for the potential therapeutic application of the identified genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3b0ae0b20f3f5d93557a996379c7107447e12e9" target='_blank'>
              Anoikis in hepatocellular carcinoma: genetic alterations and immune microenvironment insights from single-cell and computational analysis
              </a>
            </td>
          <td>
            Bing Zhou, Jialing Zhang, Mingfeng Guo
          </td>
          <td>2025-05-31</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 The KIAA1549-BRAF (KB) fusion is a prevalent genetic event in two distinct central nervous system tumors: pilocytic astrocytomas (PAs) and diffuse leptomeningeal glioneuronal tumors (DLGNTs). This chromosomal rearrangement, consisting of a 2 Mbp tandem duplication at 7q34, results in the formation of a fusion protein absent in normal cells. The KB fusion protein has been described to dimerize the BRAF kinase in a RAS-independent manner, leading to a constitutive activation of the MAPK pathway and promoting tumorigenesis. Despite its significance, the lack of pre-clinical models limits our understanding of how KB promotes DLGNTs and PAs development, so effective therapeutic strategies remain elusive. To address this, we aimed to model the KB fusion at its endogenous chromosomal location.
 Overcoming the challenges of modeling chromosomal rearrangements, we used the CRISPR-Cas9 gene-editing technique to model the fusion event ex vivo and in vivo in the mouse brain, leveraging p53- or Arf-deleted backgrounds. These models allow us to investigate the molecular mechanisms by which KB promotes tumor initiation and progression and test the therapeutic potential of Belvarafenib, a class II BRAF inhibitor that targets BRAF dimers.
 Our results demonstrate that CRISPR-mediated engineering of KB in p53- or Arf-deficient backgrounds, both ex vivo and in vivo, generates CNS tumors resembling human DLGNTs in their histology and growth patterns. Furthermore, targeting the robust MAPK pathway activation in these KB-driven tumor models with Belvarafenib significantly reduced ERK phosphorylation and tumor growth, and significantly increased the survival of treated mice.
 These findings demonstrate that KB promotes tumor initiation and progression, and highlight the therapeutic potential of Belvarafenib targeting this oncogenic event. Thus, our KB-DLGNTs model emerges as a valuable tool to evaluate the efficacy of novel pre-clinical BRAF inhibitors.


 Ylenia Cendon Florez, Ram Kannan, Matthias A. Karrajannis, Darlene Dela Cruz, Udi Segal, Danilo Maddalo, Scott Foster, Robert Benezra, Andrea Ventura. Unraveling Kiaa1549-Braf driven brain tumors: From modeling to therapy [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7033.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37141413d3e7032baebf85246c91ff6528d18e4f" target='_blank'>
              Abstract 7033: Unraveling Kiaa1549-Braf driven brain tumors: From modeling to therapy
              </a>
            </td>
          <td>
            Ylenia Cendon Florez, Ram Kannan, Matthias A. Karrajannis, Darlene Dela Cruz, Udi Segal, Danilo Maddalo, Scott Foster, R. Benezra, Andrea Ventura
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Cervical and other anogenital malignancies are largely caused by E6 and E7 oncogenes of high-risk human papillomaviruses (HPVs), which inhibit important tumor suppressors like p53 and pRb when they are persistently activated. The main goal of traditional treatments is to physically or chemically kill cancer cells, but they frequently only offer temporary relief, have serious side effects, and have a high risk of recurrence. Exploring the efficacy and accuracy of CRISPR-Cas9 gene editing in both inducing death in HPV-infected cancer cells and restoring the activity of tumor suppressors is our main goal. In this study, we propose a novel precision oncology strategy that targets and inhibits the detrimental effects of the E6 and E7 oncogenes using the CRISPR-Cas9 gene editing system. In order to do this, we create unique guide RNAs that target the integrated HPV DNA and reactivate p53 and pRb. Reactivation is meant to halt aberrant cell development and restart the cell’s natural dying pathways. This review discusses the potential of CRISPR/Cas9 in targeting HPV oncogenes, with a focus on studies that have demonstrated its promise in cancer treatment. Given the absence of a definitive treatment for papillomavirus infection and its subsequent association with various cancers, future clinical trials and experimental investigations appear essential to establish and evaluate the therapeutic potential of CRISPR-based approaches. This approach provides a less invasive alternative to conventional treatments and opens the door to personalized care that considers the genetic makeup of each patient’s tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/612cd6ffb8df66c436b36650924c944f8c2ee3b0" target='_blank'>
              HPV-driven cancers: a looming threat and the potential of CRISPR/Cas9 for targeted therapy
              </a>
            </td>
          <td>
            Atefeh Zamani Kermanshahi, Fatemeh Ebrahimi, Ahmad Taherpoor, Narges Eslami, H. B. Baghi
          </td>
          <td>2025-05-22</td>
          <td>Virology Journal</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Embryogenesis and tumorigenesis share several key biological characteristics, such as rapid cell proliferation, high plasticity, and immune evasion. This similarity indicates that developmental pathways can be hijacked, leading to the formation of malignant cell states. With regard to this, cancer can be regarded as a stem cell disease. On the contrary, a fetus, in many ways, has similar characteristics to the “ideal tumor”, such as immune evasion and rapid growth. Therefore, deciphering the molecular mechanisms beneath these phenomena will help us to understand the embryonic origins of cancer. This review discusses the relationship between embryogenesis and tumorigenesis, highlighting the potential roles played by DUX4. DUX4 is involved in the activation of the zygote genome and then facilitates the establishment of totipotency in pre-implantation embryos, whereas the misexpression of DUX4 is associated with different types of cancer. Taken together, this indicates that DUX4 performs analogous functions in these two processes and connects embryogenesis and tumorigenesis. Through examining DUX4, this review underscores the importance of developmental mechanisms in cancer biology, suggesting that the insights gained from studying embryonic processes may provide novel therapeutic strategies. As we continue to explore the complex relationship between cancer and embryogenesis, elucidating the role of DUX4 in linking these two processes will be critical for developing targeted therapies that exploit developmental pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8f5d78fbcd896e6b005b6f4da73702c30eda3a6" target='_blank'>
              Embryonic Origins of Cancer: Insights from Double Homeobox 4 Regulation
              </a>
            </td>
          <td>
            B. Fu, Hong Ma, Liang Wang, Zhenhua Guo, Fang Wang, Di Liu, Dongjie Zhang
          </td>
          <td>2025-05-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cervical cancer represents a significant global health challenge, with complex cellular and molecular mechanisms driving its progression from HPV infection to invasive malignancy. This study employed an integrated approach combining single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics (stRNA-seq) to comprehensively characterize the tumor microenvironment (TME) across different stages of cervical cancer development. Through analysis of samples from normal cervix, HPV-infected normal cervix, high-grade squamous intraepithelial lesions (HSIL), and invasive cervical cancer, we identified distinct cellular populations and their dynamic changes during disease progression. Our findings revealed significant heterogeneity in immune cell populations, particularly highlighting the role of SPP1+ macrophages that were substantially enriched in cervical cancer compared to precancerous and normal tissues. Cell-cell communication networks and spatial mapping demonstrated that SPP1+ macrophages interact extensively with immune cells through the SPP1-CD44 signaling axis. This interaction contributes to an immunosuppressive microenvironment through modulation of T cell function and promotion of tumor cell survival. Furthermore, high expression of SPP1 correlated with advanced tumor stages and poor overall survival in cervical cancer patients, highlighting its potential as a prognostic biomarker. Our comprehensive characterization of the cellular landscape and intercellular communication networks in cervical cancer progression provides valuable insights for the development of targeted therapeutic strategies aimed at modulating the TME, particularly through disruption of the SPP1-CD44 axis. These findings establish a foundation for more effective personalized approaches to improve clinical outcomes in cervical cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87cbb4dd12be6f1664aacd1df3e0b1efa6fe30d7" target='_blank'>
              Deciphering the cellular and molecular landscape of cervical cancer progression through single-cell and spatial transcriptomics
              </a>
            </td>
          <td>
            Peng Xia, Juanhong Zhou, Rong Shen, Degui Wang
          </td>
          <td>2025-05-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Cell-free DNA (cfDNA) fragments are shed into bodily fluids as a result of apoptosis and necrosis. Profiling plasma cfDNA has emerged as a powerful tool in cancer diagnostics and research, through circulating tumor DNA (ctDNA) fragments that enable genotyping of many tumors, and epigenetic analysis of cfDNA, including fragmentomics. Fragmentomics encompasses the analysis of fragment lengths, end motifs, and central dyads, which reflect epigenetic patterns, to identify the cell of origin for each fragment.
 We previously showed that fragment size diversity is strongly correlated with gene expression (Esfahani et al, NBT 2022). We hypothesize that parameters inferred from a mathematical model for the fragmentation process can enhance gene expression inference from cfDNA.



 We followed the mechanistic model previously proposed (Esfahani et al, NBT 2022), and modified it with additional steps. For each transcription start site (TSS), we input the mean and variance of the linker DNA length, fraction of nucleosomes sites occupied, cut frequency, and DNA degradation factor among other factors. Then, in accordance with the given parameters, we generate a sequence of nucleosome units, a sample of nucleosome sites occupied, and a sequence of fragments, and degrade them.
 For chromatin landscaping of plasma cfDNA, we used four cfDNA samples profiled by deep whole genome sequencing (∼250X). For each of 500 TSSs, we fitted our model to the fragment size distribution from data by minimizing the Jensen-Shannon (JS) divergence between the generated fragment size distribution and the fragment size distribution from the sample. Finally, we performed regression analysis on the fitted parameters to infer gene expression.



 We analyzed cfDNA fragmentation spectra at TSSs genome-wide in relation to transcriptome-wide gene expression levels. The regression shows that gene expression has a negative correlation (P<0.001) between fraction of nucleosome sites occupied, a positive correlation (P=0.002) with the degradation factor, and a negative correlation (P=0.009) with the mean linker length.



 We show that cfDNA fragmentation can be modeled as a multi-step stochastic process. Through surveying TSSs, we showed significant correlation between certain model parameters and gene expression. Our findings that gene expression is negatively correlated with model parameters for fraction of nucleosome sites occupied and mean linker length is consistent with previous literature suggesting that higher fraction of nucleosome sites occupied and longer linker length lead to suppressed gene expression (Valouev et al, Nature 2011). We envision this model-based approach will enhance gene expression inference that can ultimately improve cancer detection and tissue of origin classification.



 Brian Sun Liu, Mengran Zhang, Takeshi Sugio, Mohammad Shahrokh Esfahani. Mathematical modeling of cell-free DNA fragmentation to map chromatin landscapes in health and cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6243.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63affd7e1896b137912e86c03b4cbb1a8fc8724f" target='_blank'>
              Abstract 6243: Mathematical modeling of cell-free DNA fragmentation to map chromatin landscapes in health and cancer
              </a>
            </td>
          <td>
            Brian Sun Liu, Mengran Zhang, Takeshi Sugio, M. Esfahani
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Motivation The National Cancer Institute Genomic Data Commons (GDC) provides controlled access to sequencing data from thousands of subjects, enabling large-scale study of impactful genetic alterations such as simple and complex germline and structural variants. However, efficient analysis requires significant computational resources and expertise, especially when calling variants from raw sequence reads. To solve these problems, we developed bamSliceR, a R/bioconductor package that builds upon the GenomicDataCommons package to extract aligned sequence reads from cross-GDC meta-cohorts, followed by targeted analysis of variants and effects (including transcript-aware variant annotation from transcriptome-aligned GDC RNA data). Results Here, we demonstrate population-scale genomic and transcriptomic analyses with minimal compute burden using bamSliceR, identifying recurrent, clinically relevant sequence, and structural variants in the TARGET acute myeloid leukemia (AML) and BEAT-AML cohorts. We then validate results in the (non-GDC) Leucegene cohort, demonstrating how the bamSliceR pipeline can be seamlessly applied to replicate findings in non-GDC cohorts. These variants directly yield clinically impactful and biologically testable hypotheses for mechanistic investigation. Availability and implementation bamSliceR has been submitted to the Bioconductor project, where it is presently under review, and is available on GitHub at https://github.com/trichelab/bamSliceR">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb3dfbdbeef9d56dacad316cb41e99f3513a6dab" target='_blank'>
              bamSliceR: a Bioconductor package for rapid, cross-cohort variant and allelic bias analysis
              </a>
            </td>
          <td>
            Yizhou Peter Huang, Lauren Harmon, Eve Deering-Gardner, Xiaotu Ma, Josiah Harsh, Zhaoyu Xue, Hong Wen, Marcel Ramos, Sean R. Davis, Timothy J. Triche
          </td>
          <td>2025-04-28</td>
          <td>Bioinformatics Advances</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Summary: The term “anaplasia” was coined in 1890 to describe chromosomal changes common to primordial cancer cells, but ever since the mechanisms whereby a cell becomes anaplastic has been the subject of much speculation. Recent results based on genomic and epigenomic profiles of cancer patient samples provide a glimpse into early events that lead to aneuploidy, the original defining feature of an anaplastic cell. We propose that the anaplastic cell is one in which RNA Polymerase II hypertranscribes S-phase-dependent histone genes, and the resulting histone excess facilitates DNA replication while competing for CENP-A, causing centromere breaks that initiate whole-arm aneuploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e327392e26ad44de92c835846a2ea7101b5c4f9e" target='_blank'>
              Hansemann’s anaplastic theory of cancer after 135 years
              </a>
            </td>
          <td>
            S. Henikoff, K. Ahmad
          </td>
          <td>2025-05-02</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 Chromosome instability (CIN) is a prevalent hallmark of aggressive cancers that is largely absent in normal cells. Targeting tumors with CIN is a therapeutic strategy that is rapidly advancing. Volastra Therapeutics has undertaken multiple orthogonal approaches to target CIN as a cancer vulnerability. One such approach that relies on the Cancer Dependency Map (DepMap) identified BUB1B (BUB1 Mitotic Checkpoint Serine/Threonine Kinase B) as a top scoring gene required for survival of cell lines with increased genomic instability. BUB1B, a core component of the spindle assembly checkpoint that functions in cell cycle progression and chromosome segregation, was also highlighted in published screens as a major cancer vulnerability in cells with aneuploidy and whole-genome duplication.
 To corroborate the dependency of chromosomally unstable cancer cells on BUB1B, we used three independent cell models: 1) an isogenic CIN system based on modulating kinetochore-microtubule dynamics, 2) a Cas9-engineered system to induce dicentric chromosomes through telomere deprotection and 3) a non-isogenic cell line model based on differential levels of aneuploidy and micronuclei. Data in these three orthogonal systems confirmed that chromosomally unstable cells rely heavily on BUB1B for survival.
 We next explored the development of BUB1B small molecule inhibitors. The C-terminus of BUB1B is responsible for chromosome congression and contains the putative kinase domain, suggesting that catalytic activity may be responsible for CIN suppression. Whether BUB1B possesses enzymatic activity has been controversial, with some groups reporting low but inhibitable BUB1B kinase activity, while others indicate that BUB1B lacks key kinase residues and is catalytically inactive.
 To probe the function of BUB1B, we generated full-length and kinase domain recombinant proteins. Both constructs showed very weak ATPase activity comparable to published and commercially available proteins. Efforts to boost the kinase activity through improved substrate specificity or enhanced phosphorylation were not successful, and additional purification resulted in further loss of kinase activity. Furthermore, modeling of the BUB1B kinase domain suggests that ATP binding to the ATP cleft is physically hindered by bulky residues. Finally, substrate sequence mapping did not match sequences predicted for BUB1B. Together the data suggest that BUB1B is not enzymatically active and that the measured low ATPase activity may be attributable to a co-purifying kinase.
 Our work confirms chromosomally unstable cells have greater dependency on BUB1B. However, the protein lacks catalytic activity and thus cannot be targeted with a kinase inhibitor strategy. BUB1B’s function as a protein interaction hub with specific partners could permit alternative therapeutic strategies that may result in effective treatments for patients with chromosomally unstable cancers.


 Nazario Bosco, Rafael S. Depetris, Jasmina Prabhakara, Akanksha Verma, Jared L. Johnson, Lewis C. Cantley, Scott Drutman, Shinsan M. Su, Rumin Zhang, Derek A. Cogan, Christina H. Eng. BUB1B is an inactive kinase required for survival of chromosomally unstable cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2983.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3d0eb8bb7d998c887793731d6c54bbc2c2eb824" target='_blank'>
              Abstract 2983: BUB1B is an inactive kinase required for survival of chromosomally unstable cells
              </a>
            </td>
          <td>
            Nazario Bosco, R. Depetris, Jasmina Prabhakara, Akanksha Verma, Jared L. Johnson, Lewis C Cantley, Scott Drutman, Shinsan M. Su, Rumin Zhang, Derek A. Cogan, Christina H Eng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Keratinocyte carcinomas (KCs), comprising basal cell carcinoma (BCC) and cutaneous squamous cell carcinoma (cSCC), arise in the epidermis due to chronic exposure to solar ultraviolet (UV) radiation. KCs account for 99% of skin neoplasms and represent the most prevalent form of cancer worldwide. DNA methylation, an epigenetic modification of CpG dinucleotides, modulates gene expression, thereby defining cell identity and mediating cellular responses to external stimuli. Aging and environmental stressors, including UV radiation, progressively alter epidermal methylomes, thus influencing KC development and progression. Xeroderma pigmentosum (XP) is a rare genetic disorder characterized by mutations in genes of major cellular response mechanisms to UV-induced DNA damage, such as the nucleotide excision repair (NER) system or the translesion synthesis (TLS) DNA polymerase η. Consequently, XP patients exhibit a more than 10,000-fold increase in KC incidence, with tumors potentially manifesting as early as childhood. Epigenomic analysis of tumors derived from XP patients could provide valuable insights into the deregulation of DNA methylation in KC tumorigenesis, thereby contributing to the development of novel antitumor therapies for both XP patients and the broader population. Here, we used Infinium MethylationEPIC BeadChips to assess the DNA methylation status of 30 epidermal samples from XP patients with mutations in the XPC and POLH genes, representing roughly 50% of all XP cases and significantly impacted by KCs. Our sample set comprised BCC and cSCC tumors, non-tumorous epidermis, and actinic keratosis (AK), a precancerous lesion that frequently progresses to cSCC. DNA methylation analysis of XP samples clearly distinguished between BCC and cSCC tumors, whereas non-tumorous epidermis unexpectedly clustered with AK samples, reflecting substantial methylome dysregulation that is consistent with the heightened risk of KC development in XP patients. In line with this, further examination of non-tumorous XP methylomes relative to non-XP counterparts revealed features indicative of premature aging, including significant erosion of DNA methylation profiles and increased mitotic rates. Analysis of XP-derived BCC and cSCC methylation profiles also identified a higher frequency of copy number variation (CNV) events and a potential bias towards an epidermal stem cell (EpSC)-like cell-of-origin in cSCC tumors. These findings may suggest an elevated aggressiveness of XP tumors, which we now aim to verify using cellular models of KC deficient in key XP genes.


 Bea Riebesehl,Lea S. Beensen,Felix Bormann,Wilfried Roth,Carlos F. Menck,Thomas G. Hofmann,Frank Lyko,Manuel Rodríguez Paredes. Epigenomic characterization of keratinocyte carcinomas in xeroderma pigmentosum [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 208.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e4f1b205820b2198713ca41956bb0fc4b67c197" target='_blank'>
              Abstract 208: Epigenomic characterization of keratinocyte carcinomas in xeroderma pigmentosum
              </a>
            </td>
          <td>
            Bea Riebesehl, Lea S. Beensen, Felix Bormann, Wilfried Roth, C. Menck, Thomas G. Hofmann, F. Lyko, M. R. Paredes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="
 Cell cycle checkpoints serve as cellular critical safeguard mechanisms that ensure orderly progression through the cell cycle. PKMYT1, a membrane-associated kinase, functions as a negative regulator of CDK1 activity by phosphorylating CDK1 at the Thr14 residue. This enzyme is frequently overexpressed in a variety of cancers. Recent studies suggest that cancer cells harboring CCNE1 amplification (CCNE1Amp) and/or FBXW7 loss-of-function (FBXW7LoF) mutations are particularly relying on PKMYT1 for orderly entry to mitosis. As a result, inhibition of PKMYT1 can lead to premature mitotic entry, inducing DNA damage, and triggering mitotic catastrophe. CCNE1Amp is a common genetic alteration found across a spectrum of cancer types, with particular prevalence in gynecological malignancies. Elevated levels of cyclin E are linked to genomic instability, whole-genome doubling, and resistance to existing therapeutic interventions. FBXW7, a tumor suppressor gene, is frequently mutated in various cancers, including colorectal and ovarian serous carcinoma. Loss of FBXW7 function is recognized as an independent prognostic factor, significantly correlating with tumor cell resistance to chemotherapy and unfavorable disease outcomes. Moreover, FBXW7 acts as an E3 ubiquitin ligase for cyclin E, potentially leading to increased levels of cyclin E through post-translational mechanisms. QLS1209, an orally bioavailable, potent, and selective inhibitor of PKMYT1 was discovered and evaluated preclinically. QLS1209 exhibited single-digit nanomolar potency in inhibiting PKMYT1's enzymatic activity and demonstrated approximately 30-fold selectivity over its paralogue Wee1. QLS1209 potently suppressed phosphorylation of CDK1 at Thr14, with a minimal impact on Tyr15. QLS1209 was well tolerated and demonstrated anti-tumor activity in CCNE1Amp HCC1569 and OVCAR3 xenograft models in vivo, it also showed enhanced efficacy when combined with Gemcitabine. In a gastric patient-derived xenograft (PDX) model harboring an FBXW7LoF mutation, QLS1209 dose-dependently inhibited tumor growth, with higher dose leading to tumor regression. QLS1209 exhibits excellent pharmacokinetic profile, and promising in vitro ADME properties, along with a favorable safety profile. IND enabling studies are currently underway and are expected to be completed by the end of 2024. Collectively, these findings highlight the promising potential of QLS1209 as a therapeutic strategy, either as monotherapy or in combination with other anti-cancer therapies, for a variety of cancer types characterized by CCNE1Amp and/ or FBXW7LoF mutations.


 Yuxing Zhang, Jiasheng Fu, Rongyan Wang, Jianping Chen, Changliang He, Junguo Hao, Xinrui Shao, Ling Li, Hua Qin, Guangyue Lei, Ping Chen, Dong Yang, Xinghua Cheng, Qingmei Zheng, Zheng Wang, Guqin Shi, Wenyuan Qian, Daqing Sun, Weimei Sun, Weikang Tao. QLS1209, A highly potent and selective PKMYT1 inhibitor demonstrates activity in tumors with CCNE1Amp and/ or FBXW7LoF mutations [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1743.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e324f756104548defdd734daed3dcc1c70d2889" target='_blank'>
              Abstract 1743: QLS1209, A highly potent and selective PKMYT1 inhibitor demonstrates activity in tumors with CCNE1Amp and/ or FBXW7LoF mutations
              </a>
            </td>
          <td>
            Yuxing Zhang, Jiasheng Fu, Rongyan Wang, Jianping Chen, Changliang He, Junguo Hao, Xinrui Shao, Ling Li, Hua Qin, Guangyue Lei, Ping Chen, Dong Yang, Xinghua Cheng, Qingmei Zheng, Zheng Wang, Guqin Shi, Wenyuan Qian, Daqing Sun, Weimei Sun, Weikang Tao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Most solid tumors harbor somatic mutations attributed to off-target activities of APOBEC3A (A3A) and/or APOBEC3B (A3B). However, how APOBEC3A/B enzymes affect tumor evolution in the presence of exogenous mutagenic processes is largely unknown. Here, multi-omics profiling of 309 lung cancers from smokers identifies two subtypes defined by low (LAS) and high (HAS) APOBEC mutagenesis. LAS are enriched for A3B-like mutagenesis and KRAS mutations; HAS for A3A-like mutagenesis and TP53 mutations. Compared to LAS, HAS have older age at onset and high proportions of newly generated progenitor-like cells likely due to the combined tobacco smoking- and APOBEC3A-associated DNA damage and apoptosis. Consistently, HAS exhibit high expression of pulmonary healing signaling pathway, stemness markers, distal cell-of-origin, more neoantigens, slower clonal expansion, but no smoking-associated genomic/epigenomic changes. With validation in 184 lung tumor samples, these findings show how heterogeneity in mutational burden across co-occurring mutational processes and cell types contributes to tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c82f658add82b575c331a2df70337786a4843a5f" target='_blank'>
              APOBEC affects tumor evolution and age at onset of lung cancer in smokers
              </a>
            </td>
          <td>
            Tongwu Zhang, Jian Sang, P. Hoang, Wei Zhao, J. Rosenbaum, K. E. Johnson, L. Klimczak, John P. McElderry, Alyssa Klein, Christopher Wirth, E. Bergstrom, M. Díaz-Gay, Raviteja Vangara, Frank J. Colón-Matos, A. Hutchinson, Scott M Lawrence, N. Cole, Bin Zhu, T. Przytycka, Jianxin Shi, N. E. Caporaso, Robert J Homer, A. Pesatori, D. Consonni, Marcin Imieliński, Stephen J Chanock, David C Wedge, D. Gordenin, Ludmil B. Alexandrov, R. Harris, Marion Landi
          </td>
          <td>2025-05-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="
 The development of metastatic disease is a conserved feature across most solid organ malignancies and accounts for the majority of cancer-related deaths. While adaptations within the local immune microenvironment of primary tumors are well-appreciated hallmarks of tumor progression, there remains much unknown about the specific impact of metastasis on both local and systemic immune responses. Here, we present an analysis of changes in the adaptive immune response to tumors during metastasis and clonal selection. Our group previously developed a syngeneic melanoma mouse model of spontaneous lymph node metastasis, performing nine generations of in vivo serial passaging to select for metastatic phenotypes. As previous work suggests, we find that tumors undergo clonal selection during metastasis using lentivirus integration mapping to measure clonality. Even amidst reduced genomic diversity, we find that metastases exhibit comparable or increased transcriptomic diversity when compared to primary tumors. In light of this discordance between genomic and transcriptomic diversity, we ask whether the immune response, especially T cells, exhibit similar dynamics. Performing both bulk and single cell T cell receptor (TCR) sequencing, we find that T cell clonality mirrors the reduced clonal diversity of metastatic tumors both in the total T cell compartment and specifically in regulatory T cells. Through paired investigation of evolution of tumors and the responding immune cells, we develop a model of tumor-immune coevolution in which reduced tumor clonality selects for narrower T cell repertoires; yet, diverse tumor transcriptional profiles support the differentiation of a variety of T cell subsets that are advantageous for tumors including regulatory T cells and exhausted CD8+ T cells. We further perturb this dynamic by evolving new metastatic cell lines in the absence of T cell pressure, finding unique evolution of metastatic pathways when the immune system is altered, impairing metastasis when reimplanted into immunocompetent mice. This model of tumor-immune coevolution establishes the importance of ongoing, dynamic communication between the host and tumors throughout metastatic progression.


 Cort B. Breuer, Marcos Labrado, Nathan E. Reticker-Flynn. Antagonism between tumors and lymphocytes drives coevolutionary processes throughout metastasis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3458.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d9f71c97b02d4a1d072a400c199093fb87572c0" target='_blank'>
              Abstract 3458: Antagonism between tumors and lymphocytes drives coevolutionary processes throughout metastasis
              </a>
            </td>
          <td>
            Cort B Breuer, Marcos Labrado, Nathan E. Reticker-Flynn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Half of of colorectal cancer (CRC) patients develop metastatic CRC (mCRC) with a low survival rate. Epigenetic abnormalities in promoters and enhancers play a significant role in CRC advancement and hold promise for diagnosis and therapy and over 5, 000 variable enhancer loci (VELs) have been shown to be gained/lost in mCRC. Here, the functional impact of these VELs on driving CRC progression was explored. We designed a CRISPRi library including 1, 524 VELs gained or lost specifically in CRC cell lines, the top 500 essential genes for the cell line as positive controls, 1, 000 non-targeting guides as negative controls and 6 sgRNAs per site, cloned into a lentiviral plasmid. CRISPR screen was applied to the HCT116-dCas9-KRAB-MeCP2 cells for NGS. Anticancer potentials of silenced VELs were evaluated by in vitro assays (i.e MTT assay, apoptosis, cell cycle assay, 3D bioprinting). Chromosomal interactions and epigenetic state were analyzed by Chromosome conformation capture (3C) coupled with sequencing (Hi-C), chromatin immunoprecipitation (ChIP) with massively parallel DNA sequencing (ChIP-seq) and RNA sequencing. We identified 12 depleted VELs causing a significant decrease in HCT116 proliferation after CRISPR-mediated silencing. VEL_1243 and VEL_1084 caused the highest level of cell growth inhibition as well as increased apoptosis and cell cycle arrest in HCT116 cells. After silencing both enhancers, 3D bioprinting showed a reduction in key markers of metastasis. HiC-Seq and transcription factor binding data showed long-range interaction of both enhancers with genes such as MYO3B, UBR3 and GINS3 and enrichment for disease-associated TF including RFX7, and ELF3. RNA seq results showed significant transcriptome-wide expression alterations regulated by VEL_1243 and VEL_1084, exemplified by their impact on disease-associated signaling pathways including PI3K-Akt and MAPK. IFNγ response pathway -which is a critical aspect of anti-tumor immunity - was downregulated following both VEL_1243 and VEL_1084 silencing. Given that immune/inflammatory cells (e.g., tumor-associated macrophages (TAMs), T-cells, natural killer (NK) and cancer-associated fibroblasts (CAFs) are the important components of the tumor microenvironment (TME), we further examined if VELs mediated regulation of TAMs or immune cells controls the TME in a way that promotes tumour growth. M1 were co-cultured with CRC cells knock-down of VELs. Both VELs repression could induce M1 polarization in HCT116 and SW620 cells. Considering M1 macrophages have the capability to promote the Th1 response and engulf and kill target tumor cells, VEL_1084 and VEL_1243 could be potential therapeutic targets for CRC by affecting the polarization of macrophages. VEL_1243 and VEL_1054 regulate CRC-associated phenotypes and warrant further exploration to determine their regulatory role and potential as a novel therapeutic target in CRC.


 Flavia Genua, Xinxin Shao, Ellen Finnegan, Christina Caragine, Sarah Maguire, Paulien Van Minsel, Grainne Kirwan, Bernard Thienpont, Neville Sanjana, Weipeng Wang, Sudipto Das. Gene enhancers as novel rationale therapeutic targets for impeding colorectal cancer progression [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7181.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d3123c40ae6a85b19d981bbd4cd36ea0d8081af" target='_blank'>
              Abstract 7181: Gene enhancers as novel rationale therapeutic targets for impeding colorectal cancer progression
              </a>
            </td>
          <td>
            F. Genua, Xinxin Shao, E. Finnegan, Christina Caragine, Sarah L. Maguire, Paulien Van Minsel, G. Kirwan, Bernard Thienpont, Neville Sanjana, Weipeng Wang, Sudipto Das
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Intratumor heterogeneity and evolution underlie treatment failures but are poorly understood due to a limited number of tumor biopsies. In this study, we examine IDH mutant gliomas, utilizing a 3D whole tumor sampling approach and exome sequencing (32 patients, 345 samples), along with RNAseq (12 of the 32 patients, 88 samples). We investigate and redefine intratumoral heterogeneity and evolution of insertion/deletion (indel) and single nucleotide variant (SNV) in driver genes on a whole tumor scale.
 In the tumor suppressor capicua transcriptional repressor (CIC), a canonical driver in IDH mutant 1p/19q co-deleted (IDHmut-Codel) tumors, our exome and RNAseq data analysis highlights a wide spectrum of heterogeneity and inactivation by SNVs and indels in independent subclonal expansions. For instance, in nine samples from one patient’s recurrent tumor, five showed distinct CIC SNVs and eight exhibited indels. Notably, all samples showed one or more CIC SNVs or indels. Among five samples harboring SNVs, three had 2 to 3 co-occurring SNVs, and among the 8 samples with indels, three had 2 and one had 5 indels. None had the same combinations of indels and SNVs, indicating extreme heterogeneity and subclonality. Additionally, TA-cloning of a long-range PCR from one sample suggested that each individual SNV and indel were in different tumor cells, raising the possibility of early diversification of subclonal expansions driven by independent CIC alterations. Similar patterns were observed in IDH-mutant and 1p/19q non-codeleted tumors involving tumor suppressor ATRX.
 These findings demonstrate that lower grade IDH-mutant gliomas (LGGs) exhibit significant intratumoral heterogeneity, a trait typically associated with glioblastoma. To determine if CIC alterations were inactivating or simply passengers, we analyzed gene expression in those spatially mapped samples. Within the ETS-domain transcription factor family, ETS1, ETV1, ETV4, and ETV5 were upregulated in CIC mutant samples versus wild type samples. We also identified novel candidate genes potentially regulated by CIC, suggesting possible mechanisms contributing to tumorigenesis and new treatment targets for IDHmut-Codel tumors.
 This study elucidates the substantial intratumor heterogeneity and evolutionary dynamics in the drivers of LGGs on a whole tumor scale, highlighting their important role in tumorigenesis. By utilizing spatially mapped samples within the same tumors, our research reveals convergence on CIC loss of function through many mutually exclusive SNVs and indels in IDHmut-Codel tumors. Our findings demonstrate the critical nature of CIC inactivation across entire tumors, emphasizing the importance of indels and SNVs in tumor progression. This research also identifies candidate genes upregulated by CIC inactivation as possible treatment targets for IDHmut-Codel tumors.


 Lee H. Chen, Nicholas O. Stevers, Chibo Hong, Ivan V. Smirnov, Nadia R. Grishanina, Joshua S. Wu, Anny Shai, Susan M. Chang, Mitchel S. Berger, Shawn L. Hervey-Jumper, Joanna J. Phillips, Joseph F. Costello. Three-dimensional whole tumor genomic analysis redefine intratumor heterogeneity of gliomas, with implications for early tumorigenesis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3894.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a70d8ffb57c8661082ae6d08c258b2d7875bf6f" target='_blank'>
              Abstract 3894: Three-dimensional whole tumor genomic analysis redefine intratumor heterogeneity of gliomas, with implications for early tumorigenesis
              </a>
            </td>
          <td>
            Lee H. Chen, Nicholas O Stevers, Chibo Hong, Ivan V. Smirnov, Nadia R. Grishanina, Joshua S. Wu, Anny Shai, Susan Marina Chang, Mitchell S. Berger, Shawn L. Hervey-Jumper, Joanna J Phillips, Joseph F Costello
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Centromere protein F (CENP‐F) is an important nuclear matrix protein that regulates mitosis and the cell cycle, and plays a crucial role in recruiting spindle checkpoint proteins to maintain the accuracy of chromosome segregation. Studies have shown that CENP‐F is closely involved in the pathogenesis of various diseases, particularly in the development and progression of malignant tumors, where it exhibits significant oncogenic activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e907c4941d09e4b8b2b05c38929140acf5c506d" target='_blank'>
              Centromere Protein F in Tumor Biology: Cancer's Achilles Heel
              </a>
            </td>
          <td>
            Zitong Wan, Miaomiao Wen, Chunlong Zheng, Ying Sun, Yinxi Zhou, Yahui Tian, Shaowei Xin, Xuejiao Wang, Xiaohong Ji, Jie Yang, Yanlu Xiong, Yong Han
          </td>
          <td>2025-05-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94ef251e1dfb7904ef7b82484076e545a6c98075" target='_blank'>
              DNA remnants in red blood cells enable early detection of cancer.
              </a>
            </td>
          <td>
            Haobo Sun, Xingyun Yao, Yurong Jiao, Xiangxing Kong, Yuehua Han, Ying Li, Jianping Ge, Yanfei Cao, Hongsheng Lu, Pingli Wang, Yu Xu, Jun Li, Kefeng Ding, Xiaofei Gao
          </td>
          <td>2025-05-09</td>
          <td>Cell research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 e17570


 Background:
 Homologous recombination deficiency (HRD) is a pivotal biomarker for predicting response to PARP inhibitors in multiple cancers. This study evaluates the analytical performance of a 53-gene somatic NGS panel combined with low-pass whole genome sequencing (shallow WGS, sWGS) for HRD assessment.
 Methods:
 In the present study, 36 FFPE tissues from ovarian cancer patients were analyzed for 52 genes implicated in DNA repair pathways, along with sWGS for the evaluation of tumor genomic instability (GI). Libraries were generated with the KAPA HyperPlus Kit (Roche) and sequenced in duplicate using the Avity sequencing instrument (Element Bioscience) and the DNBSEQ-G400 NGS platform, subsequent to conversion for compatibility with MGI chemistry. HRD positivity was defined as either the presence of pathogenic BRCA1/2 mutations or high GI. Data analysis was performed using the SeqOne platform. The performance of the sWGS assay in detecting GI was initially assessed using the OncoScan CNV assay for 15 samples, as well as 4 reference materials with known HRD status. The efficacy of sWGS was also evaluated by comparing its agreement with the GI scores obtained from the validated HRD test (Myriad MyChoice) in 21 samples.
 Results:
 A 93% agreement was observed between the sWGS assay and OncoScan, demonstrating strong concordance. Additionally, the assay accurately evaluated the HRD status in all 4 reference samples with known values. The results obtained were highly similar between the two sequencing platforms. In addition, shallow NGS achieved >95% overall percentage agreement (OPA) with the validated HRD test for genomic instability score (Table 1). 5 of the 6 samples with discordant GI findings between Myriad and OncoScan aligned with the Myriad results when analyzed with sWGS. This implies that sWGS has the potential to resolve the discrepancies that were observed with OncoScan, thereby indicating its reliability as a robust alternative for HRD analysis.
 Conclusions:
 The assay demonstrated robust performance in detecting HRD-associated genomic signatures, supporting its applicability in clinical use. The reliability of the NGS assay for clinical use is guaranteed by its high concordance (95%) with the validated test, which entails the incorporation of shallow WGS for HRD analysis.


 Concordance between Myriad MyChoice and shallow WGS.






 Validated HRD Test





 HRD +


 HRD -


 PPV


 NPV


 OPA


 r





 Shallow WGS HRD
 HRD +
 13
 0
 100%
 88%
 95.24%
 0.9014


 HRD -
 1
 7
 p< .00001




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6547c6450cb8424763762c6834edc39b66aca16" target='_blank'>
              Performance of a 52-gene NGS panel combined with shallow WGS for accurate HRD analysis.
              </a>
            </td>
          <td>
            Paul Kubelac Milan, Iulian-Andrei Hotinceanu, Andreea Truican, Mioara-Maria Caldaraș, Vlad Manolescu, Iulia Monica Groza, Cristian-Ervin Gal, D. Stănculeanu, Aurelia Alexandru, P. Iorga, D. Median, Loredana Tuinea, Bogdan Gafton, Simona Volovaţ, Rareș Tiberiu Moldovan, Norina Macra, Daniela Nagy, Florinel Pop, E. Papadopoulou, G. Nasioulas
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Aneuploidy refers to chromosome number abnormality that is not an exact multiple of the haploid chromosome set. Aneuploidy has largely negative consequences in cells and organisms, manifested as so-called aneuploidy-associated stresses. A major consequence of aneuploidy is proteotoxic stress due to abnormal protein expression from imbalanced chromosome numbers. Recent advances have improved our understanding of the nature of the proteostasis imbalance caused by aneuploidy and highlighted their relevance with respect to organellar homeostasis, dosage compensation, or mechanisms employed by cells to mitigate the detrimental stress. In this review, we highlight the recent findings and outline questions to be addressed in future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbf2fd04594715e7a9fb0ca77097f24cb5000871" target='_blank'>
              The proteostasis burden of aneuploidy
              </a>
            </td>
          <td>
            P. S. Amponsah, Zuzana Storchová
          </td>
          <td>2025-04-14</td>
          <td>Biological Chemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/973aff9217cd2f6bd7d72f2139353fd552c0fa4e" target='_blank'>
              SARE: an optimized method for detection functional regulatory elements in genome
              </a>
            </td>
          <td>
            Mohammadjavad Hosseinpoor
          </td>
          <td>2025-04-16</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Colorectal cancer (CRC) has been classified into molecular subtypes using gene-level data from both bulk and single-cell RNA sequencing (scRNA-seq), leading to the development of the consensus molecular subtypes (CMS) and intrinsic CMS (iCMS). In addition to CRC studies, scRNA-seq and spatial transcriptomics characterization of human polyps, aligned with the conventional adenoma and serrated polyps, identified distinct driver mutations, cells-of-origin and iCMS traits associated with each polyp type. These gene-level transcriptional classification approaches were both strongly aligned to mutations underpinning the Vogelstein model of CRC development.
 However, when viewed using pathway-level information, the most dominant mechanistic signaling cascades in CRC were completely independent of these same mutational drivers. Therefore, to deliver a holistic landscape-view of the tumor ecosystem, we proposed that pathway-level subtyping could offer a broader understanding of non-genetic phenotypic signaling in CRC, leading to more informative stratification and therapeutic decision-making.
 Using this new approach in combination with bulk transcriptional data from n=1200+ tumors, alongside n=29, 452 scRNA-seq profiles derived from murine organoids, and n=49, 155 epithelial cells derived from n=63 human tumor samples, we identified three pathway-derived subtypes (PDS1-3), revealing previously overlooked changes in epithelial cell identities and phenotypic states. PDS2 represents tumors with stroma/immune-dense, regenerative stem-like features. PDS1 were associated with elevated MYC signaling, proliferation and canonical stem-cells with good prognosis, while PDS3 displayed a differentiated-like phenotype associated with reduced polycomb repressive complex 2 (PRC)-mediated gene repression and poorer prognosis.
 The inverse relationship between MYC/PRC targets within PDS1/PDS3 tumors supported the development of stem maturation index, which was directly related to cellular entropy and differentiation status of colonic cells allowing our tumor data to be projected onto a Waddington’s landscape. Given these associations with developmental patterns in established tumors, we next utilized the same pathway-level approach in combination with transcriptional data derived from n=2641 human polyps, where we observe that PDS traits can be established early in precancer development. Using a combination of trajectory analyzes, lineage identities and cellular phenotyping, these approaches have again revealed that, even in very early precancers, there is no association between Vogelstein-related mutational drivers and prominent phenotypic signaling cascades.
 While work is ongoing, the use of pathway-level information provides a unique insight into the dynamic nature of phenotypic evolution across early-to-late stages of colorectal precancers and tumors.


 Sudhir B. Malla, Natalie Fisher, Raheleh Amirkhah, Joanne Edwards, Owen Sansom, Philip D. Dunne. Tracking evolution of transcriptional landscapes during colorectal cancer development and progression [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 503.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c5fa7d80af5b43992f24ebd211378eba4c94dea" target='_blank'>
              Abstract 503: Tracking evolution of transcriptional landscapes during colorectal cancer development and progression
              </a>
            </td>
          <td>
            Sudhir B. Malla, Natalie Fisher, Raheleh Amirkhah, J. Edwards, Owen J. Sansom, Philip D Dunne
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Eukaryotic genomes are physically organized into chromatin, which regulates virtually every dimension of genome biology, including replication, recombination, transcription, and DNA repair. Precocious Dissociation of Sisters 5 (PDS5) protein is a key regulator of the cohesin complex, responsible for generating higher-order chromatin structures within and between chromosomes. In yeast and vertebrates, PDS5 can lead to cohesin establishment or disassembly, depending on its interaction with other proteins. In this study, we explored the diversity of PDS5 proteins across eukaryotes; HEAT-repeats forming a hook-like structure (PDS5 domain), followed by a predicted disordered region of variable lengths, are observed across the Tree of Life. Interestingly, two homolog types (A and B) with signs of varying chromatin localization emerged independently in later branches of animal and plant kingdoms, in vertebrates and vascular plants, respectively (∼400-500 MYA). We also investigated the emergence of two plant-specific features in the PDS5 proteins: 1.) An ancestral acquisition of a Tudor domain with conserved H3K4me1-binding amino acids in land plants 2.) A truncated version of PDS5 proteins that lacks essential features of PDS5 domain and is involved in the RNA-directed DNA methylation (RdDM) plant-specific pathway which emerged in seed plants. This work provides the most comprehensive study of PDS5 diversity, and suggests the emergence of specialized cohesin regulatory mechanisms that could shape distinct strategies for genome organization across eukaryotes. Short summary Here, we analyzed the diversification of PDS5 proteins; sequence, domain content, and structure, providing new observations that evidence different evolutionary trajectories of this protein across the Tree of Life. These results suggest neofunctional evolution in cohesin regulatory proteins that could shape distinct strategies for genome organization across eukaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0c1664fc04ea9ae186e4173c40cb2e49fd9d273" target='_blank'>
              Evolutionary trajectories of cohesin gatekeepers across the Tree of Life
              </a>
            </td>
          <td>
            Daniela Quiroz, Alice Pierce, Grey Monroe
          </td>
          <td>2025-05-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Toxoplasma gondii is an obligate intracellular parasite with a high replication rate that can lead to DNA replicative stress, in turn associated with the generation of DNA double-strand breaks (DSBs). Cells have two main pathways to repair DSBs: non-homologous end joining and homologous recombination repair (NHEJ and HRR respectively). RAD51 is the key recombinase in the HRR pathway. In this work, we achieved endogenous tagging of the RAD51 gene using the Auxin Inducible Degron (AID) system, to generate the clonal line RH RAD51HA-AID. Here we demonstrate that RAD51 is expressed in replicative tachyzoites and establishes damage foci. Auxin-induced knock-down (KD) affects the correct replication of tachyzoites which show loss of synchronization. The use of the RAD51 inhibitor B02 also affects parasite growth, with an IC50 of 4.8 µM. B02 produced alterations in tachyzoite replication and arrest in the S phase of the cell cycle. Additionally, B02 induced tachyzoite to bradyzoite differentiation showing small cyst-like structures. In conclusion, the HRR pathway is necessary for maintaining proper tachyzoite replication under normal growth conditions, supporting that replicative stress occurs during the cell cycle. Our findings also suggest that DNA replication stress can induce bradyzoite differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a99ccd548e29dfb7eccfd9c686305c90fbab532" target='_blank'>
              Toxoplasma gondii RAD51 recombinase is required to overcome DNA replication stress and its inactivation leads to bradyzoite differentiation
              </a>
            </td>
          <td>
            Ana M. Saldarriaga Cartagena, Ayelén Aparicio Arias, C. Cristaldi, Agustina Ganuza, M. M. Gonzalez, María M. Corvi, William J. Sullivan, Laura Vanagas, Sergio O. Angel
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46739b61c6c2d94cf2bbcd86bcaf1b74602dd7eb" target='_blank'>
              cellSTAAR: Incorporating single-cell-sequencing-based functional data to boost power in rare variant association testing of non-coding regions
              </a>
            </td>
          <td>
            E. V. Buren, Yi Zhang, Xihao Li, M. Selvaraj, Zilin Li, Hufeng Zhou, Nicholette D. Palmer, Donna K. Arnett, J. Blangero, Eric Boerwinkle, B. Cade, Jenna C. Carlson, April P. Carson, Yii-DerIda Chen, Joanne E. Curran, R. Duggirala, Myriam Fornage, Nora Franceschini, M. Graff, C. Gu, Xiuqing Guo, Jiang He, Nancy Heard-Cosa, Lifang Hou, Yi-Jen Hung, Rita R. Kalyani, Sharon L. R. Kardia, Eimear Kenny, C. Kooperberg, B. Kral, Leslie Lange, Dan Levy, Changwei Li, Simin Liu, Donald Lloyd-Jones, R. F. Loos, A. Manichaikul, Lisa W. Martin, Rasika A Mathias, R. Minster, Braxton D. Mitchell, J. Mychaleckyj, T. Naseri, Kari North, Jeffery R O'connell, James A. Perry, P. Peyser, B. Psaty, L. Raffield, Vasan S Ramachandran, S. Redline, Alexander P. Reiner, Steve Rich, Jennifer A. Smith, Brian W. Spitzer, Hua Tang, Kent D Taylor, Russell Tracy, Satupa'itea Viali, Lisa Yanek, Wei Zhao, Jerome I. Rotter, G. Peloso, Pradeep Natarajan, Xihong Lin
          </td>
          <td>2025-04-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="


 Cancer genomes are characterized by instability, with significant levels of DNA damage and mutation due to DNA repair defects. In colorectal cancer (CRC), defects in mismatch repair define the microsatellite instable (MSI) subtype, with stronger immunogenicity and better prognosis than the more common chromosomally instable (CIN) subtype. The mechanisms behind these differences are unknown, but their understanding could identify strategies to improve CIN CRC prognosis. The STING pathway detects foreign DNA in the cytosol, leading to induction of innate and adaptive immune responses. In cancer cells, STING also recognizes self cytosolic DNA (cyDNA) in the form of micronuclei or diffuse fragments generated by endogenous or treatment induced DNA damage. Our lab previously identified this pathway as a key player in the increased cytotoxic T cell infiltration characteristic of MSI CRCs. As MSI CRCs do not contain more cyDNA than CIN, we hypothesize that specific sequence or structural patterns exist within cyDNA from different CRC subtypes that lead to differential STING activation and thereby induction of overall anti-tumor immunity.



 We first aimed to examine whether cyDNA produced in MSI vs CIN CRC cells differentially induce STING and thereby anti-tumor immunity. CyDNA isolated from MC38 CRC cells with mutations in Mlh1 (modeling MSI) or Kras (CIN) were used to stimulate dendritic cells (DCs), key mediators of anti-tumor immunity. We found equal concentrations of MSI cyDNA more strongly activates STING, and thereby cytotoxic T cells, than CIN cyDNA. CyDNA from MSI CRC patient organoids showed similar results. Next, we aimed to characterize specific changes in cyDNA composition in MSI vs CIN CRC cells and examine how these alterations affect STING signalling. Sequencing identified patterns prevalent in highly stimulatory cyDNAs, including microsatellites. Stimulation with oligos and cyDNA enriched for microsatellites indicate enhanced STING and cytotoxic T cell activation through improved cGAS binding. Likewise, adoptive transfer of DCs activated by microsatellite rich DNAs induced stronger DC and T cell activation in orthotopic CIN tumors. Preliminary data suggests secondary DNA structures may influence this process. Finally, we aim to evaluate the contribution of micronuclei on STING activation in MSI vs CIN CRCs. DC stimulations with micronuclei isolated from MSI and CIN cells indicate MSI micronuclei also lead to enhanced STING activation over CIN micronuclei. We will evaluate micronuclei composition to identify underlying mechanisms.



 CRC is the second most deadly cancer worldwide, and immune recognition is crucial to treatment efficacy. This project will further our understanding of immune differences between MSI and CIN CRCs that may be used to develop therapeutic strategies, such as STING agonists, to improve CIN patient prognosis.



 Shayla Mosley, Natalie Lapa, Courtney Mowat, Kristi Baker. Genomic instability regulates STING mediated anti-tumor immunity through distinctive cytosolic DNA structures [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6127.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e18a54b51e51772ed2be11a7f9c312e5504aa79" target='_blank'>
              Abstract 6127: Genomic instability regulates STING mediated anti-tumor immunity through distinctive cytosolic DNA structures
              </a>
            </td>
          <td>
            Shayla R. Mosley, Natalie Lapa, Courtney Mowat, Kristi Baker
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Two genes are synthetically lethal if loss of function of either one of the two genes does not result in cell death, whereas loss of function of both genes together results in being detrimental to cell survival. This concept has been the basis for developing personalized, precision treatments, which can selectively damage tumor cells and minimize toxicity to normal tissues. Tumor cells often harbor mutations in genes involved in DNA repair pathways, forcing them to switch to alternative repair pathways, leading to chemotherapeutic resistance. These interactions, if targeted, could be synthetically lethal. We aimed to summarize synthetically lethal gene pairs that could be utilized to selectively target cancer cells and minimize side effects on normal tissues. The objective of this review is to study druggable synthetically lethal gene pairs for targeted cancer therapy that have been identified through various genetic screens and functional studies. Methods A systematic literature search will be conducted to extract synthetically lethal gene pairs that can be specifically targeted to cancer cells. Owing to the relatively recent research pertaining to this field, the literature search will incorporate data from 1956. The search will be conducted on PubMed, Web of Science, Embase, and Scopus. The narrative approach will guide the analysis and synthesis of the results. Discussion This review highlights scientific articles that report druggable synthetically lethal gene pairs by testing the efficacy of targeted inhibitors in clonogenic assays. These include research studies that identify synthetically lethal gene pairs detected through CRISPR screens by knocking out one or two genes within the same cell and testing the potency of inhibitors to specifically kill malignant cells. Systematic review registration 10.17605/OSF.IO/5BCW6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a6415589cc03c514ef18895752ecf172aaaab60" target='_blank'>
              Synthetic lethality in cancer: a protocol for scoping review of gene interactions from synthetic lethal screens and functional studies
              </a>
            </td>
          <td>
            Raashi Chauhan, R. Damerla, V. Dhyani
          </td>
          <td>2025-04-08</td>
          <td>Systematic Reviews</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Mutational single base substitution (SBS) and insertion/deletion (indel; ID) signatures are characteristic patterns of somatic mutations in cancer that may reflect underlying etiologic factors or pathogenetic mechanisms, 1 for example, deamination of 5 ‐ methylcytosine to thymine (SBS1), activity of the APOBEC family of cytidine deaminases (SBS2, SBS13), DNA damage caused by reactive oxygen species (ROS; SBS18), and defects of DNA replication and repair (ID1) (https://cancer.sanger. ac.uk/cosmic/signatures). In recent years, sequencing studies of B ‐ cell precursor acute lymphoblastic leukemias (BCP ALL) have identified SBS signatures of etiological/pathogenetic importance. 2 – 8 However, much remains to be ascertained, such as ID signatures and differences in signatures between major and minor clones. We performed whole genome sequencing (WGS) of 84 pediatric BCP ALLs with high hyperdiploidy (HeH; n = 23), ETV6 :: RUNX1 ( n = 23), TCF3 :: PBX1 ( n = 9), and B ‐ other ( n = 29) to identify SBS/ID signatures and to explore differences between major clones and minor subclones. In addition, WGS and RNA ‐ seq were used to detect fusion genes, single gene rearrangements, single ‐ nucleotide variants">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7030b71d700aea527ada3adaf24955243fbb85cd" target='_blank'>
              Mutational signatures and kataegis in pediatric B‐cell precursor acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Rebeqa Gunnarsson, Minjun Yang, Andrea Biloglav, Kristina B Lundin-Ström, H. Lilljebjörn, A. Castor, T. Fioretos, Linda Olsson-Arvidsson, K. Paulsson, B. Johansson
          </td>
          <td>2025-05-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="


 Efficacy of targeted therapies in metastatic colorectal cancer (mCRC) is hindered by onset of resistance, therefore novel strategies are needed. A subpopulation of drug-tolerant persister cells (DTPs) survives to targeted therapies and initiates adaptive mutability response. DTPs experience an increased DNA damage and a shift to error-prone DNA damage repair, thus promoting mutability and resistance.



 The purpose of this project is to explore key players modulating adaptive mutability to prevent DTPs’ evolution to resistance, thus preventing tumor recurrence.



 CAS9-expressing CRC cells were infected with a custom CRISPR library targeting over 500 DNA damage response (DDR) genes, divided in 12 subpopulations, and grown either for 4 weeks in normal cell culture conditions (untreated) or treated with specific oncogenic inhibitors until the development of resistance in a Time-To-Progression (TTP) assay. CAS9-expressing clones infected with empty library were grown in parallel, in absence or presence of targeted therapies, as control. Next Generation Sequencing-based analysis of sgRNA (single guides RNA) was then performed on resistant subpopulations grown in the presence of targeted therapies.



 We initially isolated CAS9-expressing clones from a microsatellite stable (MSS) CRC cell line and selected clones with growth rate and drug-sensitivity comparable to that of parental population. The selected CAS9-expressing clone was then infected with the CRISPR DDR library or empty library and, after the selection with puromycin, the selected populations were divided in 12 subpopulations, to exclude the possibility that pre-existing resistant cells might outgrow and interfere with the effect of DDR genes knock-outs (KO) on the development of drug-resistance. The subpopulations were then grown in absence or presence of oncogenic inhibitors for several weeks until the onset of resistance. Notably, while the 12 subpopulations infected with the empty library simultaneously developed resistance, the development of resistance in the DDR library-infected cells was spread over multiple weeks, thus highlighting that KO of DDR genes might either accelerate or delay the onset of resistance. sgRNA of resistant population were then analyzed by next-generation sequencing. A negative selection analysis will be applied to identify sgRNA that are lost in the resistant populations, thus targeting DDR genes whose depletion is potentially synthetical lethal with targeted therapies in mCRC.



 A DDR CRISPR screening identified novel vulnerabilities of CRC DTPs able to improve efficacy of targeted therapies and prevent tumor relapse in mCRC.



 Martina Miotto, Simona Lamba, Gaia Grasso, Giorgio Corti, Julie Bonetto, Alberto Sogari, Alberto Bardelli, Mariangela Russo. CRISPR-CAS9 library screening identifies novel vulnerabilities in CRC drug-tolerant persister cells [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4190.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50a3bde6f2ebc33e7239f55d9b0ddb6732733f31" target='_blank'>
              Abstract 4190: CRISPR-CAS9 library screening identifies novel vulnerabilities in CRC drug-tolerant persister cells
              </a>
            </td>
          <td>
            Martina Miotto, S. Lamba, G. Grasso, G. Corti, Julie Bonetto, A. Sogari, A. Bardelli, Mariangela Russo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Melanoma, a highly aggressive form of skin cancer, presents a significant clinical challenge. The identification of BRAFV600E mutations as a driver in roughly half of all melanoma cases has led to the development of targeted therapies, exemplified by Vemurafenib (Vem), which inhibits overactivated MAPK signaling. However, frequent relapses remain a major obstacle. To investigate the complex mechanisms driving drug resistance in melanoma, we developed PerturbFate, a scalable single-cell combinatorial-indexing strategy that integrates multi-modal profiling with CRISPR perturbations. This approach simultaneously captures newly synthesized and pre-existing transcriptomes, chromatin accessibility, and CRISPR sgRNA identity from individual cells. Using this platform, we profiled BRAFV600E melanoma cells (A375) after knocking down each of 143 potential drug resistance candidate genes, both with and without Vem treatment, in a pooled manner. We identified extensive phenotypic state transitions in melanoma cells under genetic and chemical perturbations, revealing that an undifferentiated state is associated with enhanced drug resistance, while a more benign, Vem-responsive melanocytic state exists at the opposite end of the phenotypic spectrum. Temporal gene regulatory network analyses revealed that AP-1 transcription factors (TFs), particularly FOSL1, pioneer other diverse TFs to drive the melanoma dedifferentiation. In contrast, SOX10 is the key determinant of the melanocytic state. We identified MAPK pathway, which primarily promotes proliferation, and Hippo/YAP signaling, which predominantly drives the undifferentiated state transition, as convergent key pathways mediating drug resistance. Their synergistic interplay, executed through the collaborative activation of downstream TFs (e.g., FOSL1, RREB1, SMADs, KLF5)-mediated gene expression, is a critical driver of Vem resistance. To elucidate the mechanisms underlying the observation that perturbation of genes encoding the Mediator Complex components (e.g., MED12, CCNC, MED13, MED19, MED15, MED24) induced autonomous dedifferentiation and/or resistance to Vem in melanoma, we established linkages between their distinct biochemical properties and heterogeneous perturbation responses. We also identified the convergence of dysregulated inflammation, potentially mediated by VEGFC and interleukins (ILs), with Hippo/YAP-mediated transcriptional responses in driving melanoma dedifferentiation and Vem resistance following perturbation of mediator complex genes. Altogether, our study introduces a novel single-cell multi-modal CRISPR perturbation platform, which we applied to comprehensively chart the responses of melanoma cells to the perturbation of all potential Vem resistance candidate genes. These findings pave the way for the identification of precise targets for combination therapies with enhanced efficacy against melanoma drug resistance.


 Zihan Xu, Ziyu Lu, Aileen Ugurbil, Abdulraouf Abdulraouf, Jianxiang Zhang, Wei Zhou, Junyue Cao. Functional characterization of genetic regulators driving melanoma drug resistance via Single-Cell high-content CRISPR screening [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771e1b2442ed827fe07315db2d2daebd9063c39b" target='_blank'>
              Abstract LB048: Functional characterization of genetic regulators driving melanoma drug resistance via Single-Cell high-content CRISPR screening
              </a>
            </td>
          <td>
            Zihan Xu, Ziyu Lu, A. Ugurbil, Abdulraouf Abdulraouf, Jianxiang Zhang, Wei Zhou, Junyue Cao
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Small cell carcinoma of the ovary, hypercalcemic type (SCCOHT) is a rare, aggressive cancer affecting young women, driven by inactivating mutations in SMARCA4, a key SWI/SNF chromatin remodeling gene. To uncover its druggable vulnerabilities, we performed a compound screen and found that SCCOHT cells and tumors were sensitive to PARP inhibitors. Paradoxically, SCCOHT displayed BRCA-deficient traits despite retaining wild-type BRCA1 expression. Elevated R-loop in SCCOHT sequestered BRCA1, limiting its availability for DNA damage repair. Proximity-dependent biotin identification revealed that wild-type SMARCA4, but not its pathogenic variants, promoted RNA polymerase II (Pol II) elongation by mediating the assembly of the polymerase-associated factor 1 (PAF1) complex. Thus, SMARCA4 loss increased Pol II pausing, resulting in elevated R-loops and BRCA1 redistribution. The suppression of BRCA1 activity sensitized SMARCA4-deficient SCCOHT cells and tumors to PARP inhibitors, which was further enhanced by the addition of a CDK9 inhibitor targeting Pol II elongation. Co-targeting PARP/CDK9 also elicited synergistic effects against other undifferentiated ovarian cancer cells with SMARCA4 loss. These findings link SMARCA4 loss to perturbed Pol II elongation and compromised DNA repair by BRCA1, providing a therapeutic opportunity to target SCCOHT and other SWI/SNF-deficient ovarian cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/350e8c98aa41fa4d992b18cc3ee2eb0de4ea4277" target='_blank'>
              SMARCA4 loss increases RNA Polymerase II pausing and elevates R-loops to inhibit BRCA1-mediated repair in ovarian cancer.
              </a>
            </td>
          <td>
            Xianbing Zhu, Z. Fu, Giulio Aceto, Jonathan R St-Germain, Kexin Liu, A. Arabzadeh, Yuxuan Qi, Yibo Xue, L. Witkowski, Elise Graulich, Jutta Steinberger, Selim Misirlioglu, Nicklas Bassani, Racim Sansal, Amber Yasmeen, Geneviève Morin, Jingjie Guo, A. Monast, V. Pilon, Alica Valachová, K. Pavlakis, Lili Fu, Walter H. Gotlieb, W. McCluggage, D. Huntsman, Alexander J R Bishop, Douglas A Levine, Morag Park, Yemin Wang, B. Raught, W. Foulkes, Sidong Huang
          </td>
          <td>2025-05-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 As we age, cells accumulate somatic mutations, undergo epigenetic alterations, and experience telomere shortening. In older individuals, normal tissues are often overtaken by expanding clones of cells that have acquired mutations promoting proliferation. Whether epigenetic changes can also provide selective advantage and induce clonal expansion in ageing is less clear.
 Here, we sequenced 719 whole genomes and methylomes of single-cell colonies derived from HSCs of three healthy individuals and three breast cancer patients. Using somatic mutations, we built lineage trees of the HSCs for each individual. To establish whether loss or gain of methylation was heritable, we developed a method that accurately infers, for each CpG site, its zygotic methylation state as well as the number and timing of ancestral methylation changes that explain the observed methylation state of each somatic cell.
 We analysed ∼24 million CpG sites per individual and find that methylation states are, in general, stably heritable over decades of life. We find that during embryogenesis, when the embryo comprises a few thousand cells, developmental cells lay down unique methylation haplotypes, spanning hundreds of base pairs in size, that are stably inherited by descendants. This phase of rewiring the methylome has striking properties: (1) thousands of regions genome-wide show heterogeneity of methylation profiles laid down during this phase; (2) these regions are often conserved across different individuals; (3) it precedes gastrulation, so that the same changes are seen in all germ layers; (4) the process happens quickly, completing within a few generations of cell division; (5) it coincides with establishment of inactive X chromosome methylation in females; and (6) the methylation affects many key regulatory regions of the genome.
 To assess whether embryonic methylation rewiring can contribute to cancer development, we studied women with breast cancer who had BRCA1 promoter methylation. The tumour samples did not have germline or somatic mutation of BRCA1 (or other DNA repair genes), but had BRCA1 methylation along with the typical mutational signatures of homologous recombination deficiency. Intriguingly, we found that the same methylation change was present heterozygously in a subset of HSCs; that these HSCs all derived from one embryonic cell that pre-dated gastrulation; and that the somatic mutations defining this lineage of HSCs were also present clonally in the breast cancer. Thus, aberrant methylation of the BRCA1 promoter in embryogenesis was transmitted across germ layers, in blood and breast epithelium, and (with loss-of-heterozygosity) drove homologous recombination deficiency in a breast clone that transformed to cancer 4-6 decades later.
 By comparing normal healthy individuals of different ages with cancer patients, we have been able to unravel the process of normal ageing from cancer development at an unprecedented resolution.


 Lori D. Kregar, Nicholas Williams, Joe Lee, Michael Spencer-Chapman, Oleksii Nikolaienko, Emily Mitchell, Liv B. Gansmo, Per E. Lønning, Elisa Laurenti, Lucy Yates, Stian Knappskog, Jyoti Nangalia, Peter Campbell. Rewiring of methylation during embryogenesis can seed cancer decades later [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB308.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993964715fda888d384887c1b0e033eb189929c" target='_blank'>
              Abstract LB308: Rewiring of methylation during embryogenesis can seed cancer decades later
              </a>
            </td>
          <td>
            Lori D. Kregar, N. Williams, Joe Lee, Michael Spencer-Chapman, O. Nikolaienko, E. Mitchell, Liv B Gansmo, P. Lønning, Elisa Laurenti, Lucy Yates, S. Knappskog, J. Nangalia, Peter J Campbell
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 Mutations in somatic cells are inflicted by both extrinsic and intrinsic sources and contribute over time to cancer. Tobacco smoke contains chemical carcinogens that have been causatively implicated with cancers of the lung and head & neck. APOBEC family DNA cytosine deaminases have emerged as endogenous sources of mutation in cancer, with hallmark mutational signatures (SBS2/SBS13) that often co-occur in tumors of tobacco smokers with an equally diagnostic mutational signature (SBS4). Here we challenge the dogma that mutational processes are thought to occur independently and with additive impact by showing that 4-nitroquinoline 1-oxide (NQO), a model carcinogen for tobacco exposure, sensitizes cells to APOBEC3B (A3B) mutagenesis and leads to synergistic increases in both SBS2 mutation loads and oral carcinomas in vivo. NQO-exposed/A3B-expressing mice exhibit twice as many head & neck lesions as carcinogen-exposed wildtype animals. This increase in carcinogenesis is accompanied by a synergistic increase in mutations from APOBEC signature SBS2, but not from NQO signature SBS4. Interestingly, a large proportion of A3B-catalyzed SBS2 mutations occurs as strand-coordinated pairs within 32 nucleotides of each other in transcribed regions, suggesting a mechanism in which removal of NQO-DNA adducts by nucleotide excision repair exposes short single-stranded DNA tracts to enzymatic deamination. These highly enriched pairs of APOBEC signature mutations are termed didyma (Greek for twins) and are mechanistically distinct from other types of clustered mutation (omikli and kataegis). Computational analyses of lung and head & neck tumor genomes show that both APOBEC mutagenesis and didyma are elevated in cancers from smokers compared to non-smokers. APOBEC signature mutations and didyma are also elevated in normal lung tissues in smokers prior to cancer initiation. Collectively, these results indicate that DNA adducting mutagens in tobacco smoke can amplify DNA damage and mutagenesis by endogenous APOBEC enzymes and, more broadly, suggest that mutational mechanisms can interact synergistically in both cancer initiation and promotion.


 Reuben S. Harris. Tobacco smoke carcinogens exacerbate APOBEC mutagenesis and carcinogenesis [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB476.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63aefe6d3e777332427bd46910b51b4d2635b3ae" target='_blank'>
              Abstract LB476: Tobacco smoke carcinogens exacerbate APOBEC mutagenesis and carcinogenesis
              </a>
            </td>
          <td>
            R. Harris
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="


 Genomic DNA is continually exposed to endogenous and exogenous stressors, leading to DNA damage and genomic instability. The base excision repair (BER) mechanism is crucial for maintaining genetic integrity by repairing damaged nucleobases. BER begins when DNA glycosylases cleave the N-glycosidic bond to remove oxidized pyrimidines. DNA glycosylases can be monofunctional or bifunctional. In this study, we explore the bifunctional NTHL1 variant in inflammation-associated colorectal cancer (CRC). NTHL1 tumor syndrome is linked to increased lifetime risks of CRC, breast cancer, and colorectal polyposis. The rare polymorphism rs3087468 (D239Y) results in a loss of glycosylase activity and acts as a dominant-negative mutation.



 We hypothesize that the NTHL1-D239Y variant increases CRC susceptibility under chronic inflammation in the AOM/DSS cancer initiation model.



 Male and female C57BL/6 mice (6-8 weeks old) with wild-type (NTHL1WT/WT), heterozygous (NTHL1WT/D239Y), or homozygous (NTHL1D239Y/D239Y) NTHL1 variants were used. In the first experiment, mice were injected with 7.5 mg/kg azoxymethane (AOM), followed by three cycles of 2.5% dextran sulfate sodium (DSS) (5 days DSS alternating with 14 days of normal water). The second experiment used DSS alone. We monitored body weight, collected fecal samples for 16S rRNA sequencing, and measured colon length and polyp number/size at euthanasia. Histology was assessed for inflammation and tumor development. Blood samples were analyzed for micronuclei formation, and immunohistochemistry (IHC) staining was used for tissue proliferation.



 In the AOM/DSS model, male NTHL1WT/D239Y and NTHL1D239Y/D239Y mice had significantly more polyps compared to NTHL1WT/WT controls. Female NTHL1D239Y/D239Y mice had more polyps than NTHL1WT/D239Y females. In the DSS-only experiment, male NTHL1WT/D239Y mice developed a significant number of polyps compared to wild-type males, while female NTHL1D239Y/D239Y mice developed more polyps than NTHL1WT/WT females. No significant differences in polyp size were observed in the DSS-only experiment. Tumors extending into the proximal colon were seen only in female NTHL1D239Y/D239Y mice. Male NTHL1WT/D239Y mice developed high-grade adenomas in the distal colon compared to wild-type males.Micronucleated erythrocytes were significantly higher in male NTHL1D239Y/D239Y mice compared to controls, but this was not observed in the DSS-only group. No changes in colitis score or colon length were observed. Genotype or AOM/DSS treatment did not affect microbial alpha diversity, but treatment-dependent changes in microbial composition were noted based on sex and genotype. IHC data indicated elevated KI67 expression in NTHL1D239Y/D239Y and NTHL1WT/D239Y mice following AOM/DSS treatment.



 The NTHL1-D239Y variant promotes malignancy only under chronic inflammation and AOM/DSS treatment. This variant confers a complex, sex- and zygosity-dependent susceptibility to inflammation-associated CRC, with females showing increased tumor development in the proximal colon and males displaying enhanced tumor size and progression.



 Tarek Michael Masannat, Fayez K. Ghishan, Pawel R. Kiela, Joann B. Sweasy. NTHL1-D239Y variant enhances tumorigenesis in colorectal cancer through cancer initiation and chronic inflammation [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB304.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e93f87b45a480b89d7a05bed3b25376e4e282650" target='_blank'>
              Abstract LB304: NTHL1-D239Y variant enhances tumorigenesis in colorectal cancer through cancer initiation and chronic inflammation
              </a>
            </td>
          <td>
            Tarek Masannat, F. Ghishan, P. Kiela, J. Sweasy
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb7defb0dc0b2dcfb00df9b73a0a0c72cd7d43d4" target='_blank'>
              Transposable elements as instructors of the immune system.
              </a>
            </td>
          <td>
            Lisa Schmidleithner, Philipp Stüve, Markus Feuerer
          </td>
          <td>2025-04-29</td>
          <td>Nature reviews. Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Understanding the vast noncoding cancer genome requires cutting-edge, high-resolution, and accessible strategies. Artificial intelligence (AI) is revolutionizing cancer research, enabling advanced models to analyze genome regulation. This review examines illustrative examples of noncoding mutations in cancer, focusing on both key regulatory elements and risk-associated variants that remain poorly understood, and compares key AI models developed over the last decade for identifying functional noncoding variants, predicting gene expression impacts, and uncovering cancer-associated mutations. The discussion of the goals, data requirements, features, and outcomes of the models offers practical insights to help cancer researchers integrate these technologies into their work, regardless of computational expertise.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59192b80596e6fd10ef23d12789a787453b253b7" target='_blank'>
              Illuminating the Noncoding Genome in Cancer using Artificial Intelligence.
              </a>
            </td>
          <td>
            M. D. M. Álvarez-Torres, Xi Fu, Raúl Rabadán
          </td>
          <td>2025-04-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Cancer cells achieve replicative immortality through telomere maintenance mechanisms (TMMs), primarily via telomerase activation or alternative lengthening of telomeres (ALT). Sarcomas frequently employ the ALT pathway, which traditionally correlates with adverse clinical outcomes. However, chondrosarcomas represent a unique context where the role and prognostic significance of ALT remain largely unexplored. Methods We performed comprehensive analyses of single-cell RNA-sequencing data from patients with chondrosarcoma and integrated this with 90 bulk RNA-seq datasets. This approach enabled detailed characterization of TMM at single-cell resolution, identification of ALT-specific signatures, and evaluation of the tumor microenvironment in chondrosarcomas. Results Patients with ALT-like chondrosarcomas exhibited significantly improved survival compared to those with non-ALT-like chondrosarcomas. Analysis of the tumor immune microenvironment revealed distinct metabolic and immune landscapes between the ALT-like and non-ALT-like groups. Single-cell analysis showed that high-entropy stem-like cells in high-grade chondrosarcomas predominantly adopted telomerase activation over the ALT pathway as their TMM. Additionally, we identified a 100-gene signature that reliably distinguishes ALT-like chondrosarcomas, providing a robust molecular marker for classification and prognosis. Conclusions Our study reveals ALT-like state as a marker of favorable prognosis in chondrosarcomas—contrasting with its typically adverse implications in other sarcomas. We establish a robust 100-gene signature that reliably identifies ALT-like chondrosarcomas and characterize their distinct immune microenvironment profile. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/881425c49c7bf310ad569e7f1a7bdeb0d0fff86f" target='_blank'>
              Alternative lengthening of telomeres confers favorable prognosis in chondrosarcomas
              </a>
            </td>
          <td>
            Ji-Yong Sung, Jin-Hong Kim, Yi-Jun Kim
          </td>
          <td>2025-05-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer development requires an evolutionary transformation from mammalian cells fully regulated by and integrated into multicellular tissue to cancer cells that, as single cell protists, are individually subject to Darwinian selection. Through genetic and epigenetic mechanisms of inheritance, the evolving cancer phenotype must acquire independence from host controls, downregulate differentiated functions that benefit the host but not individual cells, and generate phenotypic traits that increase fitness in the context of the selection forces within the local microenvironment. Here, we investigate this evolutionary transition in breast (BRCA) and lung (LUAD, without EGFR, KRAS or BRAF driver mutations) adenocarcinomas using bulk mutation and expression data from the TCGA database. We define evolution selection for genes and molecular pathways based on 1) changes in gene expression compared to normal tissue, and 2) significantly larger or smaller observed mutation rates compared to those expected based on the gene size. We find BRCA and LUAD disable different genes and gene pathways associated with tissue-specific signaling and differentiated functions but promote common molecular pathways associated with cell cycle, cell-cell interactions, cytoskeleton, voltage gated ion channels, and microenvironmental niche construction. Thus, tissue-specific parallel evolution in early cancer development is followed by convergence to a common cancer phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ced41c3cd6dbbc3e387b68d444b592a5918ecc9" target='_blank'>
              Parallel and convergent dynamics in the evolution of primary breast and lung adenocarcinomas
              </a>
            </td>
          <td>
            Robert A. Gatenby, Jamie K Teer, Kenneth Y. Tsai, Joel S. Brown
          </td>
          <td>2025-05-21</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite significant progress in its prevention, diagnosis, and treatment, head and neck cancer (HNC) remains a major global health issue due to its multifactorial pathogenesis. Indeed, HNCs have been found to be associated with different environmental and lifestyle factors, as well as with infection with oncogenic viruses. To date, seven viruses are recognized for their tumorigenic properties and have been proposed as implicated in HNC development, including Merkel Cell Polyomavirus (MCPyV). MCPyV is well recognized as the major etiological agent of Merkel cell carcinoma (MCC), a rare but rapidly metastasizing skin neoplasm. Specifically, in almost 80% of MCC cases, viral genome integration occurs, and a truncated form of Large T Antigen (tLT) is expressed. Although MCC is a rare cancer, MCPyV is a ubiquitous virus, widely distributed among the human population. Therefore, a plausible role of the virus has been proposed, even for other tumors. The current review provides an overview of the available data describing the presence of MCPyV in non-MCC tumors, such as HNCs, with the aim of elucidating the potential contribution of MCPyV to oral cancer. Understanding the role of viral infections in the etiology of cancer opens up the opportunity for developing preventive measures and targeted therapies that effectively address HNC progression while reducing treatment-related side effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8cb154aa103ba21fb13b74cfd8a73cf9551de0a" target='_blank'>
              Merkel Cell Polyomavirus (MCPyV) and Its Possible Role in Head and Neck Cancers
              </a>
            </td>
          <td>
            S. Passerini, Sara Messina, U. Moens, V. Pietropaolo
          </td>
          <td>2025-05-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="
 SMCHD1 is a component of the structural maintenance of chromosomes (SMC) protein family involved in epigenetic gene silencing of the X chromosome and other autosomal chromosome regions. SMCHD1 encodes an ATPase domain and binds chromatin at both N- and C-term domains to facilitate long-range chromatin interactions. However, the exact mechanism by which it elicits transcriptional silencing remains unknown. We have identified a selective dependency on SMCHD1 in cell lines exhibiting a gene expression signature characterized by high expression of multiple cancer testis antigen (CTA) genes. This CTA signature is observed in multiple cancers with >60% prevalence in Chronic Myeloid Leukemia. Importantly, SMCHD1’s effect on cell growth was independent of its ATPase activity, and a knock-in degron tag phenocopied SMCHD1 KO indicating that the proliferative advantage offered by SMCHD1 might be due to structural SMCHD1 function unrelated with its enzymatic ATPase role. Finally, SMCHD1 depletion led to the upregulation of multiple genes, including a cluster of CTA genes, in sensitive cell lines with the CTA-positive signature, while CTA-negative cell lines remained unaffected. These data confirm that SMCHD1 is a potential therapeutic target for patients with elevated CTA expression, and provides insight to what therapeutic modality would be most effective.


 Victor Quereda, Youyou Zhang, Diana Klimaszewska, Stefano Tonin, Theodore Groth, Yanhua Rao, Michael T. McCabe, Benjamin Schwartz, Markus Queisser, Rachel Buxton, Thomas J. Berrodin, Simon Peace. SMCHD1 is a synthetic lethal strategy for cancers with a cancer-testis antigen gene expression signature [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4294.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0f99bba46d4c798400f4f104ff216c0c2a15d1" target='_blank'>
              Abstract 4294: SMCHD1 is a synthetic lethal strategy for cancers with a cancer-testis antigen gene expression signature
              </a>
            </td>
          <td>
            Victor Quereda, Youyou Zhang, Diana Klimaszewska, Stefano Tonin, Theodore Groth, Yanhua Rao, Michael T. McCabe, Benjamin Schwartz, Markus Queisser, Rachel Buxton, Tom Berrodin, Simon Peace
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Gastric cancer (GC) is a leading cause of cancer-related deaths globally. Emerging evidence implicates aberrant DNA polymerase beta (POLB) expression in genomic instability and tumorigenesis. Previously, we demonstrated that the human gastric cancer-associated variant of POLB (Leu22Pro or L22P) lacks dRP lyase function in vitro and induces replication associated genomic instability and cellular transformation. In addition , in vivo mouse model experiment has shown that L22P mutation of POLB , leads to inefficient BER and exacerbate genomic instability and carcinogenesis .



 We conducted an in-silico analysis of GC datasets from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) to assess POLB expression levels and their association with the tumor immune microenvironment (TIME). Survival analyses were performed using Kaplan-Meier estimates, and the TIMER algorithm was employed to evaluate immune cell infiltration. In parallel, we utilized a dRP lyase-deficient POLB variant (Leu22Pro, L22P) in vivo model to investigate the impact of dRP lyase deficient POLB on genomic stability and inflammatory signaling. Also, Mitotic dysfunction and cytosolic DNA-mediated inflammatory responses were assessed, with and without poly(ADP-ribose) polymerase 1 (PARP1) inhibition.



 Our analysis revealed that POLB is overexpressed in approximately 32% of GC cases, significantly correlating with the intestinal subtype of GC. POLB overexpression was associated with reduced expression of innate immune signaling genes and lower infiltration of immune cells, indicating a suppressed immune microenvironment. Conversely, tumors with high POLB expression exhibited increased mutation frequency and microsatellite instability (MSI). Notably, patients with POLB-overexpressing tumors and high immune score demonstrated improved overall survival. ROC analysis suggested that POLB overexpression holds potential as a prognostic marker for GC. In in vivo model, the L22P POLB variant increased mitotic dysfunction-associated genomic instability, leading to a cytosolic DNA-mediated inflammatory response. PARP1 inhibition exacerbated chromosomal instability and enhanced the inflammatory response in these cells.



 Aberrant POLB expression in GC is associated with a dysregulated TIME and may serve as a prognostic indicator. Defective POLB function contributes to genomic instability and activates cytosolic DNA-mediated inflammatory signaling. These findings suggest that deregulated POLB in tumor likely provide a potential opportunity to patient stratification for better immune-based therapies in GC.



 Aashirwad Shahi. Aberrant DNA polymerase beta expression is associated with dysregulated tumor immune microenvironment and its prognostic value in gastric cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB386.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21340650146a334354ebce820c4c26cb9ab3ba1b" target='_blank'>
              Abstract LB386: Aberrant DNA polymerase beta expression is associated with dysregulated tumor immune microenvironment and its prognostic value in gastric cancer
              </a>
            </td>
          <td>
            Aashirwad Shahi
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In addition to accumulating mutations that may drive tumor growth, cancer cells also show broad changes in DNA methylation patterns genomewide. For example, a colorectal cancer sample may have fewer than 10 driver mutations, fewer than 100 passenger mutations, but may have nearly 800 differentially methylated regions. In contrast to the traditional method of using sequencing to find these scarce mutations in the genome, probing for these relatively abundant aberrantly methylated regions in the genome can be a powerful alternative approach for the detection of cancer.
 However, assessing DNA methylation via methyl sequencing is complicated by artifacts introduced during the traditional NGS library prep process. In vivo, CpG sites in mammals are typically methylated on both strands of the double stranded DNA; however, we observe that the majority of read fragments display some level of hemimethylation (only one strand displays methyl modifications), and roughly 10% of all reads are hemimethylated across the read. Furthermore, we observe that the fraction of methylated bases gradually decreases over the course of a sequencing read; the methylation state of DNA should not be expected to be impacted by the cycle number of a sequencing run. These artifacts are collectively known as M-bias and can be mitigated computationally, but these methods are an incomplete solution and may sacrifice valuable base coverage.
 We have devised a strategy to correct M-bias biochemically, using a custom formulated end repair mix. We show that this novel library prep workflow reduces M-bias by ∼50% in diverse ctDNA assays. This reduction in M-bias results in more accurate methylation status calls and reduces the number of reads needed to confidently cover a CpG region of interest. Finally, we show that biochemical M-bias mitigation methods lower healthy hypomethylation background signal by ∼10x. These changes increase robustness of hypomethylation biomarker quantitation and utility in ctDNA detection assays.


 James Yu, Peter George, Jason Close, Andy Eang, Neil Peterman, Yu Sun, Nafisa Bulsara, Eileen Dimalanta, Alexander Robertson, Yexun Wang, Nicole Lambert. Biochemical mitigation of technical artifacts improves performance of a ctDNA methyl sequencing assay [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4568.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b042bad9ffec7e55a0de94ab786a57c1a323f2d" target='_blank'>
              Abstract 4568: Biochemical mitigation of technical artifacts improves performance of a ctDNA methyl sequencing assay
              </a>
            </td>
          <td>
            James Yu, Peter George, J. Close, Andy Eang, N. Peterman, Yu Sun, Nafisa Bulsara, Eileen Dimalanta, Alex Robertson, Yexun Wang, Nicole Lambert
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18c7e36e3b876db6b83aaa4bdaf8e2434f722c8a" target='_blank'>
              Oncogenic fusions converge on shared mechanisms in initiating astroblastoma.
              </a>
            </td>
          <td>
            Yixing Shi, Qianqian Sun, Fuchuan Jia, Xiangyu Xie, Xiangyu Zhou, Rong Guo, Yangfan Zeng, Shanshan Chen, Zhenzhen Guo, Wenli Sun, Tong Guo, Yu Xia, Wenlong Li, Li Zhang, Wei Shi, Yang Yu
          </td>
          <td>2025-05-14</td>
          <td>Nature</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 3046


 Background:
 Prior work has shown that detection of circulating tumor DNA (ctDNA) at time of diagnosis of leiomyosarcoma (LMS) is associated with lower likelihood of objective response and patients with detectable ctDNA after two cycles of chemotherapy have worse survival. However, because ctDNA exists at much lower concentrations in plasma compared to cell-free DNA of non-tumor origin, the sensitivity of these prognostic measures to tumor signals remains unclear. With increasing evidence of ctDNA fragments being shorter than the background non-tumor cell-free DNA, we sought to test whether restricting our analysis to smaller fragment sizes would improve detection of ctDNA in LMS.
 Methods:
 Plasma was serially collected from patients with LMS undergoing chemotherapy. Cell-free DNA extracted from these samples was profiled by ultra-low-passage whole-genome sequencing (ULPWGS). Copy number alterations were identified and used to detect ctDNA using the ichorCNA algorithm before and after restricting the dataset to fragments of 90-150bp (short). We compared detectability of ctDNA via ichorCNA between analyses using all sequencing data and those using data restricted to short fragments.
 Results:
 From 28 patients, 126 plasma samples were profiled. The median fragment length of cell-free DNA was 240bp (IQR 142-349) for patients with LMS, compared to 307bp (IQR 171-465, p < 0.001) in samples collected from healthy controls. Short fragments made up 19.48% of LMS libraries at diagnosis when ctDNA levels were highest, 15.27% of all LMS samples, and 12.96% of libraries from healthy controls. While ctDNA was detectable by ULPWGS in 17% of all samples, detection increased to 40% when analyzed using only short cell-free DNA fragments (p < 0.0001). The proportion of diagnostic samples with detectable ctDNA was nominally higher when analyzed by short fragments (39% vs. 64% with 90-150bp size restriction, p = 0.1078, n = 28) and was significantly higher in samples collected after two cycles of chemotherapy (5% vs. 40% with 90-150bp size restriction, p = 0.0197, n = 20). Increases in ctDNA detectability with fragment size restriction were also observed in each of localized and metastatic subgroups (metastatic: 16% without restriction, 38% with restriction, n = 97, p = 0.0012; localized: 17% vs. 45%, n = 39, p = 0.04).
 Conclusions:
 Our results demonstrate that detection of ctDNA is improved by analyzing short fragments of cell-free DNA in samples collected from patients with LMS. These findings represent a potential to increase the sensitivity of an affordable, low-coverage liquid biopsy assay and may enhance the prognostic value of ctDNA detection in these patients. Further study of the association between ctDNA detection and outcome is needed to fully validate the impact of fragment size restricted analysis of cell-free DNA samples in patients with LMS and is currently ongoing in a prospective study.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8ceaaf3e27c10098ba0b8ae1de9b437c08db2f3" target='_blank'>
              Improved detection of circulating tumor DNA in patients with leiomyosarcoma with fragment size restriction.
              </a>
            </td>
          <td>
            N. Ruzgar, K. Klega, Cora A. Ricker, Mohammad Tanhaemami, Amanda M. Shafer, Denise K. Reinke, Brittany L. Siontis, S. Schuetze, B. Crompton
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background Mitophagy, essential for cellular homeostasis, is involved in eliminating damaged mitochondria and is associated with cancer progression and chemoresistance. The specific impact of mitophagy on microsatellite instability-high (MSI-H) colorectal cancer (CRC) is still under investigation. Ubiquitination, a post-translational modification, is essential for controlling protein stability, localization, and function. This study identifies USP14, a deubiquitinating enzyme, as a key regulator of mitophagy in MSI-H CRC. Methods A deubiquitinating enzyme (DUBs) siRNA library screening identified USP14 as a key regulator of mitophagy. Tissue samples from patients were analyzed using immunohistochemistry and Western blot. USP14 knockdown cell lines were generated using lentiviral transfection. Protein interactions between USP14 and BAG4 were confirmed by co-immunoprecipitation, while quantitative PCR was used to measure gene expression. Mitochondrial proteins were extracted to analyze mitophagy, and flow cytometry was used to assess apoptosis. Finally, a mouse xenograft model was employed to study USP14’s role in tumor growth and oxaliplatin sensitivity. Results Screening reveals that USP14 inhibits mitophagy and CRC (MSI-H) show high USP14 expression which correlates with poor prognosis. Functional analyses reveal that knocking down USP14 reduces tumor growth, and increases sensitivity to oxaliplatin. Mechanically, USP14 inhibits mitophagy by K48-deubiquitinating and stabilizing BAG4 at K403, which prevents the recruitment of Parkin to damaged mitochondria. The significant clinical relevance of USP14, BAG4, and PRKN are proved in tumor tissues. Conclusions The study highlights the USP14/BAG4/PRKN axis as a critical pathway in CRC (MSI-H), suggesting that targeting USP14 could inhibit tumor progression and improve chemotherapeutic outcomes. These findings underscore the importance of ubiquitination and mitophagy in cancer biology, indicating a potential therapeutic target for MSI-H CRC. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01182-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbc4b33209665ea2d4b62576cf3d6c685a1afc91" target='_blank'>
              USP14 inhibits mitophagy and promotes tumorigenesis and chemosensitivity through deubiquitinating BAG4 in microsatellite instability-high colorectal cancer
              </a>
            </td>
          <td>
            Zhiyong Wang, Cheng Yu, Gengchen Xie, K. Tao, Zhijie Yin, Qing Lv
          </td>
          <td>2025-05-02</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract The three-dimensional (3D) organization of the genome is crucial for gene regulation, with disruptions linked to various diseases. High-throughput Chromosome Conformation Capture (Hi-C) and related technologies have advanced our understanding of 3D genome organization by mapping interactions between distal genomic regions. However, capturing enhancer–promoter interactions at high resolution remains challenging due to the high sequencing depth required. We introduce pC-SAC (probabilistically Constrained Self-Avoiding Chromatin), a novel computational method for producing accurate high-resolution Hi-C matrices from low-resolution data. pC-SAC uses adaptive importance sampling with sequential Monte Carlo to generate ensembles of 3D chromatin chains that satisfy physical constraints derived from low-resolution Hi-C data. Our method achieves over 95% accuracy in reconstructing high-resolution chromatin maps and identifies novel interactions enriched with candidate cis-regulatory elements (cCREs) and expression quantitative trait loci (eQTLs). Benchmarking against state-of-the-art deep learning models demonstrates pC-SAC’s performance in both short- and long-range interaction reconstruction. pC-SAC offers a cost-effective solution for enhancing the resolution of Hi-C data, thus enabling deeper insights into 3D genome organization and its role in gene regulation and disease. Our tool can be found at https://github.com/G2Lab/pCSAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bc34f801c6a85a73e83d49ad30fa8158531f9d" target='_blank'>
              pC-SAC: A method for high-resolution 3D genome reconstruction from low-resolution Hi-C data
              </a>
            </td>
          <td>
            J. C. Angel, Narjis El Amraoui, Gamze Gürsoy
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma undergoes a complex and dynamic evolution involving genetic and epigenetic changes. Understanding the mechanisms underlying this evolution is vital for the development of efficient therapeutic strategies. Although treatment resistance is associated with intratumoral heterogeneity in glioblastoma, it remains uncertain whether hypometabolic and hypermetabolic lesions observed through clinical positron emission tomography (PET) imaging are influenced by spatial intratumoral genomic evolution.


METHODS
In this study, we precisely isolated autologous hypometabolic and hypermetabolic lesions from glioblastoma using advanced neurosurgical and brain tumor imaging technologies, followed by comprehensive whole-genome, exome, transcriptome, and imaging analyses.


RESULTS
Our findings unveil that hypermetabolic lesions, originating from hypometabolic lesions, exhibit strategic focal amplifications and deletions, and heightened APOBEC3 activity. Furthermore, we identify dipeptidase 1 as a novel vascular endothelial tip marker for hypermetabolic lesions in glioblastoma, facilitating angiogenesis and tumor metabolism by regulating transporter activities.


CONCLUSIONS
Hypermetabolic lesions are associated with a higher frequency of genomic abnormalities and dipeptidase 1 emerges as a novel diagnostic and prognostic vascular marker for hypermetabolic lesions. This study underscores a spatial genomic evolution with diagnostic implications and elucidates challenges and opportunities crucial for the development of novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c12841ca7664e84f80538f2ab403344da4bec905" target='_blank'>
              Multiomic profiling of glioblastoma metabolic lesions reveals complex intratumoral genomic evolution and dipeptidase-1-driven vascular proliferation.
              </a>
            </td>
          <td>
            A. Anand, J. K. Petersen, L. Andersen, M. Burton, Clara Rosa Levina Oudenaarden, M. Larsen, Philip Ahle Erichsen, C. B. Pedersen, F. Poulsen, Peter Grupe, T. Kruse, M. Thomassen, B. Kristensen
          </td>
          <td>2025-05-04</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14a450aef8a5e62f014140a56818ef21010ef060" target='_blank'>
              Genome complexity, not ploidy, dictates long-read variant-calling accuracy
              </a>
            </td>
          <td>
            Yoshinori Fukasawa
          </td>
          <td>2025-05-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 In the 20 years since the completion of the Human Genome Project, cancer biology remains rooted in the belief that the human genome encodes ∼20,000 protein-coding genes. This assumption has been fundamentally challenged by the discovery of microproteins, which are encoded across the genome by thousands of non-canonical open reading frames (ORF). Whether these microproteins reflect a previously untapped source of cancer genes may therefore have a profound impact on our understanding of cancer biology. To address this question, we have interrogated the prevalence and impact of non-canonical ORF translation across multiple cancer types. Using ribosome profiling, ORF cDNA libraries, expression profiling and CRISPR/Cas9 screens, we find that thousands of non-canonical ORFs are translated in cancer cells; nearly half (46%) of tested non-canonical ORFs may make a detectable microprotein; nearly 30% induced gene expression changes in cancer cell lines; and 10% induced a cancer cell viability defect when knocked-out. High-resolution CRISPR saturation mutagenesis identified a subset of non-canonical ORFs with exquisite genetic selectivity in about one-third of cases, providing high-confidence genetic evidence for a causal role of the non-canonical ORF in cancer cell viability. To position specific candidates in the context of cancer biology, we have characterized two cancer-associated microproteins encoded by non-canonical ORFs. In breast cancer, we described G029442 - renamed glycine-rich extracellular protein-1 (GREP1) - as a secreted protein that is a preferential genetic dependency in this disease through its regulation of the oncogenic cytokine, GDF15. Moreover, GREP1 expression is prognostic for poor patient outcomes. In childhood medulloblastoma, we found that MYC-driven medulloblastoma cells preferentially upregulated an upstream open reading frame in the ASNSD1 5’ untranslated region, termed ASNSD1-uORF. ASNSD1 uORF was a top genetic vulnerability in multiple models of medulloblastoma and it functions as a component of the prefoldin-like complex, which post-translationally regulates nascent proteins. Together, ASNSD1-uORF and the prefoldin-like complex impacted cancer cell cycle through the proteomic regulation of proteins involved in mitosis and chromosome spindle complexes. In summary, our work supports a generalizable principle that ncORFs commonly encode biologically-active microproteins in diverse malignancies. Ongoing investigation of ncORFs therefore represents a new frontier in cancer research with the potential to define the next generation of therapeutic target genes.


 John R. Prensner. Mapping the functional microproteome in cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY04-01.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5d3048f7fc1ab2dec5d8c000f11dda761b0959e" target='_blank'>
              Abstract SY04-01: Mapping the functional microproteome in cancer
              </a>
            </td>
          <td>
            John R. Prensner
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Objective: To undertake a literature review on the development of cervical cancer, emphasizing epidemiological data, human papillomavirus (HPV) infection in carcinogenesis, and the role of the XRCC1 gene in DNA repair, in order to provide valuable information for the medical and scientific community. The goal is to provide a valuable resource for clinical practice. Bibliographic Review: Cervical cancer ranks as the fourth most common malignancy worldwide. Despite known environmental, behavioral, and genetic risk factors, their management remains inadequate. Persistent infection with oncogenic HPV is a well-established risk factor; however, not all infected individuals develop cervical cancer, suggesting the involvement of additional mechanisms. Among these, polymorphisms in the XRCC1 gene have gained attention. XRCC1 encodes a scaffold protein essential for DNA repair coordination and cell cycle regulation. Three main XRCC1 polymorphisms—Arg194Trp, Arg280His, and Arg399Gln—are associated with impaired DNA repair, genomic instability, and increased cervical cancer susceptibility. Final Considerations: A deeper understanding of the genetic and cellular mechanisms underlying cervical cancer may facilitate the development of targeted medical interventions for women, contributing to improved quality of life reduced mortality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/607876d22f013a85473f817b1c9f164dd5e62623" target='_blank'>
              Cervical cancer and XRCC1 gene polymorphisms
              </a>
            </td>
          <td>
            Andréia Michelle Alves Cunha de Alcântara, Ivan Barbosa Barros, Israel Santos da Silva, Karla Roberta Lemos Maul de Souza, Ivan Barbosa Barros, M. Maia, Paulo Roberto Eleutério de Souza
          </td>
          <td>2025-05-30</td>
          <td>Revista Eletrônica Acervo Saúde</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Recent molecular biological studies have revealed that chromosomal instability (CIN) promotes the malignant phenotype of tumors. CIN causes chromosomal segregation errors, leading to micronuclei (MN) formation. Ruptured MNs activate the cGAS-STING pathway, an intracellular DNA sensor, and induce metastasis and invasion. Histological confirmation of MNs is thought to be uncertain and not reproducible. On the other hand, cGAS is localized in a subset of MN by immunohistochemistry, so we attempted to apply cGAS-positive MN as an indicator of CIN and tumor aggressiveness in liposarcoma, which is reported as one of the tumors with the highest frequency of CIN among various malignant tumors.



 Formalin-fixed paraffin-embedded biopsy and surgical specimens of liposarcoma of four histological types, namely well-differentiated (WLPS) (n=18), dedifferentiated (DLPS) (n=32), myxoid (MLPS) (n=6), and pleomorphic liposarcoma (PLPS) (n=7) were examined. In addition, benign lipomatous tumors (n=17) were also tested. Ten cases of non-lipomatous high-grade sarcoma were used as controls. Because DLPS results from the dedifferentiation of WLPS, it comprises differentiated (W) and dedifferentiated (D) components. Immunohistochemical staining for cGAS was performed, and the number of cGAS-positive micronuclei in 500 tumor cells of each tumor was counted. For DLPS, dedifferentiated cells (D cells) in the D component and lipogenic cells (L cells) and non-lipogenic cells (N cells) in the W component were evaluated separately.



 cGAS immunostain highlighted a subset of micronuclei, as well as mitotic figures, and some small punctate structures within the main nuclei. The average number of cGAS-positive MNs was 1.2 for benign lipomatous tumor (n=17), 1.0 for myxoid liposarcoma (MLPS, n=6), 34.1 for pleomorphic liposarcoma (PLPS, n=7), 23.2 for D cells in (DLPS, n=32), 3.1 for L cells and 3.6 for N cells in WLPS / W-component (n=43), 24.2 for referenced sarcomas, respectively. The number of cGAS-positive micronuclei was significantly higher in aggressive sarcomas (PLPS and D components of DLPS) than in translocation-associated sarcoma (MLPS), WLPS / W-component, and benign tumors. Although the D-component and W-component / WLPS share a similar complex karyotype, the number of c-GAS positive MNs was significantly higher in D cells rather than L cells (t-test, p=0.0001) and N cells (t-test, p=0.0002).



 MNs can be visualized by cGAS immunostain. Significant increases in cGAS-positive MNs were observed in aggressive liposarcomas with complex karyotypes. Furthermore, increased cGAS-positive MNs are associated with malignant progression in DLPS. cGAS-positive MNs may be applicable as an objective indicator of tumor aggressiveness and CIN.



 Kiyota Kato, Toru Motoi, Yu Naito, Masumi Ogawa, Chihiro Onda, Hideo Arai, Shin-ichiro Horiguchi, Toshihide Hirai, Tsunekazu Hishima. Application of cGAS-immunopositive micronuclei as an indicator of tumor aggressiveness in liposarcoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3345.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/879969a6652e4d8acc441456b7bcb05f2b8a551c" target='_blank'>
              Abstract 3345: Application of cGAS-immunopositive micronuclei as an indicator of tumor aggressiveness in liposarcoma
              </a>
            </td>
          <td>
            Kiyota Kato, Toru Motoi, Yu Naito, Masumi Ogawa, Chihiro Onda, Hideo Arai, Shin-ichiro Horiguchi, Toshihide Hirai, T. Hishima
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 3577


 Background:
 HRD is linked to sensitivity to platinum-based chemotherapy and poly (ADP-ribose) polymerase inhibitors across various tumor types. However, the presence of HRD, driver events (such as pathogenic mutations in HR genes or other genomic alterations), and its clinical relevance in mCRC remain underexplored.
 Methods:
 We performed the VHIO-300 test, an ISO15189 accredited custom NGS panel profiling over 450 genes (including HR-related genes:
 ATM, BRCA1, BRCA2, BRIP1, CHEK2 or PALB2
 ) on 356 Stage IV mCRC patients (corresponding to 247 primary colorectal and 109 to metastatic samples) enrolled in the Vall d’Hebron Institute of Oncology’s Molecular Prescreening Program from June 2021 to December 2024. All samples had a tumor cellularity > 40% as per pathologist evaluation
 .
 An HR score (sHR) based on genome-wide copy number alterations (CNA) and loss of heterozygosity (LOH) patterns is generated. After cross-validation with Myriad MyChoice, sHR ≥ 56 was established based on a cohort of ovarian tumors and used to identify HRD in mCRC tumor samples.
 Results:
 HRD prevalence was 3.4% in our mCRC cohort (12/356), but, much higher in metastatic lesions, (6.4%, 7/109) than in primary samples (2.03%, 5/247) (p = 0.05). In fact, the median sHR between primary (21) vs. metastases (30) in CRC was significantly different (p < 0.01). Regarding HR gene status, HR-mutated samples were not significantly within the HRD group (p = 0.27) and only 6.5% (2/31) were HRD. Noteworthy,
 BRCA2
 exhibits a frameshift deletion in a homopolymer stretch, that is a frequent hotspot in microsatellite instable (MSI) tumors, but this event was not found to be associated with HRD. In fact, all HRD tumors (n = 12) were microsatellite stable (MSS). Other frequent alterations in mCRC were studied and
 BRAF
 mutations were found to be present in 42% of HRD tumors (p < 0.01). Inversely, HRD was rare in
 KRAS
 -mutated samples (0.7%; 1/140) and, in fact, highly correlated with non-HRD status (p = 0.02), especially the G12 mutation (p < 0.01). Interestingly, CNA profiles also revealed a strong association between the
 BCL2L1
 loci gain and HRD (p < 0.01). Clinically, HRD was not significantly associated with prognostic value nor clinical benefit to oxaliplatin-based combinations. However, the limited sample size and heterogeneous treatment lines restricted robust statistical analysis.
 Conclusions:
 This study identified a small, yet significant subset of mCRC that displays HRD. sHR and HRD rates were higher in metastatic lesions vs primary tumors, indicating HR scarring could be accumulating over time in some mCRC patients. No clear association between pathogenic HR gene mutations and HRD were found, suggesting the involvement of alternative molecular mechanisms in this process. Frequent co-occurring events, such as
 BRAF
 mutations or
 BCL2L1
 gains could be drivers in CRC HRD, and shape, eventually, new therapeutic options for these patients in the metastatic setting.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14cea3ee627505991df2a2a9cb9438dafa79fb0" target='_blank'>
              Drivers of homologous recombination deficiency (HRD) in metastatic colorectal cancer (mCRC).
              </a>
            </td>
          <td>
            Paula Romero Lozano, Javier Ros Montañá, Eduardo García-Galea, Maria Vila-Casadesús, R. Comas, Ágatha Martín, Jenifer González-Zorelle, Paolo Nuciforo, H. G. Pálmer, J. Tabernero, E. Élez, Ana Vivancos-Prellezo
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="
 Colitis-associated colorectal cancer (CAC) is a severe complication of inflammatory bowel disease, particularly ulcerative colitis. Despite its clinical significance, the spatiotemporal dynamics of neoplastic progression in CAC remain poorly understood. This study aimed to elucidate the evolutionary trajectory of CAC using spatial transcriptomics. We employed spatial transcriptomics to analyze four tissue samples from three patients with ulcerative colitis at various stages of neoplastic progression. Whole-slide spatial transcriptomics RNA sequencing was performed, enabling simultaneous analysis of gene expression and tissue architecture. Integrated spatial transcriptomic data revealed recurring epithelial programs along the progression of copy number variation (CNV). Our analysis using CNV score and hematoxylin and eosin staining image revealed progressive changes in gene expression patterns as tissue transitioned from non-tumor to pre-tumor and finally to tumor states. This progression was quantified using beta coefficient values, allowing us to track gradual alterations in gene expression across the disease continuum. We identified a core set of 412 genes that exhibited consistent changes in at least three of the analyzed samples. Among these, 349 genes showed a progressive decrease in expression, while 63 genes, including CLDN1, E2F1, and RECQL4, demonstrated a gradual increase in expression as the tissue advanced towards a tumorous state. These upregulated genes pointed to potential oncogenic drivers or compensatory mechanisms activated during tumor development. Pathway analysis revealed that downregulated genes were enriched in processes such as KRAS signaling, hypoxia, and inflammatory response, while upregulated genes were enriched in the MYC targets pathway. This study provides high-resolution mapping of evolutionary trajectory of CRC, revealing key molecular events and spatial features associated with neoplastic progression. Our findings enhance the understanding of CAC pathogenesis and highlight potential targets for early intervention and personalized treatment strategies, contributing significantly to the field and potentially informing future therapeutic approaches.


 Yoonho Lee, Yongjae Kim, Minseok Lee, Do Kyung Yoon, Jiwon Baek, Sung Wook Hwang, Jong Lyul Lee, Sang Hyoung Park, Dong-Hoon Yang, Jihun Kim, Suk-Kyun Yang, Buhm Han, Kyuyoung Song, Yong Sik Yoon, Byong Duk Ye, Ho-Su Lee. Spatiotemporal mapping of colitis-associated colorectal cancer evolution. [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2076.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a91bf888a9dbd6ca367ec1fcc781f2777b99ddae" target='_blank'>
              Abstract 2076: Spatiotemporal mapping of colitis-associated colorectal cancer evolution
              </a>
            </td>
          <td>
            Yoonho Lee, Yongjae Kim, Minseok Lee, D. K. Yoon, J. Baek, S. Hwang, Jong Lyul Lee, S. Park, Dong-Hoon Yang, Jihun Kim, Suk-Kyun Yang, B. Han, K. Song, Yong Sik Yoon, B. Ye, Ho-Su Lee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="
 e15561


 Background:
 Colorectal cancer (CRC), the third most prevalent cancer globally, remains a major cause of cancer-related mortality. Genomic and epigenomic plasticity during CRC progression, particularly metastasis, poses significant challenges to effective treatment. Patient-derived organoids (PDOs) reflect the heterogeneous genetic and morphological composition of cancer cells, enabling ex vivo disease modeling. This study investigates the epigenomic makeup of human primary and metastatic CRC to advance precision epigenetic therapies.
 Methods:
 We analyzed 45 organoid models, including 40 CRC organoids (from primary tumors and liver/lung metastases) and 5 generated from healthy colonic tissue. Using integrative epigenomic profiling, including chromatin accessibility, chromatin state mapping (via six histone modifications), and 3D genome conformation, we constructed a comprehensive CRC epigenomic landscape.
 Results:
 Our analysis of CRC patient-derived organoids revealed a core set of accessible genomic regions shared across all organoids, irrespective of tumor location or metastatic status. This conserved chromatin signature highlights common regulatory pathways in CRC during disease progression. Unsupervised clustering based on epigenomic features revealed four distinct CRC subtypes, encompassing both primary and metastatic models. Notably, organoids derived from liver and lung metastases did not cluster by metastatic site, underscoring epigenomic convergence. Subtype D comprised 57.1% (8/14) of primary tumors, while subtypes B and C contained 47.3% (9/19) of liver metastases and 42.8% (3/7) of lung metastases, respectively. KRAS mutation status further stratified subtypes, with wild-type KRAS observed exclusively in subtypes A and B, while KRAS-mutant variants were enriched in subtypes C (6/7) and D (4/10). Histone modification patterns, defined by six marks (H3K4me1, H3K4me3, H3K27me3, H3K27ac, H3K36me3, and H3K9me3) revealed subtype-specific regulatory chromatin landscapes. Three-dimensional chromatin interactions were mapped, and genome-wide interaction contacts are used to identify various chromatin structures that may influence metastatic capacity.
 Conclusions:
 This study highlights distinct and shared epigenomic signatures in CRC PDOs, offering novel insights into tumor progression and metastasis. Integrative chromatin analysis provides a framework for identifying epigenetic biomarkers and epigenome targeting strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66760aad28585c2f41842514d0645e6e13f51968" target='_blank'>
              Deciphering colorectal cancer progression through integrative epigenomic analysis for precision therapeutics.
              </a>
            </td>
          <td>
            Ghazaleh Tavallaee, Amin Nooranikhojasteh, Elias Orouji
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Telomeres, nucleotide repeats and a six-protein complex called shelterin at chromosome ends, are critical for genomic integrity. Rare heterozygous pathogenic or likely pathogenic (P/LP) germline variants in shelterin complex genes have been associated with long telomeres and risk of certain cancers, including melanoma, lymphoproliferative diseases, glioma, papillary thyroid cancer, and sarcoma. Traditional phenotype-first studies of P/LP germline variants and disease are often limited by ascertainment of the most severely affected individuals. Genome-first approaches can reduce this bias by first assessing germline variants and then evaluating the associated diseases. Since most studies, to date, have focused on variants in POT1 and cancer, we sought to estimate associations between cancer and rare P/LP variants in the five less well studied shelterin genes (ACD, TERF1, TERF2, TERF2IP, and TINF2) in the UK Biobank (UKB) to understand their role in cancer etiology at the population level. Exome sequencing data on 469, 578 participants were retrieved from the UKB DNA Nexus platform. Variants with minor allele frequency <1%, exonic or splice site, genotype quality >30, depth >20, and allele fraction >0.3 were retained. Variant classification used Automated Germline Variant Pathogenicity (PMID: 38426335). Logistic regression analyses adjusted for age, sex, alcohol use, smoking, and BMI were used to calculate odds ratios (OR) and 95% confidence intervals (CI). There were 177 participants (97 female, 80 male, median age = 57.0 years, range = 40.0 - 69.0) with 59 unique P/LP germline variants (30 ACD, 9 TERF2, and 20 TINF2). Strikingly, there were no rare variants in TERF1 or TERF2IP. The population prevalence of individuals with P/LP germline variants in these five shelterin genes is estimated at 1:2, 514 (174/437, 486 unrelated individuals, 95% CI 2, 189-2, 953). Forty-eight individuals with P/LP germline variants were diagnosed with at least one cancer with a median age at first cancer of 59.3 years (range 19.4 - 80.6). Overall, P/LP variants were significantly associated with melanoma (OR 2.93, 95% CI 1.20-7.15, p = 0.018) and cancers of the urinary system (OR 2.78, 1.02-7.57, p = 0.045). By gene, P/LP variants were associated with cancers of the oral cavity/pharynx (ACD, OR 4.78, 95% CI 1.18-19.4, p = 0.030), soft tissue including heart (ACD, OR 8.90, 95% CI 1.24-63.9, p = 0.030), colon excluding the rectum (TERF2, OR 8.44 , 95%CI 1.05-68.0, p= 0.045), female genital system (TERF2, OR 12.98, 95% CI 1.43-117.8, p = 0.023), and melanoma (TINF2, OR 7.63, 2.35-24.75, p = 0.001).Rare P/LP germline variants in ACD, TERF2, and TINF2 may be associated with increased risk of certain cancers in the UKB. Analyses applying the genome-first approach other population-based cohorts and assessment of telomere length are underway to better understand the extent to which variation in shelterin genes contributes to cancer etiology.


 Akanksha Nagarkar, Hasset Nurelegne, Jung Kim, Kelvin C. de Andrade, Sharon A. Savage. Genome-first approach identifies shelterin complex gene variants and associated cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4891.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5281b0b530161da0728f26a1aa6d170b1c6d1d79" target='_blank'>
              Abstract 4891: Genome-first approach identifies shelterin complex gene variants and associated cancers
              </a>
            </td>
          <td>
            Akanksha Nagarkar, Hasset T Nurelegne, Jung Kim, K. D. de Andrade, Sharon A. Savage
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Synthetic lethality, a phenomenon in which alterations in two or more genes together induce cell death while changes in either gene alone do not, provides a promising approach for cancer treatment. Indeed, various inhibitors targeting synthetic lethal interactions are approved by the Food and Drug Administration or in clinical trials. Seeking to nominate additional synthetic lethal targets, we analyzed large-scale CRISPR knockout screening data and identified Pelota mRNA surveillance and ribosome rescue factor (PELO) as a promising therapeutic target for two independent and frequently observed molecular subtypes of cancer: biallelic deletion of chromosomal region 9p21.3 (9p21.3-/-) and microsatellite instability-high (MSI-H).
 Homozygous deletion of chromosomal region 9p21.3 is one of the most frequently observed somatic copy number alterations in human cancers, occurring in approximately 15% of all cancers. Many 9p21.3-/- cancers are associated with poor clinical outcomes including subsets of glioblastoma, mesothelioma, urothelial, pancreatic, esophageal, and non-small cell lung cancers. The primary driver of 9p21.3-/- is thought to be loss of the tumor suppressor cyclin-dependent kinase inhibitor 2A. Additionally, studies have linked the loss of the interferon gene cluster on 9p21.3 to immune evasion and primary resistance to immune checkpoint inhibitors. Extensive deletions of chromosomal regions can provide opportunities for cancer treatment. For instance, PRMT5 and MAT2A were found to be promising targets in 9p21.3-/- cancers due to synthetic lethal interactions with deletions of the 9p21.3 gene methylthioadenosine phosphorylase. While clinical trials are underway examining PRMT5 and MAT2A inhibitors, the frequency and poor outcomes of many 9p21.3−/− cancers underscores the urgent need for additional therapies to treat this diverse group of cancers. Using large-scale functional genomic datasets, we identified PELO as the top preferential dependency in 9p21.3-/- cell lines.
 We also observed that some cell lines with intact 9p21.3 (9p21.3+) were dependent on PELO for survival. Examining other genomic features, we found that these cells were characterized as MSI-H, a hypermutable state observed in subsets of colon, endometrial, gastric, and ovarian cancers. Indeed, when we compared 9p21.3+/MSI-H and 9p21.3+/microsatellite stable (MSS) cell lines, we found that PELO scored as a strong preferential dependency in MSI-H cell lines, second only to the previously described synthetic lethal target Werner helicase (WRN).
 To validate these findings, we interrogated the viability effects of PELO knockdown across a panel of cell lines representing 9p21.3+/MSS, 9p21.3-/-/MSS, or 9p21.3+/MSI-H. CRISPR interference (CRISPRi)-mediated PELO knockdown with three distinct guide RNAs impaired the viability of 9p21.3-/-/MSS cell lines and 9p21.3+/MSI-H cell lines, but not 9p21.3+/MSS cell lines. We also sought to validate PELO dependency in tumor organoid models for which pre-existing CRISPR data were unavailable, and therefore, we could only predict their dependence on PELO based on the two identified biomarkers. We demonstrated that 9p21.3-/- or MSI-H tumor organoids were preferentially dependent on PELO as compared to 9p21.3+/MSS models. We also sought to evaluate the effect of PELO knockdown in tumor maintenance in vivo. Leveraging a doxycycline-inducible CRISPRi system to suppress PELO expression, we showed that PELO knockdown significantly shrank 9p21.3-/- tumors grown on the flanks of nude mice.
 We next pursued the culprit lesion in 9p21.3-/- cancers that conferred PELO dependency. Hypothesizing that the loss of a 9p21.3 gene confers PELO dependency, we performed a focused loss-of-function CRISPR/Cas12a screen targeting 9p21.3 genes in the context of PELO knockdown or control conditions. We identified the mRNA surveillance gene FOCAD as a 9p21.3 gene whose knockout lead to preferential impairment in cells with PELO depletion. This led us to hypothesize that FOCAD deletions were conferring PELO dependency. In focused validation, we showed that FOCAD deletion sensitizes cell lines to PELO knockdown. Furthermore, we discovered that restoring FOCAD rescued 9p21.3-/- cells from PELO depletion. Together, these data demonstrated FOCAD loss was both necessary and sufficient for PELO dependency in the 9p21.3-/- context.
 Next, we sought to determine the mechanistic basis for PELO dependency in MSI-H cancers. We hypothesized that a mutation associated with MSI-H sensitizes cells to PELO loss. We observed that deletions in the splicing acceptor site at intron 29 of the superkiller complex (SKIc) member tetratricopeptide repeat domain 37 (TTC37) was the most correlated microsatellite mutation with PELO dependency. Thymidine deletions in this splicing acceptor site were predictive of decreased TTC37 protein levels in cell lines. These data led us to hypothesize that functional loss of TTC37 is the culprit lesion that renders MSI-H cancers dependent on PELO for survival. Evaluating this hypothesis, we found that TTC37 knockout sensitized cells to PELO depletion. We also showed that exogenous expression of TTC37, but not control, cDNA rescued the viability of MSI-H cell lines from PELO knockdown. Taken together, these data demonstrated that functional TTC37 impairment is sufficient and necessary for PELO dependency in the MSI-H setting and could serve as a refined predictive biomarker for PELO dependency.
 We next sought to investigate how FOCAD loss and TTC37 mutations converge to confer PELO dependency. TTC37 interacts with Ski2-like RNA helicase (SKIV2L) and WD Repeat Containing Protein 61 (WDR61) to form the SKIc, which promotes the 3’-5’ exosome degradation of mRNA from stalled ribosomes. We examined the Cancer Cell Line Encyclopedia proteomics dataset and demonstrated positive correlations between FOCAD, TTC37, and SKIV2L. Consistent with these datasets, we observed decreased TTC37 and SKIV2L protein levels in 9p21.3-/- cell lines and MSI-H cell lines. Based on these results, we asked if FOCAD was required and/or sufficient to maintain TTC37 and SKIV2L protein stability. We expressed FOCAD cDNA in 9p21.3-/- cells and observed increased TTC37 and SKIV2L protein levels. Furthermore, FOCAD knockout reduced SKIV2L and TTC37 protein levels in 9p21.3+/MSS cells. Together, these data support FOCAD as a critical regulator of SKIV2L and TTC37 stability. We hypothesized that FOCAD deletions and TTC37 mutations were capturing SKIc loss, thereby increasing reliance on PELO for survival. We reasoned that if functional loss of the SKIc was responsible for PELO dependency, SKIV2L deletion would also confer increased PELO dependency. We knocked SKIV2L out and observed that loss of SKIV2L sensitized cells to PELO loss. These data support our hypothesis that loss of SKIc function confers increased dependence on PELO for survival.
 To assess how SKIc-deficient cancer cells respond to PELO suppression, we performed gene expressing profiling. We found that SKIc-deficient cells had a robust transcriptional response with gene set enrichment analyses demonstrating upregulation of the unfolded protein response, a signaling network responding to aggregated unfolded or misfolded proteins. We validated these findings by demonstrating that PELO depletion preferentially upregulated C/ebp homologous protein (CHOP) expression and induced X-BOX binding protein (XPB1) splicing, markers of UPR activation, in the SKIc-deficient setting.
 Together, our observations reveal that MSI-H associated mutations and large 9p21.3 deletions involving TTC37 and FOCAD, respectively, independently impair the SKIc and confer a synthetic lethal relationship with PELO. Since MSI-H and large 9p21.3 deletions are frequently observed in patients, a PELO-based therapeutic could have broad implications for clinical oncology.


 Patricia Borck, Isabella Boyle, Kristina Jankovic, Nolan Bick, Kyla Foster, Anthony Lau, Lucy Parker-Burns, Daniel Lubicki, Ashir Borah, Nicholas Lofaso, Sohani Das Sharma, Riya V. Kishen, Joshua Dempster, Francisca Vazquez, Edmond M. Chan. Exploiting dysregulated ribosomal homeostasis in chromosome 9p21.3 deleted cancers and microsatellite unstable cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY14-03.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aee6939711394df481ecca9ad3d57b55fd487283" target='_blank'>
              Abstract SY14-03: Exploiting dysregulated ribosomal homeostasis in chromosome 9p21.3 deleted cancers and microsatellite unstable cancers
              </a>
            </td>
          <td>
            Patricia Borck, Isabella Boyle, Kristina Jankovic, Nolan R. Bick, Kyla Foster, Anthony Lau, Lucy Parker-Burns, Daniel A Lubicki, Ashir A. Borah, Nicholas J Lofaso, Sohani Das Sharma, Riya V Kishen, Joshua M Dempster, Francisca Vazquez, Edmond M Chan
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The 12th 3R + 3C international symposium focused on cutting‐edge research into the molecular mechanisms and regulatory systems of DNA replication, repair, and recombination (3R) as well as those of chromatin dynamics, chromosome architecture, and the cell cycle (3C). It also covered pioneering research into how these processes control cell growth, cell homeostasis, differentiation, development, and aging, in addition to how they contribute to diseases such as cancer, chromosomal abnormalities, and evolution of organisms. In terms of methodology, the symposium highlighted new trends in single‐molecule/single‐cell analysis, cryo‐electron microscopy analysis, kinetic analysis of higher‐order protein complexes, informatic analysis of genome dynamics, and new mathematical and theoretical analyses. Held in Fukuoka City center from November 18 to 22, 2024, this symposium attracted about 250 participants, including approximately 150 from Japan and nearly 100 from overseas. To foster mutual understanding and exchange between different fields, all the oral presentations took place in a single conference hall throughout the symposium. This format facilitated active and in‐depth discussions among participants, including young researchers, graduate students, and postdoctoral fellows.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/041f509af47278c9ea1863c279f50106d1d6e43b" target='_blank'>
              The 12th 3R + 3C International Symposium: A Meeting for Research Into DNA Replication, Repair, and Recombination, as Well as Chromatin, Chromosomes, and the Cell Cycle
              </a>
            </td>
          <td>
            Tsutomu Katayama, Masatoshi Fujita, Tatsuro S Takahashi
          </td>
          <td>2025-04-10</td>
          <td>Genes to Cells</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="This review focuses on the experimental methods and technologies of cytogenomics and how they can be combined in the process of chromosomic diagnostics and research. It is stressed that no cytogenomic methods can be comprehensive on their own. The strengths and weaknesses of each method have to be considered. This is especially important in a time where the main stream of human genetics diagnostics is actively proclaiming that high throughput methods are able to replace all other established tests.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4829c289229594dea05af81329ec9848b137cc21" target='_blank'>
              The Need for a Concert of Cytogenomic Methods in Chromosomic Research and Diagnostics
              </a>
            </td>
          <td>
            Yiping Wang, Thomas Liehr
          </td>
          <td>2025-04-29</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Solid cancers are complex 'ecosystems' comprised of diverse cell types and extracellular molecules, where heterotypic interactions significantly influence disease etiology and therapeutic response. However, our current understanding of tumor microenvironments (TMEs) remains incomplete, hindering the development and implementation of novel TME-targeted drugs. To maximize cancer therapeutic development we require a systems-level understanding of the malignant, stromal, and immune states that define the tumor and determine treatment response. In their recent study, Liu and colleagues took a new approach to resolving the complexity of stromal heterogeneity. They leveraged extensive single-cell spatial multi-omics datasets across various cancer types and platforms to identify four conserved spatial Cancer-Associated Fibroblast (CAF) subtypes, classified by their spatial organization and cellular neighborhoods. Their work expands upon prior efforts to develop a CAF taxonomy, which primarily relied on single-cell RNA-Sequencing (scRNA-Seq) and yielded a multitude of classification systems. This study advances our understanding of CAF biology by establishing a link between spatial context and CAF identity across diverse tumor types. Departing from recent single-cell transcriptomics studies that employed a marker-based approach for sub-state identification, Liu and colleagues conducted de novo discovery of CAF subtypes using spatial neighborhood information alone. By positioning spatial organization as the defining axis of CAF heterogeneity, this research redefines CAF classification and provides a new framework for exploring the rules governing tumor ecosystems and developing novel ecosystem-based therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/940389ee957141f2833e0d9de53711c0ead3a30c" target='_blank'>
              Pan-Cancer Spatial Profiling Reveals Conserved Subtypes and Niches of Cancer-Associated Fibroblasts.
              </a>
            </td>
          <td>
            Hani Jieun Kim, Travis Ruan, Alex Swarbrick
          </td>
          <td>2025-05-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Cancer heterogeneity poses a significant medical challenge, necessitating therapies tailored not only to individual patients but also to the unique cancer clones that exist within diverse microenvironmental contexts. Despite this diversity, nearly all cancer cells share a reliance on telomeres to maintain genomic stability. The telomerase (hTERT) gene is hyperactive in many cancers and has been identified as a potential target for treatment. While this has led to the development of multiple telomerase-targeting approaches, disappointingly, none have been approved in clinical applications yet. To circumvent this concern, our team has applied a genetic approach called synthetic dosage lethality (SDL), to exploit hTERT overexpression to identify potential targets to treat cancer. SDL is a genetic concept, where a normally non-lethal gene inactivation kills cells only in the context of overexpression of another gene like hTERT. Our laboratories have used lentiviral-based pooled CRISPR/Cas9 and pooled shRNA-screening platforms to systematically query the genome and recently identified several SDL partners of hTERT. Through extensive validation using pooled in vivo CRISPR/Cas9 screens across multiple cell lines, tumor xenografts, and patient-derived organoids, we identified RMT1, an RNA 2'-O-methyltransferase, as a prominent SDL partner of hTERT. In collaboration with the AtomWise and Thoth Biosimulations companies, we have developed novel inhibitors for RMT1 that preferentially suppress hTERT-overexpressing cell lines growth and viability. We have also performed a large-scale screen of FDA-approved small molecule inhibitors with the goal of repurposing these drugs to target hTERT-overexpressing cancers. Subsequently, we plan to apply these therapies in combination to amplify the efficiency of treatment against cancers, yielding preclinical evidence to support the development of novel cancer therapies. Since hTERT activity is essential in all cancer cells, our findings introduce a transformative approach to overcoming cancer cell heterogeneity by targeting the SDL interactions of hTERT. This strategy also capitalizes on the ubiquitous overexpression of hTERT in all cancer types, enabling the selective eradication of hTERT-dependent malignancies.


 Vincent Maranda, Frederick S. Vizeacoumar, Yue Zhang, Liliia Kyrylenko, Andrew Freywald, Franco J. Vizeacoumar. Developing combination therapies for telomerase-overexpressing cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 5412.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41aaa58f8e3f0c715544a400d743c926b588c9a" target='_blank'>
              Abstract 5412: Developing combination therapies for telomerase-overexpressing cancers
              </a>
            </td>
          <td>
            Vincent Maranda, F. Vizeacoumar, Yue Zhang, L. Kyrylenko, A. Freywald, Franco J Vizeacoumar
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Understanding the role of noncoding regions of the human genome in cancer requires innovative, high-resolution approaches. Artificial intelligence (AI) is transforming cancer research by providing advanced tools to identify, analyze, and interpret cancer genomes. This study aims to 1) discuss how noncoding mutations in key regulatory elements and other regions can contribute to cancer development, and 2) evaluate the leading AI models used for analyzing noncoding variants, focusing on their evolution and application in cancer research. We present examples of mutations in promoters, e.g. TERT [Huang et al., Science 2013] APC [Snow et al., Clin Gen 2015]), enhancers (a SNP in gliomas [Yanchus et al., Science 2022]), and super-enhancer hypermutation in B-cell lymphomas [Bal et al., Nature 2022; Leeman-Neil et al., Nat Gen 2023]. These cases demonstrate the functional impact of noncoding variants in tumorigenesis. However, for many noncoding regions and mutations, our understanding of the underlying mechanisms, binding motifs, and their roles in gene regulation remains limited. Here, AI can play a crucial role by analyzing large genomic datasets to uncover hidden patterns and interactions. This study compares AI models developed over the past decade, including DeepSEA[ Zhou et al., Nat Meth 2015], DeepBind [Alipanahi et al., Com Bio 2015], DeltaSVM [Lee et al., Nat Gen 2015], Basset [Kelley et al., Gen Res 2015], CADD [Rentzsch et al., Nuc Ac Res 2018], Expecto [Zhou et al., Nat Gen 2018], DeepC [Schwessinger et al., Nat Meth 2020], Enformer [Avsec et al., Nat Meth 2021], and GET [Fu et al., Nature 2024 (in press)]. We assess and scored key factors such as model complexity, data integration, cell-type specificity, interpretability, and usability within cancer research. Older models (2015-2018) score lower in complexity than recent ones (2018-2024), reflecting simpler architectures due to earlier computational limitations. In contrast, newer models leverage advanced computational power and deep learning, enhancing their ability to capture complex biological patterns and improve data integration and cell-type specificity. However, usability in practical research does not correlate with publication year, as user-friendly interfaces and comprehensive documentation play crucial roles in application, regardless of model complexity. In conclusion, AI has proven instrumental in elucidating the role of noncoding mutations in cancer, revealing intricate regulatory patterns and previously unknown mechanisms. Over 11, 000 publications cite the models discussed, underscoring their scientific impact. Such efforts will support predictive models that can generalize across diverse contexts, advancing genomic research and its implications for cancer biology. Our work highlights the potential of these AI models to assist researchers, particularly those less familiar with computational techniques, in advancing cancer genomics.


 Maria del Mar Alvarez-Torres, Xi Fu, Raul Rabadan. Illuminating the dark genome in cancer using artificial intelligence [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7429.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c16d5ebf6f1d5ea4dbccbf5a0178380f13bbf886" target='_blank'>
              Abstract 7429: Illuminating the dark genome in cancer using artificial intelligence
              </a>
            </td>
          <td>
            M. D. M. Álvarez-Torres, Xi Fu, Raul Rabadan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Merkel Cell Polyomavirus (MCPyV) is recognized as the major aetiological agent of Merkel Cell Carcinoma (MCC), an aggressive skin tumor. MCPyV-mediated oncogenesis is strictly dependent on viral integration and the expression of a truncated form of the Large T Antigen (LT). Moreover, like other oncogenic DNA viruses, MCPyV may interfere with the DNA damage response (DDR) machinery, thus promoting genomic instability and tumorigenesis. Therefore, the objective of this study was to characterize MCPyV infection in 7 MCC patients and to elucidate the plausible role of the virus in the DDR pathway. MCPyV DNA was detected in 3/7 MCC patients and, as expected, viral integration and LT truncation were observed in virus-positive MCCs, along with the expression of early genes only. Over-expression of DDR genes such as ATM, ATR and their downstream kinases Chk1 and Chk2 was reported in MCPyV-positive MCCs supporting the potential role of the virus in interfering with DDR. Our findings support the established viral aetiology of MCC, and describe, for the first time, an over-expression of DDR components in MCPyV-positive MCC, laying the basis for future studies aimed at investigating the contribution of this pathway to MCPyV-mediated carcinogenesis and exploring the plausible clinical implications of host DDR factors for the treatment of MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cee01380ef93ba0651f961c5653ef76df058128" target='_blank'>
              Investigation of mRNA expression levels of DNA damage response genes in Merkel Cell Polyomavirus-positive Merkel Cell Carcinoma: a pilot study
              </a>
            </td>
          <td>
            S. Passerini, M. Fracella, A. Ferlosio, U. Moens, C. Scagnolari, Guido Antonelli, Marco Ciotti, V. Pietropaolo
          </td>
          <td>2025-05-21</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6116cf0972c0f5ba1621a1614ba9f7a9a3add325" target='_blank'>
              The SOX gene superfamily in oncogenesis: unraveling links to ncRNAs, key pathways, chemoresistance, and gene editing approaches.
              </a>
            </td>
          <td>
            Ammar Yasir Ahmed, Subasini Uthirapathy, Enwa Felix Oghenemaro, R. M, Rohit Kumawat, Yasser Fakri Mustafa, Muthena Kariem, A. Kadhim, Shilpa Sharma, M. R. Kumar
          </td>
          <td>2025-05-20</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 The retinoblastoma protein 1 (RB1), encoded by RB1 gene, plays a critical role in regulating the G1 to S-phase transition of the cell cycle through its interaction with the E2F family of transcription factors. In addition, RB1 has been shown to directly regulate DNA damage response and apoptosis. Loss of RB1 function, predominately via gene deletion, occurs frequently in various human cancers, with the highest prevalence in small cell lung cancer (almost ubiquitous) and basal-like breast cancer (up to 72%). RB1 loss is often associated with aggressive phenotypes, especially in cancers that harbor concurrent TP53 mutations. In TP53 mutated cancer cells, further loss of RB1 can drive uncontrolled entry into S phase, leading to increased replication stress and DNA damage. WEE1 kinase regulates essential cell cycle checkpoints, preventing premature mitosis and allowing time for DNA repair. Therefore, we hypothesized that loss of RB1 would sensitize TP53-mutant cancer cells to the selective WEE1 inhibitor azenosertib. We tested this hypothesis using small cell lung cancer (SCLC) and triple-negative breast cancer (TNBC) models. In TP53-mutant SCLC (N=9) and TNBC (N=3) cell lines, concurrent RB1 loss increased sensitivity to azenosertib, as evidenced by greater cell growth inhibition compared to cells with wild-type RB1 (median GRmax = -0.23 vs -0.81, 0.01 vs -0.55, respectively). Knockdown of RB1 in RB1 wild-type DMS53 (SCLC) and MDA-MB-231 (TNBC) cell lines sensitized them to azenosertib (GRmax = -0.47 vs -0.94, -0.02 vs -0.5, respectively). Downstream signaling analysis revealed that azenosertib treatment induced higher levels of DNA damage and apoptosis, as shown by the significant increase in γH2AX and cleaved-Caspase 3 levels in RB1 knockdown cells compared to control cells. Conversely, overexpression of RB1 in RB1-null cell lines significantly desensitized the cells to azenosertib with GRmax increased from -0.81 to -0.45 in NCI-H1048 (SCLC), and from -0.63 to -0.34 in MDA-MB-468 (TNBC), respectively. Consistently, RB1 overexpressed cells demonstrated lower levels of DNA damage and apoptosis after azenosertib treatment. In xenograft models, azenosertib demonstrated enhanced tumor growth inhibition (TGI) in models with both TP53 and RB1 loss (NCI-H146 [SCLC]: 90%, MDA-MB-468 [TNBC], 87%) compared to those with TP53 mutation alone (DMS53 [SCLC]: 49%, MDA-MB-231 [TNBC] 50%). Together, our data demonstrates loss of RB1 sensitizes TP53-mutant cancer cells to WEE1 inhibition in multiple models of SCLC and TNBC, suggesting RB1 loss can be used as a predictive marker to identify TP53-mutated patients of these cancers who are likely to benefit from WEE1 inhibition by azenosertib.


 Daehwan Kim, Olivier Harismendy, Catherine Lee, Erika Cabrera, Danielle Jandial, Doris Kim, Mark R. Lackner, Jianhui Ma. Loss of RB1 sensitizes TP53-mutated cancer cells to WEE1 inhibition by azenosertib [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 372.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a59583f3996a4cd3fc1170e30a4a01eaa9a2e62" target='_blank'>
              Abstract 372: Loss of RB1 sensitizes TP53-mutated cancer cells to WEE1 inhibition by azenosertib
              </a>
            </td>
          <td>
            Daehwan Kim, O. Harismendy, Catherine Lee, Erika Cabrera, Danielle Jandial, Doris S Kim, M.R. Lackner, Jianhui Ma
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39cd802b620e803fe61dfbfe7d21f2c3b02ece22" target='_blank'>
              Spatial mapping of transcriptomic plasticity in metastatic pancreatic cancer.
              </a>
            </td>
          <td>
            Guangsheng Pei, Jimin Min, K. Rajapakshe, Vittorio Branchi, Yunhe Liu, B. Selvanesan, F. Thege, Dorsay Sadeghian, Daiwei Zhang, K. S. Cho, Yanshuo Chu, Enyu Dai, Guangchun Han, Mingyao Li, Cassian Yee, Kazuki Takahashi, Bharti Garg, Hervé Tiriac, Vincent Bernard, A. Semaan, Jean L. Grem, T. Caffrey, Jared K. Burks, A. Lowy, Andrew J. Aguirre, P. Grandgenett, Michael A. Hollingsworth, Paola A. Guerrero, Linghua Wang, Anirban Maitra
          </td>
          <td>2025-04-23</td>
          <td>Nature</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad3be920b77d7bc3bd67d114edfbe77637b6103c" target='_blank'>
              Learning the dynamic organization of a replicating bacterial chromosome from time-course Hi-C data
              </a>
            </td>
          <td>
            J. Harju, J. Messelink, Lucas Tröger, Grzegorz Gradziuk, I. Rathnayaka, M. Thanbichler, Muriel C.F. van Teeseling, Chase P. Broedersz
          </td>
          <td>2025-04-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b13792be21e8e149fad24f47b79b2c67bef2640" target='_blank'>
              The protein deacetylase HDAC10 controls DNA replication in malignant lymphoid cells.
              </a>
            </td>
          <td>
            Andreas O Mieland, Giuseppe Petrosino, Mario Dejung, Jia-Xuan Chen, Amitkumar Fulzele, F. Mahmoudi, J.-W. Tu, Al-Hassan M. Mustafa, Yanira Zeyn, Christoph Hieber, Matthias Bros, T. Schnöder, F. Heidel, Sara Najafi, Ina Oehme, Ilse Hofmann, Mike Schutkowski, Sebastian Hilscher, Christian Kosan, F. Butter, S. Bhatia, W. Sippl, Oliver H. Krämer
          </td>
          <td>2025-04-29</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="
 The MECOM locus at chromosome 3q26.2 encodes for the transcriptional regulator EVI1. In ∼10% of de novo AML, aberrant overexpression of EVI1 is observed. In approximately 50% of these it is due to inv(3)(q21;q26.2) or due to the chromosomal rearrangement [3q26.2-r], in which GATA2 enhancer is dislocated and drives EVI1 overexpression. 3q26.2-r AMLs tend to be more stem-like, exhibit therapy resistance and poor patient survival. To determine potential druggable targets, we conducted an unbiased drug screen in 3q26.2-r AML cell lines. We screened a library of over 2400 mechanistically annotated drugs (NCATS Mechanism Interrogation Plates), against seven AML cell lines (four 3q26.2-r and three non-3q26.2-r AML). Exploiting the mechanistic redundancy in the library, we identified target-level sensitivities in 3q26.2-r AML cells and relative activity as compared to other AML subtypes. Corroborating previous studies, in 3q26.2-r AMLs, BET protein inhibitors showed absolute sensitivity. We conducted RNA-seq and mass spectrometry analyses after BET inhibitor (BETi) mivebresib (ABBV-075) treatment. The findings revealed a significant overlap with the expression signature determined by RNA-seq and mass spectrometry performed after CRISPR knockout (KO) of EVI1. The dysregulated proteins included MYC, mTOR and TNF-alpha, indicating them to be potential targets or co-targets with BETi in 3q26.2-r AMLs. Since mTOR, PIK3CA, XIAP and Bcl-xL were also identified as druggable sensitivities in 3q26.2-r AML, we tested the activity of the combined mTOR/PI3K inhibitor dactolisib (BEZ235). Exposure to BEZ235 for 48 hours induced moderate levels of apoptosis in 3q26.2-r AML cell lines and patient-derived (PD) AML cells. This was associated with a decrease in the levels of the EVI1 targets, ERG, Bcl-xL, and c-Myc, as well as of c-Myb and EVI1, with simultaneous increase in the cleaved PARP levels. Combination of dactolisib with mivebresib exerted synergistic in vitro lethality in 3q26.2-r AML cell lines and PD AML blasts. Co-targeting with BETi and other identified sensitivities such as the IAP family inhibitor (LCL161) or Bcl-xL inhibitor (navitoclax or A1155463) also exhibited synergistic lethality in 3q26.2-r AML cell lines and PD AML cells. Importantly, compared to the vehicle control, monotherapy with mivebresib (BETi), dactolisib or LCL161 significantly reduced the AML burden and improved survival of the NSG mice in the xenograft models of 3q26.2-r AML. Co-treatment of mivebresib with dactolisib or LCL161 was superior than each drug alone in reducing AML burden and improving survival. These findings demonstrate promising preclinical activity of novel BETi-based combination with IAP or Bcl-xL inhibitor against AML with EVI1 overexpression.


 Christine E. Birdwell, Warren C. Fiskus, Christopher P. Mill, Tapan M. Kadia, Naval Daver, Courtney D. DiNardo, Koji Sasaki, John A. Davis, Kaberi Das, Hanxi Hou, Surbhi Sharma, Rashmi K. Shamanna, Michele Ceribelli, Kapil N. Bhalla. Novel therapeutic strategies targeting MECOM rearranged AMLs discovered by unbiased drug screen [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6825.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b17c8f4c66aab3898b741804dfbd546dbdd92394" target='_blank'>
              Abstract 6825: Novel therapeutic strategies targeting MECOM rearranged AMLs discovered by unbiased drug screen
              </a>
            </td>
          <td>
            Christine E. Birdwell, W. Fiskus, Christopher P. Mill, T. Kadia, Naval G Daver, C. Dinardo, K. Sasaki, John A. Davis, Kaberi Das, Hanxi Hou, Surbhi Sharma, R. K. Shamanna, Michele Ceribelli, K. Bhalla
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac2fa84e64df92a9b47c54f45baf0d93294fea97" target='_blank'>
              Substrate heterogeneity promotes cancer cell dissemination
              </a>
            </td>
          <td>
            Zuzana Dunajova, Saren Tasciyan, Juraj Májek, J. Merrin, E. Sahai, M. Sixt, É. Hannezo
          </td>
          <td>2025-05-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Extrachromosomal circular DNAs (eccDNAs) have been identified in various eukaryotic organisms and are known to play crucial roles in genomic plasticity. However, in crop plants, the role of eccDNAs in responses to environmental cues, particularly nutritional stresses, remains unexplored. Rice (Oryza sativa ssp. japonica), a vital crop for over half the world’s population and an excellent model plant for genomic studies, faces numerous environmental challenges during growth. Therefore, we conduct comprehensive studies investigating the distribution, sequence, and potential responses of rice eccDNAs to nutritional stresses. We describe the changes in the eccDNA landscape at various developmental stages of rice in optimal growth. We also identify eccDNAs overlapping with genes (ecGenes), transposable elements (ecTEs), and full-length repeat units (full-length ecRepeatUnits), whose prevalence responds to nitrogen (N) and phosphorus (P) deficiency. We analyze multiple-fragment eccDNAs and propose a potential TE-mediated homologous recombination mechanism as the origin of rice’s multiple-fragment eccDNAs. We provide evidence for the role of eccDNAs in the rice genome plasticity under nutritional stresses and underscore the significance of their abundance and specificity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4749bc544ae6bce267e8a661f0fd6d51c2fdcee2" target='_blank'>
              Adaptive dynamics of extrachromosomal circular DNA in rice under nutrient stress
              </a>
            </td>
          <td>
            Hanfang Ni, Lenin Yong-Villalobos, Mian Gu, D. López-Arredondo, Min Chen, Liyan Geng, Guohua Xu, L. Herrera-Estrella
          </td>
          <td>2025-05-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Single-cell RNA sequencing (scRNA-seq) has revolutionized our understanding of cellular heterogeneity. However, scRNA-seq primarily focuses on gene expression, limiting our ability to fully characterize cellular states and their underlying regulatory mechanisms. Here, we present InTraSeq, a novel multimodal assay that enables simultaneous measurement of mRNA, surface markers, cytoplasmic proteins, and nuclear proteins within individual cells. InTraSeq utilizes oligo-barcoded antibodies to profile protein expression in conjunction with scRNA-seq. We demonstrate the application of InTraSeq to study Th17 cell differentiation, revealing novel insights into the dynamics of this complex process. InTraSeq provides a powerful tool for investigating cellular heterogeneity and regulatory mechanisms, enabling a more comprehensive understanding of cellular function in health and disease.


 Majd Ariss. InTraSeq unveils transcriptomic and proteomic landscapes of Th17 differentiation [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB449.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d3f608a4fe82676c1f869676898474a93427b4d" target='_blank'>
              Abstract LB449: InTraSeq unveils transcriptomic and proteomic landscapes of Th17 differentiation
              </a>
            </td>
          <td>
            Majd M. Ariss
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Pulmonary pleomorphic carcinoma (PPC) is a rare and aggressive subtype of non-small cell lung cancer characterized by the presence of both epithelial and sarcomatoid components. We conducted multi-omics analyses on surgically resected specimens from nine primary PPC cases to explore characteristics of this tumor at molecular level.



 Whole-exome sequencing and RNA sequencing on bulk tumor specimens to identify mutational profiles. Spatial transcriptomics using the Digital Spatial Profiler was employed to analyze gene expression separately for epithelial and sarcomatoid tumor components. Single-cell RNA sequencing (scRNA-seq) was conducted on three cases to explore intratumoral heterogeneity and clonal evolution. Gene signature based on extracellular matrix (ECM) remodeling was developed and validated in public lung cancer datasets to assess its association with tumor aggressiveness and prognosis.



 Mutational analyses revealed MET exon 14 skipping in two cases and an ALK-EML4 fusion in one case, while no EGFR mutations were observed. We found various differential expressed genes (DEGs) between components. Further analysis (GSEA and Transcriptional factor scoring) revealed there was a common enrichment trend observed among pathways related to MYC, such as those involved in cell proliferation, division, and the cell cycle. On the other hand, significant differences were observed in pathways related to cancer invasion and cell-cell interactions, such as extracellular matrix (ECM) remodeling and hypoxia. identified an atypical cell population bridging epithelial cells and fibroblasts, which we defined as sarcomatoid cells. The clusters defined as epithelial tumor cells and sarcomatoid cells showed upregulation of DEGs corresponding to the respective component identified through spatial transcriptomics. Trajectory analysis visualized tumor evolution, suggesting a transition from epithelial tumor cells to sarcomatoid cells. A consistent upregulation of seven ECM remodeling-related genes was observed in both tumor components compared to normal lung tissue. When applied to a GEO lung adenocarcinoma cohort, this gene signature was associated with more aggressive histological subtypes (P < 0.01). Survival analysis using TCGA demonstrated that high ECM remodeling signature scores correlated with significantly poorer prognosis (P < 0.01).



 This study provides a comprehensive molecular analysis of PPC, revealing both shared and distinct features between its components. The trajectory analysis supports the hypothesis of a common cellular origin for these components and highlights the transition to a more aggressive sarcomatoid phenotype. These findings provide new insights into the biology of PPC and identify potential therapeutic targets to address its intratumoral heterogeneity and clonal evolution.



 Atsushi Matsuoka, Kazuhiko Shien, Shuta Tomida, Hirofumi Inoue, Kazuya Hisamatsu, Ryota Fujiwara, Kousei Ishimura, Ryunosuke Fujii, Ryo Yoshichika, Tomoaki Higashihara, Naohiro Hayashi, Kazuhiro Okada, Fumiaki Mukohara, Mao Yoshikawa, Ken Suzawa, Hidetaka Yamamoto, Shinichi Toyooka. Spatial and single-cell transcriptomics reveal molecular heterogeneity in pulmonary pleomorphic carcinoma. [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2664.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0640112dbbdf90099adbfa03bf4452da14a83a22" target='_blank'>
              Abstract 2664: Spatial and single-cell transcriptomics reveal molecular heterogeneity in pulmonary pleomorphic carcinoma
              </a>
            </td>
          <td>
            Atsushi Matsuoka, K. Shien, Shuta Tomida, Hirofumi Inoue, Kazuya Hisamatsu, Ryota Fujiwara, Kousei Ishimura, Ryunosuke Fujii, Ryo Yoshichika, Tomoaki Higashihara, Naohiro Hayashi, Kazuhiro Okada, Fumiaki Mukohara, Mao Yoshikawa, K. Suzawa, Hidetaka Yamamoto, Shinichi Toyooka
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Since its advent about ten years ago, the CRISPR-Cas9 system has been frequently used in biomedical applications. It has advanced various fields, and CRISPR-Cas9-based therapeutics have shown promising results in the treatment of specific hematological diseases. Furthermore, CRISPR gene editing technologies have revolutionized cancer research by enabling a broad range of genetic perturbations, including genetic knockouts and precise single nucleotide changes. This perspective focuses on the state-of-the-art methodology of CRISPR knock-ins to engineer immune cells. Since this technique relies on homology-directed repair (HDR) of double-strand breaks (DSBs) induced by the Cas9 enzyme, it can be used to introduce specific mutations into the target genome. Therefore, this methodology offers a valuable opportunity to functionally study specific mutations and to uncover their impacts not only on overall cell functions but also on the mechanisms behind cancer-related alterations in common signaling pathways. This article highlights CRISPR knock-in strategies, protocols, and applications in cancer and immune research, with a focus on diffuse large B cell lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d15e1b343be2ebd40cfad2b5fa562b6f9cb4962e" target='_blank'>
              Strategies for CRISPR-based knock-ins in primary human B cells and lymphoma cell lines
              </a>
            </td>
          <td>
            Sophie Lund, Chun Gong, Xin Yu, Louis M. Staudt, Daniel J Hodson, Sebastian Scheich
          </td>
          <td>2025-05-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 e15149


 Background:
 Cells possess a complex DNA damage response (DDR) system to prevent detrimental mutations from accumulating. The genes encoding various DDR components are frequently mutated in cancer cells and provide potential sites for therapeutic intervention. DNA damaging agents, with one exception, are repaired by the global genome nucleotide excision repair (GG-NER) pathway. Our novel class of drugs, Illudins, are the exception as Illudin-induced DNA damage is not recognized by GG-NER and can only be repaired by the transcription-coupled nucleotide excision repair (TC-NER) pathway. Studies utilizing isogeneic cell lines completely deficient in specific TC-NER proteins (e.g. homozygous ERCC6 knockout) indicate these cells are extremely sensitive to Illudins (up to 30-fold). In contrast, cells deficient in GG-NER activity are not sensitive to our drugs. Moreover, non-replicating or normal cells are minimally affected by Illudins, as evidenced by the lack of systemic toxicity in humans treated with one of our drugs. In TC-NER deficient cancer cells, however, our drug ITX-0121 blocks the TC-NER polymerase complex, which disengages from DNA, and initiates the irreversible process of apoptosis or cellular suicide. Many cancers exhibit deficiencies in DDR mechanisms and cannot properly repair DNA damage.
 Methods:
 Analysis of the GENIE Database indicates that the overall incidence of TC-NER deficient cancers in solid tumors is ~10%, and includes all major histological classifications (breast, ovarian, endometrial, prostate, renal, bladder, bone, colorectal, liver, lung, skin, pancreatic, thyroid, etc.) or ~175,000 new patients annually. These cancers, however, possess monoallelic mutations, as opposed to biallelic, and thus have some residual TC-NER functionality. To determine the relative sensitivity of cancers with monoallelic alterations in TC-NER genes, we generated colorectal HCT-116 cells with a monoallelic deletion in a single TC-NER gene. For example, we are generating daughter cell lines with small monoallelic deletions in ERCC2, ERCC3, ERCC4, ERCC6, and ELOF1 genes.
 Results:
 We are comparing sensitivity of these daughter lines to our lead drug ITX-0121, as well as to other DNA damaging agents such as cisplatin. Preliminary results indicate a monoallelic alteration in a TC-NER gene can confer > 6-fold sensitivity to ITX-0121, which is clinically relevant based on PK results. Based on these PK results, we believe the maximum tolerated dose (MTD) of ITX-0121 in patients to be > 1.2 mg/kg; yet only 0.4 mg/kg will be sufficient for clinical efficacy against cancers that are phenotypically TC-NER deficient.
 Conclusions:
 Thus, despite the administration of a cytotoxic agent, TC-NER deficient cancer patients can be treated at a non-toxic dose; if toxicity occurs, it should be limited to transient thrombocytopenia, as indicated by prior studies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15a4f45d8c212b489e848805a2ff3ec30d91ef9b" target='_blank'>
              ITX-0121: A therapeutic strategy for transcription-coupled repair-deficient cancers.
              </a>
            </td>
          <td>
            M. Kelner, Raymond T. Suhandynata, Venkata R. Kotamraju
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Intrahepatic cholangiocarcinoma (ICC) is a highly lethal cancer of the bile ducts that exhibits a diversity of driver oncogenic mutations, several of which have been shown to be frequently subclonal. However, existing tumor evolution studies in ICC have been limited by either the number of sub-regions assessed or a paucity of driver mutation heterogeneity. Here, we conducted a multi-omic analysis of 90 tumor regions from 11 discrete tumors in 10 patients, identifying subclonal driver mutations in 5 of the patients, in the genes PBRM1, PIK3CA, TP53, ARID1A, BRAF, MLH1, and CDKN2A. This high-number multi-region approach also allowed us to unambiguously identify distinct subclonal evolutionary branches and map them to the spatial location of the samples. Importantly, each subclonal driver mutation was tightly associated with these defined branches, implicating them as a frequent mechanism of driving subclonal evolution. Additionally, subclonal branching showed a spatial correlation on surgical resection depicting the strength of multi-region sequencing in treatment. In total, we conclude that while subclonal branching can be driven by a number of factors, the acquisition of bona fide driver mutations strongly propels subclonal evolution and spatial patterns of tumor growth.


 Evit John, Pooja Shukla, Anish Jain, Alexandria Lau, Tom Lesluyes, Huw Ogilvie, Maxime Tarabichi, Jason Roszik, Yinyin Yuan, Peter Van Loo, Lawrence Kwong, Yun Shin Chun. Driver mutations propel subclonal evolution and spatial patterns of tumor growth in intrahepatic cholangiocarcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7499.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea1a8d6967192526fb7f6ecd11f9a23af479e446" target='_blank'>
              Abstract 7499: Driver mutations propel subclonal evolution and spatial patterns of tumor growth in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Evit R. John, Pooja Shukla, Anish J Jain, Alexandria Lau, T. Lesluyes, Huw Ogilvie, M. Tarabichi, J. Roszik, Yinyin Yuan, P. Van Loo, Lawrence Kwong, Yun Shin Chun
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Aneuploidy, characterized by the gain or loss of chromosomes, plays a critical role in both cancer and congenital aneuploidy syndromes. For any aneuploidy, we can distinguish between its general effects and its chromosome-specific effects. General effects refer to the common cellular stresses induced by aneuploidy, such as impaired proliferation, proteotoxic stress, and altered metabolism, which occur regardless of the specific chromosome involved and profoundly impact cellular and organismal functions. These generalized stresses often hinder cell fitness but can also, under certain conditions, contribute to cancer progression. In contrast, chromosome-specific effects arise from the altered dosage of particular genes on the gained or lost chromosome. These effects vary depending on the chromosome involved and can provide specific fitness effects in cancer cells or distinct developmental phenotypes in congenital aneuploidies like Down syndrome. Understanding the interplay between these two levels of effects is crucial for deciphering the outcomes of aneuploidy. This review synthesizes current knowledge and discusses future directions for unraveling the hallmarks of aneuploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72635ce4b3e9d86105cc5d392c530bed90825c86" target='_blank'>
              The Hallmarks of Aneuploidy in Cancer and Congenital Syndromes.
              </a>
            </td>
          <td>
            Pan Cheng, Karan Singh, Roger H. Reeves, Teresa Davoli
          </td>
          <td>2025-05-07</td>
          <td>Annual review of genomics and human genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 e15034


 Background:
 Endometrial cancer (EC) illustrates diverse nucleotide substitution and chromosomal re-arrangement signatures like other cancers. Namely, four molecular types are identified, replicative polymerases, microsatellite stable, copy number high/serous-like, and microsatellite instability characterized by mutations in mismatch repair genes. With advances in molecular techniques, early prediction of disease aggressiveness can be revealed using comprehension in ECs. Although few cell proliferative genes have been documented, a complete molecular landscaping is essential for effective treatment planning and decision-making. We report, the association between clinical disease status and corresponding molecular findings to establish the role of the latter in the early prediction of disease aggression in endometrial cancer.
 Methods:
 Retrospectively, we investigated the association of genomic mutational profiles of 40 endometrial cancer patients and their clinical disease status from PET scans and HPE reports. Next Generation Sequencing (NGS) test was performed using a comprehensive genomic panel (CGP) OncoIndx assay.
 Results:
 Amongst 40 patients, 65% (n = 26/40) of patients had progressive and metastatic disease from PET scans and HPE reports indicative of disease aggressiveness. Almost 80% (n = 20/26) of the aggressive disease cohort were detected to have a minimum of one genomic alteration. Their mutational profiles showed a diverse spectrum of pathway-related aberrations from cell proliferation, progression, double-strand DNA (dsDNA) repair, mismatch repair (MMR), and tumor suppression constituting a total of 83 genomic alterations detected in the whole cohort. Further, the cell proliferation and dsDNA repair pathways exhibited the highest mutational load of 57.8% (n = 48/83) and 14.5% (n = 12/83) respectively in the cohort. Likewise, tumor suppressor genes constituted the lowest frequency at 6% (n = 5/83).In addition, five patients were detected with high MSI scores and one patient with an intermediate MSI score of which two MSI-high patients were also found to harbor MMR gene mutations (MSH3:p.K383Rfs*32, p.D403H, p.E456* and p.E512*, MSH6:p.F1088Sfs*2).
 Conclusions:
 Using CGP, we observed mutational frequencies of cell proliferation and dsDNA repair pathways that act as crucial markers for disease aggression in EC. We also noted that high MSI scores were not always concordant with mutations in the classical MMR genes and vice versa. This also indicates the individual significance of estimating MSI scores in these patients for clinical decision-making. Since the identified landscape of mutations well correlated with the disease states categorized from their clinical findings, they could potentially transform into molecular attributes for early prediction of progression and metastasis aiding treatment planning and decision making.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8df9baa418d967a1f642449b698f5fd35ecbbbc" target='_blank'>
              Mutational spectrum of cell proliferation genes as early predictive markers for aggressive disease in endometrial cancers.
              </a>
            </td>
          <td>
            Aarthi Ramesh, Rohit Ranade, Sandhya Iyer, Nidhi Patel, A. D'Souza, Sangeeta Prajapati, Bhagwat Jadhav, A. Vasudevan, A. Bharde, J. Khandare, G. Shafi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Loss of tumor suppressor genes in cancer often results in the co-deletion of nearby genes around the chromosomal locus, creating potential collateral damage contexts for targeted therapy. One such example is the loss of MTAP, which is frequently co-deleted with the tumor suppressor genes CDKN2A/B and creates a synthetic lethal dependency on PRMT5 and susceptibility to MTA-cooperative PRMT5 inhibitors. To identify additional novel drug targets for patients with chromosome 9p21 loss, we first conducted unbiased, combinatorial CRISPR screens evaluating the pairwise knockout of ‘context’ genes lost on Ch9p21 against a focused library of ∼5, 000 ‘target’ genes. We identified loss of FOCAD, the gene encoding the SKI complex interacting exonuclease, as synthetic lethal with loss of PELO, a ribosome rescue factor. To expand on this genetic interaction, we conducted genome-wide CRISPR screens in FOCAD isogenic cell lines discovering both PELO and its binding partner HBS1L as synthetic lethal targets in FOCAD-null cells. In FOCAD-deleted xenografts, knockout of HBS1L eliminates tumor growth and results in protein loss of PELO, likely through destabilization of the HBS1L/PELO protein complex. The dependency of FOCAD-null cells on HBS1L/PELO could be rescued by exogenous cDNA expression of FOCAD, and conversely, knockout of FOCAD in WT cells rendered them dependent on HBS1L, demonstrating the robustness of this interaction and validating the HBS1L/PELO protein complex as a promising target for drug discovery. Through cellular phenotypic screens and a medicinal chemistry campaign, we identified pharmacological disruptors of the HBS1L/PELO complex that selectively kill FOCAD-deleted, but not FOCAD-proficient, cells across diverse models. Oral dosing of mice harboring FOCAD-deleted xenografts with our leading compound led to dose-dependent tumor regression and modulation of PD biomarkers. Mechanistically, HBS1L and PELO form a complex responsible for resolving stalled elongation complexes during protein translation. We propose a model in which loss of FOCAD (and/or other SKI complex members) leads to accumulation of aberrant mRNAs on which ribosomes become stalled, leading to hyperdependence on HBS1L/PELO for ribosome rescue. This model is supported by the observation of translational arrest and activation of the unfolded protein response upon HBS1L/PELO inactivation. Clinically, FOCAD loss occurs in ∼30% of all MTAP deleted tumors, representing ∼3% of all solid tumors (including ∼7% of NSCLC). Our lead molecule provides preclinical validation for targeting the HBS1L/PELO complex as a novel strategy for the potential treatment of a large population of patients with FOCAD-deleted tumors.


 Lauren Catherine M. Martires, Matthew Tonini, Hongxiang Zhang, Minjie Zhang, Chengyin Min, Charlotte B. Pratt, Katherine Lazarides, Frank J. Bruzzese, Douglas A. Whittington, Patrick McCarren, Teng Teng, Madhavi Bandi, Shanchuan Zhao, Stephen Paik, Eden Gordon, Shangtao Liu, Alvin Lu, Maria Dam Ferdinez, Binzheng Shen, Hannah Stowe, Yi Yu, Samuel R. Meier, Ashley H. Choi, Lauren Grove, Alan Huang, Alice Tsai, William D. Mallender, Wenhai Zhang, Jannik N. Andersen, Ed Wu, Hilary E. Nicholson. Genetic and pharmacological disruption of the HBS1L/PELO complex impairs mRNA homeostasis and leads to in vivo tumor regressions in FOCAD-deleted cancers [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4252.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11550165fed57fd5f16146624c3f1bdaea238042" target='_blank'>
              Abstract 4252: Genetic and pharmacological disruption of the HBS1L/PELO complex impairs mRNA homeostasis and leads to in vivo tumor regressions in FOCAD-deleted cancers
              </a>
            </td>
          <td>
            Lauren Martires, M. Tonini, Hongxiang Zhang, Minjie Zhang, Chengyin Min, Charlotte B Pratt, Katherine Lazarides, Frank J. Bruzzese, D. Whittington, Patrick McCarren, T. Teng, Madhavi Bandi, Shanchuan Zhao, Stephen Paik, Eden N. Gordon, Shangtao Liu, Alvin Lu, Maria Dam Ferdinez, Binzheng Shen, Hannah Stowe, Yi Yu, Samuel R. Meier, Ashley H Choi, Lauren Grove, A. Huang, Alice Tsai, W. Mallender, Wenhai Zhang, Jannik N Andersen, Edward Wu, Hilary E. Nicholson
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Ewing sarcoma (ES) is the second most prevalent sarcoma that develops in the bones of adolescents and children. A major aberration in Ewing sarcoma is the chromosomal translocation between chromosomes t(11;22), resulting in the loss of one allele of EWSR1 and acquisition of EWSR1-FLI1 fusion gene. Trisomy 8 (∼50%) is the major aneuploidy in ES tumors and is associated with low survival rate in patients. This study aims to elucidate the molecular mechanism underlying the aneuploidy induction in ES. To study the role of loss of one EWSR1 allele and gain of the EWSR1-FLI1 fusion gene inducing aneuploidy, we established a conditional DLD-1 cell line that enables knockdown of EWS derived from one EWSR1 allele using Auxin-induced Degron system and expression of EWSR1-FLI1 using a Tet-on system. The DLD-1 cell line was employed in this study because it is a near diploid cell line that has been utilized to investigate aneuploidy induction by the gene of interest. Treatment of the cell with Auxin (AUX) degrades EWSR1 protein expressed from one allele of EWSR1, while treatment with Doxycycline (DOX) induces the expression of EWS-FLI1. Simultaneous degradation of EWSR1 and EWSR1-FLI1 expression is achieved by treating the cell line with both AUX and DOX. To investigate the effect of loss of one EWSR1 allele and gain of EWSR1-FLI1 fusion gene, cells were treated with AUX (EWSR1 knockdown derived from one allele), DOX (EWSR1-FLI1expression), and AUX/DOX (EWSR1 knockdown and EWSR1-FLI1 expression), for 2 to 8 days. The results showed that all three-drug treated samples led to an increase in cell size. Further, Fluorescence-activated Cell Sorting analysis (FACS) revealed that the majority of the small cells were diploid (2N), whereas the majority of large cells were aneuploid (>3N). While Ewing sarcoma cells are described as small round cells, there was a variation observed in Ewing sarcoma cell sizes, A673 and SKES1, but not in the normal MSC cells.Next, we investigated whether the treatment of AUX and DOX induces trisomy 8 using FISH assay. Both AUX and DOX treated cells displayed a higher incidence of trisomy 8 compared to the control group. Furthermore, the AUX/DOX treated cells displayed a significantly higher incidence of trisomy 8 compared to AUX, DOX treated, and control cells, suggesting that the expression of EWSR1-FLI1 fusion protein and EWSR1 knockdown have a cumulative effect on trisomy 8 induction. While trisomy 8 has been reported in various cancers with a poor prognosis, the induction mechanism remains unclear. To the best of our knowledge, this is the first observation describing the collaborative activities between the EWSR1-FLI1 fusion protein and EWSR1 knockdown in inducing trisomy 8. We are currently investigating the molecular function of trisomy 8 induction in this cell line and in Ewing sarcoma cell lines.


 Priyani H. Hapugaswatta, Mizuki Azuma. Aneuploidy induction mechanism in Ewing sarcoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2624.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/029a236a2721717e13973bdc5e5b49afdd737065" target='_blank'>
              Abstract 2624: Aneuploidy induction mechanism in Ewing sarcoma
              </a>
            </td>
          <td>
            Priyani H. Hapugaswatta, M. Azuma
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 e15032


 Background:
 Homologous recombination deficiency (HRD) is an event triggered by impairment of the homologous recombination repair (HRR) pathway which acts on DNA double-strand breaks and interstrand cross-links (ICL). Although HRD has been shown to elevate the overall genomic instability in tumors inducing favorable responses to immune checkpoint inhibitors (ICI), its clinical utility is observed in specific cancer types like breast, ovarian, low-grade glioma, testicular germ cell tumors and kidney cancers. On the contrary, in many other cancer types, HRD is conducive to the development of immune-cold tumor microenvironment leading to unfavorable treatment responses to ICI. We present, HRD predictive scores with the corresponding IO biomarkers viz. microsatellite instability (MSI) and tumor mutation burden (TMB) pan-cancers.
 Methods:
 Retrospectively, we investigated the biomarker profile (HRD, MSI, TMB) and genomic (HRR pathway genes) profiles of 642 pan-cancer patients using a comprehensive genomic assay using OncoIndx.
 Results:
 Of the cohort of 642 pan-cancer patients, a total of 12.3% (n = 79/642) of patients were detected with high HRD scores, another 16.0% (n = 103/642) were detected with intermediate HRD scores and while 71.7% (n = 460/642) of patients showed low HRD scores. Among the cohorts of high, intermediate and low HRD scores, BRCA1/2 mutation-positive patients were 6.3% (n = 5/79), 5.8% (n = 6/103), and 5.2 (n = 24/460) respectively and HRR gene mutation-positive patients were 17.7% (n = 14/79), 25.2% (n = 26/103), 17.17% (n = 79/460) respectively. Most importantly, the immune-oncology biomarkers MSI and TMB scores were high only in 1.3% (n = 1/79) and 8.9% (n = 7/79) in HRD-high patients which were not significant compared to 1.7% (n = 8/460) and 10.4% (n = 48/460) observed in HRD-low patient cohort. Even in the cohort of patients with intermediate HRD scores, MSI and TMB did not vary significantly. Overall, no significant correlation was observed between high HRD scores vs BRCA1/2 mutations, HRR pathway gene mutations, MSI and TMB statuses. Thus, HRD scores could be predicted high, irrespective of all the associated biomarkers and pathway mutations. This indicates the utility of additional treatment options in BRCA1/2 wild-type or HRR pathway gene wild-type patients.
 Conclusions:
 Although HRD-high patients could be predictive of favorable PARPi and platinum-based therapies, their role in immune activation and ICI therapy response should be carefully considered. Even at a high score, HRD seems to be inducing non-inflammatory tumor microenvironment as opposed to triggering novel cell surface antigens and high TMB aiding in immunotherapy response. However, HRD as an individual biomarker alone adds immense therapeutic benefits even among BRCA-negative patients who could be potential for PARPi/platinum-based therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92c61b0baeee9c5a60d5b38acb29b5a9cb82e2f" target='_blank'>
              The non-canonical role of homologous recombination deficiency in the development of immune-cold tumor microenvironment.
              </a>
            </td>
          <td>
            A. Vasudevan, Aarthi Ramesh, Ankit Jain, Sandhya Iyer, Nidhi Patel, A. Joshi, M. K. Singhal, Sameer Shrirangwar, Chinmaya Kumar Pani, Ganapathi Bhat, V. Patil, Dr T. Raja, J. Khandare, G. Shafi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT
Lymphomas represent one of the most common malignant diseases in young men and an important issue is how treatments will affect their reproductive health. It has been hypothesized that chemotherapies, similarly to environmental chemicals, may alter the spermatogenic epigenome. Here, we report the genomic and epigenomic profiling of the sperm DNA from a 31-year-old Hodgkin lymphoma patient who faced recurrent spontaneous miscarriages in his couple 11-26 months after receiving chemotherapy with adriamycin, bleomycin, vinblastine, and dacarbazine (ABVD). In order to capture the potential deleterious impact of the ABVD treatment on mutational and methylation changes, we compared sperm DNA before and 26 months after chemotherapy with whole-genome sequencing (WGS) and reduced representation bisulfite sequencing (RRBS). The WGS analysis identified 403 variants following ABVD treatment, including 28 linked to genes crucial for embryogenesis. However, none were found in coding regions, indicating no impact of chemotherapy on protein function. The RRBS analysis identified 99 high-quality differentially methylated regions (hqDMRs) for which methylation status changed upon chemotherapy. Those hqDRMs were associated with 87 differentially methylated genes, among which 14 are known to be important or expressed during embryo development. While no variants were detected in coding regions, promoter regions of several genes potentially important for embryo development contained variants or displayed an altered methylated status. These might in turn modify the corresponding gene expression and thus affect their function during key stages of embryogenesis, leading to potential developmental disorders or miscarriages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2daedc5d36f880d0e2fc581751bb206d1ca1d20" target='_blank'>
              Recurrent spontaneous miscarriages from sperm after ABVD chemotherapy in a patient with Hodgkin's lymphoma: sperm DNA and methylation profiling.
              </a>
            </td>
          <td>
            G. Lecuyer, A. Rolland, A. Neyroud, Bertrand Evrard, Nathan Alary, Clemence Genthon, Nathalie Dejucq-Rainsford, Célia Ravel, Jessika Moreau, N. Moinard, Mohamed Hadi Mohamed Abdelhamid, Christophe Klopp, Louis Bujan, Frédéric Chalmel
          </td>
          <td>2025-04-15</td>
          <td>Asian journal of andrology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 e15068


 Background:
 The heterogeneity of tumors promoting the development of precision medicine，however, tumors also share certain common Hallmarks, suggesting the existence of common epigenetic mechanisms. Aberrant DNA methylation is consistently present throughout the process of tumor formation, especially in the pre-cancerous stage, and identification of common biomarkers in tumors is urgently needed. we have successfully developed Guide Positioning Sequencing (GPS) to cover more than 96% CpG sites. By the GPS, we have identified a group of methylation sites that are observed in more than 17 cancer types, termed as Universal Cancer-Only Marker (UCOM), which holds potential for broad application across diverse cancer types. While hrHPV screening has reduced cervical cancer incidence, an effective triage strategy for hrHPV-positive women is still needed to minimize unnecessary colposcopy referrals. Similarly, current urothelial carcinoma (UC) detection methods are invasive and lack sufficient sensitivity and specificity. Therefore, more accurate and practical diagnostic tools for multiple cancer types are required.
 PCDHGB7
 , a UCOM, shows promise as a noninvasive marker for detecting hypermethylated
 PCDHGB7
 in cervical and UC.
 Methods:
 In the cervical cancer cohort, 3251 Thinprep Cytologic Test (TCT) samples from hrHPV positive women will be collected and divided into training and validation group to evaluate the performance of
 PCDHGB7
 in detecting cervical cancer (ChiCTR2100048972). In the UC cohort,
 PCDHGB7
 hypermethylation in urine samples from urothelial carcinoma (UC) patients, to explore its utility in noninvasive UC detection (ChiCTR2100052507).
 Results:
 For cervical cancer,
 PCDHGB7
 detection demonstrated high sensitivity (82.4%) and specificity (91.1%) in distinguishing cervical intraepithelial neoplasia grade 2 (CIN2) or worse from normal cases. It showed superior specificity (92.1% vs. 74.9%) and sensitivity (80.8% vs. 61.5%) compared to cytology, particularly among women positive for HPV16/18 and other hrHPV types. In the UC cohort,
 PCDHGB7
 hypermethylation exhibited exceptional sensitivity (87.3%) and specificity (91.0%) in distinguishing UC from BUD patients. It outperformed standard urine cytology in detecting bladder cancer (82.1% vs. 34.5%), ureter cancer (78.1% vs. 34.4%), and renal pelvis cancer (90.9% vs. 22.7%).
 Conclusions:
 The detection of
 PCDHGB7
 hypermethylation holds significant promise as a UCOM for diagnosing diverse cancers. In cervical cancer, it offers a noninvasive, highly accurate triage strategy for hrHPV-positive women, reducing unnecessary colposcopy referrals. In UC,
 PCDHGB7
 detection provides an effective, noninvasive method for early diagnosis. Overall,
 PCDHGB7
 has the potential to revolutionize cancer diagnostics by offering a versatile, high-performance tool for multiple tumor types.
 Clinical trial information: ChiCTR2100052507
 .
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61b6b3ccca56ae199ddee8ce8e84ab2e6c0af3b3" target='_blank'>
              PCDHGB7
 : A universal cancer-only marker (UCOM) and novel diagnostic tool for diverse tumors.
              </a>
            </td>
          <td>
            Wenqiang Yu, Yaping Dong, Wei Li
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Classically, cancer initiation is attributed to a sequential acquisition of discrete genetic events such as driver mutations. However, homeostatically-stable tissues, including skin, harbor numerous somatic mutations spread across the tissue’s clones. Yet, most of these clones never give rise to cancer. Using the model of UV-driven cuSCC, we tested our hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of goliaths as evidenced by the presence of cell density hotspots and higher mutational burden (evolutionary driver), and 3) emergence of macroscopic lesions. We tracked these tissue level drivers via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25, 085 clades from 14 mice over 3 months, and 14, 525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little across UV exposure, our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3) observed essentially only within UV-exposed skin and occupying the extreme tail of the distribution of clade sizes. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes, some with areas of unusually high local densities (“micro-lumps”). Targeted DNA sequencing of goliath revealed Kmt2a, Kmt2d, Prex2, and SCN11A as the most mutated genes. Giant clades do not typically possess common driver mutations of SCC, e.g. TP53 and NOTCH, and display two extremes of mutational profiles, with some goliaths having dominant clones (VAF >0.75), and others having no discernable clonal structure (VAF <0.025), demonstrating that strong mutational drivers are not a prerequisite for developing goliaths. scRNAseq analysis of both bulk skin and sorted clades revealed upregulation of the fraction of cycling basal-like cells, and downregulation of Alox12e-expressing granular cells, along with increased proportions of plasmacytoid dendritic cells (pDC) and CD4+ regulatory T-cells (Tregs) in UV-exposed samples, reflecting early epidermal de-differentiation and immunosuppression. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions emerged between months 6 and 7, only in UV-exposed skin. Based on our adaptation of the Drake equation, we conclude that the probability of observing macroscopic lesions arising in 2 of the 21 randomly sampled goliaths we followed for 3 months to be less than 10-7. Taken together, our results support the interplay between cell autonomous and non-cell autonomous compartments during cancer initiation, the earliest phase of which presage the acquisition of driver mutations. Our data constitute an unprecedented look at clonal dynamics in UV exposed skin defining the transition from normal tissue to tumor.


 Stanislav S. Avdieiev, Leticia Tordesillas, Karol Prieto, Omar Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, Luiza Silva Simoes, Y. Ann Chen, Noemi Andor, Robert A. Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai. Eco-evolutionary profiling of the UV-driven clonal dynamics and tumor initiation [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2672.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18fc5411928817db2e4ae82f8b06dabe1ec683eb" target='_blank'>
              Abstract 2672: Eco-evolutionary profiling of the UV-driven clonal dynamics and tumor initiation
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A. Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="

 Founder and President of InsideOutBio, Alan Herbert, explores the evolving understanding of genome information storage, and the significance of repetitive sequences called flipons in genome training. These flipons can alter their shape without breaking DNA and are vital in cell biology, especially in responding to environmental stress. Our view of how genomes store information has rapidly changed. In the classical view, codons contain the important content. Each codon consists of three DNA bases uniquely mapped to an amino acid. The order of codons gives the position of amino acids in the protein product that determines its function. Mutations that change codons or their expression then cause disease. In this view, the repetitive sequences that comprise over 50% of the human genome are just a nuisance. They cause genomic instability by frequently breaking and disrupting DNA repair. However, some repeats are useful. For example, telomeric repeats help protect chromosome ends and prevent senescence, while those at centromeres play a role in cell division. Otherwise, most repeat sequences are confined by nucleosomal proteins to limit the harm that they can cause. Recently, a more nuanced understanding of how genomes work has emerged. With this new focus, repeats that dynamically change their conformation in a programmable manner play important roles in cell biology. This class of repeats is called flipons. They enable genomes to learn and optimize cell responses to environmental stressors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/187ef3faa0e6c95091fda1daef8c22d574a2979f" target='_blank'>
              Training your genome
              </a>
            </td>
          <td>
            Alan Herbert
          </td>
          <td>2025-04-08</td>
          <td>Open Access Government</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Leveraging synthetic lethal interactions is an approach to selectively kill tumor cells. A SL interaction between two genes occurs when the disruption of either gene alone is viable, but the disruption of both genes simultaneously results in cell death. Although targeting SL interactions holds immense therapeutic potential, only PARP inhibitors have translated to the clinic. PARP inhibitors are unique in that they cause a gain-of-function (GoF) change in the PARP protein resulting in DNA trapping and selective cytotoxicity in BRCA1/2-mutated tumors. This study employs a novel method of dominant SL screening that aims to identify GoF protein states, modelled by missense mutations, to guide inhibitor design. Here, we evaluate the potential of the cohesin complex as a drug target. Cohesin is composed of four core subunits (SMC1, SMC3, RAD21, and STAG2) and plays roles in sister chromatid cohesion, DNA damage repair, and chromatin organization. This study shows that mutations in the C-terminal ATPase domain cause dominant effects and cluster in a druggable domain, suggesting this state could be phenocopied therapeutically. Using S. Cerevisiae as a model, dominant SL screening of a random mutagenesis library was used to identify states of the target protein, Smc1, that cause dominant lethality in a cohesin-compromised background. This revealed 17 key residues, all residing in the C-terminal ATPase domain, with specific clustering in the signature motif. Directed SL testing of 11 mutants was performed with a panel of cancer-like backgrounds. These data showed that expression of the mutants cause dominant SL in backgrounds with altered cohesin regulation, mitotic defects and replication deficiencies. Further, cell cycle analysis was performed on select mutants, showing that mutations are causing dominant G2 arrest, leading to SL cell death in a cohesin-compromised background. To test if these mutants rely on cohesin complex formation, we tested reliance on scMcd1(hsRAD21) and Smc3. Overexpressing the cohesin ring rate-limiting factor Mcd1 in conjunction with mutant protein shows worsened phenotypes, while introducing a secondary mutation that disrupts the Smc1-Smc3 interface abolishes dominant lethality. These data suggest that the mutant Smc1 proteins rely on the creation of complexes to elicit their dominant negative properties. Importantly, structural mapping reveals that the hits from the screen are conserved and map to ATP interacting residues on the human structure, revealing a potentially druggable domain. The performed mutational mapping can guide in silico molecular docking to design small molecules that can then be tested for phenocopying of the dominant state for better clinical translation.


 Elizabeth Stephens, Nigel O’Neil, Peter Stirling, Philip Hieter. Revealing therapeutically relevant states of cohesin through dominant-negative mutational mapping of Smc1 as a synthetic lethal anti-cancer approach [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1467.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c70be44318df75f89eda9a331333e9af3f2cf21" target='_blank'>
              Abstract 1467: Revealing therapeutically relevant states of cohesin through dominant-negative mutational mapping of Smc1 as a synthetic lethal anti-cancer approach
              </a>
            </td>
          <td>
            Elizabeth Stephens, N. O’Neil, Peter C Stirling, Philip Hieter
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The TP53 gene encodes the critical tumor suppressor p53, which is involved in the regulation of cell cycle control, apoptosis, and genomic stability. In cancer biology, p53 holds an elevated position because high mutation rates in the TP53 gene across human cancers lead to abnormal p53 protein. This would allow limitless replication of genetically damaged cells with malignant potential and so facilitate tumorigenesis. Normally referred to as the “guardian of the genome,” this protein assumes a central role in different cellular functions spanning metabolism through cell cycle checkpoints and apoptosis to DNA repair mechanisms. Its significance extends beyond tumor suppression because it controls metabolic pathways that are central to maintaining homeostasis under normal physiological conditions. The normal functions of p53 as a tumor suppressor are abrogated by mutations; indeed, many such mutant forms act as oncogenes themselves when expressed in cancer cells. The review paper presents information from recent scientific investigations on the chemical structure of p53 protein and its mutations with a discussion on implications for cancer treatment. It also integrates various research findings to clarify the clinical effects of the p53 gene, especially in breast cancer and myelodysplastic syndromes (MDS), and looks into changing treatment plans that focus on p53 problems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c0adda59bbabd91047006215c78f72a09302aad" target='_blank'>
              Structure, Function and Clinical Significance of P53 Protein: A Review Article
              </a>
            </td>
          <td>
            Farah mumtaz, Hadeel Nasser, Ruqaya Yahya Abd AL-Shaheed, Ali A. Al-fahham
          </td>
          <td>2025-05-28</td>
          <td>International Journal of Health &amp; Medical Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 e17631


 Background:
 Poly (ADP-ribose) polymerase inhibitors (PARPi) has been studied to show positive efficacy in endometrial cancer (EC). However, rational candidates for precision treatment with PARPi need to be further explored. Recently, it has been reported that tumors with homologous recombination deficiency (HRD) are sensitive to PARPi. Here, we aimed to assess the relationship between HRD and molecular subgroups in EC.
 Methods:
 HRD statues and gene characteristics of 372 EC patients were retrospectively collected from August, 2022 to August, 2024. HRD score was calculated as the sum of loss of heterozygosity (LOH), telomeric allelic imbalance (TAI), and large-scale state transitions (LST) scores based on the next-generation sequencing (NGS) data.
 Results:
 Of the EC that underwent NGS testing, there were 5.38% (20/372) patients with
 BRCA1/2
 mutations, and the top two
 BRCA1/2
 mutated subgroups were microsatellite instability-high (MSI-H) (2.96%) and copy number low (CN-L) (1.88%) subgroups. However, there were 10.75% (40/372) patients with positive HRD, which is largely restricted to copy number high (CN-H) (4.57%), CN-L (2.96%) and MSI-H (2.96%) subgroup. Interestingly, based on HRD score with the cutoff of 30, there were mostly CN-H and CN-L subgroups and almost no
 POLE
 mutation and MSI-H subgroups in non
 BRCA1/2
 mutation and positive HRD patients.
 Conclusions:
 BRCA1/2
 mutations and positive HRD were seen in 5.38% and 10.75% EC patients, respectively. HRD score testing, based on LOH, TAI, and LST, may help select patients benefiting from PARPi with particular reference to CN-H and CN-L subgroups.


 Prevalence of HRD statues in different molecular subgroups of EC.





 HRD
 Non-HRD




 Molecular Subgroups
 All

 BRCA1/2
 mutation

 HRD score≥30




 POLE
 mut

 1(0.27%)
 1(0.27%)
 0(0%)
 8(2.15%)


 MSI-H
 11(2.96%)
 11(2.96%)
 0(0%)
 64(17.20%)


 CN-H
 17(4.57%)
 1(0.27%)
 16(4.30%)
 64(17.20%)


 CN-L
 11(2.96%)
 7(1.88%)
 4(1.08%)
 196(52.69%)






 HRD: Homologous recombination deficiency; EC: endometrial carcinomas;
 POLE
 mut:
 POLE
 mutations; MSI-H: microsatellite instability-high; CN-L: copy number low; CN-H: copy number high.




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66ff21a16e023a00431cd6937437785071b41ecc" target='_blank'>
              Prevalence of homologous recombination deficiency in different molecular subgroups of endometrial carcinomas.
              </a>
            </td>
          <td>
            Yang Rao, Wenwen Zhang, Na Zhang, Han Han, Yong Xiao, Ping Liu
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Head and neck squamous cell carcinoma (HNSCC) is the seventh most common malignancy worldwide. Most cases are linked to alcohol, tobacco, and areca nut, while a smaller proportion is driven by high-risk human papillomavirus (HPV). HPV-negative HNSCC is associated with poor prognosis with a 5-year survival rate of 40-50%. This dismal outcome has remained relatively static over the past several decades, highlighting the critical need for the discovery of novel targets and therapeutics to better manage HPV-negative HNSCC patients.Methodology: We performed parallel in vitro and in vivo CRISPR-Cas9 knockout kinome screens to identify kinases essential for the fitness of HPV-negative HNSCC cells. CAL27 Cas9 EGFP cells were transduced with a pooled sgRNA kinome library containing 6, 104 sgRNAs at a MOI of ∼0.4. After 7 days of puromycin selection, these transduced cells were cultured in vitro for either 7 or 14 days or implanted into the flanks of athymic nude mice, with tumor harvesting at 6 weeks. DNA was isolated from both cells and tumors, followed by next-generation sequencing to identify and quantify individual sgRNAs. Fitness genes identified in both the in vitro and in vivo screens were further validated using shRNA technology in a panel of HPV-negative HNSCC cell lines. Results: We identified 24 fitness genes that were common to both our in vitro and in vivo screening approaches. These kinases are primarily involved in pathways critical for DNA damage and repair, metabolism, and cellular proliferation. Among them, we prioritize the validation of polynucleotide kinase 3’phosphatase (PNKP) as an essential gene in a panel of HPV-negative HNSCC cell lines. shRNA-mediated knockdown of PNKP significantly reduced the cellular proliferation and clonogenic survival of CAL27, Detroit-562 and FaDu cells. Furthermore, PNKP knockdown severely impaired tumorigenicity in vivo, with CAL27 and FaDu xenografts showing markedly reduced tumor growth compared to controls. Phosphoproteomic analysis revealed that PNKP knockdown resulted in dephosphorylation of 762 unique sites (log2FC>1.5; FDR p<0.05), with a particular impact on proteins involved in nuclear and mitochondrial DNA repair pathways.Conclusion: Our study identified and confirmed PNKP as an essential gene for the survival of HPV-negative HNSCC cells. This discovery highlights the potential of PNKP as a novel therapeutic target, which may lead to the development of innovative therapeutics that could improve the clinical outcomes of HPV-negative HNSCC patients.



 Gurleen Kaur Tung, Kate Chatfield-Reed, Masaru Miyagi, Wendi Quinn O'Neill, Quintin Pan. CRISPR/Cas9 kinome screen identifies PNKP as a novel fitness gene in HPV-negative head and neck squamous cell carcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 6947.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81014681c316f4c01121749b9d1b735ce752673e" target='_blank'>
              Abstract 6947: CRISPR/Cas9 kinome screen identifies PNKP as a novel fitness gene in HPV-negative head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Gurleen Kaur Tung, Kate Chatfield-Reed, Masaru Miyagi, W. O'Neill, Quintin Pan
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Breast cancer (BC) is one of the leading causes of death in women worldwide. Modern molecular medicine includes techniques like deoxyribonucleic acids (DNA) microarray and polymerase chain reaction (PCR); however, these techniques require specialized and expensive equipment to determine DNA concentration and purity. In breast cancer (BC), tumor growth promotes rapid cell replication, leading to errors and instability in DNA molecules. As a result, total DNA may exhibit differential degradation and changes in its electrical properties. DNA molecules possess an electronegative charge and implicit viscosity, so the number of double-helix structures and their electrical and physical properties are expected to alter volumetric bioimpedance based on DNA concentration and purity. In this study, a biosensor of BC DNA concentration and purity based on Electrical Bioimpedance Spectroscopy (EBiS) is proposed. Twenty DNA samples extracted from BC biopsies were quantified in concentration and purity by UV-Vis spectrophotometry and then evaluated through EBiS measurements from 100 Hz to 10 MHz Spearman correlation coefficients for EBiS measurements in the entire frequency range with respect to DNA concentration and purity were estimated through SPSS statistics software (Ver 21.0). The findings show impedance magnitude Z with a significant statistical correlation to DNA concentration and purity at 9.12 and 5.75 MHz, respectively (R 9.12 MHz = 0.936, R 5.75 MHz = 0.56, p < 0.05). The findings indicate the use of a biosensor based on EBiS measurement is a feasible technique to estimate BC DNA concentrations and purity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd040d5e3b25bafc8991d72e77f8f3858f9e013f" target='_blank'>
              Correlation of EBiS vs. Concentration and Purity of Total DNA in Breast Cancer Samples
              </a>
            </td>
          <td>
            Leticia Arias-González, J. Cruz-Ramos, N. Pérez-Vielma, Jacobo E. Mungia-Cervantes, Enrique Hernández-Sánchez, Marlon Rojas-López, Modesto Gomez-López
          </td>
          <td>2025-05-01</td>
          <td>Journal of Physics: Conference Series</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Numeric chromosomal imbalance, known as aneuploidy, is linked to both malignant and potentially malignant epithelial lesions. Aneuploidy has also been investigated in oral potentially malignant disorders (OPMDs) due to its high incidence in head and neck cancers, particularly in oral squamous cell carcinoma (OSCC). The study aimed to evaluate the potential of aneuploidy, a marker of chromosomal imbalance, as a prognostic tool for assessing malignant transformation risk in oral lichen planus (OLP) patients. Methods Fluorescent in situ hybridization (FISH) analysis targeting centromeric probes for chromosomes 2 and 8 was conducted on samples from 245 patients, with follow-up in 135 cases. Results Aneuploid cells (ACs) were detected in 73 patients (29.8%); 24 (32.9%) exhibited non-diploid cells in a normal looking mucosa. Only 2 (0.8%) patients developed OSCC during the follow-up. Among the 135 followed, 11 (8.1%) were positive for Acs in both samples, 15 (11.1%) were were negative initially but positive later. In contrast, 3 patients (2.2%) were initially positive but later negative. Conclusion These results indicate a low malignant transformation rate (< 1%), despite a high rate of aneuploidy. These also demonstrate variability in aneuploidy results over time. The dynamic nature of aneuploidy observed suggests that it may not be a reliable predictive tool for malignant transformation in OLP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b2028a4a76ed543b84deecc013d50688f9c0b83" target='_blank'>
              Is Aneuploidy a Consistent Marker for Malignant Transformation Risk in Oral Lichen Planus?
              </a>
            </td>
          <td>
            Aya Pessing-Shabi, A. Zlotogorski-Hurvitz, N. Yarom, Ilana Kaplan, Luba Trakhtenbrot, Abraham Hirshberg
          </td>
          <td>2025-05-08</td>
          <td>Head and Neck Pathology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55e359c7ef64214d02915af71ac1f575ebe2c2dd" target='_blank'>
              Chinmo defines the region-specific oncogenic competence in the Drosophila central nervous system
              </a>
            </td>
          <td>
            Phuong-Khanh Nguyen, Francesca Froldi, Louise Y. Cheng
          </td>
          <td>2025-05-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Transposable elements (TEs) make up 45% of the human genome, are a source of genetic variability difficult to detect, and involved in processes related to gene regulation and disease. Nanopore sequencing is recognized as one of the best technologies for detecting TEs; however, tools for analyzing of human TE insertions and deletions with nanopore-based data can be improved. RetroInspector is an easy to use, configurable Snakemake pipeline that performs detection, annotation, enrichment, and genotyping of TEs. RetroInspector requires the FASTQ files of the samples and the reference genome to start the identification and analysis of TEs. The user can also set the threshold for the number of supporting reads for the variant filtering. RetroInspector also allows users to compare the results of two samples. Different versions of the reference genome can be used and the presence of retrotransposition features can be annotated. RetroInspector has been run on three nanopore sequencing datasets and validated experimentally using proprietary and public data with over 80% precision.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471f8f2239ee3f6cc9ddafb9c7703071d9cf785f" target='_blank'>
              Advanced analysis of retrotransposon variation in the human genome with nanopore sequencing using RetroInspector
              </a>
            </td>
          <td>
            Javier Cuenca-Guardiola, B. de la Morena-Barrio, Javier Corral, J. Fernández-breis
          </td>
          <td>2025-04-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="SET domain bifurcated histone methyltransferase 1 (SETDB1) is a histone H3 lysine 9 (H3K9) methyltransferase, functioning in transcriptional silencing of the transposable elements (TEs), endogenous retroviruses (ERVs), interferon-stimulated genes (ISGs), and immune cell-related molecules. SETDB1 collaborates with cofactors such as ATF7IP and TRIM28 and functions in concert with DNA methylation to maintain gene repression in both stem cells and differentiated somatic cells. Given its gene targets, recent studies have shown that SETDB1 is a critical immune checkpoint gene in both solid and hematological tumors. In this review, we first discuss the role of SETDB1 in gene regulation through its histone methyltransferase activity, including an overview of its structural features and key cofactors. We then highlight the lineage-specific roles of SETDB1 in both normal hematopoietic processes and hematological malignancies, emphasizing its function as an immune checkpoint molecule that suppresses natural killer (NK) cell-mediated antileukemia responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b622142a48ee47c5894f0bcff4fb87ea6d35f6c" target='_blank'>
              The Role of Histone Methyltransferase SETDB1 in Normal and Malignant Hematopoiesis.
              </a>
            </td>
          <td>
            Yu-Hsuan Chang, S. Goyama
          </td>
          <td>2025-06-04</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Spindle Pole Body Component 25 (SPC25), a critical component of the NDC80 kinetochore complex, plays an essential role in maintaining chromosomal stability during mitosis. Recent studies have revealed its aberrant expression in various cancers, highlighting its potential as both a diagnostic biomarker and a therapeutic target. This review provides a comprehensive analysis of the molecular mechanisms underlying SPC25’s involvement in tumorigenesis, including its regulation of cell cycle progression and interaction with key oncogenic pathways. Furthermore, we discuss its prognostic significance across different cancer types and its potential impact on therapy resistance. The emerging evidence underscores SPC25’s multifaceted role in cancer biology, offering novel insights into its clinical applications. We conclude by exploring future research directions, emphasizing the need for in-depth studies to unravel the precise molecular functions of SPC25 and its therapeutic potential in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1c39ca98ea2816bab12e16bae857d1896574f49" target='_blank'>
              The multifaceted functions of SPC25 in cancer: from molecular pathways to targeted therapy
              </a>
            </td>
          <td>
            Yigang Jin, Minjie Chen, Fei Chen, Zhao-feng Gao, Xiaoping Li, Lingyu Hu, Dandan Cai, Siqi Zhao, Zhengwei Song
          </td>
          <td>2025-05-07</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Recently, prostate cancer (PCa) has been increasing in incidence and mortality, which seriously threatens men’s physical and mental health. Adenosine (A)-to-inosine (I) RNA editing, contributing to nearly 90% of all editing events in human, has been reported to contribute to pathogenesis and progression of cancer. Here, we aimed to elaborate the role and mechanism of A-to-I-edited POLA2 in PCa. Methods RT-qPCR, Western blotting, and immunohistochemistry were used to assess gene expression. RNA editing levels were determined by Sanger sequencing. Colony formation, CCK-8, and Transwell assays were conducted to detect cell proliferation and metastasis. And Flow cytometry assay was applied to examine CD8+ T cell activity and tumor cell apoptosis. Dual-luciferase reporter assay demonstrated the relationship between gene and miRNA. The ability of glycolysis was measured by Seahorse XF96 Analyzer. Results A-to-I RNA overediting of POLA2 was identified in PCa patients, which was related to unfavorable clinical outcomes and prognosis. The A-to-I RNA editing of POLA2 was mediated by ADAR1 enzyme in human cancers. Functionally, A-to-I RNA editing endowed POLA2 with carcinogenicity in PCa development, and POLA2 overediting aggravated cell viability and metastasis of PCa. More importantly, POLA2 overediting fortified glycolysis and impaired CD8+ T cell cytotoxicity in PCa. Mechanically, edited POLA2 upregulates BTBD7 expression in PCa by binding to miR-596. Conclusion A-to-I RNA edited POLA2 attained carcinogenesis in PCa by impeding immune infiltration, fortifying glycolysis and upregulating BTBD7, indicating that edited POLA2 has the potential to become a tool for gene therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd5a0ab9a9ad629e2895479ccb1798aa2061077e" target='_blank'>
              A-to-I RNA edited POLA2 attains carcinogenesis in prostatic cancer by impeding immune infiltration and upregulating BTBD7
              </a>
            </td>
          <td>
            Shengyi Lin, Hang Huang, Yeping Li, Yongyong Lu, Tingyu Ye
          </td>
          <td>2025-05-14</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="RNA small fragment aberrances are associated with diseases by mediating a range of pathogenesis and pathological processes. DNA assembly-based barcoding and amplification technologies are currently being actively explored for RNA in situ analysis. However, these modular integrated DNA assembly processes are inevitably accompanied with false positive signals caused by unexpected misassembly. Completely avoiding this phenomenon through simple and universal methods is challenging. Here, a novel dual-input to dual-output in situ analysis paradigm is proposed, aiming to improve target specificity through co-recognition (dual-input) and to eliminate false positive misassembly through fluorescent signal co-localization (dual-output). Based on this paradigm, Gemini molecular assembly co-localization (GOAL) in situ imaging system is launched to accurately distinguish the fusion gene subtypes associated with chronic myeloid leukemia (CML), and to precisely report the proportion of minimum residual cancer cells in clinical samples by intelligent co-localization counting and sorting. GOAL achieves highly sensitive and accurate genotyping recognition of 0.01% CML tumor cells and realizes fully automatic rapid diagnosis with a customized Intelligent Cell Image Sorter (iCis). iCis-assisted GOAL represents an innovative and versatile molecular toolkit for accurate, rapid, user-friendly, and professional-independent profiling of cancer cells with RNA small fragment aberrances, providing efficient clinical decision support for disease diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd630ef4621d438baadd64541aa5f505561cc528" target='_blank'>
              Gemini Molecular Assembly Colocalization (GOAL): Accurate and Efficient Fusion Genotyping for Chronic Myeloid Leukemia Intelligent Diagnosis.
              </a>
            </td>
          <td>
            Yuanyuan Huang, Zixin Zhang, Tiantian Yang, Yangli Zhang, Xiaoxue Cheng, Yuexi Kang, Yujie Guang, Yuting Zou, Xiaoying Zhang, Zewei Luo, Junman Chen, W. Cheng
          </td>
          <td>2025-04-14</td>
          <td>Small methods</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 3585


 Background:
 Colorectal cancer is the third most common cancer worldwide, accounting for about 10% of all cancer cases, and is expected to claim more than 50,000 lives in 2025. Approximately 33% of CRC patients will develop metastases throughout their cancer continuum, and their 5-year survival rate is about 15%. The majority of patients with metastatic colorectal cancer (mCRC) cannot be cured. However, a subset of mCRC patients with localized recurrence or isolated metastases in the liver and/or lungs may achieve a cure through surgical intervention. Yet, current methods for identifying patients who are candidates for more favorable responseremain inadequate. Therefore, there is a critical need for predictive biomarkers to accurately identify patients who are likely to experience better outcomes following surgery. 5-hydroxymethylcytosine (5hmC) is an epigenetic modification that is associated with active genes and regulatory regions that are cell type- and disease-specific. Here, we developed a model using cell-free DNA (cfDNA) 5hmC profiles to detect CRC and identified pathways distinguishing mCRC patient outcomes following treatment.
 Methods:
 Plasma was collected from 294 CRC patients and 588 non-cancer individuals to obtain cfDNA. cfDNA was enriched for 5hmC-containing DNA fragments. Input and 5hmC-enriched cfDNA were subsequently used to generate sequencing libraries to obtain WGS and 5hmC profiles, respectively. Machine learning operating on 5hmC and WGS data was used to develop a CRC detection model which was subsequently tested on an independent set of mCRC (n = 69) and non-cancer samples (n = 70). Differential 5hmC analysis was performed using edgeR and Gene Set Enrichment Analysis (GSEA).
 Results:
 The performance of the CRC prediction model was evaluated through 10-fold cross-validation producing an auROC curve of 0.86. An independent validation set of mCRC and non-cancer patients displayed an auROC of 0.94. Comparative GSEA using gene body 5hmC levels revealed biological pathways associated with CRC biology such as Myc signaling. cfDNA 5hmC profiling of plasma obtained from mCRC patients before surgery revealed quantitative differences in patients who show recurrence of disease within 2 years post-surgery from the patients who remain recurrence-free for at least 2 years after surgery. These differences between relapsed and non-relapsed groups included 5hmC changes over genes involved in pathways known in mCRC, such as the Wnt/β-catenin signaling (p < 0.05). Lastly, quantitative changes in 5hmC profiles measured in pre-surgery plasma samples enabled prediction of disease recurrence in patients within 2 years post-surgery.
 Conclusions:
 5hmC analysis of cell free DNA) offers a novel, non-invasive approach for identification of colorectal cancer biology and assessment of treatment response in blood samples.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b009d2d1bedf97a401e06e828fb8e966e7e7b8bf" target='_blank'>
              Cell-free DNA 5-hydroxymethylcytosine profiling for the assessment of colorectal cancer biology and treatment response in blood.
              </a>
            </td>
          <td>
            Ceyda Coruh, Y. Ning, Shimul Chowdhury, Jacqueline L Mudd, De'Vana Crutcher, Netra Navadkar, Christopher A. Maher, Ryan Fields, G. Guler, S. Levy
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 A principle in the development of cancer therapies is that robust death of the malignant cell is critical. Highly effective strategies, such as immunotherapy, engage this principle. Here I will provide the structural and molecular underpinnings for an approach that leverages chemical induced proximity to produce specific cell killing of diffuse large B cell lymphoma, the most common non-Hodgkin’s lymphoma. We developed Transcriptional/epigenetic Chemical Inducers of Proximity (TCIPs): bivalent molecules that redirect epigenetic factors to activate programmed cell death genes normally repressed by the oncogenic driver, BCL6. Acute treatment rapidly reprograms the epigenome to initiate apoptosis and repress c-MYC. The crystal structure of the chemically induced complex of the histone acetyltransferases p300/CBP and BCL6 reveals how chance interactions between the proteins can be systematically exploited to produce exquisite potency and selectivity. Biophysical, proteomic, and genomic experiments demonstrate a sub-stoichiometric, gain-of-function mechanism. Our findings imply that the malignant function of an oncogenic driver can be systematically exploited using chemically induced proximity to activate robust cell death.


 Sai Gourisankar. Gain-of-function small molecules to reprogram oncogenic transcription factors [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr SY07-01.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/950a3c88211fd8f62863e7c5d3ab9f041ca13d73" target='_blank'>
              Abstract SY07-01: Gain-of-function small molecules to reprogram oncogenic transcription factors
              </a>
            </td>
          <td>
            S. Gourisankar
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Lung cancer is the leading cause of cancer-related deaths worldwide, with non-small cell lung carcinoma (NSCLC) accounting for 85% of cases and associated with poor prognosis. The lysine-specific demethylase KDM6A, responsible for removing H3K27 trimethylation at gene promoters and enhancers, is frequently mutated in various cancers. While KDM6A is predominantly considered as a tumor suppressor gene, recent studies suggusted it may support oncogenic transcriptional regulation. To investigate the role of KDM6A in tumorigenesis, we utilized CRISPR-Cas9/sgRNA to knock out (KO) KDM6A in NCI-H1975 NSCLC cells. Phenotypic assessments demonstrated cell cycle arrest with inhibitory effects in cell proliferation with KDM6A-depleted NCI-H1975 cells. RNA sequencing in conjunction with CUT&RUN revealed KDM6A target genes involved in cell cycle regulation, DNA replication, and DNA damage repair pathways, which were further validated using RT-qPCR and CUT&RUN-qPCR. Immunofluorescence confirmed the effects of KDM6A loss on DNA replication and DNA damage-induced repair mechanisms. These findings support our hypothesis that KDM6A is a highly pleiotropic factor in tumorigenesis, playing a critical role in regulating DNA replication-associated gene activation and facilitating the repair of replication stress-induced DNA damage. Notably, loss of KDM6A induced hyperactivation of checkpoint signaling pathways such as the ATR-CHK1 axis, suggesting KDM6A-depleted tumor cells attemption to alleviate replication stress to avoid apoptosis. To evaluate the therapeutic potential of KDM6A along with inhibition of the KDM6A-depletion-induced ATR-CHK1 pathway, ceralasertib (formerly AZD6738), an oral selective ATR inhibitor, was combined with KDM6A depletion for in vivo evaluation. Introducing KDM6A depletion significantly reduced NCI-H1975 tumor volume compared to the control group, demonstrating anti-tumor activity. Consistent with in vitro cell-based assays, the combination therapy of ceralasertib with KDM6A depletion showed enhanced tumor growth inhibition and regression in the NCI-H1975 xenograft model. Our findings reveal a pro-oncogenic role of KDM6A in sustaining tumor maintenance through engagement in DNA replication and repair pathways. Targeting KDM6A alone emerges as a promising therapeutic strategy for NSCLC. Furthermore, combining KDM6A depletion with ceralasertib significantly enhances anti-tumor activity by disrupting the ATR-CHK1 checkpoint signaling pathway, leading to an enhanced anti-tumor activity.


 Pony Yu-Ling Lee, Kun-Yuan Lin, Chao-Di Chang, Shan-Yun Cheng, Ya-Wen Hung, Chih-Chieh Yang, Yu-Hsien Chang, Chien-Chang Shen, Shu-Ping Wang. Targeting histone H3K27 demethylase for anti-tumor activity: Monotherapy and combination with ceralasertib in non-small cell lung carcinoma [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 1671.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa9bdc18fb66ff82d4c5e1813d9708441f70c7c7" target='_blank'>
              Abstract 1671: Targeting histone H3K27 demethylase for anti-tumor activity: Monotherapy and combination with ceralasertib in non-small cell lung carcinoma
              </a>
            </td>
          <td>
            Pony Yu-Ling Lee, Kun-Yuan Lin, C. Chang, Shan-Yun Cheng, Y. Hung, Chih-Chieh Yang, Yu-Hsien Chang, Chien-Chang Shen, Shu-Ping Wang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 e23526


 Background:
 Kaposi sarcoma (KS) is a tumor caused by Kaposi sarcoma herpesvirus (KSHV), also known as human herpesvirus 8 (HHV-8). Unlike other virus-associated tumors, where disease progression often continues even after viral eradication, KS requires the persistent presence of KSHV for disease progression. This unique dependency underscores the potential of strategies targeting the virus to also target the tumor. This study aims to characterize the expression of KSHV viral proteins and identify factors associated with differences in viral gene expression, ultimately formulating treatment targets.
 Methods:
 A total of 43 unique samples were collected at a single academic institution between 2001 and 2024. RNA was extracted from formalin-fixed paraffin-embedded (FFPE) tissue blocks and subjected to bulk RNA sequencing. Reads were mapped to the human genome (GRCh38.p14) using STAR and to the KSHV genome (NC_009333) using Salmon. Transcript abundance in units of Transcripts Per Million (TPM) and estimated read counts to each transcript was quantified with Salmon. Clinical and demographic data from the time of sample collection were extracted from medical records. Differential gene expression analysis, PCA using DESeq2 and hierarchical clustering were done in R.
 Results:
 The 43 samples analyzed were derived from 25 patients, with 23 (92%) being male. Of the 18 patients with known HIV status, 4 (22%) were HIV-positive. None had a history of organ transplantation. Among the 14 patients with available disease status, 5 (38%) had disseminated disease at the time of sample collection. KSHV gene expression was detected in 42 out of 43 samples. The most consistently expressed genes across all samples were ORF75 (42 samples, proximal to the poly-A tail), ORF72/v-cyclin (42 samples), and ORF73/LANA (39 samples). A pronounced pattern of high expression was observed in the latency-associated region of the KSHV genome, which was conserved across all KS tumors. The three clusters of samples generated from hierarchical clustering exhibited different lytic and latent gene profile (lytic, latent, mixed), which are similar to previous studies. The differentially expressed genes between different tumor morphology or HIV status of the host showed a mix profile of lytic and latent genes.
 Conclusions:
 This study highlights the latency-associated region of KSHV as the most consistently and highly expressed region, regardless of baseline characteristics, presenting a potential conserved therapeutic target.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69dfe0589aae838d9a59485221f8f4d5df8fd8a5" target='_blank'>
              Transcriptomic landscape of Kaposi sarcoma: Insights into therapeutic targeting of KSHV.
              </a>
            </td>
          <td>
            Yuewei Fei, P. Costa, M. Junejo, Michelle Li, Curtis J. Perry, William Damsky, J. Ishizuka
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Genome maintenance is of the utmost importance in stem cells, as mutations can be propagated and cause defects in derivative tissues. Many stem cell types display low mutation rates, with embryonic stem cells (ESCs) being a notable example. The bases for this property are unclear but may be achieved by optimization of various processes including high-fidelity DNA repair, cell cycle checkpoint controls, and hypersensitivity to genotoxic insults that trigger cell death. Here, we investigate the mechanisms underlying the unique responses of mouse ESCs (mESCs) to replication stress (RS) using an array of small molecule inhibitors and genotoxins. We find that whereas mESCs survive under acute RS in an ATR- and CHK1-dependent manner similar to somatic cells, they lack a strong G2/M checkpoint and fail to repair DNA crosslinks in the absence of ATR signaling. Despite the lack of a strong G2/M checkpoint, mESCs maintain a spindle assembly checkpoint (SAC). We posit that mESCs preferentially repair DNA crosslinks in S phase via homology-directed mechanisms, and cells that fail to complete repair before mitosis undergo mitotic catastrophe and cell death. These findings shed light on mutation avoidance mechanisms in ESCs that may extend to other stem cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dca127bc92b2f97069c67210cae368db947e036c" target='_blank'>
              Embryonic Stem Cell-Specific Responses to DNA Replication Stress
              </a>
            </td>
          <td>
            Ryan C. James, Jerry K. Wang, Siddhanth R. Bhatt, Sophie E. Shadid, Daryl J. Phuong, J. Schimenti
          </td>
          <td>2025-05-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Abstract Background and Objectives Cancer continues to be a predominant cause of mortality worldwide, underscoring the critical need to identify and develop novel biomarkers to improve prognostic accuracy and therapeutic approaches. The dysregulation of ELAVL1 is linked to various diseases, including cancer. Nevertheless, its role across different cancer types remains insufficiently investigated. Methods We conducted a systematic investigation into the expression patterns, prognostic significance, genomic alterations, modifications, and functional implications of ELAVL1 in pan-cancer types. Besides, we performed in vitro and in vivo experiments to confirm the role of ELAVL1 in nasopharyngeal carcinoma (NPC). Results By utilizing multi-omics datasets, we found obvious overexpression of ELAVL1 in various cancer types at both the mRNA and protein levels, with predominant expression in malignant cells. Survival analysis revealed that increased ELAVL1 expression was linked to unfavorable outcomes in certain cancers; however, its effect difers among various cancer types. Additionally, we found that the genomic alterations and modifications of ELAVL1 were related to tumor progression. We discovered that ELAVL1 was elevated in NPC tissues. In addition, survival analysis indicated that NPC patients with higher ELAVL1 expression had worse prognoses. Functional assays demonstrated that ELAVL1 suppression led to decreased proliferation and migration in NPC cell lines. Moreover, ELAVL1 knockdown effectively inhibited NPC progression in the lymph node and lung metastasis models. Conclusions In summary, ELAVL1 exhibits diverse and complex involvement in tumor progression. Targeting it might inhibit tumor progression, making it a promising biomarker and therapeutic target for enhancing cancer treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbb3845989ed113cdceeda02fbe18eb5b7d67ac9" target='_blank'>
              Deciphering the role of ELAVL1: Insights from pan-cancer multiomics analyses with emphasis on nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Jindong Xie, Yi Xie, Wencheng Tan, Yimeng Ye, Xueqi Ou, Xiong Zou, Zhiqing He, Jiarong Wu, X. Deng, Hailin Tang, Longjun He, Kailai Li, Peng Luo, Kunhao Bai, Guoxian Huang, Jianjun Li
          </td>
          <td>2025-04-01</td>
          <td>Journal of Translational Internal Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="INTRODUCTION
Prostate cancer (PCa) is one of the most commonly diagnosed can-cers in men, with a high global incidence. The Meiotic Nuclear Division 1 (MND1) protein is essential for the repair of DNA double-strand breaks during meiosis, but its role in PCa re-mains poorly understood. This study aims to explore the function of MND1 in PCa progression and the mechanism involved.


METHODS
RNA-Seq data from the TCGA and GEO databases were analyzed. Kaplan-Meier (KM) method and χ2 test examined the association between MND1 expression, prognosis, and clinical parameters. PCa cell lines (22RV1 and C4-2) were used for functional assays. CCK-8, EdU, colony formation assay, flow cytometry analysis and xenograft model were used to evaluate the effects of MND1 on PCa cell proliferation in vitro and in vivo.


RESULTS
MND1 expression was significantly upregulated in PCa tissues, particularly in cases with Gleason scores ≥8, and correlated with poorer disease-free survival (DFS) and adverse clinical features. Functionally, elevated MND1 expression promoted PCa cell proliferation both in vitro and in vivo. Mechanistically, MND1 facilitated cell cycle progression from G0/G1 to S phase via activation of the CCNB1/p53 signaling pathway.


CONCLUSION
MND1 promotes prostate cancer progression by facilitating the G0/G1 to S phase transition via the CCNB1/p53 pathway, making it a promising prognostic marker and potential therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60b45c03c95b63c6a22aac8942e528f6deec86c2" target='_blank'>
              MND1 Promotes the Proliferation of Prostate Cancer Cell via the CCNB1/p53 Signaling Pathway.
              </a>
            </td>
          <td>
            Zhongxiang Zhao, Yesong Zou, Qian Lv, Chenxiao Wu, Ke Tang, Fazhong Dai, Jiayao Feng, Hongshen Lai, Wenjie Lai, Xiaofu Qiu
          </td>
          <td>2025-05-08</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Up to 25% of oral cavity squamous cell carcinoma (OCSCC) cases have no significant exposure to tobacco and alcohol. Compared to the rest of the OCSCC population, these cases tend to be older, more female, and display distinct patterns of oral cavity subsite dominance. The underlying molecular pathophysiology in atypical OCSCC cases remains poorly defined. We hypothesized that analysis of OCSCC genomes using single base substitution (SBS) mutational signatures, which computationally discern patterns associated with discrete mutational processes, would reveal discrete subgroups within the OCSCC population.



 We performed an exploratory genomics study using data from The Cancer Genome Atlas (TCGA). Cases included primary oral cavity tumors with whole exome sequencing data. Cases were excluded if oral cavity subsite was not specified or if the tumor had fewer than fifty single bases substitutions. SBS signatures were generated using python packages SigProfilerMatrixGenerator and SigProfilerAssignment with COSMIC SBS Signatures V3.4 as the reference. APOBEC-dominant cases were defined as those where SBS2 and SBS13 accounted for the greatest proportion of mutations relative to the tumor’s total mutational burden.



 A total of 217 OCSCC cases were included. Of these, 135 (62.2%) showed some degree of ABPOEC-related mutational burden, and in 62 (28.6%) cases APOBEC was the dominant mutational process. APOBEC-dominant cases showed greater SBS mutational burden (median difference 2.65 mutations/MB, Wilcoxon P = 2.22e-09). Compared to the rest of the OCSCC population, APOBEC-dominant cases were older in age (mean difference 4.08 years, 95% CI 0.35 - 7.81, T-test P 0.032), more female (46.8% vs 27.8%, OR 2.28, 95% CI 1.18 - 4.40), and had a significantly lower rate of tobacco use at diagnosis (OR 0.47, 95% CI 0.22 - 0.94). APOBEC mutational burden varied by oral cavity subsite (Kruskal-Wallis P = 1.66e-05), with the majority of hard palate and buccal mucosal tumors being APOBEC-dominant (85.7% and 66.7%, respectively) and floor of mouth tumors showing the lowest rate of APOBEC-dominance (16.1%). APOBEC-dominant cases showed a greater degree of mutations in RAC1, EP300, CASP8, and FAT1 (FDR corrected P <0.05). APOBEC dominant cases showed a trend toward a lower rate of TP53 mutations (OR 0.48, 95% CI 0.23 - 1.01).



 In TCGA, ABPOEC-dominant OCSCC cases resemble these atypical populations by being enriched for female sex, older age, lower rates of tobacco use, and certain oral cavity subsites. However, APOBEC-dominance was not exclusive to any clinical characteristic, highlighting the potential importance of molecular class labels in understanding population heterogeneity in OCSCC. Further investigation focused on induction of APOBEC in OCSCC is warranted and may elucidate the pathophysiology of oral cavity cancer beyond tobacco and alcohol exposure.



 Benjamin M. Wahle, Angela L. Mazul, D. Neil Hayes, Jose P. Zevallos, Sidharth V. Puram. APOBEC dominant mutational signature defines distinct oral cavity cancer subgroup [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2288.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ac9f447b3fc81a85d390c97b2bbb0300f8285d6" target='_blank'>
              Abstract 2288: APOBEC dominant mutational signature defines distinct oral cavity cancer subgroup
              </a>
            </td>
          <td>
            B. Wahle, A. Mazul, D. N. Hayes, J. Zevallos, Sidharth V. Puram
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="
 Intratumoral tumor heterogeneity characterized by the presence of genetically and phenotypically distinct cell populations (clones) within the same lesion, represents a significant obstacle to accurate diagnosis, prognosis, and therapeutic targeting based on molecular investigation, when biopsy may yield incorrect results due to insufficient representation of all tumor clones in the sample.
 Liquid biopsy, defined as the detection and quantification of somatic aberrations of circulating tumor DNA (ctDNA) in blood plasma, has long been considered an approach that can be effectively applied to obtain full spectrum of mutations without any bias. This assumption based on condition that the release of ctDNA fragments occurs quantitatively from all present cell populations.
 Our study aims to verify this hypothesis based on continuous liquid biopsies in patients undergoing treatment for advanced solid tumors. In the field of liquid biopsy, the use of ctDNA detection for continuous monitoring of ongoing therapy has recently come to the forefront. The so-called "tumor informed" approach, in which somatic mutations found in tumor tissue are subsequently detected in ctDNA. The procedure is characterized by relatively low costs, which allows repeated examinations based on frequent sampling for continuous monitoring of the disease and treatment. In ctDNA-positive patients, we subsequently perform broad-spectrum longitudinal profiling of mutational clones in ctDNA using an NGS panel based on the principle of hybrid capture of a panel covering a total of 127 genes. This method is used to monitor mutation profiles in ctDNA from plasma of patients with metastatic colorectal cancer undergoing multimodal treatment including surgical resection of metastatic tissue and chemotherapy.
 The results indicate that the response to treatment, which is manifested by a decrease or increase in ctDNA represented by different mutations, may not occur homogeneously in all present mutational clones. In fact, some clonal populations appear to indicate earlier response than others, characterized by shifts in mutation fraction dynamics.


 Petra Kroupova, Barbora Minarikova, Andrea Chmelarova, Marek Minarik. A longitudinal investigation of temporal/intratumoral heterogeneity by plasma NGS profiling of mutation clones in metastatic colorectal cancer [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 3698.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47975e1afc850e6c92f5d1873544c03b447f35c7" target='_blank'>
              Abstract 3698: A longitudinal investigation of temporal/intratumoral heterogeneity by plasma NGS profiling of mutation clones in metastatic colorectal cancer
              </a>
            </td>
          <td>
            Petra Kroupova, Barbora Minarikova, Andrea Chmelarova, Marek Minarik
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Over the past decade, long-read sequencing has evolved into a pivotal technology for uncovering the hidden and complex regions of the genome. Significant cost efficiency, scalability, and accuracy advancements have driven this evolution. Concurrently, novel analytical methods have emerged to harness the full potential of long reads. These advancements have enabled milestones such as the first fully completed human genome, enhanced identification and understanding of complex genomic variants, and deeper insights into the interplay between epigenetics and genomic variation. This mini-review provides a comprehensive overview of the latest developments in long-read DNA sequencing analysis, encompassing reference-based and de novo assembly approaches. We explore the entire workflow, from initial data processing to variant calling and annotation, focusing on how these methods improve our ability to interpret a wide array of genomic variants. Additionally, we discuss the current challenges, limitations, and future directions in the field, offering a detailed examination of the state-of-the-art bioinformatics methods for long-read sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1e2b7afb3f6eb1ebb1086fd39bb7b9d4ff2e777" target='_blank'>
              A Hitchhiker's Guide to long-read genomic analysis.
              </a>
            </td>
          <td>
            M. Mahmoud, D. P. Agustinho, F. Sedlazeck
          </td>
          <td>2025-04-01</td>
          <td>Genome research</td>
          <td>1</td>
          <td>49</td>
        </tr>

        <tr id="BACKGROUND
Hepatocellular carcinoma (HCC) is one of the primary liver tumors with high incidence and mortality. RNA G-quadruplexes (rG4) are nucleic acid structures involved in gene expression and genome duplication. rG4 exerts its function by interacting with rG4-binding proteins. The carboxypyridostatin (cPDS), a specific ligand of rG4, are widely studied in numerous tumors. However, the role of cPDS in HCC and its regulatory mechanisms are not yet fully understood. Our study aimed to discuss the regulatory mode of cPDS on Baculoviral IAP Repeat Containing 3 (BIRC3) expression and its impact on proliferation, apoptosis, and other biologic functions in HCC.


METHODS
We conducted colony formation, CCK8, Edu incorporation, scratch healing, and cell spheroid formation assays to analyze the function of cPDS on cell proliferation and migration. Additionally, we explored the role of cPDS in regulating BIRC3 expression by Western blot and qRT-PCR. Furthermore, we evaluated the impact of BIRC3 on cell proliferation and subcutaneous tumor formation in nude mice. Finally, we analyzed the regulatory mechanisms of cPDS on cell apoptosis by Western blot, qRT-PCR, flow cytometry, and Annexin V-FITC staining.


RESULTS
Our results demonstrated that cPDS inhibited HCC cells proliferation and migration. Moreover, cPDS elevated the mRNA level while inhibiting the protein expression of BIRC3 in HCC cells. Overexpression of BIRC3 significantly enhanced the proliferation of HCC cells. In the nude mice model, BIRC3 significantly increased the tumor volume and weight. Mechanistically, cPDS promoted cell apoptosis via inhibiting BIRC3-mediated anti-apoptotic effect.


CONCLUSION
Our findings revealed a critical role of rG4 ligand cPDS in HCC progression and indicate that cPDS may be used for HCC treatment considering its tumor inhibitory properties by regulating cell apoptosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9048979ec2c41de6795897f31e2e9beeb9214d3a" target='_blank'>
              cPDS Promotes Cell Apoptosis by Reducing the Translational Efficiency of BIRC3 mRNA in HCC.
              </a>
            </td>
          <td>
            Yiheng Liu, Qingqing Liu, Tianyi Huang, Shengjie Zhang, Zicheng Zhou, Chiyu Gu, Cuihua Lu
          </td>
          <td>2025-04-16</td>
          <td>Digestive diseases and sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genome instability is a key driver of malignant progression in cancer and is char-acterized by chromoanagenesis, including spontaneous events, such as chromothripsis, chromoanasynthesis, and chromoplexy. These genome catastrophes create the heterogeneity necessary for tumor cells to adapt, evolve, and resist therapy. Ergodic anticancer therapy rep-resents a novel strategy for targeting cancer stem cells by manipulating their genome chaos. Two approaches have been proposed: ergodynamic anticancer therapy (EDAT), which en-hances genome chaos beyond a critical threshold and leads to self-destruction, and ergostatic anticancer therapy (ESAT), which suppresses chaos and limits malignant progression. This brief review explores the conceptual foundations, molecular mechanisms, and therapeutic potential of ergostatic and ergodynamic therapies in treating cancer, highlighting their role in personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6433d2779e29625955556be47f98de50e3b8dc0a" target='_blank'>
              Ergodic Manipulation of Genome Chaos: Innovative Strategies against Malignant Progression.
              </a>
            </td>
          <td>
            Sergey Shityakov, Viacheslav Kravtsov
          </td>
          <td>2025-04-25</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Synthetic lethality (SL) has emerged as a highly promising therapeutic strategy, gaining significant attention in the field of cancer treatment in recent years. The principle behind SL is that the simultaneous loss or inhibition of two genes results in cell death, while the inhibition of only one gene does not. SL drugs exploit this mechanism by targeting the partner gene of an already inactivated or mutated gene in cancer cells, inducing their death. This approach allows for the precise targeting of genetic vulnerabilities specific to cancer cells, sparing normal tissues from damage. Additionally, SL therapies expand the range of druggable targets, including proteins that are difficult to inhibit with traditional small molecules. By affecting both enzymatic and non-enzymatic protein functions, SL therapies reduce the risk of developing drug resistance, providing an effective option for treating cancers that are resistant to conventional therapies.Kyinno offers an extensive collection of knockout cell lines targeting a range of genes associated with synthetic lethality, including SHLD2, P53, T53BP1, AQR, ATRX, SETD2, ATM, WRN, BRCA1, and BRCA2. These cell lines, developed using the CRISPR/Cas9 genome editing technique and validated through sequencing and western blot analysis, are crucial for investigating the complex mechanisms underlying synthetic lethality and for identifying novel therapeutic strategies in cancer treatment. For example, our knockout lines for BRCA1 and BRCA2 enable researchers to investigate vulnerabilities in homologous recombination repair, paving the way for the development of innovative PARP inhibitors and targeted therapies. Additionally, lines with deletions in ATM and WRN provide essential insights into DNA damage response pathways, which can enhance the effectiveness of DNA-damaging agents. The diverse range of gene knockouts, including those involved in cell cycle regulation, DNA repair, and apoptosis, allows for comprehensive studies into the synergistic effects of dual gene inhibition. By utilizing our extensive library of synthetic lethality cell lines, researchers can effectively validate hypotheses, screen novel drug candidates, and optimize combination therapies to improve outcomes for patients with resistant cancers. We highly recommend these resources for advancing research in synthetic lethality and targeted cancer therapies.


 Yue Huang, Yunpeng Zhai, Yan Feng, Xinyue Jia, Xiaohuan Zhang, Jinying Ning, Feng Hao. CRISPR-edited cell lines targeting key molecular pathways: Accelerating the development of synthetic lethal therapeutics beyond PARP inhibitors [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 4189.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ea7a11f970c3515d6da1945980a0668b04d9e68" target='_blank'>
              Abstract 4189: CRISPR-edited cell lines targeting key molecular pathways: Accelerating the development of synthetic lethal therapeutics beyond PARP inhibitors
              </a>
            </td>
          <td>
            Yue Huang, Yunpeng Zhai, Yan Feng, Xinyue Jia, Xiaohuan Zhang, Jinying Ning, F‐J Hao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4bf206646c787c811c3e18fa137c014843f6fee" target='_blank'>
              Emergence of coordinated cell division during the evolution of multicellularity
              </a>
            </td>
          <td>
            Luis F. Cedeño-Pérez, R. Pineau, Thomas C. Day, W. Ratcliff, Peter L. Conlin
          </td>
          <td>2025-04-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="High-Grade Serous Ovarian Carcinoma (HGSOC) represents the most prevalent and lethal subtype of ovarian cancer, with approximately 225,000 new cases reported globally each year and a five-year survival rate of merely 49.1%. The clinical management of HGSOC encounters substantial challenges, primarily attributable to its intricate drug resistance mechanisms, which involve multiple biological processes, including tumor cell heterogeneity, microenvironment remodeling, gene mutations, and drug efflux. This study systematically reviews the most recent advancements in HGSOC drug resistance research, concentrating on the molecular biological foundations of resistance mechanisms, innovative detection strategies, and potential therapeutic approaches. The research indicates that HGSOC drug resistance constitutes a complex process characterized by multifactorial interactions, involving aberrant cell signaling pathways, dynamic alterations in the tumor microenvironment, and specific expressions of molecular markers. In this review, we systematically analyzed and investigated the intricate biological behaviors associated with HGSOC drug resistance, which not only enhances the understanding of disease progression but also provides essential theoretical foundations for the development of more precise and effective targeted therapies. This review firstly illustrated the detailed drug resistance cellular and molecular mechanisms underlying HGSOC chemotherapy, which can pave the way for future studies in HGSOC drug resistance practices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f295e8c7119ec53d42252f9fd6519f2938abfa0" target='_blank'>
              Unveiling drug resistance pathways in high-grade serous ovarian cancer(HGSOC): recent advances and future perspectives
              </a>
            </td>
          <td>
            Ruiting Fu, Ruiyue Hu, Wenting Li, Xifang Lv, Hanwei Zhao, Fuxia Li
          </td>
          <td>2025-04-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dedcae3bf4b0bbb920237681839ff45066ae6f5" target='_blank'>
              Accurate detection of pathogenic structural variants guided by multi-platform comparison
              </a>
            </td>
          <td>
            Nico Alavi, M. Moeinzadeh, J. Hertzberg, U. Souto Melo, Lion Ward Al Raei, Paolo Infantino, Maryam Ghareghani, Marco Savarese, Stefan Mundlos, Martin Vingron
          </td>
          <td>2025-05-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 e17634


 Background:
 The non-specific molecular profile (NSMP) subtype accounts for 50% of endometrial carcinoma (EC). It's a heterogeneous group of tumors, characterized by minimal cell mutations and low copy number variations, including both aggressive and clinically low-risk ECs.
 Methods:
 In January 2022 our institution launched a comprehensive cancer genome profiling (CGP) (FPG500 IRB approval 3837; NCT06020625) enrolling also EC patients. Oncogenic and likely oncogenic alterations were reported according to OncoKB and classified as Tier I-II-III according to ESCAT classification. The aim of the current analysis was to describe ESCAT gene actionability findings.
 Results:
 From January 1
 st
 2022 to December 31
 st
 2023, 253 patients with any FIGO stage, NSMP ECs were enrolled. Median age was 62 years and the most frequent histotype was endometrioid (96.9%). Estrogen receptors were positive in 94.5%. Overall, 233 patients (92%) had at least one tier III ESCAT alteration. The more frequent variants were found in PTEN (88%), PIK3CA (43%), FGFR2 (15%) and AKT1 (6%). 18% of patients had an ESR1 variant, while a KRAS G12C variant was found in 11% of patients. The majority of PTEN variants were R130X: R130G (40 pts, 17.2%), R130Q (17 pts, 7.3%) and R130* (10 pts, 4.3%). The PIK3CA's more frequently altered hotspots were H1047R (21 pts, 9.0%), E545D/K/Q/A (13 pts, 5.6%) and E542K (9 pts, 3.9%). The most frequent FGFR2 hotspot was S252W (16 pts, 6.9%). Mutually exclusive genes (p<0.05) were AKT and PTEN, PIK3CA and PIK3R1, FGFR2 with CTNNB1 and KRAS. Significant co-occurrence (p<0.05). was found between PTEN and PIK3CA and ARID1A. Considering ESCAT Tiers, the PI3K pathway was altered in 85% of patients.
 Conclusions:
 Our descriptive analysis aligns with existing literature data. Further investigation correlating survival and recurrence rates with molecular findings could unveil prognostic subgroups and potentially targetable variants.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f3dd3543e71d5bb80edc693e6a23eaed8a00bd4" target='_blank'>
              A monocentric analysis of ESCAT gene actionability detection in non-specific molecular profile (NSMP) endometrial cancer.
              </a>
            </td>
          <td>
            Floriana Camarda, Luca Mastrantoni, C. Parrillo, T. Pasciuto, M. Manfredelli, R. Trozzi, Luciano Giacò, F. Persiani, Iolanda Mozzetta, A. Minucci, Elisa De Paolis, E. Perrone, Valentina Iacobelli, Gian Franco Zannoni, F. Fanfani, G. Scambia, C. Nero
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="
 Large scale efforts using CRISPR based gene loss-of-function screens have produced comprehensive catalogs of gene essentiality in hundreds of well annotated cancer models. The equivalent catalogs have not been generated for gene gain-of-function. While loss-of-function (LOF) mutations in TP53 are frequent in cancer, targeting these mutations pharmaceutically remains challenging. To explore synthetic lethalities with p53 LOF we performed genome-wide CRISPR activation screens in wildtype and TP53 knockout isogenic cancer cell lines. We discovered a cluster of genes associated with oxidative phosphorylation (OxPhos) whose overexpression were synthetic lethal with p53 loss. This is consistent with the literature describing a metabolic switch from OxPhos to glycolysis after p53 mutation/loss. Our screen results suggest that this metabolic switch could be a protective action for cancer cells with mutant p53 if OxPhos activation is synthetic lethal with functional p53 deficiency. Indeed we found that overexpression of two of the top OxPhos gene hits from the screen, namely ADCK3 and SCO2, were synthetic lethal with p53 loss in vitro. Moreover, overexpression of ADCK3 resulted in significant growth inhibition of tumors with p53 loss in vivo. Mechanistically, overexpression of either ADCK3 or SCO2 significantly induced cellular ROS production and mitochondrial DNA damage in cancer cells with p53 loss. Our results suggest a model that wildtype p53 is important for maintaining the homeostasis between OxPhos and glycolysis. In the absence of p53 cells cannot tolerate the ROS generated by OxPhos and they switch to glycolytic metabolism. Our results are consistent with the model that hyperactivation of ADCK3, a critical component for electron transport chain, results in an increase in OxPhos and subsequently ROS which in the context of p53 deficient cells is lethal. Similar to p53 deficient cells, cancer cells with hotspot TP53 mutations have been reported to undergo a similar OxPhos to glycolysis metabolic shift and are deficient in the repair of mitochondrial DNA damage caused by OxPhos generated ROS. Our results support that an ADCK3 agonist therapeutic could be an effective strategy to target mutant p53 cancers.


 Jianhuang Lin, Dimitra Georgopoulou, Shuzhen Wu, Svaty Sachania, Rebecca Saunders, Biju Mangatt, Geeta Sharma, Euan Stronach, Anthony Mazurek, Fiona M. Behan, Benjamin Schwartz, Brendan A. Veeneman. Loss of p53 sensitizes cancer cells to oxidative phosphorylation agonism [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 2993.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52e12e4fc2e2ae45882770e50fef3981ce0d4239" target='_blank'>
              Abstract 2993: Loss of p53 sensitizes cancer cells to oxidative phosphorylation agonism
              </a>
            </td>
          <td>
            Jianhuang Lin, Dimitra Georgopoulou, Shuzhen Wu, Svaty Sachania, Rebecca Saunders, Biju Mangatt, Geeta Sharma, Euan Stronach, Anthony Mazurek, Fiona M. Behan, Benjamin Schwartz, Brendan A. Veeneman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Heterozygous inversion breakpoints inhibit the formation of chiasmata in their vicinity, and it has been suggested that they do so through the same mechanism that also causes interference between chiasmata. In this paper, I therefore extend my earlier model of chiasma interference to account for interference between breakpoints and chiasmata in pericentric inversion heterokaryotypes. Using this model to analyze recombination and sterility datasets for Drosophila melanogaster, I find support for the hypothesis that inversion breakpoints interfere with chiasmata in the same way and to the same degree that other chiasmata do. I also find that breakpoints, like chiasmata, appear to show negative interference in the pericentromeric region, and positive interference elsewhere. I discuss the implications of these findings in light of the recent HEI10 coarsening interference hypothesis, and conclude with some remarks about the evolutionary origin of chiasma interference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2176d5dcc00779b3e3004169c06a0d6857d91384" target='_blank'>
              Breakpoint–chiasma interference in pericentric inversion heterokaryotypes
              </a>
            </td>
          <td>
            Øystein Kapperud
          </td>
          <td>2025-05-06</td>
          <td>Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/947370cc2d12c5cdc69fc5d3361f46b97266d6c4" target='_blank'>
              Deconstructing the Cancer Epigenome using Reduced Representation Bisulfite Sequencing (RRBS)
              </a>
            </td>
          <td>
            Nicole Glendinning, Dongjun Chung, Gary Hardiman
          </td>
          <td>2025-05-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 The BAF (SWI/SNF) chromatin remodeling complex comprises of two mutually exclusive ATPases, SMARCA2 (BRM) and SMARCA4 (BRG1), that affect the mobilization and positioning of nucleosomes on DNA and thereby regulates important cellular functions including transcription, DNA recombination, DNA repair and chromosome decatenation during mitosis. SMARCA4 is frequently overexpressed in several types of cancers. Overexpression has been linked to increased proliferation and survival, as well as aggressive tumors and poor prognosis. SMARCA4 knockdown in these tumors lead to inhibition of proliferation and increased sensitivity to known chemotherapeutic agents, supporting the validity of targeting SMARCA4. In addition, PTEN and SMARCA4 appear to share synthetic lethality relationship based on reported in vitro and in vivo knockdown studies. Considering this well-established biological rationale, selective degradation of SMARCA4 should offer significant clinical benefit in multiple disease settings. Here, we report first-in-class SMARCA4 selective degraders that demonstrate distinct cellular phenotype. As part of the initial design plan, selective SMARCA2/4 Bromodomain inhibitors and specific ligands of several E3 ligases were chosen to arrive at different degrader designs. Several linkers and different exit vectors were considered to construct a variety of hetero-bifunctional degrader molecules. Our proprietary ternary complex modeling algorithm, ALMOND4 (ALgorithm for MOdeling Neosubstrate Degraders) helped in prioritizing the designs. Shortlisted compounds were synthesized and profiled in multiple cellular assays to understand their potential to degrade SMARCA4 selectively. These compounds exhibited very potent anti-proliferative activity in cell lines of hematological and prostate cancer origin (e.g. MV-4-11, VCaP, etc.) whereas no significant potency is observed in SMARCA4 mutant cell lines. Identified compounds have shown selective degradation of SMARCA4 over SMARCA2 in multiple cell lines. Few potent SMARCA4 selective degraders were optimized for pharmacokinetic properties and further evaluation of tolerability and efficacy in rodents is in progress.


 Chandrasekhar Abbineni, Kiran Aithal, Bilash Kuila, Sandeep Vitthal Dukare, Arya Desai, Megha Goyal, Khaji Abdul Rawoof, Gauri Rahul Petkar, Gopinath CH, T Jagadeesh Kumar, Charamanna KB, Rahul B. Chavan, Nandish C, Suraj Tgore, Leena Khare, DS Samiulla, Subhendu Mukherjee, Thomas Antony, Kavitha Nellore, Sanjeev Giri, Rajesh Eswarappa, Girish Daginakatte, Susanta Samajdar, Murali Ramachandra. Identification of first-in-class paralog selective SMARCA4 degraders for treatment of hematological and prostate malignancies [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 1 (Regular Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_1):Abstract nr 7019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b258d63d3d2f73aa514ce2ae24163ec83da23207" target='_blank'>
              Abstract 7019: Identification of first-in-class paralog selective SMARCA4 degraders for treatment of hematological and prostate malignancies
              </a>
            </td>
          <td>
            C. Abbineni, K. Aithal, Bilash Kuila, S. Dukare, Arya Desai, Megha Goyal, Khaji Abdul Rawoof, Gauri Rahul Petkar, Gopinath Ch, T. J. Kumar, Charamanna Kb, Rahul B. Chavan, Nandish C, Suraj Tgore, Leena Khare, D. Samiulla, Subhendu Mukherjee, T. Antony, K. Nellore, Sanjeev Giri, R. Eswarappa, G. Daginakatte, S. Samajdar, Murali Ramachandra
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Nanopore long-read sequencing has expanded the capacity of long-range, single-base, and single-molecule DNA-methylation (DNAme) detection and haplotype-aware allele-specific epigenetic phasing. Previously, we benchmarked and ranked the robustness of seven computational tools for DNAme detection using nanopore sequencing. Overall, the top performers were Megalodon, Nanopolish, DeepSignal and Guppy. However, these algorithms exhibit lower performance at regions with discordant non-singleton DNAme patterns, intergenic regions, low CG density regions, and repetitive regions compared to genome-wide region. Furthermore, long-read sequencing analysis of mammalian genomes requires higher computational resources than next-generation sequencing. To address these issues, we developed NANOME, the first cloud-compatible, Nextflow-based container environment for consensus DNAme detection. We designed a consensus DNAme predictive model using XGBoost, which integrates the output of Megalodon, Nanopolish, and Deepsignal for analyzing data obtained using Oxford Nanopore Technologies (ONT). The consensus model enhanced DNAme detection at single-base resolution by 12% (mean square error) and improved accuracy at single-molecule resolution by 3% (F1-score accuracy) for human B-lymphocyte cell lines (NA12878 and NA19240). The consensus model also detected ∼200,000 more CpGs than Nanopolish. Combing variant calling and long-read phasing, NANOME can detect haplotype-aware allele-specific DNAme in known imprinting controls. We conducted haplotype-aware methylation detection on the T2T genome for dataset NA12878, revealing significant variations in DMR density between gap and non-gap regions, with notable higher densities in gap regions of chromosomes 2, 10, and 20. Overall, NANOME represents a significant step forward in DNAme detection and long-range epigenetic phasing, offering a robust and accessible tool for researchers studying the epigenome.


 Yang Liu, Ziwei Pan, Christina Chatzipantsiou, Emma Wade, Thatcher Slocum, Lasya Karuturi, Yue Zhao, Shilpita Karmakar, Sheng Li. NANOME: A Nextflow pipeline for haplotype-aware allele-specific consensus DNA methylation detection by nanopore long-read sequencing [abstract]. In: Proceedings of the American Association for Cancer Research Annual Meeting 2025; Part 2 (Late-Breaking, Clinical Trial, and Invited Abstracts); 2025 Apr 25-30; Chicago, IL. Philadelphia (PA): AACR; Cancer Res 2025;85(8_Suppl_2):Abstract nr LB345.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92c251207e4077b875b3ec36d1c9e930b4eeaeed" target='_blank'>
              Abstract LB345: NANOME: A Nextflow pipeline for haplotype-aware allele-specific consensus DNA methylation detection by nanopore long-read sequencing
              </a>
            </td>
          <td>
            Yang Liu, Ziwei Pan, Christina Chatzipantsiou, Emma Wade, Thatcher Slocum, Lasya Karuturi, Yue Zhao, Shilpita Karmakar, Sheng Li
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [7],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>