<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-01-20 09:25:16 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>27</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>64</td>
          <td>134</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>104</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>308</td>
          <td>134</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>265</td>
          <td>157</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f6cf05d6f649a55a948cbad9eac9ac4b6467ec" target='_blank'>
              Modern biology of extrachromosomal DNA: A decade-long voyage of discovery
              </a>
            </td>
          <td>
            Qing-Lin Yang, Yipeng Xie, Kailiang Qiao, Jun Yi Stanley Lim, Sihan Wu
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a prevalent driver of cancer, whose random segregation promotes aggressive tumors. Acentric ecDNAs attach to chromosomes during mitosis for segregation. However, the molecular mechanism governing ecDNA-chromosome mitotic interactions remains poorly understood. This study shows that ecDNAs attach to histone 3 lysine 27 acetylation (H3K27ac)-marked chromatin during mitosis. H3K27ac depletion resulted in ecDNA detachment from mitotic chromosomes. Diverse bromodomain proteins, which are known readers of H3K27ac, stabilize ecDNAs’ mitotic interaction, exhibiting context-dependent and mutually complementary roles. Furthermore, disruptions of the Mediator complex and RNA polymerase II transcription activity both dissociate ecDNAs from mitotic chromosomes, suggesting that the transcription machinery mediates ecDNA segregation. Mis-segregated ecDNAs were expelled into the cytosol and degraded, leading to diminished oncogene expression and a reversal of therapy resistance. Our research provides new insights into the interplay between RNA transcription and acentric ecDNA inheritance in cancer, offering a novel avenue for disrupting ecDNA-driven oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b99e11fc3712d503e0a3c800fc35c4668a35ad3" target='_blank'>
              Transcription Machinery Anchors ecDNAs to Mitotic Chromosomes for Segregation
              </a>
            </td>
          <td>
            Yipeng Xie, Jun Yi Stanley Lim, Wenyue Liu, C. Gilbreath, Yoon Jung Kim, Sihan Wu
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e5d3752411db0d4a1f1f0e0c0f053217196ee" target='_blank'>
              Unveiling the mysteries of extrachromosomal circular DNA: from generation to clinical relevance in human cancers and health
              </a>
            </td>
          <td>
            Zilong Wang, Jiaying Yu, Wenli Zhu, Xiaoning Hong, Zhen Xu, Shuang Mao, Lei Huang, Peng Han, Chunxiao He, Changze Song, Xi Xiang
          </td>
          <td>2024-12-20</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="LINE-1 (L1) retrotransposition is widespread in many cancers, especially those with a high burden of chromosomal rearrangements. However, whether and to what degree L1 activity directly impacts genome integrity is unclear. Here, we apply whole-genome sequencing to experimental models of L1 expression to comprehensively define the spectrum of genomic changes caused by L1. We provide definitive evidence that L1 expression frequently and directly causes both local and long-range chromosomal rearrangements, small and large segmental copy-number alterations, and subclonal copy-number heterogeneity due to ongoing chromosomal instability. Mechanistically, all these alterations arise from DNA double-strand breaks (DSBs) generated by L1-encoded ORF2p. The processing of ORF2p-generated DSB ends prior to their ligation can produce diverse rearrangements of the target sequences. Ligation between DSB ends generated at distal loci can generate either stable chromosomes or unstable dicentric, acentric, or ring chromosomes that undergo subsequent evolution through breakage-fusion bridge cycles or DNA fragmentation. Together, these findings suggest L1 is a potent mutagenic force capable of driving genome evolution beyond simple insertions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45b6d0c6b40e4e17794e34761d2e6492abd74c31" target='_blank'>
              Chromosomal rearrangements and instability caused by the LINE-1 retrotransposon
              </a>
            </td>
          <td>
            Carlos Mendez-Dorantes, Xi Zeng, Jennifer A. Karlow, Phillip Schofield, Serafina Turner, Jupiter Kalinowski, Danielle Denisko, E. A. Lee, Kathleen H. Burns, Cheng-Zhong Zhang
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1054dfecee5e5157b77728bea8bb0a9b8823603" target='_blank'>
              Single-cell multiomics analysis reveals dynamic clonal evolution and targetable phenotypes in acute myeloid leukemia with complex karyotype
              </a>
            </td>
          <td>
            Aino-Maija Leppä, Karen Grimes, Hyobin Jeong, Frank Y Huang, Alvaro Andrades, A. Waclawiczek, T. Boch, Anna Jauch, S. Renders, Patrick Stelmach, Carsten Müller-Tidow, Darja Karpova, Markus Sohn, F. Grünschläger, Patrick Hasenfeld, Eva Benito Garagorri, Vera Thiel, A. Dolnik, Bernardo Rodriguez-Martin, L. Bullinger, Krzysztof Mrózek, Ann-Kathrin Eisfeld, Alwin Krämer, A. Sanders, J. Korbel, A. Trumpp
          </td>
          <td>2024-11-25</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. Using high resolution Micro-Capture-C in primary patient samples, we also identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may also be a mechanism that drives transcriptional heterogeneity in cancer more generally. Key Points Leukemia patients with the same driver mutations often display gene expression differences Using chromatin profiling and high resolution 3C methods we show that enhancer heterogeneity drives gene expression differences">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1af97c7a783137350d56568e2bae7a8a0304b710" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression between patients
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine M Chahrour, Kim Sharp, N. Elliott, Joe R Harman, T. Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W. Stam, Nicholas Crump, James Oliver Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2024-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="All cells possess mechanisms to maintain and replicate their genomes, whose integrity and transmission are constantly challenged by DNA damage and replication impediments. In eukaryotes, the protein kinase Ataxia-Telangiectasia and Rad3-related (ATR), a member of the phosphatidylinositol 3-kinase-like family acts as a master regulator of the eukaryotic response to DNA injuries, ensuring DNA replication completion and genome stability. Here we aimed to investigate the functional relevance of the ATR homolog in the DNA metabolism of Leishmania major, a protozoan parasite with a remarkably plastic genome. CRISPR/cas9 genome editing was used to generate a Myc-tagged ATR cell line (mycATR), and a Myc-tagged C-terminal knockout of ATR (mycATRΔC-/-). We show that the nuclear localisation of ATR depends upon its C-terminus. Moreover, its deletion results in single-stranded DNA accumulation, impaired cell cycle control, increased levels of DNA damage, and delayed DNA replication restart after replication stress. In addition, we show that ATR plays a key role in maintaining L. major’s unusual DNA replication program, where larger chromosomes duplicate later than smaller chromosomes. Our data reveals loss of the ATR C-terminus promotes the accumulation of replication signal around replicative stress fragile sites, which are enriched in larger chromosomes. Finally, we show that these alterations to the DNA replication program promote chromosome instability. In summary, our work shows that ATR acts to moderate DNA replication timing thus limiting the plasticity of the Leishmania genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3499060635df2f56eb484fb5207c0367f3df1797" target='_blank'>
              ATR, a DNA damage kinase, modulates DNA replication timing in Leishmania major
              </a>
            </td>
          <td>
            Gabriel L. A. da Silva, J. Damasceno, J. A. Black, R. McCulloch, L. Tosi
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56f0a75813c454a5e8e9bd04b719f1b32b01034b" target='_blank'>
              ZIC1 is a context-dependent medulloblastoma driver in the rhombic lip
              </a>
            </td>
          <td>
            John J. Y. Lee, Ran Tao, Z. You, P. Haldipur, Anders W Erickson, H. Farooq, Liam D Hendriske, Namal Abeysundara, Cory Richman, Evan Y Wang, Neha Das Gupta, Jennifer L. Hadley, Melissa Batts, Christopher W. Mount, Xiaochong Wu, Alexandra Rasnitsyn, Swneke Bailey, F. Cavalli, Sorana A. Morrissy, Livia Garzia, K. Michealraj, Abhi Visvanathan, Vernon Fong, Jonelle Palotta, Raúl A. Suárez, Bryn Livingston, Miao Liu, B. Luu, Craig Daniels, J. Loukides, A. Bendel, P. French, J. Kros, Andrey Korshunov, Marcel Kool, F. Chico Ponce de León, M. Perezpeña-Diazconti, Boleslaw Lach, Sheila K Singh, S. Leary, Byung-Kyu Cho, Seung-Ki Kim, Kyu-Chang Wang, Ji-Yeoun Lee, Teiji Tominaga, William A. Weiss, Joanna J Phillips, Shizhong Dai, Gelareh Zadeh, Ali G Saad, L. Bognár, Á. Klekner, Ian F. Pollack, Ronald L Hamilton, Young-Shin Ra, W. Grajkowska, M. Perek-Polnik, Reid C. Thompson, Anna M. Kenney, Michael K. Cooper, S C Mack, Nada Jabado, Mathieu Lupien, Marco Gallo, Vijay Ramaswamy, M. Suvà, Hiromishi Suzuki, K. Millen, L. F. Huang, P. Northcott, Michael D. Taylor
          </td>
          <td>2025-01-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/865e3224ea8530a73a268350cf8b48819bfe3161" target='_blank'>
              Inter-chromosomal transcription hubs shape the 3D genome architecture of African trypanosomes
              </a>
            </td>
          <td>
            Claudia Rabuffo, Markus R Schmidt, Prateek Yadav, Pin Tong, Roberta Carloni, Anna Barcons-Simon, Raúl O. Cosentino, Stefan Krebs, Keith R. Matthews, Robin C. Allshire, Nicolai Siegel
          </td>
          <td>2024-12-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e77d5a82f46b1391391ade8c249b0d27a2b059b" target='_blank'>
              Comprehensive genomic characterization of early-stage bladder cancer
              </a>
            </td>
          <td>
            F. Prip, P. Lamy, S. Lindskrog, T. Strandgaard, I. Nordentoft, K. Birkenkamp-Demtröder, N. Birkbak, Nanna Kristjánsdóttir, Asbjørn Kjær, T. Andreasen, J. Ahrenfeldt, J. S. Pedersen, A. Rasmussen, Gregers G Hermann, K. Mogensen, Astrid C Petersen, Arndt Hartmann, M. Grimm, Marcus Horstmann, R. Nawroth, U. Segersten, D. Sikic, K. V. van Kessel, E. Zwarthoff, Tobias Maurer, Tatjana Simic, Per-Uno Malmström, N. Malats, J. B. Jensen, Francisco X. Real, L. Dyrskjøt
          </td>
          <td>2025-01-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Characterization of tumor epigenetic aberrations is integral to understanding the mechanisms of tumorigenesis and provide diagnostic, prognostic, and predictive information of high clinical relevance. Among the different tumor-associated epigenetic signatures, 5 methyl-cytosine (5mC) and 5-hydroxymethylcytosine (5hmC) are the two most well-characterized DNA methylation alterations linked to cancer pathogenesis. 5hmC has a tissue-specific distribution and its abundance is subjected to changes in tumor DNA, making it a promising biomarker. Detecting tumor-related DNA methylation alterations in tissues is highly invasive, while the analysis of the cell-free DNA (cfDNA) is poised to supplement, if not replace, surgical biopsies. Despite many studies attempted to identify new epigenetic targets for liquid biopsy assays, little is known about the regulatory roles of 5hmC, its impacts on the molecular phenotypes in tumors. Most importantly, whether the oncogenic-associated 5hmC signatures found in tumor tissues can be recapitulated in patient cfDNA. In this study, we performed the unbiased and simultaneous detection of 5mC and 5hmC whole-genome DNA modifications at base-resolution from two distinct cancer cohorts, from patients with bladder cancer or B-Cell lymphoma, their corresponding normal tissues, and cfDNAs from plasma. We analyzed tissue-specific methylation patters and searched for signatures in gene coding and regulatory regions linked to cancerous states. We then looked for methylation signatures in patients cfDNA to determine if they were consistent with the tumor-specific patterns. We determined the functional significance of 5hmC in tissue specific transcription and uncovered hundreds of tumor-associated 5hmC signatures. These tumor-associated 5hmC changes, particularly in genes and enhancers, were functionally significant in tumorigenesis pathways and correlated with tumor specific gene expression. To investigate if cfDNA is a faithful surrogate for tumor-associated 5hmC, we devised a targeted capture strategy to examine the alterations of 5hmC in cfDNA from patients with bladder cancer and lymphoma with sufficient sensitivity and specificity and confirmed that they recapitulated the patterns we observed in tumor tissues. Our results provide analytic validation of 5hmC as a cancer-specific biomarker. The methods described here for systematic characterization of 5hmC at functional elements open new avenues to discover epigenetic markers for non-invasive diagnosis, monitoring, and stratifying cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a233a1b3a2e680f52000d36e68abfc546f2d4b33" target='_blank'>
              Global DNA methylomes reveal oncogenic-associated 5-hydroxylmethylated cytosine (5hmC) signatures in the cell-free DNA of cancer patients
              </a>
            </td>
          <td>
            G. E. Rech, A. C. Lau, R. L. Goldfeder, R. Maurya, A. V. Danilov, C.-L. Wei
          </td>
          <td>2025-01-15</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute myeloid leukemias (AMLs) have an overall poor prognosis with many high-risk cases co-opting stem cell gene regulatory programs, yet the mechanisms through which this occurs remain poorly understood. Increased expression of the stem cell transcription factor, MECOM, underlies one key driver mechanism in largely incurable AMLs. How MECOM results in such aggressive AML phenotypes remains unknown. To address existing experimental limitations, we engineered and applied targeted protein degradation with functional genomic readouts to demonstrate that MECOM promotes malignant stem cell-like states by directly repressing pro-differentiation gene regulatory programs. Remarkably and unexpectedly, a single node in this network, a MECOM-bound cis-regulatory element located 42 kb downstream of the myeloid differentiation regulator CEBPA, is both necessary and sufficient for maintaining MECOM-driven leukemias. Importantly, targeted activation of this regulatory element promotes differentiation of these aggressive AMLs and reduces leukemia burden in vivo, suggesting a broadly applicable differentiation-based approach for improving therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c3c42a2a7df7281b8834a5ccf4f4a2d74201a60" target='_blank'>
              CEBPA repression by MECOM blocks differentiation to drive aggressive leukemias
              </a>
            </td>
          <td>
            Travis J. Fleming, Mateusz Antoszewski, Sander Lambo, Michael Gundry, Riccardo Piussi, Lara Wahlster, Sanjana Shah, Fiona E. Reed, Kevin D. Dong, Joao A. Paulo, Steve P Gygi, C. Mimoso, Seth R Goldman, Karen Adelman, Jennifer A. Perry, Y. Pikman, K. Stegmaier, Maria N. Barrachina, Kellie R. Machlus, Volker Hovestadt, A. Arruda, Mark D. Minden, Richard A. Voit, V. G. Sankaran
          </td>
          <td>2024-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Human papillomavirus (HPV) integration has been implicated in transforming HPV infection into cancer. To resolve genome dysregulation associated with HPV integration, we performed Oxford Nanopore long-read sequencing on 72 cervical cancer genomes from an Ugandan dataset that was previously characterized using short-read sequencing. We found recurrent structural rearrangement patterns at HPV integration events, which we categorized as: del(etion)-like, dup(lication)-like, translocation, multibreakpoint, or repeat region integrations. Integrations involving amplified HPV-human concatemers, particularly multibreakpoint events, frequently harbored heterogeneous forms and copy numbers of the viral genome. Transcriptionally active integrants were characterized by unmethylated regions in both the viral and human genomes downstream from the viral transcription start site, resulting in HPV-human fusion transcripts. In contrast, integrants without evidence of expression lacked consistent methylation patterns. Furthermore, whereas transcriptional dysregulation was limited to genes within 200 kilobases of an HPV integrant, dysregulation of the human epigenome in the form of allelic differentially methylated regions affected megabase expanses of the genome, irrespective of the integrant's transcriptional status. By elucidating the structural, epigenetic, and allele-specific impacts of HPV integration, we provide insight into the role of integrated HPV in cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a8457c68dc264c60760742af32b0cda2346039" target='_blank'>
              Rearrangements of viral and human genomes at human papillomavirus integration events and their allele-specific impacts on cancer genome regulation.
              </a>
            </td>
          <td>
            Vanessa L. Porter, Michelle Ng, Kieran O’Neill, Signe MacLennan, R. Corbett, Luka Culibrk, Zeid Hamadeh, Marissa Iden, Rachel Schmidt, Shirng-Wern Tsaih, C. Nakisige, Martin Origa, Jackson Orem, Glenn Chang, Jeremy Fan, K. Nip, Vahid Akbari, Simon K. Chan, James Hopkins, Richard A. Moore, E. Chuah, K. Mungall, A. Mungall, Inanҫ Birol, S. Jones, Janet S. Rader, M. Marra
          </td>
          <td>2024-12-05</td>
          <td>Genome research</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a highly metastatic pediatric cancer arising from the neural crest lineage. Genetic amplification of the MYCN proto-oncogene is a defining feature of NB, present in about 20% of all cases. The let-7 tumor suppressor microRNA targets the 3’ UTR of MYCN mRNA. We previously demonstrated that the 3’ UTR of MYCN mRNA acquires the ability to sequester let-7 in MYCN-amplified (MA) disease, thus inhibiting its function. This work established that a noncoding element within an oncogenic mRNA can contribute independently to disease pathology and genetic patterning. To further investigate the roles of noncoding RNA elements within the MYCN mRNA, we engineered cells expressing either MYCN-ORF (MYCN open reading frame only), MYCN-GL (full-length MYCN mRNA from the intact genetic locus), and Null-GL (full-length MYCN mRNA variant where EGFP replaces MYCN protein). We observe that all constructs enhance growth compared to controls in vitro and in vivo. MYCN-GL-expressing cells displayed the most robust growth in vitro despite containing multiple regulatory RNA elements. Remarkably, the Null-GL construct induces cells to grow as fast or faster than MYCN-ORF-expressing cells. Animal studies further confirmed these observations, where the Null-GL-driven tumors had the highest incidence and lowest latency, followed by MYCN-GL and then MYCN-ORF. Further, through NGS analysis, let-7, miR-101, and miR-34a targets are enriched in both MYCN-GL and Null-GL expressing cells. Thus, the 3’UTR of MYCN, which is also targeted by these microRNAs, may interact with them in MYCN-GL and Null-GL cells to deliver a protective effect for the mRNA targets of these microRNAs. We also observed more dynamic differential gene expression in the -GL constructs than in ORF and GFP-expressing cells. In addition, MYCN-GL and Null-GL expressing cells are similarly enriched in gene ontology pathways for cancer, RNA metabolism, and microRNA processing pathways as compared to ORF and GFP. Whole genome sequencing also revealed more similarities in copy number variation in MYCN-GL and Null-GL than in ORF and GFP, suggesting that these constructs may provide selective pressure to favor specific CNV patterns. These observations show that full-length MYCN mRNA containing noncoding regulatory elements are more robust drivers of cell growth and oncogenicity than MYCN protein alone and provide insights into the mechanisms of oncogenic contribution. These results open an exciting door for our understanding of NB pathology and genetic patterning and have broad implications for other oncogene-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c9a5c9ad241e7424dc126cfe9aacbf830aa7c8" target='_blank'>
              Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            Vishwa Patel, Lorraine-Rana E. Benhamou, John T. Powers
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Phenotypic plasticity, the ability of cells to adapt their behavior in response to genetic or environmental changes, is a fundamental biological process that drives cellular diversity in both normal and pathological contexts, including in tumor evolution. While chromosomal instability and somatic copy number alterations (CNAs) are known to influence cellular states, it remains difficult to separate genetic from cell non-autonomous mechanisms that govern transcriptional variability. Here, we present Echidna, a Bayesian hierarchical model that integrates single-cell RNA sequencing (scRNA-seq) and bulk whole-genome sequencing (WGS) data to quantify the impact of CNAs on gene expression dynamics. By jointly inferring clone-specific CNA profiles and uncovering clonal dependencies, Echidna bridges genomic and transcriptomic landscapes within and across multiple time points, enabling the decoupling of gene dosage effects from cell-extrinsic factors on phenotypic plasticity. Applying Echidna to patient tumor specimens, we demonstrate its superior performance in clonal reconstruction and derive insights into resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b10523ab303976a5ed9fe5bc9d69a9a688fccfd9" target='_blank'>
              Echidna: A Bayesian framework for quantifying gene dosage effect impacting phenotypic plasticity
              </a>
            </td>
          <td>
            Joy Linyue Fan, Mingxuan Zhang, William O’Brien, Joshua D. Myers, Johannes C. Melms, Jana Biermann, Edridge K. D’Souza, Somnath Tagore, Nicolas Beltran-Velez, Kevin Hoffer-Hawlik, Alexander Preau, Isha Arora, Sharanya Chatterjee, Benjamin Izar, Elham Azizi
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Human gene expression is controlled from distance via enhancers, which can form longer ‘super-enhancer’-regions of intense regulatory activity. Whether super-enhancers constitute a separate regulatory paradigm remains unclear, largely due to the difficulty of dissecting the contributions and interactions of individual elements within their natural chromosomal context. To address this challenge, we developed enhancer scrambling, a high-throughput strategy to generate stochastic inversions and deletions of targeted enhancer regions by combining CRISPR prime editing insertion of symmetrical loxP sites with Cre recombinase-induced rearrangements. We applied our approach to dissect a distal super-enhancer of the OTX2 gene, generating up to 134 alternative regulatory configurations in a single experiment, and establishing how they drive gene expression and chromatin accessibility, as well as the individual contributions of its elements to this activity. Surprisingly, the presence of the sequence containing a single DNase I hypersensitive site predominantly controls OTX2 expression. Our findings highlight that enhancer-driven regulation of some highly expressed, cell-type-specific genes can rely on an individual element within a cluster of non-interacting, dispensable components, and suggest a simple functional core to a subset of super-enhancers. The targeted randomisation method to scramble enhancers can scale to resolve many super-enhancers and human gene regulatory landscapes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cecb86ee79d0fa4af20cbd670e19b80d640e006e" target='_blank'>
              Resolution of a human super-enhancer by targeted genome randomisation
              </a>
            </td>
          <td>
            Jonas Koeppel, Pierre Murat, Gareth Girling, Elin Madli Peets, Mélanie Gouley, Valentin Rebernig, Anant Maheshwari, Jacob Hepkema, Juliane Weller, Jenie Hannah Johnkingsly Jebaraj, Ronnie Crawford, Fabio Giuseppe Liberante, Leopold Parts
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a2ef81377921558c83081b12d35bab26833147b" target='_blank'>
              Extrachromosomal circular DNA containing DTX1 promotes cell growth in hydroquinone-induced malignantly transformed cells by regulating the transcription of DTX1
              </a>
            </td>
          <td>
            X. Ling, Qunfang Jiao, Daifan Lin, Jialong Chen, Yali Han, Jinxue Meng, Bohuan Zhong, He Zhang, Gongda Zhang, Fangling Zhu, Jiheng Qin, Yongdui Ruan, Linhua Liu
          </td>
          <td>2024-11-25</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are four-stranded alternative secondary structures formed by guanine-rich nucleic acids and are prevalent across the human genome. G4s are enzymatically resolved using specialized helicases. Previous in vitro studies showed that DEAH-box Helicase 36 (DHX36/G4R1/RHAU), has the highest specificity and affinity for G4 structures. Here, by mapping genome-wide DNA double-strand breaks (DSBs), we demonstrate that knockout (KO) of DHX36 helicase increases DSB enrichment at G4 sites and that the presence of the G4 motif is a significant mediator of genome instability at regulatory regions. The loss of DHX36 corresponds with the significant upregulation of NF-κB transcriptional programs, culminating in the production and secretion of proinflammatory cytokines. Loss of DHX36 expression results in an increase in the innate immune signaling stimulator of interferon response cGAMP interactor 1 (STING1) expression and activation of genes involved in immune response pathways. Importantly, higher levels of DHX36 mRNA expression in human B-cell acute lymphoblastic leukemia correlate with improved overall survival relative to lower expression of DHX36, highlighting its critical role in preserving genome integrity at a cellular level and in the context of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f6a2067e514e41d31ed3af27971fbc66a915c36" target='_blank'>
              Loss of DHX36/G4R1, a G4 resolvase, drives genome instability and regulates innate immune gene expression in cancer cells
              </a>
            </td>
          <td>
            Anna R Bartosik, Pei-Chi Hou, James P. Vaughn, P. J. Smaldino, A. Ratan, Marty W. Mayo, Wang Yuh-Hwa
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="A significant number of castration-resistant prostate cancer (CRPC) evolve into a neuroendocrine (NE) subtype termed NEPC, leading to resistance to androgen receptor (AR) pathway inhibitors and poor clinical outcomes. Through Hi-C analyses of a panel of patient-derived xenograft tumors, here we report drastically different 3D chromatin architectures between NEPC and CRPC samples. Such chromatin re-organization was faithfully recapitulated in vitro on isogenic cells undergoing NE transformation (NET). Mechanistically, neural transcription factor (TF) NKX2-1 is selectively and highly expressed in NEPC tumors and is indispensable for NET across various models. NKX2-1 preferentially binds to gene promoters, but it interacts with chromatin-pioneering factors such as FOXA2 at enhancer elements through chromatin looping, further strengthening FOXA2 binding at NE enhancers. Conversely, FOXA2 mediates regional DNA demethylation, attributing to NE enhancer priming and inducing NKX2-1 expression, forming a feed-forward loop. Single-cell multiome analyses of isogenic cells over time-course NET cells identify individual cells amid luminal-to-NE transformation, exhibiting intermediate epigenetic and transcriptome states. Lastly, NKX2-1/FOXA2 interacts with, and recruits CBP/p300 proteins to activate NE enhancers, and pharmacological inhibitors of CBP/p300 effectively blunted NE gene expression and abolished NEPC tumor growth. Thus, our study reports a hierarchical network of TFs governed by NKX2-1 in regulating the 2D and 3D chromatin re-organization during NET and uncovers a promising therapeutic approach to eradicate NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f60ea05e69028e7c615b1e7620f9deafcd0cdd" target='_blank'>
              Epigenetic remodeling and 3D chromatin reorganization governed by NKX2-1 drive neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            Xiaodong Lu, Viriya Keo, Irina Cheng, Wanqing Xie, Galina A Gritsina, Juan Wang, Q. Jin, Peng Jin, Feng Yue, Martin G. Sanda, Victor Corces, Nicolas Altemose, Jonathan C. Zhao, Jindan Yu
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c9f63e7b6f9db6db3b912732c476af942c10e05" target='_blank'>
              Decreased mitochondrial transcription factor A and mitochondrial DNA copy number promote cyclin-dependent kinase inhibitor 1A expression and reduce tumorigenic properties of colorectal cancer cells
              </a>
            </td>
          <td>
            Jessika Buchwaldt, Tania Fritsch, Monika Hartmann, H. Witzel, Michael Kloth, Wilfried Roth, K. Tagscherer, Nils Hartmann
          </td>
          <td>2024-11-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6960e9d1d69ddbed72b6eb04c8ec6b5d4992a9b7" target='_blank'>
              Structural polymorphism and diversity of human segmental duplications.
              </a>
            </td>
          <td>
            Hyeonsoo Jeong, Philip C. Dishuck, DongAhn Yoo, William T. Harvey, Katherine M. Munson, Alexandra P. Lewis, Jennifer R. Kordosky, Gage H. Garcia, F. Yilmaz, P. Hallast, Charles Lee, T. Pastinen, E. Eichler
          </td>
          <td>2025-01-08</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Retrotransposons, multi-copy sequences that propagate via copy-and-paste mechanisms involving an RNA intermediate, occupy large portions of all eukaryotic genomes. A great majority of their manifold copies remain silenced in somatic cells, nevertheless, some are transcribed, often in a tissue specific manner, and a small fraction retains its ability to mobilize. Retrotransposon expression or mobility are increasingly recognized to contribute to normal development and tissue homeostasis, as well as to aging and disease. While it is well characterized that retrotransposon sequences may provide cis regulatory elements for neighboring genes, how their own expression and mobility are achieved in different somatic contexts is not well understood. Here, using long-read DNA sequencing, we characterize somatic retrotransposition in the Drosophila intestine. We show that retroelement mobility does not change significantly upon aging and is limited to very few active sub-families of retrotransposons. Importantly, we identify a polymorphic donor locus of an endogenous LTR retroviral element rover, active in the intestinal tissue. We reveal that gut activity of the rover donor copy depends on its genomic environment. Without affecting local gene expression, the copy co-opts its upstream enhancer sequence, rich in transcription factor binding sites, for somatic expression. Further we show that escargot, a snail-type transcription factor critical for gut progenitor cell function, can drive transcriptional activity of the active rover copy. These data provide new insights into how locus-specific features allow active retrotransposons to produce functional transcripts and mobilize in a somatic lineage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1e2218d35b486ab25b581d00df030a25e3f5364" target='_blank'>
              An endogenous retroviral element co-opts an upstream regulatory sequence to achieve somatic expression and mobility
              </a>
            </td>
          <td>
            Natalia Rubanova, Darshika Singh, Louis Barolle, Fabienne Chalvet, Sophie Netter, Mickael Poidevin, Nicolas Servant, Allison J. Bardin, Katarzyna Siudeja
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Abstract Carcinogenesis often involves significant alterations in the cancer genome, marked by large structural variants (SVs) and copy number variations (CNVs) that are difficult to capture with short-read sequencing. Traditionally, cytogenetic techniques are applied to detect such aberrations, but they are limited in resolution and do not cover features smaller than several hundred kilobases. Optical genome mapping (OGM) and nanopore sequencing [Oxford Nanopore Technologies (ONT)] bridge this resolution gap and offer enhanced performance for cytogenetic applications. Additionally, both methods can capture epigenetic information as they profile native, individual DNA molecules. We compared the effectiveness of the two methods in characterizing the structural, copy number and epigenetic landscape of a clear cell renal cell carcinoma tumor. Both methods provided comparable results for basic karyotyping and CNVs, but differed in their ability to detect SVs of different sizes and types. ONT outperformed OGM in detecting small SVs, while OGM excelled in detecting larger SVs, including translocations. Differences were also observed among various ONT SV callers. Additionally, both methods provided insights into the tumor’s methylome and hydroxymethylome. While ONT was superior in methylation calling, hydroxymethylation reports can be further optimized. Our findings underscore the importance of carefully selecting the most appropriate platform based on specific research questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2a68c85c3d1dfc77fbfc1f8f772053fa84be16b" target='_blank'>
              Long-read structural and epigenetic profiling of a kidney tumor-matched sample with nanopore sequencing and optical genome mapping
              </a>
            </td>
          <td>
            Sapir Margalit, Zuzana Tulpová, Tahir Detinis Zur, Yael Michaeli, J. Deek, Gil Nifker, Rita Haldar, Y. Gnatek, Dorit Omer, Benjamin Dekel, H. Baris Feldman, Assaf Grunwald, Yuval Ebenstein
          </td>
          <td>2025-01-07</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c8ca5fbab3938ff9f07a03caf56814d5bc9a81f" target='_blank'>
              Engineered extrachromosomal oncogene amplifications promote tumorigenesis.
              </a>
            </td>
          <td>
            D. Pradella, Minsi Zhang, Rui Gao, Melissa A Yao, Katarzyna M. Gluchowska, Ylenia Cendon-Florez, Tanmay Mishra, Gaspare La Rocca, Moritz Weigl, Ziqi Jiao, Hieu H M Nguyen, Marta Lisi, Mateusz M Ozimek, C. Mastroleo, Kevin Chen, Felix Grimm, J. Luebeck, Shu Zhang, Andrea Alice Zolli, Eric G Sun, Bhargavi Dameracharla, Zhengqiao Zhao, Y. Pritykin, C. Sigel, Howard Y. Chang, P. Mischel, V. Bafna, C. Antonescu, Andrea Ventura
          </td>
          <td>2024-12-18</td>
          <td>Nature</td>
          <td>0</td>
          <td>132</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fcd0f4cc9961826189852b167ab5b2ef51c1fec" target='_blank'>
              Integrating Circle-Seq with transcriptomics reveals genome-wide characterization of extrachromosomal circular DNA for dilated cardiomyopathy
              </a>
            </td>
          <td>
            Zhenhao Lin, Fangjie Dai, Bo Li, Yongchao Zhao, Changqian Wang
          </td>
          <td>2024-11-29</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fc92ee9eeeb11fdc0010923fa936390ffdc668a" target='_blank'>
              Multi-omic and single-cell profiling of chromothriptic medulloblastoma reveals genomic and transcriptomic consequences of genome instability
              </a>
            </td>
          <td>
            Petr Smirnov, M. Przybilla, Milena Simovic-Lorenz, R. G. Parra, H. Susak, Manasi Ratnaparkhe, John K. L. Wong, V. Körber, Jan-Philipp Mallm, George Philippos, M. Sill, Thorsten Kolb, Rithu Kumar, Nicola Casiraghi, K. Okonechnikov, David R. Ghasemi, K. Maass, K. Pajtler, Anna Jauch, A. Korshunov, Thomas Höfer, M. Zapatka, Stefan M Pfister, W. Huber, O. Stegle, Aurélie Ernst
          </td>
          <td>2024-11-23</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>109</td>
        </tr>

        <tr id="This study delves into the intriguing world of extrachromosomal DNA (ecDNA) in breast cancer, uncovering its pivotal role in cancer's aggressiveness and genetic variability. ecDNA, a form of circular DNA found outside chromosomes, is known to play a significant role in cancer progression by increasing oncogene expression. Focusing on two contrasting cell lines, MDA-MB-231 (triple-negative) and MCF-7 (Luminal-A), we utilized advanced microscopy and fluorescence techniques to detect and characterize ecDNA. Our findings reveal a stark difference: MDA-MB-231 cells, known for their high metastatic potential, exhibit a striking abundance of ecDNA, manifested as double minutes and single form with intense fluorescence signals. In contrast, the less aggressive MCF-7 cells harbor significantly fewer ecDNA. This disparity highlights the potential of ecDNA as a key player in cancer progression and a promising target for novel therapies. This research sheds light on the unseen genetic forces driving breast cancer and opens the door to new strategies in cancer treatment. Further research is necessary to understand the mechanisms of ecDNA formation and its role in different breast cancer subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fb924ca3420181045f4a1ed603d86cd06599e30" target='_blank'>
              Extrachromosomal DNA in Breast Cancer Cell Lines: Detection and Characterization.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, Fadhillah Fadhillah, Retno Lestari, Fadilah Fadilah, A. Bowolaksono, A. Dwiranti
          </td>
          <td>2024-12-29</td>
          <td>Microscopy research and technique</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The ribosomal genes (rDNA genes) encode 47S rRNA which accounts for up to 80% of all cellular RNA. At any given time, no more than 50% of rDNA genes are actively transcribed, and the other half is silent by forming heterochromatin structures through DNA methylation. In cancer cells, upregulation of ribosome biogenesis has been recognized as a hallmark feature, thus, the reduced methylation of rDNA promoter has been thought to support conformational changes of chromatin accessibility and the subsequent increase in rDNA transcription. However, an increase in the heterochromatin state through rDNA hypermethylation can be a protective mechanism teetering on the brink of a threshold where cancer cells rarely successfully proliferate. Hence, clarifying hypo- or hypermethylation of rDNA will unravel its additional cellular functions, including organization of genome architecture and regulation of gene expression, in response to growth signaling, cellular stressors, and carcinogenesis. Using the bisulfite-based quantitative real-time methylation-specific PCR (qMSP) method after ensuring unbiased amplification and complete bisulfite conversion of the minuscule DNA amount of 1 ng, we established that the rDNA promoter was significantly hypermethylated in 107 breast, 65 lung, and 135 colon tumour tissue samples (46.81%, 51.02% and 96.60%, respectively) as compared with their corresponding adjacent normal samples (26.84%, 38.26% and 77.52%, respectively; p < 0.0001). An excessive DNA input of 1 μg resulted in double-stranded rDNA remaining unconverted even after bisulfite conversion, hence the dramatic drop in the single-stranded DNA that strictly required for bisulfite conversion, and leading to an underestimation of rDNA promoter methylation, in other words, a faulty hypomethylation status of the rDNA promoter. Our results are in line with the hypothesis that an increase in rDNA methylation is a natural pathway protecting rDNA repeats that are extremely sensitive to DNA damage in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e41a5d7f178f6e61d7cd45d8e11b2f922f3fa334" target='_blank'>
              Hypermethylation at 45S rDNA promoter in cancers
              </a>
            </td>
          <td>
            Trang Thi Quynh Tran, Trang Hien Do, Tung The Pham, Phương Thi Thu Luu, Oanh Minh Pham, Uyen Quynh Nguyen, L. Vuong, Quang Ngoc Nguyen, Tuan Van Mai, Son Van Ho, Than Thi Nguyen, Lan Thi Thuong Vo
          </td>
          <td>2025-01-07</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="DNA double strand break (DSB) is a highly toxic lesion that can generate genome instability, a major source of tumorigenesis. DSBs are mainly repaired by non-homologous end joining (NHEJ) or homologous recombination (HR). The selection of the DSB repair pathway primarily depends on the DNA resection of the DSB ends. Indeed, HR is initiated by resection at the DSB generating 3’ single stranded extension. The shieldin complex prevents resection fostering DSB repair toward NHEJ. Here, we reveal that the inflammasome sensor NLRP3 facilitates DNA end resection to promote the HR pathway in an inflammasome-independent manner. Strikingly, NLRP3 silencing decreases HR efficiency, as evidenced by RAD51 foci and functional HR assays. Mechanistically, we describe that NLRP3 interacts with REV7, a subunit of the shieldin complex, and its depletion increases REV7 recruitment to IR-induced DSBs. Similar to cancer cells harboring HR mutated genes, we find that NLRP3 deficient cells are sensitive to PARP inhibitors (PARPi) and exhibit an epistatic relationship with BRCA1 deficiency. Remarkably, loss of REV7 in NLRP3-depleted cells induces PARPi resistance by restoring HR. This study unravels the crucial role of the innate immune receptor NLRP3 in regulating the selection of DSB repair pathways to maintain genome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f88ff36a1381788be79115bbf708323b6cc93cae" target='_blank'>
              The inflammasome sensor NLRP3 interacts with REV7 to maintain genome integrity through homologous recombination
              </a>
            </td>
          <td>
            Delphine Burlet, Md Muntaz Khan, Sabine Hacot, Hannes Buthmann, Léa Bardoulet, Anne-Laure Huber, Julie Gorry, Bernard S. Lopez, Yohann Couté, Matthias Geyer, Agnès Tissier, Virginie Petrilli
          </td>
          <td>2024-11-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Squamous cancers are the most frequently diagnosed solid tumours world-wide. The closely related transcriptional co-regulators YAP and TAZ(WWTR1) have emerged as important drivers of tumour initiation and progression in various types of squamous cell carcinoma. How YAP and TAZ execute their oncogenic functions in squamous cancers is still not fully understood. Here, we report that in addition to controlling transcriptional programmes that determine the balance between proliferation and terminal differentiation, YAP also engages with RIF1, a key regulator of DNA replication timing and protector of stalled replication forks under replication stress. YAP modulates gene expression of RIF1 via TEAD transcription factors and also stabilizes RIF1 protein on the chromatin. YAP-RIF1 complex formation increases in response to exogenous replication stress, and treatment with hydroxyurea to induce replication fork stalling acts synergistically with YAP depletion to impair cell proliferation. Our results thus demonstrate that YAP’s oncogenic functions in squamous cell carcinoma involve both transcriptional and non-transcriptional mechanisms, and that RIF1 is key for squamous cell carcinoma cells to survive with their high levels of endogenous replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3985e13774d5b1194193c3f3fcbde897bc3a759f" target='_blank'>
              YAP engages RIF1 to dampen replication stress in squamous cell carcinoma
              </a>
            </td>
          <td>
            Jodie Bojko, Benjamin Flynn, Bertram Aschenbrenner, Alexander Howard, Emily Lay, Emma Bailey, Natalia Krajic, Sandra Catalan, Kelli Gallacher, Elodie Sins, Jun Wang, Ute Jungwirth, B. Lichtenberger, Gernot Walko
          </td>
          <td>2025-01-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Genome duplication, critical for cell survival and identity, requires precise origin activation to ensure accurate DNA replication and chromatin structure maintenance. In colorectal cancer cells, prolonged replication stress does not hinder cell proliferation, although the mechanisms driving cancer cell adaptability remain largely unclear. Here, we demonstrate that upon recovery, cancer cells are able to activate new replication origins in distinct domains, causing persistent changes in chromatin accessibility, nuclear morphology, and replication timing, ultimately promoting chromatin instability. Chromatin accessibility, particularly in promoter regions, increased following replication stress in a subset of cells, correlating with altered gene expression and nuclear expansion. Additionally, some genes with enhanced promoter accessibility displayed sustained expression changes, further suggesting a transcriptional shift linked to stress adaptation. Our findings reveal that colorectal cancer cells recover from severe replication stress through new origin activation, a mechanism that not only maintains cell proliferation under stress but may also accelerate tumour heterogeneity. This research underscores replication origin activation as a potential therapeutic target in combating cancer cell resilience. GRAPHICAL ABSTRACT MAIN HIGHLIGHTS Colorectal cancer cells under severe replication stress activate new replication origins upon recovery. The firing of new origins in distinct replication domains results in long-lasting changes in chromatin accessibility, especially in promoter regions, and is associated with alterations in nuclear morphology and replication timing. Increased chromatin accessibility in certain promoter regions correlates with prolonged gene expression changes. The capacity of CRC cells to recover from severe replication stress fosters chromatin instability and tumour heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04cc1950d705dd8a02a8a5d1f21cfa69a97d0c9c" target='_blank'>
              Adaptive replication origin activation alters chromatin dynamics and stability in cancer cells
              </a>
            </td>
          <td>
            Alejandro Pérez-Venteo, Fernando Unzueta, Sonia Feu, P. Llinàs-Arias, Caroline Mauvezin, Neus Agell
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Structural variation is a major contributor to human diversity, adaptation, and disease. Simple structural variant (SV) types include deletions, insertions, duplications, inversions, and translocations, and SVs account for most of the variable bases between genomes. Complex structural variants (CSVs) that consist of one or more simple events in cis appear more frequently in diseases and cancers where DNA repair, apoptosis, and cell cycle checkpoints are compromised, although CSVs can also appear in germline genome sequences of healthy individuals. CSVs are often characterized by short tracts of homology or no homology, and while CSVs are more prevalent in complex regions that contain large repeats, smaller stretches of homology can also enable their formation across more unique loci. Long-read assemblies have increased the size of detectable SVs and expanded variant detection into more complex regions of the genome, and while they reconstruct CSVs, methods for identifying CSVs from assemblies is limited. Here, we have developed a new assembly-based approach to trace through complex loci rather than relying upon reference representations of alignments. We can now access CSVs in large complex segmental duplications, reveal structures that were previously unknown, and identify SV breakpoints with greater accuracy. We find 72 large CSVs per genome and 128 unique complex structures and CSVs in highly repetitive regions can now be detected including several distinct complex events in repetitive NBPF genes that was not previously callable with short-read or long-read CSV methods. This approach is implemented within a key assembly-based variant calling tool, PAV, and represents a substantial improvement identifying complex variants now ascertainable from contiguous genome assemblies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbfc7a889c92ffa27e1ac1ac1e0ce4aa98dae5ef" target='_blank'>
              Large complex structural rearrangements in human genomes harbor cryptic structures
              </a>
            </td>
          <td>
            P. Audano, Carolyn A. Paisie, Christine R. Beck
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is a hematological malignancy characterized by the clonal proliferation of malignant plasma cells within the bone marrow. The disease’s complexity is underpinned by a variety of genetic and molecular abnormalities that drive its progression. Methods This review was conducted through a state-of-The-art literature search, primarily utilizing PubMed to gather peer-reviewed articles. We focused on the most comprehensive and cited studies to ensure a thorough understanding of the genetic and molecular landscapes of MM. Results We detail primary and secondary alterations such as translocations, hyperdiploidy, single nucleotide variants (SNVs), copy number alterations (CNAs), gene fusions, epigenetic modifications, non-coding RNAs, germline predisposing variants, and the influence of the tumor microenvironment (TME). Our analysis highlights the heterogeneity of MM and the challenges it poses in treatment and prognosis, emphasizing the distinction between driver mutations, which actively contribute to oncogenesis, and passenger mutations, which arise due to genomic instability and do not contribute to disease progression. Conclusion & Future Perspectives We report key controversies and challenges in defining the genetic drivers of MM, and examine their implications for future therapeutic strategies. We discuss the importance of systems biology approaches in understanding the dependencies and interactions among these alterations, particularly highlighting the impact of double and triple-hit scenarios on disease outcomes. By advancing our understanding of the molecular drivers and their interactions, this review sets the stage for novel therapeutic targets and strategies, ultimately aiming to improve clinical outcomes in MM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec433a1270b41378c5571d49631b7d920ddc502b" target='_blank'>
              The Genetic and Molecular Drivers of Multiple Myeloma: Current Insights, Clinical Implications, and the Path Forward
              </a>
            </td>
          <td>
            Meghana Ram, Molly Fraser, Júnia Vieira dos Santos, R. Tasakis, Ariana Islam, Jannah Abo-Donia, Samir Parekh, A. Laganà
          </td>
          <td>2024-12-01</td>
          <td>Pharmacogenomics and Personalized Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="INTRODUCTION
The SEWA cell line, which is derived from a virus induced murine osteosarcoma (OS) ascites was established in the 1980s from a serially transplanted male derived tumor that first published in 1961. It has been applied in about 50 studies but was never genetically characterized in detail; this study fills that gap.


METHODS
The SEWA cell line was analyzed for its chromosomal constitution using molecular cytogenetic approaches. Array comparative genomic hybridization was performed to characterize copy number alterations.


RESULTS
SEWA has a near diploid karyotype without Y chromosome material. The complex karyotype includes neocentrics and simple and complex rearrangements. Amplification of MYC oncogene was detected in two homogeneously staining regions on two different derivative chromosomes.


CONCLUSION
An in silico translation of the obtained results to the human genome indicated that SEWA is suitable as a model for advanced human OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28ec328ac3891c2b11391d74ef5192bbc587c6a2" target='_blank'>
              Cytogenomic characterization of murine osteosarcoma cell line SEWA.
              </a>
            </td>
          <td>
            Thomas Liehr, Martina Rinčic
          </td>
          <td>2024-12-11</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Understanding the molecular mechanisms underlying rapid adaptation to stress is a fundamental question in evolutionary biology. A common mechanism is copy number changes, which can affect single genes, larger genomic regions, or even entire chromosomes in the case of eukaryotes (aneuploidy). Bacterial species with genomes comprising more than one chromosome are common, yet the role of adaptive copy number changes affecting entire chromosomes is unknown. Here, by using a bacterium with two chromosomes (chrI and chrII) as model system and combining evolution experiments in the presence of severe nutrient stress with whole-genome sequencing, we show that chromosomal copy number changes can drive rapid adaptation. We detected increased copy number of the entire chrII in almost half of the sequenced clones, isolated from twelve independent populations. In most cases, copy number increases were driven by mutations in the replication protein of chrII. A higher chrII copy number is likely to be beneficial because chrII codes for the operon responsible for metabolizing D-gluconic acid, the only carbon source present in the medium. Our results revealed a novel mechanism for rapid adaptation in bacteria. Given the prevalence of bacteria with secondary chromosomes, and the enrichment of secondary chromosomes in genes associated with interaction with the environment, this may be an important mechanism for adapting to novel environments that has been overlooked.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68e1750352edfd93ae418d5a02e63e98887bb1a2" target='_blank'>
              Chromosomal plasticity can drive rapid adaptation in bacteria
              </a>
            </td>
          <td>
            Luis Vega-Cabrera, O. Skovgaard, Macarena Toll-Riera
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Chromosomal inversions play a crucial role in evolution by influencing phenotypes through the linkage of co-adapted alleles. While inversions have been found across a large number of taxa, mapping and characterizing inversion breakpoint regions remains challenging, often due to the presence of complex tandem repeats and transposable elements (TEs). Here, we identify and quantify TEs in the breakpoints of the four large-scale inversions previously reported in Atlantic cod, leveraging on three high-quality long-read-based reference genome assemblies for the Norwegian Coastal cod, the Northeast Arctic cod and Celtic cod ecotypes. We detected a significant enrichment of TE orders and superfamilies with terminal inverted repeats (TIRs) within the inversion breakpoints of chromosomes 1, 7 and 12. Notably, we discovered a tandem accumulation of miniature inverted-repeat transposable elements (MITEs) belonging to a family of hAT transposons, exclusively residing in the breakpoints of the inverted haplotype on chromosomes 1 and 7 found in the Northeast Arctic cod. The accumulation of tandemly arranged TEs in breakpoint regions suggests that they have driven the appearance of inversions through ectopic recombination, further supporting the potential of TEs in facilitating chromosomal reorganizations with large evolutionary implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c69b85f1eed1a39a185be57ad812eafc541ce3b" target='_blank'>
              Chromosomal inversions mediated by tandem insertions of transposable elements
              </a>
            </td>
          <td>
            Robin Aasegg Araya, William B. Reinar, O. K. Tørresen, C. Goubert, Tara J Daughton, S. Hoff, H. T. Baalsrud, Marine Servane Ono Brieuc, Anna Komisarczuk, S. Jentoft, José Cerca, K.S. Jakobsen
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Glioblastoma Multiforme (GBM) is the most prevalent and highly malignant form of adult brain cancer characterized by poor overall survival rates. Effective therapeutic modalities remain limited, necessitating the search for novel treatments. Neurodevelopmental pathways have been implicated in glioma formation, with key neurodevelopmental regulators being re- expressed or co-opted during glioma tumorigenesis. Here we identified a serine/threonine kinase, NUAK family kinase 2 (NUAK2), as a fetal oncogene in mouse and human brains. We found robust expression of NUAK2 in the embryonic brain that decreases throughout postnatal stages and then is re-expressed in malignant gliomas. However, the role of NUAK2 in GBM tumorigenesis remains unclear. We demonstrate that CRIPSR-Cas9 mediated NUAK2 deletion in GBM cells results in suppression of proliferation, while overexpression leads to enhanced cell growth in both in vitro and in vivo models. Further investigation of the downstream biological processes dysregulated in the absence of NUAK2 reveals that NUAK2 modulates extracellular matrix (ECM) components to facilitate migratory behavior. Lastly, we determined that pharmaceutical inhibition of NUAK2 is sufficient to impede the proliferation and migration of malignant glioma cells. Our results suggest that NUAK2 is an actionable therapeutic target for GBM treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c93062bc61d08f65a71be5b4b969d76224456d0" target='_blank'>
              A fetal oncogene NUAK2 is an emerging therapeutic target in glioblastoma
              </a>
            </td>
          <td>
            Hanhee Jo, Aneesh Dalvi, Wenqi Yang, Elizabeth Morozova, Sarah Munoz, Stacey M. Glasgow
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Endogenous retroviral (ERV) RNA is highly expressed in cancer, although the molecular causes and consequences remain unknown. We found that ZC3H18 (Z18), a component of multiple nuclear RNA surveillance complexes, has recurrent truncating mutations in cancer. We show that Z18trunc mutations are oncogenic and that Z18 plays an evolutionarily conserved role in nuclear RNA surveillance of ERV RNA. In zebrafish, Z18trunc expedited melanoma onset and promoted a specific accumulation of ERV RNA. Z18 mutant human cell lines from the Cancer Cell Line Encyclopedia also expressed higher levels of ERV RNA. In engineered human melanoma cells, Z18trunc enhanced ERV RNA accumulation more than loss of one Z18 copy, indicating dominant negative activity. Z18trunc directly bound and stabilized ERV RNA. Notably, expression of ERV RNA was sufficient to expedite oncogenesis in a zebrafish model, which is the first evidence of which we are aware that ERV transcripts can play a functional role in cancer. Our work illuminates a mechanism for elevated ERV transcripts in cancer and supports that aberrant RNA accumulation is broadly oncogenic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85414cac93dcb00a3b2cbff32569ab0497071b3c" target='_blank'>
              Recurrent oncogenic ZC3H18 mutations stabilize endogenous retroviral RNA
              </a>
            </td>
          <td>
            Tanzina Tanu, Anna M. Cox, Jennifer A. Karlow, Priyanka Sharma, Xueyang He, Constance Wu, Swathy Babu, Jared Brown, Kevin M. Brown, Stephen J. Chanock, David Liu, Tongwu Zhang, Kathleen H. Burns, P. Boutz, Megan L. Insco
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="X-chromosome inactivation (XCI) is initiated during early mammalian embryogenesis by a long non-coding RNA (lncRNA) XIST, which coats one of the two X-chromosomes and facilitates epigenetic transcriptional silencing. A second, evolutionarily recent primate-specific lncRNA XACT was proposed to antagonize XIST’s ability to induce XCI. XACT expression is restricted to pluripotent states and early embryonic stages and coats the active X-chromosome in both females and males. Here, we report a novel XACT transcript expressed in normal and cancerous somatic cells from both the inactive (Xi) and active (Xa) X chromosomes. It coexists with XIST on the Xi without affecting XIST expression or (re)activating X-linked genes inactivated by XCI. During hematopoietic stem cell (HSC) differentiation, XACT is primarily expressed in myeloid progenitors in both sexes. XACT expression is activated in HSCs and peaks in megakaryocyte-erythrocyte progenitor cells (MEPs) before rapidly declining as the MEPs differentiate into megakaryocytes or erythrocytes. By combining CRISPR-based XACT perturbation with epigenomic and transcriptional studies, we revealed the key role of XACT in the self-renewal and differentiation of erythroid progenitors into erythrocytes, by recruiting cis-regulatory proteins and regulating transcription through ETS and AP-1 transcription factors. Furthermore, XACT is expressed in a subset of acute myeloid leukemia (AML) patients, with high levels found in erythroid-megakaryocytic blast cells; suggesting XACT’s potential as a marker for AML subtypes. Thus, we identified a novel XACT transcript with a unique expression profile and important roles in normal and cancer cells, revealing XACT lncRNA functions beyond its previously proposed role in XCI during embryogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7754eb09126d1d587b2765e70f76183499d8bcbf" target='_blank'>
              A Novel XACT lncRNA Transcript with Functions Transcending X-Chromosome Inactivation
              </a>
            </td>
          <td>
            Peifen Zhang, Paula Haro Angles, Hong Zhang, Tingyue Li, Roger Mulet-Lazaro, W. V. van Ijcken, S. Philipsen, Ruud Delwel, J. Gribnau, D. Huylebroeck, Frank Grosveld, C. Rougeulle, Eskeatnaf Mulugeta
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="The genome-wide chromosome conformation capture method, Hi-C, has greatly advanced our understanding of genome organization. However, its quantitative properties, including sensitivity, bias, and linearity, remain challenging to assess. Measuring these properties in vivo is difficult due to the heterogenous and dynamic nature of chromosomal interactions. Here, using Chemically Induced Chromosomal Interaction (CICI) method, we create stable intra- and inter-chromosomal interactions in G1-phase budding yeast across a broad range of contact frequencies. Hi-C analysis of these engineered cell populations demonstrates that static intra-chromosomal loops do not generate Topologically Associated Domains (TADs) and only promote 3D proximity within ∼50kb flanking regions. At moderate sequencing depth, Hi-C is sensitive enough to detect interactions occurring in 5-10% of cells. It also shows no inherent bias toward intra-versus inter-chromosomal interactions. Furthermore, we observe a linear relationship between Hi-C signal intensity and contact frequency. These findings illuminate the intrinsic properties of the Hi-C assay and provide a robust framework for its calibration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5303f7ba4784e24be34cf968c0e7ddd32d5ebcd" target='_blank'>
              Hi-C Calibration by Chemically Induced Chromosomal Interactions
              </a>
            </td>
          <td>
            Yi Li, Fan Zou, Lu Bai
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8feadfbb33b4fde036009823f20cf7ee9457518a" target='_blank'>
              BET inhibition revealed varying MYC dependency mechanisms independent of gene alterations in aggressive B-cell lymphomas
              </a>
            </td>
          <td>
            Loris Delrieu, Sieme Hamaidia, Emilie Montaut, Andrea Garcìa-Sandoval, Camille Teste, Patricia Betton-Fraisse, Thierry Bonnefoix, S. Carras, Rémy Gressin, Christine Lefebvre, Jérôme Govin, A. Emadali
          </td>
          <td>2024-12-19</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The SNF2 family chromatin remodeler HELLS has emerged as an important regulator of cell proliferation, genome stability, and several cancer pathways. Significant upregulation of HELLS has been reported in 33 human cancer types. While HELLS has been implicated in DNA damage response, its function in DNA repair is poorly understood. Here we report a new regulatory link between HELLS and single-strand break (SSB) repair in cellular responses to DNA alkylation damage. We found that loss of HELLS impairs SSB repair, and selectively sensitizes cells to DNA alkylating agents and PARP inhibitors (PARPi). Furthermore, we found that HELLS is co-expressed with PARP1 in cancer cells, and its loss is synthetic lethal with homologous recombination deficiency (HRD). This work unveils new functions of HELLS in modulating SSB repair and responses to clinically relevant DNA alkylation damage, thus offering new insights into the potential therapeutic value of targeting HELLS in cancer. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fd4d063ed0793e8f2d3e58d28ea55c516edbff0" target='_blank'>
              Chromatin regulator HELLS mediates SSB repair and responses to DNA alkylation damage
              </a>
            </td>
          <td>
            Joyous T Joseph, Christine M Wright, Estanislao Peixoto, Asad Khan, Yong Li, Jason S Romero Neidigk, Olivia Decker, Azait Imtiaz, Brianna A Jones, Yangfeng Zhang, Etsuko Shiabata, S. Gradilone, Zachary A. Lewis, Rafael Contreras-Galindo, Arko Sen, A. Dutta, Wioletta Czaja
          </td>
          <td>2024-12-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background Atypical teratoid rhabdoid tumor (ATRT) is the most common malignant brain tumor in infants, and more than 60% of children with ATRT die from their tumor. ATRT is associated with mutational inactivation/deletion of SMARCB1, a member of the SWI/SNF chromatin remodeling complex, suggesting that epigenetic events play a critical role in tumor development and progression. Moreover, disruption of SWI/SNF allows unopposed activity of epigenetic repressors, which contribute to tumorigenicity. We therefore explored the role of the CoREST repressor complex in ATRT. Methods We evaluated the effects of the bifunctional LSD1/HDAC1/2 small molecule CoREST inhibitor, corin, on ATRT tumor cell growth, apoptosis, differentiation, gene expression and chromatin accessibility. Results We found that corin inhibited the growth of ATRT cells regardless of their epigenetic subgroup, and was associated with increased tumor cell apoptosis and differentiation. ATAC-seq showed increases in chromatin accessibility in corin-treated ATRT cells, with changes seen at genes associated with neuronal differentiation and synaptic function. RNA-seq confirmed increased expression of neuronal differentiation genes and decreased DNA replication/cell cycle-associated genes in ATRT cells treated with corin. Corin suppressed orthotopic ATRT tumor growth, leading to significant extension of lifespan. In addition, increased histone acetylation (H3K9ac, H3K27ac) and methylation (H3K4Me1) was seen in corin-treated ATRT orthotopic xenografts, consistent with on-target pharmacodynamics. Conclusion The CoREST inhibitor, corin, suppresses tumor growth, induces differentiation, and promotes apoptosis in ATRT, leading to significantly increased survival of mice bearing ATRT orthotopic xenografts. Our results suggest a potential application of CoREST complex inhibitors in patients with ATRT. Key Points CoREST complex inhibition by corin leads to decreased cell growth and increased apoptosis in ATRT Corin promotes chromatin accessibility and neuronal differentiation in ATRT Corin inhibits tumor growth and extends lifespan in ATRT animal models Importance of the Study Loss of function of SMARCB1 is a hallmark of ATRT which leads to dysfunction of the mammalian SWI/SNF complex and an inability to counteract epigenetic repressor complexes. The CoREST complex functions as a chromatin remodeling complex that represses neuronal differentiation genes during development. Inhibition of the CoREST complex by corin in ATRT leads to decreased tumor cell growth, induction of apoptosis and increased survival of mice bearing ATRT orthotopic xenografts. These changes are associated with increased chromatin accessibility and expression of genes associated with neuronal differentiation. Corin therefore reverses the primary block of differentiation that maintains a stem cell state in ATRT, which contributes to tumorigenesis. These studies significantly improve our understanding of how to therapeutically address the underlying epigenetic drivers of ATRT and support further development of corin for ATRT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024c492e649d7b4c64d548dac81278e98253c902" target='_blank'>
              The CoREST complex inhibitor, corin, leads to decreased tumor growth, increased cellular differentiation and extended lifespan in atypical teratoid rhabdoid tumor xenograft models
              </a>
            </td>
          <td>
            Anupa Geethadevi, Nikhil Vaidya, Robert J. Fisher, Tyler Findlay, Yiming Deng, Khoa Pham, Vikas Kumar, Samuel Beck, Calixto-Hope G Lucas, C. Eberhart, Jinchong Xu, Philip A. Cole, Jeffrey A. Rubens, Marianne Collard, Eric H. Raabe, R. Alani
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Oncogenesis and tumor progression are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions of these two systems influence tumor development and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC to systematically integrate genetic cancer drivers and infiltration profiles of immune cells with clinical information. In an analysis of 8500 cancer samples, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell activities associate with disease outcomes. Risk associations of IGXs are potentially explained by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of therapeutic targets. In luminal-A breast cancer, MEN1 deletion combined with reduced neutrophils is associated with poor prognosis and deregulation of immune signalling pathways. These findings help elucidate how cancer drivers interact with TME to contribute to tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7541ff011e197e5ec1cc98d2bd31291176ffd633" target='_blank'>
              Combinations of genomic alterations and immune microenvironmental features associate with patient survival in multiple cancer types
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P. Klein, Alec Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C. L. Cheng, Jigyansa Mishra, Diogo Pellegrina, Kissy Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Increased expression of branched-chain amino acid (BCAA) transaminase 1 (BCAT1) often correlates with tumor aggressiveness and drug resistance in cancer. We have recently reported that BCAT1 was overexpressed in a subgroup of T-cell acute lymphoblastic (T-ALL) samples, especially those with NOTCH1 activating mutations. Interestingly, BCAT1-depleted cells showed pronounced sensitivity to DNA-damaging agents such as etoposide; however, how BCAT1 regulates this sensitivity remains uncertain. Here, we provide further clues on its chemo-sensitizing effect. Indeed, BCAT1 protein regulates the non-homologous end joining (c-NHEJ) DNA repair pathway by physically associating with the KU70/KU80 heterodimer. BCAT1 inhibition during active repair of DNA double-strand breaks (DSBs) led to increased KU70/KU80 acetylation and impaired c-NHEJ repair, a dramatic increase in DSBs, and ultimately cell death. Our results suggest that, in T-ALL, BCAT1 possesses non-metabolic functions that confer a drug resistance mechanism and that targeting BCAT1 activity presents a novel strategy to improve chemotherapy response in T-ALL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc99a18fefd0d2dfb9db2a0ff7404a49e575c24e" target='_blank'>
              BCAT1 Associates with DNA Repair Proteins KU70 and KU80 and Contributes to Regulate DNA Repair in T-Cell Acute Lymphoblastic Leukemia (T-ALL)
              </a>
            </td>
          <td>
            Valeria Tosello, Chiara Rompietti, A. Papathanassiu, G. Arrigoni, E. Piovan
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="In most cancers, including endometrial cancer, tumor suppressor genes harboring inactivating mutations have been systematically cataloged. However, locus-specific epigenetic alterations contributing to cancer initiation and progression remain only partly described, creating knowledge gaps about functionally significant tumor suppressors and underlying mechanisms associated with their inactivation. Here, we show that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event not associated with promoter hypermethylation. PAX2 is expressed throughout normal endometrial glands; however, microscopic clones with PAX2 protein loss arise spontaneously in adulthood and progress to endometrial precancers and cancers. Here we show that PAX2 protein loss occurs via transcriptional silencing in 80% of endometrial cancers. Mechanistically, transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that silencing is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive features (H3K27me3). The spread of the H3K27me3 signal is restrained by cohesin loops, preventing transcriptional dysregulation of neighboring genes. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. Genetically engineered mouse models together with organoid and human cell line studies established Pax2 as an in vivo endometrial tumor suppressor that cooperates with Pten to produce lethal cancers. Our discovery of a specific and recurring epigenetic alteration that transcriptionally silences PAX2 to initiate most endometrial cancers opens new lines of investigation into the origins of endometrial cancer with diverse implications for its diagnosis and treatment. One Sentence Summary We demonstrate that PAX2 is silenced by a highly recurrent epigenetic mechanism to initiate most endometrial cancers, establishing a novel mechanism of tumor suppressor inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90c481e61973d015ee3841a58286f99ac21dc81f" target='_blank'>
              PAX2 is Transcriptionally Silenced by a Distinct Mechanism of Epigenetic Reprogramming to Initiate Endometrial Carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Tumor heterogeneity is the substrate for tumor evolution and the linchpin of treatment resistance. Cancer cell heterogeneity is largely attributed to distinct genetic changes within each cell population. However, the widespread epigenome repatterning that characterizes most cancers is also highly heterogenous within tumors and could generate cells with diverse identities and malignant features. We show that high levels of the epigenetic regulator and oncogene, UHRF1, in zebrafish hepatocytes rapidly induced methylome disordering, loss of heterochromatin, and DNA damage, resulting in cell cycle arrest, senescence, and acquisition of stemness. Reducing UHRF1 expression transitions these cells from senescent to proliferation-competent. The expansion of these damaged cells results in hepatocellular carcinomas (HCC) that have immature cancer cells intermingled with fibroblasts, immune and senescent cells expressing high UHRF1 levels, which serve as reservoirs for new cancer cells. This defines a distinct and heterogenous HCC subtype resulting from epigenetic changes, stemness and senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2a7fc1c9934592d995c7390a3ec0bd3e0344392" target='_blank'>
              Epigenetic Disordering Drives Stemness, Senescence Escape and Tumor Heterogeneity
              </a>
            </td>
          <td>
            Elena Magnani, F. Macchi, Tijana Randic, Charlene Chen, Bhavani P. Madakashira, Shashi Ranjan, Sema Elif Eski, S. P. Singh, Kirsten C. Sadler
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The study investigates the impact of targeted chromosome engineering on telomere dynamics, chromatin structure, gene expression, and phenotypic stability in Arabidopsis thaliana. Using precise CRISPR/Cas-based engineering, reciprocal translocations of chromosome arms were introduced between non-homologous chromosomes. The subsequent homozygous generations of plants were assessed for phenotype, transcriptomic changes and chromatin modifications near translocation breakpoints, and telomere length maintenance. Phenotypically, translocated lines were indistinguishable from wild-type plants, as confirmed through morphological assessments and principal component analysis. Gene expression profiling detected minimal differential expression, with affected genes dispersed across the genome, indicating negligible transcriptional impact. Similarly, ChIPseq analysis showed no substantial alterations in the enrichment of key histone marks (H3K27me3, H3K4me1, H3K56ac) near junction sites or across the genome. Finally, bulk and arm-specific telomere lengths remained stable across multiple generations, except for minor variations in one translocation line. These findings highlight the remarkable genomic and phenotypic robustness of A. thaliana despite large-scale chromosomal rearrangements. The study offers insights into the cis-acting mechanisms underlying chromosome arm-specific telomere length setting and establishes the feasibility of chromosome engineering for studies of plant genome evolution and crop improvement strategies. Significance statement This study demonstrates the robustness of Arabidopsis thaliana in maintaining telomere stability, chromatin integrity, and wild-type phenotype despite large-scale chromosomal translocations. It underscores the potential of chromosome engineering in advancing genome evolution research and crop improvement, contributes to a deeper understanding of genome dynamics, and opens new avenues for precision breeding in plants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c4a5f7dc0b1cbe9e13ab94b7dc2055482d7e96c" target='_blank'>
              Chromosome engineering points to the cis-acting mechanism of chromosome arm-specific telomere length setting and robustness of plant phenotype, chromatin structure and gene expression
              </a>
            </td>
          <td>
            Ondřej Helia, Barbora Matúšová, Kateřina Havlová, Anna Hýsková, Martin Lyčka, Natalja Beying, Holger Puchta, Jíří Fajkus, M. Fojtová
          </td>
          <td>2024-12-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The role of GTF2I (General Transcription Factor2I) alteration has already been reported in thymic cancer as a valuable biomarker. However, the association of GTF2I mutation with renal cancer for prognosis of immunotherapy is not yet examined. The biologic and oncologic significance of GTF2I in renal cancer was examined at multiomics level such as mutation, copy number alteration, structural variants. The Cancer Genome Atlas (TCGA), Human Protein Atlas (HPA) were used to retrieve the omics data. The expression of GTF2I mRNA was quite significant in case of renal caner. Correlation among the GTF2I mRNA, mutation, CNA and structural variants was also studied. Interactome of GTF2I was also constructed using STRING database. Gain, amplification, and missense mutation exhibited a positive correlation between GTF2I mRNA expression and non-structural variants. Similarly, GTF2I mRNA expression and copy number alterations from GISTIC were positively correlated. High expression of GTF2I was associated with better overall survival indicating the less aggressive clinical features. Insight Box Investigating GTF2I's complex function as a tumor suppressor transcription factor in renal carcinoma provides fresh insights into its biologic and oncologic importance, especially when considering the prognosis of immunotherapy. Little is known about its possible use as a biomarker for renal cancer. Using a multiomics approach and utilizing information from the Human Protein Atlas (HPA) and The Cancer Genome Atlas (TCGA), our study clarifies the intricate relationship between mRNA expression, GTF2I changes, and clinical outcomes in renal cancer. Our results indicate that GTF2I expression may be used as a prognostic indicator because it is positively correlated with favorable survival outcomes. Furthermore, the molecular interactions behind GTF2I's functional significance in renal cancer are revealed by interactome analysis utilizing the STRING database, providing important information for further study and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/230d161006420b058a49294c979445ff8c60d0e8" target='_blank'>
              GTF2I acts as a novel tumor suppressor transcription factor and shows Favorable prognosis in renal cancer.
              </a>
            </td>
          <td>
            T. C. Dakal, Mony Thakur, Nancy George, T. Singh, Vinod Yadav, Abhishek Kumar
          </td>
          <td>2025-01-08</td>
          <td>Integrative biology : quantitative biosciences from nano to macro</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Simple Summary Large genomic rearrangements of the PALB2 gene, especially deletions and duplications, are associated with hereditary breast–ovarian cancer. This study investigates intronic breakpoints linked to rearrangements in PALB2 exon 11, which is crucial for understanding the mechanisms affecting patients with hereditary breast and ovarian syndrome. Through next-generation sequencing, one duplication and three deletions were identified, confirmed by Multiplex Ligation-Dependent Probe Amplification. Detailed characterization revealed a tandem duplication of 5134 base pairs mediated by AluY repeats and identical deletions in three unrelated patients promoted by AluSx elements, resulting in a truncated PALB2 protein. These findings highlight the instability of intronic regions flanking exon 11 and suggest directions for future research on the prevalence and functional implications of these genomic alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30610e3b0b4ee5300bddce9e6a181cf9f25f21c9" target='_blank'>
              Alu–Mediated Duplication and Deletion of Exon 11 Are Frequent Mechanisms of PALB2 Inactivation, Predisposing Individuals to Hereditary Breast–Ovarian Cancer Syndrome
              </a>
            </td>
          <td>
            Diletta Sidoti, Valeria Margotta, Diletta Calosci, Erika Fiorentini, C. Bacci, Francesca Gensini, Laura Papi, Marco Montini
          </td>
          <td>2024-11-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cell-cell fusion is a tightly controlled process in the human body known to be involved in fertilization, placental development, muscle growth, bone remodeling, and viral response. Fusion between cancer cells results first in a whole-genome doubled state, which may be followed by the generation of aneuploidies; these genomic alterations are known drivers of tumor evolution. The role of cell-cell fusion in cancer progression and treatment response has been understudied due to limited experimental systems for tracking and analyzing individual fusion events. To meet this need, we developed a molecular toolkit to map the origins and outcomes of individual cell fusion events within a tumor cell population. This platform, ClonMapper Duo (‘CMDuo’), identifies cells that have undergone cell-cell fusion through a combination of reporter expression and engineered fluorescence-associated index sequences paired to random barcode sets. scRNA-seq of the indexed barcodes enables the mapping of each set of parental cells and fusion progeny throughout the cell population. In triple negative breast cancer cells CMDuo uncovered subclonal transcriptomic hybridization and unveiled distinct cell-states which arise in direct consequence of homotypic cell-cell fusion. CMDuo is a platform that enables mapping of cell-cell fusion events in high-throughput single cell data and enables the study of cell fusion in disease progression and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05fe68cd8b32acb61197d1f1f0a4cc878efa91c6" target='_blank'>
              Mapping cell-cell fusion at single-cell resolution
              </a>
            </td>
          <td>
            Andrea L. Gardner, Lan Zheng, Kennedy Howland, Andrew Saunders, Andrea Ramirez, Patrik Parker, Chisom Iloegbunam, Daylin Morgan, T. Jost, Amy Brock
          </td>
          <td>2024-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Short tandem repeats (STRs) are a rich source of genetic variation, but are difficult to genotype. While specialized repeat variant callers exist, they typically assume a euploid human genome. This means recent findings regarding phenotypic effects of STR variants in human health and disease cannot be readily extended to polyploid organisms or cancer, which is characterised by copy number alterations (CNAs). Here we present ConSTRain, a novel STR variant caller that explicitly accounts for the copy number of loci in its genotyping approach. We benchmark ConSTRain using a euploid human 100X whole genome sequencing sample where it calls STR allele lengths for over 1.7 × 106 loci in under 20 minutes with an accuracy of 98.28%. Subsequently, we show that ConSTRain resolves complex STR genotypes in an artificial trisomy 21 sample and a polyploid Dwarf Cavendish banana harbouring a large duplication. Finally, we analyse a microsatellite instable colorectal cancer tumoroid, where ConSTRain tackles CNAs and whole-genome duplications. ConSTRain is the first STR variant caller that allows for the investigation of repeats affected by CNAs, aneuploidies, and polyploid genomes. This unlocks the investigation of STRs across a wide range of contexts and organisms where they previously could not be easily studied.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42abd302650eb39af080a99a7d76a61bf128e039" target='_blank'>
              Genotyping Short Tandem Repeats Across Copy Number Alterations, Aneuploidies, and Polyploid Organisms
              </a>
            </td>
          <td>
            M. Verbiest, Elena Grassi, Andrea Bertotti, Maria Anisimova
          </td>
          <td>2024-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Polyploidization is a driving force of wheat evolution and speciation, yet its impact on epigenetic regulation and gene expression remains unclear. Here, we constructed a high-resolution epigenetic landscape across leaves, spikes, and roots of hexaploidy wheat and its tetraploid and diploid relatives. Inter-species stable-expression genes exhibited conserved amino acid sequences under strong purifying selection, while dynamic-expression genes were linked to species-specific adaptation. During hexaploidization, dominant D-subgenome homoeolog expression was suppressed via reduced activating epigenetic signals, converging expression with the A and B subgenomes. Proximal chromatin regions near genes were more stable, whereas distal regions, particularly enhancer-like elements mediated by H3K27ac and H3K4me3, exhibit higher dynamism. Sequence variations in these enhancers lead to differential gene regulation, influencing traits such as spike development. For instance, the two haplotypes of dCRE region of TaDEP-B1 resulted in significant differences in its expression and spikelet numbers. We also observed a coevolution of transcription factors and their binding sites, particularly within the expanded ERF family, which regulates spike morphology. This study highlights the interplay between sequence variation and epigenetic modifications in shaping transcriptional regulation during wheat speciation, offering valuable insights for genetic improvement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3eb492b176e5f2e28bdd73ac3853852b58a5d2c" target='_blank'>
              Reshaping epigenomic landscapes in facilitating the speciation of bread wheat
              </a>
            </td>
          <td>
            Zhaoheng Zhang, Xuelei Lin, Jingjing Yue, Yong-Chang Xu, Lingfeng Miao, Wenqiang Tang, Weilong Guo, Jun Xiao
          </td>
          <td>2024-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="REV7, also known as MAD2B, MAD2L2, and FANCV, is a HORMA-domain family protein crucial to multiple genome stability pathways. REV7’s canonical role is as a member of polymerase ζ, a specialized translesion synthesis polymerase essential for DNA damage tolerance. REV7 also ensures accurate cell cycle progression and prevents premature mitotic progression by sequestering an anaphase-promoting complex/cyclosome activator. Additionally, REV7 supports genome integrity by directing double-strand break repair pathway choice as part of the recently characterized mammalian shieldin complex. Given that genome instability is a hallmark of cancer, it is unsurprising that REV7, with its numerous genome maintenance roles, is implicated in multiple malignancies, including ovarian cancer, glioma, breast cancer, malignant melanoma, and small-cell lung cancer. Moreover, high REV7 expression is associated with poor prognoses and treatment resistance in these and other cancers. Promisingly, early studies indicate that REV7 suppression enhances sensitivity to chemotherapeutics, including cisplatin. This review aims to provide a comprehensive overview of REV7’s myriad roles in genome maintenance and other functions as well as offer an updated summary of its connections to cancer and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ab376b5318f868977c071b0b61bc16785632df" target='_blank'>
              REV7: a small but mighty regulator of genome maintenance and cancer development
              </a>
            </td>
          <td>
            Lara Maggs, Mitch McVey
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gene expression is coordinated by a multitude of transcription factors (TFs), whose binding to the genome is directed through multiple interconnected epigenetic signals, including chromatin accessibility and histone modifications. These complex networks have been shown to be disrupted during aging, disease, and cancer. However, profiling these networks across diverse cell types and states has been limited due to the technical constraints of existing methods for mapping DNA:Protein interactions in single cells. As a result, a critical gap remains in understanding where TFs or other chromatin remodelers bind to DNA and how these interactions are perturbed in pathological contexts. To address this challenge, we developed a transformative single-cell immuno-tethering DNA:Protein mapping technology. By coupling a species-specific antibody-binding nanobody to a cytosine base editing enzyme, this approach enables profiling of even weak or transient factor binding to DNA, a task that was previously unachievable in single cells. Thus, our Docking & Deamination followed by sequencing (D&D-seq) technique induces cytosine-to-uracil edits in genomic regions bound by the target protein, offering a novel means to capture DNA:Protein interactions with unprecedented resolution. Importantly, this technique can be seamlessly incorporated into common single-cell multiomics workflows, enabling multimodal analysis of gene regulation in single cells. We tested the ability of D&D-seq to record TF binding both in bulk and at the single-cell level by profiling CTCF and GATA family members, obtaining high specificity and efficiency, with clear identification of TF footprint and signal retention in the targeted cell subpopulations. Furthermore, the deamination reaction showed minimal off-target activity, with high concordance to bulk ChIP-seq reference data. Applied to primary human peripheral blood mononuclear cells (PBMCs), D&D-seq successfully identified CTCF binding sites and enabled integration with advanced machine-learning algorithms for predicting 3D chromatin structure. Furthermore, we integrated D&D-seq with single-cell genotyping to assess the impact of IDH2 mutations on CTCF binding in a human clonal hematopoiesis sample, uncovering altered binding and chromatin co-accessibility patterns in mutant cells. Altogether, D&D-seq represents an important technological advance enabling the direct mapping of TF or chromatin remodeler binding to the DNA in primary human samples, opening new avenues for understanding chromatin and transcriptional regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c35b9d0d36121406dcb7489958fdbc0f064e833" target='_blank'>
              Single-cell mapping of regulatory DNA:Protein interactions
              </a>
            </td>
          <td>
            Wei-Yu Chi, Sang-Ho Yoon, Levan Mekerishvili, S. Ganesan, Catherine Potenski, Franco Izzo, Dan A Landau, Ivan Raimondi
          </td>
          <td>2025-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most aggressive breast cancer subtype with substantial disease heterogeneity, limited treatment options, and dismal clinical outcomes. Some TNBCs display homologous recombination deficiency (HRD), a phenotype with elevated genomic burden and worse prognosis if left untreated but chemotherapeutic sensitivity. While the molecular landscape of TNBC is distinct from other breast cancer subtypes, the TNBC-specific link between HRD and epigenome-wide methylation has not been established. This study reports two independent cohorts of TNBC tumors (n = 32 and n = 58) with HRD and epigenomic landscapes measured by the Multiplex Ligation-dependent Probe Amplification assay and the Illumina MethylationEPIC arrays, respectively. Genome-wide copy number and methylation alterations were significantly higher in HRD (all p <.05). Methylation of genome-wide repeat element Alu and transcriptional regulatory regions were significantly lower in HRD (all p <.05). An age-adjusted epigenome-wide association study of the continuous HRD probability scores revealed significant loci (all FDR <0.05) that were depleted from the CpG-rich "island" regions often seen in gene promoters but enriched in the CpG-poor "open sea" regions localized to gene enhancers. The significant loci implicated well-known candidate genes involved in the epithelial-to-mesenchymal transition, Wnt signaling, and DNA damage response. Supervised machine learning of HRD with nucleotide-specific methylation as the input enabled clinically relevant tumor stratification. Taken together, this study provides novel biological and translational insights into HRD in TNBCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5080511deac3bf2a96cf31c6d0433b409f61f60d" target='_blank'>
              Extensive epigenomic dysregulation is a hallmark of homologous recombination deficiency in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Youdinghuan Chen, Lucas A Salas, Jonathan D. Marotti, Nicole P. Jenkins, Chao Cheng, Todd W. Miller, A. Kettenbach, Brock C. Christensen
          </td>
          <td>2024-12-05</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Simple Summary Oncogenes are proteins promoting cancer formation. Cancers caused by mutations in the RAS oncogene have been a major focus of cancer research for decades. Despite better understanding of how the RAS oncogene works, finding an effective way to treat RAS-driven cancers remains a challenge. This review explores how RAS affects DNA replication and genome stability. It also describes how RAS can lead to inflammatory responses and senescence. Finally, we discuss how these molecular insights have led to potential new lines of treatments to target RAS-mutated cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fb79941bb5aaefe1e6a485ef57471bc907fec75" target='_blank'>
              Oncogenic RAS in Cancers from the DNA Replication Stress and Senescence Perspective
              </a>
            </td>
          <td>
            Hervé Técher, Samira Kemiha, Xieraili Aobuli, A. M. Kolinjivadi
          </td>
          <td>2024-11-28</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Formation of templated insertions at DNA double-strand breaks (DSBs) is very common in cancer cells. The mechanisms and enzymes regulating these events are largely unknown. Here, we investigated templated insertions in yeast at DSBs using amplicon sequencing across a repaired locus. We document very short (most ∼5–34 bp), templated inverted duplications at DSBs. They are generated through a foldback mechanism that utilizes microhomologies adjacent to the DSB. Enzymatic requirements suggest a hybrid mechanism wherein one end requires Polδ-mediated synthesis while the other end is captured by nonhomologous end joining (NHEJ) or by alternative end joining (Alt-EJ). This process is exacerbated in mutants with low levels or mutated RPA (rtt105Δ; rfa1-t33) or extensive resection deficiency (sgs1Δ exo1Δ). Templated insertions from various distant genomic locations also increase in RPA mutants as well as in rad27Δ and originate from fragile regions of the genome. Among complex insertions, common events are insertions of two sequences, originating from the same locus and with inverted orientation. We propose that these inversions are also formed by microhomology-mediated template switching. Together, we propose that a shortage of RPA, typical in cancer cells, may be a factor that stimulates the formation of templated insertions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63e3eaa71a9c5239af5efc218e8d2317e017347b" target='_blank'>
              RPA and Rad27 limit templated and inverted insertions at DNA breaks
              </a>
            </td>
          <td>
            Yang Yu, Xin Wang, Jordan Fox, Qian Li, Yang Yu, P. Hastings, Kaifu Chen, Grzegorz Ira
          </td>
          <td>2024-12-03</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer is among the leading causes of mortality in developed countries due to limited available therapeutic modalities and high rate of morbidity. Although malignancies might show individual genetic landscapes, recurring aberrations in the neoplastic genome have been identified in the wide range of transformed cells. These include translocations of frequently affected loci of the human genetic material like the Ewing sarcoma breakpoint region 1 (EWSR1) of chromosome 22 that results in malignancies with mesodermal origin. These cytogenetic defects frequently result in the genesis of fusion genes involving EWSR1 and a number of genes from partner loci. One of these chromosomal rearrangements is the reciprocal translocation between the q13 and q12 loci of chromosome 12 and 22, respectively, that is believed to initiate cancer formation by the genesis of a novel, chimeric transcription factor provoking dysregulated gene expression. Since soft-tissue neoplasms carrying t(12;22)(q13;q12) have very poor prognosis and clinical modalities specifically targeting t(12;22)(q13;q12)-harboring cells are not available to date, understanding this DNA aberration is not only timely but urgent. Here, we review our current knowledge of human malignancies carrying the specific subset of EWSR1 rearrangements that leads to the expression of the EWSR1::ATF1 tumor-driver chimeric protein.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0079b19a9f617ef641d141e48e5638bd7234944d" target='_blank'>
              EWSR1::ATF1 Translocation: A Common Tumor Driver of Distinct Human Neoplasms
              </a>
            </td>
          <td>
            Julia Raffaella Bianco, YiJing Li, Ágota Petrányi, Z. Fábián
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Interrogating regulatory epigenetic alterations during tumor progression at the resolution of single cells has remained an understudied area of research. Here we developed a highly sensitive single-nucleus CUT&RUN (snCUT&RUN) assay to profile histone modifications in isogenic primary, metastatic, and cisplatin-resistant head and neck squamous cell carcinoma (HNSCC) patient-derived tumor cell lines. We find that the epigenome can be involved in diverse modes to contribute towards HNSCC progression. First, we demonstrate that gene expression changes during HNSCC progression can be comodulated by alterations in both copy number and chromatin activity, driving epigenetic rewiring of cell states. Furthermore, intratumour epigenetic heterogeneity (ITeH) may predispose subclonal populations within the primary tumour to adapt to selective pressures and foster the acquisition of malignant characteristics. In conclusion, snCUT&RUN serves as a valuable addition to the existing toolkit of single-cell epigenomic assays and can be used to dissect the functionality of the epigenome during cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ccec03801dafc8df15febd551c3a9189ef4f702" target='_blank'>
              Single-nucleus CUT&RUN elucidates the function of intrinsic and genomics-driven epigenetic heterogeneity in head and neck cancer progression.
              </a>
            </td>
          <td>
            H. Womersley, Daniel Muliaditan, Ramanuj DasGupta, L. F. Cheow
          </td>
          <td>2024-12-02</td>
          <td>Genome research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Epigenetic regulator genes play critical roles in controlling cell identity and are frequently disrupted in breast cancers, suggesting a key driver role in this disease and its associated phenotypes. However, specific epigenetic drivers (epidrivers) of mammary cell plasticity and their mechanistic contributions to this phenotype are poorly characterized. Methods To identify potential epidrivers of the emergence of mesenchymal breast cancer stem cell-like phenotypes in non-tumorigenic mammary cells, we employed a CRISPR/Cas9 loss-of-function screening strategy targeting epigenetic regulator genes. This approach was followed by an in-depth validation and characterization of epigenomic, transcriptomic, proteomic and phenotypic changes resulting from the disruption of the putative epidriver gene BAP1. Results Our investigation revealed that loss of the histone deubiquitinase BAP1 impacts cellular processes associated with breast cancer cell plasticity such as epithelial-to-mesenchymal transition (EMT) and actin cytoskeleton organization. In addition, we unveiled that BAP1 loss resulted in an overall less permissive chromatin and downregulated gene expression, impacting programs that control cellular glycosylation and leading to decreased glycan abundance and complexity. BAP1 rescue restored the expression of several deregulated genes in a catalytic activity-dependent manner, suggesting that BAP1-mediated cell identity and glycosylation regulation are largely dependent on its histone deubiquitinase activity. Conclusions Overall, our results point to BAP1 disruption as a driver of mammary cell plasticity and reveal a novel role of BAP1 as an epigenetic regulator of cellular glycosylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744135b0816e44d41619b9824fc988445e6f4363" target='_blank'>
              Disruption of the epigenetic regulator BAP1 drives chromatin remodeling leading to the emergence of cells with breast cancer stem cell properties and aberrant glycosylation
              </a>
            </td>
          <td>
            Mariana Gomes da Silva Araujo, Aurélie Sallé, Vincent Cahais, Claire Renard, C. Cuenin, Caroline Pires Poubel, Stéphane Keita, Thorsten Mosler, Christine Carreira, Gabrielle Goldman Levy, Lya Parres, Ekaterina Bourova-Flin, Sophie Rousseaux, Saadi Khochbin, A. Ghantous, Siniša Habazin, M. Pučić-Baković, Erika Cosset, G. Lauc, Z. Herceg, Rita Khoueiry
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="The six subunit Origin Recognition Complex (ORC) is a DNA replication initiator that also promotes heterochromatinization in some species. A multi-omics study in a human cell line with mutations in three subunits of ORC, reveals that the subunits bind to DNA independent of each other rather than as part of a common six-subunit ORC. While DNA-bound ORC2 was seen to compact chromatin and attract repressive histone marks, the activation of chromatin and protection from repressive marks was seen at a large number of sites. The epigenetic changes regulate hundreds of genes, including some epigenetic regulators, adding an indirect mechanism by which ORC2 regulates epigenetics without local binding. DNA-bound ORC2 also prevents the acquisition of CTCF at focal sites in the genome to regulate chromatin loops. Thus, individual ORC subunits are major regulators, in both directions, of epigenetics, gene expression and chromosome structure, independent of the role of ORC in replication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f78b4b4e51086a34cc8aa6b1896dde6fc4ca7f3" target='_blank'>
              Regulation of epigenetics and chromosome structure by human ORC2
              </a>
            </td>
          <td>
            Zhangli Su, Mengxue Tian, Etsuko Shibata, Y. Shibata, Tianyi Yang, Zhenjia Wang, Fulai Jin, C. Zang, A. Dutta
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Replication timing (RT) allows us to analyze temporal patterns of genome-wide replication, i.e., if genes replicate early or late during the S-phase of the cell cycle. RT has been linked to gene expression in normal and diseased acute and chronic states such as cancer. However, studies done to date focused on bulk cell populations that required tens of thousands of cells for RT analysis. Here, we developed an affordable novel single cell (sc)-multiomics approach to simultaneously analyze RT and gene expression from cells or nuclei. We used this approach to generate sc-RT profiles and sc-gene expression data from the well-established human liver cancer cell line, HepG2. We demonstrated that as few as 17 mid S-phase cells were sufficient to produce cell-type specific pseudo bulk RT profiles that had a high correlation to previously published HepG2 bulk RT profiles. The sc-RT profiles allowed us to visualize how individual cells progressed through genome replication. We were also able to demonstrate high-resolution correlations between RT and gene expression within each individual cell, which to our knowledge, has not been reported. We observed trends that were conserved between individual cells, as well as cell-to-cell variations, which were not possible to detect with the bulk RT studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36c8d64daee99beea62c15fd3842a66ac8960670" target='_blank'>
              Development of a Novel Single Cell Multiomics Approach for Simultaneous Analysis of Replication Timing and Gene Expression
              </a>
            </td>
          <td>
            Anala V. Shetty, Clifford J. Steer, Walter C. Low
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) refers to a class of circular, non-chromosomal DNA that has recently gained widespread attention due to its potential role in aging and neurodegenerative diseases. The generation of ecDNA is closely associated with processes such as double-strand breaks, micronuclei formation, and the breakage-fusion-bridge (BFB) cycle, all of which are integral to regulation of gene expression, genetic stability, and clonal evolution. In neurodegenerative diseases such as Alzheimer's disease, Parkinson's disease, and Huntington's disease, the aberrant formation of ecDNA is closely linked to defects in DNA repair, alterations in synaptic plasticity, and neuronal dysfunction. The distinct distribution and functional roles of ecDNA in these conditions make it a potential diagnostic biomarker and therapeutic target. This review provides an overview of the mechanisms underlying ecDNA formation and its functions in the nervous system. Additionally, it explores the clinical potential of ecDNA in disease diagnosis, targeted therapy, and personalized medicine, offering new insights for future research and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d58ac80378b185b8a630604a71f2ebc6449a88f9" target='_blank'>
              Extrachromosomal DNA: Molecular perspectives in aging and neurodegenerative diseases.
              </a>
            </td>
          <td>
            Yafeng Ma, Ying Xia, Kenji Karako, Peipei Song, Xiqi Hu
          </td>
          <td>2024-11-30</td>
          <td>Intractable & rare diseases research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="SETD6 is a non-histone lysine methylatransferase, previously shown to participate in several housekeeping signaling pathways such as the NFkB pathway, Wnt signaling pathway, mitosis and more. In the current study we show evidence that SETD6 methylation is involved in the regulation of cytokinesis - the final process that divides cell contents into two daughter cells. SETD6 depleted HeLa cells presented high levels of chromatin bridges and actin patches, which are commonly observed following chromosomal segregation errors. In a proteomic screen we identified Aurora-B as a novel SETD6 substrate. Aurora-B kinase is an essential regulator of cytokinesis, known to actively delay cytokinesis as a response to the presence of chromatin in the midzone. We found that SETD6 binds and methylates Aurora-B on two adjacent lysine residues. Upon replication stress, Aurora-B methylation by SETD6 increases but is abolished when the two lysine methylation targets are substituted. In addition, replication stress led to a high tendency of SETD6 depleted cells to multinucleate, a major chromosomal-instability (CIN) phenotype. We detected a significant reduction in the Aurora-B kinase activity during cytokinesis in SETD6 knockout cells upon replication stress, which could be the mechanism underlying the accumulation of CIN phenotypes in these cells. CIN is a hallmark of cancer and is associated with tumor cell malignancy. Our findings suggest that Aurora-B methylation by SETD6 carries meaningful implications on tumorigenic cellular pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30079fd8588b1378d0602ce8bccaf091eb97d109" target='_blank'>
              AuroraB-kinase methylation by SETD6 regulates cytokinesis and protects cells from chromosomal instability
              </a>
            </td>
          <td>
            Michal Feldman, Anand Chopra, Dikla Nachmias, Kyle K. Biggar, Daniel Sevilla, Natalie Elia, D. Levy
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Tumor protein 53 (TP53) is a tumor repressor gene that is highly responsible for activating cell cycle arrest in the event of DNA damage, stopping the replication of damaged cells, which may cause cancer. However, when mutated, TP53 is an oncogene, with mutated variants of it being seen in over half of all cancers diagnosed including the aforementioned four. With groundbreaking CRISPR-Cas9 gene editing technology coming to the forefront of biotechnological research, the applications to cancer research and gene therapy have been increasing. This review explores CRISPR-Cas9 and its uses for treatment and medication investigation in various cancers, including lung, prostate, breast, and bone cancer through the lens of the TP53 gene. Due to TP53s commonality in cancer, CRISPR-Cas9 gene editing technology can be used to its maximum potential in this context as a treatment or investigative tool, with it yielding positive results in reverting tumor growth, as well as great versatility when it comes to screening related genes that inhibit or encourage tumor growth which can be targeted by drugs, or even the screening of the drugs effectiveness themselves. Thus, CRISPR-Cas9s applications in TP53-related cancer treatment and chemotherapeutic fields prove this technology to be successful.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da087d15d0c931988cddd598ffd040ef96f3cb9b" target='_blank'>
              Using CRISPR/Cas9 Technology to Target TP53 Related Cancers
              </a>
            </td>
          <td>
            Man Ho Damien Hsieh
          </td>
          <td>2025-01-09</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma is recognized as the most aggressive type of primary brain tumor. Despite recent advances in understanding the molecular mechanisms involved in the biology of glioblastoma, patient survival rates remain disappointing, primarily due to the lack of effective treatment options. Tumor necrosis factor receptor-associated protein 1 (TRAP1), a member of the heat shock protein 90 (Hsp90) family, refers to a protein predominantly localized in the mitochondria that regulates both cellular metabolic reprogramming and mitochondrial apoptosis. This protein is highly expressed in several types of tumors, including colorectal cancer, breast cancer, prostate cancer, and lung cancer, and is often associated with drug resistance. However, TRAP1 is also downregulated in certain cancers such as ovarian cancer, bladder cancer, and renal cancer, where its lower expression correlates with poorer prognoses and chemoresistance. The role of TRAP1 lies in enhancing or suppressing oxidative phosphorylation, with the impact of such regulation on tumor development and progression being a matter of ongoing debate. These observations prompt further investigation into the mechanisms responsible for the dual role of TRAP1 as both an oncogene and a tumor suppressor in specific types of tumors, particularly glioblastoma. The present study reviews the role of TRAP1 in the development and progression of glioblastoma and discusses the potential of targeting TRAP1 as a novel therapeutic approach against tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/236ff07b8d8eaf07109bcd843d2d90948b45e7ad" target='_blank'>
              Role of TRAP1 Protein in the Development and Progression of Glioblastoma
              </a>
            </td>
          <td>
            I. Gareev, A. Yasinskaya, S. Roumiantsev
          </td>
          <td>2024-12-28</td>
          <td>Creative surgery and oncology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b4cf48abf7265148385bfea0bb2728f6fd6101" target='_blank'>
              SMYD3 drives cell cycle and epithelial-mesenchymal transition pathways through dual gene transcriptional repression and activation in HPV-negative head and neck cancer
              </a>
            </td>
          <td>
            Madhavi Murali, Abbas Saeed, Sohyoung Kim, K. Burkitt, Hui Cheng, Arfa Moshiri, Jawad Akhtar, D. Tsai, Marie Luff, Baktiar Karim, V. Saloura
          </td>
          <td>2025-01-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Alterations in DNA methylation profiles are typically found in cancer cells, combining genome-wide hypomethylation with hypermethylation of specific regions, such as CpG islands, which are normally unmethylated. Driving effects in cancer development have been associated with alteration of DNA methylation in certain regions, inducing, for example, the repression of tumor suppressor genes or the activation of oncogenes and retrotransposons. These alterations represent prime candidates for the development of specific markers for the detection, diagnosis and prognosis of cancer. In particular, these markers, distributed along the genome, provide a wealth of information that offers potential for innovation in the field of liquid biopsy, in particular thanks to the emergence of artificial intelligence for diagnostic purposes. This could overcome the limitations related to sensitivities and specificities, which remain too low for the most difficult applications in oncology: the detection of cancers at an early stage, the monitoring of residual disease and the analysis of brain tumors. In addition, targeting the enzymatic processes that control the epigenome offers new therapeutic strategies that could reverse the regulatory anomalies of these altered epigenomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6734e14b7462be1947bbbaa4edaec73c090d31d" target='_blank'>
              [Epigenetics and cancer: the role of DNA methylation].
              </a>
            </td>
          <td>
            Marine Gorse, Charline Bianchi, Charlotte Proudhon
          </td>
          <td>2024-12-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Structural variation (SV), defined as balanced and unbalanced chromosomal rearrangements >1 kb, is a major contributor to germline and neoplastic disease. Large variants have historically been evaluated by chromosome analysis and now are commonly recognized by chromosomal microarray analysis (CMA). The increasing application of genome sequencing (GS) in the clinic and the relatively high incidence of chromosomal abnormalities in sick newborns and children highlights the need for accurate SV interpretation and reporting. In this review, we describe SV patterns of common cytogenetic abnormalities for laboratorians who review GS data.


CONTENT
GS has the potential to detect diverse chromosomal abnormalities and sequence breakpoint junctions to clarify variant structure. No single GS analysis pipeline can detect all SV, and visualization of sequence data is crucial to recognize specific patterns. Here we describe genomic signatures of translocations, inverted duplications adjacent to terminal deletions, recombinant chromosomes, marker chromosomes, ring chromosomes, isodicentric and isochromosomes, and mosaic aneuploidy. Distinguishing these more complex abnormalities from simple deletions and duplications is critical for phenotypic interpretation and recurrence risk recommendations.


SUMMARY
Unlike single-nucleotide variant calling, identification of chromosome rearrangements by GS requires further processing and multiple callers. SV databases have caveats and limitations depending on the platform (CMA vs sequencing) and resolution (exome vs genome). In the rapidly evolving era of clinical genomics, where a single test can identify both sequence and structural variants, optimal patient care stems from the integration of molecular and cytogenetic expertise.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80a2a826f1810ca4839038af6a42ce3daef01b34" target='_blank'>
              Structural Variation Interpretation in the Genome Sequencing Era: Lessons from Cytogenetics.
              </a>
            </td>
          <td>
            Lucilla Pizzo, M. K. Rudd
          </td>
          <td>2025-01-01</td>
          <td>Clinical chemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Single-stranded DNA secondary structures such as G-quadruplexes (G4s) can potentially disrupt transcription, replication, and repair. Using bio-informatic analysis, here we show that BMI1 is enriched at putative G4s flanked by heterochromatin domains, and that BMI1 knockdown in human dermal fibroblasts (HDFs) resulted in heterochromatin relaxation and G4 induction, followed by replication stress and genomic instability. In these cells, G4s co-localized with large 53BP1 and PCNA foci resembling replication catastrophes. Inhibiting transcription partly attenuated DNA damage, suggesting rescue of transcription-replication collisions at difficult-to-replicate sequences. In BMI1 knockdown or pyridostatin-exposed HDFs, the Werner helicase accumulated and co-localized with G4s. Acute WRN knockdown also resulted in G4 induction. In HDFs from Werner and Hutchinson-Gilford progeria syndromes, loss of heterochromatin and nuclear envelope anomalies were associated with G4 induction and DNA damage. Nuclear envelope anomalies were also prominent following BMI1 knockdown. These findings suggests that heterochromatin-mediated repression of G4s attenuates replication stress and genomic instability, and that this mechanism is shared across distinct progeroid models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/731920cfa53eda1eb3b00a50ee2eec0629da79d1" target='_blank'>
              BMI1-mediated heterochromatinization represses G-quadruplex DNA formation to maintain genomic stability during replication
              </a>
            </td>
          <td>
            Roy Hanna, Gilbert Bernier
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047a24a952e93dd146dc76725bcf8e221c664c94" target='_blank'>
              Targeting chromosomally unstable tumors with a selective KIF18A inhibitor
              </a>
            </td>
          <td>
            Aaron F Phillips, Rumin Zhang, Mia Jaffe, Ryan Schulz, Marysol Chu Carty, Akanksha Verma, Tamar Feinberg, Michael D Arensman, Alan Chiu, Reka Letso, Nazario Bosco, Katelyn A. Queen, Allison R Racela, Jason K. Stumpff, Celia Andreu-Agullo, Sarah E. Bettigole, R. Depetris, Scott Drutman, Shinsan M Su, Derek A Cogan, Christina H Eng
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Epigenetic therapies are emerging for pediatric cancers. Due to the relatively low mutational burden in pediatric tumors, epigenetic dysregulation and differentiation blockade is a hallmark of oncogenesis in some childhood cancers. By targeting epigenetic regulators that maintain tumor cells in a primitive developmental state, epigenetic therapies may induce differentiation. The most well-studied and clinically advanced epigenetic-targeted therapies include azacitidine and decitabine, which inhibit DNA methylation through competitive inhibition of the enzymatic activity of the DNA methyltransferase family enzymes. These DNA hypomethylating agents are Food and Drug Administration (FDA) approved for hematologic malignancies. The discovery that DNA hypermethylation occurs in patients with isocitrate dehydrogenase (IDH) mutations has led to the development and FDA approval of IDH inhibitors for hematologic and solid tumors. Epigenetic dysregulation in pediatric tumors is also driven by changes in the "histone code" that either promote oncogene expression or repress tumor suppressors. Cancers whose chromatin landscape is characterized by such aberrant histone posttranslational modifications may be amenable to targeted therapies that inhibit the chromatin-modifying enzymes that read, write, and erase these histone modifications. Small molecules that inhibit the enzymatic activity of histone deacetylases, acetyltransferases, and methyltransferases have been approved for the treatment of some adult cancers, and these agents are currently under investigation in various pediatric tumors. Chromatin regulatory complexes can be hijacked by oncogenic fusion proteins that are produced by chromosomal translocations, which are common drivers in pediatric cancer. Small molecules that disrupt oncogenic fusion protein activity and their associated chromatin complexes have demonstrated remarkable promise, and this approach has become the standard treatment for a subset of leukemias driven by the PML-RARA oncogenic fusion protein. A deeper understanding of the mechanisms that drive epigenetic dysregulation in pediatric cancer may hold the key to future success in this field, as the landscape of druggable epigenetic targets is also expanding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a807fa68d0895be1a6925ed8f18dcca64f6453e" target='_blank'>
              Epigenetic Therapies.
              </a>
            </td>
          <td>
            Wallace Bourgeois, Scott A Armstrong, Emily B. Heikamp
          </td>
          <td>2024-12-18</td>
          <td>Cold Spring Harbor perspectives in medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA viruses at once elicit and commandeer host pathways, including DNA repair pathways for virus replication. Despite encoding its own DNA polymerase and processivity factor, human cytomegalovirus (HCMV) recruits the cellular processivity factor, proliferating cell nuclear antigen (PCNA) and specialized host DNA polymerases involved in translesion synthesis (TLS) to replication compartments (RCs) where viral DNA (vDNA) is synthesized. While the recruitment of TLS polymerases is important for viral genome stability, the role of PCNA is poorly understood. PCNA function in DNA repair is regulated by monoubiquitination (mUb) or SUMOylation of PCNA at lysine 164 (K164). We find that mUb-PCNA increases over the course of infection, and modification of K164 is required for PCNA-mediated restriction of virus replication. mUb-PCNA plays important known roles in recruiting TLS polymerases to DNA, which we have shown are important for viral genome integrity and diversity, represented by novel junctions and single nucleotide variants (SNVs), respectively. We find that PCNA drives SNVs on vDNA similar to Y-family TLS polymerases, but that this did not require modification at K164. Unlike TLS polymerases, PCNA was dispensable for preventing large scale rearrangements on vDNA. These striking results suggest separable PCNA-dependent and - independent functions of TLS polymerases on vDNA. By extension, these results imply roles for TLS polymerase beyond their canonical function in TLS in host biology. These findings highlight PCNA as a complex restriction factor for HCMV infection, likely with multiple distinct roles, and provides new insights into the PCNA-mediated regulation of DNA synthesis and repair in viral infection. IMPORTANCE Genome synthesis is a critical step of virus life cycles and a major target of antiviral drugs. Human cytomegalovirus, like other herpesviruses, encodes machinery sufficient for viral DNA synthesis and relies on host factors for efficient replication. We have shown that host DNA repair factors play important roles in HCMV replication, but our understanding of this is incomplete. Building on previous findings that specialized host DNA polymerases contribute to HCMV genome integrity and diversity, we sought to determine the importance of PCNA, the central polymerase regulator. PCNA associates with nascent viral DNA and restricts HCMV replication. While PCNA is dispensable for genome integrity, it contributes to genome diversity. Our findings suggest that host polymerases function on viral genomes by separable PCNA-dependent and - independent mechanisms. Through revealing complex roles for PCNA in HCMV replication, this study expands the repertoire of host DNA synthesis and repair proteins hijacked by this ubiquitous herpesvirus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afe97c65ecc13c9b2841234ccb055156970288c9" target='_blank'>
              Complex roles for proliferating cell nuclear antigen in human cytomegalovirus replication
              </a>
            </td>
          <td>
            P. Longmire, Olivia Daigle, S. Zeltzer, Matias Lee, Marek Svoboda, Marco Padilla-Rodriguez, Carly Bobak, Giovanni Bosco, F. Goodrum
          </td>
          <td>2025-01-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="PURPOSE
The detection of circulating tumor DNA, which allows non-invasive tumor molecular profiling and disease follow-up, promises optimal and individualized management of patients with cancer. However, detecting small fractions of tumor DNA released when the tumor burden is reduced remains a challenge.


EXPERIMENTAL DESIGN
We implemented a new highly sensitive strategy to detect base-pair resolution methylation patterns from plasma DNA and assessed the potential of hypomethylation of LINE-1 retrotransposons as a non-invasive multi-cancer detection biomarker. The DIAMOND (Detection of Long Interspersed Nuclear Element Altered Methylation ON plasma DNA) method targets 30-40,000 young L1 scattered throughout the genome, covering about 100,000 CpG sites and is based on a reference-free analysis pipeline.


RESULTS
Resulting machine learning-based classifiers showed powerful correct classification rates discriminating healthy and tumor plasmas from 6 types of cancers (colorectal, breast, lung, ovarian, gastric cancers and uveal melanoma including localized stages) in two independent cohorts (AUC = 88% to 100%, N = 747). DIAMOND can also be used to perform copy number alterations (CNA) analysis which improves cancer detection.


CONCLUSIONS
This should lead to the development of more efficient non-invasive diagnostic tests adapted to all cancer patients, based on the universality of these factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/321c21ecf5626d3880de9fbd35f995ec3b32a1f7" target='_blank'>
              Non-invasive multi-cancer detection using DNA hypomethylation of LINE-1 retrotransposons.
              </a>
            </td>
          <td>
            M. Michel, Maryam Heidary, Anissa Mechri, Kévin Da Silva, Marine Gorse, Victoria Dixon, Klaus von Grafenstein, Charline Bianchi, C. Hego, A. Rampanou, C. Lamy, M. Kamal, C. Le Tourneau, M. Séné, I. Bieche, Cécile Reyes, D. Gentien, Marc-Henri Stern, Olivier Lantz, L. Cabel, J-Y Pierga, F. Bidard, Chloé-Agathe Azencott, C. Proudhon
          </td>
          <td>2024-12-02</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Protein domains of transposable elements (TEs) and viruses increase the protein diversity of host genomes by recombining with other protein domains. By screening 10 million eukaryotic proteins, we identified several domains that define multi-copy gene families and frequently co-occur with TE/viral domains. Among these, a Tc1/Mariner transposase helix-turn-helix (HTH) domain was captured by F-box genes in the Caenorhabditis genus, creating a new class of F-box genes. For specific members of this class, like fbxa-215, we found that the HTH domain is required for diverse processes including germ granule localisation, fertility, and thermotolerance. Furthermore, we provide evidence that HSF-1 mediates the transcriptional integration of fbxa-215 into the heat-shock response by binding to Helitron TEs directly upstream of the fbxa-215 locus. The interactome of HTH-bearing F-box factors suggests roles in post-translational regulation and proteostasis, consistent with established functions of F-box proteins. Based on AlphaFold2 multimer proteome-wide screens, we propose that the HTH domain may diversify the repertoire of protein substrates that F-box factors regulate post-translationally. We further demonstrate that F-box genes repeatedly and independently captured TE domains throughout eukaryotic evolution, and describe an additional instance in zebrafish. In conclusion, we identify recurrent TE domain captures by F-box genes in eukaryotes and provide insights into how these novel proteins are integrated within host gene regulatory networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6dd85055c13e497054453e00ae392d0e8fafaf" target='_blank'>
              Transposable elements drive regulatory and functional innovation of F-box genes
              </a>
            </td>
          <td>
            M. V. Almeida, Zixin Li, Pedro Rebelo-Guiomar, Alexandra Dallaire, Lukáš Fiedler, Jonathan L. Price, Jan Sluka, Xiaodan Liu, F. Butter, C. Rödelsperger, E. Miska
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Reference data on genomic variation forms the basis of genetics research. Limitations in identifying genetic variation from single reference sequences have recently been addressed through improvements in sequencing technologies, allowing the generation of pangenomic references from multiple accurate chromosome-level de novo assemblies. Nevertheless, global pangenomes to date have yet to include genomes from the populations of the Middle Eastern Region. To address this shortcoming, this study provides an Emirati genome reference. Its core is a diploid assembly with a Quality Value (QV) of 60 that includes ten telomere-to-telomere chromosomes. This assembly is incorporated into a pangenome graph constructed of 52 additional high-quality assemblies, half of which are trio-based. This Emirati pangenome reveals a similar level of genomic variation as the one compiled by the Human Pangenome Reference Consortium, underscoring its utility for the identification of both global and population-centered genomic variation, even in genome regions that have been traditionally challenging to assemble but are covered by the Emirati telomere-to-telomere assembly. As such, the Emirati genome reference significantly contributes to genomic research globally and is an essential resource for genomics-based personalized medicine in the United Arab Emirates and other parts of the Middle East.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bea3da7dfdfd92521cb394b0b240769b9f35a98" target='_blank'>
              An Emirati pangenome incorporating a diploid telomere-to-telomere reference
              </a>
            </td>
          <td>
            Michael Olbrich, M. Mousa, Inken Wohlers, Amira Al Aamri, Halima Alnaqbi, Aisha Hanaya Alsuwaidi, Hima Vadakkeveettil Manoharan, Nour al dain Marzouka, Sanjay Erathodi Ramachandran, Anju Annie Thomas, Mohammed Alameri, Guan K Tay, R. Hamoudi, S. Ibrahim, Noura Al Ghaithi, Habiba S. Alsafar
          </td>
          <td>2024-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="The DNA double-strand break (DSB) always occurs within genome. The pathways that repair the DSB are essential in keeping cell viability. This review will focus on the recent understanding of TMEJ, especially two key protein factors involved in the TMEJ repair pathway2, DNA polymerase theta (Polθ) and DNA polymerase delta (Polδ), and discuss the potential mechanism of choosing repair pathways when there is a DNA Double Strand Break. We dissect TMEJ's unique mechanism of action, including recognition of DNA ends, microhomology search, end pairing, and DNA synthesis, by focusing on the enzyme Polθ as the central player to understand the repair pathway choice. We also discuss how BRCA-mutated cancer uses the TMEJ repair pathway to facilitate breast cancer cells' growth3. This review aims to provide a comprehensive overview of TMEJ's cellular functions, regulatory mechanisms, and pivotal role in BRCA-mutated breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ba6a1c7fb204a897897b1b79890ca6faef600e" target='_blank'>
              Polymerase Theta-mediated DNA End-Joining Repair in the Maintenance of Genome Stability
              </a>
            </td>
          <td>
            Xiangyu Yan
          </td>
          <td>2024-12-24</td>
          <td>Transactions on Materials, Biotechnology and Life Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genome maintenance is essential for the integrity of the genetic blueprint, of which only a small fraction is transcribed in higher eukaryotes. DNA lesions occurring in the transcribed genome trigger transcription pausing and transcription-coupled DNA repair. There are two major transcription-coupled DNA repair pathways. The transcription-coupled nucleotide excision repair (TC-NER) pathway has been well studied for decades, while the transcription-coupled homologous recombination repair (TC-HR) pathway has recently gained attention. Importantly, recent studies have uncovered crucial roles of RNA transcripts in TC-HR, opening exciting directions for future research. Transcription also plays pivotal roles in regulating the stability of highly specialized genomic structures such as telomeres, centromeres, and fragile sites. Despite their positive function in genome maintenance, transcription and RNA transcripts can also be the sources of genomic instability, especially when colliding with DNA replication and forming unscheduled pathological RNA:DNA hybrids (R-loops), respectively. Pathological R-loops can result from transcriptional stress, which may be induced by transcription dysregulation. Future investigation into the interplay between transcription and DNA repair will reveal novel molecular bases for genome maintenance and transcriptional stress-associated genomic instability, providing therapeutic targets for human disease intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fd6afacedaa734f271cf97ef11d6135943f79a8" target='_blank'>
              Transcription as a double-edged sword in genome maintenance.
              </a>
            </td>
          <td>
            Ouyang Jian
          </td>
          <td>2024-12-20</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite the vast diversity of life forms and living histories, it appears that all branches of Metazoa face the challenge of tumor growth. Contrary to human tumors, which take years to form, tumors in short-living species can arise within days without accumulating multiple mutations, raising the question whether the paths to tumorigenesis in diverse species have any commonalities. In a fly tumor model caused by loss of cell polarity genes, we first identified the rise of a glycolytic cell population over time, resembling the Wartburg effect observed in human tumors. We further identified two key metabolic changes in these fly tumors. First, a systemic depletion of acetyl-CoA leads to a reduction in histone acetylation levels and stochastic silencing of actively-transcribed genes. Second, defects in the methionine cycle cause a systemic depletion of S-Adenosyl methionine, which further reduces histone methylation levels and causes stochastic activation of transposons. Perturbation of the methionine metabolic process strongly inhibits tumor growth. Finally, to understand the evolutionary origin of tumorigenesis, we performed comparative studies of fly and human tumors, and identified human tumors that exhibit metabolic signatures similar to those of fly tumors. We found that human tumors with high metabolic similarity to fly tumors have a lower mutational load, younger patient age, and lower DNA methylation levels. This study suggests that tumorigenesis processes have a deep evolutionary origin and highlights that depletion of key metabolites is an evolutionarily-ancient driving force for tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe6bf208acb7d71c056481ed083b94ace1ace601" target='_blank'>
              Epigenetic reprogramming induced by Acetyl-CoA and SAM depletion is an evolutionarily-ancient path to malignant growth
              </a>
            </td>
          <td>
            Zhe Chen, Xiaomeng Zhang, Mingxi Deng, Chongyang Li, Thi Thuy Nguyen, Min Liu, Kun Dou, Toyotaka Ishibashi, Jiguang Wang, Yan Yan
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genetic mutations are well known to influence tumorigenesis, tumor progression, treatment response and relapse, but the role of epigenetic variation in cancer progression is still largely unexplored. The lack of epigenetic understanding in cancer evolution is in part due to the limited availability of methods to examine such a heterogeneous disease. However, in the last decade the development of several single-cell methods to profile diverse chromatin features (chromatin accessibility, histone modifications, DNA methylation, etc.) has propelled the study of cancer epigenomics. In this review, we detail the current landscape of single-omic and multi-omic single-cell methods with a particular focus on the examination of histone modifications. Furthermore, we provide recommendations on both the application of these methods to cancer research and how to perform initial computational analyses. Together, this review serves as a referential framework for incorporating single-cell methods as an important tool for tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e561846c42367eca2d20c4639ff6a8849a7d2d" target='_blank'>
              A hitchhiker's guide to single-cell epigenomics: Methods and applications for cancer research.
              </a>
            </td>
          <td>
            Marta Moreno-Gonzalez, Isabel Sierra, Jop Kind
          </td>
          <td>2024-12-22</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42786039d5d90ecc6efc329111435b0ff4acfda0" target='_blank'>
              Homologous recombination deficiency (HRD) diagnostics: underlying mechanisms and new perspectives.
              </a>
            </td>
          <td>
            Andrey Kechin, M. Koryukov, Regina Mikheeva, M. Filipenko
          </td>
          <td>2024-12-26</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The nucleolus is a major subnuclear compartment where ribosomal DNA (rDNA) is transcribed and ribosomes are assembled. In addition, recent studies have shown that the nucleolus is a dynamic organizer of chromatin architecture that modulates developmental gene expression. rDNA gene units are assembled into arrays located in the p-arms of five human acrocentric chromosomes. Distal junctions (DJs) are ∼400 kb sequences adjacent to rDNA arrays that are thought to anchor them at the nucleolus, although the underlying regulatory elements remain unclear. Here we show that DJs display a dynamic chromosome conformation profile in human embryonic stem cells (hESCs). We identified a primate-specific, full-length insertion of the retrotransposon long interspersed nuclear element 1 (LINE1) in a conserved position across all human DJs. This DJ-LINE1 locus interacts with specific regions of the DJ and is upregulated in naïve hESCs. CRISPR-based deletion and interference approaches revealed that DJ-LINE1 contributes to nucleolar positioning of the DJs. Moreover, we found that the expression of DJ-LINE1 is required for maintenance of the structure and transcriptional output of the nucleolus in hESCs. Silencing of DJ-LINE1 leads to loss of self-renewal, disruption of the landscape of chromatin accessibility, and derepression of earlier developmental programs in naïve hESCs. This work uncovers specific LINE1 elements with a fundamental role in nucleolar organization in hESCs and provides new insights into how the nucleolus functions as a key genome-organizing hub.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6f03971b0054eedbed60fad051bf9b167c6012" target='_blank'>
              LINE1 elements at distal junctions of rDNA repeats regulate nucleolar organization in human embryonic stem cells.
              </a>
            </td>
          <td>
            Lamisa Ataei, Juan Zhang, Simon Monis, Krystyna Giemza, Kirti Mittal, Joshua Yang, Mayu Shimomura, B. McStay, Michael D. Wilson, M. Ramalho-Santos
          </td>
          <td>2024-12-20</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="ABSTRACT Insertion sequences (ISs) are mobile pieces of DNA that are widespread in bacterial genomes. IS movements typically involve (i) excision of the IS element, (ii) cutting of target site DNA, and (iii) IS element insertion. This process generates a new copy of the IS element and a short duplication at the target site. It has been noted that, for some extant IS copies, no target site duplications (TSDs) are readily identifiable. TSD absence has been attributed to degeneration of the TSD after the insertion event, recombination between identical ISs, or adjacent deletions. Indeed, the latter two—recombination between ISs and adjacent deletions—are frequent causes for the absence of TSDs, which we demonstrate here in an analysis of genome sequence data from the Lenski long-term evolution experiment. Furthermore, we propose that some IS movements—namely, those that occur in association with large-scale genomic rearrangements—do not generate TSDs, and occur without evidence for recombination between ISs or adjacent deletions. In support of this hypothesis, we provide two direct, empirical observations of such IS transposition events: an IS5 movement plus a large deletion in Escherichia coli C, and an IS481 movement occurring with a large duplication in Pseudomonas fluorescens SBW25. Although unlikely, it is possible that the observed deletion and associated IS movement occurred in two successive events in one overnight culture. However, an IS at the center of a large-scale duplication is not readily explained, suggesting that IS element activity may promote both large-scale deletions and duplications. IMPORTANCE Insertion sequences are the most common mobile genetic elements found in bacterial genomes, and hence they significantly impact bacterial evolution. We observe insertion sequence movement at the center of large-scale deletions and duplications that occurred during laboratory evolution experiments with Escherichia coli and Pseudomonas fluorescens, involving three distinct types of transposase. We raise the possibility that the transposase does not mediate DNA cleavage but instead inserts into existing DNA breaks. Our research highlights the importance of insertion sequences for the generation of large-scale genomic rearrangements and raises questions concerning the mechanistic basis of these mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00001ae08c5c01e65cbd4f4a4b234bdd427cfa24" target='_blank'>
              A more significant role for insertion sequences in large-scale rearrangements in bacterial genomes
              </a>
            </td>
          <td>
            Wing Y. Ngan, Lavisha Parab, Frederic Bertels, J. Gallie
          </td>
          <td>2024-12-05</td>
          <td>mBio</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Mitosis is a critical phase of the cell cycle and a vulnerable point where cancer cells can be effectively disrupted, leading to cell death and inhibition of tumor growth. However, challenges such as drug resistance remain significant in clinical applications. During mitosis, mRNA translation is generally downregulated, while non-canonical translation of specific transcripts proceeds. Here, we demonstrate that mitotic cancer cells redistribute ribosomes toward the 5’ untranslated region (5’ UTR) and the start of the coding sequence (CDS), enhancing the translation of thousands of upstream open reading frames (uORFs) and upstream overlapping open reading frames (uoORFs). This mitotic induction of uORF/uoORF enriches the presentation of immunopeptides at the surface of cancer cells following treatment with mitotic inhibitors. Functional assays indicate the potential of such neoepitopes to provoke cancer-cell killing by T cells. Altogether, our findings highlight the therapeutic potential of targeting uORF/uoORF-derived neoepitopes in combination with mitotic inhibitors to enhance immune recognition and tumor cell elimination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e58b9a8310537bb25bde77518c59ca5cd45e675f" target='_blank'>
              Upstream open reading frame translation enhances immunogenic peptide presentation in mitotically arrested cancer cells
              </a>
            </td>
          <td>
            Alexander Kowar, Jonas P. Becker, Rossella Del Pizzo, Zhiwei Tang, Julien Champagne, Pierré-René Körner, Jasmine Montenegro Navarro, Fiona Megan Tilghman, Hanan Sakeer, Angelika B. Riemer, R. Agami, Fabricio Loayza-Puch
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Background Single-cell sequencing can provide novel insights into the understanding and treatment of diseases. In cancer, for example, intratumor heterogeneity is a major cause of treatment resistance and relapse. Although technological progress has substantially increased the throughput of sequenced cells, single-cell sequencing remains cost and labor-intensive. Multiplexing, i.e., the pooling and subsequent joint preparation and sequencing of samples, followed by a demultiplexing step, is a common practice to reduce expenses and confounding batch effects, especially in single-cell RNA sequencing. Results Here, we introduce demoTape, a computational demultiplexing method for targeted single-cell DNA sequencing (scDNA-seq) data based on a distance metric between individual cells at single-nucleotide polymorphisms loci. To validate demoTape, we sequence three B-cell lymphoma patients separately and multiplexed on the Tapestri platform. We find similar genotypes, clones, and evolutionary histories in all three samples when comparing the individual with the demultiplexed samples. Using the three individually sequenced samples, we simulate multiplexed ground truth data and show that demoTape outperforms state-of-the-art demultiplexing methods designed for RNA sequencing data. Additionally, we demonstrate through downsampling that the inferred clonal composition remained largely stable for samples with fewer cells despite the inevitable loss in resolution of low-frequency clones. Conclusions Multiplexing and subsequent genotype-based demultiplexing of scDNA-seq will reduce costs and workload, eventually allowing the sequencing of more samples. This will open new possibilities and accelerate the investigation of biological questions where cellular heterogeneity on the genomic level plays a crucial role.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854d13588a800839a9fb4c222c2fbfc27f0a2f60" target='_blank'>
              DemoTape: Computational demultiplexing of targeted single-cell sequencing data
              </a>
            </td>
          <td>
            Nico Borgsmüller, Jack Kuipers, Johannes Gawron, Marco Roncador, Marcel Pohly, Erkin Acar, Thi Huong Lan Do, Stefanie Reisenauer, Mirjam Judith Feldkamp, C. Beisel, Thorsten Zenz, A. Moor, N. Beerenwinkel
          </td>
          <td>2024-12-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Abstract Background Retrotransposon‐derived extrachromosomal circular DNA (eccDNA) was extracted and sequenced from brains with Alzheimer’s disease, progressive supranuclear palsy, or healthy controls. Retrotransposon‐derived DNA was visualized outside of the nucleus in these phenotypes with phospho‐STING. Drosophila were used as a model to study extranuclear retrotransposon DNA. Method DNA was extracted from post‐mortem human patients with AD (n = 6), PSP (FTD) (n = 6), or no pathology (n = 6). The DNA was treated with exonucleases to degrade linear DNA and the circular DNA was isolated. Post‐mortem cryosections of AD, PSP, or healthy control brains were used for fluorescent in‐situ hybridization (FISH) of retrotransposon sequences. Co‐immunofluorescence of phospho‐STING was performed on FISH tissue. Drosophila brains were also used for FISH of retrotransposons. Digital qPCR was used to measure copy number of retrotransposons in human and fly tissue. Result Preliminary data suggests an increase in phospho‐STING in AD and PSP brains compared to controls. Preliminary data also suggests an enrichment of retrotransposon DNA in circular DNA sequencing in tau pathology compared to controls. Conclusion Retrotransposon‐derived eccDNA may be an activator of the cGAS‐STING pathway in AD and PSP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/010a3514f6b2b99b40748f696daa80760e610981" target='_blank'>
              Investigating retrotransposon‐derived eccDNA as a source of innate immune activation in tauopathy
              </a>
            </td>
          <td>
            Morgan Elizabeth Lambert, Paulino Ramirez, Wenyan Sun, Bess Frost
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & Dementia</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="SUMMARY DNA, is assaulted by endogenous and exogenous agents that lead to the formation of damage. In order to maintain genome integrity DNA repair pathways must be efficiently activated to prevent mutations and deleterious chromosomal rearrangements. Conversely, genome rearrangement is also necessary to allow genetic diversity and evolution. The antagonist interaction between maintenance of genome integrity and rearrangements determines genome shape and organization. Therefore, it is of great interest to understand how the whole linear genome structure behaves upon formation and repair of DNA damage. For this, we used long reads sequencing technology to identify and to characterize genomic structural variations (SV) of wild‐type Arabidopsis thaliana somatic cells exposed either to UV‐B, to UV‐C or to protons irradiations. We found that genomic regions located in heterochromatin are more prone to form SVs than those located in euchromatin, highlighting that genome stability differs along the chromosome. This holds true in Arabidopsis plants deficient for the expression of master regulators of the DNA damage response (DDR), ATM (Ataxia‐telangiectasia‐mutated) and ATR (Ataxia‐telangiectasia‐mutated and Rad3‐related), suggesting that independent and alternative surveillance processes exist to maintain integrity in genic regions. Finally, the analysis of the radiations‐induced deleted regions allowed determining that exposure to UV‐B, UV‐C and protons induced the microhomology‐mediated end joining mechanism (MMEJ) and that both ATM and ATR repress this repair pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b13004cb07afbb75c796371a93fdfc391d332b" target='_blank'>
              Characterization of radiations‐induced genomic structural variations in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Salimata Ousmane Sall, Abdelmalek Alioua, Sébastien Staerck, Stéfanie Graindorge, M. Pellicioli, Jacky Schuler, Catherine Galindo, Quentin Raffy, Marc Rousseau, J. Molinier
          </td>
          <td>2024-12-01</td>
          <td>The Plant Journal</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) are extremely effective in a subgroup of mismatch repair-deficient (MMRd) cancers, but ∼50% remain resistant to treatment. We have shown for the first time that this may be due to the differential regulation of factors linked to response to ICIs upon loss of the different MMR genes. Here, we show that increased PD-L1 expression is observed upon loss of the MMR genes MLH1, MSH2 and PMS2. However, this is not true upon loss of MSH6. Here, we show that this is due to a novel role for MSH6 as a direct regulator of PD-L1 transcription, dependent on recruitment by the histone trimethyltransferase SETD2. Next-generation sequencing of MLH1 and MSH6 knockout (KO) cells revealed that MSH6 KO cells have significantly lower microsatellite instability in comparison to MLH1 KO cells, despite MSH6 KO cells having a higher mutational burden. These findings emphasise the need for gene-specific stratification in the MMRd cohort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d6ede7d9d243c2456000d9151a205cf93b9af95" target='_blank'>
              The DNA Mismatch repair protein, MSH6 is a novel regulator of PD-L1 expression
              </a>
            </td>
          <td>
            Kirsten Brooksbank, Charlotte Smith, E. Maniati, Amy Gibson, Wai Yiu Tse, Amy Kate Hall, Jun Wang, T. Sharp, Sarah A Martin
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eff1090b04839914362d5c68932f9ab6b30614f" target='_blank'>
              Emerging roles of cohesin-STAG2 in cancer.
              </a>
            </td>
          <td>
            Julia S Scott, Loubna Al Ayadi, Emmanouela Epeslidou, Roan H van Scheppingen, Anna Mukha, Lucas J T Kaaij, C. Lutz, S. Prekovic
          </td>
          <td>2024-11-29</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Structural variations (SVs) play important roles in genetic diversity, evolution, and carcinogenesis and are, as such, important for human health. However, it remains unclear how spatial proximity of double-strand breaks (DSBs) affects the formation of SVs. To investigate if spatial proximity between two DSBs affects DNA repair, we used data from 3C experiments (Hi-C, ChIA-PET, and ChIP-seq) to identify highly interacting loci on six different chromosomes. The target regions correlate with the borders of mega-base sized Topologically Associated Domains (TADs), and we used CRISPR-Cas9 nuclease and pairs of single guide RNAs (sgRNAs) against these targets to generate DSBs in both K562 cells and H9 human embryonic stem cells (hESC). Droplet Digital PCR (ddPCR) was used to quantify the resulting recombination events, and high-throughput sequencing was used to analyze the chimeric junctions created between the two DSBs. We observe a significantly higher formation frequency of deletions and inversions with DSBs in proximity as compared to deletions and inversions with DSBs not in proximity in K562 cells. Additionally, our results suggest that DSB proximity may affect the ligation of chimeric deletion junctions. Taken together, spatial proximity between DSBs is a significant predictor of large-scale deletion and inversion frequency induced by CRISPR-Cas9 in K562 cells. This finding has implications for understanding SVs in the human genome and for the future application of CRISPR-Cas9 in gene editing and the modelling of rare SVs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9534f559b29005a3cc7a49a03b73e3b8f75c23b4" target='_blank'>
              Characterization of the role of spatial proximity of DNA double-strand breaks in the formation of CRISPR-Cas9-induced large structural variations.
              </a>
            </td>
          <td>
            Mikkel Dahl-Jessen, T. Terkelsen, R. Bak, Uffe Birk Jensen
          </td>
          <td>2025-01-13</td>
          <td>Genome research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM), a highly aggressive brain tumor, frequently develops resistance to temozolomide (TMZ), the current standard chemotherapy. Our study investigates the potential of combining TMZ with the poly (ADP-ribose) polymerase inhibitor Olaparib (OLA) to overcome TMZ resistance. Using in vitro models, including U251 cell lines and patient-derived GBM primary cultures, we demonstrate that OLA enhances TMZ efficacy by disrupting base excision repair and potentiating DNA damage-induced cytotoxicity. A CRISPR knockout screen identified DNA mismatch repair (MMR) gene deficiencies as key drivers of TMZ resistance, which OLA effectively counteracts. Co-treatment with TMZ and OLA significantly reduced tumor cell viability, even in MMR-deficient and MGMT-expressing contexts, suggesting a synergistic mechanism. Gene expression analysis revealed that the combination therapy impacts cell cycle regulation, stress responses, and extracellular matrix integrity, leading to mitotic catastrophe and apoptosis. These findings propose the TMZ-OLA combination as a promising therapeutic strategy for overcoming chemoresistance in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4edc2d11f92b839420cb397da78a6ec7ab2aa82c" target='_blank'>
              PARP inhibitor counteracts Temozolomide Resistance in Glioblastoma Multiforme
              </a>
            </td>
          <td>
            Edouard Samarut, Tabourel Gaston, Briand Joséphine, Landais Yuna, Gratas Catherine, Cartron Pierre-François, Vallette François, Oliver Lisa, Sérandour A. Aurélien
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genomic studies of autism and other neurodevelopmental disorders have identified several relevant protein-coding and noncoding variants. One gene with an excess of protein-coding de novo variants is EBF3 that also is the gene underlying the Hypotonia, Ataxia, and Delayed Development Syndrome (HADDS). In previous work, we have identified noncoding de novo variants in an enhancer of EBF3 called hs737 and further showed that there was an enrichment of deletions of this enhancer in individuals with neurodevelopmental disorders. In this present study, we generated a novel mouse line that deletes the highly conserved, orthologous mouse region of hs737 within the Rr169617 regulatory region, and characterized the molecular and phenotypic aspects of this mouse model. This line contains a 1,160 bp deletion within Rr169617 and through heterozygous crosses we found a deviation from Mendelian expectation (p = 0.02) with a significant depletion of the deletion allele (p = 5.8 × 10-4). Rr169617+/- mice had a reduction of Ebf3 expression by 10% and Rr169617-/- mice had a reduction of Ebf3 expression by 20%. Differential expression analyses in E12.5 forebrain, midbrain, and hindbrain in Rr169617+/+versus Rr169617-/- mice identified dysregulated genes including histone genes (i.e., Hist1h1e, Hist1h2bk, Hist1h3i, Hist1h2ao) and other brain development related genes (e.g., Chd5, Ntng1). A priori phenotyping analysis (open field, hole board and light/dark transition) identified sex-specific differences in behavioral traits when comparing Rr169617-/- males versus females; whereby, males were observed to be less mobile, move slower, and spend more time in the dark. Furthermore, both sexes when homozygous for the enhancer deletion displayed body composition differences when compared to wild-type mice. Overall, we show that deletion within Rr169617 reduces the expression of Ebf3 and results in phenotypic outcomes consistent with potential sex specific behavioral differences. This enhancer deletion line provides a valuable resource for others interested in noncoding regions in neurodevelopmental disorders and/or those interested in the gene regulatory network downstream of Ebf3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ce23e3e067854f3d2c474a36f36c782c44f4a0" target='_blank'>
              Generation and Characterization of a Knockout Mouse of an Enhancer of EBF3
              </a>
            </td>
          <td>
            Emily Cordova Hurtado, Janine M. Wotton, Alexander Gulka, Crystal Burke, Jeffrey K. Ng, Ibrahim Bah, Juana G. Manuel, Hillary Heins, Stephen A. Murray, David U. Gorkin, Jacqueline K. White, Kevin A. Peterson, Tychele N. Turner
          </td>
          <td>2025-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="T2T genome is a telomere-to-telomere high-quality genome with exceptional accuracy, continuity and integrity. It significantly filled some gaps of chromosome. However, the genomic features unique to the filled regions assembled in the T2T genome across species remain unclear. Here, we collected and compared T2T genomes with their older version from human (Homo sapiens), animal (Mus musculus), and plants (Arabidopsis thaliana, Musa acuminata, and Fragaria vesca). We explored the newly filled regions generated by the T2T genome in comparison to the old one, and focused on: (1) the comparison of their locations, (2) the distribution of genomic features, and (3) gene annotation and gene function for filled regions. Research indicated that the filled regions of the T2T genome ordinarily appear in the centromere and telomere regions, contain a great number of repetitive sequences. The T2T genome revealed the identification of a certain number of newly annotated genes, alongside significant variations in common annotated genes. Interestingly, the genomic features of filled regions exhibit distinct patterns between plants and human/animal. T2T genomes feature analysis serves as a crucial resource for studying pattern characteristics across diverse species and advancing the in-depth development of genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88844fa94943a306afc6b801a890b4cc98392746" target='_blank'>
              Genomic feature of filled regions of telomere-to-telomere genomes in five model species
              </a>
            </td>
          <td>
            Chu Xiong, Jun Zheng, Hui Zhang, Yunpeng Zhang, Lihong Hao, Min Wang, Zhen Liang
          </td>
          <td>2025-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="TNK2 is a ubiquitously expressed nonreceptor-type tyrosine kinase. TNK2 participates in tumorigenesis, and TNK2 activation has been found in various cancers; therefore, TNK2 is a promising target for cancer chemotherapy. While the TNK2 inhibitor XMD16-5 is highly selective, it inhibits cytokinesis at higher concentrations by targeting Aurora B kinase, a key enzyme for cell division. Cytokinesis failure frequently generates polyploid cells, and the surviving polyploid cells risk leading to cancer development and malignant progression via chromosome instability. In this study, to investigate the possibility that (R)-9bMS, a TNK2 inhibitor structurally related to XMD16-5, drives malignant progression by inducing abnormal cell division, we examined its effects on cell division, Aurora B autophosphorylation, and colony formation. Cell count results showed a reduction in the number of A431, HeLa S3, HCT116, and MCF7 cells upon TNK2 inhibitor treatment. Microscopic observation indicated the formation of multinucleated and nucleus-enlarged cells. An increase in DNA content was confirmed with flow cytometry, which was underpinned by an increased number of centrosomes. Time-lapse imaging revealed mitotic failure, such as mitotic slippage and cytokinesis failure, as a cause of polyploidization. Of note, TNK2 knockdown significantly increased multinucleated cells, but the effect was quite weak, suggesting that TNK2 inhibition may only partially contribute to mitotic failure and polyploidization. Expectedly, Aurora B phosphorylation was reduced by (R)-9bMS like XMD16-5, but not by TNK2 knockdown. Collectively, TNK2 inhibitors (R)-9bMS and XMD16-5 induce polyploidization via mitotic failure caused by the inhibition of Aurora B kinase rather than TNK2. Notably, (R)-9bMS treatment promoted anchorage-independent colony formation, a hallmark of cancer. Our findings suggest that (R)-9bMS at a high concentration risks promoting cancer development or malignant progression. Therefore, caution should be used when using TNK2 inhibitors for cancers where TNK2 activation is not the transforming mutation and higher concentrations of TNK2 inhibitors are required to slow proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83021c8768a54cea8919a992b5bb8d7150bf2fc1" target='_blank'>
              TNK2 Inhibitor (R)-9bMS Causes Polyploidization Through Mitotic Failure by Targeting Aurora B.
              </a>
            </td>
          <td>
            Mayu Murata, Hiroki Kuwajima, Junna Tanaka, Nanami Hasegawa, Ryuzaburo Yuki, Youhei Saito, Yuji Nakayama
          </td>
          <td>2024-12-01</td>
          <td>Cell biochemistry and function</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5730f3b1a7e75be7288a46c1b4d8d857387d1346" target='_blank'>
              Positive Selection Shapes Breast Cancer Tumor Suppressor Genes: Unveiling Insights into BRCA1, BRCA2, and MDC1 Stability.
              </a>
            </td>
          <td>
            Youssef M. Fadel, Marwan Khaled, Mohamed Emam, Nour H Marzouk, Sief El-Din Sobih, Habiba Abd-Elaty, Wafaa M Elrashedy, Gehad Mostafa, Salma Alm Eldeen, Mohaned Bador, Agostinho Antunes, Mohamed El Hadidi
          </td>
          <td>2024-12-16</td>
          <td>Journal of molecular evolution</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e3aede480e99c41ff823fe07aee10fb1b86fa8b" target='_blank'>
              DNA methylation-based analysis reveals accelerated epigenetic aging in giant cell-enriched adult-type glioblastoma
              </a>
            </td>
          <td>
            P. Cakmak, Philipp Jurmeister, I. Divé, P. Zeiner, Joachim P. Steinbach, Tim R Fenton, K. H. Plate, M. Czabanka, Patrick Harter, Katharina J. Weber
          </td>
          <td>2024-12-11</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Intra tumor heterogeneity complicates cancer therapy by providing tumors with the ability to alter their phenotypes and become more therapy resistant. Here, we tested the hypothesis that identifying and modulating expression of key state-specific transcription factors could be used as a strategy for driving cells to a more therapy-sensitive state. Recent single-cell studies have explored the inter and intra tumoral heterogeneity of osteosarcoma and identified gene pathways enriched in specific cell states. For example, metastatic tumors are characterized by an expression of genes in the TNF-α, PI3K, TGFß and mTOR pathways. We identified similar profiles in osteosarcoma patient-derived xenograft-derived cell lines and potential transcription factor drivers of these states. We then used perturb-seq to downregulate expression of key transcription factors and evaluated the effect of these modulations on single cell RNA profiles and drug responses. Knockdown of NFE2L3 or NR0B1 increased the proportion of cells sensitive to targeted therapy. This approach, which could potentially be applied to other cancers, could be used as a strategy to increase the response to targeted therapies by increasing the proportion of cells in a drug-sensitive state. Highlights Distinct transcriptomic states were identified in osteosarcoma cell lines using single-cell RNA sequencing. Lineage tracing identified states with differential sensitivity to therapy. Using perturb-seq, we identified transcription factors that drive cells towards a more sensitive state. The transcription factors NR0B1 and NFE2L3 were identified as targets capable of reprogramming cells to a sensitive state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6bc16f1a1ca4653fbc0122d06703e74482e021a" target='_blank'>
              Targeting Osteosarcoma heterogeneity to improve therapeutic response
              </a>
            </td>
          <td>
            Brenda Melano, Truc Dinh, Bahar Zirak, Mehran Karimzadeh, Brian J Woo, Benedict Choi, Hani Goodarzi, E. A. Sweet-Cordero, San Francisco
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background: The human FOXM1 transcription factor controls cell cycle progression and genome stability, and it has been correlated to the onset and progression of many tumor types. Methods: In our study, we collected all recent sequence and quantitative transcriptomics data about FOXM1, testing its presence across vertebrate evolution and its upregulation in cancer, both in bulk tissue contexts (by comparing the TCGA tumor dataset and the GTEx normal tissue dataset) and in single-cell contexts. Results: FOXM1 is significantly and consistently upregulated in all tested tumor types, as well as in tumor cells within a cancer microenvironment. Its upregulation reverberates in the upregulation of its target genes and can be used as a biomarker for poor cancer outcome in at least four tumor types. Conclusions: Despite its lack of cancer-related mutations and amplifications, the recurring upregulation of FOXM1 in all tumors puts a focusing lens on this gene as a candidate pan-cancer master regulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e19cd5cddb455ecb73607992ac0679d96d8dc34" target='_blank'>
              Pan-Cancer Upregulation of the FOXM1 Transcription Factor
              </a>
            </td>
          <td>
            Daniele Pozzobon, Arianna Bellezza, F. Giorgi
          </td>
          <td>2025-01-06</td>
          <td>Genes</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Here, we have discussed the molecular mechanisms of p53-responsive microRNAs dysregulation in response to genotoxic stress in diffuse large B-cell lymphoma (DLBCL) patients. The role of micro ribonucleic acids (microRNAs) in p53-signaling cellular stress has been studied. MicroRNAs are the small non-coding RNAs, which regulate genes expression at post-transcriptional level. Many of them play a crucial role in carcinogenesis and may act as oncogenes or suppressor of tumor growth. The understanding of the effect of p53-responsive microRNA dysregulation on oncogenesis achieved in recent decades opens wide opportunities for the diagnosis, prediction and of microRNA-based cancer therapy. Development of new bioinformatics tools and databases for microRNA supports DLBCL research. We overview the studies on the role of miRNAs in regulating gene expression associated with tumorigenesis processes, with particular emphasis on their role as tumor growth-suppressing factors. The starting point is a brief description of the classical microRNA biogenesis pathway and the role of p53 in regulating the expression of these molecules. We analyze various molecular mechanisms leading to this dysregulation, including mutations in the TP53 gene, DNA methylation, changes in host-genes expression or microRNA gene copy number, mutations in microRNA and microRNA biogenesis genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/740d5b189850125f69ad77979cf0d447a7b59eef" target='_blank'>
              Deregulation mechanisms and therapeutic opportunities of p53-responsive microRNAs in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            E. Voropaeva, Yuriy L. Orlov, Anastasia B. Loginova, O. Seregina, Vladimir N. Maksimov, T. I. Pospelova
          </td>
          <td>2025-01-07</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Colorectal cancer (CRC) ranks as the third most prevalent cancer globally and is the second leading cause of cancer mortality. FAM49B, a member of the FAM49 gene family, is a recently identified, evolutionarily conserved gene. Emerging studies indicate that FAM49B plays a role in various cancers, though its specific mechanism in CRC remains largely unexplored. In this study, we observed that FAM49B was abnormally expressed in CRC tissues and cell lines, with elevated expression correlating with poor patient prognosis. FAM49B knockdown markedly suppressed CRC cell proliferation by arresting the cell cycle and reducing cell migration and invasion. Single-cell RNA-seq (ScRNA-seq) analysis revealed that high FAM49B expression in malignant epithelial cell clusters was strongly linked to c-Myc oncogene activation. Further, FAM49B knockdown significantly reduced c-Myc expression by enhancing its K48 ubiquitination. We identified NEK9 as a direct interacting partner of FAM49B, with FAM49B knockdown inhibiting NEK9-Thr210 phosphorylation. Similarly, high NEK9 expression was linked to unfavorable prognosis in CRC. In FAM49B-overexpressing CRC cells, NEK9 knockdown significantly suppressed c-Myc expression, c-Myc-ser62 phosphorylation, and reduced cell proliferation, migration, and invasion. Thus, directly targeting the FAM49B/NEK9/c-Myc pathway presents a promising therapeutic approach for c-Myc positive CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96aecd72da94ed813728ad7c7716f5f1747f110c" target='_blank'>
              FAM49B drives colorectal cancer progression by stabilizing c-Myc through NEK9 phosphorylation.
              </a>
            </td>
          <td>
            Chen Lu, Tianyu Liu, E. Yimin, Lin Miao, Chunzhao Yu, Jianping Zhang, Xiagang Luo
          </td>
          <td>2025-01-01</td>
          <td>BioFactors</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e62a212b47508047875bfe6549eb4ee11cd53c6" target='_blank'>
              Survey for Activating Oncogenic Mutation Variants in Metazoan Germline Genes.
              </a>
            </td>
          <td>
            Karl E Krueger
          </td>
          <td>2024-11-26</td>
          <td>Journal of molecular evolution</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pseudouridine (Ψ) is a post-transcriptional modifier of RNA, often referred to as the ‘fifth nucleotide’ owing to its regulatory role in various biological functions as well as because of its significant involvement in the pathogenesis of human cancer. In recent years, research has revealed various Ψ modifications in different RNA types, including messenger RNA, transfer RNA, ribosomal RNA, small nuclear RNA, and long noncoding RNA. Pseudouridylation can significantly alter RNA structure and thermodynamic stability, as the Ψ-adenine (A) base pair is more stable than the typical uridine (U)-A base pair is due to its structural similarity to adenine. Studies have linked Ψ expression to the development and progression of several digestive system cancers, such as liver cancer and colorectal cancer, and nondigestive system cancers, such as breast cancer, non-small cell lung cancer, prostate cancer, glioblastoma, ovarian cancer, oral squamous cell carcinoma, and pituitary cancer. The present review briefly outlines the chemical structure, synthesis, and regulatory mechanisms of Ψ. This review summarizes the effects of pseudouridylation on various substrates of RNA and briefly discusses methods for detecting Ψ. Last, it focuses on how RNA pseudouridylation influences different cancers, emphasizing the search for novel approaches to cancer diagnosis, treatment, and prognosis through Ψ modification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9466c719e8cf6ff18a146061c808211210df53e3" target='_blank'>
              Advancements in pseudouridine modifying enzyme and cancer
              </a>
            </td>
          <td>
            Kaijie Liu, Shujun Zhang, Yafeng Liu, Xinjun Hu, Xinyu Gu
          </td>
          <td>2024-12-16</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) are important regulators of gene expression whose dysregulation is widely linked to tumourigenesis, tumour progression and Epithelial-Mesenchymal Transition (EMT), a developmental process that promotes metastasis when inappropriately activated. However, controversy has emerged regarding how many functional miRNAs are encoded in the genome, and to what extent non-regulatory products of RNA degradation have been mis-identified as miRNAs. Central to miRNA function is their capacity to associate with an Argonaute (AGO) protein and form an RNA-Induced Silencing Complex (RISC), which mediates target mRNA suppression. We report that numerous “miRNAs” previously reported in EMT and cancer contexts, are not incorporated into RISC and are not capable of endogenously silencing target genes, despite the fact that hundreds of publications in the cancer field describe their roles. Apparent function can be driven through the expression of artificial miRNA mimics which is not necessarily reflective of any endogenous gene regulatory function. We present biochemical and bioinformatic criteria that can be used to distinguish functional miRNAs from mistakenly annotated RNA fragments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d1ebefc068bdba0793a707a24cbd7cd0a5f423e" target='_blank'>
              Chasing non-existent “microRNAs” in cancer
              </a>
            </td>
          <td>
            Ayla Orang, Nicholas I Warnock, M. Migault, B. Dredge, A. Bert, Julie M Bracken, Philip A Gregory, Katherine A. Pillman, Gregory J Goodall, C. Bracken
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Glioblastoma (GBM) cells are highly heterogeneous and invasive, leading to treatment resistance and relapse. However, the molecular regulation in and distal to tumors remains elusive. Here, we collected paired tissues from the tumor core (TC) and peritumoral brain (PTB) for integrated snRNA-seq and snATAC-seq analyses. Tumor cells infiltrating PTB from TC behave more like oligodendrocyte progenitor cells than astrocytes at the transcriptome level. Dual-omics analyses further suggest that the distal regulatory regions in the tumor genome and specific transcription factors are potential determinants of regional heterogeneity. Notably, while activator protein 1 (AP-1) is active in all GBM states, its activity declines from TC to PTB, with another transcription factor, BACH1, showing the opposite trend. Combined inhibition of AP-1 and BACH1 more efficiently attenuates the tumor progression in mice and prolongs survival than either single-target treatment. Together, our work reveals marked molecular alterations of infiltrated GBM cells and a synergy of combination therapy targeting intratumor heterogeneity in and distal to GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c193d462d0c2dbf4aee7a761498e3e53b84808" target='_blank'>
              Single-cell multi-omics sequencing uncovers region-specific plasticity of glioblastoma for complementary therapeutic targeting
              </a>
            </td>
          <td>
            Xin Wang, Qian Sun, Tianbin Liu, Haoran Lu, Xuyi Lin, Weiwen Wang, Yang Liu, Yunting Huang, Guodong Huang, Haixi Sun, Qianxue Chen, Junmin Wang, Daofeng Tian, Fan-En Yuan, Longqi Liu, Bo Wang, Ying Gu, Baohui Liu, Liang Chen
          </td>
          <td>2024-11-22</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="With the practical implementation of the CRISPR/Cas technology for targeted genome editing, it has become possible to carry out genetic engineering manipulations with eukaryotic genomes with high efficiency. One of the key stages of this technology is the targeted induction of site-specific DNA cleavages (breaks). The cell repairs these breaks via one of two pathways: nonhomologous end joining or homologous recombination. The choice of DNA repair pathway is determined by the architecture of the sites at the DNA break area formed as a result of terminal resection and depends on the phases of the cell cycle. Nonhomologous end joining is the main pathway for repair of double-stranded DNA breaks in mammalian cells. It involves a nonspecific ligation reaction, the accuracy of which depends on the structure of the ends of the break, and can result in various insertions or deletions in the target region of the genome. Integration of the desired sequence into the genome occurs along the path of homologous recombination, the implementation of which requires a matrix with homology regions on both sides of the double-strand break. The introduction of a genetic construct into a given location in the genome is an important, but currently complex and labor-intensive task. At the same time, for fundamental studies of gene function and the creation of animal models of human diseases, the choice of the repair pathway can be of fundamental importance. This review is an attempt to combine and structure all known information on approaches to increasing the efficiency of DNA repair involving homologous recombination. The article lists the most effective strategies to shift the balance towards homologous repair, such as the use of inhibitors of the non-homologous end joining mechanism, regulation of key factors of homologous recombination, control of the cell cycle, chromatin status, construction of templates for homologous recombination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23816bdc012e17808d654e03c85cd07d45917c31" target='_blank'>
              How to shift the equilibrium of dna break repair in favor of homology recombination
              </a>
            </td>
          <td>
            O. A. Averina, S. A. Kuznetsova, O. Permyakov, P. V. Sergiev
          </td>
          <td>2024-12-30</td>
          <td>Molekulârnaâ biologiâ</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Transposable elements (TEs) are significant drivers of genome evolution, yet their recent dynamics and impacts within and among species, as well as the roles of host genes and non-coding RNAs in the transposition process, remain elusive. With advancements in large-scale pan-genome sequencing and the development of open data sharing, large-scale comparative genomics studies have become feasible. Here, we performed complete de novo TE annotations and identified active TEs in 310 plant genome assemblies across 119 species and seven crop populations. Using 811 high-quality genomes, we detected 13 844 553 TE-induced structural variants (TE-SVs), providing unprecedented resolution in delineating recent TE activities. Our integrative analysis revealed a mutual evolutionary relationship between TEs and host genomes. On one hand, host genes and ncRNAs are involved in the transposition process, as evidenced by their colocalization and coactivation with TEs, and may play a role in chromatin regulation. On the other hand, TEs drive genetic innovation by promoting the duplication of host genes and inserting into regulatory regions. Moreover, genes influenced by active TEs are linked to plant growth, nutrient absorption, storage metabolism and environmental adaptation, aiding in crop domestication and adaptation. This TE dynamics atlas not only reveals evolutionary and functional features linked to transposition activity but also highlights the role of TEs in crop domestication and adaptation, paving the way for future exploration of TE-mediated genome evolution and crop improvement strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e3e6dd3ee1f3baea529da4c3c4ab4bf7659f210" target='_blank'>
              Deciphering recent transposition patterns in plants through comparison of 811 genome assemblies.
              </a>
            </td>
          <td>
            Yan Huang, Sunil Kumar Sahu, Xin Liu
          </td>
          <td>2025-01-10</td>
          <td>Plant biotechnology journal</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The cohesin complex is a critical regulator of gene expression. STAG2 is the most frequently mutated cohesin subunit across several cancer types and is a key tumor suppressor in lung cancer. Here, we coupled somatic CRISPR-Cas9 genome editing and tumor barcoding with an autochthonous oncogenic KRAS-driven lung cancer model and showed that STAG2 is uniquely tumor-suppressive among all core and auxiliary cohesin components. The heterodimeric complex components PAXIP1 and PAGR1 have highly correlated effects with STAG2 in human lung cancer cell lines, are tumor suppressors in vivo, and are epistatic to STAG2 in oncogenic KRAS-driven lung tumorigenesis in vivo. STAG2 inactivation elicits changes in gene expression, chromatin accessibility, and 3D genome conformation that impact the cancer cell state. Gene expression and chromatin accessibility similarities between STAG2- and PAXIP1-deficient neoplastic cells further relate STAG2-cohesin to PAXIP1/PAGR1. These findings reveal a STAG2-PAXIP1/PAGR1 tumor-suppressive axis and uncover novel PAXIP1-dependent and PAXIP1-independent STAG2-cohesin-mediated mechanisms of lung tumor suppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b77c9e3bd051e20d8c03518cc31ea52a29e0f1f0" target='_blank'>
              A STAG2-PAXIP1/PAGR1 axis suppresses lung tumorigenesis.
              </a>
            </td>
          <td>
            Emily L Ashkin, Yuning J Tang, Haiqing Xu, King L. Hung, J. Belk, Hongchen Cai, Steven S. Lopez, D. N. Dolcen, Jess D. Hebert, Rui Li, Paloma A. Ruiz, Tula Keal, Laura Andrejka, Howard Y. Chang, D. Petrov, Jesse R. Dixon, Zhichao Xu, M. Winslow
          </td>
          <td>2024-12-09</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Protoplast-based systems have been utilised in a wide variety of plant species to enable genome editing without chromosomal introgression of foreign DNA into plant genomes. This allows elite cultivars to be edited without further genetic segregation, preserving their unique genetic composition and their regulatory status as non-transgenic. This can be achieved by DNA-free genome editing in protoplasts, followed by regeneration. However, protoplast isolation presents a barrier to the development of advanced breeding technologies in raspberry and no protocol has been published for DNA-free genome editing in the species. Pre-assembled ribonucleoprotein complexes (RNPs) do not require cellular processing and the commercial availability of Cas9 proteins and synthetic guide RNAs has streamlined genome editing protocols. This study presents a novel high-yielding protoplast isolation protocol from raspberry stem cultures and RNP-mediated transfection of protoplast with CRISPR-Cas9. Targeted mutagenesis of the phytoene desaturase gene at two intragenic loci resulted in an editing efficiency of 19%, though estimated efficiency varied depending on the indel analysis technique. Only amplicon sequencing was sensitive enough to confirm genome editing in a low efficiency sample. To our knowledge, this study constitutes the first use of DNA-free genome editing in raspberry. This protocol provides a valuable platform for understanding gene function and facilitates the development of precision breeding in this important soft fruit crop.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/497a86a932270b0bc385e49fa5362b3ca003c8a0" target='_blank'>
              DNA-free CRISPR Genome Editing in Raspberry (Rubus idaeus) through RNP-mediated Protoplast Transfection and Comparison of Indel Analysis Techniques
              </a>
            </td>
          <td>
            Ryan Creeth, Andrew Thompson, Zoltan Kevei
          </td>
          <td>2025-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Targeting DNA damage response (DDR) pathways represents one of the principal approaches in cancer therapy. However, defects in DDR mechanisms, exhibited by various tumors, can also promote tumor progression and resistance to therapy, negatively impacting patient survival. Therefore, identifying new molecules from natural extracts could provide a powerful source of novel compounds for cancer treatment strategies. In this context, we investigated the role of oleanolic acid (OA), identified in fermented Aglianico red grape pomace, in modulating the DDR in response to camptothecin (CPT), an inhibitor of topoisomerase I. Specifically, we found that OA can influence the choice of DNA repair pathway upon CPT treatment, shifting the repair process from homologous recombination gene conversion to single-strand annealing. Moreover, our data demonstrate that combining sub-lethal concentrations of OA with CPT enhances the efficacy of topoisomerase I inhibition compared to CPT alone. Overall, these findings highlight a new role for OA in the DDR, leading to a more mutagenic DNA repair pathway and increased sensitivity in the HeLa cancer cell line.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cba28716fad27e1778281b97439ea3f957928277" target='_blank'>
              Oleanolic Acid Modulates DNA Damage Response to Camptothecin Increasing Cancer Cell Death
              </a>
            </td>
          <td>
            Giulio Mazzarotti, Maria Cuomo, Maria Carmen Ragosta, Andrea Russo, Margherita D’Angelo, Annamaria Medugno, Giuseppe Maria Napolitano, C. A. Iannuzzi, I. Forte, Rosa Camerlingo, Sharon Burk, F. Errichiello, Luigi Frusciante, M. Forino, M. R. Campitiello, M. de Laurentiis, A. Giordano, Luigi Alfano
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The human genome is packaged within a three-dimensional (3D) nucleus and organized into structural units known as compartments, topologically associating domains (TADs), and loops. TAD boundaries, separating adjacent TADs, have been found to be well conserved across mammalian species and more evolutionarily constrained than TADs themselves. Recent studies show that structural variants (SVs) can modify 3D genomes through the disruption of TADs, which play an essential role in insulating genes from outside regulatory elements' aberrant regulation. However, how SV affects the 3D genome structure and their association among different aspects of gene regulation and candidate cis-regulatory elements (cCREs) have rarely been studied systematically. Here, we assess the impact of SVs intersecting with TAD boundaries by developing an integrative Hi-C analysis pipeline, which enables the generation of an in-depth catalog of TADs and TAD boundaries in human lymphoblastoid cell lines (LCLs) to fill the gap of limited resources. Our catalog contains 18,865 TADs, including 4596 sub-TADs, with 185 SVs (TAD-SVs) that alter chromatin architecture. By leveraging the ENCODE registry of cCREs in humans, we determine that 34 of 185 TAD-SVs intersect with cCREs and observe significant enrichment of TAD-SVs within cCREs. This study provides a database of TADs and TAD-SVs in the human genome that will facilitate future investigations of the impact of SVs on chromatin structure and gene regulation in health and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fe7b730dd8aa8992f72d3009d5b16854b631e1b" target='_blank'>
              An integrative TAD catalog in lymphoblastoid cell lines discloses the functional impact of deletions and insertions in human genomes.
              </a>
            </td>
          <td>
            Chong Li, M. Bonder, Sabriya Syed, Matthew Jensen, Mark B. Gerstein, Michael C Zody, Mark J. P. Chaisson, M. Talkowski, T. Marschall, Jan O. Korbel, E. Eichler, Charles Lee, Xinghua Shi
          </td>
          <td>2024-12-05</td>
          <td>Genome research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Cancer’s epigenetic landscape, a labyrinthine tapestry of molecular modifications, has long captivated researchers with its profound influence on gene expression and cellular fate. This review discusses the intricate mechanisms underlying cancer epigenetics, unraveling the complex interplay between DNA methylation, histone modifications, chromatin remodeling, and non-coding RNAs. We navigate through the tumultuous seas of epigenetic dysregulation, exploring how these processes conspire to silence tumor suppressors and unleash oncogenic potential. The narrative pivots to cutting-edge technologies, revolutionizing our ability to decode the epigenome. From the granular insights of single-cell epigenomics to the holistic view offered by multi-omics approaches, we examine how these tools are reshaping our understanding of tumor heterogeneity and evolution. The review also highlights emerging techniques, such as spatial epigenomics and long-read sequencing, which promise to unveil the hidden dimensions of epigenetic regulation. Finally, we probed the transformative potential of CRISPR-based epigenome editing and computational analysis to transmute raw data into biological insights. This study seeks to synthesize a comprehensive yet nuanced understanding of the contemporary landscape and future directions of cancer epigenetic research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf231d1889782a99759e5b17e3a47e9d23349015" target='_blank'>
              Mechanisms and technologies in cancer epigenetics
              </a>
            </td>
          <td>
            Z. Sherif, O. Ogunwobi, H. Ressom
          </td>
          <td>2025-01-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Gene conversion is a process in which genetic material from a donor sequence is unidirectionally copied to an acceptor sequence during the homologous recombination repair of a DNA double-strand break. Although gene conversion has been widely studied in the context of meiosis, hereditary diseases, and cancer development, gene conversion between parental homologs in the zygotes remains controversial. Here, we developed a method to detect interparental gene conversions by focusing on Mendelian errors and identified gene conversion events in one out of every 21.8 births. Some of these events were observed in genetic regions, potentially affecting offspring phenotypes. Interparental gene conversion leads to the offspring inheriting two identical alleles from one parent, resulting in a loss of heterozygosity. Our findings suggest that naturally occurring interparental gene conversions may provide a novel mechanism for the development of certain genetic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26ddbca341c42c36a2e20339125b0cf1e5d09505" target='_blank'>
              Interparental Gene Conversion in General Population: A Novel Mechanism For Loss of Heterozygosity
              </a>
            </td>
          <td>
            Jumpei Toratani, Masahito Tachibana, Junichi Sugawara, Atsushi Sugawara, Takeki Sato, Yuri Takahashi, Hiroaki Hiraga, Emi Yokoyama, Zen Watanabe, Masatoshi Saito, Nobuo Yaegashi, Gen Tamiya, J. Takayama
          </td>
          <td>2024-12-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="DNA repair dysregulation is a key driver of cancer development. Understanding the molecular mechanisms underlying DNA repair dysregulation in cancer cells is crucial for cancer development and therapies. Here, we report that enhancer of zeste homolog 2 (EZH2) directly methylates poly(adenosine diphosphate–ribose) polymerase-1 (PARP-1), an essential enzyme involved in DNA repair, and regulates its activity. Functionally, EZH2-catalyzed methylation represses PARP1 catalytic activity, down-regulates the recruitment of x-ray repair cross-complementing group-1 to DNA lesions and its associated DNA damage repair; on the other hand, it protects the cells from nicotinamide adenine dinucleotide overconsumption upon DNA damage formation. Meanwhile, EZH2-mediated methylation regulates PARP1 transcriptional and oncogenic activity, at least in part, through impairing PARP1-E2F1 interaction and E2F1 transcription factor activity. EZH2 and PARP1 inhibitors synergistically suppress prostate cancer growth. Collectively, our findings uncover an insight of EZH2 functions in fine-tuning PARP1 activity during DNA damage repair and cancer progression, which provides a rationale for combinational targeting EZH2 and PARP1 in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95e5d1675eeb186efe28f57213417662345e8e12" target='_blank'>
              EZH2 directly methylates PARP1 and regulates its activity in cancer
              </a>
            </td>
          <td>
            Qingshu Meng, Jiangchuan Shen, Yanan Ren, Qi Liu, Rui Wang, Qiaqia Li, Weihua Jiang, Quan Wang, Yixiang Zhang, Jonathan C Trinidad, Xiaotong Lu, Ting-You Wang, Yanqian Li, Chaehyun Yum, Yang Yi, Yongyong Yang, Dongyu Zhao, C. Harris, S. Kalantry, Kaifu Chen, Rendong Yang, Hengyao Niu, Qi Cao
          </td>
          <td>2024-11-27</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02c600bc41b490f1030b42efdd32168810e5ead4" target='_blank'>
              The dynamics of loss of heterozygosity events in genomes.
              </a>
            </td>
          <td>
            Abhishek Dutta, J. Schacherer
          </td>
          <td>2025-01-02</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="ABSTRACT Background SET domain‐containing protein 4 (SETD4) is a histone methyltransferase that has been shown to modulate cell proliferation, differentiation, and inflammatory responses by regulating histone H4 trimethylation (H4K20me3). Previous reports have demonstrated its function in the quiescence of cancer stem cells as well as drug resistance in several cancers. A limited number of systematic studies have examined SETD4's role in the tumor microenvironment, pathogenesis, prognosis, and therapeutic response. Methods Utilizing The Cancer Genome Atlas database, and other publicly accessible platforms, we comprehensively analyzed SETD4 gene expression, methylation patterns, and prognostic significance. Furthermore, we investigated its association with cancer‐related pathways, the immune microenvironment, immunotherapy markers, and drug resistance signatures of chemotherapy. Additionally, qRT‐PCR was performed to validate SETD4 expression in clinical specimens. Results The expression of SETD4 was abnormal across a variety of cancer types and the expression of SETD4 in colorectal cancer tissues was verified in clinical specimens. The upregulation of SETD4 may be a prognostic risk factor predicting poor overall survival and progression‐free survival. The analysis revealed that the mRNA level of SETD4 was modulated by promoter methylation, and patients with lower methylation levels showed shorter survival times. Pathway analysis showed that SETD4 influenced several key cell cycle pathways, including the G2M checkpoint, and mitotic spindle pathways. In addition, SETD4 negatively affects immune cell infiltration in most cancers, including B cells, CD8 T cells, and macrophages. The correlation between SETD4 and cancer stemness as well as homologous recombination deficiency varied across tumor types, suggesting that SETD4 may play a multifaceted role in tumor resistance. Notably, we identified several potential agents targeting SETD4. Conclusions This study demonstrates that SETD4 is an immune‐oncogenic molecule in multiple cancers, with the potential to be a diagnosis, prognosis, and targeted therapy marker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f76f7155098d30f047c5ee80fc1fcf8c03b367a" target='_blank'>
              Identification of SETD4 as an Onco‐Immunological Biomarker Encompassing the Tumor Microenvironment, Prognoses, and Therapeutic Responses in Various Human Cancers
              </a>
            </td>
          <td>
            Yuyun Zhong, Ruiqi Wang, Zijie Huang, Zhaoting Hu, Bin Peng, Bin Chen, Liyue Sun
          </td>
          <td>2025-01-01</td>
          <td>Immunity, Inflammation and Disease</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="5-Methylcytosine (5mC) is a common source of somatic mutations. Deamination of 5mC to thymine generates a G/T mismatch, which occurs spontaneously and must be repaired prior to DNA replication to avoid mutation. We generated genetically engineered mice and cell lines to define DNA repair pathways that protect against methylation damage. We observed a low background mutation rate in mouse bone marrow or colon, typically 0.2-0.5 CG>TG mutations/genome/day. This increased 3-7 fold in cells lacking the glycosylase Methyl-binding domain 4 (Mbd4), one of the few glycosylases capable of excising thymine from G/T mismatches. We found no role for Thymine DNA glycosylase (Tdg) in methylation damage repair. Instead, our results support cooperation between Mbd4 and the mismatch repair (MMR) complex MutSα (Msh6:Msh2), evident through elevated rates of methylation damage in Msh6-deficient cells; increasing to 2.6-4.8 CG>TG mutations/genome/day in primary cells and up to 13.9 CG>TG mutations/genome/day in cell lines. Our findings support the view that MutSα has DNA repair activity outside of replication. While loss of Mbd4 elevates methylation damage selectively, the broader functionality of MutSα explains why mutational signatures linked to Msh6-deficiency are variable and reflect the replicative history of the cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d89af6527eda652ec94285879400d97b8fdfca77" target='_blank'>
              Mbd4 and MutSα protect cells from spontaneous deamination of 5-methylcytosine
              </a>
            </td>
          <td>
            R. Bilardi, C. Flensburg, Zhen Xu, Emily B. Derrick, Andrew Kueh, I. Majewski
          </td>
          <td>2024-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The precise regulation of chromatin composition is critical to gene expression and cellular identity, and thus a key component in development and disease. Robust assays to study chromatin features, including histone post-translational modifications (PTMs) and chromatin-associated proteins (e.g., transcription factors or PTM readers), are essential to understand their function and identify novel therapeutic strategies. To this end, Cleavage Under Targets and Release Using Nuclease (CUT&RUN) has emerged as a powerful tool for high-resolution epigenomic profiling. The approach has been successfully applied to numerous cell and tissue types, informing on target genomic distribution with unprecedented sensitivity and throughput. Here, we provide a detailed CUT&RUN protocol from sample collection through data analysis, including best practices and defined controls to ensure specific, efficient, and robust target profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e8623eabf6a7c5745150c39568eca453c8aedb8" target='_blank'>
              High-efficiency genomic mapping of chromatin-associated targets with CUT&RUN
              </a>
            </td>
          <td>
            Tessa M. Firestone, Bryan J. Venters, Katherine Novitzky, L. Albertorio-Sáez, Courtney A. Barnes, Karlie N. Fedder-Semmes, N. Hall, Allison R. Hickman, Mark Kaderli, C. L. Windham, M. Marunde, Danielle N. Maryanski, Kelsey Noll, Leslie Shannon, Jennifer Spengler, M. Cowles, Zu-Wen Sun, Michael-Christopher Keogh, Andrea L. Johnstone, Ellen N. Weinzapfel, Lu Sun
          </td>
          <td>2024-12-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT Pleomorphic adenoma (PA) is the most common salivary gland tumor. PAs are characterized by chromosomal rearrangements of 8q12 and 12q14‐15, leading to gene fusions involving the PLAG1 and HMGA2 oncogenes. Here, we performed the first comprehensive study of the transcriptomic and gene fusion landscape of 38 cytogenetically characterized PAs. RNA‐seq identified PLAG1 or HMGA2 fusions in 33/38 cases (87%), of which 15 were novel fusions. Fusions were found also in tumors with normal karyotype, demonstrating that they are generated by cryptic rearrangements. PLAG1 was mainly activated by promoter swapping and HMGA2 by truncation of its 3′‐part. RNA‐seq revealed upregulation of genes involved in extracellular matrix production, WNT‐signaling, and epithelial‐mesenchymal transition in PA compared to normal salivary tissue. Principal component analysis identified two PA subclusters characterized by PLAG1‐ and HMGA2‐activation, respectively, that differed in expression of genes involved in the immune system, cell adhesion, and microenvironment remodeling. Moreover, comparative analyses of PA and salivary carcinomas revealed that PA resembles myoepithelial carcinoma. Our study reveals new oncogenic gene fusions and expands our knowledge about the molecular underpinnings of PA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d4eee126d1c832c25e92d4149703cc29efb6133" target='_blank'>
              The Transcriptomic and Gene Fusion Landscape of Pleomorphic Salivary Gland Adenomas
              </a>
            </td>
          <td>
            M. K. Afshari, Paloma Tejera Nevado, A. Fehr, Junchi Huang, Fredrik Jäwert, Jonas A. Nilsson, Göran Stenman, Mattias K. Andersson
          </td>
          <td>2025-01-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc8fc819ffccd10115ccb040d51408d1fd6a5f7" target='_blank'>
              Nanopore sequencing reveals that DNA replication compartmentalisation dictates genome stability and instability in Trypanosoma brucei
              </a>
            </td>
          <td>
            M. Krasiļņikova, Catarina A Marques, E. Briggs, C. Lapsley, Graham Hamilton, Dario Beraldi, K. Crouch, R. McCulloch
          </td>
          <td>2025-01-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Protein phosphatases are critical for regulating cell signaling, cell cycle, and cell fate decisions, and their dysregulation leads to an array of human diseases like cancer. The dual specificity phosphatases (DUSPs) have emerged as important factors driving tumorigenesis and cancer therapy resistance. DUSP12 is a poorly characterized atypical DUSP widely conserved throughout evolution. Although no direct substrate has been firmly established, DUSP12 that has been implicated in protecting cells from stress, regulating ribosomal biogenesis, and modulating cellular DNA content. In this study, we used affinity- and proximity-based biochemical purification approaches coupled to mass spectrometry to identify the zinc finger protein ZPR9 as a novel DUSP12 interactor, which was validated by in-cell and in-vitro IP assays. Interestingly, ZPR9 binds to the unique zinc-binding domain of DUSP12, which previous reports indicated was important for many of DUSP12’s functions within the cell. Prior studies had implicated ZPR9 as a modulator of apoptosis, but it remained unclear if and how ZPR9 participated in the cell cycle and, more so, how it promoted cell death. Using mass spectrometry analyses, we found that overexpression of DUSP12 promoted de-phosphorylation of ZPR9 at Ser143. Overexpression of ZPR9, but not Ser143 phosphomimetic and phosphorylation-deficient mutants, led to an increase in pre-metaphase mitotic defects while knockdown of DUSP12 also showed mitotic defects in metaphase. Furthermore, knockdown of DUSP12 promoted, while knockdown of ZPR9 suppressed, stress-induced apoptosis. Our results support a model where DUSP12 protects cells from stress-induced apoptosis by promoting de-phosphorylation of ZPR9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7479ca7476284506ba203b0cbc2b3084cb60b315" target='_blank'>
              DUSP12 promotes cell cycle progression and protects cells from cell death by regulating ZPR9
              </a>
            </td>
          <td>
            Mai Abdusamad, Xiao Guo, Ivan Ramirez, Erick F. Velasquez, Whitaker Cohn, Ankur A. Gholkar, Julian P Whitelegge, Jorge Z. Torres
          </td>
          <td>2025-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract H-DNA is an intramolecular DNA triplex formed by homopurine/homopyrimidine mirror repeats. Since its discovery, the field has advanced from characterizing the structure in vitro to discovering its existence and role in vivo. H-DNA interacts with cellular machinery in unique ways, stalling DNA and RNA polymerases and causing genome instability. The foundational S1 nuclease and chemical probing technologies originally used to show H-DNA formation have been updated and combined with genome-wide sequencing methods for large-scale mapping of secondary structures. There is evidence for triplex H-DNA’s role in polycystic kidney disease (PKD), cancer, and numerous repeat expansion diseases (REDs). In PKD, an H-DNA forming repeat region within the PKD1 gene stalls DNA replication and induces fragility. H-DNA-forming repeats in various genes have a role in cancer; the most well-studied examples involve H-DNA-mediated fragility causing translocations in multiple lymphomas. Lastly, H-DNA-forming repeats have been implicated in four REDs: Friedreich’s ataxia, GAA-FGF14-related ataxia, X-linked Dystonia Parkinsonism, and cerebellar ataxia, neuropathy and vestibular areflexia syndrome. In this review, we summarize H-DNA’s discovery and characterization, evidence for its existence and function in vivo, and the field’s current knowledge on its role in physiology and pathology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d258222649c5f8f00b18b553e105c9ceb00bafd" target='_blank'>
              Triplex H-DNA structure: the long and winding road from the discovery to its role in human disease
              </a>
            </td>
          <td>
            Julia A. Hisey, Chiara Masnovo, S. Mirkin
          </td>
          <td>2024-10-01</td>
          <td>Nar Molecular Medicine</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Aneuploidy, the major cause of miscarriages in humans, is pervasive in early embryos but robustly dampened during development, allowing for healthy births. Later in life, aneuploidy correlates with pathological conditions including cancer. Identification of the mechanisms underlying the elimination of aneuploid cells is relevant in development and disease. Here we generated cells carrying molecularly defined segmental monosomies and trisomies and characterized their immediate impact on cellular behavior. Our data reveal signs of out-competition of cells carrying monosomies in genomic regions devoid of previously known haploinsufficient genes. Dose-dependent effects of single genes or a discrete number of genes contribute to the observed cellular behaviors. By simultaneously inducing cells carrying monosomies and trisomies of the same genomic location, we present evidence that segmental trisomies potentiate or alleviate the negative effects of the monosomy on growth, thus revealing a key role of cell interactions in defining the in vivo elimination of aneuploid cells. Highlights Segmental monosomies cause growth impairment and are out-competed. Growth impairment relies on dose-dependent effects of single genes or a discrete number of genes. Segmental trisomies of up to 1500 genes do not have a major impact on proliferation and survival. Cell competition is modulated by the presence of cells trisomic for the same genomic region. eTOC Blurb Aneuploidy has a negative impact on the growth and proliferation of all animal cells analyzed so far. Fusari et al. unravel a role of cell interactions in defining the in vivo elimination of aneuploid cells through cell competition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc128d47b336584a75037f0f1e1bbbd450a62027" target='_blank'>
              Depletion of aneuploid cells is shaped by cell-to-cell interactions
              </a>
            </td>
          <td>
            Elena Fusari, M. Muzzopappa, Juliette Gracia, Marco Milán
          </td>
          <td>2024-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract The phosphorylation of histone H2A.X into γH2A.X is a crucial early event in the DNA damage response, marking DNA damage sites and initiating repair processes. While ATM kinase is traditionally recognized as the primary mediator of H2A.X phosphorylation, our study identifies serine/threonine kinase 39 (STK39) as a novel enhancer of this critical signaling pathway. We demonstrate that after DNA damage, STK39 undergoes phosphorylation by the ATM kinase, facilitating its interaction with the Mre11-Rad50-Nbs1 complex and subsequent recruitment to chromatin. This recruitment enables STK39 to further phosphorylate H2A.X, thus amplifying γH2A.X production and promoting homologous recombination repair. Notably, we observe a significant upregulation of STK39 in pancreatic adenocarcinoma (PAAD) tissues, correlating with heightened resistance to PARPi therapy. Furthermore, we demonstrate the synergistic efficacy of combining STK39 inhibition with PARP inhibitors in suppressing and reversing PAAD growth. This study not only provides new insights into the molecular dynamics of H2A.X phosphorylation but also highlights the therapeutic potential of targeting STK39 to enhance PARPi sensitivity in PAAD (created with BioRender).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7b5527588649efe249bff697a10bc6cd9dfa473" target='_blank'>
              STK39-mediated amplification of γ-H2A.X promotes homologous recombination and contributes to PARP inhibitor resistance
              </a>
            </td>
          <td>
            Yi Xu, Changying Li, Huan Yin, S. Nowsheen, Xin Xu, Wenjuan Kang, Xin Liu, Lifeng Chen, Zhenkun Lou, Junlin Yi, Min Deng
          </td>
          <td>2024-11-26</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Cancer development involves a complex interplay between genetic and epigenetic factors, with emerging evidence highlighting the pivotal role of competitive endogenous RNA (ceRNA) networks in regulating gene expression. However, the influence of ceRNA networks by aberrant DNA methylation remains incompletely understood. In our study, we proposed DMceNet, a computational method to characterize the effects of DNA methylation on ceRNA regulatory mechanisms and apply it across eight prevalent cancers. By integrating methylation and transcriptomic data, we constructed methylation-driven ceRNA networks and identified a dominant role of lncRNAs within these networks in two key ways: (i) 17 cancer-shared differential methylation lncRNAs (DMlncs), including PVT1 and CASC2, form a Common Cancer Network (CCN) affecting key pathways such as the G2/M checkpoint, and (ii) 24 cancer-specific DMlncs construct unique ceRNA networks for each cancer type. For instance, in LUAD and STAD, hypomethylation drives DMlncs like PCAT6 and MINCR, disrupting the Wnt signaling pathway and apoptosis. We further investigated the characteristics of these methylation-driven ceRNA networks at the cellular level, revealing how methylation-driven dysregulation varies across distinct cell populations within the tumor microenvironment. Our findings also demonstrate the prognostic potential of cancer-specific ceRNA relationships, highlighting their relevance in predicting patient survival outcomes. This integrated transcriptomic and epigenomic analysis provides new insights into cancer biology and regulatory mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7483d8802056f68722b0432bf0cd28325b090cf3" target='_blank'>
              Constructing methylation-driven ceRNA networks unveil tumor heterogeneity and predict patient prognosis.
              </a>
            </td>
          <td>
            Xinyu Li, Chuo Peng, Hongyu Liu, Mingjie Dong, Shujuan Li, Weixin Liang, Xia Li, Jing Bai
          </td>
          <td>2024-11-27</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/067cac427326fdea9459d93348affd2bc0e1346c" target='_blank'>
              Nucleus-translocated GCLM promotes chemoresistance in colorectal cancer through a moonlighting function
              </a>
            </td>
          <td>
            Jin-Fei Lin, Ze-xian Liu, Dong-Liang Chen, , Fen Cao, Kai Yu, Ting Li, Hai-Yu Mo, Hui Sheng, Zhi-Bing Liang, Kun Liao, Yi Han, Shan-Shan Li, Zhao-Lei Zeng, Song Gao, Huai-qiang Ju, Rui-Hua Xu
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4623fb2b0df3467d1b2cf2d35e43a732c3aed069" target='_blank'>
              DNA targeting by compact Cas9d and its resurrected ancestor
              </a>
            </td>
          <td>
            Rodrigo Fregoso Ocampo, Jack P. K. Bravo, Tyler L. Dangerfield, Isabel Nocedal, Samatar Jirde, Lisa M. Alexander, Nicole C. Thomas, Anjali Das, Sarah Nielson, Kenneth A. Johnson, Christopher T. Brown, Cristina N. Butterfield, Daniela S. A. Goltsman, David W. Taylor
          </td>
          <td>2025-01-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Extensive intratumor heterogeneity in glioblastoma (GBM) impedes successful treatment and complicates drug discovery as it is not obvious which cells a tumor is most dependent on. Here, we posit that single-cell-resolution transcriptomic data can be integrated with loss-of-function screens to identify the most critical cells to target within a tumor. We parsed CRISPR screen data from the Dependency Map (DepMap) Consortium and identified a GBM Dependency Signature (GDS) − 168 genes that are essential for GBM cell viability in vitro. Through similarity scoring of GDS transcriptomic profiles in single-cell RNA-sequencing (scRNA-seq) data and iterative hierarchical clustering, we identify and report here 3 single-cell vulnerability states (VS) characterized in 49 GBM tumors using both scRNA-seq and spatial transcriptomic data. These VS reflect single-cell gene dependencies and differ significantly in enrichment profiles and spatial distributions. Importantly, the proportion of VS in each GBM tumor is variable, suggesting a means of stratifying patients in clinical trials. Collectively, we have developed a novel computational pipeline to identify unique vulnerability states in GBM and other cancers, which can be used to identify existing or novel drugs for incurable diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4aef7e027ff56302092f68c8a779be84bebb597e" target='_blank'>
              Identification of disease-specific vulnerability states at the single-cell level
              </a>
            </td>
          <td>
            Matthew D’Antuono, Madison Sharp, Rishika Chowdary, Michael E. Ivan, Ricardo J Komotar, Robert K. Suter, N. Ayad
          </td>
          <td>2024-12-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Precursor B cell acute lymphoblastic leukemia (pre-B-ALL) arises as a result of precursor B cells acquiring driver mutations that lead to arrested differentiation and increased proliferation. Identification of driver mutations and understanding their biological function is critical to understanding pre-B-ALL development and advancing disease treatment. Using a mouse model of pre-B-ALL driven by deletion of genes encoding the related E26-transformation-specific (ETS) transcription factors PU.1 and Spi-B, we performed whole exome sequencing to identify secondary driver mutations. We identified recurrent variants in E26 transformation-specific transcription variant 5 (ETV5) resulting in R392P, V444I, and T505A amino acid changes. We found that the R392P and V444I variants altered the ability of ETV5 to bind to DNA using electrophoretic mobility shift assay. R392P and V444I variants did not activate a Dual-Specificity-Phosphatase 6 (DUSP6) reporter. In contrast, T505A ETV5 could interact with DNA and activate the DUSP6 promoter. To determine biological function, we forced expression of wild type, R392P, V444I, or T505A ETV5 in an interleukin-7-dependent pre-B cell line. Proliferation and apoptosis assays showed that T505A ETV5 conferred a proliferative advantage to pre-B cells. RNA sequencing showed that expression of ETV5 variants significantly altered gene expression in cultured cells. Through gene set enrichment analysis, T505A was suggested to downregulate the p53 pathway and the anti-proliferative protein, B cell translocation gene 2 (encoded by Btg2). In summary, these data suggest that ETV5 mutations play a role in pre-B-ALL by affecting proliferation and cell survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50f29fab27dbb4d851eea7f525bf89bbde3e863a" target='_blank'>
              T505A variant of ETV5 promotes proliferation of precursor B cells in a mouse model of acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Joshua Yi, Michael Wu, Michaela L. Dowling, Allanna C. E. MacKenzie, James Iansavitchous, R. DeKoter
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Simple Summary Histone methyltransferases (HMTs) and histone demethylases (HDMs) are enzymes that modify histones, the proteins around which DNA is wrapped. These modifications play a crucial role in regulating gene expression by adding or removing methyl groups, which can activate or silence genes. In this review, we explore the roles of HMTs and HDMs in the development of cancer and neurodegenerative diseases, specifically, Alzheimer’s, Parkinson’s, and Huntington’s diseases. Regarding cancer, HMTs and HDMs can drive tumor growth and progression by altering gene expression patterns that regulate cell proliferation and survival. For example, the dysregulation of enzymes like KMT2D (an HMT) and KDM2A (an HDM) is linked to various cancers including gastric cancer by influencing the chromatin structure and gene activity. In neurodegenerative diseases, these enzymes impact the health of neurons by modifying genes involved in brain function. In Alzheimer’s, Parkinson’s, and Huntington’s diseases, changes in HMT and HDM activity can lead to the dysregulation of genes critical for neuron survival, contributing to cognitive decline and motor dysfunction. Herein, we highlight the similarities and differences in how HMTs and HDMs function in cancer and neurodegeneration. Understanding these roles may reveal new therapeutic targets that address the epigenetic underpinnings of both cancer and neurodegenerative diseases, offering hope for innovative treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb434586ac7de7e0c22b593818a08f0a3dc6b5b6" target='_blank'>
              Epigenetic Modifiers: Exploring the Roles of Histone Methyltransferases and Demethylases in Cancer and Neurodegeneration
              </a>
            </td>
          <td>
            Lauren Reed, Janak Abraham, Shay Patel, Shilpa S. Dhar
          </td>
          <td>2024-12-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3245a4598675a5f03862517e366728182b100458" target='_blank'>
              Molecular mechanism targeting condensin for chromosome condensation.
              </a>
            </td>
          <td>
            Menglu Wang, Daniel Robertson, Juan Zou, Christos Spanos, J. Rappsilber, A. Marston
          </td>
          <td>2024-12-17</td>
          <td>The EMBO journal</td>
          <td>1</td>
          <td>85</td>
        </tr>

        <tr id="Mutations in the exonuclease domains of the replicative nuclear DNA polymerases POLD1 and POLE are associated with increased cancer incidence, elevated tumor mutation burden (TMB), and enhanced response to immune checkpoint blockade (ICB). Although ICB is approved for treatment of several cancers, not all tumors with elevated TMB respond, highlighting the need for a better understanding of how TMB affects tumor biology and subsequently immunotherapy response. To address this, we generated mice with germline and conditional mutations in the exonuclease domains of Pold1 and Pole. Engineered mice with Pold1 and Pole mutator alleles presented with spontaneous cancers, primarily lymphomas, lung cancer, and intestinal tumors, while Pold1 mutant mice also developed tail skin carcinomas. These cancers had highly variable tissue-type dependent increased TMB with mutational signatures associated with POLD1 and POLE mutations found in human cancers. The Pold1 mutant tail tumors displayed increased TMB, however, only a subset of established tumors responded to ICB. Similarly, introducing the mutator alleles into mice with lung cancer driven by mutant Kras and Trp53 deletion did not improve survival, whereas passaging these tumor cells in vitro without immune editing and subsequently implanting them into immune-competent mice caused tumor rejection in vivo. These results demonstrated the efficiency by which cells with antigenic mutations are eliminated in vivo. Finally, ICB treatment of mutator mice earlier, before observable tumors had developed delayed cancer onset, improved survival, and selected for tumors without aneuploidy, suggesting the potential of ICB in high-risk individuals for cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/274ef0221dce61440b76214e33f127197aac5688" target='_blank'>
              Immune Checkpoint Blockade Delays Cancer Development and Extends Survival in DNA Polymerase Mutator Syndromes.
              </a>
            </td>
          <td>
            Akshada Sawant, Fuqian Shi, E. Cararo Lopes, Zhixian Hu, Somer Abdelfattah, Jennele Baul, Jesse R. Powers, Christian S. Hinrichs, Joshua D. Rabinowitz, Chang S. Chan, Edmund C. Lattime, Shridar Ganesan, Eileen White
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The orchestration of DNA repair is of fundamental importance to the maintenance of genomic integrity and tumor suppression. DNA damage must be detected in the context of the varied chromatin landscape, its presence must be communicated throughout the cell to alter many ongoing processes, and the machinery that will mend the lesion must be recruited to the damage site. In my presentation, I will discuss our recent efforts in mapping genome maintenance pathways using genome-scale CRISPR/Cas9 screens in human cells. I will highlight how these screens can be used to identify new genome stability factors, characterize drug responses and provide new insights into the genetic architecture of the genome stability network. I will finally present how these screens can also identify potentially actionable synthetic lethal genetic interactions that could form the basis of new oncology drug discovery efforts.
 Citation Format: Daniel Durocher. Synthetic lethality: Pathways to therapeutic discovery. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Optimizing Therapeutic Efficacy and Tolerability through Cancer Chemistry; 2024 Dec 9-11; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(12_Suppl):Abstract nr IA003">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5b7589059a6134690d03df4a91c9f05b4284ea" target='_blank'>
              Abstract IA003: Synthetic lethality: Pathways to therapeutic discovery
              </a>
            </td>
          <td>
            Daniel Durocher
          </td>
          <td>2024-12-09</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ae5f6bb3e804b17358c04ea87f466f1ee8583c6" target='_blank'>
              Shedding light on DNA methylation and its clinical implications: the impact of long-read-based nanopore technology
              </a>
            </td>
          <td>
            Alexandra Chera, Mircea Stancu-Cretu, N. Zabet, Octavian Bucur
          </td>
          <td>2024-12-30</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e1421e272a6e4ff8131686f1fc164e53532b276" target='_blank'>
              Altered chromatin landscape and 3D interactions associated with primary constitutional MLH1 epimutations
              </a>
            </td>
          <td>
            Paula Climent-Cantó, Marc Subirana-Granés, M. Ramos-Rodríguez, Estela Dámaso, F. Marín, Covadonga Vara, Beatriz Pérez-González, Helena Raurell, Elisabet Munté, José Luis Soto, Ángel Alonso, GiWon Shin, Hanlee Ji, Megan Hitchins, Gabriel Capellá, Lorenzo Pasquali, M. Pineda
          </td>
          <td>2024-12-31</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="An ADP‐ribosylating enzyme called poly (ADP–ribose) polymerase 1 (PARP1) is necessary to start several types of DNA repair. PARP1 also contributes significantly to the regulation of gene expression through enzyme‐independent motif recognition and enzyme‐dependent chromatin remodeling. Thus, synthetic lethality is achieved in the therapy of tumors and cancers by blocking undesired DNA repair by the inhibition of its enzyme activity with small molecules. In addition to outlining the most recent research from 2019 to 2024 on the precise ways in which PARP1 regulates each process independently, we discussed how transcription and DNA repair are interdependent enough that perturbations caused by PARP1 enzymatic inhibition, disease‐related enzyme hyperactivation. Herein, this concept review systematically summarizes the different approach for designing selective PARP1 inhibitor that are i) PARP1–DNA trapping approach, ii) hybrid approach, iii) PROTAC approach, and iv) dual target inhibitor approach and updates on clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec80be41f9967e872ba7ea36021cfd284d0a854" target='_blank'>
              Recent Progress in Poly(ADP–Ribose) Polymerase‐1 Inhibitors for Application in Cancer
              </a>
            </td>
          <td>
            Anuradha Mehra, Shahnaz Islam, Rekha Sangwan
          </td>
          <td>2025-01-01</td>
          <td>ChemistrySelect</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca8eb632e8dcc4d371aa3266d6271acffef83ed2" target='_blank'>
              The integrated stress response drives MET oncogene overexpression in cancers.
              </a>
            </td>
          <td>
            M. Cerqua, Marco Foiani, C. Boccaccio, Paolo M Comoglio, D. Altintas
          </td>
          <td>2025-01-07</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Medulloblastoma (MB) is the most prevalent malignant brain tumor in children, exhibiting clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), display high levels of MYC and metastasis rates. Despite treatment with surgery, radiation, and chemotherapy, patients with Group 3 MB are more likely to develop aggressive recurrent tumors with poor survival. To examine resistance mechanisms, single nuclei multiome analysis of matched primary and recurrent tumors was performed in this study. A persistent progenitor population supporting resistance to therapy was identified. Additionally, distinct chromatin landscapes are linked to altered transcription and correspond to metabolic reprogramming. In vivo modeling of radiation resistance resolves similar chromatin-based metabolic reprogramming focused on wild-type isocitrate dehydrogenase (IDH1) activity. IDH1 inhibition reverses resistance-mediated chromatin changes and enables radiation re-sensitization. Ultimately, these findings demonstrate the efficacy of single-cell multiome analysis in elucidating resistance mechanisms and identifying novel target pathways for MYC-driven medulloblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beeb800863a129b3f026b22eed6f7669390988ee" target='_blank'>
              Single-cell multi-omics analysis identifies metabolism-linked epigenetic reprogramming as a driver of therapy-resistant medulloblastoma
              </a>
            </td>
          <td>
            Rajeev Vibhakar, Bethany L. Veo, Dong Wang, J. DeSisto, Angela M Pierce
          </td>
          <td>2024-12-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="DNA is frequently damaged by genotoxic stresses such as ionizing radiation, reactive oxygen species, and nitrogen species. DNA damage is a key contributor to cancer initiation and progression, and thus the precise and timely repair of these harmful lesions is required. Recent studies revealed transcription as a source of genome instability, and transcription-coupled DNA damage has been a focus in cancer research. Impaired mRNA export is closely related to DNA damage through R-loop formation. The molecular machineries of transcription-coupled DNA damage have been extensively analyzed in Saccharomyces cerevisiae. However, the molecular basis of these phenomena in higher eukaryotes remains elusive. In this review, we focus on the relationship between deregulated mRNA export through the transcription-export-2 (TREX-2) complex and cancer development. Particularly, the expression of germinal center-associated nuclear protein (GANP), a molecular scaffold in the TREX-2 complex, is highly associated with tumorigenesis in mice and humans. Although the deregulated expression of other components in the TREX-2 complex might affect cancer development, we have directly demonstrated the significance of GANP in tumorigenesis using genetically modified mice. Additionally, we describe recent evidence for medical applications demonstrating that the downregulation of the other components may be a good candidate for a chemotherapeutic target in terms of reducing the side effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99e429b3bb884f7cc3d479875de747738fafbdce" target='_blank'>
              Tumorigenesis Caused by Aberrant Expression of GANP, a Central Component in the Mammalian TREX-2 Complex—Lessons from Transcription-Coupled DNA Damages
              </a>
            </td>
          <td>
            Andri Rezano, Naomi Gondo, Yasuhiro Sakai, Yuko Nakamura, S. Phimsen, Tokio Tani, Akihiko Ito, Seiji Okada, Kazuhiko Kuwahara
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="It is thought that cells surviving ionizing radiation exposure repair DNA double-strand breaks (DSBs) and restore their genomes. However, the recent biochemical and genetic characterization of DSB repair pathways reveals that only homologous recombination (HR) can function in an error-free manner and that the non-homologous end joining (NHEJ) pathways canonical NHEJ (c-NHEJ), alternative end joining (alt-EJ), and single-strand annealing (SSA) are error-prone, and potentially leave behind genomic scars and altered genomes. The strong cell cycle restriction of HR to S/G2 phases and the unparalleled efficiency of c-NHEJ throughout the cell cycle, raise the intriguing question as to how far a surviving cell "reaches" after repairing the genome back to its pre-irradiation state. Indeed, there is evidence that the genomes of cells surviving radiation treatment harbor extensive genomic alterations. To directly investigate this possibility, we adopted next-generation sequencing (NGS) technologies and tested a normal human fibroblast cell line, 82-6 hTert, after exposure up to 6 Gy. Cells were irradiated and surviving colonies expanded and the cells froze. Sequencing analysis using the Illumina sequencing platform and comparison with the unirradiated genome detected frequent genomic alterations in the six investigated radiation survivor clones, including translocations and large deletions. Translocations detected by this analysis and predicted to generate visible cytogenetic alterations were frequently (three out of five) confirmed using mFISH cytogenetic analysis. PCR analysis of selected deletions also confirmed seven of the ten examined. We conclude that cells surviving radiation exposure tolerate and pass to their progeny a wide spectrum of genomic alterations. This recognition needs to be integrated into the interpretation of biological results at all endpoints, as well as in the formulation of mathematical models of radiation action. NGS analysis of irradiated genomes promises to enhance molecular cytogenetics by increasing the spectrum of detectable genomic alterations and advance our understanding of key molecular radiobiological effects and the logic underpinning DSB repair. However, further developments in the technology will be required to harness its full potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b3247759666806f5cf74d149fcc038397e2c814" target='_blank'>
              NGS Detects Extensive Genomic Alterations in Survivors of Irradiated Normal Human Fibroblast Cells.
              </a>
            </td>
          <td>
            A. Soni, Daniela Beisser, E. Mladenov, Matthias Höller, Inken Wohlers, Vladimir Nikolov, S. Magin, Tamara Mussfeldt, Ludger Klein-Hitpass, Michael N. Cornforth, B. Loucas, Sven Rahmann, G. Iliakis
          </td>
          <td>2024-12-27</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/930e1c577469260394dd63a8fa1b6c077a095083" target='_blank'>
              In-depth inference of transcriptional regulatory networks reveals NPM1 as a therapeutic ribosomal regulator in MYC-amplified medulloblastoma
              </a>
            </td>
          <td>
            Tong Chen, Huiyao Chen, Mingyang Xia, Yunfei Liao, Hao Li, Xinran Dong, Yifeng Lin, Wenhao Zhou
          </td>
          <td>2025-01-10</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28cbf9783c62fb0f01914152c3e6d88182c98411" target='_blank'>
              LKB1 inactivation promotes epigenetic remodeling-induced lineage plasticity and antiandrogen resistance in prostate cancer
              </a>
            </td>
          <td>
            Fei Li, Pengfei Dai, Huili Shi, Yajuan Zhang, Juan He, Anuradha Gopalan, Dan Li, Yu Chen, Yarui Du, Guoliang Xu, Weiwei Yang, Chao Liang, Dong Gao
          </td>
          <td>2025-01-01</td>
          <td>Cell Research</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Large genetic variants can be generated via homologous recombination (HR), such as polymerase theta-mediated end joining (TMEJ) or single-strand annealing (SSA). Given that these HR-based mechanisms leave specific genomic signatures, we developed GDBr, a genomic signature interpretation tool for DNA double-strand break repair mechanisms using high-quality genome assemblies. We applied GDBr to a draft human pangenome reference. We found that 78.1% of non-repetitive insertions and deletions and 11.0% of non-repetitive complex substitutions contained specific signatures. Of these, we interpreted that 98.7% and 1.3% of the insertions and deletions were generated via TMEJ and SSA, respectively, and all complex substitutions via TMEJ. Since population-level pangenome datasets are being dramatically accumulated, GDBr can provide mechanistic insights into how variants are formed. GDBr is available on GitHub at https://github.com/Chemical118/GDBr.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24886490d8b6e2e3d68ad1604f388da5f8ce50c" target='_blank'>
              GDBr: genomic signature interpretation tool for DNA double-strand break repair mechanisms
              </a>
            </td>
          <td>
            Hyunwoo Ryu, H. Han, Chuna Kim, Jun Kim
          </td>
          <td>2025-01-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Polycomb repressive complex 2 (PRC2) is known to regulate gene expression and chromatin structure as it methylates H3K27, resulting in gene silencing. Studies have shown that PRC2 has dual functions in oncogenesis that allow it to function as both an oncogene and a tumor suppressor. Because of this, nuanced strategies are necessary to promote or inhibit PRC2 activity therapeutically. Given the therapeutic vulnerabilities and associated risks in oncological applications, a structured literature review on PRC2 was conducted to showcase similar cofactor competitor inhibitors of PRC2. Key inhibitors such as Tazemetostat, GSK126, Valemetostat, and UNC1999 have shown promise for clinical use within various studies. Tazemetostat and GSK126 are both highly selective for wild-type and lymphoma-associated EZH2 mutants. Valemetostat and UNC1999 have shown promise as orally bioavailable and SAM-competitive inhibitors of both EZH1 and EZH2, giving them greater efficacy against potential drug resistance. The development of other PRC2 inhibitors, particularly inhibitors targeting the EED or SUZ12 subunit, is also being explored with the development of drugs like EED 226. This review aims to bridge gaps in the current literature and provide a unified perspective on promising PRC2 inhibitors as therapeutic agents in the treatment of lymphomas and solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d39c17f73d3909cec9400437d3632effbc2233b4" target='_blank'>
              Pharmacological Advancements of PRC2 in Cancer Therapy: A Narrative Review
              </a>
            </td>
          <td>
            Michael S. Wang, Jonathan H Sussman, Jessica A. Xu, Reema Patel, Omar Elghawy, Prashanth Rawla
          </td>
          <td>2024-12-01</td>
          <td>Life</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="KRAS is one of the most frequently mutated oncogenes in cancer and is involved in various tumorigenic processes. Among its diverse roles, KRAS mutations have been shown to upregulate macropinocytosis, a form of endocytosis that allows cells to engulf extracellular fluid and its contained solutes, subsequently supporting the increased need for cancer cells' growth and proliferation. The nutrient uptake function of macropinocytosis was initially described within the framework of KRAS-driven pancreatic tumors. Although constitutive macropinocytosis can be induced by activating mutations of proteins that are commonly found in cancer, KRAS-induced macropinocytosis constitutes a fundamental area of research due to the high occurrence of mutated KRAS in cancer (~1/3 of all cancers).  Furthermore, KRAS-mutated macropinocytosis not only contributes to tumorigenesis but also plays a critical role in developing resistance to treatments, as it was found to be implicated in Multi-Drug Resistance (MDR) in cancer cells. This mini-review aims to synthesize current knowledge of mechanisms of KRAS-mutated macropinocytosis briefly and examines the relationship between KRAS and macropinocytosis in the light of its role in cancer progression and drug resistance, highlighting therapeutic implications, targeting potential vulnerabilities and outlining clinical advancements in pertinent therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a4b47f4761d206b1edf858bd1fc0d092897567" target='_blank'>
              The Role of KRAS in Macropinocytosis and its Implications of Multi-Drug Resistance in Cancer
              </a>
            </td>
          <td>
            F. Fenyvesi, Zeinab Ibrahim
          </td>
          <td>2024-12-23</td>
          <td>De Remediis</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Telomere maintenance is crucial for preventing the linear eukaryotic chromosome ends from being mistaken for DNA double-strand breaks, thereby avoiding chromosome fusions and the loss of genetic material. Unlike most eukaryotes that use telomerase for telomere maintenance, Drosophila relies on retrotransposable elements-specifically HeT-A, TAHRE, and TART (collectively referred to as HTT)-which are regulated and precisely targeted to chromosome ends. Drosophila telomere protection is mediated by a set of fast-evolving proteins, termed terminin, which bind to chromosome termini without sequence specificity, balancing DNA damage response factors to avoid erroneous repair mechanisms. This unique telomere capping mechanism highlights an alternative evolutionary strategy to compensate for telomerase loss. The modulation of recombination and transcription at Drosophila telomeres offers insights into the diverse mechanisms of telomere maintenance. Recent studies at the population level have begun to reveal the architecture of telomere arrays, the diversity among the HTT subfamilies, and their relative frequencies, aiming to understand whether and how these elements have evolved to reach an equilibrium with the host and to resolve genetic conflicts. Further studies may shed light on the complex relationships between telomere transcription, recombination, and maintenance, underscoring the adaptive plasticity of telomeric complexes across eukaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc71fe25065e2464b3633178f88c0066a196f35c" target='_blank'>
              Maintaining Telomeres without Telomerase in Drosophila: Novel Mechanisms and Rapid Evolution to Save a Genus.
              </a>
            </td>
          <td>
            S. Cacchione, G. Cenci, Anne-Marie Dion-Côté, D. Barbash, G. Raffa
          </td>
          <td>2024-12-18</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="ABSTRACT Epstein‐Barr virus (EBV) is a ubiquitous human ɣ‐herpesvirus implicated in various malignancies, including Burkitt's lymphoma and gastric carcinomas. In most EBV‐associated cancers, the viral genome is maintained as an extrachromosomal episome by the EBV nuclear antigen‐1 (EBNA1). EBNA1 is considered to be a highly stable protein that interacts with the ubiquitin‐specific protease 7 (USP7). Here, we show that pharmacological inhibitors and small interfering RNA (siRNA) targeting USP7 reduce EBNA1 protein levels in a proteosome‐dependent manner. Proteomic analysis revealed that USP7 inhibitor GNE6776 altered the EBNA1 protein interactome, including disrupting USP7 association with EBNA1. GNE6776 also inhibited EBNA1 binding to EBV oriP DNA and reduced viral episome copy number. Transcriptomic studies revealed that USP7 inhibition affected chromosome segregation and mitotic cell division pathways in EBV+ cells. Finally, we show that GNE6776 selectively inhibited EBV+ gastric and lymphoid cell proliferation in cell culture and slowed EBV+ tumor growth in mouse xenograft models. These findings suggest that USP7 inhibitors perturb EBNA1 stability and function and may be exploited to treat EBV latent infection and tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f5972601207babc010d4b0b866abd5a2b3ddcc3" target='_blank'>
              USP7 Inhibitors Destabilize EBNA1 and Suppress Epstein‐Barr Virus Tumorigenesis
              </a>
            </td>
          <td>
            Christopher Chen, Kush Addepalli, S. Soldan, L. J. Castro-Muñoz, Sarah Preston-Alp, Rishi J Patel, Coltin J Albitz, Hsin-Yao Tang, I. Tempera, P. Lieberman
          </td>
          <td>2025-01-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="In many eukaryotes, meiotic recombination occurs preferentially at discrete sites, called recombination hotspots. In various lineages, recombination hotspots are located in regions with promoter-like features and are evolutionarily stable. Conversely, in some mammals, hotspots are driven by PRDM9 that targets recombination away from promoters. Paradoxically, PRDM9 induces the self-destruction of its targets and this triggers an ultra-fast evolution of mammalian hotspots. PRDM9 is ancestral to all animals, suggesting a critical importance for the meiotic program, but has been lost in many lineages with surprisingly little effect on meiosis success. However, it is unclear whether the function of PRDM9 described in mammals is shared by other species. To investigate this, we analyzed the recombination landscape of several salmonids, the genome of which harbors one full-length PRDM9 and several truncated paralogs. We identified recombination initiation sites in Oncorhynchus mykiss by mapping meiotic DNA double-strand breaks (DSBs). We found that DSBs clustered at hotspots positioned away from promoters, enriched for the H3K4me3 and H3K36me3 and the location of which depended on the genotype of full-length Prdm9. We observed a high level of polymorphism in the zinc finger domain of full-length Prdm9, indicating diversification driven by positive selection. Moreover, population-scaled recombination maps in O. mykiss, Oncorhynchus kisutch and Salmo salar revealed a rapid turnover of recombination hotspots caused by PRDM9 target motif erosion. Our results imply that PRDM9 function is conserved across vertebrates and that the peculiar evolutionary runaway caused by PRDM9 has been active for several hundred million years.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f09c603402d3b52a99c356601f8730c3715c9b0" target='_blank'>
              PRDM9 drives the location and rapid evolution of recombination hotspots in salmonid fish
              </a>
            </td>
          <td>
            Marie-Laure Raynaud, Paola Sanna, Julien Joseph, J. Clément, Yukiko Imai, J. Lareyre, Audrey Laurent, Nicolas Galtier, F. Baudat, Laurent Duret, P. Gagnaire, B. de Massy
          </td>
          <td>2025-01-01</td>
          <td>PLOS Biology</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="From RNA interference to chromatin silencing, diverse genome defense pathways silence selfish genetic elements to safeguard genome integrity. Despite their diversity, different defense pathways share a modular organization, where numerous specificity factors identify diverse targets and common effectors silence them. In the PIWI-interacting RNA (piRNA) pathway, target RNAs are first identified by complementary base pairing with piRNAs and then silenced by PIWI-clade nucleases. Such a binary architecture allows the defense systems to be readily adaptable, where new targets can be captured via innovation of specificity factors. Thus, our current understanding of genome defense against lineage-specific selfish genes has been largely limited to specificity factor innovations, while it remains poorly understood whether other types of innovations are required. Here, we describe a new type of innovation, which escalates the genome defense capacity to control a recently expanded selfish gene in Drosophila melanogaster. Through a targeted RNAi screen for repressors of Stellate-a recently evolved meiotic driver-we identified a defense factor, Trailblazer. Trailblazer is a transcription factor that promotes the expression of two PIWI-clade nucleases, Aub and AGO3, to match Stellate in abundance. Recent innovation in the DNA-binding domain of Trailblazer enabled it to elevate Aub and AGO3 expression, thereby escalating the silencing capacity of piRNA pathway to tame expanded Stellate and safeguard fertility. As copy-number expansion is a recurrent feature of diverse selfish genes across the tree of life, we envision that augmenting the defense capacity to quantitatively match selfish genes is a repeatedly employed defense strategy in evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b459a2d133898ca4732a04488f0f54cfda8ecf00" target='_blank'>
              Escalation of genome defense capacity enables control of an expanding meiotic driver.
              </a>
            </td>
          <td>
            Peiwei Chen, Katherine C. Pan, Eunice H. Park, Yicheng Luo, Y. C. G. Lee, A. Aravin
          </td>
          <td>2025-01-07</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Background Gene methylation in cells is an important factor in tumorigenesis, and radiotherapy can change DNA methylation in cells. In this study, complete genome methylation sequencing (BS-Seq) technology was used to analyze the genome-wide methylation of patients with cervical cancer before and after radiotherapy. Methods Three pairs of cervical squamous cell carcinoma samples were collected from patients before and after radiotherapy in July 2020. Genome-wide DNA methylation profiles were generated using WGBS. Bioinformatics analysis was conducted to identify differential methylation regions (DMRs) and their associated genes and pathways. The study focused on the methylation changes of LHX2, LHX5, and LHX9 genes, assessing their expression levels using qRT-PCR and correlating these changes with cervical cancer stages. Results MCG was the main way of genomic DNA methylation in the three patients. The DNA methylation level and methylation density on each chromosome varied greatly. As revealed by comparison of methylation before and after radiation in the three patients, 1287, 1261 and 789 differential methylation genes were identified, respectively. 3) Combined with clinical treatment, methylation level difference and correlation enrichment analysis, it was found that LHX2, LHX5 and LHX9 were closely related to the occurrence and development of cervical cancer. After 5-Aza-DC and radiotherapy, the methylation of the CpG islands in LHX2, LHX5 and LHX9 genes in these patients was decreased (p < 0.01), and the mRNA and protein expression levels were relatively increased (p < 0.01). Conclusion In our present work, genome-wide DNA methylation maps of cervical cancer tissues before and after radiotherapy were successfully constructed. We found that LHX5 and LHX9 genes are closely related to cervical cancer. LHX5 and LHX9 have a negative effect on cervical cancer. The migration ability of LHX9 silenced cells was significantly enhanced after irradiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fea2269d56f3d3d8896922414dbb511f2559dfa8" target='_blank'>
              Whole-Genome Methylation Sequencing Analysis and Functional Verification of LIM-Homeobox Family Genes in Cervical Cancer
              </a>
            </td>
          <td>
            Rong Yu, Qin Yu, Jie Shi, Xue Meng, Zhiyuan Deng, Jing Suo, Hao Yang
          </td>
          <td>2025-01-01</td>
          <td>International Journal of General Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94b123662301192f16a8e24048c0300f73a9a5cc" target='_blank'>
              SiCLAT: simultaneous imaging of chromatin loops and active transcription in living cells
              </a>
            </td>
          <td>
            Xin Wan, Jie Kong, Xiaodi Hu, Lulu Liu, Yuanping Yang, Hu Li, Gaoao Liu, Xingchen Niu, Fengling Chen, Dan Zhang, Dahai Zhu, Yong Zhang
          </td>
          <td>2025-01-02</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="DNA damage can lead to mutations that can alter the function of oncogenes or tumor suppressor genes, thus promoting the development of cancer. p53 plays a multifaceted and complex role in the DNA damage response and cancer progression and is known as the ‘guardian of the gene’. When DNA damage occurs, p53 is activated through a series of post-translational modifications, which stabilize the protein and enhance its function as a transcription factor. It regulates processes including cell cycle checkpoints, DNA repair and apoptosis, thereby preventing the spread of damaged DNA and maintaining genome integrity. On the one hand, p53 can initiate cell cycle arrest and induce cells to enter the G1/S and G2/M checkpoints, preventing cells with damaged DNA from continuing to proliferate and gaining time for DNA repair. At the same time, p53 can promote the activation of DNA repair pathways, including base excision repair, nucleotide excision repair and other repair pathways, to ensure the integrity of genetic material. If the damage is too severe to repair, p53 will trigger the apoptosis process to eliminate potential cancer risks in time. p53 also plays a pivotal role in cancer progression. Mutations in the p53 gene are frequently found in many cancers, and the mutated p53 not only loses its normal tumor suppressor function but may even acquire pro-cancer activity. Therefore, we also discuss therapeutic strategies targeting the p53 pathway, such as the use of small-molecule drugs to restore the function of wild-type p53, the inhibition of negative regulatory factors and synthetic lethality approaches for p53-deficient tumors. This review therefore highlights the important role of p53 in maintaining genomic stability and its potential in therapeutic strategies for cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b423291d0566fe3b6571def47ce2c8576918e9f7" target='_blank'>
              Unraveling the Guardian: p53’s Multifaceted Role in the DNA Damage Response and Tumor Treatment Strategies
              </a>
            </td>
          <td>
            Han Zhang, Jianxiong Xu, Yuxuan Long, Ayitila Maimaitijiang, Zhengding Su, Wenfang Li, Jinyao Li
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Cancer cells arise from multiple complementary genomic and epigenomic abnormalities that deregulate the pathways controlling cell proliferation, cell survival, and tissue homeostasis. Epigenetic modifications of histones, DNA, chromatin structure, and non-coding RNA expression are emerging as critical determinants of gene expression and essential drivers of tumor cell heterogeneity and neoplastic cell phenotypes. Multipotent stem-like cells (also referred to as cancer stem cells, CSCs) are critical determinants of tumor propagation, therapeutic resistance, and recurrence following treatment. However, our understanding of how these multidimensional epigenetic modifications cooperatively drive cancer cell stemness and how to efficaciously target them is currently inadequate to positively impact patient outcomes. Abstract Background/Objectives: CSCs are critical drivers of the tumor and stem cell phenotypes of glioblastoma (GBM) cells. Chromatin modifications play a fundamental role in driving a GBM CSC phenotype. The goal of this study is to further our understanding of how stem cell-driving events control changes in chromatin architecture that contribute to the tumor-propagating phenotype of GBM. Methods: We utilized computational analyses to identify a subset of clinically relevant genes that were predicted to be repressed in a Polycomb repressive complex 2 (PRC2)-dependent manner in GBM upon induction of stem cell-driving events. These associations were validated in patient-derived GBM neurosphere models using state-of-the-art molecular techniques to express, silence, and measure microRNA (miRNA) and gene expression changes. Advanced Poly(β-amino ester) nanoparticle formulations (PBAEs) were used to deliver miRNAs in vivo to orthotopic human GBM tumor models. Results: We show that glioma stem cell (GSC) formation and tumor propagation involve the crosstalk between multiple epigenetic mechanisms, resulting in the repression of the miRNAs that regulate PRC2 function and histone H3 lysine 27 tri-methylation (H3K27me3). We also identified miR-217-5p as an EZH2 regulator repressed in GSCs and showed that miR-217-5p reconstitution using advanced nanoparticle formulations re-activates the PRC2-repressed genes, inhibits GSC formation, impairs tumor growth, and enhances the effects of ionizing radiation in an orthotopic model of GBM. Conclusions: These findings suggest that inhibiting PRC2 function by targeting EZH2 with miR-217-5p advanced nanoparticle formulations could have a therapeutic benefit in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a650a5c81b8e5180cf08ed0736baa5ed09de3eb0" target='_blank'>
              miR-217-5p NanomiRs Inhibit Glioblastoma Growth and Enhance Effects of Ionizing Radiation via EZH2 Inhibition and Epigenetic Reprogramming
              </a>
            </td>
          <td>
            Jack Korleski, Sweta Sudhir, Yuan Rui, Christopher Caputo, Sophie Sall, Amanda L. Johnson, Harmon Khela, Tanmaya Madhvacharyula, Anisha Rasamsetty, Yunqing Li, B. Lal, Weiqiang Zhou, Karen Smith-Connor, Stephany Tzeng, Jordan Green, J. Laterra, H. Lopez-Bertoni
          </td>
          <td>2024-12-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1113f4b65f7bb6543b590129c22b6e32ee76e8" target='_blank'>
              Transposable element 5mC methylation state of blood cells predicts age and disease.
              </a>
            </td>
          <td>
            Francesco Morandini, Jinlong Y Lu, Cheyenne Rechsteiner, A. Shadyab, Ramon Casanova, Beverly M Snively, A. Seluanov, Vera Gorbunova
          </td>
          <td>2024-11-27</td>
          <td>Nature aging</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/125e72b0b5ffa707da90fa52c6a8e151fa428862" target='_blank'>
              TRACERx analysis identifies a role for FAT1 in regulating chromosomal instability and whole-genome doubling via Hippo signalling
              </a>
            </td>
          <td>
            Wei-Ting Lu, L. Zalmas, Chris Bailey, James R. M. Black, C. Martínez-Ruiz, O. Pich, F. Gimeno-Valiente, Ieva Usaite, Alastair Magness, K. Thol, Thomas A. Webber, Ming Jiang, R. Saunders, Yun-Hsin Liu, D. Biswas, Esther O. Ige, Birgit Aerne, E. Grönroos, Subramanian Venkatesan, Georgia Stavrou, Takahiro Karasaki, M. Al Bakir, M. Renshaw, Hang Xu, Deborah Schneider-Luftman, Natasha Sharma, Laura Tovini, M. Jamal-Hanjani, S. McClelland, K. Litchfield, Nicolai J. Birkbak, Michael Howell, N. Tapon, K. Fugger, N. Mcgranahan, J. Bartek, N. Kanu, C. Swanton
          </td>
          <td>2024-12-30</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>69</td>
        </tr>

        <tr id="Introduction Cancer’s inherent heterogeneity, marked by diverse genetic and molecular alterations, presents significant challenges for developing effective treatments. One such alteration is the regulation of disulfidoptosis, a recently discovered programmed cell death pathway. RPN1, a key regulator associated with disulfidoptosis, may influence various aspects of tumor biology, including immune evasion and cellular senescence. This study aims to dissect the role of RPN1 in pan-cancer and its potential as a therapeutic target. Methods We employed a pan-cancer analysis to explore RPN1 expression and its association with clinical outcomes across multiple tumor types. Immune cell infiltration and expression of immune checkpoint genes were analyzed in relation to RPN1. Additionally, cellular senescence markers were assessed in RPN1 knockdown tumor cells. Gene regulatory mechanisms were studied through gene copy number variations, DNA methylation analysis, and transcriptional regulation by SP1. Results RPN1 is overexpressed in a wide range of tumor types and correlates with poor clinical outcomes, including overall survival, disease-specific survival, and progression-free intervals. Our analysis shows that RPN1 is involved in immune evasion, correlating with the presence of myeloid dendritic cells, macrophages, and tumor-associated fibroblasts, and influencing T-cell activity. RPN1 knockdown led to reduced tumor cell proliferation and induced cellular senescence, marked by increased senescence-associated biomarkers and β-galactosidase activity. RPN1 expression was found to be regulated by gene copy number variations, reduced DNA methylation, and transcriptional control via SP1. Discussion These findings highlight RPN1 as a key pan-cancer regulator, influencing immune microenvironment interactions and cellular senescence. The regulation of disulfidoptosis by RPN1 presents a promising avenue for therapeutic intervention. Targeting RPN1 could enhance immunotherapy efficacy and help mitigate tumor progression, offering a potential strategy for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2952a313a9a344ecc3d40275b633ebc51cd3bfe9" target='_blank'>
              Dissecting the functions and regulatory mechanisms of disulfidoptosis-related RPN1 in pan-cancer: modulation of immune microenvironment and cellular senescence
              </a>
            </td>
          <td>
            Lexin Qin, Tingting Liang, Hailong Zhang, Xian Gong, Meidan Wei, Xiangrong Song, Yaoyu Hu, Xinyu Zhu, Wentao Hu, Jianxiang Li, Jin Wang
          </td>
          <td>2024-12-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae1a74726e395311f7dc9708215816307d41962f" target='_blank'>
              Asymmetric distribution of G-quadruplex forming sequences in genomes of retroviruses
              </a>
            </td>
          <td>
            Filip Kledus, Michaela Dobrovolná, J. Mergny, Václav Brázda
          </td>
          <td>2025-01-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) is one of the deadliest human cancers, with the overall 5-year survival rate stagnating in recent decades due to the lack of innovative treatment approaches. Apart from the recently Food and Drug Administration-approved epidermal growth factor receptor inhibitor and immune checkpoint inhibitor, alternative therapeutic strategies that target epigenetic abnormalities, an emerging cancer hallmark, remain to be fully explored. A pathological epigenetic landscape, characterized by widespread reprogramming of chromatin modifications such as DNA methylation and histone modifications, which drives transcription deregulation and genome reorganization, has been extensively documented in numerous cancers, including HNSCC. Growing evidence indicates that these frequent epigenomic alterations play pivotal roles in regulating malignant transformation, promoting metastasis and invasion, and reshaping the tumor microenvironment. Furthermore, these epigenetic changes also present unique vulnerabilities that open new avenues for identifying novel prognostic biomarkers and developing targeted antitumor therapies. In this review, we summarize recent discoveries of epigenetic dysregulations in HNSCC, with a focus on deregulated chromatin modifications, which include aberrant DNA methylation, oncohistone H3 lysine 36 to methionine (H3K36M) mutation, as well as recurrent mutations or altered expression of chromatin-modifying enzymes such as NSD1, EZH2, and KMT2C/D. Importantly, we discuss the various molecular mechanisms underlying the contributions of these epigenetic alterations to HNSCC development, particularly their involvement in deregulated cell proliferation and cell death, metabolic reprogramming, tumor immune evasion, and phenotypic plasticity. Finally, we conclude by highlighting the translational and clinical implications of targeting the epigenetic machinery, which offers promising prospects for overcoming resistance to conventional radiotherapy/chemotherapy and enhancing the response to immunotherapy in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0f6940a95157f857134f2791aea649c5fe14cbd" target='_blank'>
              Targeting Epigenetic Dysregulations in Head and Neck Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Y. Li, C. Lu
          </td>
          <td>2024-12-19</td>
          <td>Journal of dental research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acca60e9e5684fac9b42ec113ffc78d1e571a20d" target='_blank'>
              Pan-inhibition of super-enhancer-driven oncogenic transcription by next-generation synthetic ecteinascidins yields potent anti-cancer activity.
              </a>
            </td>
          <td>
            Max Cigrang, Julian Obid, Maguelone Nogaret, Léane Seno, Tao Ye, G. Davidson, Philippe Catez, P. Berico, Clara Capelli, Clara Marechal, Amélie Zachayus, Clémence Elly, Marie Jose Guillen Navarro, Marta Martínez Diez, Gema Santamaría Nuñez, Tsai-Kun Li, E. Compe, Pablo Aviles, Irwin Davidson, J. Egly, Carmen Cuevas, F. Coin
          </td>
          <td>2025-01-08</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/089aac455387b5d1dba7549c993d3541b59537a0" target='_blank'>
              Circular RNA circPHLPP2 promotes tumor growth and anti-PD-1 resistance through binding ILF3 to regulate IL36γ transcription in colorectal cancer
              </a>
            </td>
          <td>
            Yan Hu, Ze-Rong Cai, Ren-Ze Huang, De-Shen Wang, Huai-qiang Ju, Dong-Liang Chen
          </td>
          <td>2024-12-18</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Abstract Background Chromosomal instability (CIN), a hallmark of cancer, is commonly linked to poor prognosis in high‐grade prostate cancer (PCa). Paradoxically, excessively high levels of CIN may impair cancer cell viability. Consequently, understanding how tumours adapt to CIN is critical for identifying novel therapeutic targets. Methods Bioinformatic analyses were conducted to identify genes overexpressed in PCa tissues using The Cancer Genome Atlas (TCGA) and GEO datasets. Western blotting and immunohistochemistry assays were applied to determine the expression levels of euchromatic histone lysine methyltransferase 2 (EHMT2), pT232‐Aurora B and Cullin 3 (CUL3). The proliferation of cells was measured through CCK‐8 tests, clonogenesis and subcutaneous xenografts of human PCa cells in BALB/c nude mice. Live cell imaging, immunofluorescence (IF) and flow cytometry were used to confirm the role of EHMT2 in PCa cell mitosis. Co‐immunoprecipitation, Western blotting and IF assays further elucidated the underlying molecular mechanisms. Results EHMT2 was highly expressed in metastatic PCa tissues exhibiting elevated CIN and was strongly associated with adverse clinical outcomes in patients with PCa. Silencing EHMT2 impaired cell division, inducing G2/M‐phase arrest and mitotic catastrophe in PCa cells. Mechanistically, EHMT2 is indispensable to ensure the full activation of Aurora B through centromeric R‐loop‐driven ATR–CHK1 pathway, with EHMT2 protein expression peaking during the G2/M‐phase. Moreover, CUL3 was identified as a binding partner of EHMT2, mediating its polyubiquitination and destabilising its protein levels. Conclusions This study reveals a CUL3–EHMT2–Aurora B regulatory axis that safeguards accurate chromosome segregation in PCa cells, supporting the potential therapeutic application of EHMT2 inhibitors. Key points Euchromatic histone lysine methyltransferase 2 (EHMT2) is overexpressed in advanced prostate cancer, restraining catastrophic chromosomal instability (CIN) and enhancing cell fitness. EHMT2 functions via the centromeric R‐loop‐driven ATR–CHK1–Aurora B pathway to promote chromosomal stability. EHMT2 confers enzalutamide resistance via activating Aurora B. Cullin 3 (CUL3) promotes EHMT2 destabilisation via deubiquitination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc5ece350ce05d9ec1131335e290684b93468c8" target='_blank'>
              EHMT2‐mediated R‐loop formation promotes the malignant progression of prostate cancer via activating Aurora B
              </a>
            </td>
          <td>
            Yuyang Zhang, Mingqin Su, Yiming Chen, Li Cui, Wei Xia, Renfang Xu, Dong Xue, Xiansheng Zhang, Xingliang Feng
          </td>
          <td>2025-01-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5cabef889301b036a508a4127fd265270ae7d40" target='_blank'>
              DNA hypermethylation of tumor suppressor genes among oral squamous cell carcinoma patients: a prominent diagnostic biomarker.
              </a>
            </td>
          <td>
            Nistha Agarwal, A. Jha
          </td>
          <td>2024-12-07</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Breast cancer is known as one of the most predominant cancers that affect both females and
males worldwide. The most crucial risk factor in breast cancer is the mutations in the RAD51C
gene that have been considered in most hereditary breast cancers. RAD51C, the RAD51
paralogs, is also a deoxyribonucleic acid (DNA) repair protein related to breast and ovarian
cancers. DNA double-strand breaks (DSBs) account for the significant detrimental form of
DNA damage. RAD51C mutants also have been recognized in breast/ovarian cancer patients.
However, the role of the RAD51C protein in hereditary breast cancer and its three-dimensional
(3D) structures remains unclear. Thus, this study was conducted to identify the 3D structure of
RAD51C protein from its amino acid sequences. The homology modeling for the 3D structure
of the RAD51C protein was carried out by using three automated webservers: I-TASSER,
SWISS-MODEL, and Phyre2. PyMOL was applied to visualize the 3D structure of RAD51C
protein. Next, the MolProbity, ProSA, and SAVES v6.0 programs have been employed to
check the stereo-chemical quality of RAD51C protein. The RAD51C-IT models were found to
be the best models for the RAD1C protein after being evaluated and validated, and the models
were constructed using full-length RAD51C protein sequences. Thus, these protein models can
be utilized as a virtual screening tool in discovering potential inhibitors of RAD51C protein.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be054c5f435a476d5df616bc8bacb7f34a1b8877" target='_blank'>
              Homology Modeling of Human DNA Repair Protein RAD51 Homolog 3 (RAD51C) in Breast Cancer
              </a>
            </td>
          <td>
            Ruzianisra Mohamed
          </td>
          <td>2024-12-30</td>
          <td>International Journal of Pharmaceuticals, Nutraceuticals and Cosmetic Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cell cycle checkpoints are stringent quality control mechanisms that regulate cell cycle progression and division. Cancer cells often develop a dependency on the G2/M cell cycle checkpoint to facilitate DNA repair and resolve intrinsic or therapy-induced DNA damage. This dependency leads to therapy resistance, continuous cell division, and disease progression. Targeting G2/M checkpoints has been heavily pursued over the past two decades and has progressed into clinical studies. Recent genome-scale functional genomic studies have revealed that Myt1 kinase, an essential but previously overlooked molecule for the G2/M checkpoint, is a promising target for multiple types of cancers. In this work, we summarize the latest discoveries in molecular targeting of Myt1 kinase and discuss the challenges and limitations in expanding its clinical application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17deb173d71c056d6283f804746aaf46a913ddbb" target='_blank'>
              Myt1 kinase: An Emerging Cell Cycle Regulator for Cancer Therapeutics.
              </a>
            </td>
          <td>
            Fengchao Lang, Karambir Kaur, Javeria Zaheer, Diego Luis Ribeiro, Chunzhang Yang
          </td>
          <td>2025-01-16</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA-damaging agents (DDAs) have long been used in cancer therapy. However, the precise mechanisms by which DDAs induce cell death are not fully understood and drug resistance remains a major clinical challenge. Schlafen 11 (SLFN11) was identified as the gene most strongly correlated with the sensitivity to DDAs based on mRNA expression levels. SLFN11 sensitizes cancer cells to DDAs by cleaving and downregulating tRNALeu(TAA). Elucidating the detailed mechanism by which SLFN11 induces cell death is expected to provide insights into overcoming drug resistance. Here, we show that, upon administration of DDAs, SLFN11 cleaves tRNALeu(TAA), leading to ER stress and subsequent cell death regulated by inositol-requiring enzyme 1 alpha (IRE1α). These responses were significantly alleviated by SLFN11 knockout or transfection of tRNALeu(TAA). Our proteomic analysis suggests that tRNALeu(TAA) influences proteins essential for maintaining proteostasis, especially those involved in ubiquitin-dependent proteolysis. Additionally, we identified the cleavage sites of tRNALeu(TAA) generated by SLFN11 in cells, and revealed that tRNA fragments contribute to ER stress and cell death. These findings suggest that SLFN11 plays a crucial role in proteostasis by regulating tRNAs, and thus determines cell fate under DDA treatment. Consequently, targeting SLFN11-mediated tRNA regulation could offer a novel approach to improve cancer therapy. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/811a0f920ef5bd12afa937061ab8c997f4e1a5be" target='_blank'>
              SLFN11-mediated tRNA regulation induces cell death by disrupting proteostasis in response to DNA-damaging agents
              </a>
            </td>
          <td>
            Yuki Iimori, Teppei Morita, Takeshi Masuda, Shojiro Kitajima, N. Kono, Shun Kageyama, Josephine Galipon, Atsuo T. Sasaki, Akio Kanai
          </td>
          <td>2025-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Retrotransposable elements are implicated in genome rearrangements and gene expression alterations that result in various human disorders. In the current study, we sought to investigate the potential effects of long interspersed elements-1 (LINE-1) overexpression on the integrity and methylation of DNA and on the expression of three major pluripotency factors (OCT4, SOX2, NANOG) during the preimplantation stages of human embryo development. Human MI oocytes were matured in vitro to MII and transfected through intracytoplasmic sperm injection (ICSI) either with an EGFP vector carrying a cloned active human LINE-1 retroelement or with the same EGFP vector without insert as control. The occurrence of retrotransposition events was screened by fluorescent microscopy. The in vitro preimplantation development as well as the methylation, pluripotency, and DNA double-strand breaks (DSBs) of the transfected embryos were examined. LINE-1 retrotransposons gave rise to new retrotransposition events in the transfected embryos. LINE-1 injected embryos were characterized by accelerated asymmetrical cell division, multiple cellular fragments, cleavage arrest, and degeneration. Early OCT4 expression remained unaltered, but cleavage arrest and a high fragmentation rate hindered the expression of SOX2/NANOG at the morula stage. Increased DNA DSBs were observed in cleavage-stage blastomeres, while no methylation changes were detected before the cleavage arrest. Our data provide evidence that LINE-1 retrotransposition in human preimplantation embryos may induce DNA DSBs, while at the same time, it appears to interfere with the expression patterns of pluripotency factors. The morphological, structural, and cleavage abnormalities of the transfected embryos show that aberrant retroelement expression may negatively affect human embryo development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524c448bfb57d8b4a52f5e6135b0463d8036afab" target='_blank'>
              LINE-1-Induced Retrotransposition Affects Early Preimplantation Embryo DNA Integrity and Pluripotency
              </a>
            </td>
          <td>
            P. Sakaloglou, L. Lazaros, I. Bouba, S. Markoula, Athanasios Zikopoulos, E. Drakaki, Ismini Anagnostaki, Anastasios Potiris, S. Stavros, A. Gerede, Ekaterini Domali, Peter Drakakis, Theodoros Tzavaras, Ioannis Georgiou
          </td>
          <td>2024-11-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Tumor suppressor gene TP53 is ubiquitously mutated in ovarian cancer precursor lesions that undergo pervasive accumulation of DNA damage. Prior literature provides evidence that the expression of mutant p53 protein in epithelial cells is associated with increased survival in response to DNA-damaging treatments. Hence, identification and understanding of the mechanisms that, in response to DNA damage accumulation, support survival of ovarian cancer precursor cells carrying p53 mutations might provide important information about the evolution of the disease. Here we used a combination of OC precursor cell models, biochemistry, microscopy, and flow cytometry to provide evidence that the taurine transporter, the SLC6A6 molecule, contributes to cell protection from DNA-damaging (cisplatin) treatment. We found that expression of mutant p53R175H in OC precursor cells, the fallopian tube non-ciliated epithelial (FNE) cells, induced resistance to the DNA-damaging agent cisplatin. Most importantly, shRNA-mediated targeting of SLC6A6 transcript re-sensitized FNE cells expressing mutant p53R175H to cisplatin treatment. Our studies are consistent with the model that the loss of SLC6A6 alters mechanisms involved in the regulation of cell survival in response to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbf2e1956ceee35cd1cab8cf25a742bffadc692d" target='_blank'>
              Mutant p53R175H-Associated Protection of Ovarian Cancer Precursor Cells from Cisplatin Requires Expression of SLC6A6 Taurine Transporter
              </a>
            </td>
          <td>
            Teagan Polotaye, Daniel Centeno, Marcin Iwanicki
          </td>
          <td>2025-01-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite recent advances, improvements to long-term survival in metastatic carcinomas, such as pancreatic or ovarian cancer, remain limited. Current therapies suppress growth-promoting biochemical signals, ablate cells expressing tumor-associated antigens, or promote adaptive immunity to tumor neoantigens. However, these approaches are limited by toxicity to normal cells using the same signaling pathways or expressing the same antigens, or by the low frequency of neoantigens in most carcinomas. Here, we report a fundamentally different strategy for designing safer and more effective anti-cancer therapies through the sensing of cancer-driving biochemical signals and their rewiring to virotherapeutic activation. Specifically, we rationally engineer a RNA vector to self-replicate and cause cytotoxicity in cancer cells exhibiting hyperactive HER2 (ErbB2), but not in normal cells with normal HER2 signaling. Compared to a widely tested virotherapeutic from the same vector family, our hyperactive ErbB2-restricted RNA vector (HERV) exhibits lower toxicity and greater activity against metastatic HER2-positive ovarian cancer in mice, extending survival independently of tumor antigenicity. Most importantly, HERV synergizes with standard-of-care chemotherapy against ovarian cancer metastases in vivo, with 43% of combination-treated subjects surviving for months beyond subjects treated with either therapy alone. Taken together, these results introduce rewiring of cancer-driving signaling pathways to virotherapeutic activation as a strategy for more specific and effective cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/519eb18e674bcbdbaf122e51b6002c650afbd522" target='_blank'>
              Rewiring oncogenic signaling to RNA vector replication for the treatment of metastatic cancer
              </a>
            </td>
          <td>
            Xinzhi Zou, Cynthia Zhao, Kevin T. Beier, Chil-Yong Kang, Michael Z. Lin
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e4d241948b318cfaca392b0e51828be765d7828" target='_blank'>
              Stable minichromosome and functional neocentromere derived from rye 7R chromosome arm
              </a>
            </td>
          <td>
            Zong-xiang Tang, Qian Liu, Zijin Pan, Chang Liu, Jieran Dong, Fang Han, S. Fu
          </td>
          <td>2024-12-18</td>
          <td>BMC Plant Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbc5c8e78609811ba357b3aeb0141c3d25fbb990" target='_blank'>
              Heterochromatin-dependent transcription links the PRC2 complex to small RNA-mediated DNA elimination
              </a>
            </td>
          <td>
            Therese Solberg, Chundi Wang, Ryuma Matsubara, Zhiwei Wen, M. Nowacki
          </td>
          <td>2024-11-29</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a321392f82837178158946451c3a5d2c2f02daa" target='_blank'>
              Genomic and transcriptomic profiling of radioresistant prostate and head and neck cancers implicate a BAHD1-dependent modification of DNA damage at the heterochromatin
              </a>
            </td>
          <td>
            Chaw Yee Beh, Celestia P X Yeo, Boon Hao Hong, E. Tan, Kah M Tan, Dennis J J Poon, Pek L Chu, Dewi Susanti, Pei Ling Tai, Monica Ryu, James Proudfoot, E. Yeo, Khee Chee Soo, Melvin L. K. Chua
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Multiple oncogenic mutations and the cross stalk between normal pathways are the sources of the complexity involved in Colorectal Cancer (CRC) therapy. Mutated from of a type of gene called protooncogenes, which is involved in normal cell growth and disunion. A tumor suppressor gene, also known as an antioncogene, controls a cell's division and replication. Activation of different protooncogenes can lead to the carcinogenesis of colorectal cancer. Despite important progress attained in the symptomatic and therapeutic management of patients with colorectal cancer, there has been newly a significant growth in the prevalence of large intestine cancer in individuals below the age of 50 years. Primal inception of colorectal cancer has a complex incidence of mucinous histology, a more remote site, a different deoxyribonucleic acid (DNA) methylation profile, and a reduced survival rate. Understanding the pathophysiological mechanisms underlying this oncogenesis process may also make it easier to identify new targets for treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40d123e7ceee593665184d8be221d58c53cbbb2c" target='_blank'>
              A review on role of oncogene (KRAS) & tumor suppressor gene (TP53) in colorectal cancer
              </a>
            </td>
          <td>
            Nori Kodanda Ram, K. Triveni, P. Akshay, Sk. Shameera, J. Tharun Naik, T. Gopinath, J.N. Suresh Kumar
          </td>
          <td>2024-12-01</td>
          <td>International Journal of Allied Medical Sciences and Clinical Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/548909e3fc8f58fb6f1caf3e51e4cd2b9742adb3" target='_blank'>
              Unique structural configuration of EV-DNA primes Kupffer cell-mediated antitumor immunity to prevent metastatic progression.
              </a>
            </td>
          <td>
            I. Wortzel, Yura Seo, Ife Akano, Lee Shaashua, G. Tobias, Jakob Hebert, Kyung-A Kim, DooA Kim, Shani Dror, Yanshen Liu, Griffin Campbell Azrak, Michele Cioffi, Kofi Ennu Johnson, Tammy Hennika, Meshulam Zisha Twerski, Alexis Kushner, Robert Math, Yoon Dae Han, Dai Hoon Han, Minsun Jung, Juyeong Park, Soonmyung Paik, Jeon-Soo Shin, Min Goo Lee, Marco Vincenzo Russo, D. Zakheim, Jesse Barnes, Sunjoy Mehta, Katia Manova, R. Schwartz, B. K. Thakur, Nancy Boudreau, Irina R Matei, Haiying Zhang, Simone Sidoli, Jacqueline Bromberg, Yael David, H. Kim, D. Lyden
          </td>
          <td>2024-12-03</td>
          <td>Nature cancer</td>
          <td>1</td>
          <td>25</td>
        </tr>

        <tr id="Identifying the roles of genes in cancer is critical in discovering potential genetic therapies for cancer care. Translocon-associated protein delta (TRAPδ), also known as signal sequence receptor 4 (SSR4), is a constituent unit in the TRAP/SSR complex that resides in the endoplasmic reticulum and plays a key role in transporting newly synthesized proteins into the endoplasmic reticulumn. However, its biological role in disease development remains unknown to date. This is the first study to identify the role of TRAPδ/SSR4 in colorectal cancer cells in vitro. Upon successful transient knockdown of TRAPδ/SSR4, we observed significant reduction of cell viability in all colorectal cancer cell lines tested. Both HCT 116 and SW480 cell lines were significantly arrested at S and G1 phases, while DLD-1 cells were significantly apoptotic. Moreover, TRAPδ/SSR4 stable knockdown HCT 116 and SW480 cells showed significantly lower viability, anchorage-independent growth, and increased S and G1 phase arrests. Overall, we conclude TRAPδ/SSR4 is a potential oncogene in human colorectal cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c566aa31384f1776417a4c1880816c68b25b7d5" target='_blank'>
              Identification of Translocon-associated Protein Delta as An Oncogene in Human Colorectal Cancer Cells
              </a>
            </td>
          <td>
            Darshika Amarakoon, Wu-Joo Lee, Jing Peng, Seong-Ho Lee
          </td>
          <td>2024-12-30</td>
          <td>Journal of Cancer Prevention</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract As a chromatin remodelling factor, high mobility group A1 (HMGA1) plays various roles in both physiological and pathological conditions. However, its role in DNA damage response and DNA damage‐based chemotherapy remains largely unexplored. In this study, we report the poly ADP‐ribosylation (PARylation) of HMGA1 during DNA damage, leading to desensitization of esophageal squamous cell carcinoma (ESCC) cells to the poly(ADP‐ribose) polymerase 1 (PARP1) inhibitor, olaparib. We found that HMGA1 accumulates at sites of DNA damage, where it interacts with PARP1 and undergoes PARylation at residues E47 and E50 in its conserved AT‐hook domain. This modification enhances the accumulation of Ku70/Ku80 at the site of DNA damage and activates the DNA‐dependent protein kinase catalytic subunit, facilitating nonhomologous end‐joining repair. In both subcutaneous tumour models and genetically engineered mouse models of in situ esophageal cancer, HMGA1 interference increased tumour sensitivity to olaparib. Moreover, HMGA1 was highly expressed in ESCC tissues and positively correlated with PARP1 levels as well as poor prognosis in ESCC patients. Taken together, these findings reveal a mechanistic link between HMGA1 and PARP1 in regulating cell responses to DNA damage and suggest that targeting HMGA1 could be a promising strategy to increase cancer cell sensitivity to olaparib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/419174c65c94af6161e05b00b1d3b8c1060c25e0" target='_blank'>
              PARylation of HMGA1 desensitizes esophageal squamous cell carcinoma to olaparib
              </a>
            </td>
          <td>
            Xin-Yuan Lei, Kai-Yue He, Qiu-Tong Li, Lei Zhang, Dan-Hui Wu, Jing-Yu Yang, Jin-Rong Guo, Meng-Jie Liu, Zi-Long Zhao, Jun-Qi Li, Huai Liu, Yuan Zhao, Yu-Jia Li, Qian-Hui Sun, Chen-Guang Wu, Yun-Fan Wang, Geng-Sheng Cao, Gang Wang, Yong-Ping Jian, Z. Xu
          </td>
          <td>2024-12-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bb4f23b9e1892130e632d422e0888edddfeef1" target='_blank'>
              Molecular heterogeneity and MYC dysregulation in triple-negative breast cancer: genomic advances and therapeutic implications.
              </a>
            </td>
          <td>
            Priya, Arun Kumar, Dhruv Kumar
          </td>
          <td>2025-01-01</td>
          <td>3 Biotech</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Digital papillary adenocarcinoma (DPAC) is a rare but aggressive cutaneous malignant sweat gland neoplasm that occurs on acral sites. Despite its clinical significance, the cellular and genetic characteristics of DPAC remain incompletely understood.


METHODS
We conducted a comprehensive genomic and transcriptomic analysis of DPAC (n = 14) using targeted next-generation DNA and RNA sequencing, along with gene expression profiling employing the Nanostring Technologies nCounter IO 360 Panel. Gene expression in DPAC was compared to that in hidradenoma (n = 10). Immunohistochemistry was employed to validate gene expression.


RESULTS
Two out of eight DPACs showed fusion gene rearrangements (CRTC3::MAML2 and TRPS1::PLAG1). No uniform mutational signature was detected in DPAC. Comparative gene expression analysis revealed an enrichment of genes related to matrix remodeling, metabolism, and DNA damage repair. Hallmark pathway analysis demonstrated significant upregulation of E2F target genes in DPAC compared to hidradenoma (p = 0.00710). Human papillomavirus-42 was found to be positive in all of our tested DPAC cases. Immunohistochemistry confirmed increased protein expression of CD56, CDC20, and SOX10 in DPAC. Notably, most DPAC tumors also exhibited B-cell infiltration, as indicated by CD20 staining.


CONCLUSIONS
Our findings reveal novel fusions and validate altered replication pathways related to HPV42 in DPAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ef537853ab84e6a0c47fe3518be7d94ccfae770" target='_blank'>
              Genomic and Transcriptomic Profiling of Digital Papillary Adenocarcinomas Reveals Alterations in Matrix Remodeling and Metabolic Genes.
              </a>
            </td>
          <td>
            Erol Bayraktar, Phyu P Aung, Pavandeep Gill, G. Shen, Varshini Vasudevaraja, Zongshan Lai, Luis Chiriboga, D. Ivan, P. Nagarajan, J. Curry, C. Torres‐Cabala, Victor G. Prieto, G. Jour
          </td>
          <td>2025-01-06</td>
          <td>Journal of cutaneous pathology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Cancer is caused by mutations that drive aberrant growth, proliferation, and invasion, thus overriding regulatory mechanisms that normally link these processes to organismal needs and cellular physiology. This imposes demands for the production of energy and biomass and for survival in microenvironments that are often nonphysiologic and nutrient-poor, which are met by rewiring of cellular metabolism. The resultant dependence of tumor cells on altered metabolism can induce sensitivity to specific metabolic perturbations that can be exploited for cancer therapy. Some cancers are caused by mutations that impart a novel function to metabolic enzymes, leading to the production of a tumor-promoting metabolite that is dispensable in normal cells, representing an ideal therapeutic target. Tumors can also exploit metabolic regulation of cellular immunity to evade antitumor immune responses, and deciphering this biology has revealed potential targets for therapeutic intervention. Here, we discuss a number of illustrative examples highlighting the therapeutic potential and the challenges of targeting metabolism for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/096951e0d89ec39b14a1cc5abbb929dff9f4e14a" target='_blank'>
              Cancer Therapies Targeting Cellular Metabolism.
              </a>
            </td>
          <td>
            Benjamin Morris, Alejandro Gutierrez
          </td>
          <td>2024-11-25</td>
          <td>Cold Spring Harbor perspectives in medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af37e793e7291e5ec4163cfb7481755bd374769" target='_blank'>
              Spatial transcriptomics reveals unique metabolic profile and key oncogenic regulators of cervical squamous cell carcinoma
              </a>
            </td>
          <td>
            Limin Zhou, Jiejie Liu, Peipei Yao, Xing Liu, Fei Chen, Yu Chen, Li Zhou, Chao Shen, You Zhou, Xin Du, Junbo Hu
          </td>
          <td>2024-12-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) is the most common soft tissue sarcoma in children, but the outcomes of high-grade RMS patients remain poor, underscoring the critical need for novel therapeutic strategies. Although metabolic pathways in RMS are incompletely characterized, emerging evidence suggests that metabolic adaptations in RMS resemble those in other malignancies. Here, we identify elevated cholesterol biosynthesis driven by the PROX1 transcription factor as a defining feature of RMS. Our findings demonstrate that the cholesterol biosynthesis pathway is essential for RMS cell growth, proliferation, and survival. Blocking this pathway through genetic or pharmacological inhibition of the key cholesterol biosynthesis enzymes significantly impairs RMS cell proliferation, halts cell cycle progression, and triggers apoptosis through activation of endoplasmic reticulum stress pathways. We furthermore validate the critical role of cholesterol biosynthesis in RMS progression in tumor xenograft models, demonstrating that silencing of the DHCR7 gene significantly suppresses tumor growth. Transcriptomic analysis revealed widespread downregulation of cell cycle-related genes following DHCR7 silencing, further supporting the role of cholesterol metabolism in cell cycle regulation. These results highlight the vulnerability of RMS cells to cholesterol biosynthesis inhibition and suggest that targeting this metabolic pathway as a promising therapeutic approach for improving RMS outcomes. Our findings provide a rationale for the development of novel therapies targeted to cholesterol biosynthesis in this aggressive cancer. Significance This study reveals that targeting cholesterol biosynthesis in rhabdomyosarcoma induces ER-stress, apoptosis and cell cycle arrest, highlighting a potential therapeutic strategy for treating this aggressive pediatric cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1168b7a6afff81a3b6df61f48c435d90b22e7cf3" target='_blank'>
              Targeting de novo cholesterol synthesis in rhabdomyosarcoma induces cell cycle arrest and triggers apoptosis through ER stress-mediated pathways
              </a>
            </td>
          <td>
            N. Y. Gizaw, Kalle Kolari, Kari Alitalo, R. Kivelä
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="While the genetic paradigm of cancer etiology has proven powerful, it remains incomplete as evidenced by the widening spectrum of non-cancer cell-autonomous “hallmarks” of cancer. Studies have demonstrated the commonplace presence of high oncogenic mutational burdens in homeostatically-stable epithelia. Hence, the presence of driver mutations alone does not result in cancer. Here, we reveal additional forces governing the eco-evolutionary dynamics of carcinogenesis. Using a UV-driven mouse model of cutaneous squamous cell carcinoma, we tested our central hypothesis that cancer initiation occurs in three phases: 1) tissue disruption and the emergence of unusually large “goliath” clades (ecological driver), 2) clonal selection within a subset of these goliaths as evidenced by the presence of areas of unusually high local densities of cells (termed “micro-lumps”) with higher mutational burdens (evolutionary driver), and 3) emergence of macroscopic lesions. We tracked these tissue level ecological and evolutionary drivers of cancer initiation via in-vivo serial imaging and 3-D reconstruction of fluorescently labeled keratinocyte clades, yielding 25,085 clade measurements from 14 mice over 3 months, and 14,525 clades from 4 of these mice over 6 months. While median and mean clade sizes differed little between UV and non-UV exposure (cc. 65,000 µm3), our ecological survey revealed the emergence of large goliath clades (> cc. 4.2×106 µm3), almost exclusively within UV-exposed skin. Goliath clades emerged by month 2, increased dramatically in number by months 3-4, and plateaued between months 5-6. Goliath clades arose as serpentine structures, intercalating between adjacent keratinocytes. Unexpectedly, targeted DNA sequencing revealed mutations with very low variant allele frequencies within clades, but substantial differences among clades, suggesting that positive selection for these mutations is superfluous to the development of goliaths early in carcinogenesis. scRNAseq analysis of both bulk skin and sorted clades revealed epidermal de-differentiation and immune suppression as early events. By month 7, mutational burdens were significantly larger in goliath clades, particularly for those with micro-lumps. Finally, lesions began to emerge between months 6 and 7, only in UV-exposed skin. To confirm that goliath clades are orders of magnitude more likely to spawn lesions, we randomly selected 21 goliath clades at months 6-7, prior to the time of the emergence of detectable lesions, and followed them over time. Remarkably, 2 of these developed into macroscopic lesions. Our adaptation of the Drake equation estimated the probability of this to be <10-6. Taken together, our results support the presence of three phases of cancer initiation, the earliest of which presage the acquisition of driver mutations and also explains why cancers are relatively rare in relation to the degree of somatic mosaicism present in UV-exposed skin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75272d3f8095e90a23d2fb0d732b9102d07a0066" target='_blank'>
              Goliath clades and in vivo tracking of clonal dynamics show three phases of UV-induced skin carcinogenesis
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chavez Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Cancer remains one of the most significant public health challenges worldwide. A widely recognized hallmark of cancer is the ability to sustain proliferative signaling, which is closely tied to various cell cycle processes. Centromere Protein A (CENPA), a variant of the standard histone H3, is crucial for selective chromosome segregation during the cell cycle. Despite its importance, a comprehensive pan-cancer bioinformatic analysis of CENPA has not yet been conducted. Methods Data on genomes, transcriptomes, and clinical information were retrieved from publicly accessible databases. We analyzed CENPA’s genetic alterations, mRNA expression, functional enrichment, association with stemness, mutations, expression across cell populations and cellular locations, link to the cell cycle, impact on survival, and its relationship with the immune microenvironment. Additionally, a prognostic model for glioma patients was developed to demonstrate CENPA’s potential as a biomarker. Furthermore, drugs targeting CENPA in cancer cells were identified and predicted using drug sensitivity correlations and protein-ligand docking. Results CENPA exhibited low levels of gene mutation across various cancers. It was found to be overexpressed in nearly all cancer types analyzed in TCGA, relative to normal controls, and was predominantly located in the nucleus of malignant cells. CENPA showed a strong association with the cancer cell cycle, particularly as a biomarker for the G2 phase. It also emerged as a valuable diagnostic and prognostic biomarker across multiple cancer types. In glioma, CENPA demonstrated reliable prognostic potential when used alongside other prognostic factors. Additionally, CENPA was linked to the immune microenvironment. Drugs such as CD-437, 3-Cl-AHPC, Trametinib, BI-2536, and GSK461364 were predicted to target CENPA in cancer cells. Conclusion CENPA serves as a crucial biomarker for the cell cycle in cancers, offering both diagnostic and prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd288f4dd8225d97f5117baac02d9eac80a26ed1" target='_blank'>
              Single-cell and bulk RNA sequencing analysis reveals CENPA as a potential biomarker and therapeutic target in cancers
              </a>
            </td>
          <td>
            Hengrui Liu, Miray Karsidag, Kunwer S. Chhatwal, Panpan Wang, Tao Tang
          </td>
          <td>2025-01-16</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mobile group II introns are site-specific retrotransposons composed of a large self-splicing ribozyme and an intron-encoded reverse transcriptase that are widespread in bacterial and organellar genomes. Sequence and structural variations of the ribozyme and the associated reverse transcriptase define several lineages of bacterial group II introns. Interestingly, some of these intron families evolved different mobility strategies while others colonize particular genetic contexts. Here, we have investigated the mobility activity of an Escherichia coli group II intron that is inserted into the stop codon of the stress-response gene groEL. Using mobility assays based on over-expression from a donor plasmid, we demonstrate that this intron is a highly efficient and site-specific retrotransposon, capable of colonizing the groEL gene of an E. coli host strain according to the insertion pattern observed in natural genomes. Furthermore, we provide evidence that a chromosomal copy of the full-length retrotransposon can be expressed from its native genetic locus to yield mobile retroelement particles. This intron constitutes a novel model system that could help reveal original mobility strategies used by some group II intron retrotransposons to colonize bacterial genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd2d620dc3d0d9203fcc02022053cec2afa25859" target='_blank'>
              A specialized bacterial group II intron is a highly efficient retrotransposon
              </a>
            </td>
          <td>
            Lucie Gomes, C. Toffano-Nioche, Daniel Gautheret, Maria Costa
          </td>
          <td>2024-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef930bd56fde447e683b8e74f0950f751b9d7434" target='_blank'>
              Reassessing the roles of oxidative DNA base lesion 8-oxoGua and repair enzyme OGG1 in tumorigenesis
              </a>
            </td>
          <td>
            Jing Wang, Chunshuang Li, Jinling Han, Yaoyao Xue, Xu Zheng, Ruoxi Wang, Z. Radák, Y. Nakabeppu, Istvan Boldogh, Xueqing Ba
          </td>
          <td>2025-01-01</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Genetic modifications using CRISPR-Cas9 have revolutionized cancer research and other pre-clinical studies. Exceptionally, these efficient tools are inadequate in a few disease models and cell lines due to the aberrant differentiation states and the accumulation of excessive somatic mutations that compromise the robustness of viral gene delivery and stable transduction. A couple of B lymphoma cell lines fall into this category where lentiviral transfection becomes inefficient and exhibits variable efficiency. Additionally, lentiviral delivery requires high biosafety levels. To address this challenge, we have developed a two-step strategy that supports CRISPR-Cas9 through lentivirus and murine ecotropic γ-retrovirus. By engineering B lymphoma cell lines to express Cas9 and mCat-1, a specific receptor for ecotropic retroviruses, we enable efficient and safe gene editing through ecotropic γ-retrovirus. We demonstrate the efficacy of this method by generating IgM-deficient B lymphoma cell lines. This innovative approach simplifies protocols, enhances accessibility, and paves the way for standardized gene manipulation of B cell lymphoma models for molecular cell biology research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88a2c45cfa5f916fccda93d3f9c47decafd7f279" target='_blank'>
              Facilitating Gene Editing in Human Lymphoma Cells Using Murine Ecotropic γ-Retroviruses
              </a>
            </td>
          <td>
            Manish Kumar, Eva Gentner-Göbel, Palash Chandra Maity
          </td>
          <td>2024-12-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="IDH-mutant low-grade gliomas (LGGs) are slow-growing brain tumors that frequently progress to aggressive high-grade gliomas that have dismal outcomes. In a recent study, Wu and colleagues provide critical insights into the mechanisms underlying malignant progression by analyzing single-cell gene expression and chromatin accessibility across different tumor grades. Their findings support a two-phase model: in early stages, tumors are primarily driven by oligodendrocyte precursor-like cells and epigenetic alterations that silence tumor suppressors like CDKN2A and activate oncogenes such as PDGFRA. As the disease advances, the tumors become sustained by more proliferative neural precursor-like cells, where genetic alterations, including PDGFRA, MYCN, and CDK4 amplifications and CDKN2A/B deletion, drive tumor progression. The study further highlights a dynamic regulation of interferon (IFN) signaling during progression. In low-grade IDH-mutant gliomas, IFN responses are suppressed through epigenetic hypermethylation, which can be reversed with DNMT1 inhibitors or IDH inhibitors, leading to reactivation of the IFN pathway. In contrast, higher-grade gliomas evade IFN signaling through genetic deletions of IFN gene clusters. These findings emphasize a broader epigenetic-to-genetic shift in oncogenic regulation that drives glioma progression, provides a valuable framework for understanding the transition from indolent tumors to lethal malignancies, and has implications for therapy and clinical management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/598e5f4259516cb6e86cc401bbe6fe663aa489ab" target='_blank'>
              Switching Drivers: Epigenetic Rewiring to Genetic Progression in Glioma.
              </a>
            </td>
          <td>
            K. Drucker, Robert B. Jenkins, Daniel Schramek
          </td>
          <td>2024-12-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Constitutively active NOTCH2 signaling is a hallmark in chronic lymphocytic leukemia (CLL). The precise underlying defect remains obscure. Here we show that the mRNA sequence coding for the NOTCH2 negative regulatory region (NRR) is consistently deleted in CLL cells. The most common NOTCH2ΔNRR-DEL2 deletion is associated with two intronic single nucleotide variations (SNVs) which either create (CTTAT, G>A for rs2453058) or destroy (CTCGT, A>G for rs5025718) a putative splicing branch point sequence (BPS). Phylogenetic analysis demonstrates that rs2453058 is part of an ancient NOTCH2 gene variant (*1A01) which is associated with type 2 diabetes mellitus (T2DM) and is two times more frequent in Europeans than in East Asians, resembling the differences in CLL incidence. In contrast, rs5025718 belongs to a recent NOTCH2 variant (*1a4) that dominates the world outside Africa. Nanopore sequencing indicates that somatic reciprocal crossing over between rs2453058 (*1A01) and rs5025718 (*1a4) leads to recombined NOTCH2 alleles with altered BPS patterns in NOTCH2*1A01/*1a4 CLL cases. This would explain the loss of the NRR domain by aberrant pre-mRNA splicing and consequently the NOTCH2 gain-of-function phenotype. Together, our findings suggest that somatic recombination of inherited NOTCH2 variants might be relevant to CLL etiology and may at least partly explain its geographical clustering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76d5b526c4b4d5a4555e25526b7ffec423997f32" target='_blank'>
              Somatic Recombination Between an Ancient and a Recent NOTCH2 Gene Variant Is Associated with the NOTCH2 Gain-of-Function Phenotype in Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            Rainer Hubmann, M. Hilgarth, Tamara Löwenstern, Andrea Lienhard, Filip Sima, Manuel Reisinger, Claudia Hobel-Kleisch, E. Porpaczy, Torsten Haferlach, Gregor Hoermann, Franco Laccone, C. Jungbauer, P. Valent, Philipp B. Staber, M. Shehata, Ulrich Jäger
          </td>
          <td>2024-11-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Biomolecular condensates (BCs), RNA–protein complexes, have emerged as potential therapeutic targets for various cellular pathologies, including cancer. The mechanisms underlying liquid–liquid phase separation rely on the properties of proteins with intrinsically disordered regions and prion-like domains, which drive phase separation in conjunction with their RNA partners. These ribonucleoprotein complexes are distinct in their localization, compartmentalization, epigenetic regulation, and dynamics. BCs are categorized as either nuclear—such as promyelocytic leukemia nuclear bodies, speckles, paraspeckles, and Cajal bodies—or cytosolic, including stress granules (SGs), P bodies, and U bodies. Regulatory RNAs, assembled with protein partners through phase separation, exhibit oncogenic properties and perform key biological functions, including gene transcription, euchromatin/heterochromatin formation, mRNA splicing, mRNA translation, protein compartmentalization, and degradation. Epitranscriptome-modifying enzymes regulate the stability and oncogenic potential of noncoding RNAs (ncRNAs). These RNAs, along with their associated epitranscriptomic and protein modifications, play critical roles in the functioning and dissolution of condensates. Recent advancements in cancer therapy have focused on developing drugs targeting the epitranscriptomic machinery, including the writers, readers, and erasers of RNA modifications. Therapeutic strategies aim to target oncogenic RNAs, tumor-promoting proteins, and RNA–protein interaction domains. Several cancer therapeutic compounds have been developed using the PROTAC and RIBOTAC approaches. Moreover, small molecules targeting protein–RNA interactions and antisense oligonucleotides have been developed. Promising avenues in cancer therapeutics involve the inhibition of ncRNAs and their associated protein complexes, modulation of BCs, regulation of SGs and paraspeckles, and development of small molecule compounds with potential applications across various cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/562198ac466da5ff5183f52f8424cad1457ee4aa" target='_blank'>
              Targeting the interplay between biomolecular condensates and regulatory RNAs in cancer
              </a>
            </td>
          <td>
            P. Poltronieri, Sudipta Joardar
          </td>
          <td>2024-12-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Myc hyperactivation coordinately regulates numerous metabolic processes to drive lymphomagenesis. Here, we elucidate the temporal and functional relationships between the medley of pathways, factors, and mechanisms that cooperate to control redox homeostasis in Myc-overexpressing B cell lymphomas. We find that Myc overexpression rapidly stimulates the oxidative pentose phosphate pathway (oxPPP), nucleotide synthesis, and mitochondrial respiration, which collectively steers cellular equilibrium to a more oxidative state. We identify Myc-dependent hyperactivation of the phosphoribosyl pyrophosphate synthetase (PRPS) enzyme as a primary regulator of redox status in lymphoma cells. Mechanistically, we show that genetic inactivation of the PRPS2 isozyme, but not PRPS1, in Myc-driven lymphoma cells leads to elevated NADPH levels and reductive stress-mediated death. Employing a pharmacological screen, we demonstrate how targeting PRPS1 or PRPS2 elicits opposing sensitivity or resistance, respectively, to chemotherapeutic agents affecting the thioredoxin and glutathione network, thus providing a therapeutic blueprint for treating Myc-driven lymphomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/654099247bbca4f140ebd9c31e764d3e528bd649" target='_blank'>
              PRPS activity tunes redox homeostasis in Myc-driven lymphoma
              </a>
            </td>
          <td>
            Austin C. Macmillan, Bibek R. Karki, Juechen Yang, Karmela R. Gertz, Samantha Zumwalde, M. Czyzyk-Krzeska, Jarek Meller, J. T. Cunningham
          </td>
          <td>2025-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Kaposi’s sarcoma-associated herpesvirus (KSHV), a γ-herpesvirus, is predominantly associated with Kaposi’s sarcoma (KS) as well as two lymphoproliferative disorders: primary effusion lymphoma (PEL) and multicentric Castleman disease (MCD). Like other herpesviruses, KSHV employs two distinct life cycles: latency and lytic replication. To establish a lifelong persistent infection, KSHV has evolved various strategies to manipulate the epigenetic machinery of the host. In latently infected cells, most viral genes are epigenetically silenced by components of cellular chromatin, DNA methylation and histone post-translational modifications. However, some specific latent genes are preserved and actively expressed to maintain the virus’s latent state within the host cell. Latency is not a dead end, but the virus has the ability to reactivate. This reactivation is a complex process that involves the removal of repressive chromatin modifications and increased accessibility for both viral and cellular factors, allowing the activation of the full transcriptional program necessary for the subsequent lytic replication. This review will introduce the roles of epigenetic modifications in KSHV latent and lytic life cycles, including DNA methylation, histone methylation and acetylation modifications, chromatin remodeling, genome conformation, and non-coding RNA expression. Additionally, we will also review the transcriptional regulation of viral genes and host factors in KSHV infection. This review aims to enhance our understanding of the molecular mechanisms of epigenetic modifications and transcriptional regulation in the KSHV life cycle, providing insights for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e65f8fc61715f58044e9d19bdbbd3b68cc4bfb9d" target='_blank'>
              Rewriting Viral Fate: Epigenetic and Transcriptional Dynamics in KSHV Infection
              </a>
            </td>
          <td>
            Chunyan Han, Danping Niu, Ke Lan
          </td>
          <td>2024-11-30</td>
          <td>Viruses</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="DNA topology is a direct consequence of the double helical nature of DNA and is defined by how the two complementary DNA strands are intertwined. Virtually every reaction involving DNA is influenced by DNA topology or has topological effects. It is therefore of fundamental importance to understand how this phenomenon is controlled in living cells. DNA topoisomerases are the key actors dedicated to the regulation of DNA topology in cells from all domains of life. While significant progress has been made in the last two decades in understanding how these enzymes operate in vivo in Bacteria and Eukaryotes, studies in Archaea have been lagging behind. This review article aims to summarize what is currently known about DNA topology regulation by DNA topoisomerases in main archaeal model organisms. These model archaea exhibit markedly different lifestyles, genome organization and topoisomerase content, thus highlighting the diversity and the complexity of DNA topology regulation mechanisms and their evolution in this domain of life. The recent development of functional genomic assays supported by next-generation sequencing now allows to delve deeper into this timely and exciting, yet still understudied topic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a4860c875d7d4484e2675f291d10098c1fde66" target='_blank'>
              Regulation of DNA Topology in Archaea: State of the Art and Perspectives.
              </a>
            </td>
          <td>
            Paul Villain, Tamara Basta
          </td>
          <td>2024-12-22</td>
          <td>Molecular microbiology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly aggressive and clinically challenging subtype of breast cancer, lacking the expression of estrogen receptor (ER), progesterone receptor (PR), and HER2/neu. The absence of these receptors limits therapeutic options necessitating the exploration of novel treatment strategies. Epigenetic modifications, which include DNA methylation, histone modifications, and microRNA (miRNA) regulation, play a pivotal role in TNBC pathogenesis and represent promising therapeutic targets. This review delves into the therapeutic potential of epigenetic interventions in TNBC, with a focus on DNA methylation, histone modifications, and miRNA therapeutics. We examine the role of DNA methylation in gene silencing within TNBC and the development of DNA methylation inhibitors designed to reactivate silenced tumor suppressor genes. Histone modifications, through histone deacetylation and acetylation in particular, are critical in regulating gene expression. We explore the efficacy of histone deacetylase inhibitors (HDACi), which have shown promise in reversing aberrant histone deacetylation patterns, thereby restoring normal gene function, and suppressing tumor growth. Furthermore, the review highlights the dual role of miRNAs in TNBC as both oncogenes and tumor suppressors and discusses the therapeutic potential of miRNA mimics and inhibitors in modulating these regulatory molecules to inhibit cancer progression. By integrating these epigenetic therapies, we propose a multifaceted approach to target the underlying epigenetic mechanisms that drive TNBC progression. The synergistic use of DNA methylation inhibitors, HDACi, and the miRNA-based therapies offers a promising avenue for personalized treatment strategies, aiming to enhance the clinical outcome for patients with TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f782a5505b5b2b26e898e78e4e01384ddeb62d3" target='_blank'>
              Unlocking the epigenetic code: new insights into triple-negative breast cancer
              </a>
            </td>
          <td>
            G. Mahendran, Ann Dharshika, Ricardo Romero-Moreno, Nadeeshika Wickramarachchige Dona, S. H. G. S. Sarasija, Sumeth Perera, Gayathri N. Silva, Fang Yang, Guochun Zhang, Wickramarachchige Dona, Perera Sarasija, Silva. This
          </td>
          <td>2024-12-18</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A retrovirus inserts its genome into the DNA of a cell, occasionally a germ-line cell that gives rise to descendants of the host organism: it is then called an endogenous retrovirus (ERV). The human genome contains relics from many kinds of ancient ERV. Some relics contributed new genes and regulatory elements. This study finds further kinds of ancient ERV, in the thoroughly-studied human genome version hg38: ERV-Hako, ERV-Saru, ERV-Hou, ERV-Han, and ERV-Goku. It also finds many relics of ERV-V, previously known from just two copies on chromosome 19 with placental genes. It finds a type of ERV flanked by MER41E long terminal repeats (LTRs), with surprisingly little similarity to the known MER41 ERV. ERV-Hako has subtypes that contain sequence from host genes SUSD6 and SPHKAP : the SUSD6 variant was transferred between catarrhine and platyrrhine primates. A retrovirus uses tRNA to prime reverse transcription: Hako is the only human ERV relic that used tRNA-Trp (tryptophan, symbol W), and HERV-W is misnamed because it used tRNA-Arg, based on the Genomic tRNA Database. One ERV-Saru LTR is the previously-described enhancer of AIM2 in innate immunity. This study contributes to understanding primate ERV history, but also shows that related ERVs can have drastic differences, challenging the goal of clearly annotating all ERV relics in genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/362de87510ddffd6ce47f2c96a4e681ca10206a6" target='_blank'>
              Further varieties of ancient endogenous retrovirus in human DNA
              </a>
            </td>
          <td>
            Martin C. Frith
          </td>
          <td>2024-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b32d455915f14c9cee5327c8bd03503f5d7e8b33" target='_blank'>
              SNAI1 promotes epithelial-mesenchymal transition and maintains cancer stem cell-like properties in thymic epithelial tumors through the PIK3R2/p-EphA2 Axis
              </a>
            </td>
          <td>
            H. E, Lei Zhang, Zhenhua Yang, Long Xu, Tao Wang, Junhong Guo, Lang Xia, Juemin Yu, Heyong Wang, Y. She, Junqi Wu, Yue Zhao, Chang Chen, Deping Zhao
          </td>
          <td>2024-12-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Cancer cells within tumors exhibit a wide range of phenotypic states driven by non-genetic mechanisms in addition to extensively studied genetic alterations. Conversions among cancer cell states can result in intratumoral heterogeneity which contributes to metastasis and development of drug resistance. However, mechanisms underlying the initiation and/or maintenance of such phenotypic plasticity are poorly understood. In particular, the role of intercellular communications in phenotypic plasticity remains elusive. In this study, we employ a multiscale inference-based approach using single-cell RNA sequencing (scRNA-seq) data to explore how intercellular interactions influence phenotypic dynamics of cancer cells, particularly cancers undergoing epithelial-mesenchymal transition. Our inference approach reveals that signaling interactions between cancerous cells in small cell lung cancer (SCLC) result in seemingly contradictory behaviors—reinforcing the cellular phenotypes and maintaining population-level intratumoral heterogeneity. Additionally, we find a recurring signaling pattern across multiple types of cancer in which the mesenchymal-like subtypes utilize signals from other subtypes to reinforce its phenotype, further promoting the intratumoral heterogeneity. We use a mathematical model based on ordinary differential equations to show that inter-subtype communication accelerates the development of heterogeneous tumor populations. Our work highlights the critical role of intercellular signaling in sustaining intratumoral heterogeneity, and our approach of computational analysis of scRNA-seq data can infer inter- and intra-cellular signaling networks in a holistic manner. Significance Single cell-based inference approach reveals a key role of intercellular signaling in maintaining intratumoral heterogeneity. Cell-cell communications stabilize newly acquired cell states and diverse phenotypes of cell populations in multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa5cf6c7e9bc3e2c2f27d636cb203f597857db1" target='_blank'>
              Intercellular signaling stabilizes single-cell level phenotypic transitions and accelerates the reestablishment of equilibrium of heterogeneous cancer cell populations
              </a>
            </td>
          <td>
            Daniel Lopez, Darren R Tyson, Tian Hong
          </td>
          <td>2025-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The three most prevalent malignant skin tumors are basal cell carcinoma (BCC), squamous cell carcinoma (SCC), and cutaneous melanoma (CM). While BCC and SCC generally exhibit low levels of malignancy, some tumors can become invasive and metastasize. In contrast, CM is the most malignant and lethal form of skin cancer among these three. In recent years, the increasing incidence of skin tumors has garnered significant attention within the medical community. The advent of single-cell RNA sequencing (scRNA-seq) technology has revolutionized the analysis of the tumor microenvironment (TME) in skin tumors, offering novel insights for clinical research. In this review, we provide a concise introduction to scRNA-seq technology and its application in delineating the heterogeneity of the TME in skin tumors, elucidating the mechanisms of tumor progression and pathogenicity, and uncovering new therapeutic targets. Our review offers a comprehensive overview for researchers, offering insights that may advance the understanding of the skin TME in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c806552e713787b83ea0933cd31a52d109ce5cf9" target='_blank'>
              Advances and applications of single-cell RNA sequencing in deciphering the tumor microenvironment of malignant skin tumors
              </a>
            </td>
          <td>
            Yiting Feng, Lanlan Liu, Chunlan Hu, Jie Sun, Mingzhu Yin, Xin Li
          </td>
          <td>2025-01-10</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mitochondrial function and dynamics are crucial for maintaining cellular homeostasis and overall health. Disruptions in these processes can lead to various diseases, including cancer. The Hippo signaling pathway, a universal growth regulator, plays central roles in cancer through its main effector, the Yes-associated protein (YAP), known as Yorkie (Yki) in Drosophila. Yki upregulation drives benign tissue overgrowth in Drosophila imaginal discs. Our research demonstrates that the conserved metabolic regulator dPGC1 limits Yki-driven tissue hyperplasia and maintains tissue homeostasis in vivo. Yki upregulation and dPGC1 depletion lead to tumors characterized by enlarged and hyperfused mitochondria, a condition both necessary and sufficient for Yki-driven oncogenic growth. Our findings demonstrate that mitochondrial hyperfusion elevates the levels of the cell cycle regulator Cyclin E, which is crucial for tumor development. Our findings identify dPGC1 as a context-dependent tumor suppressor that coordinates mitochondrial dynamics and cell cycle regulation in response to oncogene activation. Defects in these processes are commonly found in cancer cells, highlighting major implications for cancer development in humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/238724bd493ec2c48bc12750dc3d9f0183bf9e62" target='_blank'>
              Control of mitochondrial dynamics by dPGC1 limits Yorkie-induced oncogenic growth in Drosophila
              </a>
            </td>
          <td>
            Wei Qi Guinevere Sew, Zhiquan Li, L. J. Rasmussen, Héctor Herranz
          </td>
          <td>2024-12-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae8d3d24f9c6f481141224932a60797062129316" target='_blank'>
              Combatting cellular immortality in cancers by targeting the shelterin protein complex
              </a>
            </td>
          <td>
            Sohini Chakraborty, Satarupa Banerjee
          </td>
          <td>2024-11-22</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA methylation is a crucial epigenetic modification that orchestrates chromatin remodelers that suppress transcription, and aberrations in DNA methylation result in a variety of conditions such as cancers and developmental disorders. While it is understood that methylation occurs at CpG-rich DNA regions, it is less understood how distinct methylation profiles are established within various cell types. In this work, we develop a molecular-transport model that depicts the genomic exploration of DNA methyltransferase within a multiscale DNA environment, incorporating biologically relevant factors like methylation rate and CpG density to predict how patterns are established. Our model predicts DNA methylation-state correlation distributions arising from the transport and kinetic properties that are crucial for the establishment of unique methylation profiles. We model the methylation correlation distributions of nine cancerous human cell types to determine how these properties affect the epigenetic profile. Our theory is capable of recapitulating experimental methylation patterns, suggesting the importance of DNA methyltransferase transport in epigenetic regulation. Through this work, we propose a mechanistic description for the establishment of methylation profiles, capturing the key behavioral characteristics of methyltransferase that lead to aberrant methylation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e458fbb01165e95120272a65d7850681688311a0" target='_blank'>
              Modeling DNA methyltransferase function to predict epigenetic correlation patterns in healthy and cancer cells.
              </a>
            </td>
          <td>
            Ariana Y Tse, A. Spakowitz
          </td>
          <td>2025-01-10</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Esophageal squamous cell carcinoma (ESCC) is a global cancer related to the sixth largest cause of death. Circular RNAs (circRNAs) have affected the progress of ESCC during recent years, but the mechanism is not completely clear. So here we probed the effects of hsa_circ_0096710 (circ_0096710) in ESCC. Methods Relative levels of circ_0096710, miR‐1294, and ADAM10 were quantified by the quantitative real‐time reverse transcription‐polymerase chain reaction in ESCC tissues. Western blot assessed ADAM10, PCNA, MMP2, VEGFA, and OCT4 protein levels. Cell proliferative capacity was assessed by cell counting and cell colony‐forming assays. Transwell assays assessed cell migration and invasion. Angiogenesis was detected by tube formation assays. Stemness of cancer cells was estimated by sphere formation assays. Dual‐luciferin reporter and RNA immunoprecipitation assays determined the targeting relationship between miR‐1294 and circ_0096710 or ADAM10. Results Relative levels of circ_0096710 and ADAM10 mRNA were upregulated in ESCC cells, yet miR‐1294 was downregulated. Circ_0096710 silencing repressed ESCC cell proliferation, migration, invasion, angiogenesis, and stem‐like properties. Moreover, circ_0096710 was an upstream target of miR‐1294, and miR‐1294 inhibition reversed the role of circ_0096710 downregulation in ESCC cells. Furthermore, ADAM10 was a downstream target of miR‐1294, and miR‐1294 overexpression suppressed ESCC cell proliferation, migration, invasion, angiogenesis, and stem‐like properties by targeting ADAM10. Meanwhile, circ_0096710 upgraded ADAM10 expression through sponging miR‐1294. Also, circ_0096710 downregulation restrained tumor growth in mouse models. Conclusion Circ_0096710 upregulates ADAM10 via mediating miR‐1294 expression so as to accelerate the occurrence of ESCC, suggesting that circ_0096710 may be a potential therapeutic target for ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8afaf2aab928485dbc4481effb0cf767394c462" target='_blank'>
              Circ_0096710 facilitates tumor growth via controlling ADAM10 expression in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Chaoqun Dong, Zhilong Li
          </td>
          <td>2024-12-02</td>
          <td>Thoracic Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Prostate cancer is a malignant tumor caused by the malignant proliferation of epithelial cells, which is highly heterogeneous and drug-resistant, and neuroendocrine prostate cancer (NEPC) is an essential cause of drug resistance in its late stage. Elucidating the evolution of NEPC and the resistance process of enzalutamide, a novel antiandrogen, will be of great help in improving the prognosis of patients. As a research hotspot in the field of molecular biology in recent years, the wide range of biological functions of long noncoding RNAs (lncRNAs) has demonstrated their position in the therapeutic process of many diseases, and a large number of studies have revealed their critical roles in tumor progression and drug resistance. Therefore, elucidating the involvement of lncRNAs in the formation of NEPCs and their interrelationship with enzalutamide resistance may provide new ideas for a deeper understanding of the development of this disease and the occurrence of enzalutamide resistance and give a new direction for reversing the therapeutic dilemma of advanced prostate cancer. This article focuses on lncRNAs that regulate enzalutamide resistance and the neuroendocrine transition of prostate cancer through epigenetic, androgen receptor (AR) signaling, and non-AR pathways that act as “molecular sponges” interacting with miRNAs. Some insights into these mechanisms are used to provide some help for subsequent research in this area.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50743b295e6011754fbae884a2dd3b94e2ad534d" target='_blank'>
              Long noncoding RNA mediates enzalutamide resistance and transformation in neuroendocrine prostate cancer
              </a>
            </td>
          <td>
            Zhe Zhu, Wenjing Xuan, Chaohui Wang, Chancan Li
          </td>
          <td>2024-11-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="This study explored the genomic alterations in Yarrowia lipolytica, a key yeast in industrial biotechnology, under both spontaneous and mutagen-induced conditions. Our findings reveal that spontaneous mutations occur at a rate of approximately 4 × 10-10 events per base pair per cell division, primarily manifesting as single-nucleotide variations (SNVs) and small insertions and deletions (InDels). Notably, C-to-T/G-to-A transitions and C-to-A/G-to-T transversions dominate the spontaneous SNVs, while 1 bp deletions, likely resulting from template slippage, are the most frequent InDels. Furthermore, chromosomal aneuploidy and rearrangements occur, albeit at a lower frequency. Exposure to ultraviolet (UV) light, methylmethane sulfonate (MMS), and Zeocin significantly enhances the rates of SNVs and alters their mutational spectra in distinct patterns. Notably, Zeocin-induced SNVs are predominantly T-to-A and T-to-G substitutions, often occurring within the 5'-TGT*-3' motif (* denotes the mutated base). Additionally, Zeocin exhibits a higher potency in stimulating InDels compared to UV and MMS. Translesion DNA synthesis is implicated as the primary mechanism behind most Zeocin-induced SNVs and some InDels, whereas non-homologous end joining serves as the main pathway for Zeocin-mediated InDels. Intriguingly, the study identifies the gene YALI1_E21053g, encoding a protein kinase, as negatively associated with Zeocin resistance. Overall, our results not only deepened our knowledge about the genome evolution in Y. lipolytica but also provided reference to develop innovative strategies to harness its genetic potential.IMPORTANCEYarrowia lipolytica exhibits high environmental stress tolerance and lipid metabolism capabilities, making it a microorganism with significant industrial application potential. In this study, we investigated the genomic variation and evolutionary patterns of this yeast under both spontaneous and induced mutation conditions. Our results reveal distinctive mutation spectra induced by different mutagenic conditions and elucidate the underlying genetic mechanisms. We further highlight the roles of non-homologous end joining and translesion synthesis pathways in Zeocin-induced mutations, demonstrating that such treatments can rapidly confer drug resistance to the cells. Overall, our research enhances the understanding of how yeast genomes evolve under various conditions and provides guidance for developing more effective mutagenesis and breeding techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b4727fb2831092d46e2f4802e9e2fadf0e55a7" target='_blank'>
              Patterns of spontaneous and induced genomic alterations in Yarrowia lipolytica.
              </a>
            </td>
          <td>
            Yuan-Ru Xiong, Yuan-Chun Fang, Min He, Keke Li, Lei Qi, Yang Sui, Ke Zhang, Xuechang Wu, Liang Meng, Ou Li, Dao-Qiong Zheng
          </td>
          <td>2024-12-23</td>
          <td>Applied and environmental microbiology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9029d65e0d9d3415e021015d3185275320222a73" target='_blank'>
              G-quadruplex stabilization provokes DNA breaks in human PKD1, revealing a second hit mechanism for ADPKD
              </a>
            </td>
          <td>
            Agata M Parsons, Seth Byrne, Jesse Kooistra, John Dewey, Aaron L Zebolsky, Gloria Alvarado, Gerrit J Bouma, Gregory B Vanden Heuvel, Erik D Larson
          </td>
          <td>2025-01-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Deoxyribose nucleic acid (DNA) methylation is an important epigenetic modification that plays an important role in the occurrence and development of tumors. Identifying key methylation-driven genes that affect the prognosis of lung squamous cell carcinoma (LUSC) can provide direction for targeted therapy research. Methods and results Methylation and RNA-seq data were downloaded from The Cancer Genome Atlas (TCGA). The MethylMix package was used to integrate and analyze the methylation and gene expression data from TCGA, and the LUSC dataset (GSE37745) was downloaded from GEO for validation. Forty-five DNA-methylation-driven genes (MDGs) were obtained, and 3 genes (TRIM61, SMIM22, and ALDH7A1) were significantly associated with survival by using univariate and multivariate Cox regression. A risk model was constructed. KM analysis showed that patients with high-risk scores had poor survival. A nomination plot for prognosis prediction of LUSC patients was constructed, which showed a good predictive efficiency for tumor prognosis. The high expression of ALDH7A1 was an independent risk factor for poor prognosis in LUSC. The expression of ALDH7A1 in LUSC was negatively correlated with its methylation status (COR = −0.655). GSEA analysis showed that high expression of ALDH7A1 could activate multiple signaling pathways (JAK-STAT signaling pathway and mTOR signaling pathway). In vitro cell experiments confirmed that in LUSC, silencing ALDH7A1 could inhibit tumor progression, while overexpression of ALDH7A1 could promote tumor progression. Conclusion Our results indicated that ALDH7A1, a newly discovered MDG in LUSC, could act as an independent prognostic factor for OS in LUSC, with the potential to become a potential target for LUSC diagnosis and treatment. High expression of ALDH7A1 in LUSC could promote the occurrence and development of tumors. Signaling pathways, such as JAK-STAT and mTOR signaling pathways, might regulate the high expression of ALDH7A1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fa4fa0f5654e845798a1c36db8d638da005aaf3" target='_blank'>
              Identification of ALDH7A1 as a DNA-methylation-driven gene in lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Gaofeng Liang, Jinxian He, Tian Chen, Liang Zhang, Kaizhong Yu, Weiyu Shen
          </td>
          <td>2024-12-23</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2fc69ff58eaa4bfa3b4fa917eb7dfeb9e6159b0" target='_blank'>
              Genomic and phenotypic stability of fusion-driven pediatric sarcoma cell lines
              </a>
            </td>
          <td>
            Merve Kasan, Florian H. Geyer, Jana Siebenlist, Martin Sill, R. Öllinger, Tobias Faehling, Enrique de Álava, D. Surdez, U. Dirksen, Ina Oehme, Katia Scotlandi, O. Delattre, Martina Müller-Nurasyid, R. Rad, Konstantin Strauch, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2025-01-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fc6572745a00744a0ffe9484307c354e0370541" target='_blank'>
              Short tandem repeats delineate gene bodies across eukaryotes
              </a>
            </td>
          <td>
            William B. Reinar, A. Krabberød, V. O. Lalun, M. Butenko, K.S. Jakobsen
          </td>
          <td>2024-12-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The presence of specific genetic mutations in patients with glioblastoma multiforme (GBM) is associated with improved survival outcomes. Disruption of the DNA damage response (DDR) pathway in tumor cells enhances the effectiveness of radiotherapy drugs, while increased mutational burden following tumor cell damage also facilitates the efficacy of immunotherapy. The ATRX gene, located on chromosome X, plays a crucial role in DDR. The aim of this research is to elucidate the correlation between ATRX mutations and GBM. Dataset obtained from TCGA-GBM were conducted an analysis on the genomic features, biological characteristics, immunopathological markers, and clinical prognosis of patients carrying ATRX mutations. Our findings revealed a significantly elevated level of microsatellite instability in individuals with ATRX mutants, along with significant alterations in the receptor-tyrosine kinase (RTK)-ras pathway among patients exhibiting combined ATRX mutations. TCGA-GBM patients with concurrent ATRX mutations exhibited sensitivity to 26 chemotherapeutic and anticancer drugs, which exerted their effects by modulating the DDR of tumor cells through highly correlated mechanisms involving the RTK-ras pathway. Additionally, we observed an enrichment of ATRX mutations in specific pathways associated with DDR among TCGA-GBM patients. Our model also demonstrated prolonged overall survival in patients carrying ATRX mutations, particularly showing strong predictive value for 3- and 5-year survival rates. Furthermore, additional protective factors such as younger age, female gender, combined IDH mutations, and TP53 mutations were identified. The results underscore the protective role and prognostic significance of ATRX mutations in GBM as a potential therapeutic target and biomarker for patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89334fb95e9947311852d880d8c780526377835e" target='_blank'>
              ATRX mutation modifies the DNA damage response in glioblastoma multiforme tumor cells and enhances patient prognosis
              </a>
            </td>
          <td>
            Rou Yu, Keru Huang, Xinyan He, Jingwen Zhang, Yushan Ma, Hui Liu
          </td>
          <td>2025-01-10</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceda101cbd27e107d725418d6adeff717071b560" target='_blank'>
              RTEL1 is upregulated in gastric cancer and promotes tumor growth
              </a>
            </td>
          <td>
            Chunyu Yang, Suzeng Wang, Ge Gao, Peiwen Xu, Mengyuan Qian, Yuan Yin, Surui Yao, Zhaohui Huang, Zehua Bian
          </td>
          <td>2024-12-26</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Methyltransferase-like 3 (METTL3) is a primary RNA methyltransferase that catalyzes N6-methyladenosine (m6A) modification. The current study aims to further delineate the effect and mechanism of METTL3 in hepatocellular carcinoma (HCC). By using a murine model of hepatocellular cancer development induced via hydrodynamic tail vein injection, we showed that METTL3 enhanced HCC development. In cultured human HCC cell lines (Huh7 and PLC/PRF/5), we observed that stable knockdown of METTL3 by shRNA significantly decreased tumor cell proliferation, colony formation and invasion, in vitro. When Huh7 and PLC/PRF/5 cells with shRNA knockdown of METTL3 were inoculated into the livers of SCID mice, we found that METTL3 knockdown significantly inhibited the growth of HCC xenograft tumors. These findings establish METTL3 as an important oncogene in HCC. Through N6-methyladenosine-sequencing (m6A-Seq), RNA sequencing (RNA-Seq) and subsequent validation studies, we identified BMI1 and RNF2, two key components of the polycomb repressive complex 1 (PRC1), as direct downstream targets of METTL3-mediated m6A modification in HCC cells. Our data indicated that METTL3 catalyzed m6A modification of BMI1 and RNF2 mRNAs which led to increased mRNA stability via the m6A reader proteins IGF2BP1/2/3. Furthermore, we showed that the METTL3 inhibitor, STM2457, significantly inhibited HCC cell growth in vitro and in mice. Collectively, this study provides novel evidence that METTL3 promotes HCC development and progression through m6A modification of BMI1 and RNF2. Our findings suggest that the METTL3-m6A-BMI1/RNF2 signaling axis may represent a new therapeutic target for the treatment of HCC. Implications: The METTL3-m6A-BMI1/RNF2 signaling axis promotes HCC development and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2828db6fb6b4ab3788a6525cfb5da2199636a58" target='_blank'>
              METTL3-mediated m6A modification regulates the polycomb repressive complex 1 (PRC1) components BMI1 and RNF2 in hepatocellular carcinoma cells.
              </a>
            </td>
          <td>
            Tong Wu, Weina Chen, Jinqiang Zhang, Wenbo Ma, Nianli Liu
          </td>
          <td>2024-12-03</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5605a78d444cbbefd6f70041fabf52971a9499b" target='_blank'>
              Transcriptional regulation of KCNA2 coding Kv1.2 by EWS::FLI1: involvement in controlling the YAP/Hippo signalling pathway and cell proliferation
              </a>
            </td>
          <td>
            Maryne Dupuy, Anaïs Postec, Mathilde Mullard, A. Chantôme, Philippe Hulin, Régis Brion, Maxime Guéguinou, Laura Regnier, M. Potier-Cartereau, Bénédicte Brounais-Le Royer, M. Baud’huin, Steven Georges, François Lamoureux, B. Ory, Françoise Rédini, C. Vandier, Franck Verrecchia
          </td>
          <td>2024-12-18</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="DNA methylation is an essential epigenetic modification that plays a crucial role in regulating gene expression and maintaining genomic stability. With the advancement in sequencing technology, methylation studies have provided valuable insights into the diagnosis of rare diseases through the various identification of episignatures, epivariation, epioutliers, and allele-specific methylation. However, current methylation studies are not without limitations. This mini-review explores the current understanding of DNA methylation in rare diseases, highlighting the key mechanisms and diagnostic potential, and emphasizing the need for advanced methodologies and integrative approaches to enhance the understanding of disease progression and design more personable treatment for patients, given the nature of rare diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30b0846fe63347313e7e0457ade21ff242a0edeb" target='_blank'>
              Expanding Upon Genomics in Rare Diseases: Epigenomic Insights
              </a>
            </td>
          <td>
            Jia W Tan, Emily J. Blake, Joseph D. Farris, Eric W. Klee
          </td>
          <td>2024-12-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Prostate cancer (PC) progresses from benign epithelium through pre-malignant lesions, localized tumors, metastatic dissemination, and castration-resistant stages, with some cases exhibiting phenotype plasticity under therapeutic pressure. However, high-resolution insights into how cell phenotypes evolve across successive stages of PC remain limited. Here, we present the Prostate Cancer Cell Atlas (PCCAT) by integrating ∼710,000 single cells from 197 human samples covering a spectrum of tumor stages. This comprehensive analysis dissects the cellular landscape and characterizes key cell types and molecular features that associate with PC progression and prognosis. In malignant cells, we highlight a distinctive profile denoted by high Major Histocompatibility Complex (MHC) expression, low Androgen Receptor (AR) activity, and enhanced stemness programs associated with enzalutamide resistance. Moreover, we reveal several cell states strongly correlated with PC progression and adverse prognosis, including lineage plasticity-like malignant cells (LPCs), neuroendocrine tumor cells, pericytes, and matrix cancer-associated fibroblasts (mCAFs). Furthermore, we uncover shared cell states that underpin the immune suppressive tumor microenvironment in advanced PC, including activated regulatory T cells, exhausted CD8+ T cells, and SPP1-expressing macrophages. Lastly, we pinpoint a spatial niche composed of mCAFs and SPP1-expressing macrophages localized near the tumor boundary in aggressive PC, which correlates with poor prognosis. Overall, our work provides a valuable resource and offers deeper insights into the diverse cell states, dynamics, and functional characteristics involved in PC progression at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b731bdee7235cc5667d4336486935b5dd643fd" target='_blank'>
              Deciphering single-cell heterogeneity and cellular ecosystem dynamics during prostate cancer progression
              </a>
            </td>
          <td>
            Faming Zhao, Jianming Zeng, Canping Chen, Xiaofan Zhao, Tingting Zhang, George V. Thomas, Rosalie C. Sears, J. Alumkal, Amy E Moran, G. Mills, Peter S Nelson, Zheng Xia
          </td>
          <td>2024-12-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Breast cancer is one of the most common cancers in women and poses a serious threat to women's health. Circular RNAs (circRNAs) have been found to be specifically expressed in cancers and regulate the growth and death of tumor cells. The role of circRNAs in breast cancer remain unknown. In this study, we explored the impacts of circRNAs on the progression of breast cancer cells.Using bioinformatics analysis, we screened out one up-regulated circRNA in breast cancer, and its function and regulatory mechanisms were confirmed by quantitative real-time PCR, cell counting kit-8 experiment, migration assay, dual luciferase reporter assay, Kyoto Encyclopedia of Genes and Genomes enrichment analysis, cell immunofluorescence, clone formation assay, scratch wound healing experiment, RNA immunoprecipitation and subcutaneous tumor-bearing experiments.Circ_0022382 was highly expressed in breast cancer cell lines MDA-MB-231, MCF-7 as well as breast cancer tissues, and promoted the proliferative and migratory capacity of breast cancer cells. In terms of regulatory mechanisms, circ_0022382 activated PI3K/AKT/mTOR signaling pathway and SLC7A11 by sponging let-7a-5p, while knockdown of circ_0022382 contributed to the occurrence of disulfidptosis. In addition, EIF4A3 promoted the expression of circ_0022382 in MDA-MB-231 and MCF-7. Consistently, knockdown of circ_0022382 inhibited the growth of breast cancer cells in vivo.Circ_0022382 and its related molecules may be effective targets for diagnosis or targeted therapy of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2884bbbc76cab2394ad82e1d267d901c415df5c0" target='_blank'>
              EIF4A3-induced circ_0022382 promotes breast cancer cell progression through the let-7a-5p/PI3K/AKT/mTOR signaling pathway and SLC7A11 axis
              </a>
            </td>
          <td>
            Wei Liu, Jun Zhang, Jiawen Zhang, Yu Ye, Jianqin Zhu, Qiwen Yu, Tao Li, Xiaochun Sun, Huabiao Chen
          </td>
          <td>2025-01-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7746fcc03c2930c5c3eb9fafbd71a061c1f28654" target='_blank'>
              KDM1A epigenetically enhances RAD51 expression to suppress the STING-associated anti-tumor immunity in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Qingyuan Yang, Shiyin Wei, Cen Qiu, Chenjie Han, Zunguo Du, Ning Wu
          </td>
          <td>2024-12-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="T-cell acute lymphoblastic leukaemia (T-ALL) is a rare aggressive haematological malignancy characterised by the clonal expansion of immature T-cell precursors. It accounts for 15% of paediatric and 25% of adult ALL. T-ALL is associated with the overexpression of major transcription factors (TLX1/3, TAL1, HOXA) that drive specific transcriptional programmes and constitute the molecular classifying subgroups of T-ALL. Although the dysregulation of transcription factor oncogenes is frequently associated with chromosomal translocations in T-ALL, epigenetic dysregulation resulting in changes to post-translational modifications of histones has also been reported. This includes non-coding intergenic mutations that form oncogenic neo-enhancers. This review will focus on the known epigenetically activating intergenic mutations reported in T-ALL, and will discuss the wider implications of neo-enhancer mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12f059393054a743b5ecf3a79dba4386f2b175eb" target='_blank'>
              Neo-enhancers in T-cell acute lymphoblastic Leukaemia (T-ALL) and beyond.
              </a>
            </td>
          <td>
            Charlotte Smith, V. Asnafi, A. Touzart
          </td>
          <td>2025-01-03</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Juxtaglomerular cell tumor (JxGCT) is a rare type of renal neoplasm demonstrating morphologic overlap with some mesenchymal tumors such as glomus tumor (GT) and solitary fibrous tumor (SFT). Its oncogenic drivers remain elusive, and only a few cases have been analyzed with modern molecular techniques. In prior studies, loss of chromosomes 9 and 11 appeared to be recurrent. Recently, whole-genome analysis identified alterations involving genes of MAPK-RAS pathway in a subset, but no major pathogenic alterations have been discovered in prior whole transcriptome analyses. Considering the limited understanding of the molecular features of JxGCTs, we sought to assess a collaborative series with a multiomic approach to further define the molecular characteristics of this entity. Fifteen tumors morphologically compatible with JxGCTs were evaluated using immunohistochemistry for renin, single-nucleotide polymorphism array (SNP), low-pass whole-genome sequencing, and RNA sequencing (fusion assay). In addition, methylation analysis comparing JxGCT, GT, and SFT was performed. All cases tested with renin (n=11) showed positive staining. Multiple chromosomal abnormalities were identified in all cases analyzed (n=8), with gains of chromosomes 1p, 10, 17, and 19 and losses of chromosomes 9, 11, and 21 being recurrent. A pathogenic HRAS mutation was identified in one case as part of the SNP array analysis. Thirteen tumors were analyzed by RNA sequencing, with 2 revealing in-frame gene fusions: TFG::GPR128 (interpreted as stochastic) and NAB2::STAT6. The latter, originally diagnosed as JxGCT, was reclassified as SFT and excluded from the series. No fusions were detected in the remaining 11 cases; of note, no case harbored NOTCH fusions previously described in GT. Genomic methylation analysis showed that JxGCT, GT, and SFT form separate clusters, confirming that JxGCT represents a distinct entity (ie, different from GT). The results of our study show that JxGCTs are a distinct tumor type with a recurrent pattern of chromosomal imbalances that may play a role in oncogenesis, with MAPK-RAS pathway activation being likely a driver in a relatively small subset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c488efbf43622bbc2ea5641f954b2249e30e2219" target='_blank'>
              Renal Juxtaglomerular Cell Tumors Exhibit Distinct Genomic and Epigenomic Features and Lack Recurrent Gene Fusions: Comprehensive Molecular Analysis of a Multi-institutional Series.
              </a>
            </td>
          <td>
            Květoslava Michalová, Petr Martínek, Roman Mezencev, Sounak Gupta, Sean R. Williamson, M. Wasco, S. Mohanty, C. Magi-Galluzzi, Sofia Cañete-Portillo, M. Aron, Shivani Kandukuri, João Lobo, G. Barkan, Irem Kilic, Andrea Straková-Peteříková, K. Pivovarcikova, Michael Michal, Michael Michal, T. Ulbright, Andres M Acosta
          </td>
          <td>2024-12-27</td>
          <td>The American journal of surgical pathology</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer deaths in the United States. New targeted therapies against the once-deemed undruggable oncogenic KRAS are changing current therapeutic paradigms. However, resistance to targeted KRAS inhibitors almost inevitably occurs; resistance can be driven by tumor cell–intrinsic changes or by changes in the microenvironment. Here, we utilized a genetically engineered mouse model of KRASG12D-driven lung cancer that allows for inducible and reversible expression of the oncogene: activation of oncogenic KRASG12D induces tumor growth; conversely, inactivation of KRASG12D causes tumor regression. We showed that in addition to regulating cancer cell growth and survival, oncogenic KRAS regulated the transcriptional status of cancer-associated fibroblasts and macrophages in this model. Utilizing ex vivo approaches, we showed that secreted factors from cancer cells induced the expression of multiple cytokines in lung fibroblasts, and in turn drove expression of immunosuppressive factors, such as arginase 1, in macrophages. In summary, fibroblasts emerged as a key source of immune regulatory signals, and a potential therapeutic target for improving the efficacy of KRAS inhibitors in lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdae42d5132ec7ce05cdb9d5dd6bd5ef386a7a36" target='_blank'>
              KRASG12D drives immunosuppression in lung adenocarcinoma through paracrine signaling
              </a>
            </td>
          <td>
            Emily L. Lasse-Opsahl, Ivana Barravecchia, Elyse McLintock, Jen-Mei Lee, Sarah F Ferris, Carlos E Espinoza, Rachael Hinshaw, Sophia Cavanaugh, Marzia Robotti, Lily Rober, Kristee L Brown, Kristena Y. Abdelmalak, C. Galbán, Timothy L. Frankel, Yaqing Zhang, M. Pasca di Magliano, Stefanie Galbán
          </td>
          <td>2025-01-09</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Abstract We developed Del-read, an algorithm targeting medium-sized deletions (6–100 bp) in short-reads, which are challenging for current variant callers relying on alignment. Our focus was on Micro-Homolog mediated End Joining deletions (MMEJ-dels), prevalent in myeloid malignancies. MMEJ-dels follow a distinct pattern, occurring between two homologies, allowing us to generate a comprehensive list of MMEJ-dels in the exome. Using Del-read, we identified numerous novel germline and somatic MMEJ-dels in BEAT-AML and TCGA-breast datasets. Validation in 672 healthy individuals confirmed their presence. These novel MMEJ-dels were linked to genomic features associated with replication stress, like G-quadruplexes and minisatellite. Additionally, we observed a new category of MMEJ-dels with an imperfect-match at the flanking sequences of the homologies, suggesting a mechanism involving mispairing in homology alignment. We demonstrated robustness of the repair system despite CRISPR/Cas9-induced mismatches in the homologies. Further analysis of the canonical ASXL1 deletion revealed a diverse array of these imperfect-matches. This suggests a potentially more flexible and error-prone MMEJ repair system than previously understood. Our findings highlight Del-read's potential in uncovering previously undetected deletions and deepen our understanding of repair mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece74b68770d9549840bdc9ed51b1bf59c99a07" target='_blank'>
              Utilizing insights of DNA repair machinery to discover MMEJ deletions and novel mechanisms
              </a>
            </td>
          <td>
            Aditee Kadam, Shay Shilo, Hadas Naor, Alexander Wainstein, Yardena Brilon, T. Feldman, Mark D. Minden, N. Kaushansky, N. Chapal-Ilani, L. Shlush
          </td>
          <td>2024-11-28</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="

Soft tissue and bone cancers, collectively known as sarcomas, constitute a diverse array of uncommon tumors originating from connective tissues. Among sarcomas, leiomyosarcoma (LMS) is one of the most frequently encountered subtypes. This study aims to investigate the expression, clinical significance, biological regulation, and dysregulation mechanisms of extra spindle pole bodies like 1 (ESPL1), a gene critical for cell cycle regulation in LMS.



Bioinformatics analysis was performed using the data from The Cancer Genome Atlas-Sarcoma and Genotype-Tissue Expression datasets. Functional experiments to assess cell proliferation and the cell cycle were performed in LMS cells (SK-LMS-1) after ESPL1 knockdown. Bioinformatics analyses were conducted to identify the potential transcriptional regulators of ESPL1. The regulatory relationship between ESPL1 and the E2F transcription factor 1 (E2F1) was validated through the various molecular assays.



ESPL1 is significantly overexpressed in LMS compared with normal muscle tissue. High ESPL1 expression is associated with a shorter progression-free interval (PFI) in sarcoma patients, particularly in the LMS subset. ESPL1 expression might be an independent prognostic factor for poor overall survival and PFI in LMS patients. Functional studies in the LMS cell line SK-LMS-1 demonstrated that ESPL1 knockdown slowed cell proliferation and increased G2/M cell cycle arrest, suggesting its crucial role in maintaining LMS cell viability and genomic integrity. Further bioinformatics analysis identified the E2F1 transcription factor as a key regulator of ESPL1 expression in LMS. Mechanistic investigations demonstrated that E2F1 interacts with the ESPL1 promoter, leading to transcriptional activation.



These findings highlight the ESPL1-E2F1 axis as a potential prognostic biomarker and therapeutic target in LMS.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d850a39b6a0de260cbb41a095ba4d698be8f6a7" target='_blank'>
              E2F1-mediated ESPL1 transcriptional activation predicts poor prognosis and promotes the proliferation of leiomyosarcoma
              </a>
            </td>
          <td>
            Xiaojuan Yang, Guihua Miao, Qin Wang, Qin Yu, Qinsheng Hu, Gang Tan
          </td>
          <td>2025-01-08</td>
          <td>Cytojournal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cytosine (DNA) methylation plays important roles in silencing transposable elements, plant development, genomic imprinting, stress responses, and maintenance of genome stability. To better understand the functions of this epigenetic modification, several tools have been developed to manipulate DNA methylation levels. These include mutants of DNA methylation writers and readers, targeted manipulation of locus-specific methylation, and the use of chemical inhibitors. Here, we summarize the effects of commonly used cytidine analog chemical inhibitors represented by zebularine, 5-azacytidine, and their related compounds on plants. These analogs are incorporated into the chromosomal DNA, where they block the activity of the replicative CG DNA methyltransferase 1 (MET1). This leads to manifold alterations in plant epigenome, modified developmental programs, or suppression of hybridization barriers. We also highlight the DNA-damaging effects of cytidine analogs, particularly the formation of stable DNA-protein crosslinks between DNA and MET1. This sheds new light on specific phenotypes observed upon cytidine analog treatments. In conclusion, cytidine analogs remain a vital tool for plant genome research and have the potential to open new promising avenues for applications in plant biotechnology and breeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/347eb5947bdb94f55760bce2a406c75b6df28aaf" target='_blank'>
              Cytidine analogs in plant epigenetic research and beyond.
              </a>
            </td>
          <td>
            Eva Dvořák Tomaštíková, A. Pečinka
          </td>
          <td>2024-12-28</td>
          <td>Journal of experimental botany</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background: Epigenetic dysregulation is a common feature of cancer. Promoter demethylation of tumor-promoting genes and global DNA hypomethylation may trigger tumor progression. Epigenetic changes are unstable; thus, research has focused on detecting remedies that target epigenetic regulators. Previous studies have suggested that concordantly targeting hypomethylation and hypermethylation is beneficial for suppressing both the oncogenic and pro-metastatic functions of cancer cells. Therefore, we aimed to investigate the effect of a combination of S-adenosylmethionine (SAM) and the demethylating agent decitabine on prostate cancer cells. Materials and Methods: Prostate cancer cells (PC-3) were treated with SAM, decitabine, or a combination of both. Proliferation, migration, invasion, and methylation assays were also performed. A transcriptome study was conducted to detect different gene clusters between the treatment groups, followed by analyses using the Kyoto Encyclopedia of Genes and Genomes pathway and ingenuity pathway analysis. Finally, to gain information on differential gene expression, promoter methylation studies were performed. Results: Groups treated with decitabine, SAM, or their combination showed reduced proliferative capacity. The decitabine-treated group showed a marginal increase in cell migration and invasion, whereas the SAM-treated and combination treatment groups showed reduced invasion and migration potential. Methylation assays demonstrated the restoration of decitabine-induced demethylation in prostate cancer samples, whereas the transcriptome study revealed the upregulation of different gene clusters between the treatment groups. Methylation studies confirmed that SAM could restore the decitabine-induced demethylation of proto-oncogenes, but it did not induce the re-methylation of tumor-suppressor genes. Conclusions: Combination treatment with SAM and decitabine had an additive effect and did not nullify each other.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60ece44a8b2382c13d866bb1b93e63772b3f622f" target='_blank'>
              The Simultaneous Treatment of PC-3 Cells with the DNA-Demethylating Agent Decitabine and S-Adenosylmethionine Leads to Synergistic Anticancer Effects
              </a>
            </td>
          <td>
            Thomas Schmidt, Carsten Sticht
          </td>
          <td>2024-12-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edb16603152832a7b5a7238cbeef6e092d7b2df9" target='_blank'>
              PRMT5-regulated splicing of DNA repair genes drives chemoresistance in breast cancer stem cells.
              </a>
            </td>
          <td>
            Matthew S Gillespie, Kelly Chiang, Gemma L Regan-Mochrie, Soo-Youn Choi, Ciara Ward, Debashish Sahay, Paloma Garcia, Roland Arnold, Clare C Davies
          </td>
          <td>2024-12-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1708204ce5d02d098867433430233edb7d10ec3" target='_blank'>
              Profiling the epigenome using long-read sequencing.
              </a>
            </td>
          <td>
            Tianyuan Liu, Ana Conesa
          </td>
          <td>2025-01-08</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/481e81722a6ce523752049f844fedd532d84c573" target='_blank'>
              Transcriptomic landscape of Hras12V oncogene-induced hepatocarcinogenesis with gender disparity
              </a>
            </td>
          <td>
            Huaiyuan Di, Zhuona Rong, Nan Mao, Huiling Li, Jun Chen, Renwu Liu, Aiguo Wang
          </td>
          <td>2025-01-16</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Motivation Allele-specific methylation (ASM) refers to differential DNA methylation patterns between two alleles at a given locus. This phenomenon is often driven by genetic variants, such as single nucleotide variants (SNVs), which influence methylation in cis by affecting transcription factor or methylation regulator binding, leading to allele-specific differences. Understanding ASM is critical for elucidating gene regulation, as it impacts gene expression and contributes to normal biological variation as well as disease processes, including cancer [1] and autoimmune disorders [2],[3]. Another key driver of ASM is genomic imprinting, an epigenetic mechanism in which gene expression is regulated in a parent-of-origin-specific manner. Imprinted regions, marked during gametogenesis and maintained through cell divisions, are essential for growth, development, and metabolism. Dysregulation of imprinting is associated with developmental and metabolic disorders, such as Prader-Willi and Angelman syndromes, and certain cancers. Detecting ASM across the genome remains challenging due to its tissue- and cell-specific nature and the technical difficulty of phasing reads to assign methylation patterns to specific alleles. Current ASM detection pipelines (e.g., [4])) often require phasing via genetic variants, a computationally intensive process that is limited in regions with low heterozygosity. Results To address these limitations, we developed asms (Allele-Specific Methylation Scanner), a tool designed to detect ASM directly from methylation data without the need for prior phasing. asms offers a faster and complementary approach to uncovering the regulatory effects of ASM, particularly in genomic regions where genetic variants or imprinting play a critical role. For demonstration purposes we leverage the fact that reads generated by Oxford Nanopore (ONT) technology measure sequence and methylation status at once, but the same software can be used with other sequencing technologies. asms can check thousands of loci in a short time. The initial list of loci to examine can be given by the user, or generated by asms through a genomic scan. The asms cluster subcommand separates the reads based on methylation. If phasing results are available, asms can use them to verify whether distinct alleles correspond to distinct base modifications patterns. We benchmark our software using publicly available Ashkenazi trio data [5]. Implementation and availability asms is implemented in rust and python. The software is available at https://github.com/ecmra/asms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c7dfc00aae1ea08fcc56a90d449bd2018337af0" target='_blank'>
              ASMS: finding allele specific methylation in human genomes without phasing
              </a>
            </td>
          <td>
            E. Raineri, Miguel Ángel Esteve Marco, Anna Esteve Codina
          </td>
          <td>2024-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="DHX9 is a multifunctional DExH-box RNA helicase with important roles in the regulation of transcription, translation, and maintenance of genome stability. Elevated expression of DHX9 is evident in multiple cancer types, including colorectal cancer (CRC). Microsatellite instable-high (MSI-H) tumors with deficient mismatch repair (dMMR) display a strong dependence on DHX9, making this helicase an attractive target for oncology drug discovery. In this report, we show that DHX9 knockdown increased RNA/DNA secondary structures and replication stress, resulting in cell cycle arrest and the onset of apoptosis in cancer cells with MSI-H/dMMR. ATX968 was identified as a potent and selective inhibitor of DHX9 helicase activity. Chemical inhibition of DHX9 enzymatic activity elicited similar selective effects on cell proliferation as seen with genetic knockdown. In addition, ATX968 induced robust and durable responses in an MSI-H/dMMR xenograft model but not in a microsatellite stable (MSS)/proficient mismatch repair (pMMR) model. These preclinical data validate DHX9 as a target for the treatment of patients with MSI-H/dMMR. Additionally, this potent and selective inhibitor of DHX9 provides a valuable tool with which to further explore the effects of inhibition of DHX9 enzymatic activity on the proliferation of cancer cells in vitro and in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ff4f51f0456e38f5d0ec9d35528d65af9bd7b59" target='_blank'>
              A Potent, Selective, Small-Molecule Inhibitor of DHX9 Abrogates Proliferation of Microsatellite Instable Cancers with Deficient Mismatch Repair.
              </a>
            </td>
          <td>
            Jennifer Castro, Matthew H Daniels, David Brennan, Brian Johnston, D. Gotur, Young-Tae Lee, Kevin E Knockenhauer, Chuang Lu, Jie Wu, S. Nayak, Cindy Collins, Rishabh Bansal, Shane M Buker, April Case, Julie Liu, Shihua Yao, Brian A Sparling, E. Sickmier, Serena J Silver, Stephen J Blakemore, P. Boriack-Sjodin, K. Duncan, S. Ribich, Robert A Copeland
          </td>
          <td>2024-11-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Transformative technologies to sequence tumor genomes at large scale and single-cell resolution have exposed the repertoire of genetic alterations that are present in leukemia genomes, the timing of their acquisition and patterns of their co-occurrence. In parallel, single-cell multi-omics technologies are allowing us to map the differentiation paths and hierarchical structures of malignant cells and giving us a glimpse into hematopoietic development in prenatal life. We propose that interrogating how the genetic evolution, differentiation hierarchy and ontogeny of malignant myeloid cells intersect with each other, using new experimental systems and multimodal technologies, will fuel the next generation of research breakthroughs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f504f6d5de23edcc7a44b61924d8a2be55a1c9e" target='_blank'>
              The crossroads of clonal evolution, differentiation hierarchy and ontogeny in leukemia development.
              </a>
            </td>
          <td>
            Christopher M Sturgeon, Elvin Wagenblast, Franco Izzo, E. Papapetrou
          </td>
          <td>2024-12-09</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02fb4ad81cdcfd96f458e312f0500d09b4e4efb2" target='_blank'>
              Computational analysis of MYC gene variants: structural and functional impact of non-synonymous SNPs.
              </a>
            </td>
          <td>
            Plabita Bhuyan, Varshabi Bharali, Sangju Basumatary, Aido Lego, Juman Sarma, Debasish Borbora
          </td>
          <td>2024-12-14</td>
          <td>Journal of applied genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The EML4–ALK oncogene drives tumour progression in approximately 5% of cases of non-small-cell lung cancers. At least 15 EML4–ALK variants have been identified, which elicit differential responses to conventional ALK inhibitors. Unfortunately, most, if not all, patients eventually acquire resistance to these inhibitors and succumb to the disease, which warrants the need for alternative targets to be identified. The most aggressive variant, EML4–ALK variant 3 (V3), assembles into a complex on interphase microtubules together with the NEK9 and NEK7 kinases, which leads to the downstream phosphorylation of NEK7 substrates. Overall, this promotes an elongated cell morphology and an enhanced migratory phenotype, which likely contributes to the increased metastasis often seen in V3 patients. Here, using two separate approaches to displace V3 from microtubules and a variety of in vitro assays, we show that microtubule association of EML4–ALK V3 is required for both V3 phenotypes, as removal of the oncogenic fusion protein from microtubules led to the dissociation of the V3–NEK9–NEK7 complex and the reversal of both phenotypic changes. Overall, we propose that targeting the interaction between EML4–ALK V3 and microtubules might offer a novel therapeutic option, independent of ALK activity, for V3+ NSCLC patients with acquired resistance to ALK inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df2861ca0c2468f59f9063b573d82cf9d569f6d7" target='_blank'>
              Microtubule Association of EML4–ALK V3 Is Key for the Elongated Cell Morphology and Enhanced Migration Observed in V3 Cells
              </a>
            </td>
          <td>
            S. Papageorgiou, Sarah L. Pashley, Laura O’Regan, K. Straatman, Andrew M. Fry
          </td>
          <td>2024-11-25</td>
          <td>Cells</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="HAP1 is a near-haploid human cell line commonly used for mutagenesis and genome editing studies due to its hemizygous nature. We noticed an unusual hypersensitivity of HAP1 to camptothecin, an antineoplastic drug that stabilizes topoisomerase I cleavage complexes (TOP1ccs). We have attributed this hypersensitivity to a deficiency of TDP1, a key phosphodiesterase involved in resolving abortive TOP1ccs. Through whole-exome sequencing and subsequent restoration of TDP1 protein via CRISPR-Cas9 endogenous genome editing, we demonstrate that TDP1 deficiency and camptothecin hypersensitivity in HAP1 cells are a result of a splice-site mutation (TDP1 c.660-1G > A) that causes exon skipping and TDP1 loss of function. The lack of TDP1 in HAP1 cells should be considered when studying topoisomerase-associated DNA lesions and when generalizing mechanisms of DNA damage repair using HAP1 cells. Finally, we also report the generation of HAP1 STAR clones with restored TDP1 expression and function, which may be useful in further studies to probe cellular phenotypes relating to TOP1cc repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7891032e07f96c869d97db1867e4b8a4b2832cd9" target='_blank'>
              TDP1 splice-site mutation causes HAP1 cell hypersensitivity to topoisomerase I inhibition.
              </a>
            </td>
          <td>
            Chen Gang Goh, Aldo S. Bader, Tuan-Anh Tran, R. Belotserkovskaya, Giuseppina D'Alessandro, S. P. Jackson
          </td>
          <td>2024-12-11</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Multiple myeloma (MM), a clonal B-cell neoplasia, is an incurable and heterogeneous disease where survival ranges from a few months to more than 10 years. The clinical heterogeneity of MM arises from multiple genomic events that result in tumour development and progression. Recurring genomic abnormalities including cytogenetic abnormalities, gene mutations and abnormal gene expression profiles in myeloma cells have a strong prognostic power. With the advancement in technologies and the development of novel drugs, the prognostic factors and treatment paradigms of MM have been fast evolving over the past few years. Following the introduction of new highthroughput cytogenomic technologies such as array comparative genome hybridisation (aCGH) or single nucleotide polymorphism array (SNP array) and molecular techniques such as gene expression profiling (GEP) and massively parallel genomic sequencing, the prediction of survival in MM no longer solely depends on conventional cytogenetics and interphase fluorescence in situ hybridisation (iFISH) analysis findings. These new technologies enable screening for all possible chromosomal aberrations and other genomic alterations, identifying each aberration on a case-bycase basis and discovering new aberrations that are relevant in unraveling the tumor cells' complex biology. This in turn allows a better understanding of the disease complexity and heterogeneity. The objective of this review on the genetic architecture of MM is to discuss the latest developments on the cytogenetic/cytogenomic-based risk classification of MM that are currently in use and their prognostic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f51a37b55e510ee5aeb00a67198a8c2f2e860c2a" target='_blank'>
              Genetic architecture of Multiple Myeloma and its prognostic implications - An updated review.
              </a>
            </td>
          <td>
            E. Foong, R. Ankathil
          </td>
          <td>2024-12-01</td>
          <td>The Malaysian journal of pathology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer, the most prevalent cancer in females, is a heterogeneous disease with various molecular subtypes, which presents challenges in diagnosis and treatment. Ubiquitination is one of the most critical post-translational protein modifications, that plays regulatory roles in numerous cellular processes including cell cycle progression, DNA replication & repair, apoptosis, transcription regulation, protein localization, trafficking and signal transduction. This modification can be reversed by deubiquitinases, or DUBs, a superfamily of cysteine proteases and metalloproteases that cleave ubiquitin-protein bonds. Dysregulation of DUBs has been associated to various diseases including cancer, making them promising targets for cancer therapy. We leveraged publicly available breast cancer datasets and employed various bioinformatics tools to identify differentially expressed DUBs in breast cancer. Our analysis identified six genes (COPS5, EIF3H, MINDY1, MINDY2, PSMD14 and USP26) with significant differential expression and survival implications. We further validated our findings experimentally and found upregulation of COPS5, EIF3H and MINDY 1 in MCF-7 and T47D breast cancer cell lines using qPCR analysis. To identify the role of these genes, EIF3H and COPS5, in disease progression, we constructed a protein-protein interaction (PPI) network with genes associated with metastasis and explored their correlation at the gene expression level in breast cancer patients. Together, this comprehensive study sheds light on DUB gene expression patterns in breast cancer with the potential to identify novel targets for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1182059f20169c0a82d0630262bfb5d46b482ee5" target='_blank'>
              Deubiquitinating enzymes in breast cancer: in silico analysis of gene expression and metastatic correlation.
              </a>
            </td>
          <td>
            Gaurav Sahoo, Shruti Bandyopadhyay, Ekta Tripathi, Prashanthi Karyala
          </td>
          <td>2024-12-13</td>
          <td>Journal of biomolecular structure & dynamics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id=". The development of cancer involves an intri‑ cate mechanism, including oncogene activation and tumor suppressor gene inactivation, often driven by epigenetic altera‑ tions. Notably, these epigenetic modifications, such as DNA hypermethylation, are dynamic and reversible, and emerging research suggests that dietary factors can influence them. Recent findings have examined the intricate associations between dietary compounds and cancer development, with a particular focus on the mechanisms through which natural components affect epigenetic alterations. Natural compounds have gained significant attention for their potential in chemo‑ prevention and treatment due to their capacity to modulate DNA methylation patterns, particularly at promoter CpG islands. Chemotherapy and radiation therapy are effective cancer treatments and are often associated with severe adverse effects. As a result, there is increasing interest in exploring natural substances as alternative therapeutic options. The present review focuses on natural compounds that can reverse hypermethylation, providing a potentially safer approach for targeted cancer treatment. These bioactive agents, abundant in natural compounds, exhibit the potential to combat cancer by inhibiting metastasis, inducing cell cycle arrest and reversing DNA hypermethylation. The present review aimed to provide further in‑depth mechanistic insight into the mechanisms through which natural compounds, such as genistein, resvera‑ trol, quercetin and capsaicin modulate DNA hypermethylation in various types of cancer, including breast, cervical, prostate and neuroblastoma. By combining in silico , in vitro and in vivo approaches, it uniquely integrates computational and experimental evidence to demonstrate compound‑specific reversal of epigenetic marks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eaa3ae9852c9314b773d0fad7daf6e3a6798eb1" target='_blank'>
              Natural compounds as epigenetic modulators: Reversing DNA hypermethylation in cancer (Review)
              </a>
            </td>
          <td>
            Farheen Farheen, Kantimahanti Shreya, R. Mathur, A. Jha
          </td>
          <td>2024-12-30</td>
          <td>World Academy of Sciences Journal</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="BACKGROUND
The transfer of mitochondrial DNA into the nuclear genomes of eukaryotes (Numts) has been linked to lifespan in non-human species and recently demonstrated to occur in rare instances from one human generation to the next.


METHOD
Here we investigated numtogenesis dynamics in humans in two ways. First, we quantified Numts in 1,187 post-mortem brain and blood samples from different individuals. Second, we tested the dynamic transfer of Numts using a repeated-measures whole genome sequencing design in a human fibroblast model that recapitulates several molecular hallmarks of aging.


RESULT
In the ROSMAP dataset, compared to circulating immune cells (n = 389), post-mitotic brain tissue (n = 798) contained more Numts, consistent with their potential somatic accumulation. Within brain samples we observed a 5.5-fold enrichment of somatic Numt insertions in the dorsolateral prefrontal cortex compared to cerebellum samples, suggesting that brain Numts arose spontaneously during development or across the lifespan. Moreover, more brain Numts was linked to earlier mortality. The brains of individuals with no cognitive impairment who died at younger ages carried approximately 2 more Numts per decade of life lost than those who lived longer. In the lifespan model, the longitudinal experiments revealed a gradual accumulation of one Numt every ∼13 days. Numtogenesis was independent of large-scale genomic instability and unlikely driven cell clonality. Targeted pharmacological perturbations including chronic glucocorticoid signaling or impairing mitochondrial oxidative phosphorylation (OxPhos) only modestly increased the rate of numtogenesis, whereas patient-derived SURF1-mutant cells exhibiting mtDNA instability accumulated Numts 4.7-fold faster than healthy donors.


CONCLUSION
Combined, our data document spontaneous numtogenesis in human cells and demonstrate an association between brain cortical somatic Numts and human lifespan. These findings open the possibility that mito-nuclear horizontal gene transfer among human post-mitotic tissues produces functionally-relevant human Numts over timescales shorter than previously assumed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/576cc9de30e3b7f80b5c6a3ef93ab242e8481166" target='_blank'>
              Basic Science and Pathogenesis.
              </a>
            </td>
          <td>
            Weichen Zhou, Kalpita R. Karan, Wenjin Gu, Hans Klein, Gabriel Sturm, P. L. De Jager, David A. Bennett, M. Hirano, Martin Picard, Ryan E Mills
          </td>
          <td>2024-12-01</td>
          <td>Alzheimer's & dementia : the journal of the Alzheimer's Association</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes led to a better understanding of tumorigenesis, and prompted the development of molecularly targeted therapy. Over the past 30 years, many new drugs, which are primarily aimed at activated oncogenic proteins in signal transduction pathways involved in cell proliferation and survival, have been introduced in the clinic. Despite its rational design, the overall efficacy of targeted therapy has been modest. Recently, the noncoding RNAs (ncRNAs) have emerged as key regulators of important cellular processes in addition to the known regulatory proteins. It now appears that dual epigenetic regulatory systems exist in higher eukaryotic cells: a ncRNA network that governs essential cell functions, like cell fate decision and maintenance of homeostasis, and a protein-based system that presides over core physiological processes, like cell division and genomic maintenance. Modifications of the ncRNA network due to altered ncRNAs can cause the cell to shift towards to neoplastic phenotype; this is cancer initiation. Mutations in the well-known cancer driver genes provide the incipient cancer cell with a selective growth advantage and fuel its consequent clonal expansion. Because of the crucial role of the altered ncRNAs in tumorigenesis, targeting them may be a reasonable therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4bbdeb17c896767d39cab33f2cc2532e16bb98" target='_blank'>
              Implications of noncoding RNAs for cancer therapy: Are we aiming at the right targets?
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-01-16</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ASXL transcriptional regulator 1 (ASXL1) is one of the three most frequently mutated genes in age-related clonal hematopoiesis (CH), alongside DNA methyltransferase 3 alpha (DNMT3A) and Tet methylcytosine dioxygenase 2 (TET2). CH can progress to myeloid malignancies including chronic monomyelocytic leukemia (CMML) and is also strongly associated with inflammatory cardiovascular disease and all-cause mortality in humans. DNMT3A and TET2 regulate DNA methylation and demethylation pathways, respectively, and loss-of-function mutations in these genes reduce DNA methylation in heterochromatin, allowing derepression of silenced elements in heterochromatin. In contrast, the mechanisms that connect mutant ASXL1 and CH are not yet fully understood. CH/CMML-associated ASXL1 mutations encode C-terminally truncated proteins that enhance the deubiquitinase activity of the ASXL-BAP1 "PR-DUB" deubiquitinase complex, which removes monoubiquitin from H2AK119Ub. Here, we show that ASXL1 mutant proteins interact with the euchromatic histone lysine methyltransferases 1 and 2 (EHMT1-EHMT2) complex, which generates H3K9me1 and me2, the latter a repressive modification in constitutive heterochromatin. Compared to cells from age-matched wild-type mice, we found that expanded myeloid cells from old (≥18-mo-old) Asxl1tm/+ mice, a heterozygous knock-in mouse model of CH, display genome-wide decreases of H3K9me2, H3K9me3, and H2AK119Ub as well as an associated increase in expression of transposable elements (TEs) and satellite repeats. Increased TE expression was also observed in monocytes from ASXL1-mutant CMML patients compared to monocytes from healthy controls. Our data suggest that mutant ASXL1 proteins compromise the integrity of both constitutive and facultative heterochromatin in an age-dependent manner by reducing the levels of H3K9me2/3 and H2AK119Ub. This increase in TE expression correlated with increased expression of nearby genes, including many interferon-inducible (inflammation-associated) genes (ISGs).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/112e7123fc0bf81467e6f3de419413862ec10652" target='_blank'>
              A mutant ASXL1-BAP1-EHMT complex contributes to heterochromatin dysfunction in clonal hematopoiesis and chronic monomyelocytic leukemia.
              </a>
            </td>
          <td>
            Zhen Dong, Hugo Sepulveda, Leo J. Arteaga-Vazquez, Chad Blouin, Jenna A. Fernandez, Moritz Binder, Wen-Chien Chou, H. Tien, M. Patnaik, Geoffrey J Faulkner, Samuel A Myers, Anjana Rao
          </td>
          <td>2025-01-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3c872fe0219ee9d415c23e2abaa5474c89585b2" target='_blank'>
              KIF18A inhibition: the next big player in the search for cancer therapeutics.
              </a>
            </td>
          <td>
            Ain Syafiza Mohd Amin, Sarah Eastwood, Courtney Pilcher, Jia Q Truong, Richard Foitzik, Joanne Boag, Kylie L Gorringe, Jessica K Holien
          </td>
          <td>2024-11-24</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Occurrence of degenerative interactions is thought to serve as a mechanism underlying hybrid unfitness in most animal systems. However, the molecular mechanisms underpinning the genetic interaction and how they contribute to overall hybrid incompatibilities are limited to only a handful of examples. A vertebrate model organism, Xiphophorus, is used to study hybrid dysfunction, and it has been shown from this model that diseases, such as melanoma, can occur in certain interspecies hybrids. Melanoma development is due to hybrid inheritance of an oncogene, xmrk, and loss of a co-evolved tumor modifier. It was recently found that adgre5, a G protein-coupled receptor involved in cell adhesion, is a tumor regulator gene in naturally hybridizing Xiphophorus species Xiphophorus birchmanni (X. birchmanni) and Xiphophorus malinche (X. malinche). We hypothesized that 1 of the 2 parental alleles of adgre5 is involved in regulation of cell growth, migration, and melanomagenesis. Accordingly, we assessed the function of adgre5 alleles from each parental species of the melanoma-bearing hybrids using in vitro cell growth and migration assays. In addition, we expressed each adgre5 allele with the xmrk oncogene in transgenic medaka. We found that cells transfected with the X. birchmanni adgre5 exhibited decreased growth and migration compared to those with the X. malinche allele. Moreover, X. birchmanni allele of adgre5 completely inhibited melanoma development in xmrk-transgenic medaka, while X. malinche adgre5 expression did not exhibit melanoma suppressive activity in medaka. These findings provide evidence that adgre5 is a natural melanoma suppressor and provide new insight in melanoma etiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b35473f18fdca14702fd87c87ffcf454b6421341" target='_blank'>
              Functional test of a naturally occurred tumor modifier gene provides insights to melanoma development.
              </a>
            </td>
          <td>
            Mateo Garcia-Olazabal, M. Adolfi, B. Wilde, Anita Hufnagel, Rupesh Paudel, Yuan Lu, Svenja Meierjohann, Gil Rosenthal, Manfred Schartl
          </td>
          <td>2025-01-17</td>
          <td>G3</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Renal clear cell carcinoma (ccRCC) presents a unique landscape of genetic and epigenetic modifications, the understanding of which is crucial for the development of targeted therapies and improved prognostication. This study explores the differential expression of histone‐related genes (HRGs) in ccRCC and correlates these findings with patient clinical outcomes. By leveraging the Cancer Genome Atlas (TCGA) data, we performed a robust multidimensional analysis of HRG expression profiles in ccRCC versus adjacent normal tissues. Our results demonstrate a significant upregulation of several key epigenetic regulators, including UHRF1, KDM5A, EZH2, PRDM6, and TWIST1, in tumor samples, with statistical significance suggesting their involvement in tumorigenesis and progression. Paired expression analysis within patient‐matched samples confirmed the consistency of overexpression in tumors. The prognostic relevance of these genes was underscored through survival analyses, which revealed a clear stratification of patients into distinct risk categories based on their expression profiles. The integration of these genetic markers with clinical parameters facilitated the development of a predictive nomogram, yielding a quantifiable tool for survival prediction. Our comprehensive analysis elucidates the profound impact of epigenetic dysregulation in ccRCC and proposes a novel set of biomarkers for disease diagnosis, prognostic stratification, and potential therapeutic targeting, marking a significant stride toward precision oncology in renal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/057b9174164151a3b1cfe3112a48500f0dca6ee3" target='_blank'>
              Epigenetic Alterations in Renal Clear Cell Carcinoma: A Comprehensive Expression and Clinical Outcome Analysis
              </a>
            </td>
          <td>
            Dafei Weng, Zhenqian Qin, Xuefeng Yuan, Xuping Jiang, Hua Rui
          </td>
          <td>2025-01-01</td>
          <td>Journal of Clinical Pharmacy and Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The memory of cellular identity is crucial for the correct development of an individual and is maintained throughout life by the epigenome. Chromatin marks, such as DNA methylation and histone modifications, ensure the stability of gene expression programmes over time and through cell division. Loss of these marks can lead to severe pathologies, including cancer and developmental syndromes. However, reprogramming of cellular identity is also a natural phenomenon that occurs early in mammalian development, particularly in the germ line, which enables the production of mature and functional gametes. The germ line transmits genetic and epigenetic information to the next generation, contributing to the survival of the species. Primordial germ cells (PGCs) undergo extensive chromatin remodelling, including global DNA demethylation and erasure of the parental imprints. This review introduces the concept of epigenetic reprogramming, its discovery and key steps, as well as the transcriptional and chromatin changes that accompany germ cell formation in mice. Finally, we discuss the epigenetic mechanisms of genomic imprinting, its discovery, regulation and relevance to human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2611fcc1b3dfdfc6647da0d47d718b5970da542" target='_blank'>
              [Epigenetic reprogramming, germline and genomic imprinting].
              </a>
            </td>
          <td>
            Clara Roidor, Karim Chebli, Maud Borensztein
          </td>
          <td>2024-12-01</td>
          <td>Medecine sciences : M/S</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/452c65f4f9faa535e4f661db53c177e8e8787cfc" target='_blank'>
              KANK1 promotes breast cancer development by compromising Scribble-mediated Hippo activation
              </a>
            </td>
          <td>
            S. S. Guo, Zhiying Liu, Guan M. Wang, Zhiqi Sun, Kaikai Yu, James P Fawcett, Reinhard Buettner, Bo Gao, Reinhard Fässler
          </td>
          <td>2024-11-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genomic integrity is critical for cellular homeostasis, preventing the accumulation of mutations that can drive diseases such as cancer. Among the mechanisms safeguarding genomic stability, the Base Excision Repair (BER) pathway plays a pivotal role in counteracting oxidative DNA damage caused by reactive oxygen species. Central to this pathway are enzymes like 8-oxoguanine glycosylase 1 (OGG1), which recognize and excise 8-oxo-7,8-dihydro-2′-deoxyguanosine (8-oxodG) lesions, thereby initiating a series of repair processes that restore DNA integrity. BER inhibitors have recently been identified as a promising approach in cancer therapy, increasing the sensitivity of cancer cells to radiotherapy and chemotherapy. By exploiting tumor-specific DNA repair dependencies and synthetic lethal interactions, these inhibitors could be used to selectively target cancer cells while sparing normal cells. This review provides a robust reference for scientific researchers, offering an updated perspective on small-molecule inhibitors targeting the 8-oxodG-BER pathway and highlighting their potential role in expanding cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e72df5cdd609ef616fe0cccc3b1963c63863d272" target='_blank'>
              Targeting the 8-oxodG Base Excision Repair Pathway for Cancer Therapy
              </a>
            </td>
          <td>
            Anna Piscone, Francesca Gorini, Susanna Ambrosio, Anna Noviello, Giovanni Scala, Barbara Majello, Stefano Amente
          </td>
          <td>2025-01-14</td>
          <td>Cells</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/937fb72f5c07d87edcc2493e093c8ba719c5c0a0" target='_blank'>
              TET2 cascade: a novel regulator of chromatin structure and leukaemogenesis
              </a>
            </td>
          <td>
            Wolfram C. M. Dempke, K. Fenchel
          </td>
          <td>2025-01-06</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract In mammals, fertilization is followed by extensive reprogramming and reorganization of the chromatin accompanying the transcriptional activation of the embryo. This reprogramming results in blastomeres with the ability to give rise to all cell types and a complete organism, including extra‐embryonic tissues, and is known as totipotency. Transcriptional activation occurs in a process known as zygotic genome activation (ZGA) and is tightly linked to the expression of transposable elements, including endogenous retroviruses (ERVs) such as endogenous retrovirus with leucine tRNA primer (ERVL). Recent studies discovered the importance of ERVs in this process, yet the race to decipher the network surrounding these elements is still ongoing, and the molecular mechanism behind their involvement remains a mystery. Amid a recent surge of studies reporting the discovery of various factors and pathways involved in the regulation of ERVs, this review provides an overview of the knowns and unknowns in the field, with a particular emphasis on the chromatin landscape and how ERVs shape preimplantation development in mammals. In so doing, we highlight recent discoveries that have advanced our understanding of how these elements are involved in transforming the quiescent zygote into the most powerful cell type in mammals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de59ad5f3d58d084f36e3b3323bf2a35634e22da" target='_blank'>
              The impact of retrotransposons on zygotic genome activation and the chromatin landscape of early embryos
              </a>
            </td>
          <td>
            Therese Solberg, Mie Kobayashi-Ishihara, Haruhiko Siomi
          </td>
          <td>2024-11-22</td>
          <td>Annals of the New York Academy of Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f33f540ba07c7d118037674f121aaa35d22c9a7" target='_blank'>
              Prolonged persistence of mutagenic DNA lesions in somatic cells.
              </a>
            </td>
          <td>
            M. Spencer Chapman, E. Mitchell, Kenichi Yoshida, N. Williams, Margarete A. Fabre, A. Ranzoni, Philip S. Robinson, Lori D. Kregar, Matthias Wilk, Steffen Boettcher, Krishnaa Mahbubani, Kourosh Saeb Parsy, K. Gowers, Sam M Janes, Stanley W K Ng, Matt Hoare, Anthony R Green, G. Vassiliou, A. Cvejic, M. Manz, E. Laurenti, I. Martincorena, Michael R Stratton, J. Nangalia, Tim H. H. Coorens, P. J. Campbell
          </td>
          <td>2025-01-15</td>
          <td>Nature</td>
          <td>1</td>
          <td>87</td>
        </tr>

        <tr id="

Cancer is a highly fatal disease that is typified by aberrant cells proliferating out of
control. The properties of acquired cells, including as genomic instability and mutations, cellular
death evasion, and proliferative signaling sustenance, are evidenced by the hallmarks of cancer
and contribute to the establishment of malignant tumors. A protein called poly(ADP-ribose) polymerase-
1 (PARP1) is essential for both cell survival and DNA damage repair, two processes that
affect cellular control. Since homologous recombination deficient cells exhibit cellular death upon
suppression of PARP1, the PARP protein has gained attention as a potential target for anticancer
treatments. The Food and Drug Administration (FDA) has already approved a number of
effective PARP1 inhibitors, including olaparib and niraparib. The final chemical has a 1,3,4-
thiadiazole core in its structure. In fact, heterocyclic moieties have gained attention due to their
numerous medicinal advantages, which include their capacity to combat cancer. The compounds
derived from these substances have been studied as PARP1 inhibitors, with promising results.
Thus, the goal of this study is to go over the importance of PARP1 in cancer as well as its role in
cell regulation. It also attempts to offer a comprehensive assessment of the literature that has been
published in the previous fifteen years about PARP1 inhibitors, including the different scaffolds,
with an emphasis on features of the structure-activity relationship. This will provide vital information
for the creation of fresh, more effective PARP1 inhibitors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0f1054bd8e7e159c150671b8309129cfedf96dc" target='_blank'>
              PARP1 Inhibitors: An Important Part in Cancer Treatment
              </a>
            </td>
          <td>
            B. Gowramma, Subhranil Mandal, Kaviarasan Laksmanan, Silpa R S, Sowmiya Arun, K. Rajagopal, Vinethmartin Jacob Stanley
          </td>
          <td>2024-12-17</td>
          <td>Current Bioactive Compounds</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Pituitary neuroendocrine tumors (PitNETs) are unusual among neoplasms in that sporadic tumors are not typically associated with genetic mutations. Instead, epigenetics, a non-mutational process by which gene expression is modified via a variety of mechanisms, may be a driving factor in PitNET growth and behavior. DNA methylation is one of the most well understood forms of epigenetic modification. Research on DNA methylation profiles of PitNETs has identified a large number of genes silenced or upregulated by DNA methylation, particularly when methylated at CpG islands of gene promoter regions. Global patterns of DNA methylation may provide valuable insight into origins of pituitary adenoma subtypes, assist with PitNet diagnostics and have been found to correspond to the current WHO classification of PitNETs based on transcription factor lineage. Analysis of differentially methylated regions of individual genes may have prognostic value as well as guide research toward nonsurgical therapeutic strategies. Pituitary epigenetics and DNA methylation analysis are rapidly growing areas of interest with the potential to shape the future of pituitary tumor diagnostics and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c91afb11897061f0dd790a1feb3aca845da9bfbb" target='_blank'>
              Current understanding of the role of DNA methylation in pituitary neuroendocrine tumors
              </a>
            </td>
          <td>
            Racheal Peterson, David J Cote, G. Zada
          </td>
          <td>2025-01-02</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DICER1 syndrome predisposes children and young adults to tumor development across various organs. Many of these cancers are sarcomas, which uniquely express the RNase IIIb domain-deficient form of DICER1 and exhibit consistent histological and molecular similarities regardless of their anatomical origins. To uncover their cellular origin and developmental hierarchy, we established a lineage-traceable genetically engineered mouse model that allows for precise activation of Dicer1 mutations in Hic1+ mesenchymal stromal cells. This model resulted in the development of renal tumors closely mirroring human DICER1 sarcoma histologically and molecularly. Single-cell transcriptomics coupled with targeted spatial gene expression analysis revealed a Hic1+ progenitor population marked by Pdgfra, Dpt, and Mfap4, corresponding to universal fibroblasts of steady-state kidneys. These fibroblastic progenitors exhibit the capacity to undergo rhabdomyoblastic differentiation or transition to highly proliferative anaplastic sarcoma. Investigation of patient samples identified analogous cell states. This study uncovers a fibroblastic origin for DICER1 sarcoma and provides a faithful model for mechanistic investigation and therapeutic development for tumors within the rhabdomyosarcoma spectrum.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7612b5f96fea7c628834a8ae0728263c8ee1372" target='_blank'>
              Spatial single cell transcriptomic analysis of a novel DICER1 Syndrome GEMM informs the cellular origin and developmental hierarchy of associated sarcomas
              </a>
            </td>
          <td>
            F. Kommoss, Joyce Zhang, Branden Lynch, S. Chen, J. Senz, Yana Moscovitz, Lesley A. Hill, Wilder Scott, Jonathan Bush, Kenneth S. Chen, Andreas von Deimling, William D. Foulkes, Gregg B. Morin, T. M. Underhill, Yemin Wang, D. Huntsman
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ac025963bbd7b574699ac753639c82b08e0f94a" target='_blank'>
              DNA methylation of ACADS promotes immunogenic cell death in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Ze Qian, Yifan Jiang, Yacong Wang, Yu Li, Lin Zhang, Xiaofeng Xu, Diyu Chen
          </td>
          <td>2025-01-12</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Long non-coding RNA (lncRNA) is a type of non-coding RNA distinguished by a length exceeding 200 nucleotides. Recent studies indicated that lncRNAs participate in various biological processes, such as chromatin remodeling, transcriptional and post-transcriptional regulation, and the modulation of cell proliferation, death, and differentiation, hence influencing gene expression and cellular function. ADAMTS9-AS1, an antisense long non-coding RNA situated on human chromosome 3p14.1, has garnered significant interest due to its pivotal involvement in the advancement and spread of diverse malignant tumors. ADAMTS9-AS1 functions as a competitive endogenous RNA (ceRNA) that interacts with multiple microRNAs (miRNAs) and plays a crucial role in regulating gene expression and cellular functions by modulating essential signaling pathways, including PI3K/AKT/mTOR, Wnt/β-catenin, and Ras/MAPK pathways. Dysregulation of this factor has been linked to tumor development, migration, invasion, and resistance to apoptotic mechanisms, including as iron-induced apoptosis, underscoring its intricate function in cancer pathology. While current research has clarified certain pathways involved in cancer formation, additional clinical and in vivo investigations are necessary to enhance comprehension of its specific involvement across various cancer types. This review encapsulates the recent discoveries on the correlation of ADAMTS9-AS1 with numerous malignancies, clarifying its molecular mechanisms and its prospective role as a therapeutic target in oncology. Furthermore, it identifies ADAMTS9-AS1 as a potential early diagnostic biomarker and therapeutic target, offering novel opportunities for targeted intervention in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd8babd07e146bd73db5bdfb569c7497c29a95ac" target='_blank'>
              The Dual Role of ADAMTS9-AS1 in Various Human Cancers: Molecular Pathogenesis and Clinical Implications.
              </a>
            </td>
          <td>
            Haodong He, Jingjie Yang, Yan Zhou, Xinyan Zheng, Lihan Chen, Zhujun Mao, Chuyuan Liao, Tongtong Li, Haoran Liu, Gang Zhou, Houdong Li, Chengfu Yuan
          </td>
          <td>2025-01-03</td>
          <td>Anti-cancer agents in medicinal chemistry</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61ff37609fff22e8bd80e0eafac5fa66fc5b8ff0" target='_blank'>
              Aurora B inhibition induces hyper-polyploidy and loss of long-term proliferative potential in RB and p53 defective cells
              </a>
            </td>
          <td>
            Shivam Vora, Saptarshi Chatterjee, Ariel Andrew, Ramyashree Prasanna Kumar, Martina Proctor, Zhen Zeng, Rituparna Bhatt, Deborah Nazareth, Madushan Fernando, M. J. Jones, Yaowu He, John D. Hooper, N. A. McMillan, Jelena Urosevic, Jamal Saeh, Jon Travers, Daniela Cimini, Jing Chen, Brian Gabrielli
          </td>
          <td>2025-01-08</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The de-ubiquitinase USP33 has been shown to possess either tumour-promoting or inhibitory effect on human cancer cells. However, all these findings are mainly based on in vitro cell culture models, and the in vivo evidence, which is more plausible to digest the functional role of USP33 in carcinogenic process, is still lacking. Here, we demonstrate that USP33 modulates DNA damage responses including cell cycle arrest and apoptosis induction through associating with p53. It directly interacts with p53 to mediate its de-ubiquitination and further  stabilisation under DNA damage condition. Depletion of USP33 induces an enhanced level of p53 ubiquitination, which de-stabilises p53 protein leading to impaired DNA damage responses. Furthermore, USP33 silencing shows either promoted or inhibited effect on cell proliferation in human cancer cells with p53 WT and mutant background, respectively. Consistently, mice with hepatocyte-specific USP33 knockout are more sensitive to nitrosodiethylamine (DEN)-induced hepatocarcinogenesis compared to wild type mice. Thus, our in vitro and in vivo evidences illustrate that USP33 possesses anti-tumour activity via regulating p53 stability and activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf79241c93d773eae434c4f3d8db032442a0c49" target='_blank'>
              USP33 Regulates DNA Damage Response and Carcinogenesis Through Deubiquitylating and Stabilising p53.
              </a>
            </td>
          <td>
            Yuqi Zhu, Zixiang Chen, Kaifeng Niu, Mengge Li, Yuchun Deng, Ji Zhang, D. Wei, Jiaqi Wang, YongLiang Zhao
          </td>
          <td>2024-12-18</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The article provides a thorough and up-to-date analysis of the role that microRNAs (miRNAs) within the realm of cancer therapy, paying specific attention to their diagnostic, prognostic as well as therapeutic capabilities. The miRNAs (small non-coding RNAs) are the current major genes that regulate gene expression. They are a key factor in the genesis of cancer. They are oncogenes, or tumor suppressors that play key functions in the signaling pathway that contribute to the development of cancer. This article focuses on the double importance of microRNAs for cancer oncogenesis. This includes both their ability to inhibit cancer suppressor genes and the stimulation of cancer-causing oncogenes. MicroRNAs have been identified for a long time as biomarkers to help in diagnosing cancer and have distinct signatures specific to different kinds of cancer. There are many detection strategies including RT-qPCR, Next Generation Sequencing (NGS) as well as Microarray Analysis that have been evaluated to prove their effectiveness in aiding the non-invasive diagnosis of cancer. The paper provides an overview of the importance of miRNAs to prognosis, highlighting their ability to forecast tumor progression as well as outcomes for cancer patients. In addition, their therapeutic value remains a subject of research. Research is being conducted in order to investigate miRNA-targeting therapy including antisense oligonucleotides, or small molecules inhibitors as possible treatment options for cancer. These methods could favor more specific and individualized approaches than the current techniques. The article also focuses on the current challenges and future prospects linked to miRNA research and demonstrates the complex biological functions they play as well as clinical applications that require investigation. The review is the source of information for researchers, clinicians and scientists who are interested in advancing studies into cancer research as well as personalized treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84f25c91b8e4b7052d56dd5ef6ce88e798f5c5b3" target='_blank'>
              MicroRNA Significance in Cancer: An Updated Review on Diagnostic, Prognostic, and Therapeutic Perspectives.
              </a>
            </td>
          <td>
            Vijay Singh, Aniruddha Sen, Sapna Saini, Shailendra Dwivedi, Ruchika Agrawal, Akash Bansal, Shashank Shekhar
          </td>
          <td>2024-12-01</td>
          <td>EJIFCC</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="SET domain bifurcated histone lysine methyltransferase 1 (SETDB1/ESET), a pivotal H3K9 methyltransferase, has been extensively studied since its discovery over two decades ago. SETDB1 plays critical roles in immune regulation, including B cell maturation, T-cell activity modulation, and endogenous retrovirus (ERV) silencing. While essential for normal immune cell function, SETDB1 overexpression in cancer cells disrupts immune responses by suppressing tumor immunogenicity and facilitating immune evasion. This is achieved through the repression of anti-tumor immune cell production, ERV silencing, and interference with the type I interferon pathway leading to inhibiting immune checkpoint blockade (ICB) efficacy. Beyond its immunological implications, SETDB1 overexpression fosters tumor growth and metastasis via transcriptional silencing of tumor suppressor genes through histone regulation and activating oncogenic signaling by non-histone regulation. These multifaceted roles make SETDB1 an attractive epigenetic target for novel cancer therapies. This review explores SETDB1's dual function in immune regulation and tumor progression, emphasizing its potential in the development of innovative cancer treatments targeting epigenetic dysregulation and oncogenic signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a673433d5768545e8da0592cd5b401ba83b5720" target='_blank'>
              Targeting SETDB1 in cancer and immune regulation: Potential therapeutic strategies in cancer.
              </a>
            </td>
          <td>
            Bo-Syong Pan, Cheng‐Yu Lin, Gilbert Aaron Lee, Hui-Kuan Lin
          </td>
          <td>2025-01-07</td>
          <td>The Kaohsiung journal of medical sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="BACKGROUND AND AIMS
Ambiguous understanding of tumors and the tumor microenvironments (TMEs) hinders accurate diagnosis and available treatment for multifocal hepatocellular carcinoma (HCC) covering intrahepatic metastasis (IM) and multicentric occurrence (MO). Here, we characterized the diverse TMEs of IM and MO identified by whole-exome sequencing (WES) at single-cell resolution.


APPROACH AND RESULTS
We performed parallel WES and single-cell RNA sequencing (scRNA-seq) on twenty-three samples from seven patients to profile their TMEs when major results were validated by immunohistochemistry in the additional cohort. Integrative analysis of WES and scRNA-seq found that malignant cells in IM showed higher intra-tumor heterogeneity, stemness and more activated metabolism than those in MO. Tumors from IM shared similar TMEs while distinct TMEs were noticed in those from MO. Furthermore, CD20+ B cells, plasma cells and conventional type II dendritic cells (cDC2s) were decreased in IM relative to MO while T cells in IM exhibited a more terminally exhausted capacity with a higher proportion of proliferative/exhausted T cells than that in MO. Both CD20 and CD1C correlated with better prognosis in multifocal HCC. Additionally, MMP9+ tumor-associated macrophages were enriched across IM and MO which formed cellular niches with regulatory T cells (Tregs) and proliferative/exhausted T cells.


CONCLUSIONS
Our findings deeply decipher the heterogeneous TMEs between IM and MO, which provide a comprehensive landscape of multifocal HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df2c7a45eee2228edb84e8693a219534a9ab548a" target='_blank'>
              Genomic and the tumor microenvironment heterogeneity in multifocal hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yongheng Yang, Qingqiang Ni, Hongguang Li, Jiuzheng Sun, Xia Zhou, Lingxin Qu, Liyuan Wang, Chuanzong Zhao, Xiaolu Zhang
          </td>
          <td>2024-12-12</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4d5925d1e02fdeb24a29a2a8e41aed5142d614d" target='_blank'>
              Advances in A-to-I RNA editing in cancer
              </a>
            </td>
          <td>
            Yi Zhang, Lvyuan Li, Juana Jessica Mendoza, Dan Wang, Qijia Yan, Lei Shi, Zhaojian Gong, Zhaoyang Zeng, Pan Chen, Wei Xiong
          </td>
          <td>2024-12-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Osteosarcoma (OS) is a prevalent primary malignant bone tumor, typically managed through a combination of neoadjuvant chemotherapy and surgical interventions. Recent advancements in early detection and the use of novel chemotherapeutic agents have significantly improved the 5-year survival rate of OS patients. However, some patients fail to achieve the desired treatment outcomes despite undergoing intensive chemotherapy and surgicals procedures, with chemotherapy resistance emerging as a critical factor contributing to therapeutic failure in OS. Noncoding RNAs (ncRNAs) are a group of RNAs that lack protein-coding capacity but play a crucial role in tumor progression by modulating various biological characteristics of cancer cells, such as proliferation, apoptosis, migration, invasion, and drug resistance. Emerging evidence indicates that the dysregulated expression of numerous ncRNAs in OS cells can influence the response to chemotherapeutic agents by modulating processes such as cell apoptosis, signaling pathways, intracellular drug concentrations, and cell autophagy. This review aims to elucidate the roles and mechanisms of ncRNAs in mediating drug resistance in OS, offering new insights for investigating novel pathways underlying drug resistance, overcoming tumor resistance to therapeutics, and developing innovative chemotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbb08d17a2f1faca4973ae29ad1b2a3dacb83f65" target='_blank'>
              Unraveling the impact of noncoding RNAs in osteosarcoma drug resistance: a review of mechanisms and therapeutic implications.
              </a>
            </td>
          <td>
            Y. Pei, Shenglong Li
          </td>
          <td>2024-12-31</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d75a7884ad18d1874dfac1a870353553ecb1b092" target='_blank'>
              Targeting FOXM1 condensates reduces breast tumour growth and metastasis.
              </a>
            </td>
          <td>
            Feng Xie, Xiaoxue Zhou, Yu Ran, Ran Li, Jing Zou, Shiyun Wan, Peng Su, Xuli Meng, Haiyan Yan, Huasong Lu, Heng Ru, Hai Hu, Zhengwei Mao, Bing Yang, Fangfang Zhou, Long Zhang
          </td>
          <td>2025-01-15</td>
          <td>Nature</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab3b31b19a2854fcc9c5d78fda08a89b093f37f4" target='_blank'>
              DCAF13-mediated K63-linked ubiquitination of RNA polymerase I promotes uncontrolled proliferation in Breast Cancer
              </a>
            </td>
          <td>
            Zhi-Zhi Yang, Bing Yang, Haiyan Yan, Xingyu Ma, Bin Tian, Bing-Qi Zheng, Yong-Xian Chen, Yi-Ming Dong, Jinsi Deng, Ziling Zhan, Yanmei Shi, Jing Yuan Zhang, Daning Lu, Jie-Hua He, Yin Zhang, K. Hu, Shuang Zhu, Keda Zhou, Yu-Chan Zhang, Yiqing Zheng, Dong Yin, Jian-You Liao
          </td>
          <td>2025-01-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2aefa7260397c41402a599b2218ad6360743839" target='_blank'>
              eEF2K as an important kinase associated with cancer survival and prognosis
              </a>
            </td>
          <td>
            Nan Wang, Lilan Cen, Zhe Tian, Miao-Miao An, Qian Gu, Xin-Hong Zhou, Yi-He Zhang, Lucas Liu, Jun Zhang, Di Yang, Yong-Zhi Huang, Xi-Dai Long, Qian Yang
          </td>
          <td>2024-11-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d182842a9c5b9ec9f888e5c17ac019a50f732fb3" target='_blank'>
              Beginning at the ends: telomere and telomere-based cancer therapeutics.
              </a>
            </td>
          <td>
            Zahra Sadr, Masoumeh Ghasemi, Soheyla Jafarpour, Reyhaneh Seyfi, Aida Ghasemi, Elham Boustanipour, H. Khorshid, Naeim Ehtesham
          </td>
          <td>2024-12-06</td>
          <td>Molecular genetics and genomics : MGG</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The DREAM (dimerization partner, RB-like, E2F, and multi-vulval class B) complex is an evolutionarily conserved transcriptional repression complex that coordinates nearly one thousand target genes, primarily associated with the cell cycle processes. The formation of the DREAM complex consequently inhibits cell cycle progression and induces cellular quiescence. Given its unique role in cell cycle control, the DREAM complex has gained significant interest across various physiological and pathological contexts, particularly in conditions marked by dysregulated cell cycles, such as cancer. However, the specific cancer types most significantly affected by alterations in the DREAM complex are yet to be determined. Moreover, the possibility of restoring or pharmacologically targeting the DREAM complex as a therapeutic intervention against cancer remains a relatively unexplored area of research and is currently under active investigation. In this review, we provide an overview of the latest advances in understanding the DREAM complex, focusing on its role in cancer. We also explore strategies for targeting the DREAM complex as a potential approach for cancer therapeutics. Advances in understanding the precise role of the DREAM complex in cancer, combined with ongoing efforts to develop targeted therapies, may pave the way for new options in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08f83080159a75138829f6897856a0dcb6a27d73" target='_blank'>
              Emerging Role of the DREAM Complex in Cancer and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Ye-Jin Hwang, Moon Jong Kim
          </td>
          <td>2025-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Infections have imposed strong selection pressures throughout human evolution, making the study of natural selection's effects on immunity genes highly complementary to disease-focused research. This review discusses how ancient DNA studies, which have revolutionized evolutionary genetics, increase our understanding of the evolution of human immunity. These studies have shown that interbreeding between modern humans and Neanderthals or Denisovans has influenced present-day immune responses, particularly to viruses. Additionally, ancient genomics enables the tracking of how human immunity has evolved across cultural transitions, highlighting strong selection since the Bronze Age in Europe (<4,500 years) and potential genetic adaptations to epidemics raging during the Middle Ages and the European colonization of the Americas. Furthermore, ancient genomic studies suggest that the genetic risk for noninfectious immune disorders has gradually increased over millennia because alleles associated with increased risk for autoimmunity and inflammation once conferred resistance to infections. The challenge now is to extend these findings to diverse, non-European populations and to provide a more global understanding of the evolution of human immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b729418da43553d3cfa26e68fd325ebc987180" target='_blank'>
              Tracing the Evolution of Human Immunity Through Ancient DNA.
              </a>
            </td>
          <td>
            E. Patin, L. Quintana-Murci
          </td>
          <td>2024-12-20</td>
          <td>Annual review of immunology</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="An important direction in cancer prevention is the development of novel strategies that can be used in the treatment by affecting the mechanisms of the regulatory pathways and genomic elements involved in malignant transformation. Strategy proposed in this study is to initiate cell death mechanisms in response to the depletion of the energy resources within a cancer cell due to the uncontrolled spread of mobile genetic elements throughout its genome. The calculation results obtained using a preliminary mathematical model that simulates cellular bioenergetic balance, taking into account the energy consumption for retrotransposition of the mobile elements such as LINE-1 and SINE, show that this scenario is plausible. Intracellular resources undergo a critical redistribution when affecting the genomic defense mechanisms and the LINE-1 transcription rate. This leads to a sharp increase in energy consumption for retrotransposon transcription that causes a significant decrease in the pool of free ATPs in the cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05095e01db5773f04351486017740af52565a16e" target='_blank'>
              Control of the Activity of Mobile Elements in Cancer Cells as a Strategy for Anticancer Therapy
              </a>
            </td>
          <td>
            S. R. Pavlov, V. V. Gursky, M. Samsonova, A. Kanapin, A. A. Samsonova
          </td>
          <td>2024-12-15</td>
          <td>Biofizika</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The long non-coding RNA (lncRNA), HAR1A is emerging as a putative tumour suppressor. In non-neoplastic brain cells, REST suppresses HAR1A expression. In gliomas REST acts as an oncogene and is a potential therapeutic target. It is therefore conceivable that REST promotes glioma progression by down-regulating HAR1A. To test this hypothesis, glioma clinical databases were analysed to study: (I) HAR1A/REST correlation; (II) HAR1A and REST prognostic role; (III) molecular pathways associated with these genes. HAR1A expression and subcellular localization were studied in glioblastoma and paediatric glioma cells. REST function was also studied in these cells, by observing the effects of gene silencing on: (I) HAR1A expression; (II) cancer cell proliferation, apoptosis, migration; (III) expression of neural differentiation genes. The same phenotypes (and cell morphology) were studied in HAR1A overexpressing cells. Our results show that REST and HAR1A are negatively correlated in gliomas. Higher REST expression predicts worse prognosis in low-grade gliomas (the opposite is true for HAR1A). REST-silencing induces HAR1A upregulation. HAR1A is primarily detected in the nucleus. REST-silencing dramatically reduces cell proliferation and induces apoptosis, but HAR1A overexpression has no major effect on investigated cell phenotypes. We also show that REST regulates the expression of neural differentiation genes and that its oncogenic function is primarily HAR1A-independent.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3631f5927fc44541e398927b977abcbc19ff8853" target='_blank'>
              REST-dependent glioma progression occurs independently of the repression of the long non-coding RNA HAR1A.
              </a>
            </td>
          <td>
            Ella Waters, Perla Pucci, Ruman Rahman, A. P. Yatsyshyna, Harry Porter, Mark Hirst, R. Gromnicova, Igor Kraev, V. Mongiardini, Benedetto Grimaldi, Jon Golding, Helen L Fillmore, Balázs Győrffy, Priyadarsini Gangadharannambiar, Christos N Velanis, Christopher J Heath, F. Crea
          </td>
          <td>2024-11-27</td>
          <td>PloS one</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04e5297d8cec280df0e89a9099a47d824b54a4d" target='_blank'>
              Molecular leveraging of HOX-embedded non-coding RNAs in the progression of acute myeloid leukemia.
              </a>
            </td>
          <td>
            C. Wilson, Priyanka Swaroop, Sachin Kumar, Anita Chopra, S. Sharawat
          </td>
          <td>2024-11-30</td>
          <td>Human cell</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cellular senescence features a durable exit from the cell cycle triggered by stress or carcinogens. The INK4 locus is inactivated in various cancers, yet in senescence, p16Ink4a is activated. Whether senescence is tumor-suppressing or -promoting remains a conundrum. We discovered an evolutionally-conserved Vertebrata INK4-homolog. This ink4ab triggers senescence upon oxidative- and/or carcinogenic-stress. Adult Ink4ab-deficient animals failed to activate senescence and developed spontaneous cancers. Combined Ink4ab and Tp53 deficiency revealed a reciprocal senescence and apoptosis regulation, controlling tumorigenesis, including retinoblastoma. INK4-hematopoietic-deficient mice exhibited p19Arf-dependent enhanced senescence-like phenotypes, uncontrolled cell proliferation, defective stem cell differentiation, and splenomegaly, with single-splenocytes spatially-enriched in senescence-associated secretory profiles. Our studies reveal the evolutionary origin of paradigms co-regulating senescence and tumor suppression and offer strategies to exploit these reciprocal pathways for cancer prevention and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9243e085293669a418053332a11c9df19226137a" target='_blank'>
              Evolutionary conserved reciprocal senescence and tumor suppressor signals limit lifetime cancer
              </a>
            </td>
          <td>
            Liqiong Liu, Stephani Davis, Shamila Yusuff, Ann Strange, Yonghua Zhuang, Prasanna Vaddi, Kathleen Flaherty, Kelly Jara, Joseph Kramer, Christine Archer, Schuyler Lee, Bifeng Gao, A. van Bokhoven, Wei Wang, Sharon R. Pine, Tetsuya Nakamura, Hatim E. Sabaawy
          </td>
          <td>2024-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Simple Summary In addition to being essential for DNA replication and repair, proliferating cell nuclear antigen (PCNA) has recently been linked to regulation of signalling, metabolism and apoptosis. As PCNA is a new and unexplored target for cancer treatment, we have here investigated the regulatory role of PCNA in 10 multiple myeloma (MM) cell lines using a multi-omics approach and a PCNA-targeting peptide, ATX-101, currently in clinical development. We have found that ATX-101’s efficacy is linked to PCNA’s role in regulation of proteasomal and ER stress in MM cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a84301979821512ede666f571de223ed396c3f64" target='_blank'>
              Multiple Myeloma Cells with Increased Proteasomal and ER Stress Are Hypersensitive to ATX-101, an Experimental Peptide Drug Targeting PCNA
              </a>
            </td>
          <td>
            Camilla Olaisen, L. M. Røst, Animesh Sharma, C. K. Søgaard, T. Khong, Sigrid Berg, Mi Jang, A. Nedal, Andrew Spencer, P. Bruheim, M. Otterlei
          </td>
          <td>2024-11-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Karyoptosis is a type of regulated cell death (RCD) characterized by explosive nuclear rupture caused by a loss of nuclear membrane integrity, resulting in the release of genomic DNA and other nuclear components into the cytosol and extracellular environment. The mechanism underlying karyoptosis involves a delicate balance between the following forces: the expansion force exerted by the tightly packed DNA in the nucleus, the resistance provided by the nuclear lamina at the inner nuclear membrane (INM), and the tensile force from the cytoskeleton that helps position the nucleus at the center of the cytoplasm, allowing it to remain maximally expanded. In addition, CREB3, a type II integral membrane protein with DNA-binding ability, tethers chromatin to the INM, providing a tightening force through chromatin interactions that prevent nuclear membrane rupture. UVB radiation can trigger this process, inducing CREB3-FL cleavage and producing CREB3-CF. Therefore, UVB acts as an intrinsic factor in the induction of karyoptosis. Importantly, biochemical analysis of RCD markers shows that karyoptosis is distinct from other forms of cell death, such as apoptosis, autophagy, necroptosis, and pyroptosis. This review explores the mechanisms involved in maintaining nuclear membrane integrity and the role of CREB3 in triggering karyoptosis and provides brief suggestions on the potential implications for targeting cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e9b81aa2650efbd4287e4521e56cb949db56f3f" target='_blank'>
              Karyoptosis as a novel type of UVB-induced regulated cell death.
              </a>
            </td>
          <td>
            Weidong Chen, Ji-Young Byun, Han Chang Kang, Hye Suk Lee, Joo Young Lee, Young Jik Kwon, Yong-Yeon Cho
          </td>
          <td>2024-12-03</td>
          <td>Free radical research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b91b716a484954fac596a012c478c6ddaee7a8e" target='_blank'>
              HMGA2 overexpression with specific chromosomal abnormalities predominate in CALR and ASXL1 mutated myelofibrosis.
              </a>
            </td>
          <td>
            Shivani Handa, Christoph Schaniel, J. Tripodi, Daiva Ahire, Md Babu Mia, Sophie Klingborg, D. Tremblay, Bridget K Marcellino, R. Hoffman, V. Najfeld
          </td>
          <td>2024-12-23</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Simple Summary Melanoma is an aggressive malignancy defined by significant intratumoral heterogeneity, driving its capacity for therapeutic resistance and recurrence. This study adopts a systems-level approach to dissect the melanoma microenvironment, focusing on intricate interactions between malignant cells and immune infiltrates. We identified critical regulatory networks and intercellular communication pathways that appear to influence disease progression. These findings highlight the dynamic interplay between tumor-intrinsic factors and the surrounding microenvironment, revealing potential mechanisms underlying immune evasion and therapy resistance. By mapping these complex interactions, the present study builds on our foundation for precision-based therapeutic strategies tailored to the unique biological landscape of melanoma, offering promise for improved clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f383353b5d7dd45f21551edcc7dd626f8651941" target='_blank'>
              Integrated Analysis of Single-Cell and Bulk RNA Data Reveals Complexity and Significance of the Melanoma Interactome
              </a>
            </td>
          <td>
            Michael J Diaz, J. T. Tran, Arthur M. Samia, M. Forouzandeh, Jane M. Grant-Kels, Marjorie E. Montanez-Wiscovich
          </td>
          <td>2025-01-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a highly aggressive cancer with distant metastasis. Accumulated evidence has demonstrated that exosomes are involved in TNBC metastasis. Elucidating the mechanism underlying TNBC metastasis has important clinical significance. Extracellular vesicles (EVs) present a promising avenue for diagnosing and treating triple-negative breast cancer (TNBC) through a technique called "liquid biopsy," offering a new wellspring of biomarkers. These tiny lipid bilayer vesicles, released by most cells, carry a diverse array of RNA molecules that can influence the behaviour of recipient cells. Among these, circular RNAs (circRNAs) have emerged as a subtype of noncoding RNAs capable of modulating gene expression by sponging microRNAs, thus playing crucial roles in various aspects of cancer development and progression, including TNBC. Despite their significance, our understanding of circRNAs involvement in TNBC remains incomplete. However, studies have shown that circRNAs are abundant in EVs, with exosomal circRNAs (exo-circRNAs) particularly influential in cancer biology. These exo-circRNAs can be taken up by neighboring or distant cells, impacting numerous aspects of their physiological states, thereby enhancing cell communication and tumor dissemination. This review provides an overview of EVs key characteristics and functions before delving into exo-circRNAs potential roles in driving or suppressing TNBC, as well as their implications for cancer diagnosis, prognosis, and monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bbb3d750dda1ec3bc792d436bc16135579839b9" target='_blank'>
              CircRNAs in extracellular vesicles associated with triple-negative breast cancer.
              </a>
            </td>
          <td>
            A. A. Qurtam
          </td>
          <td>2025-01-12</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc7636cc7694bca6c468bc6e4c750272c2be1508" target='_blank'>
              SPRTN metalloprotease participates in repair of ROS-mediated DNA-protein crosslinks
              </a>
            </td>
          <td>
            Luke Erber, Arnold Scott Groehler IV, Cesar I. Cyuzuzo, Jahan Baker-Wainwright, Reeja S. Maskey, Lei Li, Yuichi J. Machida, Natalia Tretyakova
          </td>
          <td>2024-12-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Senescence is a tumor suppressor mechanism triggered by oncogene expression and chemotherapy treatment. It orchestrates a definitive cessation of cell proliferation through the activation of the p53-p21 and p16-Rb pathways, coupled with the compaction of proliferative genes within heterochromatin regions. Some cancer cells have the ability to elude this proliferative arrest but the signaling pathways involved in circumventing senescence remain to be characterized. We have recently described that malignant cells capable of evading senescence have an increased expression of specific tRNAs, such as tRNA-Leu-CAA and tRNA-Tyr-GTA, alongside the activation of their corresponding tRNA ligases, namely LARS and YARS. We have previously shown that YARS promotes senescence escape by activating proliferation and cell cycle genes but its functions during this proliferative arrest remain largely unknown. In this study, we have continued to characterize the functions of YARS, describing non-canonical transcriptional functions of the ligase. Our results show that YARS is present in the nucleus of proliferating and senescent cells and interacts with the Trim28 transcriptional regulator. Importantly, YARS binds to the LIN9 promoter, a critical member of the Dream complex responsible for regulating cell cycle gene transcription. The ligase facilitates the binding and the phosphorylation of the type II RNA polymerase and promotes the deposition of activating epigenetic marks on the LIN9 promoter. Consequently, during senescence escape, YARS activates LIN9 expression and both proteins are necessary to induce the proliferation of emergent cells. These results underscore unconventional transcriptional functions of YARS in activating LIN9 expression in proliferating cells and during senescence escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c36e3380440225cd14c09aec61c163d6f628c872" target='_blank'>
              A non-canonical role for the tyrosyl tRNA synthetase: YARS regulates senescence induction and escape and controls the transcription of LIN9.
              </a>
            </td>
          <td>
            H. Coquelet, Geraldine Leman, Amine Maarouf, Coralie Petit, Bertrand Toutain, Cécile Henry, Alice Boissard, Catherine Guette, Eric Lelièvre, Pierre Vidi, Jordan Guillon, O. Coqueret
          </td>
          <td>2025-01-05</td>
          <td>The FEBS journal</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="BACKGROUND Esophageal squamous cell carcinoma (ESCC) is a malignant tumor with high morbidity and mortality, and easy to develop resistance to chemotherapeutic agents. Telomeres are DNA-protein complexes located at the termini of chromosomes in eukaryotic cells, which are unreplaceable in maintaining the stability and integrity of genome. Telomerase, an RNA-dependent DNA polymerase, play vital role in telomere length maintain, targeting telomerase is a promising therapeutic strategy for cancer. AIM To investigate the efficacy and underlying mechanisms of BIBR1532, a telomerase inhibitor, in ESCC. METHODS KYSE150 and KYSE410 cells were cultured and exposed to various concentrations of BIBR1532. Cell viability was assessed at 48 hours and 72 hours to determine the IC50 values. The effects of BIBR1532 on ESCC cell proliferation, migration, and cellular senescence were evaluated using the cell counting kit-8 assay, plate colony formation assay, scratch assay, transwell assay, and β-galactosidase staining, respectively. Western blotting was performed to detect the expression of proteins in BIBR1532-treated ESCC cells, such as human telomerase reverse transcriptase (hTERT), key molecules involved in DNA damage response (DDR) or cellular senescence, as well as telomere-binding proteins. Additionally, a tumor-bearing nude mouse model was established to evaluate the anti-cancer effect of BIBR1532 in vivo. RESULTS The IC50 values for KYSE150 and KYSE410 cells after 48 hours of BIBR1532 exposure were 48.53 μM and 39.59 μM, respectively. These values decreased to 37.22 μM and 22.71 μM, respectively, following a longer exposure of 72 hours. BIBR1532 exhibited dose-dependent effects on KYSE150 and KYSE410 cells, including decreased hTERT expression, inhibition of proliferation and metastasis, and induction of cellular senescence. Mechanistically, BIBR1532 upregulated the expression of the DDR protein, γ-H2AX, and activated the ataxia telangiectasia and Rad3-related protein (ATR)/ check point kinase 1 (CHK-1) and ataxia-telangiectasia mutated gene (ATM)/CHK2 pathways. BIBR1532 downregulated the expression of telomere-binding proteins, including telomeric-repeat binding factor 1 (TRF1), TRF2, protection of telomeres 1, and TIN2-interacting protein 1. In a nude mouse xenograft model, BIBR1532 significantly suppressed tumor growth, reduced hTERT expression, and increased γ-H2AX protein levels. Hematoxylin and eosin staining of various organs, including the heart, liver, spleen, lungs, and kidneys, revealed no apparent adverse effects. CONCLUSION BIBR1532 exerts anti-cancer effects on ESCC by inducing DDR through the ATR/CHK1 and ATM/CHK2 pathways and downregulating the expression of telomere-binding proteins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11c9a7cb4627655be894378fe9cf7a95af0081d7" target='_blank'>
              BIBR1532 inhibits proliferation and metastasis of esophageal squamous cancer cells by inducing telomere dysregulation
              </a>
            </td>
          <td>
            Qin Wang, Qingrong Li, Lei Xu, Zichun Yuan, Xiao Liu, Mao-Ju Tang, Man Luo, Xiao-Wu Zhong, Qiang Ma, Xiao-Lan Guo
          </td>
          <td>2025-01-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Bloom Syndrome helicase (Blm) is a RecQ family helicase involved in DNA repair, cell-cycle progression, and development. Pathogenic variants in human BLM cause the autosomal recessive disorder Bloom Syndrome, characterized by predisposition to numerous types of cancer. Prior studies of Drosophila Blm mutants lacking helicase activity or protein have shown sensitivity to DNA damaging agents, defects in repairing DNA double-strand breaks (DSBs), female sterility, and improper segregation of chromosomes in meiosis. Blm orthologs have a well conserved and highly structured RecQ helicase domain, but more than half of the protein, particularly in the N-terminus, is predicted to be intrinsically disordered. Because this region is poorly conserved across metazoa, we compared closely related species to identify regions of conservation that might be associated with important functions. We deleted two Drosophila-conserved regions in D. melanogaster using CRISPR/Cas9 gene editing and assessed the effects on several Blm functions. Each deletion had distinct effects. Deletion of either conserved region 1 (CR1) or conserved region 2 (CR2) compromised DSB repair through synthesis-dependent strand annealing and resulted in increased mitotic crossovers. In contrast, CR2 is critical for embryonic development but CR1 is less important. Loss of CR1 leads to defects in meiotic crossover designation and patterning but does not impact meiotic chromosome segregation, whereas deletion of CR2 does not result in significant meiotic defects. Thus, while the two regions have overlapping functions, there are distinct roles facilitated by each. These results provide novel insights into functions of the N-terminal region of Blm helicase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09ba52476bc754401f768625b0a2ee1259db3c9a" target='_blank'>
              Functions of the Bloom Syndrome Helicase N-terminal Intrinsically Disordered Region.
              </a>
            </td>
          <td>
            Colleen Bereda, Evan B Dewey, Mohamed A. Nasr, Venkat R. Chirasani, J. Sekelsky
          </td>
          <td>2025-01-10</td>
          <td>Genetics</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d60a3bedb4bf1a029a736f9136afc2a07aef47f" target='_blank'>
              Decoding the functional impact of the cancer genome through protein-protein interactions.
              </a>
            </td>
          <td>
            Haian Fu, Xiulei Mo, Andrey A. Ivanov
          </td>
          <td>2025-01-14</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Transgenic mice and organoid models, such as three-dimensional tumoroid cultures, have emerged as powerful tools for investigating cancer development and targeted therapies. Yet, the extent to which these preclinical models recapitulate the cellular identity of heterogeneous malignancies, like neuroblastoma (NB), remains to be validated. Here, we characterized the transcriptional landscape of TH-MYCN tumors by single-cell RNA sequencing (scRNA-seq) and developed ex vivo tumoroids. Integrated analysis with murine fetal adrenal samples confirmed that both TH-MYCN tumors and tumoroids closely mirror the cellular profiles of normal embryonic sympathoblasts and chromaffin cells. Comprehensive comparison between tumors from NB patients and TH-MYCN mice demonstrated similarities in adrenergic tumor cell composition. Ex vivo tumoroid cultures displayed histological resemblance and shared transcriptional profiles with the originating TH-MYCN tumors and human NB. Importantly, subpopulations within tumoroids exhibited gene expression associated with poor NB patient survival. Notably, recurrent observations of a low-proliferative chromaffin phenotype connected to the highly proliferative sympathetic phenotype suggested that pushing sympathoblasts into a chromaffin-like state may offer an interesting therapeutic strategy for NB. Together, this study not only deepens our understanding of a widely used transgenic mouse NB model but also introduces an ex vivo model that maintains critical adrenergic cell state identity, thereby enhancing its translational potential for NB research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee5619b05ffd717eb9d8928b0e83ee66ca9cda8b" target='_blank'>
              Comparative Single-Cell Transcriptomics of Human Neuroblastoma and Preclinical Models Reveals Conservation of an Adrenergic Cell State.
              </a>
            </td>
          <td>
            Bethel Tesfai Embaie, Hirak Sarkar, A. Alchahin, Jörg Otte, T. Olsen, Conny Tümmler, Polina Kameneva, Artem V Artemov, N. Akkuratova, I. Adameyko, J. Stukenborg, Malin Wickström, P. Kogner, J. Johnsen, Shenglin Mei, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2025-01-14</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Long non-coding RNA Rhabdomyosarcoma 2-associated Transcript (RMST) is a crucial regulator in various biological processes, particularly in neurogenesis and cancer progression. This review summarizes current knowledge on structure, expression patterns, and functional roles across different organs and diseases of RMST. RMST exhibits tissue-specific expression, notably in brain tissues and vascular endothelial cells, and plays a significant role in neuronal differentiation through interaction with SRY-box 2 (SOX2). In cancer, RMST predominantly functions as a tumor suppressor, with context-dependent roles observed across different cancer types. RMST is also implicated in neurological disorders, cardiovascular diseases, and Hirschsprung's disease. Mechanistically, RMST acts as a competing endogenous RNA and a transcriptional regulator, interacting with various microRNAs and proteins to modulate gene expression. The potential of RMST as a biomarker and therapeutic target is increasingly recognized, particularly in atherosclerosis and cancer. While current findings are promising, further research is needed to fully elucidate the functions and translate these insights into clinical applications of RMST. This review underscores the significance of RMST in cellular processes and disease pathogenesis, highlighting its potential as a novel target for diagnostic and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/274785be6ba4141ed74a6cfcddf7cc469ab67f46" target='_blank'>
              RMST: A long noncoding RNA involved in cancer and disease.
              </a>
            </td>
          <td>
            Hidenori Tani
          </td>
          <td>2024-11-29</td>
          <td>Journal of biochemistry</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Adenocarcinoma of the esophagogastric junction (AEGJ) is a highly aggressive tumor that frequently metastasizes to the liver. Understanding the cellular and molecular mechanisms that drive this process is essential for developing effective therapies.We employed single-cell RNA sequencing to analyze the tumor heterogeneity and microenvironmental landscape in patients with AEGJ liver metastases. This approach enabled us to characterize the diverse cell populations involved in the liver metastatic process.Our analysis revealed a significant involvement of fibroblasts and mural cells in AEGJ liver metastasis. We identified a specific fibroblast type in AEGJ liver metastasis and observed distinct gene expression patterns between adenocarcinoma of the esophagogastric junction and other stomach adenocarcinomas. Our study demonstrated high expression of the SFRP2 gene in pericyte cells during the liver metastasis of AEGJ. The incorporation of GEO, TCGA, and immunofluorescence staining of SFRP2 expression enhanced our study. High expression of SFRP2 in pericytes may influence vascular stability and angiogenesis through the Wnt pathway.Our study provides novel insights into the cellular interactions and molecular mechanisms that underlie AEGJ liver metastasis. Targeting the identified subtype of fibroblasts or influencing SFRP2 gene expression in pericytes may offer new therapeutic strategies for combating this aggressive tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c76cf1003bffe0097e28dee22754d6f4e46b48db" target='_blank'>
              Integrated single-cell RNA sequencing reveals the tumor heterogeneity and microenvironment landscape during liver metastasis in adenocarcinoma of esophagogastric junction
              </a>
            </td>
          <td>
            Junrui Xu, Ussama Sadiq, Wangruizhi Zhao, Hengbo Xia, Yiwei Liu, Renquan Zhang, Aman Xu
          </td>
          <td>2025-01-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Therapy resistance still constitutes a common hurdle in the treatment of many human cancers and is a major reason for treatment failure and patient relapse, concomitantly with a dismal prognosis. In addition to “intrinsic resistance”, e.g., acquired by random mutations, cancer cells typically escape from certain treatments (“acquired resistance”) by a large variety of means, including suppression of apoptosis and other cell death pathways via upregulation of anti-apoptotic factors or through inhibition of tumor-suppressive proteins. Therefore, ideally, the tumor-cell-restricted induction of apoptosis is still considered a promising avenue for the development of novel, tumor (re)sensitizing therapies. A growing body of evidence has highlighted the multifaceted role of tripartite motif 25 (TRIM25) in controlling different aspects of tumorigenesis, including chemotherapeutic drug resistance. Accordingly, overexpression of TRIM25 is observed in many tumors and frequently correlates with a poor patient survival. In addition to its originally described function in antiviral innate immune response, TRIM25 can play critical yet context-dependent roles in apoptotic- and non-apoptotic-regulated cell death pathways, including pyroposis, necroptosis, ferroptosis, and autophagy. The review summarizes current knowledge of molecular mechanisms by which TRIM25 can interfere with different cell death modalities and thereby affect the success of currently used chemotherapeutics. A better understanding of the complex repertoire of cell death modulatory effects by TRIM25 is an essential prerequisite for validating TRIM25 as a potential target for future anticancer therapy to surmount the high failure rate of currently used chemotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eb1e40b262aff1ec5083769b832df042944b012" target='_blank'>
              TRIM25: A Global Player of Cell Death Pathways and Promising Target of Tumor-Sensitizing Therapies
              </a>
            </td>
          <td>
            W. Eberhardt, Usman Nasrullah, Josef Pfeilschifter
          </td>
          <td>2025-01-07</td>
          <td>Cells</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Epigenetics which examines the regulation of genes without modification of the DNA sequence, plays a crucial role in various biological processes and disease mechanisms. Among the different forms of epigenetic modifications, histone post-translational modifications (PTMs) are important for modulating chromatin structure and gene expression. Aberrant levels of histone PTMs are implicated in a wide range of diseases, including cancer, making them promising targets for biomarker discovery and therapeutic intervention. In this context, blood, tissues, or cells serve as valuable resources for epigenetic research and analysis. Traditional methods such as mass spectrometry and western blotting are widely used to study histone PTMs, providing qualitative and (semi)quantitative information. However, these techniques often face limitations that could include throughput and scalability, particularly when applied to clinical samples. To overcome these challenges, we developed and validated 13 Nu.Q® immunoassays to detect and quantify specific histone PTM-nucleosomes from K2EDTA plasma samples. Then, we tested these assays on other types of samples, including chromatin extracts from frozen tissues, as well as cell lines and white blood cells Our findings demonstrate that the Nu.Q® assays offer high specificity, sensitivity, precision and linearity, making them effective tools for epigenetic profiling. A comparative analysis of HeLa cells using mass spectrometry, Western blot, and Nu.Q® immunoassays revealed a consistent histone PTMs signature, further validating the effectiveness of these assays. Additionally, we successfully applied Nu.Q® assays across various biological samples, including human tissues from different organs and specific white blood cell subtypes, highlighting their versatility and applicability in diverse biological contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ea54269010715f8d7295e217ed70c9663ed44c1" target='_blank'>
              High-Throughput Epigenetic Profiling Immunoassays for Accelerated Disease Research and Clinical Development
              </a>
            </td>
          <td>
            P. Van den Ackerveken, Clotilde Hannart, Dorian Pamart, Robin Varsebroucq, Marion Wargnies, Olivia Thiry, Marie Lurkin, Séverine Vincent, Muriel Chapelier, Guillaume Rommelaere, M. Herzog
          </td>
          <td>2024-12-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Cancer research has historically focused on the Somatic Mutation Theory (SMT), viewing cancer as a consequence of genetic mutations. However, this perspective has limitations in explaining phenomena like tumor reversion and cancer heterogeneity. This paper introduces an alternative approach: viewing cancer as a complex information processing system shaped by its microenvironment. By integrating historical data on tumor reversion and insights into evolutionary dynamics, I propose a reframing of cancer biology. This process-oriented perspective highlights the role of cellular plasticity and adaptive behaviors, offering new pathways for therapeutic development">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d33c3cc92fad268ef72893a64fd8d179664753cb" target='_blank'>
              Rethinking Cancer Evolution: From Genetic Mutations to Complex Information Systems in Tumor Reversion
              </a>
            </td>
          <td>
            M. Tez
          </td>
          <td>2024-12-13</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Lung cancer is one of the most common cancers in humans. However, there is still a need to understand the underlying mechanisms of a normal cell developing into a cancer cell. Here, we develop the chromosome dynamic structural model and quantify the important characteristics of the chromosome structural ensemble of the normal lung cell and the lung cancer A549 cell. Our results demonstrate the essential relationship among the chromosome ensemble, the epigenetic marks, and the gene expressions, which suggests the linkage between chromosome structure and function. The analysis reveals that the lung cancer cell may have a higher level of relative ensemble fluctuation (micro CFI) and a higher degree of phase separation between the two compartments than the normal lung cell. In addition, the significant conformational "switching off" events (from compartment A to B) are more than the significant conformational "switching on" events during the lung cancerization. We identify "nucleation seeds" or hot spots in chromosomes, which initiate the transitions and determine the mechanisms. The hot spots and interaction network results reveal that the lung cancerization process (from normal lung to A549) and the reversion process have different mechanisms. These investigations have revealed the cell fate determination mechanism of the lung cancer process, which will be helpful for the further prevention and control of cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ced4f0cd9c3dbc1395b811ded415faaab1d4100" target='_blank'>
              Uncovering the lung cancer mechanisms through the chromosome structural ensemble characteristics and nucleation seeds.
              </a>
            </td>
          <td>
            Wen-Ting Chu, Jin Wang
          </td>
          <td>2024-12-11</td>
          <td>The Journal of chemical physics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d1fd954b718bc92e2f3d26e9d0a2654e108d08" target='_blank'>
              Prostate cancer epigenetics - from pathophysiology to clinical application.
              </a>
            </td>
          <td>
            V. Constâncio, J. Lobo, J. P. Sequeira, Rui Henrique, Carmen Jerónimo
          </td>
          <td>2025-01-16</td>
          <td>Nature reviews. Urology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Synthetic lethality is defined as a type of genetic interaction where the combination of two genetic events results in cell death, whereas each of them separately does not. Synthetic lethality can be a useful tool in personalised oncology. MLH1 is a cancer-related gene that has a central role in DNA mismatch-repair and TP53 is the most frequently mutated gene in cancer. To identify genetic events that can lead to tumour death once either MLH1 or TP53 is mutated, a genome-wide genetic screening was performed. Thus, mutations in all protein-coding genes were introduced into haploid human embryonic stem cells (hESCs) with and without loss-of-function mutations in the MLH1 or TP53 genes. These experiments uncovered a list of putative hits with EXO1, NR5A2, and PLK2 genes for MLH1, and MYH10 gene for TP53 emerging as the most promising candidates. Synthetic lethal interactions of these genes were validated genetically or chemically using small molecules that inhibit these genes. The specific effects of SR1848, which inhibits NR5A2, ON1231320 or BI2536, which inhibits PLK2, and blebbistatin, which inhibits MYH10, were further validated in cancer cell lines. Finally, animal studies with CCL xenografts showed the selective effect of the small molecule BI2536 on MLH1-null tumours and of blebbistatin on TP53-mutated tumours. Thus, demonstrating their potential for personalised medicine, and the robustness of genetic screening in haploid hESCs in the context of cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf8258266a9c323d57d9c42f16e5c72059537055" target='_blank'>
              Genome-Wide Screening in Haploid Stem Cells Reveals Synthetic Lethality Targeting MLH1 and TP53 Deficient Tumours.
              </a>
            </td>
          <td>
            R. Cashman, Guy Haim-Abadi, Elyad Lezmi, Hagit Philip, J. Nissenbaum, Ruth Viner-Breuer, Chen Kozulin, T. Golan‐lev, Aseel Gadban, Shiri Spinner-Potesky, O. Yanuka, O. Kopper, N. Benvenisty
          </td>
          <td>2025-01-15</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Abstract Xeno nucleic acids (XNAs) are unnatural analogues of the natural nucleic acids in which the canonical ribose or deoxyribose rings are replaced with alternative sugars, congener structures or even open-ring configurations. The expanding repertoire of XNAs holds significant promise for diverse applications in molecular biology as well as diagnostics and therapeutics. Key advantages of XNAs over natural nucleic acids include their enhanced biostability, superior target affinity and (in some cases) catalytic activity. Natural systems generally lack the mechanisms to transcribe, reverse transcribe or replicate XNAs. This limitation has been overcome through the directed evolution of nucleic acid-modifying enzymes, especially polymerases (pols) and reverse transcriptases (RTs). Despite these advances, the mechanisms by which synthetic RT enzymes read these artificial genetic polymers remain largely unexplored, primarily due to a scarcity of structural information. This study unveils first structural insights into an evolved thermostable DNA pol interacting with the XNA 1,5-anhydrohexitol nucleic acid (HNA), revealing unprecedented HNA nucleotide conformations within a ternary complex with the enzyme. These findings not only deepen our understanding of HNA to DNA reverse transcription but also set the stage for future advancements of this and similar enzymes through deliberate design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a70ba41782df746ce8d0f1ebc756856322bb7e1c" target='_blank'>
              Structural insights into a DNA polymerase reading the xeno nucleic acid HNA
              </a>
            </td>
          <td>
            Cédric Gutfreund, K. Betz, M. Abramov, Frédérick Coosemans, Phillipp Holliger, Piet Herdewijn, Andreas Marx
          </td>
          <td>2024-12-03</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Kirsten rat sarcoma viral oncogene homolog (KRAS) is the most frequently mutated oncogene in lung adenocarcinoma, with G12C and G12V being the most predominant forms. Recent breakthroughs in KRASG12C inhibitors have transformed the clinical management of patients with the G12C mutation and advanced our understanding of the function of this mutation. However, little is known about the targeted disruption of KRASG12V, partly due to a lack of specific inhibitors. Here, we leverage the degradation tag (dTAG) system to develop a KRASG12V-transgenic mouse model. We explored the therapeutic potential of KRASG12V degradation and characterized its effect on the tumor microenvironment (TME). Our study reveals that degradation of KRASG12V abolished lung and pancreatic tumors in mice and caused a robust inhibition of KRAS-regulated cancer-intrinsic signaling. Importantly, targeted degradation of KRASG12V reprogrammed the TME toward a stimulatory milieu and drove antitumor immunity, elicited mainly by effector and cytotoxic CD8+ T cells. Our work provides insights into the effect of KRASG12V degradation on both tumor progression and the immune response, highlighting degraders as a powerful strategy for targeting KRAS-mutant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7557f24725afcb0b1e2df48764b1f6cd0b9ab54" target='_blank'>
              Targeted degradation of oncogenic KRASG12V triggers antitumor immunity in lung cancer models
              </a>
            </td>
          <td>
            Dezhi Li, Ke Geng, Yuan Hao, Jiajia Gu, Saurav Kumar, Annabel T Olson, Christina C. Kuismi, Hye Mi Kim, Yuanwang Pan, Fiona Sherman, Asia M. Williams, Yiting Li, Fei Li, Ting Chen, C. Thakurdin, Michela Ranieri, Mary Meynardie, Daniel S. Levin, Janaye Stephens, Alison Chafitz, Joy Chen, Mia S Donald-Paladino, Jaylen M Powell, Ze-Yan Zhang, Wei Chen, Magdalena Ploszaj, Han Han, S. Gu, Tinghu Zhang, Baoli Hu, B. Nacev, Medard Ernest Kaiza, Alice H. Berger, Xuerui Wang, Jing Li, Xuejiao Sun, Yang Liu, Xiaoyang Zhang, T. Bruno, N. Gray, Behnam Nabet, Kwok-Kin Wong, Hua Zhang
          </td>
          <td>2024-12-24</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="ABSTRACT Missing an entire chromosome or chromosome arm in normal diploid cells has a deleterious impact on cell viability, which may contribute to the development of specific birth defects. Nevertheless, the effects of chromosome loss in human cells have remained unexplored due to the lack of suitable model systems. Here, we developed an efficient, selection‐free approach to generate partial monosomy in human induced pluripotent stem cells (iPSCs). The introduction of Cas9 proteins and a pair of gRNAs induces over megabase‐sized interstitial chromosomal deletions. Using human chromosome 21 (HSA21) as a model, partial monosomy 21q (PM21q) iPSC lines with deletions ranging from 4.5 to 27.9 Mb were isolated. A 33.6 Mb deletion, encompassing all protein‐coding genes on 21q, was also achieved, establishing the first 21q monosomy human iPSC line. Transcriptome and proteome analyses revealed that the abundances of mRNA and protein encoded by the majority of genes in the monosomic regions are half of the diploid expression level, indicating an absence of dosage compensation. The ability to generate customized partial monosomy cell lines on an isogenic, karyotypically normal background should facilitate the gain of novel insights into the impact of chromosome loss on cellular fitness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f457fb7efd63146141d89c41994690b979076d69" target='_blank'>
              Generation of Monosomy 21q Human iPS Cells by CRISPR/Cas9‐Mediated Interstitial Megabase Deletion
              </a>
            </td>
          <td>
            Masaya Egawa, N. Uno, Rina Komazaki, Yusuke Ohkame, Kyotaro Yamazaki, Chihiro Yoshimatsu, Yuki Ishizu, Yusaku Okano, Hitomaru Miyamoto, Mitsuhiko Osaki, Teruhiko Suzuki, Kazuyoshi Hosomichi, Yasunori Aizawa, Yasuhiro Kazuki, Kazuma Tomizuka
          </td>
          <td>2024-11-24</td>
          <td>Genes to Cells</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cancer, a complex and heterogeneous disease, continues to be a major global health concern. Despite advancements in diagnostics and therapeutics, the aggressive nature of certain cancers remain a significant challenge, necessitating a deeper understanding of the underlying molecular mechanisms driving their severity and progression. Cancer severity and progression depend on cellular properties such as cell migration, cell division, cell shape changes, and intracellular transport, all of which are driven by dynamic cellular microtubules. Dynamic properties of microtubules, in turn, are regulated by an array of proteins that influence their stability and growth. Among these regulators, End Binding (EB) proteins stand out as critical orchestrators of microtubule dynamics at their growing plus ends. Beyond their fundamental role in normal cellular functions, recent research has uncovered compelling evidence linking EB proteins to the pathogenesis of various diseases, including cancer progression. As the field of cancer research advances, the clinical implication of EB proteins role in cancer severity and aggressiveness become increasingly evident. This review aims to comprehensively explore the role of microtubule-associated EB proteins in influencing the severity and aggressiveness of cancer. We also discuss the potential significance of EB as a clinical biomarker for cancer diagnosis and prognosis and as a target for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e8a843dc90163f1664e05266a959467a1810e6f" target='_blank'>
              End Binding Proteins: Drivers of Cancer Progression.
              </a>
            </td>
          <td>
            Dhakshmi Sasankan, Renu Mohan
          </td>
          <td>2024-12-19</td>
          <td>Cytoskeleton</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Mycoplasma bovis (M. bovis) is characterized by a reduced genomic size and limited synthetic capacity, including the inability to synthesize nucleotides de novo, relies on nucleases for nutrient acquisition and survival. A number of nucleases have been implicated in M. bovis pathogenicity, facilitating substrate degradation and contributing to DNA repair mechanisms that enhance bacterial persistence. The present study confirmed that the T5.808 mutant, in which a novel nuclease gene (Mbov_0701) was disrupted by the mini-Tn4001 transposon, exhibits a growth defect when co-cultured with EBL cells. However, the restoration of Mbov_0701 resulted in the resumption of growth in the mutant. The characterization of MbovP701 revealed that it had high activity in hydrolyzing dsDNA with 5′- to 3′- polarity. Furthermore, the substrates of MbovP701 were extended to include linear dsDNA, ssDNA, RNA, and plasmid DNA. The exonuclease activity is dependent on the presence of Mn2+ and/or Mg2+ ions, with an optimal pH and temperature of 8.3 and 43 °C, respectively. The truncation experiments of rMbovP701 revealed that YqaJ (41–185 aa) is the key functional domain of MbovP701 exonuclease. In conclusion, the present study identified a novel nuclease in M. bovis that plays an essential role in the proliferation of this minimal organism. This finding elucidates the survival strategy and pathogenesis of M. bovis, suggesting a potential therapeutic strategy for the treatment of M. bovis through targeting the inhibition of MbovP701. Moreover, it provides a foundation for future investigations into the interactions between MbovP701 and other nucleases involved in M. bovis biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edf88c44ad6d18a2167a54f0906a3be9810ae56b" target='_blank'>
              Novel Nuclease MbovP701 with a Yqaj Domain Is Interrelated with the Growth of Mycoplasma bovis
              </a>
            </td>
          <td>
            Zhiyu Hao, Doukun Lu, Xixi Li, Abdul Raheem, Gang Zhao, A. Dawood, Yingyu Chen, Xi Chen, Changmin Hu, Jianguo Chen, Lei Zhang, Xifang Zhu, A. Guo
          </td>
          <td>2024-12-01</td>
          <td>Microorganisms</td>
          <td>0</td>
          <td>27</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [5, 3],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>