<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-02-09 10:10:35 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>48</td>
          <td>107</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>96</td>
          <td>140</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>126</td>
          <td>146</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>414</td>
          <td>140</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>349</td>
          <td>160</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f09bfb27dbaed264c5656346c24d74a25e18488" target='_blank'>
              An Atlas of Extrachromosomal DNA Structures Illuminates Its Evolution and Biogenesis in Cancer
              </a>
            </td>
          <td>
            Chee Hong Wong, Kuo-Chen Wei, Kuan-Yu Kung, Li-Hao Yang, Yin-Cheng Huang, Chiung-Yin Huang, I-Jiuan Chung, Meng-Fan Huang, Pei-Hua Peng, Meihong Li, C. Ngan, Kou-Juey Wu, Chia-Lin Wei
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are large, acentric, circular DNA molecules that occur pervasively across many human cancers. EcDNA can drive tumor formation and evolution, contribute to drug resistance, and associate with poor patient survival outcomes. Beyond mediating high copy numbers, the circular topology and dynamic conformational changes of ecDNA disrupt topological domains and rewire regulatory networks, thereby conferring an important role in the transcriptional regulation of oncogenes. Here, we develop ec3D, a computational method for reconstructing the three-dimensional structures of ecDNA and analyzing significant interactions from high-throughput chromatin capture (Hi-C) data. Given a candidate ecDNA sequence and the corresponding whole-genome Hi-C as input, ec3D reconstructs the spatial structure of ecDNA by maximizing the Poisson likelihood of observed interactions. Ec3D’s performance was validated using both simulated ecDNA structures with varying conformations and Hi-C data from previously-characterized cancer cell lines. Our reconstructions reveal that ecDNAs occupy spherical configurations and mediate unique long-range interactions involved in gene regulation. Through algorithmic innovations, ec3D can resolve complex ecDNA structures with duplicated genomic segments, identify multi-way interactions, and distinguish between interactions arising from direct spatial proximity and secondary interactions resulting from alternative folding patterns or intermolecular (trans) contacts of ecDNA molecules. Our findings provide insights into how the spatial organization of ecDNA may influence gene regulation and contribute to increased oncogene expression. Code availability https://github.com/AmpliconSuite/ec3D">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec860e1d2a390c1994942055749c19eed5ff637" target='_blank'>
              Reconstructing the three-dimensional architecture of extrachromosomal DNA with ec3D
              </a>
            </td>
          <td>
            Biswanath Chowdhury, Kaiyuan Zhu, Chaohui Li, J. Luebeck, Owen S. Chapman, Katerina Kraft, Shu Zhang, Lukas Chavez, A. Henssen, P. Mischel, Howard Y. Chang, V. Bafna
          </td>
          <td>2025-05-19</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>75</td>
        </tr>

        <tr id="Segmental copy-number gains are major contributors to human genetic variation and disease, but how these alterations arise remains incompletely understood. Here, based on the analyses of both experimental evolution and human disease genomes, we describe a general mechanism of segmental copy-number gain from a rearrangement process termed ‘breakage–replication/fusion’. The hallmark genomic feature of breakage–replication/fusion is adjacent parallel breakpoints: two or more rearrangement breakpoints derived from replication of a single ancestral DNA end. We show that adjacent parallel breakpoints are a widespread feature of DNA duplications in human disease genomes and experimental models of chromothripsis. In addition to adjacent parallel breakpoints, breakage–replication/fusion also explains two other patterns of complex rearrangements with unclear provenance: chains of short (≤1 kb) insertions and high-level amplification consisting of inverted segments. Together, these findings revise the mechanistic model for chromothripsis and provide a new conceptual framework for understanding the origin of segmental DNA duplication during genome evolution. This paper introduces breakage–replication/fusion, a genomic rearrangement process underpinning three patterns of copy-number gains found in cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/943c5640094a2cf860368d879c898eeb29d9cb6e" target='_blank'>
              A breakage–replication/fusion process explains complex rearrangements and segmental DNA amplification
              </a>
            </td>
          <td>
            Cheng-Zhong Zhang, Carlos Mendez-Dorantes, Kathleen H. Burns, David Pellman
          </td>
          <td>2026-01-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Canine osteosarcoma (OS) is an aggressive bone cancer with remarkable similarity to human OS. Despite its value as a comparative oncology model, the drivers of genomic instability remain poorly defined, hindering cross-species insights into disease progression. We applied long-read PacBio HiFi whole genome sequencing to matched tumor and blood samples from four purebred dogs to generate high-resolution maps of somatic structural variants (SVs). Tumor genomes showed extensive rearrangement complexity, with satellite repeat regions, particularly SAT1_CF, enriched near somatic SV breakpoints and occurring within genomic contexts marked by focal hypomethylation. We also identified multiple extrachromosomal DNA (ecDNA) elements carrying oncogenes indicating ecDNA-mediated oncogene amplification. Together, these data provide an integrated genomic and epigenomic view of canine OS, implicating satellite repeats as recurrent substrates for rearrangement and ecDNA as a prominent mode of genome remodeling. This work advances our understanding of OS genome dynamics and establishes a comparative framework for investigating repeat-driven instability and oncogene amplification across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5300af19b1916e9abe3db71ff1fa34ffceb5e24d" target='_blank'>
              Satellite DNA fragility accompanies complex genome rearrangements and ecDNA oncogene amplification in canine osteosarcomas
              </a>
            </td>
          <td>
            F. Yilmaz, Wonyoung Kong, Sabriya A. Syed, Francis H. O’Neill, Jody T. Lombardi, P. K. Ng, Ching C. Lau, Charles Lee
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genomic alterations are a hallmark of cancer, and extrachromosomal DNA (ecDNA) has emerged as a key source of oncogene selection, tumor growth, and drug resistance. The intratumor heterogeneity and clonal selection of ecDNA is, however, poorly understood. In this study, we pursue a computational approach that leverages allelic imbalance and outlier expression from standard single-cell RNA sequencing (scRNA-seq) to deconvolve the tumor heterogeneity of ecDNA at the single-cell level (ecSingle). Using this approach, we identify oncogene-carrying ecDNAs in tumor samples at the single-cell level, which we validate using genome sequencing. Moreover, we show the superiority of using single-molecule long-read sequencing in resolving ecDNA. ecDNAs displayed extensive intratumor heterogeneity, including subclonal oncogene-carrying ecDNA in primary tumor cells that segregate with distinct transcriptional cell states. Importantly, we show that a rare ecDNA+ clone in the primary tumor can expand to form dominant clones in relapse tumors. Our study introduces a novel approach to studying ecDNA at the single-cell level, enabling both clonal evolution and transcription cell state analysis. We apply this approach to cancer samples to gain deeper insights into the role of ecDNA in intratumor heterogeneity and cellular plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7df14db8ad5c3596e9b413772fed54a722b4bb2b" target='_blank'>
              Resolving clonal evolution and selection of extrachromosomal DNA at single-cell resolution
              </a>
            </td>
          <td>
            J. Hendriksen, A. Locallo, B. Schlotmann, Francisco Germán Rodríguez González, Jane Skjøth-Rasmussen, C. Yde, D. Nørøxe, H. S. Poulsen, U. Lassen, J. Weischenfeldt
          </td>
          <td>2026-01-29</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Oncogene amplifications fuel some of the most lethal, therapy‑refractory cancers, yet remain clinically untargeted. We report a single‑guide CRISPR/Cas9 strategy that converts the sheer copy‑number excess of oncogene amplicons into an Achilles’ heel. A solitary intronic double‑strand break is innocuous in diploid genomes but collapses oncogene amplification‑positive cells across neuroblastoma, small‑cell lung and colorectal carcinoma models, driving > 90% loss of viability, G₂/M blockade and catastrophic DNA‑damage signalling. Amplified‑locus cleavage rewires transcription toward cell death activation, necroptosis and cGAS–STING–mediated immunogenic cell death, enabling dendritic‑cell cross‑priming and T‑cell activation and proliferation. In xenografts, delivery of the intronic sgRNA shrinks tumours by 90%, prolongs survival and remodels the innate tumour microenvironment. Deep sequencing confirms negligible off‑target editing, and combination with doxorubicin achieves supra‑additive killing. These findings establish amplification density, not sequence content, as a tractable, tumour‑exclusive target and unveil a dual‑action platform that is simultaneously cytotoxic and immunostimulatory. Editing of tumor amplifications therefore offers a blueprint for translating copy‑number aberrations into precision genome‑editing therapies for treatment‑resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8784639587a78e80b9c245135bde519d167d493" target='_blank'>
              Selective genome editing of amplified oncogenes triggers immunogenic cell death and tumor remodeling
              </a>
            </td>
          <td>
            A. Nieto-Sanchez, M. Martinez-Lage, P. Puig-Serra, S. Carpintero, A. Alonso-Yanez, P. Ojeda-Walczuk, M. Ibañez-Navarro, G. Pita, FJ. Moya, C. Moreno, MC. Martin, R. Alonso, R. Nuñez-Torres, VJ. Sanchez-Arevalo Lobo, L. Alonso-Guirado, N. Malats, A. González-Neira, L. Fernandez, P. Roda-Navarro, R. Torres-Ruiz, S. Rodríguez-Perales
          </td>
          <td>2025-12-22</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/794649b9036f93693e77ad613d6e85889d3eff72" target='_blank'>
              Targeting extrachromosomal DNA in human cancers.
              </a>
            </td>
          <td>
            I. Wong, Hyerim Yi, Bruno Melillo, Benjamin F. Cravatt, Howard Y. Chang, P. Mischel
          </td>
          <td>2026-02-03</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Extrachromosomal DNA (ecDNA) amplifications are key drivers of human cancers. Here, we show that ecDNAs are major platforms for generating and amplifying oncogene fusion transcripts across diverse cancer types. By integrating analysis of whole genome and transcriptome sequences from 1,825 tumor samples and cancer cell lines of a wide variety of tissue types, we reveal that ecDNAs have the highest rate of oncogene fusion events of any copy number alteration. Focusing on the most common ecDNA fusion hotspot, we find that fusion of the 5' end of the long noncoding RNA gene, PVT1–with exon 1 joined to diverse 3' partners–confers increased RNA stability, and enhances MYC-dependent transcription and cancer cell survival. These results demonstrate that ecDNA fosters genome instability and frequent oncogene fusion formation under strong selection in cancer.



 Shu Zhang, Hyerim Yi, Jason Swinderman, Yanbo Wang, Vishnupriya Kanakaviti, King L. Hung, Ivy Tsz-Lo. Wong, Suhas Srinivasan, Ellis J. Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G. Jones, Jens Luebeck, Chris Bailey, Yanding Zhao, Julia A. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Kabir S. Mahajan, Frances Liang, Mariam Jamal-Hanjani, Dean W. Felsher, Luke A. Gilbert, Vineet Bafna, Paul S. Mischel, Howard Y. Chang. EcDNA-derived structural variants: A common pathway to oncogenic fusion transcript amplification in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A007.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b68513e53c43f7bee512369c94fed2f8b73ac179" target='_blank'>
              Abstract A007: EcDNA-derived structural variants: A common pathway to oncogenic fusion transcript amplification in cancer
              </a>
            </td>
          <td>
            Shu Zhang, Hyerim Yi, J. Swinderman, Yanbo Wang, Vishnupriya Kanakaviti, King L. Hung, I. Wong, Suhas Srinivasan, Ellis J Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G. Jones, J. Luebeck, Chris Bailey, Yanding Zhao, J. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Kabir S. Mahajan, Frances Liang, M. Jamal-Hanjani, D. Felsher, Luke A. Gilbert, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d983622827e375feb6c8bdb7968042a7608e5a98" target='_blank'>
              Genomic Flexibility Through Extrachromosomal Amplifications: A Leishmania Survival Strategy
              </a>
            </td>
          <td>
            Atia B. Amin, A. Ibarra-Meneses, Mathieu Blanchette, Christopher Fernández-Prada, David Langlais
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is an aggressive malignancy associated with single-nucleotide variants in VHL, PBRM1, and SETD2. However, the role of structural variation (SV)—which have broader genomic impacts—and 3D genome architecture in ccRCC remains inadequately understood. Here, we reported a comprehensive molecular characterization. Through multi-omics analysis, we identify novel SV-associated oncogenic targets and reveal multidimensional 3D genome reorganization during ccRCC progression. We elucidate the dynamic interplay between SV and 3D chromatin architecture, demonstrating how structural rearrangements drive oncogenic dysregulation through 3D genome reorganization. Notably, an unrecognized pathogenic enhancer hijacking event is discovered and experimentally validated, leading to constitutive activation of the proto-oncogene SEMA5B. Furthermore, we developed a machine learning-based prognostic framework using enhancer hijacking signatures. Collectively, this work establishes a valuable resource for ccRCC research by elucidating how SVs and 3D genome reorganization collectively drive oncogenesis, and translates these findings into a clinically applicable prognostic tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f8e6235c583e0191c138b2862bedc511ab3f80" target='_blank'>
              Structural variation drives enhancer hijacking via 3D genome disruption in ccRCC
              </a>
            </td>
          <td>
            Yu Dong, Wenjiao Xia, Zitong Yang, Liangliang Ren, Hongru Wang, Yiyang Zhou, Qinchen Li, Zhi Chen, Zhinan Xia, Yichun Zheng, Feifan Wang, Ning He, Bing Cheng, Dongmei Ma, Wei Shao, Wei Guo, Shuwen Wang, Ziqiao Liu, Junxiao Shen, Yiming Qi, Xuke Gong, Juan Jin, Bo Xie, Guixin Zhu, Cheng Zhang
          </td>
          <td>2026-01-06</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chromosome 3p (chr3p) is frequently deleted in multiple cancers, indicating the presence of shared tumor suppressors. Analysis of genomic alterations in 33 different cancer types implicates the deletion or deleterious mutations of SET-domain-containing 2 (SETD2) at chr3p21 in significantly facilitating the formation of isochromosomes, consisting of two identical mirror-imaged arms, thereby promoting genomic instability conducive to large-scale chromosomal rearrangements and rapid cancer genome evolution. Fracturing of dicentric isochromosomes during cell division is pervasive and follows the dynamic fragmentation pattern of solids under impulse. Across cancers, isochromosomes form most frequently on chr8 to amplify the MYC-containing q-arm and chr17 to delete the TP53-containing p-arm. In the most aggressive uveal melanoma (UVM) subtype, chr3 deletion also includes MITF, a critical melanocyte differentiation and survival factor, and co-occurs with chr8q amplification. We demonstrate that MITF is a master transcriptional regulator of GNAQ/GNA11, mutated in 90% of UVM patients, and the associated synthetic-lethal genes identified by recent CRISPR screening studies. MITF maintains MAPK and calcium homeostasis in UVM, and its deletion is thus accidental, creating an early crisis during oncogenesis. We further show that MITF, MYC, and GNAQ/GNA11 form coupled regulatory feedback loops in the melanocyte lineage, and MITF deletion in UVM creates acute dependency on MYC-mediated rescue via chr8q amplification, often as a consequence of isochromosome formation. The discovered feedback loops predict both overall and relapse-free patient survival within the most aggressive UVM subtype, explain sensitivity to therapeutic gene perturbations, and inform effective combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f31378489089eb1239e625480e1522a21f193cf" target='_blank'>
              Chromosome 3p deletion leads to extensive genomic alterations in diverse cancers and confers synthetic lethality in uveal melanoma
              </a>
            </td>
          <td>
            Mitchell C. Cutler, P. B. Howland, M. Hejna, Jun S. Song
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) amplification represents an emerging mechanism underlying oncogene amplification, tumor heterogeneity, and drug resistance in cancer. However, the biology of ecDNA remains poorly understood because tools to engineer ecDNAs and precisely monitor their dynamics are limited. In particular, genome engineering strategies have not been established for ecDNA, which exists in tens to hundreds of copies within a single cell. Here, we report a systematic validation of ecDNA editing using standard CRISPR-Cas9 system and optimized CRISPR-Cas9 system with safeguard single-guide RNAs (sgRNAs), in which the addition of cytosine extensions finely reduces excessive Cas9 activity. The conventional CRISPR-Cas9 system induced severe cytotoxicity and markedly reduced ecDNA copy number, together with frequent micronucleus formation. Knock-in efficiency was remarkably low, highlighting an intrinsic difficulty in editing ecDNA. In contrast, the safeguard sgRNA strategy not only alleviated cytotoxicity and ecDNA loss in a cytosine-length–dependent manner but also enabled efficient knock-in into multiple ecDNA per cell. Computational simulations suggested that the degree and temporal patterns of multiple DNA cleavage events shape cell death, micronucleus formation, and rapid expansion of knock-in ecDNA. Collectively, optimization of Cas9 activity using safeguard sgRNAs enables efficient and nondisruptive ecDNA engineering, providing a powerful tool to study ecDNA biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85607d9d15aeb32494567c03b21204776a014e41" target='_blank'>
              Optimized CRISPR-Cas9 system for efficient engineering of ecDNA in cancer cells
              </a>
            </td>
          <td>
            Yohei Sugimoto, Takeru Kachi, Yu Watanabe, Mei Kubokawa, Koichi Ogami, Masaki Kawamata, Seiko Yoshino, Hiroshi I. Suzuki
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are circular DNA fragments frequently found in human cancers, where they amplify oncogenes, drive tumor heterogeneity, and promote therapy resistance and poor prognosis. Despite their prevalence, how ecDNAs interact with the immune system remains poorly understood. Here, we show that the cytosolic DNA sensor cGAS detects ecDNA fragments in the cytoplasm and activates the innate immune response. cGAS and STING are frequently silenced in ecDNA+ tumors through promoter hypermethylation. Restoring cGAS or STING in human and murine ecDNA+ cancer cells reactivates innate immune signaling and selectively suppresses ecDNA+ tumor growth in an immunocompetent mouse model. Using two ecDNA biogenesis models, we show that the cGAS-STING pathway restricts de novo ecDNA formation. Together, our findings identify innate immune sensing as a natural barrier to ecDNA-driven oncogenesis and establish cGAS-STING reactivation as a therapeutic strategy for ecDNA+ cancers. Highlights The cGAS-STING pathway is frequently silenced in ecDNA+ tumors Restoration of cGAS in ecDNA+ cells activates innate immune responses cGAS expression suppresses ecDNA+ tumor growth in vivo The cGAS-STING pathway restricts de novo ecDNA biogenesis">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/663b6039e119f140e6b2d7e0ab0731409522972e" target='_blank'>
              Innate immune sensing via the cGAS-STING pathway restricts extrachromosomal DNA–driven tumorigenesis
              </a>
            </td>
          <td>
            Tuo Li, Qing-Lin Yang, Kailiang Qiao, Anli Zhang, Chenglong Sun, Huocong Huang, P. Mischel, Sihan Wu, Zhijian J. Chen
          </td>
          <td>2026-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="TFE3 translocation renal cell carcinoma (tRCC), an aggressive kidney cancer driven by TFE3 gene fusions, is frequently misdiagnosed owing to morphologic overlap with other kidney cancer subtypes. Conventional liquid biopsy assays that detect tumor DNA via somatic mutations or copy number alterations are unsuitable for tRCC since it often lacks recurrent genetic alterations and because fusion breakpoints are highly variable between patients. We reasoned that epigenomic profiling could more effectively detect tRCC because the driver fusion constitutes an oncogenic transcription factor that alters gene regulation. By defining a TFE3-driven epigenomic signature in tRCC cell lines and detecting it in patient plasma using ChIP-seq, we distinguished tRCC from clear-cell RCC (AUC = 0.86) and samples of individuals without evidence of cancer (AUC = 0.92) at low tumor fractions (<1%). This work establishes a framework for noninvasive epigenomic detection, diagnosis, and monitoring of tRCC, with implications for other mutationally quiet, fusion-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181d702745222ebeaadeb7b01c3a274dc1851478" target='_blank'>
              Cell-free DNA epigenomic profiling enables noninvasive detection and monitoring of translocation renal cell carcinoma
              </a>
            </td>
          <td>
            S. Garinet, Karl Semaan, Jiao Li, Ze Zhang, Prathyusha Konda, A. Sadagopan, J. Canniff, Noa Phillips, K. Klega, Medha Pandey, H. Savignano, Matthew P Davidsohn, Kevin Lyons, Alessandro Medda, Prateek Khanna, M. Achom, B. Fortunato, Rashad Nawfal, Razane El Hajj Chehade, J. O’Toole, J. Horst, D. Freeman, Rachel Trowbridge, C. Chau, William D. Figg, Jacob E. Berchuck, Brian D. Crompton, J-H. Seo, T. Choueiri, M. Freedman, S. Baca, S. Viswanathan
          </td>
          <td>2026-02-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac603fef64d1ae37e58513b3a4b599d750808d42" target='_blank'>
              XAI-ecDNA: A Proof-of-Concept Framework for Explainable Multi-Modal Extrachromosomal DNA Detection Using Synthetic FISH and Genomics
              </a>
            </td>
          <td>
            Asim Manzoor, Minahal Amin
          </td>
          <td>2026-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Over 15% of cancers worldwide are caused by viruses. Merkel cell polyomavirus (MCPyV) is the most recently discovered human oncovirus and is the only polyomavirus that drives malignant tumors in humans. Here, we show that MCPyV+ Merkel cell carcinoma is defined by neuroendocrine-lineage core regulatory (CR) transcription factors (TFs) (ATOH1, INSM1, ISL1, LHX3, POU4F3, and SOX2) that were essential for tumor survival and that co-bound chromatin with the viral small T antigen at super enhancers. Moreover, MCPyV integration sites were enriched at these neuroendocrine super enhancers. We further discovered that the MCPyV noncoding control region contained a homeodomain binding motif absent in other polyomaviruses that bound ISL1 and LHX3 and depended on them for T antigen expression. To therapeutically target the CR factors, we used histone deacetylase (HDAC) inhibitors to collapse the chromatin architecture and induce topological blurring of superenhancer loops, abrogating core TF expression and halting tumor growth. To our knowledge, our study presents the first example of oncogenic cross-regulation between viral and human epigenomic circuitry to generate interlocking and essential transcriptional feedback circuits that explain why MCPyV causes neuroendocrine cancer and represent a tumor dependency that can be targeted therapeutically.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb14a3c17daae13a9f8ee1cd3546f3c3832ec32" target='_blank'>
              Interlocking host and viral cis-regulatory networks drive Merkel cell carcinoma
              </a>
            </td>
          <td>
            L. Miao, D. Milewski, Amy Coxon, T. Gelb, K. Garman, Jadon Porch, Arushi Khanna, L. Collado, Natasha T. Hill, Kenneth Daily, S. Vilasi, Danielle Reed, Tiffany Alexander, G. Starrett, Maharshi Chakraborty, Young Song, Rachel Choi, Vineela Gangalapudi, Josiah D. Seaman, Andrew Morton, K. Busam, C. Vakoc, Daniel J. Urban, Min Shen, Matthew D. Hall, Richard Sallari, Javed Khan, B. Gryder, Isaac Brownell
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>2</td>
          <td>98</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c21032e04b711532f2c1af652faef7052c5a06e" target='_blank'>
              PanCNV-Explorer: Deciphering copy number alterations across human cancers
              </a>
            </td>
          <td>
            Nadine S. Kurz, Kevin Kornrumpf, Anna-Rosa Krüger, Jürgen Dönitz
          </td>
          <td>2026-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The human genome is dominated by noncoding sequences, most of which are poorly conserved across species. How genetic information is distributed between coding and noncoding regions remains a fundamental unresolved question. Using CRISPR saturation mutagenesis at base-pair resolution, we mapped the functional fitness landscape of the 10-kb human MYC locus with a near-PAMless, high-fidelity SpRY-Cas9. This unbiased interrogation revealed that the majority (67%) of functionally essential base-pairs in this locus are noncoding. Paradoxically, the phenotypic impact of noncoding sequences correlates inversely with evolutionary conservation, driven in part by rapidly diverging cis-regulatory DNA elements that remain functionally constrained in humans. Within this landscape, we identified an ultraconserved RNA element in the 3’ untranslated region (UTR) that is indispensable for MYC-dependent cancer cells. Remarkably, steric-blocking antisense oligos targeting this RNA element selectively eliminate MYC-addicted cancer cells by suppressing MYC function without reducing MYC abundance. Mechanistically, this 3′ UTR element promotes perinuclear localization of MYC mRNA and efficient nuclear import of the short-lived MYC protein, enabling its function as a nuclear transcription factor. Together, these findings highlight noncoding sequences as major carriers of functional genetic information, provide a comprehensive fitness map of the MYC locus, and uncover a therapeutically actionable RNA element that disables MYC-driven cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9f715a277680bacb435d2c9602b972a237e4aca" target='_blank'>
              Decoding the MYC locus reveals a druggable ultraconserved RNA element
              </a>
            </td>
          <td>
            Peiguo Shi, Feiyue Yang, Fnu Tala, Wesley Huang, Alexis O. Aparicio, Colin H. Kalicki, Aditi Trehan, Michael R. Murphy, Esther Rotlevi, Linqing Xing, Muredach P. Reilly, Jianwen Que, Xuebing Wu
          </td>
          <td>2026-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Eukaryotic genomes are abundant in satellite DNA (satDNA): large blocks of tandemly-repeated sequences that accumulate in heterochromatic genome regions. SatDNAs are dynamic in their genomic location and abundance across species. Some satDNAs overlap essential genome regions such as centromeres and telomeres, but even pericentromeric satDNA can have effects on phenotypes, raising questions about their functional significance. However, it remains unclear whether these effects depend on satDNA sequence, copy number, higher-order structural organization, or genomic context. The highly repetitive nature of satDNA arrays has long hindered detailed genomic and genetic analyses. Recent advances in long-read sequencing now facilitate both the detailed characterization of satDNA structure and the development of more targeted approaches to genetic analysis. Here we present a sequential CRISPR/Cas9-based strategy to make mutations in satDNA arrays and demonstrate its utility using an autosomal pericentromeric satDNA in Drosophila melanogaster called Responder (Rsp). Rsp is the target of a sperm-killing male meiotic driver, Segregation Distorter (SD), where sensitivity to sperm killing positively correlates with Rsp copy number. Using our CRISPR/Cas9 approach, we generated an allelic series of Rsp deletion and expansion variants in two genetic backgrounds and examined their effects on spermatogenesis. Our approach produced precise satDNA variants efficiently, with minimal detectable off-target effects. The resulting mutations affect sensitivity to SD that scale with Rsp copy number. This work establishes a new framework for experimentally dissecting satDNA function and provides insights into the evolutionary and functional roles of satDNA in genome organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f2bac6b9fbac768b087852174f053e8622b039f" target='_blank'>
              Targeted editing of pericentromeric satellite DNA alters sensitivity to meiotic drive
              </a>
            </td>
          <td>
            D. Eickbush, J. Rahmat, M. Lindsay, J. Bowers, N. Fuda, A. Larracuente
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94cb28bdcaf79e7d74e6f5d2dbc03edf13bb5c93" target='_blank'>
              SMAD4 loss drives cancer chromosomal instability through transcription-replication conflict-induced replication stress
              </a>
            </td>
          <td>
            Atmika Paul, Gargee Joshi, Philipp Jungk, Simranjeet Kaur, Shengyu Yao, Ioannis Tsirkas, N. Böhly, Maren Sitte, Gabriela Salinas, Stephan Hamperl, Maik Kschischo, Holger Bastians
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mutations in epigenetic regulators are common in bladder cancer, yet their impact on therapeutic responses remains unclear. Here, we identify that loss-of-function mutations in KDM6A, a histone demethylase altered in about 26% of advanced bladder cancers, are associated with poor survival after cisplatin chemotherapy, whereas they correlate with improved outcomes with anti-PD-1 therapy. Using CRISPR-Cas9-engineered murine and human bladder cancer models, we show that KDM6A deficiency increases formation of extrachromosomal circular DNA carrying chemoresistance loci, promoting cisplatin resistance. In parallel, KDM6A loss impairs DNA repair and rewires tumor metabolism, reducing glycolysis and lactate output. This metabolic shift diminishes histone lactylation in regulatory T cells, suppressing immunoregulatory genes and limiting expansion of PD-1hi regulatory T cells. Collectively, our findings establish KDM6A mutation as a key regulator of therapeutic responses, providing a foundation for its use in guiding precision therapy in advanced bladder cancer. Genes encoding key epigenetic regulators, including Lysine Demethylase 6A (KDM6A), are frequently mutated in bladder cancer. Here, the authors show that loss of KDM6A promotes formation of extrachromosomal circular DNA (eccDNA), genomic instability, and metabolic reprogramming, driving resistance to cisplatin chemotherapy while simultaneously enhancing sensitivity to immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9139b0ede19d5b7c3ea40131744831c88cd696ed" target='_blank'>
              Loss of KDM6A-mediated genomic instability and metabolic reprogramming regulates response to therapeutic perturbations in bladder cancer
              </a>
            </td>
          <td>
            Pratishtha Singh, R. D’Rozario, Bidisha Chakraborty, Swadhin Meher, D. Raychaudhuri, Aminah J. Tannir, Yang Li, Anurag Majumdar, Jessalyn Hawkins, Yun Xiong, Philip L. Lorenzi, Padmanee Sharma, K. Akdemir, P. Pilié, Abhinav K. Jain, Byron H Lee, Sangeeta Goswami
          </td>
          <td>2026-01-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Cancer driver mutations alone are often insufficient to fully explain tumorigenesis. We demonstrate that these mutations cooperate with somatic copy number variations (CNVs) in a tissue-specific pattern of genomic epistasis. Analyzing 93,462 tumors, we identified 54 gene-cancer type pairs with significant co-occurrence of somatic mutations and CNVs. Our new Binoculars algorithm, which resolved phased DNA/RNA reads, revealed frequent preferential amplification in oncogenic mutation alleles, including AKT1 p.E17K, BRAF p.V600E, KRAS p.G12C/D/V, NRAS p.Q61K, and a fraction of gain-of-function TP53 p.R175H. Conversely, deletions selectively targeted the reference alleles, leading to loss of heterozygosity of IDH1 p.R132H and tumor suppressor mutations, including CDKN2A and TP53 truncations. Lung cancer patients carrying co-occurrences of somatic mutation-CNVs in TP53 and KRAS showed poorer survival than those carrying the same gene mutations. These findings reveal epistasis of cancer mutations and CNVs at an allelic resolution, suggesting specific genomic events to enhance patient stratification and therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c67a7d5e6bdabbcb4e758c260b7827b1f0b38f4" target='_blank'>
              Widespread Epistasis between Cancer Driver Mutations and Allele-Specific Copy Number Variations
              </a>
            </td>
          <td>
            Serge Merzliakov, Guanlan Dong, Andrea Castro, Mahad Bihie, Yve Nichols-Evans, Hanna Carter, Kuan-lin Huang
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Enhancers have been proposed to act as privileged loading sites for cohesin, raising the idea that they actively fold the genome to engage distal target promoters for transcription. Supporting this idea, NIPBL/MAU2, which is required for cohesin loading, binds at enhancers in mouse embryonic stem cells. However, we find that driving cohesin recruitment near an enhancer strongly inhibits transcription from its target distal promoter, indicating that strong focal cohesin loading at enhancers is not compatible with their long-range regulatory functions. Quantitative experiments and biophysical modeling further indicate that cohesin loading at enhancers does not make major contributions to genome-wide cohesin binding and chromosome folding patterns. Instead, cohesin must load throughout the genome to extrude it, regardless of enhancer proximity, with the major determinants of cohesin traffic being extrusion barriers such as transcription and clustered CTCF sites. These findings indicate that enhancer function is ancillary to the general mechanisms of chromosome folding, informing further study of the relationship between genome architecture and transcriptional regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0804135380a756dde59061bbd3010283fb083527" target='_blank'>
              The role of cohesin loading at enhancers in the flux of loop extrusion and long-range transcriptional control
              </a>
            </td>
          <td>
            Erika C. Anderson, H. Rahmaninejad, Abrar Aljahani, Emily M. Arnold, Annie S. Adachi, Rini Shah, Karissa L. Hansen, Ivana Čavka, A. Boettiger, Geoffrey Fudenberg, Elphѐge P. Nora
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background: Long INterspersed Element-1 (LINE-1) retrotransposons comprise 17–20% of the human genome. These retroelements are normally silenced early in embryonic development through epigenetic mechanisms and reawakened during oncogenesis, leading to transcriptional dysregulation, genomic instability, and immune evasion. Methods: In the present study, we categorized LINE-1 transcripts across 121 non-small cell lung cancer (NSCLC) cell lines from the Cancer Cell Line Encyclopedia (CCLE) by subfamily, length, orientation, chromosomal origin, and distribution. In addition, high-prevalence insertions were mapped to nearby genes to assess potential functional interactions. Results: LINE-1 transcript abundance and length in NSCLC were dominated by evolutionarily young subfamilies, particularly L1HS and L1PA2 through L1PA5. Chromosomal patterns were conserved across NSCLC subtypes, with modest enrichment of L1HS activity on Chromosome 4 and the X Chromosome. The lung squamous cell carcinoma (LSQCC) subtype exhibited the highest total levels of L1HS expression relative to other NSCLC subtypes. Race modestly influenced LINE-1 transcript abundance, with cell lines derived from self-identified African American individuals showing elevated overall LINE-1 and L1HS expression. Age showed a weak positive correlation with total LINE-1 abundance. Integrative analysis revealed recurrent hotspots at 22q12.1 and 20p11.21 that were transcriptionally active across subtypes and coincided with previously reported intact LINE-1 elements active in epithelial cancers. Recurrent insertions were located near cancer-associated genes, including RB1, NEDD4, FTO, LAMA2, NOD1, and KCNB2, implicating LINE-1 activity in cis-regulatory remodeling of oncogenic pathways. Conclusions: Together, these findings indicate that LINE-1 transcript heterogeneity in NSCLC is shaped by host genomic architecture and conserved functional hotspots, providing new insights into the mechanisms of genetic and epigenetic dysregulation associated with LINE-1 retroelements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a76a2793974308abd948a4e791ee9224ae6f4f7f" target='_blank'>
              LINE-1 Transcript Heterogeneity in Non-Small Cell Lung Cancers Is Driven by Host Genomic Context and Conserved Functional Hotspots
              </a>
            </td>
          <td>
            Yingshan Wang, Kenneth S. Ramos
          </td>
          <td>2026-01-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The short arms of human acrocentric chromosomes are characterized by nucleolar organizer regions essential for ribosome biogenesis, but their highly repetitive nature has hindered genomic analysis. Leveraging the recently completed genomes of all major ape lineages, we identified recurrent features of their acrocentrics, including enriched repeat classes, centromere repositioning by whole-arm inversion, interchromosomal sequence exchange, and birth-and-death evolution of multiple gene families. Together, these processes have enabled the repeated amplification and diversification of the FRG1 gene family over 25 million years of ape evolution, and, in gorilla, the formation and amplification of a novel IGSF3-GGT fusion gene under positive selection. Similar evolutionary events also explain the distribution of segmental duplications and heterochromatin in the modern human genome, predisposing it to karyotypic abnormalities such as Robertsonian translocations. Our findings highlight acrocentric chromosomes as key drivers of evolution in the great apes, with implications for speciation, adaptation, and clinical genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d00257fab76c105ca582971affa83606fdc93ba" target='_blank'>
              Origin and evolution of acrocentric chromosomes in human and great apes
              </a>
            </td>
          <td>
            Steven J. Solar, Prajna Hebbar, L. G. de Lima, Alex Sweeten, A. Rhie, T. Potapova, Luciana de Gennaro, A. Guarracino, Juhyun Kim, Brandon D. Pickett, Benedict Paten, Melissa A. Wilson, S. Koren, Erik Garrison, E. Eichler, M. Ventura, J. Gerton, A. Phillippy
          </td>
          <td>2025-12-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Elevated levels of extrachromosomal DNAs (ecDNAs) are associated with poor prognoses of many cancer types. These large circular DNAs typically harbour oncogenes and regulatory elements which, together with high levels of ecDNA transcription, confer a growth advantage to cancer cells. Replication of ecDNAs, followed by their unequal distribution at mitosis, further promotes rapid cancer evolution. In contrast to ecDNAs, the role of circular DNA by-products from V(D)J recombination in cancer development has largely been overlooked. Developing lymphocytes generate millions of excised signal circles (ESCs) each day through gene rearrangement at the immunoglobulin and T-cell receptor loci. Despite their similar size to ecDNAs, ESCs were long assumed to be inert and lost during cell division. However, it is now known that ESCs potently trigger genome instability when complexed with recombinase proteins. Not only this, but new data show that just like ecDNAs, ESCs replicate and persist, with high levels strongly correlating with poor prognosis of B-cell precursor acute lymphoblastic leukaemia (BCP-ALL). Despite these striking similarities, the properties of ESCs and ecDNAs are seldom linked. Here, we provide the first comparative review of ecDNAs and ESCs, and highlight the reasons why these molecules are more closely related than once assumed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec100c92fb04b0426668a2656e80f3ac4a05ad94" target='_blank'>
              The newfound relationship between extrachromosomal DNAs and excised signal circles.
              </a>
            </td>
          <td>
            Dylan Casey, Zeqian Gao, J. Boyes
          </td>
          <td>2026-01-03</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Introduction Cancer cell proliferation occurs within the context of persistent genomic instability. In this review, we propose the RS–CCD–CIN axis as a systems-level framework in which replication stress (RS), cell cycle deregulation (CCD) and chromosomal instability (CIN) form an interdependent triad that shapes tumour evolution. This axis represents a constrained metastable state in which genomic instability is tolerated and buffered. The objective of this review is to synthesize the current understanding of how the RS–CCD–CIN axis contributes to tumour heterogeneity, adaptability and therapy response. Discussion Evidence indicates that RS, CCD and CIN operate as a dynamic, interconnected network rather than as independent processes. Replication stress induces DNA damage and mutagenesis, while partial checkpoint disruption permits cells with unresolved lesions to proliferate. Chromosomal instability generates both structural and numerical alterations, contributing to intratumoural heterogeneity. Together, these processes facilitate adaptation to environmental and therapeutic pressures. Extrachromosomal DNA, micronuclei formation and cytosolic DNA signalling, including the cGAS–STING pathway, connect genomic instability to adaptive responses and immune modulation. Single-cell and spatial profiling reveal temporal and spatial variability in RS, CCD and CIN states, highlighting the limitations of static biomarkers. Therapeutically, targeting individual components often yields limited durability, whereas approaches that simultaneously perturb multiple aspects of the RS–CCD–CIN axis may improve clinical outcomes. Conclusions This review highlights the RS–CCD–CIN axis as a fragile and metastable architecture that supports cancer evolution, while also being susceptible to collapse. A deeper understanding of this interconnected framework may inform the development of therapeutic strategies and enhance the management of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/294582a14e94664c9f90d67cf0b0db4402660bea" target='_blank'>
              A system-level metastable model of cancer evolution: integrating replication stress, cell cycle deregulation and chromosomal instability
              </a>
            </td>
          <td>
            Dongjing Ma, Jinbin Wang, Yicong Chen, Ling Yao, Qianya Wei, Jianjun Wu
          </td>
          <td>2026-01-28</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf830ded86f6aa48fcef1356b0883f8c090233e1" target='_blank'>
              Patterns of hypermutation shape tumorigenesis and immunotherapy response in mismatch-repair-deficient glioma
              </a>
            </td>
          <td>
            Nick Fernandez, Yuan Chang, N. Nunes, J. Dimayacyac, A. Levine, Amit Ringel, L. Negm, A. B. Ercan, J. Hess, Olfat Ahmad, Caitlin Lee, L. Stengs, V. Bianchi, M. Edwards, Sheradan Doherty, J. Chung, L. Nobre, Julie Bennett, A. Dodgshun, David T. W. Jones, S. Pfister, A. Villani, David Malkin, Vijay Ramaswamy, A. Huang, É. Bouffet, M. Aronson, Peter B. Dirks, A. Shlien, Gaddy Getz, Y. Maruvka, B. Ertl-Wagner, Cynthia E. Hawkins, Anirban Das, U. Tabori
          </td>
          <td>2025-12-22</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74e725a85a73539cf9bc922a83eb56481b485413" target='_blank'>
              Pan-cancer silencer transcription atlas reveals leukemia silencer hijacking FOXP1 to enhance MYC oncogene
              </a>
            </td>
          <td>
            M. Guo, Liutao Chen, Shengcheng Deng, Jingkai Zhang, Zhijie Hu, Weiwen Wang, Songyang Zhou, Yuanyan Xiong
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/450603a806f69ebb4194f3b7b3ab19ed0a5b5513" target='_blank'>
              A holocentric pangenome links karyotype evolution to meiotic recombination
              </a>
            </td>
          <td>
            Meng Zhang, Stefan Steckenborn, Marco Castellani, Laia Marín-Gual, A. S. Câmara, Laura A. Robledillo, Letícia Maria-Parteka, N. Sargheini, Magdalena Marek, Eduardo Chacón-Madrigal, Andrés Gatica Arias, L. Felix, William W. Thomas, B. Huettel, A. Pedrosa‐Harand, John T. Lovell, André L. L. Vanzela, Aurora Ruiz-Herrera, A. Marques
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Ribosomal DNA (rDNA) encodes the 18S, 5.8S, and 28S rRNA, accounting for ∼70% of cellular transcription. Despite its essential role and links to cancer and aging, quantifying rDNA instability in mammals remains challenging due to its repetitive organization and inherent heterogeneity. Here, we developed a murine rDNA FISH probe and genomic tools tailored for laboratory mouse strains. The results confirmed rDNA cluster locations, revealed substantial inter- and intra-strain as well as intercellular heterogeneity in rDNA organization within inbred mice and unstressed cells, and identified sources of spontaneous and replication-associated DNA double-strand breaks in the rDNA transcription termination region. Using mouse embryonic stem cells, we showed that BRCA1-mediated homologous recombination promotes rDNA instability, the non-homologous end joining factor XRCC1, but not Ku, suppresses intra-cluster deletions, and ATM kinase preserves rDNA cluster stability. Together, these findings establish a platform and tools for studying rDNA instability in animal models relevant to aging and cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/377fa5f7328f7f52cd2825bb12b7a086a7b2a641" target='_blank'>
              Visualization and quantification of rDNA instabilities in mammalian cells and mouse models
              </a>
            </td>
          <td>
            Xiaolu Zhu, Wenxia Jiang, Wei Wu, Brian J. Lee, Demis Menolfi, Anthony Tubbs, Olivia M. Cupo, Eli Malkovskiy, Mattie Nester, Xiaobin S. Wang, Peter A. Sims, Chyuan-Sheng V Lin, Lorraine Symington, Andre Nussenzweig, Brian McStay, Shan Zha
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Human artificial chromosomes (HACs) are inherited through cell divisions alongside natural chromosomes, serving as tools for interrogating chromosomal elements and as vectors for large genetic cargoes (1–5). Despite recent progress (6–8), large (i.e., multiple Mb) HACs have not been reported. Further, HAC formation via epigenetic seeding of centromeric chromatin currently requires prior engineering of recipient human cells (6–8), hampering their potential deployment in many useful cell types. Here, we designed, built, and delivered to human cells a 2 Mb HAC construct that is ∼3 times larger than the prior generation. We also report a robust epigenetic centromere seeding approach that initiates immediately upon delivery to the human cell cytoplasm and bypasses genetic engineering of target cells. The HACs are then faithfully inherited in the absence of selection. Thus, formation of functional centromeric chromatin in the same cell cycle of HAC delivery drives high efficiency HAC formation. Teaser A system for efficient protein-only centromere seeding permits visualizing the first step of HAC formation">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/755b604be820a14cd765634bf7256cd13a7ee21b" target='_blank'>
              Protein-only centromeric chromatin assembly streamlines human artificial chromosome formation
              </a>
            </td>
          <td>
            Gabriel J. Birchak, P. K. Allu, Prakriti Kashyap, G. K. Anbalagan, Shu-Cheng Chuang, Glennis A. Logsdon, Daniel G. Gibson, John I. Glass, Ben E. Black
          </td>
          <td>2026-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638cde0ab64b6b068ce716defb37f6b2cb4ab7b9" target='_blank'>
              Unraveling the molecular mechanisms underlying spontaneous multipolar mitosis through CIN-seq
              </a>
            </td>
          <td>
            Pin-Rui Su, Ting-Chun Chou, M. T. López-Cascales, Junfeng Hunag, Tsai-Ying Chen, Sjoukje van Wieren, Chan Li, Cecile Beerens, Li You, J. Storteboom, M. Chien
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7695d6c89af83065453f86bff9d21aad9be71838" target='_blank'>
              A distal CTCF-binding site drives MYC expression plasticity in a negative feed-forward loop
              </a>
            </td>
          <td>
            Chenying Gao, M. Martino, Ayushi Chaurasiya, F. Casagrande, Jia Pei Lim, Yuri S. van Ekelenburg, Wei Feng, Yan Liu, Barbara A. Scholz, Pragyan Mishra, Mallikarjuna Thippana, Dennis Torkornoo, Phong KC Chau, Rabia Khan, T. Tannaes, I. Tzelepis, Ilyas Chacoua, Natalie Geyer, S. Meltzer, A. H. Ree, Marco Gerling, Johan Hartman, Narsis Kiani, Rolf Ohlsson, A. Göndör
          </td>
          <td>2026-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fc0251706b1f215af0fc6511724feaeecccbbd1" target='_blank'>
              Wakhan: reconstruction of chromosome-scale copy number profiles of tumor genomes with long-read sequencing
              </a>
            </td>
          <td>
            T. Ahmad, A. Keskus, S. Aganezov, A. Goretsky, I. Rodriguez, B. Yoo, L. Lansdon, E. Repnikova, L. Zhang, Y. Liu, A. Donmez, A. Bryant, S. Tulsyan, J. Park, J. Gardner, B. McNulty, S. Sacco, J. Shetty, Y. Zhao, B. Tran, S. Malikić, C.-P. Day, K. Miga, B. Paten, C. Sahinalp, M. Farooqi, M. Dean, M. Kolmogorov
          </td>
          <td>2025-12-15</td>
          <td>None</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="MYC dysregulation occurs in most cancers provoking, profound changes in gene expression. Although sometimes considered to be an E-box-dependent transcription factor, an alternate model posits MYC to be a universal amplifier of active genes. Although MYC is associated with accelerated pause release, the full extent of its participation in the transcription cycle remains poorly illuminated. MYC also stimulates topoisomerases to resolve topological issues that complicate DNA transactions; whether and how this stimulation is coordinated with or independent of its transcriptional role has not been examined. We have developed a genetic tool that discriminates between MYC’s ability to activate versus amplify reporter gene expression in any cell. This system enables the interrogation of genes, cofactors, and compounds that execute or modulate MYC activity. Using a combination of biochemical and cellular assays, we reveal the dynamic interplay between MYC-driven transcription amplification and DNA supercoiling. The early stages of transcription are highly sensitive to the level of DNA supercoiling. Reducing the activity of TOP1 either through genetic knockdown or low-dose inhibition potentiates transcription amplification by MYC. This enhancement is associated with pre-initiation complex (PIC) stabilization by DNA supercoiling. MYC helps to mobilize these stabilized PICs. Significance Statement MYC is dysregulated in most cancers, yet its precise functions in transcription are poorly understood. Using a genetic system that distinguishes transcriptional activation from amplification, we reveal a functional interplay between MYC-driven amplification and DNA supercoiling. Reduced topoisomerase I activity increases DNA-supercoiling that in turn stabilizes PICs to provide more substrate for MYC-dependent transcription amplification. Thus, DNA topology as a critical regulator of MYC-dependent transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a0f98bd86315e8e5aa3b48daf84b94980e7dd61" target='_blank'>
              Dynamic Supercoiling Sponsors Transcription Amplification by MYC
              </a>
            </td>
          <td>
            R. Jha, Fedor Kouzine, Bo Wang, James D. Phelan, S. K. Das, Brian A Lewis, D. Levens
          </td>
          <td>2026-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="In a recent study published in Nature, Cosenza et al. 1 introduced MAGIC (machine-learning-assisted genomics and imaging convergence), a versatile platform that photolabels cells with micronuclei and other nuclear atypia, enabling direct quanti ﬁ cation of de novo chromosomal abnormality (CA) formation and ongoing chromosomal instability (CIN) at single-cell resolution. Using this framework, the authors show how spontaneous and induced lesions from pre-mitotic DNA damage and mitotic errors drive distinct CIN trajectories, with TP53 inactivation markedly amplifying CA burden and complexity, highlighting how CIN emergence and propagation prior to clonal expansion may inform tumour initiation, progression and the identi ﬁ cation of actionable biomarkers and therapeutic vulnerabilities. CIN is the ongoing process by which cells acquire new CAs through repeated chromosome missegregation and structural rearrangements, altering chromosome number and structure across divisions (Fig. 1a). CAs are pervasive across cancers and typically affect more genes than substitutions or small insertion/ deletion alterations, fuelling clonal diversi ﬁ cation and tumour adaptation. 2 – 5 Accurate chromosome segregation and genome integrity rely on coordinated processes such as DNA replication and repair, telomere maintenance, spindle assembly and mitotic exit. 2 When these are compromised and cells bypass senescence and apoptosis, concurrent pre-mitotic and mitotic defects drive ongoing CIN. 2,5 However, most CIN">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0c5eb4ff0382eff914e4d728cb5754b2a5b73b7" target='_blank'>
              It’s a kind of MAGIC: uncovering the origins of chromosomal instability
              </a>
            </td>
          <td>
            D. Gómez-Peregrina, César Serrano
          </td>
          <td>2026-02-05</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6dd597d404f6624bed3a43a3ecd045b834d7d8b" target='_blank'>
              Integration of an anellovirus genome in the SKNO-1 acute myeloid leukemia cell line
              </a>
            </td>
          <td>
            Na Cui, S. Goya, E. Piliper, Alexander L. Greninger
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Hepatocellular carcinoma (HCC) is the most common primary liver malignancy worldwide and is associated with a high mortality rate. The unique genetic features and structural properties of extrachromosomal circular DNA (eccDNA) offer a promising approach for early cancer diagnosis.


OBJECTIVE
This study aims to investigate the potential of eccDNA as a diagnostic marker by identifying its distinguishing characteristics using a two-dimensional cell culture model.


METHODS
EccDNA was isolated from the supernatant of HepG2 and THLE-2 cell cultures. Long-read sequencing and bioinformatic analysis were employed to detect and characterize eccDNA. The size distribution, chromosomal origin, and genomic annotation of eccDNA from both cell types were compared. PCR was performed to validate DNA fragments at the junctions of the closed-circular forms.


RESULTS
The total number of cell-free eccDNA derived from HepG2 cells averaged 2,669,673 bases, encompassing 2,542 sequences, while THLE-2 cells averaged 857,718 bases with 975 sequences. Size distribution analysis revealed that most cell-free eccDNA from both cell types ranged from 351 to 600 bp. Bioinformatic analysis revealed that approximately 80% of eccDNAs corresponded to gene coding regions. Ten eccDNAs were the most frequently detected in HepG2 cells, with four eccDNAs harboring CDC27P11, RAC1P3, LOC112268123, and LOC124902279 genes successfully validated and uniquely detected in HepG2 cells, suggesting their potential as biomarkers for HCC diagnosis.


CONCLUSION
Distinct eccDNA types were identified in HepG2 cells, which may serve as promising biomarkers for HCC diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2857a0fc338d1571175ba6c4c210b5ca1dafdfd9" target='_blank'>
              Identification of Extrachromosomal Circular DNA Isolated from Cell Culture Supernatant and Its Potential Applications for Hepatocellular Carcinoma Diagnosis.
              </a>
            </td>
          <td>
            Suchitraporn Sukthaworn, Suchanuch Ondee, Opas Choksupmanee, Suparat Taengchaiyaphum, Prasert Yodsawat, Ponsit Sathapondecha
          </td>
          <td>2026-01-21</td>
          <td>Asian Pacific journal of cancer prevention : APJCP</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/308ec9e744b73dd3ecd51f7b2385a3a46667c2cb" target='_blank'>
              Pangenome analysis reveals the evolutionary dynamics of repeat-based holocentromeres
              </a>
            </td>
          <td>
            Piotr Włodzimierz, Estela Perez-Roman, A. S. Câmara, Laura A. Robledillo, Gokilavani Thangavel, Meng Zhang, Jacob González Isa, L. M. Parteka, Marco Castellani, B. Huettel, André L. L. Vanzela, Ian R. Henderson, Alexandros Bousios, A. Marques
          </td>
          <td>2026-01-18</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1248ad827da125e00e0f24cf372ada5d8bfee2" target='_blank'>
              DNA replication errors drive genome-wide small inverted triplication dynamics
              </a>
            </td>
          <td>
            Yi Lei, Yu Zhou, Haitao Sun, Hang Yuan, Xinyu Pei, Jessica D. Hess, Yao Yan, Zunsong Hu, Mian Zhou, Zhaohui Gu, Li Zheng, Xiwei Wu, Binghui Shen
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Changes in the copy number of large genomic regions, termed copy number variations (CNVs), contribute to important phenotypes. CNVs are readily identified using conventional approaches when present in a large fraction of the cell population. However, CNVs in only a few genomes are often overlooked but important; if beneficial, a de novo CNV that arises in a single genome can expand during selection to create a population of cells with novel characteristics. While single cell methods for studying de novo CNVs are increasing, we continue to lack information about CNV dynamics in rapidly evolving microbial populations. Here, we investigated de novo CNVs in the genome of the Plasmodium parasite that causes human malaria. The highly AT-rich Plasmodium falciparum genome readily accumulates CNVs that facilitate rapid adaptation. We employed low-input genomics and specialized computational tools to evaluate the impact of sub-lethal stress on the de novo CNV rate. We observed a significant increase in genome-wide de novo CNVs following treatment with an antimalarial compound that inhibits replication. De novo CNVs encompassed genes from various cellular pathways participating in human infection. This snapshot of CNV dynamics emphasizes the connection between replication stress, DNA repair, and CNV generation in this important microbial pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cc90f4b9d9a1317ece4f68d2ac843d4fc5a32e4" target='_blank'>
              Replication stress increases de novo CNVs across the malaria parasite genome
              </a>
            </td>
          <td>
            Noah Brown, A. Luniewski, Xuanxuan Yu, Michelle Warthan, Shiwei Liu, Julia Zulawinska, Syed Ahmad, Nadia Prasad, Molly Congdon, Webster Santos, Feifei Xiao, J. Guler
          </td>
          <td>2026-01-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/566c090308e26a25e5654df687dc08aebd75d6cb" target='_blank'>
              EZH2 Inhibition Reshapes 3D Chromatin Architecture to Induce Immunogenic Phenotype in Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Sana Parveen, Ramesh Adhinaveni, Kun Fang, Lavanya Choppavarapu, Meijun Du, Gustavo Leone, N. De Sarkar, Victor X. Jin, Hui-Zi Chen
          </td>
          <td>2026-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Osteosarcoma (OS) exhibits substantial genomic complexity and inter-patient heterogeneity, necessitating longitudinal, patient-matched analyses to understand acquired features of tumor evolution. However, most published OS data is limited to primary tumor samples, limiting insight into patient-specific resistance mechanisms. To address this, we characterized the genomic landscape of paired primary and metastatic tumor samples from dogs with spontaneous OS. Whole-genome and single-cell RNA sequencing reveal mutation and gene expression profiles that are predominantly organized by patient identity. Mutational burden and pathway alterations such as those involving PI3K, NOTCH, TP53, MAPK, RAS and epigenetic regulation differ between primary and metastatic samples. Variants present in tumor tissue are readily detectable in paired cfDNA samples, demonstrating the utility of this assay for identifying tumor-specific alterations associated with treatment resistance. Analysis of bulk RNA-seq data to estimate cell-type composition shows greater immune cell representation in metastases, underscoring the importance of immune signaling pathways in OS. These findings exemplify the presence of patient-specific alterations in genomic architecture over the course of tumor progression, linking CNV amplification, pathway reprogramming, and immune evasion in metastatic OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d967dfc80d383c8de7c87cc500998e454e27b0e7" target='_blank'>
              Multi-omic Longitudinal Analysis of Canine Osteosarcoma Identifies Inter-Patient Heterogeneity and Immune Enrichment in Metastatic Lesions
              </a>
            </td>
          <td>
            Christopher Husted, Kerstin Seidel, Tanya T. Karagiannis, Cornelia Peterson, Kate Megquier, Diane Genereux, Jillian M. Richmond, Elinor K. Karlsson, Dave Shulman, Brian Crompton, Cheryl A London, Heather L. Gardner
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="In higher eukaryotic cells, genomic DNA is packaged into dynamic chromatin domains whose physical behavior is coupled to DNA transactions such as transcription and DNA repair. Although chromatin organization is altered in cancer, how oncogenic signals modulate chromatin dynamics over time remains unclear. To address this issue, we established a doxycycline-inducible carcinogenesis model in hTERT-immortalized human RPE-1 cells expressing HPV16 E6/E7, MYC(T58A), and KRAS(G12V) (EMR cells) and investigated chromatin behavior during oncogene-driven transformation. Upon induction, EMR cells displayed accelerated proliferation, loss of contact inhibition, anchorage-independent growth in soft agar, and tumor formation in nude mice. Using time-resolved single-nucleosome imaging to track local chromatin dynamics over days to weeks of oncogene induction, we found that local nucleosome motion was unchanged at 1-3 days, significantly increased at 5-7 days, and returned to parental levels by 4 weeks, despite sustained oncogene expression and stable malignant growth. To explore the basis of this transient increase, we quantified DNA damage, histone marks, and transcription. γH2AX foci were elevated in EMR cells, but ATM/ATR inhibition had only minor effects on local chromatin motion, indicating that the DNA damage response is not the principal driver. By contrast, H3/H4 acetylation and nascent RNA synthesis were upregulated specifically during the early window of enhanced dynamics, whereas the heterochromatin mark H3K9me3 decreased, consistent with transient chromatin loosening associated with increased transcription. These findings reveal a biphasic change in local chromatin dynamics during human oncogene-driven transformation and provide a physical and temporal framework for understanding how oncogenic pathways reorganize chromatin.Key words: cancer, oncogenesis, single-nucleosome imaging, chromatin dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fda7eca5af9dad7ecc50b2fe449402fc658287ea" target='_blank'>
              Single-nucleosome imaging uncovers biphasic chromatin dynamics in inducible human transformed cells.
              </a>
            </td>
          <td>
            Aoi Otsuka, Masa A. Shimazoe, Shigeaki Watanabe, Katsuhiko Minami, Sachiko Tamura, Tohru Kiyono, Fumitaka Takeshita, Kazuhiro Maeshima
          </td>
          <td>2025-12-23</td>
          <td>Cell structure and function</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Genomic instability is a hallmark of cancer and contributes to disease progression through various mechanisms. DNA copy number variation (CNV) is one of the most significant consequences causing a phenomenon called aneuploidy. Aneuploidy is characterized by an imbalanced genome and is associated with lethal progression in prostate cancer. Currently, sequencing-based CNV analysis is widely used to identify cancer-specific aneuploidy that contribute to tumorigenesis, immune evasion, and disease progression. Performing low-pass whole genome sequencing (LP-WGS) on DNA extracted from bulk tissues or cell cultures, is a convenient approach to use for detecting aneuploidy and focal CNVs. However, conventional bulk LP-WGS methods average genomic signals across thousands of cells, often overlooking tumor heterogeneity and small populations with distinct CNV patterns. To address this limitation, we optimized a microfluid nanowell-based system to analyze aneuploidy patterns at a single-cell level using disassociated tissues and isolated nuclei from prostate samples. This pipeline is compatible with both preclinical models and clinical specimens and can be integrated with other multi-omics platforms to generate multimodal datasets for basic and translational research. By leveraging high-throughput single-cell sequencing technologies, our approach enables direct DNA CNV measurement with high sensitivity at a single-cell resolution. Ultimately, this approach aims to improve the resolution of CNV detection, enhance molecular stratification and enable multimodal data integration for better understanding of clinical prostate cancer samples.



 Chloe Springer, Faith Kim, Kathryn Echandía-Monroe, Aditi Shirke, Gustavo Guitierrez-Cruz, Madeline Wong, Kun-Lin Ho, Duanduan Ma, Elise G. DeArment, Amina Ali, William D. Figg, Gregory Chesnut, Matthew G. Vander Heiden, Xiaofeng A. Su. Characterizing aneuploid tumor heterogeneity in prostate cancer through single-cell DNA sequencing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A060.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6642b028922ed7c8904563bae6731f81ce2d7a38" target='_blank'>
              Abstract A060: Characterizing aneuploid tumor heterogeneity in prostate cancer through single-cell DNA sequencing
              </a>
            </td>
          <td>
            Chloe Springer, Faith Kim, Kathryn Echandía-Monroe, Aditi Shirke, Gustavo Guitierrez-Cruz, Madeline M. Wong, Kun-Lin Ho, Duanduan Ma, Elise G. DeArment, Amina Ali, W. Figg, Gregory T. Chesnut, M. V. Vander Heiden, Xiaofeng A. Su
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Extrachromosomal circular DNA has emerged as a pivotal factor in tumor biology, contributing to intratumor heterogeneity, oncogene amplification, and tumor evolution. Despite its relevance, the presence and molecular characteristics of plasma-derived cell-free extrachromosomal circular DNA (eccDNA) in cancer patients remain insufficiently explored. In this study, we profiled plasma-derived cell-free eccDNA from a multi-cancer cohort consisting of 413 cancer patients and 239 healthy individuals. We analyzed eccDNA fragment size distributions using a patient-derived xenograft (PDX) mouse model and identified fragment size features to distinguish tumor-derived eccDNA from non-tumor-derived eccDNA. We further performed in silico fragment size selection and developed a gene-based annotation method to characterize the gene content carried by cell-free eccDNA across different cancer types. By utilizing these cell-free eccDNA signatures, we developed ScanTecc (screening cancer types with cell-free eccDNA), a machine learning-based approach for cancer detection and tissue-of-origin classification. The classification performance of ScanTecc was further evaluated at the individual sample level using multiple machine learning classifiers, including adaptive boosting and logistic regression. We observed a significantly higher abundance and longer fragment lengths of eccDNA in cancer patients’ plasma. Analysis of the PDX mouse model revealed a distinct fragment size threshold of approximately 1,000 base pairs that effectively differentiates tumor-derived eccDNA from non-tumor-derived counterparts. Following fragment size selection, gene-based annotation of large-sized eccDNA revealed cancer type–specific enrichment of tumor-associated genes. Leveraging these features, ScanTecc achieved an overall AUC of 0.92 for distinguishing cancer patients from healthy individuals, with consistently high performance across disease stages, including stage I (AUC = 0.92) and stage IV (AUC = 0.93). ScanTecc also enabled accurate tissue-of-origin classification and achieved an overall AUC of 0.79 in identifying specific cancer types, with AUC values ranging from 0.70 for gastric cancer to 0.81 for ovarian cancer. Our study establishes a multi-cancer plasma cell-free eccDNA landscape and introduces a non-invasive cancer screening framework based on cell-free eccDNA features, highlighting the potential of plasma cell-free eccDNA for early cancer detection and tumor classification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd2c300c02b9f286a9e46bf7c8db5ce03368c9f" target='_blank'>
              Detection of primary cancer types via fragment size selection in circulating cell-free extrachromosomal circular DNA
              </a>
            </td>
          <td>
            Jingwen Fang, Songwen Luo, Shouzhen Li, Yehong Xu, Jing Wang, Benjie Shan, Mingjun Hu, Qiaoni Yu, Wen Zhang, Ke Liu, Yunying Shao, Jiaxuan Yang, Youyang Zhou, Guangtao Xu, Xinfeng Yao, Ruoming Sun, Mengyuan Zhang, Kun Li, Xihai Xu, Yongliang Zhang, Zhihong Zhang, Xinghua Han, Yueyin Pan, Chuang Guo, Kun Qu
          </td>
          <td>2026-01-06</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e732ad2e253dbdf86a5a9ac4cf431aea3406d6e9" target='_blank'>
              Genome wide analysis of YBX1-mediated redistribution of JMJD6 in ER+ breast cancer cells
              </a>
            </td>
          <td>
            Aritra Gupta, Siddharth Bhardwaj, K. Desai
          </td>
          <td>2026-01-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Ewing sarcoma is an aggressive bone tumor of adolescence characterized by a hallmark EWSR1::ETS fusion oncogene. The resulting chimeric oncoprotein drives tumorigenesis by reshaping transcriptional and epigenetic landscapes. However, how it is transcriptionally regulated and whether additional master transcription factors (MTFs) form a core regulatory circuit (CRC) in Ewing sarcoma remain unclear. Using an extensive panel of Ewing sarcoma cell lines and primary tumors, we mapped super-enhancers and identified enrichment of GGAA microsatellites, confirming their specificity to Ewing sarcoma as compared to other pediatric cancers and normal tissues. Integrating transcriptomic, epigenetic, 3D chromatin conformation, and dependency data, we predicted a set of MTFs potentially forming a CRC. However, functional validation demonstrated that these MTFs neither establish auto-regulatory loops nor confer robust proliferative dependencies typical of CRCs in other pediatric tumors. Instead, EWSR1::FLI1 emerged as an “hegemonic” oncoprotein, regulating expression of these MTFs without reciprocal regulation. Knockdown of EWSR1::FLI1 strongly shifted H3K27ac profiles toward mesenchymal states, whereas silencing individual or combined MTFs did not alter cell growth or EWSR1::FLI1 expression. These findings highlight the absence of a classical CRC in Ewing sarcoma and emphasize EWSR1::FLI1 as the dominant oncoprotein and a major vulnerability in this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/941b3d130e41a819129bbb3b8a4a261f87cf3d37" target='_blank'>
              The hegemonic EWSR1::ETS oncoprotein overrules core regulatory circuitry principles in Ewing sarcoma
              </a>
            </td>
          <td>
            Sandrine Grossetête, Sakina Zaidi, M. Orth, Sarah Morice, Karine Laud, Caroline Louis-Brennetot, Anna Sole Ferre, Virginie Perrin, F. Cidre‐Aranaz, Virginie Raynal, M. Aynaud, Sylvain Baulande, Marco Wachtel, I. Janoueix-Lerosey, Erika Brunet, Thomas G P Grünewald, Olivier Delattre, Didier Surdez
          </td>
          <td>2025-12-25</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a5adbde5a06ec4e198106c731d03948bb0cc143" target='_blank'>
              CAD-C reveals centromere pairing and near-perfect alignment of sister chromatids
              </a>
            </td>
          <td>
            Axel Delamarre, Maurane Reveil, David Pintor Pichon, M. Boemo, I. Whitehouse
          </td>
          <td>2025-12-20</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f796949ca22238ba707f7eb2bf4bbb66b809b4f" target='_blank'>
              Single-cell whole-genome sequencing reveals convergent evolution in Burkitt lymphoma
              </a>
            </td>
          <td>
            A. Steemers, M. V. van Roosmalen, Rico Hagelaar, L. Trabut, M. Verheul, Siem Jongsma, J. D. de Kanter, F. Meyer-Wentrup, R. van Boxtel
          </td>
          <td>2025-12-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The dynamic three-dimensional (3D) organization of the human genome (the 4D nucleome) is linked to genome function. Here we describe efforts by the 4D Nucleome Project1 to map and analyse the 4D nucleome in widely used H1 human embryonic stem cells and immortalized fibroblasts (HFFc6). We produced and integrated diverse genomic datasets of the 4D nucleome, each contributing unique observations, which enabled us to assemble extensive catalogues of more than 140,000 looping interactions per cell type, to generate detailed classifications and annotations of chromosomal domain types and their subnuclear positions, and to obtain single-cell 3D models of the nuclear environment of all genes including their long-range interactions with distal elements. Through extensive benchmarking, we describe the unique strengths of different genomic assays for studying the 4D nucleome, providing guidelines for future studies. Three-dimensional models of population-based and individual cell-to-cell variation in genome structure showed connections between chromosome folding, nuclear organization, chromatin looping, gene transcription and DNA replication. Finally, we demonstrate the use of computational methods to predict genome folding from DNA sequence, which will facilitate the discovery of potential effects of genetic variants, including variants associated with disease, on genome structure and function. The 4D Nucleome Project demonstrates the use of genomic assays and computational methods to measure genome folding and then predict genomic structure from DNA sequence, facilitating the discovery of potential effects of genetic variants, including variants associated with disease, on genome structure and function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/376c7aa7da9195c4ec11c0015d608d47cf9af797" target='_blank'>
              An integrated view of the structure and function of the human 4D nucleome
              </a>
            </td>
          <td>
            J. Dekker, B. A. Oksuz, Yang Zhang, Ye Wang, Miriam K. Minsk, Shuzhen Kuang, Liyan Yang, J. Gibcus, Nils Krietenstein, Oliver J Rando, Jie Xu, D. Janssens, Steven Henikoff, A. Kukalev, Willemin Andréa, Warren Winick-Ng, Rieke Kempfer, Ana Pombo, Miao Yu, Pradeep Kumar, Liguo Zhang, Andrew S. Belmont, Takayo Sasaki, Tom van Schaik, Laura Brueckner, Daan Peric-Hupkes, B. van Steensel, Ping Wang, Haoxi Chai, Minji Kim, Yijun Ruan, Ran Zhang, SA Quinodoz, Prashant Bhat, M. Guttman, Wenxin Zhao, Shu Chien, Yuan Liu, Sergey V. Venev, Dariusz Plewczyński, Ibai Irastorza Azcarate, Dominik Szabó, Christoph J. Thieme, Teresa Szczepińska, Mateusz Chiliński, Kaustav Sengupta, Mattia Conte, Andrea Esposito, Alex Abraham, Ruochi Zhang, Yuchuan Wang, Xingzhao Wen, Qiuyang Wu, Yang Yang, Jie Liu, L. Boninsegna, Aslı Yıldırım, Yuxiang Zhan, A. Chiariello, Simona Bianco, Lindsay Lee, Ming Hu, Yun Li, R. J. Barnett, Ashley L. Cook, Daniel J. Emerson, Claire Marchal, Peiyao Zhao, Peter J. Park, Burak H Alver, Andrew J. Schroeder, Rahi Navelkar, Clara Bakker, William Ronchetti, Shannon R. Ehmsen, Alexander D. Veit, Nils Gehlenborg, Ting Wang, Daofeng Li, Xiaotao Wang, Mario Nicodemi, Bing Ren, Sheng Zhong, Jennifer E. Phillips-Cremins, David M. Gilbert, Katherine S. Pollard, Frank Alber, Jian Ma, W. Noble, Feng Yue
          </td>
          <td>2025-12-17</td>
          <td>Nature</td>
          <td>2</td>
          <td>92</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cb53da812dbac416c29e7f7225ad2ab6decfdb9" target='_blank'>
              Exploring the Space of Tumor Phylogenies Consistent with Single-Cell Whole-Genome Sequencing Data
              </a>
            </td>
          <td>
            Samin Rahman Khan, P. Sashittal
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Colorectal cancer (CRC) develops through evolutionary processes involving genomic alterations, epigenetic regulation, and microenvironmental interactions. While traditionally explained by the stepwise accumulation of driver mutations, contemporary evidence supports a ‘Big Bang’ model in which many early-arising clones expand simultaneously to establish extensive heterogeneity. We reviewed recent studies employing spatially resolved multi-omic sequencing of tumour glands combined with computational modelling. These approaches enable high-resolution reconstruction of clonal architecture, transcriptional states, and chromatin accessibility. Findings show that although early clonal mutations shape tumour expansion, gene expression variability can be independent of genetic ancestry and instead reflects phenotypic plasticity driven by microenvironmental cues. Epigenomic analyses identified recurrent somatic chromatin accessibility alterations in promotors and enhancers of oncogenic pathways, frequently in the absence of DNA mutations, suggesting alternative mechanisms of gene regulation. Immune-focused studies demonstrated that early silencing of antigen-presenting genes and loss of neoantigens facilitate immune escape despite active surveillance. CRC is shaped by an interplay of genome, epigenome, and immune evolution, with non-genetic mechanisms and tumour plasticity emerging as important drivers of progression and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00d0ede4db6fdd44f5a7c3b138a4161c7d8d0f7" target='_blank'>
              Advances in Colorectal Cancer Cell Biology and Clonal Evolution
              </a>
            </td>
          <td>
            Sopozme Toghey, Elizabeth J. Harvey-Jones, Jonathan D. Towler, Charlotte J. H. Hafkamp, Irene Y. Chong
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Whole-genome duplication (WGD) fuels tumor evolution and therapy resistance, yet the molecular mechanisms governing the switch from the canonical mitotic cell cycle to the endoreplication cycle remain unclear. Here, we combine single-cell proteomics, manifold learning, and live-cell imaging to map the intersection of the mitotic and endoreplication cycles in breast cancer cells exposed to genotoxic agents. We identify two distinct routes to WGD driven by distinct p21 dynamics. High p21 induction induces G2 exit and endocycling, whereas insufficient p21 permits mitotic entry followed by slippage and endomitosis. This therapy-induced switch acts as a facultative stress response, generating drug-resistant polyploid populations that propagate genomic instability through replication stress and the generation of replication-competent micronuclei. Both paths to WGD converge on a common polyploid G0 state dependent on cyclin D1:CDK4/6 activity to complete the transition to the endoreplication cycle, revealing a shared vulnerability. Sequential treatment with genotoxic agents followed by CDK4/6 inhibitors preserves the cytotoxic efficacy of DNA-damaging drugs while simultaneously blocking entry into the endoreplication cycle and WGD-driven evolutionary rescue. These findings reveal the molecular rules governing the switch from the mitotic to endoreplication cycle and highlight the potential of WGD-blocking drugs as adjuvant therapies to inhibit drug resistance and suppress tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64dace37f687cf3f6f5d90e43b0621daaa3dd0b0" target='_blank'>
              Distinct p21 dynamics drive alternative routes to whole-genome duplication through a common CDK4/6-dependent polyploid G0 state
              </a>
            </td>
          <td>
            Yian Yang, Anuraag Bukkuri, Dante Poe, Janet McLaughlin, Sihan Hao, Daniel D. Brown, Katarzyna M. Kedziora, Wayne Stallaert
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b11efdad5d60344c302ed6d2c56a7d648ae4da7" target='_blank'>
              Targeted therapy-induced chromosomal instability dictates mitotic dependency on Aurora Kinase A
              </a>
            </td>
          <td>
            Chendi Li, Varuna Nangia, Melissa D. Vieira, Anahita Nimbalkar, Christopher J. Graser, Jeremy B Chang, Mohammad U. Syed, Yi Shen, Radhika Koranne, Lee Zou, Xueqian Gong, Sabrina L Spencer, Aaron N. Hata
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d6260543e09115d78b28611e811c1654d5ea522" target='_blank'>
              Mutant ASXL1 Drives Transcriptional Activation and Repression in Human Hematopoiesis
              </a>
            </td>
          <td>
            Madison L. Hall, Aliya Quintal, Samantha Worme, Thai T. Nguyen, Zachary Schonrock, Mitsuhiro Tsuchiya, Sonia N. Acharya, Hanqian L Carlson, Itallia V. Pacentine, Shawn B Shrestha, Jommel Macaraeg, Randall Armstrong, James McGann, Sara Evans-Dutson, Theresa A. Lusardi, Brendan L. O’Connell, Andrew C. Adey, G. Yardımcı, Julia E. Maxson, Theodore P. Braun
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Over the past 5–7 million years, humans and chimpanzees have diverged in brain size, structural complexity, and cognitive abilities despite high conservation of protein-coding genes. Notably, the endogenization and proliferation of retroviral infections within host genomes has introduced numerous species-specific regulatory elements that have the potential to influence gene regulation. However, the role of these endogenous retroviruses in hominoid brain evolution remains unclear. A burst of lineage-specific PTERV1 retroviruses recently invaded the chimpanzee genome but are absent in humans. We conducted an epigenomic analysis of PTERV1 insertions in chimpanzee neural organoids and found that they are heavily covered by DNA methylation, representing more than 150 species-specific heterochromatin domains with the capacity to influence gene regulatory networks. We identified one such chimpanzee-specific PTERV1 insertion on chromosome 19 that blocks the expression of the long noncoding RNA LINC00662, via DNA methylation spread to the adjacent genomic region. The expression of LINC00662 was restored in chimpanzee induced pluripotent stem cells when we deleted the PTERV1 insertion using CRISPR editing. We found that LINC00662, a human-specific RNA, is highly expressed in the developing brain and plays an important role in the posttranscriptional control of neuronal maturation, axon outgrowth, and neural organoid development. In summary, our findings describe how endogenous retroviral insertions contributed to the functional divergence of the human and chimpanzee brains. This provides a new mechanism by which retroviral pandemics influenced primate brain speciation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b7aafdaae0e68f3439418fdcde81bbb4ad6e768" target='_blank'>
              Retroviral insertions contributed to the divergence of human and chimpanzee brains
              </a>
            </td>
          <td>
            Patricia Gerdes, Ofelia Karlsson, R. Garza, Diahann A. M. Atacho, Pinghsun Hsieh, Bharat Prajapati, Chandramouli Muralidharan, Anita Adami, Carrie Davis-Hansson, Emily M. Johansson, Laura Castilla-Vallmanya, Wiktor West, Meghna Vinoud, Lucian Domitrovic, Pia A. Johansson, Jenny G. Johansson, C. Douse, C. Kanduri, Patric Jern, E. Eichler, J. Jakobsson
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="In placental XX females, one X chromosome is silenced during a narrow developmental time window by X-chromosome inactivation, which is mediated by Xist noncoding RNA. Although most X-linked genes are silenced during X-chromosome inactivation, some genes can escape. Here, by increasing its endogenous level, we show that Xist RNA can silence escapees well beyond early embryogenesis both in vitro, in differentiated cells, as well as in vivo, in mouse pre- and post-implantation embryos. We further demonstrate that Xist RNA plays a role in eliminating topologically associating domain-like structures spanning clusters of escapees, and this is dependent on SPEN. The function of Xist in silencing escapees and eliminating topological domains is initially fully reversible, but sustained Xist upregulation leads to irreversible silencing and CpG island DNA methylation of escapees. Thus, gene activity and three-dimensional topology of the inactive X chromosome are directly controlled by Xist, well beyond an early developmental time window. The authors show that increased Xist RNA levels can induce de novo silencing of genes that normally escape X inactivation. SPEN depletion prevents the silencing of escape genes upon Xist RNA overexpression in neural progenitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b274cefc089887937597429bef5c9660ece124a" target='_blank'>
              Escape from X inactivation is directly modulated by Xist noncoding RNA
              </a>
            </td>
          <td>
            Antonia Hauth, Jasper Panten, Emma Kneuss, Christel Picard, N. Servant, Isabell Rall, Yuvia A. Pérez-Rico, Lena Clerquin, N. Servaas, L. Villacorta, Ferris Jung, Christy Luong, Howard Y. Chang, Judith B. Zaugg, O. Stegle, D. T. Odom, Edith Heard, Agnese Loda
          </td>
          <td>2025-12-15</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>89</td>
        </tr>

        <tr id="The three-dimensional (3D) folding of the genome plays a crucial role in genome regulation. However, how 3D genome structure varies between individuals and consequently influences genome function and evolution remains poorly understood. One potential source of this variation is transposable elements (TEs), genomic parasites whose location and composition vary between species and individuals. Hosts typically silence TEs through enriching them with repressive epigenetic marks, turning euchromatic TEs into heterochromatin islands, which were shown to spatially interact with pericentromeric heterochromatin (PCH). Because most TE insertions are present in only a few individuals within Drosophila populations, we asked whether polymorphism in the presence/absence of TEs drives varying 3D structures through TE-PCH spatial interactions. We performed deep-coverage Hi-C of two wild-type Drosophila strains and developed a Hi-C analysis framework enabling allelic comparisons of spatial interactions with PCH. Supporting our hypothesis, nearly 40% of strain-specific euchromatic TEs cause their adjacent euchromatic regions to be spatially closer to PCH than TE-free homologous alleles. These interactions are not limited to specific TE families, and, surprisingly, telomere-proximal TEs show a similar propensity as centromere-proximal TEs to enhance PCH interactions. The most defining feature of TEs involved in PCH interactions is H3K9me3 enrichment, revealing a chromatin-based mechanism for TE-mediated 3D genome organization broadly applicable across TE families and genome locations. Importantly, TEs involved in PCH interactions reduce the expression of adjacent genes and are evolutionarily young, indicating stronger selection against them. Our study reveals a previously uncharacterized mechanism by which TEs influence the function and evolution of host genomes by generating polymorphic 3D genome organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85a4185cccd9e0a3e27a9efe577810b6e58e6b70" target='_blank'>
              Polymorphic 3D genome architecture mediated by transposable elements
              </a>
            </td>
          <td>
            Harsh G. Shukla, Yuheng Huang, Zita Y. Gao, Y. C. G. Lee
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0842575f97c19bbdfdf55ac42444d8e91ee3215" target='_blank'>
              Intrinsic DNA sequence determinants and tissue-specific regulation of human replication origins
              </a>
            </td>
          <td>
            Marcell Veiner, Marina Salvadores, Iván Galván-Femenía, Fran Supek
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Retrotransposons are genomic parasites frequently reactivated in cancers, where their mobility can cause genetic alterations. However, it remains unclear whether their gene products contribute to cancer beyond mutagenesis. Here, we uncover a chromatin-associated function of RNAs from Long Interspersed Element-1 (LINE-1), the only autonomous retrotransposon in the human genome. Subcellular-resolved transcriptomics revealed that LINE-1 RNAs are primarily nascent transcripts produced by full-length, cell-type-specific genomic copies of evolutionarily young subfamilies. Using a long-read chromosome conformation assay, we identified a class of highly interactive LINE-1 loci required for gene expression across cancer subtypes, revealing an unexpected regulatory role for LINE-1 locus transcription in oncogenic gene control. LINE-1 RNA depletion disrupted LINE-1-centric chromatin interactions and downregulated associated genes, whereas genomic insertion of an inducible LINE-1 generated de novo chromatin interactions in a transcription-dependent manner. Therefore, beyond their mutagenic potential, retrotransposons also regulate cancer gene expression by nucleating chromatin architecture through their transcriptional activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e9e6bb366cb35c566b3052102db69c89368901c" target='_blank'>
              LINE-1 Locus Transcription Nucleates Oncogenic Chromatin Architecture.
              </a>
            </td>
          <td>
            Michael Lee, Yuannyu Zhang, Jun Yi Stanley Lim, Tao Dai, James S Ye, Margaret B Cervantes, Varun Sondhi, Sisi Zheng, Yoon Jung Kim, Brandon Chen, R. DeBerardinis, Jian Xu
          </td>
          <td>2026-01-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dfa4ba8535d059c25df03570621ab7be09a0c95" target='_blank'>
              Chromatin compaction upon CDK12 inhibition drives long gene silencing and a combinatorial lethal dependency on PAF1
              </a>
            </td>
          <td>
            Jing Liang, Harri M. Itkonen
          </td>
          <td>2026-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1504ceb179d5285f33902944e8fdc793d756f87c" target='_blank'>
              Extrachromosomal DNA drives molecular and clinical heterogeneity in hepatocellular carcinoma: a multi-omics analysis and prognostic model development.
              </a>
            </td>
          <td>
            Kai Huang, Guangquan Zhang, Shuai Hu, Yongfei He, Hang Zhai, Junming Xu, Jicai Wang, Shengjie Hong, Fenfang Wu, Xianjie Shi
          </td>
          <td>2026-02-03</td>
          <td>Human genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e50ce208a78ff6653e59e288e486f934aca2d39" target='_blank'>
              Three-Dimensional Epigenome Roadmap of Human B-cell Differentiation Uncovers Mechanisms of Humoral Immunity and Oncogenesis
              </a>
            </td>
          <td>
            Raúl de Haro-Blázquez, Laureano Tomás-Daza, Lucía Fanlo-Escudero, Paula López-Martí, Nicolás Byrne-Álvarez, Llorenç Rovirosa, Blanca Valero-Martínez, Juan Ochoteco, Maria Rigau, Galina Medvedeva, Lucía Álvarez-González, Blanca Urmeneta, Ainoa Planas-Riverola, Jose Carbonell, Teresa Robert-Finestra, A. Alqahtani, Nicholas Brittain, José Falcon-Bermejo, Marta Kulis, J. Martín-Subero, María Dolores Guerrero-Gilabert, Emilio Amilibia, Anna Costa, Sara Pérez, Lisa J. Rusell, Daniel Rico, Alfonso Valencia, B. Javierre
          </td>
          <td>2025-12-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="ABSTRACT Human papillomavirus (HPV) and Merkel cell polyomavirus (MCPyV) are DNA tumor viruses that cause human cancer. The mechanisms by which HPV and MCPyV oncoproteins induce genomic instability are not well defined. This minireview discusses the influence of these oncoproteins on the repertoire of proteins at replicating DNA, known as the host replisome, and discusses how new technologies like isolation of proteins on nascent DNA (iPOND) can drive the discovery of viral dysregulation of the host replisome to enhance our understanding of viral oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d034cfb38775ad20a94958e31b3237066d26d7c8" target='_blank'>
              Viruses and the host replisome: discovering oncogenic mechanisms of small DNA tumor viruses
              </a>
            </td>
          <td>
            Christopher D Collins, Matthew Stefely, Kavi Mehta, Megan E. Spurgeon
          </td>
          <td>2026-01-07</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f294dc7fa07420ce7e76b025f057d7e7a72cbe2a" target='_blank'>
              Endogenous retroviral elements LTR8B and MER65 regulate the PSG9 locus that promotes trophoblast syncytialization: Insights into placental evolution and pre-eclampsia pathology
              </a>
            </td>
          <td>
            Manvendra Singh, Yuliang Qu, Amit Pande, Julianna Zadora, F. Herse, M. Gauster, Xuhui Kong, Rongyan Zheng, Rabia Anwar, Katarina Stevanovic, Ralf Dechend, Marie Cohen, Attila Molvarec, Jichang Wang, Miriam K. Konkel, Bin Zhang, Cédric Feschotte, Gabriela Dveksler, S. M. Blois, Laurence D. Hurst, Z. Izsvák
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="ABSTRACT Oncogenic viruses can induce epigenetic changes in host cells, which may contribute to cell transformation and tumorigenesis by altering gene expression patterns. Human adenoviruses, while primarily known for their lytic infections, have also been implicated in oncogenic transformation in experimental settings, but the mechanisms underlying adenovirus oncoprotein-induced epigenetic dysregulation remain poorly defined. Building on our recent findings that the adenoviral oncoprotein E1B-55K coordinates cellular transformation through interactions with DNA-bound host transcription factors, we next sought to determine whether these interactions also influence the chromatin landscape. For this purpose, we investigated the epigenetic consequences of E1B-55K expression in primary human mesenchymal stromal cells using histone-targeting MNase chromatin immunoprecipitation-seq, complemented by RNA-seq analysis. We demonstrate that stable expression of HAdV-C5 E1B-55K leads to widespread changes in histone post-translational modifications across the genome. Most notably, E1B-55K expression results in a marked loss of the activating histone marks H3K4me3 and H3K27ac at specific promoters and enhancers. These epigenetic alterations correspond with significant changes in gene expression, suggesting that E1B-55K interferes with normal transcriptional programming by altering the establishment of key histone modifications. Our findings help explain how E1B-55K drives viral transformation by changing epigenetic patterns and improving our understanding of virus-host interactions at the chromatin level. By describing how human adenoviruses disrupt epigenetic homeostasis, we provide insights into the mechanisms by which oncogenic viruses promote early events in cellular transformation. IMPORTANCE Oncogenic viruses can reshape host cell epigenomes to promote transformation, yet the mechanisms by which adenoviral oncogenes exert such control remain poorly understood. Here, we reveal that the human adenovirus E1B-55K oncoprotein induces widespread loss of activating histone modifications, leading to transcriptional silencing of key cellular genes in primary human cells. Building on our previous demonstration of efficient adenoviral transformation of primary human mesenchymal stromal cells, this study provides the first comprehensive view of E1B-55K-mediated epigenetic reprogramming. Our findings uncover a link between E1B-55K-host transcription factor interactions and chromatin remodeling, offering new insight into how adenoviruses disrupt epigenetic homeostasis to initiate early events in oncogenic transformation. These results provide direct evidence for the transcriptional repression function long postulated for E1B-55K and advance understanding of virus-host interplay at the chromatin level. Oncogenic viruses can reshape host cell epigenomes to promote transformation, yet the mechanisms by which adenoviral oncogenes exert such control remain poorly understood. Here, we reveal that the human adenovirus E1B-55K oncoprotein induces widespread loss of activating histone modifications, leading to transcriptional silencing of key cellular genes in primary human cells. Building on our previous demonstration of efficient adenoviral transformation of primary human mesenchymal stromal cells, this study provides the first comprehensive view of E1B-55K-mediated epigenetic reprogramming. Our findings uncover a link between E1B-55K-host transcription factor interactions and chromatin remodeling, offering new insight into how adenoviruses disrupt epigenetic homeostasis to initiate early events in oncogenic transformation. These results provide direct evidence for the transcriptional repression function long postulated for E1B-55K and advance understanding of virus-host interplay at the chromatin level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77947d567465053bbac94ee829a515fc47178965" target='_blank'>
              The adenovirus oncoprotein E1B-55K reshapes epigenetic histone modifications in primary human cells
              </a>
            </td>
          <td>
            Konstantin von Stromberg, Laura Seddar, Britta Gornott, Thomas Dobner, L. D. Bertzbach, W. Ip
          </td>
          <td>2026-01-26</td>
          <td>mBio</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) was characterized by a highly complex genome, with structural variations (SVs) playing a significant role in its development. In this study, we employed Oxford Nanopore Technology long‐read sequencing in paired tumor and adjacent normal liver tissues from 74 Chinese HCC patients to thoroughly characterize the landscape of somatic SVs. Our analysis revealed that somatic SVs were more prevalent in hepatitis B virus (HBV)‐related HCC, with chromosome 1 emerging as a major hotspot, and several members of the chromosome 1 open reading frame (C1orf) family genes expression level exhibited significant age‐related difference. Notably, HBV‐related HCC cases exhibited a higher frequency of deletions, particularly among younger ones (≤ 35 years old). In addition, we observed an increased burden of HBV integration events in younger ones. Remarkably, the divergent‐paired related homeobox (DPRX) loci was identified as a novel gene for HBV integration in younger patients. Together, these findings delineated the somatic SV landscape in HCC and underscored age‐associated HBV‐related genomic alterations as key pathological features of hepatocarcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83cdc214e4351424b89dd91edf26352c6acad965" target='_blank'>
              Landscape of Somatic and Age‐Related Pathogenic Structural Variations in Hepatocellular Carcinoma Revealed by Long‐Read Sequencing
              </a>
            </td>
          <td>
            Zhewen Wei, Yinghao Cao, Hongchao Liu, Mei Liu, Bolun Zhang, Jianming Ying, Jianqiang Cai, Xinyu Bi, Jian‐jun Zhao, Jianguo Zhou, Zhi-yu Li, Zhen Huang, Jianmei Liu, Xueyan Lv, Zhiwen Luo, Zhicheng Wei, Xiaoshi Zhang, Yi Yang, Yiqiao Deng, Yanjiang Yin, Jinghua Chen, Junbo Liang, Xiaoyue Wang, Ye‐fan Zhang, Hong Zhao
          </td>
          <td>2026-01-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a387e7c68c6390622f2c2d1c86f9db7afbe8e8ce" target='_blank'>
              miRISC inhibition causes mitotic defects and synergizes with genotoxic agents in cancers
              </a>
            </td>
          <td>
            Minsi Zhang, Ziqi Jiao, Ylenia Cendon Florez, Ivana Lessel, Xiaoyi Li, Melissa A. Yao, Moritz Weigl, Mateusz M. Ozimek, Turgut Dogruluk, Kevin Chen, Keith Conrad Fernandez, Vinagolu K. Rajasekhar, Olesja Popow, João A. Paulo, Davide Pradella, Tanmay Mishra, C. Mastroleo, Juliana I. Delgado, Francesco Enrico D’Amico, Vincenzo Cavalieri, Carol D. Morris, M. B. Tsou, Kevin M. Haigis, Jayanta Chaudhuri, R. Benezra, Joana A. Vidigal, Davor Lessel, Andrea Ventura, Gaspare La Rocca
          </td>
          <td>2025-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Activation-induced cytidine deaminase (AID) converts cytosines to uracils within the actively transcribed switch regions to initiate DNA repair and formation of DNA double strand breaks required for immunoglobulin class switch recombination (CSR). How AID specifically targets switch regions remains a key unanswered question. Using a multimodal live cell single molecule imaging approach, we demonstrate that intronic switch regions promote robust transcription by enhancing polymerase loading and persistent transcriptional bursts, resulting in the formation of a dynamic RNA hub consisting of numerous nascent switch transcripts simultaneously tethered to the IgH locus. We further demonstrate that AID interacts with switch region RNA in vivo, and that this interaction is required for recruitment of AID to the IgH locus. Together, our findings show that the RNA hub formed by nascent switch region transcripts may be part of a “class switch recombination center” and drives the specific recruitment of AID to the IgH locus to initiate CSR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaf4802ad2cbc2f54795b5db78233be1eb3d1360" target='_blank'>
              A dynamic RNA hub facilitates activation induced cytidine deaminase recruitment to the immunoglobulin heavy chain locus
              </a>
            </td>
          <td>
            Mariia Mikhova, Mackenzie Kapanka, Li Han, Jens C. Schmidt, Kefei Yu
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="CCR4-NOT regulates multiple steps in gene regulation and has been well studied in budding yeast. Although primarily cytoplasmic, where it plays an essential role in mRNA degradation, the human complex has poorly characterized nuclear functions. Here, we used auxin-induced degradation to rapidly deplete the scaffold subunit CNOT1 and the E3 ligase CNOT4, and characterized the transcriptional functions of the human CCR4-NOT complex. Using transient transcriptome profiling (TT-Seq) to measure ongoing transcription, we found widespread activation of RNA synthesis in depleted cells across genic and intergenic regions. Interestingly, fewer genes were repressed, including KRAB-Zinc-Finger-protein (KZNF) genes, especially those on chromosome 19. KZNFs repress genes and retrotransposable elements (rTEs), and consistent with decreased KZNF expression, rTEs, mainly Long Interspersed Nuclear Elements (LINEs), were activated. Full-length active LINEs and rTEs lying outside of genes were activated, suggesting that the increased transcription is not the direct result of transcription of the genes the rTEs are embedded in. We found that most activated transcription events were in proximity to KZNF binding sites, suggesting that KZNF regulation contributes to the suppression of genic and rTE transcription. Finally, we demonstrate that CCR4-NOT regulates the stability of rTE RNAs, indicating that the complex tightly controls transposon expression by repressing transcription and targeting their RNAs for decay.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52e5545d6ea0cdff6467688544916e041cb664d5" target='_blank'>
              Human CCR4-NOT suppresses pervasive transcription and retrotransposable elements
              </a>
            </td>
          <td>
            S. Kulkarni, Alexis Morrissey, A. Sebastian, O. T. Akinniyi, C. Keller, I. Albert, Shaun Mahony, J. Reese.
          </td>
          <td>2026-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="ABSTRACT Aims Mitochondrial DNA copy number (mtDNA-CN) is associated with several age-related chronic diseases and is a predictor of all-cause mortality. Here, we examine site-specific differential nuclear DNA (nDNA) methylation and differential gene expression resulting from in vitro reduction of mtDNA-CN to uncover shared genes and biological pathways mediating the effect of mtDNA-CN on disease. Materials and methods Epigenome and transcriptome profiles were generated for three independent human embryonic kidney (HEK293T) cell lines harboring a mitochondrial transcription factor A (TFAM) knockout generated via CRISPR-Cas9, and matched control lines. Results We identified 2924 differentially methylated sites, 67 differentially methylated regions, and 102 differentially expressed genes associated with mtDNA-CN. Integrated analysis uncovered 24 Gene-CpG pairs. GABAA receptor genes and related pathways, the neuroactive ligand signaling pathway, ABCD1/2 gene activity, and cell signaling processes were overrepresented, providing insight into the underlying biological mechanisms facilitating these associations. We also report evidence implicating chromatin state regulatory mechanisms as modulators of mtDNA-CN effect on gene expression. Conclusions We demonstrate that mitochondrial DNA variation signals to the nuclear DNA epigenome and transcriptome and may lead to nuclear remodeling relevant to development, aging, and complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcbea560ce049511e5c017437908e0cfbb595232" target='_blank'>
              Mitochondrial DNA copy number reduction via in vitro TFAM knockout remodels the nuclear epigenome and transcriptome
              </a>
            </td>
          <td>
            Phyo W. Win, Julia Nguyen, Elly H. Shin, T. S. Nagano, Brent Selimi, Katie Hong, A. Meybodi, Bradley P Yates, Emma V Burke, David E Carter, Gregory A Newby, Charles E Newcomb, D. Arking, C. Castellani
          </td>
          <td>2025-12-23</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f08e435f05db1aa85f8917f417c0101833b4e71" target='_blank'>
              Meiotic pairing through barcode-like satellite DNA repeats
              </a>
            </td>
          <td>
            Lena Skrutl, Ankita Chavan, A. Sintsova, Ilaria Ceppi, Corin J. Ropp, S. Sunagawa, Petr Cejka, Madhav Jagannathan
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b0405c36f0a0962f0b72a0d5777b890f28a6e5d" target='_blank'>
              Human POLD3 coordinates leading and lagging strand in Mitotic DNA Synthesis
              </a>
            </td>
          <td>
            Demetrio Turati, Vasilios S Dionellis, L. Tropia, T. Halazonetis
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b069e3e7f863bc3b24b70272c8b819a793918112" target='_blank'>
              Clastogenesis by nucleotide lesions requires the completion of two cell cycles
              </a>
            </td>
          <td>
            J. Jansen, Piya Temviriyanukul, Daniel de Groot, K. Szuhai, S. van Hees-Stuivenberg, Anastasia Tsaalbi-Shtylik, Heinz Jacobs, Niels de Wind
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Z-DNA/Z-RNA is an alternative left-handed nucleic acid conformation with established and emerging roles in gene regulation, immunity, and genome instability. However, its occurrence dynamics and lineage specificity across the tree of life have not yet been fully characterized. Utilizing the recently developed and improved Z-DNA searching tool, ZSeeker, we analyzed 281,139 complete organismal genomes, including multiple Telomere-to-Telomere genome assemblies, and generated genome-wide Z-nucleic acid maps, examined their topography, and compared them to dinucleotide-preserving controls. Cellular genomes featured pervasive Z-DNA enrichment relative to expectation, with enrichments of ∼1.5 and ∼1.7-fold in Bacteria and Archaea and ∼3-fold in Eukaryota. In contrast, Viruses exhibited large differences between lineages, with modest enrichment in several DNA viral groups and pronounced depletion across RNA clades, most notably Influenza A/B strains. We built a LASSO regression model trained on non-Influenza viruses (cross-validated R² ≈ 0.73), which identified GC content, genome type, and host type as the leading predictors for Z-nucleic acid density, yet it significantly over-predicted Z-RNA density in Influenza A/B. More than 99% of assemblies exceeded the +2 SD threshold, and a “typical Influenza” genome was predicted at 2.76 bp/kb compared to ∼0.016 bp/kb observed (a ∼170-fold overestimation based on chance alone). Together, these results reveal domain- and lineage-specific regimes: cellular genomes are enriched for Z-DNA consistent with regulatory roles, whereas influenza viruses appear to have undergone strong, lineage-specific depletion of Z-RNA-forming sequences, likely reflecting evolutionary pressure tied to host sensing pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d5326854d4ddc954dec631bbab0b254c15320a" target='_blank'>
              Characterization of Z-DNA dynamics across the tree of life
              </a>
            </td>
          <td>
            Georgios Megalovasilis, Eleftherios Bochalis, Michail Patsakis, Dionysios V. Chartoumpekis, Guliang Wang, Karen M. Vasquez, IIias Georgakopoulos-Soares
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f6c40c7ea1ba98d656117fae44f7bccd796a048" target='_blank'>
              Incomplete developmental silencing of cancer-testis antigen BORIS in humanized mouse model promotes cancer susceptibility
              </a>
            </td>
          <td>
            Emma Price, E. Pugacheva, D. N. Bhatt, Yon Ji, Sheila Yeboah, Arielle Scott, D. Loukinov, Victor V. Lobanenkov
          </td>
          <td>2025-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Immune dysfunction in cancer is enacted by multiple programs, including tumor cell-intrinsic responses to distinct immune subpopulations. A subset of these immune evasion programs can be systematically recapitulated through direct tumor-immune interactions in vitro. Here, we present an integrated, high-throughput single-cell CRISPR screening framework focused on the protein kinome for mapping the tumor-intrinsic regulation of T cell-driven immune pressure in glioblastoma (GBM). We combine pooled CRISPR interference and activation (CRISPRi/a) with immune-matched NY-ESO-1 antigen-specific allogeneic GBM-T cell co-culture and massively multiplexed single-cell transcriptomics to systematically quantify how genetic perturbation reshapes baseline tumor state and adaptive responses across graded effector-to-target ratios. We further leverage deep generative models for analyzing pooled CRISPR screens to decipher the effects of genetic perturbations on the mechanisms of tumor resistance. This framework resolves distinct modules of immune evasion and survival, including the regulation of the antigen-presentation machinery, interferon/NF-κB signaling, oxidative stress resilience, and checkpoint/cytokine programs, while identifying perturbations that reroute the continuous tumor transcriptional trajectory induced by T cell engagement. A secondary chemical screen in patient-derived GBM cultures identified putative kinase targets of immune evasion phenotypes (e.g., EPHA2 and PDGFRA), whose inhibition leads to the blockade of evasive programs and enhances T cell-mediated GBM killing. Together, this workflow provides a scalable blueprint for comprehensive charting of the genetic control of tumor-immune interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e76a058bed6b1c1e4d91cd692a05e66fab6beee" target='_blank'>
              Mapping kinase-dependent tumor immune adaptation with multiplexed single-cell CRISPR screens
              </a>
            </td>
          <td>
            Lingting Shi, Ross M. Giglio, Qingyuan Cai, Mathini Vaikunthan, Justin Hong, Abdullah Naqvi, Marina Milea, Hannah Khanshali, Anna Schoonen, Nicholas Hou, Jonathan Guo, Melanie Fraidenburg, Xumin Shen, Seth Malinowski, K. L. Ligon, Raúl Rabadán, Elham Azizi, José L. McFaline-Figueroa
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a627c9a572d6d8b3961581475fe8124b860d37" target='_blank'>
              Protein buffering of aneuploidy is driven by coordinated factors identified through machine learning.
              </a>
            </td>
          <td>
            Erik Marcel Heller, Karen Barthel, M. Räschle, Klaske M. Schukken, J. Sheltzer, Zuzana Storchová
          </td>
          <td>2026-01-22</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c8690c99df95e16ef7ad300af3d1501e42ea930" target='_blank'>
              Anticipating on-target resistance to WRN inhibitors in microsatellite unstable cancers
              </a>
            </td>
          <td>
            Mark E. Orcholski, Nancy Laterreur, Wardah Masud, Shamika Shenoy, Vincent Chapdelaine-Trépanier, Julian Bowlan, Amisha Minju-Op, Júlia-Jié Cabré-Romans, Tanja Sack, Chris Fiore, J. Young, Alejandro Álvarez-Quilón, Raquel Cuella-Martin
          </td>
          <td>2026-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0b876781bf210db756c35db260709c192ea56b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution.
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A McIntyre, Akimasa Hayashi, Nicolas Lecomte, Marc Hilmi, Wungki Park, Nan Pang, Eileen M. O’Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2026-01-22</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="ABSTRACT Secondary plasma cell leukemia (sPCL) is a rare, aggressive manifestation of multiple myeloma (MM). We report a 75‐year‐old Japanese man with anemia as the chief complaint and IgG‐λ MM that rapidly progressed to sPCL. Whole genome sequencing using Canopy revealed a major clone with a monoallelic TP53 mutation. During progression, a subclone with a biallelic TP53 mutation expanded, and the 1q21 copy number increased. Single‐cell RNA‐sequencing identified an emergent PCL population with high CKS1B expression. These data demonstrate genomic instability and clonal evolution during sPCL development, underscoring the need for approaches accounting for tumor heterogeneity in MM. Trial Registration: The authors have confirmed clinical trial registration is not needed for this submission.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2edcd3927754b5f89f94a2aa218c66d909ec380e" target='_blank'>
              Whole Genome and Single‐Cell RNA Sequencing Reveals Clonal Evolution and Heterogeneity of Secondary Plasma Cell Leukemia: A Case Report
              </a>
            </td>
          <td>
            Tomotaka Suzuki, Rui Yokomori, T. Sanda, M. Ri, Shinsuke Iida
          </td>
          <td>2026-01-13</td>
          <td>EJHaem</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract Termination of DNA replication is a surprisingly complex process that contributes critically to genome stability and cell viability. And even though progress was made to establish the consequences that arise if termination is going awry, the precise molecular mechanisms of fork fusion events and the coordination with key factors that ensure that DNA replication is brought to a successful conclusion remain poorly understood. We therefore investigated replication termination in Escherichia coli, focusing specifically on the interplay between replication fork fusions and genomic stability, the Tus–ter replication fork trap, and key DNA-processing enzymes. By utilizing whole genome sequencing, immunoblotting, and recombination reporter assays, we demonstrate that local hyper-recombination is induced wherever forks meet and that the combined loss of factors such as RecG helicase and 3′ exonucleases causes extreme over-replication in the terminus region of the chromosome. Unexpectedly, cells lacking Tus exhibit elevated R-loop levels, revealing an unanticipated connection between the fork trap and R-loop metabolism. These findings underscore the complexity of replication termination and its central role in maintaining bacterial genome stability, while providing mechanistic insights with implications for understanding replication termination in more complex organisms and developing new antimicrobial strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7313e7e0c5ea82dfcde902877c06fc46c8baafab" target='_blank'>
              Termination of DNA replication drives genomic instability via multiple mechanisms
              </a>
            </td>
          <td>
            Daniel J. Goodall, J. U. Dimude, M. A. Hashemloo, Emma L Dunbar, Iren Grigoryan, Amy L. Upton, E. Bolt, Christian J. Rudolph
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6532d2e7cd4f584634ccf28fbab489accbcdd3b5" target='_blank'>
              DNA repeats: origins, conservation, and role in human disease
              </a>
            </td>
          <td>
            A. K. Rana
          </td>
          <td>2026-01-06</td>
          <td>Genome Instability & Disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/596e7d27f8f208d19adb1081b3f37426d428f0a7" target='_blank'>
              PTEN deficiency linked to chromosome 10q loss leads to aggressive NF2 mutant meningioma biology
              </a>
            </td>
          <td>
            Abigail G. Parrish, H. N. Thirimanne, Sonali Arora, C. Herold-Mende, P. Sievers, F. Sahm, Frank Szulzewsky, Eric C. Holland
          </td>
          <td>2025-12-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Circular DNA molecules—mitochondrial DNA (mtDNA) and extrachromosomal circular DNA (ecDNA)—are fundamental genetic elements whose structural integrity is essential for cellular homeostasis. Pathological alterations in their length, caused by deletions or duplications, are directly implicated in a broad spectrum of human diseases, from inherited mitochondrial disorders to aggressive cancers. This review provides a comprehensive methodological guide for detecting these specific structural changes. We systematically evaluate classical techniques (long-range PCR, Southern blotting, electron microscopy) that offer direct visualization and validation, and modern high-throughput approaches (short- and long-read sequencing) that enable genome-wide discovery. Special emphasis is placed on quantitative clinical tools like digital droplet PCR (ddPCR) for ultrasensitive monitoring. By presenting a comparative framework, we guide the selection of optimal methods based on resolution, throughput, and clinical applicability. We conclude that an integrated, multi-method strategy is indispensable for robust analysis, positioning the precise detection of circular DNA length alterations as a cornerstone of emerging precision diagnostics and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9367ea411656b4af8c5e9c55df093cb7809d8cc8" target='_blank'>
              The Circular DNA Size Code
              </a>
            </td>
          <td>
            Zurab Abashidze
          </td>
          <td>2026-01-31</td>
          <td>Longevity Horizon</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromosomal translocations are prevalent genetic events across multiple pediatric cancers, notably in CNS tumors, solid tumors, and leukemias. For decades, Fusion oncoproteins resulting from chromosomal translocations have been proposed as a hallmark of cancers, some of which can drive the process of cancers as the initial event of the disease. In addition, studies have shown that some tumor cells become addicted to the activity of fusion proteins, and cell death occurs when the fusion proteins are depleted. These researches suggest that fusion oncoproteins are one of the most promising targets for cancer treatment. Although fusion proteins are already recognized as critical oncogenic drivers, increasing evidence suggests that they can also form positive feedback loops with other proteins. In cancer patients, positive feedback loops have been shown to activate various oncogenic signals to drive tumor development, and influencing tumor cells’ sensitivity to different therapies. Therefore, these loops not only amplify the functions of the fusion proteins but also render single-agent targeting of the fusion protein insufficient to suppress tumor growth, highlighting the therapeutic potential of combination strategies in treating fusion-positive tumors. This review highlights the oncogenic roles of fusion protein-driven positive feedback loops in tumor initiation and progression, outline the molecular mechanisms underlying their formation and function, and summarize emerging therapeutic strategies targeting these circuits, offering new insights into the treatment of fusion-positive cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43211d0c7c2e449b055a88bf1178c869727d1a0b" target='_blank'>
              Fusion oncoproteins orchestrate tumorigenesis and sustain malignant progression via a positive feedback mechanism
              </a>
            </td>
          <td>
            Wenwen Ying, Xiaomin Wang, Jiayi Yu, Jinhu Wang, Qiao-Chu He, Bo Yang, Yifan Chen, Meidan Ying
          </td>
          <td>2026-01-03</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00cba125444257d124df7ba1ec49280e8dc284a3" target='_blank'>
              Single-cell transcriptomics reveals a conserved embryonic progenitor in different human cancers
              </a>
            </td>
          <td>
            A. Khozyainova, Vera G. Subrakova, M. Menyailo, D. Zhigalina, T. N. Kireeva, A. Galiakberova, Mikhail A. Berestovoy, Elena E. Kopantseva, A. Korobeynikova, E. Dashinimaev, Dmitry M. Loos, M. S. Tretyakova, U. A. Bokova, N. Skryabin, E. Denisov
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Receptor tyrosine kinase (RTK) fusions are a large class of oncoproteins found in ∼5% of cancers. Key questions remain, however, about how RTK fusions transmit oncogenic signals, including how these largely cytoplasmic proteins activate downstream pathways that originate at the plasma membrane. Fusions are multimeric and can form mesoscale condensates in cancer cells, and condensation has been implicated as an essential mechanism to enable signal transmission from the cytoplasm. However, whether condensates play a causal role, or whether smaller ‘diffuse’ assemblies are sufficient to transduce signals, has been challenging to establish. Here we apply advanced microscopy, single-cell analysis, and synthetic fusions to determine the principles by which multimerization and condensation drive signaling from cytoplasmic RTK fusions. For EML4-ALK, a prominent fusion that forms condensates, we found poor correlation between condensation and signaling. By contrast, EML4-ALK activity was abundant in the diffuse phase, and the kinetics of diffuse-phase activity aligned more closely with downstream Erk signaling than did kinetics of signaling within condensates. Synthetic RTK fusions showed that cytoplasmic ALK or RET fusion dimers—and even constitutively active monomers—were sufficient to induce strong Ras-Erk signaling despite the absence of condensates, and diffuse fusions were sufficient to transform cells in vitro and in subcutaneous tumor models. A panel of various other cancer-driving RTK fusions showed that low-order multimerization was universal across fusions, whereas mesoscale condensation was rare and did not correlate with signaling. Our results suggest that low-order fusion multimerization is sufficient to drive its phosphorylation, which is necessary and sufficient to trigger downstream oncogenic signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef6563f441f99dac16cec7dceb7a7c5f130e146e" target='_blank'>
              Low-order assemblies drive oncogenic RTK fusion signaling without condensation
              </a>
            </td>
          <td>
            D. Gonzalez-Martinez, Thomas R. Mumford, Delaney Wilde, Sofia Wissert, Y. Gao, Emily Brackhahn, R. Kriwacki, Elizabeth Rhoades, Lukasz J. Bugaj
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70b4b4433336cdba04cf2ea66983445def093bd8" target='_blank'>
              Genomic insights into chromosomal fusion and its evolutionary implications for zokors.
              </a>
            </td>
          <td>
            Zhuoran Kuang, Xiaojie Yang, Na Wan, Jiaqi Chen, Qijiao Duan, Bowen Li, Xi Liu, Xiaolong Liang, Xinfeng Liu, Wenyu Liu, E. Nevo, Kexin Li
          </td>
          <td>2026-02-03</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Dysplastic nodules (DN) are precursors to cirrhosis‐associated malignancy, and the HBV DNA integration in the human genome plays a critical role in tumorigenesis. However, the precise relationship between DN and HBV integration remains unclear. We performed HBV‐capture sequencing on 19 cirrhosis patients with HBV infection (DN, 10; RN, 9), out of which 10 subjects (DN, 9; RN, 1) underwent RNA sequencing. We performed HBV‐capture sequencing on 19 cirrhosis patients with HBV infection (DN, 10; RN, 9), out of which 10 subjects (DN, 9; RN, 1) underwent RNA sequencing. 1936 and 1450 HBV integration sites were identified in the DN and RN samples, respectively. The number of HBV integration sites in DN correlated with nodule size. Breakpoints in HBV genome were concentrated within 100 bps towards the 5' or 3' end of involved HBV genes. Furthermore, integration numbers also positively correlated with number of point mutations in DN samples. We identified 53 and 29 recurrent genes containing HBV integrations in >=2 samples in DN and RN samples, respectively. Higher clonality was observed for HBV integrations in recurrent genes than other HBV‐integrated genes. The HBV integrations in recurrent genes were predominantly located in intron regions. Notably, among those recurrently HBV‐integrated genes in DN samples, SCHIP1, ZDHHC14, YPEL2, RABGAP1L, and SOX5 displayed significant expression alterations. Moreover, clinical indicators revealed significant prolongation of prothrombin time in two DN patients with HBV integrations in SCHIP1 and ZDHHC14. As a new insight regarding HBV integrations in DN stage, our findings suggest a possible role of HBV integration in the transformation of DN to early‐stage liver cancer by affecting the expression of key genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef5e9a714b1a7dd80b4097dd4b95245da6b9365c" target='_blank'>
              Analysis of HBV Integration Reveals New Insights of Oncogenic Mechanism in Dysplastic Liver Nodule
              </a>
            </td>
          <td>
            Xi Zeng, Hui Liu, Zheqi Xu, Xinjie Rao, Yuyouye Wang, Fang Peng, Wei Dong, Ziying Wang, Zhenguang Wang, Xing Gu, Fuchen Liu, Guoliang Li, Weiping Zhou, Linghao Zhao
          </td>
          <td>2026-01-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract The xenogeneic silencer protein Lsr2 from Mycobacterium tuberculosis plays a critical role in its survival and pathogenesis. Lsr2 is a nucleoid-associated protein (NAP) that interacts with DNA in vivo and regulates many genes. Purified Lsr2 forms nucleoprotein filaments with DNA molecules, leading to highly compacted DNA conformations. However, the physical mechanism underlying Lsr2-mediated DNA compaction, resulting in gene regulation, remains elusive. We employed a combination of biochemical assay, single-molecule imaging, and molecular dynamics simulations to investigate the governing principles of Lsr2-mediated DNA compaction. We show that, while Lsr2 alone undergoes phase separation, addition of DNA substantially lowers the required concentration for its phase separation. Strikingly, our single-molecule and simulation data establish that Lsr2 forms condensates with long stretches of AT-rich DNA, providing strong evidence for sequence-dependent co-condensation. We further validate our findings by carrying out in vivo imaging of endogenously expressing Lsr2 tagged with eGFP in Mtb cells. This observation is contrary to the classical view of sequence-dependent binding of individual protein molecules to DNA; our findings rather suggest that protein–DNA co-condensates “sense” the average binding energy landscape. We present a physical model for Lsr2-mediated DNA compaction and gmycene regulation, describing a novel mechanism for NAP-mediated genome organization in bacteria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9fe9c6575903dcbbc1b5c771711d6f1e938d3c5" target='_blank'>
              Sequence-dependent co-condensation of Lsr2 with DNA elucidates the mechanism of genome compaction in Mycobacterium tuberculosis
              </a>
            </td>
          <td>
            Prakshi Gaur, Thejas Satheesh, R. Singh, Hussain Beig, Sneha Shahu, Saminathan Ramakrishnan, Mansi Srivastava, Shreyasi Neogi, Ayesha Dash, Amit Singh, Sandeep Choubey, Mahipal Ganji
          </td>
          <td>2026-01-05</td>
          <td>Nucleic Acids Research</td>
          <td>2</td>
          <td>2</td>
        </tr>

        <tr id="SUMMARY Plant nuclear genomes contain a variable, though typically minor, fraction of DNA sequences of plastid origin known as NUPTs. Unlike the massive transfer of DNA and genes from the proto‐organelle genome to the nucleus that occurred during the endosymbiotic event that gave rise to plastids, the formation of NUPTs is an ongoing process that does not imply concomitant DNA loss. Although NUPTs are generally considered to be potentially deleterious insertions that are continuously generated and rapidly eliminated at near‐constant turnover rates, accumulating evidence reveals alternative evolutionary trajectories. In this review, we discuss recent findings that highlight the episodic formation of NUPTs, their subsequent proliferation, and their eventual long‐term fixation within the nuclear genome. We also explore their non‐random spatial association with specific genomic elements. NUPTs show preferential overlap with specific superfamilies of transposable elements, which may facilitate their proliferation and dispersal throughout the nuclear genome. Regarding protein‐coding genes, the contribution of NUPTs varies among species. In contrast, NUPTs are found to be consistently enriched among certain classes of non‐coding RNA genes, notably rRNA, tRNA, and specific regulatory RNA families, suggesting that they are involved in the evolution of gene regulation and translational machinery. Overall, these findings underscore the unexpected complexity of the mechanisms underlying NUPT formation and support the idea that they are a significant source of genome variation and evolutionary innovation. Further research is necessary to fully elucidate the mechanisms underlying NUPT formation, as well as to determine their potential adaptive significance in plant genome evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72824b12a89eeb048c0f674beb8dd4ae9c4603a5" target='_blank'>
              Nuclear DNA of plastid origin (NUPTs), neglected driver of genome variation and evolutionary innovation
              </a>
            </td>
          <td>
            Lorenzo Carretero-Paulet, Juan Pablo Marczuk-Rojas, Aaron Gálvez‐Salido
          </td>
          <td>2026-02-01</td>
          <td>The Plant Journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Eukaryotic genomes contain numerous transposable elements (TEs), whose dysregulation threatens genome stability and may contribute to cancer. Pancreatic adenocarcinoma (PAAD) is among the deadliest cancers, marked by abundant stroma that obscures tumor-specific molecular signals, complicating bulk-tissue analyses. Here, using 71 patient-derived PAAD organoids, we show that TE activities may potentially promote tumorigenesis and provide a source of novel immunotherapeutic targets. We identify 16 new TE-derived transcripts fused with 15 known oncogenes, exhibiting potential oncogenic function and prognostic value. Notably, LTR7-PLAAT4, present in 29% of tumors, encodes a protein variant transcriptionally regulated by FOXM1 binding to the LTR7 promoter. LTR7-PLAAT4 isoform 2 is associated with increased cholesterol ester accumulation and lipid droplet formation mediated through BSCL2 coexpression, potentially fostering tumor progression. On the immunogenic front, HLA-I immunopeptidomics of AsPC-1 cells and DAC13 organoids identify over 11,000 peptides respectively. Althought mutation-derived neoantigens are rare, several peptides are originated from TE-chimeric transcripts, including four predicted by TEprof2. The peptide FLIQHLPLV, detected in 27% of organoids, exhibits robust immunogenicity, validated by T2 binding, mass spectrometry and ELISPOT assays with HLA-genotyped PBMCs. Together, these findings suggest that TE activities may contribute to PAAD progression and diversify its immunopeptidome, providing new opportunities for molecular subtyping and potential immunotherapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/112bba488753be082c706c6e45c781f5f0d2fb01" target='_blank'>
              Transcriptomic landscape of transposable elements reveals LTR7-PLAAT4 as a potential oncogene and therapeutic target in pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Meilong Shi, Chuanqi Teng, Shan Zhang, Xiaobo He, Lingyun Xu, Fengxian Han, Rongqi Wen, Ganjun Yu, Jingwen Liu, Yang Feng, Yanfeng Wu, Yan Ren, G. Jin, Jing Li
          </td>
          <td>2026-01-08</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Enhancers are regulatory DNA sequences that increase the transcription of a target gene when bound by transcription factors. Decades of research endeavors have shed light on the crucial roles of enhancers in cardiovascular development and disease. Besides working as cis-regulatory elements, enhancers also play critical roles in transactivating mechanisms such as enhancer RNA transcription and transcriptional condensates. Technological advances, such as single-cell multiomics, CRISPR (clustered regularly interspaced short palindromic repeats)-based enhancer perturbation screens, and machine learning technologies, provide us with unprecedented opportunities to gain novel insights into the dynamic function of cardiovascular enhancers. In this brief review, we discuss cutting-edge techniques that are being or can be leveraged to fill knowledge gaps in studying cardiovascular enhancers. Like gene expression, enhancers display functional dynamics in concordance with specific cellular stress. We summarize the findings of inducible enhancer functions in the cardiovascular system. We also discuss several recent studies that are advancing our understanding of cardiovascular enhancer functions and are potentially paradigm-shifting. These insights have great potential to inform novel therapeutic strategies targeting enhancer-mediated gene dysregulation in cardiovascular disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04775b01e20c7d3f0f15530bb34c511d087e60a8" target='_blank'>
              Enhancer Dynamics for Gene Regulation in the Cardiovascular System.
              </a>
            </td>
          <td>
            Zhaoning Wang, Peiheng Gan
          </td>
          <td>2026-01-08</td>
          <td>Arteriosclerosis, thrombosis, and vascular biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c961a3f1619198d178054a670088d76568b3783" target='_blank'>
              Friend and Foe: Genome-Wide Analysis of the Tardigrade Dsup Protein Expressed in Yeast Reveals Trade-offs of DNA Protection
              </a>
            </td>
          <td>
            H. Gaikani, Marjan Barazandeh, Lewis Hitchens, G. Giaever, Corey Nislow
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fc8f584519863394cabad2d307177d0a23fce8d" target='_blank'>
              An EGR1-dependent cascade modulates genome architecture at the CSF1R locus.
              </a>
            </td>
          <td>
            Sandra Deliard, Elisa Barbieri, Marco Trizzino, Kelsey A. Leach, Avery J. Zucco, Francis Picone, Filippo Veglia, Alessandro Gardini
          </td>
          <td>2026-01-14</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background: Optical genome mapping (OGM) detects genome-wide structural variants (SVs), including balanced rearrangements and complex copy-number alterations beyond standard-of-care cytogenomic assays. In chronic lymphocytic leukemia (CLL), cytogenetic and genomic risk stratification is traditionally based on fluorescence in situ hybridization (FISH), karyotyping, targeted next-generation sequencing (NGS), and immunogenetic assessment of immunoglobulin heavy chain variable region (IGHV) somatic hypermutation status, each of which interrogates only a limited aspect of disease biology. Methods: We retrospectively evaluated fifty patients with CLL using OGM and integrated these findings with cytogenomics, targeted NGS, IGHV mutational status, and clinical time-to-first-treatment (TTFT) data. Structural variants were detected using OGM and pathogenic NGS variants were derived from a clinical heme malignancy panel. Clinical outcomes were extracted from the electronic medical record. Results: OGM identified reportable structural variants in 82% (41/50) of cases. The most frequent abnormality was del(13q), observed in 29/50 (58%) and comprising 73% (29/40) of all OGM-detected deletions with pathologic significance. Among these, 12/29 (42%) represented large RB1-spanning deletions, while 17/29 (58%) were focal deletions restricted to the miR15a/miR16-1 minimal region, mapping to the non-coding host gene DLEU2. Co-occurrence of adverse lesions, including deletion 11q/ATM, BIRC3 loss, trisomy 12, and deletion 17p/TP53, were recurrent and strongly associated with shorter TTFT. OGM also uncovered multiple cryptic rearrangements involving chromosomal loci that are not represented in the canonical CLL FISH probe panel, including IGL::CCND1, IGH::BCL2, IGH::BCL11A, IGH::BCL3, and multi-chromosomal copy-number complexity. IGHV data were available in 37/50 (74%) of patients; IGHV-unmutated status frequently co-segregated with OGM-defined high-risk profiles (del(11q), del(17p), trisomy 12 with secondary hits, and complex genomes whereas mutated IGHV predominated in OGM-negative or structurally simple del(13q) cases and aligned with indolent TTFT. Integration of OGM with NGS further improved genomic risk classification, particularly in cases with discordant or inconclusive routine testing. Conclusions: OGM provides a comprehensive, genome-wide view of structural variation in CLL, resolving deletion architecture, identifying cryptic translocations, and defining complex multi-hit genomic profiles that tracked closely with clinical behavior. Combining OGM and NGS analysis refined risk stratification beyond standard FISH panels and supports more precise, individualized management strategies in CLL. Prospective studies are warranted to evaluate the clinical utility of OGM-guided genomic profiling in contemporary treatment paradigms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28f49f64c743393cb2116dc5e9531fab4be082e7" target='_blank'>
              Optical Genome Mapping Enhances Structural Variant Detection and Refines Risk Stratification in Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            S. Chakraborty, Michelle A. Bickford, Narcisa A. Smuliac, Kyle A. Tonseth, Jing Bao, Farzana Murad, Irma G. Domínguez Vigil, Heather B. Steinmetz, Lauren M. Wainman, Parth Shah, Elizabeth M. Bengtson, Swaroopa PonnamReddy, Gabriella A. Harmon, Liam L. Donnelly, L. Tafe, Jeremiah X. Karrs, Prabhjot Kaur, Wahab A. Khan
          </td>
          <td>2026-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Abstract The occupancy of DNA-binding factors is critical for establishing the chromatin context of DNA-templated processes like transcription and replication. Many studies have investigated relationships between chromatin and transcription in response to genetic and environmental perturbations, but a comprehensive view of the dynamics of chromatin organization and its effect on transcription during the cell-intrinsic process of the cell division cycle has been lacking. In this study, we use the model organism Saccharomyces cerevisiae to investigate the interplay between the cell-cycle-regulated dynamics of chromatin and transcription. In a time series across two consecutive cell cycles, we profile the genome-wide chromatin occupancy landscape with next-generation sequencing of micrococcal-nuclease-digested chromatin while simultaneously profiling the transcriptome with RNA-seq. Surprisingly, among cell-cycle-regulated genes, only a small subset exhibits a change in protein occupancy in their promoter or gene body that strongly mirrors the dynamics of their expression level. However, we separately identify widespread chromatin changes that are not directly linked to cell-cycle-regulated genes, revealing transcription-independent chromatin dynamics during cell-cycle progression. Using entropy as a proxy for nucleosome disorganization, we observe widespread nucleosome disruption in S phase followed by re-organization in M phase, associated with replicative and mitotic activities, respectively. A notable exception is the set of genes expressed in mitosis, which exhibit marked M-phase nucleosome disorganization, likely due to elevated transcription. We develop a Gaussian process statistical model that uses simple features from these chromatin data to improve the prediction of transcript dynamics through the cell cycle. Collectively, our data reveal complex relationships between the dynamics of chromatin occupancy, transcription, and replication during the cell cycle.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4654fa6db895d00f5770811dad7f4f95e97a44ed" target='_blank'>
              Comprehensive profiling of chromatin occupancy dynamics through the cell cycle
              </a>
            </td>
          <td>
            Yulong Li, D. MacAlpine, A. Hartemink
          </td>
          <td>2026-01-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="A large number of extrachromosomal circular DNAs (eccDNAs) are found in eukaryotic cells, but the mechanism behind the formation of eccDNAs remains unknown. EccDNAs smaller than 1000 bp are commonly referred to as microDNAs. In a prior study, we identified a 542 bp microDNA, designated eccDNAfib-L, mapping to Chromosome 14: 9,692,083-9,692,624 nt in the silk gland of Bombyx mori. There is a direct short repeat “GAGT” at both 5′ and 3′ break points of eccDNAfib-L, but only one copy of “GAGT” is retained in eccDNAfib-L. Here, we find that the specific junction observed can also be detected by transfecting with a DNA fragment containing upstream sequences of the 5′ break point and downstream sequences of the 3′ break point of eccDNAfib-L. Additionally, the length of the flanking sequences and the direct short repeats “GAGT” affect the formation efficiency of eccDNAfib-L. We confirm that eccDNAfib-L formation is associated with DNA repair pathways, with the expression of eccDNAfib-L undergoing significant alterations following the silencing of genes related to DNA repair pathways. In vitro, a cell-free reaction system confirms that Polθ and the direct short repeat “GAGT” are essential for the circularization of eccDNAfib-L, and the DExH-box helicase domain of Polθ plays critical role in mediating this circularization process. Collectively, our data support a mechanism whereby Polθ can mediate the joining of linear DNA fragments with direct short repeat at the 5′ and 3′ ends to form a circular DNA through MMEJ. The results not only clarify the essential factors driving eccDNAfib-L formation but also complement previous reports on eccDNA formation. DNA repair pathways are involved in eccDNAfib-L formation. Specifically, Polθ’s DExH-box helicase domain and direct short repeat “GAGT” are essential for its MMEJ-mediated circularization, delineating key factors driving this microDNA’s biogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/853b0d1066b56e039e179bc83a12e0f1707f8de1" target='_blank'>
              DNA repair pathway-related proteins are involved in the circularization step of microDNA eccDNAfib-L
              </a>
            </td>
          <td>
            Xinyu Tong, Qunnan Qiu, Xiaolong Hu, Zhe Liu, Mei Yin, Liuyang Li, Chao Lei, Yongjie Feng, Min Zhu, C. Gong
          </td>
          <td>2025-12-19</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 The minor spliceosome (MiS) is a specialized RNA-processing machinery upregulated in cancer to promote expression of oncogenic genes. Here, we identify its catalytic component, U6atac snRNA, as a druggable vulnerability in prostate and breast cancers. U6atac knockdown triggers R-loop–mediated DNA damage while impairing DNA repair by downregulating key factors such as BRCA1, PARP1, TP53BP1, and CHK1/2, disabling both homologous recombination and non-homologous end joining. This dual effect sensitizes tumors to PARP inhibitors, cisplatin, and radiation, independent of BRCA status. Moreover, we uncover an adaptive resistance mechanism driven by extracellular vesicles enriched in U6atac, which amplify MiS activity and facilitate therapy escape; a process reversed by U6atac depletion. Across multiple in vitro and in vivo models, MiS targeting demonstrates tumor-selective activity with minimal toxicity. These findings position U6atac as a central regulator of genome stability and establish MiS targeting as a promising approach to potentiate genotoxic therapy and overcome resistance.



 Anke Katharina. Augspach, Mark Rubin, Paola Francica, Sven Rottenberg, Rahul Kanadia. Targeting Minor Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c71eaaa6cc7e28ae4d142a474ab9e0350163c6b" target='_blank'>
              Abstract B003: Targeting Minor Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer
              </a>
            </td>
          <td>
            Anke Augspach, Mark A. Rubin, P. Francica, Sven Rottenberg, Rahul Kanadia
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da2aeba8554dfbf99a705198a393f82144d0d18d" target='_blank'>
              Menin maintains enhancer-promoter interactions in a leukemia-specific manner
              </a>
            </td>
          <td>
            Vassilena Sharlandjieva, Catherine Chahrour, F. Lassen, Joseph C. Hamley, Andreas Damianou, N. Denny, Alastair L. Smith, Svenja S Hester, I. Vendrell, Ronald W. Stam, Marina Konopleva, A. Roy, J. O. Davies, Nicholas T. Crump, Benedikt M. Kessler, Thomas A. Milne
          </td>
          <td>2026-01-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="MYC genomic amplification and pathway activation is associated with aggressive behavior and poor prognosis in osteosarcoma (OS). However, a discordance exists between copy-number gains and transcriptional output from MYC in OS, and defining these mechanisms is critical to understand and intercept persistent MYC signaling. Here, we showed that cytoplasmic mRNA (poly(A)) sustains MYC activation in OS. Multi-omics profiling and single-cell transcriptomics identified TENT5A, a non-canonical RNA-binding poly(A) polymerase, as selectively upregulated in MYC-activated tumors and enriched in proliferative, stem-like populations. Biochemical and genetic evidence demonstrated that TENT5A directly bound MYC mRNA via its PAP/OAS1 domain, extended its poly(A) tail, and stabilized the transcript, thereby reinforcing MYC-driven stemness and chemoresistance. Gain- and loss-of-function assays, orthotopic xenografts, and patient-derived organoids confirmed that elevated TENT5A enhanced tumor-initiating capacity and reduced chemotherapy sensitivity. Pharmacologic inhibition of TENT5A disrupted MYC mRNA stabilization, shortened poly(A) tails, and reversed chemoresistance in preclinical models. These findings delineate a post-transcriptional RNA-stabilization pathway that reconciles the disconnect between MYC genetic alterations and transcriptional activity and nominate the RNA-binding protein TENT5A as a therapeutically tractable target in OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73d515c1b7f0b534d63c1002ea77732498dd44f1" target='_blank'>
              TENT5A Maintains MYC mRNA Stability to Enhance Osteosarcoma Stemness.
              </a>
            </td>
          <td>
            Yining Tao, Qi Zhang, Haoyu Wang, Binghui Yang, Haoran Mu, Kaiyuan Liu, Weisong Zhao, Xiyu Yang, Bowen Zhao, D. Zuo, Liu Yang, Zhengdong Cai, Zongyi Wang, Hongsheng Wang, Yingqi Hua, Wei Sun
          </td>
          <td>2026-01-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Analytical challenges in detecting sequencing-unknown extrachromosomal circular DNA (eccDNA)─a molecule with critical roles in tumorigenesis and therapeutic potential─primarily arise from interference by long linear genomic and mitochondrial DNA. Here, we for the first time introduce a selective real-time rolling circle amplification (sRT-RCA) method that overcomes this limitation by combining RecBCD and PacI dual-enzyme digestion to remove interfering nucleic acids, thus permitting direct Phi29 polymerase-mediated RT-RCA without the need for conventional physical purification step. This approach achieves unprecedented sensitivity (2 pg, ∼4 × 10-19 mol circular DNA molecules) within a 104-fold higher background of genomic DNA. Furthermore, the established method, utilizing a simplified direct-lysis workflow, is capable of detecting eccDNA from as few as 125 cells. By this approach, we estimated that eccDNA averages 540 copies (8kb plasmid equivalents) in certain cancer cells (K562). Clinically, eccDNA was detected in 3/10 early stage and 6/10 advanced-stage patients. These findings indicate that eccDNA is widespread in cancer patients and demonstrate the potential utility of our method in clinical applications and translational research, thereby bridging a key technical gap and accelerating the translation of eccDNA-based diagnostics and personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad0f1ea2bf7309e407f53aa8233632e2a1ae6a7f" target='_blank'>
              Ultrasensitive and Rapid Screening of Tumor-Associated Extrachromosomal Circular DNA by Selective Real-Time Rolling Circle Amplification.
              </a>
            </td>
          <td>
            Yiran Liu, Gang Li, Jing Zheng, Qing Li, Wei-Min Tong, Weiyi Lai, Hailin Wang
          </td>
          <td>2026-02-03</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Significance Our studies show the levels of type I topoisomerases are increased in human papillomaviruses (HPV)-positive cells, leading to enhanced levels of DNA breaks and activation of DNA repair pathways. Furthermore, knockdown of either TOP1α or TOP3β impaired viral replication and transcription. Importantly, these two topoisomerases act by distinct mechanisms to regulate viral functions. Elucidating the mechanisms by which topoisomerases regulate viral gene expression and replication offers a promising avenue for targeted therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/843f6cbaaa32b6f9a8d2bc8fb897a0a89ef42241" target='_blank'>
              Differential roles of type I topoisomerases in regulating HPV pathogenesis
              </a>
            </td>
          <td>
            A. Vats, Conor W Templeton, Lou Laimins
          </td>
          <td>2026-01-02</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The total DNA of an organism is called the genome. DNA contains all the instructions for how to build an organism and how that organism will function. Some DNA sequences are repeated thousands and thousands of times throughout the genome. Repeats lined up one after another are called satellite DNA. The number of copies of a given satellite DNA sequence can rapidly change, and differ among individuals. Although satellite DNAs were once considered useless, researchers are continuously discovering their important roles in various organisms. Satellite DNAs are essential for keeping organisms functioning properly. They help cells divide and keep the genome integrity. They can affect behavior, health, and help the organism to overcome stressful conditions. In these ways, satellite DNAs increase Earth’s biodiversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e93be9ccc179d27f89aeee0b54094657efcfefe" target='_blank'>
              Mysteries of Repetitive DNA Segments
              </a>
            </td>
          <td>
            Eva Šatović-Vukšić, Miroslav Plohl
          </td>
          <td>2025-12-29</td>
          <td>Frontiers for Young Minds</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Most deaths from cancer are caused by their metastases. Along with the ability to evade the immune system and to invade and disrupt very different tissue environments, metastases usually develop resistance to whatever therapeutic approaches are tried. Their extreme adaptability requires a diversity of traits from which natural selection can choose, but current models, such as the epithelial-to-mesenchymal transition (EMT), do not directly address how this diversity is generated. We observed that single cell clones of Panc1 human pancreas cancer cells that had had crispR knock outs (KO) of the gene for activin-like kinase 4 (ALK4) had developed a markedly diverse morphology. Time-lapse and fluorescence microscopy, FACS analysis and mitotic chromosome squashes provided evidence that the cells diversify profoundly in size, behavior and chromosome number. This diversity appears to develop through cytokinesis failure, the generation of very large multinuclear cells that exhibit a high range of nuclear configurations, coupled with continued cell divisions that sometimes generate three, four, and more daughter cells in one cell division. This astounding activity often resulted in cells that were much smaller than the normal Panc1 cells, and indirect evidence suggests they were significantly sub-ploidy, yet underwent regular cell divisions. A subset of the smaller cells were highly motile, and were observed to sometimes merge together or even into larger cells. This phenomenon could provide an unappreciated venue for transferring mutated genes, chromosome fragments, whole chromosomes or sets of chromosomes into an invaded cell. We suggest that these observations may be a serendipitous illustration of the missing link between the genetic mutations implicated in the development of primary carcinoma, and the extreme genomic diversity that characterizes malignant metastases. Graphical Summary Single cell clones of Panc 1 ALK4 KO cultures changed from largely uniformly epithelioid in character (A) to cultures composed of an unexpectedly wide variety of cells, and that this variety is not explained by the traditional “EMT” (epithelial-to-mesenchymal-transition). These cells range from very small to very large, have varying degrees of motility, chromosome number, and cell division behavior. We describe the new morphologies and behaviors, and present time lapse movies and microscopic evidence that help explain the evolution of this variety. These new cell types integrate into colonies where the smallest cells gather into spherical masses (D). We believe that the heterogeneity in genomic content and behavior provides the variety needed for natural selection to select cells that are resistant to most treatments for metastatic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba6b16627321b67c2cf57a151fe0aa1bc2c09235" target='_blank'>
              A Model for the Diversity Explosion Fundamental to Metastasis
              </a>
            </td>
          <td>
            Edward Tim O’Brien
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

 Circular RNAs (circRNAs), a unique class of endogenous noncoding RNAs characterized by their covalently closed loop structure, have emerged as pivotal players in cancer biology due to their exceptional stability, tissue‐specific expression, and diverse regulatory functions. This comprehensive review delves into the multifaceted roles of circRNAs as epigenetic modulators in cancer pathogenesis, systematically exploring their biogenesis via back‐splicing—regulated by
 cis
 ‐elements and
 trans
 ‐factors—and their general functions, including directly recruiting chromatin‐modifying enzymes or DNA methyltransferases or by sequestering noncoding RNAs that feed back to the chromatin, thereby establishing heritable and DNA sequence–invariant expression states. A distinctive feature of this work is its focused examination of how circRNAs interface with key epigenetic regulators, such as histone modifiers, DNA methyltransferases, and chromatin remodeling complexes, thereby influencing gene expression and driving tumor initiation, metastasis, and drug resistance. The review further integrates recent insights from single‐cell sequencing and discusses stoichiometric and structural nuances that define authentic miRNA sponge functions, setting it apart from previous summaries. By synthesizing current evidence and highlighting both mechanistic depth and clinical relevance—such as the potential of circRNAs as biomarkers and therapeutic targets—this article not only advances our understanding of circRNA‐mediated epigenetic dysregulation in cancer but also outlines future research directions and technological challenges, underscoring its significance in bridging basic molecular insights with translational oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df145e1c14bbc9d33031580b823f8b55632795bd" target='_blank'>
              Circular RNAs: Epigenetic Puppeteers Pulling the Strings of Cancer Pathogenesis
              </a>
            </td>
          <td>
            Yimao Wu, Zichang Chen, Junying Lee, Gökhan Zengin, Mengyao Li
          </td>
          <td>2026-01-14</td>
          <td>Medicine Bulletin</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/422a810751b71895a76ea9dd11ff1a3be414db1d" target='_blank'>
              The FANCM-RMI1/2 complex promotes genomic instability and PARP inhibitor sensitivity in BRCA2-deficicient cells
              </a>
            </td>
          <td>
            François Dossin, Mikael Attia, Fabricio Montero Zúñiga, Daniel González Acosta, R. Brough, Fei Fei Song, Aurélien Bore, Raphaël Margueron, Massimo Lopes, C. J. Lord, L. Deriano
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="ABSTRACT Human papillomavirus 16 (HPV16) is a causative agent of oropharyngeal, cervical, and anogenital cancers. The viral E2 protein is essential for viral genome replication, transcriptional regulation, episome maintenance, and activation of the host DNA damage response. Despite its central role, the full network of HPV16 E2 interactions with host proteins remains incompletely defined, particularly under differentiating conditions, which support the complete viral life cycle. In this study, we used TurboID-based proximity labeling to characterize the interactome of HPV16 E2 and its known host partner protein TOPBP1 in both undifferentiated monolayer and differentiating keratinocytes. We generated stable keratinocyte lines expressing doxycycline-inducible TurboID-tagged HPV16 E2 and confirmed that the tagged protein retained transcriptional, replicative, and DNA damage-inducing functions. Mass spectrometry analysis of streptavidin-enriched proteins identified both known and novel E2-associated host factors, including chromatin regulators, DNA repair proteins, and nucleolar components. Comparative analysis revealed substantial overlap between E2 and TOPBP1 interactomes, and in situ validation by proximity ligation assay identified nucleolin (NCL) as a differentiation-dependent factor whose interaction with E2 is stabilized by TOPBP1. Functional studies demonstrated that NCL is required for episomal genome maintenance, highlighting a cooperative E2–TOPBP1–NCL axis critical for viral genome stability during differentiation. These findings provide a comprehensive view of the E2-associated protein landscape in stratified epithelial cells and reveal a mechanistic pathway through which HPV16 co-opts host factors to support genome maintenance, productive replication, and persistence. IMPORTANCE Human papillomaviruses (HPVs) establish persistent infections in stratified epithelia and rely on host DNA damage and repair factors to support their replication. The E2 protein is central to viral genome replication and maintenance and depends heavily on its interaction with the host factor TOPBP1 for these functions. Here, we define the E2 and TOPBP1 interactomes in differentiating keratinocytes and identify nucleolin (NCL) as a critical differentiation- and TOPBP1-dependent E2 partner required for episomal genome stability. These findings expand the understanding of how HPV16 coordinates viral replication with host chromatin and DNA repair networks, uncovering a cooperative E2–TOPBP1–NCL axis that may represent a new target for antiviral intervention. Human papillomaviruses (HPVs) establish persistent infections in stratified epithelia and rely on host DNA damage and repair factors to support their replication. The E2 protein is central to viral genome replication and maintenance and depends heavily on its interaction with the host factor TOPBP1 for these functions. Here, we define the E2 and TOPBP1 interactomes in differentiating keratinocytes and identify nucleolin (NCL) as a critical differentiation- and TOPBP1-dependent E2 partner required for episomal genome stability. These findings expand the understanding of how HPV16 coordinates viral replication with host chromatin and DNA repair networks, uncovering a cooperative E2–TOPBP1–NCL axis that may represent a new target for antiviral intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ded8316db339bd64df9132a4a8179bca47c1336a" target='_blank'>
              Differentiation-dependent proximity proteomics identifies novel host factors linked to HPV16 E2 function
              </a>
            </td>
          <td>
            C. James, Aya H. Youssef, A. T. Prabhakar, Jenny D Roe, Elinor Lu, Austin J. Witt, S. Giri, Molly L Bristol, Phoebe Bridy, Xu Wang, Arjun Rijal, Charles Lyons, I. Morgan
          </td>
          <td>2026-01-12</td>
          <td>mBio</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdbf7c97f85876b986624ce952ff6a0df44fa34d" target='_blank'>
              A genomic and epigenomic view of human centromeres.
              </a>
            </td>
          <td>
            Kate E Jaggi, Savannah J. Hoyt, Rachel J. O’Neill, B. Sullivan
          </td>
          <td>2026-01-06</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df3fdd78ac1c63cef7b04dfd2c26d05276098c61" target='_blank'>
              Visualizing Interchromosomal Interactions at Sub-Megabase Resolution Using Network Clustering Coefficients
              </a>
            </td>
          <td>
            Yingjie Xu, Ian J. Anderson, R. P. McCord, Tongye Shen
          </td>
          <td>2026-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/707bcd4a686a2ce90d2809d1732e3cd5eef3e800" target='_blank'>
              Chromosome-associated spot formation by human cytomegalovirus immediate early 1 (IE1) protein
              </a>
            </td>
          <td>
            Mamata Savanagouder, Tejasv Gupta, M. Messerle, E. Borst, T. F. Schulz, Divya Das, E. Poole, John Sinclair, Wojciech Zdanowski, Tomasz Waśniewski, Marek Zygmunt, Sławomir Wołczyński, Magdalena Weidner-Glunde
          </td>
          <td>2025-12-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Simple Summary In many types of cancer, cells double their entire set of chromosomes, a state known as tetraploidy. This change can accelerate the cancer’s growth, spread, and resistance to drugs. We wanted to understand how this chromosome doubling changes the cell’s mitochondria. We compared cells with a normal versus a doubled set of chromosomes from human cancers, yeast, and fungi. We found that in all cases, cells with doubled chromosomes were larger and contained more mitochondria, which were also more active. By identifying this shared trait, our research points to these adapted mitochondria as a potential new target for treating cancers with unstable chromosomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9e2f77cdf8945e8a66c23348025539c8ac24aa7" target='_blank'>
              Mitochondrial Adaptations Underlying Tetraploidization in Human Cancer, Fungal, and Yeast Models
              </a>
            </td>
          <td>
            Mohamed Jemaà, Ameni Bedoui, Nihel Ammous, Ali Gargouri, Mohamed Guerfali
          </td>
          <td>2026-01-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, may also play an important role in tetraploid (4N) cancer cell physiology. Here, we showed that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. Small size was associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size may contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e3ce0e5eeb35d45bde98218a3b940e6aa2d9ace" target='_blank'>
              Cell and Nuclear Size are Associated with Chromosomal Instability and Tumorigenicity in Cancer Cells that Undergo Whole Genome Doubling.
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney M Huth, Daniella S McCausland, Ron Saad, Nazia Bano, Tran N Chau, Megan L Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, U. Ben-David, Daniela Cimini
          </td>
          <td>2026-01-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Essential meiotic structure‐specific endonuclease 1 (EME1) is integral to the maintenance of genomic stability in various cancers. However, its biological role and expression profile of this molecule in nasopharyngeal carcinoma (NPC) remain to be explored. In this study, we found that EME1 was overexpressed in NPC specimens compared to adjacent noncancerous tissues and was correlated with poorer overall survival outcomes. Furthermore, EME1 knockdown significantly inhibited the proliferation and migration of NPC cells in vitro and in vivo, with a corresponding sensitization to either camptothecin (CPT) or olaparib, evidenced by a further suppression of proliferation upon drug treatment. Notably, silencing EME1 significantly increased the sensitivity of NPC cell lines to CPT by enhancing ATM‐CHEK2 phosphorylation and inducing nuclear abnormalities. Collectively, our findings suggest that combining EME1 modulation with agents such as CPT or olaparib could be an effective treatment strategy for NPC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64d4b0d132ea96aa28bec17ad81c4b1af8056bcb" target='_blank'>
              Targeting EME1 Increases the Sensitivity of Camptothecin in Nasopharyngeal Carcinoma Cells
              </a>
            </td>
          <td>
            Xizhen Jiang, Falian Liang, Zhirui Lin, Fang Yang, Dong-ping Chen, Gengde Hong, Jinquan Liu, Wenjing Yin, Mengyao Wang, Bin Qi
          </td>
          <td>2026-01-14</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a highly aggressive cancer of the biliary tract, distinguished by significant intratumoral heterogeneity (ITH), which contributes to therapy resistance and unfavorable clinical outcomes. Traditional genome profiling has revealed recurring driver changes in CCA; yet, genomic data alone fails to elucidate functional pathway activation, adaptive signaling, and the diverse treatment responses reported among tumor locations and disease subtypes. This review analyses the use of integrated sequencing technologies, proteogenomics, and phosphoproteomics to systematically characterize intratumoral heterogeneity in cholangiocarcinoma and convert molecular diversity into therapeutically applicable discoveries. We present evidence that the combination of genomic sequencing and mass spectrometry–based proteomics facilitates the direct correlation of genetic mutations with protein expression, post-translational modifications, and signaling system activity. Phosphoproteomic profiling specifically offers functional insights into kinase-driven networks that dictate tumor aggressiveness, therapeutic susceptibility, and adaptive resistance mechanisms, which cannot be anticipated only from DNA-level analysis. We propose that integrating proteogenomic and phosphoproteomic analyses into diagnostic and therapeutic assessments can enhance molecular classification, reveal subtype- and region-specific therapeutic dependencies, and guide rational combination treatment strategies, based on recent extensive proteogenomic studies and functional proteomic investigations in CCA. Pathway-level analysis of intratumoral heterogeneity provides a framework for selecting targeted medicines, predicting resistance, and informing personalized treatment strategies in CCA. The combination of sequencing, proteogenomics, and phosphoproteomics is essential for advancing precision oncology in cholangiocarcinoma. The implementation of this multi-layered analytical approach may better patient classification, refine therapy choices, and eventually improve clinical outcomes for individuals with this particular heterogeneous cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a96c2c6b9428d020026e895b0e1d0644f1ab872b" target='_blank'>
              Integrative Sequencing and Proteogenomic Approaches to Intratumoral Heterogeneity in Cholangiocarcinoma: Implications for Precision Diagnosis and Therapy
              </a>
            </td>
          <td>
            Sirinya Sitthirak, Arporn Wangwiwatsin, Apinya Jusakul, N. Namwat, Poramate Klanrit, Sitthirug Roytrakul, Hasaya Dokduang, Thitinat Duangchan, Yanisa Rattanapan, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Teh Bin Tean, Luke Boulter, Yoshinori Murakami, W. Loilome
          </td>
          <td>2026-01-07</td>
          <td>Medical Sciences</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c066b01c86150bb1cb77ee7e3befe19a5fcc47" target='_blank'>
              Orphan Non-Coding RNAs Drive Tumorigenesis and Enable Pre-Diagnostic Detection in HPV-Negative Head and Neck Cancer
              </a>
            </td>
          <td>
            Hoang C. B. Nguyen, Lishi Li, M. Karimzadeh, Jack Ghanam, Jeffrey Wang, Yana Al Inaya, G. Lumaj, F. Hormozdiari, B. Alipanahi, A. Feng, J. Richmon, Daniel G. Deschler, M. Varvares, D. Faden, Derrick Lin, Hani Goodarzi
          </td>
          <td>2025-12-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="RecQ family of DNA helicases play pivotal roles in DNA replication, repair and responses to DNA damage or replication stress. Several human RecQ helicases are defective in diseases associated with chromosomal instability, premature aging and cancer. We recently discovered novel mutations in the RECQL4 gene in glioblastoma (GBM), the most malignant brain tumor in adults. Transcriptomic profiles of GBMs with REQCL4 mutations resembled those in REQCL4 KO glioma cells. We employ structural modelling and biochemical approaches to elucidate impacts of novel mutations on RECQL4 helicase activities. Using recombinant RECQL4P532S and RECQL4R766Q proteins we demonstrate that P532S substitution reduces the RECQL4 ability to unwind DNA and disrupts DNA-coupled ATP-hydrolysis activity. WT and mutated RECQL4 were overexpressed in RECQL4 KO glioma cells to study interactions with BLM helicases, cell viability and specific responses to UVC- and chemotherapy-induced DNA damage/repair. Overexpression of RECQL4P532S or RECQL4R766Q variants affected DNA repair and responses to chemotherapeutics in glioma cells, and RECQL4R766Q disturbed interactions with the BLM helicase. Our results reveal deleterious consequences of novel RECQL4 mutations in GBMs. The newly identified RECQL4 mutations affect RECQL4 helicases and their interactions with BLM contributing to glioma progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0650345dd7db490bb44b3ecdcc3b0ad865cd59a8" target='_blank'>
              Novel mutations in the RECQL4 gene affect its helicase functions, interactions with the BLM helicase and chemotherapeutics-induced cell death
              </a>
            </td>
          <td>
            Agnieszka Kaczmarczyk, Mikołaj Sokołowski, K. Wojnicki, Marta Pabiś, B. Wojtaś, I. Ciechomska, K. Poleszak, B. Gielniewski, S. K. Król, Matthew Guille, Sebastian Glatt, B. Kaminska
          </td>
          <td>2025-12-01</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The genome folds inside the cell nucleus into hierarchical architectural features, such as chromatin loops and domains. If and how this genome organization influences the regulation of gene expression remains only partially understood. The structure-function relationship of genomes has traditionally been probed by population-wide measurements after mutation of critical DNA elements or by perturbation of chromatin-associated proteins. To circumvent possible pleiotropic effects of such approaches, we have developed OptoLoop, an optogenetic system that allows direct manipulation of chromatin contacts by light in a controlled fashion. OptoLoop is based on the fusion between a nuclease-dead SpCas9 protein and the light-inducible oligomerizing protein CRY2. We demonstrate that OptoLoop can bring together genomically distant, repetitive DNA loci. As a proof-of-principle application of OptoLoop, we probed the functional role of DNA looping in the regulation of the human telomerase gene TERT by long-range contacts with the telomere. By analyzing the extent of chromatin looping and nascent RNA production at individual alleles, we find evidence for looping-mediated repression of TERT. In sum, OptoLoop represents a novel means for the interrogation of structure-function relationships in the genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5676ff6c24cf2aa8e41cc35c8a6bbe06bad743e9" target='_blank'>
              OptoLoop: An optogenetic tool to probe the functional role of genome organization.
              </a>
            </td>
          <td>
            Martin Stortz, Adib Keikhosravi, Gianluca Pegoraro, Tom Misteli
          </td>
          <td>2026-01-19</td>
          <td>Journal of cell science</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Homologous recombination (HR) is traditionally portrayed as a DNA double-strand break repair pathway. However, emerging evidence positions RAD51, its partners BRCA1, BRCA2, and other HR factors at the core of a broader genome-maintenance network that operates by a “prevent and protect” strategy extending beyond repair. Here, we review how RAD51 can shield DNA from nucleolytic processing mediated by MRE11 and related nucleases, promote fork reversal, suppress replicative DNA gaps accumulation, and bind abasic sites, averting their conversion into cytotoxic intermediates. These extended functions counteract endogenous replication stress as shown in BRCA1- or BRCA2-deficient contexts, where failure to prevent gaps, protect forks, and safeguard abasic DNA accelerates genomic instability. The functional impairment of HR proteins, which interface with base-excision repair and translesion synthesis, rewires these pathways, driving distinctive base-substitution mutational signatures of HR-defective tumors. Abasic sites, especially from methyl-cytosine metabolism, put replication forks at risk of breaking, amplifying the need for RAD51-mediated defense. Such redefinition of homologous recombination protein function as part of an anticipatory surveillance and protective system, rather than a repair-only module, bears important implications for understanding tumorigenesis, therapy resistance, and aging. Vincenzo Costanzo and coworkers review how RAD51 can shield DNA from nucleolytic processing, promote fork reversal, suppress replicative DNA gaps accumulation, and bind abasic sites to avert their conversion into cytotoxic intermediates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edfd2702367051da70ddb6fa2941183d106513bc" target='_blank'>
              The expanding roles of homologous recombination proteins in genome stability
              </a>
            </td>
          <td>
            Lorenzo Sassi, Andrea Martinez Marroquin, Salli Waked, Alessandra Ardizzoia, Vincenzo Costanzo
          </td>
          <td>2026-01-03</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Loss-of-function mutations in DNMT3A, a DNA methyltransferase, or NSD1, a histone methyltransferase, cause overgrowth syndromes. Conversely, disruption of the DNMT3A domain that binds NSD1-deposited H3K36 dimethylation (H3K36me2) results in growth restriction. To investigate the molecular basis of these opposing growth outcomes, we generated isogenic human embryonic stem cells carrying growth syndrome–associated mutations in DNMT3A and NSD1. Unexpectedly, both overgrowth- and growth restriction–associated DNMT3A mutations led to DNA hypomethylation in a shared subset of active enhancers, implicating H3K36me2 in directing enhancer methylation maintenance. In contrast, bivalent promoters—marked by both active and repressive histone modifications—showed divergent DNA methylation changes: hypermethylation in growth restriction-associated DNMT3A mutants and hypomethylation in overgrowth-associated DNMT3A or NSD1 loss-of-function mutants. These findings identify locus-specific DNA methylation defects as a common molecular feature and nominate dysregulated DNA methylation at bivalent promoters as a potential driver of abnormal growth phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f699bcc558592153abc552376bed77e07ec2c5e0" target='_blank'>
              Convergent DNA methylation abnormalities at enhancers and bivalent promoters in human growth disorders
              </a>
            </td>
          <td>
            Marie E S Wheeler, Yoshiko Takahashi, Jihye Lee, Camille T Perez, Xiaoting Chen, Yuri Lee, Zachary S. Pope, Daniella J Lu, Marcus M. Seldin, Ivan Marazzi, Hongseok Yun, M. Weirauch, Minji Byun
          </td>
          <td>2025-12-27</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a5c7aee5589d3f24718b4f0d1ab22f036f7184d" target='_blank'>
              Retrotransposon Activation in the Aged and Alzheimer’s Disease Brain Examined by Nanopore Long-read DNA Sequencing
              </a>
            </td>
          <td>
            M. Kelsey, Anjalika Chongtham, J. LaCava, Martin S. Taylor, J. Boeke, F. H. Gage, A. Seluanov, Vera Gorbunova, Ana C. Pereira, J. Sedivy
          </td>
          <td>2025-12-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>114</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/271f2b42923ccb0c522e0defc7aee243eaf5e39d" target='_blank'>
              Reconstructing clone-resolved transcriptional programs from bulk tumor sequencing
              </a>
            </td>
          <td>
            Jiaying Lai, Yi Yang, Kathleen Noller, YunZhou Liu, Archana Balan, Prathima B. Nagendra, Luciane T. Kagohara, E. Fertig, Laura D. Wood, Rachel Karchin
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Characterizing the physical organization of the genome is essential for understanding long-range gene regulation, chromatin compartmentalization, and epigenetic accessibility. Hi-C experiments generate two-dimensional (2D) genome-wide contact maps of chromatin interactions by capturing the spatial proximity between genomic loci, which reveal interaction frequencies but lack the spatial resolution needed to interpret the three-dimensional (3D) genome structure(s). Emerging evidence suggests that epigenetic regulation is closely linked to 3D genome architecture, and that structural changes over time (4D) drive key biological processes in development, disease, and environmental response. Thus, integrating 3D structure with functional data is critical for a more complete understanding of genome regulation. Previous work, most notably the 4DHiC chromosome modeling framework, has shown that physical multi-dimensional modeling approaches rooted in polymer physics and molecular dynamics can resolve these structures at biologically meaningful resolutions by integrating temporal Hi-C data with physical constraints to uncover dynamic chromosome reorganization. Thus, molecular dynamics simulations, constrained by Hi-C contact matrices, can resolve fine-scale structural changes and reveal functionally significant transitions in chromatin conformation. Herein, we present the 4D Genome Browser Workflow (4DGBWorkflow) and the 4D Genome Browser (4DGB). The algorithm is based on the 4DHiC method, and the containerized tool is an end-to-end workflow that can transform, filter, and view 4D epigenomics and chromatin datasets, allowing non-specialists to apply three-dimensional modeling principles to diverse datasets and experimental conditions. The software executes on a laptop running macOS, Linux or Windows. From input Hi-C files (.hic), the 4DGBWorkflow produces 3D reconstructions of chromosomes, integrates the reconstruction with track data (e.g., epigenetic marks, transcriptome profiles), and provides comparative visualization of the results in a single workflow. The 4DGBWorkflow and 4D Genome Browser are open-source tools for comparative analysis and visualization of 4D chromosome datasets, including chromatin architecture and epigenomic signals. Automatic integration of Hi-C data with molecular dynamics democratizes the construction of time resolved 3D genome structures, simplifying complex simulations and data integration schemes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e99ba06ed5c338fe5fbe39dbd1637f6375018c6c" target='_blank'>
              From 2D to 4D: a containerized workflow and browser to explore dynamic chromatin architecture
              </a>
            </td>
          <td>
            David H. Rogers, Cullen Roth, Cameron Tauxe, Jeannie T. Lee, Christina R. Steadman, K. Sanbonmatsu, Anna Lappala, Shawns Starkenburg
          </td>
          <td>2026-01-08</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Organisms regulate cell size and shape to function efficiently. Aberrant cell morphogenesis is commonly associated with disease, yet gene-regulatory mechanisms remain unknown. CISTR-ACT was the first lncRNA involved in inter-chromosomal proximities and Mendelian disease, and it is associated with mean corpuscular volume (red blood cell size). Here, functional dissection of CISTR-ACT’s DNA- and RNA-encoded mechanisms by in vitro and in vivo perturbations reveals that CISTR-ACT regulates cell size across cell types and species. CISTR-ACT’s locus is embedded in a stable inter-chromosomal environment which contains cell size genes that are regulated by CISTR-ACT in trans. CISTR-ACT’s RNA also has function and directly interacts with transcription factor FOSL2 to guide its regulation of cell morphogenesis and cell-cell adhesion genes. In the absence of CISTR-ACT, the FOSL2-chromatin binding is perturbed. Our study exemplifies how a functionally conserved lncRNA regulates cell size with multiple modes of action and ultimately contributes to clinically relevant phenotypes. Establishment and maintenance of appropriate cell size is a prerequisite for cells to function efficiently. Here, Kiriakopulos et al. reveal that the lncRNA CISTR-ACT maintains cell size across cell types in humans and mice by regulating cell morphogenesis genes in trans via guidance of the transcription factor FOSL2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2257e0e8bb5b6556885e88aef4e0f1ede88e699" target='_blank'>
              LncRNA CISTR-ACT regulates cell size in human and mouse by guiding FOSL2
              </a>
            </td>
          <td>
            Katerina Kiriakopulos, Katty Soleimanpour, Brandon J. McMurray, Benjamin-Israel Moke, Jordan J. Chalmers, Milad Mokhtaridoost, Jeremy D Newton, Taylor De Young, Kate Delfosse, Mai Ahmed, Cassandra J. Wong, S. Stricker, Yun Li, Brian J. Nieman, A. Gingras, Monica J. Justice, Julie L. Lefebvre, Philipp G. Maass
          </td>
          <td>2025-12-16</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>110</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumor (ATRT) is a highly aggressive but genetically simple pediatric central nervous system tumor, defined by biallelic inactivation of the chromatin regulator SMARCB1 with remarkably few other cooperating mutations. Despite its genetic homogeneity, ATRT exhibits profound clinical and epigenetic heterogeneity, with three major subgroups (ATRT-TYR, ATRT-MYC, and ATRT-SHH) defined by DNA methylation and transcriptional signatures. Beyond these subgroup-defining features, we aimed to investigate epigenetic variability within tumors by applying whole-genome bisulfite sequencing and probabilistic modeling to quantify stochastic DNA methylation in primary ATRT samples encompassing all three subgroups. We show that ATRT exhibits a destabilized and increasingly stochastic methylome. While ATRT global methylation patterns diverge according to subgroup, some methylation perturbations, such as hypermethylation and increased methylation entropy over bivalent promoters, are consistent across subgroups. We find that methylation stochasticity alterations map onto potential drivers of ATRT, such as LIN28a, the HOXD cluster for ATRT-MYC, and OTX2 for ATRT-TYR, and identify actionable targets, such as hypermethylation of the tumor suppressor CDKN2a across all subgroups. We investigate the sensitivity of the aberrant DNA methylation landscape of ATRT to pharmacologic DNA methyltransferase inhibition (DNMTi) and histone deacetylase inhibition (HDACi). We show that decitabine leads to profound demethylation of patient-derived ATRT cell lines, including reversal of hypermethylation at bivalent promoters and the CDKN2a locus. The addition of HDACi leads to dramatic gene expression changes, including upregulation of innate immune signaling pathways, such as STING/interferon signaling, genes under the regulation of bivalent promoters, and reactivation of the tumor suppressor CDKN2A. The combination of DNMTi and HDACi synergistically reduces cell viability. Taken together, we show that ATRT has a highly stochastic methylome sensitive to epigenetic manipulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e14cbda965cb78ec2d086d8bbd22a534ed3d3a3" target='_blank'>
              Disordered DNA methylation leads to targetable transcriptional plasticity in ATRT
              </a>
            </td>
          <td>
            Ashley R Tetens, Tyler R Findlay, Jordyn Craig-Schwartz, Athanasia Liapodimitri, Oscar Camacho, Kegan O Skalitzky, Adrian Idrizi, Rakel Tryggvadottir, Kayleigh Lunsford, Eric H. Raabe, Michael A Koldobskiy
          </td>
          <td>2025-12-17</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="DEAD-box (DDX) RNA helicases are essential regulators of RNA metabolism and gene expression. Among them, DDX10 remains poorly characterized despite growing evidence supporting its involvement in human diseases. This review provides a comprehensive analysis of DDX10, from its structural and functional features to its emerging roles in solid tumors and hematologic malignancies. We discuss how DDX10, through its conserved domains, contributes to pre-rRNA processing, ribosome biogenesis, and cell proliferation, and explore potential links between DDX10 and processes such as liquid–liquid phase separation (LLPS) and epigenetic regulation, which may underlie its roles in cancer cell plasticity and stress response. We argue that the dysregulation of these fundamental cellular processes positions DDX10 as a focal point where aberrant RNA metabolism and altered molecular condensates converge to disrupt transcriptional homeostasis and drive oncogenic transformation. Aberrant DDX10 expression is a recurrent feature across multiple cancers, where it promotes tumor progression, therapy resistance, and poor prognosis. Moreover, DDX10 participates in oncogenic fusion events, most notably the NUP98::DDX10 fusion identified in a subset of acute myeloid leukemias, which drives leukemogenesis by disrupting transcriptional regulation and cellular differentiation. Given its tumor-associated expression and diverse biological functions, DDX10 is increasingly recognized as a potential diagnostic biomarker and a promising target for therapeutic strategies. By consolidating current knowledge under this unifying framework, this review highlights the multifaceted roles of DDX10 in cancer biology, advocating further research into its molecular functions and translational potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6180292fbacfd0407c4b7281fb3b5202eef78597" target='_blank'>
              DDX10 RNA Helicase: Structure, Function, and Oncogenic Roles Across Solid and Hematologic Tumors
              </a>
            </td>
          <td>
            Giorgia Isinelli, Genny Scacci, Arianna Capocchia, C. Emiliani, Cristina Mecucci, R. la Starza, Danika Di Giacomo
          </td>
          <td>2026-01-27</td>
          <td>Genes</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Compacting chromatin within the cellular nucleus presents a significant challenge for biology. Chromosomes must be both condensed and spatially organized to enable essential processes such as transcription and replication. Chromosome conformation capture experiments (e.g., Hi-C) provide valuable information about the spatial organization and, therefore, the connectivity between different genomic regions. These experiments inspired polymer models that describe the physical mechanism of the chromosomal energy landscape. The Full-Inversion Chromatin model (FI-Chrom), a data-driven approach for modeling genome organization, uses Hi-C contact maps to infer pairwise interaction potentials between all chromosomal loci. It combines Graphics Processing Unit (GPU)-accelerated simulations with efficient training of tens of millions of parameters derived from the maximum-entropy principle to determine 3D structures of chromosomes that accurately reproduce Hi-C-like data. FI-Chrom does not make any a priori assumptions regarding chromosome architecture, making it applicable to any chromosome conformation capture experiment. Its derived structural ensembles capture all essential features from the short- and long-range interactions of typical chromosome organization, such as segregated compartments, chromosome territories, and fully or partially formed loops. Although Hi-C contains only structural information, FI-Chrom extends these data by revealing an emergent dynamical mechanism encoded in the inferred energy landscape. For example, simulations show that chromatin loops are not static architectural features but rather transient structural elements. Statistical analyses further indicate that loops confined within a single compartment occur more frequently than those spanning multiple compartments, highlighting the dynamic and compartment-dependent nature of chromatin organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5e464495ba1f67ebcfcfd2e8d42c98c8759239d" target='_blank'>
              A data-driven chromatin model reveals spatial and dynamic features of genome organization.
              </a>
            </td>
          <td>
            A. B. Oliveira Junior, Matheus F. Mello, Ronaldo J Oliveira, Esteban Dodero-Rojas, Sumitabha Brahmachari, V. Contessoto, J. Onuchic
          </td>
          <td>2026-01-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>98</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52aead1bb294c5c8ab3425b1373986d70ceb1c0f" target='_blank'>
              Blocking Minor Intron Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer
              </a>
            </td>
          <td>
            Anke Augspach, P. Francica, Kaitlin N. Girardini, Muriel Jaquet, Marika Lehner, A. Rodriguez-Calero, Charlotte Komarek, Martín González-Fernández, Hannah L. Williams, Xiaoyue Deng, Lea Lingg, Cristina Zivko, Vera Fuchs, Seynabou Diop, S. Maletti, Simone de Brot, A. Ewe, Elena Perugini, Nigel B Jamieson, Y. Doncheva, Claire K Dietrich, Phillip Thienger, Dilara Akhoundova, Andrej Benjak, A. Aigner, R. Kanadia, Sven Rottenberg, Mark A. Rubin
          </td>
          <td>2025-12-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e82aa8297f80f569a06497cad1d25c2344da9284" target='_blank'>
              The 3D Genome of Gigaspora margarita Unveils Stable Chromatin and Nucleolar Organization and Symbiont-Dependent Genome Dynamics
              </a>
            </td>
          <td>
            Ken Mugambi, Jordana Oliveira, F. Magurno, Alessandra Salvioli di Fossalunga, M. Novero, L. Lanfranco, Stefano Ghignone, Gokalp Yildirir, Yan Wang, Paola Bonfante, Nicolas Corradi
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The evolutionary history of cancers is encoded in molecular patterns that provide critical insights into the mechanisms of tumor progression. However, reconstructing cancer lineages in patient tumors remains challenging, as conventional sequencing captures only static snapshots of an ongoing evolutionary process. DNA methylation at CpG sites is a heritable epigenetic mark that accumulates stochastic errors - termed epimutations - at rates far exceeding those of somatic DNA mutations. These epimutations serve as a 'molecular clock' and can be harnessed for retrospective lineage tracing. When applied at single-cell resolution, methylation-based lineage tracing enables the reconstruction of cancer phylogenies and provides a powerful framework for studying the dynamics of treatment resistance, cell-state heritability, and metastasis directly in human tumors. In this review, we outline the strengths and challenges of single-cell DNA methylation-based lineage tracing, highlight key insights gained from its application with an emphasis on hematological cancers where relevant, and discuss its future potential in advancing our understanding of cancer evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1728d59760495508f83df1d25fc69028b0e41513" target='_blank'>
              Methylation-based lineage tracing in cancer.
              </a>
            </td>
          <td>
            Jiaoyi Chen, Benson Wu, Federico Gaiti
          </td>
          <td>2025-12-15</td>
          <td>Blood</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a21b610301b7b55b77a33ac6317a40f28dcdb985" target='_blank'>
              A gastric microbial chromatin remodeler drives gastric cancer progression and immune evasion by reprogramming the host epigenome
              </a>
            </td>
          <td>
            Xiaoshan Xie, Yue Wei, Zhikai Zheng, Jiaying Zheng, Xijie Chen, Jiarui Wang, Ning Ma, Xiaoling Huang, Peng Zhang, Boyu Zhang, Hanyong Cai, Li Ma, Lishi Xiao, Qingxin Liu, Wenyu Wang, Sachiyo Nomura, Shi Chen, Xiangqi Meng, Mong-Hong Lee
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Background and objectives Somatic mutation patterns in cancer remain largely unexplored outside humans, despite their significance for aging and oncogenesis. Chimpanzees (Pan troglodytes), sharing >98% genomic similarity with humans, display markedly different cancer spectra. To gain comparative insights into cancer susceptibility and resistance, we sequenced chimpanzee hepatocellular carcinoma (HCC) genomes and analyzed their mutational profiles alongside human counterparts. Methodology HCC and matched non-cancerous tissues from five chimpanzees were examined using histopathology, immunohistochemistry (β-catenin, ARID1A, TSC2, FAP, vimentin, TGF-β), whole-genome sequencing (one pair), and whole-exome sequencing (four pairs). Somatic variants were identified with GATK MuTect2, annotated with Ensembl VEP, and analyzed for functional enrichment. Comparative analyses were performed with subsets of human HCC datasets (TCGA, ICGC) including TSC2-positive and TSC2-negative cases. Results Chimpanzee HCCs exhibited histological and immunohistochemical features similar to human tumors but displayed sharply divergent genomic landscapes. Chimpanzee tumors carried significantly higher coding mutation loads (mean 5632 per sample vs. 96–275 in humans). Non-synonymous TSC2 mutations occurred in 80% of chimpanzees, versus ~7% in human HCC, suggesting a species-specific oncogenic pathway linked to the scirrhous subtype. Additional recurrently mutated genes included ARID1A, FAT1–4, TP53, and FGA. Despite greater heterogeneity in chimpanzee tumors, humans showed stronger enrichment of non-synonymous single nucleotide variants, implying more intense positive selection. Shared alterations across species involved canonical drivers such as TP53, CTNNB1, FAT4, and TTN. Conclusions and implications Chimpanzee HCCs are defined by high mutational burden and frequent TSC2 alterations, contrasting with the more selectively constrained mutation spectrum of human HCC. Divergent evolutionary patterns highlight species-specific oncogenic routes while underscoring conserved pathways. Comparative primate cancer genomics offers novel insights into cancer evolution, biomarkers, and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c7fdead0d4830205015111b5a2fbfd3bc9dc0ec" target='_blank'>
              Divergent evolution of hepatocellular carcinoma genomes in chimpanzees and humans
              </a>
            </td>
          <td>
            Lin Kang, K. Michalak, Robin T. Varghese, R. Anandakrishnan, Edward J Dick, Z. A. Abd Elmageed, Pawel Michalak
          </td>
          <td>2025-12-15</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27e64bdfdb14f37106c0cabb353f42df99006bef" target='_blank'>
              Ancient eukaryotic immunity through genome editing of viral sequences
              </a>
            </td>
          <td>
            Lisa Mettrop, A. Lipzen, Gilles Mirambeau, K. Barry, Igor V. Grigoriev, G. Piganeau, Marc Krasovec
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="High mobility group A1 (HMGA1), a non-histone chromatin structural protein encoded by the HMGA1 gene, plays a critical role in cancer. Recent studies have increasingly focused on its functions in genomic stability and cell death, revealing its involvement in tumorigenesis, cancer progression, and chemotherapy resistance. Consequently, inhibiting HMGA1 represents a promising strategy for developing novel cancer therapies. This review summarizes the cellular and molecular functions of HMGA1 in regulating genomic integrity and cell death in cancer. Furthermore, we discuss current HMGA1-targeting strategies, with emphasis on approaches leveraging its structural and functional characteristics, aiming to provide new insights for future research on HMGA1-targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9dfc0cc9a26137eed4ac1b69fa1e9407df182b1" target='_blank'>
              The role of HMGA1 in genome stability: Implications in human cancer
              </a>
            </td>
          <td>
            Xin-Yuan Lei, Kai-yue He, Yong-Ping Jian, Z. Xu
          </td>
          <td>2026-01-26</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa668452cd533a08523fe27084c36abb8bbb84f0" target='_blank'>
              esBAF and INO8°C fine-tune subcompartments and differentially regulate enhancer-promoter interactions.
              </a>
            </td>
          <td>
            Braulio Bonilla, Benjamin J. Patty, Snehal Sambare, Job Dekker, T. Fazzio, Sarah J. Hainer
          </td>
          <td>2026-02-05</td>
          <td>Genetics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="PHF20 encodes plant homeodomain finger protein 20 (PHF20), a component of the KAT8‐containing nonspecific lethal (NSL) complex that deposits acetylation on histone H4 to activate gene expression. We report two unrelated individuals with developmental delay, microcephaly, and distinctive facial features, in whom exome sequencing and chromosomal microarray analysis revealed a homozygous deletion of PHF20 that segregated with the disease phenotype in their families. Breakpoint junction sequencing revealed an Alu–Alu‐mediated deletion event. Western blot in cells from an affected individual showed undetectable PHF20, while levels of other NSL complex subunits were unaltered. Transcriptomic and epigenomic analysis revealed significant downregulation of gene pathways related to cell projection and neuronal development, associated with reduced histone H4K16 acetylation at these genes. In conclusion, our data suggest that homozygous deletion of PHF20 leads to a neurodevelopmental syndrome, potentially through targeted epigenetic dysregulation and altered gene expression essential for neuronal development. Identifying additional families with biallelic PHF20 variants will further delineate the phenotypic spectrum, and molecular studies in neuronal cell lines will be essential for understanding the disease mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ebe2b3854a5aa35254b0ecad5b6631cc63aebb0" target='_blank'>
              Homozygous Deletion of the Epigenetic Regulator PHF20 in Individuals With Neurodevelopmental Disorder
              </a>
            </td>
          <td>
            Shira Yanovsky Dagan, Hongwen Xuan, Jonathan Rips, Emuna Paz-Ebstein, Talia Baer, Shira Gross, A. Frumkin, Xiaobing Shi, T. Harel
          </td>
          <td>2025-01-01</td>
          <td>Human Mutation</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cigarette smoke (CS) is a major risk factor for many respiratory diseases, including lung cancer, contributing to genomic instability, chronic inflammation, and impaired immune responses. Macrophages are among the most affected cell types, and CS-induced polarization has been increasingly associated with tumor promotion and reduced anti-tumor activity. G quadruplexes (G4) are non-canonical structures of RNA and DNA, enriched in G-rich sequences (such as telomeres or gene promoters), influencing several layers of gene expression regulation and DNA stability. More recently, genome-wide approaches have enabled high-resolution mapping of G4 structures in human cells, confirming the formation and functionality of G4 in multiple biological contexts. G4 structures can be bound and stabilized by small molecules such as RHPS4, with a consequent impairment of cell proliferation in cancer cells and downregulation of several pro-oncogenic factors. However, no studies have previously addressed whether CS exposure affects G4 formation or stability. Here, we found that CS exposure destabilizes G4 structure formation in cultured THP-1 monocytes differentiated into macrophages, antagonizing the effect of the G4 ligand RHPS4. In this context, CS exposure strongly induces the activation of IL-1β and TNF-α, both containing a G4 putative forming region in their promoters, suggesting that CS-linked cytokine modulation could involve a G4-dependent mechanism. Moreover, in proliferating THP-1, CS antagonizes the anti-proliferative effect of RHPS4 and has an opposite effect on the expression of G4 containing pro-oncogenic genes (like myc and bcl2). Overall, these findings showed for the first time a relationship between CS exposure and G4 structures, representing a new unexplored regulatory mechanism underlying smoking-related conditions and carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8705d372279f0282d03f4315e222b69b48ba1b1d" target='_blank'>
              Cigarette smoke destabilizes G-quadruplex structures and antagonizes G4-ligand effects in macrophages
              </a>
            </td>
          <td>
            D. Mirra, Erica Salvati, R. Esposito, G. Spaziano, Francesca Panico, Agnese Perelli, Roberto Cutolo, Salvatore Di Maro, Sandro Cosconati, Bruno D’Agostino
          </td>
          <td>2025-12-30</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background & objectives Genetic instability is frequent in tumour cells and might occur due to an imbalance of homologous recombination (HR). HR is a crucial mechanism of DNA double-strand break (DSB) repair that depends on the formation and resolution of Holliday junctions for genomic stability maintenance. The SMC6 complex with SMC5 is involved in DSB repair. We sought to investigate the association between SMC6 expression, genomic instability, and prognosis of breast cancer. Methods This was an observational retrospective cohort study. We assessed SMC6 expression and copy number variation (CNV) data measured by qRT-PCR and whole-genome comparative genomic hybridization in 33 women with breast cancer who are non-carriers of BRCA1/BRCA2 mutations. According to nuclear staining, the SMC6 protein expression was evaluated on a tissue microarrayer containing 481 samples classified as SMC6low (negative/weak) or SMC6high (moderate/strong). Results SMC6low tumours tend to show higher CNV. SMC6high group presented poorer disease-free survival than the SMC6low group (P=0.050), mainly for the luminal subtype (P=0.005). SMC6low/ERpos were protective biomarkers for recurrence. Interpretation & Conclusions There is a possible association between SMC6 expression and relapse of breast cancer, also suggesting that SMC6 abnormal expression may indicate tumour genetic instability in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cee362a99ed17056295307f3fac61c772083810" target='_blank'>
              SMC6 expression & outcome of breast cancer
              </a>
            </td>
          <td>
            F. Mangone, A. Krepischi, A. C. Pavanelli, Pedro Henrique Fernandes Gatti, D. Carraro, M. Nagai
          </td>
          <td>2025-11-01</td>
          <td>The Indian Journal of Medical Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="ABSTRACT High-quality reference genomes are essential for comparative genomics and accurate genotype-phenotype mapping. Here, we corrected the Mycobacterium tuberculosis Erdman strain reference genome (ErdmanTI) using ultra-deep HiFi sequencing. Among the small variants (n = 275) between ErdmanTI and the current Erdman reference NC_020559.1 (ErdmanSTJ), numerous are likely errors in ErdmanSTJ. We identified a novel bias toward in-frame structural variations (SVs) in pe/ppe genes and 28 SVs between ErdmanTI and ErdmanSTJ, half representing likely errors in ErdmanSTJ. Other SVs were consistent with in vitro evolution, including copy number variation (CNV) of promoter tandem repeats (PTRs). PTR CNVs were polyphyletic and within isogenic populations (10−2–10−3 CNVs/chromosome), demonstrating the impact of phase-variable CNV across evolutionary timescales. These hypervariable PTRs pinpoint a genomic basis for rapidly switching nitric oxide resistance (Dop), biofilm formation (LpdA), drug tolerance (EfpA), and glycerol utilization (GlpD2) phenotypes. This work uncovers a common phase variation mechanism obscured by short-read sequencing limitations and provides an improved reference for comparative studies. IMPORTANCE Mycobacterium tuberculosis (Mtb), the pathogen responsible for tuberculosis, is often described as genetically stable. Our findings reveal an overlooked evolutionary adaptation mechanism: phase variation driven by tandem repeat copy number changes in gene promoters. Enabled by ultra-deep, long-read sequencing, we corrected errors in the Erdman reference genome and uncovered frequent, spontaneous expansions and contractions of promoter repeats upstream of genes linked to nitric oxide resistance, drug efflux, and biofilm formation. Through altering promoter strength, these dynamic promoter variants may generate phenotypic diversity within subpopulations and across diverse clinical lineages, suggesting a conserved evolutionary advantage for navigating host-imposed stress. This reframes Mtb’s evolutionary potential, highlighting how adaptive flexibility has been underestimated due to reliance on short-read sequencing and limited resolution of subpopulations at standard genomic depths. Our findings underscore the need to integrate structural variation-aware approaches into studies of Mtb pathogenesis, evolution, and drug response. Mycobacterium tuberculosis (Mtb), the pathogen responsible for tuberculosis, is often described as genetically stable. Our findings reveal an overlooked evolutionary adaptation mechanism: phase variation driven by tandem repeat copy number changes in gene promoters. Enabled by ultra-deep, long-read sequencing, we corrected errors in the Erdman reference genome and uncovered frequent, spontaneous expansions and contractions of promoter repeats upstream of genes linked to nitric oxide resistance, drug efflux, and biofilm formation. Through altering promoter strength, these dynamic promoter variants may generate phenotypic diversity within subpopulations and across diverse clinical lineages, suggesting a conserved evolutionary advantage for navigating host-imposed stress. This reframes Mtb’s evolutionary potential, highlighting how adaptive flexibility has been underestimated due to reliance on short-read sequencing and limited resolution of subpopulations at standard genomic depths. Our findings underscore the need to integrate structural variation-aware approaches into studies of Mtb pathogenesis, evolution, and drug response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b608d990d2aa84315a169548ac17ba8eb0b7b1e2" target='_blank'>
              Updated Erdman reveals tandem repeat copy number is phase-variable and impacts M. tuberculosis adaptation across evolutionary timescales
              </a>
            </td>
          <td>
            Samuel J. Modlin, Nachiket Thosar, Paulina M. Mejía-Ponce, Raegan L. Lunceford, Gaëlle Guiewi Makafe, Brian Weinrick, F. Valafar
          </td>
          <td>2025-12-17</td>
          <td>mSystems</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f26e5c36cf6fc942909dc97f82cb84d88175870d" target='_blank'>
              Transient centrosome loss in cultured prostate epithelial cells induces chromosomal instability to produce an oncogenic genotype that correlates with poor clinical outcomes.
              </a>
            </td>
          <td>
            Jiawen Yang, John M. Ryniawec, Diogo de Oliveira Pessoa, Matthew R Coope, Daniel W. Buster, Emily Loertscher, Mengdie Wang, Chen Chen, A. Cress, Megha Padi, Gregory C Rogers
          </td>
          <td>2026-01-07</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Transposable elements (TEs) are mobile DNA sequences capable of self-replication (especially retrotransposons) within the genome, which may lead to various forms of DNA damage. The introduction of this review encompasses the diverse classes and subclasses of TEs, particularly emphasizing the most active TEs present in the human genome. An analysis of the retrotransposition process of TEs is presented, illustrating how this mechanism can result in DNA damage and gene rearrangements. Furthermore, the review meticulously examines the implications of TE insertions on gene expression and genomic organization, which may contribute to the development of various diseases, including cancer. The relationship between TE activation and the aging process is also explored, with an emphasis on that epigenetic modifications associated with aging can lead to the derepression of TEs, thereby promoting genomic instability and inflammation. These factors may play a significant role in the pathogenesis of age-related diseases, such as cancer, cardiovascular disorders, and neurodegenerative conditions. Finally, the review considers potential therapeutic approaches aimed at targeting TE activity to alleviate the impacts of aging and associated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1552ca333c184d411cd1fdc06105d2ca4b0a5435" target='_blank'>
              The role of transposable elements activity in genomic instability and their relationship to aging process.
              </a>
            </td>
          <td>
            Jingran Hu, Tianhao Mao, Kainan Huang, Shangzhi Yang, Wenrui Yu, Jiacheng Huang, Shiqi Jin, Chuanyu Sun, Zeyidan Jiapaer, Xianli Wang
          </td>
          <td>2026-01-18</td>
          <td>Cell cycle</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddfc626bda7c6db9e371ab841d5c95cbd7dff747" target='_blank'>
              RNAseq analysis reveals the recurrent loss of heterozygosity in lung cancer and associated transcription patterns
              </a>
            </td>
          <td>
            Ruslan Gumerov, Wangzhen He, Phong Luong, Filippo Dall’Olio, Y. Vassetzky, Anna Schwager
          </td>
          <td>2026-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e04b464c0e1b167410299c1bc664ca1e5464bb8" target='_blank'>
              cellSTAAR: incorporating single-cell-sequencing-based functional data to boost power in rare variant association testing of noncoding regions.
              </a>
            </td>
          <td>
            Eric Van Buren, Yi Zhang, Xihao Li, M. Selvaraj, Zilin Li, Hufeng Zhou, Nicholette D. Palmer, Donna K. Arnett, J. Blangero, Eric Boerwinkle, B. Cade, Jenna C. Carlson, April P. Carson, Yii-DerIda Chen, J. Curran, R. Duggirala, Myriam Fornage, Nora Franceschini, M. Graff, C. Gu, Xiuqing Guo, Jiang He, Nancy Heard-Cosa, Lifang Hou, Yi-Jen Hung, Rita R. Kalyani, Sharon L. R. Kardia, Eimear E. Kenny, C. Kooperberg, B. Kral, Leslie Lange, Daniel Levy, Changwei Li, Simin Liu, D. Lloyd-Jones, R. Loos, A. Manichaikul, L. W. Martin, Rasika A Mathias, R. Minster, Braxton D. Mitchell, J. Mychaleckyj, T. Naseri, Kari E. North, Jeffry R. O'Connell, James A. Perry, P. Peyser, B. Psaty, L. Raffield, Ramachandran S. Vasan, S. Redline, Alexander P. Reiner, Stephen S. Rich, Jennifer A. Smith, Brian W. Spitzer, Hua Tang, Kent D. Taylor, Russell Tracy, S. Viali, Lisa R. Yanek, Wei Zhao, Jerome L. Rotter, G. Peloso, P. Natarajan, Xihong Lin
          </td>
          <td>2025-12-31</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Background Primary malignant melanoma in the cervix (PMMC) currently has no standardized therapy. Its pathogenesis remains unclear, and the prognosis is poor. Given the high mortality of PMMC, its causes and pathogenic mechanisms need to be unraveled, and novel biomarkers must be identified. Methods To further understand the genomic characteristics of PMMC, whole-exome sequencing was performed on the cancerous and adjacent tissues from three patients with PMMC, and PMMC-related tumor-susceptibility genes, mutation spectrum, mutation characteristics, driver genes, and high-frequency tumor mutations were analyzed. Concurrently, SNP and copy number variation (CNV) data from melanoma patients in the COSMIC and TCGA datasets were analyzed.Key protein interactions were further validated in vitro using co-immunoprecipitation assays. Results The results showed that C>T/G>A is predominant in PMMC, with mutation characteristics more closely resembling those of SBS5 and SBS40, though not completely matching those of melanoma. Comparison with the COSMIC and TCGA databases revealed overlapping genes between PMMC and melanoma, such as TP53, AHNAK2, and PMMC, as well as previously unreported mutated genes such as AKT3, SMYD4, ATAD3A, SIRPB1, XKR6, etc. In vitro experiments and co-immunoprecipitation validation revealed that P53 interacts with AKT3, SMYD4, and ATAD3A, forming a network potentially involved in the pathogenesis of PMMC. Finally, integration of significantly mutated genes and CNV analysis revealed that copy number gains may regulate the expression of SIRPB1, AHNAK2, and XKR6, thereby activating the nuclear factor kappa B (NFkB) and protein kinase (AKT) signaling pathways, which could further promote the occurrence and development of PMMC. Conclusion This study shows that while PMMC has similarities with other melanomas, it also harbors unique mutant genes. These findings provide a basis for exploring PMMC-related molecular markers and developing personalized diagnosis and treatment plans for patients with PMMC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8169bcb0a16009a6f92e0cc377919f341d410d5" target='_blank'>
              The genetic landscape of primary malignant melanoma of the cervix using integrated bioinformatics analysis and whole-exome sequencing
              </a>
            </td>
          <td>
            Jialiu Zhou, Tian Xu, Chujun Chen, Yi Ping
          </td>
          <td>2025-12-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76b560eb7504f0651c24b57c1e03e1ecc5ce721e" target='_blank'>
              DNAJB6 as an immuno-oncogenic hub in liver hepatocellular carcinoma: multi-omic profiling reveals prognostic significance and therapeutic vulnerability
              </a>
            </td>
          <td>
            Yiting Deng, Jin Li, Peng Wang, Xuelian Luo, Hongxia Xu, Yu Xiao, Zhenchuan Wang, Yifei Lu, Ying Chen, Feng Ge
          </td>
          <td>2026-01-21</td>
          <td>Journal of Molecular Histology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Transvection is a phenomenon where gene regulatory elements interact between different chromosomes, adding an additional layer of regulatory control beyond traditional cis-interactions. Although transvection effects have been characterized for many individual genes in Drosophila, it remains unclear how trans-interactions occur among multiple co-regulated genes where enhancers are shared. Here we demonstrate that transvection is supported at the engrailed-invected (en-inv) locus, where transcription of the two developmental genes is coordinated by common enhancers. Our data show that the presence of the en promoter in cis to the enhancers prevents trans-activation of inv, but removal of this promoter enables robust transvection, demonstrating competition between heterologous promoters in trans. We also find that local Polycomb Response Elements (PREs) enhance transvection reliability but are not strictly required for trans-activation. Furthermore, our analysis reveals that transvection at this locus is developmentally regulated, occurring efficiently in third instar larval tissues and late-stage embryos but not in early embryonic stages. Finally, we show that en-inv transvection can be reconstituted using transgenic constructs at an ectopic chromosomal location where it produces a striking reciprocal expression pattern between en in cis and inv in trans, suggesting that these enhancers can choose to activate one promoter or the other in a stochastic manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88e31a2cc622180074bc01c940ac5b5dbfb14d2a" target='_blank'>
              Promoter competition and Polycomb Response Elements govern transvection efficiency between co-regulated engrailed and invected genes in Drosophila.
              </a>
            </td>
          <td>
            Yuzhong Cheng, Adrienne T Perkins, Fountane Chan, Natalie D. Gehred, Jack R. Bateman, J. Kassis
          </td>
          <td>2025-12-30</td>
          <td>Genetics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Hereditary paragangliomas (PGLs) caused by germline SDHD pathogenic variants (PVs) exhibit a parent‐of‐origin effect, with tumors arising almost exclusively when the PV is inherited from the paternal allele. The Hensen model proposes that a cluster of maternally expressed tumor suppressor genes (TSGs) on chromosome 11p15.5 may play a crucial role in SDHD‐related PGL tumorigenesis, wherein somatic loss of maternal 11p and wild‐type SDHD allele, in conjunction with a paternally inherited SDHD PV, triggers tumor development. To systematically localize and identify the most crucial maternal‐expressed TSGs within 11p15.5, we developed a novel single nucleotide variant (SNV)‐oriented, capture‐based targeted enrichment approach followed by next‐generation sequencing (NGS) to enable high‐resolution loss‐of‐heterozygosity (LOH) analysis. Among 13 SDHD‐related PGLs and 23 non‐SDHD‐related PGLs, a somatic loss of 11p15.5‐15.4 was detected in 92% and 47%, respectively, a significant difference (p = 0.0035). Parental genotype analysis confirmed that the lost chromosome was of maternal origin. In our studies, 12/13 SDHD‐related tumors demonstrated complete loss of the maternal 11p15.5‐15.4 region, preventing localization of a specific driver TSG. Only one exceptional SDHD‐related tumor retained this region, warranting further investigation into the mechanism underlying parent‐of‐origin tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27b365a662db25feb9efe53f1684d83f7862183" target='_blank'>
              A Novel Targeted Sequence for Chromosome 11p15.5 Maternal Loss in SDHD‐Related Paragangliomas
              </a>
            </td>
          <td>
            Huei-Pin Lai, Chen-Hui Lee, Hsuan Hu, An-Ko Chung, Pei-Lung Chen, Wei-Shiung Yang, Wan-Chen Wu
          </td>
          <td>2026-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ffb82464a4840ef8616fc6853b84c7285b6e602" target='_blank'>
              Functional footprints of homologous recombination deficiency in prostate cancer revealed by ctDNA fragmentation and transcription factor accessibility.
              </a>
            </td>
          <td>
            Georgios Vlachos, T. Moser, Isaac Lazzeri, Matthias J. Moser, Lisa Glawitch, Emil Bauernhofer, Anna Eberhard, C. Beichler, Hanieh Sadeghi, J. Blatterer, Stefan Kühberger, N. Monsberger, A. Terbuch, Karl Kashofer, J. Geigl, T. Bauernhofer, Ellen Heitzer
          </td>
          <td>2026-01-09</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Human T-lymphotropic virus type 1 (HTLV-1), the first human retrovirus identified, is linked to adult T-cell leukemia and HTLV-1-associated myelopathy/tropical spastic paraparesis. However, its post-transcriptional regulation remains poorly understood. Here, we used Oxford Nanopore direct RNA sequencing to profile the HTLV-1 transcriptome and epitranscriptome in MT2 cells. We identified 23 transcript isoforms, encompassing canonical and novel splice variants. Polyadenylation analysis revealed a predominant poly(A) tail length of around 50–100 nucleotides with transcript-specific variations. Distinct RNA modifications, including pseudouridine, N6-methyladenosine, and 5-methylcytidine, were enriched near the 3′ end and varied among transcript classes, with generally lower modification ratios in viral transcripts. These findings provide a more comprehensive map of HTLV-1 RNA splicing, polyadenylation, and modifications in MT2 cells, offering new insights into viral gene regulation and pathogenic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a1e4aff36600c0fd3356ace5c0413dcc761d3f3" target='_blank'>
              Transcriptomic and Epitranscriptomic Landscape of Integrated HTLV-1 in MT2 Cells
              </a>
            </td>
          <td>
            Shuanglong Wei, Bohan Zhang, Jingwan Han, Hanping Li, Yongjian Liu, Lei Jia, Jingyun Li, Xiaotian Huang, Lin Li
          </td>
          <td>2025-12-30</td>
          <td>Viruses</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Micronuclei are formed during cell division when acentric fragments or lagging chromosomes cannot be incorporated into the primary nucleus. Macroautophagy/autophagy may reduce chromosomal instability (CIN) by clearing isolated, atypical micronuclei. Other studies implicate that the loss of autophagy disrupts DNA repair pathways. However, whether aberrant mitosis contributing to CIN occurs when autophagy is inhibited has yet to be evaluated. We found impaired autophagy initiation contributes to CIN and facilitates the formation of micronuclei and other abnormal nuclear phenotypes either by genetic or pharmacological manipulation in multiple cell lines. We also found that loss of the integral autophagy protein ATG9A resulted in various types of mitotic errors that can contribute to the formation of micronuclei. ATG9A also localizes to centrosomes and midbody during cell division. Autophagy inhibition causes the overactivation and mislocalization of TBK1 (TANK binding kinase 1) into cytoplasmic, punctate structures that colocalize with SQSTM1/p62. This overactivation interferes with its function in cell division as a mitotic kinase and its role at the centrosome. These results indicate that loss of autophagy contributes to genomic instability from multiple angles, one of which being aberrant cell division.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f7652d445dff3bc9d16db0af8750b841d562ae2" target='_blank'>
              Disrupted autophagy overactivates TBK1 and results in mitotic defects promoting chromosomal instability.
              </a>
            </td>
          <td>
            Swagatika Paul, Porter L Tomsick, Julia P. Milner, S. Biswas, Samantha Brindley, Nicole DeFoor, Leila Zavar, G. Wright, Yairis Soto, A. Pickrell
          </td>
          <td>2026-01-16</td>
          <td>Autophagy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b544cab2c7f5ac6e8301325b2bf082f8d3ca21" target='_blank'>
              TRMT112 drives a tumor growth and metastasis-promoting program in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Amr R. Elhamamsy, Brandon J. Metge, M. H. Elbahoty, Bhavya Papineni, Heba M. Alsheikh, Dongquan Chen, Rajeev S. Samant, L. Shevde
          </td>
          <td>2026-01-08</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Human papillomavirus (HPV) infection is a leading cause of cervical cancer and a significant contributor to anogenital and oropharyngeal malignancies worldwide. While the oncogenic functions of HPV oncoproteins E6 and E7 in disrupting nuclear tumor suppressor pathways are well established, their influence on mitochondrial biology has only recently emerged as a critical facet of HPV-driven carcinogenesis. This review synthesizes current evidence on the qualitative and quantitative alterations of mitochondrial DNA (mtDNA) and their functional consequences in HPV-associated cancers. We discuss how E6 and E7 modulate mitochondrial dynamics, bioenergetics, and redox balance, contributing to metabolic reprogramming, resistance to apoptosis, and adaptation to tumor microenvironmental stress. We also examine the clinical significance of mtDNA mutations, deletions, and copy number variations as potential biomarkers for diagnosis, prognosis, and therapy response. Advances in multi-omics approaches, high-throughput sequencing, and patient-derived organoid models have accelerated the exploration of mitochondria as therapeutic targets. Integrating mitochondrial profiling into HPV-related cancer research holds promise for identifying novel metabolic vulnerabilities and guiding the development of mitochondria-directed treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25dca4a25a42e375c179e79296851a7aeadff46b" target='_blank'>
              Mitochondrial DNA Alterations in HPV-Related Cancers: Emerging Insights and Future Directions
              </a>
            </td>
          <td>
            M. O. Cakir, Melis Selek, Guldide Kayhan, Betül Yılmaz, Mustafa Ozdogan, G. H. Ashrafi
          </td>
          <td>2026-02-02</td>
          <td>DNA</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Highlights What are the main findings? We established and characterized a unique pair of isogenic cervical cancer cell lines (AdMer35/AdMer43) from a patient with squamous cell carcinoma of the cervix before and after chemoradiotherapy, revealing that resistance is not caused by a single factor but by complex reorganization of cancer cells. Single-cell analysis identified key resistance hallmarks: transcriptional reprogramming towards embryonic stemness (HOX, POU5F1), a shift from fibrillar to non-fibrillar collagen expression, and activation of interferon/inflammatory pathways. What are the implications of the main findings? The study maps a remodeled tumor ecosystem with a metabolically reprogrammed, pro-tumorigenic senescent-like cell population and a progenitor-like cell reservoir, identifying them as critical therapeutic targets for eliminating resistant cell pools. The identified key pathways, Wnt, TGFβ, BMP, and the specific ligands EREG and SEMA3C, along with the metabolic vulnerabilities of the senescent compartment, provide a robust framework for future therapeutic exploration. Abstract Cervical cancer remains a significant global health burden, with chemoradioresistance representing a major obstacle to successful treatment. To elucidate the mechanisms underlying this resistance, we established a unique pair of isogenic primary cervical cancer cell lines, AdMer35 and AdMer43, obtained from a patient with squamous cell carcinoma of the cervix before and after radiation therapy. The aim of our study was to characterize the transcriptomic and cellular heterogeneity of these cells. We conducted an in-depth comparative analysis using single-cell RNA sequencing. Analysis of this paired, patient-derived isogenic model suggests that chemoradioresistance can arise through coordinated multilevel cellular adaptations. Resistant AdMer43 cells demonstrated transcriptional reprogramming, with the upregulation of embryonic stemness factors (HOX, POU5F1, SOX2), a shift in extracellular matrix from fibrillar to non-fibrillar collagens, and activation of inflammatory pathways. We identified and characterized critical cell-state dynamics: resistant cells exhibited a remodeled ecosystem with a metabolically reprogrammed senescent-like cell population showing an enhanced pro-tumorigenic communication via EREG, SEMA3C, BMP, and WNT pathways. Furthermore, we identified a progenitor-like cell population with a minimal CNV burden, potentially serving as a reservoir for tumor persistence. These findings offer novel insights for developing targeted strategies to eliminate resistant cell pools and improve cervical cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ccd2a5a0a730a290acb2a66420d2bdad799661d" target='_blank'>
              Single-Cell Transcriptomic Landscape of Cervical Cancer Cell Lines Before and After Chemoradiotherapy
              </a>
            </td>
          <td>
            Dmitriy V Semenov, Irina S Tatarnikova, A. S. Chesnokova, Vadim A. Talyshev, M. Zenkova, E. Logashenko
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Retrotransposon mobilization in germline cells enables the rewriting of genetic information to drive genome innovation, species evolution, and adaptation through the generation of de novo mutations. However, uncontrolled mobilization can cause DNA breaks and genome instability, often leading to sterility. How retrotransposon mobilization that can be retained for genome evolution persists despite negative outcomes of retrotransposon activity remains poorly understood. Here, we used Drosophila spermatogenesis as a model to investigate retrotransposon mobilization dynamics. Although many retrotransposon families are transcriptionally active, we found that the LTR retrotransposon nomad completes the full mobilization cascade (including mRNA export, protein translation, and reverse transcription) to produce double-stranded DNA (dsDNA) the most efficiently. Strikingly, despite successfully generating dsDNA, nomad rarely achieves genomic reintegration. Instead, its newly synthesized DNA predominantly forms extrachromosomal circular DNA (ecDNA). These findings show that retrotransposon-derived DNA largely remains as ecDNA. This could prevent widespread genomic integration during spermatogenesis, potentially preserving genome stability with the presence of limited retrotransposon activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9420ae83c1dc149dca66d3f18d2a4cfd87487588" target='_blank'>
              Retrotransposon activation during spermatogenesis achieves massive ecDNA biogenesis but rare integration.
              </a>
            </td>
          <td>
            Lauren Tracy, Yao Chen, Z. Zhang
          </td>
          <td>2026-01-22</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer progression involves dynamic crosstalk between tumor-intrinsic pathways and microenvironmental remodeling, and identifying pan-cancer biomarkers is critical for precision oncology. CDKN2AIPNL exhibits a paradoxical role in cancer, acting as a tumor suppressor in myeloid malignancies but promoting solid tumor progression, yet its systematic pan-cancer characteristics remain unelucidated. This study aimed to comprehensively analyze CDKN2AIPNL’s expression patterns, prognostic value, genetic alterations, and molecular mechanisms across multiple tumor types using public datasets including TCGA, GTEx, HPA, and tools such as GEPIA2, cBioPortal, TIMER2, STRING, and BioGRID. We performed expression difference analysis, survival analysis (overall survival, disease-free survival, progression-free survival), genetic alteration analysis, cancer-associated fibroblast (CAF) infiltration analysis, and gene/protein interaction enrichment analysis. Results showed that CDKN2AIPNL was significantly upregulated in multiple tumors (e.g., LIHC, UVM, BRCA, LUAD) and downregulated in others (e.g., KICH, KIRP, THCA), with high tumor specificity. Elevated CDKN2AIPNL expression correlated with poor overall survival in LIHC (HR = 1.7, p = 0.0026), UVM (HR = 26, p = 2.2e-6), BRCA (HR = 26, p = 2.2e-6), LUAD (HR = 1.36, p = 0.049), PCPG (HR = 1.71, p = 0.0012), and TGCT (HR = 0.37, p=0.023), and was associated with advanced tumor stages in metabolically active cancers. Genetic alterations (amplifications and mutations) were frequent in KIRC (>5%) and ACC (>4%), with all mutations localized to the XTBD region, and amplification predicted poor prognosis in PRAD (p = 0.008) while mutations conferred favorable outcomes in BLCA. CDKN2AIPNL expression positively correlated with CAF infiltration in ESCA, KICH, UVM, and other tumors, and interacted with MYC, XRN2, and CHAMP1 to regulate metabolic reprogramming, cell cycle, and immune suppression. Our findings systematically reveal CDKN2AIPNL’s dual role in tumorigenesis and validate it as a potential pan-cancer prognostic biomarker, providing novel insights for cancer diagnosis and targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1804517cd011102843d1d1b41c7599a28bf25c60" target='_blank'>
              CDKN2AIPNL: a potential pan-cancer biomarker
              </a>
            </td>
          <td>
            Yulin Yuan, Shengjie Ma, Heshi Liu, Yang Gong, Xuan Sun, Quan Wang, Weifu Zhang
          </td>
          <td>2026-01-21</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Transitions of cancer cells between distinct cell states, which are typically driven by transcription reprogramming, fuel tumor plasticity, metastasis, and therapeutic resistance. Whether the transitions between cell states can be therapeutically targeted remains unknown. Here, using the epithelial-to-mesenchymal transition (EMT) as a model, we show that the transcription reprogramming during a cell-state transition induces genomic instability through R-loops and transcription-replication conflicts, and the cell-state transition cannot occur without the ATR kinase, a key regulator of the replication stress response. ATR inhibition during EMT not only increases transcription- and replication-dependent genomic instability but also disrupts transcription reprogramming. Unexpectedly, ATR inhibition elevates R-loop-associated DNA damage at the SNAI1 gene, a key driver of the transcription reprogramming during EMT, triggering ATM- and Polycomb-mediated transcription repression of SNAI1. Beyond SNAI1, ATR also suppresses R-loops and antagonizes repressive chromatin at a subset of EMT genes. Importantly, inhibition of ATR in tumors undergoing EMT reduces tumor growth and metastasis, suggesting that ATR inhibition eliminates cancer cells in transition. Thus, during EMT, ATR not only protects genome integrity but also enables transcription reprogramming, revealing that ATR is a safeguard of cell-state transitions and a target to suppress tumor plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3e77506695b216bcfa4d8c23ede1c2f9a8846d6" target='_blank'>
              ATR Safeguards Epithelial-to-Mesenchymal Transition by Countering R-loops and Enabling Transcription Reprogramming.
              </a>
            </td>
          <td>
            Parasvi S Patel, Jacob P Matson, Xiaojuan Ran, Marcello Stanzione, Ajinkya S Kawale, Mingchao Wang, Sneha Saxena, Conrad Sander, Jacquelyn M Curtis, Jessica L Hopkins, Edmond Wong, R. Corcoran, Daniel A. Haber, Nicholas J. Dyson, S. Maheswaran, Lee Zou
          </td>
          <td>2026-01-22</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="GapR is an α-proteobacterial nucleoid-associated protein reported to either directly regulate transcription of AT-rich DNA or to regulate transcription indirectly through sensing DNA topology and modulating topoisomerase activity. We use single-DNA micromechanics, biolayer interferometry (BLI), and computational analysis to study GapR transcriptional regulation. Micromechanics experiments show that GapR overtwists DNA and shortens its contour length. GapR binding also substantially increases DNA bending persistence length and twist stiffness, while promoting duplex DNA melting at higher forces. Nonequilibrium binding experiments show GapR-DNA complexes to be extremely stable, with essentially no dissociation on hour-long time scales. Strikingly, our BLI experiments show that GapR has high affinity for AT-rich DNA while our micromechanics show that GapR binding is enhanced by pre-twisting of DNA, validating GapR affinity for both forms. By analyzing published GapR binding data, we differentiate between AT-rich and overtwisted DNA binding, revealing that overtwisted DNA primarily determines GapR localization. Lastly, we show that these two binding modes exert opposite effects on transcription. AT-rich binding stimulates initiation, likely by promoting RNA polymerase progression, while overtwisted DNA binding represses transcription, possibly by stabilizing local overtwist at promoters. We propose that GapR binding promotes a conformation of DNA ideal for efficient replication and transcription progression. 3 Key points GapR is a DNA structuring protein that binds AT-rich and overtwisted, positively supercoiled DNA GapR binding stiffens DNA, limits plectoneme formation, and melts undertwisted DNA at higher force GapR binding modes exert opposing effects on transcription initiation while promoting elongation">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19ce59d692ce3bfb13c956f1bf270e498226eb75" target='_blank'>
              GapR is a nucleoid-associated protein that stiffens and overtwists DNA to promote transcription and replication
              </a>
            </td>
          <td>
            Xinjue Wei, Lucy M. Kwiatkowski, Ryo Kawamura, John F. Marko, Monica S. Guo
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Immune aging is being increasingly recognized as a critical barrier to effective cancer immunotherapy, as the aged tumor microenvironment (TME) drives T cell dysfunction and impairs immune control of cancer1–4. However, the key molecular drivers of this process as well as potential targets to rescue T cell dysfunction in aged tumors remain incompletely understood. Therefore, we performed in vivo single-cell CRISPR screens in CD8⁺ T cells within aged tumors and tumor-draining lymph nodes (tdLNs). We identified Dusp5 and Zfp219 as key regulators of T cell persistence and effector differentiation in aged hosts. Loss of Dusp5, a negative regulator of ERK signaling, increased ERK1/2 phosphorylation and enhanced T cell proliferation in both young and aged tumors. In contrast, loss of Zfp219, a transcriptional repressor, induced epigenetic reprogramming of cytotoxic gene programs, thereby increasing granzyme secretion and enhancing antitumor immunity. Moreover, expression of the human ortholog gene ZNF219 is increased within intratumoral CD8⁺ T cells in older cancer patients. High ZNF219 expression correlates with poorer survival following immune checkpoint blockade (ICB) and reduces persistence of human intratumoral T cells. Notably, Zfp219 ablation synergized with anti-PD-1 blockade in mice to expand effector-like CD8⁺ T cells, leading to significantly enhanced anti-tumor immunity and tumor clearance in aged hosts. Together, these findings highlight Dusp5 and Zfp219 as critical drivers of age-related T cell dysfunction and as potential therapeutic targets to rejuvenate T cell antitumor immunity in older cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/017b5a7413706a4c2c7fbdc538a3a98539127c42" target='_blank'>
              CRISPR screens identify targets to rescue age-related T cell dysfunction in cancer
              </a>
            </td>
          <td>
            Alex C. Y. Chen, Keely Ji, Cansu Yerinde, Nelson H. Knudsen, Kevin Bi, Daniela Martinez, Thomas J Carmona-LaSalle, Kazuhiro Taguchi, Katherine H Xu, Elizabeth M. Seider, Marc A. Schwartz, Maria Zschummel, Linda T. Nieman, Kathleen B. Yates, T. R. Mempel, R. Manguso, N. Hacohen, Debattama R. Sen
          </td>
          <td>2026-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d73e2496f48eaf63e522263faed717c4d2cc83e" target='_blank'>
              Exploring the Role of APC in Modulating Chemotherapeutic Response in Triple-Negative Breast Cancer cells
              </a>
            </td>
          <td>
            T. Nair, M. K. Vanklompenberg, Jenifer R Prosperi
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15ea2c9c7605b557b8e4ec203f1280961cf9800c" target='_blank'>
              Exploring the oncogenic mechanism of plasmacytoma variant translocation 1 (PVT1) gene in solid cancers; emphasis on microRNA regulation pathways.
              </a>
            </td>
          <td>
            Mojtaba Zehtabi, Shirin Azizidoost, Abdolah Mousavi Salehi, Mahrokh Abouali Gale Dari, Mohammad Amin Zargar, Maryam Khombi Shooshtari, Krzysztof Data, Dominika Domagała, Julia Niebora, Małgorzata Józkowiak, P. Chmielewski, Aleksandra Partyńska, Aleksandra Górska, A. Bryja, M. Kulus, Adam Kamiński, Teresa Wysocka, Marek Spaczyński, Piotr Dzięgiel, P. Mozdziak, Maryam Farzaneh, B. Kempisty
          </td>
          <td>2026-01-29</td>
          <td>Cancer cell international</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f88af5bb8874c2112f831be5ec9601df23ed25e" target='_blank'>
              MER57E3 transposable elements regulate gene expression in a human cell model of neural development.
              </a>
            </td>
          <td>
            Michelle Almeida da Paz, Umut Yildiz, Minyoung Kim, Víctor Campos-Fornés, Marina Pinkasz, Thomas Dahlet, K. Noh, Leila Taher
          </td>
          <td>2026-01-31</td>
          <td>Genome biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Clinical genomic sequencing has revealed that prostate cancer (PCa) progression is associated with changes to chromosome number and structure, termed chromosomal instability (or CIN). While mutations in common driver genes are rare in primary PCa, CIN is detected in >90% of cases, suggesting that CIN may be a driver of the disease. However, the underlying mechanisms that cause CIN in PCa are poorly understood. Cells are susceptible to CIN during mitosis, when the duplicated genome must be equally segregated into two daughter cells. Normally, two centrosomes (cytoplasmic, non-membranous organelles) nucleate and organize microtubules to guide assembly of the mitotic spindle, thus ensuring the fidelity of cell division. However, we recently discovered that cells within primary PCa lack centrosomes, the first cancer type to show this phenomenon. Furthermore, we showed that centrosome elimination in non-tumorigenic cells generates CIN which can transform these cells, causing them to form xenograft tumors in mice. Therefore, we sought to understand the underlying mechanisms that drive the centrosome loss observed in PCa. The prostate is a naturally hypoxic organ, averaging around 4% oxygen tension in the healthy organ which declines as men age. Additionally, tumor hypoxia is strongly correlated with high CIN, particularly in prostate cancers. However, there are few mechanistic links between hypoxia and CIN. Therefore, we hypothesized that hypoxia could cause centrosome loss and, thus, predispose cells to CIN. Indeed, we found that immortalized prostate RWPE-1 cells progressively lost centrosomes when exposed to 1% O2 for as short as 6 hours in vitro. Using ultrastructure expansion microscopy (U-ExM), we find centrosomes are replaced by non-functional microtubule husks devoid of centrosome proteins which we call ‘remnants’. These phenomena also occur in PC-3 and LNCaP prostate cancer lines. Furthermore, we observe centrosome loss specifically in the hypoxic regions of xenograft tumors. Using a combination of RNAseq and chemical inhibitors, we determined that centrosome elimination sits at the crossroads of 3 transcriptional programs: (1) centrosome elimination occurs in basal cells and basal-like cancers that express p63; (2) cells must be confluent with active Hippo pathway; and (3) hypoxia induces a HIF-independent transcriptional response. When these conditions are met, cells transcriptionally downregulate 4 key centrosome proteins (Cep63, Cep152, Cep192, and Cep215), resulting in the depletion of a protective shell of pericentriolar material (PCM) and subsequent centrosome elimination. By overexpressing the mitotic kinase Polo-like kinase 1 (PLK1), we can strengthen the PCM and prevent elimination. Finally, we identify PCa cells containing remnants within prostatectomy tissue samples. This work is significant as it identifies the first examples and mechanisms of centrosome elimination in human somatic and cancer cells, provides a mechanistic link between hypoxia and CIN, and identifies remnants as a cellular indicator for CIN predisposition.



 John M. Ryniawec, Natalya K. Seppanen, Gregory C. Rogers, Anne E. Cress. Hypoxia-induced centrosome elimination as a driver of chromosomal instability in prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B065.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0fec5efccdcc580bc977b0a1bb739d683cd475" target='_blank'>
              Abstract B065: Hypoxia-induced centrosome elimination as a driver of chromosomal instability in prostate cancer
              </a>
            </td>
          <td>
            John M. Ryniawec, Natalya K. Seppanen, Gregory C Rogers, A. Cress
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af7630e9ff15d614dc5c29742755271640ed25f5" target='_blank'>
              KMT2A-Mediated transcriptional regulation in stemness and cancer: molecular mechanisms and therapeutic opportunities
              </a>
            </td>
          <td>
            Md Rashedunnabi Akanda, Md Shiblee, Sadik Sabuj, Tanvir Ahmed, Md Jamilur Rahman, S. M. A. Salam, Byung-Yong Park
          </td>
          <td>2025-12-24</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 ZFTA-RELA (ZR) is the most recurrent genetic alteration seen in pediatric supratentorial ependymoma (EPN) and is sufficient to initiate tumors in mice. Despite the oncogenic potential of ZR, it is observed nearly exclusively in childhood EPN, with tumors located distinctly in the supratentorial region of the central nervous system (CNS). We hypothesized that specific chromatin modules accessible during brain development would render distinct cell lineage programs at direct risk of transformation by ZR.



 To this end, we performed combined single nucleus ATAC and RNA (snMultiome) sequencing of the developing mouse forebrain, as compared to ZR-driven mouse and human EPN.



 We demonstrate that specific developmental lineage programs present in transient progenitor cells and regulated by PLAG/L family transcription factors (TF) are at risk of neoplastic transformation. Binding of this chromatin network by ZR or other PLAG/L family motif targeting fusion oncoproteins lead to persistent chromatin accessibility at oncogenic loci and oncogene expression. Cross-species analysis of mouse and human ZR EPN reveals significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling progenitor-like or radial glial-like cells that establish a putative tumor cell hierarchy. In vivo lineage tracing studies reveal neoplastic clones that aggressively dominate tumor growth and establish the entire EPN cellular hierarchy.



 These findings unravel developmental epigenomic states critical for fusion oncoprotein-driven transformation and elucidate how these states continue to shape tumor progression.



 Alisha Kardian, Hua Sun, Stephen Mack. Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr LT004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/885b82368395c60d63823ec56b4ae77477004d30" target='_blank'>
              Abstract LT004: Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development
              </a>
            </td>
          <td>
            A. Kardian, Hua Sun, S. Mack
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Somatic structural variations (SVs) are common in cancer. Although a small fraction of SVs in breast and ovarian cancers can be attributed to homologous recombination deficiency, the underlying molecular mechanisms for the vast majority of somatic SVs remain unclear. Here, we focus on the roles of transcription and DNA replication collisions in genomic instability in cancer. Such collisions are unavoidable in cells since both transcription and replication use the same DNA as template. We hypothesized that transcription replication collisions (TRCs), if not properly repaired, would lead to collapsed replication forks and result in SVs. To this end, we studied somatic SVs in 6193 high-coverage whole-genome sequenced primary and metastatic tumors from three independent pan-cancer cohorts. A total of 13 conserved SV signatures, representing independent molecular mechanisms, were deconvoluted from these cohorts using non-negative matrix factorization approach. We detected replicated-strand bias, the expected footprint of transcription-replication collision, in large tandem duplications (TDs) across multiple cohorts. This bias was only observed in expressed genes, consistent with TRCs depending on transcription activity. Large TDs were abundant in female-specific (breast, ovarian and uterus), upper gastric-intestinal tract, and prostate cancers. They were associated with CDK12 mutations and worse patient survival. CDK12 is a cyclin-dependent kinase, a key regulator of transcription elongation and termination. Deleting or suppressing CDK12 using CRISPR-Cas9 in prostate cell lines increased RNA:DNA hybrids (R-loops), promoted TRCs, and ultimately led to large TDs. Finally, we found that cells lacking CDK12 were sensitive to WEE1, CHK1 and ATR inhibitors. In summary, our data suggest that large TDs in cancer form due to impaired TRC repair and can be used as a biomarker for prognosis and treatment.



 Lixing Yang. A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A078.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a771b9c6597f1781d70e2b6d5df9ff54196c4d85" target='_blank'>
              Abstract A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer
              </a>
            </td>
          <td>
            Lixing Yang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Approximately 80% of Merkel cell carcinoma (MCC) cases are caused by Merkel cell polyomavirus (MCV), driven by its T antigen oncogene. Why MCV drives MCC, a skin cancer that displays the neuroendocrine Merkel cell phenotype, remains unclear. In this issue of the JCI, Miao et al. demonstrated that MCC tumor survival requires neuroendocrine-lineage transcription factors, which are recruited to superenhancers (SEs) with the viral small T antigen oncoprotein to promote the neuroendocrine Merkel cell lineage of the cancer. Surprisingly, SEs mapped near the MCV integration site in MCC, and two SE-associated neuroendocrine transcription factors drove viral T antigen gene expression. MCV oncogene and neuroendocrine transcriptional network interactions rendered this viral tumorigenesis dependent on the Merkel cell lineage. Together with reports from other groups, the findings explain why MCV-associated cancer is specifically linked to the Merkel cell phenotype and identify epigenetic strategies targeting of lineage-dependent oncogene circuitry to treat virus-positive MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972de34b3fee7d7c0f171474dfc31f7dfd102967" target='_blank'>
              Tumor virus–induced lineage survival circuit drives Merkel cell carcinogenesis
              </a>
            </td>
          <td>
            M. Shuda
          </td>
          <td>2025-12-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Papillary renal cell carcinoma (pRCC) is characterized by marked intratumoral heterogeneity, which contributes to therapeutic resistance and disease progression. In this study, we identify STK38 as a key regulator of tumor heterogeneity in pRCC, functioning through non-canonical activation of the Hedgehog (Hh) signaling pathway. STK38 interacts with both KIF7 and GSK3β to promote Hh signaling by facilitating KIF7 ciliary localization and reprogramming GSK3β substrate selectivity, leading to GLI1 stabilization and β-catenin suppression. Moreover, GLI1 directly enhances STK38 transcription, establishing a positive feedback loop that reinforces pathway activation. Notably, depletion of STK38 sensitizes tumor cells to a NETosis-like chromatin release process (tNET release), a form of stress-induced nuclear expulsion associated with immune evasion and metastatic potential. Given the potential pro-metastatic consequences of STK38 inhibition, we instead targeted its downstream effector GLI1 using Glabrescione B, which potently suppressed tumor growth and induced apoptosis in both xenograft and patient-derived organoid models, particularly in STK38-high tumors. These findings position STK38 as a critical modulator of pRCC heterogeneity and support GLI1 inhibition as a promising strategy to disrupt oncogenic signaling while minimizing adverse effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a249254f9dd46bc3c20e8d81598af9b9ed7c0662" target='_blank'>
              STK38-mediated feedback loop regulation of the hedgehog pathway governing tumor heterogeneity in renal papillary carcinoma
              </a>
            </td>
          <td>
            Yifan Du, Xiuyuan Sui, Zeyuan Zheng, Zhengying Zhang, Bin Liu, Yang Bai, Yue Zhao, Qingqing Wu, Haodong Wu, Min Zhong, Liyan Li, Huimin Sun, Chen Shao
          </td>
          <td>2026-01-15</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="A fundamental challenge in studying therapy resistance is understanding whether it results from pre-existing cellular states (“priming”) or drug-induced changes (“adaptation”). While lineage barcoding enables retrospective analysis of cells before and after treatment, current methods struggle to efficiently capture rare lineages in single-cell RNA sequencing (scRNA-seq) or isolate multiple specific lineages simultaneously for functional study. To overcome these limitations, we developed CloneSweeper, a multiplexed lineage tracking platform that pools enrichment libraries to isolate or enrich multiple rare lineages. CloneSweeper utilizes a dual-function barcode expressed as both a Cas9 gRNA for live-cell sorting and a 3’ UTR transcript for high-recovery detection in 10x Genomics scRNA-seq. We applied CloneSweeper to a model of BRAF V600E melanoma, where we identified that resistance to targeted therapy emerges from a polyclonal population of rare, pre-existing lineages. By simultaneously targeting and enriching 21 distinct rare lineages prior to treatment, we defined a heritable, primed state characterized by de-differentiation and elevated mesenchymal markers. We demonstrate that these primed cells are not quiescent but instead exhibit upregulated inflammatory and stress response signaling, specifically via the AP-1 and NF-κB1 pathways. CloneSweeper thus provides a robust framework for dissecting the molecular mechanisms of rare biological phenomena through simultaneous, multiplexed lineage isolation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cfe1b1a36a95588a97f5cc3024b429ba771c00" target='_blank'>
              Multiplexed enrichment and tracking of lineages with CloneSweeper
              </a>
            </td>
          <td>
            Robert J. Vander Velde, Raymond W. S. Ng, Christopher Coté, Sydney M. Shaffer
          </td>
          <td>2026-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The tumor suppressor RAD51D is essential for homologous recombination (HR). Pathogenic variants in RAD51D are associated with breast and ovarian cancers. However, most clinical missense variants are of unknown significance. We performed a multiplex assay of variant effect to test 6,888 RAD51D coding variants for loss-of-function. The resulting variant-to-function map perfectly separates known pathogenic and benign variants and is validated by orthogonal HR and biochemical assays across 70 clinical variants. Our screen shows that variants in the DNA-binding or ATPase core most severely compromise HR, and we identify the RAD51D-RAD51C interface within the BCDX2 complex as essential for regulating its ATPase activity. We hypothesize that, paradoxically, the primary function of RAD51D is to slow the ATPase activity of BCDX2, thereby allowing sufficient time and space for RAD51 filament assembly. Together, we identify hotspots of deleterious RAD51D variants and uncover the mechanisms by which variants compromise its biochemical functions. Highlights Used a multiplexed assay of functional effect (MAVE) to assess the functionality via olaparib sensitivity of 6,888 RAD51D coding variants, which can be used for variant classification Provided cellular functional analysis for 70 clinically-identified breast and ovarian cancer RAD51D variants Identified key regions and enzymatic activities of RAD51D critical for its function in the BCDX2 and the X3CDX2 complexes Determined mechanism of RAD51D-mediated regulation of BCDX2 ATPase activity">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b147f7b191823354d4eee4bfd6f57b8ab44617ab" target='_blank'>
              High-throughput mapping of 6,888 RAD51D variants identifies distinct biochemical functions needed for homologous recombination and olaparib response
              </a>
            </td>
          <td>
            Kristie E. Darrah, Shelby L. Hemker, Yashpal Rawal, Noah J. Goff, Phoebe S. Parker, Gayatri Ganesan, Caleb M. Stratton, Katherine Oppenheimer, Ella Roberts, Elena Glick, Nicole Banks, Arjun Kumar, Silvia Casadei, Matthew W Snyder, Katherine L Nathanson, S. Domchek, Lea M. Starita, Shaun K. Olsen, Patrick Sung, J. Kitzman, Kara A. Bernstein
          </td>
          <td>2026-01-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2770d616654ff2fea428335622c1756448d7d587" target='_blank'>
              APE1 Coordinates Its Disordered Region and Metal Cofactors to Drive Genome Surveillance
              </a>
            </td>
          <td>
            Donghun Lee, Subin Kim, Gyeongpil Jo, Juwon Kim, Jungmin Yoo, Jejoong Yoo, Ja Yil Lee, Gwangrog Lee
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Chromosomal aberrations are frequent in cancer and play a pivotal role in disease initiation and progression. In prostate cancer (PCa), deletions at chromosome 13q14 are common and enriched in advanced disease, suggesting selective advantage in tumor progression. Beyond BRCA2, this region harbors other tumor suppressor genes, including RB1 and RNASEH2B, which impact sensitivity to androgen receptor inhibitors (ARi) and PARP inhibitors (PARPi). This study aims to elucidate the effect of individual vs concomitant loss of RB1 and RNASEH2B in response to dual AR and PARP inhibition.



 To explore the prevalence of deletions in 13q we interrogated copy number alterations in the Hartwig Medical Foundation (HMF) PCa WGS dataset (n=486). In vitro, we generated CRISPR-Cas9-mediated knock-outs (KO) of RB1, RNASEH2B, or both genes in PCa cell lines (LNCaP, C4-2, 22Rv1, DU145). Gene editing was confirmed by Sanger, and western blot analysis validated protein loss. Antitumor activity of enzalutamide (ARi), and olaparib or saruparib (PARPi) was determined using CCK8 kit and colony formation assays.



 RB1 deletions were present in 42% of HMF PCa samples (205/486), being mostly heterozygous deletions (200/205). RB1 deletions co-occurred with RNASEH2B loss in 94% of cases (192/205) and with BRCA2 deletions in 42% of cases (87/205). In vitro, RB1-KO increased resistance to ARi and PARPi in hormone-sensitive lines LNCaP and C4-2, while no change in sensitivity was detected in AR-independent models 22Rv1 and DU145. Loss of RNASEH2B increased sensitivity to PARPi in all models. Next, we generated RB1 and RNASEH2B double KO models in LNCaP and C4-2 by sequentially editing single KO populations. After single-cell seeding, we obtained: (1) RB1-KO (homozygous) clones with complete protein loss; (2) RNASEH2B-heterozygous (RNASEH2B-Het) models with partial protein expression; and (3) RB1-KO/RNASEH2B-Het models. No clones with complete RNASEH2B loss were obtained, suggesting that RNASEH2B-loss population is transient and that clones with complete loss do not proliferate. RB1-KO models maintained resistance to both treatments. RNASEH2B-Het cells showed increased sensitivity to PARPi, although to lower extent than the population with complete RNASEH2B loss. Importantly, introducing RB1-loss in RNASEH2B-Het cells restored PARPi resistance, and combined AR/PARP inhibition did not overcome resistance in RB1-KO or RB1-KO/RNASEH2B-Het cells.



 Our findings indicate that 13q14 deletions in PCa predominantly occur as heterozygous events encompassing key genes influencing ARi and PARPi responses. Heterozygous loss of RNASEH2B confers sensitivity to PARPi, although to a lesser extent than in models with complete RNASEH2B loss. RB1-loss confers higher resistance to ARi and PARPi, restoring resistance to PARPi in RNASEH2B heterozygous models. These results highlight the need for models that better mirror the heterozygous nature of 13q14 deletions to clarify their biological effects and refine therapeutic strategies for advanced PCa.



 Victor Esquefa, Pablo Cresta Morgado, Richard Norris, Lara de Llobet, Julian Brandariz, Sara Arce-Gallego, Teresa Casals, Manuel Ramos, Daniel Aguilar, Gisela Mir, Irene Casanova-Salas, Francisco M. Barriga, Joaquin Mateo. Impact of 13q.14 loss in prostate cancer progression and sensitivity to targeted agents [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B018.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15430fc4c6c79184273233030845f22a6f4ecd5a" target='_blank'>
              Abstract B018: Impact of 13q.14 loss in prostate cancer progression and sensitivity to targeted agents
              </a>
            </td>
          <td>
            Victor Esquefa, P. C. Morgado, Richard Norris, L. D. De Llobet, Julian Brandariz, Sara Arce-Gallego, Teresa Casals, Manuel J. Ramos, Daniel Aguilar, Gisela Mir, I. Casanova-Salas, Francisco M. Barriga, Joaquin Mateo
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="How endogenous retroviral elements (ERVs), a family of transposable elements, may promote tumor progression is not well understood. Tripartite motif-containing 28 (TRIM28/TIF1b/KAP1) is a key transcriptional co-repressor protein that represses ERV expression in many cell types including embryonic stem cells, neural progenitor cells, differentiated adult cells, and cancer cells. In this study, we investigated the effect of Trim28 deletion on the expression of ERVs using an immune competent genetically engineered mouse model for prostate cancer. We found Trim28 deletion in prostate tumors led to the expression of ERVs in prostates from both hormonally intact and castrated mice. ERVs can regulate the expression of neighboring genes, and we detected increased expression of several protein-coding genes near overexpressed ERVs. Our data suggest that Trim28 deletion in prostate tumor epithelial cells may promote an innate immune response. However, Trim28 deletion also led to excessive deposition of tumor extracellular matrix (ECM). Our findings suggest that ECM alterations downstream of ERV derepression could affect immune cells in the tumor microenvironment and may promote tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0f3ac90397a0b7a69b88c44582035578d15374" target='_blank'>
              Expression of endogenous retroviral elements is associated with extracellular matrix remodeling in prostate cancer
              </a>
            </td>
          <td>
            Emily C. Williams, Dewanga R. Mayarata, Anelia Horvath, Katherine B. Chiappinelli, Maho Shibata
          </td>
          <td>2026-01-08</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89c6e2a39a6240d55a8cd9334672f5c4459bbc0c" target='_blank'>
              A compendium of chromatin interaction maps in the Giant Panda genome.
              </a>
            </td>
          <td>
            Pengliang Liu, Jiaman Zhang, Kailai Cai, Juan Wang, Hong Liu, Liang Zhang, Wei Xu, Yuliang Liu, Fujun Shen, Rong Hou, Yan Li
          </td>
          <td>2026-01-13</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 In Non-Small Cell Lung Cancer (NSCLC), low-fidelity transcription mechanisms such as ecDNA (extra-chromosomal DNA) are an increasingly relevant aspect of progression and prognosis, warranting comprehensive investigation into stability and prevalence of chimeric RNA transcripts. Here, we describe global distribution of fusion transcripts in encounter-matched primary tumor and lymph node metastases and their prognostic implications.



 RNAseq was performed on resected FFPE (encounter-matched primary and lymph nodes) from 121 stage IIB-IIIA (T1-4N1-2M0) NSCLC patients, in addition to 693 Stage IA1-IB cases (T1a-2aN0M0). STAR and STAR-Fusion were used to align data to the GRCh37 genome. Differential gene expression was based on specimen source (primary or lymph node). Aggregate analyses were also performed, including determination of a fusion burden metric, assessing the relationship between primary tumors, lymph node metastases, and recurrence. Deconvolution of Tumor Microenvironment was performed via Quantiseq.



 Comparing primary tumors to paired lymph node metastases, 7,930 ensembl IDs were differentially-expressed at FDR < 0.001 (n=101 pairs). A total of 613 fusions (685 unique transcripts) were detected from paired primaries and lymph node metastases (n = 242). Frequency of specific gene fusions in primaries ranged from 0-0.963 (median 0.019), compared to 0-0.870 (median 0.0093) in LN metastases (p=1.87×10-4). We observed fusion of TULP4-RP11-732M18.3 as pervasive in NSCLC, occurring in 84-87% of primary tumors and lymph nodes, with a 91% retention rate. Conversely, fusions involving CCDC7 or SEPT14 were preferentially observed in metastases, where 82% of CCDC7 fusions and 75% of SEPT14 fusions were not observed in the primary tumor despite occurring at high frequency (66% and 18.5%, respectively) among LN metastases. Interestingly, median fusion burden (unique gene-gene fusions) for primaries was 14 compared to 11 (p=2.06×10-7) in LN metastases (median change -4). Among N0 cases that did not receive adjuvant therapy, median fusion burden in non-recurrent tumors was 15 (n=570), compared to 13 in those that recurred over 3-60 months (n=123; p = 0.04). Interestingly, there was no significant difference in fusion burden between recurrent cases and primary tumors with LN metastases (n = 108). For both recurrent cases and LN-paired primaries, the difference in fusion burden compared to LN metastases (n = 110) was significant (p=2.10×10-4 and 2.06×10-7). A subset of LN metastases (n=27) increased in fusion burden relative to the primary tumor; these primaries uniquely exhibited 33% decrease in regulatory T cell content (p=0.008), 20% decrease in M2 macrophage content (p = 0.01 with M1 macrophages static), and 40% decrease in B-cell content (p=0.05).



 Fusion transcripts are pervasive in RNAseq data, suggesting a broader and more variable landscape than accounted for by primary tumors alone. Concurrent ongoing studies explore the role of ecDNA in fusion burden changes related to NSCLC progression and metastasis.



 Kelly M. Cagin, Keri L. Denson, Sierra R. Broad, Wara M. Naeem, Arsalan M. Khan, Michael M. Liptay, Christopher W. Seder, Jeffrey A. Borgia. A global analysis of fusion transcripts and recurrence in NSCLC primary-lymph node pairs [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B027.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83abeb8dbe1ed35e15732a9cc5bda99a8030ed67" target='_blank'>
              Abstract B027: A global analysis of fusion transcripts and recurrence in NSCLC primary-lymph node pairs
              </a>
            </td>
          <td>
            K. Cagin, Keri L. Denson, Sierra R. Broad, Wara Naeem, Arsalan M. Khan, Michael M. Liptay, C.W. Seder, Jeffrey Borgia
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colon cancer (CC) is a malignancy with high global incidence and mortality, and elucidating its underlying molecular mechanisms is critical for improving prognostic assessment and therapeutic strategies. In this study, transcriptomic data from a large cohort of CC samples and a limited number of normal controls from the TCGA database were used to construct a multigene prognostic risk model using univariate Cox and least absolute shrinkage and selection operator (LASSO) regression analyses. The expression of key prognostic genes, including immunoglobulin superfamily member 9 (IGSF9), was further validated in CC tissues by PCR and Western blotting. Functional assays were performed in HCT116 cells to investigate the biological effects of IGSF9 overexpression and its regulatory relationship with p53. The prognostic model identified IGSF9 as a gene significantly associated with patient survival. Although IGSF9 expression was reduced at both the mRNA and protein levels in CC tissues, its overexpression in vitro markedly promoted apoptosis, alleviated DNA damage, and suppressed cell migration and invasion. Notably, silencing of p53 partially reversed the tumor‐suppressive effects induced by IGSF9 overexpression, indicating that IGSF9 exerts its biological functions in a p53‐dependent manner. Collectively, these findings demonstrate that IGSF9 acts as a tumor suppressor in colorectal cancer and regulates DNA damage responses and apoptosis through a mechanism partially dependent on p53, highlighting its potential value as a prognostic biomarker and therapeutic target in CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ff548ce8387a244b0397236c92065d62ddf1f1" target='_blank'>
              P53 Inhibition Diminishes IGSF9 Gene Activity to Promote DNA Repair and Exacerbate the Progression of Colon Cancer
              </a>
            </td>
          <td>
            Huan-Yu Zhang, Dan Tian, Wan-Fu Zhang, Ying-Hui Zhang, Jia-Li Feng, Juan Sheng, Xue-Qin Shang
          </td>
          <td>2026-01-01</td>
          <td>Journal of Biochemical and Molecular Toxicology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The Polycomb Repressive Complex 2 (PRC2) is an ancient, conserved chromatin-interacting complex that controls gene expression, facilitating differentiation and cellular identity during development. Its regulation is critical in most eukaryotes. EZHIP was recently characterized as a PRC2 inhibitor and ‘oncohistone mimic’ in mammals. Although Ezhip expression is typically restricted to the germline, its aberrant expression in pediatric brain tumors inhibits PRC2-mediated H3K27 methylation and drives disease progression. To gain a deeper understanding of its normal functions, we systematically examined Ezhip evolution across 70 mammals using comparative genomics, synteny analysis, and motif discovery. Bolstering previous work, we find that Ezhip originated and has been strictly retained on the X chromosome in placental mammals. In addition to the highly conserved H3K27M-like histone mimic motif, our motif analysis reveals seven previously unidentified EZHIP motifs, including a putative nuclear localization signal, and tandem repeats that are largely well-conserved in placental mammals, except in some lineages. We hypothesize that these motifs are also critical to EZHIP’s functions, including in PRC2 interaction and inhibition. We show that Ezhip has evolved under strong diversifying selection in primates and underwent dynamic expansions and losses across species. Some paralogs, such as Ezhip2 in primates, also evolved under positive selection. Based on its evolutionary attributes and germ-cell expression, we propose that Ezhip arose and evolved rapidly due to inter-parental conflict over fetal development in utero in placental mammals. Our work provides a foundation for further investigations into EZHIP’s essential, potentially multifaceted roles in mammalian reproduction and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a5aaeda0a67409aa497058125c0f7abe2b1f4a7" target='_blank'>
              Dynamic evolution of EZHIP, an inhibitor of the Polycomb Repressive Complex 2 in mammals
              </a>
            </td>
          <td>
            P. Raman, Hana Khan, Janet M. Young, T. Tsukiyama, Harmit S. Malik
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="


 Fusion oncogenes drive malignant transformation by altering transcriptional programs, disrupting cellular homeostasis, and frequently rewiring protein condensate networks. Emerging evidence suggests that aberrant condensate formation is a key mechanism through which these fusion proteins exert oncogenic control. Specifically, by amplifying transcriptional output, sequestering cofactors, and enforcing pathological gene expression states. Despite their genetic validation as oncogenic drivers, many fusion proteins remain pharmacologically intractable due to the absence of catalytic domains or well-defined binding pockets. To overcome this, we developed an integrated discovery platform that directly assesses the biophysical and functional properties of fusion protein-driven condensates and enables identification of small molecules that disrupt these oncogenic assemblies. Using protein language models and in silico predictors of phase separation, we systematically mapped condensate-forming potential across recurrent oncogenic fusions catalogued in TCGA. This analysis revealed that most fusion proteins (including EML4-ALK and EWS-FLI1) harbor extensive intrinsically disordered regions (IDRs) that promote abnormal condensation and transcriptional dysregulation. We further identified disease-specific variants that alter condensate formation relative to wild-type proteins, providing mechanistic insight into oncogenic gain- and loss-of-function events. To experimentally validate and therapeutically exploit these findings, we established PhaseScan™, a high-throughput droplet microfluidics screening platform capable of quantitatively profiling condensate dynamics and screening thousands of compounds against reconstituted fusion condensates to identify therapeutically relevant small molecule inhibitors. Across multiple fusion proteins, PhaseScan™ identified small molecules that selectively disrupt aberrant condensates while sparing physiological assemblies such as nuclear speckles and stress granules. Using EML4-ALK as a representative model, we demonstrated that compound-mediated condensate dissolution disrupts oncogenic ALK signaling and reverses oncogenic phenotypes in cell-based systems. Our results highlight condensate biology as a generalizable vulnerability across fusion-driven cancers and demonstrate the feasibility of discovering small molecules against fusion proteins traditionally considered undruggable. By integrating AI-driven predictive modeling, and on-target molecular high-throughput screening, our platform provides a roadmap for rationally identifying and drugging fusion protein condensates. This approach establishes condensate-targeted therapeutics as a novel class of precision medicines for patients with fusion-positive malignancies.



 Andrew Seeber, William E. Arter, Seema Qamar, Cinzia Sgambato, Amal Alex, Kadi Saar, Julia Doh, Martin Kulander, Tuomas Knowles, Shilpi Arora. Targeting aberrant condensate formation in fusion-positive cancers with an integrated discovery platform [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr PR004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee0c93f691d35862b44375fdee303e25769c39e0" target='_blank'>
              Abstract PR004: Targeting aberrant condensate formation in fusion-positive cancers with an integrated discovery platform
              </a>
            </td>
          <td>
            A. Seeber, W. Arter, S. Qamar, Cinzia Sgambato, Amal Alex, K. Saar, Julia Doh, M. Kulander, Tuomas Knowles, Shilpi Arora
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) in plasma consists of short DNA fragments resulting from a non-random fragmentation process, with distinct fragmentomic characteristics that are related with their cellular origins. Here, we report that somatic variant signatures in cfDNA markedly differ between non-cancerous controls and cancer patients, indicating that tumor-associated signals are retained in these variants. Surprisingly, even in controls, cfDNA molecules harboring somatic variants exhibit cancer-like fragmentomic characteristics, such as reduced size, decreased DNA methylation, and altered end motif usages and distributions in the nucleosome structure. Further investigations suggest that such cancer-like traits are associated with somatic variants derived from clonal hematopoiesis. Importantly, these somatic variants-associated fragmentomic aberrations are more pronounced in cancer patients, enabling cancer diagnosis. In a large pan-cancer cohort, we utilize AI to integrate genomic, fragmentomic, and epigenomic features to develop diagnostic models named FreeSV and FreeSV+. Leveraging somatic variant-associated features alone, the FreeSV model achieved area under the ROC curves (AUCs) between 0.81-0.92 across cancer types; however, when genomewide features are also included, the AUCs of FreeSV+ model substantially increased to 0.93-0.99 across cancer types, highlighting the significance of integrative genomic and fragmentomic analyses in cfDNA for cancer liquid biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07d931b70db2d8495e7d9b12e6b1f59f47b890a1" target='_blank'>
              Cancer-Like Fragmentomic Characteristics of Somatic Variants in Cell-Free DNA.
              </a>
            </td>
          <td>
            Zhenyu Zhang, Yunyun An, Mengqi Yang, Yuqi Pan, Xiaoyi Liu, Fanglei Gong, Huizhen Lin, Bianbian Tang, Yunxia Bai, Xin Zhao, Yu Zhao, Changzheng Du, Kun Sun
          </td>
          <td>2026-01-22</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Rho GTPases are Ras-like proteins that contribute to many aspects of human cancer. However, unlike Ras, Rho proteins are not often mutagenically activated in cancers, but require activation by upstream proteins known as Rho GDP exchange factors (RhoGEFs). The neuroepithelial transforming gene 1 (Net1) is a RhoA/RhoB-specific GEF that is overexpressed in many cancers, and high levels of Net1 protein expression often correlate with reduced disease-free and overall survival. Net1 promotes multiple phenotypes in cancer cells, including cell motility, invasion, and proliferation. It does so by regulating diverse signaling pathways that ultimately control actin cytoskeletal organization, transcription, DNA damage signaling, and mitosis. In tumors, Net1 is required for tumor cell proliferation, tumor angiogenesis, and distant metastasis. In this review we will delineate the evidence supporting a role for Net1 in tumor progression, and describe mechanisms that regulate Net1 expression and cellular activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c51a562624e69f2f4d9024970a172be9b919059" target='_blank'>
              The Role of the RhoA Activating Protein Net1 in Cancer Initiation and Progression.
              </a>
            </td>
          <td>
            Yan Zuo, Stephanie A Planque, J. Frost
          </td>
          <td>2025-12-25</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Breast cancer is one of the most severe global health challenges, with its incidence continuing to rise. The development of this malignancy is associated with numerous risk factors; however, its primary etiology often remains unclear. Research on the potential association between breast cancer and carcinogenic human papillomavirus (HPV) has been conducted for three decades, yet a definitive consensus has not been reached. The development of prophylactic cervical cancer vaccines has greatly heightened the urgency of this issue: if a causal relationship can be established, it would open a feasible avenue for preventing this common cancer. This review aims to systematically integrate relevant evidence and provide a critical commentary on the association between HPV and breast cancer. We analyzed evidence from 34 studies on HPV DNA detection in breast cancer cells. We discuss the prevalence and genotypes of HPV detected in breast cancer across different geographical regions, the state of the viral genome (integrated vs. episomal), viral load, and potential mechanisms for HPV-associated breast carcinogenesis—particularly its role in inducing genomic instability via activation of APOBEC3B. Additionally, we explore the histopathological and clinical characteristics of HPV-positive breast cancer. The synthesized evidence suggests that high-risk HPV types are present in a subset of breast cancers worldwide, but with lower viral loads compared to cervical cancer, indicating a potentially different mode of action. The association between HPV and breast cancer warrants further rigorous investigation to clarify its clinical and preventive implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91c5a2843a4cdacf97e3a7acb855e5ff458bbd2b" target='_blank'>
              HPV, APOBEC3B, and the origins of breast cancer: a narrative review and perspectives on novel mechanisms
              </a>
            </td>
          <td>
            Zhiyong Liu, Ran Chen
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Most intracellular proteins undergo degradation by proteasomes, which exist in constitutive, immune, and intermediate forms. The diversity arises from the presence of different catalytic β subunits incorporated into the 20S core particle. Constitutive proteasomes contain β1, β2 and β5 subunits, whereas immunoproteasomes integrate β1i, β2i and β5i proteins. Intermediate proteasomes comprise combinations of constitutive and immune subunits, but typically lack the β2i subunit. Distinct proteasome configurations have widespread impact on cellular metabolism, gene expression, immune signaling, stress adaptation and tumorigenesis. However, the biological functions of proteasome subtypes remain incompletely defined, underscoring the need for refined experimental systems to uncover their specific activities. Methods Previously, we generated cancer cell lines expressing mCherry-tagged β5i subunit under the control of endogenous regulatory elements. Using CRISPR/Cas9 nickase technology, we further modified the genomes of the colon adenocarcinoma SW620B8-mCherry and cervical adenocarcinoma TZM-blB8-mCherry cells. In SW620B8-mCherryB5-GFP cells, the eGFP gene was inserted at the 3’ end of the PSMB5 gene, which encodes the β5 subunit. Similarly, the β2i-encoding gene (PSMB10) was fused with the photo-switchable cyan fluorescent protein gene (PS-CFP2) in TZM-blB8-mCherryB10-CFP cells. Results Efficient integration of fluorescent protein-encoding sequences into target genomes provided robust expression and integration of chimeric subunits into proteasomes. In TZM-blB8-mCherryB10-CFP cells, photo-conversion of PS-CFP2 allowed visualization of both intermediate and immunoproteasomes within the cellular nuclei following treatment with IFN-γ/TNF. In SW620B8-mCherryB5-GFP cells, analysis of intracellular proteasome localization revealed discrete regions enriched in β5i-containing proteasomes. Moreover, IFN-γ/TNF exposure induced a fluorescence shift from green to red, accompanied by redistribution of intracellular proteasomes in SW620B8-mCherryB5-GFP cells. Multiphoton microscopy of tumors grafted into immunocompromised mice showed the formation of β5i-containing, proteasome-enriched inclusions and suggested their potential release from cancer cells. Collectively, the TZM-blB8-mCherryB10-CFP cell line provides a tool to interrogate immunoproteasome functions independently of intermediate proteasomes, whereas SW620B8-mCherryB5-GFP cells enable visual discrimination between constitutive and β5i-containing proteasomes both in vitro and in vivo. Conclusions Generated cell lines provide a novel platform for dissecting proteasome functions, addressing the distinct properties of individual proteasome forms, and assessing the impact of diverse compounds on the proteasome repertoire in cultured cells and in tumor xenografts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8f185dc9839ab4abec820b173f8f7dd6f377d1a" target='_blank'>
              Next-generation cell lines for profiling different proteasome forms and their implications in cancer
              </a>
            </td>
          <td>
            A. Burov, Anastasia Yakovleva, Georgy Linovskiy, Andrei Fedotov, V. Popenko, Olga Leonova, Aleksandr Lanin, Marina Drutskaya, Vadim Karpov, Alexey Morozov
          </td>
          <td>2025-12-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b856e4af500cf320f5945f41664e5ac6ae569d14" target='_blank'>
              Targeting thymine DNA glycosylase induces synthetic lethality in p53-deficient cancers.
              </a>
            </td>
          <td>
            Jiaxin Zhou, Zhen-Yu Shao, Lin Zhang, Jiannan Guo, Meng Wang, Qing Xu, Yiqin Wang, Qing Xu, Dan Zhou, Sheng-Xiang Ren, Yanhao Yu, Zhi-Hao Lu, Guo-Zheng Pang, Yao Cao, Yilu Liu, Bin Zhou, Hong-Bin Ji, Yi-Han Chen, Hai-Ping Wu, Guo-Liang Xu, Liang Zhang, Ya-Rui Du
          </td>
          <td>2026-01-22</td>
          <td>Nature chemical biology</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="


 The CBFA2T3–GLIS2 (C/G) fusion defines an exceptionally aggressive, epigenetically driven subtype of pediatric AML (pAML) confined to infants (<3 years). Arising from a pericentric inversion of chromosome 16, it is the most frequent lesion in acute megakaryoblastic leukemia (AMKL; ∼30%) and also occurs in non-AMKL subtypes, indicating lineage-independent pathogenicity. Clinically, C/G+ leukemia shows relapse rates > 90% and < 15% five-year survival, making it among the most treatment-refractory pediatric cancers. Lacking cooperating mutations, C/G acts as a singular oncogenic driver that reprograms the enhancer landscape, yet how it sustains epigenetic programs enforcing leukemic identity and chemoresistance remains unknown. We integrated multi-omic and functional epigenetic profiling across patient samples, C/G+ AML lines, and a developmentally faithful cord-blood CD34+ HSPC co-culture model. DNA methylation (DNAm) was mapped by meEM-seq, chromatin states by CUT&RUN and ATAC-seq, and transcription by RNA-seq. Functional studies included DNMT3B knockout, gRNA-resistant rescue, and inducible dCas9–TET1 demethylation, complemented by promoter-capture Hi-C and C/G+ xenograft validation. C/G+ pAML exhibited a promoter-biased hypermethylation signature (Δβ ≥ 0.2 vs NBM), with ∼50% of altered CpGs at promoters. The fusion bound the DNMT3B promoter, inducing its transcription without global 5mC gain but driving focal hypermethylation at active promoters linked via long-range enhancer loops. These hypermethylated promoters paradoxically stabilized expression of CRE-residing genes (BMP2, MED12, ITGB2) enriched for EMT, KRAS, heme-metabolism, and apoptotic pathways. In the fetal C/G–UCB-HSPC model, these signatures were faithfully recapitulated, confirming that the fusion alone reprograms the fetal hematopoietic epigenome. Motif analysis identified E2F2 and E2F4 as transcription factors stabilizing methylated yet transcriptionally active CREs. DNMT3B loss reduced locus-specific 5mC and target-gene expression but upregulated DNMT1 and UHRF1, suggesting a compensatory maintenance shunt that preserves DNAm at apoptotic loci and raises the apoptotic threshold. Targeted demethylation of PMAIP1 (NOXA) restored its transcription and apoptosis, confirming that DNAm stabilizes but constrains inducibility of pro-apoptotic genes. In vivo, DNMT3B-deficient xenografts regained Venetoclax sensitivity and extended survival, phenocopying DNMT inhibition (DNMTi). This study identifies C/G as an epigenetic architect converting DNAm from a repressive to a stabilizing signal, locking leukemic enhancers into an active yet apoptosis-resistant state. By defining the C/G–DNMT3B–DNMT1/UHRF1 axis and transcription-factor-mediated maintenance of methylated CREs, we reveal a unified mechanism linking developmental origin, transcriptional persistence, and therapy resistance in pAML. These findings position aberrant DNAm as a druggable epigenetic vulnerability and provide the mechanistic rationale for DNMTi + BCL-2 blockade in high-risk, treatment-refractory pAML.



 Samrat Roy Choudhury, Arundhati Chavan, Rhonda E. Ries, Giselle Almeida Gonzalez, Soheil Meshinchi, Jason Farrar. CBFA2T3–GLIS2 fusion reprograms enhancer-linked DNA methylation to enforce apoptotic resistance and defines an epigenetic vulnerability in pediatric AML [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr PR006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e01e458a3959338b14b992ffa07b3d02aa22cbfe" target='_blank'>
              Abstract PR006: CBFA2T3–GLIS2 fusion reprograms enhancer-linked DNA methylation to enforce apoptotic resistance and defines an epigenetic vulnerability in pediatric AML
              </a>
            </td>
          <td>
            S. R. Choudhury, Arundhati Chavan, R. Ries, Giselle Almeida Gonzalez, S. Meshinchi, Jason Farrar
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Gene transcription in eukaryotes is orchestrated by intricate regulatory architectures that depend on chromatin context and a diverse repertoire of transcription factors. Consequently, eukaryotic promoters are thought to be incompatible with the streamlined transcriptional machinery of bacteria. Here, we challenge this assumption by demonstrating that a minimal 90-bp fragment of the human TMIGD1 promoter functions as an active, constitutive promoter in Escherichia coli. This compact human sequence drives reproducible reporter expression at levels comparable to established moderate-strength bacterial promoters, revealing an unexpected convergence between human and bacterial transcriptional recognition mechanisms. The ability of a human promoter to operate across domains of life suggests that fundamental features of promoter architecture may be more deeply conserved─or evolutionarily constrained─than previously appreciated. Our findings expand the boundaries of promoter compatibility and open new opportunities for designing hybrid regulatory systems that leverage cross-kingdom transcriptional activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af813299d8850ad64cb73a41bc68dc9fd7821ccb" target='_blank'>
              Human TMIGD1 Promoter Exhibits Robust Activity in Prokaryotic Cells.
              </a>
            </td>
          <td>
            Rosana Meyer, Kaneyoshi Yamamoto, Bridget Whelpley, Daniel Dempsey, Nader Rahimi
          </td>
          <td>2025-12-24</td>
          <td>Biochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transcription-associated torsional stress presents a continuous threat to genome integrity that needs to be tightly controlled. Here we present a method for the visualization of genome maintenance events at sites of active transcription, which builds on a previously developed reporter system that allows for the detection of nascent, MS2-repeat containing transcripts with a YFP-tagged MS2 coat protein. We describe steps to monitor Topoisomerase 1 engagement as a result of transcription-associated topological stress in both S phase and non-S phase cells, and detail procedures to visualize TOP1 cleavage complex formation and concomitant DNA repair factor engagement. We expect this system to provide a powerful means to study effectors of the transcription-associated topological stress response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b2876e36a19d9d9dd1e784475aea19dba3a2ec1" target='_blank'>
              Visualizing DNA repair factor recruitment at sites of transcription in single cells
              </a>
            </td>
          <td>
            Tianyi Guan, Yuepeng Shi, Tae-Hee Lee, Philipp Oberdoerffer
          </td>
          <td>2026-01-03</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Like many cancers, prostate cancer (PCa) relies on tissue- and lineage-specific transcription factors essential for normal tissue function and tumor progression. A key master factor is the androgen receptor (AR), which regulates prostate tissue identity and differentiation, maintaining normal, growth-suppressive, prostate-specific luminal programs via androgen response elements (AREs). However, during tumorigenesis, AR is co-opted to drive oncogenic transcription programs, including reprogramming factors like FOXA1 and HOXB13. Despite AR’s critical role, the mechanisms and functional implications of both its oncogenic and growth-suppressive programs in PCa remain incompletely understood. We hypothesized that the methyl-CpG reader and DNA repair protein MBD4 safeguards lineage trajectories by restraining pioneer factor engagement.



 We performed an epigenetic-focused CRISPR screen (2,508 genes) in an LNCaP ARE-activated model to nominate the regulators of AR programs. We engineered MBD4 knockout and inducible overexpression models and profiled chromatin and transcription by ATAC-seq; CUT&RUN (AR, FOXA1, H3K27ac, H3K4me3, H3K27me3); and RNA-seq across AR+ and AR- lines. Proliferation and lineage-identity programs were also evaluated in normal mouse prostate organoids. Clinical relevance was assessed using multi-cohort patient transcriptomes and cancer-dependency datasets.



 MBD4 emerged as a gatekeeper of the AR/ARE-mediated growth-suppressive program. MBD4 loss increased chromatin accessibility and enhancer acetylation at FOXA1-enriched loci, with FOXA1 binding expanding upon loss and decreasing with MBD4 overexpression. MBD4 localized to AR/FOXA1 enhancers and limited FOXA1 engagement. Functionally, MBD4 knockout accelerated proliferation in AR+ lines and slowed growth in AR- lines (e.g., PC3, DU145). In normal mouse prostate organoids, MBD4 disruption biased luminal epithelial identity programs and reprogrammed enhancer architecture. Across patient cohorts, MBD4 expression and dependency patterns tightly tracked with AR status/lineage, linking lineage context to chromatin plasticity.



 MBD4 acts as a gatekeeper of the AR/ARE-mediated growth-suppressive program by limiting lineage-specific enhancers and restraining FOXA1 engagement. Loss of MBD4 promotes FOXA1-driven reprogramming and lineage plasticity, nominating the MBD4–FOXA1 axis as a potential therapeutic target. AI DISCLOSURE: Generative AI was used to help draft the wording of this abstract; all content was supplied, reviewed, and approved by the authors.



 Xuanrong Chen, Janny Villa-Pulgarin, Jiansheng Wu, Un In Chan, Jude Owiredu, Anjali Yadav, Andrea Sboner, Christopher Barbieri. MBD4 regulates FOXA1 lineage-specific enhancers to promote prostate tumorigenesis and progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/144dd6ef15d0789330238173b369f5abfb772a9e" target='_blank'>
              Abstract A010: MBD4 regulates FOXA1 lineage-specific enhancers to promote prostate tumorigenesis and progression
              </a>
            </td>
          <td>
            Xuanrong Chen, J.A. Villa-Pulgarín, Jiansheng Wu, Un In Chan, Jude Owiredu, Anjali Yadav, A. Sboner, C. Barbieri
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Oncofetal reprogramming has recently emerged as a critical concept in translational cancer research, particularly for its role in driving therapeutic resistance across a variety of malignancies. This biological process refers to a pattern of gene expression that is restricted to embryogenesis, but becomes expressed again in a subpopulation of cancer cells. These genes are typically suppressed after embryogenesis, and their aberrant re-expression in tumors endows cancer cells with stem-like properties and enhanced adaptability. The goal of this review is the following: (i) comprehensively examine the multifaceted nature of oncofetal reprogramming; (ii) elucidate its underlying molecular mechanisms, including its regulators and effectors; and (iii) evaluate its consequences for the therapeutic response in different cancer types. We comprehensively integrate the latest findings from colorectal, breast, lung, liver, and other cancers to provide a detailed understanding of how oncofetal programs interfere with tumor response to treatment. Among the candidates, YAP1 and AP-1 have emerged as central transcriptional drivers of this reprogramming process, especially in colorectal and breast cancers. We also explore the distinct expression patterns of oncofetal genes across different tumor types and how these patterns correlate with treatment outcomes and patient survival. Lastly, we propose a dual-targeting therapeutic strategy that simultaneously targets both cancer stem cells and oncofetal-reprogrammed populations as a more effective approach to overcome resistance and limit recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a262b7681553fb103b5d82c8b95c6fdfada4bf5" target='_blank'>
              Oncofetal Reprogramming: A New Frontier in Cancer Therapy Resistance
              </a>
            </td>
          <td>
            Anh L Nguyen, Molly Lausten, Bruce M. Boman
          </td>
          <td>2026-01-29</td>
          <td>International Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fc898aa0428ac9d6508c8e1b95d29a7714c4715" target='_blank'>
              The role of autoproteolysis and mitoribosomal proteins in regulation of mitochondrial LACTB tumor suppressor
              </a>
            </td>
          <td>
            Sara Escudeiro-Lopes, Juan M. González-Morena, Alzbeta Baudysova, Michal Svoboda, Pavel Marasek, Gabriel Rodriguez-Gomez, Jéssica Mariane Ferreira Mendes, Chung Weng Phang, Sravan Kumar Miryala, Pavel Jakoube, Beata Kovarova, Susana Machado, Yuxiong Feng, Zuzana Keckesova
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Most colorectal cancers are resistant to immune checkpoint inhibitors due to an immunosuppressive tumor microenvironment. LINC00673, previously classified as a long non-coding RNA, has been implicated in tumor progression, but its role in antitumor immunity remains poorly understood. We used single-cell RNA sequencing, flow cytometry, and functional assays in Linc00673 knockout and wild-type mouse models treated with programmed death 1 (PD-1) blockade to dissect changes in immune landscape and tumor cell behavior. Linc00673 deletion enhanced the efficacy of PD-1 therapy, reducing tumor burden and prolonging survival. KO tumors showed increased infiltration of cytotoxic CD8⁺ T cells, reduced exhaustion, and improved antigen presentation. The tumor microenvironment shifted toward a pro-inflammatory state, with elevated dendritic cell function, reduced immunosuppressive macrophages, and improved T cell–tumor interactions. Linc00673 also promoted tumor cell migration and immune evasion independently of immune cells, suggesting dual tumor-intrinsic and -extrinsic roles. LINC00673 suppresses antitumor immunity and promotes tumor aggressiveness. Its deletion synergizes with PD-1 blockade, revealing a promising therapeutic target to overcome immune resistance in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb9ef0a9a3f2685b8b34be389ae9c671f82a1666" target='_blank'>
              Single-cell RNA sequencing unveils enhanced antitumor immunity in colorectal cancer with PD-1 blockade and LINC00673 deletion
              </a>
            </td>
          <td>
            Yifei Zhu, Yanxi Yao, Yaxian Wang, Zhibing Qiu, Dingpei Zhou, Huixia Huang, Keji Chen, Qingyang Sun, Jiayu Chen, Yuxue Li, Jianqiang Tang, Dawei Li, Ping Wei
          </td>
          <td>2026-01-03</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Many studies have shown an association between increased risk of bladder cancer and hypomethylation of retroelements in human peripheral blood, as well as activation of LINE1 (long interspersed nuclear elements), HERV-K (human endogenic retrovirus K) and Alu with insertions into new genomic loci in bladder cancer tissues. Retroelement hypomethylation is accompanied by inactivation of tumor suppressor genes, such as APC, SFRP1, RASSF1A, DAPK1, RARB2, CDKN2A, TP53, RB1, CDKN2A, ERCC2, RUNX3, as well as stimulation of proto-oncogenes FGFR3, TERT, KDM6A, ELF3, PLA2G4A. According to the CancerHERVdb database, between 72.7 and 100 % of bladder cancer tissues are positive for HERV expression. Retroelements are mobile genetic elements and serve as a rich source of microRNA and long non-coding RNA genes which also participate in bladder cancer development. Analysis of the MDTE (miRNA-Derived from Transposable Elements) database allowed to describe 15 microRNAs evolved from mobile genetic elements and possessing oncogenic capabilities and 17 possessing suppressor capabilities. Long non-coding RNAs potentially can be used for targeted therapy of bladder cancer, especially inoperable metastatic drug-resistant cancer. Hereditary predisposition to bladder cancer can be explained by the fact that retroelements are located in intergene, intron and regulatory areas near most of bladder cancer-associated SNPs (single-nucleotide polymorphism). Factors of retroelement activation are aging and viral infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3f3d880fcbdd2957545f29e2396b19549c1b1e4" target='_blank'>
              Role of retroelements in bladder cancer development
              </a>
            </td>
          <td>
            R. Mustafin
          </td>
          <td>2025-12-18</td>
          <td>Cancer Urology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2eabb33613261d21b9be588be9b205da952bb4e2" target='_blank'>
              Polyploidy drives structural and functional evolution in Camellia mitochondrial genomes.
              </a>
            </td>
          <td>
            Jianbin Gao, Yuan Zeng, Boyong Liao, Zuoyun Yin, Wei Wu, Yong-quan Li
          </td>
          <td>2026-01-28</td>
          <td>BMC genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="A significant cancerous growth with a high global incidence and death rate, cancer of the lungs—particularly non-small cell lung cancer (NSCLC), which makes up 80% to 85% of cases—poses a serious threat to public health. Drug resistance, damage to cells that are healthy, and limited effectiveness in advanced stages represent a few of the drawbacks of conventional treatments such radiation and targeted therapy. CRISPR-Cas9 gene editing technology, consisting of key elements like Cas9 protein and guide RNA (sgRNA), enables precise DNA cleavage and editing, offering new prospects for NSCLC treatment with advantages such as strong targeting and high efficiency. The procedures for employing CRISPR-Cas9 for curing NSCLC are explained in detail in this publication, including the use of delivery vectors optimized to overcome lung physiological barriers, treatment strategies like oncogene knockout using Cas9 nickases and adjuvant CAR-T therapy through PD-1 gene knockout, and its application in detection and diagnosis for early screening. Additionally, it analyzes the technical advantages of CRISPR-Cas9 compared to traditional therapies and discusses the challenges and solutions in clinical transformation, with the intention of offering a theoretical guide for the accurate management of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3f6e41f0ec4c718c50a0a00f99bdd6b6402f217" target='_blank'>
              Applying CRISPR-Cas9 technologies for lung cancer treatment
              </a>
            </td>
          <td>
            Shimiao Huang
          </td>
          <td>2025-12-28</td>
          <td>Academic Journal of Science and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966381538c6447576a47e18b17f5babd106c5c90" target='_blank'>
              Structural organization and function of telomeric chromatin.
              </a>
            </td>
          <td>
            Ruben van der Lugt, J. Jacobs
          </td>
          <td>2026-01-07</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Failures in chromosome segregation during meiosis result in aneuploid gametes and are the leading cause of miscarriage. The position and number of crossovers, genetic exchanges between homologous chromosomes, are essential to their accurate disjunction in meiosis. Previous research analyzing nondisjunction of acro- and telocentric chromosomes in human and Drosophila has identified altered positioning and number of crossovers that differs between meiosis I and meiosis II nondisjunction. Limited data from positioning in chromosomes that segregated normally in metacentric trisomies has suggested discrepancies between the behavior of these chromosomes and the acro/telocentrics in nondisjunction, which warrants further investigation. Here, we employ whole-genome sequencing to study spontaneous meiotic nondisjunction of the metacentric chromosome 2 in Drosophila melanogaster. Several patterns of recombination may differentially impact telo- versus metacentric chromosomes: lack of a crossover, distal crossovers, and proximal crossovers. We find that unlike meiotic nondisjunction of the Drosophila X chromosome, nondisjunction of chromosome 2 is not associated with dramatic changes in crossover landscape, but is associated with reduced recombination. Differences in the proportions of NDJ events with altered recombination patterns between chromosomes X and 2 suggest that abnormal crossover positions disparately affect chromosomes with different shapes. Taken together, these findings highlight that the underlying triggers of meiotic nondisjunction are chromosome-specific.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8208b29f245fc09b23a7d8f70794f7fb4b094e6" target='_blank'>
              Chromosome-specific differences in the recombination landscape of spontaneous meiotic nondisjunction
              </a>
            </td>
          <td>
            Carolyn A. Turcotte, J. Sekelsky
          </td>
          <td>2026-01-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The senescence-associated secretory phenotype (SASP) exerts dual roles in tumor suppression and progression, yet how it is regulated in head and neck squamous cell carcinoma (HNSCC) remains unclear. Here, we identify LIM homeobox 1 (LHX1) as a key transcriptional suppressor of STING, whose downregulation enables evasion of SASP-mediated tumor surveillance. Notably, high LHX1 expression correlated with poor prognosis in HNSCC patients. Mechanistically, LHX1, in complex with LDB1, directly bound to the STING promoter to mediate transcriptional repression via the deposition of the repressive histone mark H3K9me3, thereby blocking SASP activation. Depletion of LHX1 restored STING-dependent SASP and impaired cancer stem cell self-renewal. Therapeutic disruption of the LHX1-LDB1 complex using engineered peptides re-activated STING signaling, induced SASP, and significantly suppressed tumor growth. In this study, we employed human and mouse-derived HNSCC cell lines, xenograft models, and clinical samples to assess the functional relevance of LHX1 in regulating SASP and tumor progression. Our findings reveal LHX1 as a master transcriptional repressor of STING-mediated senescence and highlight the therapeutic potential of targeting the LHX1-LDB1 axis to restore tumor-suppressive SASP in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91c542eb07fdf0c718c2bb9764fab625fc0a071c" target='_blank'>
              Targeting the LHX1-LDB1 Complex Restores STING-dependent Senescence Surveillance and Inhibits Head and Neck Cancer Progression
              </a>
            </td>
          <td>
            Mingshu Long, Yang Chen, Ruixue Du, Jiejie Yang, Jingjing Wang, Yunqing Sun, Xuezhang Tian, Shaowei Wang, Yunhong Zhong, Weilian Liang, Junjie Zhang, Z. Shang
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Torsional stress from DNA supercoiling is receiving renewed attention as a driving force for chromosome folding and the establishment of gene activity states. Transcription is a major source of DNA supercoiling, while topoisomerases relax supercoils and solve topological problems that arise during DNA replication, transcription, and chromosome segregation. Recent technological advancements have allowed for the mapping of how torsional stress distributes within the genome and distinguishing between occupancy of topoisomerases on chromatin and sites where they are catalytically engaged. Coupling these innovations to assessments of 3D chromosome conformation and nascent transcription at high resolution have provided a new understanding of the relationships between supercoiling and topoisomerase activity. Here, we summarize the insights obtained from these recent studies and discuss how the interplay between transcription, supercoiling, and topoisomerases shapes cellular gene activity states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10dbaff5bfcd28408c238f71589ba70e8a6c9363" target='_blank'>
              The Roles of Topoisomerases in Transcriptional Regulation
              </a>
            </td>
          <td>
            Kelli D. Fenelon, Ram Madabhushi
          </td>
          <td>2026-02-04</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Ectonucleotide pyrophosphatase/phosphodiesterase 1 (ENPP1), 1st identified in breast cancer and subsequently in multiple other cancer types, is an innate immune checkpoint regulator that recently emerged as a promising biomarker and therapeutic target. Homologous recombination deficiency (HRD) has gained clinical relevance with therapeutic vulnerability, particularly in breast and ovarian cancers. Despite the increasing significance of ENPP1 and HRD in cancer biology and treatment, their potential relationships have not yet been comprehensively investigated. We analyzed the relationship between ENPP1 expression and HRD score across the Cancer Genome Atlas pan-cancer and individual tumor types using the Pearson and Spearman correlations. To account for heterogeneity, pan-cancer samples were clustered using linear regression into 3 groups based on Bayesian Information Criterion. Differential expression, functional enrichment, and survival analyses were performed for these clusters at both the pan-cancer and representative tumor type levels. Although the pan-cancer relationship between ENPP1 expression and HRD score was heterogeneous, significant correlations were observed in 11 tumor types. Linear regression-based clustering resolved this heterogeneity into 3 functionally and clinically distinct groups: Cluster 1 was characterized by proliferation programs; Cluster 2 by extracellular matrix remodeling, differentiation, and immune response; and Cluster 3, by metabolic reprogramming. Clinically, Cluster 3 was associated with better survival than Clusters 1 and 2 in a pan-cancer analysis (P < .0001). At the individual tumor type level, these global cluster features were further modified in tissue-specific contexts, reflecting local microenvironment adaptation. Significant survival differences were observed in patients with adrenocortical carcinoma, chromophobe renal cell carcinoma, low grade glioma, and mesothelioma, further underscoring the tissue-specific modification of global cluster features. Our comprehensive pan-cancer analysis revealed the intrinsic heterogeneity of ENPP1 expression and HRD score, which may arise from complex and dynamic interactions with diverse cancer hallmarks, including proliferation, extracellular matrix remodeling, immune response, and metabolic reprogramming, and can be generalized into 3 clusters with distinct molecular and clinical characteristics. At the individual tumor type level, these global cluster features were further modified to adapt to a tissue-specific microenvironment, manifesting distinct tissue-specific patterns. Collectively, these findings provide a foundation for refining biomarker-driven precision medicine strategies for diverse tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad738ec3c685dfb5e7e6594798ea773356192816" target='_blank'>
              Interaction between ENPP1 and homologous recombination deficiency defines distinct pan-cancer signatures: A retrospective observational study
              </a>
            </td>
          <td>
            Yong Min Kim, Sung‐Hak Lee, Woong Na, Il Ju Lee, Mihye Kwon, Oyeon Jo, Young Soo Song
          </td>
          <td>2026-01-02</td>
          <td>Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background: Head and neck squamous cell carcinoma remains a highly aggressive malignancy with limited predictive biomarkers for prognosis and radiotherapy response. Increasing evidence indicates that pseudogenes are functionally active regulators of cancer biology, yet their clinical relevance in HNSCC is poorly defined. Methods: Using transcriptomic and clinical data from The Cancer Genome Atlas, we analyzed the expression and clinical significance of two pseudogenes, ANXA2P2 and PA2G4P4, in HNSCC. Associations with clinicopathological features, HPV status, tumor subtypes, survival, genomic instability, radiotherapy response, and immune landscape were assessed using bioinformatic tools. Results: Both pseudogenes were significantly upregulated in HNSCC compared to normal tissues. Higher expression levels correlated with adverse clinicopathological features, increased tumor proliferation and wound-healing capacity, and unfavorable TCGA molecular subtypes. High ANXA2P2 and PA2G4P4 expression was associated with reduced overall survival, while their combined low-expression signature identified patients with significantly improved overall and disease-free survival. Notably, lower expression of both pseudogenes was observed in patients responding to radiotherapy, whereas higher expression was linked to genomic instability parameters and enrichment of oncogenic pathways, including MYC, PI3K/AKT/mTOR, cell cycle regulation, and DNA repair. ANXA2P2 expression differed significantly by HPV status, showing reduced levels in HPV-positive tumors. Furthermore, pseudogene expression stratified distinct immune profiles, including immune subtypes, stromal and immune scores, and specific immune cell populations. Conclusions: ANXA2P2 and PA2G4P4 are clinically relevant pseudogenes associated with tumor aggressiveness, immune modulation, and radiotherapy response in HNSCC. These findings support their potential utility as prognostic and predictive biomarkers and provide a rationale for further functional validation in experimental models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33872a456524f0aadce25f6d9d2640840e11a5a0" target='_blank'>
              ANXA2P2 and PA2G4P4 Pseudogenes Are Associated with the Response to Ionizing Radiation and Could Be Used as Potential Biomarkers: In Silico Study
              </a>
            </td>
          <td>
            Tomasz Kolenda, Piotr Białas, Kacper Kamiński, Maria Dziuba, Małgorzata Czernecka, Aleksandra Leszczyńska, Kacper Guglas, Joanna Kozłowska-Masłoń, Paulina Potter, Klaudia Dudek, Nina Grzejda, Karina Tylkowska, Anna Zapłata, Marlena Janiczek-Polewska, Paulina Gieremek, Katarzyna Regulska, Patrycja Mantaj, Anna Florczak-Substyk, Anna Przybyła, U. Kazimierczak, E. Leporowska, Zefiryn Cybulski, Beata J Stanisz, Anna Teresiak
          </td>
          <td>2026-01-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Clonal hematopoiesis (CH) is exemplified by the expansion of genomically altered hematopoietic stem and progenitor cells that may give rise to altered immune repertoires. Driven largely by age and chronic inflammation, CH has been historically linked to myeloid malignancies. Although emerging evidence points to a modest prevalence of CH within solid tumors, the impact of these naturally occurring alterations on anti-tumor immunity and therapeutic responses remains less defined. Here, we employed somatic mutation calling algorithms to detect single nucleotide variants (SNVs) in both epithelial and immune compartments within tumors. Our preliminary analyses of single-cell RNA-sequencing (scRNA-seq) of 32 primary, therapy-naïve prostate tumors encompassing 11 patients identified conventional mutations representative of both CH and prostate cancer. Within the tumor-epithelial compartment, we identified recurrent FOXA1 class 1C mutations in 27% (3/11), alongside NCOR1 mutations in 18% (2/11), IDH1 in 9% (1/11), and SPOP in 9% (1/11) of patients. Of note, these SNVs were detected across multiple samples for the same patient. Interestingly, we also detected germline HOXB13 G84E variants, associated with increased risk for prostate cancer, in 2 cases at a high clonality within epithelial cells (>75%). CH alterations were identified in 27% (3/11) of patients based on the detection of exonic, nonsynonymous SNVs within myeloid cells. Specifically, these variants affected SF3B2 (variant allele frequency; VAF = 0.30), UBE2A (VAF = 0.30), and KMT2C (VAF = 0.43) that are predicted deleterious across multiple classifier algorithms. Patients with detectable CH alterations displayed immunologically hotter tumor microenvironments, with CH+ patients having significantly higher proportions of tumor-infiltrating monocytes (16.22% vs 7.13%, Fisher’s exact test p<0.0001), macrophages (6.49% vs 2.88%, p<0.0001), CD8+ T cells (10.35% vs 3.33%, p<0.0001), CD4+ T cells (2.81% vs 0.74%, p<0.0001), and B cells (1.81% vs 0.59%, p<0.0001), and lower fractions of epithelial cells (50.94% vs 69.81%, p<0.0001). In summary, we leveraged de novo mutational algorithms to identify somatic alterations using scRNA-seq. To our knowledge, our study is the first to detect CH alterations in prostate cancer at the single-cell resolution. Further work will integrate our mutational signatures with transcriptomics to elucidate the impact of both cancer- and immune-intrinsic alterations on anti-tumor immunity. Additionally, we will incorporate mitochondrial DNA-based lineage tracing algorithms on matched peripheral blood, primary, and metastatic tumor samples to deconvolute the spatiotemporal regulation of naturally occurring CH alterations across cancer progression. We envision our findings will illuminate novel drivers of immune function and regulation within prostate cancer that may be exploited for future immunomodulatory therapies.



 John R. Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang. Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03587209005d8227c89ae44e47e8117ddd350a30" target='_blank'>
              Abstract A040: Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum
              </a>
            </td>
          <td>
            John R Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 Fibrolamellar Carcinoma (FLC) is a liver cancer of adolescents and young adults driven by the fusion of DNAJB1, a heat shock protein and PRKACA, the catalytic subunit of protein kinase A (PKA). It is characterized by increased PKA activity leading to widespread changes. A profile of FLC at genomics, transcriptomics, methylomics and proteomics in a multi-omic study that studied 1,412 tumor-normal paired liver samples from patients with FLC, Hepatoblastoma, Intrahepatic Cholangiocarcinoma, and Hepatocellular Carcinoma, yielded the FLC Data Base (FLC-DB). To facilitate the visualization of the multiomics, we developed a user-friendly web interface for the FLC-DB. It uses functions from the R-library OmicsKit and enables interactive exploration for non-bioinformaticians to retrieve information and generate publication-ready plots. It comprises 3 main tabs. First, Documentation, which describes the samples and datasets, the transcriptomic FLC Signature, and a reduced set of 35 genes with strong and recurrent upregulation across all samples, selected for clinical purposes. Second, the Transcriptomics tab, where users can interactively explore the distribution of gene counts across liver cancers, comparing normal samples against tumors and tumor subtypes. This has the option to directly perform Differential Expression Analysis using subsets that can be dynamically defined by the user in the web interface. Third, the Genomics tab, which allows exploration of Single Nucleotide Variants (SNV) from Whole-Genome and Whole-Exome Sequencing (WGS/WES) and download of the SNVs annotations and perform downstream analyses. This allows exploring questions such as: Is a transcript consistently upregulated across FLC studies? Are there recurrent SNVs in FLC? What gene signatures are shared between FLC and HBL? The FLC-DB web interface facilitates exploration of complex multi-omics data, broadening its access to non-bioinformatic researchers, patients and the community. It is currently available at https://fibrolamellardatabase.org/.



 Daniel F. Guevara-Diaz, David Requena, Sanford M. Simon. The fibrolamellar carcinoma database [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6039ebc7493e6eddced2c3191b76583f0aea8b24" target='_blank'>
              Abstract B031: The fibrolamellar carcinoma database
              </a>
            </td>
          <td>
            Daniel F. Guevara-Diaz, David Requena, Sanford M. Simon
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35607d62497aa71c83442fc00cbc1c011c30044b" target='_blank'>
              DICER1 syndrome mutations lead to a gain of 3p-miRNA function, HERVH activity and increased metastatic potential
              </a>
            </td>
          <td>
            K. Gordon, N. Bellora, J. Witteveldt, Joanna K. Wojtus, Felix Mueller, Katharina Kases, Pilar G. Marchante, G. Peris, Shelagh Boyle, Sara R. Heras, Atlanta G. Cook, Cei Abreu-Goodger, S. Macias
          </td>
          <td>2025-12-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63700b1580a28e6e919137fb8581425a7fd29e67" target='_blank'>
              Digging deeper into cancer resistance in the extraordinarily long-lived naked mole-rat
              </a>
            </td>
          <td>
            Tanvi T Patel, Vince G Amoroso, Kaitlyn N. Lewis Hardell, Vikram Narayan, Martha A. Delaney, Rochelle Buffenstein
          </td>
          <td>2026-01-27</td>
          <td>Mammalian Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer cell adaptation to their physical tumor microenvironment is a key driver of malignancy. Recent experimental evolution experiments show that the soft extracellular matrix (ECM) can impose a selection pressure on genetically variable tumor populations. Over months of sustained culture, the selection pressure leads to enrichment of specific genetic variants with high fitness, but the mechanisms underlying the high fitness of these soft-selected clones are not fully understood. Here, we used a combination of RNA-seq, ATAC-seq, and RRBS-seq to compare soft-selected populations with non-selected ancestral populations cultured on soft ECM. We demonstrate that ancestral populations grown on soft ECM for short durations are characterized by a stressed cell state with low fitness marked by cell cycle arrest and distinct metabolic shifts, whereas sustained culture selects for a robust proliferative phenotype. Mechanistically, selected cells exhibit a silenced ancestral stress response through epigenetic modifications, characterized by reduced chromatin accessibility and de novo DNA methylation, including CDH1 promoter hypermethylation. This repressive landscape supports a high-fitness state defined by elevated MYBL2 and FAK levels. An in-silico mechanism-based model shows that these molecular differences, together with high YAP1 nuclear localization in soft-selected cells, are salient features of genetic clones capable of FAK upregulation. These findings uncover a coordinated genetic and epigenetic mechanism driving cancer cell evolution in mechanically soft microenvironments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4db71dfbf014de56876df2b0949f05913ac70b1" target='_blank'>
              Multi-omics investigation reveals molecular determinants of cancer cell evolution on soft extracellular matrix
              </a>
            </td>
          <td>
            Sarthak Sahoo, S. Khanna, Ting-Ching Wang, Tanmay P. Lele, M. Jolly
          </td>
          <td>2025-12-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Micronuclei (MN) are small extranuclear chromosomal fragments that arise from genomic instability and serve as established biomarkers for genotoxicity and disease susceptibility. Once considered only markers of disease, they are now recognized as active, causative drivers of disease progression, driven by emerging evidence of epigenetic regulation within these structures. The micronucleus assay is recognized as a cost-effective and minimally invasive method for monitoring genotoxicity resulting from both chronic and early exposure to environmental factors such as arsenic and lead, as well as from genetic instability associated with cancer progression. This review critically examines the expanding role of MN beyond traditional cytogenetic endpoints, with particular emphasis on recent insights into their epigenetic landscape. Mass spectrometry-based studies have demonstrated that MN possess distinct histone posttranslational modification signatures compared to primary nuclei, including alterations in H3K27ac, H3K9ac, and H3K18ac. These modifications affect chromatin structure, gene expression, and DNA repair mechanisms. In the context of xenobiotic exposures, MN-associated epigenetic changes may function as early indicators of disease progression. Additionally, rupture of the MN envelope can activate innate immune responses through the cGAS-STING pathway or result in chromothripsis, both of which contribute to cancer progression. The concept of "Micronuclear Epigenetics" is highlighted, with its potential application in high-throughput diagnostic platforms, particularly liquid biopsy, discussed. This approach may enhance early detection and risk stratification in exposure-induced toxicity and diseases such as cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a963e7666be290f5a7f2a006e9788007c4d9584f" target='_blank'>
              Potential of Micronuclear Epigenetic Signatures in Analyses of Toxicity and Genomic Instability.
              </a>
            </td>
          <td>
            Somnath Paul, Ankita Das, A. Bandyopadhyay, Ashok K. Giri
          </td>
          <td>2026-01-27</td>
          <td>Cell biology international</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce1dd81fa69ca7596c6e2f6203643899d73e28c1" target='_blank'>
              In vivo isogenic modelling unveils a TP53-mediated relapse phenotype in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Stéphanie Gachet, S. Quentin, L. Hernandez, Loïc Maillard, M. Passet, R. Kim, Hugo Bergugnat, Melha Benlebna, Maxime Boy, Véronique Parietti, P. Fenaux, André Baruchel, Hervé Dombret, N. Boissel, François Sigaux, Hugues de Thé, E. Clappier, J. Soulier
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Colorectal cancer ranks third in global incidence and second in cancer mortality. Patient-derived models are irreplaceable for studying tumor biology. We established a human epithelial cell line from a rectal adenocarcinoma overexpressing cancer stem cell marker ALDH1A1, and we investigated the effect of ALDH1A1 knockout on tumor cell traits. The cell line and its CRISPR-Cas9 ALDH1A1 knockouts were characterized by genomic and cytogenetic methods (CNV, WES, RNAseq, karyotype), in vitro (proliferation, response to chemotherapy, migration, invasion, apoptosis), and in vivo methods. We identified the landscape of somatic mutations and copy number alterations in the original tumor and the derived cell line. Genetic attenuation of ALDH1A1 was characterized by an increase in migratory potential and extensive metastatic ability, accompanied by reduced growth of subcutaneous xenografts and alterations in gene expression associated with inhibited proliferation and promoted invasion and metastasis, ultimately resulting in dysregulation of the Wnt signaling pathway. Increased metastatic potential was also confirmed in HT-29 cells after ALDH1A1 genetic attenuation. CRISPR-Cas9-mediated editing led to functional, cellular, and molecular changes confirming the role of ALDH1A1 in colorectal cancer carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b72d554a35d6005b3610cd9e7621ca3ab6650f1a" target='_blank'>
              Genetic attenuation of ALDH1A1 increases metastatic potential and aggressiveness in colorectal cancer.
              </a>
            </td>
          <td>
            M. Poturnajova, Z. Kozovska, Ondrej Pos, Kristína Pavlov, Sachin Gulati, Peter Makovický, Kristina Jakic, M. Burikova, Eva Sedlackova, Barbora Svitková, S. Tyčiaková, V. Bystrý, Nicolas Blavet, Boris Tichý, Matej Hrnciar, J. Budiš, Miroslav Tomas, Peter Dubovan, Georgina Kolnikova, Veronika Repaska, Nikoleta Mojzesova, E. Zomborská, D. Pindak, Michal Mego, T. Szemeš, M. Matúšková
          </td>
          <td>2026-02-03</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="As a tumor suppressor gene, CCDC6 encodes a coiled-coil domain-containing protein that is ubiquitously expressed and involved in crucial cellular processes such as DNA damage response and apoptosis, although its precise mechanisms remain elusive. Initially identified as part of a fusion gene, CCDC6 can form fusion genes with a variety of proto-oncogenes, including both kinase- and non-kinase-coding genes, thereby facilitating oncogenesis. Alterations in CCDC6 expression across various cancers underscore its intricate role and potential influence on the efficacy of anticancer therapies. Recent findings have demonstrated that CCDC6 can undergo liquid-liquid phase separation (LLPS) and facilitate the LLPS of its associated fusion proteins, providing new perspectives on its functional characterization and potential therapeutic implications in related diseases. We present a comprehensive overview of CCDC6, encompassing its protein characteristics and physiological and genomic aspects. Furthermore, we explored the association between CCDC6 alterations and carcinogenesis, as well as their implications for therapeutic interventions. The objective of this review is to furnish the medicinal community with current information and valuable insights pertaining to diseases associated with CCDC6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872145069c4ef81fa1ace0d713f50e945a6721cc" target='_blank'>
              The multiple roles of the disordered protein CCDC6 in cancer development.
              </a>
            </td>
          <td>
            Guifeng Wei, Yiji Chen, Yichao Kong, Donglai Li, Yang Wang, Ting Qiu, Xiabin Chen
          </td>
          <td>2026-01-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The extraordinary repetitive content of human acrocentric short arms has prevented detailed investigations into recombination and de novo mutation. Integrating multiple sequencing technologies, we created 156 phased short arms and assessed 107 intergenerational transmissions from 23 samples in a four-generation pedigree. We observed a significant depletion (P<0.0001) of p-arm allelic recombination but one ectopic chr13–chr21 recombination breakpoint mediated by a 630 kbp segmental duplication mapping 1.6 Mbp distal to the SST1 array. In contrast, 18 maternal-biased q-arm allelic recombinations are significantly enriched within 5 Mbp of the centromere. Compared to autosomal euchromatin, the overall p-arm de novo single-nucleotide variant rate (1.33×10⁻⁷ per base pair per generation) is 10-fold higher, with a significant reduction of C>T but increased C>G and A>C mutations. We hypothesize that acrocentric sequence composition biases and the dearth of allelic recombination contribute to an elevated mutation rate and unique mutational signatures suggestive of mismatch repair defects and oxidative stress-induced DNA lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dde5730289623fe97755031a7f3f19f52605e3c3" target='_blank'>
              Human acrocentric chromosome short arm de novo mutation and recombination
              </a>
            </td>
          <td>
            Jiadong Lin, F. K. Mastrorosa, Michelle D. Noyes, DongAhn Yoo, A. Rhie, David Porubsky, Kendra Hoekzema, Katherine M. Munson, Nidhi Koundinya, W. S. Watkins, Lynn B. Jorde, Aaron R. Quinlan, Deborah W. Neklason, A. Phillippy, E. Eichler
          </td>
          <td>2025-12-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Objectives: Although Aurora kinase A (AURKA) expression has been investigated in many cancer types, studies focusing on its role in hepatitis B virus-associated hepatocellular carcinoma (HBV-HCC) are limited. In this study, we examined the activity of AURKA and its substrates (PLK1, P53, and BRCA1) in HBV-HCC and cryptogenic hepatocellular carcinoma (Cr-HCC) cases. Methods: The study groups consisted of HBV-HCC, Cr-HCC, and healthy liver tissue cases. AURKA copy number variation (CNV) was analyzed using molecular methods. AURKA expression was evaluated by molecular and immunohistochemical (IHC) methods. AURKA substrates P53Ser315, PLK1Thr210, and BRCA1 were also analyzed by IHC. Results: There was no increase in AURKA gene copy number among the groups (2−∆∆Ct < 2). AURKA level was significantly increased in both test groups (p < 0.001). At the protein level, AURKA was significantly higher in both cancer groups compared to the control group (p < 0.001). Phospho-P53Ser315 levels were significantly higher in both HBV-HCC and Cr-HCC groups compared to the control group (p = 0.002 and p < 0.001, respectively). Cr-HCC cases also showed significantly higher levels compared to HBV-HCC (p = 0.025). For phospho-PLK1Thr210, Cr-HCC cases showed statistically higher expression compared to both the control group and HBV-HCC cases (p = 0.001).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e170fbdf0949ac51576d078567eb9acab99746e" target='_blank'>
              The Role of Aurora Kinase A in HBV-Associated Hepatocellular Carcinomas: A Molecular and Immunohistochemical Study
              </a>
            </td>
          <td>
            M. Huz, Nese Karadag Soylu, Ahmet Koç, Z. Kucukakcali, Nefsun Danis, O. Ozhan
          </td>
          <td>2026-01-01</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="In cells, highly coordinated multivalent interactions give rise to discrete functional assemblies-commonly referred to as biomolecular condensates-that compartmentalize the molecular components required for specific biological reactions. These condensates are increasingly recognized as organizational entities with central roles in normal cellular regulation and in the pathogenesis of human cancers. In a recent issue of Cell, Datar and colleagues investigated the condensation of NPM1c, a common gene mutation in acute myeloid leukemias (AMLs). They demonstrated the necessity and sufficiency of NPM1c in forming nuclear condensates termed coordinating bodies (C-bodies), which show co-partitioning of a suite of transcriptional coactivators such as NUP98, KMT2A/MLL1, Menin and XPO1/CRM1. While C-bodies are necessary for driving NPM1c-mutant AMLs, blockade of the co-partitioned component within C-bodies, such as the XPO1/CRM1 or Menin interaction by inhibitors, significantly alters the condensate composition and functionality. Likewise, a systematic deletion study of various regions within NPM1c pointed to a role for the coordinated multivalent interaction in establishing the functional condensates, as previously reported in studies of the Wilms tumor-causing ENL mutants and AML-causing NUP98 onco-fusions. Co-mixing of C-bodies and condensates formed by the onco-fusion of NUP98 or KMT2A/MLL1 in cells suggested them to be biophysically indistinguishable, indicative of a shared pathogenic mechanism. Altogether, recent studies of multiple genetic drivers in human cancers have revealed a type of chromatin-bound multi-component onco-condensates, which shall motivate the development of onco-condensate disruptors that could potentially be used as the broad treatments for cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cc999451ae4cdf83273f4d0ad6380f1649aa9ab" target='_blank'>
              A unified model: chromatin-bound multi-component onco-condensates.
              </a>
            </td>
          <td>
            Benjamin K Lau, Elena Haarer, Jeong Hyun Ahn, G. Wang
          </td>
          <td>2026-01-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Most genes are transcribed from multiple transcription start sites (TSSs), also known as alternative TSSs, which are highly regulated and can lead to various gene regulatory outcomes, including changes in translation efficiency and protein isoform expression. Transcription factors and chromatin regulators control alternative TSS selection. DNA supercoiling affects multiple aspects of transcription, including transcription initiation; however, its regulatory effect on genes with multiple TSSs is not known. Here, we investigated how depletion of topoisomerases, which resolve DNA supercoiling events, impacts alternative TSS usage in Saccharomyces cerevisiae. We depleted topoisomerases (Top1 and Top2) during early meiosis, where alternative TSS usage is prevalent, and applied an improved TSS sequencing protocol. We show that supercoiling affects alternative TSS usage at almost 600 genes. Increased alternative and aberrant TSS usage was observed near and within open reading frames, likely resulting from transcription-induced supercoiling originating from upstream alternative TSSs. Top1/Top2 co-depletion most strongly affected genes with highly used, widely spaced alternative TSSs. Our correlative analyses support a model where DNA supercoiling release during transcription is critical for correct TSS selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5b0512c4413bf00c8391b3e207d2e934a5c9598" target='_blank'>
              Topoisomerases regulate alternative transcription start site selection in yeast.
              </a>
            </td>
          <td>
            Eleanor Elgood Hunt, Claudia Vivori, Waleed S. Albihlal, F. V. van Werven
          </td>
          <td>2026-01-06</td>
          <td>G3</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Dysregulation of repair pathways, specifically the base excision repair (BER) axis, is frequently observed in cancer. This dysfunction is associated with accelerated tumor growth and reduced patient survival, suggesting that BER components may serve as potential targets for cancer therapy. This study focused on the expression of Uracil-DNA glycosylase (UNG), a key enzyme in BER, in colorectal tumor tissues and adjacent normal tissues. We systematically explored the clinical significance, biological functions, and molecular mechanisms of UNG in colorectal cancer (CRC). Our findings revealed that UNG was significantly upregulated in CRC, and its higher expression correlated with poorer patient outcomes. Functional assays demonstrated that silencing UNG inhibited cell proliferation, reduced migration and invasion capabilities, and induced apoptosis along with S/G2 phase cell cycle arrest. Mechanistically, UNG appears to interact with the AMPK, AKT/mTOR, and ERK signaling pathways. These results suggested that UNG exhibited oncogenic potential in CRC by regulating malignant behaviors and oncogenic signaling cascades, positioned it as a potential prognostic biomarker and therapeutic target for CRC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40071783d4dcf5a7a90c5996af36927f2f827940" target='_blank'>
              Silencing Uracil-DNA glycosylase inhibits colorectal cancer progression
              </a>
            </td>
          <td>
            Jingjing Yang, Li-zhou Shi, Yong-wei Hu, Hua Chen, Qing-Hua Wang, Wei Han
          </td>
          <td>2026-01-04</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8bb1ad1f1afa1b592836b21a838884cdc277c82" target='_blank'>
              Targeting EZH2 Oncogenic Splicing: Decoding the Regulatory Network and Antisense Correction
              </a>
            </td>
          <td>
            Md Rafikul Islam, Preeti Nagar, Naomi McNaughton, Shabiha Afroj Heeamoni, Md Mahbub Hasan, Shila Kandel, Panagiotis Tsakiroglou, W. Dalton, Omar Abdel-Wahab, A. Krainer, M. A. Rahman
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="The AID/APOBEC gene family originated during evolution to perform functions related to organismal defense. At present, AID/APOBEC genes encode proteins that are structurally similar but functionally diverse. Most AID/APOBEC proteins exhibit deaminase activity toward cytidyl nucleotides in single-stranded DNA and RNA. In addition to cytidine deamination within viral genomes – which restricts infection – these deaminases are also capable of inducing mutations in the human genome. APOBEC-associated mutations represent one of the most common mutational signatures in cancer: they have been detected in approximately 75 % of cancer types and in more than 50 % of all tumors. 
The aim of this review is to analyze the role of AID/APOBEC-induced mutations in the genesis of human malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/631a1dbb121589589ddf78ee79bee08d68b13140" target='_blank'>
              AID/APOBEC family and its role in carcinogenesis
              </a>
            </td>
          <td>
            G. Volgareva
          </td>
          <td>2025-12-14</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6da8c58eace175b556f1165210d11f554c8a306d" target='_blank'>
              Regulatory Mechanisms of Maternal Imprinting at the Dlk1-Dio3 Domain
              </a>
            </td>
          <td>
            Ariella Weinberg-Shukron, Frances L Dearden, Ana Moreno-Barriga, Lok Ting Nick New, Marco Müller, Rebecca C. Rancourt, C. Edwards, A. Ferguson-Smith
          </td>
          <td>2026-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Tumor evolution involves genetic, transcriptional, and phenotypic alterations that shape cancer cell behavior and interactions with the microenvironment. While single-cell technologies have advanced our understanding of this process, spatial dynamics remain incompletely characterized. Here, whole-exome sequencing (WES), imaging mass cytometry (IMC), and spatial transcriptomics (ST) were integrated to study molecular evolution and immune responses in two lung adenocarcinoma (LUAD) mouse models: a genetically engineered model (129S4/Sv-KrasLSL-G12D, termed 129S4 K) and a carcinogen-induced precancer model (129S4 U). Compared to 129S4 K, 129S4 U tumors exhibited higher mutational, neoantigen but lower copy number variation (CNV) burdens at matched developmental timepoints, consistent with findings of higher mutational burden in human smoking-related LUAD than nonsmoking LUAD. We profiled over 1.4 million spatial single cells from 284 IMC regions of interest and 51,531 spatial transcriptomic spots from 156 lesions across 141 mice. Macrophage abundance increased with tumor progression, while CD8 T-cell and B-cell densities declined in late-stage LUAD. 129S4 U showed greater immune infiltration in both tumor and adjacent normal tissue, higher T-cell cytotoxicity signature score in line with its higher mutational and neoantigen burdens. LUAD progression was marked by early morphological shifts and late-stage changes in cell states and interactions. These data define spatial and genetic landscapes of LUAD development and provide a framework for investigating immune evolution and therapeutic strategies in early carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0e7ad7c0953d4cab7fb1f9dcb251447c108ead" target='_blank'>
              Spatial Profiling Reveals Distinct Molecular and Immune Evolution of Mouse Lung Adenocarcinoma Precancers with or Without Carcinogen Exposure.
              </a>
            </td>
          <td>
            Bo Zhu, M. Aminu, Pingjun Chen, Jian-Rong Li, Chuanpeng Dong, Chenyang Li, Yanhua Tian, Shao-Wei Lu, Hong Chen, Chenxi Ma, Xin Hu, Jie-Tong Ye, Andrew Y Liu, Beibei Huang, Frank R. Rojas, Parra Cuentas Edwin Roger, Ou Shi, M. Nilsson, A. Poteete, Khaja B Khan, Wei Lu, Luisa M. Solis Soto, Junya Fujimoto, C. Haymaker, Ignacio I. Wistuba, Zhubo Wei, Linghua Wang, Don L. Gibbons, Ken Chen, Alexandre Reuben, Jason M Schenke, J. Heymach, Chao Cheng, Jia Wu, Jianjun Zhang
          </td>
          <td>2026-01-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="


 Prostate cancer is among the most prevalent cancer types and advanced castration-resistant prostate cancer (CRPC) remains a lethal disease. It is driven by oncogenic transcription factors, most notably the androgen receptor (AR) which represents to this day the main therapeutic target. That said, more transcription factors have emerged supporting tumor growth in late stages when prostate cancer cells lose AR expression through lineage plasticity. However, targeting this plethora of transcriptional drivers throughout disease progression is a daunting task. Yet, disrupting common enablers upstream of oncogenic transcription factors may provide an opportunity to shut down oncogenic output of prostate cancer cells regardless of their CRPC subtype. Indeed, the pioneer transcription factor FOXA1 is a strong and highly selective Achilles' heel in prostate cancer (www.depmap.org), critical for opening chromatin and enabling AR to exert its oncogenic function. Moreover, its homologue FOXA2 has been shown to be upregulated in AR-negative lineage plasticity prostate cancers as well. That said, effective FOXA1/2 inhibitors are currently not available. Here, we show that the FOXA1 homologue FOXA2 cooperates with the AR-pioneer factor FOXA1 in mediating AR-independent cell proliferation in various prostate cancer subtypes. In AR-negative cells, this colocalization is reinforced and supports the gene transcription of lineage-specific oncogenic transcription factors. Joint loss-of-function of both FOXA1 and FOXA2 leads to the collapse of lineage-specific oncogenic transcription factors. Importantly, using virtual drug screening we identify a non-covalent small molecule capable of directly binding and disrupting the interaction of the conserved forkhead domains with DNA. The inhibitors target both FOXA1 wild-type tumors and tumor characterized by recurrent driver mutations. Mechanistically, the inhibitors bind to a hydrophobic cavity within the N-terminus of the conserved forkhead domains of FOXA1 and FOXA2. Treatment with T and derivatives inhibits specifically the oncogenic transcriptional output programs related to different AR-positive and AR-negative prostate cancer subtypes and effectively blocks tumor growth in culture and after intraperitoneal injection in xenograft models in vivo. In summary, our findings uncover an unexpected collaboration of FOXA1 and FOXA2 in disease progression and a druggable key dependency for AR-positive and -negative CRPC regardless of their specific lineage-specific oncogenic program.



 Nicolo Formaggio, Jacopo Sgrignani, Gayathri Thillaiyampalam, Claudio Lorenzi, Yanick Uebelhart, Diego Camuzi Cassiano, Andrea Rinaldi, Matteo Pecoraro, Roger Geiger, Raffaella Santoro, Andrea Cavalli, Jean-Philippe Paul. Theurillat. Targeting of FOXA1 and FOXA2 by small molecules disables the oncogenic output of castration resistant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR020.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4af0e2296f18efc47feaa25f0dfc62945878e4ac" target='_blank'>
              Abstract PR020: Targeting of FOXA1 and FOXA2 by small molecules disables the oncogenic output of castration resistant prostate cancer
              </a>
            </td>
          <td>
            Nicolò Formaggio, Jacopo Sgrignani, Gayathri Thillaiyampalam, Claudio Lorenzi, Yanick Uebelhart, Diego Camuzi Cassiano, Andrea Rinaldi, Matteo Pecoraro, Roger Geiger, Raffaella Santoro, Andrea Cavalli, Jean-Philippe Paul. Theurillat
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df3ef8e203c378a8d1a7856db6aaa8c3b98aa293" target='_blank'>
              Dissecting the contribution of transposable elements to interphase chromosome structure.
              </a>
            </td>
          <td>
            Liyang Shi, Zhen Xiao, Xuemeng Zhou, I. A. Babarinde, Xiuling Fu, Gang Ma, Zhuoqi Huang, Li Sun, Jiangping He, Alexander V. Strunnikov, A. P. Hutchins
          </td>
          <td>2026-01-20</td>
          <td>Genome biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c91fe51488198f3ff4051c25194da67b2c9762e1" target='_blank'>
              Human Accelerated Regions: Functional Studies and Methodological Approaches (A Review)
              </a>
            </td>
          <td>
            A. Knyazeva, T. A. Shnaider
          </td>
          <td>2026-01-30</td>
          <td>Cell and Tissue Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC), a highly lethal disease often developing in the context of chronic liver disease, is marked by high relapse rates and low 5‐year survival. To investigate the multicellular ecosystem and molecular features of hepatocarcinogenesis and progression, a comprehensive analysis integrating single‐cell and spatial transcriptomics was conducted. Significant alterations were observed in various cell types, notably an increase in liver parenchymal and endothelial cells, which play critical roles in tissue repair and angiogenesis. Scissor+ cells, predominantly hepatocytes, were identified as potentially evolving into malignant cells with poor prognosis and enhanced immune evasion capabilities. Key genes associated with Scissor+ cells exhibited potential as cancer prognostic markers. Additionally, the disruption of cellular communication networks during malignant progression revealed the evolution of hepatocytes into cancerous phenotypes. The enhanced signaling pathways, such as MDK‐SDC4 and COL4A‐SDC, were identified as pivotal in driving the malignant transformation of hepatocytes. This transformative process, wherein Scissor+ cells acquire malignant features, highlights novel mechanisms of cancer progression. These findings provide deeper insights into hepatocarcinogenesis and offer potential avenues for targeted and personalized therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ea05c9d911e5acfb02ab472396d64afacfb09aa" target='_blank'>
              Spatiotemporal Evolution of Malignant Hepatocyte Clones Unveils Immune Evasion Features and Therapeutic Vulnerabilities in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Shizheng Xiong, Dekang Ren, Jiaming Jin, Linjie Zhao, Chengjun Gong, Jiarui Liu, Wanjie Zheng, Zhiheng He, Haochuan Guo, Li Guo, Tingming Liang
          </td>
          <td>2026-01-01</td>
          <td>Biotechnology Journal</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The ubiquitin-proteasome system (UPS) is a crucial determinant of protein stability and activity in various aspects of physiological function and disease development. The well-characterized class of regulatory enzymes within the UPS is deubiquitinating enzymes (DUBs), and their effects, particularly those of the ubiquitin-specific proteases (USPs), oppose ubiquitination. All DUB activities can be more or less disrupted in various cancers, including breast cancer, and DUB alterations contribute substantially to tumor initiation, progression, and many forms of therapeutic resistance. In this review, we discuss the various molecular mechanisms of action of USPs on breast cancer hallmarks, including proliferation, aggression or metastasis, evasion of apoptosis, immune evasion, and metabolic programming. We evaluate how specific USPs stabilize oncogene members by deubiquitinating target proteins or deubiquitinating tumor suppressors, thereby influencing a variety of cellular behaviors, from regulating the cell cycle to modulating immune responses. Due to their important role in breast cancer pathology, the alterations of USP activities and the functional roles of selective USPs will also demonstrate some ways USPs present promising therapeutic targets in breast cancer. We will provide a comprehensive overview of USP inhibitors to date, focusing on their utilization in developing and describing efficacy in breast cancer models. Pharmacological inhibitors of specific USPs, such as pimozide, trifluoperazine, rottlerin, 6-thioguanine, and costunolide, are highlighted for their potential to inhibit proliferation, metastasis, induce apoptosis, and circumvent therapy resistance across breast cancer subtypes (triple-negative and HER-2 positive). The review highlights the complex and often contradictory roles of USPs in breast cancer and points to the immense promise of targeting these enzymes to develop new and efficacious anticancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f7da39e8cc82d88b1919d7feec1b7d98af8afce" target='_blank'>
              Ubiquitin-Specific Proteases in Breast Cancer: Unraveling Oncogenic Roles and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Jitendra Gupta, Nisreen Salah Majeed, Mudhr Kedhem, Mohammad Y Alshahrani, R. Jyothi S., Priya Priyadarshini Nayak, Ashish Singh Chauhan, Harshit Gupta, Sami G Almalki, Mohammed Jawad Alnajar
          </td>
          <td>2026-01-13</td>
          <td>DNA and Cell Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Prostate cancer lineage plasticity, characterized by histologic transformation of prostate adenocarcinoma (PRAD) to neuroendocrine prostate cancer (NEPC), is an emerging mechanism of treatment resistance. NEPC accounts for up to 15% of treatment-resistant prostate cancers and is associated with poor prognosis, highlighting an unmet need for new therapies. NEPC lineage reprogramming is primarily driven by epigenetic dysregulation including differential activity of histone and DNA methyltransferases. EZH2, the catalytic component of the Polycomb repressive complex 2 (Polycomb), is overexpressed in most treatment-resistant prostate cancers and is implicated as a driver of disease progression. In this study, we define the differential, lineage-specific action of Polycomb in both PRAD and NEPC subtypes to better understand its role in modulating differentiation and lineage plasticity, and to identify novel targetable drivers of NEPC. Epigenetic H3K27me3 CUT&Tag profiling of treatment-resistant PRAD (n=9) and NEPC (n=9) rapid autopsy clinical samples revealed that Polycomb targets, including NE-lineage transcription factors, are de-repressed in NEPC. Mechanistically, Polycomb modulates H3K4/K27me3 bivalent promoters, leading to the upregulation of NEPC-associated transcriptional drivers (e.g., ASCL1) and neuronal gene programs which facilitate forward differentiation following EZH2 targeting in NEPC patient-derived organoid/xenograft models. Notably, we identified prospero-homeobox 1 (PROX1) as an understudied cell-fate determining transcription factor that is epigenetically de-repression by differential Polycomb activity. Integrative H3K27ac CUT&RUN and Capture Hi-C analyses revealed potential enhancers of PROX1 in NEPC that may be regulated by ASCL1. We sought to functionally characterize the role of PROX1 in NEPC. An unbiased CRISPR screen in two NEPC patient-derived organoid models demonstrated high cellular dependency for PROX1; knockout of PROX1 impeded tumor growth in NEPC models, while overexpression of PROX1 in PRAD promoted tumor growth and spontaneous metastases. Transcriptomic and cistromic analyses across models of CRPC and NEPC pointed to PROX1 regulation of neuroendocrine-lineage transcriptional programs. Immunoprecipitation followed by mass spectrometry identified three novel phosphorylated sites in the DNA-binding domain of PROX1 that are critical for its stability and function. In silico analyses of these phosphorylation sites predicted CHEK1 as a potential upstream kinase which could be exploited for therapeutic targeting of PROX1. Our findings provide insights into the potential role for bivalent promoters in Polycomb-mediated lineage reprogramming, which may facilitate forward differentiation in NEPC upon EZH2 inhibition. Further investigation of de-repressed candidates defines the role of PROX1 as a driver of NEPC and a potential therapeutic target.



 Varadha Balaji Venkadakrishnan, Nathaniel C. E. Voss, Nicole Traphagen, Richa Singh, James Neiswender, Keira Prenza. Sosa, Kenny Weng, Francisca Vazquez, David S. Rickman, Myles Brown, Himisha Beltran. Polycomb dysregulation shapes chromatin bivalency critical for prostate cancer lineage plasticity [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr PR022.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d16b90ea02984495a11ed149a37089c1519a17fc" target='_blank'>
              Abstract PR022: Polycomb dysregulation shapes chromatin bivalency critical for prostate cancer lineage plasticity
              </a>
            </td>
          <td>
            V. B. Venkadakrishnan, Nathaniel C E Voss, Nicole A. Traphagen, Richa Singh, James Neiswender, Keira Prenza Sosa, Kenny Weng, Francisca Vazquez, D. Rickman, Myles Brown, H. Beltran
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Transposable elements (TEs) comprising nearly 50% of the genome are generally silenced by epigenetic mechanisms. Epigenetic anti-cancer drugs can lead to their re-expression, however, the role of TEs in tumorigenesis is unknown. Here, we demonstrate that TEs and their activation of a viral mimicry response plays an important tumor suppressive role in inflammation. We discovered that both patients and mice with colitis express TEs that lead to a viral mimicry response. Interestingly, this response inhibits stemness of cancer-initiating cells. Further activation of viral mimicry by DNA hypomethylation inhibits tumorigenesis. Conversely, knockout of the anti-viral signaling protein MAVS promotes tumorigenesis and reverses the anti-tumor effect of DNA hypomethylation, confirming a tumor suppressive role of viral mimicry. Consistent with this finding, patients with colitis-associated dysplasia show decreased expression of TEs and interferon-related genes. These findings suggest that activation of viral mimicry inhibits stemness and plays a key tumor suppressive role in inflammation. The role of normally silenced transposable elements (TEs) in tumorigenesis remains unclear. Here, the authors show that increased expression of TEs in both patients and mice with colitis or by DNA hypomethylating drugs elicits a viral mimicry response that suppresses tumorigenesis. This viral mimicry response inhibits the stemness of cancer initiating cells in a cell autonomous manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e00113e47a2ed214bc6bbfa7c35e4aa3dc3c8fb9" target='_blank'>
              Viral mimicry acts as a tumor suppressor in colitis
              </a>
            </td>
          <td>
            F. Larsen, Will Jeong, Daniel Schep, H. Good, Alice E. Shin, Liyue Zhang, M. Derouet, Timothy C. Wang, C. Castellani, Parisa Shooshtari, Samuel Asfaha
          </td>
          <td>2026-02-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genomic alterations affecting components of the fibroblast growth factor (FGF) signaling axis can trigger aberrant pathway activation and tumor development. Genomic truncation of the FGF receptor 2 (FGFR2) exon 18 (E18) disrupts the FGFR2 carboxy-terminal tail (C-tail), acting as a potent driver alteration across multiple tumor types. Here, we analyzed human oncogenomic datasets to reveal that E18 truncations are similarly prevalent in FGFR3, an FGFR2 paralog. FGFR3 E18 truncations primarily occurred due to rearrangements (REs) that involve transforming acidic coiled-coil containing protein 3 (TACC3), resulting in FGFR3ΔE18-TACC3 gene fusions. In contrast to E18-truncated FGFR2, functional in vitro and in vivo examination of Fgfr3 variants demonstrated that the truncation of Fgfr3 E18 is insufficient to promote oncogenic activity in cell lines or in the lungs and mammary glands of mice. Only the combination of an Fgfr3 E18 truncation with a RE partner gene that encodes a receptor-dimerizing domain resulted in the development of tumors, which were sensitive to FGFR inhibition. Overall, these findings suggest that patients with cancers that are positive for rearranged FGFR3 resulting in E18 truncation and a fusion to dimerizing partners should be considered for FGFR-targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94a9640747833b06de97dd43802396e30e71e136" target='_blank'>
              C-terminal Truncation and Fusion Partner Determine Oncogenicity of FGFR3.
              </a>
            </td>
          <td>
            Julia Yemelyanenko, J. Bhin, Eline van der Burg, Anne Paulien Drenth, Jessica K. Lee, C. Lutz, Lea Dörner, E. Wientjens, S. Klarenbeek, Ji-Ying Song, Hyeonjin Moon, S. Annunziato, Natalie Proost, Bjørn Siteur, J. S. Ross, M. van de Ven, O. van Tellingen, Shridar Ganesan, L. Wessels, Daniel Zingg, Jos Jonkers
          </td>
          <td>2025-12-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="ERK activity governs diverse cellular responses and has significant implications in cancer biology and treatment. Cellular heterogeneity is a major feature of cancer and a barrier for therapy success, allowing cancer cells to adapt and survive in challenging environments. Here, we used a genetic live-cell reporter to explore the heterogeneity of ERK signaling activity within cellular populations and colonies of glioblastoma (GB) cells. GB cells showed a wide spectrum of ERK activation levels in basal culture conditions and throughout state transitions. Treatment with the chemotherapeutic agent temozolomide increased the phenotypic heterogeneity in ERK activity within cells even in clonal populations. Using the MEK inhibitor trametinib in combination with temozolomide to homogenize ERK activity reduced cell fitness in colonies and decreased fractional killing in GB clonal cells. Our study contributes to the growing understanding of the complexity in ERK activity and dynamics, pointing out the consequences of cell-to-cell ERK phenotypic variability in fitness and therapy survival. The complexity of ERK signaling phenotypes in the context of chemotherapy treatment is shown, offering valuable insights about the intricacies of ERK signaling heterogeneity and chemotherapy treatment. Heterogeneity in ERK activity in live GBM cells is high in basal culture conditions. Temozolomide alters ERK activity in GBM cells and generates phenotypic heterogeneity. Clonal populations behave heterogeneously in ERK activity, and this heterogeneity impacts fitness. Targeting the generation of ERK heterogeneity reduces fitness and fractional killing in GBM colonies. Heterogeneity in ERK activity in live GBM cells is high in basal culture conditions. Temozolomide alters ERK activity in GBM cells and generates phenotypic heterogeneity. Clonal populations behave heterogeneously in ERK activity, and this heterogeneity impacts fitness. Targeting the generation of ERK heterogeneity reduces fitness and fractional killing in GBM colonies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eba7cfb5da57498c2e07248e865b70479b2da77" target='_blank'>
              Temozolomide increases the generation of cell heterogeneity in ERK activity in glioma cells
              </a>
            </td>
          <td>
            K. R. Begnini, J. Marcolin, L. C. Pereira, Letícia Cunha Pereira de Souza, Frederico Kraemer-Mattos, Daphne Tórgo, Carolina Machemer, Georgia da Silva Goulart, Andrew Oliveira Silva, Jephesson A. dos Santos, Guido Lenz
          </td>
          <td>2026-01-29</td>
          <td>Journal of Molecular Medicine (Berlin, Germany)</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Fusion oncoproteins generated by chromosomal translocations are defining molecular drivers of several pediatric cancers, functioning in part by reprogramming gene expression and disrupting cellular differentiation. Ewing sarcoma, an aggressive cancer of bone and soft tissue, exemplifies this paradigm and is driven in the majority of cases by the oncogenic fusion protein EWS::FLI1, formed by joining the low-complexity domain (LCD) of the RNA-binding protein EWS to the DNA-binding domain (DBD) of the transcription factor FLI1. EWS::FLI1 alters transcriptional and RNA processing programs and assembles into aberrant biomolecular condensates, yet the molecular basis for these fusion-driven assemblies remains poorly understood. Here, we integrate NMR spectroscopy, mutagenesis, biophysical and cellular assays, microscopy, and all-atom molecular dynamics simulations to dissect how sequence features and domain interactions shape the condensate properties of EWS and EWS::FLI1. We find that the EWS LCD undergoes extensive self-association mediated by tyrosine-rich motifs that generate compact intramolecular conformations and robust multivalent networks driving phase separation. The RNA-binding domains of EWS engage the LCD through transient electrostatic and aromatic interactions, modulating condensate dynamics, whereas the FLI1 DBD of the fusion protein directly interacts with the EWS LCD via flexible “wings”. Incorporation of the FLI1 DBD markedly alters the material properties of condensates, producing emergent biophysical behaviors distinct from either parental protein. These altered condensate properties possibly contribute to the cellular dysregulation and toxicity associated with EWS::FLI1, in part through mis-localization and aberrant interactions with native EWS that disrupt normal nuclear organization and RNA processing. Together, these findings define a molecular framework in which weak, multivalent, and chemically specific interactions give rise to the emergent properties of the EWS::FLI1 fusion protein. By linking sequence-encoded features, interdomain crosstalk, and condensate material states, we propose a mechanistic description of how fusion with ETS-family DBDs reprograms the physical behavior of EWS assemblies. These emergent condensate properties provide a potential unifying description of how EWS::FLI1 perturbs transcriptional regulation, RNA metabolism, and nuclear organization in pediatric Ewing sarcoma.



 David S. Libich, Erich J. Sohn, Kandarp A. Sojitra, Aiola Stoja, Courtney N. Johnson, Jeetain Mittal, Alex A J. Bishop. Emergent condensate behaviors of the EWS::FLI1 fusion protein in Ewing sarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B005.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6d38099a1e2be11dc3ccd83fb0db5c31397244f" target='_blank'>
              Abstract B005: Emergent condensate behaviors of the EWS::FLI1 fusion protein in Ewing sarcoma
              </a>
            </td>
          <td>
            David S. Libich, Erich J. Sohn, Kandarp A. Sojitra, Aiola P. Stoja, Courtney N. Johnson, J. Mittal, Alex A J. Bishop
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/077d8d4a45150f5dcadd79e304aadcb9aba01396" target='_blank'>
              YAP1 Is a Crucial Nexus in the Tumor Microenvironment.
              </a>
            </td>
          <td>
            Raghavan Narasimhan, Anshula Narayanasamy, Jaya Padmanabhan, Srikumar Chellappan, Durairaj Mohan Kumar
          </td>
          <td>2026-01-13</td>
          <td>Current medical science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Oncogenic mutations of SF3B1 are common in myeloid cancers, chronic lymphocytic leukemia (CLL) and select solid tumors. Their mechanistic basis for promoting oncogenesis has been investigated in detail, with the stereotyped missplicing of mRNA protein coding sequences most intensively studied. These changes, in genes such as MAP3K7, BRD9, and ABCB7, typically lead to loss-of-function, thus contributing to cancer pathogenesis.Here we systematically analyzed the impact of mutant SF3B1 on non-coding regions of mRNA transcripts across disease types, in both cell lines and primary patient specimens. This identified numerous novel and highly reproducible splicing alterations in such regions. Studies of one target gene, DCAF16, revealed multiple complex mutation-induced alterations in its 5' and 3' untranslated regions (5', 3' UTRs). Remarkably, these were mechanistically associated with increased DCAF16 protein levels in SF3B1 mutant cells, representing the first time that oncogenic SF3B1 has been shown to increase levels of a target protein in a gain-of-function manner. DCAF16 is a substrate recognition adapter for the DDB1/CUL4 E3 ubiquitin ligase complex. Novel protein degrader small molecules which co-opt DCAF16 to degrade BRD4 as a neosubstrate demonstrated preferential selectivity for SF3B1 mutant cancers and CLL primary patient specimens due to increased DCAF16 protein levels. In turn, this reveals the therapeutic relevance of mutant SF3B1 dysregulation of transcript untranslated regions and uncovers a novel strategy for the treatment of these important neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc898e0fdc805e68eb2bae19245eee2675983440" target='_blank'>
              Oncogenic SF3B1 mutations alter the splicing of mRNA noncoding regions to induce a novel therapeutic vulnerability.
              </a>
            </td>
          <td>
            Michal Sekrecki, Agata Sekrecka, Rohan Lattupally, Kenny Le, Xiaokang Jin, Chen Mozes, Brendan G. Dwyer, Zhe Zhuang, Bryan A. Romero, J. Pineda, Xunhong Cao, Linh Nguyen, Vicky W Chen, Crystal Zhou, Jared A Wallace, Kailee L Tanaka, C. Tiwari, Austin Gabel, Won Jun Kim, Robert F Stanley, Salima Benbarche, Jaspreet Thind, Abhinaya Muruganandham, T. Y. Zhang, Peter L Greenberg, J. Gotlib, G. Mannis, William Shomali, G. Salmasi, Calvin J Kuo, T. Shanafelt, Irtisha Singh, Daichi Inoue, Finn K. Hansen, N. Gray, Capucine Van Rechem, B. Fakhri, Xiaoyu Zhang, Sydney X. Lu
          </td>
          <td>2026-01-26</td>
          <td>Blood</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fc10f187417e69aa6fbb8d577d4530419eafe0a" target='_blank'>
              The bacterial RNA polymerase-associated CarD protein couples promoter activity to DNA supercoiling.
              </a>
            </td>
          <td>
            D. Forrest, Emily A. Warman, D. Grainger
          </td>
          <td>2026-02-03</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="BRCA1 and BRCA2 are highly conserved tumor suppressor genes, which have dispensable roles in maintaining genomic integrity and whose coordinated functions are mainly owed to DNA repair conducted by homologous recombination (HR) mechanisms. BRCA1/2 loss-of-function mutations cause defective repair of DNA double-strands breaks, destabilization of stalled replication forks, and instability of replication stress responses, leading to increased chromosomal instability and extensive genomic restructuring. BRCA1 deficiency is associated, at the molecular level, with alterations in the DNA end resection program, the checkpoint signaling program, and chromatin remodeling; BRCA2 dysfunction is associated with RAD51 loading and filament stabilization, and changes in the bias of cells toward error-prone repair programs. Besides these canonical repair errors, BRCA1/2 mutations have an effect on transcriptional regulation, chromatin structure and cell-cycle control which further promotes mutational gain and altered cell fate choices. A combination of faulty DNA repair, replication stress, and failure of genome maintenance form a permissive environment for oncogenic transformation. Such interactive molecular pathways can be used to explain the significant predisposition to breast, ovarian, and other solid tumors that are seen in subjects that carry pathological BRCA1/2 mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f5fac4b549ba24ec939a11edbcaa77ad4453e42" target='_blank'>
              The Mechanisms of Tumor Susceptibility Associated with BRCA1/BRCA2 Genetic Mutations
              </a>
            </td>
          <td>
            Zhenan Shi
          </td>
          <td>2026-02-02</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77d8e8d77d073fc56fdafdf9e5ca39c23154e49" target='_blank'>
              A thermodynamic chromatin polymer model characterizes the epigenetic conditions for Hox collinearity
              </a>
            </td>
          <td>
            Yoshifumi Asakura, Yoshihiro Morishita, Kyosuke Adachi, Takayuki Suzuki
          </td>
          <td>2025-12-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Prostate cancer (PCa) is the second leading cause of cancer-related deaths among men in the U.S. Overexpression of PVT1 exon 9 has been shown to transform non-tumorigenic prostate epithelial cells into aggressive tumor-forming cells. We have recently identified a non-canonical promoter within PVT1 that is associated with PVT1 exon 9 expression and hypothesize that epigenetic factors like DNA methylation and transcription factors regulate its overexpression in prostate cancer, given that gene expression is modulated by epigenetic alterations. Using enzymatic methyl-sequencing (EM-Seq), we profiled tumor and adjacent normal prostate tissues. At a differential methylation threshold of ≥±5% and q ≤0.05, we identified 1050 differentially methylated cytosines (DMCs), with 108 hypomethylated and 80 hypermethylated sites in tumor tissues. Regional tiling analysis (1kb windows) also revealed 172 differentially methylated regions (DMRs), including 42 hypomethylated and 16 hypermethylated DMRs in tumor tissue. PVT1 exon 9-associated promoter was visibly hypomethylated in tumors relative to adjacent normal tissues. In vitro, 5-azacytidine (a DNA methyltransferase inhibitor) treatment induced significant upregulation of PVT1 exons 4A and 9 in WPE1-NA22, DU145, PC-3 and MDA-PCa-2b cell lines versus the non-tumorigenic RWPE-1 cell line. Computational transcription factor analysis revealed CDX1 binds to the newly-identified non-canonical promoter, while CRISPR-mediated base editing of its signal sequence reduced PVT1 exon 9 expression in PVT1 exon 9 overexpressing cells. In addition, CDX1 expression was also found to have a positive correlation with high grade PCa. These findings indicate that PVT1 exon 9 overexpression and oncogenicity in PCa is driven by promoter hypomethylation and CDX1 regulation.



 Chinedum C. Udekwu, Siti Khaula. Nurazizah, Seidu Adams, Bonnacci Rachel, E.Oluwabunmi Olapade-Olaopa, Olorunseun O. Ogunwobi. The oncogenic role of PVT1 exon 9 overexpression in prostate Cancer is mediated by DNA methylation and CDX1 [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A065.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9a9e20af1415b799b87aab15045ca553659c6c2" target='_blank'>
              Abstract A065: The oncogenic role of PVT1 exon 9 overexpression in prostate Cancer is mediated by DNA methylation and CDX1
              </a>
            </td>
          <td>
            Chinedum C. Udekwu, Siti Khaula. Nurazizah, Seidu Adams, Bonnacci Rachel, E. Oluwabunmi Olapade-Olaopa, O. Ogunwobi
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) is characterized by profound genomic instability and spatial heterogeneity. Liquid biopsy, utilizing circulating tumor DNA (ctDNA), offers a non-invasive approach to capture the comprehensive mutational landscape of the disease. This pilot study evaluated the concordance of genomic alterations between cell-free DNA (cfDNA) and matched tumor tissue in patients with HGSOC. Twelve patients with HGSOC undergoing primary cytoreductive surgery were enrolled. Using the Macrogen® Axen™ Cancer Panel 2 with unique molecular identifier (UMI) technology for error suppression, we achieved a theoretical limit of detection of ~0.36% VAF. The mean cfDNA concentration was 107.3 ng/mL, showing a significant positive correlation with FIGO stage (p = 0.016). While the sensitivity of cfDNA to detect tissue-confirmed mutations was 57.6%, the overall gene-level concordance was 95.3%, largely driven by negative agreement in wild-type genes. Liquid biopsy revealed a significantly broader mutational spectrum (mean 9.67 alterations/patient) compared to tissue (5.50/patient). Crucially, concordant mutations exhibited high variant allele frequencies (VAFs) (mean 41.4%), whereas plasma-unique discordant mutations showed significantly lower VAFs (mean 7.31%, p < 0.001). These preliminary findings suggest that while tissue biopsy likely reflects the dominant clonal population, liquid biopsy may serve as a potential molecular mirror, capturing subclonal variants from spatially distinct metastatic sites and hypoxic niches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fbd4fe1db0bd75682837f32c9c2cfec2b1ed6cb" target='_blank'>
              Concordance of Genomic Alterations in Ovarian Cancer Tissues and Circulating-Tumor DNA: A Pilot Study
              </a>
            </td>
          <td>
            Bowon Kang, Seongmin Kim, Sanghoon Lee, Jae-Yun Song
          </td>
          <td>2026-01-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="RNA editing is a critical regulatory mechanism that acts after transcription. Two types of RNA editing are A-to-I editing and C-to-U editing. The first one uses adenosine deaminase acting on RNA, while the second uses APOBEC complexes. RNA editing has an important role in cancer development in controls tumorigenesis and tumor progression through multiple mechanism. It allows for precise edits to RNA sites linked to cancer at a molecular level. At the microenvironmental level, it regulates the immune response by remodeling the immune landscape. RNA editing has dual roles in tumor progression and anti-tumor effects. The ability of some non-coding RNAs to perform the reverse function is shown to be useful in various important biological processes. The important processes include drug resistance, agitation, tumor suppressor gene expression, tumor microenvironment, metabolic regulation, and cancer cell migration. Both the over and under editing of RNA can cause tumors. Therefore, initiating and stopping editing needs to be strictly controlled for clinical purposes. Studies indicate RNA editing has a unique therapeutic potential by downregulating oncogenes, restoring mutated genes, and enhancing the efficacies of immunotherapy. Problems remain; however, with regard to targeting precision, safety, functional characterization, and the regulation of editing efficiency and extent. The ongoing evolution of specificity and tools may ensure that RNA editing emerges as a vital strategy in cancer therapeutics. Nevertheless, progress is hindered by enduring hurdles, such as off-target effects and variable editing efficiency, and has yet to be validated in larger and multi-site clinical trials. In the same vein, RNA editing may serve as useful references for surgical decisions. For example, RNA editing can be used to identify surgery risk, recurrence risk, and surgical approach selection due to its exact targeting capacity. It also helps intraoperative resection margins and adjuvant therapy to properly maintain the outcome postoperatively in the long term. Hence, surgical oncology researchers must explore RNA editing in-depth. It is worth noting that limited antitumor activity was observed in solid tumors in preclinical studies due to insufficient delivery of editing tools, indicating realistic translation expectation is needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a2398f8e95c73580d87fcee9bf8e56692501fff" target='_blank'>
              RNA editing: an emerging frontier in cancer therapy - explorations, opportunities, and challenges.
              </a>
            </td>
          <td>
            Jun Chen, Qi Sun, Na Gong, Zengkan Du, Zhijie Zhao, Dongyang Ding, Haoling Zhang, Q. Yan, Shengxian Yuan, Shuya Jiang
          </td>
          <td>2026-01-12</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The present study focused on the role of coiled‑coil domain‑containing protein 137 (CCDC137), an RNA‑binding and epigenetic protein with high expression and poor prognosis in various types of cancer. Bioinformatics analysis, cellular experiments and animal models investigated the functions of CCDC137 in RNA post‑transcriptional regulation, epigenetic modification and the tumor microenvironment. The results showed that CCDC137 can bind specific mRNAs to affect physiological processes such as the cell cycle, participate in epigenetic regulation such as DNA methylation and histone modification and influence tumor immune escape by affecting the functions of tumor‑associated macrophages. Based on the multidimensional regulatory functions of CCDC137, it is expected to become a new target for cancer diagnosis and treatment and provide a theoretical basis for precision cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e296b6b77c5bade8d6a248d8b774b6b213bd0e20" target='_blank'>
              CCDC137: A key hub for RNA and epigenetic regulation in cancer research (Review).
              </a>
            </td>
          <td>
            Xiaoying Zhao, Wenjing Zhang, Xiang Yu, Xiaohan Zhao, Xu Dai, Sirui Chen, Yan Wang, Jianghua Cheng, Weiwei Zheng
          </td>
          <td>2025-12-16</td>
          <td>International journal of oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db9ee68261862574cb9b7d1551d82bf740d99cfd" target='_blank'>
              Elevated mutation in haploid yeast driven by translesion synthesis
              </a>
            </td>
          <td>
            Jacob Fredette-Roman, Denise R. Smith, Sanad B. Omari, Nathaniel P. Sharp
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92529c21b1d903977db7b6f99cc016a10cc689ad" target='_blank'>
              Molecular analysis of tumor recurrence using established cancer stem cell-line and drug discovery.
              </a>
            </td>
          <td>
            Kiyotaka Nakano, E. Oki, Masaki Yamazaki, Masami Suzuki, S. Kawai, Y. Zaitsu, Chiyoko Nishime, K. Ando, G. Nagae, N. Harimoto, Mitsuhiko Ota, T. Kawazoe, Kentaro Nonaka, Keita Natsugoe, Sachie Omori, H. Saeki, Hiroyuki Aburatani, Tatsumi Yamazaki, Yoshihiko Maehara
          </td>
          <td>2026-01-20</td>
          <td>International journal of clinical oncology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Sorex araneus, the Eurasian common shrew, has seasonal brain size plasticity (Dehnel's phenomenon) and abundant intraspecific chromosomal rearrangements. Genomic contributions to these traits, however, remain unknown. We couple a chromosome-scale genome assembly with seasonal brain transcriptomes to discover relationships between molecular changes and both traits. While positively selected genes enriched the Fanconi anemia DNA repair pathway (FANCI, FAAP100), which is likely involved in chromosomal rearrangements by preventing the accumulation of chromosomal aberrations, genes under positive selection or showing seasonal differential expression in the brain implicate neurogenesis (PCDHA6, SOX9, Notch signaling) and metabolic regulation (VEGFA, SPHK2) as key mechanisms underlying Dehnel's phenomenon. We also find that both positively selected and differentially expressed genes in the hippocampus are overrepresented near Sor. araneus evolutionary breakpoints. This relates both positive selection and differential expression to accessible chromatin configuration, suggesting that chromosomal rearrangements are integral to adaptive evolution and the regulation of brain size plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60e5d59bc3d794d733f5961db0a47844813d84f1" target='_blank'>
              Genomic comparisons shed light on the adaptive basis of brain size plasticity and chromosomal instability in the Eurasian common shrew.
              </a>
            </td>
          <td>
            William R. Thomas, Tanya M. Lama, Cecilia Baldoni, Laia Marín-Gual, Diana D Moreno Santillán, Marta Farré, Linelle Abueg, Jennifer R. Balacco, O. Fedrigo, G. Formenti, Nivesh Jain, J. Mountcastle, Tatiana Tilley, Ying Sims, Alan Tracey, Jonathan M. D. Wood, David A. Ray, Dominik von Elverfeldt, J. Nieland, Angelique P. Corthals, A. Ruiz-Herrera, D. Dechmann, Erich D. Jarvis, Liliana M. Dávalos
          </td>
          <td>2026-01-09</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="ANLN is a highly conserved actin-binding protein that plays a critical role in cytokinesis by interacting with key cytoskeletal components such as actin, myosin, and RhoA. Increasing evidence shows that ANLN is aberrantly overexpressed in various cancers, including lung, breast, and liver cancers, and that its elevated expression is associated with enhanced tumor cell proliferation, migration, and invasion. Because of its central involvement in tumorigenesis and disease progression, ANLN has emerged as a promising prognostic biomarker and potential therapeutic target. Recent studies have demonstrated that ANLN contributes to resistance to chemotherapy, targeted therapy, and immunotherapy through multiple molecular mechanisms. This review provides a systematic overview of the physiological functions of ANLN, its roles in cancer initiation and progression, and its regulatory mechanisms in treatment resistance, offering biological insights into precision oncology and potential strategies for overcoming therapeutic resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27a40af0d42ab2b0397c3e4bc1fc5ecc54b07af4" target='_blank'>
              The role of ANLN in malignant tumors: pathogenesis, treatment resistance and targeted strategies
              </a>
            </td>
          <td>
            Yixuan Wang, Yang Xiao, Jielin Yu, YuanHua Zou, Zhe Wang, Xianghong Yang
          </td>
          <td>2026-01-07</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Recapitulation of the nuclear auxin response pathway in yeast led to the hypothesis that corepressors prime loci for rapid bursts of transcription. Specifically, the TPL corepressor, which inhibits transcription in the absence of auxin, interfaces with Mediator and core components of the transcription pre‐initiation complex (PIC). Once auxin is perceived, RNA Pol‐II is likely to essentially displace TPL, allowing for rapid transcription initiation (and likely re‐firing). Here, we present the current evidence supporting the role of TPL‐type corepressors in multiple levels of transcriptional control, including modulation of regulatory transcription factor activity, interactions between cis‐elements and the core promoter, and in organizing multiple loci within the nucleoplasm. We also highlight critical areas for future investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d47b25bb88b986213fcf383cdae22d916c6e97b4" target='_blank'>
              Primed to Burst: Corepressors Coordinate Transcriptional Activation and Efficient Switching Between Cell States
              </a>
            </td>
          <td>
            Alexander R Leydon, J. Nemhauser
          </td>
          <td>2025-12-15</td>
          <td>Bioessays</td>
          <td>1</td>
          <td>44</td>
        </tr>

        <tr id="Primary hepatic angiosarcoma is a very rare and extremely aggressive malignancy that has hadfew therapeutic options and has been closely linked to aristolochic acid which is a strongenvironmental carcinogen commonly found in some herbal medicines. The hallmark mutationalsignature of aristolochic acid is the observed A: T↔ T: A transversion at context-specifictrinucleotide motifs, and this pattern can be used to discriminatively identify AA-induced tumorswith genome-wide analyses. This signature can act as a molecular signature that has the potentialto explain the etiology of cancers, can aid retrospective exposure assessment, as well ascontribute to early diagnosis. Coincidently, the molecular profiling has highlighted newbiomarkers, serine/threonine kinase p53 pathways, aberrations of ERCC1 DNA repair enzyme,and up-regulation of endothelial cell markers. which have collectively enhanced diagnosis andstratification outcomes. These markers have been tested as potential predictive targets andprovide the basis of precision approaches that encompass PARP inhibitors that target DNA-repairdefects, targeted kinase inhibitors, and monoclonal antibodies against angiogenic pathways.Combination therapies that include the targeted with conventional chemotherapy have beenshown to be more effective. Even more, the development of innovative technologies, includingliquid biopsy, single-cell sequencing, and artificial intelligence-mediated searches in mutationalsignatures, further expand personalized medicine of Primary hepatic angiosarcoma and allow anearlier manifestation and precision intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b9f27b6bcd4ee7287251279079994596ebf2bc" target='_blank'>
              Uncovering Aristolochic Acid–Driven Mutational Signatures and Novel Biomarkers inPrimary Hepatic Angiosarcoma for Targeted Therapeutic Development
              </a>
            </td>
          <td>
            Ramzan Mohammad, Jasvir Kaur, Ruhit Ashraf, Shafkat Hussain Malik, Pursuing B. Pharmacy
          </td>
          <td>2026-01-10</td>
          <td>Pan-African Journal of Health and Psychological Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Ovarian cancers with similar histopathologic but diverse immunogenomic profiles may require different treatment regimens to achieve the best clinical outcomes. The development of optimal treatment regimens will thus require an understanding of the main ways in which ovarian cancers differ with respect to their immunogenomic profiles. We used a multimodal latent variable model to enable an integrated analysis of somatic mutation, mRNA expression, and multiplex immunofluorescence data to uncover the principal drivers of intertumor heterogeneity across 197 patients with ovarian cancer. We found that the majority of the immunogenomic intertumor heterogeneity was driven by gene expression and immune infiltration in the tumor core and invasive margin. Moreover, much of this heterogeneity could not be explained by histologic subtype; somatic mutation patterns explained much of the difference between high-grade serous and other subtypes. Clustering of samples according to their positions in the latent space revealed a distinct subgroup of endometrioid ovarian cancer tumors, characterized by increased CST2 expression and improved prognosis, among other immunogenomic features. We identified a group of collagen-related genes, for which expression in all subtypes was inversely correlated with the density of proliferating tumor cells in the tumor core and associated with increased levels of inflammatory fibroblasts, independent of tumor purity. In summary, our findings advocate for a paradigm shift in how we classify and approach the treatment of ovarian cancer. By moving beyond the constraints of histologic subtypes and delving into the molecular intricacies of patient-specific tumor microenvironments, we unveil a new opportunity for targeted immuno-oncological treatment of a subset of patients with ovarian cancer. Significance: Our study employs a multimodal latent variable model across 197 patients to identify principal drivers of immunogenomic intertumor heterogeneity. We uncover a distinct subgroup of endometrioid ovarian cancer with a unique immunogenomic signature and better prognosis and a set of collagen-related genes associated with the tumor microenvironment. This work challenges the adequacy of existing histologic classifications in capturing the molecular diversity of ovarian cancer, potentially informing more personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc2a2b95c1893d86c4e80e7a4dcaee2ec82bbf3" target='_blank'>
              Characterization of the Immunogenomic Landscape of Ovarian Cancer Uncovers a Distinct Subset of Endometroid Tumors Associated with High CST2 Expression and a Favorable Prognosis
              </a>
            </td>
          <td>
            James Boyle, Jan Zaucha, Felicia Ng, A. Spitzmüller, Miljenka Vuko, F. Segerer, Monica Azqueta-Gavaldon, M. Testori, Amelia Raymond, Barrett Nuttall, A. Storti, Sakshi Gulati, Shaan Gill, Michael Surace, Paola Marco-Casanova, Emma V Jones, Ben S Sidders, J. Reis-Filho, M. Scaltriti, Anysia Semertzidou, Helen K Angell
          </td>
          <td>2025-12-18</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Down syndrome (DS; trisomy 21) confers a ∼100-fold increased risk of Hirschsprung disease (HSCR), yet the causal contributions of specific chromosome 21 genes remain unresolved. Here we show that increased dosage of SOD1 alone is sufficient to perturb enteric nervous system (ENS) development. We engineered a humanized SOD1 trisomic mouse line by inserting a 28 kb human SOD1 locus into ROSA26 and genetically profiled the distal colon at postnatal day 0 using single-cell RNA-seq and immunofluorescence. Sod1/SOD1 was elevated ∼1.5X in the ENS but varied by cell type, with transcriptionally active progenitors showing the greatest increase. Cell composition shifted toward transcriptionally active cells and glia, with concomitant loss of excitatory and inhibitory motor neurons and interneurons. Genetically, Sod1/SOD1 trisomy downregulated synaptic and neuronal communication programs but upregulated DNA replication/cell-cycle and genome maintenance pathways, especially within glia. Consequently, key HSCR genes were dysregulated: Ret, Ednrb, and Sema3c were decreased, while Sema3a ,a negative guidance cue, was increased. Ret was selectively reduced in inhibitory and excitatory motor neurons and progenitors, unchanged in glia, and reduced at the protein level in vivo. Within glia, Sod1/SOD1 was particularly elevated in proliferating/active glia with a glia-specific bias toward endogenous mouse Sod1 expression. Taken together, these data support a dual mechanism whereby increased Sod1/SOD1 dosage suppresses RET-dependent neurogenesis while independently promoting reactive/proliferative glial states. Thus, SOD1 is sufficient to alter ENS development significantly and provide the susceptibility substrate for HSCR with further reductions in RET gene expression leading to aganglionosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd42270480a6f6019c599fcabcddd6cbc7151813" target='_blank'>
              Sod1 trisomy causes ENS developmental defects and susceptibility to Hirschsprung disease via neuronal Ret suppression and glial remodeling
              </a>
            </td>
          <td>
            Gabriel Grullon, Jarod A. Rollins, Lauren Wilkes, A. Zuberi, A. Chakravarti, S. Chatterjee
          </td>
          <td>2026-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the fourth leading cause of cancer-related death worldwide and shows significant heterogeneity. Cadherin 13 (CDH13) is abnormally expressed in various malignancies, but its role in HCC and the tumor microenvironment remains unclear. This study aimed to systematically explore the expression, regulatory mechanisms, and potential functions of CDH13 in HCC using a multi-omics strategy. We integrated transcriptomics, proteomics, spatial transcriptomics, single-cell RNA sequencing (scRNA-seq), immunohistochemistry (IHC), clustered regularly interspaced short palindromic repeats/CRISPR-associated protein 9 (CRISPR/Cas9) gene editing, chromatin immunoprecipitation sequencing (ChIP-seq), weighted gene co-expression network analysis (WGCNA), gene set variation analysis (GSVA), immune infiltration analysis, molecular docking, phenotypic analysis, and survival analysis. In multiple cohorts, CDH13 mRNA was significantly upregulated in 5145 HCC samples (standardized mean difference (SMD) = 1.17) and associated with poor prognosis (hazard ratio (HR) = 2.06). Protein levels were elevated in 165 public and 476 clinical paired samples (361 from Guangxi Medical University, 115 from Yulin Red Cross Hospital; SMD = 2.48), mainly localized to the cytoplasm and membrane. Spatial transcriptomics showed CDH13 enrichment in tumor regions and spatial association with endothelial cells, which was confirmed in an independent HCC tissue section. Focusing on cell function, CRISPR/Cas9 knockout of CDH13 significantly inhibited proliferation in HCC cell lines such as SNU182, SNU739, HUH7, etc. Further analysis showed CDH13 was expressed in hepatocytes, endothelial cells, and hepatitis B virus (HBV)-infected regions; pseudotime analysis indicated late-stage upregulation in endothelial cells and activation of the macrophage migration inhibitory factor (MIF) pathway. IHC confirmed moderate CDH13 positivity in tumor cells and strong positivity in microvascular endothelial cells. CDH13-high endothelial cells were enriched in pyrimidine metabolism, oxidative phosphorylation, and propanoate metabolism. WGCNA and GSVA linked CDH13-related genes to angiogenesis, invasion, and metabolic reprogramming. ChIP-seq identified binding of Forkhead box A1 (FOXA1) to CDH13 regulatory regions. Clinically, CDH13 negatively correlated with regulatory T cells (Tregs, R = −0.311) and γδ T cells (R = −0.206), and positively with stromal scores (R = 0.400). High CDH13 predicted poor immunotherapy response (Area Under the Curve (AUC) = 0.745–0.795) and prognosis. Molecular docking shows a binding with docetaxel (−7.6 kcal/mol), while phenotypic analysis showed no obvious physiological toxicity. CDH13 is highly expressed in HCC and may promote angiogenesis, immune evasion, and metabolic reprogramming via the FOXA1-CDH13 axis, suggesting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e26bc66a34e745a67c0425c16be84d04301c9194" target='_blank'>
              Multimodal analysis reveals potential association of CDH13 with endothelial cells and its overexpression in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Ke-Jun Wu, Li-Hua Yang, Dong-Ming Li, Rong-Quan He, Di-Yuan Qin, Shi-De Li, Jian-Di Li, Y. Dang, Ming-Jie Li, Qi Li, Jian-jun Li, Lu Zhang, Han He, Ji-Feng He, Gang Chen, Xiao-Bo Jiang
          </td>
          <td>2025-12-12</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fef918e1d4b64862104d2dc24b6aa7569489a94" target='_blank'>
              Genomic profiling implicates candidate genes and mutagenic pathways driving lung cancer recurrence
              </a>
            </td>
          <td>
            L. Luhari, A. Valter, A. Bahcheli, K. C. Cheng, M. Bayati, A. Ustav, A. Velthut-Meikas, K. Oselin, J. Reimand
          </td>
          <td>2026-01-01</td>
          <td>None</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Malaria remains one of the devastating illnesses, and drug-resistant malaria has incurred enormous societal costs. A few host kinases are vital for the liver stage malaria and might be promising drug targets against drug-resistant malaria. STK35L1 is one of the host kinases that is highly upregulated during the liver stage of malaria, and the knockdown of STK35L1 significantly suppresses Plasmodium sporozoite infection. In this study, we retrieved the promoter region of STK35L1 based on 5′ complete transcripts, transcription start sites, and cap analysis of gene expression tags. Furthermore, we identify transcriptionally active regions by analyzing CpG islands, histone acetylation (H3K27ac), and histone methylation (H3K4me3). It suggests that the identified promoter region is active and has cis-regulatory elements and enhancer regions. We identified various putative transcription factors (TFs) from the various high-throughput ChIP data that might bind to the promoter region of STK35L1. These TFs were differentially regulated during the infection of Plasmodium sporozoites in HepG2 cells. Our molecular modeling study suggests that, except for SMAD3, the identified TFs may be directly bound to the promoter. Together, the data suggest that these TFs may play a role in sporozoite infection and in regulating STK35L1 expression during the liver stage of malaria.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5083d80d5c64f4a36358495eb61056154d577e3" target='_blank'>
              Differential Expression of STK35L1-Associated Transcription Factors in Plasmodium Infection During the Liver Stage of Malaria
              </a>
            </td>
          <td>
            Arpana Yadav, P. Sharma, Mayuree Hazarika, Pragya Gehlot, Saloni Bage, Mahesh Saini, Kritika Gaur, Acham Parambath Aswathi, Malti Thakur, Devesh M. Sawant, A. Singh, D. Brünnert, Pankaj Goyal
          </td>
          <td>2025-12-12</td>
          <td>Kinases and Phosphatases</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Introduction Immunotherapy has shown positive response in many patients with microsatellite instable (MSI-H) tumors, but its effectiveness in microsatellite stable (MSS) tumors remains limited. We hypothesize that tumor mutational burden (TMB) can help identify a biologically distinct subset of MSS tumors that can benefit from immunotherapy. Methods We analyzed the molecular characteristics, including mutational landscape, mutational signatures, immune cell profiles and neoantigen load of MSS tumors with high TMB using data from colorectal cancer datasets (TCGA-COAD and TCGA-READ). After that, we extended these findings across other cancer types with MSI classification, further supporting the potential of using TMB as a biomarker for predicting immunotherapy response in MSS tumors. Results Our results show that MSS tumors with TMB greater than 50 mutations per megabase have POLE gene mutations, which lead to hypermutation. These hypermutated tumors show immune cell signatures that are more similar to MSI-H tumors, rather than MSS tumors with low TMB. We also found that MSS tumors with high TMB have a substantially higher number of neoantigens compared to low-TMB MSS tumors, suggesting they may respond better to immunotherapy, including a high proportion of predicted high-affinity neoantigens. Discussion These findings support the clinical relevance of TMB as a biomarker for neoantigen prediction and immunotherapy-relevant features in MSS tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/395cb7d68e2b00553e27d494f32391f3a74717e5" target='_blank'>
              Tumor mutational burden predicts neoantigen profiles and immunotherapy response in microsatellite stable tumors across different cancer types
              </a>
            </td>
          <td>
            Olesia Kondrateva, Tugce Bilgin Sonay, Inti Zlobec, Maria Anisimova
          </td>
          <td>2026-01-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Kar4 is a putative transcription factor required for efficient mating in the budding yeast Saccharomyces cerevisiae. Kar4 functions with Ste12, the master transcriptional regulator of the yeast mating pheromone response, to promote the transcription of a subset of Ste12 targets required for mating. However, the mechanism by which Kar4 modulates Ste12 activity has remained uncertain. Here, we examined Kar4’s function at the levels of transcription (RNA-seq) and Ste12 DNA binding (ChIP-exo). We show that Kar4 promotes Ste12 binding to nearly all Ste12 DNA-binding sites associated with transcriptionally-upregulated genes, even if their upregulation is not dependent on Kar4. We further found that the majority of Ste12-binding sites have two pheromone response elements (PREs) separated by four nucleotides in a head-to-tail orientation (H-T 4 motif). Sites associated with Kar4-dependent transcription have PREs with more mismatches and substantially lower Ste12 occupancy in kar4Δ cells than sites linked to Kar4-independent transcription. During the pheromone response in kar4Δ cells, Ste12 exhibits increased binding to non-H-T 4 motifs, particularly T-T 3 motifs, resulting in the abnormal upregulation of many genes not transcribed during the wild-type pheromone response. Therefore, we propose that Kar4 functions by increasing the DNA-binding specificity of Ste12 globally, promoting its binding primarily to H-T 4 motifs. Our model is consistent with previously-observed slower induction kinetics of Kar4-dependent genes via a feed-forward mechanism. Lastly, we uncovered several novel aspects of the pheromone response, including a broad role for the Crz1 transcription factor, induction of stress responses, and the identification of pheromone-responsive intergenic transcripts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a0182b1d0b691ffdab0d53110f09b61359e4d1" target='_blank'>
              Kar4 acts as a Ste12 regulator in Saccharomyces cerevisiae, promoting Ste12 binding to a specific DNA motif genome-wide
              </a>
            </td>
          <td>
            Jason V. Rogers, Amanda Yeo, Val Meleshkevich, Hernan Lorenzi, Mark D. Rose, O. Cohen-Fix
          </td>
          <td>2025-12-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The BK polyomavirus (BKPyV), a ubiquitous human pathogen, has garnered significant attention for its potential oncogenic role, particularly in immunocompromised populations such as transplant recipients. This review synthesizes current evidence on the molecular mechanisms underlying BKPyV-associated carcinogenesis, with a focus on the viral Large T antigen (LT-Ag). LT-Ag drives oncogenic transformation primarily by inactivating tumor suppressor proteins p53 and pRb, disrupting cell cycle regulation, and inhibiting apoptosis. Additionally, it dysregulates critical signaling pathways, including Wnt/β-catenin, PI3K/Akt/mTOR, Ras/Raf/MAPK, and STAT-3, which collectively promote uncontrolled proliferation, survival, and genomic instability. A hallmark of BKPyV's oncogenicity is its capacity to integrate into the host genome, often near tumor suppressor loci, further amplifying chromosomal damage. Clinically, BKPyV has been implicated in urothelial carcinoma, renal cell carcinoma, and prostate cancer, with viral DNA and LT-Ag detected in 20-50% of tumor specimens from kidney transplant recipients (KTRs). However, epidemiological data remain contentious, as studies in immunocompetent cohorts report inconsistent associations, and viral presence in non-neoplastic tissues complicates causal interpretations. This duality BKPyV as both a latent commensal and a potential oncogenic driver underscores the need for nuanced investigation into host-virus interactions, particularly in the context of immunosuppression. Emerging diagnostic approaches, such as next-generation sequencing to map viral integration sites, and therapeutic strategies targeting LT-Ag-mediated pathways hold promise for early detection and intervention. By bridging molecular insights with clinical observations, this review advocates for longitudinal studies to clarify BKPyV's role in carcinogenesis and to refine management protocols for high-risk populations, ultimately mitigating the burden of virus-associated malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ccf36c56bb73223ea7f615349a48e3ade542e5" target='_blank'>
              Cancer Inducing Role of BK Polyomavirus Large T Antigen: Molecular Signaling View.
              </a>
            </td>
          <td>
            Maryam Rahimi Foroudi, Ramin Yaghobi, A. Afshari, J. Roozbeh
          </td>
          <td>2025-12-26</td>
          <td>Cancer investigation</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="A large number of reports have suggested that the cancer is associated with the gene mutations and chromosomal aberrations, and thus it has been a focus of research. Today new insights are proposed by analyzing relevant literature, suggesting that the cancer is not caused by the gene mutations and chromosomal aberrations, but by the regulatory dysfunction of gene expression resulted from the nuclear damage, among which, the gene mutations and chromosomal aberrations are only an accompany phenomena also resulting from the nuclear damage. The gene mutations generally do not cause diseases, but if they do, they mainly cause the single-gene diseases. The chromosomal aberrations usually refer to the abnormalities in chromosome structure and number, often leading to cell death or inducing various chromosomal diseases. Some chronic refractory diseases, such as the cancer, Alzheimer's disease, degenerative diseases, are probably not caused by the gene mutations and chromosomal abnormalities, but rather by the abnormal regulation of gene expression and the dysfunction of DNA transcription-protein synthesis resulted from the nuclear damage. These diseases are all the diseases of nuclear dysfunction and collapse of differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4826bbb55a85cfdd2750ad0771ebe3b7d95aef3d" target='_blank'>
              Differences and Pathogenicity Between the Gene Mutation and Chromosomal Aberration
              </a>
            </td>
          <td>
            Maojin Li
          </td>
          <td>2025-12-29</td>
          <td>Medical Research Frontiers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a malignant tumor that occurs in the mucosal epithelial cells of the colon or rectum, with high mortality and morbidity. Therefore, the study of its molecular principle is more important. In recent years, more attention has been paid to the regulation of CRC by epigenetic factors, especially the regulation of non-coding RNAs (ncRNAs) and RNA modifications. ncRNAs include miRNA, circRNA, and lncRNA. Among them, miRNAs regulate MAPKs, Wnt, and TGF-β signaling pathways and are involved in the tumor process. CircRNAs, as regulators of gene expression, can participate in various biological processes. lncRNA interacts with DNA, RNA, and proteins to regulate signaling pathways in various ways, potentially acting as biomarkers in non-invasive examinations. RNA modifications, especially N6-methyladenosine (m6A), affect CRC development. m6A consists of three proteins, writers, erasers, and readers, which maintain the stability of RNA modifications. Different levels of expression of these three proteins in CRC can lead to changes in the tumor microenvironment, gene expression, and other factors, and thus play an important role in CRC. ncRNAs and RNA modifications comprise the CRC epigenetic regulatory network, providing a new direction for the diagnosis and treatment of CRC in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11d5caa5bf293c7752dde20eabcef8521208dcce" target='_blank'>
              Center Molecular Mechanisms of Non-coding RNAs and RNA Modifications in Colorectal Cancer
              </a>
            </td>
          <td>
            Tongyue Yu
          </td>
          <td>2025-12-19</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="PURPOSE
In renal cell carcinoma (RCC), loss of the NF2 tumor suppressor gene encoding the merlin protein is associated with aggressive clinical behavior. However, data regarding the clinical course and additional molecular features in the context of contemporary systemic therapies remains limited.


METHODS
Clinical outcomes were evaluated in patients with RCC exhibiting merlin loss by immunohistochemistry in one academic cohort. Integrative genomic analyses were performed, including targeted DNA sequencing via the institutional OncoPanel assay and RNA sequencing data from The Cancer Genome Atlas (TCGA).


RESULTS
In the institutional cohort (n=33), most patients had biphasic hyalinizing psammomatous RCC (66.6%) and metastatic disease (78.8%). Among 23 patients receiving systemic therapy, those treated with non-immunotherapy (IO)-based regimens (n=5) had numerically longer overall survival (24.5 vs 16.5 months, p=0.2) and progression-free survival (9.8 vs 5 months, p=0.5) compared to IO-based therapies (n=18), though differences were not statistically significant. Genomic analysis (n=17) revealed frequent truncating mutations of NF2 (82.3%) and recurrent alterations in chromatin remodeling and DNA damage-response signaling genes with prominent deletions in tumor suppressor genes such as CDKN2A/B. Transcriptomic profiling of NF2-inactivated tumors from TCGA (n=13) demonstrated enrichment of cellular proliferation and concurrent suppression of metabolic and immune pathways, suggesting an aggressive phenotype compared to clear cell RCC without NF2 inactivation (n=529).


CONCLUSIONS
Biallelic NF2 inactivation and merlin protein deficiency drives an aggressive RCC phenotype marked by immune dysfunction, high proliferation, and frequent cell cycle and DNA damage-response signaling alterations. Limited treatment response to immunotherapy highlights the need for molecularly tailored therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ed3d6f6e41d2538d4480bbe119a1ba105f4f83a" target='_blank'>
              Integrative Clinical and Molecular Evaluation of Renal Cell Carcinoma with Merlin Protein Deficiency and Biallelic Loss of NF2.
              </a>
            </td>
          <td>
            Emre Yekedüz, Weiwei Bian, Stephanie E. Siegmund, M. Machaalani, Jad El Masri, Mustafa Saleh, E. Saad, Liliana Ascione, Razane El Hajj Chehade, Clara Steiner, P. Barrios, S. Berg, B. McGregor, C. Mantia, P. Ravi, Wenxin Xu, Michelle S. Hirsch, T. Choueiri, M. Serzan
          </td>
          <td>2026-01-12</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be59cd38ce72b5622d35251bdca260d68eabf2d9" target='_blank'>
              pCancer and FOXK2 (Forkhead Box K2): Oncogenic and Tumor-Suppressive Roles of FOXK2 in Cancer.
              </a>
            </td>
          <td>
            Şeyda Akin, İbrahim Ozturk, C. Hepokur
          </td>
          <td>2026-02-03</td>
          <td>Biochemical genetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="BACKGROUND
Approximately 10% of cancers achieve replicative immortality through a telomerase-independent mechanism of telomere maintenance, termed Alternative Lengthening of Telomeres (ALT). ALT is particularly prevalent in certain subtypes of malignant gliomas, such as IDH-mutant astrocytoma and pediatric glioblastoma, and frequently co-occurs with ATRX inactivating mutations. Although ALT is an adaptive mechanism through which cancer cells achieve proliferative immortality, the elevated levels of replication stress observed in ALT tumors constitute a potential therapeutic vulnerability.


METHODS
Leveraging CRISPR/Cas9 screening data from the Cancer Dependency Mapping Project, coupled with patient-derived cell lines and xenografts, we identified SMARCAL1 as a novel synthetic lethal vulnerability in ATRX-deficient glioma models that engage ALT. Using complementary molecular assays for DNA damage, telomere maintenance, and telomeric replication stress, we define the mechanisms underlying cytotoxicity induced by SMARCAL1 depletion in ALT-positive glioma cells.


RESULTS
Our data demonstrate the annealing helicase SMARCAL1 is a highly specific synthetical lethal vulnerability in cancers that use ALT. SMARCAL1 localizes to ALT-associated PML bodies in ALT-positive glioma cell lines, including IDH-mutant astrocytomas. SMARCAL1 depletion, via doxycycline-induced RNAi, led to a hyperactivation of the ALT phenotype, high levels of DNA double-strand breaks in G2 phase, and cell death via mitotic catastrophe. In mice bearing intracranial xenografts derived from high-grade IDH-mutant astrocytoma, inducible SMARCAL1 depletion prolonged animal survival.


CONCLUSIONS
Our findings demonstrate that the molecular processes orchestrating ALT-mediated telomere maintenance constitute a targetable synthetic lethal vulnerability that can be exploited by SMARCAL1 inhibition, thus supporting the future development of small molecule inhibitors of SMARCAL1 as anti-cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c97a1417417751706742470fa54409032d4aaa86" target='_blank'>
              SMARCAL1 is a targetable synthetic lethal therapeutic vulnerability in ATRX-deficient gliomas that use Alternative Lengthening of Telomeres.
              </a>
            </td>
          <td>
            Alexandrea Brown, Laura Strickland, Elise N Erman, Christopher J. Pirozzi, Justin T. Low, B. Diplas, Emiley A Gibson, Mariah Shobande, Taher Khambati, Marharyta Krylova, Heng Liu, R. McLendon, Zachary J. Reitman, S. Keir, Lee Zou, David M. Ashley, Matthew S. Waitkus
          </td>
          <td>2026-01-10</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="


 Fusion-positive rhabdomyosarcoma is an aggressive pediatric cancer caused by a chromosomal translocation. The most common translocations occur between PAX3 or PAX7 and FOXO1, which generates a chimeric transcription factor with gain-of-function activities. These fusion proteins contain the PAX3/7 DNA-binding domains (paired and homeobox) and the FOXO1 transactivation domain. For over thirty years, these fusions have been established as the defining genetic drivers of this disease; however, they remain challenging to target pharmacologically. As such, improvements to the standard of care, which include surgery, radiation, and general chemotherapy, have focused on optimizing chemotherapy dosage and are rarely curative. Therapeutic development has, in part, been hindered by unclear in vivo regulatory mechanisms of PAX3/7::FOXO1 across cellular contexts and tumorigenic stages. Previously, our groups showed that PAX3::FOXO1, the most common fusion, is a pioneer transcription factor that reshapes chromatin architecture and can transform zebrafish cells to recapitulate human tumors. Recently, we aimed to understand how PAX3::FOXO1 initially interfaces with and modulates chromatin in vivo. We developed a zebrafish mRNA injection model and expressed human PAX3::FOXO1 during early vertebrate development. PAX3::FOXO1 utilized its homeobox domain and partial homeobox motif recognition for initial genomic binding, a new binding mode that varies from in vitro models. Moreover, PAX3::FOXO1 bound and activated inaccessible chromatin in vivo, two hallmarks of pioneer transcription factor activity. Chromatin and transcriptional profiling revealed that PAX3::FOXO1 binding resulted in activation of rhabdomyosarcoma and neural developmental gene signatures. Despite the common association of rhabdomyosarcoma with incomplete myogenesis, neural pathways and populations are present in primary patient tumors. The neural transcription factor HES3 is one example that we previously characterized as a PAX3::FOXO1 cooperating gene. The neural population is also enriched during chemotherapy treatment in patient-derived xenografts. We are continuing to utilize our zebrafish modeling and parallel approaches to understand the functional role and regulation of these neural pathways. Currently, we are investigating how HES3 modulates this activation, given that it developmentally promotes neural stem renewal, is an established target of PAX3::FOXO1, and correlates with worse patient outcomes. Neural signatures are seen in both PAX3::FOXO1 and PAX7::FOXO1 tumors, therefore we are leveraging in vivo comparative analyses between these fusions to assess if differences in neural activation can explain their divergent clinical outcomes. Altogether, this work highlights neural pathway activation as a core activity of PAX3::FOXO1 and suggests that these programs could play a critical role across multiple stages of tumorigenesis. Our long-term goal is to delineate how PAX3/7::FOXO1 cooperate with neural factors across cellular contexts to identify novel therapeutic vulnerabilities.



 Jack Kucinski, Alexi Tallan, Matthew Kent, Andrew Vontell, Katherine Silvius, Cenny Taslim, Benjamin Stanton, Genevieve Kendall. In vivo activation of neural gene signatures in fusion-positive rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B021.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51dec928b1aed79a0677d3cbac12867c633157e2" target='_blank'>
              Abstract B021: In vivo activation of neural gene signatures in fusion-positive rhabdomyosarcoma
              </a>
            </td>
          <td>
            Jack Kucinski, Alexi Tallan, Matthew R. Kent, Andrew M. Vontell, Katherine M. Silvius, C. Taslim, Benjamin Z. Stanton, Genevieve C. Kendall
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Comparative transcriptome analyses of gamma herpesviruses have been revolutionized by long‐read sequencing and integrative multi‐omics approaches, enabling unprecedented resolution of viral gene expression. Cross‐species studies of Kaposi's sarcoma‐associated herpesvirus (KSHV), Epstein–Barr virus (EBV), and murine gamma herpesvirus 68 (MHV68) have uncovered a vast diversity of transcript isoforms, including alternative 5′/3′ UTR variants, alternative splice isoforms, and transcripts encoding truncated or extended coding sequences. These efforts revealed novel transcript isoforms harboring partial open reading frames, entirely new ORFs, and an expanding repertoire of viral non‐coding RNAs. Additional regulatory mechanisms, such as promoter dependence on viral replication factors, selective polyadenylation site usage, frameshifting via alternative splicing, and extensive readthrough transcription, further illustrate the complex strategies governing gamma herpesvirus transcriptomes. This review synthesizes these discoveries, highlighting transcriptional strategies employed by gamma herpesviruses for persistence through different phases of replication, while outlining how integrative transcriptomics has reshaped our understanding of herpesvirus gene regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8150269b600bc18ee3d2193456ba4418ddf8a3de" target='_blank'>
              Cross‐Species Insights Into Gamma Herpesvirus Transcriptomes: Long‐Read and Multi‐Omics Perspectives
              </a>
            </td>
          <td>
            Ritu Shekhar, Sarah McMahon, S. Tibbetts, Erik K. Flemington, Rolf Renne
          </td>
          <td>2026-01-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Ewing sarcoma is a pediatric bone and soft tissue cancer thought to arise from mesenchymal stem cells (MSCs). It is characterized by fusion of the RNA-binding protein EWS to an ETS family transcription factor, most often FLI1. EWS::FLI1 induces changes in gene expression that have been well-studied and have been linked to its ability to bind to and alter chromatin state at GGAA repeat sequences within the genome. In addition to these effects, EWS::FLI1 alters mRNA splicing; however, the mechanism of this activity remains unclear.



 We curated a sarcoma-focused transcriptome from published long-read RNA sequencing of Ewing sarcoma, osteosarcoma, and synovial sarcoma patient tumors and matched adjacent normal tissue (215M reads). We supplemented this with published long-read RNA sequencing data from relevant normal cell types, osteosarcoma cell lines, and Ewing sarcoma lines with or without EWS::FLI1 knockdown (105M reads). We used this transcriptome as a reference to evaluate expression and splicing from short-read RNA-seq of Ewing sarcoma cell lines, MSCs, and MSC-based models of Ewing sarcoma and established Ewing-associated expression and splicing signatures.



 We generated scores based on these signatures in published knockdown-rescue experiments in which endogenous EWS::FLI1 was silenced by shRNA and replaced with exogenous shRNA-resistant mutants. Expression and splicing scores were highly correlated within these datasets (r2 = 0.89), suggesting that similar regions of the fusion protein might be required for both its functions. In contrast, the scores had limited correlation within an internal dataset of RNA-binding protein perturbations (r2 = 0.02). We next examined the effects of pharmacological agents thought to alter EWS::FLI1 activity. Interestingly, we found that lurbinectedin which disrupts EWS::FLI1’s DNA-binding could partially revert both expression and splicing scores. In contrast, YK-4-279 which disrupts the fusion’s protein-protein interactions and MS0621 which reverts a Ewing-associated chromatin signature had limited effects on these scores. To further interrogate the potential role of EWS::FLI1 DNA-binding in splicing regulation, we silenced EWS::FLI1 in TC-32 cells and replaced it with either the wildtype fusion or a DNA-binding null mutant. Unlike the wildtype construct, the DNA-binding null mutant could not rescue expression or splicing. Finally, we examined the effects of the previously published KRAB-ZFA7 repressor which silences chromatin at GGAA repeats while sparing single GGAA sites and found that it could revert both scores.



 Together, these data establish a potential connection between EWS::FLI1 activity at GGAA repeats and its effects on mRNA splicing. Additionally, we find that lurbinectedin, an FDA-approved agent currently in a phase 1/2 trial for Ewing sarcoma, suppresses both the splicing and expression effects of EWS::FLI1. Emerging compounds designed to target EWS::FLI1 DNA-binding directly will likely have similar effects.



 David V. Allegakoen, Jeffrey A. Toretsky. EWS::FLI1 activity at GGAA repeats is essential for maintenance of Ewing-like splicing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6934f37ed326d496d296a4806fccea6f323a4f63" target='_blank'>
              Abstract B013: EWS::FLI1 activity at GGAA repeats is essential for maintenance of Ewing-like splicing
              </a>
            </td>
          <td>
            D. Allegakoen, J. Toretsky
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a widely occurring malignancy with significant mortality on a global scale, making up close to 10% of all diagnosed cancers in 2020. While traditional CRC diagnostics and research have focused on nuclear genomic alterations, emerging evidence has highlighted the multifaceted roles of mitochondrial DNA (mtDNA) in the pathogenesis and clinical management of CRC. In this review, we examine three interlaced aspects of mtDNA in CRC: (1) Liquid biopsy biomarkers: Cell-free mtDNA circulating in the blood serving as a minimally invasive diagnostic and monitoring tool; (2) Nuclear mtDNA (NUMT)-driven genomic instability: The somatic nuclear incorporation of mtDNA (NUMT segments, or NUMT), contributing to mutational burden and chromosomal disruption in tumours; and (3) Mitochondria-encoded micropeptide signaling - small peptides encoded by the mtDNA that modulate cellular pathways and tumour behaviour. Recent high-impact studies have demonstrated that tumour-derived mtDNA in biofluids can augment cancer detection sensitivity, although technical challenges remain due to mtDNA fragmentation and background noise. Meanwhile, genomic analyses have uncovered a significant increase in NUMT insertion events in CRC cells, linking mitochondrial genome escape to nuclear genome instability and identifying potential numtogenesis suppressor genes. A novel dimension of mito-nuclear interactions in cancer was discovered in mitochondrial microproteins, such as humanin and mitochondrial open reading frame of the 12S rRNA type-c. Humanin exhibits both tumour-promoting and cytoprotective properties under specific conditions, while mitochondrial open reading frame of the 12S rRNA type-c possesses tumour-suppressive activities under other conditions. The outcomes of clinical, mechanistic, and translational research, revealing how mtDNA-based biomarkers and involvement contribute towards early detection, prognostication, and treatment of CRC, are presented.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba21880b5e33a449e69dca805fc3ec140c49eb10" target='_blank'>
              Interlaced roles of mitochondrial DNA in colorectal cancer: Liquid-biopsy biomarkers, nuclear mtDNA-driven genomic instability, and mito-encoded micro peptide signaling
              </a>
            </td>
          <td>
            T. Koo, X. Leong, Y. Lee, Firdaus Hayati, A. Zakaria
          </td>
          <td>2025-12-15</td>
          <td>World Journal of Gastrointestinal Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/468c034edcdec5179082358a5f9944277909facb" target='_blank'>
              Identification of DNA methylation based prognostic subtype and signature in epithelial ovarian cancer
              </a>
            </td>
          <td>
            Jingjing Zhou, Yuxin Chen, Wei Liu, Chao Wang, Lei Liu, Wei Geng, Linqing Chai, Dong Yang, H. Dai, Haixin Li, Yanrui Zhao, Kexin Chen, Hong Zheng, Lian Li
          </td>
          <td>2025-12-27</td>
          <td>Human Genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third most common malignancy worldwide, and the five-year survival rate for patients with metastatic disease remains below 15% despite advances in current therapeutic approaches. Post-translational modifications (PTMs) play a pivotal role in CRC initiation and progression, among which neddylation—a critical ubiquitin-like modification—is closely associated with tumor cell proliferation, migration, and chemotherapy resistance. This modification covalently attaches neural precursor cell-expressed developmentally downregulated protein 8 (NEDD8) to lysine residues of substrate proteins, thereby regulating protein stability, DNA repair, and immune responses. In CRC, key enzymes in the neddylation pathway, such as NAE1, UBE2M, and DCUN1D1, are frequently aberrantly activated, leading to the stabilization of key oncoproteins and cell-cycle regulators by preventing their ubiquitin-mediated degradation, thereby promoting tumor progression and drug resistance. Although neddylation has been extensively studied in various cancer types, its precise role in CRC has not been fully elucidated. Recent studies have shown that targeting neddylation—particularly with NAE1 inhibitors such as MLN4924—can significantly suppress tumor progression and offer new therapeutic opportunities to overcome chemoresistance. This review systematically summarizes the roles of neddylation in CRC pathogenesis, chemoresistance, and immune microenvironment remodeling, with a focus on the clinical potential of combining neddylation-targeted inhibitors with chemotherapy and immunotherapy, as well as the prospective application of liquid biopsy in precision monitoring, aiming to provide a theoretical basis and future directions for molecular targeted therapy and clinical translation in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/700a3383d26d6893d3a5b7f74acc16f607b31dc5" target='_blank'>
              The role of neddylation in colorectal cancer and its therapeutic potential
              </a>
            </td>
          <td>
            Chen Mi, Feng Cui, Siyang Wang, Yongyue Du, Yongzhao Li, Hanteng Yang
          </td>
          <td>2025-12-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Chemoresistance remains a major barrier in cancer therapy, frequently resulting in treatment failure and reduced patient survival. This multifaceted phenomenon arises from the interplay of well-established mechanisms such as genetic mutations, non-genetic adaptations, and tumor microenvironment (TME) mediated influences as well as newly emerging findings from recent research (2020-present). Key biochemical contributors include diminished intracellular drug accumulation through altered uptake or efflux, dysregulation of drug metabolism and bioactivation involving multiple Phase I and Phase II enzymes, genomic instability affecting DNA repair pathways, disruption of cell cycle control, and evasion of apoptosis. In addition, recent evidence highlights the roles of epigenetic reprogramming, metabolic reconfiguration, and TME-derived signaling in amplifying chemoresistance. This review integrates both foundational concepts and recent advancements in understanding drug resistance, with particular emphasis on updated insights into drug-metabolizing enzymes and their impact on therapeutic failure. It also evaluates current and emerging strategies to overcome resistance including targeting metabolic enzymes, modulating the TME, and implementing polytherapy's that address multiple resistance pathways. By synthesizing established knowledge with recent discoveries, this review highlights promising directions for improving the efficacy of cancer treatments and enhancing patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92ce9c249ee2d1807438a427a4bdf1ac44ab032" target='_blank'>
              Molecular basis of cancer chemoresistance: biochemical insights.
              </a>
            </td>
          <td>
            R. Guneidy
          </td>
          <td>2026-01-26</td>
          <td>Drug metabolism reviews</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Processing bodies (P-bodies) are membrane-less organelles composed of condensed mRNAs and proteins that play essential role in mRNAs decay and storage, contributing to the translational control of cellular proteostasis. Regulation of P-body assembly/disassembly by signaling events, cellular stress or specific environmental conditions shapes the rate of RNA turnover and protein synthesis, controlling cell growth, differentiation and survival. Deregulation of protein translation is an important factor for tumor development and progression and cancer cells benefit from P-bodies to reshape their proteome to support specific metabolic needs and promote tumor development, progression and metastasis. Hence, understanding the composition and the regulation of P-bodies, both under physiological and pathological conditions, will define the mechanisms underlying cancer cell plasticity and develop novel therapeutic strategies to inhibit cancer growth and metastasis. Here, we will discuss the principal mechanisms of P-body regulation and function, with special focus on the role of these ribonucleoprotein condensates in cancer. This review summarizes the molecular mechanisms that regulate P-body assembly/disassembly and their impact in different aspects of human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/985073e49b272efce01c573118f58597fd6fcf2f" target='_blank'>
              Reshaping cancer cell plasticity by P-body dynamics and protein translation
              </a>
            </td>
          <td>
            Emanuela Senatore, Laura Rinaldi, F. Chiuso, A. Bianco, Antonio Feliciello
          </td>
          <td>2026-01-02</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ABSTRACT Renal cell carcinoma (RCC) presents a significant global health challenge, with a substantial proportion of patients diagnosed with advanced or metastatic disease due to the limitations of current diagnostic imaging and the lack of validated non‐invasive biomarkers. These conventional methods, including computed tomography and magnetic resonance imaging, often lack the sensitivity and specificity to differentiate benign from malignant small renal masses reliably or to detect minimal residual disease (MRD) post‐treatment. This review explores the transformative potential of liquid biopsy, explicitly focusing on circulating tumour DNA (ctDNA) fragmentomics and epigenetic signatures, to overcome these clinical hurdles. This review also explores how the analysis of ctDNA fragmentation patterns—such as size distribution, end motifs, and nucleosome footprints—provides a mutation‐independent method to enhance RCC detection, even in low‐shedding tumours. Concurrently, RCC‐specific epigenetic alterations, particularly DNA methylation profiles, offer particular biomarkers for early detection, tumour classification, and prognostication. This Review examines evidence that integrating these multi‐analyte approaches—combining fragmentomic and epigenetic data—synergistically improves diagnostic accuracy, enables sensitive MRD assessment, and allows precision monitoring of treatment response and tumour evolution. Despite existing technical and biological challenges, the convergence of ctDNA fragmentomics and epigenetic profiling heralds a new era for the non‐invasive, dynamic, and personalised management of RCC, promising to improve patient outcomes through earlier intervention and tailored therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba930e2032c840e4948e71d0668510521c1d91f2" target='_blank'>
              Emerging Role of ctDNA Fragmentomics and Epigenetic Signatures in the Early Detection, Minimal Residual Disease Assessment, and Precision Monitoring of Renal Cell Carcinoma
              </a>
            </td>
          <td>
            H. Kamli, Najeeb Ullah Khan
          </td>
          <td>2026-01-30</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="


 Whilst most mutations cells accumulate are neutral, some can alter cellular functions leading to positive selection and clonal expansions. In inflammatory bowel disease (IBD), epithelial injury-repair cycles raise mutation rates and favour clones with growth advantages. Previous studies [1-3] showed that clones with IL-17 signalling pathway mutations, including PIGR, NFKBIZ and ZC3H12A, recurrently expand in IBD despite being uncommon in colorectal cancer. Organoid work suggested that NFKBIZ mutations provide resistance to IL-17A-induced apoptosis and loss of PIGR was hypothesised to worsen inflammation by promoting dysbiosis. Thus, a positive feedback loop could be established, in which mutant clones drive tissue damage while expanding further. We aim to improve our understanding of how clonal expansions and driver genes influence disease biology in IBD. To achieve this, we performed paired DNA-RNA sequencing of individual human colonic crypts to characterise the transcriptional consequences of driver mutations.



 Multiomics laser-microdissection of single colonic crypts was possible with an improved low-input protocol. Fresh biopsies were collected during colonoscopies from inflamed and uninflamed colon regions. We obtained over 200 new biopsies from 25 patients which, combined with prior data, enabled selection analysis across 1500+ crypts from 50+ patients. We generated 700+ single crypts with paired DNA-RNA data, enabling differential expression analysis (DGE) between mutant and wild-type crypts.



 Our work confirms elevated mutation rates and characteristic mutational signatures associated with azathioprine treatment. Beyond identifying PIGR, NFKBIZ and ZC3H12A as strongly selected genes, consistent with prior work, we show that NFKBIZ is also highly prevalent in individuals of European ancestry. We also expand the catalogue of putative driver genes, particularly highlighting IL13RA1, ELF3 and MED12. DGE analysis suggests mutations in some genes e.g., NFKBIZ, may protect crypts from inflammatory effects, while others remain influenced by inflammation, potentially expanding primarily through increased crypt fission. Crypts with cancer-related mutations display elevated inflammatory signalling and epithelial-to-mesenchymal transition features compared to crypts with IBD-characteristic mutations e.g., ZC3H12A. These findings reveal how distinct driver mutations shape epithelial behaviour in IBD.



 Overall, paired DNA and RNA sequencing of single colonic crypts enabled us to broaden the catalogue of driver genes implicated in IBD and characterise their transcriptomic consequences in human tissue for the first time. Thus, improving our understanding of how mutations shape the epithelial landscape in IBD.
 References:
 1.Olafsson S, McIntyre RE, Coorens T, Butler T, Jung H, Robinson PS, Lee-Six H, Sanders MA, Arestang K, Dawson C, Tripathi M, Strongili K, Hooks Y, Stratton MR, Parkes M, Martincorena I, Raine T, Campbell PJ, Anderson CA. Somatic Evolution in Non-neoplastic IBD-Affected Colon. Cell. 2020 Aug 6;182(3):672-684.e11. doi: 10.1016/j.cell.2020.06.036. Epub 2020 Jul 21. PMID: 32697969; PMCID: PMC7427325.
 2.Kakiuchi N, Yoshida K, Uchino M, Kihara T, Akaki K, Inoue Y, Kawada K, Nagayama S, Yokoyama A, Yamamoto S, Matsuura M, Horimatsu T, Hirano T, Goto N, Takeuchi Y, Ochi Y, Shiozawa Y, Kogure Y, Watatani Y, Fujii Y, Kim SK, Kon A, Kataoka K, Yoshizato T, Nakagawa MM, Yoda A, Nanya Y, Makishima H, Shiraishi Y, Chiba K, Tanaka H, Sanada M, Sugihara E, Sato TA, Maruyama T, Miyoshi H, Taketo MM, Oishi J, Inagaki R, Ueda Y, Okamoto S, Okajima H, Sakai Y, Sakurai T, Haga H, Hirota S, Ikeuchi H, Nakase H, Marusawa H, Chiba T, Takeuchi O, Miyano S, Seno H, Ogawa S. Frequent mutations that converge on the NFKBIZ pathway in ulcerative colitis. Nature. 2020 Jan;577(7789):260-265. doi: 10.1038/s41586-019-1856-1. Epub 2019 Dec 18. PMID: 31853061.
 3.Nanki K, Fujii M, Shimokawa M, Matano M, Nishikori S, Date S, Takano A, Toshimitsu K, Ohta Y, Takahashi S, Sugimoto S, Ishimaru K, Kawasaki K, Nagai Y, Ishii R, Yoshida K, Sasaki N, Hibi T, Ishihara S, Kanai T, Sato T. Somatic inflammatory gene mutations in human ulcerative colitis epithelium. Nature. 2020 Jan;577(7789):254-259. doi: 10.1038/s41586-019-1844-5. Epub 2019 Dec 18. PMID: 31853059.
 4.EllisP,MooreL,SandersMA,ButlerTM,BrunnerSF,Lee-SixH,OsborneR,FarrB,CoorensTHH,LawsonARJ,CaganA,Stratton MR, Martincorena I, Campbell PJ. Reliable detection of somatic mutations in solid tissues by laser-capture microdissection and low-input DNA sequencing. Nat Protoc. 2021 Feb;16(2):841-871. doi: 10.1038/s41596-020-00437-6. Epub 2020 Dec 14. PMID: 33318691.
 Conflict of interest:
 Torra I Benach, Maria: No conflict of interest
 Marshall, Henry: No conflict of interest
 Jung, Hyunchul: No conflict of interest
 Abascal, Federico: No conflict of interest
 Lawson, Andrew: No conflict of interest
 Hu, May: No conflict of interest
 Goodwin, Scott: No conflict of interest
 Garri, Wendy: No conflict of interest
 Harris, Bradley: Personal Fees: I have received honoraria for a presentation at a BridgeBio board meeting.
 Garcia Salinas, Isaac: No conflict of interest
 Jarman, Georgee: No conflict of interest
 El Garwany, Omar: No conflict of interest
 Hooks, Yvette: No conflict of interest
 Sanghvi, Rashesh: No conflict of interest
 O’Neill, Laura: No conflict of interest
 Yogakanthi, Saiumaeswar: No conflict of interest
 Brezina, Biljana: No conflict of interest
 Chappel, Lia: No conflict of interest
 Cotobal Martin, Cristina: No conflict of interest
 Rahbari, Raheleh: No conflict of interest
 Raine, Timothy: Grant: Abbvie, Takeda Personal Fees: TR has received research/educational grants and/or speaker/consultation fees from Abbvie, Alfasigma, Arena, Aslan, AstraZeneca, Boehringer-Ingelheim, BMS, Celgene, Domain Therapeutics, Eli Lilly, Ferring, Galapagos, Gilead, GSK, Heptares, LabGenius, Janssen, MonteRosa, Mylan, MSD, Novartis, Numab, Pfizer, Roche, Sandoz, Scientia, Takeda, UCB and XAP therapeutics
 Anderson, Carl: Personal Fees: Carl Anderson (CAA) has received consultancy or speaker fees from Genomics plc, BridgeBio Ltd and Glaxo Smith Kline.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c73cadd26922fc11658f33d34a414918b714ee81" target='_blank'>
              DOP109 Investigating the role of driver mutations in Ulcerative Colitis through paired DNA and RNA sequencing of single colonic crypts
              </a>
            </td>
          <td>
            M. Torra, I. Benach, H. Marshall, H. Jung, F. Abascal, A. Lawson, M. Hu, S. Goodwin, W. Garri, B. Harris, I. G. Salinas, G. Jarman, O. El Garwany, Y. Hooks, R. Sanghvi, L. O'Neill, S. Yogakanthi, B. Brezina, L. Chappel, C. Martin, R. Rahbari, T. Raine, C. Anderson
          </td>
          <td>2026-01-01</td>
          <td>Journal of Crohn’s and Colitis</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/490f7019972e1dff0c8c3442eb3b1f9903b34509" target='_blank'>
              Exosome-Derived Circular RNAs in Colorectal Cancer: Emerging Roles in Tumorigenesis, Diagnosis, and Therapy.
              </a>
            </td>
          <td>
            Amir Mohammad Aghaie, Fereydoun Rahmani, N. F. Maroufi, Vahid Zarezade, Soleyman Bafadam, Mostafa Mostafazadeh
          </td>
          <td>2026-02-01</td>
          <td>Journal of biochemical and molecular toxicology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Merkel cell polyomavirus (MCV) T antigen functions as an oncoprotein that drives the transformation of Merkel cell carcinoma (MCC) cells by activating transcription factors involved in cell proliferation. The viral T antigen promoter requires the activity of the cellular coactivator CREB-binding protein (CBP)/p300 for its expression. Inhibition of CBP/p300 with two distinct small-molecule inhibitors suppresses T antigen expression, leading to cell cycle arrest and upregulation of the cell cycle inhibitor p27Kip1. This shift promotes neuronal differentiation, associated with neurite outgrowth in MCC cells. RNA sequencing revealed downregulation of genes involved in E2F, Myc, mTORC1 oncogenic signaling, as well as markers of the Merkel cell lineage including Sox2 and Atoh1. Notably, a rare MCC case exhibiting a mixed cellular composition, with loss of T antigen expression and neuroblastic phenotype, showed a transcriptomic profile resembling that of MCC cells treated with CBP/p300 inhibitors. This suggests that similar differentiation processes may contribute to tumor heterogeneity in patients. This study presents the model system enabling reversible switching between a transformed and differentiated cell state in a human cancer using small-molecule treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c3adc8efe8ee8ff8d0fd773026a2aa619b608d8" target='_blank'>
              Pharmacologic reversion of Merkel cell carcinoma via CBP/p300 inhibition.
              </a>
            </td>
          <td>
            Joseph L. Collura, Kuan Cheok Lei, Mitalee Chandra, T. Kervarrec, Hyun Jung Park, Mazdak Dalkoohi, Jürgen C Becker, Yuan Chang, Patrick S. Moore, M. Shuda
          </td>
          <td>2025-12-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Accurate detection of gene fusions is critical for the diagnosis and management of sarcomas. However, occasional false-negative results may occur due to factors such as low tumor cellularity, limited assay coverage of relevant genomic regions, or mis-annotation by bioinformatic pipelines. We recently encountered a case in which our clinical SARCP fusion detection assay failed to identify an HMGA2::NCOR2 (exon 3–exon 16) fusion.



 The SARCP assay is a targeted, amplicon-based RNA-seq next-generation sequencing (NGS) assay designed to detect gene fusions across 138 genes, including HMGA2. Although NCOR2 is not among the targeted genes, the assay is capable of detecting fusions when only one gene partner is represented.



 Suspicion of a false-negative result arose when the pathologist noted that the tumor’s histologic features—keratin-positive, giant cell-rich soft tissue—were consistent with a subtype previously associated with the HMGA2::NCOR2 fusion. This prompted a review of the bioinformatic output, which had erroneously reported an HMGA2::MAML2 fusion. The misannotation was traced to sequence homology involving a CAA/CAG repeat at the 5′ end of NCOR2 exon 16, likely resulting from DNA polymerase slippage and misalignment in low-complexity, partially complementary repetitive regions. This erroneously lead to the target genomic sequence getting mapped to MAML2 exon 2 as opposed to the NCOR2 exon 16.



 This case highlights how bioinformatic mis-annotation can lead to occasional false-negative fusion calls. It underscores the importance of manual review in cases with strong clinical or pathologic suspicion and calls attention to the need for fusion-calling algorithms that better handle repetitive and homopolymeric sequences.



 Ganesh P. Pujari, Mitch L. Klebig, Kurt S. Willkom, Montha Chhunn, Stefan W. Nelson, Neiladri Saha, Kevin C. Halling. Mis-Annotation of an HMGA2::NCOR2 Fusion in FFPE Sarcoma Samples: A Case Illustrating Bioinformatic Limitations in Fusion Detection [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b640522fda2138d045848d46734cd774a873c147" target='_blank'>
              Abstract A012: Mis-Annotation of an HMGA2::NCOR2 Fusion in FFPE Sarcoma Samples: A Case Illustrating Bioinformatic Limitations in Fusion Detection
              </a>
            </td>
          <td>
            Ganesh P. Pujari, Mitch L. Klebig, Kurt S. Willkom, Montha Chhunn, Stefan W. Nelson, Neiladri Saha, K.C. Halling
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer therapy resistance emerges from highly integrated molecular systems that enable tumor cells to evade cell death and survive cytotoxic therapeutic stress. High Mobility Group Box 1 (HMGB1) is increasingly gaining recognition as a central coordinator of these resistance programs. This review delineates how HMGB1 functions as a molecular switch that dynamically redistributes between cellular compartments in response to stress, with each localization enabling a distinct layer of resistance. In the nucleus, HMGB1 enhances chromatin accessibility and facilitates the recruitment of DNA repair machinery, strengthening resistance to radio- and chemotherapeutic damage. Cytosolic HMGB1 drives pro-survival autophagy, maintains redox stability, and modulates multiple regulated cell death pathways, including apoptosis, ferroptosis, and necroptosis, thereby predominantly shifting cell-fate decisions toward survival under therapeutic pressure. Once released into the extracellular space, HMGB1 acts as a damage-associated molecular pattern (DAMP) that activates key pro-survival and inflammatory signaling pathways, establishing microenvironmental circuits that reinforce malignant progression and therapy escape. HMGB1 further intensifies resistance through upregulation of multidrug resistance transporters, amplifying drug efflux. Together, these compartmentalized functions position HMGB1 as a central node in the networks of cancer therapy resistance. Emerging HMGB1-targeted agents, ranging from peptides and small molecules to receptor antagonists and nanoformulations, show promise in reversing resistance, but clinical translation will require precise, context- and redox-informed HMGB1 targeting to overcome multifactorial resistance program in refractory cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcd9023ae13f0952b85c73d6ba319988bfa2cf22" target='_blank'>
              HMGB1: A Central Node in Cancer Therapy Resistance
              </a>
            </td>
          <td>
            Bashar A. Alhasan, Boris A. Margulis, Irina V. Guzhova
          </td>
          <td>2025-12-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Unraveling how gene regulations are remodeled during evolution is central to understanding how biological processes evolve. We explored this question in the frame of X-chromosome inactivation, a process under strong selective constraint, governed by the XIST lncRNA and its cis-regulators. Using functional approaches on closely related primate species, we show that XIST regulation has uniquely diverged over a short evolutionary timescale. In human and marmoset embryonic stem cells (ESCs), the JPX lncRNA gene is a major regulator of XIST expression. In contrast, JPX has a minor effect on XIST in macaque ESCs, where it acts together with a macaque-specific enhancer. This occurs within a reshuffled 3D organization of the XIST neighborhood triggered by the insertion of a HERVK transposon in the macaque lineage. Retrospective sequence comparisons revealed that many XIST regulators are not evolutionarily constrained, supporting the hypothesis that neutrally evolving noncoding elements harbor adaptive potential. These results illuminate how evolutionary recent elements are integrated into preexisting regulatory landscapes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/565e8acbf3d02200d648afa59e906c351636a475" target='_blank'>
              Remodeling of XIST regulatory landscape during primate evolution
              </a>
            </td>
          <td>
            Emmanuel Cazottes, Charbel Alfeghaly, C. Rognard, Anamaria Necsulea, Agnese Loda, G. Castel, L. Villacorta, Michael Dong, Edith Heard, Irène Aksoy, P. Savatier, C. Morey, C. Rougeulle
          </td>
          <td>2026-01-16</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="BRC repeats are integral to BRCA2 function and mediate RAD51 loading during homologous recombination (HR). Their number varies across species, but mammals, including mice and humans, harbor eight repeats. Prior studies have suggested functional redundancy among these repeats, but the biological significance of maintaining multiple repeats remains unresolved. Here, we demonstrate that the presence of a single BRC repeat, either BRC2 or BRC4, is sufficient for mouse embryonic stem cell (mESC) viability, RAD51 loading, PARP inhibitor resistance and protection of stalled replication forks. Consistent with these findings, we show that knock-in mice with a single BRC repeat 2 or 4 are viable and exhibit normal growth and fertility. In contrast, embryonic fibroblasts from these mice display genomic instability and impaired RAD51 recruitment. Notably, this defect is rescued under low oxygen culture conditions, which mimics physoxic levels, whereas exposure to oxidative stress impairs RAD51 recruitment in mESCs harboring a single BRC repeat. Together, our findings indicate that while a single BRC repeat is sufficient under physiological conditions, the evolutionary retention of multiple BRC repeats likely ensures robust genome stability under extreme oxidative stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f87b204f3013e5a6252054983a01e00ba9bd045" target='_blank'>
              A single BRCA2 BRC repeat supports viability while multiple repeats ensure resilience under stress
              </a>
            </td>
          <td>
            A. P. Mishra, Sounak Sahu, S. Sengodan, Gemma Moore, Swati Priya, Natasha Oberoi, Julia R. Jensen, Eileen Southon, Mellissa Galloux, Dylan Caylor, Teresa Sullivan, Sandra S. Burkett, Margaret Albaugh, Parirokh P Awasthi, F. T. Ardori, L. Tessarollo, Ryan B. Jensen, Shyam K. Sharan
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28a3d75c8b930697a81a7aeceffc4df22265c789" target='_blank'>
              3D confinement reshapes RNA folding and enhances circularisation in the Zika virus
              </a>
            </td>
          <td>
            J. K. Novev, J. Y. Lau, D. Marenduzzo, G. Kudla
          </td>
          <td>2026-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64122def353963318315fa81fc6a3976164d571b" target='_blank'>
              DOT1L activity limits transcription elongation velocity and favors RNAPII pausing to facilitate mutagenesis by AID.
              </a>
            </td>
          <td>
            Poorani Ganesh Subramani, Noé Seija, Jana Ridani, J. Boulais, Paul A Donnelly, Mélanie Provencher, H. Bagci, Christian Poitras, Jean-François Côté, François Robert, J. D. Di Noia
          </td>
          <td>2026-01-12</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f687787e5f5da10c05fbeb66dd1e619139451c70" target='_blank'>
              Multiomics reveals the mechanism of KNG1 in promoting tumor progression in adamantinomatous craniopharyngioma
              </a>
            </td>
          <td>
            Lin Zhou, Zhicheng Xu, Chao Fang, Hui Ren, C. Yang, Zhigang Wang, Tao Hong, Jie Wu
          </td>
          <td>2026-01-01</td>
          <td>Journal of Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Immunotherapies have transformed cancer treatment by harnessing the immune system to recognize and eliminate malignant cells, offering durable clinical benefit across diverse tumor types. Despite successes with immune checkpoint inhibitors (ICIs) and other strategies like cytokines, oncolytic viruses, cancer vaccines, bispecific antibodies, and adoptive cell therapies, substantial fractions of patients still fail to respond or develop resistance. The oncogene MYC, deregulated in ~70% of human cancers, has emerged as a central driver of immune evasion and a key contributor to immunotherapy failure. MYC regulates broad transcriptional networks controlling proliferation, metabolism, angiogenesis, and cell survival, while also orchestrating profound remodeling of the tumor microenvironment (TME). Mechanistically, oncogenic MYC suppresses antigen processing and presentation, attenuates interferon signaling, and upregulates immune checkpoints such as PD-L1 and CD47. Concurrently, MYC stimulates secretion of immunosuppressive cytokines and chemokines that recruit regulatory T cells, myeloid-derived suppressor cells, and M2 macrophages, while driving metabolic reprogramming that fosters nutrient competition, hypoxia, and acidosis, impairing effector T- and NK-cell function. Through these pathways, MYC promotes primary, adaptive, and acquired resistance to immunotherapy. Targeting MYC, directly or indirectly, holds promise to restore immune surveillance and potentiate immunotherapeutic efficacy. This review highlights MYC as a master regulator of tumor–immune interactions and underscores the therapeutic potential of MYC inhibition to overcome resistance and expand the clinical impact of cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/437ab7ab32dc49430fcd2b151a4005d089060a97" target='_blank'>
              MYC at the tumor–immune interface: mechanisms of immune escape and immunotherapy resistance
              </a>
            </td>
          <td>
            Íñigo González-Larreategui, Manrique Valdés-Bango Martín, S. Casacuberta-Serra, Laura Soucek
          </td>
          <td>2026-01-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Circulating tumor DNA (ctDNA) represents a promising noninvasive biomarker for cancer diagnosis, including oral cancer. However, its clinical translation is currently limited by the lack of precise and reliable detection techniques. In this study, we developed a novel fluorescent biosensor for the detection of single-nucleotide variations in ctDNA, which integrates dual dCas9/sgRNA complexes for target recognition, proximity ligation-initiated terminal hairpin formation and self-priming amplification (PS-THSP), and Cas12a/crRNA-mediated signal output. A key innovation of this design is its multilayered specificity strategy, combining mutation-specific recognition by dual dCas9/sgRNA, proximity-dependent ligation, and Cas12a/crRNA-assisted verification of PS-THSP amplicons. This integrated approach offers a significant advance over existing CRISPR/Cas-based methods that rely primarily on signal amplification. Furthermore, the biosensor achieves high sensitivity through the synergistic coupling of PS-THSP amplification and Cas12a trans-cleavage activity, enabling a broad dynamic range spanning 6 orders of magnitude and a detection limit as low as 0.12 fM within 120 min. When applied to serum samples, the biosensor reliably detected ctDNA with high accuracy, demonstrating its strong potential for clinical cancer diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/509c19751ef2d8c94d6cf7c0447fb541e0c4ad69" target='_blank'>
              Single-Nucleotide Variation Analysis in Oral Squamous Cell Carcinoma-Related ctDNA by dCas9/sgRNA Recognition-Mediated Proximity Ligation-Triggered Terminal Hairpin Formation and Self-Priming Amplification.
              </a>
            </td>
          <td>
            Xiaobo Yuan, Fan Yang, Xianglian Chen, Pin Xiao, Liangfang Shen
          </td>
          <td>2026-01-28</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The loss of chromosome 3p and the inactivation of the tumor suppressor gene von Hippel-Lindau (VHL) were identified in clear cell renal cell carcinomas (ccRCC) over three decades ago. Since then, mutations in genes for the three chromatin modulators, polybromo 1 (PBRM1), SET domain-containing 2 (SETD2), and BRCA1-associated protein-1 (BAP1), have been recognized as common in ccRCC. Although these genomic alterations are central to understanding ccRCC’s development, other deregulated cellular processes are also prominent in these tumors. Metabolic reprogramming is a key hallmark of this disease, characterized by various changes linked to the stabilization of hypoxia-inducible factors (HIF), including increased aerobic glycolysis, elevated lipid levels, and glutamine dependence for cell survival. Additionally, HIF-α stabilization plays a crucial role in regulating the immune system, thereby enhancing CD8+ T lymphocyte cytotoxicity. Immune checkpoint inhibitors (ICI) are now used as first-line treatments to target the often highly infiltrated tumor microenvironment of ccRCC. However, the effectiveness of ICI varies and is difficult to predict. Although emerging studies are beginning to provide insight, evidence suggests roles for PBRM1, SETD2, and BAP1 in metabolic regulation and in shaping the tumor immune microenvironment in ccRCC. Here, we review recent advances in this field and examine their impact on the management of ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d8124d8b083fe78630c316b5f74781f77ea65c" target='_blank'>
              The fall of the genome protectors triad: PBRM1, SETD2, and BAP1’s impact on metabolism and immunity in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Mathieu Johnson, Sandra Turcotte
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Pediatric acute myeloid leukemias (AMLs) are driven by transcription factor (TF) fusions that reprogram the epigenome, creating fusion-specific subtypes with distinct biomarkers, prognoses, and vulnerabilities. Pediatric AMLs driven by the CBFA2T3::GLIS2 (C/G) fusion are among the highest-risk subtypes, with fewer than 15% patients surviving 60 months post-diagnosis. These dismal outcomes stem from an incomplete understanding of how C/G rewires chromatin to establish subtype-specific regulatory dependencies, limiting development of targeted therapies. Comprehensive, multi-omic dissection of oncofusion-driven gene regulation is essential to reveal druggable mechanistic vulnerabilities.



 To detect C/G-interacting factors, we performed unbiased immunoprecipitation mass spectrometry (IP-MS) profiling of C/G in patient-derived AML cell lines. This revealed a novel interaction with mammalian SWI/SNF (SWItch/Sucrose Non-Fermentable) chromatin remodeling complexes, which normally guide hematopoietic differentiation. CRISPR-based viability assays showed that C/G cell survival relies on BRG1-containing mSWI/SNF, revealing a fusion-specific chromatin remodeling dependency. To investigate how this interaction reprograms myeloid cells, we compared C/G-transformed hematopoietic stem and progenitor cells (HSPCs) to non-transformed counterparts. CUT&RUN profiling of C/G-mSWI/SNF occupancy and histone modifications, integrated with RNA-seq, showed that C/G retargets mSWI/SNF to oncogenic regulatory elements, sustaining overexpression of cell cycle genes CCND2 and CDK6. This C/G-directed chromatin remodeling creates a clinically targetable mSWI/SNF-G1 regulatory axis. Pharmacologic inhibition of this axis using palbociclib, an FDA-approved CDK4/6 kinase inhibitor, reduced leukemic burden and extended survival by ∼56% in an aggressive C/G AML mouse model, highlighting a clinically actionable vulnerability.



 These findings reveal how the C/G oncofusion repurposes BRG1-containing mSWI/SNF complexes to drive cell cycle gene activation, establishing a chromatin–cell cycle regulatory circuit that is selectively targetable through G1 restriction. This mechanistic insight provides a rationale for repurposing CDK4/6 inhibitors for children with this otherwise lethal AML subtype.



 Gabriel E. Boyle, Alexander W. Ying, Lara Yao, Chao-Jen Wong, Liam Caven, Morgan Merrill, Farzane Sivandzade, Matthew Wither, Soheil Meshinchi, Rachel Rau, Shan Lin, Cigall Kadoch, Jay F. Sarthy. Targeting Chromatin–Cell Cycle Vulnerabilities in High-Risk Pediatric Acute Myeloid Leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr PR005.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51536a8c7256334de8b62c77727b8c51d68fcc39" target='_blank'>
              Abstract PR005: Targeting Chromatin–Cell Cycle Vulnerabilities in High-Risk Pediatric Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Gabe Boyle, Alexander W. Ying, Lara Yao, Chao-Jen Wong, Liam Caven, Morgan Merrill, Farzane Sivandzade, Matthew Wither, S. Meshinchi, Rachel Rau, Shan Lin, Cigall Kadoch, Jay F. Sarthy
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="NUTM1-rearrangement (NUTM1-r) defines a significant subset of B-cell acute lymphoblastic leukemia (B-ALL), particularly in infants lacking KMT2A-rearrangements (KMT2A-r), yet its underlying molecular characteristics remain poorly understood. Here, we establish that NUTM1-r leukemia is a discrete entity characterized by a unique transcriptional and epigenetic landscape, notably featuring global DNA hypomethylation, irrespective of the 5' fusion partner. Functional interrogation of NUTM1 fusions reveals a dual oncogenic role: they drive commitment towards the B-lymphoid lineage while concurrently conferring potent leukemic stem cell properties. Strikingly, expression of a representative fusion, BRD9-NUTM1, is sufficient to induce serially-transplantable prepro-B-like leukemia in vivo, faithfully recapitulating the key molecular and immunophenotypic features of human NUTM1-r B-ALL. Mechanistically, NUTM1 fusions establish an aberrant chromatin state, marked by global enhancement of H3K27 acetylation and the creation of distinctive open chromatin regions that co-opt both B-lineage and stemness-related transcriptional programs, including those involving NF-κB and posterior HoxA genes. In stark contrast to resistant KMT2A-r leukemias, NUTM1-r leukemic cells exhibit a profound sensitivity to chemotherapy. This vulnerability is mechanistically linked to the leukemia's dependence on active transcription. Our findings delineate the unique molecular profile of NUTM1-r leukemias, revealing specific vulnerabilities that rationalize their favorable clinical outcomes and suggest opportunities for modified therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec942964a1f8b552f79cec056ff910b4f7724fe5" target='_blank'>
              Comprehensive Molecular and Functional Analysis of NUTM1-Rearranged Leukemia.
              </a>
            </td>
          <td>
            Koutarou Nishimura, Tomoya Isobe, Tsukasa Shigehiro, Masaki Nomura, Weijia Zang, Muran Xiao, Wenjuan Liao, Yui Koike, Akira Nishimura, A. Sato-Otsubo, Hiromi Yamazaki, Hiromi Ito, Shinri Okada, Naomi Matsumoto, Wataru Saika, Yifan Zhang, Yumi Aoyama, Chihiro Hasegawa, Takaya Yamasaki, Yasuo Kubota, Kentaro Oki, N. Kiyokawa, G. Nagae, Kenichi Yoshida, Y. Nannya, Hiroo Ueno, Shiro Fukuda, K. Tatsuno, S. Tsutsumi, Yusuke Shiozawa, Y. Shiraishi, K. Chiba, Hiroko Tanaka, Mariko Eguchi, Yuki Arakawa, K. Koh, Takao Deguchi, Daisuke Tomizawa, T. Miyamura, Eiichi Ishii, Shuki Mizutani, Satoru Miyano, Hiroyuki Aburatani, Seishi Ogawa, A. Takaori-Kondo, Akihiko Yokoyama, Omar Abdel-Wahab, Patricia Ernst, Junko Takita, Tomokatsu Ikawa, Masatoshi Takagi, Daichi Inoue
          </td>
          <td>2025-12-29</td>
          <td>Blood</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Long non-coding RNAs (lncRNAs) constitute a large class of ribonucleic acids, participating in multiple biological events within tumor cells, especially the regulation of transcription. A growing body of literature has revealed that abnormalities of lncRNA expression could result in carcinogenesis and oncogenesis by exerting inhibition or oncogenic effects. LINC01094 is a recently identified lncRNA found to be dys-regulated in an assortment of cancer tissues and control multiple biological processes via competing endogenous RNA (ceRNA) mechanisms. Specifically, LINC01094 functions as a ceRNA to modulate tumor cell growth, invasion, and migration by regulating critical signaling pathways including PI3K/AKT, PTEN/AKT, and Wnt/β-catenin, while also exerting oncogenic effects through transcriptional regulatory networks. Numerous recently published reports have shown that LINC01094 exerts critical functions during the regulation of malignant cell growth, migrating ability, and invasiveness, thereby controlling cancer cell growth and metastasis. In this review, multiple cancer biology functions of LINC01094 documented in published literature are summarized, aiming to inspire innovations in the management of human malignancies under laboratory and clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/974ccdcd3757f01ef8b5c443955ddd79a4ec4e69" target='_blank'>
              Implications of LINC01094 for human malignancies
              </a>
            </td>
          <td>
            Yan Wang, Yang Yang, Quan-ying Zhang, Ying Zeng, Yilin Cai, Hai-qing Luo, Xiangyong Li
          </td>
          <td>2026-01-27</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="INTRODUCTION
Sarcomas are rare, molecularly diverse malignancies that challenge accurate diagnosis and effective therapy. Japan has implemented nationwide comprehensive cancer genomic profiling (CGP) since 2019, creating an integrated framework that enables precision oncology in routine care.


AREAS COVERED
This review synthesizes the status of cancer genomic medicine in Japan, delineates the genomic landscape of sarcomas, and summarizes clinically actionable alterations. We describe the diagnostic value of CGP - particularly the contribution of RNA sequencing for fusion detection - and outline national infrastructures (expert panels, registries) and platform trials that operationalize genome-matched therapy access. We also examine emerging directions, including liquid biopsy, multi-omics integration, whole-genome approaches, and artificial intelligence, and discuss practical barriers (tissue quality, turnaround time, drug access) with potential solutions.


EXPERT OPINION
Japan's coordinated model - CGP embedded in care, centralized data, and adaptive trial platforms - offers a blueprint for rare cancer precision medicine. Over the next five years, earlier CGP use, broader RNA-enabled diagnostics, and trial-mediated access to targeted agents should expand; coupling liquid biopsy with multi-omics and AI will refine classification and treatment selection. Addressing access and implementation gaps will be critical to translate genomic insights into better outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6d29025ec515e35f9718c74eca1a1a7206ef836" target='_blank'>
              Cancer genome profiling in sarcomas: current status and future perspectives in Japan.
              </a>
            </td>
          <td>
            Tatsusnori Shimoi, Shosuke Kita, K. Yonemori
          </td>
          <td>2026-01-16</td>
          <td>Expert review of anticancer therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer remains a significant challenge in the field of medicine, primarily due to its inherent plasticity and the development of resistance to conventional therapeutic interventions. Genomic mutations and the activation of oncogenes enable cancer cells to resist senescence and apoptosis, leading to uncontrolled growth with harmful consequences. Small peptides are molecules with interesting anti-tumour properties and represent a valid alternative to conventional treatments. Our group has previously identified a class of small peptides bound to the DNA that can be extracted from the chromatin of various tissues, including wheat germ and trout. These peptide pools have been shown to possess interesting antiproliferative and apoptotic properties, and they are associated with cell cycle regulation. However, given the complexity of the extraction process, it is necessary to identify a substrate that will enable a more efficient extraction of these peptides, while also ensuring a composition that is simple to investigate. The present study developed a method for the extraction of this group of peptides from yeast, and the extract was then tested on cancer cells in order to confirm its anti-tumoral properties. The peptides were obtained from chromatin extracted from Saccharomyces cerevisiae cells through alkalisation and purification by gel filtration chromatography. The extract was tested on HeLa cells to verify its effects on vitality and the cell cycle. The data demonstrate that the chromatographic profile of this peptide extract indicates a more basic composition than the pool extracted from other tissues and exhibits comparable antiproliferative properties. The ability to rapidly obtain a biologically active, analytically accessible, and adequately purified fraction from the widely available substrate Saccharomyces cerevisiae represents a significant advance in the study of these DNA-binding peptides.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/862bf194362564eb7cdcf05366b510c07b0d9e77" target='_blank'>
              Extraction of pH-Dependent DNA-Binding Anti-Tumoral Peptides from Saccharomyces cerevisiae
              </a>
            </td>
          <td>
            F. Ragonese, L. Mancinelli
          </td>
          <td>2026-01-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Circular RNA (circRNA) is a type of closed-loop noncoding RNA that is highly resistant to exonucleases. Research has shown that circRNA plays a key regulatory role in the occurrence, progression, immune escape and drug resistance of breast cancer (BC). In terms of cell proliferation and apoptosis, circRNA promotes cell proliferation and inhibits apoptosis through sponge adsorption and binding to the binding protein FUBP1. In the regulation of the tumor microenvironment, circRNA carried by exosomes can form an immune-privileged microenvironment; remodel stromal fibroblasts and promote stromal hardening; enhance angiogenesis and facilitate metastasis. In terms of drug resistance mechanisms, circRNA relieves the inhibition of EGFR, leading to tamoxifen resistance and relieves the inhibition of ABCB1, enhancing drug efflux and causing paclitaxel resistance. Gene intervention targeting circRNA can significantly improve the efficacy of HER2-targeted therapy and estrogen receptor antagonists, suggesting that it is a potential diagnostic marker and therapeutic target. However, the main bottlenecks remain in specific delivery, in vivo stability and long-term safety. In the future, continuous innovation is needed in precise delivery systems, combined multi-target strategies and standardized detection platforms to realize the clinical value of circRNA in BC precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39362d7b4e2da9761cb69824b8e8a8beb1ae9203" target='_blank'>
              The Application of Circular RNA in the Pathogenesis of Breast Cancer
              </a>
            </td>
          <td>
            Wanying Yang
          </td>
          <td>2025-12-19</td>
          <td>MedScien</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The review examines aspects of the interrelationships of the morpho-functional activity of the nucleolus and ribosome biogenesis with the occurrence and progression of malignant neoplasms. To prepare this literature review, an electronic systematic search was conducted for review and research articles in the PubMed database devoted to the study of the nucleolar apparatus of cells in tumor growth. The information was searched in English for the following keywords: nucleolus, nuclear stress, ribosome biogenesis, cancer. The mechanisms of nucleolar stress and the molecular mechanisms leading to disruption of the nucleolar apparatus during tumor growth are highlighted. It has been shown that mutations and changes in the expression of suppressor proteins, oncogenes, and cell cycle regulators lead to impaired nucleolar function and accelerated ribosome biogenesis. An increase in the number and size of nucleoli is an important prognostic factor in malignant neoplasms. Disruption of the function of the nucleolar apparatus and changes in the structure of ribosomes is important in the pathogenesis of cancer development against the background of ribosomopathy and viral diseases. The study of the molecular mechanisms underlying the pathology of the nucleolus, biogenesis and pathology of ribosomes can contribute to the development of new targeted drugs for the treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff62248ca27fa1dd28b5b35369e7344f487de02f" target='_blank'>
              Nucleolus, ribosome biogenesis and tumor growth: literature review
              </a>
            </td>
          <td>
            I. P. Bobrov, A. F. Lazarev, N. Korsikov, A. Dolgatov, A. V. Lepilov, E. S. Dolgatova, A.A. Pimenova, Elena O. Lushnikova, M. A. Bakarev
          </td>
          <td>2025-12-24</td>
          <td>Russian Journal of Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025', '2026'],
    y: [11, 8],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>