<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-03-17 09:28:04 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>30</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>66</td>
          <td>136</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>107</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>318</td>
          <td>136</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>274</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) are circular DNA bodies that play critical roles in tumor progression and treatment resistance by amplifying oncogenes across a wide range of cancer types. ecDNA lack centromeres and are thus not constrained by typical Mendelian segregation, enabling their unequal accumulation within daughter cells and associated increases in copy number. Despite intrinsic links to their oncogenic potential, the fidelity and mechanisms of ecDNA inheritance are poorly understood. Here, we show that ecDNA are protected against cytosolic mis-segregation through mitotic clustering and by tethering to the telomeric and subtelomeric regions of mitotic chromosomes. ecDNA-chromosome tethering depends on BRD4 transcriptional co-activation and mitotic transcription of the long non-coding RNA PVT1, which is co-amplified with MYC in colorectal and prostate cancer cell lines. Disruption of ecDNA-chromosome tethering through BRD4 inhibition, PVT1 depletion, or inhibiting mitotic transcription results in cytosolic mis-segregation, ecDNA reintegration, and the formation of homogeneously staining regions (HSRs). We propose that nuclear inheritance of ecDNA is facilitated by an RNA-mediated physical tether that links ecDNA to mitotic chromosomes and thus protects against cytosolic mis-segregation and chromosomal integration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a63a915cf820dec9dd414321fe61f18977108c0" target='_blank'>
              Mitotic transcription ensures ecDNA inheritance through chromosomal tethering
              </a>
            </td>
          <td>
            Ashley Nichols, Roshan Norman, Yanyang Chen, Yujin Choi, Josefine Striepen, Eralda Salataj, Eléonore Toufektchan, R. Koche, J. Maciejowski
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="In mammalian cells, gene copy number is tightly controlled to maintain gene expression and genome stability. However, a common molecular feature across cancer types is oncogene amplification, which promotes cancer progression by drastically increasing the copy number and expression of tumor-promoting genes. For example, in tyrosine kinase inhibitor (TKI)-resistant lung adenocarcinoma (LUAD), oncogene amplification occurs in over 40% of patients’ tumors. Despite the prevalence of oncogene amplification in TKI-resistant tumors, the mechanisms facilitating oncogene amplification are not fully understood. Here, we find that LUADs exhibit a unique chromatin signature demarcated by strong CTCF and cohesin deposition in drug-naïve tumors, which correlates with the boundaries of oncogene amplicons in TKI-resistant LUAD cells. We identified a global chromatin priming effect during the acquisition of TKI resistance, marked by a dynamic increase of H3K27Ac, cohesin loading, and inter-TAD interactions, which occurs before the onset of oncogene amplification. Furthermore, we have found that the METTL7A protein, which was previously reported to localize to the endoplasmic reticulum and inner nuclear membrane, has a novel chromatin regulatory function by binding to amplified loci and regulating cohesin recruitment and inter-TAD interactions. Surprisingly, we discovered that METTL7A remodels the chromatin landscape prior to large-scale copy number gains. Furthermore, while METTL7A depletion has little effect on the chromatin structure and proliferation of drug-naïve cells, METTL7A depletion prevents the formation and maintenance of TKI resistant-clones, highlighting the specific role of METTL7A as cells are becoming resistant. In summary, we discovered an unexpected mechanism required for the acquisition of TKI resistance regulated by a largely uncharacterized factor, METTL7A. This discovery sheds light into the maintenance of oncogene copy number and paves the way to the development of new therapeutics for preventing TKI resistance in LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864e77e3ffbc341dd750e7bc0c5fd24e1c53b031" target='_blank'>
              Epigenetic priming promotes acquisition of tyrosine kinase inhibitor resistance and oncogene amplification in human lung cancer
              </a>
            </td>
          <td>
            Rebecca M. Starble, Eric G. Sun, Rana Gbyli, Jonathan S. D. Radda, Jiuwei Lu, Tyler B. Jensen, Ning Sun, Nelli Khudaverdyan, Bomiao Hu, M. Melnick, Shuai Zhao, Nitin Roper, Gang Greg Wang, Jikui Song, Katerina Politi, Siyuan Wang, Andrew Xiao
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87febc9c07e2846c41020472ac578cf946a0ca8e" target='_blank'>
              Breakage fusion bridge cycles drive high oncogene number with moderate intratumoural heterogeneity
              </a>
            </td>
          <td>
            Siavash Raeisi Dehkordi, I. Wong, Jing Ni, J. Luebeck, Kaiyuan Zhu, Gino Prasad, Lena Krockenberger, Guanghui Xu, Biswanath Chowdhury, Utkrisht Rajkumar, Ann Caplin, Daniel Muliaditan, Aditi Gnanasekar, C. Coruh, Q. Jin, Kristen Turner, Shu Xian Teo, A. W. Pang, Ludmil B. Alexandrov, Christelle En Lin Chua, F. Furnari, J. Maciejowski, Thomas G Paulson, Julie A. Law, Howard Y. Chang, Feng Yue, Ramanuj DasGupta, Jean J. Zhao, P. Mischel, V. Bafna
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are large, acentric, circular DNA molecules that occur pervasively across many human cancers. EcDNA can drive tumor formation and evolution, contribute to drug resistance, and associate with poor patient survival outcomes. Beyond mediating high copy numbers, the circular topology and dynamic conformational changes of ecDNA disrupt topological domains and rewire regulatory networks, thereby conferring an important role in the transcriptional regulation of oncogenes. Here, we develop ec3D, a computational method for reconstructing the three-dimensional structures of ecDNA and analyzing significant interactions from high-throughput chromatin capture (Hi-C) data. Given a candidate ecDNA sequence and the corresponding whole-genome Hi-C as input, ec3D reconstructs the spatial structure of ecDNA by maximizing the Poisson likelihood of observed interactions. Ec3D’s performance was validated using both simulated ecDNA structures with varying conformations, and Hi-C data from previously-characterized cancer cell lines. Our reconstructions reveal that ecDNAs occupy spherical configurations and mediate unique long-range interactions involved in gene regulation. Through algorithmic innovations, ec3D can resolve complex ecDNA structures with duplicated copies of large genomic segments, identify multi-way interactions, distinguish between interactions arising from direct spatial proximity and secondary interactions resulting from alternative folding patterns or intermolecular (trans) contacts of ecDNA molecules. Our findings provide insights into how the spatial organization of ecDNA may influence gene regulation and contribute to increased oncogene expression. Code availability https://github.com/AmpliconSuite/ec3D">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aebb6e832c80fde421190f885801636b3438e629" target='_blank'>
              Reconstructing the three-dimensional architecture of extrachro-mosomal DNA with ec3D
              </a>
            </td>
          <td>
            Biswanath Chowdhury, Kaiyuan Zhu, Chaohui Li, J. Luebeck, Owen S. Chapman, Katerina Kraft, Shu Zhang, Lukas Chavez, A. Henssen, P. Mischel, Howard Y. Chang, V. Bafna
          </td>
          <td>2025-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Dear Editor Extrachromosomal DNA (ecDNA), a major focal oncogene amplification mode found across cancer, has recently regained attention as an emerging cancer hallmark1,2, with a pervasive presence across cancers3,4. With technical advancements such as high-coverage sequencing and live-cell genome imaging, we can now investigate ecDNA’s behaviors and functions 5-7. However, we still lack an understanding of how to eliminate ecDNA. We observed depletion of cells containing ecDNA during lentiviral but not transposon-based transduction while we sought to investigate the mechanism of ecDNA behavior. This discovery may provide critical information on utilizing a lentiviral system in emerging ecDNA research. Additionally, this observation suggests specific sensitivities for cells with ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db1a49566751259c29d9900a0683040a703d19a" target='_blank'>
              Selective depletion of cancer cells with extrachromosomal DNA via lentiviral infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D. Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) are commonly produced within the nucleus to drive genome dynamics and heterogeneity, enabling cancer cell evolution and adaptation. However, the mechanisms underlying ecDNA biogenesis remain poorly understood. Here using genome-wide CRISPR screening in human cells, we identified the BRCA1-A and the LIG4 complexes mediate ecDNA production. Following DNA fragmentation, the upstream BRCA1-A complex protects DNA ends from excessive resection, promoting end-joining for circularization. Conversely, the MRN complex, which mediates end resection and thus antagonizes the BRCA1-A complex, suppresses ecDNA formation. Downstream, LIG4 conservatively catalyzes ecDNA production in Drosophila and mammals, with patient tumor ecDNA harboring junctions marked by LIG4 activity. Notably, disrupting LIG4 or BRCA1-A in cancer cells impairs ecDNA-mediated adaptation, hindering resistance to both chemotherapy and targeted therapies. Together, our study reveals the roles of the LIG4 and BRCA1-A complexes in ecDNA biogenesis, and uncovers new therapeutic targets to block ecDNA-mediated adaptation for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65472598115e4d60921b909d76b9f2af808705a0" target='_blank'>
              BRCA1-A and LIG4 complexes mediate ecDNA biogenesis and cancer drug resistance
              </a>
            </td>
          <td>
            Oliver W. Chung, Shun Yao, Fu Yang, Ling Wang, Christian G Cerda-Smith, Haley M Hutchinson, Kris C. Wood, Weijia Su, Mustafa Khasraw, Lee Zou, Dale A. Ramsden, Z. Zhang
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c6d858496cf48cc3c25b9e46942a18e35bc4d" target='_blank'>
              A compendium of Amplification-Related Gain Of Sensitivity genes in human cancer
              </a>
            </td>
          <td>
            Verónica Rendo, Michael Schubert, Nicholas Khuu, Maria F Suarez Peredo Rodriguez, Declan Whyte, Xiao Ling, A. van den Brink, Kaimeng Huang, Michelle L Swift, Yizhou He, Johanna Zerbib, Ross Smith, J. Raaijmakers, P. Bandopadhayay, Lillian M. Guenther, Justin H Hwang, A. Iniguez, Susan Moody, J-H. Seo, Elizabeth H Stover, Levi A. Garraway, William C. Hahn, K. Stegmaier, René H. Medema, Dipanjan Chowdhury, Maria Colomé-Tatché, Uri Ben-David, R. Beroukhim, F. Foijer
          </td>
          <td>2025-01-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/effa3cdf47f124f2fc25cc824cb0bcd26047dfd8" target='_blank'>
              Comprehensive dissection of cis-regulatory elements in a 2.8 Mb topologically associated domain in six human cancers
              </a>
            </td>
          <td>
            Christina M. Caragine, Victoria T Le, Meer Mustafa, Bianca Jay Diaz, John A Morris, Simon Müller, A. Méndez-Mancilla, Evan Geller, Noa Liscovitch-Brauer, Neville E. Sanjana
          </td>
          <td>2025-02-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chromosomal instability in gastric cancer cells is associated with the amplification of oncogenes that encode receptor tyrosine kinases (RTKs), such as HER2 and FGFR2; such gene amplification varies from cell to cell and manifests as genetic heterogeneity within tumours. The intratumoural genetic heterogeneity of RTK gene amplification causes heterogeneity in RTK protein expression, which has been suggested to be associated with therapeutic resistance to RTK inhibitors; however, the underlying mechanism is not fully understood. Here, we show that extrachromosomal DNA (ecDNA) causes intratumoural genetic heterogeneity in RTKs and drug resistance due to diverse dynamic changes. We analysed the dynamics of FGFR2 and MYC ecDNA in a gastric cancer cell line after single-cell cloning. Similar to those in parental cells, the copy numbers of FGFR2 and MYC in subclones differed significantly between cells, indicating intraclonal genetic heterogeneity. Furthermore, the ecDNA composition differed between subclones, which affected FGFR2 protein expression and drug sensitivity. Interestingly, clone cells that were resistant to the FGFR2 inhibitor AZD4547 presented diverse changes in ecDNA, including chimeric ecDNA, large ecDNA and increased ecDNA numbers; these changes were associated with high expression and rephosphorylation of FGFR2. Conversely, when resistant clone cells were cultured under conditions that excluded AZD4547, the ecDNA status became similar to that of the original clone cells, and the inhibitory effect on cell growth was restored. Implications: Our results show that dynamic quantitative and qualitative changes in ecDNA can drive the intratumoural genetic heterogeneity of RTKs and resistance to RTK inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895b7aa47450acfbdeb8f8f1748f0e8b623eb4a8" target='_blank'>
              Extrachromosomal DNA dynamics contribute to intratumoural receptor tyrosine kinase genetic heterogeneity and drug resistance in gastric cancer.
              </a>
            </td>
          <td>
            Kazuki Kanayama, Hiroshi Imai, Ryotaro Hashizume, Chise Matsuda, Eri Usugi, Y. Hirokawa, Masatoshi Watanabe
          </td>
          <td>2025-02-19</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="MYC-driven (MYC+) cancers are aggressive and often fatal. MYC dysregulation is a key event in these cancers, but overexpression of MYC alone is not always enough to cause cancer. Plasmocytoma Variant Translocation 1 (PVT1), a long non-coding RNA (lncRNA) adjacent to MYC on chromosome 8 is a rearrangement hotspot in many MYC+ cancers. In addition to being co-amplified with MYC, the genomic rearrangement at PVT1 involves translocation, which has had obscure functional consequences. We report that translocation at the PVT1 locus cause asymmetric enrichment of 5’-PVT1 and loss of 3’-PVT1. Despite being classified as a non-coding RNA, the retained 5’ region of PVT1 generates a circular RNA (CircPVT1) that codes for the novel peptide we call Firefox (FFX). FFX augments AKT signaling and synergistically activates MYC and mTORC1 in these cells. Further, the 3’ end of PVT1, which is lost during the translocation, codes for a tumor-suppressing micropeptide we named as Honeybadger (HNB). We demonstrate that HNB interacts with KRAS and disrupts the activation of KRAS effectors. Loss of HNB leads to activation of RAS/MAPK signaling pathway, and enhances MYC stability by promoting phosphorylation of MYC at Ser62. These findings identify PVT1 as a critical node that synchronizes MYC, AKT, and RAS-MAPK activities in cancer. Our study thus identifies a key mechanism by which rearrangements at the PVT1 locus activate additional oncogenic pathways that synergize with MYC to exacerbate the aggressiveness of MYC+ cancers. This newfound understanding explains the poor prognosis associated with MYC+ cancers and offers potential therapeutic targets that could be leveraged in treatment strategies for these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e076955c4c71d3d95422db7f1fcfc733c5d61e3" target='_blank'>
              Synergistic RAS-MAPK and AKT Activation in MYC-Driven Tumors via Adjacent PVT1 Rearrangements
              </a>
            </td>
          <td>
            Ashutosh Tiwari, Utkarsha Paithane, Jordan Friedlein, Kojiro Tashiro, Olivier Saulnier, Karina Barbosa, Quang Trinh, Bryan Hall, Shrawantee Saha, Aditi Soni, Takuma Nakashima, Andrey Bobkov, Lynn Fujimoto, Rabi Murad, Svetlana Maurya, Mayank Saraswat, Shahab Sarmashghi, Joshua T. Lange, Sihan Wu, M. Masihi, Srija Ghosh, Gazal Hemmati, Owen S. Chapman, Liam D Hendrikse, Brian James, J. Luebeck, Tanja Eisemann, Theophilos Tzaridis, Deepak Rohila, Robyn Leary, Jyotika Varshney, Badrinath R. Konety, S. Dehm, Yasuhiko Kawakami, R. Beroukhim, D. Largaespada, Lincoln Stein, Lukas Chavez, Hiromichi Suzuki, William A. Weiss, Jianhua Zhao, Aniruddha J. Deshpande, R. Wechsler-Reya, Michael D Taylor, Anindya Bagchi
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7394db232e99965882ed6981c1decbce7cc434ab" target='_blank'>
              8q24 derived ZNF252P promotes tumorigenesis by driving phase separation to activate c-Myc mediated feedback loop
              </a>
            </td>
          <td>
            Tianyu Qu, Chang Zhang, Xiyi Lu, Jiali Dai, Xuezhi He, Wei Li, Liang Han, Dandan Yin, Erbao Zhang
          </td>
          <td>2025-02-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db22625a85ea34b5a5adadae1847c161e783d66" target='_blank'>
              MYC ecDNA promotes intratumour heterogeneity and plasticity in PDAC.
              </a>
            </td>
          <td>
            Elena Fiorini, A. Malinova, Daniel Schreyer, Davide Pasini, Michele Bevere, Giorgia Alessio, Diego Rosa, Sabrina D'Agosto, Luca Azzolin, Salvatore Milite, S. Andreani, F. Lupo, L. Veghini, Sonia Grimaldi, S. Pedron, Monica Castellucci, C. Nourse, R. Salvia, G. Malleo, A. Ruzzenente, A. Guglielmi, M. Milella, R. Lawlor, C. Luchini, A. Agostini, Carmine Carbone, Christian Pilarsky, A. Sottoriva, Aldo Scarpa, D. Tuveson, Peter Bailey, V. Corbo
          </td>
          <td>2025-03-12</td>
          <td>Nature</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="Abstract Recent research reveals that eukaryotic genomes form circular DNA from all parts of their genome, some large enough to carry whole genes. In organisms like yeast and in human cancers, it is often observed that extrachromosomal circular DNA (eccDNA) benefits the individual cell by providing resources for rapid cellular growth. However, our comprehension of eccDNA remains incomplete, primarily due to their transient nature. Early studies suggest they arise when DNA breaks and is subsequently repaired incorrectly. In this review, we provide an overview of the evidence for molecular mechanisms that lead to eccDNA formation in human cancers and yeast, focusing on nonhomologous end joining, alternative end joining, and homologous recombination repair pathways. Furthermore, we present hypotheses in the form of molecular eccDNA formation models and consider cellular conditions which may affect eccDNA generation. Finally, we discuss the framework for future experimental evidence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd036a4aeb41767279a21c8043d9cb8d21781a98" target='_blank'>
              Molecular mechanisms of extrachromosomal circular DNA formation
              </a>
            </td>
          <td>
            Rasmus A B Eugen-Olsen, J. M. Hariprakash, V. Oestergaard, Birgitte Regenberg
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="High grade serous ovarian cancer (HGSOC) is the most lethal gynecologic malignancy, killing more than 9,000 women each year in the United States alone. Nearly 80% of patients with HGSOC tumors will experience recurrence within 5 years, but little is known about the mechanisms that drive this process. Intratumor heterogeneity is believed to be a key feature of recurrence and resistance in HGSOC tumors, with early studies reporting diverse and complex mechanisms of the seeding of metastatic sites, including metastasis reseeding the primary tumor. Few studies have investigated temporal changes to clonality and structural variants through disease recurrence and the development of chemoresistance as surgical debulking is not frequently performed at this later stage. We performed multi-omic profiling in paired chemo-naive and chemoresistant tumors from 32 HGSOC patients to investigate the mutational and genomic landscape of disease progression. Somatic mutation profiles were largely conserved through disease progression. Mutational burdens did not significantly differ across recurrence but were driven by homologous recombination repair deficiency status. Clonal composition and dynamics were measured through variant allele frequency alterations as tumors progressed from primary chemo-naïve to recurrent chemo-resistant tumors. A novel candidate driver gene, MDC1, from the homologous recombination repair pathway, was significantly mutated and over-represented in patients with homologous recombination proficient tumors, with somatic mutations clustered in a single exon. Tumor evolution and phylogeny revealed that few changes in clonal abundance and complexity occur across the disease course in these patients. Taking structural variants into account, homologous recombination repair proficient (HRP) tumors tend to be polyclonal while homologous recombination repair deficient (HRD) tumors tend to be monoclonal, accompanied by a longer progression-free survival than the HRP patients. Three distinct classes of tumors were identified by structural variant signature analysis: tumors defined by DNA losses, tumors defined by DNA gains, and tumors defined by copy number neutral changes, which were largely defined by HRD status. Each class displayed distinct regions of the chromosome that were frequently affected by large scale SV events (>5Mb). Although no regions were frequently altered in recurrent tumors, GO analysis revealed that recurrent tumors have a significantly reduced immune response, which was not seen in the primary tumors. Ultra long read sequencing validated a majority of the SVs identified in short read sequencing and identified additional SVs undetected by short reads. These analyses identify that the phenotype of high grade serous ovarian tumors as defined by mutation and clonality profiles is established early in disease development and remain largely unchanged through chemotherapy and recurrence. This, when considered with the significant inter-patient heterogeneity identified in HGSOC, demonstrates the need for personalized therapies based on tumor profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ce536f8f2260934bcc6a48fd33209bc44e0997d" target='_blank'>
              The mutation and clonality profile of genomically unstable high grade serous ovarian cancer is established early in tumor development and conserved throughout therapy resistance
              </a>
            </td>
          <td>
            Michael Diaz, Nicole Gull, Pei-Chen Peng, Kruttika Dabke, Jessica Baker, Jennifer Sun, Juan Alvarado, Benjamin Bastin, Nimisha Mazumdar, C. Santiskulvong, Andrew J. Li, Beth Y. Karlan, B. Rimel, Sarah J. Parker, S. Gayther, Michelle R. Jones
          </td>
          <td>2025-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="While there is compelling genetic evidence for the role of enhancers in regulating promoter activity even over large genomic distances, it is unclear to what extent physical proximity with the respective promoters is involved. Here we combine DNA fluorescence in situ hybridization (FISH) with confocal and stimulated emission depletion (STED) super resolution microscopy to investigate enhancer-promoter (E-P) distances at selected loci (Dppa3, Nanog, Dnmt3a, Sox2, Prdm14) that contain multiple enhancers and undergo transcriptional changes at the transition from naive to primed pluripotency in mouse embryonic stem cells. Automated distance measurements in thousands of cells revealed that both pairwise and multiway E-P conformations undergo only small (Δ median distance = ∼22 nm) changes, despite large, up to 1500-fold changes in transcription, arguing against lasting changes in genome architecture at these loci. As transcription often occurs in transient bursts in a small fraction of cells, we performed RNA FISH to identify actively transcribed alleles. We found that in actively transcribed Dppa3 alleles (<10%) the median E-P distances are significantly shorter (Δ median distance = ∼90 nm) than in non-transcribed ones. These data are consistent with a transient spatial interaction of enhancers and promoter during the initiation of transcription. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ff38d00d6901c7e7b148aae70803c82b0c36574" target='_blank'>
              Nanoscale Dynamics of Enhancer-Promoter Interactions during Exit from Pluripotency
              </a>
            </td>
          <td>
            G. Stumberger, David Hörl, Clemens Steinek, H. Leonhardt, H. Harz
          </td>
          <td>2025-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Chromosome conformation in mammals is closely related to gene regulation. Within topologically associating domains, where genomic contacts are enriched, genes tend to show correlated expression across tissues and conditions, suggesting domain-wide mechanisms coregulating multiple genes, such as enhancer sharing or local histone mark spreading. At the single-cell level, where transcription occurs in sporadic bursts, transcriptional coordination has been observed between proximal genes, but how the local folding of mammalian chromosomes influences gene coregulation in cis at individual alleles remains unclear. Using single-molecule microscopy, we imaged nascent transcription from three adjacent genes located around a strong contact insulation site at the FOS locus, during the estrogen response in human breast cancer cells. To interpret this data, we developed two new analysis approaches to dissect the sources of (co)variation in gene activities: one to separate allele-extrinsic, allele-intrinsic, and gene-autonomous components; and another one to quantify the contributions of burst co-occurrence and burst size correlations. We find that transcriptional variability is largely gene-autonomous, yet correlations between genes display distinct patterns and occur almost exclusively in cis. Correlations are stronger between proximal and less insulated genes. However, unexpectedly, substantial correlations also occur across the strong insulation site and, under certain conditions, two proximal genes on the same side can exhibit uncorrelated burst occurrences. By disentangling burst co-occurrence from burst size correlations, we reveal transcriptional patterns suggesting two distinct coregulatory mechanisms influenced by local chromosome folding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8149ddb3736f68e357841e3d3a66278f75e1232" target='_blank'>
              Transcriptional coregulation in cis around a contact insulation site revealed by single-molecule microscopy
              </a>
            </td>
          <td>
            Maciej A. Kerlin, Ilham Aboulfath-Ladid, Julia Roensch, Chloé Jaubert, A. Battistella, K. J. Borgman, A. Coulon
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Lampbrush chromosomes are giant meiotic bivalents in growing oocyte nuclei that have served as a classic model system for studying chromatin organization and RNA synthesis for over a century. Despite their importance, the molecular mechanisms underlying lampbrush chromosomes formation and their distinctive chromomere-loop architecture have remained poorly understood. Here, we provide the first comprehensive genomic, cytological, and biophysical analysis of lampbrush chromosome organization by integrating single-cell Hi-C, RNA-seq, NOMe-seq, FISH mapping, and chromatin simulations. Single-nucleus Hi-C analysis revealed CTCF-independent contact domains with stable boundaries defined by transcription units in a convergent orientation. Сontact domains identified through Hi-C analysis correspond to insulated chromomeres in lampbrush chromosomes. Small transcriptionally inactive contact domains surrounded by transcription units in the diverged orientation form ‘chromatin knots’, which are often detached from the chromosome axis. Transcription loops frequently manifest as a ‘cross’ pattern with reduced contacts within chromatin domains. Integrative analysis of the whole-genome data uncovers the mechanisms underlying lampbrush chromosome structure, revealing how hypertranscription modulates chromatin stiffness and repositions SMC complexes to establish the distinctive chromomere-loop organisation. Biophysical modeling through polymer simulation reproduces key features of lampbrush chromosomes, including transcription loop formation, chromomere compaction, and insulation patterns. These findings offer a unifying framework for understanding the remarkable chromatin architecture of lampbrush chromosomes and their transcription-dependent organization. Highlights - First integration of single-cell Hi-C, RNA-seq, NOMe-seq and microscopy methods uncovers molecular mechanisms underlying lampbrush chromosome architecture. - Hi-C reveals contact patterns corresponding to lampbrush chromomeres and transcription loops, validated through BAC-based FISH mapping. - Lampbrush chromosomes are segmented into contact domains formed via a CTCF-independent mechanism, with boundaries coinciding with convergently oriented gene pairs. - Hypertranscription shapes lampbrush chromosome through multiple mechanisms, increasing stiffness and decreasing compaction of transcribed units, generating outward pressure, pushing transcription loops away from the chromosome axis, and repositioning SMC complexes to form transcription-dependent domains with stable boundaries. - Polymer simulations including these mechanisms replicate key features of LBC organization, supporting the proposed model of lampbrush chromosomes formation. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46e642c22825b99c48d9f70a1c1f9236c18b4d5e" target='_blank'>
              The 3D genomics of lampbrush chromosomes highlights the role of active transcription in chromatin organization
              </a>
            </td>
          <td>
            T. Lagunov, M. Gridina, A. Nurislamov, T. Kulikova, A. Maslova, V. Konstantinov, A. Popov, A. Krasikova, V. Fishman
          </td>
          <td>2025-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Originating from, but independent of, linear chromosomes, extrachromosomal DNA (ecDNA) exists in a more active state of transcription and autonomous replication. It plays a crucial role in the development of malignancies and therapy resistance. Since its discovery in eukaryotic cells more than half a century ago, the biological characteristics and functions of ecDNA have remained unclear due to limitations in detection methods. However, recent advancements in research tools have transformed ecDNA research. It is believed that ecDNA exhibits greater activity in the abnormal amplification of oncogenes, thereby driving cancer progression through their overexpression. Notably, compared to linear DNA, ecDNA can also function as a genomic element with regulatory roles, including both trans- and cis-acting functions. Its critical roles in tumorigenesis, evolution, progression, and drug resistance in malignant tumors are increasingly recognized. This review provides a comprehensive summary of the evolutionary context of ecDNA and highlights significant progress in understanding its biological functions and potential applications as a therapeutic target in malignant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf3896d81e701493b4d97e132a04a0cb6710489" target='_blank'>
              Insights into Extrachromosomal DNA in Cancer: Biogenesis, Methodologies, Functions, and Therapeutic Potential.
              </a>
            </td>
          <td>
            Xudong Mao, Guocheng Rao, Gonghui Li, Shihan Chen
          </td>
          <td>2025-02-13</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) presents a promising target for cancer therapy; however, its spatial-temporal diversity and influence on tumor evolution and the immune microenvironment remain largely unclear. We apply computational methods to analyze ecDNA from whole-genome sequencing data of 595 urothelial carcinoma (UC) patients. We demonstrate that ecDNA drives clonal evolution through structural rearrangements during malignant transformation and recurrence of UC. This supports a model wherein tumors evolve via the selective expansion of ecDNA-bearing cells. Through multi-regional sampling of tumors, we demonstrate that ecDNA contributes to the evolution of multifocality and increased intratumoral heterogeneity. EcDNA is present in 36% of UC tumors and correlates with an immunosuppressive phenotype and poor prognosis. Single-cell RNA sequencing analyses reveal that ecDNA+ malignant cells exhibit diminished expression of major histocompatibility complex class I molecules, enabling them to evade T-cell immunity. Finally, we show that sequencing of urinary sediment-derived DNA has excellent specificity in detecting ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27eaa871f09220399d901baadcfefa6c53b148cd" target='_blank'>
              Spatial-Temporal Diversity of Extrachromosomal DNA Shapes Urothelial Carcinoma Evolution and Tumor-Immune Microenvironment.
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, Huiying Tao, Yanfen Zhu, Xiaolong Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez-Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xi Xiang, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, J. Luebeck, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-03-11</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="The mutational landscape of phylogenetically ultraconserved elements (UCEs), especially those in noncoding DNAs (ncUCEs), and their functional relevance in cancers remain poorly characterized. Here, we perform a systematic analysis of whole-genome and in-house targeted UCE sequencing datasets from more than 3000 patients with cancer of 13,736 UCEs and demonstrate that ncUCE somatic alterations are common. Using a multiplexed CRISPR knockout screen in colorectal cancer cells, we show that the loss of several altered ncUCEs significantly affects cell proliferation. In-depth functional studies in vitro and in vivo further reveal that specific ncUCEs can be enhancers of tumor suppressors (such as ARID1B) and silencers of oncogenic proteins (such as RPS13). Moreover, several miRNAs located in ncUCEs are recurrently mutated. Mutations in miR-142 locus can affect the Drosha-mediated processing of precursor miRNAs, resulting in the down-regulation of the mature transcript. These results provide systematic evidence that specific ncUCEs play diverse regulatory roles in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50d62f517b45fea38f094fa47dd4b743a633f64" target='_blank'>
              The mutational landscape and functional effects of noncoding ultraconserved elements in human cancers
              </a>
            </td>
          <td>
            R. Bayraktar, Yitao Tang, M. Dragomir, Cristina Ivan, Xinxin Peng, L. Fabris, Jianhua Zhang, A. Carugo, S. Aneli, Jintan Liu, M. Chen, Sanjana Srinivasan, Iman Sahnoune, E. Bayraktar, K. Akdemir, Meng Chen, P. Narayanan, Wilson Huang, L. Ott, A. K. Eterovic, O. Villarreal, M. M. Mohammad, M. Peoples, Danielle M Walsh, J. A. Hernandez, Margaret B Morgan, K. Shaw, Jennifer S Davis, David Menter, Constantine S Tam, Paul Yeh, S. Dawson, L. Rassenti, T. Kipps, Tanja Kunej, Z. Estrov, S. Joosse, Luca Pagani, C. Alix-Panabières, Klaus Pantel, Alessandra Ferajoli, Andrew Futreal, I. Wistuba, Milan Radovich, S. Kopetz, Michael J Keating, Giulio F. Draetta, John S. Mattick, Han Liang, George A Calin
          </td>
          <td>2025-02-19</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="Breast cancer is the most prevalent neoplasm in women. ER+ (Luminal subtype), representing over 70% of breast tumors, is a genetically diverse group. Structural and Numerical-Chromosomal instability initiates tumor development and is recognized as the primary driver of genetic alteration in luminal breast tumors. Genomic instability refers to the increased tendency of cancer cells to accumulate genomic alterations during cell proliferation. The cell cycle check-point response to constant and stable genomic alterations in tumor cells drives this process. The impact of CNV patterns and aneuploidies in cell cycle and proliferation perturbation has recently been highlighted by scientists in Luminal breast tumors. The impact of chromosomal instability on cancer therapy and prognosis is not a new concept. Still, the degree of emerging genomic instability leads to prognosis alteration following cell cycle deregulation by chromosomal instability could be predicted by CNVs-based reclassification of breast tumors. In this review, we try to explain the effect of CIN in the cell cycle that ended with genomic instability and altered prognosis and the impact of CIN in decision-making for a therapy strategy for patients with luminal breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06bef9c6cdff104c948a88f734e3a20c0747386c" target='_blank'>
              A new insight into the impact of copy number variations on cell cycle deregulation of luminal-type breast cancer
              </a>
            </td>
          <td>
            Amir Mahdi Khamaneh, Davoud Jafari-Gharabaghlou, Khalil Ansarin, Pouya Pazooki, Zahra Akbarpour, B. Naghili, Nosratollah Zarghami
          </td>
          <td>2025-02-12</td>
          <td>Oncology Reviews</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Transposable elements (TEs) are reactivated in tumors and serve as significant contributors to tumor transcriptomic complexity and heterogeneity. However, their repetitive nature and diverse transcriptional forms have hindered efforts to characterize TE-derived transcripts independent of host gene contexts. Here, using our self-developed splicing-junction analysis tool ASJA, we systematically interrogated TE transcription across 32 cancer types, identifying transcripts autonomously initiated from and exclusively spliced among TEs with high TE content, termed TEtrans. Our analysis revealed 5,361 TEtrans exhibiting pan-cancer prevalence, tumor-specific expression and features of mature transcripts, including canonical splicing, 5’ caps and polyA tails. Mostly unannotated and enriched in intergenic regions, TEtrans are predominantly derived from primate-specific classes with conserved splice pattern. TEtrans burden demonstrates heterogeneous associations with prognosis and immune activity across cancers, while their expression can be epigenetically dual-regulated by stemness- and inflammation-associated transcription factors. Functional studies uncovered that TEtrans could act as oncogenes, exemplified by tsTE1, a HERVH-derived transcript that promotes colorectal cancer proliferation by enhancing TOP1-mediated DNA supercoil relaxation. Remarkably, ∼7.2% of TEtrans are identified to encode tumor neoantigens, including viral proteins and unannotated peptides, which are shared among patients and validated by proteogenomic analysis. These TEtrans-derived neoantigens are immunogenic both in vitro and in vivo, and exceed neoepitopes from genomic alterations in abundance per tumor, particularly in cancers with low mutational burden. Collectively, as a gene-independent form of TE-derived transcripts, TEtrans represents a unique source of oncogenes and tumor neoantigens, expanding the functional repertoire of TEs in cancer biology and offering new avenues for therapeutic targeting. Highlights TEtrans are tumor-specific transcripts with conserved splicing patterns, predominantly derived from primate TEs in intergenic regions. TEtrans expression is epigenetically activated and dually regulated by stemness/inflammation transcription factors. TEtrans function as oncogenes, exemplified by tsTE1 enhancing TOP1-mediated DNA relaxation. TEtrans encode tumor neoantigens that elicit CD8+ T cell responses, providing targets for immunotherapy, especially in low-mutation tumors. Abstract figure">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f90153db4362d7e931500fbc74364483bb05472a" target='_blank'>
              Transcriptomic splicing analysis reveals a gene-independent form of TE reactivation in tumors representing unrecognized source of oncogenes and neoantigens
              </a>
            </td>
          <td>
            Ziteng Li, Yichao Bao, Qin Li, Hongwu Yu, He-Yun Zhang, Yifan Wen, Yu Yang, Jingjing Zhao, Peng Lin, Yan Li, Zhixiang Hu, Xichun Hu, Xiaodong Zhu, Shenglin Huang
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72bc6fce345e0628d21c6c9477e460943646693b" target='_blank'>
              Spatial deconvolution from bulk DNA methylation profiles determines intratumoral epigenetic heterogeneity
              </a>
            </td>
          <td>
            Binbin Liu, Yumo Xie, Yu Zhang, Guannan Tang, Jinxin Lin, Ze Yuan, Xiaoxia Liu, Xiaolin Wang, Meijin Huang, Yanxin Luo, Huichuan Yu
          </td>
          <td>2025-01-23</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tumors with low expression of Interferon-Stimulated Genes (ISG) and Antigen Presentation (AP) genes respond relatively poorly to current immunotherapies. One of the early hallmarks of cancer is DNA hypomethylation in genomic repeat regions, resulting in the expression of normally silenced endogenous “viral” elements. Such epigenetic changes have the potential to augment anti-tumor immune responses as well as reduce tumor cell fitness through the generation of aberrant nucleic acid species (NAS) and consequent activation of NAS-sensing pathways. Therefore, tumor evolution should favor additional selective events that suppress NAS generation, possibly yielding specific therapeutic vulnerabilities. Here, we show that the Lysine Demethylase 5 (KDM5) family of epigenetic regulatory enzymes suppress R-loop formation in genomic repeat regions in cancer cells. We find that KDM5 inhibition in luminal breast cancer cells results in R-loop-mediated DNA damage, reduced cell fitness and an increase in ISG and AP signatures as well as cell surface Major Histocompatibility Complex (MHC) class I, mediated by RNA:DNA hybrid activation of the CGAS/STING pathway. KDM5 inhibition does not result in DNA damage or activation of the CGAS/STING pathway in normal breast epithelial cells, suggesting that KDM5 inhibitors may enable a wide therapeutic window in this setting, as compared to STING agonists or Type I Interferons. These findings provide new insights into the interplay between epigenetic regulation of genomic repeats, R-loop formation, innate immunity, and cell fitness in the context of cancer evolution and therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd1c02bf99dc94856478bd198e131107c59bd8c" target='_blank'>
              KDM5 demethylases suppress R-loop-mediated “viral mimicry” and DNA damage in breast cancer cells
              </a>
            </td>
          <td>
            Lena Lau, Kurt W Henderson, Ahu Turkoz, Sara Linker, Dörte Schlesinger, Brad Townsley, Brian Egan, Shoba Ragunathan, Robert Rollins, Xianju Bi, Zhijian J. Chen, Oleg Brodsky, Clifford Restaino, Murali Gururajan, Kristen Jensen-Pergakes, Anders Malarstig, Chames Kermi, Paul C Moore, Marie Classon
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Significance Chemotherapeutics effectively kill cancer cells but can also induce mutations in surviving cells, leading to therapy resistance and secondary malignancies. We analyzed the DNA rearrangements caused by doxorubicin in murine HER2+ breast cancer organoids, focusing on genomic structural variants (SVs). Using single-cell sequencing, we revealed extensive therapy-induced SVs and genomic instability in a subset of posttreatment cells. Additionally, by advancing computational methods to analyze nucleosome occupancy (NO) in different cell types of the mammary lineage, we link SVs to gene expression changes that could potentially lead to breast cancer progression. This research underscores the need for strategies to minimize drug-induced genomic damage to improve treatment efficacy and patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95047a549d97c358dede6e2ed75d74511cbd0c06" target='_blank'>
              Structural variant and nucleosome occupancy dynamics postchemotherapy in a HER2+ breast cancer organoid model
              </a>
            </td>
          <td>
            Maja Starostecka, Hyobin Jeong, Patrick Hasenfeld, Eva Benito-Garagorri, Tania Christiansen, Catherine Stober Brasseur, Maise Gomes Queiroz, Marta Garcia Montero, Martin Jechlinger, J. Korbel
          </td>
          <td>2025-02-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="Summary The epigenetic state of chromatin, gene activity and chromosomal positions are interrelated in plants. In Arabidopsis thaliana, chromosome arms are DNA‐hypomethylated and enriched with the euchromatin‐specific histone mark H3K4me3, while pericentromeric regions are DNA‐hypermethylated and enriched with the heterochromatin‐specific mark H3K9me2. We aimed to investigate how the chromosomal location affects epigenetic stability and gene expression by chromosome engineering. Two chromosomal inversions of different sizes were induced using CRISPR/Cas9 to move heterochromatic, pericentric sequences into euchromatic regions. The epigenetic status of these lines was investigated using whole‐genome bisulfite sequencing and chromatin immunoprecipitation. Gene expression changes following the induction of the chromosomal inversions were studied via transcriptome analysis. Both inversions had a minimal impact on the global distribution of histone marks and DNA methylation patterns, although minor epigenetic changes were observed across the genome. Notably, the inverted chromosomal regions and their borders retained their original epigenetic profiles. Gene expression analysis showed that only 0.5–1% of genes were differentially expressed genome‐wide following the induction of the inversions. CRISPR/Cas‐induced chromosomal inversions minimally affect epigenetic landscape and gene expression, preserving their profiles in subsequent generations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8552d926f37133e02cf503023e99c3ac17866bc9" target='_blank'>
              Epigenetic state and gene expression remain stable after CRISPR/Cas‐mediated chromosomal inversions
              </a>
            </td>
          <td>
            Solmaz Khosravi, Rebecca Hinrichs, Michelle Rönspies, Reza Haghi, Holger Puchta, Andreas Houben
          </td>
          <td>2025-01-29</td>
          <td>The New Phytologist</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="In eukaryotic cells, gene-distal regulatory elements (REs) facilitate long-range gene regulation, ensuring cell type-specific transcriptional programs. This mechanism is frequently disrupted in cancer, often driven by transcription factors (TFs) that serve as targets for cancer therapy. However, targeting these TFs can lead to acquired resistance mechanisms that are not fully understood. We demonstrate that mesothelioma cancer cells, dependent on the oncogenic driver TF family TEAD, develop resistance to a pan-TEAD inhibitor and revert to an evolutionarily ancient, promoter-centric gene regulatory mechanism to recover gene expression following TEAD inhibition. Base-pair-resolution 3D chromatin conformation mapping reveals that RE-promoter interactions are disrupted in resistant cells, despite epigenetic and transcriptomic recovery. Mechanistically, in resistant cells, TF complexes, including resistance-specific FOSL1 and KLF4, preferentially bind and enhance promoter activity to recover gene expression, rendering distal REs dispensable. Our findings highlight promoter elements and promoter-specific TFs as potential therapeutic targets using a model of drug-resistant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fe445b4115593745bb7783131bb805e9f85aeb" target='_blank'>
              Promoter-centric gene regulation in drug-resistant cancer
              </a>
            </td>
          <td>
            Vasumathi Kameswaran, Sayantanee Paul, Daniel Le, Jonathan Hoover, Luke Y. Zhao, Alissa D. Guarnaccia, Thijs J. Hagenbeek, Jessica M. Lund, Ana Xavier-Magalhães, Minyi Shi, Julia Lau, Marco De Simone, Yuxin Liang, Anwesha Dey, Z. Modrusan, Bence Daniel
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a75bdacf15f78eabb80ad03155a6f7ee66350fd9" target='_blank'>
              Genomic alterations and transcriptional phenotypes in circulating free DNA and matched metastatic tumor
              </a>
            </td>
          <td>
            N. Takahashi, L. Pongor, Shivam P. Agrawal, Mariya Shtumpf, Ankita Gurjar, V. Rajapakse, Ahmad Shafiei, Christopher W. Schultz, Sehyun Kim, Diana Roame, Paula Carter, R. Vilimas, Samantha Nichols, Parth Desai, W. Figg, Mohammad Bagheri, V. B. Teif, Anish Thomas
          </td>
          <td>2025-02-25</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 Nearly half of the human genome is composed of transposable elements (TEs), which were once considered “junk” DNA due in part to their repetitive and non-coding characteristics. However, TEs are now emerging as important regulators of gene activity, especially when they escape typical epigenetic silencing mechanisms like DNA methylation and histone modifications. When activated, TEs can contribute sequences that are exploited as disruptors of genomic stability or novel gene regulatory elements, making them important factors in both development and disease, including cancer. In cancer, these previously silenced elements can act as alternative promoters of oncogenes, leading to the production of TE-chimeric transcripts in a process called “onco-exaptation”. While these oncoexaptation events are reported to be widespread across many cancers, only a few TE-chimeric transcripts have been validated as active contributors to cancer progression. Additionally, it remains unclear whether these TE-chimeric transcripts are merely a consequence of the inherent genomic instability in cancer or whether they can initiate cancer. To address these gaps, we designed a cDNA overexpression library incorporating our previously identified pan-cancer TE-chimeric transcript candidates. This library is used in early passage (EP) BJ fibroblast, a cell line known for its transformation potential in the well-established Weinberg transformation model, which allows us to systematically investigate these onco-exaptation events during cancer progression. Our initial analyses indicate that most TE-chimeric oncogenes are not expressed in EP fibroblast, but are activated in subsequent transformation stages following hTERT, SV40 and HRas induction, underscoring the model viability. We are currently using this model to distinguish TE-chimeric transcripts that act as drivers (TE-chimeric oncogenes) from those requiring previous vulnerabilities, such as telomere inactivation by hTERT or p53 and Rb inactivation by SV40. Ultimately, this work will identify novel biomarkers and therapeutic targets, enhance insights into gene regulation, and provide opportunities for early detection and intervention. These findings have the potential to improve cancer treatment and patient outcomes.
 Citation Format: Wesley N Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang. The functional role of transposable element-oncogene chimeric transcript during cancer progression [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9f7a8875ef14376de973322ef301e4c4ddf4aab" target='_blank'>
              Abstract B005: The functional role of transposable element-oncogene chimeric transcript during cancer progression
              </a>
            </td>
          <td>
            Wesley N. Saintilnord, Yixin Wu, Yonghao Liang, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bd6b6473526b044dda123e9fa9d72ac0fc045ca" target='_blank'>
              A dual role of Cohesin in DNA DSB repair
              </a>
            </td>
          <td>
            Michael Fedkenheuer, Yafang Shang, Seolkyoung Jung, Kevin Fedkenheuer, Solji Park, Davide Mazza, Robin Sebastian, Hiroyuki Nagashima, D. Zong, Hua Tan, S. Jaiswal, Haiqing Fu, Anthony Cruz, Supriya V Vartak, Jan Wisniewski, Vittorio Sartorelli, John J O'Shea, L. Elnitski, A. Nussenzweig, M. Aladjem, Fei Meng, Rafael Casellas
          </td>
          <td>2025-01-20</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfacdbb6d5b3609b2de9ba21e1d307534c0157c2" target='_blank'>
              SETD2 loss-of-function uniquely sensitizes cells to epigenetic targeting of NSD1-directed H3K36 methylation
              </a>
            </td>
          <td>
            Ryan T Wagner, R. Hlady, Xiaoyuan Pan, Liguo Wang, Sungho Kim, Xia Zhao, Louis Y El Khoury, Shafiq Shaikh, Jian Zhong, Jeong-Heon Lee, J. Grembecka, Tomasz Cierpicki, Thai H Ho, Keith D. Robertson
          </td>
          <td>2025-02-05</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="Prostate cancer is characterized by profound heterogeneity in its clinical trajectory. While genomic heterogeneity has been well-characterized, epigenomic heterogeneity remains less understood. To fill this gap, we compiled 2,149 multi-ancestric prostate methylomes spanning normal tissue through localized disease of varying grades to poly-metastatic disease, most with multi-omic DNA and/or RNA characterization. We identify four subtypes that varied by stage, grade and mutational subtype. We identify extensive interplay between DNA ploidy and DNA methylation, with transcriptional consequences that vary across driver-genes. Each major prostate cancer driver gene mutation triggers a specific epigenetic dysregulation, and we define a set of 14 reusable models that accurately predict clinico-molecular features of a prostate cancer from DNA methylation data. Specific epigenetic features predict disease aggression, including metastasis, with epigenomic and genomic features synergizing to optimize predictions. These results define a complex interplay between tumour genetics and epigenetics that converges to modify gene-expression programs, and subsequent clinical presentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0b1fd2e12bcf45ef4e0b8a0b897b112e846447" target='_blank'>
              The Landscape of Prostate Tumour Methylation
              </a>
            </td>
          <td>
            J. Arbet, Takafumi N. Yamaguchi, Yu-Jia Shiah, Rupert Hugh-White, Adriana Wiggins, Jieun Oh, Tsumugi Gebo, Adrien Foucal, R. Lesurf, Chol-Hee Jung, Rachel M. A. Dang, Raag Agrawal, J. Livingstone, A. Salcedo, C. Yao, S. M. G. Espiritu, Kathleen E. Houlahan, Fouad Yousif, Lawrence E Heisler, M. Fraser, Amar U. Kishan, Alejandro Berlin, Melvin LK Chua, T. H. van der Kwast, Christopher M Hovens, Rob Bristow, P. Boutros
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="PURPOSE
Monitoring disease progression in patients with high-grade gliomas (HGGs) is challenging due to treatment-related changes on imaging and the requirement for neurosurgical intervention to obtain diagnostic tissue. DNA junctions in HGGs often amplify oncogenes, making these DNA fragments potentially more abundant in blood than monoallelic mutations. Herein, we piloted a cell-free DNA approach for disease detection in plasma of patients with HGGs by leveraging patient-specific DNA junctions associated with oncogene amplifications.


EXPERIMENTAL DESIGN
Whole genome sequencing of grade 3 or 4 IDH-mutant or wild-type astrocytomas was utilized to identify amplified junctions. Individualized qPCR assays were developed using patient-specific primers designed for the amplified junction. ctDNA levels containing these junctions were measured in patient plasma samples.


RESULTS
Unique amplified junctions were evaluated by individualized semi-quantitative PCR assays in presurgical plasma of 18 patients, 15 with tumor-associated focal amplifications and three without. High-copy number junctions were robustly detected in plasma of 14/15 patients (93.3%) with amplified junctions and none of the controls. Changes in junction abundance correlated with disease trajectory in serial plasma samples from five patients, including increased abundance of amplified junctions preceding radiographic disease progression.


CONCLUSION
In patients with grade 3 or 4 astrocytomas who had tumor-associated amplifications, patient-specific amplified junctions were successfully detected in assayed plasma from most patients. Longitudinal analysis of plasma samples correlated with disease trajectory, including cytoreduction and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6be79817ee79c2803379821d2392c413312af98" target='_blank'>
              Personalized tumor-specific amplified DNA junctions in peripheral blood of patients with glioblastoma.
              </a>
            </td>
          <td>
            Mohamed F Ali, C. Riviere-Cazaux, Sarah H Johnson, Rebecca Salvatori, Alan R Penheiter, James B Smadbeck, Stephen J. Murphy, F. Harris, Lex F McCune, Lucas P Carlstrom, Michael T Barrett, F. Kosari, Leila A Jones, Cristiane M Ida, M. Borad, B. Bendok, Alfredo Quiñones-Hinojosa, Alyx B Porter, Maciej M Mrugala, K. Jaeckle, P. Anastasiadis, S. Kizilbash, John C Cheville, David M Routman, Terry C Burns, G. Vasmatzis
          </td>
          <td>2025-02-28</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="One fundamental yet open question is how eukaryotic chromosomes fold into segregated territories, a process essential for gene transcription and cell fate. Through analyzing Hi-C and chromatin-tracing DNA-FISH data, we identify long-range chromo skeleton loop structures that span over 100 Mb, extending beyond the reach of loop extrusion models. Spatial density analyses point to assembly formation independent of major nuclear structures. A subset of genomic loci serves as nucleation centers, driving loop clustering. These complexes are highly stable, as shown by live-cell imaging with sequence-specific fluorescent labeling, and biophysical model analyses reveal a multivalent binding mechanism. Our findings suggest a redundant, distributed cluster mechanism that ensures robustness across cell types and against mutations, guiding both chromosome compaction and the formation of smaller-scale chromosomal structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04b5ac6a8c671d63e4c9bd85fcca4ea46155b48" target='_blank'>
              Long-range genomic loci stochastically assemble into combinatorial forms of chromosome skeleton
              </a>
            </td>
          <td>
            Jingyu Zhang, Siyuan Wang, Simon C Watkins, Jianhua Xing
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Two decades after the initial report on increased micronuclei in human chronic liver disease (CLD) and hepatocellular carcinoma (HCC), their role in HCC development is still poorly understood. Here we show that micronuclei in hepatocytes trigger hepatic immune response and promote HCC development via an increased level of extrachromosomal circular DNA (eccDNA). Livers from a CLD model (Mcl1Δhep mice) show increased micronuclei in parallel to eccDNA. Circular sequencing confirms higher eccDNA levels in micronuclei compared to primary nuclei. We developed nuclei-segregated DNA fiber (NuSeF) assay to show that micronuclei are more susceptible to replication stress, showing increased replication fork slowing. By comparing different murine liver disease models, high eccDNA is correlated with increased tumor incidence. EccDNA is a strong immunostimulant and promotes a crosstalk between hepatocytes and immune cells through the cGAS-STING pathway. Deletion of Sting1 in Mcl1Δhep mice reduces immune cell chemotaxis, as well as tumor incidence. Our findings suggest that eccDNA from micronuclei mediates inflammation-driven liver carcinogenesis in CLD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/563897a8a27ca1930e83755c188ad3a2e992e96a" target='_blank'>
              Extrachromosomal circular DNA promotes inflammation and hepatocellular carcinoma development
              </a>
            </td>
          <td>
            Lap Kwan Chan, Juanjuan Shan, Elias Rodriguez-Fos, Marc E. Healy, Peter Leary, Rossella Parrotta, Nina Desboeufs, Gabriel Semere, Nadine Wittstruck, A. Henssen, Achim Weber
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7170c44494d1fc7e8ae358052e6bacf3d4c1c009" target='_blank'>
              Recruitment and rejoining of remote double-strand DNA breaks for enhanced and precise chromosome editing
              </a>
            </td>
          <td>
            Mingyao Wang, Pengchong Fu, Ziheng Chen, Xiangnan Wang, Hanhui Ma, Xuedi Zhang, Guanjun Gao
          </td>
          <td>2025-03-11</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Ductal carcinoma in situ (DCIS) harbors genomic copy number changes reflecting early events in advancing invasive breast cancer, and changes in specific cell states related to cancer initiation and progression. DNA methylation is a covalent modification to cytosines with cell type specificity, is stable in archival samples, and provides most of the specificity in multiple noninvasive pan-cancer screens. However, the creation of single-cell DNA methylomes is technically challenging and analysis is immature, leaving cell type and state level interpretation understudied. This has led to a gap in knowledge: it is not understood how the methylome reacts to gene dosage changes brought on by early copy number events, and it is not known how long these potential methylation changes persist. To this end, we have created and optimized a high throughput assay to measure these methylation changes at single-cell level, pairing our analysis with a multimodal view of these early copy number events. Here we describe three major steps in our work on elucidating methylome response to early breast cancer events: 1) We have generated methylome libraries for breast cancer cell lines with known large scale copy number alterations and validated our copy number calling in methylomes with those of gold-standard single-cell whole genome libraries. 2) We have generated thousands of single cell methylomes on normal breast samples attained from reduction surgeries paired with our groups’ work on the Human Breast Cell Atlas project. We were able to identify all major cell types of the human breast using methylation data for the first time. We used this data set to generate high-quality pseudobulk cell type specific methylomes. 3) We have profiled 5 DCIS and 1 synchronous invasive ductal carcinoma sample with subclonal copy number events, paired with single-cell transcriptomics. Through this we identified subclones in our methylome libraries and validated these through inferred copy number changes in our single-cell transcriptome libraries. We have uncovered thousands of hypomethylated regions enriched for specific transcription factor motifs, and widespread methylation changes across DCIS subclones. Notably many differential methylation changes occur outside of the genomic boundaries of copy number effects, suggesting distal changes by transcription factor gene dosage, chromatin perturbation, or tumor microenvironment interactions. These data will serve as a basis for future analyses into long-term follow through of archival samples and expand our understanding of this understudied epigenomic mark and its importance in early cancer evolution.
 Citation Format: Ryan M Mulqueen, Mariam Mosaad, Shanshan Bai, Jiahong Li, Emi Sei, Bora Lim, Savitri Krishnamurthy, Alastair Thompson, Nicholas Navin. Copy number analysis of single-cell methylomes reveal distinct subclone methylation in early breast cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fa4f976dba878ba71cab253948818a9b0e43066" target='_blank'>
              Abstract A006: Copy number analysis of single-cell methylomes reveal distinct subclone methylation in early breast cancer
              </a>
            </td>
          <td>
            Ryan M Mulqueen, Mariam Mosaad, S. Bai, Jiahong Li, Emi Sei, Bora Lim, Savitri Krishnamurthy, Alastair Thompson, Nicholas Navin
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="An aggressive subtype of acute myeloid leukemia (AML) is caused by enhancer hijacking resulting in MECOM overexpression. Several chromosomal rearrangements can lead to this: the most common (inv(3)/t(3;3)) results in a hijacked GATA2 enhancer, and there are several atypical MECOM rearrangements involving enhancers from other hematopoietic genes. The set of enhancers which can be hijacked by MECOM can also be hijacked by BCL11B. Enhancer deregulation is also a driver of oncogenesis in a range of other malignancies. The mechanisms of enhancer deregulation observed in other cancer types, including TAD boundary disruptions and the creation of de novo (super-) enhancers, may explain overexpression of MECOM or other oncogenes in AML without enhancer hijacking upon translocation. Gaining mechanistic insight in both enhancer deregulation and super-enhancer activity is critical to pave the way for new treatments for AML and other cancers that are the result of enhancer deregulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc88804fb71dad9d4f7cceb8ac265d5c25968818" target='_blank'>
              Mechanisms of enhancer-driven oncogene activation.
              </a>
            </td>
          <td>
            Joyce Vriend, Ruud Delwel, Dorien Pastoors
          </td>
          <td>2025-01-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Prostate cancer (PCa) is a prevalent male cancer with high survival rates, except in advanced or metastatic stages, for which effective treatments are lacking. Metastatic PCa involves complex mechanisms including loss of tumor suppressor genes and DNA repair molecules, which impacts therapy responses. We have reanalyzed data from a CRISPR/Cas9 genome wide screening previously performed to identify essential regulators of invasive abilities of the metastatic cell line DU145 identifying SYCP3 as a regulator of metastatic invasion. Subsequent analyses of tumor samples demonstrated that SYCP3 expression is frequently upregulated in PCa tumors from patients in advanced stages. Furthermore, SYCP3 genetic depletion significantly reduced the invasive and migratory abilities of DU145 cells and increased their adhesion capacity. Additionally, and due to the implication of SYCP3 on DNA repair processes, we have analyzed the role of SYCP3 on the cellular response to radiotherapy (RT) and found that its depletion induced RT resistance, suggesting a role for SYCP3 in DNA damage response and genomic instability. All these data support a role for SYCP3 in PCa metastasis and provides opportunities for personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016d79d4fcdaa6ad45de27d2da94398a7826f930" target='_blank'>
              CRISPR screening reveals SYCP3 as a key driver of metastasis in prostate cancer
              </a>
            </td>
          <td>
            Maria Rodrigo-Faus, Inés del Monte-García, Marina Hermosilla-Trespaderne, Alicia Gordo-Vega, Natalia Vidal, Javier Puente, Melchor Saiz-Pardo, Ángel M Cuesta, Hui-Qi Qu, H. Hakonarson, Almudena Porras, Daniel Sanchez-Parcerisa, Paloma Bragado, Á. Gutierrez-Uzquiza
          </td>
          <td>2025-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Genome stability is safeguarded by a finely orchestrated cascade of events that collectively represent the DNA damage response (DDR). In eukaryotes, the DDR operates within the dynamic chromatin landscape, where the interplay between DNA repair factors, chromatin remodeling, replication, transcription, spatial genome organization, and cytoskeletal forces is tightly coordinated. High-resolution studies have unveiled chromatin alterations spanning multiple scales, from localized kilobase-level changes to megabase-scale reorganization, which impact chromatin’s physical properties and enhance the mobility of damaged regions. Leveraging this knowledge could pave the way for innovative therapeutic strategies, particularly in targeting chromatin dynamics to destabilize cancer cells selectively. This review, focusing on DNA double-strand breaks (DSBs), sheds light on how chromatin undergoes dynamic modifications in response to damage and how these changes influence the DDR at both local and global levels, offering a glimpse into how nuclear architecture contributes to the delicate balance between genome stability and adaptability and highlighting the importance of exploring these interactions in the context of cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a627a3749979e4f865f8823e54ada96686be7874" target='_blank'>
              Interplay and Dynamics of Chromatin Architecture and DNA Damage Response: An Overview
              </a>
            </td>
          <td>
            Susanna Ambrosio, Anna Noviello, Giovanni Di Fusco, Francesca Gorini, Anna Piscone, Stefano Amente, Barbara Majello
          </td>
          <td>2025-03-11</td>
          <td>Cancers</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Oncogene activation in normal untransformed cells induces DNA replication stress and creates a dependency on DNA Damage Response (DDR) mechanisms for cell survival. Different oncogenic stimuli signal via distinct mechanisms in every cancer setting. The DDR is also pathologically re-programmed and deployed in diverse ways in different cancers. Because mutant KRAS is the driver oncogene in 90% of Pancreatic Ductal Adenocarcinomas (PDAC), here we have investigated DDR mechanisms by which KRAS-induced DNA replication stress is tolerated in normal human pancreatic epithelial cells (HPNE). Using a candidate screening approach, we identify TRIP13 as a KRASG12V-induced mRNA that is also expressed at high levels in PDAC relative to normal tissues. Using genetic and pharmacological tools, we show that TRIP13 is necessary to sustain ongoing DNA synthesis and viability specifically in KRASG12V-expressing cells. TRIP13 promotes survival of KRASG12V-expressing HPNE cells in a Homologous Recombination (HR)-dependent manner. KRASG12V-expressing HPNE cells lacking TRIP13 acquire hallmark HR-deficiency (HRD) phenotypes including sensitivity to inhibitors of Trans-Lesion Synthesis (TLS) and Poly-ADP Ribose Polymerase (PARP). Established PDAC cell lines are also sensitized to intrinsic DNA damage and therapy-induced genotoxicity following TRIP13-depletion. Taken together our results expose TRIP13 as an attractive new and therapeutically-tractable vulnerability of KRAS-mutant PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89fa9b68b58350edbc77be5948633226c0fa98f1" target='_blank'>
              TRIP13 protects pancreatic cancer cells against intrinsic and therapy-induced DNA replication stress
              </a>
            </td>
          <td>
            J. R. Anand, G. Droby, Sayali Joseph, Urvi Patel, Xingyuan Zhang, Jeffrey A. Klomp, C. Der, Jeremy E. Purvis, Samuel C. Wolff, Jessica Bowser, Cyrus Vaziri
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>122</td>
        </tr>

        <tr id="Background Transcriptional activation of otherwise repressed endogenous retroelements (RTEs) is a hallmark of cancer, shaping tumour progression and immunogenicity by multifaceted, yet incompletely understood mechanisms. Methods We used an extended pan-cancer transcriptome assembly to identify potential effects of RTEs on the genes near or within which they have integrated. These were subsequently verified in test cases by further analysis of transcriptional profiles in cancer patient data, and by in vitro studies involving restoration of gene activity, and proliferation and migration assays in cancer cell lines. Results We report that cancer-specific transcriptional activation of RTEs causes frequent reduction or loss of gene function. Exonisation and alternative splicing of RTEs creates non-functional RNA and protein isoforms and derepressed RTE promoter activity initiates antisense transcription, both at the expense of the canonical isoforms. Contrary to theoretical expectation, transcriptionally activated RTEs affect genes with established tumour-promoting function, including the common essential RNGTT and the lung cancer-promoting CHRNA5 genes. Furthermore, the disruptive effect of RTE activation on adjacent tumour-promoting genes is associated with slower disease progression in clinical data, whereas experimental restoration of gene activity enhances tumour cell in vitro growth and invasiveness Conclusions These findings underscore the gene-disruptive potential of seemingly innocuous germline RTE integrations, unleashed only by their transcriptional utilisation in cancer. They further suggest that such metastable RTE integrations are co-opted as sensors of the epigenetic and transcriptional changes occurring during cellular transformation and as executors that disrupt the function of tumour-promoting genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf39ff907245282864eb90768249daa905476758" target='_blank'>
              Retroelement co-option disrupts the cancer transcriptional programme
              </a>
            </td>
          <td>
            Jane Loong, Rachael Thompson, Callum Hall, Laura Doglio, Judith Pape, G. Kassiotis
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a highly heterogeneous gastrointestinal malignancy. Despite significant advances in molecular targeted therapies for CRC in recent years, the increase in the overall survival rates for CRC patients remains limited. Therefore, there is an urgent need to explore novel drug targets. Herein, we show that heparin binding growth factor (HDGF) is highly expressed in CRC, and that its overexpression is associated with a poor disease-free interval. Additionally, we reveal that HDGF knockout reduces proliferation, migration, and invasion, while enhancing apoptosis in CRC cells, thereby validating HDGF as a potential therapeutic target for CRC. Mechanistically, we found that HDGF modulates DNA damage response and, by recruiting C-terminal binding protein-interacting protein (CtIP), it facilitates homologous recombination repair to influence CRC drug sensitivity. Furthermore, we propose that HDGF may serve as a recognition protein for H3K36me3, participating in the repair of damaged transcriptionally active genes, thus maintaining genomic stability in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f394455983264a89c22ede32239ab6721086ed77" target='_blank'>
              HDGF Knockout Suppresses Colorectal Cancer Progression and Drug Resistance by Modulating the DNA Damage Response
              </a>
            </td>
          <td>
            Riya Su, Qin Wang, Qun Hu, Wendurige, Kexin Li, Changshan Wang, Liang Tao
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Centromeres are an important part of chromosomes which direct chromosome segregation during cell division. Their modifications can therefore explain the unusual mitotic and meiotic behaviour of certain chromosomes, such as the germline-restricted chromosome (GRC) of songbirds. This chromosome is eliminated from somatic cells during early embryogenesis and later also from male germ cells during spermatogenesis. Although the mechanism of elimination is not yet known, it is possible that it involves a modification of the centromeric sequence on the GRC, resulting in problems with the attachment of this chromosome to the mitotic or meiotic spindle and its lagging during anaphase, which eventually leads to its elimination from the nucleus. However, the repetitive nature and rapid evolution of centromeres make their identification and comparative analysis across species and chromosomes challenging. Here, we used a combination of cytogenetic and genomic approaches to identify the centromeric sequence of two closely related songbird species, the common nightingale (Luscinia megarhynchos) and the thrush nightingale (L. luscinia). We found a 436-bp satellite repeat present in the centromeric regions of all regular chromosomes, making it a strong candidate for the centromeric repeat. This centromeric repeat was highly similar between the two nightingale species. Interestingly, hybridization of the probe to this satellite repeat on meiotic spreads suggested that this repeat is missing on the GRC. Our results indicate that the change of the centromeric sequence may underlie the unusual inheritance and programmed DNA elimination of the GRC in songbirds.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cc2a0f86985b1b169e70e74228b3c32d8f2ac99" target='_blank'>
              Germline-restricted chromosome of songbirds has different centromere compared to regular chromosomes
              </a>
            </td>
          <td>
            Jakub Rídl, D. Dedukh, Zuzana Halenková, Stephen A. Schlebusch, V. Beneš, Mireia Osuna Lopez, T. Osiejuk, Francisco J Ruiz-Ruano, Alexander Suh, Tomáš Albrecht, Jiří Reif, Radka Reifová
          </td>
          <td>2025-01-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70f85dd49412748b7fb2dbe319b405d34092c20d" target='_blank'>
              Multi-omics analysis reveals the panoramic picture of TOP2A in pan-cancer
              </a>
            </td>
          <td>
            Jin Zhang, Tianxiao Yang, Kenie Wang, Jie Pan, Junjun Qiu, Shengnai Zheng, Xuesong Li, Guanghao Li
          </td>
          <td>2025-02-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Chromosome instability is highly prevalent in cancer and drives large scale chromosomal imbalances, known as aneuploidies. However, how aneuploidy contributes to tumorigenesis remains difficult to study due to the vast numbers of genes affected. To address these limitations, we have developed a CRISPR-Knock Out and Activation Linked Assay (CRISPR-KOALA), which enables systematic high-throughput bidirectional genetic screens in immune-competent mouse models of cancer. Using CRISPR-KOALA we screened the mouse orthologs of all 3,752 genes residing on the ten most frequently altered human chromosome arms in basal-like breast cancer (BLBC), which to date is the largest bidirectional screen performed in vivo. Our screen identified 90 cancer driver genes, the vast majority of which have hitherto unknown functions in cancer. These genes drive distinct signalling pathways, reflecting the high degree of BLBC heterogeneity. Individual manipulation of the identified cancer driver genes completely overcomes the need for copy number alterations (CNAs) in p53-mutant BLBC mouse models. Mechanistically, we uncover PLGRKT as a potent oncogene and show that its tumor-promoting activity is associated with the creation of highly stress-resistant mitochondria that promote tumor cell survival, rather than through its canonical role of regulating the extracellular matrix. Overall, our work reveals that arm-level CNAs can function to select specific driver genes to promote heterogenous biological processes.
 Citation Format: Khalid N Al-Zahrani, Ellen R Langille, Andreea Obersterescu, Christopher Lowden, Katie Teng, Lauren Caldwell, David Cook, Miguel Pérez-Castro, Cynthia Chiu, Alexander Bahcheli2, Ricky Tsai, Jacob Berman, Kin Chan, Linkang Zhang, K.W. Annie Bang, Michael Parsons, Adele Lopes, Jocelyn Nurtanto, E. Idil Temel, Iosifina Fotiadou, Julien Dessapt, Hartland Jackson, Sean Egan, Jüri Reimand, Jeffrey Wrana, Daniel Schramek. Decoding aneuploidy: Identifying drivers and therapeutic targets in recurrent breast cancer copy number alterations [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr A022.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471fa22c73f7c1a8083ff473dd937339ee492d0c" target='_blank'>
              Abstract A022: Decoding aneuploidy: Identifying drivers and therapeutic targets in recurrent breast cancer copy number alterations
              </a>
            </td>
          <td>
            Khalid N. Al-Zahrani, Ellen Langille, A. Obersterescu, Christopher Lowden, Katie Teng, Lauren Caldwell, David Cook, Miguel Pérez-Castro, Cynthia H Chiu, Alec Bahcheli, R. Tsai, Jacob Berman, Kin Chan, Linkang Zhang, K.W. Annie Bang, Michael Parsons, Adele Lopes, Jocelyn Nurtanto, E. I. Temel, Iosifina Fotiadou, Julien Dessapt, Hartland Jackson, Sean E Egan, J. Reimand, Jeffrey Wrana, Daniel Schramek
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/560c8effcee249e562abe69a29bf3bcf7c2bff96" target='_blank'>
              Unravelling single-cell DNA replication timing dynamics using machine learning reveals heterogeneity in cancer progression
              </a>
            </td>
          <td>
            Joseph M. Josephides, Chun-Long Chen
          </td>
          <td>2025-02-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Activation of synthetic centromeres on chromosome 4 in maize leads to its breakage and formation of trisomic fragments called neochromosomes. A limitation of neochromosomes is their low and unpredictable transmission rates due to trisomy. Here we report that selecting for dicentric recombinants through male crosses uncovers stabilized chromosome 4 fission events which split it into 4a-4b complementary chromosome pairs, where 4a carries a native centromere and 4b a synthetic one. The cells rapidly stabilized chromosome ends by de novo telomere formation and the new centromeres spread among genes without altering their expression. When both 4a and 4b chromosomes were present in a homozygous state, they segregated through meiosis indistinguishably from wild-type, and gave rise to healthy plants with normal seed set. This work leverages synthetic centromeres to engineer chromosome fission, effectively raising the diploid chromosome number of maize from 20 to 22. TEASER Using synthetic centromeres, we divide one maize chromosome into two chromosomes, and show that both chromosomes are fully functional.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffa270994f0b80ef6d572c92b9c5ee380872d96e" target='_blank'>
              Increased maize chromosome number by engineered chromosome fission
              </a>
            </td>
          <td>
            Yibing Zeng, Mingyu Wang, J. Gent, R. Dawe
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Kinase-related gene fusion and point mutations play pivotal roles as drivers in cancer, necessitating optimized, targeted therapy against these alterations. The efficacy of molecularly targeted therapeutics varies depending on the specific alteration, with great success reported for such therapeutics in the treatment of cancer with kinase fusion proteins. However, the involvement of actionable alterations in solid tumors, especially regarding kinase fusions, remains unclear. Therefore, in this study, we aimed to compare the number of actionable alterations in patients with tyrosine or serine/threonine kinase domain fusions, mutations, and copy number alterations (CNAs). We analyzed 613 patients with 40 solid cancer types who visited our division between June 2020 and April 2024. Furthermore, to detect alterations involving multiple-fusion calling, we performed comprehensive genomic sequencing using FoundationOne® companion diagnostic (F1CDx) and FoundationOne® Liquid companion diagnostic (F1LCDx). Patient characteristics and genomic profiles were analyzed to assess the frequency and distribution of actionable alterations across different cancer types. Notably, 44 of the 613 patients had fusions involving kinases, transcriptional regulators, or tumor suppressors. F1CDx and F1LCDx detected 13 cases with kinase-domain fusions. We identified 117 patients with kinase-domain mutations and 58 with kinase-domain CNAs. The number of actionable alterations in patients with kinase-domain fusion, mutation, or CNA (median [interquartile range; IQR]) was 2 (1–3), 5 (3–7), and 6 (4–8), respectively. Patients with kinase fusion had significantly fewer actionable alterations than those with kinase-domain mutations and CNAs. However, those with fusion involving tumor suppressors tended to have more actionable alterations (median [IQR]; 4 [2–9]). Cancers with kinase fusions exhibited fewer actionable alterations than those with kinase mutations and CNAs. These findings underscore the importance of detecting kinase alterations and indicate the pivotal role of kinase fusions as strong drivers of cancer development, highlighting their potential as prime targets for molecular therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e5249feb71c9f79a3c5d96e6debe6951cd92ec6" target='_blank'>
              Comparison of actionable alterations in cancers with kinase fusion, mutation, and copy number alteration
              </a>
            </td>
          <td>
            S. Suzuki, T. Akahane, A. Tanimoto, M. Higashi, I. Kitazono, M. Kirishima, M. Nishigaki, T. Ikeda, S. Kanemitsu, J. Nakazawa, E. Akahane, H. Nishihara, Kimiharu Uozumi, M. Yoshimitsu, K. Ishitsuka, S.-i. Ueno
          </td>
          <td>2025-01-23</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a0977541bd5b827628ce81b0938c0df52c0a295" target='_blank'>
              Proteogenomic analysis reveals adaptive strategies for alleviating the consequences of aneuploidy in cancer.
              </a>
            </td>
          <td>
            Jan-Eric Bökenkamp, Kristina Keuper, Stefan Redel, Karen Barthel, Leah Johnson, Amelie Becker, A. Wieland, M. Räschle, Zuzana Storchová
          </td>
          <td>2025-02-10</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="In this study, we show that comprehensive DNA profiling using whole genome sequencing (WGS) can offer novel insights into ovarian carcinoma (OC). First, using WGS of 185 OC with HRDetect profiling, a mutational signature-based algorithm for homologous recombination repair deficiency (HRD), we found that patients with HRDetect-high OC had significantly better overall survival, with a median of 6.2 years compared to 4.1 years for those with lower score OC. Second, the rest of the OCs are not simply non-HRD - some carry distinguishing rearrangement signatures that are strongly prognostic. Third, in an independent cohort of 77 patients, we demonstrate predictive value; HRDetect-high patients exhibited better responses to PARP inhibitors (PARPi) (median progression-free survival of 11.1 months versus 7.1 months; overall response rate of 54% versus 22.5%). Critical next steps include fine-tuning therapeutic vulnerabilities associated with the diverse rearrangement signatures, and harnessing the full potential of WGS for OC clinical trial stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc713b72314da7faa35c83a1d8de687d85095e55" target='_blank'>
              HRDetect in Ovarian Carcinoma: Stratification and Therapeutic Implications.
              </a>
            </td>
          <td>
            K. Banda, H. R. Davies, Y. Kumar, Y. Memari, A. Degasperi, M. Radke, I. Rodriguez, T. A. Gooley, F. Menghi, T. Harding, K. Lin, E. T. Liu, S. Nik-Zainal, E. M. Swisher
          </td>
          <td>2025-03-06</td>
          <td>None</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Telomerase reverse transcriptase (hTERT) overexpression, a hallmark of most cancers, drives tumorigenesis by enabling limitless replicative potential. Direct targeting of hTERT is challenging, necessitating alternative strategies. Through genome-wide synthetic dosage lethality (SDL) screening in cancer models, including patient-derived organoids, we identify FTSJ3, an RNA 2’-O-methyltransferase, as a critical vulnerability in hTERT-overexpressing cells. FTSJ3 methylates telomeric repeat-containing RNA (TERRA), a modification essential for recruiting SUV39H1 to telomeric ends to mediate H3K9 trimethylation and establish stable heterochromatin. Loss of FTSJ3 disrupts this cascade, impairing H3K9 trimethylation, HP1-alpha recruitment, and telomeric heterochromatin maintenance. Notably, this reveals an unexpected dependency on TERRA methylation for telomeric heterochromatin stability in hTERT-driven cancers. Non-malignant cells, lacking telomerase activity and de novo telomere repeat synthesis, are unaffected by FTSJ3 suppression. Our findings establish the FTSJ3/TERRA/SUV39H1 axis as a critical mechanism supporting telomeric heterochromatin stability in hTERT-driven cancers. This telomere-directed epigenetic strategy provides a robust framework for translational therapeutic innovation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f728a2708c415752990d708812f36363b77d5df" target='_blank'>
              Epigenetic Control of TERRA by FTSJ3 is Critical for Telomerase-Driven Cancers
              </a>
            </td>
          <td>
            Jared D.W. Price, Frederick S. Vizeacoumar, Omar Abuhussein, Vincent Maranda, Yue Zhang, H. Adachi, Liliia Kyrylenko, A. Rangel-Pozzo, He Dong, Lihui Gong, Prachi Walke, Ashtalakshmi Ganapathysamy, Connor Denomy, T. Freywald, Renuka Dahiya, Hussain Elhasasna, Anjali Saxena, Jeff P. Vizeacoumar, Hardikkumar Patel, Karthic Rajamanickam, Kathryn Nguyen, D. M. de Oliveira, Mary Lazell-Wright, Alain Morejon Morales, Aanchal Aggarwal, Jia Lin Xu, N. Alli, E. P. Munhoz, Peng Gao, Jayme Salsman, Dinesh K Dahiya, Cristina Gonzalez-Lopez, Patricia Thibault, Michael Levin, Graham Dellaire, Nicholas Jette, Gary Groot, Anand Krishnan, Shahid Ahmed, Christopher Eskiw, Khaled H. Barakat, Yuliang Wu, Ronald A. DePinho, Sabine Mai, Yi-Tao Yu, J. Wong, A. Freywald, Franco J. Vizeacoumar
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Disseminated tumor cells (DTCs) contribute to metastasis formation by adapting to the hostile environmental conditions encountered at the seeded secondary organs. This phenotypic plasticity is supported by the epigenetic reprogramming of cis-regulatory elements (CREs), representing a hallmark of metastatic cells. DTCs can enter a reversible state of cancer dormancy, resulting from their ability to enter quiescence after mitosis and escape immune surveillance. In this respect, mitosis may represent a window of opportunity to increase DTCs’ fitness by priming CREs, ensuring the reactivation of a dormancy-related transcriptional program while preserving the epigenetic information. By profiling the chromatin changes occurring during the early steps of mitosis, we found that DTCs experience fluctuations in chromatin accessibility at CREs, which are bookmarked by pro-metastatic TFs. Among these, the pro-dormancy TF SOX9 remained associated with mitotic chromatin, facilitating transcriptional reactivation upon exiting mitosis. By dissecting the consequences of degrading SOX9 before mitosis, we found that its priming of dormancy-related CREs, combined with the recruitment of the Mediator kinase module, is instrumental for metastatic cells to enter quiescence and escape NK-mediated immune surveillance. These findings indicate that cancer dormancy is primed during mitosis by pro-metastatic TFs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0606469a945ca0abe5515873ee7687ae21b3a877" target='_blank'>
              Metastatic cell plasticity is maintained by SOX9 function during mitosis in breast cancer
              </a>
            </td>
          <td>
            Sven Beyes, Daniela Michelatti, Maria Luce Negri, Leonardo Morelli, Enrico Frigoli, L. Fagnocchi, A. Zippo
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) analysis offers a powerful, non-invasive approach to cancer diagnostics and monitoring by revealing tumor-specific genomic and epigenetic alterations. Here, we demonstrate the versatility of MeD-seq, a methylation-dependent sequencing assay, for comprehensive cfDNA analysis, including methylation profiling, chromosomal copy number (CN) alterations, and tumor fraction (TF) estimation. MeD-seq-derived CN profiles and TF estimates from 38 colorectal cancer with liver metastases (CRLM) and 5 ovarian cancer patients were highly comparable to shallow whole-genome sequencing (sWGS) validating our approach. These findings establish MeD-seq as a robust and cost-effective platform for detecting cancer-specific signals directly from plasma without prior tissue-based information. Future work should expand its application to other cancer types, solidifying MeD-seq as a versatile tool for minimally-invasive cfDNA profiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8a8bea4181cf5b880c2869962e79354192f059" target='_blank'>
              From Background to Signal: Simultaneously derived Copy Numbers from Methylation-Dependent Sequencing cell-free DNA
              </a>
            </td>
          <td>
            Daan M. Hazelaar, R. Boers, J. Boers, Vanja de Weerd, M. Jansen, Henk M. Verheul, Cornelis Verhoef, Joost Gribnau, John W M Martens, Stavros Makrodimtris, S. Wilting
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive cancer characterized by profound desmoplasia and cellular heterogeneity, which cannot be fully resolved using traditional bulk sequencing approaches. To understand the contribution of this heterogeneity to PDAC biology, we analyzed a large cohort of primary human PDAC samples (n = 62), profiling 443,451 single cells and 53,236 spatial transcriptomic spots using a combined single-cell RNA sequencing and spatial transcriptomics approach. Our analysis revealed significant intratumoral heterogeneity, with multiple genetically distinct neoplastic clones co-existing within individual tumors. These clones exhibited diverse transcriptional states and subtype profiles, challenging the traditional binary classification of PDAC into basal and classical subtypes; instead, our findings support a transcriptional continuum influenced by clonal evolution and spatial organization. Additionally, these clones each interacted uniquely with surrounding cell types in the tumor microenvironment. Phylogenetic analysis uncovered a rare but consistent classical-to-basal clonal transition associated with MYC amplification and immune response depletion, which were validated experimentally, suggesting a mechanism driving the emergence of a more aggressive basal clonal phenotype. Spatial analyses further revealed dispersed clones enriched for epithelial-to-mesenchymal transition (EMT) activity and immune suppression, correlating with metastatic potential and colonization of lymph node niches. These dispersed clones tended to transition toward a basal phenotype, contributing to disease progression. Our findings highlight the critical role of clonal diversity, transcriptional plasticity, and TME interactions in shaping human PDAC biology. This work provides new insights into the molecular and spatial heterogeneity of PDAC and offers potential avenues for therapeutic intervention targeting clonal evolution and the mechanisms driving metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f29b4b1fd404eda30da076650d1b340e06687894" target='_blank'>
              Clonal Heterogeneity in Human Pancreatic Ductal Adenocarcinoma and Its Impact on Tumor Progression
              </a>
            </td>
          <td>
            Despoina Kalfakakou, Daniel C. Cameron, Emily A. Kawaler, Motoyuki Tsuda, Lidong Wang, Xiaohong Jing, Cristina Hajdu, Dylan L. Tamayo, Yoona Shim, Amanda Ackermann, Daniel Weissinger, Hayley Zimny, R. Hernandez, Matthew Beier, D. Dimartino, Peter Meyn, Kalina Rice, Shanmugapriya Selvaraj, Cynthia Loomis, Adriana Heguy, Amanda W. Lund, Rosalie C. Sears, T. Welling, Igor Dolgalev, A. Tsirigos, Diane M. Simeone
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Purpose
The epidermal growth factor receptor (EGFR) is a therapeutic target with confirmed clinical efficacy for several cancer types. We aimed to identify EGFR aberrations and their associations with other genomic alterations in patients with metastatic diseases of various cancers.


Materials and Methods
We used real-world data from the next-generation sequencing (NGS) of 3,286 patients with metastatic cancer at the Samsung Medical Center. We analyzed the distribution of EGFR amplification, mutation, and fusion, as well as their correlations with microsatellite instability (MSI), tumor mutation burden (TMB), and other gene aberrations.


Results
A total of 3,286 patients were tested using NGS of a panel covering 523 cancer-related genes (TSO500, Illumina) as part of clinical practice between October 2019 and October 2022. Patients with lung cancer and gliomas were not included in the analysis. Of the 3,286 patients, 175 (5.3%) had EGFR amplification, 38 (1.2%) had EGFR mutations, and 8 (0.2%) had EGFR fusion. All 175 patients with EGFR amplifications had microsatellite-stable (MSS) tumors, but 102 had co-amplifications in other cancer-related genes, and 78 had mutations with clinical significance (tier I/II). Among the 38 patients with EGFR mutations, three (8%) showed MSI-high status, and eleven (29%) demonstrated high TMB (≥ 10 mutations/mb). Among eight patients with EGFR fusion, three exhibited possible functionalities of the EGFR gene.


Conclusion
EGFR aberrations, mainly amplification, followed by mutation and fusion, were present in 6.4% of patients with metastatic solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ae0df005b2d8a0f47444591e98d3dc4bb550830" target='_blank'>
              Epidermal Growth Factor Receptor Aberrations Identified by Next-generation Sequencing in Patients with Metastatic Cancers.
              </a>
            </td>
          <td>
            Minkyue Shin, Dae-Ho Choi, Jaeyun Jung, Deokgeun Kim, Sung Hee Lim, Seung-Tae Kim, Jung Yong Hong, Se Hoon Park, Joon-Oh Park, Kyoung-Mee Kim, Jeeyun Lee
          </td>
          <td>2025-02-21</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Large eukaryotic genomes are packaged into the restricted area of the nucleus to protect the genetic code and provide a dedicated environment to read, copy and repair DNA. The physical organisation of the genome into chromatin loops and self-interacting domains provides the basic structural units of genome architecture. These structural arrangements are complex, multi-layered, and highly dynamic and influence how different regions of the genome interact. The role of chromatin structures during transcription via enhancer-promoter interactions is well established. Less understood is how nuclear architecture influences the plethora of chromatin transactions during DNA replication and repair. In this review, we discuss how genome architecture is regulated during the cell cycle to influence the positioning of replication origins and the coordination of DNA double strand break repair. The role of genome architecture in these cellular processes highlights its critical involvement in preserving genome integrity and cancer prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7d6e724c4765bfa8612f020a665aed0f98394cb" target='_blank'>
              Roles for the 3D genome in the cell cycle, DNA replication, and double strand break repair
              </a>
            </td>
          <td>
            Katherine A Giles, P. Taberlay, Anthony J. Cesare, Mathew J. K. Jones
          </td>
          <td>2025-02-27</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Many tumors evolve through cellular mutation and selection, where subpopulations of cells (subclones) with shared ancestry compete for dominance. Introduction of next generation sequencing enables subclone identification using small somatic. However, there are several advantages to marking subclones with structural variants: they have greater functional impact, play a crucial role in late-stage tumors, and provide a more complete view of genomic instability driving tumor evolution. Here, we present SVCFit, a scalable method to estimate the cellular prevalence of somatic deletions, duplications and inversions. We demonstrate that cellular prevalence estimation can be improved by incorporating distinct read patterns for each structural variant type. Additionally, this improvement is achieved without prior knowledge of tumor purity, which is often inaccurate. Using a combination of simulated data and patient-derived metastatic samples with known mixture proportions, we show that our algorithm achieves significantly greater accuracy than state-of-the-art in estimating the structural variants cellular prevalence (p<0.05). The speed of SVCFit estimation from cost-effective bulk whole-genome sequencing (WGS) makes it well-suited for analyzing large cohorts of sequenced tumor samples, enhancing the accessibility of SV-based clonal reconstruction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77e71e2d20d9e52879b8c2d8cc5014dd16b7a335" target='_blank'>
              SVCFit: Inferring structural variant cellular fraction in tumors
              </a>
            </td>
          <td>
            Yunzhou Liu, Jiaying Lai, Laura D. Wood, Rachel Karchin
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Immunotherapy has been a promising treatment for various cancer types. This therapy heavily relies on the efficiency of the targeted neoantigens for T cell recognition and downstream immune responses. Neoantigens derived from tumor-specific nonsynonymous somatic mutations have been a great resource and manifested remarkable therapeutic effects, yet with high patient specificity and high dependency on mutation rate. Here, we propose a novel strategy for discovering neoantigens derived from epigenetic mutations, rather than genetic mutations, from transposable elements (TEs) sequences. TEs make up 50% of the human genome but have been long considered to be “junk DNA” and therefore overlooked in medical genomics. However, recent works from our lab have shown that, despite being epigenetically silenced in somatic cells, certain TEs contribute significantly to the evolution of regulatory networks and shape the cancer transcriptome landscape, including functioning as promoters and producing novel, tumor-specific TE-derived transcripts (TSTETs) that can be translated into proteins. Given the plasticity of epigenome, neoantigens resulted from epigenetic mutations could be pan-cancer with high recurrence. Thus, we hypothesize that cryptic promoters embedded in TEs can be re-activated due to epigenetic dysregulation in cancer, leading to the generation of highly recurrent, pan-cancer, TE-derived, tumor-specific protein products that could be employed as neoantigens for immuno-therapy. In this study, we developed a computational pipeline termed TE Promoter Finder 3 (TEProf3) to precisely identify TE-derived promoters and transcripts genome-wide from transcriptomic data, including data from both long-read and short-read sequencing. We applied TEProf3 to analyze 12,015 tumor samples encompassing 33 cancer types from The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) datasets. Our analysis revealed 14,417 of TSTETs and hundreds of tumor-specific TE-derived antigens (TSTEAs) from the tumor transcriptomic and mass spectrometry data, respectively. Notably, most TSTETs exhibited high recurrence rates, with over 10% recurrence in the majority and some exceeding 90% recurrence. Each tumor sample displayed a median number of 19 TSTETs. Certain cancer types showed a comparable landscape of tumor-specific TE-derived promoters, potentially due to similarities in epigenetic mutations and cell types of origin. The number of TSTETs correlated with tumor progression in some cancer types, suggesting a link between progressive dysregulation of TE-derived promoters and tumorigenesis. We further assessed the efficiency of antigen presentation in cancer cell lines as a proof of concept. We successfully identified T cell receptors sequences from CD8+ T cells that exhibited strong binding affinity for TSTEAs and demonstrated cytotoxicity effects on the cancer cell lines. In summary, this work represents one of the first comprehensive catalogs of TSTEAs, providing a valuable resource for future immunotherapy strategies.
 Citation Format: Yonghao Liang, Xiaoyun Xing, Ting Wang. Transposable elements activation produces pan-cancer tumor-specific neoantigens [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a628523c6f85200cb9e347eacaf757c8785c94bb" target='_blank'>
              Abstract B024: Transposable elements activation produces pan-cancer tumor-specific neoantigens
              </a>
            </td>
          <td>
            Yonghao Liang, Xiaoyun Xing, Ting Wang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Non-canonical roles for growth factors in the nucleus have been previously described, but their mechanism of action and biological roles remain enigmatic. Platelet-derived growth factor B (PDGFB) can drive formation of low-grade glioma and here we show that it localizes to the nucleus of human glioma cells where it binds chromatin to preserve genome stability and cell lineage. Failure of PDGFB to localize to the nucleus leads to chromosomal abnormalities, aberrant heterochromatin architecture and accelerated tumorigenesis. Furthermore, nuclear localization of PDGFB is reliant upon the expression levels and mutation status of isocitrate dehydrogenase (IDH). Unexpectedly, we identified macrophages as the predominant source of PDGFB in human, finding that immune-derived PDGFB can localize to the nucleus of glioma cells. Collectively, these studies show that immune derived PDGFB enters the nucleus of glioma cells to maintain genomic stability, while identifying a new mechanism by which IDH mutations promote gliomagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/646b1a4f23d92e4dc1293f0abe42bdd8c61153d0" target='_blank'>
              Mutant IDH impairs chromatin binding by PDGFB to promote chromosome instability
              </a>
            </td>
          <td>
            R. Curry, Malcolm F McDonald, Peihao He, B. Lozzi, Yeunjung Ko, Isabella O'Reilly, Anna Rosenbaum, Wookbong Kwon, Leyla Fahim, Joshua Marcus, Noah Powell, Su Wang, Jin Ma, Asha Multani, Dongjoo Choi, Debo Sardar, Carrie Mohila, Jason Lee, Marco Gallo, Arif O. Harmanci, A. Harmanci, Benjamin Deneen, Ganesh Rao
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 DNA methylation alterations are observed across multiple cancers, occurring early during tumor initiation. However, evolution and drivers of these changes remain poorly understood and difficult to assess at the clonal level. Consequently, we developed a method for profiling genome-wide DNA methylation from low DNA amounts and applied it to the organoid model of BRAFV600E oncogene-induced colon cancer (CC) tumorigenesis. Single organoid clones, derived from single stem cells, were profiled every month for 5 months to map nucleotide-level dynamic heterogeneity in methylation changes. Uninduced control organoids exhibit a pattern of epigenetic drift leading towards clonal convergence not until 5 months. Whereas BRAFV600E-induced organoids exhibit a step-wise divergence throughout the different time points, likely explained by progressive clonal takeovers. The methylation changes in BRAFV600E induced organoids are explained by the age-in-culture associated methylation changes; especially BRAFV600E did not induce epigenetic changes, as previously thought. The CpG sites affected in the control and BRAFV600E comparison were not shared but mapped to the same genomic elements, indicating common drivers of methylation changes regardless of oncogene. Investigation of promoter CpG-island methylation gains revealed that they are decoupled from replication. External validation in other pre-cancerous and CC datasets, and experimental cell line data sets, show that methylation gains at regulatory elements are a function of “biological aging” and independent of replication or cell division cycles. Our studies provide new insights into the early epigenetic changes involved in BRAFV600E driven carcinogenesis rooted in the age-dependent drivers of epigenetic alterations.
 Citation Format: Sara-Jayne Thursby, Zhicheng Jin, Nibedita Patel, Yong Tao, Yuba Bhandari, Daniel Petkovich, Shangzhi Lu, Lijing Yang, Tza-Huei Wang, Thomas R. Pisanic II, Stephen B Baylin, Hariharan Easwaran. Promoter methylation gains in aging and cancer are independent of replication [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B004.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/082f1b7fdd5a838908f885cbac828f34e7906f35" target='_blank'>
              Abstract B004: Promoter methylation gains in aging and cancer are independent of replication
              </a>
            </td>
          <td>
            Sara-Jayne Thursby, Zhicheng Jin, Nibedita Patel, Yong Tao, Yuba Bhandari, Daniel Petkovich, Shangzhi Lu, Lijing Yang, Tza-Huei Wang, T. R. Pisanic, Stephen B. Baylin, H. Easwaran
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The cellular programs that mediate therapy resistance are often important drivers of metastasis, a phenomenon that needs to be understood better to improve screening and treatment options for cancer patients. Although this issue has been studied extensively for chemotherapy, less is known about a causal link between resistance to radiation therapy and metastasis. We investigated this problem in triple-negative breast cancer (TNBC) and established that radiation resistant tumor cells have enhanced metastatic capacity, especially to bone. Resistance to radiation increases the expression of integrin β3 (ITGβ3), which promotes enhanced migration and invasion. Bioinformatic analysis and subsequent experimentation revealed an enrichment of RNA metabolism pathways that stabilize ITGβ3 transcripts. Specifically, the RNA binding protein heterogenous nuclear ribonucleoprotein L (HNRNPL), whose expression is regulated by Nrf2, mediates the formation of circular RNAs (circRNAs) that function as competing endogenous RNAs (ceRNAs) for the family of let-7 microRNAs that target ITGβ3. Collectively, our findings identify a novel mechanism of radiation-induced metastasis that is driven by alterations in RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14903a27d305650e2051ca5d1898fdc092718345" target='_blank'>
              Resistance to Radiation Enhances Metastasis by Altering RNA Metabolism
              </a>
            </td>
          <td>
            Ayush Kumar, Kensei Kishimoto, H. Goel, C. Wisniewski, Rui Li, Brendan Pacheco, L. J. Zhu, William A. Flavahan, A. Mercurio
          </td>
          <td>2025-02-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9d0020163f84c6d4b9e21837bb90e991331c10b" target='_blank'>
              Enabling sensitive and precise detection of ctDNA through somatic copy number aberrations in breast cancer
              </a>
            </td>
          <td>
            Riccardo Scandino, A. Nardone, Nicola Casiraghi, F. Galardi, Mattia Genovese, Dario Romagnoli, Marta Paoli, C. Biagioni, Andrea Tonina, I. Migliaccio, M. Pestrin, E. Moretti, L. Malorni, L. Biganzoli, M. Benelli, Alessandro Romanel
          </td>
          <td>2025-03-08</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="ZNF93 is a primate-restricted KRAB zinc finger protein responsible for repressing 20- to 12-million-year-old L1 transposable elements. Here, we reveal that ZNF93 also regulates the key cancer driver APOBEC3B—a mutagenic enzyme linked to tumorigenesis and cancer progression. ZNF93 depletion impairs DNA synthesis, activates replication and DNA damage checkpoints, and triggers proinflammatory phenotypes. Conversely, its overexpression enhances resistance to exogenous genotoxic stress, mirroring the effects observed with APOBEC3B depletion. ZNF93 expression correlates with cell proliferation rates and is overexpressed in many cancer types. These findings suggest that ZNF93 serves as a critical guardian of genome integrity, co-opted by cancer cells to counterbalance APOBEC3B-induced and L1-derived genomic instability and inflammation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a66cfad86856f62baa2febbde7f74ab58053f5f4" target='_blank'>
              Cancer cells subvert the primate-specific KRAB zinc finger protein ZNF93 to control APOBEC3B
              </a>
            </td>
          <td>
            Romain Forey, Cyril Pulver, C. Raclot, Olga Rosspopoff, Sandra Offner, Julien Duc, E. Planet, Filipe Martins, P. Turelli, Didier Trono
          </td>
          <td>2025-03-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13edecae9b7bef7cd629efeb486e0288bd13530b" target='_blank'>
              Tumour hypoxia in driving genomic instability and tumour evolution.
              </a>
            </td>
          <td>
            Alexandru Suvac, Jack Ashton, Robert G Bristow
          </td>
          <td>2025-01-28</td>
          <td>Nature reviews. Cancer</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="The DUX4 transcription factor is briefly expressed in the early embryo and is epigenetically repressed in somatic tissues. Loss of epigenetic repression can result in the aberrant expression of DUX4 in skeletal muscle and can cause facioscapulohumeral dystrophy (FSHD). Multiple factors have been identified as necessary to maintain epigenetic silencing of DUX4 in skeletal muscle, but whether specific sequences at the DUX4 locus are sufficient for epigenetic silencing has been unknown. We cloned fragments of the D4Z4 macrosatellite repeat, the DNA region that encompasses the DUX4 retrogene, adjacent to a reporter driven by a constitutive promoter and identified a single fragment sufficient to epigenetically repress reporter gene expression. Previously identified suppressors of DUX4 expression—SETDB1, ATF7IP, SIN3A/B, and LRIF1—were necessary for silencing activity and p38 inhibitors enhanced suppression. These findings identify a key regulatory sequence for D4Z4 epigenetic repression and establish a model system for mechanistic and discovery studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c637d49373bdbc8ffeb399dfd912513e88f9743f" target='_blank'>
              A discrete region of the D4Z4 is sufficient to initiate epigenetic silencing
              </a>
            </td>
          <td>
            Ellen M. Paatela, Faith G. St. Amant, Danielle C Hamm, Sean R. Bennett, Taranjit S. Gujral, Silvère M. van der Maarel, Stephen J. Tapscott
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Background: Gastroesophageal adenocarcinomas (GEA) are characterized by chromosomal instability (CIN) predominantly consisting of high amplitude focal amplifications of oncogenes, including Cyclin E1. An initial pan-cancer analysis revealed that CCNE1 is one of the top genes whose expression correlates with focal CIN. Objective: We proposed a causal role for excess CCNE1 as a driver of CIN and gastroesophageal tumorigenesis, and investigated the therapeutic vulnerabilities of CCNE1 amplification/overexpression. Design: We engineered a novel CCNE1-overexpressing mouse gastric organoid model with Trp53 deletion and studied the functional effects on tumor initiation and progression. Whole genome sequencing of tumors was performed to characterize the effects of CCNE1 overexpression on genomic instability. To identify dependencies of CCNE1-amplified GEA, we executed a CRISPRi functional genomic screen. Top genetic dependencies were translated into potential pharmacologic vulnerabilities and tested in monotherapy and combination. Results: CCNE1 overexpression accelerated tumor initiation and progression of mouse gastric organoids with Trp53 deletion. These tumors demonstrated increased chromosomal complexity, mainly focal amplifications. Our functional genomic screen identified specific genetic dependencies of CCNE1-amplified GEA, including CCNE1/CDK2 as the strongest hits, and SKP2 as a novel dependency likely through its function as a p27 degrader. Pharmacologic inhibition of CDK2 was insufficient and antagonistic with concurrent inhibition of the DNA damage response via targeting the ATR/CHEK1 axis. However, staggered inhibition of these distinct vulnerabilities demonstrated efficacy. Conclusions: Our study identifies CCNE1 as a driver of CIN in GEA and builds a foundation for further exploration of therapies for CCNE1-amplified GEA characterized by genomic instability.
 Citation Format: ZHONG WU, Ankur Nagaraja, Chunyang Bao, Jin Zhou, Adam Bass. Cyclin E1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr B019.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344c45deee3ee4d6c328b84a6e73872831228859" target='_blank'>
              Abstract B019: Cyclin E1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Chunyang Bao, Jin Zhou, A. Bass
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Recently revised brain tumor classification suggested a glioma treatment strategy that takes into consideration molecular variants in IDH1 and TP53 marker genes. While pathogenic variants of IDH1 and TP53 can be accompanied by chromosomal instability (CIN), the impact of IDH1 and TP53 mutations on genome stability remains unstudied. Elevated CIN might provide therapeutic targets, based on synergistic effects of chemotherapy with CIN-inducing drugs. Using an assay based on human artificial chromosomes, we investigated the impact of common glioma missense mutations in IDH1 and TP53 on chromosome transmission and demonstrated that IDH1R132H and TP53R248Q variants elevate CIN. We next found enhanced CIN levels and the sensitivity of IDH1R132H/WT and TP53R248Q/R248Q genotypes, introduced into U87 MG glioma cells by CRISPR/Cas9, to different drugs, including conventional temozolomide. It was found that U87 MG cells carrying IDH1R132H/WT exhibit dramatic sensitivity to paclitaxel, which was independently confirmed on cell cultures derived from patients with naturally occurring IDH1R132H/WT. Overall, our results suggest that the development of CIN-enhancing therapy for glioma tumors with the IDH1R132H/WT genotype could be advantageous for adjuvant treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/641df7ee46092e60a05501256dd4211c20f30f9f" target='_blank'>
              Therapy enhancing chromosome instability may be advantageous for IDH1R132H/WT gliomas
              </a>
            </td>
          <td>
            Nikolay V Goncharov, Ivan Baklanov, Valeriia S Gulaia, Anastasiia P Shuliak, Daria V Lanskikh, Valeriia Zhmenia, M.E. Shmelev, N. Shved, Jing Wu, Mikhail Liskovykh, Vladimir Larionov, N. Kouprina, Vadim Kumeiko
          </td>
          <td>2025-02-13</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Homologous recombination maintains genome stability by repairing double strand breaks and protecting replication fork stability. Defects in homologous recombination results in cancer predisposition but can be exploited due to increased sensitivity to certain chemotherapeutics such as PARP inhibitors. The NEK8 kinase has roles in the replication response and homologous recombination. NEK8 is overexpressed in breast cancer, but the impact of NEK8 overexpression on homologous recombination has not been determined. Here, we demonstrate NEK8 overexpression inhibits RAD51 focus formation resulting in a defect in homologous recombination and degradation of stalled replication forks. Importantly, NEK8 overexpression sensitizes cells to the PARP inhibitor, Olaparib. Together, our results suggest NEK8 overexpressing tumors may be recombination-deficient and respond to chemotherapeutics that target defects in recombination such as Olaparib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8173c08c1d4d30ea1a0d792e303a9db46e92a70" target='_blank'>
              Overexpression of the NEK8 kinase inhibits homologous recombination
              </a>
            </td>
          <td>
            Joshua L. Turner, Georgia Moore, Tyler J. McCraw, Jennifer M. Mason
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0491183160fe7bbdae2e8ae2e68e3e5bf8212367" target='_blank'>
              Intratumor heterogeneity of HPV integration in HPV-associated head and neck cancer
              </a>
            </td>
          <td>
            Noa Sasa, T. Kishikawa, Masashi Mori, R. Ito, Yumie Mizoro, Masami Suzuki, Hirotaka Eguchi, Hidenori Tanaka, T. Fukusumi, Motoyuki Suzuki, Y. Takenaka, K. Nimura, Yukinori Okada, H. Inohara
          </td>
          <td>2025-01-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The identification of recurrent mutations in genes encoding the cohesin complex in cancer was among the most unexpected findings from cancer exome sequencing studies. Cohesin is a multi-subunit protein complex that is essential for sister chromatid cohesion, three-dimensional chromosome organization, DNA damage repair, and gene regulation. It forms a ring around DNA, with four structural subunits, SMC1A, SMC3, RAD21, and either STAG1 or STAG2. In particular, the cohesin subunit STAG2 is one of only 12 human genes to be significantly mutated in four or more distinct types of cancer. Cohesin mutations are typically heterozygous and result in haploinsufficiency and/or loss-of-function, and although they might be expected to cause defects in chromosome segregation, the sister chromatid cohesion function of the complex is unlikely the driving mechanism of tumorigenesis, given the lack of aneuploidy in cohesin-mutant cancers. In this review, we will focus on the prevalence of somatic mutations in cohesin subunits across different cancer types, the influence of these mutations on chromatin organization and gene regulation, the resulting cellular and disease phenotypes, and the therapeutic potential of targeting the mutant cohesin complex.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/391f121bccfd400a13a99c3af96bbd051ffc5650" target='_blank'>
              Cohesin mutations and chromatin changes in cancer.
              </a>
            </td>
          <td>
            Ariel D Swett, Zuzana Tothova
          </td>
          <td>2025-03-10</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279ce0f97c7d66cc42d068594f84880c4fe69c7b" target='_blank'>
              Alternative splicing of transposable elements in human breast cancer
              </a>
            </td>
          <td>
            Alex Nesta, Diogo F. T. Veiga, Jacques Banchereau, Olga Anczuków, Christine R. Beck
          </td>
          <td>2025-02-22</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Mutations in the TP53 gene and chromosomal instability (CIN) are two of the most common alterations in cancer. CIN, marked by changes in chromosome numbers and structure, drives tumor development, but is poorly tolerated in healthy cells, where developmental and tissue homeostasis mechanisms typically eliminate cells with chromosomal abnormalities. Mechanisms that allow cancer cells to acquire and adapt to CIN remain largely unknown. Tumor suppressor protein p53, often referred to as the “guardian of the genome”, plays a critical role in maintaining genomic stability. In cancer, CIN strongly correlates with TP53 mutations, and recent studies suggest that p53 prevents the propagation of cells with abnormal karyotypes arising from mitotic errors. Furthermore, p53 dysfunction is frequent in cells that underwent whole-genome doubling (WGD), a process that facilitates CIN onset, promotes aneuploidy tolerance, and is associated with poor patient prognosis across multiple cancer types. This review summarizes current insights into p53’s role in protecting cells from chromosome copy number alterations and discusses the implications of its dysfunction for the adaption and propagation of cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ee772438cd2bfb17d973a17539b1ffdae477941" target='_blank'>
              The Role of p53 Mutations in Early and Late Response to Mitotic Aberrations
              </a>
            </td>
          <td>
            Anna Hertel, Zuzana Storchová
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Lung adenocarcinoma, the most common subtype of lung cancer, is genomically complex, with tumors containing tens to hundreds of non-synonymous mutations. However, little is understood about how genes interact with each other to enable the evolution of cancer in vivo, largely due to a lack of methods for investigating genetic interactions in a high-throughput and quantitative manner. Here, we employed a novel platform to generate tumors with inactivation of pairs of ten diverse tumor suppressor genes within an autochthonous mouse model of oncogenic KRAS-driven lung cancer. By quantifying the fitness of tumors with every single and double mutant genotype, we show that most tumor suppressor genetic interactions exhibited negative epistasis, with diminishing returns on tumor fitness. In contrast, Apc inactivation showed positive epistasis with the inactivation of several other genes, including synergistic effects on tumor fitness in combination with Lkb1 or Nf1 inactivation. Sign epistasis was extremely rare, suggesting a surprisingly accessible fitness landscape during lung tumorigenesis. These findings expand our understanding of the interactions that drive tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e03ac1e8a6ee59b9db5be527feeaddbf8b397192" target='_blank'>
              Combinatorial In Vivo Genome Editing Identifies Widespread Epistasis and an Accessible Fitness Landscape During Lung Tumorigenesis
              </a>
            </td>
          <td>
            Jess D. Hebert, Yuning J Tang, Márton Szamecz, Laura Andrejka, Steven S. Lopez, D. Petrov, Gábor Boross, M. Winslow
          </td>
          <td>2025-02-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 Embryonal Tumors with Multilayered Rosettes (ETMRs) are relatively rare but aggressive embryonal brain tumors that mainly affect children under the age of 4 and which have a poor outcome. Around 90% of ETMRs harbor amplification of the C19MC miRNA cluster, in most cases fused to the TTHY1 gene. The second most recurrent aberration, present in about half of the C19MC negative cases, are bi-allelic mutations in DICER1. ETMRs are also characterized by overall lower DNA methylation levels and high LIN28A gene expression as compared with other embryonal brain tumors like medulloblastomas (MB) and atypical teratoid rhabdoid tumors (ATRT). However, a more detailed description of the epigenetic landscape marked by DNA methylation and histone modifications, and how this affects the ETMR transcriptome is still lacking. To decode the epi-genome, we have collected whole genome bisulfite sequencing (WGBS) data, ChIP sequencing (ChIP-seq) data of various (active and repressive) histone marks and epigenetic modifiers (EZH2, BRD4, CTCF), and RNA sequencing (RNA-seq) data from a series of ETMR patient tumors, and similar data from a series of MBs and ATRTs covering all molecular subtypes for comparison reasons. Using all these data and performing bioinformatic analyses, we have depicted chromatin states and DNA-methylation-defined genomic segments of ETMR for the first time. ETMR genomes are generally hypomethylated, with wider distributions of un-methylated regions and DNA methylation valleys (DMVs) on its genome compared with MB and ATRT. By integrating these data with all other omics data, we found that BRD4-regulated overexpression of TET demethylases may cause global hypomethylation in ETMRs. Polycomb and TETs are recruited at DMVs to ensure DMVs are longer in ETMRs than MBs and ATRTs. Furthermore, genes involved in embryo development and cell proliferation are enriched in entity-shared DMVs and are highly expressed in ETMRs and other embryonal brain tumors. We also depicted that TET family members, BRD4, and Polycomb complex (EZH2) are potential therapeutic targets for ETMR patients. In conclusion, we have partially described the epi-genome and epigenetic regulations of ETMR, and are currently working on more detailed regulation axes and performing drug test experiments for these potential therapeutic targets to understand better the underlying biology of these tumors and how this may lead to alternative and more effective therapies.
 Citation Format: Shanzheng Wang, Sander Lambo, Monika Mauermann, Robert J Autry, Natalie Jäger, Stefan M Pfister, Marcel Kool. Epigenetic landscape and gene regulations in embryonal brain tumors with multilayered rosettes (ETMR) [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B028.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0e28f4d695edfbb9dadc1b324a1b929e51f3786" target='_blank'>
              Abstract B028: Epigenetic landscape and gene regulations in embryonal brain tumors with multilayered rosettes (ETMR)
              </a>
            </td>
          <td>
            Shanzheng Wang, S. Lambo, Monika Mauermann, Robert J Autry, N. Jäger, Stefan M. Pfister, M. Kool
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="Background The escalating global cancer burden necessitates the development of biomarkers with enhanced specificity and sensitivity for early diagnosis and therapeutic efficacy monitoring. The CNIH4 gene, an emerging biomarker, is increasingly recognized for its role in the malignant progression across various cancers. Methods We conducted a comprehensive multi-omics analysis of CNIH4, including pan-cancer expression profiles, epigenetic alterations, immune microenvironment characteristics, and therapeutic response patterns. Our focus was on clinical features, molecular underpinnings, and drug sensitivity in breast cancer (BRCA) associated with CNIH4. In vitro studies were also performed to assess the effects of CNIH4 knockdown on cell proliferation and cell cycle in the MDA-MB-231 cell line. Results CNIH4 upregulation was observed in multiple cancers, significantly correlating with genomic instability. High CNIH4 expression levels were linked to poor prognosis across cancers and associated with key cancer-related pathways, particularly those in cell cycle regulation and DNA repair. Correlation analyses suggest a role for CNIH4 in the tumor immune microenvironment, as evidenced by its association with immune subtypes, immune-related genes, and immune cell infiltration. Single-cell and spatial transcriptome analyses confirmed that CNIH4 expression in BRCA predicts tumor malignancy. Drug sensitivity analysis revealed a significant correlation between CNIH4 and responsiveness to various kinase inhibitors and chemotherapeutic agents. In vitro experiments demonstrated that CNIH4 knockdown significantly impacts the proliferation and cell cycle of MDA-MB-231 cells. Conclusion Our study highlights CNIH4 as a promising pan-cancer biomarker with significant implications for tumor progression and a critical role in cell cycle regulation in BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de7a2eade185851de2705c33f65b5768b5aafc69" target='_blank'>
              Deciphering the role of CNIH4 in pan-cancer landscapes and its significance in breast cancer progression
              </a>
            </td>
          <td>
            Yao Xu, Zengzhen Lai, Chaolin Li
          </td>
          <td>2025-02-20</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Solid tumors often have an abundance of collagen-I that stiffens the tissue, and they are invariably driven by mutations that include chromosome losses and gains. These observations are linked here by showing that 3D matrix stiffness induces heritable changes to a cell’s DNA. We use live-cell chromosome reporters (ChReporters) and hydrogels of tunable stiffness to show mitotic compression, micronuclei counts, ChReporter losses and heterogeneity all increase as functions of stiffness. Increased mistakes occur despite suppressed cell division in stiff matrix and minimal size variation between spheroids. Colonies of ChReporter-negative cells within cancer spheroids align with Luria-Delbruck’s seminal theory for heritable mutations, which predicts inter-spheroid variances that exceed Poisson statistics. Suppression of the contractility motor Myosin-II also increases chromosome loss in 3D but not 2D and does not affect spheroid growth – thus clarifying Myosin-II’s putative role as a tumor suppressor. Consistent with experiments, pan-cancer analyses of clinical data associates chromosome losses and gains with collagen-I levels and genetic variation. Stiff extracellular matrix thus drives mechano-evolution of solid tumors as a Darwin-Lamarck process with heterogeneity that complicates therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e21422305f6e2cd46c2231ab78870bc6be52cefd" target='_blank'>
              Matrix stiffness induces heritable changes in chromosome numbers, consistent with solid tumor heterogeneity
              </a>
            </td>
          <td>
            Alişya A. Anlaş, Markus T. Sprenger, Mai Wang, Nicholas M. Ontko, Steven H. Phan, D.E. Discher
          </td>
          <td>2025-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60d43986e3ec1bf10fa7949b61cdcc071a4afe56" target='_blank'>
              Chromosomal domain formation by archaeal SMC, a roadblock protein, and DNA structure
              </a>
            </td>
          <td>
            Kodai Yamaura, Naomichi Takemata, Masashi Kariya, Ayami Osaka, S. Ishino, Masataka Yamauchi, Tomonori Tamura, I. Hamachi, Shoji Takada, Y. Ishino, H. Atomi
          </td>
          <td>2025-02-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/861d404b3fd5272bc30487378b974c5ae16b843d" target='_blank'>
              CTCF is selectively required for maintaining chromatin accessibility and gene expression in human erythropoiesis
              </a>
            </td>
          <td>
            Xue Yang, Li Cheng, Ye Xin, Jianxiang Zhang, Xinfeng Chen, Jinchao Xu, Mengli Zhang, Ruopeng Feng, Judith Hyle, Wenjie Qi, Wojciech Rosikiewicz, Beisi Xu, Chunliang Li, Peng Xu
          </td>
          <td>2025-02-28</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Mutations with a synergistic effect on tumorigenesis were positively selected during tumor evolution, showing a tendency to co-occur in cancer genomes. To systematically gain insight of tumorigenic crosstalk at the mutation level, we utilized the FP-Growth algorithm in frequent pattern mining to create an unbiased map of co-occurring mutations from clinical genome sequencing data. We identified 100,933 frequent co-occurring mutation pairs across 22 cancer types. The co-mutation pairs involving cancer driver genes highlighted potential mechanisms for collaborative tumor formation, suggesting promising targets to disrupt tumorigenesis. Additionally, the large number of intra-gene co-mutations with strong dependencies indicated possible combinational effects on the oncogenic properties of specific proteins, which might be overlooked when individual mutations are not hotspots. Furthermore, our whole-genome mining strategy revealed a significant number of co-mutations in non-coding regions, with an enrichment of regulatory elements suggesting their potential role in modulating the co-expression of related genes. Finally, we demonstrated that our pipeline could systematically identify high-order co-mutations. In summary, our study provides the most extensive co-occurring mutations in tumor development, offering valuable insights into mechanisms and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10963d53c10fd0cd38d8c059ea7d0809367afa03" target='_blank'>
              Unbiased co-occurring mutational landscapes in tumorigenesis
              </a>
            </td>
          <td>
            Yizhou Li, Yangfang Xiong, Wensheng Wei
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Despite the development of HER2-targeting drugs such as trastuzumab and T-DXd, treatment resistance is a substantial challenge, often leading to relapse and distant metastasis. Tumor heterogeneity in HER2-positive breast cancer drives the evolution of resistant clones following therapeutic stress. However, the targetable drivers of anti-HER2 treatment resistance are not thoroughly identified. This study aims to use neoadjuvant-targeted therapy cohorts and a patient-derived organoid in vitro treatment model to uncover the potential targetable drivers of anti-HER2 treatment resistance. it is found that LINC01235 significantly enhances DNA replication licensing and chromosomal instability, fostering clonal expansion and evolution, and ultimately increasing resistance to therapeutic interventions. LINC01235 regulates global H3K27ac, H3K9ac, and H3K36me3 modifications, promotes H2A.Z expression in regulatory regions, and increases the accessibility of DNA licensing factors to their promoter regions. XRCC5 is identified as a key component for maintaining genomic stability, crucial for LINC01235's role in replication licensing. Furthermore, therapeutic strategies targeting LINC01235, including the use of antisense oligonucleotides or ATR inhibitors, which showed promise in overcoming treatment resistance are explored. These findings underscore the pivotal role of LINC01235 in driving resistance mechanisms and highlight novel avenues for targeted therapies to improve the outcomes of patients with HER2-positive breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fde95930ed4b5d2e1c4571f86786e616e7f2cf6" target='_blank'>
              LINC01235 Promotes Clonal Evolution through DNA Replication Licensing-Induced Chromosomal Instability in Breast Cancer.
              </a>
            </td>
          <td>
            Qi Zhang, Xuliren Wang, Zhibo Shao, Yi Zhang, Li-Ying Zhang, Ming Chen, Xujie Zhou, Han Zhu, Yue Zhou, Xinya Lu, Pei Li, W. Chi, Lun Li, Zhibo Shao, Shenglin Huang, J. Xue, Y. Chi, Jiong Wu, B. Xiu
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The ability to replicate and pass information to descendants is a fundamental requirement for life. In addition to the DNA-based genetic information, modifications of the DNA or DNA-associated proteins can create patterns of heritable gene regulation. Such epigenetic inheritance allows for adaptation without mutation, but its limits and regulation are incompletely understood. Here we developed a C. elegans system to study the transgenerational epigenetic inheritance of H3K27me3, a conserved histone posttranslational modification associated with gene repression. We find that induced alterations of the genome-wide H3K27me3 landscape and the associated fertility defects persist for many generations in genetically wildtype descendants under selective pressure. We uncover that the inheritance of the altered H3K27me3 landscape is regulated by two chromodomain proteins with antagonizing functions, and provide mechanistic insight into how this molecular memory is initiated and maintained. Our results demonstrate that epigenetic inheritance can act as a mutation-independent, heritable mechanism of adaptation. In Brief Özdemir et al. demonstrate that an altered genomic distribution of the histone modification H3K27me3 can be epigenetically inherited across many generations through the activity of HERI-1/SET-32/MES-4, which is antagonized by CEC-6/PRC2 in C. elegans. Highlights Altered H3K27me3 landscapes can be inherited for at least 15 generations in C. elegans. The chromodomain proteins CEC-6 and HERI-1 have opposite roles in antagonizing or promoting the maintenance of the altered H3K27me3 landscape. H3K23me3 and H3K36me3 replace H3K27me3 to promote the intergenerational and transgenerational inheritance of the altered epigenome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d9477951c95e1934dfca6ddb054dc1db83139c5" target='_blank'>
              Transgenerational maintenance of H3K27me3 heterochromatin is balanced by chromodomain proteins in Caenorhabditis elegans
              </a>
            </td>
          <td>
            Isa Özdemir, Anna Höfler, Kamila Delaney, Joanna M. Wenda, Chengyin Li, Arneet L. Saltzman, Andreas Boland, Florian A. Steiner
          </td>
          <td>2025-03-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The impact of cancer driving mutations in regulating immunosurveillance throughout tumor development remains poorly understood. To better understand the contribution of tumor genotype to immunosurveillance, we generated and validated lentiviral vectors that create an epi-allelic series of increasingly immunogenic neoantigens. This vector system is compatible with autochthonous Cre-regulated cancer models, CRISPR/Cas9-mediated somatic genome editing, and tumor barcoding. Here, we show that in the context of KRAS-driven lung cancer and strong neoantigen expression, tumor suppressor genotype dictates the degree of immune cell recruitment, positive selection of tumors with neoantigen silencing, and tumor outgrowth. By quantifying the impact of 11 commonly inactivated tumor suppressor genes on tumor growth across neoantigenic contexts, we show that the growth promoting effects of tumor suppressor gene inactivation correlate with increasing sensitivity to immunosurveillance. Importantly, specific genotypes dramatically increase or decrease sensitivity to immunosurveillance independently of their growth promoting effects. We propose a model of immunoediting in which tumor suppressor gene inactivation works in tandem with neoantigen expression to shape tumor immunosurveillance and immunoediting such that the same neoantigens uniquely modulate tumor immunoediting depending on the genetic context. One Sentence Summary Here we uncover an under-appreciated role for tumor suppressor gene inactivation in shaping immunoediting upon neoantigen expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a2622766d9e8241bb1972eff5c441baddd14069" target='_blank'>
              Tumor suppressor genotype influences the extent and mode of immunosurveillance in lung cancer
              </a>
            </td>
          <td>
            Keren M. Adler, Haiqing Xu, Amy C. Gladstein, Valerie M. Irizarry-Negron, Maggie R. Robertson, Katherine R. Doerig, D. Petrov, M. Winslow, D. Feldser
          </td>
          <td>2025-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Chromosomal deletion of tumor suppressor genes often occurs in an imprecise manner, leading to co-deletion of neighboring genes. This collateral damage can create novel dependencies specific to the co-deleted context. One notable example is the dependency on PRMT5 activity in tumors with MTAP deletion, which co-occurs with CDKN2A/B loss, leading to the development of MTA-cooperative PRMT5 inhibitors. To identify additional collateral damage context/target pairs for chromosome 9p and other common loci of chromosomal deletions, we conducted a combinatorial CRISPR screen knocking out frequently co-deleted genes in combination with a focused target library. We identified the gene encoding the ribosome rescue factor PELO as synthetic lethal with the SKI complex interacting exonuclease FOCAD, which is frequently co-deleted alongside MTAP and CDKN2A/B on chromosome 9p. A genome-wide screen in FOCAD isogenic cells further identified the ribosome rescue GTPase and PELO binding partner HBS1L as the top synthetic lethal target for FOCAD loss. Analysis of publicly available data and genetic manipulation of HBS1L using orthogonal modalities validated this interaction. HBS1L dependency in FOCAD-deleted cells was rescued by FOCAD re-expression, and FOCAD intact cells could be rendered HBS1L-dependent by FOCAD knockout, demonstrating the context specificity of this interaction. Mechanistically, HBS1L loss led to translational arrest and activated the unfolded protein response in FOCAD-deleted cells. In vivo, HBS1L deletion eliminated growth of FOCAD-deleted tumors. Here we propose a model where the FOCAD/SKI complex and HBS1L/PELO work together to resolve aberrant mRNA-induced ribosomal stalling, making the HBS1L/PELO complex an intriguing novel target for treating FOCAD-deleted tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1ae2ca598b5cdc041c4c163b222a0b0f6d55027" target='_blank'>
              Tumor suppressor collateral damage screens reveal mRNA homeostasis protein HBS1L as a novel vulnerability in ch9p21 driven FOCAD deleted cancer
              </a>
            </td>
          <td>
            Hongxiang Zhang, M. Tonini, Lauren Martires, Charlotte B Pratt, Helena N. Jenkins, Eden N. Gordon, Shanchuan Zhao, Ashley H Choi, Samuel R. Meier, Tenzing Khendu, Shangtao Liu, B. Shen, Hannah Stowe, Katerina Pashiardis, Xuewen Pan, Madhavi Bandi, Minjie Zhang, Yi Yu, Chengyin Min, A. Huang, Jannik N Andersen, Hilary E. Nicholson, T. Teng
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Genome three-dimensional organization is essential for the coordination of eukaryote gene expression. The chromosomes of the pathogen Trypanosoma brucei contain hundreds of silent variant surface glycoprotein (VSGs) genes in subtelomeric regions. However, T. brucei transcribes a single VSG gene and periodically changes the VSG expressed by transcriptional or recombination mechanisms, altering its surface coat to escape host antibodies by antigenic variation. We show that VSG-rich silent subtelomeric regions form distinct chromosome compartments from transcribed regions, with subtelomeric compartments of different chromosomes co-interacting. We uncovered chromatin-associating factors at the boundaries of transcribed and silent compartments. Among these, repressor activator-protein 1 (RAP1) marks the compartment boundaries and spreads over silent regions. Inactivation of phosphatidylinositol phosphate 5-phosphatase removed RAP1 from compartment boundaries and subtelomeric regions, disrupting compartment assembly and derepressing all VSG genes. The data show spatial segregation of repressed from transcribed chromatin and phosphoinositides regulation of silent compartment assembly and genome organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a481ddec30044015c744ee5e03cef2d29b31493" target='_blank'>
              Chromosome compartment assembly is essential for subtelomeric gene silencing in trypanosomes
              </a>
            </td>
          <td>
            Luiza Berenguer Antunes, Tony Isebe, O. Kutova, I. Cestari
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Endemic Burkitt lymphoma (eBL) is one of the most prevalent cancer in children in sub-Saharan Africa, and while prior studies have found that Epstein-Barr virus (EBV) type and variation may alter the tumor driver genes necessary for tumor survival, the precise relationship between EBV variation and EBV-associated tumorigenesis remains unclear due to lack of scalable, cost-effective, viral whole-genome sequencing from tumor samples. This study introduces a rapid and cost-effective method of enriching, sequencing, and assembling accurate EBV genomes in BL tumor cell lines through a combination of selective whole genome amplification (sWGA) and subsequent 2-tube multiplex polymerase chain reaction along with long-read sequencing with a portable sequencer. The method was optimized across a range of parameters to yield a high percentage of EBV reads and sufficient coverage across the EBV genome except for large repeat regions. After optimization, we applied our method to sequence 18 cell lines and 3 patient tumors from fine needle biopsies and assembled them with median coverages of 99.62 and 99.68%, respectively. The assemblies showed high concordance (99.61% similarity) to available Illumina-based assemblies, and differences were consistent with improved accuracy. The improved method and assembly pipeline will allow for better understanding of EBV variation in relation to BL and is applicable more broadly for translational research studies, especially useful for laboratories in Africa where eBL is most widespread.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dc231a1e340bf069a402e88df346d750d87f574" target='_blank'>
              Comparison of Nanopore with Illumina Whole Genome Assemblies of the Epstein-Barr Virus in Burkitt Lymphoma
              </a>
            </td>
          <td>
            Isaac E. Kim, A. Fola, Enrique Puig, Titus K. Maina, Sin Ting Hui, Hongyu Ma, Kaleb Zuckerman, Eddy O. Agwati, A. Leonetti, Rebecca Crudale, Micah A. Luftig, A. Moormann, Cliff I. Oduor, Jeffrey A. Bailey
          </td>
          <td>2025-02-25</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Genes involved in the regulation of chromatin structure are frequently disrupted in cancer, contributing to an aberrant transcriptome and phenotypic plasticity. Yet, therapeutics targeting mutant forms of chromatin-modifying enzymes have yielded only modest clinical utility, underscoring the difficulty of targeting the epigenomic underpinnings of aberrant gene regulatory networks. Here, we sought to identify novel epigenetic vulnerabilities in diffuse large B-cell lymphoma (DLBCL). Through phenotypic screens and biochemical analysis, we demonstrated that inhibition of the H3K9 demethylases KDM4A and KDM4C elicits potent, subtype-agnostic cytotoxicity by antagonizing transcriptional networks associated with B-cell identity and epigenetically rewiring heterochromatin. KDM4 demethylases associated with the KRAB zinc finger ZNF587, and their enzymatic inhibition led to DNA replication stress and DNA damage-induced cGAS-STING activation. Broad surveys of transcriptional data from patients also revealed KDM4 family dysregulation in several other cancer types. To explore this potential therapeutic avenue, we performed high-throughput small molecule screens with H3K9me3 nucleosome substrates and identified novel KDM4 demethylase inhibitors. AI-guided protein-ligand binding predictions suggested diverse modes of action for various small molecule hits. Our findings underscore the relevance of targeting fundamental transcriptional and epigenetic mechanisms for anti-cancer therapy. HIGHLIGHTS Phenotypic screens identified JIB-04 as a potent anti-cancer agent for multiple subtypes of diffuse large B-cell lymphoma JIB-04 binds and inhibits KDM4 demethylases resulting in epigenomic rewiring of heterochromatin KDM4 demethylases cooperate with KRAB zinc fingers to limit DNA replication stress, and KDM4 inhibition instigates DNA-damage and cGAS-STING activation in several human cancers High-throughput small molecule screens with semi-synthetic nucleosome substrates and AI-guided molecular docking simulations identify novel KDM4 inhibitors">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502a11d348f0facd56363bda5866813863b5fcc9" target='_blank'>
              Heterochromatin fidelity is a therapeutic vulnerability in lymphoma and other human cancers
              </a>
            </td>
          <td>
            M. Najia, Deepak K. Jha, Cheng Zhang, Benoit Laurent, Caroline Kubaczka, Arianna Markel, Christopher Li, Vivian M Morris, Allison Tompkins, Luca Hensch, Yue Qin, B. Chapuy, Yu-Chung Huang, Michael A. Morse, M. Marunde, Anup Vaidya, Zachary B. Gillespie, S. A. Howard, Trista E. North, Daniel Dominguez, Michael-Christopher Keogh, Thorsten M Schlaeger, Yang Shi, Hu Li, Margaret M. Shipp, P. Blainey, George Q. Daley
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="ABSTRACT Human endogenous retroviruses (HERVs) are proviral relics of infections that affected primates' germ line. Many HERV elements retain a residual capacity to encode transcripts and proteins that have been occasionally domesticated for the host physiology. In addition, HERV transcriptional modulation is of great interest to clarify the etiology of complex disorders such as cancer, even if a few studies assessed the specific HERV loci modulated in tumor tissues. In the present work, we used a transcriptomic approach to investigate the specific expression of ~3300 HERV loci in paired tumor and normal tissues of 7 colorectal cancer (CRC) patients. A total of 102 HERVs were significantly modulated in CRC, with a general tendency towards downregulation. Of note, among the 42 upregulated HERVs 23 belonged to the HERV‐H group, that is the most investigated in CRC. De novo transcriptome reconstruction and qPCR validation allowed to identify a transcript from a HERV‐H locus on chromosome Xp22.3 with high specific expression in CRC samples, potentially encoding for a partial Pol protein. These results provide a detailed description of HERV transcriptional variations in CRC and its interindividual variability, identifying a HERV‐H transcript that deserves further investigation for its possible impact on tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8d57225f528a23471e9e5e65e4532aff3132bd2" target='_blank'>
              HERV Modulation in Colorectal Carcinoma Patients: A Snapshot of Endogenous Retroviral Transcriptome
              </a>
            </td>
          <td>
            Nicole Grandi, Ching-Hsuan Liu, Saili Chabukswar, Daniele Carta, Yun Yen, Liang-Tzung Lin, E. Tramontano
          </td>
          <td>2025-02-24</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Whole genome duplication, or polyploidy, has been implicated in driving genome instability and tumorigenesis. Recent studies suggest that polyploidy in tumors promotes cancer genome evolution, progression, and chemoresistance resulting in worse prognosis of survival. The mechanisms by which whole genome duplications confer genome instability are not yet fully understood. In this study, we use Schizosaccharomyces pombe (fission yeast) diploids to investigate how whole genome duplication affects genome maintenance and response to stress. We find that S. pombe diploids are sensitive to replication stress and DNA damage, exhibit high levels of loss of heterozygosity, and become dependent on a group of ploidy-specific lethal genes for viability. These findings are observed in other eukaryotic models suggesting conserved consequences of polyploidy. We further investigate ploidy-specific lethal genes by depleting them using an auxin-inducible degron system to elucidate the mechanisms of genome maintenance in diploids. Overall, this work provides new insights on how whole genome duplications lead to genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/185bf82df0621d29d2214d3aa74691b2937269d4" target='_blank'>
              Diploidy confers genomic instability in Schizosaccharomyces pombe
              </a>
            </td>
          <td>
            Joshua M. Park, Daniel F. Pinski, S. Forsburg
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Approximately 97% of the human genome comprises non-coding sequences, with nearly half originating from transposable elements. Among these, retrotransposons represent a critical subclass that replicates via a "copy-and-paste" mechanism and significantly influences the regulation of host genomes. In both normal and pathological contexts, retrotransposons contribute a vast reservoir of regulatory elements that can modulate the expression of genes. If left unchecked, retrotransposons can substantially affect host transcriptional programs and genomic integrity. Therefore, various mechanisms, including epigenetic modifications, are employed to mitigate their potentially deleterious effects. In diseases such as cancers, the epigenome is often significantly reprogrammed, which can lead to retrotransposon dysregulation. Drawing insights from recent studies conducted in human and murine cells, this review examines how retrotransposons expand the complexity of mammalian genomes, describes the impact of their epigenetic dysregulation in cancer development, and highlights the potential of targeting these sequences for therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acf5c3f4d65d70d88355c0196d8616e76468f57c" target='_blank'>
              Epigenetic dysregulation of retrotransposons in cancer.
              </a>
            </td>
          <td>
            Danny Leung, Kwok Yu Liu
          </td>
          <td>2025-02-13</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07762c4bd9219b33748e6d7ae6f755f4437dd352" target='_blank'>
              Extrachromosomal circular DNA: a double-edged sword in cancer progression and age-related diseases.
              </a>
            </td>
          <td>
            Shadira Anindieta Irdianto, A. Dwiranti, A. Bowolaksono
          </td>
          <td>2025-02-19</td>
          <td>Human cell</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Breast cancer is characterized by genetic and epigenetic deregulations, leading to aberrant expression of tissue-specific genes that are normally silent in healthy breast tissue. Our previous work identified the embryonic stem cell-specific gene DNMT3B, a de novo DNA methyltransferase, as aberrantly activated in breast cancer, correlating with aggressive tumour behaviour and high relapse risk, regardless of molecular subtype. Through integrative bioinformatic analyses of DNA methylation and transcriptomic data, we identified 154 genes downregulated via DNMT3B-driven promoter hypermethylation, many of which are associated with high relapse risk. Notably, the tumour suppressor gene GATA3 emerged as a primary target of functional inactivation through either loss-of-function mutations or DNMT3B-controlled hypermethylation, in a mutually exclusive manner. Both mechanisms of GATA3 inactivation were associated with similar molecular signatures linked to tumour progression, increased malignancy, and poorer prognosis. However, distinct differences were observed, with immune- and inflammation-related genes enriched in GATA3 hypermethylation cases but depleted in mutation-driven silencing. Additionally, our analysis uncovered other potential tumour suppressor genes epigenetically repressed in aggressive breast cancers. These findings underscore a broader role of GATA3 inactivation beyond genetic alterations and suggest therapeutic opportunities to target epigenetically silenced tumour suppressors in aggressive breast tumours.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c122f003440fc5017c72299634c64a7c4caded88" target='_blank'>
              Discovery of Epigenetically Silenced Tumour Suppressor Genes in Aggressive Breast Cancer Through a Computational Approach
              </a>
            </td>
          <td>
            Anne-Laure Vitte, Florent Chuffart, Emmanuelle Jacquet, Eleni Nika, Mireille Mousseau, Ina Jung, S. Tabone-Eglinger, Thomas Bachelot, Sophie Rousseaux, Saadi Khochbin, Ekaterina Bourova-Flin
          </td>
          <td>2025-02-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Aneuploidy is one of the most common mechanisms of adaptation to environmental selection in vegetatively growing cells. However, it is still unclear what advantages aneuploidy provides over other types of genomic alterations. In this study, we aimed to determine if beneficial aneuploidy phenotypes are driven primarily by combinatorial expression changes of multiple genes using budding yeast as a model system. To determine the impact of complex aneuploidy on cellular fitness and resistance to a variety of drug treatments, we generated yeast collections with nearly every possible combination of two chromosome gains or losses. In addition, we genetically dissected aneuploid chromosomes using partial deletions to identify chromosomal regions that contribute to aneuploidy-driven drug resistance. Strong resistance phenotypes consistently came from combinations of chromosomes or chromosomal regions. We identified up to five regions on a single chromosome that contribute to resistance to a single drug treatment. Finally, we found that the strongest aneuploidy phenotypes came from synergistic effects between copy number changes of different chromosomes or chromosomal regions. These results provide insights into how combining subtle gene expression changes across the genome impacts aneuploidy phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288386141655e3d2d8bd580753d7c57e3ca32011" target='_blank'>
              Combinatorial effects of multiple genes contribute to beneficial aneuploidy phenotypes
              </a>
            </td>
          <td>
            M. S. Koller, Claudia Himmelbauer, Sarah Fink, Madhwesh C. Ravichandran, Christopher S. Campbell
          </td>
          <td>2025-03-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25b8ef7036ff1fb3a03cc5984eb57541db6e9f2c" target='_blank'>
              Functional screen identifies RBM42 as a mediator of oncogenic mRNA translation specificity.
              </a>
            </td>
          <td>
            Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Nimmy Mohan, Kaylee Yiakis, Isabelle Liu, Kwun Wah Wen, Grace E. Kim, S. Miglani, Alma L. Burlingame, Hani Goodarzi, Rushika M. Perera, Davide Ruggero
          </td>
          <td>2025-02-04</td>
          <td>Nature cell biology</td>
          <td>2</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6879a0f12b521e3cdca31a407d6e24ccb72cd9a1" target='_blank'>
              AZD7648 (DNA-PKcs inhibitor): a two-edged sword for editing genomes.
              </a>
            </td>
          <td>
            Muhammad Waseem Sajjad, Ifrah Imran, Fatima Muzamil, R. Naqvi, Imran Amin
          </td>
          <td>2025-03-01</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cancer cell lines are valuable models for studying tumor biology, yet their genomic evolution during culture can compromise experimental reproducibility. We conducted a detailed genomic analysis of the triple-negative breast cancer cell line MDA-MB-231, examining sublines obtained from different sources, at various time points, and across distinct passages. We introduce the concept of intraline heterogeneity (ILH) to highlight the genomic variability observed among these sublines. Our analyses revealed extensive genomic diversity, including differences in single nucleotide variants (SNVs) and copy number alterations (CNAs). In particular, CNAs exhibited remarkable heterogeneity, with pronounced chromosomal gains and losses between sublines, underscoring the impact of genomic instability on ILH. These findings suggest that ILH may influence experimental outcomes, emphasizing the importance of considering passage-specific genomic characterization to ensure consistency and reliability in cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c7a90a295650f938dfb30675bf97324372367f9" target='_blank'>
              Intraline genomic heterogeneity of the triple-negative breast cancer MDA-MB-231 cell line
              </a>
            </td>
          <td>
            Nair Varela-Rouco, N. Estévez-Gómez, C. Fernández-Santiago, Laura Tomás, Miriam Pérez, Daniel García-Souto, J. Pasantes, R. Piñeiro, J. M. Alves, D. Posada
          </td>
          <td>2025-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5181ddbe51c7dee5ac944fbf5ce920c81fec81b8" target='_blank'>
              Enhancer regulatory networks globally connect non-coding breast cancer loci to cancer genes
              </a>
            </td>
          <td>
            Yihan Wang, Daniel A Armendariz, Lei Wang, Huan Zhao, Shiqi Xie, Gary C. Hon
          </td>
          <td>2025-01-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Oncogenic gene fusions are key drivers of cancer, yet most remain untargetable by current therapies. Here, we establish CRISPR-PspCas13b as a personalizable platform for systematic silencing of various fusion transcripts. We reveal that recognition and cleavage of the breakpoint sequence by PspCas13b disrupts the fusion transcript, resulting in unexpected RNA nicking and ligation near the cleavage site, which generates out-of-frame, translation-incompetent transcripts. This approach efficiently degrades canonical and drug-resistant BCR::ABL1 mutants (e.g., T315I), a primary cause of resistance to tyrosine kinase inhibitors (TKIs) and relapse in chronic myeloid leukemia (CML). Silencing T315I BCR::ABL1 mRNA in drug-resistant CML cells triggers extensive transcriptomic, proteomic, and phosphoproteomic remodelling, causing erythroid differentiation and apoptosis. Beyond BCR-ABL1 mutants, personalized design of PspCas13b effectively silences other undruggable fusions, including RUNX1::RUNX1T1 and EWSR1::FLI1, key drivers in acute myeloid leukemia and in Ewing sarcoma, respectively. Collectively, this study establishes a framework for systematic, precise, and personalizable targeting of otherwise undruggable or drug-resistant oncogenic transcripts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52d993b12a12216338adf29eefc996f4f14d40a3" target='_blank'>
              Systematic silencing of oncogenic fusion transcripts with ultra-precise CRISPR-Cas13b
              </a>
            </td>
          <td>
            Wenxin Hu, Honglin Chen, J. Casan, Carolyn Shembrey, Steven He, Lauren M Brown, Timothy P. Hughes, Deborah L. White, Antoine de Weck, I. Voskoboinik, Joseph A Trapani, Paul G Ekert, Teresa Sadras, Mohamed Fareh
          </td>
          <td>2025-02-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Immune escape is a critical hallmark of cancer progression and underlies resistance to multiple immunotherapies. However, it remains unclear when the genetic events associated with immune escape occur during cancer development. Here, we integrate functional genomics studies of immunomodulatory genes with a tumor evolution reconstruction approach to infer the evolution of immune escape across 38 cancer types from the Pan-Cancer Analysis of Whole Genomes dataset. Different cancers favor mutations in different immunomodulatory pathways. For example, the antigen presentation machinery is highly mutated in colorectal adenocarcinoma, lung squamous cell carcinoma, and chromophobe renal cell carcinoma, and the protein methylation pathway is highly mutated in bladder transitional cell carcinoma and lung adenocarcinoma. We also observe different timing patterns in multiple immunomodulatory pathways. For instance, mutations impacting genes involved in cellular amino acid metabolism were more likely to happen late in pancreatic adenocarcinoma. Mutations in the glucocorticoid receptor regulatory network pathway tended to occur early, while mutations in the TNF pathways were more likely to occur late in B-cell non-Hodgkin lymphoma. Mutations in the NOD1/2 signaling pathway and DNA binding transcription factor activity tended to happen late in breast adenocarcinoma and ovarian adenocarcinoma. Together, these results delineate the evolutionary trajectories of immune escape in different cancer types and highlight opportunities for improved immunotherapy of cancer. Significance Despite its critical role in cancer progression, the evolution of immune escape is poorly understood. We integrate functional genomics and tumor evolution reconstruction and infer immune escape trajectories across cancer types. Our results have important implications for developing biomarkers for immunoprevention and treatment strategies for immune escape of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a491b543872787c05b0f657359563c86a9093f6" target='_blank'>
              Evolutionary trajectories of immune escape across cancers
              </a>
            </td>
          <td>
            Wenjie Chen, Toby M. Baker, Zhihui Zhang, Huw A. Ogilvie, P. Van Loo, S. Gu
          </td>
          <td>2025-01-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Understanding the mechanistic basis of genetic disease requires annotating the regulatory elements in the human genome. To this end, the International Human Epigenome Consortium (IHEC) has generated thousands of epigenomic datasets—including ChIP-seq, DNase-seq, and ATAC-seq—that measure various biochemical activities in the genome, including transcription factor binding, histone modification, and DNA accessibility. Currently, the predominant methods for integrating these data sets to annotate regulatory elements are segmentation and genome annotation (SAGA) algorithms such as ChromHMM and Segway. The majority of annotations generated by these methods are cell type-specific. However, as the number of profiled cell types has grown into the thousands, using thousands of cell type-specific chromatin state annotations proves undesirable for many applications. Therefore, recently, researchers have sought a single unified annotation of regulatory elements across all cell types, known as a “pan-cell type” annotation. Here, we present a pan-cell type annotation that summarizes all IHEC epigenomes using the recently-developed method, epigenome-ssm. This pan-cell type annotation comprises 33 genome-wide chromatin state feature signal tracks, each of which captures a regulatory program driving genomic activity in one or more cell types. We show that these feature maps constitute an intuitive and visualizable summary of epigenomic data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30732cfd43295287b3ea69f83c8a2c64e5ceda15" target='_blank'>
              Pan-cell type continuous chromatin state annotation of all IHEC epigenomes
              </a>
            </td>
          <td>
            Habib Daneshpajouh, Ismail Moghul, Kay C Wiese, Maxwell W. Libbrecht
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee3a4ef449036c9f0b9b5728bbd660366041a80a" target='_blank'>
              Activation of FGFR genes by genetic and epigenetic alterations in uterine leiomyomas
              </a>
            </td>
          <td>
            Vilja Jokinen, Aurora Taira, Åsa Kolterud, Isa Ahlgren, Kimmo Palin, Riku Katainen, Maritta Räisänen, Eevi Kaasinen, Sini Ilves, A. Raitila, Helena Kopp Kallner, Emma Siili, Ralf Bützow, O. Heikinheimo, A. Pasanen, A. Karhu, Niko Välimäki, L. Aaltonen
          </td>
          <td>2025-02-27</td>
          <td>BJC Reports</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89eb232b40b0237c425e76f52fd0c4fca33e33a1" target='_blank'>
              Pan‑cancer analysis of the oncogenic role of telomeric repeat binding factor 2 (TERF2) in human tumors and in vitro validation in gastric cancer by TERF2 knockdown
              </a>
            </td>
          <td>
            Qiang Ma, Wen Xu, Xiaolan Guo
          </td>
          <td>2025-02-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Background Information Mitosis is crucial for the faithful transmission of genetic material, and disruptions can result in chromosomal instability (CIN), a hallmark of cancer. CIN is a known driver of tumor heterogeneity and anti‐cancer drug resistance, thus highlighting the need to assess CIN levels in cancer cells to design effective targeted therapy. While micronuclei are widely recognized as CIN markers, we have recently identified the toroidal nucleus, a novel ring‐shaped nuclear phenotype arising as well from chromosome mis‐segregation. Results Here, we examined whether increasing nuclear envelope stiffness through lamin A/C overexpression could affect the formation of toroidal nuclei and micronuclei. Interestingly, lamin A/C overexpression led to an increase in toroidal nuclei while reducing micronuclei prevalence. We demonstrated that chromatin compaction and nuclear stiffness drive the formation of toroidal nuclei. Furthermore, inhibition of autophagy and lysosomal function elevated the frequency of toroidal nuclei without affecting the number of micronuclei in the whole cell population. We demonstrated that this divergence between the two CIN biomarkers is independent of defects in lamin A processing. Conclusions and Significance These findings uncover a complex interplay between nuclear architecture and levels of CIN, advancing our understanding of the mechanisms supporting genomic stability and further contributing to cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ec53c24194133a00410a7a2f29a26e297572eb9" target='_blank'>
              Nuclear stiffness through lamin A/C overexpression differentially modulates chromosomal instability biomarkers
              </a>
            </td>
          <td>
            Mireia Bosch-Calvet, Alejandro Pérez-Venteo, Alex Cebria-Xart, Marta Garcia-Cajide, Caroline Mauvezin
          </td>
          <td>2025-02-01</td>
          <td>Biology of the Cell</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35d622cb4408809bb4fc5d6f453132e02ad78dfe" target='_blank'>
              Mechanism for local attenuation of DNA replication at double-strand breaks.
              </a>
            </td>
          <td>
            Robin Sebastian, Eric G Sun, Michael Fedkenheuer, Haiqing Fu, Seolkyoung Jung, Bhushan L. Thakur, Christophe E Redon, Gianluca Pegoraro, Andy D Tran, Jacob M. Gross, Sara Mosavarpour, N. A. Kusi, Anagh Ray, A. Dhall, L. Pongor, Rafael Casellas, M. Aladjem
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="This article reviews the impact of single-cell sequencing (SCS) on cancer biology research. SCS has revolutionized our understanding of cancer and tumor heterogeneity, clonal evolution, and the complex interplay between cancer cells and tumor microenvironment. SCS provides high-resolution profiling of individual cells in genomic, transcriptomic, and epigenomic landscapes, facilitating the detection of rare mutations, the characterization of cellular diversity, and the integration of molecular data with phenotypic traits. The integration of SCS with multi-omics has provided a multidimensional view of cellular states and regulatory mechanisms in cancer, uncovering novel regulatory mechanisms and therapeutic targets. Advances in computational tools, artificial intelligence (AI), and machine learning have been crucial in interpreting the vast amounts of data generated, leading to the identification of new biomarkers and the development of predictive models for patient stratification. Furthermore, there have been emerging technologies such as spatial transcriptomics and in situ sequencing, which promise to further enhance our understanding of tumor microenvironment organization and cellular interactions. As SCS and its related technologies continue to advance, they are expected to drive significant advances in personalized cancer diagnostics, prognosis, and therapy, ultimately improving patient outcomes in the era of precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b38b69885e5e53fa9260297634ff329303ec057" target='_blank'>
              Single-Cell Sequencing: Genomic and Transcriptomic Approaches in Cancer Cell Biology
              </a>
            </td>
          <td>
            Ana Ortega-Batista, Yanelys Jaén-Alvarado, Dilan Moreno-Labrador, Natasha Gómez, Gabriela García, Erika N. Guerrero
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Background: Aberrant expression and mutations in the fibroblast growth factor (FGF) family play crucial roles in cell differentiation, growth, and migration, contributing to tumor progression across various cancers. Nasopharyngeal carcinoma (NPC), a malignancy prevalent in East Asia, is primarily treated with radiotherapy; however, radioresistance remains a major challenge, leading to recurrence and poor outcomes. While FGFs are known to activate signaling pathways such as MAPK, PI3K/AKT, and JAK/STAT to promote cancer progression, the specific role of individual FGFs in NPC radioresistance remains unclear. Emerging evidence highlights FGF5 as a key player in NPC progression, metastasis, and radioresistance, underscoring its potential as a therapeutic target to overcome treatment resistance and improve clinical outcomes. Methods: We analyzed single nucleotide variation (SNV) data, gene expression, and DNA methylation patterns using cancer datasets, including TCGA and GTEx, to investigate FGF5 expression. Differentially expressed genes (DEGs) were identified and interpreted using functional enrichment analysis, while survival analysis and gene set enrichment analysis (GSEA) were conducted to identify clinical correlations. DNA methylation patterns were specifically assessed using the HumanMethylation850 BeadChips on tissue samples from nine recurrent and nine non-recurrent NPC patients. Functional assays, including cell viability, migration, invasion, and clonogenic survival assays, were performed to evaluate the effects of FGF5 on NPC cell behavior in vitro and in vivo. Results: FGF5 showed elevated SNV frequencies across multiple cancers, particularly in HNSC and NPC. DNA methylation analysis revealed an inverse relationship between FGF5 expression and methylation levels in recurrent NPC tumors. Functional assays demonstrated that FGF5 enhances migration, invasion, and radioresistance in NPC cells. High FGF5 expression was associated with reduced distant metastasis-free survival (DMFS) and increased radioresistance, highlighting its role in metastatic progression and recurrence. Conclusions: FGF5 plays a significant role in the progression and recurrence of nasopharyngeal carcinoma. Its elevated expression correlates with increased migration, invasion, and radioresistance as well as reduced distant metastasis-free survival. These findings suggest that FGF5 contributes to the metastatic and recurrence potential of NPC, making it a potential target for therapeutic intervention in treating these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5304b34e9ca6a3e7c770b19af10ccf63ce40f9c2" target='_blank'>
              Multi-Omics Characterization of Genome-Wide Abnormal DNA Methylation Reveals FGF5 as a Diagnosis of Nasopharyngeal Carcinoma Recurrence After Radiotherapy
              </a>
            </td>
          <td>
            Z. Long, Ran Ding, Ting-Qiu Quan, Rui Xu, Zhuo-Hui Huang, D. Wei, W. Zheng, Ying Sun
          </td>
          <td>2025-02-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The tumor suppressor p53 can trigger tumor resistance to chemotherapy by facilitating DNA damage repair and maintaining genomic integrity. Here, we report that a p53-induced circular RNA circASCC3 promotes chemotherapeutic resistance by resolving R-loops. Our results reveal that p53 directly activates the transcription of ASCC3, the host gene of circASCC3. In addition, the RNA-binding protein SFPQ is identified to inhibit the formation of circASCC3 by associating with its flanking regions. Importantly, p53 facilitates the formation of circASCC3 by repressing the expression of SFPQ. CircASCC3 has a marginal effect on the survival and growth of cancer cells under normal growing conditions but surprisingly boosts their survival and growth in response to DNA damage stress. Mechanistic analysis reveals that circASCC3 binds to the DEAD-box RNA helicase DDX5 to inhibit its proteasomal degradation. This results in the prevention of R-loop accumulation due to DNA damage, thereby conferring tumor resistance to chemotherapy. Together, our study uncovers that p53 activates circASCC3 to promote R-loop resolution, which maintains genomic stability and potentially contributes to chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c687b99bd05ab27cade57161e55043204c9469c" target='_blank'>
              p53 activates circASCC3 to repress R-loops and enhance resistance to chemotherapy.
              </a>
            </td>
          <td>
            Mingming Cao, Yu Gan, Yingdan Huang, Jing Tong, Chen Xiong, Yajie Chen, Bing Chen, Ruixuan Huang, B. Xie, Jun Deng, Shenglin Huang, Xianghuo He, Qian Hao, Xiang Zhou
          </td>
          <td>2025-03-11</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Clonal lineage inference ("tumor phylogenetics") has become a crucial tool for making sense of somatic evolution processes that underlie cancer development and are increasingly recognized as part of normal tissue growth and aging. The inference of clonal lineage trees from single-cell sequence data offers particular promise for revealing processes of somatic evolution in unprecedented detail. However, most such tools are based on fairly restrictive models of the types of mutation events observed in somatic evolution and of the processes by which they develop. The present work seeks to enhance the power and versatility of tools for single-cell lineage reconstruction by making more comprehensive use of the range of molecular variant types by which tumors evolve. We introduce Sc-TUSV-ext, an integer linear programming-based tumor phylogeny reconstruction method that, for the first time, integrates single nucleotide variants, copy number alterations, and structural variations into clonal lineage reconstruction from single-cell DNA sequencing data. We show on synthetic data that accounting for these variant types collectively leads to improved accuracy in clonal lineage reconstruction relative to prior methods that consider only subsets of the variant types. We further demonstrate the effectiveness of real data in resolving clonal evolution in the presence of multiple variant types, providing a path toward more comprehensive insight into how various forms of somatic mutability collectively shape tissue development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279bd9fa8c8b49a99cb9a4bb9596137f5ef0a74d" target='_blank'>
              Sc-TUSV-Ext: Single-Cell Clonal Lineage Inference from Single Nucleotide Variants, Copy Number Alterations, and Structural Variants.
              </a>
            </td>
          <td>
            N. Bristy, Xuecong Fu, Russell Schwartz
          </td>
          <td>2025-03-06</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bea01c369ba68e6382727474294c60f827396f68" target='_blank'>
              Super-enhancers in tumors: unraveling recent advances in their role in Oncogenesis and the emergence of targeted therapies
              </a>
            </td>
          <td>
            Yumeng Ji, Baixue Li, Rongjin Lin, Jing Yuan, Yang Han, Yuping Du, Yang Zhao
          </td>
          <td>2025-01-21</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Aneuploidy is a hallmark of cancer, yet the specific genes driving recurrent chromosome-arm losses remain largely unknown. Here, we present a systematic framework integrating gene expression, mutation, and copy number data to identify candidate driver genes of cancer type-specific recurrent chromosome-arm losses across 20 cancer types, using ∼7,500 tumors from The Cancer Genome Atlas. By analyzing focal deletions and point mutations that co-occur with, or are mutually exclusive with, chromosome-arm losses, we pinpoint 311 candidate drivers associated with 160 cancer type-specific recurrent events. Our approach identifies known aneuploidy drivers such as TP53 and PTEN, while revealing multiple novel candidates, including established tumor suppressor genes not previously linked to aneuploidy. Furthermore, we leverage gene expression changes associated with these chromosome-arm losses to propose pathway-level alterations that may drive cancer progression. Integrating these findings highlights key candidate drivers underlying the observed gene expression alterations, thereby reinforcing their biological relevance. This work provides the first comprehensive catalogue of candidate driver genes for recurrently lost chromosome-arms in human cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e688482cb691231e9728515c865331942ee770" target='_blank'>
              Integrating gene expression, mutation and copy number data to identify driver genes of recurrent chromosome-arm losses
              </a>
            </td>
          <td>
            Ron Saad, Ron Shamir, Uri Ben-David
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 291


 Background:
 5-hydroxymethylcytosine (5hmC) is an epigenetic modification which regulates gene expression and is associated with active transcription. The optimization of 5hmC sequencing in cell-free DNA (cfDNA) could therefore enable assessment of gene activity through a liquid biopsy. We aimed to investigate the 5hmC landscape of colorectal cancer (CRC) in plasma-derived cfDNA to evaluate the potential of 5hmC modifications as a liquid biopsy-based biomarker of CRC.
 Methods:
 Genome-wide 5hmC modifications were analyzed with a low-input whole-genome 5hmC sequencing method based on selective chemical labeling in cfDNA from 72 CRC patients and 70 healthy samples. Differential 5hmC analysis between CRC and healthy samples was conducted using DESeq2 (padj < 0.05 and log2FC <= -1 and >=1). Low-pass whole genome sequencing (LP-WGS) was performed on all cfDNA samples. Tumor fraction was estimated using ichorCNA to classify samples into high ct-fraction (>=0.1) and low ct-fraction (<0.1) groups. Nucleosome profiling was performed on LP-WGS to reveal chromatin accessibility patterns. Elastic Net Regression (ENR) was applied to identify changes in 5hmC levels for classifying samples as cancerous or healthy.
 Results:
 Differential cfDNA 5hmC levels of 139 genes distinguished high ct-fraction CRC (n=16) and healthy samples (n=70). Although the remaining CRC samples (n=56) had similar ct-fractions as the healthy samples, as estimated by ichorCNA, we found a differentially hydroxymethylated gene signature could discriminate between these low ct-fraction CRC and healthy samples. Nucleosome profiling analysis with LP-WGS in cfDNA was used to identify active transcriptional drivers of the genes with increased 5hmC levels. The analysis revealed increased chromatin accessibility near the binding sites for the transcription factor caudal-related homeobox transcription factor 2 (
 CDX2
 ), a master transcriptional regulator of intestinal cell fate and cancer, in CRC patients relative to healthy samples. Corroborating this,
 CDX2
 and its downstream targets had increased 5hmC levels in cfDNA in CRC relative to healthy samples, consistent with colorectal-derived tumor DNA in CRC plasma samples.
 Conclusions:
 Genes with differential 5hmC levels in cfDNA can discriminate CRC and healthy samples. Our 5hmC signature identified CRC samples that did not harbor copy number gains or losses; which is typically used to estimate ct-fraction and routinely included in targeted ctDNA sequencing assays. Integration of 5hmC and LP-WGS can be used to identify gene activation in disease. In the context of CRC, CDX2 displayed increased chromosome accessibility in the promoters of its target genes which in turn displayed increased 5hmC levels. Further studies are aimed at optimizing and validating 5hmC-based biomarkers throughout patient treatment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80a577e616a7dc533785f9599dc6feaa89e4aef7" target='_blank'>
              5-hydroxymethylcytosine as a liquid biopsy biomarker in colorectal cancer.
              </a>
            </td>
          <td>
            Christopher A. Maher, Netra Navadkar, Muheng Liao, De'Vana Crutcher, Jacqueline L. Mudd, Ceyda Coruh, G. Guler, Y. Ning, Shimul Chowdhury, Emily Rozycki, Amy Ly, S. Levy, Ryan Fields
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Infections by the major opportunistic pathogen of human Candida albicans are commonly treated with echinocandin (ECN) drugs. However, C. albicans can adapt to grow in the presence of certain amounts of ECNs. Prior studies by several laboratories have defined multiple genes, as well as mechanisms involving induced aneuploidy, that can govern this. Still, the mechanisms of ECN adaptation are not fully understood. Here, we use genome-wide profiling of chromatin accessibility by ATAC-seq to determine if ECN adaptation is reflected in changes in the chromatin landscape in the absence of aneuploidy. We find that drug adaptation is coupled with multiple changes in chromatin accessibility genome-wide, which occur predominantly in gene promoter regions. Areas of increased accessibilities in promoters are enriched with the binding motifs for at least two types of transcription factors: zinc finger and basic leucine zipper. We also find that chromatin changes are often associated with differentially expressed genes including genes with functions relevant to the ECN-adapted phenotype, such as cell wall biosynthesis. Consistent with this, we find that the cell wall is remodeled in ECN-adapted mutants, with chitin up and glucan down and increased cell surface exposure. A full understanding of ECN adaptation processes is of critical importance for the prevention of clinical resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3773ea0cdd0e2f8c5db7467a8fc51861d1557f3" target='_blank'>
              Echinocandin Adaptation in Candida albicans Is Accompanied by Altered Chromatin Accessibility at Gene Promoters and by Cell Wall Remodeling
              </a>
            </td>
          <td>
            S. Sah, Anshuman Yadav, Tyler Stahl, Jeffrey J. Hayes, Michael Bulger, E. Rustchenko
          </td>
          <td>2025-02-01</td>
          <td>Journal of Fungi</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09816baa7e13a8a81394d1d42faa90a68058ed21" target='_blank'>
              Stochastic demethylation and redundant epigenetic suppressive mechanisms generate highly heterogeneous responses to pharmacological DNA methyltransferase inhibition
              </a>
            </td>
          <td>
            Mie K. Jakobsen, S. Traynor, A. Y. Nielsen, Christina Dahl, Mette Staehr, S. T. Jakobsen, M. S. Madsen, Rasmus Siersbaek, M. Terp, Josefine B. Jensen, C. B. Pedersen, Anup Shrestha, J. R. Brewer, P. H. Duijf, Odd L. Gammelgaard, Henrik J Ditzel, Alexei F Kirkin, Per Guldberg, Morten F. Gjerstorff
          </td>
          <td>2025-01-23</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Unscheduled whole genome duplication (WGD), also described as unscheduled or non-physiological polyploidy, can lead to genetic instability and is commonly observed in human cancers. WGD generates DNA damage due to scaling defects between replication factors and DNA content. As a result, DNA damage repair mechanisms are thought to be critical for ensuring cell viability and proliferation under these conditions. In this study, we explored the role of homologous recombination and Holliday junction resolution in non-physiological polyploidy in vivo. Using Drosophila genetics and high-resolution imaging, we identified a key and surprising role for Gen/Gen1 nuclease. Our findings revealed that loss-of-function and overexpression of Gen have opposing effects, delaying or accelerating the proliferation of polyploid cells, respectively. These changes ultimately impact cell proliferation, nuclear asynchrony and mitotic DNA damage levels. Surprisingly, our findings show that this effect is unrelated with the expected Gen’s function in DNA damage repair. Instead, Gen seems to influence polyploid DNA replication rates. This work identifies a novel function for Gen nuclease and provides new insights into the cellular and molecular requirements of non-physiological polyploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbd8ff52437b49af76b524b72bb72040b51dd497" target='_blank'>
              A novel DNA repair-independent role for Gen nuclease in promoting unscheduled polyploidy cell proliferation
              </a>
            </td>
          <td>
            Manon Budzyk, Anthony Simon, Anne-Sophie Macé, R. Basto
          </td>
          <td>2025-02-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background: In eukaryotes with a double-stranded linear DNA genome, the loss of terminal DNA during replication is inevitable due to an end-replication problem; here, telomeres serve as a buffer against DNA loss. Thus, the activation of the telomere maintenance mechanism (TMM) is a prerequisite for malignant transformation. Methods: We compared neurofibroma (NF, benign) and malignant peripheral nerve sheath tumors (MPNSTs) occurring in the same patient with type 1 neurofibromatosis, where each NF–MPNST pair shared the same genetic background and differentiation lineage; this minimizes the genetic bias and contrasts only those changes that are related to malignant transformation. A total of 20 NF–MPNST pairs from 20 NF1 patients were analyzed. Whole-transcriptome sequencing (WTS) was conducted to profile the transcriptional relationship, and whole-genome sequencing (WGS) was performed to measure the telomere length. Results: We identified 22 differentially expressed genes (DEGs) during the malignant transformation of MPNSTs. Among them, NELL2 activated PAX7, which sequentially activated RAD52, the recombinase of RAD52-dependent alternative lengthening of telomeres (ALT). RAD52 elongated MPNSTs–telomeres (p = 0.017). Otherwise, neither NELL2 nor PAX7 affected telomere length (p = 0.647 and p = 0.354, respectively). RAD52 increased MPNSTs–telomeres length, independently of NELL2 and PAX7 in multiple analyses (p = 0.021). The group with increased telomere length during the malignant transformation showed inferior overall survival (OS) (HR = 3.809, p = 0.038) to the group without increased telomere length. Accordingly, the group with increased PAX7 showed inferior OS (HR = 4.896, p = 0.046) and metastasis-free survival (MFS) (HR = 9.129, p = 0.007) in comparison to the group without increased PAX7; the group with increased RAD52 showed inferior MFS (HR = 8.669, p = 0.011) in comparison to the group without increased RAD52. Conclusions: We suggest that the NELL2-PAX7 transcriptional cascade activates RAD52-dependent ALT to increase telomere length during the malignant transformation of MPNSTs, resulting in a poor prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec682f73e8b37948b58b7e542b7ea83e47ad44c3" target='_blank'>
              NELL2-PAX7 Transcriptional Cascade Suggests Activation Mechanism for RAD52-Dependent Alternative Lengthening of Telomeres During Malignant Transformation of Malignant Peripheral Nerve Sheath Tumors: Elongation of Telomeres and Poor Survival
              </a>
            </td>
          <td>
            Jungwoo Lee, Eunji Choi, Hyoju Kim, Young-Joon Kim, Seung Hyun Kim
          </td>
          <td>2025-01-23</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe40161d2aa729850b5719de103597d5d50c55e2" target='_blank'>
              EV DNA from pancreatic cancer patient-derived cells harbors molecular, coding, non-coding signatures and mutational hotspots
              </a>
            </td>
          <td>
            Appolinaire A Olou, Wesley A Tom, Gary D. Krzyzanowski, Chao Jiang, Dinesh S Chandel, Nirmalee Fernando, Adrian W. Draney, Joel Destino, Danny R Welch, M. R. Fernando
          </td>
          <td>2025-03-05</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22ab73f4c8cf8c6df493069f963fc3cc43cc9298" target='_blank'>
              The epigenetic basis of hepatocellular carcinoma - mechanisms and potential directions for biomarkers and therapeutics.
              </a>
            </td>
          <td>
            Hong-Yi Lin, Ah-Jung Jeon, Kaina Chen, Chang Jie Mick Lee, Lingyan Wu, S. Chong, C. Anene-Nzelu, R. Foo, P. Chow
          </td>
          <td>2025-03-08</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Cancers that retain wild type TP53 presumably harbor other clonal alterations that permitted their precursors to bypass p53-mediated growth suppression. Consequently, studies that employ TP53-wild type cancer cells and their isogenic derivatives may systematically fail to appreciate the full scope of p53 functionality. Several TP53 phenotypes are known to be absent in the widely used isogenic HCT116 colorectal cancer (CRC) model, which originated from a tumor that had retained wild type TP53. In contrast, we show that restoration of p53 in the TP53-mutant CRC cell line DLD-1 impeded cell proliferation, increased levels of senescence and sensitized cells to ionizing radiation (IR). To study p53 in a non-cancer context, we disrupted TP53 in hTERT-RPE1 cells. Derived from primary cells that were immortalized in vitro, hTERT-RPE1 expressed striking p53-dependent phenotypes and appeared to select for p53 loss during routine culture. hTERT-RPE1 expressed a p53-responsive transcriptome that was highly representative of diverse experimental systems. We discovered several novel downstream p53 targets of potential clinical relevance including ALDH3A1, which is involved in the detoxification of aldehydes and the metabolism of reactive oxygen species, and nectin cell adhesion molecule 4 (NECTIN4) which encodes a secreted surface protein that is overexpressed in many tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56bfbd25a9a6de3dee6da6ae82f799aa171d1483" target='_blank'>
              Robust p53 phenotypes and prospective downstream targets in telomerase-immortalized human cells
              </a>
            </td>
          <td>
            Jessica J. Miciak, Lucy Petrova, Rhythm Sajwan, Aditya Pandya, Mikayla Deckard, Andrew J Munoz, Fred Bunz
          </td>
          <td>2025-02-18</td>
          <td>Oncotarget</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Merkel cell polyomavirus (MCPyV) causes most Merkel cell carcinomas (MCC). The virus is one of the few known human tumor viruses, and due to its direct role in this skin cancer development, it is a bona fide model for viral tumorigenesis and tumorigenesis in general. Chronic viruses in humans such as MCPyV are highly adapted to their host and current models to study infection, persistence and pathogenesis are highly limited. We here use an induced pluripotent stem cell (iPSC)-derived hair-bearing skin organoid (SkO) system to demonstrate efficient viral infection, progression and spread of MCPyV. Using bulk-, single cell - and spatial-transcriptomics, combined with immunostaining and nucleic acid hybridization technologies, we show that MCPyV ensures persistence due to a quasi-latency state within the majority of dermal fibroblasts carry the viral genome. Further, we identify the cell type of productive infection with papillary fibroblasts and dermal sheath fibroblasts supporting viral replication and progeny production. Our high-resolution methods demonstrate that the virus in these cells evades the innate immune response, as evidenced by the efficacy of interferon-beta treatment or ruxolitinib, a JAK/STAT inhibitor, in suppressing or stimulating viral replication. We show that iPSC-derived SkOs are able to support infection and long-term persistence of the virus under conditions very similar to those found in humans. Thus, this infection model provides a robust platform for understanding and characterizing the interaction of this virus with the immune system in infection, testing treatment strategies to control reactivation and map processes involved in tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebeb9fa82784b82f60e0038c9b75808f64425555" target='_blank'>
              Merkel cell polyomavirus infection and persistence modelled in skin organoids
              </a>
            </td>
          <td>
            Silvia Albertini, Manja Czech‐Sioli, Thomas Günther, Sanamjeet Virdi, Patrick Blümke, Lisann Röpke, Veronika Brinschwitz, M. Dandri, L. Allweiss, Rudolf Reimer, Carola Schneider, Arne Hansen, S. Krasemann, E. Wyler, M. Landthaler, A. Grundhoff, Nicole Fischer
          </td>
          <td>2025-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="
 Precision medicine in oncology is rapidly emerging as the next frontier for cancer therapy, and accordingly there exists a need to identify new therapeutically actionable targets. Gene fusions have long been established as biomarkers and therapeutic targets for cancer treatment, with targetable fusions such as BCR-ABL1, EML4-ALK, and TPM3-NTRK1 already having shown sustained success in the clinic. While historically studies have focused on DNA rearrangements as the source of fusion transcripts and proteins, recent advances in RNA sequencing have revealed a new landscape of fusions originating at the RNA level through intergenic cis and trans splicing. Combined, this class of molecule, termed “chimeric RNA”, presents new opportunities for molecular profiling of the fusion transcriptome. The purpose of this study is to systematically characterize chimeric RNAs across cancer to identify molecular drivers of cell growth. To build an atlas of chimeric transcripts, we applied multiple fusion-calling programs followed by stringent filtering to predict fusions in 9509 primary tumors from The Cancer Genome Atlas (TCGA) and 1019 cell lines from The Cancer Cell Line Encyclopedia (CCLE). From this we identified 7047 recurrent chimeric transcripts, 2085 of which were not found in normal tissue. To further characterize these fusions, we integrated matched structural variant predictions from whole-genome sequencing to establish the origin of formation at the DNA or RNA level and mapped transcript breakpoint coordinates to protein annotations to establish the potential for the formation of a fusion protein. To identify fusions driving growth and proliferation we developed a functional screen, integrating shRNA dependency screens from DepMap with our cell line chimeric RNA predictions. By mapping the shRNA probes to the sequences of the fusion parental genes, we are able to compare the difference in cell growth between the chimera-mapping and parent specific probes. Using this method, we are able to identify fusions functioning as cancer dependencies independently of the parental genes. Our method accurately detected well-established fusions such as EML4-ALK, FGFR3-TACC3, and TPM3-NTRK1 and showed a significant enrichment of oncogenic fusions from the COSMIC database, fusions containing established oncogenes, and fusions predicted to code for proteins. Out of the 2225 total fusion transcripts screened, we identified 47 cancer-specific transcripts predicted to function as promoters of growth in cancer. Based on degree of essentiality, expression across multiple cancer indications, and potential of forming a targetable fusion protein, we selected candidate fusions for further characterization through cell-based gain and loss of function studies to confirm oncogenicity. Our results reveal a new landscape of chimeric transcripts as drivers of cancer and hold potential for the development of new precision medicine-based treatments.
 Citation Format: Samir Lalani, Mason Ingram, Sehajroop Gadh, Hui Li. Functional profiling of chimeric RNA transcripts identifies new biomarkers and therapeutic targets for precision medicine in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr A035.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a306f36a45e59527afe8dc83f82d2317d23a26f" target='_blank'>
              Abstract A035: Functional profiling of chimeric RNA transcripts identifies new biomarkers and therapeutic targets for precision medicine in cancer
              </a>
            </td>
          <td>
            Samir Lalani, Mason Ingram, Sehajroop Gadh, Hui Li
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract During X chromosome inactivation (XCI), Xist RNA establishes silencing by coating the chromosome in cis and binding diverse proteins to promote formation of a heterochromatic domain. However, Xist repeat B role beyond initiation of XCI remains unclear. Here, we find that loss of Xist repeat B in female mice allows survival and leads to a small body size persisting throughout life. Epigenetic and transcriptomic analyses reveal low levels of H3K27me3 and H2AK119ub occupancy on the X chromosome, except in certain CpG island regions, and partial reactivation of X-linked genes on the inactive X across multiple tissues. Notably, overdosage of Usp9x promotes centrosome amplification and chromosome instability. We further demonstrate that Usp9x overdosage alters asymmetric cell division, thereby affecting the process of cell differentiation. Thus, Xist repeat B is necessary for gene-specific silencing during XCI maintenance and impacts cell proliferation and differentiation during development. This provides insights into repeat B importance in maintaining XCI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e9e14e58c0496a8253f79c766a6a837dca408f9" target='_blank'>
              Deletion of Xist repeat B disrupts cell cycle and asymmetric cell division through Usp9x hyperactivation in mice
              </a>
            </td>
          <td>
            Mingming Liang, Lichao Zhang, Heng Gong, Li Yang, Haijun Wang, Na Song, Liangxue Lai, Wanhua Xie, Zhanjun Li
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="All cancers are diseases of the genome, since the cancer cell genome typically consists of 10,000s of passenger alterations, 5-10 biologically relevant alterations, and 1-2 "actionable" alterations. Therefore, somatic mutations in cancer cells can have diagnostic, prognostic, and predictive value. Traditional methods are widely used for testing, such as immunohistochemistry, Sanger sequencing, and allele-specific PCR. However, due to the low throughput, these methods are focused exclusively on testing the most common mutations in target genes. The modern next generation sequencing (NGS) is a technology that enables precision oncology in its current form. ESCAT and ESMO Guidelines defined NGS for routine use in patients with advanced cancers such as non-squamous non-small cell lung cancer, prostate cancer, ovarian cancer, and cholangiocarcinoma. The high sensitivity of the NGS method allows it to be used to search for specific mutations in circulating tumor DNA in blood plasma and other body fluids. NGS testing has evolved from hotspot panels, actionable gene panels, and disease-specific panels to more comprehensive panels. The exome and whole genome sequencing approaches are just beginning to emerge, that is why panel-based testing remains most optimal in oncology practice. NGS is also widely used to identify new and rare mutations in cancer genes and detect inherited cancer mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/207a8f97841f6a0acdc06c228c5f1967ea8ee827" target='_blank'>
              Application of Next-Generation Sequencing to Realize Principles of Precision Therapy in Management of Cancer Patients.
              </a>
            </td>
          <td>
            N. Khranovska, O. Gorbach, O. Skachkova, G. Klimnyuk
          </td>
          <td>2025-02-20</td>
          <td>Experimental oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56da4ec3ca3ec43d5746b51ede7497a19adee094" target='_blank'>
              Regulatory non-coding somatic mutations as drivers of neuroblastoma
              </a>
            </td>
          <td>
            Annalaura Montella, Matilde Tirelli, V. A. Lasorsa, Vincenzo Aievola, Vincenza Cerbone, Rosa Manganiello, Achille Iolascon, Mario Capasso
          </td>
          <td>2025-01-23</td>
          <td>British Journal of Cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c6b9704e0bc8ddb9fedc0099d11922be33aebac" target='_blank'>
              Deletion of 17p in cancers: Guilt by (p53) association
              </a>
            </td>
          <td>
            Francisca van Kampen, Abigail Clark, Jamie Soul, Aditi Kanhere, Mark Glenn, Andrew R Pettitt, Nagesh Kalakonda, J. Slupsky
          </td>
          <td>2025-02-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39fada8ad025d98e9dfc80fcb5a196964bddeb3e" target='_blank'>
              Decoding chromosomal instability insights in CRC by integrating omics and patient-derived organoids
              </a>
            </td>
          <td>
            Federica Papaccio, M. Cabeza-Segura, B. Garcia-Micò, F. Gimeno-Valiente, S. Zúñiga-Trejos, V. Gambardella, M. F. Gutierrez-Bravo, C. Martínez-Ciarpaglini, P. Rentero-Garrido, Tania Fleitas, Susana Roselló, J. Carbonell-Asins, M. Huerta, D. Moro-Valdezate, D. Roda, N. Tarazona, Manuel M. Sánchez del Pino, Andrés Cervantes, Josefa Castillo
          </td>
          <td>2025-02-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Comparative genomics has emerged as a pivotal methodology for elucidating genetic variations in microbial studies. However, conventional analytical approaches for diploid and polyploid microorganisms have demonstrated limited efficacy in discriminating new mutations from background heterozygosity. This study presents an innovative microbial comparative genomics framework adapted from tumor-normal sequencing methodology. Our approach establishes the original strain as the “normal sample” and the derived strain as the “tumor sample”, enabling precise identification of new mutations (“somatic variants”) while filtering pre-existing heterozygous sites (“germline variations”). The analytical pipeline also includes assessment of loss-of-heterozygosity (LOH) events and genome-wide detection of copy number variations (CNVs) with resolution to identify both regional CNV and whole-chromosome aneuploidy through integrated CNV and variant allele frequency analysis. We validated this framework using diploid Saccharomyces cerevisiae strains before successfully extending its application to Kluyveromyces marxianus, Candida spp., and Hortaea werneckii, encompassing haploid, diploid, polyploid, and aneuploid states. The methodology revealed previously undetected variations across experimental evolution studies, demonstrating superior resolution compared to conventional approaches. This adaptable platform establishes a new paradigm for microbial genome studies, particularly for organisms with diploid or polyploid states where traditional comparative genomics methods prove inadequate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c06f8340582c06e98073914beb7e687d2327b602" target='_blank'>
              Advancing Microbial Comparative Genomics Through Tumor-Normal Inspired Framework
              </a>
            </td>
          <td>
            Junjie Yang, Sheng Yang
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The structural integrity of the chromosomes is essential to every functional process within eukaryotic nuclei. Chromosomes are DNA-histone complexes that are essential for the inheritance of genetic information to the offspring, and any defect in them is linked to mitotic errors, cancer growth, and cellular aging. Changes in the mechanical properties of a chromosome could lead to its compromised function and stability, leading to chromosome breaks. Here, we studied the changes in chromosome physical properties using metaphase chromosomes isolated from moderately malignant (MCF7) and highly malignant (MDA-MB-231) human breast cancer cells exposed to valproic acid (VPA), a known epigenetic modifier drug involved in histone hyperacetylation and DNA demethylation. Due to chromosomal structural intricacy and preparative and technical limitations of analytical tools, we employed a label-free atomic force microscopy approach for simultaneously visualizing and mapping single chromosome elasticity and stretching modulus. Additionally, we performed electron transport characteristics through metaphase chromosomes to elucidate the effect of VPA. The chromosomal elasticity and electron transport alterations are manifestations of VPA-mediated chromatin’s epigenetic changes. Our multiparametric strategy, as shown by receiver operating characteristics analyses with the physical properties of chromosomes, offers a new scope in terms of analytical tools for studying chromosomal structural changes/aberrations linked to cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4081e3a40a851c1d9b5593b224de0890423bfe7" target='_blank'>
              Epigenetic Modifier Drug Valproic Acid Enhances Cancer Metaphase Chromosome Elasticity and Electron Transport: An Atomic Force Microscopy Approach
              </a>
            </td>
          <td>
            Tanya Agrawal, Debashish Paul, Amita Mishra, Ganesan Arunkumar, T. Rakshit
          </td>
          <td>2025-02-12</td>
          <td>JACS Au</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="This paper provides a critical analysis of the molecular mechanisms presently used to explain transcriptional strand asymmetries of single base substitution (SBS) signatures observed in cancer genomes curated at the Catalogue of Somatic Mutations in Cancer (COSMIC) database (Wellcome Trust Sanger Institute). The analysis is based on a deaminase-driven reverse transcriptase (DRT) mutagenesis model of cancer oncogenesis involving both the cytosine (AID/APOBEC) and adenosine (ADAR) mutagenic deaminases. In this analysis we apply what is known, or can reasonably be inferred, of the immunoglobulin somatic hypermutation (Ig SHM) mechanism to the analysis of the transcriptional stand asymmetries of the COSMIC SBS signatures that are observed in cancer genomes. The underlying assumption is that somatic mutations arising in cancer genomes are driven by dysregulated off-target Ig SHM-like mutagenic processes at non-Ig loci. It is reasoned that most SBS signatures whether of “unknown etiology” or assigned-molecular causation, can be readily understood in terms of the DRT-paradigm. These include the major age-related “clock-like” SBS5 signature observed in all cancer genomes sequenced and many other common subset signatures including SBS1, SBS3, SBS2/13, SBS6, SBS12, SBS16, SBS17a/17b, SBS19, SBS21, as well as signatures clearly arising from exogenous causation. We conclude that the DRT-model provides a plausible molecular framework that augments our current understanding of immunogenetic mechanisms driving oncogenesis. It accommodates both what is known about AID/APOBEC and ADAR somatic mutation strand asymmetries and provides a fully integrated understanding into the molecular origins of common COSMIC SBS signatures. The DRT-paradigm thus provides scientists and clinicians with additional molecular insights into the causal links between deaminase-associated genomic signatures and oncogenic processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a875e39ec433bc73f24db2e1195da36a00350df" target='_blank'>
              Deaminase-Driven Reverse Transcription Mutagenesis in Oncogenesis: Critical Analysis of Transcriptional Strand Asymmetries of Single Base Substitution Signatures
              </a>
            </td>
          <td>
            Edward J. Steele, R. Lindley
          </td>
          <td>2025-01-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Genome functions are regulated by the means of chromatin and 3D chromosome organization. Dynamic condensation and relaxation of chromosomes during cell cycle is largely controlled by the Condensin complexes. We mapped mutants in Condensin II subunits SMC2A, CAP-D3 and CAP-H2 as hypersensitive to DNA-protein crosslink (DPC) inducer zebularine. This suggested that Condensin II complex is required for resistance to genotoxic stress in Arabidopsis and prompted us to explore the underlying phenotypes. We show that the Condensin II role in resistance to zebularine is independent of the DNA damage response signaling by SOG1 and the homology-directed repair pathway. Furthermore, we found that Arabidopsis Condensin II mutants have incompletely condensed mitotic chromosomes and show abnormal chromatin connections during anaphase. Based on the structural and temporal analyses, we propose that the connections represent catenated chromatids. The chromosome connections were more frequent when the Condensin II mutants were treated by zebularine or TOP2 crosslinker and inhibitor ICRF-187. Altogether, we demonstrate that the proper large-scale chromatin organization by Condensin II is important for resistance to DNA damage response-inducing agents in Arabidopsis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6084d605a281f1ce54dec4a816d64fd93e825187" target='_blank'>
              Condensin II mediates efficient chromatid resolution and resistance to genotoxic stress in Arabidopsis
              </a>
            </td>
          <td>
            Jovanka Vladejić, Klára Procházková, E. Tomaštíková, Zynke de Cock, Jana Zwyrtková, Kateřina Kaduchová, Markéta Pavlíková, A. Pečinka
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Prostate cancer is a common malignancy that in 5%-30% leads to treatment-resistant and highly aggressive disease. Metastasis-potential and treatment-resistance is thought to rely on increased plasticity of the cancer cells-a mechanism whereby cancer cells alter their identity to adapt to changing environments or therapeutic pressures to create cellular heterogeneity. To understand the molecular basis of this plasticity, genomic studies have uncovered genetic variants to capture clonal heterogeneity of primary tumors and metastases. As cellular plasticity is largely driven by non-genetic events, complementary studies in cancer epigenomics are now being conducted to identify chromatin variants. These variants, defined as genomic loci in cancer cells that show changes in chromatin state due to the loss or gain of epigenomic marks, inclusive of histone post-translational modifications, DNA methylation and histone variants, are considered the fundamental units of epigenomic heterogeneity. In prostate cancer chromatin variants hold the promise of guiding the new era of precision oncology. In this review, we explore the role of epigenomic heterogeneity in prostate cancer, focusing on how chromatin variants contribute to tumor evolution and therapy resistance. We therefore discuss their impact on cellular plasticity and stochastic events, highlighting the value of single-cell sequencing and liquid biopsy epigenomic assays to uncover new therapeutic targets and biomarkers. Ultimately, this review aims to support a new era of precision oncology, utilizing insights from epigenomics to improve prostate cancer patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8f247f6de2138b02c15cb85166ee95210b75593" target='_blank'>
              Epigenomics-guided precision oncology: Chromatin variants in prostate tumor evolution.
              </a>
            </td>
          <td>
            K. Furlano, Tina Keshavarzian, Nadine Biernath, Annika Fendler, M. De Santis, Joachim Weischenfeldt, Mathieu Lupien
          </td>
          <td>2025-01-23</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e2c4e60ca9cf87a1f35e280844de54a071afba7" target='_blank'>
              Super-enhancer-hijacking RBBP7 potentiates metastasis and stemness of breast cancer via recruiting NuRD complex subunit LSD1
              </a>
            </td>
          <td>
            Yuanyin Xi, Ruoding Wang, Man Qu, Qinwen Pan, Minghao Wang, X. Ai, Zi-han Sun, Chao Zhang, Peng Tang, Jun Jiang, Ying Hu
          </td>
          <td>2025-03-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The interplay between genetics and cancer has been a focal point of research for decades, leading to profound understandings into the molecular mechanisms driving tumorigenesis. In this comprehensive review article, we explore the genetic basis of cancer, encompassing the diverse array of alterations that underline oncogenic transformation. From oncogenes to tumor suppressor genes, and from point mutations to chromosomal rearrangements, we delve into the molecular hallmarks of cancer and their implications for diagnosis, treatment, and prevention. Drawing on recent advancements in genomic technologies, we discuss the role of next-generation sequencing, single-cell sequencing, and computational modeling in unraveling the complexity of cancer genetics. Furthermore, we examine the clinical implications of genetic predisposition to cancer, highlighting the importance of genetic testing and counselling in cancer risk assessment and management. Through an exploration of tumor heterogeneity, clonal evolution, and therapeutic resistance, we underscore the challenges and opportunities in precision oncology. Finally, we discuss future directions in cancer genetics research, including precision prevention strategies and ethical considerations.Keywords: Cancer Genetics; Oncogenes; Tumor Suppressor Genes; Genetic Alterations; Precision Oncology; Clonal Evolution   ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b6286c0dc712fbb40ada9cedccccb4c8abdd42a" target='_blank'>
              Understanding the Role of Genetics in Tumour and Cancer Biology
              </a>
            </td>
          <td>
            A. Asiri, Amer Al Ali, Mohammed H. Abu-Alghayth
          </td>
          <td>2025-01-29</td>
          <td>Advancements in Life Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Super-enhancers (SEs) are expansive cis-regulatory elements known for amplifying oncogene expression across various cancers. However, their role in cervical cancer (CC), a remarkable global malignancy affecting women, remains underexplored. Here we applied integrated epigenomic and transcriptomic profiling to delineate the distinct SE landscape in CC by analyzing paired tumor and normal tissues. Our study identifies a tumor-specific SE at the EFNA1 locus that drives EFNA1 expression in CC. Mechanically, the EFNA1 SE region contains consensus sequences for the transcription factor FOSL2, whose knockdown markedly suppressed luciferase activity and diminished H3K27ac enrichment within the SE region. Functional analyses further underlined EFNA1's oncogenic role in CC, linking its overexpression to poor patient outcomes. EFNA1 knockdown strikingly reduced CC cell proliferation, migration, and tumor growth. Moreover, EFNA1 cis-interacted with its receptor EphA2, leading to decreased EphA2 tyrosine phosphorylation and subsequent activation of the Src/AKT/STAT3 forward signaling pathway. Inhibition of this pathway with specific inhibitors substantially attenuated the tumorigenic capacity of EFNA1-overexpressing CC cells in both in vitro and in vivo models. Collectively, our study unveils the critical role of SEs in promoting tumor progression through the FOSL2-EFNA1-EphA2-Src/AKT/STAT3 axis, providing new prognostic and therapeutic avenues for CC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3e483c9466e24512f997df84699534d6031c04" target='_blank'>
              Super-enhancer-driven EFNA1 fuels tumor progression in cervical cancer via the FOSL2-Src/AKT/STAT3 axis.
              </a>
            </td>
          <td>
            Shu-Qiang Liu, Xi-Xi Cheng, Shuai He, Tao Xia, Yi-Qi Li, Wan Peng, Ya-Qing Zhou, Zi-Hao Xu, Mi-Si He, Yang Liu, P.-P. Wei, Song-Hua Yuan, Chang Liu, Shu-Lan Sun, Dong-Ling Zou, Min Zheng, Chun-Yan Lan, Chun-Ling Luo, Jin-Xin Bei
          </td>
          <td>2025-02-18</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Human colorectal cancer (CRC) most often arises as a result of spontaneous genetic driver mutations such as APC, BRAF, KRAS, SMAD4, and TP53. These are among the most common drivers of CRC and are thought to contribute directly to cancer initiation and growth. CRC tumors harboring BRAFV600E mutation exhibit a CpG Island Methylator Phenotype (CIMP). Other drivers are also more weakly associated with distinct DNA methylation profiles. It is not clear whether these DNA methylation profiles are a result of differences in the cells-of-origin for these cancers, with pre-existing methylation patterns or whether there is selective pressure for various DNA methylation alterations that cooperate with each driver. To address this question, we have modeled this in mice by inducing colon tumors in a controlled cell population at the same age, with either BRAFF-V600E, KRASLSLG12D, Smad4, or Trp53 drivers in combination with Apc heterozygosity. We have profiled DNA methylation in tumors arising in these mice to determine the interaction between different driver mutations and the tumor methylome. Using a multiple linear regression model, we have identified DNA methylation alterations that are shared across all tumor samples, as well as changes that are specific to each of the different genetic drivers. The DNA methylation profiles accurately recapitulate well-established findings, including high levels of hypermethylation in KRASLSLG12D and BRAF F-V600E tumors reminiscent of CpG Island Methylator Phenotype (CIMP), universal tumor hypermethylation at binding sites for transcription factors involved in intestinal epithelial differentiation, and hypomethylation at AP1 complex component binding sites. Driver-specific histology findings broadly align with those observed in human cancers, such as sessile serrated morphology in BRAFF-V600E mutated tumors. The tumor models with different driver mutations also show variation in hypermethylation of polycomb repressive complex target sites and replicative history. We are using the differences in the DNA methylation profiles between the driver tumor models in conjunction with RNA-seq analysis and conservation of the alterations in human CRC to identify epigenetic events cooperative with the drivers and functionally relevant to tumorigenesis.
 Citation Format: Manpreet Kalkat, Toshinori Hinoue, Liang Kang, Peter W. Laird, Mary Olesnavich. Inducible mouse models of cancer driver mutations as a powerful tool for in vivo and epigenetic studies of colorectal cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c32f4b048765e136fdf9884436d3047629c3c3d9" target='_blank'>
              Abstract B025: Inducible mouse models of cancer driver mutations as a powerful tool for in vivo and epigenetic studies of colorectal cancer
              </a>
            </td>
          <td>
            Manpreet Kalkat, T. Hinoue, Liang Kang, Peter W. Laird, Mary Olesnavich
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The interaction between RAD51 and BRCA2 plays a key role in homologous recombination (HR), a critical DNA repair mechanism essential for the survival of cancer cells. Disrupting this interaction increases the sensitivity of cancer cells to chemotherapeutic agents. Here, we employed in silico methods to design a novel class of aptamers—customized single-stranded oligonucleotides—specifically engineered to bind RAD51. These aptamers were developed with the aim of selectively modulating RAD51’s nuclear recruitment and its role in DNA repair processes. The leading candidate displays high affinity for RAD51, competing with BRCA2 for the same interaction site in vitro, as confirmed through biolayer interferometry (BLI) and fluorescence lifetime imaging microscopy (FLIM). We tested the efficacy of the leading aptamer in pancreatic cancer cells and observed that it significantly impedes RAD51 nuclear localization, reduces homologous recombination (HR) efficiency, and increases DNA damage. Critically, our aptamer potentiates the cytotoxicity of the PARP inhibitor olaparib, exploiting synthetic lethality (SL) to induce cancer cell death. Our study showcases an aptamer-based approach for selectively targeting protein interactions within DNA repair pathways, introducing a promising avenue for SL-based treatments applicable to a wide range of cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9dc2579e152970b51a457421873665b384bcc5d3" target='_blank'>
              Computationally-designed aptamers targeting RAD51-BRCA2 interaction inhibit RAD51 nuclear recruitment
              </a>
            </td>
          <td>
            Giulia Milordini, E. Zacco, Alexandros Armaos, Francesco Di Palma, Mirco Masi, Martina Gilodi, Jakob Rupert, Laura Broglia, Giulia Varignani, Michele Oneto, Marco Scotto, Roberto Marotta, S. Girotto, Andrea Cavalli, Gian Gaetano Tartaglia
          </td>
          <td>2025-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer initiation and progression are associated with numerous somatic mutations, genomic rearrangements, and structure variants. The transformation of a normal cell into a cancer cell involves spatio-temporal changes in the regulation of different gene networks. The accessibility of these genes within the cell nucleus is manipulated via nucleosome remodeling ATPases, comprising one of the important mechanisms. Here, we reviewed studies of an ATP-dependent chromatin remodulator, chromodomain helicase DNA-binding 4 (CHD4), in cancer. Multiple domains of CHD4 are known to take part in nucleosome mobilization and histone binding. By binding with other proteins, CHD4 plays a vital role in transcriptional reprogramming and functions as a key component of Nucleosome Remodeling and Deacetylase, or NuRD, complexes. Here, we revisit data that demonstrate the role of CHD4 in cancer progression, tumor cell proliferation, DNA damage responses, and immune modulation. Conclusively, CHD4-mediated chromatin accessibility is essential for transcriptional reprogramming, which in turn is associated with tumor cell proliferation and cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a438ec9f6a0d935eb8720173993e895011ee0d" target='_blank'>
              Chromatin Remodulator CHD4: A Potential Target for Cancer Interception
              </a>
            </td>
          <td>
            Krishnendu Goswami, K. Venkatachalam, Surya P. Singh, C. Rao, Venkateshwar Madka
          </td>
          <td>2025-02-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9ab65a9ee1a176544b4d48a367f9b18440b017c" target='_blank'>
              R-loops acted on by RNase H1 influence DNA replication timing and genome stability in Leishmania
              </a>
            </td>
          <td>
            J. Damasceno, E. Briggs, M. Krasiļņikova, Catarina A Marques, C. Lapsley, R. McCulloch
          </td>
          <td>2025-02-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Transposable elements (TEs) comprise the majority of the human genome. In most somatic tissues, TEs are silenced by DNA methylation and histone modifications. Tumors exhibit changes in DNA methylation including global loss of methylation at regions that are silenced for genome stability, like TEs, and gain of methylation at the promoter regions of tumor suppressor genes. We showed that low doses of DNMT inhibitors (DNMTis) activate type I interferon signaling by reducing methylation and increasing expression of TEs that activate dsRNA sensors. DNMTis increase TEs in murine models of cancer, activating interferon signaling and recruiting CD8+ T cells to kill cancer cells and sensitize to immune checkpoint blockade therapy. We performed a genome-wide analysis of the response to DNMTi in a panel of ovarian cancer cell lines. Alu elements, which directly bind to the dsRNA sensor MDA5, are the most significantly demethylated TEs after DNMTi treatment. Our RNA-sequencing analyses indicated that A-to-I RNA editing of TEs by the RNA editing enzyme ADAR1 was increased after DNMTi treatment. This A-to-I editing makes TE RNA less immunogenic and inhibits the immune response to DNMTi. We find that combining ADAR1 knockdown and DNMTi treatment significantly increases type I interferon signaling and cancer cell secretion of downstream cytokines, including CCL5 and CXCL10. Further, this combination increased recruitment and activation of host immune cells and significantly improved survival in the aggressive ID8 Trp53-/- murine model of ovarian cancer. CITE-Seq (cellular indexing of transcriptomes and epitopes, a type of single-cell RNA sequencing that utilizes antibody tagging of each cell type before sequencing) demonstrated significantly higher infiltration of T and B cells in the tumor microenvironment. To understand the immune cell types causing the DNMTi anti-tumor response, we used antibodies to deplete CD8 T cells and NK cells in the ID8 Trp53-/- model with ADAR1 knockdown and DNMTi treatment. In addition, we performed these experiments in the C57BL/6-Tg(IghelMD4)4Ccg/J mice, which only produce antibodies against “Hel” antigen and have no other B cell responses. Of these lymphocyte populations, depleting T cells had the biggest effect and we observed specific killing of tumor cells by T cells isolated from the mice. We performed immunopeptidomics on cancer cell lines with and without DNMTi and showed that DNMTi upregulate TE peptide presentation by MHC molecules on the surface of tumor cells. Using a novel immuno-engineering approach, we are able to generate antigen-specific T cells targeting TEs and other antigens in murine and human cancer cell line models. These T cells specifically recognize (produce IFNgamma and TNFalpha) and kill the target tumor cells. As DNMTi have also been shown to demethylate genes crucial for T cell function, including IFNgamma and TCF7, we are currently combining these T cells with DNMTis for a novel cell therapy against cancer.
 Citation Format: Katherine B Chiappinelli. Epigenetic regulation of transposable elements to reverse cancer immune evasion [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr PR008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d21853f30868268b4b8dd99d7c90c4354a570de" target='_blank'>
              Abstract PR008: Epigenetic regulation of transposable elements to reverse cancer immune evasion
              </a>
            </td>
          <td>
            Katherine B. Chiappinelli
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3389d87da3fd17d24b405d1383fde338ac5299e" target='_blank'>
              MYC in cancer: from undruggable target to clinical trials.
              </a>
            </td>
          <td>
            Jonathan R Whitfield, Laura Soucek
          </td>
          <td>2025-02-19</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Primary cilia have been considered tumor-suppressing organelles in cholangiocarcinoma (CCA), though the mechanisms behind their protective role are not fully understood. This study investigates how the loss of primary cilia affects DNA damage response (DDR) and DNA repair processes in CCA. Human cholangiocyte cell lines were used to examine the colocalization of DNA repair proteins at the cilia and assess the impact of experimental deciliation on DNA repair pathways. Deciliation was induced using shRNA knockdown or CRISPR knockout of IFT20, IFT88, or KIF3A, followed by exposure to the genotoxic agents cisplatin, methyl methanesulfonate (MMS), or irradiation. Cell survival, cell cycle progression, and apoptosis rates were evaluated, and DNA damage was assessed using comet assays and γH2AX quantification. An in vivo liver-specific IFT88 knockout model was generated using Cre/Lox recombination. Results showed that RAD51 localized at the cilia base, while ATR, PARP1, CHK1 and CHK2 were found within the cilia. Deciliated cells displayed dysregulation in critical DNA repair. These cells also showed reduced survival and increased S-phase arrest after genotoxic challenges as compared to ciliated cells. Enhanced DNA damage was observed via increased γH2AX signals and comet assay results. An increase in γH2AX expression was also observed in our in vivo model, indicating elevated DNA damage. Additionally, key DDR proteins, such as ATM, p53, and p21, were downregulated in deciliated cells after irradiation. This study underscores the crucial role of primary cilia in regulating DNA repair and suggests that targeting cilia-related mechanisms could present a novel therapeutic approach for CCA. New and Noteworthy: Our findings reveal a novel connection between primary cilia and DNA repair in cholangiocytes. We showed that DDR and DNA repair proteins localize to cilia, and that deciliation leads to impaired cell survival and S-phase arrest under genotoxic stress. Deciliated cells exhibit heightened DNA damage, evidenced by increased γH2AX signals and comet assay results, a phenotype mirrored in in vivo IFT88 knockout mice. Furthermore, key DDR regulators, including ATM, p53, and p21, are downregulated in deciliated cells following irradiation, highlighting a crucial role for primary cilia in maintaining genome stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad4692fdd18b494d4b749d2c1b7ef3cd9972515b" target='_blank'>
              Cholangiocytes’ Primary Cilia Regulate DNA Damage Response and Repair
              </a>
            </td>
          <td>
            E. Peixoto, Kishor Pant, Seth Richard, J. Abrahante, Wioletta Czaja, S. Gradilone
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="All organisms depend on specific proteins to compact and organize their genomes. In eukaryotes, histones fulfil this role, while bacterial chromosomes are shaped by nucleoid-associated proteins (NAPs). Among its pleiotropic functions, the NAP Hfq plays a pivotal role in bacterial genome organization. In this study, we characterized the structure of the C-terminal extension of Hfq, which mediates chromosomal compaction, in its DNA-bound state. Using an integrative approach that combined transmission electron microscopy, neutron scattering, site-directed mutagenesis, and molecular modeling, we identified an amyloid module formed by the C-terminal region of Hfq. This module uniquely bridges and compacts six DNA molecules, marking the first documented instance of an amyloid structure with DNA-bridging properties. Our findings redefine the functional landscape of amyloids, linking them to genome architecture and gene regulation. This result suggests that amyloid-DNA interactions may represent a conserved mechanism across biological systems, with profound implications for understanding genome organization and the regulation of gene expression in both prokaryotes and eukaryotes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c1a71c24ec17a7b470740db10c59903a0c75c1" target='_blank'>
              Amyloid-like DNA bridging: a new mode of DNA shaping.
              </a>
            </td>
          <td>
            Frank Wien, Marcos Gragera, Tatsuhito Matsuo, Gautier Moroy, M. Bueno-Carrasco, Rocío Arranz, A. Cossa, Anne Martel, Heloisa N Bordallo, S. Rudić, Marisela Vélez, J. V. D. van der Maarel, J. Peters, V. Arluison
          </td>
          <td>2025-02-27</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Whole-genome duplications (WGD) and transposable element (TE) activity can act synergistically in genome evolution. WGDs can increase TE activity directly through cellular stress or indirectly by relaxing selection against TE insertions in functionally redundant, duplicated regions. Because TEs can function as, or evolve into, TE-derived cis-regulatory elements (TE-CREs), bursts of TE activity following WGD are therefore likely to impact evolution of gene regulation. Yet, the role of TEs in genome regulatory evolution after WGDs is not well understood. Here we used Atlantic salmon as a model system to explore how TE activity after the salmonid WGD ~100MYA shaped CRE evolution. We identified 55,080 putative TE-CREs using chromatin accessibility data from liver and brain. Retroelements were both the dominant source of TE-CREs and had higher regulatory activity in MPRA experiments compared to DNA elements. A minority of TE-subfamilies (16%) accounted for 46% of TE-CREs, but these "CRE-superspreaders" were mostly active prior to the WGD. Analysis of individual TE insertions, however, revealed enrichment of TE-CREs originating from WGD-associated TE activity, particularly for the DTT (Tc1-Mariner) DNA elements. Furthermore, coexpression analyses supported the presence of TE-driven gene regulatory network evolution, including DTT elements active at the time of WGD. In conclusion, our study supports a scenario where TE activity has been important in genome regulatory evolution, either through relaxed selective constraints, or strong selection to recalibrate optimal gene expression phenotypes, during a transient period following genome doubling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08496aae0238cce493a4d3baa60b2ce62c4ee152" target='_blank'>
              The role of transposon activity in shaping cis-regulatory element evolution after whole genome duplication.
              </a>
            </td>
          <td>
            Oystein Monsen, L. Grønvold, Alex Datsomor, T. Harvey, James Kijas, Alexander Suh, T. R. Hvidsten, S. Sandve
          </td>
          <td>2025-02-12</td>
          <td>Genome research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Background: Plasmablastic multiple myeloma (PMM), a subtype of multiple myeloma (MM), is characterized by immature plasma cells in the bone marrow. PMM is considered a high-risk disease due to its aggressive nature and reduced survival rates. The dramatic epigenetic remodeling that occurs in cancer genomes is associated with loss of silencing at transposable elements. Several consequences of this loss of silencing at TEs have been documented, from TEs being co-opted by the cancer genome for gene regulation, a process known as ‘onco-exaptation’, to genome instability mediated by transposition of LINE-1 (L1) elements. Among the mechanisms involved in the epigenetic repression of TEs are Krüppel-associated box (KRAB) domain zinc finger proteins (KZFPs) that target TEs through sequence specific binding and recruit repressor complexes to epigenetically suppress repeat elements. Methods: Formalin-fixed paraffin-embedded samples from PMM patients were characterized using stranded RNA sequencing for transcriptome profiling and whole-genome bisulfite sequencing for methylome analysis. Publicly available RNA-seq data from normal bone marrow plasma cells (BMPCs) and MM patients were obtained from published studies and the Multiple Myeloma Research Foundation (MMRF). Pipelines for TE transcript detection and methylation status were optimized to ensure accurate alignment within highly repetitive regions. Results: Loss of DNA methylation in PMM samples, as compared to normal BMPCs, results in the formation of partially methylated domains that are variable across patient samples. This loss of DNA methylation coincides with the expression of several hundred transcripts driven by LINE-1 (L1) retrotransposons that are epigenetically silenced in normal cells. Evolutionarily young L1 subfamilies, such as L1PA2 and L1PA3, preferentially lose methylation and are transcribed. While DNA methylation at CpG islands overlapping L1 elements is reduced in PMM samples compared to resting B cells, DNA methylation entropy is increased at these CpG islands within L1 elements in PMM samples. Transcript levels of specific KZFPs, including KZFPs that target L1PA2 and L1PA3 elements, are inversely correlated with the activity of L1 subfamilies. A similar degree of L1 activation is present in myeloma samples from the MMRF, where L1 activity correlates with poor survival outcomes. Samples with elevated L1 activity have a distinct gene expression signature related to a more proliferative, less differentiated state and inhibition of p53 and interferon pathways. Conclusion: The heterogeneous loss of DNA methylation at LINE1 regions in PMM samples contributes to heterogenous transposon activity with the potential to alter genome regulation. Patients with high L1 activity display unique gene expression signatures, potentially mediating the tolerance of high TE activity. Targeting LINE1 regulation may represent a novel strategy to slow PMM progression. Further studies are needed to assess how LINE1 activation affects genome regulation.
 Citation Format: Qianhui Wan, Amy Leung, Mahek Vinod Bhandari, Hiroyuki Kato, Joo Song, Dustin Schones. Methylation variability and LINE-1 activation in plasmablastic multiple myleoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7556721a82c0422b8d26aaeaf4a37ba842556557" target='_blank'>
              Abstract B006: Methylation variability and LINE-1 activation in plasmablastic multiple myleoma
              </a>
            </td>
          <td>
            Qianhui Wan, Amy Leung, Mahek Vinod Bhandari, Hiroyuki Kato, Joo Song, Dustin Schones
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background UCK2 (Uridine-Cytidine Kinase 2) is a promising prognostic marker for malignant tumors, but its association with immune infiltration and cancer stemness in pan-cancer remains to be fully understood. we find that gene UCK2 is closed related to RNA stemness scores (RNAss) and DNA stemness scores (DNAss), which is measured the tumor stemness. We also discover an association between UCK2 expression and immune cells by CIBERSORT algorithm, ESTIMATE algorithm and ssGSEA algorithm, especially, related to T cell, monocytes, mast cells, and macrophages. This study aims to shed light on the role and possible mechanism of UCK2 in pan-cancer. Methods We used the R programming language for pan-cancer bulk sequencing data analysis, which were obtained from the University of California, Santa Cruz (UCSC) datasets. UCSC database is a very useful for explore data from TCGA and other cancer genomics datasets, The data we explored at the UCK2 transcriptome level came from TCGA data in the UCSC database. We explored differential UCK2 expression between tumor and normal samples. Immunohistochemistry (IHC) was utilized to validate the expression of UCK2 in different types cancers using tumor tissue chips. The correlations of UCK2 with prognosis, genetic instability, DNA repair, cancer stem cell characteristics, and immune cell infiltration were investigated. Furthermore, single-cell datasets, acquired from the Gene Expression Omnibus (GEO) database, were used to validate the relationship between UCK2 and immune cells. GEO is a famous public genomics database supporting freely disseminates microarray data. Finally, we analyzed the correlation between UCK2 and drug sensitivity. Results UCK2 expression was observed to be high in most cancers and was remarkably related to the prognosis of pan-cancers. We found that the increased UCK2 expression was associated with higher genetic instability. Additionally, positive relationships were observed between UCK2 expression and mismatch repair genes, homologous recombination repair genes, and cancer stemness across different cancer types. There were significant correlations between UCK2 and T cells, monocytes, mast cells, and macrophages. Moreover, as expected, the immune checkpoint human leucocyte antigen (HLA) was found to be negatively related to UCK2. Similarly, UCK2 was also observed to have a negative association with major histocompatibility complex (MHC) genes. We noted that UCK2 had significant correlations with the sensitivity to various anti-cancer drug. Conclusion We have observed that UCK2 plays pivotal roles in prognosis and tumor immunity, and it is associated with DNA repair and cancer stemness. The UCK2 gene exhibits a strong correlation with the immune checkpoints HLA. This study highlights its potential impact on drug sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f3d615de6acd1c023835e11b2fb04b8e2aecd39" target='_blank'>
              Uridine-cytidine kinase 2 is correlated with immune, DNA damage repair and promotion of cancer stemness in pan-cancer
              </a>
            </td>
          <td>
            Jinlong Tian, Yanlei Li, Yu Tong, Yuan Zhang, Tingxiao Zhao, Yao Kang, Qing Bi
          </td>
          <td>2025-01-27</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Motivation Understanding the factors involved in DNA double-strand break (DSB) repair is crucial for the development of targeted anti-cancer therapies, yet the roles of many genes remain unclear. Recent studies show that perturbations of certain genes can alter the distribution of sequence-specific mutations left behind after DSB repair. This suggests that genome-wide screening could reveal novel DSB repair factors by identifying genes whose perturbation causes the mutational distribution spectra observed at a given DSB site to deviate significantly from the wild-type. However, designing proper controls for a genome-wide perturbation screen could be challenging. We explore the idea that a genome-wide screen might allow us to forgo the use of traditional non-targeting controls by reframing the analysis as an outlier detection problem, assuming that most genes have minimal influence on DSB repair. Results We propose MUSICiAn (Mutational Signature Catalogue Analysis), a compositional data analysis method that ranks gene perturbation-specific mutational spectra without controls by measuring deviations from the central tendency in the distributions of all spectra. We show that MUSICiAn can effectively estimate pseudo-controls for the existing Repair-seq dataset, screening 476 genes and 60 non-targeting controls. We further apply MUSICiAn to a genome-wide dataset profiling mutational outcomes induced by CRISPR-Cas9 at three target sites across cells with individual perturbations of 18,406 genes. MUSICiAn successfully recovers known genes, highlights the spliceosome as a lesser-appreciated player in DSB repair, and reveals candidates for further investigation. Availability github.com/joanagoncalveslab/MUSICiAn.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d86446b59ac2517d127c22b884f9cd2e873cbddf" target='_blank'>
              MUSICiAn: Genome-wide Identification of Genes Involved in DNA Repair via Control-Free Mutational Spectra Analysis
              </a>
            </td>
          <td>
            Colm Seale, Marco Barazas, R. van Schendel, M. Tijsterman, Joana P. Gonçalves
          </td>
          <td>2025-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d06e293f47afb8c1e77d741db124642957b624" target='_blank'>
              Characterization of RNF144B and PPP2R2A identified by a novel approach using TCGA data in ovarian cancer
              </a>
            </td>
          <td>
            P. Manasa, S. Krishnapriya, C. Sidhanth, S. Vasudevan, Kanchan Murhekar, T. S. Ganesan
          </td>
          <td>2025-02-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The human fungal pathobiont Candida albicans displays extensive genomic plasticity, including large-scale chromosomal changes such as aneuploidy. Chromosome trisomy appears frequently in natural and laboratory strains of C. albicans. Trisomy of specific chromosomes has been linked to large phenotypic effects, such as increased murine gut colonization by strains trisomic for chromosome 7 (Chr7). However, studies of whole-chromosome aneuploidy are generally limited to the SC5314 genome reference strain, making it unclear whether the imparted phenotypes are conserved across C. albicans genetic backgrounds. Here, we report the presence of a Chr7 trisomy in the “commensal-like” oral candidiasis strain, 529L, and dissect the contribution of Chr7 trisomy to colonization and virulence in 529L and SC5314. These experiments show that strain background and homolog identity (i.e., AAB vs ABB) interact with Chr7 trisomy to alter commensal and virulence phenotypes in multiple host niches. In vitro filamentation was the only phenotype altered by Chr7 trisomy in similar ways across the two strain backgrounds. Oral colonization of mice was increased by the presence of a Chr7 trisomy in 529L but not SC5314; conversely, virulence during systemic infection was reduced by Chr7 trisomy in SC5314 but not 529L. Strikingly, the AAB Chr7 trisomy in the SC5314 background rendered this strain avirulent in murine systemic infection. Increased dosage of NRG1 failed to reproduce most of the Chr7 trisomy phenotypes. Our results demonstrate that aneuploidy interacts with background genetic variation to produce complex phenotypic patterns that deviate from our current understanding in the genome reference strain. AUTHOR SUMMARY Candida albicans is a clinically important fungus of humans that is also part of the microbiota that typically colonizes our bodies. Changes in the number of copies of chromosome 7 (Chr7) in the genome of the reference strain of C. albicans enhances host colonization. We identified a third copy of Chr7 in a strain classified as a good colonizer of humans that doesn’t cause disease, mimicking the increased colonization of the reference strain. Here, we demonstrate that the number of copies of Chr7 alters phenotypes differently between these strains, and three copies of Chr7 does not universally produce a strain that is only a better commensal strain. Additionally, which copy of Chr7 in this diploid species exists as the third copy influences commensal and pathogen phenotypes. Therefore, trisomic changes of the same chromosome in different C. albicans strains have substantially different effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2332e9447e18d3247be54ad40b31742ed672d8fa" target='_blank'>
              Strain background interacts with chromosome 7 aneuploidy to determine commensal and virulence phenotypes in Candida albicans
              </a>
            </td>
          <td>
            Abhishek Mishra, N. Solis, Siobhan M Dietz, Audra L Crouch, S. Filler, Matthew Z. Anderson
          </td>
          <td>2025-01-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d71f4d600cde047c309be7d9f03d4993b2ab30" target='_blank'>
              EXO1's pan-cancer roles: diagnostic, prognostic, and immunological analyses through bioinformatics
              </a>
            </td>
          <td>
            Zheng Liu, Shu Huang, Rui Luo, Xiaomin Shi, Mingzhu Xiu, Yizhou Wang, Ruiyu Wang, Wei Zhang, Muhan Lv, Xiaowei Tang
          </td>
          <td>2025-03-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Human immune inflammatory response reflects the evolutionary adaptation of immune-cell regulatory elements1, where recent mutations can control both pathogen defence and susceptibility to chronic inflammatory and autoimmune diseases2–4. The impact of the deeper evolutionary history of these elements within primate genomes on human inflammatory responses remains poorly understood. Evolutionary young transposons have uniquely reshaped primate genomes5 and spread novel cis-regulatory sequences6. To understand how these events influenced human inflammation, we traced sequence changes in annotated human immune-cell enhancers back to macaque. We show that Alu elements and endogenous retroviruses dispersed motifs for the inflammation-related NF-κB and IRF1, redefining their binding patterns and contributing most prominently to great ape-specific binding sites. After the human-macaque split, many of these motifs shifted toward higher predicted binding affinity. In humans, population genetics analyses reveal that positive selection favors alleles, often Alu-derived, that increase enhancer affinity toward NF-κB. Enhancers containing Alu elements are more likely to undergo positive selection at the locus level, particularly when associated with chronic inflammatory diseases. As the most mutable enhancer sequences, Alus harbor disproportionately high numbers of single nucleotide polymorphisms and significantly contribute to selected alleles, some associated with chronic inflammatory diseases. We propose that the invasion of primate-specific transposons has created unique opportunities to adapt inflammatory responses in rapidly evolving great apes, with ancestral Alus continuing to influence evolutionary potential in humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d4cd878cd7db5b4df899d876eaba8e4013a874a" target='_blank'>
              Transposon invasion of primate genomes shaped human inflammatory enhancers and susceptibility to inflammatory diseases
              </a>
            </td>
          <td>
            Mengliang Ye, M. Rotival, Sebastian Amigorena, Elina Zueva
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Purpose: Pancreatic cancer (PC) is an aggressive lethal tumor with an unmet need for novel therapeutic approaches. KRAS activating mutations occur in 90-95% of PC and contribute to tumor progression and resistance to therapy, including radiation. The mechanisms by which oncogenic KRAS promotes radiation resistance are critical to understand in order to identify novel therapies. Methods: We first analyzed the expression levels of DNA damage response and repair genes using Affymetrix RNA expression microarray in isogenic HCT116 and SW48 cells with KRAS wide-type and KRASG13D activating mutations to identify novel targets by which KRAS mutations may confer radiation resistance. We analyzed the expression of STN1 in pancreas normal and cancer tissues and assessed the correlation with PC clinical outcomes using TCGA dataset. Human tumor xenografts were generated to explore the role of STN1 on tumor growth in vivo. Radiation response was assessed through clonogenicity and gH2AX foci assays. Homologous recombination (HR) and non-homologous end joining (NHEJ) repair reporter assays, chromatin spreading assay, cell cycle analysis, mitotic catastrophe, Annexin-V assays were performed to investigate the mechanisms of radiation-induced cell death. Mass spectrometry analysis was performed to identify STN1 interacting proteins important in DNA damage response and further validated by immunoprecipitation and immunoblotting. Results: We find that KRAS activation increases STN1 expression to enhance DNA double strand break repair capacity in PC. STN1 is a component of the CST complex normally important for telomere duplication and maintenance. We find that STN1 is significantly upregulated in PC, especially in aggressive subtypes of PC, associates with KRAS oncogenic mutations, and correlates with poor patient clinical outcomes. Genetic silencing or pharmacologic inhibition of KRAS signaling decreases STN1 expression in PC cells, suggesting KRAS signaling positively regulates STN1 expression. Interestingly, STN1 depletion reduces tumor growth in a heterotopic model of KRAS mutant PC. Mechanistically, depletion of STN1 potentiates DNA damage, replication stress, and sensitizes PC cells to ionizing radiation independent of CTC1 and TEN1. In support of this finding, STN1 silencing reduces both HR and NHEJ repair of DSBs. Furthermore, knockdown of STN1 impairs cell cycle arrest at the G2/M phase in response to ionizing radiation, which is accompanied with increased mitotic catastrophe, radiation-induced apoptosis. Proteomic analysis reveals that STN1 physically interacts with many proteins important for DNA repair, replication and cell cycle progression, including ATM, DICER1, CEP164, and CEP250. Conclusion: Our findings have revealed a novel, potentially CST complex-independent role of STN1 in DSB repair after radiation. STN1 may function at one of the apical nodes in the DNA damage response pathway by interacting with ATM. Our findings suggest STN1 may be a promising target for improving genotoxic therapies in KRAS mutant cancers, including PC.
 Citation Format: Tiantian Cui, Changxian Shen, Linlin Yang, Ling Gui, Sergio Corrales-Guerrero, Sindhu Nair, Joanna M. Karasinska, James T. Topham, Xiaoli Ping, Jeremy M. Stark, Terence M. Williams.STN1 (OBFC1) promotes DNA double-strand break repair in a potentially CTC1-STN1-TEN1 (CST) complex-independent role in pancreatic cancer.[abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr P011">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ce1e95ed7e839e47ffa944834be9b3c39c5d903" target='_blank'>
              Abstract P011: STN1 (OBFC1) promotes DNA double-strand break repair in a potentially CTC1-STN1-TEN1 (CST) complex-independent role in pancreatic cancer
              </a>
            </td>
          <td>
            Tiantian Cui, Changxian Shen, Linlin Yang, Ling Gui, S. Corrales-Guerrero, Sindhu Nair, J. Karasinska, J. Topham, Xiaoli Ping, Jeremy M. Stark, Terence M. Williams
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Chromosomal rearrangements involving the Mixed Lineage Leukemia gene (MLL1, KMT2A) are defining a genetically distinct subset in about 10% of human acute leukemias. Translocations involving the KMT2A-locus at chromosome 11q23 are resulting in the formation of a chimeric oncogene, where the N-terminal part of KMT2A is fused to a variety of translocation partners. The most frequently found fusion partners of KMT2A in acute leukemia are the C-terminal parts of AFF1, MLLT3, MLLT1 and MLLT10. Unfortunately, the presence of an KMT2A-rearrangements is associated with adverse outcomes in leukemia patients. Moreover, non-rearranged KMT2A-complexes have been demonstrated to be crucial for disease development and maintenance in NPM1-mutated and NUP98-rearranged leukemia, expanding the spectrum of genetic disease subtypes that are dependent on KMT2A. Recent advances in the development of targeted therapy strategies to disrupt the function of KMT2A-complexes in leukemia have led to the establishment of Menin-KMT2A interaction inhibitors that effectively eradicate leukemia in preclinical model systems and show favorable tolerability and significant efficacy in early-phase clinical trials. Indeed, one Menin inhibitor, Revumenib, was recently approved for the treatment of patients with relapsed or refractory KMT2A-rearranged acute leukemia. However, single agent therapy can lead to resistance. In this Review article we summarize our current understanding about the biology of pathogenic KMT2A-complex function in cancer, specifically leukemia, and give a systematic overview of lessons learned from recent clinical and preclinical studies using Menin inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e763a1f504a59703ce0db774f75dc6bbb915a958" target='_blank'>
              Targeting the Menin-KMT2A interaction in leukemia: Lessons learned and future directions.
              </a>
            </td>
          <td>
            Florian Perner, Jayant Y. Gadrey, Scott A. Armstrong, Michael W. M. Kühn
          </td>
          <td>2025-01-30</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Treatment decisions in metastatic castration-resistant prostate cancer (mCRPC) are mostly guided by clinical variables, but efforts to molecularly monitor the disease remain hampered by challenges in acquiring tumor tissue repeatedly. Here, we simultaneously profiled the genome copy number and exome in longitudinal plasma circulating tumor DNA (ctDNA) acquired before, during, and upon progression to serial treatments with androgen signaling inhibitors (ASI) and taxane chemotherapy from 60 mCRPC patients (2-10 samples per patient). The genomic data was used to delineate the clonal substructure and evolutionary dynamics of each patient, and an evolutionary dynamic index (EDI) was developed to measure the longitudinal changes of the tumor subclones. Treatment with ASI resulted in greater subclonal selection and population structure changes than taxane treatment. The subclones that emerged in association with serial therapy resistance harbored recurrent aberrations in previously identified and new candidate genes, with particular enrichment in genes related to PI3K-AKT signaling. These findings indicate that the integration of detailed clinical and genomics data can provide a framework for future unbiased genomic applications for ctDNA in the clinic to enable precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b66fc40e62ecd1d0cde61c3ba23696dc636af88" target='_blank'>
              Longitudinal Profiling of Circulating Tumor DNA Reveals the Evolutionary Dynamics of Metastatic Prostate Cancer During Serial Therapy.
              </a>
            </td>
          <td>
            Yuehui Zhao, Naveen Ramesh, Ping Xu, Emi Sei, Min Hu, S. Bai, Patricia Troncoso, Ana M Aparicio, Christopher J. Logothetis, Paul G Corn, Nicholas E Navin, A. Zurita
          </td>
          <td>2025-02-24</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The retinoic acid-related orphan receptor gamma (RORγt) acts as the major transcriptional activator in Th17 cell development and function to mediate adaptive immune defenses against pathogenic infection. RORγt engages accessible DNA response elements in the genome and interplays with coactivator proteins and accessory transcription factors to drive gene expression. However, how the chromatin environment mediates RORγt structure, dynamics, and function remains unclear. Here, we profile how the nucleosome promotes or restricts access to the main RORγt DNA response elements found in native enhancers and promoters, revealing preferential binding in regions of free DNA and nucleosomal entry/exit sites, with single base-pair resolution. Solution phase measurements using hydrogen deuterium exchange coupled to mass spectrometry identify novel allosteric effects that influence RORγt binding and mediate chromatin dynamics. A high-resolution structure of RORγt bound to the nucleosome reveals how structured elements assemble to confer binding specificity and avidity to chromatin substrates. The observations suggest an activation model where RORγt binding to chromatinized DNA promotes coregulator recruitment and chromatin decompaction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5624b8ad84bcb73e88ab4dda70b0e6af9169b6fa" target='_blank'>
              Nucleosome Engagement Regulates RORγt Structure and Dynamics
              </a>
            </td>
          <td>
            Timothy S. Strutzenberg, Matthew D. Mann, Xiandu Li, Hyejeong Shin, Jordan Kelsey, S. Aiyer, Jingting Yu, Gennavieve Gray, Zeyuan Zhang, Zelin Shan, Bo Zhou, Ye Zheng, Patrick R. Griffin, Dmitry Lyumkis
          </td>
          <td>2025-03-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3815b96ede148fbe1798a5178e21eca65c618ab6" target='_blank'>
              FOXM1, a super enhancer-associated gene, is related to poorer prognosis and gemcitabine resistance in pancreatic cancer.
              </a>
            </td>
          <td>
            Jian Jiang, Tianci Shen, Dan Chen, Zihao Dai, Xuelong Wang, Qiang Meng, Zhuo Yang, Di Zhang, Xiaoyi Guo, Jianqiang Xu, Jiangning Gu, Changmiao Wang
          </td>
          <td>2025-02-03</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mutations in somatic cells are inflicted by both extrinsic and intrinsic sources and contribute over time to cancer. Tobacco smoke contains chemical carcinogens that have been causatively implicated with cancers of the lung and head & neck1,2. APOBEC family DNA cytosine deaminases have emerged as endogenous sources of mutation in cancer, with hallmark mutational signatures (SBS2/SBS13) that often co-occur in tumors of tobacco smokers with an equally diagnostic mutational signature (SBS4)3,4. Here we challenge the dogma that mutational processes are thought to occur independently and with additive impact by showing that 4-nitroquinoline 1-oxide (NQO), a model carcinogen for tobacco exposure, sensitizes cells to APOBEC3B (A3B) mutagenesis and leads to synergistic increases in both SBS2 mutation loads and oral carcinomas in vivo. NQO-exposed/A3B-expressing animals exhibit twice as many head & neck lesions as carcinogen-exposed wildtype animals. This increase in carcinogenesis is accompanied by a synergistic increase in mutations from APOBEC signature SBS2, but not from NQO signature SBS4. Interestingly, a large proportion of A3B-catalyzed SBS2 mutations occurs as strand-coordinated pairs within 32 nucleotides of each other in transcribed regions, suggesting a mechanism in which removal of NQO-DNA adducts by nucleotide excision repair exposes short single-stranded DNA tracts to enzymatic deamination. These highly enriched pairs of APOBEC signature mutations are termed didyma (Greek for twins) and are mechanistically distinct from other types of clustered mutation (omikli and kataegis). Computational analyses of lung and head & neck tumor genomes show that both APOBEC mutagenesis and didyma are elevated in cancers from smokers compared to non-smokers. APOBEC signature mutations and didyma are also elevated in normal lung tissues in smokers prior to cancer initiation. Collectively, these results indicate that DNA adducting mutagens in tobacco smoke can amplify DNA damage and mutagenesis by endogenous APOBEC enzymes and, more broadly, suggest that mutational mechanisms can interact synergistically in both cancer initiation and promotion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de637cc3bf2890bf07d4603dfc1f19004aa5b7c2" target='_blank'>
              Tobacco smoke carcinogens exacerbate APOBEC mutagenesis and carcinogenesis
              </a>
            </td>
          <td>
            Cameron Durfee, E. Bergstrom, M. Díaz-Gay, Yufan Zhou, N. A. Temiz, Mahmoud A. Ibrahim, Shuvro P. Nandi, Yaxi Wang, Xingyu Liu, Christopher D. Steele, Joshua Proehl, Rachel I Vogel, P. Argyris, Ludmil B. Alexandrov, R. Harris
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Ovarian cancer (OC) is a highly heterogeneous malignancy, often characterized by complex genomic alterations that drive tumor progression and therapy resistance. In this paper, we report a novel de novo BRCA2 germline variant NM_000059.3:c.(8693_8695delinsGT) associated with early-onset OC that featured two regions with differential MMR (Mismatch Repair) gene expression. To date, only six cases of de novo BRCA2 variants have been reported, none of which were associated with early-onset high-grade serous OC. The immunohistochemical analysis of MMR genes revealed two distinct tumor areas, separated by a clear topographic boundary, with the heterogeneous expression of MLH1 and PMS2 proteins. Seventy-five percent of the tumor tissue showed positivity, while the remaining 25% exhibited a complete absence of expression, underscoring the spatial variability in MMR gene expression within the tumor. Integrated comparative spatial genomic profiling identified several tumor features associated with the genetic variant as regions of loss of heterozygosity (LOH) that involved BRCA2 and MLH1 genes, along with a significantly higher mutational tumor burden in the tumor area that lacked MLH1 and PMS2 expression, indicating its further molecular evolution. The following variants were acquired: c.6572C>T in NOTCH2, c.1852C>T in BCL6, c.191A>T in INHBA, c.749C>T in CUX1, c.898C>A in FANCG, and c.1712G>C in KDM6A. Integrated comparative spatial proteomic profiles revealed defects in the DNA repair pathways, as well as significant alterations in the extracellular matrix (ECM). The differential expression of proteins involved in DNA repair, particularly those associated with MMR and Base Excision Repair (BER), highlights the critical role of defective repair mechanisms in driving genomic instability. Furthermore, ECM components, such as collagen isoforms, Fibrillin-1, EMILIN-1, Prolargin, and Lumican, were found to be highly expressed in the MLH1/PMS2-deficient tumor area, suggesting a connection between DNA repair deficiencies, ECM remodeling, and tumor progression. Thus, the identification of the BRCA2 variant sheds light on the poorly understood interplay between DNA repair deficiencies and ECM remodeling in OC, providing new insights into their dual role in shaping tumor evolution and suggesting potential targets for novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b3d22f654834c47296af7f38cdf9042ad40753" target='_blank'>
              Novel De Novo BRCA2 Variant in an Early-Onset Ovarian Cancer Reveals a Unique Tumor Evolution Pathway
              </a>
            </td>
          <td>
            G. Miolo, Giovanni Canil, Maurizio Polano, M. Dal Bo, Alessia Mondello, Antonio Palumbo, F. Puglisi, Giuseppe Corona
          </td>
          <td>2025-03-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Prostate cancer is a prevalent tumor type that, despite being highly curable, progresses to metastatic disease in a fraction of patients, thus accounting for more than 350.000 annual deaths worldwide. In turn, uncovering the molecular insights of metastatic disease is instrumental to improve the survival rate of prostate cancer patients. By means of gene expression metanalysis in multiple prostate cancer patient cohorts, we identified a set of genes that are differentially expressed in aggressive prostate cancer. Transcription factor 19 (TCF19) stood out as an unprecedented epithelial gene upregulated in metastatic disease, with prognostic potential and associated with the activity of androgen receptor. By combining computational and empiric approaches, our data revealed that TCF19 is required for full metastatic capacity and its depletion influences core cancer-related processes, such as vascular permeability, supporting the role of this gene in the dissemination of prostate tumor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ecdf804c61fcb68669b4e697cde8e1d82f6075a" target='_blank'>
              Transcription factor 19 is an androgen responsive gene that modulates vessel homeostasis and sustains metastatic prostate cancer
              </a>
            </td>
          <td>
            Amaia Ercilla, Jana R. Crespo, Saioa Garcia-Longarte, Marta Fidalgo, Natalia Martín-Martín, Onintza Carlevaris, I. Astobiza, Sonia Fernández-Ruiz, M. Guiu, Laura Bárcena, Isabel Mendizabal, A. Aransay, M. Graupera, R. Gomis, Arkaitz Carracedo
          </td>
          <td>2025-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="ABSTRACT The human genome is like a museum of ancient retroviral infections. It contains a large number of endogenous retroviruses (ERVs) that bear witness to past integration events. About 5,000 of them are so-called long terminal repeat 12 (LTR12) elements. Compared with 20,000 human genes, this is a remarkable number. Although LTR12 elements can act as promoters or enhancers of cellular genes, the function of most of these retroviral elements has remained unclear. In our mini-review, we show that different LTR12 elements share many similarities, including common transcription factor binding sites. Furthermore, we summarize novel insights into the epigenetic mechanisms governing their silencing and activation. Specific examples of genes and pathways that are regulated by LTR12 loci are used to illustrate the regulatory network built by these repetitive elements. A particular focus is on their role in the regulation of antiviral immune responses, tumor cell proliferation, and senescence. Finally, we describe how a targeted activation of this fascinating ERV family could be used for diagnostic or therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4041a47b44612c0a5daa01efba716fa89ab732e8" target='_blank'>
              Viral influencers: deciphering the role of endogenous retroviral LTR12 repeats in cellular gene expression
              </a>
            </td>
          <td>
            Veronika Krchlikova, Yueshuang Lu, D. Sauter
          </td>
          <td>2025-01-31</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract G-quadruplex (G4) DNAs are alternative nucleic acid structures, proposed to play important roles in regulating DNA replication, gene transcription, and translation. Several specialized DNA helicases are involved in cellular G4 metabolism, in some cases with redundant functions. Among them, human FANCJ/BRIP1, which has orthologs in all metazoans, is one of the most powerful G4 resolvases, believed to act mainly at DNA replication forks. Here, we tested the effects of a set of hydrazone-derivative G4 ligands in a FANCJ-knocked-out HeLa cell line and in a Caenorhabditis elegans strain, where DOG-1, a FANCJ ortholog, was disrupted, as a whole organism model system. Our results revealed that loss of FANCJ specifically sensitized cancer cells to FIM-15, a mono-guanylhydrazone derivative bearing the diimidazopyrimidine core, among the tested hydrazone-based compounds and induced enhanced DNA damage in different chromosomal sites including telomeric ends. Moreover, dietary administration of FIM-15 to dog-1−/− nematodes stabilized G4 structures in gonadal cell nuclei and resulted in compromised embryonic development in the first-generation post-treatment. Collectively, our findings unveil a specific vulnerability of FANCJ-knocked-out cancer cells (and DOG-1-lacking worms) to G4 stabilization by the FIM-15 compound. This study provides an important proof-of-principle for use of G4 ligands in synthetic lethality-based therapeutic approaches targeting FANCJ-defective cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a485077681fc1cddbc3b2edd2dc8610a684c82b" target='_blank'>
              Effects of hydrazone-based G-quadruplex ligands on FANCJ/BRIP1-depleted cancer cells and a Caenorhabditis elegans dog-1−/− strain
              </a>
            </td>
          <td>
            Marcello Germoglio, Federica D'Aria, G. Cortone, Antonello Prodomo, M. Mahtab, Rita Morigi, Jussara Amato, Francesca M. Pisani, C. Giancola
          </td>
          <td>2025-02-08</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a deadly malignancy and new treatments are needed to improve survival rates. Many lung tumours exhibit a feature called chromosomal instability, which is characterized by ongoing acquisition of genetic alterations that promote tumour aggression. Interestingly, chromosomal instability is a “double-edged sword”. A moderate level can support tumour growth and progression but a high level causes cell death. Therefore, chromosomal instability represents a weakness in cancer cells that could be exploited to kill lung tumours. Kinesins are a family of proteins with several members having key roles in chromosome segregation during mitosis. Because of this function, inhibiting mitotic kinesins can amplify chromosomal instability. This review summarizes the potential of targeting kinesins to elevate chromosomal instability to lethal levels as a strategy for killing chromosomally unstable lung cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994a7c70e4769153ecb42042c5f04caa4a869a11" target='_blank'>
              Targeting Kinesins for Therapeutic Exploitation of Chromosomal Instability in Lung Cancer
              </a>
            </td>
          <td>
            Christopher Zhang, Benson Z. Wu, Kelsie L. Thu
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff1e1e85866d4cc9a2e1386acb32f48ee6d86bb6" target='_blank'>
              AZIN1 level is increased in medulloblastoma and correlates with c-Myc activity and tumor phenotype
              </a>
            </td>
          <td>
            Julie Sesen, Tyra Martinez, Sara Busatto, L. Poluben, Hassan Nassour, Caroline Stone, Karthik Ashok, Marsha A. Moses, Edward R. Smith, Aram Ghalali
          </td>
          <td>2025-02-17</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Transposable elements (TEs), once considered as "junk" DNA, are now recognized for their epigenetic and regulatory influence, especially in cancer. Cryptic regulatory elements within TEs can be epigenetically reactivated in cancer, contributing to oncogenesis and altering gene expression patterns. This reactivation results in TE-derived transcripts, which have the potential to be translated and form neoantigens that can be presented by the human leukocyte antigens (HLA) complex and recognized by T cells, ultimately impacting the tumor immune response. In this study, we analyzed RNA-seq data from melanoma patients undergoing PD1/PDL1 immunotherapy. By identifying tumor specific TE-derived transcripts and utilizing in silico translation, we detected TE-derived neoantigens in patients with differing responses to immunotherapy. Our findings indicate that the number of TE-derived neoantigens is correlated with patients' response to immunotherapy, with a higher number of neoantigens observed in patients who responded to treatment compared to those who did not. These results imply that TE-derived neoantigens may serve as potential biomarkers for predicting the prognosis of PD1/PDL1 immunotherapy treatment. Incorporating TE-derived neoantigen profiles into clinical decision-making could enhance patient selection for immunotherapy, thereby improving treatment outcomes.
 Citation Format: Yixin Wu, Ting Wang, Yonghao Liang. Epigenetic reactivation of transposable elements as prognostic markers for PD1/PDL1 immunotherapy in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B007.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c61593260331e8272d84d80f0c9cc19c5bdc07d" target='_blank'>
              Abstract B007: Epigenetic reactivation of transposable elements as prognostic markers for PD1/PDL1 immunotherapy in cancer
              </a>
            </td>
          <td>
            Yixin Wu, Ting Wang, Yonghao Liang
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="A large proportion of disease variants is found in non-coding RNAs (ncRNAs), gene loci that have been identified as key regulatory elements. However, for most ncRNAs, their targets are unknown, hindering our ability to understand complex diseases. Here, we found that allele-specific ncRNAs were enriched nearby allelic protein-coding genes (pcGenes), suggesting that the allele-specific information could be used to predict cis-acting ncRNA-targets. We translated this concept into the Allelome.LINK framework and applied it to the major mouse organs, revealing 397 events where the allele-specific expression (ASE) of a ncRNA correlated with the ASE of a nearby pcGene, suggesting either enhancing or repressive regulatory interactions. Next, we applied our strategy to the largest human dataset including tissues from nearly 1,000 individuals. Given the outbred nature of humans, each individual allows for the discovery of novel ASE correlation events. We uncovered 2,291 shared ASE events along with their mechanisms, which we benchmarked against sample-matched eQTLs, yielding a high validation rate of 77.47%. Further GWAS integration assigned 30.59% of variants overlapping informative ncRNAs to their pcGene targets. As more sequencing data and risk variants become available, this strategy has the potential to decode the entire cis-acting landscape of the non-coding genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8599e8314fc9026e2db49b39f76e5203a6d7080" target='_blank'>
              Allele-specific genomics decodes gene targets and mechanisms of the non-coding genome
              </a>
            </td>
          <td>
            Tim P Hasenbein, Sarah Hoelzl, S. Engelhardt, Daniel Andergassen
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Mitoregulin (MTLN) is a 56-amino-acid mitochondrial microprotein known to modulate mitochondrial energetics. MTLN gene expression is elevated broadly across most cancers and has been proposed as a prognostic biomarker for non-small cell lung cancer (NSCLC). In addition, lower MTLN expression in lung adenocarcinoma (LUAD) correlates with significantly improved patient survival. In our studies, we have found that MTLN silencing in A549 NSCLC cells slowed proliferation and, in accordance with this, we observed the following: (1) increased proportion of cells in the G1 phase of cell cycle; (2) protein changes consistent with G1 arrest (e.g., reduced levels and/or reduced phosphorylation of ERK, MYC, CDK2, and RB, and elevated p27Kip1); (3) reduction in clonogenic cell survival and; (4) lower steady-state cytosolic and mitochondrial H2O2 levels as indicated by use of the roGFP2-Orp1 redox sensor. Conflicting with G1 arrest, we observed a boost in cyclin D1 abundance. We also tested MTLN silencing in combination with buthionine sulfoximine (BSO) and auranofin (AF), drugs that inhibit GSH synthesis and thioredoxin reductase, respectively, to elevate the reactive oxygen species (ROS) amount to a toxic range. Interestingly, clonogenic survival after drug treatment was greater for MTLN-silenced cultures versus the control cultures. Lower H2O2 output and reduced vulnerability to ROS damage due to G1 status may have jointly contributed to the partial BSO + AF resistance. Overall, our results provide evidence that MTLN fosters H2O2 signaling to propel G1/S transition and suggest MTLN silencing as a therapeutic strategy to limit NSCLC growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/728830e41932164b648250623dd9f7dc576066d5" target='_blank'>
              Mitoregulin Promotes Cell Cycle Progression in Non-Small Cell Lung Cancer Cells
              </a>
            </td>
          <td>
            Colleen S. Stein, Connor R. Linzer, Collin D. Heer, Nathan H. Witmer, Jesse D. Cochran, D. R. Spitz, Ryan L. Boudreau
          </td>
          <td>2025-02-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Intratumoral heterogeneity and clonal evolution are pivotal in the progression and metastasis of melanoma. However, when combined with variable tumor cellularity, intratumoral heterogeneity limits the sensitivity and accuracy of uncovering a cancer’s clonal evolution. In this study, we combined fluorescence-activated cell sorting (FACS) with whole-exome sequencing (WES) to investigate the clonal composition and evolutionary patterns in seven melanoma biopsies obtained from three patients, each having both primary site and metastatic samples. We employed a multiparameter ploidy sorting approach to isolate tumor populations based on DNA ploidy and melanoma biomarkers (SOX10 or S100), enabling us to investigate clonal evolution with high resolution. Our approach increased the mean tumor purity from 70% (range 19–88%) in unsorted material to 91% (range 87–96%) post-sorting. Our findings revealed significant inter- and intratumor heterogeneity, with one patient exhibiting two genomically distinct clonal tumor populations within a single primary site biopsy, each giving rise to different metastases. Our findings highlight the critical role of intratumoral heterogeneity and clonal evolution in melanoma, especially when analyzing tumor trajectories. The unique combination of multiparameter FACS and WES provides a powerful method for identifying clonal populations and reconstructing clonal evolution. This study provides valuable insights into the clonal architecture of melanoma and lays the groundwork for future research with larger patient groups.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d2f5ebd9cec46f8081721cfbefc573bc09f8fb" target='_blank'>
              Tracking the Evolution of Cutaneous Melanoma by Multiparameter Flow Sorting and Genomic Profiling
              </a>
            </td>
          <td>
            L. Roma, T. Lorber, Sabrina Rau, Michael T. Barrett, Caner Ercan, Federica Panebianco, Salvatore Piscuoglio, Katharina Glatz, L. Bubendorf, Christian Ruiz
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Chromatin remodeling proteins contribute to DNA replication, transcription, repair, and recombination. The chromodomain helicase DNA-binding (CHD) family of remodelers plays crucial roles in embryonic development, neurogenesis, and carcinogenesis. As the founding member, CHD1 is capable of assembling nucleosomes, remodeling chromatin structure, and regulating gene transcription. Dysregulation of CHD1 at genetic, epigenetic, and post-translational levels is common in prostate cancer and other malignancies. Through interacting with different genetic alterations, CHD1 possesses the capabilities to exert oncogenic or tumor-suppressive functions in context-dependent manners. In the past decade, my study uncovered the novel cell-intrinsic and -extrinsic roles of CHD1 during prostate cancer development, progression, and therapy resistance (Nature 2017, Cancer Discover 2020, Cancer Research 2022, Nature Cancer 2024). Cancer genome studies have uncovered a unique subtype of prostate cancer characterized by speckle-type BTB/POZ protein (SPOP) mutations, which stabilize AR and its co-activators and have been associated with prolonged responses to androgen deprivation therapies and favorable prognoses. Loss of CHD1 frequently co-occurs with SPOP mutations; however, how concurrent CHD1 loss impacts SPOP-mutated prostate cancer remains unclear. Most recently, by generating the first-in-the-field genetically engineered mouse models, we provided genetic evidence supporting that CHD1 loss accelerates SPOP-mutated prostate cancer progression and confers resistance to castration in this context. We uncover a non-canonical epigenetic function of CHD1 and demonstrate that CHD1 reprograms lipid metabolism by governing SREBPs. CHD1 loss–induced lipid reprogramming supplies intratumoral androgen biosynthesis and retains AR transcriptional activity in SPOP-mutated prostate tumors upon castration. Furthermore, the mechanistic studies facilitate the development of a novel biomarker-driven combinatorial therapy for castration-resistant prostate cancer harboring CHD1 loss and SPOP mutations. These findings advance our understanding of an emerging prostate cancer subtype and mechanism of action and provide novel therapeutic strategies for men with advanced prostate cancer. *This research article has been accepted by Nature Cancer for publication.
 Citation Format: Feiyu Chen, Di Zhao. Decipher actionable tumor suppressor defects for personalized cancer medicine [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr B022.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/255726480b1fba8b96c7a797db393035b828db59" target='_blank'>
              Abstract B022: Decipher actionable tumor suppressor defects for personalized cancer medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87b3f437af02a37eaf52895dfc3343570b14c40b" target='_blank'>
              The role of Box A of HMGB1 in producing γH2AX associated DNA breaks in lung cancer
              </a>
            </td>
          <td>
            Sirapat Settayanon, P. Chanvorachote, Apiwat Mutirangura
          </td>
          <td>2025-01-25</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="The link between the “stress phenotype”-a well-established hallmark of cancer-and its role in tumor progression and intratumor heterogeneity remains poorly defined. The integrated stress response (ISR) is a key adaptive pathway that enables tumor survival under oncogenic stress. While ISR has been implicated in promoting tumor growth, its precise role in driving tumor evolution and heterogeneity has not been elucidated. In this study, using a genetically engineered mouse models, we demonstrate that ISR activation—indicated by elevated levels of phosphorylated eIF2 (p-eIF2) and ATF4—is essential for the emergence of dedifferentiated, therapy-resistant cell states. ISR, through the coordinated actions of ATF4 and MYC, facilitates the development of tumor cell populations characterized by high plasticity, stemness, and an epithelial-mesenchymal transition (EMT)-prone phenotype. This process is driven by ISR-mediated expression of genes that maintain mitochondrial integrity and function, critical for sustaining tumor progression. Importantly, genetic, or pharmacological inhibition of the p-eIF2–ATF4 signaling axis leads to mitochondrial dysfunction and significantly impairs tumor growth in mouse models of lung adenocarcinoma (LUAD). Moreover, ISR-driven dedifferentiation is associated with poor prognosis and therapy resistance in advanced human LUAD, underscoring ISR inhibition as a promising therapeutic strategy to disrupt tumor evolution and counteract disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/984621d182e32177c7e7be128d21ab0e1df4a135" target='_blank'>
              Lineage plasticity of the integrated stress response is a hallmark of cancer evolution
              </a>
            </td>
          <td>
            Shiqi Diao, Jia Yi Zou, Shuo Wang, N. Ghaddar, Jason E. Chan, Hyungdong Kim, Nicolas Poulain, C. Koumenis, M. Hatzoglou, Peter Walter, Nahum Sonenberg, John Le Quesne, Tuomas Tammela, A. Koromilas
          </td>
          <td>2025-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Ku70 and Ku80 form Ku, a ring-shaped protein that initiates the non-homologous end-joining (NHEJ) DNA repair pathway.1 Specifically, Ku binds to double-stranded DNA (dsDNA) ends and recruits other NHEJ factors (e.g., DNA-PKcs and LIG4). While Ku binds to double-stranded RNA (dsRNA)2 and traps mutated-DNA-PKcs on ribosomal RNA in vivo,3,4 the physiological significance of Ku-dsRNA interactions in otherwise wild-type cells remains elusive. Intriguingly, while dispensable for murine development,5,6 Ku is essential in human cells.7 Despite similar genome sizes, human cells express ∼100-fold more Ku than mouse cells, implying functions beyond NHEJ, possibly through a dose-sensitive interaction with dsRNA, which is ∼100 times weaker than with dsDNA.2,8 While investigating the essentiality of Ku in human cells, we found that depletion of Ku - unlike LIG4 - induces profound interferon (IFN) and NF-kB responses reliant on the dsRNA-sensor MDA5/RIG-I and adaptor MAVS. Prolonged Ku-degradation also activates other dsRNA-sensors, e.g. PKR that suppresses protein translation, and OAS/RNaseL that cleaves rRNAs and eventually induces growth arrest and cell death. MAVS, RIG-I, or MDA5 knockouts suppressed IFN signaling and, together with PKR knockouts, partially rescued Ku-depleted human cells. Ku-irCLIP analyses revealed that Ku binds to diverse dsRNA, predominantly stem-loops in primate-specific Alu elements9 at anti-sense orientation in introns and 3’-UTRs. Ku expression rose sharply in higher primates tightly correlating with Alu-expansion (r = 0.94/0.95). Together, our study identified a vital role of Ku in accommodating Alu-expansion in primates by mitigating a dsRNA-induced innate immune response, explaining the rise of Ku levels and its essentiality in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24649fa8cee2fc610da3934361ee2a3da1b227a9" target='_blank'>
              Ku suppresses RNA-mediated innate immune responses in human cells to accommodate primate-specific Alu expansion
              </a>
            </td>
          <td>
            Yimeng Zhu, Angelina Li, Suvrajit Maji, Brian J. Lee, Sophie M. Korn, Jake A. Gertie, Tyler J. Dorrity, Jianhua Wang, Kyle J. Wang, Amandine Pelletier, Daniel F. Moakley, Rachel D. Kelly, Antony B. Holmes, Raul Rabadan, D. Edgell, Caroline Schild Poulter, Mauro Modesti, Anna-Lena Steckelberg, Eric A. Hendrickson, Hachung Chung, Chaolin Zhang, Shan Zha
          </td>
          <td>2025-02-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Breast cancers of the IntClust-2 type, characterized by amplification of a small portion of chromosome 11, have a median survival of only five years. Several cancer-relevant genes occupy this portion of chromosome 11, and it is thought that overexpression of a combination of driver genes in this region is responsible for the poor outcome of women in this group. In this study we used a gene editing method to knock out, one by one, each of 198 genes that are located within the amplified region of chromosome 11 and determined how much each of these genes contributed to the survival of breast cancer cells. In addition to well-known drivers such as CCND1 and PAK1, we identified two different genes (SERPINH1 and P4HA3), that encode proteins involved in collagen synthesis and organization. Using both in vitro and in vivo functional analyses, we determined that P4HA3 and/or SERPINH1 provide a critical driver function on IntClust-2 basic processes, such as viability, proliferation, and migration. Inhibiting these enzymes via genetic or pharmacologic means reduced collagen synthesis and impeded oncogenic signaling transduction in cell culture models, and a small-molecule inhibitor of P4HA3 was effective in treating 11q13 tumor growth in an animal model. As collagen has a well-known association with tissue stiffness and aggressive forms of breast cancer, we believe that the two genes we identified provide an opportunity for a new therapeutic strategy in IntClust-2 breast cancers. Implications: Breast cancers with 11q13/14 chromosomal amplifications may be vulnerable to inhibitors of collagen synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd01b0833b0dda4d17087283c1b41d82a5f1780" target='_blank'>
              Identifying and targeting key driver genes for collagen production within the 11q13/14 breast cancer amplicon.
              </a>
            </td>
          <td>
            D. Araiza-Olivera, T. Prudnikova, Cristina Uribe-Alvarez, Kathy Q Cai, J. Franco-Barraza, Jesus M Dones, Ronald T. Raines, J. Chernoff
          </td>
          <td>2025-01-21</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Genomic imprinting is an epigenetic mechanism that drives monoallelic gene expression depending on parental origin. Loss of imprinting (LOI) is associated with human imprinting disorders, fetal development, and cancer progression. Imprinted genes, organized in clusters, are regulated by methylation at imprint control regions (ICRs), differentially methylated regions (DMRs) between parental chromosomes. Somatic cell reprogramming into induced pluripotent stem cells (iPSCs) is a valuable tool for studying pluripotency and holds promise for patient-specific therapies. Discerning whether genomic imprinting changes during reprogramming represent epigenetic abnormalities or essential adaptations linked to pluripotency is crucial. Here, we perform RNA-seq and MeDIP-seq analysis on mouse iPSCs derived from neural stem cells (NSCs). Our findings reveal that ICRs undergo DNA hypomethylation, confirming widespread LOI in pluripotent cells. However, the IG-DMR within the Dlk1-Dio3 imprinted cluster resists hypomethylation, a hallmark of successful pluripotency acquisition. We also identify a non-canonical role of TET3 in IG-DMR methylation protection through transcriptional regulation of Oct4 and Trim28. These findings highlight genomic imprinting as a key mechanism of gene dosage control in pluripotency acquisition and maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c296d4d11f624a61242abca49dfb84c1148d060b" target='_blank'>
              TET3 protects the Dlk1-Dio3 Imprinted Locus from DNA hypomethylation during adult NSC Reprogramming
              </a>
            </td>
          <td>
            Laura Lázaro-Carot, Esteban Jiménez-Villalba, Jordi Planells, Anna Lozano-Ureña, Jennifer Díaz-Moncho, Raquel Montalbán-Loro, Adela Lleches-Padilla, Martina Kirstein, Mitsu Ito, Elisabeth J. Radford, Sacri R. Ferrón
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Human genome sequencing efforts in healthy and diseased individuals continue to identify a broad spectrum of genetic variants associated with predisposition, progression, and therapeutic outcomes for diseases like cancer1–6. Insights derived from these studies have significant potential to guide clinical diagnoses and treatment decisions; however, the relative importance and functional impact of most genetic variants remain poorly understood. Precision genome editing technologies like base and prime editing can be used to systematically engineer and interrogate diverse types of endogenous genetic variants in their native context7–9. We and others have recently developed and applied scalable sensor-based screening approaches to engineer and measure the phenotypes produced by thousands of endogenous mutations in vitro10–12. However, the impact of most genetic variants in the physiological in vivo setting, including contextual differences depending on the tissue or microenvironment, remains unexplored. Here, we integrate new cross-species base editing sensor libraries with syngeneic cancer mouse models to develop a multiplexed in vivo platform for systematic functional analysis of endogenous genetic variants in primary and disseminated malignancies. We used this platform to screen 13,840 guide RNAs designed to engineer 7,783 human cancer-associated mutations mapping to 489 endogenous protein-coding genes, allowing us to construct a rich compendium of putative functional interactions between genes, mutations, and physiological contexts. Our findings suggest that the physiological in vivo environment and cellular organotropism are important contextual determinants of specific gene-variant phenotypes. We also show that many mutations and their in vivo effects fail to be detected with standard CRISPR-Cas9 nuclease approaches and often produce discordant phenotypes, potentially due to site-specific amino acid selection- or separation-of-function mechanisms. This versatile platform could be deployed to investigate how genetic variation impacts diverse in vivo phenotypes associated with cancer and other genetic diseases, as well as identify new potential therapeutic avenues to treat human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2f71f3527a2a35f380d7482a2555f7e6eeef4ab" target='_blank'>
              Multiplexed in vivo base editing identifies functional gene-variant-context interactions
              </a>
            </td>
          <td>
            Jonuelle Acosta, Grace A. Johnson, Samuel I. Gould, Kexin Dong, Yovel Lendner, Diego Detrés, Ondine Atwa, Jari Bulkens, Samuel Gruber, Manuel E. Contreras, Alexandra N. Wuest, Varun K. Narendra, Michael T. Hemann, Francisco J. Sánchez-Rivera
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Aberrant activation of transposable elements (TEs) has been a well-documented source of genomic instability and disease, stemming from their insertion into genes and their imposition of epigenetic effects on nearby loci. However, the extent to which their disruptive effects involve concomitant or subsequent formation of DNA:RNA hybrids (R-loops) remains unknown. Here we used DNA:RNA immunoprecipitation followed by high-throughput sequencing (DRIP-seq) to map the R-loop profiles of TEs and satellites in D. melanogaster ovaries in control and rhino knockout flies, where dozens of TE families are derepressed. We observe that R-loops form primarily in LTR retrotransposons that carry A/T-rich sequence motifs, which are known to favor R-loop formation at genes in Drosophila and other species. We also report evidence of R-loop formation at 11 of 14 highly abundant D. melanogaster DNA satellites. R-loop formation is positively correlated with expression level for both TEs and satellites; however, neither sequence content nor expression fully explain which repeat families form R-loops, suggesting other factors are at play. Finally, by analyzing population frequencies of R-loop forming TEs, we present evidence that TE copies with high R-loop signal may be under stronger negative selection, which suggests that R-loop formation by TEs may be deleterious to their host. Collectively, these results provide insight into the determinants of R-loop formation at repetitive elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a23fd0fe606d16164cbf75d226bfad5eec1b690f" target='_blank'>
              Complex determinants of R-loop formation at transposable elements and major DNA satellites.
              </a>
            </td>
          <td>
            Timothy J. Stanek, Adam Kneebone, Matthew A Lawlor, Weihuan Cao, Christopher E Ellison
          </td>
          <td>2025-02-28</td>
          <td>Genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e1743fa19de0544fdab9fff89543caabd4c7a18" target='_blank'>
              D-2-hydroxyglutarate impairs DNA repair through epigenetic reprogramming
              </a>
            </td>
          <td>
            Fengchao Lang, Karambir Kaur, Haiqing Fu, Javeria Zaheer, Diego Luis Ribeiro, M. Aladjem, Chunzhang Yang
          </td>
          <td>2025-02-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 Among the many features of cancer cells, they commonly display aberrant accumulation of RNA-DNA hybrids known as R-loops. R-loops are formed in healthy cells during many cellular processes, including transcription and DNA replication, and are crucial for proper maintenance of telomeres and centromeres, as well as for adequate mitochondrial function. However, aberrant accumulation of R-loops can lead to increases in DNA damage and inflammation. R-loop accumulation has been documented in different types of cancer, including BRCA1-mutant breast cancer, diffuse large B cell lymphoma, and myelodysplastic syndrome (MDS). Indeed, accumulation of R-loops has been proposed as a unifying mechanism underlying MDS and other myeloid malignancies. Ten-eleven translocation enzymes (TET1, TET2, TET3) are 5-methylcytosine dioxygenases that undergo frequent mutational or functional inactivation in human cancers. We have shown that mice with profound, acute TET deficiency induced by deletion of all Tet genes (hereafter referred to as Tet iTKO mice) show rapid onset of an uncontrolled malignant myeloid expansion. Expanded myeloid cells from Tet iTKO mice show increased levels of R-loops, together with increased DNA damage and inflammation, compared to control myeloid cells. Using Tet iTKO hematopoietic stem and progenitor cells (HSPC), we performed an in vivo CRISPR knockout screen of genes encoding RNA and DNA sensors, as well as R-loop regulators, to find how loss of genes related to nucleic acid sensing and innate immune response would affect the proliferation of TET-deficient myeloid cells. We observed that loss of enzymes that prevent the accumulation of R-loops impaired the uncontrolled malignant myeloid expansion of Tet iTKO cells. CRISPR-mediated depletion of the gene encoding RNase H1, an enzyme that specifically degrades the RNA strand in R-loops, further increased R-loop levels and DNA damage, and decreased proliferation and in vivo expansion of TET-deficient myeloid cancer cells, compared to Tet iTKO cells treated with non-targeting sgRNAs. Moreover, the accumulation of DNA breaks in Tet iTKO cells upon Rnaseh1 loss was not observed in Tet wild-type cells similarly depleted for RNase H1, suggesting a role for TET enzymes and potentially TET-oxidized 5-methylcytosines in DNA repair, which will be discussed in this presentation. Overall, these findings indicate that further increasing R-loops in TET-deficient cancer cells blocks their proliferation, and suggest that RNase H depletion or inhibition could be a new therapeutic strategy for cancer treatment in cancers that display augmented levels of R-loops, independent of the presence of TET mutations.
 Citation Format: Isaac F López-Moyado, Otokani Ortega de la Tejera, Lot M Hernández-Espinosa, Arthur X Cheng, Roberta Nowak, Zhen Dong, Caitlin Brown, André Nussenzweig, Anjana Rao. Forced accumulation of R-loops through inhibiton of RNase H impairs the proliferation of TET-deficient cancer cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr PR005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c95c688477a3d43d5e5b015493b97f465e9711d" target='_blank'>
              Abstract PR005: Forced accumulation of R-loops through inhibiton of RNase H impairs the proliferation of TET-deficient cancer cells
              </a>
            </td>
          <td>
            Isaac F. López-Moyado, Otokani Ortega de la Tejera, Lot M Hernández-Espinosa, A. X. Cheng, Roberta Nowak, Zhen Dong, Caitlin Brown, André Nussenzweig, Anjana Rao
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 For the last thirty years an increasing amount of evidence has shown the relevance of epigenetics in cell biology and tissue physiology, being DNA methylation aberrations in cancer the flag-ship for the recognition of its disturbance in human diseases. From the candidate gene approaches, new powerful technologies such as comprehensive DNA methylation microarrays and whole genome bisulfite sequencing has recently emerged that have reinforced the notion of epigenetic disruption in the crossroad of many sickness. From the poster-boy case of MGMT hypermethylation in the prediction of alkylating drug response to the personalized treatment of leukemia with small molecules targeted to fusion proteins involving histone modifiers, the field has walked a long path. The current talk will focus in the epigenetic profiling, basically at the level of DNA methylation and histone modifications, that is starting to provide clinical value in the diagnosis, prognosis and prediction of response to drug therapies. For cancer, we have already a wide view of the undergoing DNA methylation events that expand beyond classical promoter CpG islands of tumor suppressor genes and we have a growing list of mutated chromatin remodeler genes that contributes to the tumorigenesis process. It is time to apply this knowledge in practical clinical situations like the diagnosis of cancers of unknown primary, the screening of malignancies in high-risk populations or a biomarker selection of the patients that should receive treatment with anticancer drugs, including immunotherapy. Beyond cancer, DNA methylation is starting to be recognized as playing a major role in infectious diseases, and in this regard, the present lecture will also address the epigenomic component of COVID-19. Most important, Epigenomics can be added to Genomics, Transcriptomics and Proteomics plus Bioinformatics to create a Multomics view of human diseases.
 Citation Format: Manel Esteller. DNA methylation in cancer: From knowledge to applications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr IA007">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/855f061aa62959677f73a3c7e17c250ce96d93c4" target='_blank'>
              Abstract IA007: DNA methylation in cancer: From knowledge to applications
              </a>
            </td>
          <td>
            M. Esteller
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241144354fceff41346aae0d566101a01ccaa823" target='_blank'>
              Advances in cancer genomics and precision oncology.
              </a>
            </td>
          <td>
            Yonjong Heo, Woo-Jin Kim, Yong-Joon Cho, Jae-Won Jung, Nam-Soo Kim, Ik-Young Choi
          </td>
          <td>2025-01-23</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7c7a49a64fc95173d062b597a609f1c34ffa6ed" target='_blank'>
              PIF/harbinger transposon-derived protein promotes 7SL expression to enhance pathogen resistance
              </a>
            </td>
          <td>
            S. Geng, Xing Lv, T. Xu
          </td>
          <td>2025-01-30</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Resistance to radiotherapy remains a critical barrier in treating colorectal cancer (CRC), particularly in cases of locally advanced rectal cancer (LARC). To identify key kinases involved in CRC radioresistance, we employed a kinase‐targeted CRISPR‐Cas9 library screen. This approach aimed to identify potential kinase inhibitors as radiosensitizers. Our screening identified salt‐inducible kinase 2 (SIK2) as a critical factor in CRC radioresistance. Increased SIK2 expression correlated with reduced tumor regression and poorer outcomes in LARC patients undergoing neoadjuvant chemoradiotherapy. The depletion of SIK2 significantly enhanced radiation‐induced apoptosis and tumor regression. Mechanistically, SIK2 interacts with valosin‐containing protein (VCP), promoting its hyperphosphorylation. This modification improves VCP's capacity to extract K48‐linked ubiquitin‐conjugated proteins from chromatin, thus aiding the recruitment of RPA and RAD51 to DNA damage sites. This mechanism strengthens homologous recombination–mediated DNA repair, which contributes to radioresistance. Importantly, ARN‐3236, a SIK2 inhibitor, markedly sensitized CRC cells to radiation both in vivo and in vitro, providing a potential strategy to overcome radioresistance. In summary, our findings reveal a novel mechanism by which SIK2 contributes to the radioresistance of CRC, proposing SIK2 as a potential therapeutic target with its inhibitor significantly enhancing CRC radiotherapy efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27d0bffcf12a04202c03cb4fe50e95cbe2133af6" target='_blank'>
              Salt‐inducible kinase 2 confers radioresistance in colorectal cancer by facilitating homologous recombination repair
              </a>
            </td>
          <td>
            Yuan Meng, Shuo Li, Da‐Shan Lu, Xue Chen, Lu Li, You‐fa Duan, Gao‐yuan Wang, Wenlin Huang, Ran-Yi Liu
          </td>
          <td>2025-01-28</td>
          <td>MedComm</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer is one of the leading causes of cancer-related deaths in women worldwide. Circular RNAs (circRNAs), a novel class of endogenous noncoding RNA with a covalently closed continuous loop that lacks the 5'-cap structure and the 3'-poly A tail, are more stable than linear RNAs and less susceptible to degradation by nucleases. CircRNAs are widespread in multiple mammalian genomes and have been detected in various tissues, cells and body fluids. Increasing evidence shows that abnormal expression of circRNAs is involved in the development of a variety of diseases, including breast cancer. Numerous studies have explored the potential of circRNAs as biomarkers in various malignant tumors. In this review, we aim to provide a comprehensive overview of the latest advances in circRNAs as promising biomarkers in the early diagnosis, prognosis, molecular type, metastasis and drug resistance of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4edebdbfc4e64a85c8a092875820d3d39ea54f7" target='_blank'>
              Circular RNAs as Biomarkers in Breast Cancer Diagnosis, Prognosis, Molecular Types, Metastasis and Drug Resistance.
              </a>
            </td>
          <td>
            Li-Xin Li, Yun Hao, Lei Dong, Zhong-Qi Qiao, , Yan-Duo Chen, Kai Zhang, Ya-Wen Wang
          </td>
          <td>2025-03-01</td>
          <td>Technology in cancer research & treatment</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The detection of cytosolic dsDNA is tightly regulated to avoid pathological inflammatory responses. A major pathway involved in their detection relies on the cyclic GMP-AMP synthase (cGAS) that triggers activation of the Stimulator of interferon genes (STING) which subsequently drives the expression of inflammatory genes and type I Interferons (IFNs). Here, we show that the methyl-CpG-binding protein 2 (MECP2), a major transcriptional regulator, controls dsDNA-associated inflammatory responses. We show that the presence of cytosolic dsDNA promotes MECP2 export from the nucleus to the cytosol where it interacts with dsDNA, dampening cGAS activation. Our data also indicate that MECP2 export from the nucleus partially phenocopies MECP2 deficiency, leading to the expression of inflammatory and interferon stimulated genes, enforcing an antiviral state. Finally, we also show that MECP2 displacement from the nucleus following dsDNA stimulation is sufficient to disrupt its canonical function, leading to the reactivation of otherwise repressed genes, such endogenous retroelements of the Long interspersed nuclear element-1 (LINE-1) family. Re-expression of the latter led to the accumulation of DNA species feeding cGAS-dependent signaling and can be dampened by reverse transcriptase inhibitors. We thus establish a previously unforeseen direct role of MECP2 in the regulation of the breadth and nature of dsDNA-associated inflammatory responses. Furthermore, our results suggest that targeting dsDNA-associated pathways or pharmacological inhibition of LINE-1 may bear therapeutic hopes for Rett syndrome (RTT) patients that present with MECP2 deficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3893a0d8a3ff4afd4b28c233f6c8386b6ca8970" target='_blank'>
              The methyl-CpG-binding protein 2 inhibits cGAS-associated signaling
              </a>
            </td>
          <td>
            Hanane Chamma, Soumyabrata Guha, Moritz Schüssler, Yasmine Messaoud-Nacer, Pierre Le Hars, Mohammad Salma, Joe McKellar, Joanna Re, Morgane Chemarin, Arnaud Carrier, Michael A. Disyak, C. Taffoni, Robin Charpentier, Zoé Husson, E. Valjent, Charlotte Andrieu-Soler, Eric Soler, Karim Majzoub, I. Vila, Nadine Laguette
          </td>
          <td>2025-02-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The nuclear pore complex (NPC) is an evolutionarily conserved structure that maintains the traffic between the nucleus and cytoplasm. Here, we profiled the expression of nucleoporins (NUPs) in glioblastoma stem cells (GSCs) and found that NUP98 promoted GSC maintenance and therapeutic resistance. GSCs preferentially expressed NUP98, which is essential for GSC tumorigenesis in vitro and in vivo. RNA sequencing demonstrated that NUP98 regulated the expression of key DNA damage and repair pathways. NUP98 formed a complex with transcription factor p65 to directly activate genes involved in homologous repair. Attenuation of NUP98 or p65 expression induced unrepaired intrinsic DNA damage and sensitized GSC to ionizing radiation. Clinically, overexpression of NUP98 informs poor clinical outcome among glioblastoma (GBM) patients. Collectively, our results demonstrate that NUP98-p65 represents a novel node in the regulation of DNA repair, suggesting a therapeutic strategy with potential clinical benefits for GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f65a9f26936f417e9c87891e0b2241ca67fff8c1" target='_blank'>
              NUP98-p65 complex regulates DNA repair to maintain glioblastoma stem cells.
              </a>
            </td>
          <td>
            Feifei Li, Ying Zhang, Jiahui Li, Ranran Jiang, Shusheng Ci
          </td>
          <td>2025-02-17</td>
          <td>FASEB journal : official publication of the Federation of American Societies for Experimental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The Exon Junction Complex (EJC) is formed by the essential eIF4A3, MAGOH and Y14 core proteins. It is universally deposited during splicing at exon-exon junctions. The EJC is known to impact almost every post-transcriptional regulatory step throughout the life of mRNAs including their modifications, splicing, decay and trafficking. Its dysregulation leads to neurodevelopmental pathologies. Here we show that EJC-i, a compound known to block the ATPase activity of eIF4A3, inhibits de novo EJC assembly. EJC-i and targeted knockdown of either eIF4A3 or Y14 core EJC subunits lead to very similar phenotypes by impacting the destiny of mRNAs due to alterations in alternative splicing, nonsense-mediated mRNA decay, genome-wide m6A methylation and proper intracellular addressing of specific transcripts, in particular to the centrosome. Both EJC impairment methods disrupt the centrosome function that might be responsible for mitotic arrest at prometaphase. As a small molecule that readily diffuses into cells, EJC-i is a particularly attractive easy to use and versatile tool to investigate EJC functions in live cells or whole organism that are not prone to genetic manipulation. Indeed, this property was used to disrupt ciliogenesis in primary neural stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d82ec08c9ad540baf17cea17836bedb8e4f45c6" target='_blank'>
              Chemical inhibition of Exon Junction Complex assembly impairs mRNA localization and neural stem cells ciliogenesis
              </a>
            </td>
          <td>
            Tommaso Villa, Oriane Pourcelot, D. Dierks, Marion Faucourt, Cindy Burel, Floric Slimani, Léa Guyonnet, Nathalie Spassky, Schraga Schwartz, Edouard Bertrand, Olivier Bensaude, H. Le Hir
          </td>
          <td>2025-03-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb8360ddf104ac6e4b2843cdfd9b7edf6a857c0d" target='_blank'>
              Dysregulation of transposable elements and PIWI-interacting RNAs in myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Z. Krejcík, D. Kundrát, Jirí Kléma, A. Hruštincová, I. Trsova, M. Belickova, Jaroslav Čermák, A. Jonášová, Jiri Dostal, M. Dostalova Merkerova
          </td>
          <td>2025-01-23</td>
          <td>Biomarker Research</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="In eukaryotes, DNA-associated protein complexes coevolve with genomic sequences to orchestrate chromatin folding. We investigate the relationship between DNA sequence and the spontaneous loading and activity of chromatin components in the absence of coevolution. Using bacterial genomes integrated into Saccharomyces cerevisiae, which diverged from yeast more than 2 billion years ago, we show that nucleosomes, cohesins, and associated transcriptional machinery can lead to the formation of two different chromatin archetypes, one transcribed and the other silent, independently of heterochromatin formation. These two archetypes also form on eukaryotic exogenous sequences, depend on sequence composition, and can be predicted using neural networks trained on the native genome. They do not mix in the nucleus, leading to a bipartite nuclear compartmentalization, reminiscent of the organization of vertebrate nuclei.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/653fd5d6406b6894ce87859c219648b7e181e16b" target='_blank'>
              Sequence-dependent activity and compartmentalization of foreign DNA in a eukaryotic nucleus.
              </a>
            </td>
          <td>
            Léa Meneu, Christophe Chapard, Jacques Serizay, Alex Westbrook, Etienne Routhier, Myriam Ruault, Manon Perrot, Alexandros Minakakis, Fabien Girard, Amaury Bignaud, Antoine Even, G. Gourgues, Domenico Libri, C. Lartigue, Aurèle Piazza, Agnès Thierry, Angela Taddei, Frédéric Beckouët, J. Mozziconacci, Romain Koszul
          </td>
          <td>2025-02-07</td>
          <td>Science</td>
          <td>1</td>
          <td>26</td>
        </tr>

        <tr id="Introduction Traditional drug discovery efforts primarily target rapid, reversible protein-mediated adaptations to counteract cancer cell resistance. However, cancer cells also utilize DNA-based strategies, often perceived as slow, irreversible changes like point mutations or drug-resistant clone selection. Extrachromosomal DNA (ecDNA), in contrast, represents a rapid, reversible, and predictable DNA alteration critical for cancer’s adaptive response. Methods In this study, we developed a novel post-processing pipeline for automated detection and quantification of ecDNA in metaphase Fluorescence in situ Hybridization (FISH) images, leveraging the Microscopy Image Analyzer (MIA) tool. This pipeline is tailored to monitor ecDNA dynamics during drug treatment. Results Our approach effectively quantified ecDNA changes, providing a robust framework for analyzing the adaptive responses of cancer cells under therapeutic pressure. Discussion The pipeline not only serves as a valuable resource for automating ecDNA detection in metaphase FISH images but also highlights the role of ecDNA in facilitating swift and reversible adaptation to epigenetic remodeling agents such as JQ1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed078aa348654cdad2d2c2164c322729a574c316" target='_blank'>
              Leveraging AI to automate detection and quantification of extrachromosomal DNA to decode drug responses
              </a>
            </td>
          <td>
            Kohen Goble, Aarav Mehta, Damien Guilbaud, Jacob Fessler, Jingting Chen, William Nenad, Christina Ford, Oliver Cope, Darby Cheng, William Dennis, Nithya Gurumurthy, Yue Wang, Kriti Shukla, Elizabeth Brunk
          </td>
          <td>2025-02-03</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Rationale: Tumor cell epigenetics, especially chromosome accessibility, has been reported to be closely related to the tumor immune landscape and immunotherapy. However, the exact mechanism remains unknown. Methods: Whole-exome sequencing was used to analyze 13 colorectal tumor samples treated with PD1 immunotherapy. The assays for transposase-accessible chromatin using sequencing (ATAC-seq) and RNA sequencing were used to detect tumor cells' chromosome accessibility status and screen regulatory pathways. Results: Polybromo-1 (PBRM1) was one of the 12 genes with the highest frequency of somatic mutations associated with immunotherapy sensitivity. PBRM1/Pbrm1 deficiency in colorectal cancer promoted PD-1 immunotherapy sensitivity and chemotaxis of CD8+ T and NK cells in the microenvironment in vivo and in vitro. ATAC sequencing revealed that deletion of Pbrm1, a critical component of the SWI/SNF complex, increased chromosomal accessibility in tumor cells and triggered the release of cytokines, such as CCL5 and CXCL10, by activating the NF-κB signaling pathway. Application of ACBL1, a PROC inhibitor of PBRM1, in BALB/C mice or colorectal patient-derived tumor organoids (PDTOs) significantly promoted the sensitivity to PD1 antibody immunotherapy. Conclusions: Our study established that PBRM1/Pbrm1 deficiency was positively correlated with PD1 immunotherapeutic sensitivity in colorectal cancer. The underlying molecular mechanisms involved regulation of chromosome accessibility, activation of the NF-κB signaling pathway, and immune cell infiltration in the microenvironment. These findings identify potential molecular targets for enhancing immunotherapy for colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba7444e0c65d77850e20585c8a5ec38a58620ecd" target='_blank'>
              PBRM1 deficiency enhances PD1 immunotherapeutic sensitivity via chromosomal accessibility in colorectal cancer
              </a>
            </td>
          <td>
            Rui Li, Jie He, Chaoqun Liu, Zesheng Jiang, Jiasheng Qin, Kun Liang, Zhuocheng Ji, Liang Zhao
          </td>
          <td>2025-02-18</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Somatic alterations, like mutations and copy number changes, driver oncogenesis and cancer progression. Their inhibition has been exploited in the clinic, with several targeted therapies approved for patients with specific mutations or amplifications. However, the response rate of these treatments remains low. The causes are several, ranging from clonal heterogeneity to off target binding. For this reason, CRISPR assays have been developed to study the exact effect of a gene’s deletion. Still, the results from them are puzzling with the same alterations responding different to knockout even in the same cellular context. For this reason, we have developed SAEG, a novel deep learning architecture for somatic alterations in cancer. Our architecture is able to model mutations and copy number alterations and protein-protein interactions to predict if a cell will be susceptible to a gene knockout. SAEG outperforms other models and we show that it learns patterns that can be traced back to the biochemical and biological properties of genes and amino acids. Code Availability https://github.com/Luisiglm/SAEG Contact luis.iglesiarmatinez@ucd.ie">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caba93fa83f15f4694f814ad4d5d269b25293aad" target='_blank'>
              SAEG: A Novel Deep Learning Architecture for Somatic Alterations
              </a>
            </td>
          <td>
            Luis F. Iglesias-Martinez, Dan Yang Wester, Walter Kolch
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA damage responses primarily coordinate intracellular processes, but their effects on the cell surface remain poorly understood. We uncovered a DNA damage response at the cell surface that regulates complement dependent cytotoxicity (CDC), a key mechanism of antibody-mediated tumor cell killing. We found that several mCRPs, such as CD46, CD55, and CD59, are upregulated in diffuse large B-cell lymphoma (DLBCL) cells following chemotherapy. These mCRPs protect cells from CDC, including CDC triggered by anti-CD20 immunotherapy. Chemotherapy pre-treatment reduced the efficacy of rituximab-induced CDC, with CD59 exerting a dominant function. A high-throughput kinase screen revealed that Chk1 inhibition restored CDC sensitivity. Mechanistically, chemotherapy-induced CD59 expression was Chk1-dependent and regulated at an Sp1-driven promoter site. Inhibition of Sp1 with Mithramycin A abrogated chemotherapy-induced CD59 transcription. Co-immunoprecipitation mass spectrometry revealed that DNA damage releases transcriptional repressors from Sp1, an effect reversed by Chk1 inhibition. These findings define a Chk1-Sp1-CD59 signaling axis linking genotoxic stress to complement modulation. Our study reveals potential antagonism between chemotherapy and antibody-mediated CDC and highlights consideration in sequencing combination therapies and a therapeutic potential of Chk1 inhibitors in maintaining CDC efficacy during chemoimmunotherapy. Short Summary Rituximab-based chemoimmunotherapy is the first-line treatment for DLBCL. Our study reveals that chemotherapy induces resistance to rituximab-mediated cell killing. We identify this resistance as driven by a Chk1-Sp1-CD59 signaling axis and provide a therapeutic rationale for targeting this pathway to enhance chemoimmunotherapy efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0992e4a25dabd2c0752a1440000c36e4af2e792" target='_blank'>
              A Chk1-Sp1-CD59 axis of the DNA damage response impedes rituximab-mediated complement-dependent cytotoxicity
              </a>
            </td>
          <td>
            Allison S. Y. Chan, P. Jaynes, A. Anbuselvan, C. Z. Ong, M. Hoppe, Wai Khang Yong, Vartika Khanchandani, Jie Min Lee, Nurulhuda Mustafa, M. I. Azaman, Phuong M. Hoang, Guo Hong, W. Chng, M. Cragg, Dennis Kappei, C. Tripodo, Anand D. Jeyasekharan
          </td>
          <td>2025-02-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="SUMMARYHuman papillomaviruses (HPVs) are small DNA viruses that are responsible for significant disease burdens worldwide, including cancers of the cervix, anogenital tract, and oropharynx. HPVs infect stratified epithelia at a variety of body locations and link their productive life cycles to the differentiation of the host cell. These viruses have evolved sophisticated mechanisms to exploit cellular pathways, such as DNA damage repair (DDR), to regulate their life cycles. HPVs activate key DDR pathways such as ATM, ATR, and FA, which are critical for maintaining genomic integrity but are often dysregulated in cancers. Importantly, these DDR pathways are essential for HPV replication in undifferentiated cells and amplification upon differentiation. The ability to modulate these DDR pathways not only enables HPV persistence but also contributes to cellular transformation. In this review, we discuss the recent advances in understanding the mechanisms by which HPV manipulates the host DDR pathways and how these depend upon enhanced topoisomerase activity and R-loop formation. Furthermore, the strategies to manipulate DDR pathways utilized by high-risk HPVs are compared with those used by other DNA viruses that exhibit similarities and distinct differences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c3977d9b3bc13e1ba0a9ea26fcf6605bb8b2b35" target='_blank'>
              How human papillomavirus (HPV) targets DNA repair pathways for viral replication: from guardian to accomplice.
              </a>
            </td>
          <td>
            A. Vats, Lou Laimins
          </td>
          <td>2025-01-27</td>
          <td>Microbiology and molecular biology reviews : MMBR</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Human endogenous retroviruses (HERV) represent nearly 8% of the human genome. Of these, HERV-K subtype HML-2 is a transposable element that plays a critical role in embryonic development and in the pathogenesis of several diseases. Quantification and characterization of these multiple HML-2 insertions in the human chromosome has been challenging due to their size, sequence homology with each other, and their repetitive nature. We examined a cohort of 222 individuals for HML-2 proviruses 6q14.1 and 7p22.1a, two loci that are capable of producing full-length viral proteins and have been previously implicated in several cancers, autoimmune disorders and neurodegenerative diseases, using long-read DNA sequencing. While the reference genome for both regions suggests these two loci are structurally dissimilar, we found that for both loci about 5% of individuals have a unique tandem repeat-like sequence (three long terminal repeat sequences sandwiching two internal, potentially protein coding sequences), while most individuals have a standard proviral structure (one internal region sandwiched by two long terminal repeats). Moreover, both proviruses can make full-length, or nearly full-length, HERV-K proteins in multiple transcription orientations. The amino acid sequences from different loci in the same transcriptional orientation share sequence homology with each other. These results demonstrate a clear, previously unreported, relationship between HML-2 loci 6q14.1 and 7p22.1a and highlight the utility of long-read sequencing to study repetitive elements. Future studies need to determine if these polymorphisms determine genetic susceptibility to diseases that are associated with them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd67bb0a75d66d24c8cfbfff72aa27068e59aecc" target='_blank'>
              Characterization of novel human endogenous retrovirus structures on chromosomes 6 and 7
              </a>
            </td>
          <td>
            Nicholas Pasternack, Ole Paulsen, Avindra Nath
          </td>
          <td>2025-01-27</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb7e063d7f27e7ff5ba777247b0efc2ec0bcfdb4" target='_blank'>
              Functional analysis of cancer-associated germline risk variants.
              </a>
            </td>
          <td>
            Laura N. Kellman, Poornima H. Neela, Suhas Srinivasan, Z. Siprashvili, Ronald L. Shanderson, Audrey W. Hong, Deepti S Rao, Douglas F Porter, D. Reynolds, Robin M. Meyers, M. Guo, Xue Yang, Yang Zhao, Glenn G Wozniak, L. Donohue, R. Shenoy, Lisa A Ko, Duy T. Nguyen, Smarajit Mondal, Omar S. Garcia, L. Elcavage, Ibtihal Elfaki, Nathan S. Abell, Shiying Tao, Christopher M Lopez, Stephen B. Montgomery, Paul A. Khavari
          </td>
          <td>2025-02-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ef97a03979c9e266e81720f51438d4d5cd7b23" target='_blank'>
              BRCA1 is involved in sustaining rapid antler growth possibly via balancing of the p53/endoplasmic reticulum stress signaling pathway
              </a>
            </td>
          <td>
            Qianqian Guo, Zhen Wang, Jiping Li, Chao Ma, Junjun Zheng, H. Ba, Guokun Zhang, Chunyi Li
          </td>
          <td>2025-01-24</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Historically considered downstream effects of tumorigenesis—arising from changes in DNA content or chromatin organization—nuclear alterations have long been seen as mere prognostic markers within a genome‐centric model of cancer. However, recent findings have placed the nuclear envelope (NE) at the forefront of tumor progression, highlighting its active role in mediating cellular responses to mechanical forces. Despite significant progress, the precise interplay between NE components and cancer progression remains under debate. In this review, we provide a comprehensive and up‐to‐date overview of how changes in NE composition affect nuclear mechanics and facilitate malignant transformation, grounded in the latest molecular and functional studies. We also review recent research that uses advanced technologies, including artificial intelligence, to predict malignancy risk and treatment outcomes by analyzing nuclear morphology. Finally, we discuss how progress in understanding nuclear mechanics has paved the way for mechanotherapy—a promising cancer treatment approach that exploits the mechanical differences between cancerous and healthy cells. Shifting the perspective on NE alterations from mere diagnostic markers to potential therapeutic targets, this review calls for further investigation into the evolving role of the NE in cancer, highlighting the potential for innovative strategies to transform conventional cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53ec969f050943713ac6f56187d24a844cc59674" target='_blank'>
              At the nucleus of cancer: how the nuclear envelope controls tumor progression
              </a>
            </td>
          <td>
            F. Paganelli, Alessandro Poli, Serena Truocchio, Alberto M. Martelli, Carla Palumbo, Giovanna Lattanzi, Francesca Chiarini
          </td>
          <td>2025-01-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Protein methyltransferase (PRMT)-catalysed arginine methylation is a widespread post-translational modification that regulates numerous RNA-binding proteins and frequently becomes dysregulated in cancer. While PRMT inhibitors have shown promise as an anti-cancer strategy, greater understanding of the downstream pathways linking arginine methylation to tumour-promoting phenotypes is needed to improve patient stratification and develop more effective therapeutic approaches. Here, we reveal arginine methylation as a critical regulator of alternative polyadenylation (APA) patterns that are fundamental to tumour progression. 3′ RNA-sequencing assays uncover a rapid and global shift toward longer 3′ UTR isoforms upon dual (symmetric and asymmetric) methylation (DMAi), impacting a broad range of cellular proliferation and signalling genes. Arginine methylation is required for sustaining proximal poly(A) site usage under high proliferative demand, as DMAi treatment blocks use of such sites in activated T cells, various cancer cell lines and patient-derived lung organoids. DMAi also counteracts the 3′ UTR shortening caused by reduced CFIM25 expression, which normally promotes oncogenic isoforms. DMAi treatment affects APA in many of the same mRNAs as impaired cleavage and polyadenylation activity, and these mRNAs contain characteristic signatures such as high GC-content and long 3’ UTRs. This systematic impact of PRMT activity on APA regulation broadens the potential utility of PRMT inhibitors as therapeutic agents for both cancer and immune-related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43a623593b9a20fd048f7dbc4cfe070fa57a8f07" target='_blank'>
              PRMT activity promotes global 3’ UTR shortening in proliferating cells
              </a>
            </td>
          <td>
            Llywelyn Griffith, Charlotte Capitanchik, Shaun Moore, Anca Farcas, Mags Gwynne, Martina Pedna, D. M. Jones, A. Chakrabarti, Dimitris Lagos, Jelena Urosevic, James T. Lynch, J. Ule
          </td>
          <td>2025-03-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Epigenetic modifications, which reversibly regulate gene expression without altering the DNA sequence, are increasingly described in the literature as essential elements in the processes leading to cancer development. SETDB1 regulates histone 3 (H3) K9 di- and trimethylation, promoting heterochromatin formation, and plays a key role in gene silencing. Epigenetic deregulation of SETDB1 expression appears to be involved in different cancers types, particularly in aggressive, relapsing or treatment-resistant subtypes. Despite advances in research, the full range of mechanisms through which this protein acts remains unclear; however, it is evident that SETDB1 has a pivotal role, particularly in the mesenchymal stem cells differentiation, tumor evasion and treatment resistance. Its role in genetically complex sarcomas, such as osteosarcoma, has not been fully explored, although recent Omics analyses suggest its presence and amplification in osteosarcoma. Given its involvement in osteoblastogenesis and adipogenesis, we discuss the potential of SETDB1 as a key target for new therapeutic strategies in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b378d2be6324f27ab95439dedf132da8d385c12" target='_blank'>
              SETDB1 amplification in osteosarcomas: Insights from its role in healthy tissues and other cancer types
              </a>
            </td>
          <td>
            Elodie Verdier, Nathalie Gaspar, Maria Eugénia Marques da Costa, A. Marchais
          </td>
          <td>2025-02-12</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer remains a major global health challenge, with prostate cancer, lung cancer, colorectal cancer, and breast cancer accounting for nearly half of all diagnoses. Despite advancements in cancer treatment, metastasis to distant organs continues to be the leading cause of cancer-related mortality. The progression of cancer involves the alteration of numerous genes, with dynamic changes in chromatin organization and histone modifications playing a critical role in regulating cancer-associated genes. Special AT-rich sequence-binding protein 1 (SATB1), a critical chromatin organizer, plays a pivotal role in cancer progression by regulating gene expression, chromatin remodeling, and cell signaling pathways. SATB1 binds to AT-rich DNA sequences, acting as a scaffold for chromatin-modifying enzymes and transcription factors, thus coordinating the regulation of extensive gene networks. Its overexpression has been implicated in a wide range of cancers and is associated with poor prognosis, aggressive tumor phenotypes, and enhanced epithelial–mesenchymal transition (EMT). Moreover, SATB1’s activity is modulated by microRNAs (miRNAs) and post-translational modifications, further contributing to its complex regulatory functions. Given its crucial involvement in cancer progression and metastasis, SATB1 has emerged as a promising target for novel therapeutic strategies. This review delves into the molecular mechanisms of SATB1 in cancer and explores potential therapeutic approaches for targeting this key regulator in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/013b48f08c648e41770b775be56070039b081ae8" target='_blank'>
              SATB1 in cancer progression and metastasis: mechanisms and therapeutic potential
              </a>
            </td>
          <td>
            Jinping Bai, Gege Yang, Qi Yu, Qianya Chi, Xianlu Zeng, Wenjing Qi
          </td>
          <td>2025-02-25</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa423d61a8720aa08805a6d061e12902556221c5" target='_blank'>
              Comprehensive landscape and oncogenic role of extrachromosomal circular DNA in malignant biliary strictures
              </a>
            </td>
          <td>
            Zhuo Cheng, Xuanmei Luo, Wenzheng Liu, Xiaofang Lu, Hong Chang, Yingchun Wang, Wei Zheng, Xiue Yan, Yonghui Huang
          </td>
          <td>2025-02-07</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Four out of five patients with autoimmune diseases are women. The XIST ribonucleoprotein (RNP) complex, comprising the female-specific long noncoding RNA XIST and over 100 associated proteins, may drive several autoimmune diseases that disproportionately affect women, who have elevated levels of autoantibodies against the XIST RNP. However, the structural distribution, potential origin, and clinical significance of XIST RNP autoantibodies remained unexplored. Here, we find that XIST RNP is associated with autoantigens associated with six female-biased autoimmune conditions. Mapping autoantibody targets to their occupancy sites on XIST shows that these autoantigens are concentrated at discrete “hotspots” along the XIST lncRNA, notably the A-repeat. Cell type-specific protein expression data nominated neutrophils as a predominant source of hotspot antigens, and we confirmed the presence of both XIST and hotspot antigens in neutrophil extracellular traps during NETosis, an immunogenic programmed cell death pathway triggered by neutrophil activation upon which neutrophils extrude their nuclear content. Furthermore, we found that levels of autoantibodies against a top hotspot antigen, SPEN, that binds the A-repeat, correlate with severe digital ischemia in systemic sclerosis in two independent cohorts. Together, these data show a plausible mechanism for the origin of AXA, guided by RNA structure and RNA-protein interactions, and show that antibodies to XIST RNP holds promise for disease endotyping and prognostication in female-biased autoimmune conditions. One Sentence Summary Novel autoantibodies target hotspots on XIST ribonucleoprotein complex in female-biased autoimmune diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2def919ff2ecf3dfede1a094bde8e83c3054f368" target='_blank'>
              Autoantibody hotspots reveal origin and impact of immunogenic XIST ribonucleoprotein complex
              </a>
            </td>
          <td>
            Bingyu Yan, Jinwoo Lee, Suhas Srinivasan, Quanming Shi, Diana R. Dou, S. Davuluri, Swarna Nandyala, A. Woods, Gwendolyn Leatherman, Yanding Zhao, Roman E. Reggiardo, Manasi Sawant, H. R. Thiam, Ami A. Shah, David F. Fiorentino, Lorinda S. Chung, Howard Y. Chang
          </td>
          <td>2025-01-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) remains a leading cause of cancer associated deaths in women owing to its highly metastatic potential and limited treatment options. Recent studies have shown that expression of proteins associated with epigenetic regulation of gene expression are associated with metastatic relapse, however targeting epigenetic regulatory proteins has not resulted in effective therapies for TNBC in the clinic. The ABL tyrosine kinases promote metastasis of breast cancer cells in mouse models. However, a role of ABL kinases in the regulation of epigenetic processes in solid tumor metastasis remains unexplored. Here we show that inactivation of ABL kinases in bone metastatic TNBC cells led to a significant enrichment in gene signatures associated with the PRC2 protein complex, revealing a functional link between ABL kinases and the PRC2 complex. ABL inactivation promotes EZH2-T487 phosphorylation through the regulation of a FAK-CDK1 signaling axis. We find that phosphorylated EZH2 T487 or a phosphomimic EZH2 T487D mutant exhibit increased binding to non-canonical binding partners of EZH2 including c-MYC and ZMYND8. Notably, we identify a therapeutic vulnerability in TNBC cells whereby combination treatment with ABL allosteric inhibitors and EZH2 inhibitors elicits a synergistic decrease in TNBC cell survival in vitro, and impairs TNBC metastasis, prolonging survival of tumor-bearing mice treated with the combination therapy. One Sentence Summary ABL Kinases indirectly impact EZH2 catalytic activity by blocking a signaling cascade that leads to changes in the phosphorylation, protein interactions, and function of the PRC2 catalytic component EZH2 in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a73bead74e9165b080991fb6c70e4def4cd0e8e6" target='_blank'>
              ABL Kinases Modulate EZH2 Phosphorylation and Signaling in Metastatic Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Ashley Colemon, Tatiana Prioleau, Clay Rouse, A. Pendergast
          </td>
          <td>2025-02-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Amplified centrosome number causes genomic instability, most severely through division into more than two aneuploid daughter cells (multipolar mitosis). Several mechanisms that suppress multipolar division have been uncovered, yet mechanisms that favor viable multipolar division are poorly understood. To uncover factors that promote viability in cells with frequent centrosome amplification and multipolar division, we conducted an unbiased Drosophila genetic screen. In 642 mutagenized lines, we exploited the ability of intestinal papillar cells to form and function despite multipolar divisions. Our top hit is an unnamed gene, CG3168. We name this gene synaptic vesicle glycoprotein 2, reflecting homology to human Synaptic Vesicle Glycoprotein 2 (SV2) proteins. GFP-tagged SV2 localizes to the plasma membrane. In cells with amplified centrosomes, SV2 positions membrane-adjacent centrosomes, which prevents severe errors in chromosome alignment and segregation. Our results uncover membrane-based multipolar division regulation and reveal a novel vulnerability in cells with common cancer properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d9667c9a72868a1c9d7c6fb195bb8c2fd73a80a" target='_blank'>
              Synaptic vesicle glycoprotein 2 enables viable aneuploidy following centrosome amplification
              </a>
            </td>
          <td>
            Jane E. Blackmer, Erin A. Jezuit, Archan Chakraborty, Ruth A. Montague, Nora G Peterson, William Outlaw, Donald T. Fox
          </td>
          <td>2025-02-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 DNA double strand breaks (DSBs) result in activation of several key DNA damage response (DDR) kinases including ATM, ATR, and DNA-PK. These protein kinases not only promote DNA damage-induced checkpoint control, but also facilitate DSB repair in humans. Thus, these DDR kinases have become promising drug targets for cancer therapy. However, the benefits of targeting DDR kinases remain to be realized, in part due to the lack of predictive biomarkers. By undertaking CRISPR screens with inhibitors targeting key DDR kinases, we obtained a global and unbiased view of genetic interactions with DDR inhibition. Additionally, we compared the synergistic effects of combining different DDR inhibitors and found that ATM and PARP inhibitors showed remarkable synergy, which depends on the dominant negative function of ATM inhibition. Moreover, to provide comprehensive and unbiased perspective of DDR signaling pathways, we performed 30 fluorescence-activated cell sorting–based genome-wide CRISPR screens with antibodies recognizing distinct endogenous DNA damage–signaling proteins to identify new regulators involved in DNA damage response (DDR). We discovered that proteasome-mediated processing is an early and prerequisite event for cells to trigger camptothecin- and etoposide-induced DDR signaling. Furthermore, we identified PRMT1 and PRMT5 as new modulators that regulate ATM protein level. Additionally, we discovered that GNB1L is a master regulator of DDR signaling via its role as a co-chaperone for PIKK proteins. Collectively, these screens offer a rich resource for further investigation of DDR, which may provide insight into strategies of targeting these DDR pathways to improve therapeutic outcomes. More recently, we have adopted dTAG technology for the investigation of many essential DDR and replication proteins, which provide mechanistic insights into the roles of these essential proteins in genome maintenance. Additionally, we have expanded FACS-based screens for the studies of other cancer-associated pathways and explored key biological processes critical for cancer progression in vivo. These studies will facilitate the development of better therapies for cancer patients.
 Citation Format: Junjie Chen. Targeting DNA damage responsive pathways in cancer therapy. [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Translating Targeted Therapies in Combination with Radiotherapy; 2025 Jan 26-29; San Diego, CA. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(2_Suppl):Abstract nr A009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b542469fcebc2361e69bec6d43fd73238a87bdc" target='_blank'>
              Abstract A009: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-01-26</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

Globally, lung cancer continues to be the primary cause of cancer-related fatalities. Despite
significant advancements in chemotherapy, targeted therapy, and immunotherapy that have improved
treatment effectiveness and increased survival rates, the five-year relative survival rate for
lung cancer patients remains very low. Most cases receive a diagnosis at an advanced stage, which
primarily contributes to this. Drug resistance is still a significant challenge, which is responsible for
90% of cancer-related deaths. Epigenetic alterations, including DNA methylation, RNA modification,
and histone modification, play crucial roles in cancer treatment resistance. Among these, the methyltransferase-
like 3 (METTL3)-induced N6-methyladenosine (m6A) RNA modification has received a
lot of attention because it plays a part in lung cancer tumor growth, carcinogenesis, proliferation, and
resistance to treatment. Researchers have identified or developed several METTL3 inhibitors that
have shown promising therapeutic benefits in both in vivo and in vitro models. This review focuses
on the progress that has been made in the research investigating the role of METTL3 in lung cancer
treatment resistance. It also elucidates the processes that are at play and investigates prospective therapeutic
options that target METTL3 in order to enhance treatment results and overcome drug resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b3a2a7fdf1a317ed911bfc6a066f5d8abc14563" target='_blank'>
              Exploring the Function of METTL3 in Lung Cancer: Understanding Drug Resistance and Exploring Innovative Therapeutic Approaches
              </a>
            </td>
          <td>
            Shiqing Huang, Ye Li, F. Liew
          </td>
          <td>2025-03-12</td>
          <td>Current Cancer Therapy Reviews</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/352956df3f50dd3a8802a00f3ce2fb6c110258ba" target='_blank'>
              SMC translocation is unaffected by an excess of nucleoid associated proteins in vivo
              </a>
            </td>
          <td>
            Zhongqing Ren, Lindsey E. Way, Xindan Wang
          </td>
          <td>2025-01-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Gene fusion events result in chimeric proteins that are frequently found in human cancers. Specific targeted therapies are available for several types of cancer fusions including receptor tyrosine kinase gene moieties. RNA sequencing (RNAseq) can directly be used for detection of gene rearrangements in a single test, along with multiple additional biomarkers. However, tumor biosamples are usually formalin-fixed paraffin-embedded (FFPE) tissue blocks where RNA is heavily degraded, which in theory may result in decreased efficiency of fusion detection. Here, for the first time, we compared the efficacy of gene fusion detection by RNAseq for matched pairs of freshly frozen in RNA stabilizing solution (FF) and FFPE tumor tissue samples obtained from 29 human colorectal cancer patients. We detected no statistically significant difference in the number of chimeric transcripts in FFPE and FF RNAseq profiles. The known fusion KANSL1-ARL17A/B occurred with a high frequency in 69% of the patients. We also detected 93 new fusion genes not mentioned in the literature or listed in the ChimerSeq database. Among them, 11 were found in two or more patients, suggesting their potential role in carcinogenesis. Most of the fusions detected most probably represented read-through, microdeletion or local duplication events. Finally, in one patient, we detected a potentially clinically actionable in-frame fusion of LRRFIP2 and ALK genes not previously described in colorectal cancer with an intact tyrosine kinase domain that can be potentially targeted by ALK inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/620d2dc5c8d58680ec55371dff6419f2e6129a29" target='_blank'>
              Detection of fusion events by RNA sequencing in FFPE versus freshly frozen colorectal cancer tissue samples
              </a>
            </td>
          <td>
            M. Sorokin, Vladimir Lyadov, M. Suntsova, Marat Garipov, Anna Semenova, Natalia Popova, E.P. Guguchkin, R. Heydarov, Marianna A. Zolotovskaia, Xiaowen Zhao, Qing Yan, Ye Wang, Evgeny Karpulevich, Anton Buzdin
          </td>
          <td>2025-01-21</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="DNA G-quadruplexes (G4s) are non-B form secondary DNA structures that are highly conserved across evolution. G4 structures occupy key regulatory sites in the mammalian genomes and are implicated in several cellular processes. However, the mechanisms by which G4s contribute to distinct facets of genome function are not well understood. Here, we conduct a proteomics screen with G4s of diverse topologies to uncover novel G4 binding activities in genomic regulators of nucleosome remodeling, paraspeckle assembly, RNA splicing, and 3D genome organization. Among the most prominent hits, we identify the genomic architectural protein, CTCF, as one of the strongest G4 binders. Building on this discovery, we perform extensive biochemical validation of CTCF-G4 interaction and identify CTCF mutants, with pronounced affinity for G4s over its consensus DNA motif. By implementing well-established approaches and developing new G4 mapping tools, we define a comprehensive catalog of genomic G4s and investigate their association with CTCF binding. Using genetic reconstitution of mouse embryonic stem cells with a G4-specific CTCF mutant, we ascribe genomic functions to CTCF-G4 interactions in regulation of CTCF occupancy, chromatin looping and gene expression. Interestingly, our studies reveal a subset of G4-linked chromatin loop anchors that form “persistent loops” which are retained even upon CTCF depletion. Collectively, our work establishes the G4 binding activity of CTCF and provides new insights into the functional significance of G4 structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a92ea67e64484dade77882133de154f4d5bb2f2" target='_blank'>
              CTCF directly binds G-quadruplex structures to regulate genome topology and gene expression
              </a>
            </td>
          <td>
            Daniela Samaniego-Castruita, Isabella Han, Roxroy C Morgan, Samantha Carpenter, Bryce Williams, Abhijit Chakraborty, Ishwar Radhakrishnan, F. Ay, Samuel A. Myers, Anjana Rao, Vipul Shukla
          </td>
          <td>2025-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The nucleotide excision repair (NER) mechanism is responsible for the removal of bulky DNA damage such as pyrimidine dimers induced by ultraviolet light. The NER pathway detects such lesions and excises the damaged strand through incisions at 5 ’ and 3’ of the damage. The 5’ incision is catalyzed by a heterodimeric endonuclease composed of XPF (catalytic subunit) and ERCC1 (non-catalytic). Here, we show that the genome of Trypanosoma brucei, the causal agent of human African trypanosomiasis or sleeping sickness, codes for an XPF ortholog. RNAi silencing of TbXPF sensitizes cells to UV irradiation, thus providing evidence that NER operates in these parasites. In addition, TbXPF confers protection against intra- and inter-strand crosslinks induced by cisplatin and mitomycin C respectively. Consistent with a role in DNA repair, XPF localizes to the cell nucleus, and is found associated to nucleoplasmic and nucleolar regions. The presence of a functional NER pathway in trypanosomes suggests that in vivo, they are susceptible to undergo replication and transcription-blocking DNA damages. The results obtained with various antitumor agents provide proof of concept for the potential of NER inhibition as a means to improve antiparasitic therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4eda64b2f73b09173646bdfd81a7c10d708ef6e" target='_blank'>
              Role of the Nucleotide Excision Repair endonuclease XPF in the kinetoplastid parasite Trypanosoma brucei
              </a>
            </td>
          <td>
            Claudia Gomez-Liñan, María Sáez-Maldonado, L. Ruiz‐Pérez, Dolores González-Pacanowska, Antonio E. Vidal
          </td>
          <td>2025-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e1b48d57468e7ee31e0d2b20955ec88056a5d42" target='_blank'>
              LINE-1 ORF1p expression occurs in clear cell ovarian carcinoma precursors and is a candidate blood biomarker
              </a>
            </td>
          <td>
            Pamela R. de Santiago, Sho Sato, Stephanie J. Zhang, Meaghan Dougher, Kyle M. Devins, A. Bilecz, Sagar Rayamajhi, Gabriel Mingo, Hannah S. Rendulich, Yi Feng, Connie Wu, Martin S Taylor, Yelena Zhuravlev, Euihye Jung, Dalia K Omran, Tian-Li Wang, I. Shih, Lauren E. Schwartz, Sarah Kim, Mark Aloysuis Morgan, J. Tanyi, Kathleen H. Burns, Ernst Lengyel, C. Parra‐Herran, Andrew K Godwin, David R. Walt, Ronny Drapkin
          </td>
          <td>2025-03-06</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="MET fusions (MET-Fs) are oncogenic drivers that remain poorly characterized. Analysis of 56 MET-F-positive tumors from an institutional cohort of 91,119 patients (79,864 DNA-seq plus 11,255 RNA-seq) uncovered two forms of MET-F pathobiology. The first group featured 5' partners with homodimerization domains fused in-frame with MET-tyrosine kinase domain (TKD), primarily originated from translocations, frequently excluded MET exon 14, mediated oncogenesis through cytoplasmic aggregation and constitutive activation, and were markedly sensitive to MET tyrosine kinase inhibitors (TKI) in pre-clinical models and patients with lung cancer. The second group lacked partner homodimerization motifs and retained MET transmembrane and extracellular domains. Their pathogenesis involved intrachromosomal rearrangements, resulting in partner selection for promoter hijacking and fusion allele amplification. Membrane-bound fusions were enriched in gliomas with RTK co-alterations. We provide a framework to comprehend the heterogeneous landscape of MET-Fs, supporting that fusion oncogenicity and MET-TKI sensitivity are determined by structural topology and pathogenomic context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020be0c73016e3bdb9e553a5f923d18e28c49838" target='_blank'>
              Pan-cancer analysis of oncogenic MET fusions reveals distinct pathogenomic subsets with differential sensitivity to MET-targeted therapy.
              </a>
            </td>
          <td>
            Christopher A Febres-Aldana, M. Vojnic, I. Odintsov, Tom Zhang, Ryan Cheng, Catherine Z. Beach, Daniel Lu, M. Mattar, Andrea M Gazzo, Leo Gili, M. Harshan, Ali Ameri, S. Machnicki, Xiuying Xiao, William W Lockwood, Xiaoyan Zhou, Qianlan Yao, A. Drilon, Natasha Rekhtman, Nameeta Shah, Anqi Li, Zebing Liu, Soo-Ryum Yang, M. Davare, Marc Ladanyi, R. Somwar
          </td>
          <td>2025-02-20</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="In hypermutant cancers, both the mutation rate per cell division and the genomic locations of new mutations are vastly altered. Such cancers often display atypical driver gene variants, which may provide sub-optimal oncogenic advantages. Why such weaker cancer drivers prevail in these tumours is unclear. Here, we examine the null hypothesis that aberrant mutational processes alone can account for the detection of weaker, rather than canonical, driver mutations. Using a mathematical modelling analysis, we find that simply increasing the mutation rate per-division can lead to the detection of weaker drivers, an effect which is further compounded by the biassing of mutations towards weaker driver sequence contexts. Focusing on POLE-mutant (DNA polymerase epsilon proofreading-deficient) colorectal cancers and KRAS driver mutations, we quantify the mutational bias towards non-canonical KRAS drivers in both POLE-mutant and non-hypermutant colorectal cancers. The observed bias, coupled with the increased mutation rate of POLE-mutant cancers, is sufficient to explain the enrichment of atypical KRAS drivers in POLE-mutant cancers with plausible selection parameters. Thus, within our model, differential selection between these cancer types need not be invoked to understand the observed variation in driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7891a3a8df530bd356f3a5da9253356957c1253d" target='_blank'>
              Hyper-mutational processes provide a head-start for non-optimal cancer driver mutations explaining atypical KRAS variants
              </a>
            </td>
          <td>
            Michael D. Nicholson, Ian Tomlinson
          </td>
          <td>2025-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal cancer (CRC) represents the third-leading cause of cancer-related deaths. Knowledge covering diverse cellular and molecular data from individual patients has become valuable for diagnosis, prognosis, and treatment selection. Here, we present an in-depth comparative mRNA-seq and microRNA-seq analysis of tissue samples from 32 CRC, pairing tumors with adjacent healthy tissues. The differential expression gene (DEG) analysis revealed an interconnection between nutrients, metabolic programs, and cell cycle pathways. We focused on the impact of overexpressed SLC7A11 (xCT) and SLC3A2 genes which compose the cystine/glutamate transporter (Xc-) system. To assess the oncogenic potency of the Xc-system in a cellular setting, we applied a knowledge-based approach for analyzing gene perturbations from CRISPR screens across various cell types as well as using a variety of functional assays in five primary patient-derived organoid cell models to functionally verify our hypothesis. We identified a previously undescribed cell surface protein signature predicting chemotherapy resistance and further highlighted the causality and potential of pharmacological blockage of ferroptosis as promising avenue for cancer therapy. Biological processes such as redox homeostasis, ion/amino acid transporters and de novo nucleotide synthesis were associated with these co-dependent genes in patient specimens. This study highlighted a number of overlooked genes as potential clinical targets for CRC and promotes stem cell-based, patient-individual in vitro model systems as a versatile partner platform to functionally validate in silico predictions, with focus on SLC7A11 and its associated genes in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06923ea47f54b55109b8b50bd2b097751afa1aed" target='_blank'>
              Patient-Specific Pharmacogenomics unveils xCT as a Key Regulator for Druggable Metabolic and Proliferation Pathways in Colon Cancer
              </a>
            </td>
          <td>
            Marco Strecker, Keren Zohar, Martin Böttcher, Thomas Wartmann, Henry Freudenstein, Maximilian Doelling, Mihailo Andric, W. Shi, O. Kakhlon, Katrin Hippe, Beatrix Jahnke, Dimitrios Mougiakakos, Franziska Baenke, Daniel Stange, R. Croner, M. Linial, U. D. Kahlert
          </td>
          <td>2025-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2e7583ed7379bcbc35a51a5c2875a8894ab63e6" target='_blank'>
              Global characterization of extrachromosomal circular DNAs in four body fluids.
              </a>
            </td>
          <td>
            Lanrui Jiang, Mengyuan Song, Feng Song, Shuangshuang Wang, Yuxiang Zhou, Ze Wang, Chaoran Sun, Hewen Yao, Zhirui Zhang, Xindi Wang, Miao Liao, Yanyun Wang, Haibo Luo
          </td>
          <td>2025-02-13</td>
          <td>International journal of legal medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Colorectal cancer (CRC) remains a significant global health challenge, underscoring the need for innovative therapeutic strategies. Oncogenic miRNAs (oncomiRs) play a significant biological role in the initiation and progression of colorectal cancer. Inspired by the cooperative mechanisms of plant nanovirus, which employ multiple circular single-stranded DNA (CssDNA) genomes, it is hypothesized that the development and delivery of CssDNA to target oncomiRs would achieve therapeutic benefits in CRC. In this study, a multi-omics approach is utilized to identify key tumor suppressor genes (TSGs) and their related oncomiRs implicated in CRC, followed by the development of CssDNA, each of which is loaded with multiple miRNA binding sites targeting one oncomiR. When transfected into the cells, these CssDNA can effectively target and sequester the corresponding oncomiRs to restore the expression of TSGs, leading to a marked reduction in CRC development both in vitro and in vivo. The findings highlight the therapeutic potential of nanovirus-inspired CssDNA in modulating the miRNA-mediated regulatory network in CRC. This study lays the groundwork for the development of non-coding DNA-based therapies with broad implications for the treatment of colorectal cancer and potentially other malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a389d3e4c09149bfce4cb4ae8bf6fdc4a619a01" target='_blank'>
              Circular Single-Stranded DNA-Based Artificial Nanoviruses Mitigate Colorectal Cancer Development.
              </a>
            </td>
          <td>
            Jinghao Wang, Pengfei Zhang, Yonglian Huang, Guang Hu, Kexuan Zou, Songtao Zhou, Dandan Shao, Jianming Wang, Jie Song
          </td>
          <td>2025-01-21</td>
          <td>Small methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Purpose: To look into the influence of DNAJC12 knockdown on the progression and radio-sensitivity of esophageal squamous cell carcinoma (ESCC), with a focus on cellular mechanics and tumor microenvironment interactions. Methods: The TCGA database combined with immunohistochemical staining was used to validate the DNAJC12 expression in ESCC patients from the perspective of the clinic. DNAJC12 knockdown was performed in TE-1 and KYSE-150 cell lines to assess changes in proliferation, migration, invasion, apoptosis, and cellular mechanical properties (e.g., stiffness, adhesion, and contractility). The downstream molecule regulated by DNAJC12 was explored using Western blotting and biomechanical assays. The effect of DNAJC12 knockdown on tumor radiosensitivity was evaluated in vivo, with a focus on tumor stiffness and extracellular matrix (ECM) remodeling under irradiated conditions. Results: Upon analyzing the TCGA database and examining tumor tissue samples from patients, it was discovered that DNAJC12 exhibited high expression levels in tissues of ESCC. Vitro experiments showed that DNAJC12 knockdown significantly decreased cellular proliferation and migration (P < 0.05). Biomechanical assays revealed that DNAJC12 knockdown decreased cellular stiffness and contractility, suggesting a role in regulating cytoskeletal dynamics. Molecular analysis showed downregulation of P-ERK, MMP-2, N-Cadherin, P-P38, Snail, Vimentin, β-Catenin, Fibronectin, and Twist alongside upregulation of E-Cadherin (P < 0.05). Overexpression of SNAI1 could restore the proliferative and migratory capabilities of cells with downregulated DNAJC12. In vivo experiments, knockdown of DNAJC12 resulted in faster tumor growth under irradiated conditions (P < 0.05). Conclusion: DNAJC12 knockdown slows ESCC progression by modulating cellular biomechanical properties and molecular pathways. However, it enhances tumor growth post-radiotherapy, potentially due to altered mechanosensitive signaling and ECM remodeling. These findings highlight the interplay between molecular biology and biomechanics in ESCC progression and treatment response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49eb9686b4a983fe402a837ae9d2a07843ebb7fe" target='_blank'>
              Knockdown of DNAJC12 slows tumor progression and affects tumor radiosensitivity in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiao Ju, Jianbo Zhang, Linke Yang, Pei Li, Ping Wang
          </td>
          <td>2025-03-06</td>
          <td>Molecular &amp; Cellular Biomechanics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 11


 Background:
 Metastatic squamous cell carcinoma of the anus (SCCA) is associated with significant morbidity and mortality. Due to its relative rarity, there is limited evidence to support systemic therapy regimens informed by genomic data. This study aims to utilize whole-genome and transcriptome sequencing to enhance the understanding of the genetic alterations driving metastatic SCCA and to identify potentially actionable therapeutic targets.
 Methods:
 This report examines nine cases of metastatic SCCA in patients enrolled in the Personalized Oncogenomics (POG) Program at the BC Cancer Agency. Comprehensive genomic profiling, including whole-genome and transcriptome analysis (WGTA) was conducted on fresh tumor biopsy or formalin fixed paraffin embedded (FFPE) tissue. Tumor genomic alterations were analyzed in detail including: SNVs, indels, copy number alterations, structural variants, mutation signatures, viral presence, tumor mutation burden (TMB), expression outliers, and immune cell scores. The somatic alterations were integrated with existing knowledge of drug-target interactions to identify actionable therapeutic targets.
 Results:
 Of the nine patients, eight were found to be HPV-positive, with one exhibiting high tumor mutational burden (TMB>10mut/Mb). Mutation signatures included seven cases with AID/APOBEC signatures (commonly seen with HPV), and one case with an ID2 DNA replication slippage signature. The PI3K/AKT/mTOR pathway was the most commonly affected signaling pathway (n=4), followed by FGFR amplification and overexpression (n=3), and RAS/RAF alterations (
 BRAF
 mutation and
 KRAS
 overexpression, n=2). EGFR amplification was identified in two cases, one of which also exhibited
 EGFR
 overexpression. Additionally,
 FBXW7
 mutations were present in two cases. Immunotherapy was recommended for all nine patients: eight based on HPV positivity, either alone or in combination with other alterations, such as high TMB, SWI/SNF complex alterations, and presence of immune infiltrating cells, and the remaining case based on a SWI/SNF complex mutation. Only two patients ultimately received immunotherapy, both of whom derived clinical benefit. One patient, treated with atezolizumab as part of the phase II CAPTIV-8 trial (NCT04273061), demonstrated progression-free survival (PFS) of 10 months (ongoing), while the other, treated with nivolumab, achieved a PFS of 6 months.
 Conclusions:
 Genomic analysis using WGTA supported the recommendation of immunotherapy for all patients in this cohort. Although only two patients received immunotherapy, both experienced clinical benefit, underscoring the potential of personalized genomic-guided treatment in metastatic SCCA.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c65e46bc3b8c5ed99a2bc82c211bf31bd2460d61" target='_blank'>
              Personalized oncogenomic analysis of metastatic squamous cell carcinoma of the anus: Utilizing whole-genome sequencing to guide clinical decision-making.
              </a>
            </td>
          <td>
            R. D. Peixoto, Melissa McConechy, Howard Lim, M. Marra, Jing Xu, Kathleen Wee, E. Pleasance, J. Laskin
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Colorectal cancer poses a major global health issue, profoundly affecting both mortality and morbidity rates across the world. A key obstacle in understanding the pathogenesis of colorectal cancer lies in its high inter-patient and spatial heterogeneity, making standard treatments ineffective. Commonly, the study on colorectal cancer relies on bulk RNA sequencing, offering an average gene expression profile for a heterogenous cell population. However, this approach obscures the heterogeneous characteristics of the cancer cells and hinders a comprehensive understanding of the complex interplay among different cell populations. Recently, the advent of single-cell RNA sequencing has been revolutionary, enabling researchers to analyze individual immune cells and overcome the limitations of bulk RNA sequencing. Through single-cell RNA sequencing, researchers have gained insights into the dynamic nature of the immune response to cancer and potential targets for immunotherapy. In this review, we discuss the technical aspects of single-cell RNA sequencing, the application of single-cell RNA sequencing in cancer immunology, and the potential of single-cell RNA in clinical settings. We believe that harnessing singlecell RNA sequencing in cancer research holds immense potential to drive the development of personalized immunotherapies, aiming to improve patient outcomes in colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddac82763361480546b876aad89a46e0a8e233fa" target='_blank'>
              Single Cell RNA Sequencing in Colorectal Cancer Immunology: Recent Updates, Application, and Emerging Challenges.
              </a>
            </td>
          <td>
            Sabrina George, Nor Adzimah Johdi
          </td>
          <td>2025-02-12</td>
          <td>Current molecular medicine</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/824026b9cf3feecceded2241b45f5f7f89022d53" target='_blank'>
              A CRISPR-Cas9 screen reveals genetic determinants of the cellular response to decitabine.
              </a>
            </td>
          <td>
            Pinqi Zhang, Zhuqiang Zhang, Yiyi Wang, Wenlong Du, Xingrui Song, Weiyi Lai, Hailin Wang, Bing Zhu, J. Xiong
          </td>
          <td>2025-02-10</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="SKA3, an important factor in cell cycle regulation, is involved in spindle assembly and kinetochore function, playing a critical role in maintaining cancer cell proliferation and division. However, its specific roles and regulatory mechanisms in cancer remain not fully understood. Large-scale datasets from multiple public databases, including The Cancer Genome Atlas and Genotype-Tissue Expression, covering various cancer types, were integrated. Systematic analysis revealed that SKA3 exhibits aberrant expression patterns in multiple cancers and is significantly associated with tumor progression and poor patient prognosis in certain cancers. We explored the status of SKA3 gene mutation, gene amplification and promoter region methylation in various tumors. In the context of immunotherapy, we assessed the value of SKA3 in cancer. Analyzing the correlation between SKA3 expression levels and immune checkpoints and immune cell infiltration, we discovered that SKA3 could serve as a novel immunotherapy biomarker across multiple cancers, guiding clinical immunotherapy decisions. Finally, SKA3 knockdown inhibited lung adenocarcinoma cell proliferation and metastasis. In conclusion, this study provides new insights into the role of SKA3 in cancer and offers significant theoretical and experimental evidence for its development as a diagnostic and prognostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cdcaafe55411db23c30a2e5da0f6e25b45d3a4b" target='_blank'>
              Pan-Cancer Analysis Reveals SKA3 as a Potential Diagnostic and Prognostic Biomarker.
              </a>
            </td>
          <td>
            Chunlin Li, Min Gao, Hua Huang, Nashunbayaer Zha, Gang Guo
          </td>
          <td>2025-02-25</td>
          <td>APMIS : acta pathologica, microbiologica, et immunologica Scandinavica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 226


 Background:
 Metastatic prostate cancer (PC) accumulates significant genomic alterations as the disease progresses, exhibiting the highest levels of chromosomal instability (CIN) among all metastatic tumors (Bakhoum, Nature 2018). Furthermore, tumor suppressor genes (TSG;
 RB1
 ,
 PTEN
 and
 TP53
 ) are frequently altered in metastatic PC and are associated with adverse outcomes. Here, we aim to investigate CIN in metastatic hormone-sensitive PC (mHSPC) harbouring alterations in TSG.
 Methods:
 TSG and whole transcriptome gene expression was assessed by nCounter platform (N=354) and RNA-seq (N=60) in mHPSC samples, respectively. TSG
 low
 was considered when ≥2 out of 3 TSG presented low expression of a previously stablished cut-off, and TSG
 wt
 in the remaining cases. Differential gene expression analysis between TSG
 low
 vs TSG
 wt
 tumors of our RNA-seq cohort (N=60) and TCGA cohort (N=333) was performed. Differential expressed genes (DEGs) were considered if FDR <0.05 and log2FoldChange (LFC)>0.5. DEGs were correlated with castration-resistance PC-free survival (CRPC-FS) and overall survival (OS) by Cox analysis. CIN gene signatures were obtained from literature and the rest of Hallmark signatures from MSigDB.
 Results:
 In our cohort of mHSPC patients treated with androgen deprivation therapy + docetaxel, TSG
 low
 tumors (16.7%) displayed higher levels of CIN compared with TSG
 wt
 (CIN25 normalized enrichment score [NES] 2.13, p<0.001 and CIN70 NES 2.54, p<0001). Cell cycle and DNA-repair signatures were also overexpressed in TSG
 low
 tumors, contrasting with a low expression of androgen receptor pathway. Results were validated in the TCGA cohort, where TSG
 low
 presented also overexpression of CIN signatures. In our RNA-seq cohort, we found 22 DEGs associated with CIN (29.3% of the total DEGs) in TSG
 low
 tumors compared with TSG
 wt
 . 51 genes were significantly correlated with CRPC-FS in a multivariate analysis and 16 of them (31,3%) were CIN-associated genes. Among these genes, 5 encode proteins that are targetable for metastatic PC, with specific inhibitors already developed:
 BUB1
 (LFC 1.06, FDR 0.034),
 CDC7
 (LFC 12.4 FDR 0.003),
 KIFC1
 , (LFC 11,7, FDR 0.01),
 PLK1
 (LFC 1.08, FDR <0.001) and
 WEE1
 (LFC 0.79, FDR 0.011).
 Conclusions:
 TSG-altered mHSPC tumors harbor high levels of CIN, and differentially expressed CIN-associated genes could potentially be targets in mHSPC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/243ddfb9d9bb3f9a966b17fe378947695a450d86" target='_blank'>
              Chromosomal instability in tumor suppressor genes–altered prostate cancer.
              </a>
            </td>
          <td>
            M. Garcia de Herreros, Natalia Jiménez, J. Padrosa, Òscar Reig, L. Fernández-Mañas, L. Ferrer-Mileo, S. García-Esteve, A. Font Pous, V. Ruiz de Porras, Begoña Mellado
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34beca577def84b7592da411aab4f563850a4d1a" target='_blank'>
              Tumour-wide RNA splicing aberrations generate actionable public neoantigens
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O Stevers, Iñaki Etxeberria, Takahide Nejo, Maggie Colton Cove, Lee H. Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Gary Chan, Jerry Liu, Samuel H. Wu, Emilio Ramos, A. Yamamichi, P. Watchmaker, Hirokazu Ogino, A. Saijo, Aidan Du, Nadia Grishanina, James Woo, Aaron Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideo Okada
          </td>
          <td>2025-02-19</td>
          <td>Nature</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="
 Epigenetic therapies that induce global DNA hypomethylation activate transcriptionally repressed transposable elements (TEs). The activation of a class of TEs known as human endogenous retroviruses (HERVs) can trigger an innate antiviral response in cells called “viral mimicry”. Viral mimicry induced by aberrantly expressed HERVs stimulates interferon signaling genes resulting in an adaptive immune response that can ultimately lead to immunogenic cell death. This heightened immune activity can enhance the efficacy of immune checkpoint inhibitor therapies. Identifying cancers that are susceptible to a viral mimicry response or treatment regimens that induce aberrant transcription of TEs has not yet been fully explored. Recent computational tools enable the measurement of TE signal from short-read RNA-seq data and several large-scale reprocessing pipelines have uniformly processed RNA-seq data of cancer cell lines treated with epigenetic and cytotoxic drugs. However, these large-scale pipelines omit the quantification of TEs. To address this gap and fully utilize publicly available data, we have developed a computational pipeline and database of TE expression for more than 7,000 samples from 220 cancer cell lines treated with over 700 cytotoxic and epigenetic drugs, yielding over 2,100 differential expression experiments. To illustrate the usefulness of this database, we ranked drugs by %TE activation and found that the top three targets were DNMT (Decitabine), CDK9 (MC180295), and Topoisomerase II (Doxorubicin). We enable the exploration of this dataset by implementing several meta-analysis methods and visualizations made available as an interactive web application. This database and web application can serve as a valuable resource for the cancer and epigenetics community by enabling hypothesis generation and providing insights into how specific drugs may induce TE expression.
 Citation Format: Gennaro H Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre Issa. TEDEdb: Transposable element differential expression database [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr B022.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5443bedc86457ac044103f87e00c65e563c2339" target='_blank'>
              Abstract B022: TEDEdb: Transposable element differential expression database
              </a>
            </td>
          <td>
            G. Calendo, Morgan Chaunzwa, Iman Dehzangi, Jozef Madzo, Jean-Pierre J Issa
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Radioresistance is one of the main reasons for the recurrence and metastasis of non-small cell lung cancer. Cyclin Y has been implicated in various cellular processes such as cell growth, proliferation, autophagy, and tumor progression. However, the function and regulatory mechanism of Cyclin Y in lung cancer radioresistance remain poorly understood. In this study, we find that Cyclin Y is overexpressed in non-small cell lung cancer and correlates with poor prognosis. Furthermore, knockdown of Cyclin Y results in inhibited cell growth and proliferation, increases DNA damage, impairs DNA damage repair, and enhances radiosensitivity in vitro and in vivo. Mechanistically, we uncover that Cyclin Y interacts with Chk1 and positively regulate both the mRNA and protein levels of RRM2, resulting in increased STAT3 phosphorylation. Rescue experiments confirm that the effects of Cyclin Y on lung cancer are mediated partially by RRM2. Collectively, we reveal for the first time that Cyclin Y promotes lung cancer radioresistance by binding to Chk1 to activate RRM2/STAT3 signaling, indicating that targeting Cyclin Y may be a promising strategy for enhancing the efficacy of radiotherapy in the treatment of non-small cell lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd382c38ca99977b603b78c35d219d19e3b14901" target='_blank'>
              Cyclin Y interacts with Chk1 to activate RRM2/STAT3 signaling and promotes radioresistance in non-small cell lung cancer
              </a>
            </td>
          <td>
            Zhiwei Liu, Huichan Xue, Zhi Wang, Ye Zhao, Shuang-Bing Xu, Xiaorong Dong
          </td>
          <td>2025-02-18</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55bd28583d34995b3cfd1d2547de4bdf67a2acf7" target='_blank'>
              Integrative single cell transcriptomic analysis reveals 3p deletion associated tumor microenvironment and chemoresistance in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Xiaochuan Chen, Siqi Xu, Junping Pan, Wenqian Xu, Hanxuan Yang, Xin Chen, Ronghui Chen, Yuan Wang, Sufang Qiu
          </td>
          <td>2025-03-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Simple Summary Prostate cancer (PCa) is a heterogenous malignancy in men, representing the second most diagnosed cancer and the fifth leading cause of cancer-related death globally in 2022. PCa incidence and progression is dependent on several predisposing factors, most significantly genetics and germline/somatic mutations. With the development of whole-genome sequencing (WGS) technology, it is now possible to determine the mutational statuses of PCa-related genes for improved clinical decision making and individualized patient care. The National Comprehensive Cancer Network (NCCN) Guidelines now recommend germline multigene testing followed by post-test genetic counseling. Depending on the germline mutations in homologous recombination (HR) repair genes or in mismatch repair (MMR) genes, specific treatment options, including poly-ADP-ribose polymerase (PARP) inhibition or PD-1 blockade, may be of more clinical benefit. This review highlights specific PCa-related germline mutations in racially diverse populations and therapeutic approaches to treat PCa patients with DNA damage repair gene alterations in advanced disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53f685df01c211531229e0572b44cf7405f9d7cc" target='_blank'>
              Ancestry-Specific DNA Damage Repair Gene Mutations and Prostate Cancer
              </a>
            </td>
          <td>
            Talaibek Borbiev, Kevin Babcock, K. Sinopole, Gregory T. Chesnut, G. Petrovics
          </td>
          <td>2025-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="ABSTRACT Tumour heterogeneity will accumulate and amplify during cell culture and passage, even if derived from the same strain. In the current study, multiple batches of colorectal cancer HCT116 and HT29 cell lines were obtained using different conditions of trypsin digestion and processed for RNA sequencing. CD39 was identified as a biomarker highly expressed in easily trypsin‐digested cells compared to the undigestible ones. Furthermore, CD39 was determined to enhance cell invasion and suppress cell apoptosis but not affect cell proliferation. Moreover, CD39 could activate the TGF‐β/SMAD3 signalling pathway, whereas the expression of CD39 was negatively regulated by SMAD3 via recruitment of SETDB1 and adding H3K9me3 to the CD39 promoter in HCT116 cells. Overall, our study uncovered distinct gene signatures amongst different heterogeneities of colorectal cells and revealed the effect of CD39 on cell invasion and apoptosis, as well as determined the epigenetic role in regulating CD39 transcription.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f4bb713ea12ae36c80b493256ba51492ce28889" target='_blank'>
              CD39 Contributes to the Ability of Cell Invasion in Heterogeneity of Colorectal Cancer
              </a>
            </td>
          <td>
            Xiaosong Li, Yifen Shen, Jianhua Lang, Jianzhong Wu, Zhenhai Qian, Genhai Shen, Yihang Shen
          </td>
          <td>2025-03-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0818d31f654dc2a6f66d1620f983e96d2fa6b573" target='_blank'>
              Genomic and fragmentomic landscapes of cell-free DNA for early cancer detection.
              </a>
            </td>
          <td>
            D. C. Bruhm, Nicholas A. Vulpescu, Zachariah H. Foda, J. Phallen, Robert B Scharpf, V. Velculescu
          </td>
          <td>2025-03-04</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="The current treatment of triple-negative breast cancer (TNBC) is still primarily based on platinum-based chemotherapy. However, TNBC cells frequently develop resistance to platinum and experience relapse after drug withdrawal. It is crucial to specifically target and eliminate cisplatin-tolerant cells after platinum administration. Here, it is reported that upregulated N 6-methyladenosine (m6A) modification drives the development of resistance in TNBC cells during cisplatin treatment. Mechanistically, histone deacetylase 2 (HDAC2) mediates delactylation of methyltransferase-like 3 (METTL3), facilitating METTL3 interaction with Wilms'-tumor-1-associated protein and subsequently increasing m6A of transcript-associated DNA damage repair. This ultimately promotes cell survival under cisplatin. Furthermore, pharmacological inhibition of HDAC2 using Tucidinostat can enhance the sensitivity of TNBC cells to cisplatin therapy. This study not only elucidates the biological function of lactylated METTL3 in tumor cells but also highlights its negative regulatory effect on cisplatin resistance. Additionally, it underscores the nonclassical functional mechanism of Tucidinostat as a HDAC inhibitor for improving the efficacy of cisplatin against TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c5ff71ca33fdaff595792bb621bbec0ae0038c4" target='_blank'>
              HDAC2-Mediated METTL3 Delactylation Promotes DNA Damage Repair and Chemotherapy Resistance in Triple-Negative Breast Cancer.
              </a>
            </td>
          <td>
            Xiaoniu He, Yuanpei Li, Jian Li, Yu Li, Sijie Chen, Xia Yan, Zhangrong Xie, Jiangfeng Du, Guoan Chen, Jianbo Song, Qi Mei
          </td>
          <td>2025-02-14</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer's inherent ability to evolve presents significant challenges for its categorization and treatment. Cancer evolution is driven by genetic, epigenetic, and phenotypic diversity influenced by microenvironment changes. Aging plays a crucial role by altering the microenvironment and inducing substantial genetic and epigenetic heterogeneity within an individual's somatic cells even before cancer initiation.This review highlights the clinical significance of epigenetic mechanisms in cancer evolution, focusing on hematopoietic and solid tumors. The review aims to explore opportunities for integrating evolutionary principles and data science into cancer research.The review synthesizes recent advancements in omics technologies, single‐cell sequencing, and genetic barcoding to elucidate epigenetic mechanisms and aging's role in cancer evolution.Epigenetic mechanisms' high plasticity generates heritable phenotypic diversity, driving malignant evolution toward poor prognosis. Advances in single‐cell sequencing and genetic barcoding enable the precise detection and tracking of biomarkers, allowing early, personalized interventions. Incorporating data science into cancer research has the potential to map, predict, and prevent cancer evolution effectively.Understanding cancer evolution through novel technologies and data analysis offers a proactive approach to cancer prevention and treatment. By predicting key evolutionary events and leveraging personalized strategies, patient outcomes can be improved, and healthcare burdens reduced, marking a transformative shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afc8601f31e88365526c6e3fa1b322df2e0a2b70" target='_blank'>
              Deciphering Aging, Genetic, and Epigenetic Heterogeneity in Cancer Evolution: Toward Personalized Precision Preventative Medicine
              </a>
            </td>
          <td>
            Lamis Naddaf, Sheng Li
          </td>
          <td>2025-01-28</td>
          <td>Aging and Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab8b68e7b5a2f581513237acff3c4e88418f66b" target='_blank'>
              Haploinsufficient phenotypes promote selection of PTEN and ARID1A-deficient clones in human colon
              </a>
            </td>
          <td>
            Nefeli Skoufou-Papoutsaki, Sam Adler, Shenay Mehmed, Claire Tume, Cora Olpe, Edward Morrissey, Richard Kemp, Anne-Claire Girard, Elisa B. Moutin, C. Chilamakuri, Jodi L Miller, Cecilia Lindskog, Fabian Werle, Kate Marks, Francesca Perrone, Matthias Zilbauer, D. Tourigny, Doug Winton
          </td>
          <td>2025-02-07</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9961e6ed0a6d198d471203dc74fb5135efa471b" target='_blank'>
              CDKN2A, a key gene in copper-induced cell death model, influencing melanoma invasion and apoptosis
              </a>
            </td>
          <td>
            Jing Li, Xi Yang, Cunli Yin, Siru Li, Yan Xu, Bin Liu
          </td>
          <td>2025-02-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 854


 Background:
 The question still remains for the presence of yet unknown genetic modifiers in the initiation and/or progression of bladder cancer (BC). It remains challenging to identify the key factors, the mechanism and potential target behind. A genome-wide CRISPR library colony formation assay was performed using UROtsa cells, an immortalized ureter epithelium cell line, in order to identify novel genes that promote the transformation of normal urothelial cells. As one of the top candidate genes, the Prolactin Releasing Hormone Receptor (PRLHR) was identified. To date, only a few studies addressed the impact and the correlation of PRLHR in cancer and reports in the context of BC are missing altogether.
 Methods:
 We assessed the gene and protein expressions via public database, qPCR and Western blot analyses in a set of nontumour cell strains (Y235T, UROtsa) and a set of BC cell lines (RT4, T24). Following the shRNA knockdown in the PRLHR-proficient cells(Y235T) and the ectopic PRLHR expression in the cell lines with low/absent PRLHR expression(T24), we performed functional experiments, such as soft agar colony formation assay, MTT, invadopodia and gelatine degradation assays. We determined the invasion and migration abilities of these genetically modified cells by using the and wound-healing, Boyden chamber and porcine bladder ex vivo organ culture invasion assay. Metaphase spread assay and Immunofluorescence was performed in modified Y235T cells for chromosome counting and microscopy observation (centrsomes, chromosomal misalignments and lagging chromosomes).
 Results:
 The available in silico expression data and expression analyses in BC cell lines and matched normal/tumour patient samples revealed the repression of PRLHR early in BC tumorigenesis. Soft agar experiments confirmed that ectopic PRLHR impairs colony formation of T24 cells, providing indirect evidence that the CRISPR-Cas9 mediated deficiency of PRLHR promoted UROtsa cells transformation in the screening assay. Functional experiments using the MTT and the wound healing assays showed that ectopic PRLHR improved the viability and migration abilities of BC cells. The Boyden chamber assay and the porcine bladder organ culture model revealed that PRLHR impairs BC cell invasion, likely by the inhibition of the AKT/PKB pathway. Moreover, we observed an abnormal number of centrosomes, an increasing number of lagging chromosomes in the anaphase and telophase of the PRLHR-knockdown cells during mitosis.
 Conclusions:
 The screening procedure identified PRHLR as a new candidate tumour suppressor factor, and that loss/down-regulation of this protein contributes to the progression of bladder cancer. Downregulation of PRLHR leads to aneuploidy and genome instability in urothelial cells. PRLHR might serve as a novel biomarker or potential target for BC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0fddfdbd53dd2702eeb20d67caa73e1dfbdec2c" target='_blank'>
              Impact of loss of PRLHR on aneuploidy and initiation of bladder cancer.
              </a>
            </td>
          <td>
            Xi Zheng, Bingjie Yang, Wenya Wang, A. Azoitei, Felix Wezel, Christian Bolenz, Hongqian Guo, Cagatay Günes
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Post-transcriptional regulation by RNA-binding proteins (RBPs) is critical for mRNA stability, localization, and translation, primarily through interaction with the 3’-untranslated region. Dysregulation of RBPs has been associated with various cancers, with fragile X-related protein 1 (FXR1) emerging as a critical RBP involved in tumorigenesis through its interactions with target mRNAs. Despite its significance, the specific role of FXRI in cancer progression remains underexplored. In this study, we investigated FXR1’s function using SH-SY5Y cells. RNA immunoprecipitation (RNA-IP) assay was employed to isolate RNA complexes associated with FXR1. We generated stable cell lines with either FXR1 overexpression or silencing to assess the impact of FXRI binding to mRNA complexes. Subsequent analyses, including quantitative reverse transcription polymerase chain reaction, correlation analysis, gene expression profiling, and survival analysis, were performed to validate the interactions of FXR1 with target mRNAs. Our RNA-IP analysis identified several mRNAs significantly enriched in FXR1-bound RNA complexes, including SHISAL1, SLC43A3, NBAT1, PDZK1IP1, ACKR3, KCNN3, NECAB2, ANO5, ATOH8, IGFBP7, LEMD1, GPR35, WNT7A, and F2RL3. Notably, we observed changes in the expression levels of these genes following FXR1 overexpression or depletion, indicating FXR1-mediated functional regulation. Co-expression analysis further supported FXR1’s association with these target mRNAs. These findings highlight the significant role of FXR1 in regulating the stability and expression of key mRNAs implicated in malignancies. The dysregulation of FXR1 and its interaction with these target transcripts suggest that FXR1 plays a critical role in tumor biology, potentially offering new avenues for therapeutic interventions. This study provides a deeper understanding of FXR1’s involvement in cancer and underscores the importance of RBPs in the post-transcriptional regulatory landscape of cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c68584e4b61388edd6ff9ff31b6a17585ebe8fc4" target='_blank'>
              FXR1 modulates the expression of oncogenes and tumor suppressor genes associated with poor cancer prognosis
              </a>
            </td>
          <td>
            F. Khan, Xin-Ying Ji, Shaoping Ji, Jingcheng Dong
          </td>
          <td>2025-02-05</td>
          <td>Gene &amp; Protein in Disease</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The p53 tumor suppressor was first identified as a cellular protein that bound to the large T antigen in SV40-transformed cells. Initially thought to be the product of an oncogene, p53 turned out to be an anticancer protein whose loss or mutation could promote tumorigenesis. Subsequent work revealed it functions as a DNA-binding transcription factor central to the DNA damage response and cell cycle control. In this excerpt from his forthcoming book on the history of cancer research, Joe Lipsick looks back at the discovery of p53 and the groundbreaking work that revealed its role as "guardian of the genome."">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c78c5dd12eb473d301e95ef3085c40bdc1f7b393" target='_blank'>
              A History of Cancer Research: The P53 Pathway.
              </a>
            </td>
          <td>
            Joseph Lipsick
          </td>
          <td>2025-02-03</td>
          <td>Cold Spring Harbor perspectives in medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="

 Glioblastoma(GBM) is a lethal intracranial neoplasm that inevitably recurs despite treatment with DNA damaging therapies, radiation(RT) and temozolomide(TMZ). Recurrence is likely driven by inherently resistant glioma stem-like cells(GSCs), which have baseline upregulation of DNA damage response(DDR) pathways due to elevated DNA replication stress(RS). Elevated DDR confers resistance to RT/TMZ however factors mediating elevated RS and therefore upregulated DDR in GSC are poorly defined.



 THY1 was overexpressed or silenced in patient-derived neurosphere and organoid models utilizing inducible shRNA systems. Models were irradiated and collected at various timepoints. Alterations in activation and completion of DNA repair were evaluated using yH2AX immunofluorescence and ATM/ATR activation via western blot. In parallel, GBM tissue from over 20 patients underwent single cell RNA sequencing and spatial transcriptomics to assess pathways associated with THY1.



 Spatial transcriptomics and single cell RNA sequencing data demonstrate significant upregulation of GSC pathways in high-THY1 cells, and constitutive upregulation of DDR pathways in high-THY1 cells compared to low-THY1 cells(p<2e-16), especially homologous recombination. In high-THY1 in vitro models, DNA repair activation is upregulated at baseline. Following irradiation, high-THY1 models mediate more rapid repair of DNA, with peak repair 2 hours after radiation.



 These findings demonstrate that GSCs exhibit elevated levels of DNA replication stress, and high baseline DDR pathway activation which allows for extremely rapid DNA repair. Here we demonstrate that THY1 mediates treatment resistance as result of constitutively upregulated DNA repair. Elucidation of the mechanisms mediating the GSC phenotype in high-THY1 cells is underway to better characterize the resulting DNA replication stress and upregulation of DNA repair, and identify a targetable pathway.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3eb942377bafe8f7e43907ef0e3522eb17cee58" target='_blank'>
              1333 THY1 Upregulates Glioma Stem Cell Phenotype and Induces Treatment Resistance in Glioblastoma
              </a>
            </td>
          <td>
            Arushi Tripathy, S. Syed, V. Ravikumar, , A. Rehemtulla, Arvind Rao, W. Al-Holou
          </td>
          <td>2025-04-01</td>
          <td>Neurosurgery</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f17dd1fdac811901b5625401f98738e2abb40163" target='_blank'>
              IP6K2 mutations as a novel mechanism of resistance to oncolytic virus therapy
              </a>
            </td>
          <td>
            Zhijian Huang, Xiangqian Zhao, Zirong Jiang, Xiaoting Qiu, Xinhao Sun, Dawei Wang, Hucheng Zhang, Qi Chen, Ruirong Tan, Yangkun Shen
          </td>
          <td>2025-03-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="DNA decoys are expected to be among the nucleic acid drugs used to downregulate the transcription process. However, spatially controlling the on/off efficacy of DNA decoys to avoid side effects on normal cells is challenging. To reduce undesired decoy function in normal cells, we adopted catalytic hairpin assembly (CHA) to produce a DNA duplex from a hairpin DNA pair in response to a specific microRNA (miRNA). We designed the DNA hairpin pairs to form a DNA decoy that binds to NF-kB, whose overexpression is related to many diseases, including cancer. The transformation of the DNA hairpin pair to the NF-kB DNA decoy was catalyzed by miR-21. Intracellular CHA progression and the inhibitory effect against NF-kB were observed only in miR-21 overexpressing cancer cells. The intracellular miR-21-catalyzed production of the NF-kB DNA decoy has the potential to reduce side effects on normal cells, thereby strengthening the therapeutic profile of the CHA-decoy system. This study is the first to use the CHA product itself as a selective therapeutic substance. The ability to customize the combination of catalytic miRNA and target transcription factors would allow our technology to serve as a "personalized drug discovery system" for a variety of challenging diseases, including cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a859188c2bcbbbd269e7e181ed7e2f2850df16d" target='_blank'>
              Amplified Production of a DNA Decoy Catalyzed by Intracellular MicroRNA.
              </a>
            </td>
          <td>
            Soshu Yasuda, K. Morihiro, Shuichiro Koga, Akimitsu Okamoto
          </td>
          <td>2025-02-03</td>
          <td>Angewandte Chemie</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Summary Recently formed allopolyploid species offer unprecedented insights into the early stages of polyploid evolution. This review examines seven well‐studied neopolyploids (we use ‘neopolyploid’ to refer to very recently formed polyploids, i.e. during the past 300 years), spanning different angiosperm families, exploring commonalities and differences in their evolutionary trajectories. Each neopolyploid provides a unique case study, demonstrating both shared patterns, such as rapid genomic and phenotypic changes, and unique responses to hybridization and genome doubling. While previous studies of these neopolyploids have improved our understanding of polyploidy, significant knowledge gaps remain, highlighting the need for further research into the varied impacts of whole‐genome duplication on gene expression, epigenetic modifications, and ecological interactions. Notably, all of these neopolyploids have spontaneously arisen due to human activity in natural environments, underscoring the profound consequences of polyploidization in a rapidly changing world. Understanding the immediate effects of polyploidy is crucial not only for evolutionary biology but also for applied practices, as polyploidy can lead to novel traits, as well as stress tolerance and increased crop yields. Future research directions include investigating the genetic and epigenetic mechanisms underlying polyploid evolution, as well as exploring the potential of neopolyploids for crop improvement and environmental adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac5a204f000a3e9de1ebd9bce03b0ead5230c52b" target='_blank'>
              Natural neopolyploids: a stimulus for novel research
              </a>
            </td>
          <td>
            P. Edger, D. Soltis, Shunsuke Yoshioka, Mario Vallejo-Marín, Rie Shimizu‐Inatsugi, Kentaro K. Shimizu, A. Salmon, Simon Hiscock, Malika Ainouche, Pamela S Soltis
          </td>
          <td>2025-02-14</td>
          <td>The New Phytologist</td>
          <td>0</td>
          <td>142</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ed27bb6383161df0bd1281e88568674842389e" target='_blank'>
              Transposition element MERVL regulates DNA demethylation through TET3 in oxidative-damaged mouse preimplantation embryos
              </a>
            </td>
          <td>
            Lihong Liu, Siyao Ha, Dan Cao, MingQing Li, Zhiling Li
          </td>
          <td>2025-03-12</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16718d2277420c09078544ae1939c2c13bd37fd2" target='_blank'>
              Exome sequencing shows same pattern of clonal tumor mutational burden, intratumor heterogenicity and clonal neoantigen between autologous tumor and Vigil product.
              </a>
            </td>
          <td>
            David Willoughby, E. Bognar, L. Stanbery, Casey Nagel, G. Wallraven, Aman Pruthi, Nicholas Bild, Ericca Stamper, Donald Rao, Adam Walter, John Nemunaitis
          </td>
          <td>2025-03-13</td>
          <td>Scientific reports</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Identifying genes involved in cancer is crucial for understanding the underlying molecular mechanisms of the disease and developing effective treatment strategies. Differential expression analysis (DEA) is the predominant method used to identify cancer-related genes. This approach involves comparing gene expression levels between different samples, such as cancerous and non-cancerous tissues, to identify genes that are significantly upregulated or downregulated in cancer. DEA is based on the commonly believed assumption that genes upregulated in cancerous tissues have the potential to function as oncogenes. Their expression levels often correlate with cancer advancement and unfavorable prognosis, whereas downregulated genes display the opposite correlation. However, contrary to the prevailing belief, our analysis utilizing The Cancer Genome Atlas (TCGA) databases revealed that the upregulated or downregulated genes in cancer do not always align with cancer progression or prognosis. These findings emphasize the need for alternative approaches for identifying cancer-related genes that may be more accurate and effective. To address this need, we compared the effectiveness of machine learning (ML) methods with that of traditional DEA in the identification of cancer-related genes. ML algorithms have the advantage of being able to analyze large-scale genomic data and identify complex patterns that may go unnoticed by traditional methods. Our results demonstrated that ML methods significantly outperformed DEA in the screening of cancer-related genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6167fa4f27157ad0473050eb8b2f5d919ef194a8" target='_blank'>
              Challenging Conventional Perceptions of Oncogenes and Tumor Suppressor Genes: A Comprehensive Analysis of Gene Expression Patterns in Cancer.
              </a>
            </td>
          <td>
            Mingyuan Zou, Li Qiu, Wentao Wu, Hui Liu, Han Xiao, Jun Liu
          </td>
          <td>2025-02-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
Genomic and transcriptomic sequencing technologies have revolutionized our ability to characterize prostate cancer at the molecular level. The underlying premise of next-generation sequencing technologies and their current and evolving applications in prostate cancer management are provided in the review.


RECENT FINDINGS
Improved methodologies are allowing timely sequencing of the coding regions or both the coding and noncoding regions of the genome to help identify potential mutations and structural variations in the prostate cancer genome, some of which are currently also targetable therapeutically. DNA microarray- based differential gene expression has been supplanted by RNA sequencing (RNA-seq), which not only allows for more accurate quantitation but also nucleotide-level resolution to investigate the entire transcriptome, including alternative gene spliced transcripts and noncoding RNA transcripts, whose full clinical implications have yet to be fully understood and realized. Gene classifier platforms that predict risk of recurrence or metastasis are being incorporated into prostate cancer management algorithms. In the appropriate clinical context, not only somatic but also germline mutation testing is being recommended.


SUMMARY
Continued clinical integration of sequencing technologies and ongoing research will lead to improved understanding of prostate cancer biology and prostate cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e945cd17bb6878accfa941b57523d14ed0d229c" target='_blank'>
              Genomic and transcriptomic sequencing in prostate cancer.
              </a>
            </td>
          <td>
            Safiullah Rifai, Azimullah Rifai, Xiaolei Shi, M. A. Khan, Wei Guang, Linbo Wang, Luke Tallon, Arif Hussain
          </td>
          <td>2025-02-28</td>
          <td>Current opinion in oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="DNA damage is a common event in cells, resulting from both internal and external factors. The maintenance of genomic integrity is vital for cellular function and physiological processes. The inadequate repair of DNA damage results in the genomic instability, which has been associated with the development and progression of various human diseases. Accumulation of DNA damage can lead to multiple diseases, such as neurodegenerative disorders, cancers, immune deficiencies, infertility and aging. This comprehensive review delves the impact of alterations in DNA damage response genes (DDR) and tries to elucidate how and to what extent the same traits modulate diverse major human diseases, such as cancer, neurodegenerative diseases, and immunological disorders. DDR is apparently the trait connecting important complex disorders in humans. However, the pathogenesis of the above disorders and diseases are different and leading to the divergent consequences. It is important to discover the switch(es) that direct further the pathogenic process either to proliferative, or degenerative diseases. Our understanding the influence of DNA damage on diverse human disorders may enable a development of the strategies to prevent, diagnose, and treat these diseases. In our article, we analysed publicly available GWAS summary statistics from the NHGRI-EBI GWAS Catalog and identified 12,009 single nucleotide polymorphisms (SNPs) associated with cancer. Among these, 119 SNPs were found in DDR pathways, exhibiting significant p-values. Additionally, we identified 44 SNPs linked to various cancer types and neurodegenerative diseases (NDDs), including four located in DDR-related genes: ATM, CUX2, and WNT3. Furthermore, 402 SNPs were associated with both cancer and immunological disorders, with two found in DDR gene RAD51B. This highlights the versatility of the DDR pathway in multifactorial diseases. However, the specific mechanisms that regulate DDR to initiate distinct pathogenic processes remain to be elucidated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9bdcdd5d823de10dac2aaf9306d09b471c64a04" target='_blank'>
              Defective DNA Repair: A Putative Nexus Linking Immunological Diseases, Neurodegenerative Disorders, and Cancer.
              </a>
            </td>
          <td>
            Safaa Andarawi, L. Vodickova, Anusha Uttarilli, Petr Hanak, P. Vodicka
          </td>
          <td>2025-02-12</td>
          <td>Mutagenesis</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Chromosome aberrations (CAs) play an important role in tumor initiation, followed by metastasis. Short-term peripheral blood lymphocyte (PBL) cultures
using RPMI-1640 media with 5 % FBS were used to evaluate the frequency of structural and numerical chromosome aberrations in breast cancer patients.
Highest frequency of aneuploidy (11.53%) were observed followed by trisomy -21(8.93%) in karyotypes. De-novo mutation involving 12q21 with loss
of 16.38 Mbp DNA fragment is the most relevant finding in breast cancer (BC) patients and has not been reported earlier. Secondly, role of trisomy-21
(8.93%) might have increase genetic susceptibility of disease because of “giant satellites”, are the active sites of rRNA sub unit of 18S and 28S. These active
sites might have increase gene-expression of truncated protein during tumor Significant increase in the frequency of aneuploidy with increased number
of non-homologous chromosome was the striking feature due to increase of non-disjunction event followed by unequal crossing-over and synapse
formation by known environmental factors like arsenic. However, the present study is small, but interesting to explore the etiopathology of BC patients
associated chromosome instability. However, further study is required to confirm these changes, whether these mutations are either familial (inherited)
or spontaneous in nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f684d4b189095eeab5e425ab3b1be62a314e19" target='_blank'>
              Genomic Instability with Non- Homologues Chromosomes in Aneuploid Cell Increase Risk Factor for the Development of Breast Cancer Patients
              </a>
            </td>
          <td>
            A. K. Saxena
          </td>
          <td>2025-01-31</td>
          <td>Journal of Medicine and Healthcare</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22c8fd6c1c580a3737a18f3485f8040f1c52d76c" target='_blank'>
              TRIM28-dependent developmental heterogeneity determines cancer susceptibility through distinct epigenetic states
              </a>
            </td>
          <td>
            Ilaria Panzeri, Luca Fagnocchi, S. Apostle, Megan Tompkins, Emily Wolfrum, Z. Madaj, G. Hostetter, Yanqing Liu, Kristen Schaefer, Chih-Hsiang Yang, Alexis Bergsma, Anne Drougard, Erez Dror, Darrell P Chandler, Daniel Schramek, Timothy J. Triche, J. Pospisilik
          </td>
          <td>2025-01-24</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Anaplastic lymphoma kinase (ALK) fusion tyrosine kinases (TKs) are commonly found in various cancers and are considered as promising targets for therapy due to their intricate biological processes. However, the reasons for the higher frequency of ALKs fusion compared to other TKs are not well elucidated. Physicochemical properties, secondary and tertiary structures, and phylogenetic trees, along with protein sequence alignments of receptor tyrosine kinases (RTKs) and ALK fused partner genes, were examined using the resources provided by the National Center for Biotechnology Information (NCBI) and the Catalogue of Somatic Mutations in Cancer (COSMIC). Sequence alignments were performed to identify common sequences between partner genes and search for common breakpoints within the COSMIC database. ALK is a large, unstable, acidic protein with similarly conservation among RTKs. ALK fusion partners are mostly acidic, unstable proteins, mostly consisting of α-helices and random coil. However, EML4 and NPM1 are the most frequently occurring partner genes and have their own unique structural characteristics. By functional domain analysis, we found that the functions of the first half of the ALK partner gene (the part fused to ALK) are mostly focused on signaling. ALK is identified as a large hydrophilic protein，exhibits a higher proportion of random coils. Compared to other RTKs, ALK has fewer structural domains (PTKC_ALK_LTK domain). Pairwise comparison with fusion partner genes revealed a conserved sequence predicted to have structural stability and act as a common binding site for nucleases. Exon 20 of ALK is a fusion frequent site according to COSMIC database analysis. The structural instability of ALK and partner genes, coupled with the inherent variability of breakpoint sequences, leads to the formation of potent kinase-activated oncogenes, which play a critical role in tumorigenesis. While the occurrence of ALK fusions with partner genes is random, specific combinations lead to the generation of oncogenes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d901b9b33dfa1544156c22b0678e85042f152af" target='_blank'>
              Genomic landscaping of receptor tyrosine kinase ALK with highly frequent rearrangements in cancers.
              </a>
            </td>
          <td>
            Wei Ye, Wen-Bin Ou
          </td>
          <td>2025-02-01</td>
          <td>IUBMB life</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddb80961cd695209a157b69a05f62bd12ae66e96" target='_blank'>
              Role of circular RNAs in cancer therapy resistance
              </a>
            </td>
          <td>
            Wenjuan Liu, Jiling Niu, Yanfei Huo, Long Zhang, Linyu Han, Nasha Zhang, Ming Yang
          </td>
          <td>2025-02-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="
 DNA double strand breaks (DSBs) result in activation of several key DNA damage response (DDR) kinases including ATM, ATR, and DNA-PK. These protein kinases not only promote DNA damage-induced checkpoint control, but also facilitate DSB repair in humans. Thus, these DDR kinases have become promising drug targets for cancer therapy. However, the benefits of targeting DDR kinases remain to be realized, in part due to the lack of predictive biomarkers. By undertaking CRISPR screens with inhibitors targeting key DDR kinases, we obtained a global and unbiased view of genetic interactions with DDR inhibition. Additionally, we compared the synergistic effects of combining different DDR inhibitors and found that ATM and PARP inhibitors showed remarkable synergy, which depends on the dominant negative function of ATM inhibition. Moreover, to provide comprehensive and unbiased perspective of DDR signaling pathways, we performed 30 fluorescence-activated cell sorting–based genome-wide CRISPR screens with antibodies recognizing distinct endogenous DNA damage–signaling proteins to identify new regulators involved in DNA damage response (DDR). We discovered that proteasome-mediated processing is an early and prerequisite event for cells to trigger camptothecin- and etoposide-induced DDR signaling. Furthermore, we identified PRMT1 and PRMT5 as new modulators that regulate ATM protein level. Additionally, we discovered that GNB1L is a master regulator of DDR signaling via its role as a co-chaperone for PIKK proteins. Collectively, these screens offer a rich resource for further investigation of DDR, which may provide insight into strategies of targeting these DDR pathways to improve therapeutic outcomes. More recently, we have adopted dTAG technology for the investigation of many essential DDR and replication proteins, which provide mechanistic insights into the roles of these essential proteins in genome maintenance. Additionally, we have expanded FACS-based screens for the studies of other cancer-associated pathways and explored key biological processes critical for cancer progression in vivo. These studies will facilitate the development of better therapies for cancer patients.
 Citation Format: Junjie Chen. Targeting DNA damage responsive pathways in cancer therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr B008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb7435e798afc5c276c5bcf5f266a5cdfa975999" target='_blank'>
              Abstract B008: Targeting DNA damage responsive pathways in cancer therapy
              </a>
            </td>
          <td>
            Junjie Chen
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Base of tongue squamous cell carcinoma (BOTSCC) is a prevalent and aggressive form of oral cancer, often associated with poor patient outcomes. The tumor microenvironment (TME) of HPV-positive BOTSCC is critical in influencing cancer progression and treatment response. Objective This study aims to analyze the TME of HPV-positive BOTSCC by examining the expression of key genes involved in various biological processes. Methods We utilized the RT2 Profiler PCR Array to quantify the expression of 168 genes related to inflammation, immunity, oncogenesis, tumor suppression, apoptosis, and angiogenesis. Enrichment analysis of cancer hallmarks was performed on all upregulated genes. Additionally, we investigated the correlation between the expression levels of the ten most highly upregulated genes and survival prognosis in HPV-associated BOTSCC patients. Results Our analysis revealed dysregulation of 42 genes associated with tumor-immune interactions, with 20 genes upregulated and 22 downregulated. Furthermore, we identified 64 genes linked to cancer development, with 33 upregulated and 31 downregulated. High-risk HPV (hr-HPV) genotypes were found in 81% of patients, predominantly HPV-35 and HPV-16. Conclusion This study highlights the complexity of the HPV-positive BOTSCC TME, underscoring the need for further research into molecular pathways and immune interactions to identify new therapeutic targets for improved cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edc25ed1594bab521984d64540a18029b5d937fb" target='_blank'>
              Profiling the Tumor Immune Microenvironment of HPV-Associated Base of Tongue Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Reham M Alahmadi, M. Awadalla, N. Marraiki, Mohammed Alswayyed, Hajar Alshehri, Amjad Alsahli, H. Khoja, Osamah Khojah, Rawan M Alahmadi, Nada Farid, B. Alosaimi
          </td>
          <td>2025-02-01</td>
          <td>OncoTargets and Therapy</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Fanconi Anemia (FA) is a heritable syndrome characterized by DNA damage repair deficits, frequent malformations and a significantly elevated risk of bone marrow failure, leukemia, and mucosal head and neck squamous cell carcinomas (HNSCC). Hematopoietic stem cell gene therapy can prevent marrow failure and lower leukemia risk, but mucosal gene therapy to lower HNSCC risk remains untested. Major knowledge gaps include an incomplete understanding of how rapidly gene-corrected cellular lineages could spread through the oral epithelium, and which delivery parameters are critical for ensuring efficient gene correction. To answer these questions, we extended an agent-based model of the oral epithelium to include the delivery of gene correction in situ to FA cells and the competitive dynamics between cellular lineages with and without gene correction. We found that only gene-corrected lineages with substantial proliferative advantages (probability of resisting displacement out of the basal layer ≥ 0. 1) could spread on clinically relevant timelines, and that these lineages were initially at high risk of loss in the generations following correction. Delivering gene correction to many cells minimizes the risk of loss, while delivery to many distinct locations within a tissue maximizes the rate of spread. To determine the impact of mucosal gene therapy in preventing the clonal expansion of pre-cancerous mutations, we compared the expected burden of TP53 mutations in simulated tissue sections with and without gene correction. We found that when FA cells have elevated genome instability or a TP53-dependent proliferative advantage, gene correction can substantially reduce the accumulation of pro-tumorigenic mutations. This model illustrates the power of computational frameworks to identify critical determinants of therapeutic success to enable experimental optimization and support novel and effective gene therapy applications. Author summary We investigated factors influencing the success of oral mucosal gene therapy for Fanconi Anemia (FA), a genetic syndrome marked by DNA repair defects in conjunction with a heightened risk of cancer. We used a computational model of the oral epithelium to determine how gene therapy corrected cells compete with FA background cells and the best gene delivery approaches to promote effective tissue replacement by gene-corrected cells. We find that gene-corrected cells require strong proliferative advantages to spread effectively, and that initially delivering gene correction to more cells reduces the chance that these cells are stochastically eliminated before they can spread. We also demonstrate that gene correction reduces the accumulation of pro-tumorigenic TP53 mutations in an FA context, where genomic instability can elevate the mutation rate and FA-specific selective pressures could favor accelerated TP53 clonal expansion. This research provides a useful framework for guiding mucosal gene therapy experiments and the development of effective oral gene therapy protocols for cancer prevention in FA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b88eb3b56fe96b81aec45dd517d7b11cc8ca9c35" target='_blank'>
              Epithelial competition determines gene therapy potential to suppress Fanconi Anemia oral cancer risk
              </a>
            </td>
          <td>
            Hunter L. Colegrove, Raymond J Monnat, Alison F Feder
          </td>
          <td>2025-02-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer genomic studies have identified frequent alterations in genes encoding components of the SWI/SNF chromatin remodeling complex, including SMARCA4 and ARID1A. Importantly, clinical reports indicate that SMARCA4-mutant lung cancers respond poorly to immunotherapy and have dismal prognosis. Here, we corroborated the clinical findings by using immune-humanized, syngeneic, and genetically engineered mouse models of lung cancer harboring SMARCA4 deficiency. Specifically, models with SMARCA4 loss showed decreased response to anti-PD1 immunotherapy associated with significantly reduced infiltration of dendritic cells and CD4+ T cells into the tumor microenvironment. SMARCA4 loss in tumor cells led to profound downregulation of STING1, IL1B, and other components of the innate immune system, as well as inflammatory cytokines that are required for efficient recruitment and activity of immune cells. The deregulation of gene expression was caused by cancer cell-intrinsic reprogramming of the enhancer landscape with marked loss of chromatin accessibility at enhancers of genes involved in innate immune response, such as STING1, IL1B, type I interferon, and inflammatory cytokines. Interestingly, the transcription factor NF-κB binding motif was enriched in enhancers that lose accessibility upon SMARCA4 deficiency. Furthermore, SMARCA4 and NF-κB co-occupied the same genomic loci on enhancers associated with STING1 and IFNB1, indicating a functional interplay between SMARCA4 and NF-κB. Taken together, these findings provide the mechanistic basis for the poor response of SMARCA4-mutant tumors to immunotherapy and establish a functional link between SMARCA4 and NF-κB in innate immune and inflammatory gene expression regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1178b1ac2d65f9262b22b808cb25c2ac38fa27ab" target='_blank'>
              Mutation of SMARCA4 Induces Cancer Cell-Intrinsic Defects in the Enhancer Landscape and Resistance to Immunotherapy.
              </a>
            </td>
          <td>
            Yawen Wang, Ismail M. Meraz, Md Qudratullah, Sasi Kotagiri, Yanyan Han, Yuanxin Xi, Jing Wang, Kadir C Akdemir, Jack A Roth, Yonathan Lissanu
          </td>
          <td>2025-03-13</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The regulatory circuitry of cell-specific transcriptional programs is thought to be influenced by transposable elements (TEs), whereby TEs serve as raw material for the diversification and genome-wide distribution of genetic elements that contain cis-regulatory activity. However, the transcriptional activators of TEs in relevant physiological contexts are largely unknown. Here, we undertook an evolutionary approach to identify regulators of two main families of MERVL, a major regulator of transcription during early mouse development. Using a combination of phyloregulatory, transcriptomic, and loss-of-function approaches, we demonstrate that SRF is a novel regulator of MERVL and embryonic transcription during zygotic genome activation. By resolving the phylogenetic history of two major MERVL families, we delineate the evolutionary acquisition of SRF and DUX binding sites and show that the acquisition of the SRF site precedes that of DUX. SRF contributes to embryonic transcription through the regulation of MERVLs, which in turn serve as promoters for host genes. Our work identifies new transcriptional regulators and TEs that shape the gene expression programs in early embryos and highlights the process of TE domestication via the sequential acquisition of transcription factor binding sites and coevolution with the host.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c19557776012bc520b8b5ecf4614f5b4aaa6f641" target='_blank'>
              The transcription factor SRF regulates MERVL retrotransposons and gene expression during zygotic genome activation.
              </a>
            </td>
          <td>
            C. Hermant, Carlos Michel Mourra-Díaz, Marlies E. Oomen, Luis Altamirano-Pacheco, Mrinmoy Pal, Tsunetoshi Nakatani, M. Torres-Padilla
          </td>
          <td>2025-02-27</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Cancer cells exhibit high heterogeneity and lineage plasticity, complicating studies of tumorigenesis and development of therapies. Recently, preneoplastic cells, although histologically normal, have been shown to possess high plasticity and early genetic alterations, yet their origins and lineage trajectories remain unclear. Herein, we introduce a lineage-tracing tool integrating genetic barcoding with single-cell RNA sequencing to map preneoplastic esophageal cell lineages. We identified preneoplastic precursor cells (PNPCs) as a distinct progenitor-like population with unique transcriptional profiles and high plasticity, contributing to proliferative and basal cell populations. To enhance lineage mapping, we developed the eXamined Ridge (XR) score, accurately identifying high-plasticity cells. Nfib and Qk emerged as conserved PNPC markers, peaking in early preneoplasia and declining after malignant transformation. These findings reveal PNPCs as key players in early tumorigenesis and highlight their potential as biomarkers for early cancer detection and therapeutic intervention, offering new strategies for preventing esophageal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37dcd14a4aabf910a9a90379f13bfe9df5a44b29" target='_blank'>
              Deciphering Precursor Cell Dynamics in Esophageal Preneoplasia via Genetic Barcoding and Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Jinho Jang, Kyung-Pil Ko, Jie Zhang, Sohee Jun, Jae-Ll Park
          </td>
          <td>2025-02-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are unique nucleic acid structures composed of guanine-rich (G-rich) sequences that can form diverse topologies based on the arrangement of their four strands. G4s have attracted attention for their potential roles in various biological processes and human diseases. In this review, we focus on the G4 structures formed by human telomeric sequences, (GGGTTA)n, and the hexanucleotide repeat expansion, (GGGGCC)n, in the first intron region of the chromosome 9 open reading frame 72 (C9orf72) gene, highlighting their structural diversity and biological significance. Human telomeric G4s play crucial roles in telomere retention and gene regulation. In particular, we provide an in-depth summary of known telomeric G4s and focus on our recently discovered chair-type conformation, which exhibits distinct folding patterns. The chair-type G4s represent a novel folding pattern with unique characteristics, expanding our knowledge of telomeric G4 structural diversity and potential biological functions. Specifically, we emphasize the G4s formed by the (GGGGCC)n sequence of the C9orf72 gene, which represents the most common genetic cause of amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD). The thorough structural analysis in this review advances our comprehension of the disease mechanism and provides valuable insights into developing targeted therapeutic strategies in ALS/FTD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4afb7326b8980cb609e9c6033f3c9e1b8d005a43" target='_blank'>
              G-Quadruplex Structures Formed by Human Telomere and C9orf72 GGGGCC Repeats
              </a>
            </td>
          <td>
            Bing Yan, M. C. Suen, Naining Xu, Chao Lu, Changdong Liu, Guang Zhu
          </td>
          <td>2025-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="During zygotic genome activation (ZGA) and early development, hierarchical levels of chromatin structure undergo remarkable perturbations: changes in the nuclear-to-cytoplasmic ratio of various components; changes in chromatin accessibility; histone exchange; and the formation of 3D structures such as loops, topologically associated domains, and compartments. Here, we review the peculiarities, variability, and emergence of the chromatin structural features during ZGA in different organisms. Focusing on newly found structures called fountains, we describe the prerequisites for cohesin loading on DNA and possible mechanisms of genome organization in early development. Fountains resulting from asymmetric bidirectional cohesin extrusion spread from cohesin-loading points in a CTCF-independent manner. We discuss that fountains may not possess specific functions, unlike conventional chromatin structures, and could be found in other biological processes where cohesin loading occurs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e35698f05d68e24632b02180e50c1bfb7426f2b2" target='_blank'>
              The establishment of the 3D genome structure during zygotic genome activation.
              </a>
            </td>
          <td>
            Anastasiia L Sivkina, O. Iarovaia, S. V. Razin, S. Ulianov
          </td>
          <td>2025-03-03</td>
          <td>Annals of the New York Academy of Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a1ce69e97a9adaccd0f62087c29dd1f173f2a27" target='_blank'>
              Advances in epigenetic treatment of adult T-cell leukemia/lymphoma: a comprehensive review
              </a>
            </td>
          <td>
            Arash Letafati, Rabeeh Mehdigholian Chaijani, Fahimeh Edalat, Nazila Eslami, Hanieh Askari, Farideh Askari, Sara Shirvani, Hamed Talebzadeh, Mahdiyeh Tarahomi, Nila MirKhani, Faeze Karimi, Mehdi Norouzi, Sayed-Hamidreza Mozhgani
          </td>
          <td>2025-03-01</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The DNA double-stranded genome of human adenoviruses (HAdV) is preferentially targeted by host factors involved in chromatin remodeling, such as SPOC1 and KAP1 to inhibit efficient viral gene expression. HAdV genomes undergo alterations through association with host histones and epigenetic modifications; however, the precise underlying mechanism remains elusive. A recently discovered silencing mechanism for retrotransposons and retroviruses involves the Human Silencing Hub (HUSH) complex. This complex of MPP8, TASOR, and PPHLN1 safeguards the human genome by utilizing histone H3 Lys9 trimethylation (H3K9me3) to block transcription. Through the recruitment of SETDB1 and MORC2, the HUSH complex silences host genes and condenses target genomes to combat infections such as HIV, MLV, and AAV. Here, we present evidence that the HUSH complex effectively restricts HAdV infection. To counteract the repressive function of this epigenetic silencing complex, HUSH factors are inhibited through binding of HAdV proteins and subsequent relocalization. We observe that MPP8 is targeted by the adenoviral E3 ubiquitin ligase, thus recruited by the viral early proteins E1B-55K and E4orf6 for proteasomal degradation. In summary, we provide evidence that the HUSH complex is a previously unrecognized host factor that restricts HAdV gene expression and replication. Based on these novel findings, we propose that HUSH represents a promising therapeutic target to combat HAdV infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26334a481a2a25fb7be72ec2c09273c472c1ce03" target='_blank'>
              Intrinsic immunity against HAdV is achieved by a novel epigenetic silencing complex
              </a>
            </td>
          <td>
            Julia Mai, Maryam Karimi, A. Kuderna, Anna Heider, Thomas Günther, C. Friedel, A. Grundhoff, Thomas Stamminger, Sabrina Schreiner
          </td>
          <td>2025-02-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="
 Rationale: Leukemic evolution occurs through gradual clonal selection at early and late stages of disease development. The early stage of this evolutionary process is called clonal hematopoiesis (CH) and is typified by expansion, and increased self-renewal capacity, of non-malignant, somatically mutated hematopoietic stem and progenitor cells (HSPCs). CH-mutant genes result in either genomic hypomethylation through loss of DNMT3A, or hypermethylation either by genetic loss of TET2 or its’ inhibition by the oncometabolite 2-HG generated by mutant IDH1/IDH2. Leukemic transformation is often associated with subsequent mutagenesis in a broader range of genes including mitogenic signaling genes that confer increased proliferative capacity (FLT3, NRAS). The stepwise acquisition of mutations generates a polyclonal “ecosystem” of clones with increasing clonal diversity indicative of disease progression and poor prognosis. The goal of this study is to 1) identify genetic evolutionary trends associated with distinct CH-related mutations in DNMT3A and TET2 and 2) implement innovative single cell DNA sequencing protocols to identify genotype-selective DNA methylation alterations at single cell resolution. Approach: We compiled patients diagnosed with de novo AML at the University of Pennsylvania that had multiple genomic profiling studies, a cohort comprising 182 patients. All patients were profiled at diagnosis, with n=119 at first remission and n=76 at first relapse. 84.6% of patients received induction chemotherapy with combination anthracycline and nucleoside analog therapy. To investigate the genomic changes in AML disease course, we calculated the change in variant allele frequencies (VAFs) for all identified variants, evaluating matched samples from individual patients across time. We infer phylogenetic relationships from matched genomic profiles across timepoints in individual patients using CALDER. Finally, we performed single cell DNA methylation sequencing on 3 patients (n=4000 cells) using the Single Cell Methylation Kit from Scale Biosciences. This is a combinatorial indexing-based whole genome bisulfite sequencing based approach. Results: Comparing VAFs between diagnosis and relapse for DNMT3Amut vs TET2mut AML, we observed that FLT3 and NPM1 variants were similarly split between increased and decreased VAFs at relapse vs diagnosis. NRAS mutations were both gained and lost in DNMT3Amut samples between diagnosis and relapse, but no NRAS mutations were identified at all in TET2mut samples. When inferring phylogeny associations, we found DNMT3Amut samples were more likely to undergo subclonal swapping from diagnosis to relapse, whereas TET2mut samples relapsed with a more stable mutation profile. Analyses are currently underway for single cell DNA methylation data. Conclusions: Our analyses suggest that DNMT3A and TET2 have distinct sets of co-occurring genetic alterations at diagnosis and relapse. Early CH-associated mutations preceding transformation can shape the evolutionary trajectories of AML throughout disease development.
 Citation Format: Nisargbhai Shah, Ryan Cho, Priya Velu, Selina Luger, Martin Carroll, Jennifer Morrissette, Michael Bowman, Robert Bowman. Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: DNA Methylation, Clonal Hematopoiesis, and Cancer; 2025 Feb 1-4; San Diego, CA. Philadelphia (PA): AACR; Cancer Res 2025;85(3 Suppl):Abstract nr A021.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da48ac3143bd55d667bb07036faa9f708e803fd9" target='_blank'>
              Abstract A021: Evolutionary trajectories and single cell methylation analysis of DNMT3A and TET2 mutant acute myeloid leukemia
              </a>
            </td>
          <td>
            Nisargbhai Shah, Ryan D Chow, Priya Velu, Selina M Luger, Martin P Carroll, Jennifer J D Morrissette, Michael Bowman, Robert L Bowman
          </td>
          <td>2025-02-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adb0953d261751e4de7ab51c018f54f11e89daf6" target='_blank'>
              p53-loss induced prostatic epithelial cell plasticity and invasion is driven by a crosstalk with the tumor microenvironment
              </a>
            </td>
          <td>
            Darya Yanushko, Beatriz German Falcon, Rana El Bizri, Despoina Pervizou, Robin Dolgos, Céline Keime, Tao Ye, C. Thibault-Carpentier, Clémentine Le Magnen, S. Henri, Gilles Laverny, Daniel Metzger
          </td>
          <td>2025-01-26</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="While the mutational landscape across early T-cell precursor acute lymphoblastic leukemia (ETP-ALL) and ETP-like leukemia is known, establishing a unified framework that activates stem cell genes characteristic of these tumors remains elusive. Using complementary mouse and human models, chromatin mapping, and enhancer profiling, we show that the coactivator ZMIZ1 promotes normal and malignant ETP population growth by inducing the transcription factor MYB in feedforward circuits to convergently activate oncogenes (MEF2C, MYCN, and BCL2) through essential enhancers. A key superenhancer, the N-Myc regulating enhancer (NMRE), drives malignant ETP population growth but is dispensable for normal lymphopoiesis. This network of stem cell superenhancers identifies treatment-resistant tumors and poor survival outcomes; unifies diverse ETP-ALLs; and contributes to cardinal features of the recently genomically identified high-risk bone marrow progenitor-like (BMP-like) ETP-ALL tumor-stem cell/myeloid gene expression, inhibited NOTCH1-induced T-cell development, aggressive clinical behavior, and venetoclax sensitivity. Since ZMIZ1 is dispensable for essential homeostasis, it might be possible to safely target this network to treat high-risk diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c779b27de9c7947d2736cb472ee433351114fe43" target='_blank'>
              Native stem cell transcriptional circuits define cardinal features of high-risk leukemia.
              </a>
            </td>
          <td>
            Qing Wang, Francesco Boccalatte, Jason Xu, Giovanni Gambi, Bettina Nadorp, Fatema Akter, Carea Mullin, A. Melnick, Elizabeth Choe, Anna C. McCarter, Nicole A Jerome, Siyi Chen, Karena Lin, Sarah Khan, R. Kodgule, Jonathan H. Sussman, P. Pölönen, Javier Rodriguez-Hernaez, Sonali Narang, Kleopatra Avrampou, B. King, A. Tsirigos, Russell J H Ryan, C. Mullighan, D. Teachey, Kai Tan, I. Aifantis, M. Chiang
          </td>
          <td>2025-02-19</td>
          <td>The Journal of experimental medicine</td>
          <td>0</td>
          <td>103</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae55557a79f951a485decd6834444eb8baa9c9b2" target='_blank'>
              Recent updates of centromere proteins in hepatocellular carcinoma: a review
              </a>
            </td>
          <td>
            Zhongyuan Yang, Wenjiao Chen, Yunhui Liu, Yuxin Niu
          </td>
          <td>2025-02-06</td>
          <td>Infectious Agents and Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52a090853d43190aed49536f43b138c297c3e652" target='_blank'>
              Telomerase in cancer- ongoing quest and future discoveries.
              </a>
            </td>
          <td>
            Apurwa Mishra, Trupti N Patel
          </td>
          <td>2025-01-25</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer (BC) is the most prevalent malignancy among women, characterized by extensive heterogeneity stemming from molecular and genetic alterations. This review explores the intricate epigenetic landscape of BC, highlighting the significant role of epigenetic modifications—particularly DNA methylation, histone modifications, and the influence of non-coding RNAs—in the initiation, progression, and prognosis of the disease. Epigenetic alterations drive crucial processes, including gene expression regulation, cell differentiation, and tumor microenvironment interactions, contributing to tumorigenesis and metastatic potential. Notably, aberrations in DNA methylation patterns, including global hypomethylation and hypermethylation of CpG islands, have been associated with distinct BC subtypes, with implications for early detection and risk assessment. Furthermore, histone modifications, such as acetylation and methylation, affect cancer cell plasticity and aggressiveness by profoundly influencing chromatin dynamics and gene transcription. Finally, non-coding RNAs contribute by modulating epigenetic machinery and gene expression. Despite advances in our knowledge, clinical application of epigenetic therapies in BC is still challenging, often yielding limited efficacy when used alone. However, combining epi-drugs with established treatments shows promise for enhancing therapeutic outcomes. This review underscores the importance of integrating epigenetic insights into personalized BC treatment strategies, emphasizing the potential of epigenetic biomarkers for improving diagnosis, prognosis, and therapeutic response in affected patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67a596476c40bed6ddb55424b08c446ae8d78faf" target='_blank'>
              Decoding the Epigenome of Breast Cancer
              </a>
            </td>
          <td>
            Elisa Cortellesi, Isabella Savini, Matteo Veneziano, Alessandra Gambacurta, M. Catani, V. Gasperi
          </td>
          <td>2025-03-13</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Recent studies have demonstrated that the production of bidirectional enhancer-derived transcripts (eRNAs) is a characteristic of an active Cis-regulatory element (CRE). Higher levels of eRNA synthesis correlate with the activation of histone modifications, a potentially valuable tool for deciphering the complexity of the gene regulatory network. To understand the changes of CREs during gonadal development in mice, we collected gonadal WT1-positive cells from the piggyBac-Wt1-mCherry-2A-EGFP (PBWt1-RG) reporter strain at E13.5, E16.5, and P0 in both sexes and conducted Cap Analysis of Gene Expression analysis (CAGE) which is capable to capture transcriptional starting site (TSS). We compared the levels of intergenic bidirectional RNA, i.e, potentially eRNA, according to sex at each stage (testis somatic cells vs ovary somatic cells at E13.5, E16.5, and P0) and stage in each sex (E13.5 vs E16.5, E16.5 vs P0, and E13.5 vs P0 in testis somatic cells or ovary somatic cells). Intergenic RNAs with significant changes (|Log2FC| > 1, p < 0.05) were selected. The TSS profile of intergenic RNA changed more profoundly in testis somatic cells than in ovary somatic cells, suggesting embryonic testicular development is driven by larger changes in a transcriptional regulatory network than ovarian development. Based on the profiles of the predicted transcription factors (TFs) that would bind to the active CREs during gonadal development, the NR4A, EGR, and TCF3 families would be novel TFs to play pivotal roles in gonadal development. Identifying active CREs using eRNAs would provide a means to comprehensively understand the transcriptional regulatory system, leading to valuable insights into the gonadal development of male and female individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8877e8a45402ec1d0fc04d7b02b6aa2118971232" target='_blank'>
              Analysis of Functional Cis-regulatory Elements Reveals Novel Transcriptional Regulatory Mechanisms in Gonadal Development.
              </a>
            </td>
          <td>
            S. Kirino, Ryuichi Nakagawa, Maki Gau, K. Takasawa, Yasuhiro Murakawa, H. Kawaji, Y. Hayashizaki, Tomohiro Morio, K. Kashimada
          </td>
          <td>2025-01-20</td>
          <td>Sexual development : genetics, molecular biology, evolution, endocrinology, embryology, and pathology of sex determination and differentiation</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Melanoma is the deadliest form of skin cancer, and its treatment poses significant challenges due to its aggressive nature and resistance to conventional therapies. Long non-coding RNAs (lncRNAs) represent a new frontier in the search for suitable targets to control melanoma progression and invasiveness. Indeed, lncRNAs exploit a wide range of regulatory functions along chromatin remodeling, gene transcription, post-transcription, transduction, and post-transduction to ultimately tune multiple cellular processes. The understanding of this intricate and flexible regulatory network orchestrated by lncRNAs in pathological conditions can strategically support the rational identification of promising targets, ultimately speeding up the setup of new therapeutics to integrate the currently available approaches. Here, the most recent findings on lncRNAs involved in melanoma will be analyzed. In particular, the functional links between their mechanisms of action and some frequently underestimated features, like their different subcellular localizations, will be highlighted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f08119f9fccb5c4f5a79dd1ca1781581cce224f1" target='_blank'>
              Unravelling the Regulatory Roles of lncRNAs in Melanoma: From Mechanistic Insights to Target Selection
              </a>
            </td>
          <td>
            Beatrice Moras, C. Sissi
          </td>
          <td>2025-02-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="BRCA1 deficiency is observed in approximately 25% of triple-negative breast cancer (TNBC). BRCA1, a key player of homologous recombination (HR) repair, is also involved in stalled DNA replication fork protection and repair. Here, we investigated the sensitivity of BRCA1-deficient TNBC models to the frequently used replication chain terminator gemcitabine, which does not directly induce DNA breaks. A large fraction of BRCA1-deficient cells was sensitive to gemcitabine, in contrast to their isogenic BRCA1-proficient counterparts. Gemcitabine treated BRCA1-deficient cells accumulated massive levels of single strand DNA (ssDNA) and presented no RPA or RAD51 nuclear foci. The gemcitabine-induced accumulation of ssDNA in BRCA1-deficient cells was strongly diminished by targeting MRE11 with inhibitors and by siRNA attenuation. In contrast, treatment with the PARP1/2 inhibitor olaparib did not result in MRE11 dependent over-resection. Furthermore, a fraction of gemcitabine treated BRCA1-deficient cells that showed massive ssDNA accumulation slipped into mitosis, producing mitotic bridges and strongly stained BrdU and γH2AX micronuclei (MN). The BrdU-positive MN and DNA bridges also stained positively for cGAS. In conclusion, these data suggest that gemcitabine treatment in BRCA1-deficient TNBC exposes unprotected nascent DNA linked to replication fork reversal, which leads to MRE11 over-resection and ssDNA accumulation. Therefore, the observed hypersensitivity to gemcitabine indicates that it could be a beneficial addition to BRCA1-deficient TNBC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d16464aa7109edc218441355d65fc601eddce9b" target='_blank'>
              Replication-Poison Treatment in BRCA1-Deficient Breast Cancer Causes MRE11 Over-Resection that Induces Single-Stranded DNA Accumulation and Mitotic Catastrophe.
              </a>
            </td>
          <td>
            I. Tabet, Esin Orhan, Ermes Candiello, Lise Fenou, C. Velázquez, B. Orsetti, Geneviéve Rodier, William Jacot, C. Ribeyre, Claude Sardet, C. Theillet
          </td>
          <td>2025-01-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a39dc1a99977067c119e44f09575781707edcc62" target='_blank'>
              Interplay of aurora kinase a functional residues and Epstein-barr Nuclear Antigen 1 in Epstein-barr virus associated Gastric cancer using AGS cells
              </a>
            </td>
          <td>
            N. Varshney, Siddharth Singh, M. Kandpal, Vaishali Saini, Erle S. Roberston, Hem Chandra Jha
          </td>
          <td>2025-01-24</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 255


 Background:
 Homologous recombination repair (HRR) genes, including DNA repair genes, are increasingly recognized for impacting treatment resistance and prognosis of metastatic prostate cancer (mPC). Next-generation sequencing of circulating tumor DNA (ctDNA) enables non-invasive, longitudinal monitoring of molecular alterations in patients (pts) undergoing systemic therapies. This study explores the dynamic changes in ctDNA profiles in mPC, particularly focusing on the HRR alterations.
 Methods:
 We utilized GuardantINFORM, a clinical-genomics database containing de-identified ctDNA test results and commercial payer-claims data. Eligible pts had a diagnosis of mPC and underwent ctDNA testing pre (within 3 months before initiation) and post (within 3 months of discontinuation before initiating subsequent treatment) Androgen Receptor Pathway Inhibitors (ARPi), poly ADP-ribose polymerase inhibitors (PARPi), and taxane chemotherapy. Mann-Whitney test assessed the differences in mutations. We also examined the association between ctDNA burden and overall survival (OS).
 Results:
 From a database of 21,682 pts with mPC, 145 had ctDNA collected pre/post-ARPi, 54 pre/post PARPi and 115 pre/post taxane chemotherapy. Pre-to-post-ARPI, individual HRR alterations increased from 21 to 34, with the most common (%pre/%post) genes to be altered, including ATM (5.5/7.6), BRCA2 (3/6) and CDK12 (3/2). In the PARPi group, the most prevalent (%pre/%post) genes to be altered were ATM (28/22), BRCA2 (28/20) and BRCA1 (4/6). Although overall BRCA2 prevalence declined, 11 new unique alterations were detected post-PARPi, along with an increase in unique BRCA1 (2 to 4) and BRCA2 (16 to 21) alterations. Three of 25 patients (12%) who underwent serial testing pre- and post-PARPi exhibited BRCA reversion mutations. In the taxane group, notable (%pre/%post) HRR genes to be altered included ATM (4/3), BRCA1 (1/3), and BRCA2 (1/3). Patients with higher ctDNA burden in pre-treatment samples exhibited worse OS.
 Conclusions:
 This study reveals dynamic changes in HRR gene alterations in mPC using ctDNA profiling. We observed increased unique alterations in ATM, BRCA1, and BRCA2 post-treatment, along with BRCA reversion mutations, indicating potential treatment resistance mechanisms. These findings underscore the importance of tailoring therapeutic approaches based on HRR gene profiles in mPC.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3f25e9aede0dd0d8017cba72b4efd6a1c07f803" target='_blank'>
              Molecular evolution of HRR gene alterations in metastatic prostate cancer: A ctDNA-based study.
              </a>
            </td>
          <td>
            Chinmay T Jani, Eli Tran, Nicole Zhang, J. Tsai, Jianjie Dong, Elizabeth Pan, Justine Panian, Rana R McKay
          </td>
          <td>2025-02-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Long non-coding RNAs (lncRNAs) refer to RNA molecules that exceed 200 nucleotides in length. While lncRNAs do not possess the capacity to encode proteins, they play crucial roles in gene expression, chromatin remodeling, and protein relocation. PSMA3 antisense RNA 1 (PSMA3-AS1) is a newly discovered lncRNA located on human chromosome 14q23.1. Convincing evidence shows that it acts as a tumor-promoting factor in several forms of human cancers. Moreover, high expression of PSMA3-AS1 is linked to poor clinical and pathological features and adverse prognosis in eight types of cancer. The molecular mechanisms of PSMA3- AS1 are diverse and complex. Existing evidence demonstrates that PSMA3-AS1 is activated by two transcription factors, PAX5 and YY-1, and influences cancer cell growth, metastasis, apoptosis, drug resistance, oxidative stress, and autophagy by acting as a competing endogenous RNA, activating signaling pathways, directly interacting with RNA or proteins, as well as participating in the epithelial-mesenchymal transition process. Therefore, PSMA3-AS1 holds promise as a biomarker for cancer detection and prediction, as well as a novel therapeutic target. This review explores the expression features, biological roles, potential mechanisms, and clinical significance of PSMA3-AS1 in various human cancers and provides directions for future research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a5575613f9e7c6e034f80189343ae1eb7afbdba" target='_blank'>
              Revisiting the Role of Long Non-coding RNA PSMA3-AS1 in Human Cancers: Current Evidence and Future Directions.
              </a>
            </td>
          <td>
            Jingjie Yang, Kexing Liu, Lihan Chen, Haodong He, Tongtong Li, Li Li, Xiaolan Li, Chengfu Yuan
          </td>
          <td>2025-01-31</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Purpose The mechanisms that lead to pterygium pathogenesis are poorly understood. Ultraviolet (UV) exposure is a leading environmental risk factor. We propose that UV triggers the de-repression of transposable elements (TEs), including endogenous retroviruses (ERVs), and subsequently activate double-stranded RNA and DNA sensors such as RIG-I and cGAS. Methods In the present study, we analyzed publicly available RNA sequencing data from pterygium and healthy conjunctiva specimens. We used Ingenuity Pathway Analysis (IPA) to examine the relative expression of nucleic acid sensing pathways and related processes. We also used Telescope to identify TEs. We then repeated this analysis using RNA collected from cultured human conjunctival fibroblasts exposed over multiple days to a range of UV intensities. Results We found that pathways involved in cellular stress, nucleic acid sensing, and innate immune signaling were upregulated in both cell lines exposed to increased amounts of UV energy as well as pterygium. We also observed differential expression of TEs. Conclusions Taken together, our findings suggest a mechanistic link between environmental factors and pterygium. Understanding the molecular pathways activated by UV exposure may aid in developing therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8234f024c61b7af4ccdfb30cc2f8dd5336b81ea" target='_blank'>
              Differential expression of transposable elements and upregulation of nucleic acid sensing pathways in pterygium
              </a>
            </td>
          <td>
            Patrick C. Demkowicz, Emma Hammes, Despoina Theotoka, Jessica Chow, Mathieu F. Bakhoum
          </td>
          <td>2025-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1bc42a12825a4c11c2ddcf3582debb617c1482e" target='_blank'>
              EccDNA atlas in male mice reveals features protecting genes against transcription-induced eccDNA formation
              </a>
            </td>
          <td>
            Xue Liang, Gerard Arrey, Yating Qin, Lucía Álvarez-González, J. M. Hariprakash, Jie Ma, Sylvester Holt, Peng Han, Yonglun Luo, Hanbo Li, Aurora Ruiz-Herrera, Henriette Pilegaard, Birgitte Regenberg
          </td>
          <td>2025-02-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) poses a formidable clinical challenge due to multidrug resistance (MDR) caused by tumor heterogeneity. To elucidate the intricate mechanisms underlying HGSOC heterogeneity, we conducted a comprehensive analysis of five single-cell transcriptomes and eight spatial transcriptomes derived from eight HGSOC patients. This study provides a comprehensive view of tumor heterogeneity across the spectrum of gene expression, copy number variation (CNV), and single-cell profiles. Our CNV analysis revealed intratumor heterogeneity by identifying distinct tumor clones, illuminating their evolutionary trajectories and spatial relationships. We further explored the homogeneity and heterogeneity of CNV across tumors to pinpoint the origin of heterogeneity. At the cellular level, single-cell RNA sequencing (scRNA seq) analysis identified three meta-programs that delineate the functional profile of tumor cells. The communication networks between tumor cell clusters exhibited unique patterns associated with the meta-programs governing these clusters. Notably, the ligand-receptor pair MDK - NCL emerged as a highly enriched interaction in tumor cell communication. To probe the functional significance of this interaction, we induced NCL overexpression in the SOVK3 cell line and observed enhanced tumor cell proliferation. These findings indicate that the MDK - NCL interaction plays a crucial role in promoting HGSOC tumor growth and may represent a promising therapeutic target. In conclusion, this study comprehensively unravels the multifaceted nature of HGSOC heterogeneity, providing potential therapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/375bc93ef6f0dd0954afbd4a3ef7ab6a5339dc00" target='_blank'>
              Deciphering ovarian cancer heterogeneity through spatial transcriptomics, single-cell profiling, and copy number variations
              </a>
            </td>
          <td>
            Songyun Li, Zhuo Wang, Hsien-da Huang
          </td>
          <td>2025-03-04</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Immortalized cell lines are commonly used as proxies for primary cells in human biology research. For example, Jurkat leukemic T cells fundamentally contributed to uncovering T cell signaling, activation, and immune responses. However, the immortalization process can alter key cellular properties, and researchers widely believe this process could significantly change RNA modification machinery and modification sites. In this study, we focus on pseudouridine (Ψ), one of the most abundant mRNA modifications, and compare Ψ profiles in mRNA from primary and immortalized T cells using direct RNA sequencing (DRS). Surprisingly, 87% of Ψ-sites were shared between the two cell types, primarily in transcripts encoding proteins involved in essential cellular processes, including RNA-modification regulation. Furthermore, the analysis of the 13% of sites unique to each cell type reveals that Jurkat cells contained transcripts linked to immune activation and oncogenesis, while primary T cells contained transcripts associated with calcium signaling and intracellular trafficking. We provide a list of these genes, which should be considered when using immortalized cells to study RNA modifications in immunology contexts. Most differences were driven by whether the mRNA was present or absent in the immortalized or primary cell type. Interestingly, RNA-modification enzyme expression levels were highly conserved in both cell types. This suggests that site-specific differences in Ψ levels arise from regulatory processes acting in trans rather than differences in modification enzyme levels. Significance Statement It is widely believed that RNA modification machinery and modification sites could be significantly altered in immortalized cells, yet this has never been tested. Focusing on pseudouridine (Ψ), we map Ψ in the transcriptomes of primary and immortalized human T cells. Surprisingly, most sites are conserved in the primary and immortalized T cells, with several important examples of cases with cell type specificity and should be considered on a case-by-case basis. Furthermore, we evaluated RNA-modification machinery levels in primary and immortalized T cells, finding high conservation across the cell lines. Our findings demonstrate that RNA modifications are largely conserved between primary and immortalized cells, and the edge cases can be considered individually.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5b46d555b004bfc6033b490f3c93648df1820f4" target='_blank'>
              Direct RNA sequencing of primary human T cells reveals the impact of immortalization on mRNA pseudouridine modifications
              </a>
            </td>
          <td>
            Oleksandra Fanari, Dylan Bloch, Yuchen Qiu, Michele Meseonznik, Amr Makhamreh, M. Wanunu, S. Rouhanifard
          </td>
          <td>2025-03-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Head and Neck Squamous Cell Carcinomas (HNSCC) are the seventh most prevalent form of cancer and are associated with human papilloma virus infection (HPV-positive) or with tobacco and alcohol use (HPV-negative). HPV-negative HNSCCs have a high recurrence rate, and individual patients’ responses to treatment vary greatly due to the high level of cellular heterogeneity of the tumor and its microenvironment. Here, we describe a HNSCC single cell atlas, which we created by integrating six publicly available datasets encompassing over 230,000 cells across 54 patients. We contextualized the relationships between existing signatures and cell populations, identified new subpopulations, and show the power of this large-scale resource to robustly identify associations between transcriptional signatures and clinical phenotypes that would not be possible to discover using fewer patients. We reveal a previously undefined myeloid population, sex-associated changes in cell type proportions, and novel interactions between CXCL8-positive fibroblasts and vascular endothelial cells. Beyond our findings, the atlas will serve as a public resource for the high-resolution characterization of tumor heterogeneity of HPV-negative HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfe2478d77674f8edfddeb9019aa5fa535133e0f" target='_blank'>
              A highly resolved integrated single-cell atlas of HPV-negative head and neck cancer
              </a>
            </td>
          <td>
            Lina Kroehling, Andrew Chen, Anthony J Spinella, Eric Reed, Maria Kukuruzinka, X. Varelas, S. Monti
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="
 239


 Background:
 Inactivating genomic alteration (GA) of the TP53 tumor suppressor gene leads to inactivation of p53 protein and is currently the most frequently associated GA across all cancers. Colorectal cancer is a common and lethal subtype of cancer, in the top 5 for incidence, prevalence and cancer-related deaths worldwide. TP53 is the second most frequent GA identified in clinically advanced CRC. It has been postulated that GA in TP53 not only affects the molecular biology of malignant cells but also impacts the tumor microenvironment in CRC. This GA is currently untargetable by approved drugs however, recently a novel approach designed to revert to wild type functioning of TP53 inactivated by Y220C base substitution has gained significant clinical interest.
 Methods:
 67,301 cases of clinically advanced CRC underwent hybrid capture based comprehensive genomic profiling to assess all classes of GA. Cases were sequenced to a mean coverage depth of 650X with microsatellite instability (MSI) status and tumor mutation burden (TMB) determined from the sequencing data and PD-L1 expression measured by immunohistochemistry using the DAKO tumor proportional staining system (low positive set at 1-49% staining and high positive set at ≥50% staining).
 Results:
 422 (0.6%) of the CRC featured the
 TP53
 Y220C GA (Y220C+). When compared with the CRC that were
 TP53
 Y220C negative (Y220C-), the Y220C+ cases were of similar age, gender and number of GA per tumor. The Y220C- cases has significantly higher frequencies of MSI status (6.1% vs 2.2%; p=.0007) and TMB levels greater >10 mutations/Mb (8.7% vs 4.7%; p=.004). The Y220C+ featured lower frequencies of GA in
 PIK3CA
 (12.9% vs 18.6%; p=.002),
 BRAF
 (8.8% vs 10.1%; NS) and
 PTEN
 (7.4% vs 8.3%, NS) and a higher frequency of
 ERBB2
 amplification (5.0% vs 2.7%; p=.004). GA in
 APC
 ,
 KRAS
 ,
 NRAS
 ,
 BRCA2
 ,
 EGFR
 and
 MET
 were similar in both Y220C+ and Y220C- CRC. PD-L1 low expression was similarly infrequent (12.1% to 16.7% range) in both groups.
 Conclusions:
 The potentially targetable
 TP53
 Y220C short variant mutation is uncommon in clinically advanced CRC but is associated with unique genomic and biomarker characteristics. Further research might elucidate subpopulations or CRC subtypes with a greater frequency of this mutation who might benefit from a novel targeted therapy.






 CRC
 TP53
 Y220C+(422 Cases)


 CRC
 TP53
 Y220C-(66,879 Cases)

 P Value




 Gender
 41.2% F/ 58.8% M
 44.4% F/55.6% M
 NS


 Median Age (range)
 62 (23-89+)
 61 (8-89+)
 NS


 GA/tumor
 6.1
 6.5
 NS



 TP53
 non-Y220C GA

 16.1%
 75.7%
 <.0001



 PIK3CA

 12.9%
 18.6%
 .002



 ERBB2

 6.2% (5.0% amp)
 5.1% (2.7% amp)
 .004 (amp)


 MSI High
 2.2%
 6.1%
 .0007


 TMB > 10 mut/Mb
 4.7%
 8.7%
 .004


 PD-L1 Low (1-49%TPS)/ PD-L1 High (>50% TPS)
 16.7% (132 cases)/2.3%
 12.1% (20,759 cases)/1.4%
 NS




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99ba4427e7070f06aa4eaa87971edb3bcc24e315" target='_blank'>
              TP53
 Y220C mutations in colorectal carcinoma (CRC): A genomic landscape study.
              </a>
            </td>
          <td>
            Deevyashali Parekh, J. Ross, Nimisha Srivastava, Tamara Jamaspishvili, A. Basnet
          </td>
          <td>2025-02-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Mutations in the shelterin protein POT1 are associated with diverse cancers and thought to drive carcinogenesis by impairing POT1's suppression of aberrant telomere elongation. To classify clinical variants of uncertain significance (VUSs) and identify cancer-driving loss-of-function mutations, we developed a locally haploid human stem cell system to evaluate >1900 POT1 mutations, including >600 VUSs. Unexpectedly, many validated familial cancer-associated POT1 (caPOT1) mutations are haplosufficient for cellular viability, indicating that some pathogenic alleles do not act through a loss-of-function mechanism. Instead, POT1's DNA damage response suppression and telomere length control are genetically separable. ATR inhibition enables isolation of frameshift mutants, demonstrating that the only essential function of POT1 is to repress ATR. Furthermore, comparison of caPOT1 and frameshift alleles reveals a class of caPOT1 mutations that elongate telomeres more rapidly than full loss-of-function alleles. This telomere length-promoting activity is independent from POT1's role in overhang sequestration and fill-in synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25aca8c4eb6696cc9c11ef54c9581b2900ee7f04" target='_blank'>
              Active telomere elongation by a subclass of cancer-associated POT1 mutations.
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2025-02-27</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="R-loops are three-stranded non-canonical nucleic acid structures composed of nascent RNA hybridized with the template DNA strand, leaving the non-template DNA strand displaced. These structures play crucial roles in regulating gene expression, DNA replication, and transcription processes. However, R-loops have also been increasingly described as highly deleterious, causing genomic instability and DNA damage. To maintain R-loops at a relatively safe level, complex regulatory mechanisms exist to prevent their excessive formation. The growing understanding of R-loop functions has provided valuable insights into their structure and potential clinical applications. Emerging research indicates that R-loops contribute to the pathogenesis of various disorders, including neurodegenerative, immune-related, and neoplastic diseases. This review summarizes R-loop metabolism and its significance in the etiology of associated disorders. By elucidating the regulatory mechanisms governing R-loops, we aim to establish a theoretical foundation for understanding disease pathogenesis and exploring novel therapeutic strategies targeting these hybrid nucleic acid structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b259f1c79c41dc7ac4a71263a4f7f0603a6d956" target='_blank'>
              Mechanisms underlining R-loop biology and implications for human disease
              </a>
            </td>
          <td>
            Junzhe Liu, Fengze Li, Yulong Cao, Yonghui Lv, K. Lei, Zewei Tu, Chuandong Gong, Haiyan Wang, Feng Liu, Kai Huang
          </td>
          <td>2025-02-21</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Non-coding RNA transcribed from active enhancers, known as enhancer RNA (eRNA), is a critical element in gene regulation with a highly specific expression pattern in the regulatory networks of tumor-infiltrating cells. Therefore, eRNA signatures could potentially be applied to represent anti-tumor immune cells and to improve cancer immunotherapy. Herein, we identified thousands of eRNAs that were significantly correlated with infiltrating immune cell abundance in more than 10,000 patient samples across a variety of cancer types. The expression of these eRNAs was mediated by transcription factors with high expression in anti-tumor immune cells, as identified through single-cell assays. An eRNA immunotherapy signature (eRIS) developed using the anti-tumor eRNAs was highly associated with the objective response rate (ORR) of immunotherapy and was elevated in patients who benefited from immune checkpoint blockade (ICB) treatment. In comparison with a signature based on protein-coding genes, the eRIS was more effective in predicting the response to immunotherapy. Integration of the eRIS with pharmacogenomic data revealed hundreds of anti-cancer drugs that have the potential to enhance immunotherapy efficacy. Finally, treatment of a mouse model of IDH mutant glioma with the histone deacetylase inhibitor vorinostat improved the effects of anti-PD-1 immunotherapy through increased abundance of infiltrating immune cells. Taken together, this study developed an eRIS with demonstrated efficacy in predicting immunotherapy response and used the eRIS to identify a series of effective combination drugs, thus highlighting the clinical utility of the eRIS in immunotherapy enhancement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb404431d7b2aee513f3b1e9892ee993135259d" target='_blank'>
              Characterization of an Enhancer RNA Signature Reveals Treatment Strategies for Improving Immunotherapy Efficacy in Cancer.
              </a>
            </td>
          <td>
            Chenyang Zhang, Yan-Yan Chen, Shuyu Chen, Yunzhe Wang, Yifan Yuan, Xiwen Yang, Wei Hu, Bo Chen, Zengxin Qi, Jason T Huse, Yun Liu, Bo Wen, Xiuping Liu, Leng Han, Yuxiang Wang, Zhao Zhang
          </td>
          <td>2025-02-04</td>
          <td>Cancer research</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/286f65c0d127245d474792c410ee7afae8aec904" target='_blank'>
              Complicated crosstalk between HMGA and non-coding RNAs modulates hallmarks of cancer
              </a>
            </td>
          <td>
            Lijie Zhang, Xiaomin Zhao, Xianghong Gao, Hao Qin, Feng Chen, Zhijuan Lin
          </td>
          <td>2025-03-07</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Adoptive immune cell therapies, such as CAR T, have revolutionized how we can approach cancer therapy, however challenges remain to increase their efficacy, response rates, and resistance to dysfunction. The systematic investigation of gene programs governing human T cell function would be advantageous for reprogramming cell products for clinical benefit. Recent studies have demonstrated the power of CRISPR-based forward genetic screens in primary human T cells to nominate candidate perturbations for enhanced T cell function, however combinations of perturbations will likely be necessary to overcome the most challenging tumors. To enable the systematic and high-throughput interrogation of combinations of gain and loss-of-function perturbations, we have developed a bidirectional CRISPR activation and interference (CRISPRai) system, where pairs of genes can be simultaneously activated and repressed in primary human T cells. Leveraging hits from previous genome-wide CRISPRa and CRISPRi screens, we constructed a library of >66,000 sgRNA combinations and screened for regulators of acute T cell stimulation responses, including cytokine production and proliferation. Reassuringly, these screens identified expected gene epistasis interactions (e.g. rescue of PLCG1 repression by PLCG2 activation), while also discovering novel genetic interactions, providing mechanistic insights into the regulatory networks governing stimulation response. To further identify gene “building blocks” for evaluation as combinations, we completed genome-wide CRISPRa and CRISPRi screens for regulators of human CD8+ CAR T cell exhaustion. These screens identified known potential enhancements (e.g. gain of BATF, loss of MED12), and also uncovered novel regulators of T cell function, with CRISPRa discovering genes not normally expressed in T cells but whose overexpression could cause resistance to exhaustion by synthetic mechanisms. Gene expression programs controlled by perturbation of key regulators were mapped by pairing CRISPR with single-cell RNA-seq (Perturb-seq) and arrayed functional assays. Finally, a top gain-of-function candidate was evaluated for in vivo tumor control, using an antigen-inducible Synthetic Notch system. Together, this work demonstrates how advanced forward genetics approaches can discover and elucidate both natural and synthetic programs that regulate T cell states, and the nomination of perturbations to encode in logic-gated synthetic biology designs for cell therapies.
 Citation Format: Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole Roybal, Brian Shy, Alexander Marson. Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens [abstract]. In: Proceedings of the AACR IO Conference: Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2025 Feb 23-26; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2025;13(2 Suppl):Abstract nr A026.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/996e0da402cc341f0ff28812c9362a9660b3a867" target='_blank'>
              Abstract A026: Deciphering natural and synthetic genetic programs governing human T cell function with combinatorial CRISPR activation and interference screens
              </a>
            </td>
          <td>
            Zachary Steinhart, Jeffrey Perera, Dmytro Dorovskyi, Luke Workley, Carinna Tran, Rosmely Hernandez, Zhongmei Li, Max Foisey, Daniel Goodman, Kole T. Roybal, B. Shy, Alexander Marson
          </td>
          <td>2025-02-23</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The transcription factor p53 is the most frequently impaired tumor suppressor in human cancers. In response to various stress stimuli, p53 activates transcription of genes that mediate its tumor-suppressive functions. Distinctive characteristics of p53 outlined here enable a well-defined program of genes involved in cell cycle arrest, apoptosis, senescence, differentiation, metabolism, autophagy, DNA repair, anti-viral response, and anti-metastatic functions, as well as facilitating autoregulation within the p53 network. This versatile, anti-cancer network governed chiefly by a single protein represents an immense opportunity for targeted cancer treatment, since about half of human tumors retain unmutated p53. During the last two decades, numerous compounds have been developed to block the interaction of p53 with the main negative regulator MDM2. However, small molecule inhibitors of MDM2 only induce a therapeutically desirable apoptotic response in a limited number of cancer types. Moreover, clinical trials of the MDM2 inhibitors as monotherapies have not met expectations and have revealed hematological toxicity as a characteristic adverse effect across this drug class. Currently, combination treatments are the leading strategy for enhancing efficacy and reducing adverse effects of MDM2 inhibitors. This review summarizes efforts to identify and test therapeutics that work synergistically with MDM2 inhibitors. Two main types of drugs have emerged among compounds used in the following combination treatments: first, modulators of the p53-regulated transcriptome (including chromatin modifiers), translatome, and proteome, and second, drugs targeting the downstream pathways such as apoptosis, cell cycle arrest, DNA repair, metabolic stress response, immune response, ferroptosis, and growth factor signaling. Here, we review the current literature in this field, while also highlighting overarching principles that could guide target selection in future combination treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3895fe8ca5319763195e6c375cf525ba1915bfa" target='_blank'>
              Harnessing p53 for targeted cancer therapy: new advances and future directions.
              </a>
            </td>
          <td>
            Z. Andrysik, Joaquin M Espinosa
          </td>
          <td>2025-03-03</td>
          <td>Transcription</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67046daa5df3cebbea573edba163f96ed0a30fda" target='_blank'>
              Visualization of chromosomal reorganization induced by heterologous fusions in the mammalian nucleus
              </a>
            </td>
          <td>
            Meng Yan, X. Zhang, Zhenhua Yang, Miao Jia, Rongyu Liao, Jinsong Li
          </td>
          <td>2025-02-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Synthetic lethality approaches in BRCA1/2-mutated cancers have focused on poly(ADP-ribose) polymerase (PARP) inhibitors, which are subject to high rates of innate or acquired resistance in patients. Here, we used CRISPR/Cas9-based screening to identify DNA Ligase I (LIG1) as a novel target for synthetic lethality in BRCA1-mutated cancers. Publicly available data supported LIG1 hyperdependence of BRCA1-mutant cells across a variety of breast and ovarian cancer cell lines. We used CRISPRn, CRISPRi, RNAi, and protein degradation to confirm the lethal effect of LIG1 inactivation at the DNA, RNA, and protein level in BRCA1-mutant cells in vitro. LIG1 inactivation resulted in viability loss across multiple BRCA1-mutated cell lines, whereas no effect was observed in BRCA1/2 wild-type cell lines, demonstrating target selectivity for the BRCA1-mutant context. On-target nature of the phenotype was demonstrated through rescue of viability with exogenous wild-type LIG1 cDNA. Next, we demonstrated a concentration-dependent relationship of LIG1 protein expression and BRCA1 mutant cell viability using a titratable, degradable LIG1 fusion protein. BRCA1 mutant viability required LIG1 catalytic activity, as catalytically dead mutant LIG1K568A failed to rescue viability loss caused by endogenous LIG1 depletion. LIG1 perturbation produced proportional increases in PAR staining in BRCA1 mutant cells, indicating a mechanism consistent with the function of LIG1 in sealing ssDNA nicks. Finally, we confirmed LIG1 hyperdependence in vivo using a xenograft model in which LIG1 loss resulted in tumor stasis in all mice. Our cumulative findings demonstrate that LIG1 is a promising synthetic lethal target for development in patients with BRCA1 mutant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/831e285c6dc25f79bcb13593b00dacbe2a024fd9" target='_blank'>
              LIG1 is a synthetic lethal target in BRCA1 mutant cancers.
              </a>
            </td>
          <td>
            Lauren Martires, Leanne G Ahronian, Charlotte B Pratt, Nikitha M Das, Xiaobin Zhang, D. Whittington, Hongxiang Zhang, B. Shen, Jon Come, Patrick McCarren, Mu-Sen Liu, Chengyin Min, Tianshu Feng, Haris Jahic, Janid A Ali, Daniel R Aird, Fang Li, Jannik N Andersen, A. Huang, William D Mallender, Hilary E. Nicholson
          </td>
          <td>2025-01-27</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="ABSTRACT Disrupted nuclear shape is associated with multiple pathological processes including premature aging disorders, cancer-relevant chromosomal rearrangements and DNA damage. Nuclear blebs (i.e. herniations of the nuclear envelope) can be induced by (1) nuclear compression, (2) nuclear migration (e.g. cancer metastasis), (3) actin contraction, (4) lamin mutation or depletion, and (5) heterochromatin enzyme inhibition. Recent work has shown that chromatin transformation is a hallmark of bleb formation, but the transformation of higher-order structures in blebs is not well understood. As higher-order chromatin has been shown to assemble into nanoscopic packing domains, we investigated whether (1) packing domain organization is altered within nuclear blebs and (2) whether alteration in packing domain structure contributed to bleb formation. Using dual-partial wave spectroscopic microscopy, we show that chromatin-packing domains within blebs are transformed both by B-type lamin depletion and the inhibition of heterochromatin enzymes compared to what is seen in the nuclear body. Pairing these results with single-molecule localization microscopy of constitutive heterochromatin, we show fragmentation of nanoscopic heterochromatin domains within bleb domains. Overall, these findings indicate that chromatin within blebs is associated with a fragmented higher-order chromatin structure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/691331448a7d0baa47fbb1a0f67d9e5391519de2" target='_blank'>
              Nuclear blebs are associated with destabilized chromatin-packing domains
              </a>
            </td>
          <td>
            Emily M. Pujadas Liwag, Nicolas Acosta, L. Almassalha, Yuanzhe Su, Ruyi Gong, Masato T. Kanemaki, A. Stephens, Vadim Backman
          </td>
          <td>2025-01-29</td>
          <td>Journal of Cell Science</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="
 The development of tumoroid (or cancer organoid) technology has enabled the in vitro culture of patient-derived cancer cells that retain key morphological, mutational, and transcriptomic characteristics of patient tumors. In a number of studies, the response of tumoroids to therapies has mirrored that seen in the patients from which the tumoroids were derived. As a result, there is increasing interest in the use of tumoroids in functional precision oncology, in which an individual patient’s cancer cells are used as ex vivo tumor avatars and tested for response to a range of various potential treatments to aid in therapy selection. Here, we explored a workflow to integrate genomic sequencing with functional drug sensitivity testing for precision medicine applications. Excess FFPE tissue from 6 colorectal cancer donors was processed to generate H&E slides and unstained sections. Slides (2 x 10 µm sections per donor) were digested, and RNA and DNA were isolated using an automated nucleic acid purification system. The output samples were analyzed on the Genexus™ Integrated Sequencer for targeted detection of SNVs, CNVs, gene fusions, and indels for each sample using the Oncomine™ Comprehensive Assay v3. Purified RNA was also utilized for bulk RNA sequencing to generate transcriptomic profiles for each donor. In parallel, viably cryopreserved dissociated tumor cells (DTCs) were thawed and plated in Gibco™ OncoPro™ Tumoroid Culture Medium, a commercially available, serum-free, and conditioned medium-free system. OncoPro medium was developed and optimized for the culture of patient-derived tumor cells from solid cancers to provide an easy-to-use, reproducible, cGMP manufactured culture system for these samples. Where sample size allowed, RNA was isolated from DTC samples and sequenced to compare transcriptomic results generated from FFPE vs. viably cryopreserved material. Additionally, DTCs were sampled for untargeted proteomic and metabolomic profiling. DTCs were cultured for 3-7 days in OncoPro medium prior to exposure to chemotherapy drugs used in standard-of-care treatment for colorectal cancer, and cell response was quantified. Dissociated tumoroid cells formed tumoroids at high rates in culture, in line with previous studies showing that patient-derived cancer cells across colorectal, endometrial, lung, breast, and head and neck cancers form tumoroids in OncoPro medium in >85% of samples within 7 days of tissue receipt. We observed heterogeneous genomic, proteomic, transcriptomic, and drug response profiles across donors. Current efforts are focused on development of computational methods to tie multiomic characterization with functional response. Overall, this data presents an integrated workflow to combine genomic approaches with functional testing to inform patient care.
 Citation Format: Colin D Paul, Amber Bullock, Anthony Chatman, Km Shams Ud Doha, Harrison Leong, Sylvia Beam, Chris Yankaskas, Pradip Shahi Thakuri, Matt Dallas, David Kuninger. An integrated workflow for multiomic characterization and functional precision medicine testing of solid tumor samples [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr B017.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/564696ec86cb40931955901184003710aa166dc0" target='_blank'>
              Abstract B017: An integrated workflow for multiomic characterization and functional precision medicine testing of solid tumor samples
              </a>
            </td>
          <td>
            Colin Paul, Amber Bullock, Anthony Chatman, Km Shams Ud Doha, Harrison Leong, Sylvia Beam, Chris Yankaskas, Pradip S. Thakuri, Matt Dallas, David Kuninger
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Noncoding microRNAs tend to evolve within introns of coding genes that provide them with transcriptional opportunity. As an outcome of natural selection, the intragenic position of microRNAs is crucial for their expression, evolution, and functional cooperation with the host gene. Therefore, understanding the evolution of intragenic microRNA structures may bring novel insights into genetic and phenotypic evolution. However, it remains largely unexplored. Here, by analyzing microRNA genomics in 34 metazoan species, we found that the majority (630/1,154) of microRNA families originated from introns of coding genes that provided them with initial transcriptional capacity. The most rapid expansion of intragenic microRNAs happened at the advent of vertebrates when 21 microRNA families emerged from introns of neural genes and reorganized the gene regulatory network, leading to the rise of vertebrate-specific neural crest cells, which transformed the invertebrate head and enabled the ecological shift from filter feeding to active predation. Intragenic microRNAs gradually gain independence from their host genes, which is accelerated by whole-genome duplications. After a whole-genome duplication, the purging of redundant host genes often set an orphaned microRNA “free” to diversify with the transcriptional elements inherited from the host. Whole-genome duplications facilitated a dramatic microRNA diversification during the initial divergence of vertebrates, as the intragenic status of 12 neural crest-regulating microRNAs was retained in jawed vertebrates but was lost in jawless cyclostomes, which diverged their neural crest development. We propose that coding genes not only facilitate the origination of new microRNAs, but also “sacrifice” themselves to help microRNA diversification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec14a5673e5811f8ba88226799b41e9414365cb1" target='_blank'>
              Coding Genes Helped the Origination and Diversification of Intragenic MicroRNAs
              </a>
            </td>
          <td>
            Chengzhang Liu, Xiaojun Zhang, Jianbo Yuan, J. Xiang, Fuhua Li
          </td>
          <td>2025-02-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="
 Advancing precision medicine research relies heavily on laboratory models that accurately reflect the rich intra- and inter-patient diversity of human tumors. While large-scale resources built on historical cell lines have provided valuable insights, these models often lack the molecular diversity needed for comprehensive translational research and may undergo genetic drift in culture, reducing their fidelity. Recent advances in technologies, such as organoid culture, have improved model derivation; however, systematic pan-cancer comparisons of genomic, transcriptomic, and epigenomic integrity following long-term ex vivo expansion remain limited. Here, we report on the Human Cancer Models Initiative (HCMI), an international program that has generated and systematically characterized 665 organoid, spheroid, and cell line models derived from over 2,500 consented donors. These models have undergone comprehensive whole-genome, exome, methylome, and transcriptome analyses. The HCMI collection includes 47 unique models representing 16 rare and histologically distinct cancer subtypes, including one of a kind disease models (e.g. desmoid tumor) and greatly broadening the diversity of available cancer models. Through integrative concordance analysis, we demonstrate that 96% of the 417 analyzed models closely mirror the molecular profiles of their parental tumors. Interesting rare exceptions to this concordance include subsets of models, where cell culture media conditions, tumor stroma, or population selection appear to influence cellular characteristics. We validated observed cellular state shifts using single-cell RNA sequencing, providing insights into the impact of ex vivo culture on model integrity. We further explored the translational utility of this resource by investigating treatment exposures, post-treatment mutational signatures, and extrachromosomal DNA (ecDNA) amplifications and their potential implications on treatment resistance within these models. This community resource offers a comprehensive roadmap for capturing a broader spectrum of cancer diversity in preclinical models, ultimately supporting and advancing therapeutic discovery.
 Citation Format: Dina ElHarouni, Mushriq Al-Jazrawe, Seongmin Choi, Merve Dede, Toshinori Hinoue, Sean A Misek, Heeju Noh, Luca Zanella, Moony Tseng, Hayley E Francies, Priya Sridevi, Rachana Agarwal, Cindy W Kyi, Julyann Perez-Mayoral, Megan J Stine, Eva Tonsing-Carter, James M Clinton, The HCMI Network, Peter W Laird, Calvin J Kuo, Olivier Elemento, David L Spector, Andrew D Cherniack, Kyle Ellrott, Martin L Ferguson, Rameen Beroukhim, Katherine A Hoadley, Nicolas Robine, Mathew Garnett, Andrea Califano, Paul T Spellman, David A Tuveson, Keith L Ligon, Daniela S Gerhard, Louis M Staudt, Jesse Boehm. Integrative clinical and molecular analysis of 665 next-generation in vitro cancer models generated by the the Human Cancer Models Initiative (HCMI) for advancing precision medicine and functional drug discovery [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr B025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38d11642e20fef657158a832ae45b3c22aa7b26f" target='_blank'>
              Abstract B025: Integrative clinical and molecular analysis of 665 next-generation in vitro cancer models generated by the the Human Cancer Models Initiative (HCMI) for advancing precision medicine and functional drug discovery
              </a>
            </td>
          <td>
            Dina Elharouni, Mushriq Al-Jazrawe, Seongmin Choi, Merve Dede, T. Hinoue, Sean A Misek, Heeju Noh, Luca Zanella, Moony Tseng, Hayley E. Francies, Priya Sridevi, Rachana Agarwal, Cindy W Kyi, Julyann Pérez-Mayoral, Megan J Stine, Eva Y. Tonsing-Carter, James M Clinton, The Hcmi Network, Peter W. Laird, Calvin J Kuo, O. Elemento, David L. Spector, A. Cherniack, Kyle Ellrott, M. Ferguson, R. Beroukhim, Katie Hoadley, N. Robine, Mathew Garnett, Andrea Califano, Paul T Spellman, D. Tuveson, Keith L. Ligon, Daniela S Gerhard, Louis M Staudt, Jesse S Boehm
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>148</td>
        </tr>

        <tr id="
 Background: Drug resistance is a major challenge in cancer treatment, often driven by tumor cell heterogeneity. We cultivated circulating tumor cells (CTCs) from the blood of patients with solid tumors, including those preserved as frozen PBMCs, into cancer organoids. Two organoid lines, derived from renal and colon cancers, have been maintained and used extensively to assess drug sensitivity. During these experiments, viability assays revealed that a subset of organoids displayed resistance to certain drugs, resuming growth after initial inhibition. This prompted us to investigate this heterogeneity using specialized assays. Methods: To study variability in drug responses among CTC-derived organoids, we employed three complementary approaches: 1. Microcavity Plates: Corning's Elplasia plates with microcavities were used to segregate organoids, facilitating high-resolution, kinetic imaging for monitoring individual organoid growth and survival during drug treatment. 2. Image Analysis: Morphological changes in organoids during drug exposure were tracked using image analysis, allowing for the quantitation of resistant organoids and providing insights into variations in growth patterns and morphological changes. 3. Transcriptome Profiling: Using individual organoids, each with sufficient RNA for analysis, allowed detailed transcriptomic profiling and provided a comprehensive assessment of genetic responses in drug-resistant organoids. Results: Preliminary results from a limited set of one renal and one colon cancer organoids indicate that the three methods we employed show promise in detecting heterogeneity in drug sensitivity. The use of Elplasia plates allowed for consistent kinetic monitoring of individual organoid survival, and image analysis effectively captured morphological changes associated with varying drug responses. Initial transcriptomic profiling hinted at distinct genetic signatures in drug-resistant organoids, suggesting a potential reflection of physiological heterogeneity seen in patient tumors, potentially including metastatic ones. While these findings are encouraging, further studies with more samples are needed to confirm these observations and refine our approach. Conclusion and Future Directions: Our integrated approach appears to be promising in identifying and characterizing drug sensitivity heterogeneity in CTC-derived cancer organoids. We plan to expand this study to include more patient samples and cancer types, aiming to confirm our findings and identify potential biomarkers of resistance.
 Citation Format: Yi-chun Han, David Hsieh, Shian-Jiun Shih. Evaluating drug sensitivity heterogeneity in cancer organoids derived from circulating tumor cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Functional and Genomic Precision Medicine in Cancer: Different Perspectives, Common Goals; 2025 Mar 11-13; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(5 Suppl):Abstract nr B035.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53dcae9646767b55b828bf57d0d3c3835c11bb8d" target='_blank'>
              Abstract B035: Evaluating drug sensitivity heterogeneity in cancer organoids derived from circulating tumor cells
              </a>
            </td>
          <td>
            Y. Han, David Hsieh, Shian-Jiun Shih
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The PPM1D gene and its protein product (serine-threonine protein phosphatase, PPM1D or Wip1) are involved in regulation of cell's DNA damage response, cell cycle control, and repair. Amplification, overexpression, or mutations of the PPM1D gene have a significant impact on cell responses to stress factors and genetic instability as well as impairments of processes of double-strand break repair, nucleotide excision repair, base excision repair, cell cycle, and apoptosis. PPM1D dephosphorylates and thus inactivates p53, proteins that respond to DNA strand integrity damage, cell cycle checkpoint proteins, and apoptotic proteins. This contributes to tumor development, growth, and maintenance of the tumor phenotype. In this review we consider data on the role of the PPM1D gene in the formation and maintenance of various oncological processes, including tumors of the mammary glands, ovaries, prostate gland, esophagus, stomach, intestines, liver and pancreas, hemoblastoses, and others.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e1769adfe2db6b9f47beac988f9dcd9d78de7ec" target='_blank'>
              The role of the PPM1D gene in tumor pathogenesis.
              </a>
            </td>
          <td>
            A.S. Kucheryavenko, E. A. Muzyko, V. N. Perfilova, K.D. Kaplanov, M.Yu. Frolov
          </td>
          <td>2025-02-01</td>
          <td>Biomeditsinskaia khimiia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Historical studies performed nearly a century ago using mouse skin models identified two key steps in cancer evolution: initiation, a likely mutational event, and promotion, driven by inflammation and cell proliferation. Initiation was proposed to be permanent, with promotion as the critical rate-limiting step for cancer development. Here, we carried out whole genome sequencing to demonstrate that initiated cells with thousands of mutagen-induced mutations can persist for long periods and are not removed by cell competition or by immune intervention, thus mimicking the persistence of cells with cancer driver mutations in normal human tissues. In the mouse, these cells do not give rise to tumors unless exposed to the tumor promoter TPA. Tissue damage and regenerative proliferation, but not normal cell turnover, consistently trigger tumor formation. Wounding, promoter treatment, and obesity enhance promotion without increasing mutational burden, supporting the possibility of future cancer prevention efforts directed at promotional risk factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd9f59a39a075b237d1c5fa6c39932e43dffde05" target='_blank'>
              Long term latency of highly mutated cells in normal mouse skin is reversed by exposure to tumor promoters and chronic tissue damage.
              </a>
            </td>
          <td>
            Y. R. Li, E. Kandyba, Kyle Halliwill, R. Delrosario, Quan Tran, Nora Bayani, Di Wu, Olga K. Mirzoeva, Melissa Q Reeves, S M Ashiqul Islam, Laura Riva, E. Bergstrom, Kavya Achanta, John DiGiovanni, Ludmil B. Alexandrov, Allan Balmain
          </td>
          <td>2025-03-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/590a8f98137453dc965f4fb64612753d1428ec38" target='_blank'>
              The functions and modifications of tRNA-derived small RNAs in cancer biology.
              </a>
            </td>
          <td>
            Abdulaziz Ahmed A Saad, Kun Zhang, Qianqian Deng, Jiawang Zhou, Lichen Ge, Hongsheng Wang
          </td>
          <td>2025-03-01</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df4d7f1f1862a0b62b5b5df29d2416fddab0290c" target='_blank'>
              lncRNAs: the unexpected link between protein synthesis and cancer adaptation
              </a>
            </td>
          <td>
            Mila Gugnoni, Manoj Kumar Kashyap, Kishore K. Wary, A. Ciarrocchi
          </td>
          <td>2025-01-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The presence of circulating tumor DNA (ctDNA) in patients with colorectal adenomas remains uncertain. Studies using tumor-agnostic approaches report ctDNA in 10-15% of patients, though with uncertainty as to whether the signal originates from the adenoma. To obtain an accurate estimate of the proportion of patients with ctDNA, a sensitive tumor-informed strategy is preferred, as it ensures the detected signal originates from the adenoma. Here, tumor-informed whole-genome sequencing-based ctDNA analysis (MRD-EDGESNV) was applied to two independent cohorts. Cohort 1, comprising 93 patients with stage III colorectal cancer (CRC) and 40 healthy individuals, was used to establish the signal threshold at 95% specificity. This threshold was then applied to Cohort 2, consisting of 22 patients with symptomatic and 20 with asymptomatic adenomas. In stage III, MRD-EDGESNV had an area under the curve of 0.98. ctDNA was detected in 50% and 25% of patients with symptomatic and asymptomatic adenomas, respectively. The median adenoma plasma tumor fraction was 5.9 × 10-5. These finding not only demonstrate the feasibility of ctDNA detection in patients with colorectal adenomas, but also provides an estimate of the necessary sensitivity required to detect these lesions, paving the way for future ctDNA-based screening strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b4d40c77bd0aacf8ed113c3130ce3e34e7639b7" target='_blank'>
              Whole-genome sequencing of cell-free DNA reveals DNA of tumor origin in plasma from patients with colorectal adenomas.
              </a>
            </td>
          <td>
            A. Frydendahl, Adam J. Widman, N. Øgaard, Anushri Arora, Daniel Halmos, J. Nors, J. Ahrenfeldt, T. Henriksen, Christina Demuth, Line Raaby, M.H. Rasmussen, C. Therkildsen, Dan A Landau, C. Andersen
          </td>
          <td>2025-01-20</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a819a83329e7d0a414c5918852d0411278218a5" target='_blank'>
              High CDC20 levels increase sensitivity of cancer cells to MPS1 inhibitors
              </a>
            </td>
          <td>
            Siqi Zheng, Linoy Raz, Lin Zhou, Yael Cohen-Sharir, Ruifang Tian, M. Ippolito, Sara Gianotti, Ron Saad, René Wardenaar, Mathilde Broekhuis, Maria F Suarez Peredo Rodriguez, Soraya Wobben, A. van den Brink, Petra Bakker, S. Santaguida, F. Foijer, Uri Ben-David
          </td>
          <td>2025-01-21</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>35</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [13],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>