<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-10-06 09:30:04 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>39</td>
          <td>105</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>83</td>
          <td>138</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>114</td>
          <td>146</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>364</td>
          <td>138</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>318</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="In tumors, extrachromosomal DNA (ecDNA) is an important driver of oncogene expression, genomic instability, the evolution of drug resistance, and poor patient prognosis. ecDNA is present in various tumors but is rarely found in normal cells. Here, we provide a detailed review of the structure, genetics, occurrence, outcomes, and functions of ecDNA, offering further reference for research on ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854e01c4800ec9b678143d63d0ad28cdee5f8010" target='_blank'>
              The role of extrachromosomal DNA in tumorigenesis and progression
              </a>
            </td>
          <td>
            Xiaoyang Ma, Xiaolin Yu, Chuan Wu, Lixing Song
          </td>
          <td>2025-09-15</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Extrachromosomal circular DNA (eccDNA) is an emerging class of genetic material that exists outside of the chromosomal genome. These circular DNA molecules are gaining increasing attention as important biomarkers in various cancers because of their roles in gene amplification, genetic heterogeneity, and drug resistance. In this review, we explore in depth the impacts of eccDNAs on cancer biology, their potential to predict treatment sensitivity and resistance, and their involvement in the development of new anticancer therapies. eccDNAs can be used as biomarkers for various tumor types to help diagnose and predict prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1f8c896f37f0d86135daabcae627eb40e4e88c" target='_blank'>
              Extrachromosomal Circular DNA as a Cancer Biomarker: From Diagnosis to Treatment
              </a>
            </td>
          <td>
            Hexin Li, Jiahui Cai, Gang Zhao, Lihui Zou
          </td>
          <td>2025-09-08</td>
          <td>Cancer Innovation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Extrachromosomal circular DNA (eccDNA) refers to small, circular DNA molecules that originate from chromosomal sequences and are prevalent across nearly all eukaryotic organisms. In humans, eccDNAs are widely distributed in normal tissues, cancerous tissues, and body fluids, where they play important roles in tumorigenesis and are often associated with poor clinical outcomes. Given their biological and clinical significance, a well-integrated and high-quality database is essential for advancing eccDNA-related research. To address this need, we developed eccDNABase, a comprehensive and curated resource for browsing, searching, and analyzing eccDNAs across multiple species. The database systematically catalogs eccDNA–disease associations from diverse tissues and organisms. Currently, eccDNABase contains 1,875,452 eccDNA–disease associations, encompassing 8,398 ecDNA entries across nine species, 63 diseases, and healthy individuals. Each entry provides detailed information, including eccDNA ID, type, chromosomal localization, species, tissue or cell line source, disease name and Disease Ontology ID, overlap length and percentage with genes, oncogene overlap, detection method, and links to literature and source databases. Given its extensive and curated datasets, eccDNABase serves as a valuable resource for both basic and translational research, offering deeper insights into the role of eccDNA in health and disease. The database is publicly accessible at http://cgga.org.cn/eccDNABase/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf15dcf8d0c3a1e284faf711bb5268a1015b46d0" target='_blank'>
              eccDNABase: A Comprehensive and High-Quality Database for Extrachromosomal Circular DNA
              </a>
            </td>
          <td>
            Jin-Hao Zhang, Chang-Lin Yang, Chang-Yuan Ren, Ji Shi, Han-Xiao Zhou, Ying Zhang, Zhi-Yan Sun, Tao Jiang, Zheng Zhao
          </td>
          <td>2025-09-13</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/127ba85daf25e695241e6cc43a8f791c28684c9d" target='_blank'>
              DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, E. Larose Cadieux, Nana E. Mensah, Xiaohong Liu, Kezhong Chen, O. Chervova, Takahiro Karasaki, C.E. Weeden, Corentin Richard, Siqi Lai, Carlos Martínez-Ruiz, E. Lim, A. Frankell, T. Watkins, Georgia Stavrou, Ieva Usaite, Wei-Ting Lu, Daniele Marinelli, S. Saghafinia, G. Wilson, P. Dhami, H. Vaikkinen, Jonathan Steif, S. Veeriah, R. Hynds, Martin Hirst, C. Hiley, A. Feber, Özgen Deniz, M. Jamal-Hanjani, N. Mcgranahan, Stephan Beck, J. Demeulemeester, M. Tanić, C. Swanton, P. Van Loo, N. Kanu
          </td>
          <td>2025-09-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Abstract Background Extrachromosomal circular DNA (eccDNA) has emerged as a critical driver of oncogenesis, yet its functional roles in high‐grade serous ovarian cancer (HGSOC) remain poorly characterized. This highlights the need for comprehensive investigations into the abundance, biogenesis, and functional implications of eccDNA in HGSOC. Methods To characterize eccDNA in HGSOC, we performed comprehensive Circle‐seq analysis to assess eccDNA abundance and genomic annotation in HGSOC tissues compared to normal ovarian tissue. For mechanistic validation of eccDNA biogenesis pathways, targeted knockdown experiments of microhomology‐mediated end‐joining (MMEJ) dependent on LIG3 and POLQ were conducted. Functional characterization of HGSOC‐specific eccDNA‐harboring precursor microRNAs (eccMIRs) included in vitro assays using HGSOC cells and in vivo tumor growth experiments. Results Circle‐seq analysis revealed a 13‐fold increase in eccDNA abundance in HGSOC compared to normal ovarian tissue, with significant enrichment in promoter and coding regions. The MMEJ pathway was identified as the predominant pathway for eccDNA biogenesis in HGSOC, supported by characteristic microhomologies at junction sites and validation via LIG3 and POLQ knockdown experiments. Notably, HGSOC‐specific eccDNA frequently contained functional eccMIRs (eccMIR3661, eccMIR618, and eccMIR2277), which generate oncogenic miRNAs. These miRNAs promote tumor progression by downregulating tumor suppressor genes and activating key oncogenic pathways. Functional assays confirmed that these eccMIRs significantly enhanced HGSOC cell proliferation, migration, and invasion in vitro and promoted tumor growth in vivo. Conclusions These results underscore eccDNA as an oncogenic driver in HGSOC through non‐coding RNA‐mediated regulatory mechanisms, revealing novel therapeutic opportunities for targeting eccDNA biogenesis in this aggressive malignancy. Key points This study revealed a 13‐fold increase of eccDNA in HGSOC compared to normal tissues, with significant enrichment in promoter and coding regions. eccDNA‐derived miRNAs (eccMIRs) were shown to enhance cancer cell proliferation, invasion, and tumor growth through the expression of oncogenic miRNA sequences. The study highlights the importance of the MMEJ pathway in eccDNA generation and proposes that targeting eccDNA biogenesis in this aggressive malignancy presents a novel therapeutic opportunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc21093a2da7863b736808acd2b5cbd885658bf0" target='_blank'>
              Extrachromosomal circular DNA expressing miRNA promotes ovarian cancer progression
              </a>
            </td>
          <td>
            Ning Wu, Ling Wei, Qiyu Liu, T. He, Cuiyu Huang, Yunpeng Jiang, Kailong Li, Hongyan Guo, Fengbiao Mao, Xiaolu Zhao
          </td>
          <td>2025-09-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lung cancer is a highly heterogeneous disease primarily driven by tobacco smoking. About 20% of lung cancers occur among patients who have never smoked (LCINS) with differences in patient ancestry, sex, tumor histology, and clinical features. Our understanding of chromosomal instability in lung cancer, especially LCINS, is still limited. Here, we perform a comprehensive study of 182,429 somatic structural variations (SVs) detected in 1,209 whole-genome sequenced lung cancers, of which 864 LCINS. SVs are more abundant in tumors from patients who have smoked (LCSS); however, they are more complex and play more important roles in tumorigenesis in LCINS. EGFR mutations and KRAS mutations profoundly and independently shape the SV landscape. EGFR-mutant tumors have higher SV burden and more cancer-driving SVs. In contrast, KRAS mutations are associated with lower SV burden and less driver SVs. We decompose 16 SV signatures for both complex and simple SVs that likely represent divergent molecular mechanisms. The SV breakpoints have distinct distributions across the genome depending on the signatures due to mutagenic mechanisms and positive selection. Many established cancer-driving genes are recurrently rearranged by multiple SV signatures suggesting functional convergence of these genome instability mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e02ec2c65d55813a962cf1f4c6e68eaaf0b28784" target='_blank'>
              Panorama of Chromosomal Instability in Lung Cancer
              </a>
            </td>
          <td>
            Yang Yang, Xiaoming Zhong, William Phillips, Wei Zhao, P. Hoang, Christopher Wirth, S. Yang, Charles Leduc, M. Baine, William D. Travis, L. Sholl, Philippe Joubert, Robert J Homer, Jian Sang, Azhar Khandekar, John P. McElderry, Thi-Van-Trinh Tran, Caleb Hartman, Mona Miraftab, Monjoy Saha, Olivia W. Lee, Sunandini Sharma, Kristine Jones, Bin Zhu, M. Díaz-Gay, E. Edell, Jacobo Martínez Santamaría, M. Schabath, Sai S. Yendamuri, Marta Mańczuk, J. Lissowska, B. Świątkowska, A. Mukeria, O. Shangina, D. Zaridze, I. Holcatova, V. Janout, D. Mates, Simona Ognjanovic, Milan Savic, M. Kontić, Y. Bossé, B. G. Rothberg, D. Christiani, V. Gaborieau, Paul Brennan, Geoffrey Liu, P. Hofman, Maria Pik Wong, Kin Chung Leung, Chih-Yi Chen, C. A. Hsiung, A. Pesatori, Dario Consonni, Nathaniel Rothman, Qing Lan, Martin A. Nowak, David C Wedge, Ludmil B. Alexandrov, Stephen J Chanock, Jianxin Shi, Tongwu Zhang, Lixing Yang, Marion Landi
          </td>
          <td>2025-09-12</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cfc80059f8767909d0c8bd930a6c33c0803bbb" target='_blank'>
              Preservation and Clonal Behavior of Extrachromosomal DNA in Patient-Derived Xenograft Models of Childhood Cancers.
              </a>
            </td>
          <td>
            Rishaan Kenkre, Jon D. Larson, Owen S. Chapman, J. Luebeck, Yan Yuen Lo, Megan Paul, Wenshu Zhang, V. Bafna, Robert J Wechsler-Reya, Lukas Chavez
          </td>
          <td>2025-08-19</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs), which replicate and segregate in a non-Mendelian manner, serve as vectors for accelerated tumor evolution. By integrating chromatin accessibility, whole-genome sequencing, and Hi-C-based genome topology data from a cohort of metastatic Castration-Resistant Prostate Cancer (mCRPC) cases, we show that epigenetically activated repeat DNA, amplified in ecDNAs, drive oncogene overexpression. Specifically, we identify a subgroup of mCRPCs (20%) characterized by clusters of accessible LINE1 repeat DNA elements flanking the androgen receptor (AR) gene. These LINE1 elements are co-amplified with AR and provide binding sites for prostate-lineage transcription factors, including AR, FOXA1 and HOXB13. Accessible LINE1 elements establish novel 3D chromatin interactions with the AR gene, forging a new regulatory plexus driving AR overexpression and confers resistance to androgen signaling inhibitors. Our findings indicate how tumor evolution is driven by the convergence of genetic and epigenetic alterations on repeat DNA, activating and amplifying them to allow oncogene overexpression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6c5dce2ec927f5cff069554fa04e5223fd82a5" target='_blank'>
              Genetic and Epigenetic Reprogramming of Transposable Elements Drives ecDNA-Mediated Metastatic Prostate Cancer
              </a>
            </td>
          <td>
            Lisanne Mout, Thaidy Moreno-Rodriguez, Giacomo Grillo, Ankita Nand, Tina Keshavarzian, S. Bahl, Komaldeep Kang, M. Bootsma, Emma Minnee, Stanley Zhou, Kathleen H. Burns, E. Corey, Peter Nelson, S. Dehm, Shuang G Zhao, W. Zwart, Felix Y. Feng, David A Quigley, M. Lupien
          </td>
          <td>2025-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Oncogenes amplified on extrachromosomal DNA (ecDNA) contribute to treatment resistance and poor survival across cancers. Currently, the spatiotemporal evolution of ecDNA remains poorly understood. In this study, we integrate computational modeling with samples from 94 treatment-naive human glioblastomas (GBM) to investigate the spatiotemporal evolution of ecDNA. We observe oncogene-specific patterns of ecDNA spatial heterogeneity, emerging from random ecDNA segregation and differing fitness advantages. Unlike PDGFRA-ecDNAs, EGFR-ecDNAs often accumulate prior to clonal expansions, conferring strong fitness advantages and reaching high abundances. In corroboration, we observe pretumor ecDNA accumulation in vivo in genetically engineered mouse neural stem cells. Variant and wild-type EGFR-ecDNAs often coexist in GBM. Those variant EGFR-ecDNAs, most commonly EGFRvIII-ecDNA, always derive from preexisting wild-type EGFR-ecDNAs, occur early, and reach high abundance. Our results suggest that the ecDNA oncogenic makeup determines unique evolutionary trajectories. New concepts such as ecDNA clonality and heteroplasmy require a refined evolutionary interpretation of genomic data in a large subset of GBMs.


SIGNIFICANCE
We study spatial patterns of ecDNA-amplified oncogenes and their evolutionary properties in human GBM, revealing an ecDNA landscape and ecDNA oncogene-specific evolutionary histories. ecDNA accumulation can precede clonal expansion, facilitating the emergence of EGFR oncogenic variants, reframing our interpretation of genomic data in a large subset of GBMs. See related article by Korsah et al., p. XX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a02add441c27bf4e8e365b8edcace16e00d2cda" target='_blank'>
              Extrachromosomal DNA-Driven Oncogene Spatial Heterogeneity and Evolution in Glioblastoma.
              </a>
            </td>
          <td>
            I. Noorani, Magnus Haughey, J. Luebeck, Andrew Rowan, E. Grönroos, Francesco Terenzi, I. Wong, Davide Pradella, Marta Lisi, Jeanette Kittel, Natasha Sharma, Chris Bailey, C. Weeden, Donald M Bell, Eric Joo, Vittorio Barbè, Matthew G. Jones, King L. Hung, E. Nye, M. Green, Lucy Meader, Emma Norton, Mark Fabian, N. Kanu, M. Jamal-Hanjani, T. Santarius, Andrea Ventura, James A R Nicoll, D. Boche, Howard Y. Chang, V. Bafna, Weini Huang, P. Mischel, C. Swanton, Benjamin Werner
          </td>
          <td>2025-09-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="DNA abnormalities characterized by cytogenetic imaging at the single cell resolution, i.e. karyotyping, have long served as cancer diagnostic and prognostic biomarkers. To enable in-silico karyotyping using unpaired whole-genome sequencing data, we developed Seq2Karyotype (S2K), a tool that fits karyotype models with clonality estimation based on read-depth and allelic imbalance in a bulk sample and supports visualization-guided refinement. Analysis on 19 adult and pediatric cancer cell lines revealed unexpected intratumoral heterogeneity involving multiple copy number variation (CNV) states including whole-genome duplication, which were validated by imaging and single-cell omics profiling. Analyses on patient samples showed high concordance with clinical cytogenetic reports for acute myeloid leukemia, and revealed evolutionary trajectories from multi-region metastatic neuroblastomas implicating reversion. These findings highlight extensive and dynamic intratumoral heterogeneity contributed by CNV in both cell line models and patient samples, which may inform future research on tumor evolution under selective pressure such as drug exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0572920d9d208e3e04940659c463f83c4e0642eb" target='_blank'>
              Seq2Karyotype (S2K): A Method for in-silico Karyotyping Using Single-Sample Whole-Genome Sequencing Data
              </a>
            </td>
          <td>
            Limeng Pu, Karol A. Szlachta, Virginia Valentine, Meiling Jin, Sivaraman Natarajan, Robert Greenhalgh, Nadezhda V. Terekhanova, Jian Wang, Daniel K. Putnam, Li Dong, Lingxiang Jiang, Soumya Tumbath, Xiumei Huang, Xiaotu Ma, Thomas Look, Marcin W. Włodarski, Lu Wang, Steven Burden, J. Easton, Xiang Chen, Jinghui Zhang
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f0c6259c21cf5580851e190063781e3ca6d347" target='_blank'>
              Cancer subclone detection based on DNA copy number in single-cell and spatial omic sequencing data.
              </a>
            </td>
          <td>
            Chi-Yun Wu, Jiazhen Rong, A. Sathe, Paul R Hess, Billy T Lau, Susan M. Grimes, Sijia Huang, Hanlee P Ji, Nancy R. Zhang
          </td>
          <td>2025-09-01</td>
          <td>Nature methods</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Abstract Histone acetylation is widely assumed to directly instruct gene activation. Among acetylated residues, H4K16ac is one of the most abundant modifications, conserved across all eukaryotes. Despite its established role in X-chromosome hyperactivation in Drosophila, its function in mammalian cells has remained elusive. Here, we show that in human somatic cells, H4K16ac does not substantially affect gene expression, but instead controls the spatiotemporal program of genome replication. By combining a meta-analysis of public datasets and perturbation experiments designed to minimize confounding effects, we found that H4K16ac is neither associated with nor required for transcriptional activity. Rather, H4K16ac depletion resulted in premature replication of heterochromatic regions and widespread alterations in replication timing across the genome. These defects were driven by the aberrant activation of cryptic replication origins at long terminal repeats—repetitive elements typically marked by H4K16ac and whose sequence context resembles that of canonical origins in euchromatic regions. Our findings reveal an unexpected role for one of the most prevalent chromatin modifications and uncover a new regulatory mechanism that safeguards genome replication fidelity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58d31c25c7ef175d7c8a34dda15d005ef97496d8" target='_blank'>
              Mammalian H4K16ac regulates the spatiotemporal order of genome replication rather than gene expression
              </a>
            </td>
          <td>
            Marta Milan, Valeria Runfola, Manthan Patel, Lucia Falbo, R. Noberini, C. Soriani, S. Rodighiero, Tiziana Bonaldi, Vincenzo Costanzo, Madapura M. Pradeepa, Paola Scaffidi
          </td>
          <td>2025-09-23</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Increasing evidence underscores the driving role of coding and non-coding variants in cancer development. Analyzing gene sets in biological processes offers deeper insights into the molecular mechanisms of carcinogenesis. Here, we developed geMER to identify candidate driver genes genome-wide by detecting mutation enrichment regions within coding and non-coding elements. We subsequently designed a pipeline to identify a core driver gene set (CDGS) that broadly promotes carcinogenesis across multiple cancers. CDGS comprising 25 genes for 25 cancers displayed instability in DNA aberrations. Variants within the TTN enrichment region may influence the folding of the I-set domain by altering local polarity or side-chain chemistry properties of amino acids, potentially disrupting its antigen-binding capacity in LUAD. Multi-omics analysis revealed that APOB emerged as a candidate oncogene in LIHC, whose genetic alterations within the enrichment region may activate key TFs, upregulate DNA methylation levels, modulate critical histone modifications, and enhance transcriptional activity in the HepG2 and A549 cell lines compared to Panc1. Additionally, CDGS mutation status was an independent prognostic factor for the pan-cancer cohort. High-risk patients tended to develop an immunosuppressive microenvironment and demonstrated a higher likelihood of responding to ICI therapy. Finally, we provided a user-friendly web interface to explore candidate driver genes using geMER (http://bio-bigdata.hrbmu.edu.cn/geMER/).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe1ede7baa4480b19bb1878aa07140d59ffd789" target='_blank'>
              A core driver gene set identified based on geMER reveals its potential driver mechanism in pan-cancer
              </a>
            </td>
          <td>
            Jing Gan, Yuncong Wang, Zhuoran Shi, Haoyu Hu, Manyi Xu, Xinrong Li, Wenbo Dong, Jiaheng He, Yusen Zhao, Yakun Zhang, Yue Sun, Caiyu Zhang, Qianyi Lu, Shangwei Ning, Yan Jin, Hui Zhi
          </td>
          <td>2025-08-09</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d343e029068c438cb11fe609f9a8f46f24c108" target='_blank'>
              Multi-Layered Dysregulation of NRCAM in Gliomas: Insights from TCGA Copy Number and Epigenetic Analyses
              </a>
            </td>
          <td>
            MD Steven Lehrer, MD JD MS Peter H. Rheinstein
          </td>
          <td>2025-08-21</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The interplay between DNA methylation and super-enhancer activity is emerging as a key focus in cancer research. Super-enhancers, a specialized class of enhancers, strongly activate transcription of their target genes due to their dense clustering with essential transcription factors (TFs) and cofactors. In cancer, especially, these super-enhancers control key oncogenic drivers and often display abnormal DNA methylation patterns that can repress or overexpress target genes in both solid and blood cancers. Furthermore, DNA methylation in the super-enhancer region has been found to influence their regulatory capacity. Although enhancers are typically characterized by low DNA methylation, dysregulated methylation at super-enhancers is seen in most malignancies, affecting TF and chromatin regulator recruitment. Hypomethylation at these sites often accompanies oncogene hyperactivation, while hypermethylation can repress tumor suppressor mechanisms. Recent research highlights DNA methylation as a promising source of cancer biomarkers. This review examines the intricate relationship between DNA methylation and super-enhancer activity in cancer, concentrating on how methylation regulates super-enhancers, modulates oncogene expression, promotes oncogenesis, and serves as a target for novel oncology therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80856f95cde56634ccf673f640abf635b4018a59" target='_blank'>
              Super-enhancer DNA methylation in cancer: the mechanism of action and therapeutic directions
              </a>
            </td>
          <td>
            Yue Bi, Kang Zou, Ke Wang
          </td>
          <td>2025-10-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="OncoDB was initially developed to advance cancer research by integrating RNA expression profiles, DNA methylation patterns, clinical annotations, and oncoviral signatures derived from the Cancer Genome Atlas (TCGA) and Genotype-Tissue Expression datasets. We now present OncoDB 2.0, an extensively expanded platform that offers a more comprehensive and integrated view of cancer omics. The updated version includes an atlas of somatic mutations discovered from combined DNA and RNA sequencing, enabling in-depth investigation of mutation patterns across tumor types and their association with clinical features. Furthermore, we have integrated proteomic data from the Clinical Proteomic Tumor Analysis Consortium (CPTAC) and chromatin accessibility data from TCGA, offering new dimensions for oncogene regulation studies. OncoDB 2.0 also introduces advanced multi-omics analysis modules that facilitate the combined exploration of RNA expression, DNA methylation, and somatic mutations, allowing researchers to examine complex cross-omic relationships with greater depth and flexibility. Together, these enhancements make OncoDB 2.0 a robust and invaluable tool for the cancer research community. OncoDB 2.0 is freely available at https://oncodb.org.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2597afaecc32114e7746cc65fb5c9507bfe36bf0" target='_blank'>
              OncoDB 2.0: a comprehensive platform for integrated pan-cancer omics analysis.
              </a>
            </td>
          <td>
            Minsu Cho, Gongyu Tang, Charles S Rogers, Makayla Dove, Xinyi Liu, Yuanxiang Li, Xiaowei Wang
          </td>
          <td>2025-09-23</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Mouse heterochromatin is characterized by A/T-rich, 234 bp DNA repeat arrays, called major satellite repeats (MSR). We investigated MSR expression in response to a variety of stress conditions by using small molecule compounds. We identified the isoflavone genistein to selectively stimulate MSR transcription, but not that of other DNA repeat elements. Genistein is a natural compound that is frequently used in dietary supplements and has been associated with reducing cancer risk. A 24 h exposure of mouse embryonic fibroblasts (MEF) to genistein results in a more than 100-fold induction of MSR transcripts. This up-regulation depends on the activity of RNA polymerase II and requires a cycling G1 cell population. Blocking the cell cycle at the G2/M stage significantly attenuates genistein-mediated stimulation of MSR transcription. Mechanistically, DNA topoisomerase poisons phenocopy the genistein-dependent up-regulation of MSR expression. Together, these data suggest that MSR transcriptional response is guided by an altered topology of the underlying A/T-rich MSR DNA repeat arrays and reveal a novel function for genistein that may contribute to the anticancer properties of this natural compound. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00623-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64fed7546750b813d379740d82aa6947375846a7" target='_blank'>
              The isoflavone genistein selectively stimulates major satellite repeat transcription in mouse heterochromatin
              </a>
            </td>
          <td>
            Thomas Fuhrmann, N. Shukeir, Reagan W. Ching, Galina Erikson, Yuan Dou, Zoe Sawitzki, Megumi Onishi-Seebacher, Carmen Galán, Thomas Jenuwein
          </td>
          <td>2025-08-25</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Objective The diagnosis of precancerous lesions of colorectal cancer (CRC) presents significant challenges in clinical practice. In this study, we conducted a clinical investigation using the UCAD technique after analyzing chromosomal copy number variations (CNVs) in formalin-fixed, paraffin-embedded (FFPE) samples from various pathological stages, aiming to evaluate the value of detecting chromosomal instability (CIN) in CRC diagnosis. Methods Based on colonoscopic pathological findings, we selected 39 FFPE specimens of tubular adenomas, 8 FFPE specimens of villous adenomas, 16 cases diagnosed as tubular-villous adenomas, and 14 cases without defined pathological subtype classification. The UCAD technique was employed to analyze these specimens, with the objective of delineating differences in chromosomal instability among the various pathological subtypes. Results UCAD analysis confirmed that among 39 patients diagnosed with tubular adenomas, 12 (30.76%) exhibited CIN positivity, primarily characterized by amplifications of chromosomal segments on 13q, 7, and 8, and losses on 18q and 14q. In the 8 patients diagnosed with villous adenomas, 6 (75%) were CIN-positive, displaying amplifications at 13q, 7, 8q, and 20, along with losses at 18q and 14q. Among 16 patients diagnosed with tubular-villous adenomas, 8 (50%) demonstrated CIN positivity. Additionally, 8 out of 14 cases lacking a defined pathological subtype were CIN-positive. Conclusion The assessment of CIN correlates with both pathological subtypes and disease progression. UCAD-based detection of CIN contributes to the diagnosis of colorectal adenomas (CRA), with aberrations in chromosomes 7 and 8 potentially being closely associated with PLCRA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db3a5e74d13312e81a68afaaa89165cdbc3cca06" target='_blank'>
              Application of low-coverage whole-genome sequencing technology in risk stratification of colorectal adenomas
              </a>
            </td>
          <td>
            Guo Zhili, Xue Yuyue, Yu Fang, Dianqun Ren, Zhang Qin, Liu Jie
          </td>
          <td>2025-08-22</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80c7c66e7f054de7552ccd4ec8d0186447c75c2a" target='_blank'>
              Single-cell Genomic Copy Number Evolution Reveals Frequent Loss of the Y chromosome in Esophageal Adenocarcinoma.
              </a>
            </td>
          <td>
            Shiwei Yin, Na Zhong, Swati Agrawal, Bingru Feng, Sneha Pradhan, Nathan K. Jobalia, Maryam Vaziripour, Sampath K Poreddy, L. Kresty, Erin A Gillaspie, Nicholas E Dietz, W. Al-Refaie, K. Nandipati, Yusi Fu, Jun Xia
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb14185df59742e1c30589cf1835a68e0538e9b9" target='_blank'>
              Tracking clonal evolution during treatment in ovarian cancer using cell-free DNA.
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Duaa H Al-Rawi, Gryte Satas, Hannah Lees, J. Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, Neeman Mohibullah, Yulia Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitry Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2025-10-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a lethal cancer of the bile duct and is a major health concern in several parts of the world, including northeastern Thailand, where CCA incidence is the highest due to the endemic liver fluke Opisthorchis viverrini. Multiple studies have characterised genomic alterations in CCA tumours, and specific chromosomal alterations can predict prognosis. However, it is not known whether chromosomal instability (CIN), ongoing genomic alteration characteristic of most cancer types, is present in CCA tumours. In this study we leveraged a panel of cancer cell lines derived from fluke-positive CCA patients, as well as a matched normal cholangiocyte line as a control, to characterise CIN in CCA. We observed elevated rates of chromosome segregation errors compared to normal cells, although overall CIN rates were lower than those for highly genomically unstable cancers, such as colorectal or ovarian cancer. Chromosome segregation errors in CCA cell lines were potentially driven by elevated DNA replication stress and centrosome duplication. Single-cell genome sequencing and karyotyping of the cell lines showed extensive structural and numerical chromosomal aberrations, as well as copy number alterations that were heterogeneous between individual cells, supporting the presence of ongoing CIN in these cell line models. Low-pass whole-genome sequencing of 33 CCA tumour samples with matched normal tissue from northeastern Thailand, a liver fluke-endemic region showed increased whole and subchromosomal level alterations, with a higher extent of genomic alterations in intrahepatic tumours compared to extrahepatic. Eight tumours carried focal amplifications and/or deletions involving known cancer genes, as well as potential chromosomal instability-associated genes, including CCNE1 amplifications and a rare amplification of BRCA1. This study provides increased understanding of the rate and potential mechanisms of CIN in CCA that may inform new therapeutic strategies that synergise with specific ongoing CIN mechanisms. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4433abbfa4bf430e31ec8b07fb91e4f3d1fdbb22" target='_blank'>
              Chromosomal instability and genomic alterations in cholangiocarcinoma from Northeastern Thailand.
              </a>
            </td>
          <td>
            Raksawan Deenonpoe, MA Guscott, Sasithorn Watcharadetwittaya, Isabel L McNeill, D. Moralli, N. Shaikh, SE McClelland
          </td>
          <td>2025-09-15</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Somatic human cells contain a diploid genome consisting of 23 pairs of chromosomes. The maintenance of this diploid state is essential across all layers of biological organization, ranging from the physiology of individual cells to the proper regulation of tissue homeostasis and organismal development. Most cancer cells, however, harbor an aneuploid genome with an abnormal number of chromosomes, including whole and/or partial chromosome gains and losses. These alterations arise as a consequence of mitotic chromosome segregation errors and/or ongoing chromosomal instability (CIN). While aneuploidy usually imposes a fitness cost to nontransformed cells, certain recurrent aneuploidies confer adaptive advantages that are subjected to positive selection throughout tumorigenesis. In this review, we discuss how aneuploidy impacts cellular physiology, fitness, and adaptability in the context of cancer development. We also examine how the aneuploid state and CIN can create vulnerabilities that may be exploited for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb17fd1a7e9ab93107d809a69e7ad9c212a84ff9" target='_blank'>
              Impact of Chromosomal Instability and Aneuploidy in Cancer Development
              </a>
            </td>
          <td>
            Amanda K. Mennie, Brittiny Dhital, Peter Ly
          </td>
          <td>2025-09-30</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The distinctive three-dimensional (3D) chromatin architecture adopted by chromosomes during meiosis is essential for fertility, yet the functional implications of the fine-scale spatial organization remain poorly understood. Here, we investigate the impact of RAD21L deletion, a meiosis-specific cohesin subunit, on the 3D genome architecture and gene expression in the male germ line. Using fluorescence-activated cell sorting, high-throughput chromosome conformation capture, and single-cell RNA sequencing, we demonstrate that RAD21L deficiency impairs meiotic chromatin organization and inter/intrachromosomal interactions. This conspicuous 3D genome reorganization also disrupts bouquet formation, resulting in increased telomeric interactions between heterologous chromosomes in primary spermatocytes. Genome reorganization was accompanied by detectable transcriptional dysregulation in spermatogonia and primary spermatocytes, mainly affecting sex chromosomes. Collectively, our findings establish the cohesin RAD21L as a germline-specific critical regulator of genome-wide 3D genome reorganization during spermatogenesis and a modulator of the male germline transcriptional landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2272faa3ad286c58e292f1de1607d234286a48c" target='_blank'>
              Meiotic cohesin RAD21L shapes 3D genome structure and transcription in the male germline
              </a>
            </td>
          <td>
            Laia Marín-Gual, Covadonga Vara, Raquel Sainz-Urruela, Yasmina Cuartero, Lucía Álvarez-González, Natalia Felipe-Medina, Francisca Garcia, E. Llano, M. Martí-Renom, A. Pendás, A. Ruiz-Herrera
          </td>
          <td>2025-10-01</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Functional inactivation of tumor suppressor genes drives cancer initiation, progression, and treatment responses. Most tumor suppressor genes are inactivated through 1 of 2 well-characterized mechanisms: DNA-level mutations, such as point mutations or deletions, and promoter DNA hypermethylation. Here, we report a distinct third mechanism of tumor suppressor inactivation based on alterations to the histone rather than DNA code. We demonstrated that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event in more than 80% of human endometrial cancers. Integrative transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that PAX2 transcriptional downregulation is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive chromatin features (H3K27me3) in a framework dictated by 3D genome organization. The spread of the repressive H3K27me3 signal resembled a pearl necklace, with its length modulated by cohesin loops, thereby preventing transcriptional dysregulation of neighboring genes. This mechanism, involving the loss of a promoter-proximal superenhancer, was shown to underlie transcriptional silencing of PAX2 in human endometrial cancers. Mouse and human preclinical models established PAX2 as a potent endometrial tumor suppressor. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. The discovery that most endometrial cancers originate from a recurring epigenetic alteration carries profound implications for their diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5897c9bf68c93e4fb62e86f78e7178d27155eec3" target='_blank'>
              A distinct mechanism of epigenetic reprogramming silences PAX2 and initiates endometrial carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad R. Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2025-08-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Transcription is a core hallmark of cancer, wherein many different proteins assemble at specific sites in the nucleus and act in concert to transcribe functionally relevant genes. Central to this process are transcription factors that bind to their cognate DNA motifs on enhancers and super-enhancers to recruit cofactors, coactivators, and epigenetic modifiers, thereby inducing or repressing gene expression. Super-enhancers drive oncogenic transcription, to which cancer cells become highly addicted and confer tumor dependencies on super-enhancer-driven transcription machinery. Transcriptional condensates (TCs) are nuclear membrane-less assemblies of DNA-binding transcription factors, transcription co-activators, and the transcriptional machinery (such as RNA polymerases, non-coding RNAs) formed through liquid–liquid phase separation (LLPS). The function of transcriptionally active oncogenic proteins and their interplay with nucleic acids are carried out within these biomolecular condensates, allowing them to spatiotemporally regulate oncogene expression and lead to the induction and maintenance of cancer. With this growing understanding, specific inhibitors and strategies targeting TC assembly and activation should be considered promising therapeutic opportunities for treating various tumors, including hematological malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd11a72e245b5549ddbe26e0421c105063b1fbf1" target='_blank'>
              Transcriptional Condensates: Epigenetic Reprogramming and Therapeutic Targets in Hematologic Malignancies
              </a>
            </td>
          <td>
            Kevin Qiu, Qing Yin, Chongzhi Zang, Jianguo Tao
          </td>
          <td>2025-09-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Most advanced melanomas initially respond to targeted therapy but eventually relapse. Rather than acquiring new mutations, resistance is driven by drug-tolerant persister cells that enter a reversible drug-refractory state. We developed MeRLin, a high-resolution lineage tracing platform integrating cellular barcoding, single-cell transcriptomics, RNA fluorescence in situ hybridization (FISH), and computational analyses to track clonal and transcriptional dynamics in patient-derived melanoma models during prolonged therapy. Clonal dynamics revealed that persister subpopulations first responded to treatment but persisted and expanded during minimal residual disease, ultimately leading to tumor recurrence. Pre-treatment melanoma populations diversified into four conserved persister states characterized by stress-like, lipid metabolism, PI3K signaling, and extracellular matrix remodeling programs associated with adaptive resistance. Spatial transcriptomics showed the organization of these adaptive programs and a complex signaling network of autocrine and paracrine interactions among persister subpopulations. Barcoded RNA-FISH enabled spatial mapping of clonal identity and gene expression, revealing in situ co-localization of a dominant resistant clone with SLC2A1 expression. MeRLin provides a robust framework for dissecting cancer heterogeneity and identifying vulnerabilities in persister populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c4ead317fd2f11125da9ef801fb0c7bcc0217f3" target='_blank'>
              Clonal dynamics shaped by diverse drug-tolerant persister states in melanoma resistance
              </a>
            </td>
          <td>
            Haiyin Li, Yeqing Chen, Jessica Kaster, Maggie Dunne, Min Xiao, Ling Li, Monzy Thomas, Nazifa Promi, Dylan Fingerman, G. Brown, Qiuxian Zheng, Xingyue Zhu, McKenna Reale, Andrew Patterson, Le Gao, Xuxiang Zhang, Siqi Jiang, Tianxing Hu, Hanzhang Fang, Jianlan Ren, Cong Qi, Luyang Wang, Haiwei Mou, Gatha Thacker, Eric Ramirez Salazar, J. Villanueva, Arjun Raj, Dave SB Hoon, Tian Bin, Jozef Madzo, Zhi Wei, Noam Auslander, M. Herlyn
          </td>
          <td>2025-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84b7fddece1de57c4f545b96bbc40fa5c54a7ad7" target='_blank'>
              Enhancer regulation in cancer: from epigenetics to m6A RNA modification.
              </a>
            </td>
          <td>
            Arathy Vasukutty, Poshan Yugal Bhattarai, Hong Seok Choi
          </td>
          <td>2025-08-19</td>
          <td>Archives of pharmacal research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/952548d2d250db0e8e23c4ea5a20ae1aba4ce302" target='_blank'>
              Coregulators determine androgen receptor activity in prostate cancer.
              </a>
            </td>
          <td>
            Kerim Yavuz, Nathan A. Lack
          </td>
          <td>2025-08-20</td>
          <td>Bioscience reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Significance This study highlights a specific source of endogenous DNA replication stress, G-quadruplexes (G4s), that can increase genome instability in BRCA2-deficient cerebellar granule cell progenitors (GCPs), leading to medulloblastoma. BRCA2 is necessary for maintenance of replication speed at G4s in GCPs, and its absence allows these structures to become hotspots of increased mutagenesis. The G4-unwinding PIF1 helicase was found to be upregulated in tumor cells as a potential way to cope with G4-related replication stress. Targeting PIF1 may represent a therapeutic strategy for BRCA2-deficient medulloblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba0cd9b13650ad33751efc7afb07cdd2a585f9d5" target='_blank'>
              G-quadruplexes as a source of vulnerability in BRCA2-deficient granule cell progenitors and medulloblastoma
              </a>
            </td>
          <td>
            Danielle L. Keahi, Mathijs A. Sanders, M. Paul, A. L. Webster, Yin Fang, Tom Wiley, Samer Shalaby, Thomas S. Carroll, S. Chandrasekharappa, C. Sandoval-Garcia, Margaret L MacMillan, John E. Wagner, M. Hatten, A. Smogorzewska
          </td>
          <td>2025-08-25</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Background BRD9 is involved in multiple physiological and pathological pathways, yet its functional role and molecular mechanisms in gastric cancer (GC) remain largely unexplored. Addressing this knowledge gap is critical given the persistent global mortality burden of GC and the limited efficacy of current therapeutic strategies. Methods BRD9 expression in GC patients was systematically analyzed using immunohistochemical (IHC) assays and transcriptomic datasets. Comprehensive functional validation, employing cellular and murine tumor models, elucidated BRD9’s role in GC progression. Molecular pathways underlying BRD9-mediated gastric carcinogenesis were delineated through integrated approaches, including RNA sequencing, co-immunoprecipitation (co-IP), subcellular fractionation, and luciferase reporter assays. Results BRD9 was significantly overexpressed in GC and associated with poor patient prognosis. Functionally, BRD9 promoted GC cell proliferation and enhanced DNA damage repair capacity. Mechanistically, elevated BRD9 expression inhibited p53 nuclear translocation via direct binding, subsequently activating the E2F transcription factor family. Notably, we identified that E2F1 directly bound to and transactivated the BRD9 promoter, establishing a positive feedback loop that sustains BRD9 expression. Additionally, BRD9 knockdown sensitized GC cells to cisplatin and oxaliplatin treatment. Conclusions These findings highlight the critical role of BRD9 in GC progression and its therapeutic potential. The BRD9-p53-E2F1 axis acts as a crucial regulator of GC cell proliferation and DNA damage response. Targeting BRD9 pharmacologically could be a novel therapeutic approach to enhance chemotherapy efficacy and improve treatment outcomes in GC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02449-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dadff4be7ebdf814a21760a8db20acee07e5dfa" target='_blank'>
              BRD9-p53-E2F1 circuit orchestrates cell growth and DNA damage repair in gastric cancer
              </a>
            </td>
          <td>
            Qingqing Zhou, Qi Wang, Yantao Duan, Chi Zhang, Tengfei Liu, Hengrui Liu, Mindi Zhao, Zhihuang Hu
          </td>
          <td>2025-10-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinomas (HGSOC) have one of the highest burdens of focal copy number alterations amongst cancers studied by The Cancer Genome Atlas. These focal copy number alterations are driven by chromosomal instability and rearrangements. Our current understanding of structural variants (SVs) in HGSOC come largely from a combination of macroscale karyotyping and base-level short-read Illumina sequencing. However, short-read sequencing is inherently hindered in accurate SV calling due to limited information of exact breakpoints captured using this technology. With the development of long-read sequencing, including Oxford Nanopore Technologies (ONT) sequencing used here, SVs are resolved with higher accuracy. Additionally, ONT sequencing allows better haplotype phasing and information about 5-methylcytosine (5mC; gene silencing) and 5-hydroxymethylcytosine (5hmC; gene activation). Here we sought to examine SVs and allele-specific methylation (ASM) in HGSOC cell lines, organoids, and tumors. Our cell line data, from 7 HGSOC cell lines, sequenced the genome to an average coverage of ∼35x. Using the epi2me human variation pipeline (ONT), supplemented by additional long-read SV caller cuteSV and filtering with MAVIS, we identified Insertions, Deletions, Translocations, Duplications, and Inversions. Because our samples lacked germline calls, we leveraged coverage information from the gnomAD database across the Ten-Thousand Genomes Project to remove regions of frequent variation in the population, which greatly filtered the number of insertions and deletions. A strong peak of 300bp was observed in insertions and deletions, which was nearly entirely repetitive elements (SINE, LINE, LTR, and simple repeats), making up over 50% of these SVs. MAVIS annotates SVs predicted to make non-synonymous changes to protein sequences; of 95 non-synonymous SVs across 7 cell lines, 57 were examined by PCR, with successful validation of 85.9%. Long-read sequencing was also haplotype phased, and DSS was used to determine regions of differential methylation between haplotypes. A mean of 828 promoters with ASM were observed, with about 40% shared between at least two cell lines. While 91 different tumor suppressor genes (TSG) had promoters with ASM were identified (mean 25 TSG per cell line), none were confirmed to have a mutation to the expressed haplotype. We are continuing to confirm whether ASM leads to bias in allelic expression of RNA, as well as expanding our study to examine long-read sequencing in organoids and tumor datasets. This work adds granularity to existing data on chromosomal rearrangements in HGSOC. Future directions of this work will examine how translocations between chromosomes influence novel enhancer-gene interactions.



 Jessica D. Lang, Ryan Moreno, Kristin M. Adams, Lauren Kersten. Long-read sequencing of high-grade serous ovarian carcinoma genomes: Insights into structural variants and allele-specific DNA methylation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6303026df4b6b8b5a1d48c24d59b28bf2267c15" target='_blank'>
              Abstract B010: Long-read sequencing of high-grade serous ovarian carcinoma genomes: Insights into structural variants and allele-specific DNA methylation
              </a>
            </td>
          <td>
            Jessica D. Lang, Ryan Moreno, Kristin M Adams, Lauren Kersten
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53f0d40328372c75f3f03439d0b2ad59f2263c92" target='_blank'>
              Transcriptional regulation of Ligase IV by an intronic regulatory element directs thymocyte development.
              </a>
            </td>
          <td>
            Matthew D Estrada, Christopher J Gebhardt, Mariam A Salem, Christina N Rau, Kruthika Sharma, Rebecca A Glynn, Craig H Bassing, Eugene M Oltz, Patrick L. Collins
          </td>
          <td>2025-09-05</td>
          <td>Genes and immunity</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Breast cancer (BC) remains one of the most prevalent cancers worldwide and a significant cause of cancer-related mortality among women. Despite significant advancements in understanding the genetic foundations of BC, numerous research initiatives have historically focused on protein-coding genes, which constitute merely about 2% of the human genome. This focus has produced significant insights into oncogenes such as HER2, TP53, and BRCA1, along with tumor suppressor genes. Nonetheless, it has led to the non-coding portions of the genome garnering relatively less focus. Recent studies illuminate the crucial significance of non-coding DNA in cancer biology, highlighting its regulatory roles and influence on tumor formation, metastasis, and treatment resistance. This review examines the importance of non-coding DNA in BC. It provides an in-depth analysis of essential non-coding regions, their functions in gene regulation and chromatin structure, and their implications for various BC subtypes. Examining these facets, we seek to reveal the potential of non-coding DNA as a viable source of novel diagnostic markers and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966867774ecd7f0803dece977fed36246e346fe6" target='_blank'>
              The Role of Non-Coding Regions in Breast Cancer: From Gene Regulation to Therapeutic Implications
              </a>
            </td>
          <td>
            Hussein Sabit, Sarah Sobhy, Shaimaa Abdel-Ghany, Al-Hassan Soliman Wadan, Olubukola Ayodele, Yasser Albrahim, Hirendra Banerjee, Ahmed Elhashash, B. Arneth
          </td>
          <td>2025-09-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="While the coding genome of pancreatic cancer has been characterized in detail, features of the noncoding genome remain relatively unexplored. We used whole genome sequencing to study the coding and noncoding genome (promoters, enhancers and noncoding-NOS) in two unique patient cohorts. We find that treated cancers have a significantly higher mutational burden than untreated cancers in all four genomic regions. However, the relative proportion of mutations in each region are preserved despite treatment-induced genetic bottlenecks. Compared to other noncoding regions enhancers have a lower number of mutations/Mb. Enhancers also have a distinct mutational signature with enrichment of SBS39. Enhancer sequences were segregated into conserved and nonconserved regions based on the overlap of predicted orthologous regions in the human and mouse genomes and the conserved regions screened for recurrent somatic mutations. We find recurrent somatic mutations in conserved enhancer regions largely correspond to those associated with known transcription factors with a role in pancreatic development and cancer including KLF5 and TP63. Transcriptional expression based on RNA-seq data of cancers with enhancer mutations showed significantly different levels of expression, most often a loss of expression, compared to cancers without enhancer mutations suggesting a functional effect. These findings expand our knowledge of the noncoding genome in pancreatic cancer and point to an unexplored role of conserved enhancer mutations for pancreatic cancer as well.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edbe76ca8bf694ac9c6f9820ca8a2fe03fe38724" target='_blank'>
              The Non-Coding Mutational Landscape of Pancreatic Cancer Reveals Recurrent Somatic Mutations in Enhancer Regions.
              </a>
            </td>
          <td>
            Akimasa Hayashi, Yu-Jui Ho, Alvin P. Makohon-Moore, Amanda Zucker, Jungeui Hong, Shigeaki Umeda, Elias-Ramzey Karnoub, Jinlong Huang, Priscilla Baez, R. Kappagantula, Jerry P Melchor, Wungki Park, Eileen M. O’Reilly, Nicholas D Socci, Shinya Oki, Christine A. Iacobuzio-Donahue
          </td>
          <td>2025-09-19</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is a primary cause of therapeutic failure, driving tumor progression. Within tumors, diverse cell states coexist, maintained by a specific chromatin landscape that influences various cell functions, including cancer stemness. Among factors that induce chromatin changes affecting cell state fitness, DNA damage and its repair have emerged as significant contributors. This perspective examines recent advances that elucidate the interplay between DNA repair, epigenome, and cell plasticity. We discuss how epigenome affects DNA repair and, conversely, how DNA repair-induced chromatin changes influence cell plasticity. Finally, we discuss emerging concepts and highlight the therapeutic implications of these interconnected mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0841716149f708372bcf7e59a0f69e9d199da1" target='_blank'>
              The interplay of DNA damage, epigenetics and tumour heterogeneity in driving cancer cell fitness
              </a>
            </td>
          <td>
            Celia D Rouault, E. Charafe-Jauffret, C. Ginestier
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Polycomb domains safeguard cell identity by maintaining lineage-specific chromatin states enriched in repressive histone modifications, preserving the epigenetic memory of cell lineages. While Polycomb Repressive Complex 2 (PRC2) can re-establish its occupancy after perturbation, the mechanisms that guide de novo Polycomb recruitment remain unclear. To address this, we engineered an auxin-inducible degradation system to reversibly deplete and reintroduce the endogenous PRC2 core subunit Suz12 in mouse embryonic stem cells (mESCs). Genome-wide profiling at an early recovery time point revealed ~1,100 PRC2 nucleation sites, characterized by rapid Suz12 and histone H3K27me3 re-accumulation with strong signal, with minimal impact on gene expression. These sites were significantly enriched at bivalent promoters, coinciding with unmethylated CpG islands and chromatin states associated with developmental regulation, and were largely conserved in differentiated cells. Motif analysis identified G/C-rich DNA sequences associated with E2F and zinc-finger proteins, alongside strong co-occupancy with MTF2 and JARID2, two PRC2 cofactors previously implicated in Polycomb targeting. Notably, a subset of nucleation sites overlapped with long-range chromatin interaction anchors in histone H3K27me3 HiChIP datasets. These findings reveal that PRC2 de novo nucleation sites are associated with a combination of chromatin states, DNA sequence features, cofactor co-occupancy and spatial genome organization, suggesting that epigenetic memory can be re-established through defined genomic and chromatin features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6acdb65cd9855008ca98cbba75e663b43fcb1515" target='_blank'>
              Mapping early PRC2 nucleation sites upon Suz12 reintroduction reveals features of de novo Polycomb recruitment
              </a>
            </td>
          <td>
            I. Hernández-Romero, C. A. Peralta-Alvarez, Abraham Román-Figueroa, Nallely Cano-Domínguez, Maribel Soto‐Nava, Hongwei Zhou, Xin Huang, Félix Recillas-Targa, A. C. Poot-Hernández, Mayra Furlan-Magaril, Santiago Ávila-Ríos, David Valle-Garcia, Jianlong Wang, V. J. Valdés
          </td>
          <td>2025-09-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="R-loop is a special DNA–RNA hybrid genomic structure. Since its identification, the molecular mechanisms of physiological processes such as class switch recombination have been revealed, uncontrolled regulation of which has become the underlying cause of many diseases. With the development of molecular biology techniques, R-loops found at different sites and formed via different mechanisms have been discovered. These non-classical R-loops participate in various cellular activities via different mechanisms. A set of sophisticated regulatory mechanisms are required to control the number of various R-loops in vivo. The levels of non-classical R-loops in tumor cells differ significantly from those in normal cells; in addition, the regulatory mechanisms for establishing R-loop homeostasis differ, which may be utilized to develop breakthrough therapies for various tumors. In this review, we summarize the current state of knowledge regarding non-classical R-loops in tumor cells, the mechanisms that promote or inhibit this structure, the effects of this structure on tumor cells, and the possible therapeutic targets. By systematically elucidating the pathogenic mechanisms of atypical R-loops, we can achieve precision targeting of tumors and revolutionize clinical precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676d0195748900abca38448fb2ef5985171fe4bf" target='_blank'>
              Atypical R-loops in cancer: decoding molecular chaos for therapeutic gain
              </a>
            </td>
          <td>
            Yuan Sun, Sheng Wang, N. Ge, Jintao Guo, Guoxin Wang, Fan Yang, Siyu Sun
          </td>
          <td>2025-08-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 The genomic landscape of ovarian high grade serous carcinoma (HGSC) is typified by extensive genomic instability and heterogeneity, near-ubiquitous TP53 mutations, copy number aberrations, and defects in homologous recombination (HR) repair in approximately 50% of patients. Due to different mechanisms underlying therapy resistance and a lack of predictive biomarkers, matching best treatment modalities to patients with HGSC is challenging. Assessing genomic instability scores to indicate HR-deficiency (HRD) status is used clinically to guide optimal maintenance regimens including PARP-inhibitor treatment. Recently, we demonstrated the degree of phenotypic and genomic heterogeneity in advanced, high tumor burden HGSC patients. Here, we aim to show that the spatial inter-tumoral heterogeneity in HGSC expands to variability in whole genome duplication (WGD) states and timing, and in HRD-score derivation. Systematic mapping of tumor dissemination patterns was performed in chemotherapy-naïve advanced HGSC patients (n=22) following maximal-effort cytoreduction. Multi-site biopsies from 4-15 different anatomical sites were collected, and paired relapse samples were collected where possible (n=10). Germline and Tumor DNA was extracted (5 tumors per case, plus all relapse samples) and underwent whole genome sequencing (WGS, ∼30x depth). We observed spatial and temporal genomic heterogeneity for common HGSC features including mutations, gene amplifications (e.g. CCNE1 and MYC), and copy number and mutational signatures. Somatic, pathogenic loss-of-function TP53 mutations were observed in all tumors in the cohort. We observed discordance in HRDsum classification measures, with 4/22 patients presenting with mixed-HRD scores, i.e. both HRD-positive and HRD-negative scores in their tumors. In each case, the HRDsum scores had a mean value close to the threshold of 42 (range 35.8-45.8). HRD-negative or mixed-HRD status patients had poorer progression free (p=0.0026) and overall survival (p=0.0015) than HRD-positive patients. Cases with mixed states for WGD (no WGD, one WGD or second sub-clonal WGD) were observed in 41% of the cohort with variable timings (early vs late) within disseminated tumors across the cohort, in patients with HRD-negative and mixed HRD-status. The significant inter-tumoral genomic diversity observed in our analysis in measures of HRD and WGD status across the various tumor sites in our cohort of advanced HGSC patients highlights the challenges facing clinicians to derive accurate clinically actionable or prognostic information from a single tumor biopsy alone to direct patient treatment.



 Paula Cunnea, Nikki Burdett, Marc Lorentzen, Ahwan Pandey, James Clark, Katherine Nixon, Jennifer Ploski, Shirya Varghese, David Bowtell, Jonathan Krell, Elizabeth Christie2, Christina Fotopoulou. Evaluating spatial inter-tumor heterogeneity in whole genome duplication states and homologous recombination deficiency status in advanced ovarian high grade serous carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28933fb2fad0d299521abc6704fd16fc40c4a93c" target='_blank'>
              Abstract B004: Evaluating spatial inter-tumor heterogeneity in whole genome duplication states and homologous recombination deficiency status in advanced ovarian high grade serous carcinoma
              </a>
            </td>
          <td>
            P. Cunnea, Nikki L Burdett, Marc Lorentzen, A. Pandey, James Clark, K. Nixon, J. Ploski, Shirya Varghese, D. Bowtell, J. Krell, Elizabeth L. Christie, C. Fotopoulou
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Tumor suppressor NF1 is recurrently mutated in glioblastoma, leading to aberrant activation of Ras/rapidly accelerated fibrosarcoma (RAF)/MEK signaling. However, how tumor heterogeneity shapes the molecular landscape and efficacy of targeted therapies remains unclear. Here, we combined bulk and single-cell genomics of human somatic NF1-mutant, isocitrate dehydrogenase (IDH) wild-type glioblastomas with functional studies in cell lines and mouse intracranial tumor models to identify mechanisms of tumor heterogeneity underlying clinical outcome and MEK inhibitor response. Targeted DNA sequencing identified CDKN2A/B homozygous deletion as a poor prognostic marker in somatic NF1-mutant, but not NF1 wild-type, glioblastoma. Single-nucleus RNA sequencing of human patient NF1-mutant glioblastomas demonstrated that mesenchymal-like (MES-like) tumor cells were enriched for MEK activation signatures. Single-cell RNA-sequencing of mouse intracranial glioblastomas treated with the MEK inhibitor selumetinib identified distinct responses among tumor subpopulations. MEK inhibition selectively depleted MES-like cells, and selumetinib-resistant MES-like cells upregulated Ras signaling while resistant non-MES cells expressed markers of glial differentiation. Finally, genome-wide CRISPR interference screens validated Ras/RAF/MEK signaling as a key mediator of selumetinib response. Repression of the RAF regulator SHOC2 sensitized glioblastomas to selumetinib in vitro and in vivo, suggesting a synergistic treatment strategy. Taken together, these results highlighted the heterogeneity of NF1-mutant glioblastomas and informed future combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9b553a3b7f2afede17ca71f5d4515ccec597b2" target='_blank'>
              Tumor heterogeneity underlies clinical outcome and MEK inhibitor response in somatic NF1-mutant glioblastoma
              </a>
            </td>
          <td>
            S. Pan, K. Mirchia, E. Payne, S. J. Liu, N. Al-Adli, Zain Peeran, Poojan D. Shukla, Jacob S. Young, Rohit Gupta, Jasper Wu, J. Pak, T. Ozawa, Brian Na, Alyssa T Reddy, Steve E Braunstein, Joanna J. Phillips, Susan Chang, D. Solomon, A. Perry, D. R. Raleigh, Mitchel S Berger, Adam Abate, H. Vasudevan
          </td>
          <td>2025-09-23</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Although there are known sex differences in cancer incidence, severity, and treatment, the sex chromosomes are typically excluded from genomic analyses because of the unique technical challenges associated with assessing their copy number, sequence variation, and expression. Here we assess sex chromosome complement in three widely-used human genomics datasets from normal (non-cancerous) tissues, primary tumors, and cancer cell lines and study the effects on genome-wide gene expression. Expected sex chromosome complements based on reported patient sex were observed in non-cancerous tissues, but about half of tumors and cancer cell lines showed loss of typical sex chromosome gene expression across tissue types with three categories: loss of chromosome Y (LOY), loss of chromosome X (LOX) and reactivation of the inactive X chromosome (XaXa). Genes consistently differentially expressed in tumors with loss of chromosome X, loss of chromosome Y, or loss of X chromosome inactivation are associated with the hallmarks of cancer and include both sex-linked and autosomal genes from nearly all chromosomes, druggable genes, and genes with molecular functions relevant to cancer signaling, such as kinase activity. Strikingly, tumors that are X0, including tumors from female patients that have lost an X chromosome and tumors from male patients that have lost a Y chromosome, cluster together by gene expression profile. Patients with tumors that have LOX or LOY had poorer survival outcomes compared to those with tumors that had maintained their sex chromosome complement. Further, LOX and LOY eliminates nearly all of the differential gene expression between tumors from different patient sexes, affecting sex chromosomal and autosomal gene expression. Going forward, considering patient sex as well as the entire genome, including assessment of the sex chromosome complement, will provide additional insights into personalized tumor etiology, progression, treatment, and patient outcome. Teaser Loss of typical sex chromosome complement is present in primary tumors and cancer cell lines across tissue types. We identify consistent autosomal gene expression changes across multiple cancers when the sex chromosome complement is altered, and show that loss of sex chromosomes reduces tumor gene expression differences between patient sexes. We show that loss of sex chromosomes is also associated with poorer patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10da2a14fc8a155d62290f61a032edc3e7218d82" target='_blank'>
              Reproducible autosomal gene expression changes with loss of typical X and Y complement across tumor types
              </a>
            </td>
          <td>
            Seema B. Plaisier, Robert Phavong, Mason Farmwald, Mariah Lee, Teagen Allen, Malli Swamy, Ilsa Rodriguez, Mackenzie Wells, Nadia Phaneuf, Susan Christine Massey, Jared Del Rosario, Juvelyn Hart, Alexander Mangelsdorf, Martin Van Der Jagt, Alex R. DeCasien, K. Buetow, 
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Comprehensive epigenomic studies in multiple myeloma (MM) that unravel the connections between major epigenetic regulators, their intertwined collaboration and the potential of combinatorial targeting remain limited. Utilizing ChIP-seq, ATAC-seq, RNA-seq, and DNA methylation (DNAme) data, we generated whole-genome chromatin annotations from normal plasma cells and MM patients, revealing epigenomic re-configuration affecting downstream genes involved in tumour growth and survival. Primary MM samples showed global DNA hypomethylation but site-specific hypermethylation was observed at transcription start sites, promoters, and enhancers. Moreover, increased deposition of H3K27me3 was observed in clinically relevant functional chromatin clusters. Combined EZH2 and DNMTs inhibition resulted in extensive epigenomic alterations activating apoptosis and cell cycle genes, leading to increased G2/M arrest and apoptosis in MM cell lines. Our findings provide novel insights into the role of epigenetic gene silencing in MM tumorigenesis and the interplay between the Polycomb repressive complex 2 and DNAme. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17093-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1dfe4ebbad2d35b5cf5e512e8911342a3522d08" target='_blank'>
              Combinatorial DNMTs and EZH2 inhibition reprograms the H3K27me3 and DNAme-mediated onco-epigenome to suppress multiple myeloma proliferation
              </a>
            </td>
          <td>
            Alba Atienza Párraga, P. Nylund, K. Diamanti, Berta Garrido-Zabala, Stefania Iliana Tziola, Louella Vasquez, P. Pyl, D. Raykova, Aron T. Skaftason, Anqi Ma, Jian Jin, J. Martín-Subero, Fredrik Öberg, Elke De Bruyne, Jan Komorowski, Helena Jernberg Wiklund, A. Kalushkova
          </td>
          <td>2025-08-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Oncomicrobes are estimated to cause 15% of cancers worldwide. When cancer whole-genome sequencing (WGS) data are collected, the microbes present are also sequenced, allowing the investigation of potential etiological and clinical associations. Interrogating the microbial community for 8908 patients encompassing 22 cancer types from the Genomics England WGS dataset revealed that only colorectal tumors exhibited unmistakably distinct microbial communities that can reliably be used to distinguish anatomical site [positive predictive value (PPV) = 0.95]. This pattern was validated in two independent datasets. Potential clinical relevance uncovered by our analyses included accurate detection of alphapapillomaviruses [human papillomavirus (HPV)] in oral cancers, when compared with current clinical standards, and the detection of rare, highly pathogenic viruses such as human T-lymphotropic virus-1. Biomarker investigations demonstrated statistically significant associations (P < 0.05) between a subset of anaerobic bacteria and survival in certain subtypes of sarcoma. Our results contradict previous claims that each cancer type has a distinct microbiological signature but highlight the potential value of microbial analysis for certain cancers as WGS of tumor samples becomes common in the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82e28dcd99723ccc30fe571b2d59df15e8ee150f" target='_blank'>
              The landscape of microbial associations in human cancer.
              </a>
            </td>
          <td>
            Abraham Gihawi, H.M. Wood, Jeremy Clark, Justin O'Grady, Rosalind A. Eeles, David C Wedge, G. M. Jakobsdottir, G. Magiorkinis, A. Schache, Liam Masterson, Matt Lechner, Tim R. Fenton, Terence M Jones, Adrienne M Flanagan, Solange De Noon, Alex Rubinsteyn, Rachel Hurst, Colin S Cooper, Daniel S. Brewer
          </td>
          <td>2025-09-03</td>
          <td>Science translational medicine</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="The RNA helicase DDX41 is a DEAD-box helicase that is well known as a virus sensor in dendritic cells and a tumor suppressor that is frequently mutated in myeloid neoplasms. However, the functions and relevance of DDX41 in solid tumors remain largely unexplored. In this study, through in vivo CRISPR screening, we demonstrate that DDX41 is highly expressed in various solid tumor types and promotes tumorigenicity in liver cancer. Mechanistically, DDX41 facilitates R-loop processing and accelerates the transcription of RPL/RPS genes, thereby promoting ribosome biogenesis and protein synthesis. Additionally, we show that the acetyltransferase KAT8 is required for H3K9ac modification of the DDX41 promoter and that NR2C1/NR2C2 are responsible for DDX41 expression. Moreover, elevated DDX41 levels increase liver cancer cell sensitivity to protein synthesis inhibitors; treatment with homoharringtonine (HHT), an approved drug, significantly inhibits tumor growth in DDX41-overexpressing liver cancer models. Taken together, the results of this study highlight that DDX41 acts as an oncogene in liver cancer and suggest that protein synthesis inhibition may be a promising therapy for liver cancers with high DDX41 expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98d12c5da526b704dc3b4f45e9292335c86b8f71" target='_blank'>
              CRISPR screening reveals that RNA helicase DDX41 triggers ribosome biogenesis and cancer progression through R-loop-mediated RPL/RPS transcription
              </a>
            </td>
          <td>
            Hongquan Li, Yifei He, Jiawen Jiang, Zhen Liu, Yizhe Liu, Qili Shi, Jie Ding, Haochen Li, Weili Sun, Xin Hu, Zhiao Chen, Xianghuo He
          </td>
          <td>2025-08-11</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa29e8cd8f6fdd768b529f92b9275ea04ce76dc" target='_blank'>
              Genetic landscape of cancer: mechanisms, key genes, and therapeutic implications.
              </a>
            </td>
          <td>
            Arun Karnwal, Joydeep Dutta, Aqueel-Ur-Rehman, Abdel Rahman Mohammad Said Al-Tawaha, Natalia Nesterova
          </td>
          <td>2025-08-17</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Repairing programmed DNA double-strand breaks (DSBs) is crucial in the lifecycle of Paramecium tetraurelia, especially during its sexual reproduction phase when its somatic polyploid macronucleus is lost. The formation of a new macronucleus involves programmed genome rearrangements, introducing DNA DSBs at ∼45 000 loci. Paramecium tetraurelia employs a non-homologous end joining (NHEJ) mechanism for the faithful repair of these DSBs. There are four genes encoding DNA polymerases of family X in the genome, one of which was found recently to colocalize with other NHEJ proteins in the nucleus. Here we have characterized all four enzymes and shown that they are generally very faithful. They fall into two functional classes that may specialize in the distinct repair contexts encountered during DSB DNA repair. Biochemical assays, site-directed mutagenesis, and X-ray structures of mutants of human Polλ incorporating sequence determinants from P. tetraurelia PolX or metazoan Polβ are used to investigate the origin of their fidelity. Our findings suggest that Paramecium PolX enzymes may represent evolutionary intermediates between metazoan Polβ and Polλ. A general classification of DNA PolXs based on clustering methods indicates that our results can be generalized to plant DNA PolXs (Polλ-like) involved in DSB DNA repair generated by CRISPR–Cas9 engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e54e53d0ac06ac21d27fa1e914141a18179e83e" target='_blank'>
              Fidelity, specialization, and evolution of Paramecium PolX DNA polymerases involved in programmed double-strand break DNA repair
              </a>
            </td>
          <td>
            Nourisson Antonin, Missoury Sophia, Lucas-Staat Soizick, Haouz Ahmed, Delarue Marc
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromatin, the spatial organizer of genomic DNA, is hierarchically folded into higher-order structures to facilitate DNA compaction, enabling genome surveillance. Understanding the organization and function of the three-dimensional (3D) genome is critical to profile chromatin accessibility and functional interactions that govern gene regulation across multiple biological processes, including aging and one of its hallmarks, cellular senescence. Cellular senescence constitutes a defensive stress response to various intrinsic and extrinsic stimuli, preserving cellular and organismal homeostasis through a generally irreversible cell cycle arrest. In this review article we discuss epigenetic alterations occurring to DNA and chromatin that drive and fuel the onset of this complex phenomenon. As such, we describe major large-scale chromatin events, including the formation of higher-order chromatin structures and the 3D spatial alterations of the genome that occur during senescence. We also discuss global heterochromatin loss, deficiencies in nuclear lamins, the depletion of core histones and their modifications, as well as the epigenetic regulation of the senescence-associated secretory phenotype (SASP), all of which serve key roles in the epigenome of senescent cells. To clearly demonstrate the significance of epigenetic modifications, data from a computational meta-analysis are presented, aiming to further underpin key epigenetic mechanisms occurring in senescent cells. Last, we highlight promising epigenetic modulators implemented in therapeutic strategies for senescent cell detection and elimination, possibly leading to significant clinical advances against various age-related diseases as well as the delay and prevention of the aging onset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fd36c31f599c330b53d9bff8eb9f2a73d497006" target='_blank'>
              Interrogating the regulatory epigenome of cellular senescence
              </a>
            </td>
          <td>
            Dimitris-Foivos Thanos, Orestis A. Ntintas, Emmanouil Athanasiadis, A. Papaspyropoulos, R. Petty, V. Gorgoulis
          </td>
          <td>2025-08-31</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Abstract The murine endogenous retrovirus MERVL is dynamically activated in a small population of in vitro cultured mouse embryonic stem cells (mESCs) exhibiting totipotent-like features. Yet, the relationship between MERVL activation and cell fate decisions of mESCs is incompletely understood. Through a genome-wide knockout screen, we discovered that MERVL activity is intrinsically linked to DNA damage response pathways. Loss of Ints7, a backbone subunit of the Integrator complex, increased DNA damage and triggered MERVL expression. Mechanistically, Ints7 depletion induced phosphorylation of Kap1, increased chromatin accessibility at MERVL loci, and activated the p53–Dux axis to drive MERVL transcription. Intriguingly, DNA damage-induced MERVL resurgence followed the cleavage of caspase-3, often accompanying a process known as anastasis—cell survival after transient apoptotic signaling. Collectively, our study uncovered that MERVL activation in mESCs is integrated into the cellular circuit for decision-making in response to DNA damage, suggesting that sublethal caspase activation can influence the developmental potential of stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9647f46def2d9561823487f197db2f3a1327b93a" target='_blank'>
              Ints7 deficiency activates DNA damage response to elicit resurgence of endogenous retrovirus MERVL and anastasis of embryonic stem cells
              </a>
            </td>
          <td>
            Yunfan Shen, Li Li, Huiling Ni, Hui Li, Mingrui Xu, Xiaoyang Tan, Zhangjie Li, Pishun Li, Fang Chen, Song Mao, Gongping Sun, Kai Yuan
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee3a629089847ef194ea5479a4ede2bc91a09e2d" target='_blank'>
              Detection and monitoring of translocation renal cell carcinoma via plasma cell-free epigenomic profiling.
              </a>
            </td>
          <td>
            S. Garinet, Karl Semaan, Jiao Li, Ananthan Sadagopan, J. Canniff, Noa Phillips, K. Klega, Medha Panday, H. Savignano, Matthew P Davidsohn, Kevin Lyons, Alessandro Medda, Prateek Khanna, M. Achom, Prathyusha Konda, Brad Fortunato, R. Nawfal, Razane El Hajj Chehade, Ze Zhang, Jillian O’Toole, J. Horst, Dory A Freeman, Rachel Trowbridge, C. Chau, William D. Figg, Jacob E. Berchuck, Brian D Crompton, J-H. Seo, T. Choueiri, M. Freedman, S. Baca, Srinivas R Viswanathan
          </td>
          <td>2025-09-04</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Both single nucleotide variants (SNVs) and somatic copy number alterations (SCNAs) accumulate in cancer cells during tumour development, fuelling clonal evolution. However, accurate estimation of clone-specific copy numbers from bulk DNA-sequencing data is challenging. Here we present allele-specific phylogenetic analysis of copy number alterations (ALPACA), a method to infer SNV and SCNA coevolution by leveraging phylogenetic trees reconstructed from multi-sample bulk tumour sequencing data using SNV frequencies. ALPACA estimates the SCNA evolution of simulated tumours with a higher accuracy than current state-of-the-art methods1–4. ALPACA uncovers loss-of-heterozygosity and amplification events in minor clones that may be missed using standard approaches and reveals the temporal order of somatic alterations. Analysing clone-specific copy numbers in TRACERx421 lung tumours5,6, we find evidence of increased chromosomal instability in metastasis-seeding clones and enrichment for losses affecting tumour suppressor genes and amplification affecting CCND1. Furthermore, we identify increased SCNA rates in both tumours with polyclonal metastatic dissemination and tumours with extrathoracic metastases, and an association between higher clone copy number diversity and reduced disease-free survival in patients with lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69d250508121e21820a44d64995e2f038263f7bb" target='_blank'>
              Clone copy number diversity is linked to survival in lung cancer
              </a>
            </td>
          <td>
            Piotr Pawlik, Kristiana Grigoriadis, Abigail Bunkum, Helena Coggan, A. Frankell, Carlos Martínez-Ruiz, Takahiro Karasaki, A. Huebner, Andrew Rowan, Jasmin Fisher, A. Hackshaw, C. Swanton, S. Zaccaria, N. Mcgranahan
          </td>
          <td>2025-08-13</td>
          <td>Nature</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Abstract DNA double-strand breaks (DSBs) represent critical events in genome integrity, arising from both endogenous cellular processes and exogenous factors. These breaks are implicated in various genomic aberrations and chromosomal rearrangements, leading to cancers and genetic disorders. Common and rare fragile sites, containing repetitive elements and non-B DNA structures, are particularly prone to breakage under replication stress, which play a pivotal role in cancer development and genetic diseases. Accurate quantification of DNA breaks in the context of repetitive sequences such as microsatellites or non-B DNA structures is technically challenging. We have been comparing four different methods to reliably quantify DSBs in repetitive DNA, namely Southern blot, DSB-PCR, real-time DSB-qPCR, and digital PCR (dPCR). We show here that dPCR offers enhanced sensitivity and specificity compared to other methods. This provides significant applications for future disease diagnosis, understanding molecular mechanisms generating chromosomal breakage and for the development of gene therapies for microsatellite expansion disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccab30f2a498f797c04595f2b9078ba3ca2f3603" target='_blank'>
              Fast and accurate quantification of double-strand breaks in microsatellites by digital PCR
              </a>
            </td>
          <td>
            Cécile Palao, Adèle Kovacs, M. Teixeira, G. Richard
          </td>
          <td>2025-08-09</td>
          <td>Biology Methods & Protocols</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Small round cell sarcomas (SRCS) are highly aggressive tumors in soft tissues and bone of mostly children and young adults. Despite being different in many aspects, including genetics, possible cell-of-origin, and pathology, patients with any of these entities all receive the same therapeutic regimen. Although several pre-clinical models of Ewing sarcoma have been established, such as cell lines and patient-derived tumor xenografts, few models exist for other SRCS. Here, we describe a pediatric SRCS tumor organoid (tumoroid) biobank containing long-term tumoroid cultures with different translocations, including EWSR1::FLI1, EWSR1::ERG, CIC::DUX4, and BCOR-rearrangements. Using histology, whole genome sequencing and RNA sequencing, we demonstrate that these tumoroids retain histological characteristics, known marker gene expression and chromosomal rearangements of their matching patient tumors. In addition, we compare mutation clusters in the tumoroids across patient-matched longitudinal samples, which shows that cellular heterogeneity is maintained. Drug screening on the tumoroid models unveils entity-specific drug sensitivity to various cytotoxic compounds and targeted compounds, including MCL-1 inhibitors for CIC::DUX4 sarcomas. Taken together, this newly established SRCS patient-derived tumoroid biobank represents a promising source of material for future basic cancer research and drug screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d54a3ce373321b31d8cc83feaca65eb77c05c889" target='_blank'>
              Small round cell sarcoma tumoroid biobank reveals CIC::DUX4 sarcoma vulnerability to MCL-1 inhibition
              </a>
            </td>
          <td>
            F. Ringnalda, Gijs J. F. van Son, Laurens H G Verweij, Seok-Young Kim, Vicky Amo-Addae, U. E. Flucke, L. Hiemcke-Jiwa, Karin P S Langenberg, Jos A. M. Bramer, L. Heimans, M. A. van de Sande, W. V. van Houdt, Max M. van Noesel, H. Kerstens, Marcel Santoso, Georg Seifert, O. Delattre, Katia Scotlandi, B. Geoerger, J. H. Merks, J. Molenaar, R. van Boxtel, Marc van de Wetering, K. Sanders, H. Clevers
          </td>
          <td>2025-08-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Over 95% of ependymomas that arise in the cortex are driven by a gene fusion involving the zinc finger translocation-associated (ZFTA) protein. Here, using super-resolution and lattice light-sheet microscopy, we demonstrate that the most frequent fusion variant, ZFTA–RELA (ZR), forms dynamic nuclear condensates that are required for oncogene expression and tumorigenesis. Mutagenesis studies of ZR reveal a key intrinsically disordered region (IDR) in RELA that governs condensate formation. Condensate-modulating IDR mutations introduced into ZR impaired its genomic occupancy at oncogenic loci and inhibited the recruitment of transcriptional effector proteins, such as MED1, BRD4 and RNA polymerase II. Using nuclear magnetic resonance spectroscopy, we examined the DNA-binding residues of the critical zinc finger (ZF1) found in ZR and characterized their significance for condensate formation, genomic binding and oncogene activation. We generated synthetic ZFTA fusion proteins where IDRs from known condensate-forming proteins were grafted into ZR. Synthetic ZFTA fusion oncoproteins utilizing IDRs from EWS and FUS restored condensate formation, oncogene transcription and tumour initiation in mice. These findings provide key insights into the oncogenic mechanism of ZR and the importance of IDR acquisition in fusion oncoproteins in brain cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d5d4e3905b6133ceff9f0311d9fcea6d57257e" target='_blank'>
              Synthetic ZFTA fusions pinpoint disordered protein domain acquisition as a mechanism of brain tumorigenesis
              </a>
            </td>
          <td>
            A. Arabzade, H. K. Shirnekhi, S. Varadharajan, S. M. Ippagunta, A. Phillips, N. Laboe, D. W. Baggett, Wahiduzzaman, M. Jo, T. Zheng, R. Pathak, D. Gee, D. Bhimsaria, H. Wu, X. Gao, J. Liu, E. Emanus, A. Bland, A. Kardian, A. Hancock, B. Holcomb, T. Wright, T. Bugbee, H. Sun, M. Zhai, E. Caesar, M. Park, S. Tripathi, A. Shirinifard, K. Lowe, A. Khalighifar, R. A. Petersen, S. King, D. Stabley, A. Pitre, G. Campbell, C-G Park, W. T. Freyaldenhoven, B. Chandra, Y. Xia, E. Bonten, A. Achari, S. Kandikonda, A. Carisey, S. Pounds, J. Xu, D. W. Ellison, B. Deneen, K. Bertrand, R. Kriwacki, S. C. Mack
          </td>
          <td>2025-08-27</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Cancer arises from a convergence of genetic mutations and epigenetic alterations. Among the consequences of epigenetic deregulation is the aberrant activation of transposable elements, particularly retrotransposons, the most expansive and functionally impactful class. This review examines the paradoxical role of retrotransposons in cancer, emphasizing their dual capacity to support oncogenesis while simultaneously triggering innate and adaptive immune responses that may limit tumor growth. A central question is discussed: Is retrotransposon activation merely a stochastic byproduct of epigenetic instability, or could it represent an intrinsic layer of early tumor surveillance that inadvertently turns against the host as malignancy progresses? Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00368-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7a2bf5f051e3bd6c0fa28e6e14cd088f5983472" target='_blank'>
              A plan or pandemonium? The conundrum of retrotransposon activation in cancer
              </a>
            </td>
          <td>
            Elina Zueva
          </td>
          <td>2025-08-09</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Oncogenic p53 mutations (Onc-p53) are frequent in lung and many other solid tumors often associated with chromosome aberrations. Why cells with Onc-p53 develop chromosomal aberrations and whether the abnormalities contribute to tumor growth remain elusive. Evidence in this communication demonstrate for the first time that replication stress induced by Onc-p53 triggers re-copying of DNA replication forks, which generates replication intermediates that cause persistent mitotic aberration and DNA segregation errors. Replication intermediates from re-copied replication forks induced by Onc-p53 activate ATM signaling, which stabilizes Onc-p53, reinforces its ability to upregulate replication factors for sustaining replication stress, thus generating a feedforward cycle accelerating tumor formation. In agreement with this observation our time lapse video microscopy show in real time that persistent mitotic aberration and DNA segregation errors induced by Onc-p53 confer selective growth advantage. Accordingly, human lung tumors with Onc-p53 show selection of cells with mitotic aberration during serial passages. Knock down of active replication forks reduces re-copied fork generation by Onc-p53 and specifically induces apoptotic death of lung cancer cells expressing Onc-p53 in xenograft lung tumors synergistically in cooperation with inhibitors of ATM activation, deselecting cells with Onc-p53 with mitotic errors. This communication reveals a novel mechanism which interconnects replication stress induced by Onc-p53 to its stabilization and ability to generate chromosomal aberration in lung cancer cells that both accelerate tumor growth and serve as a targetable therapeutic vulnerability. These findings will be extremely valuable for tumor-specific treatment of a high percentage of cancer patients with p53 mutation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b77d562f308b7a32a1524622e9f1f8039e4924f9" target='_blank'>
              Oncogenic p53 induces mitotic errors in lung cancer cells by recopying DNA replication forks conferring targetable proliferation advantage
              </a>
            </td>
          <td>
            S. Deb, Shilpa Singh, Lilia Gheghiani, R. Frum, Steven Grossman, Brad Windle, S. Deb
          </td>
          <td>2025-08-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Alterations in 3D chromatin conformation may disrupt the interplay between promoters and distal enhancers. How gene regulatory circuits are reshaped during ionizing radiation-induced tumorigenesis remains unclear, and little is known about the mediators that drive these processes. To decipher the chromatin alterations in radiation-induced lung cancer, we performed ATAC-seq, RNA-seq and Hi-C analyses of human bronchial epithelial cells and corresponding radiation-induced malignantly transformed cell lines. We found that this malignant transformation is accompanied by chromatin switching from the inactive B compartment to the active A compartment, an increased number of TADs and gained ATAC-seq peaks that mediate new distal chromatin contacts. We identified tumour protein 63 (TP63) as a mediator of new chromatin-accessible sites that anchor tumour-specific chromatin contacts in radiation-induced tumour cells. A TP63-mediated accessible chromatin site anchors a tumour-specific TAD boundary and multiple tumour-specific chromatin loops, which might underlie MYC oncogene activation during malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b8217e99ba9977673505cae49c7863f218434b" target='_blank'>
              TP63 mediates the generation of tumour-specific chromatin loops that underlie MYC activation in radiation-induced tumorigenesis
              </a>
            </td>
          <td>
            Qiaowei Liu, Shuai Jiang, Xiang Xu, Kang Xu, Yawen Luo, Zongyuan Yu, Meida Xiang, Zhouyang Xu, Lijie Wang, Sujie Zhang, H. Tao, Xuhui Yang, Chongbo Jiang, Xiaoguang Qi, Hao Li, Yiming Gao, Yao Li, Shihui Fu, Pingkun Zhou, Xiaochen Bo, Hebing Chen, Xiaohua Chen, Yi Hu
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background/Objectives: DNA-damaging agents can contribute to genetic instability, and such agents are often used in cancer chemotherapeutic regimens due to their cytotoxicity. Thus, understanding the mechanisms involved in DNA damage processing can not only enhance our knowledge of basic DNA repair mechanisms but may also be used to develop improved chemotherapeutic strategies to treat cancer. The high-mobility group box protein 1 (HMGB1) is a known nucleotide excision repair (NER) cofactor, and its family member HMGB3 has been implicated in chemoresistance in ovarian cancer. Here, we aim to understand the potential role(s) of HMGB3 in processing DNA damage. Methods: A potential role in NER was investigated using HMGB3 knockout human cell lines in response to UV damage. Subsequently, potential roles in DNA interstrand crosslink (ICL) and DNA double-strand break (DSB) repair were investigated using mutagenesis assays, metaphase spreads, foci formation, a variety of DNA repair assays, and TagSeq analyses in human cells. Results: Interestingly, unlike HMGB1, HMGB3 does not appear to play a role in NER. We found evidence to suggest that HMGB3 is involved in the processing of both DSBs and ICLs in human cells. Conclusions: These novel results elucidate a role for HMGB3 in DNA damage repair and, surprisingly, also indicate a distinct role of HMGB3 in DNA damage repair from that of HMGB1. These findings advance our understanding of the role of HMGB3 in chemotherapeutic drug resistance and as a target for new chemotherapeutic strategies in the treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1261da2765ba41543d61e2f0da2291c8e870115b" target='_blank'>
              High-Mobility Group Box Protein 3 (HMGB3) Facilitates DNA Interstrand Crosslink Processing and Double-Strand Break Repair in Human Cells
              </a>
            </td>
          <td>
            Jillian Dangerfield, Anirban Mukherjee, W. Reh, Anna M. Battenhouse, Karen M. Vasquez
          </td>
          <td>2025-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="ABSTRACT Liquid biopsy represents a transformative approach in oncology, enabling noninvasive disease detection and monitoring through epigenetic signals in circulating tumor DNA (ctDNA), nucleosomes, and noncoding RNAs. Tumor initiation is driven by epigenetic modifications, including DNA methylation, histone alterations, and dysregulated noncoding RNAs, which disrupt gene regulation, cell cycle control, DNA repair, and metastatic processes. This review systematically examines recent evidence on DNA methylation, histone marks (e.g., H3K27me3, H3K18ac), and noncoding RNAs (miRNAs, lncRNAs) as biomarkers for early cancer detection, prognosis, and therapeutic response. Particular focus is placed on aberrant DNA methylation (e.g., hypermethylation of CDKN2A, RASSF1A) and altered histone modifications (e.g., EZH2‐mediated silencing) as indicators of tumor heterogeneity and evolution. Stable and specific in biofluids, noncoding RNAs such as oncogenic miR‐21, tumor‐suppressive miR‐34a, and metastasis‐associated MALAT1/HOTAIR further enhance clinical applicability. Recent detection methods, including bisulfite sequencing, ChIP‐seq, and RNA‐seq, have advanced biomarker profiling, though challenges remain in standardization and low‐abundance detection. With over 12 active clinical studies validating their utility, integration of epigenetic markers with AI and multiomics holds promise for individualized, dynamically guided oncology care. Future innovations, such as chromatin accessibility analysis and cfDNA fragmentation profiling, may further refine diagnostic precision and therapeutic monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77f80572fe0354408f2985b22c2727d7355f5dac" target='_blank'>
              Transforming Cancer Diagnostics: The Emergence of Liquid Biopsy and Epigenetic Markers
              </a>
            </td>
          <td>
            Debalina Saha, Pritam Kanjilal, Mandeep Kaur, Soumya V. Menon, Ayash Ashraf, M. Kumar, T. Alqahtani, Shikha Atteri, D. Uti, Bikram Dhara
          </td>
          <td>2025-09-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer genomes frequently carry apolipoprotein B mRNA editing catalytic polypeptide-like (APOBEC)-associated DNA mutations, suggesting APOBEC enzymes as innate mutagens during cancer initiation and evolution. However, the pure mutagenic impacts of the specific enzymes among this family remain unclear in human normal cell lineages. Here, we investigate the comparative mutagenic activities of APOBEC3A and APOBEC3B, through whole-genome sequencing of human normal gastric organoid lines carrying doxycycline-inducible APOBEC expression cassettes. Our findings demonstrate that transcriptional upregulation of APOBEC3A leads to the acquisition of a massive number of genomic mutations in just a few cell cycles. In contrast, despite clear deaminase activity and DNA damage, APOBEC3B upregulation does not generate a significant increase in mutations in the gastric epithelium. APOBEC3B-associated mutagenesis remains minimal even in the context of TP53 inactivation. Further analysis of the mutational landscape following APOBEC3A upregulation reveals a detailed spectrum of APOBEC3A-associated mutations, including indels, primarily 1 bp deletions, clustered mutations, and evidence of selective pressures acting on cells carrying the mutations. Our observations provide a clear foundation for understanding the mutational impact of APOBEC enzymes in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ca9c435f5c5a8910902a03b815334d801814d13" target='_blank'>
              APOBEC3A drives deaminase mutagenesis in human gastric epithelium
              </a>
            </td>
          <td>
            Yohan An, Ji-Hyun Lee, Joonoh Lim, J. Youk, Seongyeol Park, Ji-Hyung Park, Kijong Yi, Taewoo Kim, Chang Hyun Nam, Won Hee Lee, Soo A Oh, Yoo Jin Bae, Thomas M. Klompstra, Haeun Lee, Jinju Han, Junehwak Lee, Jung Woo Park, Jie-Hyun Kim, Hyunki Kim, Hugo Snippert, B. Koo, Young Seok Ju
          </td>
          <td>2025-08-26</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is characterized by rapid progression, therapeutic resistance, and a profoundly immunosuppressive tumor microenvironment. Emerging evidence suggests that endoplasmic reticulum (ER)-associated macromolecules play critical roles in tumor adaptation. In this study, we performed a multi-omics investigation of orosomucoid-like protein 2 (ORMDL2), a conserved ER membrane protein involved in sphingolipid biosynthesis and ER stress regulation, and uncovered its regulatory functions in GBM progression. Transcriptomic analyses across The Cancer Genome Atlas (TCGA), and Chinese Glioma Genome Atlas (CGGA) revealed elevated ORMDL2 expression in GBM tissues which causes poor prognosis. The MetaCore pathway and Gene Set Enrichment Analysis (GSEA) identified ORMDL2's involvement in antigen presentation via a major histocompatibility complex I (MHC class I), unfolded protein response (UPR), and mitochondrial apoptotic signaling. Single-cell RNA-sequencing data and the Human Protein Atlas showed ORMDL2 enrichment in tumor stromal cells. Pharmacogenomic correlation via the Genomics in Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) database suggested that ORMDL2 expression was associated with resistance to DNA damage response inhibitors such as etoposide, doxorubicin, talazoparib, and might interact with sphingolipid-targeting compounds. Collectively, our findings establish ORMDL2 as a multi-functional macromolecular regulator of immune suppression and therapeutic resistance in GBM, providing new mechanistic insights and potential targets for translational medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5a5f2b3cb520c8099ca5e5eaf8e443e9c0dbc4" target='_blank'>
              Targeting ORMDL2 in glioblastoma through integrated landscape of high-throughput sequencing and pharmacogenomic analysis
              </a>
            </td>
          <td>
            Fitria Sari Wulandari, , Sachin Kumar, Ngoc Phung Ly, Juan Lorell Ngadio, Dahlak Daniel Solomon, Do Thi Minh Xuan, Gabriela Natasha Sutandi, , , Yung-Kuo Lee, , , Ching-Chung Ko
          </td>
          <td>2025-09-27</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The majority of life-threatening cancers arise from epithelial tissues. These epithelial cancers include cutaneous squamous cell carcinoma (cSCC), the second-most common cancer. cSCC is highly invasive and accounts for an estimated 15,000 deaths each year. We identified SRCAP, a chromatin remodeler that regulates the chromatin occupancy of the histone H2A variant H2A.Z, as a frequently mutated gene in cSCC. Analysis of cSCC mutations in epithelial cancers identified a hotspot truncating mutation in SRCAP, which removes 42% of the protein sequences after amino acid 1879. While SRCAP mutations have been previously connected to the pathogenesis of Floating-Harbor syndrome (FHS), these typically occur downstream, with a hotspot mutation leading to protein truncation after amino acid 2444. We found that expressing the SRCAP-1879 truncation in an HRas-CDK4-driven cSCC model was sufficient to increase proliferation, impair terminal differentiation, and accelerate invasion. Mechanistically, the expression of SRCAP-1879 in primary human keratinocytes was sufficient to dysregulate genes crucial for carcinogenesis (e.g., proliferation, differentiation, and motility) without altering H2A.Z occupancy. In particular, the expression of SRCAP-1879 truncation led to strong induction of the matrix metalloproteinase MMP9 expression level, accompanied by increased keratinocyte cell motility, which was sensitive to matrix metalloprotease inhibition. In contrast, the expression of the SRCAP-FHS truncation did not increase but instead reduced cell motility as well as the expression of MMP9. Taken together, our findings identify a previously under-characterized role of the SRCAP-1879 truncating mutation in promoting multiple aspects of epithelial cancer progression, including invasion, distinct from the well-recognized roles of SRCAP mutations in FHS pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b5e5845afc4ca4f1754b4e5d0d5637c1e90c8a" target='_blank'>
              A dominant SRCAP truncating mutation promotes squamous cell carcinoma progression
              </a>
            </td>
          <td>
            S. Droll, Elena I. O. Dewar, Celia Xue, Maxwell C Levine, Benny J Zhang, Xiaomin Bao
          </td>
          <td>2025-08-26</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Base excision repair, initiated by DNA glycosylases, preserves genomic integrity by removing damaged bases. DNA glycosylases Ogg1 and Mutyh were shown to alter the hippocampal transcriptome independently of DNA damage repair. However, the role of DNA glycosylases as transcriptional regulators and their involvement in cognition remain unclear. Here, we combine transcriptomic and epigenomic analyses of the hippocampus in mice deficient in DNA glycosylases. We report that the combined deficiency of Ogg1 and Mutyh impairs spatial long-term memory. Mechanistically, Ogg1 and Mutyh modulate DNA methylation of polycomb repressive complex 2 (PRC2) target genes. PRC2 occupancy and associated histone post-translational modifications depend on Ogg1 and Mutyh in neurons and glia. These epigenetic changes correlate with cell-type specific differences in gene expression of PRC2 targets. Finally, human genetic data link DNA glycosylase-regulated genes to cognitive function, highlighting a novel role for Ogg1 and Mutyh in modulating the epigenome to control transcriptional responses relevant for brain-related diseases. Graphical Abstract Created in BioRender. Scheffler, K. (2025). https://BioRender.com/jwscej4. Supplementary Information The online version contains supplementary material available at 10.1007/s00018-025-05730-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/788c473f610d3e71e5a7b1644717d7c2a8e6b1a6" target='_blank'>
              DNA glycosylases Ogg1 and Mutyh influence gene expression of PRC2 targets associated with cognition
              </a>
            </td>
          <td>
            Andreas Abentung, Teri Sakshaug, Rabina Dumaru, N. Liabakk, Mingyi Yang, Junbai Wang, Magnar Bjørås, Katja Scheffler
          </td>
          <td>2025-08-08</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background Numerous cellular processes rely on biomolecular condensates formed through liquid–liquid phase separation (LLPS). Recently, it has become evident that somatic mutations can interfere with or over-activate the formation of phase-separated condensates. Results Here, we set out to systematically study the connection between cancer and biological condensation, specifically mapping the extent to which LLPS is affected in cancer and understanding the molecular pathomechanisms and therapeutic consequences of mutations affecting LLPS scaffolds. We identify both known and novel combinations of molecular functions that are specific to oncogenic fusion proteins and thus have a high potential for driving tumorigenesis. Protein regions driving condensate formation show an increased association with DNA- or chromatin-binding domains of transcription regulators within oncogenic fusion proteins, indicating a common molecular mechanism underlying several soft tissue sarcomas and hematologic malignancies where phase-separation-prone oncogenic fusion proteins form abnormal condensates along the DNA and thereby dysregulate gene expression programs. Conclusions We find that proteins initiating LLPS are frequently implicated in somatic cancers, even surpassing their involvement in neurodegeneration. Our data shows that cancer-driving LLPS scaffolds tend to be potent oncogenes, giving rise to dominant phenotypes and lacking targeting options by current FDA-approved drugs. Finding the currently missing drugs to shut down oncogenic fusion proteins, to disrupt the condensation enabled by them, and to offset their downstream effects could provide cancer drugs widely applicable to diverse cancer incidences previously defying standard treatments. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03787-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54d550a595352429b7ad7fdab681cd564170f566" target='_blank'>
              Phase-separating fusion proteins drive cancer by upsetting transcription regulation
              </a>
            </td>
          <td>
            Nazanin Farahi, Tamas Lazar, Peter Tompa, Bálint Mészáros, Rita Pancsa
          </td>
          <td>2025-10-02</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background Non-invasive urine tests for bladder cancer (BC) could reduce dependence on flexible cystoscopy for diagnosis and surveillance. Most recent developments in urine testing are based on targeted detection of genomic and/or epigenomic markers. We hypothesised that long-read whole-genome sequencing of urinary DNA with direct methylation profiling may allow accurate BC detection and insights into disease biology. However, the feasibility of such an approach has not yet been reported. Methods We applied long-read whole-genome sequencing with direct methylation detection to urine cell pellet DNA (ucpDNA) from 21 haematuria clinic patients: 13 BCs and 8 non-BCs. The modkit Hidden Markov Model algorithm was used to define differentially methylated regions across the genome. The ability to discriminate between BC and non-BC, and the cellular pathways affected were tested using PCA, h-clust and GSEA. Results We observed global hypomethylation and cancer-specific patterns of promoter hypermethylation in urine from BC patients. Sequencing of a single ucpDNA sample per flow cell yielded read depths of 18-34x; furthermore, BC methylation patterns were also evident with 2–5x multiplex sequencing. Copy number changes were also evident in ucpDNAs from BC patients. A limitation of the study is the small number of samples analysed; however, the detection of cancer-specific events demonstrates the feasibility of the approach, both in single and multiplexed flow-cell runs. Conclusions Even at low-read depths, genome-wide methylation patterns in urinary DNA reflect the presence of BC, potentially permitting rapid, non-invasive and cost-effective BC detection. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01946-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8390cf4d4c35b389c0ed48f2ce2f3090aabaa5c3" target='_blank'>
              Detection of genome-wide methylation changes in bladder cancer by long-read sequencing of urinary DNA
              </a>
            </td>
          <td>
            A. Goel, B. Tura, Joanne D. Stockton, Nicholas Tovey, Luke Ames, Andrew D. Beggs, M. Zeegers, Nicholas D. James, K. Cheng, Richard T. Bryan, D. G. Ward, R. Arnold
          </td>
          <td>2025-08-11</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Persistence of JAK2-mutated clones that may undergo clonal evolution and malignant transformation remains a challenge in myeloproliferative neoplasms (MPN), Novel therapeutic approaches to attenuate clonal evolution and progression to blast-phase are therefore urgently needed. LSD1 (KDM1A) inhibitors reduce symptoms and clonal burden in MPN, but whether these compounds may be effective in advanced disease stages remained so far elusive. Using a chromatin-focused CRISPR-Cas9 screen, we identified the histone methyltransferase DOT1L as a synthetic lethal target under pharmacologic LSD1 inhibition. DOT1L knockout impaired cellular fitness, reduced proliferation, and prolonged survival in xenografts. Furthermore, genetic inactivation of DOT1L increased LSD1 inhibitor sensitivity up to 100-fold resulting in cell cycle arrest and apoptosis induction in TP53 mutant blast-phase MPN. Mechanistically, we have identified a novel, non-canonical function of DOT1L which co-occupied LSD1-bound enhancers and contributed to the repression of transcriptional programs independent of its enzymatic activity. DOT1L loss cooperated with LSD1 inhibitors to activate tumor suppressive programs, while pharmacologic inhibition of DOT1Ls catalytic activity failed to elicit comparable effects. These findings indicate that leveraging DOT1L targeting via protein degradation or RNA interference, rather than conventional enzymatic inhibition, could enhance the therapeutic efficacy of LSD1 inhibitors in blast-phase MPN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd3997846e172a52e0436c7851168e81909eb612" target='_blank'>
              The histone-methyltransferase DOT1L cooperates with LSD1 to control cell division in blast-phase MPN
              </a>
            </td>
          <td>
            Karl Kapahnke, Thomas Plenge, Tabea Klaus, Manoj K. Gupta, Disha Anand, Tamer T. Önder, Birgit Perner, T. Schnöder, Felicitas R. Thol, Frederik Damm, Florian H. Heidel, Florian Perner
          </td>
          <td>2025-08-08</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Deciphering the pre-malignant cell of origin (COO) of different cancers is critical for understanding tumor development and improving diagnostic and therapeutic strategies in oncology. Prior work demonstrates that somatic mutations preferentially accumulate in closed chromatin regions of a cancer’s COO. Leveraging this information, we combine 3,669 whole genome sequencing patient samples, 559 single-cell chromatin accessibility cellular profiles, and machine learning to predict the COO of 37 cancer subtypes with high robustness and accuracy, confirming both the known anatomical and cellular origins of numerous cancers, often at cell subset resolution. Importantly, our data-driven approach predicts a basal COO for most small cell lung cancers and a neuroendocrine COO for rare atypical cases. Our study also highlights distinct cellular trajectories during cancer development of different histological subtypes and uncovers an intermediate metaplastic state during tumorigenesis for multiple gastrointestinal cancers, which have important implications for cancer prevention, early detection, and treatment stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba12099e8d95054e5c9ddd1d07125ef4b9a79476" target='_blank'>
              Learning the cellular origins across cancers using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Wooseung Lee, Bruno Giotti, Akhil Kumar, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Class switch recombination (CSR) and locus suicide recombination (LSR) are critical processes involved in the immune system’s ability to diversify antibody responses. Both are initiated by activation-induced cytidine deaminase, which induces DNA double-strand breaks (DSBs) at specific regions within the immunoglobulin heavy chain (IgH) locus. In CSR, DSBs occur at the switch (S) regions, allowing B cells to replace the IgM heavy chain constant region (CH) with other isotypes, thereby enhancing immune adaptability. This process is regulated by both cis and trans mechanisms, including the IgH super-enhancer 3′ regulatory region (3′RR) and the production of enhancer RNAs (eRNAs). A recent study highlighted the role of MED12 in CSR through enhancer activation and the transcription of eRNA. Now, we show that heterogeneous ribonucleoprotein L (hnRNPL) acts as an additional regulator of CSR and LSR by forming an eRNA-associated complex with CstF64, a polyadenylation factor. This complex facilitates RNA polymerase II elongation and eRNA transcription at the 3′RR. Moreover, the hnRNPL/CstF64 complex promotes NHEJ-mediated DNA repair at both S and 3′RR regions, facilitating 53BP1 and Ku80 recruitment, thereby impacting the efficiency of CSR and LSR. This discovery highlights the intricate, multimodal regulation of these processes, linking eRNA transcription to DNA repair in the process of antibody diversification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d7bc08d8df641c457cf3f47b9d1b5907694cc1f" target='_blank'>
              hnRNPL–CstF64 complex: coordinating CSR and LSR in IgH locus recombination dynamics through eRNA and NHEJ regulation
              </a>
            </td>
          <td>
            Farazul Haque, Mikiyo Nakata, H. Kosako, Tasuku Honjo, N. Begum
          </td>
          <td>2025-08-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Abstract Transposable elements (TEs), once considered genomic “junk”, are now recognized as critical regulators of genome function and human disease. These mobile genetic elements—including retrotransposons (long interspersed nuclear elements [LINE-1], Alu, short interspersed nuclear element-variable numbers of tandem repeats-Alu [SVA], and human endogenous retrovirus [HERV]) and DNA transposons—are tightly regulated by multilayered mechanisms that operate from transcription through to genomic integration. Although typically silenced in somatic cells, TEs are transiently activated during key developmental stages—such as zygotic genome activation and cell fate determination—where they influence chromatin architecture, transcriptional networks, RNA processing, and innate immune responses. Dysregulation of TEs, however, can lead to genomic instability, chronic inflammation, and various pathologies, including cancer, neurodegeneration, and aging. Paradoxically, their reactivation also presents new opportunities for clinical applications, particularly as diagnostic biomarkers and therapeutic targets. Understanding the dual role of TEs—and balancing their contributions to normal development and disease—is essential for advancing novel therapies and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9a4403306942197c6791291eb7e939f11dbad7" target='_blank'>
              Transposable elements in health and disease: Molecular basis and clinical implications
              </a>
            </td>
          <td>
            Yaqiang Hong, Nian Liu
          </td>
          <td>2025-08-15</td>
          <td>Chinese Medical Journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Background Nonsense-mediated messenger RNA decay (NMD) is a highly conserved surveillance system that degrades mRNAs with premature termination codons (PTCs), and regulates the mRNA quantity. Cancer cells hijack NMD to compensate for the imbalanced DNA levels and modulate their antigenicity. We identified the LTO1/YAE1 complex as NMD factors regulating ribosome biogenesis. Our study aimed to investigate their roles in modulating NMD and major histocompatibility complex class I (MHC-I) antigen presentation, which is vital for cancer immunotherapy. Methods Clustered regularly interspaced short palindromic repeats (CRISPR) and the CRISPR-associated protein 9 (Cas9)-mediated gene knockouts, overexpression, and mutational analysis were used to explore the role of LTO1 and YAE1 in NMD and MHC-I expression across various tumor cell lines, assayed by fluorescent reporter-based assays, fluorescence-activated cell sorting analysis, reverse transcription-quantitative PCR, mRNA decay assay, and polysome profiling. Transcriptomic analyses were used to assess the expression of LTO1/YAE1 and their correlation with MHC-I molecules in human cancers. T cell receptor (TCR)-T cells and tumor cells coculture were employed to monitor the effect of LTO1 and YAE1 loss on T cell activation. Iron chelators were further tested as NMD inhibitors to enhance cancer immunotherapy using in vitro coculture assay and a mouse tumor model for immune checkpoint blockade (ICB) therapy. Results We demonstrate that deficiency in LTO1, YAE1, or their downstream target ABCE1 impairs NMD, causes the overexpression of key regulators of MHC-I, including NLR family CARD domain containing 5 (NLRC5), interferon regulatory factor 1 (IRF1), and nuclear factor-kappa B (NFκB), which results in enhanced T cell activation and tumor cell killing in TCR-T models. Transcriptomic analyses reveal that the LTO1/YAE1 complex is frequently overexpressed in human cancers, where it negatively regulates MHC-I. Moreover, low doses of iron chelators inhibit NMD and enhance MHC-I expression, leading to improved recognition and activation of CD8+ T cells, thereby promoting antigen-specific killing and increasing the efficacy of TCR-T and ICB therapy. Conclusions Collectively, our findings establish novel roles for the LTO1/YAE1 complex in regulating MHC-I expression via NMD. As MHC-I is crucial for antigen presentation and T cell activation, these results reveal a previously unappreciated link between NMD and tumor immunogenicity, with potential implications for cancer immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed845b43734d7679aa3a6108491b7b64691ef9ed" target='_blank'>
              LTO1 and YAE1 regulate MHC-I expression via nonsense-mediated RNA decay in tumor cells
              </a>
            </td>
          <td>
            Zhengning Yang, Zhongxuan Meng, Shangyuan Liu, Yuxin Chen, Xuhong Gan, Zi-Mei Zhang, Xiaoshan Luo, Jiahong Yan, Huijin Chen, Danqing Huang, Lihong Huang, Xiaomin Zeng, Wenyu Ma, Jun Liu, Linlin Wang, Haotian Huang, Xinyi Yang, Mei Hu, Lin Chen, Hongli Du, Qibin Leng, Li Qiu, Xiaoling Wang
          </td>
          <td>2025-09-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Lower-grade glioma (LGG) is a common primary brain tumor with potential progression to glioblastoma. Its development is closely linked to DNA damage repair. DNA polymerase delta subunit 3 (POLD3), a subunit of the DNA polymerase δ complex, participates in DNA replication and repair. POLD3 may act as a therapeutic target and prognostic biomarker in LGG. Methods We collected primary data from The Cancer Genome Atlas (TCGA), the Chinese Glioma Genome Atlas (CGGA), the Tumor Immune Estimation Resource (TIMER), and Gene Set Enrichment Analysis (GSEA). The CGGA dataset included messenger RNA (mRNA) sequencing and clinical data from 403 LGG cases; its microarray subset contained 142 additional cases. The TCGA dataset provided clinical data for 503 LGG cases. Employing bioinformatics tools, we investigated the correlation of POLD3 expression with the clinical characteristics, pathological features, survival outcomes, and tumor immune microenvironment in LGG. Subsequent functional validation of POLD3’s impact in LGG was performed using both cellular and animal models. Results Integrative analysis combining bioinformatics and experimental approaches revealed significant POLD3 overexpression in LGG. Elevated POLD3 levels exhibited significant associations with distinct clinicopathological features, altered immune cell infiltration profiles, and adverse clinical outcomes. Functionally, POLD3 was demonstrated to drive LGG cell proliferation, invasion, and modulate the tumor microenvironment. Conclusion This study establishes POLD3 as a critical molecular determinant in LGG pathogenesis. Our findings position POLD3 as a promising multifaceted target, with significant implications for advancing LGG diagnostics, therapeutic development, immune modulation strategies, and prognostic assessment, thereby providing a fresh conceptual framework for understanding and managing this tumor. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14934-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b766711209d5df5ec31c2b94f1204021afe9d8" target='_blank'>
              POLD3 knockdown effects on low-grade glioma: insights from bioinformatics and experimental validation
              </a>
            </td>
          <td>
            Zhihui Yang, Zhonghao Dou, Hao Wang, Zhaojiang Hu, Yanbo Gao, Yilei Bai, Junfeng Liu, Rongjun Qian
          </td>
          <td>2025-10-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. This study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2), and triple-negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb3089644af163030736f3789ed79a1b202fc7" target='_blank'>
              Genetic background and oncogenic driver determines the genomic evolution and transcriptomics of mammary tumor metastasis
              </a>
            </td>
          <td>
            Christina R Ross, Karol Szczepanek, Jack Sanford, Gabriel A. Needle, Tinghu Qiu, Larry Pearce, Maxwell P Lee, Kent W. Hunter
          </td>
          <td>2025-08-14</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Integrative conjugative elements (ICEs) are mobile DNA that remain integrated within the host bacterial genome until they activate their excision and transfer. ICE transfer is initiated in a small subpopulation of cells that leads to transfer competence. Here we study the role of the ParB-like protein BisD for ICEclc transfer competence formation in Pseudomonas putida. Genetic dissection and single cell microscopy showed that mCherry-BisD fusion proteins form fluorescent foci on parS-like DNA sequences on ICEclc. Chromatin immunoprecipitation followed by DNA sequencing showed that BisD accumulates at two bisS sites, and further enriches around 11 ICEclc transfer competence promoters. BisD enrichments suggest its sliding over ca. 50 kb of ICE-DNA in an asymmetric manner, following the direction of inwards ICE gene transcription. BisD has a unique domain absent from ParB, which is essential for ICE promoter activation in conjunction with the ICEclc encoded factor BisC, and for ICE transfer. Similar to ParB, purified BisD preferentially binds and hydrolyses cytidine triphosphate (CTP). Mutation of a conserved Arg- at position 95 to an Ala-residue abolished nucleotide binding by BisD, and also inhibited ICE promoter activation and transfer. BisD action might thus serve two purposes, orchestrating ICE gene activation and ICE-DNA clamping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f3e12fbb10a91cc84c5206bb1304d91859cfdbd" target='_blank'>
              Orchestrated long-distance gene activation by a ParB-like BisD-CTP DNA clamp in low-frequency transfer competence development in Pseudomonas putida
              </a>
            </td>
          <td>
            Hammam Antar, Nicolas Carraro, Hanna Budny, Stephan Gruber, J. R. van der Meer
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 In this talk I will discuss new insights into the nature of tumor evolution in high grade serous ovarian cancer (HGSOC). I will focus on two key areas that have been informed by recent advances in single cell whole genome sequencing. First, I will discuss the nature of how HGSOCs acquire genome doubled states in their evolutionary histories. I will show that genome doubling is an ongoing process throughout evolution with important consequences for phenotypic states and immunosuppression. Second, I will discuss recent work to track evolution from diagnosis to relapse using structural variants and cell-free DNA technology. I will show how clonal populations either contract or persist over a multi-year intervals in patients with relapsed disease and highlight genomic structural features linked to the evolution of drug resistance.



 Sohrab Shah. Frontiers of tumor evolution in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c181ca2707243d8dc30253d7814569567a3fe09e" target='_blank'>
              Abstract IA024: Frontiers of tumor evolution in ovarian cancer
              </a>
            </td>
          <td>
            Sohrab P. Shah
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Cell division cycle 6 (CDC6) is a key licensing factor for DNA replication in the G1 and S phases. Besides initiating replication, CDC6 also helps establish and maintain the S-M checkpoint, ensuring genomic stability. Emerging evidence highlights its dysregulation in various cancers, implicating CDC6 in tumor progression and therapy resistance. However, a comprehensive pan-cancer analysis evaluating its diagnostic, prognostic and immunomodulatory potential remains lacking, underscoring the need for further investigation. Methods By integrating multi-omics datasets from The Cancer Genome Atlas (TCGA), Genotype-Tissue Expression Project (GTEx), cBioPortal, Human Protein Atlas (HPA), UALCAN and SangerBox, we employed systematic bioinformatics approaches to investigate the oncogenic role of CDC6 across multiple cancer types. Our analysis encompassed prognostic associations, mutational landscapes, tumor immune microenvironment (TIME) infiltration patterns and epigenetic regulation via DNA methylation, providing a pan-cancer perspective on CDC6’s potential role in tumorigenesis. In addition, CDC6’s role in melanoma cell proliferation, invasion and migration was experimentally assessed. Results Pan-cancer analysis demonstrated CDC6 as a consistently upregulated oncogene across multiple malignancies, exhibiting significantly elevated expression compared to normal tissues. Notably, CDC6 is closely associated with prognosis across various cancer types. Our investigation further revealed robust correlations between CDC6 expression and immune cell infiltration patterns. Epigenetic profiling identified significant associations between CDC6 expression and DNA methylation alterations in nine cancer types. Functional studies validated CDC6’s oncogenic role, where its overexpression significantly promoted cellular proliferation, migration and invasion in melanoma. Conclusions Our study demonstrates that CDC6 serves as a crucial oncogenic driver across diverse tumor types, establishing its dual utility as a diagnostic biomarker and independent prognostic indicator. Importantly, we identified a significant correlation between elevated CDC6 expression and specific immune microenvironment alterations, suggesting its potential as a predictive biomarker for immunotherapy response. These findings demonstrate CDC6’s dual role in cancer development and immune regulation, warranting further investigation into its mechanisms and therapeutic potential. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14782-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b86557cdc5c9f375fa0289acd10e271c4227fce8" target='_blank'>
              CDC6 as a pan-cancer immunological and prognostic biomarker and its role in suppressing melanoma malignancy
              </a>
            </td>
          <td>
            Lijun Mo, Min Jia, Qian Wu, Qingqing Xu, Z. Mai, Bo Fu, Zhengqiang He, Mingchang Huang, Yinyuan Lan, Qiaozhen Xu, Zhenmou Xie, Xiaoni He, Yaohua Xue
          </td>
          <td>2025-09-24</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 There is global epigenetic dysregulation during transformation from normal to cancerous cells. One of the many consequences of this global epigenetic dysregulation is the reactivation of transposable elements (TEs). TEs comprise roughly half of the human genome and are typically repressed through epigenetic mechanisms, including DNA methylation and histone modifications. Endogenous retroviruses (ERVs) are a class of TEs that contain long terminal repeats (LTRs) with putative gene regulatory motifs. LTRs can be co-opted to be pro-tumorigenic by becoming non-canonical promoters for oncogenic genes. While several examples of TE-derived promoters have been reported in cancer, the potential significance of TE-derived enhancers remains largely unexplored. Here, we utilized publicly available multi-omics datasets (bulk and single-cell chromatin profiling and RNA-sequencing) of high-grade serous carcinoma (HGSC) and fallopian tube (FT) tissues to determine if there are tumor-specific, TE-derived enhancers that have gene regulatory function. Once putative enhancers were identified, we used Clustered Regularly Interspaced Short Palindromic Repeats interference (CRISPRi) to functionally validate a putative TE-derived enhancer and gene regulatory axis. We discovered that on the bulk tissue level, several TE families, notably LTR2B, LTR18A, and LTR18B, are enriched at open chromatin sites and sites marked by H3K27Ac (enhancer mark) in HGSC cells compared to FT cells, suggesting the presence of HGSC-specific, TE-derived enhancers. On the single-cell level, we discovered that TE transcripts from the same families are more highly expressed in tumor cells compared to stromal and immune cells in the tumor microenvironment, suggesting that the increased TE-derived enhancer activity observed on the bulk level is specific to tumor cells. Lastly, we discovered that silencing a specific LTR18A locus upstream of the TCDD-Inducible Poly ADP Ribose Polymerase (TIPARP) gene was sufficient to repress TIPARP gene expression suggesting that these putative TE-derived enhancers have a gene regulatory function. TIPARP expression has been shown to be correlated with HGSC, and our studies show a novel mechanism by which TIPARP may be regulated in HGSC. Determining the regulatory role of these TE-derived enhancers will provide novel insights into not only HGSC, but also other cancer-related transcriptomes.



 Lily Nguyen, Abigail Jeong, Atma Ivancevic, Benjamin Bitler, Edward B. Chuong. Transposable element derived enhancers in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e45139333a87abd3b3e296653a1c558db2322706" target='_blank'>
              Abstract B012: Transposable element derived enhancers in ovarian cancer
              </a>
            </td>
          <td>
            Lily Nguyen, Abigail D Jeong, A. Ivancevic, Benjamin G. Bitler, Edward B. Chuong
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708353dcfdc8ca7c4e2b434fced2427c746e282e" target='_blank'>
              Tracing the evolution of single-cell 3D genomes in Kras-driven cancers.
              </a>
            </td>
          <td>
            Miao Liu, Shengyan Jin, Sherry S Agabiti, Tyler B Jensen, Tianqi Yang, Jonathan S. D. Radda, Christian F. Ruiz, Gabriel Baldissera, Moein Rajaei, Fangyong Li, Jeffrey P. Townsend, M. Muzumdar, Siyuan Wang
          </td>
          <td>2025-08-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Increased expression of lnc-FANCI-2, a newly discovered long noncoding RNA, is associated with cervical lesion progression from cervical intraepithelial neoplasia stage 1 (CIN1, low grade), CIN2–3 (high grade), to cervical cancer. Viral oncoprotein E7 of high-risk human papillomaviruses (HR-HPVs) and host transcription factor YY1 are two major factors promoting lnc-FANCI-2 expression. Using CRISPR-Cas9 technology, we knocked out the expression of lnc-FANCI-2 in the HPV16-positive cervical cancer cell line, CaSki cells. The selected knockout (KO) single-cell clones displayed altered cell morphology and proliferation with changes of cellular soluble receptors, but normal HPV16 E6 and E7 expression. Relative to the parental cells, lnc-FANCI-2 KO cells exhibited significantly increased RAS signaling and epithelial-mesenchymal transition, but decreased response to IFN signaling, along with increased p-Akt and p-Erk1/2 (two RAS signaling effectors), IGFBP3, MCAM, VIM, and CCND2 (cyclin D2) and decreased expression of RAC3. lnc-FANCI-2 in CaSki interacts with cellular proteins H13, HNRH1, K1H1, MAP4K4, and RNPS1. MAP4K4 knockdown led to enhance the expression of p-Erk1/2 and p-Akt. High lnc-FANCI-2 and low MCAM levels in cervical cancer tissues were found to be associated with patients’ survival. A key function of lnc-FANCI-2 intrinsically regulates RAS signaling to impact cervical lesion progression and cervical cancer prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8c94a8ea5a8fd05397e7326fdf7e23695122a41" target='_blank'>
              The long noncoding RNA lnc-FANCI-2 intrinsically restricts RAS signaling in human papillomavirus type 16-infected cervical cancer cells
              </a>
            </td>
          <td>
            Haibin Liu, Lulu Yu, V. Majerciak, Thomas J Meyer, Ming Yi, Peter F. Johnson, Margaret C Cam, Douglas R. Lowy, Zhi-Ming Zheng
          </td>
          <td>2025-08-29</td>
          <td>eLife</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Epigenetic regulation is critical to B cell development, guiding gene expression via DNA methylation, histone modifications, chromatin remodeling, and noncoding RNAs. In mature B cell neoplasms, particularly diffuse large B cell lymphoma (DLBCL), follicular lymphoma (FL), and chronic lymphocytic leukemia (CLL), these mechanisms are frequently disrupted. Recurrent mutations in key epigenetic regulators such as EZH2, KMT2D, CREBBP, and TET2 lead to altered chromatin states, repression of tumor suppressor genes, and enhanced oncogenic signaling. Dysregulation of specific microRNAs (e.g., miR-155, miR-21) further contributes to pathogenesis and therapeutic resistance. In DLBCL, hypermethylation of SMAD1 and CREBBP mutations are associated with immune evasion and chemoresistance. In FL, EZH2 gain-of-function and KMT2D loss-of-function mutations alter germinal center B cell programming, while in CLL, DNA hypomethylation patterns reflect the cell of origin and correlate with clinical outcome. Targeted therapies such as the EZH2 inhibitor tazemetostat have demonstrated efficacy in EZH2-mutant FL, while HDAC and BET inhibitors show variable responses across B cell malignancies. The limitations of current epigenetic therapies reflect the complexity of targeting epigenetic dysregulation rather than therapeutic futility. These challenges nonetheless highlight the relevance of epigenetic alterations as biomarkers and therapeutic targets, with potential to improve the management of mature B cell neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a068788c20e74398875dae829a5f27eeefbe604a" target='_blank'>
              Unraveling the Epigenetic Landscape of Mature B Cell Neoplasia: Mechanisms, Biomarkers, and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Nawar Maher, F. Maiellaro, J. Ghanej, S. Rasi, R. Moia, G. Gaidano
          </td>
          <td>2025-08-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28b36c19d01be738ee1237dc98c176d6789c1e47" target='_blank'>
              Chromatin-associated circRNA ciCRLF3(2) regulates cell differentiation blockage via activating non-homologous end joining-based DNA repair.
              </a>
            </td>
          <td>
            Ke-Jia Pu, Xiao-Tong Chen, Shun-Xin Zhu, Yan An, Xin-Yi Feng, Heng-Jing Huang, Cheng-Lin Zhou, Mei-Ying Ye, Yun-Chun Wei, Yi-Xuan Ma, Chen Fang, Nan Zhang, Dan Wang, Bin-Rong Han, Jun-Yi Lian, Tian-Qi Chen, Yumeng Sun, Yue-Qin Chen, Wentao Wang
          </td>
          <td>2025-09-04</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b074d01f6fb78ae72372c0d26a26df7f99fb224" target='_blank'>
              Fungi as models of centromere innovation: from DNA sequence to 3-dimensional arrangement.
              </a>
            </td>
          <td>
            Srijana Dutta, Krishna Bhat, Rashi Aggarwal, K. Sanyal
          </td>
          <td>2025-08-11</td>
          <td>Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Small-cell lung cancer (SCLC), an aggressive neuroendocrine tumor strongly associated with exposure to tobacco carcinogens, is characterized by early dissemination and dismal prognosis with a five-year overall survival of less than 7%. High-frequency gain-of-function mutations in oncogenes are rarely reported, and intratumor heterogeneity (ITH) remains to be determined in SCLC. Here, via multiomics analyses of 314 SCLCs, we found that the ASCL1+/MKI67+ and ASCL1+/CRIP2+ clusters accounted for 74.38% of the 190,313 SCLC cancer cells from 39 patients, with the ASCL1+SOX1+ stem-like cell cluster across SCLC subtypes. The major histocompatibility complex (MHC) class I molecules were expressed at low levels in six and high levels in five cancer cell clusters and were inversely associated with the KI67 expression level. Abnormal splicing of mRNAs was a feature of SCLC, with focal adhesion kinase (FAK) splicing variants identified in 119 (77.3%) of 154 patients. FAK variants exhibited elevated kinase activity, were associated with the worst prognosis, and were sensitive to FAK inhibitors in patient-derived organoids and xenograft models. Eleven high-frequency mutations were identified in addition to TP53 and RB1, and smoking status and tumor stage did not affect microbiota variance in SCLC. Taken together, our data further revealed the complicated ITH and discovered that FAK splicing variants represent high-frequency gain-of-function alterations in oncogene in SCLC and potential therapeutic targets for this recalcitrant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3bde80c19d7166cce5785be8ae3a4a5e971fbbd" target='_blank'>
              Characterization of the extrinsic and intrinsic signatures and therapeutic vulnerability of small cell lung cancers
              </a>
            </td>
          <td>
            Gui-zhen Wang, Zheng Wang, Shi-Hao Bai, Yun Tan, Wen-Zhao Zhong, Guo-Gui Sun, Yu-Tao Liu, B. Pan, Chen Huang, Di Wang, Beibei Sun, Dong-Ni Chen, Bin Zhang, Yong-Chun Zhou, Sheng Li, Xiang-Wei Zhang, Si-Chong Han, Fu-ying Yang, Xue-Yan Shi, Xiao-Liang Jie, Yu-Ke Shen, L. Liang, Zhe-Sheng Wen, Li Zhang, Mingkun Li, Na Wang, Jin-Song Liu, Ying Dong, Manli Wang, Yan Wang, Chang-Li Wang, Dawei Xie, Ze-Guang Han, Jian-Ming Ying, Chong Chen, Yunchao Huang, Hongbin Ji, Yuan-Yuan Zhang, Yan Yu, Guang-Biao Zhou
          </td>
          <td>2025-09-10</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Repeated oncogenic mutations and polyclonal proliferation are evident in cancers. However, little is known about the polyclonal principles governing the systemic cancerous lineage during immunotherapy. Here, we examine a unique autopsy case of metastatic urothelial carcinoma that exhibits different treatment responses to anti-PD-1 therapy at each tumor site. By performing in-depth analyses of different multiregional bulk tumor masses, we reveal that subsets of subclones acquire potential driver mutations under treatment selection pressure. Spatial transcriptomics analysis reveals that subclones resistant to immunotherapy form distinct immunosuppressive environments consistent with their habitats. Furthermore, different cancer hallmarks are identified in each of the subclones that expand under immunotherapy at single-cell level; for example, one subclone is more proliferative, and another is more stem-cell-like. In summary, this study provides an overall picture of the polyclonal competition and changes in the immune microenvironment that are related to resistance to immunotherapy in patients with malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f357f9c7e57b9e95e8ee48a6378f3d690f3275bd" target='_blank'>
              Clonal diversity shapes the tumour microenvironment leading to distinct immunotherapy responses in metastatic urothelial carcinoma
              </a>
            </td>
          <td>
            Takashi Kamatani, K. Umeda, T. Iwasawa, Fuyuki Miya, K. Matsumoto, S. Mikami, Kensuke Hara, Masayuki Shimoda, Yutaka Suzuki, Jo Nishino, Mamoru Kato, Kazuhiro Kakimi, Nobuyuki Tanaka, M. Oya, T. Tsunoda
          </td>
          <td>2025-08-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is one of the most prevalent adult leukemias, derived from mature B-cells and exhibiting a highly heterogeneous disease course. Standard cytogenetic analysis of CLL includes FISH and karyotyping. However, conventional chromosome analysis of cancer specimens is often constrained by low chromosomal resolution, and FISH analysis is limited by the number of probes that can be applied. This study highlights the application of optical genome mapping (OGM), a high-resolution cytogenomic tool that visualizes ultra-long, sequence-labeled DNA molecules, to uncover the structural complexity of the cancer genome and assess the clinical relevance of chromothripsis in CLL. Comprehensive cytogenetic analysis was conducted on a 43-year-old male diagnosed with chronic lymphocytic leukemia. Karyotyping revealed a complex rearrangement: 46,XY,der(3)t(3;13)(p2?3;q14.3),der(4)t(?3;4)(p23;p16),add(11)(p13),del(13)(q14)[12]/46,sl,del(11)(q2?2.2q23.3)[6]. FISH analysis further identified the loss of ATM and a partial deletion of the D13S319 locus. OGM analysis performed on bone marrow revealed a complex genotype including chromothripsis of chromosome 13, and structural rearrangements involving chromosomes 3, 4, and 11. Additionally, multiple intrachromosomal translocations and interstitial microdeletions of chromosome 13 were identified. The resolution of these aberrations has been significantly enhanced with examples including: ogm[GRCh38] t(3;13)(p26.3;q33.1)(2,706,645~2,721,113;103,142,901~103,154,241][VAF0.45], ogm[GRCh38] t(4;13)(p15.31;q32.1)(20,869,721~20,907,265;96,617,837~96,630,317)[VAF0.42],. In conclusion, OGM revealed the intricate structural alterations of the cancer genome. The high resolution provided by OGM could facilitate the discovery of oncogenic mechanisms, novel fusion genes, prognostic markers, and potential therapeutic targets. OGM serves as a powerful tool for revisiting CLL disease classification by offering deeper insights into complex genomic rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4656aef9b3e1a0de1b2f5258ea921c03f0e3137" target='_blank'>
              Case Report: Unraveling complex genomic alterations in a case of chronic lymphocytic leukemia using optical genome mapping
              </a>
            </td>
          <td>
            Leila Youssefian, Trilochan Sahoo, Jia-Chi Wang
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Alternative lengthening of telomeres (ALT) is a telomerase-independent telomere maintenance mechanism observed in 15% of human cancers. A hallmark of ALT cancers is the presence of C-circles, circular single-stranded DNAs (ssDNAs) enriched with cytosine-rich telomere (C-rich, CCCTAA) sequences. G-circles, containing guanosine-rich telomere (G-rich, GGGTTA) ssDNAs, also exist but are much less abundant. Recent studies indicate that excessive displacement of Okazaki fragments during lagging-strand synthesis is a unique feature of ALT telomeres and responsible for generating C-circles/C-rich ssDNAs. However, the distinct characteristics of C-circles compared to G-circles remain unclear. Here, we demonstrate that co-deficiency of the DNA translocases SMARCAL1 and FANCM leads to abundant generation of G-circle/G-rich ssDNAs. These G-rich ssDNAs mainly exist in linear form, ranging in size from 500 to 3000 nucleotides, which differs significantly from the structure and size of C-circle/C-rich ssDNAs. Mechanistically, both C-rich and G-rich ssDNAs originate from BLM/POLD-mediated excessive strand displacement; however, they differ in their origins and initiation mechanisms. Specifically, C-rich ssDNAs arise from lagging daughter strands initiated by the CST complex, whereas G-rich ssDNAs originate from leading daughter strands through RAD51-dependent G-strand synthesis. Our findings propose two distinct mechanisms for generating two different extrachromosomal telomere DNAs, C- and G-circles, during ALT-mediated telomere elongation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12e4731f01013d1812909194114a8c4a4ce7f91c" target='_blank'>
              Distinct mechanisms underlying extrachromosomal telomere DNA generation in ALT cancers
              </a>
            </td>
          <td>
            Junyeop Lee, Eric J. Sohn, Jina Lee, Ashley Y Cheng, Angelo Taglialatela, Alberto Ciccia, Jaewon Min
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background DNA methylation is an epigenetic mechanism involved in gene regulation and cellular differentiation. Accurate and comprehensive assessment of DNA methylation patterns is thus essential for understanding their role in various biological processes and disease mechanisms. Bisulfite sequencing has long been the default method for analyzing methylation marks due to its single-base resolution, but the associated DNA degradation poses a concern. Although several methods have been proposed to circumvent this issue, there is no clear consensus on which method might be better suited for specific study designs. Results We conducted a comparative evaluation of four DNA methylation detection approaches: whole-genome bisulfite sequencing (WGBS), Illumina methylation microarray (EPIC), enzymatic methyl-sequencing (EM-seq) and third-generation sequencing by Oxford Nanopore Technologies (ONT). DNA methylation profiles were assessed across three human genome samples derived from tissue, cell line, and whole blood. We systematically compared these methods in terms of resolution, genomic coverage, methylation calling accuracy, cost, time, and practical implementation. EM-seq showed the highest concordance with WGBS, indicating strong reliability due to their similar sequencing chemistry. ONT sequencing, while showing lower agreement with WGBS and EM-seq, captured certain loci uniquely and enabled methylation detection in challenging genomic regions. Despite a substantial overlap in CpG detection among methods, each method identified unique CpG sites, emphasizing their complementary nature. Conclusions Our findings underscore the strengths and limitations of current DNA methylation detection methods. EM-seq and ONT emerge as robust alternatives to WGBS and EPIC, offering unique advantages: EM-seq delivers consistent and uniform coverage, while ONT excels in long-range methylation profiling and access to challenging genomic regions. These insights provide practical guidance for method selection based on specific experimental goals. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00616-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1115f5c6b6dae937448bce4077f0951e9ac581f3" target='_blank'>
              Comparison of current methods for genome-wide DNA methylation profiling
              </a>
            </td>
          <td>
            Ana Regina de Abreu, J. Ibrahim, Vasileios Lemonidis, Ligia Mateiu, Guy Van Camp, K. Op de Beeck
          </td>
          <td>2025-08-25</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="SUMMARY Ovarian high-grade serous carcinoma (HGSC) is characterized by extensive intra-peritoneal dissemination and tumor heterogeneity. In the metastatic cascade, tumors utilize several transcriptional programs to translocate and survive in distant tissues. Here, we analyzed multi-modal, real-world data from 350 tumor samples across 160 patients with HGSC to identify transcriptional programs that drive intra-peritoneal metastasis and heterogeneity. We identified nine transcriptional programs, including those regulating epithelial-mesenchymal transition and immune modulation and cytoskeletal reorganization, which shape distinct metastatic trajectories to solid and ascitic environments and are associated to treatment response. Our results reveal pronounced intra-patient transcriptional heterogeneity, which in some cases surpassed inter-patient heterogeneity, highlighting the importance of multi-site sampling for accurate prognostication and combinatorial treatments in HGSC. Our extensive characterization offers novel insights into intra-peritoneal metastasis with significant prognostic implications, reveals histomorphological biomarkers for patient stratification and paves the way for innovative therapeutic strategies aimed at impairing cancer cell adaptability and limiting metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32c6dc18825437bb12a5a64bda09f91c2c6d329" target='_blank'>
              Multi-modal characterization of transcriptional programs that drive metastatic cascades to solid sites and ascites in ovarian cancer
              </a>
            </td>
          <td>
            Kaiyang Zhang, Essi Kahelin, G. Marchi, Oskari Lehtonen, S. Salloum, K. Lavikka, Yilin Li, Felix Dietlein, A. Lahtinen, J. Oikkonen, S. Hietanen, J. Hynninen, Antti Häkkinen, A. Virtanen, S. Hautaniemi
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Mitotic chromosome formation depends on coordinated SMC complex activities, yet how condensin engages cohesin during this process remains unclear. Samejima et al. combined synchronized mitotic entry, auxin-inducible degrons, high-resolution Hi-C, live-cell imaging, quantitative proteomics, and polymer simulations to dissect condensin I, condensin II, and cohesin interplay in vertebrate cells. They showed that condensins actively displace extrusive cohesin to dismantle interphase chromatin and build nested mitotic loops. Condensin II generates large, helical loops, and condensin I forms finer loops, together yielding the canonical rodlike mitotic chromosome. Cohesin, while preserving sister-chromatid cohesion, relocates to loop tips without blocking condensin. The study also reports the first in vivo measurements of condensin loop-extrusion speed. This work establishes a mechanistic, quantitative framework for mitotic chromosome architecture and offers predictive models for future genome-organization- and SMC-related pathology studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/643a4e7b08010ff47e07b3e48096592c6d6e3bce" target='_blank'>
              Decoding Mitotic Chromosome Assembly: Three Rules Governing Condensin–Cohesin Engagement
              </a>
            </td>
          <td>
            Haiyan Yan, Xinyu Zhou, Fangwei Wang
          </td>
          <td>2025-08-19</td>
          <td>Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cutaneous melanoma (CM), with a continuously rising incidence worldwide, represents the most aggressive type of skin cancer, and it leads to the majority of skin cancer-related deaths. Approximately 50% of CM carry the activating BRAFV600 mutation and, although BRAF inhibitors have demonstrated clinical efficacy, most patients often develop early resistance to treatment. Aberrant expression of non-coding RNAs (ncRNAs), which represent less than 2% of the entire transcriptome, has been implicated in CM development and progression. By using BRAFV600-mutant CM in vitro and in vivo models, we have recently demonstrated that the loss of Spry1 expression impairs BRAFV600-mutant CM progression. Therefore, the extensive long and small ncRNA datasets generated in this study might represent a valuable resource for the characterization of their roles in BRAFV600-mutant CM initiation and progression upon Spry1 loss, thus providing a comprehensive resource to support future studies on BRAFV600-mutant CM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c46752f6a68d20467ff6b95252e5ee4402a37" target='_blank'>
              Non-coding RNA profiling in BRAFV600E-mutant cutaneous melanoma before and after Spry1 depletion
              </a>
            </td>
          <td>
            J. Lamberti, Domenico Memoli, Barbara Montico, Francesco Silvestro, R. Guerrieri, F. Colizzi, Alessandro Weisz, A. Salvati, E. Fratta, G. Nassa
          </td>
          <td>2025-09-02</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="ABSTRACT The circular, double-stranded DNA genomes of Human papillomaviruses (HPV) exist in a nucleosomal state throughout the infectious cycle and rely on host histone epigenetic modifications and chromatin assembly processes to promote various phases of the viral life cycle. Here, we show that the histone H3.3 chaperone HIRA and its associated complex members are recruited to HPV replication factories during the late phase of the HPV life cycle. HIRA is also recruited to HPV replication factories generated by amplification of a replicon with a minimal origin and expression of the viral replication proteins E1 and E2, demonstrating that the E1 and E2 proteins are sufficient for HIRA recruitment. Downregulation of HIRA expression reduces HPV31 DNA amplification and viral transcription in differentiated keratinocytes. Histone H3.3 that is highly phosphorylated on serine residue 31 is also enriched at sites of HPV replication and this modification links the DNA damage response to chromatin that supports rapid gene activation. We propose that deposition of histone H3.3 generates viral minichromosomes that are highly primed to support the late stages of the HPV life cycle.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a5b34a817df65cd52a7a01714509e40fb97f7ea" target='_blank'>
              Chromatin assembly by the histone chaperone HIRA facilitates Human Papillomavirus replication
              </a>
            </td>
          <td>
            Ashley N. Della Fera, Dan Chen, Alison A. McBride
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT The integrator complex (INT) is an essential regulator of RNA biogenesis across evolution. Most current findings describe INT’s function in states of equilibrium, presenting a research gap in INT’s role in dynamic states, such as in infections and cancers. Viruses hijack cellular RNA machinery to transcribe their genes and produce viral progeny, presenting a unique condition to investigate INT-dependent RNA regulation under perturbation. Kaposi’s sarcoma-associated herpesvirus (KSHV) is an oncogenic DNA virus that causes two deadly cancers, Kaposi’s sarcoma and primary effusion lymphoma. KSHV undergoes a highly regulated and robust transcription of viral genes upon lytic reactivation, providing a complex and dynamic system to investigate integrator-mediated viral/host RNA regulation. We find that integrator subunit 11 (INTS11), the enzymatic core of INT, is essential for KSHV lytic replication triggered by reactivation or primary infection. Further RNA-seq analyses revealed a dynamic and unique signature of human transcriptomes during each lytic stage, respectively. Although the knockdown of INTS11 resulted in selective upregulation and downregulation of certain human gene transcription, INTS11’s loss globally repressed the KSHV transcriptome throughout KSHV lytic replication. This inhibited viral lytic gene expression, viral genome replication, and virion production. Integrator subunits 9 and 6 are also important for KSHV lytic replication. Mechanistically, ChIP-seq analysis showed that INTS11 is increasingly recruited to the KSHV genome with some unique binding patterns as the lytic cycle progresses, suggesting that KSHV hijacks INTS11 during lytic gene transcriptions. In all, our findings reveal the essential roles of the Integrator complex in KSHV lytic replication. IMPORTANCE The integrator complex (INT) is essential for RNA metabolism and is fundamental to all organisms, but its function and regulation during viral infection are not well described. Kaposi’s sarcoma-associated herpesvirus (KSHV) infection establishes lifelong infection and causes two deadly cancers; however, no vaccine is available. Using KSHV as a model, we found that integrator subunit 11 (INTS11), the enzymatic core of INT, is recruited to the KSHV genome under lytic phases and plays an essential role in facilitating global KSHV lytic mRNA transcription and viral production. This reveals the critical role of INT in viral infection, a common and inevitable event in human life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1301fb87c8bd6f125375d3b77eeefe0bb4b06d09" target='_blank'>
              Essential function of the integrator complex in Kaposi’s sarcoma-associated herpesvirus lytic replication
              </a>
            </td>
          <td>
            Amy Nguyen, Tianqi Li, Conner M Traugot, Kimberly Paulsen, Tiffany S Nelson, Mingyi Xie, Zhe Ma
          </td>
          <td>2025-08-13</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Mutations in numerous genes contribute to human cancer, with different oncogenic lesions prevalent in different cancer types. However, the malignant phenotype is simple, characterized by unrestricted cell growth, invasion, and often metastasis. One possible hypothesis explaining this dichotomy is that cancer genes regulate common targets, which then function as master regulators of essential cancer phenotypes. To identify mechanisms that drive the most fundamental feature shared by all tumors-unrestricted cell proliferation-we used a multiomic approach, which identified translation and ribosome biogenesis as common targets of major oncogenic pathways across cancer types. Proteomic analysis of tumors and functional studies of cell cultures established nucleolar and coiled-body phosphoprotein 1 as a key node, whose convergent regulation, both transcriptionally and posttranslationally, is critical for tumor cell proliferation. Our results indicate that lineage-specific oncogenic pathways regulate the same set of targets for growth control, revealing key downstream nodes that could be targeted for therapy or chemoprevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e15820af1f4a4174736d844aaa4b18b01cd6fd" target='_blank'>
              Diverse oncogenes use common mechanisms to drive growth of major forms of human cancer.
              </a>
            </td>
          <td>
            O. Kauko, Mikko Turunen, Päivi Pihlajamaa, Antti Häkkinen, R. Queiroz, M. Pääkkönen, S. Ventelä, Massimilliano Gaetani, Susanna L. Lundström, Antonio Murgia, Biswajyoti Sahu, J. Routila, Gong-Hong Wei, H. Irjala, J.L. Griffin, Kathryn S. Lilley, Teemu Kivioja, S. Hautaniemi, J. Taipale
          </td>
          <td>2025-08-22</td>
          <td>Science advances</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Our research investigates the spatiotemporal organization of nuclear architecture via biomolecular condensation, focusing on the role of liquid–liquid phase separation (LLPS) in gene regulation during cancer progression. We identify phosphatidylinositol 4,5 bisphosphate (PIP2) as a key nuclear lipid forming membrane less condensates with RNA, affecting transcriptional regulators and super-enhancer assembly. PIP2 shows RNA- dependent nuclear localization and interacts with RNA polymerases I and II, suggesting a structural role in transcriptionally active chromatin. Mass spectrometry revealed the RNA-dependent PIP2-associated (RDPA) nuclear proteome, enriched for intrinsically disordered regions and polybasic motifs, supporting LLPS. Among RDPA proteins, BRD4 undergoes PIP2- and RNA-dependent condensation. Using a system for inducible nuclear PIP2 depletion, we demonstrate its role in organizing transcriptional condensates and modulating super-enhancer hubs. These findings establish PIP2-driven phase separation as a key mechanism in nuclear domain organization and cancer-related gene regulation. This project is supported by the project National Institute for Cancer Research (Programme EXCELES, ID Project No. LX22NPO5102) by the European Union – Next Generation EU.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ebf08b77918307ebbd285055dc8c3a13dd93180" target='_blank'>
              SPATIO-TEMPORAL ORGANIZATION OF THE SUPER-ENHANCER STRUCTURE IN CANCER CELLS
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Transposable elements (TEs) represent a threat to genome integrity due to their proliferative capacity. Eukaryotic cells silence TEs through different molecular mechanisms, including the deposition of repressive histone marks. Previous studies have shown that TE repressive marks can spread to neighboring sequences. However, evidence for this spreading leading to nearby gene silencing remains limited. Similarly, whether TEs induce changes in the enrichment of active histone marks genome-wide, and the potential impact on gene expression have not been widely studied. Results In this work, we perform a comprehensive study of the epigenetic effects of 2235 TEs and their potential effects on nearby gene expression on Drosophila melanogaster head, gut, and ovary. While most TEs (816) induce the enrichment of the H3K9me3 repressive mark, with stronger epigenetic effects in the ovary, a substantial number (345 TEs) induce the enrichment of the H3K27ac active mark, particularly in the gut. We find that 70% of the H3K9me3 enriched TEs associated with expression changes downregulate the nearby gene, and 50% of the H3K27ac enriched TEs associated with expression changes lead to gene upregulation. These changes in expression affect specific regulatory networks only in the head. Furthermore, TE epigenetic effects on gene expression are genomic context dependent. Finally, we find that 221 TEs also affect gene expression by disrupting regions enriched for histone marks. Conclusions Overall, our results show that TEs contribute to the generation of regulatory novelty through epigenetic changes, with these epigenetic effects not restricted to gene silencing and being genomic context dependent. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03705-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75c9f4509d1e24d95361876bc4f99d2975a0de9c" target='_blank'>
              The epigenetics effects of transposable elements are genomic context dependent and not restricted to gene silencing in Drosophila
              </a>
            </td>
          <td>
            Marta Coronado-Zamora, Josefa González
          </td>
          <td>2025-08-18</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1110fcfacea74bd5588618a3f029d0c49019fbe" target='_blank'>
              Mechanisms and genomic implications of break-induced replication.
              </a>
            </td>
          <td>
            Adel Atari, Haoyang Jiang, Roger A. Greenberg
          </td>
          <td>2025-08-22</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background DNA methylation (DNAm) plays an important functional role and is influenced by genetic variants known as methylation QTLs (meQTLs). The majority of meQTL studies have been conducted in human blood. Despite its unique landscape, the genetic regulation of sperm DNAm remains largely unexplored. In this study, we leveraged DNAm measured in sperm from 405 Holstein bulls using reduced representation bisulfite sequencing (RRBS) and performed sequence-level genome-wide association studies for 166,985 variable CpGs (s.d. >5%). We reported heritability estimates and have mapped both cis-meQTLs and trans-meQTLs. Results Heritability estimates ranged from 0 to 1 and averaged 0.26 across all selected CpGs, with 76% of estimates above 0.1. The meQTL mapping revealed that 32.9% of the CpGs had a cis-meQTL, 3.6% had a trans-meQTL and 1.0% had both cis- and trans-meQTLs. The cis-CpGs were located on average 261 kb (absolute mean) from their cis-meQTL top SNPs (defined by the most significant association). MeQTLs were enriched in featured genomic annotations, including regions surrounding transcription start sites and ATAC-seq peaks. We also identified spurious trans-associations by analyzing data across multiple genome assemblies, including the construction of a partial pangenome. Additionally, eight trans-meQTL hotspots, defined as variants associated with at least 30 trans-CpGs, were identified and overlapped with genes involved in epigenetic regulation. Using peripheral blood mononuclear cell DNAm from 54 out of the 405 bulls, we did not observe a similar effect of the trans-meQTL hotspots to that one observed in sperm. Conclusions For the first time, meQTLs have been detected and characterized in bovine sperm, contributing to a better understanding of the transmission of paternally inherited DNAm marks. These findings provide useful information for further research aimed at integrating epigenetic information into the prediction of performance traits. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11934-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03d42a2f9879b2547c8e0d23b4ffde22fa6774da" target='_blank'>
              Genetic regulation of sperm DNA methylation in cattle through meQTL mapping
              </a>
            </td>
          <td>
            C. Fouéré, Valentin Costes, C. Hozé, Amrita Raja Ravi Shankar, Florian Besnard, G. Costa Monteiro Moreira, Valentin Sorin, C. Le Danvic, A. Chaulot-Talmon, Francesca Ali, M. Deloche, Aurélie Bonnet, E. Sellem, Hélène Jammes, Sébastien Fritz, M. Boussaha, D. Boichard, Hélène Kiefer, M. Sanchez
          </td>
          <td>2025-08-22</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background KAT6A-CBP (K/C) and KAT6A-P300 (K/P) fusions are recurrent genetic alterations in acute myeloid leukemia (AML) associated with poor prognosis. Despite their strong oncogenic potential, the mechanisms underlying their genomic targeting and leukemogenic function remain unclear. A major challenge has been their large size, which has impeded preclinical model development and mechanistic studies. Results We employ a domain-focused truncation strategy to generate de novo murine models of K/C and K/P fusions, which faithfully recapitulate the morphological, immunophenotypic, and transcriptomic features of KAT6A-rearranged AML. Genomic profiling reveals that KAT6A fusions preferentially localize to H3K4me2/3-marked regions, while biochemical analyses uncover that KAT6A interacts with the Nucleosome Remodeling Factor (NURF), a key H3K4me2/3 reader. Disrupting NURF-chromatin interactions via depletion or small-molecule inhibition of its subunit, Bromodomain PHD Finger Transcription Factor (BPTF), impairs K/C recruitment and disrupts MLL/COMPASS-mediated H3K4me2 deposition, defining a functional epigenetic module involving KAT6A chimeras, NURF, and MLL/COMPASS. Notably, CBP/P300 inhibition reduces histone acetylation and chromatin accessibility, further impairing the recruitment of this epigenetic module. Targeting this module via NURF or CBP/P300 inhibition demonstrates efficacy in K/C leukemia models, with enhanced therapeutic effects observed when combined. Conclusions Our study identifies a self-reinforcing epigenetic module of histone modifiers and readers in KAT6A-rearranged AML, providing mechanistic insights into the genomic targeting of KAT6A chimeras and highlighting promising combinatorial therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03743-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4b1939d769bfdab61413c545354658eef418fa8" target='_blank'>
              KAT6A chimeras form a self-reinforcing epigenetic module with NURF and MLL/COMPASS to sustain AML
              </a>
            </td>
          <td>
            Junhui Lv, Zhinang Yin, Conghui Li, Honglin Wen, Jian Ni, Peiyuan Yang, Zemin Song, Ying Xiang, Honghong Wang, Rui Lu, Li Huang, Ying Zhou, Hai-Bing Zhou, Ruijing Xiao, Pingping Fang, Kaiwei Liang
          </td>
          <td>2025-08-19</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Invasive cervical cancer (ICC) is the leading cause of cancer-related deaths among women in low-income regions, with approximately 70% of cases caused by high-risk human papillomavirus (HPV) strains—primarily HPV 16 and HPV 18. HPV infection triggers the production of viral oncoproteins E6 and E7, which interfere with the tumor suppressor proteins p53 and pRb, respectively, leading to disrupted cell cycle control and uncontrolled cellular proliferation. Moreover, only a few targeted therapies are approved for treatment.



 To address this gap, we conducted exome sequencing on 400 ICC fresh frozen samples from Guatemala, the country with one of the highest incidences of cervical cancer globally. We also performed long-read whole genome sequencing (WGS) on 29 cervical cancer cell lines with a coverage depth ranging from 50 to 100x. Additionally, ICC data from public database like TCGA was included in the analysis.



 Our lab previously identified that YAP1 oncogene amplification (11%) is associated with earlier diagnosis (12 years earlier on average), poorer survival, and a novel aggressive cervical cancer subtype, more prevalent in minority populations. Treatment with the PIK3CA inhibitor Alpelisib reduced proliferation in YAP1 amplified cervical cancer cell lines, like CaSki, SNU1000, SNU1299, QG-U, C4I and CERV-215 but had no effect on PIK3CA wildtype cells. We also observed that STK11, a tumor suppressor gene, is frequently altered in ICC. Exome sequencing data from Guatemala revealed that 32% of tumors (25% in adenocarcinoma (ASC) and 7% in squamous cell carcinoma (SCC)) have STK11 alterations, consistent with our WGS data showing alterations in 35% of cervical cancer cell lines. These mutations are associated with a 10-year earlier age of diagnosis in minority populations. Furthermore, we noticed that STK11 and YAP1 are both functionally involved in the same Hippo signaling pathway. Interestingly, we found in TCGA that 24% of STK11 wild-type cases have YAP1 amplifications while only 8% of STK11 altered cases have YAP1 amplification (p=0.009), exhibiting a co-occurrence of STK11 alterations with YAP1 amplification.



 We propose a model that HPV E7 oncoprotein and STK11 alterations activate YAP1, promoting oncogenesis. Therefore, co-occurrence of STK11 alterations and HPV infection may synergize to promote more aggressive cancers through hyper-activation of YAP1. Presently, we are validating our observations in highly expressed STK11 cervical cancer cell lines (direct RNA sequencing data) through siRNA transfection and protein expression. This suggests that evaluating YAP1 expression in STK11-deleted cell lines could uncover additional opportunities for targeted therapies and screening invasive cervical cancer cases for STK11 and YAP1 gene alterations may help identify women who are more likely to respond favorably to targeted therapies.



 Sonam Tulsyan, Emma Robinson, Hong Lou, Wen Luo, Yi Xie, Qian Yang, Isabele Rodriguez, Eduardo Gharzouzi, Enrique Alvirez, Michael Dean. Long-read whole genome sequencing identifies STK11 and YAP1 genes as novel biomarkers for treating invasive cervical cancer [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr A085.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f27666748c2b6097853c42aef3cfa545e01c9e3" target='_blank'>
              Abstract A085: Long-read whole genome sequencing identifies STK11 and YAP1 genes as novel biomarkers for treating invasive cervical cancer
              </a>
            </td>
          <td>
            Sonam Tulsyan, Emma Robinson, Hong Lou, Wen Luo, Yi Xie, Qian Yang, Isabel Rodríguez, E. Gharzouzi, E. Alvirez, Michael Dean
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Transcriptional enhancers are DNA regulatory elements that control the levels and spatiotemporal patterns of gene expression during development, homeostasis, and pathophysiological processes. Enhancer identification and characterization at the genome-wide scale rely on their structural characteristics, such as chromatin accessibility, binding of transcription factors and cofactors, activating histone modifications, 3D interactions with other regulatory elements, as well as functional characteristics measured by massively parallel reporter assays and sequence conservation approaches. Recently, machine learning approaches and particularly deep learning models (Enformer, BPNet, DeepSTARR, etc.) allow the prediction of enhancers, the impact of variants on their activity and the inference of transcription factor binding sites, leading, among others, to the construction of the first completely synthetic enhancers. We present the above computational tools and discuss their diverse applications towards cracking the enhancer regulatory code, which could have far-reaching ramifications for uncovering essential regulatory mechanisms and diagnosing and treating diseases. With an emphasis on sepsis, a leading cause of morbidity and mortality in hospitalized patients, we discuss computational approaches to identify sepsis-associated endotypes, circuits, and immune cell states and signatures characteristic of this condition, which could aid in developing novel therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b2615cfff1cc435acd6a5c1c36b7693a74f3bbf" target='_blank'>
              Machine learning tools for deciphering the regulatory logic of enhancers in health and disease
              </a>
            </td>
          <td>
            S. Foutadakis, Vasiliki Bourika, Ioanna Styliara, Panagiotis Koufargyris, A. Safarika, Eleni Karakike
          </td>
          <td>2025-08-13</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer cells arise from the cumulative acquisition of genetic and epigenetic alterations that affect vital cellular functions. Genomic instability results from deficiencies in protective mechanisms, such as cell cycle checkpoints, DNA replication control, or DNA repair. Claspin integrates a group of crucial proteins that maintain genome integrity. It participates in key cellular events such as DNA damage checkpoint activation, DNA replication, replication stress responses, DNA repair, epigenetic memory, and apoptosis. Given its crucial functions, a role for Claspin in cancer is not a surprise. Indeed, there is a considerable body of evidence linking Claspin deregulation with cancer. For instance, over-expression of Claspin and Tim promoted the survival of cancer cells by enabling adaptation to oncogene-induced replication stress. In addition, Claspin gene (CLSPN) mutations that affect checkpoint regulation have been identified in cancer patients, suggesting that they may contribute to cancer development. Changes in Claspin expression levels may be used as a prognostic marker in several types of cancer. Finally, several therapy-resistance signaling pathways seem to converge onto Claspin’s stabilization, turning Claspin into an attractive target for chemo- and radio-sensitization. In this review, we will focus on the role of Claspin in cancer and ways in which Claspin can be exploited in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86699ba9ba454ef5c75f676effbf8547c5ac5e0c" target='_blank'>
              Claspin and Cancer: Where Are We Now?
              </a>
            </td>
          <td>
            D. Azenha, Teresa C. Martins
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Lethal prostate cancer (PCa) has passed through at least two evolutionary bottlenecks: acquisition of metastatic potential and development of castration resistance. A better understanding of how this impacts genetic heterogeneity across metastatic sites is needed to develop strategies to block metastatic spread and overcome resistance. By leveraging deep whole-exome sequencing of 93 tumors from 26 patients, we examined patterns of metastatic dissemination and clonal evolution of PCa. Phylogenetic reconstruction and mathematical modeling enabled quantification of the number of mutations and clones in the cancer ecosystem and characterization of each patient's disease as an evolutionary process. While mutations of the earliest pathogenic genetic drivers of PCa were truncal, most other likely passenger mutations, copy-number alterations, and clones arose after the cancer had spread and were confined to individual metastatic sites due to polyclonal and polyphyletic seeding. Single-tissue sequencing tended to overestimate mutation clonality and, apart from truncal drivers, underestimate mutation rates for both individual patients and cohorts. This study highlights the independent evolution of metastatic lesions, which has implications for diagnostic and targeted therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f198ff2332798cb036a19b4692ee525e77c812c" target='_blank'>
              Divergent Clonal Evolution and Early Dissemination Promote Genetic Heterogeneity of Metastases in Castration Resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Noshad Hosseini, R. Mannan, Ryan J. Rebernick, Fengyun Su, Rui Wang, Xuhong Cao, Ana Lako, Dattatreya Mellacheruvu, Jing Hu, J. Alumkal, Zachery R Reichert, Rohit Malik, Rohit Mehra, A. Chinnaiyan, M. Cieslik
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9490feaeec24c63eafa0bf22c93391f3883ea64" target='_blank'>
              [Exploration of ribosomal DNA copy number as an early biomarker for exposure to chemical carcinogens].
              </a>
            </td>
          <td>
            C. Ding, H. D. Xu, J. L. Lou
          </td>
          <td>2025-08-20</td>
          <td>Zhonghua lao dong wei sheng zhi ye bing za zhi = Zhonghua laodong weisheng zhiyebing zazhi = Chinese journal of industrial hygiene and occupational diseases</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="B-cell acute lymphoblastic leukemia (B-ALL) remains a major clinical challenge in hematologic oncology, characterized by a continuous evolution of molecular drivers that shape its heterogeneity across the age spectrum. Pediatric B-ALL is generally associated with high cure rates, while adult forms of the disease are often more aggressive and less responsive to treatment. This review examines the age-specific genetic and epigenetic landscapes that contribute to this disparity, revealing how the nature and timing of molecular alterations point to fundamentally different leukemogenic processes. Favorable genetic aberrations, such as ETV6::RUNX1 and hyperdiploidy, are predominant in children, whereas adults more frequently present with high-risk features, including BCR::ABL1 fusions and IKZF1 deletions. Epigenetic distinctions are similarly age-dependent, involving divergent patterns of DNA methylation, histone modifications, and non-coding RNA expression. For example, pediatric B-ALL frequently harbors mutations in epigenetic regulators like SETD2 and CREBBP, while adult B-ALL is more commonly affected by alterations in TET2 and IDH1/2. These molecular differences are not only prognostic but also mechanistic, reflecting distinct developmental trajectories and vulnerabilities. Understanding these age-driven transitions is essential for improving risk stratification and developing precision therapies tailored to the unique biology of B-ALL across the lifespan.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb81ee6e89e6609c04144954df008c19fbd5a755" target='_blank'>
              Age-Driven Genetic and Epigenetic Heterogeneity in B-ALL
              </a>
            </td>
          <td>
            Yoana Veselinova, M. Esteller, Gerardo Ferrer
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Glioblastoma are hallmarked by clonal evolution and transcriptional heterogeneity linked to genetic and environmental diversity resulting in therapy resistance and disease progression. In this study, we integrate heritable molecular barcoding and spatial transcriptomics to enable high-resolution cell lineage tracing in the spatial context of the glioblastoma TME.



 Patient-derived glioblastoma cells from two individual cases were isolated from anatomically and pathologically distinct regions: the primary tumor core, the tumor periphery, and the recurrent lesion. Each subline was transduced with predefined FlexTrack lentiviral plasmids encoding GFP, resulting in uniquely barcoded (9N) cell lines. Immediately before injection, the barcoded glioblastoma populations were pooled and introduced into ex vivo human cortical slices. Live-imaging was used to trace GFP-positive, barcoded cell populations. After incubation, the fresh-frozen slices were processed for spatial transcriptomics and the line-specific barcodes were mapped according to the 3′ poly(A) tails of captured transcripts. Graph neural networks based on single-cell deconvolution were employed to spatially analyze the cellular ecosystems.



 We successfully deconvoluted and reconstructed the trajectories of injected glioblastoma cells linking their transcriptome phenotypes along with clonal identity. Multi-omic integration of spatial copy-number alterations and barcoding based pseudo-trajectories enabled reconstruction of the temporal transcriptional dynamics linked to the heterogeneity of tumor ecosystems.



 This study serves as a proof-of-concept for integrating molecular barcoding and spatial transcriptomics in cortical slice culture. Our approach to molecular barcoding in complex model systems opens new avenues to investigate the contextual dynamics of clonal evolution in glioblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f785ed6bb81bfa9c1d1d1cee449d66b313a3ee5" target='_blank'>
              P02.23.B MOLECULAR BARCODING OF GLIOBLASTOMA IN HUMAN ORGANOTYPIC CORTICAL SLICE CULTURE
              </a>
            </td>
          <td>
            G. Kastner, M. Mühlbauer, J. Jung, S. Langer, L. Müller, A. Baginska, P. Berger, C. Dobersalske, S. Ertmer, F. M. Gerach, N. Neidert, K. Herrmann, L. Rauschenbach, L. van Hijfte, U. Sure, K. D. Kokkaliaris, B. M. Grüner, D. Heiland, B. Scheffler
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Rhabdomyosarcoma (RMS) is the most frequent soft tissue sarcoma of childhood. The fusion positive (FP) subtype expressing the chimeric transcriptional factor PAX3-FOXO1, the driver of malignancy, has limited treatment options due to resistance to treatments. As a pediatric cancer, RMS has a low mutational burden with epigenetic transcriptional deregulations playing key roles. Thus, targeting the basic transcriptional machinery including catalytic effectors of transcription is an emerging anti-cancer approach. The transcriptional cyclin-dependent kinase CDK12 regulates the transcription elongation phase, particularly of the DNA damage response (DDR) genes, by phosphorylating Ser2 (pSer) on RNA PolII. In Ewing sarcoma, CDK12 inhibition leads to a DNA repair deficient phenotype and high sensitivity to DNA damaging agents. PAX3-FOXO1 RMS cells display elevated levels of replication stress highly relying to DDR for survival. Therefore, in this study we evaluate the impact of CDK12 inhibition on PAX3-FOXO1 RMS tumorigenic features. We found that CDK12 mRNA levels were highly up-regulated in RMS primary samples compared to healthy control muscles, with one of the highest expression among the adult and pediatric tumors analyzed. Then, we treated PAX3-FOXO1 cell lines with THZ531, an inhibitor of CDK12 and its paralog CDK13. Six hours of THZ531 treatment at nanomolar concentrations decreased pSer2 of RNA PolII and reduced PAX3-FOXO1 transcripts and protein levels. The drug also reduced cell survival in 2D and 3D culture conditions. THZ531-treated cells accumulated in G2/M phase and underwent PARP- and Casp 3/7-dependent apoptosis, increased markers of DNA damage associated to DNA double-strand breaks (neutral comet assay), and reduced the levels of the DDR players ATM, ATR, RAD51, BRCA1 and BRCA2. Similar results were obtained after treatment with another CDK12/13i with Cyclin K degrader ability, SR-4835, and with a CDK12-specific degrader, BSJ-4-116, suggesting on-target effects. The three drugs showed similar low nanomolar concentrations effective on PAX3-FOXO1 RMS organoids. Preliminary RNAseq data in two PAX3-FOXO1 RMS cell lines showed that THZ531 downregulated only a low number of genes, common to the two cell lines. The most downregulated gene families were those of DDR pathways. Moreover, a negative correlation between gene length and gene expression levels was found. CDK12 KO in two RMS cell lines affected cell survival in 2D and 3D conditions compared to control cells suggesting on-target activity of the drugs. Phosphoproteomics analysis after THZ531 treatment showed a significant decrease of a number of serines and threonine phosphopeptides also involving a subset on DDR proteins. THZ531 treatment resulted synergic in vitro with Olaparib (PARPi) and with JQ1 (BRD4i). Interestingly, BRD4 is needed for PAX3-FOXO1 activity on oncogenic super-enhancers. Analysis of nascent RNAs and mapping of RNA PolII to identify direct targets of CDK12 is ongoing. Supported by Associazione Italiana per la Ricerca sul Cancro (AIRC) IG #27794 grant to RR



 Marika Attili, Lucrezia D'Archivio, Riccardo Mazzocchi, Francesca Antonella Aiello, Matteo Cassandri, Silvia Pomella, Adam D. Durbin, Francesco Marampon, Kim PJ Schellekens, Michael T Meister, Jarno Drost, Concetta Quintarelli, Simone Sidoli, Michele Ceribelli, Craig J Thomas, Franco Locatelli, Biagio De Angelis, Kristy R Stengel, Rossella Rota. CDK12 pharmacological targeting promotes cell death by abrogating DNA damage response, impairs PAX3-FOXO1 expression and synergizes with BRD4 and PARP inhibitors in fusion positive rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B015.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cd33df041a8224c723527c37b828d7c14637d9c" target='_blank'>
              Abstract B015: CDK12 pharmacological targeting promotes cell death by abrogating DNA damage response, impairs PAX3-FOXO1 expression and synergizes with BRD4 and PARP inhibitors in fusion positive rhabdomyosarcoma
              </a>
            </td>
          <td>
            Marika Attili, Lucrezia D'Archivio, Riccardo Mazzocchi, F. Aiello, Matteo Cassandri, S. Pomella, A. Durbin, Francesco Marampon, Kim PJ Schellekens, Michael T. Meister, J. Drost, C. Quintarelli, Simone Sidoli, Michele Ceribelli, Craig J Thomas, Franco Locatelli, B. De Angelis, Kristy Stengel, R. Rota
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Numerous covalently closed circular RNAs (circRNAs) are produced from back-splicing of eukaryotic exons, and multiple circRNAs can be generated from a single gene locus through alternative circularization (AC). However, functions of most AC circRNAs remain poorly understood. Here, we profile the landscape of AC across multiple cell lines and colorectal cancer (CRC) tissues, identifying predominantly expressed circRNAs (pe-circRNA) in each AC gene locus. Other than cell-type-specific expression of most top pe-circRNAs, circMAN1A2(2,3,4,5), a universally expressed pe-circRNA in examined samples, plays an important role in cell proliferation and CRC progression. Mechanistically, circMAN1A2(2,3,4,5) directly interacts with the 3′ untranslated region of Centromere Protein B (CENPB) mRNA via its characteristic back-splicing junction site, which enhances IGF2BP2-mediated CENPB mRNA stability. Inhibition of circMAN1A2(2,3,4,5) expression with locked nucleic acids represses CRC progression. These results uncover the prevalence of AC and the regulatory role of a specific AC circRNA, circMAN1A2(2,3,4,5), in cell proliferation and tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/333a174f8178b96c1b80005e5f597d344cda4871" target='_blank'>
              Direct circMAN1A2(2,3,4,5)-CENPB mRNA interaction regulates cell proliferation and cancer progression
              </a>
            </td>
          <td>
            Mei Cao, Guo-Hua Yuan, Shi-Meng Cao, Yu-Xin Liu, Chu-Xiao Liu, Yi-Feng Xu, Jia Wei, Yi-Lin Guo, Ling-Ling Chen, Li Yang
          </td>
          <td>2025-09-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The impact of ionizing radiation (IR) with induction of various DNA damage is based not only on genetic but also on epigenetic effects. Epigenetic modifications determine the chromatin structure and DNA accessibility, thereby regulating cellular functions through the expression of individual genes or entire groups of genes. However, the influence of DNA repair processes on the restoration of local chromatin structures and global nuclear architectures is still insufficiently understood. In multicellular organisms, epigenetic mechanisms control diverse cellular functions of specific cell types through precise temporal and spatial regulation of gene expression and silencing. How altered epigenetic mechanisms regulate the pathophysiological function of cells, tissues, and ultimately entire organs following IR exposure remains to be investigated in detail. Radiation-induced epigenetic processes are particularly critical for immature cell populations such as tissue-specific stem and progenitor cells during development and differentiation of organ tissues. Genome-wide patterns of DNA and histone modifications are established cell types—specifically during the development and differentiation of organ tissues but can also be fundamentally altered in adult organism by stress responses, such as radiation-induced DNA damage. Following IR exposure, epigenetic factors are not always fully restored to their original state, resulting in epigenetic dysfunction that causes cells to lose their original identity and function. Moreover, severe radiation-induced DNA damage can induce premature senescence of cells in complex tissues, which ultimately leads to signs of aging and age-related diseases such as cancer. In this work, we provide an overview of the most important epigenetic changes following IR exposure and their pathophysiological significance for the development of acute and chronic radiation reactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78d105ae28e3df5bffe779131184bc03d9d06867" target='_blank'>
              Role of Ionizing Radiation in Shaping the Complex Multi-Layered Epigenome
              </a>
            </td>
          <td>
            C. Rübe, M. A. A. Al-razaq, Carola Meier, Markus Hecht, Christian Rübe
          </td>
          <td>2025-08-08</td>
          <td>Epigenomes</td>
          <td>1</td>
          <td>34</td>
        </tr>

        <tr id="Knowledge of the structure, regulation, and function of genes is a fundamental pillar of molecular biology and modern medicine. Genes are DNA sequences that contain the information necessary to produce functional proteins or RNA, and are not only made up of coding regions (exons), but also introns and regulatory elements such as promoters, enhancers, and silencers. Mutations in any of these regions can alter normal cellular function and lead to various diseases. The regulation of gene expression ensures that not all genes are activated at the same time, but rather in a controlled manner depending on the cell type, stage of development, or environmental conditions. This control is mediated by transcription factors, epigenetic modifications such as DNA methylation, and the action of non-coding RNAs, mechanisms that allow multiple specialized cell types to be generated from the same DNA sequence. The function of genes is reflected in the products they encode, whether proteins or RNA, which participate in vital processes such as metabolism, cell signaling, or DNA repair. Genetic variants can cause loss of function, as in the case of cystic fibrosis due to mutations in the CFTR gene, or gain of function, as occurs in certain oncogenes involved in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ccb5bb302f5b6f126acbfc92ab37e76b9351d4d" target='_blank'>
              Estructura de un gen y sus elementos de control
              </a>
            </td>
          <td>
            Ana Laura Téllez García, Kevin Chung, Karen Hou, Stephania Hylton, J. Mosquera, Crital Zhang
          </td>
          <td>2025-08-31</td>
          <td>Genética y Genómica Clínica</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Multicellular life depends on the ability to activate and repress genes in a highly context-specific manner. With each cell state transition, a new transcriptional profile is established. As non-coding DNA elements, enhancers mediate their regulatory potential through the effectors they recruit. While ultimately instructed by the underlying DNA sequence, enhancer activity depends on several factors, such as transcription factor availability, chromatin state, and promoter proximity, all of which are dynamically regulated within the cell. Even when we understand the regulation of one enhancer, its genomic impact is dependent on its integration within the regulatory landscape. Thus, a full picture of enhancer dynamics can only be painted through broad, but controlled, approaches that integrate investigations into multiple levels of gene regulatory mechanisms. In this review, we will present the exit of naive pluripotency as a prime setting to do just that and contextualize how its contemporary use has been, and could be, used to reveal the intricacies of enhancer mechanistics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a198ff34681e848852fded55aab3ace3539920dc" target='_blank'>
              The exit from naive pluripotency: a platform for the study of enhancer mechanistics
              </a>
            </td>
          <td>
            Mattias Enar Jonasson, C. Buecker
          </td>
          <td>2025-08-18</td>
          <td>Biochemical Society Transactions</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Complex chromosomal rearrangements (CCRs) are rare structural abnormalities involving at least three chromosomal breakpoints and often two or more chromosomes. Owing to their inherent genomic complexity, CCRs are frequently associated with abnormal phenotypes, including developmental delay, congenital anomalies, and infertility. In this study, we report four male patients, three of them with de novo rare structural chromosomal rearrangement detected through a combination of Giemsa-Trypsin (GTG) banding, fluorescence in situ hybridization (FISH), and high-resolution microarray techniques (SNP array and array CGH). Each of the four cases turned out to be of a different type: in addition to two exceptional CCRs, an inv dup del 18q and a cluster rearrangement involving the long arm of chromosome 4 were identified. Despite the limitations of the testing methods, we performed a detailed analysis of the relationship between the most detailed genotype data and the associated phenotype. Our study provides further valuable evidence that the use of molecular cytogenetic methods is of paramount importance even in cases with abnormal karyotypes detected by light microscopy, as high-resolution data may reveal unsuspected genomic complexity, which is essential for genetic counseling in these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/974abec08ecfb2c8d46a921b99de6ab00061bf03" target='_blank'>
              Uncovering Rare Structural Chromosomal Rearrangements: Insights from Molecular Cytogenetics
              </a>
            </td>
          <td>
            M. Czakó, A. Szabó, Á. Till, Anna Zsigmond, K. Hadzsiev
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Human cytomegalovirus extensively alters nuclear organization and the cellular transcriptome, yet understanding of these genome-wide events remains relatively limited. Here, chromatin conformation capture (Hi-C) revealed how cytomegalovirus alters chromosome organization at both large- and small-scales. Nascent transcriptomics further revealed how transcriptional changes correlate with genomic reorganization, while also uncovering infection-induced transcriptional dysregulation that contributes to the induction of neuronal gene signatures in infected fibroblasts. Combining Hi-C and Cleavage Under Targets & Release Using Nuclease (CUT&RUN) we find that viral genomes preferentially localize to highly euchromatic compartments, further dysregulating transcription of host genes. Finally, RNAi-mediated depletion of two key effectors of microtubule-based forces that are exerted on the nucleus provides insights into their diverging roles in regulating compartment-scale contacts and viral genomic interactions with host chromatin. Combined, we reveal the extent to which HCMV interacts with and alters host chromatin and transcription, and the influence of microtubule mechanotransduction on these processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0c2779fbea7b388894e9412fc49c8051594da3" target='_blank'>
              Microtubule mechanotransduction refines cytomegalovirus interactions with and remodeling of host chromatin
              </a>
            </td>
          <td>
            Celeste D Rosencrance, Derek Walsh
          </td>
          <td>2025-08-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background Human endogenous retroviruses (HERVs), particularly the HERV-K family, are increasingly recognized for their roles in cancer biology, yet the function of ERVK3-1 (HERVK_3p21.31) in liver hepatocellular carcinoma (LIHC) remains largely unexplored. Methods We analyzed transcriptomic data from the TCGA-LIHC cohort to identify differentially expressed genes (DEGs) between high and low ERVK3-1 expression groups, followed by functional enrichment analyses (GO, KEGG, GSEA), protein-protein interaction (PPI) network construction, and hub gene identification. The immunological relevance of ERVK3-1 was assessed through TIP immune cycle analysis, single-cell RNA sequencing datasets, and correlation with immune checkpoint expression. Immunotherapy responsiveness was evaluated using TIDE and TCIA databases. Results High ERVK3-1 expression was associated with enrichment in metabolic and oxidative stress-related pathways, while low expression correlated with cell cycle and DNA replication. PPI analysis revealed mitosis-related hub genes (e.g., CCNB1, CDK1). ERVK3-1 expression promoted early immune cell recruitment but was inversely correlated with later stages of the cancer immunity cycle, including immune infiltration and T-cell killing. Single-cell data showed high ERVK3-1 expression in immunosuppressive subsets, alongside positive associations with inhibitory immune checkpoints (e.g., PD-1, CTLA-4, TIM-3). High ERVK3-1 expression also correlated with greater immune evasion and reduced immunotherapy responsiveness. Conclusions ERVK3-1 plays a multifaceted role in LIHC progression, contributing to metabolic reprogramming, immune suppression, and resistance to immunotherapy. These findings highlight ERVK3-1 as a potential prognostic biomarker and therapeutic target in liver cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12985-025-02928-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/186dc1e93c9e9bba5bd5b29915ff6bacc720805a" target='_blank'>
              Human endogenous retrovirus ERVK3-1 characterizes a metabolically active and immunosuppressive subtype of liver cancer
              </a>
            </td>
          <td>
            Xiaofen Wen, Shulv Weng, Minna Chen, Danxia Lin, Wenwu Xue, De Zeng, Jiaxin Shen
          </td>
          <td>2025-10-01</td>
          <td>Virology Journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction: Long interspersed element-1 (L1) and Alu elements are reported to regulate tumor development and progression by forming chimeric transcripts with diverse sequences. While these elements have been implicated in various cancers, the landscape of chimeric transcript formation in breast cancer remains poorly characterized.
Objective: Therefore, in this study, we comprehensively investigated the profiles of L1 and Alu chimeric transcripts in breast tumors by analyzing RNA sequences from The Cancer Genome Atlas (TCGA) database.
Methods: More than 50 billion paired-end RNA sequencing reads from 606 primary breast tumor tissues and 59 matched adjacent normal tissues from the TCGA database were aligned to L1 and Alu consensus sequences with Bowtie2 software and further analyzed.
Results: Our analysis revealed significantly elevated expression levels of L1 and Alu retrotransposons in breast tumor tissues compared to adjacent non-tumor tissues. Importantly, higher expression of L1 and Alu in breast tumors was associated with poorer 10-year overall survival, suggesting their potential as prognostic biomarkers. Further exploration demonstrated that L1 and Alu were extensively expressed in chimeric form with adjacent transcripts. A significant difference in the frequency of 651 L1-mRNA and 1,525 Alu-mRNA chimeric transcripts was observed between non-tumor and tumor tissues. Similarly, 1,009 L1-long non-coding RNA (lncRNA) and 2,575 Alu-lncRNA chimeric transcripts exhibited significantly different frequencies between non-tumor and tumor tissues. Functional enrichment analysis revealed that these differentially expressed chimeric genes played crucial roles in multiple facets of tumorigenesis, including metabolism, signal transduction, immunity, cell cycle regulation, and apoptosis.
Conclusion: Our findings demonstrate that Chimeric transcript formation with L1 and Alu retrotransposons represents a widespread phenomenon in breast cancer that impacts multiple oncogenic pathways. These results significantly expand our understanding of retrotransposon involvement in breast cancer and highlight potential novel targets for diagnostic and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99e34ef62da2b088b93dd7d2cad99c0994dc0a0e" target='_blank'>
              Landscape of long interspersed element-1 and Alu chimeric transcripts in breast tumors
              </a>
            </td>
          <td>
            Jian Li, Yin Zhang, Yiting Zhou, Jianyou Liao, Yabin Guo
          </td>
          <td>2025-09-03</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) is the most common subtype of ovarian cancer (∼80%) and is one of the most lethal gynecological malignancies in the United States. HGSOC is characterized by universal mutations in the TP53 gene and defects in DNA repair pathways, contributing to high incidence of chromosomal instability (CIN). CIN refers to an ongoing process involving the gain or loss of parts of chromosomes, leading to chromosomal mis-segregation, aneuploidy, and copy number variations (CNV). CINhigh cancers display a greater dependency on KIF18A, an essential cytoskeletal motor protein for cell division, in contrast to normal somatic cells. Therefore, KIF18A represents a promising target for therapeutic interventions in CINhigh cancers. We investigated the use of a novel KIF18A inhibitor (KIF18Ai), ATX020, for selectively targeting CINhigh HGSOC cells. Our approach involved advanced techniques such as RNA sequencing (RNA-seq), siRNA-based gene silencing, cell cycle analysis, and immunofluorescence. We utilized DEPMAP and flow cytometric analysis to categorize ovarian cancer cell lines according to their aneuploidy score (AS) and ploidy to investigate their response to ATX020. Cell growth inhibition studies demonstrated a correlation between AS/ploidy levels and sensitivity to ATX020 in a panel of HGSOC cells including PEO4, OVCAR3, OVCAR8 and A2780. Assessments of cell viability, cell cycle analysis, phospho-histone H3 expression, clonogenic assays, and invasion assays indicated that the growth inhibition observed over 72 hours was primarily due to the toxic effects of ATX020 on tumorigenicity and mitotic arrest. Mechanistically, ATX020 exerts its toxic effects by inhibiting the plus-end movement of KIF18A on spindle fibers, which increases the inter-kinetochore (Inter-KT) distance and induces adverse effects such as chromosomal mis-segregation, aneuploidy, and double-strand DNA damage. We also observed that KIF18Ai-resistant cells (A2780, PEO4) showed inhibited KIF18A plus-end movement and increased inter-KT distance but did not exhibit the accompanying toxic effects. In summary, our findings indicate that ATX020 inhibits CINhigh HGSOC cells primarily by inducing mitotic arrest, causing DNA damage, and reducing tumorigenicity. This effect is facilitated by ATX020's ability to inhibit the plus-end directed movement of KIF18A along spindle fibers, which is detrimental to CINhigh cells that rely on KIF18A for normal mitotic divisions. Further mechanistic investigation is ongoing.



 Jayakumar R. Nair, Maureen M. Lynes, Laura Ghisolfi, Serena J. Silver, Jung-Min Lee. Investigating a novel KIF18A inhibitor ATX020 in chromosomally instable ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/256c1d3be3ec4d6c1fa4908dbe7af1b6e3ef12d5" target='_blank'>
              Abstract B024: Investigating a novel KIF18A inhibitor ATX020 in chromosomally instable ovarian cancer
              </a>
            </td>
          <td>
            Jayakumar R. Nair, Maureen M. Lynes, Laura Ghisolfi, Serena J Silver, Jung-Min Lee
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract RECQ5 is a member of the RECQ helicase family that maintains genomic stability. However, the molecular mechanism of RECQ5 in this biological process remains elusive. Here, we show that RECQ5 localizes in the dense fibrillar component of nucleolus and associates with several pre-rRNA processing factors. It recognizes pre-rRNA and is able to unwind double-stranded RNA in vitro. Loss of RECQ5 induces the accumulation of 47S, 30SL5′, and 30S pre-rRNA, and the reduction of 21S pre-rRNA, suggesting that it regulates the processing of pre-rRNA. Cancer-associated mutations of RECQ5 abolish its nucleolar localization as well as its helicase activities. Moreover, lacking RECQ5 leads to the unprocessed pre-rRNA hybridizing with rDNA, triggering the R-loop formation and ATR activation. Since both RECQ5 and ATR protect genomic stability in nucleolus, suppression of RECQ5 sensitizes tumor cells to the ATR inhibitor treatment. Collectively, this study reveals that RECQ5 plays a crucial role in pre-rRNA processing and ribosomal DNA (rDNA) stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd78de72f99e82103addd1e28df765e8c65273f5" target='_blank'>
              RECQ5 mediates pre-rRNA processing in nucleolus
              </a>
            </td>
          <td>
            Yidi Ma, Xiao Yu
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="One of the most prevalent illnesses in women, breast cancer is thought to be a complex and diverse condition that continues to be a significant public health issue. Long non-coding RNAs have drawn a lot of interest lately because of the advancement of next-generation sequencing methods. Long non-coding RNAs have been shown in several studies to play significant roles in the formation of tumors. Recent studies have shown that a variety of long non-coding RNAs express aberrantly in malignancies, including breast cancer, despite the fact that the biological role and molecular processes of these RNAs are still unknown. With an emphasis on the many molecular functions of regulatory long non-coding RNAs that control cell proliferation, invasion, metastasis, and apoptosis, we have included the most recent research on long non-coding RNAs in breast cancer here.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b79bc9fa5e82ee9b8b63a734e31b110e1608509" target='_blank'>
              Role of Long non-coding RNA Gene Expression in Breast Cancer
              </a>
            </td>
          <td>
            Dalia Abdelnasser Awad Mohammed, A. M. Elzawawy, Hoda Yousry Abdullah, M. Atef, L. O. E. Abdel Salam, Hanan Ezzat Shafik
          </td>
          <td>2025-09-01</td>
          <td>Egyptian Reviews for Medical and Health Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In mammals, repression of germline-specific gene expression is essential for preserving somatic cell identity and preventing disease. Germline gene silencing is often dependent on the presence of promoter 5-cytosine DNA methylation (5meC). Early mammalian development, however, is marked by a dramatic loss of 5meC levels genome-wide. Recent research has highlighted a specialized variant of the Polycomb Repressive Complex 1, PRC1.6, as a key regulator that maintains transient silencing of germline genes in this embryonic window. Eventually, PRC1.6 seems to stimulate the recruitment of de novo DNA methyltransferases (DNMTs), although the precise mechanisms remain to be fully elucidated. Evidence suggests a coordinated epigenetic relay, potentially involving direct protein interactions and shaping the local histone modification landscape. This review explores PRC1.6 as a central epigenetic hub that connects Polycomb repression, histone 3 lysine 9 (H3K9) methylation, and DNA methylation pathways. Unraveling this interplay will provide key insights into the mechanisms that maintain the critical barrier between the germline and the soma, essential not only for proper development but also for preserving somatic cell function and health throughout life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ed478b995bf1b398b0bfe5ba03fe4e97c29208" target='_blank'>
              Epigenetic relay: Polycomb-directed DNA methylation in mammalian development
              </a>
            </td>
          <td>
            Teresa Urli, Maxim V. C. Greenberg
          </td>
          <td>2025-09-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="A substantial portion of lung cancer-associated genetic elements in East Asian populations remains unidentified, underscoring the need for large-scale genome-wide studies, particularly on non-coding regulation. We conducted a whole genome sequencing (WGS)-based genome-wide scan in 13,722 Chinese individuals to identify regulatory elements associated with lung cancer. We verified common-variant-based loci by meta-analysis across the available East Asian studies. Integrating a genome-transcriptome reference panel of lung tissue in 297 Chinese, we bridged the variant-lung cancer associations, highlighting genes including TP63 and DCBLD1. Implementing the STAAR pipeline for rare variant aggregate analysis, we identified and replicated novel genes, including PARPBP, PLA2G4C, and RITA1 in the context of non-coding regulation. Adapting a deep learning-based approach, potential upstream regulators such as TP53, MYC, ZEB1, and NFKB1 were revealed for the lung cancer-associated genes. These findings offered crucial insights into the non-coding regulation for the etiology of lung cancer, providing additional potential targets for intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/218dccec773052bdb16c39b8293ccbf37c8ec89f" target='_blank'>
              Non-coding genetic elements of lung cancer identified using whole genome sequencing in 13,722 Chinese
              </a>
            </td>
          <td>
            Dan Zhou, Ming Wu, Qilong Tan, Liyang Sun, Yuanxing Tu, Weifang Zheng, Yun Zhu, Min Yang, Kejia Hu, Fang Hu, Xiaohang Xu, Hanyi Zhou, Tian Luo, Fangming Yang, Fuqiang Li, Xin Jin, Huakang Tu, Wenyuan Li, Kui Wu, Xifeng Wu
          </td>
          <td>2025-08-09</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="Mammalian genomes undergo pervasive transcription in both genic and intergenic regions. Trimethylation of histone H3 lysine 4 (H3K4me3) is a deeply conserved and functionally important histone modification enriched at transcriptionally active promoters, where it facilitates RNA polymerase activity. H3K4me3 is also commonly found in intergenic regions, where its role is poorly understood. We interrogated the role of H3K4me3 at intergenic regulatory elements by using epigenetic editing to efficiently deposit H3K4me3 at intergenic loci. We found that H3K4me3 amplifies RNA polymerase activity and is actively remodeled at intergenic regions, shedding light on these important but poorly understood regions of the genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b45f64bb56f63b370548214321fa4126548d4ca8" target='_blank'>
              H3K4me3 amplifies transcription at intergenic active regulatory elements.
              </a>
            </td>
          <td>
            Haoming Yu, Yongyan Zhang, Zhicong Liao, B. W. Walters, Bluma J. Lesch
          </td>
          <td>2025-08-18</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The GATA3 noncoding variant rs3824662 has been implicated in the pathogenesis of Ph-like B-ALL, where it is associated with extensive chromatin reorganisation, resulting in the dysregulation of multiple genes, including CRLF2 overexpression. Given the altered chromatin landscape and increased accessibility of GATA3 binding regions associated with the rs3824662 variant, we investigated the potential role of enhancer RNAs (eRNAs) located near the GATA3 locus in regulating CRLF2 expression. We found that the expression of eRNA_G3, located at chr10:8,443,562–8,449,563, was positively correlated with CRLF2 expression. Notably, eRNA_G3 was significantly upregulated in Ph-like ALL cases carrying the GATA3 rs3824662 variant. These findings suggest that both rs3824662 and eRNAs may cooperatively contribute to the regulatory mechanisms governing GATA3 and CRLF2 expression in Ph-like ALL. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10262-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a6bfc4308f5a1eb04611b56714ff4964a37c1d8" target='_blank'>
              Enhancer RNAs contribute to genome reprogramming driven by a GATA3 noncoding variant in leukaemia
              </a>
            </td>
          <td>
            Thayana da Conceição Barbosa, C. P. Poubel, A.L.T. Maciel, C. B. Blunck, Rafael Lima Dos Santos, M. Boroni, M. B. Mansur, M. Emerenciano
          </td>
          <td>2025-08-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The most common cause of death in pediatric cancer patients is a treatment-resistant tumor, compounded by metastatic spread, making surgery, radiation therapy, and chemotherapy unfeasible as curative treatment options. However, the mechanisms behind metastatic spread in pediatric tumors remain largely unexplored. We conducted whole-genome copy number profiling on 171 primary tumor and metastases samples from 17 patients with neuroblastoma, Wilms tumor, or gonadal tumors, and performed targeted deep sequencing on a subset. Phylogenetic reconstruction enabled spatiotemporal tracking of subclones. In total, 11 of 17 patients displayed at least one metastasis arising earlier, defined as occurring before the most recent common ancestor in the primary tumor. In eight patients, metastatic spread was observed several times during tumor evolution, with different subclones from the same primary tumor having metastatic capability, even colonizing the same site. Strikingly, dissemination between metastases (intermetastatic spread) occurred in eight of nine patients with metastases in at least two different sites, indicating that this is a common phenomenon in pediatric malignancy. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e24f7bd63ad2744ced5ab2ba8c285ffb7719db78" target='_blank'>
              Tracing metastatic spread in pediatric solid tumors using copy number and targeted deep sequencing.
              </a>
            </td>
          <td>
            Natalie Andersson, Michele Ferro, Caroline Jansson, Subhayan Chattopadhyay, Jenny Karlsson, David Gisselsson
          </td>
          <td>2025-09-23</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Hepatitis B virus (HBV) integration is a key driver of hepatocellular carcinoma (HCC) occurrence and progression; however, its oncogenic mechanisms remain incompletely understood because of limitations in detection methods and sample availability. In this study, we employed Oxford Nanopore Technologies (ONT) whole-genome sequencing and full-length transcriptome sequencing to characterize HBV integration events at the genomic and transcriptomic levels, along with their regulatory effects on structural variations (SVs) and gene expression. Functional validation was performed using dual-luciferase assays and cell-based experiments. Our findings revealed that integrated HBV sequences form long concatemers, mediating inter- and intrachromosomal recombination in the human genome. Notably, integrated HBV enhancer I (HBV-Enh I) was detected in 6 of 7 tumor tissues and was associated with aberrant gene expression. HBV integration induced oncogenic SVs, such as focal MYC amplification and NAV2 deletion, and directly modulated gene expression. Additionally, ectopic overexpression of MYOCD, driven by HBV-Enh I integration, promoted HCC cell migration and invasion. In summary, HBV integration acts as a major driver of large-scale genomic SVs and transcriptomic dysregulation, through either direct alterations in genome dosage or cis-regulatory mechanisms. HBV-Enh I is frequently integrated in HCC and might play a pivotal role in abnormal gene expression, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/428a53d8eb2aca52cd5c27a1ec4e813d0e350e58" target='_blank'>
              Long-read sequencing reveals HBV integration patterns and oncogenic impact on early-onset hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yao Wang, Dong Yu, Yue Mei, Zhida Fu, Jian Lin, Di Wu, Yuan Yang, Hongli Yan
          </td>
          <td>2025-09-16</td>
          <td>Genome research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In cellular genomes, CCCTC-binding factor (CTCF) insulators impact transcription over small distances in a one-dimensional manner and over much longer distances in a three-dimensional manner by maintaining chromatin loops. We have previously shown that the latent HSV-1 genome contains CTCF insulators that function to regulate lytic transcription of adjacent genes in a one-dimensional manner. Here, we test the hypothesis that HSV-1 CTCF insulators nucleate chromatin loops to regulate the expression of distance-separated gene regions through three-dimensional organization of viral genomes. We used 4C-seq methods to identify multiple long-range cis interactions in HSV-1 genomes that generate viral chromatin domains, including those nucleated by the viral CTCF insulator CTRL2. Deletion of the CTRL2 insulator disrupted these viral chromatin domains. Loop-nucleating interactions were quantitated with a novel approach (UMI-4C-seq) that utilizes unique molecular identifiers to label and count chromatin interactions associated with specific viewpoint primers. Cis-interaction peaks across four different viewpoints were quantified. Viral genomes lacking CTRL2 displayed more cis-interaction peaks and wider ranges of interaction lengths compared to wt virus, suggesting altered chromatin organization. Furthermore, differential looping analysis showed that viral genomes lacking CTRL2 displayed a more transcriptionally permissive chromatin environment. Thus, the CTRL2 insulator functions as a critical regulator of long-range chromatin interactions, and its deletion reshapes the viral chromatin landscape, leading to a more accessible and dynamic regulatory environment that may influence HSV-1 transcriptional programs and latency-associated chromatin states.IMPORTANCEHSV-1 is a significant lifelong human pathogen that infects 70% of adults worldwide. The latent HSV-1 genome is chromatinized and maintained in distinct chromatin structures that silence the virus, while reactivation is facilitated by transient reversal of host factors that maintain those chromatin domains. Understanding how this happens is critical for the development of novel therapeutics. It is becoming clear that CTCF insulators play a key role in the reversal that leads to reactivation. CTCF insulators are essential regulators of chromatin structure and gene expression in mammalian cells and play vital regulatory roles in transcriptional control of DNA viruses by organizing chromatin architecture during both latent and lytic stages of virus lifecycles. Here, we present the first report that latent HSV-1 genomes are organized into 3D structures to support latency yet allow the viral genome to reactivate, opening the door for future therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95a535267a81e37204f0c265ca04fb430e6f4497" target='_blank'>
              Multiple long-range cis interactions generate CTCF insulator-dependent viral chromatin domains in quiescent HSV-1 genomes.
              </a>
            </td>
          <td>
            Alyssa Richman, Sophie Kogut, Terri G. Edwards, Joseph R. Boyd, Princess D. Rodriguez, Michael Mariani, Mason A. Shipley, Kayley A Manuel, Ziyun A Ye, David C. Bloom, Seth Frietze, Donna M Neumann
          </td>
          <td>2025-08-28</td>
          <td>mBio</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Transcription factor E3 (TFE3) oncofusions are frequently detected in the Microphthalmia transcription factor (MiT) family translocation renal cell carcinoma (tRCC), a rare pediatric renal cancer with limited treatment options. The mechanisms by which TFE3 oncofusions promote tRCC malignancy remain inadequately defined. Here, we demonstrate that the RNA-binding capability conferred by TFE3 fusion partners drives the formation of TFE3 condensates. This further enables TFE3 oncofusions to co-condensate with RNA polymerase II (RNAPII) and other RNA-binding proteins, such as paraspeckle component 1 (PSPC1), ultimately driving the formation of transcriptional hubs to promote pro-oncogenic transcription. Dissolution of oncofusion condensates through nanobody-based chemogenetic manipulation effectively curtails tRCC cell growth both in vitro and in vivo, suggesting the therapeutic potential for targeting oncofusion condensation in tRCC. Collectively, our study establishes the causal role of RNA and RNA-binding proteins in facilitating oncofusion condensation to promote renal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79cdc6e21b779dea89a691271b52beca3b734a9" target='_blank'>
              RNA-mediated condensation of TFE3 oncofusions facilitates transcriptional hub formation to promote translocation renal cell carcinoma
              </a>
            </td>
          <td>
            Lei Guo, Rongjie Zhao, Yi-Tsang Lee, Junhua Huang, James Wengler, Logan Rivera, Tingting Hong, Tianlu Wang, Kunjal Rathod, Ashley Suris, Yitian Wu, Xiaoli Cai, Rui Wang, Yubin Zhou, Yun Huang
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumor metastasis is a major contributor for mortality in patients with pancreatic ductal adenocarcinoma (PDAC). While circular RNAs (circRNAs) have emerged as pivotal regulators of tumor progression and metastasis, their functional roles in PDAC remain poorly understood. Through comprehensive circRNA profiling of 10 pairs of PDAC tumors and adjacent normal tissues, we identified circPRKD3 (hsa_circ_0000992) as being substantially up-regulated in malignant specimens. Functional characterization demonstrated that circPRKD3 overexpression potently enhanced cellular migration, invasion, and metastatic capacity in vitro and in vivo, whereas its knockdown produced opposite phenotypic effects. Mechanistic investigations revealed that circPRKD3 directly interacted with the oncogenic RNA-binding protein Musashi-2, protecting it from β-TRCP-mediated ubiquitination and subsequent proteasomal degradation. Clinical correlation analysis revealed a close association between elevated circPRKD3 expression and shorter survival of PDAC patients. Notably, we validated the translational potential of circPRKD3 as a liquid biopsy marker, showing that serum detection, when combined with conventional biomarkers (CA19-9, CEA, and CA125), dramatically improved diagnostic performance. These findings not only delineate a novel circRNA-mediated regulatory axis in PDAC metastasis but also identify circPRKD3 as a promising diagnostic and prognostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1441280c9bbaab596bfcee43907f7ebe16003d78" target='_blank'>
              A Circular RNA Promotes Tumor Metastasis through Stabilizing MSI2 Protein in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Zhang Li, Lanyang Gao, Yang Yang, Yue Ming, Wenrong Liu, Tingting Zhang, Zixia Ye, Fuyan Xu, Juan He, Jiao Li, Jiawei Guo, Xiaojuan Yang, Qing Zhu, Yong Peng
          </td>
          <td>2025-09-10</td>
          <td>Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Simple Summary BRG1 protein expression or its normal functions are lost in many cancers, including ~10% of lung cancer cases, due to mutation. However, how these losses contribute to the development of lung cancer is not clearly known. This study used human lung epithelial cell-derived normal, pre-cancer, and cancer cell lines to investigate the initiation and progression of lung cancer. Our results show that BRG1 loss induces epigenetic (non-genetic) and expression changes in thousands of genes genome-wide that lead to the early transformation of normal cells into pre-cancer cells. In established cancer cells, BRG1 loss and its downstream epigenetic and gene expression changes make the cancer cells become sensitive to epigenetic drugs. Taken together, these results indicate that BRG1 loss contributes to early lung cancer development and could be exploited to improve the treatment of this deadly disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23222db0933f5999fd32623b0319bdf7442135af" target='_blank'>
              BRG1 Loss Is Frequent in Lung Cancer and Transforms Lung Epithelial Cells via Transcriptional and Epigenetic Reprograming
              </a>
            </td>
          <td>
            M. Tessema, Christin M. Yingling, Loryn M. Phillips, K. Do, M. Picchi, R. Willink, Steven A. Belinsky
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA), shed by malignant tumor cells into extracellular fluid, provides valuable epigenetic information indicative of cancer status. Nipple aspirate fluid (NAF), a noninvasive liquid biopsy from at-risk women, contains nucleic acid and protein biomarkers from adjacent cancer cells, showing promise for breast cancer (BrC) detection. However, despite its potential, the application of cfDNA in NAF for BrC screening is still underexplored. Here, we report a proof-of-concept study for using cfDNA bisulfite sequencing (cfBS) to assess tumor DNA methylation signatures from NAF samples. For four healthy individuals and three BrC patients, cfBS achieved greater than 20× sequencing depth with an average coverage of 26.5× on the genome. A total of 7471 differentially methylated regions were identified, with significant hypermethylation in BrC samples compared to healthy controls. Gene set enrichment analysis indicated that the differentially methylated genes (DMGs) were significantly associated with epithelial–mesenchymal transition (EMT). By developing a novel EMT scoring metric, we found that BrC samples had more of a mesenchymal phenotype than samples from healthy individuals. CDH1, WNT2, and TRIM29 were hypermethylated near the promoter region, while COL5A2 was hypermethylated in the coding region. The DNA methylation and EMT changes were validated through The Cancer Genome Atlas Breast Invasive Carcinoma study, which confirmed that DMGs were associated with gene expression change and that our methylation-based EMT score reliably distinguished tumors from healthy controls. Our findings support the utilization of the NAF cfDNA cfBS methylation profile for noninvasive BrC screening and pave the way for enhanced early detection of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/910913725dce178fd3c9aae5a2e95c8cd7107e54" target='_blank'>
              Cell-Free DNA Bisulfite Sequencing Reveals Epithelial–Mesenchymal Transition Signatures for Breast Cancer
              </a>
            </td>
          <td>
            M. S. Jeon, Zehuan Ding, Casey Pei, Jing Li, Linglin Xie, Edward Sauter, Ke Kurt Zhang
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a4fa771302c68d0f49b58a8ac556af478b3c9db" target='_blank'>
              DUX4-rearranged B-ALL: deciphering a biological and clinical conundrum.
              </a>
            </td>
          <td>
            Jack Bakewell, Anthony V Moorman, S. Ryan
          </td>
          <td>2025-09-12</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Our previous work demonstrated that oncogenic KRAS mutations in pancreatic cancer induce aberrant transcription-replication conflicts (TRCs). TRCs arise when DNA replication forks collide with transcription complexes due to loss of spatial and temporal coordination between these processes. TRC resolution mechanisms are and adaptive dependency to KRAS-mediated oncogenic stress. Here we identify a key pathway that regulates TRCs in PDAC and performs mechanistic studies towards clinical translation.



 TRC factors were identified through an siRNA screen against 200 DNA repair genes in Panc1 cells using a high-throughput proximity ligation assay (HiPLA). Knock-out and complementation experiments were performed using CRISPR-Cas9 systems. Cell proliferation was measured using incucyte. Replication stress and DNA damage were measured using multiple orthogonal methods. Protein-protein interactions were assessed by co-immunoprecipitation mass spectrometry and validated by western blot. Chromatin occupancy of key factors was determined using ChIP-seq.



 Through high-throughput proximity ligation assay (HiPLA) screening of 200 known DDR genes, we identified the BER pathway as a key mechanism for TRC resolution. Among these, Neil2 - a bifunctional glycosylase in the base excision repair (BER) pathway - emerged as one of the highest-scoring BER factors; and the only BER glycosylase involved—likely due to its transcription-coupled repair activity. Using CRISPR/Cas9-mediated knockdown, we found that loss of Neil2 led to elevated double-stranded DNA breaks (DSBs) and replication stress. Notably, this significantly impaired cell cycle progression. colony formation, and proliferation in pancreatic cancer cell lines, with minimal effects on normal pancreatic epithelial cells, highlighting Neil2’s selective oncogenic function. To elucidate the underlying mechanisms, we combined proteomic approaches—including co-immunoprecipitation and iPOND-mass spectrometry—with transcriptomic and chromatin profiling with ChIP-seq and RNAseq. Our data revealed that Neil2 plays a critical role in transcription termination, and that its dysregulation promotes RNA Polymerase II readthrough, likely contributing to R-loop accumulation, downstream TRCs, and subsequent genotoxicity. Intriguingly, deletion of Neil2’s DNA binding domain—which abolishes its DNA lesion binding capability—did not compromise its regulation of TRCs or support of cancer cell viability. This suggests that Neil2’s oncogenic role in pancreatic cancer may be DNA-binding independent.



 We uncovered a noncanonical and essential function of Neil2 in TRC regulation and oncogenic maintenance in pancreatic cancer, and we propose Neil2 as a promising therapeutic target for selective intervention in this disease.



 Fan Meng, Mustafa Raoof. Neil2 is a potential theratputic targert in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d45ba09d3e2c9d65e7a73127234b8bec15900c3d" target='_blank'>
              Abstract A041: Neil2 is a potential theratputic targert in pancreatic cancer
              </a>
            </td>
          <td>
            Fan Meng, Mustafa Raoof
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Most of the human genome is transcribed into noncoding RNAs (ncRNAs), which although do not encode proteins, are often involved in key cellular processes. Recent technological advances have accelerated the discovery of ncRNA as well as our understanding of ncRNAs as cancer biomarkers and targets. In this “In the Limelight” special issue of FEBS Open Bio, four review articles by leading experts provide an overview of the role of ncRNAs in cancer, from their association to liver cancer to their role as oncogenes in head and neck squamous cell carcinoma (HNSCC).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39d403967dd7059c9aa239237ba085b5894f7584" target='_blank'>
              Non‐coding RNAs as key player in cancer diagnosis and treatment
              </a>
            </td>
          <td>
            Marcin Majka
          </td>
          <td>2025-09-01</td>
          <td>FEBS Open Bio</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Lysine methyltransferase 5a (KMT5a) plays a key role in the pathogenesis of many human diseases. Here, we review the diverse impacts of KMT5a activity on human cancer development and progression. First, KMT5a is the only Mammalian enzyme that specifically induces monomethylation of histone 4 (H4) on lysine 20, thus regulating chromatin organization and, in turn, the transcription of several oncogenes and tumor suppressor genes. KMT5a, by inducing H4 methylation, also critically establishes the choice between different pathways of DNA double-strand break repair, with important consequences for genomic instability and cancer origin. Finally, KMT5a also methylates lysine residues on nonhistone proteins, and KMT5a-induced methylation of key oncogenic and tumor suppressor proteins, including TP53, strongly affects cancer cell functions. Overall, KMT5a is overexpressed in a high percentage and wide variety of human cancers and has protumorigenic activity, which makes it a target for innovative therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0d2924a6f5a75832faa0ef5f5bf3e3b48c7c2e1" target='_blank'>
              Multiple functions of the lysine methyltransferase KMT5a in cancer: potential targets for innovative therapies
              </a>
            </td>
          <td>
            R. Della Monica, Michela Buonaiuto, Mariella Cuomo, Davide Costabile, Claudio Schonauer, G. Catapano, Lorenzo Chiariotti, Roberta Visconti
          </td>
          <td>2025-09-29</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Enhancer of Zeste Homolog 2 (EZH2) is a key epigenetic regulator known for its role in global gene silencing and is involved in a variety of cellular processes, including cell survival, proliferation, invasion, and self-renewal. As a core component of the Polycomb Repressive Complex 2 (PRC2), EZH2 catalyzes the trimethylation of histone H3 at lysine 27 (H3K27me3), leading to chromatin compaction and transcriptional repression. Dysregulated EZH2 expression is observed in a wide range of solid tumors and hematological malignancies and is frequently associated with increased metastatic potential and poor clinical outcomes. While EZH2 primarily mediates gene silencing through its canonical PRC2-dependent activity, it also exerts oncogenic effects via non-canonical mechanisms. In its non-canonical role, EZH2 acts independently of PRC2, interacting with other signaling molecules as a transcriptional activator or co-activator, thereby promoting the activation of oncogenic pathways. Through both canonical and non-canonical mechanisms, EZH2 significantly contributes to tumor initiation and its subsequent progression. Given its critical role in oncogenesis and cancer progression, EZH2 is under investigation as a potential biomarker for cancer diagnosis and prognosis. This review provides a comprehensive overview of EZH2’s function and oncogenic roles across human cancers. Enhanced insight into EZH2’s complex regulatory network may facilitate the development of more effective strategies to manage EZH2-driven malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac9083fda14d6cccbdbf2f40711aac3833884d4a" target='_blank'>
              EZH2 Dysregulation and Its Oncogenic Role in Human Cancers
              </a>
            </td>
          <td>
            Shiv Verma, Nikita Goyal, Suhani Goyal, Parminder Kaur, Sanjay Gupta
          </td>
          <td>2025-09-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Recent researches have revealed the potential utility of extrachromosomal circular DNAs (eccDNAs) as biomarkers in various diseases. However, the association between plasma eccDNAs and myocardial infarction (MI) remains unclear. In this study, we extracted plasma eccDNA from blood samples of individuals with acute MI and conducted Circle-Seq. We identified an MI-related eccDNA (MIRECD) with high expression levels in the plasma of patients with MI. Sanger sequencing validated its loop construction and sequence. Mechanistically, MIRECD could aggravate necroptosis via regulating the expression of mixed-lineage-kinase-domain-like pseudokinase (MLKL). Kaplan–Meier analysis demonstrated that the incidences of major adverse cardiac events (MACEs) and cardiovascular mortality were higher in individuals with elevated MIRECD levels. Univariate and multivariate Cox regression analyses indicated that MIRECD independently predicted the occurrence of MACEs in patients with MI. The addition of MIRECD enhanced the discrimination and reclassification ability compared with conventional risk factors. In conclusion, our study identified a novel eccDNA, MIRECD, which might regulate myocardial necroptosis through MLKL. MIRECD has the potential to serve as a reliable indicator for predicting the prognosis and stratifying the risk of MI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a8ed2aed986868beea935ead77f232c45fe1fb" target='_blank'>
              Extrachromosomal Circular DNA MIRECD Enhances Necroptosis and Predicts Prognosis of Myocardial Infarction
              </a>
            </td>
          <td>
            Yiheng Zhao, Yujia Zhou, Shuchen Zhang, Boyang Xiang, Xiang Zhou
          </td>
          <td>2025-08-08</td>
          <td>Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="INTRODUCTION
Long intergenic non-coding RNA 01123 (LINC01123) is a lncRNA located on the human chromosome 2q13. It is upregulated in various cancers and has been identified as an oncogene. Its expression is associated with the risk and poor prognosis of multiple cancers.


METHODS
A systematic literature search was conducted in PubMed, Web of Science, and Google Scholar databases using "LINC01123" as the search term. The retrieved studies were reviewed to analyze the expression patterns, oncogenic mechanisms, and clinical significance of LINC01123 in cancers.


RESULTS
LINC01123 is activated by transcription factors such as c-Myc, ZEB1, and FOXC1. It promotes cancer progression, metastasis, and drug resistance by acting as a "molecular sponge" for miRNAs, activating signaling pathways, or interacting with proteins. Its upregulation correlates with adverse clinicopathological features and poor prognosis in multiple cancers.


DISCUSSION
The findings suggest that LINC01123 plays a multifaceted role in cancer biology. Its ability to regulate gene expression through various mechanisms highlights its potential as both a prognostic biomarker and a therapeutic target. However, further research is needed to elucidate its mechanisms fully and to explore its clinical applications across different cancer types.


CONCLUSION
LINC01123 has potential as a novel prognostic biomarker and therapeutic target for cancer. Further research is needed to elucidate its mechanisms and clinical applications fully.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f2bf850ec743e66fdfe68d29b02bfd81015a455" target='_blank'>
              The Emerging Landscape of LINC01123 in Cancer: Roles, Mechanisms, and Clinical Significance.
              </a>
            </td>
          <td>
            Jingjie Yang, Fan Peng, Haodong He, Haoran Liu, Kexing Liu, Zhouya Xu, Yaqi Hu, Guihua Liao, Yan Cai, Chengfu Yuan
          </td>
          <td>2025-09-11</td>
          <td>Current drug targets</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Retroelements are repetitive sequences that comprise 42% of the human genome and are strictly regulated through various epigenetic mechanisms. Examining retroelement expression on a locus-specific level in relation to cancer can uncover distinct disease signatures. Results Using over 5000 RNA-sequencing samples, we assessed retroelement transcription across 23 tissue systems, 159 cell types, 1019 cancer cell lines, and cells isolated from various stages of embryogenesis using the specialized software tool, Telescope. In healthy individuals, 11,388 retroelements were found to be actively transcribed and dynamically regulated in a tissue- and cell type-dependent manner. Using the adult human body as a reference, we observed that 94% of cancer cell lines displayed elevated transcription of at least one cancer-specific retroelement, providing a three-fold larger reservoir of cancer biomarkers (1182) than our comparable analysis of annotated protein-coding genes (338). The precise retroelements that were transcribed following tumorigenesis were influenced by the originating location, with cancers of the blood, lungs, and soft tissue displaying the highest origin-specific activation. Moreover, nearly half of the cancer-specific retroelement loci, mostly from the HERV-H family, were found to be expressed during early embryonic development. Conclusions Our data demonstrate that elevated transcription of certain tissue-specific and embryonic retroelements can be considered as a hallmark of tumorigenesis. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00376-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e92e95f88b68527f2a00c19cdab4d4246b04c" target='_blank'>
              Strict retroelement regulation is frequently lost following cancer transformation and generates a promising reservoir of cancer biomarkers
              </a>
            </td>
          <td>
            Eric Russ, Sergey Iordanskiy
          </td>
          <td>2025-10-03</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction Skeletal muscle stem cells (MuSCs) have strong regenerative abilities, but as we age, their ability to regenerate decreases, leading to a decline in muscle function. Although the methylation reprogramming of super-enhancers (SEs) plays a pivotal role in regulating gene expression associated with the aging process, our understanding of the molecular diversity of stem cells during aging remains limited. This study aimed to identify the methylation profile of SEs in MuSCs and explore potential therapeutic molecular targets associated with aging. Methods The ROSE software was employed to identify super enhancers from the ChIP-seq data obtained from the ENCODE database. Additionally, the ALLCools and Methylpy packages were applied to analyze the methylation profile of SEs and to identify differentially methylated regions (DMRs) between aged and control samples using single-cell bisulfite sequencing (scBS-seq) data from the Gene Expression Omnibus (GEO) database. Overlap analysis was used to assess the regions of SEs and DMRs. The target genes and motifs were analyzed using KEGG, GO, and HOMER to identify key biological pathways and functions, followed by validation through snATAC-seq and immunofluorescence techniques. Results In conclusion, we conducted a multi-omics and cross-species analysis of MuSCs, creating a detailed methylation profile of SEs during aging. We identified key motifs and genes affected by SE methylation reprogramming, revealing important molecular pathways involved in aging. Notably, further analysis of the key gene PLXND1 revealed a decreasing expression trend in aged MuSCs, which appears to be linked to the hypermethylation of SE Rank 869. This epigenetic alteration is likely to contribute to the dysregulation of the SEMA3 signaling pathway, with profound implications for muscle regeneration in MuSCs during aging. Conclusion These findings suggest that epigenetic alterations in the methylation reprogramming of SEs are closely linked to the disruption of transcriptional networks during MuSCs aging. Moreover, our results offer valuable insights into the mechanisms driving SE methylation reprogramming, shedding light on how these epigenetic changes contribute to the molecular processes underlying aging. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00619-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/417be044ed4d0dde0738375aa74ebba806518514" target='_blank'>
              Multi-omic integration of single-cell data uncovers methylation profiles of super-enhancers in skeletal muscle stem cells
              </a>
            </td>
          <td>
            Anyu Zeng, Hailong Liu, Shuling He, Xuming Luo, Zhiqi Zhang, Ming Fu, Baoxi Yu
          </td>
          <td>2025-08-11</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="DNA methylation inhibitors are widely used in treating myeloid malignancies, yet their precise effects on chromatin organization and nuclear architecture remain incompletely understood. Here, the integrated molecular, cellular, and biophysical approaches to investigate the impact of azacitidine (AZA) and decitabine (DEC) on chromatin structure and nuclear mechanics in AML-007 leukemia cells are presented. Confocal microscopy revealed drug-induced alterations in nuclear morphology and actin cytoskeleton organization, with DEC inducing significant nuclear enlargement and disorganization at lower concentrations (1.0 µM) compared to AZA (5.0 µM). Chromatin condensation assays demonstrated that DEC increased chromatin accessibility in a concentration-dependent manner, while AZA produced subtler effects. Optical tweezers measurements showed both agents reduced nuclear stiffness, with DEC exerting a greater impact. Spectroscopic analysis confirmed differential drug incorporation into DNA, with higher methylation loss and structural changes observed following DEC treatment. Refractive index mapping revealed chromatin decompaction, aligning with increased accessibility and nuclear softening. These findings demonstrate that DNA hypomethylating agents exert distinct, concentration-dependent effects on nuclear organization and chromatin structure, which can be quantified through molecular and biophysical readouts. This study underscores the value of integrative methods for revealing epigenetic drug effects on chromatin architecture in leukemia cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4906405a8ab8442becabd2d89c99471c5e4596eb" target='_blank'>
              Effects of DNA Methylation Inhibitors on Molecular and Structural Changes in Chromatin Organization in Leukemia Cells.
              </a>
            </td>
          <td>
            Przemysław Sareło, A. Kaczorowska, Marlena Gąsior-Głogowska, Kinga Godkowicz, Weronika Lamperska, Sławomir Drobczyński, Ewa Zioło, Anna A Licznerska, Tomasz Wróbel, Halina Podbielska, Wojciech Kałas, M. Kopaczyńska
          </td>
          <td>2025-09-29</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The strength of the repressive histone H3K27me3 signal varies across silencers. Focusing on regions with unusually strong signals—super-silencers—we show that B-cell super-silencers are initially linked to gene upregulation in development, with target genes highly expressed in stem cells. About 13% of B-cell super-silencers convert to super-enhancers in B-cell lymphoma; 22% of these recur in over half of patients. Genes like BCL6 and BACH2 tied to these conversions are downregulated faster by JQ1, a super-enhancer-disrupting anti-cancer agent. Super-silencers are enriched for B-cell cancer-associated variants—both somatic and germline—and translocation breakpoints, exceeding levels in other regulatory elements like CTCF binding sites. Over 80% of B-cell lymphoma t(3;14)(q27;q32) translocations fuse BCL6 super-silencers with enhancer-rich regions. Super-silencer repression depends on CpG content: CpG-rich elements block promoter–enhancer contacts; CpG-poor — inhibit looping. These findings highlight super-silencers’ key role in B-cell regulation and suggest their alteration may be a primary factor of B-cell carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cab550e747142b856a2c8f404a6e1c22dc14158c" target='_blank'>
              Super-silencers are crucial for development and carcinogenesis in B cells
              </a>
            </td>
          <td>
            Di Huang, H. Petrykowska, Dhaneshwar Kumar, L. Kardava, S. Moir, B. Afzali, L. Elnitski, Ivan Ovcharenko
          </td>
          <td>2025-09-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Abstract DNA-damaging agents (DDAs) have long been used in cancer therapy. However, the mechanisms by which DDAs induce cell death are not fully understood, and drug resistance remains a major clinical challenge. Schlafen 11 (SLFN11) was identified as the gene most strongly correlated with sensitivity to DDAs based on mRNA expression levels. SLFN11 sensitizes cancer cells to DDAs by cleaving and decreasing tRNALeu(TAA) levels. Elucidating the detailed mechanism by which SLFN11 induces cell death is expected to provide insights into overcoming drug resistance. Here, we show that, upon administration of DDAs, SLFN11 cleaves tRNALeu(TAA), triggering ER stress and protein aggregate formation, leading to cell death regulated by inositol-requiring enzyme 1 alpha (IRE1α). These responses were significantly alleviated by SLFN11-knockout or transfection of tRNALeu(TAA). Proteomic analysis suggests tRNALeu(TAA) influences proteins essential for maintaining proteostasis, especially those involved in ubiquitin-dependent proteolysis. Additionally, we identified the cleavage sites of tRNALeu(TAA) generated by SLFN11 in cells and revealed that tRNA fragments contribute to ER stress and cell death. These findings suggest that SLFN11 plays a crucial role in proteostasis by regulating tRNAs and thus determines cell fate under DDA treatment. Consequently, targeting SLFN11-mediated tRNA regulation could offer a novel approach to improve cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04bd7e8511d20689e4f33be87517895928d1815" target='_blank'>
              SLFN11-mediated tRNA regulation induces cell death by disrupting proteostasis in response to DNA-damaging agents
              </a>
            </td>
          <td>
            Yuki Iimori, Teppei Morita, Takeshi Masuda, Shojiro Kitajima, N. Kono, Shun Kageyama, Josephine Galipon, A. Sasaki, Akio Kanai
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Background Variations in DNA methylation within the DNA damage response (DDR) mechanism could have significant implications for glioma prognosis and immune responses. This study aimed to explore the global DNA methylation landscape of DDR genes in gliomas and identify key epigenetically regulated genes influencing glioma biology and immunity. Methods This study incorporated a range of public and local glioma datasets. Multiple clinical, bioinformatic, and in vitro experimental analyses were conducted to explore clinical and biological aspects. Results Global DNA methylation variations in DDR genes correlated with distinct glioma prognoses, with five CpGs identified as potent predictors. Hierarchical clustering and a risk-score model based on these CpGs unveiled immune-related prognostic subgroups in glioblastomas (GBMs) and lower-grade gliomas (LGGs). NSUN5, epigenetically regulated by one of these CpGs, highlighted the biological significance of the DDR CpG panel. In vitro, NSUN5 displayed tumor-suppressor-like activities in GBM cells, but clinically, it was an unfavorable prognostic marker. Depletion of NSUN5 shifted cytosolic DNA sensing from a STING-dependent (cGAS-STING) pathway to a STING-independent (DNA-PK-HSPA8) pathway, leading to a delayed but more robust type I interferon (IFN) response in GBM cells and enhancing microglial M1 polarization and chemotaxis. This may partially account for the functional discrepancy of NSUN5 observed between experimental and clinical contexts. Conclusion This study highlights the complex interplay between DNA methylation, the DDR mechanism, cytosolic DNA sensing, and glioma immunity. These findings may inspire novel strategies for DNA sensing-based immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06360-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc54757ac8e99a0979bc781abac88c0c6433dc38" target='_blank'>
              DNA methylation variations of DNA damage response in glioblastoma: NSUN5 modulates tumor-intrinsic cytosolic DNA-sensing and microglial behavior
              </a>
            </td>
          <td>
            An-An Yin, Yan-Qiu Yao, Yi-Feng Liu, Yu-Sha Ji, Yalong He, Tian-Chi Ma, Wen-Heng Guo, A. Etcheverry, M. Aubry, Jean Mosser, Wei Lin, Yuan-Ming Wu, Kun Chen
          </td>
          <td>2025-08-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Replication stress (RS) poses a threat to genome stability and drives genomic rearrangements. The homologous recombination (HR) pathway repairs stalled replication forks (RFs) and prevents such instability. Through an E3 ubiquitin ligase screen aimed at identifying regulators of RAD51, we identified macrophage erythroblast attacher (MAEA), a core component of C-terminal to Lish (CTLH) E3 ubiquitin ligase complex, as a regulator of the HR pathway. Loss of MAEA impairs RAD51 recruitment at stalled RFs, leading to increased sensitivity to RS-inducing agents and excessive degradation of nascent DNA strands. Mechanistically, MAEA associates with and mediates the ubiquitylation of Ku80, enabling its removal from RF ends and facilitating the loading of RAD51. Notably, MAEA deficiency is associated with a developmental disorder involving microcephaly, craniofacial abnormalities, ocular defects, and heart malformations. Functional assays show that disease-linked MAEA variants (R34C, E349G, Y394D, and M396R) are defective in RS response. These findings establish MAEA as an essential factor in RF protection and genome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c499707b53080fc3870cbe7d66d1176114a45109" target='_blank'>
              Pathogenic variants in MAEA disrupt DNA replication fork stability and are associated with developmental abnormalities in humans
              </a>
            </td>
          <td>
            Elham Zeinali, Fatemeh Mashayekhi, Rabih Abou Farraj, Talah Hasanni, Marie-christine Caron, Yan Coulombe, A. Fitieh, J. N. M. Glover, Jean-Yves Masson, Ismail Hassan Ismail
          </td>
          <td>2025-08-29</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4c68bd6ad67db140a21e19e70ce02281cc63746" target='_blank'>
              Transcriptional Rewiring of BET Inhibitor Treated Ewing Sarcoma Cells Augments their Dependency on Focal Adhesion Kinase.
              </a>
            </td>
          <td>
            Shireen S Ganapathi, Emma D. Wrenn, Nicolas M Garcia, Neerja Katiyar, Stephanie I. Walter, Aya Miyaki, Jonah R. Valenti, Yuqi Kang, Marina Chan, Taranjit S. Gujral, Elizabeth R. Lawlor
          </td>
          <td>2025-09-06</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Epigenetic regulation, encompassing DNA methylation, histone modifications, and non-coding RNA activities, is a crucial mechanism through which gene expression is modified without corresponding changes in genomic DNA sequences. Dysregulation of these mechanisms can lead to histone and DNA modifications that either suppress or enhance the expression of disease progression-related genes. Among these regulatory processes, histone modifications are particularly significant, as they contribute to genomic stability, DNA repair, and chromatin dynamics, all of which influence breast cancer initiation and progression. This review offers a detailed analysis of the current state of research centered on epigenetic regulatory factors, with a particular focus on the role that histone modifications play in the treatment of breast cancer. It also examines the interplay between epigenetic modifications and the effectiveness of radiotherapy and chemotherapy when treating breast cancer. Lastly, this article explores the potential of epigenetic regulatory factors as viable targets for the future design of new anticancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/379da6773863c74043489d2ccffa9dd0b5e23c2c" target='_blank'>
              Epigenetic regulation espeically histone modifications in breast cancer: A viable and emerging targeted therapeutic strategy
              </a>
            </td>
          <td>
            Yibing Zhou, Haotian Liu, Weimin Hong, Haotian Su, Yuxiao Mu, Yijie Cheng, Chaoshen Wu, Xuli Meng, Da Qian
          </td>
          <td>2025-09-22</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3d9cc4974c393dd5c8e640f80106dc4e3c5a03a" target='_blank'>
              Targeting the 3D genome by anthracyclines for chemotherapeutic effects.
              </a>
            </td>
          <td>
            Minkang Tan, Shengnan Sun, Yuchen Liu, Andrea A. Perreault, D. Phanstiel, Liping Dou, Baoxu Pang
          </td>
          <td>2025-10-07</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Humans are continually exposed to a diverse array of environmental chemicals that can damage DNA and compromise genomic integrity. Among these genotoxic agents, acetaldehyde, acrolein, aristolochic acids, and vinyl chloride are particularly concerning due to their widespread presence in industrial emissions, dietary sources, and lifestyle-related exposures such as smoking and alcohol consumption. These compounds can induce structurally distinct forms of DNA damage including bulky DNA adducts, interstrand crosslinks, and other replication-blocking lesions. While canonical DNA repair pathways serve as the primary defense against such DNA damage, some lesions persist, challenging the capacity of DNA repair systems. If not efficiently repaired, DNA lesions may disrupt replication and transcription. In many cases, translesion synthesis polymerases are recruited to bypass unrepaired lesions, introducing mutations that contribute to agent-specific mutational signatures found in cancer genomes. This review systematically examines how each of these four exogenous chemicals induces DNA damage, the DNA repair pathways responsible for removing their lesions, and the role of translesion synthesis in shaping their mutational signatures. We also highlight how three-dimensional genome organization regulates lesion susceptibility and repair, contributing to variability of mutational landscapes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c10fba52c520395f5d989cae353584818e523a" target='_blank'>
              Mechanisms of DNA repair and mutagenesis induced by acetaldehyde, acrolein, aristolochic acids, and vinyl chloride.
              </a>
            </td>
          <td>
            Kyle Jones, Emmaline Dye, Nyla Gilkes, Dian Xia, Shan Jiang, Wentao Li
          </td>
          <td>2025-08-16</td>
          <td>Journal of environmental science and health. Part C, Toxicology and carcinogenesis</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDA) is a highly aggressive disease with a 5-year survival rate of 12%. Patients are typically diagnosed at the metastatic stage due to lack of symptoms and current treatment options do not effectively control the disease. There is an urgent need to develop sensitive strategies to detect PDA at an early stage before it spreads. Characterizing circulating tumor DNA (ctDNA) in plasma is an effective approach to detect and monitor cancer. Typically, cancer genomes contain thousands of mutations, but targeted sequencing approaches only focus on pre-determined driver genes, which limits their performance in detecting ctDNA. Whole-genome sequencing (WGS) however, tracks all the tumor mutations simultaneously, and has a higher sensitivity than targeted approaches, especially in low tumor burden samples. In this study, we aim to use plasma WGS to build a liquid biopsy platform for PDA detection and characterization. To date, we have established the largest plasma WGS (20-60x) PDAC cohort which consists of over 250 baseline and longitudinal plasma samples from 205 PDA patients, with an additional 45 plasma samples from age-matched healthy individuals. Paired tumor tissue WGS and RNAseq are also available. We developed and validated a bioinformatics pipeline to detect cancer mutations in plasma and estimate plasma tumor fraction (TFx) for tumor burden. In the PDA cohort, plasma TFx increased with clinical stages (medians of 0.5% in resectable, 0.8% in locally advanced and 5% in metastatic), as did the ctDNA detection rate which was measured by whether the plasma TFx was significantly higher than controls (87%, 91% and 95%). Plasma TFx positively correlated with liver metastasis size (R=0.68, P<0.001), but not with the primary pancreatic tumor size. These results suggest that in PDA, metastases are the major site to shed ctDNA, but not primary tumors. We also found that liver metastasis size had a stronger correlation with plasma TFx than with serum CA19-9 (R=0.32, P=0.002). Baseline plasma TFx was associated with shorter overall survival (OS) in both resectable (P=0.013) and metastatic (P=0.022) cases. By analyzing the paired tumor WGS and RNAseq, we found that higher plasma TFx was associated with higher cell cycle, lower immunity and polyploid genomes in the tumors. No difference was found in plasma TFx between basal-like and classical PDA subtypes. In summary, the current study provides insights into the interplay between ctDNA dynamics, tumor biology and clinical features in PDA. Alongside with these results, we will provide a freely accessible computational platform for ctDNA genomic profiling, PDA early detection and progression monitoring using plasma.



 Yuanchang Fang, Michelle Chan-Seng-Yue, Karen Ng, Amy Zhang, Tuan Hoang, Gun Ho Jang, Sabiq Chaudhary, Eugenia Flores-Figueroa, Daniela Bevacqua, Stephanie Ramotar, Ayelet Borgida, Shawn Hutchinson, Anna Dodd, Julie Wilson, Barbara Grünwald, Robert Grant, Erica Tsang, George Zogopoulos, Jennifer Knox, Masoom Haider, Steven Gallinger, Faiyaz Notta. Plasma whole-genome sequencing to monitor pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B072.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41904b2fb7e8d9c5cced0dd565b2ced62da7f071" target='_blank'>
              Abstract B072: Plasma whole-genome sequencing to monitor pancreatic cancer
              </a>
            </td>
          <td>
            Yuanchang Fang, M. Chan-Seng-Yue, Karen Ng, Amy Zhang, Tuan Hoang, G. Jang, Sabiq Chaudhary, Eugenia Flores-Figueroa, Daniela Bevacqua, S. Ramotar, A. Borgida, S. Hutchinson, A. Dodd, Julie M. Wilson, Barbara Grünwald, Robert C. Grant, Erica S. Tsang, G. Zogopoulos, Jennifer J Knox, Masoom A. Haider, Stephen Gallinger, F. Notta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Copy number control of DNA and centrosomes is essential for accurate genetic inheritance. DNA replication and centrosome duplication have been recognized as parallel key events for cell division. Here, we discover that the DNA replication machinery directly regulates the licensing and execution processes of centrosome duplication to prevent centrosome amplification. We find that the microcephaly protein DONSON couples DNA replication initiation with Cdc6 translocation to centrosomes. The Cdc6 signal prevents the precocious occurrence of centriole disengagement, the licensing step for centrosome duplication. During DNA replication, DONSON inhibits replisome disassembly by interacting with the CMG helicase, maintaining the intrinsic S/G2 checkpoint signal that blocks centriole-to-centrosome conversion, the execution step for centrosome duplication. Disruption of these dual signals causes precocious centrosome duplication and chromosome mis-segregation, observed in DONSON patient cells. Our results reveal that the DNA replication machinery not only duplicates genetic material but also controls the system for its accurate segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895f8bba3e9e4612c1b72098d2d19a10b12b8583" target='_blank'>
              The DNA replication machinery transmits dual signals to prevent unscheduled licensing and execution of centrosome duplication
              </a>
            </td>
          <td>
            Kyohei Matsuhashi, Kei K Ito, Kaho Nagai, Akira Sanada, Koki Watanabe, Kasuga Takumi, Atsushi Toyoda, Masamitsu Fukuyama, Shohei Yamamoto, Takumi Chinen, Grant S Stewart, Shoji Hata, Daiju Kitagawa
          </td>
          <td>2025-09-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Diverse fusions of EML4 and ALK are oncogenic drivers in lung adenocarcinomas. EML4-ALK variants have distinct breakpoints within EML4, but their functional differences remain poorly understood. Here, we use somatic genome editing to generate autochthonous mouse models of EML4-ALK-driven lung tumors and show that V3 is more oncogenic than V1. By employing multiplexed genome editing and quantifying the effects of 29 putative tumor suppressor genes on V1- and V3-driven lung cancer growth, we show that many tumor suppressor genes have variant-specific effects on tumorigenesis. Pharmacogenomic analyses further suggest that tumor genotype can influence therapeutic responses. Analysis of human EML4-ALK-positive lung cancers also identified variant-specific differences in their genomic landscapes. These findings suggest that EML4-ALK variants behave more like distinct oncogenes rather than a uniform entity and highlight the dramatic impact of oncogenic fusion partner proteins and coincident tumor suppressor gene alterations on the biology of oncogenic fusion-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c8a78e3c9e4fd68a0441eaf2f2edc5f59a5afae" target='_blank'>
              EML4-ALK variant-specific genetic interactions shape lung tumorigenesis.
              </a>
            </td>
          <td>
            Alberto Diaz-Jimenez, Emily G. Shuldiner, Kálmán Somogyi, Karen Y. Shih, Óscar González-Velasco, Mulham Najajreh, Stewart Kim, Filiz Akkas, Christopher W Murray, Laura Andrejka, Min K. Tsai, B. Brors, Ilse Hofmann, S. Sivakumar, S. Sisoudiya, Ethan S Sokol, Hongchen Cai, D. Petrov, M. Winslow, R. Sotillo
          </td>
          <td>2025-09-23</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="SUMMARY DNA replication is molecularly asymmetric, due to distinct mechanisms for lagging and leading strand DNA synthesis. Whether chromatin assembly on newly replicated strands is also asymmetric remains unknown, as visualizing this short-lived state in cells is impossible. To circumvent this limitation, we combine in vitro reconstitution of the Saccharomyces cerevisiae DNA and chromatin replication machineries with replication-aware single-molecule chromatin footprinting, to study how chromatin is re-assembled on replicated DNA. Leveraging the non-destructive, single-molecule, and strand-specific nature of our data, we discover an intrinsic asymmetry in nucleosome positioning patterns and organization between lagging- and leading-strand chromatin created by the yeast replisome. This asymmetry is only partially restored upon addition of chromatin assembly factors involved in de novo histone deposition and the ATP-dependent chromatin remodeler Isw1a, implying that other regulatory factors must resolve this asymmetry in cells. In sum, our data reveal the complexity of chromatin re-establishment following DNA replication, and suggest an asymmetric chromatin assembly intermediate on each daughter chromatid. These pathways have implications for essential chromatin-templated processes such as DNA repair, transcription, and gene silencing at replication forks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a40a3157eae67e28210affbf8a47af9f74eab0a3" target='_blank'>
              The eukaryotic replisome intrinsically generates asymmetric daughter chromatin fibers
              </a>
            </td>
          <td>
            Bruna V. Eckhardt, Hannah J Richter, Inge Rondeel, Keerthi Renduchintala, Francesca Mattiroli, Vijay Ramani
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a59a21b3122614696fcbf37a9973f0709eb05a0" target='_blank'>
              Unravelling the genetics and epigenetics of the ageing tumour microenvironment in cancer.
              </a>
            </td>
          <td>
            H. Easwaran, A. Weeraratna
          </td>
          <td>2025-09-08</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) systematically orchestrate multiple biological processes via gene regulation in various organisms. Conventionally, miRNAs bind to untranslated regions of mRNAs to modulate gene silencing in the cytoplasm. Recent studies have revealed that a subset of miRNAs located in the nucleus can efficiently activate gene transcription by targeting enhancers, named as Nuclear Activating miRNAs (NamiRNAs). The fundamental principle of this regulatory pattern is that NamiRNAs as "wedges" open double strands of genomic DNA and provide the prerequisite for RNA polymerase II-mediated transcription. Specifically, NamiRNAs bind to their target enhancers and form the hybrids of miRNAs and single-stranded DNA, which are recognized by Argonaute 2 (AGO2) to maintain the single-stranded state. Meanwhile, AGO2 protects these miRNA-DNA hybrids from RNase H-mediated degradation to maintain their stability. Additionally, NamiRNAs induce H3K27ac enrichment at target enhancers and put them into an active state conducive to gene activation. Notably, similar to NamiRNAs, Human Identical Sequences as exogenous small RNAs as critical pathogenic factors for RNA viruses, facilitate infectious disease progression. Collectively, this review systematically elucidates the theory of NamiRNA-Enhancer-mediated Gene Activation in combination with corresponding evidence, summarizes the functions and challenges of NamiRNAs, and discusses their potential in fundamental research and clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff4fdc35f7062b8b76f4cff58088fb91115a9279" target='_blank'>
              NamiRNA-Enhancer Network: The Changing Era Challenge and Chance of miRNA.
              </a>
            </td>
          <td>
            Lu Chen, Ying Liang, Hongyan Wei, Mengxing Liu, Jin Wu, Wei Li, Wenxuan Li, Yuxiao Jin, Yinshan Li, Wei Zhao, Min Xiao, Kaicheng Zhou, Shuai Yang, Wenqiang Yu
          </td>
          <td>2025-08-26</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0643ed748b04336ec828931abc9d5fbd2911609" target='_blank'>
              [Progress of scRNA-seq technology in nasopharyngeal carcinoma research].
              </a>
            </td>
          <td>
            Bin Zheng, Guanqiao Jin
          </td>
          <td>2025-09-01</td>
          <td>Lin chuang er bi yan hou tou jing wai ke za zhi = Journal of clinical otorhinolaryngology head and neck surgery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA topoisomerase IIα (TOP2A) plays a key role in resolving DNA topological stress during replication, transcription, and chromosome segregation. Yatein, a biogenic precursor of deoxypodophyllotoxin possesses antitumor effects. Herein, we explored its antitumor effect in breast cancer. Yatein significantly inhibited the proliferation of breast cancer cells by MTT, colony formation, EdU assay, and cell cycle analysis. Moreover, yatein suppressed the stemness of cancer cells, causing inhibition of migration and epithelial-mesenchymal transition. Mechanism study revealed that yatein inhibited TOP2A activity and stabilized TOP2A-DNA cleavage complexes, caused DNA damage and apoptosis, suggesting its potential in the treatment of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2d219f76fc877a179547f374e85d00ebe2c5225" target='_blank'>
              Yatein-induced DNA damage of breast cancer cells by interfering with DNA topoisomerase IIα.
              </a>
            </td>
          <td>
            Yu-Yao Liu, Ya Chen, Qi-Qi Wang, Xi Zhang, Xiao-Xia Ma, Xing-De Wu, Jia Su
          </td>
          <td>2025-08-25</td>
          <td>Journal of Asian natural products research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Post-transcriptional RNA modifications, such as N6-methyladenosine (m6A) methylation and adenosine to inosine (A-to-I) editing, are critical regulators of hematopoietic stem cell (HSC) self-renewal and differentiation, yet their precise contributions to malignant transformation are not fully elucidated. In this study, we uncovered the epitranscriptomic landscape caused by knockdown of genes from the methyltransferase (METTL)-family in hematopoietic stem and progenitor cells (HSPCs). We identified both converging and distinct roles of METTL3 and METTL14, known members of the m6A writer complex, as well as orphan gene METTL13. Notably, METTL13 was uniquely upregulated by adenosine deaminase acting on RNA 1 (ADAR1) overexpression, while other METTL genes were downregulated. Knockdown of METTL13 altered the expression of multiple genes involved in oncogenic development in HSPCs. Furthermore, METTL13 was associated with a high-risk profile in pediatric T-cell acute lymphoblastic leukemia (T-ALL), and functional studies confirmed that METTL13 is required for T-ALL cell proliferation and survival both in vitro and in vivo. Collectively, our results indicate a previously unrecognized, oncogenic role for METTL13 in pre-leukemic transformation and T-ALL pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047267bf0f7cb87eac1cee31332201b93b817d49" target='_blank'>
              METTL13 Promotes Pre-Leukemic Transformation and the Development of Pediatric Leukemia
              </a>
            </td>
          <td>
            Sabina Enlund, Chae-eun Lim, Isabella Hoang, Sonali Joshi, Amanda Ramilo Amor, Cecilia Thomsson, Indranil Sinha, Shahrzad Shirazi Fard, Anna Nilsson, Ola Hermanson, Qingfei Jiang, Frida Holm
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="DNA transactions introduce torsional constraints that pose an inherent risk to genome integrity. While topoisomerase 1 (TOP1) activity is essential for DNA supercoil removal, the aberrant stabilization of TOP1:DNA cleavage complexes (TOP1ccs) can result in cytotoxic DNA lesions. What protects genomic hot spots of topological stress from excessive TOP1cc accumulation remains unknown. Here, we identify chromatin context as an essential means to coordinate TOP1cc resolution. Through its ability to bind poly(ADP-ribose) (PAR), the histone variant macroH2A1.1 facilitates TOP1cc repair factor recruitment and lesion turnover, thereby preventing DNA damage in response to transcription-associated topological stress. The alternatively spliced macroH2A1.2 isoform is unable to bind PAR or protect from TOP1ccs. Impaired macroH2A1.1 splicing, a frequent cancer feature, was predictive of increased sensitivity to TOP1 poisons in a pharmaco-genomic screen in breast cancer cells, and macroH2A1.1 inactivation mirrored this effect. We propose macroH2A1 alternative splicing as an epigenetic modulator of TOP1-associated genome maintenance and a potential cancer vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfdec4fbd7a5287d669fa0b66871ebd9fb8a9c98" target='_blank'>
              Epigenetic control of topoisomerase 1 activity presents a cancer vulnerability
              </a>
            </td>
          <td>
            Tae-Hee Lee, Colina X Qiao, Vladislav Kuzin, Yuepeng Shi, Marina Farkas, Zhiyan Zhao, Vijayalalitha Ramanarayanan, Tongyu Wu, Tianyi Guan, Xianzhen Zhou, David Corujo, M. Buschbeck, Laura Baranello, Philipp Oberdoerffer
          </td>
          <td>2025-08-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="


 Fusions between protein-coding genes are common oncogenic drivers across cancers, typically pairing a proto-oncogene with a partner that does not independently drive cancer. In all therapeutically actionable fusions, the proto-oncogene is the drug target; the contributions to oncogenicity of the fusion partner have largely been ignored. KIAA1549::BRAF (K::B) is the most common fusion protein across brain tumors, and particularly in pediatric gliomas. We found that KIAA1549 is necessary for oncogenicity of K::B, in contrast to the notion that the BRAF kinase domain alone is enough for transformation. We therefore performed genome-wide CRISPR/Cas9 anti-transformation screens in isogenic neural stem cell models with K::B overexpression to identify genetic dependencies specific to this oncogenic driver. We found that the BRAF fusion partner KIAA1549 engenders a striking and specific dependency on the protein O-mannosyltransferase complex (POMT1/2). Specifically, we show that genetic silencing or pharmacologic inhibition of the POMT1/2 complex reverses K::B transformation, thereby representing a novel therapeutic target beyond the MAPK pathway. Furthermore, POMT1/2 is required to glycosylate and enable maturation of K::B, reflecting the native biology of KIAA1549 but representing a non-canonical process for BRAF. These findings represent a proof-of-concept for targeting the partners in oncogenic fusions as a potential cancer therapeutic strategy.



 Anna Borgenvik, Sean Misek, Daniel Christen, Gloria Kyrila, Alexander Zhang, Sarah Reel, Michelle Boisvert, Kelly Cai, Kevin Zhou, Elizabeth M Gonzales, Lorena Lazo de la Vega, Amy Goodale, Esteban Miglietta, Jacquelyn Jones, Seth Malinowski, Lobna Elsadek, Merve Ozdemir, Zachary Eisenbies, Joohee Lee, April Apfelbaum, Jenna Robinson, Antonio Maldera, Daniel Bondeson, Jason Kwon, Mounica Vallurupalli, Sangita Pal, Todd Golub, William Hahn, Eric Fischer, Jesse Boehm, Jörn Dengjel, Henrik Clausen, Nada Jabado, Till Milde, Beth A Cimini, Keith Ligon, Kathrine Janeway, Michael Eck, David Root, David Jones, Timothy N Phoenix, Rameen Beroukhim, Hiren Jitendra Joshi, Tilman Brummer, Adnan Halim, Pratiti Bandopadhayay. O-mannosylation represents a therapeutic opportunity in BRAF fusion protein oncogenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e2365acc2ab5bd8824942a859ad9e18262c35e6" target='_blank'>
              Abstract B019: O-mannosylation represents a therapeutic opportunity in BRAF fusion protein oncogenesis
              </a>
            </td>
          <td>
            Anna Borgenvik, Sean A. Misek, Daniel Christen, Gloria Kyrila, Alexander Zhang, Sarah Reel, Michelle Boisvert, Kelly Y Cai, Kevin N Zhou, Elizabeth M Gonzales, Lorena Lazo de la Vega, A. Goodale, Esteban Miglietta, Jacquelyn Jones, Seth Malinowski, Lobna Elsadek, Merve Ozdemir, Zachary Eisenbies, Joohee Lee, April A Apfelbaum, Jenna Robinson, Antonio Maldera, Daniel Bondeson, Jason Kwon, Mounica Vallurupalli, Sangita Pal, Todd Golub, William Hahn, Eric S Fischer, Jesse Boehm, J. Dengjel, Henrik Clausen, Nada Jabado, T. Milde, Beth Cimini, Keith L. Ligon, Kathrine Janeway, Michael J Eck, David E. Root, David Jones, T. Phoenix, R. Beroukhim, H. Joshi, Tilman Brummer, Adnan Halim, P. Bandopadhayay
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>150</td>
        </tr>

        <tr id="In a recent study published in Science , Tiebang Kang and his colleagues 1 identify ASB7 as a key negative regulator of heterochromatin maintenance through targeted degradation of the H3K9me3 methyltransferase SUV39H1. The work uncovers a cell cycle-regulated chromatin-associated ubiquitin ligase circuit that maintains epigenetic homeostasis and reveals a novel vulnerability in cancer that may be exploited therapeutically. H3K9me3 is a de ﬁ ning feature of constitutive heterochromatin, vital for transcriptional repression and genome stability. This repressive histone mark is propagated through a self-reinforcing loop, wherein HP1 binds pre-existing H3K9me3 and recruits SUV39H1 to methylate adjacent nucleosomes. While this mechanism ensures robust inheritance of silencing, unrestrained propagation could lead to widespread gene silencing and excessive chromatin compaction. How this feedback loop is constrained to maintain H3K9me3 homeostasis has remained poorly understood. To uncover regulators that limit H3K9me3 spread, authors conducted a genome-wide CRISPR-Cas9 screen using H3K9me3 immunostaining in asynchronous and S-phase – arrested cells. Among known heterochromatin regulators such as SUV39H1/2, SETDB1, and EHMT1/2, the screen identi ﬁ ed ASB7, a substrate adaptor of the CUL5 E3 ligase complex, as a top hit. ASB7 knockout led to genome-wide H3K9me3 accumulation across both repetitive and non-repetitive regions, con ﬁ rmed by CUT&RUN and immuno ﬂ uorescence. Conversely, ASB7 overexpression suppressed H3K9me3, supporting its role as a global repressor">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f123d6e18344d7fb0613cfecce34c6744d13f4e" target='_blank'>
              Fencing the genome: ubiquitin signaling restricts heterochromatin spread
              </a>
            </td>
          <td>
            Collin Bakker, Joanna Paulson, Nitika Taneja
          </td>
          <td>2025-08-21</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is the age-related expansion of mutated hematopoietic stem cells without other hematologic abnormalities. In patients with solid tumors, CH is associated with higher mortality and may evolve to therapy-related myeloid neoplasms; however, the mechanisms by which cancer treatments promote CH dynamics remain largely unknown. Here, we analyzed 392 serial samples from a prospective cohort of breast cancer patients and showed that cytotoxic treatments (chemotherapy {+/-} radiation) led to strong therapeutic bottlenecks, resulting in significant reductions in hematopoietic allelic populations and differential clonal selection. CH clones that were positively selected and expanded through dose-dependent therapeutic bottlenecks frequently harbored mutations in TP53, PPM1D, SRCAP, DNMT3A, and YLPM1. Patients with positively selected CH during treatment had the shortest progression-free and overall survival compared to patients with unchanging or negatively selected CH across all therapies. These findings, corroborated in independent breast cancer and pan-cancer cohorts, provide strong evidence for clinical relevance of changes in CH during cancer treatment which may help identify patients at high risk for inferior outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6a553a5184b2aa0287d71bedac04551bf6a5da6" target='_blank'>
              Clonal Hematopoiesis Dynamics and Evolutionary Fitness During Cancer Treatment Impact Clinical Outcomes
              </a>
            </td>
          <td>
            Mona Arabzadeh, Yi-Han Tang, Christelle M. Colin-Leitzinger, S. Marzban, Daniel Walgenbach, Stefania Morganti, Vaidhyanathan Mahaganapathy, Erika Harper, Mingxiang Teng, Jacob Kresovich, Iman Washington, H. Parsons, Judy E. Garber, J. West, Shridar Ganesan, Hossein Khiabanian, Nancy Gillis
          </td>
          <td>2025-09-02</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Double Homeobox 4 (DUX4) is a potent transcription factor encoded by a retrogene mapped in D4Z4 repeated elements on chromosome 4q35. DUX4 has emerged as pivotal in the pathomechanisms of facioscapulohumeral muscular dystrophy (FSHD), a relatively common hereditary muscle wasting condition, although classified as a rare disease. DUX4 contributes to zygote genome activation before its expression is repressed in most somatic tissues through epigenetic mechanisms, including DNA methylation and chromatin modifications. In FSHD, inappropriate activation of DUX4 expression is driven by a complex interplay of genomic and epigenetic alterations. The ectopic presence of DUX4 in skeletal muscle cells activates genes, viral elements and pathways that are typical of very early embryonic development, disturbing cell function and ultimately contributing to muscle weakness and wasting. This review first traces the history of DUX4, from the FSHD genetic linkage studies in the early 1990s, through to identification and characterization of the DUX4 gene in 1999. We then discuss the seminal studies that showed how and why DUX4 is expressed in FSHD and the effects of this ectopic expression in muscle, notably cellular toxicity. Other pathological roles of DUX4, such as participation in cancer and viral infection, are also highlighted. Maintenance of DUX4 in the genome was explained by discovery of the function of DUX4 in zygotic genome activation to institute the totipotent cells of the embryo. Thus, we encompass the gradual transition of DUX4 over the past 25 years from being considered a pseudogene in “junk DNA” to becoming central to understanding the molecular pathogenesis of FSHD and the primary focus for FSHD therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad639b9256d2a222879537cb6ffce2084c7d6dd4" target='_blank'>
              DUX4 at 25: how it emerged from “junk DNA” to become the cause of facioscapulohumeral muscular dystrophy
              </a>
            </td>
          <td>
            Alexandra Belayew, Alberto L. Rosa, Peter S. Zammit
          </td>
          <td>2025-08-25</td>
          <td>Skeletal Muscle</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Transposable elements (TEs), comprising over one-third of the human genome, play a crucial role in its evolution, serving as a significant source of regulatory sequences. Under normal circumstances, their activity is tightly controlled by DNA methylation mechanisms; however, the effectiveness of this suppression varies substantially across tissues. The placenta, characterized by global hypomethylation, represents a unique environment where retroviruses and retrotransposons, typically silenced in somatic cells, gain the opportunity for activation. This distinct epigenetic landscape of the placenta allows transposons to participate in the regulation of genomic activity, influencing processes ranging from early embryogenesis to postnatal development. DNA hypomethylation in the placenta not only promotes TE mobilization, but also opens the possibility of using their components as independent genes and regulatory elements – promoters, enhancers, and other functional modules. These elements are involved in key aspects of placental development, including syncytiotrophoblast formation, extravillous trophoblast invasion, spiral artery remodeling, and endometrial decidualization. Importantly, TEs can serve as sources of alternative promoters for neighboring genes, and ancient mammalian transposons contain multiple transcription factor binding sites, enabling coordinated regulation of genes sharing a common function. Despite the growing interest in the role of transposable elements in placental development and function, many questions remain unanswered. In particular, the mechanisms of non-long terminal repeat (non-LTR) retrotransposon function during pregnancy remain poorly understood. A deep understanding of these processes is necessary to elucidate regulatory disorders in the placenta associated with major obstetric syndromes. This review examines the contribution of transposable elements to the functioning of the human genome, particularly their impact on gene expression, in the context of pregnancy and placental development">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe4a3e7ac959ca8083b328f8c6eeb28a76f946c" target='_blank'>
              Transposable elements as key regulators of placental development
              </a>
            </td>
          <td>
            M. A. Zhilkina, E. Tolmacheva, S. Vasilyev
          </td>
          <td>2025-09-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d20ba148e7406fb91b956b71267ccb358bcdb11" target='_blank'>
              MNase stratification reveals heterogeneous 5hmC in naive B cells.
              </a>
            </td>
          <td>
            Michael Hsu, Heng-Yi Chen, Fang-Yun Lay, Qin Ma, Ben Delatte, C. J. Lio
          </td>
          <td>2025-08-28</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Genes encoding the RNA-binding proteins FUS, EWSR1, and TAF15 (FET proteins) are involved in chromosomal translocations in rare sarcomas. The encoded fusion oncoproteins typically include a DNA-binding domain and function as oncogenic transcription factors. FET-rearranged sarcomas are often aggressive malignancies affecting patients of all ages. Oncogenic fusion transcription factors are challenging to target directly, and new therapies are needed. Treatment with the small molecule SP-2509 results in reversal of the transcriptional activity of the FET fusion that causes Ewing sarcoma, EWSR1::FLI1. A similar molecule, seclidemstat (SP-2577), is currently in clinical trials for FET-rearranged sarcomas (NCT03600649), but its pharmacologic activity in FET fusion–positive sarcomas has not been demonstrated. We found potent seclidemstat cytotoxicity against both FET-rearranged and other fusion-positive sarcoma cell lines in vitro, including Ewing sarcoma, desmoplastic small round cell tumor, clear cell sarcoma, myxoid liposarcoma, and fusion-positive rhabdomyosarcoma. We also define transcriptomic effects of seclidemstat with bulk RNA sequencing. Seclidemstat recapitulated much of SP-2509 transcriptional activity in Ewing sarcoma. Widespread transcriptional changes were seen in all tested cell lines after seclidemstat treatment, regardless of the fusion protein expressed. This included reversal of FET fusion transcriptional signatures for EWSR1::WT1, EWSR1::ATF1, and EWSR1::ERG. Though novel inhibitors are unlikely to display single-agent efficacy in the clinic, these data suggest that seclidemstat remains a promising new treatment strategy for patients with FET-rearranged and other fusion-positive sarcomas. Significance: In this study, we show seclidemstat has in vitro activity in multiple rare and aggressive sarcomas caused by FET fusion proteins. With 13 RNA sequencing experiments, including multiple FET-rearranged sarcoma cell lines, this dataset is a rich resource for those studying FET-rearranged sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fb78384576a9a5b2487874950835b687b754a22" target='_blank'>
              Seclidemstat (SP-2577) Induces Transcriptomic Reprogramming and Cytotoxicity in Multiple Fusion–Positive Sarcomas
              </a>
            </td>
          <td>
            G. Rask, C. Taslim, A. Bayanjargal, Rachel D. Dreher, Matthew V. Cannon, Julia Selich-Anderson, Jesse C. Crow, Aundrietta Duncan, Emily R. Theisen
          </td>
          <td>2025-08-25</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f43327ba689a502fd2aba63a16648713112594" target='_blank'>
              Epigenomic Heterogeneity of Non-Functional Pancreatic Neuroendocrine Tumors Uncovered by Single nucleus and Spatial ATAC Profiling.
              </a>
            </td>
          <td>
            Dejiang Wang, Xiangjun Di, Fu Gao, Gaocai Li, Liyun Lin, Shujie He, Di Zhang, Junyi Yu Jin, Yuanxin Liang, Michael Cecchini, Jill Lacy, John Kunstman, Pamela Kunz, Jing Du, Yang Liu
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA-binding molecules regulate gene expression, replication, repair, and transcription, making their research crucial to molecular biology. This review investigated their architectures, processes, and impacts on cancer research and therapy to determine their biological and therapeutic potential. A thorough examination of published data on DNA-binding agents, including intercalators, groove binders, and metal complexes, focused on their chemical properties, biological activity, and therapeutic significance. Doxorubicin intercalated between base pairs to inhibit replication and transcription, whereas cisplatin produced covalent cross-links with guanine bases to cause tumor cell death. Multiple derivatives of metal complexes reduced tumor development by over 70% in leukemia models as DNA probes and therapeutics. Also reviewed, Dps proteins in *Escherichia coli* showed that their non-specific DNA binding offered up to 65% oxidative stress resistance compared to control cultures, validating DNA-protein protection as a survival strategy. In Phase II studies, amsacrine caused substantial remission in acute leukemia patients, whereas doxorubicin was more effective across many cancer types but had greater cardiotoxicity concerns. Selectivity, toxicity, and resistance limit DNA-targeting medicines, although they are successful. Chemical modifications like hydrophobic tailoring and sequence-specific binding have enhanced binding affinity and therapeutic index, yet only 10% of candidate compounds get clinical approval. Recent studies show that AI-driven design has expedited screening, lowering development costs by 30% and durations by 3–5 years. These data suggest that DNA-binding medicines have great potential in cancer, but safer, more selective, and resistance-free therapy are still needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bfaa0d838d07a9aad5a057a0a95be735af4700a" target='_blank'>
              Chemistry of DNA-binding Molecules
              </a>
            </td>
          <td>
            Evon Akram, Dina A. Najeeb, Asmaa A. Jawad, N. Bedair, A. M. Hussein, Saba R. Jaafar, Ruaa H. Ali, R. F. Shaher, Marwa A. Hussein, Daniah M. Hamid, Reem H. Al-Tabra, Alyaa K. Abood, Salam Mohammed
          </td>
          <td>2025-08-30</td>
          <td>Al-Salam Journal for  Medical Science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Gastric cancer (GC) remains one of the leading causes of cancer-related mortality worldwide due to its aggressive nature and high metastatic potential. Circular RNAs (circRNAs), characterized by covalently closed loop structures and high molecular stability, have emerged as important regulatory molecules in cancer biology. However, the biological functions and molecular mechanisms of most circRNAs in GC remain largely unexplored. Methods We conducted circRNA microarray profiling on three pairs of primary GC tissues and matched adjacent normal tissues to identify differentially expressed circRNAs. The expression of circ-ANKRD36C was validated by quantitative reverse transcription polymerase chain reaction (qRT-PCR) in additional clinical specimens. Functional assays were performed to assess the effects of circ-ANKRD36C knockdown on GC cell proliferation, migration, and invasion. Dual-luciferase reporter assays provided evidence of direct interaction between circ-ANKRD36C and miR-15a-5p. Clinicopathological correlations and survival outcomes were analyzed in 84 paired GC and normal tissue samples. Results Circ-ANKRD36C was significantly upregulated in GC tissues. Knockdown of circ-ANKRD36C suppressed GC cell proliferation, migration, and invasion in vitro. Dual-luciferase assays supported its direct interaction with miR-15a-5p, potentially influencing oncogenic targets such as BMI1 and CCND1. Clinically, high circ-ANKRD36C expression was associated with shorter overall survival (OS). Multivariate Cox regression analysis suggested its independent prognostic value (hazard ratio [HR] = 2.41; 95% confidence interval [CI] = 1.22–4.77; P = 0.011). Conclusion Circ-ANKRD36C is upregulated in gastric cancer and may serve as a prognostic biomarker. Functional evidence suggests it could act as a candidate regulator, but its mechanistic role requires further validation. Supplementary Information The online version contains supplementary material available at 10.1186/s12876-025-04287-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aec96a53ffb97eb8a42c6e93ec17c0d2fe31bed9" target='_blank'>
              Circular RNA profiling and functional analysis implicate circ-ANKRD36C as a potential prognostic biomarker in gastric cancer
              </a>
            </td>
          <td>
            Rongpeng Chen, Tianxing Ji, Yuanqing Chu, Feng Xiao, Zhicheng Feng, Guoqiang Wang
          </td>
          <td>2025-09-29</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer progression is often accompanied by epigenetic silencing of tumor-suppressor microRNAs such asmiR-200c, a key regulator of epithelial-to-mesenchymal transition (EMT) and metastasis. Given the reversible nature of DNA methylation, we employed a CRISPR/dCas9-TET1 system to target the miR-200c promoter and restore its expression in MCF-7 and MDA-MB-231 breast cancer cell lines. Two gRNAs were designed to flank CpG-rich regions of the miR-200c promoter, and their individual or combined delivery enabled site-specific demethylation. Co-transfection with both gRNAs resulted in a synergistic increase in miR-200c expression, likely due to expanded coverage of dCas9-TET1 recruitment. This upregulation led to the downregulation of key EMT-related transcription factors ZEB1, ZEB2, and the oncogene KRAS, as well as increased E-cadherin expression in MDA-MB-231 cells. However, E-cadherin changes in MCF-7 cells were minimal, highlighting the complex and context-dependent nature of epigenetic regulation. Functional assays further confirmed the anti-tumorigenic effects of miR-200c restoration, with reduced cell viability and increased apoptosis, effects more pronounced in MDA-MB-231 cells, which initially exhibited higher miR-200c promoter methylation. Collectively, our findings demonstrate that CRISPR/dCas9-TET1-mediated epigenetic editing effectively reactivates miR-200c, reverses EMT-associated gene expression, and impairs tumor cell aggressiveness, supporting its potential as a targeted therapeutic strategy in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e592b60e9ed7df393f38257baac9b31bbd24eb" target='_blank'>
              CRISPR/dCas9-TET1–mediated epigenetic editing reactivates miR-200c in breast cancer cells
              </a>
            </td>
          <td>
            Mahyar Zahraei, Yasamin Azimi, Morteza Karimipour, Fatemeh Rahimi-Jamnani, Vahideh Valizadeh, Masoumeh Azizi
          </td>
          <td>2025-08-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Human oncogenic viruses contribute significantly to the global health burden and include seven types; Epstein-Barr virus, hepatitis B virus, human T-cell leukemia virus type 1, human papillomavirus, hepatitis C virus, Kaposi's sarcoma-associated herpesvirus, and Merkel cell polyomavirus. While the roles of latent or integrated viral genomes in cancer have been documented, emerging evidence highlights the contribution of defective viruses-those carrying intragenic deletions or loss-of-function mutations-in promoting viral oncogenesis. These altered genomes often lack genes essential for lytic replication or immune recognition, which enhances their persistence and immune evasion. In virus-associated diseases, specific patterns of gene retention and deletion suggest that host-driven selective pressures drive the emergence of these altered genomes. This review examines the generation, prevalence, and functional impact of these viruses, reframing them as active participants in disease development and progression. Recognizing their role offers new insights into viral tumor evolution and creates opportunities for applications in viral diagnostics and targeted intervention strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/389381b25524966b33d29c79924df9e5f4c7676f" target='_blank'>
              Defective But Tumorigenic: The Evolutionary and Functional Roles of Mutated Oncoviruses.
              </a>
            </td>
          <td>
            Yoshitaka Sato, Yusuke Okuno, Takayuki Murata, Hiroshi Kimura
          </td>
          <td>2025-09-26</td>
          <td>FEMS microbiology reviews</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Genomic amplification may result in aberrant gene expression and support development of cancer, including chronic myeloid leukemia (CML). In CML cell line K-562, we recently reported overexpression of TBX1 located at chromosomal position 22q11, focally co-amplified together with BCR, part of the CML hallmark fusion gene BCR::ABL1. Here, we extended that study, by identifying genomically amplified and overexpressed MAPK1/ERK2 at 22q11 together with MCTS1 at Xq22. Using pharmacological inhibitors and siRNA-mediated knockdown assays, our data collectively revealed novel regulatory connections between TBX1, MAPK1 and MCTS1, which may play a role in drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11cfe5a18303d3c15ad4fb64c39efcc6da10d99d" target='_blank'>
              Aberrant expression of MAPK1 and MCTS1 in chronic myeloid leukemia (CML)
              </a>
            </td>
          <td>
            Leo Kortendick, C. Meyer, S. Nagel
          </td>
          <td>2025-09-18</td>
          <td>microPublication Biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 The androgen receptor (AR) plays a critical role in the development and progression of prostate cancer by regulating key cellular processes such as cell proliferation and apoptosis. Although traditional AR-targeted therapies have shown initial success, acquired resistance remains a significant clinical challenge, often driven by AR alterations and somatic gene mutations associated with homologous recombination deficiency (HRD). Approximately 20% of advanced prostate cancer cases exhibit HRD, resulting in substantial genomic instability and complicating treatment. Fortunately, Food and Drug Administration–approved poly(ADP-ribose) polymerase inhibitors, including olaparib and rucaparib, exploit synthetic lethality to target prostate cancer with HRD, and additional drugs targeting DNA damage response (DDR) proteins are under development. Emerging evidence suggests that AR activity enhances DDR gene expression, with multiple DDR proteins localized near androgen-regulated regions, highlighting a close interaction between AR and DDR pathways. Consequently, recent preclinical and clinical studies have investigated combining AR-targeted therapies with treatments that induce DNA damage, such as radiation therapy, or inhibit DNA repair mechanisms. This review discusses AR's role in cellular processes, the interplay between AR and DDR, and recent advances in prostate cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4bcf547c4cdb4775dcc9944086243c67a7115b7" target='_blank'>
              The interconnection between AR and DNA damage response pathways in prostate cancer
              </a>
            </td>
          <td>
            Mallory Sands, Sam Adams, Jihaeng Lee, Michael Li, Mae Wang, Thomas Walsh, Leo Leon, Adriana Zablah, Matthew Haerens, Zhichao Liu, Jianneng Li
          </td>
          <td>2025-08-19</td>
          <td>Current Urology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a malignant tumor with high morbidity and mortality rates worldwide and only presents symptoms in later stage; no ideal biomarker is available for the early diagnosis of CRC. Therefore, it is important to explore novel molecules that significantly contribute to CRC progression. The cohort contains different stages of CRC were downloaded and comprehensive bioinformatics analyses were performed by Mfuzz, Protein–Protein Interaction (PPI), MCODE, ESTIMATE, and ssGSEA.The results revealed that Protein Kinase, Membrane Associated Tyrosine/Threonine 1 (PKMYT1) served as a functional hub gene and its high expression might be associated with an immunosuppressive microenvironment, therapeutic sensitivity and tumor progression. PKMYT1-related genes are linked to DNA replication, the cell cycle, and mismatch repair, indicating PKMYT1 functions as an oncogene and potential biomarker in CRC development. Moreover, in vitro experimental investigation was conducted and the data found that CRC tumor tissues and cells have elevated PKMYT1 expression. Knockdown of PKMYT1 by siRNAs significantly impaired the proliferation, cell cycling, migration, and invasion of CRC cells. In summary, this study demonstrated that PKMYT1 may be a promising target for therapeutic intervention and play a significant role in the development of CRC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18430-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c70064227e93e681809cf44b6b66e461f9dc4b0" target='_blank'>
              Exploring PKMYT1 as a potential marker for colorectal cancer progression through bioinformatics analyses and experimental validation
              </a>
            </td>
          <td>
            Yuqiang Zhang, Yingwei Chang, Yaorui Hu, Zhichao Ding, Zhihui Zhang, Yanbei Wei, Ben Liu, Minghao Yang, Wei Chen
          </td>
          <td>2025-09-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Targeting the DNA damage response (DDR) exhibits potent efficacy in inducing immune activation and enhancing patient prognosis. However, the benefits of DDR regulation are not universally observed across all patients, owing to the intricate compensatory mechanisms operative in certain cancers. There still exists a gap in the function of activated DDR protein in esophageal squamous cell carcinoma (ESCC). Here, we demonstrate that increased expression of DDR genes contributes to the progression of esophageal squamous cell carcinoma and suppresses the tumor immune microenvironment. Notably, the abundant presence of the DDR protein KIN in ESCC tissues facilitates efficient DNA damage clearance and promotes escape from apoptosis. Depletion of KIN significantly inhibited proliferation and induced DNA damage accumulation in ESCC cells. Mechanistically, KIN functions to support the recruitment of the R-loop regulator DHX9 to R-loop sites, thereby addressing DNA damage associated R-loops. Intriguingly, the depletion of KIN activates the STING pathway via NFκB signaling, which is induced by the accumulation of R-loops, ultimately initiating an innate immune response. Depletion of KIN improved the immune microenvironment and the effect of immune therapy in mouse model. Collectively, our findings identify KIN as a novel R-loop binding protein that facilitates the recruitment of the R-loop resolution complex and suppresses tumor-intrinsic innate immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e3c03a2576fc82b57363fed2fb18edfc9c71a5" target='_blank'>
              DNA/RNA-binding protein KIN17 supports esophageal cancer progression via resolving noncanonical STING activation induced by R-loop
              </a>
            </td>
          <td>
            Zichao Wei, N. Zhao, Lu Kuang, J. Cong, Sujuan Zheng, Yi Li, Zhihua Liu
          </td>
          <td>2025-08-15</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 The complex heterogeneity of Acute lymphoblastic leukemia (ALL) often predicts poor prognosis, high morbidity, and drug resistance. Yet, the molecular drivers that initiate aggressive ALL have yet to be fully elucidated, especially in the context of the >30 subtypes of human T- and B-ALL. Here, we used a large-scale F0 transgenic screen in zebrafish to identify synergistic combinations of 65 putative oncogenes at inducing both T- and B-cell leukemia. This approach identified a new proto-oncogene SET that collaborates with both notch1a icn and mutationally activated Interleukin 7 Receptor (IL7R) to initiate aggressive leukemia in vivo. SET is a multifunctional protein that can act as a transcriptional regulator, a histone chaperone, and an inhibitor of protein phosphatase 2A (PP2A). The high expression of SET along with mutational activation of NOTCH1 and IL7R was also found in human B- and T-ALL, supporting their oncogenic roles in leukemogenesis. Our discovery that SET is capable of independently synergizing with known drivers to initiate ALL presents a unique opportunity to explore the nature of oncogene driven ALL and define the mechanisms by which it drives transformation. We next used Non-negative Matrix Factorization to unbiasedly identify six gene programs shared across the zebrafish ALL subtypes and assessed if each was predictive of outcome in human disease. One program comprised a MYC-driven transcription signature that unexpectedly predicted good outcomes in human T-ALL patients. Additional conserved programs independently stratified patients into poor outcomes and when combined with the MYC signature profile were able to better discern disease outcomes in patients, including the aggressive ETP-like and TAL1 DP-like T-ALL. Taken together, we have uncovered new roles for SET in driving ALL initiation, uncovered unexpected high MYC expression as predictive of good outcome, and defined a new combination biomarker gene signature that predicts poor overall survival in aggressive subtypes of human T-ALL. Our zebrafish-based screening approach is a powerful tool for comparative genomic studies to identify new oncogenic drivers, vulnerability pathways, and new biomarkers of aggression in ALL.



 James R Allen, Luis Antonio Corchete Sanchez, Mohamed N Bakr, Miriam Fernández-Lajarín, Alexandra Hazelwood, Nathan Ford, Anna M Lucianò, Alexandra Veloso, Alexander D Weissman, Olivia A Strom, Esther Rheinbay, David M Langenau. Zebrafish modeling predicts clinical outcomes in human acute lymphoblastic leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B035.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79aacf42881085b0060797945bc5360d7773e354" target='_blank'>
              Abstract B035: Zebrafish modeling predicts clinical outcomes in human acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            J. R. Allen, L. A. Corchete Sanchez, M. N. Bakr, Miriam Fernández-Lajarín, Alexandra Hazelwood, Nathan Ford, Anna M Lucianò, Alexandra Veloso, Alexander D Weissman, Olivia A Strom, Esther Rheinbay, David M Langenau
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chromosomal instability (CIN) and epigenetic reprogramming are central drivers of breast cancer progression, yet the mechanisms connecting them remain elusive. We uncover a direct role of EZH2 histone methyltransferase in promoting CIN in triple-negative breast cancer (TNBC). Across breast cancers, EZH2 expression correlates with copy number alterations, and its catalytic activity is associated with increased CIN in metastasis-initiating cells. Pharmacological EZH2 inhibition suppressed CIN, revealing an unexpected vulnerability. Integrated chromatin and transcriptome profiling identified Tankyrase (TNKS), a poly(ADP-ribose) polymerase, as a direct transcriptional target of EZH2. Mechanistically, EZH2-mediated TNKS suppression disrupts CPAP (centrosomal P4.1-associated protein), driving centrosome overduplication, multipolar mitosis and exacerbated CIN. In vivo, CIN suppression is a critical mechanism underlying the anti-metastatic effects of EZH2 inhibition. These findings delineate a previously unrecognized epigenetic mechanism governing CIN and establish EZH2 inhibitors as the first therapeutic agents capable of directly suppressing CIN, underscoring the need for trials with metastasis-focused endpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5010199d8cc937113c87be8a59e3c3e7e0efc720" target='_blank'>
              Epigenetic regulation of chromosomal instability by EZH2 methyltransferase.
              </a>
            </td>
          <td>
            Yang Bai, Albert S. Agustinus, Shira Yomtoubian, Cem Meydan, Dylan R McNally, Liron Yoffe, Melissa J Hubisz, Marvel Tranquille, Sneha Pramod, Christy Hong, Magdalena L Plasilova, Aakanksha R Kapoor, Arshdeep Singh, Henry G. Withers, Lukas E. Dow, A. Laughney, B. Bhinder, O. Elemento, Ari M Melnick, S. Bakhoum, Vivek Mittal
          </td>
          <td>2025-10-02</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="


 Epigenetic dysregulation drives many pediatric tumors, including neuroblastoma, an aggressive malignancy that accounts for approximately 15% of childhood cancer deaths. Drugs targeting epigenetic regulators have emerged, with FDA approvals of tazemetostat for patients with metastatic or locally advanced epithelioid sarcoma with incomplete resection and for patients with relapsed or refractory follicular lymphoma with EZH2 mutations. More recently, the menin inhibitor revumenib was FDA approved for children and adults with relapsed/refractory KMT2A-rearranged acute myeloid leukemia. Unfortunately, these drugs are generally not curative, and resistance develops by multiple mechanisms suggesting that combination therapies are needed. Our laboratory recently identified some neuroblastomas, particularly MYCN amplified, that are dependent on the SAGA complex, a transcriptional coactivator complex that regulates gene expression by modifying histones. Knockout of TADA2B, a gene essential to the histone acetyltransferase function of SAGA, or treatment with GSK699, a selective degrader of KAT2A and KAT2B, the canonical catalytic acetyltransferases of SAGA, causes cell cycle arrest and delayed progression in xenograft models of neuroblastoma. In order to nominate combination strategies with SAGA inhibitors, we used orthogonal approaches: DepMap nearest neighbor analysis and a CRISPR sensitizer screen. Utilizing the Broad Institute’s Cancer Dependency Map (DepMap), we identified EZH2, a core member of the PRC2 complex, as a co-dependency with the SAGA complex in neuroblastoma. Furthermore, we conducted a CRISPR/Cas9 screen with a focused sgRNA library (1,350 genes) to identify additional epigenetic sensitizers and drivers of resistance in neuroblastoma cells treated with GSK699. EZH2 knockout scored as sensitizing to GSK699 while knockout of BRD2, a member of the BET family of proteins and an epigenetic reader, scored as rendering resistance to GSK699. The utilization of both in vitro and preclinical studies confirmed that pharmacological inhibition of KAT2A/B using GSK699 and EZH2 using tazemetostat induced a combinatorial inhibitory effect on neuroblastoma tumor proliferation compared to single agent treatment, with marked prolongation of survival in xenograft models with combination therapy. Transcriptional analysis of tumors treated with combination therapy showed increases in epithelial to mesenchymal transition (EMT) programs and myogenesis differentiation pathways, a surprising finding under active investigation. In conclusion, our data demonstrates that inhibition of the SAGA complex in combination with EZH2 inhibition effectively suppresses the proliferation of MYCN-amplified neuroblastoma supporting further evaluation of this combination in models of neuroblastoma, including deeper mechanistic studies.



 Melinda Soeung, Kenneth Ross, Lucy Merickel, Silvi Salhotra, Audrey Taillon, Allen Basanthakumar, Kimberly Stegmaier. Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A043.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5216fb3aee2d0a8a7d9f8e5c04d43544aa31dd4b" target='_blank'>
              Abstract A043: Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma
              </a>
            </td>
          <td>
            Melinda Soeung, Kenneth Ross, L. Merickel, S. Salhotra, A. Taillon, A. Basanthakumar, K. Stegmaier
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c39fcf56ce4b3277a5d1391cbbde0aec7e6c90" target='_blank'>
              Deletion of the Mis12C-Binding Domain of CENP-C Promotes Chromosomal Aneuploidy in Cutaneous Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Megumi Saito, Kazuhiro Okumura, Yurika Tokunaga, Sora Tanaka, Keisuke Otoyama, Yoshinori Hasegawa, Masatoshi Hara, Masakazu Hashimoto, Toshihiko Fujimori, Tatsuo Fukagawa, Yuichi Wakabayashi
          </td>
          <td>2025-10-03</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Chromatin remodeling by the SWI/SNF complex is essential for hematopoietic lineage commitment and differentiation. While core subunits ARID1A and ARID1B are frequently mutated in B cell malignancies, their lineage-specific functions remain unclear. Here, we used CD19-Cre-mediated conditional deletion initiated at the pro B cells stage to investigate the function of Arid1a/1b in vivo in mice. Loss of either gene partially blocked B cell differentiation in the bone marrow, reducing immature/recirculating B cell output, and impaired germinal center formation following antigen challenge. Combined deletion resulted in a more severe phenotype, with marked reduction in peripheral B cells, shortened survival, and development of an aggressive leukemia. Unexpectedly, the malignancy was of myeloid origin and arose from a small subset of CD19-expressing multipotent progenitors (MPPs). Arid1a/Arid1b-deficient MPPs exhibited abnormal expansion, reduced colony formation in methylcellulose, and transcriptional dysregulation of stemness and lineage-priming programs, including diminished CBFA2T3 (ETO2) and Fli1 signatures. In established B cell lymphoma cells in vitro, double ARID1A/ARID1B loss only modestly affected cell growth, whereas loss of ARID1A increased sensitivity to pharmacological EZH2 inhibition. Transcriptomic analysis revealed major alterations in cell adhesion/migration pathways, cytokine-receptor interactions and DNA repair mechanisms. Collectively, these findings reveal stage-specific and compensatory roles for ARID1A and ARID1B in B cell development, uncover a mechanism by which SWI/SNF loss in early progenitors can drive transformation towards myeloid leukemia, and suggests context-dependent therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53d7d9360a90c28b3536e18cfb6b090769877ccb" target='_blank'>
              ARID1A and ARID1B safeguard B cell identity, prevent myeloid transformation and expose therapeutic vulnerabilities in lymphoma
              </a>
            </td>
          <td>
            Phyo Nay Lin, Joohyung Park, Yoon-A Kang, George P. Souroullas
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) remains the most lethal gynecologic malignancy, with a 5-year survival rate below 50%. Despite initial responsiveness to platinum-based chemotherapy and the implementation of targeted therapies such as PARP inhibitors, the majority of patients relapse and develop treatment-resistant disease. Emerging evidence suggests that the spatial organization and dynamic interactions among tumor, immune, and stromal cells within the tumor microenvironment (TME) play a critical role in therapeutic response and disease progression. However, traditional transcriptomic approaches lack the spatial context necessary to fully understand these interactions. The Xenium spatial transcriptomics (ST) platform represents an innovative technology that enables high-resolution, in situ mapping of gene expression within intact tissue architecture. By preserving the spatial relationships between cell types and molecular states, ST provides the opportunity to explore how the TME evolves under therapeutic pressure and contributes to resistance. We performed spatial transcriptomic profiling on FFPE tumor sections from three matched longitudinal pairs of HGSOC samples obtained prior to treatment and upon PARPi progression. Utilizing the Xenium 5K Plex platform from 10X Genomics, we quantified the expression of 5000 genes in over 2.5 million cells. Cell type annotation was performed via reference mapping and marker gene expression. Differential expression and gene ontology enrichment analyses were conducted to identify key transcriptional changes in tumor clusters between paired naïve and progression samples. We furthermore describe spatial cellular neighborhoods and their changes upon development of resistance. Characterization of the tumors reveals great morphological heterogeneity between patients. Comparative analysis revealed distinct shifts in the cancer cell abundance, cancer cell area and spatial arrangement of the TME between naïve and progression samples. Progression samples exhibited increased expression of, for example, hypoxia-related genes, particularly in cancer cells. Spatial mapping identified distinct cellular neighborhoods, including niches with a high density of neutrophil- or macrophage-associated transcripts, which were notably more abundant in progression samples, suggesting a role in immune evasion and tumor progression. This study demonstrates the potential of spatial transcriptomics to elucidate the architectural and transcriptional changes of the ovarian cancer microenvironment at the single-cell level. While based on early findings, our results suggest the relevance of hypoxia, stromal remodeling, and niches with distinct cellular compositions in contributing to disease progression. These data support further investigation into spatially targeted therapeutic strategies and the use of ST as a tool for microenvironment-driven biomarker discovery.



 Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera. Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cf6f443298e3305bcbf89470d2e118a90600ac1" target='_blank'>
              Abstract A003: Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance
              </a>
            </td>
          <td>
            Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Germ cells are the only cells capable of transmitting heritable genetic material to future generations. Epigenetic mechanisms that regulate germ cell formation are essential for optimizing offspring production, which is particularly important in farm animals like chicken. Primordial germ cells (PGCs), the precursors of gametes, could be derived from the pluripotent blastoderm cells (BC) or embryonic stem cell (ESCs) in chicken but the germline induction efficiency remain low and require further improvements. Methods We systematically profiled key histone modifications and chromatin states during the germ/soma specification from chicken pluripotent blastoderm cells to either PGCs or fibroblasts to uncover the chromatin regulators that direct the germline specification. The histone methyltransferase was perturbed during germ cell differentiation to assess the effect of histone modification on germline induction. Results The specific alterations of chromatin states could instruct the expression of germline genes and repress the pluripotency or somatic gene program in distinct cell types. The dynamic chromatin activation at both promoters and enhancers contribute to germline induction from pluripotency. Interestingly, the diminished active histone modification H3K4me3 regulate the transitions of bivalent states into repressive to facilitate the specification of the germ cell lineage. We demonstrated that selectively erase the H3K4me3 modifications could block the expression of BMP signaling antagonists, thereby enhancing the creation of PGC-like cells (PGCLCs) in chicken. Conclusions The comprehensive analysis of gene expression and chromatin regulation patterns during germ/soma segregation reveals that chromatin reprogramming mechanisms play crucial roles in controlling germline specification. This research also provides new epigenetic strategies to enhance the production of germ cells. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s13287-025-04584-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/accc81fe5868770f6c262fa56cdbe0c4579d11f1" target='_blank'>
              Chromatin remodeling and H3K4me3 depletion regulate germline specification from pluripotency
              </a>
            </td>
          <td>
            Sheng Wang, Lu Meng, Xiaochen Huang, Zhelun Peng, Yao Hua, Yinlong Liao, Ruimin Ren, Heng Wang, Guiyu Zhu
          </td>
          <td>2025-08-26</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Somatic cell nuclear transfer (SCNT) holds promise for animal cloning but remains limited by low efficiency and phenotypic abnormalities, often attributed to incomplete nuclear reprogramming. This study presents an integrative genomic and epigenomic analysis of cloned buffaloes and their respective donors using long-read Oxford Nanopore sequencing. Our results showed a high degree of genomic similarity between clones and donors, with most variations located in non-coding regions and structural variants (SV) distributions highly correlated at the chromosomal level. Gene and protein level overlap of SV-affected loci revealed 70.9–73.3% gene-level and 69.7–72.5% protein-level similarity. Despite this genetic similarity, DNA methylation analysis identified differentially methylated regions (DMRs), particularly in intergenic and promoter regions. Clones exhibited slightly lower CpG methylation than the donors. The DMRs in donor vs. clone comparisons indicated higher hypomethylated regions than hypermethylated regions. Functional enrichment of DMR-associated genes highlighted pathways linked to mitochondrial function, oxidative phosphorylation, and reproductive processes. Although clones showed moderate genome-wide methylation correlation with donors, key differences in methylation suggest incomplete epigenetic reprogramming. Despite these epigenetic differences, all clones were phenotypically normal and healthy into adulthood. This study offers the first comprehensive SV and methylome profile of SCNT-derived buffaloes and emphasizes the role of epigenetic mechanisms in clone development and health, providing valuable insights to enhance cloning efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ed0c9146dce33964d38cc27a7e6aa2903443aa1" target='_blank'>
              Unravelling High Nuclear Genomic Similarity and Mitochondria Linked Epigenetic Divergence in SCNT Derived Buffalo Clones via Long-Read Nanopore Genome Sequencing
              </a>
            </td>
          <td>
            Meeti Punetha, Dharmendra Kumar, Satish Kumar, Bhavya Maggo, Priya Dahiya, Pradeep Kumar, Rakesh K. Sharma, Yash Pal, P. S. Yadav
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fd03ff61ce13e9709e7c822f2e992ee5cbe1a4e" target='_blank'>
              Targeting histone H2B acetylated enhanceosomes via p300/CBP degradation in prostate cancer.
              </a>
            </td>
          <td>
            Jie Luo, Zhixiang Chen, Yuanyuan Qiao, J. Tien, E. Young, R. Mannan, Somnath Mahapatra, Rupam Bhattacharyya, Lanbo Xiao, Tongchen He, Sanjana Eyunni, Yuping Zhang, Yang Zheng, Fengyun Su, Xuhong Cao, Rui Wang, Yunhui Cheng, Rithvik Seri, James M. George, Miriam Shahine, Stephanie J Miner, M. Rees, M. Ronan, Jennifer A Roth, U. Vaishampayan, Mi Wang, Shaomeng Wang, A. Parolia, A. Chinnaiyan
          </td>
          <td>2025-10-03</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="In mammals, primordial germ cells (PGCs) undergo global erasure of DNA methylation with delayed demethylation of germline genes and selective retention of DNA methylation at evolutionarily young retrotransposons. However, the molecular mechanisms of persistent DNA methylation in PGCs remain unclear. Here we report that resistance to DNA methylation reprogramming in PGCs requires UHRF2, the paralog of the DNMT1 cofactor UHRF1. PGCs from Uhrf2 knock-out mice show loss of retrotransposon DNA methylation, while DNA methylation is unaffected in somatic cells. This is not associated with changes in the expression of retrotransposons in E13.5 PGCs, indicating that other mechanisms compensate for retrotransposon control at this stage. Furthermore, Uhrf2-deficient PGCs show precocious demethylation of germline genes and overexpress meiotic genes in females. Subsequently, Uhrf2-deficient mice show impaired oocyte development and female-specific reduced fertility, as well as incomplete remethylation of retrotransposons during spermatogenesis. These findings reveal a crucial function for the UHRF1 paralog UHRF2 in controlling DNA methylation in the germline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea0f6cead9b7479f9deb5d526e8e54ad404a663" target='_blank'>
              UHRF2 mediates resistance to DNA methylation reprogramming in primordial germ cells
              </a>
            </td>
          <td>
            Ambre Bender, Marion Morel, Michael Dumas, M. Klopfenstein, Naël Osmani, Maxim V. C. Greenberg, Déborah Bourc'his, Norbert B Ghyselinck, Michael Weber
          </td>
          <td>2025-08-09</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f396f45a05f69851b7610dc8198d4d5a27e35207" target='_blank'>
              Characterization of cis-regulatory elements and functional variants in colorectal cancer using epigenomics and CRISPRi screenings.
              </a>
            </td>
          <td>
            Zequn Lu, Can Chen, Heng Zhang, Bin Li, Yizhuo Liu, Jiayi Guo, Runying Xu, Ke Shi, Qianying Ma, Ming Zhang, Yimin Cai, Jinyu Huang, Hui Geng, Linyun Fan, Caibo Ning, Yanmin Li, Shuo Chen, W. Tian, Kexin Hu, Haijie Li, Xiaojun Yang, Chaoqun Huang, Yongchang Wei, Xu Zhu, Xiangpan Li, Zhen Xiong, M. Cai, Xiaoyang Wang, Shaokai Zhang, Hongda Chen, M. Dai, Kun Chen, Mingjuan Jin, Meng Jin, Ying Zhu, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2025-08-25</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The tumour suppressor protein p53 plays a central role in safeguarding genomic integrity through the regulation of DNA repair, cell cycle arrest, and apoptosis. Mutations in TP53, particularly within its DNA-binding domain, are among the most frequent genetic alterations in human cancers and are strongly associated with chemoresistance and poor prognosis. In this study, all TP53 mutations reported in the COSMIC database were systematically mapped onto all experimentally resolved TP53 three-dimensional structures available in the Protein Data Bank, supplemented with AlphaFold-predicted models to achieve full structural coverage. Mutations were classified according to their structural context—protein core, interface regions, ligand- and zinc-binding sites, and intrinsically disordered regions—and evaluated using complementary sequence- and structure-based predictive tools. The analysis revealed distinct mutational hotspots, differential distribution across structural regions, and context-dependent effects on stability and DNA-binding capacity. Notably, a subset of mutations exhibited consistent predictions of high destabilisation across all structural contexts, underscoring their potential as drivers of functional inactivation. By providing a comprehensive structural map of TP53 alterations, this work offers a valuable resource for understanding mutation-specific mechanisms of p53 dysfunction and for guiding the development of precision therapeutic strategies aimed at restoring its tumour-suppressive functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85f3fc7b1e2b5e98df635979848fcd53d6ac30ab" target='_blank'>
              Mutational Disruption of TP53: A Structural Approach to Understanding Chemoresistance
              </a>
            </td>
          <td>
            Ali F. Alsulami
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Aberrant pre-mRNA splicing is a hallmark of many cancers, yet the underlying genetic mechanisms driving these alterations remain incompletely understood. While splice-altering variants can explain a subset of events, many tumor-specific splicing changes arise in the absence of nearby sequence variants. This study aims to systematically dissect the contribution of proximal splice-altering variants to tumor-specific splicing in pediatric brain tumors and to define the residual cohort likely driven by regulatory mechanisms such as mutations affecting splicing factors or the spliceosome machinery. We hypothesized that some fraction of tumor-specific splice events in pediatric brain tumors are not associated with nearby genetic variants, and that these may highlight broader disruptions to splicing regulation.



 We applied replicate Multivariate Analysis of Transcript Splicing (rMATS-turbo) to stranded total RNA-seq data (N=635) from the Pediatric Brain Tumor Atlas (PBTA). Tumor-specific single exon (SE) splicing events were defined as a change in percent spliced in (| ΔPSI |) > 0.3 between tumors and healthy brain samples sourced from GTEx (<40 years, n=2,642) and pediatric normals (n=7). Tumor-normal matched whole genome sequencing data were used to identify somatic and rare (<0.1% Allele Frequency in gnomad) germline SNVs/Indels proximal (<250nt) to rMATS-defined splice junctions.



 Across the cohort (N=635), we identified 107,827 tumor-specific SE alternative splicing events. Only 44 tumors harbored a variant proximal to the affected splice site (11 germline, 39 somatic), leaving most events without a clear cause. We are now investigating this variant-negative subgroup as a preliminary cohort. Current efforts focus on excluding structural drivers before turning to potential regulatory mechanisms, including the disruption of spliceosome components. Preliminary findings suggest some tumors show altered expression or mutations in spliceosome components, though further validation is needed.



 This dual-ended analysis of tumor-specific splicing events reveals that while a subset can be attributed to local genetic variation, the majority likely arise from broader disruptions in splicing regulation. These findings underscore the importance of integrating RNA- and DNA-based analyses to fully capture the molecular basis of splicing dysregulation in cancer. Moreover, the variant-negative cohort may serve as a rich resource to uncover novel regulatory mechanisms and potential therapeutic targets in splicing-driven pediatric brain tumors.



 Patricia J Sullivan, Ryan J Corbett, Ammar S Naqvi, Alexander Sickler, Rebecca Kaufman, Bo Zhang, Chuwei Zhong, Sharon J Diskin, Jo Lynne Rokita. Uncovering genetic and regulatory drivers of tumor-specific splicing in pediatric brain tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0c6326494c3f8ecc5b90e1fc51e9fc24433d6c" target='_blank'>
              Abstract B031: Uncovering genetic and regulatory drivers of tumor-specific splicing in pediatric brain tumors
              </a>
            </td>
          <td>
            Patricia J Sullivan, Ryan J. Corbett, Ammar S. Naqvi, Alexander Sickler, Rebecca S. Kaufman, Bo Zhang, Chuwei Zhong, S. Diskin, J. L. Rokita
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="INTRODUCTION
The epigenetic silencing of O6-methylguanine DNA methyltransferase (MGMT) is associated with reduced DNA repair capacity, carcinogenesis and increased sensitivity to alkylating chemotherapy. However, the biological role and clinical significance of MGMT overexpression in cancer remains poorly understood.


METHODS
Using multiplexed quantitative immunofluorescence we measured the localized levels of MGMT protein, γH2AX and CD8+ T cells in multiple retrospective colorectal cancer (CRC) cohorts. Genomic and transcriptomic features of selected cases were also studied with whole exome DNA sequencing and genome-wide methylation analysis. MGMT-methylated human CRC cells SW620 were transfected with an MGMT-containing plasmid and co-cultured with allogeneic peripheral blood mononuclear cells.


RESULTS
A subset of CRCs showed MGMT protein upregulation associated with lower γH2AX, reduced CD8+ tumor infiltrating lymphocytes (TILs), mismatch repair proficient (pMMR) status and shorter survival. CD8+ TILs were more distant from MGMT-expressing cells than MGMT-negative cells and the MGMT promoter methylation status did not highly correlate with MGMT protein levels in CRC. In genomic/transcriptomic analysis, high MGMT expression was associated with a lower nonsynonymous somatic mutational burden, higher transition-to-transversion mutation ratio, increased deleterious TP53 variants and distinct transcriptomic profiles. The exogenous expression of MGMT in SW620 CRC cells reduced the number of spontaneous nonsynonymous mutations, reproduced mutational features of MGMT-high CRC and limited the in vitro T-cell-mediated killing of malignant cells induced by proinflammatory cytokines in tumor/immune cell co-cultures.


CONCLUSIONS
MGMT overexpression identifies a previously undescribed subset of CRCs with distinct biological and clinical properties including reduced mutagenesis, adaptive immune evasion, predominantly pMMR phenotype and aggressive clinical course. Direct, quantitative assessment of MGMT protein expression using spatially resolved analysis is more reliable than inference of MGMT expression by promoter methylation status in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43cc0eaa491cd34a4c5b928fccd34ad143431db3" target='_blank'>
              High MGMT expression identifies aggressive colorectal cancer with distinct genomic features and immune evasion properties.
              </a>
            </td>
          <td>
            J. Zhang, B. K. Rajendran, Shruti S Desai, Joanna Gibson, Jassim DiPalermo, Patricia LoRusso, Yong Kong, Hongyu Zhao, Michael Cecchini, K. Schalper
          </td>
          <td>2025-09-01</td>
          <td>Journal for immunotherapy of cancer</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Deoxyribonucleic acid (DNA) serves as the fundamental blueprint for life, governing both the structural development and functional regulation of living organisms. The technology explores the modern understanding of DNA role in heredity and molecular genetics that highlighted key discoveries that have shaped the field. The discovery of intricate molecular mechanisms such as epigenetic modifications, non-coding RNA and the gene environment interactions has evolved beyond Mendelian genetics. The mechanism of genetic inheritance, replication and expression illustrates how traits are passed from one generation to the next generation. This review article highlights how mutation and variations in DNA sequences influence health, evolution and disease susceptibility. A significant focus is placed on revolutionary CRISPR-Cas9 Jain editing technology which has transformed genetic research and holds immense promise for future therapeutic applications. After enabling precise modifications in the genome, CRISPR opens new avenues for the treatment of genetic disorders and advancement in biotechnology. This review emphasizes how the integration of classical molecular genetics with cutting edge gene editing techniques offers a comprehensive understanding of DNA function for addressing agricultural and global health challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd606e61309a0083af92e77dc5cd4dc92727de21" target='_blank'>
              MODERN INSIGHTS OF DNA FUNCTION AND HEREDITY: A REVIEW OF MOLECULAR GENETICS AND CRISPR GENE
              </a>
            </td>
          <td>
            Mariam Fatima, Mariam Fatima¹, Muhammad Haseeb, Anwar Tarar¹, Arooba Maryam², Muhammad Azhar, Ud Din³˒⁴, Ifra Muzaffar¹, H.M. Saif, UR Rahman⁵, Rida Sattar⁶
          </td>
          <td>2025-08-21</td>
          <td>Journal of Medical &amp; Health Sciences Review</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract DNA methylation is altered in all cancers, but the mechanisms responsible for these changes are not well understood. Using data from 100 primary samples, we show that regions with intermediate methylation in normal hematopoietic cells are hotspots for stable, clonal epimutations in acute myeloid leukemia. Analysis of hematopoietic stem cell clones demonstrated that intermediate methylation at epimutation hotspots reflects random allele-specific methylation and expression at the single cell level, representing a previously unrecognized form of somatically acquired imprinting. We identified somatically imprinted regions in other tissues and show they are epimutation hotspots in other cancer types. This demonstrates that random allele-specific methylation is both a general property of normal cells, and a vulnerability that renders them susceptible to cooperating events and clonal selection during cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e7fa3ac12fa0ce8e4468a1db8da8c5facb616d2" target='_blank'>
              Intermediately Methylated Regions in Normal Cells Are Epimutation Hotspots in Cancer
              </a>
            </td>
          <td>
            Mohamed Mahgoub, Haley Abel, Nidhi Davarapalli, Heidi Struthers, E. Kotnik, Brittany Johnson, Christopher Markovic, C. Fronick, Robert Fulton, Michael P. Meers, David H. Spencer
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Tumor heterogeneity, a hallmark of cancer, frequently leads to treatment failure and relapse. However, the intricate communication between various cell types within the tumor microenvironment and their roles in tumor progression in vivo remain poorly understood. Here we establish a novel tumor heterogeneity model in the Drosophila larval eye disc epithelium and dissect the in vivo mechanisms by combining sophisticated genetics with single-cell RNA sequencing. We found that mutation of the tricellular junction protein M6 in cells surrounding RasV12 benign tumors promotes their malignant transformation. Mechanistically, early RasV12//M6−/− tumors secrete Pvf1, which activates the Pvr receptor on hemocytes, facilitating their recruitment to the tumor site. These tumor-associated hemocytes secrete the Spätzle (Spz) ligand to activate the Toll receptor within the RasV12 tumors. This enhanced activation of the Toll pathway synergizes with RasV12 to promote malignant transformation through the JNK-Hippo signaling cascade. In summary, our study elucidates the complex interplay between genetically distinct oncogenic cells and between tumors and hemocytes, highlighting how hemocytes exploit the ancient innate immune system to coordinate tumor heterogeneity and drive tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d320162041d03e1b05ee82602e945a36e317508" target='_blank'>
              Hemocytes facilitate interclonal cooperation-induced tumor malignancy by hijacking the innate immune system in Drosophila
              </a>
            </td>
          <td>
            Sihua Zhao, Yifan Guo, Xiaoyu Kuang, Xiaoqin Li, Chenxi Wu, Peng Lin, Qi Xie, Zongzhao Zhai, D. Kong, Xianjue Ma
          </td>
          <td>2025-08-22</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is an aggressive solid malignancy with poor overall prognosis and limited response to standard treatments. Growing interest in the modulation of DNA repair mechanisms, including the homologous recombination (HR) repair pathway, has opened new avenues for therapeutic development. BARD1 (BRCA1-Associated RING Domain 1) plays a complex role in tumor biology, functioning either as a tumor suppressor or as an oncogenic driver, depending on isoform expression, cellular context, and regulatory environment. In this review, we examine the dual roles of BARD1, focusing on its regulation and paradoxical activities in PDAC. We summarize evidence that BARD1 and BARD1 isoforms differentially affect DNA repair, apoptosis, and drug resistance and evaluate the therapeutic potential of targeting BARD1 and other DNA damage response (DDR) proteins. We also review ongoing clinical trials and investigational agents designed to exploit DDR vulnerabilities in PDAC. Together, these insights highlight BARD1’s context-dependent roles in PDAC and support continued efforts to profile BARD1 isoforms, clarify their functions, and leverage DDR pathways through precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6c070f9d1ec5a160a1dcacf90475e9772e358bc" target='_blank'>
              BARD1: A Friend or Foe in Pancreatic Ductal Adenocarcinoma?
              </a>
            </td>
          <td>
            Lily Zekavat, Aditi Jain
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Human Papillomavirus (HPV), particularly high-risk strains such as HPV16 and HPV18, is a leading cause of cervical cancer and a significant risk factor for several other epithelial malignancies. While the oncogenic mechanisms of viral proteins E6 and E7 are well characterized, the broader effects of HPV infection on host transcriptional regulation remain less clearly defined. This study explores the hypothesis that conserved genomic motifs within the HPV genome may act as molecular decoys, sequestering human transcription factors (TFs) and thereby disrupting normal gene regulation in host cells. Such interactions could contribute to oncogenesis by altering the transcriptional landscape and promoting malignant transformation.We conducted a computational analysis of the genomes of high-risk HPV types using MEME-ChIP for de novo motif discovery, followed by Tomtom for identifying matching human TFs. Protein–protein interactions among the predicted TFs were examined using STRING, and biological pathway enrichment was performed with Enrichr. The analysis identified conserved viral motifs with the potential to interact with host transcription factors (TFs), notably those from the FOX, HOX, and NFAT families, as well as various zinc finger proteins. Among these, SMARCA1, DUX4, and CDX1 were not previously associated with HPV-driven cell transformation. Pathway enrichment analysis revealed involvement in several key biological processes, including modulation of Wnt signaling pathways, transcriptional misregulation associated with cancer, and chromatin remodeling. These findings highlight the multifaceted strategies by which HPV may influence host cellular functions and contribute to pathogenesis. In this context, the study underscores the power of in silico approaches for elucidating viral–host interactions and reveals promising therapeutic targets in computationally predicted regulatory network changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/019a01c1572bf807f787812d1b2ed2b11781bb4f" target='_blank'>
              HPV as a Molecular Hacker: Computational Exploration of HPV-Driven Changes in Host Regulatory Networks
              </a>
            </td>
          <td>
            M. Chetta, Alessandra Rosati, N. Bukvic
          </td>
          <td>2025-08-27</td>
          <td>Viruses</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 DNA damaging agents remain the mainstay for high grade serous ovarian cancer (HGSOC) therapy. While many HGSOCs are sensitized to DNA damaging agents via dysregulated DNA repair, those patients harboring tumors with amplification or overexpression of CCNE1 (encoding cyclin E1) respond poorly in part due to proficiency in the homologous recombination (HR) DNA repair pathway. Here, we put forth a novel paradigm that HR proficiency of CCNE1-driven HGSOCs is promoted by altered cellular metabolism. We found that alpha-ketoglutarate (aKG) is upregulated in cyclin E1-driven HGSOC cells, and suppression of aKG sensitized these cells to DNA damaging agents both in vitro and in vivo, demonstrating that aKG drives chemoresistance in these models. aKG is required for the activity of aKG-dependent dioxygenases (aKGDD). There are >60 known aKGDD, and prior work has almost exclusively focused on aKGDD-mediated histone demethylation. Through a targeted CRISPR knockout library, we discovered that Trimethyllysine Hydroxylase Epsilon (TMLHE), the first and rate-limiting enzyme in de novo carnitine synthesis, is necessary for chemoresistance to DNA damaging agents in cyclin E1-driven cells. Unexpectedly, aKG-mediated TMLHE-dependent carnitine synthesis was required for histone acetylation, while histone methylation was affected but dispensable. These data point to a previously unrecognized role for aKG in acetylation reactions. The increase in histone acetylation via aKG-dependent carnitine synthesis promoted HR-mediated repair through site-specific histone acetylation. Additionally, analysis of HGSOC patient samples demonstrated that TMLHE positively correlates with histone acetylation, and both high tumor TMLHE or high serum acetylcarnitine are associated with worse progression free survival (PFS) in HGSOC patients treated with DNA damaging agents. Pharmacological targeting of carnitine synthesis via mildronate in vivo was sufficient to decrease tumor burden >30% in combination with cisplatin. Since mildronate is a non-toxic and extensively used drug in humans, this combination shows potential applicability of inhibition of this axis in humans. These data demonstrate for the first time that HR-proficiency in cyclin E1-driven HGSOCs is mediated through a metabolic axis that directly influences histone acetylation and highlight the translational potential of targeting this axis to induce HR deficiency and sensitize these tumors to DNA damaging agents.



 Apoorva Uboveja, Nathaniel W. Snyder, Katherine M. Aird. aKG-mediated carnitine synthesis promotes chemoresistance in CCNE1-high ovarian cancers via enhancing histone acetylation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7139762235894a9ff9c39c95032bcf77d46290e" target='_blank'>
              Abstract B031: aKG-mediated carnitine synthesis promotes chemoresistance in CCNE1-high ovarian cancers via enhancing histone acetylation
              </a>
            </td>
          <td>
            Apoorva Uboveja, Nathaniel W. Snyder, K. Aird
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Endometrial cancer (EC) is a common gynecologic malignancy that often exhibits molecular features such as extensive somatic copy number alterations, microsatellite instability and frequent TP53 mutations, which considerably affect the physical and mental well-being of women. The Fanconi anemia (FA) pathway is a DNA damage repair pathway involving multiple FA genes that play crucial roles in DNA damage repair as well as the maintenance of genome stability. Abnormalities in FA, such as deletions or mutations, may lead to defects in DNA damage repair, resulting in increased genomic instability and/or an abnormal cell cycle, ultimately leading to EC. This comprehensive review provides a systematic summary of EC-related FA genes, elucidates the roles of various FA genes in EC and further speculates on their related mechanisms to facilitate the development of targeted therapies that specifically target key genes, leading to a more accurate and efficient treatment for EC. The present review searched PubMed and Google Scholar for articles published in English up to June 2025 using keywords such as Fanconi anemia pathway, 22 FA genes (FANCA, FANCB, FANCC, FANCD1/BRCA2, FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ/BRIP1, FANCL, FANCM, FANCN/PALB2, FANCO/RAD51C, FANCP/SLX4, FANCQ/XPF, FANCR/RAD51, FANCS/BRCA1, FANCT/UBE2T, FANCU/XRCC2, FANCV/REV7, FANCW/RFWD3), endometrial cancer (type I: Endometrioid adenocarcinoma; Type II Uterine serous carcinoma, clear-cell carcinoma, carcinosarcoma), somatic copy number alterations, microsatellite instability, TP53 mutations, pathogenesis, genomic instability, target therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd983a8338fd34520960aa5e9a79e6e3166a39ae" target='_blank'>
              Potential role of Fanconi anemia pathway in the pathogenesis of endometrial cancer (Review)
              </a>
            </td>
          <td>
            Mengmeng Yao, Chuqi Liu, Huiyu Ping, Kaidi Meng, Xinru Li, Qingxin Li, Yuanmin Qi, Ziming Zhu, Li Zhang, Aizhong Han
          </td>
          <td>2025-08-21</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) with rearrangement of the mixed lineage leukemia gene express MLL-AF9 fusion protein, a transcription factor that impairs differentiation and drives expansion of leukemic cells. We report here that the zinc finger protein GFI1 together with the histone methyltransferase LSD1 occupies the promoter and regulates expression of the lncRNA ELDR in the MLL-r AML cell line THP-1. Forced ELDR overexpression enhanced the growth inhibition of an LSD1i/ATRA combination treatment and reduced the capacity of these cells to generate leukemia in xenografts, leading to a longer leukemia-free survival. We found that ELDR binds the clamp protein PCNA and the MCM5 helicases causing defects of DNA replication fork progression. Moreover, AML cells overexpressing ELDR showed reduced chromatin accessibility and transcription at α-satellite repeats in centromeres. In addition, ELDR RNA was detected close to MLL-AF9 at centromeres suggesting that it impedes leukemic progression preferentially of MLL-r AML by interfering with both DNA replication and centromeric transcription. Our findings reveal novel functions of the lncRNA ELDR in DNA replication and centromere biology when expressed at high levels in AML cells with MLL rearrangements. These discoveries could provide rationale for future strategies to treat MLL-r AML, which has a poor prognosis in children and adults. Delivery of the ELDR RNA could potentially be utilized as an adjunct to LSD1i/ATRA treatment or other currently used chemotherapeutic drugs to develop novel therapies for these AML subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bca6f7f7a7b4fc0402c634bb0d6e344f0b3e15b" target='_blank'>
              The lncRNA ELDR suppresses tumorigenicity of AML by interfering with DNA replication and chromatin accessibility.
              </a>
            </td>
          <td>
            K. Arman, Julie Ross, Eva-Maria Piskor, Luca Lazzari, Virginie Calderon, Gaspard Reulet, Maria Vera, Edlie St Hilaire, Hugo Wurtele, Brian T Wilhelm, T. Möröy
          </td>
          <td>2025-09-08</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Under stress conditions, endogenous biomolecules such as nucleic acids or proteins can be released from damaged cells and considered as damage-associated molecular patterns (DAMPs) activating innate immune system and context-dependent responses. In the present study, self-extracellular RNA was obtained from dying (RNA D) and senescent (RNA S) cellular models of osteosarcoma (OS), characterized by NGS, and tested against proliferating and non-proliferating (etoposide-indued senescent) OS cells (U-2 OS, SaOS-2, MG-63, 143B). RNA D and RNA S induced apoptosis, nitro-oxidative stress, nucleic acid sensing pathways and cytokine production, and RNA m5C methyltransferase-based responses (TRDMT1 and NSUN2) in proliferating OS cells. In drug-induced senescent OS cells, TRDMT1 gene knockout (KO) prevented STING activation, related proinflammatory response, and cell death. Furthermore, IFN-β binding RNA partners were identified, namely NSUN2, NSUN5, NSUN6, CDKN1A, MYC, and RAD51 transcripts and these interactions were compromised in TRDMT1 KO cells and upon RNA D and RNA S treatment. TRDMT1 KO also resulted in replication stress in OS cells that was potentiated by RNA D and RNA S stimulation and associated with elevated levels of APOBEC3A and APOBEC3G, members of the cytidine deaminase protein family. In conclusion, we showed that TRDMT1 KO restricted STING-based immune and cell death response to RNA D and RNA S in non-proliferating drug resistant OS cells that might have potential therapeutic implications. Supplementary Information The online version contains supplementary material available at 10.1007/s00018-025-05835-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72593d57bace30d9c5d2b56afce6bbdcb7efd0be" target='_blank'>
              TRDMT1 methyltransferase gene knockout attenuates STING-based cell death signaling during self-extracellular RNA-mediated response in drug-induced senescent osteosarcoma cells
              </a>
            </td>
          <td>
            Gabriela Betlej, A. Deręgowska, M. Wnuk, D. Błoniarz, T. Szmatoła, Katarzyna Klimczak, J. Adamczyk-Grochala, Julia Świętoń, A. Lewińska
          </td>
          <td>2025-08-13</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Bladder cancer (BC) is a disease that predominantly affects older adults, with aging playing a critical role in its onset and progression. Age-associated phenomena, including immunosenescence and chronic inflammation, form a pro-tumor milieu, while genomic instability and epigenetic drift further increase cancer risk. The review highlights the dual role of DNA methylation in BC: global hypomethylation can activate transposable elements and oncogenes, whereas focal hypermethylation silences tumor-suppressor genes like CDKN2A, especially detrimental in older tissues that rely on these genes for senescence control. In parallel, frequent mutations in chromatin modifiers (e.g., KDM6A, KMT2D) and overexpression of histone-modifying enzymes (e.g., EZH2) alter the tumor epigenome to promote immune evasion and tumor aggressiveness. At the non-coding RNA level, dysregulated microRNAs (miRNAs) and long non-coding RNAs (lncRNAs) in BC contribute to aberrant proliferation, metastatic potential, and immune suppression, with aging-associated declines in miRNA processing further exacerbating these effects. Collectively, the accumulation of epigenetic alterations in older patients appears to facilitate both tumor progression and resistance to therapy. Looking forward, epigenetic biomarkers may improve early detection and risk stratification. Furthermore, “epigenetic therapies,” such as DNA methyltransferase inhibitors (DNMTi), EZH2 inhibitors (EZH2i), or histone deacetylases inhibitors (HDACi), hold promise to restore tumor-suppressor function and enhance immunogenicity, offering an attractive avenue for improving outcomes in older patients with BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42bc5ece634826938f00d72c674db3193d520a4b" target='_blank'>
              Epigenetic regulation of bladder cancer in the context of aging
              </a>
            </td>
          <td>
            Xuewei Liu, Guofeng Ding, Yifan Liu, Xiaoli Yan, Yan Zhao, Hailin Lv, Xiaojuan Xu
          </td>
          <td>2025-08-21</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The global population constantly breathes particulate matter with an aerodynamic diameter of ≤10 µm (PM10)—a human carcinogen linked to lung cancer. Previous studies have indicated that PM10 causes DNA damage, including double-strand breaks (DSBs). In particular, DSBs are primarily repaired by the non-homologous end joining (NHEJ) pathway, which is essential for maintaining genomic stability; however, the effects of PM10 exposure on this pathway are unknown. To address this, A549 lung epithelial cells were exposed to 10 µg/cm2 of PM10 for 6, 12, and 24 h. We determined that DSBs increased with prolonged exposure, and an increase in the frequency of micronuclei was found. Despite the accumulated DNA damage, no changes in the cell cycle were observed. Reductions in the levels of the Ku80 gene and protein, as well as the Ku70–Ku80 heterodimer—which is essential for initiating NHEJ-mediated repair—were observed. Levels of Artemis (which is responsible for processing DNA damage) remained stable, while levels of the XRCC4 gene and protein (responsible for completing repair) decreased. We conclude that PM10 disrupts two key proteins in the NHEJ pathway, impairing the capacity for DSB repair. This could promote the accumulation of DNA damage and induce genomic instability, contributing to the development of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91b5b62f2a8611e0c9322a1102ef2e5535b148c7" target='_blank'>
              Airborne PM10 Decreases Ku80 Expression and Ku70–Ku80 Heterodimer Levels of the Non-Homologous End Joining Repair Pathway in Lung Epithelial Cells
              </a>
            </td>
          <td>
            E. M. Quezada-Maldonado, Javier Ivan Lozolla-Ortiz, Miguel Santibáñez-Andrade, Rocío Morales-Bárcenas, C. García-Cuellar, Y. Sánchez-Pérez
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Replication stress (RS), if not effectively and timely addressed, could result in DNA damage in mitosis. However, the relationship between RS and other mitotic events, such as nuclear envelope (NE) breakdown and reassembly, remains poorly understood. Here we report that RS can lead to NE defect. Importantly, rather than de novo NE rupture, the defect per se is a result of nuclear envelope reassembly defect (NERD) during mitosis. Interestingly, NERD is associated with mitotic DNA damage, and repair of the damage by DNA polymerase theta (Polθ)-mediated end joining (TMEJ) ameliorates NERD. Genomic mapping of lamina associated domains (LADs) by cleavage under targets and tagmentation (CUT&Tag) identifies a population of replication stress-sensitive LADs (RESSLADs). Strikingly, a substantial portion of RESSLADs reside in the common fragile sites (CFSs). The loss of RESSLADs-NE interaction under RS might be attributed to the sustained phosphorylation of Lamin A/C at the sites of NERD. In addition, prominent NE defect is observed under multiple conditions of synthetic lethality. Altogether, these findings establish a link between genome instability and nuclear vulnerability under replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/939377138dd121885a6964852dac3b04dd8c9c53" target='_blank'>
              Mitotic DNA repair by TMEJ suppresses replication stress-induced nuclear envelope reassembly defect
              </a>
            </td>
          <td>
            Guojun Ye, Yide He, Yihui Zhang, Dongchen Li, Fuhai Liu, Yi Li, Qinglian Ge, Qiong Guo, Shuya Han, Chunyu Song, Weiping Chang, Haoyue Zhang, Qin Peng, Kun Sun, Weike Ji, Lin Deng
          </td>
          <td>2025-10-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Osteosarcoma is a genomically complex tumor characterized by widespread structural rearrangements. This complexity has limited development of therapeutic strategies informed by molecular mechanisms of oncogenesis. We hypothesized that epigenetic mechanisms could drive distinct subtypes of osteosarcoma. Through analysis of chromatin accessibility, we identified an "early osteoblast-derived" (EOD) cell state characterized by upregulation of transcription factors associated with early bone development, and a "late osteoblast-derived" state (LOD), characterized by upregulation of genes involved in late bone development. We then defined core regulatory circuitries governing the underlying gene expression programs in these two cell states. Multiomic single-cell analysis indicates that these cell states co-exist in a single tumor. Finally, using a panel of patient-derived xenograft models, we identified differential drug responses dependent on these cellular states. These findings create opportunities for developing new combination therapy strategies for osteosarcoma treatment and underscore the value of defining epigenetic subtypes in highly genomically complex cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061de65b88f0b6c7fc51a022f24db3f11873c460" target='_blank'>
              Epigenetic and transcriptional programs define osteosarcoma subtypes and establish targetable vulnerabilities.
              </a>
            </td>
          <td>
            E. López-Fuentes, Andrew S Clugston, Alex G Lee, L. Sayles, Natalie Sorensen, María V Pons Ventura, Stanley G Leung, Truc Dinh, Marcus R. Breese, E. Sweet-Cordero
          </td>
          <td>2025-10-03</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Breast cancer biology is influenced by both genetic ancestry and the environment with implications for disease outcomes. However, we lack the knowledge how non-cancerous cells within tumors are functionally altered by these factors at single-cell resolution. Here, we created a single-nucleus (sn) dataset from 33 African American, 25 Kenyan, and 24 European American women with breast cancer using combined RNA and ATAC sequencing. We successfully isolated intact, high-quality single nuclei from archival frozen breast tumor tissue using an optimized combination of enzymatic digestion and automated tissue homogenization. SnMultiome sequencing of 82 tumors was performed using the 10x Genomics platform. Following filtering, normalization, peak calling, and integration, our dataset includes a total of 292,458 nuclei. Cancerous (163,419 nuclei) and non-cancerous (129,039 nuclei) cells were distinguished based on DNA copy number. Within the microenvironment, 11 major immune, epithelial, and stromal cell types were identified. CD45+, EPCAM- cells were further analytically extracted, clustered, and manually annotated based on known marker genes, detecting 26 distinct immune subpopulations across our samples. After adjustment for potential confounders (age, body mass index, and tumor molecular subtype) in all analyses, we found that women of African descent possessed distinct, activated mesenchymal and endothelial cell subpopulations within the tumor vasculature that promote inflammation, angiogenesis, and cancer cell invasion. Women of African descent exhibited a more immune-suppressive tumor microenvironment, with elevated expression of immune checkpoint markers (PD-1, TIGIT, CTLA-4, and LAG-3), functional impairment of dendritic cells, and enrichment of regulatory and tolerogenic lymphocytic subpopulations. Further, malignant cells from African descent patients expressed a discrete set of CAR-T targets with potential implications for precision immunotherapy. Lastly, we developed three gene signatures within the microenvironment that are predictive of survival with external validation in both tumor-adjacent normal and cancerous tissues from the TCGA. Together, we uncovered molecular breast tumor characteristics of clinical significance in women of African ancestry.



 Alexandra R. Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H. Dorsey, Eytan Ruppin, Shahin Sayed, Francis Makokha, Gretchen L. Gierach, Stefan Ambs. Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C109.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b4c16fead2784b8ef235041a0b9c641f582e01" target='_blank'>
              Abstract C109: Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy
              </a>
            </td>
          <td>
            Alexandra R Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H Dorsey, E. Ruppin, Shahin Sayed, Francis W. Makokha, Gretchen L. Gierach, S. Ambs
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a highly heterogeneous disease, with significantly higher incidence and fatality rates in the elderly. Even with recent decades of research progress in AML, the exact etiology of this deadly disease is still not fully understood, with recent advancements in sequencing technologies highlighting the role of a growing number of non-coding RNAs (ncRNAs) that are intimately associated with AML leukemogenesis. These ncRNAs have been found to have a significant role in leukemia-related cellular processes such as cell division, proliferation, and death. A few of these non-coding RNAs exhibit potential as prognostic biomarkers. The three main groups of ncRNAs that contribute unique activities, especially in cancer, are microRNAs (miRNAs), long non-coding RNAs (lncRNAs), and circular RNAs (circRNAs). Their existence or altered expression levels frequently offer vital information on the diagnosis, course of treatment, and follow-up of cancer patients. The identification of ncRNAs has opened up new avenues for the diagnosis, prognosis, and therapy of acute myeloid leukemia. In order to provide a clear understanding of the significant influence that lncRNAs have on prognostic predictions and diagnostic accuracy in AML, this review aims to provide a comprehensive and insightful understanding of how these molecules actively participate in the complex landscape of the disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f9b9b43383afdfc6c7d6a505ee096f783aeb161" target='_blank'>
              Role of Non-Coding RNAs in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Shailendra S Maurya, Sarita Maurya, S. Chaturvedi
          </td>
          <td>2025-09-19</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a particularly aggressive subtype of breast cancer with high metastatic potential, limited treatment options, and low patient survival rates. By combining functional proteomics and genomics approaches, we identified an oncogenic transcriptional network in mesenchymal and invasive TNBC involving the glucocorticoid receptor (GR), GATA6, MYC, and AP-1 transcription factors. Although these transcription factors bound extensively to shared enhancers, they utilized different enhancer repertoires from this shared enhancer pool to drive distinct downstream oncogenic pathways. The SWI/SNF chromatin remodeling complex was a common gatekeeper of chromatin access for these transcription factors, and SWI/SNF inhibition decommissioned the distinct enhancer networks and pro-tumorigenic gene programs they control. Consistently, SWI/SNF inhibition repressed TNBC proliferation, glucocorticoid-induced chemoresistance, and invasion in vitro and suppressed tumor growth and metastasis in vivo. Thus, these findings indicate that SWI/SNF-mediated chromatin remodeling drives TNBC progression, positioning SWI/SNF as a promising therapeutic target in this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5aecd9fb0bc36400b799414dcc4be0d90b23c11" target='_blank'>
              SWI/SNF Functions as a Gatekeeper of Enhancer Chromatin Access to Control Progression of Mesenchymal Triple-Negative Breast Cancer.
              </a>
            </td>
          <td>
            Rikkemaja Isahbell J Møldrup, M. Terp, S. T. Jakobsen, Tina Ravnsborg, M. S. Madsen, Daniel D Brown, Laurits R Sloth, Michael C. Gorry, S. Oesterreich, Adrian V. Lee, Henrik J Ditzel, Ole N. Jensen, Rasmus Siersbæk
          </td>
          <td>2025-09-10</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Introduction Ovarian Cancer remains a significant global health concern, with high mortality rates, largely due to late-stage diagnosis and limited treatment options. These extrinsic factors are driven or exacerbated by intrinsic mechanisms such as persistent activation or upregulation of Signal Transducer and Activator of Transcription 3 (STAT3). STAT3 promotes tumor growth, inhibits apoptosis, accelerates angiogenesis and metastasis, facilitates immune evasion, and contributes to chemoresistance. Consequently, STAT3 activation fosters an aggressive ovarian cancer phenotype, contributing to treatment failure, poor prognosis and low survival rates, highlighting the urgent need for novel, safe, effective and affordable STAT3-targeted therapeutic strategies. In this study, we developed a novel double-stranded DNA minicircle (mcDNA) inhibitor, designed to act as a decoy for STAT3, preventing its binding to target gene promoters. Methods Utilizing the SKOV3 ovarian cancer cell line, we evaluated the effects of our inhibitor in vitro on cell viability through MTS assay, its apoptotic and necrotic effects using flow cytometry and the expression modulation of downstream STAT3-regulated genes, assayed through RT-qPCR and Western blot analysis. Results We demonstrate that anti-STAT3 mcDNA significantly reduces the viability of SKOV3 cells at low nanomolar concentrations, while sparing the control group. The effect observed was dose-dependent. Mechanistically, anti-STAT3 mcDNA induces apoptosis and necrosis in treated cells, also revealing a certain dose-dependency, while also decreasing cell proliferation. Finally, our inhibitor significantly downregulates STAT3-dependent anti-apoptotic genes MCL1 and PIM1. Conclusion These findings suggest that anti-STAT3 mcDNA is a promising, effective and specific candidate for targeted STAT3 inhibition in SKOV3 ovarian cancer cells, warranting further validation in ovarian cancer, in vivo exploration and potential application in other types of malignancies, where STAT3 acts as an oncogenic factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/298edc0f21db1280384ca9a0eea5b54ca57a8bc1" target='_blank'>
              Breaking the cancer code: a novel DNA minicircle to disable STAT3 in ovarian cancer cells SKOV3
              </a>
            </td>
          <td>
            Adina-Gabriela Vasilescu, Andrei-Mihai Vasilescu, Livia E. Sima, Natalia Baran, S. Szedlacsek
          </td>
          <td>2025-09-15</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Graphical Abstract Abstract Dysfunction of several WD40 family proteins causes diverse endocrine diseases. Until recently, MEP50, a WD40 protein, was considered a gene of unknown significance because no inherited disease had been linked to its function. However, genetic inactivation of MEP50 in mouse models or somatic mutations in humans drives oncogenesis in several endocrine-related cancers, including those of the prostate, breast, and uterus. In this study, we generate new knowledge through a multi-tier integration of evolutionary genomics, sequence and structural analyses, molecular mechanic calculations, and dynamic simulations of wild-type and cancer-mutated MEP50 proteins. Indeed, we find that a conserved splicing event across evolution generates an alternative MEP50 isoform, which is smaller than the canonical MEP50 and lacks the final β-sheet of the first WD40 domain, the entirety of the second WD40 domain, and the first β-sheet of the third WD40 domain. Notably, we find that this novel, short MEP50 (s-MEP50) transcript encodes a 278 amino acid protein that retains aspects of the key regulatory and interaction sites, including those critical for androgen receptor and PRMT5 binding. Finally, we analyze the mutational landscape of MEP50 in endocrine-regulated cancers and use molecular mechanic calculations and dynamic simulations to reveal that cancer-associated mutations disrupt conserved bonding networks and induce widespread structural destabilization within the WD40 domain architecture. Thus, by combining evolutionary, structural, and biophysical approaches, we advance the understanding of MEP50 genomics, providing significant mechanistic and clinically relevant insights into endocrine-regulated tissues and their cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e4d5f2f3bba5deb45827f67cca6b9cab9291043" target='_blank'>
              Genomic features of the methylosome protein MEP50 and its implications in hormone signaling and cancer
              </a>
            </td>
          <td>
            Gareth Pollin, Young-In Chi, Raul Urrutia, G. Lomberk
          </td>
          <td>2025-09-01</td>
          <td>Endocrine Connections</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Oncogenesis, tumor progression and therapy response are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions between these two systems influence tumor evolution and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC that systematically integrates genetic cancer drivers and infiltration profiles of immune cells to find pairwise combinations of drivers and TME characteristics that jointly associate with clinical outcomes. By analyzing 8500 primary tumor samples of 26 cancer types, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell levels were significantly correlated with patient survival. Subsets of tumor samples defined by some IGXs were characterized by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of immunotherapy target genes. In luminal-A breast cancer, an IGX involving MEN1 deletion combined with reduced levels of neutrophils associated with lower progression-free survival and deregulation of immune signaling pathways, as observed in two independent cancer genomics datasets. These results showcase the ability of PACIFIC to integrate complex multi-omics datasets with clinical information, enabling the identification of clinically relevant immunogenomic interactions. Such interactions provide a rich set of hypotheses for mechanistic studies and the development of biomarkers and therapeutic targets. Implications: Co-occurrence patterns of cancer drivers and TME characteristics highlight synergistic interactions with prognostic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3413ab1802170ae0154c57fdbd8827be618414" target='_blank'>
              Cancer genomic alterations and microenvironmental features encode synergistic interactions with disease outcomes.
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P Klein, Alexander T Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C L Cheng, Jigyansa Mishra, Diogo Pellegrina, K. Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2025-09-03</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC) remains a lethal malignancy with limited therapeutic advancements, underscoring the need to identify novel oncogenic drivers. This study integrates multi-omics analyses-including ChIP-seq, bulk RNA-seq, single-cell ATAC/RNA-seq, and spatial transcriptomics-to delineate enhancer-driven transcriptional networks in 59 matched LUSC tumors and adjacent normal tissues. The analyses identify 3,447 tumor-specific oncogenic enhancers (SOEs) enriched for master transcription factors (SOX2, TP63, KLF5, GRHL2) that orchestrate malignant programs. Single-cell epigenomic profiling reveals these SOEs to be exclusively active in a cancer cell-like cluster, pinpointing PTPRZ1, a receptor tyrosine phosphatase, as a top SOE-driven candidate. Functional studies demonstrate that PTPRZ1 is essential for LUSC-tumorigenesis and tumor cells proliferation and migration in vitro and in vivo. Spatial transcriptomic analysis further implicates midkine (MDK) as the ligand activating PTPRZ1 in LUSC tumor tissue. Mechanistically, PTPRZ1 mediates MDK-induced PI3K phosphorylation that is essential for LUSC tumor growth. This work illustrates enhancer addiction as a hallmark of LUSC, identifies the PTPRZ1-MDK axis as an important targetable pathway, and establishes a paradigm for dissecting epigenetic vulnerabilities in solid tumors through multi-omics integration. These findings advance precision oncology strategies for LUSC, bridging epigenomic dysregulation to therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdd6b0f8500832ade4c7417d38653f9c0588ff5" target='_blank'>
              Enhancer Reprogramming Reveals the Tumorigenic Role of PTPRZ1 in Lung Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yong-Qiang Ning, Jie Wang, Yihui Zhang, Hongrui Li, Zenan Jiang, Xinyu Su, Xiao Yang, Xiaoxuan Wu, Bing Wei, Jianming Ying, Jian Yan, Wenbin Li
          </td>
          <td>2025-09-03</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f45e0bd555bafab74ee26abfc65e0d5026258f51" target='_blank'>
              Acetic acid produced by Staphylococcus epidermidis remodels chromatin architecture and suppresses gene expression in Malassezia restricta.
              </a>
            </td>
          <td>
            Jae Min Lee, Hyun Oh Yang, Hideki Tanizawa, Ken-ichi Noma, Tae Kwon Lee, W. H. Jung, Yong-Joon Cho, Kyoung-Dong Kim
          </td>
          <td>2025-09-12</td>
          <td>mBio</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01fb40ddbc348d6bdf921a2ddd2696429a42111c" target='_blank'>
              Energy Deficiency-Induced ATG4B Nuclear Translocation Inhibits PRMT1-Mediated DNA Repair and Promotes Leukemia Progression.
              </a>
            </td>
          <td>
            Zhenkun Wang, Xianli Zhang, Yuanyuan Zhou, Shuyi Lin, Yuting Fu, Haiping Liu, Qiongdan Gao, YanHong Xiao, Zhao Yin, Shuang Liu, Kexiu Huang, Suqi Deng, Juan Du, Hui Zeng, Jia Wang, Jinping Zheng, Qinghua Zhou, Tianfeng Chen, Xiaoling Gao, Zhenyu Ju, Bo Liu
          </td>
          <td>2025-08-11</td>
          <td>Advanced science</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Telomeres are DNA-protein structures that protect chromosome ends. The DNA component of telomeres shortens as cells divide, and telomere length (TL) is a key biomarker of aging and disease risk. Most previous studies in humans of TL have analyzed average TL; thus, our knowledge of TL variability across chromosome arms remains limited. The availability of long-read whole-genome sequencing (lrWGS) data has enabled the development of computational methods to measure chromosome-specific TL (csTL). We generated lrWGS-based csTLs for > 2,500 All of Us participants and characterized variability in csTL attributable to individuals, chromosome arms, participant characteristics, and technical factors. We found that TL varies by chromosome arm (9.1% of the variance in csTL), mirroring patterns observed in prior studies and highlighting the potential for chromosome-specific mechanisms of TL regulation. Substantial variance in csTL (8.9%) was attributable to individual, independent of age, supporting the hypothesis that individuals are endowed with short or long TL in early development, which is maintained throughout life. While age is inversely associated with TL across all arms, the strength of the association varied, with longer arms showing stronger associations. We demonstrate that csTL estimates can be used to estimate disease associations at individual telomeres, including outlying values in the csTL distribution. Our work identifies lrWGS quality metrics that impact csTL estimation, providing a framework to guide future studies. This study demonstrates the utility of lrWGS data for csTL profiling in population cohorts. Larger studies of csTL are needed to advance our understanding of telomeres in aging and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa69a885ec60b7e17b0882bed745c7bd4aa5094b" target='_blank'>
              Determinants of chromosome-specific telomere lengths among 2,573 All of Us participants
              </a>
            </td>
          <td>
            Niyati Jain, Jiajun Luo, Yuqing Yang, B. Aschebrook-Kilfoy, Habibul Ahsan, Lin Chen, Brandon Pierce
          </td>
          <td>2025-08-18</td>
          <td>Research Square</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) is a treatment-resistant subtype that arises through lineage plasticity, allowing tumor cells to bypass androgen receptor (AR)–targeted therapies. In a recent study, Lu et al. define a transcriptional and epigenetic hierarchy that drives this transdifferentiation. The pioneer factor FOXA2 initiates enhancer remodeling and regional DNA demethylation, while the neural lineage transcription factor NKX2-1 is required to complete the NE program. Together, these factors reorganize 3D chromatin architecture and activate lineage-specific genes through enhancer–promoter looping. Crucially, the histone acetyltransferase p300/CBP is essential cofactor in this process. Pharmacologic inhibition of p300/CBP with CCS1477 suppresses NE gene expression and impairs tumor growth in NEPC models. These findings offer a mechanistic insight of lineage plasticity and highlight p300/CBP as promising therapeutic targets. The study also raises key questions about the stability and reversibility of chromatin remodeling and sets a framework for understanding enhancer-driven plasticity in other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3331955821f5eab1873b6edce226793e7651773a" target='_blank'>
              FOXA2 and NKX2-1 Orchestrate Neuroendocrine Lineage Plasticity in Prostate Cancer
              </a>
            </td>
          <td>
            Ka-wing Fong
          </td>
          <td>2025-10-01</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer development and response to treatment are evolutionary processes1,2, but characterizing evolutionary dynamics at a clinically meaningful scale has remained challenging3. Here we develop a new methodology called EVOFLUx, based on natural DNA methylation barcodes fluctuating over time4, that quantitatively infers evolutionary dynamics using only a bulk tumour methylation profile as input. We apply EVOFLUx to 1,976 well-characterized lymphoid cancer samples spanning a broad spectrum of diseases and show that initial tumour growth rate, malignancy age and epimutation rates vary by orders of magnitude across disease types. We measure that subclonal selection occurs only infrequently within bulk samples and detect occasional examples of multiple independent primary tumours. Clinically, we observe faster initial tumour growth in more aggressive disease subtypes, and that evolutionary histories are strong independent prognostic factors in two series of chronic lymphocytic leukaemia. Using EVOFLUx for phylogenetic analyses of aggressive Richter-transformed chronic lymphocytic leukaemia samples detected that the seed of the transformed clone existed decades before presentation. Orthogonal verification of EVOFLUx inferences is provided using additional genetic data, including long-read nanopore sequencing, and clinical variables. Collectively, we show how widely available, low-cost bulk DNA methylation data precisely measure cancer evolutionary dynamics, and provides new insights into cancer biology and clinical behaviour.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57090ca5bdab1ab7fa8186d6d31b9c968e9579a" target='_blank'>
              Fluctuating DNA methylation tracks cancer evolution at clinical scale
              </a>
            </td>
          <td>
            C. Gabbutt, M. Duran-Ferrer, Heather E. Grant, D. Mallo, F. Nadeu, J. Househam, N. Villamor, Madlen Müller, Simon Heath, E. Raineri, O. Krali, J. Nordlund, T. Zenz, I. Gut, E. Campo, A. López-Guillermo, J. Fitzgibbon, C. P. Barnes, D. Shibata, J. Martín-Subero, Trevor A. Graham
          </td>
          <td>2025-09-10</td>
          <td>Nature</td>
          <td>1</td>
          <td>95</td>
        </tr>

        <tr id="


 Genetic alterations in p53 (TP53) are found in most solid malignancies, including pancreatic cancer (PANC). Mutant p53 (p53mut) contributes to cancer progression and metastasis. Existing therapeutic options for p53mut cancers are ineffective and cause toxic side effects, stressing the need for better therapies. To address these clinical problems, we developed a two-drug therapeutic strategy that selectively targets p53mut cancers. This novel strategy combines a thymidine analogue (trifluorothymidine (TFT), a component of TAS102, acting as an inducer of DNA damage), and an inhibitor of poly (ADP) ribose polymerase (PARPi) that works as an amplifier of DNA damage. Incorporation of TFT into DNA provokes post-replicative repair generating single-strand DNA break intermediates. Repair of these intermediates is assisted by PARP, while inhibition of PARP increases more lethal double-strand DNA breaks. In normal p53 wild-type (WT) cells, TAS102 and PARPi activate p53-dependent G1/S checkpoint, limiting DNA damage and promoting repair. In p53mut cells with compromised G1/S checkpoint, TAS102 and PARPi cooperate as the inducer-amplifier pair to increase lethal DNA breaks and cell death. The TAS102-PARPi strategy was validated in preclinical studies. Our first-in-human phase I study with TAS102-talazoparib regimen (NCT04511039) showed that this regimen is safe with promising efficacy. The current work investigated the mechanisms underlying the response to TAS102-PARPi regimen in p53mut cancer cells. Our data show that TAS102-PARPi induces the G2/M checkpoint mediated by ATR kinase in a p53-independent manner. ATR acts through the G2-checkpoint kinases (CHK1 and WEE1) to inactivate CDK1 and stop transition to mitosis. The transcriptome analysis showed that TAS102-PARPi treatment increased expression of Claspin and inactivated PLK1-driven mitotic pathway. Claspin is critical for activation of DNA damage and replication checkpoints by facilitating ATR-CHK1 signaling, whereas PLK1 mediates inactivation of the G2-checkpoint kinases and degradation of Claspin for normal cell cycle progression and re-entry into mitosis. Thus, the PLK1-Claspin network may play a key regulatory role in G2-checkpoint in response to TAS102-PARPi by restraining premature reentry into cell cycle of cells with unrepaired DNA. The next major question was whether TAS102-PARPi regimen can cooperate with drugs disrupting G2-checkpoint. New data show that the efficacy of TAS102-PARPi therapy is greatly enhanced by a sequential application of G2-kinase WEE1 inhibitor. Our data show that this sequential triple-drug strategy works in a synthetic-lethality manner to cause massive cell death in p53mut cancers. The triple-drug strategy greatly improved tumor control in PANC patient-derived xenografts with no signs of major toxicities to normal tissues. In summary, this study provides the scientific foundation for our novel sequential triple-drug strategy that demonstrates greater efficacy against pancreatic cancer with lower toxicity.



 Andrei Bakin, Mohammed Alruwaili, Ashley Guo, Thomas Melendy, Barbara Foster, Christos Fountzilas. Effective targeting p53 mutant pancreatic cancer by novel drug-combination strategy leveraging their DNA damage liabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a93f85c005599dc813a5323d69a7be402a3898b5" target='_blank'>
              Abstract B001: Effective targeting p53 mutant pancreatic cancer by novel drug-combination strategy leveraging their DNA damage liabilities
              </a>
            </td>
          <td>
            A. Bakin, M. Alruwaili, Ashley Guo, T. Melendy, Barbara Foster, C. Fountzilas
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41c845deccf83a402934c97c50b9796b93645c5b" target='_blank'>
              Establishment of chromatin architecture interplays with embryo hypertranscription.
              </a>
            </td>
          <td>
            Guang Yu, Kai Xu, Weikun Xia, Ke Zhang, Qianhua Xu, Lijia Li, Zili Lin, Ling Liu, Bofeng Liu, Zhenhai Du, Xia Chen, Qiang Fan, Fangnong Lai, Wenying Wang, Lijuan Wang, Feng Kong, Chao Wang, Haiqiang Dai, Huili Wang, Wei Xie
          </td>
          <td>2025-08-13</td>
          <td>Nature</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Excised signal circles (ESC) are circular DNA molecules generated during T- and B-cell maturation. Previously considered biologically inert, recent work by Gao et al. now show that ESCs can replicate and accumulate in healthy lymphocytes. Moreover, the authors link higher levels of ESCs to an increased risk of relapse in B-cell leukemia patients and propose that this phenomenon is due to the unique ability of ESCs to induce genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a90caaf42de342ff59afcf74bb20559cdc1183c5" target='_blank'>
              Not just a by-product: circular DNA molecules derived from V(D)J recombination are linked to worse prognosis in B-cell leukemia.
              </a>
            </td>
          <td>
            Davide Pradella, Andrea Ventura
          </td>
          <td>2025-09-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Cancer research has long focused on mutations in normal body cells, but this approach has not produced major breakthroughs for most cancers. Our study explores a different concept that some aggressive cancers may actually arise from early reproductive cells called primordial germ cells, which normally develop into eggs and sperm. We created a new experimental model showing how a virus can transform human primordial germ cell-like cells into virus-positive Merkel cell carcinoma, a rare but deadly skin cancer. This model shows that cancers can emerge through changes in developmental states rather than relying solely on genetic mutations. By linking cancer development to early germ cells, our findings suggest a unifying explanation for both germ cell cancers and body cancers. This new perspective may guide more effective approaches to study, diagnose, and treat cancer by focusing on early human development rather than only DNA mutations and later developmental stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de5f636d947e93ff13e3e8ff3f419d1f1d6cb51" target='_blank'>
              A Polyomavirus-Positive Merkel Cell Carcinoma Mouse Model Supports a Unified Origin for Somatic and Germ Cell Cancers
              </a>
            </td>
          <td>
            Wendy Yang, Sara Contente, Sarah Rahman
          </td>
          <td>2025-08-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Targeted hypermutation tools are useful for engineering proteins and pathways, and exploring the evolutionary landscapes. However, existing targeted hypermutation tools for genomic loci mostly exhibit restricted mutation windows and limited mutational types. Here, by integrating mutagenic, high-processivity bacteriophage T5 or T7 DNA polymerases (DNAPs) with CRISPR-Cas9, the study develops an in vivo mutagenesis system that enables all possible types of nucleotide substitutions and an expanded mutation window of up to 2 kilobases, achieving a maximum mutation rate 1.1 × 106-fold higher than wild-type Escherichia coli. Through MS2-mediated recruitment of T5 or T7 DNAP for co-localization with nickase nCas9, off-target rate is reduced by up to 96.8% without compromising on-target rate. Further benefiting from the dTnpB-based transcriptional repression system, the mutagenesis process can be properly regulated during continuous evolution. Finally, the CRISPR-TDNAP-assisted targeted mutagenesis for regulable laboratory evolution (CTRLE) confers cellular triple-antibiotic resistance in 8 days, and enhances the efficiency of the twin-arginine translocation pathway by over threefold in 6 days. Furthermore, CTRLE proves effective in Bacillus subtilis and Kluyveromyces lactis, yielding targeted mutation rates 1.2 × 10⁵-fold and 5 × 10⁷-fold higher than host backgrounds, respectively. Collectively, CTRLE provides an efficient and universal way to accelerate the continuous evolution of different microbial cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/453cdc9bc3fd8236cad5dbabb388b99d7317237d" target='_blank'>
              CRISPR-DNA Polymerase Assisted Targeted Mutagenesis for Regulable Laboratory Evolution.
              </a>
            </td>
          <td>
            Shuaili Chen, Xiangdi Chen, Yifan Peng, Qinghua Li, Jingwen Zhou, Jianghua Li, Guocheng Du, Jian Chen, Guoqiang Zhang
          </td>
          <td>2025-09-23</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Epstein-Barr Virus (EBV) establishes life-long latent infection in >90% of adults and is a causal agent for diverse cancers and autoimmune diseases. EBV has a complex life cycle in multiple different tissue types that involve dynamic variations in viral gene expression. These gene expression changes account for the success of the virus in long-term persistence and evading host immune control, as well as its potential for driving cancer evolution and autoimmune disease. Here, we review some of the salient features of EBV gene regulation highlighting the many variations of viral transcription. We review recent advances in our understanding of the factors that bind and regulate EBV gene expression. Based on this diversity of viral transcription patterns, we propose that EBV genome consists of gene modules regulated by local promoter-proximal transcription factor combinations that are further regulated by distal regulatory interactions among the various modules that interact through architectural factors, such as CTCF and cohesion. These modules are likely to represent chromatin architectural domains, and can also interact with host chromosome domains that further regulate viral and host gene expression. We propose that this gene regulatory hierarchy provides EBV with necessary plasticity for viral persistence, as well as a strong potentiator for cancer and autoimmune disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a251ae41cd508722e48d62d5ca0f4e635efbec9f" target='_blank'>
              Transcriptional regulation of gene modules in Epstein-Barr virus.
              </a>
            </td>
          <td>
            Paul M. Lieberman
          </td>
          <td>2025-09-22</td>
          <td>Transcription</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="An essential category of non-protein-coding RNA molecules, known as long noncoding RNAs (lncRNAs), are naturally occurring RNA sequences exceeding 200 nucleotides in length. Accumulating evidence has unveiled the significant involvement of lncRNAs in various cancers, where they are known to be a key factor for tumors’ statement changing, e.g., modulating proliferative, apoptotic, migratory and chemotherapy resistance in cells. In addition, lncRNAs are also regarded as potential diagnostic and prognostic tumor biomarkers. Studies have suggested that lncRNAs modulate biological processes by regulating linear RNA transcription and protein production. A recently discovered long non-coding RNA, SNHG3, is identified to have associations with cancer, which displays atypical expression patterns and functions as an oncogene in various cancers. Additionally, SNHG3 has been demonstrated to hold promise as a diagnostic biomarker and a tool for prognostic assessment in cancer patients. This review summarizes current knowledge about SNHG3’s expression, functional roles, molecular mechanisms, and implications for diagnosis and prognosis in human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ffaa9b71dfc7869fc1372cea5b3cd75f3cbc0b5" target='_blank'>
              The role of LncRNA SNHG3 in human cancers
              </a>
            </td>
          <td>
            Jinming Tang, Min Su, Yuhang Xiao, Wei Ou, Wenxiang Wang, Ren-Wang Peng, Zhining Wu
          </td>
          <td>2025-10-03</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0641382c381145c3c4ec79eaca66ce9a1d4b8df0" target='_blank'>
              Telomere Crisis Shapes Cancer Evolution.
              </a>
            </td>
          <td>
            Joe Nassour, J. Karlseder
          </td>
          <td>2025-08-11</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13c548af869e91466a22ceb7887ea1b6fdb37dc0" target='_blank'>
              Integrative proteogenomics and forward genetics reveals a novel mitotic vulnerability in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Nicholas J Neill, S. Satpathy, Karsten Krug, Jitendra K Meena, Nivetha Ramesh Babu, Cheyenne Calderon, Desmon Reed, Marcus J Weber, L. Dobrolecki, Alaina Lewis, Christina Sallas, M. Anurag, Kimberly R Holloway, Chen Huang, S. Vasaikar, Maria F Cardenas, Beom-Jun Kim, Doug W. Chan, Shayan C. Avanessian, S. Tyagi, Mayra Orellana, Sufeng Mao, Heyuan Li, Fade Gong, S. Kurley, K. Meerbrey, Calla M Olson, Amritha Nair, Tingting Sun, Hsiang-Ching Chung, Elizabeth A. Bowling, Jarey H Wang, Pengju Zhang, Peng Xiao, Duxiao Yang, F. Stossi, Mei-Yin C Polley, Alexander B. Saltzman, F. Mundt, D. Mani, Michael A Gillette, S. Hilsenbeck, George Miles, Carolina Gutierrez, C. K. Osborne, Charles Y. Lin, N. Gray, Jinpeng Sun, David A. Wheeler, C. Perou, A. Malovannaya, Michael T. Lewis, Bing Zhang, Matthew J Ellis, Steven A. Carr, Thomas F Westbrook
          </td>
          <td>2025-08-11</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>165</td>
        </tr>

        <tr id="ABSTRACT Background Chronic myeloid leukemia (CML) is a clonal malignancy propelled by the BCR::ABL1 fusion gene originating from the Philadelphia chromosome. This gene activates ABL tyrosine kinase, which enhances the survival of leukemic cells. Although tyrosine kinase inhibitors (TKIs) have significantly advanced the treatment of CML, resistance to these inhibitors presents a substantial hurdle. Consequently, novel therapeutic strategies targeting resistance mechanisms independent of BCR::ABL1 are urgently needed. Methods This study investigated the potential impact of combining WEE1 inhibitors, particularly MK‐1775, with vitamin K2 (VK2) in treating CML. To analyze differentially expressed and spliced transcripts in CML, we examined mRNA profiles from peripheral blood mononuclear cells of five patients with CML (during chronic and blast phases) and five healthy controls. The samples were analyzed using deep sequencing. Differential expression analyses were performed using RaNA‐Seq and Heatmapper, the latter of which was designed for complex data set visualizations. Results WEE1 controls the G2/M checkpoint to prevent early mitosis, and blocking it increases the cytotoxicity of agents that damage deoxyribonucleic acid, especially in cancers lacking p53. VK2, a micronutrient, exerts anticancer effects against various malignancies. Gene expression studies have indicated that PKMYT1 expression is elevated in CML but not WEE1 cells. MK‐1775 successfully halted the growth of both standard and TKI‐resistant CML cell lines by triggering apoptosis via caspase 3/7 activation. VK2 reduced the viability of CML cells and increased cytotoxicity. A combined regimen of MK‐1775 and VK2 markedly decreased colony growth, disrupted mitochondrial membrane potential, and increased death in CML cells, including those resistant to TKIs. Conclusions The results suggest that a combination of MK‐1775 and VK2 represents a potentially effective treatment strategy for CML, especially in drug‐resistant cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/635521a554ad2c4e931009718d411cefa0b2c2c7" target='_blank'>
              Combination of WEE1 Inhibitor and Vitamin K2 Enhances Therapeutic Efficacy in Chronic Myeloid Leukemia
              </a>
            </td>
          <td>
            S. Okabe, Yuya Arai, Akihiko Gotoh, D. Akahane
          </td>
          <td>2025-08-28</td>
          <td>Cancer Innovation</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Tumorigenesis is a multistep process involving germline and somatic alterations, also known as "drivers." Understanding how these alterations cooperate is crucial for assessing cancer risk and developing strategies to intercept malignant transformation.



 To investigate the interplay between germline pathogenic variants (GPVs) and somatic drivers in childhood cancers, and to compare tumor molecular profiles based on the origin of initiating alterations.



 We analyzed whole-genome (paired tumor-germline) and transcriptomic data from 2,397 children enrolled in the ZERO Childhood Cancer Program (2018–2025). A subset of 199 participants with autosomal dominant cancer predisposition syndromes consistent with their tumor type was selected for detailed analysis.



 GPVs were distributed across multiple cancer predisposition genes (CPGs), with TP53, NF1, SMARCB1, and DICER1 being the most prevalent. A second somatic hit in the same CPG was identified in 83% of cases (166/199), 56% resulting from loss of heterozygosity. RNA sequencing revealed additional variants (10% of cases) not detected in the DNA data. Additional somatic drivers were identified in 149 tumors (75%), 82 of which exhibited recurrent drivers for their cancer type (e.g., CDKN2A in NF1-driven malignant peripheral nerve sheath tumor). In 33% of them (27/82), no second hit was present in the CPG (e.g., NF1 GPV in low-grade glioma with somatic FGFR1 mutation), suggesting the GPV may act as a cooperating enabler. Of interest, 67 tumors harbored additional drivers atypical for their cancer type, with most (62/67) showing a second hit in the same CPG, suggesting a distinct molecular profile in this group of GPV-driven tumors (e.g. GPV in FBXW7 and Wilms tumor with somatic CDKN2A loss). In 49 cases, the only somatic event was a somatic second hit in the CPG, indicating that some GPVs may independently initiate tumorigenesis (e.g., SMARCB1, DICER1). This supports the hypothesis that GPVs can enable and promote tumour progression without the need for cooperation with other genomic drivers. Conversely, 25 cases lacked a second hit in the CPG but harbored other somatic drivers, including unexpected tumor alterations (e.g., GPV in RECQL4 and osteosarcoma without TP53 alteration), suggesting possible GPV haploinsufficiency in driving tumorigenesis through an alternative molecular pathway.



 Our study highlights the multifactorial nature of tumour initiation and promotion in GPV-driven tumorigenesis involving direct initiation and cooperation with somatic drivers. Genes like SMARCB1 can drive cancer alone, while others like NF1 need additional somatic alterations for malignancy. Notably, biallelic inactivation of CPG rarely leads directly to high-grade tumors, emphasizing gene and cancer-type specificity of this process. Future research, such as the study of induced pluripotent cancer stem cells, can help clarify the role of GPV and the sequence of multi-step events. This may guide strategies for preventing malignant transformation and enabling earlier detection.



 Noemi A Fuentes-Bolanos, Dianne Sylvester, Eliza Courtney, Chelsea Mayoh, Meera Warby, Dong Anh Khuong Quang, Paulette Barahona, Ashleigh Sullivan, Ann Altekoester, Megan Rumford, Hetal Dholaria, Marie Wong, Vaness Tyrrell, Michelle Haber, Katherine Tucker, Mark J Cowley, David S Ziegler, Mark Pinese, Loretta M S Lau. Cooperation between germline and somatic drivers in childhood cancer: insights from the ZERO cohort [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75b9cb084afe69c38e9d831c9072216bcdeed64f" target='_blank'>
              Abstract A045: Cooperation between germline and somatic drivers in childhood cancer: insights from the ZERO cohort
              </a>
            </td>
          <td>
            N. Fuentes-Bolanos, Dianne Sylvester, Eliza Courtney, Chelsea Mayoh, Meera Warby, Dong Anh Khuong Quang, P. Barahona, Ashleigh Sullivan, A. Altekoester, Megan Rumford, Hetal Dholaria, Marie Wong, Vanessa Tyrrell, Michelle Haber, Katherine Tucker, Mark J. Cowley, D. S. Ziegler, M. Pinese, L. Lau
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Precision oncology is witnessing an increasing number of molecular targets fueled by the continuous improvement of cancer genomics and drug development. Tumor genomic profiling is nowadays (August 2025) part of routine cancer patient care, guiding therapeutic decisions day by day. Nevertheless, implementing and distilling the increasing number of potential gene targets and possible precision drugs into therapeutically relevant actions is a challenge. The availability of prescreening programs for clinical trials has expanded the description of the genomic landscape of gastrointestinal tumors. The selection of the genomic test to use in each clinical situation, the correct interpretation of the results, and ensuring clinically meaningful implications in the context of diverse geographical drug accessibility, economic cost, and access to clinical trials are daily challenges of personalized medicine. In this context, well-established negative predictive biomarkers, such as extended RAS extended mutations for anti-EGFR therapy in colorectal cancer, and positive predictive biomarkers, such as MSI status, BRAF p.V600E hotspot mutation, ERBB2 amplification, or even NTRK1, NTRK2, NTRK3, RET, and NRG1 fusions across gastrointestinal cancers, are mandatory to provide tailored clinical care, improve patient selection for treatment and clinical trials, maximize therapeutic benefit, and minimize unnecessary toxicity. In this review, we provide an updated overview of actionable genomic alterations in GI cancers and discuss their implications for clinical decision making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a041af7c3fe90ad802a0f68558db067f344766d8" target='_blank'>
              Clinical Actionability of Genes in Gastrointestinal Tumors
              </a>
            </td>
          <td>
            Nadia Saoudi Gonzalez, Giorgio Patelli, G. Crisafulli
          </td>
          <td>2025-09-25</td>
          <td>Genes</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Introduction Mre11 is a multisubunit nuclease involved in DNA repair, and its dysfunction often causes DNA damage sensitivity, genomic instability, telomere shortening, and aberrant meiosis. However, the specific roles of Mre11 in porcine oocyte meiosis remain unclear. Methods In this study, porcine oocytes were treated with the Mre11-specific inhibitor mirin to investigate the function of Mre11 during meiotic maturation. Meiotic progression, spindle and chromosome structure, spindle migration, cytoplasmic actin polymerization, and DNA damage levels were assessed using immunofluorescence and relevant molecular markers including BubR1 and γH2A.X. Results Inhibition of Mre11 activity led to failure of first polar body extrusion, with sustained BubR1 presence at kinetochores, indicating activation of the spindle assembly checkpoint (SAC). Mre11-inhibited oocytes showed disrupted spindle and chromosome organization due to decreased microtubule stability. Additionally, spindle migration to the oocyte cortex was impaired, correlating with reduced cytoplasmic actin polymerization. Elevated DNA damage levels were observed in treated oocytes as evidenced by increased γH2A.X staining. Discussion These findings demonstrate that Mre11 is essential for porcine oocyte meiotic progression by maintaining normal spindle assembly, actin cytoskeleton dynamics, and SAC activity. DNA damage accumulation following Mre11 inhibition likely contributes to meiotic failure, highlighting its critical role in ensuring oocyte quality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35c9791f0e71996b2445b7ae813bc649f58ab883" target='_blank'>
              MRE11 orchestrates porcine oocyte meiotic progression by modulating the spindle assembly checkpoint
              </a>
            </td>
          <td>
            Dandan Zhang, Zaishan Yang, Yongteng Zhang, Fugui Fang, Hongguo Cao, Yunsheng Li, Zubing Cao, Yanfeng Xue, Mianqun Zhang
          </td>
          <td>2025-08-08</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="

Breast cancer is the main cause of death for women, even with major improvements
in treatment. Through processes like DNA damage, estrogen metabolism, and immunological
regulation, bacterial populations have been shown to have an impact on breast cancer development
in recent studies.



This review aimed to examine and evaluate current research on the involvement of bacteria
in breast cancer progression, with an emphasis on gene action mechanisms and potential future
treatments targeting the microbiome.



A thorough literature analysis was carried out to identify pertinent research published between
1989-2024 across various databases, including PubMed, Google Scholar, Google, and Scopus.



Bacterial dysbiosis in the gut and breast tissue contributes to the progression of breast cancer
through different pathways. Double-strand breaks in DNA are linked to various bacteria, like
Escherichia coli, Staphylococcus epidermidis, Helicobacter pylori, and Fusobacterium, which contribute
to genomic instability. Breast cancers are influenced by hormones that are influenced by gut
microbiota, namely the estrobolome, which regulates estrogen levels. Bacteria also impact immune
responses by preventing anti-tumor immunity. These results suggest that restoring microbial balance
to specific bacterial taxa may open up new treatment options. Different genes may contribute
to variations, including an increase in regulatory T (Treg) cells, while FOXP3+ T cells are linked to
shorter relapse-free survival. Understanding the microbiota's role in DNA damage, hormone regulation,
and immune modulation is important.



Bacteria contribute significantly to the development of breast cancer through genelevel
processes. Probiotics, immunomodulatory techniques, and microbiome-targeted treatments are
potential future developments that could improve therapy effectiveness and reduce resistance.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2a52b781fcea2d4ef9f890f1a8d1daa10502bc" target='_blank'>
              The Bacterial Role in the Progression of Breast Cancer through Mechanism of Gene Action: Future Prospects with Existing Studies
              </a>
            </td>
          <td>
            Yasir Nawaz, Saba Munir, Sidra Aslam, Fizza Rimal Butt, Fouzia Tanvir, Basit Nawaz, Kazam Bashir, Rikza Tahreem
          </td>
          <td>2025-09-22</td>
          <td>Current Genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94f6767c83b49c5f93547ae2e51f853756e7c466" target='_blank'>
              DNA methylation influences human centromere positioning and function.
              </a>
            </td>
          <td>
            Catalina Salinas-Luypaert, Danilo Dubocanin, Rosa Jooyoung Lee, Lorena Andrade Ruiz, Riccardo Gamba, Marine Grison, Léonid Velikovsky, Annapaola Angrisani, Andrea Scelfo, Yuan Xu, Marie Dumont, Viviana Barra, Therese Wilhelm, Guillaume Velasco, Marialucrezia Losito, René Wardenaar, C. Francastel, F. Foijer, G. Kops, K. Miga, Nicolas Altemose, D. Fachinetti
          </td>
          <td>2025-09-04</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f5b13afa4acec24579eb468c6dc31248a66bb3e" target='_blank'>
              Detection of clonal hematopoiesis of indeterminate potential via genome or exome sequencing profoundly underestimates disease associations
              </a>
            </td>
          <td>
            R. Corty, Y. Pershad, C. Vlasschaert, L. Luo, T. Mack, K. Amancherla, C. Robinson-Cohen, M. Savona, A. G. Bick
          </td>
          <td>2025-08-16</td>
          <td>None</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Epigenetic regulation plays a key role in the progression, proliferation, and dedifferentiation of thyroid cancer. Epigenetic control occurs at multiple levels, including DNA methylation, RNA modification, histone modification, chromatin remodeling, and chromatin accessibility. Genetic alterations in chromatin regulators are commonly observed in thyroid cancer, which includes papillary thyroid carcinoma (PTC), medullary thyroid carcinoma (MTC), anaplastic thyroid carcinoma (ATC), and follicular thyroid carcinoma (FTC). These cancers exhibit distinct characteristics in terms of genetics, biology, and clinical presentation. Therefore, we review the disease biology driven by changes in chromatin pathways in thyroid cancer. Specifically, we summarize examples of epigenetic dysregulation at each level, with mechanisms involving alterations in enzymes regulating DNA methylation, RNA modification and posttranslational modifications of histones, as well as the loss or fusion of subunits involved in chromatin remodeling and chromatin accessibility. Finally, on the basis of clinical applications, we review the current and potential future approaches for thyroid cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4b8e029502363c1309237f110ef671aea68cd77" target='_blank'>
              Epigenetic control in thyroid cancer: mechanisms and clinical perspective
              </a>
            </td>
          <td>
            Jiahui Zhang, Shengkai Zheng, Ruiwang Xie, Junsi Zhang, Xiangjin Chen, Sunwang Xu
          </td>
          <td>2025-08-17</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Menin protein is encoded by the multiple endocrine neoplasia type 1 gene, which typically serves an oncogenic role in endocrine organs. However, in mice, menin protein is a key mediator of leukemic transformation, particularly in acute myeloid leukemia (AML), and is involved in the disease process through epigenetic regulatory mechanisms. This functional paradox may be due to the unique gene expression regulatory properties of menin, which can both activate and inhibit the expression of target genes. At the molecular level, menin protein regulates the gene transcription process by interacting with multiple protein complexes and forms a complex network with multiple signaling pathways. As the core hub of transcriptional regulation, menin protein is essential for the maintenance of cellular homeostasis and its aberrant function can lead to gene expression disorders, which contributes to the development of AML. Despite the druggability challenges of several transcriptionally regulated proteins, inhibitors of menin protein have made breakthroughs in clinical development. Particularly in AML subtypes [for example, lysine methyltransferase 2A (KMT2A) rearrangement or nucleophosmin (NPM1) mutation], menin protein inhibitors have demonstrated favorable efficacy. The present study systematically reviewed biological functions of the menin protein and its application in targeted therapy for specific AML subtypes. Notably, menin inhibitors have demonstrated potential in KMT2A/NPM1-mutant leukemia, however, the off-target effects, resistance mechanisms and a lack of biomarkers due to the extensive nature of their binding interface remains to be elucidated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18b23d4e7fbebe4140893833cf2070b2e3e6ebf" target='_blank'>
              Targeting menin in lysine methyltransferase 2A/nucleophosmin-mutated leukemia: A novel strategy from epigenetic dysregulation to clinical therapy (Review)
              </a>
            </td>
          <td>
            Junjie Bi, Hongjun Zhou
          </td>
          <td>2025-09-10</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Cancer remains a leading cause of global morbidity and mortality, underscoring the need for innovative diagnostic and prognostic biomarkers. PIWI-interacting RNAs (piRNAs) have emerged as critical regulators of gene expression and genomic stability, with increasing evidence linking their dysregulation to oncogenesis. piRNAs can modulate DNA methylation through interactions with DNA methyltransferases (DNMTs), contributing to tumor suppressor gene silencing and broader epigenetic reprogramming. However, the functional implications of piRNA-mediated methylation changes in cancer are not fully elucidated. This study investigates the circulating levels of piR-823 and piR-651, their associations with DNMT3B expression and global DNA methylation, and their correlations with clinical and molecular markers in colorectal, breast, and prostate cancers. Materials and methods A total of 300 participants, including 150 patients with histologically confirmed cancers and 150 age- and sex-matched healthy controls, were enrolled. Plasma piRNA levels were measured via qRT-PCR, while global 5-methylcytosine (5mC) and DNMT3B concentrations were quantified using ELISA. Immunohistochemistry was performed on resected tumor tissues to assess DNMT3B, Ki-67, p53, E-cadherin, vimentin, HER2, and androgen receptor (AR). Correlations were analyzed using Pearson’s test and multivariate regression models. Diagnostic performance was evaluated via Receiver Operating Characteristic (ROC) curves. Results piR-823 was significantly upregulated in colorectal and breast cancers (p < 0.001), while piR-651 was markedly downregulated in prostate cancer (p = 0.002). Cancer patients exhibited elevated levels of DNMT3B and 5mC (both p < 0.001), which correlated strongly with Ki-67 and p53 expression (r > 0.7, p < 0.001). E-cadherin expression was inversely correlated with DNMT3B and piR-823, consistent with a role in epithelial–mesenchymal transition (EMT). cfDNA levels were also significantly elevated in cancer cases (p < 0.001), suggesting potential utility as a non-invasive biomarker. Conclusion The observed associations between piRNA dysregulation, epigenetic markers, and tumor aggressiveness indicators support the potential of circulating piRNAs—particularly piR-823 and piR-651—as novel biomarkers for cancer detection and progression monitoring. These findings warrant further mechanistic studies to validate their functional roles in tumor epigenetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec55cb867b96ef67ad7cc822f9de2fa5e24bf5e8" target='_blank'>
              Epigenetic functions and biomarker potential of PIWI-interacting RNAs in oncogenesis
              </a>
            </td>
          <td>
            Kldiashvili Ekaterina, Abiatari Ivane, Kekelia Elene, Iordanishvili Saba, Metreveli Tornike, Dumbadze Eter
          </td>
          <td>2025-09-30</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Nuclear architecture and chromosome folding are often speculated to influence genome replication. In yeasts, centromeres cluster close to the spindle pole body and telomeres position at the nuclear periphery. This ‘Rabl configuration’ spatially segregates the most early and late replicating parts of the genome, centromeres and telomeres, respectively, suggesting that origin position along the centromere–telomere axis may influence origin activity. Here, we investigated DNA replication in a wild-type, 16-chromosome Saccharomyces cerevisiae strain and in its single-chromosome counterpart engineered by chromosome fusion and elimination of all but two telomeres and one centromere, which strongly affects genome folding and abrogates the Rabl conformation. Using nanopore sequencing-based methods, we found that the DNA replication program of both strains was virtually indistinguishable, with the exception of origin inactivation next to deleted centromeres and changes in origin efficiency and fork direction at chromosome fusions, as anticipated from the known origin-regulation properties of centromeres and telomeres. Only a handful of replication changes, mostly due to local origin repression, were observed elsewhere. Fork speed was also unaffected except at deleted centromeres. In conclusion, the DNA replication program of budding yeast is remarkably resilient to perturbations of chromosome folding and loss of the Rabl conformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e28f5bc4a9bd5e110c57ffd165964c5f1a57708" target='_blank'>
              Replication program of a single-chromosome budding yeast strain
              </a>
            </td>
          <td>
            Jade Pellet, Laurent Lacroix, Bertrand Theulot, Emma Simonin Chavignier, Alan Tourancheau, F. Proux, Sylvie Hermann-Le Denmat, G. Millot, Benoît Le Tallec, O. Hyrien
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background Brain development and neuronal cell specification are accompanied by epigenetic changes that enable the regulation of diverse gene expression patterns. During these processes, transcription factors interact with cell-type-specific epigenetic marks, binding to unique sets of cis-regulatory elements in different cell types. However, the detailed mechanisms through which cell-type-specific gene regulation is established in neurons remain to be explored. Results In this study, we conducted a comparative histone modification analysis between excitatory and inhibitory neurons. Our results revealed that neuronal cell-type-specific histone modifications are enriched in super enhancer regions that contain abundant EGR1 motifs. Further CUT&RUN assay confirmed that excitatory neurons exhibit more EGR1 binding sites, primarily located in enhancers. Integrative analysis demonstrated that EGR1 binding is strongly correlated with various epigenetic markers of open chromatin regions and is linked to distinct gene pathways specific to neuronal subtypes. In inhibitory neurons, most genomic regions containing EGR1 binding sites become accessible during early embryonic stages, whereas super enhancers in excitatory neurons, which also host EGR1 binding sites, gain accessibility during postnatal stages. Conclusions This study highlights the crucial role of transcription factor binding, such as EGR1, to enhancer regions, which may be key to establishing cell-type-specific gene regulation in neurons. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02357-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b332587abb6a2d2f6f1559fe6cf75f2d99d13f89" target='_blank'>
              Elevated EGR1 binding at enhancers in excitatory neurons correlates with neuronal subtype-specific epigenetic regulation
              </a>
            </td>
          <td>
            Liduo Yin, Xiguang Xu, Benjamin Conacher, Yu Lin, G. Carrillo, Yupeng Cun, Michael A. Fox, Xuemei Lu, Hehuang Xie
          </td>
          <td>2025-08-11</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5de0d15fb6fe47895cf59141995dd4d8303baf" target='_blank'>
              Temporal genomic dynamics shape clinical trajectory in multiple myeloma.
              </a>
            </td>
          <td>
            F. Maura, Marcella Kaddoura, A. Poos, Linda B. Baughn, B. Ziccheddu, Marc-Andrea Bärtsch, Anthony Cirrincione, K. Maclachlan, M. Chojnacka, B. Diamond, M. Papadimitriou, P. Blaney, L. John, Philipp Reichert, S. Huhn, Dylan C Gagler, Yanming Zhang, Ahmet Dogan, A. Lesokhin, Faith Davies, H. Goldschmidt, Roland Fenk, Katja C. Weisel, E. Mai, N. Korde, Gareth J Morgan, S. Rajkumar, Shaji K. Kumar, S. Usmani, O. Landgren, M. Raab, Niels Weinhold
          </td>
          <td>2025-08-20</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="Extracellular vesicles (EVs) are promising biomarkers for cancer diagnosis and prognosis due to their ability to carry specific biomolecular cargo, including DNA. However, the clinical utility of DNA methylation-based liquid biopsies using EV-DNA remains underexplored. The low quantity and relatively long length of EV-DNA complicate whole-genome methylation profiling. To address this, we develop Tn5-assisted Enzymatic Methyl-sequencing with Post-conversion Tailing (TEMPT), a bisulfite-free whole-genome profiling method for EV-DNA. TEMPT employs single-adapter Tn5 tagmentation, enzymatic conversion of unmodified cytosines, and post-conversion tailing to generate high-depth whole-genome EV-DNA methylomes. We apply TEMPT to EV-DNA from 58 gastric cancer and polyp samples, generating methylomes from sub-nanogram inputs and identifying differentially methylated regions (DMRs) that distinguish cancer from controls. We identify potential cancer biomarkers through DMR-associated genes, highlighting the roles of EVs in cellular communication. Our findings suggest that immune cells may serve as an alternative source of EV-DNA. This approach holds significant promise for advancing EV-DNA research and its applications in early disease diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad67a59c8fc16bd26c5fe5c57db4a838137c7af1" target='_blank'>
              Whole-genome methylation profiling of extracellular vesicle DNA in gastric cancer identifies intercellular communication features
              </a>
            </td>
          <td>
            Bingqian Lin, Zhenna Jiao, Shouquan Dong, Weikai Yan, Jinting Jiang, Yanfang Du, Xiaocheng Weng, Hongling Wang, Zhiyuan Hu, Yibin Liu, Xiang Zhou
          </td>
          <td>2025-08-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background One of the most intriguing and unusual features of trypanosomatids is their mitochondrial DNA, known as kinetoplast DNA (kDNA), which is organized into a network of concatenated circles. The kDNA is contained within the mitochondrial matrix and can exhibit distinct arrangements across different species and during cell differentiation. In addition to kDNA, the kinetoplast contains multiple proteins, including those involved in mitochondrial DNA topology and metabolism, such as the kinetoplast-associated proteins (KAPs). In this work, we obtained mutant cells to investigates the role of KAP7 in two trypanosomatid species, Trypanosoma cruzi and Angomonas deanei, which have distinct kinetoplast shapes and kDNA arrangements. Methods For this purpose, the kDNA replication process and cell morphology and ultrastructure were evaluated using microscopy methods. Furthermore, the proliferation of cells treated with genotoxic agents, such as cisplatin and ultraviolet radiation, was analyzed. Results In A. deanei, which contains a symbiotic bacterium, KAP7 seems to be essential, since the deletion of one KAP7 allele generated mutants with a decay in cell proliferation, as well as changes in kDNA structure and replication. In T. cruzi, null mutants exhibited disturbances in kDNA replication, although the overall topology remained unaltered. The use of cisplatin and ultraviolet (UV) radiation affected the ultrastructure of A. deanei and T. cruzi. Cisplatin promoted increased kDNA compaction in both KAP7 mutants, but only in T. cruzi did the proliferative capacity fail to recover after treatment, as was also observed following UV radiation exposure. Conclusions Proteins associated with DNA are evolutionarily conserved and usually perform similar functions in different organisms. Our findings reveal that KAP7 is involved in kDNA replication, but its roles differ in trypanosomatid species: in A. deanei, KAP7 is associated with kDNA arrangement, while in T. cruzi, it is related to mitochondrial metabolism, such as kDNA replication and damage response. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s13071-025-06985-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbf00b20cf0ea3a2d1d73d597428bf9948be9fc1" target='_blank'>
              From mitochondrial DNA arrangement to repair: a kinetoplast-associated protein with different roles in two trypanosomatid species
              </a>
            </td>
          <td>
            C. S. Gonçalves, C. M. Catta-Preta, B. Repolês, Wesley Roger Rodrigues Ferreira, Flávia Souza Morini, Jeremy C. Mottram, D. P. Cavalcanti, Wanderley de Souza, S. Fragoso, Carlos Renato Machado, M. Motta
          </td>
          <td>2025-08-28</td>
          <td>Parasites & Vectors</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1cae074f29f78d9d694a20a3e12fe166bc58ebc" target='_blank'>
              RAD51-mediated homologous recombination is a pro-tumour driver pathway.
              </a>
            </td>
          <td>
            Bernard S. Lopez
          </td>
          <td>2025-09-24</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eb04dfc69e03ddcb4aa348ab8eef9592fa55873" target='_blank'>
              p300/CBP is an essential driver of pathogenic enhancer activity and gene expression in Ewing sarcoma.
              </a>
            </td>
          <td>
            Laura C Godfrey, Brandon Regalado, Sydney R Schweber, Charlie Hatton, Daniela V. Wenge, Yanhe Wen, Meaghan Boileau, Maria Wessels, Jun Qi, Christopher J Ott, K. Stegmaier, Miguel N Rivera, Scott A Armstrong
          </td>
          <td>2025-09-01</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) evolves as an aggressive phenotype during prolonged androgen deprivation therapy, lacking effective clinical management. Here, we elucidated a reciprocal metabolic-epigenetic mechanism involving a positive feedback loop between glycolysis and the transcription factor LHX2 that promotes PCa progression. Mechanistically, enzalutamide-mediated inhibition of the androgen receptor (AR) led to upregulation of key glycolytic enzymes. Elevated glycolysis resulted in lactate accumulation and subsequent histone lactylation, which in turn enhanced LHX2 expression. Reciprocally, LHX2 transcriptionally activated the lactate dehydrogenase LDHA, which further amplified lactate production. Furthermore, LHX2 augmented the expression of the lineage-determining enzyme DNMT1, potentiating neuroendocrine gene expression and tumor growth. Significantly, the antiviral agent paritaprevir could directly bind to LHX2, effectively suppressing neuroendocrine marker expression and tumor progression. These findings uncover a potential role for LHX2 in orchestrating NEPC development, offering putative therapeutic targets for treating this aggressive cancer phenotype and overcoming drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef2fefc98e5fd3352706deab4254aa3b33bda2c" target='_blank'>
              LHX2 Rewires the Metabolic and Epigenetic Landscape to Drive Tumor Progression in Prostate Cancer.
              </a>
            </td>
          <td>
            Jun Jiang, Shaojie Liu, Chao Xu, Limin He, Hongji Li, Yike Zhou, Zhengxuan Li, Yu Li, Fa Yang, Yuchen Wei, Tong Lu, Keying Zhang, Jingliang Zhang, Pang Wang, Li Guo, Changhong Shi, Weihong Wen, Rui Zhang, Donghui Han, Weijun Qin
          </td>
          <td>2025-09-03</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The presence of the PRDM16::SKI fusion gene was described, for the first time, in a T prolymphocytic leukemia (T-PLL) patient with a long indolent period and a late development treatment requiring disease. The fusion transcript was detected by RNA sequencing and validated by reverse transcriptase polymerase chain reaction and Sanger/Cycle sequencing. The chimera occurs between exon 1 of the PR/SET Domain 16 (PRDM16) gene and exon 2 of the oncogene V-Ski Avian Sarcoma Viral Oncogene Homolog (SKI) gene. The finding provides insight into the role of genetic alterations, including fusion genes, in development and progression of T-PLL and may possibly lead to the development of effective and precise targeted therapy for this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/314985a5230a419836262a0347f8d007d1be6717" target='_blank'>
              Identification of a novel PRMD16::SKI fusion gene in T-prolymphocytic leukemia
              </a>
            </td>
          <td>
            M. Brunetti, H. K. Gjelberg, H. Reikvam, F. Micci
          </td>
          <td>2025-08-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Simple Summary DNA repair mechanisms are crucial for maintanence of DNA and cell integrity. When disrupted, they can lead to increased mutation rate and contribute to the cancerogenesis. On the other hand, cancer cells that bear mutations in genes involved in DNA repair are more prone to DNA damaging insults which opens opportunities for treatments involving synthetic lethality. microRNAs are short non-coding RNAs that regulate expression by binding to gene transcripts and inducing mRNA degradation or inhibition of translation. Here, we review the miRNA-mediated dysregulation of genes involved in DNA damage response (DDR) and DNA repair pathways in ovarian cancer (OvCa), one of the deadliest female malignancy. We also discuss miRNAs, which affect response to OvCa therapy by regulating PARP1 (Poly(ADP-Ribose) Polymerase-1), a central gene of the OvCa synthetic lethality treatments. Finally, we address the limitations of miRNAs as diagnostic biomarkers and potential targets for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd4db6c3d00cf07b80c8f608dd0b112d9dfce41" target='_blank'>
              DNA Damage and Repair in Ovarian Cancer: Focus on MicroRNAs
              </a>
            </td>
          <td>
            Katarzyna D. Arczewska, A. Piekiełko-Witkowska
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Nuclear biomolecular condensates are essential sub-compartments within the cell nucleus and play key roles in transcription and RNA processing. Bottom-up construction of nuclear architectures in synthetic settings is non-trivial but vital for understanding the mechanisms of condensates in real cellular systems. Here, we present a facile and versatile synthetic DNA protonucleus (PN) platform that facilitates localized transcription of branched RNA motifs with kissing loops (KLs) for subsequent condensation into complex condensate architectures. We identify salinity, monomer feeding, and KL-PN interactions as key parameters to control co-transcriptional condensation of these KLs into diverse artificial nuclear patterns, including single and multiple condensates, interface condensates, and biphasic condensates. Over time, KL transcripts co-condense with the PN matrix, with the final architecture determined by their interactions, which can be precisely modulated using a short DNA invader strand that outcompetes these interactions. Our findings deepen the understanding of RNA condensation in nuclear environments and provide strategies for designing functional nucleus-mimetic systems with precise architectural control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13da8ed75a9cef66af63d63bcdff443dc8cb37d3" target='_blank'>
              Constructing synthetic nuclear architectures via transcriptional condensates in a DNA protonucleus
              </a>
            </td>
          <td>
            Miao Xie, Weixiang Chen, Maria Vonk-de Roy, Andreas Walther
          </td>
          <td>2025-09-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Detection of cancer at early stage can significantly improve the five-year survival rate of patients. Bisulfite-based methylation detection can cause DNA damage, especially in high GC content regions which was associated with the development of cancers. Loss of aberrant methylated CpG sites in cfDNA will lead to the undetectability of certain circulating tumor DNA (ctDNA), consequently may affect the cancer detection. Our study uses enzymatic method to detect whole genome abnormal methylation regions in esophageal squamous cell carcinoma (ESCC). We also provide a pretrained neural network, hybrid of BERT and CNN (BCNN), to identify ctDNA robustly. Maximum posterior probability is utilized to estimate the fraction of ESCC-derived ctDNA in plasma for predicting the risk of ESCC cancer. Our results analysis indicated that enzyme-based whole-genome methylation sequencing retained more longer cfDNA and detected more CpG sites than bisulfite-based method in both gDNA and cfDNA. Enrichment analysis of differentially methylated regions (DMR) showed that top five pathways were associated with ESCC. Compared to traditional models, our BCNN demonstrates the best performance in identifying ctDNA (AUC = 0.970). By estimating the fraction of plasma ctDNA, our BCNN exhibits high accuracy in ESCC detection even at ultralow sequencing depths (AUC = 0.946). Specifically, in the validation cohort, when the specificity is 93.75%, 7 out of 8 early-stage ESCC (TNM Stage I) were identified as positive in our preliminary results. In conclusion, our preliminary results reinforce the idea of employing BCNN as a novel strategy for ESCC early detection in clinical practice. However, to be applied in clinical, further validation with a larger sample size is necessary. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18278-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5af110454f8624475f32847f585947aa544c81e" target='_blank'>
              Improved circulating tumor DNA identification for detection of esophageal squamous cell carcinoma by enzymatic methyl sequencing and hybrid neural network
              </a>
            </td>
          <td>
            Jiangfeng Shen, Yuqi Ren, Ziyong Mao, Qiuxiang Hu, Yilong Wan, Jiangcun Du, Yanting Lai, Shengxiang Shao, Liuqing Zhang, Hao Wu, Jiaxi Li, Sheng Ju, Xin Tong, Jun Zhao, Lei Cao, Deyi Xiong, ChengCheng Xu, Jun-Chi Xu, Dong Jiang
          </td>
          <td>2025-09-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Glioblastoma multiforme represents the most common primary malignant brain tumor in adults, with a 90% 2-year recurrence rate despite standard therapy of surgical resection followed by chemo- and radiotherapy. Yet, the cellular mechanisms underlying therapeutic resistance remain poorly understood. By integrating analysis of single-cell RNA sequencing results of 41 primary and 43 recurrent tumor specimens with experimental validation, we identify neuro-oligodendrocyte precursors as the predominant cellular population driving tumor recurrence through establishment of protective microenvironmental niches. We discover that CTNNA3 and MBP exhibit paradoxical roles in promoting survival in primary tumors while conferring chemoresistance in recurrent GBM, revealing that these factors determine treatment efficacy rather than directly driving relapse. Functional assays confirmed their role as mediators of extracellular matrix remodeling, establishing a mechanistic link between tumor microenvironmental remodeling and therapeutic resistance. By revealing how recurrent tumors engineer self-protective niches, our study provides a new framework for developing combination therapies that disrupt these resistance mechanisms and provides foundation for overcoming treatment failure in this universally fatal disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ec11a04512f1a7cb7a16592e9fd8fa7748da070" target='_blank'>
              CTNNA3 and MBP Mediate Treatment Resistance Through Tumor Microenvironment Remodeling in Recurrent Glioblastoma
              </a>
            </td>
          <td>
            Wenna Mao
          </td>
          <td>2025-09-24</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) can be effectively treated inhibiting the disease-causing BCR::ABL1 kinase by tyrosine kinase inhibitors (TKIs). Although therapy is initially tremendously successful, resistance may occur in up to 25% of CML patients. Besides aberrations in the BCR::ABL1 kinase domain, a variety of resistance mechanisms are currently discussed, among them epigenetic reprogramming. The histone-modifying enzyme lysine methyltransferase 2D (KMT2D/MLL2) belongs to the most frequently mutated genes in cancer and is also known for its association with hereditary Kabuki syndrome. However, its role in CML is widely unknown. In the present study, we analyzed the role of the KMT2D p. (Arg191Trp) variant in imatinib-resistant CML, which was recurrently acquired in imatinib resistance in vitro. SiRNA-mediated KMT2D knockdown, but also introduction of the p. (Arg191Trp) variant into treatment-naïve K-562 cells led to impaired imatinib susceptibility visible by increased cell numbers, proliferation rates and metabolic activities under imatinib exposure (p < 0.001). The effect of KMT2D p. (Arg191Trp) could be overcome by inhibiting histone demethylation with the demethylase inhibitor LSD1. In addition, rescue of KMT2D expression in imatinib-resistant cells reinstated the response to imatinib treatment. Furthermore, gene expression analysis revealed upregulation of CCNE2 in cells harboring KMT2D p. (Arg191Trp) potentially explaining increase in cell proliferation under imatinib exposure. Overall, our findings demonstrate that the loss of the tumor suppressor KMT2D promotes TKI resistance in CML. Thus, KMT2D status could serve as an additional biomarker for TKI resistance, while restoration of its expression might be a therapeutic option to overcome this resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59b4b4f7562cd8617674ef60daedf2504f6b6932" target='_blank'>
              The role of the lysine histone methylase KMT2D in chronic myeloid leukemia
              </a>
            </td>
          <td>
            Lina Schlemminger, Inga Nagel, I. Vater, Ingolf Cascorbi, Meike Kaehler
          </td>
          <td>2025-09-16</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="With the increase in dog life expectancy, canine mammary cancer has become the most critical and frequent neoplasm in dogs. As in humans, this type of cancer has spontaneous development and is influenced by several risk factors, such as age and hormonal exposure in addition to genetic and epigenetic factors. Thus, knowing that epigenetic patterns influence a variety of biological mechanisms, such development cancer, this study aimed to correlate the methylation pattern of LINE-1 elements in healthy and neoplastic canine mammary tissue, as well as in isolated cell lines of carcinoma canine with tumoral aggressiveness degree. To this end, a retrospective study was conducted in which genomic DNA was extracted from histological sections of biopsies and cell lines using the phenol/chloroform method. To determine the percentages, the genomic DNA was pretreated with methylation-sensitive endonucleases and then amplified by qPCR with primers specific for the L1td1 gene. The results revealed that both the tumor cell lines and the canine adenoma and inflammatory carcinoma samples are hypomethylated relative to normal tissue. However, it was not possible to confirm the relationship between the percentage of LINE-1 hypomethylation and tumor aggressiveness in canine breast cancer. Therefore, the combined analysis of our results allows us to conclude that the methylation pattern of LINE-1 elements can be used as a molecular biomarker to differentiate normal canine mammary tissue from neoplastic canine mammary tissue. However, to confirm whether there is a relationship between the degree of methylation of LINE-1 elements and tumor staging, further studies with canine samples are needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da3e474a09cd5273ac4d9d3cd5402967aa34da4d" target='_blank'>
              ASSOCIATION BETWEEN LINE-1 HYPOMETHYLATION IN CANINE MAMMARY TUMOR TISSUES: IMPLICATIONS FOR DIAGNOSIS AND RESEARCH
              </a>
            </td>
          <td>
            Patrícia A. dos Santos de Carvalho Joaquim, José Guilherme Xavier, M. Lallo, Rodrigo A. Silva
          </td>
          <td>2025-09-24</td>
          <td>Revista CPAQV - Centro de Pesquisas Avançadas em Qualidade de Vida</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Vincristine is a classical chemotherapeutic agent widely used for its ability to disrupt microtubule polymerization, yet additional molecular effects may contribute to its anticancer activity. G-quadruplexes (G4s), non-canonical nucleic acid structures enriched in regulatory regions of the genome and in mitochondrial DNA, have emerged as relevant modulators of cellular homeostasis. In this study, we investigated whether vincristine can influence G4 biology. Cancer cells treated with vincristine were analyzed by immunofluorescence, revealing a consistent increase in nuclear and mitochondrial G4 foci. In particular, mitochondrial G4s were significantly elevated by approximately 1.5–2.5 fold compared to untreated cells, an effect accompanied by a detectable reduction in membrane potential, indicative of impaired organelle function. In addition, biophysical analyses on representative G4-forming sequences were carried out. Proton nuclear magnetic resonance titrations showed localized chemical shift perturbations upon vincristine addition, circular dichroism confirmed preservation of G4 topology, and isothermal titration calorimetry indicated weak but enthalpically favorable interactions. Taken together, these results suggest that vincristine perturbs both the cellular G4 landscape and mitochondrial homeostasis, while also engaging G4 DNA in vitro. Although additional studies are required to establish the mechanistic details, this work provides proof-of-concept for a previously unrecognized dimension of vincristine’s anticancer action.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9ceb68af52929ce6ba68361c5ca182e12edea35" target='_blank'>
              Vincristine Beyond Mitosis: Uncovering a First Link to G-Quadruplex DNA in Cancer Cells
              </a>
            </td>
          <td>
            Anna Di Porzio, Carolina Persico, Francesca Romano, Alessandra Barra, Immacolata Aiello, Ludovica D'Auria, Sara Abate, Federica D’Aria, C. Giancola, Elpidio Cinquegrana, F. S. Di Leva, Jussara Amato, Simona Marzano, Nunzia Iaccarino, Antonio Randazzo
          </td>
          <td>2025-10-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="This study examines the contribution of dysregulated long non-coding RNAs (lncRNAs) to disease progression and prognosis in breast invasive carcinoma (BRCA) and cervical squamous cell carcinoma with endocervical adenocarcinoma (CESC) using data from The Cancer Genome Atlas (TCGA). Comparative analyses identified both shared and cancer-specific lncRNAs linked to tumorigenesis. Survival analysis using the Kaplan–Meier method highlighted several candidates with prognostic relevance. Notably, FAM83H-AS1 showed increased expression in advanced stages of BRCA and was linked with lymph node involvement and metastatic status. Its elevated expression was associated with poor survival outcomes in both BRCA and CESC cohorts. In contrast, lncRNAs such as FGF14-AS2, which were downregulated, demonstrated links to tumor-suppressive mechanisms and improved survival in BRCA patients. Subcellular localization analysis indicated predominant cytoplasmic presence of FAM83H-AS1, implying possible roles in post-transcriptional regulation. To support our in-silico observations, we carried out qRT-PCR experiments in breast cancer cell lines with varying degrees of aggressiveness, which confirmed the differential expression patterns of these lncRNAs. Overall, our findings highlight the clinical importance of lncRNA dysregulation in gynaecological cancers and point to their potential utility as biomarkers for prognosis and targeted therapy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-16436-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96a627efdc32c519561de180942228807a9ccfd0" target='_blank'>
              Functional characterization of dysregulated LncRNAs in BRCA and CESC highlights their role in cancer prognosis
              </a>
            </td>
          <td>
            Madhur Sharma, Nidhi Chourasia, Priyanka Priyanka, D. P. Sarkar, A. Nag, Sandeep Saxena
          </td>
          <td>2025-10-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Simple Summary Breast cancer is a complex disease influenced not only by genes but also by the way cancer cells generate and use energy. One rare change is the loss of a gene called fumarate hydratase (FH), which normally helps cells produce energy efficiently. We examine how this change influences how tumors grow and respond to treatment. By studying thousands of breast cancer samples, we found that tumors missing FH tend to create a special environment that helps them grow and survive, especially by encouraging new blood vessel formation. We also describe a patient with this FH change who had a remarkable and long-lasting response to a therapy that blocks blood vessel growth (known as anti-VEGF treatment). These findings suggest that identifying this rare energy-related alteration may help clinicians determine which patients are most likely to benefit from this therapy. Such insights could contribute to advancing more personalized approaches to breast cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c214b499120e8d0fc3e3396e45d4d3eb6e6c018" target='_blank'>
              Revealing the Angiogenic Signature of FH-Deficient Breast Cancer: Genomic Profiling and Clinical Implications
              </a>
            </td>
          <td>
            L. A. Sinberger, Noa Keren-Khadmy, Assaf Goldberg, T. Peretz-Yablonski, A. Sonnenblick, Mali Salmon-Divon
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Methylation of DNA CpG domains in cellular DNA is a key mechanism of epigenetic regulation. Disruptions in the processes maintaining DNA methylation can lead to diseases like cancer. The radiation response of certain cancer cells may be affected by their DNA methylation levels, which may have consequences in their response to radiotherapy. In this work, we utilized DNA origami nanotechnology to examine whether DNA methylation impacts DNA response to ionizing radiation in solution before biological processes come into play. Our findings reveal that a protective effect is achieved with just a few methylated CpG adducts. Both low-LET (electron) and high-LET (carbon ion) irradiation show a reduced lesion count in methylated DNA, as indicated by qPCR results. AFM single-molecule observations using DNA origami nanoframes suggest fewer double-strand breaks in methylated DNA after carbon ion irradiation. This radioprotective effect may contribute to the differential radiation response of cellular DNA and should be considered when predicting and evaluating DNA radiation damage yields.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60446411a5ce1e0fa6ca09e48e25c2f668acc1a4" target='_blank'>
              CpG Methylation Protects DNA against Ionizing Radiation
              </a>
            </td>
          <td>
            Leo Sala, Tereza Zápotocká, Jana Šáchová, Václav Olšanský, David Chvatil, François Chevalier, Violaine Vizcaino, Alain Mery, J. Kočišek
          </td>
          <td>2025-08-20</td>
          <td>The Journal of Physical Chemistry. B</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Small intestinal neuroendocrine tumors (SI-NETs) represent the most frequent neoplasms of the small intestine, with loss of chromosome 18 as the most common genetic aberration. To date, no highly frequently mutated genes have been identified on chromosome 18 or elsewhere. This study aimed to explore the potential role of methyl-CpG binding domain protein 2 (MBD2), located at 18q21.2, as a tumor suppressor gene in SI-NETs. By immunohistochemistry, we found undetectable or very low levels of MBD2 expression in 64% of the analyzed SI-NETs, and overall, very low levels of mRNA and protein expression were observed. Overexpression of MBD2 in the SI-NET cell line, GOT1, reduced cell growth and induced apoptosis, though cell migration remained unaffected. In addition, MBD2 expression decreased cell proliferation and migration capacity of the neuroendocrine cells NCI-H727 but showed no effect on apoptosis. Furthermore, MBD2 expression in both cell lines was found to upregulate E-cadherin but downregulate the levels of N-cadherin and vimentin in GOT1 and NCI-H72, respectively, as determined by western blot analysis. These results suggest that MBD2 plays a role in inhibiting the epithelial-mesenchymal transition process and may function as a potential tumor suppressor gene in SI-NETs. Future studies to elucidate the underlying mechanisms are warranted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14eeef15fb15c0cced59de17e27889264700227a" target='_blank'>
              Loss of MBD2 is associated with tumor growth in small intestinal neuroendocrine tumors
              </a>
            </td>
          <td>
            Elham Barazeghi, S. Backman, Per Hellman, O. Norlén, P. Stålberg
          </td>
          <td>2025-09-01</td>
          <td>Endocrine Connections</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Despite numerous studies investigating intratumoral microorganisms and their significant roles in cancer initiation, progression, and treatment efficacy, a systematic understanding of intratumoral microorganisms remains lacking. Herein, we conducted a study using 16S rDNA data on seven types of cancer, comprising a total of 783 samples. It’s worth noting that Pseudomonas, Streptococcus, and Prevotella were found to be shared with the microbial communities of the seven cancers, suggesting that these may be associated with the occurrence and development of cancers. We anticipate establishing a foundation for related research and exploring potential methods for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80274119fe0660855f0a9a793c84d8230ca38ecb" target='_blank'>
              Pan-cancer analysis reveals intratumoral microbial diversity in multiple cancers by amplicon technology
              </a>
            </td>
          <td>
            Xudong Liu, Yuteng Yao, Aoyi Xiao, Dingyan Cao, Jingheng Zhang, Yanan Shi, Qing Zhong, Zilong He, Wenming Wu
          </td>
          <td>2025-09-02</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Rad51 is a pivotal DNA repair protein that plays a central role in homologous recombination repair. Its significance in tumorigenesis and cancer treatment resistance has garnered increasing attention. In recent years, studies have shown that Rad51 not only plays an important role in the biological characteristics of tumor cells but is also closely related to tumor immunity. The complexity of the tumor immune microenvironment makes the action mechanism of Rad51 more complex and affects the immune system's ability to recognize and clear tumors by influencing the interaction between tumor cells and immune cells. Although preliminary studies have suggested a potential role of Rad51 in tumor immunity, the underlying molecular mechanisms require further elucidation. Furthermore, the research on Rad51 as a therapeutic target is also continuously deepening, indicating its application potential in tumor treatment. In summary, this review aims to comprehensively analyze the role of Rad51 in tumor immunity and its challenges and opportunities in future tumor treatment research and clinical application in order to provide new ideas for clinical tumor immunotherapy. However, the current understanding of Rad51's immunomodulatory functions remains incomplete, and further preclinical and clinical studies are needed to validate its therapeutic potential, address potential off‐target effects, and elucidate tumor‐type‐specific mechanisms and microenvironmental interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00a9c3b142aaabf70111312c22aa00050983254" target='_blank'>
              Rad51 at the Crossroads: Bridging DNA Repair, Tumorigenesis, and Immune Regulation
              </a>
            </td>
          <td>
            Zhendong Qin, Mingjun Lu, Jingwei Guo, Chenyang Wang, Teng Ma
          </td>
          <td>2025-09-24</td>
          <td>Med Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Multiple myeloma (MM) is characterized by extensive intratumoral heterogeneity and complex interactions within the bone marrow microenvironment, yet the cellular and molecular drivers of treatment resistance remain poorly defined. Protein tyrosine phosphatase receptor gamma (PTPRG) has emerged as a candidate tumor suppressor in various malignancies by antagonizing proliferative and survival signaling, but its functional and prognostic relevance in MM has not been established. Methods We analyzed 103,171 single‐cell transcriptomes from 18 MM samples (10 optimal responders [OR] and 8 suboptimal responders [SOR] to bortezomib–melphalan–prednisone) to investigate cell‐type composition, malignant plasma cell subclusters, and tumor–microenvironment crosstalk. InferCNV was used to distinguish malignant plasma cells, which were further reclustered and correlated with bulk prognostic phenotypes. Differential expression, pathway enrichment, transcription‐factor activity, pseudotime trajectory, and ligand–receptor interaction analyses were performed. Finally, bulk datasets (GSE9782, GSE2658, MMRF-CoMMpass) and in vitro knockdown assays in U266 and NCI-H929 cells were used to validate the prognostic and functional role of PTPRG. Results Eleven major cell types were annotated, with plasma cells, T/NK cells, and CD14+ monocytes predominating; SOR samples exhibited an expanded plasma‐cell fraction and reduced T/NK, CD14+ monocyte, pre-B, and HSPC populations. Among 35,944 malignant plasma cells, five subclusters were defined; one subcluster (MalPlasma3) was enriched in SOR samples and harbored 93.1% of cells associated with poor survival. MalPlasma3 and “worse‐survival” cells showed activation of stemness, E2F/MYC targets, and G2M checkpoint pathways, driven by transcription factors E2F8, E2F7, FOXM1, E2F1, and TIMELESS. Pseudotime analysis revealed a bifurcating differentiation toward a resistant phenotype, accompanied by upregulation of cell‐cycle and proliferation modules. In the OR group, enhanced cytotoxic features in NK, effector, and naïve T cells, along with IGF1–IGF1R and IFNG–IFNGR signaling, suggested a supportive microenvironment. In contrast to the known role as a tumor suppressor in solid and hematologic cancers, our integrative analyses identified PTPRG among seven stemness‐related genes upregulated in MalPlasma3 and poor‐survival cells, which was echoed in the observed reduced cell viability and increased apoptosis in MM cell lines following siRNA‐mediated PTPRG knockdown. Conclusions This single‐cell multi‐omic dissection implicates a proliferative, stem‐like MalPlasma3 subcluster and identified PTPRG as a key mediator of drug resistance and poor outcome in MM, offering novel prognostic biomarkers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48b1aa46cace7effec1d51940ecccc2bdb09fa3e" target='_blank'>
              Single-cell profiling uncovers PTPRG-driven stemness in malignant plasma cells and signatures of treatment failure in multiple myeloma
              </a>
            </td>
          <td>
            Jiewen Tan, Jinman Zhong, Yueping He, Yunman Xu, Chang Chen, Dan Xiong
          </td>
          <td>2025-09-18</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Gemykibivirus, an emerging single-stranded DNA (ssDNA) virus of the recently established genus in the family of Genomoviridae, had been discovered in human blood and cerebrospinal fluid and a variety of other body fluids. However, the molecular mechanisms of gemykibivirus entrance into the host cells and its pathogenicity remain poorly understood. To investigate the host response of gemykibivirus, we used an infectious clone of gemykibivirus previously established through molecular biology techniques to rescue virus in HEK293T cells and analyzed the changes in the host transcriptome during the infection period by RNA-Seq. Our findings indicate that gemykibivirus can both express viral proteins and accomplish replication, and high-throughput transcriptome analysis identified a total 1732 significantly different genes. Functional enrichment analysis of Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways for differentially expressed genes (DEGs) showed gemykibivirus involving several important pathways, including MAPK signaling pathway, Chemical carcinogenesis-reactive oxygen species and Oxidative phosphorylation. Interestingly, mitochondrial DNA-encoded mRNAs exhibited varying levels of upregulation, suggesting that gemykibivirus may be involved in mitochondrial fission and the regulation of mitochondrial function. Subsequently, a series of experiments proved that gemykibivirus can lead an increase in mitochondrial DNA copy number, promote the release of mtDNA into the cytoplasm, enhance reactive oxygen species production and trigger other cellular antiviral responses. Overall, we lay a foundation for revealing the relationship between Gemykibivirus and human diseases through mitochondrial functional alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a67fc3d43e01f79ea8442e27ecafc7aa6d559e12" target='_blank'>
              Transcriptome Analysis Reveals Gemykibivirus Infection Induces Mitochondrial DNA Release in HEK293T Cells
              </a>
            </td>
          <td>
            Runbo Yang, Hao Yan, Yifan Wang, Wenqing Yang, Jianru Qin
          </td>
          <td>2025-09-30</td>
          <td>Viruses</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies, with limited treatment options and poor prognosis. Recent advances in cancer genomic analysis enable the identification of actionable gene alterations, opening new opportunities for personalized therapy. Among these, homologous recombination DNA repair (HRR) gene alterations are associated with distinct biological behavior, favorable prognosis, and increased sensitivity to platinum-based chemotherapy. However, the prognostic impact of coexisting mutations in key driver genes—KRAS, TP53, CDKN2A, and SMAD4—within HRR-altered PDAC remains poorly understood. Methods We retrospectively analyzed PDAC patients who underwent genomic profiling testing with FoundationOne® CDx between June 2019 and December 2021 through the Center for Cancer Genomics and Advanced Therapeutics (C-CAT) database. We compared the prevalence and prognostic significance of key gene alterations between HRR-altered and HRR–wild-type (WT) tumors. Results Of 2,381 PDAC patients, 274 (11.5%) harbored HRR alterations. These patients showed significantly longer overall survival (OS) than those with HRR-WT tumors (HR = 0.66, p = 0.002). The frequencies of KRAS, TP53, and CDKN2A mutations were less frequent in HRR-altered tumors. TP53 mutation was independently associated with poorer OS across both HRR subgroups, while CDKN2A alteration was a poor prognostic factor in HRR-WT tumors. Interestingly, SMAD4 alteration was linked to improved survival in the HRR-altered group. Conclusion HRR-altered PDAC has a distinct genomic profile and is associated with a favorable prognosis. Our findings demonstrate that coexisting alterations are significant prognostic factors in both HRR-altered and HRR–wild-type tumors. These results highlight the clinical relevance of incorporating comprehensive genomic profiling into routine care to stratify patient prognosis better and inform individualized treatment strategies in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada42c90fd2263fd4d9bb1466e612ccff4b0a39f" target='_blank'>
              Prognostic impact of gene alterations via homologous recombination DNA repair gene alteration status in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Yusuke Kawanaka, Chiaki Inagaki, Masaki Okura, S. Mitani, T. Takahama, K. Yonesaka, Y. Chiba, Kazuhiko Nakagawa, H. Kawakami, H. Hayashi
          </td>
          <td>2025-08-25</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by a pronounced inter- and intra-tumoral heterogeneity, which fuels chemoresistance and contributes to high mortality rates. Previously, we have developed a branched organoid system embedded in collagen matrices that robustly recapitulates the phenotypic heterogeneity seen in both murine and human PDAC1,2. These organoids display complex, self-organized branching morphogenesis and give rise to distinct, spatially ordered tumor cell populations that reflect their underlying molecular profiles and differentiation states. Importantly, we show that the observed heterogeneity is not random but governed by defined transcriptional programs, particularly epithelial-to-mesenchymal plasticity, that drive the emergence of discrete tumor-cell states. Using integrated phenotypic and transcriptomic profiling, we map this diversity to specific biological functions in vivo, demonstrating that each organoid phenotype corresponds to a tumor-cell state with unique metastatic potential, and therapeutic vulnerabilities. Moreover, we identify dynamic, treatment-induced phenotype reprogramming events that are targetable, paving the way for rational design of state-specific therapeutic interventions. Building on our findings, we are now focusing on elucidating intra-organoid heterogeneitysuch as tip–trunk hierarchies at the single-cell level, to uncover the mechanisms of self-organization and to determine how distinct organoid phenotypes and subpopulations contribute to liver metastasis and treatment resistance. In summary, we have established a scalable and mechanistically informative organoid platform that enables in vitro modeling of PDAC heterogeneity. This system provides a framework for dissecting the tumor cell–intrinsic drivers of phenotypic plasticity and for developing phenotype-guided treatment strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47eb75a781046a1e97c826bc3797a33d46a03686" target='_blank'>
              LEVERAGING ORGANOID TECHNOLOGIES TO CHARACTERIZE DYNAMIC TUMOR CELL STATES DRIVING TREATMENT RESISTANCE IN PANCREATIC CANCER
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Human papillomavirus (HPV) is a prevalent viral pathogen that causes a variety of malignancies, including cervical cancer, one of the leading causes of cancer-related deaths among women worldwide. The HPV E2 protein is a central regulator of viral replication and oncogene expression, making it a critical determinant of HPV-associated malignancies. While its core functions are conserved, variations within the E2 protein are thought to contribute to the differential oncogenic potential among HPV types, though the structural basis for this remains incompletely understood. Previous research from our laboratory suggests that mutations within a 12–base pair segment of the long control region that encompasses the E2 binding sites may influence the oncogenic potential of certain HPV strains. Methods Computational methods, including multiple sequence alignment, phylogenetic analysis, and protein structural modeling were employed to identify conserved regions and correlate these with potential cancer-associated mutations in the coding region. Results Structural modeling using AlphaFold3 and visualization in PyMOL revealed that conserved E2 residues cluster near the DNA-binding surface in the C-terminal domain and at critical interaction sites in the N-terminal transactivation domain, important for E1 DNA helicase binding and potentially other host factor interactions. Notably, species-specific adaptations, including the T309P substitution in the HPV52 subfamily B2, which may induce structural changes in the DNA-binding domain, and variations in the 12-base pair spacer, could modulate oncogene expression. Conclusions Collectively, these findings refine our understanding of E2’s essential role in viral pathogenesis and highlight promising targets for therapeutic intervention in high-risk HPV strains. Supplementary Information The online version contains supplementary material available at 10.1186/s12985-025-02903-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c91329ed72f3d8eb23cb138c9e1fce1ab89b380" target='_blank'>
              Structural and functional roles of conserved residues of human papillomavirus (HPV) E2 protein and biological consequences
              </a>
            </td>
          <td>
            Sean Fletcher, E. Biswas-Fiss, Subhasis B. Biswas
          </td>
          <td>2025-08-13</td>
          <td>Virology Journal</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de38d4909461a183185045ade884781ed7e9b99d" target='_blank'>
              Genome imbalance modulates the expression of long non-coding RNAs in maize.
              </a>
            </td>
          <td>
            Wenjie Yue, Chuang Yu, Hua Yang, Shiqi Lin, Shuyi Liu, James A. Birchler, Xiaowen Shi
          </td>
          <td>2025-09-01</td>
          <td>The Plant journal : for cell and molecular biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Both DNA methylation and homologous recombination (HR) are extensively studied. In bacteria, Dam methylation is the most studied DNA modification, while RecA-mediated HR is a primary mechanism to repair DNA damages including double-stranded breaks, single-stranded gaps, and stalled replication forks. While HR regulation by proteins is extensively studied, whether methylation of DNA itself directly affects the functions of RecA and HR remains unclear. Mainly by single-molecule experiments, we report that Dam methylation of single-stranded DNA (ssDNA) promotes RecA assembly, partially by reducing the effective charge of ssDNA under counterion screening. Furthermore, Dam methylation of double-stranded DNA promotes homologous pairing, joint molecule growth, and strand exchange. In cellular experiments, dam deletion impairs HR, whereas hypermethylation of the adenines in the genome enhances HR in P1 transduction assays and DNA-damage sensitivity tests without significantly upregulated HR-related genes. In addition, the preference of RecA for Dam-methylated DNA in RecA assembly and homologous pairing is conserved across divergent species covering a gram-negative bacterium Klebsiella pneumoniae, a gram-positive bacterium Bacillus subtilis, and a flowering plant Arabidopsis thaliana. Dam methylation of ssDNA increases the ATPase activity of molecular motors such as RecQ helicase that containing RecA-like domains. These findings reveal effects of DNA methylation and mechanisms regulating RecA-mediated HR and molecular motors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a366efaa4f71dbaa4fd5242b1fedea2c3fbc0149" target='_blank'>
              N6-methyladenine modification of DNA enhances RecA-mediated homologous recombination.
              </a>
            </td>
          <td>
            Xiaocong Zhao, Bin Wu, Ya-Jun Yang, Ying Li, Qi-Yuan Qiu, Liu Wang, Ya-Peng Xu, Han Gong, Lun Song, Xuejie Wang, Jing-Yao Shi, Xuefeng Chen, Shao-Ran Zhang, Qi Zong, Liang Dai, Shi-Shen Du, Yan Zhang, Wenqiang Wu, Xinghua Zhang
          </td>
          <td>2025-08-20</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The EZH2 histone methyltransferase inhibitors tazemetostat and valemetostat have recently received approval for clinical use in follicular lymphoma and adult T-cell leukemia/lymphoma, respectively. In follicular lymphoma, the expression of AID is responsible for increased mutational signatures and genomic instability. Because EZH2 inhibitors induce epigenetic and transcriptional changes in normal and lymphoma B cells, we studied whether these inhibitors could alter the pattern of AID-mediated chromosomal translocations. Here we show that treatment with EZH2 inhibitors did not significantly change AID expression or AID-dependent chromosomal translocation frequency when used as monotherapyin either CH12F3 mouse B cells or MEC-1 human B cells. In contrast, when combined with PI3Kδ inhibition, which enhances AID expression, EZH2 inhibition significantly increased the frequency of chromosomal translocations compared to either EZH2 or PI3Kδ inhibition alone both in mouse CH12F3 cells and human MEC-1 cells. EZH2 inhibition also further enhanced translocation formation in mouse B cells that were Ligase4 deficient. Mechanistically, EZH2 inhibition in B cells depletes the repressive histone modification H3K27me3 while concurrently enhancing the active histone modification H3K27ac, thereby selectively increasing transcriptional activity and facilitating chromosomal translocation formation in the presence of high AID activity or Ligase4 deficiency. These findings highlight the impact of drugs that induce epigenetic changes to influence chromosomal translocations and demonstrate the genetic safety of EZH2 inhibitors as monotherapy while highlighting the increased risk of genomic instability when used in cells prone to translocations, such as B cells with high AID levels or DNA-repair deficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7976efe4db0e0c962e5f359566e424689b7f04c0" target='_blank'>
              Epigenetic changes by EZH2 inhibition increase translocations in B cells with high AID activity or DNA repair deficiency.
              </a>
            </td>
          <td>
            Jianli Tao, Luca Alessandri, Alessandro Gasparetto, Lijuan Zhao, Xuefei Zhang, Frederick W Alt, Roberto Chiarle
          </td>
          <td>2025-08-18</td>
          <td>Blood</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Recent technological advances have facilitated the assembly of telomere-to-telomere (T2T) genomes. The current T2T CHM13 showcases the complete architecture of the human genome, yet its use in functional experiments is limited by discrepancies with the actual genome of the specific biological system under study. Access to reference assemblies for experimentally relevant cell lines is therefore essential in advancing sequencing-based analyses and precise manipulation, particularly in highly variable regions such as centromeres. Here, we present RPE1v1.1, the near-complete diploid genome assembly of the hTERT RPE-1 cell line, a non-cancerous human retinal epithelial model with a stable karyotype. Using high-coverage Pacific Biosciences and Oxford Nanopore Technologies long-read sequencing, we generate a high-quality de novo assembly, validate it through multiple methods, and phase it by integrating high-throughput chromosome conformation capture (Hi-C) data. Our assembly includes chromosome-level scaffolds that span centromeres for all chromosomes. Comparing both haplotypes with the CHM13 genome, we detect haplotype-specific genomic variations, including the translocation between chromosome 10 and chromosome X t(X;10)(Xq28;10q21.2) characteristic of RPE-1 cells, and divergence peaking at centromeres. Altogether, the RPE1v1.1 genome provides a reference-quality diploid assembly of a widely used cell line, supporting high-precision genetic and epigenetic studies in this model system.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cb559c0519b9dfa48eac6893a79e9c8d2446468" target='_blank'>
              The reference genome of the human diploid cell line RPE-1
              </a>
            </td>
          <td>
            Emilia Volpe, A. Colantoni, Luca Corda, Elena Di Tommaso, Franca Pelliccia, Riccardo Ottalevi, A. Guarracino, Danilo Licastro, Luigi Faino, M. Capulli, G. Formenti, Evelyne Tassone, Simona Giunta
          </td>
          <td>2025-09-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fa3ab0bece749f3dd05030992350c29b3fbf920" target='_blank'>
              Targeting transcription in neuroblastoma: focus on the Core regulatory circuit.
              </a>
            </td>
          <td>
            O. Kuchur, S. S. Pogodaeva, Victoria I Zhdankina, Anna V Scherbakova, Alexander A. Shtil, N. V. Antipova
          </td>
          <td>2025-08-11</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="Summary Adult women are typically exposed to estradiol (E2) concentrations of ~100–200 pM, yet most cell-based studies use 100 nM. We determined the molecular effects of E2 concentrations spanning six orders of magnitude (1 pM to 100 nM) in breast cancer cells. Estrogen receptor alpha (ERα) enhancers formed at low physiological doses of E2 (1–100 pM) are mechanistically distinct from those that form at high pharmacological doses (10–100 nM). They (1) form in open chromatin bound by FOXA1, (2) produce enhancer RNAs enriched with functional eRNA regulatory motifs (FERMs), and (3) drive expression of cell proliferation genes with promoter-proximal paused RNA polymerase II. Importantly, low dose ERα enhancer usage is elevated in breast cancer patients with poor responses to aromatase inhibitors, likely as a continued response to low circulating levels of E2. Collectively, our results identify mechanistic differences between low and high dose ERα enhancers that specify distinct biological outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99c754115c7eb96b7aba70ecc92a81903133740f" target='_blank'>
              Estrogen Receptor Enhancers Sensitive to Low Doses of Hormone Specify Distinct Molecular and Biological Outcomes
              </a>
            </td>
          <td>
            Hyung Bum Kim, T. Nandu, C. Camacho, W. L. Kraus
          </td>
          <td>2025-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Esophageal squamous cell carcinoma (ESCC) is a highly aggressive malignancy with poor prognosis and limited treatment options. While circular RNAs (circRNAs) are frequently dysregulated in ESCC, their functional roles and molecular mechanisms in tumor progression remain largely unexplored. Methods We characterized circRNA using RT-PCR, Sanger sequencing, and fluorescent in situ hybridization. Gain- and loss-of-function studies in vitro and in vivo were performed to assess circRNA function. Molecular interactions were investigated via RNA pull-down assays coupled with mass spectrometry, electrophoretic mobility shift assays, co-immunoprecipitation, and immunoblot analysis. Clinical relevance was evaluated by RT-qPCR and immunohistochemistry in patient specimens. Results We identified circLNF, a novel circRNA derived from the long non-coding RNA FIRRE gene, which is significantly upregulated in ESCC. Functional assays demonstrated that circLNF promotes proliferative and migratory capacities in cultured cells and accelerates tumor progression and metastasis in animal models. Mechanistically, circLNF directly interacts with IGF2BP family proteins through their “CAUC” motifs, protecting them from TRIM25-mediated ubiquitination and proteasomal degradation. Importantly, circLNF expression positively correlated with IGF2BP1 protein levels in ESCC patient tissues, underscoring the clinical relevance of the circLNF-IGF2BP axis in ESCC progression. Conclusions Our findings reveal an oncogenic role of circLNF in ESCC by inhibiting TRIM25-mediated proteasomal degradation of IGF2BP proteins and highlight circLNF as a potential therapeutic target for ESCC. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02442-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fde46bb0cd6a0e4ea2cb6ea52b1582df490fa428" target='_blank'>
              A circRNA promotes esophageal squamous cell carcinoma progression by inhibiting TRIM25-mediated degradation of IGF2BP family members
              </a>
            </td>
          <td>
            Shuangyan Tan, Yue Ming, Jiawei Guo, Wenrong Liu, Hulin Ma, Yu Liu, Zhijie Xu, A-Lai Gu-Ha, Lunzhi Dai, , Yong Peng
          </td>
          <td>2025-10-03</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35317e7a4cee78e23641ba79f278922203278977" target='_blank'>
              A chemical-genetic interaction between PAF1 and ENL/AF9 YEATS inhibition.
              </a>
            </td>
          <td>
            Paige A. Barta, Leopold Garnar-Wortzel, T. Bishop, Rachel E Hayward, Lauren M. Hargis, James B. Shaum, H. Kwok, Brian B Liau, Benjamin F. Cravatt, Michael A. Erb
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c38bcffc5c385d9f9befa33ae4e8c24ff19f54" target='_blank'>
              Synthesis of Oligonucleotides Containing a Dna Distorting Interstrand Crosslink Produced by Mitomycins.
              </a>
            </td>
          <td>
            Sarah Marks, Gabriel Martinez, Michael Adamov, Christopher Alley, Bianca Davila, Ana G Petrovic, R. Abzalimov, Padmanava Pradhan, Marta Concheiro-Guisan, Elise Champeil
          </td>
          <td>2025-08-14</td>
          <td>Chembiochem : a European journal of chemical biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="N6-Methyladenosine (m6A) is one of the most abundant RNA modifications that occur in eukaryotes. The relationship between m6A methylation and DNA damage repair (DDR) is still unclear. As an important chaperone protein of m6A methyltransferase, the role of Wilm’s tumor–associated protein (WTAP) in DDR has not been studied in detail. We identified that WTAP was markedly upregulated in HCC cells with DNA damage induction. Further investigations revealed that WTAP was engaged in DDR and WTAP knockdown resulted in a significant decrease of the stability of the DDR-related transcription factor Forkhead Box M1 (FOXM1) mRNA, eventually preventing DDR. WTAP deficiency inhibited cell growth both in vivo and in vitro. Moreover, WTAP silencing sensitized HCC cells to cisplatin, a well-known DNA damage agent. Altogether, our findings demonstrate that WTAP is the dominant m6A -related modulator upon DNA damage and functions by stabilizing FOXM1. In addition, WTAP deficiency saliently sensitized HCC cells to cisplatin both in vitro and in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32fcd24463d28078ba4ecd4a46e0acbc5ea89720" target='_blank'>
              WTAP participates in the DNA damage response via an m6A-FOXM1-dependent manner in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Nan Huang, Z. Bian, Chang Xu, Yue Zhang, Li Liu, Zhongqi Cui, Shasha Zhao, Qiangyuan Fan, Shaobo Xue, Yan Chen, Qiuhui Pan, F. Sun
          </td>
          <td>2025-08-22</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2ab9d9615fda47177afd65cb9d942ab1bd6dc87" target='_blank'>
              Thymic epithelial cells amplify epigenetic noise to promote immune tolerance.
              </a>
            </td>
          <td>
            Noah Gamble, Jason A Caldwell, Joshua McKeever, Caroline Kaiser, Alexandra Bradu, Peyton J. Dooley, Sandy Klemm, William J. Greenleaf, Narutoshi Hibino, Aaron R. Dinner, Andrew S. Koh
          </td>
          <td>2025-08-20</td>
          <td>Nature</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor drug resistance remains a significant barrier to effective cancer treatment, undermining the clinical efficacy of current therapies through adaptive and evolving mechanisms. Cuproptosis, a recently discovered form of programmed cell death, emerges as a promising tumor suppressor by targeting mitochondrial metabolic pathways, offering a novel strategy to combat drug resistance. However, targeting mitochondrial metabolism presents challenges due to the essential physiological roles of mitochondria in normal tissues, raising concerns about potential on-target toxicity. Recent studies have revealed the dynamic role of non-coding RNAs (ncRNAs) in modulating susceptibility to cuproptosis via epigenetic alterations, potentially serving as key factors in therapeutic response. In this review, we first summarize the complex interplay between chemotherapeutic resistance and cancer progression/relapse. Then, we outline the key molecular mechanisms underlying cuproptosis, its upstream epigenetic regulators, and the downstream effects on drug resistance. Finally, this review discusses the latest findings on how ncRNAs influence resistance to chemotherapy, immunotherapy, and targeted therapy by modulating cuproptosis. This review aims to untangle the regulatory mechanisms of ncRNA-mediated epigenetic modifications on cuproptosis, highlighting the critical role of the ncRNA-cuproptosis interaction network in tumor drug resistance progression. These insights provide a conceptual foundation for developing ncRNA-targeted therapies in conjunction with cuproptosis-inducing approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/368bafe9b516f67cfc94213e12ae103aa111fe6a" target='_blank'>
              Epigenetic modification of cuproptosis by non-coding RNAs in cancer drug resistance
              </a>
            </td>
          <td>
            Fei Du, Lu Tang, Fang Wang, Xian Jiang, Junxin Li, Guojun Wang, Zhuo Zhang
          </td>
          <td>2025-09-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Colorectal cancer is one of the most prevalent malignant tumors worldwide, and it has one of the highest rates of mortality and morbidity. Its emergence and progression are intricately linked to various genetic and epigenetic factors influencing colonic epithelial cells. Long non-coding RNAs function as crucial regulators of the epigenetic landscape, playing significant roles in the development, progression, invasion, metastasis, and drug resistance of various tumors, particularly colorectal cancer. Epigenetic modifications, such as DNA methylation, RNA methylation, and histone modification, facilitate heritable gene expression regulation without changing the underlying DNA sequence. Recent studies have increasingly highlighted the intricate interactions between lncRNAs and these epigenetic modifications in colorectal cancer, illuminating their participation in critical processes, including cell proliferation, apoptosis, invasion, and metastasis. This review concentrates on the relationship between long non-coding RNAs and several significant epigenetic modifications that drive colorectal cancer development, offering fresh insights for future research that aims at devising more effective treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31b080b73592dba79e60d81f2a7fb3048fe78b62" target='_blank'>
              Long non-coding RNAs promote colorectal cancer development through other epigenetic modifications
              </a>
            </td>
          <td>
            Guojuan Wang, Qingqing Ma, Naicheng Zhu, Yuwei Yan, Chen Zhong, Xinlin Hong, Yujuan Lai, Xiudan Chen, Nanxin Li, Wenyan Yu
          </td>
          <td>2025-09-28</td>
          <td>Gastroenterology Report</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 In contrast to prevailing wisdom that cancer is a genetic disease that evolves via Darwinian selection, it is increasingly recognized that in addition to genetic mutations, intratumoral heterogeneity also results through non-genetic mechanisms. Furthermore, it is now evident that the malignant phenotype results from complex interactions between genetic and non-genetic mechanisms. These interactions contribute to functional changes across multiple spatiotemporal scales, from molecular to cellular to tissue level, creating a heterogeneous cancer cell population. Cancer Systems Biology is a comprehensive, up-to-date treatise that addresses these issues. The main intent of this compendium is to encourage cross-pollination of ideas from experts in multiple disciplines including basic and translational science researchers, systems/mathematical biologists, and biophysicists and clinicians interested in decoding a systems-level emergent view of cancer so that a deeper understanding of disease progression can be gained and more effective treatment strategies can be designed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a03788cb0b91aa174c4a7535dcca74621b417df9" target='_blank'>
              Cancer Systems Biology
              </a>
            </td>
          <td>
            Ravi Salgia, M. Jolly, Prakash Kulkarni, Govindan Rangarajanm
          </td>
          <td>2025-09-01</td>
          <td>None</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="In this study, we delineated the distinct transcriptomic effects of VHL mutation and chromosome 3p (chr3p) loss, revealing that chr3p loss is specifically associated with immune suppression in clear cell renal cell carcinoma (ccRCC). Furthermore, we developed driver genomic aberration (DGA) gene signatures that demonstrate superior performance in predicting both patient prognosis and treatment response compared to traditional mutation-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8edc71bb1913e06b7ed3c5e92c20979a6f59e6" target='_blank'>
              Unravelling the distinct effects of VHL mutations and chromosome 3p loss in clear cell renal cell carcinoma: Implications for prognosis and treatment
              </a>
            </td>
          <td>
            Xiang Wang, Jian-Rong Li, N. Chowdhury, Lang Wu, Cheng Chao
          </td>
          <td>2025-09-25</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) involves the expansion of hematopoietic stem cells with ageacquired mutations linked to myeloid malignancy. Advances in next-generation and single-cell sequencing, along with computational modeling, have expanded our ability to detect both common and rare CH drivers, including single-nucleotide variants and mosaic chromosomal alterations, with increasing sensitivity. While sequencing methods differ in accuracy, cost, and ability to detect low-frequency variants, they have deepened our understanding of CH biology. A growing body of evidence has identified both somatic drivers, such as variants in DNMT3A, TET2, and ASXL1, and germline genetic variants that modify CH risk, highlighting the complex interplay between inherited and acquired factors. These collective discoveries are guiding the development of targeted therapies and interventions, particularly for individuals at risk of progression to myeloid neoplasms or cardiovascular disease. Additionally, CH is emerging as a clinically relevant factor in the treatment of solid tumors, where it may influence the tumour microenvironment, treatment response and the risk of therapy-related complications. Risk stratification models are facilitating earlier identification and monitoring of high-risk individuals, enabling personalized treatment decisions. The scope of CH management continues to expand, from surveillance to intervention, with ongoing trials testing preventive strategies in high-risk populations. Emerging trial frameworks emphasize risk stratification, age-appropriateness, inclusive recruitment, decentralized trial models, and the use of traditional clinical and novel endpoints. Together, these advances reflect a shift from passive observation to proactive intervention, charting a course for early detection, precision treatment, and prevention in CH care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6faa3f8b3f8e8d6ae046148a99055b41b7b424" target='_blank'>
              CHIP ahoy: charting a decade of discovery in clonal hematopoiesis.
              </a>
            </td>
          <td>
            Casey K. Wong, Alexandra McDonald, M. Buttigieg, M. Rauh
          </td>
          <td>2025-09-04</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="


 Essential epigenetic dependencies have become evident in many cancers. Based on functional antagonism between BAF/SWI/SNF and PRC2 in SMARCB1-deficient sarcomas, we and colleagues recently completed the clinical trial of the EZH2 inhibitor tazemetostat, leading to its FDA approval. However, the principles of tumor response to epigenetic therapy in general, and tazemetostat in particular, remain unknown. First, using functional genomics of patient tumors and diverse experimental models, we define molecular mechanisms of tazemetostat resistance in SMARCB1-deficient sarcomas and rhabdoid tumors. We found distinct classes of acquired mutations that converge on the RB1/E2F axis and decouple EZH2-dependent differentiation and cell cycle control. This allows tumor cells to escape tazemetostat-induced G1 arrest despite EZH2 inhibition, and suggests a general mechanism for effective EZH2 therapy. We provide prospective biomarkers for therapy stratification, including the cell cycle-regulated PRICKLE1 gene. To circumvent tazemetostat resistance and improve response in vitro and in vivo we develop the rational combination of tazemetostat with barasertib, to target the downstream cell cycle kinase AURKB. Second, through transcriptomic inference of SMARCB1-deficient tumor cells, we predicted ATR as a target to overcome resistance to EZH2 inhibition. We found that EZH2 inhibition induces DNA damage, at least in part via upregulation of the transposase-derived PGBD5 nuclease. Finally, we establish a second rational combination strategy for increased anti-tumor efficacy between tazemetostat and the ATR inhibitor, elimusertib, by leveraging the synthetic lethal interaction with PGBD5-dependent DNA damage repair signaling via ATR. Thus, the epigenetic therapy, EZH2 inhibition, induces a targetable collateral vulnerability through activation of the PGBD5-ATR axis. This establishes a mechanistically rational combination strategy for targeting EZH2 and ATR in SMARCB1-deficient cancers based on an EZH2-PGBD5-ATR synthetic lethal axis and is immediately translatable to clinical trials for patients with these aggressive malignancies.



 Helen S Mueller. Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e20005fa9b7768828704bae3a839702d20b1fe7d" target='_blank'>
              Abstract A048: Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy
              </a>
            </td>
          <td>
            Helen S. Mueller
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [12],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>