<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-08-25 09:30:05 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>37</td>
          <td>104</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>77</td>
          <td>138</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>114</td>
          <td>145</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>349</td>
          <td>138</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>307</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a317a5c4269a3d4d0174cf78851658866c2ba8" target='_blank'>
              Extrachromosomal DNA-Driven Oncogene Dosage Heterogeneity Promotes Rapid Adaptation to Therapy in MYCN-Amplified Cancers.
              </a>
            </td>
          <td>
            Giulia Montuori, Fengyu Tu, Di Qin, R. Schmargon, Elias Rodriguez-Fos, K. Helmsauer, Hui Hui, Susmita Mandal, Karin Purshouse, Lara Fankhänel, Bartolomeo Bosco, B. Spanjaard, H. Seyboldt, Laura Grunewald, M. Schmitt, D. Gürgen, Viktoria Buck, M. Rosenfeldt, Frank P B Dubois, S. Schallenberg, Annika Lehmann, J. Theißen, S. Taschner-Mandl, Arend Koch, P. Hundsdoerfer, A. Künkele, Angelika Eggert, Matthias Fischer, G. Gargiulo, Teresa G Krieger, Lukas Chavez, Fabian Coscia, Benjamin Werner, Weini Huang, A. Henssen, J. Dörr
          </td>
          <td>2025-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Circular extrachromosomal DNA (ecDNA) is a common form of oncogene amplification in aggressive cancers. The frequency and diversity of ecDNA has been catalogued in adult and some childhood cancers; however, its role in most pediatric cancers is not well-understood. To address this gap, we accessed large pediatric cancer genomics data repositories and identified ecDNA algorithmically using cloud computing. This retrospective cohort comprises 2,968 patients covering all major childhood solid tumor types. Aggressive tumor types had particularly high incidences of ecDNA. Pediatric patients whose tumors harbored extrachromosomal DNA had poorer five-year overall survival than children whose tumors contain only chromosomal amplifications. We catalogue known and potentially novel oncogenes recurrently amplified on ecDNA and show that ecDNA often evolves during disease progression. These results highlight patient populations that could potentially benefit from future ecDNA-directed therapies. To facilitate discovery, an interactive catalogue of ecDNA in childhood cancer is available at https://ccdi-ecdna.org/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8b9576b9fdef51f927edfe8dbc69e8e7f1f4c7f" target='_blank'>
              Extrachromosomal DNA associates with poor survival across a broad spectrum of childhood solid tumors
              </a>
            </td>
          <td>
            Owen S. Chapman, S. Sridhar, Eugene Yui-Ching Chow, Rishaan Kenkre, Jonathan Kirkland, Aditi Dutta, Shanqing Wang, Wenshu Zhang, Miguel Brown, J. Luebeck, Yan Yuen Lo, Elias Rodriguez-Fos, A. Henssen, K. Okonechnikov, David R. Ghasemi, K. Pajtler, Daisuke Kawauchi, V. Bafna, Megan Paul, K. Yip, Jill P. Mesirov, Lukas Chavez
          </td>
          <td>2025-07-23</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) is a critical driver of cancer progression, contributing to tumour growth, evolution, and therapeutic resistance through oncogene amplification. Despite its significance, the replication of ecDNA remains poorly understood. In this study, we investigated the replication dynamics of ecDNA using high-resolution replication timing analysis (Repli-seq) and DNAscent, a method for measuring origin firing and replication fork movement, that we applied to both bulk DNA and to ecDNA isolated with FINE (Fluorescence-activated cell sorting-based Isolation of Native ecDNA), a new method for isolating, chromatinized ecDNA without DNA or protein digestion. We demonstrate that ecDNA in the COLO 320DM colorectal cancer cell line exhibits largely asynchronous replication throughout the S phase, contrasting with the conserved replication timing of the corresponding chromosomal DNA in RPE-1 cells and the chromosomally reintegrated ecDNA in COLO 320HSR. Replication origins on ecDNA are redistributed, and replication forks exhibit reduced velocity and increased stalling. Under replication stress induced by hydroxyurea treatment, ecDNA replication is further compromised, leading to altered origin activation, reduced fork velocity and eventual ecDNA depletion from cells. Our findings reveal fundamental differences in the replication dynamics of ecDNA, providing insights that could inform the development of therapies targeting ecDNA-associated oncogene amplification in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec71fc330fef12833948b0057870ba85688ca5b1" target='_blank'>
              ecDNA replication is disorganized and vulnerable to replication stress
              </a>
            </td>
          <td>
            Jedrzej J Jaworski, Pauline L Pfuderer, Pawel Czyz, Gianluca Petris, M. Boemo, Julian E. Sale
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Oncogene amplification is a key driver of cancer pathogenesis and is often mediated by extrachromosomal DNA (ecDNA). EcDNA amplifications are associated with increased pathogenicity of cancer and poorer outcomes for patients. EcDNA can be detected accurately using fluorescence in situ hybridization (FISH) when cells are arrested in metaphase. However, the majority of cancer cells are non-mitotic and must be analyzed in interphase, where it is difficult to discern extrachromosomal amplifications from chromosomal amplifications. Thus, there is a need for methods that accurately predict oncogene amplification status from interphase cells. Here, we present interSeg, a deep learning-based tool to cytogenetically determine the amplification status as EC-amp, HSR-amp, or not amplified from interphase FISH images. We trained and validated interSeg on 652 images (40,446 nuclei). Tests on 215 cultured cell and tissue model images (9,733 nuclei) showed 89% and 97% accuracy at the nuclear and sample levels, respectively. The neuroblastoma patient tissue hold-out set (67 samples and 1,937 nuclei) also revealed 97% accuracy at the sample level in detecting the presence of focal amplification. In experimentally and computationally mixed images, interSeg accurately predicted the level of heterogeneity. The results showcase interSeg as an important method for analyzing oncogene amplifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a539ca1a35b8049ad46a136d79392bb325111564" target='_blank'>
              Accurate Prediction of ecDNA in Interphase Cancer Cells using Deep Neural Networks
              </a>
            </td>
          <td>
            Utkrisht Rajkumar, Gino Prasad, Ellis J Curtis, I. Wong, Xiaowei Yan, Shu Zhang, Lotte Brückner, Kristen Turner, Julie Wiese, Justin Wahl, Sihan Wu, J. Theißen, Matthias Fischer, Howard Y. Chang, A. Henssen, P. Mischel, V. Bafna
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNAs (ecDNAs), which replicate and segregate in a non-Mendelian manner, serve as vectors for accelerated tumor evolution. By integrating chromatin accessibility, whole-genome sequencing, and Hi-C-based genome topology data from a cohort of metastatic Castration-Resistant Prostate Cancer (mCRPC) cases, we show that epigenetically activated repeat DNA, amplified in ecDNAs, drive oncogene overexpression. Specifically, we identify a subgroup of mCRPCs (20%) characterized by clusters of accessible LINE1 repeat DNA elements flanking the androgen receptor (AR) gene. These LINE1 elements are co-amplified with AR and provide binding sites for prostate-lineage transcription factors, including AR, FOXA1 and HOXB13. Accessible LINE1 elements establish novel 3D chromatin interactions with the AR gene, forging a new regulatory plexus driving AR overexpression and confers resistance to androgen signaling inhibitors. Our findings indicate how tumor evolution is driven by the convergence of genetic and epigenetic alterations on repeat DNA, activating and amplifying them to allow oncogene overexpression. Statement of significance We show how tumor evolution is driven by the convergence of genetic and epigenetic alterations on repeat DNA elements, resulting in their activation as regulatory elements and co-amplification in ecDNAs with oncogenes in mCRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6c5dce2ec927f5cff069554fa04e5223fd82a5" target='_blank'>
              Genetic and Epigenetic Reprogramming of Transposable Elements Drives ecDNA-Mediated Metastatic Prostate Cancer
              </a>
            </td>
          <td>
            Lisanne Mout, Thaidy Moreno-Rodriguez, Giacomo Grillo, Ankita Nand, Tina Keshavarzian, S. Bahl, Komaldeep Kang, M. Bootsma, Emma Minnee, Stanley Zhou, Kathleen H. Burns, Eva Corey, Peter Nelson, S. Dehm, , W. Zwart, Felix Feng, David Quigley, M. Lupien
          </td>
          <td>2025-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Abstract Large chromosomal copy number gains are ubiquitous throughout cancer types. However, which genes drive their selective advantage is not well established, and therefore they are hardly utilized in clinical practice. Our analysis of copy number patterns in pan-cancer datasets suggests that the selective advantage of copy number gains is largely driven by known oncogenes. Analysis of CRISPR screening data identifies a list of 101 genes that are likely to mediate the effect of these gains, which is highly enriched in annotated oncogenes but also contains genes that have not been implicated in cancer so far. Moreover, we show that specific gains are associated with drug sensitivity or resistance, with a strong enrichment of gains of oncogenes with increased sensitivity to inhibitors targeting these specific genes. Finally, we provide examples where gains can function as relevant clinical biomarkers for diagnosis and treatment. Thus, large copy number gains exert their selective advantage through known and novel oncogenes, and their systematic analysis could advance precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecd5fd07460798ed6e6a37e20560052b55679a6c" target='_blank'>
              Low-amplitude copy number gains shape cancer through known and novel oncogenes with associated therapeutic vulnerabilities
              </a>
            </td>
          <td>
            T. F. Eleveld, B. Ylstra, Leendert H J Looijenga
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) of chromosomal origin is commonly present in all eukaryotic organisms and tissue tested so far. EccDNA populations exhibit immense diversity and a characteristically low degree of overlap between samples, suggesting low inherence of eccDNA between cells or a deficiency the methods by which eccDNA is detected. This study revisits the Circle-seq approach for enrichment of eccDNA to address if these limitations, hypothesizing that experimental procedures significantly contribute to the observed low eccDNA overlap. We optimized the protocol by reducing the time. Linear DNA is digested by increasing exonuclease V activity. We employed CRISPR-Cas9 for mitochondrial linearization, which proved superior to restriction enzymes. A key finding is the critical role of random hexamer primer concentration and genomic DNA input in Rolling Circle Amplification (RCA) for generating high-quality long amplicons from eccDNA (concatemeric tandem copy, CTC), essential for confident de novo eccDNA construction from long-read sequencing data. Lower primer concentrations substantially increased the percentage of CTC-derived eccDNA and improved the overlap of identified eccDNAs in technical replicates. Applying this revisited approach to human myeloma and breast cancer cell lines, as well as xenograft models, demonstrated that the optimized conditions enhanced the overlap of detected eccDNA up to over 50% overlap which substantially improved over previous studies (less than 1%). Moreover, Additionally, the oncogenic signature of eccDNAs can be identified across all replicates. These findings provide guidelines for developing standardized procedures for eccDNA profiling, advancing our understanding of eccDNA biology and its potential clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/842e0482ffc6f8bbb0e4561726b21f49e5e47e0b" target='_blank'>
              An Enhancement of Extrachromosomal Circular DNA Enrichment and Amplification to Address the Extremely Low Overlap Between Replicates
              </a>
            </td>
          <td>
            Charles M. Burnham, Alongkorn Kurilung, Visanu Wanchai, Birgritte Regenberg, J. Delgado-Calle, Alexei G. Basnakian, I. Nookaew
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The spatial organization of the genome within the nucleus is a fundamental regulator of gene expression, genome stability, and cell identity. This review addresses the central question of how nuclear genome architecture contributes to disease mechanisms and diagnostics, and how technological advances enable its clinical exploration. We first outline the principles of nuclear genome architecture, including chromosome territories, replication timing, and 3D domains, and their role in gene regulation and disease. We then explore the mechanisms and consequences of chromosomal rearrangements, and how replication dynamics intersect with epigenetic regulation and genome stability. Diagnostic tools are presented in chronological progression, from conventional cytogenetics to high-resolution genomic and single-cell techniques. A dedicated section focuses on cancer cytogenomics and its clinical implications. We further highlight emerging technologies for 3D genome and epigenome profiling and their integration into diagnostic workflows. Finally, we discuss current challenges, such as standardization and cost, and the transformative potential of multi-omics and artificial intelligence for future precision diagnostics. Overall, we provide a comprehensive overview of how cytogenetics and cytogenomics contribute to the understanding and clinical diagnosis of genetic and neoplastic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0253c2480a5ecc52bcd5c1eda31a7ec8e922ab1" target='_blank'>
              Cytogenetics and Cytogenomics in Clinical Diagnostics: Genome Architecture, Structural Variants, and Translational Applications
              </a>
            </td>
          <td>
            Concetta Federico, Desiree Brancato, Francesca Bruno, Elvira Coniglio, V. Sturiale, Salvatore Saccone
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background Increases in the copy number of large genomic regions, termed amplifications, are an important adaptive strategy for many organisms. Numerous amplifications across the AT-rich Plasmodium falciparum genome contribute directly to drug resistance or impact the fitness of this protozoan parasite. During the characterization of malaria parasites selected with a dihydroorotate dehydrogenase (DHODH) inhibitor that targets pyrimidine biosynthesis, we detected increased copies of a genomic region that encompassed 3 genes (~ 5 kb) including GTP cyclohydrolase I (GCH1 amplicon). While amplification of this gene is reported in antifolate-resistant parasites, GCH1 amplicons had not previously been implicated in DHODH inhibitor resistance. Results Here, we explored the expansion of the GCH1 locus in this family of parasite lines using long-read sequencing and single-read visualization. We directly quantified higher numbers of tandem GCH1 amplicons in selected parasite lines (up to 9 GCH1 amplicons) compared to parental P. falciparum parasites (strictly 3 GCH1 amplicons). Because each read represents DNA from an individual genome, we were able to appreciate hidden variation within a single parasite line (3, to 5, to 7 amplicons) that was not reflected in other DNA-based analysis methods. While all GCH1 amplicons shared a consistent structure, expansions arose in precise 2-unit steps within selected lines. We found conserved AT-rich sequences at amplicon boundaries, which is consistent with the Plasmodium model of CNV formation. Parasite lines with expanded GCH1 also had DHODH amplicons on a separate chromosome. When we evaluated prior DHODH inhibitor selections, we observed that GCH1 amplification was not required for resistance; however, selection outcomes suggest that pre-existing GCH1 amplicons may support amplification at the DHODH locus. Conclusions We identified previously undetected heterogeneity in gene copy number by viewing long pieces of DNA from individual genomes. This approach was possible due to the amplicon’s tandem orientation and relatively small size that can be spanned by a single long ONT read. The positive association between DHODH and GCH1 copy number, combined with the metabolic connection between P. falciparum pyrimidine and folate biosynthesis, justifies further investigation into the adaptive evolution of these two genomic loci. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11859-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df91538345e9b090469612002190dc6883763df6" target='_blank'>
              Direct long-read visualization reveals hidden variation in GCH1 gene copy number and precise expansion steps
              </a>
            </td>
          <td>
            Shiwei Liu, Julia Zulawinska, Emily R. Ebel, A. Luniewski, Charles Danis, Mary Lewis Simpson, Jane Kim, Nnenna Ene, Thomas Braukmann, Molly Congdon, Webster Santos, Ellen Yeh, J. Guler
          </td>
          <td>2025-07-17</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acb47b6253f4e8e1db726d8195373534a90d452c" target='_blank'>
              Selective depletion of cancer cells with extrachromosomal DNA via lentiviral infection.
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, Emmanuel E Korsah, Noah A Dusseau, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-08-11</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69d250508121e21820a44d64995e2f038263f7bb" target='_blank'>
              Clone copy number diversity is linked to survival in lung cancer.
              </a>
            </td>
          <td>
            Piotr Pawlik, Kristiana Grigoriadis, Abigail Bunkum, Helena Coggan, A. Frankell, Carlos Martínez-Ruiz, Takahiro Karasaki, A. Huebner, Andrew Rowan, Jasmin Fisher, A. Hackshaw, C. Swanton, S. Zaccaria, N. Mcgranahan
          </td>
          <td>2025-08-13</td>
          <td>Nature</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="


 In tumors, clonal populations of malignant cells evolve a diverse repertoire of copy number variants (CNVs), often resulting in profound intra-tumor heterogeneity (ITH). ITH is associated with poor prognosis, cancer progression, increased risk of therapy resistance, metastasis and relapse. Traditionally, researchers have studied clonal heterogeneity and tumor evolution using bulk-tissue assays, however, these techniques provide little or no spatial information and have low sensitivity for focal/low copy-number CNVs, particularly when they are subclonal or rare. DNA FISH is the gold standard for detecting CNVs at cellular resolution, but it is limited to one gene per tissue section. To overcome the above limitations, we developed multiplexed-CNV FISH, a spatial omics assay that detects gain and loss of DNA loci at single-gene (∼10kb) and single-cell resolution in human tissues. Remarkably, CNV-FISH analysis of CRC tumor sections identified small and subclonal CNVs not detected by existing methods with resolution of CNV spanning whole chromosome arms. Our results indicate that CNV-FISH can reveal unexpected and previously undetected CN heterogeneity within tumors, which could potentially underlie treatment resistance and eventual metastasis of cancer cells.



 Mei Suen KONG, Hui Ting Grace YEO, Jiamin TOH, Lin Li, Jonathan Shaozhong Aow, Joanito Irwan IGNASIUS, Jeeranan BOONRUANGKAN, Jen Yi WONG, Nirmala ARUL RAYAN, Lavanya MUTHUKRISHNAN, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon LIM, Bee Huat Iain TAN, Kok Hao CHEN, Shyam PRABHAKAR. Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P06.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae7336ba20d4eb282822f6335d4e3b64784f2e7c" target='_blank'>
              Abstract P06: Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer
              </a>
            </td>
          <td>
            Mei Suen KONG, Hui Ting Grace YEO, Jiamin Toh, Lin Li, Jonathan Aow, Joanito Irwan IGNASIUS, Jeeranan Boonruangkan, Jen Yi WONG, N. A. Rayan, Lavanya Muthukrishnan, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon Lim, Bee Huat Iain TAN, Kok Hao CHEN, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT Tumors of infectious origin globally represent 13%. Oncogenic DNA viruses such as human papillomavirus (HPV), hepatitis B virus (HBV), and Epstein-Barr virus (EBV) are responsible for approximately 60% of these tumors. These oncoviruses are extensively studied to understand their role in cancer development, particularly through viral genome integration into the host DNA. Retroviruses require integration mediated by viral integrase for persistence, whereas DNA oncoviruses do not need integration for replication; instead, integration occurs incidentally. This process often targets fragile sites in the human genome, causing structural rearrangements that disrupt genes, activate proto-oncogenes, and increase genomic instability, all contributing to tumorigenesis. Integration near promoter regions and active genes is closely linked to carcinogenesis, highlighting its importance in developing diagnostic and therapeutic strategies. This review summarizes viral integration’s role in oncogenesis, mechanisms of integration, and methods to study this process, focusing on DNA tumor viruses such as HBV, EBV, HPV, and Merkel cell polyomavirus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ef1eb5cd84b08745894e948edf6d9875f648ea7" target='_blank'>
              Genome integration of human DNA oncoviruses
              </a>
            </td>
          <td>
            Zuzana Vojtěchová, R. Tachezy
          </td>
          <td>2025-07-23</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2a0c8aae8bd565763175795d01ffdf375af883" target='_blank'>
              Ongoing genome doubling shapes evolvability and immunity in ovarian cancer.
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Duaa H Al-Rawi, Matthew Zatzman, A. Weiner, Samuel Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nicholas Ceglia, Danguolė Norkūnaitė, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, Ginger J Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, R. Grisham, Ying L Liu, A. Viale, Nicole Rusk, Yulia Lakhman, Lora H. Ellenson, S. Tavaré, Sam Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2025-07-16</td>
          <td>Nature</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e0ce2d3aba4ba6a29f1b308846500c4a3d7ec7" target='_blank'>
              Beyond inversions and deletions: the evolutionary and functional insights from translocations, fissions, and fusions in animal genomes.
              </a>
            </td>
          <td>
            Célian Diblasi, M. Saitou
          </td>
          <td>2025-08-01</td>
          <td>Heredity</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chromosome-derived circular DNAs have different characters from linear DNAs and RNAs, which exert distinctive functions, and participate in a widespread range of physiological and pathological processes. Although, the composition, structure and genome-wide frequency of circular DNAs have not yet been illuminated systemically, circular DNAs are identified to act as unanticipated major sources of somatic rearrangements, which is an important genomic feature in tumorigenesis and represents a multihit and ongoing mutagenic process. And, this process remarkably contributes to oncogenic remodeling and oncogenes amplification through chimeric circularization and reintegration of circular DNAs into the linear genome, which is closely associated with high cancer mortality and morbidity. It has been found that circular DNAs-stimulated oncogenes amplification and overexpression implicate in diverse cytopathological processes during cancer progression, including cell proliferation, apoptosis, autophagy, epithelial-mesenchymal transition (EMT), extracellular matrix (ECM) stiffness and angiogenesis. Herein, we comprehensively introduced circular DNAs’ formation and characteristics, and emphatically elaborated circular DNAs-induced oncogenes amplification and overexpression and their functional mechanisms regulating cancer behaviors. Lastly, some mysteries in oncogenic circular DNAs’ study were introduced, and the perspectives were also discussed in cancer diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/825b11b400d7213811736a403636f24a5a529f78" target='_blank'>
              Circular DNAs Participate in Tumorigenesis and Its Mechanisms
              </a>
            </td>
          <td>
            Yuan Tan, Faqing Tang
          </td>
          <td>2025-07-30</td>
          <td>Clinical Medicine and Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9eeecebde718f44128a5af1add184f4e02b1d49" target='_blank'>
              NuMA promotes constitutive heterochromatin compaction by stabilizing linker histone H1 on chromatin
              </a>
            </td>
          <td>
            Yao Wang, Wenxue Zhao, Jiahao Niu, Cuifang Liu, Xiaotian Wang, Weihong Yuan, Shanshan Ai, Wolfgang Baumeister, Guohong Li, Aibin He, Peng Xu, Cheng Li, Yujie Sun
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains a lethal malignancy with poor prognosis. We investigated intratumoral deoxyribonucleic acid methylation heterogeneity by analyzing 44 tumor samples and 5 normal samples from 6 cases of PDAC by using high-resolution methylation arrays. Two distinct methylation profiles were identified: T1, which is similar to normal pancreatic tissue and is associated with well-differentiated histology, and T2, which is significantly different from normal tissue and is linked to poorly differentiated morphology and squamous features. Validation using The Cancer Genome Atlas (TCGA) confirmed these profiles and revealed the association of T2 with shorter disease-free survival (p = 0.04). Differentially methylated region analysis identified the substantial hypomethylation of transcription regulation genes in T2 profiles (false discovery rate [FDR] q < 0.001). Gene set enrichment analysis with TCGA gene expression data demonstrated the upregulation of DNA repair and MYC target genes in T2 samples (FDR q < 0.001). Phylogenetic analysis with our multi-sampling dataset suggested an evolutionary trajectory from T1 to T2 profiles coinciding with aggressive phenotypes and increased genomic instability. Cases exhibited varying degrees of intratumoral heterogeneity from distinctly separated clusters to minimal differences. This comprehensive characterization of the epigenetic landscape of PDAC provides insights into tumor evolution and heterogeneity with potential implications for patient stratification and the development of epigenetic-based diagnostic and therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-13919-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd12eab03975f1f1da59d2699d262a5bc0933404" target='_blank'>
              Epigenetic evolution and clinicopathological implications of distinct DNA methylation profiles in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            K. Kitahama, Yu-Jui Ho, Kaishi Satomi, Takahiro Shibayama, Kiyotaka Nagahama, K. Ohtsuka, Hiroaki Ohnishi, Yoshihiro Sakamoto, Junji Shibahara, Akimasa Hayashi
          </td>
          <td>2025-08-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are linked to cancer progression and poor prognosis, offering valuable genetic insights into tumors. Accurate detection of genomic alterations in CTCs is essential for improving cancer diagnosis and treatment. To address this, we develop Uniform Chromosome Conformation Capture (Uni-C), a method for profiling 3D chromatin architecture and genomic alterations at the single-cell level. Using Uni-C, we analyze CTCs from pancreatic cancer patient-derived xenograft (PDX) and spontaneous tumor mouse models. In the PDX model, integrating data from seven CTCs captures 88.7% of SNPs and INDELs, and 75.0% of structural variants present in tumor tissue. These findings indicate that variants detected in CTCs reflect tumor genomic features. Notably, we observe chromatin conformation differences between mitotic and interphase CTCs, suggesting potential markers of cell vitality. In the spontaneous tumor model, we identify driver gene mutations in CTCs and predict neoantigens, advancing early cancer detection and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcdb93a8b0e33fd2f466feefe41fc91b323a6623" target='_blank'>
              Identification of multiple genomic alterations and prediction of neoantigens from circulating tumor cells at the single-cell level
              </a>
            </td>
          <td>
            Xiaochen Gao, Xinyu Li, Weize Xu, Ming Jiao, Yu Guo, Jiajia Wang, Weihao Wang, Jiling Feng, Qianqian Guo, Chengchao Wu, Taiyu Zhang, Yuqin Yang, Da Lin
          </td>
          <td>2025-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Cancer-directed drugs are often clinically deployed without definitive understanding of their molecular mechanisms of action (MOA). Hypomethylating agents (HMAs), which result in the degradation of the DNA methyltransferase 1 (DNMT1), have been deployed for decades in the treatment of haematological malignancies1,2. The precise mechanism of action of these drugs, however, has been debated, rendering the design of rational combination therapies challenging. Here, we identified the deubiquitinating enzyme USP48 as a crucial regulator of posttranslational histone modification in the context of DNA demethylation. USP48 loss selectively enhances response to DNMT1 inhibition, leading to a rapid induction of cell death. We demonstrate that USP48 is localized at sites of DNA damage and deubiquitinates H2A variants and proteins important for DNA damage repair. Functionally, loss of USP48 triggers an increase in chromatin accessibility upon HMA treatment, rendering AML cells more susceptible to DNA damage. Our results support USP48 as a posttranslational histone modifier for chromatin stability and DNA damage in response to HMA-related DNA demethylation. These findings propose USP48 as a new target for combination therapy with HMAs for acute myeloid leukaemia (AML).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42e9d360cbac606431a70f9cd70adaccce3729a4" target='_blank'>
              Chromatin Perturbation Promotes Susceptibility to Hypomethylating Agents
              </a>
            </td>
          <td>
            C. Schneider, G. Alexe, L. Merickel, Ashleigh Meyer, Michelle L Swift, Rodolfo B. Serafim, A. Basanthakumar, A. Taillon, S. Salhotra, Fabio Boniolo, Sander Lambo, Yara Rodriguez, Björn Häupl, Rani E. George, D. E. Root, T. Oellerich, Volker Hovestadt, Dipanjan Chowdhury, K. Stegmaier
          </td>
          <td>2025-07-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Acral melanoma (AM) is an aggressive melanoma subtype with limited therapeutic options and poor outcomes. In non-European descent and admixed populations, like those residing in Latin America, AM accounts for a significant proportion of cutaneous melanoma cases. Here, we performed comprehensive genomic and functional profiling of AM from a uniquely diverse Brazilian cohort. Whole-exome and transcriptome sequencing revealed low mutation burden and predominance of copy number alterations, including high-amplitude focal amplifications termed hailstorms. These hailstorms frequently affected chromosomes 11, 5 and 22 and key oncogenes such as CCND1, GAB2, CDK4, and TERT. The presence of hailstorms in the long arms of chromosomes 11 and 22 was associated with higher focal copy number burden and loss of DNA damage response genes (ATM, CHEK1), suggesting a permissive genomic environment driving structural instability. To explore the unique genomic context of AM, we established a comprehensive collection of patient-derived xenograft (AM-PDX) models that faithfully retain the histopathological and genomic features of the original tumours. Functional exploration of AM-specific vulnerabilities through pharmacological and CRISPR/Cas9 knockout screenings identified strong sensitivity to targeting MAPK, CDK4/6, MDM2, and WEE1 pathways. Notably, the pan-RAS(ON) inhibitor RMC-7977 effectively reduced viability in NRAS-, KRAS-, and KIT-mutant AM cell lines. Finally, CRISPR screens revealed AM-enriched dependencies, including CRKL and SF3B4, highlighting previously unrecognized vulnerabilities. Our findings emphasize the distinct biology of AM compared to other subtypes of melanoma, provide a valuable resource of models reflective of Latin American ancestry, and identify potential drivers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0407eee77f76a83d31f10dc4bc4ede91f52ba5a" target='_blank'>
              Modelling Acral Melanoma in Admixed Brazilians Uncovers Genomic Drivers and Targetable Pathways
              </a>
            </td>
          <td>
            Annie Cristhine Moraes Sousa-Squiavinato, Sara Santos Bernardes, Flavia C. Aguiar, Antonio C. Facciolo, Martín del Castillo Velasco Herrera, J. R. C. Wong-Ramirez, P. Basurto-Lozada, Aretha Brito Nobre, Geethanjali Annamalai, Rebecca Martins Cadimo do Nascimento, Jacqueline M Boccacino, Rafaela Fagundes, Pedro Sodré do R. Barros, Mariana de Moraes Pitombo, R. Olvera-León, Larissa Satiko Alcantara Sekimoto Matsuyama, J. Billington, I. Vermes, Irving Simonin-Wilmer, Danielle G. Carvalho, João Pedro Cavalcante Simoes, Priscila Valverde Fernandes, Luiz Fernando Nunes, Andreia Cristina de Melo, Jadivan Leite de Oliveira, M. Herlyn, Andrew E. Aplin, C. D. Robles-Espinoza, David J. Adams, P. Possik
          </td>
          <td>2025-08-12</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17c19d5f1ca11d81a47f9f7924f6f192beb7cb85" target='_blank'>
              Cancer type-specific association of p53 deficiency with aneuploidy and chromosome losses
              </a>
            </td>
          <td>
            Joana F. Marques, Marco António Dias Louro, Teresa Davoli, G. Kops
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Objectives: To investigate the clinical diagnostic and prognostic value of extrachromosomal circular DNA (eccDNA) in breast cancer, eccDNA profiles were constructed for 81 breast cancer tumor tissues and 33 adjacent non-tumor tissues. Methods: The distribution characteristics of eccDNA across functional genomic elements and repetitive sequences were systematically analyzed. Furthermore, a diagnostic model for differentiating malignant and normal breast tissues, as well as a prognostic prediction model, was developed using a random forest algorithm. Results: EccDNA in breast cancer tissues harbor a higher proportion of functional elements and repetitive sequences, with their annotated genes significantly enriched in tumor- and immune-related pathways. However, no significant differences in eccDNA features were observed across breast cancer subtypes or pathological stages. In the validation cohort, the eccDNA-based diagnostic model achieved an AUC of 0.83, with repetitive elements and enhancer-associated features contributing the most to diagnostic performance. The prognostic model achieved an AUC of 0.78, with repetitive element annotations also showing strong prognostic relevance. Conclusions: These findings highlight the promising potential of eccDNA in the development of precision diagnostics and prognostic systems for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f033fc855abaceeac39bacf8c5fcd1d56a705c9" target='_blank'>
              A Study on the Diagnostic and Prognostic Value of Extrachromosomal Circular DNA in Breast Cancer
              </a>
            </td>
          <td>
            Fuyu Li, Wenxiang Lu, Lingsong Yao, Yunfei Bai
          </td>
          <td>2025-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Ribonucleoside monophosphates (rNMPs) are abundant in DNA, but their distribution and function in human nuclear genomes remain unknown. Here, we mapped nearly one million rNMPs per genome across diverse human cell types, defining a nuclear “ribome” with non-random distribution patterns. rNMPs are enriched in C/G-rich sequences, epigenetically marked regions, and telomeres. Conserved ribonucleotide-enriched zones (REZs) overlap with CpG islands and R-loops. rNMP concentration near transcription start sites (TSSs) correlates positively with gene expression. Wild-type cells display a broader gene-expression range than ribonuclease H2A (RNH2A) knockouts, in which loss of rNMP cleavage causes pronounced retention of embedded rG and strand-biased rC near TSSs, both increasing with gene expression. These findings establish DNA-embedded rNMPs as a novel epigenetic mark that modulates human gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a34d4cbfcfa7f6007c5ecd2489744cc0bcfeecb" target='_blank'>
              Human ribomes reveal DNA-embedded ribonucleotides as a new type of epigenetic mark
              </a>
            </td>
          <td>
            Deepali L. Kundnani, Taehwan Yang, Tejasvi Channagiri, Penghao Xu, Yeunsoo Lee, Mo Sun, Francisco Martinez-Figueroa, Supreet Randhawa, Ashlesha Gogate, Youngkyu Jeon, Stefania Marsili, Gary Newnam, Yilin Lu, Vivian S. Park, Sijia Tao, Justin A Ling, Raymond F. Schinazi, Z. Pursell, Abdulmelik Mohammed, P. Opresko, Bret D. Freudenthal, Baek Kim, Soojin V Yi, Nataša Jonoska, F. Storici
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Mammalian genomes are subdivided into euchromatic A compartments that contain mostly active chromatin, and inactive, heterochromatic B compartments. However, it is not well understood how A and B genome compartments are established and maintained. Here we study SMCHD1, an SMC-like protein best known for its role in X chromosome inactivation, in human male myoblasts. SMCHD1 colocalizes with Lamin B1 and the heterochromatin mark H3K9me3. Loss of SMCHD1 leads to extensive heterochromatin and Lamin B1 depletion at the nuclear lamina, acquisition of active chromatin states and increased DNA methylation along chromosomes. In absence of SMCHD1, long range intra-chromosomal contacts between B compartments are lost while many new TADs and loops are formed. Inactivation of SMCHD1 promotes numerous B to A compartment transitions accompanied by activation of silenced genes. The data suggests that SMCHD1 functions as an anchor for heterochromatin domains at the nuclear lamina ensuring that these domains are poorly accessible to DNA methyltransferases and to epigenome modification enzymes that typically operate in active chromatin. Thus, the properties of SMCHD1 in heterochromatin maintenance extend well beyond its role in X chromosome inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80d576a82dc3af4516675e7e1ac94325c2cc7c05" target='_blank'>
              SMCHD1 maintains heterochromatin, genome compartments and epigenome landscape in human myoblasts
              </a>
            </td>
          <td>
            Zhijun Huang, Wei Cui, Ishara Ratnayake, Kristin Gallik, Lorna Cohen, Rabi Tawil, Gerd P Pfeifer
          </td>
          <td>2025-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Copy number variants (CNVs) are duplications and deletions of DNA sequences that contribute to genetic variation between individuals and underlie rapid adaptive evolution. Increases in gene copy number can confer a strong selective advantage in some circumstances, but CNVs can also incur fitness costs. Prior research has shown that experimental evolution of Saccharomyces cerevisiae in nutrient-limited chemostats recurrently selects for amplifications of nutrient transporter genes, including GAP1, MEP2, and PUT4 in glutamine, ammonium and proline limited conditions, respectively. However, the fate of these CNVs upon return of the organism to a non-selective environment, is unknown. To investigate the fitness costs and stability of CNVs upon reversal of the original selection pressure, we studied 15 unique CNV lineages that had been selected in different nitrogen-limited chemostats, including both segmental amplifications and whole-chromosome aneuploidies. We studied the stability of CNVs using a fluorescent CNV reporter system during propagation in nutrient-rich media using serial dilution of batch cultures for 110–220 generations. We found that only one-third of CNV lineages repeatedly underwent loss of CNVs and reversion to a single-copy genotype. All lineages containing aneuploidies showed rapid reversion dynamics, whereas lineages containing segmental amplifications were remarkably stable – only one of 11 strains reverted to a single copy. Pairwise competitive fitness assays revealed strong fitness defects associated with reverting CNVs in the nutrient-rich environment but minimal fitness defects in lineages that did not revert. Reversion of CNVs led to increased fitness. Using simulation-based inference to estimate reversion rates and fitness effects we find that negative selection is the primary driver of CNV loss although aneuploid reversion rates are high with estimates of 10−5 to 10−3. Whole-genome sequencing of clones isolated from evolved populations revealed that reversion of both aneuploids and a segmental amplification left no evidence of prior existence of the CNV, making the genomes of revertants indistinguishable from the single-copy ancestor. Our findings provide novel evidence of the low fitness cost and high stability of most segmental CNVs upon removal of the selection pressure, and that gene amplifications with large fitness costs are readily reversible, highlighting the significance of CNVs for genome evolution as well as rapid and reversible adaptation to transient selection pressures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/478ede766edac8c6526ad5e7502f78feea49ddaf" target='_blank'>
              Segmental copy number amplifications are stable in the absence of selection
              </a>
            </td>
          <td>
            Titir De, Nadav Ben Nun, P. Spealman, Ina Suresh, Grace Avecilla, Farah Abdul-Rahman, Yoav Ram, David Gresham
          </td>
          <td>2025-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1231b220c32fa68e474d1bf7533e08b4effde6bb" target='_blank'>
              Cell enlargement causes mitotic errors and aneuploidy in cells that evade senescence after CDK4/6 inhibition
              </a>
            </td>
          <td>
            Aanchal U. Pareri, Marialucrezia Losito, F. Foijer, AT Saurin
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53236a139dc9b0a444537137348625f035c225eb" target='_blank'>
              Allelic Imbalance and Chromothripsis Lead to Diversity in Japanese Tumor Genomes With Whole-Genome Duplication.
              </a>
            </td>
          <td>
            K. Hatakeyama, Takeshi Nagashima, Sumiko Ohnami, S. Ohnami, Koji Maruyama, K. Ohshima, Y. Shimoda, A. Naruoka, H. Kenmotsu, K. Urakami, Y. Akiyama, Ken Yamaguchi
          </td>
          <td>2025-08-06</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5ccf1087bb37643567d38d9f74e846184322997" target='_blank'>
              SHLD2 loss is a synthetic vulnerability to Polθ inhibition combined with radiotherapy
              </a>
            </td>
          <td>
            G. Rodriguez-Berriguete, Purusotha Thambiayah, Alessandro Cicconi, Nicole D. Machado, Celia Gotorbe, David Nderitu, Wei-Chen Cheng, Marie Laure Boursier, Aurora Cerutti, V. Grinkevich, Bethany Rebekah Hill, Katjusa Koler, Sophie Langdon, Jayesh Majithiya, Suraj Menon, Shaun Moore, Joana Neves, Natalie M Palmer-Deverill, E. Rajendra, Marina Roy-Luzarraga, Asmita Thapa, Robert A Heald, Graeme C M Smith, Helen M R Robinson, M. Ranzani, Geoff S. Higgins
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="It was recently shown that inhibition of polo-like kinase 4 (PLK4) induces TP53-dependent synthetic lethality in cancers with chromosome 17q-encoded TRIM37 copy number gain due to cooperative regulation of centriole duplication and mitotic spindle nucleation. We show here that chromosome 17q/TRIM37 gain is a pathognomonic feature of high-risk neuroblastoma and renders patient-derived cell lines hypersensitive to the novel PLK4 inhibitor RP-1664. We demonstrate that centriole amplification at low doses of RP-1664 contributes to this sensitivity in a TRIM37- and TP53-independent fashion. CRISPR screens and live cell imaging reveal that upon centriole amplification, neuroblastoma cells succumb to multipolar mitoses due to an inability to cluster or inactivate supernumerary centrosomes. RP-1664 showed robust anti-tumor activity in 14/15 neuroblastoma xenograft models and significantly extended survival in a transgenic murine neuroblastoma model. These data support biomarker-directed clinical development of PLK4 inhibitors for high-risk neuroblastoma and other cancers with somatically acquired TRIM37 overexpression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b9f40478af2f555a373ad66dd3ebf45d31430e0" target='_blank'>
              The PLK4 inhibitor RP-1664 demonstrates potent single-agent efficacy in neuroblastoma models through a dual mechanism of sensitivity
              </a>
            </td>
          <td>
            John M. Maris, I. Soria-Bretones, Matias Casás-Selves, M. Samanta, David Groff, Jayne E. Murray, Jamie I. Fletcher, Alvin Farrel, Steven Pastor, Khushbu Patel, Elliot Goodfellow, Li Li, Cathy Caron, Ariya Shiwram, Hyeyeon Kim, Danielle Henry, Nancy Laterreur, Julian Bowlan, K. Krytska, Steven Neuhauser, Timothy M. Stearns, J. Schubert, Jinhua Wu, Lea Surrey, Alejandro Álvarez-Quilón, Frédéric Vallée, Parham Nejad, J. Schonhoft, Joanna Li, A. Veloso, Jordan T. F. Young, M. Hyer, S. Morris, Y. Mossé, Gary Marshall, Michelle Haber, Michal Zimmermann
          </td>
          <td>2025-07-29</td>
          <td>Research Square</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eec7c3cc61289a3954a3ddd8b996ed6f9c80850e" target='_blank'>
              The adaptive molecular landscape of reprogrammed telomeric sequences
              </a>
            </td>
          <td>
            Melania D’Angiolo, Benjamin P. Barré, Sakshi Khaiwal, J. Muenzner, Johan Hallin, Matteo De Chiara, Nicolò Tellini, J. Warringer, M. Ralser, Eric Gilson, Gianni Liti
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Objective Repetitive DNA comprises the majority of nuclear genomes in eukaryotes and is critical for genome stability, gene regulation and evolutionary innovation. Yet, most genomic surveys focus on lowcopy regions, leaving repeats underexplored. Here, we assess how repetitive elements shape genomic diversity and rapid radiation in Dyckia (Bromeliaceae) by characterizing the abundance, composition and variability of major repeat families using lowcoverage whole-genome sequencing and the RepeatExplorer2 pipeline. Results description A substantial proportion of Dyckia genome consists of repetitive DNA, reaching ~ 71% in D. densiflora, D. elata, and D. consimilis. Notable interspecific and intraspecific variation was observed, with D. consimilis ranging from 50.3 to 69.1%. Satellite DNA, though present in all species, varied in abundance (0.1–4.7%), indicating heterochromatin flexibility. Ty3/Gypsy and Ty1/Copia LTR retrotransposons dominate the repeatome, with lineage-specific expansions of Tekay and Ogre elements. Despite general conservation, population-level differences in repeat composition suggest a role in genome restructuring and phenotypic plasticity. These patterns point to repeat dynamics as a key driver of genome evolution, taxonomic complexity, and ecological adaptability in Dyckia. Supplementary Information The online version contains supplementary material available at 10.1186/s13104-025-07359-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3a9ee02793b5181e16f379a04487e1d2a8906d6" target='_blank'>
              Genomic characterization of repetitive DNA and transposable elements in Dyckia (Pitcairnioideae) species
              </a>
            </td>
          <td>
            João Victor da Silva Rabelo-Araujo, A. F. Francisconi, Zirlane Portugal da Costa, Caroline Bertocco Garcia, Otavio Batista de Castro Ribeiro, Ana Cristina Silva Amoroso Anastacio, Maria Imaculada Zucchi
          </td>
          <td>2025-07-09</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The strength of the repressive histone H3 lysine 27 trimethylation modification signal varies drastically at individual silencers. Focusing on cases of an unusually strong repressive signal in regions that we refer to as super-silencers, we demonstrate that the regions that become B-cell super-silencers are originally associated with gene upregulation during development, and their target genes are highly expressed in stem cells, especially during early developmental stages. About 13% of B-cell super-silencers transmute to super-enhancers in B-cell lymphoma and 22% of these conversions recur across more than half of patients. Notably, genes associated with these conversions, like BCL6 and BACH2, are downregulated more swiftly than others when subjected to JQ1, a super-enhancer-disrupting bromodomain and extra-terminal domain inhibitor utilized in cancer chemotherapy. Furthermore, super-silencers are characterized by an over-representation of B-cell-cancer-associated mutations, both somatic and germline, and B-cell-cancer translocation breakpoints. This surpasses the prevalence found in other regulatory elements, such as CTCF binding sites, underlining the crucial role of super-silencers in forming and stabilizing regulatory topologies in standard B cells. For example, over 80% of cases involving the B-cell-lymphoma translocation t(3;14)(q27;q32) fuse super-silencers in the BCL6 locus with enhancer-rich domains. Finally, we demonstrate that the repressive mechanisms of super-silencers are partially governed by the CpG content in their sequences. While CpG-rich super-silencers often prevent promoters from interacting with enhancers, CpG-depleted super-silencers typically suppress the chromatin looping of nearby enhancers. In summary, our findings accentuate the critical role super-silencers play in the normal function of B-cells, suggesting that sequence mutations and activity modifications in these elements could be primary factors in B-cell carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d12eb40d352b5dd3f41e425053f82e55565a55cc" target='_blank'>
              Super-silencers are crucial for development and carcinogenesis in B cells
              </a>
            </td>
          <td>
            Di Huang, H. Petrykowska, Dhaneshwar Kumar, L. Kardava, S. Moir, B. Afzali, L. Elnitski, Ivan Ovcharenko
          </td>
          <td>2025-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="The DNA repair protein RAD18 activates ‘Y-family’ Trans-Lesion Synthesis (TLS) DNA polymerases that are DNA damage-tolerant and potentially error-prone. RAD18 is also frequently overexpressed and pathologically activated in cancer cells. However, the extent to which RAD18 shapes cancer genomes and impacts tumorigenesis is unclear. Therefore, we tested the effect of Rad18 status on chemically-induced and oncogene-driven tumorigenesis. In a chemically-induced oral carcinogenesis model, acute (2-16 days) 4NQO-treatment induces expression of Rad18 and TLS polymerase mRNAs in mouse oral epithelial cells prior to emergence of oral squamous cell carcinomas (OSCCs). Chronic (8 week) 4NQO-treatment leads to onset of oral tumors that is accelerated in Rad18-/- mice when compared with Rad18+/+ animals. Analysis of OSCC genomes reveals increased levels of G(C)>T(A) transversions in Rad18-/- tumors when compared with Rad18+/+. Therefore, Rad18 promotes error-free bypass of 4NQO-induced DNA lesions and suppresses 4NQO-induced oral carcinogenesis. In a KrasG12D-induced lung carcinogenesis model, Rad18-deficiency did not affect rates or incidence of oncogene-induced lung tumors or mutations. Taken together, we demonstrate that Rad18 has context-specific tumor-suppressive activity. Given the prevalence of 4NQO-like environmental exposures, RAD18 is highly likely to shape human cancer genomes and perhaps influence other aspects of the tumorigenic process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/318ddf85ddf5e7011f464ee8a8d2e2b51144fcdd" target='_blank'>
              DIFFERENTIAL ROLES OF RAD18 IN REPRESSING CARCINOGEN- AND ONCOGENE-DRIVEN MUTAGENESIS IN VIVO
              </a>
            </td>
          <td>
            J. R. Anand, Bethany Wagner, Jitong Lou, Qisheng Gu, Yang Yang, Gaith N Droby, Di Wu, Akankshya Jena, Jialiu Xie, Yuri Fedoriw, Bernard E Weissman, Scott E. Williams, Cyrus Vaziri
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumors (ATRTs) are aggressive central nervous system tumors mainly affecting young children. Extensive molecular characterization based on gene expression and DNA methylation patterns has solidly established three major ATRT subtypes (MYC, SHH and TYR), which show distinct clinical features, setting the basis for more effective, targeted treatment regimens. Transcriptional activity of transposable elements (TEs), like LINE1s and LTRs, is tightly linked with human cancers as a direct consequence of lifting epigenetic repression over TEs. The sole recurrent biallelic loss-of-function mutation in SMARCB1 in ATRTs, a core component of the SWI/SNF chromatin remodeling complex, raises the question of how TE transcription contributes to ATRT development. Here, we comprehensively investigate the transcriptional profiles of 1.9M LINE1 and LTR elements across ATRT subtypes in primary human samples. We find TE transcription profiles allow sample stratification into ATRT subtypes. The TE activity signature in the ATRT-MYC subtype is unique, setting these tumors apart from SHH and TYR ATRTs. More specifically, ATRT-MYC show broadly reduced transcript levels of LINE1 and ERVL-MaLR subfamilies. ATRT-MYC also displayed significantly less LTR and LINE1 loci with bidirectional promoter activity. Furthermore, we identify 849 differentially transcribed TEs in primary samples, which are predictive towards established ATRT-SHH and -MYC cell line models. In summary, including TE transcription profiles into the molecular characterization of ATRTs might reveal new tumor vulnerabilities leading to novel therapeutic interventions, such as immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02078-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c60e5327059223ad1194aac3ab94faeb7d65828" target='_blank'>
              Expression of LTR and LINE1 transposable elements defines atypical teratoid/rhabdoid tumor subtypes
              </a>
            </td>
          <td>
            M.V. Hamann, Shweta Godbole, Maisha Adiba, Sabrina M. Leddy, Jelena Navolić, Ghazaleh Tabatabai, Daniel J Merk, Ulrike C. Lange, J. Neumann
          </td>
          <td>2025-07-22</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT KRAS is mutationally activated in 45-50% of colorectal cancer (CRC) cases, and while KRAS-targeted therapies have shown some clinical promise, upfront and acquired resistance limit their efficacy. To explore the acute response and mechanisms underlying KRAS inhibitor resistance, we used targeted exome sequencing and single-cell spatial transcriptomics to analyze patient-matched pre-treatment, on-treatment, and progression biopsies from patients treated with combined KRAS G12C and EGFR inhibition. Acquired genetic events were identified in most patients at progression but were often subclonal and coexisted with transcriptional adaptive states. Mesenchymal, YAP, and fetal-like transcriptional signatures predominated in resistant tumors, while tumor cell-intrinsic inflammatory programs were induced in the early treatment phase. Single-cell spatial analysis revealed significant intratumoral heterogeneity, with diverse adaptive states predominating in different zones of individual tumors. Using human and murine organoid models, we show that these drug-induced inflammatory programs are cancer-cell autonomous and precede the emergence of regenerative fetal-like programs associated with drug resistance. We uncover TBK1 as a promising target to abrogate the early inflammatory adaptive phase and enhance responses to KRAS inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75a7967138a21f862a2f0477898f16663439bb11" target='_blank'>
              Concurrent genetic and non-genetic resistance mechanisms to KRAS inhibition in CRC
              </a>
            </td>
          <td>
            Salvador Alonso, Kevan Chu, Marie J Parsons, Elizabeth Granowsky, H. Gunasinghe, Jinru Shia, R. Yaeger, Lukas E. Dow
          </td>
          <td>2025-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Summary DNA damage can result from external sources or occur during programmed genome rearrangements in processes like immunity or meiosis. To maintain genome integrity, cells activate DNA repair pathways that prevent harmful outcomes such as cancer or immune dysfunction. In this study, we uncover a novel role for DNA damage during the terminal differentiation of multiciliated cells (MCCs). MCCs, which line the airways, reproductive tracts, and brain ventricles, produce hundreds of motile cilia, each anchored by a centriole. Therefore, MCCs must generate hundreds of centrioles during differentiation. Normally, centriole duplication is tightly linked to the S and G2 phases of the cell cycle, raising questions about how MCCs override numerical and temporal restrictions on centriole duplication. We find that differentiating MCCs accumulate extensive double-stranded DNA breaks during centriole amplification, with damage levels correlating with centriole number. DNA damage response (DDR) kinases are essential for supporting centriole biogenesis and ciliogenesis. We also observe that transcriptional activity, required for the expression of centriole and cilia genes, produces RNA-DNA hybrids (R-loops) that co-localize with DNA damage. This suggests that transcription-coupled DNA damage helps initiate a pseudo–cell cycle program, permitting centriole amplification without triggering full S/G2 phase processes. Our findings indicate that MCCs harness DDR signaling as part of their developmental program, revealing a broader principle in which the canonical cell cycle is adaptively rewired during differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/025db960c1d15e49e5330c45782572ca46dc80ab" target='_blank'>
              The DNA damage response pathway is required for multiciliated cell differentiation
              </a>
            </td>
          <td>
            Cayla E. Jewett, Andrew J Holland, Chad G Pearson
          </td>
          <td>2025-07-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Genome instability is a hallmark of cancer, often driven by mutations and altered expression of genome maintenance factors involved in DNA replication and repair. Rap1 Interacting Factor 1 (RIF1) plays a crucial role in genome stability and is implicated in cancer pathogenesis. Cells express two RIF1 splice variants, RIF1-Long and RIF1-Short, which differ in their ability to protect cells from DNA replication stress. Here, we investigate differential expression and splicing of RIF1 in cancer cell lines following replication stress and in patients using matched normal and tumour data from The Cancer Genome Atlas (TCGA). Overall RIF1 expression is altered in several cancer types, with increased transcript levels in colon and lung cancers. RIF1 also exhibits distinct splicing patterns, particularly in specific breast cancer subtypes. In Luminal A (LumA), Luminal B (LumB), and HER2-enriched breast cancers (HER2E), RIF1 Exon 31 tends to be excluded, favouring RIF1-Short expression and correlating with poorer clinical outcomes. These breast cancer subtypes also tend to exclude other short exons, suggesting length-dependent splicing dysregulation. Basal breast cancer also shows exon exclusion, but unlike other subtypes, it shows no short-exon bias. Surprisingly, however, in basal breast cancer, RIF1 Exon 31 is not consistently excluded, which may impact prognosis since RIF1-Long protects against replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ca94768efdf45848f937fd47caaaca2fe19dbc3" target='_blank'>
              Dysregulated Alternative Splicing in Breast Cancer Subtypes of RIF1 and Other Transcripts
              </a>
            </td>
          <td>
            Emma Parker, Laura Akintche, Alexandra Pyatnitskaya, S. Hiraga, A. Donaldson
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Transposable elements (TEs) are abundant in the human genome, and they provide the source for genetic and functional diversity. Previous studies have suggested that TEs are repressed by DNA methylation and chromatin modifications. Here through integrating transcriptome and 3D genome architecture studies, we showed that haploinsufficient loss of NIPBL selectively activates alternative promoters (altPs) at the long terminal repeats (LTRs) of the TE subclasses. This activation occurs through the reorganization of topologically associating domain (TAD) hierarchical structures and the recruitment of proximal enhancers. These observations indicate that TAD hierarchy restricts transcriptional activation of LTRs that already possess open chromatin features. Perturbation of hierarchical chromatin topology can lead to co-option of LTRs as functional altPs, driving aberrant transcriptional activation of oncogenes. These data uncovered a new layer of regulatory mechanisms of TE expression and posit TAD hierarchy dysregulation as a new mechanism for altP-mediated oncogene activation and transcriptional diversity in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13b5cab876cec5cb4938bafcc68be31f6630d5cb" target='_blank'>
              Disruption of TAD hierarchy promotes LTR co-option in cancer
              </a>
            </td>
          <td>
            Elissa W P Wong, Merve Sahin, Rui Yang, UkJin Lee, Dan Li, Y. Zhan, Rohan Misra, Fanny Tomas, Nawaf Alomran, A. Polyzos, Cindy J. Lee, Tuan Trieu, A. Martínez-Fundichely, Thomas Wiesner, Andrew Rosowicz, Shuyuan Cheng, Christina Liu, Morgan Lallo, A. Shoushtari, T. Merghoub, Pierre-Jacques Hamard, R. Koche, Ekta Khurana, E. Apostolou, Deyou Zheng, Yu Chen, Christina S. Leslie, P. Chi
          </td>
          <td>2025-06-30</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>83</td>
        </tr>

        <tr id="The identification of CENPA, CENPB, and CENPC by Earnshaw and Rothfield 40 years ago has revealed the remarkable diversity and complexity of centromeres and confirmed most seed plants and animals have centromeres comprised of complex satellite arrays. The rapid evolution of centromeres and positive selection on CENPA and CENPC led to the centromere drive model, in which competition between tandem satellite arrays of differing size and centromere strength for inclusion in the egg of animals or megaspore of seed plants during female meiosis drives rapid evolution of centromeres and kinetochore proteins. Here we review recent work showing that non-B-form DNA structures in satellite centromeres make them sites of frequent replication fork stalling, and that repair of collapsed forks by break-induced replication rather than unequal sister chromatid exchange is likely the primary mode of satellite expansion and contraction, providing the variation in satellite copy number that is the raw material of centromere drive. Centromere breaks at replication, rather than errors at mitosis, can account for most centromere misdivisions that underlie aneuploidies in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65fb737fe39c4474778577289e08e6983374bf35" target='_blank'>
              Centromeres drive and take a break
              </a>
            </td>
          <td>
            P. Talbert, S. Henikoff
          </td>
          <td>2025-08-04</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Significance Modified cytosine bases are essential components of mammalian genomes, but how they are distributed in double-stranded DNA remains unclear. Here, we report a method to simultaneously sequence the two most abundant modified bases—5-methylcytosine and 5-hydroxymethylcytosine (hmC)—across both strands of genomic DNA. Applying this method to DNA from mouse embryonic stem cells, we find that at CpG sites, hmC forms different combinations with cytosine variants across the DNA double helix, exhibiting unique distributions at enhancers and gene bodies, and distinct relationships with transcription. Our findings demonstrate that the double-stranded context of hmC is a key feature of the epigenome, revealing a further layer of regulatory information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8822428faae43abf3590008b224a007bfe559580" target='_blank'>
              SCoTCH-seq reveals that 5-hydroxymethylcytosine encodes regulatory information across DNA strands
              </a>
            </td>
          <td>
            Jack S Hardwick, Somdutta Dhir, Angie Kirchner, A. Simeone, Sean M. Flynn, J. M. Edgerton, Rafael de Cesaris Araujo Tavares, Isabel Esain-Garcia, D. Tannahill, Pauline Y. Golder, Jack M Monahan, Walraj S. Gosal, Shankar Balasubramanian
          </td>
          <td>2025-07-31</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Functional inactivation of tumor suppressor genes drives cancer initiation, progression, and treatment responses. Most tumor suppressor genes are inactivated through 1 of 2 well-characterized mechanisms: DNA-level mutations, such as point mutations or deletions, and promoter DNA hypermethylation. Here, we report a distinct third mechanism of tumor suppressor inactivation based on alterations to the histone rather than DNA code. We demonstrated that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event in more than 80% of human endometrial cancers. Integrative transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that PAX2 transcriptional downregulation is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive chromatin features (H3K27me3) in a framework dictated by 3D genome organization. The spread of the repressive H3K27me3 signal resembled a pearl necklace, with its length modulated by cohesin loops, thereby preventing transcriptional dysregulation of neighboring genes. This mechanism, involving the loss of a promoter-proximal superenhancer, was shown to underlie transcriptional silencing of PAX2 in human endometrial cancers. Mouse and human preclinical models established PAX2 as a potent endometrial tumor suppressor. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. The discovery that most endometrial cancers originate from a recurring epigenetic alteration carries profound implications for their diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5897c9bf68c93e4fb62e86f78e7178d27155eec3" target='_blank'>
              A distinct mechanism of epigenetic reprogramming silences PAX2 and initiates endometrial carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad R. Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2025-08-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="DNA carries genetic information, ensuring the stable transmission of genetic material through generations. However, DNA sequences can be constantly rewritten, allowing evolution and adaptation to occur at both the cellular and species levels. To facilitate this dynamic process, the genome is enriched with mobile genetic elements that can move within DNA sequences. While transposable elements are the classic example of mobile DNA, we propose that extrachromosomal circular DNA (ecDNA) represents another class of mobile genetic elements. This class can also introduce a new layer of genome dynamics, significantly shaping host biology. This review traces the historical discoveries and conceptual evolution of ecDNA, categorizing its diverse forms across organisms and highlighting its life cycle from biogenesis to maintenance and clearance. We discuss ecDNA's pivotal roles in physiological processes, including development, stress adaptation, and species evolution, and emphasize its pathological significance in cancer progression, drug resistance, and viral infections. Finally, we explore therapeutic strategies targeting ecDNA biology, offering a framework for translational advances in oncology and antiviral treatment. Together, these insights position ecDNA as a critical force in shaping genome dynamics and as a promising target for future biomedical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a2c075da6c2a7912686c8ef608ee9b31391b746" target='_blank'>
              Extrachromosomal Circular DNA: A Mobile Genetic Element Shaping Host Biology.
              </a>
            </td>
          <td>
            Shun Yao, Oliver W. Chung, Ling Wang, Z. Zhang
          </td>
          <td>2025-08-05</td>
          <td>Annual review of cell and developmental biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The role of G-quadruplexes (G4s) in gene regulation has been widely documented, especially in gene promoters. However, the transcriptional mechanisms involving G4s in other regulatory regions remain largely unexplored. In this study, we integrated the G4-DNA data derived from 22 breast cancer patient-derived tumor xenograft (PDTX) models and MCF7 cell line as potential breast cancer-associated G4s (BC-G4s). Genome-wide analysis showed that BC-G4s are more prevalent in gene promoters and the first introns. The genes accommodating promoter or intronic BC-G4s show significantly higher transcriptional output than their non-G4 counterparts. The biased distribution of BC-G4s in close proximity to the transcription start site (TSS) is associated with an enrichment of transcription factor (TF) interactions. A significant negative correlation was detected between the G4–TF interactions within the first introns and their cognate promoters. These different interactions are complementary rather than redundant. Furthermore, the differentially expressed genes (DEGs) harboring promoter and first intron BC-G4s are significantly enriched in the cell cycle pathway. Notably, promoter BC-G4s of DEGs could be a central hub for TF–TF co-occurrence. Our analysis also revealed that G4-related single nucleotide variants (SNVs) affect the stability of G4 structures and the transcription of disease-related genes. Collectively, our results shed light on how BC-G4s within promoters and first introns regulate gene expression and reinforce the critical role of G4s and G4-related genes in breast cancer-associated processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6951122fcae4ae20d0e429a03632112d1f8ad46" target='_blank'>
              The Relevance of G-Quadruplexes in Gene Promoters and the First Introns Associated with Transcriptional Regulation in Breast Cancer
              </a>
            </td>
          <td>
            Huiling Shu, Ke Xiao, Wenyong Zhu, Rong Zhang, Tiantong Tao, Xiao Sun
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Microhomology-mediated tandem duplication (TD) is a distinct mutational signature in BRCA1-deficient tumors. While several mechanisms have been proposed for its generation, much understanding comes from repeat-based reporters, which may not represent the region of the human genome lacking neighboring repeats. To address this limitation, we developed a repeat-less TD reporter and a PCR-based endogenous site-specific TD assay to examine TDs induced by replication-coupled single-ended DNA double-strand breaks (seDSBs) in Brca1-deficient mouse embryonic stem cells. While TDs induced by seDSBs were detectable in wild-type cells, they were significantly elevated in Brca1-deficient cells, independent of classical non-homologous end joining. Some of these TDs arose from allelic DNA synthesis followed by a shift to microhomology-mediated non-allelic sister chromatid recombination, suggesting increased premature termination of allelic DNA synthesis, a late step of allelic break-induced replication (aBIR), in Brca1-deficient cells. Disrupting RAD51 loading reduced TDs in wild-type cells but not in Brca1-deficient cells, indicating both RAD51-dependent and -independent TD formation. Additionally, RAD54 and BRCA1–BARD1 suppressed TDs in a RAD51-dependent manner, with the effect of BARD1 partially mediated through RAD51 interaction. These findings together implicate late steps of aBIR in TD suppression and provide novel insights into the mechanisms underlying BRCA1-linked TD formation in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7edf834c059a1df782121d4a36a790a684d7b6d4" target='_blank'>
              Late steps of allelic break-induced replication suppress tandem duplication associated with BRCA1 deficiency
              </a>
            </td>
          <td>
            Zhi-cheng Huang, , Qian Liu, Ruo-Dan Chen, Si-Cheng Liu, Meng Wang, A. Xie
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="R-loop is a special DNA–RNA hybrid genomic structure. Since its identification, the molecular mechanisms of physiological processes such as class switch recombination have been revealed, uncontrolled regulation of which has become the underlying cause of many diseases. With the development of molecular biology techniques, R-loops found at different sites and formed via different mechanisms have been discovered. These non-classical R-loops participate in various cellular activities via different mechanisms. A set of sophisticated regulatory mechanisms are required to control the number of various R-loops in vivo. The levels of non-classical R-loops in tumor cells differ significantly from those in normal cells; in addition, the regulatory mechanisms for establishing R-loop homeostasis differ, which may be utilized to develop breakthrough therapies for various tumors. In this review, we summarize the current state of knowledge regarding non-classical R-loops in tumor cells, the mechanisms that promote or inhibit this structure, the effects of this structure on tumor cells, and the possible therapeutic targets. By systematically elucidating the pathogenic mechanisms of atypical R-loops, we can achieve precision targeting of tumors and revolutionize clinical precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676d0195748900abca38448fb2ef5985171fe4bf" target='_blank'>
              Atypical R-loops in cancer: decoding molecular chaos for therapeutic gain
              </a>
            </td>
          <td>
            Yuan Sun, Sheng Wang, N. Ge, Jintao Guo, Guoxin Wang, Fan Yang, Siyu Sun
          </td>
          <td>2025-08-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The DNA-incorporating nucleoside analogs azacytidine (AZA) and decitabine (DEC) have clinical efficacy in blood cancers, yet the precise mechanism by which these agents kill cancer cells has remained unresolved – specifically, whether their anti-tumor activity arises from conventional DNA damage or DNA hypomethylation via DNA methyltransferase 1 (DNMT1) inhibition. This incomplete mechanistic understanding has limited their broader therapeutic application, particularly in solid tumors, where early clinical trials showed limited efficacy. Here, through the assessment of drug sensitivity in over 600 human cancer models and comparison to a non-DNA-damaging DNMT1 inhibitor (GSK-3685032), we establish DNA hypomethylation, rather than DNA damage, as the primary killing mechanism of AZA and DEC across diverse cancer types. In further support of an epigenetic killing mechanism, CRISPR drug modifier screens identified a core set of chromatin regulators, most notably the histone deubiquitinase USP48, as AZA and DEC protective factors. We show that USP48 is recruited to newly hypomethylated CpG islands and deubiquitinates non-canonical histones, establishing USP48 as a key molecular link between the two components of epigenetic gene regulation: DNA methylation and chromatin modification. Furthermore, loss of USP48, which occurs naturally through biallelic deletions in human cancers, sensitized both hematologic and solid tumors to DNMT1 inhibition in vitro and in vivo. Our findings elucidate the epigenetic mechanism of action of AZA and DEC and identify a homeostatic link between DNA methylation and chromatin state, revealing new therapeutic opportunities for DNMT1 inhibitors in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4849479727b73a833dd0ff467a2ebd5a79ba5e8" target='_blank'>
              Discovery of chromatin-based determinants of azacytidine and decitabine anti-cancer activity
              </a>
            </td>
          <td>
            Rishi V. Puram, Qiangzong Yin, YuhJong Liu, Justine C. Rutter, Daniel Bondeson, Maria C. Saberi, Lisa Miller, Michael Du, Khanh Nguyen, Donovan L. Batzli, Hilina B. Woldemichael, Christian Taeger, Anna Goldstein, Ming Y. Chu, Qi Guo, D. Mani, Michael Naumann, M. Ronan, M. Rees, Blanche C. Ip, Mustafa Kocak, Mikołaj Słabicki, John G Doench, Jennifer A. Roth, Steven A. Carr, N. Udeshi, Jingyi Wu, Todd R. Golub
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Abstract DNA–protein crosslinks (DPCs) form following exposure to various alkylating agents, including environmental carcinogens, cancer chemotherapeutics, and reactive aldehydes. If not repaired, DPCs can interfere with key biological processes such as transcription and replication and activate programmed cell death. A growing body of evidence implicates nucleotide excision repair (NER), homologous recombination, and other mechanisms in the removal of DPCs. However, the effects of genomic context on DPC formation and removal have not been comprehensively addressed. Using a combination of next-generation sequencing and DPC enrichment via protein precipitation, we show that DPCs induced following exposure to formaldehyde are non-randomly distributed across the human genome, based on chromatin state. The data further show that the efficiency of DPC removal correlates with transcription at loci transcribed by RNA polymerase II. Data presented herein indicate that efficient removal of chromosomal DPCs requires both the Cockayne syndrome group B gene as well as “downstream” TC-NER factor xeroderma pigmentosum group A gene. In contrast, loci transcribed by RNA polymerase I showed no evidence of transcription-coupled DPC removal. Taken together, our results indicate that complex interactions between chromatin organization, transcriptional activity, and numerous DNA repair pathways dictate genomic patterns of DPC formation and removal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/138482f47c7713ca6d5e082edda14419c0390855" target='_blank'>
              Genome-wide mapping of formaldehyde-induced DNA–protein crosslinks reveals unique patterns of formation and transcription-coupled removal in mammalian cells
              </a>
            </td>
          <td>
            Duha Alshareef, Charlie T Nguyen, Kayla N Tucker, Micah D. Gearhart, Natalia Y Tretyakova, Colin Campbell
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Jumping translocations (JTs) are rare chromosomal abnormalities that play a crucial role in the pathogenesis of various cancer types. These rearrangements, especially those involving chromosome 1q, are frequently associated with tumor progression, therapeutic resistance, and poor prognosis. One gene of particular interest, human Jumping Translocation Breakpoint (JTB), has been identified at the site of translocation breakpoints and exhibits complex, context-dependent roles in cancer biology. JTB protein functions as a pivotal regulator in mitosis, chromosomal segregation, apoptosis, and cellular metabolism. It is functionally linked with the chromosomal passenger complex (CPC) and is implicated in processes such as epithelial–mesenchymal transition (EMT), immune evasion, and therapy resistance, especially in breast and prostate cancers. Advances in genomic, transcriptomic, and proteomic research have highlighted the significant potential of JTB as a diagnostic biomarker and a target for therapeutic interventions. This review underscores the dual role of JTB as both a tumor suppressor and oncogene, depending on the cellular context, and advocates for its continued investigation at the genomic, transcriptomic, and proteomic levels. Understanding JTB’s multifaceted contributions to tumor biology may pave the way for novel biomarkers and targeted treatments in cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48960a3d042990da8ae30d21a39837a5c48dded4" target='_blank'>
              From Jumping Gene to Cancer: Revisiting the Role of JTB Protein
              </a>
            </td>
          <td>
            Taniya M Jayaweera, Madhuri Jayathirtha, Krishan Weraduwage, Petra Kraus, C. Darie, A. Neagu
          </td>
          <td>2025-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The capacity of a plasmid to express genes is constrained by its length and copy number. However, the interplay between these parameters and their constraints on plasmid evolution have remained elusive due to the absence of comprehensive quantitative analyses. Here, we present ‘Pseudoalignment and Probabilistic Iterative Read Assignment’ (pseuPIRA), a computational method that overcomes previous computational bottlenecks, enabling rapid and accurate determination of plasmid copy numbers at large scale. We apply pseuPIRA to all microbial genomes in the NCBI RefSeq database with linked short-read sequencing data (4644 bacterial and archaeal genomes including 12,006 plasmids). The analysis reveals three scaling laws of plasmids: first, an inverse power-law correlation between plasmid copy number and plasmid length; second, a positive linear correlation between protein-coding genes and plasmid length; and third, a positive correlation between metabolic genes per plasmid and plasmid length, particularly for large plasmids. These scaling laws imply fundamental constraints on plasmid evolution and functional organization, indicating that as plasmids increase in length, they converge toward chromosomal characteristics in copy number and functional content.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96d0a945dacbeb8801a3cb57a4c7f7eb8870d06a" target='_blank'>
              Scaling laws of bacterial and archaeal plasmids
              </a>
            </td>
          <td>
            Rohan Maddamsetti, Irida Shyti, Maggie L. Wilson, Hye-in Son, Yasa Baig, Zhengqing Zhou, Jia Lu, Lingchong You
          </td>
          <td>2025-07-02</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa29e8cd8f6fdd768b529f92b9275ea04ce76dc" target='_blank'>
              Genetic landscape of cancer: mechanisms, key genes, and therapeutic implications.
              </a>
            </td>
          <td>
            Arun Karnwal, Joydeep Dutta, Aqueel-Ur-Rehman, Abdel Rahman Mohammad Said Al-Tawaha, Natalia Nesterova
          </td>
          <td>2025-08-17</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Repairing programmed DNA double-strand breaks (DSBs) is crucial in the lifecycle of Paramecium tetraurelia, especially during its sexual reproduction phase when its somatic polyploid macronucleus is lost. The formation of a new macronucleus involves programmed genome rearrangements, introducing DNA DSBs at ∼45 000 loci. Paramecium tetraurelia employs a non-homologous end joining (NHEJ) mechanism for the faithful repair of these DSBs. There are four genes encoding DNA polymerases of family X in the genome, one of which was found recently to colocalize with other NHEJ proteins in the nucleus. Here we have characterized all four enzymes and shown that they are generally very faithful. They fall into two functional classes that may specialize in the distinct repair contexts encountered during DSB DNA repair. Biochemical assays, site-directed mutagenesis, and X-ray structures of mutants of human Polλ incorporating sequence determinants from P. tetraurelia PolX or metazoan Polβ are used to investigate the origin of their fidelity. Our findings suggest that Paramecium PolX enzymes may represent evolutionary intermediates between metazoan Polβ and Polλ. A general classification of DNA PolXs based on clustering methods indicates that our results can be generalized to plant DNA PolXs (Polλ-like) involved in DSB DNA repair generated by CRISPR–Cas9 engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e54e53d0ac06ac21d27fa1e914141a18179e83e" target='_blank'>
              Fidelity, specialization, and evolution of Paramecium PolX DNA polymerases involved in programmed double-strand break DNA repair
              </a>
            </td>
          <td>
            Nourisson Antonin, Missoury Sophia, Lucas-Staat Soizick, Haouz Ahmed, Delarue Marc
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary This study examined unclassified chromosomal abnormalities (UnCA) in patients with Hodgkin’s lymphoma (HL) before, during, and after anticancer treatment. HL is a rare cancer affecting 2–4 people per 100,000 individuals annually. Although treatment is highly effective, survivors face an increased risk of developing other cancers. In this study, we analyzed blood samples from HL patients and survivors at different stages of treatment, focusing on changes in their chromosomes that do not fit standard categories. These changes include alterations in chromosomes and nuclei. We found that UnCA levels were higher in HL patients than in healthy individuals at all stages; the general UnCA frequency decreased during treatment but increased again one year after treatment, at the expense of two specific UnCAs, free chromatin and micronuclei clusters. Given their persistence and association with genomic instability, these UnCAs might serve as early indicators of secondary cancers in HL survivors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb5a19d761a4166c2c377dcfbbe9dbb9f00192f1" target='_blank'>
              Unclassified Chromosomal Abnormalities as an Indicator of Genomic Damage in Survivors of Hodgkin’s Lymphoma
              </a>
            </td>
          <td>
            S. Ramos, B. Molina, María Del Pilar Navarrete-Meneses, D. Cervantes-Barragan, Valentín Lozano, S. Frias
          </td>
          <td>2025-07-23</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Transposable elements (TEs) are dynamic repetitive regions constituting 50% of the genome. In adult cancers, TEs contribute to carcinogenesis through structural rearrangements that disrupt tumor suppressors and by onco-exaptation, which drives aberrant oncogene expression. TEs are characterized in adult malignancies, with epithelial cancers harbouring significantly more new insertions. However, the role of TEs in pediatric cancer is unknown. Many pediatric cancers arise from predisposition, such as Li-Fraumeni Syndrome, which is caused by germline TP53 (gTP53) variants. TP53 suppresses TEs by binding L1 elements to inhibit transcription and by inducing apoptosis in cells with high retrotransposition. Moreover, adult cancers with somatic TP53 variants harbour significantly more TEs than wildtype tumors. To investigate the pediatric context, we conducted a pan-cancer analysis of germline and tumor TE landscapes in pediatric patients with (n = 66 germline; n = 40 tumor) and without (n = 465 germline; n = 276 tumor) gTP53 variants. Germline TEs were identified using MELT, xTEA, and INSurVeyor, while tumor TEs were called with xTEA and TotalReCall. TEs were consolidated across individuals using JASMINE and annotated with AnnotSV. We excluded 21% of germline and 2% of tumor TEs classified as common, defined as present in over 3% of gnomAD or pediatric non-cancer controls (n=189). Overall, 11% of pediatric tumors harbored at least one L1 insertion, with a median of 0 insertions per tumor, substantially lower than the 21% reported in adult cancers using the same TE callers (Solovyov et al). Frequency in germline or tumor TEs did not vary significantly by tumor type, age of onset, sex, prior treatment, or disease state. Germline and tumor TEs were found in 35 and 147 cancer genes respectively. Tumor TEs significantly affected many cancer pathways, including those involved in mitosis. While no structural variants (SVs) involved TEs at both breakpoints, 171 SVs in six samples were associated with a TE at one breakpoint. Importantly, both germline and tumor TE burden did not differ between gTP53 carriers and wildtype individuals, nor did the number of evolutionarily young or full-length elements. Pathway enrichment analyses revealed no significant differences in TE-affected pathways between gTP53 and wildtype groups. However, a gradient-boosted tree model trained on TE distribution across genomic windows predicted gTP53 status with an AUPRC of 0.77, suggesting that gTP53 variants shape the germline TE landscape at a positional level rather than through global frequency changes. In summary, pediatric cancers exhibit a quiescent TE landscape, with low insertion rates across tumor types. gTP53 variants do not alter the germline or tumor TE frequency,but influence the location of germline insertions. TE analysis in pediatric cancer and gTP53 carriers will enhance our understanding of tumorigenesis, informing future treatment approaches.



 Brianne Laverty, Shilpa Yadahalli, Safa Majeed, Ashby Kissoondoyal, Alexander Solovyov, Scott Davidson, Yisu Li, Mehdi Layeghifard, Adam Shlien, Vallijah Subasri, David Malkin. Pediatric pan-cancer characterization of transposable elements and their modulation by germline TP53 variants [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A039.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ba8aeceb59adfe162dadfd953074aed2a705a85" target='_blank'>
              Abstract A039: Pediatric pan-cancer characterization of transposable elements and their modulation by germline TP53 variants
              </a>
            </td>
          <td>
            B. Laverty, Shilpa Yadahalli, Safa Majeed, Ashby Kissoondoyal, Alexander Solovyov, Scott Davidson, Yisu Li, M. Layeghifard, A. Shlien, Vallijah Subasri, D. Malkin
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Cancer arises from a convergence of genetic mutations and epigenetic alterations. Among the consequences of epigenetic deregulation is the aberrant activation of transposable elements, particularly retrotransposons, the most expansive and functionally impactful class. This review examines the paradoxical role of retrotransposons in cancer, emphasizing their dual capacity to support oncogenesis while simultaneously triggering innate and adaptive immune responses that may limit tumor growth. A central question is discussed: Is retrotransposon activation merely a stochastic byproduct of epigenetic instability, or could it represent an intrinsic layer of early tumor surveillance that inadvertently turns against the host as malignancy progresses? Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00368-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7a2bf5f051e3bd6c0fa28e6e14cd088f5983472" target='_blank'>
              A plan or pandemonium? The conundrum of retrotransposon activation in cancer
              </a>
            </td>
          <td>
            Elina Zueva
          </td>
          <td>2025-08-09</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="As DNA variants accumulate in somatic stem cells, become selected or evolve neutrally, they may ultimately alter tissue function. When, and how, selection occurs in homeostatic tissues is incompletely understood. Here, we introduce SCIFER, a scalable method that identifies selection in an individual tissue, without requiring knowledge of the driver event. SCIFER also infers self-renewal and mutation dynamics of the tissue’s stem cells, and the size and age of selected clones. Probing bulk whole-genome sequencing data of nonmalignant human bone marrow and brain, we detected pervasive selection in both tissues. Selected clones in hematopoiesis, with or without known drivers, were initiated uniformly across life. In the brain, we found pre-malignant clones with glioma-initiating mutations and clones without known drivers. In contrast to hematopoiesis, selected clones in the brain originated preferentially from childhood to young adulthood. SCIFER is broadly applicable to renewing somatic tissues to detect and quantify selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6381edb3d3b2ef9cddaaae818d10d5852bbd41f" target='_blank'>
              Detecting and quantifying clonal selection in somatic stem cells
              </a>
            </td>
          <td>
            Verena Körber, N. A. Jakobsen, N. Ansari-Pour, Rachel Moore, Nina Claudino, M. Metzner, Eva Thielecke, Franziska Esau, B. Usukhbayar, Mirian Angulo Salazar, Simon Newman, B. Kendrick, Adrian H Taylor, Rasheed Afinowi-Luitz, R. Gundle, B. Watkins, Kim Wheway, D. Beazley, S. Dakin, Antony Palmer, Andrew J. Carr, Paresh Vyas, Thomas Höfer
          </td>
          <td>2025-07-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Oncogenic p53 mutations (Onc-p53) are frequent in lung and many other solid tumors often associated with chromosome aberrations. Why cells with Onc-p53 develop chromosomal aberrations and whether the abnormalities contribute to tumor growth remain elusive. Evidence in this communication demonstrate for the first time that replication stress induced by Onc-p53 triggers re-copying of DNA replication forks, which generates replication intermediates that cause persistent mitotic aberration and DNA segregation errors. Replication intermediates from re-copied replication forks induced by Onc-p53 activate ATM signaling, which stabilizes Onc-p53, reinforces its ability to upregulate replication factors for sustaining replication stress, thus generating a feedforward cycle accelerating tumor formation. In agreement with this observation our time lapse video microscopy show in real time that persistent mitotic aberration and DNA segregation errors induced by Onc-p53 confer selective growth advantage. Accordingly, human lung tumors with Onc-p53 show selection of cells with mitotic aberration during serial passages. Knock down of active replication forks reduces re-copied fork generation by Onc-p53 and specifically induces apoptotic death of lung cancer cells expressing Onc-p53 in xenograft lung tumors synergistically in cooperation with inhibitors of ATM activation, deselecting cells with Onc-p53 with mitotic errors. This communication reveals a novel mechanism which interconnects replication stress induced by Onc-p53 to its stabilization and ability to generate chromosomal aberration in lung cancer cells that both accelerate tumor growth and serve as a targetable therapeutic vulnerability. These findings will be extremely valuable for tumor-specific treatment of a high percentage of cancer patients with p53 mutation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b77d562f308b7a32a1524622e9f1f8039e4924f9" target='_blank'>
              Oncogenic p53 induces mitotic errors in lung cancer cells by recopying DNA replication forks conferring targetable proliferation advantage
              </a>
            </td>
          <td>
            S. Deb, Shilpa Singh, Lilia Gheghiani, R. Frum, Steven Grossman, Brad Windle, S. Deb
          </td>
          <td>2025-08-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background The selection of replication origins is a defining characteristic of DNA replication in eukaryotes, yet its mechanism in humans has not been well defined. Potential DNA replication origins are licensed through the recruitment of a pair of minichromosome maintenance complexes (MCMs). In yeast, a subset of MCMs is selected for initiation by the SLD3–SLD7 firing factors during G1. Like in yeast, excessive numbers of MCM complexes are loaded onto chromatin during G1 in human cells, but it is unclear how MCMs are selected for firing. Results We examine genomic binding locations for TICRR/TRESLIN and MTBP, the human orthologs for the yeast replication initiation factors Sld3 and Sld7. TRESLIN and MTBP binding patterns are detectable in a G1 synchronized population and have a higher binding signal compared to asynchronously cycling cells. MTBP binds to early-mid replicating regions in an early S population. Our data suggest MTBP is dependent on TRESLIN for proper association with chromatin during G1 but not S phase. We show that TRESLIN and MTBP binding during G1 does not require origins licensed with loaded MCMs. Conclusions We provide evidence for a chromatin binding mechanism of TRESLIN-MTBP during G1 that is dependent on TRESLIN and does not require interactions with licensed origins. MTBP binding location and footprint during S phase differs from that seen in G1, implicating two separate modes of binding. These data highlight binding mechanisms for DNA replication initiation factors in human cells that have diverged from those shown in yeast, suggesting differences in origin selection. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03665-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b74245a8a7e7194eb9255132a18f436ef65a462" target='_blank'>
              Cell cycle-dependent TICRR/TRESLIN and MTBP chromatin binding mechanisms and patterns
              </a>
            </td>
          <td>
            Tyler D. Noble, Courtney G. Sansam, Kimberlie A. Wittig, Blanka Majchrzycka, Christopher L. Sansam
          </td>
          <td>2025-07-07</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="With age, clonal expansions occur pervasively across normal tissues yet only in rare instances lead to cancer, despite being driven by well-established cancer drivers. Characterization of the factors that influence clonal progression is needed to inform interventional approaches. Germline genetic variation influences cancer risk and shapes tumor mutational profile, but its influence on the mutational landscape of normal tissues is not well known. Here we studied the impact of germline genetic variation on clonal hematopoiesis (CH) in 731,835 individuals. We identified 22 new CH-predisposition genes, most of which predispose to CH driven by specific mutational events. CH-predisposition genes contribute to unique somatic landscapes, reflecting the influence of germline genetic backdrop on gene-specific CH fitness. Correspondingly, somatic–germline interactions influence the risk of CH progression to hematologic malignancies. These results demonstrate that germline genetic variation influences somatic evolution in the blood, findings that likely extend to other tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bcdcfe442326883ec1ff1d1b7880290f079cb82" target='_blank'>
              Germline genetic variation impacts clonal hematopoiesis landscape and progression to malignancy
              </a>
            </td>
          <td>
            Jie Liu, Duc Tran, Liying Xue, B. Wiley, C. Vlasschaert, Caroline J. Watson, Hamish A J MacGregor, X. Zong, I. Chan, Indraniel Das, M. Uddin, A. Niroula, Gabriel Griffin, Benjamin L Ebert, T. Mack, Y. Pershad, B. Sharber, Michael F. Berger, A. Zehir, R. Ptashkin, Ross L. Levine, E. Papaemmanuil, V. Joseph, Teng Gao, Y. Kemel, D. Mandelker, Konrad H Stopsack, Paul D. P. Pharoah, Semanti Mukherjee, Li Ding, Yinyin Cao, MJ Walter, Jamie R Blundell, Nilanjan Chatterjee, K. Offit, Lucy A. Godley, Daniel C. Link, Z. Stadler, Alexander G. Bick, P. Natarajan, Kelly L Bolton
          </td>
          <td>2025-07-15</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>132</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) plays a significant role in cancer development. However, our understanding of its role in normal cells is limited. This study aimed to explore the roles of eccDNA in the differentiation of human bone marrow mesenchymal stem cells (BMSCs). Methods Through circular DNA sequencing (Circle-seq) and computational analysis, we documented eccDNAs in human BMSCs and their differentiation into osteoblasts, adipocytes, and chondrocytes. Next, the assay for transposase-accessible chromatin with high throughput sequencing (ATAC-seq) and RNA sequencing (RNA-seq) data were integrated with Circle-seq data. The roles of phosphatidylinositol 4-kinase alpha (PI4KA) and eccDNA as functional enhancers in human BMSC differentiation were assessed in vitro. Results Our results demonstrate that eccDNA is common in human BMSCs. In the differentiated groups including osteoblasts, adipocytes and chondrocytes, eccDNA-encoded genes presented higher expression compared to the background. We found eccDNA displayed enhanced chromatin accessibility in human BMSCs, which contribute to increased transcription of genes, such as runt-related transcription factor 2 (RUNX2), a key transcription factor for BMSC osteogenesis. We further found PI4KA, an eccDNA-encoded gene, promoted osteogenic differentiation of human BMSCs via the PI3K/AKT/mTOR pathway. Finally, we demonstrated eccDNA as functional enhancers to regulate BMSC differentiation in a dose-dependent manner. Conclusions Our study provides the evidence of eccDNA regulating BMSC differentiation and highlights the roles of eccDNA as transcription template and enhancer in normal cells, which will facilitate future research and clinical applications. Supplementary Information The online version contains supplementary material available at 10.1186/s13287-025-04516-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9298ef8d097b3fec00680e4438db26b85d21b6" target='_blank'>
              Extrachromosomal circular DNAs in the differentiation of human bone marrow mesenchymal stem cells
              </a>
            </td>
          <td>
            Yuxi Gu, Yidan Song, Shuhua Wang, Jun Liu
          </td>
          <td>2025-07-18</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: KRAS is one of the most frequently mutated oncogenes in humans. KRAS aberrations play a significant role in various solid tumors, affecting patient prognosis and treatment outcomes. Objectives: We identified features of genetic alterations in KRAS, including single amino acid substitutions and amplifications, based on the results of next-generation sequencing tests in 1667 advanced solid tumor patients. Design: Retrospective cohort study. Methods: Among 1667 patients, 28.1% (N = 468) had KRAS aberrations (single-nucleotide variant (SNV): N = 438, 26.1%; amplification (copy number variation): N = 48, 2.9%) in metastatic solid tumors. Results: The incidence rate of SNVs was higher in pancreatic cancer (N = 89, 89.9%) than in other tumors, including colorectal cancer (N = 259, 47.3%) and small bowel cancer (N = 4, 40.0%). Most of the mutations were missense mutations (N = 434, 99.1%). In addition, we examined the specific location of SNVs; the mutational type G12D (N = 178, 40.6%) was the most frequent, followed by G12V (N = 95, 21.7%) and G13D (N = 47, 10.7%). In the survival analysis, the mutational types of G12V and G13D influenced the poor survival of patients (G12V, the mutant type; 245 vs the wild type; 531 days; G13D, the mutant type; 435 vs the wild type; 531 days). Conclusion: In patients with KRAS amplification, the copy number range varied among the tumor types. Bladder cancer (147.9), cholangiocarcinoma (16.2), and gastric cancer (11.0) had relatively high median copy numbers of KRAS. Overall, our data are expected to provide valuable information for patients with various metastatic solid tumors and KRAS aberrations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c7721ce0c12159b0e8741b55a39a7ada4755088" target='_blank'>
              Comprehensive analysis of KRAS aberrations in pan-cancer, with a focus on prognostic and therapeutic implications
              </a>
            </td>
          <td>
            Jaeyun Jung, Jung Yong Hong, Se Hoon Park, Joon-Oh Park, Y. Park, Ho Yeong Lim, Won-Ki Kang, Jeeyun Lee, S. Ha, Soomin Ahn, Sung Hee Lim, Seung-Tae Kim
          </td>
          <td>2025-08-17</td>
          <td>Therapeutic Advances in Medical Oncology</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Genomic repetitive elements are essential structural and functional components of most eukaryotes and have attracted the attention of researchers in recent years. Despite the well-documented features of the karyotype of the tapeworm species Caryophyllaeus laticeps, knowledge about the content and distribution of repetitive elements on its chromosomes is limited. To gain a more comprehensive understanding of the composition of the repetitive DNA, we used a combination of bioinformatic, cytogenetic and molecular techniques to characterise the repetitive DNA content of this species. RepeatExplorer2 analysis revealed that transposable elements are predominantly present in the C. laticeps genome. The most suitable repeat families were selected for fluorescence mapping on the chromosomes, and most of them showed a scattered hybridisation pattern. In this study, we identified three new chromosomal markers: small nuclear U1 DNA (snDNA), the 5S ribosomal DNA (5S rDNA) and a satellite DNA named ClatSat14-167, each of which hybridises to a single locus at different chromosomal locations, allowing accurate identification of three pairs of chromosomes. These markers could be very valuable tools for future evolutionary studies and greatly enrich the limited cytogenetic data for tapeworms. In addition, an increased frequency of chromosomal aberrations was detected, reaffirming previous findings on the possible adverse effects of environmental pollution on the chromosomes of the species studied. Supplementary Information The online version contains supplementary material available at 10.1007/s00436-025-08530-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f4e24b509116b93b7418ab18f10e22249cbe4d2" target='_blank'>
              Discovery of new chromosomal markers through repeatome analysis of Caryophyllaeus laticeps (Caryophyllidea)
              </a>
            </td>
          <td>
            Anna Marková, M. Orosová
          </td>
          <td>2025-07-18</td>
          <td>Parasitology Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Epigenetic modifications are heritable, reversible alterations that causally reshape chromatin architecture and thereby influence DNA repair without changing nucleotide sequence. DNA methylation, histone modifications and non-coding RNAs profoundly influence DNA repair mechanisms and genomic stability. Aberrant epigenetic patterns in cancer compromise DNA damage recognition and repair, therefore impairing homologous recombination (HR), non-homologous end joining (NHEJ), and base excision repair (BER) by suppressing key repair genes and lowering access to repair sites. Then it is dissected how loss-of-function mutations in Switch/Sucrose non-fermentable, imitation switch and CHD (Chromodomain helicase DNA-binding) chromatin-remodeling complexes impair nucleosome repositioning, preventing effective damage sensing and assembly of repair machinery. Non-coding RNAs contribute to epigenetic silencing at DNA break sites, exacerbating repair deficiencies. This review evaluates recent advances concerning epigenetic dysfunction and DNA repair impairment. It is also highlighted that nanoparticle-mediated delivery strategies are designed to overcome pharmacologic resistance. It is presented how epigenetic dysregulation of DNA repair can guide more effective and drug-resistant cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d922eca60c4fcba055a3683ed299a04149ee715f" target='_blank'>
              Epigenetic Dysregulation in Cancer: Implications for Gene Expression and DNA Repair-Associated Pathways
              </a>
            </td>
          <td>
            Nina Rembiałkowska, Katarzyna Rekiel, Piotr Urbanowicz, Mateusz Mamala, Karolina Marczuk, Maria Wojtaszek, Marta Żywica, Eivina Radzevičiūtė-Valčiukė, V. Novickij, J. Kulbacka
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Sporadic colorectal cancer (CRC), the third leading cause of cancer-related death globally, arises through a continuum from normal tissue to adenomas, progressing from low-grade (LGD) to high-grade dysplasia (HGD); yet, the early epigenetic drivers of this transition remain unclear. To investigate these events, we profiled LGD and HGD adenomas using EM-seq, and identified a consensus differential methylation signature (DMS) of 626 regions through two independent bioinformatics pipelines. This signature effectively distinguished LGD from HGD in both tissue and plasma-derived cell-free DNA (cfDNA), highlighting specific methylation patterns. Functional annotation indicated enrichment for regulatory elements associated with transcription factor activity and cell signaling. Applying the DMS to the TCGA CRC dataset revealed three tumor subtypes with increasing hypermethylation and one normal cluster. The most hypermethylated subtype exhibited poor survival, high mutation burden, and disrupted transcriptional networks. While overlapping with classical CpG Island Methylator Phenotype (CIMP) categories, the DMS captured a broader spectrum of methylation alterations. These findings suggest that the DMS captures functionally relevant, antecedent epigenetic alterations in CRC progression, enabling the robust stratification of dysplasia severity and tumor subtypes. This signature holds promise for enhancing preclinical detection and molecular classification, and warrants further evaluation in larger prospective cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c27b0ef917c9876876a5f66fb35ed8c368fc721c" target='_blank'>
              Multiomics Signature Reveals Network Regulatory Mechanisms in a CRC Continuum
              </a>
            </td>
          <td>
            Juan Carlos Higareda-Almaraz, Francesco Mattia Mancuso, P. Canal-Noguer, Kristi Kruusmaa, Arianna Bertossi
          </td>
          <td>2025-07-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Somatic mutations are desirable targets for selective cancer elimination, yet most are found within noncoding regions. We have adapted CRISPR-Cas9 as a novel, cancer-specific killing strategy by targeting the subset of somatic mutations that create protospacer adjacent motifs (PAMs), which have evolutionally allowed bacterial cells to distinguish between self and non-self DNA for Cas9-induced double strand breaks (DSBs). We hypothesized that somatic 5’-NGG-3’ PAMs arising from base substitutions (e.g. NCG→NGG) could serve as CRISPR-Cas9 targets for selective cancer elimination through the induction of multiple DSBs. Whole genome sequencing (WGS) of paired tumor-normal (T-N) samples from three PC patients were put through PAMfinder (software for somatic PAM discovery). On average, 417 somatic PAMs per PC case were found (>99% of them in non-exons) and 97% of the selected subset were Sanger confirmed. We then analysed T-N samples from ICGC which revealed medians of 455 somatic PAMs per tumor in two PC cohorts (N=174 combined), 2790 in lung (N=29), and 3236 in esophageal (N=388) cancer cohorts. We performed γH2AX staining on PC cell lines transduced with 4-9 sgRNAs designed from our PAM discovery approach, and observed more foci with increased number of sgRNAs in the target cell line but not in the non-target cell line. Deep sequencing and WGS on clones transduced with cancer-specific sgRNAs also demonstrated on-target activity in the target cell line but not in either the patient’s normal cells or an irrelevant cancer cell line. No off-target events were detected. To test selective toxicity in vitro and in vivo, we transduced 4-9 sgRNAs into PC co-cultures and found 69-99% selective cell death of three target cell lines through flow cytometry. Finally, using mouse xenograft models, 92% weight reduction (N=10, P=0.0004) of tumors transduced with a pool of 9 cancer-specific sgRNAs was achieved compared to tumors transduced with non-targeting sgRNAs (N=8).



 Selina Shiqing K. Teh, Kirsten Bowland, Alexis Bennett, Eitan Halper-Stromberg, Akhil Kotwal, Hong Liang, Sarah Wheelan, Chien-Fu Hung, Michael Goldstein, Nicholas J. Roberts, and James R. Eshleman. CRISPR-Cas9 as a Novel Cancer Gene Therapy Approach against Pancreatic Cancers [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr LT02.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b50e15f1763d831356c9974d95d4708f8806a3e" target='_blank'>
              Abstract LT02: CRISPR-Cas9 as a Novel Cancer Gene Therapy Approach against Pancreatic Cancers
              </a>
            </td>
          <td>
            S. Teh, Kirsten Bowland, Alexis Bennett, Eitan Halper-Stromberg, Akhil Kotwal, Hong Liang, Sarah Wheelan, Chien-Fu Hung, M. Goldstein, Nicholas J. Roberts, James R. Eshleman
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b369f79b329c117da716bed78cecbcb615823b8f" target='_blank'>
              JAK2 signalling to HNRNPA1 represses retrotransposon activity in haematopoietic stem cells
              </a>
            </td>
          <td>
            Jeremy W. Deuel, Andrejs Makarovs, Sheidasadat Goosheh, George Winder, Hisham Mohammed, Jason S. Carroll, Michael Spencer Chapman, C. Kapadia, Peter J. Campbell, J. Nangalia, E. Laurenti, C. Schneider, T. Oellerich, Anthony R Green, S. Loughran
          </td>
          <td>2025-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Genetic lesions drive cancer development and progression, and understanding their origins will reveal the mechanisms of carcinogenesis. We showed that murine FAM72A promotes mutagenic DNA repair during antibody maturation by acting as a substrate adaptor of the CTLHMKLN1 E3 ligase to induce the proteasome degradation of Uracil DNA glycosylase 2 (UNG2), a pivotal enzyme of the base excision repair. In humans, the FAM72 gene has expanded to include four paralogues named FAM72A-D. Bioinformatic studies suggested that the human FAM72 genes are overexpressed in a broad range of cancers. However, the functional roles of FAM72A-D in human biology and cancer are unknown. Here, we show that FAM72 family members are minimally expressed in most healthy tissues except for thymus, and that FAM72A, B and D are overexpressed in primary tumorigenic tissues. Human FAM72 expression inversely correlates with UNG2 protein level in human cell lines and primary tumorigenic tissues suggesting that human FAM72 promotes UNG2 degradation. However, only FAM72A is able to bind to and induce UNG2 degradation in human cells. Our results suggest that the ability of FAM72A to induce UNG2 degradation contributes to neoplasia in a variety of cancer types by promoting mutagenic repair of genomic dUs. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-07723-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369de028cb26608e44c32340a1371447a2c75ba9" target='_blank'>
              FAM72A promotes UNG2 degradation and mutagenesis in human cancer cells
              </a>
            </td>
          <td>
            Yuqing Feng, P. Barbulescu, Chetan K Chana, Melika Shirdarreh, Hong Yang, Lizhen Wu, Sami Mamand, Mohammad Kashem, Amin Zia, Ming Han, Julissa Tsao, Trevor Pugh, David Cescon, David G. Schatz, Frank Sicheri, Alberto Martin, R. Pezo
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) encompasses tumors arising in the colon (CC) and rectum (RC), often treated as a single disease despite emerging evidence of biological divergence. Understanding the molecular differences between CC and RC is critical for improving diagnosis, prognosis, and therapeutic strategies. Methods We performed an integrated genomic and transcriptomic analysis of CC and RC data from The Cancer Genome Atlas (TCGA) to investigate their degree of similarity and observed that these tumors present distinct molecular profiles, which suggest an evolution through divergent pathways. Comparative analyses included copy number alterations (CNAs), somatic mutations, driver gene prediction, differential gene expression, pathway enrichment, and survival analysis. Results Chromosomal analyses revealed that 43% of focal and 77% of large-scale CNAs were specific of CC, while 10.5% and 57% were specific of RC with 8% of mutant genes unique to CC and 0.18% to RC. CC and RC presented distinct profiles of gene mutations, with CC showing significantly higher tumor mutational burden (0.51 muts/Mb vs 0.28 muts/Mb in RC). Distinct mutational signatures were identified, with CC characterized by a higher frequency of PIK3CA, BRAF, and DNAH1 mutations, while RC showed enrichment for TP53 and NRAS mutations. Importantly, analysis of predicted non-canonical driver genes identified ACVR1B, LTBP4, SETD1A as CC-specific drivers and C4BPA, EHD1 as RC-specific drivers, underscoring divergent oncogenic mechanisms. However, the most substantial divergence was observed in transcriptomic profiling, with 56% and 33% of DEGs (in CC and RC, respectively) that were tumor-type specific. Notably, RC tumors segregated into two distinct transcriptional subtypes (Cluster 1 and Cluster 2), with Cluster 1 showed a more heterogeneous Consensus Molecular Subtypes (CMS) distribution, while Cluster 2 enriched in CMS4 (mesenchymal) and CMS3 (metabolic) consensus molecular subtypes. Accordingly, Gene Set Enrichment Analysis revealed CC-specific upregulation of Wnt, MYC, and mTOR signaling pathways, and RC-specific enrichment of GPCR and neuronal development pathways. On the other hand, pseudogene expression was significantly higher in CC, suggesting differential mechanisms of transcriptional dysregulation. Finally, we identified an RC-specific multigene survival signature as a prognostic model involving upregulation of C2CD4B, HSPD1P1, LINC01356, CBX3P9, GATA2-AS1 and downregulation of ATP5F1EP2, HSP90AB3P and SNRPFP1. Conclusions Collectively, our findings provide robust molecular evidence that CC and RC follow divergent oncogenic pathways, emphasizing the need for site-specific biomarker development and therapeutic targeting in colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06908-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab7228806c5ae698ac668178d1fa7e354b1b537" target='_blank'>
              Distinctive chromosomal, mutational and transcriptional profiling in colon versus rectal cancers
              </a>
            </td>
          <td>
            Maria Teresa De Angelis, Antonia Rizzuto, Angela Amaddeo, Carlo Sagnelli, Niccolò Vono, Michela Reda, Valentina Lise, Luca Parrillo, Carmela de Marco, Donatella Malanga, Gianluca Santamaria, Giuseppe Viglietto
          </td>
          <td>2025-08-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Ovarian high-grade serous carcinoma (HGSC) is characterized by pervasive genomic instability and high inter- and intra-tumor heterogeneity. Approximately half of HGSC tumors harbor homologous recombination deficiency (HRD), rendering them vulnerable to PARP inhibitors and platinum-based chemotherapy. In contrast, patients lacking HRD (HR-proficient, HRP) generally respond poorly to the current therapies. To overcome heterogeneity and identify relevant HGSC subtypes, we characterized the genomic landscape of 640 tumors from 243 patients using whole-genome sequencing. Our chromosomal instability (CIN) signature-based analysis characterized the structural variation landscape and revealed five HGSC subtypes, validated in an independent dataset. Two HRD subtypes, associated with BRCA1- or BRCA2-driven alterations, demonstrated favorable treatment responses. Strikingly, three HRP subtypes emerged, marked by unique structural alterations and gene expression patterns, tumor microenvironment interactions, and different chemotherapy responses. Finally, organoid experiments showed subtype-specific sensitivity to CHK1 inhibition, suggesting prexasertib as a potential targeted treatment for most currently untreatable HRP patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57e62a189c25cb112512c18bbcd2dda397ed67ef" target='_blank'>
              Decoding the Genomic and Functional Landscape of Emerging Subtypes in Ovarian Cancer.
              </a>
            </td>
          <td>
            Giulia Micoli, K. Lavikka, Yilin Li, A. Pirttikoski, D. Afenteva, Wojciech Senkowski, G. Marchi, Anna Vaharautio, T. Muranen, Titta Joutsiniemi, S. Hietanen, A. Virtanen, K. Wennerberg, J. Hynninen, J. Oikkonen, S. Hautaniemi
          </td>
          <td>2025-07-31</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Abstract Aurantiochytrium limacinum holds great promise for producing sustainable single-cell oil as an alternative to fish oil. However, research into its complex biological and biochemical characteristics and efforts toward strain improvement have been hampered by insufficient genetic tools. Until now, genetic transformations of A. limacinum have relied solely on chromosome integration, which is inefficient and prone to insertional mutagenesis and other issues related to genetically modified organisms (GMOs). This paper describes the first centromeric plasmid for A. limacinum. Amplification of this shuttle vector by E. coli enables direct delivery into A. limacinum via electroporation, where it undergoes stable replication and segregation into daughter cells. The key to the stable plasmid maintenance lies in a 500 bp segment derived from chromosome 24 of Phaeodactylum tricornutum. While this segment does not significantly enhance the efficiency of vector transformation, it enables the replication and maintenance of the shuttle vector in the host cell as closed circular DNA. The plasmid from three transformants demonstrates a high segregation efficiency of 96.8 ± 0.3% (n = 3), even in the absence of antibiotic selection. This novel centromeric plasmid considerably enhances the flexibility of genetic manipulations and gene expression in A. limacinum, opening new avenues for its study and industrial application. Key points • First centromeric plasmid developed for genetic transformation in A. limacinum. • The novel plasmid enhances flexibility in genetic manipulation and gene expression. • The plasmid achieves 96.8 ± 0.3% (n = 3) segregation efficiency without antibiotic selection. Supplementary Information The online version contains supplementary material available at 10.1007/s00253-025-13527-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e1a22ec96ed65caa34d28f985f7648b9334af03" target='_blank'>
              Novel centromeric plasmid for stable extrachromosomal gene expression in Aurantiochytrium limacinum
              </a>
            </td>
          <td>
            P. P. Renta, Cian-Huei Syu, Ta-Yu Huang, Yi-Ting Chang, Yu-Feng Liang, Ssu-Ting Chen, Po-Wei Weng, Ming-Chen Hsu, Keng-Hung Lin, Tsu-tung Liu, Anna C-C Jang, Che-Chia Tsao, Han-Jia Lin, Hung-Yun Lin, Yi-Min Chen
          </td>
          <td>2025-07-04</td>
          <td>Applied Microbiology and Biotechnology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Gastric cancer (GC) is classified into four molecular subtypes according to the Epstein-Barr virus-positive status, microsatellite instability, genomic stability and chromosomal instability (CIN). The CIN subtype is characterized by a high frequency of gene amplifications in receptor tyrosine kinases (RTKs) and a poor prognosis. In addition, the CIN subtype often exhibits intratumoral heterogeneity and indicates insensitivity to targeted drugs. Elucidating the molecular mechanisms of CIN in GC is therapeutically crucial; however, the molecular mechanisms involved are not yet fully understood. The kinetochore-associated 1 (KNTC1) gene encodes kinetochore-associated protein 1 (KNTC1), a major component of the outer kinetochore. The downregulation of KNTC1 causes a high frequency of lagging chromosomes and consequent aneuploidy and CIN in Drosophila and Caenorhabditis elegans. However, the association between KNTC1 and CIN in GC has not yet been clarified. Therefore, the present study investigated the role of KNTC1 in GC CIN. It was found that GC cell lines with a high frequency of lagging chromosomes had a low KNTC1 mRNA expression. Notably, KNTC1 knockdown increased the frequency of lagging chromosomes in GC cell lines. In particular, GC cell lines with the amplification of RTK genes exhibited a significant increase in the frequency of lagging chromosomes. On the whole, the findings of the present study suggest that the suppression of KNTC1 expression may contribute to CIN in GC and may be involved in the generation of intratumoral genetic heterogeneity in GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa4f34e8d31ebf5a0db6caf271a09a3caf4a8164" target='_blank'>
              Downregulation of kinetochore-associated 1 gene increases lagging chromosomes and contributes to chromosomal instability in gastric cancer cells
              </a>
            </td>
          <td>
            Daiki Ohsaki, Kazuki Kanayama
          </td>
          <td>2025-08-04</td>
          <td>Medicine International</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Tumors represent dynamically evolving populations of mutant cells, and many advances have been made in understanding the biology of their progression. However, there are key unresolved questions about the conditions that support a cell’s initial transformation, which cannot be easily captured in patient populations and are instead modeled using transgenic cellular or animal systems. Results Here, we use extensive patient atlas data to define common features of the tumor DNA methylation landscape as they compare to healthy human cells and apply this benchmark to evaluate 21 engineered human and mouse models for their ability to reproduce these patterns. Notably, we find that genetically induced cellular transformation rarely recapitulates the widespread de novo methylation of Polycomb regulated promoter sequences as found in clinical samples, but can trigger global changes in DNA methylation levels that are consistent with extensive proliferation in vitro. Conclusions Our results raise pertinent questions about the relationship between genetic and epigenetic aspects of tumorigenesis as well as provide an important molecular reference for evaluating existing and emerging tumor models. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03650-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b654b38672fde0e6d98d2fd32826d3930c1b68a1" target='_blank'>
              Direct genetic transformation bypasses tumor-associated DNA methylation alterations
              </a>
            </td>
          <td>
            Sara Hetzel, Eran Hodis, Elena Torlai Triglia, Alexander Kovacsovics, Kathleen Steinmann, A. Gnirke, Meiying Cui, Daniel McQuaid, Raha Weigert, Georg Pohl, M. Muzumdar, Serge Leyvraz, Ulrich Keilholz, M. Yaspo, Aviv Regev, H. Kretzmer, Zachary D. Smith, Alexander Meissner
          </td>
          <td>2025-07-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Neuroblastoma is a heterogeneous disease where patient stratification is critical for prognosis and treatment decisions and where it recently has been suggested that the presence of telomere maintenance mechanisms (TMM) should be considered in risk stratification. We investigated the utility of DNA methylation-based classification for neuroblastoma diagnostics by analysing 303 tumours samples from two cohorts. We show that of the total number of cases, an average of 90% of the samples classified as neuroblastoma, while 66% also achieved confident classification into the three NB subclasses: “MYCN-type”, “ALT/TERT TMM positive” and “TMM negative”. The tumours classified as MYCN-type showed genomic amplification of MYCN (MNA); however, some MYCN-type cases lacked evident MNA, suggesting that epigenetic states might be influenced by other factors such as activating ALK mutations. Survival analysis indicated similar poor survival probabilities for patients classified as TMM positive or MYCN type, distinct from the inferior survival of TMM-negative cases. All cases, with complementary genomic data available, associated with TMM positivity also presented features associated with telomere lengthening mechanisms, including TERT or ATRX alteration. However, some tumours positive for these features, especially TERT rearrangement, classified as MYCN type rather than TMM positive, indicating that MNA and other mechanisms introduce a methylation pattern that supersede or overlap with pattern imposed by TERT. Chromosomal copy number alterations (CNAs) characteristic of methylation subclasses were identified, including 1p deletion and 17q gain in MYCN type and combinations of 11q loss, 3p loss, 7q gain, and 17q gain in TMM-positive cases, highlighting the potential of the methylation arrays to replace SNP arrays for prognostic genomic assessments. Our study demonstrates that DNA methylation-based classification stratifies neuroblastoma into clinically relevant subgroups, aiding diagnostic and prognostic decisions, although discrepancy between genomic features and methylation classification does occur. The interplay between genomic alterations and methylation patterns could give clues into the discrepancy and underscores the complexity of neuroblastoma biology and the need for further research and validation of clinical outcomes of the patients in the respective subclasses. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01936-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43f713d2afc90db793e049e383b6f7ea1cd34f79" target='_blank'>
              DNA methylation profiling for molecular classification of neuroblastoma
              </a>
            </td>
          <td>
            Maja Löfgren, A. Djos, Shiva Rezaei, Medha Suman, P. Kogner, Tommy Martinsson, S. Fransson, Helena Carén
          </td>
          <td>2025-07-27</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="A major goal in evolutionary biology and biomedicine is to understand the complex interactions between genetic variants, the epigenome, and gene expression. However, the causal relationships between these factors remain poorly understood. mSTARR-seq, a methylation-sensitive massively parallel reporter assay, is capable of identifying methylation-dependent regulatory activity at many thousands of genomic regions simultaneously and allows for the testing of causal relationships between DNA methylation and gene expression on a region-by-region basis. Here, we develop a multiplexed mSTARR-seq protocol to assay naturally occurring human genetic variation from 25 individuals from 10 localities in Europe and Africa. We identify 6957 regulatory elements in either the unmethylated or methylated state, and this set was enriched for enhancer and promoter chromatin annotations, as expected. The expression of 58% of these regulatory elements is modulated by methylation, which is generally associated with decreased transcription. Within our set of regulatory elements, we use allele-specific expression analyses to identify 8020 sites with genetic effects on gene regulation; further, we find that 42.3% of these genetic effects vary in direction or magnitude between methylated and unmethylated states. Sites exhibiting methylation-dependent genetic effects are enriched for GWAS and EWAS annotations, implicating them in human disease. Compared with data sets that assay DNA from a single European ancestry individual, our multiplexed assay is able to uncover more genetic effects and methylation-dependent genetic effects, highlighting the importance of including diverse genomes in assays that aim to understand gene regulatory processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d8c0aba589430fda6268dbe62323f10c40bdeec" target='_blank'>
              Uncovering methylation-dependent genetic effects on regulatory element function in diverse genomes
              </a>
            </td>
          <td>
            Rachel M. Petersen, Christopher M. Vockley, Amanda J. Lea
          </td>
          <td>2025-07-14</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7238ca759d86a62210ebf70bb830d0369e9ff71e" target='_blank'>
              Excised DNA circles from V(D)J recombination promote relapsed leukaemia.
              </a>
            </td>
          <td>
            Zeqian Gao, James N. F. Scott, Matthew P Edwards, Dylan Casey, Xiaoling Wang, Andrew D Gillen, S. Ryan, Lisa J Russell, Anthony V Moorman, Ruth de Tute, Catherine Cargo, Anthony M Ford, D.R. Westhead, J. Boyes
          </td>
          <td>2025-08-06</td>
          <td>Nature</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Abstract Phenotypic plasticity is a recognized mechanism of therapeutic resistance in prostate cancer (PCa), however current knowledge of driver mechanisms and therapeutic interventions are limited. Using genetically engineered mouse models (GEMMs) devoid of Pten and Rb1, we previously demonstrated the chromatin reprogramming factor enhancer of zeste homolog 2 (EZH2) as an important regulator of alternative transcription programs promoting phenotypic plasticity. Here, using a multi-omics approach we demonstrate that EZH2 regulates multilineage cell states dependent on the RNA binding protein Tristetraprolin (TTP) that mediates RNA stability and activation of translation. Combined chemical inhibition of EZH2 and PI3K/mTORC1 resulted in superior anti-tumor activity in murine and human phenotypic plastic models and was most significant when this combination was used with castration or enzalutamide. Together, these data indicate phenotypic plasticity dependence on coordination between EZH2, TTP and mTORC1 signaling that represent novel therapeutic dependencies for this lethal PCa phenotype. Graphical abstract Significance Statement EZH2 plays a key role in driving cell state transitions in neuroendocrine prostate cancer (NEPC), guiding cancer cells towards a more aggressive, therapy-resistant cell type. This transformation creates a specific vulnerability, as NEPC cells become highly reliant on both EZH2 and PI3K/mTORC1-translation signaling. Targeting this dependency, we demonstrate that combining EZH2 with PI3K/mTORC1 inhibition provides effective suppression of NEPC cell growth, offering a promising therapeutic strategy for treating this challenging and aggressive prostate cancer subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12b27ab9c67e47d28ae41dcca4d51f6a0aabec7f" target='_blank'>
              EZH2-TTP-mTORC1 Axis Drives Phenotypic Plasticity and Therapeutic Vulnerability in Lethal Prostate Cancer
              </a>
            </td>
          <td>
            Beatriz German, Katherine L. Morel, Teia Noel, N. Boufaied, Deborah L. Burkhart, Sujun Chen, Felipe Dezem, Xintao Qui, Henry W. Long, Stefan DiFazio, S. Baca, Ayesha A. Shafi, Matthew L. Freedman, Himisha Beltran, Christopher J. Sweeney, H. He, Myles Brown, Jasmine T. Plummer, Simon R V Knott, David P. Labbé, Leigh Ellis
          </td>
          <td>2025-08-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Background. Medulloblastoma (MB) prognosis and response to therapy depend largely on genetic changes in tumor cells. Many genes and chromosomal abnormalities have been identified as prognostic factors, including amplification of MYC oncogenes, gains in 1q and 17q, deletions in 10q and 21p, or isochromosomes 17 (i(17)(q10)). The frequency of these abnormalities varies greatly between ethnic populations, but the frequency of specific abnormalities, such as MYCC and MYCN amplification, 17q gain, and deletions, in the Russian population is unknown. Objective: The aim is to study the frequency of MYCC and MYCN amplifications, 17q gain, and 17p deletion and determine their prognostic value in Russian patients with MB. Methods. This study was performed on MB cells obtained from 18 patients (12 boys and 6 girls, aged between 3 months and 17 years, with a median age of 6.5 years). Determination of cytogenetic aberrations was carried out using FISH assays with MYCC-SO, MYCN-SO, and MYCN-SG/cen2 probes, as well as cen7/p53 dual color probes and PML/RARα dual color probes (Abbott Molecular, USA). One-way ANOVA and Fisher’s F-test were used to compare the two groups. The differences were considered significant when p < 0.05. Results. In 77.7% of patients (14/18), the classical type of MB was present; in 16.7% (3/18), desmoplastic type; and in 5.6% (1/18), nodular desmoplasic types of neoplasms. Amplification of MYC genes was detected in 22.2% of Russian patients (n = 4 out of 18). Patients with MYC amplification had the worst overall survival (OS: 0% vs. 68%, p = 0.0004). Changes on the 17th chromosome were found in 58.3% of patients. Deletion of 17p occurred in 23.1%, and gain of 17q occurred in 46.2%. There were no significant differences in OS, clinical signs, or the presence of additional 17q material or 17p deletion among patients with MB. Conclusions: Amplification of the MYC gene is a predictor of poor overall survival to therapy and a high risk of metastatic relapse. This allows us to more accurately stratify patients into risk groups in order to determine the intensity and duration of therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5f5e5b1eac08421c304a6da77fd1d4ce4707e64" target='_blank'>
              The Prognostic Value of Amplification of the MYCC and MYCN Oncogenes in Russian Patients with Medulloblastoma
              </a>
            </td>
          <td>
            A. Chernov, Ekaterina Batotsyrenova, Sergey N Zheregelya, S. Pyurveev, V. Kashuro, Dmitry Ivanov, Elvira Galimova
          </td>
          <td>2025-07-27</td>
          <td>Diseases</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 DNA repair and autophagy are distinct biological processes vital for cell survival. Although autophagy helps maintain genome stability, there is no evidence of its direct role in the repair of DNA lesions. We discovered that lysosomes process Topoisomerase 1-cleavage complexes (TOP1cc) DNA lesions in human cancer cell. TOP1 resolves DNA topological stress ahead of DNA replication and transcription. If not promptly repaired, TOP1cc hinder the progression of DNA replication and transcription fostering genomic instability. Thus, stabilising TOP1cc with TOP1 poisons, such as Camptothecin or its analogue, is highly effective and widely used in cancer therapy. Proteolysis of TOP1cc by the proteasome is described at excessively high doses of CPT that are not relevant in a clinical setting, calling into question the relevance of the proteasome role in TOP1cc repair for cancer therapy and patient response. Here we described that repair of TOP1cc induced by clinically relevant dose of Camptothecin depends on selective autophagy. Proteomics, sequencing, biochemistry techniques and live imaging all demonstrate the lysosomal uptake of TOP1cc upon induction of replication stress. Selective degradation of TOP1cc by autophagy directs DNA damage repair and cell survival. TOP1cc are exported from the nucleus to lysosomes through transient alteration of the nuclear envelope and independent of the proteasome. Mechanistically, the autophagy receptor TEX264 acts as a TOP1cc sensor at DNA replication forks, triggering TOP1cc processing by the p97 ATPase and mediating the delivery of TOP1cc to lysosomes in an MRE11 nuclease and ATR kinase-dependent manner. We found an evolutionarily conserved role for selective autophagy in DNA damage repair that enables cell survival, protects genome stability, and is clinically relevant for colorectal cancer patients.



 Pauline Lascaux, Gwendoline Hoslett, Sara Tribble, Ivan Antičević, Cecile Otten, Ignacio Torrecilla, Yichen Zhao, Wei Song, Cristiano Peron, Giulio Deangeli, Enric Domingo, James Bancroft, Loïc Carrique, Errin Johnson, Alvin Wei Tian Ng, Joanne Ngeow, Nuno Raimundo, Tim Maughan, Marta Popović, Ira Milošević, and Kristijan Ramadan. TEX264 DRIVES SELECTIVE AUTOPHAGY OF DNA LESIONS TO PROMOTE DNA REPAIR AND CELL SURVIVAL [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P11.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ada9e8b0a0e36cefc3f3e65806469a480324003" target='_blank'>
              Abstract P11: TEX264 DRIVES SELECTIVE AUTOPHAGY OF DNA LESIONS TO PROMOTE DNA REPAIR AND CELL SURVIVAL
              </a>
            </td>
          <td>
            Pauline Lascaux, Gwendoline Hoslett, Sara Tribble, Ivan Anticevic, Cecile Otten, Ignacio Torrecilla, Yichen Zhao, Wei Song, Cristiano Peron, Giulio Deangeli, Enric Domingo, James Bancroft, L. Carrique, Errin Johnson, Alvin Wei Tian Ng, Joanne Ngeow, Nuno Raimundo, Tim Maughan, Marta Popović, Ira Milošević, Kristijan Ramadan
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract G-quadruplexes (G4s) are prevalent at promoters and superenhancers (SEs), exclude nucleosomes, and recruit transcription factors. This study sought to determine whether the nucleosome exclusion affects the recruitment of the SE marker BRD4, which typically binds to acetylated histones and facilitates SE–promoter contacts via the phase separation-dependent mechanism. Analyses of the available whole-genome data revealed that SEs with the highest G4 density were depleted of nucleosomes but not of BRD4. This led us to test the possibility of histone-independent BRD4 maintenance at G4-rich SEs. A typical SE G4 destabilized a nearby nucleosome in vitro and, unlike B-DNA, bound weakly to BRD4 bromodomains. Similar to an acetylated nucleosome, the G4 promoted phase separation in BRD4 solutions. This effect was not altered by the histone competitor JQ1. However, it was attenuated by two known G4 ligands, suggesting that they could disrupt SE–promoter communication in cells. Consistently, these ligands downregulated several genes regulated by G4-rich SE-contacting promoters more efficiently than they did SE-independent genes. Our findings underscore the significance of G4-rich SEs as transcriptional regulators and provide new insights into their organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ca6b9e0a4bd9850a35faa9e07c7255af9d5b35c" target='_blank'>
              G-quadruplexes as potential traps for superenhancer marker BRD4: ligand-sensitive binding and co-separation in vitro
              </a>
            </td>
          <td>
            Iuliia Pavlova, Olga M. Ivanova, Mikhail S Iudin, Anastasiya V Surdina, Nikolay A Barinov, M. E. Bogomiakova, Sergey D Oreshkov, Z. Shenkarev, V. Severov, Dmitriy V Klinov, Victoria O Shender, A. Bogomazova, M. Lagarkova, A. Varizhuk, Vladimir B Tsvetkov
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes provided important insights into the molecular pathogenesis of cancer but also revealed some contradictions in the prevailing somatic mutation theory. The evidence that noncoding RNAs (ncRNAs) form an elaborate network that regulates the flow of genetic information in eukaryotic cells offers an explanation for the inconsistencies. ncRNAs comprise a wide variety of molecules that interact with one another as well as with other RNAs, DNA, and proteins, over whose activities they exert a regulatory influence. The outcome of the dynamic interactions of the cell’s biomolecules is the emergence of higher-order states of equilibrium, called attractor states, which correspond to the gene-expression configurations of distinct cell types. Attractor states are relatively stable systems, but they are susceptible to perturbation by a disturbing force, such as mutation. Mutations that disrupt the ncRNA network can enable the cell to undergo a state transition towards a potentially neoplastic one. This is the crux of tumorigenesis. An inquiry into the architecture of the ncRNA network and its role in tumorigenesis is required to complement our knowledge of the well-known cancer genes as well as serve as a guide in the design of new anticancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/751cbcc8f9af147a3e904b8dfcf8de5075149f58" target='_blank'>
              The Primary Role of Noncoding RNA in the Pathogenesis of Cancer
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Structural maintenance of chromosomes (SMC) complexes organize genomes by extruding DNA loops, while replisomes duplicate entire chromosomes. These essential molecular machines must collide frequently in every cell cycle, yet how such collisions are resolved in vivo remains poorly understood. Taking advantage of the ability to load SMC complexes at defined sites in the Bacillus subtilis genome, we engineered head-on and head-to-tail collisions between SMC complexes and the replisome. Replisome progression was monitored by genome-wide marker frequency analysis, and SMC translocation was monitored by time-resolved ChIP-seq and Hi-C. We found that SMC complexes do not impede replisome progression. By contrast, replisomes restrict SMC translocation regardless of collision orientations. Combining experimental data with simulations, we determined that SMC complexes are blocked by the replisome and then released from the chromosome. Occasionally, SMC complexes can bypass the replisome and continue translocating. Our findings establish that the replisome is a barrier to SMC-mediated DNA-loop extrusion in vivo, with implications for processes such as chromosome segregation, DNA repair, and gene regulation that require dynamic chromosome organization in all organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/686025c7ec346de8dd0c0dabfe2ed3d2aaa2a6a1" target='_blank'>
              Replisomes restrict SMC translocation in vivo
              </a>
            </td>
          <td>
            Qinchan Liao, Hugo B. Brandão, Zhongqing Ren, Xindan Wang
          </td>
          <td>2025-08-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0d94bb593ee184379443d2fb409c739871b84fe" target='_blank'>
              tugMedi: simulator of cancer-cell evolution for personalized medicine based on the genomic data of patients
              </a>
            </td>
          <td>
            Iurii S. Nagornov, E. Furukawa, M. Nagai, S. Yagishita, Tatsuhiro Shibata, M. Kato
          </td>
          <td>2025-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="ABSTRACT Kaposi's sarcoma‐associated herpesvirus (KSHV), the etiological agent of Kaposi's sarcoma and other lymphoproliferative disorders, presents a delicate balance between host cell manipulation, and orchestrating its transcriptional repertoire. Like all herpes viruses, the virus can reside in latency, expressing only a handful of proteins or launch a lytic program that expresses over 80 proteins responsible for the generation and release of new virions. Enhancers, as their name suggests, can enhance the transcription rate from a promoter depending on the cellular environment and tissue. While the promoters in KSHV have been extensively studied, the enhancers were only recently identified. Here we will describe our current view on KSHV enhancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0b1a93efde5ee2df29eb9e253f26d14a84bff84" target='_blank'>
              Viral Enhancers Orchestrate Kaposi's Sarcoma‐Associated Herpesvirus Transcription
              </a>
            </td>
          <td>
            Nilabja Roy Chowdhury, Meir Shamay
          </td>
          <td>2025-07-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Despite the comprehensive advancement in the field of cancer therapeutics, there remains an urgent need to identify new pathophysiological mechanisms that can be targeted in isolation or in combination with existing therapeutic regimens. The epithelial-to-mesenchymal transitions (EMT) induced by hypoxia, cytokines, and growth factors involves acquisition of invasive and migratory properties by cancer cells. Epigenetic alterations of DNA methylations and/or histone modifications cause substantial transcriptomic reprogramming in cancer cells during EMT and metastasis, which can be therapeutically targeted by a thorough understanding of the mutual interactions among the epigenetic processes. Previously, the mammalian DNA methyltransferases (DNMTs) have been shown to possess redox- and Ca++- dependent active DNA 5mC demethylation activities in addition to the cytosine methylation activity. Methods In this study, we have carried out experiments using a range of molecular, cellular, and genome editing approaches including cell culturing, CRISPR/Cas9-editing, si- or sh-RNA-mediated knockdown, quantitative RT-PCR, western blotting, ChIP-qPCR, Na-bisulfite sequencing, EMT and lung colonization assays in conjunction with DNA methylome and DNMT3A ChIP-Seq analyses, Results We found that active DNA demethylation activity of DNMT3A is essential for hypoxia-induced EMT of the SW480 colon cancer cells, its global genomic DNA demethylation, and promoter DNA demethylation/transcriptional activation of EMT-associated genes including TWIST1 and SNAIL1. DNMT3A also regulates hypoxia-induced HIF-1α binding to and transcriptional activation of the TWIST1 promoter as well as genome-wide DNA demethylation and EMT of breast cancer and liver cancer cells. Mechanistic analysis supports a regulatory model where hypoxia-induced H3K36me3 mark recruits DNMT3A to demethylate CpG in the hypoxia-responsive element (HRE), thereby facilitating HIF-1α binding and activation of the promoters of EMT genes. Conclusions Altogether, this study has provided the first demonstration of a physiological function of the active DNA demethylation activity of the DNMTs. Equally important, our findings have revealed a missing link between the HIF-1α pathway and the O2-sensing KDM pathway both of which are known to be essential for a wide set of normal and disease-associated cellular processes. Finally, the active DNA demethylation activity of DNMT3A has now emerged as a new potential target for therapeutic development to prevent EMT and metastasis of cancer cells. Clinical trial number Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s11658-025-00775-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/754e95e75763fae58f7d2af9f820e9cabdaae0fe" target='_blank'>
              Hypoxia-induced genome-wide DNA demethylation by DNMT3A and EMT of cancer cells
              </a>
            </td>
          <td>
            Biswanath Chatterjee, P. Majumder, Chun-Chang Chen, Jing-Ping Wang, P. Su, Hung-Cheng Lai, Ching-Chen Liu, Hsin-Nan Lin, Chen-Hsin A Yu, Hanna S Yuan, Che-Kun James Shen
          </td>
          <td>2025-08-05</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2944900d47d857ea99c57a79dff86ef949dfd7c" target='_blank'>
              Genomic alterations in the YAP/TAZ pathway are associated with stem cell-like castration-resistant prostate cancer
              </a>
            </td>
          <td>
            M. Roskes, A. Martínez-Fundichely, S. Cohen, M. Balaban, C. K. Wong, W. Li, T. A. Gonzalez, A. Tehim, H. Xu, S. ElNaggar, M. Myers, R. Bareja, P. Dorsaint, K. Gorski, M. Asad, M. A. Assaad, B. D. Robinson, M. Sigouros, E. Barnett, J. Manohar, S. Tagawa, D. Nanus, A. Molina, J. Nauseef, C. N. Sternberg, J. Mosquera, H. Scher, A. Sboner, B. Raphael, Y. Chen, E. Khurana
          </td>
          <td>2025-07-02</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Abstract R-loops or DNA–RNA hybrids are prominent nucleic acid structures that commonly arise during transcription. These structures play important biological functions, such as regulating gene expression and DNA repair. However, when unresolved by nucleic acid processing factors, pathological R-loops can be harmful and lead to genome instability. N6-Methyladenosine (m6A), the most prevalent modification in messenger RNA, has been recently identified to be crucial for regulating R-loop balance and maintaining genome stability. Strikingly, m6A-modified R-loop formation can have opposing consequences, either stabilization or resolution, depending on the biological context. In this review, we discuss the current knowledge of the regulatory roles of m6A on R-loops across various processes, including gene transcription, DNA repair, and centromere and telomere stability. Additionally, we explore other m6A-mediated processes, such as nascent transcription and chromatin landscape, that potentially affect R-loop dynamics. Finally, we discuss the current limitations and future directions of studying the m6A–R-loop axis, as well as the opportunities to target this pathway as a potential therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5d8ee286e956882ce3a5149e41ece752c38e0d" target='_blank'>
              m6A modification in R-loop homeostasis: a potential target for cancer therapeutics
              </a>
            </td>
          <td>
            Minh-Anh Vu, Manuela Spagnuolo, Chun-Long Chen
          </td>
          <td>2025-08-11</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. This study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2), and triple-negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb3089644af163030736f3789ed79a1b202fc7" target='_blank'>
              Genetic background and oncogenic driver determines the genomic evolution and transcriptomics of mammary tumor metastasis
              </a>
            </td>
          <td>
            Christina R Ross, Karol Szczepanek, Jack Sanford, Gabriel A. Needle, Tinghu Qiu, Larry Pearce, Maxwell P Lee, Kent W. Hunter
          </td>
          <td>2025-08-14</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Epigenetic traits are persistent cellular and organismal properties that do not result from changes in DNA sequence. One such property involves transmission of chromosomes, which entails the formation of highly specialized chromatin structures, the kinetochores, on selected chromosomal regions, called centromeres. Centromere function is essential and centromeres are determined epigenetically by the deposition of a variant histone H3 CENP-A (CENH3 in plants). Either reduced or ectopic function alone leads to genome instability, decreased fitness, aneuploid syndromes, and cancer. At times, however, centromeres malfunction in an apparently programmed mode. This is exemplified by a peculiar centromeric syndrome involving selective elimination of a chromosome set, which can affect a wide range of organisms, including plants. Over half a century ago, plant geneticists described this syndrome in interspecific crosses of barley. Building on their work, we examine the growing understanding of how CENH3 function can be modified to affect epigenetic regulation of centromeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024266b37112d5916123a1fab820a516c26c7be4" target='_blank'>
              Pioneers of chromosome elimination
              </a>
            </td>
          <td>
            Luca Comai, M. P. A. Marimuthu
          </td>
          <td>2025-07-29</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="R-loops, chromatin structures containing DNA-RNA hybrids with displaced single-stranded DNA, play crucial roles in various cellular processes. Their formation is influenced by factors such as DNA topology, RNA stability, and the presence of GC-rich regions. However, excessive or uncontrolled R-loop accumulation can threaten genomic stability, leading to DNA damage, particularly double-strand breaks. To preserve genome integrity, cells have developed mechanisms to regulate R-loop formation and resolution. Dysregulation of these processes is linked to several diseases, including cancer and neurodegenerative disorders. In this review, we will explore the dynamics of R-loop formation and resolution and how they are detected, their roles in DNA damage and repair, and how their dysregulation may lead to immune responses and disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3728d9fd65e0fa0dfc3340b92946a313355d5d1" target='_blank'>
              The paradox of R-loops: guardians of the genome or drivers of disease?
              </a>
            </td>
          <td>
            Shachinthaka D Dissanayaka Mudiyanselage, Phillip Wulfridge, K. Sarma
          </td>
          <td>2025-08-06</td>
          <td>Genome research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Homologous recombination is a high-fidelity DNA repair mechanism essential for maintaining genome stability. Impairment of this pathway, often due to BRCA1 or BRCA2 inactivation, leads to homologous recombination deficiency (HRD), forcing cells to rely on error-prone mechanisms for repairing DNA double-strand breaks, such as nonhomologous or microhomology-mediated end joining. HRD is a clinically important biomarker, particularly in breast and ovarian cancers, as it predicts responsiveness to platinum-based chemotherapies and PARP inhibitors. However, current tests in the clinical setting, mostly based on targeted panel sequencing, lack specificity and lead to a substantial number of false positives. In contrast, whole-genome sequencing, despite its high accuracy, remains largely confined to research because of high costs and logistical constraints. In this issue of Cancer Research, Abbasi and colleagues present HRProfiler, a machine learning-based tool that accurately detects HRD using whole-exome sequencing (WES) data, which is increasingly accessible in clinical oncology. Notably, it demonstrates improved sensitivity in the WES setting compared with existing tools, such as HRDetect and SigMA. As WES continues to gain traction, HRProfiler offers a promising step toward democratizing HRD detection and enabling more precise, genomics-guided treatment strategies. This article is part of a special series: Driving Cancer Discoveries with Computational Research, Data Science, and Machine Learning/AI. See related article by Abbasi et al., p. 2504.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab81bfb225ffd3e5881d30b8dbaccdbb884d86b5" target='_blank'>
              Leveraging Whole-Exome Sequencing and Mutational Signatures to Detect Homologous Recombination Deficiency in Cancer.
              </a>
            </td>
          <td>
            Joonoh Lim, Young Seok Ju
          </td>
          <td>2025-07-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The Mre11 nuclease, part of the conserved MRX complex involved in the repair of DNA double-strand breaks (DSBs), is also essential to initiate meiotic recombination in budding yeast by promoting Spo11-induced DSBs. Recruitment of Mre11 to meiotic DSB sites depends on Rec114-Mei4 and Mer2 (RMM) that organize the meiotic DSB machinery by a mechanism involving biomolecular condensation. Here, we explored the role of Mre11 during meiosis and its relationship to RMM condensation. We show that both Mre11 and MRX complexes form DNA-dependent, hexanediol sensitive condensates in vitro. In vivo, Mre11 assembles into DNA damage-dependent foci in vegetative cells and DSB-independent foci in meiotic cells. In vitro condensates and in vivo foci both depend on the C-terminal intrinsically-disordered region (IDR) of Mre11. Importantly, while the Mre11 IDR is dispensable for vegetative DNA repair it is essential during meiosis. The C-terminus of Mre11 forms a short α-helix that binds a conserved region of Mer2, and mutating residues within this interface reduces Mre11 foci and DSB formation. Finally, we identified a SUMO-interacting motif within the Mre11 IDR that enhances recruitment of Mre11 during meiosis and facilitates DSB formation. This work identifies multiple mechanisms that collaborate to recruit Mre11 during meiosis to initiate recombination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d416cf934fcb6f095caa1612b3bfc936b858edeb" target='_blank'>
              Recruitment of Mre11 to recombination sites during meiosis
              </a>
            </td>
          <td>
            Priyanka Priyadarshini, Mahesh Survi, Wael El Yazidi Mouloud, Regina Bohn, Steven Ballet, Neil Hunter, Alexander N Volkov, Corentin Claeys Bouuaert
          </td>
          <td>2025-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708353dcfdc8ca7c4e2b434fced2427c746e282e" target='_blank'>
              Tracing the evolution of single-cell 3D genomes in Kras-driven cancers.
              </a>
            </td>
          <td>
            Miao Liu, Shengyan Jin, Sherry S Agabiti, Tyler B Jensen, Tianqi Yang, Jonathan S. D. Radda, Christian F. Ruiz, Gabriel Baldissera, Moein Rajaei, Fangyong Li, Jeffrey P. Townsend, M. Muzumdar, Siyuan Wang
          </td>
          <td>2025-08-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Multicellular organisms arise from a single genome template in the zygote, necessitating the cells of the developing embryo to up- and downregulate specific genes to establish and maintain their identity. This template is maintained, propagated, and interpreted as chromatin, a polymer of nucleic acids and associated structural and regulatory proteins. Recent genome-wide surveys documented a wealth of disease-associated mutations in chromatin factors, indicating their fundamental significance and potential for therapeutic targeting. However, chromatin factors exist in a complex balance, with a single deficiency often leading to pleiotropic downstream effects. Here, we review the mechanisms of chromatin regulation and partitioning, highlighting examples of how these processes are altered in human diseases. We argue that loss of chromatin fidelity, both locally at specific genes and regulatory elements, and globally at the megabase-scale, contributes to many pathological states and may thus represent an intriguing target for corrective interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70b3c6ccb1965f22efefabec61c021483fcae985" target='_blank'>
              Aere perennius: how chromatin fidelity is maintained and lost in disease
              </a>
            </td>
          <td>
            Dustin Fetch, Amina Jumamyradova, Cameron Chapa, Yong Ge, Mansour Mohamadzadeh, Alexey A Soshnev
          </td>
          <td>2025-07-01</td>
          <td>Nar Molecular Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Methylglyoxal (MG) is a highly reactive aldehyde that is produced endogenously during metabolism, and from exogenous sources like sugary food and cigarette smoke. Unless detoxified by glyoxalases, MG can readily react with DNA and proteins, generating characteristic glycation-derived lesions. As a result, MG exposure has been linked to a variety of human diseases, including cancers. Prior studies show that MG preferentially makes adducts on guanine residues, causing DNA damage. However, in vivo, how such events impact genome-wide MG mutagenicity is poorly understood. Such information is essential to comprehend the true contribution of MG to genome instability and global mutational burden. In the present study, we show that MG can robustly mutagenize single-stranded DNA in the yeast genome, within a guanine-centered mutable motif. We demonstrate that genome-wide MG mutagenesis is greatly elevated in the absence of the glyoxalase Glo1, and abrogated in the presence of the aldehyde quencher aminoguanidine. Importantly, we uncovered strand slippage and mispairing as the predominant mechanism for generation of all MG-associated mutations, and demonstrate that the translesion polymerase Rev1 is a key player in this pathway. Finally, we find that the primary MG-associated mutation is enriched in a variety of sequenced tumor datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2ac197fe47da2319b761beabd3304e807a2cc0f" target='_blank'>
              Methylglyoxal mutagenizes single-stranded DNA via Rev1-associated slippage and mispairing
              </a>
            </td>
          <td>
            Sriram Vijayraghavan, Alessandra Ruggiero, Samuel Becker, P. Mieczkowski, George S Hanna, Mark T. Hamann, Natalie Saini
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Castration-resistant prostate cancer is a heterogeneous disease with variable phenotypes commonly observed in later stages of the disease. These include cases that retain expression of luminal markers and those that lose hormone dependence and acquire neuroendocrine features. While there are distinct transcriptomic and epigenomic differences between castration-resistant adenocarcinoma and neuroendocrine prostate cancer, the extent of overlap and degree of diversity across tumor metastases in individual patients has not been fully characterized. Here we perform combined DNA methylation, RNA-sequencing, H3K27ac, and H3K27me3 profiling across metastatic lesions from patients with CRPC/NEPC. Integrative analyses identify DNA methylation-driven gene links based on location (H3K27ac, H3K27me3, promoters, gene bodies) pointing to mechanisms underlying dysregulation of genes involved in tumor lineage (ASCL1, AR) and therapeutic targets (PSMA, DLL3, STEAP1, B7-H3). Overall, these data highlight how integration of DNA methylation with RNA-sequencing and histone marks can inform intraindividual epigenetic heterogeneity and identify putative mechanisms driving transcriptional reprogramming in castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdf4a71accfefcd4f60da4f3b6e81d8837cae193" target='_blank'>
              Intraindividual epigenetic heterogeneity underlying phenotypic subtypes of advanced prostate cancer
              </a>
            </td>
          <td>
            Kei Mizuno, Sheng-Yu Ku, V. B. Venkadakrishnan, Martin K. Bakht, M. Sigouros, Joanna Chan, A. Trigos, Jordan H. Driskill, J. Manohar, Abigail King, Adam G. Presser, Min Jin Kim, Alok K. Tewari, H. Long, David Quigley, T. Choueiri, Steven P. Balk, Sarah J. Hill, J. Mosquera, David J Einstein, Shahneen Sandhu, M. Taplin, H. Beltran
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b074d01f6fb78ae72372c0d26a26df7f99fb224" target='_blank'>
              Fungi as models of centromere innovation: from DNA sequence to 3-dimensional arrangement.
              </a>
            </td>
          <td>
            Srijana Dutta, Krishna Bhat, Rashi Aggarwal, K. Sanyal
          </td>
          <td>2025-08-11</td>
          <td>Chromosome research : an international journal on the molecular, supramolecular and evolutionary aspects of chromosome biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Alternative lengthening of telomeres (ALT) is a telomerase-independent telomere maintenance mechanism observed in 15% of human cancers. A hallmark of ALT cancers is the presence of C-circles, circular single-stranded DNAs (ssDNAs) enriched with cytosine-rich telomere (C-rich, CCCTAA) sequences. G-circles, containing guanosine-rich telomere (G-rich, GGGTTA) ssDNAs, also exist but are much less abundant. Recent studies indicate that excessive displacement of Okazaki fragments during lagging-strand synthesis is a unique feature of ALT telomeres and responsible for generating C-circles/C-rich ssDNAs. However, the distinct characteristics of C-circles compared to G-circles remain unclear. Here, we demonstrate that co-deficiency of the DNA translocases SMARCAL1 and FANCM leads to abundant generation of G-circle/G-rich ssDNAs. These G-rich ssDNAs mainly exist in linear form, ranging in size from 500 to 3000 nucleotides, which differs significantly from the structure and size of C-circle/C-rich ssDNAs. Mechanistically, both C-rich and G-rich ssDNAs originate from BLM/POLD-mediated excessive strand displacement; however, they differ in their origins and initiation mechanisms. Specifically, C-rich ssDNAs arise from lagging daughter strands initiated by the CST complex, whereas G-rich ssDNAs originate from leading daughter strands through RAD51-dependent G-strand synthesis. Our findings propose two distinct mechanisms for generating two different extrachromosomal telomere DNAs, C- and G-circles, during ALT-mediated telomere elongation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12e4731f01013d1812909194114a8c4a4ce7f91c" target='_blank'>
              Distinct mechanisms underlying extrachromosomal telomere DNA generation in ALT cancers
              </a>
            </td>
          <td>
            Junyeop Lee, Eric J. Sohn, Jina Lee, Ashley Y Cheng, Angelo Taglialatela, Alberto Ciccia, Jaewon Min
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7bd3ebe84ec1189f788307a3b3b752befef2ae" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals an Aging-Associated Fibroblast Subtype Linked to Tumor Progression in Human Skin
              </a>
            </td>
          <td>
            Haibin Wu, Danping Pan
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA topoisomerases are essential for maintaining DNA topology, gene expression and the accurate transmission of genetic information. Mitochondria possess circular DNA (mtDNA), which, unlike nuclear chromosomes, lacks protective histones and exists in nucleoprotein complexes called nucleoids, which are vital for mtDNA stability. Although the mitochondrial genome encodes essential genes involved in ATP production via oxidative phosphorylation, it does not encode crucial mtDNA maintenance genes and depends entirely on nuclear-encoded proteins for mtDNA maintenance. These include nuclear-encoded topoisomerases (i.e. Top1mt, Top2α, Top2β and Top3α), which alleviate topological stress during mtDNA transcription and replication, and mitochondrial transcription factor A (TFAM), are crucial for ensuring proper nucleoid structure and mtDNA packaging. Furthermore, tyrosyl-DNA phosphodiesterase 1 and 2 (TDP1 and TDP2) participate in the repair of mtDNA damage associated with trapped topoisomerase-mtDNA complexes, which can compromise mtDNA integrity and contribute to neurodegeneration, cancer and premature aging. Drugs that stabilize these protein-DNA adducts (PDAs) to induce mtDNA damage and mitochondrial dysfunction are promising new strategies for cancer therapy. This Review explores the essential roles of mitochondrial topoisomerases, overviews mechanisms involved in mtDNA repair and discusses how mitochondrial fission and mitophagy are employed as a survival strategy for clearing damaged mtDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc13f830961da01cf0f5401d6373dc48a97645fa" target='_blank'>
              Mitochondrial topoisomerases, nucleoid architecture and mtDNA repair in human disease.
              </a>
            </td>
          <td>
            Sangheeta Bhattacharjee, B. Das
          </td>
          <td>2025-07-01</td>
          <td>Journal of cell science</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3da656da84cf3e9dc1144dda4829e9c83aef2891" target='_blank'>
              CHAMP1 Complex Promotes Heterochromatin Assembly and Reduces Replication Stress.
              </a>
            </td>
          <td>
            Feng Li, Amira Elbakry, Felix Y Zhou, Tianpeng Zhang, Ramya Ravindranathan, Huy Nguyen, Aleem Syed, Lifang Sun, S. Mukkavalli, Roger A. Greenberg, Alan D. D’Andrea
          </td>
          <td>2025-08-05</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b0586d8fafa9a29f6a58bf48055edaf80d268f3" target='_blank'>
              RIPOR2 promotes multinucleation of melanoma cells downstream of the RAS/ERK oncogenic pathway
              </a>
            </td>
          <td>
            Axelle Wilmerding, Aurélie Richard, Nicolas Macagno, E. Hirsinger, Tarek Gharsalli, Léa Bellenger, N. Naouar, Caroline Gaudy, S. Mallet, N. Degardin, Lauranne Bouteille, Nathalie Caruso, D. Duprez, Y. Graba, S. Medjkane, H. Etchevers, Marie-Claire Delfini
          </td>
          <td>2025-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Breast cancer in women <40, accounting for ~5% of all breast cancer cases diagnosed in the U.S., is more aggressive and associated with worse outcomes compared to breast cancer in older women. We performed a first-ever integrated proteogenomic study from a matched cohort of laser-microdissected tumors of 34 young (<40 years) and 34 older (≥60 years) women to identify molecular features that may underlie the worse outcomes in young women. Progression-free interval was shorter in young women, and their tumors were enriched for more aggressive molecular subtypes. Our multi-omic analysis identified distinct clusters between age groups in luminal but not basal-like cancers. Notably, GATA3 mutations were enriched in luminal tumors from young women while TP53 and PIK3CA mutations were more common in luminal tumors from older women. Young women’s tumors exhibited lower estrogen receptor (ER) expression yet paradoxically enhanced ER response pathways and increased expression of tamoxifen-resistance-associated genes (IRS1, FERMT1). Immune pathway activity and immune scores were lower in tumors from young women, whereas proliferative and MYC pathways were notably elevated, identifying potential therapeutic targets. Transcriptomic data from TCGA and METABRIC confirmed our findings, with 10 of 11 observed pathways corroborated. Finally, differential expression of four immune-related surface proteins also suggested the potential of age-specific responses to immune-based therapies. Together, these findings may contribute to the understanding of the molecular mechanisms underlying worse outcomes in young women and offer new insight to therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2307e76c48702faeae76ee479073bb1744121b0" target='_blank'>
              Proteogenomic characterization of invasive breast tumors in young women
              </a>
            </td>
          <td>
            Praveen-Kumar Raj-Kumar, Jianfang Liu, A. Soltis, N. Bateman, Qing-rong Chen, Lori Sturtz, B. Deyarmin, M. Pierobon, Tamara A Abulez, Anupama Praveen-Kumar, Xijun Zhang, Trinh Nguyen, Chunhua Yan, Ying Hu, Kate Guion, J. Hooke, A. Kovatich, Leigh Fantacone-Campbell, Bradley J. Mostoller, L. Kvecher, S. Somiari, Patricia S Steeg, P. Rajagopal, Kathleen M Darcy, Jerry S H Lee, Clifton L. Dalgard, T. Conrads, Emanuel F. Petricoin, D. Meerzaman, Matthew D Wilkerson, , C. Larson, Heather Blackburn, Katie Miller, Jennifer L. Kane, Melissa Cannada, Kimberly Murtha, Deborah W. Sims, Kimberly A. Knapp, Heather A. Kabala, Erin L. Goins, Lorraine Tafra, Jay Wareham, Lynne Mullican, Julie Joseph, Justin Wells, Nicole L. Wolf, Margaret M. Kildee, Jerez Te, C. Viollet, G. Sukumar, Dagmar Bacikova, C. Alba, Elisa McGrath, Sraavya Polisetti, Jatinder Singh, A. Adeleye, Harvey Pollard, D. Mitchell, G. Gist, Victoria Olowu, K. Wilson, K. Conrads, B. Hood, Sasha C. Makohon-Moore, G. L. Maxwell, E. Baldelli, Matthew Nealeigh, M. Cutler, Mariano Russo, A. Zimmer, Sean Hanlon, Xiaoying Lin, Craig D. Shriver, S. Lipkowitz, Hai Hu
          </td>
          <td>2025-08-18</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Abstract Transposable elements (TE), also called transposons, are repetitive DNA sequences making up half of the human genome. Initially, TEs were described as “junk DNA” because they lack specific function. However, they have been recognized for their ability to replicate and integrate into different genomic locations; this “jumping gene” activity results in genomic instability, variation of the chromosomal architecture, and transcriptional dysregulation, all of which represent major hallmarks of cancer. In this respect, the involvement of TE in tumorigenesis was extensively studied and their role as diagnostic and therapeutic tools in cancer is now well-established. Transposons’ products, including TE-derived cancer antigens, transcripts, and associated epigenetic modifications, mainly hypomethylation, were found to be promising biomarkers in several types of cancer ensuring early disease detection and prognosis. In addition, TE are currently used to design innovative approaches, with transposon-based systems, namely, Sleeping Beauty and PiggyBac, enabling precise genomic modifications and novel strategies for cancer therapy. Therefore, the aim of this review is to provide an overview on the dual application of TE as diagnostic and therapeutic tools in cancer, paving the way to improved clinical outcomes. Clinical trial number Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4636ade80b58aaf2a75f12fb6871d3d8ab2244c5" target='_blank'>
              Exploring transposable elements: new horizons in cancer diagnostics and therapeutics
              </a>
            </td>
          <td>
            Ahlam Chaaban, Reem Sleem, Jana Santina, Mohamad Rima, José-Noel Ibrahim
          </td>
          <td>2025-07-12</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The HOXA gene locus coordinates body patterning, hematopoiesis, and differentiation. While studying blood phenotype-associated variation within the HOXA locus, we identified a genetic variant, rs17437411, associated with globally reduced blood counts, protection from blood cancers, and variation in anthropometric phenotypes. We find that this variant disrupts the activity of a previously unstudied antisense long non-coding RNA (lncRNA) located between HOXA7 and HOXA9 , which we have named HOTSCRAMBL . The HOTSCRAMBL variant disrupts lncRNA function and reduces human hematopoietic stem cell (HSC) self-renewal. Mechanistically, HOTSCRAMBL enables appropriate expression and splicing of HOXA genes in HSCs, most notably HOXA9 , in an SRSF2-dependent manner. Given the critical role of HOXA gene expression in some blood cancers, we also demonstrate that HOTSCRAMBL variation or deletion compromises HOXA -dependent acute myeloid leukemias. Collectively, we show how insights from human genetic variation can uncover critical regulatory processes required for effective developmental gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854c6d558bc4abf3b6a29b21c9a18aa5b2bee350" target='_blank'>
              Genetic variation reveals a homeotic long noncoding RNA that modulates human hematopoietic stem cells
              </a>
            </td>
          <td>
            Peng Lyu, Gaurav Agarwal, Chun-Jie Guo, Tianyi Ye, Chen Weng, Mateusz Antoszewski, Samantha Joubran, Alexis Caulier, Michael Poeschla, V. G. Sankaran
          </td>
          <td>2025-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f198ff2332798cb036a19b4692ee525e77c812c" target='_blank'>
              Divergent Clonal Evolution and Early Dissemination Promote Genetic Heterogeneity of Metastases in Castration Resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Noshad Hosseini, R. Mannan, Ryan J. Rebernick, Fengyun Su, Rui Wang, Xuhong Cao, Ana Lako, Dattatreya Mellacheruvu, Jing Hu, J. Alumkal, Zachery R Reichert, Rohit Malik, Rohit Mehra, A. Chinnaiyan, M. Cieslik
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="Elephants exhibit remarkable resistance to cancer, and understanding these mechanisms has focused on their potential applications in cancer prevention and treatment in humans. A genome-wide comparative analysis identified that the accelerated regions in elephants are enriched in Fanconi anemia (FA) complementation group L (FANCL), a ubiquitin E3 ligase that mediates the monoubiquitylation of FANCD2 as an essential step in the FA pathway. The FA pathway plays a crucial role in DNA interstrand crosslink (ICL) repair, contributing substantially to genome stability and cancer resistance. In this study, we investigated the differences in ICL repair via the FA pathway, including the function of FANCL, as well as the DNA damage response to ICLs between elephants and humans. We found that elephant fibroblasts exhibited higher sensitivity to ICL-inducing treatments, such as mitomycin C and trimethylpsoralen plus UVA (PUVA), than human fibroblasts, while showing comparable or reduced sensitivity to other DNA-damaging agents, such as doxorubicin and bleomycin. Functional analyses revealed that elephant and human FANCL performed similarly in mediating FANCD2 monoubiquitylation and cell viability following mitomycin C treatment. Interestingly, elephant fibroblasts exhibited a more potent and prolonged activation of p21 and sustained DNA repair responses, such as FANCD2 monoubiquitylation and increased RAD51expression, following ICL-induced treatments. Moreover, elephant fibroblasts showed significantly greater RAD51 foci formation than human fibroblasts after PUVA treatment, even under comparable levels of DNA damage. These findings suggest that elephants efficiently repair ICLs in growth-arrested cells likely through robust p21 activation. This study provides new insights into the cancer resistance mechanisms of elephants and offers novel approaches for cancer prevention and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5eb799fee658096c981c29abe3d04c60e57627a" target='_blank'>
              Enhanced sensitivity, robust p21 activation, and sustained DNA repair responses to interstrand crosslinks in elephant cells compared to humans
              </a>
            </td>
          <td>
            Taisuke Kitano, Zida Zhu, Naoya Minami, K. Orino, Yasunaga Yoshikawa
          </td>
          <td>2025-07-10</td>
          <td>Frontiers in Veterinary Science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Hepatoblastoma is the most common pediatric liver cancer, with the fastest rising incidence among childhood malignancies. Early genomic studies revealed that hepatoblastoma has the lowest mutational burden of any human cancer, however, recent advances in single-cell RNA-seq, multiomics, spatial transcriptomics, and functional genomics screenings have revealed substantial complexity. Diverse cellular subpopulations, divergent WNT signaling, key developmental pathways, and intricate interactions between the tumor cells and tumor immune microenvironment (TME) collectively shape tumor heterogeneity, disease progression, therapeutic responses, and genetic dependencies. Mosaic embryonic loss of heterozygosity (LOH) at chromosome 11p15.5 may be a hepatoblastoma-initiating event, as clonal expansion of 11p15.5 LOH occurs in adjacent normal liver tissue. A cholangiocyte-like subpopulation expresses FGF19, in a SOX4-dependent, paracrine manner, to drive the proliferation of neighboring embryonal hepatoblastoma cells. WNT-signaling dependent MDK promotes the immunosuppressive TME, which impairs immune cell infiltration. The TME may also be driven by islands of erythroblasts, which influence treatment resistance. Plasticity driven by changes in chromatin accessibility enables differentiation transition between hepatocytic and liver progenitor cell types, which is associated with treatment resistance. Here, we review recent findings in pediatric hepatoblastoma cells, tumor-associated cell types, and genetic dependencies that will serve to advance hepatoblastoma therapy. Clinical trial number: Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02405-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/361b9ebd3fa588c3f28e25ba36c91c45f4b07fb6" target='_blank'>
              From preneoplastic lesion to heterogenous tumor: recent insights into hepatoblastoma biology and therapeutic opportunities
              </a>
            </td>
          <td>
            Jun Yang, A. Davidoff, Andrew J. Murphy
          </td>
          <td>2025-07-19</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="DNA polymerase delta (Pol δ) is a cornerstone of genomic stability, orchestrating DNA replication and repair through its catalytic subunit, POLD1. This subunit’s 3’–5’ exonuclease domain proofreads replication errors, ensuring fidelity. However, POLD1 mutations—particularly in this domain—disrupt this function, triggering genomic instability and a hypermutated state in cancers. This review delves into the multifaceted roles of POLD1 mutations, spotlighting their contributions to tumorigenesis and immunotherapy responses. Beyond their established link to syndromes like polymerase proofreading-associated polyposis (PPAP), these mutations unexpectedly enhance tumor immunogenicity in microsatellite-stable (MSS) tumors, previously considered largely resistant to immune checkpoint inhibitors (ICIs). By elevating tumor mutation burden and generating unique mutational signatures (e.g., SBS10d), POLD1 mutations sensitize MSS tumors to ICIs, challenging the dominance of microsatellite instability (MSI) as an immunotherapy predictor. Integrating structural insights, molecular mechanisms, and clinical data, this review positions POLD1 mutations as both a driver of cancer progression and a promising biomarker, redefining therapeutic possibilities in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02d2442be2c75ad66ffe5186a1b712852b468974" target='_blank'>
              Beyond proofreading: POLD1 mutations as dynamic orchestrators of genomic instability and immune evasion in cancer
              </a>
            </td>
          <td>
            Huiqing Chen, Jiayu Wei, Qi Tang, Guohui Li, Yajing Zhou, Zhen Zhu
          </td>
          <td>2025-06-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The majority of cancer cells experience replication stress, which ultimately causes them to enter mitosis with underreplicated DNA. To alleviate the consequences of replication stress, cells utilize a mechanism known as MiDAS that functions to complete synthesis of underreplicated DNA in early mitosis. This process is considered an Achilles heel for highly replicative cancers. In this study, we show that human TopBP1 localizes to sites of underreplicated DNA marked by FANCD2 and promotes MiDAS through recruitment of the nuclease scaffold protein SLX4. Additionally, we demonstrate that the recruitment of SLX4 to TopBP1 foci in mitosis depends on TopBP1-K704, SLX4-T1260, and several SUMO-interaction motifs in SLX4. Lastly, we show that the recruitment of SLX4 to TopBP1 foci in mitosis is important to prevent transmission of DNA damage to daughter cells. Based on this, we hypothesize that targeting the TopBP1-SLX4 interaction in mitosis may be a potential strategy for anti-cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d03ab5b01bb44008997c89ac819106d74fa906d" target='_blank'>
              TopBP1 coordinates DNA repair synthesis in mitosis via recruitment of the nuclease scaffold SLX4
              </a>
            </td>
          <td>
            Jonas Bagge, Kamilla Vandsø Petersen, Sinem N Karakus, Thorbjørn M Nielsen, Johanne Rask, Christian R Brøgger, Jonas Jensen, Meliti Skouteri, Antony M. Carr, I. Hendriks, V. Oestergaard, Michael Lisby
          </td>
          <td>2025-07-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Tumor mutational burden (TMB) has emerged as a potential surrogate for neoantigen load and an indicator of immune checkpoint (IC)-blockade response; however, its precise significance in breast cancer (BC) is not fully understood. Here, we comprehensively characterized the genomic repertoire of BCs with a TMB ≥ 10 mut/Mb (TMB-high [n = 527]) to identify putative predictors of importance. The predominant mutational signature was apolipoprotein B mRNA-editing enzyme catalytic polypeptide (APOBEC) in 64.7% of tumors. TMB-high BCs were enriched in KMT2C, ARID1A, PTEN, NF1, and RB1 alterations, which are associated with APOBEC mutagenesis. Further identified were loss-of-function ARID1A and PTEN alterations, which are linked to immune cell exclusion. ESR1 p.E380Q prevailed among all ESR1 hotspot mutations, supporting APOBEC-mediated effects. Finally, mutations in DNA damage response and repair genes were seen at a higher frequency than in non-TMB-high BCs. These findings provide justification for exploring combined pharmacologic inhibition to improve IC-based efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e8980a3a4147f92cf6718bb89169af5da92bd71" target='_blank'>
              Genomic characterization of tumor mutational burden-high breast carcinomas
              </a>
            </td>
          <td>
            Theodore Vougiouklakis, Chad M. Vanderbilt, Satshil Rana, A. Mohanty, F. Pareja, E. Brogi, Christopher J. Schwartz, M. Arcila, Marc Ladanyi, Hanna Y Wen, D. Ross
          </td>
          <td>2025-08-08</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Many essential functions of organisms are encoded in highly repetitive genomic regions, including histones involved in DNA packaging, centromeres that are core components of chromosome segregation, ribosomal RNA comprising the protein translation machinery, telomeres that ensure chromosome integrity, piRNA clusters encoding host defenses against selfish elements, and virtually the entire Y Chromosome. These regions, formed by highly similar tandem arrays, pose significant challenges for experimental and computational studies, impeding sequence-level descriptions essential for understanding genetic variation. Here, we report the assembly and variation analysis of such repetitive regions in Drosophila melanogaster, offering significant improvements to the existing community reference assembly. Our work successfully recovers previously elusive segments, including complete reconstructions of the histone locus and the pericentric heterochromatin of the X Chromosome, spanning the Stellate locus to the distal flank of the rDNA cluster. To infer structural changes in these regions where alignments are often not practicable, we introduce landmark anchors based on unique variants that are putatively orthologous. These regions display considerable structural variation between different D. melanogaster strains, exhibiting differences in copy number and organization of homologous repeat units between haplotypes. In the histone cluster, although we observe minimal genetic exchange indicative of meiotic crossing over, the variation patterns suggest mechanisms such as unequal sister chromatid exchange. We also examine the prevalence and scale of concerted evolution in the histone and Stellate clusters and discuss the mechanisms underlying these observed patterns.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/624ae86af4f411108a369c23810a20e4ee6dde43" target='_blank'>
              Genetic variation in recalcitrant repetitive regions of the Drosophila melanogaster genome.
              </a>
            </td>
          <td>
            Harsh G. Shukla, Mahul Chakraborty, J. Emerson
          </td>
          <td>2025-07-16</td>
          <td>Genome research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Targeted demethylation by DNA glycosylases (DNGs) results in differential methylation between parental alleles in the endosperm, which drives imprinted expression. Here, we performed RNA sequencing on endosperm derived from DNG mutant mdr1 and wild-type endosperm. Consistent with the role of DNA methylation in gene silencing, we find 96 gene and 86 TE differentially expressed (DE) transcripts that lost expression in the hypermethylated mdr1 mutant. Compared with other endosperm transcripts, the mdr1 targets are enriched for TEs (particularly Helitrons), and DE genes are depleted for both core genes and GO term assignments, suggesting that the majority of DE transcripts are TEs and pseudo-genes. By comparing DE genes to imprinting calls from prior studies, we find that the majority of DE genes have maternally biased expression, and approximately half of all maternally expressed genes (MEGs) are DE in this study. In contrast, no paternally expressed genes (PEGs) are DE. DNG-dependent imprinted genes are distinguished by maternal demethylation and expression primarily in the endosperm, so we also performed EM-seq on hybrids to identify maternal demethylation and utilized a W22 gene expression atlas to identify genes expressed primarily in the endosperm. Overall, approximately ⅔ of all MEGs show evidence of regulation by DNA glycosylases. Taken together, this study solidifies the role of MDR1 in the regulation of maternally expressed, imprinted genes and TEs and identifies subsets of genes with DNG-independent imprinting regulation. Significance Statement This work investigates the transcriptome changes resulting from the loss of function of DNA glycosylase MDR1, revealing that, in wild-type endosperm, targets of MDR1 are expressed predominantly from the maternal allele and this expression is suppressed in mutants. Furthermore, by combining expression data, DNA methylation data, and developmental expression data, we are able to categorize all maternally expressed, imprinted genes based on DNA glycosylase dependent or independent regulatory methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e38df6b4bc62b7ce1643805d5dd45feb9a54b999" target='_blank'>
              MDR1 DNA glycosylase regulates the expression of genomically imprinted genes and Helitrons
              </a>
            </td>
          <td>
            Kaitlin M. Higgins, J. Gent, Sarah N. Anderson
          </td>
          <td>2024-08-01</td>
          <td>The Plant Journal</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Throughout the last century, aneuploidy has been cemented as a hallmark of cancer. Although the association of aneuploidy with tumorigenesis has been well established, the role of these genetic imbalances in tumor formation has only recently begun to be elucidated. Advancements in genomics have revealed the complexity and context dependence of the effect of aneuploidy on cancer growth, while developments in genetic editing have allowed for proper modeling of specific aneuploidies. In this review, we discuss the key factors to consider when studying the role of aneuploidy in cancer and the tools that are available to do so. We then highlight recent studies that establish phenotypic contributions of aneuploidy to tumorigenicity. In particular, we highlight how general aneuploidy and chromosomal instability affect the tumor microenvironment and how specific chromosomal alterations, including the loss of chromosome 9p and the gain of chromosomes 8q and 1q, influence tumor behavior and therapeutic responses. Finally, we emphasize the potential of targeting aneuploidy-induced vulnerabilities to improve cancer treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b1804dd6a9ac26725f995589c82459ba79d112" target='_blank'>
              Modeling and targeting general and chromosome-specific aneuploidy in cancer.
              </a>
            </td>
          <td>
            Aleah Goldberg, Maria Trifas, Teresa Davoli
          </td>
          <td>2025-07-28</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Long-read sequencing (LRS) has improved sensitivity to discover variation in complex repetitive regions, assign parent-of-origin, and distinguish de novo germline from postzygotic mutations (PZMs). Most studies have been limited to population genetic surveys or a few families. We applied three orthogonal sequencing technologies—lIlumina, Oxford Nanopore Technologies, and Pacific Biosciences—to discover and validate de novo mutations (DNMs) in 73 children from 42 autism families (157 individuals). Assaying 2.77 Gbp of the human genome using read-based approaches, we discover on average 95 DNMs per transmission (87.5 de novo single-nucleotide variants and 7.8 indels), including sex chromosomes. We estimate that LRS increases DNM discovery by 20–40% over previous Illumina-based studies of the same families, and more than doubles the discoverable number of PZMs that emerged early in embryonic development. The strict germline mutation rate is 1.30×10−8 substitutions per base pair per generation, strongly driven by the father’s germline (3.95:1), while PZMs increase the rate by 0.23×10−8 with a modest but significant bias toward paternal haplotypes (1.15:1). We show that the mutation rate is significantly increased for classes of repetitive DNA, where segmental duplication (SD) mutation shows a dependence on the length and percent identity of the SD. We find that the mutation rate enrichment in repeats occurs predominantly postzygotically as opposed to in the germline, a likely result of faulty DNA repair and interlocus gene conversion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b545aeba0586ba78cffa00d695ca70d8352ddc9" target='_blank'>
              Long-read sequencing of trios reveals increased germline and postzygotic mutation rates in repetitive DNA
              </a>
            </td>
          <td>
            Michelle D. Noyes, Yang Sui, Youngjun Kwon, Nidhi Koundinya, Isaac Wong, Katherine M. Munson, Kendra Hoekzema, Jennifer R. Kordosky, Gage H. Garcia, Jordan Knuth, Alexandra P. Lewis, E. Eichler
          </td>
          <td>2025-07-19</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d169a9346a546b4f3b12a60658db5bed2132a3f" target='_blank'>
              "In situ" MYC rearrangement and amplification as a secondary change in a case of High-grade B cell lymphoma with 11q aberration.
              </a>
            </td>
          <td>
            G. Di Stefano, S. Bens, Anja Fischer, Manuel Ciceri, L. Nassi, Reiner Siebert, Raffaella Santi
          </td>
          <td>2025-08-07</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="ABSTRACT In this study, we report the atypical cell cycle organization of the unicellular eukaryotic pathogen Toxoplasma gondii. The remarkably flexible cell division of T. gondii and other apicomplexan parasites differs considerably from the cell division modes employed by other model eukaryotes. In addition, there is a lack of recognizable cell cycle regulators, which has contributed to the difficulties in deciphering the order of events in the apicomplexan cell cycle. To aid in studies of the cell cycle organization of the T. gondii tachyzoite, we have created the Fluorescent Ubiquitination-based Cell Cycle Indicator probes, ToxoFUCCIS and ToxoFUCCISC. We introduced a DNA replication factor TgPCNA1 tagged with NeonGreen that can be used alone or in conjunction with an mCherry-tagged budding indicator TgIMC3 in the auxin-induced degradation parental strain. The varied localization and dynamic cell cycle oscillation have confirmed TgPCNA1 to be a suitable T. gondii FUCCI probe. The ToxoFUCCIS analysis showed that tachyzoite DNA replication starts at or near centromeric regions and has a bell-shaped dynamic and a significant degree of the cell cycle asynchrony within the vacuoles. Quantitative live and immunofluorescence microscopy analyses of ToxoFUCCIS and its derivatives co-expressing epitope-tagged cell cycle markers have revealed an unusual composite cell cycle phase that incorporates overlapping S, G2, mitosis, and cytokinesis (budding). We identified five intervals of the composite phase and their approximate duration: S (19%), S/G2/C (3%), S/M/C (9%), M/C (18%), and C/G1 (<1%). The ToxoFUCCIS probe efficiently detected G2/M and Spindle Assembly Checkpoints, as well as the SB505124-induced TgMAPK1-dependent block. Altogether, our findings showed an unprecedented complexity of the cell cycle in apicomplexan parasites. IMPORTANCE The cell division rates directly correlate with the severity of the diseases caused by apicomplexan parasites. Despite its clinical importance, little is known about the apicomplexan cell cycle that controls parasite division rates. Previous studies implied that the apicomplexan cell cycle is organized differently from the cell cycle of their host cells. However, the order of cell cycle events had never been established. In the current study, we present evidence of the highly unusual organization of the Toxoplasma gondii cell cycle. Using a new cell cycle indicator, we measured the duration of individual cell cycle processes in Toxoplasma tachyzoites and revealed the unprecedented overlaps of four cell cycle phases. Our findings explain how the apicomplexan cell cycle accommodates the flexibility of the division modes and identify unique steps of the parasite survival program that can be explored in the future. The cell division rates directly correlate with the severity of the diseases caused by apicomplexan parasites. Despite its clinical importance, little is known about the apicomplexan cell cycle that controls parasite division rates. Previous studies implied that the apicomplexan cell cycle is organized differently from the cell cycle of their host cells. However, the order of cell cycle events had never been established. In the current study, we present evidence of the highly unusual organization of the Toxoplasma gondii cell cycle. Using a new cell cycle indicator, we measured the duration of individual cell cycle processes in Toxoplasma tachyzoites and revealed the unprecedented overlaps of four cell cycle phases. Our findings explain how the apicomplexan cell cycle accommodates the flexibility of the division modes and identify unique steps of the parasite survival program that can be explored in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e64a3a52147cec95c741b1106e281890e68bde6" target='_blank'>
              Deciphering cell cycle organization of Toxoplasma endodyogeny
              </a>
            </td>
          <td>
            Mrinalini Batra, Clem Marsilia, Danya Awshah, L. M. Hawkins, Chengqi Wang, Dale Chaput, Daria A. Naumova, E. Suvorova
          </td>
          <td>2025-07-01</td>
          <td>mBio</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="ABSTRACT Transposable elements (TEs) have hijacked cellular machineries to replicate and spread throughout host genomes. TEs now make up a significant portion of eukaryotic genomes and play notable roles in genomic evolution, driving both speciation and providing raw material for genetic innovation. Barbara McClintock's pioneering work on these “jumping genes” laid the foundation for modern TE research; however, her paradigm‐shifting theories in which TEs act as “controlling elements” were initially rejected due to the long‐held belief that TEs were “junk” or parasitic DNA elements. Historically, the highly repetitive nature of TEs made it challenging to both identify and investigate functions. However, recent advances in genomics have greatly accelerated our understanding of TEs. Despite their potential to cause insertional mutagenesis and disease, many transposable elements have been co‐opted by host genomes to contribute to gene regulation and development. In contrast to protein‐coding genes that typically begin their journey as DNA, are transcribed into RNA, and reach their ultimate functional form as proteins, TEs can function as cis‐regulatory DNA, functional RNA, and in rare cases, domesticated proteins and fusion events between TE and host genes. Driven by rapidly advancing technologies, the roles of TEs in both development and disease are being uncovered faster than ever, making current and future work an exciting continuation of Barbara McClintock's groundbreaking legacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/831a0252c52d2b92a8d6f28251facae6b2cf9eff" target='_blank'>
              From Junk DNA to Genomic Treasure: Impacts of Transposable Element DNA, RNA, and Protein in Mammalian Development and Disease
              </a>
            </td>
          <td>
            Ten D. Li, Katelyn Toohill, A. Modzelewski
          </td>
          <td>2025-07-01</td>
          <td>Wiley Interdisciplinary Reviews. RNA</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d205d2c83b4378b5ded0fd2f961df7d35dcb8b85" target='_blank'>
              Assemblatron: An Automated Workflow for High-Throughput Assembly of Big-DNA Libraries
              </a>
            </td>
          <td>
            Antonio Vela Gartner, Luisa F. Quispe, Klaus-Peter Rühmann, J. A. Martin, Jordan M. Welker, Mingyu Kenneth Li, Camila Coelho, Yu Zhao, Ran Brosh, J. Boeke
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="Genomic instability underlies various diseases, including cancer. This instability arises from defects in critical cellular processes, particularly those involved in DNA damage repair. Therefore, a detailed understanding of these repair mechanisms is essential for developing strategies to prevent or diagnose such diseases. The MRN complex, composed of MRE11, NBS1, and RAD50, is among the earliest elements involved in detecting DNA damage. Upon detecting DNA breaks, this complex triggers a cascade of signaling events that regulate both cell cycle arrest and DNA repair. These signaling pathways are tightly controlled by various post-translational modifications, notably ubiquitination. Although several ubiquitin ligases have been implicated in different stages of the DNA damage response, our knowledge remains limited. In this study, we reveal that βTrCP, a substrate-recognizing subunit of the SCF (SKP1/CUL1/F-box protein) ubiquitin ligase, interacts in vivo with the proteins of the MRN complex. These interactions occur in normally proliferating cells and are dependent on the GSK3 kinase. Moreover, we show that βTrCP enhances the recruitment of the MRN complex to chromatin through MRE11, thereby promoting the efficient DNA damage repair. Hence, alterations in βTrCP function affecting MRN dynamics could have severe consequences for the cell homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47da9e36f36ee547f6e7e42f2ff12a54a26f4687" target='_blank'>
              βTrCP facilitates MRN complex localization on chromatin to enhance DNA repair
              </a>
            </td>
          <td>
            Alejandro Belmonte-Fernández, Joaquín Herrero-Ruíz, Carmen Sáez, M. Japón, M. Mora-Santos, F. Romero
          </td>
          <td>2025-07-11</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Epstein–Barr virus, the first identified human DNA tumour virus, is detectable in more than 90% of nasopharyngeal carcinoma patients in endemic regions. The 3D chromosome conformation analysis reveals that virus‒host chromatin interactions induce the spatial reorganisation of loops and compartments, resulting in “enhancer infestation” and switch of “H3K27 bivalency” at EBV-interacting regions. Through this mechanism, EBV hijacks KDM5B, a gatekeeper of genome stability, and its binding targets, leading to aberrant activation of an EBVIR-enhancer-KDM5B signature. Cancer cells with this signature present increased MYC activation, DNA damage responses, and epigenetic plasticity of epithelial-immune dual features with metastatic potential. Our multicentre multiomics study confirms that this signature is the prerequisite for chromosome instability and can be utilised as a risk factor for distant metastasis. This study highlights a mechanism in which latent viral episomes can alter the host high-order epigenotype, promoting transcriptional rewiring and metastasis in NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d24b257eddf1aac8b8f7ed9ed3a3ecfd24278af6" target='_blank'>
              Virus-human chromatin interactions reorganise 3D genome and hijack KDM5B for promoting metastasis in nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            D. Chung, Zhaozheng Hou, Ying Wang, Kazi Anisha Islam, Songran Liu, Jiayan Liu, Larry Ka-Yue Chow, Yuki Yuk-Wun Wong, Cyrus Paak-Ting Chak, Yingpei Zhang, Lanqi Gong, Ziyang Qi, K. Cheng, Zhuoyou Yu, Ping Feng, Zilu Huang, R. Ngan, Xin-Yuan Guan, W. Ng, Zhonghua Liu, A. Tsang, D. Kwong, Ann Wing-Mui Lee, V. H. Lee, Honglin Chen, Yunfei Xia, Wei Dai
          </td>
          <td>2025-08-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1369dba20e9061226bb6a83875ed3d5ee27fc660" target='_blank'>
              Saturation genome editing of BRCA1 across cell types accurately resolves cancer risk
              </a>
            </td>
          <td>
            Phoebe Dace, Nicole M. Forrester, Maria Zanti, L. Cubitt, Chloé Terwagne, Megan Buckley, Elke M. van Veen, Thomas Stuart Wilson, Paola Scaffidi, K. Michailidou, Gregory M. Findlay
          </td>
          <td>2025-08-16</td>
          <td>None</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Abstract Human cytomegalovirus (HCMV) genetic manipulation traditionally relies on bacterial artificial chromosome (BAC) recombineering, necessitated by its large ~236 kb genome. This approach is limited by the scarcity of HCMV strains engineered into BACs and often requires the deletion of ‘non-essential’ genes to accommodate the BAC cassette. We developed a novel approach using temperature-sensitive Sendai virus (SeV) vectors to deliver CRISPR/Cas9 for targeted HCMV genome editing without these constraints. This system achieves high editing efficiency (80–90%) in fibroblasts, epithelial cells and endothelial cells without BAC intermediates. As proof of principle, we targeted the HCMV (TB40/E strain) pentamer complex (PC) genes UL128 and UL130, crucial for viral entry into non-fibroblast cells. Edited viruses showed significantly reduced infectivity in epithelial cells, confirming functional disruption of the PC. Plaque purification yielded isogenic clones with phenotypes comparable to AD169, a naturally PC-deficient strain. Furthermore, multiplexed editing created precise 663 bp deletions in over 60% of viral genomes. Importantly, this method enables HCMV editing in physiologically relevant cell types without fibroblast passaging, which typically introduces mutations. This SeV-Cas9 system represents a significant advancement for studying HCMV biology in diverse cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5126745691429dc45be87ace4ed9512b7bb87284" target='_blank'>
              Precision engineering of human cytomegalovirus without BAC constraints: a Sendai virus-delivered CRISPR/Cas9 approach
              </a>
            </td>
          <td>
            Jillian C. Carmichael, C. Stevens, Kristina E. Atanasoff, S. Kowdle, Rebecca A. Reis, Domenico Tortorella, Benhur Lee
          </td>
          <td>2025-07-01</td>
          <td>The Journal of General Virology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is a highly lethal, aggressive malignancy. Little is known about the evolutionary trajectories of HCC and how clinical decision-making could be informed based on biopsies of the initial tumour. Here, we report the whole-exome sequencing of a unique series of resected HCC tumours and matched recurrences. This cohort included patients who received a liver transplant and who were immunosuppressed at time of recurrence, in comparison to patients who underwent liver resection for HCC and immunocompetent at time of recurrence, therefore facilitating analyses of immune selection in driving evolutionary divergence. We find extensive evolutionary divergence between baseline and recurrent tumours, with the majority of mutations in our cohort being private, in the process informing sampling guidelines for precision oncology in this disease. We also find no evidence that immunosuppression relaxes immune selection pressures, given the absence of a genomic footprint reflecting the presentation of neoantigens or altered dynamics of genomic evolution. We attribute this to the presence of genetic lesions that confer the capabilities of immune evasion in these tumours prior to transplantation, and then validate the link between immune selection pressures and the emergence of these lesions in publicly available HCC datasets. Our findings point to HCC as a cancer with extensive evolutionary divergence over time, partly defined by an irreversible, genetically determined trajectory towards immune escape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9c9a24625027fee25b759620a813b537ff8ac7" target='_blank'>
              Evolutionary dynamics of recurrent hepatocellular carcinoma under divergent immune selection pressures
              </a>
            </td>
          <td>
            Ankur Chakravarthy, Elisa Pasini, Xun Zhao, Jeffrey To, Shu Yi (Roxana) Shen, Sandra Fischer, A. Ghanekar, Arndt Vogel, Robert C. Grant, Jennifer Knox, Gonzalo Sapisochin, Gregory J. Gores, Daniel De Carvalho, Mamatha Bhat
          </td>
          <td>2025-08-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Human cytomegalovirus extensively alters nuclear organization and the cellular transcriptome, yet understanding of these genome-wide events remains relatively limited. Here, chromatin conformation capture (Hi-C) revealed how cytomegalovirus alters chromosome organization at both large- and small-scales. Nascent transcriptomics further revealed how transcriptional changes correlate with genomic reorganization, while also uncovering infection-induced transcriptional dysregulation that contributes to the induction of neuronal gene signatures in infected fibroblasts. Combining Hi-C and Cleavage Under Targets & Release Using Nuclease (CUT&RUN) we find that viral genomes preferentially localize to highly euchromatic compartments, further dysregulating transcription of host genes. Finally, RNAi-mediated depletion of two key effectors of microtubule-based forces that are exerted on the nucleus provides insights into their diverging roles in regulating compartment-scale contacts and viral genomic interactions with host chromatin. Combined, we reveal the extent to which HCMV interacts with and alters host chromatin and transcription, and the influence of microtubule mechanotransduction on these processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0c2779fbea7b388894e9412fc49c8051594da3" target='_blank'>
              Microtubule mechanotransduction refines cytomegalovirus interactions with and remodeling of host chromatin
              </a>
            </td>
          <td>
            Celeste D Rosencrance, Derek Walsh
          </td>
          <td>2025-08-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="ABSTRACT Purpose Metastatic castration-resistant prostate cancer (mCRPC) remains a significant therapeutic challenge and a leading cause of cancer-related mortality in men. PARP inhibitors like Olaparib are effective in homologous recombination repair (HRR)-deficient tumors, but resistance often arises through DNA repair restoration. This study explores the role of the structure-specific endonuclease subunit SLX1, a catalytic subunit of the SLX1-SLX4 endonuclease complex, in Olaparib resistance. Methods Data from The Cancer Genome Atlas (TCGA) were used for expression and survival analyses. The CRPC cell line DU145, which harbors BRCA1 and BRCA2 mutations, was used as a cell model for both in vitro and in vivo studies. Results Elevated SLX1A expression in prostate cancer tissues was associated with significantly reduced progression-free and overall survival. SLX1 protein was upregulated in androgen-resistant prostate cancer cell lines (DU145, 22RV1, PC3) and further increased in Olaparib-resistant DU145 (DU145-OR) cells. Silencing SLX1 via shRNA enhanced Olaparib sensitivity, reducing colony formation and increasing DNA damage and apoptosis in DU145 and DU145-OR cells. Mechanistically, SLX1 knockdown disrupted SLX4 interactions with critical DNA repair proteins (ERCC1-XPF, PLK1, and TOPBP1), impairing DNA repair complex stability. In vivo, SLX1-silenced DU145 xenografts treated with Olaparib showed significantly reduced tumor growth with decreased Ki-67 expression and increased apoptosis/necrosis compared to controls. Conclusion This study highlights SLX1 as both a prognostic marker and potential therapeutic target to enhance PARPi efficacy in advanced prostate cancer. Targeting SLX1 may be a promising strategy to overcome Olaparib resistance in mCRPC patients with homologous recombination deficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02640e28be0a0d352e94d2990883b754339fb49e" target='_blank'>
              SLX1 silencing overcomes Olaparib resistance in metastatic castration-resistant prostate cancer by disrupting SLX4-mediated DNA repair complexes
              </a>
            </td>
          <td>
            Xin Zhao, Shiyun Feng, Xiaoping Nitie, Shibu Muluo, Yi Lei
          </td>
          <td>2025-08-11</td>
          <td>Cancer Biology & Therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Zwilch Kinetochore Protein(ZWILCH) has been reported to prevent cells from prematurely exiting mitosis. However, the underlying mechanisms or involvement of ZWILCH in the tumor immune microenvironment in various cancers remain largely unknown. Methods Generalized dysregulation of ZWILCH was observed through the whole transcriptome analysis in this study. The spatial transcriptome analysis was utilized to identify expressed regions of ZWILCH. Next, cells that mainly expressed ZWILCH in the tumor microenvironment were determined using the single-cell transcriptome analysis. Also, the “cellchat” R package was applied to estimate the effect of ZWILCH on malignant cell communication. Combining multiple analytic approaches including GSEA, GSVA, KEGG enrichment analysis, and Aucell, with TCPA functional protein data, Genome-wide CRISPR screening, potential functions of ZWILCH and the pathways in which ZWILCH participated were thoroughly exploited. Univariate Cox regression analysis calculated the association between ZWILCH and cancer patients’ adverse outcomes. Results ZWILCH is universally highly expressed in tumors. The spatial transcriptome analysis showed that ZWILCH overexpression comes from the tumoral region or mixed tumoral region. At the single-cell level, ZWILCH is chiefly expressed by malignant cells and proliferative T cells. The expression of ZWILCH mRNA is positively correlated with cell proliferation, repair of DNA damage, and cell cycle score. Plenty of metabolic pathways are inhibited in patients with high expression of ZWILCH. Moreover, after ZWILCH knockout, a large number of cancer cell lines are stagnated, inhibited, or died. Additionally, the malignant cells with positive expression of ZWILCH have a stronger ability for cell communication. In short, ZWILCH is meant to be a risk factor for clinical outcomes of multiple tumors. Conclusions ZWILCH is a promising therapeutic target that influences patient prognosis by participating in cell proliferation, cell communication, and reshaping the tumor microenvironment across different cancers. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02981-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73f14c1c066485285bab172ec02a8c4271d565b9" target='_blank'>
              Zwilch kinetochore protein affects the prognosis of cancer patients by participating in cell proliferation, enhancing cell communication, and reshaping the tumor microenvironment
              </a>
            </td>
          <td>
            Long Yao, Lianpo Liu, Jinsong Wu, Yunlong Huang, Renquan Zhang, Haoxue Zhang
          </td>
          <td>2025-07-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3dcbc0047fe628bc7d0d1f2c63478bc9274ca9ad" target='_blank'>
              Punctuated mutagenesis promotes multi-step evolutionary adaptation in human cancers
              </a>
            </td>
          <td>
            Christopher J Graser, Wenbo Wu, Cole Christini, Mia Petljak, Franziska Michor
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e6f16ce9f63cfac7b2ed1f793c4962d5de13510" target='_blank'>
              Telomeric Repeat-Containing lncRNA TERRA Targets Non-Telomeric DNA in Trans via R-Loops
              </a>
            </td>
          <td>
            Saifeldin N. Shehata, Akram Mendez, Tanmoy Mondal, Roshan Vaid
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Introduction Ovarian cancer (OC) is one of the leading causes of cancer-related death in women worldwide. Treatment with PARP-inhibitors has significantly improved survival in patients with high-grade serous cancer (HGSC) bearing BRCA1/2 mutations (~22% of the cases), and/or homologous recombination deficiency (HRD, ~50%). Unfortunately, limited therapeutic alternatives are available for BRCA1/2 wild type/HR proficient HGSC patients, who usually exhibit resistance to standard treatments and poor prognosis. Methods Herein, we present the results of a comprehensive genomic profiling (CGP) analysis using the Oncomine Comprehensive Assay® (OCA) Plus in a consecutive retrospective cohort of 102 HGSC patients characterized in our institution. Results Genomic instability, measured by Genomic Instability Metric (GIM) >16, was found in 40% of the cases and was significantly associated with BRCA1/2 mutations (p=0.009), with a better prognosis in terms of recurrence-free survival (p=0.01). CCNE1 amplification was observed in 29% of cases and was negatively correlated with BRCA1/2 mutations (p=0.001), without any association with GIM, supporting CCNE1 as a strong and independent driver of tumorigenesis. Additionally, CCNE1 amplification was validated with fluorescent in situ hybridization (FISH), supporting the analytical robustness of NGS data (rho=0.93), and investigated by immunohistochemistry (IHC), revealing that CCNE1 protein overexpression was observed in the absence of gene amplification in 45% of cases. Discussion Our real-world study supports the clinical utility of the GIM metric and the analytical validity of CCNE1 amplification, a new promising biomarker for personalizing treatment in HR proficient HGSC patients. The discordance between CCNE1 amplification and protein expression raises intriguing questions about the mechanisms of CCNE1-driven tumorigenesis and warrants further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef09810cfae4b459a30bafb300e6ffa23d0d9a9a" target='_blank'>
              Genomic instability and CCNE1 amplification as emerging biomarkers for stratifying high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            E. Conca, Daniele Lorenzini, E. Minna, Luca Agnelli, Matteo Duca, Marco Gentili, Beatrice Bodini, M. Polignano, Mara Mantiero, Silvia Damian, A. Devecchi, Gianpaolo Dagrada, Rita Carminati, Alice Ardore, Francesca Barbetta, Nathalia Brito Da Chuna, Andrea Guerrizio, A. Busico, I. Capone, Alberta Piccolo, E. Tamborini, Federica Perrone, Massimo Milione, Biagio Paolini, A. Vingiani, Francesco Raspagliesi, F. de Braud, Giancarlo Pruneri
          </td>
          <td>2025-08-06</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Summary Intratumoral heterogeneity, originating from genetic, epigenetic, and phenotypic cellular diversity, is pervasive in cancer. As these heterogeneous states employ diverse mechanisms to promote tumor progression, metastasis, and therapy resistance, emergence of cancer heterogeneity is one of the most significant barriers to curative treatment. Here we leverage deep learning approaches to develop a high-throughput image-analysis paradigm with subcellular resolution that quantifies and predicts colorectal cancer (CRC) patient outcome based on tissue architecture and nuclear morphology. We further combine this approach with spatial transcriptomics, multiplex immunohistochemistry, and patient-derived organoids to uncover a dynamic, co-evolutionary relationship between tumor architecture and cell states. We identify clinically relevant architectural interfaces in CRC tissue that diversify cellular identities by favoring distinct cancer stem cell states and thus promote evolution of tumor heterogeneity. Specifically, tissue fragmentation and associated compressive forces promote loss of classic stem cell signature and acquisition of fetal/regenerative stem cell states, which initially emerges as a hybrid state with features of epithelial-to-mesenchyme (EMT) transition. Additional tumor stroma communication then diversifies these states into distinct stem cell and EMT states at the invasive margins of tumors. Reciprocally, Wnt signaling state of tumor cells tunes their responsiveness to tissue architecture. Collectively, this work uncovers a feedback loop between cell states and tissue architecture that drives cancer heterogeneity and cell state diversification. Machine learning-based analyses harness this co-dependency, independently of mutational status or cancer stage, to predict patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e8d83b8caf4ac682c803318ebb577d7836c261e" target='_blank'>
              Colorectal cancer heterogeneity co-evolves with tumor architecture to determine disease outcome
              </a>
            </td>
          <td>
            Fabien Bertillot, Solène Hervé, Nelly Drobjazko, Noemi Kedei, Mona El Gendi, Maria O. Hernandez, Kai Kruse, E. Erkan, Mirjam I. Binner, Karolina Punovuori, T. Pellinen, M. Ahtiainen, Jan Böhm, J. Mecklin, Caj Haglund, Jaana Hagström, Mike Nikolaev, N. Gjorevski, Matthias Lütolf, Toni T Seppälä, Sara A. Wickström, Yekaterina A. Miroshnikova
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Colon cancer is a highly aggressive solid tumor. Because most studies have focused on the intrinsic oncogenic pathways of tumors, we focused on the relationship between DNA methylation genes in the tumor immune microenvironment and colon cancer prognosis. We download RNA-seq data from the TCGA dataset. DNA methylation genes were scored using the GSVA method, and this score was subjected to GSEA, GO and KEGG enrichment analysis. Then, in vitro experiments examined the effect of silencing LARS2 on the proliferation and apoptosis of HCT116 cells. We obtained 2635 genes, 144 were obtained after single factor screening, and 8 genes were obtained through random forest dimensionality reduction. They are LARS2, TEX2, BRIP1, QSOX2, HOOK1, COX19, NEK4 and STXBP4. Survival analysis indicated that patients with high Riskscore had poor prognosis. There were significant differences in the expression of DNA methylation genes between the two groups with high and low Riskscore. The immune checkpoints of different Riskscore groups include Antigen present, Ligand, Receptor, Co-inhibitor, Co-stimulator, Other, and Cell adhesion. There were significant differences in the expression of genes and DNA integrity checkpoint, histone deacetylation, mitotic DNA damage checkpoint, TGF-beta signaling pathway, Platinum drug resistance and Protein processing in endoplasmic reticlum pathway at the level of Antigen present. Silencing of LARS2 decreased the proliferative capacity and increased the apoptosis rate of HCT116. Our findings suggest that LARS2 methylation could affects colon cancer development. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10669-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/146fe9925f0f237535e6dcfdafa1d3f52c10c07d" target='_blank'>
              LARS2 DNA methylation predicts the prognosis in colon cancer
              </a>
            </td>
          <td>
            Yingzhou Fu, Wei Xie, Qifan Ren, Jiao Wu, Ying Long, Ying Liu
          </td>
          <td>2025-07-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Significance The protein kinase gene CDK12 is frequently altered genetically in human cancers including prostate, ovarian, and breast cancer. Loss of CDK12 has been linked to enhanced anticancer immunity in prostate cancer. Here, we report that CDK12 binds to and phosphorylates the nuclease/helicase DNA2, heralding phosphorylation-dependent proteasomal degradation of DNA2. On the contrary, loss of CDK12 results in DNA2 protein accumulation, DNA damage, cGAS–STING activation, and immunogenicity. Our findings identify DNA2 as a key signaling module connecting CDK12 deficiency to genomic instability and highlight the relevance of immune response to CDK12-mutant tumors, a trait distinct from CDK12-intact tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f37f20719fbb3acf29c2fa7f8d91b4dd2f7baf76" target='_blank'>
              DNA2 protein destruction dictates DNA hyperexcision, cGAS–STING activation, and innate immune response in CDK12-deregulated cancers
              </a>
            </td>
          <td>
            Rui Sun, Peng Jiang, Zhijun Wang, Binyuan Yan, Shouhai Zhu, Zhenlin Huang, Jianong Zhang, Donglin Ding, Xiang Li, Liguo Wang, Zhenkun Lou, Baiye Jin, Jun Pang, Haojie Huang, Dan Xia
          </td>
          <td>2025-07-07</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea1387277252c00fad4b66b55a8349b13921b432" target='_blank'>
              MitoPerturb-Seq identifies common and gene-specific single-cell responses to mitochondrial DNA depletion and heteroplasmy
              </a>
            </td>
          <td>
            Stephen P. Burr, Kathryn Auckland, Angelos Glynos, Abhilesh S. Dhawanjewar, Wei Wei, Cameron Ryall, Antony M. Hynes-Allen, Malwina Prater, Matylda Sczaniecka-Clift, Julien Prudent, Patrick F. Chinnery, Jelle van den Ameele
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="ABSTRACT Objective To illustrate a new barcoded orthotopic patient‐derived xenograft (PDX) mouse model where one can investigate phenotypic effects of single‐cell level gene manipulation in a pooled format. To address some concerns of current PDX mouse models of head and neck squamous cell carcinoma (HNSCC): (1) genomic evolution with passage by generating high‐purity cancer cells, which can also be utilized for other downstream applications, including cell culture‐based studies, and (2) cost‐effectiveness of current PDX models. Methods Two‐millimeter tumor cubes from nine patients were implanted into immunodeficient mouse flanks subcutaneously. Purified tumor cells were obtained from subcutaneous xenografts. Various numbers of purified tumor cells were then injected into the lingual tissue of immunodeficient mice, and the lowest amount of cells needed to achieve a 100% orthotopic engraftment rate were identified. Clonal stability was tested using a lentiviral barcoding system. The orthotopic PDXs' genetic landscapes were characterized using whole exome sequencing. Results This approach yielded an overall engraftment rate of 88.9%. The purification process increased cancer cell purity from 34% to 92%. Lingual injection of 100,000 purified tumor cells achieved a 100% orthotopic engraftment rate from purified subcutaneous PDX tumor cells while maintaining clonal and genetic stability. Conclusion Our study presents a barcoded orthotopic patient‐derived xenograft model for head and neck squamous cell carcinoma with clonal stability. This model provides a way to study phenotypic effects of single cell level gene manipulation in a pooled format. The method can be adapted for in vitro work as well.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/770f65963b9ed4389a6e68906eb989805afcd015" target='_blank'>
              Barcoded Orthotopic Patient‐Derived Head & Neck Squamous Cell Carcinoma Model Demonstrating Clonal Stability and Maintenance of Cancer Driver Mutational Landscape
              </a>
            </td>
          <td>
            Peiran Zhou, Claire B Mills, Zhao Ming Dong, Brittany R Barber, Slobodan Beronja
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="2’-deoxyuridine (dU) is a common DNA lesion resulting from cytosine deamination and from dUMP incorporation by DNA polymerases, both of which are prevalent in cancer. The primary mechanism that repairs dU lesions in genomic DNA is base excision repair initiated by Uracil-DNA Glycosylase 1 (UNG1). We generated Ung knockout mouse B16 melanoma cells to investigate the consequences of UNG deficiency in a well-characterized, immunoproficient, syngeneic mouse cancer model. We show that UNG-deficient (ΔUNG) B16 tumors have altered growth kinetics in vivo and that their delayed growth is T-cell dependent. Immune profiling revealed reduced CD8+ T cell infiltration but augmented CD4+ Th1 responses in ΔUNG tumors. In vitro, ΔUNG tumor cells exhibit strongly suppressed cell-intrinsic type-I interferon, type-II interferon, and inflammatory signaling gene expression signatures as well as altered cytokine and chemokine secretion. In vivo, ΔUNG tumors exhibit a modified inflammatory cytokine and chemokine milieu. Furthermore, ΔUNG tumor cells have altered sensitivity to exogenous interferons in vitro, with increased sensitivity to IFN-γ but decreased sensitivity to IFN-α/β. Collectively, our data show that tumor cell-specific UNG deficiency results in an altered tumor microenvironment in vivo and provide proof-of-concept data for the use of UNG inhibitors to modulate inflammatory pathways in tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bc26c13b6ac7757d4f5f6a8fbbc2a735a2ecafa" target='_blank'>
              Uracil-DNA glycosylase deficiency is associated with repressed tumor cell-intrinsic inflammatory signaling and altered sensitivity to exogenous interferons
              </a>
            </td>
          <td>
            F. Vendetti, Pinakin Pandya, Carina R. Sclafani, Reyna E. Jones, Daniel Ivanov, Robert W. Sobol, Christopher J. Bakkenist
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Retroviral transduction of cancer cell lines has been used to find genes related to therapy resistance. Characterization of virus integration sites (VIS) pinpointing these genes has been cumbersome. This study defines a sequencing-based protocol to rapidly characterize genomic loci near VIS. The protocol selectively amplifies VIS-genome junctions using the retroviral vector neomycin (NEO) gene, Genomic Walker Adapter approach, linker-mediated NEO-PCR (LM-NEO-PCR) or biotinylated NEO-capture, followed by long terminal repeats PCR (LTR-PCR). LTR-genome junctions were sequenced (NGS), reads mapped, quantified, and linked to genes. The protocol was tested on DNA from single clones holding twenty reported VIS loci and on multiplex and diluted clone DNA samples. Our VIS-NGS protocol enriched significantly (p < 0.02) more loci at high reads coverage in samples with VIS compared to negative controls. The protocol found seventeen reported VIS loci (85%) in single clone DNAs, of which fifteen loci (88%) were also detected in multiplex samples. Six loci were evaluated for all dilutions, with three loci detected at lowest 1% clone proportion. The protocol can be conducted in two weeks and successfully found almost all VIS loci in single VIS clones and detected half of the evaluated loci at low clone proportion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc457805a586e034f6d53bc20d3aa71d5bdfc7ad" target='_blank'>
              Novel protocol for mapping virus integration sites in genes involved in therapy resistance
              </a>
            </td>
          <td>
            M. Massimino, Zhangzan Huang, J. Helmijr, Camilla Muritti, Anna Uijtdewillegen, P. Vigneri, John W M Martens, Maurice P H M Jansen
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f36466ce59313bfdfbd1f2c615f7d2712a743403" target='_blank'>
              Zyxin directly binds to chromosomal DNA and is linked with mitochondrial integrity and apoptosis.
              </a>
            </td>
          <td>
            M. Q. Siddiqui, Marcin Wojewodzic, Shridhar Sanghvi, H. S. Pereira, Gunjan Vasudeva, Kieran Meadows, Rutu Prajapati, Mitchell Geeraert, Maulik D. Badmalia, Raymond J Owens, Neil A Hotchin, Harpreet Singh, Trushar R. Patel
          </td>
          <td>2025-07-17</td>
          <td>Biochemistry and cell biology = Biochimie et biologie cellulaire</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Non-homologous end joining (NHEJ) is a DNA repair pathway that directly ligates broken DNA ends without the need for a homologous template, whereas translesion synthesis (TLS) is a DNA damage tolerance mechanism in which specialized DNA polymerases bypass lesions on the template strand. Although both pathways play critical roles in maintaining genome integrity across organisms, they inherently introduce mutations. Here, we investigate how these two pathways contribute to spontaneous and genotoxic stress–induced genomic alterations in the yeast Yarrowia lipolytica. A NHEJ-deficient mutant (ku70) and three TLS-deficient mutants (rev1, rev3, and rad30) are subjected to mutation accumulation experiments, followed by whole-genome sequencing. Our results show that the deletion of KU70 has no significant effect on the rates of spontaneous single-nucleotide variations (SNVs), small insertions and deletions, or chromosomal rearrangements, while the deletion of REV1 and REV3 leads to significant reductions in spontaneous SNV rates. These findings indicate that TLS but not the NHEJ pathway is a major contributor to spontaneous mutagenesis in Y. lipolytica. Moreover, exposure to 0.02% methyl methanesulfonate and 80 J/m2 ultraviolet (UV) radiation resulted in 48- and 107-fold increases in SNV rates, respectively. These induced SNVs were largely dependent on DNA polymerases Rev1 and ζ, further underscoring their central roles in genotoxic stress–induced mutagenesis. We observe that DNA polymerase η can suppress C to T and C to A substitutions while promoting T to C mutations, exhibiting a dual function in regulating mutagenesis under UV treatment. Phenotypic evolution experiments reveal that TLS activity enhances the adaptive potential of Y. lipolytica under oxidative stress, underlying its broader impact on environmental fitness. Together, these findings provide new insights into the distinct roles of the NHEJ and TLS pathways in preserving genome integrity in Y. lipolytica. Key points • The NHEJ pathway has a limited role in spontaneous genomic alterations in Y. lipolytica. • DNA polymerases Rev1 and ζ contribute to most UV- and MMS-induced mutations. • The dual roles of Pol η in UV-induced mutations were revealed. • NHEJ and TLS pathways are crucial to phenotypic evolution of Y. lipolytica. Supplementary Information The online version contains supplementary material available at 10.1007/s00253-025-13575-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15d93f494460655f7859b1e3dd9cb7f63f2e1036" target='_blank'>
              The roles of NHEJ and TLS pathways in genomic alterations and phenotypic evolution in the yeast Yarrowia lipolytica
              </a>
            </td>
          <td>
            Cen Yan, Ye-Ke Wang, Yuan-Ru Xiong, Xin-Qiu Zhou, Yuan-Chun Fang, Ruo-Tian Nie, Cunqi Ye, Ke Zhang, Dao-Qiong Zheng
          </td>
          <td>2025-08-15</td>
          <td>Applied Microbiology and Biotechnology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Diversification and selective propagation are the main driving forces of evolution, resulting in the emergence of organisms possessing various strategies. Here, we conducted in vitro evolution of the replication origin (oriC) under the pressure of an AT-rich mini-chromosome amplification using a reconstituted Escherichia coli replication-cycle reaction (RCR). Using next-generation sequencing, we identified that these evolved oriCs contain select mutations within the duplex unwinding element (DUE) and DnaA-binding site (DnaA box) regions, which are crucial for replication initiation. Real-time detection of RCR amplification (real-time RCR) revealed that the DUE mutations, which decreased the GC content, along with the introduction of a specific A/T sequence in DUE-M and a consecutive KAK (K = T or G) motif in DUE-R, enhanced the RCR amplification efficiency compared to the wild-type oriC (oriCwt). A competitive amplification assay also elucidated that the DnaA box mutations confer a competitive advantage over coexistent oriCwt. Although these DUE and DnaA box mutations were selected through the same amplification reaction, they exhibited distinct competitive amplification strategies. The DUE mutant represents a faster propagation strategy (r-strategy), while the DnaA box mutant represents an adaptive strategy with a competitive advantage (K-strategy), representing the r/K-selection theory in molecular evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76d8adc8e642b9b4fca6ba9da0b2f823fd1341a1" target='_blank'>
              Adaptive evolution of oriC through in vitro propagation of a mini-chromosome in RCR
              </a>
            </td>
          <td>
            Shota Suzuki, M. Su’etsugu
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Ubiquitin carboxyl terminal hydrolase L3 (UCHL3) is a deubiquitinase belonging to UCH protease family. Previous studies have demonstrated that abnormal expression of UCHL3 is correlated with the development of human diseases, especially in tumors. Notably, UCHL3 exhibits contradictory biological functions across different cancer types. Initial studies identified UCHL3 as a tumor suppressor in prostate cancer. However, emerging evidence suggests that UCHL3 serves as an oncogene in other cancers. Several UCHL3 inhibitors have shown a promising anti-tumor effect in vitro and in vivo. UCHL3 has also been linked to DNA damage repair, which is essential for genome stability. Overexpression of UCHL3 enhances DNA damage repair induced resistance to chemotherapy or radiotherapy in certain tumor types. This review aims to summarize the promoting role of UCHL3 in DNA damage repair and its dual, paradoxical roles in tumor progression, along with the associated mechanisms, and to provide insights into the use of UCHL3 as a therapeutic target for overcoming DNA damage repair-mediated resistance to chemotherapy and radiotherapy, as well as for treating tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce3d5f1b68a091ec87b927bf44542627cac87c63" target='_blank'>
              UCHL3: a crucial deubiquitinase in DNA damage repair and tumor progression
              </a>
            </td>
          <td>
            Jiahao Liu, Shulu Hu, Junqi Xiao, Jumei Zhou
          </td>
          <td>2025-07-21</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Current cancer chemotherapy uses etoposide to induce DNA double-strand breaks for cell killing, which can cause fluctuations in stemness promoters like OCT4A, SOX-2, or NANOG in relation to senescence promoters p21Cip1, p27, or p16ink4A, and mitigate anti-cancer therapy. These fluctuations take place over several days and are associated with changes in the organization of euchromatin and heterochromatin. Here, we show how the induction of double-strand breaks by etoposide led to the compaction of euchromatin and de-condensation of heterochromatin on day 3. This was accompanied by a co-localization of euchromatin and heterochromatin marks as detected by Single-Molecule Localization Microscopy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a3f66309c268ad41eb3fdc8156d8889c120362" target='_blank'>
              Chemotherapy (Etoposide)-Induced Intermingling of Heterochromatin and Euchromatin Compartments in Senescent PA-1 Embryonal Carcinoma Cells
              </a>
            </td>
          <td>
            Marc Bayer, Jaroslava Zajakina, Myriam Schäfer, K. Salmina, Fēlikss Rūmnieks, J. Jansons, F. Bestvater, R. Kurg, J. Erenpreisa, Michael Hausmann
          </td>
          <td>2025-07-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="ABSTRACT RNA epigenetics, also referred to as epitranscriptomics, has emerged as a critical regulatory layer in cancer biology, extending beyond the scope of traditional DNA and histone modifications. It encompasses a series of dynamic posttranscriptional processes—including RNA biosynthesis, splicing, transport, stability, degradation, translation, and chemical modifications—orchestrated by RNA‐binding proteins (RBPs) and noncoding RNAs (ncRNAs). Collectively, these mechanisms influence mRNA fate, shape transcriptional output, and reprogram the tumor microenvironment. Importantly, both coding RNA and ncRNA are themselves subjected to epigenetic regulation, forming intricate feedback loops that contribute to oncogenesis, immune evasion, metastasis, and therapeutic resistance. In this review, we systematically synthesize the current understanding of RNA metabolism and RNA epigenetic modifications during tumor progression, with a particular focus on the roles of RBPs and RNA modifications. Furthermore, we highlight recent advances in RNA‐based therapeutic strategies, including mRNA vaccines, antisense oligonucleotides, siRNAs, and circRNA scaffolds. These innovative approaches offer promising avenues for targeting transcriptionally active yet genomically “undruggable” cancer drivers. Together, our synthesis provides a comprehensive framework for understanding RNA epigenetics in tumor biology and lays the groundwork for precision oncology guided by transcriptome plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de7fa009ea26201e23b24a2c7b7bd45eb23b5831" target='_blank'>
              RNA Epigenetics in Cancer: Current Knowledge and Therapeutic Implications
              </a>
            </td>
          <td>
            Shanhe Huang, Zonglin Li, Weilong Lin, Ruihui Xie, Hai Huang
          </td>
          <td>2025-08-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="CRISPR/Cas technology has revolutionized genome engineering, unlocking unprecedented therapeutic potential. However, beyond well-documented concerns of off-target (OT) mutagenesis, recent studies reveal a more pressing challenge: large structural variations (SVs), including chromosomal translocations and megabase-scale deletions, particularly in cells treated with DNA-PKcs inhibitors. These undervalued genomic alterations raise substantial safety concerns for clinical translation. As more CRISPR-based therapies progress toward the clinic, understanding and mitigating these risks is paramount. Here, we review emerging evidence on on-target aberrations and chromosomal translocations, identify key gaps in our understanding of the DNA repair pathways underlying these adverse effects, and discuss strategies to improve the safety of genome editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c34e819820702e599ff9b58db37b63e7aab2e777" target='_blank'>
              The hidden risks of CRISPR/Cas: structural variations and genome integrity
              </a>
            </td>
          <td>
            Clotilde Aussel, Toni Cathomen, Carla Fuster-García
          </td>
          <td>2025-08-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background: The long-term goal of chronic hepatitis B research is a functional cure (HBsAg seroclearance). Although currently used nucleos(t)ide analogs can efficiently inhibit viral replication, they do not reduce viral RNAs or proteins produced from covalently closed circular DNA (cccDNA), and rarely achieve a functional cure. To overcome this situation, revealing the mode of the existence of cccDNA is required, including identifying the interreacting proteins with cccDNA. Here, we aimed to identify novel proteins that interact with cccDNA. Methods: Using an in vitro HBV infection model and a sequence-specific proximity labelling method consisting of dead Cas9 and biotin identification (BioID2), we comprehensively determined proteins that possibly interact with cccDNA. After identifying the candidate proteins, the HBV RNA transcription levels were examined by knocking out the associated genes. Results:﻿ We identified ABHD14B as a protein that interacts with cccDNA and inhibits HBV RNA transcription from cccDNA. ABHD14B decreases the acetylation levels of histone proteins that control the transcription levels of HBV RNA in cccDNA. Moreover, ABHD14B interacts with TFII-I, which binds directly to cccDNA in a sequence-dependent manner. These results suggest that the host protein, ABHD14B, is recruited to cccDNA via the TFII-I protein, inhibiting HBV RNA transcription from cccDNA by deacetylating cccDNA histones. Conclusions: ABHD14B was newly identified as a suppressor of HBV RNA transcription from cccDNA, which may improve our understanding of the mode of existence of cccDNA, providing a basis for development of a functional cure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bd95fe07e8a95469056b65d704b3979e1b6c0b9" target='_blank'>
              CRISPR-mediated proximity labeling unveils ABHD14B as a host factor to regulate HBV cccDNA transcriptional activity
              </a>
            </td>
          <td>
            Kazuma Sekiba, Nozomi Miyake, Yu Miyakawa, Chikako Shibata, Takahiro Seimiya, T. Kishikawa, Mitsuhiro Fujishiro, Motoyuki Otsuka
          </td>
          <td>2025-07-14</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Gene expression is finely controlled by the abundance and activation status of transcription factors and their regulators, as well as by a number of reversible modifications of DNA and histones that are commonly referred to as epigenetic marks. Such alterations (i.e., methylation, acetylation, and ubiquitination) are catalyzed by an array of dedicated enzymes with antagonistic activity, including methyltransferases and demethylases, acetyltransferases and deacetylases, as well as ubiquitin ligases and deubiquitinating enzymes. The epigenetic control of transcription is critical not only for embryonic and postembryonic development but also for the preservation of homeostasis in all adult tissues. In line with this notion, epigenetic defects have been associated with a variety of human disorders, including (but not limited to) congenital conditions as well as multiple hematological and solid tumors. Here, we provide an in-depth discussion of the impact of epigenetic alterations on cancer stemness, i.e., the ability of a small population of poorly differentiated malignant cells to (1) self-renew while generating a more differentiated progeny, and (2) exhibit superior tumor initiating/repopulating potential along with exceptional plasticity and improved resistance to environmental and therapy-elicited stress. Moreover, we critically evaluate the potential and limitations of targeting epigenetic modifiers as a means to eradicate cancer stem cells for therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3477193515567caf9d7adf26496dfc00e067cba2" target='_blank'>
              Epigenetic regulation of cancer stemness
              </a>
            </td>
          <td>
            C. Galassi, G. Manic, Manel Esteller, Lorenzo Galluzzi, Ilio Vitale
          </td>
          <td>2025-08-01</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Wilms tumor (WT), the most common pediatric renal malignancy, exhibits a relatively low mutational burden compared to adult cancers, which hinders the development of targeted therapies. To elucidate the molecular landscape of WT, we perform integrative proteomic, phosphoproteomic, transcriptomic, and whole-exome sequencing analyses of WT and normal kidney tissue adjacent to tumor. Our multi-omics approach uncovers prognostic genetic alterations, distinct molecular subgroups, immune microenvironment features, and potential biomarkers and therapeutic targets. Proteome- and transcriptome-based stratification identifies three molecular subgroups with unique signatures, correlating with different histopathological subtypes and putative cellular origins at different stages of embryonic kidney development. Notably, we identify EHMT2 as a promising prognostic biomarker and therapeutic target associated with epigenetic regulation and Wnt/β-catenin pathway. In this work, we provide a comprehensive molecular characterization of WT, offering valuable insights into its pathogenesis and a foundational resource for future therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3be681b81ddbc1d74bdc72b4011124ef4e90944" target='_blank'>
              Integrative proteogenomic characterization of Wilms tumor
              </a>
            </td>
          <td>
            Cheng Cheng, Li Zhang, Xiaofeng Chang, Kai Chen, Tian He, Jia Shi, Fan Lv, Lijia Pan, Yangkun Wu, Qianqian Cheng, Dong Ren, Y. Guo, Weiping Zhang, Huanmin Wang, Tieliu Shi, Jing Li, Xinxin Ni, Yeming Wu, Yaqiong Jin, Zhixiang Wu
          </td>
          <td>2025-08-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Structural variations (SVs) represent genomic variations that involve breakage and rejoining of DNA segments. SVs can alter normal gene dosage, lead to rearrangements of genes and regulatory elements within a topologically associated domain, and potentially contribute to physical traits, genomic disorders, or complex traits. Recent advances in sequencing technologies and bioinformatics have greatly improved SV detection and interpretation at unprecedented resolution and scale. Despite these advances, the functional impact of SVs, the underlying SV mechanism(s) contributing to complex traits, and the technical challenges associated with SV detection and annotation remain active areas of research. This review aims to provide an overview of structural variations, their mutagenesis mechanisms, and their detection in the genomics era, focusing on the biological significance, methodologies, and future directions in the field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f005d90812c5790ee3488e2503f389fb8c3ab57b" target='_blank'>
              Structural Variants: Mechanisms, Mapping, and Interpretation in Human Genetics
              </a>
            </td>
          <td>
            Shruti Pande, Moez Dawood, Christopher M. Grochowski
          </td>
          <td>2025-07-29</td>
          <td>Genes</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Pancreatic cancer is known for its lethal condition, with most cases being diagnosed at advanced stage. Recently, liquid biopsy has emerged as a promising tool in cancer detection. Here we develop both an early detection model and a prognostic model for pancreatic cancer using cell-free DNA (cfDNA) end motif, fragmentation, nucleosome footprint (NF), and copy number alteration (CNA) features from plasma cfDNA. A total of 975 individuals were enrolled in our study. We developed an integrated model that demonstrated superior performance in distinguishing patients with early-stage pancreatic cancer from non-cancer controls. Moreover, we find that cfDNA features are associated with prognostic outcomes among pancreatic cancer patients. In this study, a cfDNA-based liquid biopsy signature is established for the early detection and prognostic prediction of pancreatic cancer. CfDNA may become a valuable tool for enhancing early diagnosis and prognosis assessment in this challenging disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2b38ea3144ac464585d901d0e6435daee2918d3" target='_blank'>
              Cell-free DNA testing for the detection and prognosis prediction of pancreatic cancer
              </a>
            </td>
          <td>
            Jianmin Wu, Xiongfei Xu, Qingzheng Zhang, Peilong Li, Tong Wu, Shiwei Guo, Lutao Du, Dongdong Xue, S. Shen, Fuming Sun, Ji Hu, Lu Zheng, Xuan Wu, Jian Bai, Yin Wang, Lin Wu, Wei-wen Liu, Hongyang Wang, G. Jin, Lei Chen
          </td>
          <td>2025-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is the most common renal malignancy. However, the combined clinical and biological scores commonly used to predict the clinical outcome are imperfect and need improvement. The main goal of our study was to assess the effect of mtDNA genetics on the prognosis of ccRCC patients and to explore morphologic correlation. Mitochondrial DNA copy number (mtDNAcn) variation between tumor and paired matched healthy kidney tissue was assessed by real-time quantitative PCR and expressed as a ratio in 105 patients. According to this median ratio, the cohort was divided into two groups: “LOW” (n = 53) and “HIGH” (n = 52). Cancer-Specific Survival (CSS) and Disease-Free Survival were assessed in each group. The tumor samples were classified into two subtypes (Clear or Eosinophilic cells) according to the cytoplasmic morphology. CSS was significantly reduced in the “HIGH” than in the “LOW” group with respective 5-year survival rates: 78.7% (CI 95: 64.8–95.5) and 95.5% (CI 95 87.1–100.0) (Hazard Ratio: 7.4 (CI 95: 1.9–29.9, p = 0.027*) in multivariate analysis, including pathological classification, tumor size, International Society of Urological Pathology grade, lymphovascular invasion, dedifferentiated pattern, necrosis and adjuvant therapy. Next-generation sequencing of mtDNA was performed on 14 tumors and matched healthy kidney tissue. No hotspot mutation or redundant large deletion was found. None of the variants or large deletions identified had an impact on prognosis. MtDNAcn variation in tumor relative to normal kidney appears as an independent prognostic factor in ccRCC, which was also associated with eosinophilic morphology. MtDNA content could be considered an additional prognostic factor, in combination with other predictive parameters. Furthermore, these results underline the importance of the role of mitochondria in ccRCC and the need for further functional studies to understand the pathophysiological mechanisms better and consider therapies targeting mitochondrial metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288c14d8ba3e745b3421617281d655151662ced6" target='_blank'>
              MtDNA copy number enrichment is associated with poor prognosis and eosinophilic morphology in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Sarah Bellal, C. Rolley, Jérémy Richard, Nolwenn Bounaix, Vincent Le Corre, Marie-Christine Copin, Odile Blanchet, Pierre Bigot, Vincent Procaccio, C. Bris
          </td>
          <td>2025-07-23</td>
          <td>Pathology and Oncology Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background
 Breast cancer (BC) remains the second leading cause of cancer-related mortality among women worldwide. Liquid biopsy based on circulating tumor DNA (ctDNA) offers a promising noninvasive approach for early detection; however, differentiating malignant tumors from benign abnormalities remains a significant challenge. Results Here, we developed a multimodal approach to analyze cfDNA methylation and fragmentomic patterns in 273 BC patients, 108 individuals with benign breast conditions, and 134 healthy controls. Genome-wide analyses revealed distinct cfDNA copy number alterations and cytosine-enriched cleavage sites in BC patients. Targeted sequencing further revealed unique methylation patterns, including hypermethylation in GPR126, KLF3, and TLR10 and hypomethylation in TOP1 and MAFB. Our machine-learning model achieved an AUC of 0.90, with 93.6% specificity and 62.1–66.3% sensitivity for stage I–II cancers. In symptomatic populations, sensitivities were 50.0%, 68.2%, and 64.7% for BI-RADS categories 3, 4, and 5, respectively, with 96.1% specificity. Conclusions These findings underscore the potential of cfDNA biomarkers to enhance BC detection and reduce the rate of unnecessary biopsies. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02371-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18aacd3f9fbc0c3fe9a757e1b6d9cf892dfdded8" target='_blank'>
              Multimodal analysis of cell-free DNA enhances differentiation of early-stage breast cancer from benign lesions and healthy individuals
              </a>
            </td>
          <td>
            Thi Tuong Vi Van, T. Tran, Thi Hue Hanh Nguyen, Van Thien Chi Nguyen, Dac Ho Vo, Giang Thi Huong Nguyen, Trong-Hieu Nguyen, Kim Sang To, Anh Luan Nguyen, Cao Hong An Tran, T. Jasmine, Thi Loan Vo, Thi Huong Thoang Nai, T. Tran, My Hoang Truong, Ngan Chau Tran, Thi Loc Le, Thi Hong Nhung Nguyen, Ngoc Hieu Tu, Thanh Son Tran, Bao Toan Le, Van Phong Tang, Pham Thanh Nhan Nguyen, Khac Tien Nguyen, Van Chien Ho, Xuan Vinh Nguyen, Nhu Nhat Tan Doan, Thi Trang Tran, Thi Minh Thu Tran, V. Tran, M. Le, Thi Luyen Vu, B. Tieu, Huu-Phuc Nguyen, Luu Hong Dang Nguyen, Ngoc Minh Phan, Thi Van Phan, T. Do, Thi Huyen Dao, Hung-Sang Tang, Duy Nguyen, Hoa Giang, Minh-Duy Phan, H. Nguyen, Duc Hieu Vo, Levis Tran
          </td>
          <td>2025-08-20</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The nucleotide excision repair (NER) mechanism is responsible for removing bulky DNA damage, such as pyrimidine dimers induced by ultraviolet (UV) light. The NER pathway excises the damaged strand through incisions at the 5’ and 3’ ends of the damage, with the 5’ incision catalyzed by the XPF-ERCC1 endonuclease complex. Here, we identify an XPF ortholog in Trypanosoma brucei, the causative agent of human African trypanosomiasis (sleeping sickness). XPF-deficient parasites exhibit hypersensitivity to UV irradiation and a slower rate of DNA damage repair. Consistent with its role in DNA repair, XPF localizes to the nucleus, associating with nucleoplasmic and nucleolar regions. Additionally, we demonstrate that TbXPF protects against intra- and inter-strand crosslinks induced by cisplatin and mitomycin C, respectively. The presence of a functional NER pathway in trypanosomes suggests that these organisms are susceptible to replication- and transcription-blocking DNA damage in vivo. Under genotoxic stress, genome stability and parasite survival may heavily rely on DNA repair systems such as NER which, for this reason, could be an effective target for chemotherapeutic interventions. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-08659-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbdd4ff3d6a3681f1e698b78ba5f4c131040ae39" target='_blank'>
              Role of the nucleotide excision repair endonuclease XPF in the kinetoplastid parasite Trypanosoma brucei
              </a>
            </td>
          <td>
            Claudia Gomez-Liñan, María Sáez-Maldonado, Laura Montosa-Hidalgo, L. Ruiz‐Pérez, Dolores González-Pacanowska, Antonio E. Vidal
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="ABSTRACT R-loops are three-stranded nucleic acid structures consisting of a DNA/RNA hybrid and a displaced single-stranded DNA. Once considered rare byproducts of transcription, R-loops are now recognized as important regulators of various nuclear processes. In particular, evidence indicates a role for R-loops in regulating DNA methylation dynamics. R-loops have been shown to promote active DNA demethylation—the enzymatic reversal of 5-methylcytosine (5mC) back into cytosine—by recruiting associated proteins, providing an attractive targeting mechanism. Nevertheless, many important aspects of this process, including whether the associated proteins bind to and function on R-loops, remain to be substantiated. In this study, we demonstrate for the first time that thymine DNA glycosylase (TDG), a key enzyme in the active DNA demethylation pathway, binds tightly to R-loops in vitro and can excise DNA demethylation intermediates 5-formylcytosine (5fC) and 5-carboxycytosine (5caC) from DNA in DNA/RNA hybrid duplexes. We also show that R-loops guide the strand-specific activity of TDG at CpG sites, potentially explaining the asymmetric distribution of 5fC/5caC at gene promoters. Furthermore, we provide important mechanistic insights into base excision on DNA/RNA hybrid duplexes using 19 F NMR. Finally, our findings suggest that TDG–R-loop interactions may be widespread in human cells. Collectively, our results provide strong evidence that R-loops play a critical role in DNA demethylation and support a mechanism in which 5fC/5caC are directly removed from DNA/RNA hybrids in cells. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f720c25376cfceef4fd5beb6c69c7fd9ad1d0f50" target='_blank'>
              Thymine DNA Glycosylase Binds to R-Loops and Excises 5-Formyl and 5-Carboxyl Cytosine from DNA/RNA Hybrids
              </a>
            </td>
          <td>
            Baiyu Zhu, L. Pidugu, Mary E. Cook, Xinyu Y Nie, E. A. P. T. Amarasekara, Jerome S. Menet, A. Drohat, Jonathan T. Sczepanski
          </td>
          <td>2025-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract The MYC and RAS oncogenes were found early on to cooperate in the transformation of nonimmortalized primary cells, and in tumor development in transgenic mouse models. MYC prevents RAS-induced senescence. Moreover, tumor regression resulting from the suppression of MYC expression is associated with cellular senescence. How MYC prevents RAS-induced senescence and why suppression of MYC in tumors causes senescence remains to be elucidated. Here, we show that MYC interacts with the Pol β DNA polymerase and stimulates its enzymatic activities to accelerate the repair of oxidative DNA damage by the base excision repair (BER) pathway, thereby enabling RAS-driven cancer cells to avoid senescence and continue to proliferate despite producing excess levels of reactive oxygen species. The carboxy-terminal domain of MYC, which is needed for heterodimerization with its MAX partner and DNA binding, is not required for the DNA repair activity of MYC. Ectopic expression of MYCΔCTD accelerates DNA repair, protects against RAS-induced senescence, and cooperates with RAS in the transformation of primary cells, whereas MYC mutants inactive in DNA repair exhibit weaker or no activity in these assays. These results demonstrate that the function of MYC in BER plays an important role in the MYC–RAS cooperation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c13c7980326ac55cbce4b685dfa42090cd38be7" target='_blank'>
              The function of MYC in base excision repair protects against RAS-induced senescence
              </a>
            </td>
          <td>
            C. C. F. Faraco, Wanting Zhu, Anne-Marie Fortier, Z. Ramdzan, Elise Vickridge, Virakbott Ong, Félix Goudreau, Billel Djerir, Jérôme Bellemare, Shujing Gu, Huanyi Zhang, Brittany L Carroll, Shreya S Krishnan, L. Leduy, Alba Guarné, Morag Park, Alexandre Maréchal, Alain Nepveu
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are distinct nucleic acid secondary structures formed by guanine-rich sequences in both DNA and RNA. These structures readily form and fulfill diverse biological functions. The structural diversity of G4s is influenced by several factors, including their strand orientation, glycosidic bond angles, and loop configurations. G4s are widely distributed in functionally significant genomic regions, including telomeres, promoter regions, exons, 5' untranslated region (5' UTR), intron region, and 3' untranslated region (3' UTR). G4s are implicated in critical biological processes, including telomere elongation, DNA replication, DNA damage repair, transcription, translation, and epigenetic regulation. This overview offers a comprehensive analysis of the determinants of G4 structure and their impact on associated biological processes. Briefly, it describes the effects of G4s on cancers, viruses, and other pathogenic substances. This overview aims to contribute new ideas for the regulation of related mechanisms and their potential impact on the treatment strategies of related diseases. This article is categorized under: RNA Structure and Dynamics > RNA Structure, Dynamics and Chemistry RNA Structure and Dynamics > Influence of RNA Structure in Biological Systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0eb5827dbe52a8e0e2bc9b47ded6461cd148b5b" target='_blank'>
              G-Quadruplexes in Gene Regulation and Cellular Function.
              </a>
            </td>
          <td>
            Qi Wen, Lijin Guo, Farhad Bordbar, Qinghua Nie
          </td>
          <td>2025-07-01</td>
          <td>Wiley interdisciplinary reviews. RNA</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background The mammalian Tumor Susceptibility Gene 101 (TSG101) encodes a protein with diverse functions that control the proliferation and survival of cells, but its role in malignant transformation and cancer development has remained enigmatic. Methods To study the pro-tumorigenic functions of TSG101, we developed a bi-transgenic mouse model that expresses exogenous TSG101 along with a luciferase reporter in a ligand-controlled manner in the mammary gland epithelium. We performed a comprehensive histopathologic, biochemical, and molecular characterization of ductal hyperplasia and mammary tumors. Unsupervised hierarchical clustering based on 1,723 intrinsic genes of ten TSG101-overexpressing cancers alongside 251 tissue samples representing 31 reference mammary tumor models and normal mammary glands was conducted. Results Females overexpressing TSG101 develop ductal hyperplasia, adenomyoepitheliomas, and palpable adenosquamous carcinomas at an average latency of approximately ten months. These metaplastic mammary tumors are comprised of transforming basal and luminal epithelial cells. Using a GFP reporter strain to monitor the transgene activation at the single-cell level, we determined that the epithelial heterogeneity within transforming ducts and ensuing carcinomas originated from the luminal epithelium. At the molecular level, TSG101-induced mammary tumors are triple-negative and exhibit gene expression signatures of Wnt and inflammatory cytokine signaling, which are key regulators of epithelial cell fate. The ligand-controlled downregulation of exogenous TSG101 in established carcinomas led to tumor regression. We demonstrated that the TSG101-mediated activation of PI3K/AKT signaling, as well as upregulation of Cyclin D1 and MDM2, are dependent on the perpetual expression of the TSG101 oncoprotein. Conclusions The collective findings of this study provide in vivo evidence that TSG101 possesses pro-tumorigenic properties that extend to cancer progression and maintenance, suggesting that this protein could be a rational molecular target to prevent and treat a subset of mammary tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02007-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca345fd462ac7f96e7ae408ea2e3598885c80b06" target='_blank'>
              Overexpression of TSG101 causes the development of adenosquamous mammary carcinoma
              </a>
            </td>
          <td>
            Rayane Dennaoui, Patrick D. Rädler, Madison N. Wicker, Kerry Vistisen, R. Ferraiuolo, A. Triplett, Hridaya Shrestha, Tessa A. Liner, Karoline C. Manthey, Hallgeir Rui, R. Cardiff, Teresa M. Gunn, C. Perou, Kay-Uwe Wagner
          </td>
          <td>2025-07-07</td>
          <td>Breast Cancer Research : BCR</td>
          <td>1</td>
          <td>165</td>
        </tr>

        <tr id="

Cancer is characterized by the uncontrolled proliferation of abnormal cells that escape
the body's standard regulatory mechanisms. Under normal conditions, cells grow, divide, and die in
an orderly manner, but cancerous cells lose this control, growing uncontrollably and invading surrounding
tissues. Poly(ADP-ribose) polymerase 1 (PARP1) is a crucial enzyme in this DNA repair
process, helping to fix single-strand breaks. PARP inhibitors (PARPi) are a class of drugs that target
and block the activity of the PARP1 enzyme, impairing its ability to repair DNA damage. By inhibiting
PARP1, these drugs lead to an accumulation of DNA damage in cancer cells, which eventually
becomes overwhelming and leads to cell death. This mechanism is particularly effective in cancers
with deficiencies in other DNA repair pathways, such as those with BRCA1 and BRCA2 gene mutations.
Several PARPi, including Olaparib, Niraparib, and Rucaparib, have been approved by the
U.S. Food and Drug Administration (FDA) for use in treating cancers like breast, ovarian, and prostate
cancer, particularly in patients with BRCA mutations. The evolution and development of PARP
inhibitors have focused on modifying their chemical structure to increase their effectiveness. The
design of PARPi also aims to improve their bioavailability, ensuring that the drugs are effectively
absorbed into the body and can reach the tumor site in sufficient concentrations. Further developments
may also involve combining PARPi with other treatments, such as chemotherapy or immunotherapy,
to enhance their overall efficacy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/751a24c2008023c3f532255ac2cf61fcbded6c01" target='_blank'>
              Advances in Structure-Based PARP1 Inhibitors: Implications for Cancer Treatment
              </a>
            </td>
          <td>
            Hardha Balachandran, Sinjini Das, Gowramma Byran, S. Murugesan, Sathya Narayana Reddy Karri, K. Rajagopal
          </td>
          <td>2025-07-08</td>
          <td>Current Enzyme Inhibition</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cyclin dependent kinases 4 and 6 inhibitors have brought great improvements in the treatment of luminal breast cancer, but resistance is a major clinical hurdle. Multiple biomarkers of resistance have been proposed, but none is currently utilized in clinical practice. By performing single-cell RNA sequencing of seven palbociclib-naïve luminal breast cancer cell lines and palbociclib-resistant derivatives, we show that established biomarkers and pathways related to CDK4/6i resistance present marked intra- and inter- cell-line heterogeneity. Transcriptional features of resistance could be already observed in naïve cells correlating with levels of sensitivity (IC50) to palbociclib. Resistant derivatives showed transcriptional clusters that significantly varied for proliferative, estrogen response signatures or MYC targets. This marked heterogeneity was validated in the FELINE trial where, compared to the sensitive ones, ribociclib-resistant tumors developed higher clonal diversity at genetic level and showed greater trascriptional variability for genes associated with resistance. A potential signature of resistance inferred from the cell-line models, positively enriched for MYC targets and negatively enriched for estrogen response markers, was probed on the FELINE trial, separating sensitive from resistant tumors and revealing higher heterogeneity in resistant versus sensitive cells. These data suggest that heterogeneity for CDK4/6 inhibitors resistant markers might facilitate the development of resistance and challenge the validation of clinical biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab65930181b447a2afea51540e3bba84721ca231" target='_blank'>
              Single-cell transcriptomics reveals biomarker heterogeneity linked to CDK4/6 Inhibitor resistance in breast cancer cell lines
              </a>
            </td>
          <td>
            I. Migliaccio, M. Bonechi, Dario Romagnoli, Giulia Boccalini, F. Galardi, C. Guarducci, A. Nardone, R. Schiff, L. Biganzoli, L. Malorni, M. Benelli
          </td>
          <td>2025-07-31</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Background In-solution hybridization enrichment of genetic markers is a method of choice in paleogenomic studies, where the DNA of interest is generally heavily fragmented and contaminated with environmental DNA, and where the retrieval of genetic data comparable between individuals is challenging. Here, we benchmark the commercial “Twist Ancient DNA” reagent from Twist Biosciences using sequencing libraries from ancient human samples of diverse demographic origin with low to high endogenous DNA content (0.1–44%). For each library, we tested one and two rounds of enrichment and assessed performance compared to deep shotgun sequencing. Results We find that the “Twist Ancient DNA” assay provides robust enrichment of approximately 1.2M target SNPs without introducing allelic bias that may interfere with downstream population genetics analyses. Additionally, we show that pooling up to 4 sequencing libraries and performing two rounds of enrichment is both reliable and cost-effective for libraries with less than 27% endogenous DNA content. Above 38% endogenous content, a maximum of one round of enrichment is recommended for cost-effectiveness and to preserve library complexity. Conclusions In conclusion, we provide researchers in the field of human paleogenomics with a comprehensive understanding of the strengths and limitations of different sequencing and enrichment strategies, and our results offer practical guidance for optimizing experimental protocols. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03622-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c477c495125e3cd66f843708e054b0141348cbd8" target='_blank'>
              Optimized in-solution enrichment of over a million ancient human SNPs
              </a>
            </td>
          <td>
            Roberta Davidson, Xavier Roca-Rada, Shyamsundar Ravishankar, Leonard Taufik, Christian Haarkötter, Evelyn Collen, Matthew P. Williams, Peter Webb, M. I. Mahmud, Erlin Novita, Idje Djami, Gludhug A. Purnomo, Cristina Santos, A. Malgosa, Linda R. Manzanilla, Ana Maria Silva, Sofia Tereso, Vítor Matos, Pedro C. Carvalho, Teresa Fernandes, A. Maurer, João C. Teixeira, Ray Tobler, Lars Fehren-Schmitz, B. Llamas
          </td>
          <td>2025-07-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71630599b9ea7d81e787095673033ab508c0165a" target='_blank'>
              The GGH/HuR Complex Binds and Stabilizes mRNAs to Maintain Tumor Cell Cycle and DNA Replication.
              </a>
            </td>
          <td>
            Yu Li, Xinrui Li, Yuhui Du, Sijie Chen, Xiaoniu He, Zhangrong Xie, Zhiqing Zhou, Huijie Zhao, Xiaofei Zeng, Guoan Chen
          </td>
          <td>2025-08-19</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Comprehensive genomic profiling (CGP) aims to assist clinicians with the diagnosis, treatment decisions and early detection of recurrence in patients with cancer. CGP using tumor tissue is widely implemented, whereas circulating tumor DNA (ctDNA) analysis is a noninvasive method that uses peripheral blood. This pilot study included eight patients with locally advanced tumors (two each of breast, lung, pancreatic, and head and neck cancers). The concordance of somatic variants with tumor tissues and paired ctDNA from pre- and post-resection samples was evaluated. This study demonstrated that the overall concordance rate in all genes between tissue and postoperative blood was high (94.2%), but the concordance rate in genes with somatic variants was low (4.76%). In patient 8 with head and neck cancer, the MAP2K1 variant was concordant between the tissue and blood after surgery. The patient was found to have a small lung tumor at 10 months after surgery, indicating recurrence in the lung. In patient 6 with pancreas cancer, the TP53 variant was concordant between the blood before and after surgery, but no recurrence was observed. In patient 5 with pancreas cancer, recurrence was identified; however, the somatic variants were not concordant between the tissue and blood. Furthermore, a case, such as patient 8, of recurrence with somatic variants matching the tissue and postoperative blood was encountered, suggesting that detecting a somatic variant in postoperative ctDNA matching the same variant in the tissue may predict recurrence. However, since the major limitation of this study was the limited sample size, subsequent studies with larger sample sizes and more extensive research designs are warranted. The study was entered in the Japan Registry of Clinical Trials (April 10, 2023; no. 072230003).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2572d19f3aca51d05413f2ac2fd5a7dbca05f04" target='_blank'>
              Comprehensive genetic profiling using tissue and blood in locally advanced tumors
              </a>
            </td>
          <td>
            Kenji Fujiyoshi, Rie Sugihara, Naoki Miyamoto, Yoriko Watanabe, Tomoya Sudo, Sanae Numata, Jun Akiba, H. Abe, Yuka Ichinose, Kenji Inoue, Shuich Ozono, T. Ono, Kentaro Orioka, Masaki Kashihara, Ryousuke Kajiwara, Hiroyuki Kawano, Akihiko Kawahara, Ryuta Takase, Uhi Toh, Kazuaki Hashimoto, Toru Hisaka, Shingo Hirai, M. Mitsuoka, Daiki Miyazaki, Fumi Yoshitomi, Ken Yamamoto, H. Umeno, Masahisa Nomura, Yoshiki Naito
          </td>
          <td>2025-08-05</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Heritable gene activity variations that do not alter the underlying DNA sequence are known as epigenetic modifications. Mutations that disrupt genome structure and function are key drivers of oncogenesis. In addition to genetic mutations that cause direct disruptions in the DNA sequence, epigenetic changes can affect gene expression, which helps the development of cancerous traits. Changes in DNA methylation and histone mark patterns are the main drivers of these epigenetic modifications, as they encourage the growth and spread of tumors. In this review, we explore the substantial implications of epigenetic control on tumor genesis, metastatic behaviour, metabolic pathways, and the tumor microenvironment, delving into the intricacies of this intricate regulation. We pay special attention to the dysregulation at every stage of epigenetic modulation, which includes, but is not limited to, abnormalities in the enzymes that modify histones and methylate DNA, subunit loss or fusions in chromatin remodeling complexes, and disruptions in higher-order chromatin structure. We also highlight the development of inhibitors of medications targeted at epigenetic enzymes and summarize the abnormal roles of enzymes in DNA methylation, histone acetylation, and histone methylation during tumour progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d006bb544ee4859c5808a78f45579e1cfd4a19" target='_blank'>
              Epigenetic regulators in cancer therapy and progression
              </a>
            </td>
          <td>
            Hany E. Marei
          </td>
          <td>2025-06-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction V(D)J recombination, initiated by recombination-activating gene (RAG) endonucleases, is a crucial process for the generation of diversified antigen receptors of T and B lymphocytes but regarded dispensable for innate natural killer (NK) lymphocytes lacking clonotypic receptors. Methods To explore the impact of potential rearrangements on NK cell maturation, RAG-fate mapping reporter human induced pluripotent stem cell (iPSC) lines were generated by introduction of RSS-invEGFP constructs into the AAVS1 locus using CRISPR/Cas9 and differentiated into NK cells in vitro. Results GFP expression was observed in up to 14% of mature NK cells characterized by a CD45dim CD56dimCD57+NKG2C+/−KIR+/− phenotype and unproductive genetic rearrangements in the IGH locus. Advanced maturation was further revealed by transcriptomic studies using RNA sequencing. Despite their strong effector function, DNA damage response and survival to ionizing radiation were compromised. Discussion These findings suggest a role of RAG expression in NK cell ontogeny supporting the development of a terminally differentiated effector population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6697344fa4f70950fad5843a2ad7ef88eb6f129" target='_blank'>
              RAG recombinase expression discriminates the development of natural killer cells
              </a>
            </td>
          <td>
            Jasmin Sprissler, U. Pannicke, E. Rump, H. Schrezenmeier, N. Casadei, M. Pogoda, Laurence Kuhlburger, M. B. Oquendo, Stefan Czemmel, K. Debatin, Miriam Erlacher, Klaus Schwarz, Kerstin Felgentreff
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Replication timing (RT), the temporal order of DNA replication during S phase, influences regional mutation rates, yet the mechanistic basis for RT-associated mutagenesis remains incompletely defined. To identify drivers of RT-dependent mutation biases, we analyzed whole-genome sequencing data from cells with disruptions in DNA replication/repair genes or exposed to mutagenic compounds. Mutation distributions between early- and late-replicating regions were compared using bootstrapping and statistical modeling. We identified 14 genes that exhibit differential effects in early- or late-replicating regions, encompassing multiple DNA repair pathways, including mismatch repair (MLH1, MSH2, MSH6, PMS1, and PMS2), trans-lesion DNA synthesis (REV1) and double-strand break repair (DCLRE1A and PRKDC), DNA polymerases (POLB, POLE3, and POLE4), and other genes central to genomic instability (PARP1 and TP53). Similar analyses of mutagenic compounds revealed 19 compounds with differential effects on replication timing. These results establish replication timing as a critical modulator of mutagenesis, with distinct DNA repair pathways and exogenous agents exhibiting replication timing-specific effects on genomic instability. Our systematic bioinformatics approach identifies new DNA repair genes and mutagens that exhibit differential activity during the S phase. These findings pave the way for further investigation of factors that contribute to genome instability during cancer transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13a232208698bfbf694bd7e822cd1f4798018e9a" target='_blank'>
              Unveiling Replication Timing-Dependent Mutational Biases: Mechanistic Insights from Gene Knockouts and Genotoxins Exposures
              </a>
            </td>
          <td>
            Hadas Gross-Samuels, A. Koren, Itamar Simon
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="ABSTRACT Maintaining genome integrity is essential for the proper functioning and development of organisms. An intriguing aspect is that neocentromeres can form at non-centromeric sites. CENP-A, a key epigenetic marker of centromeres, is often mislocalized to ectopic sites in cancers when overexpressed. Its deposition on centromeres relies on transcription of centromeric non-coding RNAs. Subsequently, ectopic CENP-A is frequently found at transcriptionally active and chromosome breakpoint regions. We previously engineered a stable ectopic CENP-A site on a naïve chromosome by overexpressing PCAT2, a non-centromeric oncogenic lncRNA that recruits CENP-A to its transcribing locus. We tracked cells with this transgene to analyze the longevity of ectopic CENP-A. We discovered that this induced epigenetic memory was lost due to suppression by epigenetic silencing mechanisms, restoring CENP-A to previous levels. These findings suggest that cells have mechanisms to prevent neocentromere formation at ectopic sites by suppressing transcription unless selective pressure favors it">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aec7b7b4ef76af3d3bde1c36b2a65ab677cc6c45" target='_blank'>
              Oncogenic lncRNA transgene transcription modulates epigenetic memory at a naïve chromosomal locus
              </a>
            </td>
          <td>
            Sweta Sikder, Songjoon Baek, Yamini Dalal, Ganesan Arunkumar
          </td>
          <td>2025-07-31</td>
          <td>Nucleus</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6ffe6d676c4d407af20326fa835958244e0ac3e" target='_blank'>
              PCBP2 Mediates Olaparib Resistance in Breast Cancer by Inhibiting m6A Methylation to Stabilize PARP1 mRNA.
              </a>
            </td>
          <td>
            Zhaochang Qi, Lifang He, Zemei Xu, Xi Luo, Likeng Ji, Chenting Lin, Armando E Giuliano, Xiaojiang Cui, Zihao Deng, Jundong Wu, S. L. Lin, Yukun Cui
          </td>
          <td>2025-08-07</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Amplified centrosome number causes genomic instability, most severely through division into more than two aneuploid daughter cells (multipolar mitosis). Several mechanisms that suppress multipolar division have been uncovered, yet mechanisms that favor viable multipolar division are poorly understood. To uncover factors that promote viability in cells with frequent centrosome amplification and multipolar division, we conducted an unbiased Drosophila genetic screen. In 642 mutagenized lines, we exploited the ability of intestinal papillar cells to form and function despite multipolar divisions. Our top hit is an unnamed gene, CG3168. We name this gene synaptic vesicle glycoprotein 2, reflecting homology to human Synaptic Vesicle Glycoprotein 2 (SV2) proteins. GFP-tagged SV2 localizes to the plasma membrane. In cells with amplified centrosomes, SV2 positions membrane-adjacent centrosomes, which prevents severe errors in chromosome alignment and segregation. Our results uncover membrane-based multipolar division regulation and reveal a novel vulnerability in cells with common cancer properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/604d54b8cce75407680ad35bfbf26bb3a7a65d49" target='_blank'>
              Synaptic vesicle glycoprotein 2 enables viable aneuploidy following centrosome amplification.
              </a>
            </td>
          <td>
            Jane E Blackmer, Erin A. Jezuit, Archan Chakraborty, Satya N Yalamanchi, Ruth A. Montague, Erin E. Dickert, Nora G Peterson, William Outlaw, Donald T. Fox
          </td>
          <td>2025-07-31</td>
          <td>Genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="DNA transactions introduce torsional constraints that pose an inherent risk to genome integrity. While topoisomerase 1 (TOP1) activity is essential for DNA supercoil removal, the aberrant stabilization of TOP1:DNA cleavage complexes (TOP1ccs) can result in cytotoxic DNA lesions. What protects genomic hot spots of topological stress from excessive TOP1cc accumulation remains unknown. Here, we identify chromatin context as an essential means to coordinate TOP1cc resolution. Through its ability to bind poly(ADP-ribose) (PAR), the histone variant macroH2A1.1 facilitates TOP1cc repair factor recruitment and lesion turnover, thereby preventing DNA damage in response to transcription-associated topological stress. The alternatively spliced macroH2A1.2 isoform is unable to bind PAR or protect from TOP1ccs. Impaired macroH2A1.1 splicing, a frequent cancer feature, was predictive of increased sensitivity to TOP1 poisons in a pharmaco-genomic screen in breast cancer cells, and macroH2A1.1 inactivation mirrored this effect. We propose macroH2A1 alternative splicing as an epigenetic modulator of TOP1-associated genome maintenance and a potential cancer vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfdec4fbd7a5287d669fa0b66871ebd9fb8a9c98" target='_blank'>
              Epigenetic control of topoisomerase 1 activity presents a cancer vulnerability
              </a>
            </td>
          <td>
            Tae-Hee Lee, Colina X Qiao, Vladislav Kuzin, Yuepeng Shi, Marina Farkas, Zhiyan Zhao, Vijayalalitha Ramanarayanan, Tongyu Wu, Tianyi Guan, Xianzhen Zhou, David Corujo, M. Buschbeck, Laura Baranello, Philipp Oberdoerffer
          </td>
          <td>2025-08-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Amplification of human chromosome 20q has been reported as the most frequently recurring genetic abnormality associated with large scale changes in mRNA and protein levels in sporadic colorectal carcinomas. While some studies have found 20q amplification to be consistent between primary and metastatic samples from the same patient with a role in the development of metastasis and worse patient prognosis, others have reported association with improved overall survival for a subset of these patients with colorectal cancer (CRC). To fine map the Minimal Common Regions (MCRs) of amplification on chromosome 20q and identify the candidate genes playing roles in progression of the disease, microarray comparative genomic hybridization analyses of two in vitro cultured CRC liver metastasis cell line model systems was utilized. Microarray expression analysis led to the identification of a candidate gene signature comprising of four genes, BMP7, DNMT3B, UBE2C and YWHAB, residing in the MCRs that were over expressed in CRC cells. By validating our results in a training set of 23 adenocarcinomas (tumors) and five adenomas (polyps) using reverse transcription-quantitative PCR, as well as analyses of two larger colorectal cancer test data sets derived from 195 The Cancer Genome Atlas and 182 MD Anderson Cancer Center patients with colorectal adenocarcinoma patients, this gene signature was ascertained to be associated with lymph node spread and/or distant metastasis (P<0.05). Previously reported functional studies of the gene signature indicated their involvement in inflammatory and immune response pathways driving CRC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19ea2107086249d394733ab4fbf6c82be7552a16" target='_blank'>
              Chromosome 20q gene signature associated with colorectal cancer progression
              </a>
            </td>
          <td>
            Jennifer Carter Jones, A. Hegde, Yu-Jing Huang, G. Manyam, Vibhuti Srivastava, J. Song, Yulan Cheng, Ralf Krahe, Warapen Treekitkarnmongkol, Stephen J. Meltzer, S. Kopetz, Stanley R Hamilton, Hiroshi Katayama, Subrata Sen
          </td>
          <td>2025-07-18</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Post-transcriptional modifications on mRNA are crucial for mRNA fate and function. The current lack of a comprehensive method for high-coverage and sensitive quantitative analysis of mRNA modifications significantly limits the discovery of new mRNA modifications and understanding mRNA modifications’ occurrence, dynamics and function. Here, a highly sensitive, high-throughput and robust LC-MS/MS-based technique, mRQuant, was developed to simultaneously detect and quantify 84 modified ribonucleosides in cellular mRNA. Using mRQuant, we quantified 32–34 modified ribonucleosides across several human cancer and non-cancer cell lines and uncovered cancer- and cancer type-specific signatures. Analyses of cisplatin- and paclitaxel-treated HeLa cells and drug-resistant variants revealed several drug resistance-associated modifications. Among them, m1A exhibited significant differences across multiple cell types and between cancerous and non-cancerous cells. Knocking down mRNA m1A writer or eraser protein resulted in altered cell viability, cell cycle and apoptosis in HeLa cells, suggesting a role of mRNA m1A in cancer. Transcriptomic and proteomic analyses further revealed the molecular mechanisms underlying the phenotypic variation. Supplementary information The online version contains supplementary material available at 10.1007/s00018-025-05805-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b614524ca44941ff0254c44b4d860cba08475a2" target='_blank'>
              Quantifying the mRNA epitranscriptome reveals epitranscriptome signatures and roles in cancer
              </a>
            </td>
          <td>
            Ying Feng, Xiaoli He, Mingxin Guo, Ying Tang, Guantong Qi, Qian Huang, Wenran Ma, Hong Chen, Yifan Qin, Ruiqi Li, Jin Wang, Yu Liu
          </td>
          <td>2025-07-28</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor evolution is shaped by selective pressures imposed by physiological factors as the tumor naturally progresses to colonize local and distant tissues, as well as by therapy. However, the distinction between these two types of pressures and their impact on tumor evolution remain elusive, mainly, due to extensive intra-tumor heterogeneity. To disentangle the effects of these selective pressures, we analyze data from diverse cohorts of patients, of both treated and untreated cancers. We find that, despite the wide variation across patients, the selection strength on tumor genomes in individual patients is stable and largely unaffected by tumor progression in the primary settings, with some cancer-specific signatures detectable in the progression to metastases. However, we identify a nearly universal shift toward neutral evolution in tumors that resist treatment and demonstrate that this regime is associated with worse prognosis. We validate these findings on both published and original datasets. We suggest that monitoring the selection regime during cancer treatment can assist clinical decision-making in many cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95ee20056cad563b0d086b8e70065c4daacf4f" target='_blank'>
              Genome-level selection in tumors as a universal marker of resistance to therapy
              </a>
            </td>
          <td>
            Erez Persi, P. Sudalagunta, Yuri I. Wolf, Rafael R. Canevarolo, Mehdi Damaghi, K. Shain, A. Silva, Eugene V. Koonin
          </td>
          <td>2025-07-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Mitochondria are critical for cellular energy, and while large deletions in their genome (mtDNA) are linked to primary mitochondrial diseases, their significance in cancer is less understood. Given cancer’s metabolic nature, investigating mtDNA deletions in tumors at various stages could provide insights into disease origins and treatment responses. In this study, we analyzed 148 bone marrow samples from 129 pediatric patients with B-cell (B-ALL) and T-cell (T-ALL) acute lymphoblastic leukemia at diagnosis, remission, and relapse using long-range PCR, next-generation sequencing, and the Splice-Break2 pipeline. Both T-ALL and B-ALL exhibited significantly more mtDNA deletions than did the controls, with T-ALL showing a ~100-fold increase and B-ALL a ~15-fold increase. The T-ALL samples also exhibited larger deletions (median size > 2000 bp) and greater heterogeneity, suggesting increased mitochondrial instability. Clustering analysis revealed distinct deletion profiles between ALL subtypes and across disease stages. Notably, large clonal deletions were detected in some B-ALL remission samples, including one affecting up to 88% of mtDNA molecules, which points toward treatment-driven selection or toxicity. A multivariate analysis confirmed that disease type, timepoint, and WHO subtype significantly influenced mtDNA deletion metrics, while age and gender did not. These findings suggest that mtDNA deletion profiling could serve as a biomarker for pediatric ALL and may indicate mitochondrial toxicity contributing to late effects in survivors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c4c62c1b9c4085a3dab116b06e29653419eb877" target='_blank'>
              Distinct Mitochondrial DNA Deletion Profiles in Pediatric B- and T-ALL During Diagnosis, Remission, and Relapse
              </a>
            </td>
          <td>
            Hesamedin Hakimjavadi, Elizabeth Eom, Eirini Christodoulou, Brooke E. Hjelm, A. A. Omidsalar, D. Ostrow, Jaclyn A. Biegel, Xiaowu Gai
          </td>
          <td>2025-07-23</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="DNA sequencing technologies are widely used to study tumor evolution within a cancer patient. However, analyses require various computational methods, including those to infer clone sequences (genotypes of cancer cell populations), clone frequencies within each tumor sample, clone phylogeny, mutational tree, dynamics of mutational signatures, and metastatic cell migration events. Therefore, we developed GenoPath, a streamlined pipeline of existing tools to perform tumor evolution analysis. We also developed and added tools to visualize results to assist interpretation and derive biological insights. We have illustrated GenoPath’s utility through a case study of tumor evolution using metastatic prostate cancer data. By reducing computational barriers, GenoPath broadens access to tumor evolution analysis. The software is available at https://github.com/SayakaMiura/GP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e039aed15ffae5155d36f38b8ec85975e0960dc5" target='_blank'>
              GenoPath: a pipeline to infer tumor clone composition, mutational history, and metastatic cell migration events from tumor DNA sequencing data
              </a>
            </td>
          <td>
            Ryan M. Tobin, Shikha Singh, Sudhir Kumar, Sayaka Miura
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Simple Summary Cancer cells manage to live longer and grow uncontrollably by protecting the ends of their chromosomes, known as telomeres. In healthy cells, telomeres naturally shorten over time, leading to aging and death of the cell. But in cancer, special mechanisms keep these telomeres from shrinking, allowing the cells to avoid death and continue dividing. This review explains how cancer cells achieve this and the relationship between telomere protection and the systems that repair damaged DNA. The study also explores new treatment approaches that aim to block these processes in cancer cells. These include drugs, immune therapies, and even natural substances that target the proteins or structures involved in telomere maintenance. While some treatments are still being tested in early trials, others have already shown promise in specific cancers. The authors also highlight the challenges of targeting telomeres without harming normal cells, and how telomere length and behavior can help doctors predict treatment success or detect cancer earlier. This work is important because it may lead to better, more personalized treatments and help fight cancer more effectively in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d33d40917a29f13f31b9de10ea636994f17fba8" target='_blank'>
              Telomere Maintenance and DNA Repair: A Bidirectional Relationship in Cancer Biology and Therapy
              </a>
            </td>
          <td>
            Nina Rembiałkowska, Mikołaj Sędzik, Monika Kisielewska, Wiktoria Łuniewska, Kamil Sebastianka, Klaudia Molik, Katarzyna Skinderowicz, Jacek Kuźnicki, Joanna Tunikowska, J. Kulbacka
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The mitotic inhibitor docetaxel (DTX) is often used to treat endocrine-refractory metastatic breast cancer, but initial responses are mitigated as patients eventually have disease progression. Using a cohort of ex vivo cultures of circulating tumor cells (CTCs) from patients with heavily pretreated breast cancer (n=18), we find two distinct patterns of DTX susceptibility, independent of clinical treatment history. In CTCs cultured from some patients, treatment with a single dose of DTX results in complete cell killing, associated with accumulation of non-viable polyploid (≥8N) cells arising from endomitosis. In others, a transient viable drug-tolerant persister (DTP) population emerges, ultimately enabling renewed proliferation of CTCs with preserved parental cell ploidy and DTX sensitivity. In these CTC cultures, efficient cell cycle exit generates a ≤4N drug-tolerant state dependent on CDKN1B (p27Kip1). Exposure to DTX triggers stabilization of CDKN1B through AKT-mediated phosphorylation at serine 10. Suppression of CDKN1B reduces the number of persister CTCs, increases ≥8N mitotic cells and abrogates regrowth after DTX exposure. Thus, CDKN1B-mediated suppression of endomitosis contributes to a reversible persister state following mitotic inhibitors in patient-derived treatment refractory breast cancer cells. Summary in bullets Transient DTX tolerant persister cells emerge in some patient-derived cultured CTCs. DTX-tolerant persisters restrict endoreduplication and polyploidy through CDKN1 (p27kip1). DTX exposure induces CDKN1B stabilization through AKT mediated phosphorylation at serine 10. Suppression of polyploidy underlies a drug tolerant persister state specific to mitotic inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c80ac0aa94dfd6f02a8336f8c8d7fde16ba668bb" target='_blank'>
              CDKN1B (p27/kip1) enhances drug-tolerant persister CTCs by restricting polyploidy following mitotic inhibitors
              </a>
            </td>
          <td>
            Elad Horwitz, Taronish D. Dubash, Annamaria Szabolcs, Ben S. Wittner, J. Kreuzer, Robert Morris, Aditya Bardia, Brian Chirn, Devon F. Wiley, Dante Che, Hunter C. Russel, Xcanda Ixchel Herrera Lopez, Douglas B. Fox, Ezgi Antmen, David T. Ting, Wilhelm Haas, Moshe Sade-Feldman, S. Maheswaran, Daniel A Haber
          </td>
          <td>2024-02-22</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="INTRODUCTION
Long noncoding RNAs are essential regulators in numerous biological processes and have been linked to various diseases, including cancer. Despite their initial classification as transcriptional byproducts, lncRNAs have been shown to modulate chromatin structure, transcription, RNA processing, protein translation, and intranuclear transport. LINC-PINT, a lncRNA induced by P53, is particularly noteworthy for its role in tumor suppression across multiple cancers.


METHODS
By utilizing the PubMed database and applying inclusion criteria based on relevance, literature quality, and data availability, we conducted a comprehensive analysis of 128 studies to provide an overview of the functions of LINC-PINT and its mechanisms of action in cancers.


RESULTS
LINC-PINT was confirmed to function as a tumor suppressor factor in many cancers, such as triple-negative breast cancer, non-small cell lung cancer, gastric cancer, glioma, melanoma, osteosarcoma, laryngeal squamous cell carcinoma, esophageal cancer, colorectal cancer, nasopharyngeal carcinoma, retinoblastoma, ovarian cancer, thyroid cancer, hepatocellular carcinoma, and pancreatic cancer by promoting apoptosis and senescence, inhibiting proliferation, migration, invasion, drug resistance, cell stemness, EMT, radioresistance, and DNA damage repair.


DISCUSSION
LINC-PINT serves as a tumor suppressor with its ability to sponge miRNAs, regulate epigenetic modulation, DNA damage repair, etc. Despite the promising findings, the complex and tissue-specific functions of LINC-PINT, along with the need for further clinical validation, underscore the importance of continued research to fully understand its mechanisms and potential as a therapeutic target.


CONCLUSION
LINC-PINT is a potential target in cancer progression and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c614cdfeae321548caa4f0f7d2f981f35ae5a41" target='_blank'>
              LINC-PINT: A Distinctive Long Non-Coding RNA Functioning as a Potential Suppressor in Tumorigenesis.
              </a>
            </td>
          <td>
            Jiayi Li, Yining Pan, Songqiang Li, Cheng Chen, Chengfu Yuan
          </td>
          <td>2025-07-22</td>
          <td>Current topics in medicinal chemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) represent one of the most severe forms of genomic damage. Although substantial progress has been made in elucidating general patterns associated with DSBs, the influence of 3D chromatin structure on DSB formation remains underexplored, particularly concerning its spatial configuration. Here, the ChromInSight framework is introduced. Using standardized datasets,Hi-DSB is developed and deployed in ChromInSight, a genome-wide DSB prediction model based on graph contrastive learning (GCL), and applied advanced interpretability techniques to identify DSB-associated genomic patterns. The findings reveal that the spatial cluster-scene between hub nodes and DSB sites is predominantly shaped by the 3D conformation of chromatin, rather than by linear genomic distance. This phenomenon is validated at both the Loop and topologically associating domain (TAD) levels and proposed a "spatial isolation - damage containment" hypothesis, which illustrates the genome strategy for managing damage. These findings support the role of 3D genome architecture in genomic instability. Consequently, the framework provides a powerful tool for investigating the intricate relationship between chromatin structure and genomic stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31e5b0b135f6e23dfe9751ad20057b548c477ad6" target='_blank'>
              ChromInSight: Revealing DNA Double-Strand Breaks Through Chromatin Structural Insights With an Interpretable Graph Neural Network Framework.
              </a>
            </td>
          <td>
            Kang Xu, Zongyuan Yu, Canzhuang Sun, Conglin Gou, Jiangyue Zhu, Jun Wang, Xiaochen Bo, Guoxian Yu, Hao Li, Hebing Chen
          </td>
          <td>2025-06-30</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Gastric cancer (GC), as one of the most prevalent malignant tumors, significantly impacts individuals’ health. Many studies have examined the relationship between mitochondrial DNA (mtDNA) copy number and GC. However, conclusions remain inconclusive, with conflicting findings. The genome-wide association study summary statistics for mtDNA copy number were obtained from 2 sources: one from a robust cohort of 465,809 White individuals provided by the Cohorts for Heart and Aging Research in Genomic Epidemiology consortium and the UK Biobank; and the other from a dataset comprising 395,718 UK Biobank participants. In addition, a total of 5 sets of genome-wide association study summary statistics for GC were obtained through datasets from Finland, the European Bioinformatics Institute, and the Integrative Epidemiology Unit at the University of Bristol, encompassing a total of 937,663 participants. Furthermore, we undertook a 2-sample bidirectional Mendelian randomization analysis to explore the association between mtDNA copy number and GC. The Inverse variance weighted (IVW) method was primarily utilized, complemented by 4 other validation methods. Based on our comprehensive investigation, no discernible causal relationship was found between mtDNA copy number and GC in both the training and validation cohorts (IVW, P >.05). Furthermore, in the reverse Mendelian randomization analysis, no association was observed between GC and mtDNA copy number either (IVW, P >.05). Additionally, all analyses showed no evidence of horizontal pleiotropy or heterogeneity. The findings of this study provide evidence that there is no causal relationship between mtDNA copy number and GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4105ef9b843658ffce3da6be2a184656f4791db1" target='_blank'>
              Unveiling the enigma: Investigating the controversy surrounding mitochondrial DNA copy number and gastric cancer using Mendelian randomization analysis
              </a>
            </td>
          <td>
            Jie Zhou, Taohua Shangguan, Yixin Xu, Chao Chen, Kun Wang
          </td>
          <td>2025-08-15</td>
          <td>Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Erythropoiesis, the process driving the differentiation of hematopoietic stem and progenitor cells to mature erythrocytes, unfolds through tightly orchestrated developmental stages, each defined by profound epigenetic remodeling. From the initial commitment of hematopoietic progenitors to the terminal enucleation of erythrocytes, dynamic changes in chromatin accessibility, transcription factor occupancy, and three-dimensional genome architecture govern lineage specification and stage-specific gene expression. Advances in our understanding of the regulatory genome have uncovered how non-coding elements, including enhancers, silencers, and insulators, shape the transcriptional landscape of erythroid cells. These elements work in concert with lineage-determining transcription factors to establish and maintain erythroid identity. Disruption of these epigenetic programs—whether by inherited mutations, somatic alterations, or environmental stress—can lead to a wide range of hematologic disorders. Importantly, this growing knowledge base has opened new therapeutic avenues, enabling the development of precision tools that target regulatory circuits to correct gene expression. These include epigenetic drugs, enhancer-targeted genome editing, and lineage-restricted gene therapies that leverage endogenous regulatory logic. As our understanding of erythroid epigenomics deepens, so too does our ability to design rational, cell-type-specific interventions for red blood cell disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d4488b6509a29acbbc7dd6eded382cd65c2fb32" target='_blank'>
              Epigenetic Regulation of Erythropoiesis: From Developmental Programs to Therapeutic Targets
              </a>
            </td>
          <td>
            Ninos-Ioannis Vasiloudis, K. Paschoudi, Christina Beta, G. Georgolopoulos, N. Psatha
          </td>
          <td>2025-06-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="This review examines the increasingly prominent role of mechanics within cancer formation and progression. The extremely varied and contradictory genetic landscape of cancer is in stark contrast to the seemingly universal mechanical characteristics of cancer cells and their tumour microenvironment, and mechanics may be a principal unifying trait of this disease. The tight regulation of innate cell mechanical properties raises the possibility that destabilisation of the cell drives tumour formation in an attempt to restore cell mechanical homeostasis. With losses in cell stiffness more pronounced at the cell nucleus, we hypothesise that destabilisation occurs within the nucleus, likely within the nucleosome. Beyond the mechanical properties of the cell, this compromise to the chromatin structure holds significant repercussions for both genetic and epigenetic regulation, providing scope for significant genetic dysregulation and mutation. However, the nature of such genetic events will be dependent upon the region of mechanical destabilisation; thus, introducing greater variability and heterogeneity to genetic changes. We conclude with the hypothesis that cancer has a mechanical genesis, in which cell nuclear destabilisation functions as the enabling hallmark of cancer. It is theorised that both genetic and structural dysfunction stem from this nuclear destabilisation, driving disease pathology and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/440a5c35e068d9fe1560a5f26a4055f7a6f2a62d" target='_blank'>
              Nuclear destabilisation - a possible genesis of cancer?
              </a>
            </td>
          <td>
            Daniel D Scott, Francesco Bettariga, Marco Ventin, Chris Bishop, Britta Stordal
          </td>
          <td>2025-07-14</td>
          <td>Biological reviews of the Cambridge Philosophical Society</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="PLXNA3 is a transmembrane protein essential for axon guidance and synapse formation, gaining attention in cancer research for its roles in cell migration, invasion, and signal transduction. However, its biological significance in breast cancer remains underexplored. Here, bulk RNA sequencing and survival analysis revealed that elevated PLXNA3 levels were correlated with increased malignancy and poorer survival in breast cancer. Single-cell RNA sequencing demonstrated that PLXNA3 was predominantly enriched in tumor cell clusters within the tumor immune microenvironment, rather than in immune cell clusters. In vitro experiments further demonstrated that knockdown and overexpression of PLXNA3 significantly altered the proliferative, invasive, and migratory capacities of breast cancer cells. Additionally, the overexpression of PLXNA3 was negatively correlated with immune activation status and served as a predictor of poor response to anti-PD1 immunotherapy. Our findings suggested that high expression of PLXNA3 is associated with poor prognosis in breast cancer and plays a crucial role in cancer immunity, making it a promising novel target for intervention in breast cancer. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-13843-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f126a6deb0547875700a586994249ce5224fe17" target='_blank'>
              Comprehensive analysis of the PLXNA3 gene on prognosis and immune characteristics in breast cancer
              </a>
            </td>
          <td>
            Wen Sun, Yifan Xu, Qi Qi, Weiya Zhang, H. Pan, Guangshun Sun, Shui Wang, Wenbin Zhou
          </td>
          <td>2025-08-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Centromeres are epigenetically specified by distinct chromatin, whereas their DNA varies between species and individuals. This extensive sequence divergence makes comparative analyses between centromeres challenging. In this study, we identified a chromosome-specific architectural pattern across the human genome, defined by the conserved spacing of a functionally relevant centromeric DNA motif. The distribution of these sites along chromosome arms constitutes the human "centeny map." By using a custom Genomic Centromere Profiling (GCP) pipeline, we leveraged the motif's position, orientation, and organization to construct structural models that enable reclassification of human chromosomal clusters, detection of centromere expansion, and identification of structural variants and misassembled regions. The high-resolution maps derived from this pattern not only provide a framework for comparative analysis of centromeres across evolution and disease but also offer a new dimension for chromosome annotation, assembly, and characterization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cea4b7a430cb7cedf73a935a8f04bee13cba7321" target='_blank'>
              Chromosome-specific centromeric patterns define the centeny map of the human genome.
              </a>
            </td>
          <td>
            Luca Corda, Simona Giunta
          </td>
          <td>2025-07-03</td>
          <td>Science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Two epigenetically labile subsets of genes that link embryonic environmental exposures with adult disease susceptibility are those that are imprinted and those with metastable epialleles. The expression of genes with metastable epialleles, like the agouti gene in Agouti viable yellow (Avy) mice, is highly variable between individuals but uniform in tissues within an individual. We used the Avy mouse to demonstrate that exposure to nutritional supplements, chemical toxicants, and low-dose ionizing radiation during embryogenesis alters adult disease susceptibility by modifying the epigenome. Genomic imprinting is a unique species-dependent epigenetic form of gene regulation that evolved approximately 150 million years ago in a common ancestor to Therian mammals. It resulted in monoallelic parent-of-origin-dependent gene silencing. Thus, imprinted genes are functionally haploid disease susceptibility loci, since only a single genetic or epigenetic event is required to alter their function. Expression of imprinted genes in the human genome is regulated by hemi-methylated imprint control regions (ICRs) in the human imprintome. Furthermore, human imprintome ICRs associated with chronic diseases (e.g., cancer, diabetes, and obesity) and behavioral disorders (e.g., autism, bipolar disorder, psychopathy, and schizophrenia) can now be identified with the use of cells from peripheral samples and the human imprintome array. The importance of metastable epialleles and imprinted genes in the etiology of environmentally induced human chronic diseases is discussed in this review.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/960e9fe6811efda91a98ebc102150c171e03b920" target='_blank'>
              Epigenetics, human imprintome, and chronic diseases
              </a>
            </td>
          <td>
            Randy L Jirtle
          </td>
          <td>2025-07-04</td>
          <td>Essays in Biochemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/813fcceb7d9e7dd4a1d1ee8db0535b3e857d7b20" target='_blank'>
              Unraveling the Potential of ctDNA in Precision Medicine for Breast Cancer.
              </a>
            </td>
          <td>
            Juscelino Carvalho de Azevedo Junior, Fernanda Jardim da Silva, Anna Carolina Lima Rodrigues, Stefanie Braga Maia de Sousa, Jéssica Manoelli Costa da Silva, Iago Barroso Ramos, Bárbara do Nascimento Borges, Vanessa Morais Freitas, D. Calcagno
          </td>
          <td>2025-08-07</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 The androgen receptor (AR) plays a critical role in the development and progression of prostate cancer by regulating key cellular processes such as cell proliferation and apoptosis. Although traditional AR-targeted therapies have shown initial success, acquired resistance remains a significant clinical challenge, often driven by AR alterations and somatic gene mutations associated with homologous recombination deficiency (HRD). Approximately 20% of advanced prostate cancer cases exhibit HRD, resulting in substantial genomic instability and complicating treatment. Fortunately, Food and Drug Administration–approved poly(ADP-ribose) polymerase inhibitors, including olaparib and rucaparib, exploit synthetic lethality to target prostate cancer with HRD, and additional drugs targeting DNA damage response (DDR) proteins are under development. Emerging evidence suggests that AR activity enhances DDR gene expression, with multiple DDR proteins localized near androgen-regulated regions, highlighting a close interaction between AR and DDR pathways. Consequently, recent preclinical and clinical studies have investigated combining AR-targeted therapies with treatments that induce DNA damage, such as radiation therapy, or inhibit DNA repair mechanisms. This review discusses AR's role in cellular processes, the interplay between AR and DDR, and recent advances in prostate cancer treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4bcf547c4cdb4775dcc9944086243c67a7115b7" target='_blank'>
              The interconnection between AR and DNA damage response pathways in prostate cancer
              </a>
            </td>
          <td>
            Mallory Sands, Sam Adams, Jihaeng Lee, Michael Li, Mae Wang, Thomas Walsh, Leo Leon, Adriana Zablah, Matthew Haerens, Zhichao Liu, Jianneng Li
          </td>
          <td>2025-08-19</td>
          <td>Current Urology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The XPO1 (exportin 1) gene encodes exportin 1 protein responsible for transporting proteins and RNA from the nucleus to the cytoplasm. It has been used as a biomarker for lymphoma detection. XPO1E571K mutation has been frequently observed and identified as a good prognostic indicator for lymphoma patients. The detection of a target molecule released by lymphoma cells into blood circulation (cell-free circulating tumor DNA, cfDNA) is a better method than tissue biopsy. However, cfDNA concentration in blood circulation is very low in cancer patients. Therefore, a precise and sensitive method is needed. In this study, cfDNA was extracted, and then the XPO1 gene was detected and amplified using conventional PCR. Sanger sequencing was employed to verify the DNA sequences. FAST-COLD-PCR was developed to detect XPO1E571K gene mutation using a CFX96 Touch Real-Time PCR System. The optimal critical temperature (Tc) was 73.3 °C, allowing selective amplification of XPO1E571K mutant DNA while wild-type XPO1 could not be amplified. XPO1E571K gene mutation can be detected by this method with high specificity and sensitivity in lymphoma patients. This approach facilitates rapid and straightforward detection in a timely manner after the diagnosis. Accordingly, the optimized FAST-COLD-PCR conditions can be used as a prototype for XPO1E571K mutant detection in lymphoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2337549ca06913503c667aac38d66d4d170ff79" target='_blank'>
              Detection of XPO1E571K Gene Mutation from Cell-Free DNA in Blood Circulation of Lymphoma Patients by FAST-COLD PCR
              </a>
            </td>
          <td>
            S. Duangmano, Natsima Viriyaadhammaa, Pinyaphat Khamphikham, Nutjeera Intasai, A. Tantiworawit, Teerada Daroontum, S. Chiampanichayakul, S. Anuchapreeda
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Targeting the DNA damage response (DDR) exhibits potent efficacy in inducing immune activation and enhancing patient prognosis. However, the benefits of DDR regulation are not universally observed across all patients, owing to the intricate compensatory mechanisms operative in certain cancers. There still exists a gap in the function of activated DDR protein in esophageal squamous cell carcinoma (ESCC). Here, we demonstrate that increased expression of DDR genes contributes to the progression of esophageal squamous cell carcinoma and suppresses the tumor immune microenvironment. Notably, the abundant presence of the DDR protein KIN in ESCC tissues facilitates efficient DNA damage clearance and promotes escape from apoptosis. Depletion of KIN significantly inhibited proliferation and induced DNA damage accumulation in ESCC cells. Mechanistically, KIN functions to support the recruitment of the R-loop regulator DHX9 to R-loop sites, thereby addressing DNA damage associated R-loops. Intriguingly, the depletion of KIN activates the STING pathway via NFκB signaling, which is induced by the accumulation of R-loops, ultimately initiating an innate immune response. Depletion of KIN improved the immune microenvironment and the effect of immune therapy in mouse model. Collectively, our findings identify KIN as a novel R-loop binding protein that facilitates the recruitment of the R-loop resolution complex and suppresses tumor-intrinsic innate immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e3c03a2576fc82b57363fed2fb18edfc9c71a5" target='_blank'>
              DNA/RNA-binding protein KIN17 supports esophageal cancer progression via resolving noncanonical STING activation induced by R-loop
              </a>
            </td>
          <td>
            Zichao Wei, N. Zhao, Lu Kuang, J. Cong, Sujuan Zheng, Yi Li, Zhihua Liu
          </td>
          <td>2025-08-15</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05361b43fcc5f75fe89674ef423d0f325e486737" target='_blank'>
              Characterizing neurofibromin 1-altered breast cancer through genomic, functional, and clinical analyses.
              </a>
            </td>
          <td>
            Xueyan Mao, Kang Ma, Yunjie Wang, Cheng Long, Xuejing Tan, Xi Chen, Bo Chen
          </td>
          <td>2025-08-14</td>
          <td>Virchows Archiv : an international journal of pathology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f99846544207b1ae90590a8e709eb82ec0d28561" target='_blank'>
              RAD18 Methylation by the Methyltransferase SETD6 Attenuates DNA Breaks
              </a>
            </td>
          <td>
            L. Weil, Michal Feldman, J. V. Duine, Ji Qiu, Joshua LaBaer, D. Levy
          </td>
          <td>2025-07-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="A substantial amount of castration-resistant prostate cancer (CRPC) progresses into a neuroendocrine (NE) subtype, known as NEPC, which is associated with poor clinical outcomes. Here we report distinct three-dimensional chromatin architectures between NEPC and CRPC tumors, which were recapitulated by isogenic cell lines undergoing NE transformation (NET). Mechanistically, pioneer factors such as FOXA2 initiate binding at NE enhancers to mediate regional DNA demethylation and induce neural transcription factor (TF) NKX2-1 expression. NKX2-1 preferentially binds gene promoters and interacts with enhancer-bound FOXA2 through chromatin looping. NKX2-1 is highly expressed in NEPC and indispensable for NET of prostate cancer. NKX2-1/FOXA2 further recruits p300/CBP to activate NE enhancers, and pharmacological inhibition of p300/CBP effectively blunts NE gene expression and abolishes NEPC tumor growth. Taken together, our study reports a hierarchical network of TFs governed by NKX2-1 in critically regulating chromatin remodeling and driving luminal-to-NE transformation and suggests promising therapeutic approaches to mitigate NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f79a3ac68a2814bb628761a75464adfb40b777a" target='_blank'>
              NKX2-1 drives neuroendocrine transdifferentiation of prostate cancer via epigenetic and 3D chromatin remodeling
              </a>
            </td>
          <td>
            Xiaodong Lu, Viriya Keo, Irina Cheng, Wanqing Xie, Galina A Gritsina, Juan Wang, Lina Lu, Cheng-Kai Shiau, Yueying He, Qiushi Jin, Peng Jin, Martin G Sanda, Victor Corces, Nicolas Altemose, Ruli Gao, Jonathan C Zhao, Jindan Yu
          </td>
          <td>2025-07-21</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Radiotherapy is a mainstay of treatment for head and neck squamous cell carcinoma (HNSCC), either definitive or adjuvant to surgery. Biological factors known to affect radiation response are hypoxia and DNA repair capacity, but several lines of evidence indicate that other genes and pathways in the tumor cells might be involved that have not been elucidated. Here, we report the results of a genome-wide CRISPR-Cas9 functional genomics screen in HNSCC cells to identify radiosensitizing genes. Remarkably, microtubule organizing genes were identified with CLASP1 as most unexpected radiosensitizing hit. Clonogenic assay confirmed the radiosensitizing effect of CLASP1 knockout. Functional analysis showed that CLASP1 knockout has major impact during S-phase, and resulted in mitotic cells with broken chromosomes and cell death. CLASP1 and possibly the microtubule machinery in broader sense seem involved in protection of HNSCC cells against radiation–induced DNA damage. This newly identified mechanism provides an outlook for novel treatment approaches in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/284fd55d39146101abef95bbef149eda9c32d030" target='_blank'>
              An unexpected role of CLASP1 in radiation response and S-phase regulation of head and neck cancer cells
              </a>
            </td>
          <td>
            R. D. de Roest, Marijke Buijze, Myrthe Veth, Klaas de Lint, G. Pai, M. Rooimans, Rob M. F. Wolthuis, A. Brink, J. Poell, R. Brakenhoff
          </td>
          <td>2025-08-06</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Background NSUN6 is a regulator of tRNA methylation that partially exists in the Golgi and centrosomes. It catalyzes the methylation of tRNACys and tRNAThr at the C72 site, which affects tRNA generation. However, its expression, prognosis, and immune invasion in pan-cancer have not been studied. This extensive research tapped into the TCGA database, gathering 33 carefully paired sets of cancer and nearby normal tissues, covering a broad spectrum of 33 different cancer types. Moreover, this study explored how NSUN6 expression relates to the presence of immune and stromal cells in the tumor’s intricate immune environment. To further understand NSUN6’s role, a detailed Gene Set Enrichment Analysis (GSEA) was conducted, revealing its enrichment patterns in various carcinoma subtypes and hinting at its involvement in tumor growth. Analysis of NSUN6 expression patterns revealed a significant increase in eight cancer types, including lung adenocarcinoma and pancreatic cancer (P < 0.001). Additionally, a reduction was observed in six cancer types, such as cholangiocarcinoma and thyroid cancer (P < 0.001). Survival analysis indicated that individuals with lung adenocarcinoma, pancreatic cancer, and other cancers expressing high levels of NSUN6 had improved prognosis. The association between NSUN6 expression and patient outcomes was evident, including overall survival, disease-specific survival, disease-free interval, and progression-free interval. Furthermore, individuals with high levels of NSUN6 expression showed significantly longer survival times than those with low levels (P < 0.05). The involvement of NSUN6 in immune infiltration was evident, GSEA showed a significant correlation between NSUN6 expression and the five most significant enrichment pathways in different tumors. The NSUN6 gene functions as an initial indicator for diagnosis in renal clear cell carcinoma, pancreatic cancer, lung adenocarcinoma, and low-grade brain glioma. There is an association between its elevated expression and the predictive outcome, as well as the immune system’s infiltration, in 33 varieties of cancer. This study is limited by its reliance on in silico analyses without experimental validation. Additionally, the use of publicly available datasets may introduce variability due to differences in data sources and platforms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba0727a8de06b6d8752b41beb8058ddf29b34aed" target='_blank'>
              NSUN6 expression correlates with prognosis and immune infiltration in human cancers based on pan-cancer analysis
              </a>
            </td>
          <td>
            Jie Gao, Yao Xu, Chan Tian, Qiuhong Zhang, Ming Zhang, H. Shan, Jie Shi, Zongjuan Ming, Shuanying Yang, Xia Yang
          </td>
          <td>2025-07-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma accounts for 90% of primary liver malignancies and ranks as the fourth leading cause of global cancer mortality. DNA methylation drives HCC pathogenesis by orchestrating tumor suppressor silencing and oncogenic pathway activation, fundamentally governing tumor heterogeneity and therapeutic resistance. While alterations in white blood cell counts have emerged as independent prognostic indicators in HCC progression, their causal relationship with tumor epigenetics remains unclear. Given that HCC tumors actively secrete cytokines and remodel systemic immunity through epigenetic mechanisms, peripheral white blood cell dynamics serve as a compelling surrogate for studying tumor-driven immune dysregulation and identifying methylation-based therapeutic targets. Methods We integrated DNA methylation profiles from The Cancer Genome Atlas-Liver Hepatocellular Carcinoma (TCGA-LIHC) with summary statistics from six genome-wide association studies of white blood cell counts. Using these data, we applied a two-sample Mendelian Randomization (MR) framework to systematically investigate the causal effects of HCC-related CpG sites on white blood cell counts. The robustness of our findings was established through stringent criteria, comprehensive sensitivity analyses, and Bayesian colocalization, followed by functional validation with TCGA data and a transcriptome-wide association study (TWAS). Results Our analysis identified 26 HCC-specific CpG sites with causal effects on white blood cell counts. A multi-step validation pipeline integrating Bayesian colocalization and tumor-specific expression analysis robustly confirmed three core mediator genes: BTN3A2, TRIM27, and S100A12. TWAS provided independent, gene-level validation, revealing BTN3A2 as a powerful regulator of lymphocyte and neutrophil counts, TRIM27 as a key immunometabolic checkpoint, and S100A12 as a modulator of innate immunity. Conclusion This study establishes epigenome-immune crosstalk in HCC by identifying BTN3A2, S100A12 and TRIM27 as central methylation-immunoregulatory hubs. The discovered methylation-immune axes provide mechanistic insights for developing combination therapies targeting epigenetic checkpoints to reverse immunosuppressive microenvironments in advanced HCC. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14693-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/282ecf09d34731818d82e7690045faf501ada4b4" target='_blank'>
              Decoding the epigenetic-immune nexus in hepatocellular carcinoma: a Mendelian randomization study reveals BTN3A2, S100A12 and TRIM27 as white blood cell regulators
              </a>
            </td>
          <td>
            Yang Qiu, Haiyan Zhang, Xiao Yu, Jielu Pan, Dinghong Xiao, Hongyu Miao, Ruiqing Wang, Nan Shen, Peimin Pu, Yinling Zhou, Zhidong Liu, Lin He, Lianjun Xing
          </td>
          <td>2025-08-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background SPOCK1, a matricellular glycoprotein, has been implicated in tumor progression, metastasis, and the tumor immune microenvironment, yet its specific roles in breast cancer (BRCA) remain unclear. This study aimed to systematically explore the expression pattern, prognostic significance, mutation landscape, immune association, and spatial localization of SPOCK1 in breast cancer through integrated multi-omics analyses. Methods Transcriptomic, genomic, and clinical data from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) were utilized. Bulk RNA sequencing and single-cell RNA sequencing (scRNA-seq) analyses were conducted, including functional enrichment, immune infiltration assessments, mutation profiling, and transcription factor activity analysis. Multiplex immunohistochemistry (mIHC) was performed to validate the spatial distribution of SPOCK1+ cancer-associated fibroblasts (CAFs) within the tumor microenvironment. Statistical analyses were performed using R and GraphPad Prism. Results SPOCK1 was broadly overexpressed in multiple cancer types and significantly associated with poor prognosis in BRCA. High SPOCK1 expression correlated with immune checkpoint activation, enhanced immune infiltration, and enriched metastasis-related pathways such as epithelial–mesenchymal transition (EMT) and TGF-β signaling. Single-cell analysis identified CAFs as the primary cell population expressing SPOCK1, with spatial mIHC confirming their close proximity to tumor cells. Furthermore, SPOCK1-high CAFs exhibited stronger intercellular communications with malignant cells via collagen, fibronectin, and IGFBP signaling pathways, alongside distinct transcription factor and metabolic profiles. In breast cancer CAF cell lines with knockdown of ANXA2 we found that the expression of both SPOCK1 and IGF1 was reduced. Conclusion SPOCK1 serves as a critical regulator of breast cancer progression, influencing tumor metastasis and reshaping the immune microenvironment via CAF-mediated mechanisms. These findings suggest that targeting SPOCK1+ CAFs could offer new therapeutic opportunities for breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd6ec4d3a2efd20e6cfad0b54c715da9bed1f6fb" target='_blank'>
              SPOCK1 promotes the progression of breast cancer by modulating cancer-associated fibroblasts and exerts a synergistic effect with ANXA2
              </a>
            </td>
          <td>
            Yuan Jie, Xin Fei, Meng Fan
          </td>
          <td>2025-08-05</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b570c21348c29c543cb996a976f642797b0edb0" target='_blank'>
              A SETD2-CDK1-lamin axis maintains nuclear morphology and genome stability.
              </a>
            </td>
          <td>
            Abid Khan, Cheng Zhang, Phu G Nguyen, James M. Metts, Lucas C. Collins, Kanishk Jain, C. A. Mills, Logan Vlach, Kelin Li, Amanda L. Brademeyer, Brittany M. Bowman, M. Major, Jeffrey Aubé, Laura E. Herring, W. Rathmell, Frank M. Mason, Ian J. Davis, Qing Zhang, B. Strahl
          </td>
          <td>2025-08-01</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="The chromosomal passenger complex (CPC), comprising Survivin, Aurora B kinase, INCENP, and Borealin, is classically known for its essential functions during mitosis. However, recent findings expand the CPC's role beyond cell division, uncovering novel functions in replication stress response and genome stability maintenance. In our recent study by Falke et al. [1], we demonstrate that CPC components, particularly Survivin and Aurora B, contribute to the protection of stalled replication forks and facilitate translesion synthesis (TLS) - a specialized DNA damage tolerance mechanism. This regulation involves a functional interaction with proliferating cell nuclear antigen (PCNA), a central coordinator of DNA replication and repair. The work not only reshapes our understanding of CPC biology but also reveals a mechanistic link how tumor cells may exploit CPC activity to maintain replication under genotoxic stress. These findings open new avenues for targeting CPC-PCNA interactions in cancers characterized by elevated replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27440a8fa21c0d8a560b047855971e8e0f1ea441" target='_blank'>
              From mitosis to mutagenesis: Chromosomal passenger proteins at the crossroads of replication stress and cancer resilience
              </a>
            </td>
          <td>
            S. Knauer, Roland H Stauber
          </td>
          <td>2025-07-07</td>
          <td>Cell Signaling</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Telomerase is a ribonucleoprotein enzyme that maintains telomeric repeats on chromosome ends in continuously dividing cells. Telomere maintenance via telomerase is dependent on the correct assembly of the enzyme complex, complex stabilization by associated cofactors, and effective recruitment to the telomere. Here, we show that telomerase is regulated in each of these processes by the Drosophila behaviour/human splicing (DBHS) family of RNA/DNA binding proteins (NONO, SFPQ and PSPC1). The DBHS proteins associate with catalytically active telomerase through the hTR RNA template component. Cells lacking the DBHS proteins display telomerase retention in nuclear Cajal bodies and impaired telomerase recruitment to the telomere, with NONO and PSPC1 depletion culminating in progressive telomere shortening in several cell lines, with the exception of long-term NONO depletion in 293 and 293T. Our results reveal the DBHS protein family as components of the telomerase trafficking machinery integral to telomere maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a7e5284a186f4721a8dfef97b71d4b22f3a92b8" target='_blank'>
              NONO, SFPQ, and PSPC1 promote telomerase recruitment to the telomere
              </a>
            </td>
          <td>
            A. Sobinoff, Jadon K Wells, Maurice Chow, Christopher B. Nelson, Xinyi Wu, Scott B. Cohen, Yu Heng Lau, Tracy M. Bryan, Archa H. Fox, Hilda A. Pickett
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827b6d7c12ecbffd0d9766944bd9dcb122ff76bc" target='_blank'>
              Epithelial state-transitions permit inflammation-induced tumorigenesis
              </a>
            </td>
          <td>
            E. Jarman, Anabel Martinez Lyons, Yuelin Yao, Aleksandra Rozyczko, S. Waddell, Andreea Gradinaru, Paula Olaizola, Kyle Davies, Rachel V. Guest, Stephanie Röessler, Timothy J. Kendall, Owen J. Sansom, A. Khamseh, Luke Boulter
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Specialized DNA polymerases facilitate various cellular processes. Despite extensive research, the mutagenic effects of these error-prone enzymes on genomes are not fully understood. Here we show that Pol IV promotes genomic instability in Pseudomonas aeruginosa by misincorporating oxidized guanine nucleotides. This activity led to a distinctive mutational signature, characterized by A-to-C transversions occurring preferentially at AT sites flanked by a 5’G and/or 3’C. Furthermore, Pol IV preferentially targeted pathogenicity genes located at specific chromosomal locations near the replication termination region and rRNA-encoding operons. Half of the mutation events catalyzed by Pol IV impaired gene function. This can be attributed to the bias of Pol IV for mutating codons with its preferred sequence contexts, leading to substitutions to unreactive alanine and glycine residues. Remarkably, mutation signatures identified for Pol IV were found in clinical isolate genomes of P. aeruginosa, providing compelling evidence for its role in genetic diversification during pathogen adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54f7c02761e12b8b73eebc31b5beda14fbe3ac0c" target='_blank'>
              The low-fidelity DNA Pol IV accelerates evolution of pathogenicity genes in Pseudomonas aeruginosa
              </a>
            </td>
          <td>
            Sofía D. Castell, Consuelo M. Fernandez, Ignacio N. Tumas, L. M. Margara, María C. Miserendino, Danilo Ceschin, R. Pezza, Mariela R. Monti
          </td>
          <td>2025-08-02</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="In mammals, primordial germ cells (PGCs) undergo global erasure of DNA methylation with delayed demethylation of germline genes and selective retention of DNA methylation at evolutionarily young retrotransposons. However, the molecular mechanisms of persistent DNA methylation in PGCs remain unclear. Here we report that resistance to DNA methylation reprogramming in PGCs requires UHRF2, the paralog of the DNMT1 cofactor UHRF1. PGCs from Uhrf2 knock-out mice show loss of retrotransposon DNA methylation, while DNA methylation is unaffected in somatic cells. This is not associated with changes in the expression of retrotransposons in E13.5 PGCs, indicating that other mechanisms compensate for retrotransposon control at this stage. Furthermore, Uhrf2-deficient PGCs show precocious demethylation of germline genes and overexpress meiotic genes in females. Subsequently, Uhrf2-deficient mice show impaired oocyte development and female-specific reduced fertility, as well as incomplete remethylation of retrotransposons during spermatogenesis. These findings reveal a crucial function for the UHRF1 paralog UHRF2 in controlling DNA methylation in the germline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea0f6cead9b7479f9deb5d526e8e54ad404a663" target='_blank'>
              UHRF2 mediates resistance to DNA methylation reprogramming in primordial germ cells
              </a>
            </td>
          <td>
            Ambre Bender, Marion Morel, Michael Dumas, M. Klopfenstein, Naël Osmani, Maxim V. C. Greenberg, Déborah Bourc'his, Norbert B Ghyselinck, Michael Weber
          </td>
          <td>2025-08-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Mitotic bookmarking, the retention of regulatory proteins and lncRNAs on chromatin during mitosis, epigenetically sustains competency for phenotype-specific gene expression in progeny cells. Gene expression is predominantly suppressed during mitosis. Bookmarking provides the guidance for the resumption of gene expression in progeny cells that is obligatory for physiological control of lineage commitment, specialized cell structure and phenotypic function. While regulatory continuity is supported by the persistence of genome-associated regulatory complexes, altered bookmarking mediates plasticity for responsiveness to physiological cues. Bookmarking fidelity ensures genome integrity and controls expression of tumor suppressors and proto-oncogenes. Cancer-compromised aberrations in bookmarking results in transcriptional dysregulation and the initiation of tumor-associated processes. Keywords: Mitotic Bookmarking, Mitosis, Epigenetic Control, Gene Expression, Cell Division.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1d78052a96f4610f52d1801e1448869b1e34100" target='_blank'>
              Mitotic Bookmarking Provides Epigenetic Persistence or Plasticity for Biological Control and Cancer.
              </a>
            </td>
          <td>
            Andrew J. Fritz, Emory Pacht, R. Toor, Janine Warren, Haley W Greenyer, Jackson R Del Porto, Abigail G Person, Sadie J Korzec, Georgiy Zotkin, Jessica L Heath, Prachi N Ghule, J. Gordon, Andre J. van Wijnen, Seth E Frietze, Karen C. Glass, Jane Lian, Jane L. Stein, Gary S. Stein
          </td>
          <td>2025-07-16</td>
          <td>Biochemistry and cell biology = Biochimie et biologie cellulaire</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Radon (Rn) exposure has a strong association with lung cancer risk and is influenced by epigenetic modifications. To investigate the characterization of DNA methylation (DNAm) episignatures for radon-induced lung cancer, we detected the specific changes in DNAm in blood and lung tissues using reduced representation bisulfite sequencing (RRBS). We identified the differentially methylated regions (DMRs) induced by radon exposure. The bioinformatics analysis of the DMR-mapped genes revealed that pathways in cancer were affected by radon exposure. Among them, the DNAm episignatures of MAPK10, PLCG1, PLCβ3 and PIK3R2 were repeated between lung tissue and blood, and validated by the MassArray. In addition, radon exposure promoted lung cancer development in the genetic engineering mouse model (GEMM), accompanied by decreased MAPK10 and increased PLCG1, PLCβ3, and PIK3R2 with mRNA and protein levels. Conclusively, radon exposure significantly changes the genomic DNAm patterns in lung tissue and blood. The DNAm episignatures of MAPK10, PLCG1, PLCβ3 and PIK3R2 have a significant influence on radon-induced lung cancer. This brings a new perspective to understanding the pathways involved in radon-induced lung cancer and offers potential targets for developing blood-based biomarkers and epigenetic therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c05a3529ac357a7fffda3b4fd4ee34dc43dd89f" target='_blank'>
              Characterization of DNA Methylation Episignatures for Radon-Induced Lung Cancer
              </a>
            </td>
          <td>
            Ziyan Yan, Huixi Chen, Yuhao Liu, Lin Zhou, Jiaojiao Zhu, Yifan Hou, Xinyu Zhang, Zhongmin Chen, Yilong Wang, Ping-Kun Zhou, Yongqing Gu
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT The mechanisms underlying the assembly and regulation of enzymatic complexes responsible for plant organellar DNA replication remain poorly characterized. Unlike the monophyletic origin of the gene products involved in animal mitochondrial replication, derived from T‐odd bacteriophages, plant organellar DNA replication relies on genes either unique to plants or with origins traceable to bacteria and bacteriophages. Here, we demonstrate that the bacteriophage‐related primase‐helicase from Arabidopsis thaliana (AtTwinkle) is essential for double‐stranded DNA unwinding. AtTwinkle functionally interacts with bacterial‐related organellar DNA polymerases (AtPolIs), which lack the ability to unwind large regions of dsDNA, coupling DNA unwinding to processive DNA synthesis at the leading strand of the replisome. Analysis of two T‐DNA insertion mutants of AtTwinkle reveals distinct phenotypic outcomes; these mutant lines are hereafter referred to as ph. The ph1 (−/−) mutant, which carries a T‐DNA insertion in the 5´ UTR region, is viable and exhibits no noticeable developmental differences compared to wild‐type plants. In contrast, the ph2 mutant, with a T‐DNA insertion in the 19th exon, displays embryo lethality. Despite these differences, both ph1 (−/−) and heterozygous ph2 (+/−) mutants show a reduction in organellar DNA copy numbers under non‐stress conditions and exhibit heightened sensitivity to DNA‐damaging agents. In summary, our findings demonstrate that AtTwinkle is essential for organellar DNA replication. The heightened sensitivity of insertion mutants to organelle‐specific genotoxic agents indicates that loss of AtTwinkle function reduces the availability of template DNA necessary for double‐strand break (DSB) repair. Collectively, our findings reveal that two proteins of distinct evolutionary origins—AtTwinkle and plant organellar DNA polymerases—coevolved to coordinate DNA replication in plant organelles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baa75f5ccf72ac5f810f8cdfaa6b416c03addc39" target='_blank'>
              A Bacteriophage‐Derived Primase‐Helicase Orchestrates Plant Organellar DNA Replication
              </a>
            </td>
          <td>
            Carlos M Morales-Vázquez, Mayra A. Dagio-Hernández, Laura D Camacho-Manriquez, Antolín Peralta-Castro, Claudia D. Raygoza, Diana Solano-Argüello, Josue D Mora-Garduño, Rogelio González-González, H. Herrera-Ubaldo, C. Díaz‐Quezada, Alfredo Cruz-Ramírez, S. de Folter, J. Pedroza-García, Luis G. Brieba
          </td>
          <td>2025-07-01</td>
          <td>Physiologia Plantarum</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Simple Summary Cancer treatments that use drugs resembling DNA building blocks, known as nucleotide analogs, that insert themselves into DNA and prevent replication often stop working over time because cancer cells become resistant. This review explains how reversible epigenetic changes help cancer cells avoid being killed by these drugs. These changes do not alter the DNA sequence but can turn important genes on or off. We describe how cancer cells use these mechanisms to repair damaged DNA, pump drugs out of the cell, or break them down. We highlight the role of DNA packaging proteins (histones), small RNA molecules, and chemical tags on DNA that control how genes behave. Understanding these changes is important because it may allow clinicians to resensitize tumors and make them sensitive to treatment again and prevent relapse. We also explore new therapies, including drugs and gene editing tools, that target these epigenetic changes to improve the success of chemotherapy. These findings could lead to better outcomes for patients with difficult-to-treat cancers, especially those with pancreatic, colorectal, or breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbeef0502ae8a946f9634749621e6cdd28f4f464" target='_blank'>
              Epigenetic Drivers of Chemoresistance in Nucleobase and Nucleoside Analog Therapies
              </a>
            </td>
          <td>
            John Kaszycki, Minji Kim
          </td>
          <td>2025-07-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="During carcinogenesis, cells must acquire a telomere maintenance mechanism in order to avoid telomere shortening-induced replicative senescence. While most tumors activate telomerase, a minority of them employ a recombinational mechanism called Alternative Lengthening of Telomeres (ALT). One of the most investigated features is the association between ALT and ATRX mutations, since this has been shown to be the gene with the highest rate of mutations among ALT tumors. However, most of these studies, and in particular, mechanistic studies in vitro, have been carried out on mesenchymal tumors (sarcomas). In the present study, using genomic and expression data from the DepMap portal, we identified several non-mesenchymal ALT cell lines, and we compared the incidence of ATRX and other gene mutations between ALT cell lines of different origins (mesenchymal, neural, epithelial, hematopoietic). We confirmed that ATRX is frequently mutated in mesenchymal and neural ALT cell lines but not in epithelial ones. Our results showed that mutations of ATRX or other proteins involved in the maintenance of telomere integrity are needed for ALT activation in all cell types, and ATRX is preferentially mutated in mesenchymal ALT cells. Besides a more precise interpretation of the role of ATRX loss in ALT establishment, we proposed a model in which mutation of this gene impairs differentiation in mesenchymal and neural cells (but not in epithelial ones). Therefore, we explained the high incidence of ATRX mutations in mesenchymal and neural tumors with the fact that they both trigger ALT and impair differentiation, thus promoting two steps at once in the process of carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f454c9a5bcf29e99c79d4426a0a12fbbf069c0ed" target='_blank'>
              Alternative Lengthening of Telomeres: The Need for ATRX Mutations Is Lineage-Dependent
              </a>
            </td>
          <td>
            I. Udroiu, Antonella Sgura
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most prevalent and deadly neoplasms globally; this fact puts emphasis on the need for accurate molecular biomarkers for early detection and accurate prognosis. Circular RNAs (circRNAs) have recently emerged as very promising cancer biomarkers. In this study, we thoroughly examined whether the expression levels of circular transcripts of the protein arginine methyltransferase 1 (PRMT1) gene can predict the prognosis of patients diagnosed with colorectal adenocarcinoma, the most frequent type of CRC. Hence, a highly sensitive quantitative PCR (qPCR) assay was developed and applied to quantify circ-PRMT1 expression in cDNAs from 210 primary colorectal adenocarcinoma tissue specimens and 86 paired normal colorectal mucosae. Extensive biostatistical analysis was then performed to assess the potential prognostic power of circ-PRMT1. Significant overexpression of this molecule was observed in colorectal adenocarcinoma tissue samples in contrast to their non-cancerous counterparts. Moreover, higher circ-PRMT1 expression was correlated with poorer disease-free survival (DFS) and worse overall survival (OS) in colorectal adenocarcinoma patients. Interestingly, multivariate Cox regression analysis revealed that the prognostic value of the expression of this circRNA does not depend on other established prognostic factors included in the prognostic model. Furthermore, the stratification of patients based on TNM staging revealed that higher circ-PRMT1 levels were significantly related to shorter DFS and OS intervals, particularly in patients with colorectal adenocarcinoma of TNM stage II or III. In summary, this original research study provides evidence that circ-PRMT1 overexpression represents a promising molecular biomarker of poor prognosis in colorectal adenocarcinoma, not depending on other established prognostic factors such as TNM staging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b06ea5f530ca9a15a72069b8a7542753ca04c1f" target='_blank'>
              Overexpression of Circular PRMT1 Transcripts in Colorectal Adenocarcinoma Predicts Recurrence and Poor Overall Survival
              </a>
            </td>
          <td>
            P. Kokoropoulos, S. Christodoulou, P. Tsiakanikas, P. Vassiliu, C. Kontos, Nikolaos Arkadopoulos
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has been described in several eukaryotic species and has been shown to impact phenomena as diverse as cancer and herbicide tolerance. EccDNA is thought to arise mainly through transposable element (TE) mobilization. Because studies based on short-read sequencing cannot efficiently identify full-length eccDNA forms generated from TEs, we employed the CIDER-Seq pipeline based on long-read sequencing, to obtain full-length eccDNAs from Arabidopsis. The generated eccDNA datasets identified centromeric/pericentromeric regions as hotspots of eccDNAs with several eccDNA molecules originating from Helitron and LTR TEs. To investigate the role of epigenetic marks on TE-derived eccDNA biogenesis, we studied Arabidopsis methylation mutants dcl3, rdr6, ros1, and ddm1. Contrasting the TE-suppression previously reported in the hypermethylated ros1 mutants, we identified activation of TEs in ros1, specifically of LTR/Gypsy TEs. An enrichment of LTR/Copia elements was identified in actively dividing calli and the shoot apical meristem (SAM). Uncharacterized “variable TEs” with high eccDNA and expression were identified in the SAM, including ATCOPIA58. Together, our study reveals the genomic origins of eccDNAs and delineates the link between epigenetic regulation, transposon mobilization, and eccDNA biogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a8fe16883b4dfe42fe6b49221fc2e6a347cfdc2" target='_blank'>
              A comprehensive atlas of full-length Arabidopsis eccDNA populations identifies their genomic origins and epigenetic regulation
              </a>
            </td>
          <td>
            S. S. Zaidi, Sara Shakir, Hanne De Kort, Devang Mehta, V. Nguyen, Ruben Gutzat, Hervé Vanderschuren
          </td>
          <td>2025-07-01</td>
          <td>PLOS Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Although the significance of intratumoral heterogeneity (ITH) has been widely acknowledged in various cancers, its role in pancreatic cancer (PC) remains underexplored and warrants further investigation. Methods Pancreatic cancer transcriptomic data were acquired from the TCGA and GEO databases. The DEPTH2 algorithm, in combination with differential expression analysis, was used to identify genes associated with intratumoral heterogeneity (ITH). We applied univariate Cox regression analysis and multiple machine learning techniques to establish a reliable prognostic model. Patients were then stratified according to their ITH scores, and differences between subgroups were examined through pathway enrichment analysis, immune cell infiltration profiling, and drug response prediction. Furthermore, we conducted subcellular localization and differential analysis of ITH using single-cell data, followed by cell-cell communication analysis to explore interaction relationships and identify key pathways. Results Patients exhibiting lower intratumoral heterogeneity (ITH) levels demonstrated poorer clinical outcomes. The constructed 11-gene signature successfully differentiated individuals into high- and low-risk categories with significant survival differences. Immune profiling revealed notable differences in immune cell composition between the two groups, with patients in the high ITH cohort exhibiting enhanced immune activation. Drug sensitivity analysis indicated a differential response to therapies, with high-risk patients more resistant to certain drugs. Single-cell RNA sequencing identified a greater ITH score in epithelial cells, highlighting key interactions, particularly involving Galectin signaling pathways. Conclusion Our results highlight intratumoral heterogeneity’s prognostic and therapeutic relevance in pancreatic cancer, suggesting its potential utility in guiding individualized treatment approaches. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03080-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c15cd4c65cc7acb2163700609aefaa2af8b33ee" target='_blank'>
              Integrating bulk and single-cell RNA sequencing reveals intratumor heterogeneity phenotypes and immune infiltration in pancreatic cancer
              </a>
            </td>
          <td>
            Li Zhao, Yao Huang, Jiang Li, Rui Xie, Kaiming Li
          </td>
          <td>2025-07-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9589673dbf4b0dcfb9624f20bcd04da6bc6ed357" target='_blank'>
              YTHDF3 recognizes DNA N6-methyladenine and recruits ALKBH1 for 6mA removal from genomic DNA.
              </a>
            </td>
          <td>
            Xin-Hui Chen, Zi-Lu Wang, Jincui Yang, Min Chen, Si-Yi Zhao, Kun-Xiong Guo, Xuelong Zheng, Zhengwei Zhao, Xiaoqiang Chen, Jing Li, Minmin Zhang, Ling Ran, Huifang Zhu, Xiaofeng Gu, Guang-Rong Yan
          </td>
          <td>2025-07-25</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Survival strategy of bacteria is expanded by extrachromosomal elements (ECEs). However, their genetic diversity and functional roles for adaptability are largely unknown. Here, we discover a novel family of intracellular ECEs using 56 saliva samples by developing an efficient microbial DNA extraction method coupled with long-read metagenomics assembly. Even though this ECE family was not hitherto identified, our global prevalence analysis using 476 salivary metagenomic datasets elucidates that these ECEs reside in 74% of the population. These ECEs, which we named, “Inocles”, are giant plasmid-like circular genomic elements of 395 kb in length, including Streptococcus as a host bacterium. Inocles encode a series of genes that contribute to intracellular stress tolerance, such as oxidative stress and DNA damage, and cell wall biosynthesis and modification involved in the interactions with oral epithelial cells. Moreover, Inocles exhibit significant positive correlations with immune cells and proteins responding to microbial infection in peripheral blood. Intriguingly, we examine and find their marked reductions among 68 patients of head and neck cancers and colorectal cancers, suggesting its potential usage for a novel biomarker of gastrointestinal cancers. Our results suggest that Inocles potentially boost the adaptive capacity of host bacteria against various stressors in the oral environment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ef6b2491714273e3a5acd966b2b147c6dd3e3d" target='_blank'>
              Giant extrachromosomal element “Inocle” potentially expands the adaptive capacity of the human oral microbiome
              </a>
            </td>
          <td>
            Yuya Kiguchi, Nagisa Hamamoto, Y. Kashima, L. Runtuwene, A. Ishizaka, Yuta Kuze, T. Enokida, N. Tanaka, Makato Tahara, S. Kageyama, T. Fujisawa, Riu Yamashita, A. Kanai, Josef S. B. Tuda, Taketoshi Mizutani, Yutaka Suzuki
          </td>
          <td>2025-08-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31620c8deab30d15f98e2b8cf7eeeade98239959" target='_blank'>
              LINC01410 in cancer: a comprehensive review of its oncogenic role, regulatory mechanisms, and clinical implications.
              </a>
            </td>
          <td>
            Saman Morovat, Pejman Morovat, Ashkan Kalantary-Charvadeh, Marzieh Mojbafan, S. Teimourian
          </td>
          <td>2025-07-12</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) originates from neuroepithelial tissue and is one of the most common intracranial malignant tumors in adults, with high recurrence rate and poor prognosis. In recent years, SOX9 has been reported to play an important role in many diseases and cancers, and is a promising target, but it has been rarely reported in GBM. Methods RNA sequencing data of GBM were obtained from the Cancer Genome Atlas (TCGA) database and the Genotype-Tissue Expression (GTEx) database for analysis of SOX9 expression and differentially expressed genes (DEGs). Moreover, functional enrichment analysis of GBM-related DEGs was performed by GO/KEGG, GSEA, and protein-protein interaction (PPI) network. Additionally, the clinical significance of SOX9 in GBM was assessed by Kaplan-Meier Cox regression and prognostic model. What’s more, we analyzed SOX9-related immune cell infiltration and expression of immune checkpoints in GBM. The incorporated studies were analyzed using the R package. Results SOX9 was highly expressed in a range of malignant tumor tissues, including GBM. Surprisingly, high SOX9 expression was remarkably associated with better prognosis in the lymphoid invasion subgroups in a sample of 478 cases (P < 0.05). Totally, 126 differentially significant genes (DSGs) were identified between high- and low- expression group, of which 29 genes were upregulated and 97 genes were downregulated. Furthermore, high expression of SOX9 was an independent prognostic factor for IDH (isocitrate dehydrogenase)-mutant in Cox regression analysis. Screening was performed by LASSO coefficients to select non-zero variables that satisfied the coefficients of lambda. min, and four genes were screened out. OR4K2 and IDH status were prognostic factors associated with THCA in multifactorial COX regression analysis. SOX9, OR4K2 and IDH status were included in the nomogram prognostic model. Correlation analysis indicated SOX9 expression was correlated with immune cell infiltration and expression of immune checkpoints in GBM. Conclusion SOX9 was identified as a diagnostic and prognostic biomarker in glioblastoma, particularly in IDH-mutant cases. Its expression was closely correlated with immune infiltration and checkpoint expression, indicating its involvement in the immunosuppressive tumor microenvironment. SOX9-based gene signatures further supported a robust nomogram model, underscoring its potential as a therapeutic and prognostic target in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a2a8c97c4df9bf9651c0f0af4a4efc671535b22" target='_blank'>
              High expression of SOX9 is a diagnostic and prognostic indicator of glioma
              </a>
            </td>
          <td>
            Libo Xu, Zhenhao Wang, Mao Li, Qingsong Li
          </td>
          <td>2025-07-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9a4403306942197c6791291eb7e939f11dbad7" target='_blank'>
              Transposable elements in health and disease: Molecular basis and clinical implications.
              </a>
            </td>
          <td>
            Yaqiang Hong, Nian Liu
          </td>
          <td>2025-08-15</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The bacterial cytoplasm is characterized by a distinctive membrane-less organelle, the nucleoid, which harbors chromosomal DNA. We investigate the effects of dynamic processes associated with transcription and translation on the structure of this organelle, using coarse-grained molecular dynamics (MD) simulations implemented with out-of-equilibrium reactions. Our model captures the scale of the entire cell and incorporates a reaction-diffusion system for ribosomes and polyribosomes, combining their out-of-equilibrium dynamics with excluded volume interactions with DNA. Our findings demonstrate that out-of-equilibrium reactions increase the size of the nucleoid. In addition, we show that the nucleoid size increase is proportional to transcriptional activity. Our model reproduces the time-dependent change in nucleoid size observed in rifampicin treatment experiments, where the pool of polyribosomes is depleted. Furthermore, we find these active processes are essential for complete sister chromosome separation and correct nucleoid positioning within the cell. Overall, our study reveals the effects of the central dogma processes on the internal organization and localization of bacterial nucleoids.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5869c17c0b2f03dcafbf482f5c2ecb59859979e3" target='_blank'>
              Effects of central dogma processes on the compaction and segregation of bacterial nucleoids
              </a>
            </td>
          <td>
            Mu-Hung Chang, Maxim O. Lavrentovich, Jan-Michael Y. Carrillo, Jaan Männik
          </td>
          <td>2025-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Introduction Pancreatic cancer is a highly malignant tumor of the digestive system with a dismal prognosis. Despite advances in diagnosis and treatment, overall survival remains extremely low. Early diagnostic markers and an improved understanding of tumor-microenvironment interactions are essential for developing more effective therapies. Methods We analyzed 74 single-cell RNA sequencing (scRNA-seq) samples, performing unsupervised clustering and marker-gene expression profiling to define major cell types. Large-scale chromosomal copy-number variation (CNV) analysis distinguished malignant from non-malignant ductal cells. Non-negative matrix factorization (NMF) identified stage-associated gene modules, which were integrated with TCGA bulk-RNA data and machine-learning feature selection to pinpoint candidate prognostic genes. Two independent cohorts were used for validation. Regulatory network inference (pySCENIC) and ligand–receptor interaction analysis (CellPhoneDB) explored cross-talk between malignant cells and macrophages. Finally, in vitro knockdown of CTSV assessed its functional role in pancreatic cancer (PAC) cell proliferation and migration. Results Three prognosis-related genes—ANLN, NT5E, and CTSV—were selected based on their strong association with clinical stage and validated in external datasets. High expression of these genes correlated with poorer overall survival and an increased infiltration of M0 macrophages. CellPhoneDB predicted significant interactions between high-expression malignant ductal cells and M0 macrophages via CXCL14–CXCR4 and IL1RAP–PTPRF axes, with SPI1 identified as an upstream regulator of IL1RAP. In vitro CTSV knockdown significantly inhibited PAC cell proliferation and migration. Discussion Our integrative single-cell and bulk-RNA workflow identifies ANLN, NT5E, and CTSV as novel prognostic biomarkers in pancreatic cancer and highlights a pro-tumorigenic interaction between malignant ductal cells and macrophages. Targeting CTSV or disrupting CXCL14–CXCR4 and IL1RAP–PTPRF signaling may offer new therapeutic avenues for PAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50ce5676a35dfc57b1b078e8894ee25f419175d" target='_blank'>
              Integration of single-cell RNA and bulk RNA sequencing revealed malignant ductal cell heterogeneity and prognosis signatures in pancreatic cancer
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-07-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Although immune checkpoint inhibitors (ICIs) hold promise for those diagnosed with advanced human lung adenocarcinoma (LUAD), notable heterogeneity in patient responses and the complex tumor microenvironment (TME) limits their clinical utility while providing clinical benefit. To identify new therapeutic targets to pursue in chimeric antigen receptor (CAR) T-cell therapy, we leveraged an integrative multi-omic approach. This study identified three oncogenic drivers, karyopherin α2 (KPNA2), golgi membrane protein 1 (GOLM1) and thymidine kinase 1 (TK1), that are overexpressed in LUAD, spatially enriched in malignant niches and associated with significantly reduced overall survival (log-rank P < 0.01). Furthermore, functional interrogation using RNAi-mediated gene silencing in LUAD cell lines (NCI-H1650, A549), demonstrated their essential roles in mediating protumorigenic pathways. Specifically, CRISPR interference of KPNA2 or TK1 inhibited cellular proliferation and invasion while also phenocopying a pro-apoptotic effect with cisplatin. GOLM1 depletion inhibited PD-L1 upregulation and restored CD8 + T-cell cytotoxicity in co-culture. Mechanistically, dual knockdown targets, KPNA2 and TK1, were shown to restore deregulated STAT3 signaling that promotes cell survival and also enhance MHC class I antigen presentation implicating functional roles for KPNA2 and TK1 in linked processes of intrinsic oncogenesis and immunoediting by immune evasion and suppression. In summary, KPNA2, GOLM1, and TK1 are functionally relevant molecular coordinators for tumor cell autonomous proliferation and TME mediated immunosuppression. Their membrane expression (notable in the case of PD-L1) and functional roles in immune modulation, make them reasonable targets of CAR T-cell immunotherapy. We posit that a combined strategy targeting KPNA2, GOLM1 and TK1 with or without PD-1/PD-L1 axis inhibitors could provide a better strategy to treat patients with ICI resistant LUAD and generate prolonged and stable immune remodeling. Supplementary Information The online version contains supplementary material available at 10.1186/s40001-025-02955-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbb248ed2c42e77c5eb8fc8c37967d8c58decf41" target='_blank'>
              Coordinating oncogenesis and immune evasion: KPNA2, GOLM1, and TK1 as novel CAR T-cell targets in lung adenocarcinoma
              </a>
            </td>
          <td>
            Wangrui Liu, Hui Zou, Lei Guo, Zhonghua Zhou, Yahui Xie, Huaqi Guo, Gang Wei, Kai Zhang, Hui Yin, Shiyin Wei, Jiachang Chi
          </td>
          <td>2025-08-19</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer cells utilize codon-biased translation to fuel tumorigenesis and drug resistance, but underlying mechanisms remain poorly understood. Here, we show ALKBH1 is overexpressed in acute myeloid leukemia (AML) and essential for leukemia stem/initiating cell (LSC/LIC) self-renewal and AML development/maintenance, whereas dispensable for normal hematopoiesis. ALKBH1 enhances mitochondrial assembly/function and oxidative phosphorylation (OXPHOS), crucial for AML survival/proliferation and resistance to venetoclax, a potent BCL2 inhibitor and widely-used first-line targeted therapy for AML in clinic. Mechanistically, ALKBH1 catalyzes 5-formylcytosine (f5C) at tRNA wobble positions, reprograming decoding and facilitating codon-biased translation, a mechanism we term "Epitranslatomic Midas touch", which in turn drives leukemogenesis and drug resistance by promoting synthesis of key oncogenic proteins like WDR43. Targeting ALKBH1, particularly together with venetoclax, exhibited potent anti-leukemia efficacy in preclinical models with favorable safety profiles. Collectively, our findings elucidate ALKBH1's pivotal role in codon-biased translation and tumorigenesis, and propose a novel therapeutic strategy for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a3e11cf9a76aba0c7266c1034a7a9bfa25e9f8a" target='_blank'>
              ALKBH1 drives tumorigenesis and drug resistance via tRNA decoding reprogramming and codon-biased translation.
              </a>
            </td>
          <td>
            Chao Shen, Yuan Che, Keren Zhou, Kitty Wang, Wei Li, Diyuan Xue, Tong Wu, Lu Yang, Meiling Chen, Yue Sheng, Chengwan Zhang, S. Robinson, Huiying Chen, Lillian Sau, Zhenhua Chen, M. Wunderlich, Li Han, Tingting Tang, Y. Qing, Dong Wu, Miao Sun, Keith Leung, Katarzyna Dąbrowska, P. Pirrotte, Yueh-Chiang Hu, Lucy Y Ghoda, Bin Zhang, J. Mulloy, Minjie Wei, Guido Marcucci, Chuan He, Xiaolan Deng, Jianjun Chen
          </td>
          <td>2025-08-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Neoplastic transformation of mammalian cells involves intricate interactions between genetic, epigenetic and architecture modifications of the nucleus. Neuroblastoma is a malignant pediatric tumor with high biological and clinical heterogeneity representing a challenging model of study. We aimed to explore the changes in genome architecture and epigenetics being associated with neuroblastoma malignancy. We employed the neuroblastoma cell line SKNBE2 overexpressing the ncRNA NDM29 to differentiate from highly malignant into neuron-like cells. By 3D confocal microscopy, we explored the nuclear architecture (volume, elongation, compactness, and chromatin density). Using super-resolution microscopy (STED) and histone H3 immunolabelling we assessed the epigenetic rearrangement, and by enzyme-linked immunoassay the global DNA methylation. Then we assessed the mRNA expression of the main epigenetic modifying enzymes by quantitative PCR, and the expression of NF-κB-regulated genes by cDNA microarray. Compared to malignant NB cells, the NDM29-overexpressing cells, assuming a neuron-like phenotype, exhibited smaller and more elongated nuclei, redistribution of H3K9-acetylated and -methylated chromatin domains and DNA hypermethylation. In line with these results, in neuron-like cells the acetyltransferase KAT2A and the DNA methyltransferase DNMT1 were up-regulated, while most of NF-κB-regulated genes were down-regulated. Our findings reveal modifications of the nuclear structure and epigenome during neuroblastoma retro-transformation induced by NDM29 overexpression, with impacts on gene expression. These results offer potential insights into better understanding the mechanism of neuroblastoma malignancy in terms of chromatin rearrangements, opening exciting prospects for prognostic and therapeutic approaches with a focus on the nuclear level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53128c07de4d9b019036fd21bd73a5a365d261f0" target='_blank'>
              Epigenome and three-dimensional genome architecture remodeling during NDM29-mediated retro-transformation of neuroblastoma cells
              </a>
            </td>
          <td>
            Francesca Baldini, Aldo Pagano, L. Zeaiter, P. Bianchini, Hawraa Zbeeb, A. Diaspro, Laura Vergani
          </td>
          <td>2025-07-31</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="H3K27M diffuse midline gliomas (DMG) are characterized by p53 mutations and hypomethylation of MGMT, a DNA-repair enzyme, leading to resistance towards chemotherapeutic agents such as temozolomide (TMZ). As an alternative, we investigated the efficacy of a functionally different DNA-damaging agent, Val-083, on our DMG models. Val-083 is a blood-brain barrier penetrant DNA targeting agent that induces DNA N7-guanine interstrand crosslinks, which is unrepairable by MGMT. As Val-083 also triggers S/G2 phase cell cycle arrest for DNA repair, we evaluated its combined efficacy with Wee1 inhibitor, AZD1775. AZD1775 functions by inhibiting Wee1, at G2/M checkpoint to prevent phosphorylation of CDK1 and propel cells into the M phase. This subsequently overrides cell cycle arrest and drives cells with DNA damage into premature mitosis and apoptosis. Our results showed that Val-083 and AZD1775 work additively on a range of p53 mutant and p53 wildtype DMG models to inhibit cell growth, induce DNA damage and alter cell cycle. In addition, the combination drugs led to significant increase in the number of cells undergoing apoptosis, and a decrease in the migration and invasion activity of the cells. In vivo, the combination of both drugs led to significant reduction in tumor growth in zebrafish xenograft models and prolongation of survival in mice xenograft models. Our findings indicate that Val-083 and AZD1775 in combination demonstrate promising efficacy in DMGs, providing a clinical rationale for positioning these arms in future therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b7afa16a43447bdfe6a53168a1d6ac7f4603a62" target='_blank'>
              Dual treatment with Val-083 and AZD1775 shows potent anti-tumor activity in diffuse midline glioma models
              </a>
            </td>
          <td>
            Bavani Subramaniam, Debasish Roy, Ayda Woldegerima, Wai Chin Chong, J. Przystal, Jie Zhang, Sandra Laternser, Bettina Kritzer, Suraj Rangnath, Truman Knowles, Sridevi Yadavilli, R. Packer, Sabine Mueller, Javad Nazarian
          </td>
          <td>2025-07-01</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d7803de2661e62cd9d27e86eb2d0ae2e148e6e4" target='_blank'>
              Chromatin Control of EBV Infection and Latency.
              </a>
            </td>
          <td>
            P. Lieberman, I. Tempera
          </td>
          <td>2025-07-13</td>
          <td>Current topics in microbiology and immunology</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01fb40ddbc348d6bdf921a2ddd2696429a42111c" target='_blank'>
              Energy Deficiency-Induced ATG4B Nuclear Translocation Inhibits PRMT1-Mediated DNA Repair and Promotes Leukemia Progression.
              </a>
            </td>
          <td>
            Zhenkun Wang, Xianli Zhang, Yuanyuan Zhou, Shuyi Lin, Yuting Fu, Haiping Liu, Qiongdan Gao, YanHong Xiao, Zhao Yin, Shuang Liu, Kexiu Huang, Suqi Deng, Juan Du, Hui Zeng, Jia Wang, Jinping Zheng, Qinghua Zhou, Tianfeng Chen, Xiaoling Gao, Zhenyu Ju, Bo Liu
          </td>
          <td>2025-08-11</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Alkyladenine DNA glycosylase (AAG) is a critical enzyme in the base excision repair (BER) pathway, responsible for removing a broad spectrum of alkylated DNA lesions. While AAG maintains genomic stability, dysregulated activity has been implicated in cancer development, drug resistance, and neurodegenerative diseases. This review synthesizes the current knowledge on AAG’s structure, catalytic mechanism, and polymorphic variants, highlighting their potential roles in disease pathogenesis. A comprehensive bioinformatics analysis of over 370 AAG single-nucleotide polymorphisms (SNPs) is presented, identifying ~40% as high-risk variants likely to impair enzymatic function. Notably, 151 SNPs were predicted to be damaging by multiple algorithms, including substitutions at catalytic residues and non-conserved sites with unknown functional consequences. Analysis of cancer databases (COSMIC, cBioPortal, NCBI) revealed 93 tumor-associated AAG variants, with 18 classified as high-impact mutations. This work underscores the need for mechanistic studies of AAG variants using structural biology, cellular models, and clinical correlation analyses. Deciphering AAG’s polymorphic landscape may unlock personalized strategies for cancer prevention and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f35650f59a2a5d69d14473a41169b47ec06f0170" target='_blank'>
              The Link Between Human Alkyladenine DNA Glycosylase and Cancer Development
              </a>
            </td>
          <td>
            O. A. Kladova, A. A. Kuznetsova
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The induced “recombinogenic situation” in hematopoietic stem cells and the activation of the cell’s reparative systems create the basis for recombination events between fragments of extracellular double-stranded DNA delivered into the cell and chromosomal DNA or other forms of the reparative-recombination process. In mouse and rat model organisms as well as in human bone marrow cells, changes in the amount of telomeric DNA in hematopoietic stem cells were assessed as an indicator of repair and recombination events that have occurred. In all experiments performed, recombinant human angiogenin was used as a comparison factor. Dot blot hybridization showed that in the colony cells obtained from the bone marrow cells of the model organisms as well as from human bone marrow cells treated with a double-stranded DNA preparation, there was a significant increase in the amount of telomeric DNA. Amplification of telomeric DNA in colony cells is not associated with contamination of the original DNA preparation with which the bone marrow cells were treated. Treatment of bone marrow cells with DNA that does not carry telomeric sequences (AluI PCR fragment) does not lead to an increase in the amount of telomeric DNA in the cells of grown colonies. This suggests the participation in the amplification of telomeric DNA of an extrachromosomal DNA template carrying telomeric DNA. It has been established that treatment of bone marrow cells with angiogenin also leads to an increase in telomeric DNA in colony cells. A comparison of the type of colonies with the intensity of hybridization (i. e. the amount of telomeric DNA in the sample) suggests that the increase in the amount of detectable telomeric DNA following treatment with angiogenin and hDNAgr has a fundamentally different origin. Western blot analysis and real-time PCR revealed that the increase in the amount of telomeric DNA following treatment of bone marrow cells with a double-stranded DNA preparation does not correlate with the activity of endogenous/exogenous telomerase. For angiogenin, it has been shown that an increase in the amount of telomeric DNA may be the result of activation of endogenous telomerase activity. A principle has been developed for the amplification of a new genetic trait that came into hematopoietic stem cells with extracellular double-stranded DNA material and was fixed in the recipient genome or was transitively present in the cell as new genetic information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11f0c77ec133a09bb58f64518224081f769056e8" target='_blank'>
              Concept of natural genome reconstruction. Part 3. Analysis of changes in the amount of telomeric DNA in colony cells as a new amplified feature that arose during the processing of hematopoietic bone marrow stem cells
              </a>
            </td>
          <td>
            V. Ruzanova, S. Oshikhmina, G. S. Ritter, E. Dolgova, S. Kirikovich, E. Levites, Y. R. Efremov, T. V. Karamysheva, A. Bogomolov, M.I. Meschaninova, A. L. Mamaev, O. Taranov, S. V. Sidorov, S. D. Nikonov, O. Leplina, A. Ostanin, E. Chernykh, N. A. Kolchanov, A. Proskurina, S. S. Bogachev
          </td>
          <td>2025-07-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Mobile genetic elements (e.g. transposons and introns) are genomic parasites, DNA sequences that can proliferate within host genomes. In prokaryotes, these elements are minimized by purifying selection and host-cell defenses. In eukaryotes, however, these parasitic sequences may comprise most of the genome. This extreme imbalance in eukaryotes is maintained by a metabolically quiescent germline through which the genomic parasites are transmitted to the next generation via gametes and spread throughout the population via sexual reproduction. Some eukaryotes avoid the threat posed by genomic parasites by eliminating them from somatic cells, whereas others retain the parasites in somatic cells and attempt to suppress them by epigenetic silencing mechanisms. Here, we review the evidence for the ongoing competition between host and parasite for genome occupancy. We conclude that defenses against mobile genetic elements vary greatly among organisms, and this variation accounts for the enormous range in genome size among organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58c5828d52742180cad7b0c29c98a298b55b8bda" target='_blank'>
              The biological and evolutionary consequences of competition between DNA sequences that benefit the cell and DNA sequences that benefit themselves
              </a>
            </td>
          <td>
            A. J. Bendich, Scott O. Rogers
          </td>
          <td>2025-07-08</td>
          <td>Nucleic Acids Research</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="Abstract Chromatin relaxation is a permissiven progress for DNA repair through enabling repair factors to access the damaged DNA. Linker histone H1 is important in maintaining chromatin compaction under physiological state. The recent evidence highlights the importance of H1 modifications in response to cellular stress. Following DNA double‐strand breaks, the metabolic enzyme phosphorylated CTP synthase 1 (CTPS1) functions as a deamidase, catalyzing the rapid conversion of H1 residues Asn76 and Asn77 into aspartate. This modification enables subsequent acetylation at Lys75 by the histone acetyltransferase p300, thereby reducing H1‐DNA affinity and promoting chromatin decompaction. This sequential modification‐H1 deamidation followed by acetylation‐facilitates the recruitment of repair factors involving both homologous recombination and non‐homologous end joining repair pathways, and consequently promoting DNA repair. Importantly, high expression of CTPS1 is associated with resistance to radiotherapy in mouse models and clinical cancer patients, suggesting that the CTPS1 may serve as a potential therapeutic target. While targeting CTPS1 may offer opportunities to enhance radiosensitivity of cancer patients, challenges related to specificity and off‐target effects require further studies. This article highlights an emerging role of H1 modification in the DNA damage repair and discusses the therapeutic potential of manipulating H1 deamidation in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57ef2febfbdf0354039134d0a6f7b0aac3792561" target='_blank'>
              H1N76/77 deamidation facilitates chromatin remodelling and genome stability during DNA damage repair
              </a>
            </td>
          <td>
            Tingting Feng, Mengyan Li, Chenmiao Hu, Yuan Tian, Wei-Guo Zhu
          </td>
          <td>2025-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fda3215ef96d1797670c5063bd5db4df44e918" target='_blank'>
              Genetic variants in oncogenic miRNA and 3' untranslated region of tumor suppressor genes: emerging insight into cancer genetics.
              </a>
            </td>
          <td>
            Anita Choudhary, Anil Kumar, A. Munshi
          </td>
          <td>2025-06-30</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41c845deccf83a402934c97c50b9796b93645c5b" target='_blank'>
              Establishment of chromatin architecture interplays with embryo hypertranscription.
              </a>
            </td>
          <td>
            Guang Yu, Kai Xu, Weikun Xia, Ke Zhang, Qianhua Xu, Lijia Li, Zili Lin, Ling Liu, Bofeng Liu, Zhenhai Du, Xia Chen, Qiang Fan, Fangnong Lai, Wenying Wang, Lijuan Wang, Feng Kong, Chao Wang, Haiqiang Dai, Huili Wang, Wei Xie
          </td>
          <td>2025-08-13</td>
          <td>Nature</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Transposons are mobile genetic elements capable of moving within the genome. Leveraging this property—particularly the cut-and-paste mechanism of DNA transposons—has enabled the development of technologies for inserting exogenous DNA fragments into host genomes. While targeted integration is a key goal for therapeutic applications, this review highlights the value of their intrinsic randomness. By combining the ability to freely design the DNA cargo with the stochastic nature of transposon integration, it becomes possible to generate highly sensitive reporter cells. These can be used to efficiently identify functional markers, uncover novel signaling pathways, and establish innovative platforms for drug screening. As more subfamilies of transposons become available for research use, their complementary biases may enhance the coverage and diversity of genome-wide screening approaches. Although inherently unpredictable, this strategy embraces randomness as a strength, and we propose that it holds great promise for driving new advances in biology, cellular engineering, and medical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4d806cc8a8f4473c66fef98ec788d4bd5b59391" target='_blank'>
              Random Insertion Reporter Gimmicks Powered by Cut-and-Paste DNA Transposons
              </a>
            </td>
          <td>
            Yamato Kasahara, Kentaro Semba, Shinya Watanabe, K. Ishikawa
          </td>
          <td>2025-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Nucleoside analogues are crucial in treating non-small cell lung cancer (NSCLC), but resistance hampers patient outcomes. The cytidine analogue RX-3117 shows promise in gemcitabine-resistant cancers, yet mechanisms underlying acquired resistance to this drug remain unexplored. This study includes a comprehensive investigation into RX-3117 resistance mechanisms by leveraging new preclinical models and cutting-edge genomic tools, including a CRISPR-Cas9 knockout screen and transcriptomics. Methods NSCLC cell lines A549 and SW1573 were exposed to stepwise increasing concentrations of RX-3117 to establish stable resistant subclones, confirmed by SRB and clonogenic assays. Intracellular RX-3117 nucleotide levels were measured via LC/MS-MS, prompting the evaluation and modulation of the expression of key metabolic enzymes by Western blot and siRNA. A CRISPR-Cas9 screen identified genes whose loss increased RX-3117 sensitivity, while RNA-sequencing with differential expression analyses revealed resistance-related pathways, further investigated through cell cycle distribution, knock-out, and ELISA assays. Results Resistant clones exhibited decreased accumulation of RX-3117 nucleotides, which however, was not associated to reduced expression of activation enzymes (UCK2, UMPK, CMPK, NME1/NDPK, RR1 and RR2). Instead, increased expression was observed in certain DNA repair and deactivation enzymes (NT5C3) but pharmacological inhibition and silencing of the latter did not circumvent resistance. Remarkably, a comprehensive approach with CRISPR-Cas9 screen highlighted DNA-repair and cell cycle determinants as key sensitizing genes. XL-PCR and RNA-sequencing confirmed aberrations in DNA-repair and pathways involved in cell cycle regulation. Knock-out and pharmacological inhibition validated the role of PKMYT1, a protein kinase involved in G2/M transition and genomic stability. RX-3117-resistant A549 cells showed enhanced sensitivity to the PKMYT1 inhibitor lunresertib and its synergism with RX-3117, suggesting further studies, especially in patients with high PKMYT1 expression who have significantly shorter survival rates, as observed in public databases and validated in an internal cohort of NSCLC patients. Conclusion By integrating CRISPR-Cas9 with functional assays and transcriptomics, our study established a framework for decoding resistance mechanisms and highlights potential therapeutic strategies to enhance RX-3117 efficacy in NSCLC. We demonstrated for the first time that aberrant DNA repair and cell cycle dysregulation led resistance, identifying PKMYT1 as a promising target. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03470-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a070b145fa4db64737d5a930cbf0a0bb16c66539" target='_blank'>
              Unraveling resistance mechanisms to the novel nucleoside analog RX-3117 in lung cancer: insights into DNA repair, cell cycle dysregulation and targeting PKMYT1 for improved therapy
              </a>
            </td>
          <td>
            M. Vahabi, Geng Xu, Dzjemma Sarkisjan, B. E. Hassouni, G. Mantini, Valentina Donati, Bing Wang, Giulia Lencioni, R. Honeywell, Dongmei Deng, S. Strano, Godefridus J. Peters, G. Blandino, E. Giovannetti
          </td>
          <td>2025-07-24</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13c548af869e91466a22ceb7887ea1b6fdb37dc0" target='_blank'>
              Integrative proteogenomics and forward genetics reveals a novel mitotic vulnerability in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Nicholas J Neill, S. Satpathy, Karsten Krug, Jitendra K Meena, Nivetha Ramesh Babu, Cheyenne Calderon, Desmon Reed, Marcus J Weber, L. Dobrolecki, Alaina Lewis, Christina Sallas, M. Anurag, Kimberly R Holloway, Chen Huang, S. Vasaikar, Maria F Cardenas, Beom-Jun Kim, Doug W. Chan, Shayan C. Avanessian, S. Tyagi, Mayra Orellana, Sufeng Mao, Heyuan Li, Fade Gong, S. Kurley, K. Meerbrey, Calla M Olson, Amritha Nair, Tingting Sun, Hsiang-Ching Chung, Elizabeth A. Bowling, Jarey H Wang, Pengju Zhang, Peng Xiao, Duxiao Yang, F. Stossi, Mei-Yin C Polley, Alexander B. Saltzman, F. Mundt, D. Mani, Michael A Gillette, S. Hilsenbeck, George Miles, Carolina Gutierrez, C. K. Osborne, Charles Y. Lin, N. Gray, Jinpeng Sun, David A. Wheeler, C. Perou, A. Malovannaya, Michael T. Lewis, Bing Zhang, Matthew J Ellis, Steven A. Carr, Thomas F Westbrook
          </td>
          <td>2025-08-11</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>165</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0641382c381145c3c4ec79eaca66ce9a1d4b8df0" target='_blank'>
              Telomere Crisis Shapes Cancer Evolution.
              </a>
            </td>
          <td>
            Joe Nassour, J. Karlseder
          </td>
          <td>2025-08-11</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The AP-1 transcription factor FOSL1, also known as Fra-1, is a crucial oncoprotein that plays an important role in human tumor progression and metastasis and has thus emerged as a promising therapeutic target. FOSL1 regulates the expression of a large protein-coding gene network, and this molecular mechanism can promote the progression of tumors. Interestingly, recent studies have shown that FOSL1 can also achieve the same protumor effect by regulating certain noncoding RNAs (ncRNAs). However, more studies have shown that ncRNAs can regulate the expression and activity of FOSL1, thereby affecting the occurrence and development of tumors, which indicates that ncRNAs can be regulators of FOSL1 in cancer. In this review, we first provide a comprehensive overview of the expression and function of FOSL1 and ncRNAs in tumors and then focus on the mutual regulatory relationship between ncRNAs and FOSL1, as well as their regulatory effects on and mechanisms of tumor progression. In addition, we further explored the potential clinical applications of the FOSL1-ncRNA system in cancer treatment, providing a theoretical basis for the study of FOSL1 and/or ncRNA-related molecular markers or targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f413c89530da64009a6941f40cf33c862cd2843" target='_blank'>
              Noncoding RNAs as regulators of FOSL1 in cancer
              </a>
            </td>
          <td>
            Xiaochang Wang, Li Wang, Shoushi Wang, Jinjie Zhang, Xueyang Wang, Ting Zhang, Linwei Li, Jin Wei, Yi Zhao, Zhixia Zhou
          </td>
          <td>2025-08-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background/Objectives: Krüppel-like factor 4 (KLF4) emerged as an epigenetically regulated gene in a variety of settings, including cell reprogramming and malignant cell proliferation. The aim of the present manuscript is to explore the relationship described in recent years between circular RNAs, miRNAs, and KLF4. These have been shown to be involved in cancers having diverse histological origins, including some of the most prevalent and deadly tumors for the human population. Expression and protein levels of this transcription factor correlate with invasiveness and prognosis in a context- and tissue-specific fashion. Methods: The literature was obtained through two main PubMed queries. The first is “miRNA and KLF4 and cancer” and is limited to the last 5 years. The second is “circRNA and KLF4”, which yielded publications between 2013 and 2024. The oncological publications were selected. Results: A number of circRNA/miRNA axes that regulate the downstream transcription factor KLF4 emerged in the last few years. circRNAs act as sponges for miRNAs and synergize with KLF4, which can function as either a tumor promoter or suppressor in different tumors. Conclusions: The axes represented by circRNA/miRNA/KLF4 emerged as a new layer of epigenetic regulation. These RNA-based modulators explain the complex regulation of this transcription factor and open the way to new therapeutic targeting possibilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9a3e7b131d3c048a24a47a22d4b39bf8b794eef" target='_blank'>
              circRNA/miRNA Networks Regulate KLF4 in Tumor Development
              </a>
            </td>
          <td>
            R. Frazzi, E. Farnetti, D. Nicoli
          </td>
          <td>2025-07-29</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1353c24c2d18ad56030f170e19dffc05bf916315" target='_blank'>
              HPV16 recruitment of SMARCAL1 to viral and host replication forks is required for the viral life cycle.
              </a>
            </td>
          <td>
            C. James, Aya H. Youssef, Jenny D Roe, Floriana Cappiello, F. Aiello, Benedetta Perdichizzi, Rachel L. Lewis, Austin J. Witt, A. T. Prabhakar, Xu Wang, Molly L Bristol, P. Pichierri, I. Morgan
          </td>
          <td>2025-07-22</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="RNA is a fundamental biological macromolecule that undergoes several post-transcriptional modifications, including adenosine to inosine (A-to-I) editing by adenosine deaminases acting on RNA (ADARs). These essential enzymes catalyze the conversion of A-to-I in double-stranded RNA (dsRNA) molecules, influencing RNA stability, splicing, and translation, all of which impact various cellular functions. More recently, RNA editing has emerged as a pivotal mechanism in cancer biology, where ADARs, primarily ADAR1 and ADAR2, exert context-dependent roles as either oncogenic drivers or tumor suppressors. Beyond their catalytic editing function, ADARs also regulate cancer-relevant pathways through editing-independent mechanisms, including RNA binding and protein-protein interactions. Dysregulated ADAR activity facilitates carcinogenesis by altering oncogene expression, impairing tumor suppressor pathways, and reprogramming the transcriptome to promote tumor progression. Furthermore, RNA editing may contribute to tumor cell immune evasion by affecting interferon signaling and altering neoantigen presentation, as well as modulating immune surveillance. Additionally, ADAR-mediated RNA modifications contribute to therapy resistance by modifying drug targets and pathways involved in cell survival and repair. This review comprehensively analyzes the multifaceted roles of RNA-editing ADAR enzymes in cancer pathogenesis, emphasizing editing-dependent and -independent mechanisms contributing to tumor progression, immune evasion, and resistance to therapy. Moreover, we highlight the potential of ADARs as prognostic biomarkers and promising therapeutic targets in oncology. This review aims to spark novel precision oncology and cancer immunotherapy strategies by bridging molecular insights with translational applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b01878fae3478e3acc0ed674ac3f3c9f7ba1c042" target='_blank'>
              Deciphering the mechanistic roles of ADARs in cancer pathogenesis, tumor immune evasion, and drug resistance
              </a>
            </td>
          <td>
            Xiaoke Wang, Fei Yin, Yangfang He, Yue Qiao, Yan Yan, Jingru Ma
          </td>
          <td>2025-08-07</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The development of multiple myeloma is typically associated with various cytogenetic abnormalities; however, these genetic changes alone do not fully account for the observed heterogeneity in patient prognosis and treatment response. Recent studies leveraging next-generation sequencing and genomic approaches have shown that epigenetic alterations are crucial in myeloma development and therapeutic resistance. These changes contribute to high levels of transcriptomic instability and enables cellular adaptation to targeted therapies and immunotherapies through diverse evolutionary trajectories. In this regard, aberrations of histone modifications and chromatin remodeling affect various cellular processes such as DNA repair, DNA damage response, cellular survival, and apoptosis signaling, which provides a strong rationale for developing epigenetic-targeted therapies for myeloma treatment. In this review, we focus on recent advances and research gaps in understanding the deregulation of histone acetylation, a widespread and versatile process of histone modification occurring at lysine residues at the N-terminus of histone tails, and its intimate interplay with chromatin remodeling complexes in orchestrating dynamic chromatin functional states and transcriptional outputs. We also provide an updated review of epigenetic modulatory drugs targeting HDAC, CBP/p300, and bromodomain and extraterminal proteins, along with a discussion of their limitations and future perspectives in myeloma treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d100926ef56ad097039cb56ac307d5417be61de8" target='_blank'>
              Deciphering the dynamics of histone acetylation and chromatin remodeling in multiple myeloma: a tale beyond the tails.
              </a>
            </td>
          <td>
            Sinan Xiong, Jianbiao Zhou, WeeJoo Chng
          </td>
          <td>2025-07-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Understanding how the assembly of nascent mRNA into a ribonucleoprotein (mRNP) influences R-loop homeostasis is crucial for gaining insight into the cellular mechanisms that prevent genome instability. Here, we identify three RNA-binding proteins, Rie1, Rim4 and She2, whose expression levels are important to limit R-loop accumulation and, thus, to prevent DNA damage. Interestingly, Rim4 and She2 are overrepresented in CBP80-containing mRNPs formed in the absence of THO. In addition, we found that an excess of the RNA exosome component Dis3 impairs its function, promoting R-loops, particularly from non-coding RNAs, which cause genomic instability. Our results indicate that changes in the availability of different RBPs or RNAs, causes R-loop-mediated DNA damage in the cell. These results may help to understand the mechanism that promotes cancer, as several RBPs are overexpressed in different types of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58254c8464c859b1341c1f452949c65b62663dc9" target='_blank'>
              Cellular imbalance of specific RNA-binding proteins associates with harmful R-loops
              </a>
            </td>
          <td>
            José Antonio Mérida-Cerro, Guillaume Chevreux, Benoît Palancade, Ana G. Rondón, A. Aguilera
          </td>
          <td>2025-07-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Long noncoding RNAs (lncRNAs), defined as noncoding transcripts exceeding 200 nucleotides without protein-coding potential, have been increasingly recognized for encoding polipeptides or proteinsimplicated in the regulation of malignant tumor initiation, progression, and prognosis. Among them, small nucleolar RNA host gene 29 (SNHG29) has attracted increasing attention because of its aberrant expression across diverse malignancies. This report reviews the latest research regarding its expression patterns, regulatory functions, and involved signaling pathways. This review synthesizes current insights into the contributions of SNHG29 to oncogenic processes, including cell proliferation, migration, epithelial‒mesenchymal transition, chemoresistance, and immunosuppression. Furthermore, evidence indicates that SNHG29 functions as a competing endogenous RNA (ceRNA), modulating tumor progression via the miRNA‒mRNA axis and signaling pathways such as the ATR‒Chk1, PI3K‒AKT, and Wnt‒β-catenin pathways. Cumulative data position SNHG29 as a promising diagnostic and prognostic biomarker, as well as a potential molecular target for therapeutic intervention in various malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef7951689b147bbb46276371501a41cc5d89fba" target='_blank'>
              The role of long noncoding RNA SNHG29 in malignant tumors
              </a>
            </td>
          <td>
            Tingcheng Bu, Wendong Gu, Yingjie Shao
          </td>
          <td>2025-07-15</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Wilms tumor (nephroblastoma), the most common pediatric renal malignancy, has a complex genetic and epigenetic landscape. While mutations in genes like WT1, CTNNB1, and WTX have been well characterized, accumulating evidence suggests that epigenetic dysregulation plays a pivotal role in WT pathogenesis. This review examines the various epigenetic mechanisms implicated in WT, including DNA methylation, histone modifications, chromatin remodeling, and non-coding RNA-mediated regulation. We discuss how epigenetic mechanisms contribute to tumor initiation, progression, and heterogeneity and their implications for improved diagnosis and targeted therapy. We also highlight recent advances in epigenomic profiling, discuss the interplay between epigenetics and developmental gene expression programs, and evaluate potential therapeutic strategies targeting epigenetic regulators.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fa17235c52d629b5c3bb715ccfc06b98890be55" target='_blank'>
              Epigenetic Regulation in Wilms Tumor
              </a>
            </td>
          <td>
            Annabelle Bolitho, Hongbing Liu
          </td>
          <td>2025-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f72e0c1deac040357dc9e7ae4b906709cffda79a" target='_blank'>
              Methylation dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Adriana V.A. Fonseca, Christopher Boniface, Caroline J. Watson, Samuel Hackett, C. Gabbutt, Akemi D. Ramos-Yamasaki, Amanda B. Tan, Jose Montoya, R. Strogantsev, Lars L. P. Hanssen, Yuexuan Zhang, S. Apostolidou, A. Gentry-Maharaj, Sadik C Esener, Paul T Spellman, Trevor Graham, U. Menon, Hisham Mohammed, J. Blundell
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Abstract Nuclear architecture and chromosome folding are often speculated to influence genome replication. In yeasts, centromeres cluster close to the spindle pole body and telomeres position at the nuclear periphery. This ‘Rabl configuration’ spatially segregates the most early and late replicating parts of the genome, centromeres and telomeres, respectively, suggesting that origin position along the centromere–telomere axis may influence origin activity. Here, we investigated DNA replication in a wild-type, 16-chromosome Saccharomyces cerevisiae strain and in its single-chromosome counterpart engineered by chromosome fusion and elimination of all but two telomeres and one centromere, which strongly affects genome folding and abrogates the Rabl conformation. Using nanopore sequencing-based methods, we found that the DNA replication program of both strains was virtually indistinguishable, with the exception of origin inactivation next to deleted centromeres and changes in origin efficiency and fork direction at chromosome fusions, as anticipated from the known origin-regulation properties of centromeres and telomeres. Only a handful of replication changes, mostly due to local origin repression, were observed elsewhere. Fork speed was also unaffected except at deleted centromeres. In conclusion, the DNA replication program of budding yeast is remarkably resilient to perturbations of chromosome folding and loss of the Rabl conformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e28f5bc4a9bd5e110c57ffd165964c5f1a57708" target='_blank'>
              Replication program of a single-chromosome budding yeast strain
              </a>
            </td>
          <td>
            Jade Pellet, Laurent Lacroix, Bertrand Theulot, Emma Simonin Chavignier, Alan Tourancheau, F. Proux, Sylvie Hermann-Le Denmat, G. Millot, Benoît Le Tallec, O. Hyrien
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Key message CRISPR/Cas9-mediated modification of OsCENH3 induces aneuploidy but fails to trigger haploid formation in rice, underscoring limited efficiency and the need for strategy refinement. Abstract The centromeric histone H3 variant (CENH3) is essential for kinetochore assembly and accurate chromosome segregation during cell division. Alterations in CENH3 have been shown to trigger haploid induction in various plant species; however, its utility in rice remains largely unexplored. In this study, six OsCENH3 mutant lines were generated through CRISPR/Cas9-mediated genome editing, complemented by a GFP-tagged construct. To assess their haploid induction potential, both self pollination and outcrossing with the cultivar Nipponbare (NIP) were performed. Flow cytometry analysis revealed that none of the mutants produced true haploids. However, two aneuploid individuals (Het-C1-1-1 and Het-C1-5-1) were identified among the hybrid progeny derived from tail domain mutants C1-1 and C1-5, suggesting partial chromosome missegregation. These aneuploid plants exhibited reduced pollen viability, abnormal morphology, and compromised agronomic performance. In addition, significantly elevated rates of embryo/endosperm abortion were observed across different crosses, which far exceeded the frequency of aneuploid production, implying that early-stage chromosomal instability may result in embryo lethality. Collectively, while specific OsCENH3 mutations can induce limited chromosomal instability, their haploid induction efficiency remains low in rice. Further refinement of editing strategies and exploration of favorable genetic backgrounds will be essential for developing effective centromere-based haploid induction systems. This study provides a theoretical framework and technical reference for engineering haploid inducers in rice through centromere manipulation. Supplementary Information The online version contains supplementary material available at 10.1007/s00299-025-03581-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e47eb21bd47c4bbfb2065f109453ba3b80c2b99" target='_blank'>
              Generation and characterization of rice OsCENH3 mutants for haploid induction
              </a>
            </td>
          <td>
            Siyi Liang, Sisi Jia, Wenyu Lu, Jiafeng Wang, Ming Huang, Chun Chen, Cuihong Huang, Danhua Zhou, Tao Guo, Hong Liu
          </td>
          <td>2025-08-01</td>
          <td>Plant Cell Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Introduction Recent advancements in transcriptomic analysis, combined with single-cell RNA sequencing (scRNA-seq) and spatial transcriptomics, have deepened our understanding of the tumor microenvironment and cellular heterogeneity, laying the groundwork for personalized therapies. The aim of this research is to explore the heterogeneity of tumor cells in colorectal cancer (CRC) and evaluate their prognostic value in different therapeutic contexts, emphasizing the impact of tumor cell heterogeneity on disease progression. Methods scRNA-seq alongside spatial transcriptomics was employed to analyze the heterogeneity of tumor cells in CRC, the spatial distribution of tumor cells, and their interactions with the microenvironment. Results We identified nine distinct tumor cell subtypes, with MLXIPL + neoplasm prevalent in advanced CRC, while ADH1C + and MUC2 + neoplasms were more common in early-stage CRC. MLXIPL + neoplasm was significantly associated with chemotherapy and targeted therapy efficacy. Analysis of spatial transcriptomics indicated that MLXIPL + neoplasm is located in the core area of the tumor cells. We developed a 13-gene prognostic signature (PS) using machine learning algorithm (StepCox backward), which predicts the prognosis of CRC patients. Furthermore, the patients with low PS score demonstrated higher immune cell infiltration and immune regulatory factors, suggesting enhanced immune surveillance and treatment response. Conclusions The findings highlight the critical role of tumor cell heterogeneity in CRC progression and treatment response, suggesting the need for personalized therapeutic strategies targeting different subpopulations. The constructed PS demonstrates significant clinical application value in predicting patient prognosis. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00805-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8065bcb7e9315d28e942e5a0a0b33a35e0d6f402" target='_blank'>
              Spatial transcriptomics and scRNA-seq: decoding tumor complexity and constructing prognostic models in colorectal cancer
              </a>
            </td>
          <td>
            Wei Song, Yatao Wang, Min Zhou, Fengqin Guo, Yanliang Liu
          </td>
          <td>2025-08-13</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="This review article explores the role of immuno-radiotherapy in the context of genome instability and tumour evolution. Genomic changes in tumours exist in a delicate balance with the immune system, offering evolutionary pathways to adapt and grow but risking provoking an immune response. Rapid developments across both immunotherapy and radiotherapy have raised questions about the potential benefits combination therapy, and how best to identify ideal treatment populations. Here we discuss foundational studies of genomic instability and tumour evolution, how these paradigms translate into immune surveillance and evasion, and subsequently go on to explore recent preclinical and clinical studies of both treatment modalities. Understanding how cancers evolve in the context of the immune system could provide a key insight in delivering better therapies that could overcome treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4d64cec361f889e2be65100e62ef8d285da2f26" target='_blank'>
              Tumour Evolution Driving Genome Instability, Immune Interactions, and Response to Radiotherapy
              </a>
            </td>
          <td>
            G. Seed, France Truong, Rahma Riahi, Ben O’Leary
          </td>
          <td>2025-06-09</td>
          <td>Cancer journal (Sudbury, Mass.)</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Bladder cancer has been considered as one of the most common urinary malignancies. Growing evidence has indicated that Centromere Protein F (CENPF) is a promising molecular biomarker for many human malignant tumors. However, the role of CENPF in bladder cancer (BC) proliferation, migration, invasion, cell cycle and apoptosis thereof remain unclear. In the present study, high-throughput sequencing technology and bioinformatics analysis were conducted to identify mRNAs profiles in 10 pairs of bladder cancer tissues and adjacent noncancerous tissues. CENPF was overexpression in bladder cancer tissues, and higher in MIBC than NMIBC. We investigated the expression of CENPF in 30 other pairs bladder cancers tissues, and the results were in accordance with the sequencing results. Furthermore, Immunohistochemical staining, showed that strong intensity of CENPF in BC tissues than normal tissues. Increased staining of CENPF was detected of tumor cells in MIBC compared with NMIBC. This suggests that CENPF might be highly expressed in aggressive and invasive tumor cells. Subsequently, in vitro functional experiments also demonstrated that the siRNA interference of CENPF expression significantly weakened the proliferation, migration, invasion and apoptosis of BC cells, and the cells were arrested in the G2/S phase in Cell cycle. Moreover, functional enrichment analyses, lncRNA-miRNA-mRNA, and protein-protein interaction networks revealed that CENPF was potentially involved in carcinogenesis and evolution of bladder cancer. Taken together, these results demonstrated that CENPF may serve as a potential biomarker of tumor occurrence, progression, and even prognosis for bladder cancer. However, further research is needed to be further clarified the pathway mechanisms of CENPF in bladder cancer. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10677-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcea81a86745e041752c09be80ee21f9ca4c7d66" target='_blank'>
              CENPF overexpression in bladder cancer cells enhances proliferation, migration, invasion, and apoptosis
              </a>
            </td>
          <td>
            Ruixiang Song, Xin Chen, Zhensheng Zhang, Huiqing Wang, Jinshan Xu, Shuxiong Zeng, Wentao Zhang, Zhaoyang Sheng, Xudong Yao
          </td>
          <td>2025-07-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The human mitochondrial helicase Twinkle is essential for mitochondrial DNA (mtDNA) replication and integrity. Using biochemical and single-molecule techniques, we investigated Twinkle’s real-time kinetics, including DNA loading, unwinding, and rewinding, and their regulation by its N-terminal Zinc-binding domain (ZBD), C-terminal tail, and mitochondrial SSB protein (mtSSB). Our results indicate that Twinkle rapidly scans dsDNA to locate the fork, where specific interactions halt diffusion. During unwinding, ZBD-DNA interactions and C-terminal tail control of ATPase activity downregulate kinetics, slowing down the helicase. Binding of mtSSB to DNA likely outcompetes ZBD-DNA interactions, alleviating the downregulatory effects of this domain. Furthermore, we show that ZBD-DNA interactions and ATP binding also regulate rewinding kinetics following helicase stalling. Our findings reveal that ZBD and C-terminal tail play a major role in regulation of Twinkle´s real-time kinetics. Their interplay constitutes an auto-regulatory mechanism that may be relevant for coordinating the mtDNA maintenance activities of the helicase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c4e9b3c40e208324f97a7ecc98eca6a14e41660" target='_blank'>
              Autoregulation of the real-time kinetics of the human mitochondrial replicative helicase
              </a>
            </td>
          <td>
            Ismael Plaza-G A, María Ortiz-Rodríguez, Seth P Buchanan, Samuel Miguez-Amil, Kateryna M. Lemishko, F. Moreno-Herrero, R. Fernández-Leiro, Grzegorz L. Ciesielski, Borja Ibarra
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Small Ubiquitin-like Modifier 1 Pseudogene 3 (SUMO1P3) is a novel long non-coding RNA (lncRNA) located at the 1q23.2 locus of the human chromosome. Recent evidence indicates that SUMO1P3 is aberrantly upregulated in nine types of human cancer and functions as an oncogene. Elevated SUMO1P3 expression is strongly associated with unfavorable clinicopathological features and poor prognosis in eight cancer types. Mechanistically, SUMO1P3 functions as a miRNA sponge, an epigenetic regulator, and directly interacting with proteins. It activates key signaling pathways, such as the Wnt/β-catenin and AKT pathways, and regulates Epithelial- Mesenchymal Transition (EMT), which facilitates cancer progression and therapy resistance. Due to its diverse functional roles, SUMO1P3 emerges as a promising diagnostic and prognostic biomarker, as well as a potential therapeutic target in precision oncology. This review provides a comprehensive summary of current research on SUMO1P3, highlighting its regulatory mechanisms, biological functions, and clinical significance in cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4f3e3fbd6a4fd472988ea5e4c6612a35b0592f" target='_blank'>
              The Vital Role of Long Non-Coding RNA SUMO1P3 in the Regulation of Human Cancer: Current Perspectives and Future Challenges.
              </a>
            </td>
          <td>
            Jingjie Yang, Yuzhang Wei, Chengran Gao, Zihang Wang, Yulong Liu, Haodong He, Hao Zhou, Guihua Liao, Gang Zhou, Chengfu Yuan
          </td>
          <td>2025-07-29</td>
          <td>Current topics in medicinal chemistry</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/760c4eaf87c1e364f138dd2080aea7e51f03d039" target='_blank'>
              Circular RNAs: A Potential Regulator in Environmental Chemical Carcinogenesis.
              </a>
            </td>
          <td>
            Huijun Huang, Yiqi Zhou, Yiyan Huang, Jiaxin Wang, Shiyi Ouyang, Meiqi Lan, Lieyang Fan, Yun Zhou
          </td>
          <td>2025-07-17</td>
          <td>Chemical research in toxicology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="SUMMARY The ring-shaped sliding clamp PCNA enables DNA polymerases to perform processive DNA synthesis during replication and repair. The loading of PCNA onto DNA is catalyzed by the ATPase clamp loader RFC. Using a single-molecule platform to visualize the dynamic interplay between PCNA and RFC on DNA, we unexpectedly discovered that RFC continues to associate with PCNA after loading, contrary to the conventional view. Functionally, this clamp-loader/clamp complex is required for processive DNA synthesis by polymerase δ (Polδ), as the PCNA-Polδ assembly is inherently unstable. This architectural role of RFC is dependent on the BRCT domain of Rfc1, and mutation of its DNA-binding residues causes sensitivity to DNA damage in vivo. We further showed the FEN1 flap endonuclease can also stabilize the PCNA-Polδ interaction and mediate robust synthesis. Overall, our work revealed that, beyond their canonical enzymatic functions, PCNA-binding proteins harbor non-catalytic functions essential for DNA replication and genome maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00baaea7b1484df608f6f240f34fa8c252f35c53" target='_blank'>
              A non-catalytic role for RFC in PCNA-mediated processive DNA synthesis
              </a>
            </td>
          <td>
            Gabriella N. L. Chua, Emily C. Beckwitt, Victoria Miller-Browne, O. Yurieva, Dan Zhang, Bryce Katch, John W Watters, Kaitlin Abrantes, Ryogo Funabiki, Xiaolan Zhao, Michael E. O’Donnell, Shixin Liu
          </td>
          <td>2025-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Introduction Retrotransposons (REs) constitute nearly half of the genome and include long terminal repeat (LTR) elements, Long INterspersed Elements (LINE), and Short INterspersed Elements (SINE). REs are typically silenced in somatic tissues via DNA methylation but can be reactivated through DNA hypomethylation, potentially impacting gene regulation. Here, we investigate genome-scale profiles of RE methylation in high-grade serous ovarian carcinoma (HGSOC) and associations with survival among Black women. Methods Methylation levels of LTR, LINE-1, and Alu (type of SINE) in 200 HGSOC tumors were predicted using a random forest approach and clustered using multiple consensus algorithms. Associations between RE methylation clusters and survival were evaluated using Cox proportional hazard regression, adjusting for age, stage, and debulking status. We performed sensitivity analyses restricted to women with late-stage disease and with adjustment for BRCA1/BRCA2 mutations. Results Two RE methylation clusters were identified. Cluster 1 exhibited a more hypomethylated RE profile (“Active”), while Cluster 2 was more hypermethylated (“Repressed”). No statistically significant differences in patient or clinical characteristics were observed between clusters. Compared to the Active Cluster, the Repressed Cluster was associated with an increased risk of mortality (HR = 2.41; 95% CI 1.04–5.59) and had a lower proportion of T cells. This association was consistent in sensitivity analyses. Conclusion A more hypermethylated RE profile was linked to worse survival among Black women with HGSOC, highlighting the potential of RE methylation as a prognostic biomarker. Further research is needed to understand the underlying biological mechanisms and their implications in ovarian cancer biology and treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01942-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bc642af0cdc56a04adbb30045ba5a19d537e41" target='_blank'>
              Retrotransposon methylation profiles and survival in Black women with high-grade serous ovarian carcinoma
              </a>
            </td>
          <td>
            Christelle M. Colin-Leitzinger, Katherine A Lawson-Michod, Courtney E Johnson, Irma M Vlasac, Sean Yoder, Tania Mesa, Dana Roeber, Chad Huff, M. Hildebrandt, Kristen Haller, Anthony J Alberg, E. Bandera, Melissa Bondy, Michele L Cote, Theresa A Hastert, Edward S Peters, Paul D Terry, Andrew B. Lawson, Andy Berchuck, Brooke L Fridley, J. Chern, J. Doherty, Jeffrey R Marks, J. Schildkraut, Brock C. Christensen, Lucas A Salas, Lauren C. Peres
          </td>
          <td>2025-07-30</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Fanconi anemia (FA) is a rare genetic disorder caused by defects in the repair of DNA interstrand crosslinks (ICLs)—highly toxic lesions that impede essential processes like DNA replication and transcription, leading to severe genome instability. Clinically, FA presents with a broad spectrum of symptoms, including progressive bone marrow failure, congenital abnormalities, and an elevated predisposition to various malignancies, particularly acute myeloid leukemia and squamous cell carcinomas. This review provides a comprehensive overview of both the endogenous and exogenous sources of ICLs and the DNA repair pathways responsible for their resolution, with a primary focus on the FA pathway. We also discuss the tumorigenic consequences of FA pathway deficiencies, highlighting the molecular mechanisms that contribute to the heightened cancer risk observed in FA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6689c7c75058ca5686e353d84e63237862e17744" target='_blank'>
              Comprehensive review on Fanconi anemia: insights into DNA interstrand cross-links, repair pathways, and associated tumors
              </a>
            </td>
          <td>
            Chenyan Fang, Zhoujun Zhu, Jun Cao, Jun Huang, Yipeng Xu
          </td>
          <td>2025-07-30</td>
          <td>Orphanet Journal of Rare Diseases</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Brain tumors are one of the most severe types of malignant tumors and glioma accounts for ~80% of malignant brain tumors. The current treatment methods for glioma are limited and patients with glioma often experience relapse following treatment, which leads to a poor prognosis for these patients. Therefore, novel therapeutic targets and methods urgently need to be explored. The present review screened studies that mainly focused on the epigenetic regulation of small guanosine triphosphate (GTP)ase in glioma. These small GTPases participate in most cellular biological processes, including differentiation, proliferation, cell migration, apoptosis, vesicle and organelle dynamics and transport, nuclear dynamics and cytoskeleton regulation. Due to the diversity and importance of the biological functions of small GTPases, an increasing number of studies have focused on them; however, the incidence of changes in the gene structure of small GTPases is considered to be low in glioma. Several studies have shown that the abnormal expression of genes encoding small GTPases is often influenced by epigenetic regulation in glioma. Epigenetic regulation is a dynamic and reversible process, which implies that the reversal of abnormal epigenetic modifications is a potential treatment strategy for glioma. These previous studies, which are summarized in the present review, not only provide new therapeutic targets and prognostic markers, but also provide information regarding the treatment of glioma. The current review may provide valuable insights for future research and promote the clinical translation of relevant research results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44b0933c82de7174662d27dd036b6327ca1cf162" target='_blank'>
              The role of abnormal epigenetic regulation of small GTPases in glioma (Review)
              </a>
            </td>
          <td>
            Mingyang Zhang, Yimin Huang, Qiang Zhang, Xiaoyan Zhang, Lumei Kang, Jianguo Wang
          </td>
          <td>2025-07-02</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d635136f6062b087a4cb1e8318c15fddd1b00672" target='_blank'>
              Rewiring DNA repair with PARP-based chemical inducers of proximity.
              </a>
            </td>
          <td>
            Bryce da Camara, Eric M. Bilotta, Erin F Broderick, Ashwini Premashankar, Paige A. Barta, Trever R. Carter, Lauren M. Hargis, D. Durocher, Michael A. Erb
          </td>
          <td>2025-07-31</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Previously, we identified that mutant p53 expression in cancer cells promotes engulfment of neighbouring cancer cells to form cell-in-cell (CIC) structures. This process gave mutant p53 cells an advantage in tumour formation in mouse xenograft experiments. TP53 can be found mutated at nearly every amino acid in cancers and mutant p53 expression is associated with various GOF (Gain-of-function) processes, including cancer cell invasion, metastasis, stemness and drug resistance. In the current manuscript, we identified SH3BGRL (Src homology 3 binding glutamate rich protein like) as a mutant p53-regulated gene and investigated to what extent SH3BGRL expression and cell engulfment are responsible for mutant p53-dependent anchorage-independent growth and chemoresistance. We demonstrate that mutant p53 expression drives cell engulfment in which the mutant p53 host cell moves in the direction of the target internal cell to form CIC structures. This is therefore more reminiscent of cell engulfment rather than cell entosis, in which cells invade into host cells. Using NGS (Next Generation Sequencing), we identified novel target genes of mutant p53 and demonstrate that cell engulfment requires SH3BGRL expression. We generated mutant p53 and p53 KO cell lines that stably overexpressed SH3BGRL and determined that SH3BGRL promotes etoposide resistance in mutant p53 cells and anchorage-independent growth independent of mutant p53 expression. Through FACS sorting of pure cell engulfing (CIC) populations, we could also show that engulfing cells have an enhanced etoposide resistance. These data suggest that SH3BGRL and cell engulfment are required for certain GOFs of mutant p53.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba6ff758bccb84d82f6118096ce3f5daf639f6e" target='_blank'>
              Mutant p53 induces SH3BGRL expression to promote cell engulfment
              </a>
            </td>
          <td>
            Lobsang Dolma, Mary I Patterson, Antonia Banyard, C. Hall, Steven J Bell, Wolfgang Breitwieser, Sudhakar Sahoo, John Weightman, Maria Pazos Gil, Garry Ashton, C. Behan, Nicola Fullard, Lewis D Williams, P. Muller
          </td>
          <td>2025-07-01</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="X chromosome inactivation (XCI) is crucial for balancing X-linked gene dosage in female cells by randomly silencing one X chromosome during early embryogenesis. However, accurately classifying cells based on the parental origin of the inactivated X chromosome in single-cell samples remains challenging. Here we present FemXpress, a computational tool leveraging X-linked single nucleotide polymorphisms (SNPs) to group cells based on the origin of the inactivated X chromosome in female single-cell RNA sequencing (scRNA-Seq) data. FemXpress performs robustly on both simulated and real datasets, without requiring parental genomic information, and can also identify genes that escape XCI. Applying FemXpress to single-cell RNA-Seq data from multiple tissues of a cynomolgus monkey, we reveal heterogeneity in XCI origin across organs and cell types. In each organ, we identify candidate XCI-escaping genes, and within each cell type, we observe gene expression differences associated with XCI origin, potentially contributing to phenotypic variability. Furthermore, FemXpress demonstrated strong performance in phasing XCI in scRNA-Seq datasets from embryos and colon tumors. In summary, FemXpress provides a powerful approach for XCI status analysis, offering new insights into XCI dynamics at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/478a669d81b3cf05029932ee293479ee1f9bcebc" target='_blank'>
              FemXpress: Systematic Analysis of X Chromosome Inactivation Heterogeneity in Female Single-Cell RNA-Seq Samples.
              </a>
            </td>
          <td>
            Xin Wang, Yingke Ma, Fan Li, Wentao Cui, Tianshi Pan, Siqi Wang, Sinan Ma, Qingtong Shan, Chao Liu, Yukai Wang, Ying Zhang, Yuanchun Zhou, Wei Li, Pengfei Wang, Qi Zhou, Guihai Feng
          </td>
          <td>2025-06-29</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="DNA methylation (5-methylcytosine, 5mC) represents the most prevalent modification in mammals, which is closely linked to disease pathogenesis and cancer development. Single-base resolution sequencing and quantitative analysis of 5mC are essential for elucidating its biological functions. However, current methods are still limited by resolution, sequencing bias, and false positives. In this study, we engineered a Naegleria TET-like protein (NTET), yielding a recombinant engineered NTET (eNTET), to improve both its oxidation activity and sequence compatibility for 5mC. Combined with pyridine borane reduction, we developed engineered NTET-assisted pyridine borane sequencing (eNAPS) to quantitatively detect 5mC in DNA at single-base resolution. In eNAPS, 5mC is oxidized by eNTET to 5-formylcytosine (5fC) and 5-carboxylcytosine (5caC), which are further reduced to dihydrouracil (DHU) by pyridine borane and read as thymine (T) in the subsequent sequencing. The direct conversion of 5mC-to-T allows for precise mapping of 5mC in DNA at single-base resolution. Compared with conventional bisulfite sequencing (BS-seq), eNAPS exhibits advantages such as non-destruction, enhanced sensitivity, improved accuracy, and greater efficiency. Using the eNAPS method, we achieved quantitative analysis of 5mC at single-base resolution in genomic DNA of lung tumor and tumor-adjacent normal tissues. Overall, eNAPS is a mild and bisulfite-free method with high accuracy, making it a valuable tool for investigating the dynamic interplay of 5mC in epigenetic regulation and disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b65e0ea364960e8375b9f55d33c362f5bb33bc66" target='_blank'>
              Direct sequencing of DNA 5-methylcytosine by engineered dioxygenase NTET-assisted eNAPS.
              </a>
            </td>
          <td>
            Shan Zhang, Neng-Bin Xie, Li Zeng, Fang-Yin Gang, Yao-Hua Gu, Min Wang, Xia Guo, T. Ji, Jun Xiong, Bifeng Yuan
          </td>
          <td>2025-07-11</td>
          <td>Chemical science</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background SUGT1 (Suppressor of the G2 allele of SKP1) and FH (fumarate hydratase) have recently garnered significant attention from the research community. SUGT1 functions as a molecular chaperone, regulating the stability and activity of various proteins, while FH is a key enzyme in the tricarboxylic acid cycle, catalyzing the reversible conversion of fumarate to malate. Existing literature has established their essential roles in signaling, tumorigenesis, and cancer progression. However, their functions and mechanisms in ovarian cancer (OC) remain poorly understood. Results We found that high SUGT1 expression is associated with a more advanced FIGO stage in OC. SUGT1 knockdown significantly inhibits OC cell proliferation and metastasis, while its overexpression has the opposite oncogenic effect. Mechanistically, we revealed that SUGT1 promotes FH protein degradation via the ubiquitin-proteasome pathway. Moreover, FH knockdown partly reversed the inhibitory effects of SUGT1 knockdown on tumor cell proliferation, migration, and proteins of phosphorylated PI3K/AKT and Vimentin. In summary, We demonstrated that SUGT1 exerts oncogenic functions in OC by regulating FH stability. Conclusions Our study is the first to provide experimental evidence elucidating the SUGT1-FH relation and its role in OC progression, offering potential significance for clinical diagnosis and therapy. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01744-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/467d1c72a28cbccbc2152c3f132424b0a5db294b" target='_blank'>
              Mechanism by which SUGT1 downregulates FH to promote proliferation and migration in serous ovarian cancer
              </a>
            </td>
          <td>
            Tianli Mu, Bo Ren, Ziteng Kuang, Runze He, Bingjie Rui, Ye Yang, Yuxi Liu, Danbo Geng, Yuci Zhang, Min Wang
          </td>
          <td>2025-07-29</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Decitabine and 5-azacytidine were approved for the treatment of patients with acute myeloid leukemia. Their main mechanism of action is considered to inhibit DNA methyltransferase, restoring expression of tumor suppressor genes. However, demethylating agents activate undesirable transcription from genome non-coding regions, including LINEs retrotransposons. Active LINE1retrotransposition is accompanied by DNA damage, genomic rearrangements and, as a result, rapid relapse of the disease. To minimize undesirable consequences, we proposed to use demethylating agents in combination with nucleoside inhibitors of viral reverse transcriptase (RTIs) actively applied in antiretroviral therapy. We studied effects of reverse transcriptase inhibitors on the level of DNA damage caused by active retrotransposition of LINE1 under the influence of hypomethylating agents decitabine and 5-azacytidine using THP-1 cells. DNA damage levels were assessed by DNA comet assay in accordance with OECD recommendations and by counting the number of γ-H2AX-foci using fluorescent microscopy. We demonstrated that in THP1 cells treated with 5-azacytidine and decitabine at IC20 concentrations statistically significant increase in the value of the “comet tail moment” relatively to untreated cells. In particular, decitabine caused 2-fold increase and 5-azacytidine - 1.7-fold increase. In THP-1 cells treated with demethylating agents we also observed statistically significant increases of the number of γ-H2AX foci. RTIs do not change significantly the level of DNA damage. RTIs decreased undesirable DNA damage caused by demethylating agents when the drugs were used in combinations. According to comet assay results and γ-H2AX-foci numbers statistically significant decrease up to the control level was observed for the combination of decitabine (IC20) + RTIs (lamivudine 20 μM/zidovudine 20 μM/efavirenz 20 μM) and for the combination of 5-azacytidine (IC20) + RTIs. Thus, we demonstrate that application of RTIs in demethylating agent therapy is promising. This work was supported by the grant of the Russian Science Foundation 23-25-00276.



 Strelkov M.S., Vlasova O.A., Antonova I. A., Magomedova Kh.M., Kirsanov K.I., Yakubovskaya M.G.. Reverse Transcriptase Inhibitors Decrease Adverse DNA Damage Caused By Anticancer Demethylating Drugs [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P70.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1d96b67cdd7843762b27866b119f5cac02e056a" target='_blank'>
              Abstract P70: Reverse Transcriptase Inhibitors Decrease Adverse DNA Damage Caused By Anticancer Demethylating Drugs
              </a>
            </td>
          <td>
            M. S. Strelkov, O. Vlasova, I. Antonova, Kh.M. Magomedova, K. Kirsanov, M. Yakubovskaya
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT DNA replication is regulated by factors that promote or inhibit initiation. In Bacillus subtilis, YabA is a negative regulator of replication initiation, while the newly identified kinase CcrZ is a positive regulator. The consequences of under-initiation or over-initiation of replication on genome stability remain unclear. In this work, we measure the origin-to-terminus ratio as a proxy for replication initiation activity. We show that ΔccrZ and several ccrZ alleles under-initiate DNA replication, while ablation of yabA or overproduction of CcrZ leads to over-initiation. We find that cells under-initiating DNA replication have few incidents of replication fork stress as determined by the low frequency formation of RecA-GFP foci compared with wild type. In contrast, cells over-initiating replication show levels of RecA-GFP foci formation analogous to cells directly challenged with DNA-damaging agents. We show that cells under-initiating and over-initiating DNA replication are both sensitive to mitomycin C, demonstrating that changes in replication initiation frequency cause an increase in sensitivity to genotoxic stress. With these results, we propose that cells under-initiating DNA replication are sensitive to DNA damage due to asynchronous DNA replication, leading to inefficient homologous recombination. In cells over-initiating replication, we propose that an increase in the number of replication forks leads to replication fork stress, which is further exacerbated by chromosomal DNA damage. Together, our study shows that DNA replication initiation frequency must be tightly controlled because changes in initiation influence replication fork fate and the capacity of cells to efficiently repair damage to their genetic material. IMPORTANCE The regulation of DNA replication is fundamental to cell growth and cell cycle control. In eukaryotes, under-initiation or over-initiation leads to genome instability. In bacteria, it is unclear how changes in replication initiation frequency impact DNA replication status and genome integrity. We show that tight regulation of DNA replication initiation is critical for maintaining genome integrity. Cells over-initiating or under-initiating DNA replication are sensitive to DNA damage. Furthermore, cells over-initiating DNA replication experience replication fork stress at levels that phenocopy those observed in cells challenged with DNA damage from mitomycin C. Our results establish the critical importance of properly regulating DNA replication initiation frequency because an imbalance in initiation results in replication fork perturbations, deficiencies in DNA repair, and genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4a27c31f5c5a118965016651f3b5fe82634d9fb" target='_blank'>
              DNA replication initiation timing is important for maintaining genome integrity
              </a>
            </td>
          <td>
            Tristan T Reed, Abigail H Kendal, Katherine J. Wozniak, Lyle A. Simmons
          </td>
          <td>2025-07-21</td>
          <td>Journal of Bacteriology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Poly(ADP-ribosyl)ation is a crucial posttranslational modification that governs gene expression, chromatin remodeling, and cellular homeostasis. This dynamic process is mediated by the opposing activities of poly(ADP-ribose) polymerases (PARPs), which synthesize poly(ADP-ribose) (pADPr), and poly(ADP-ribose) glycohydrolase (PARG), which degrades it. While PARP function has been extensively studied, the structural and mechanistic basis of PARG-mediated pADPr degradation remain incompletely understood. To investigate the role of PARG in pADPr metabolism, we employed CRISPR/Cas9-based genome editing to generate a novel Parg29b mutant mouse embryonic stem cell (ESC) line carrying a precise deletion within the PARG catalytic domain. This deletion completely abolished pADPr hydrolytic activity, resulting in massive nuclear pADPr accumulation, yet ESC viability, proliferation, and cell cycle progression remained unaffected. Using Drosophila melanogaster as a model system, we demonstrated that this mutation completely disrupted the pADPr pathway and halted developmental progression, highlighting the essential role of PARG and pADPr turnover in organismal development. Our results define a critical structural determinant of PARG catalytic function, underscore the distinct requirements for pADPr metabolism in cellular versus developmental contexts, and provide a genetically tractable model for studying the regulation of poly(ADP-ribose) dynamics and therapeutic responses to PARP inhibition in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9e822c00a22c15cec5b6420b563ebd7543bc810" target='_blank'>
              PARG Mutation Uncovers Critical Structural Determinant for Poly(ADP-Ribose) Hydrolysis and Chromatin Regulation in Embryonic Stem Cells
              </a>
            </td>
          <td>
            Yaroslava Karpova, Sara Piatz, Guillaume Bordet, A. Tulin
          </td>
          <td>2025-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSC) accounts for more than 200,000 deaths each year. Despite recent advances in treating HGSC with neoadjuvant chemotherapy, the majority of patients ultimately develop chemotherapy resistance. HGSC is characterized by TP53 mutations and widespread copy number alterations and occurs frequently in the setting of deleterious germline BRCA1/2 variations, but many cases lack putative driver mutations. Here, we performed whole-exome, whole-genome, and whole transcriptome sequencing along with mass spectrometry to characterize the molecular landscape of HGSC in 22 paired samples obtained before and after neoadjuvant chemotherapy. Responsiveness to chemotherapy was determined for each patient. Evidence at the DNA, RNA, and protein level revealed numerous defects in cell-cell and cell-matrix interactions as well as disruption of cell polarity and cytoskeletal regulation in HGSC, indicating that defects in epithelial integrity were present in the majority of HGSC patients. Non-responsive HGSC harbored subclones with putative survival mutations. Additionally, ineffective nonsense mediated decay resulted in the persistence of transcripts with frameshift mutations that were translated into aberrant proteins detectable in HGSC samples. Together, these findings suggest that HGSC may arise through defects in maintenance of epithelial integrity that lead to shedding of malignant cells throughout the peritoneum, and the presence of resistant subclones prior to chemotherapy may decrease the chemosensitivity of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/896a702c0b394e4d17c91e5aa33566545b16c4dc" target='_blank'>
              Multi-omic Characterization of Pre- and Post-neoadjuvant Chemotherapy Treated Ovarian Cancer Reveals Mediators of Tumorigenesis and Chemotherapy Response.
              </a>
            </td>
          <td>
            Mark C Valentine, Mary M Mullen, E. Kotnik, Matthew A Powell, Premal H Thaker, C. McCourt, L. Kuroki, Andrea R Hagemann, Rebecca C Arend, A. Holehouse, R. Mitra, D. Mutch, Katherine C Fuh
          </td>
          <td>2025-07-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1858682f6d07c026bc78fe000bacfa219ec68e47" target='_blank'>
              Unexplored diversity and potential functions of extra-chromosomal elements.
              </a>
            </td>
          <td>
            Haoyu Liu, Jia Sun, JuanJuan Si, Yi Liao, Jiaqi Bai, Xia Li, Limin Wang, Kuojun Cai, Wei Ni, Ping Zhou, Shengwei Hu
          </td>
          <td>2025-08-19</td>
          <td>mSystems</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cellular senescence is a double-edged sword in cancer biology, functioning as both a tumor-suppressive mechanism and a driver of malignancy. Initially, senescence acts as a protective barrier by arresting the proliferation of damaged or oncogene-expressing cells via pathways such as oncogene-induced senescence and the DNA damage response. However, persistent senescence-associated secretory phenotype and metabolic reprogramming in senescent cells create a pro-inflammatory, immunosuppressive tumor microenvironment, fueling cancer progression, therapy resistance, and metastasis. This comprehensive review systematically examines the molecular mechanisms of senescence across diverse cancers, spanning digestive, reproductive, urinary, respiratory, nervous, hematologic, endocrine, and integumentary systems, and elucidates its context-dependent roles in tumor suppression and promotion. We highlight groundbreaking therapeutic innovations, including precision senolytics, senomorphics, and combinatorial strategies integrating immunotherapy, metabolic interventions, and epigenetic modulators. The review also addresses microenvironment remodeling and cutting-edge technologies for dissecting senescence heterogeneity, epigenetic clocks for biological age prediction, and microbiome engineering to modulate senescence. Despite their promise, challenges such as off-target effects, biomarker limitations, and cellular heterogeneity underscore the need for precision medicine approaches. Finally, we propose future directions to harness senescence as a dynamic therapeutic target, offering transformative potential for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee9e8f75a567edbd4415256ca44b46f90ddbb5c2" target='_blank'>
              Cellular senescence in cancer: from mechanism paradoxes to precision therapeutics
              </a>
            </td>
          <td>
            Tiejun Feng, Fuda Xie, Leo M.Y. Lee, Zhiqiang Lin, Yifan Tu, Yang Lyu, Peiyao Yu, Jialin Wu, Bonan Chen, Ge Zhang, Gary M K Tse, K. To, Wei Kang
          </td>
          <td>2025-08-08</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Circular RNAs (circRNAs) are a class of covalently closed non-coding RNAs that regulate the progression of multiple cancers. Recent studies have revealed the presence of several post-transcriptional modifications such as N6-methyladenosine (m6A), N1-methyladenosine (m1A), 5-methylcytosine (m5C), N4-acetylcytidine (ac4C), and N7-methylguanosine (m7G) on circRNAs. This review synthesizes the results of articles retrieved through systematic searches of PubMed and Web of science databases, mainly focusing on circRNA modifications in cancers. These modifications affect the biogenesis, metabolism, stability, and function of circRNAs in cancers, highlighting the critical roles of circRNAs modifications in cancers. Moreover, circRNAs crosstalk with epigenetic modifications of mRNA across various cancers, offering new perspectives for cancer therapy. Innovations in detection techniques, such as anti-modifying antibodies and mass spectrometry, have improved the identification and sensitivity of epigenetically modified circRNAs. Emerging technologies including artificial intelligence (AI)-based bioinformatics algorithms will accelerate the development of RNA epigenetic modifications-based precision therapies. Herein, we summarize a range of epigenetically modified circRNAs and their future research directions. We hope to develop clinical protocols that targeting circRNAs modification for the treatment of refractory malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edebcdf6176f17b4538381fc1ab3dca2e4573291" target='_blank'>
              Dynamic modifications of circular RNAs drive oncogenesis.
              </a>
            </td>
          <td>
            Jiaojiao Dai, Zhe Wang, Xiaoxun Cheng, Zhengze Hu, Jinghan Hua
          </td>
          <td>2025-07-15</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ovarian cancer ranks as the eighth most common malignancy in women globally. Early detection remains a critical challenge for improving ovarian cancer diagnosis and treatment outcomes. Our prior study demonstrated that ARHGAP40 downregulation in basal cell carcinoma is attributed to CpG island hypermethylation. However, the expression and methylation status of ARHGAP40 in ovarian cancer remain unexplored. Here, we investigated ARHGAP40 protein expression in normal fallopian tubes, ovarian benign tumors, borderline tumors, low-grade serous carcinoma (LGSC) and high-grade serous carcinomas (HGSC). Methylation analysis of the ARHGAP40 was performed using bisulfite sequencing PCR (BSP). MethyLight assays were developed to detect methylated circulating tumor DNA (ctDNA) fragments of ARHGAP40. IHC results revealed absent or weak ARHGAP40 protein expression in 93.8% (30/32) of HGSC, 11.1% (2/18) of borderline tumors, and in 14.3% (1/7) of LGSC cases. ARHGAP40 protein expression was robust expressed in all normal fallopian tubes (15/15) and benign ovarian tumors (8/8). CpG island hypermethylation in the ARHGAP40 promoter showed a strong inverse correlation with protein expression (P < 0.001). Furthermore, methylated ctDNA for ARHGAP40 was detected in 80.0% (4/5) of HGSC patients’ plasma, but not in benign tumor (0/3) and healthy controls (0/15). Our findings suggest that promoter hypermethylation may be a mechanism underlying ARHGAP40 silencing in HGSC. Detection of methylated ARHGAP40 ctDNA may serve as a noninvasive biomarker for early diagnosis and monitoring of HGSC. While our findings suggest the potential of methylated ARHGAP40 as an early diagnostic biomarker, the small sample size warrants validation in larger cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a13bc1c0523b1d5d86e6d21e3c7e8792952655" target='_blank'>
              Methylated ARHGAP40 DNA as a potential biomarker for early diagnosis in high-grade ovarian serous cancer
              </a>
            </td>
          <td>
            Jiaxin Chai, Dongni Leng, Shuwei Guo, Rusong Zhang, Jiandong Wang, Yun Gu, Qiu Rao
          </td>
          <td>2025-07-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) represents the most common solid renal malignancy, accounting for 90% of kidney cancers and 3% of all malignancies. Clear cell RCC (ccRCC) is the predominant histologic subtype, comprising roughly 80% of cases. Despite a 5-year survival rate of 75%, up to 30% of patients with localized ccRCC eventually develop recurrence or distant metastases. Mitotic spindle organizing protein 2A (MZT2A), also known as FAM128A, GCP8, or MOZART2A, is located on chromosome 2q21.1 and encodes a 158-amino acid protein first described by Hutchins et al. in 2010. To date, limited studies have linked MZT2A to tumorigenesis, primarily in lung and gastric cancers. The tumor microenvironment (TME), particularly tumor-infiltrating immune cells, plays a critical role in cancer progression and prognosis. Using paired tumor and normal samples from TCGA, we assessed MZT2A expression across multiple cancer types. Collectively, these findings suggest that MZT2A is involved in immune cell infiltration and the tumor microenvironment, may promote KIRC progression, and is overexpressed in cases with poor outcomes, supporting its potential as a diagnostic and prognostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d43bd634f019908094dd923cc5fef3d5a13d2f5" target='_blank'>
              Integrated transcriptomic analysis identifies MZT2A as a prognostic and immune-related biomarker in KIRC.
              </a>
            </td>
          <td>
            Yunjing Zhang, Zekun Xin, Xinnuo Li, Yanmei Chen
          </td>
          <td>2025-07-15</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract DNA supercoiling significantly influences DNA metabolic pathways. To examine its impact on DNA–protein interactions at the single-molecule level, we developed a highly efficient and reliable protocol to modify plasmid DNA at specific sites, allowing us to label plasmids with fluorophores and biotin. We then induced physiological levels of negative or positive supercoiling in these plasmids using gyrase or reverse gyrase, respectively. By comparing supercoiled DNA with relaxed circular DNA, we assessed the effects of supercoiling on CRISPR–Cas9 and the mismatch repair protein MutS. We found that negative DNA supercoiling exacerbates off-target effects in DNA unwinding by Cas9. For MutS, we observed that both negative and positive DNA supercoiling enhance the binding interaction between MutS and a mismatched base pair but do not affect the rate of ATP-induced sliding clamp formation. These findings not only underscore the versatility of our protocol but also open new avenues for exploring the intricate dynamics of protein–DNA interactions under the influence of supercoiling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fce158336529940319e5538b8cb84bb3cbffe58d" target='_blank'>
              A high-throughput single-molecule platform to study DNA supercoiling effect on protein–DNA interactions
              </a>
            </td>
          <td>
            Huijin Lee, Fahad Rashid, Jihee Hwang, James A. London, Richard Fishel, James M. Berger, Sua Myong, Taekjip Ha
          </td>
          <td>2025-06-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) is a common characteristic of human cancers, which occurs most frequently in ovarian and breast cancers. The unique genetic vulnerabilities, coupled with the synthetic lethality effect of Poly-(ADP-ribose) polymerase (PARP) inhibitors (PARPi), provide a promising opportunity for targeting HRD cancers. However, only a few HRD patients benefit from PARPi monotherapy, and it is therefore imperative to explore effective combination therapies for HRD tumors. Growing evidence has underscored the distinct tumor microenvironment (TME) landscape of HRD cancers. Immune activation and immune suppression co-exist in a dynamic balance during the development of HRD cancers. At the late stage, however, negative immune regulation predominates, resulting in the formation of an immunosuppressive microenvironment. This intricate network reprograms cancer biology in multiple aspects and serves as a potential target for cancer treatment. In this review, we briefly outline the current HRD tests and genetic characteristics of HRD cancers, focusing on breast, ovarian, pancreatic, and prostate cancers. We then summarize the interactions and crosstalk between immune cells and cancer cells, as well as various signaling pathways within the TME of HRD cancers. Additionally, we highlight recent advances in combining PARP inhibitors with immunotherapies in preclinical models and clinical trials of HRD cancers. This review provides valuable insights and perspectives into the distinct landscape of TME in HRD cancers, and offers a rationale for expanding the application of this combined therapeutic approach to a broader range of HRD cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/befa4ba7c7bbd20ccb4cc8148dd06bbf9c57c776" target='_blank'>
              The distinct landscape of tumor immune microenvironment in homologous recombination deficient cancers
              </a>
            </td>
          <td>
            Qiuyang Xu, Yuanjia Wen, Taiyuan Huang, Huayi Li, Xingzhe Liu, Shen-nan Shi, Wenjian Gong, Gordon B. Mills, Ding Ma, Qing-cui Gao, Yong Fang
          </td>
          <td>2025-08-20</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background/Objectives: Aberrant expression of high-mobility group protein B1 (HMGB1) has been linked to cancer development and progression. Methods: To better comprehend the role of HMGB1 expression in cancer, a tissue microarray containing 14,966 samples from 134 different tumor entities and 608 samples of 76 different normal tissue types was analyzed by immunohistochemistry. Results: Strong HMGB1 staining occurred in almost all normal cell types and in most cancers. Of 11,808 evaluable cancers, only 7.8% showed complete absence of HMGB1 staining (HMGB1 deficiency) while 9.9% showed 1+, 25.0% showed 2+, and 57.2% showed 3+ HMGB1 positivity. Absence of HMGB1 staining mostly occurred in pheochromocytoma (90.0%), seminoma (72.4%), gastrointestinal stromal tumor (28.6%), adrenal cortical carcinoma (25.0%), and Hodgkin’s lymphoma (25.0%). Low HMGB1 staining was linked to poor histologic grade (p < 0.0001), advanced pT stage (p < 0.0001), high UICC stage (p < 0.0001), and distant metastasis (p = 0.0413) in clear cell renal cell carcinoma, invasive tumor growth in urothelial carcinoma (pTa vs. pT2–4, p < 0.0001), mismatch repair deficiency (p = 0.0167) in colorectal cancers, and advanced pT stage in invasive breast carcinoma of no special type (p = 0.0038). Strong HMGB1 staining was linked to nodal metastases in high-grade serous ovarian carcinomas (p = 0.0213) and colorectal adenocarcinomas (p = 0.0137), as well as to poor histological grade in squamous cell carcinomas (p = 0.0010). Conclusions: HMGB1 deficiency and reduced HMGB1 expression occur in a broad range of different tumor entities. Low rather than strong HMGB1 staining is often linked to an aggressive tumor phenotype. Whether HMGB1 deficiency renders cells susceptible to specific drugs remains to be determined.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b377eef89b94e72a5b4e3f5a4032671c6ed7e3fd" target='_blank'>
              HMGB1 Deficiency Occurs in a Broad Range of Human Cancers and Is Often Associated with Unfavorable Tumor Phenotype
              </a>
            </td>
          <td>
            Viktoria Chirico, Hena Sharifi, M. Tsourlakis, Şeyma Büyücek, Clara Marie von Bargen, Katharina Möller, F. Lutz, Davin Dum, M. Kluth, C. Hube-Magg, Georgia Makrypidi-Fraune, Piero Caneve, M. Lennartz, Morton Freytag, S. Dwertmann Rico, S. Kind, V. Reiswich, E. Burandt, T. Clauditz, P. Lebok, C. Fraune, T. Krech, S. Minner, A. Marx, W. Wilczak, R. Simon, G. Sauter, S. Steurer, K. Jansen
          </td>
          <td>2025-08-01</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6b2afacc712863315fd2993dce1b85afc512471" target='_blank'>
              Long Noncoding RNAs Enforce Pancreatic Cancer Identity and Block Reprogramming
              </a>
            </td>
          <td>
            D. Grygoryev, Seung-Won Lee, Connor Mitchell Frankston, Shauna Rakshe, Mark Berry, Alex Hirano, Taelor Ekstrom, Elise Manalo, Julien Tessier, Marilynn Chow-Castro, Jason Link, D. Keith, Brett C Sheppard, Suzanne Fei, Terry Morgan, Helen E. Remotti, Wenli Yang, Emma Furth, Sudhir Thakurela, Rosalie C. Sears, Jungsun Kim
          </td>
          <td>2025-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Ewing sarcoma is a relatively rare aggressive tumor of bones and soft tissues characterized by specific chromosomal translocations involving genes and transcription factors of the FET and ETS families. A number of studies have demonstrated the presence of molecular genetic events preceding these specific rearrangements.The article presents a clinical observation of the treatment of a 24-year-old patient with Ewing sarcoma of the X rib and CHEK2-associated cancer predisposition syndrome with an oncologically burdened family history, including Ewing sarcoma in a first-degree relative. Specific translocation of the EWSR1 gene (22q12) was identified, characteristic of tumors of the Ewing sarcoma/PNET (primitive neuroectodermal tumors) family, as well as 59 variants of different functional significance in suppressor genes and driver genes was identified in the course of a comprehensive molecular genetic study using fluorescence in situ hybridization (FISH) and next-generation sequencing (NGS) with a targeted custom panel including 415 genes involved in carcinogenesis. The patient underwent combined treatment in the volume of neoadjuvant polychemotherapy, surgical stage (extirpation of the X rib with plastic surgery) and subsequent adjuvant polychemotherapy. During dynamic observation for the period 2022–2025, there were no signs of progression and metastatic process. Molecular genetic profiling in Ewing sarcoma has identified markers that may act as risk modifiers for disease development and progression, determine sensitivity/resistance to standard treatment methods, and serve as potential targets for personalized treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89b306fbc45bbb4cc8b3926afaa4d96e92f8aed8" target='_blank'>
              Ewing sarcoma and CHEK2-related tumor predisposition syndrome (case report and review)
              </a>
            </td>
          <td>
            L. N. Lyubchenko, K. M. Chernavina, A. I. Senderovich, A. Kolomeytseva, M. Sedova, V. V. Chernyshov, A. Fedenko
          </td>
          <td>2025-07-25</td>
          <td>Bone and soft tissue sarcomas, tumors of the skin</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Non-coding RNAs (ncRNAs) in human cells constitute a substantial portion of the transcriptome but do not lead to protein synthesis. Among them, long non-coding RNAs (lncRNAs, > 200 nucleotides long) are fascinating in their ability to orchestrate critical cellular functions that govern cell development, differentiation, and metabolism. Therefore, the dysregulation of lncRNAs has been linked with several diseases, chiefly cancers. Results We focused here on colorectal cancer (CRC), the second-highest cause of mortalities related to cancer worldwide, and more particularly on three lncRNAs, i.e., LINC01589, MELTF-AS1, and UXT-AS1, known to be dysregulated in CRC. We identified a vulnerability in these lncRNAs that could be exploited from a therapeutic point of view: a part of their sequence folds into a secondary structure referred to as G-quadruplex (G4), which is suspected to play active roles in the lncRNA functions. We demonstrate here that these sequences do fold into G4s both in vitro and in CRC cells, and that these G4s can be modulated using PhpC, a prototype molecule for destabilizing G4s. Conclusion We describe an innovative anticancer strategy that fully abides by the rules of chemical biology. We indeed modulate the formation of G4s in cells using ad hoc molecular tools in the aim of disturbing the homeostasis and inner functioning of lncRNAs. By exploiting cellular outcomes, we infer how this pharmacomodulation affects CRC biology and, beyond this, the fate of CRC cells owing to the flawed repertoire of correction and/or compensatory mechanisms in cancer cells. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02322-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/754c36f086e107d8742747ebe134ba178418b705" target='_blank'>
              Pharmacomodulation of G-quadruplexes in long non-coding RNAs dysregulated in colorectal cancer
              </a>
            </td>
          <td>
            Shubham Sharma, Jérémie Mitteaux, A. Pipier, Marc Pirrotta, Marie-José Penouilh, D. Monchaud, Bhaskar Datta
          </td>
          <td>2025-08-08</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d4cff3ee3c7d90bb73c3a8d68c2134c4b228a48" target='_blank'>
              SOX2 is a dispensable modulator of NUT carcinoma oncogenesis in mouse.
              </a>
            </td>
          <td>
            Chenxiang Luo, Dejin Zheng, Ahmed Elnegiry, Sheetal Bhatara, G. Mias, M. Tsoi, Jiyang Yu, Bin Gu
          </td>
          <td>2025-07-14</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Nuclear morphology is a defining cellular feature, differing based on cell type, tissue type, and species. In healthy cells, nuclear morphology is generally tightly regulated and maintained; however, dynamic changes in nuclear morphology are observed under certain conditions, for instance in early embryos and in some immune cells. Deviations in normal nuclear morphology are linked to numerous diseases, including most cancers and premature aging syndromes. Many regulators of nuclear morphology have been identified, encompassing both intranuclear, cytoplasmic, and extracellular factors. Of note, recent studies have converged on chromatin and chromatin-associated proteins as key determinants of nuclear morphology and dynamics. In this review we discuss how the chromatin landscape regulates nuclear morphology in both normal and diseased cellular states. Additionally, we highlight emerging technologies that promise to bridge critical gaps in our understanding of nuclear morphology, including new approaches to probe nuclear structure and the use of synthetic cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/adc2c44f577e2c2679b995d4aca9623ddc124e59" target='_blank'>
              How the chromatin landscape influences nuclear morphology
              </a>
            </td>
          <td>
            Sourabh Sengupta, Haritha Prabha, Daniel L. Levy
          </td>
          <td>2025-07-03</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Clinical outcomes can be linked to the fundamental role played by the regulatory 3D architecture of the genome in a living cell and the unique properties offered as a molecular biomarker modality. Here, we describe how this molecular phenomenon acts as an imprint of the cellular genetic, epigenetic and metabolic status which form discriminatory stable conditional structures that inform on the state of the cell and its phenotype. Consequently, 3D architecture, as a biomarker, offers a more comprehensive translation from molecular readouts to clinical outcomes, with examples and evidence provided by the highly accurate diagnostic and prognostic clinical tests already reduced to practice in the US and UK. Moreover, with support from Large Language Models, we can elucidate the complexities and inter-relationships of molecular pathways and regulation using a 3D genomic architecture knowledge base, to gain insights into various molecular modalities and associated clinical conditions. Abstract Background/Objectives: The 3-dimensional (3D) architecture of the genome in the nucleus of a living cell plays an unexpected yet fundamental regulatory role in cell biology. As an imprint of the cellular genetic, epigenetic and metabolic status, it discriminates pathological conditions through conditional changes to long-range 3D interactions (up to 300 kb) and thus could act as a powerful molecular biomarker linked closely to clinical outcomes. Methods: Here an assessment is made of the latest paradigm shift in molecular biology from a supply chain where information flows from DNA to RNA to protein, to the concept of heritable 3D folding of the genome reflecting the epigenetic and metabolic state of the cell, and which serves as a molecular biomarker for complex clinical outcome. Results: While biomarkers based on individual components of the supply chain fail to accurately reflect clinical outcomes, 3D genomics offers highly informative insights, exemplified for immuno-oncology and prostate cancer diagnosis by clinical tests of superior performance, already in practice in the US and UK. Conclusions: A more complete understanding of human biology will require models that account for the flow of information to and from the 3D genomic architecture in living cells, together with its regulation and logic. Integrating these principles into biomarker discovery and therapeutic design, along with other frontline approaches in precision medicine, including multi-omics and other system-level tools, will be essential for advancing precision medicine beyond its current limitations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afe42115687f08e85dd377e033fb44be5d4f1cc1" target='_blank'>
              Paradigm Lost
              </a>
            </td>
          <td>
            Jane Mellor, E. Hunter, A. Akoulitchev
          </td>
          <td>2025-06-28</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Cancer is an enigma, and different types and subtypes add to the complexity of this disease. Innumerable molecules play important roles in glial pathogenesis. One such molecular component discovered recently and originating from the non-coding region of the human genome is the long non-coding RNA, or lncRNA. Multiple lncRNAs have been shown to play a role in tumor development in different cancer types. Although underappreciated as a therapeutic component, emerging studies have highlighted their crucial contributions to intercellular signaling. Studies suggest that lncRNAs play multiple, diverse roles in cancer cells. Their contribution to disease pathogenesis seems contrasting because the same lncRNA facilitating tumorigenesis in a particular cancer type may play a completely antagonistic role in another. In this review, we discuss multiple lncRNAs and the intracellular targets that are pertinent to glioma pathology, the sensitive expression levels, their impact on the disease progression, the pathways they affect, and their consideration as potential biomarkers for glioma. We further discuss multiple lncRNAs in the most aggressive glioma subtype, glioblastoma. Overall, we provide comprehensive information about key lncRNAs in glioma pathogenesis which could be of future fundamental and therapeutic relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f473a5d748d1ada307293cc0b8a9bbbf79222c54" target='_blank'>
              Long non-coding RNA as modulators of brain pathology: insights into cellular signaling diversity
              </a>
            </td>
          <td>
            Megha Chaudhary, Shreya Maity, Naveen Soni, Shweta Dongre, Ashwani Tiwari, Bhawana Bissa
          </td>
          <td>2025-07-24</td>
          <td>Academia Molecular Biology and Genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7976efe4db0e0c962e5f359566e424689b7f04c0" target='_blank'>
              Epigenetic changes by EZH2 inhibition increase translocations in B cells with high AID activity or DNA repair deficiency.
              </a>
            </td>
          <td>
            Jianli Tao, Luca Alessandri, Alessandro Gasparetto, Lijuan Zhao, Xuefei Zhang, Frederick W Alt, Roberto Chiarle
          </td>
          <td>2025-08-18</td>
          <td>Blood</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Despite significant successes, immunotherapy continues to face numerous challenges in practical applications. CD82 plays an important role in the tumor microenvironment (TME) across various cancers. However, the prognostic value of CD82 in immunotherapy and its role in the immune response remain unclear. The expression levels of CD82 in healthy human tissues were assessed using the human protein atlas (HPA) database tool. The association among CD82 expression levels, copy number variations (CNV), and methylation was explored using the gene set cancer analysis (GSCA) database. Timer tumor immune estimation resource (TIMER) database is used for immune checkpoint and immune infiltration. The LinkedOmincs database is used for enrichment analysis. The statistical inquiry was performed through the R program (version 4.4.1). The result has shown that CD82 mRNA levels were significantly downregulated in most tumor types. Abnormal CD82 mRNA expression was associated with CNV and methylation changes in tumor tissues. Furthermore, several immune checkpoint genes related to CD82 influenced prognosis and were linked to increased immune cell infiltration across various cancer forms, including liver hepatocellular carcinoma (LIHC). The prognosis of the patients under multiple cancer forms, particularly LIHC, was remarkably influenced by CD82, which plays a vital role in modulating immune infiltration. CD82 has the potential to serve as both a prognostic indicator and an immunological biomarker in human cancer. CD82 expression is frequently downregulated across various cancer associated with poor prognosis, influenced by CNV and DNA methylation alterations. Its abnormal expression correlates with changes in immune checkpoint gene activity and enhanced immune cell infiltration, notably in liver. These associations suggest that CD82 plays a significant role in shaping the tumor immune microenvironment and affects patient prognosis. Consequently, CD82 holds promise as both a prognostic biomarker and an immunomodulatory target in cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e6b1867f8cfd33352d9c4d7c23852fa5423b5ed" target='_blank'>
              Prognostic and immunological significance of CD82 in cancers with focus on hepatocellular carcinoma: A bioinformatics approach
              </a>
            </td>
          <td>
            Sikandar Ali, Shiyu Liu, Jiali Chen, Jawad Ullah, Guoyu Mou, Xiaohua Huang
          </td>
          <td>2025-07-25</td>
          <td>Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Despite substantial progress in discovering novel therapeutic targets and compounds, cancer hyperproliferative cells (CHCs) play a central role in driving tumor progression in gastric cancer (GC). High-resolution single-cell RNA sequencing (scRNA-seq) integrated with spatial transcriptomics sequencing (ST-seq) elucidating the role of the CHC microenvironment in driving GC progression remains lacking. Methods Through integrated analysis of 40 scRNA-seq samples and 6 ST samples from human gastric tissues spanning tumor progression, we delineate the phenotypic plasticity of tumor epithelium and characterize the transcriptional trajectory from normal gastric cells to CHCs, a process associated with RRM2 overexpression. Finally, we validated the analysis results through cellular and animal experiments. Results Our findings demonstrate that RRM2 knockdown triggers ferroptosis in GC cells by promoting lipid peroxidation and intracellular iron accumulation. We further screen a compound and RRM2 inhibitor, osalmid, which effectively suppresses tumor growth in GC cell xenografts by triggering ferroptosis, thus offering a new therapeutic option for GC. Conclusions This study elucidates the critical role of RRM2 + CHCs in gastric cancer development and progression, providing novel insights for targeted therapies. The discovery of osalmid as a potential therapeutic agent offers a promising new treatment strategy for GC. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06847-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad62270a5ce806049ffe5312712ee483297a8c1" target='_blank'>
              Integration of scRNA-seq and ST-seq identifies hyperproliferative RRM2+ cells features and therapeutic targets in gastric cancer
              </a>
            </td>
          <td>
            Shuai Ping, Xiong Jia, Yanan Tian
          </td>
          <td>2025-07-15</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fa3ab0bece749f3dd05030992350c29b3fbf920" target='_blank'>
              Targeting transcription in neuroblastoma: focus on the Core regulatory circuit.
              </a>
            </td>
          <td>
            O. Kuchur, S. S. Pogodaeva, Victoria I Zhdankina, Anna V Scherbakova, Alexander A. Shtil, N. V. Antipova
          </td>
          <td>2025-08-11</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer is the primary cause of death worldwide. Cancer occurrence and progression are closely associated with DNA damage repair. DNA glycosylase were the first enzymes to initiate base excision repair, and the Nei-like DNA glycosylase (NEIL) family has attracted increasing attention as an important component of DNA glycosylases. Here, we introduce the role of the NEIL family in the malignant biological behaviors of cancer, including cell proliferation, chemoradiotherapy resistance, invasion and migration, apoptosis, and stemness. Mechanisms affecting the expression of the NEIL protein family range from the transcriptional level and mRNA editing to the level of post-translational modification. Additionally, we emphasize the different functions of the NEIL family in various malignancies and present useful information that supports the hypothesis that the NEIL family could be a potential target in the treatment and diagnosis of various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf9582798727585833402f33b5e71abe53251dc2" target='_blank'>
              Research progress on the role of the NEIL family in cancer
              </a>
            </td>
          <td>
            Yinghan Chen, Muyun Ma, Aixue Zou, Xinjia Wang, Weiwei Dong
          </td>
          <td>2025-07-21</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer is the origin of a novel tissue that attracts resources, spreads beyond boundaries, avoids normal controls, and escapes immunity. How does a novel tissue arise? The puzzle is that two seemingly different processes appear to be the primary driving force. On the one hand, overwhelming evidence links (epi)genetic driver mutations to the origin and progression of tumors. Common oncogenic mutations such as KRAS accelerate cell division, and common knockouts of tumor suppressors such as TP53 abrogate cell death or checks on cell division. On the other hand, cancerous tissues create complex traits that require intricate changes in cells and multiple interactions between different cell types. Such novelty often arises by hijacking the developmental plasticity that normally creates the diverse cells and tissues of our bodies from a single original zygotic cell. How can we reconcile the simple genetic changes in carcinogenesis with the complex developmental plasticity that creates novel tissues? This perspective advocates a new model. (Epi)genetic mutations release developmental plasticity. That developmental plasticity creates novel cellular interactions and complex tissues. Initially, novel traits created by developmental plasticity may not be stably heritable, thus subsequent (epi)genetic changes must stabilize the phenotypic novelty. Recent studies show how classic oncogenic and tumor suppressor driver mutations, such as KRAS and TP53, may primarily act in early carcinogenesis as broad releasers of developmental plasticity rather than as stimulators of cell division or knockout of limitations on cellular clonal expansion. In the new model, genetics releases, plasticity creates, and genetics stabilizes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0abe05e544bd1630685c7bbe6b925595c19894a8" target='_blank'>
              How cancer arises: Genetics releases, plasticity creates, genetics stabilizes
              </a>
            </td>
          <td>
            Steven A Frank
          </td>
          <td>2025-07-31</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background/Objectives: Increasing evidence suggests that lncRNAs are core regulators in the field of tumor progression, with context-specific functions in oncogenic tumorigenesis. LINC01133, a lncRNA that has been identified as both an oncogene and a tumor suppressor, remains largely unexplored in terms of its molecular mechanisms. The purpose of this study was to conduct an in silico analysis, incorporating literature research on various cancer types, to investigate the structural and functional duality of LINC01133. This analysis aimed to identify pathways influenced by LINC01133 and evaluate its mechanism of action as a potential therapeutic target and diagnostic biomarker. Methods: In silico analyses and a narrative review of the literature were performed to predict conserved structural elements, functional internal loops, and overall conservation of the LINC01133 sequence among different vertebrate organisms, summarizing the empirical evidence regarding its roles as a tumor suppressor and tumor-promoting roles in various types of tumors. Results: LINC01133 harbors the evolutionarily conserved structural regions that might allow for binding to relevant driver signaling pathways, substantiating its specific functionality. Its action extends beyond classical tumor mechanisms, affecting proliferation, migration, invasion, and epigenetic pathways in various types of tumors, as indicated by the in silico results and narrative review of the literature we present here. Clinical outcome associations pointed to its potential as a biomarker. Conclusions: The dual character of LINC01133 in tumor biology further demonstrates its prospective therapeutic value, but complete elucidation of its mechanisms of action requires further investigation. This study establishes LINC01133 as a multifaceted lncRNA, supporting context-specific strategies in targeting its pathways, and calls for expanded research to harness its full potential in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f66180b93b3358b4aae4204b9c68eb1c73a7d7f" target='_blank'>
              The Good, the Bad, or Both? Unveiling the Molecular Functions of LINC01133 in Tumors
              </a>
            </td>
          <td>
            Leandro Teodoro Júnior, M.C. Sogayar
          </td>
          <td>2025-07-30</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f9d2ade8ce124c86164da260d86f13c994b3a19" target='_blank'>
              KEAP1 mutations activate the NRF2 pathway to drive cell growth and migration, and attenuate drug response in thyroid cancer.
              </a>
            </td>
          <td>
            Nicholas E Bambach, J. Ricarte-Filho, E. R. Reichenberger, Christian Hernandez-Padilla, K. Hinkle, A. Isaza, Andrew J. Bauer, A. Franco
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72ce4746633f2d4d42f9a7aafe673ea0e10428f4" target='_blank'>
              Genome-wide profiling the integration patterns with T7-PCR.
              </a>
            </td>
          <td>
            Ang Li, Chichen Xiao, Jinwen Wang, Xiao Huang, Xionglei He, Kehui Liu, Fan-Nv Xia
          </td>
          <td>2025-08-01</td>
          <td>The Plant journal : for cell and molecular biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="To what extent overdominance might contribute to the maintenance of genetic diversity within genomes is still an ongoing research question. Pseudo-overdominance created by the complementation of deleterious alleles in heterozygotes has recently become a subject of particular interest. Simulations and theory suggest that pseudo-overdominance may occur in low recombining regions. Here, we conduct a comprehensive investigation of large low-recombining (LLR) regions in cultivated populations of pearl millet, an outcrossing diploid African cereal. We examine seven large regions ranging from 5 to 88 Mb and six of them are pericentromeric. These LLR regions exhibit an excess of heterozygotes, a distinctive hallmark of overdominance. They display a tendency toward a higher diversity and a larger ratio of non-synonymous and deleterious variants. We conduct a more in-depth study of the largest 88 Mb region, identified on chromosome 3. Interestingly, haplotypes of this region have been introgressed from wild relatives. Using long read sequencing, we confirm their strong divergence and the presence of inversions across one of them. One of the haplotypes seems to be highly deleterious in the homozygous state. A total of 17% of the cultivated pearl millet genome exhibit a local population structure suggestive of overdominance or possibly pseudo-overdominance. Our empirical results contribute to the accumulation of knowledge, which will enhance our understanding of the potential role of overdominance or pseudo-overdominance in maintaining genetic diversity, particularly in low recombining regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8563879e9297d9a2f3d59a5ca6775ee20a33deef" target='_blank'>
              Interplay between large low-recombining regions and pseudo-overdominance in a plant genome
              </a>
            </td>
          <td>
            Marine Salson, Maud Duranton, Stella Huynh, C. Mariac, Christine Tranchant-Dubreuil, Julie Orjuela, Phillipe Cubry, Anne-Céline Thuillet, C. Burgarella, Miguel de Navascués, Leïla Zekraouï, M. Couderc, Sandrine Arribat, Nathalie Rodde, A. Barnaud, A. Faye, N. Kane, Y. Vigouroux, C. Berthouly-Salazar
          </td>
          <td>2025-07-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/510186795452720c77e13e46654ec25972305866" target='_blank'>
              Mechanisms for licensing origins of DNA replication in eukaryotic cells.
              </a>
            </td>
          <td>
            Bruce Stillman, J. Diffley, Janet Iwasa
          </td>
          <td>2025-06-30</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="


 Gliomas are classified as one of the most lethal cancers owing to their invasive and aggressive nature. Numerous genetic and epigenetic changes have been studied in the development and progression of GBM. Dysregulation of epigenetic function may bring about profound changes in gene expression to facilitate glioma formation and development. SETD2, a key epigenetic regulator is frequently mutated in various cancers; SETD2 regulates m6A mRNA methylation (epitranscriptome) via H3K36 trimethylation. The m6A RNA methylation modification regulates various biological functions like gene regulation, cell self–renewal and maintenance, cell differentiation, invasion, and tumorigenesis. The RNA modifications are executed by a set of complex proteins collectively known as writers, readers and erasers (RNA modifiers). These modifiers work along with the molecular cues laid down by SETD2 via H3K36 trimethylation. The positive correlation between SETD2 expression and m6A RNA modifiers highlights its direct involvement in epitranscriptomics. Glioma, a highly malignant grade IV brain tumor, is characterized by limited therapeutic options and poor clinical outcomes. However, the specific role of SETD2 in glioma and its impact on m6A RNA methylation remain inadequately defined. In this study, we investigated the expression of m6A RNA methylation regulators, the molecular pathways contributing to tumor progression, and the outcomes associated with SETD2-mediated m6A modifications in glioma. Our in vitro analyses demonstrated that knocking down SETD2 in glioma cell lines resulted in reduced oncogenic properties, such as decreased cell proliferation and invasiveness. This effect was accompanied by a global reduction in m6A methylation levels within the transcriptome, which was closely linked to the downregulation of key m6A writers, METTL3 and METTL14. These findings suggest that SETD2 plays a crucial role in maintaining m6A methylation, and its disruption can drive glioma progression through altered epitranscriptomic landscapes, offering new insights into molecular oncogenesis and potential therapeutic targets.



 Subhadra Kumari, Srinivasan M. THE ROLE OF SETD2 IN THE MOLECULAR ONCOGENESIS OF GLIOMA THROUGH m6A RNA MODIFICATION [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P75.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33a10b3b7631ab056fcfe5ee35c047e63418abb6" target='_blank'>
              Abstract P75: THE ROLE OF SETD2 IN THE MOLECULAR ONCOGENESIS OF GLIOMA THROUGH m6A RNA MODIFICATION
              </a>
            </td>
          <td>
            Subhadra Kumari, M. Srinivasan
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72bf36b7f21c4b94702b8969aa76f9a853c780b1" target='_blank'>
              PPARγ acetylation governs mammary adenocarcinoma tumor growth via acetylated residues that determine DNA sequence-specific binding.
              </a>
            </td>
          <td>
            Lifeng Tian, X. Jiao, Chenguang Wang, Danni Li, A. Ertel, Joanna Achinger-Kawecka, S. Addya, Raymond E Soccio, Eric R. Chen, Balázs Győrffy, Gabriele Di Sante, Zhijiu Zhong, Haidar Alkhafaji, Nina Entcheva, Elyssa M. Campbell, Peter A McCue, Andrew V. Kossenkov, Rita Pancsa, Peter Tompa, Susan J. Clark, Richard G Pestell
          </td>
          <td>2025-07-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Abstract The Asian icefish, Protosalanx chinensis, has undergone extensive colonization in various waters across China for decades due to its ecological and physiological significance as well as its economic importance in the fishery resource. Here, we decoded a telomereto-telomere (T2T) genome for P. chinensis combining PacBio HiFi long reads and ultra-long ONT (nanopore) reads and Hi-C data. The telomere was identified in both ends of the contig/chromosome. The expanded gene associated with circadian entrainment suggests that P. chinensis may exhibit a high sensitivity to photoperiod. The contracted genes’ immune-related families and DNA repair associated with positive selection in P. chinensis suggested the selection pressure during adaptive evolution. The population genetic analysis reported the genetic diversity and genomic footprints in 231 individuals from 7 different locations. The introduced highest alkalinity population (HRCL) exhibited higher values of inbreeding coefficients and clustered different from other groups suggested local environmental adaptation. Thus, the T2T genome and genetic variation can be valuable resources for genomic footprints in P. chinensis, shedding light on its evolution, comparative genomics, and the genetic differences between natural and introduced populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff0e41e198fbe29dbf276b6596922e3840811763" target='_blank'>
              Telomere-to-telomere genome and resequencing of 231 individuals reveal evolution, genomic footprints in Asian icefish, Protosalanx chinensis
              </a>
            </td>
          <td>
            Yanfeng Zhou, Chenhe Wang, Binhu Wang, Dongpo Xu, Xizhao Zhang, You Ge, Shulun Jiang, Fujiang Tang, Chunhai Chen, Xuemei Li, Jianbo Jian, Yang You
          </td>
          <td>2025-01-06</td>
          <td>GigaScience</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="The cyclin-dependent kinase CDK11 is an understudied kinase that has been the subject of conflicting reports regarding its function in cancer. Here, we combine genetic and pharmacological approaches to demonstrate that CDK11 is a critical regulator of cancer cell survival that is required for RNA splicing and the expression of homologous recombination genes. Inhibition of CDK11 disrupts genome stability, promotes the retention of intronic sequences in mature mRNAs, and induces synthetic lethality with PARP inhibitors. Through integrative analysis of functional genomics datasets, we identify heterozygous deletions of chromosome 1p36 - which encompasses CDK11 and its activating cyclin CCNL2 - as a recurrent and predictive biomarker of sensitivity to CDK11 inhibition. To assess the therapeutic potential of CDK11, we develop MEL-495R, a selective and orally bioavailable CDK11 inhibitor. Additionally, we establish a genetically-engineered mouse model that allows us to differentiate between the on-target and off-target effects of CDK11 inhibitors in vivo. Using this platform, we demonstrate that MEL-495R induces widespread on-target toxicity, revealing a narrow therapeutic index. Together, these findings define CDK11 as a core cancer dependency, uncover a chromosomal deletion that sensitizes tumors to CDK11 inhibition, and provide a generalizable strategy for deconvolving drug efficacy and toxicity in vivo for novel oncology targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af348abea2aee38e920e3e84752e406507b1375" target='_blank'>
              On-target toxicity limits the efficacy of CDK11 inhibition against cancers with 1p36 deletions
              </a>
            </td>
          <td>
            Linda A. Julian, Lisa Crozier, Devon A. Lukow, Sanat Mishra, Aditi Swamy, Ryan A. Hagenson, Peter Sennhenn, Erin L. Sausville, Brianna E. Mendelson, Claudio Chuaqui, Lu Qiao, Anand Vasudevan, Kuan-Ting Lin, Sonam Bhatia, Thierry Bertomeu, Andrew Chatr-Aryamontri, Li Zhang, M. Rees, M. Ronan, Jennifer A. Roth, Tim Nottoli, Suxia Bai, Jayalakshmi Lakshmipathi, Viswanathan Muthusamy, Jonathan G. Van Vranken, Steve P Gygi, Sarah L. Thompson, Joan C. Smith, K. Anderson, Sanjana Shah, R. Bindra, Martin Akerman, David L. Spector, A. Krainer, J. Sheltzer
          </td>
          <td>2025-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="Triple negative breast cancers (TNBCs) are a heterogenous subcategory of breast cancers which have significantly worse survival outcomes compared to other breast cancer subtypes. However, limited advances have been made towards a targeted TNBC therapeutic and traditional chemotherapies remain the frontline therapy. Therefore, this study aimed to examine the therapeutic potential of a novel TNBC target, Barrier-to-Autointegration Factor 1 (Banf1), including determining Banf1 expression and cellular localisation in non-malignant breast cells and a panel of TNBC cell lines. Bioinformatic analysis of patient samples demonstrated that Banf1 is overexpressed in all breast cancer stages and subtypes. Banf1 depletion inhibited proliferation and induced mitotic arrest in TNBC cells via loss of nuclear envelope integrity and aberrant nuclear morphology, inducing TNBC tumour cell-specific cell death. These findings highlight the significant overexpression and functional involvement of Banf1 in TNBC progression and suggests that it may have potential as a novel anti-cancer target, supporting further investigation. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10346-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecd1f5909e830afa31cad48c33f6b5d6c0d1eeae" target='_blank'>
              The inner nuclear membrane protein, Banf1, has an essential role in triple negative breast cancer cell proliferation and survival
              </a>
            </td>
          <td>
            M. Rose, J. Burgess, Chee Man Cheong, Iain Richard, Amila Suraweera, M. Adams, Pascal H. G. Duijf, Kenneth J. O’Byrne, Derek J. Richard, E. Bolderson
          </td>
          <td>2025-07-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Mammalian oocytes and embryos are known to exhibit a markedly low frequency of de novo mutations compared to somatic cells. We still lack efficient tools to carry out functional studies of the intergenerational mechanism of genome protection, and our view of this phenomenon is constantly being modified in light of the new results. Although oocytes were originally considered a cell type lacking DNA repair, new results indicate that mammalian oocytes might possess a set of unique properties that make them and their descendants resistant to accumulation of DNA damage. Here, we review various factors that can influence oocyte and embryo genome stability and discuss the functional evidence for the uniquely efficient response to DNA damage, particularly in the presence of minor DNA lesions and single-strand breaks. We discuss whether high levels of DNA repair proteins might be the basis for the observed low mutation rate. Finally, we present the idea that the unique characteristics of the chromatin landscape, as well as the limited replication, rather than the abundance of repair factors alone, may be responsible for the intergenerational protection of the genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5907f7fca7565b9214e0d15819fb5b0e6e30461" target='_blank'>
              Mechanisms underlying low mutation rates in mammalian oocytes and preimplantation embryos
              </a>
            </td>
          <td>
            Nataliia Dudko, Jurek W Dobrucki, Helena Fulka
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/687af5811b08e28febe794844c74138cc2e493e9" target='_blank'>
              The PARP1-EXD2 axis orchestrates R-loop resolution to safeguard genome stability.
              </a>
            </td>
          <td>
            Zhaoshuang Li, Yu Liu, Yuanhui Liu, Yiting Zhang, M. Huen, Huasong Lu, Zhigang Zhang, Jianwei Zhou, Dong Fang, Ting Liu, Jun Huang
          </td>
          <td>2025-06-27</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d71c17ea6f14ca131f9682284c6a60466ccc4f05" target='_blank'>
              Reversible DNA condensation drives natural transformation.
              </a>
            </td>
          <td>
            Joshua I. Santiago, Ishtiyaq Ahmed, J. Hahn, Abigail Rubino, David Dubnau, M. B. Neiditch, Keith J. Mickolajczyk
          </td>
          <td>2025-07-29</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cyclic GMP–AMP synthase (cGAS) serves as a crucial sentinel in innate immunity by sensing cytosolic DNA, yet the molecular mechanisms governing its activation remain incompletely understood. Here, we identify lysine demethylase 4B (KDM4B) as the specific demethylase that erases cGAS K350 methylation, facilitating its chromatin release and subsequent activation. Genetic ablation of Kdm4b compromised both antiviral immunity against HSV-1 infection and antitumor responses, while also diminishing the efficacy of anti-PD-1 immunotherapy. Mechanistically, KDM4B-mediated cGAS demethylation proved crucial for its proper subcellular distribution and activation. In the context of autoimmune diseases, we found that targeting KDM4B–cGAS axis through either genetic approaches or pharmacological inhibition of KDM4B with JIB-04 effectively ameliorated disease manifestations in both Trex1-deficient mice and peripheral blood mononuclear cells from Aicardi-Goutieres syndrome (AGS) patients. Collectively, this study demonstrated that KDM4B functions as a specific demethylase for cGAS, controlling its chromatin dissociation and subsequent activation, thereby providing a therapeutic rationale for targeting cGAS methylation in human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d02e009387f05a9eed674f2f2753383328c16438" target='_blank'>
              KDM4B enhances immune surveillance via demethylating cGAS
              </a>
            </td>
          <td>
            Qiao Peng, Huimin Zhuo, Minkang Wu, Yun Hao, Yiyi Zhang, Yuying Zheng, Lei Yu, Lin Han, Hui Ren, Yingcong Wang, Zhijie Gao, Leilei Wu, Qi Lin, Chunhua Lu, Jinghua Li, Ping Wang, Lan Fang, Haihong Yu, Meiling Lu
          </td>
          <td>2025-07-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025'],
    y: [0, 8],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>