<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-07-28 09:36:26 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>36</td>
          <td>104</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>73</td>
          <td>138</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>114</td>
          <td>145</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>341</td>
          <td>138</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>299</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a539ca1a35b8049ad46a136d79392bb325111564" target='_blank'>
              Accurate Prediction of ecDNA in Interphase Cancer Cells using Deep Neural Networks
              </a>
            </td>
          <td>
            Utkrisht Rajkumar, Gino Prasad, Ellis J Curtis, I. Wong, Xiaowei Yan, Shu Zhang, Lotte Brückner, Kristen Turner, Julie Wiese, Justin Wahl, Sihan Wu, J. Theißen, Matthias Fischer, Howard Y. Chang, A. Henssen, P. Mischel, V. Bafna
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="The role of extrachromosomal DNA (ecDNA) in lung cancer, particularly in subjects who never smoked (LCINS), remains unclear. Examination of 1,216 whole-genome-sequenced lung cancers identified ecDNA in 18.9% of patients. Enrichment of MDM2 and other oncogenes’ amplification via ecDNA possibly drives a LCINS subset. Tumors harboring ecDNA or other focal amplifications showed similarly poor survival. A strong association with whole-genome doubling suggests most ecDNA reflects genomic instability in treatment-naïve lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20bf29ecb0d2ae5c2ac96f25fe4450672918b4ba" target='_blank'>
              Examining the Role of Extrachromosomal DNA in 1,216 Lung Cancers
              </a>
            </td>
          <td>
            Azhar Khandekar, P. Hoang, J. Luebeck, M. Díaz-Gay, Wei Zhao, John P. McElderry, Caleb Hartman, Mona Miraftab, Olivia W. Lee, Kara M. Barnao, E. Bergstrom, Yang Yang, Martin A. Nowak, Nathaniel Rothman, Robert J Homer, S. Yang, Qing Lan, David C Wedge, Lixing Yang, Stephen J Chanock, Tongwu Zhang, Ludmil B. Alexandrov, Marion Landi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Chromothripsis, the chaotic shattering and repair of chromosomes, is common in cancer. Whether chromothripsis generates actionable therapeutic targets remains an open question. In a cohort of 64 patients in blast phase of a myeloproliferative neoplasm (BP-MPN), we describe recurrent amplification of a region of chromosome 21q (‘chr. 21amp’) in 25%, driven by chromothripsis in a third of these cases. We report that chr. 21amp BP-MPN has a particularly aggressive and treatment-resistant phenotype. DYRK1A, a serine threonine kinase, is the only gene in the 2.7-megabase minimally amplified region that showed both increased expression and chromatin accessibility compared with non-chr. 21amp BP-MPN controls. DYRK1A is a central node at the nexus of multiple cellular functions critical for BP-MPN development and is essential for BP-MPN cell proliferation in vitro and in vivo, and represents a druggable axis. Collectively, these findings define chr. 21amp as a prognostic biomarker in BP-MPN, and link chromothripsis to a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4397c5f80ce07fe2ffd98add28275e15180327b" target='_blank'>
              Chromothripsis-associated chromosome 21 amplification orchestrates transformation to blast-phase MPN through targetable overexpression of DYRK1A
              </a>
            </td>
          <td>
            Charlotte K Brierley, B. Yip, G. Orlando, J. Wen, S. Wen, H. Goyal, M. Levine, G. M. Jakobsdottir, Avraam Tapinos, A. Cornish, A. Rodriguez-Romera, A. Rodriguez-Meira, M. Bashton, A. Hamblin, SA Clark, Joseph C Hamley, Olivia Fox, M. Giurgiu, Jennifer O'Sullivan, L. Murphy, A. Adamo, Aude-Anais Olijnik, A. Cotton, Emily Hendrix, S. Narina, S. Pruett-Miller, A. Enshaei, Claire N Harrison, Mark Drummond, S. Knapper, A. Tefferi, I. Antony-Debré, James O. J. Davies, A. Henssen, S. Thongjuea, David C Wedge, S. Constantinescu, E. Papaemmanuil, B. Psaila, John D. Crispino, Adam J Mead
          </td>
          <td>2025-06-09</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Chromoanagenesis is a catastrophic genomic phenomenon involving sudden, extensive rearrangements within one or a few cell cycles. In osteosarcoma, the most prevalent malignant bone tumor in children and adolescents, these events dramatically alter the genomic landscape, frequently disrupting key tumor suppressor genes like TP53 and RB1, amplifying oncogene expression, and propelling tumor progression and evolution. This review elucidates how key chromoanagenic mechanisms, such as chromothripsis and chromoanasynthesis, arise from replication stress and impaired DNA repair pathways, ultimately contributing to genomic instability in osteosarcoma. Chromothripsis features prominently in osteosarcoma, occurring in up to 62% of tumor regions and driving intratumoral heterogeneity through persistent genomic crises. Next-generation sequencing, optical genome mapping, and emerging technologies like single-cell sequencing empower researchers to detect and characterize these complex structural variants, demonstrating how a single catastrophic event can profoundly influence osteosarcoma progression over time. While targeted therapies for osteosarcoma have proven elusive, innovative strategies harnessing comprehensive genomic profiling and patient-derived preclinical models hold promise for uncovering tumor-specific vulnerabilities tied to chromoanagenesis. Ultimately, unraveling how these rapid, large-scale rearrangements fuel osteosarcoma’s aggressive nature will not only refine disease classification and prognosis but also pave the way for novel therapeutic approaches to enhance patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d502c33306d89faa688fa75a0e5f2710c0c03efc" target='_blank'>
              Chromoanagenesis in Osteosarcoma
              </a>
            </td>
          <td>
            Guozhuang Li, Nan Wu, Jennifer Ghabrial, Victoria Stinnett, Melanie Klausner, Laura A Morsberger, Patty Long, Ezra G. Baraban, J. Gross, Ying S. Zou
          </td>
          <td>2025-06-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6865d140b6544d7dfc0dca54fb01a54fdf212f1a" target='_blank'>
              Mitotic BLM functions are required to maintain genomic stability
              </a>
            </td>
          <td>
            Tamara Eleanore Hamann, A. Wieland, Andrea Tirincsi, Kruno Vukušić, Farbod Mohseni, René Wardenaar, Marialucrezia Losito, Ipek Ilgin Gönenc, B. Wollnik, F. Foijer, I. Tolic, Zuzana Storchová, M. Räschle
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Testicular germ cell tumors (TGCTs) are the most common malignancy in young men, exhibit a unique developmental origin and exceptional chemosensitivity. However, the molecular distinctions between TGCT subtypes remain poorly understood. Here we present a comprehensive genomic analysis of 252 treatment-naive primary TGCTs, integrating deep whole-genome sequencing with matched transcriptomic and epigenomic data. We identify new driver genes and uncover defining features of TGCTs, including pervasive chromosome X amplification with subtype-specific X chromosome inactivation, and a germ cell-like transcriptional program. Although previously reported, whole genome doubling (WGD) in TGCTs is further characterized here as ubiquitous, developmentally early, and associated with age at onset. Seminomas are enriched for early driver mutations, secondary WGD events, sustained XIST expression and replication stress-associated indel mutational signatures, while non-seminomas show greater structural complexity, subclonal diversity, relatively earlier-onset WGD, extended tumor latency, and telomere elongation. Moreover, we identify a mutational signature, SBS87, that is exceptionally rare across cancers with exception of thiopurine-treated leukemia, but strikingly prevalent in TGCT, especially non-seminomas. SBS87 is linked to extended tumor latency and telomere elongation, implicating possible environmental or endogenous processes that mimic thiopurine-induced DNA damage in TGCT pathogenesis. Collectively, our findings define TGCTs as molecularly distinct tumors shaped by early genomic instability and highlight SBS87 as a novel mutational footprint with potential etiologic and clinical relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/797d0b2769352db6f37defdb43fa1da2c4040fb1" target='_blank'>
              A Thiopurine-like Mutagenic Process Defines TGCT Subtypes
              </a>
            </td>
          <td>
            Kevin Brown, Jun Zhong, Adriana Morales Miranda, Mengyan Zhang, Joycelyn Williams, Jacob Williams, Haoyu Zhang, Cheng Liang, Wenbo Li, Bin Zhu, Stephen J Chanock, K. Nathanson, Tongwu Zhang
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="ABSTRACT Aims & methods DNA methylation and genomic instability are critical drivers of cancer initiation and malignant progression. However, the roles of methylation aberrations and genomic instability in malignant progression have not been thoroughly investigated in intrahepatic cholangiocarcinoma (ICC). To address this, we identified differentially methylated regions (DMRs) and somatic copy number alterations (SCNAs) from 341 ICC samples across various stages. Results Our findings revealed that stages IAIB, II, IIIA, and IIIB exhibited comparable methylation changes, whereas stage IV ICC showed a pronounced accumulation of stage-specific methylation alterations. Leveraging these findings, we developed a classification model that effectively distinguished stage IV ICC from earlier stages with high accuracy using 15 DMRs. Furthermore, stage IV ICC exhibited slightly higher genomic instability, including an elevated aneuploidy score and a greater proportion of focal amplifications. We also observed a positive correlation between SCNA burden and DNA methylation entropy in the promoter, gene body, and CpG island regions, with the gene body of MDM2 serving as a notable example. Conclusions These findings highlight the potential of DNA methylation as a biomarker for metastasis diagnosis and the interplay between local genomic instability and aberrant methylation, emphasizing their synergistic roles in driving the evolutionary trajectory of ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d995c84a0cfeeb8c871fd5bd4ce5a040e83eca26" target='_blank'>
              Methylation aberrations and genomic instability synergistically drive the evolution of intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Guanghao Li, Youhuang Bai, Feng Tao, Tingting Hu, Ting Wang, Yong Zeng, Deqiang Sun
          </td>
          <td>2025-06-17</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/842e0482ffc6f8bbb0e4561726b21f49e5e47e0b" target='_blank'>
              An Enhancement of Extrachromosomal Circular DNA Enrichment and Amplification to Address the Extreme Low Overlap Between Replicates
              </a>
            </td>
          <td>
            Charles M. Burnham, Alongkorn Kurilung, Visanu Wanchai, Birgritte Regenberg, J. Delgado-Calle, Alexei G. Basnakian, I. Nookaew
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Esophageal squamous cell carcinoma (ESCC) is among the most frequently diagnosed cancer types, and affected patients frequently experience poor prognostic outcomes and high mortality rates. Many genomic studies of ESCC have been performed in recent years, yet the mutational mechanisms driving ESCC and their clinical implications remain incompletely understood. In this study, paired tumor and normal tissue samples from 22 patients with ESCC were used for whole genome sequencing-based analyses of genome-wide mutational events. These comprehensive analyses enabled the detection and characterization of various mutation subtypes in ESCC including somatic single-nucleotide variants, small insertions and deletions, copy number variations, structural variations, and circular extrachromosomal DNA. Of identified genes harboring non-silent mutations, TP53, NOTCH1, CSMD3, EP300, and FAM135B were the most frequently mutated genes in this study and they were annotated in the COSMIC Cancer Gene Census. With the exception of aging-related signatures, an APOBEC-associated mutational signature was the dominant mutational feature detected in ESCC samples, suggesting that APOBEC-mediated cytidine deamination is likely a major driver of mutations in this cancer type. Notably, our study also detected circular extrachromosomal DNA (ecDNA) events in these ESCC patient samples. The oncogenes COX6C, PVT1, and MMP12 as well as the oncogenic long non-coding RNA AZIN1-AS1 which were detected in ecDNA regions in these analyses may be associated with worse disease-free survival in ESCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b55745d3238cc107d72d837356b7050823d1fc40" target='_blank'>
              Whole genome sequencing revealed esophageal squamous cell carcinoma related biomarkers
              </a>
            </td>
          <td>
            Mingjun Li, Lei Li, Xizi Wang, Yanwei Zhao, Peina Du, Wei Wang, , Yadong Wang, Yanxing Sheng, Mingliang Gu, Xiaodong Jia
          </td>
          <td>2025-06-26</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29cfe224e356c6e68ceccf6ec7ce94377f472868" target='_blank'>
              KLF5 controls subtype-independent highly interactive enhancers in pancreatic cancer to regulate cell survival
              </a>
            </td>
          <td>
            Thomas L. Ekstrom, Zhangshuai Dai, Julia Thiel, Meghana Manjunath, Nadine Schacherer, Bishakha Joyeeta Saha, Yara Souto, A. Abdelrahman, Frank Essmann, Sven Beyes, M. Truty, Meng Dong, Steven A. Johnsen
          </td>
          <td>2025-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 3090


 Background:
 Recurrent cytogenetic abnormalities represent candidate therapeutic targets for children with neuroblastoma (NB).
 MYCN
 oncogene amplification is associated with significantly worse survival rates for children with NB and remains one of the primary predictors of patient prognosis.
 MYCN
 amplifications in NB can be found both within the linear genome and on circular extrachromosomal DNA (ecDNA), and therapeutic targeting of the mechanisms underlying
 MYCN
 amplification represents a novel and promising strategy in NB. However, the molecular features and clinical and biological significance of these amplifications in NB tumors are not sufficiently understood.
 Methods:
 Whole genome and RNA sequencing data were analyzed for NB cell lines and NCI TARGET NB samples using AmpliconSuite software for ecDNA identification and characterization. GISTIC was used for identification of recurrently amplified regions. Gene expression levels were determined using StringTie, and gene clustering heatmaps were generated using FeatureCounts software. For differential gene expression analyses, samples were divided into ecDNA
 +
 and ecDNA
 -
 , and genes contained on ecDNA were compared to the same regions on linear DNA across samples using DESeq2. Associations between ecDNA quantity, content, and patient survival were performed using multivariate Cox regression survival analysis. Associations of gene expression with patient survival were performed using the R2 Platform. The efficacy of targeting ecDNA-associated gene products was assessed using live cell imaging and cell viability assays.
 Results:
 WGS analysis confirmed 7/20 NB patient tumors from the TARGET database to be ecDNA amplified with 1-5 independent ecDNA elements and
 MYCN
 gene expression correlated with the ecDNA copy number. ecDNAs in
 MYCN
 -amplified neuroblastoma cell lines contained distinct gene combinations and possessed unique structures. MYCN overexpression in NB cells has been shown to be associated with replication stress (RS), and tumor cells containing ecDNA are hyper-reliant on the DNA damage response (DDR) kinase CHK1 to manage heightened replication stress. Expression of the
 CHK1
 gene was associated with neuroblastoma patient outcomes and neuroblastoma was most significantly associated with
 CHK1
 RNA dependency. We further validated CHK1i as a promising therapeutic strategy in
 MYCN
 amplified NB, as CHK1 inhibition with the novel inhibitor BBI-2779 was most effective against ecDNA+,
 MYCN
 -amplified neuroblastoma cell lines.
 Conclusions:
 Our results emphasize the critical role of ecDNA in NB. We identify a synthetic lethality axis shaped by ecDNA
 MYCN
 amplification and CHK1 dependence. We further demonstrate the feasibility of targeting this vulnerability through CHK1 inhibition, thus offering new avenues for treatment in
 MYCN
 amplified tumors.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e222ec29cad608a8fd4761ccb837028a1803c8a" target='_blank'>
              Effect of extrachromosomal DNA (ecDNA) on
 MYCN
 amplified neuroblastoma and patient outcomes.
              </a>
            </td>
          <td>
            Mihika Sonalkar, Carla Sampaio, Ting Yang, Taelor Getz, Catherine Shaw, Vicky Pham, Joshua Lange, Sudhir Chowdhry, Christian A Hassig, L. Alexandrov, Peter E. Zage
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10436da9e9bcdf67b6e4cfcf17b6e58df4672a34" target='_blank'>
              Spatio-temporal chromosomal arrangements by late-replicating heterochromatin
              </a>
            </td>
          <td>
            Satyadev Polisetty, Shuvadip Dutta, Rakesh Netha Vadnala, D. Notani, R. Padinhateeri, K. Sanyal
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is recognized for its aggressiveness, yet the mechanism underlying metastasis remains unclear. Here, we report that CREPT/RPRD1B, which exhibits somatic gene copy-number amplifications and elevated expression, correlates with poor patient survival and drives TNBC metastasis. We demonstrate that CREPT alters three-dimensional genome structures in topologically-associating domain (TAD) status and chromatin loops via occupying promoters and enhancers. Specifically, CREPT mediates 1082 co-operational chromatin loops configured by enhancer-promoter and promoter-termination loops, which are validated by HiChIP analyses and visualized by Tn5-FISH experiments. These loops orchestrate RNAPII loading and recycling to enhance the metastatic gene expression. Disruption of these co-operational loops using CRISPR-dCas9 suppresses TNBC metastasis in vivo. Furthermore, depletion of CREPT using an AAV-based shRNA blocks TNBC metastasis in both preventative and therapeutic mouse models. We propose that targeting CREPT to disrupt the co-operational chromatin loop structures represents a promising therapeutic strategy for metastatic TNBC. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02361-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d99f51f69c1d7213a6e3e1569c1bdb0463d7e42a" target='_blank'>
              CREPT is required for the metastasis of triple-negative breast cancer through a co-operational-chromatin loop-based gene regulation
              </a>
            </td>
          <td>
            Jianghua Li, Lu Xu, Jiayu Wang, Xuning Wang, Yuting Lin, Alex Yutian Zou, Fangli Ren, Yinyin Wang, Jun Li, Zhijie Chang
          </td>
          <td>2025-06-10</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76dc4a293cd6dc10ea69cf397e4b517d730cdc8e" target='_blank'>
              Synergistic role of Alu and core duplicon sequences in driving genomic instability at the disease-associated 16p12.3–p13.11 region
              </a>
            </td>
          <td>
            Stefania Fornezza, Giuliana Giannuzzi
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Loss-of-function mutations in genome maintenance genes fuel tumorigenesis through increased genomic instability. A subset of these tumor suppressors are challenging to identify due to context dependency, including functional interactions with other genes and pathways. Here, we searched for potential causal genes that impact tumor development and/or progression in breast cancer through functional-genetic screening of candidate genes. MYH4, encoding a class II myosin, emerged as a top hit impacting genomic stability. We show that MYH4 suppresses DNA replication stress by promoting replication licensing and replication fork progression. Moreover, we observed a strong synergistic relationship among class II myosins in suppressing replication-associated DNA damage. Genomic analysis of Pan-Cancer Analysis of Whole Genomes project breast cancer samples revealed frequent concomitant loss of TP53 with MYH4 and class II myosins on chromosome 17p. Notably, Myh4 disruption accelerated mouse mammary tumorigenesis in a Trp53-deficient background. In conclusion, our results suggest an unanticipated function of MYH4 in p53-mediated tumor suppression that can explain their combined loss in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64806f05c07b8d24d35ea0d2ce3f2fd4c89aa895" target='_blank'>
              The class II myosin MYH4 safeguards genome integrity and suppresses tumor progression
              </a>
            </td>
          <td>
            J. Thatte, Ana Moisés da Silva, Judit Börcsök, Thorkell Gudjónsson, Jan Benada, Xin Li, Muthiah Bose, H. van der Gulden, Ji-Ying Song, Renée Menezes, Elena Martín-Doncel, Luis Toledo, Valdemaras Petrosius, Cord Brakebusch, J. Jonkers, F. C. Nielsen, Maria Rossing, C. S. Sørensen
          </td>
          <td>2025-06-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23f7337991d50d13c4c6add597a9dcc86990c583" target='_blank'>
              Persistent chromatin alterations and gene expression reprogramming follow widespread DNA damage in glioblastoma
              </a>
            </td>
          <td>
            A. Modrek, Ken Chandradoss, Catherine Do, R. Ezhilarasan, Theodore Sakellaropoulos, Jerome Karp, Yingwen Ding, Ze-Yan Zhang, Melanie Graciani, Giulia Cova, Jennifer E. Phillips-Cremins, Erik P. Sulman, Jane A. Skok
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9eeecebde718f44128a5af1add184f4e02b1d49" target='_blank'>
              NuMA promotes constitutive heterochromatin compaction by stabilizing linker histone H1 on chromatin
              </a>
            </td>
          <td>
            Yao Wang, Wenxue Zhao, Jiahao Niu, Cuifang Liu, Xiaotian Wang, Weihong Yuan, Shanshan Ai, Wolfgang Baumeister, Guohong Li, Aibin He, Peng Xu, Cheng Li, Yujie Sun
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aebba9c32cce818d2e97fcfa124c22eff76b4f3" target='_blank'>
              LSH-mediated resolution of R-loops mitigates transcription-replication conflicts to preserve genomic stability in prostate cancer cells
              </a>
            </td>
          <td>
            Kai Ni, Yu Zhang, Kun Zheng, Jianwen Huang, Nuorong Xiong, Rong Liang, Yingxiao Chen, Honglin Jin, Youlong Hai, Wang Li, Jiayi Ma, Ranxing Yang, Guanggai Xia, Yueqing Bai, Lujie Song, Xiaoyong Hu, Zijian Tang, Qiang Fu
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Genome instability is a fundamental feature and hallmark of cancer, associated with aggressiveness, drug resistance and poor prognosis. RAI2 was initially identified as a novel metastasis suppressor protein specifically associated with the presence of disseminated tumour cells in the bone marrow of breast cancer patients, but its molecular function is largely unknown. Methods We analysed the consequences of RAI2 depletion on gene expression and genomic stability in luminal breast cancer cell lines, performed cytotoxicity profiling using a library of pharmacologically active compounds, and characterized a potential function of the RAI2 protein in the DNA damage response. We performed in silico validation in different breast cancer datasets. Results Analysis of clinical samples revealed that in primary breast tumours, low RAI2 gene expression is significantly associated with genomically unstable tumours and poor prognosis. RAI2 depletion in breast cancer cell lines resulted in loss of mitotic fidelity characterized by prolonged mitosis with increased chromosome segregation errors and micronuclei formation. Drug screening revealed increased sensitivity of RAI2-depleted breast cancer cells to topoisomerase I and Aurora A inhibitors. We also found that genotoxic stress induces the RAI2 protein, which has an affinity for and colocalises with poly-(ADP-ribose). We validated the association of RAI2 gene expression with DNA repair capacity in clinical samples. Conclusions Our findings support, for the first time, a functional role of RAI2 in the maintenance of genomic stability. Understanding the underlying the molecular mechanism could help to improve patient diagnosis and treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02085-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d5723a488a418c3d94e5cf45e2172b0ac676d31" target='_blank'>
              Retinoic acid-induced 2 deficiency impairs genomic stability in breast cancer
              </a>
            </td>
          <td>
            Lena Boettcher, Sarah Greimeier, Kerstin Borgmann, Shabbir S. Mughal, Bernhard Ellinger, K. Bartkowiak, B. Zobiak, Antonio V. Failla, Pascal Steffen, Ellen Claus, K. Besler, Christopher Buccitelli, Violetta Schaaf, A. Ozga, Simona Parretta, Svenja Schneegans, W. Mansour, Jan O. Korbel, Hartmut Schlueter, B. Brors, Klaus Pantel, H. Wikman, S. Werner
          </td>
          <td>2025-07-22</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Chromosomal instability (CIN) is a hallmark of cancer, closely associated with tumor evolution, metastasis, immune evasion, and resistance to treatment. CIN is driven by persistent chromosome missegregation, resulting in abnormal chromosomal copy numbers and promoting tumor progression through mitotic errors, faulty chromosome segregation, and the formation of micronuclei or chromosome bridges. Previous studies have demonstrated that p62 localizes to micronuclei, where it interacts with mitochondria, enhancing ROS-mediated cysteine oxidation and promoting p62 homo-oligomerization. This disrupts repair by recruiting CHMP7 and other ESCRT-III components, impairing their function in restoring the micronuclear envelope. Notably, we observed a time-dependent increase in nuclear p62 levels following radiation therapy, which correlated with enhanced chromatin fragmentation and chromothripsis. While p62 inhibits homologous recombination repair and promotes non-homologous end joining, we hypothesize that radiation-induced nuclear p62 accumulation impairs nuclear envelope resealing by inhibiting ESCRT-III components, thereby contributing to chromosomal fragmentation and chromothripsis. These chromosomal alterations may play a significant role in tumor evolution and treatment resistance. Our findings suggest that irradiation-induced p62 accumulation is crucial for chromothripsis and may affect nuclear repair processes, impacting chromosomal stability. Future research will focus on elucidating how p62 contributes to these abnormalities and their implications for tumor progression and treatment resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40a93ad5e20684542464f1d1902962e0915d7957" target='_blank'>
              Irradiation-induced increase in nuclear p62 levels contributes to chromosomal fragmentation and chromothripsis.
              </a>
            </td>
          <td>
            Xiaojuan Yang, Ying Zhou, Buzhe Zhang, Xue Tao, Weili Qi, Hong Wu
          </td>
          <td>2025-06-05</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcc5ca4f590540c5787b14eba5758a0b8345da44" target='_blank'>
              TrAEL-seq captures DNA replication dynamics in mammalian cells
              </a>
            </td>
          <td>
            Neesha Kara, Laura Biggins, V. Grinkevich, Alex J. Whale, Paola Garran-Garcia, Jhanavi Srinivasan, P. Rugg-Gunn, Simon Andrews, Aled J. Parry, Helen M R Robinson, J. Houseley
          </td>
          <td>2025-06-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea18029e3c5c41c740db47ee4da2a815d1304135" target='_blank'>
              5-hydroxymethylcytosine deposition mediates Polycomb Repressive Complex 2 function in MYCN-amplified neuroblastoma
              </a>
            </td>
          <td>
            Mohansrinivas Chennakesavalu, Gabriel L. Lopez, Varsha Gupta, Kelley Moore, Yuqing Xue, Josie Majowka, Sahil Veeravalli, Ryan Borchert, Monica Pomaville, A. Chlenski, Chuan He, Andrea Piunti, M. Applebaum
          </td>
          <td>2025-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1231b220c32fa68e474d1bf7533e08b4effde6bb" target='_blank'>
              Cell enlargement causes mitotic errors and aneuploidy in cells that evade senescence after CDK4/6 inhibition
              </a>
            </td>
          <td>
            Aanchal U. Pareri, Marialucrezia Losito, F. Foijer, AT Saurin
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Plasmids have fundamentally transformed how we resolve regulatory grammar across the tree of life. However, although chromatin plays an integral role in regulating the function of regulatory elements along the nuclear genome, our understanding of how, or whether, similar chromatin structures form on plasmids transfected into mammalian cells remains limited. We demonstrate that plasmid single-molecule chromatin fiber sequencing (plasmid Fiber-seq) can accurately map chromatin architectures along individual, full-length transfected plasmid molecules at near single-nucleotide resolution. Application of plasmid Fiber-seq to diverse plasmids and cell lines demonstrates that plasmids are chromatinized in a sequence-dependent organized manner and adopt a heterogeneous and incomplete chromatin architecture relative to nuclear-encoded chromatin fibers. We show that the focal occupancy of nucleosomes and transcription factors along transfected plasmids is central to their transcriptional activity within mammalian cells, and demonstrate that plasmids indeed are capable of recapitulating nuclear genome-encoded chromatin architectures, although not always. Finally, we demonstrate that combining plasmid Fiber-seq with high-throughput reporter assays can establish the molecular mechanisms underlying pathogenic non-coding variants, including disentangling the effects of transcriptional activators and repressors with near-single-nucleotide resolution. Overall, our findings reveal the principles by which plasmid-based assays can be used for accurate fine-scale mapping of chromatin-dependent regulatory grammar.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75d71ed390846f18de84c61671f46cfb09ecd9af" target='_blank'>
              Principles and functional consequences of plasmid chromatinization in mammalian cells
              </a>
            </td>
          <td>
            Ben Mallory, Thomas W. Tullius, Carina G. Biar, Jonas A. Gustafson, Stephanie C. Bohaczuk, Danilo Dubocanin, L. Starita, A. Stergachis
          </td>
          <td>2025-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The evolution of techniques used to identify structural variants (SVs) and copy number variants (CNVs) in genomes have seen significant development in the last decade. With the growing use of more technologies including chromosomal microarray, genome sequencing and genome mapping in clinical cytogenetics laboratories, reporting the frequency of SVs and CNVs has increased the complexity of genomic results. In conventional testing (e.g. karyotype or FISH) individual cells are analyzed and abnormalities are reported at the single cell level directly as a proportion of the analyzed cells. Whereas for bulk genome assays structural and sequence changes are often reported as variant allele frequencies and fractional copy number states. The International System of Cytogenomic Nomenclature (ISCN) recommends converting these values into a “proportion of the sample”, which requires different calculations and underlying assumptions based on the data type. This review illustrates how the different methods of interpreting and reporting data are performed and identifies challenges in the conversion of these values to a proportion of the sample. We stress the need for careful interpretation of data with consideration for factors that may alter how proportions are reported including overlapping SVs and CNVs or regions with acquired homozygosity. We also demonstrate, using validation data of SVs and CNVs tested by multiple techniques how results are largely consistent across methodologies, but can show dramatic differences in rare circumstances. This review focuses on illustrating many of the challenges with aligning reporting using different techniques and their underlying assumptions. As hematologic disease classifications start to incorporate numeric limits (e.g. VAF defining thresholds), it is important for laboratory geneticists, pathologists and clinicians to appreciate the differences in methodologies, potential pitfalls and the nuances when comparing bulk genome analyses to the more conventional single cell techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/883b8047db9fe9c01ef1b4982d92e6a1387b4be0" target='_blank'>
              What the VAF? A guide to the interpretation of variant allele fraction, percent mosaicism, and copy number in cancer
              </a>
            </td>
          <td>
            Adam C Smith, Hubert Tsui, Sila Usta, Jose-Mario Capo-Chichi
          </td>
          <td>2025-07-08</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Objective Repetitive DNA comprises the majority of nuclear genomes in eukaryotes and is critical for genome stability, gene regulation and evolutionary innovation. Yet, most genomic surveys focus on lowcopy regions, leaving repeats underexplored. Here, we assess how repetitive elements shape genomic diversity and rapid radiation in Dyckia (Bromeliaceae) by characterizing the abundance, composition and variability of major repeat families using lowcoverage whole-genome sequencing and the RepeatExplorer2 pipeline. Results description A substantial proportion of Dyckia genome consists of repetitive DNA, reaching ~ 71% in D. densiflora, D. elata, and D. consimilis. Notable interspecific and intraspecific variation was observed, with D. consimilis ranging from 50.3 to 69.1%. Satellite DNA, though present in all species, varied in abundance (0.1–4.7%), indicating heterochromatin flexibility. Ty3/Gypsy and Ty1/Copia LTR retrotransposons dominate the repeatome, with lineage-specific expansions of Tekay and Ogre elements. Despite general conservation, population-level differences in repeat composition suggest a role in genome restructuring and phenotypic plasticity. These patterns point to repeat dynamics as a key driver of genome evolution, taxonomic complexity, and ecological adaptability in Dyckia. Supplementary Information The online version contains supplementary material available at 10.1186/s13104-025-07359-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3a9ee02793b5181e16f379a04487e1d2a8906d6" target='_blank'>
              Genomic characterization of repetitive DNA and transposable elements in Dyckia (Pitcairnioideae) species
              </a>
            </td>
          <td>
            João Victor da Silva Rabelo-Araujo, A. F. Francisconi, Zirlane Portugal da Costa, Caroline Bertocco Garcia, Otavio Batista de Castro Ribeiro, Ana Cristina Silva Amoroso Anastacio, Maria Imaculada Zucchi
          </td>
          <td>2025-07-09</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 e18058


 Background:
 HNCs are a diverse group of malignancies characterized by genetic, epigenetic, and proteomic alterations that drive tumor progression, treatment resistance, and poor outcomes.
 Methods:
 Molecular profiling of 273 HNC samples was conducted using a semiconductor-based next-generation sequencing (NGS) platform at Datar Cancer Genetics. A subset of 106 cases underwent targeted transcriptomic analysis of 20,802 genes. Additionally, PD-L1 (22C3) IHC, microsatellite instability (MSI), and tumor mutational burden (TMB) were analyzed in selected samples.
 Results:
 Transcriptomic analysis highlighted significant alterations in critical cellular pathways, including cell cycle control, apoptosis, DNA repair mechanisms, and transcription regulation. Among the most dysregulated genes were
 DLGAP5
 ,
 CASP14
 ,
 BCL2
 , and
 JUN
 . Copy number profiling in a subset (N=74) revealed significant chromosomal instability, with 34% of cases showing losses on specific chromosome arms and 9% exhibiting gains, notably in regions such as
 4p, 19p
 and
 1p.
 Mutations were predominantly observed in tumor suppressor genes, with a lower frequency in oncogenes. Frequently altered genes include
 TP53
 (63%),
 CDKN2A
 (19%),
 PIK3CA
 (12%),
 TERT
 (15%),
 HRAS
 (6%) and
 EGFR
 (1%).
 HRAS
 mutations offer potential for targeted therapy with tipifarnib, a farnesyltransferase inhibitor.
 EGFR
 alterations, though rare, have proven challenging to effectively target with
 EGFR
 -specific therapies such as cetuximab or tyrosine kinase inhibitors, underscoring the critical need for innovative strategies to exploit this target. Amplifications were observed in
 MYC
 (9%),
 CCND1
 (8%),
 FGF19
 (6%),
 FGF3
 (4%) and
 FGF4
 (4%). Targetable amplifications were seen in
 EGFR
 (6%),
 ERBB2
 (2%) and
 MET
 (1%), offering potential opportunities for personalized therapies.
 MET
 amplifications are also linked to aggressive tumor behavior. Fusion was a relatively less common event, observed in 9.5% cases, including one case each of targetable
 BRAF-MRPS33
 and
 FGFR1-PLAG1
 fusion. High TMB (≥10 muts/mb) was seen in 21% cases (Median 12, range 0-28). None of the tumors showed MSI-high status (n=63). Higher PD-L1 positivity was observed in TMB high samples (76% vs 59%), with PD-L1 22-C3 CPS ≥1 in 66% (50/ 76) and PD-L1 28-8 TPS ≥1 in 46% (36/ 78).
 Conclusions:
 This study underscores the genetic heterogeneity of HNCs, pointing to deregulated pathways involved in cell cycle control, apoptosis, and DNA repair as promising targets for future therapeutic development and personalized treatment strategies. Also this study points towards a subset of patients who may derive a greater benefit with immunotherapy, based on TMB/PDL1.


 ESCAT TIER distribution of the molecular alterations detected in head and neck cohort (N=273).




 TIERs
 Total Cases
 % occurrence




 IC
 6
 2%


 IIA
 21
 8%


 IIIA
 85
 31%


 IIIA
 1
 0%


 IIIB
 57
 21%


 IVA
 180
 66%


 IVB
 71
 26%


 X
 142
 52%




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a204052c6edd84c3f9c00c6d7bada097fe56677e" target='_blank'>
              Molecular profiling of head and neck cancers (HNCs): Genetic heterogeneity, immune landscape, and therapeutic targets.
              </a>
            </td>
          <td>
            R. K. Choudhary, S. Limaye, Padman Vamadevan, Vivek Agarwala, Vikas Goswami, H. P, Vijay Anand Reddy, Niyati Krunal Shah, Darshit Kalpeshkumar Shah, Shivam Shingla, Dhanashri Ahire, Vinayak Rao, R. Gosavi, Priyanka Desale, R. Datar, D. Patil
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 e15108


 Background:
 Children with high-risk and relapsed neuroblastoma (NB) need improved therapies, and recurrent cytogenetic abnormalities, such as
 MYCN
 oncogene amplification, represent candidate therapeutic targets.
 MYCN
 amplification and increased
 MYCN
 expression drive deregulated hyper-transcription that leads to development and growth of NB tumors, and
 MYCN
 amplifications, which can be found both within the linear genome (HSR) and on circular extrachromosomal DNA (ecDNA), are associated with significantly worse survival rates for children with NB.
 MYCN
 overexpression has been linked to an increase in replication stress (RS), and RS and subsequent genome instability are important drivers of tumor initiation and progression. Flap Endonuclease 1 (FEN1), a non-essential DNA replication enzyme, was identified as a synthetic lethal target in
 BRCA1
 /
 2
 -deficient cancers via induction of RS. Recent success of targeting RS-elevated cancers with replication enzyme inhibitors opens a new avenue to target
 MYCN
 -amplified NB.
 Methods:
 Associations of gene expression with patient survival and prognostic features were performed on available neuroblastoma tumor databases using the R2 Genomics Analysis and Visualization Platform. The efficacy of FEN1 inhibition was assessed using live cell imaging and cell viability assays, comparing results in
 MYCN
 -amplified to -nonamplified NB cells and in NB cells with inducible
 MYCN
 expression and repression. Mechanisms of cell death and impacts on replication stress in cells treated with FEN1 inhibitors were evaluated by Western blots.
 Results:
 We have found that
 FEN1
 expression levels are associated wtih NB patient outcomes and with features of high-risk disease, including tumor stage and
 MYCN
 amplification. FEN1 inhibition was effective against NB cells and was significantly more effective in
 MYCN
 -amplified NB cells causing reduced cell growth and viability. Increased
 MYCN
 expression also led to increased sensitivity to FEN1 inhibition, and reduced MYCN expression reduced sensitivity. FEN1 inhibition led to the induction of apoptosis but not necroptosis in NB cells and responses to FEN1 inhibition were associated with markers of replication stress, including activation of the ATR-CHK1 and ATM-CHK2 pathways.
 Conclusions:
 We have discovered that
 MYCN
 -amplified NB cells are hypersensitive to FEN1 inhibition, suggesting that FEN1 inhibition may be a promising therapeutic strategy for children with high-risk and relapsed neuroblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa405fa9ce38d24eb7641c652f857fc594ac4869" target='_blank'>
              Mechanisms of efficacy of endonuclease inhibition in neuroblastoma.
              </a>
            </td>
          <td>
            Carla Sampaio, Eric Wu, Madison Cinelli, Erica Steen, Andrew Shiau, Richard Kolodner, Jean Yin Jen Wang, Peter E. Zage
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d5c649254c4e2aba35b14c09f15fffc91fc90e" target='_blank'>
              RB dependent transcriptional regulation at mitotic centromeres preserves genome stability
              </a>
            </td>
          <td>
            Elizabeth A Crowley, Amity L Manning
          </td>
          <td>2025-06-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0f3b475325be08160e158336893ab2921a98e71" target='_blank'>
              Neuronal migration induces DNA damage in developing brain
              </a>
            </td>
          <td>
            Zhejing Zhang, Andres Canela, Peilin Zou, Takahiro Furuta, Noriko Takeda, Takumi Kawaue, Naotaka Nakazawa, Mai Saeki, Masaki Utsunomiya, Junko Kurisu, Fumiyoshi Ishidate, Hiroyuki Sasanuma, Yusuke Kishi, M. Kengaku
          </td>
          <td>2025-06-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Genome instability (GIN) is a cell pathology linked to cancer promotion and tumor evolution. Transcription is an essential cellular process but also a potential source of DNA damage and GIN. Transcription-replication conflicts (TRCs) are a predominant source of GIN, and defective TRC resolution may seriously compromise genome integrity. Importantly, chromatin dynamics helps orchestrate the response to TRCs to preserve genome integrity. Multiple epigenetic deficiencies have been shown to cause transcription-induced replication stress, resulting in DNA breaks and mutations. Consistently, chromatin alterations are frequent in cancer and correlate with increased mutation burden at TRC sites in tumors. Here, we review our current knowledge of TRC processing, the consequences of its dysfunction, and its relevance in cancer. We focus on the interplay between the DNA damage response (DDR) and chromatin dynamics and discuss the clinical potential of targeting TRCs as anticancer strategies and drugging the associated epigenetic signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/633badfaea4225294a91ff97893c1fea15ce4095" target='_blank'>
              Transcription-Replication Conflicts: Unlocking New Frontiers in Cancer.
              </a>
            </td>
          <td>
            Aleix Bayona-Feliú, A. Aguilera
          </td>
          <td>2025-06-09</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mapping the spatial organization of DNA-level somatic copy number changes in tumors can provide insight to understanding higher-level molecular and cellular processes that drive pathogenesis. We describe an integrated framework of spatial transcriptomics, tumor/normal DNA sequencing, and bulk RNA sequencing to identify shared and distinct characteristics of an initial cohort of eleven gliomas of varied pathology and a replication cohort of six high-grade glioblastomas. We identify focally amplified extrachromosomal DNA (ecDNA) in four of the eleven initial gliomas, with subclonal tumor heterogeneity in two EGFR-amplified grade IV glioblastomas. In a TP53-mutated glioblastoma, we detect a subclone with EGFR amplification on ecDNA coupled to chromosome 17 loss of heterozygosity. To validate subclonal somatic aneuploidy and copy number alterations associated with ecDNA double minutes, we examine the replication cohort, identifying MDM2/MDM4 ecDNA subclones in two glioblastomas. The spatial heterogeneity of EGFR and p53 inactivation underscores the role of ecDNA in enabling rapid oncogene amplification and enhancing tumor adaptability under selective pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48c8db15868df338dc48b4c0bf5130f5e0f200b" target='_blank'>
              Resolving spatial subclonal genomic heterogeneity of loss of heterozygosity and extrachromosomal DNA in gliomas
              </a>
            </td>
          <td>
            Michelle G. Webb, Frances Chow, Carmel G. McCullough, Bohan Zhang, John J. Y. Lee, Rania Bassiouni, Norman E. Garrett, Kyle Hurth, J. Carpten, Gabriel Zada, David W. Craig
          </td>
          <td>2025-06-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f6786fefc42e4d77082eb7a0b7de5ec1cad7e96" target='_blank'>
              MITF maintains genome stability in non-melanocytic cell lineages and suppresses Hippo pathway signaling
              </a>
            </td>
          <td>
            Drifa H. Gudmundsdottir, Adrián López García de Lomana, Thejus B. Venkatesh, Kritika Kirty, S. Sigurdsson, Linda Vidarsdottir, R. Dilshat, Erla Sveinbjornsdottir, Snædís Ragnarsdóttir, Daniel H Magnusson, Maria R. Bustos, E. Steingrímsson, Thorkell Gudjónsson, Stefan Sigurdsson
          </td>
          <td>2025-06-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/779c13815a53f4713be7d07b72908d9198ac23e6" target='_blank'>
              Nucleosome-mediated conformational switches in micro-eccDNAs
              </a>
            </td>
          <td>
            Sarah Harris, Victor Velasco Berrelleza, William Sandel, T. Sutthibutpong, Victoria N. Parikh, Euan A. Ashley, Andrew Fire, S. Levene, Thomas C Bishop, M. Shoura
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a45fc2e4de0ca1a04af5876a2240acffe2c6ff19" target='_blank'>
              Role of PLK4 inhibition in cancer therapy.
              </a>
            </td>
          <td>
            Kishore Banik, Thomas J Hayman
          </td>
          <td>2025-06-13</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumors (ATRTs) are aggressive central nervous system tumors mainly affecting young children. Extensive molecular characterization based on gene expression and DNA methylation patterns has solidly established three major ATRT subtypes (MYC, SHH and TYR), which show distinct clinical features, setting the basis for more effective, targeted treatment regimens. Transcriptional activity of transposable elements (TEs), like LINE1s and LTRs, is tightly linked with human cancers as a direct consequence of lifting epigenetic repression over TEs. The sole recurrent biallelic loss-of-function mutation in SMARCB1 in ATRTs, a core component of the SWI/SNF chromatin remodeling complex, raises the question of how TE transcription contributes to ATRT development. Here, we comprehensively investigate the transcriptional profiles of 1.9M LINE1 and LTR elements across ATRT subtypes in primary human samples. We find TE transcription profiles allow sample stratification into ATRT subtypes. The TE activity signature in the ATRT-MYC subtype is unique, setting these tumors apart from SHH and TYR ATRTs. More specifically, ATRT-MYC show broadly reduced transcript levels of LINE1 and ERVL-MaLR subfamilies. ATRT-MYC also displayed significantly less LTR and LINE1 loci with bidirectional promoter activity. Furthermore, we identify 849 differentially transcribed TEs in primary samples, which are predictive towards established ATRT-SHH and -MYC cell line models. In summary, including TE transcription profiles into the molecular characterization of ATRTs might reveal new tumor vulnerabilities leading to novel therapeutic interventions, such as immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02078-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c60e5327059223ad1194aac3ab94faeb7d65828" target='_blank'>
              Expression of LTR and LINE1 transposable elements defines atypical teratoid/rhabdoid tumor subtypes
              </a>
            </td>
          <td>
            M.V. Hamann, Shweta Godbole, Maisha Adiba, Sabrina M. Leddy, Jelena Navolić, Ghazaleh Tabatabai, Daniel J Merk, Ulrike C. Lange, J. Neumann
          </td>
          <td>2025-07-22</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Chromosome instability leading to aneuploidy and accumulation of copy number gains or losses is a hallmark of cancer. Copy number alteration (CNA) signatures are increasingly used for cancer risk stratification, but size thresholds for defining CNAs across cancers are variable and the biological and clinical implications of CNA size heterogeneity and co-occurrence are incompletely understood. Here we analyze CNA and clinical data from 691 meningiomas and 10,383 tumors from The Cancer Genome Atlas to develop cancer- and chromosome-specific size-dependent CNA and CNA co-occurrence models to predict tumor control and overall survival. Our results shed light on technical considerations for biomarker development and reveal prognostic CNAs with optimized size thresholds and co-occurrence patterns that refine risk stratification across a diversity of cancer types. These data suggest that consideration of CNA size, focality, number, and co-occurrence can be used to identify biomarkers of aggressive tumor behavior that may be useful for individualized risk stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa10dcbb2ebaf779a0622a82815ec7ac7b25cca" target='_blank'>
              Pan-cancer copy number analysis identifies optimized size thresholds and co-occurrence models for individualized risk stratification
              </a>
            </td>
          <td>
            M. P. Nguyen, William C Chen, K. Mirchia, A. Choudhury, Naomi Zakimi, Vijay Nitturi, Tiemo J. Klisch, Stephen T. Magill, Calixto-Hope G. Lucas, Akash J. Patel, David R Raleigh
          </td>
          <td>2025-07-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background/Objectives: Aneuploidy is near-ubiquitous in cancer and can decrease chemotherapy efficacy while also sensitizing cells to other drugs. Methods: To systematically identify treatment strategies that target aneuploid cancers, data were integrated from The Cancer Genome Atlas (TCGA; 10,967 samples, 16,948 aneuploidy events) and the Broad Institute’s Profiling Relative Inhibition Simultaneously in Mixtures (PRISM) screen of 578 cancer cell lines and 4518 compounds. Results: Our analyses uncovered 37,720 significant positive and negative associations linking specific aneuploidies and treatments with patient prognosis or cell viability. Within TCGA data, 22 treatments correlated with improved 5-year survival for specific aneuploid cancers, whereas 46 were linked to worse outcomes. A complementary analysis of PRISM identified 17,946 compound–aneuploidy associations and 16,189 mechanism of action (MOA)–aneuploidy associations. Pathway-altering compounds that selectively reduce viability in cells with aneuploidy profiles were discovered, including an unexpectedly prominent number of glucocorticoid receptor agonists. Conclusions: This integrated dataset provides a resource for designing therapeutic decision hypotheses, identifying drug-repurposing opportunities, and informing future studies aimed at targeting aneuploidy-induced vulnerabilities in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/811f8e93329a18a966c2c968f4a1e0431dca5e53" target='_blank'>
              Half the Chromosome It Used to Be: Identifying Cancer Treatments Targeting Aneuploid Losses
              </a>
            </td>
          <td>
            Andrew O. Disharoon, Joe R. Delaney
          </td>
          <td>2025-06-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Uveal melanoma (UM) is the most common intraocular malignancy in adults and frequently metastasizes. Somatic mutations and chromatin aberrations have been implicated in the pathogenesis of this deadly disease. Despite rapid progress in elucidating the genetic landscape of UM, the epigenetic architecture underlying UM pathogenesis remains incompletely understood. Here, we describe a super-enhancer-mediated epigenetic pipeline through genome-scale histone acetylation and transcriptional profiling. We first characterized the active landscape of super-enhancer profiles in UM via chromatin immunoprecipitation sequencing (ChIP-seq). We identified master transcription factors specifically driven by UM-specific super-enhancers, and our pipeline identified transcription factor AP-2 alpha (TFAP2A), which is highly associated with metabolism and oncogenesis, as the top essential regulator in UM. TFAP2A occupied predicted super-enhancers associated with the oncogene Solute Carrier Family 7 member 8 (SLC7A8) in UM, thereby elucidating a mechanism for regulating oncogene expression. Collectively, our data illustrate the potential for epigenetic targeting of super-enhancer-mediated oncogene dependencies in UM, highlighting an epigenetic vulnerability that can be exploited for precision therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09f568892c966ac714c9bbdb7905467e036c0b85" target='_blank'>
              Super-enhancers orchestrate transcriptional dysregulation and metabolic reprogramming in uveal melanoma
              </a>
            </td>
          <td>
            Hui Pan, Weihuan Shao, Huixue Wang, Shengfang Ge, Lingyu Zhang, Xiaofang Xu, Yefei Wang, Zhuang Ai
          </td>
          <td>2025-06-23</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Transposable elements (TEs) are abundant in the human genome, and they provide the source for genetic and functional diversity. Previous studies have suggested that TEs are repressed by DNA methylation and chromatin modifications. Here through integrating transcriptome and 3D genome architecture studies, we showed that haploinsufficient loss of NIPBL selectively activates alternative promoters (altPs) at the long terminal repeats (LTRs) of the TE subclasses. This activation occurs through the reorganization of topologically associating domain (TAD) hierarchical structures and the recruitment of proximal enhancers. These observations indicate that TAD hierarchy restricts transcriptional activation of LTRs that already possess open chromatin features. Perturbation of hierarchical chromatin topology can lead to co-option of LTRs as functional altPs, driving aberrant transcriptional activation of oncogenes. These data uncovered a new layer of regulatory mechanisms of TE expression and posit TAD hierarchy dysregulation as a new mechanism for altP-mediated oncogene activation and transcriptional diversity in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13b5cab876cec5cb4938bafcc68be31f6630d5cb" target='_blank'>
              Disruption of TAD hierarchy promotes LTR co-option in cancer
              </a>
            </td>
          <td>
            Elissa W P Wong, Merve Sahin, Rui Yang, UkJin Lee, Dan Li, Y. Zhan, Rohan Misra, Fanny Tomas, Nawaf Alomran, A. Polyzos, Cindy J. Lee, Tuan Trieu, A. Martínez-Fundichely, Thomas Wiesner, Andrew Rosowicz, Shuyuan Cheng, Christina Liu, Morgan Lallo, A. Shoushtari, T. Merghoub, Pierre-Jacques Hamard, R. Koche, Ekta Khurana, E. Apostolou, Deyou Zheng, Yu Chen, Christina S. Leslie, Ping Chi
          </td>
          <td>2025-06-30</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="
 Background: Genomic structural variation (SV) is a recognized property of cancer cells, contributing to genomic instability and oncogenesis. SV breakpoints and rearrangement patterns are often tumor-specific and can reflect underlying tumor biology. The landscape and implications of SV in breast cancer has been incompletely characterized. Furthermore, the utility of using SVs for prospective circulating tumor DNA (ctDNA) detection and monitoring in early breast cancer (EBC) has not been evaluated. Methods: SV was evaluated through whole genome sequencing (WGS) analysis in two independent breast cancer datasets: (1) The 100,000 genomes project [n=3044 patients; Genomics England, GEL], made possible through access to data and findings in the National Genomic Research Library via the Genomics England Research Environment and (2) a cohort of patients with EBC treated with neoadjuvant chemotherapy [ctDNA evaluation in early breast cancer (TRACER; n=210 patients) Princess Margaret Cancer Centre, Canada]. SV burden and type was evaluated in the GEL dataset using MANTA and in TRACER using an in-house pipeline (SV size cut-off of >50 bp). Survival correlates within the GEL dataset were evaluated using Cox proportional hazard models with cases censored after 6 years. The performance of an SV-based ctDNA assay was evaluated in BT-474 (cell line, contrived samples) and in FFPE tumors (TRACER, clinical samples) using shallow depth (∼15X) WGS followed by SV detection (up to 16) via proprietary multiplex digital PCR. Longitudinal ctDNA detection was performed in the TRACER cohort. Results: SV was common across all breast cancer subtypes (GEL data; median SV burden: 108, range: 4-1448; median aggregate copy number of the top 16 SV: 57, range: 16-160). A higher proportion of inversions were seen in HER2-positive tumors, while deletions were common in TNBC. In a multivariate model (including clinical stage, subtype, and tumor mutational burden), there was a significant association between higher SV count and worse overall survival (OS) in ER+/HER2- breast cancer (HR: 2.31, p=0.0207). Patients with ER+/HER2- breast cancer who experienced a clinical recurrence had higher SV copy number in the top 16 variants than those who did not (p<0.0001). In silico analyses demonstrated that personalized SV-based ctDNA panels (fingerprints) could be successfully designed for 97.1% of GEL cases. To assess the characteristics of an SV-based ctDNA assay, a limit of detection (LoD) study was performed with BT-474 using contrived cfDNA (70 ng). The LoD95 was estimated at 0.00052% tumor fraction (5 PPM) with variants detected as low as 1 in 10 million (0.00001% or 0.1 PPM in 31% of cases). A specificity of 100% was seen in 134 healthy donors using 24 different fingerprints (assessment of 1600 SVs). In an initial cohort of 55 patients (TRACER; ER+/HER2-:19, HER2+:23, TNBC:13), shallow WGS and fingerprint design were successful in all patients. The median number of SVs was 336 (range: 73-1345) with the SV type distributed in a similar fashion (more inversions in HER2+ tumors, deletions in TNBC). A trend towards higher SV copy number based on tumor-only WGS was also seen in those with ER+/HER2- disease who experienced subsequent recurrence (TRACER; SV in no recurrence vs. recurrence: 54.3 vs. 134.1, p=0.099). Shallow WGS and SV-based ctDNA assay design is underway for additional patients in the TRACER cohort. Conclusion: SVs are prevalent in breast cancer and associated with prognosis. Personalized SV-based panels permitted ultrasensitive ctDNA detection with high sensitivity and specificity in contrived samples, supporting assay feasibility for early breast cancer. Further analysis of the prognostic impact of SV-burden and type as well as on-treatment and adjuvant ctDNA detection in patients with EBC (TRACER) using SV tracking will be presented at the meeting.
 Citation Format: Mitchell Elliott, Karen Howarth, Sasha Main, Jesús Fuentes Antrás, Philippe Echelard, Aaron Dou, Eitan Amir, Michelle B. Nadler, Elizabeth Shah, Celeste Yu, Scott Bratman, Taylor Bird, June Roh, Elza C. de Bruin, Christopher Rushton, Sofia Birkeälv, Miguel Alcaide, Lucia Oton, Sergii Gladchuk, Yilun Chen, Anthony George, Girish Putcha, Samuel Woodhouse, Philippe L. Bedard, Lillian L. Siu, Hal K. Berman, David W. Cescon. Somatic Structural Variation in Breast Cancer and its Application in Longitudinal Analysis of Circulating Tumor DNA in Early Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS9-05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8337c1a084ec3751caa33b2040b1336acad4d2d" target='_blank'>
              Abstract PS9-05: Somatic Structural Variation in Breast Cancer and its Application in Longitudinal Analysis of Circulating Tumor DNA in Early Breast Cancer
              </a>
            </td>
          <td>
            M. Elliott, Karen Howarth, S. Main, J. F. Antrás, Philippe Echelard, Aaron Dou, Eitan Amir, M.B. Nadler, Elizabeth Shah, Celeste Yu, Scott V. Bratman, Taylor Bird, June Roh, Elza C de Bruin, Christopher Rushton, Sofia Birkeälv, M. Alcaide, L. Oton, Sergii Gladchuk, Yilun Chen, A. George, Girish Putcha, Samuel Woodhouse, P. Bedard, Lillian L. Siu, H. Berman, D. Cescon
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e24cfd9851764715ead654d1c47914a25b505dff" target='_blank'>
              Die Vermessung des Genoms
              </a>
            </td>
          <td>
            Clemens Steinek, H. Leonhardt
          </td>
          <td>2025-06-01</td>
          <td>BIOspektrum</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The spatial organization of the genome within the nucleus is a fundamental regulator of gene expression, genome stability, and cell identity. This review addresses the central question of how nuclear genome architecture contributes to disease mechanisms and diagnostics, and how technological advances enable its clinical exploration. We first outline the principles of nuclear genome architecture, including chromosome territories, replication timing, and 3D domains, and their role in gene regulation and disease. We then explore the mechanisms and consequences of chromosomal rearrangements, and how replication dynamics intersect with epigenetic regulation and genome stability. Diagnostic tools are presented in chronological progression, from conventional cytogenetics to high-resolution genomic and single-cell techniques. A dedicated section focuses on cancer cytogenomics and its clinical implications. We further highlight emerging technologies for 3D genome and epigenome profiling and their integration into diagnostic workflows. Finally, we discuss current challenges, such as standardization and cost, and the transformative potential of multi-omics and artificial intelligence for future precision diagnostics. Overall, we provide a comprehensive overview of how cytogenetics and cytogenomics contribute to the understanding and clinical diagnosis of genetic and neoplastic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0253c2480a5ecc52bcd5c1eda31a7ec8e922ab1" target='_blank'>
              Cytogenetics and Cytogenomics in Clinical Diagnostics: Genome Architecture, Structural Variants, and Translational Applications
              </a>
            </td>
          <td>
            Concetta Federico, Desiree Brancato, Francesca Bruno, Elvira Coniglio, V. Sturiale, Salvatore Saccone
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/264c746b03fe346aca4a7ae7979467caf2a477ca" target='_blank'>
              Molecular Phenotypic Plasticity Informs Possible Adaptive Change of Triple-Negative Breast Cancer Cells In Vivo
              </a>
            </td>
          <td>
            Md. Iftehimul, P. Muganda, Robert H. Newman, C. J. Rorie, Scott H. Harrison, M. T. Hossain, Misty D. Thomas, Joseph L. Graves, Dipongkor Saha
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Chromosome arm-level aneuploidies (CAAs) are a common consequence of genomic instability during the evolution of solid tumor and often associated with metastasis and therapy resistance. In breast cancer, chromosome arms 1q and 8q are the frequent amplified regions for CAAs. This study aims to investigate whether critical genes amplified in genomic regions with CAAs contribute to the pathophysiology in triple-negative breast cancer (TNBC). We established multiple long-term culturable TNBC patient-derived cells (PDCs) using spheroid culture technique, which is favorable for the enrichment of cell population with cancer stemness. We performed a genome-wide chromatin immunoprecipitation study for histone H3K27ac and super-enhancer analysis in these TNBC PDCs and cell lines. Super-enhancer analysis revealed that 8q is a substantial chromosomal region with a high density of super-enhancers. Among genes in the vicinity of 8q super-enhancers, we identified that MYB proto-oncogene like 1 (MYBL1) is a transcription factor with high abundance in TNBC PDCs as well as in basal-like BT549 cells. In TCGA breast cancer database, 8q gain is a common genomic feature in MYBL1 gene-amplified breast cancer tissues and MYBL1 expression is substantially correlated with 8q-related genes and proliferation-related genes like MKI67 and BUB1. In TNBC PDCs and cell lines, we showed that MYBL1-specific siRNAs significantly repressed TNBC cell proliferation and migration. RNA sequencing analysis in TNBC cells revealed that MYBL1 silencing substantially downregulated genes involved in the transcription machinery such as nuclear envelope, chromatin dynamics, and DNA replication. In immunohistochemical analysis of clinical TNBC tissues from a Japanese cohort, we showed that a positive MYBL1 immunoreactivity (IR) was significantly associated with shorter disease-free survival (DFS). Among candidate MYBL1 targets, we demonstrated that NCAPH, a subunit of condensin I complex, was a prognostic factor for patients with TNBC based on the immunohistochemistry of clinical TNBC tissues. Notably, patients with double IR positivity for MYBL1 and NCAPH showed shorter DFS than those with MYBL1 or NCAPH positivity alone. MYBL1 has been characterized as an essential transcriptional regulator in male meiosis and in female mammary gland development. Our findings indicate that MYBL1 plays an essential role in the pathophysiology for advanced breast cancers such as TNBC, and MYBL1 and its downstream genes can be potential diagnostic and therapeutic targets for the disease.
 Citation Format: Akihiro Fujimoto, Kazuhiro Ikeda, Keiichi Kinowaki, Hidetaka Kawabata, Akihiko Osaki, Satoshi Inoue, Kuniko Horie. Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-02-08.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bc0fd91427900f720610a3a8305e7e955f0eb39" target='_blank'>
              Abstract P2-02-08: Chromosome 8q gain-related transcription factor MYBL1 is a critical regulator in triple-negative breast cancer
              </a>
            </td>
          <td>
            Akihiro Fujimoto, K. Ikeda, Keiichi Kinowaki, H. Kawabata, Akihiko Osaki, Satoshi Inoue, K. Horie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Background Repetitive elements account for a large proportion of the human genome and undergo alterations during early tumorigenesis. However, the exclusive fragmentation pattern of DNA-derived cell-free repetitive elements (cfREs) remains unclear. Methods This study enrolled 32 healthy volunteers and 112 patients with five types of cancer. A novel repetitive fragmentomics approach was proposed to profile cfREs using low-pass whole genome sequencing (WGS). Five innovative repetitive fragmentomic features were designed: fragment ratio, fragment length, fragment distribution, fragment complexity, and fragment expansion. A machine learning-based multimodal model was developed using these features. Results The multimodal model achieved high prediction performance for early tumor detection, even at ultra-low sequencing depths (0.1×, AUC = 0.9824). Alu and short tandem repeat (STR) were identified as the primary cfREs after filtering out low-efficiency subfamilies. Characterization of cfREs within tumor-specific regulatory regions enabled accurate tissue-of-origin (TOO) prediction (0.1×, accuracy = 0.8286) and identified aberrantly transcribed tumor driver genes. Conclusion This study highlights the abundance of repetitive DNA in plasma. The innovative fragmentomics approach provides a sensitive, robust, and cost-effective method for early tumor detection and localization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f71e0eea2828ba986a7986cc3d935c9f50cf8f6" target='_blank'>
              Comprehensive fragmentation of cell-free repetitive DNA for enhanced cancer detection in plasma
              </a>
            </td>
          <td>
            Mingguang Zhang, Shuohui Dong, Wei Rao, Shiwen Mei, Gang Hu, Ling Liu, Zhen Wang, Jianqiang Tang
          </td>
          <td>2025-07-09</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Mutation, deletion, or silencing of genes encoding cellular metabolism factors occurs frequently in human malignancies. Neomorphic mutations in isocitrate dehydrogenases 1 and 2 (IDH1/2) promoting the production of R-2-hydroxyglutarate (R-2HG) instead of α-ketoglutarate (αKG) are recurrent in human brain cancers and constitute an early event in low-grade gliomagenesis. Due to its structural similarity with αKG, R-2HG acts as an inhibitor of αKG-dependent enzymes. These include the JUMONJI family of lysine demethylases, among which KDM4A is particularly sensitive to R-2HG-mediated inhibition. However, the precise molecular mechanism through which inhibition of αKG-dependent enzymes by R-2HG promotes gliomagenesis remains poorly understood. Here, we show that treatment with R-2HG induces cellular senescence in a p53-dependent manner. Furthermore, expression of mutated IDH1R132H or exposure to R-2HG, which leads to KDM4A inhibition, causes telomeric dysfunction. We demonstrate that KDM4A localizes to telomeric repeats and regulates abundance of H3K9(me3) at telomeres. We show that R-2HG caused reduced replication fork progression, and that depletion of SMARCAL1, a helicase involved in replication fork reversal, rescues telomeric defects caused by R-2HG or KDM4A depletion. These results establish a model whereby IDH1/2 mutations cause R-2HG-mediated inhibition of KDM4A, leading to telomeric DNA replication defects, telomere dysfunction, and associated genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/728014013df3b457cbd616bc46c4a758e940b3f8" target='_blank'>
              R-2-hydroxyglutarate-mediated inhibition of KDM4A compromises telomere integrity
              </a>
            </td>
          <td>
            Florence Couteau, Laurence M Gagné, Karine Boulay, Philippe Rousseau, Mélissa Carbonneau, Mary McQuaid, Jyoti Sharma, C. Sawchyn, Erlinda Fernandez, Dagmar Glatz, Rana Rizk, Marie-Eve Lalonde, Yosra Mehrjoo, Tsz Wai Chu, G. Moquin-Beaudry, Christian Beauséjour, Mikhail Sergeev, Santiago Costantino, D. Avizonis, I. Topisirovic, Nada Jabado, Hugo Wurtele, Chantal Autexier, Frédérick A. Mallette
          </td>
          <td>2025-06-06</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Aneuploidy is defined as the loss or gain of a whole chromosome or its region. Even at early stages of development, it usually leads to fatal consequences, including developmental defects/abnormalities and death. For a long time, it was believed that the disruption of gene balance results in pronounced effects at both the cellular and organismal levels, adversely affecting organism formation. It has been shown that the gene imbalance resulting from aneuploidy leads to proteotoxic and metabolic stress within the cell, reduced cell proliferation, genomic instability, oxidative stress, etc. However, some organisms have exhibited tolerance to aneuploidies, which may even confer adaptive advantages, such as antibiotic resistance in pathogenic fungal strains. A significant factor likely lies in the complexity of the tissue and organ organization of specific species. Polyploid organisms are generally more tolerant of aneuploidy, particularly those that have recently undergone whole-genome duplication. This review places special emphasis on the examination of sex chromosome aneuploidies in humans. In addition to primary effects, or cis effects (changes in the quantity of the transcripts of genes located on the aneuploid chromosome), aneuploidy can induce secondary or trans effects (changes in the expression levels of genes located on other chromosomes). The results of recent studies have prompted a reevaluation of the impact of aneuploidy on the structural-functional organization of the genome, transcriptome, and proteome of both the cell and the entire organism. Despite the fact that, in the cases of aneuploidy, the expression levels for most genes correlate with their altered copy numbers in the cell, there have been instances of dosage compensation, where the transcript levels of genes located on the aneuploid chromosome remained unchanged. The review presents findings from recent studies focused on compensatory mechanisms of dosage compensation that modify gene product quantities at post-transcriptional and post-translational levels, alleviating the negative effects of aneuploidy on cellular homeostasis. It also discusses the influence of extrachromosomal elements on the spatial organization of the genome and the changes in gene expression patterns resulting from their presence. Additionally, the review specifically examines cases of segmental aneuploidy and changes in copy number variants (CNVs) in the genome. Not only the implications of their composition are considered, but also their localization within the chromosome and in various compartments of the interphase nucleus. Addressing these questions could significantly contribute to enhancing cytogenomic diagnostics and establishing a necessary database for accurate interpretation of identified cases of segmental aneuploidy and CNVs in the genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/271a442ac782b355db581186dfcccbd9f3d50dd8" target='_blank'>
              From cytogenetics to proteogenomics: new horizons in the study of aneuploidies
              </a>
            </td>
          <td>
            K. Zadesenets, N. B. Rubtsov
          </td>
          <td>2025-06-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0065672d86f3f94440ecf420ec518fca8aa5a1ec" target='_blank'>
              Integrated Spatial Analysis of Ovarian Precancerous Lesions
              </a>
            </td>
          <td>
            Tu-Yung Chang, Yen-Wei Chien, Szu-Hua Chen, Annelise Sokolow, Yeh Wang, Brant G. Wang, Tricia A Numan, M. Herman Chui, Rebecca Stone, T. Pisanic, Nicholas Papadopoulos, Tian-Li Wang, Christopher Douville, Leslie Cope, I.-M. Shih
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Enhancers, as distal cis-regulatory elements in the genome, have a pivotal influence on orchestrating precise gene expression. Enhancer RNAs (eRNAs), transcribed from active enhancer regions, are increasingly recognized as key regulators of transcription. N6-methyladenosine (m6A), the most plentiful internal modification in eukaryotic mRNAs, has garnered significant research interest in recent years. With advancements in high-throughput sequencing technologies, it has been established that m6A modifications are also present on eRNAs. An accumulative body of evidence demonstrates that aberrant enhancers, eRNAs, and m6A modifications are intimately connected with carcinoma onset, progression, invasion, metastasis, treatment response, drug resistance, and prognosis. However, the underlying molecular mechanisms governing m6A modification of eRNAs in cancer remain elusive. Here, we review and synthesize current understanding of the regulatory roles of enhancers, eRNAs, and m6A modifications in cancer. Furthermore, we investigate the possible roles of eRNAs m6A modification in tumorigenesis based on existing literature, offering novel perspectives and directions for future research on epigenetic regulatory mechanisms in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6966deaca0f1174a84788609926a02bf29bca084" target='_blank'>
              Biological roles of enhancer RNA m6A modification and its implications in cancer
              </a>
            </td>
          <td>
            Yangyang Han, Jingqi Sun, Minghui Yao, L. Miao, Mengjia Li
          </td>
          <td>2025-05-30</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In mitosis the duplicated genome is aligned and accurately segregated between daughter nuclei. CTCF is a chromatin looping protein in interphase with an unknown role in mitosis. We previously published data showing that CTCF constitutive knockdown causes mitotic failure, but the mechanism remains unknown. To determine the role of CTCF in mitosis, we used a CRISPR CTCF auxin inducible degron cell line for rapid degradation. CTCF degradation for 3 days resulted in increased failure of mitosis and decreased circularity in post-mitotic nuclei. Upon CTCF degradation CENP-E is still recruited to the kinetochore and there is a low incidence of polar chromosomes which occur upon CENP-E inhibition. Instead, immunofluorescence imaging of mitotic spindles reveals that CTCF degradation causes increased intercentromere distances and a wider and more disorganized metaphase plate, a disruption of key functions of the pericentromere. These results are similar to partial loss of cohesin, an established component of the pericentromere. Thus, we reveal that CTCF is a key maintenance factor of pericentromere function, successful mitosis, and post-mitotic nuclear shape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/529b194fb855f3438e03aaee838ab3df0a5f9eb4" target='_blank'>
              CTCF maintains pericentromere function and mitotic fidelity
              </a>
            </td>
          <td>
            Erin Walsh, A. Stephens
          </td>
          <td>2025-06-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Background: Large-scale genomic profiling has cataloged the prevalence of single base substitution (SBS) mutational signatures associated with the activity of Apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) cytidine deaminases in breast cancer (BC). These mutational signatures are enriched in metastatic BC (mBC) compared to early tumors indicating an association with poor prognosis and a potential function in therapy resistance and disease progression. We sought out to investigate whether APOBEC3 mutational signatures can serve as biomarkers for poor treatment outcomes and if APOBEC3 mutagenesis-driven genomic instability can induce therapy resistance in mBC. Methods: We analyzed SBS mutational signatures in 3,880 BC samples with paired tumor-normal sequencing by the MSK-IMPACT assay using the SigMA algorithm. We utilized the detailed clinical annotation to assess the clinical characteristics of APOBEC3-dominant tumors including survival analyses on endocrine and targeted therapies. We generated cellular models of BC to investigate the molecular drivers of APOBEC3 mutagenesis and its function in promoting therapeutic resistance. We performed whole genome sequencing (WGS) of BC models and paired primary/metastatic patient samples to identify broader genomic alterations mediated by APOBEC3 activity. Results: Building on published results, we found that APOBEC3 mutational signatures were highly prevalent in all subtypes of BC and enriched in metastatic hormone receptor-positive (HR+) and triple-negative breast cancers (TNBC) compared to unmatched primary tumors (p < 0.0001 for HR+/HER2-, p < 0.01 for HR+/HER2+ and TNBC). APOBEC3 mutational signatures were independently associated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor combination therapy in patients with HR+ mBC (HR 1.5, 95% CI 1.2 - 1.8, p < 0.001). Expression of APOBEC3A (A3A) and APOBEC3B (A3B) enzymes generated APOBEC3-associated alterations including single nucleotide variants, copy number alterations (CNAs), and clustered mutations in a deamination-dependent manner, and promoted resistance acquisition of estrogen receptor-positive (ER+) BC cells to agents including an ER degrader and CDK4/6 inhibitors. In HER2+ cells, endogenous A3A-driven APOBEC3 activity was also necessary for faster resistance development against anti-HER2 therapies. WGS analyses of resistant models identified CNA events such as loss of heterozygosity in chromosome 13 exclusively in APOBEC3-positive cells. Upon exposure to the CDK4/6 inhibitor abemaciclib, these cells acquired APOBEC3-context truncating mutations in RB1 tumor suppressor gene, a well-characterized mechanism of resistance. Detailed analyses of WGS of five paired patient samples also highlighted acquired resistance-linked alterations such as PIK3CA E54XK mutations in APOBEC3-dominant tumors, which were corroborated in paired pre/post-treatment samples in our clinical cohort. The acquisition of APOBEC3-context alterations in APOBEC3-dominant samples highlights the causality of APOBEC3 mutagenesis in driving resistance-promoting changes. Lastly, comparison of mutational signatures in the paired cohort demonstrated pre-existence of APOBEC3 signatures in 75% of pre-treatment samples that became APOBEC3-dominant post-treatment illustrating that APOBEC3 mutagenesis can be active during early stages of BC. Conclusions: Our work reveals that APOBEC3 mutational signatures predict poor treatment outcomes of HR+ mBC. We demonstrate that APOBEC3 mutagenesis, primarily through the enzymatic activity of A3A and A3B, drives resistance to endocrine and targeted therapies by causing APOBEC3-context resistance-associated changes. We further show that the presence of APOBEC3 mutagenesis can be detected before therapy exposure and may therefore represent a valuable biomarker and therapeutic target.
 Citation Format: Avantika Gupta, Andrea Gazzo, Pier Selenica, Anton Safonov, Fresia Pareja, Edaise M. da Silva, David N. Brown, Yingjie Zhu, Juber Patel, Juan Blanco-Heredia, Bojana Stefanovska, Michael A. Carpenter, Xin Pei, Denise Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, Britta Weigelt, Nadeem Riaz, Simon N. Powell, Pedram Razavi, Reuben S. Harris, Jorge S. Reis-Filho, Antonio Marra, Sarat Chandarlapaty. APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr GS2-11.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b57625ce2cff9cbb61ff4f9d159b42b1e5eb578" target='_blank'>
              Abstract GS2-11: APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, B. Weigelt, N. Riaz, Simon Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b369f79b329c117da716bed78cecbcb615823b8f" target='_blank'>
              JAK2 signalling to HNRNPA1 represses retrotransposon activity in haematopoietic stem cells
              </a>
            </td>
          <td>
            Jeremy W. Deuel, Andrejs Makarovs, Sheidasadat Goosheh, George Winder, Hisham Mohammed, Jason S. Carroll, Michael Spencer Chapman, C. Kapadia, Peter J. Campbell, J. Nangalia, Elisa Laurenti, Constanze Schneider, T. Oellerich, Anthony R Green, S. Loughran
          </td>
          <td>2025-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Genetic lesions drive cancer development and progression, and understanding their origins will reveal the mechanisms of carcinogenesis. We showed that murine FAM72A promotes mutagenic DNA repair during antibody maturation by acting as a substrate adaptor of the CTLHMKLN1 E3 ligase to induce the proteasome degradation of Uracil DNA glycosylase 2 (UNG2), a pivotal enzyme of the base excision repair. In humans, the FAM72 gene has expanded to include four paralogues named FAM72A-D. Bioinformatic studies suggested that the human FAM72 genes are overexpressed in a broad range of cancers. However, the functional roles of FAM72A-D in human biology and cancer are unknown. Here, we show that FAM72 family members are minimally expressed in most healthy tissues except for thymus, and that FAM72A, B and D are overexpressed in primary tumorigenic tissues. Human FAM72 expression inversely correlates with UNG2 protein level in human cell lines and primary tumorigenic tissues suggesting that human FAM72 promotes UNG2 degradation. However, only FAM72A is able to bind to and induce UNG2 degradation in human cells. Our results suggest that the ability of FAM72A to induce UNG2 degradation contributes to neoplasia in a variety of cancer types by promoting mutagenic repair of genomic dUs. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-07723-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369de028cb26608e44c32340a1371447a2c75ba9" target='_blank'>
              FAM72A promotes UNG2 degradation and mutagenesis in human cancer cells
              </a>
            </td>
          <td>
            Yuqing Feng, P. Barbulescu, Chetan K Chana, Melika Shirdarreh, Hong Yang, Lizhen Wu, Sami Mamand, Mohammad Kashem, Amin Zia, Ming Han, Julissa Tsao, Trevor Pugh, David Cescon, David G. Schatz, Frank Sicheri, Alberto Martin, R. Pezo
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="For chromosome abnormalities (CA), such as Down syndrome (DS), the influence of genomic variations to chromosome conformation and gene transcription remains elusive. Based on the complete genomic sequence from the parents of the DS trisomy patient, we systematically delineated an atlas of parental-specific haploid single nucleotide polymorphism (SNP), copy number variation (CNV), three-dimensional (3D) genome, and RNA expression profiles of the diencephalon in the DS patient. The integrated haploid multi-omics analysis demonstrated that one-dimensional genomic variations including SNPs and CNVs in the DS patient are highly correlated with the alterations of the 3D genome and the subsequent gene transcription. The correlation relationship remains valid in haploid-levels. Moreover, we revealed the 3D genome alteration associated mis-regulation of DS-related genes, which facilitates to understanding the pathogenesis of CA. Together, our study contributes to decipher the coding from one-dimensional genomic variations to 3D genomic architecture and the subsequent gene transcription in healthy and diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33d9de44c4aa76967640e3414bc7acd3b2037c35" target='_blank'>
              Deciphering Haploid Chromosome Conformation Alternation in Down Syndrome by Multiple Haploid Omics Analysis.
              </a>
            </td>
          <td>
            Chengchao Wu, Tianshu Zhou, Wenfu Ke, Wei Xiong, Zhihui Zhang, Siheng Zhang, Jinyue Wang, Lulu Deng, Keji Yan, Man Wang, Shenglong He, Qi Gong, Chao Ma, Xiaping Chen, Yan Li, He Long, Chong Guo, Gang Cao, Zhijun Zhang
          </td>
          <td>2025-06-12</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="The human genome contains regulatory DNA elements, enhancers, that can activate gene transcription over long chromosomal distances. Here, we show that enhancer distance can be critical for gene silencing. We demonstrate that linear recruitment of the normally distal strong HBB enhancer to developmentally silenced embryonic HBE or fetal HBG promoters, through deletion or inversion of intervening DNA sequences, results in their strongly reactivated expression in adult erythroid cells and ex vivo differentiated hematopoietic stem and progenitor cells. A similar observation is made in the HBA locus, where deletion-to-recruit of the distal enhancer strongly reactivates embryonic HBZ expression. Overall, our work assigns function to seemingly non-regulatory genomic segments: by providing linear separation they may support genes to autonomously control their transcriptional response to distal enhancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d408a95c592c57473a14d7c2700a606f3cb349c" target='_blank'>
              Reactivation of developmentally silenced globin genes through forced linear recruitment of remote enhancers.
              </a>
            </td>
          <td>
            Anna-Karina Felder, Sjoerd J D Tjalsma, H. Verhagen, Rezin Majied, M. Verstegen, Thijs C. J. Verheul, Jeffrey van Haren, Rebecca Mohnani, Richard Gremmen, Peter H. L. Krijger, S. Philipsen, Emile van den Akker, W. de Laat
          </td>
          <td>2025-06-02</td>
          <td>Blood</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="PARP inhibitors (PARPi), recently introduced for treating metastatic castration-resistant prostate cancer (mCRPC), have heightened interest in molecular profiling for pathogenic aberrations in homologous recombination DNA repair (HRR) genes in all mCRPC patients. Liquid biopsy offers a viable alternative to archival tumor tissue for genetic analysis. In this study, we assessed the feasibility and utility of combining mutational panel sequencing with shallow whole genome sequencing (sWGS) to refine HRR status determination from plasma in prostate cancer (PCa) patients. The mutational profile of 16 HRR genes was assessed in 63 PCa patients: 28.6% of patients harbored putative pathogenic variants in HRR-related genes. A HRR-mutant status was defined for 10 patients (15.8%). Through the integration of sWGS data, plasma samples non-informative about somatic alterations were identified, and germline/somatic origin of HRR mutations was defined. Matched tumor tissue was available for 41 patients, with an 85.7% concordance rate between plasma and tissue mutational analyses. Additionally, we explored the copy number variation (CNV) profile using sWGS and it was found concordant with the literature PCa profiles. Our findings demonstrated that ctDNA analysis through liquid biopsy is a reliable alternative to tissue-based methods for identifying SNVs and CNVs. However, concordance was affected by ctDNA levels in plasma and clonal hematopoiesis. The data highlight the utility of integrating sWGS with targeted mutation analysis for comprehensive molecular profiling of PCa patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05384-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad00381f62d746252d19dc78da101336888d0016" target='_blank'>
              Mutational and low-coverage whole genome sequencing identifies actionable DNA repair alterations in prostate cancer plasma DNA
              </a>
            </td>
          <td>
            A. Virga, Milena Urbini, Maurizio Polano, E. Petracci, G. Tedaldi, G. Gurioli, G. Marisi, D. Angeli, Andrea Ambrosini-Spaltro, Giovanni De Luca, Ilaria Cangini, Valentina Zampiga, Giovanna Cenacchi, Giuseppe Toffoli, C. Casadei, M. C. Cursano, V. Conteduca, U. de Giorgi, Paola Ulivi
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Abstract Aurantiochytrium limacinum holds great promise for producing sustainable single-cell oil as an alternative to fish oil. However, research into its complex biological and biochemical characteristics and efforts toward strain improvement have been hampered by insufficient genetic tools. Until now, genetic transformations of A. limacinum have relied solely on chromosome integration, which is inefficient and prone to insertional mutagenesis and other issues related to genetically modified organisms (GMOs). This paper describes the first centromeric plasmid for A. limacinum. Amplification of this shuttle vector by E. coli enables direct delivery into A. limacinum via electroporation, where it undergoes stable replication and segregation into daughter cells. The key to the stable plasmid maintenance lies in a 500 bp segment derived from chromosome 24 of Phaeodactylum tricornutum. While this segment does not significantly enhance the efficiency of vector transformation, it enables the replication and maintenance of the shuttle vector in the host cell as closed circular DNA. The plasmid from three transformants demonstrates a high segregation efficiency of 96.8 ± 0.3% (n = 3), even in the absence of antibiotic selection. This novel centromeric plasmid considerably enhances the flexibility of genetic manipulations and gene expression in A. limacinum, opening new avenues for its study and industrial application. Key points • First centromeric plasmid developed for genetic transformation in A. limacinum. • The novel plasmid enhances flexibility in genetic manipulation and gene expression. • The plasmid achieves 96.8 ± 0.3% (n = 3) segregation efficiency without antibiotic selection. Supplementary Information The online version contains supplementary material available at 10.1007/s00253-025-13527-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e1a22ec96ed65caa34d28f985f7648b9334af03" target='_blank'>
              Novel centromeric plasmid for stable extrachromosomal gene expression in Aurantiochytrium limacinum
              </a>
            </td>
          <td>
            P. P. Renta, Cian-Huei Syu, Ta-Yu Huang, Yi-Ting Chang, Yu-Feng Liang, Ssu-Ting Chen, Po-Wei Weng, Ming-Chen Hsu, Keng-Hung Lin, Tsu-tung Liu, Anna C-C Jang, Che-Chia Tsao, Han-Jia Lin, Hung-Yun Lin, Yi-Min Chen
          </td>
          <td>2025-07-04</td>
          <td>Applied Microbiology and Biotechnology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The identification of recurrent oncogenic drivers has enabled targeted therapeutic strategies for subsets of non-small cell lung carcinoma (NSCLC) and other malignancies. Oncogenic fusions involving the RAS-activating guanine exchange factor (GEF) RASGRF1 are reported in multiple tumors, but their prevalence and genetic heterogeneity remain undefined. Here, we query RNA-seq data from a real-world database of diverse human malignancies and identify 40 tumors with rearrangements involving RASGRF1 or the related RASGRF2 predicted to generate chimeric proteins. Half of these fusions occur in NSCLC, pancreatic cancer, and melanoma and are enriched in tumors without other established driver alterations. A subset of RASGRF fusions contains transmembrane partners, and membrane localization enhances RAS activation and transforming activity. Loss of N-terminal PH1 and DH domains in RASGRF fusions also promotes transformation. Although some fusions lack the PH1 but not the DH domain, our functional assays indicate that loss of the PH1 domain alone is insufficient to drive cellular transformation. Our findings provide insights about the tissue distribution, structural diversity, and oncogenic mechanisms of RASGRF fusions. As cell models driven by these fusions are sensitive to MAPK pathway inhibition, oncogenic RASGRF fusions may represent a therapeutic target in rare molecular subsets of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bb9a75bf2438a8363d43341f77587497989fb9b" target='_blank'>
              Genetic and oncogenic features of RASGRF fusions
              </a>
            </td>
          <td>
            Sreya Das, Daniel S Lenchner, Ellen Jaeger, Lisa Hunihan, Dana F DeSantis, Stamatina Fragkogianni, Karyn Ronski, Frederick H Wilson
          </td>
          <td>2025-07-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Understanding the role of cancer hotspot mutations is essential for unraveling mechanisms of tumorigenesis and identifying therapeutic vulnerabilities. Correcting cancer mutations with base editing is a novel, yet promising approach for investigating the biology of driver mutations. Results Here, we present a versatile platform to investigate the functional impact of cancer hotspot mutations through adenine base editing in combination with transcriptomic profiling. Using this approach, we correct TP53 hotspot mutations in cancer cell lines derived from diverse tissues, followed by mRNA sequencing to evaluate transcriptional changes. Remarkably, correcting these mutations not only reveals the dependency on mutant allele expression but also restores highly conserved tumor-suppressive transcriptional programs, irrespective of tissue origin or co-occurring mutations, highlighting a shared p53-dependent regulatory network. Our findings demonstrate the utility of this base editing platform to systematically interrogate the functional consequences of cancer-associated mutations and their downstream effects on gene expression. Conclusions This work establishes a robust framework for studying the transcriptional dynamics of cancer hotspot mutations and sheds light on the conserved biological processes reinstated by p53 correction, offering potential avenues for future targeted therapies. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03667-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d441d58b01f4910f8c55a860db9d7cbefe24b5" target='_blank'>
              A base editing platform for the correction of cancer driver mutations unmasks conserved p53 transcription programs
              </a>
            </td>
          <td>
            Pascal Wang, Rituparno Sen, Frank Buchholz, Shady Sayed
          </td>
          <td>2025-07-22</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eded65f17b590b88c69cd9abd535f6f282a95b44" target='_blank'>
              The mutational dynamics of the Arabidopsis centromeres
              </a>
            </td>
          <td>
            Xiao Dong, Wenbiao Jiao, J. A. Campoy, Fernando A. Rabanal, Jurriaan Ton, Lisa M Smith, Detlef Weigel, Korbinian Schneeberger
          </td>
          <td>2025-06-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="
 e16580


 Background:
 Non-muscle invasive bladder cancer (NMIBC) presents a significant clinical challenge due to its high recurrence rate. Recently, a deeper understanding of tumor epigenetics has provided new opportunities for cancer detection and treatment. While significant efforts have been made to understand NMIBC biology, the molecular and epigenetic mechanisms driving its progression remain elusive.
 Methods:
 We used multiomics analysis including expression, DNA methylation, and cell-free DNA chromatin immunoprecipitation sequencing (cfChIP-seq) assays to uncover the molecular and epigenetic landscape of NMIBC from human tissue samples and urine.
 Results:
 High-throughput sequencing revealed more than 6,000 differentially methylated regions (DMRs) within regulatory elements, many of which were linked to cancer-related pathways and exhibited distinct signatures in cancer tissues compared to healthy controls. Next, we distinguished high and low-grade tumors based on unique DNA methylation profiles. These patterns correlated with transcription level of genes associated with cell cycle regulators, consistent with the higher proliferative capacity of high-grade tumors. Moreover, Among the DMRs, we also identified genes involved in T cell regulation. Supporting this, we demonstrated reduced CD8+ T cells infiltration in high-grade tumors, further highlighting immune dysregulation in the more aggressive disease. We extended our investigation to urine samples, performing cfChIP-seq on samples from healthy donors and NMIBC patients. By using the H3K4me3 promoter histone mark on cfDNA we could detect cell origin. Utilizing this assay, we were able to identify changes in the urine of NMIBC patients, owing to cf-nucleosome contributions from both the tumor and immune cells. Additionally, we stratified the cancer cohort by tumor grade and found enrichment in the number of differentially marked genes in high-grade tumors.
 Conclusions:
 Our detection of unique epigenetic signatures offer opportunities for biomarker development, enabling early detection through non-invasive urine based diagnostic approach. This could grade tumor more accurately and tailor personalized patient care and monitoring.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38dd958dabb53d9b68b4717906734c6068947417" target='_blank'>
              Elucidating the epigenetic landscape of non-muscle invasive bladder cancer.
              </a>
            </td>
          <td>
            T. Falick Michaeli, Batia Azria, M. Lotem, Nir Friedman
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef02803fbaf5cb1abbd79351dc6a4ca1c68ba79" target='_blank'>
              Constitutional copy number amplifications: rare or under-evaluated? Revisiting a 25-year-old cold case.
              </a>
            </td>
          <td>
            Eliana Salvo, Romano Tenconi, R. Giorda, Sara Bertuzzo, Luca Cesana, Roberta Murru, S. Giglio, Mana M Mehrjouy, Niels Tommerup, Orsetta Zuffardi, M. Bonaglia
          </td>
          <td>2025-06-04</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is a highly aggressive malignancy with extremely poor prognosis. SCLC cells exhibit high plasticity and can progress from neuroendocrine (NE) to non-NE phenotypes. This dynamic evolution promotes treatment resistance and relapses, representing a challenge for targeted therapies in this elusive disease. Here we identify the transcription factor ONECUT2 (OC2) as a driver of plasticity in SCLC, leading to non-NE transcriptional states. OC2 is highly expressed in SCLC tumors compared to normal lung tissue and its expression is associated with heightened clinical stage and lymph node metastasis. We show that OC2 is a repressor of ASCL1, the NE master regulator transcription factor. In addition, OC2 upregulates non-NE programs through activation of c-MYC and Notch signaling. We also demonstrate that OC2 is required for growth and survival of SCLC cells and that it can be targeted with a small molecule inhibitor that acts synergistically with the standard combination of cisplatin and etoposide, providing a novel therapeutic strategy for OC2 active SCLC tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01267-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736f50c299b3fe9c13bfbd3d67e9601f1567eb2" target='_blank'>
              ONECUT2 reprograms neuroendocrine fate and is an actionable therapeutic target in small cell lung cancer
              </a>
            </td>
          <td>
            Mirian Gutiérrez, Irene Zamora, Raquel Iriarte, M. Pajares, Qian Yang, Chen Qian, N. Otegui, J. Fernández-Irigoyen, Enrique Santamaria, N. Alcala, A. Sexton-Oates, L. Fernandez-Cuesta, Miguel Barajas, Alfonso Calvo, Luis Montuenga, Beatrice S. Knudsen, Sungyong You, Michael R. Freeman, Ignacio J. Encío, Mirja Rotinen
          </td>
          <td>2025-06-11</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background DNA methylation plays a crucial role in cancer development and progression and has been linked to genetically and clinically distinct tumor classes, including IDH-mutated and IDH-wildtype adult-type diffuse gliomas. Here, we identify a CpG-island methylator phenotype (CIMP) that characterizes the receptor tyrosine kinase 2 (RTK2) subtype of IDH-wildtype glioblastoma. Results This RTK2-CIMP affects genomic locations and cell functions distinct from those of IDH mutation-associated IDH-CIMP and suppresses the expression of its target genes. The RTK2-CIMP-region chromatin is characterized by a combination of repressive and activating marks, including polycomb-associated H3K27me3 and enhancer-associated H3K4me1, consistent with DNA methylation-mediated silencing of genes with bivalent-state promoters in neural progenitor cells. Functionally, RTK2-CIMP affects neuronal lineage genes and is significantly associated with astrocyte-like glioblastoma, suggesting that RTK2-CIMP is an epigenetic signature of the astrocyte-like cell state. Furthermore, we demonstrate that RTK2-CIMP can be induced by genetic manipulation in glioblastoma cells. Conclusions Our results suggest that RTK2-CIMP is a key contributor to cell-state plasticity in glioblastoma. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03670-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94bee397b61a1f7011617f82ad34a983c326fca3" target='_blank'>
              A new IDH-independent hypermethylation phenotype is associated with astrocyte-like cell state in glioblastoma
              </a>
            </td>
          <td>
            Ana Luisa Costa, Daria Doncevic, Yonghe Wu, Lin Yang, Ka-Hou Man, Anna-Sophie Spreng, Hannah Winter, M. Fletcher, B. Radlwimmer, Carl Herrmann
          </td>
          <td>2025-07-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Retinoblastoma (RB) is the most common malignant eye tumor in children. Cytotoxic drugs like melphalam often cause significant side effects and acquired resistance in RB treatment. Here, the impact of Paraneoplastic Ma1 (PNMA1) on RB progression and chemotherapy resistance was investigated. A series of bioinformatics methods were used for differential expression genes and drug resistance genes screening. Western blot was performed for the detection of genes expression. CCK-8 and EdU staining were assessed for cell viability. Clone formation and TUNEL staining were examined for cell viability and apoptosis. Wound healing and transwell assays were analyzed for the cell migrated and invasive abilities. Immunofluorescence was observed for γ-H2AX expression. Colorimetric PARP/Apoptosis Assay was used for PARP activity. PNMA1 was upregulated in RB tumor tissues, and high expression of PNMA1 promotes RB cells’ chemoresistance to melphalam, while downregulation of PNMA1 reversed chemoresistance in resistant cells. Furthermore, the mechanism by which PNMA1 mediated chemoresistance to melphalam, discovered that PNMA1 might facilitate melphalam resistance through the c-myc-mediated DNA damage repair pathway. PNMA1 functions as a tumor-promoting factor in retinoblastoma and may regulate melphalam resistance through the upregulation of the c-myc gene and the involvement in DNA damage repair. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-12206-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0daf04e5f29491e7cfbbb6a4daba0a96d6c93b3" target='_blank'>
              PNMA1 mediates melphalam chemoresistance in retinoblastoma via promoting DNA repair
              </a>
            </td>
          <td>
            Longge Sun, Zhaokang Xu, Xin Zhang, Xing Du, Guanghui Li, Yuanyuan Guo
          </td>
          <td>2025-07-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Heterogeneity in cancer gene expression is typically linked to genetic and epigenetic alterations, yet post-transcriptional regulation likely influences these patterns as well. However, the quantitative contribution of post-transcriptional mechanisms to cancer transcriptome dynamics remains unclear. Here, we systematically measured mRNA dynamics across diverse breast cancer models, revealing that mRNA stability significantly shapes gene expression variability. To decipher the regulatory grammar underlying these dynamics, we developed GreyHound, an interpretable multimodal deep-learning framework integrating RNA sequence features and RNA-binding protein (RBP) expression. GreyHound identified an extensive network of RBPs and their regulons underlying variations in mRNA stability. Among these, we uncovered a metastasis-suppressive regulatory axis centered on the RNA-binding protein RBMS3 and its post-transcriptional control of the redox regulator TXNIP. Functional and molecular analyses revealed that RBMS3 depletion resulted in targeted transcript destabilization, which was associated with poor clinical outcomes and enhanced metastatic potential in xenograft models. Using in vivo epistasis studies, we confirmed that RBMS3 regulation of TXNIP mRNA stability drives this metastasis-suppressive program. These findings position the RBMS3-TXNIP regulatory axis as a key post-transcriptional mechanism in breast cancer and illustrate how interpretable models of RNA dynamics can uncover hidden regulatory programs in disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a157de4d34fa592e866b9d4ddb110508f65adde6" target='_blank'>
              Integrative analysis of mRNA stability regulation uncovers a metastasis-suppressive program in breast cancer
              </a>
            </td>
          <td>
            Heather Karner, Tabea Mittmann, Vicky W. Chen, Ashir A. Borah, Andreas Langen, Hassan Yousefi, L. Fish, Balyn W. Zaro, A. Navickas, Hani Goodarzi
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 11561


 Background:
 Liposarcoma (LPS) is characterized by unstable genomes and high occurrence of gene fusions. Hotspots of both recurrent and non-recurrent gene fusions can provide information about structural alterations in certain LPS subtypes. For instance, myxoid (M) LPS expresses the oncogenic
 FUS-DDIT3
 protein and is known to be fusion-driven. Moreover, disrupted locus 12q13-15 is an important feature of well-dedifferentiated (WD) and dedifferentiated (DD) LPS, and is the site of copy number alterations (CNAs) and gene fusions. Here, we used RNA sequencing to uncover hotspots of intra- and transchromosomal gene fusions in LPS patient samples, identifying potentially clinically relevant events in certain chromosomal regions.
 Methods:
 The BostonGene internal LPS cohort (n=150) was analyzed by bulk whole-transcriptome sequencing, using STAR-fusion for sequence calling. Quality control was performed using FastQC, FastQ Screen, RSeQC, and MultiQC. Tumor purity was assessed via pathological and bioinformatics examination with a threshold of 20%. One sample Poisson rate test was used to evaluate statistical significance of gene fusion hotspots.
 Results:
 We identified 4,080 gene fusions among four LPS subtypes (DDLPS, WDLPS, MLPS, and PLPS - pleomorphic LPS). Of those, 2,302 (56.4%) were intrachromosomal and 1,778 (43.6%) were transchromosomal. Over half of these fusions (1,263/2,302 intrachromosomal fusions, 54.9%; 1,047/1,778 transchromosomal fusions, 58,9%) were detected on chromosome 12. Most identified fusions occurred in the 12q13-15 region (q-value < 0.001), with q15 being especially prevalent in transchromosomal fusions (q-value < 0.001). The most prevalent recurrent fusion across our LPS cohort was
 FUS-DDIT3
 (N=21, MLPS samples). Another notable recurrent fusion was
 TRIO-TERT
 (N=4, DDLPS). We also identified extended gene fusion hotspots in regions containing important oncogenes such as
 MDM2
 and
 FRS2
 in LPS subtypes (DDLPS, WDLPS, PLPS) that are not considered fusion-driven (Table). These findings suggest an oncogenic role of such fusions in these LPS subtypes, along with known CNAs like
 MDM2
 amplification.
 Conclusions:
 Our comprehensive transcriptomic analysis of gene fusions in LPS samples uncovered both new and established hotspots of chromosomal rearrangements. Identification of such hotspots improves our understanding of LPS oncogenesis and thus can enhance the diagnostic accuracy and discovery of new biomarkers.


 Hotspots of gene fusions in liposarcoma.




 Hotspot cytoband
 Diagnosis
 q-value
 Important genes




 1q23.3
 DDLPS
 < 0.001

 ATF6



 1q24.3
 DDLPS, WDLPS
 < 0.001

 DNM3



 12q13.3
 MLPS
 < 0.001

 DDIT3



 12q14.1
 DDLPS, WDLPS
 < 0.001

 CDK4



 12q14.3
 DDLPS, WDLPS
 < 0.001

 HMGA2, YEATS4



 12q15
 DDLPS, WDLPS, PLPS
 < 0.001

 MDM2, FRS2, CPM



 16p11.2
 MLPS
 < 0.001

 FUS





">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df3d585e6b1b252046eb1085a1fdfa70c3a17980" target='_blank'>
              Detecting hotspots of intra- and transchromosomal fusions in liposarcomas by RNA sequencing.
              </a>
            </td>
          <td>
            Dmitrii Grachev, Danil Ivanov, Oleg Baranov, Vladimir Kushnarev, Melissa Clemons, Sheila T. Yong, N. Kotlov, Konstantin Chernyshov, A. Bagaev, N. Fowler, A. Conley, G.M. Cote, Sant P Chawla
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 10615


 Background:
 ATM
 is a tumor suppressor gene involved in DNA repair and telomere maintenance.
 ATM
 biallelic pathogenic or likely pathogenic germline variants (gPV) are associated with

 ataxia-telangiectasia syndrome. Monoallelic ATM gPV are associated with increased cancer risk; however, their contribution to carcinogenesis has not been elucidated. We sought to characterize the genomic landscape of gPV
 ATM
 -associated cancers in a pan-cancer cohort.
 Methods:
 ATM
 alterations (germline and somatic) were identified in patients (Pts) with a solid tumor diagnosis sequenced with MSK-IMPACT, an FDA-approved, tumor-normal paired targeted NGS. Genetic-imputed ancestry clinicopathologic characteristics and FACETS estimated allele-specific copy number profiles were evaluated. Samples with purity <0.2 or with undetermined copy number profiles were excluded. HRD score was evaluated using FACETS profiles. Both somatic and germline sequencing data were analyzed within protocol NCT01775072.
 Results:
 Among 40,136 Pts with cancer who underwent germline testing, 1.1% (n=442) harbored an
 ATM
 gPV, inclusive of 2 Pts with biallelic
 ATM
 gPVs. The most frequent mutations were: R2547_S2549del (n=17), K2756* (n=14) and V1268* (n=11). Among these, R2547_S2549del and K2756* were identified exclusively in Pts with European ancestry, while E343Ifs*2 (n= 9/10) and c.1065+1G>T (n= 6/7) were most prevalent in Pts with Ashkenazi Jewish (AJ) ancestry. Concomitant gPVs in other genes were observed in 10% (n=47), with the most frequent ones being
 APC
 (I1307K),
 MUTYH
 ,
 BRCA2, CHEK2
 , and
 BRCA1
 . 63,270 out of 86,039 tumor samples had usable FACETS profiles.
 ATM
 somatic allele-specific information was available for 67% (297/442) of Pts. 62% (197/317) of tumor samples from Pts with monoallelic
 ATM
 gPVs exhibited somatic biallelic (Bi)
 ATM
 inactivation, while 38% (120/317) retained monoallelic (Mono) ATM status. The underlying mechanism of Bi-ATM inactivation was loss of heterozygosity in 75% (149/197) and additional somatic
 ATM
 mutations in 25% (48/197). Samples from Pts with tumors known to be associated with ATM gPVs (Breast, Pancreatic and Prostate cancers) had significant enrichment in Bi-ATM inactivation compared to tumors without strong association with ATM gPVs (88% Vs 58%; p<0.01). Although samples with Bi-
 ATM
 had higher overall HRD-scores compared to Mono-
 ATM
 (median = 34 Vs 20; p<0.01), no significant enrichment in HRD-High phenotype was seen in Bi-
 ATM
 (23 Vs 18; p=0.49).
 Conclusions:
 Evaluation of a pan-cancer Pt population with
 ATM
 gPVs demonstrated a high prevalence of biallelic somatic inactivation. Most Pts with
 ATM
 gPVs who had malignancies implicated in
 ATM
 -associated cancer risk, had biallelic somatic inactivation in their tumors suggestive of their contribution to tumorigenesis. While biallelic ATM inactivation is associated with higher genomic instability, no enrichment in HRD phenotype was observed.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/171b6148e94d64a55d3944e70772bbb94d153ffa" target='_blank'>
              Clinicopathologic and allele-specific analysis of germline
 ATM
 alterations in a pan-cancer cohort.
              </a>
            </td>
          <td>
            Matteo Repetto, Daniel Muldoon, P. Selenica, Kanika Arora, Ying L. Liu, M. Carlo, Lauren Banaszak, Alicia J Latham, C. Bandlamudi, E. O'Reilly, M. E. Robson, B. Rousseau, D. Mandelker, A. Drilon, Britta Weigelt, K. Offit, Michael F. Berger, Y. Murciano-Goroff, Z. Stadler, Mohammad Abbass
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="Background Precursor lesions like polyps and adenomas in the colon commonly precede colorectal cancer (CRC), advancing through the “normal-polyp-adenoma-carcinoma” sequence towards malignancy. Yet, the cellular heterogeneity and molecular mechanisms involved in CRC development remain inadequately characterized. Methods To understand the molecular mechanisms driving the onset and progression of CRC, we conducted a comprehensive analysis of ten clinical colorectal samples representing sequential pathological stages using single-cell RNA sequencing (scRNA-seq). Validation was performed through immunofluorescence and immunohistochemistry analyses in a separate human colorectal tissue cohort. Additional verification was carried out using bioinformatics analyses of public TCGA and GEO datasets. Results Our comprehensive analyses not only reveal the cellular diversity and transcriptomic differences throughout disease progression but also highlight the importance of leveraging ligand–receptor gene expression to distinguish various cell subtypes. Subsequent examination and validation with a larger sample cohort uncover the specific involvement of ligand–receptor genes, transcription factors, immunoglobulin genes, and heat shock genes in regulating immune responses and microenvironment changes during colorectal tumorigenesis. Conclusions Our extensive transcriptome dataset provides valuable insights and acts as a fundamental resource to deepen our understanding of the complex molecular landscape in CRC. This dataset facilitates improved diagnostic accuracy and the creation of more personalized therapeutic approaches. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06785-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70e15cc889bc4d3cf181a1a8c15dbc72bc14a7b3" target='_blank'>
              A dynamic molecular landscape in colorectal cancer progression at single-cell resolution
              </a>
            </td>
          <td>
            Jianwen Sheng, Nianshuang Li, Shuai Li, Yuman Ye, Yuanhang Liao, Yupeng Zhang, Huan Chen, Mengxian Tu, Guping Zhong, Chaoliang Xiong, Pan Zheng, Yuting Lei, Zekun He, Xingxing He, Yao Zhang, Lei Wang, Xiaolin Gao, Yin Zhu, Jianping Liu, Huizhen Fan
          </td>
          <td>2025-07-01</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract MYC proteins are potent oncoproteins that drive tumorigenesis in a wide range of cancers, making it critical to understand their oncogenic functions and underlying mechanisms. Although MYC overexpression induces transcriptional and replication‐associated stress, recent studies have paradoxically identified MYC as a key resilience factor that protects cancer cells from these stressors. In this review, we explore the dual role of MYC in both driving and mitigating cellular stress to achieve its oncogenic function. We also examine how MYC‐induced transcriptional and replicative stress generates potentially immunogenic nucleic acid species while simultaneously helping cancer cells evade host immune recognition. We propose a model in which MYC plays a critical role in managing the stress it induces, thereby maintaining a balance that promotes tumor growth. Based on this model, we discuss potential therapeutic strategies targeting MYC‐dependent stress responses, offering new avenues for cancer treatment and highlighting the complexity of MYC‐driven oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78ce286c658baf7f61e9c844f34e8e87797cdc79" target='_blank'>
              MYC: The Guardian of Its Own Chaos
              </a>
            </td>
          <td>
            A. Gaballa, Bastian Krenz, Leonie Uhl
          </td>
          <td>2025-06-09</td>
          <td>Bioessays</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Oncogenic viruses typically manipulate host cellular mechanisms to drive tumorigenesis. They exploit pioneering transcription factors to modify gene expression, enabling uncontrolled proliferation. These viruses alter chromatin accessibility and induce chromatin remodelling, disrupting DNA repair and promoting viral genome integration. Additionally, epigenetic reprogramming through mechanisms like DNA methylation and histone modifications silences tumor suppressor genes and activates oncogenes. Understanding these mechanisms is critical for identifying more improved therapeutic targets, improving diagnostics, and predicting disease progression. Advances in this field can guide the development of innovative treatments and early detection tools. This comprehensive review synthesizes existing knowledge on the contributions of oncogenic viruses such as hepatitis B virus (HBV), hepatitis C virus (HCV), human papillomavirus (HPV), and human T-cell leukaemia virus type 1 (HTLV-1), Epstein–Barr virus (EBV), human herpesvirus 8 (HHV-8), and Merkel cell polyomavirus (MCV) to cancer development, highlighting their therapeutic relevance and driving forward research in viral oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36b6df879c24dbe78b1949e0a2207cf7cabf08ee" target='_blank'>
              The role of pioneering transcription factors, chromatin accessibility and epigenetic reprogramming in oncogenic viruses
              </a>
            </td>
          <td>
            M. Kgatle, S. Mbambara, Leon Khoza, Olalekan Fadebi, T. Mashamba-Thompson, M. Sathekge
          </td>
          <td>2025-06-16</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c37e8efeea3f80c12980365d094cd561d8a7d326" target='_blank'>
              LINE-1 ORF1p is a shared and immunogenic antigen in cancer
              </a>
            </td>
          <td>
            Wilson McKerrow, Megan Snyder, Chong Chu, Deena Kelly, Heike Keilhack, Liyang Diao
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2944900d47d857ea99c57a79dff86ef949dfd7c" target='_blank'>
              Genomic alterations in the YAP/TAZ pathway are associated with stem cell-like castration-resistant prostate cancer
              </a>
            </td>
          <td>
            M. Roskes, A. Martínez-Fundichely, S. Cohen, M. Balaban, C. K. Wong, W. Li, T. A. Gonzalez, A. Tehim, H. Xu, S. ElNaggar, M. Myers, R. Bareja, P. Dorsaint, K. Gorski, M. Asad, M. A. Assaad, B. D. Robinson, M. Sigouros, E. Barnett, J. Manohar, S. Tagawa, D. Nanus, A. Molina, J. Nauseef, C. N. Sternberg, J. Mosquera, H. Scher, A. Sboner, B. Raphael, Y. Chen, E. Khurana
          </td>
          <td>2025-07-02</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Aging and cancer, though distinct biological processes, share overlapping molecular pathways, particularly in epigenetic regulation. Among these, DNA methylation is central to mediating gene expression, maintaining cellular identity, and regulating genome stability. This review explores how age-associated changes in DNA methylation, characterized by both global hypomethylation and focal hypermethylation, contribute to the emergence of cancer. We discuss mechanisms of DNA methylation drift, the development of epigenetic clocks, and the role of entropy and epigenetic mosaicism, in aging and tumorigenesis. Emphasis is placed on how stochastic methylation errors accumulate in aging cells and lead to epiallelic shifts and gene silencing, predisposing tissues to malignant transformation, even despite recently increased cancer incidences at younger ages. We also highlight the translational potential of DNA methylation-based biomarkers, and therapeutic targets, in age-related diseases. By framing cancer as a disease of accelerated epigenetic aging, this review offers a unifying perspective and calls for age-aware approaches to both basic research and clinical oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e80ee6ccb4c2cbc13d84b76cf48a4e779ceb4234" target='_blank'>
              DNA Methylation, Aging, and Cancer
              </a>
            </td>
          <td>
            Himani Vaidya, Jaroslav Jelinek, Jean-Pierre J. Issa
          </td>
          <td>2025-06-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Homologous recombination is a high-fidelity DNA repair mechanism essential for maintaining genome stability. Impairment of this pathway, often due to BRCA1 or BRCA2 inactivation, leads to homologous recombination deficiency (HRD), forcing cells to rely on error-prone mechanisms for repairing DNA double-strand breaks, such as nonhomologous or microhomology-mediated end joining. HRD is a clinically important biomarker, particularly in breast and ovarian cancers, as it predicts responsiveness to platinum-based chemotherapies and PARP inhibitors. However, current tests in the clinical setting, mostly based on targeted panel sequencing, lack specificity and lead to a substantial number of false positives. In contrast, whole-genome sequencing, despite its high accuracy, remains largely confined to research because of high costs and logistical constraints. In this issue of Cancer Research, Abbasi and colleagues present HRProfiler, a machine learning-based tool that accurately detects HRD using whole-exome sequencing (WES) data, which is increasingly accessible in clinical oncology. Notably, it demonstrates improved sensitivity in the WES setting compared with existing tools, such as HRDetect and SigMA. As WES continues to gain traction, HRProfiler offers a promising step toward democratizing HRD detection and enabling more precise, genomics-guided treatment strategies. This article is part of a special series: Driving Cancer Discoveries with Computational Research, Data Science, and Machine Learning/AI. See related article by Abbasi et al., p. 2504.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab81bfb225ffd3e5881d30b8dbaccdbb884d86b5" target='_blank'>
              Leveraging Whole-Exome Sequencing and Mutational Signatures to Detect Homologous Recombination Deficiency in Cancer.
              </a>
            </td>
          <td>
            Joonoh Lim, Young Seok Ju
          </td>
          <td>2025-07-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Breast cancer is the most common malignancy that threaten women's health seriously. Many studies have shown that long noncoding RNAs can play significant role in the tumorigenesis of breast cancer. By analyzing the TCGA breast cancer genome data and transcriptome data, we found that copy number amplification drives the activation of long noncoding RNA MNX1-AS in breast cancer and indicates poor prognosis. Functionally, MNX1-AS1 could regulate pathogenesis of breast cancer in vitro and in vivo. Mechanistically, MNX1-AS1 could bind IGF2BP1, which increased the interaction of IGF2BP1 with MET mRNA to promote its stability, thus promoting tumorigenesis of breast cancer. Moreover, combination of MNX1-AS1 inhibition and MET small molecule inhibitor (PHA-665752, PHA) exhibited better antitumor efficacy in xenograft model, suggesting the therapeutic potential. Overall, our findings indicated that MNX1-AS1/MET regulatory axis may serve as a potential therapeutic target in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb4e1e2cea9e0a9b0b2c1fbdca7c268c8199624a" target='_blank'>
              Long Noncoding RNA MNX1-AS1 Promotes Tumorigenesis of Breast Cancer by Binding IGF2BP1 to Activate c-MET.
              </a>
            </td>
          <td>
            Jinrong Xie, Jianchun Gu, Wenjie Lv, Limin Cheng, Shuxian Chen, Ran Wu, Yan Liang, Meiling Zhu, Siyu Chen, Mawei Jiang, Jun Su
          </td>
          <td>2025-07-24</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Ultraviolet (UV) light causes cyclobutane pyrimidine dimers (CPDs) and other DNA lesions that must be efficiently repaired to prevent cell death and mutagenesis. While mammals utilize the nucleotide excision repair (NER) pathway to repair CPDs, many species primarily utilize photolyase enzymes to repair UV damage. Our understanding of how different genomic and chromatin features impact photolyase repair across a eukaryotic genome is limited. Here, we map repair of CPDs by photolyase across the yeast genome at single-nucleotide resolution. Our data indicate that yeast photolyase repairs CPDs more rapidly than NER, but photolyase activity is inhibited at certain classes of transcription factor binding sites and in nucleosomes. Repair in nucleosomes is particularly inhibited when CPDs are located along the 3′ side of the nucleosomal DNA or at minor-in rotational settings. Our data indicate that photolyase efficiently repairs the non-transcribed strand of yeast genes, but repair of the transcribed strand (TS) is inhibited. Genome-wide analysis of UV-induced mutations in NER-deficient photoreactivated yeast reveals a striking enrichment of mutations along the TS of yeast genes. These data indicate that inhibition of photolyase repair along the TS, likely due to occlusion of CPDs by RNA polymerase II stalling, promotes UV mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba7e81bd67cc35d46ad0fa3937b62cb144d357b6" target='_blank'>
              Genome-wide maps of UV damage repair and mutation suppression by CPD photolyase
              </a>
            </td>
          <td>
            Kaitlynne A. Bohm, Marian F. Laughery, P. A. Mieczkowski, Steven A. Roberts, John J Wyrick Data Curation
          </td>
          <td>2025-06-06</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Simple Summary A comprehensive study analyzed circulating cell-free DNA (cfDNA) from 30 oral squamous cell carcinoma (OSCC) patients, 199 public OSCC, and 192 normal samples to explore its potential in early-stage diagnostics and prognosis. The findings suggest that the cfDNA concentration correlates with the tumor stage, malignancy, and survival prognosis in OSCC patients. Deep genome sequencing of cfDNA revealed several genomic alterations: 1. Copy number variation analysis identified amplifications and deletions at loci 1q, 2q, 3p, 3q, and 8q22. Notably, amplifications of TP53 and PIK3CA, along with other genes related to keratinization, were identified in relapsed OSCC cases. Single nucleotide variations were observed in genes highly mutated in OSCC. 2. A novel fusion gene, TRMO-TRNT1, was detected in seven high-grade tumor samples. The parental genes, TRMO and TRNT1, are involved in tRNA modification and DNA repair, respectively. 3. Integrations of human papillomavirus, simian virus, and enterovirus were found in the OSCC samples, potentially indicating viral involvement in OSCC pathogenesis. These results underscore the utility of cfDNA as a non-invasive biomarker for OSCC, offering insights into tumor genomics and aiding in early detection and prognosis assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55c52e96cadb3f868e649f16559709defe911b5c" target='_blank'>
              Applications for Circulating Cell-Free DNA in Oral Squamous Cell Carcinoma: A Non-Invasive Approach for Detecting Structural Variants, Fusions, and Oncoviruses
              </a>
            </td>
          <td>
            M. Bhattacharya, D. Yaniv, D. P. D’Souza, Eyal Yosefof, Sharon Tzelnick, R. Detroja, Tal Wax, A. Levy-Barda, G. Baum, A. Mizrachi, G. Bachar, Milana Frenkel Morgenstern
          </td>
          <td>2025-06-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Castration-resistant prostate cancer is a heterogeneous disease with variable phenotypes commonly observed in later stages of the disease. These include cases that retain expression of luminal markers and those that lose hormone dependence and acquire neuroendocrine features. While there are distinct transcriptomic and epigenomic differences between castration-resistant adenocarcinoma and neuroendocrine prostate cancer, the extent of overlap and degree of diversity across tumor metastases in individual patients has not been fully characterized. Here we perform combined DNA methylation, RNA-sequencing, H3K27ac, and H3K27me3 profiling across metastatic lesions from patients with CRPC/NEPC. Integrative analyses identify DNA methylation-driven gene links based on location (H3K27ac, H3K27me3, promoters, gene bodies) pointing to mechanisms underlying dysregulation of genes involved in tumor lineage (ASCL1, AR) and therapeutic targets (PSMA, DLL3, STEAP1, B7-H3). Overall, these data highlight how integration of DNA methylation with RNA-sequencing and histone marks can inform intraindividual epigenetic heterogeneity and identify putative mechanisms driving transcriptional reprogramming in castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdf4a71accfefcd4f60da4f3b6e81d8837cae193" target='_blank'>
              Intraindividual epigenetic heterogeneity underlying phenotypic subtypes of advanced prostate cancer
              </a>
            </td>
          <td>
            Kei Mizuno, Sheng-Yu Ku, V. B. Venkadakrishnan, Martin K. Bakht, M. Sigouros, Joanna Chan, A. Trigos, Jordan H. Driskill, J. Manohar, Abigail King, Adam G. Presser, Min Jin Kim, Alok K. Tewari, H. Long, David Quigley, T. Choueiri, Steven P. Balk, Sarah J. Hill, J. Mosquera, David Einstein, Shahneen Sandhu, M. Taplin, H. Beltran
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="


 To investigate whether synthetic embryos (blastoids) derived from three different human stem-cell lines mimic human blastocyst molecular dynamics, including chromosome instability and cellular composition.



 Although human blastoids were derived from different stem cell lines, they exhibit highly similar transcriptomes. Their genome appears to be more stable than human embryos.



 Blastoids—stem cell-derived models of human blastocysts—provide a scalable and ethically-justified model to study early embryonic development. Different stem cell sources influence blastoid differentiation, but their specific impact remains unclear. While blastoids resemble human embryos in both transcriptional and morphological aspects, their genomic stability is largely unexplored. Single-cell RNA-seq and DNA-seq studies have shown that aneuploidy, a hallmark of early development, affects >80% of human blastocysts. Understanding whether blastoids exhibit similar rate of chromosomal instability and how aneuploid cells may be corrected is crucial for assessing their relevance in modeling early human development and advancing reproductive medicine and developmental biology.



 Three different stem cell lines, including human naïve embryonic (HNES1, H9) and induced pluripotent stem cells (niPSC HDF75) were used to generate blastoids. From each blastoid, we isolated 8-16 cells for single cell sequencing. We aimed to (i) investigate how starting cell cultures influence resultant blastoid molecular composition, (ii) determine whether blastoids recapitulate the high levels of chromosome instability observed in preimplantation human embryos, and (iii) characterize abnormal cells within blastoid.



 We collected initial stem cells (Day 0) and blastoid cells 72 hours after trophectoderm (TE) versus inner cell mass (ICM) induction (Day 3). We profiled their genome and transcriptome using single cell G&T-seq, performed copy number profiling, trajectory and differential expression analyses. Additionally, we identified copy number variations from published blastoid single-cell RNA-seq data using the inferCNV algorithm. This allowed us to compare blastoids derived from different stem cell sources, specifically HNES1 and niPSC.



 Transcriptome analysis of Day 3 blastoid cells (n = 155), derived from NHES1 and niPSC, revealed high similarity between them. However, single-cell genome profiling uncovered a mix of genomic stability and imbalance. While 43% of cells displayed chaotic copy number alterations, 57% showed more stable karyotypes, with 38% consistently exhibiting segmental gains on chromosomes 5 and 19. The transcriptomic cell scoring analysis revealed that cells with more disbalanced profiles were predominantly in S-phase or G2/M-phase. In contrast, genomically stable cells were mostly in G1-phase. Notably, copy number inference from public single-cell transcriptomic data of blastoids (n = 120 randomly selected cells; 30 cells per timepoint across 4 timepoints) revealed slight copy number gains, including chromosome 19, at 60 hours post-differentiation, followed by a sharp increase in copy number gains and losses at 96 hours post-differentiation.



 A limitation of our study is that a limited number of cells (5-8) were analyzed per single blastoid.



 Our findings highlight that chromosome instability also emerges in blastoids, though at rates differing from human preimplantation development. However, analyzing genomic instability in these models can help to refine and optimize them to better mimic human development.



 No
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cb157ed086d09d3d17646e65718ea01bd71ec8a" target='_blank'>
              O-205 Single-cell multi-omics to study genome plasticity in synthetic models of early human development
              </a>
            </td>
          <td>
            D. Obukhova, D. Luijkx, J. Paulissen, A. Swinnen, W. H. Loneus, M. Drüsedau, W. van Dijk, M. Bergamelli, E. Vrij, H. Brunner, A. Salumets, M. Zamani Esteki
          </td>
          <td>2025-06-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddc1079a033fb614c3439063ef9f1c82d7614ef4" target='_blank'>
              Phage-Encoded TelN Inhibits Mre11-Rad50 to Protect Hairpin Telomeres
              </a>
            </td>
          <td>
            Maya Houmel, Nicolas Pellaton, Anna Anchimiuk, Stephan Gruber
          </td>
          <td>2025-06-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 The apolipoprotein B mRNA editing enzyme, catalytic polypeptide- like 3 (APOBEC3 or A3) family of cytosine deaminases has been implicated in some of the most prevalent mutational signatures in cancer. A3-associated mutational signatures have been identified in more than 70% of cancer types and around 50% of all cancer genomes, with prominence in breast cancer as well as other cancer types. Many tumors are hypermutated by C-to-T/G mutations induced by A3 within TCW (W:T,A) motifs, accounting for many driver mutations in genes such as PIK3CA, ERBB2, and PPP2R1A. The genome and transcriptome of tumor tissue biopsies contain a wealth of information about these A3-associated signatures. However, quantification of A3 expression changes in tumor cells is confounded by the ubiquitous expression of these enzymes in infiltrating immune cells. To overcome this we will utilize single-cell RNA-seq analysis across a wide range of publicly available breast cancer datasets in order to follow the relative changes of A3 expression within individual cell types. Results from this analysis will give our lab insights into to changes in A3 expression in tumor cells which will be helpful in unveiling the complex relationships among breast cancer cells, stromal elements, and immune cells, facilitating the development of personalized therapies and prognostic markers, while also shedding light on breast cancer heterogeneity and potential vulnerabilities.
 Citation Format: Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi. Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-06-27.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4bd3dd6ddb39a7b246a7fddf688cff30066edc6" target='_blank'>
              Abstract P2-06-27: Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data
              </a>
            </td>
          <td>
            Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Kaiso (ZBTB33) is a transcription factor involved in mitotic clonal expansion and tumorigenesis in association with Adenomatous Polyposis Coli (APC) loss of heterozygosity. ENCODE data show strong overlap of the Kaiso promoter-binding site-encode-derived Kaiso-binding site (eKBS) and many other transcription factors, including BRCA1. Here we sought to determine whether BRCA1 is a component of the Kaiso enhanceosome that regulates gene transcription. Using proximal ligation assays, immunoprecipitation followed by mass spectrometry, luciferase assays and ChIP-seq experiments, we evaluated the association between BRCA1 and Kaiso. Kaiso nuclear extract immunoprecipitation experiments revealed that Kaiso associates strongly with genes involved in RNA splicing and processing. When Kaiso was not crosslinked to DNA, BRCA1 was not detected among Kaiso-binding proteins. However, overexpression of BRCA1 increased Kaiso-mediated gene transcription in luciferase assays in a Kaiso-dependent manner. Comparison of BRCA1 ChIP-seq and Kaiso ChIP-seq data from HCT116 cells revealed both BRCA1 and Kaiso commonly bind to the promoters of 379 genes. The most enriched term associated with these genes where BRCA1 and Kaiso bind their promoters is metabolism of RNA. Disease processes associated with these BRCA1/Kaiso gene promoters indicate that BRCA1 is functionally linked to a Kaiso-directed programme of RNAP2-mediated gene transcription and likely associated with colorectal cancer development and maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/867932990d4ec7d192a115e8b90063ba2e09603d" target='_blank'>
              Kaiso mediates transcription and RNA splicing in colorectal carcinoma: role of BRCA1 in the Kaiso enhanceosome
              </a>
            </td>
          <td>
            Weifeng Luo, Manish K Tripathi, Qi Liu, Lei Chen, Robert W Cowan, Juliet M. Daniel, Chi Yan, Ann Richmond, Albert B. Reynolds
          </td>
          <td>2025-06-01</td>
          <td>Open Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Epitranscriptomics has emerged as a rapidly evolving field that focused on studying post-transcriptional RNA modifications and their role in spatiotemporal regulation of gene expression. N6-methyladenosine (m6A) and 5-methylcytosine (m5C) represent the most extensively studied modifications on mRNAs. These reversible modifications, mediated by ‘writer,’ ‘eraser,’ and ‘reader’ proteins, dynamically fine-tune mRNA stability, splicing, and translation. Growing evidence links their dysregulation to pathological states, including cancer progression and metastasis, where their aberrant deposition on oncogenes or tumor suppressors alters cellular signaling and therapeutic responses. In the current study, we present a detailed analysis of the m5C epitranscriptomic landscape across distinct breast cancer molecular subtypes. Using CRISPR/Cas9, we confirm NSUN2 as a key m5C writer in human mRNAs. NSUN2 loss was validated by targeted sequencing and Western blotting. Furthermore, we demonstrate the regulatory effects of NSUN2 on its canonical mRNA targets, revealing its critical role in maintaining proper gene expression networks. Our findings strongly suggest that additional m5C writers contribute to m5C methylation machinery. Additionally, we assessed the functional impact of NSUN2 depletion on mRNAs harboring m5C sites using mRNA stability assays. Furthermore, our analysis revealed distinct m5C methylation patterns among breast cancer subtypes, highlighting unique m5C signatures associated with the disease. Notably, we identified specific hypomethylated and hypermethylated m5C sites in each breast cancer cell line, representing a universal m5C methylation signature for breast cancer. Our study constitutes the first comprehensive m5C epitranscriptomic atlas in human breast cancer and paves the way for future research aimed at developing targeted therapeutic interventions that leverage the m5C methylation landscape. Supplementary Information The online version contains supplementary material available at 10.1007/s10142-025-01648-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c7ab96a995706a16195a85c4f63f01b36d8d6b" target='_blank'>
              Exploring the m5C epitranscriptome of mRNAs in breast cancer cells through genome engineering and long-read sequencing approaches
              </a>
            </td>
          <td>
            Konstantina Athanasopoulou, P. Adamopoulos, P. Tsiakanikas, Andreas Scorilas
          </td>
          <td>2025-06-25</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea81a914a4a5b1e0240796b0fe52e36431943de7" target='_blank'>
              Genetic barcoding uncovers the clonal makeup of solid and liquid biopsies and their ability to capture intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            A. Serrano, Tom S. Weber, J. Berthelet, Sarah Ftouni, Farrah El-Saafin, Verena C. Wimmer, K. Rogers, Elgene Lim, Emmanuelle Charaffe-Jauffret, Christophe Ginestier, David Williams, Frédéric Hollande, B. Yeo, Sarah-Jane Dawson, S. Naik, D. Merino
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61956d7589478355803ed0351b120e6a0e356110" target='_blank'>
              SLFN11 Enhances Cisplatin Sensitivity in Pediatric Cancer via Activation of Stress-Response and Suppression of Survival Pathways
              </a>
            </td>
          <td>
            A. Jabeen, Dina Awartani, S. Sherif, Eiman I. Ahmed, R. Alanany, Ayman Saleh, W. Hendrickx, C. Raynaud
          </td>
          <td>2025-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Retroviral and retrotransposon invasion pose a constant threat to genome integrity and have driven the evolution of host defense pathways able to counter these attacks. The human silencing hub (HUSH complex) is an epigenetic transcriptional repressor complex that recognizes and silences newly integrated retroelements through the establishment of ectopic heterochromatin and chromatin compaction. HUSH provides a genome-wide immunosurveillance system whose challenging task is to detect and silence any newly integrated retroelements, and it thus plays a key role in host defense. To distinguish self from nonself genomic DNA, HUSH recognizes long single-exon (intronless) DNA, the essential hallmark of reverse transcription. Retroelements, being RNA derived, lack classical, noncoding cellular introns, so a long, intronless sequence of DNA is the abnormal molecular pattern that allows HUSH to distinguish invading retroelements from intron-containing host genes. As a newly identified component of the innate immune system, HUSH protects the genome from the reverse flow of genetic information from RNA to DNA, revealing an unanticipated mechanism of postintegration genome immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8b1d5426e615e5a445a45b06d1bcad8bd22f0c5" target='_blank'>
              Silencing by the HUSH Epigenetic Transcriptional Repressor Complex.
              </a>
            </td>
          <td>
            Paul J. Lehner
          </td>
          <td>2025-06-01</td>
          <td>Annual review of biochemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7bd3ebe84ec1189f788307a3b3b752befef2ae" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals an Aging-Associated Fibroblast Subtype Linked to Tumor Progression in Human Skin
              </a>
            </td>
          <td>
            Haibin Wu, Danping Pan
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA topoisomerases are essential for maintaining DNA topology, gene expression and the accurate transmission of genetic information. Mitochondria possess circular DNA (mtDNA), which, unlike nuclear chromosomes, lacks protective histones and exists in nucleoprotein complexes called nucleoids, which are vital for mtDNA stability. Although the mitochondrial genome encodes essential genes involved in ATP production via oxidative phosphorylation, it does not encode crucial mtDNA maintenance genes and depends entirely on nuclear-encoded proteins for mtDNA maintenance. These include nuclear-encoded topoisomerases (i.e. Top1mt, Top2α, Top2β and Top3α), which alleviate topological stress during mtDNA transcription and replication, and mitochondrial transcription factor A (TFAM), are crucial for ensuring proper nucleoid structure and mtDNA packaging. Furthermore, tyrosyl-DNA phosphodiesterase 1 and 2 (TDP1 and TDP2) participate in the repair of mtDNA damage associated with trapped topoisomerase-mtDNA complexes, which can compromise mtDNA integrity and contribute to neurodegeneration, cancer and premature aging. Drugs that stabilize these protein-DNA adducts (PDAs) to induce mtDNA damage and mitochondrial dysfunction are promising new strategies for cancer therapy. This Review explores the essential roles of mitochondrial topoisomerases, overviews mechanisms involved in mtDNA repair and discusses how mitochondrial fission and mitophagy are employed as a survival strategy for clearing damaged mtDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc13f830961da01cf0f5401d6373dc48a97645fa" target='_blank'>
              Mitochondrial topoisomerases, nucleoid architecture and mtDNA repair in human disease.
              </a>
            </td>
          <td>
            Sangheeta Bhattacharjee, B. Das
          </td>
          <td>2025-07-01</td>
          <td>Journal of cell science</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="ABSTRACT RNA alternative splicing is a fundamental cellular process implicated in cancer development. Kaposi’s sarcoma-associated herpesvirus (KSHV), the etiological agent of multiple human malignancies, including Kaposi’s sarcoma (KS), remains a significant concern, particularly in AIDS patients. A CRISPR-Cas9 screening of matched primary rat mesenchymal stem cells (MM) and KSHV-transformed MM cells (KMM) identified key splicing factors involved in KSHV-induced cellular transformation. To elucidate the mechanisms by which KSHV-driven splicing reprogramming mediates cellular transformation, we performed transcriptomic sequencing, identifying 131 differentially alternative spliced transcripts, with exon skipping as the predominant event. Notably, these transcripts were enriched in vascular permeability, multiple metabolic pathways, and ERK1/2 signaling cascades, which play key roles in KSHV-induced oncogenesis. Further analyses of cells infected with KSHV mutants lacking latent genes, including vFLIP, vCyclin, and viral miRNAs, as well as cells overexpressing LANA, revealed their involvement in alternative splicing regulation. Among the identified splicing factors, FAM50A, a component of the spliceosome complex C, was found to be crucial for KSHV-mediated transformation. FAM50A knockout resulted in distinct splicing profiles in both MM and KMM cells and significantly inhibited KSHV-driven proliferation, cellular transformation, and tumorigenesis. Consistently, FAM50A knockdown suppressed the proliferation of PEL cells. Mechanistically, FAM50A knockout altered SHP2 splicing, promoting an isoform with enhanced enzymatic activity that led to reduced STAT3 Y705 phosphorylation in KMM cells. These findings reveal a novel paradigm in which KSHV hijacks host splicing machinery, specifically FAM50A-mediated SHP2 splicing, to sustain STAT3 activation and drive oncogenic transformation. IMPORTANCE Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies. Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33b22cecd1973d138799718526caad5582306174" target='_blank'>
              KSHV reprograms host RNA splicing via FAM50A to activate STAT3 and drive oncogenic cellular transformation
              </a>
            </td>
          <td>
            Shenyu Sun, Ling Ding, Karla Paniagua, Xian Wang, Yufei Huang, Mario A. Flores, Shou‐Jiang Gao
          </td>
          <td>2025-06-12</td>
          <td>mBio</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe12fe3c0e52aeffbd01092c11945a919867298" target='_blank'>
              Multimodal single-cell analyses reveal distinct fusion-regulated transcriptional programs in Ewing sarcoma
              </a>
            </td>
          <td>
            Olivia G. Waltner, April A. Apfelbaum, Emma D. Wrenn, Shruti S Bhise, Sami B. Kanaan, R. Gladden, Mark A. Mendoza, R. Volden, Z. Kronenberg, Anand G. Patel, Michael A Dyer, J. Sarthy, Elizabeth R. Lawlor, Scott N. Furlan
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Transposable elements (TE), also called transposons, are repetitive DNA sequences making up half of the human genome. Initially, TEs were described as “junk DNA” because they lack specific function. However, they have been recognized for their ability to replicate and integrate into different genomic locations; this “jumping gene” activity results in genomic instability, variation of the chromosomal architecture, and transcriptional dysregulation, all of which represent major hallmarks of cancer. In this respect, the involvement of TE in tumorigenesis was extensively studied and their role as diagnostic and therapeutic tools in cancer is now well-established. Transposons’ products, including TE-derived cancer antigens, transcripts, and associated epigenetic modifications, mainly hypomethylation, were found to be promising biomarkers in several types of cancer ensuring early disease detection and prognosis. In addition, TE are currently used to design innovative approaches, with transposon-based systems, namely, Sleeping Beauty and PiggyBac, enabling precise genomic modifications and novel strategies for cancer therapy. Therefore, the aim of this review is to provide an overview on the dual application of TE as diagnostic and therapeutic tools in cancer, paving the way to improved clinical outcomes. Clinical trial number Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4636ade80b58aaf2a75f12fb6871d3d8ab2244c5" target='_blank'>
              Exploring transposable elements: new horizons in cancer diagnostics and therapeutics
              </a>
            </td>
          <td>
            Ahlam Chaaban, Reem Sleem, Jana Santina, Mohamad Rima, José-Noel Ibrahim
          </td>
          <td>2025-07-12</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Among the various genotypes of the human papillomavirus (HPV), the HPV16 genotype is the most oncogenic by far, and some (sub)lineages may be associated with an increased risk of cancer. Thus, characterizing the full genome is important to understand the link between the HPV16 genome variability and its transforming role. We set up a multiplex PCR approach combined with Oxford Nanopore Technologies for the HPV16 full-genome characterization. The primers were designed with PrimalScheme, and the optimization of the monoplex/multiplex PCR was performed on SiHa cells. After library preparation, the sequencing was performed on a GridION sequencing device to determine the sensitivity, the specificity, and the performance of the method. Fourteen primer pairs were selected to span the full genome. With the monoplex PCR, 12 primer pairs showed good amplification, while a double concentration of two primers was required to improve the amplification. We then performed a multiplex PCR approach by generating two pools comprising non-overlapping primer pairs. The multiplex PCR showed good sensitivity, allowing the amplification of HPV16 with a Ct value below 27 and providing a coverage greater than 99.9% and a sequencing depth greater than 100×. In addition, the specificity of the method was validated by the absence of amplification of other high-risk HPV genotypes compared with HPV16. We set up an amplicon-based approach to characterize the full genome and diversity of the HPV16 (sub)lineages. This approach shows good sensibility and specificity with limited cost, opening new perspectives in the field of whole-genome HPV sequencing. IMPORTANCE HPV16 is by far the most oncogenic genotype, so characterizing the genetic variability of its genome is important to better understand the link with its transforming role. We developed an amplicon-based approach combined with Oxford Nanopore Technologies next-generation sequencing to overlap the HPV16 genome, which is easy to implement in the laboratory and inexpensive in the field. HPV16 is by far the most oncogenic genotype, so characterizing the genetic variability of its genome is important to better understand the link with its transforming role. We developed an amplicon-based approach combined with Oxford Nanopore Technologies next-generation sequencing to overlap the HPV16 genome, which is easy to implement in the laboratory and inexpensive in the field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/836ca2c5fafa3f7dae222db6854bdcbbe566eaa9" target='_blank'>
              An amplicon-based approach for full-genome characterization of HPV16
              </a>
            </td>
          <td>
            I. Malet, Inès Draa, V. Leducq, Fanny Vuong, Pascale Bonnafous, A. Marcelin, V. Calvez, A. Jary
          </td>
          <td>2025-06-09</td>
          <td>Microbiology Spectrum</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="
 Background: While recent advances have improved outcomes for Triple Negative Breast Cancer (TNBC), it continues to have a poor prognosis. Long non-coding RNAs are among a recent class of epigenetic regulators that function in the nucleus to support the stability of cells and maintain the fidelity of chromatin interactions. Our laboratory discovered the long noncoding RNA MANCR (LINC00704) as being upregulated in human breast cancer. Furthermore, our work demonstrated that MANCR is enriched in TCGA breast cancer patient samples that are not estrogen or progesterone receptor positive. We also demonstrated that the 10-year survival in this TCGA analysis is substantially worse in patients with high MANCR expression. More recently, we have found MANCR to be aberrantly expressed in TNBC cells, and these cells are highly dependent on MANCR to retain their tumorigenic characteristics. Methods: Functional in vitro studies in MDA-MB-231 TNBC cells used short antisense nucleic acids (GapmerRs) to knockdown MANCR. For in vivo studies, TNBC cells were injected into the mammary fat pad of mice, allowing tumor formation, and subsequent treatment with 2 nmol/g of a negative control or MANCR targeting GapmeR. We also identified genome interaction sites at single nucleotide resolution by chromatin isolation by RNA purification sequencing (ChIRP-seq) to determine the mechanism of MANCR activity in TNBC cells. Results: We now demonstrate that MANCR knockdown promotes DNA damage and decreases cell proliferation, migration, anchorage-independent colony formation, transwell invasion, and cellular survival. Additionally, in vivo targeting of MANCR drastically inhibited tumor growth over time and the end-point tumor mass. The MANCR GapmeR treatment also inhibited the ability of TNBC cells to circulate and disseminate to distant organs in vivo. After performing ChIRP-seq in the MDA-MB-231 cells, we identified 1206 genome-wide binding sites that exhibit MANCR interactions, of which 48% are intergenic and 52% are in genic regions. Furthermore, many MANCR ChIRP peaks were found to overlap with fragile sites in the genome, indicating MANCR provides stability to these sites. Conclusions: These data suggest that targeting MANCR has therapeutic potential for patients with “MANCR-high” TNBC tumors by disrupting genome stability. Indeed, our in vivo studies demonstrate that “MANCR-high” TNBC tumors require MANCR to rapidly grow and promote disease progression. Significantly, many of the MANCR-chromatin interactions identified were found in intergenic regions and overlap with fragile sites within the genome. Collectively, these data strongly indicate that MANCR stabilizes the TNBC genome, and disrupting genome stability by targeting MANCR has therapeutic potential.
 Citation Format: Janine S. A. Warren, Bodhisattwa Banerjee, Jonathan A. R. Gordon, Prachi N. Ghule, Janet L. Stein, Gary S. Stein, Jane B. Lian, Peter A. Kaufman. Mitotically-associated long noncoding RNA (MANCR): A novel long noncoding RNA that promotes genomic stability and cellular proliferation in Triple Negative Breast Cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-07-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbb40dcaca0c24edab0b1fb3286bc3d79cdad166" target='_blank'>
              Abstract P1-07-17: Mitotically-associated long noncoding RNA (MANCR): A novel long noncoding RNA that promotes genomic stability and cellular proliferation in Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Janine Warren, Bodhisattwa Banerjee, J. Gordon, Prachi N Ghule, Jane L. Stein, G. S. Stein, J. Lian, Peter A. Kaufman
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The protein autoubiquitination has emerged as a significant focus in pan-cancer genetic research due to its potential impact on cancer progression and treatment. Protein autoubiquitination regulates the stability, activity, and localization of involved proteins, playing a crucial role in various cellular processes, including signal transduction, protein quality control, and immune response regulation. This mechanism is vital for maintaining cellular homeostasis and adapting to environmental changes or stress, such as tumor growth. Insights into these processes could lead to novel therapeutic strategies targeting the ubiquitin-proteasome system. This study examines the clinical relevance of transcription factors associated with protein autoubiquitination genes, including CNOT4, MTA1, NFX1, RNF10, RNF112, RNF115, RNF13, RNF141, RNF4, RNF8, TAF1, TRIM13, and UHRF1. Using multi-omics profiling data and Gene Set Cancer Analysis (GSCA) with normalized SEM mRNA expression, the study evaluates differential expression, gene mutations, and drug correlations. The analysis revealed that the single nucleotide variant (SNV) heatmap indicated high mutation frequencies for many of these genes across various cancer types. Gene expression analysis showed limited overall significance, but TAF1 was notably upregulated in uterine corpus endometrial carcinoma (UCEC), while RNF115 and RNF141 were downregulated in the same cancer type. Copy number variation (CNV) profiles exhibited diverse patterns across cancer types, and methylation profiles suggested differences in methylation levels between tumor and normal tissues. Additionally, single-cell transcriptomic analysis uncovered cancer-type-specific functional states. This research highlights the importance of understanding autoubiquitination genes in cancer biology, which may aid in developing effective diagnostic and prognostic strategies. However, the analysis is limited to experimental evidence. However, these findings derive solely from publicly available datasets and lack experimental validation, which may introduce bias. Single-cell analyses cover only a few tumor types, drug-gene relationships remain correlative, and the absence of longitudinal clinical data prevents evaluation of true prognostic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ee1fde6115331cd6538d86495d1787ac8e59d8c" target='_blank'>
              An in-silico pan-cancer bulk and single-cell profiling of transcription factors in protein autoubiquitination
              </a>
            </td>
          <td>
            Angela Dong, A. Rasteh, Panpan Wang, Hengrui Liu
          </td>
          <td>2025-07-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1ead10baece9545ae1e3801dde43ce3d766e518" target='_blank'>
              METTL5 Blockade Enhances Anti-Tumor Immune Response via Inducing Neoantigen Generation
              </a>
            </td>
          <td>
            Yangyi Zhang, Xiaoyan Shi, Yuci Wang, Ruiqi Wang, Yanlan Cao, Hao Chen
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 11529


 Background:
 Epigenetic dysregulation is a key mechanism in cancer biology, driving both immune evasion and the generation of potentially druggable neoantigen targets. Transposable elements (TEs) are proviral sequences mostly found in epigenetically repressed genomic regions. In cancer, TEs are one potential source of neoantigens. The tumor suppressor gene
 ATRX
 is an epigenetic regulator that is lost in up to 30% of soft tissue sarcomas. Loss of function of this heterochromatin modulator leads to widespread alterations to transcription patterns. Our prior work in sarcoma, along with studies in other tumors, showed that ATRX loss leads to the de-repression of TEs. We hypothesize that this mechanism leads to TE neoantigen presentation on MHC class I (MHC-I) molecules.
 Methods:
 Leveraging the gEVE database of proviral sequences with open reading frames, we examined the differential expression of TEs using RNA sequencing of multiple models comparing ATRX lost and ATRX wild-type tumors. Analysis included our previously published data from human undifferentiated pleomorphic sarcoma cell lines (2 clones, 3 replicates per group; GEO: GSE240030) and a publicly available dataset from a mouse model of soft tissue sarcoma (n = 3 per group; GEO: GSE167537). Additionally, we analyzed analogous data from a mouse model of glioblastoma multiforme (GBM), another malignancy with frequent ATRX loss (n = 3 per group; GEO: GSE178113). RNA-seq data was analyzed using Salmon for transcript-to-genome pseudo-alignment, DESeq2 for differential TE expression analysis, and arcasHLA for sample-level MHC-I subtype identification. Differentially overexpressed TEs in ATRX mutant tumors were analyzed for predicted neoantigens using the NetMHCpan4.1 package.
 Results:
 Across multiple models, ATRX loss of function alterations were associated with both activation and repression of TEs. Within two sarcoma contexts, upregulated TEs were found to have protein-coding sequences with the potential to lead to MHC-I neoantigen presentation. Within a human undifferentiated pleomorphic sarcoma cell line with ATRX loss, there were 7 upregulated TEs shared between the clones, each predicted to contain 12 – 50 MHC-I binding epitopes. Within a mouse soft tissue sarcoma model, 7 TEs were upregulated with ATRX loss, each with 5 – 28 MHC-I binding epitopes. Since ATRX loss alters TE expression in other malignancies, parallel data from a mouse GBM model was additionally analyzed. In this model, 308 TEs were found to be upregulated with ATRX loss, and all were predicted to generate neoantigens. ERVs and LINE1 elements were recurrently identified, making up 97 – 100% of the activated TEs.
 Conclusions:
 Transposable element transcription altered by ATRX loss in sarcoma and GBM creates a predicted source of MHC-I-binding neoantigens. These neoantigens come from TEs of the ERV and LINE1 families.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d406b838be4f31bb8e68fe40b2f943691a36673" target='_blank'>
              Predicting transposable element-derived neoantigens in ATRX-mutated sarcoma.
              </a>
            </td>
          <td>
            Simon Cao, Ziyang Wang, B. Nacev
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) contributes to 3% of all childhood cancers with roughly 400-500 cases diagnosed each year in the United States. The World Health Organization classifies rhabdomyosarcoma into four histological subtypes which include alveolar, embryonal, spindle-cell and pleomorphic. The primary genetic drivers in a subset of alveolar and spindle-cell histological subtypes are gene fusions. This review explores the fusion oncogenes identified in RMS such as PAX-and NCOA2-based fusions, along with discussing studies defining fusion oncogene biology and tumorigenic mechanisms. Focus areas include data around transformation events and progression along with dysregulated biological processes. Furthermore, we summarize model systems, ranging from cell to animal models, that have been implemented to study fusion oncogenes identified in RMS. With the constant identification of novel fusion oncogenes, this review also emphasizes the need for genetically characterizing RMS tumors and rapidly developing new model systems. These models are critical to study fusion oncogene activity and to delineate key regulatory players and potential therapeutic targets that suppress tumorigenesis. The identification of RMS fusion oncogenes and integration with animal and cell culture models will help identify conserved molecular targets, optimize therapeutic approaches, and ultimately improve clinical outcomes for children with RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b23d84102b5431b0bc6bd415779fd94bf54425c" target='_blank'>
              Fusion oncogenes in rhabdomyosarcoma: model systems, mechanisms of tumorigenesis, and therapeutic implications
              </a>
            </td>
          <td>
            Chinmay S. Sankhe, Lisa Hall, Genevieve C. Kendall
          </td>
          <td>2025-06-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 e15100


 Background:
 Intratumoral heterogeneity, driven by phenotypic plasticity, contributes to disease progression and therapy failures. However, a thorough understanding of its underlying mechanisms is still lacking. Breast cancer exemplifies this with its histological and molecular diversity; estrogen receptor-positive (ER+), progesterone receptor-positive (PR+), and HER2- luminal A subtypes, such as the MCF7 cell line, are not the exception. While single-cell RNA sequencing (scRNA-seq) and three-dimensional multicellular tumor spheroids (MCTS) represent initial approaches to understanding heterogeneity, integrating these tools with computational strategies offers new paths to uncover the regulatory groundwork of tumor behavior. When modeled via Boolean frameworks, transcriptional regulatory networks (TRNs) provide a dynamic platform for studying gene interactions, identifying attractors (stable cellular states corresponding to distinct phenotypes), and simulating cellular transitions. This study represents the first step in constructing such a framework for ER+ breast cancer to advance precision medicine and improve therapeutic strategies.
 Methods:
 Regulatory elements relevant to the MCF7 cell line were identified through an extensive review of NCBI literature. Interactions were strictly curated based on evidence quality (ranging from robust experimental validation to predictive hypotheses), and evaluated for type and significance, such as feedback loops, activation, coactivation, and inhibition. Logical operators (AND, OR, NOT) were used to translate regulatory connections into Boolean functions to simulate binary gene expression dynamics. The final TRN was visualized using Cytoscape (v3.10.3) with a hierarchical layout.
 Results:
 The TRN consists of 54 nodes and 81 interactions, representing the regulatory landscape of ER+, PR+, and HER2- breast cancer. Boolean equations offer a framework for simulating the evolution of binary states to attractors, enabling the observation of how perturbations, such as gene overexpression or silencing, influence heterogeneity. While simulation and attractor analyses are ongoing, the network provides a foundation for elucidating the regulatory mechanisms underlying breast cancer plasticity.
 Conclusions:
 This study highlights the potential of integrating experimental data, TRN reconstruction, and Boolean modeling to understand intratumoral heterogeneity. Our rigorously curated model establishes a foundational framework for precision oncology and the integration of experimental and computational methods.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10947cef519fbed46d1a42e2676b4ea1fd778e16" target='_blank'>
              Boolean network modeling: The first step toward a new understanding of breast cancer heterogeneity.
              </a>
            </td>
          <td>
            Lia Medina Montalvo, Thelma Escobedo-Tapia, Osbaldo Resendis-Antonio
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Comprehensive analysis of single-cell transcriptome and chromatin accessibility will contribute to interpret the heterogeneity of acute myeloid leukemia (AML). We hypothesize that integrating single-cell transcriptomic and chromatin accessibility landscapes underlying t(8;21) AML will provide valuable insights into its heterogeneous cellular properties and gene regulatory programs. Methods Here, we conducted paired single-cell RNA-sequencing (scRNA-seq) and single-cell ATAC sequencing on bone marrow samples from newly diagnosed t(8;21) AML patients and healthy controls. Genetic signatures extracted from scRNA-seq were built and validated across three independent cohorts (German AMLCG1999, GSE106291 and TCGA LAML). Results We identified TCF12, a core component of AML1-ETO-containing transcription factor complex (AETFC), as the most active transcription factor in blast cells, driving a universally repressed chromatin state. Furthermore, we delineated two functionally distinct T cell subsets, revealing that EOMES-mediated transcriptional regulation promotes the expansion of a cytotoxic T cell population (T cells_2; high GNLY, NKG7 and GZMB expression), with an increased clonality and a tendency for drug resistance. In addition, we discovered a novel leukemic CMP-like cluster characterized by high TPSAB1, HPGD and FCER1A expression. Leveraging machine learning-based integration of multi-omic profiles, we identified a robust 9-gene prognostic signature, which demonstrated significant predictive value for AML outcomes across three independent cohorts. Conclusions This multi-omics study provides unprecedented insights into the transcriptional and epigenetic heterogeneity of t(8;21) AML, providing a potential actionable tool for clinical risk stratification. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06659-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb6d672d2579a9875bb647a603a272b4cc733a93" target='_blank'>
              A multi-omic single-cell landscape reveals transcription and epigenetic regulatory features of t(8;21) AML
              </a>
            </td>
          <td>
            Xue-Ping Li, Yan Gao, Bai-Tian Zhao, Yujun Dai, J. Mao, Yang Liang, Lu Jiang
          </td>
          <td>2025-07-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Recent advancements in genomic technologies have become critical tools for deciphering the genetic complexities of cancer tissues, enabling precision medicine strategies aimed at improving patient clinical outcomes. Here we performed a comprehensive analysis of clinically annotated whole genome and whole transcriptome sequences from 1,364 breast cancer cases. Our investigation provides the most detailed genomic landscapes of breast cancer to date, which allowed us to comprehensively correlate genomic changes with clinical characteristics. Our findings include, but not limited to, (1) whole-genome-based homologous recombination deficiency profiles effectively predict responses to adjuvant chemotherapy and first-line CDK4/6 inhibitor treatment, (2) focal ERBB2 amplifications, often arising via extrachromosomal DNA mechanisms, are strongly associated with a positive response to neoadjuvant chemotherapy. Additionally, our study identified (3) recurrent copy number amplifications linked to patient survival, and (4) whole-genome-based intratumoral heterogeneity profiles that can predict patient survival as well as response to anti-HER2 therapies. Our work underscores the power of highly annotated whole-genome and transcriptome sequencing of a population-scale cancer cohorts for uncovering clinically relevant genomic mutations and novel molecular targets, thereby advancing precision oncology strategies for breast cancer.
 Citation Format: Ryul Kim,JongHan Yu, Joonoh Lim, Brian Baek-Lok Oh, Seok Jin Nam, Seok Won Kim, Jeong Eon Lee, Byung Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, Pil Sun Baek, Soo Yeon Bae, Chang Ik Yoon, Young Joo Lee, Dooreh Kim, Kabsoo Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, Erin Connolly-Strong, Sangmoon Lee, Bo Rahm Lee, Yuna Lee, Ki Jong Yi, Young Oh Kwon, In Hwan Chun, Junggil Park, Jihye Kim, Chahyun Choi, Jong Yeon Shin, Hyunjung Lee, Minji Kim, Hansol Park, Ilecheon Jeong, Boram Yi, Won-Chul Lee, Jeong Seok Lee, Woo Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Yeon Hee Park. Whole-genome landscapes of 1,364 breast cancers with clinical outcomes [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-01-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea155495e72da0b7ab53fc77458b87559af5c5e0" target='_blank'>
              Abstract P1-01-06: Whole-genome landscapes of 1,364 breast cancers with clinical outcomes
              </a>
            </td>
          <td>
            Ryul Kim, J. Yu, Joonoh Lim, Brian Baek-Lok Oh, S. Nam, S. Kim, Jeong-Eon Lee, Byung-Joo Chae, Ji-Yeon Kim, Ga Eun Park, Bong Joo Kang, Pil Sun Baek, Soo Yeon Bae, C.I. Yoon, Young Joo Lee, D. Kim, K. Shin, Ji Eun Lee, Jun Kang, Ahwon Lee, E. Connolly-Strong, Sangmoon Lee, Bo-Rahm Lee, Yuna Lee, Kijong Yi, Youngoh Kwon, In Hwan Chun, Jung-Woon Park, Jihye Kim, Chahyun Choi, Jong-Yeon Shin, Hyunjun Lee, Minji Kim, H. Park, Ilecheon Jeong, Boram Yi, W. Lee, Jeongmin Lee, Woo Chan Park, Sung Hun Kim, Yoon-La Choi, Jeongmin Lee, Young Seok Ju, Y. Park
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The RYR3-DT divergent transcript is a long noncoding RNA overexpressed in gliomas. DNA methylation is frequently perceived as a crucial process associated with epigenetic changes and among the preeminent mechanisms involved in gene inactivation. Ferroptosis is a recently discovered class of cell necrosis and death. Ferroptosis stimulation can potentially eliminate malignancies. RYR3-DT expression and its exact mechanism in glioma remain ambiguous. This investigation found that RYR3-DT was markedly elevated in gliomas. In-vitro functional assays, including Transwell assay, CCK-8, Western blot, and Annexin V assay, revealed that RYR3-DT silencing inhibits glioma cells growth, migration, and invasion, while stimulated cellular apoptosis. Western blot analysis and RNA sequencing proved that RYR3-DT regulated the ferroptosis CoQ10/FSP1 pathway, concluding that lncRNA RYR3-DT deletion might suppress glioma cell growth migrating and invading ability, and also increase cellular apoptosis via CoQ10/FSP1 pathway. Our research recommends lncRNA RYR3-DT is an expected target for future discoveries on glioma treatment. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03115-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58170cd15bdf825855f6cf4264eebb6a91f04f1b" target='_blank'>
              Aberrant DNA hypermethylation-regulated long non-coding RNA RYR3-DT affects ferroptosis and progression of glioma cell through the CoQ10/FSP1 axis
              </a>
            </td>
          <td>
            Yang Cui
          </td>
          <td>2025-07-02</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is a deadly disease known for its genetic heterogeneity. LTR12C is an endogenous retrovirus-derived regulator of pro-apoptotic genes and is normally silenced by epigenetic regulation. In this study, we found that the treatment of two glioblastoma cell lines, T98-G and U87-MG, with DNA methyltransferase (DNMT) and histone deacetylase (HDAC) inhibitors activated LTR12C expression. Combined treatment with these epigenetic drugs exerted a synergistic action on the LTR12C activation in both cell lines, while treatment with each drug as a single agent had a far weaker effect. A strong induction of the expression of the TP63 gene was seen in both cell lines, with the pro-apoptotic isoform GTA-p63 accounting for most of this increase. Coherently, downstream targets of p63, such as p21 and PUMA, were also induced by the combined treatment. Furthermore, we observed a significant reduction in the GBM cell growth and viability following the dual DNMT/HDAC inhibition. These findings reveal that the reactivation of LTR12C expression has the potential to modulate survival pathways in glioblastoma and provide information regarding possible epigenetic mechanisms that can be used to treat this deadly disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b329500c4414f15e1eee2caec6a0bfaed4d1e7f7" target='_blank'>
              Functional Characterization of LTR12C as Regulators of Germ-Cell-Associated TA-p63 in U87-MG and T98-G In Vitro Models
              </a>
            </td>
          <td>
            Lucia Meola, Sohum Rajesh Shetty, A. Peschiaroli, Claudio Sette, Camilla Bernardini
          </td>
          <td>2025-06-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e8a7efbbd0fffc709e336945d36a6f61377b28f" target='_blank'>
              Intratumoral heterogeneity in microsatellite instability at single cell resolution
              </a>
            </td>
          <td>
            Harrison Anthony, C. Seoighe
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background The human genome contains approximately 3,200 near full-length autonomous human endogenous retroviral (HERV) genomes distributed across the 23 chromosomes. These autonomous HERV proviral genomes include long terminal repeats (LTRs) capable of promoting RNA transcription. In quiescent cells, most HERV loci remain transcriptionally silent. However, environmental changes, such as epigenetic remodeling of chromatin, can activate these silenced loci. Methods To study HERV reactivation, we previously analyzed autonomous HERV expression patterns in monocytes isolated from peripheral blood mononuclear cells (PBMCs) identified in single-cell RNA sequencing (scRNA-seq) databases using the Azimuth application. We developed a Window-based HERV Alignment (WHA) method, which analyzes aligned DNA sequences using sequential, non-overlapping windows of defined lengths. Samples were scored as positive (>= 9 good/usable windows) or negative (<= 8 good/usable windows). Results Using WHA, we established a control set from 31 normal individuals, with fewer than 8 windows at selected HERV loci. We analyzed scRNA-seq data from three studies of hospitalized COVID-19 patients and found distinct HERV expression patterns in monocytes. Unique patterns were also found in patients with influenza, Dengue virus, or sepsis. We next examined HERV expression at early (<7 days) and late (>14 days) timepoints post COVID-19 recovery and detected HERV loci in both groups. Analyzing 12 patients with post-acute sequelae of COVID-19 (PASC), we identified three HERV loci expressed in all patients. Some loci showed amplified numbers of good/usable windows, indicating longer transcripts and greater sequence depth. The most amplified locus was located within an intron of JAKMIP2, which, along with neighboring host genes, also showed increased transcription. Conclusion Previous studies have shown that viral infections, including COVID-19, influenza, and Dengue virus, as well as sepsis, can induce innate immune memory in monocytes through epigenetic remodeling of hematopoietic stem and myeloid precursor cells. The identification of co-amplified HERV loci and neighboring host gene transcripts in monocytes from PASC patients suggests expansion of epigenetically remodeled myeloid progenitors. The identification of these HERV-host gene patterns provides a foundation needed to understand the clinical features of patients with PASC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3890c566bb6e206114c5472e39818cdcd77102fc" target='_blank'>
              Amplification of select autonomous HERV loci and surrounding host gene transcription in monocytes from patients with post-acute sequelae of COVID-19
              </a>
            </td>
          <td>
            Hyunmin Koo, Casey D. Morrow
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Genomic instability is a hallmark of cancer, enabling the generation of mutations and gross chromosomal rearrangements to drive neoplastic cell transformation and oncogenesis. The BRCA1-BARD1 protein complex acts to eliminate highly toxic DNA double-strand breaks, to ensure the faithful propagation of our genetic blueprint and to suppress cancer development. BRCA1 is a well-described tumor suppressor protein associated with hereditary breast and ovarian cancers as well as sporadic breast cancers, with loss or mutation of BRCA1 leading to triple negative breast cancer and poor patient prognosis. The BRCA1-BARD1 complex promotes homologous recombination (HR), which is the major pathway for the accurate repair of double-strand breaks. However, there is little information regarding the intricate roles fulfilled by BRCA1-BARD1 in this process, or how loss of specific BRCA1-BARD1 functions leads to tumorigenesis. BRCA1 has been previously reported to physically interact with both DNA and RAD51, key factors in HR, but the contributions of the interaction attributes to DNA damage repair remain unknown. Here, we delineate major sites of DNA and RAD51 binding in BRCA1 and use a combination of biochemical and NMR methods to identify the specific residues mediating interactions with these ligands. This has allowed us to develop mutations to ablate BRCA1’s ability to interact with these substrates without affecting BRCA1-BARD1’s interaction with other key DNA repair substrates. Using these BRCA1 mutants impaired for either DNA or RAD51 binding, we have interrogated the contributions of these interaction attributes to BRCA1-BARD1’s function by comparing the activity of WT to mutant BRCA1-BARD1 in biochemical assays to reconstitute various steps of HR. We have found that both DNA and RAD51 binding are indispensable for BRCA1-BARD1’s ability to promote RAD51-mediated D-loop formation, thus helping to delineate the mechanism by which BRCA1 promotes HR. Our studies provide the foundation to determine the functional consequences of cancer mutations in BRCA1-BARD1 and for the development of therapeutic strategies to target HR-deficient tumors.
 Citation Format: Angela Jasper, Hoang Dinh, Cody M. Rogers, Sameer Salunkhe, Hardeep Kaur, Antoine Baudin, David S. Libich, Patrick Sung. The Mechanistic Role of BRCA1 DNA and RAD51 Binding in DNA Double-Strand Break Repair [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19be3dd24b8cbf64d48385ebe0e058a4d636d0fd" target='_blank'>
              Abstract P5-06-06: The Mechanistic Role of BRCA1 DNA and RAD51 Binding in DNA Double-Strand Break Repair
              </a>
            </td>
          <td>
            Angela M Jasper, Hoang Dinh, Cody M Rogers, Sameer Salunkhe, Hardeep Kaur, Antoine Baudin, David S. Libich, Patrick Sung
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: Epigenetic dysregulation constitutes one of the mechanisms in the pathogenesis of HCC, thereby underscoring its critical scientific importance for early diagnosis and therapeutic interventions. The chromatin remodeling factor AT-rich interaction domain 1A (ARID1A) serves as an essential epigenetic regulator and is frequently subject to inactivating mutations across various cancer types. Nevertheless, the precise molecular pathways by which ARID1A deficiency facilitates the progression of HCC remain inadequately elucidated. Methods: RNA sequencing (RNA-seq) was employed to identify genes that were significantly upregulated following the knockdown of ARID1A. Immunohistochemistry was used to assess the expression levels of ARID1A and histone deacetylase 7 (HDAC7) in HCC samples. Chromatin immunoprecipitation and luciferase reporter assays were conducted to validate PU.1 as a positive transcriptional regulator of HDAC7. Protein immunoprecipitation coupled with mass spectrometry was used to identify potential substrates of HDAC7. A xenograft assay was performed to evaluate the therapeutic potential of HDAC7 inhibitors in ARID1A-deficient HCC. Results: Our investigation demonstrated that the loss of ARID1A in HCC cells led to an upregulation of HDAC7 expression. Mechanistic analyses revealed that ARID1A deficiency facilitated PU.1-mediated transcriptional regulation of HDAC7. Additionally, the overexpression of HDAC7 resulted in a reduced acetylation level of Enolase 1 (ENO1), thereby enhancing the malignant proliferation of HCC cells. Targeting HDAC7 effectively inhibited the growth of ARID1A-deficient tumors in tumor-bearing mice. Conclusions: This study provides novel insights into the regulatory interplay between chromatin remodeling and acetylation modification, 2 pivotal epigenetic processes influencing tumorigenesis. It also suggests that pharmacological inhibition of HDAC7 offers a promising therapeutic strategy for treating HCC with ARID1A mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c41de19c6a3fb6defa2ffb422608efe57057f498" target='_blank'>
              ARID1A deficiency promotes malignant proliferation of hepatocellular carcinoma by activating HDAC7/ENO1 signaling pathway
              </a>
            </td>
          <td>
            Shanshan Zhang, Gang Wu, Lan Shi, Pengsheng Ding, Gan Liu, Wenjiao Chang, Yuanyuan Dai, Xinghua Han, Xiaoling Ma
          </td>
          <td>2025-06-19</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="VIRMA (also known as KIAA1429), as a core regulatory subunit of the m6A methyltransferase complex, plays a key role in tumorigenesis and progression by dynamically regulating RNA methylation modifications. Studies have shown that VIRMA is aberrantly overexpressed in more than 20 types of malignant tumors, including liver cancer, breast cancer, and lung cancer, and is significantly associated with chromosome 8q amplification and poor prognosis. Its mechanism of action involves regulating the expression of tumor-associated genes through both m6A-dependent and m6A-independent pathways, thereby promoting tumor proliferation, metastasis, and drug resistance. These findings suggest that VIRMA has the potential to serve as a pan-cancer diagnostic and prognostic biomarker. This review summarizes the role of VIRMA in malignant tumors from multiple perspectives and explores its potential applications in clinical diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1f0fb758f358894db60bb1aa6efc139d365cc4" target='_blank'>
              The Multifaceted Role of VIRMA, a Core Component of the Methyltransferase Complex, in Cancer and Cancer Therapy
              </a>
            </td>
          <td>
            Jinmeng Lu, Chengyu Zhang, Mengshuang Yin, Huili You, Chao Xiong, Jing Wu, Ying Gong, Zhangang Xiao, Jing Shen
          </td>
          <td>2025-06-22</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Aberrant DNA methylation is a hallmark of colorectal cancer (CRC), contributing to tumor progression through the silencing of tumor suppressor genes and activation of oncogenes. Indicaxanthin (IND), a dietary betalain pigment from Opuntia ficus indica, has shown antiproliferative effects in CRC models, yet its epigenetic impact remains unexplored. In this study, we investigated the effects of IND on the methylome of Caco-2 cells using Reduced Representation Bisulfite Sequencing (RRBS). IND induced a global hypermethylation profile, particularly at gene promoters and CpG islands. Among the differentially methylated genes, 60% were protein-coding, and 10% encoded transcription factors, including PAX5 and TFAP4, both hypermethylated at active enhancers. Functional enrichment analysis revealed pathways beyond canonical intestinal functions, suggesting altered cell identity and plasticity. Transcription factor targets (SOX10, NFKB1, AHR, ARNT) were significantly enriched among the affected genes, several of which are involved in transdifferentiation processes. Methylation changes also indicated potential reprogramming toward epithelial cell types from pulmonary or neuroectodermal origin. Moreover, IND induced selective hypomethylation of Alu elements on chromosome 21 and hypermethylation of rDNA loci, hinting at suppressed ribosomal biogenesis. Overall, these findings highlight the epigenetic remodeling potential of IND and its possible role in modulating cell fate and metabolism in CRC cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aad87b105620f84af351d2656d30853c48a91d89" target='_blank'>
              Epigenetic Remodeling of Regulatory Regions by Indicaxanthin Suggests a Shift in Cell Identity Programs in Colorectal Cancer Cells
              </a>
            </td>
          <td>
            M. Ragusa, C. Gentile, Aldo Nicosia, Salvatore Costa, Sara Volpes, Laura Greco, F. Naselli, Fabio Caradonna
          </td>
          <td>2025-06-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Inflammatory breast cancer (IBC), of which HER2 + is the predominant subtype, is extremely aggressive and difficult to treat. Previous studies have suggested that targeting the tumour microenvironment (TME) may provide new directions for IBC diagnosis and treatment. Methods In this study, we used single-cell transcriptome technology (scRNA-seq) to investigate the molecular features of the TME of HER2 + IBC patients and performed a comprehensive and detailed comparison of the cellular components and molecular phenotypes of the TME between IBC patients and noninflammatory breast cancer (nIBC) patients to elucidate the cell types that are specifically enriched in the TME of IBC patients, as well as the molecular features that are responsible for the preferential remodelling of the cellular functional state in the TME. Results A total of 15,832 cells, including epithelial cells, endothelial cells, stromal cells, T cells, B cells, antibody secreting cells (ASCs) and myeloid cells, were obtained from tumour tissues from 3 HER2 + IBC patients for scRNA analysis. By comparing the TME with that of HER2 + nIBC patients in a public database, we found that the TME of HER2 + IBC patients had a greater level of lymphocyte infiltration than that of nIBC patients did, and an especially significant enrichment of ASCs (mainly plasmablasts or plasma cells). In the TME of HER2 + IBC patients, tumour-infiltrating T cells exhibited a dual molecular phenotype of high activation and high exhaustion, with tumour-infiltrating B cells preferring the extrafollicular developmental pathway, and tumour-infiltrating myeloid and mesenchymal cells exhibiting a greater immunosuppressive status. By performing a cellular interaction analysis, we revealed that PTN molecules were significantly overexpressed in HER2 + IBC tumour cells and that the cellular interactions mediated by these molecules were strongly correlated with the functional polarisation of the cellular components in the TME. We observed that HER2 + IBC tumour cells have an active interferon response and epithelial mesenchymal transition (EMT) signalling, and that their malignant process is strongly correlated with the inflammatory response. Moreover, we found that HER2 + IBC tumour-infiltrating B cells promoted necroptosis of endothelial cells through high expression of TNF, thus promoting inflammatory responses. Conclusion We found a strong correlation between high expression of PTN molecules in HER2 + IBC tumour cells and their highly invasive characteristics and highly immunosuppressive TME. These results suggest that HER2 + IBC tumour cells can promote an inflammatory response by upregulating the expression of TNF molecules in B cells via PTN molecules and that the enhanced inflammatory response in turn promotes tumour progression, a malignant cycle that shapes a more immunosuppressive TME. Therefore, diagnostic and therapeutic strategies targeting the PTN-TNF molecular axis may have considerable potential for development in HER2 + IBC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03454-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9c28eda21e6edafced575cd62b5c13be2df5e23" target='_blank'>
              Single-cell transcriptome analysis reveals the malignant characteristics of tumour cells and the immunosuppressive landscape in HER2-positive inflammatory breast cancer
              </a>
            </td>
          <td>
            Juan Huang, Yong-Gui Zhu, Wenjing Zeng, Yulong Zhang, Weizhi Xia, Fan Xia, Liyu Liu, Kuansong Wang, Yidi Guan, Taohong Shen, Bingjian Jiang, Lunquan Sun, Ayong Cao, Shouman Wang, Zhi Li
          </td>
          <td>2025-07-08</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaee95d8b33babc294494d923d1f29bffe920cba" target='_blank'>
              Gain of Function p53 mutant R273H confers distinct methylation profiles and consequent partial or full EMT states to colon tumour
              </a>
            </td>
          <td>
            H. Rani, Seemadri Subhadarshini, M. Jolly, V. Mahadevan
          </td>
          <td>2025-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5484870331a7d0dee6989fa1ddf5ea8e0b01c6cb" target='_blank'>
              Functional genomics and tumor microenvironment analysis reveal prognostic biological subtypes in Mantle cell lymphoma
              </a>
            </td>
          <td>
            Sunandini Sharma, Roshia Ali, Alyssa Bouska, Dylan Jochum, Meghana Kesireddy, Simeon Mahov, Joseph Lownik, Weiwei Zhang, Waseem G Lone, Mahfuza Afroz Soma, Alicia Gamboa, Vaishnavi Devarkonda, Dalia Elgamal, Atqiya Fariha, Adnan Mansoor, Douglas Stewart, Peter Martin, Brian K. Link, R. Advani, Paul M. Barr, A. Goy, A. Mehta, M. Kamdar, D. Stephens, Veronika Bachanova, L. Smith, Ryan D Morin, P. Pararajalingam, M. Lunning, Kai Fu, Dennis Wiesenberger, Wing C. Chan, J. Khoury, T. Greiner, J. Vose, Akil Merchant, Chengfeng Bi, J. Iqbal
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="Background Transcriptomic and genomic analyses of bladder cancer (BC) reveal a highly diverse disease stratified into molecular subtypes with distinct molecular features and biological behaviors. Intratumor heterogeneity (ITH) and plasticity can significantly impact diagnosis and patient management, yet their extent in BC remains highly debated. Here, we investigated whether the three main bladder cancer subtypes maintain or alter their identity in response to changes in the microenvironment and during metastatic colonization. Methods Seven patient-derived xenograft (PDX) models representing the major BC subtypes were propagated into three distinct tissue microenvironments: subcutaneous, mammary fat pad and under the kidney capsule. Metastatic lesions were generated via systemic injection of tumor cells. Tumor samples were analysed using RNA- and exome sequencing, SNP-arrays and histopathology to assess subtype fidelity, genomic evolution, and clonal dynamics. Results A comprehensive, longitudinal multiomics analysis showed that tumors consistently maintain their molecular subtype, as well as their transcriptomic and genomic profiles, across different environments. No evidence of emerging ITH or subtype transitions was observed, regardless of the microenvironment. The transcriptomic adaptations observed in metastases and different implantation sites are limited and are associated primarily with hypoxia, epithelial-mesenchymal transition (EMT), and invasion. Conclusions Our results suggest that invasive bladder cancers have a strong intrinsic tumor identity that is not easily reprogrammed by the microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00682-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68687878ba972fb7712b212201473f3b4152d812" target='_blank'>
              Bladder cancer subtypes exhibit limited plasticity across different microenvironments and in metastases
              </a>
            </td>
          <td>
            C. Bernardo, Subhayan Chattopadhyay, Natalie Andersson, P. Eriksson, Benjamin Medle, Lena Tran, Nour Al Dain Marzouka, Adam Mattsson, Aymeric Zadoroznyj, Malin Larsson, F. Liedberg, Mattias Höglund, G. Sjödahl
          </td>
          <td>2025-07-02</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Glioma is a highly malignant primary tumor of the central nervous system, characterized by highly invasive behavior and a high recurrence rate. The current treatment options for gliomaoften have unsatisfactory outcomes. Replication protein A1 (RPA1), a component of the RPA heterotrimer, has been identified as a potential oncogene implicated in the development and clinical prognosis of various types of solid tumors, including liver cancer, nasopharyngeal carcinoma and gastrointestinal cancer. However, studies on the associations between RPA1 expression and glioma cell proliferation or patient prognosis are limited, and the underlying mechanisms remain unclear. The aim of the present study was to explore the expression of RPA1 in glioma and its clinical relevance in the clinicopathology and prognosis of glioma. In addition, the biological functions and signal transduction pathways mediated by RPA1 were predicted using bioinformatics analysis to provide basic theoretical support for further mechanistic research and in vivo experiments. The Cancer Genome Atlas (TCGA) data analysis, and reverse transcription-quantitative PCR, western blot analysis and immunohistochemical (IHC) staining were performed to analyze RPA1 expression in glioma cells or tissues of different grades. The associations between RPA1 expression and various pathological parameters related to tumor cell proliferation and patient prognosis were also investigated. Furthermore, the biological processes and signaling pathways influenced by RPA1-related target genes were predicted. The results revealed that RPA1 expression varies among different grades of gliomas, with higher World Health Organization (WHO) grades exhibiting higher RPA expression levels. Furthermore, both bioinformatics analysis and IHC staining results revealed that high RPA1 expression was associated with a shorter overall survival (OS). In addition, RPA1 expression exhibited a significant association with WHO glioma grade and key markers of malignancy, namely Ki-67 expression and p53 mutation status. Gene Ontology and Kyoto Encyclopedia of Genes and Genomes enrichment analyses suggested that RPA1 may promote the development of glioma through pathways including ‘DNA replication’, ‘Fanconi anemia pathway’, ‘mismatch repair’, ‘homologous recombination’, ‘nucleotide excision repair’ and ‘cell cycle’. In conclusion, RPA1 expression in glioma tissues and cells is positively associated with the degree of glioma malignancy, and high RPA1 expression is associated with a shorter OS in patients with glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb9154b84dfbf0461fd48c7dd062feb43da7f114" target='_blank'>
              Role and mechanism of RPA1 in the development and progression of glioma
              </a>
            </td>
          <td>
            Zhiming Zhao, Yu Sun, Ping Yin, Quan Zhang, Jianfu Zhang
          </td>
          <td>2025-06-26</td>
          <td>Experimental and Therapeutic Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4eaa0126e7f9afd60d57ab46895cddf6ea4cbe3" target='_blank'>
              Pangenome-based identification of cryptic pathogenic variants in undiagnosed rare disease patients
              </a>
            </td>
          <td>
            S. S. Jang, S. Kim, S. Lee, S. Kim, J. Moon, J. Kim, J. Chae
          </td>
          <td>2025-07-11</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes provided important insights into the molecular pathogenesis of cancer but also revealed some contradictions in the prevailing somatic mutation theory. The evidence that noncoding RNAs (ncRNAs) form an elaborate network that regulates the flow of genetic information in eukaryotic cells offers an explanation for the inconsistencies. ncRNAs comprise a wide variety of molecules that interact with one another as well as with other RNAs, DNA, and proteins, over whose activities they exert a regulatory influence. The outcome of the dynamic interactions of the cell’s biomolecules is the emergence of higher-order states of equilibrium, called attractor states, which correspond to the gene-expression configurations of distinct cell types. Attractor states are relatively stable systems, but they are susceptible to perturbation by a disturbing force, such as mutation. Mutations that disrupt the ncRNA network can enable the cell to undergo a state transition towards a potentially neoplastic one. This is the crux of tumorigenesis. An inquiry into the architecture of the ncRNA network and its role in tumorigenesis is required to complement our knowledge of the well-known cancer genes as well as serve as a guide in the design of new anticancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/751cbcc8f9af147a3e904b8dfcf8de5075149f58" target='_blank'>
              The Primary Role of Noncoding RNA in the Pathogenesis of Cancer
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Estrogen receptor 1 (ESR1; also known as ERα, encoded by ESR1 gene) is the driving transcription factor in breast cancer development and progression. ESR1 genomic action is thought to operate under tight epigenetic control, with its chromatin binding and subsequent transcriptional output heavily reliant on the pioneer transcription factor FOXA1, which renders chromatin accessible for ESR1 binding. However, the exact contribution of the epigenome to selective enhancer activation by ESR1 remains to be fully elucidated. To address this, we employ a massively parallel reporter assay to profile 7576 individual ESR1 binding sites for hormone responsiveness. Only a minority of ESR1-occupied enhancers exhibit hormone-induced activity. These findings are confirmed by genomic data in situ, indicating that enhancer activation within a chromatinized context is robustly captured in a plasmid-based reporter assay. In silico integration of our findings with publicly available functional genomics data sets from breast cancer cell lines and tumor samples reveal distinct transcription complex compositions, 3D genome contexts, and regulatory dynamics associated with different classes of ESR1 binding sites. Overall, our results establish a comprehensive framework to highlight and elucidate the molecular basis underlying ESR1 genomic heterogeneity and its contribution to breast cancer biology and clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d35cc452fcf0927d4ead83569deaccc08af058d" target='_blank'>
              Comprehensive functional annotation of ESR1-driven enhancers in breast cancer reveals hierarchical activity independent of genomic and epigenomic contexts.
              </a>
            </td>
          <td>
            Y. Zekri, Sebastian Gregoricchio, Elif Yapıcı, C. Huang, Tunc Morova, Umut Berkay Altıntaş, Gozde Korkmaz, Nathan A. Lack, W. Zwart
          </td>
          <td>2025-06-10</td>
          <td>Genome research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Unchecked growth in three-dimensions (3D) in culture is a key feature of immortalized cells on the path to malignant transformation and hence a potential target phenotype for prevention. Also, expression of genes driving this process, but not that of 2D growth, would likely be more specific to cancer development and their inhibition would be less toxic to normal cells, many of which can grow in 2D but rarely in 3D culture. To define such genes, we compared CRISPR depletion screens performed in HBLAK, a spontaneously immortalized, non-tumorigenic human urothelial cell line, grown in 2D to those in 3D. Using the CRISPR Bassik DTKP (drug target kinase phosphatase) deletion library targeting 2,333 genes, we identified 85 genes which were specifically lost in 3D cultures. Prioritizing hits to those associated with bladder cancer in patients provided us with a set of 11 genes. Only one gene, MAPK1 remained relevant if a human pan-cancer criteria was applied. Single gene in vitro validation confirmed that MAPK1 inhibition was specific to 3D growth. We also found that MAPK1 depletion led to significant growth reductions in human tumor xenografts in vivo. Inhibition of MAPK1 by Ulixertinib, an orally active MAPK1 inhibitor, led to human bladder cancer growth inhibition in both 3D in vitro and in vivo models. In summary, screening for genes specifically driving 3D growth in immortalized cells may provide targets for both prevention and early therapy in bladder and other cancers while potentially limiting therapeutic toxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c39ec53d27180d1be55595d4ca83d125348b0422" target='_blank'>
              Genes driving three-dimensional growth of immortalized cells and cancer
              </a>
            </td>
          <td>
            Mukta Basu, Jin-Fen Xiao, Saravana Kumar Kailasam Mani, Fangyuan Qu, Yongqi Lin, Jason Duex, Huihui Ye, Vanessa Neang, Dan Theodorescu
          </td>
          <td>2025-06-10</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Three Prime Repair Exonuclease 1 (TREX1) is the most abundant 3' exonuclease in human cells, playing a pivotal and complex role in both autoimmune regulation and cancer immunity. Despite its importance, existing methods have been insufficient for directly tracking the real-time dynamics of endogenous TREX1 in living cells. To address this gap, we developed a dual-modified DNA fluorescent probe (Probe-TREX1) that enables rapid quantification of TREX1 activity, live-cell imaging across subcellular compartments, and visualization of its distribution in tumor and adjacent nontumor tissues. Our results reveal that TREX1 levels in the nuclei of normal and cancer cells are comparable to or even higher than those in the cytoplasm. Upon stimulation with anticancer drugs, nuclear TREX1 levels, especially in euchromatin regions, rose rapidly, matching or exceeding the increases seen in the cytoplasm, suggesting a significant role for TREX1 in the nucleus. Notably, TREX1 expression was lower in hepatoblastoma and the other two investigated tumor tissues compared to adjacent non-tumor tissues, highlighting the complexity of TREX1's roles in DNA repair and immune response across different tumor types. This study introduces a valuable molecular tool for advancing our understanding of the diverse functions of TREX1, with a broad potential for evaluating and improving cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e3b92bad7ec2ed80d64938ec4a5c00715b0c783" target='_blank'>
              First Fluorescent Probe for TREX1 Enables Real-Time Tracking of DNA Repair and Immune Regulation in Cancer Cells.
              </a>
            </td>
          <td>
            Ying Sun, Jinghui Zheng, Yupei Zhang, Ruilan Zhang, Xiangjian Cao, Xiang Liu, Jungui Liu, Weihong Duan, Meiping Zhao
          </td>
          <td>2025-06-16</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c71b63f87cb576d755f4a88747d8d75f40eef18" target='_blank'>
              Multimodal profiling unveils a reversible basal-like breast cancer cell state resistant to AKT inhibition
              </a>
            </td>
          <td>
            Colin D. H. Ratcliffe, Hugh Sparks, G. L. Boezio, Yuriy Alexandrov, Robert P. Jenkins, A. Le Marois, Pablo Soro-Barrio, Rebecca Lee, S. Strohbuecker, Sudeep Joshi, James K. Ellis, Anne-Marie Fortier, Nils Gustafsson, Ana C. Cunha, Morag Park, J. Macrae, A. Serio, J. Briscoe, Andy Riddell, S. Barry, Chris Dunsby, E. Sahai
          </td>
          <td>2025-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="XPO1 is a nuclear export receptor that is essential for cell survival. Previous genomic analyses have identified recurrent XPO1 hotspot mutations in cancer. Here, we conducted a large-scale genomic analysis of 217,570 cancer patients to identify and characterize XPO1 variants from real-world patient tumors. XPO1 harbored a R749Q mutation in various solid tumors, with a clear enrichment in endometrial and colorectal cancers, and XPO1R749Q mutations significantly co-occurred with POLE mutations. Analysis of isogenic colon cancer cell lines revealed that XPO1R749Q localized more in the cytoplasm than wildtype XPO1, with enhanced export of a large number of proteins. Structural modeling of XPO1R749Q suggested an increase in RanGTP affinity, which is consistent with enhanced protein export. A compound library screen using over 200 FDA-approved anticancer drugs indicated a general trend towards chemoresistance, specifically to topoisomerase I inhibition, in XPO1R749Q mutant cells. Mechanistically, XPO1R749Q mutant cells exhibited enhanced DNA damage response via RPA phosphorylation in response to topoisomerase I inhibition. Combining XPO1 and topoisomerase I inhibitors reduced DNA damage-induced RPA phosphorylation and mediated synergistic antitumor effects in cells harboring the XPO1R749Q mutation. Finally, the combination of selinexor and irinotecan overcame chemotherapeutic resistance in xenograft mouse models, prolonging survival. These findings suggest that XPO1 alterations in cancer are selected for in POLE mutant tumors and may confer resistance to DNA-damaging chemotherapies, which have implications for patients with tumors bearing XPO1R749Q and for XPO1 inhibitor development in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e52bf0adf008e92bc55a03de218e8fe730ea2f" target='_blank'>
              XPO1R749Q Mutations Co-occur with POLE Mutations in Cancer and can be Targeted to Overcome Chemoresistance.
              </a>
            </td>
          <td>
            T. Totiger, Wannasiri Chiraphapphaiboon, Y. Baca, S. Chaudhry, R. Notti, S. Montoya, M. Chojnacka, Gabriel Gaidosh, Jumana Afaghani, Maurizio Affer, Chris Armstrong, Paul M Kavanaugh, Efe Karaca, Jenna Zabroski, Michael Lewis, Alyssa Maye, Jacob Jahn, R. Soni, Daniel Bilbao, Phil Walker, Andrew Elliott, E. Lou, Wafik S. El-Deiry, Josean Rodriguez, Hai Dang Nguyen, Justin Taylor
          </td>
          <td>2025-06-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Oncogene activation through DNA amplification or overexpression plays a critical role in cancer initiation and progression. Chromosome 17 harbors numerous cancer-associated genetic anomalies, some of which are strongly linked to poor prognosis in breast cancer patients. The FOXK2 gene, located on 17q25, encodes a transcription factor with a forkhead DNA binding domain. Integrative analyses of publicly available genomic datasets reveal that FOXK2 is frequently amplified and overexpressed in breast cancer, correlating with poor overall survival. FOXK2 knockdown via shRNA significantly inhibited the proliferation, migration, and anchorage-independent growth of breast cancer cells, causing G0/G1 cell cycle arrest. The C-terminal nuclear localization signal (NLS) is crucial for FOXK2's oncological function. Additionally, FOXK2 knockdown in MCF7 breast cancer cells delayed tumor growth in a xenograft mouse model. Inhibiting FOXK2 expression also sensitized breast cancer cells to frontline chemotherapy agents, including doxorubicin, 5-fluorouracil, and etoposide in vitro. Co-overexpression of FOXK2 and PI3KCA with oncogenic mutations (E545K or H1047R) induced the transformation of non-tumorigenic MCF-10A cells, suggesting that FOXK2 acts as an oncogene in breast cancer and contributes to PI3KCA-driven tumorigenesis. CCNE2, PDK1, and ESR1 were identified as direct transcriptional targets of FOXK2 in MCF-7 cells. Moreover, co-inhibition of FOXK2 function, either through knockdown or inhibition of its transcriptional targets (CCNE2/CDK2 by Dinaciclib or PDK1 by DCA) and PI3KCA by alpelisib, had synergistic anti-tumor effects in breast cancer cells with FOXK2 overexpression and PI3KCA oncogenic mutations. FOXK2 functional inhibition by the CDK2 inhibitor Dinaciclib enhanced the tumor-suppressive effect of alpelisib in a MCF7 breast cancer xenograft mouse model. These findings provide compelling evidence that FOXK2 plays an oncogenic role in breast tumorigenesis. The combination of PIK3CA and CCNE2/CDK2 inhibition by alpelisib and Dinaciclib offers a potential therapeutic strategy for breast cancer with FOXK2 overexpression and PIK3CA oncogenic mutation.
 Citation Format: Hong Zhang, Yang Yu, Weng-Ming Cao, Feng Cheng, Edaise da silva, Higinio Dopeso, Hui Chen, Xiaosong Wang, Chunchao Zhang. Targeting FOXK2 signaling in breast cancers with FOXK2 amplification/overexpression and PIK3CA oncogenic mutations: a promising therapeutic strategy? [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-02-24.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/caaa74a368ec1f757b26670f47abf7954355f919" target='_blank'>
              Abstract P5-02-24: Targeting FOXK2 signaling in breast cancers with FOXK2 amplification/overexpression and PIK3CA oncogenic mutations: a promising therapeutic strategy?
              </a>
            </td>
          <td>
            Hong Zhang, Yang Yu, Wen-Ming Cao, Feng Cheng, E. D. da Silva, Higinio Dopeso, Hui Chen, Xiaosong Wang, Chunchao Zhang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Extensively studied over the past four decades, the TP53 gene has emerged as a pivotal watchman in cellular defense and a key factor in cancer biology. TP53 is the most frequently mutated gene in human malignancies, 50% of which carry alterations to it. Initially, the functions of p53 were thought to be restricted to cell‐cycle arrest and apoptosis. With time, however, a growing number of functions have been discovered, illustrating p53's role as a master switch between any cellular stress and cellular or multicellular responses that contribute to its anti‐tumor activity. Indeed, the peculiar landscape of TP53 mutations and its high heterogeneity are linked both to the structure of the protein and its ubiquitous function in regulating cellular homeostasis. Mutations in p53 are associated with poor response to therapy and shorter survival in most cancer types, and the diagnosis of p53 mutations is currently used to improve case management in some types of leukemia and lymphoma. Although TP53 has been defined as a tumor suppressor gene, overexpressed mutated p53 variants found in human tumors are defined as dominant oncogenes with a potential gain of function, which makes the gene a very attractive target for developing new cancer treatments. Beyond its role in cancer, our review also highlights TP53's significance in non‐neoplastic conditions, such as bone marrow failure syndromes and certain developmental disorders, where chronic p53 activation plays a crucial role in cellular stress responses, demonstrating its broader biological importance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f422279b37d3fc7b3177be1ae5e2e34940dcc8f7" target='_blank'>
              The TP53 tumor suppressor gene: From molecular biology to clinical investigations
              </a>
            </td>
          <td>
            Panagotis Baliakas, Thierry Soussi
          </td>
          <td>2025-06-16</td>
          <td>Journal of Internal Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8503e81194e8e95ebd9edbe7edc4c0fd9d960853" target='_blank'>
              Mechanisms of resistance to VHL loss-induced genetic and pharmacological vulnerabilities
              </a>
            </td>
          <td>
            Jianfeng Ge, Shoko Hirosue, Saroor A. A. Patel, L. Wesolowski, Anna Dyas, Cissy Yong, Leticia Castillon, Sanne de Haan, J. Drost, Grant D. Stewart, Anna C Obenauf, D. Muñoz-Espín, Sakari Vanharanta
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Organotropism results from the functional versatility of metastatic cancer cells to survive and proliferate in diverse microenvironments. This adaptivity can originate in clonal variation of the spreading tumor and is often empowered by epigenetic and molecular reprogramming of cell regulatory circuits. Related to organotropic colonization of metastatic sites are environmentally-sensitive, differential responses of cancer cells to therapeutic attack. Accordingly, understanding the organotropic profile of a cancer and probing the underlying driver mechanisms are of high clinical importance. However, determining systematically the organotropism of one cancer versus the organotropism of another cancer, potentially with the granularity of comparing the same cancer type between patients or tracking the evolution of a cancer in a single patient for the purpose of personalized treatment, has remained very challenging. It requires a host organism that allows observation of the spreading pattern over relatively short experimental times. Moreover, organotropic patterns often tend to be statistically weak and superimposed by experimental variation. Thus, an assay for organotropism must give access to statistical powers that can separate ‘meaningful heterogeneity’, i.e., heterogeneity that determines organotropism, from ‘meaningless heterogeneity’, i.e., heterogeneity that causes experimental noise. Here we describe an experimental workflow that leverages the physiological properties of zebrafish larvae for an imaging-based assessment of organotropic patterns over a time-frame of 3 days. The workflow incorporates computer vision pipelines to automatically integrate the stochastic spreading behavior of a particular cancer xenograft in tens to hundreds of larvae allowing subtle trends in the colonization of particular organs to emerge above random cell depositions throughout the host organism. We validate our approach with positive control experiments comparing the spreading patterns of a metastatic sarcoma against non-transformed fibroblasts and the spreading patterns of two melanoma cell lines with previously established differences in metastatic propensity. We then show that integration of the spreading pattern of xenografts in 40 – 50 larvae is necessary and sufficient to generate a Fish Metastatic Atlas page that is representative of the organotropism of a particular oncogenotype and experimental condition. Finally, we apply the power of this assay to determine the function of the EWSR1::FLI1 fusion oncogene and its transcriptional target SOX6 as plasticity factors that enhance the adaptive capacity of metastatic Ewing sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5025875828f543583bc1d12169d95499fa95592" target='_blank'>
              Robust statistical assessment of Oncogenotype to Organotropism translation in xenografted zebrafish
              </a>
            </td>
          <td>
            D. Saucier, Xuexia Jiang, Divya Rajendran, Roshan Ravishankar, Erin Butler, A. Marchetto, R. Kurmasheva, T. G. Grünewald, James F. Amatruda, G. Danuser
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="DEAD-box helicase 41 (DDX41) is implicated in germline (GL)-predisposed myeloid neoplasms, where pathogenic GL variants often lead to disease following the acquisition of a somatic variant in trans, most commonly p.R525H. However, the precise molecular mechanisms by which DDX41 variants contribute to the pathogenesis of myeloid neoplasms remain poorly understood, partly due to challenges in establishing cellular and animal models that faithfully recapitulate the human disease phenotype. This limitation highlights the necessity of directly analyzing primary human disease cells. In this case report, conducted to pursue this objective, we implemented single-cell RNA sequencing integrated with genotyping at the p.R525 locus in a myelodysplastic neoplasm (MDS) harboring both germline and somatic DDX41 variants, leveraging highly efficient Terminator-Assisted Solid-phase cDNA amplification and sequencing. We found that acquiring p.R525H induced G2/M cell cycle arrest selectively in colony-forming unit-erythroid cells, accompanied by R-loop accumulation, which impaired erythropoiesis through DNA damage. In hematopoietic stem and myeloid progenitor populations, gene expression profiles were largely similar between p.R525H-positive and -negative cells. However, ligand-receptor interaction and transcriptional regulation analyses suggested a non-cell-autonomous influence from p.R525H-expressing cells on GL variant-only cells. This interaction drove convergence toward a shared expression profile, highlighting an intricate interplay shaping the patient’s MDS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43fbd448cf6560b9e09e5f237fa9deac7aa303c9" target='_blank'>
              Genotype-integrated single-cell transcriptome analysis reveals the role of DDX41 pR525H in a patient with myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Hirotaka Matsui, Akiko Nagamachi, Minori Koizumi, Rei Kudo, Masahiko Ajiro, H. Harada, Yuka Harada, Shigeyuki Shichino, Akihide Yoshimi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Throughout the cell cycle, DNA molecules convert between hierarchical intramolecular (cis) and intermolecular (trans) associations. Cohesin ATPase complexes produce both types of DNA associations which collectively are required for sister chromatid segregation, chromatin condensation, genomic architecture, gene transcription, and DNA repair. The mechanisms that regulate cohesin cis- and trans-activities, however, remain controversial. A popular model is that a regulatory complex (Pds5, Irr1/Scc3, and Rad61) sits atop a core ring-like complex (Mcd1/Scc1, Smc1, and Smc3), the latter of which exhibits the inherent ATPase activities responsible for producing cis-and trans-DNA conformations. Additional proteins transiently interact with cohesins to promote cohesin deposition onto DNA (Scc2 and Scc4) or stabilize cohesin-DNA binding (Eco1/Ctf7). Of these nine components, only RAD61 is non-essential. Recent findings, however, identified pairs of suppressor mutations that support the viability of cells individually deleted for either PDS5 or ECO1/CTF7 (herein ECO1). Intriguingly, CLN2 deletion is common in both suppressor pairs, suggesting that combined suppressor mutations may support the viability of cells co-deleted for both ECO1 and PDS5. These results further suggest that the addition of other suppressor mutations (such as ELG1 and RAD61) may support the viability of cells deleted of all auxiliary subunits - including IRR1/SCC3 (herein SCC3). Here, we test these predictions and report on novel gene deletion combinations required for cell cycle progression and cell viability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a5f10ca02abc4152347a73b1ca267645760052e" target='_blank'>
              Analysis of Combinatorial Cohesin Subunit Gene Deletions in Budding Yeast.
              </a>
            </td>
          <td>
            Grace Duke, R. Skibbens
          </td>
          <td>2025-06-10</td>
          <td>Genetics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The nucleolus is a large membraneless subnuclear structure, the main function of which is ribosome biogenesis. However, there is growing evidence that the function of the nucleolus extends beyond this process. While the nucleolus is the most transcriptionally active site in the nucleus, it is also the compartment for the location and regulation of repressive genomic domains and, like the nuclear lamina, is the hub for the organization of inactive heterochromatin. Studies in human and Drosophila cells have shown that a decrease in some nucleolar proteins leads to changes in nucleolar morphology, heterochromatin organization and declustering of centromeres. This work is devoted to the study of the effects of Novel nucleolar protein 3 (Non3) gene mutations in D. melanogaster on the organization of chromatin in the nucleus. Previously, it was shown that partial deletion of the Non3 gene leads to embryonic lethality, and a decrease in NON3 causes an extension of ontogenesis and formation of a Minute-like phenotype in adult flies. In the present work, we have shown that mutations in the Non3 gene suppress the position effect variegation (PEV) and increase the frequency of meiotic recombination. We have analyzed the classical heterochromatin markers in Non3 mutants and shown that the amount of the HP1 protein as well as the modification of the histone H3K9me2 do not change significantly in larval brains and salivary glands compared to the control in Western blot analysis. Immunostaining with antibodies to HP1 and H3K9me2 did not reveal a significant reduction or change in the localization patterns of these proteins in the pericentromeric regions of salivary gland polytene chromosomes either. We analyzed the localization of the HP1 protein in Non3 mutants using DNA adenine methyltransferase identification (DamID) analysis and did not find substantial differences in protein distribution compared to the control. In hemocytes of Non3 mutants, we observed changes in the morphology of the nucleolus and in the size of the region detected by anti-centromere antibodies, but this was not accompanied by declustering of centromeres and their untethering from the nucleolar periphery. Thus, the NON3 protein is important for the formation/function of the nucleolus and is required for the correct chromatin packaging, but the exact mechanism of NON3 involvement in these processes requires further investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14d67fd0595392b8b07d4241837bf9f0f2261d7f" target='_blank'>
              The effects of Non3 mutations on chromatin organization in Drosophila melanogaster
              </a>
            </td>
          <td>
            A. A. Yushkova, A. Ogienko, E. N. Andreyeva, A. Pindyurin, A. E. Letiagina, E. S. Omelina
          </td>
          <td>2025-06-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54cf680837c223860a295c7d973297324c3a3da0" target='_blank'>
              From predictive biomarker to therapeutic target: the dual role of SLFN11 in chemotherapy sensitivity.
              </a>
            </td>
          <td>
            Yue Feng, Yingze Li, Zhenhao Zhang, Yuxuan Dai, Xingchun Gou, Kejing Lao, Running Zhang
          </td>
          <td>2025-06-18</td>
          <td>Cancer chemotherapy and pharmacology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="R-loops, three-stranded RNA-DNA hybrid nucleic acid structures, are recognized for their roles in both physiological and pathological processes. Regulation of R-loops is critical for genome stability as disruption of R-loop homeostasis can lead to aberrant gene expression, replication stress, and DNA damage. Recent studies suggest that the RNA modification, N6-methyladenosine (m6A), can modify R-loops and the writers, erasers, and readers of m6A are involved in the dynamic regulation of R-loops. Here, we discuss the reported functions of various m6A regulatory proteins in relation to R-loops, highlighting their distinct roles in recognizing and modulating the formation, stability, and resolution of these structures. We further examine the functional implications of m6A and R-loop interaction in human diseases, with a particular emphasis on their roles in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/178d222f0ed2c1781f83d81616226ea68d9cb6b9" target='_blank'>
              The Dynamic Interactions of m6A Modification and R-Loops: Implications for Genome Stability
              </a>
            </td>
          <td>
            Nicholas Kim, Hong Sun
          </td>
          <td>2025-06-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract The homologous recombination (HR) system repairs DNA double-strand breaks produced by the DNA damage response, which is a complex signaling pathway consisting of the key proteins BRCA1/2 and other DNA repair proteins, such as the ATM, PALB2, BARD1, RAD51, and Fanconi anemia proteins. Mutations and epigenetic alterations in HR-related genes may lead to homologous recombination deficiency (HRD), resulting in genomic instability and contributing to the development of certain solid tumors. The biological significance and molecular mechanism of BRCA1/2 mutation-related HRD are well understood, but the relationships of other HR-related genes and their variant forms with HRD have not been sufficiently studied. These genes exhibit multiple forms of variation, including one or more HR genes, germline or somatic mutations, monoallelic or biallelic variants, and not all variants present HRD. Therefore, HRD is usually defined as HR-related gene variation, but recent studies have shown that defining it as the combined score of loss of heterozygosity, LST and TAI, known as the HRD score, can more accurately assess genomic instability. In patients with HRD, platinum-based therapy and poly ADP-ribose polymerase enzyme inhibitor (PARPi) have been shown to perform well in ovarian, breast, and prostate cancers. For gastrointestinal cancer (GI cancer), HRD has been relatively well studied in pancreatic cancer, but its role in other cancers has rarely been reported. Herein, we review the pathogenesis and predictive value of HRD, including the use of platinum drugs, PARPi, and immunotherapy, in digestive system tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b86ad0d0eb3efaff60321645af70d194c1741a2" target='_blank'>
              The prognostic and predictive value of homologous recombination deficiency in gastrointestinal cancer
              </a>
            </td>
          <td>
            Beibei Jiang, Yile Shang, Xiang Zhang, Wenguang He, Hanju Hua, Feng Ye, Xile Zhou, Yandong Li, Weixiang Zhong, Wei-wei Jiang, Guosheng Wu
          </td>
          <td>2025-06-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="How the host immune system loses its surveillance function during the evolution from normal cell to malignancy is still largely unknown. Here, we investigate the dynamics changes of the pancreatic ductal adenocarcinoma (PDAC) tumor microenvironment by profiling 132,115 single-cell transcriptomes derived from 51 tissues, including healthy pancreatic tissue, non-metastatic PDAC primary tumors, metastatic primary tumors, and patient-matched liver metastases. The cellular proportion, bio-functional, and interaction between each cell type are carefully characterized. Aberrant copy number variations (CNVs) indicating malignant intensity are identified at chromosomes 7 and 20 of epithelial cells during tumor development. A bio-functional transition of predominant genes from physiology to pancreatic oncogenesis and metastasis is observed. Combinatorial analysis of epithelial cells and immunocytes indicates a gradient loss of immune surveillance during the malignant transformation. By dissecting cellular interaction, we unravel an incremental tumor cell-triggered apoptosis of DCs through molecular pair ANXA1-FPR1/3. Consequently, the activation and infiltration of cytotoxic CD8+ T cells are dampened progressively. Remarkably, we unveil a novel subtype of stress-response NK cells (strNK), which are characterized by robust proliferation, diminished cytolytic capabilities, and negative immune regulation. Notably, the presence of strNK cells is associated with poor prognosis of PDAC patients, implying a potential pro-tumor function. Taken together, our results not only shed light on the intricate mechanisms underlying step-wise evasion of immune surveillance during PDAC tumor development, but also provide a potential strategy for holding back malignant transition by reinforcing DCs’ function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f087912453ba83b002b8d59f2aad4f9cb821703a" target='_blank'>
              Single-cell transcriptional dissection illuminates an evolution of immunosuppressive microenvironment during pancreatic ductal adenocarcinoma metastasis
              </a>
            </td>
          <td>
            Xiaowei Liu, Jinen Song, Meiling Yuan, Fengli Zuo, Huihui Li, Leyi Tang, Xinmin Wang, Xueyan Wang, Qian Xiao, Li Li, Xinyu Liu, Zhankun Yang, Jian-Lin Wu, Jing Jing, Xuelei Ma, Hubing Shi
          </td>
          <td>2025-06-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Heritable gene activity variations that do not alter the underlying DNA sequence are known as epigenetic modifications. Mutations that disrupt genome structure and function are key drivers of oncogenesis. In addition to genetic mutations that cause direct disruptions in the DNA sequence, epigenetic changes can affect gene expression, which helps the development of cancerous traits. Changes in DNA methylation and histone mark patterns are the main drivers of these epigenetic modifications, as they encourage the growth and spread of tumors. In this review, we explore the substantial implications of epigenetic control on tumor genesis, metastatic behaviour, metabolic pathways, and the tumor microenvironment, delving into the intricacies of this intricate regulation. We pay special attention to the dysregulation at every stage of epigenetic modulation, which includes, but is not limited to, abnormalities in the enzymes that modify histones and methylate DNA, subunit loss or fusions in chromatin remodeling complexes, and disruptions in higher-order chromatin structure. We also highlight the development of inhibitors of medications targeted at epigenetic enzymes and summarize the abnormal roles of enzymes in DNA methylation, histone acetylation, and histone methylation during tumour progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d006bb544ee4859c5808a78f45579e1cfd4a19" target='_blank'>
              Epigenetic regulators in cancer therapy and progression
              </a>
            </td>
          <td>
            Hany E. Marei
          </td>
          <td>2025-06-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mitotic genome folding, or mitotic chromosome assembly, is essential for the faithful segregation of genetic information into daughter cells. While this process was once thought to be highly complex, requiring a myriad of protein components, recent studies have begun to revise this conventional view. An emerging view is that the core reaction of mitotic genome folding is mediated by a dynamic interplay of a limited number of structural components, namely, condensins, topoisomerase II (topo II), and histones. Condensins and topo II are two distinct classes of ATPases that cooperate to actively form and manipulate DNA loops, both accumulating at the central axial regions of the resulting chromosomes. In contrast, nucleosomes and linker histones help to compact DNA loops by cooperating and competing with the action of these ATPases. In this review, I will focus on the recent advances in the field, with an emphasis on the mechanistic aspects of mitotic genome folding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a72d38cae80b2ee1dda7240ab411f39d6901aa41" target='_blank'>
              Mitotic genome folding.
              </a>
            </td>
          <td>
            Tatsuya Hirano
          </td>
          <td>2025-07-07</td>
          <td>The Journal of cell biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Super-enhancers are a super-cluster of enhancers formed by serially arranged regulatory elements that can strongly drive the expression of cell-related genes. Hundreds of SEs in cells affect cell identity and fate-determining processes. Previous studies have verified that the expression of pathogenic genes is highly correlated with the abnormal activation of SEs in malignant tumorigenesis, dementia, diabetes, and many autoimmune diseases. Also, enhancer RNAs (eRNAs) can be regarded as crucial markers for SEs. Here, we summarize the discovery process and basic concepts of SEs, describe the structural characteristics and functional regulation of SEs in different tumor diseases, Alzheimer’s disease, and immune-related diseases, with a focus on typical diseases such as rheumatoid arthritis, systemic lupus erythematosus, and multiple sclerosis. In this review, we also discuss the potential clinical applications of SE, as well as the research prospects in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f255bb941ea062fdb933b412745b2b17de7146b1" target='_blank'>
              Current progress and future perspective of super-enhancers: a viable and effective bridge between the transcriptional apparatus and disease
              </a>
            </td>
          <td>
            Ran Wang, Aiying Li, Zongran Pang
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioma is the most frequent malignant tumor in the brain. Super-enhancer (SE) is a class of transcriptional activator, which drives gene expression. SE-related genes (SERGs) affect occurrence and development of several tumors. We explored the predictive role of SERGs in the prognosis and immune features of glioma. A total of 1557 glioma patients were collected from four data sets, including The Cancer Genomic Atlas (TCGA, n = 691), the Chinese Glioma Genomic Atlas (CGGA) array (n = 286), the CGGA sequencing (n = 316), and GSE16011 (n = 264) from Gene Expression Omnibus (GEO) database. SERGs were selected from SEdb (http://www.licpathway.net/sedb), a comprehensive human SE database. Survival analysis and visualization were performed using the R packages survival (v3.3-1) and survminer (v0.4.9). Immune subtype classification was conducted with the ImmuneSubtypeClassifier (v0.1.0) R package. A nomogram was generated using the rms (v6.7-1) package. A risk score model based on 13 super-enhancer-related genes (SERGs) was constructed, demonstrating that patients in the low-risk group had significantly better prognosis. The SERGs signature significantly correlated with age, molecular and immune subtypes, IDH mutation, MTMG promoter methylation, 1p19q co-deletion, and expression of immune checkpoint genes in glioma patients. The SERGs signature could predict the prognosis and immune features of glioma, and SERGs might serve as novel immunotherapy options for glioma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98722f844f3653a60f19c5d5304fa09c01ae6373" target='_blank'>
              A super-enhancer-related gene signature predicts prognosis and immune microenvironment features in glioma.
              </a>
            </td>
          <td>
            Huijun Li, Bin Luo, Yibadaiti Tulufu, Xiong Wang, Daoyuan Yue
          </td>
          <td>2025-07-06</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b91ec8476ea04c15a90514f720adb0e97fd88bcc" target='_blank'>
              ASXL1 truncating mutations drive leukemic resistance to T cell attack
              </a>
            </td>
          <td>
            D. McCurry, Zhongqi Ge, Jaehyun Lee, Rishi Pasumarthi, Xiaohong Leng, Thomas Koehnke, O. Pasvolsky, Pranaya Raparla, Vinhkhang N. Nguyen, K. Maurer, Shuqiang Li, K. Livak, Emerie Danson, Bijal Thakkar, Elham Azizi, Robert J. Soiffer, S. Shukla, Ravi Majeti, Jerome Ritz, Catherine J. Wu, J. Molldrem, P. Bachireddy
          </td>
          <td>2025-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa7409bece8a61b55b1fe618c705451021becc29" target='_blank'>
              Tissue-Specific Activation of Microhomology-Mediated End Joining with Age Reflects Dynamic Rewiring of DNA Repair in Rats
              </a>
            </td>
          <td>
            Diksha Rathore
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="

Long noncoding RNAs are essential regulators in numerous biological
processes and have been linked to various diseases, including cancer. Despite their initial classification
as transcriptional byproducts, lncRNAs have been shown to modulate chromatin structure,
transcription, RNA processing, protein translation, and intranuclear transport. LINC-PINT, a
lncRNA induced by P53, is particularly noteworthy for its role in tumor suppression across multiple
cancers.



By utilizing the PubMed database and applying inclusion criteria based on relevance,
literature quality, and data availability, we conducted a comprehensive analysis of 128 studies to
provide an overview of the functions of LINC-PINT and its mechanisms of action in cancers.



LINC-PINT was confirmed to function as a tumor suppressor factor in many cancers, such
as triple-negative breast cancer, non-small cell lung cancer, gastric cancer, glioma, melanoma,
osteosarcoma, laryngeal squamous cell carcinoma, esophageal cancer, colorectal cancer, nasopharyngeal
carcinoma, retinoblastoma, ovarian cancer, thyroid cancer, hepatocellular carcinoma, and
pancreatic cancer by promoting apoptosis and senescence, inhibiting proliferation, migration, invasion,
drug resistance, cell stemness, EMT, radioresistance, and DNA damage repair.



LINC-PINT serves as a tumor suppressor with its ability to sponge miRNAs, regulate
epigenetic modulation, DNA damage repair, etc. Despite the promising findings, the complex and
tissue-specific functions of LINC-PINT, along with the need for further clinical validation, underscore
the importance of continued research to fully understand its mechanisms and potential as a
therapeutic target.



LINC-PINT is a potential target in cancer progression and treatment
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c614cdfeae321548caa4f0f7d2f981f35ae5a41" target='_blank'>
              LINC-PINT: A Distinctive Long Non-Coding RNA Functioning as a Potential Suppressor in Tumorigenesis
              </a>
            </td>
          <td>
            Jiayi Li, Yining Pan, Songqiang Li, Cheng Chen, Chengfu Yuan
          </td>
          <td>2025-07-22</td>
          <td>Current Topics in Medicinal Chemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The overexpressed gene MYEOV in multiple myeloma, as an oncogene, has been widely recognized for its high expression levels in various malignant tumors. MYEOV plays a significant role in multiple malignancies, particularly in diseases such as multiple myeloma, breast cancer, lung cancer, pancreatic cancer, and esophageal cancer. The presence of the open reading frame of MYEOV in humans and other primates suggests its potential protein-coding capacity, although direct evidence of functional MYEOV protein is currently lacking. The role of MYEOV in various tumors is not limited to its direct effects as an oncogene; it also includes its complex role in tumor cell signaling pathways and its ability to participate in miRNA regulatory networks as a competing endogenous RNA (ceRNA). Studies have shown that MYEOV may affect the expression of cancer-related genes through enhancer activity. This article aims to provide a comprehensive review of the role of MYEOV, its involvement in signaling pathways in tumor cells, and the latest advancements in MYEOV-targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e799fb69a5049d179cc77bd3716c162048abf1c" target='_blank'>
              The role of MYEOV gene: a review and future directions
              </a>
            </td>
          <td>
            Yidan Xi, Xuefei Liu, Yanan Ge, Mingzhe Jiang, Dong Zhao, Ershu Zhao, Fei Cai, Xinlong Wang, Jiatong Li, Tingting Li, Zhendong Zheng
          </td>
          <td>2025-06-04</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Somatic cell reprogramming into human induced pluripotent stem cells entails significant intracellular changes, including modifications in mitochondrial metabolism and a decrease in mitochondrial DNA copy number. However, the mechanisms underlying this decrease in mitochondrial DNA copy number during reprogramming remain unclear. Here we aimed to elucidate these underlying mechanisms. Through a meta-analysis of several RNA sequencing datasets, we identified genes responsible for the decrease in mitochondrial DNA. We investigated the functions of these identified genes and assessed their regulatory mechanisms. In particular, the expression of the thymidine kinase 2 gene (TK2), located in the mitochondria and required for mitochondrial DNA synthesis, is decreased in human pluripotent stem cells as compared with its expression in somatic cells. TK2 was significantly downregulated during reprogramming and markedly upregulated during differentiation. Collectively, this decrease in TK2 levels induces a decrease in mitochondrial DNA copy number and contributes to shaping the metabolic characteristics of human pluripotent stem cells. However, contrary to our expectations, treatment with a TK2 inhibitor impaired somatic cell reprogramming. These results suggest that decreased TK2 expression may result from metabolic conversion during somatic cell reprogramming.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afec2f521a0fb9cff12157659a7e8ef543de136a" target='_blank'>
              Changes in mitochondrial thymidine metabolism and mtDNA copy number during induced pluripotency
              </a>
            </td>
          <td>
            Hyun Kyu Kim, Yena Song, Minji Kye, Byeongho Yu, Hyungkyu Choi, Sung-Hwan Moon, M. Lee
          </td>
          <td>2025-06-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Alterations in the homologous recombination repair genes, such as BRCA1 and BRCA2, are prevalent in various cancers, presenting a unique opportunity to develop synthetic lethal strategies that target homologous recombination deficiency (HRD). Poly ADP-ribose polymerase inhibitors (PARPis) have been developed to induce synthetic lethality in tumors with HRD by inhibiting the repair of single-strand DNA breaks. Beyond the initial approach to target cancers associated with HRD, the utility of PARPis has expanded to combination therapy with immune checkpoint inhibitors, anti-angiogenic drugs, or anti-androgen drugs based on the molecular biological rationale. In the field of genitourinary (GU) cancer, PARPis, such as olaparib, rucaparib, and talazoparib, are approved by the Food and Drug Administration in metastatic prostate cancer patients with BRCA1/2 mutations, sometimes in combination with other agents (e.g., olaparib plus abiraterone acetate, or talazoparib plus enzalutamide). More recently, pivotal clinical trials have broadened the potential of PARPis to the other GU cancers, including urothelial carcinoma and renal cell carcinoma. In this review, we examine the biomarkers for the response to PARPis beyond mutations in BRCA1/2 and discuss the current state and future perspectives of PARPis in GU cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28ab480620686400ea1a47db842007f871807c33" target='_blank'>
              PARP Inhibitors in Genitourinary Cancer: A New Paradigm Beyond Prostate Cancer.
              </a>
            </td>
          <td>
            Yohei Okuda, Taigo Kato, Y. Ishizuya, Takuji Hayashi, Y. Yamamoto, K. Hatano, A. Kawashima, Junko Murai, Norio Nonomura
          </td>
          <td>2025-06-09</td>
          <td>International journal of urology : official journal of the Japanese Urological Association</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="DNA polymerase delta (Pol δ) is a cornerstone of genomic stability, orchestrating DNA replication and repair through its catalytic subunit, POLD1. This subunit’s 3’–5’ exonuclease domain proofreads replication errors, ensuring fidelity. However, POLD1 mutations—particularly in this domain—disrupt this function, triggering genomic instability and a hypermutated state in cancers. This review delves into the multifaceted roles of POLD1 mutations, spotlighting their contributions to tumorigenesis and immunotherapy responses. Beyond their established link to syndromes like polymerase proofreading-associated polyposis (PPAP), these mutations unexpectedly enhance tumor immunogenicity in microsatellite-stable (MSS) tumors, previously considered largely resistant to immune checkpoint inhibitors (ICIs). By elevating tumor mutation burden and generating unique mutational signatures (e.g., SBS10d), POLD1 mutations sensitize MSS tumors to ICIs, challenging the dominance of microsatellite instability (MSI) as an immunotherapy predictor. Integrating structural insights, molecular mechanisms, and clinical data, this review positions POLD1 mutations as both a driver of cancer progression and a promising biomarker, redefining therapeutic possibilities in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02d2442be2c75ad66ffe5186a1b712852b468974" target='_blank'>
              Beyond proofreading: POLD1 mutations as dynamic orchestrators of genomic instability and immune evasion in cancer
              </a>
            </td>
          <td>
            Huiqing Chen, Jiayu Wei, Qi Tang, Guohui Li, Yajing Zhou, Zhen Zhu
          </td>
          <td>2025-06-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Erythropoiesis, the process driving the differentiation of hematopoietic stem and progenitor cells to mature erythrocytes, unfolds through tightly orchestrated developmental stages, each defined by profound epigenetic remodeling. From the initial commitment of hematopoietic progenitors to the terminal enucleation of erythrocytes, dynamic changes in chromatin accessibility, transcription factor occupancy, and three-dimensional genome architecture govern lineage specification and stage-specific gene expression. Advances in our understanding of the regulatory genome have uncovered how non-coding elements, including enhancers, silencers, and insulators, shape the transcriptional landscape of erythroid cells. These elements work in concert with lineage-determining transcription factors to establish and maintain erythroid identity. Disruption of these epigenetic programs—whether by inherited mutations, somatic alterations, or environmental stress—can lead to a wide range of hematologic disorders. Importantly, this growing knowledge base has opened new therapeutic avenues, enabling the development of precision tools that target regulatory circuits to correct gene expression. These include epigenetic drugs, enhancer-targeted genome editing, and lineage-restricted gene therapies that leverage endogenous regulatory logic. As our understanding of erythroid epigenomics deepens, so too does our ability to design rational, cell-type-specific interventions for red blood cell disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d4488b6509a29acbbc7dd6eded382cd65c2fb32" target='_blank'>
              Epigenetic Regulation of Erythropoiesis: From Developmental Programs to Therapeutic Targets
              </a>
            </td>
          <td>
            Ninos-Ioannis Vasiloudis, K. Paschoudi, Christina Beta, G. Georgolopoulos, N. Psatha
          </td>
          <td>2025-06-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) is a common and aggressive malignancy. While significant advances have been made in the management of low-grade cancer, treatment of advanced HNSCC remains challenging. Here, we used a proteomic approach to find binding partners of the oncogene p63, the most frequently amplified transcription factor in HNSCC. We identified a zinc finger protein, ZNF148, which is coexpressed and physically binds p63 in carcinoma cells. Genome occupancy analyses identified a functional transcribed enhancer-derived RNA (eRNA) upstream of the CCND1 gene occupied by both factors. Mechanistically, p63 and ZNF148 control transcription of this eRNA, leading to overexpression of cyclin D1 to reinforce tumor cell proliferation. Importantly, this axis is specific for cancer cells and remains inactive in normal epithelial cells. The expression levels of these factors and the eRNA are positively correlated in cancer and associated with advanced stage and metastasis. Collectively, our data reveal a molecular pathway controlling HNSCC progression and identify potential selective targets for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5e810ce5ca8ccc51a55dbf7814b137367f41651" target='_blank'>
              p63 and ZNF148 cooperate to regulate head and neck squamous cell carcinoma.
              </a>
            </td>
          <td>
            R. Pecorari, Mara Mancini, Artem Smirnov, Giacomo Corleone, Flavia Novelli, A. Lena, Mariacristina Franzese Canonico, M. Montanaro, A. Cappello, A. Sacconi, Gabriella Misso, Michela Falco, Chiara Tammaro, F. Ciccosanti, Mauro Piacentini, M. Scimeca, M. Caraglia, G. Blandino, A. Mauriello, Maurizio Fanciulli, G. Melino, E. Candi
          </td>
          <td>2025-07-15</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="While it is straightforward to understand why most mutations affecting functional sequence are harmful, how genomic changes result in new beneficial traits is harder to understand. Domestication of transposable elements (TEs) is an important source of both new genes and new regulatory systems as, for their own propagation, TEs need to have transcription factor binding sites and functional products that predispose to their recruitment. But are such predispositions to gain-of-function sufficient? Here we consider the case of the endogenous retrovirus, HERVH. Knockdown data supports HERVH having roles in pluripotency, self-renewal and defence against transpositionally-active retroelements in the early human embryo. We clarify the pluripotent cell types associated with HERVH expression and, in the process, note a key unresolved issue, framed by the unwanted transcript hypothesis: how can some cell types have 2% of their transcripts being HERVH-derived but survive the multiplicity of cellular devices that suppress foreign transcripts, be this by transcriptional repression or post-transcriptional filtering? We note a common coupling between novelty generation and suppression evasion. For example, pluripotency-associated KLF4 binding is thought to compete with transcriptional suppressor binding. Similarly, HERVH has a strong splice site enabling efficient novel chimeric transcript formation, the resulting exon–intron junctions enabling evasion of the unwanted transcript filters that recognize low or absent intron presence. We conclude that to better understand domestication, a focus on predispositions to avoidance of unwanted transcripts filters, as well as predispositions to gain of functions, is necessary. The same insights will be valuable for transgene design (eg for gene therapy) and instructive of gain-of-function in tumours, as HERVH is known to be involved in onco-exaptation events. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00361-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1580e3fbff7b45d8d3d4ce0772f34b993e7ad4c0" target='_blank'>
              Co-option of an endogenous retrovirus (LTR7-HERVH) in early human embryogenesis: becoming useful and going unnoticed
              </a>
            </td>
          <td>
            Z. Izsvák, Jin Ma, Manvendra Singh, Laurence D. Hurst
          </td>
          <td>2025-07-05</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2885eb27b7a94cdea06da95207f47a0022e95a39" target='_blank'>
              NSD1 governs H3K36me2-mediated DNA methylation and drives differentiation of human iPSCs by regulating ‘HIDEN’ lncRNA expression
              </a>
            </td>
          <td>
            Anna Hansknecht, Martina Mankarious, Monica Varona Baranda, Esra Dursun Torlak, Wolfgang Wagner, Kira Zeevaert, Deepika Puri
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2c94a8a149ed8b89b155e50f8c13949352a8f13" target='_blank'>
              Molecular dynamics driving phenotypic divergence among KRAS mutants in pancreatic tumorigenesis
              </a>
            </td>
          <td>
            A. Grimont, David J. Falvo, Whitney J. Sisso, P. Zumbo, Christopher W. Chan, Francisco Santos, Grace Pan, Megan Cleveland, Tomer M. Yaron, Alexa Osterhoudt, Yinuo Meng, Maria Paz Zafra, William B. Fall, André F. Rendeiro, Erika Hissong, R. Yantiss, D. Betel, Mark A. Magnuson, Steven D. Leach, Anil K. Rustgi, Lukas E. Dow, R. Chandwani
          </td>
          <td>2025-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 A crucial bulwark of genome stability is the DNA repair system. The homologous recombination (HR) pathway is essential for the repair of the most harmful DNA lesions, like DNA double-strand breaks. Deficiencies in this pathway result in genomic instability and contribute to tumorigenesis. BRCA1 and BRCA2 (Breast Cancer type 1 and 2 susceptibility proteins) play critical roles in HR. Germline mutations in the BRCA1 and BRCA2 genes are responsible for nearly 50% of all familial breast and ovarian cancers. BRCA1/2-deficiencies coerce tumor cells to depend on alternative DNA repair pathways. Recent studies have demonstrated that inactivation of RAD52 protein induces lethality (synthetic lethality) in BRCA1/2-deficient cancer cells, without affecting normal cells. By exploiting this synthetic lethality relationship, targeting RAD52 with inhibitors may provide an exceptionally selective method for eliminating cancer cells. Focusing on DNA repair proteins involved in synthetically lethal relationships in cancer cells has become an important strategy in the development of specific cancer therapeutics. Our laboratory has developed small molecule inhibitors targeting RAD52. We have successfully created small molecule inhibitors of RAD52 (RAD52i) and have shown their ability to inhibit the biochemical activities RAD52 including DNA annealing and DNA strand exchange. Using the GFP-based recombination assays, we show that small molecule inhibitors of RAD52 disrupt the homologous DNA repair (HDR) and single strand annealing (SSA) repair pathways in human cells. Our compounds exhibited a notable reduction in cell viability in BRCA deficient cells. Furthermore, our small molecule RAD52i exhibited considerable efficacy in a mouse xenograft model cells markedly suppressing growth of human BRCA1-mutated cancer cells. We aim to develop small molecule RAD52i as novel cancer therapeutics.
 Citation Format: Shiva Ostadrahimi, Matthew J. Rossi, Sarah F. DiDomenico, Saiful Amin, Alexander V. Mazin. TARGETING BRCA-DEFICIENT CANCER CELLS WITH SMALL MOLECULE RAD52 INHIBITORS [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-12-25.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745e4e002b1607f75e1773159229e8749f23eb29" target='_blank'>
              Abstract P4-12-25: TARGETING BRCA-DEFICIENT CANCER CELLS WITH SMALL MOLECULE RAD52 INHIBITORS
              </a>
            </td>
          <td>
            Shiva Ostadrahimi, Matthew Rossi, Sarah DiDomenico, Saiful Amin, Alexander Mazin
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Objectives: Cancer research aims to understand the cellular and molecular mechanisms involved, in order to identify new therapeutic targets and provide patients with more effective therapies that generate fewer side undesirable and toxic effects. Previous studies have demonstrated the role of small nucleolar RNAs (snoRNAs) in many physiological and pathological cellular processes, including cancers. SnoRNAs are a group of non-coding RNAs involved in different post-transcriptional modifications of ribosomal RNAs. Recently, we identified a new snoRNA (jouvence), first in Drosophila, and thereafter, by homology, in humans. Methods: Here, we characterize the effect of the knockdown of jouvence by a sh-lentivirus on human primary patient-derived glioblastoma cells. Results: The sh-lentivirus anti-jouvence induces a significant decrease in cell proliferation and leads to cell death. EdU staining confirmed this decrease, while TUNEL also showed the presence of apoptotic cells. An RNA-Seq analysis revealed a decrease, in particular, in the level of BAALC, a gene known to potentiate the oncogenic ERK pathway and deregulating p21, leading to cell cycle blockage. Conclusions: Altogether, these results allow the hypothesis that the knockdown of jouvence could potentially be used as a new anti-cancer treatment (sno-Therapy), especially against glioblastoma and also, potentially, against acute myeloid leukemia (AML) due to the BAALC deregulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1877da20439c8df3eeaccfbcddc1a2c347beb2b9" target='_blank'>
              Knockdown of the snoRNA-Jouvence Blocks the Proliferation and Leads to the Death of Human Primary Glioblastoma Cells
              </a>
            </td>
          <td>
            Lola Jaque-Cabrera, Julia Buggiani, Jérôme Bignon, Patricia Daira, Nathalie Bernoud-Hubac, Jean-René Martin
          </td>
          <td>2025-07-18</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is the most prevalent type of leukemia in adults, characterized by a complex pathogenesis. The nuclear pore complex is pivotal in the development of various of malignant tumors, including hematological malignancies. While the NUP210 gene is upregulated in several solid tumors, its role in AML remains unreported. RNA-Sequence data and clinical profiles of AML patients were obtained from TCGA databases. Bone marrow samples were obtained from the First Affiliated Hospital of Ningbo University. GEPIA2, UALCAN and TCGA Portal databases were employed for survival prognosis analysis based on NUP210 expression. The GO/KEGG enrichment analysis of NUP210 co-expression and GSEA were performed using R software package. ENCORI and STRING databases were utilized to validate the association between NUP210 and MEIS1, PBX3 or HOXAs. CCK8, RT-qPCR, and Western blot assays were used to detect the viability and related gene expression in AML cell lines.Our findings revealed that NUP210 was significantly upregulated in AML patients compared to healthy controls. Multivariate analyses identified NUP210 as an independent poor prognostic factor in AML. Increased expression of NUP210 inhibited the viability of AML cells. The HOXA9, MEIS1, and PBX3 genes were notably upregulated in the NUP210 high-expression group. The RT-qPCR and Western blot assays demonstrated that NUP210 downregulation in leukemia cells leads to the inhibition of HOXA9 expression. Positive correlations were observed between NUP210 and MEIS1/PBX3, with MEIS1 and PBX3 proteins interacting with the HOXA9 protein. Our results demonstrated that NUP210 was remarkably upregulated in AML and may promote the malignant behavior of AML cells by upregulate HOXA9 gene expression, offering new insights for AML treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e60a036833093bfe76d9adec4575ddd5cae1b05c" target='_blank'>
              High NUP210 mRNA expression and its role in prognosis of acute myeloid leukemia
              </a>
            </td>
          <td>
            An Wu, Liyun Yang, Xiao Yan, Yanli Lai, Lieguang Chen, Shanhao Tang, Miao Zhou, Cong Shi, Ying Chen, Lixia Sheng, Qitian Mu, Guifang Ouyang
          </td>
          <td>2025-07-22</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Epigenetics and genome science have become central to current molecular biology research. Among the key mechanisms ensuring genomic integrity is the silencing of transposable elements in germline cells, a process essential for fertility in both sexes. A pivotal component of this silencing machinery involves PIWI-interacting RNAs (piRNAs), a distinct class of small non-coding RNAs that regulate gene expression and suppress transposable elements at both the transcriptional and post-transcriptional levels. piRNAs function in concert with PIWI proteins, whose expression is critical for proper oogenesis, spermatogenesis, and early zygote development. Disruptions in piRNA or PIWI protein pathways not only impair germline function but also contribute to genome instability, unchecked cell proliferation, and aberrant epigenetic modifications, hallmarks of tumorigenesis. Emerging evidence links the dysregulation of the piRNA/PIWI axis to the development and progression of various cancers, including lung and colorectal carcinomas. This review highlights the fundamental roles of piRNAs and PIWI proteins in reproductive biology and their increasingly recognized relevance in cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e264ddb82fab7df947d4ae7f4949ecb2a0652c26" target='_blank'>
              piRNA-Mediated Maintenance of Genome Stability in Gametogenesis and Cancer
              </a>
            </td>
          <td>
            Martyna Zawalska, Maciej Tarnowski
          </td>
          <td>2025-06-20</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10b2ceca0d46027db1d23222f329dce36899cf43" target='_blank'>
              Mammalian Proteome Profiling Reveals Readers and Antireaders of Strand-Symmetric and -Asymmetric 5-Hydroxymethylcytosine-Modifications in DNA
              </a>
            </td>
          <td>
            Lena Engelhard, Zeyneb Vildan Cakil, Simone Eppmann, Tye I Gonzalez, Rasmus Linser, Petra Janning, Daniel Summerer
          </td>
          <td>2025-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Nuclear atypia is associated with increased malignancy in numerous cancers, including glioblastoma (GBM). Here, we found that GBM stem cells display small nuclear size, prompting investigation of mechanisms underlying nuclear size regulation in the tumor hierarchy.


METHODS
We performed comparative gene expression and proteomics in GBM stem cells (GSCs) and neural stem cells (NSCs) to discover potential regulators of nuclear size. Through transcriptomic analysis, mass spectrometry, and pharmacologic inhibition, we interrogated the functional significance of nuclear size regulation.


RESULTS
GSCs were enriched for a nuclear sterol reductase, Lamin B Receptor (LBR). Targeting LBR increased nuclear size and decreased GSC viability and tumor initiation. Regulation of nuclear cholesterol synthesis underlaid LBR-dependency in GSCs. Loss of LBR or reduction of cholesterol levels induced double-strand DNA breaks (DSBs), activating P53-dependent DNA damage responses (DDR). The GSC proteomic LBR interactome revealed DDR mediators, including DEAD-box RNA helicase DDX5 that resolves R-loops at DSBs. Genetic targeting of LBR reduced the DDX5-R loop interaction, leading to increased R-loop formation rescued by cholesterol supplementation. Pharmacological sterol reductase inhibition mirrored genetic LBR targeting by reducing the DDX5-R loop interaction and increasing R-loops and DSBs. Targeting LBR genetically and pharmacologically inhibits GSC growth in vivo and synergizes with irradiation.


CONCLUSIONS
Stem-like GBM cells display reduced nuclear size, driven by nuclear cholesterol synthesis to regulate radiation responses, revealing a novel therapeutic paradigm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac67f62707eaa5f9643989ac2e6171dc64366dea" target='_blank'>
              Nuclear cholesterol regulates nuclear size and DNA damage responses in cancer stem cells.
              </a>
            </td>
          <td>
            Tingting Duan, S. Taori, S. Bhargava, Sisi Lai, Cuiqing Zhong, Shira Yomtoubian, Huairui Yuan, Xujia Wu, Po Zhang, Tengfei Huang, Donghai Wang, Fanen Yuan, Daqi Li, Huan Li, Hailong Mi, Weichi Wu, Rui Wang, Ahmed Habib, Farrukh Hammed, F. Vendetti, Pascal O. Zinn, Christopher J. Bakkenist, Marlies Meisel, Qiulian Wu, Jeremy N. Rich
          </td>
          <td>2025-06-07</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b5bfa4068fc673194fc62b48dc4a50c589763e7" target='_blank'>
              Ataxin-2 preserves oocyte genome integrity by promoting timely premeiotic DNA replication
              </a>
            </td>
          <td>
            V. L. Monteiro, Crystal Yu, Camilla Roselli, B. Bakthavachalu, Mani Ramaswami, T. Hurd
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background NSUN6 is a regulator of tRNA methylation that partially exists in the Golgi and centrosomes. It catalyzes the methylation of tRNACys and tRNAThr at the C72 site, which affects tRNA generation. However, its expression, prognosis, and immune invasion in pan-cancer have not been studied. This extensive research tapped into the TCGA database, gathering 33 carefully paired sets of cancer and nearby normal tissues, covering a broad spectrum of 33 different cancer types. Moreover, this study explored how NSUN6 expression relates to the presence of immune and stromal cells in the tumor’s intricate immune environment. To further understand NSUN6’s role, a detailed Gene Set Enrichment Analysis (GSEA) was conducted, revealing its enrichment patterns in various carcinoma subtypes and hinting at its involvement in tumor growth. Analysis of NSUN6 expression patterns revealed a significant increase in eight cancer types, including lung adenocarcinoma and pancreatic cancer (P < 0.001). Additionally, a reduction was observed in six cancer types, such as cholangiocarcinoma and thyroid cancer (P < 0.001). Survival analysis indicated that individuals with lung adenocarcinoma, pancreatic cancer, and other cancers expressing high levels of NSUN6 had improved prognosis. The association between NSUN6 expression and patient outcomes was evident, including overall survival, disease-specific survival, disease-free interval, and progression-free interval. Furthermore, individuals with high levels of NSUN6 expression showed significantly longer survival times than those with low levels (P < 0.05). The involvement of NSUN6 in immune infiltration was evident, GSEA showed a significant correlation between NSUN6 expression and the five most significant enrichment pathways in different tumors. The NSUN6 gene functions as an initial indicator for diagnosis in renal clear cell carcinoma, pancreatic cancer, lung adenocarcinoma, and low-grade brain glioma. There is an association between its elevated expression and the predictive outcome, as well as the immune system’s infiltration, in 33 varieties of cancer. This study is limited by its reliance on in silico analyses without experimental validation. Additionally, the use of publicly available datasets may introduce variability due to differences in data sources and platforms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba0727a8de06b6d8752b41beb8058ddf29b34aed" target='_blank'>
              NSUN6 expression correlates with prognosis and immune infiltration in human cancers based on pan-cancer analysis
              </a>
            </td>
          <td>
            Jie Gao, Yao Xu, Chan Tian, Qiuhong Zhang, Ming Zhang, H. Shan, Jie Shi, Zongjuan Ming, Shuanying Yang, Xia Yang
          </td>
          <td>2025-07-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background: Previously, we found that an uncharacterized protein CXorf38 is significantly downregulated in human ZIP8-knockout (KO) cells. Given that ZIP8 regulates essential micronutrients linked to diseases including cancer, this study aims to characterize CXorf38 and evaluate its role in lung adenocarcinoma. Methods: iTRAQ-based proteomics was previously used to identify the abundance of proteins in ZIP8-KO cells. Cell proliferation and colony formation assays were used to examine the function of CXorf38 by overexpressing the gene in lung adenocarcinoma cell lines. Kaplan–Meier survival analysis was used to assess the prognostic value of CXorf38, while TCGA clinical database analysis was used to evaluate its expression in lung cancer tissues, particularly in smokers. Bioinformatics analyses (GO, KEGG, PPI, and ICI) were performed on CXorf38-coexpressed genes derived from patients with lung cancer. Results: CXorf38 overexpression suppressed lung cancer cell proliferation and colony formation, suggesting a tumor-suppressive role. Higher CXorf38 expression correlated with improved survival in patients with lung adenocarcinoma, but not in lung squamous cell carcinoma. Clinical data showed CXorf38 downregulation with lung cancer tissues of smokers, indicating a potential role in smoking-induced cancer progression and treatment. Functional analysis using bioinformatics linked CXorf38 to immune response regulation, suggesting involvement in the tumor immune microenvironment. Conclusions: Our study reveals for the first time that CXorf38 is a potential tumor suppressor, prognostic biomarker, and/or tumor immune regulator in lung adenocarcinoma—further research is warranted to explore its role in tumor immunity and its therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0de769da2616c4ecb2cadcb3f426fcabdecfbd13" target='_blank'>
              Chromosome X Open Reading Frame 38 (CXorf38) Is a Tumor Suppressor and Potential Prognostic Biomarker in Lung Adenocarcinoma: The First Characterization
              </a>
            </td>
          <td>
            Rui Yan, H. Tan, Na-Li Cai, Le Yu, Yan Gao, Yan-Ming Xu, Andy T. Y. Lau
          </td>
          <td>2025-06-01</td>
          <td>Proteomes</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Background: Genetic instability and dysregulation of the cell cycle are hallmarks of cancer. In advanced HR+/HER2- breast cancer, the standard therapy involves cyclin-dependent kinase 4/6 inhibitors (CDK4/6i) which directly target aberrant growth by inhibiting the CDK4/6-RB1-E2F axis. In combination with endocrine therapy, CDK4/6i substantially extend progression-free survival (PFS), however, therapy resistance is frequently observed in the clinical setting. Inactivating mutations or loss of the breast cancer susceptibility gene 2 (BRCA2) have been associated with breast cancer for a long time and are observed in 30-40% of sporadic breast cancer cases. Additionally, patients with germline BRCA2 pathogenic variants (gBRCA2 PV) show decreased PFS on CDK4/6i combined with endocrine therapy. However, gBRCA2 PVs may not directly lead to CDK4/6i resistance. Instead, the proximity of the genes BRCA2 and retinoblastoma protein 1 (RB1) on chromosome 13 might result in simultaneous loss of both. Since RB1 is a direct downstream target of CDK4/6, its inactivation leads to significant resistance against CDK4/6 inhibition. Methods: In the proposed project, we used a lentiviral-based CRISPR/Cas9 system to model the loss of BRCA2 and RB1 alone and in combination in HR+/HER2- breast cancer cell lines and examined the impact on cell proliferation and treatment sensitivity. Results: The loss of RB1 resulted in increased proliferation, whereas the loss of BRCA2 did not affect growth rates. Interestingly, the combined loss of both genes led to significantly reduced proliferation. However, under therapeutic pressure, loss of RB1 was dominant and led to strong resistance towards the CDK4/6i Palbociclib. Nevertheless, Olaparib, a PARP inhibitor targeting DNA damage repair, remained effective in RB1- and especially in BRCA2-deficient cells where DNA damage repair mechanisms are already compromised. Discussion: Our results indicate that the loss of RB1 and BRCA2 significantly impacts cell proliferation, with the inactivation of both genes leading to a disadvantage in cell growth. We hypothesize that this is due to the impaired ability to repair DNA double-strand breaks. BRCA2 is crucial for homologous recombination repair, while RB1 is involved in the non-homologous end joining pathway. We believe that the inactivation of both genes increases reliance on other error-prone repair pathways, such as single-strand annealing or microhomology-mediated end joining (MMEJ, also known as alternative end joining). This could also explain the heightened response to Olaparib, which, among other PARP proteins also targets PARP1, a component of the MMEJ repair pathway. While simultaneous inactivation of both genes leads to growth disadvantage, potentially because of the diminished capacity to repair DNA-double strand breaks, our data suggests that they still retain a strong advantage under treatment pressure and that clones harbouring both mutations evolve during the time of CDK4/6i treatment.
 Citation Format: Daniela Haas, Selina Wolf, Urte Stankute, Magdalena Kröll, Philipp Jost, Nadia Dandachi, Marija Balic, Michael Dengler. Simultaneous loss of BRCA2 and RB1 results in growth disadvantage but strong CDK4/6i resistance in HR+/HER2- breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/925deda18ec871dd939e7887015a8d1c73a05d8c" target='_blank'>
              Abstract P3-02-19: Simultaneous loss of BRCA2 and RB1 results in growth disadvantage but strong CDK4/6i resistance in HR+/HER2- breast cancer
              </a>
            </td>
          <td>
            Daniela Haas, Selina Wolf, Urte Stankute, Magdalena Kröll, Philipp J Jost, N. Dandachi, Marija Balic, Michael Dengler
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The chromosomal passenger complex (CPC), comprising Survivin, Aurora B kinase, INCENP, and Borealin, is classically known for its essential functions during mitosis. However, recent findings expand the CPC's role beyond cell division, uncovering novel functions in replication stress response and genome stability maintenance. In our recent study by Falke et al. [1], we demonstrate that CPC components, particularly Survivin and Aurora B, contribute to the protection of stalled replication forks and facilitate translesion synthesis (TLS) - a specialized DNA damage tolerance mechanism. This regulation involves a functional interaction with proliferating cell nuclear antigen (PCNA), a central coordinator of DNA replication and repair. The work not only reshapes our understanding of CPC biology but also reveals a mechanistic link how tumor cells may exploit CPC activity to maintain replication under genotoxic stress. These findings open new avenues for targeting CPC-PCNA interactions in cancers characterized by elevated replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27440a8fa21c0d8a560b047855971e8e0f1ea441" target='_blank'>
              From mitosis to mutagenesis: Chromosomal passenger proteins at the crossroads of replication stress and cancer resilience
              </a>
            </td>
          <td>
            S. Knauer, Roland H. Stauber
          </td>
          <td>2025-07-07</td>
          <td>Cell Signaling</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most malignant subtype of breast cancer that lacks reliable targets for diagnosis and therapy. Non-coding RNA (ncRNA)-encoded products hold promise for addressing this unmet need. By analyzing the reported ribosomal RNA sequencing data, combined with the TCGA, ORFfinder, SmProt databases, we identified CDKN2B-AS1, a TNBC-upregulated lncRNA encoding a 66-amino-acid peptide via CUG-initiated translation. CRISPR-Cas9 gene editing and mass spectrometry confirmed endogenous expression of this peptide, designated 66CTG, in TNBC cells. Functionally independently of its host RNA, 66CTG promoted the proliferation of TNBC cells and the tumor growth of TNBC xenograft by stabilizing c-Myc protein and enhancing Cyclin D1 transcription. Immunohistochemistry of 89 clinical TNBC paraffin samples revealed positive correlations among 66CTG, c-Myc, and Cyclin D1 expression levels. Mechanistically, co-immunoprecipitation and ubiquitination assays revealed that 66CTG stabilized c-Myc by competitively interacting with FBW7α, an E3 ligase responsible for recognizing 66CTG CPDS56/S60 motif which phosphorylated by GSK-3β during the late G1 phase. In conclusion, our findings suggest 66CTG has potential to be developed as a target for TNBC diagnosis and therapy. Furthermore, it unveils a regulatory axis wherein 66CTG stabilizes c-Myc by interacting with FBW7α, offering a new mechanistic explanation for c-Myc overexpression in TNBC. Patients co-overexpressing 66CTG, c-Myc, and Cyclin D1 may benefit from therapies targeting this axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05673162f1ad18da2bfdf56d75820b6435486900" target='_blank'>
              A novel peptide 66CTG stabilizes Myc proto-oncogene protein to promote triple-negative breast cancer growth
              </a>
            </td>
          <td>
            Huichun Liang, Fubing Li, Huan Fang, Wenlong Ren, Zhongmei Zhou, Jiecheng Wang, Jialing Liu, Yongjia Tang, Xue Liu, Yingying Wu, Jing Peng, Chuanyu Yang, Jiayi Chen, Yuting Fei, Yujie Shi, Dewei Jiang, Nu Zhang, Ceshi Chen
          </td>
          <td>2025-07-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827b6d7c12ecbffd0d9766944bd9dcb122ff76bc" target='_blank'>
              Epithelial state-transitions permit inflammation-induced tumorigenesis
              </a>
            </td>
          <td>
            E. Jarman, Anabel Martinez Lyons, Yuelin Yao, Aleksandra Rozyczko, S. Waddell, Andreea Gradinaru, Paula Olaizola, Kyle Davies, Rachel V. Guest, Stephanie Röessler, Timothy J. Kendall, Owen J. Sansom, A. Khamseh, Luke Boulter
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background The study explores the role of non-coding ribonucleic acids (ncRNAs) and higher enhancer of zeste homolog 2 (EZH2) gene in breast cancer progression, examining how microRNA (miRNA) and long noncoding RNA (lncRNA) control EZH2, potentially influencing oncogene growth and treatment failure. Materials and methods The databases used in the study included Cyclebase 3.0 and CellTracer to determine EZH2’s role in cell cycle, Oncomine, OncoMX and The University of Alabama At Birmingham Cancer Data Analysis Portal (UALCAN) for Pan-cancer analysis, The Cancer Genomic Atlas Portal (TCGA Portal), Gene Expression Profiling Interactive Analysis (GEPIA2), OncoDB, CR2Cancer, Encyclopaedia of RNA Interactomes (ENCORI), and The Cancer Genome Altas Analyzer (TCGAnalyzeR v1.0) for differential expression analysis, CR2Cancer, OncoDB, MethMarkerDB, and Wanderer databases for epigenetic alteration analysis, Kaplan-Meier Plotter for survival analysis, Breast Cancer Gene Expression Miner (bc-GenExMiner v5.0) for hormone receptor analysis, Tumor-Immune System Interaction Database (TISIDB), Cancer Single Cell State Atlas (CancerSEA), TNMplot, DriverDBv4, and ENCORI for biological processes, cell cycle checkpoints and metastasis analysis, Enrichr, Tumor Immune Estimation Resource (TIMER 2.0), Gepia2 for transcription factor analysis, miRNet, Transcriptome Alterations in Cancer Omnibus (TACCO), and CancerMIRome for miRNA analysis, Enrichr, UALCAN, and ENCORI for lncRNA analysis. Results The EZH2 gene is overexpressed in breast cancer (BRCA) tumors, metastatic tissues, and circulating tumor cells, potentially leading to cancer progression. Patients with high EZH2 expression have shorter overall survival (OS), distant metastasis-free survival (DMFS), and relapse-free survival (RFS) compared to those with low expression. Estrogen receptot (ER)-negative BRCA tumors and PR-negative tumors have high EZH2 gene and eukaryotic transcription factor (E2F2) levels. The EZH2/E2F2 axis may assist ER/PR-negative BRCA by sponging homosapiens microRNA family (hsa-let-7b-5p) through lncRNA-thymopoietin antisense transcript 1 (TMPO-AS1). Overexpression of the EZH2 protein is associated with BRCA metastasis. Conclusion EZH2 overexpression in basal-like BRCA is mediated by a competing endogenous RNA (ceRNA) network and regulating their expression levels may facilitate better survival outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7050692d6924a95d1983e91c0ebe8eef26bc9ef7" target='_blank'>
              TMPO-AS1-hsa-let-7b-5p-EZH2-RNA network predicts poor survival in basal-like breast cancer patients
              </a>
            </td>
          <td>
            Prerna Vats, Bhavika Baweja, Sakshi Nirmal, Aditi Singh, Rajeev Nema
          </td>
          <td>2025-06-07</td>
          <td>Reports of Practical Oncology and Radiotherapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Personalized immunotherapy of pancreatic ductal adenocarcinoma (PDAC) through T-cell mediated targeting of tumor-specific, mutanome-encoded neoepitopes may offer new opportunities to combat this disease, in particular by countering recurrence after primary tumor resection. However, the sensitive and accurate calling of somatic mutations in PDAC tissue samples is compromised by the low tumor cell content. Moreover, the repertoire of immunogenic neoepitopes in PDAC is limited due to the low mutational load of the majority of these tumors. Methods We developed a workflow involving the combined analysis of next-generation DNA and RNA sequencing data from matched pairs of primary tumor samples and patient-derived xenograft models towards the enhanced detection of driver mutations as well as single nucleotide variants encoding potentially immunogenic T-cell neoepitopes. Subsequently, we immunized HLA/human T-cell receptor (TCR) locus-transgenic mice with synthetic peptides representing candidate neoepitopes, and molecularly cloned the genes encoding TCRs targeting these epitopes. Result Application of our pipeline resulted in the identification of greater numbers of non-synonymous mutations encoding candidate neoepitopes with increased confidence. Furthermore, we provide proof of concept for the successful isolation of HLA-restricted TCRs from humanized mice immunized with different neoepitopes, several of which would not have been selected based on mutanome analysis of PDAC tissue samples alone. These TCRs mediate specific T-cell reactivity against the tumor cells in which the corresponding mutations were identified. Conclusion Enhanced mutanome analysis and candidate neoepitope selection increase the likelihood of identifying therapeutically relevant neoepitopes, and thereby support the optimization of personalized immunotherapy for PDAC and other poorly immunogenic cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a74726107b1853e699ed8135ce18a77b1befecac" target='_blank'>
              Enhanced mutanome analysis towards the induction of neoepitope-reactive T-cell responses for personalized immunotherapy of pancreatic cancer
              </a>
            </td>
          <td>
            M. Volkmar, Dana Hoser, Claudia Lauenstein, Janina Rebmann, Agnes Hotz-Wagenblatt, Jan Rieger, I. Poschke, Jonas P. Becker, Angelika B. Riemer, M. Sprick, A. Trumpp, Oliver Strobel, Thomas Blankenstein, G. Willimsky, R. Offringa
          </td>
          <td>2025-06-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract The six-subunit origin recognition complex (ORC) loads excess MCM2–7 on chromosomes to promote initiation of DNA replication and is believed to be important for origin specification. Mapping of origins in cancer cell lines engineered to delete three of the subunits, ORC1, ORC2, or ORC5, shows that specific origins are still used and are mostly at the same sites in the genome as in wild-type cells. The few thousand origins that were upregulated in the absence of ORC suggest that GC/TA skewness and simple repeat sequences facilitate, but are not essential for, origin selection in the absence of the six-subunit ORC. Despite the lack of ORC, excess MCM2–7 is still loaded at comparable rates in G1 phase to license dormant origins and is also repeatedly loaded in the same S phase to permit re-replication. Thus, origin specification and excess MCM2–7 loading on origins do not require the six-subunit ORC in human cancer cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b9d06157a04f12f90653a1aa8763b312684dcdf" target='_blank'>
              Specific origin selection and excess functional MCM2–7 loading in ORC-deficient cells
              </a>
            </td>
          <td>
            Y. Shibata, Mihaela Peycheva, Etsuko Shibata, Daniel Malzl, Rushad Pavri, Anindya Dutta
          </td>
          <td>2025-06-06</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Aggressive and therapy-resistant glioblastoma is among the most lethal malignant tumors in humans. Complete surgical resection is often unachievable; therefore, combination chemoradiotherapy is used to target tumor cells residual beyond the resection margin. This approach induces DNA damage in tumor cells and activates the apoptosis pathway. Unfortunately, recurrence remains a major clinical challenge, frequently manifesting as more aggressive and treatmentresistant glioblastoma phenotypes. The DNA repair and damage response (DDR) pathways are critical for maintaining genome stability. While defects in these mechanisms contribute to oncogenesis, they also make tumor cells vulnerable to DNA-damaging therapy, as the cells become dependent on residual repair capacity. It is of paramount importance to understand the molecular components of these mechanisms and to identify potential therapeutic/pharmacological targets for improving outcomes in glioblastoma patients. A subpopulation of stem-like cells, designated as glioblastoma cancer stem cells (CSCs), has been identified as a critical factor in the initiation, maintenance, and recurrence of tumors. These cells exhibit therapy resistance due to enhanced DNA repair capacity. In addition, emerging evidence suggests a link between carbohydrate metabolism and DNA repair pathways, thereby revealing novel therapeutic vulnerabilities in glioblastoma. This review examines current strategies targeting DNA repair mechanisms in glioblastoma. We present a synopsis of recent advancements in research concerning the mechanisms and factors involved in the elimination of DNA damage induced by ionizing radiation and temozolomide (TMZ). Furthermore, we explore the potential of DNA repair pathway inhibitors under investigation in preclinical and clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8edb7bc241371338e935ca9559370edd806d8316" target='_blank'>
              DNA Damage and Repair in Glioblastoma: Emerging Therapeutic Perspectives
              </a>
            </td>
          <td>
            I. Gareev, O. Beylerli, S. Roumiantsev
          </td>
          <td>2025-07-01</td>
          <td>Creative surgery and oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Background: The study of circulating tumor DNA (ctDNA) and cell-free chromatin (cf-Chr) has become instrumental in advancing our understanding of cancer biology. This research delves into the antitumor mechanism of abemaciclib in ER-positive, HER2-negative metastatic breast cancer patients by examining cf-Chr fragmentation patterns. Methods: In this investigation, 72 ER-positive, HER2-negative metastatic breast cancer patients, as defined by ASCO/CAP guidelines and treated with abemaciclib at Chiba University Hospital in Japan from 2020 to 2022, were enrolled. Pre- and post-treatment serum samples were analyzed, with nucleic acids extracted for whole-genome sequencing (WGS) and differential enrichment analysis. Given the observed correspondence of cf-Chr fragment patterns with open chromatin regions (OCRs), indicative of gene activation zones, differential enrichment analysis was specifically focused on cell type-specific regulatory regions—namely enhancers and promoters—in luminal breast cancer cells and T cells, to elucidate the transcriptionally active regions within the genome. Results: Our investigations have revealed a significant decrease in cf-Chr concentrations, showing a 50% reduction following treatment with abemaciclib. The analysis of cf-Chr in breast cancer patients, both pre- and post-abemaciclib treatment, unveiled differential chromatin enrichment within regions tied to antitumor pathways that are modulated by the drug. Notably, the cf-Chr analysis preceding treatment disclosed disturbances in the Notch signaling pathway, thereby revealing an aspect of abemaciclib's antitumor mechanism that has not been previously established in human clinical specimens. Conclusions: Our study suggests for the first time alterations in cf-Chr in human patient specimens that may reflect the antitumor activity of abemaciclib that had previously been confirmed only in vitro. Our study presents a pivotal step in understanding the impact of abemaciclib on the tumor chromatin landscape, providing a direct observation of the drug's mechanism of action in patients. These insights pave the way for future research into the mechanisms of existing therapies and their optimization for treating metastatic breast cancer.
 Citation Format: Mamoru Takada, Sakuntha Gunarathna, Hideyuki Yamada, Muhan Yu, Takeshi Nagashima, Hiroshi Fujimoto, Junta Sakakibara, Hiroto Yamamoto, Regina Nguyen, Tsutomu Kawaguchi, Otsuka Masayuki, Motoki Takaku. Unveiling the Antitumor Mechanism of abemaciclib in Human Breast Cancer Through Circulating Tumor Chromatin Analysis [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-02-23.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/006d74044a867b3a3ad5d2dcfe26ae4ccbc619c6" target='_blank'>
              Abstract P5-02-23: Unveiling the Antitumor Mechanism of abemaciclib in Human Breast Cancer Through Circulating Tumor Chromatin Analysis
              </a>
            </td>
          <td>
            Mamoru Takada, Sakuntha Gunarathna, Hideyuki Yamada, Muhan Yu, Takeshi Nagashima, H. Fujimoto, Junta Sakakibara, Hiroto Yamamoto, Regina Nguyen, Tsutomu Kawaguchi, Otsuka Masayuki, Motoki Takaku
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Congenital hypogonadotropic hypogonadism (CHH) is a rare endocrine disorder characterized by gonadal dysfunction attributed to impaired gonadotropin secretion. CHH is associated with approximately 60 genes including FGFR1. Nevertheless, the nucleotide variants of these genes are only related to less than half of the cases. Herein, we report a case of CHH caused by a novel mechanism. A 6-year-old boy presented with hypomasculinized genitalia, hyposmia, and syndactyly. Endocrine examinations showed impaired gonadotropin secretion. Short-read next-generation sequencing (NGS) identified the absence of mutations in the major causative genes for CHH. However, it detected an accumulation of discordant and split reads in a genomic region within the FGFR1 gene. Array-based comparative genomic hybridization did not detect copy-number abnormalities. Targeted long-read NGS and Sanger sequencing identified a de novo 333-bp insertion in exon 9 of the FGFR1 gene. A similarity search revealed that the insertion was an Alu element. This insertion caused a frameshift and resulted in premature termination (p. His409fsTer31). Further, it had several hallmarks of retrotransposition such as target site duplication, an endonuclease cleavage site-like motif, and a poly-A tail. The study results broadened the genetic basis of CHH that considered retrotransposon insertions. Importantly, this case emphasizes the need for additional genomic analyses in patients with CHH who had negative results on short-read NGS and array-based comparative genomic hybridization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/637624b99a101c0fcbfaf275613d36c05df12aa2" target='_blank'>
              De novo retrotransposon insertion into the FGFR1 gene in a boy with congenital hypogonadotropic hypogonadism: a case report
              </a>
            </td>
          <td>
            Kentaro Sawano, Keisuke Nagasaki, E. Suzuki, Yasuko Ogiwara, Ikuko Kageyama, Maki Fukami, Y. Kuroki
          </td>
          <td>2025-06-18</td>
          <td>Frontiers in Endocrinology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Salivary gland tumors are diagnostically challenging due to major diversity of benign and malignant tumors with enormous intra-tumorous and inter-tumorous heterogeneity and, hence, frequently overlapping histologic features. DNA methylation has greatly enhanced tumor classification in several organs and led to the identification of previously unrecognized entities. In a recent study on DNA methylation of salivary gland tumors, we had identified a group of unclassifiable tumors. In this study, we characterize this group through an integrated analysis of clinical, histomorphologic, immunohistochemical, and molecular features. This group of 12 tumors is characterized by small, clinically benign appearing tumors with striking female predominance (91.7%), the latter not paralleled in other salivary tumor types. In addition to distinct DNA methylation profiling, copy number analysis revealed unique alterations with highly recurrent chromosome 5p/5q loss and frequent amplification of the MDM2 locus on chromosome 12q. Whole-exome and transcriptome sequencing detected no recurrent mutations or fusions. The histomorphologic features were only moderately distinct, comprising an obligate, thereby variable admixture of biphasic-tubular and monophasic-myoepithelial areas, low or absent nuclear atypia, and minimal proliferation. Frequent invasive behavior, a solitary lymph node metastasis, and the molecular alterations, altogether, strongly support classification as a, presumably low-grade, carcinoma. Altogether, these findings clearly distinguish this tumor group from histomorphologically similar tumor entities, in particular myoepithelial carcinoma, epithelial-myoepithelial carcinoma, and adenoid cystic carcinoma. We present thorough arguments that this tumor group represents a distinct salivary carcinoma entity rather than a variant of an existing one. We propose the provisional designation "Biphasic myoepithelial carcinoma with 5p/5q loss" for discussion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82dbe56be70a5e0ed77a5d5f76729e43d07ee1f6" target='_blank'>
              Biphasic Myoepithelial Carcinoma With 5p/5q Loss: Morphomolecular Characterization and Provisional Designation of a Proposed Novel Salivary Tumor Entity.
              </a>
            </td>
          <td>
            Philipp Jurmeister, Maximilian Leitheiser, Linda Bergmayr, Emma Payá Capilla, Liliana H. Mochmann, Yauheniya Zhdanovich, Fabian Engelhardt-Schott, Konstanze Schleich, Doreen Klingler, Edgar Chimal, Cornelia M Focke, G. Breimer, Ilse van Engen van Grunsven, Andreas von Deimling, D. Capper, F. Klauschen, Stephan Ihrler
          </td>
          <td>2025-07-01</td>
          <td>The American journal of surgical pathology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="Ribosomal DNA transcription is essential for ribosome biogenesis and the production of proteins. Using a combination of droplet digital PCR and deep bisulfite sequencing, we have quantified both the absolute number as well as the methylation level of individual rDNA transcription units (TU) in blood samples of 139 young healthy individuals and 141 sex- and age-matched individuals with unsolved syndromal developmental delay (DD), ranging from 0.02 to 18.4 years in age. There were no between-group differences in average promoter methylation, absolute copy number (CN), extreme CN, and hypomethylated (0-10%) presumably active CN. This largely excludes rDNA CN as a modulating factor in DD. The absolute CN in all 280 samples was 423.7 ± 108.4 (median 410, range 153 to 1,000) and the active CN was 175.0 ± 36.4 (median 174, range 70 to 376). Similar to adults, the absolute CN did not change from birth to sexual maturity but was strongly (Pearson ρ = 0.64, P < 0.001) correlated with promoter methylation. In contrast to adults, there was no significant correlation between age and promoter methylation and no age-related loss of active copies from birth to < 20 years. The number of completely unmethylated copies even significantly (Pearson ρ = 0.15; P = 0.01) increased during childhood and adolescence. Our results suggest that rDNA promoter methylation and the age-related loss of active rDNA TU, which are a hallmark of the aging process, start only after reaching sexual maturity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/274c60d47b8d0482945a4f0864a98745972db01c" target='_blank'>
              rDNA copy number variation and methylation from birth to sexual maturity
              </a>
            </td>
          <td>
            Alina Michler, Sarah Kießling, Jana Durackova, R. Potabattula, Asuman Koparir, Thomas Haaf
          </td>
          <td>2025-06-16</td>
          <td>Aging (Albany NY)</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) accounts for twelve percent of all breast cancer cases, with a survival rate around ten percent lower than ER+/PR+ positive breast cancers. There are limited therapeutic options as these tumors do not respond to hormonal therapy or HER2-targeted treatments. We hypothesized that new insights into pathogenic mechanisms in TNBC can be obtained from studying epigenetic alterations through Hi-C (genome-wide chromosome conformation capture) data analysis.We developed a computational strategy that captured key properties of chromatin conformation while incorporating statistical measures of interaction significance. This model addresses limitations in Hi-C data analysis without relying on predefined features like TADs and compartments. We applied this model to Hi-C and RNA-seq data from TNBC patients, representing the data as multilayer networks to identify genome-wide properties of the TNBC 3D genome.Our network-based analysis revealed distinct chromatin interaction patterns in TNBC compared to healthy contralateral controls. Hi-C data can distinguish interaction patterns related to diseased phenotypes or interaction patterns with potential to exert regulatory effects instead of incidental contacts, but some apparently random interactions may also support important genome regulatory activities.Our findings demonstrate that network-based Hi-C analysis can capture the genome-wide complexity of chromatin interactions in TNBC. This integrative approach provides new insights into the epigenetic mechanisms underlying TNBC pathogenesis and contributes to the advancement of analysis methods for future investigations into novel therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be2cde03b116eaa9a9ec374af95ff1672e86aa8b" target='_blank'>
              Integration of chromosome conformation and gene expression networks reveals regulatory mechanisms in triple negative breast cancer
              </a>
            </td>
          <td>
            Helena Reyes-Gopar, Keila Adonai Pérez-Fuentes, M. Bendall, Enrique Hernández-Lemus
          </td>
          <td>2025-07-04</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 The single-stranded DNA cytosine deaminase enzyme, APOBEC3B (A3B), has been implicated as a mutational driver in multiple human cancers1,2. Breast cancer in particular shows high levels of A3B expression and positive associations with poor clinical outcomes3. Recent studies have also demonstrated that human A3B expression in mice is carcinogenic with significantly shortened life expectancies4. The only genetic factor in breast cancer thus far associated with A3B expression and APOBEC signature single base substitutions is p53 mutation5. Therefore, here we test the hypothesis in vivo that human A3B expression will show pathlogical synergy in the form of accelerated mammary tumor penetrance upon combination with p53 haploinsufficiency due to a heterozygous deletion mutation spanning exons 2-10. Custom and commercially available animals were subjected to standard breeding practices to generate experimental (A3B, p53Delta exons 2-10/+) and control groups (p53Deltanexons 2-10/+). Surprisingly, in contrast to A3B-accelerated tumor development on a wildtype genetic background, A3B had no effect on the rates of tumor development in p53 heterozygous animals (median 13.5 months). The observed genetic epistasis was not due to a lack of A3B expression or activity, as evidenced by strong IHC positivity and a clear acculumlation of APOBEC signature single base substitution mutation in tumors. We conclude that A3B-accelerated tumor development requires full p53 function, and that a haploinsufficiency in p53 enables tumor cells (or their precursors) to better tolerate DNA damage lesions induced by A3B.
 Selected References:
 1. Petljak, M. et al. Addressing the benefits of inhibiting APOBEC3-dependent mutagenesis in cancer. Nat Genet 54, 1599-1608 (2022).
 2. Butler, K. & Banday, A. R. APOBEC3-mediated mutagenesis in cancer: causes, clinical significance and therapeutic potential. J Hematol Oncol 16, 31 (2023).
 3. Roelofs, P. A. et al. Clinical implications of APOBEC3-mediated mutagenesis in breast cancer. Clin Cancer Res 29, 1658-1669 (2023).
 4. Durfee, C. et al. Human APOBEC3B promotes tumor development in vivo including signature mutations and metastases. Cell Rep Med 4, 101211 (2023).
 5. Burns, M. B. et al. APOBEC3B is an enzymatic source of mutation in breast cancer. Nature 494, 366–370 (2013).\
 Citation Format: Joshua Proehl, Cameron Durfee, Yuan Zhao, Nuri Alpay Temiz, Reuben Harris. p53 Haploinsufficiency Epistatically Masks A3B Tumor Phenotype in vivo [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-06-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8de4891dd8d8ec2c399682371385c3645385994" target='_blank'>
              Abstract P3-06-26: p53 Haploinsufficiency Epistatically Masks A3B Tumor Phenotype in vivo
              </a>
            </td>
          <td>
            Joshua Proehl, Cameron Durfee, Yuan Zhao, N. A. Temiz, R. Harris
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="
 3046


 Background:
 Prior work has shown that detection of circulating tumor DNA (ctDNA) at time of diagnosis of leiomyosarcoma (LMS) is associated with lower likelihood of objective response and patients with detectable ctDNA after two cycles of chemotherapy have worse survival. However, because ctDNA exists at much lower concentrations in plasma compared to cell-free DNA of non-tumor origin, the sensitivity of these prognostic measures to tumor signals remains unclear. With increasing evidence of ctDNA fragments being shorter than the background non-tumor cell-free DNA, we sought to test whether restricting our analysis to smaller fragment sizes would improve detection of ctDNA in LMS.
 Methods:
 Plasma was serially collected from patients with LMS undergoing chemotherapy. Cell-free DNA extracted from these samples was profiled by ultra-low-passage whole-genome sequencing (ULPWGS). Copy number alterations were identified and used to detect ctDNA using the ichorCNA algorithm before and after restricting the dataset to fragments of 90-150bp (short). We compared detectability of ctDNA via ichorCNA between analyses using all sequencing data and those using data restricted to short fragments.
 Results:
 From 28 patients, 126 plasma samples were profiled. The median fragment length of cell-free DNA was 240bp (IQR 142-349) for patients with LMS, compared to 307bp (IQR 171-465, p < 0.001) in samples collected from healthy controls. Short fragments made up 19.48% of LMS libraries at diagnosis when ctDNA levels were highest, 15.27% of all LMS samples, and 12.96% of libraries from healthy controls. While ctDNA was detectable by ULPWGS in 17% of all samples, detection increased to 40% when analyzed using only short cell-free DNA fragments (p < 0.0001). The proportion of diagnostic samples with detectable ctDNA was nominally higher when analyzed by short fragments (39% vs. 64% with 90-150bp size restriction, p = 0.1078, n = 28) and was significantly higher in samples collected after two cycles of chemotherapy (5% vs. 40% with 90-150bp size restriction, p = 0.0197, n = 20). Increases in ctDNA detectability with fragment size restriction were also observed in each of localized and metastatic subgroups (metastatic: 16% without restriction, 38% with restriction, n = 97, p = 0.0012; localized: 17% vs. 45%, n = 39, p = 0.04).
 Conclusions:
 Our results demonstrate that detection of ctDNA is improved by analyzing short fragments of cell-free DNA in samples collected from patients with LMS. These findings represent a potential to increase the sensitivity of an affordable, low-coverage liquid biopsy assay and may enhance the prognostic value of ctDNA detection in these patients. Further study of the association between ctDNA detection and outcome is needed to fully validate the impact of fragment size restricted analysis of cell-free DNA samples in patients with LMS and is currently ongoing in a prospective study.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8ceaaf3e27c10098ba0b8ae1de9b437c08db2f3" target='_blank'>
              Improved detection of circulating tumor DNA in patients with leiomyosarcoma with fragment size restriction.
              </a>
            </td>
          <td>
            N. Ruzgar, K. Klega, Cora A. Ricker, Mohammad Tanhaemami, Amanda M. Shafer, Denise K. Reinke, Brittany L. Siontis, S. Schuetze, B. Crompton
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31620c8deab30d15f98e2b8cf7eeeade98239959" target='_blank'>
              LINC01410 in cancer: a comprehensive review of its oncogenic role, regulatory mechanisms, and clinical implications.
              </a>
            </td>
          <td>
            Saman Morovat, Pejman Morovat, Ashkan Kalantary-Charvadeh, Marzieh Mojbafan, S. Teimourian
          </td>
          <td>2025-07-12</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Pancreatic cancer is a highly malignant tumor of the digestive system with a dismal prognosis. Despite advances in diagnosis and treatment, overall survival remains extremely low. Early diagnostic markers and an improved understanding of tumor-microenvironment interactions are essential for developing more effective therapies.We analyzed 74 single-cell RNA sequencing (scRNA-seq) samples, performing unsupervised clustering and marker-gene expression profiling to define major cell types. Large-scale chromosomal copy-number variation (CNV) analysis distinguished malignant from non-malignant ductal cells. Non-negative matrix factorization (NMF) identified stage-associated gene modules, which were integrated with TCGA bulk-RNA data and machine-learning feature selection to pinpoint candidate prognostic genes. Two independent cohorts were used for validation. Regulatory network inference (pySCENIC) and ligand–receptor interaction analysis (CellPhoneDB) explored cross-talk between malignant cells and macrophages. Finally, in vitro knockdown of CTSV assessed its functional role in pancreatic cancer (PAC) cell proliferation and migration.Three prognosis-related genes—ANLN, NT5E, and CTSV—were selected based on their strong association with clinical stage and validated in external datasets. High expression of these genes correlated with poorer overall survival and an increased infiltration of M0 macrophages. CellPhoneDB predicted significant interactions between high-expression malignant ductal cells and M0 macrophages via CXCL14–CXCR4 and IL1RAP–PTPRF axes, with SPI1 identified as an upstream regulator of IL1RAP. In vitro CTSV knockdown significantly inhibited PAC cell proliferation and migration.Our integrative single-cell and bulk-RNA workflow identifies ANLN, NT5E, and CTSV as novel prognostic biomarkers in pancreatic cancer and highlights a pro-tumorigenic interaction between malignant ductal cells and macrophages. Targeting CTSV or disrupting CXCL14–CXCR4 and IL1RAP–PTPRF signaling may offer new therapeutic avenues for PAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50ce5676a35dfc57b1b078e8894ee25f419175d" target='_blank'>
              Integration of single-cell RNA and bulk RNA sequencing revealed malignant ductal cell heterogeneity and prognosis signatures in pancreatic cancer
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-07-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 e15067


 Background:
 Our group previously identified the oncogenic fusion
 CRTC1::MAML2
 , arising from t(11;19) translocations in mucoepidermoid carcinoma, the most common type of salivary gland malignancy. Subsequent studies have reported additional fusions of
 MAML2
 with
 CRTC3
 ,
 YAP1
 , and
 NR1D1
 . These fusions retain the C-terminal
 MAML2
 transactivating domain (TAD), which is believed to drive aberrant gene transcription. While the oncogenic roles of these
 MAML2
 fusions have been well-established in preclinical and clinical studies, we aimed to identify novel
 MAML2
 fusions across a broader range of human malignancies.
 Methods:
 DNA and RNA sequencing were performed on tumor samples submitted to Caris Life Sciences.
 MAML2
 fusions were identified from RNA transcripts and filtered to include only known pathogenic fusions or in-frame fusions containing a C-terminal
 MAML2
 TAD that were observed in ≥3 samples. Fusion burden was defined as the number of unique fusion isoforms per sample. Statistical significance was evaluated using Mann-Whitney, Fisher’s Exact, or Chi-squared tests.
 Results:
 Among 180,124 tumor samples analyzed, 509 specimens were found to harbor
 MAML2
 fusion transcripts. After filtering for non-recurrent fusions and those lacking a
 MAML2
 TAD, 143 unique fusions remained. Over half of these specimens harbored known fusions involving
 CRTC1
 (42/143),
 YAP1
 (37/143),
 CRTC3
 (7/143), and
 NR1D1
 (2/143). In addition, novel fusions with
 MTMR2
 (31/143),
 SESN3
 (11/143),
 CCDC82
 (6/143),
 FAM76B
 (4/143), and
 ATXN3
 (3/143) were identified. Compared to known fusions, the novel fusions were identified in fewer sequencing reads (8 vs 13 median junction reads, p = 0.0064) and tended to have a higher median fusion burden (6 vs 2 median fusion isoforms per sample, p < 0.0001), more frequent
 TP53
 co-mutations (80% vs 11.5%, p < 0.0001), and no clear association with tissue of origin. Novel fusions, excluding
 ATXN3::MAML2
 , were located near
 MAML2
 in the genome and appeared to result from duplication or deletion events. At least 75% of samples harboring each novel fusion also carried a concurrent pathogenic mutation, primarily involving
 TP53
 . In contrast,
 ATXN3::MAML2
 fusions, the product of which interacts with YAP1, arose via translocation, occurred in samples with a low fusion burden (2 fusion isoforms per sample), and were not associated with
 TP53
 mutations.
 Conclusions:
 Our comprehensive analysis of tumor samples identified several novel
 MAML2
 fusion partners. Most of these appear to represent passenger alterations arising from genomic instability, likely due to impaired p53 function. However,
 ATXN3::MAML2
 fusions, previously reported in a precancerous pancreatic disease case, may represent a pathogenic alteration warranting further investigation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e97cd82342c70352bfdec0ff8af6680522007f" target='_blank'>
              Novel
 MAML2
 fusions as oncogenic drivers in human malignancy.
              </a>
            </td>
          <td>
            Takefumi Komiya, K. Sweeney, Chao Hui Huang, A. Crymes, Beverly Wang, E. Antonarakis, A. Elliott, M. Oberley, M. Evans
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cervical cancer remains a significant global health concern, impacting over half a million women annually. The primary cause is a persistent infection with hr-HPV, which disrupts various cellular processes crucial for normal function. This disruption leads to genetic instability, including changes in the expression of microRNAs and their corresponding host genes, with far-reaching consequences for cellular regulation. Researchers have widely utilized high-throughput technologies to analyze gene expression in cervical cancer, aiming to identify distinct molecular signatures of microRNAs and genes through genomic analysis. However, discrepancies among studies have been noted, possibly due to variations in sample collection, technological platforms, and data processing methods such as normalization and filtering. Therefore, it is essential to synthesize findings from diverse studies to comprehensively understand the molecular mechanisms of regulatory genetic networks involved in the initiation and progression of cervical cancer. This review examined the evidence detailing the role of microRNA signatures and their target genes in cervical carcinogenesis and disease advancement. The accumulated data suggest the presence of widespread regulatory genetic networks active in both precancerous and cancerous cervical cells, potentially acting as key drivers of this malignancy. Identifying these molecular genomic signatures could open new avenues for developing therapeutic strategies for cervical cancer, particularly in the realm of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5110a729c4003c4b384cf7e2d5409b9bf5d54dee" target='_blank'>
              Regulatory Genetic Networks by microRNAs: Exploring Genomic Signatures in Cervical Cancer
              </a>
            </td>
          <td>
            C. Pérez-Plasencia, Y. C. Orbe-Orihuela, Armando Méndez-Herrera, Jessica Deas, Claudia Gómez-Cerón, H. Jiménez-Wences, J. Ortiz-Ortiz, G. Fernández-Tilapa, Aldo F. Clemente-Soto, J. R. Parra-Unda, J. S. Velarde-Félix, M. Rodríguez-Dorantes, O. Peralta-Zaragoza
          </td>
          <td>2025-06-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Purpose The ovarian metastases originating from colorectal cancer (CRCOM) develops rapidly and lethally. Previously, the genetic alterations and metastatic pathway in CRCOM were not well understood. The aim of this study is to explore the special molecular phenotype and dissemination patterns of CRCOM. Methods The whole-exome sequencing (WES) was performed on 65 matched tissue samples from 11 CRCOM patients, including 11 primary colorectal cancer (CRC) with 11 matched normal tissues, and 43 multi-site metastases (including 15 CRCOMs and 4 patients had bilateral ovarian metastases (OMs). Genetic landscape, neoantigens, tumor clonal origin and spread of CRCOMs were analyzed. TCGA-COAD dataset combined with our data were used for survival analysis and validation of the findings. Results There was significant intertumoral heterogeneity among patients with CRCOM and intra-tumoral heterogeneity among multiorgan metastases. 19 genes were inferred as the potential driver genes of CRCOM. USP7 and RPA1 were HRD-related mutations and potential to serve as predictive biomarkers in OM. The putative neoantigen number of the primary CRC and OM varies widely among patients. The OM showed an immune desert state, extremely deficient in each subtype of immune cells. According to COSMIC signatures features, the CRCOM patients were divided into two groups, which are different in overall survival (OS) (median OS, 720 days vs 360 days, P = 0.074) and genetic alterations. Two metastatic patterns of CRCOM were summarized, which were primary CRC to OM, and metastases to metastases (including lymph node metastases (LNM) to OM, peritoneal metastases (PM) to OM, and other metastases to OM). Interestingly, the sources of bilateral OM might be different in the two patients. Conclusion This study presents a better understanding the heterogeneity of the genetic characterizations and metastatic pattern in CRCOM. The subtypes of CRCOM with USP7 mutation, more copy number alterations, lower neoantigens, and immunoscore have a worse prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f79bcc3ee858322f861b93f18ac7e9a5b5f8b2c" target='_blank'>
              The heterogeneity of genomic alterations, metastatic patterns and immune microenvironment in metastatic ovarian cancer originating from colorectal cancer
              </a>
            </td>
          <td>
            Chao Chen, Jian Wang, Binjie Sun, Yuyan Zheng, Xiaoxu Ge, Zhiyuan Gong, Haochen Gu, Zhiwei Zhang, Akao Zhu, Yingkuan Shao, Yeting Hu, Lijia Ma, Yini Li, Kefeng Ding, Da Wang, Lifeng Sun
          </td>
          <td>2025-06-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 2519


 Background:
 Tumor heterogeneity and low mutational burden limits the availability of effective immunotherapy targets. Aberrant RNA-splicing (neojunctions) represents an underexplored yet promising source of neoantigens. To address this, we developed a neoantigen discovery platform (SNIPP) that characterizes a novel class of clonally-expressed, splicing-derived neoantigens. Furthermore, we validated the immunogenicity of these neoantigens by identifying specific TCRs that drive CD8+ T-cell-mediated tumor killing.
 Methods:
 SNIPP identified public neojunctions by analyzing TCGA RNA-seq data, selecting neojunctions with a positive sample rate (PSR) > 10% and filtering out those found in GTEx normal tissue RNA-seq data (PSR < 1%) across 12 cancer types. To characterize intratumorally conserved neojunctions, we performed maximally-distanced multi-site biopsies (
 n
 = 535) within glioma patients (
 n
 = 56) and generated RNA-seq data for each intratumoral site. Two independent algorithms were utilized to predict peptide processing likelihood and HLA-binding affinity of splicing-derived neoantigen candidates. Neoantigen-specific TCR sequences were identified via
 in vitro
 sensitization of PBMCs and subsequent 10x V(D)J scRNA-seq. These TCRs were transduced into CD8+ T-cells, which were tested downstream for immunogenicity and cytotoxicity against glioma cell lines.
 Results:
 Our pipeline identified 789 public neojunctions, including 32 neojunctions concurrently detected in transcriptomic and proteomic glioma datasets and confidently predicted to be presented by HLA-A*02:01. IVS and subsequent 10x V(D)J scRNA-seq identified TCR clonotypes reactive against neojunctions in
 RPL22
 (
 n
 = 7) and
 GNAS
 (
 n
 = 1), with the latter exhibiting high intratumoral conservation (detected in > 90% of spatially-mapped biopsies across 17/56 patients (26.78%)). TCR-transduced CD8+ T-cells recognized and were immunogenically activated and demonstrated cytotoxicity against endogenously processed and presented neoantigens in GBM and melanoma lines. Additionally, IDH1-mutant oligodendrogliomas exhibited significantly higher neojunction expression compared to IDH1-mutant astrocytomas and IDH1wt subtypes. Differential gene expression analysis (DESeq2) revealed reduced expression of splicing factors in oligodendrogliomas, attributed to their specific co-deletion of chromosomes 1p and 19q. CRISPRi-mediated knockdown of these splicing factors (e.g. SF3A3, SNRPD2) in IDH1wt glioma cells resulted in significantly increased expression of corresponding neojunctions.
 Conclusions:
 Our study highlights a novel class of neoantigens derived from tumor-wide aberrant RNA splicing. The SNIPP platform effectively identifies public intratumorally-conserved neojunctions with strong therapeutic potential. Furthermore, elevated neojunction expression in oligodendroglioma underscores the mechanistic link between dysregulated splicing factor expression and RNA splicing abnormalities.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/476976fc09e028193d5d0a3658a6d5d31763d4d8" target='_blank'>
              Tumor-wide RNA splicing aberrations and their potential as therapeutic neoantigen targets.
              </a>
            </td>
          <td>
            Darwin W. Kwok, Nicholas O. Stevers, Iñaki Etxeberria, T. Nejo, Maggie Colton Cove, Lee-may Chen, Jangham Jung, K. Okada, S. Lakshmanachetty, M. Gallus, Abhilash Barpanda, Chibo Hong, Aaron A. Diaz, Shawn L. Hervey-Jumper, Susan Marina Chang, Joanna J Phillips, Arun P Wiita, Christopher A. Klebanoff, Joseph F Costello, Hideho Okada
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Transposable elements (TEs) make up around half of the human genome. Their transcription is repressed in most somatic cells to maintain genome integrity and function. The repression is chiefly maintained by a combination of epigenetic modifications such as DNA methylation and histone modifications. However, recent research suggests that liver steatosis is associated with extensive changes to the hepatocyte epigenome. Furthermore, studies in mice have reported diet- and drug-induced changes to TE transcript levels in liver. The confirmation of these effects in human liver has not previously been undertaken. Here, we examined TE transcription in liver tissue from three patient cohorts with histologically confirmed liver steatosis caused by alcohol consumption or metabolic dysfunction. The quantitation of the number of transcripts with TE-homology in RNA-Seq data from a cohort of 90 bariatric surgery patients with metabolic dysfunction-associated steatotic liver disease (MASLD) revealed a trend for the reduction in TEs of all classes due to increasing steatosis, but no effect of fibrosis. This pattern was also present in a separate cohort of MASLD and HCC patients, as RT-qPCR also showed a reduction in Alu element transcripts in advanced steatosis, but again, no effect of fibrosis. Contrastingly, in a cohort of alcohol-related liver disease patients, the reduction in LINE-1 transcripts was associated with either increased steatosis or increased fibrosis. Moreover, the examination of LINE-1 DNA methylation levels in the MASLD and HCC cohort indicated that DNA methylation was also negatively associated with LINE-1 transcription in MASLD. This study suggests that TE transcript levels in human liver are slightly reduced by steatosis, that DNA methylation is an influential epigenetic regulator of LINE-1 retrotransposon transcription in steatosis, and that Alu transcript levels in background liver could be a new biomarker for HCC in cirrhotic and non-cirrhotic MASLD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2228b88546bad7926adb359aec31ed752195e972" target='_blank'>
              Multi-Cohort Exploration of Repetitive Element Transcription and DNA Methylation in Human Steatotic Liver Disease
              </a>
            </td>
          <td>
            N. Youngson, A. Tourna, Timothy J. Chalmers, K. Prates, J. Argemi, Ramón Bataller, K. S. Haghighi, Lindsay E. Wu, Shilpa Chokshi, Peter Starkel, Patrick S. Western, M. Morris, Stephen M. Riordan
          </td>
          <td>2025-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The CHEK2 gene serves a canonical role in the DNA damage response (DDR) pathway encoding the regulatory kinase CHK2 in the homologous recombination (HR) repair of double-strand breaks (DSB). Although CHEK2 is traditionally considered a tumor suppressor gene, recent studies suggest additional functions. Across several cohort studies, CHEK2 expression was negatively correlated with the efficacy of immune checkpoint inhibitors (ICI), which target the interaction between effector immune and tumor cells. This review explores two possible explanations for this observed phenomenon: the first relating to the canonical role of CHEK2, and the second introducing a novel role of the CHEK2 gene in immunomodulation of the tumor microenvironment (TME). DDR mutations have been implicated in increased levels of tumor mutation burden (TMB), often manifesting as neoepitope expression on the tumor cell surface recognized by effector immune cells. As a result, impaired DNA repair due to CHEK2 loss of function, either from germline deleterious variants or acquired mutations, results in the recruitment of CD8+ cytotoxic T-cells and subsequent efficacy of ICI treatment. However, functional loss of CHEK2 may be directly involved in potentiating the immune response through canonical inflammatory and anti-tumor pathways, acting through the cGAS-STING pathway. Although the exact mechanism by which CHEK2 modulates immune responses is still under investigation, combination therapy with CHEK1/2 inhibition and ICI immunotherapy has shown benefit in preclinical studies of several solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/561bae3be7aafdc8f4a911392f292dcb271c31d1" target='_blank'>
              Beyond DNA damage response: Immunomodulatory attributes of CHEK2 in solid tumors
              </a>
            </td>
          <td>
            Helen Qian, Heba Ali, Vivekanudeep Karri, Justin T. Low, David M Ashley, Amy B. Heimberger, Lucy A. Godley, A. Sonabend, C. Dmello
          </td>
          <td>2025-06-10</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 The loss of chromosome 16q is the single most common genomic event in BC, occurring in over 50% of tumors. Despite this, the literature is focused almost exclusively on the importance of focal mutations that show a maximum frequency of ∼35-40%. We have modeled 16q loss by deleted this region in mammary epithelium of genetically modified mice. This event was sufficient to induce mammary tumor formation. This finding establishes for the first time that 16q is a BC-suppressing chromosome arm. We have gone on to identify two genes on 16q that can promote tumor formation when one copy is lost (Cbfb and Ankrd11, each in cooperation with Pik3caHR). We have also identified mutations that cooperate with 16q syntenic sequence loss in spontaneous tumors from our 16q mouse model (c-Met gene amplification and p53 mutation) as well as in a Sleeping Beauty transposon-based screen (identifying mutations that enhance Ras and Rho signaling). Finally, we have used transcriptional profiling and proteomics to compare matching mammary tumors with/without 16q-syntenic sequence loss, which revealed that hemizygous deletion of this region suppressed keratinocyte-like differentiation in mammary tumors.
 Citation Format: Sean Egan, Idil Eda Temel, YeJi An, Katelyn Kozma, Amanda Loch, Wei Wang. 16q is a breast cancer suppressor arm [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e57f36d665bf1dd94705e6477b5680fa560c1a" target='_blank'>
              Abstract P3-04-17: 16q is a breast cancer suppressor arm
              </a>
            </td>
          <td>
            Sean E Egan, Idil Eda Temel, Yeji An, Katelyn J Kozma, Amanda J. Loch, Wei Wang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 A deeper understanding of the complicated transcriptional landscape of human breast cancers are vital to identifying more efficacious treatments. Investigating genetic variations among breast cancer subtypes at a single-cell level holds the potential to enhance our understanding of cancer formation, progression, and elucidate actionable targets. In this study, we compile single-cell RNA sequencing data from patient tumors and matched lymph metastasis, reduction mammoplasties, breast cancer patient-derived xenografts (PDXs), PDX-derived organoids (PDXOs), and cell lines, forming a diverse dataset consisting of 117 samples with a total of 506,719 cells. These samples include hormone receptor-positive (HR+), human epidermal growth factor receptor 2 positive (HER2+), and triple-negative breast cancer (TNBC) subtypes, along with isogenic model pairs. Within this study, we outline the similarities and differences across these models and patient samples, and assess therapeutic drug efficacy based on subtype proportions. Within the TNBC subtype, PDX models exhibited a closer resemblance to patient samples in terms of tumor heterogeneity and cell cycle features compared to cell lines. Acquired drug resistance in TNBC PDX tumors was linked to an increase in basal-like cell proportions, as defined by SCSubtype and TNBCtype cell typing predictors. Cell-wise subtyping revealed all patient samples comprised a mix of subtypes; notably, HR+ lymph node metastases had a lower proportion of HER2-Enriched cells compared to matched primary tumors. PDXOs displayed differences in metabolic-related transcripts relative to matched PDX tumors. Correlative analyses of cytotoxic drugs on PDX cells demonstrated that therapeutic efficacy was dependent on subtype proportions. Herein we provide an extensive multi-model dataset, a dynamic approach to cell-wise sample annotation, and a thorough examination of models within human breast cancer systems. This analysis and reference resource will support informed decision-making in preclinical research and therapeutic development by establishing model limitations, subtype-specific insights, and novel targetable pathways.
 Citation Format: Julia Altman, Amy L. Olex, Emily K. Zboril, Carson J. Walker, David C. Boyd, Rachel K. Myrick, Nicole S. Hairr, Jennifer E. Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. Steven Chen, Yunshun Chen, Charles M. Perou, Brian D. Lehmann, Jane E. Visvader, J. Chuck Harrell. Single-cell transcriptional features of human breast tumors and model systems [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-19.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad1272929a89f74bc5ab4bbe73f4045f2023308" target='_blank'>
              Abstract P5-06-19: Single-cell transcriptional features of human breast tumors and model systems
              </a>
            </td>
          <td>
            Julia E Altman, Amy L. Olex, Emily K. Zboril, Carson J Walker, David C. Boyd, Rachel K. Myrick, Nicole S Hairr, Jennifer E Koblinski, Madhavi Puchalapalli, Bin Hu, Mikhail G. Dozmorov, X. S. Chen, Yunshun Chen, C. Perou, B. Lehmann, Jane E. Visvader, J. Harrell
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fda3215ef96d1797670c5063bd5db4df44e918" target='_blank'>
              Genetic variants in oncogenic miRNA and 3' untranslated region of tumor suppressor genes: emerging insight into cancer genetics.
              </a>
            </td>
          <td>
            Anita Choudhary, Anil Kumar, A. Munshi
          </td>
          <td>2025-06-30</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Cancer progression and development are influenced by a complex interplay between inherited (germline) and acquired (somatic) mutations. Current cancer genomic research in Middle Eastern populations predominantly utilizes targeted panels to examine predefined genes, potentially overlooking a broader spectrum of genomic contributions to cancer predisposition. Addressing this gap, this study adopts an unbiased approach using whole exome sequencing (WES) data to identify both high- and low-penetrance genetic variants within the United Arab Emirates (UAE) population. This investigation features a case-control matching analysis comprising 62 patients diagnosed with various cancer types and 142 unrelated healthy controls. The results showed a potential association between cancer predisposition and variants within. The results demonstrate an association between cancer predisposition and variants within C-terminal Binding Protein 2 (CTBP2), DNA Polymerase Theta (POLQ) and Tektin 4 (TEKT4). Gene set enrichment analysis showed enriched pathways associated with cancer-related biological processes such as DNA repair and depleted pathways related to translation, cellular metabolic process, and mitochondrial functions. This study highlights that the distinctive genetic composition of underrepresented groups influences the penetrance of pathogenic variants that could contribute to hereditary cancer risk in ways that diverge from patterns observed in more extensively researched cohorts. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05296-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972fb28f89280ac4ce146e45e2b00c9a74cb002a" target='_blank'>
              Pan-Cancer Exome-wide analysis of germline mutational patterns and pathways
              </a>
            </td>
          <td>
            Halima Alnaqbi, Michael Olbrich, Noora Zayed, Mira Mousa, S. Azzam, S. Tirmazy, Habiba S. Alsafar
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca9aae7f279fb50f5b49767837c2e747991818a1" target='_blank'>
              TET3 facilitates bladder cancer progression through targeted modulation of stemness pathways.
              </a>
            </td>
          <td>
            Zhiren Cai, Yanqi Xie, Luyao Li, Yuhang Zheng, Xianghui Zhou, Guanghua Zhou, Ganping Wang, Xin Zeng
          </td>
          <td>2025-06-26</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Radioactive iodine-refractory (RAI-R) thyroid cancer is associated with poor prognosis and limited treatment options. Identifying RAI-R status early in treatment is crucial for optimizing therapy. This study aims to identify genetic variants distinguishing RAI-R from radioiodine-sensitive (RAI-S) thyroid cancers. The study included six RAI-R patients and six age-matched RAI-S controls. DNA from tumor tissues and peripheral leukocytes of RAI-R patients underwent long-read whole-genome sequencing, while tumor DNA from RAI-S patients underwent whole-exome sequencing. RNA and immunohistochemistry (IHC) analyses were performed to assess candidate variants. Among the six RAI-R patients, three showed disease progression. These patients harbored somatic heterozygous variants in the TG gene, including a c.7863-7_7864delTTCTCACinsG variant, an 8,232 bp deletion encompassing exons 33–34, and a missense variant (c.5488 C > T; p.Pro1830Ser). No TG variants were found in RAI-S or non-progressive RAI-R patients. Additionally, one RAI-R patient had a somatic nonsense variant in the RELA gene (c.601 C > T; p.Arg201Ter). qRT-PCR showed lower NIS expression in RAI-R patients with TG or RELA variants, whereas RAI-S patients exhibited higher NIS expression. Somatic TG and RELA variants were identified in RAI-R thyroid cancer. These variants could serve as early biomarkers for identifying RAI-R status, enabling more tailored treatment strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05605-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f2e40de7eabe036a9c22df6ac0d66c665a19302" target='_blank'>
              Somatic mutations in the TG and RELA genes specific for radioiodine-refractory thyroid cancer
              </a>
            </td>
          <td>
            Chanan Suprakun, Pattarin Nuwongsri, S. Tongkobpetch, C. Srichomthong, C. Vinayanuwattikun, S. Keelawat, Supavadee Jantasuwan, N. Tangjaturonrasme, Pasurachate Samorn, V. Shotelersuk, Supatporn Tepmongkol
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Disorders of sex development (DSDs) are characterized by discrepancies among karyotype, the gonadal phenotype and gonadal anatomy following birth. Among these, 46, XY DSD is the most complex sub-type and a major cause of birth defects associated with sexual development. However, due to the considerable heterogeneity in pathogenic genes, numerous cases remain genetically undiagnosed. In the present study, a novel gain-of-function variant was identified in the mitogen-activated protein 3 kinase 1 (MAP3K1) gene, contributing to 46, XY DSD through the induction of abnormal cell apoptosis. Genetic analysis of a pediatric male patient and his family revealed a heterozygous MAP3K1 c.4445 G>A variant, resulting in an arginine-to-glutamine substitution. The variant site is highly conserved across species and the amino acid transition induced structural changes in the MAP3K1 protein. In vitro experiments demonstrated that the MAP3K1 c.4445 G>A variant markedly increased apoptotic signaling, leading to abnormal cell apoptosis, disruption of the cell cycle and reduced cell viability. Moreover, the variant exhibited increased levels of ERK1/2 and p38 phosphorylation, indicative of a gain-of-function effect. Subsequent analysis revealed increased expression of the testis-determining gene, SOX9 and reduced expression of the ovary-determining gene, FOXL2. Notably, alterations in gene expression were associated with the MAP3K1 c.4445 G>A variant, providing a mechanistic basis for the pathogenesis of 46, XY DSD. Collectively, these findings offer novel insights into the anti-apoptotic role of MAP3K1, advancing genetic diagnosis and pre-natal screening for individuals with DSDs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2799a19a1fd170b1fea72ad9ddd73906ece00960" target='_blank'>
              A novel variant in the MAP3K1 genomic locus reveals abnormal cell apoptosis as a potential pathogenic mechanism in 46, XY disorders of sex development
              </a>
            </td>
          <td>
            Yufu Lu, Sijia Wei, Shuang Wang, Jingzhi Zhang, Yongjie Xu, Changyudong Huang, Wei Pan, Zhengrong Wang
          </td>
          <td>2025-06-05</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Human endogenous retroviruses (HERVs) comprise genetic material from germ cells that were infected with ancient retroviruses during Pan troglodytes' evolutionary divergence. These "non-coding" repetitive elements now make up approximately 8% of the human genome. Once inserted in the genome, HERVs increase copy number via retrotransposition (a 'copy-and-paste' mechanism) and are involved key functions including transcriptional control and cellular fusion during placenta formation. Aberrant re-expression of HERV-associated genes initially silenced by host epigenetic mechanisms, including DNA methylation, has been linked to many diseases including cancer. Furthermore, in addition to regulating gene expression, including of neighboring genes, reversal of silencing of HERVs may also impact diverse biological functions through the LTRs, and gag, pol, and env proteins. Although previous studies, specifically in white populations, have documented the expression of HERV-K in breast cancer, there is a paucity of data on the expression and role of HERV-R in triple-negative breast cancer (TNBC) among rural West African women from Nigeria. Breast cancer remains the most frequently diagnosed cancer among women globally, with the highest mortality rates across sub-Saharan African nations compared with other countries. Advanced-stage disease at presentation, as well as limited access to treatment options, typically contribute to the poor overall survival rate of breast cancer in African countries including Nigeria. However, the incidence and mortality rates of TNBC in women with West African ancestry are alarmingly high, with an overall survival rate of three years in Zaria, Nigeria. To better understand the molecular basis underlying these high incidence and mortality rates, as well as identify prognostic biomarkers we assessed expression patterns of HERV-R and TNBC-associated genes from Formalin-Fixed Paraffin-Embedded (FFPE) samples of breast cancer, focusing on TNBC in Nigerian women from rural West Africa. To examine gene expression in this study, we used immunohistochemistry, immunoblot, and immunofluorescence, with results showing strong expression of the HERV-R env protein in all but one TNBC patient, as well as in other basal breast cancer subtypes. Our results also strongly suggest that HERV-R expression is associated with overexpression of oncogenes, including TMPRSS2, BRAF, CCND1, BAZ1B, KTM2D, and FOXA1. We found that patients with HERV-R expression have a poor prognosis with low survival. These findings imply that activation of HERV-R may play a role in regulating the expression of genes implicated in TNBC and may serve as a prognostic biomarker in breast cancer.
 Citation Format: Faruk Mohammed, Mohammed Daniyan, Ines Hosni, Halimatu Sadiya Musa, Sani Kamarudeen Owolabi, Adoke Kasimu Umar, Yawale Iliyasu, Rebecca Garnham, Jane Carr-Wilkinson, Sani Ibrahim, Kevin Petrie. Activation of HERV-R is a biomarker of poor prognosis among rural Nigerian women with triple-negative breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P1-09-18.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50d98da126d396eab075d52af30620900c11d04d" target='_blank'>
              Abstract P1-09-18: Activation of HERV-R is a biomarker of poor prognosis among rural Nigerian women with triple-negative breast cancer
              </a>
            </td>
          <td>
            Faruk Mohammed, Mohammed Daniyan, Ines Hosni, H. S. Musa, S. K. Owolabi, A. Umar, Y. Iliyasu, Rebecca Garnham, Jane Carr-Wilkinson, Sani Ibrahim, Kevin Petrie
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Thyroid cancer frequently metastasizes, even in small lesions, significantly impacting prognosis. Circular RNAs (circRNAs) have emerged as critical regulators of tumor biology; however, their roles in thyroid cancer progression remain poorly understood. Using RNA sequencing data from clinical and GEO-derived samples, we identified 111,971 circRNAs, including 13,399 novel thyroid cancer-specific circRNAs. TNM stage-specific analysis revealed 43 circRNAs positively correlated with tumor size (Rho = 1) and 168 negatively correlated (Rho = -1). Gene Ontology enrichment analysis demonstrated that the target genes of thyroid cancer-specific circRNAs are enriched in cell developmental regulation pathways (NES = 1.74, p < 0.05). Network analysis further suggested that these circRNAs function as competing endogenous RNAs by interacting with tumor-associated microRNAs. Among these, circAQR was significantly associated with tumor size and lymph node metastasis. In vitro experiments revealed that circAQR inhibits proliferation while promoting migration and invasion in thyroid cancer cell lines. These findings highlight the pivotal role of TNM stage-specific circRNAs in thyroid cancer progression, with circAQR showing potential as a biomarker for early metastasis detection and a therapeutic target. The integration of circRNAs into clinical practice could improve early diagnosis and prognostic evaluation, contributing to more effective management of thyroid cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86497152c09fabb541a9705a2578a90e0530c85a" target='_blank'>
              Genome-wide analyses identify circAQR, a circular RNA suppressing proliferation but promoting migration and invasion in thyroid cancer
              </a>
            </td>
          <td>
            Tianjiao Zhou, Zheng Li, Weijun Huang, Enhui Zhou, Yumeng Jiang, Yueqing Bai, Chen Xu, Wen Lu, Kaiming Su, Chuan Xu, Hongliang Yi
          </td>
          <td>2025-06-11</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d66d1c34039c0624a67bfae6ffcc0194c9308ee" target='_blank'>
              Bioelectric and Epigenetic Landscapes in Lateralized Breast Tumors Reveal Distinct Tumor Microenvironment Signatures
              </a>
            </td>
          <td>
            Sebastián Real, Sergio R. Laurito, Pablo Gonzalez, Oscar Bello, Joao Carvalho, María Roqué
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 e14558


 Background:
 DNA polymerases ε (
 POLE
 ) and δ1 (
 POLD1
 ) encode enzymes involved in DNA replication and proofreading. Somatic
 POLE
 /
 D1
 mutations occur in 1-3% of cancers. Loss-of-function alterations typically reside within the
 POLE
 /
 D1
 exonuclease domain and can result in high tumor mutation burden (TMB). Ultrahigh TMB (≥100 mut/Mb) is often associated with
 POLE/D1
 mutations, and results in near-universal responses to immune checkpoint inhibitors (ICI). Deciphering the contribution of
 POLE
 /
 D1
 mutations to TMB can be challenging, as these often co-occur with microsatellite instability (MSI-H) which also drives TMB.
 Methods:
 Samples were analyzed by NGS (NextSeq or NovaSeq) (Caris Life Sciences, Phx, AZ). TMB (totaled somatic mutations found per megabase pair) was evaluated in patients with
 POLE
 /
 POLD1
 /
 D2
 /
 D3
 /
 D4
 -altered cancers. MSI-H or microsatellite stable (MSS) status was determined by IHC and NGS. Cohorts were divided into pathogenic and likely pathogenic variants (i.e. exonuclease domain mutations) versus VUS alterations (i.e. variants outside the exonuclease domain). Differences in TMB between cohorts were assessed using Fisher’s exact test.
 Results:
 Pan-cancer, TMB ≥20 / ≥50 was identified in 10.9% / 3.4% of
 POLE
 VUS MSS cancers compared to 3.1% / 0.4% of MSS cancers with
 POLE
 /
 D1
 WT status (
 p
 < 0.0001). 0.94% of
 POLE
 VUS cancers had ultrahigh TMB versus 0.05% of MSS
 POLE
 /
 D1
 WT cancers (
 p
 < 0.0001). Similar rates were observed in MSS cancers with
 POLD1
 VUSs (Table). MSS cancers with pathogenic
 POLE
 variants exhibited variable rates of ultrahigh TMB depending on cancer type, ranging from 52.6% (ovarian) to 62.9% (endometrial) to 80.0% (colorectal). Notably, ultrahigh TMB occurred in 0.53%, 0.44%, and 0.44% of MSS patients with
 POLD2
 ,
 POLD3
 , and
 POLD4
 mutations (
 p
 < 0.0001), after excluding coexisting
 POLE
 /
 D1
 mutations.
 Conclusions:
 We propose that certain non-exonuclease
 POLE
 /
 D1
 mutations can result in loss-of-function and ultrahigh TMB in MSS patients. Further, tumor type appears to influence the impact of established
 POLE
 /
 D1
 driver mutations on TMB. Finally,
 POLD2
 -
 4
 mutations (not previously studied) may also contribute to genome instability and mutational load. These findings argue for deeper characterization of DNA polymerase mutations to identify patients that may benefit from ICI.






 POLE
 /
 D1
 WT + MSS[n (%)]


 POLE
 VUS Mutation + MSS[n (%)]


 POLE
 Pathogenic Mutation + MSS[n (%)]


 POLD1
 VUS Mutation + MSS[n (%)]


 POLD1
 Pathogenic Mutation + MSS[n (%)]





 Total
 139,842
 5,658
 557
 1,793
 11


 TMB ≥20
 4,378(3.1%)[ref]

 614(10.9%)
 p
 <0.0001


 529(95.0%)
 p
 <0.0001


 168(9.4%)
 p
 <0.0001


 5(45.5%)
 p
 <0.0001



 TMB ≥50
 586(0.42%)[ref]

 193(3.4%)
 p
 <0.0001


 467(83.8%)
 p
 <0.0001


 79(4.4%)
 p
 <0.0001


 2(18.2%)
 p
 =0.0009



 TMB ≥100
 73(0.05%)[ref]

 53(0.94%)
 p
 <0.0001


 343(61.6%)
 p
 <0.0001


 28(1.6%)
 p
 <0.0001


 1(9.1%)
 p
 =0.0058







 Evaluated by Fisher’s exact test comparing to respective POLE/D1 WT + MSS cohort.




">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eb38b846148e42ca1249ce42b9e558a15a34574" target='_blank'>
              Potential impact of non-exonuclease
 POLE
 and
 POLD1
 mutations on high and ultrahigh TMB across cancers.
              </a>
            </td>
          <td>
            Alexander K. Tsai, S. Deshmukh, Sharon Wu, Dave S. B. Hoon, W. El-Deiry, Axel Grothey, Justin H Hwang, E. Antonarakis
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="In recent years, the incidence of cancer has increased, and early diagnosis and treatment are key to cancer treatment. Nucleolar and spindle-associated protein 1 (NUSAP1) is a crucial mitotic regulator. NUSAP1 binds to microtubules and mediates their attachment to chromosomes, ensuring the accurate distribution of genetic material to the two daughter cells. NUSAP1 is located on chromosome 15q15.1 and is typically expressed in proliferating cells, suggesting a potential oncogenic role. Studies have shown that NUSAP1 plays a significant role in various cancers, including breast cancer, cervical cancer, lung cancer, prostate cancer, and hepatocellular carcinoma. Furthermore, NUSAP1 may serve as a promising biomarker for cancer diagnosis and prognosis. In this review, we discuss the roles and mechanisms of NUSAP1 in human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e17ed1053e2b63b1afa28756e4052dc1859c3cc0" target='_blank'>
              The role and therapeutic value of NUSAP1 in human cancers
              </a>
            </td>
          <td>
            Shenyuan Xu, Ding Nan, Ruiqi Liu, Chenting Liu, Yuefeng Hu, Luanluan Huang, Xiaoyan Chen, Hailong Sheng, Yanwei Lu, Haibo Zhang
          </td>
          <td>2025-07-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Triple-negative breast cancer (TNBC, lacking expression of estrogen and progesterone receptors and amplification of HER2) is an aggressive and drug-resistant subtype. Current standard-of-care for early-stage disease with neoadjuvant chemoimmunotherapy results in a pathologic complete response in only ∼60% of patients. Proliferative heterogeneity naturally occurs in most tumors, where tumor cells exist in multiple different states with varying proliferative potential—states of active division, long- and short-term quiescence, pre-cell cycle entry, and stress. TNBC tumors exhibit a higher percentage of actively cycling, or Ki-67–positive cells, compared to other molecular subsets of breast cancer, which has been shown to predict higher pathologic complete response rates following neoadjuvant chemotherapy. Unfortunately, despite this, TNBC patients experience worse overall survival rates driven by higher rates of relapse. It has been proposed that a significant portion of noncycling, or Ki-67–negative, cells are in a G0-like, or quiescent, state and predicted therefore to be resistant to cytotoxic chemotherapies that rely on actively dividing cells. The presence of a tumor cell population in a quiescent/G0-like state has been increasingly recognized across many tumor types, including in breast cancer. Patient-derived tumor organoids have emerged as an appealing model system that retains more cellular heterogeneity from primary tumors compared to widely used two-dimensional cell lines. Organoids show remarkable complexity by single-cell RNA sequencing, representing many more cell states compared to best available two-dimensional cell lines that most closely resemble the mutation, copy number variation, gene expression and protein expression profiles of patient tumors. A mutant version of p27 containing mutations blocking binding to cyclin/cdk complexes, but retained proteolysis in S/G2/M, combined with an mVenus fluorescent protein (p27-mVenus) has been previously validated as a genetic reporter for steady-state G0 readouts. We have optimized lentiviral transduction of a triple-negative breast cancer patient-derived xenograft organoid with the p27-mVenus reporter to label quiescent organoid subpopulations. Optimized parameters include method of lentivirus concentration, multiplicity of infection, duration of transduction, presence of protamine, presence of a Matrigel base layer, presence of RetroNectin and use of spinoculation. We will use this p27-mVenus TNBC organoid reporter line to study the response of quiescent cell populations to current standard of care treatments and how they contribute to therapeutic resistance.
 Citation Format: Darien Reed-Perino, Sonali Arora, Alana L Welm, Cyrus M Ghajar, Patrick J Paddison. Optimization of lentiviral transduction of a triple-negative breast cancer patient-derived xenograft organoid for modeling tumor cell quiescence and associated treatment resistance [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-21.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9268e78f9e9bcde52a8f24e0b21e89d64bf2c6e3" target='_blank'>
              Abstract P3-04-21: Optimization of lentiviral transduction of a triple-negative breast cancer patient-derived xenograft organoid for modeling tumor cell quiescence and associated treatment resistance
              </a>
            </td>
          <td>
            Darien E Reed-Perino, Sonali Arora, A. Welm, Cyrus M Ghajar, Patrick J. Paddison
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="The relationship between epigenetic modifications that influence gene regulation, such as histone deacetylation, microRNA regulation and DNA methylation, and human diseases has become increasingly recognised. Dysregulation of DNA methylation has been implicated in genomic imprinting and tumourigenesis. Previous studies identified several epigenetic modifier genes that directly cause genomic DNA (gDNA) methylation such as DNA methyltransferases (DNMTs).DNMTaberrations usually affect genome-wide methylation profiles, deactivating many tumour suppressor genes by methylation of their promoters. In this study, we describe a unique case with several tumours of different lineages, likely caused by a germline pathogenic DNA (cytosine-5-)-methyltransferase 3 alpha (DNMT3A)variant.We evaluated a middle-aged female who suffered from multiple metachronous and synchronous tumours, mostly benign, including subcutaneous lipomas and mesenchymal tumours, multinodular goitre, a parathyroid adenoma, a complex meningioma, a benign breast tumour, bilateral non-functioning adrenal nodules, hepatic and renal cysts and a pancreatic neuroendocrine tumour. The patient’s gDNA was subjected to whole-exome sequencing and strict bioinformatics analysis. The possible effect of the variant was investigated using multiple in-silico analysis tools and protein three-dimensional modelling.A heterozygous missenseDNMT3Avariant (c.2063G>A, p.R688H) was identified. The identified variant was Sanger sequenced in all available patient’s samples. Protein modelling strongly supports the pathogenic role of this variant in the development of the several tumours that this patient had.We report the c.2063G>ADNMT3Avariant as a potential cause of multiple unrelated tumours. This is the first report of such a case and one of a few reports ofDNMT3Agermline variants causing tumourigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/225d0c30ea3c8c06200141f21fb55e55e3264e9c" target='_blank'>
              Multiple tumours of different lineages likely related to aDNMT3Apathogenic R688H variant
              </a>
            </td>
          <td>
            Kheloud M. Alhamoudi, Meshael M. Alswailem, B. Alghamdi, Maram Alshuwaymi, Karla A. Peña-Guerra, Stefan T. Arold, H. Al-Hindi, Ali S Alzahrani
          </td>
          <td>2025-06-01</td>
          <td>BMJ Connections Clinical Genetics and Genomics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Circular RNAs (circRNAs) constitute a large class of non-coding RNAs with gene regulatory capabilities, mainly through microRNA binding, a mechanism that has been linked to cancer development. The circRNA ciRS-7 (also known as CDR1as) is an interesting candidate as it harbors over 60 binding sites for miR-7, which is known to have tumor-suppressing properties. Here, we investigated the spatial expression patterns and epigenetic regulation of ciRS-7 across nine different adenocarcinomas originating from the colon, pancreas, ovary, endometrium, breast, stomach, bile duct, lung, and prostate. The study included primary patient samples and 18 different cell lines. ciRS-7 expression was analyzed using Reverse Transcription-quantitative PCR (RT-qPCR), single molecule in situ hybridization, and Nanostring nCounter, while epigenetic modifications were examined through bisulfite sequencing, Sensitive Melting Analysis after Real Time – Methylation Specific PCR (SMART-MSP), and chromatin immunoprecipitation. The functional relevance of epigenetic modifications was examined using DNA methyltransferase and histone deacetylase inhibitors. Across all adenocarcinomas, ciRS-7 was absent in the cancer cells in most of the primary tumor specimens, except for the breast tumors, while being expressed in the tumor microenvironment (TME). In line with this, ciRS-7 was not detected in most of the cell lines. Moreover, we demonstrated that DNA methylation and H3K9 acetylation, but not H3K27 methylation, are important epigenetic modifications that impact ciRS-7 expression. In conclusion, our data show that ciRS-7 is mainly expressed in the TME and is regulated through DNA methylation and histone acetylation across all major types of adenocarcinomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dd458d2d2876867a8c5e16e63beed678fb4697c" target='_blank'>
              ciRS-7 expression is epigenetically regulated in cancer cells across human adenocarcinomas
              </a>
            </td>
          <td>
            Thea Petersen Paasch, Morten T Jarlstad Olesen, J. L. García-Rodríguez, A. Assmus, Robert A Fenton, Jørgen Kjems, Henrik Hager, L. S. Kristensen
          </td>
          <td>2025-06-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22bcb6bbbf4e6c64ab8d5a6fa755b5306451c07b" target='_blank'>
              EHMT1 mediates cellular motility in embryonal rhabdomyosarcoma by activating SOX8 expression.
              </a>
            </td>
          <td>
            Upasana Bajaj, Dipanwita Das, Jia Yu Leung, Nurul Fateha Binti Abdul Rashi, R. Taneja
          </td>
          <td>2025-06-04</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The development of multiple myeloma is typically associated with various cytogenetic abnormalities; however, these genetic changes alone do not fully account for the observed heterogeneity in patient prognosis and treatment response. Recent studies leveraging next-generation sequencing and genomic approaches have shown that epigenetic alterations are crucial in myeloma development and therapeutic resistance. These changes contribute to high levels of transcriptomic instability and enables cellular adaptation to targeted therapies and immunotherapies through diverse evolutionary trajectories. In this regard, aberrations of histone modifications and chromatin remodeling affect various cellular processes such as DNA repair, DNA damage response, cellular survival, and apoptosis signaling, which provides a strong rationale for developing epigenetic-targeted therapies for myeloma treatment. In this review, we focus on recent advances and research gaps in understanding the deregulation of histone acetylation, a widespread and versatile process of histone modification occurring at lysine residues at the N-terminus of histone tails, and its intimate interplay with chromatin remodeling complexes in orchestrating dynamic chromatin functional states and transcriptional outputs. We also provide an updated review of epigenetic modulatory drugs targeting HDAC, CBP/p300, and bromodomain and extraterminal proteins, along with a discussion of their limitations and future perspectives in myeloma treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d100926ef56ad097039cb56ac307d5417be61de8" target='_blank'>
              Deciphering the dynamics of histone acetylation and chromatin remodeling in multiple myeloma: a tale beyond the tails.
              </a>
            </td>
          <td>
            Sinan Xiong, Jianbiao Zhou, WeeJoo Chng
          </td>
          <td>2025-07-03</td>
          <td>Blood</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Understanding how the assembly of nascent mRNA into a ribonucleoprotein (mRNP) influences R-loop homeostasis is crucial for gaining insight into the cellular mechanisms that prevent genome instability. Here, we identify three RNA-binding proteins, Rie1, Rim4 and She2, whose expression levels are important to limit R-loop accumulation and, thus, to prevent DNA damage. Interestingly, Rim4 and She2 are overrepresented in CBP80-containing mRNPs formed in the absence of THO. In addition, we found that an excess of the RNA exosome component Dis3 impairs its function, promoting R-loops, particularly from non-coding RNAs, which cause genomic instability. Our results indicate that changes in the availability of different RBPs or RNAs, causes R-loop-mediated DNA damage in the cell. These results may help to understand the mechanism that promotes cancer, as several RBPs are overexpressed in different types of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58254c8464c859b1341c1f452949c65b62663dc9" target='_blank'>
              Cellular imbalance of specific RNA-binding proteins associates with harmful R-loops
              </a>
            </td>
          <td>
            José Antonio Mérida-Cerro, Guillaume Chevreux, Benoît Palancade, Ana G. Rondón, A. Aguilera
          </td>
          <td>2025-07-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The pituitary tumor transforming gene 1 (PTTG1), discovered in 1997 by Pei and Melmed, takes part in cellular replication, cell cycle control, DNA repair mechanisms, organogenesis, metabolism regulation, cellular transformation, and senescence. Its biological actions include protein-protein interactions, modulation of gene transcription, and, other than intracellular and autocrine mechanisms, even paracrine activities. For the reasons mentioned above, PTTG1 stands out as a multifaceted regulator of cancer biology; it is involved in genomic and chromosomal instability, local invasiveness, neo-lymphangiogenesis, and metastatic spreading. In solid neoplasms, endocrine neoplasms, although deemed rare, have experienced a significant increase in diagnostic incidence in recent years. Endocrine cancers are still a major challenge in healthcare and research since several questions remain unanswered, even though researchers have made considerable efforts to uncover their causes. Twenty-seven years have passed since PTTG1's discovery, and several works have been published. However, only the tip of the iceberg has been unveiled. Herein, we review current knowledge of PTTG1's action in endocrine cancers, such as pituitary, thyroid, testicular, adrenal, pancreatic, and ovarian.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a57eb69f3187a01c5af405bf459aed1d53f3b1f4" target='_blank'>
              Pituitary tumor transforming gene 1 and endocrine cancers: an up-to-date review through history, current insights and future perspectives.
              </a>
            </td>
          <td>
            E. Vergani, E. Teveroni, Francesca Mancini, F. Di Nicuolo, S. Raia, S. Chiloiro, F. Pierconti, Antonio Bianchi, A. M. Isidori, Alfredo Pontecorvi, Domenico Milardi
          </td>
          <td>2025-06-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="BACKGROUND AIMS
Intrahepatic cholangiocarcinomas (ICCs) are aggressive liver tumors with high heterogeneity and limited therapeutic options. Although traditionally thought to arise from biliary cells, recent findings suggest that hepatocytes may also serve as a cellular origin for ICC. However, the mechanisms underlying hepatocyte malignant transformation and ICC initiation remain poorly understood.


APPROACH AND RESULTS
We employed oncogene-driven and chemically induced ICC murine models, along with cellular models, to recapitulate the transformation of hepatocytes into ICC. Our findings demonstrate that mature hepatocytes undergo a significant hyperpolyploid state during ICC initiation. Hyperpolyploidy promotes aberrant cell division and chromosomal instability, accelerating hepatocyte transformation and ICC onset. Furthermore, we identified the chromatin remodeling factor Arid1a as a critical suppressor of hyperpolyploidy. Arid1a deficiency disrupts mitotic machinery at the centrosome, driving hyperpolyploidization and ICC tumorigenesis.


CONCLUSIONS
Hepatocytes can transform into ICC through a process involving hyperpolyploidizaiton. This study offers new insights into the pathogenesis of ICC, particularly in patients harboring frequent ARID1A mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb35330c9db09a4f3ded3d2167db23a4e30f36f2" target='_blank'>
              Arid1a deficiency promotes hepatocyte hyperpolyploidy and drives intrahepatic cholangiocarcinoma in mice.
              </a>
            </td>
          <td>
            Qi Bian, Shu Wang, Zimin Song, Fang Liu, Muqing Cao, Nan Yang, Jun Ying, Jia-Syuan Hu, Xinyuan Xiong, Huiqin Zhu, Jun Wu, Jie Yang, Xiaonan Wang, Shunli Shen, Xuxu Sun
          </td>
          <td>2025-06-09</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Cancer-associated fibroblasts (CAFs) greatly contribute to the growth, invasion, metastasis and drug resistance of neuroblastoma (NB). This study aimed to construct a CAF-related prognostic model and identify the immune status of patients with NB via single-cell RNA sequencing (scRNA-seq) and bulk RNA sequencing (RNA-seq). Methods ScRNA-seq data of NB acquired from the Gene Expression Omnibus (GEO) database were used to identify cellular subpopulations. Bulk gene expression data were downloaded from GEO, The Cancer Genome Atlas (TCGA) and ArrayExpress databases and the prognostic model was constructed using univariate Cox and Least Absolute Shrinkage and Selection Operator (LASSO) analyses. Differences in immune infiltration, therapeutic responses and signaling pathways between the high- and low-risk groups were investigated. Finally, immunohistochemistry was performed to evaluate the protein expressions and a nomogram based on the risk signature and clinical characteristics was constructed. Results ScRNA-seq data of eight NB samples were integrated to identify 253 marker genes for CAF. An eight-gene prognostic CAF-related signature was established based on the GEO data. The CAF model was strongly associated with immune infiltration, drug response and active signaling pathways in tumors. Univariate and multivariate Cox regression analyses verified that the CAF model was as an independent prognostic indicator, and a nomogram integrating the clinical signature and CAF-related risk signature was constructed for clinical prediction. Conclusions The CAF-related signature can effectively predict the prognosis of NB and provide new genomic evidence for anti-CAF immunotherapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03023-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0c7a00465eed83a4f94e4167616471e1046f423" target='_blank'>
              Integrated single-cell and bulk RNA sequencing analysis establishes a cancer associated fibroblast-related signature for predicting prognosis and therapeutic responses in neuroblastoma
              </a>
            </td>
          <td>
            Zhiyao Cao, Qi Wang, Yali Han, Jianwei Lin, Qi Wu, Chencheng Xu, Jingchun Lv, Lei Zhang, Hongxiang Gao, Dapeng Jiang
          </td>
          <td>2025-07-01</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The ASXL1 gene is one of the most frequently mutated genes in acute myeloid leukemia (AML). It is associated with signs of aggressiveness and adverse clinical outcomes. The aim of the current study was to analyze the genetic profile of ASXL1 gene mutations and its impact on the overall survival in AML patients from Pakistan. Thirty-eight well characterized AML patients were enrolled, and DNA sequencing of the ASXL1 was performed using the Illumina NextSeq500 next generation sequencing (NGS) system. Standard pipeline of bioinformatics tools was used to determine the mutational profile. The mutational profile of the enrolled AML patients was compared with that of 1000 Genomes project, and TCGA AML datasets. The analysis revealed 43 genetic variants in ASXL1 across the 38 AML patients (1.13 variant/patient). Eight rare variants were observed in exons 12, 13 of the ASXL1 gene. Notably, a recurrent rare nonsynonymous deleterious variant p.G1336S in exon 13 (NM_015338 transcript) was found in two patients (5.26%). The overall survival of the ASXL1+ (but TP53, FLT3, NPM1, EZH2, and WT1 negative) AML was shorter compared with the ASXL1- (p < 0.05). Further, the overall survival of current study ASXL1 + AML was found comparable with that of the TCGA AML. In conclusion, the non-silent mutations in ASXL1 were associated with lower survival in AML. Further studies with larger cohort are suggested for subsequent clinical implementation. Supplementary Information The online version contains supplementary material available at 10.1186/s12863-025-01324-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a03f43436989187938c1f8f140ca42098c22feb4" target='_blank'>
              Genetic profile of ASXL1 gene in risk assessment in acute myeloid leukemia
              </a>
            </td>
          <td>
            Salem Ahmed Bamusa, Wardah Qureshi, Atia Gohar, Muhammad Irfan, I. Khan, Muhammad Shakeel
          </td>
          <td>2025-07-12</td>
          <td>BMC Genomic Data</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Aim: Chemotherapy resistance, often linked to the development of resistance against genotoxic agents, is a major obstacle in cancer treatment. FoxM1, a transcription factor frequently overexpressed in malignancies such as breast cancer, is strongly associated with genotoxic therapy resistance. The aim of this study is to conduct a comparative analysis of the effects of thiostrepton (THIO), a FoxM1 inhibitor, on the DNA damage response in HUVEC cells (non-malignant) and MDA-MB-231 breast cancer cells (malignant) 
Materials and Methods: THIO's impact on cell viability were evaluated in both cell lines using the MTT assay. Oxidative DNA damage levels were measured with the 8-OHdG kit, and apoptosis was assessed using the Caspase 3 ELISA kit. The expression levels of DNA damage response genes (BRCA-1, DNAPKC, FOXM1, RAD51, MRE11 and XRCC1) were analyzed by RT-PCR. 
Results: MDA-MB-231 cells exhibited greater sensitivity to the cytotoxic effects of THIO than HUVEC cells. In HUVEC cells, THIO caused a significant increase in oxidative DNA damage, whereas no such effect was observed in MDA-MB-231 cell lines. Conversely, breast cancer cells showed a significant increase in Caspase 3 levels. RT-PCR results revealed a marked downregulation of DNA damage response genes, particularly BRCA-1, DNAPKC, MRE11, FOXM1, and XRCC1, in both cell types. 
Conclusion: THIO downregulates DNA damage response genes by inhibiting FoxM1 expression in both malignant and non-malignant cells. This effect suggests that THIO may enhance the sensitivity of breast cancer cells to therapy by disrupting DNA repair pathways, presenting potential for overcoming chemotherapy resistance. Further studies are required to validate these findings and explore THIO's therapeutic utility in breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0302ae4a15025cfb10555437aa0808dee744207" target='_blank'>
              FoxM1 inhibition by Thiostrepton downregulates DNA damage response genes, enhancing sensitivity of breast cancer cells to therapy
              </a>
            </td>
          <td>
            F. Demırtaş Korkmaz, Zekeriya Düzgün, Fadime Mutlu İçduygu
          </td>
          <td>2025-06-10</td>
          <td>Ege Tıp Dergisi</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ad49026046d30dd6909e4b9cd5b26991a13bef" target='_blank'>
              Tumor aneuploidy as a prognostic and predictive biomarker in immune checkpoint blockade.
              </a>
            </td>
          <td>
            Daniel Huang, L. F. Spurr, R. Weichselbaum, S. Pitroda
          </td>
          <td>2025-06-30</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Loss of the Y chromosome (LOY) in peripheral blood mononuclear cells (PBMCs) is the most common somatic alteration in men and is associated with higher mortality from epithelial cancers1–3. In tumours, epithelial LOY is also associated with poor survival4–7. This raises several fundamental questions, such as why LOY in PBMCs drives cancer mortality and whether there is a relationship between LOY in PBMCs, PBMC-derived immune cells and cancer cells (and, if so, what its consequences are). We sought to answer these questions through a comprehensive pan-cancer analysis of bulk and single-cell RNA sequencing data from 29 human tumour types, along with autochthonous and syngeneic mouse models. In human and mouse tumours, malignant epithelial cells had the highest LOY prevalence, yet LOY was also present in tumour stromal and immune cells, with LOY in malignant epithelial cells predicting LOY in benign cells. LOY also correlated between paired tumour and PBMC samples from patients. Among benign cells, LOY induced the strongest shift in CD4+ and CD8+ T cells, with both showing transcriptomic signatures of immunosuppression. Furthermore, the magnitude of LOY in epithelial cells, CD4+ T cells and CD8+ T cells independently predicts survival, with tumours exhibiting concurrent epithelial and T cell LOY having the worst outcomes. Here we establish a model that links LOY in immune cells to LOY in malignant cells, which may explain in part why LOY in PBMCs is associated with increased cancer mortality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/521330e6a0acc4c997107eda98c17326ccf4fe70" target='_blank'>
              Concurrent loss of the Y chromosome in cancer and T cells impacts outcome
              </a>
            </td>
          <td>
            Xingyu Chen, Yiling Shen, Suhyeon Choi, Hany A Abdel-Hafiz, Mukta Basu, Lena Hoelzen, Martina Tufano, Saravana Kumar Kailasam Mani, Maryam Ranjpour, Jiani Zhu, V. K. Ramanujan, E. Koltsova, Vinicius F Calsavara, S. Knott, Dan Theodorescu
          </td>
          <td>2025-06-04</td>
          <td>Nature</td>
          <td>2</td>
          <td>6</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a diverse collection of malignant tumors that originate in the bile ducts. Mitochondria, the energy converters in eukaryotic cells, contain circular mitochondrial DNA (mtDNA) which has a greater mutation rate than nuclear DNA. Heteroplasmic variations in mtDNA may suggest an increased risk of cancer-related mortality, serving as a potential prognostic marker. In this study, we investigated the mtDNA variations of five CCA cell lines, including KKU-023, KKU-055, KKU-100, KKU213A, and KKU-452 and compared them to the non-tumor cholangiocyte MMNK-1 cell line. We used Oxford Nanopore Technologies (ONT), a long-read sequencing technology capable of synthesizing the whole mitochondrial genome, which facilitates enhanced identification of complicated rearrangements in mitogenomics. The analysis revealed a high frequency of SNVs and INDELs, particularly in the D-loop, MT-RNR2, MT-CO1, MT-ND4, and MT-ND5 genes. Significant mutations were detected in all CCA cell lines, with particularly notable non-synonymous SNVs such as m.8462T > C in KKU-023, m.9493G > A in KKU-055, m.9172C > A in KKU-100, m.15024G > C in KKU-213A, m.12994G > A in KKU-452, and m.13406G > A in MMNK-1, which demonstrated high pathogenicity scores. The presence of these mutations suggests the potential for mitochondrial dysfunction and CCA progression. Analysis of mtDNA structural variants (SV) revealed significant variability among the cell lines. We identified 208 SVs in KKU-023, 185 SVs in KKU-055, 231 SVs in KKU-100, 69 SVs in KKU-213A, 172 SVs in KKU-452, and 217 SVs in MMNK-1. These SVs included deletions, duplications, and inversions, with the highest variability observed in KKU-100 and the lowest in KKU-213A. Our results underscore the diverse mtDNA mutation landscape in CCA cell lines, highlighting the potential impact of these mutations on mitochondrial function and CCA cell line progression. Future research is required to investigate the functional impacts of these variants, their interactions with nuclear DNA in CCA, and their potential as targets for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92494ba8290eb1c5fd7c0d865dc1a82ce3683d1" target='_blank'>
              Mitochondrial genomic alterations in cholangiocarcinoma cell lines.
              </a>
            </td>
          <td>
            Athitaya Faipan, Sirinya Sitthirak, Arporn Wangwiwatsin, N. Namwat, Poramate Klanrit, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Luke Boulter, Hasaya Dokduang, W. Loilome
          </td>
          <td>2025-06-09</td>
          <td>PloS one</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e86dbb7bd01d6e90b441b62dcc070c2bc321518" target='_blank'>
              High-fidelity long-read sequencing reveals a complex RCCX locus at the single-nucleotide level in Korean patients with congenital adrenal hyperplasia
              </a>
            </td>
          <td>
            S. Kim, J.-H. Yoon, D. Kim, S. Park, G.-H. Kim,, H.-W. Yoo, J.-H. Choi,, J. Kim, J. Kim
          </td>
          <td>2025-07-11</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chromatin forms specific intranuclear structures through physical interactions between specific genomic regions mediated by DNA-binding proteins and/or RNAs. Recent efforts have revealed that these genome organizations and dynamics are involved in various functions of genomic DNA, such as regulation of gene expression, DNA replication, cell division, and epigenetic memory, which are mechanisms underlying cell differentiation and disease development. The methods to detect physically interacting chromatin regions and reconstruct 3D genomic organization can be roughly categorized into four types: (i) microscopic observation of visualized intranuclear structures, (ii and iii) sequencing-based methods including proximity ligation-dependent/independent methods, and (iv) de novo prediction from genomic sequences and other omics data. Here, we review these techniques to detect physical interactions between genomic regions, highlighting their unique advantages and limitations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e72e6fcae694238f304b974323d0587b84637810" target='_blank'>
              Techniques to detect physical interactions between genomic regions.
              </a>
            </td>
          <td>
            Takayuki Hata, H. Fujii
          </td>
          <td>2025-06-26</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 What is the prevalence of mitotic-origin aneuploidies in human blastocysts using a PGT platform capable of identifying cell division origin of aneuploidy? A single-center experience



 Although uncommon, some embryos are diagnosed with aneuploidies of mitotic origin only. Diagnosing aneuploidies without determining their cellular origin risks discarding embryos with potential viability.



 Chromosomal abnormalities in human embryos are a significant factor contributing to reproductive failure. Next-generation sequencing (NGS) platforms are widely accepted for evaluating chromosomal abnormalities, as they are copy number-focused technologies. However, in some cases, biopsies reveal a mix of normal and abnormal cells, referred to as mosaic embryos. The developmental potential of these mosaic embryos remains unclear, although successful outcomes, including normal live births, have been reported. Moreover, novel PGT approaches, such as high-density single nucleotide polymorphism (SNP) arrays, offer the capability to predict the cell division origin of aneuploidies, moving beyond a sole focus on chromosome copy numbers.



 This retrospective study analyzed 310 consecutive PGT-A biopsies obtained from 72 patients across 91 treatment cycles conducted between May 2023 and December 2024 at a private fertility clinic. Both autologous and donor cycles were included, and all blastocysts were evaluated using trophectoderm (TE) biopsy and DNA samples from the biological parents.



 Trophectoderm biopsies were obtained, with 5–10 cells collected for each sample. Parental DNA samples were also required to serve as reference material for the tested embryos. Trophectoderm biopsies and parental DNA were processed using SNP arrays to identify the cell division origin of aneuploidy, integrating both genotyping and copy-number data. Aneuploidies were classified by origin as maternal meiotic (from the oocyte), paternal meiotic (from the sperm), or mitotic (embryonic, arising post-zygotic division).



 Among the total embryos studied (n = 310), 44.8% (139/310) were identified as aneuploid. Of these, 34.8% (108/310) exhibited at least one maternal meiotic aneuploidy, with an average maternal age of 36.3 ± 4.4 years, consistent with the well-established correlation between advancing maternal age and chromosomal anomalies. Additionally, 7.7% (24/310) of the aneuploid embryos displayed paternal meiotic errors, while 8.7% (27/310) presented mitotic errors originating post-fertilization. Notably, 5.16% (16/310) of the embryos displayed exclusively mitotic-origin aneuploidies, underscoring the importance of mitotic anomalies in early embryonic development. Among the 139 abnormal embryos, 81.3% (113/139) exhibited whole chromosome aneuploidies, consistent with meiotic segregation errors involving entire chromosomes. Conversely, 13.67% (19/139) of the abnormal embryos exhibited segmental aneuploidies, involving duplications or deletions of specific chromosomal regions. A small subset of embryos, accounting for 5.04% (7/139), displayed karyotypes that included both whole chromosome aneuploidies and segmental abnormalities, reflecting the complexity of chromosomal errors in human embryos.



 This is a retrospective study with a relatively small number of embryos analyzed. Expanding the cohort of blastocysts and evaluating the clinical implications of aneuploidy origin prediction are necessary. Additionally, further research is needed to assess the reproductive potential of embryos with segmental chromosome aneuploidies.



 Identifying the cell division origin of aneuploidies provides deeper insights into embryonic chromosomal abnormalities and their developmental potential. This approach could improve clinical decision-making in assisted reproduction, enhancing embryo selection strategies and informing future studies on the implications of mitotic and segmental aneuploidies in human reproduction.



 No
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/839a948ebef49cd1b00902c78815394f99d075d1" target='_blank'>
              P-242 From Cell Division to Clinical Decisions: Unveiling the True Prevalence of Mitotic-Origin Aneuploidies in Preimplantation Embryos
              </a>
            </td>
          <td>
            M. J. Gomez Cuesta, D. Marin Chavarria, M. I. Gutierrez Estrada, M. Garcia Galicia, G. D. Cortes Ruiz, M. Alonso de Mendieta, C. Kandel Varsano, C. Lopez Valencia, E. Cervantes Bravo
          </td>
          <td>2025-06-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f8bfbccb54cc051188169f1f74b334c11e829f4" target='_blank'>
              Unravelling the role of epigenetic regulators during embryonic development of Rhipicephalus microplus
              </a>
            </td>
          <td>
            Anderson de Mendonça Amarante, Daniel Martins de Oliveira, Marcos Paulo Nicolich Camargo de Souza, Manoel Fonseca-Oliveira, Antonio Galina, Serena Rosignoli, Angélica Arcanjo, Bruno Moraes, A. Paiardini, Dante Rotili, Juan Diego de Paula Li Yasumura, S. Henaut-Jacobs, T. M. Venancio, Marcelle Uhl, R. Nunes-da-Fonseca, L. F. Parizi, I. S. V. Vaz Júnior, C. Mermelstein, L. Tirloni, C. Logullo, M. Fantappié
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cancer initiation occurs when a cell acquires and accumulates mutations in genes involved in the regulation of cell processes: each cell division throughout a person’s life introduces novel mutations in the cells’ DNA and under normal circumstances, the body is primed to prevent those from leading to cancer. Occasionally, a subset of those mutations escapes those safeguards and might eventually result in the emergence of the disease. To understand the dynamics of accumulation of somatic mutations, we have performed longitudinal whole genome sequencing of DNA obtained from whole blood from healthy individuals and cancer patients using Oxford Nanopore Technologies’ Long Read Sequencing. Here we show that the number of somatic single nucleotide variants detected increases with their age and that for specific mutational processes, changes can be detected within months. We computed aggregated metrics for unique participants at each timepoint across types of variants (based on single based substitution molecular signatures) and identified patterns of change both over an individual’s lifespan (age) and over the sampling period (months). This study showcases the suitability of long read sequencing of blood DNA for detecting coarse-grained differences over time and enable future development of “state of the system” personalized prevention programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c95c0c1b8ce2a6d46369d034aca04dad0606d256" target='_blank'>
              Monitoring the rate and variability of somatic genomic alterations using long-read sequencing
              </a>
            </td>
          <td>
            Xingyao Chen, Hagai Ligumsky, Charlie Ambrose, Denisse Sibrian, Brian Tran, Daoud Arif, Olga Castellanos, Darren Kessner, Hanyi Luo, Mukta Ubale, Abigail Coleman, Vaidhyanathan Mahaganapathy, Thomas J Jönsson, R. Basho, Jerry S H Lee, Naim Matasci, David B. Agus
          </td>
          <td>2025-06-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Breast cancer is characterized as highly heterogenous and is a representative model to understand how molecular features of tumor biology determine therapeutic strategy. LINC01133 exhibits opposing expressing patterns across different breast cancer subtypes, yet its roles and mechanisms in ER+ breast cancer remain a loaded question. Methods The expression of LINC01133 was initially assessed utilizing a public dataset TCGA and subsequently validated within clinical samples through RT-qPCR and in situ hybridization (ISH). To determine the role of LINC01133, various assays, including colony formation, Transwell, 5-ethynyl-2′-deoxyuridine (EdU) labeling, and mouse xenograft experiments, were performed. Additionally, RNA immunoprecipitation (RIP), RNA pull-down, mass spectrometry (MS), and RNA stability assays were conducted to elucidate its mechanisms. Results LINC01133 was dramatically downregulated in ER+ breast cancer, which results in unfavorable prognosis. Functionally, LINC01133 inhibited migration and invasion in vitro and metastasis in vivo of ER+ breast cancer cells. Mechanistically, LINC01133 can directly interact with IGF2BP2 protein promoting its ubiquitination and degradation. The downregulation of LINC01133 was mediated by m6A modification, catalyzed by METTL3 and recognized by YTHDF2, causing half-life reduction and accelerated degradation of LINC01133. Conclusion Our findings revealed the downregulation of LINC01133 in ER+ breast cancer and provided novel insight to the role of METTL3/YTHDF2/LINC01133/IGF2BP2 axis in ER+ breast cancer, which might offer a novel perspective in the design and development of novel anticancer drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd1c654f899d20ee944e199054f4ac06718cd700" target='_blank'>
              The m6A modification of LINC01133 suppresses ER+ breast cancer progression by modulating IGF2BP2 protein stability via a ubiquitination-dependent mechanism
              </a>
            </td>
          <td>
            Mai-dong Li, Ben-jie Shan, Lei Wang, Shuang Gao, Li Hao, Hai-yang Yu, Yue-yin Pan
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35fd329abfeae45d934694f31530eb9db169304e" target='_blank'>
              A mouse model of PTEN Hamartoma Tumour Syndrome reveals that loss of the nuclear function of PTEN drives macrocephaly, lymphoid overgrowth, and late-onset cancer
              </a>
            </td>
          <td>
            Priyanka Tibarewal, V. Rathbone, Sarah E Conduit, G. Classen, Fiona M Black, Mohammad Amin Danesh, Georgia Constantinou, Zeinab Asgarian, Koujiro Tohyama, N. Kriplani, Virginia Alvarez Garcia, Elizabeth Foxall, Djenat Belarbi, M. Leverve, Wayne Pearce, Mahreen Adil, Zofia Varyova, Lucia Conde, Adriana Alves, G. Masson, Roger L. Williams, Adrienne M Flanagan, Javier Herrero, Katerina Stroud, M. Tischkowitz, K. Lachlan, Cheryl L Scudamore, Mark G H Scott, Nicholas R. Leslie, N. Kessaris, B. Vanhaesebroeck
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3115319084e3008e2ffcee186c2141b84b2e97e1" target='_blank'>
              Assembly and delivery of large DNA via chromosome elimination in yeast.
              </a>
            </td>
          <td>
            Yuan Ma, Yao Xiong, Jiayu Xu, Hui Xu, Zong-Heng Fu, Guang-Rong Zhao, Yi Wu, Yingjin Yuan
          </td>
          <td>2025-07-10</td>
          <td>Nature protocols</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="AlkB homolog (ALKBH) belongs to 2-oxoglutarate (2OG) and ferrous iron-dependent oxygenases, which can catalyze the demethylation of DNA/RNA and play an important role in human diseases represented by cancer. Although ALKBH has been extensively studied, the biological explanation for its specific recognition of DNA/RNA and its mechanism of action in cancer remains unclear. This review explores the molecular mechanism of ALKBH protein binding to DNA/RNA through structural fitting and ChIP-Seq analysis and summarizes the structural biology factors that lead to its specific substrate recognition. Furthermore, the expression of ALKBH protein across different tissues is systematically compiled, with a tissue-specific expression map generated for 15 organs and 50 tissues. These maps provide valuable insights into the protein’s function in various physiological contexts. Additionally, this review systematically summarizes the regulatory role of the ALKBH protein in the occurrence and development of cancer. It highlights the protein’s involvement in key signaling pathways, including Wnt, AKT, AMPK, NF-κB, Hippo, and Notch, revealing its potential for application in early diagnosis and targeted cancer therapy. Overall, this review comprehensively expounds the biological functions of the ALKBH protein and its important role in cancer, which has important theoretical value and application prospects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73c3777d66ca6fa0fb7fbc4ccae8e6e2833de338" target='_blank'>
              Biological interpretation of DNA/RNA modification by ALKBH proteins and their role in human cancer
              </a>
            </td>
          <td>
            Siqi Yang, Jixiang Xing, Yancheng Song, Jiafu Cui, Lingling Bao, Qilemuge Xi, Yongchun Zuo
          </td>
          <td>2025-06-06</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The distribution of m6A across various RNA isoforms and its heterogeneity within single cells are still not well understood. Here, we develop m6A-isoSC-seq, which employs both Oxford Nanopore long-read and Illumina short-read sequencing on the same 10x Genomics single-cell cDNA library with APOBEC1-YTH induced C-to-U mutations near m6A sites. Through m6A-isoSC-seq on a pooled sample of three cell line origins, we unveil a profound degree of m6A heterogeneity at both the isoform and single-cell levels. Through comparisons across single cells, we identify widespread specific m6A methylation on certain RNA isoforms, usually those misprocessed RNA isoforms. Compared to the coding isoforms of the same genes, the expression of highly methylated misprocessed RNA isoforms is more sensitive to METTL3 depletion. These misprocessed RNAs tend to have excessive m6A sites in coding regions, which are targets of CDS-m6A decay (CMD). This study offers undocumented insights into the role of m6A in RNA surveillance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3d1afb6146144c07522f09b53199013967becd6" target='_blank'>
              Isoform characterization of m6A in single cells identifies its role in RNA surveillance
              </a>
            </td>
          <td>
            Zhijun Ren, Jialiang He, Xiang Huang, Yan Gao, Chuanchuan Wei, Zehong Wu, Wenbing Guo, Feng Wang, Qingquan Zhao, Xiang Sun, Jie Zhang, Nan Cao, Lan Lin, Jinkai Wang, Yixian Cun
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 e14570


 Background:
 Immune checkpoint blockades (ICBs) showed little efficacy in pediatric solid tumors. The reconfiguration of gene regulation in cancers is a major factor in tumor immune microenvironment (TiME) remodeling and ICB resistance. Hypothesis: A subset of pediatric tumors may exhibit an epigenetically altered TiME, potentially responsible for ICB resistance. Objective: classify the TiME of pediatric extracranial solid tumors and describe the gene regulatory networks (GRNs) of immune landscapes.
 Methods:
 We studied bulk tumor gene expression and DNA-methylation from 184 pediatric extracranial solid tumors. Similarity network fusion (SNFtool) individualized TiME phenotypes by dual-omic clustering of immune genes. The clusters were compared by differential analysis for gene expression and methylation. We studied the relationship of enhancer methylation to gene expression (ELMER package) to infer methylation reprogramming. An enrichment study of regulatory binding regions (ReMapEnrich) identified the master regulators that define the GRNs unique to each phenotype.
 Results:
 SNF clustering identified 3 phenotypes with 52 (28%), 83 (45%), 49 (27%) samples in clusters (cl) 1, 2 and 3, respectively. Cl1 exhibited low expression of immune genes (“cold” phenotype), cl2 overexpressed immune genes (“hot” phenotype), and cl3 featured global hypermethylation (epigenetically “altered” phenotype). Different tumor types were present across all phenotypes, but Ewing sarcoma was more frequent in altered, osteosarcoma and neuroblastoma in hot, and Wilms tumor in cold phenotype (
 p
 <0.05). Both hot and altered phenotypes overexpressed immune checkpoints (
 CD274
 ,
 PDCD1
 ), T-cell activator chemokines (
 CXCL9
 ,
 CXCL10
 ) and pro-inflammatory pathways, central to ICB sensitivity. However, only hot tumors overexpressed genes and pathways essential for antigen-presenting machinery and immune recognition. Methylation regulation was responsible for derepressing MHC-II genes and their regulators in hot tumors. In contrast, gene silencing in altered phenotype relied on hypermethylation/downregulation of the CIITA cofactor domain, the master control of MHC-II genes, and other immune-specific master regulators crucial for anti-tumor immunity and ICB sensitivity. The regulatory landscape of immune gene repression in the altered phenotype involved binding regions specific to the MYC/MAX network and the polycomb-group protein PCR2 (
 i.e.,
 EZH2 and SUZ12).
 Conclusions:
 We demonstrated that GRN reconfiguration participates in TiME reshaping in pediatric extracranial solid tumors. A subset of tumors with epigenetically altered immune phenotype has an immune recognition capacity repressed by MYC/MAX and PCR2 complexes. Future studies should investigate specific inhibitors to reprogram the GRN to foster immune recognition and ICB sensitivity.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b6a52d83f8f6dace4bda8bd4ea08fd3f518537c" target='_blank'>
              Dual-omic analysis to reveal gene regulatory network of immune landscape in childhood solid tumors and implications for immunotherapy.
              </a>
            </td>
          <td>
            Stéphanie Bianco, Anas Belaktib, Virgile Raufaste-cazavieille, Charles Joly-Beauparlant, Mona Patoughi, Thibault Mallevaey, Lara Herrmann, Sylvie Langlois, Thomas Sontag, Alex Richard-St-Hilaire, Noel Raynal, Vincent-Philippe Lavallée, Sonia Cellot, T. Tran, D. Sinnett, Arnaud Droit, Raoul Santiago
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="PURPOSE
Telomere maintenance mechanisms including MYCN amplification (MYCN-A) (via upregulated telomerase reverse transcriptase [TERT] expression), TERT rearrangements (TERT-RA), and ATRX mutations confer neoplastic immortality in neuroblastoma (NB) cells. Clinical characterization of patients with NB harboring activating somatic TERT promoter point mutations (TERT-PM) in NB may improve stratification.


METHODS
To identify TERT-PM and TERT-RA, tumors were profiled by the Memorial Sloan Kettering Cancer Center-Integrated Mutation Profiling of Actionable Cancer Targets targeted next-generation sequencing platform and whole-genome sequencing, respectively. Associated clinical outcomes were studied.


RESULTS
TERT-PM and TERT-RA were detected in 17 of 603 (2.8%) and 31of 168 (18.4%) tumors from individual patients, respectively. The median age at diagnosis was 32 (range, 15-79) and 48 (range, 3-168) months for TERT-PM and TERT-RA, respectively. TERT-PM were located at canonical hotspots (C228T [16/17] and C250T [1/17]) and were concurrent with MYCN-A in 9 of 17 (53%). By contrast, MYCN-A occurred in 1 of 31 TERT-RA tumors. Most patients with TERT-PM had stage M (94%) and high-risk NB (HR-NB) (81%) at diagnosis. Twenty-eight patients with TERT-RA had HR-NB (90%). After risk appropriate therapy, complete response was achieved in 7 of 17 (41%) and 17 of 31 (55%) patients with TERT-PM and TERT-RA, respectively. The median progression-free (PFS) and overall survival (OS) were 9.7 ± 1.2 and 16.5 ± 2 months for patients with TERT-PM. Corresponding PFS/OS for patients with TERT-RA were 20 ± 6.4/56.8 ± 8.6 months (P = .015 for OS). Excluding MYCN-A status produced no significant survival difference (P > .1) between the two groups. CNS relapse occurred in 11 of 31 (35%) patients with TERT-RA versus 1 of 17 (6%) patients with TERT-PM.


CONCLUSION
TERT-PM is rarer than TERT-RA but both are associated with HR-NB and poor prognosis. MYCN-A frequently co-occurs with TERT-PM and might represent an ultra-high-risk subset of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/556a0902c3148a7408804dc2c0f9318f277250b6" target='_blank'>
              Clinical Significance of TERT Promoter Mutations in Neuroblastoma.
              </a>
            </td>
          <td>
            Natasha Persaud, G. Gundem, B. Kushner, Marc Ladanyi, Neerav Shukla, S. Modak
          </td>
          <td>2025-07-01</td>
          <td>JCO precision oncology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background Melanoma brain metastases (MBM) have a unique molecular profile compared to extracranial metastases (ECM). Description of the biological features and clinical outcomes of MBM will facilitate the design of rational therapies. Methods We examined the mutational landscape and gene expression profiles of MBM (74 patients) and ECM (34 patients) in paired patient samples from a previously published dataset with whole-exome sequencing (WES) and RNA sequencing (RNAseq) data from MD Anderson Cancer Center (MDACC). We also present findings from MBM from a new cohort of 14 patients from Duke University to strengthen investigation of somatic mutations and gene expression profiles. Gene Set Enrichment Analysis (GSEA) was used to compare paired MBM versus lymph node (LN) metastases and skin metastases. Relative immune cell abundance was inferred using deconvolution methods. Survival outcomes from craniotomy and associations with biological features, BRAF mutation status, and PTEN expression were assessed. Results GSEA found that autophagy signaling pathways are enriched in MBM versus LN and skin metastases. BRAF was the most frequently mutated clinically relevant gene in MBM and ECM, with NRAS and PTEN also frequently altered in MBM. The most strongly upregulated genes in autophagy pathways were glial fibrillary acidic protein (GFAP) and hemoglobin beta (HBB). An increased proportion of immune-suppressive M2 compared to tumor-suppressive M1 macrophages in MBM and ECM was identified. There was not sufficient evidence for an association between BRAF V600 mutation status or expression and overall survival (OS) from craniotomy. Conclusions The mutational landscape and gene expression of MBM from the Duke cohort resembled those previously reported in the MDACC cohort. Upregulation of autophagy pathways was observed in patient-matched MBM versus LN and skin metastases due to upregulation of two genes, GFAP and HBB. In MBM, higher M2:M1 ratio may contribute to a therapeutically relevant immune-suppressive tumor microenvironment (TME). Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02035-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd2acbbd5230dc401bd3ec3c002ad4d3ee572431" target='_blank'>
              A comprehensive, multi-center, immunogenomic analysis of melanoma brain metastases
              </a>
            </td>
          <td>
            L. B. Kennedy, Amanda E. D. Van Swearingen, Marissa R Lee, Layne W. Rogers, A. Sibley, Jeff Sheng, Dadong Zhang, Xiaodi Qin, E.S. Lipp, Swaminathan Kumar, Aron Y Joon, Pixu Shi, Michael A. Davies, K. Owzar, Carey K. Anders, April K S Salama
          </td>
          <td>2025-06-02</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Chromatin structure in eukaryotes is organized into functional domains through protein-DNA complexes. The cis-acting DNA elements are attached to the nuclear matrix, known as scaffold/matrix attachment regions (S/MARs). The associated protein partners known as matrix-associated region binding proteins (MARBPs). The coordinated interactions between MARBP and MARs to the nuclear scaffold act as an anchor for chromatin attachment and influences the regulation of gene expression. MARBPs modulate local epigenetic landscape of chromatin through the epigenetic modifiers. This function is executed by participating in the posttranslational modifications (PTMs) of DNA and histones. Such epigenetic changes are governed by crosstalk between long noncoding RNAs (lncRNAs) and associated MARBPs. Thus, dysregulation of either MARBPs or lncRNAs may alter gene expression potentially contributing to various disease manifestations. In this review, we elaborate on regulatory crosstalk between lncRNAs and MARBPs, its implication in human diseases, and possible therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bf8ecc6737643fa1d086cc3eb51315cf7f83f47" target='_blank'>
              MARBP-lncRNA Complexes Alter Gene Function Through Modulation of Epigenetic Landscape.
              </a>
            </td>
          <td>
            Nilanjana Maji, Anindya Dutta, Animesh Anand, S. Chatterjee, Samit Chattopadhyay
          </td>
          <td>2025-07-03</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 e15149


 Background:
 Cells possess a complex DNA damage response (DDR) system to prevent detrimental mutations from accumulating. The genes encoding various DDR components are frequently mutated in cancer cells and provide potential sites for therapeutic intervention. DNA damaging agents, with one exception, are repaired by the global genome nucleotide excision repair (GG-NER) pathway. Our novel class of drugs, Illudins, are the exception as Illudin-induced DNA damage is not recognized by GG-NER and can only be repaired by the transcription-coupled nucleotide excision repair (TC-NER) pathway. Studies utilizing isogeneic cell lines completely deficient in specific TC-NER proteins (e.g. homozygous ERCC6 knockout) indicate these cells are extremely sensitive to Illudins (up to 30-fold). In contrast, cells deficient in GG-NER activity are not sensitive to our drugs. Moreover, non-replicating or normal cells are minimally affected by Illudins, as evidenced by the lack of systemic toxicity in humans treated with one of our drugs. In TC-NER deficient cancer cells, however, our drug ITX-0121 blocks the TC-NER polymerase complex, which disengages from DNA, and initiates the irreversible process of apoptosis or cellular suicide. Many cancers exhibit deficiencies in DDR mechanisms and cannot properly repair DNA damage.
 Methods:
 Analysis of the GENIE Database indicates that the overall incidence of TC-NER deficient cancers in solid tumors is ~10%, and includes all major histological classifications (breast, ovarian, endometrial, prostate, renal, bladder, bone, colorectal, liver, lung, skin, pancreatic, thyroid, etc.) or ~175,000 new patients annually. These cancers, however, possess monoallelic mutations, as opposed to biallelic, and thus have some residual TC-NER functionality. To determine the relative sensitivity of cancers with monoallelic alterations in TC-NER genes, we generated colorectal HCT-116 cells with a monoallelic deletion in a single TC-NER gene. For example, we are generating daughter cell lines with small monoallelic deletions in ERCC2, ERCC3, ERCC4, ERCC6, and ELOF1 genes.
 Results:
 We are comparing sensitivity of these daughter lines to our lead drug ITX-0121, as well as to other DNA damaging agents such as cisplatin. Preliminary results indicate a monoallelic alteration in a TC-NER gene can confer > 6-fold sensitivity to ITX-0121, which is clinically relevant based on PK results. Based on these PK results, we believe the maximum tolerated dose (MTD) of ITX-0121 in patients to be > 1.2 mg/kg; yet only 0.4 mg/kg will be sufficient for clinical efficacy against cancers that are phenotypically TC-NER deficient.
 Conclusions:
 Thus, despite the administration of a cytotoxic agent, TC-NER deficient cancer patients can be treated at a non-toxic dose; if toxicity occurs, it should be limited to transient thrombocytopenia, as indicated by prior studies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15a4f45d8c212b489e848805a2ff3ec30d91ef9b" target='_blank'>
              ITX-0121: A therapeutic strategy for transcription-coupled repair-deficient cancers.
              </a>
            </td>
          <td>
            M. Kelner, Raymond T. Suhandynata, Venkata R. Kotamraju
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="CHROMR is a primate-specific long noncoding RNA with emerging roles in homeostasis and pathophysiology. Elevated blood levels of CHROMR have been observed in patients with cardiovascular disease and several cancers, where it is correlated with poor clinical outcomes. Like many lncRNAs, CHROMR accumulates in both the nucleus and the cytoplasm, and it assumes distinct functions in each of these cellular compartments. In the nucleus, CHROMR sequesters a transcriptional repressor complex to activate interferon-stimulated gene expression and antiviral immunity. In the cytoplasm, CHROMR competitively inhibits microRNAs involved in cholesterol efflux and cell cycle regulation, thereby impacting gene pathways involved in reverse cholesterol transport, HDL biogenesis, and tumor growth. In this review, we detail the multifaceted functions of CHROMR in cholesterol metabolism, innate immunity, and cancer progression. We also explore the potential molecular mechanisms that govern its expression and dynamic subcellular localization, which may be key to its functional versatility. Advancing our understanding of the regulatory networks and cellular environments that shape CHROMR activity will be critical for assessing its promise as a therapeutic target and diagnostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e434fd00ac8710c07c05ad122a8af40ef9af54ab" target='_blank'>
              The Multifaceted Roles of CHROMR in Innate Immunity, Cancer, and Cholesterol Homeostasis
              </a>
            </td>
          <td>
            Emma R. Blaustein, Coen van Solingen
          </td>
          <td>2025-06-01</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6483fac3f504fbebfb3a60d1e21f332ce1ac0267" target='_blank'>
              Epigenetic reprogramming of mast and cancer cells modifies tumor-promoting cytokine networks.
              </a>
            </td>
          <td>
            Alejandro Schcolnik-Cabrera, Marisol Ramírez-Yautentzi, Rodolfo Soria-Castro, R. Chacón-Salinas, Rocío Morales-Bárcenas, G. Domínguez-Gómez, A. González-Fierro, Alfonso Dueñas-González, Alma D Chávez-Blanco
          </td>
          <td>2025-07-24</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha (PIK3CA) plays a critical role in cell growth and survival. PIK3CA somatic mutations are linked to the PIK3CA-related overgrowth spectrum (PROS) syndrome. Cells from those patients appeared to be associated with a moderate but significant radiosensitivity. Mutations or amplifications in this gene are common in breast, colorectal, and lung cancers. Alterations in the PIK3CA gene, including amplification and mutation, are common in cancer, but their influence on radiotherapy is not yet fully understood. This report reveals a potential association between PIK3CA gene modifications and radiosensitivity (p < 0.05) deducted from 8 established colorectal cancer cell lines (HT-29, DLD-1, HCT-116, SW480, HCT-15, Colo-320, and LoVo). Meanwhile gene amplification (> 2) seems to be linked to increased radiation sensitivity, mutations appear to be associated with increased radioresistance in colorectal cancer cells. Leveraging this relationship, PIK3CA amplification and mutations could act as biomarker to pinpoint patients who might benefit from more personalized radiotherapy regimens. However, this association is preliminary and hypothesis-generating in view of the limited number of cases. Further studies are needed to confirm this conclusion. By uncovering the distinct mechanistic effects of these PIK3CA alterations on radiosensitivity phenotype in both normal and cancerous cells, researchers can lay the groundwork for tailored radiotherapy strategies in colorectal cancer. This insight could enhance treatment effectiveness while reducing side effects, ultimately leading to improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3d2f05a3aac70c0b974b05ec72c5f8dff7113c5" target='_blank'>
              Effects of PIK3CA Gene Modifications on Radiosensitivity in Colorectal Cancer Cells.
              </a>
            </td>
          <td>
            Ghazi Alsbeih, K. Al-Hadyan
          </td>
          <td>2025-06-01</td>
          <td>Asian Pacific journal of cancer prevention : APJCP</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chromosome segregation is essential for cellular proliferation. Unlike eukaryotes, bacteria lack cytoskeleton-based machinery to segregate their chromosomal DNA (nucleoid). The bacterial ParABS system segregates the duplicated chromosomal regions near the origin of replication. However, this function does not explain how bacterial cells partition the rest (bulk) of the chromosomal material. Furthermore, some bacteria, including Escherichia coli, lack a ParABS system. Yet, E. coli faithfully segregates nucleoids across various growth rates. Here, we provide theoretical and experimental evidence that polysome production during chromosomal gene expression helps compact, split, segregate, and position nucleoids in E. coli through nonequilibrium dynamics that depend on polysome synthesis, degradation (through mRNA decay), and exclusion from the DNA meshwork. These dynamics inherently couple chromosome segregation to biomass growth across nutritional conditions. Halting chromosomal gene expression and thus polysome production immediately stops sister nucleoid migration, while ensuing polysome depletion gradually reverses nucleoid segregation. Redirecting gene expression away from the chromosome and toward plasmids causes ectopic polysome accumulations that are sufficient to drive aberrant nucleoid dynamics. Cell width enlargement experiments suggest that limiting the exchange of polysomes across DNA-free regions ensures nucleoid segregation along the cell length. Our findings suggest a self-organizing mechanism for coupling nucleoid compaction and segregation to cell growth without the apparent requirement of regulatory molecules.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b38632c47e3d185d80d1fdc81e020f23db296ddb" target='_blank'>
              Nonequilibrium polysome dynamics promote chromosome segregation and its coupling to cell growth in Escherichia coli
              </a>
            </td>
          <td>
            Alexandros Papagiannakis, Q. Yu, Sander K. Govers, Wei-Hsiang Lin, N. Wingreen, C. Jacobs-Wagner
          </td>
          <td>2025-06-24</td>
          <td>eLife</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Background Resveratrol (RSV) is one of the most studied and used biomolecules, for which many pharmacological effects targeting multiple tissues have been described. However, a common underlying mechanism driving its full pharmacological activity has not been described in detail. G-quadruplexes (G4s) are non-canonical nucleic acid structures found in regulatory genomic locations and involved in controlling gene transcription, telomere maintenance, or genome stability, among others. This study provides a genome-wide characterization of RSV G4-binding properties, explaining its multi-target traits. Methods Immunofluorescence assays using a nucleolar and a G4-specific antibody were used to characterize RSV cellular effects on the nucleolus and G4 stabilization. DNA damage and cell cycle analyses were performed via western blot and flow cytometry. Breaks lLbeling In Situ and Sequencing (BLISS) was used to map double strand breaks (DSB) in response to treatment, and identify G4s targeted by RSV. mRNA sequencing was used to identify changes at the transcriptional level upon treatment and relate them to a direct targeting of G4s. Biophysical assays (circular dichroism, ultraviolet–visible [UV–Vis] titration, differential scanning calorimetry, and nuclear magnetic resonance) were used to characterize RSV–G4 interactions. Lastly, luciferase-based transcription assays were performed to confirm RSV–G4 interaction in vitro and its direct influence on gene expression. Results In a cellular context, RSV treatment showed classic G4-ligand effects, such as nucleolar disassembly, inhibition of RNA polymerase I, DNA damage, and cell cycle arrest. RSV was shown to stabilize cellular G4s, which accumulated around double strand breaks in the promoters of differentially expressed genes. Upon treatment, G4 stabilization triggered DNA damage and controlled gene expression. The interaction between RSV and target G4s was confirmed in vitro by biophysical assays and through luciferase-based transcription assays. Conclusions A G4-dependent mode of action was demonstrated as the main mechanism underlying RSV pleiotropic effects, along with the identification of target genes and G4s. This in-depth analysis of the mode of action of RSV will be helpful to improve its therapeutic potential in a wide variety of health scenarios. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s11658-025-00747-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309b9a5bcda358645827e337bc4418daea949d2f" target='_blank'>
              In-depth analysis of the mode of action of resveratrol: genome-wide characterization of G-quadruplex binding properties
              </a>
            </td>
          <td>
            Ana Soriano-Lerma, Victoria Sánchez-Martín, Javier Murciano-Calles, Matilde Ortiz-González, Maria J. Tello-Lopez, Virginia Pérez-Carrasco, Ángel Linde-Rodríguez, Inmaculada Ramírez-Macías, Irene Gómez-Pìnto, Inmaculada López-Aliaga, M. Soriano, Jose A. Garcia Salcedo
          </td>
          <td>2025-06-20</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Triple-negative breast cancer (TNBC) is a highly aggressive disease that typically exhibits faster growth and higher metastatic spread than other subtypes. There is a paucity of treatments available for TNBC, further contributing to poor patient prognosis. The rapid growth of TNBC cells makes mitosis a compelling target; indeed, a current mainstay of treatment is Paclitaxel (Ptx), a taxane that stabilizes microtubules during mitotic progression. Despite being initially effective, acquired resistance to Ptx is common. Moreover, additional antimitotic therapies can be similarly rendered ineffective due to resistance, toxicity, or a lack of efficacy, underscoring the need for new therapeutic approaches. One understudied class of potential targets is the motor protein dynein and its associated regulatory proteins. Of these, LIS1 was highly ranked in several essentiality screens for breast cancer cell growth. LIS1 regulates mitotic spindle orientation and chromosome movement during neurodevelopment, yet little is known regarding its function in breast cancer. We found that suppressing LIS1 expression in TNBC cells, but not in non-transformed cells, reduces cell number. Live cell imaging indicates that reducing LIS1 alters cell cycle dynamics, increasing mitotic duration due to an accumulation of cells blocked in the G2/M phase. Such accumulation is commonly caused by activation of the spindle assembly checkpoint (SAC) or increased levels of DNA damage. Indeed, LIS1-silenced cells exhibit higher levels of Cyclin B1, a marker of SAC activation. Moreover, loss of LIS1 increases DNA double-strand breaks, highlighting a novel role for LIS1 in TNBC. Lastly, we found that the loss of LIS1 substantially enhances these defects in cells that are resistant to Ptx and sensitizes both parental and Ptx-resistant TNBC cells to Ptx treatment. As such, LIS1 represents a novel anti-mitotic target for treating TNBC, particularly in the context of paclitaxel resistance.
 Citation Format: Parth Majmudar, Ruth Keri. The dynein regulator LIS1 maintains cell cycle progression and DNA integrity in TNBC, and can be targeted to improve paclitaxel response [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-04-05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bd22f2742958ae854793738019d19a69bd8fb92" target='_blank'>
              Abstract P2-04-05: The dynein regulator LIS1 maintains cell cycle progression and DNA integrity in TNBC, and can be targeted to improve paclitaxel response
              </a>
            </td>
          <td>
            P. Majmudar, Ruth A Keri
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Mitotic chromosomes are considered to be universally folded as loop arrays across species and cell types. However, some studies suggest that features of mitotic chromosomes might be cell type or species specific. We previously reported that CTCF binding in human differentiated cell lines is lost in mitosis, whereas mitotic mouse embryonic stem cells (mESC) display prominent binding at a subset of CTCF sites. Here, we perform footprint ATAC-seq analyses of mESCs and somatic mouse and human cells confirming these findings. We then investigate roles of mitotically bookmarked CTCF in prometaphase chromosome organization by Hi-C. We do not find any remaining interphase structures such as TADs or loops at bookmarked CTCF sites in mESCs. This suggests that mitotic loop extruders condensin I and II are not blocked by CTCF, and thus that maintained CTCF binding does not alter mitotic chromosome folding. Lastly, we compare mitotic Hi-C data generated in this study in mouse with public data in human and chicken. We do not find any cell type-specific differences; however, we find a difference between species. The average genomic size of mitotic loops is smaller in chicken (200-300 kb), compared to human (400-600 kb) and especially mouse (1-1.5 mb). Interestingly, we find that this difference is correlated with the genomic length of q-arms in these species, a finding we confirm by microscopy measurements of chromosome compaction. This suggests that the dimensions of mitotic chromosomes can be modulated through control of loop size by condensins to facilitate species-appropriate shortening of chromosome arms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7587b4758e2a0d2dc6b16f50b764f5c00ce6820f" target='_blank'>
              Mitotic chromosomes harbor cell type and species-specific structural features within a universal loop array conformation.
              </a>
            </td>
          <td>
            Marlies E. Oomen, A. N. Fox, Inma Gonzalez, A. Molliex, Thaleia Papadopoulou, Pablo Navarro, J. Dekker
          </td>
          <td>2025-06-06</td>
          <td>Genome research</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="
 e15032


 Background:
 Homologous recombination deficiency (HRD) is an event triggered by impairment of the homologous recombination repair (HRR) pathway which acts on DNA double-strand breaks and interstrand cross-links (ICL). Although HRD has been shown to elevate the overall genomic instability in tumors inducing favorable responses to immune checkpoint inhibitors (ICI), its clinical utility is observed in specific cancer types like breast, ovarian, low-grade glioma, testicular germ cell tumors and kidney cancers. On the contrary, in many other cancer types, HRD is conducive to the development of immune-cold tumor microenvironment leading to unfavorable treatment responses to ICI. We present, HRD predictive scores with the corresponding IO biomarkers viz. microsatellite instability (MSI) and tumor mutation burden (TMB) pan-cancers.
 Methods:
 Retrospectively, we investigated the biomarker profile (HRD, MSI, TMB) and genomic (HRR pathway genes) profiles of 642 pan-cancer patients using a comprehensive genomic assay using OncoIndx.
 Results:
 Of the cohort of 642 pan-cancer patients, a total of 12.3% (n = 79/642) of patients were detected with high HRD scores, another 16.0% (n = 103/642) were detected with intermediate HRD scores and while 71.7% (n = 460/642) of patients showed low HRD scores. Among the cohorts of high, intermediate and low HRD scores, BRCA1/2 mutation-positive patients were 6.3% (n = 5/79), 5.8% (n = 6/103), and 5.2 (n = 24/460) respectively and HRR gene mutation-positive patients were 17.7% (n = 14/79), 25.2% (n = 26/103), 17.17% (n = 79/460) respectively. Most importantly, the immune-oncology biomarkers MSI and TMB scores were high only in 1.3% (n = 1/79) and 8.9% (n = 7/79) in HRD-high patients which were not significant compared to 1.7% (n = 8/460) and 10.4% (n = 48/460) observed in HRD-low patient cohort. Even in the cohort of patients with intermediate HRD scores, MSI and TMB did not vary significantly. Overall, no significant correlation was observed between high HRD scores vs BRCA1/2 mutations, HRR pathway gene mutations, MSI and TMB statuses. Thus, HRD scores could be predicted high, irrespective of all the associated biomarkers and pathway mutations. This indicates the utility of additional treatment options in BRCA1/2 wild-type or HRR pathway gene wild-type patients.
 Conclusions:
 Although HRD-high patients could be predictive of favorable PARPi and platinum-based therapies, their role in immune activation and ICI therapy response should be carefully considered. Even at a high score, HRD seems to be inducing non-inflammatory tumor microenvironment as opposed to triggering novel cell surface antigens and high TMB aiding in immunotherapy response. However, HRD as an individual biomarker alone adds immense therapeutic benefits even among BRCA-negative patients who could be potential for PARPi/platinum-based therapy.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92c61b0baeee9c5a60d5b38acb29b5a9cb82e2f" target='_blank'>
              The non-canonical role of homologous recombination deficiency in the development of immune-cold tumor microenvironment.
              </a>
            </td>
          <td>
            A. Vasudevan, Aarthi Ramesh, Ankit Jain, Sandhya Iyer, Nidhi Patel, A. Joshi, M. K. Singhal, Sameer Shrirangwar, Chinmaya Kumar Pani, Ganapathi Bhat, V. Patil, Dr T. Raja, Jayant J Khandare, Gowhar Shafi
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad0e72d616aeb3ee8d41accd4d7a574cb580739" target='_blank'>
              A continuum reaction-diffusion model for spread of gene silencing in chromosomal inactivation
              </a>
            </td>
          <td>
            Shibashis Paul, Sha Sun, Karmella A. Haynes, Tian Hong
          </td>
          <td>2025-06-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Background: Breast cancer is the most frequently diagnosed cancer and the leading cause of cancer-related death among women worldwide. Triple negative breast cancer (TNBC) represents 15-20% of all incident breast cancers and is a highly aggressive subtype. Neo(adjuvant) chemotherapy (NAC) is standard for early stage TNBC; however, for those with residual breast cancer following NAC, recurrence is common and associated with few effective therapeutic approaches. Thus, identification of novel resistance mechanisms and alternative treatment strategies remain an unmet clinical need. Circular RNAs (circRNAs) are a newly identified class of largely noncoding RNA molecules with covalently closed circular structures. Recent studies suggest that circRNAs play crucial roles in regulating tumor development, progression and chemoresistance. However, the role of circRNAs in mediating chemotherapy resistance remains unknown. Methods: As a first step towards identifying circRNAs that participate in the development of chemoresistance, and to determine if targeting such circRNAs may be an efficacious strategy, doxorubicin-resistant (Doxo-R), paclitaxel-resistant (PTX-R), and double-resistant (DP-R) cell lines were generated from MDA-MB-231 cell lines. Human circRNA microarrays were utilized to profile the expression of approximately 14,000 known circRNAs in normal breast tissue, matched TNBC patient-derived xenografts (PDX) generated prior to and following NAC, and TNBC chemosensitive and chemoresistant cell lines. Top hits were validated using RT-PCR. Bioinformatic approaches were employed to identify circRNA-miRNA-mRNA networks regulated by differentially expressed circRNA transcripts. A reverse genetics approach was also employed using CRISPR to interrogate the contributions of over 4000 specific circRNA in driving the development of resistance to doxorubicin and paclitaxel. Results: circRNA microarray profiling identified 429 and 310 transcripts differentially expressed in doxorubicin and paclitaxel resistant cells, respectively, compared to parental chemosensitive cell lines (|FC| ≥ 1.5; p value < 0.05). Given that circRNAs can act as microRNA (miRNA) sponges, thereby inhibiting miRNA activity, we computationally identified circRNA-miRNA pairs, and assessed the abundance of mRNA transcripts predicted to be targets of paired miRNAs via RNA-seq. A differentially expressed circRNA-miRNA-mRNA network was further constructed. Dysregulated canonical pathways and gene ontology (GO) enrichment analysis were determined from these networks and revealed multiple GO term included negative regulation of miRNA transcription, miRNA and RNA metabolic process, and endoplasmic reticulum-associated degradation (ERAD) pathway. The CRISPR-Cas13 screen is ongoing at this time but results will be available prior to the meeting. Conclusions: We have identified cancer specific circRNAs in TNBC cell lines, PDX models and patient tumors in the context of chemotherapy sensitive and chemotherapy resistant disease. Specific biological processes, pathways and signaling cascades have been implicated in the down-stream consequences of dysregulated circRNAs and in the potential development of resistance. Our ongoing studies seek to identify specific circRNAs sufficient to drive resistance to chemotherapy. Outcomes of these studies highlight the potential of specific circRNAs to serve as predictive/prognostic biomarkers and novel therapeutic targets for advanced forms of TNBC.
 Citation Format: Xiyin Wang, Michael J. Emch, Xiaojia Tang, Esther P.B. Rodman, Liewei Wang, Judy C. Boughey, Matthew P. Goetz, Krishna R. Kalari, John R. Hawse. The role of circular RNAs in triple-negative breast cancer and chemotherapy resistance [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-02-22.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb284df6818a5ca75827ea02126a84e48b2886e9" target='_blank'>
              Abstract P3-02-22: The role of circular RNAs in triple-negative breast cancer and chemotherapy resistance
              </a>
            </td>
          <td>
            Xiyin Wang, M. Emch, Xiaojia Tang, E. Rodman, Liewei Wang, J. Boughey, Matthew P. Goetz, K. Kalari, John R. Hawse
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 5072


 Background:
 STEAP
 1 and 2 (six-transmembrane epithelial antigen of prostate) are metalloreductase proteins involved in a variety of biologic processes.
 STEAP1/2
 are tumor-associated cell surface antigens highly expressed in prostate cancer (PC), although their role in cancer is poorly understood.
 STEAP1/2
 have emerged as successful targets for adoptive T-cell therapy trials for PC. We employed a multi-omics approach to investigate the molecular features associated with
 STEAP1
 and
 STEAP2
 expression in PC.
 Methods:
 NextGen Sequencing of DNA (592 genes or whole exome) and RNA (whole transcriptome) was performed for PC tumors (n = 7089) submitted to Caris Life Sciences (Phoenix, AZ). PC samples were stratified by
 STEAP1/2
 mRNA levels into top (high) and bottom quartile (low). Immune cell infiltration in the tumor microenvironment (TME) was inferred by quanTIseq. Transcriptomic signatures of androgen receptor signaling (AR), neuroendocrine classification (NEPC), and interferon gamma signaling (IFN) were calculated. Mann-Whitney U and X2/Fisher-Exact tests were applied where appropriate, with P-values adjusted for multiple comparisons (q< .05).
 Results:
 Of 7,089 samples, 63.2% were from the prostate; 11.7% from lymph node metastases (LNM); 7.3% from bone; and 17.8% from visceral/soft tissue metastases (V/STM).
 STEAP
 -1 and -2 were significantly correlated to each other (R= 0.90, p<.001), with significantly higher expression of
 STEAP1
 observed in primary prostate and LNMs, compared with reduced expression in V/STM (
 STEAP1
 TPM: 105.2 vs 140.6 vs 91.9 p<.001). Mutations in
 AR
 (3.8% v 1.9%),
 KDM6A
 (4.2% v 2.2%),
 SPOP
 (10.9% v 8.4%) and
 AR
 V7 (23.0% v 10.5%) were enriched in
 STEAP1
 high PC (each q<.01). Mutations in
 KDM6A
 (3.8% v 2.5%) and
 AR
 V7 (17.6% v 14.3%) were enriched in
 STEAP2
 high PC (each q<.05).
 STEAP1/2
 expression negatively correlated with TMB count (R =-0.03, p<.05) and IFN score (R = -0.26, p<.001). Concordantly, fewer proinflammatory immune cell fractions (M1 Macrophages, NK cells, CD4+/CD8+ T cells, myeloid dendritic cells) were observed within the TME of
 STEAP1/2
 high PC (p<.0001). However,
 STEAP1/2
 expression correlated positively with the AR signature (R = 0.39, p<.001) and androgen response pathways, while correlating negatively with the NEPC signature (R = -0.15, p<.001).
 Conclusions:
 PC tumors expressing high
 STEAP1/2
 display distinct genomic and transcriptomic profiles compared to
 STEAP1/2
 -low PC, and
 STEAP 1/2
 expression varies across sites of metastases. Immune biomarkers and immune cell infiltration data suggest that STEAP1/2 may be associated with a cold TME. The recent success of STEAP1-targeting T-cell redirecting therapies mechanisms by which adoptive T-cell strategies may overcome immunosuppressive factors within the TME. Ongoing development of T-cell immunotherapeutics targeting STEAP1 may account for the differential expression profiles in guiding patient selection and combination strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3db32deff1c58e26b5169abd853a15c53e234839" target='_blank'>
              Molecular characterization of STEAP1 and -2 in advanced prostate cancer.
              </a>
            </td>
          <td>
            Kevin K. Zarrabi, T. Adeyelu, A. Elliott, D. Geynisman, D. Oh, Carissa E. Chu, Rana R McKay, Nicholas A. Zorko, E. Antonarakis, Lucia R Languino, Pedro C. Barata, D. Danila, Norm Smith, William Kevin Kelly
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Ovarian cancer ranks as the eighth most common malignancy in women globally. Early detection remains a critical challenge for improving ovarian cancer diagnosis and treatment outcomes. Our prior study demonstrated that ARHGAP40 downregulation in basal cell carcinoma is attributed to CpG island hypermethylation. However, the expression and methylation status of ARHGAP40 in ovarian cancer remain unexplored. Here, we investigated ARHGAP40 protein expression in normal fallopian tubes, ovarian benign tumors, borderline tumors, low-grade serous carcinoma (LGSC) and high-grade serous carcinomas (HGSC). Methylation analysis of the ARHGAP40 was performed using bisulfite sequencing PCR (BSP). MethyLight assays were developed to detect methylated circulating tumor DNA (ctDNA) fragments of ARHGAP40. IHC results revealed absent or weak ARHGAP40 protein expression in 93.8% (30/32) of HGSC, 11.1% (2/18) of borderline tumors, and in 14.3% (1/7) of LGSC cases. ARHGAP40 protein expression was robust expressed in all normal fallopian tubes (15/15) and benign ovarian tumors (8/8). CpG island hypermethylation in the ARHGAP40 promoter showed a strong inverse correlation with protein expression (P < 0.001). Furthermore, methylated ctDNA for ARHGAP40 was detected in 80.0% (4/5) of HGSC patients’ plasma, but not in benign tumor (0/3) and healthy controls (0/15). Our findings suggest that promoter hypermethylation may be a mechanism underlying ARHGAP40 silencing in HGSC. Detection of methylated ARHGAP40 ctDNA may serve as a noninvasive biomarker for early diagnosis and monitoring of HGSC. While our findings suggest the potential of methylated ARHGAP40 as an early diagnostic biomarker, the small sample size warrants validation in larger cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a13bc1c0523b1d5d86e6d21e3c7e8792952655" target='_blank'>
              Methylated ARHGAP40 DNA as a potential biomarker for early diagnosis in high-grade ovarian serous cancer
              </a>
            </td>
          <td>
            Jiaxin Chai, Dongni Leng, Shuwei Guo, Rusong Zhang, Jiandong Wang, Yun Gu, Qiu Rao
          </td>
          <td>2025-07-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 11550


 Background:
 The development of resistance is the one of the reason for ineffective treatment of soft tissue sarcomas (STSs). The search for new molecular-biological targets, both for the personalized selection of novel targeted drugs and for predicting chemoresistance of STSs to individual antitumor agents and their combinations, is a pressing task.
 Methods:
 In this study, an experimental in vitro chemoresistance analysis, whole-genome sequencing, and single-cell sequencing of tumors in patients (N = 9) with undifferentiated pleomorphic sarcomas were performed. Chemotherapy resistance indices were determined for the obtained samples.
 Results:
 Whole-genome DNA sequencing yielded data on the type and frequency of somatic mutations in undifferentiated pleomorphic sarcomas, and driver genes of carcinogenesis were identified, among which the most frequently occurring were FCGBP, PARP4, TP53, RGPD3, PDE4DIP, and RB1. Correlations between the presence of genetic alterations and the response to chemotherapy in the in vitro test were sought. Thus, in samples resistant to the combination of doxorubicin and ifosfamide, the presence of mutations in the PEG3, USP8, NT5C3A, and WAS genes was characteristic. Bioinformatic analysis of single-cell transcriptome sequencing data, aimed at characterizing the population composition and transcriptomic landscape of undifferentiated pleomorphic sarcoma cells, revealed 15 normal and 8 tumor cell populations. Among the populations of normal cells, the following clusters were identified: M2 and M1 macrophages, T-cells, matrix-remodeling fibroblasts and myofibroblasts, monocytes, stromal cells, endothelial cells, and osteoblasts. Among tumor cells, clusters of proliferating cells, hypoxic cells, matrix-interacting cells, and tumor cells with a stem-like phenotype were identified. The most common clusters in all samples were M2 macrophages, endothelial cells, and one of the clusters of tumor cells interacting with the matrix. Comparison of the expression profiles of cells from patients with different responses to chemotherapy allowed identification and characterization of clusters associated with chemosensitivity and chemoresistance. These included clusters of macrophages, tumor cells, fibroblasts, and endothelial cells. In addition, genes common to all clusters associated with resistant samples were identified (ARGLU1, JUND, TNNT3, RHOB, CCNL2, LENG8, LUC7L3, KLF9, RSRP1, RNF213, SERPINE1, CDK5RAP3).
 Conclusions:
 The data obtained in this study will expand the understanding of the pathogenesis of undifferentiated pleomorphic sarcomas and the mechanisms underlying their development of chemotherapy resistance. In the future, this will serve as the basis for creating a test system to evaluate the expression levels of genes whose activation/repression is associated with the development of drug resistance in STSs.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9452b21e788959623418046da69bbf46291bf598" target='_blank'>
              Molecular-genetic characteristics of soft tissue sarcomas associated with the development of chemotherapy resistance.
              </a>
            </td>
          <td>
            A. Tararykova, T. Fetisov, P. Shtompel, S. Khazanova, E. Trapeznikova, Maxim Menyailo, A. Ikonnikov, Elena E Kopantseva, A. Korobeynikova, B. Bokhyan, M. Yakubovskaya, K. Kirsanov
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="DNA methylation is a key epigenetic modification that regulates gene expression, cell differentiation, and genome stability. Aberrant DNA methylation patterns, including the hypermethylation or global hypomethylation of tumor suppressor genes, are strongly associated with various human diseases, such as cancer, autoimmune disorders, and metabolic syndrome. DNA methylation predominantly occurs at CpG dinucleotides, influencing transcription by altering chromatin structure and accessibility. MBD2 (Methyl-CpG-binding proteins 2) play a crucial role in interpreting these epigenetic marks and regulating downstream gene expression. In disease contexts, aberrant DNA methylation disrupts cellular homeostasis by silencing key regulatory genes or activating pathological pathways. Current research primarily focuses on MBD2 in cancer, with less emphasis on its role in autoimmune diseases. This review discusses the role of MBD2 in regulating immune cell development and differentiation through epigenetic mechanisms, particularly DNA methylation and its regulatory components. Furthermore, it highlights the mechanistic contributions of MBD2 to autoimmune diseases such as systemic lupus erythematosus and evaluates its potential as a novel therapeutic target for these conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09aaf1514800c8b7b42acb275ab57b91b16493dd" target='_blank'>
              The role of MBD2 in immune cell development, function, and autoimmune diseases
              </a>
            </td>
          <td>
            Yunfei Zhang, Yufeng Fan, Ying Hu, Xiaocui Wang, Bin Wen, Xuemei Duan, Haonan Li, Shumin Dong, Ze Yan, Weiwei Zhang, Yukai Jing
          </td>
          <td>2025-06-19</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor development is mainly marked by the gradual transformation of cells that acquire capacities such as sustained growth signaling, evasion of growth suppression, resistance to cell death, and induction of angiogenesis, achieving replicative immortality and activating invasion and metastasis. How different epigenetic alterations like m1A, m5C, and m6A contribute to tumor development is a field that still needs to be investigated. The immune modulators, CD70, CD80, and TIGIT, mainly regulate T-cell activation and consequently the immune evasion of tumors. Here, we explored the presence and the potential consequences of RNA modifications in these regulators in pan-cancer. Our findings highlight the critical role of the m6A, m5C, and m1A in regulating CD70, CD80, and TIGIT across multiple solid tumors. By combining epitranscriptomics data with functional enrichment and survival modeling, we show that RNA modification enzymes not only modulate immune-related gene expression but also serve as potential biomarkers for patient prognosis. By constructing a robust four-gene prognostic signature involving YTHDF3, RBM15B, IGF2BP2, and TRMT61A, we demonstrate that RNA modification profiles can accurately stratify patients into risk groups with distinct overall survival outcomes. The performance of this model across eight cancer types underscores the translational promise of epitranscriptomic markers in both mechanistic understanding and personalized oncology. Altogether, our study bridges the gap between the mechanistic regulation of immune checkpoints and their clinical utility, offering novel insights into how the epitranscriptome can be leveraged to improve cancer prognosis and potentially enhance immunotherapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d03e20bb526dd9e6903a8377f001f578b9d1d1d" target='_blank'>
              Epitranscriptomics Regulation of CD70, CD80, and TIGIT in Cancer Immunity
              </a>
            </td>
          <td>
            C. Rigopoulos, Marios Gkoris, Ilias Georgakopoulos-Soares, I. Boulalas, A. Zaravinos
          </td>
          <td>2025-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Canine lymphoma, a phenotypically and genetically heterogeneous disease, represents a significant proportion of canine cancers. We present a large-scale study of 238 dogs with lymphoma to better understand the genetic landscape of canine lymphoma, as well as the relationship to clinical outcomes. Using a targeted next-generation sequencing panel comprising 308 genes, we screened somatic and germline mutations in matched tumor and normal samples. Our findings revealed key associations between genetic alterations and lymphoma subtypes, with certain somatic variants linked to significant differences in response to common chemotherapy regimens. Recurrent mutations in genes such as KMT2C, KMT2D, NOTCH2, TRAF3, CCND1, ARID1A, CREBBP, and TP53 were observed, with TRAF3 mutations standing out for their significant association with prolonged progression-free survival and overall survival in B-cell lymphomas. In contrast, mutations in PIK3CD and CREBBP were associated with inferior outcomes in T-cell lymphomas, highlighting the immunophenotype-specific impact of genetic alterations on treatment responses. These findings support the integration of comprehensive genomic profiling in planning treatment strategies and optimizing clinical outcomes in canine lymphomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e07f223413194847051e584c8f37b09d9f47c90d" target='_blank'>
              Identification of novel genetic mutations for the treatment prognostication of canine lymphoma
              </a>
            </td>
          <td>
            Josephine Tsang, Qi-Jing Yap, Sheena Kapoor, Jerry Cromarty, Sushmita Sen, Minji Kim, George Courcoubetis, Suhyeon Cho, Deanna Swartzfager, Stanley Park, Sungwon Lim, Ilona Holcomb, Jamin Koo
          </td>
          <td>2025-06-12</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dbcccd092da4df387dc9187eeb5acec5c01b0b6" target='_blank'>
              Loss of chromosome Y shapes the immune cell fate with aging in men
              </a>
            </td>
          <td>
            A. A. Z. Dawoud, L. Green, O. Rackham
          </td>
          <td>2025-06-02</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88a889f8e2ddcb6a4b7bab00376e74479c448ebb" target='_blank'>
              Structural basis of Fanconi anemia pathway activation by FANCM.
              </a>
            </td>
          <td>
            R. Bythell-Douglas, Sylvie van Twest, Lara Abbouche, Elyse Dunn, Rachel J Coulthard, D. Briggs, Vincent J. Murphy, Xinxin Zhang, Winnie Tan, Sarah S Henrikus, Dongming Qian, Yin Wu, Jana Wolf, Laurent Rigoreau, S. Shakeel, Kathryn L Chapman, N. Q. McDonald, A. Deans
          </td>
          <td>2025-05-30</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Non-coding RNAs (ncRNAs) have important regulatory functions similar to traditional oncogenes or tumor suppressor genes. Our previous research found a novel pan-cancer downexpressed ncRNA, PANC754. However, its function and underlying mechanism remain obscure in colorectal cancer (CRC). In this study, in vitro and in vivo experiments were performed to determine the function of PANC754. Loss and gain of function experiments, molecular docking experiments, and bioinformatic analysis were utilized to visualize its pathway. Co-culture system was leveraged to explore its effect on synergetic immune checkpoint blockage against CRC. Through a series of studies, we found that overexpressed PANC754 significantly inhibited cell viability, migration, and metastasis and induced notable apoptosis in CRC. The mechanical research found that PANC754 was the nuclear-located and its expression was regulated by m6A modification via METTL3 enzyme, which bound with its RBP PSPC1, then interacted with H3K4me1 to chromatin-accessible inhibit immune evasive molecule LGALS7 and led to suppress CRC progress. Furthermore, we confirmed that prominent upregulation of the immune checkpoint inhibitory (ICI) capability of anti-NKG2A, monalizumab when it was combined with PANC754 overexpression. Collectively, our study revealed that PANC754 is a tumor-suppressing ncRNA to form an ncRNA/RBP/histone repression complex with m6A-dependence, which can enhance the immune therapeutics effect of ICI, suggesting a promising therapeutic target. PANC754 is a novel pan-tumor suppressing non-coding RNA which is m6A-dependent and regulated by METTL3 modification enzyme. PANC754 is located at cellular nuclear and interacts with its RNA binding protein (RBP) PSPC1 and chromatin accessible histone H3K4me1, then can enhance immunotherapy capability of ICB anti-NKG2A against colorectal cancer through the immune evasive molecule LGALS7 signaling. Effect is that novel nuclear PANC754/PSPC1/H3K4me1 repression complex down-regulates the level “Don’t eat me” signal LGALS7 to improve the immune efficiency of ICB and induce NK or CTL cell to release perforin and cytokine to kill tumor cells. (Created by Figdraw).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef91b217bd2a213904410de9cf4af1abcc075b69" target='_blank'>
              M6A-METTL3-dependent nuclear PANC754/PSPC1/H3K4me1 repression complex regulate immune evasive LGALS7 signal to enhance immunotherapy against colorectal cancer
              </a>
            </td>
          <td>
            Jianfeng Zhang, Guilian Cao, Feng Li, Siyu Tang, Chenxi Wu, Min Jiang, Jingxin Ye, Shaoqing Ju, Fei Qian, Weifeng Ding
          </td>
          <td>2025-07-09</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary Hepatocellular carcinoma (HCC) is a highly heterogeneous malignant tumor. Among the various epigenetic alterations involved in its pathogenesis, the N6-methyladenosine (m6A) modification of eukaryotic mRNA plays a pivotal role in tumor biology and holds significant potential for diagnostic and therapeutic applications. In this study, we identified and validated an 8-gene risk score that robustly predicts prognosis in HCC patients, achieving area under the ROC curve (AUC) values greater than 0.75 across both internal and external cohorts. A high-risk score was significantly associated with poorer clinical outcomes and reduced immune tolerance, while also indicating a potentially enhanced response to immunotherapy. Single-cell RNA sequencing revealed a higher proportion of T cells and a decreased presence of immunosuppressive T cells in the high-risk group, which may account for their improved responsiveness to immune checkpoint inhibitors. Finally, in vitro experiments and immunohistochemical staining showed the biological function of the key gene ANLN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea94ab482ecd557d67969f8038f92a2cc7e807c" target='_blank'>
              Stratification of Hepatocellular Carcinoma Using N6-Methyladenosine
              </a>
            </td>
          <td>
            Nan Wang, Jia-Xin Shi, M. Bartneck, E. Dahl, Junqing Wang
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 8602


 Background:
 MTAP
 loss leads to methylthioadenosine (MTA) accumulation, disrupting downstream metabolic pathways. Targeting synthetic lethal interactions with
 MTAP
 loss offers a promising therapeutic strategy. The DNA damage response (DDR) pathway, essential for genomic stability, is commonly dysregulated in lung cancer, impacting treatment response and prognosis. Emerging evidence has highlighted potential association between
 MTAP
 loss and mutations in DDR genes, suggesting that investigating their relationship is critical for uncovering disease mechanisms and identifying novel biomarkers and therapeutic targets.
 Methods:
 Hybridization capture sequencing with StarPanel NGS Assay (1,326 genes) was conducted on 2,258 Chinese lung cancer patients' tumor and matched peripheral blood samples. Then somatic and germline mutations, and TMB values of each sample were obtained. Based on a depth-based algorithm, differences between tissue and control samples in CNVs from the gene level were analyzed. Besides, we utilized shifts in germline heterozygous SNPs within the gene region to assist in determining if the loss was homozygous or heterozygous.
 Results:
 Homozygous and heterozygous deletions of
 MTAP
 were detected in 11.07% and 6.95% of all samples. Notably, 61.67% of these samples exhibited co-loss of
 MTAP
 ,
 CDKN2A
 and
 CDKN2B
 . The median TMB was significantly higher in samples with
 MTAP
 loss (3.85 mut/Mb) compared to those with intact
 MTAP
 (2.56 mut/Mb). In samples with homozygous
 MTAP
 loss, the top somatically co-altered genes were
 EGFR
 (76.00%),
 TP53
 (57.60%) and
 CDKN2A
 (52.80%). Gain-of-function (GoF) mutations were most prevalent in
 EGFR
 (56.80%),
 KRAS
 (12.00%) and
 MDM2
 (10.00%), while Loss-of-function (LoF) mutations were most common in
 TP53
 (50.40%),
 RBM10
 (11.60%) and
 PTEN
 (7.60%). Notably, LoF mutations in
 MTAP
 -loss samples showed a higher prevalence of DDR genes compared to
 MTAP
 -intact samples, including
 RAD50
 (2.40% vs. 0.00%,
 p
 < .0001) and
 POLQ
 (2.40% vs. 0.00%,
 p
 < .0001). Enrichment analysis further revealed LoF mutations unique to
 MTAP
 -loss samples were significantly enriched in the DDR pathway (
 p
 < .0001). For top germline pathogenic mutations, two DDR genes,
 RECQL4
 (0.80% vs. 0.38%,
 p
 = 0.29) and
 BRCA2
 (0.40% vs. 0.43%,
 p
 = 1), showed no significant differences between
 MTAP
 -loss and
 MTAP
 -intact samples.
 Conclusions:
 Potential association exists between
 MTAP
 loss and DDR pathway dysregulation, which may impact tumorigenic processes and therapeutic vulnerabilities. The enrichment of DDR-related LoF mutations indicates
 MTAP
 loss could exacerbate genomic instability by impairing DNA damage repair mechanisms, thereby increasing TMB and driving cancer progression with a distinct molecular profile. These vulnerabilities are primarily driven by somatic mutations, providing a rationale for exploring personalized treatment strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/305ad8ea528f00d8a04f7506987172e9d5a5207d" target='_blank'>
              Dysregulation of DNA damage repair in lung cancer driven by
 MTAP
 loss: Mechanistic insights and target discovery.
              </a>
            </td>
          <td>
            Bo Jiang, W. Liao, Xin Wang, Hongliang Hui, Yan-Gui Lin, Linjie Qiu, Dan Li, Min Luo, Haoran Miao, Fan Qiu, Yiqian Zhang
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 e20051


 Background:
 Histone lactylation, a newly identified epigenetic modification, has been extensively implicated in the initiation and progression of tumors. However, the precise role of histone lactylation in modulating tumor cell responses to T cell-mediated cytotoxicity remains insufficiently explored.
 Methods:
 To elucidate the adaptive changes tumor cells undergo in response to T cell-induced cytotoxicity within the tumor microenvironment (TME), we performed a comprehensive, genome-wide CRISPR-Cas9 screen. This approach facilitated the identification of genetic alterations in lung cancer cells co-cultured with cytotoxic T lymphocytes (CTLs) that promote their evasion of T cell-mediated destruction. Complementary RNA sequencing (RNA-seq) and chromatin immunoprecipitation sequencing (ChIP-seq) analyses were employed to investigate the molecular mechanisms by which tumor cells modulate their response to T cell-mediated killing, particularly following the deletion of the monocarboxylate transporter 1 (MCT1).
 Results:
 MCT1 deletion significantly heightened the cancer-cell susceptibility to CTL-mediated killing. Furthermore, the loss of MCT1 led to mitochondrial dysfunction within the tumor cells. Mechanistically, MCT1 knockout resulted in a reduction of histone H3K18 lactylation and downregulation of ferridoxin reductase (FDXR), which disrupted mitochondrial homeostasis and precipitated mitochondrial dysfunction. Notably, the combination of MCT1 inhibition with immune checkpoint blockade (ICB) therapy substantially enhanced ICB efficacy, prolonged survival in both lung cancer and malignant pleural effusion (MPE) murine models, and alleviated the immunosuppressive milieu within the TME.
 Conclusions:
 Our findings reveal a novel role for lactate metabolism in modulating mitochondrial dysfunction and anti-tumor immunity, with lactylation-mediated regulation of mitochondrial homeostasis serving as a critical mechanism. These results provide new insights into the epigenetic regulation of tumorigenesis and highlight potential immune-metabolic strategies for targeted lactate metabolism therapies in the treatment of lung cancer and MPE.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db7369949ad290ca8642cb013b1fc65d9e0567a5" target='_blank'>
              MCT1-mediated histone lactylation modulates lung cancer cell sensitivity to T-cell–mediated killing by influencing mitochondrial dysfunction.
              </a>
            </td>
          <td>
            Q. Xue, W. Peng, Qiong Zhou
          </td>
          <td>2025-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="SET domain bifurcated histone methyltransferase 1 (SETDB1) is a histone H3 lysine 9 (H3K9) methyltransferase, functioning in transcriptional silencing of the transposable elements (TEs), endogenous retroviruses (ERVs), interferon-stimulated genes (ISGs), and immune cell-related molecules. SETDB1 collaborates with cofactors such as ATF7IP and TRIM28 and functions in concert with DNA methylation to maintain gene repression in both stem cells and differentiated somatic cells. Given its gene targets, recent studies have shown that SETDB1 is a critical immune checkpoint gene in both solid and hematological tumors. In this review, we first discuss the role of SETDB1 in gene regulation through its histone methyltransferase activity, including an overview of its structural features and key cofactors. We then highlight the lineage-specific roles of SETDB1 in both normal hematopoietic processes and hematological malignancies, emphasizing its function as an immune checkpoint molecule that suppresses natural killer (NK) cell-mediated antileukemia responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b622142a48ee47c5894f0bcff4fb87ea6d35f6c" target='_blank'>
              The Role of Histone Methyltransferase SETDB1 in Normal and Malignant Hematopoiesis.
              </a>
            </td>
          <td>
            Yu-Hsuan Chang, S. Goyama
          </td>
          <td>2025-06-04</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Introduction: BRCA1 promoter hypermethylation (BRCA1meth), which is a driver of homologous recombination deficiency (HRD), is observed in up to 25% of triple negative breast cancers (TNBCs) and ovarian carcinomas (OvCas). BRCA1meth is associated with worse outcome compared to BRCA1 mutations (Menghi et al., PMID: 35857626), and, when found in individuals without cancer (i.e. constitutional BRCA1meth), it is associated with increased risk of breast and ovarian cancer (Lonning, et al., PMID: 36074460). Current methodologies to determine the presence of BRCA1meth in a cancer sample require tissue biopsy and have not been adopted for standard clinical assessment. The GRAIL cell-free DNA (cfDNA)-based targeted methylation platform is a robust, biopsy-free, scalable assay that was optimized to distinguish cancer methylation patterns between different cancer signal origins. Here, we describe the GRAIL platform’s ability to detect and quantify BRCA1meth. Methods: As part of the CCGA study (NCT02889978), clinical data were recorded and plasma samples were collected and processed on GRAIL’s targeted methylation platform from 2,958 patients with cancer prior to any treatment, and from 2,790 individuals without cancer. Samples were evaluated from patients with a variety of cancers: breast, ovarian, cervical, uterine, lung, prostate, head and neck, kidney, bladder, stomach, esophageal, colorectal, and hematological malignancies. A dilution series of contrived samples was analyzed to establish analytical characteristics of BRCA1meth detection using the GRAIL assay. Results: When applied to contrived samples, the GRAIL platform was able to detect BRCA1meth at an LoD95 of <1% (as measured by BRCA1meth fraction = methylated molecules / total molecules). When applied to a cohort of 2,790 individuals without cancer, we found 4% of individuals harbored evidence of BRCA1meth in the fractional range of 0.4% to 37%, with the majority of individuals (81%) having <5% BRCA1meth fraction, similar to surveys using peripheral blood cell DNA (Lonning et al., PMID: 36074460). Consistent with recent data of constitutional BRCA1meth prevalence in umbilical cord blood of newborns (Nikolaienko, et al., PMID: 38053165), we see lower incidence of BRCA1meth in males without cancer (3%) compared to females without cancer (5%, p=0.01). BRCA1meth was significantly enriched in the cfDNA of patients with TNBC and OvCa compared to female individuals without cancer (17/112 (15%), adjusted p=0.001 for TNBC; 14/101 (14%), adjusted p=0.01 for OvCa), in concordance with previous assessments of BRCA1meth prevalence across different tumor types (Menghi et al., PMID: 35857626). In TNBC and OvCa, the per-sample BRCA1meth fraction ranged from 0% to 54% and was correlated with the methylation-based estimate of tumor fraction in both TNBC (R=0.68, p=0.0052) and OvCa (R=0.65, p=0.03). High BRCA1meth fractions (up to 16-21%) were observed in four individuals with uterine, colorectal, bladder, or kidney cancer. This suggests that BRCA1 deficiency may be implicated in these rare cases, which would imply HRD, making them candidates for specific treatments. Finally, across tumor types that affect both sexes, we found consistently lower percentages of BRCA1meth in male vs. female patients, reflecting the differential rates of constitutional BRCA1meth across the two sexes. This suggests that the dynamics of BRCA1 promoter methylation and its consequences for the development of cancer may differ by sex. Conclusions: This work shows that the GRAIL platform, as a single blood-based assay, can sensitively detect and quantify BRCA1meth from cfDNA, which may inform cancer susceptibility in non-cancer bearing individuals and may also have potential treatment selection implications for cancer patients.
 Citation Format: Francesca Menghi, Margaret Antonio, William Cance, Tracy Nance, Edison T. Liu. Quantification of Breast Cancer 1 gene (BRCA1) promoter methylation in individuals with and without cancer using cell-free DNA [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-11-03.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8ede0da8ea6d210ede4f24e81580ed41fdb9cdd" target='_blank'>
              Abstract P2-11-03: Quantification of Breast Cancer 1 gene (BRCA1) promoter methylation in individuals with and without cancer using cell-free DNA
              </a>
            </td>
          <td>
            F. Menghi, Margaret Antonio, William Cance, Tracy Nance, Edison T. Liu
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Invasive lobular carcinoma (ILC) is the second most common histological subtype of breast cancer after invasive ductal carcinoma of no special type (NST), accounting for 10–15% of diagnoses. Despite the myriad molecular, histological and clinical differences between ILC and NST tumors, patients are treated in the same way, and although prognosis initially is good, ILC patients have poorer long-term outcomes. Understanding the differences between these two subtypes and identifying ILC-enriched therapeutic targets is necessary to improve patient care. Methods Human and mouse cancer-associated fibroblasts (CAFs), ILC cell lines and patient-derived organoids were used for in vitro and in vivo studies, including western blotting, migration, organotypic invasion assays and dissemination in zebrafish embryos. RNASeq was used to identify CAF and interleukin-6 (IL-6)-derived gene signatures. Bioinformatic analysis of public databases and immunohistochemical of human tumor microarrays was carried out. Results We identified IL-6 as a paracrine CAF-derived factor that activates Signal-Transducer-and-Activator-of-Transcription-3 (STAT3) in human and mouse ILC models. Analysis of human breast tumors showed that the IL-6/JAK/STAT3 pathway is enriched in ER + ILC compared to ER + NST. A 42-gene CAF dependent IL-6 gene signature and 64-gene consensus IL-6 gene signature were generated and were significantly enriched in ER + ILC, with many of the genes overexpressed in ILC tumors. IL-6 treatment suppressed downstream estrogen signaling and also led to the acquisition of a more mesenchymal-like phenotype associated with increased migration and invasion. Finally, IL-6 treatment significantly increased ILC cell dissemination following injection into zebrafish embryos. Conclusions CAF-derived IL-6 drives paracrine activation of the IL6/JAK/STAT3 signaling pathway which is enriched in ILC. This leads to the acquisition of pro-tumorigenic phenotypes, highlighting the pathway as a potential therapeutic target in ILC. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02074-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57ff0735a08b68cfe9b8184d5f4b41c0bb6234a9" target='_blank'>
              Cancer-associated fibroblast driven paracrine IL-6/STAT3 signaling promotes migration and dissemination in invasive lobular carcinoma
              </a>
            </td>
          <td>
            Esme Bullock, Aleksandra Rozyczko, Sana Shabbir, Ifigenia Tsoupi, Adelaide I J Young, J. Travnickova, Laura Gómez-Cuadrado, Zeanap Mabruk, Giovana Carrasco, Elizabeth Morrow, K. Pennel, Pim Kloosterman, J. Houthuijzen, Jos Jonkers, L. Avalle, Valeria Poli, R. Iggo, Xue Xiao, Jingjing Guo, Xuan Zhu, Elizabeth Mallon, Joanne Edwards, E. E. Patton, Valerie G. Brunton
          </td>
          <td>2025-07-01</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 DNA repair processes are the foundation for genome integrity and survival, especially in extreme environments where DNA damage occurs more frequently and where archaea are found. Nevertheless, first-hand experimental information on repair pathways in archaea is scarce, and assignment of repair proteins is currently largely based on homology. We showed previously that DNA lesions induced by CRISPR-Cas self-targeting are repaired by microhomology mediated end joining (MMEJ). To identify proteins involved in the archaeal MMEJ pathway, we used deletion strains devoid of proteins assigned to the key steps of MMEJ, to examine changes in the repair outcome. In addition we used aphidicolin to inhibit the activity of the essential PolB1 protein. For the first time, we were thereby able to experimentally identify proteins involved in this repair pathway in the euryarchaeal model organism Haloferax volcanii. The present study confirms that Mre11, Rad50, Fen1, PolB1, LigA and LigN take part in MMEJ, as previously inferred. In addition, we show that Cas1 and Hel308a are also involved in the MMEJ pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5af42e41866e31a1c7b25ea6eee79e0cec5ab57c" target='_blank'>
              CRISPR-Cas induced self-targeting identifies key players in archaeal microhomology mediated end joining
              </a>
            </td>
          <td>
            A-L. Sailer, J. Wörtz, V. Smith, A. Stachler, F. Blau, M. Daratha, L-K Maier, T. Allers, A. Marchfelder
          </td>
          <td>2025-07-08</td>
          <td>microLife</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Background Triple-negative breast cancer (TNBC) is commonly characterized by high-grade and aggressive features, resulting in an augmented likelihood of distant metastasis and inferior prognosis for patients. Tumor immune microenvironment (TME) has been recently considered to be tightly correlated with tumor progression and immunotherapy response. However, the actual heterogenous TME within TNBC remains more explorations. Methods The thorough analyses of different cell types within TME were conducted on the self-tested single-cell RNA sequencing dataset which contained nine TNBC treatment-naïve patients, including subclusters classification, CellChat algorithm, transcription factors (TFs) expression, pseudotime analysis and functional enrichment assay. The malignant epithelial cluster was confirmed by copy number variations analysis, and subsequently LASSO-Cox regression was carried out to establish a Malignant Cell Index (MCI) model on the basis of five crucial genes (BGN, SDC1, IMPDH2, SPINT1, and UQCRFS1), which was validated in several TNBC cohorts through Kaplan–Meier survival and immunotherapy response analyses. The public spatial transcriptome, proteome data and qRT-PCR, western blotting experiments were exploited to corroborate UQCRFS1 expression in RNA and protein levels. Additionally, functional experiments were implemented to unravel the impacts of UQCRFS1 on TNBC cells. Results The diverse subclusters of TME cells within TNBC were clarified to display distinct characteristics in cell–cell interactions, TFs expression, differentiation trajectory and functional pathways. Particularly, IL32high Treg imparted an essential effect on tumor evasion and predicted a worsened prognosis of TNBC patients. Furthermore, MCI model enabled to notify the inferior prognosis and immunotherapy resistance in TNBC. Ultimately, UQCRFS1 knockdown dampened the proliferative and migratory competence in vitro as well as tumor growth in vivo of TNBC cells. Conclusions Our study offers innovative perspectives on comprehending the heterogeneity within TME of TNBC, thereby facilitating the elucidation of TNBC biology and providing clinical recommendations for TNBC patients' prognosis, such as IL32high Treg infiltration, MCI evaluation, and UQCRFS1 expression. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00672-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bde7e708bae8d3687cc62eafeff7c490a66352ee" target='_blank'>
              Multi-omics analyses of the heterogenous immune microenvironment in triple-negative breast cancer implicate UQCRFS1 potentiates tumor progression
              </a>
            </td>
          <td>
            Yuhui Tang, Aiqi Xu, Zhongbiao Xu, Jindong Xie, Wei Huang, Liulu Zhang, Yitian Chen, Lu Yang, Shasha Du, Kun Wang
          </td>
          <td>2025-06-16</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The tumor suppressor protein p53, widely known for the potency and diversity of its functions, acts as a critical barrier to tumorigenesis. Mutations in p53, particularly within its DNA-binding domain (DBD), compromise its tumor suppressing function in over 40% of human tumors. Diverse p53 mutants adopt three major types of oncogenic effects, namely the loss-of-function effect, dominant-negative effect and gain-of-function effect. However, the conformational mechanisms by which hotspot mutations (e.g., R175H, R273H/C) drive p53 dysfunction remain elusive. Here, we performed microsecond-level molecular dynamics simulations to dissect the structural dynamics of wild-type p53DBD and three oncogenic mutants. In wild-type p53DBD, multi-state conformational switching of the L1 loop was governed by hydrophobic interactions (A119/V122-P278) and an intra-loop hydrogen bond network. Notably, a previously unidentified β-hairpin conformation within the L1 loop was discovered, suggesting a latent regulatory motif. Mutations at R273 disrupted the H2 α-helix integrity, inducing helix-to-coil transitions that destabilized the DNA-binding interface. In contrast, R175H mutation triggered allosteric flexibility in both L2 and L3 loops, distorting the DNA contact surface through synergistic loop rearrangements. Interaction network analysis further revealed that these mutations remodeled non-local residue couplings, with R273H/C primarily destabilizing local interactions and R175H perturbing long-range communication with the LSH motif. Our findings provide structural insights into wild-type p53's complex activities and link mutation-specific conformational shifts to p53's loss/gain-of-function phenotypes, offering new avenues for restoring p53 activity in cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/402840c289bac9dca144ccccf06c354be1c249d6" target='_blank'>
              Structural dynamics of the wild-type p53 DNA-binding domain and hotspot mutants reveal oncogenic conformational shifts.
              </a>
            </td>
          <td>
            Ziqian Zhao, Gang Wang, Xiaoxiao Wu, Zhenyu Qian
          </td>
          <td>2025-06-12</td>
          <td>Physical chemistry chemical physics : PCCP</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Long noncoding RNAs (lncRNAs), non-protein-coding transcripts exceeding 200 nucleotides, are critical regulators of gene expression through chromatin remodeling, transcriptional modulation, and post-transcriptional modifications. While ionizing radiation (IR) induces cellular damage through direct DNA breaks, reactive oxygen species (ROS)-mediated oxidative stress, and bystander effects, the functional involvement of lncRNAs in the radiation response remains incompletely characterized. Here, through genome-wide CRISPR activation (CRISPRa) screening in non-small cell lung cancer (NSCLC) cells, we identified LOC401312 as a novel radiosensitizing lncRNA, the stable overexpression of which significantly enhanced IR sensitivity. Transcriptomic profiling revealed that LOC401312 transcriptionally upregulates carbamoyl-phosphate synthase 1 (CPS1), a mitochondrial enzyme involved in pyrimidine biosynthesis. Notably, CPS1 overexpression recapitulated the radiosensitization phenotype observed with LOC401312 activation. Mechanistic investigations revealed that CPS1 suppresses the phosphorylation of ATM kinase (Ser1981) protein, which is a key mediator of DNA damage checkpoint activation. This study established the LOC401312–CPS1–ATM axis as a previously unrecognized regulatory network governing radiation sensitivity, highlighting the potential of lncRNA-directed metabolic rewiring to impair DNA repair fidelity. Our findings not only expand the functional landscape of lncRNAs in DNA damage response but also provide a therapeutic rationale for targeting the LOC401312–CPS1 axis to improve radiotherapy efficacy in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0498ebb770b2709d819e9dcc309db3bb1e8811f" target='_blank'>
              Long Non-Coding RNA LOC401312 Induces Radiosensitivity Through Upregulation of CPS1 in Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Zhengyue Cao, Tiantian Wang, Fumin Tai, Rui Zhai, Hujie Li, Jingjing Li, Shensi Xiang, Huiying Gao, Xiaofei Zheng, Chang-Yan Li
          </td>
          <td>2025-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Nucleic acid amplification is a pivotal technology for gene investigation in molecular biology as well as an indispensable tool for nucleic acid-based analyses across various disciplines. However, the aerosolized amplicons render the subsequent reaction susceptible to carryover contamination, leading to serious false positive results, particularly in scenarios involving repeated amplification of identical sequences. Although the uracil-DNA glycosylase-based strategy can address this issue for natural DNA amplification, the uracil-containing DNA required for bisulfite conversion-based DNA methylation analysis fails to be analyzed because both aerosol and the intended template can be hydrolyzed. In this study, an innovative strategy for one-pot elimination of carryover contamination is established, applicable to both natural and uracil-containing DNA analyses using either thermal cycle-based or isotherm-based nucleic acid amplification. By incorporation of deoxyinosine triphosphate, hypoxanthine is inserted into amplicons, resulting in a different base composition from the original template. Hypoxanthine, a damaged base as the recognition site of endonuclease V, acts as a label for aerosol cleavage, while the intended template lacking hypoxanthine is unaffected. By systematic optimization, abundant aerosols are thoroughly hydrolyzed within a brief time. Despite the use of deoxyinosine triphosphate as an aberrant deoxyribonucleoside triphosphate, the amplification efficiency, sensitivity, and specificity are insusceptible. In addition, the hypoxanthine-containing amplicons can be analyzed not only for length identification by gel and capillary electrophoresis but also for sequence identification by sanger sequencing, pyrosequencing, and massively parallel sequencing. Moreover, the hypoxanthine-containing amplicons can be further used for cloning and restriction enzyme digestion, indicating the significant potential of this anti-aerosol strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5373512e1ee71db9407b5d72c170aac51f9f8cf9" target='_blank'>
              Elimination of Carryover Contamination during Nucleic Acid Amplification Based on Damaged Base Excision by DNA Repair Enzymes.
              </a>
            </td>
          <td>
            Xiaonan Liu, Wenjing Hu, Huyun Zhou, Jiaxing Zhang, Mengyang Zhao, Yali Cui, Jiangwei Yan
          </td>
          <td>2025-06-03</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Over the last two decades, significant improvements have been made in the understanding of the genomic and biological bases of acute lymphoblastic leukemia (ALL), resulting in enhanced genomic classification, more precise risk stratification and improved long-term outcomes. ALL is a hematologic malignancy defined by uncontrolled proliferation of immature B- or T- lymphoid blasts in the bone marrow, blood, and other extramedullary tissues. It affects most commonly children, representing the most common childhood cancer, but it also occurs in adults where outcome tends to be poorer compared to pediatric patients. A variety of genetic aberrations, including structural and numerical chromosome alterations, translocations generating fusion oncoproteins, cryptic genomic rearrangements, sequence mutations and genomic copy number changes, define multiple genomic subtypes, influence risk stratification and determine response to therapeutic strategies. In this review, we describe the updated genomic classification of ALL highlighting new biological insights and discussing their implications for prognostication and outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9bf200ea2c50518790c91d4e35c6ba5760ccd542" target='_blank'>
              Genomic Subclassification and Risk Stratification of ALL.
              </a>
            </td>
          <td>
            Anna Østergaard, Ilaria Iacobucci
          </td>
          <td>2025-06-30</td>
          <td>Acta haematologica</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumor protein p53 target gene 1 (TP53TG1) is a significant long non-coding RNA (lncRNA) located on human chromosome 7q21.12, exhibiting dysregulated expression in several cancer tissues, and operates both as an oncogene and a tumor suppressor. TP53TG1 modulates gene expression by several mechanisms, including “competitive RNA” activity, competitive miRNA binding, control of critical signaling pathways, and facilitation of epigenetic changes. TP53TG1 affects tumor proliferation, migration, invasion, and apoptosis through intricate mechanisms, modifies tumor cells resistance to therapies, influences the tumor immune microenvironment and metabolic reprogramming, and may present various avenues for clinical therapeutic intervention. Notwithstanding considerable advancements in comprehending the molecular function of TP53TG1, some challenges persist, especially regarding the efficient and targeted administration of TP53TG1-based therapeutics. Advanced RNA delivery technologies, including lipid nanoparticles (LNPs), sphingomyelin-based nanosystems, and polymer-based nanocarriers, have demonstrated the capacity to surmount these obstacles. This review aims to clarify the expression pattern of TP53TG1 in 10 malignant tumors, its biological relevance, its potential as a therapeutic target, and its importance in clinical applications, thereby offering a novel perspective on molecular targeting in cancer. In the future, integrating TP53TG1 with conventional medications may augment efficacy, offer personalized treatments, and eventually increase the prognosis for patients with aggressive or drug-resistant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f226208fa68047280df52857f7f84b24287ce57b" target='_blank'>
              Targeting TP53TG1: a promising prognostic biomarker and therapeutic target for personalized cancer therapy
              </a>
            </td>
          <td>
            Haodong He
          </td>
          <td>2025-07-07</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fc4d33cb171c95693c151dadaf1339c2b7a61c7" target='_blank'>
              SCLC-TumorMiner: A Directly Accessible Genomics Resource for Precision Oncology – Big Data for Small Cells
              </a>
            </td>
          <td>
            Fathi Elloumi, A. Dhall, Daiki Taniyama, Augustin Luna, Yasuhiro Arakawa, Sudhir Varma, Jeffrey Wang, Anisha Tehim, Mark Raffeld, Kenneth Aldape, Christophe E. Redon, Roshan Shresta, William C Reinhold, M. Aladjem, Jaydira Del Rivero, Nitin Roper, Anish Thomas, Y. Pommier
          </td>
          <td>2025-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Abstract Background Small cell lung cancer (SCLC) is a lethal lung malignancy and patients are often diagnosed with distant metastasis. Nearly all patients suffer from disease relapsing with inherent chemoresistance. Lack of targeted SCLC therapies further worsens disease outcomes, making it highly desirable to identify novel and effective therapeutic targets. Methods To search for potential therapeutic targets in SCLC, we analyzed publicly available single-cell and bulk RNA-sequencing (RNA-seq) data from normal, lung adenocarcinoma, and SCLC tumor tissues. To assess the targeting potential of FOXM1, we developed various in vitro models, including DOX-On-shFOXM1 (Tet-ON) inducible stable knockdown systems. Cisplatin resistant human and murine SCLC cell lines were generated to assess the role of FOXM1 in chemotherapy resistance. Immunoblotting, immunohistochemistry (IHC), and immuno-fluorescence were used to analyze the expression of FOXM1 and target proteins. ChIP-assay was used to study protein-gene interactions. Further, multicolor flow cytometry was employed to study the effect of FOXM1 inhibition on human T cells activation and differentiation. Subcutaneous xenograft and SCLC spontaneous (RPM: RB fl/fl ; TP53 fl/fl ; LSL-MYC T58A ) mouse models were used to evaluate the efficacy of FOXM1 inhibitors. Results Single-cell as well as bulk RNA-seq data revealed that FOXM1, an oncogenic transcription factor, is overexpressed in SCLC, and it was recapitulated in human and murine SCLC tissues and cell lines. Interestingly, chemo-resistant (CR) SCLC showed a substantially higher FOXM1 expression than naïve SCLC. Silencing FOXM1 genetically or pharmacologically by FOXM1 inhibitors revealed a marked reduction in cell viability, colony formation, migration and sphere formation in naïve and CR SCLC cells. Moreover, FOXM1 inhibition induced apoptosis and cell cycle arrest in SCLC cells. Furthermore, FOXM1 inhibition in combination with first-line platinum-based chemotherapy showed synergistic anticancer effects in both xenograft and RPM mouse models of SCLC. Our RNA-seq analysis revealed that FOXM1 inhibition altered the Aurora Kinase B (AURKB) signaling pathway, which is dysregulated in SCLC. Moreover, we found FOXM1 inhibition enhanced T cell activation and supported the differentiation of CD8 + cytotoxic T cells, and T cell-mediated killing of cancer cells. Conclusions Our study demonstrates that FOXM1 targeting using small molecule inhibitors has the potential to be a novel therapeutic strategy to combat SCLC progression including chemotherapeutic resistance and reshaping the anti-tumor immune response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cf5514bc8f1dd3db6ac08b065580f4ed8890ea4" target='_blank'>
              FOXM1 targeting alters AURKB activity and reshapes antitumor immunity to curb the progression of small cell lung cancer
              </a>
            </td>
          <td>
            Md Arafat Khan, Parvez Khan, Mahek Fatima, Asad Ur Rehman, Laiba Anwar, Z. Alsafwani, Aatiya Ahmad, Mohammad Ali Abbas Zaidi, Jesse L. Cox, Areem Zahid, Sameer Mohiuddin, Sung Hoon Kim, Juan A. Santamaria-Barria, Imayavaramban Lakshmanan, B. Katzenellenbogen, John A Katzenellenbogen, A. Ganti, Surinder K. Batra, M. Nasser
          </td>
          <td>2025-07-01</td>
          <td>Research Square</td>
          <td>0</td>
          <td>116</td>
        </tr>

        <tr id="
 Background: Combinations of endocrine therapy and CDK4/6 inhibitors (CDK4/6i) used in metastatic breast cancer (MBC) result in heterogenous clinical benefit with subsets of patients having disease control ranging anywhere from a few months to more than 5 years. Understanding the mechanisms that account for these differences can enable new pharmacologic approaches to convert more of these cancers into the long-term control subset. Methods: To elucidate the causal factors that underlie the observed clinical variability, we have utilized multi-omic analyses of patient samples as well as experimental modeling using a diverse cohort of ER+ breast cancers. For clinicogenomic analyses, we analyzed 467 patients with ER+/ HER2- MBC treated with first-line CDK4/6i at our institution (MSK) with detailed clinical follow annotation and follow up. For genomic/transcriptomic studies, we analyzed single cell RNA sequencing data, whole genome sequencing data, and Ki67 score from the neoadjuvant CDK4/6i containing arms in the FELINE trial (NCT02712723). Patient derived xenografts, patient derived organoids, isogenic cell lines were used to define the underlying mechanisms. Live cell imaging assays were performed by FUCCI cell cycle sensor to assess cell cycle transitions. Results: Of 467 patients from MSK cohort, 129 (27.6%) had pre-CDK4/6i loss-of-function variants in TP53, corresponding to short PFS (median PFS= 8.7 months, 95% CI: 6.6–10.3; HR = 2.12, 95% CI: 1.68 to 2.68; p = 2.64e-10). This finding was validated in two large independent clinical cohorts outside MSK. In patient derived and isogenic cell lines, p53 loss had no impact on initial drug response but promoted long-term cell outgrowth and prevented emergence of senescence phenotypes. Following cells in detailed time courses revealed the propensity of p53 mutant but not wild type cells to reenter the cell cycle after initial arrest by CDK4/6 inhibition. Gene expression analysis of RNA seq data revealed a significant positive association between TP53 mutation status and DREAM complex genes, assessed by GSEA (NES= 1.51, p = 1e-4) In models, deletion of DREAM complex component, p130-Rbl2 was found to fully recapitulate the effects of p53 loss: promoting long-term outgrowth from CDK4/6i, blocking therapy induced senescence, and promoting cell cycle re-entry. To validate these results, we analyzed the multi-omic and histologic characteristics of serial samples from the FELINE trial. Compared with WT cases, baseline TP53 LOF had a higher amount of proliferating cells at the end of treatment, measured by Ki67 > 10% (4.5% of WT cases vs. 60% of mutant cases, respectively, OR = 26.8, p = 0.014), the per cell proliferation score, measured by ssGSEA (log2FC: 0.2 vs. 2.8, respectively) as well as a higher pseudobulk expression of DREAM complex related genes (GSEA: NES = 2.0, q = 3.7e-4). Finally, to identify strategies that might facilitate stable cell cycle control of p53KO tumors, we investigated which targets of p53 regulate DREAM and identified p21. Given role of p21 in regulating both CDK4/6 and CDK2 we investigated whether CDK2 inhibitors would provide additional benefit to CDK4/6i. Addition of CDK2i was able to fully restore efficacy in p53KO lines as well as a patient derived model with TP53 mutation enabling irreversible cell cycle arrest and long-term growth suppression. Conclusion: Wild type p53 facilitates DREAM complex assembly in response to CDK4/6i in ER+ breast cancer, underlying long-term disease control. p53 loss enables cells to reenter the cell cycle and facilitates the development of CDK4/6i resistance. Combined inhibition of CDK2 with CDK4/6 represents a promising therapeutic strategy for overcoming loss of p53 and enabling long-term antitumor efficacy.
 Citation Format: Rei Kudo, Anton Safonov, Enrico Moiso, Catherine Jones, Hong Shao, Sharanya Nag, Selma Yeni Yildirim, Edaise M. da Silva, Qing Li, Marie Will, Atsushi Fushimi, Harikrishna Nakshatri, Andrew Koff, Britta Weigelt, Qamar J. Khan, Pedram Razavi, Sarat Chandarlapaty. DREAM complex assembly and stable cell cycle control underlies long-term clinical response to CDK4/6 inhibition [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr PS13-04.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3c291e911332efcfe859d324b671af4bba597c2" target='_blank'>
              Abstract PS13-04: DREAM complex assembly and stable cell cycle control underlies long-term clinical response to CDK4/6 inhibition
              </a>
            </td>
          <td>
            Rei Kudo, A. Safonov, E. Moiso, Catherine Jones, Hong Shao, Sharanya Nag, Selma Yeni Yildirim, E. D. da Silva, Qing Li, Marie Will, A. Fushimi, H. Nakshatri, Andrew Koff, B. Weigelt, Qamar J Khan, P. Razavi, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Cervical cancer is the fourth most common malignant tumor in women, lacking specific and sensitive markers. TP53 induced glycolysis regulatory phosphatase (TIGAR) plays an important role in the initiation and progression of several solid tumors. This study aims to investigate the diagnostic and prognostic and immunotherapeutic role of TIGAR in cervical cancer. We observed that TIGAR expression was upregulated in cervical cancer cells and tissues using experiments and bioinformatics analyses. Moreover, elevated TIGAR expression was associated with poor prognosis using Kaplan–Meier survival analysis, and its diagnostic value was confirmed using the Receiver operating characteristic (ROC) curve. We also analyzed the correlation between TIGAR expression and immune infiltration and assessed its correlation with a clinical cohort of cervical cancer, and found that TIGAR expression was significantly associated with the infiltration levels of Foxp3 + Treg cells, CD11b + MDSC, CTLA4, and HAVCR2. TIGAR is a potential diagnostic and prognostic biomarker of cervical cancer and is expected to offer new possibilities for cervical cancer immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-07201-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91786186f8b506b70f9791c3768dec5816483289" target='_blank'>
              TIGAR regulated by HPV E6 is correlated with disease stage, drug sensitivity, and immune microenvironment in cervical cancer
              </a>
            </td>
          <td>
            Yameng Liu, Yu Zhang, Hailong Zhang, Dianqin Xu, Xinzhu Zhou, Luhong Xie, Hongli Xi, Yurong Zhu, Hanlin Yang, Xiaoyu Zhu, Ji Ren, Shaoju Min, Yujie Tan
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c15cd4c65cc7acb2163700609aefaa2af8b33ee" target='_blank'>
              Integrating bulk and single-cell RNA sequencing reveals intratumor heterogeneity phenotypes and immune infiltration in pancreatic cancer.
              </a>
            </td>
          <td>
            Li Zhao, Yao Huang, Jiang Li, Rui Xie, Kaiming Li
          </td>
          <td>2025-07-13</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Recent pan-cancer transcriptome analysis has revealed differential activity of two alternative PHF19 gene promoters in malignant versus non-malignant gut mucosa. One of these promoters upregulated in colon cancer leads to the expression of the PHF19-207 transcript, suggesting its potential role in tumor promotion. The objective of this study was to investigate the function of PHF19-207 using in silico tools and publicly available data, as well as to assess its expression in colon cancer. Expression analyses were conducted via qPCR and RNA sequencing on RNA extracted from the immortalized colonic epithelial cell line HCEC-1CT, as well as a series of colon cancer cell lines cultured in both 2D and 3D environments. The expression of PHF19-207 was found to be elevated in all malignant cell lines compared to the non-malignant HCEC-1CT cell line in both culture conditions, with the most prominent increase observed in cell lines derived from advanced stages of the disease and in the HCEC-1CT cell line overexpressing KRAS. Furthermore, the PHF19-207 transcript was detected in exosomes derived from malignant cells. These findings suggest that PHF19-207 holds potential as a diagnostic biomarker. In addition, in silico analyses indicate that this transcript may function as a long non-coding RNA involved in the regulation of gene expression. Further functional investigations are required to elucidate its precise role in colon carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f2ed32fe4f53109416f20074827600f8e263969" target='_blank'>
              Transcript PHF19-207 May Be a Long Non-Coding RNA with Tumor-Promoting Role in Colon Cancer
              </a>
            </td>
          <td>
            Dunja Pavlovic, T. Babic, Sofija Ignjatovic, Katarina Pavlovic, S. Dragičević, A. Nikolić
          </td>
          <td>2025-07-02</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/386c1a2fe25cd319031f8c222c2cc077a06d6aae" target='_blank'>
              The long antisense non-coding RNA HOXA transcript at the distal tip (LncRNA HOTTIP) in health and disease: a comprehensive review and in silico analysis.
              </a>
            </td>
          <td>
            Mona G El-Sisi, Sara M. Radwan, Sameh S Ali, Mohamed Y Mostafa, Nadia M. Hamdy
          </td>
          <td>2025-07-05</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Epigenetic modifications prove critical to cancer development, so the disease stands as a major cause of death in people, according to scientific evidence. Environment and cellular alterations affect modifications that differ from genetic mutations because DNA methylation, histone modification, chromatin remodeling, and non-coding RNA activity show reversible potential. This paper thoroughly examines epigenetic methods while demonstrating how their regulatory functions help cancer development without altering DNA sequences. The research reviews how irregular DNA methylation patterns combined with modifiable histone codes and chromatin structural modifications affect tumorous heterogeneity and create drug resistance while preventing immune responses. The document evaluates epigenetic biomarker accuracy for early cancer detection and their forecasting abilities alongside modern developments in anti-epigenetic therapeutic strategies. This article integrates present multi-omics research with translational studies to develop full knowledge about how epigenetic factors define cancer biology for precise medical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e9bf265cc39518378d4e53c6395df1bc723ba17" target='_blank'>
              The Role of Epigenetics in Cancer Development: Investigating the Link between Epigenetic Modifications and Cancer
              </a>
            </td>
          <td>
            Sinigdha Islam, Bibi Ayesa, Dr. Mahmudul Hasan, Md. Jobaer Rahman Rashed, Banasree Roy Urmi, R. Malakar, Shekh Mohammad Mostafa, Aditta Das, Zahidul Mostafa, M. Nesa
          </td>
          <td>2025-06-12</td>
          <td>Journal of Artificial Intelligence General science (JAIGS) ISSN:3006-4023</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Cancer cells exploit epigenetic modifications and post-transcriptional regulations to form oncogenic gene expression networks. However, how these machineries collaboratively orchestrate malignancy remains elusive. One of aberrant epigenetic pathways in cancer is Polycomb repressive complex 1 (PRC)-mediated H2AK119 monoubiquitination (H2AK119ub1) with subsequent silencing of tumor suppressor genes. Despite previous efforts, the biological and clinical significance of PRC1 remains unclear in neuroblastoma (NB), an aggressive sympathoadrenal solid tumor in children. In this study, we demonstrated that knockdown of RING1A, one of the E3 ubiquitin ligases of PRC1, reduced cell viability and enrichment of H2AK119ub1 in NB cells. Transcriptional profiling revealed RING1A-specific targets, whose lower expression was associated with poor outcomes in NB patients. Among these genes, BTG2, a component of the CCR4-NOT polyA deadenylase complex, harbored a hypomethylated CpG island occupied by H2AK119ub1 and accessory proteins of noncanonical PRC1.1 (ncPRC1.1). Biological experiments uncovered that BTG2 suppressed NB cell growth in vitro and inhibited tumor formation in vivo. Moreover, BTG2 perturbed cell cycle progression and selectively destabilized the mRNAs of the cyclin genes CCNA2, CCNB1, and CCNB2. In NB patient cohorts, lower expression of BTG2 was associated with poor outcomes and inversely correlated with those cyclin gene expression. Collectively, we have uncovered a crosstalk between epigenetic modifications and post-transcriptional regulations, in which ncPRC1.1-mediated silencing of BTG2 retains cyclin gene expression and cell proliferation in NB. This study provides new insights into how epigenetic pathways contribute to NB malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38c569ff6ff6e1686d8d62fbb31dac773b0f32f0" target='_blank'>
              Noncanonical PRC1.1 targets BTG2 to retain cyclin gene expression and cell growth in neuroblastoma
              </a>
            </td>
          <td>
            S. Satoh, Mariko Hasegawa, Ryu Okada, Masayuki Haruta, H. Takenobu, Miki Ohira, Takehiko Kamijo
          </td>
          <td>2025-06-03</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Plasmodium parasites, the causative agents of malaria, undergo crucial developments within the mosquito vector, initiated by the formation of male and female gametes. Male gametogenesis involves three rapid rounds of mitosis without nuclear or cell division, followed by a single round of DNA segregation and nuclear division during gamete budding. How the cell organizes the segregation of eight genomes from a single octoploid nucleus into eight haploid gametes is currently unknown. Here we discovered an atypical Arp2/3 complex in Plasmodium important for DNA segregation during male gametogenesis. Unlike the canonical Arp2/3 complex found in other eukaryotes, Plasmodium Arp2/3 localizes to endomitotic spindles and interacts with a kinetochore-associated protein. Disruption of Arp2/3 subunits or actin polymerization interferes with kinetochore–spindle association, causes the formation of subhaploid gametes, and blocks transmission. Our work identified an evolutionary divergent Arp2/3 complex in malaria parasites, provides insights into gametogenesis, and reveals potential targets for transmission-blocking interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a0eb8cd0287d7e4de3c5f68362eedb92788d67" target='_blank'>
              An atypical Arp2/3 complex is required for Plasmodium DNA segregation and malaria transmission
              </a>
            </td>
          <td>
            Franziska Hentzschel, David Jewanski, Yvonne Sokolowski, Pratika Agarwal, Anna Kraeft, Kolja Hildenbrand, Lilian P. Dorner, Mirko Singer, Matthias Marti, Friedrich Frischknecht
          </td>
          <td>2025-06-13</td>
          <td>Nature Microbiology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78da32824622b78fcaf8b80fadf148acaedc9723" target='_blank'>
              m5C-modified circRREB1 promotes lung cancer progression by inducing mitophagy.
              </a>
            </td>
          <td>
            Dunyu Cai, Xingcai Chen, Haotian Xu, Qingyun Zhao, Xiaodong Zhou, Jiaxi Wu, Shengyi Yuan, Yihong Gao, Deqing Li, RuiRui Zhang, Wenyi Peng, Gang Li, Aruo Nan
          </td>
          <td>2025-07-14</td>
          <td>Journal of experimental & clinical cancer research : CR</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Despite advancements in cancer therapy, clinical options remain limited and therapeutic resistance inevitably develops. In melanoma, the second most commonly mutated driver oncogene is neuroblastoma rat sarcoma viral oncogene homolog (NRAS) which remains undruggable by small molecular agents. For those treated with BRAF/MEK combination therapy, in the setting of V600 BRAF mutations, or immunotherapies regardless of somatic genetic make-up, resistance eventually develops in the majority of cases. Developing rational combination therapies, as suggested by preclinical studies involving genetic manipulation of resistance mediators, necessitates a substantial expansion of druggable targets. Novel chemistry strategies, such as chemoproteomics platforms and chemical inducer of proximity (CIP) agents, offer promising solutions. Chemoproteomics enables the rapid identification of ligands to guide medicinal chemistry, while CIP agents alter the expression levels of key proteins. Using oncogene (such as NRAS) and p53 as pivotal resistance mediators, and the lineage-specific transcription factor SRY-box transcription factor 10 [SOX10]) in melanoma as illustrative examples, we demonstrate how these technologies can be leveraged to rapidly expand the druggable target pool and overcome resistance mechanisms, ultimately paving the way for regimens that produce deeper and more durable responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/525c67efe3ba67b1849a4b3d8417e86cfd7d04fc" target='_blank'>
              The Rise of New Targets in Melanoma.
              </a>
            </td>
          <td>
            Xue Bai, Keith T Flaherty
          </td>
          <td>2025-06-26</td>
          <td>The British journal of dermatology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="DNA methylation is the best-known epigenetic mechanism regulating gene expression without altering the DNA sequence. Its counterpart, known as DNA demethylation, is equally important and enables the activation of previously silenced genes. DNA demethylation has attracted interest in the scientific community following the landmark discovery that Ten-Eleven Translocation (TET) proteins can convert 5-methylcytosine to 5-hydroxymethylcytosine. A growing body of research indicates that changes in TET protein levels and 5-hydroxymethylcytosine content are hallmarks of cancer. These epigenetic changes appear to play a critical role in the development of malignancies characterized by high levels of somatic mutations and genetic instability. Bladder cancer is among the most common cancers worldwide and, despite aggressive treatment, remains associated with high mortality and poor prognosis. The lack of reliable diagnostic and prognostic markers poses a significant challenge in its management, highlighting the urgent need for novel biomarkers to enable earlier diagnosis and more accurate prediction of clinical outcomes. This review examines epigenetic alterations associated with bladder cancer and their clinical implications. We focus on the impact of DNA methylation and demethylation on oncogene regulation, summarize scientific evidence supporting their role in bladder cancer development and progression, and briefly explore novel therapeutic strategies targeting those epigenetic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38660186524a060ff11f30a8ae4e9ad5b94b808c" target='_blank'>
              The role of DNA methylation and demethylation in bladder cancer: a focus on therapeutic strategies
              </a>
            </td>
          <td>
            Wiktoria Strasenburg, Jędrzej Borowczak, Daria Piątkowska, J. Jóźwicki, Dariusz Grzanka
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a79b34bae8b1dbcf266da5a700548580b6368ded" target='_blank'>
              Bipartite chromatin recognition by Hop1 from two diverged holozoa
              </a>
            </td>
          <td>
            Alyssa A. Rodriguez, Alessandro E. Cirulli, Katie Chau, Justin Nguyen, Qiaozhen Ye, Kevin D. Corbett
          </td>
          <td>2025-06-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Loss-of-function (LoF) mutations frequently found in human cancers are generally intractable by classical small molecule inhibitor approaches. Among them are mutations affecting polycomb-group (PcG) epigenetic regulators, EZH2 and ASXL1 frequently found in haematological malignancies of myeloid or lymphoid lineage, and their concurrent mutations associates with particularly poor prognosis. While there is clear need to develop novel and effective treatments for these patients, the lack of appropriate disease models and mechanistic insights have significantly hindered the progresses. Here we show that genetic inactivation of Asxl1 and Ezh2 in murine haematopoietic stem/progenitor cells results in highly penetrant haematological malignancies as observed in corresponding human diseases. These PcG proteins regulate both coding and non-coding genomes, leading to marked reactivation of transposable elements (TEs) and DNA damage responses in PcG LoF mutated cells, which create a novel vulnerability for PARP inhibitors (PARPi)-induced synthetic lethality. Using both mouse models and primary patient samples, we demonstrate that Asxl1/Ezh2 mutated cells are highly sensitive to PARPi that induce excessive DNA damage and significantly extend disease latency. Intriguingly the observed PARPi-sensitivity can be specifically overridden by reverse transcriptase inhibitors that interrupt target-site primed reverse transcription (TPRT) and life cycle of TEs. This mechanism is contrastingly different from the current concept of BRCAness associated PARPi-induced synthetic lethality, which largely rely on deficient homologous recombination and is independent on reverse transcriptase inhibitors. Together, this study reveals a novel application and mechanism of PARPi-induced synthetic lethal targeting of blood cancers with reactivated TEs such as those carrying PcG epigenetic mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f5054befc76e369abbf5e3e565cd84241ccf574" target='_blank'>
              Transposable elements as novel therapeutic targets for PARPi-induced synthetic lethality in PcG-mutated blood cancer.
              </a>
            </td>
          <td>
            B. Zeisig, Chiou-Tsun Tsai, Clemence Virely, T. Fung, A. Akin, Estelle Troadec, Bo Jiao, Richard Thompson, Priscilla Nga Ieng Lau, Nanjun Li, Andriani Charalambous, Larissa Bomfim, Jennifer Lynch, Athina Georgiou, Robbert Hoogeboom, Claire Lynn, Si Yi Zhang, P. Patten, Cynthia Fisher, Anna Schuh, Seishi Ogawa, G. Mufti, Mohammad M Karimi, Chi Wai Eric So
          </td>
          <td>2025-06-25</td>
          <td>Blood</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Long noncoding RNAs (lncRNAs) influence the progression, metastasis, and drug resistance of various cancers including ovarian cancer (OC). Putative G-quadruplex (G4)-forming sequences that are abundant in cancer-dysregulated lncRNAs have not been systematically pursued from a structure–function correlation perspective. In this work, we have used a combination of informatics, computational tools, spectroscopy, and molecular biology experiments to identify G4 formation by the OC-dysregulated lncRNAs ERLNC1, DLX6-AS1, LINC01127, FMNL1-DT, and LINP1. The in vitro ability of the lncRNAs to fold into G4 structures was accompanied by interesting profiles of individual G-tract contributions and response to monovalent cations, ligand TMPyP4, and G4-targeting antibody. Human serum albumin (HSA) was found to interact with these G4-forming lncRNAs, albeit with different affinities and structural implications for the G4 motifs. The G4-motif likely plays a crucial role in the binding interactions of select lncRNAs with HSA. This study provides the first systematic study of putative G4-forming sequences in OC-dysregulated lncRNAs and elucidates their interactions with HSA. The interaction of lncRNAs with HSA, possibly facilitated by G4 motifs, can be valuable for OC diagnosis and therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6c83fc756cfd7ff3eaa827a9847f6f909eb7ae0" target='_blank'>
              Putative G‑Quadruplex Structures in Dysregulated Long Non-coding RNA of Ovarian Cancer and Their Binding Interactions with Human Serum Albumin
              </a>
            </td>
          <td>
            Deepshikha Singh, Chinmayee Shukla, Bhaskar Datta
          </td>
          <td>2025-06-11</td>
          <td>ACS Omega</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has recently become a focus of cardiovascular disease (CVD) research as it is possibly involved in disease mechanisms and may function as a biomarker. The literature suggests that eccDNAs may modify gene expression patterns and contribute to the pathogenesis of diseases such as pulmonary arterial hypertension (PAH). Many eccDNAs have been reported to be upregulated in PAH patients with promisingly high diagnostic sensitivity and specificity values. The advent of eccDNA detection from plasma samples using high‐throughput sequencing technologies has opened new avenues for non‐invasive diagnosis. In this review, we focus on the role of eccDNA in CVDs, concentrating on its prospects for use as a biomarker of disease development and stage. However, further studies should be conducted to explicate the functional aspects of eccDNAs in cardiovascular health and disease. This work may eventually lead to the development of novel treatment options and improved clinical outcomes for CVD patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/903ec3642b6c2ce94d9db06e20223312c1b439d8" target='_blank'>
              Extrachromosomal Circular DNA: Emerging Insights Into Cardiovascular Disease Mechanisms and Biomarker Potential
              </a>
            </td>
          <td>
            Saim Mahmood Khan, Jawairya Muhammad Hussain, Surraiya Riaz Mahmood Khan, Bushra Nadeem, Taniyat Naseem, Hammad Jawed, Aina Lashari
          </td>
          <td>2025-06-29</td>
          <td>iLABMED</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BRCA1-associated protein 1 (BAP1) and Enhancer of zeste homolog 2 (EZH2) proteins are associated with the Polycomb repressive complex (PRC), and both have key roles in cell cycle and DNA repair pathways, so variants in BAP1 and/or EZH2 could lead to tumorigenesis. Somatic and germline variants in these genes have been associated with uveal and cutaneous melanoma, mesothelioma, renal cell carcinoma (BAP1 loss-of-function variants), and lymphoma, leukemia, and triple-negative breast cancer (EZH2 gain-of-function variants). Loss of BAP1 function in the murine model has been shown to lead to an overexpression of EZH2 mRNA. Tazemetostat is a selective EZH2 inhibitor currently approved for the treatment of follicular lymphoma in the third line. The effects of tazemetostat have been studied in clinical trials for patients with BAP1 and EZH2 pathogenic variants with malignant mesothelioma and it has shown significant results when used in combination with poly (ADP-ribose) polymerase inhibitors on breast cancer cell lines. The interaction of BAP1 and EZH2 and the efficacy of tazemetostat show great potential for the future of cancer treatment for patients harboring variants in those genes due to the more aggressive nature of those tumors and the lack of targeted treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79d6acaaed33cf0b6b7cca65a93fe16d0a41274" target='_blank'>
              BRCA1-Associated Protein 1 and Enhancer of Zeste Homolog 2: Pathway Interaction and Therapeutic Intervention in Breast Cancer, Mesothelioma, and Lymphoma.
              </a>
            </td>
          <td>
            Elizabeth Casey West, Jessica Evangelista, Annalisa Campanella, F. Lococo, Laura Sun, D. Ivankovic, Christopher L Farrell, Veronica Parker, Luigi Boccuto
          </td>
          <td>2025-06-01</td>
          <td>JCO precision oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Non-coding RNAs (ncRNAs) and the significant roles they play in several diseases have been described and verified in numerous studies. Transfer RNA (tRNA)-derived fragments (tRFs) are a newly discovered class of small ncRNAs produced by mature or precursor tRNAs. In light of the development of RNA sequencing, evidence has shown that tRFs are widely involved in the generation and progression of diseases through a series of mechanisms, including RNA silencing, translational regulation, epigenetic regulation, reverse-transcriptional regulation, and cellular apoptosis. Several studies have determined that tRFs participate in several cancers in a number of ways. Furthermore, novel tRFs may hold significant potential as both diagnostic biomarkers and therapeutic targets in clinical applications. In this review, we discuss the biogenesis and classification of tRFs, illustrate their fundamental functions, and summarize the most recent tRF-related discoveries pertaining to cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0cb37675bb5bb424ff86df17c12d04390d51d71" target='_blank'>
              The Roles of tRNA-Derived Fragments in Cancer: Updates and Perspectives
              </a>
            </td>
          <td>
            Jiamian Geng, Zhaoyuan Sun, Hang Li
          </td>
          <td>2025-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d6212390f670616d04700ae40210a5bbd3973af" target='_blank'>
              Long telomere inheritance through budding yeast sexual cycles
              </a>
            </td>
          <td>
            Vasilisa Sidarava, Sarah Mearns, D. Lydall
          </td>
          <td>2025-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Mutations in PIK3CA are prevalent in advanced colorectal cancer (CRC) and are associated with advanced-stage disease and poor prognosis. However, the prognostic value of these mutations remains unclear. To better understand their role in CRC progression, we evaluated the genomic landscape of PIK3CA mutations (PIK3CAm) in advanced CRC using plasma-based ctDNA from ~17,000 samples (Guardant360 database). Mutation co-occurrence, MSI-H status, TMB scores, and exon-specific mutation frequencies were analyzed. Our study shows that the frequency of PIK3CAm (19.2%) and distribution of exons 9 and 20 are similar to those found previously in the literature. Our results demonstrate PIK3CAm having a low frequency of MSI-H co-occurrence, higher TMB scores, and increased co-occurring alterations in APC, BRAF, EGFR, ERBB2 and KRAS, which may suggest higher genomic instability. Our findings underscore the clinical significance of PIK3CAm in CRC and supports the development of tailored therapeutic strategies targeting this mutation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c39945611065d70241e847cc40e1982a682875" target='_blank'>
              Comprehensive analysis of PIK3CA mutations in advanced colorectal cancer using circulating tumor DNA profiling
              </a>
            </td>
          <td>
            Allante Milsap, J. Tsai, Baqir Jafry, Alex Kim, Syed M Ali Kazmi, Nilesh Verma
          </td>
          <td>2025-07-03</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Background: The discovery of BRCA1 and BRCA2 genes in 1994 and 1995 led to increased focus on the homologous recombination (HR) repair pathway in cancer research and therapy. Preclinical studies in 2005 demonstrated that poly ADP-ribose polymerase (PARP) inhibitors selectively target BRCA-deficient cells. Numerous phase III clinical trials have shown that PARP inhibitors can improve treatment outcomes in both early and advanced germline BRCA1/2 (gBRCA1/2)-mutant breast cancer, as well as achieve a better quality of life compared to cytotoxic chemotherapy. However, the prevalence of gBRCA1/2 mutations is estimated to be only 2-5% in the unselected general population. Therefore, it's important to identify patients beyond gBRCA1/2 carriers whose cancers may be sensitive to PARP inhibition. Studies in prostate and ovarian cancer have suggested that some patients with somatic BRCA1/2 (sBRCA1/2) mutations or mutations in HR-related genes other than BRCA1/2 may benefit from PARP inhibitors. Among HR-related genes, PALB2 is responsible for loading RAD51 onto ssDNA, stimulating RAD51-mediated strand exchange and D-loop formation via the BRCA complex (BRCA1-PALB2-BRCA2-RAD51). Germline PALB2 mutation is estimated to be present in about 1% of breast cancer patient populations and is associated with an increased risk of breast and pancreatic cancer. The epidemiology of breast cancer differs between Taiwanese (ethnically Han Chinese origin) and Caucasian populations. Taiwanese breast cancer patients have a younger median age of disease onset and carry a greater risk for disease progression and shorter interval to secondary contralateral breast cancer than Western women. As the early onset and bilaterality of breast cancer are more likely to be related to genetic predisposing factors, it is important to identify potential genetic alterations underpinning Taiwanese patients. Methods: This study aimed to assess the prevalence of BRCA1, BRCA2, and PALB2 mutations in Taiwanese breast cancer patients using targeted sequencing on tumor-only samples. The Institutional Review Board of Taipei Veterans General Hospital approved the study (protocol number: 2018-09-007A). Written informed consent was obtained from all participants prior to enrollment. A total of 879 consecutive breast cancer patients, representing 924 assays, were enrolled in the study. Patients were assigned to different groups based on their treatment plans: Group 1 [planned to receive surgery as the first-line treatment and followed by adjuvant therapy], Group 2 [planned to receive neoadjuvant therapy as the first-line treatment and followed by surgery], and Group 3 [diagnosed with de novo and treatment naive stage IV breast cancer, or stage IV breast cancer with recurrence beyond three years after surgery]. Formalin-fixed paraffin-embedded specimens were assayed using the Ion Torrent Oncomine™ Comprehensive Assay v3 (Thermo Fisher Scientific, Waltham, MA), enabling the detection of 161 cancer-related genes and identification of single nucleotide variants (SNVs), copy number variations (CNVs), gene fusions, and indels. Variants were filtered with the Oncomine™ Knowledgebase Reporter and the online VariED tool to filter out Taiwan Biobank polymorphisms. The mutational consequences of filtered variants were determined using the ClinVAR database, OncoKB™, and SNPnexus. Reflex germline testing was conducted for patients with pathogenic or likely pathogenic variants using whole exome sequencing (WES) or whole genome sequencing (WGS) through blood sample collection. To distinguish germline from somatic mutations with tumor-only sequencing, algorithms including the LOH-germline inference calculator (LOHGIC) and the somatic-germline-zygosity (SGZ) method were employed as alternatives for patients not ready for germline testing. Results: Of the 924 assays, 281 were positive for mutant BRCA1, BRCA2 and PALB2 in 130 patients. These mutations impacted 27 patients (3.1%) for BRCA1, 76 patients (8.6%) for BRCA2, and 46 patients (5.2%) for PALB2. In total, genetic alterations were noted in 14.8% (130 patients) of the cohort. The BRCA1 mutation cohort was associated with a higher proportion of advanced stages compared to those without. The BRCA2-mutant patients showed a higher incidence of family history of ovarian cancer, resulting in a significant difference in the number of mutant patients with a family history of ovarian cancer. In terms of IHC phenotypes, 13 (2.3%) of the BRCA1 mutant breast cancers were HR+/HER2−, 3 (3.3%) were HR+/HER2+, 2 (2.6%) were HR−/HER2+, and 9 (6.9%) were HR−/HER2−. For BRCA2 mutated cases, 51 (9.0%) were HR+/HER2−, 9 (9.9%) were HR+/HER2+, 6 (7.9%) were HR−/HER2+, and 9 (6.9%) were HR-/HER2-. Among the PALB2 mutated patients, 30 (5.3%) were HR+/HER2−, 4 (4.4%) were HR+/HER2+, 3 (3.9%) were HR−/HER2+, and 9 (6.9%) were HR−/HER2−. Co-occurrence was found between BRCA1/2 in 13 samples (log2 odds ratio: > 3, p-value < 0.001, and q-value < 0.001), BRCA1 and PALB2 in 8 samples (log2 odds ratio: > 3, p-value < 0.001, and q-value < 0.001), and BRCA2 and PALB2 in 8 samples (log2 odds ratio: 2.401, p-value < 0.001, and q-value < 0.001). Five patients had both BRCA1/2 mutations, 1 had both BRCA2 and PALB2 mutations, none had both BRCA1 and PALB2 mutations, and 7 had all three mutations. The study analyzed various genetic variants and identified 176 amino acid (AA) changes. There were four variants that did not notice any AA change, and three novel splice site variants (BRCA1 c.5256+1G > A, BRCA1 c.5215+1G > A, and BRCA2 c.-38-3CAG > C) were identified. Notably, 60.2% (106) of the discovered AA changes were not documented in either ClinVAR or the Oncomine™ Knowledge database. Using the OncoKB™ for annotation, 171 (97.2%) AA changes were found to have clinical implications. The most common mutations were p.K654fs (3 cases) for BRCA1, p.N372H (26 cases), p.S2186fs (5 cases), p.V2466A (5 cases), and p.X159_splice (5 cases) for BRCA2, and p.I887fs (30 cases) for PALB2. It should be noted that although p.N372H was observed 26 times in BRCA2-mutated assays, it has been confirmed to be a benign variant. Reflex germline testing was conducted for patients until November 20, 2023. Specifically, 48 cases (36.9%) were identified with pathogenic or likely pathogenic variants via tumor-targeted sequencing, as classified by the ClinVar database, and were subsequently recalled for further investigation through WGS or WES. Among 130 cases examined, 7 cases (5.4%) completed WGS uncovering crucial genetic variations. None harbored germline mutations in BRCA1/2 and PALB2. Another 20 cases (15.4%) were reached and 9 had completed WES (6.9%). The study group encountered enormous challenges including loss of follow-up and 14 were deceased. The reflex germline testing results showed that three variants (BRCA1 c.1969_1970del, BRCA1 c.3629_3630del, BRCA2 c.8755-1G > C) were classified as Pathogenic/Likely pathogenic (P/LP) by ClinVar and as likely loss-of-function or likely oncogenic by OncoKB. One variant (PALB2 c.448C > T) was not listed in ClinVar, but OncoKB annotated it as likely loss-of-function or likely oncogenic. Additionally, there were 2 cases (1.5%) of germline mutations with uncertain significance, and 5 cases (3.8%) with benign germline alterations, emphasizing the genetic intricacy involved in breast cancer development. Using alternative methods as the results of LOHGIC and SGZ analyses for determining germline and somatic mutations, among the 281 samples, 40 were identified as germline mutations, and 169 samples (60.1%) were of somatic origin. Borderline cases comprised 26 samples (9.3%), and 46 samples (16.4%) were unclassifiable due to missing data on tumor purity. Discussion: This study presents one of the largest cohorts of breast cancer patients with BRCA1, BRCA2, and PALB2 mutations detected through tumor-only sequencing in Taiwan. The high detection rate of breast cancer susceptibility genes could lead to more patients undergoing germline testing and receiving appropriate treatments. The study revealed a significant tendency of co-occurrence between BRCA1/2, BRCA1-PALB2, and BRCA2-PALB2 mutations. This finding may have implications for understanding breast carcinogenesis and creating genetic panels for predicting and prognosing hereditary breast cancer. Tumor-only targeted sequencing has gained increasing attention due to its ability to reveal pathogenic/likely pathogenic variants in genes associated with cancer predisposition and potential therapeutics with a higher level of coverage. The Dana-Farber/Harvard Cancer Center study found that over half (52.9%) of the tumor-identified P/LP patients did not meet any personal or family history criteria for clinical genetic testing (CGT). These results show the potential of tumor-only sequencing in detecting P/LP mutations in cancer predisposition genes across malignancies. Both germline and somatic alterations can affect treatment decisions and outcomes. The results of the TBCRC-048 and TBB trials suggested further exploration of PARP inhibitors in metastatic or advanced breast cancers with HR-associated mutations beyond BRCA1 and BRCA2. Identifying additional biomarkers to expand this treatment in somatic BRCA1/2-mutant or HR-related-gene-mutant advanced breast or ovarian cancers could significantly benefit patients who would otherwise receive chemotherapies as the only regimen. The significance of reflex germline testing cannot be overemphasized. However, the extent of reflex testing conducted to date is still limited. This has led to the exploration of alternative methods to determine whether the reported mutation is of germline or somatic origin. The refined LOHGIC and SGZ methodologies are designed to assess three crucial factors: tumor purity, allele frequency, and confirmation of a diploid genome. Although these algorithms are useful, they have their limitations; in an analysis of 130 cases, 10 could not be co">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9870aa3f5a2b8b41daa66289bc79b03e6830f70" target='_blank'>
              Abstract P5-04-26: Prevalence of BRCA1, BRCA2, and PALB2 genomic alterations among 924 Taiwanese breast cancer assays with tumor-only targeted sequencing: extended data analysis from the VGH-TAYLOR study
              </a>
            </td>
          <td>
            Han-Fang Cheng, Y. Tsai, Chun-Yu Liu, Chih-Yi Hsu, Pei-Ju Lien, Yen-Shu Lin, Ta-Chung Chao, Jiun-I. Lai, Chin-Jung Feng, Yen-Jen Chen, Bo Chen, Jen-Hwey Chiu, Ling-Ming Tseng, Chi-Cheng Huang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background Prostate cancer(PCa) ranks among the most frequently diagnosed malignancies in men. The progression and heterogeneity of tumors pose significant challenges to clinical prognosis and treatment strategies. Recently, extrachromosomal DNA(ecDNA) has emerged as a critical player in cancer biology, influencing tumor progression, metastasis, and resistance to therapy. Oncogenes and regulatory sequences carried on ecDNA(ecDNA genes) can significantly alter the biological characteristics of tumors and their clinical outcomes. Methods In this study, we obtained ecDNA genes specifically expressed in PCa from the ECGA database. To construct a prognostic risk model for Biochemical Recurrence-Free Survival (BRFS), the two most common types of ecDNA genes which are protein-coding genes and long non-coding RNAs, were analyzed using Cox regression and LASSO regression techniques. Through KEGG/GO pathway enrichment analysis, we identified relevant pathways and analyzed the immune cell infiltration status. Functional assays, such as colony formation, CCK-8, migration, and invasion assays, were employed to assess the cellular functions of a key lncRNA AC016394.2. Results Our analysis identified six key ecDNA lncRNAs(ec-lncRNAs), including the ec-lncRNA AC016394.2, with significant prognostic value in PCa. By employing our risk scoring model, patients were classified into high-risk and low-risk groups, revealing significant differences in their BRFS outcomes. The model demonstrated strong predictive accuracy and clinical relevance. The 1/3/5-year AUC of the model is close to 0.8, which is higher than most common clinical indicators such as Gleason score and TM staging. KEGG and GO pathway enrichment analyses revealed that the high-risk group was predominantly enriched in immune-related pathways. Additionally, immune cell infiltration analysis demonstrated notable differences in the distribution of specific immune cell populations between the high-risk and low-risk groups. Knockdown of AC016394.2 inhibited PCa cell proliferation, migration, and invasion. Conclusions This study presents a novel ecDNA gene-based prognostic risk model for PCa, highlighting the functional importance of ec-lncRNA AC016394.2. These findings offer valuable insights into the biological role of ec-lncRNAs, highlighting their potential as targets for precision oncology and therapeutic intervention. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03886-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19ee0338b68e0a302760ee35e3fa258d092e1aae" target='_blank'>
              An EcDNA gene-based risk model and functional verification of a key ec-lncRNA AC016394.2 for prostate cancer
              </a>
            </td>
          <td>
            Jiangping Qiu, Jiang Wu, Nan Zhou, Cong Lai, Xin Huang, Cheng Liu, Xiaoqing Yuan, Kewei Xu
          </td>
          <td>2025-07-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="A key hallmark of cancer tumorigenesis is the maintenance of telomere length, which occurs canonically through the reactivation of telomerase. Alternative lengthening of telomeres (ALT) is an atypical, non-canonical telomere maintenance mechanism that uses homologous recombination (HR) to maintain telomere length and is associated with replication stress and defects in genome maintenance. In preclinical models, ALT positivity (ALT+) sensitizes tumor cells to ataxia telangiectasia and Rad3-related (ATR) inhibitors. Camonsertib is a novel potent, and highly selective ATR inhibitor that is synthetic lethal with genomic alterations affecting HR and DNA damage response (DDR). Here we describe a case of confirmed clinical and molecular response to pharmacological ATR inhibition through camonsertib, in a patient with ALT+ metastatic melanoma. To our knowledge, this is the first clinical report of synthetic lethal targeting of a confirmed ALT+ tumor with an ATR inhibitor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4c42dfab8a81e858900cd580157a32f2bb58ea7" target='_blank'>
              Exceptional response to the ATR inhibitor, camonsertib, in a patient with ALT+ metastatic melanoma
              </a>
            </td>
          <td>
            Natalie Y L Ngoi, Ian M. Silverman, Adrienne Johnson, Chenfeng Meng, J. Schonhoft, Michal Zimmermann, Danielle Ulanet, Hyeyeon Kim, Carlos Torrado, Carolina Salguero, Christian Valladolid Brown, J. Rodón, V. Rimkunas, Maria Koehler, T.A. Yap
          </td>
          <td>2025-07-07</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df417637b67eb31555739f374729e00085d2dca3" target='_blank'>
              CRISPR RiPCA for Investigating eIF4E-m7GpppX Capped mRNA Interactions
              </a>
            </td>
          <td>
            Gabriela Vega-Hernández, Jesse Duque, Brandon J. C. Klein, Dalia M. Soueid, Jason Rech, Hui Wang, Wenhui Zhou, Amanda L. Garner
          </td>
          <td>2025-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract The arms race between phages and bacteria is dynamic and ongoing, with both continuously acquiring new strategies to outcompete each other during co-evolution. Here, we report bacteriophage T4 exonuclease DexA and an uncharacterized Escherichia coli exonuclease as a rare pair of attack and defense duo arising from the same mechanism. DexA, highly conserved among phages, has two well-characterized biological roles: host DNA scavenging and intron homing. Unmodified DNA is the substrate during host DNA scavenging, whereas cleavage of 5hmC (5-hydroxymethylcytosine)-modified phage DNA is required for intron homing. We reveal a temperature-driven quaternary fold switch between DexA dimer and tetramer that facilitates cleavage of distinct DNA forms, namely 5hmC-modified phage DNA and unmodified host DNA. As a countermeasure, bacteria produce DexA variants for defense against phage that only targets 5hmC-modified DNA. Thus, both phages and bacteria compete using HmC-Recognizing EXonuclease strategies (designated as HREX).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2390a6250e0b17221d839a68a53197524485a484" target='_blank'>
              A temperature-driven DNA discrimination strategy to distinguish E. coli DNA and phage 5hmC-modified DNA
              </a>
            </td>
          <td>
            Yue Kang, Yahui Liu, Haolong Zhou, Biyun Ma, Huan Chen, Kaining Zhang, Yawen Wang, Chengpeng Fan, Huaiyu Yang, Yingqi Xu, Steve Matthews, Shuai Yuan, Yan Li, Bing Liu
          </td>
          <td>2025-06-06</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="All biological processes, from embryonic development to cancer, are tightly controlled by the interactions between genetics and epigenetics. An array of epigenetic modifications, such as DNA methylation, histone/chromatin modifications, and noncoding RNA-mediated targeting, are essential to regulate the heritable changes that occur during multiple cellular processes. A failure in proper regulation results in inappropriate gene expression that ultimately leads to pathological states. Groundbreaking advances in genomics and transcriptomics have revealed the potential involvement of epigenetics in various physiological and pathological states. The promising clinical and preclinical results shown by epigenetics drugs further underscore the central role of epigenetics in multiple human diseases, including cancer. AT rich interaction domain (ARID)-containing proteins are a family of evolutionarily conserved DNA binding proteins that regulate epigenetic modifications. Genome sequencing has revealed the existence of 15 ARID family proteins that are divided into 7 subfamilies based on their sequence and domain homology. Although the ARID family of proteins are implicated in cell growth, development, differentiation, and cancer, the diverse biological functions of many family members remain to be elucidated. Here, we focus on ARID4B to summarize its prominent role in embryonic stem cell differentiation and human malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72fba835b7a96d97fab7eb6f68f890cb10aeda31" target='_blank'>
              ARID4B: An Orchestrator from Stem Cell Fate to Carcinogenesis
              </a>
            </td>
          <td>
            R. R. K. Kandy, M. Arumugam, Mukesh Pratap Yadav, Bibhuti Bhusan Mishra, Jyotika Sharma
          </td>
          <td>2025-06-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9c3ded805682d4eff5c7bc4d110a3fd55426d2b" target='_blank'>
              Selective Sensitivity of Ph-like B-ALL to BRG1 Inhibition Reveals a Novel Targeted Therapy Strategy
              </a>
            </td>
          <td>
            V. S. S. A. Ayyadevara, Shikha Gaur, Ashley Paik, Ria Perencsik, Monika M. Toma, Gerald Wertheim, H. Geng, Tomasz Skorski, Srividya Swaminathan, C. Hurtz
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ccadc7a645959f100047bcb5bd2dda2c9316267" target='_blank'>
              Genomic determinants of therapy response in ETV6::RUNX1 leukemia.
              </a>
            </td>
          <td>
            L. Oksa, Sanni Moisio, Khurram Maqbool, Roger Kramer, A. Nikkilä, Buddika Jayasingha, Artturi Mäkinen, H. Foroughi-Asl, Samuli Rounioja, Janne Suhonen, O. Krali, Miikka Voutilainen, M. Lahnalampi, Kaisa Vepsäläinen, Sui Huang, J. Duque-Afonso, Julia Hauer, J. Nordlund, V. Wirta, O. Lohi, M. Heinäniemi
          </td>
          <td>2025-07-09</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3ebf60acd25684c97159468164978486a64f35c" target='_blank'>
              Early transcriptional signatures of MeCP2 positive and negative cells in Rett syndrome
              </a>
            </td>
          <td>
            Yan Li, Ashley G. Anderson, Guangtong Qi, Sih-Rong Wu, J. Revelli, Zhandong Liu, Huda Y. Zoghbi
          </td>
          <td>2025-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The study of DNA tumor viruses has revolutionized cancer biology, partly by virtue of the unique capabilities of viral oncoproteins to manipulate key proteins and pathways involved in tumorigenesis. We find a high affinity and selective binding of the polyoma small T antigen (PyST) with the transcription cofactor TAZ. By engineering a degradable version of PyST, we demonstrate that, when TAZ activity is modulated by PyST, a surprisingly small number of genes have altered expression and thus are candidate transcription targets of TAZ. Notably, knocking out TAZ, or its target genes CTGF or CYR61, abolishes the growth-promoting properties of PyST that are evident upon growth factor withdrawal. Therefore, by controlling the protein abundance of PyST and consequently TAZ activity, we find that TAZ is a transcriptional coactivator that can achieve important biological effects by acting on a limited number of gene targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a4dcc37489aa9a0942107e458f9ebbf6f6693a5" target='_blank'>
              A degradable form of polyoma small T antigen reveals the high specificity of TAZ in regulating gene expression.
              </a>
            </td>
          <td>
            Yubao Wang, C. Manokaran, Kevin Huang, Brian Schaffhausen, Thomas M Roberts
          </td>
          <td>2025-07-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Micropeptides are endogenous peptides translated from alternative open reading frames (alt-ORFs) within coding or non-coding genes. Emerging evidence suggests that some micropeptides play critical roles in both physiological and pathological processes. Multiple myeloma (MM), a haematological malignancy, remains incurable due to frequent relapses and a limited understanding of its underlying mechanisms. In this study, we sought to investigate the function and molecular mechanism of a novel micropeptide in MM pathogenesis. We identified a novel micropeptide, altH19, encoded by the lncRNA H19, which is highly expressed in patients of MM. Functional assays revealed that altH19 promotes myeloma cell proliferation and colony formation significantly. Furthermore, altH19 induces multipolar mitosis by upregulating the expression of Aurora B, Centrin 2 and phosphorylated histone H3. Flow cytometry analyses confirmed that overexpression of altH19 enhances DNA replication and accelerates the transition from early to mid-late stages of the DNA replication process. Conversely, knockout of altH19 reverses these effects. Mechanistically, altH19 directly interacts with phosphorylated CDK2 at threonine 160, thereby enhancing CDK2 T160 phosphorylation and activating the downstream E2F1 target RB phosphorylation. Notably, altH19 was able to restore phosphorylation levels of CDK2 and RB that were otherwise suppressed by the CDK2-selective inhibitor Seliciclib. In summary, we identify altH19 as a novel lncRNA-derived micropeptide with a pivotal role in myeloma progression, highlighting the therapeutic potential of targeting the altH19-CDK2-RB axis in MM treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/790582f1ede9cdc6910628de9fe99a0b37b482f7" target='_blank'>
              LncRNA H19-Encoded Micropeptide altH19 Promotes DNA Replication and Mitosis in Myeloma Cells by Enhancing the Phosphorylation of CDK2 at Threonine 160.
              </a>
            </td>
          <td>
            Yaxin Zhang, Wenjing Li, Xu Cao, Jiwei Mao, Xiaodan Zhou, Linlin Liu, Ruosi Yao
          </td>
          <td>2025-06-27</td>
          <td>Cell proliferation</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Introduction: Cancer cells continue to replicate their DNA and survive mainly due to their unique ability to repair damaged DNA using alternate DNA repair pathways. For example, cancer cells with a deficiency in homologous recombination (HR) proteins (such as BRCA1) can repair their DNA by either relying on other highly expressed HR-related proteins (such as RAD51 or PARP1) or by using backup DNA repair mechanisms such as ALT-NHEJ. Alterations in DNA repair pathways commonly occur during breast cancer (BC) progression. For example, TNBCs have dysfunctional BRCA1/2 but express high levels of RAD51. Further, ER+BC employs ALT- NHEJ, HR, or BER to repair their DNA. The objective of this study was to identify FDA-approved non-cancer drug/s capable of inhibiting DNA repair in BC cells, thereby inhibiting their growth, with potential clinical benefits for BC patients. Methods: BC cells were treated with vehicle and FDA-approved drugs for 72 hours and were subjected to cell-titer Glo assay. The anti-tumor effect of imipramine alone and in combination with Olaparib and tamoxifen was validated using orthotopic xenograft mouse models. The effect of imipramine on DNA repair was determined by immunofluorescence using antibody against 53BP1, and functional DNA repair assays. Based on these results, a window of opportunity clinical trial was conducted to test the efficacy of imipramine in early-stage breast cancer patients. After having a breast core needle biopsy, 15 eligible patients with stage I-III breast cancer were enrolled in the trial and were treated with imipramine at a target dose of 200 mg PO daily for an average of 28 days. Patients were evaluated on day 7, day 14, day 21, and at the end of treatment for toxicity. The primary endpoint for the trial was the absolute change in the Ki67. IHC using an antibody against Ki67 was performed on core biopsy (pre-treatment) and tumor tissue after imipramine treatment. The secondary objectives of this trial were to further define the toxicity profile of imipramine. Results: Imipramine treatment significantly reduced the viability of TNBC and ER+ BC cells. Further, imipramine treatment inhibited the migration and invasion of BC cells. Systemic delivery of imipramine suppressed the growth of BC. Importantly, imipramine blocked the DNA repair capacity of BC cells by inhibiting the expression of DNA repair proteins including FOXM1 and RAD51. Notably, imipramine treatment improved the efficacy of Olapraib in TNBC and sensitized the tamoxifen response in endocrine-resistant ER+ BC cells. The clinical trial on 15 patients treated with imipramine showed a marked reduction in Ki67-positive cells in post-surgical tumor tissues compared to core needle biopsy tissues. Toxicity was mild with only grade 1 and 2 toxicities that included some instances of dizziness and nausea. There was no dose reduction, interruption, or treatment discontinuation. Discussion: Our preclinical and clinical studies showed that imipramine can block DNA repair in both TNBC and HR+ breast cancer. Short-term treatment with imipramine can effectively decrease Ki67 in breast cancer patients. Future clinical trials will involve combining imipramine with other regimens such as Olaparib for TNBC patients and elacestrant/CDK4/6 inhibitor for therapy-resistant ER+ breast cancer patients.
 Citation Format: Manjeet Rao, Arhan Rao, Santosh Timilsina, Subapriya Rajamanickam, Panneerdoss Subbarayalu, Ismail Jatoi, Yidong Chen, Kate Lathrop, Ratna Vadlamudi, Virginia Kaklamani. Imipramine: A Promising Therapeutic Regimen for Breast Cancer Patients [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-08-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/896ae671f03f0b00708634d7ce87a54ad2b6ba0a" target='_blank'>
              Abstract P3-08-26: Imipramine: A Promising Therapeutic Regimen for Breast Cancer Patients
              </a>
            </td>
          <td>
            M. Rao, Arhan Rao, Santosh Timilsina, Subapriya Rajamanickam, Panneerdoss Subbarayalu, Ismail Jatoi, Yidong Chen, Kate Lathrop, Ratna K Vadlamudi, Virginia Kaklamani
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Hypopharyngeal carcinoma is a highly aggressive malignancy in the head and neck region with poor prognosis due to challenges in early diagnosis, high invasiveness, recurrence rate, and metastatic potential. Small non-coding RNAs (sncRNAs) play crucial roles in tumorigenesis and progression and hold potential as clinical diagnostic biomarkers and therapeutic targets. However, the ability of traditional RNA-sequencing technologies to detect modified sncRNAs is limited, potentially leading to the failure to accurately identify some functionally relevant sncRNAs. In this study, we employed PANDORA-seq technology for the first time to systematically profile sncRNA expression in cancerous and adjacent normal tissues from five patients with hypopharyngeal carcinoma. Our results revealed dynamic changes in sncRNA expression in hypopharyngeal carcinoma tissues and found 4798 significantly differentially expressed sncRNAs. Among these, differentially expressed miRNAs and tsRNAs were primarily involved in key signaling pathways, including MAPK, FoxO, and TGF-β. Additionally, we validated the differential expression of eight sncRNAs in hypopharyngeal carcinoma tissues, which may represent potential diagnostic biomarkers and therapeutic targets. This study lays the foundation for the application of PANDORA-seq technology in human cancers and offers new directions for exploring the underlying molecular mechanisms of hypopharyngeal carcinoma and potential targets for its clinical diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dd8cc6530b05785d0d4d3e2bfeefa62e349e5c7" target='_blank'>
              PANDORA-Seq Unveils the Hidden Small Non-Coding RNA Landscape in Hypopharyngeal Carcinoma
              </a>
            </td>
          <td>
            Miaoyan Pu, Luyu Shi, Haiyu Ma, Chuntao Tao, Ying Zhang, Youquan Bu, Junhong Ye
          </td>
          <td>2025-06-21</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [4],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>