<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-10-13 09:31:34 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>39</td>
          <td>105</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>83</td>
          <td>138</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>115</td>
          <td>146</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>365</td>
          <td>138</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>320</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="In tumors, extrachromosomal DNA (ecDNA) is an important driver of oncogene expression, genomic instability, the evolution of drug resistance, and poor patient prognosis. ecDNA is present in various tumors but is rarely found in normal cells. Here, we provide a detailed review of the structure, genetics, occurrence, outcomes, and functions of ecDNA, offering further reference for research on ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854e01c4800ec9b678143d63d0ad28cdee5f8010" target='_blank'>
              The role of extrachromosomal DNA in tumorigenesis and progression
              </a>
            </td>
          <td>
            Xiaoyang Ma, Xiaolin Yu, Chuan Wu, Lixing Song
          </td>
          <td>2025-09-15</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Extrachromosomal circular DNA (eccDNA) is an emerging class of genetic material that exists outside of the chromosomal genome. These circular DNA molecules are gaining increasing attention as important biomarkers in various cancers because of their roles in gene amplification, genetic heterogeneity, and drug resistance. In this review, we explore in depth the impacts of eccDNAs on cancer biology, their potential to predict treatment sensitivity and resistance, and their involvement in the development of new anticancer therapies. eccDNAs can be used as biomarkers for various tumor types to help diagnose and predict prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1f8c896f37f0d86135daabcae627eb40e4e88c" target='_blank'>
              Extrachromosomal Circular DNA as a Cancer Biomarker: From Diagnosis to Treatment
              </a>
            </td>
          <td>
            Hexin Li, Jiahui Cai, Gang Zhao, Lihui Zou
          </td>
          <td>2025-09-08</td>
          <td>Cancer Innovation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6f51d8b7476391e7b48e8b1303fc52862802717" target='_blank'>
              Comparative Analysis of CHEK1, Extrachromosomal DNA Dynamics and 1p/19q Status in Lower Grade and High Grade Glioma
              </a>
            </td>
          <td>
            MD Steven Lehrer, MD JD MS Peter Rheinstein
          </td>
          <td>2025-10-05</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Extrachromosomal circular DNA (eccDNA) refers to small, circular DNA molecules that originate from chromosomal sequences and are prevalent across nearly all eukaryotic organisms. In humans, eccDNAs are widely distributed in normal tissues, cancerous tissues, and body fluids, where they play important roles in tumorigenesis and are often associated with poor clinical outcomes. Given their biological and clinical significance, a well-integrated and high-quality database is essential for advancing eccDNA-related research. To address this need, we developed eccDNABase, a comprehensive and curated resource for browsing, searching, and analyzing eccDNAs across multiple species. The database systematically catalogs eccDNA–disease associations from diverse tissues and organisms. Currently, eccDNABase contains 1,875,452 eccDNA–disease associations, encompassing 8,398 ecDNA entries across nine species, 63 diseases, and healthy individuals. Each entry provides detailed information, including eccDNA ID, type, chromosomal localization, species, tissue or cell line source, disease name and Disease Ontology ID, overlap length and percentage with genes, oncogene overlap, detection method, and links to literature and source databases. Given its extensive and curated datasets, eccDNABase serves as a valuable resource for both basic and translational research, offering deeper insights into the role of eccDNA in health and disease. The database is publicly accessible at http://cgga.org.cn/eccDNABase/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf15dcf8d0c3a1e284faf711bb5268a1015b46d0" target='_blank'>
              eccDNABase: A Comprehensive and High-Quality Database for Extrachromosomal Circular DNA
              </a>
            </td>
          <td>
            Jin-Hao Zhang, Chang-Lin Yang, Chang-Yuan Ren, Ji Shi, Han-Xiao Zhou, Ying Zhang, Zhi-Yan Sun, Tao Jiang, Zheng Zhao
          </td>
          <td>2025-09-13</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/127ba85daf25e695241e6cc43a8f791c28684c9d" target='_blank'>
              DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, E. Larose Cadieux, Nana E. Mensah, Xiaohong Liu, Kezhong Chen, O. Chervova, Takahiro Karasaki, C.E. Weeden, Corentin Richard, Siqi Lai, Carlos Martínez-Ruiz, E. Lim, A. Frankell, T. Watkins, Georgia Stavrou, Ieva Usaite, Wei-Ting Lu, Daniele Marinelli, S. Saghafinia, G. Wilson, P. Dhami, H. Vaikkinen, Jonathan Steif, S. Veeriah, R. Hynds, Martin Hirst, C. Hiley, A. Feber, Özgen Deniz, M. Jamal-Hanjani, N. Mcgranahan, Stephan Beck, J. Demeulemeester, M. Tanić, C. Swanton, P. Van Loo, N. Kanu
          </td>
          <td>2025-09-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Abstract Background Extrachromosomal circular DNA (eccDNA) has emerged as a critical driver of oncogenesis, yet its functional roles in high‐grade serous ovarian cancer (HGSOC) remain poorly characterized. This highlights the need for comprehensive investigations into the abundance, biogenesis, and functional implications of eccDNA in HGSOC. Methods To characterize eccDNA in HGSOC, we performed comprehensive Circle‐seq analysis to assess eccDNA abundance and genomic annotation in HGSOC tissues compared to normal ovarian tissue. For mechanistic validation of eccDNA biogenesis pathways, targeted knockdown experiments of microhomology‐mediated end‐joining (MMEJ) dependent on LIG3 and POLQ were conducted. Functional characterization of HGSOC‐specific eccDNA‐harboring precursor microRNAs (eccMIRs) included in vitro assays using HGSOC cells and in vivo tumor growth experiments. Results Circle‐seq analysis revealed a 13‐fold increase in eccDNA abundance in HGSOC compared to normal ovarian tissue, with significant enrichment in promoter and coding regions. The MMEJ pathway was identified as the predominant pathway for eccDNA biogenesis in HGSOC, supported by characteristic microhomologies at junction sites and validation via LIG3 and POLQ knockdown experiments. Notably, HGSOC‐specific eccDNA frequently contained functional eccMIRs (eccMIR3661, eccMIR618, and eccMIR2277), which generate oncogenic miRNAs. These miRNAs promote tumor progression by downregulating tumor suppressor genes and activating key oncogenic pathways. Functional assays confirmed that these eccMIRs significantly enhanced HGSOC cell proliferation, migration, and invasion in vitro and promoted tumor growth in vivo. Conclusions These results underscore eccDNA as an oncogenic driver in HGSOC through non‐coding RNA‐mediated regulatory mechanisms, revealing novel therapeutic opportunities for targeting eccDNA biogenesis in this aggressive malignancy. Key points This study revealed a 13‐fold increase of eccDNA in HGSOC compared to normal tissues, with significant enrichment in promoter and coding regions. eccDNA‐derived miRNAs (eccMIRs) were shown to enhance cancer cell proliferation, invasion, and tumor growth through the expression of oncogenic miRNA sequences. The study highlights the importance of the MMEJ pathway in eccDNA generation and proposes that targeting eccDNA biogenesis in this aggressive malignancy presents a novel therapeutic opportunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc21093a2da7863b736808acd2b5cbd885658bf0" target='_blank'>
              Extrachromosomal circular DNA expressing miRNA promotes ovarian cancer progression
              </a>
            </td>
          <td>
            Ning Wu, Ling Wei, Qiyu Liu, T. He, Cuiyu Huang, Yunpeng Jiang, Kailong Li, Hongyan Guo, Fengbiao Mao, Xiaolu Zhao
          </td>
          <td>2025-09-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ABSTRACT Background This article presents an extensive review of the advancements in research concerning extrachromosomal circular DNA (ecDNA) within the context of breast cancer. As a distinct form of DNA, ecDNA is critically involved in the initiation, progression, diagnosis, and treatment of breast cancer. Methods The article provides a comprehensive analysis of the mechanisms underlying the formation of ecDNA, highlighting factors such as aberrant DNA damage repair and chromosomal rearrangements. It further examines the biological roles of ecDNA in augmenting oncogene expression, fostering tumor heterogeneity, and facilitating immune evasion. Results Epidemiological studies indicate significant variability in the distribution of ecDNA across different breast cancer subtypes, with a notable association with invasive subtypes, such as triple‐negative breast cancer. The presence of ecDNA is linked to poor prognosis and treatment resistance in patients. Current detection technologies for ecDNA include molecular biology and imaging techniques, and its detectability in plasma underscores its potential as a biomarker for liquid biopsy. Conclusions The development of ecDNA‐targeted therapies and their integration with immunotherapy strategies offers promising new avenues for breast cancer treatment. Despite challenges such as incomplete elucidation of mechanisms, standardization of detection methods, and ethical considerations, ecDNA remains a valuable biomarker and therapeutic target in breast cancer. Future research is anticipated to advance its clinical transformation and application in individualized treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3be1ca65e1813757b54ee00006ec2026ac1e51f0" target='_blank'>
              Research Advances of Extrachromosomal Circular DNA in Breast Cancer
              </a>
            </td>
          <td>
            Xiaodan Zhu, Yiyang Qian, Quan Tang, Jiahui Li, Yuqing Geng, Shixia Gong, Chunhui Jin
          </td>
          <td>2025-10-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cfc80059f8767909d0c8bd930a6c33c0803bbb" target='_blank'>
              Preservation and Clonal Behavior of Extrachromosomal DNA in Patient-Derived Xenograft Models of Childhood Cancers.
              </a>
            </td>
          <td>
            Rishaan Kenkre, Jon D. Larson, Owen S. Chapman, J. Luebeck, Yan Yuen Lo, Megan Paul, Wenshu Zhang, V. Bafna, Robert J Wechsler-Reya, Lukas Chavez
          </td>
          <td>2025-08-19</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="DNA abnormalities characterized by cytogenetic imaging at the single cell resolution, i.e. karyotyping, have long served as cancer diagnostic and prognostic biomarkers. To enable in-silico karyotyping using unpaired whole-genome sequencing data, we developed Seq2Karyotype (S2K), a tool that fits karyotype models with clonality estimation based on read-depth and allelic imbalance in a bulk sample and supports visualization-guided refinement. Analysis on 19 adult and pediatric cancer cell lines revealed unexpected intratumoral heterogeneity involving multiple copy number variation (CNV) states including whole-genome duplication, which were validated by imaging and single-cell omics profiling. Analyses on patient samples showed high concordance with clinical cytogenetic reports for acute myeloid leukemia, and revealed evolutionary trajectories from multi-region metastatic neuroblastomas implicating reversion. These findings highlight extensive and dynamic intratumoral heterogeneity contributed by CNV in both cell line models and patient samples, which may inform future research on tumor evolution under selective pressure such as drug exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0572920d9d208e3e04940659c463f83c4e0642eb" target='_blank'>
              Seq2Karyotype (S2K): A Method for in-silico Karyotyping Using Single-Sample Whole-Genome Sequencing Data
              </a>
            </td>
          <td>
            Limeng Pu, Karol A. Szlachta, Virginia Valentine, Meiling Jin, Sivaraman Natarajan, Robert Greenhalgh, Nadezhda V. Terekhanova, Jian Wang, Daniel K. Putnam, Li Dong, Lingxiang Jiang, Soumya Tumbath, Xiumei Huang, Xiaotu Ma, Thomas Look, Marcin W. Włodarski, Lu Wang, Steven Burden, J. Easton, Xiang Chen, Jinghui Zhang
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99f0c6259c21cf5580851e190063781e3ca6d347" target='_blank'>
              Cancer subclone detection based on DNA copy number in single-cell and spatial omic sequencing data.
              </a>
            </td>
          <td>
            Chi-Yun Wu, Jiazhen Rong, A. Sathe, Paul R Hess, Billy T Lau, Susan M. Grimes, Sijia Huang, Hanlee P. Ji, Nancy R. Zhang
          </td>
          <td>2025-09-01</td>
          <td>Nature methods</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Human papillomavirus (HPV) integration is recognized as a hallmark event in cervical carcinogenesis. However, it does not represent a routine phase of the viral life cycle but rather a stochastic occurrence, often constituting a dead-end pathway for the virus. High-risk human papillomavirus (hr-HPV) exhibits a greater propensity for integration. The progression from initial infection to genomic integration constitutes a dynamic multi-step oncogenic process in the development of cervical cancer (CC). This process involves viral entry, immune evasion, persistent infection, and ultimately integration. This article innovatively provides a comprehensive overview of this multi-stage mechanism: HPV, via the L1/L2 proteins, mediates internalization and establishes infection. Subsequently, under the influence of factors such as the host’s genetic background, vaginal microbiota imbalance, and immune evasion, the host’s DNA damage response (DDR) pathways are activated. Viral DNA integrates into host genome vulnerable sites (e.g., 3q28 and 8q24) through microhomology-mediated end joining (MMEJ) or other alternative pathways. Following integration, the expression of viral oncogenes persists, triggering host genomic rearrangements, aberrant epigenetic modifications, and immune microenvironment remodeling, all of which collectively drive cervical cancer progression. The study further reveals the clinical potential of HPV integration as a highly specific molecular biomarker, offering new perspectives for precision screening and targeted therapy. This dynamic model deepens our understanding of the HPV carcinogenic mechanism and provides a theoretical basis for intervention strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2f3540f21b81b3f4af2b24e5ab200e71a6ae9d" target='_blank'>
              From Viral Infection to Genome Reshaping: The Triggering Role of HPV Integration in Cervical Cancer
              </a>
            </td>
          <td>
            Junlan Li, Shuang Li
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Histone acetylation is widely assumed to directly instruct gene activation. Among acetylated residues, H4K16ac is one of the most abundant modifications, conserved across all eukaryotes. Despite its established role in X-chromosome hyperactivation in Drosophila, its function in mammalian cells has remained elusive. Here, we show that in human somatic cells, H4K16ac does not substantially affect gene expression, but instead controls the spatiotemporal program of genome replication. By combining a meta-analysis of public datasets and perturbation experiments designed to minimize confounding effects, we found that H4K16ac is neither associated with nor required for transcriptional activity. Rather, H4K16ac depletion resulted in premature replication of heterochromatic regions and widespread alterations in replication timing across the genome. These defects were driven by the aberrant activation of cryptic replication origins at long terminal repeats—repetitive elements typically marked by H4K16ac and whose sequence context resembles that of canonical origins in euchromatic regions. Our findings reveal an unexpected role for one of the most prevalent chromatin modifications and uncover a new regulatory mechanism that safeguards genome replication fidelity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58d31c25c7ef175d7c8a34dda15d005ef97496d8" target='_blank'>
              Mammalian H4K16ac regulates the spatiotemporal order of genome replication rather than gene expression
              </a>
            </td>
          <td>
            Marta Milan, Valeria Runfola, Manthan Patel, Lucia Falbo, R. Noberini, C. Soriani, S. Rodighiero, Tiziana Bonaldi, Vincenzo Costanzo, Madapura M. Pradeepa, Paola Scaffidi
          </td>
          <td>2025-09-23</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Recent evidence highlights the role of extrachromosomal circular DNAs (eccDNAs) in cancers. However, reports regarding its role in hepatocellular carcinoma (HCC) are infrequent. The abundance of eccDNAs from five HCC/adjacent tissue pairs was explored using Circle-Sequencing. eccDNA PAFAH1B3 was selected as one of the objects. The effect of eccDNA PAFAH1B3 on HCC progression was determined using EdU, Transwell, and apoptosis assays. Additionally, the expressions of eccDNA PAFAH1B3, mRNA PAFAH1B3, and epithelial–mesenchymal transition (EMT)-related markers were determined using RT-PCR and WB. A xenograft tumor model was established to explore the function of PAFAH1B3 in vivo, and EMT-related markers were detected using RT-PCR and IHC analyses. The abundance of eccDNA PAFAH1B3 was significantly increased in HCC cell lines after transfection with eccDNA PAFAH1B3, and promoted the proliferation, migration, and invasion of liver cells while inhibiting apoptosis. The levels of mRNA PAFAH1B3 were also upregulated. Furthermore, intratumoral injection of PAFAH1B3 inhibitor suppressed tumor growth, and PAFAH1B3 knockdown increased and decreased the levels of the E-cadherin and N-cadherin, respectively. Our study findings reveal that eccDNA PAFAH1B3 may promote the occurrence and development of HCC by enhancing the expression of PAFAH1B3 and regulating EMT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/173a3d83ec58d1fbf7f0f7dcd22ef9f85da2bb94" target='_blank'>
              PAFAH1B3 Exists in Linear Chromosomal and Extrachromosomal Circular DNA and Promotes HCC Progression via EMT
              </a>
            </td>
          <td>
            Dandan Li, Huishan Sun, Yingjie Wang, Yicong Yin, Ying Zhu, Xia Qian, Shanshan Wang, Longhao Zhang, Haitao Zhao, Ling Qiu
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Extrachromosomally amplified oncogenes are shown to drive evolution in a large subset of IDH wild-type glioblastomas, and the spatial patterns of extrachromosomal DNA contain information on a tumor’s evolutionary history.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a02add441c27bf4e8e365b8edcace16e00d2cda" target='_blank'>
              Extrachromosomal DNA–Driven Oncogene Spatial Heterogeneity and Evolution in Glioblastoma
              </a>
            </td>
          <td>
            I. Noorani, Magnus Haughey, J. Luebeck, Andrew Rowan, E. Grönroos, Francesco Terenzi, I. Wong, Davide Pradella, Marta Lisi, Jeanette Kittel, Natasha Sharma, Chris Bailey, C. Weeden, Donald M Bell, Eric Joo, Vittorio Barbè, Matthew G. Jones, King L. Hung, E. Nye, M. Green, Lucy Meader, Emma Norton, Mark Fabian, N. Kanu, M. Jamal-Hanjani, T. Santarius, Andrea Ventura, James A R Nicoll, D. Boche, Howard Y. Chang, V. Bafna, Weini Huang, P. Mischel, C. Swanton, Benjamin Werner
          </td>
          <td>2025-09-08</td>
          <td>Cancer Discovery</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d343e029068c438cb11fe609f9a8f46f24c108" target='_blank'>
              Multi-Layered Dysregulation of NRCAM in Gliomas: Insights from TCGA Copy Number and Epigenetic Analyses
              </a>
            </td>
          <td>
            MD Steven Lehrer, MD JD MS Peter H. Rheinstein
          </td>
          <td>2025-08-21</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The interplay between DNA methylation and super-enhancer activity is emerging as a key focus in cancer research. Super-enhancers, a specialized class of enhancers, strongly activate transcription of their target genes due to their dense clustering with essential transcription factors (TFs) and cofactors. In cancer, especially, these super-enhancers control key oncogenic drivers and often display abnormal DNA methylation patterns that can repress or overexpress target genes in both solid and blood cancers. Furthermore, DNA methylation in the super-enhancer region has been found to influence their regulatory capacity. Although enhancers are typically characterized by low DNA methylation, dysregulated methylation at super-enhancers is seen in most malignancies, affecting TF and chromatin regulator recruitment. Hypomethylation at these sites often accompanies oncogene hyperactivation, while hypermethylation can repress tumor suppressor mechanisms. Recent research highlights DNA methylation as a promising source of cancer biomarkers. This review examines the intricate relationship between DNA methylation and super-enhancer activity in cancer, concentrating on how methylation regulates super-enhancers, modulates oncogene expression, promotes oncogenesis, and serves as a target for novel oncology therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80856f95cde56634ccf673f640abf635b4018a59" target='_blank'>
              Super-enhancer DNA methylation in cancer: the mechanism of action and therapeutic directions
              </a>
            </td>
          <td>
            Yue Bi, Kang Zou, Ke Wang
          </td>
          <td>2025-10-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mouse heterochromatin is characterized by A/T-rich, 234 bp DNA repeat arrays, called major satellite repeats (MSR). We investigated MSR expression in response to a variety of stress conditions by using small molecule compounds. We identified the isoflavone genistein to selectively stimulate MSR transcription, but not that of other DNA repeat elements. Genistein is a natural compound that is frequently used in dietary supplements and has been associated with reducing cancer risk. A 24 h exposure of mouse embryonic fibroblasts (MEF) to genistein results in a more than 100-fold induction of MSR transcripts. This up-regulation depends on the activity of RNA polymerase II and requires a cycling G1 cell population. Blocking the cell cycle at the G2/M stage significantly attenuates genistein-mediated stimulation of MSR transcription. Mechanistically, DNA topoisomerase poisons phenocopy the genistein-dependent up-regulation of MSR expression. Together, these data suggest that MSR transcriptional response is guided by an altered topology of the underlying A/T-rich MSR DNA repeat arrays and reveal a novel function for genistein that may contribute to the anticancer properties of this natural compound. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00623-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64fed7546750b813d379740d82aa6947375846a7" target='_blank'>
              The isoflavone genistein selectively stimulates major satellite repeat transcription in mouse heterochromatin
              </a>
            </td>
          <td>
            Thomas Fuhrmann, N. Shukeir, Reagan W. Ching, Galina Erikson, Yuan Dou, Zoe Sawitzki, Megumi Onishi-Seebacher, Carmen Galán, Thomas Jenuwein
          </td>
          <td>2025-08-25</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80c7c66e7f054de7552ccd4ec8d0186447c75c2a" target='_blank'>
              Single-cell Genomic Copy Number Evolution Reveals Frequent Loss of the Y chromosome in Esophageal Adenocarcinoma.
              </a>
            </td>
          <td>
            Shiwei Yin, Na Zhong, Swati Agrawal, Bingru Feng, Sneha Pradhan, Nathan K. Jobalia, Maryam Vaziripour, Sampath K Poreddy, L. Kresty, Erin A Gillaspie, Nicholas E Dietz, W. Al-Refaie, K. Nandipati, Yusi Fu, Jun Xia
          </td>
          <td>2025-09-17</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb14185df59742e1c30589cf1835a68e0538e9b9" target='_blank'>
              Tracking clonal evolution during treatment in ovarian cancer using cell-free DNA.
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Duaa H Al-Rawi, Gryte Satas, Hannah Lees, J. Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, Neeman Mohibullah, Yulia Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitry Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2025-10-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a lethal cancer of the bile duct and is a major health concern in several parts of the world, including northeastern Thailand, where CCA incidence is the highest due to the endemic liver fluke Opisthorchis viverrini. Multiple studies have characterised genomic alterations in CCA tumours, and specific chromosomal alterations can predict prognosis. However, it is not known whether chromosomal instability (CIN), ongoing genomic alteration characteristic of most cancer types, is present in CCA tumours. In this study we leveraged a panel of cancer cell lines derived from fluke-positive CCA patients, as well as a matched normal cholangiocyte line as a control, to characterise CIN in CCA. We observed elevated rates of chromosome segregation errors compared to normal cells, although overall CIN rates were lower than those for highly genomically unstable cancers, such as colorectal or ovarian cancer. Chromosome segregation errors in CCA cell lines were potentially driven by elevated DNA replication stress and centrosome duplication. Single-cell genome sequencing and karyotyping of the cell lines showed extensive structural and numerical chromosomal aberrations, as well as copy number alterations that were heterogeneous between individual cells, supporting the presence of ongoing CIN in these cell line models. Low-pass whole-genome sequencing of 33 CCA tumour samples with matched normal tissue from northeastern Thailand, a liver fluke-endemic region showed increased whole and subchromosomal level alterations, with a higher extent of genomic alterations in intrahepatic tumours compared to extrahepatic. Eight tumours carried focal amplifications and/or deletions involving known cancer genes, as well as potential chromosomal instability-associated genes, including CCNE1 amplifications and a rare amplification of BRCA1. This study provides increased understanding of the rate and potential mechanisms of CIN in CCA that may inform new therapeutic strategies that synergise with specific ongoing CIN mechanisms. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4433abbfa4bf430e31ec8b07fb91e4f3d1fdbb22" target='_blank'>
              Chromosomal instability and genomic alterations in cholangiocarcinoma from Northeastern Thailand.
              </a>
            </td>
          <td>
            Raksawan Deenonpoe, MA Guscott, Sasithorn Watcharadetwittaya, Isabel L McNeill, D. Moralli, N. Shaikh, SE McClelland
          </td>
          <td>2025-09-15</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Somatic human cells contain a diploid genome consisting of 23 pairs of chromosomes. The maintenance of this diploid state is essential across all layers of biological organization, ranging from the physiology of individual cells to the proper regulation of tissue homeostasis and organismal development. Most cancer cells, however, harbor an aneuploid genome with an abnormal number of chromosomes, including whole and/or partial chromosome gains and losses. These alterations arise as a consequence of mitotic chromosome segregation errors and/or ongoing chromosomal instability (CIN). While aneuploidy usually imposes a fitness cost to nontransformed cells, certain recurrent aneuploidies confer adaptive advantages that are subjected to positive selection throughout tumorigenesis. In this review, we discuss how aneuploidy impacts cellular physiology, fitness, and adaptability in the context of cancer development. We also examine how the aneuploid state and CIN can create vulnerabilities that may be exploited for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb17fd1a7e9ab93107d809a69e7ad9c212a84ff9" target='_blank'>
              Impact of Chromosomal Instability and Aneuploidy in Cancer Development
              </a>
            </td>
          <td>
            Amanda K. Mennie, Brittiny Dhital, Peter Ly
          </td>
          <td>2025-09-30</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The distinctive three-dimensional (3D) chromatin architecture adopted by chromosomes during meiosis is essential for fertility, yet the functional implications of the fine-scale spatial organization remain poorly understood. Here, we investigate the impact of RAD21L deletion, a meiosis-specific cohesin subunit, on the 3D genome architecture and gene expression in the male germ line. Using fluorescence-activated cell sorting, high-throughput chromosome conformation capture, and single-cell RNA sequencing, we demonstrate that RAD21L deficiency impairs meiotic chromatin organization and inter/intrachromosomal interactions. This conspicuous 3D genome reorganization also disrupts bouquet formation, resulting in increased telomeric interactions between heterologous chromosomes in primary spermatocytes. Genome reorganization was accompanied by detectable transcriptional dysregulation in spermatogonia and primary spermatocytes, mainly affecting sex chromosomes. Collectively, our findings establish the cohesin RAD21L as a germline-specific critical regulator of genome-wide 3D genome reorganization during spermatogenesis and a modulator of the male germline transcriptional landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2272faa3ad286c58e292f1de1607d234286a48c" target='_blank'>
              Meiotic cohesin RAD21L shapes 3D genome structure and transcription in the male germline
              </a>
            </td>
          <td>
            Laia Marín-Gual, Covadonga Vara, Raquel Sainz-Urruela, Yasmina Cuartero, Lucía Álvarez-González, Natalia Felipe-Medina, Francisca Garcia, E. Llano, M. Martí-Renom, A. Pendás, A. Ruiz-Herrera
          </td>
          <td>2025-10-01</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Functional inactivation of tumor suppressor genes drives cancer initiation, progression, and treatment responses. Most tumor suppressor genes are inactivated through 1 of 2 well-characterized mechanisms: DNA-level mutations, such as point mutations or deletions, and promoter DNA hypermethylation. Here, we report a distinct third mechanism of tumor suppressor inactivation based on alterations to the histone rather than DNA code. We demonstrated that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event in more than 80% of human endometrial cancers. Integrative transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that PAX2 transcriptional downregulation is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive chromatin features (H3K27me3) in a framework dictated by 3D genome organization. The spread of the repressive H3K27me3 signal resembled a pearl necklace, with its length modulated by cohesin loops, thereby preventing transcriptional dysregulation of neighboring genes. This mechanism, involving the loss of a promoter-proximal superenhancer, was shown to underlie transcriptional silencing of PAX2 in human endometrial cancers. Mouse and human preclinical models established PAX2 as a potent endometrial tumor suppressor. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. The discovery that most endometrial cancers originate from a recurring epigenetic alteration carries profound implications for their diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5897c9bf68c93e4fb62e86f78e7178d27155eec3" target='_blank'>
              A distinct mechanism of epigenetic reprogramming silences PAX2 and initiates endometrial carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad R. Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2025-08-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Transcription is a core hallmark of cancer, wherein many different proteins assemble at specific sites in the nucleus and act in concert to transcribe functionally relevant genes. Central to this process are transcription factors that bind to their cognate DNA motifs on enhancers and super-enhancers to recruit cofactors, coactivators, and epigenetic modifiers, thereby inducing or repressing gene expression. Super-enhancers drive oncogenic transcription, to which cancer cells become highly addicted and confer tumor dependencies on super-enhancer-driven transcription machinery. Transcriptional condensates (TCs) are nuclear membrane-less assemblies of DNA-binding transcription factors, transcription co-activators, and the transcriptional machinery (such as RNA polymerases, non-coding RNAs) formed through liquid–liquid phase separation (LLPS). The function of transcriptionally active oncogenic proteins and their interplay with nucleic acids are carried out within these biomolecular condensates, allowing them to spatiotemporally regulate oncogene expression and lead to the induction and maintenance of cancer. With this growing understanding, specific inhibitors and strategies targeting TC assembly and activation should be considered promising therapeutic opportunities for treating various tumors, including hematological malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd11a72e245b5549ddbe26e0421c105063b1fbf1" target='_blank'>
              Transcriptional Condensates: Epigenetic Reprogramming and Therapeutic Targets in Hematologic Malignancies
              </a>
            </td>
          <td>
            Kevin Qiu, Qing Yin, Chongzhi Zang, Jianguo Tao
          </td>
          <td>2025-09-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84b7fddece1de57c4f545b96bbc40fa5c54a7ad7" target='_blank'>
              Enhancer regulation in cancer: from epigenetics to m6A RNA modification.
              </a>
            </td>
          <td>
            Arathy Vasukutty, Poshan Yugal Bhattarai, Hong Seok Choi
          </td>
          <td>2025-08-19</td>
          <td>Archives of pharmacal research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies and is projected to become the second leading cause of cancer-related death within the next decade. Population-scale genomic analyses of PDAC tumors have revealed extensive chromosomal instability (CIN), indicating high levels of genomic diversification that may influence therapy response. However, the mechanisms driving CIN during PDAC progression and its role in treatment response remain poorly understood. We hypothesize that specific patterns of genomic rearrangements emerging during PDAC evolution can inform two key aspects of tumor biology: (i) the underlying processes driving CIN, which may be therapeutically targetable, and (ii) subclonal CIN signatures that predispose tumors to develop chemoresistance.



 To investigate the dynamics of CIN and chemoresistance, we employed an in-house single-cell clonal outgrowth assay integrated with single-cell whole-genome sequencing (sc-WGS) and mathematical modelling of clonal growth. Experiments were performed in PDX-derived primary cultures and patient-derived 3D organoids. We tracked clonal outgrowth under both untreated conditions and following exposure to chemotherapeutic agents relevant to PDAC treatment. Comparative analyses of single-cell copy number profiles between parental populations and drug-exposed clones allow us to identify any de novo genomic alterations acquired during treatment, as well as pre-existing events enriched by drug selection.



 We observed ongoing CIN in PDAC cells, leading to continuous generation of new CNAs and enhanced cell-to-cell genomic diversity. Our data demonstrate pronounced heterogeneity in copy number states and growth dynamics among individual clones from the same cell line. This intrinsic heterogeneity provides a reservoir for evolutionary selection and may promote the emergence of drug-resistant phenotypes. We were able to analyse which clones carried pre-existing alterations and/or which generated new ones under the pressure of chemotherapy, allowing us to understand the diversity of responses and general principles of CNA changes in response to therapy. These findings underscore the innate adaptability of PDAC cells under therapeutic pressure.



 Our study highlights the dynamic nature of CIN in PDAC and its critical role in promoting chemoresistance. The identification of specific genomic alterations enriched in resistant subclones could offer potential biomarkers predictive of therapy response. This approach provides new insights into the interplay between CIN, cell-to-cell heterogeneity, and therapeutic adaptation, with potential future implications for guiding personalized treatment strategies in PDAC.



 Audrey Lumeau, Molly Anne. Guscott, Nathaniel Mon Pere, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland. Tracking genome evolution and chemoresistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B129.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b332fbacb3e055d135e8c8f4dfd0dbc1124774d" target='_blank'>
              Abstract B129: Tracking genome evolution and chemoresistance in pancreatic cancer
              </a>
            </td>
          <td>
            A. Lumeau, Molly A Guscott, N. M. Père, Isabel Nichols, Benjamin Werner, Nelson Dusetti, Sarah E. McClelland
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/952548d2d250db0e8e23c4ea5a20ae1aba4ce302" target='_blank'>
              Coregulators determine androgen receptor activity in prostate cancer.
              </a>
            </td>
          <td>
            Kerim Yavuz, Nathan A. Lack
          </td>
          <td>2025-08-20</td>
          <td>Bioscience reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Significance This study highlights a specific source of endogenous DNA replication stress, G-quadruplexes (G4s), that can increase genome instability in BRCA2-deficient cerebellar granule cell progenitors (GCPs), leading to medulloblastoma. BRCA2 is necessary for maintenance of replication speed at G4s in GCPs, and its absence allows these structures to become hotspots of increased mutagenesis. The G4-unwinding PIF1 helicase was found to be upregulated in tumor cells as a potential way to cope with G4-related replication stress. Targeting PIF1 may represent a therapeutic strategy for BRCA2-deficient medulloblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba0cd9b13650ad33751efc7afb07cdd2a585f9d5" target='_blank'>
              G-quadruplexes as a source of vulnerability in BRCA2-deficient granule cell progenitors and medulloblastoma
              </a>
            </td>
          <td>
            Danielle L. Keahi, Mathijs A. Sanders, M. Paul, A. L. Webster, Yin Fang, Tom Wiley, Samer Shalaby, Thomas S. Carroll, S. Chandrasekharappa, C. Sandoval-Garcia, Margaret L MacMillan, John E. Wagner, M. Hatten, A. Smogorzewska
          </td>
          <td>2025-08-25</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Background BRD9 is involved in multiple physiological and pathological pathways, yet its functional role and molecular mechanisms in gastric cancer (GC) remain largely unexplored. Addressing this knowledge gap is critical given the persistent global mortality burden of GC and the limited efficacy of current therapeutic strategies. Methods BRD9 expression in GC patients was systematically analyzed using immunohistochemical (IHC) assays and transcriptomic datasets. Comprehensive functional validation, employing cellular and murine tumor models, elucidated BRD9’s role in GC progression. Molecular pathways underlying BRD9-mediated gastric carcinogenesis were delineated through integrated approaches, including RNA sequencing, co-immunoprecipitation (co-IP), subcellular fractionation, and luciferase reporter assays. Results BRD9 was significantly overexpressed in GC and associated with poor patient prognosis. Functionally, BRD9 promoted GC cell proliferation and enhanced DNA damage repair capacity. Mechanistically, elevated BRD9 expression inhibited p53 nuclear translocation via direct binding, subsequently activating the E2F transcription factor family. Notably, we identified that E2F1 directly bound to and transactivated the BRD9 promoter, establishing a positive feedback loop that sustains BRD9 expression. Additionally, BRD9 knockdown sensitized GC cells to cisplatin and oxaliplatin treatment. Conclusions These findings highlight the critical role of BRD9 in GC progression and its therapeutic potential. The BRD9-p53-E2F1 axis acts as a crucial regulator of GC cell proliferation and DNA damage response. Targeting BRD9 pharmacologically could be a novel therapeutic approach to enhance chemotherapy efficacy and improve treatment outcomes in GC patients. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02449-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dadff4be7ebdf814a21760a8db20acee07e5dfa" target='_blank'>
              BRD9-p53-E2F1 circuit orchestrates cell growth and DNA damage repair in gastric cancer
              </a>
            </td>
          <td>
            Qingqing Zhou, Qi Wang, Yantao Duan, Chi Zhang, Tengfei Liu, Hengrui Liu, Mindi Zhao, Zhihuang Hu
          </td>
          <td>2025-10-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 High-grade serous ovarian carcinomas (HGSOC) have one of the highest burdens of focal copy number alterations amongst cancers studied by The Cancer Genome Atlas. These focal copy number alterations are driven by chromosomal instability and rearrangements. Our current understanding of structural variants (SVs) in HGSOC come largely from a combination of macroscale karyotyping and base-level short-read Illumina sequencing. However, short-read sequencing is inherently hindered in accurate SV calling due to limited information of exact breakpoints captured using this technology. With the development of long-read sequencing, including Oxford Nanopore Technologies (ONT) sequencing used here, SVs are resolved with higher accuracy. Additionally, ONT sequencing allows better haplotype phasing and information about 5-methylcytosine (5mC; gene silencing) and 5-hydroxymethylcytosine (5hmC; gene activation). Here we sought to examine SVs and allele-specific methylation (ASM) in HGSOC cell lines, organoids, and tumors. Our cell line data, from 7 HGSOC cell lines, sequenced the genome to an average coverage of ∼35x. Using the epi2me human variation pipeline (ONT), supplemented by additional long-read SV caller cuteSV and filtering with MAVIS, we identified Insertions, Deletions, Translocations, Duplications, and Inversions. Because our samples lacked germline calls, we leveraged coverage information from the gnomAD database across the Ten-Thousand Genomes Project to remove regions of frequent variation in the population, which greatly filtered the number of insertions and deletions. A strong peak of 300bp was observed in insertions and deletions, which was nearly entirely repetitive elements (SINE, LINE, LTR, and simple repeats), making up over 50% of these SVs. MAVIS annotates SVs predicted to make non-synonymous changes to protein sequences; of 95 non-synonymous SVs across 7 cell lines, 57 were examined by PCR, with successful validation of 85.9%. Long-read sequencing was also haplotype phased, and DSS was used to determine regions of differential methylation between haplotypes. A mean of 828 promoters with ASM were observed, with about 40% shared between at least two cell lines. While 91 different tumor suppressor genes (TSG) had promoters with ASM were identified (mean 25 TSG per cell line), none were confirmed to have a mutation to the expressed haplotype. We are continuing to confirm whether ASM leads to bias in allelic expression of RNA, as well as expanding our study to examine long-read sequencing in organoids and tumor datasets. This work adds granularity to existing data on chromosomal rearrangements in HGSOC. Future directions of this work will examine how translocations between chromosomes influence novel enhancer-gene interactions.



 Jessica D. Lang, Ryan Moreno, Kristin M. Adams, Lauren Kersten. Long-read sequencing of high-grade serous ovarian carcinoma genomes: Insights into structural variants and allele-specific DNA methylation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6303026df4b6b8b5a1d48c24d59b28bf2267c15" target='_blank'>
              Abstract B010: Long-read sequencing of high-grade serous ovarian carcinoma genomes: Insights into structural variants and allele-specific DNA methylation
              </a>
            </td>
          <td>
            Jessica D. Lang, Ryan Moreno, Kristin M Adams, Lauren Kersten
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Breast cancer (BC) remains one of the most prevalent cancers worldwide and a significant cause of cancer-related mortality among women. Despite significant advancements in understanding the genetic foundations of BC, numerous research initiatives have historically focused on protein-coding genes, which constitute merely about 2% of the human genome. This focus has produced significant insights into oncogenes such as HER2, TP53, and BRCA1, along with tumor suppressor genes. Nonetheless, it has led to the non-coding portions of the genome garnering relatively less focus. Recent studies illuminate the crucial significance of non-coding DNA in cancer biology, highlighting its regulatory roles and influence on tumor formation, metastasis, and treatment resistance. This review examines the importance of non-coding DNA in BC. It provides an in-depth analysis of essential non-coding regions, their functions in gene regulation and chromatin structure, and their implications for various BC subtypes. Examining these facets, we seek to reveal the potential of non-coding DNA as a viable source of novel diagnostic markers and treatment approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966867774ecd7f0803dece977fed36246e346fe6" target='_blank'>
              The Role of Non-Coding Regions in Breast Cancer: From Gene Regulation to Therapeutic Implications
              </a>
            </td>
          <td>
            Hussein Sabit, Sarah Sobhy, Shaimaa Abdel-Ghany, Al-Hassan Soliman Wadan, Olubukola Ayodele, Yasser Albrahim, Hirendra Banerjee, Ahmed Elhashash, B. Arneth
          </td>
          <td>2025-09-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="While the coding genome of pancreatic cancer has been characterized in detail, features of the noncoding genome remain relatively unexplored. We used whole genome sequencing to study the coding and noncoding genome (promoters, enhancers and noncoding-NOS) in two unique patient cohorts. We find that treated cancers have a significantly higher mutational burden than untreated cancers in all four genomic regions. However, the relative proportion of mutations in each region are preserved despite treatment-induced genetic bottlenecks. Compared to other noncoding regions enhancers have a lower number of mutations/Mb. Enhancers also have a distinct mutational signature with enrichment of SBS39. Enhancer sequences were segregated into conserved and nonconserved regions based on the overlap of predicted orthologous regions in the human and mouse genomes and the conserved regions screened for recurrent somatic mutations. We find recurrent somatic mutations in conserved enhancer regions largely correspond to those associated with known transcription factors with a role in pancreatic development and cancer including KLF5 and TP63. Transcriptional expression based on RNA-seq data of cancers with enhancer mutations showed significantly different levels of expression, most often a loss of expression, compared to cancers without enhancer mutations suggesting a functional effect. These findings expand our knowledge of the noncoding genome in pancreatic cancer and point to an unexplored role of conserved enhancer mutations for pancreatic cancer as well.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edbe76ca8bf694ac9c6f9820ca8a2fe03fe38724" target='_blank'>
              The Non-Coding Mutational Landscape of Pancreatic Cancer Reveals Recurrent Somatic Mutations in Enhancer Regions.
              </a>
            </td>
          <td>
            Akimasa Hayashi, Yu-Jui Ho, Alvin P. Makohon-Moore, Amanda Zucker, Jungeui Hong, Shigeaki Umeda, Elias-Ramzey Karnoub, Jinlong Huang, Priscilla Baez, R. Kappagantula, Jerry P Melchor, Wungki Park, Eileen M. O’Reilly, Nicholas D Socci, Shinya Oki, Christine A. Iacobuzio-Donahue
          </td>
          <td>2025-09-19</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is a primary cause of therapeutic failure, driving tumor progression. Within tumors, diverse cell states coexist, maintained by a specific chromatin landscape that influences various cell functions, including cancer stemness. Among factors that induce chromatin changes affecting cell state fitness, DNA damage and its repair have emerged as significant contributors. This perspective examines recent advances that elucidate the interplay between DNA repair, epigenome, and cell plasticity. We discuss how epigenome affects DNA repair and, conversely, how DNA repair-induced chromatin changes influence cell plasticity. Finally, we discuss emerging concepts and highlight the therapeutic implications of these interconnected mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0841716149f708372bcf7e59a0f69e9d199da1" target='_blank'>
              The interplay of DNA damage, epigenetics and tumour heterogeneity in driving cancer cell fitness
              </a>
            </td>
          <td>
            Celia D Rouault, E. Charafe-Jauffret, C. Ginestier
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Background Understanding tumor heterogeneity is essential for advancing cancer treatment. Clonal reconstruction methods play a pivotal role in deciphering this heterogeneity. Our goal is to develop a clonal reconstruction approach that is clinically applicable, easy to implement, and capable of delivering both high-speed performance and excellent reconstruction accuracy. Results We present MyClone, a probabilistic method designed to reconstruct the clonal composition of tumors using deep sequencing genomic data. MyClone processes read counts and copy number information of single nucleotide variants derived from deep sequencing data, enabling it to determine the mutational composition of clones and the cancer cell fractions of these mutations. Compared to existing clonal reconstruction methods, MyClone enhances clustering accuracy and cancer cell fraction prediction when applied to deep-targeted sequencing data and bulk tumor sequencing data with deep sequencing coverage. Additionally, MyClone achieves a substantial improvement in computational speed. We rigorously validated MyClone’s performance using both simulated and real clinical datasets and applied it to analyze a circulating tumor DNA sequencing dataset from 139 metastatic breast cancer patients. In this analysis, we explored mechanisms of drug resistance in metastatic breast cancer and identified 10 mutated genes potentially associated with drug resistance or sensitivity. Conclusions For deeply sequenced data, MyClone outperforms existing methods on both targeted sequencing and bulk tumor data. Its high computational efficiency and reconstruction accuracy position MyClone as a promising tool for broad clinical application in cancer treatment. The source code is publicly available at https://github.com/Hansen0413/myclone-code. Supplementary Information The online version contains supplementary material available at 10.1186/s12859-025-06252-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/008b5a4c97ae8500318c69c4b98363d3ad251446" target='_blank'>
              MyClone: rapid and precise reconstruction of clonal population structures for tumors
              </a>
            </td>
          <td>
            Qihan Guo, Dan Lv, Bo Lan, Haili Qian, Fei Ma, Ting Chen
          </td>
          <td>2025-10-08</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Liquid biopsy analyzing cell free tumor DNA (ctDNA) from Cerebrospinal Fluid (CSF) is a feasible and relatively low-risk alternative to a tissue biopsy for diagnosis of Central Nervous System (CNS) tumors. CNS liquid biopsy is challenging due to molecular and clinical diversity of primary CNS tumors, the multitude of type of metastatic tumors and molecular drivers, and the lack of assays that distinguish tumors from non-neoplastic diseases. An integrated epigenetic and genomic analysis could provide simultaneous DNA methylation classification with mutational, copy number and gene fusion analysis. Current methylation techniques lead to DNA loss due to bisulfite conversion destruction. Enzymatic 5-base chemistries enable simultaneous high throughput genomic analysis and DNA methylation classification without DNA loss.



 We analyzed ctDNA from 60 CSF samples from patients with primary and metastatic CNS tumors. Library preparation was performed using TET-assisted pyridine borane sequencing (TAPS) chemistry (Watchmakers Genomics). Libraries were sequenced on the NOVASeq 2000 with 500 mil reads/sample. Fastq files were processed using NF-CaseReports pipeline, and coverage bias corrected using fragCounter and rPCA denoising. Methylation signals were extracted using Rastair, followed by integration with the Infinium MethylationEPIC v2.0 array probe set, facilitating downstream clustering analyses and clinical DNA methylation Classifier. Copy number analysis was performed with CBS, and structural variant, using GRIDSS.



 TAPS provides comprehensive genomic landscape analysis including mutation, genome-wide and focal copy number analysis, and gene fusions and DNA methylation classifier in the same assay. For example, we detected 1p19q losses in oligodendroglioma (N=3), trisomy of chromosome 7 (N=7) and loss of chromosome 10 (N=8), as well as high level amplifications of EGFR, MET in GBM (N=11) and ERBB2 (Her2) in metastatic breast cancer. Mutational analysis clearly classified IDH mutant (N=15) and wild-type (N=25) gliomas, based on detection of IDH1/2, ATRX, TP53 and TERT promoter mutations, amplifications, and CDKN2A/B, PTEN loss. Structural variant analysis detected EGFRvIII (N=3) and MET fusions (N=1) in GBM. Furthermore, TAPS enabled targeted analysis of the MGMT promoter concordant with methylation array. In pediatric tumors we detected MN1: : CXXC5 fusion in a case of astroblastoma and isochromosome 17 in Group 4 medulloblastoma, notably a tumor lacking somatic mutations. In 13 cases (21%), we did not detect any informative mutations or epigenetic changes, likely due to low level of CSF ctDNA.



 Integrated genomic-epigenetic ctDNA analysis using TAPS chemistry is highly sensitive liquid biopsy approach that provides simultaneous genetic and epigenetic analysis from CSF ctDNA and requires minimum ctDNA input.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c953510ee5c4758248718340692da6ec1d03a9d8" target='_blank'>
              P05.12.B ULTRASENSITIVE INTEGRATED GENOMIC-EPIGENETIC CTDNA CNS TUMOR ASSAY USING TET-ASSISTED PYRIDINE BORANE SEQUENCING (TAPS)
              </a>
            </td>
          <td>
            M. Snuderl, C. Fang, E. Freitag, A. Oviedo, C. Schroff, K. Hadi, A. Deshpande, J. Rafailov, H. Weiss, N. Dhasmana, D. Orringer, K. Wrzeszczynski, M. Imielinski, A M Miller
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 The genomic landscape of ovarian high grade serous carcinoma (HGSC) is typified by extensive genomic instability and heterogeneity, near-ubiquitous TP53 mutations, copy number aberrations, and defects in homologous recombination (HR) repair in approximately 50% of patients. Due to different mechanisms underlying therapy resistance and a lack of predictive biomarkers, matching best treatment modalities to patients with HGSC is challenging. Assessing genomic instability scores to indicate HR-deficiency (HRD) status is used clinically to guide optimal maintenance regimens including PARP-inhibitor treatment. Recently, we demonstrated the degree of phenotypic and genomic heterogeneity in advanced, high tumor burden HGSC patients. Here, we aim to show that the spatial inter-tumoral heterogeneity in HGSC expands to variability in whole genome duplication (WGD) states and timing, and in HRD-score derivation. Systematic mapping of tumor dissemination patterns was performed in chemotherapy-naïve advanced HGSC patients (n=22) following maximal-effort cytoreduction. Multi-site biopsies from 4-15 different anatomical sites were collected, and paired relapse samples were collected where possible (n=10). Germline and Tumor DNA was extracted (5 tumors per case, plus all relapse samples) and underwent whole genome sequencing (WGS, ∼30x depth). We observed spatial and temporal genomic heterogeneity for common HGSC features including mutations, gene amplifications (e.g. CCNE1 and MYC), and copy number and mutational signatures. Somatic, pathogenic loss-of-function TP53 mutations were observed in all tumors in the cohort. We observed discordance in HRDsum classification measures, with 4/22 patients presenting with mixed-HRD scores, i.e. both HRD-positive and HRD-negative scores in their tumors. In each case, the HRDsum scores had a mean value close to the threshold of 42 (range 35.8-45.8). HRD-negative or mixed-HRD status patients had poorer progression free (p=0.0026) and overall survival (p=0.0015) than HRD-positive patients. Cases with mixed states for WGD (no WGD, one WGD or second sub-clonal WGD) were observed in 41% of the cohort with variable timings (early vs late) within disseminated tumors across the cohort, in patients with HRD-negative and mixed HRD-status. The significant inter-tumoral genomic diversity observed in our analysis in measures of HRD and WGD status across the various tumor sites in our cohort of advanced HGSC patients highlights the challenges facing clinicians to derive accurate clinically actionable or prognostic information from a single tumor biopsy alone to direct patient treatment.



 Paula Cunnea, Nikki Burdett, Marc Lorentzen, Ahwan Pandey, James Clark, Katherine Nixon, Jennifer Ploski, Shirya Varghese, David Bowtell, Jonathan Krell, Elizabeth Christie2, Christina Fotopoulou. Evaluating spatial inter-tumor heterogeneity in whole genome duplication states and homologous recombination deficiency status in advanced ovarian high grade serous carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28933fb2fad0d299521abc6704fd16fc40c4a93c" target='_blank'>
              Abstract B004: Evaluating spatial inter-tumor heterogeneity in whole genome duplication states and homologous recombination deficiency status in advanced ovarian high grade serous carcinoma
              </a>
            </td>
          <td>
            P. Cunnea, Nikki L Burdett, Marc Lorentzen, A. Pandey, James Clark, K. Nixon, J. Ploski, Shirya Varghese, D. Bowtell, J. Krell, Elizabeth L. Christie, C. Fotopoulou
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) has been suggested as a key factor of tumor heterogeneity at the genomic and transcriptional levels, via unequal segregation and ecDNA hub formation, respectively. In this issue of Cancer Discovery, two studies further expand our understanding of the roles of ecDNA in enhancing tumor heterogeneity at the phenotypic and spatiotemporal levels, as well as in providing a strong selective advantage during tumor evolution. See related article by Montuori et al., p. 2054 See related article by Noorani et al., p. 2078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cba883c44a904170e7fcec05ae3700dab4a7298" target='_blank'>
              Extrachromosomal DNA: A Trusted Path to Tumor Heterogeneity.
              </a>
            </td>
          <td>
            Emmanuel E Korsah, Noah A Dusseau, Eunhee Yi
          </td>
          <td>2025-10-06</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Human papillomaviruses (HPVs) maintain their genomes as minichromosomes in the nuclei of infected keratinocytes. This study investigates the association of HPV31 genomes with host chromatin using both HiC and 4C-seq chromosome conformation capture techniques. We show that HPV31 genomes preferentially associate with transcriptionally active A compartments of host chromatin, regions of open chromatin defined by ATAC-seq, and super-enhancers defined by Brd4 and H3K27ac ChIP-seq. The viral genome association sites were also highly correlated with genomic loci previously identified as common HPV integration sites in cervical cancers. Recent studies have shown that transcriptionally active sites are prone to dsDNA breaks, and we find a strong correlation among dsBREAK datasets with transcriptionally active and open regions of host chromatin and the HPV31 genome association sites defined in our study. These findings suggest that HPV genomes associate with cellular transcriptional epicenters to maintain active viral gene expression during persistent infection, but also indicate that the susceptibility of these regions to dsDNA breaks could explain their propensity for viral DNA integration in HPV-associated cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38ecc91def6b800389ed078be4c219004519589f" target='_blank'>
              Human papillomavirus genomes associate with active host chromatin during persistent viral infection
              </a>
            </td>
          <td>
            Alix Warburton, T. E. Markowitz, JJ L. Miranda, Kinjal Majumder, Alison A. McBride
          </td>
          <td>2025-09-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Comprehensive epigenomic studies in multiple myeloma (MM) that unravel the connections between major epigenetic regulators, their intertwined collaboration and the potential of combinatorial targeting remain limited. Utilizing ChIP-seq, ATAC-seq, RNA-seq, and DNA methylation (DNAme) data, we generated whole-genome chromatin annotations from normal plasma cells and MM patients, revealing epigenomic re-configuration affecting downstream genes involved in tumour growth and survival. Primary MM samples showed global DNA hypomethylation but site-specific hypermethylation was observed at transcription start sites, promoters, and enhancers. Moreover, increased deposition of H3K27me3 was observed in clinically relevant functional chromatin clusters. Combined EZH2 and DNMTs inhibition resulted in extensive epigenomic alterations activating apoptosis and cell cycle genes, leading to increased G2/M arrest and apoptosis in MM cell lines. Our findings provide novel insights into the role of epigenetic gene silencing in MM tumorigenesis and the interplay between the Polycomb repressive complex 2 and DNAme. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17093-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1dfe4ebbad2d35b5cf5e512e8911342a3522d08" target='_blank'>
              Combinatorial DNMTs and EZH2 inhibition reprograms the H3K27me3 and DNAme-mediated onco-epigenome to suppress multiple myeloma proliferation
              </a>
            </td>
          <td>
            Alba Atienza Párraga, P. Nylund, K. Diamanti, Berta Garrido-Zabala, Stefania Iliana Tziola, Louella Vasquez, P. Pyl, D. Raykova, Aron T. Skaftason, Anqi Ma, Jian Jin, J. Martín-Subero, Fredrik Öberg, Elke De Bruyne, Jan Komorowski, Helena Jernberg Wiklund, A. Kalushkova
          </td>
          <td>2025-08-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa29e8cd8f6fdd768b529f92b9275ea04ce76dc" target='_blank'>
              Genetic landscape of cancer: mechanisms, key genes, and therapeutic implications.
              </a>
            </td>
          <td>
            Arun Karnwal, Joydeep Dutta, Aqueel-Ur-Rehman, Abdel Rahman Mohammad Said Al-Tawaha, Natalia Nesterova
          </td>
          <td>2025-08-17</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Repairing programmed DNA double-strand breaks (DSBs) is crucial in the lifecycle of Paramecium tetraurelia, especially during its sexual reproduction phase when its somatic polyploid macronucleus is lost. The formation of a new macronucleus involves programmed genome rearrangements, introducing DNA DSBs at ∼45 000 loci. Paramecium tetraurelia employs a non-homologous end joining (NHEJ) mechanism for the faithful repair of these DSBs. There are four genes encoding DNA polymerases of family X in the genome, one of which was found recently to colocalize with other NHEJ proteins in the nucleus. Here we have characterized all four enzymes and shown that they are generally very faithful. They fall into two functional classes that may specialize in the distinct repair contexts encountered during DSB DNA repair. Biochemical assays, site-directed mutagenesis, and X-ray structures of mutants of human Polλ incorporating sequence determinants from P. tetraurelia PolX or metazoan Polβ are used to investigate the origin of their fidelity. Our findings suggest that Paramecium PolX enzymes may represent evolutionary intermediates between metazoan Polβ and Polλ. A general classification of DNA PolXs based on clustering methods indicates that our results can be generalized to plant DNA PolXs (Polλ-like) involved in DSB DNA repair generated by CRISPR–Cas9 engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e54e53d0ac06ac21d27fa1e914141a18179e83e" target='_blank'>
              Fidelity, specialization, and evolution of Paramecium PolX DNA polymerases involved in programmed double-strand break DNA repair
              </a>
            </td>
          <td>
            Nourisson Antonin, Missoury Sophia, Lucas-Staat Soizick, Haouz Ahmed, Delarue Marc
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromatin, the spatial organizer of genomic DNA, is hierarchically folded into higher-order structures to facilitate DNA compaction, enabling genome surveillance. Understanding the organization and function of the three-dimensional (3D) genome is critical to profile chromatin accessibility and functional interactions that govern gene regulation across multiple biological processes, including aging and one of its hallmarks, cellular senescence. Cellular senescence constitutes a defensive stress response to various intrinsic and extrinsic stimuli, preserving cellular and organismal homeostasis through a generally irreversible cell cycle arrest. In this review article we discuss epigenetic alterations occurring to DNA and chromatin that drive and fuel the onset of this complex phenomenon. As such, we describe major large-scale chromatin events, including the formation of higher-order chromatin structures and the 3D spatial alterations of the genome that occur during senescence. We also discuss global heterochromatin loss, deficiencies in nuclear lamins, the depletion of core histones and their modifications, as well as the epigenetic regulation of the senescence-associated secretory phenotype (SASP), all of which serve key roles in the epigenome of senescent cells. To clearly demonstrate the significance of epigenetic modifications, data from a computational meta-analysis are presented, aiming to further underpin key epigenetic mechanisms occurring in senescent cells. Last, we highlight promising epigenetic modulators implemented in therapeutic strategies for senescent cell detection and elimination, possibly leading to significant clinical advances against various age-related diseases as well as the delay and prevention of the aging onset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fd36c31f599c330b53d9bff8eb9f2a73d497006" target='_blank'>
              Interrogating the regulatory epigenome of cellular senescence
              </a>
            </td>
          <td>
            Dimitris-Foivos Thanos, Orestis A. Ntintas, Emmanouil Athanasiadis, A. Papaspyropoulos, R. Petty, V. Gorgoulis
          </td>
          <td>2025-08-31</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee3a629089847ef194ea5479a4ede2bc91a09e2d" target='_blank'>
              Detection and monitoring of translocation renal cell carcinoma via plasma cell-free epigenomic profiling.
              </a>
            </td>
          <td>
            S. Garinet, Karl Semaan, Jiao Li, Ananthan Sadagopan, J. Canniff, Noa Phillips, K. Klega, Medha Panday, H. Savignano, Matthew P Davidsohn, Kevin Lyons, Alessandro Medda, Prateek Khanna, M. Achom, Prathyusha Konda, Brad Fortunato, Rashad Nawfal, Razane El Hajj Chehade, Ze Zhang, Jillian O’Toole, J. Horst, Dory A Freeman, Rachel Trowbridge, C. Chau, William D. Figg, Jacob E. Berchuck, Brian D Crompton, J-H. Seo, T. Choueiri, M. Freedman, S. Baca, Srinivas R Viswanathan
          </td>
          <td>2025-09-04</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Double-strand break (DSB) repair occurs through non-homologous end joining (NHEJ) or homologous recombination (HR). To identify non-canonical factors that influence DSB repair outcomes, we parsed data from pooled genetic screens. Through this approach, we identified the splicing factor SFPQ, which has been previously reported to associate with DSBs and promote repair. Here, we show that SFPQ depletion alters DSB repair via HR. However, in contrast to other published work, we find that SFPQ does not localize to DSBs but instead stabilizes the expression of RAD51 and its paralogs independently of p53 activation or DNA damage. Our findings suggest that SFPQ contributes to constitutive DSB repair by maintaining RAD51 paralog mRNA stability rather than through direct interaction with DSBs or RAD51 protein and highlight indirect mechanisms by which RNA-binding proteins can influence genome stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1f2a926bfc3d7a9fcc9c4f329c1455e2b3d6c2b" target='_blank'>
              SFPQ Promotes Homologous Recombination via mRNA Stabilization of RAD51 and Its Paralogs
              </a>
            </td>
          <td>
            Sofia Gotthold, Keile R. Hansen, Andrew N. Brown, Soham P. Chowdhury, Hannah I. Ghasemi, Amanda C. Yoon, Christine M. Joyce, Julien Bacal, Brooke M. Gardner, Chris D. Richardson
          </td>
          <td>2025-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Small round cell sarcomas (SRCS) are highly aggressive tumors in soft tissues and bone of mostly children and young adults. Despite being different in many aspects, including genetics, possible cell-of-origin, and pathology, patients with any of these entities all receive the same therapeutic regimen. Although several pre-clinical models of Ewing sarcoma have been established, such as cell lines and patient-derived tumor xenografts, few models exist for other SRCS. Here, we describe a pediatric SRCS tumor organoid (tumoroid) biobank containing long-term tumoroid cultures with different translocations, including EWSR1::FLI1, EWSR1::ERG, CIC::DUX4, and BCOR-rearrangements. Using histology, whole genome sequencing and RNA sequencing, we demonstrate that these tumoroids retain histological characteristics, known marker gene expression and chromosomal rearangements of their matching patient tumors. In addition, we compare mutation clusters in the tumoroids across patient-matched longitudinal samples, which shows that cellular heterogeneity is maintained. Drug screening on the tumoroid models unveils entity-specific drug sensitivity to various cytotoxic compounds and targeted compounds, including MCL-1 inhibitors for CIC::DUX4 sarcomas. Taken together, this newly established SRCS patient-derived tumoroid biobank represents a promising source of material for future basic cancer research and drug screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d54a3ce373321b31d8cc83feaca65eb77c05c889" target='_blank'>
              Small round cell sarcoma tumoroid biobank reveals CIC::DUX4 sarcoma vulnerability to MCL-1 inhibition
              </a>
            </td>
          <td>
            F. Ringnalda, Gijs J. F. van Son, Laurens H G Verweij, Seok-Young Kim, Vicky Amo-Addae, U. E. Flucke, L. Hiemcke-Jiwa, Karin P S Langenberg, Jos A. M. Bramer, L. Heimans, M. A. van de Sande, W. V. van Houdt, Max M. van Noesel, H. Kerstens, Marcel Santoso, Georg Seifert, O. Delattre, Katia Scotlandi, B. Geoerger, J. H. Merks, J. Molenaar, R. van Boxtel, M. van de Wetering, K. Sanders, H. Clevers
          </td>
          <td>2025-08-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Cyclin-dependent kinases (CDKs) coordinate DNA replication and cell division, and play key roles in tissue homeostasis, genome stability and cancer development. The first step in replication is origin licensing, when minichromosome maintenance (MCM) helicases are loaded onto DNA by CDC6, CDT1 and the origin recognition complex (ORC). In yeast, origin licensing starts when CDK activity plummets in G1 phase, reinforcing the view that CDKs inhibit licensing. Here we show that, in human cells, CDK4/6 activity promotes origin licensing. By combining rapid protein degradation and time-resolved EdU-sequencing, we find that CDK4/6 activity acts epistatically to CDC6 and CDT1 in G1 phase and counteracts RB pocket proteins to promote origin licensing. Therapeutic CDK4/6 inhibitors block MCM and ORC6 loading, which we exploit to trigger mitosis with unreplicated DNA in p53-deficient cells. The CDK4/6-RB axis thus links replication licensing to proliferation, which has implications for human cell fate control and cancer therapy design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae1df687f2a244dc847159544edc08f4fb36a5cf" target='_blank'>
              Temporal control of human DNA replication licensing by CDK4/6-RB signalling and chemical genetics
              </a>
            </td>
          <td>
            Anastasia Sosenko Piscitello, Ann-Sofie Nilsson, Michael Hawgood, Abid H Sayyid, V. Dionellis, Giovanni Giglio, Bruno Urién, Pratikiran Bajgain, Sotirios G. Ntallis, J. Bartek, T. Halazonetis, B. Lemmens
          </td>
          <td>2025-09-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Over 95% of ependymomas that arise in the cortex are driven by a gene fusion involving the zinc finger translocation-associated (ZFTA) protein. Here, using super-resolution and lattice light-sheet microscopy, we demonstrate that the most frequent fusion variant, ZFTA–RELA (ZR), forms dynamic nuclear condensates that are required for oncogene expression and tumorigenesis. Mutagenesis studies of ZR reveal a key intrinsically disordered region (IDR) in RELA that governs condensate formation. Condensate-modulating IDR mutations introduced into ZR impaired its genomic occupancy at oncogenic loci and inhibited the recruitment of transcriptional effector proteins, such as MED1, BRD4 and RNA polymerase II. Using nuclear magnetic resonance spectroscopy, we examined the DNA-binding residues of the critical zinc finger (ZF1) found in ZR and characterized their significance for condensate formation, genomic binding and oncogene activation. We generated synthetic ZFTA fusion proteins where IDRs from known condensate-forming proteins were grafted into ZR. Synthetic ZFTA fusion oncoproteins utilizing IDRs from EWS and FUS restored condensate formation, oncogene transcription and tumour initiation in mice. These findings provide key insights into the oncogenic mechanism of ZR and the importance of IDR acquisition in fusion oncoproteins in brain cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d5d4e3905b6133ceff9f0311d9fcea6d57257e" target='_blank'>
              Synthetic ZFTA fusions pinpoint disordered protein domain acquisition as a mechanism of brain tumorigenesis
              </a>
            </td>
          <td>
            A. Arabzade, H. K. Shirnekhi, S. Varadharajan, S. M. Ippagunta, A. Phillips, N. Laboe, D. W. Baggett, Wahiduzzaman, M. Jo, T. Zheng, R. Pathak, D. Gee, D. Bhimsaria, H. Wu, X. Gao, J. Liu, E. Emanus, A. Bland, A. Kardian, A. Hancock, B. Holcomb, T. Wright, T. Bugbee, H. Sun, M. Zhai, E. Caesar, M. Park, S. Tripathi, A. Shirinifard, K. Lowe, A. Khalighifar, R. A. Petersen, S. King, D. Stabley, A. Pitre, G. Campbell, C-G Park, W. T. Freyaldenhoven, B. Chandra, Y. Xia, E. Bonten, A. Achari, S. Kandikonda, A. Carisey, S. Pounds, J. Xu, D. W. Ellison, B. Deneen, K. Bertrand, R. Kriwacki, S. C. Mack
          </td>
          <td>2025-08-27</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Abstract Human positive coactivator 4 (PC4) is a highly abundant non-histone chromatin protein involved in diverse cellular processes, including transcription regulation, genome organization, autophagy, B-cell differentiation, neurogenesis, DNA repair, etc. Most PC4 is phosphorylated in cells, which interacts with core histones and the linker histone H1 to confer the compact heterochromatin state of the genome. Knocking down PC4 at both cellular and organismal levels leads to significant chromatin decondensation, altered epigenetic landscape, enhanced autophagy, and increased DNA damage susceptibility. Here, we report that besides p300, PC4 also gets acetylated by DNA repair, facilitating lysine acetyltransferase KAT5 (Tip60) at a specific lysine residue (PC4K80) when the cells are subjected to DNA damage. The vulnerability of DNA in PC4 devoid cells was substantially reduced by reintroducing wild-type PC4 to the cells but not the mutant PC4 (PC4K80R), defective in KAT5-mediated acetylation. High-resolution microscopy techniques, including transmission electron microscopy and atomic force microscopy, are employed to visualize chromatin structural changes in response to DNA damage and repair in a Tip60-mediated PC4 acetylation-dependent manner. Presumably, KAT5-mediated acetylation of PC4 at K80 residue facilitates access to the damaged DNA by altering chromatin structures at damage sites, thus promoting DNA repair. This process could be highly significant both in cancer and in aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c83d5690dbc70212ddc740cb45975fd69abb4f3" target='_blank'>
              KAT5-mediated acetylation of PC4 facilitates DNA repair by promoting chromatin reorganization
              </a>
            </td>
          <td>
            Aa yushi A gr a w al, eta Sikder, Siddharth Singh, Rakesh Kumar, Nikhil Pallaprolu, Kalyan Mitra, Sourav Mondal, Rupa Mukhopadhy, anta Sarkar, Ramaling am Per aman, Ra vic, Tapas K. Kundu
          </td>
          <td>2025-09-23</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Alterations in 3D chromatin conformation may disrupt the interplay between promoters and distal enhancers. How gene regulatory circuits are reshaped during ionizing radiation-induced tumorigenesis remains unclear, and little is known about the mediators that drive these processes. To decipher the chromatin alterations in radiation-induced lung cancer, we performed ATAC-seq, RNA-seq and Hi-C analyses of human bronchial epithelial cells and corresponding radiation-induced malignantly transformed cell lines. We found that this malignant transformation is accompanied by chromatin switching from the inactive B compartment to the active A compartment, an increased number of TADs and gained ATAC-seq peaks that mediate new distal chromatin contacts. We identified tumour protein 63 (TP63) as a mediator of new chromatin-accessible sites that anchor tumour-specific chromatin contacts in radiation-induced tumour cells. A TP63-mediated accessible chromatin site anchors a tumour-specific TAD boundary and multiple tumour-specific chromatin loops, which might underlie MYC oncogene activation during malignant transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b8217e99ba9977673505cae49c7863f218434b" target='_blank'>
              TP63 mediates the generation of tumour-specific chromatin loops that underlie MYC activation in radiation-induced tumorigenesis
              </a>
            </td>
          <td>
            Qiaowei Liu, Shuai Jiang, Xiang Xu, Kang Xu, Yawen Luo, Zongyuan Yu, Meida Xiang, Zhouyang Xu, Lijie Wang, Sujie Zhang, H. Tao, Xuhui Yang, Chongbo Jiang, Xiaoguang Qi, Hao Li, Yiming Gao, Yao Li, Shihui Fu, Pingkun Zhou, Xiaochen Bo, Hebing Chen, Xiaohua Chen, Yi Hu
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background/Objectives: DNA-damaging agents can contribute to genetic instability, and such agents are often used in cancer chemotherapeutic regimens due to their cytotoxicity. Thus, understanding the mechanisms involved in DNA damage processing can not only enhance our knowledge of basic DNA repair mechanisms but may also be used to develop improved chemotherapeutic strategies to treat cancer. The high-mobility group box protein 1 (HMGB1) is a known nucleotide excision repair (NER) cofactor, and its family member HMGB3 has been implicated in chemoresistance in ovarian cancer. Here, we aim to understand the potential role(s) of HMGB3 in processing DNA damage. Methods: A potential role in NER was investigated using HMGB3 knockout human cell lines in response to UV damage. Subsequently, potential roles in DNA interstrand crosslink (ICL) and DNA double-strand break (DSB) repair were investigated using mutagenesis assays, metaphase spreads, foci formation, a variety of DNA repair assays, and TagSeq analyses in human cells. Results: Interestingly, unlike HMGB1, HMGB3 does not appear to play a role in NER. We found evidence to suggest that HMGB3 is involved in the processing of both DSBs and ICLs in human cells. Conclusions: These novel results elucidate a role for HMGB3 in DNA damage repair and, surprisingly, also indicate a distinct role of HMGB3 in DNA damage repair from that of HMGB1. These findings advance our understanding of the role of HMGB3 in chemotherapeutic drug resistance and as a target for new chemotherapeutic strategies in the treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1261da2765ba41543d61e2f0da2291c8e870115b" target='_blank'>
              High-Mobility Group Box Protein 3 (HMGB3) Facilitates DNA Interstrand Crosslink Processing and Double-Strand Break Repair in Human Cells
              </a>
            </td>
          <td>
            Jillian Dangerfield, Anirban Mukherjee, W. Reh, Anna M. Battenhouse, Karen M. Vasquez
          </td>
          <td>2025-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="IDH-mutant gliomas represent a subtype of diffuse gliomas that primarily affect patients in early to mid-adolescence. These tumors are classified into three distinct CNS WHO grades of malignancy. Accurate grading is essential for selecting an appropriate treatment maximizing anti-tumor efficacy while minimizing adverse effects. However, grading of oligodendrogliomas with 1p/19q codeletion currently relies on qualitative tumor characteristics that may be influenced by observer subjectivity, sampling bias, and tumor heterogeneity. This study aimed to explore DNA methylation-based tumor deconvolution into latent methylation components (LMCs) to evaluate their potential as objective grading tools in a cohort of 137 IDH-mutant gliomas. LMCs were analyzed in relation to malignancy markers, cellular composition, and underlying methylation signatures of the chromatin landscape. Glioma subtypes were associated with distinct LMCs. Two LMCs correlated with higher cellular density and advanced epigenetic age as well as with microvascular proliferation, necrosis, and epigenetically defined high-grade astrocytoma. The epigenetic patterns defining high-grade astrocytoma or oligodendroglioma, respectively, were similar. Higher-grade oligodendrogliomas, identified by LMC-based grading, were associated with more copy number alterations. Among patients of an external cohort who died during the assessment period, higher LMC1 proportions were associated with poorer overall survival. Therefore, LMCs hold the potential to support IDH-mutant glioma grading by incorporating objective epigenetic markers. Supplementary Information The online version contains supplementary material available at 10.1007/s00401-025-02926-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d16c74139be26cefe8b9fdf9ca1fbad5b85b836" target='_blank'>
              DNA methylation analysis reveals an epigenetic signature distinctive of high-grade oligodendroglioma
              </a>
            </td>
          <td>
            Katharina J. Weber, M. Dettki, M. Münzberg, P. Zeiner, Marie-Thérèse Forster, Eike Steidl, I. Divé, Patrick N Harter, Michael Scherer
          </td>
          <td>2025-08-23</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The ability to complete DNA replication as replisomes converge has recently been shown to be a highly regulated, multi-enzymatic process. Converging forks also are likely to generate unique supercoiled, tangled, or knotted substrates. These structures are typically resolved by one of the four topoisomerases encoded by Escherichia coli. However, identifying the cellular substrates and specific function for these essential enzymes which contain overlapping biochemical activities has remained challenging. Here, we show that Topo I and Topo IV are required to allow converging forks to complete chromosome replication. Impaired Topo I function leads to amplifications where forks converge, whereas inactivation of Topo IV prevents forks from converging and produces a dramatic loss of this chromosome region. The results are consistent with previous studies suggesting Topo I suppresses illegitimate initiations in the terminus region by disrupting R- and D-loops and demonstrate a specific requirement for Topo IV acting before replication completes to allow convergent forks to reach their doubling point. We propose that the positive supercoils arising between convergent forks are converted to precatenanes and resolved by Topo IV, when diminishing space may preclude gyrase from binding and functioning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00942d9d408f53959a1274fc3d6f6fac0c437dbd" target='_blank'>
              Topo IV is required to allow replisomes to converge and complete replication on the chromosome
              </a>
            </td>
          <td>
            Parisa D Mokhtari, Tannos Seyedjavadi, Thulni A Liyanaarachchi, C. T. Courcelle, J. Courcelle
          </td>
          <td>2025-09-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="ABSTRACT Liquid biopsy represents a transformative approach in oncology, enabling noninvasive disease detection and monitoring through epigenetic signals in circulating tumor DNA (ctDNA), nucleosomes, and noncoding RNAs. Tumor initiation is driven by epigenetic modifications, including DNA methylation, histone alterations, and dysregulated noncoding RNAs, which disrupt gene regulation, cell cycle control, DNA repair, and metastatic processes. This review systematically examines recent evidence on DNA methylation, histone marks (e.g., H3K27me3, H3K18ac), and noncoding RNAs (miRNAs, lncRNAs) as biomarkers for early cancer detection, prognosis, and therapeutic response. Particular focus is placed on aberrant DNA methylation (e.g., hypermethylation of CDKN2A, RASSF1A) and altered histone modifications (e.g., EZH2‐mediated silencing) as indicators of tumor heterogeneity and evolution. Stable and specific in biofluids, noncoding RNAs such as oncogenic miR‐21, tumor‐suppressive miR‐34a, and metastasis‐associated MALAT1/HOTAIR further enhance clinical applicability. Recent detection methods, including bisulfite sequencing, ChIP‐seq, and RNA‐seq, have advanced biomarker profiling, though challenges remain in standardization and low‐abundance detection. With over 12 active clinical studies validating their utility, integration of epigenetic markers with AI and multiomics holds promise for individualized, dynamically guided oncology care. Future innovations, such as chromatin accessibility analysis and cfDNA fragmentation profiling, may further refine diagnostic precision and therapeutic monitoring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77f80572fe0354408f2985b22c2727d7355f5dac" target='_blank'>
              Transforming Cancer Diagnostics: The Emergence of Liquid Biopsy and Epigenetic Markers
              </a>
            </td>
          <td>
            Debalina Saha, Pritam Kanjilal, Mandeep Kaur, Soumya V. Menon, Ayash Ashraf, M. Kumar, T. Alqahtani, Shikha Atteri, D. Uti, Bikram Dhara
          </td>
          <td>2025-09-01</td>
          <td>MedComm</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Cancer genomes frequently carry apolipoprotein B mRNA editing catalytic polypeptide-like (APOBEC)-associated DNA mutations, suggesting APOBEC enzymes as innate mutagens during cancer initiation and evolution. However, the pure mutagenic impacts of the specific enzymes among this family remain unclear in human normal cell lineages. Here, we investigate the comparative mutagenic activities of APOBEC3A and APOBEC3B, through whole-genome sequencing of human normal gastric organoid lines carrying doxycycline-inducible APOBEC expression cassettes. Our findings demonstrate that transcriptional upregulation of APOBEC3A leads to the acquisition of a massive number of genomic mutations in just a few cell cycles. In contrast, despite clear deaminase activity and DNA damage, APOBEC3B upregulation does not generate a significant increase in mutations in the gastric epithelium. APOBEC3B-associated mutagenesis remains minimal even in the context of TP53 inactivation. Further analysis of the mutational landscape following APOBEC3A upregulation reveals a detailed spectrum of APOBEC3A-associated mutations, including indels, primarily 1 bp deletions, clustered mutations, and evidence of selective pressures acting on cells carrying the mutations. Our observations provide a clear foundation for understanding the mutational impact of APOBEC enzymes in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ca9c435f5c5a8910902a03b815334d801814d13" target='_blank'>
              APOBEC3A drives deaminase mutagenesis in human gastric epithelium
              </a>
            </td>
          <td>
            Yohan An, Ji-Hyun Lee, Joonoh Lim, J. Youk, Seongyeol Park, Ji-Hyung Park, Kijong Yi, Taewoo Kim, Chang Hyun Nam, Won Hee Lee, Soo A Oh, Yoo Jin Bae, Thomas M. Klompstra, Haeun Lee, Jinju Han, Junehwak Lee, Jung Woo Park, Jie-Hyun Kim, Hyunki Kim, Hugo Snippert, B. Koo, Young Seok Ju
          </td>
          <td>2025-08-26</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Introduction The epigenetic silencing of O6-methylguanine DNA methyltransferase (MGMT) is associated with reduced DNA repair capacity, carcinogenesis and increased sensitivity to alkylating chemotherapy. However, the biological role and clinical significance of MGMT overexpression in cancer remains poorly understood. Methods Using multiplexed quantitative immunofluorescence we measured the localized levels of MGMT protein, γH2AX and CD8+ T cells in multiple retrospective colorectal cancer (CRC) cohorts. Genomic and transcriptomic features of selected cases were also studied with whole exome DNA sequencing and genome-wide methylation analysis. MGMT-methylated human CRC cells SW620 were transfected with an MGMT-containing plasmid and co-cultured with allogeneic peripheral blood mononuclear cells. Results A subset of CRCs showed MGMT protein upregulation associated with lower γH2AX, reduced CD8+ tumor infiltrating lymphocytes (TILs), mismatch repair proficient (pMMR) status and shorter survival. CD8+ TILs were more distant from MGMT-expressing cells than MGMT-negative cells and the MGMT promoter methylation status did not highly correlate with MGMT protein levels in CRC. In genomic/transcriptomic analysis, high MGMT expression was associated with a lower nonsynonymous somatic mutational burden, higher transition-to-transversion mutation ratio, increased deleterious TP53 variants and distinct transcriptomic profiles. The exogenous expression of MGMT in SW620 CRC cells reduced the number of spontaneous nonsynonymous mutations, reproduced mutational features of MGMT-high CRC and limited the in vitro T-cell-mediated killing of malignant cells induced by proinflammatory cytokines in tumor/immune cell co-cultures. Conclusions MGMT overexpression identifies a previously undescribed subset of CRCs with distinct biological and clinical properties including reduced mutagenesis, adaptive immune evasion, predominantly pMMR phenotype and aggressive clinical course. Direct, quantitative assessment of MGMT protein expression using spatially resolved analysis is more reliable than inference of MGMT expression by promoter methylation status in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43cc0eaa491cd34a4c5b928fccd34ad143431db3" target='_blank'>
              High MGMT expression identifies aggressive colorectal cancer with distinct genomic features and immune evasion properties
              </a>
            </td>
          <td>
            J. Zhang, B. K. Rajendran, Shruti S Desai, Joanna Gibson, Jassim DiPalermo, Patricia LoRusso, Yong Kong, Hongyu Zhao, Michael Cecchini, K. Schalper
          </td>
          <td>2025-09-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is characterized by rapid progression, therapeutic resistance, and a profoundly immunosuppressive tumor microenvironment. Emerging evidence suggests that endoplasmic reticulum (ER)-associated macromolecules play critical roles in tumor adaptation. In this study, we performed a multi-omics investigation of orosomucoid-like protein 2 (ORMDL2), a conserved ER membrane protein involved in sphingolipid biosynthesis and ER stress regulation, and uncovered its regulatory functions in GBM progression. Transcriptomic analyses across The Cancer Genome Atlas (TCGA), and Chinese Glioma Genome Atlas (CGGA) revealed elevated ORMDL2 expression in GBM tissues which causes poor prognosis. The MetaCore pathway and Gene Set Enrichment Analysis (GSEA) identified ORMDL2's involvement in antigen presentation via a major histocompatibility complex I (MHC class I), unfolded protein response (UPR), and mitochondrial apoptotic signaling. Single-cell RNA-sequencing data and the Human Protein Atlas showed ORMDL2 enrichment in tumor stromal cells. Pharmacogenomic correlation via the Genomics in Drug Sensitivity in Cancer (GDSC) and Cancer Therapeutics Response Portal (CTRP) database suggested that ORMDL2 expression was associated with resistance to DNA damage response inhibitors such as etoposide, doxorubicin, talazoparib, and might interact with sphingolipid-targeting compounds. Collectively, our findings establish ORMDL2 as a multi-functional macromolecular regulator of immune suppression and therapeutic resistance in GBM, providing new mechanistic insights and potential targets for translational medicines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f5a5f2b3cb520c8099ca5e5eaf8e443e9c0dbc4" target='_blank'>
              Targeting ORMDL2 in glioblastoma through integrated landscape of high-throughput sequencing and pharmacogenomic analysis
              </a>
            </td>
          <td>
            Fitria Sari Wulandari, Chih-Yang Wang, Sachin Kumar, Ngoc Phung Ly, Juan Lorell Ngadio, Dahlak Daniel Solomon, Do Thi Minh Xuan, Gabriela Natasha Sutandi, Hung-Yun Lin, Hui-Ru Lin, Yung-Kuo Lee, Hao-Chien Yang, , Ching-Chung Ko
          </td>
          <td>2025-09-27</td>
          <td>International Journal of Medical Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="


 Glioblastoma (GBM) remains a devastating disease with a median survival of 15 months. Its molecular definition illustrates underlying unstable genomes with copy number variation (chr 7+/10-), epigenetic modification in DNA repair enzymes (MGMT methylation), and changes to telomere maintenance genes (TERT or ATRX), all recognised as key factors in gliomagenesis and patient outcomes. Telomeres protect chromosome-ends from aberrant DNA damage responses (DDR), and telomere erosion has been shown to drive such genome instability and clonal evolution in several cancers. As more innovative platform trial designs emerge, identifying novel markers to better stratify patients and predict response to targeted therapies is a priority. We conducted single molecule telomere length (TL) analysis and long-read fusion sequencing in GBM patients and related this to clinical outcome.



 146 adults with molecular GBMs had tumour TL assessed with high-throughput single telomere length analysis. Assays for extrachromosomal c-circles and TERT promoter mutation status were performed, to assess underlying telomere maintenance mechanisms. Uni- and multi-variable analysis was conducted to assess tumour TL impact on OS and PFS. Single-molecule PCR was used to identify telomere fusions which were further characterised with long-read nanopore sequencing, including in patients with paired samples from index surgery and at recurrence.



 GBMs displayed shortened telomeres (median TL 4.0kb), unrelated to age at diagnosis or MGMT methylation status. Short TL was negatively predictive for OS on multivariable analysis (HR 4.81, p=0.009). This impact was more pronounced in patients with gross total resection at index surgery (HR 21.55, p=0.0074). Long read Fusion-Seq demonstrated widespread telomere fusions in GBMs and revealed microhomology, telomeric deletions and insertions of genomic loci, including from known protein-coding genes with driver mutations, long non-coding RNA, canonical telomeric repeats, and subtelomeric regions. This mutational spectrum suggests utilisation of specific DNA repair pathways in mediating telomere fusion.



 This study provides mechanistic evidence of dysfunctional telomeres in GBMs, with high resolution TL analysis together with MGMT methylation status providing powerful independent prognostic information. The presence of short telomeres in patients with unmethylated MGMT defines a subgroup of GBM patients that may benefit from alternative treatment regimens. The presence of telomere fusions offers direct evidence for significant genomic instability, potentially favouring targeted DDR inhibitors for patients exhibiting this signature. HB is supported by a WCAT fellowship and Brain Research UK. Work in the Baird Lab is supported by Cancer Research UK and US NIH. Thanks go to BRAIN UK and all our patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5786e68cc90683f81be8d14b8359cb26641b2fd7" target='_blank'>
              P05.18.B TELOMERE DYSFUNCTION AND FUSIONS IN GLIOBLASTOMA - DEFINING PROGNOSIS AND GENOME INSTABILITY
              </a>
            </td>
          <td>
            H. Bhatt, K. Norris, K. Cleal, M. Hodgson, C. Herbert, W. Gray, K. Kurian, D. Baird
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The majority of life-threatening cancers arise from epithelial tissues. These epithelial cancers include cutaneous squamous cell carcinoma (cSCC), the second-most common cancer. cSCC is highly invasive and accounts for an estimated 15,000 deaths each year. We identified SRCAP, a chromatin remodeler that regulates the chromatin occupancy of the histone H2A variant H2A.Z, as a frequently mutated gene in cSCC. Analysis of cSCC mutations in epithelial cancers identified a hotspot truncating mutation in SRCAP, which removes 42% of the protein sequences after amino acid 1879. While SRCAP mutations have been previously connected to the pathogenesis of Floating-Harbor syndrome (FHS), these typically occur downstream, with a hotspot mutation leading to protein truncation after amino acid 2444. We found that expressing the SRCAP-1879 truncation in an HRas-CDK4-driven cSCC model was sufficient to increase proliferation, impair terminal differentiation, and accelerate invasion. Mechanistically, the expression of SRCAP-1879 in primary human keratinocytes was sufficient to dysregulate genes crucial for carcinogenesis (e.g., proliferation, differentiation, and motility) without altering H2A.Z occupancy. In particular, the expression of SRCAP-1879 truncation led to strong induction of the matrix metalloproteinase MMP9 expression level, accompanied by increased keratinocyte cell motility, which was sensitive to matrix metalloprotease inhibition. In contrast, the expression of the SRCAP-FHS truncation did not increase but instead reduced cell motility as well as the expression of MMP9. Taken together, our findings identify a previously under-characterized role of the SRCAP-1879 truncating mutation in promoting multiple aspects of epithelial cancer progression, including invasion, distinct from the well-recognized roles of SRCAP mutations in FHS pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b5e5845afc4ca4f1754b4e5d0d5637c1e90c8a" target='_blank'>
              A dominant SRCAP truncating mutation promotes squamous cell carcinoma progression
              </a>
            </td>
          <td>
            S. Droll, Elena I. O. Dewar, Celia Xue, Maxwell C Levine, Benny J Zhang, Xiaomin Bao
          </td>
          <td>2025-08-26</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13261ddc42d1763ebb460f41abcabf618281627b" target='_blank'>
              Targeting the TRIB3-MYC axis in cancer: mechanistic insights and therapeutic disruption strategies.
              </a>
            </td>
          <td>
            Emadeldin M. Kamel, Sulaiman A. Alsalamah, A. A. Allam, Noha A. Ahmed, F. F. Alkhayl, Al Mokhtar Lamsabhi
          </td>
          <td>2025-09-04</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Numerous cellular processes rely on biomolecular condensates formed through liquid–liquid phase separation (LLPS). Recently, it has become evident that somatic mutations can interfere with or over-activate the formation of phase-separated condensates. Results Here, we set out to systematically study the connection between cancer and biological condensation, specifically mapping the extent to which LLPS is affected in cancer and understanding the molecular pathomechanisms and therapeutic consequences of mutations affecting LLPS scaffolds. We identify both known and novel combinations of molecular functions that are specific to oncogenic fusion proteins and thus have a high potential for driving tumorigenesis. Protein regions driving condensate formation show an increased association with DNA- or chromatin-binding domains of transcription regulators within oncogenic fusion proteins, indicating a common molecular mechanism underlying several soft tissue sarcomas and hematologic malignancies where phase-separation-prone oncogenic fusion proteins form abnormal condensates along the DNA and thereby dysregulate gene expression programs. Conclusions We find that proteins initiating LLPS are frequently implicated in somatic cancers, even surpassing their involvement in neurodegeneration. Our data shows that cancer-driving LLPS scaffolds tend to be potent oncogenes, giving rise to dominant phenotypes and lacking targeting options by current FDA-approved drugs. Finding the currently missing drugs to shut down oncogenic fusion proteins, to disrupt the condensation enabled by them, and to offset their downstream effects could provide cancer drugs widely applicable to diverse cancer incidences previously defying standard treatments. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03787-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54d550a595352429b7ad7fdab681cd564170f566" target='_blank'>
              Phase-separating fusion proteins drive cancer by upsetting transcription regulation
              </a>
            </td>
          <td>
            Nazanin Farahi, Tamas Lazar, Peter Tompa, Bálint Mészáros, Rita Pancsa
          </td>
          <td>2025-10-02</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49580bd801398c666e64f8508dfb8bf04f7a7c7d" target='_blank'>
              Epidrugs in cancer: mechanisms, applications, and future direction.
              </a>
            </td>
          <td>
            Roshni Bibi, Lakshmi Varshetha, Rakshit Kamal Lahary, Jahnavi Namburi, Fardeen Ahmed Laskar, Koustav Sarkar
          </td>
          <td>2025-10-08</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Deciphering the pre-malignant cell of origin (COO) of different cancers is critical for understanding tumor development and improving diagnostic and therapeutic strategies in oncology. Prior work demonstrates that somatic mutations preferentially accumulate in closed chromatin regions of a cancer’s COO. Leveraging this information, we combine 3,669 whole genome sequencing patient samples, 559 single-cell chromatin accessibility cellular profiles, and machine learning to predict the COO of 37 cancer subtypes with high robustness and accuracy, confirming both the known anatomical and cellular origins of numerous cancers, often at cell subset resolution. Importantly, our data-driven approach predicts a basal COO for most small cell lung cancers and a neuroendocrine COO for rare atypical cases. Our study also highlights distinct cellular trajectories during cancer development of different histological subtypes and uncovers an intermediate metaplastic state during tumorigenesis for multiple gastrointestinal cancers, which have important implications for cancer prevention, early detection, and treatment stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba12099e8d95054e5c9ddd1d07125ef4b9a79476" target='_blank'>
              Learning the cellular origins across cancers using single-cell chromatin landscapes
              </a>
            </td>
          <td>
            Mohamad D. Bairakdar, Wooseung Lee, Bruno Giotti, Akhil Kumar, Paula Štancl, Elvin Wagenblast, Dolores Hambardzumyan, P. Polak, R. Karlić, A. Tsankov
          </td>
          <td>2025-09-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Overexpression of the proto-oncogene MYC occurs in >70% of cancers and is especially prevalent in breast cancer. Myc partners with transcription factor Max to bind to the E-box DNA response element. By patterning our frankenproteins on the basic region/helix-loop-helix/leucine zipper motif of Max, we designed MEF and MEF/C93 to bind to the E-box. In bacterial one-hybrid assays and quantitative electrophoretic mobility shift assay, both proteins bound specifically to the E-box with high sequence-specificity and affinity (Kd = 8 nM) rivaling native transcription factors. Quantitative PCR revealed that MEF and MEF/C93 selectively downregulated Myc target genes in Myc-dependent MDA-MB-231 breast cancer cells, but not in Myc-independent MCF-7 cells. Fluorescence colocalization demonstrated transport into cell nuclei. Our proteins displayed IC50 values of 1-2 μM in cell viability assays in MDA-MB-231, compared with ∼25 μM in MCF-7. These results demonstrate the specificity of targeting Myc-dependent cancer cells and mark significant progress toward protein-based therapies aimed at Myc-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8cc39bbdc2499f24c52dd2589dd7f515be4ffe6" target='_blank'>
              Designer Frankenproteins That Halt the Proliferation of Myc-Driven Cancer Cells.
              </a>
            </td>
          <td>
            Maryam Ali, Raneem Akel, Francine He, Micheline Piquette-Miller, Jumi A Shin
          </td>
          <td>2025-09-03</td>
          <td>Journal of medicinal chemistry</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="Anti-bacteriophage systems like restriction-modification and CRISPR-Cas have DNA substrate specificity mechanisms that enable identification of invaders. How Gabija, a highly prevalent nuclease-helicase anti-phage system, executes self- vs. non-self-discrimination remains unknown. Here, we propose that phage-encoded DNA end-binding proteins that antagonize host RecBCD sensitize phages to Gabija. When targeting a temperate Lambda-like phage in Pseudomonas aeruginosa, Gabija protects the cell by preventing phage genome circularization and subsequent replication. Phage and plasmid sensitivity to Gabija is licensed by DNA end-binding complexes such as a phage exonuclease together with a ssDNA-annealing protein or GamMu dimers, which prevent loading of host repair complex RecBCD. Escape phages lacking these end-binding proteins become protected from Gabija by RecBCD translocation activities. RecBCD activity on the bacterial genome also prevents Gabija from targeting self-DNA. Therefore, we propose that Gabija antagonizes circularization of linear DNA devoid of RecBCD as a mechanism to identify foreign invaders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daea778f2bb1880e2bb5de9ac8bdf131b6aa7185" target='_blank'>
              Gabija restricts phages that antagonize a conserved host DNA repair complex
              </a>
            </td>
          <td>
            Alex Hong, Miaoxi Liu, Alexis Truta, Alexander Talaie, Gerald R Smith, Joseph Bondy-Denomy
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Class switch recombination (CSR) and locus suicide recombination (LSR) are critical processes involved in the immune system’s ability to diversify antibody responses. Both are initiated by activation-induced cytidine deaminase, which induces DNA double-strand breaks (DSBs) at specific regions within the immunoglobulin heavy chain (IgH) locus. In CSR, DSBs occur at the switch (S) regions, allowing B cells to replace the IgM heavy chain constant region (CH) with other isotypes, thereby enhancing immune adaptability. This process is regulated by both cis and trans mechanisms, including the IgH super-enhancer 3′ regulatory region (3′RR) and the production of enhancer RNAs (eRNAs). A recent study highlighted the role of MED12 in CSR through enhancer activation and the transcription of eRNA. Now, we show that heterogeneous ribonucleoprotein L (hnRNPL) acts as an additional regulator of CSR and LSR by forming an eRNA-associated complex with CstF64, a polyadenylation factor. This complex facilitates RNA polymerase II elongation and eRNA transcription at the 3′RR. Moreover, the hnRNPL/CstF64 complex promotes NHEJ-mediated DNA repair at both S and 3′RR regions, facilitating 53BP1 and Ku80 recruitment, thereby impacting the efficiency of CSR and LSR. This discovery highlights the intricate, multimodal regulation of these processes, linking eRNA transcription to DNA repair in the process of antibody diversification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d7bc08d8df641c457cf3f47b9d1b5907694cc1f" target='_blank'>
              hnRNPL–CstF64 complex: coordinating CSR and LSR in IgH locus recombination dynamics through eRNA and NHEJ regulation
              </a>
            </td>
          <td>
            Farazul Haque, Mikiyo Nakata, H. Kosako, Tasuku Honjo, N. Begum
          </td>
          <td>2025-08-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Abstract Transposable elements (TEs), once considered genomic “junk”, are now recognized as critical regulators of genome function and human disease. These mobile genetic elements—including retrotransposons (long interspersed nuclear elements [LINE-1], Alu, short interspersed nuclear element-variable numbers of tandem repeats-Alu [SVA], and human endogenous retrovirus [HERV]) and DNA transposons—are tightly regulated by multilayered mechanisms that operate from transcription through to genomic integration. Although typically silenced in somatic cells, TEs are transiently activated during key developmental stages—such as zygotic genome activation and cell fate determination—where they influence chromatin architecture, transcriptional networks, RNA processing, and innate immune responses. Dysregulation of TEs, however, can lead to genomic instability, chronic inflammation, and various pathologies, including cancer, neurodegeneration, and aging. Paradoxically, their reactivation also presents new opportunities for clinical applications, particularly as diagnostic biomarkers and therapeutic targets. Understanding the dual role of TEs—and balancing their contributions to normal development and disease—is essential for advancing novel therapies and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9a4403306942197c6791291eb7e939f11dbad7" target='_blank'>
              Transposable elements in health and disease: Molecular basis and clinical implications
              </a>
            </td>
          <td>
            Yaqiang Hong, Nian Liu
          </td>
          <td>2025-08-15</td>
          <td>Chinese Medical Journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="MYCN and c-MYC are critical driver oncogenes in several childhood cancers, including neuroblastoma. Currently, the clinical development of MYC inhibitors has been hindered by the intrinsically disordered structure of MYC proteins, which lack well-defined ligand-binding pockets. Proliferation-associated protein 2G4 (PA2G4) directly binds to and stabilizes MYCN protein, leading to markedly increased MYCN levels in neuroblastoma cells. Here, we demonstrate that PA2G4 is essential for MYCN-driven tumor growth in neuroblastoma in vivo. Moreover, PA2G4 elevates c-MYC protein levels in neuroblastoma cells by inhibiting its ubiquitin-mediated degradation. In turn, c-MYC upregulates the transcription and protein expression of PA2G4, creating an oncogenic feed-forward expression loop. A small molecule PA2G4 inhibitor, WS6, directly disrupts the PA2G4-c-MYC protein–protein interaction, resulting in decreased levels of both PA2G4 and c-MYC. WS6 exhibited selective cytotoxicity in c-MYC-overexpressing cell lines. Together, these findings identify PA2G4 as a shared cofactor for both the c-MYC and MYCN oncoproteins and highlight its interaction with MYC family oncoproteins as a promising therapeutic vulnerability in MYC-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c59a9ccaa43b0ac475d7ab175811386fe341ce8d" target='_blank'>
              PA2G4 Functions as a Cofactor for MYC Family Oncoproteins in MYC-Driven Malignancies
              </a>
            </td>
          <td>
            Sukriti Krishan, J. Koach, Taylor Lim, Kenny K. L. Yeo, Faith Cheong, Jie-Si Luo, Hassina Massudi, Xiaomian Gao, Sopheakwealthy Heangsarath, Andrew J Kueh, Marco J. Herold, B. Cheung, Glenn M. Marshall
          </td>
          <td>2025-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background Lower-grade glioma (LGG) is a common primary brain tumor with potential progression to glioblastoma. Its development is closely linked to DNA damage repair. DNA polymerase delta subunit 3 (POLD3), a subunit of the DNA polymerase δ complex, participates in DNA replication and repair. POLD3 may act as a therapeutic target and prognostic biomarker in LGG. Methods We collected primary data from The Cancer Genome Atlas (TCGA), the Chinese Glioma Genome Atlas (CGGA), the Tumor Immune Estimation Resource (TIMER), and Gene Set Enrichment Analysis (GSEA). The CGGA dataset included messenger RNA (mRNA) sequencing and clinical data from 403 LGG cases; its microarray subset contained 142 additional cases. The TCGA dataset provided clinical data for 503 LGG cases. Employing bioinformatics tools, we investigated the correlation of POLD3 expression with the clinical characteristics, pathological features, survival outcomes, and tumor immune microenvironment in LGG. Subsequent functional validation of POLD3’s impact in LGG was performed using both cellular and animal models. Results Integrative analysis combining bioinformatics and experimental approaches revealed significant POLD3 overexpression in LGG. Elevated POLD3 levels exhibited significant associations with distinct clinicopathological features, altered immune cell infiltration profiles, and adverse clinical outcomes. Functionally, POLD3 was demonstrated to drive LGG cell proliferation, invasion, and modulate the tumor microenvironment. Conclusion This study establishes POLD3 as a critical molecular determinant in LGG pathogenesis. Our findings position POLD3 as a promising multifaceted target, with significant implications for advancing LGG diagnostics, therapeutic development, immune modulation strategies, and prognostic assessment, thereby providing a fresh conceptual framework for understanding and managing this tumor. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14934-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1b766711209d5df5ec31c2b94f1204021afe9d8" target='_blank'>
              POLD3 knockdown effects on low-grade glioma: insights from bioinformatics and experimental validation
              </a>
            </td>
          <td>
            Zhihui Yang, Zhonghao Dou, Hao Wang, Zhaojiang Hu, Yanbo Gao, Yilei Bai, Junfeng Liu, Rongjun Qian
          </td>
          <td>2025-10-01</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Aberrant expression of long noncoding RNAs (lncRNAs) serves as a critical driver in carcinogenesis and tumor evolution across various human cancers. Among these lncRNAs, small nucleolar RNA host gene 15 (SNHG15) has emerged as a key regulator in multiple cellular processes. Its dysregulation is frequently implicated in disease pathogenesis, particularly cancer. In this investigation, we delineated the functional landscape and mechanistic basis of SNHG15 in lung carcinoma. Cell cycle synchronization experiments demonstrated phase-dependent SNHG15 expression, showing maximal transcript abundance during G2/M transition. SNHG15 knockdown substantially curbed tumor growth in vitro (A549 cell line) and in vivo (subcutaneous PDX models). Notably, SNHG15 orchestrated genomic surveillance mechanisms through tripartite regulation: (1) cell cycle checkpoint coordination, (2) transcriptional machinery reactivation, and (3) nucleotide excision repair. Mechanistically, proteomic profiling coupled with RNA immunoprecipitation identified CREB5 as a direct binding partner mediating SNHG15-dependent survival signaling under genotoxic stress. These results position the SNHG15-CREB5 axis as an important player governing genomic fidelity in lung adenocarcinoma, suggesting druggable targets for precision oncology approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca1f937a74141f7d8c6f5eb7bb745d4f086d4cf3" target='_blank'>
              The role of lncRNA SNHG15 in UV-induced DNA damage repair
              </a>
            </td>
          <td>
            Xin Zhang, Jiahui Lin, Zixuan Fan, Sai Zhang, Meiqi Yang, Tingting Zhu, Qing Lu, Yuming Wang
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT The skin microbiome is composed of diverse microbial communities that engage in interkingdom interactions, influencing host physiology and microbial balance. Although Malassezia restricta and Staphylococcus epidermidis are codominant members of the human skin microbiome, the molecular mechanisms underlying their interactions remain poorly understood. We aimed to investigate the mechanism by which S. epidermidis-derived acetic acid affects chromatin organization and gene expression in M. restricta. S. epidermidis modulated chromatin structure and transcriptional activity in M. restricta by secreting acetic acid (AcOH), a common skin-associated organic acid. Using in situ Hi-C, we established the first three-dimensional genome architecture map of M. restricta and identified putative centromeric loci based on inter-chromosomal association scores. Co-culture with S. epidermidis or direct treatment with AcOH induced large-scale chromatin decompaction and enhanced centromeric clustering, indicating significant reorganization of the nuclear architecture. Through chromatin immunoprecipitation (ChIP)-seq analysis, we observed that AcOH exposure led to a redistribution of histone acetylation from promoter regions to gene bodies. This chromatin remodeling was further associated with extensive transcriptional repression, particularly of genes involved in translation, metabolism, and virulence, as revealed by RNA-seq analysis. Of note, these changes were specific to AcOH and were not replicated under inorganic acid stress (HCl), indicating a metabolite-specific epigenetic response. This study reveals a novel form of interkingdom communication in the skin microbiome, in which S. epidermidis-derived AcOH acts as an epigenetic modulator in M. restricta. Our findings provide key mechanistic insights into how bacterial metabolites influence fungal chromatin architecture and transcription, with implications for microbial community dynamics and skin health. IMPORTANCE This study provides essential insights into interkingdom interactions within the human skin microbiome, highlighting how microbial metabolites influence fungal biology at the chromatin level. Specifically, we identify acetic acid (AcOH), secreted by Staphylococcus epidermidis, as a key regulator that induces significant chromatin remodeling and transcriptional changes in Malassezia restricta. By presenting the first three-dimensional genome architecture map of M. restricta, our findings uncover metabolite-specific chromatin dynamics that cannot be replicated by inorganic acid stress. Additionally, the conservation of this chromatin response in other Malassezia species suggests broader implications for understanding microbial adaptation mechanisms in the skin environment. This work underscores the critical role of bacterial metabolites as modulators of microbial interactions and provides new avenues for investigating microbial community balance and potential therapeutic strategies for skin health.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f45e0bd555bafab74ee26abfc65e0d5026258f51" target='_blank'>
              Acetic acid produced by Staphylococcus epidermidis remodels chromatin architecture and suppresses gene expression in Malassezia restricta
              </a>
            </td>
          <td>
            Jae Min Lee, Hyun Oh Yang, Hideki Tanizawa, Ken-ichi Noma, Tae Kwon Lee, W. H. Jung, Yong-Joon Cho, Kyoung-Dong Kim
          </td>
          <td>2025-09-12</td>
          <td>mBio</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Polymerase theta (POLQ) is essential for theta-mediated end joining (TMEJ), an error-prone DNA repair pathway that is hyperactive in homologous recombination-deficient (HRD) cancers such as high-grade serous ovarian cancer (HGSOC). Inhibiting POLQ is a promising therapeutic strategy, but the molecular determinants of sensitivity and downstream mechanisms remain incompletely defined.



 We assessed the growth-inhibitory activity of two structurally distinct POLQ inhibitors, novobiocin and ART558, across 18 well-characterized HGSOC cell lines. Genomic profiling data were integrated with drug response values (GR50s) to identify predictive biomarkers, including mutational status, HRD scores, and copy number signatures (CX1–CX17). Mechanistic assays included POLQ ATPase activity, cell cycle analysis (FUCCI), immunofluorescence for γH2AX and POLQ, apoptosis markers and DNA fiber assays.



 POLQ inhibitor sensitivity varied independently from olaparib or carboplatin response, indicating a distinct mode of action. Novobiocin sensitivity significantly correlated with CX3 and CX5 copy number signatures, which reflect impaired homologous recombination and replication stress. In contrast, resistant lines were enriched for CX14 and CX6, signatures linked to mitotic errors and telomere shortening. Sensitive cell lines showed increased DNA damage (γH2AX foci), POLQ accumulation on chromatin, and apoptosis, particularly in BRCA1/2-mutant backgrounds. Cell cycle profiling revealed impaired G1/S progression and S-phase depletion. DNA fiber assays demonstrated replication fork degradation upon POLQ inhibition, consistent with POLQ’s role in fork protection.



 We identify genomic features, specifically CX3 and CX5 copy number signatures as candidate biomarkers of POLQ inhibitor sensitivity in HGSOC. Mechanistically, POLQ inhibition induces replication stress, persistent DNA damage, and apoptosis in HR-deficient settings. These findings support the clinical development of POLQ inhibitors and rational patient stratification based on genome instability signatures.



 Aleksandra Kruglikov. POLQ inhibitors induce replication stress and apoptosis in high-grade serous ovarian cancer: Functional and genomic biomarkers of response [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A037.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fe824b1195032b879fa6dbdb3ed12cc08d24058" target='_blank'>
              Abstract A037: POLQ inhibitors induce replication stress and apoptosis in high-grade serous ovarian cancer: Functional and genomic biomarkers of response
              </a>
            </td>
          <td>
            Aleksandra Kruglikov
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Berberine (BBR) is an isoquinoline alkaloid with a variety of biological activities, including anti‐microbial and anti‐tumoral activities. However, the cellular targets of BBR and the roles of BBR in the radiosensitivity of breast cancer cells are not well defined. In this study, we investigated the effects of BBR on the radiosensitivity of BT549 triple‐negative breast cancer cells. Through RNA‐seq results, we found that BBR significantly down‐regulated the expression of DNA ligases. The results of western blot and comet assay confirmed that BBR attenuated DNA ligase III (LIGIII) expression and caused DNA damage in a dose‐dependent manner. The results of electron microscopy showed that BBR decreased mitochondrial copies and induced mitochondrial autophagy. Clonal formation analysis showed that BBR sensitised breast cancer cells to irradiation. The genetic complementation and interference assays showed that the effect of BBR on the radiosensitivity of BT549 breast cancer cells is dependent on the expression of LIGIII. These results contribute to the understanding of the effects of BBR on cellular DNA repair and the clinical application of BBR in breast cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d7148d132ad7e87d8b8ae13f9a1fb9fcaf3cd90" target='_blank'>
              Berberine Sensitises Breast Cancer Cells to Radiation via the Attenuation of DNA Ligase III
              </a>
            </td>
          <td>
            Yuxin Sun, Cong Li, Hang Yin, Kunyan Li, Ying Wang, Shuailong Zhang, Yi Zhao, Jing Wang, Weifeng Mao
          </td>
          <td>2025-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 In this talk I will discuss new insights into the nature of tumor evolution in high grade serous ovarian cancer (HGSOC). I will focus on two key areas that have been informed by recent advances in single cell whole genome sequencing. First, I will discuss the nature of how HGSOCs acquire genome doubled states in their evolutionary histories. I will show that genome doubling is an ongoing process throughout evolution with important consequences for phenotypic states and immunosuppression. Second, I will discuss recent work to track evolution from diagnosis to relapse using structural variants and cell-free DNA technology. I will show how clonal populations either contract or persist over a multi-year intervals in patients with relapsed disease and highlight genomic structural features linked to the evolution of drug resistance.



 Sohrab Shah. Frontiers of tumor evolution in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr IA024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c181ca2707243d8dc30253d7814569567a3fe09e" target='_blank'>
              Abstract IA024: Frontiers of tumor evolution in ovarian cancer
              </a>
            </td>
          <td>
            Sohrab P. Shah
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background The dysregulation of the DNA damage response (DDR) plays a vital role in cancer development and inadequate response to anti-cancer therapy. We investigated the involvement of DNA polymerase theta (Polθ), the principal component of the microhomology-mediated end joining (MMEJ) pathway, in the pathogenesis of multiple myeloma (MM). Methods We explored the expression levels of Polθ and assessed its prognostic relevance in a cohort of 124 MM patients at our institution. We demonstrated the impacts of Polθ inhibition on genomic stability and cell viability in MM using shRNA and a pharmacological inhibitor. To elucidate the role of Polθ-mediated MMEJ in repairing melphalan-induced DNA damage, we employed Western blot and EJ2-GFP reporter assay to evaluate the activation of the MMEJ pathway following melphalan exposure. We analyzed the synergistic cytotoxicity between the Polθ inhibitor and melphalan in MM, both in vitro and in vivo. The impacts of combination therapy on genomic damage and mitotic catastrophe were examined by comet assay, Western blot, and immunofluorescence. Results We revealed that Polθ was markedly upregulated in MM. The enhanced expression of Polθ was found to be significantly associated with advanced disease stages and adverse clinical outcomes. Polθ inhibition promoted genomic instability and impaired MM growth, underscoring the polymerase's function in sustaining myeloma survival under ongoing genomic stress. Furthermore, we observed a further increase in Polθ expression and MMEJ repair efficiency following melphalan treatment. Polθ inhibitors sensitized MM cells to melphalan by increasing unrepaired DNA damage and inducing mitotic catastrophe, both in vitro and in vivo. Conclusions Collectively, our results confirm Polθ as a promising prognostic biomarker and highlight its potential to serve as a synthetic lethal target that can be used in combination with melphalan for the treatment of MM. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-07065-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a916ab34fef94b64abac58ae251e6ce253205a64" target='_blank'>
              Polymerase theta inhibition impairs tumor growth and amplifies melphalan-induced DNA damage in multiple myeloma
              </a>
            </td>
          <td>
            Qun Li, Chi Ma, Liping Zuo, Aoshuang Xu, Bo Zhang, Fei Zhao, Lifei Liu, Xin Zhao, Yu Hu, Chunyan Sun
          </td>
          <td>2025-10-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Breast cancer (BCa) is a major contributor to female mortality worldwide. Treatment resistance and tumor heterogeneity contribute to the lack of effective therapeutic targets, posing a significant challenge in BCa management. CENPI, a core centromere protein involved in chromosome segregation, has emerging evidence implicating it in oncogenesis across diverse malignancies. However, its functional and molecular mechanisms in BCa remain unclear. Methods We analyzed CENPI expression and its clinical significance by using the BCa dataset from the Cancer Genome Atlas (TCGA) and immunohistochemical staining of 3 human BCa tissue samples. Cellular functional assays and mice xenograft models were utilized to assess the effects of CENPI on BCa growth. RNA sequencing combined with bioinformatics analysis was conducted to elucidate the molecular mechanisms underlying CENPI function, with further validation through Western blotting, immunofluorescence, and TOP/FOP flash assays. Results CENPI was aberrantly overexpressed in BCa, with elevated expression levels strongly associated with disease progression and poor prognosis. Functional assays demonstrated that CENPI significantly promoted breast carcinogenesis in both cellular and animal models. Mechanistically, CENPI increased BCa progression and malignant phenotypes by modulating the Wnt/β-catenin axis. Conclusions CENPI is a critical oncogene in BCa, driving tumorigenesis and disease progression via the Wnt/β-catenin axis, which represents a promising biomarker and therapeutic target for BCa. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-04001-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33eef316f873dd58287f9f4d71241b7970f53004" target='_blank'>
              Centromere protein I facilitates breast cancer tumorigenesis and disease progression through modulation of Wnt/β-Catenin signaling
              </a>
            </td>
          <td>
            Chaoshen Wu, Yibing Zhou, Yuxiao Mu, Yijie Cheng, Wenqian Xu, Mengna Huang, Jingyuan Xu, Ming Li, Jianfeng Gu, Xuli Meng, Da Qian
          </td>
          <td>2025-10-10</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 There is global epigenetic dysregulation during transformation from normal to cancerous cells. One of the many consequences of this global epigenetic dysregulation is the reactivation of transposable elements (TEs). TEs comprise roughly half of the human genome and are typically repressed through epigenetic mechanisms, including DNA methylation and histone modifications. Endogenous retroviruses (ERVs) are a class of TEs that contain long terminal repeats (LTRs) with putative gene regulatory motifs. LTRs can be co-opted to be pro-tumorigenic by becoming non-canonical promoters for oncogenic genes. While several examples of TE-derived promoters have been reported in cancer, the potential significance of TE-derived enhancers remains largely unexplored. Here, we utilized publicly available multi-omics datasets (bulk and single-cell chromatin profiling and RNA-sequencing) of high-grade serous carcinoma (HGSC) and fallopian tube (FT) tissues to determine if there are tumor-specific, TE-derived enhancers that have gene regulatory function. Once putative enhancers were identified, we used Clustered Regularly Interspaced Short Palindromic Repeats interference (CRISPRi) to functionally validate a putative TE-derived enhancer and gene regulatory axis. We discovered that on the bulk tissue level, several TE families, notably LTR2B, LTR18A, and LTR18B, are enriched at open chromatin sites and sites marked by H3K27Ac (enhancer mark) in HGSC cells compared to FT cells, suggesting the presence of HGSC-specific, TE-derived enhancers. On the single-cell level, we discovered that TE transcripts from the same families are more highly expressed in tumor cells compared to stromal and immune cells in the tumor microenvironment, suggesting that the increased TE-derived enhancer activity observed on the bulk level is specific to tumor cells. Lastly, we discovered that silencing a specific LTR18A locus upstream of the TCDD-Inducible Poly ADP Ribose Polymerase (TIPARP) gene was sufficient to repress TIPARP gene expression suggesting that these putative TE-derived enhancers have a gene regulatory function. TIPARP expression has been shown to be correlated with HGSC, and our studies show a novel mechanism by which TIPARP may be regulated in HGSC. Determining the regulatory role of these TE-derived enhancers will provide novel insights into not only HGSC, but also other cancer-related transcriptomes.



 Lily Nguyen, Abigail Jeong, Atma Ivancevic, Benjamin Bitler, Edward B. Chuong. Transposable element derived enhancers in ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e45139333a87abd3b3e296653a1c558db2322706" target='_blank'>
              Abstract B012: Transposable element derived enhancers in ovarian cancer
              </a>
            </td>
          <td>
            Lily Nguyen, Abigail D Jeong, A. Ivancevic, Benjamin G. Bitler, Edward B. Chuong
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708353dcfdc8ca7c4e2b434fced2427c746e282e" target='_blank'>
              Tracing the evolution of single-cell 3D genomes in Kras-driven cancers.
              </a>
            </td>
          <td>
            Miao Liu, Shengyan Jin, Sherry S Agabiti, Tyler B Jensen, Tianqi Yang, Jonathan S. D. Radda, Christian F. Ruiz, Gabriel Baldissera, Moein Rajaei, Fangyong Li, Jeffrey P. Townsend, M. Muzumdar, Siyuan Wang
          </td>
          <td>2025-08-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Epigenetic regulation is critical to B cell development, guiding gene expression via DNA methylation, histone modifications, chromatin remodeling, and noncoding RNAs. In mature B cell neoplasms, particularly diffuse large B cell lymphoma (DLBCL), follicular lymphoma (FL), and chronic lymphocytic leukemia (CLL), these mechanisms are frequently disrupted. Recurrent mutations in key epigenetic regulators such as EZH2, KMT2D, CREBBP, and TET2 lead to altered chromatin states, repression of tumor suppressor genes, and enhanced oncogenic signaling. Dysregulation of specific microRNAs (e.g., miR-155, miR-21) further contributes to pathogenesis and therapeutic resistance. In DLBCL, hypermethylation of SMAD1 and CREBBP mutations are associated with immune evasion and chemoresistance. In FL, EZH2 gain-of-function and KMT2D loss-of-function mutations alter germinal center B cell programming, while in CLL, DNA hypomethylation patterns reflect the cell of origin and correlate with clinical outcome. Targeted therapies such as the EZH2 inhibitor tazemetostat have demonstrated efficacy in EZH2-mutant FL, while HDAC and BET inhibitors show variable responses across B cell malignancies. The limitations of current epigenetic therapies reflect the complexity of targeting epigenetic dysregulation rather than therapeutic futility. These challenges nonetheless highlight the relevance of epigenetic alterations as biomarkers and therapeutic targets, with potential to improve the management of mature B cell neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a068788c20e74398875dae829a5f27eeefbe604a" target='_blank'>
              Unraveling the Epigenetic Landscape of Mature B Cell Neoplasia: Mechanisms, Biomarkers, and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Nawar Maher, F. Maiellaro, J. Ghanej, S. Rasi, R. Moia, G. Gaidano
          </td>
          <td>2025-08-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28b36c19d01be738ee1237dc98c176d6789c1e47" target='_blank'>
              Chromatin-associated circRNA ciCRLF3(2) regulates cell differentiation blockage via activating non-homologous end joining-based DNA repair.
              </a>
            </td>
          <td>
            Ke-Jia Pu, Xiao-Tong Chen, Shun-Xin Zhu, Yan An, Xin-Yi Feng, Heng-Jing Huang, Cheng-Lin Zhou, Mei-Ying Ye, Yun-Chun Wei, Yi-Xuan Ma, Chen Fang, Nan Zhang, Dan Wang, Bin-Rong Han, Jun-Yi Lian, Tian-Qi Chen, Yumeng Sun, Yue-Qin Chen, Wentao Wang
          </td>
          <td>2025-09-04</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Small-cell lung cancer (SCLC), an aggressive neuroendocrine tumor strongly associated with exposure to tobacco carcinogens, is characterized by early dissemination and dismal prognosis with a five-year overall survival of less than 7%. High-frequency gain-of-function mutations in oncogenes are rarely reported, and intratumor heterogeneity (ITH) remains to be determined in SCLC. Here, via multiomics analyses of 314 SCLCs, we found that the ASCL1+/MKI67+ and ASCL1+/CRIP2+ clusters accounted for 74.38% of the 190,313 SCLC cancer cells from 39 patients, with the ASCL1+SOX1+ stem-like cell cluster across SCLC subtypes. The major histocompatibility complex (MHC) class I molecules were expressed at low levels in six and high levels in five cancer cell clusters and were inversely associated with the KI67 expression level. Abnormal splicing of mRNAs was a feature of SCLC, with focal adhesion kinase (FAK) splicing variants identified in 119 (77.3%) of 154 patients. FAK variants exhibited elevated kinase activity, were associated with the worst prognosis, and were sensitive to FAK inhibitors in patient-derived organoids and xenograft models. Eleven high-frequency mutations were identified in addition to TP53 and RB1, and smoking status and tumor stage did not affect microbiota variance in SCLC. Taken together, our data further revealed the complicated ITH and discovered that FAK splicing variants represent high-frequency gain-of-function alterations in oncogene in SCLC and potential therapeutic targets for this recalcitrant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3bde80c19d7166cce5785be8ae3a4a5e971fbbd" target='_blank'>
              Characterization of the extrinsic and intrinsic signatures and therapeutic vulnerability of small cell lung cancers
              </a>
            </td>
          <td>
            Gui-zhen Wang, Zheng Wang, Shi-Hao Bai, Yun Tan, Wen-Zhao Zhong, Guo-Gui Sun, Yu-Tao Liu, B. Pan, Chen Huang, Di Wang, Beibei Sun, Dong-Ni Chen, Bin Zhang, Yong-Chun Zhou, Sheng Li, Xiang-Wei Zhang, Si-Chong Han, Fu-ying Yang, Xue-Yan Shi, Xiao-Liang Jie, Yu-Ke Shen, L. Liang, Zhe-Sheng Wen, Li Zhang, Mingkun Li, Na Wang, Jin-Song Liu, Ying Dong, Manli Wang, Yan Wang, Chang-Li Wang, Dawei Xie, Ze-Guang Han, Jian-Ming Ying, Chong Chen, Yunchao Huang, Hongbin Ji, Yuan-Yuan Zhang, Yan Yu, Guang-Biao Zhou
          </td>
          <td>2025-09-10</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Repeated oncogenic mutations and polyclonal proliferation are evident in cancers. However, little is known about the polyclonal principles governing the systemic cancerous lineage during immunotherapy. Here, we examine a unique autopsy case of metastatic urothelial carcinoma that exhibits different treatment responses to anti-PD-1 therapy at each tumor site. By performing in-depth analyses of different multiregional bulk tumor masses, we reveal that subsets of subclones acquire potential driver mutations under treatment selection pressure. Spatial transcriptomics analysis reveals that subclones resistant to immunotherapy form distinct immunosuppressive environments consistent with their habitats. Furthermore, different cancer hallmarks are identified in each of the subclones that expand under immunotherapy at single-cell level; for example, one subclone is more proliferative, and another is more stem-cell-like. In summary, this study provides an overall picture of the polyclonal competition and changes in the immune microenvironment that are related to resistance to immunotherapy in patients with malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f357f9c7e57b9e95e8ee48a6378f3d690f3275bd" target='_blank'>
              Clonal diversity shapes the tumour microenvironment leading to distinct immunotherapy responses in metastatic urothelial carcinoma
              </a>
            </td>
          <td>
            Takashi Kamatani, K. Umeda, T. Iwasawa, Fuyuki Miya, K. Matsumoto, S. Mikami, Kensuke Hara, Masayuki Shimoda, Yutaka Suzuki, Jo Nishino, Mamoru Kato, Kazuhiro Kakimi, Nobuyuki Tanaka, M. Oya, T. Tsunoda
          </td>
          <td>2025-08-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Background DNA methylation is an epigenetic mechanism involved in gene regulation and cellular differentiation. Accurate and comprehensive assessment of DNA methylation patterns is thus essential for understanding their role in various biological processes and disease mechanisms. Bisulfite sequencing has long been the default method for analyzing methylation marks due to its single-base resolution, but the associated DNA degradation poses a concern. Although several methods have been proposed to circumvent this issue, there is no clear consensus on which method might be better suited for specific study designs. Results We conducted a comparative evaluation of four DNA methylation detection approaches: whole-genome bisulfite sequencing (WGBS), Illumina methylation microarray (EPIC), enzymatic methyl-sequencing (EM-seq) and third-generation sequencing by Oxford Nanopore Technologies (ONT). DNA methylation profiles were assessed across three human genome samples derived from tissue, cell line, and whole blood. We systematically compared these methods in terms of resolution, genomic coverage, methylation calling accuracy, cost, time, and practical implementation. EM-seq showed the highest concordance with WGBS, indicating strong reliability due to their similar sequencing chemistry. ONT sequencing, while showing lower agreement with WGBS and EM-seq, captured certain loci uniquely and enabled methylation detection in challenging genomic regions. Despite a substantial overlap in CpG detection among methods, each method identified unique CpG sites, emphasizing their complementary nature. Conclusions Our findings underscore the strengths and limitations of current DNA methylation detection methods. EM-seq and ONT emerge as robust alternatives to WGBS and EPIC, offering unique advantages: EM-seq delivers consistent and uniform coverage, while ONT excels in long-range methylation profiling and access to challenging genomic regions. These insights provide practical guidance for method selection based on specific experimental goals. Supplementary Information The online version contains supplementary material available at 10.1186/s13072-025-00616-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1115f5c6b6dae937448bce4077f0951e9ac581f3" target='_blank'>
              Comparison of current methods for genome-wide DNA methylation profiling
              </a>
            </td>
          <td>
            Ana Regina de Abreu, J. Ibrahim, Vasileios Lemonidis, Ligia Mateiu, Guy Van Camp, K. Op de Beeck
          </td>
          <td>2025-08-25</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3493648729eb60ce21f81ce27852f8a58bffd93" target='_blank'>
              Influence of Epigenetic Regulators Expression on Gastric Cancer Prognosis
              </a>
            </td>
          <td>
            Lucas Brabo Rotella, A. Khayat, Monique Feitoza, Victoria Pereira Costa, Diego Pereira, Fabiano Cordeiro Moreira, S. Demachki, Samir Mansour Moraes Casseb, G. Ishak, W. Barra, R. Burbano, P. P. D. Assumpção
          </td>
          <td>2025-09-15</td>
          <td>None</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Histones are key epigenetic factors that regulate the accessibility and compaction of eukaryotic genomes, affecting DNA replication and repair, and gene expression. Recent studies have demonstrated that histone missense mutations can perturb normal histone function, promoting the development of phenotypically distinguishable cancers. However, most histone mutations observed in cancer patients remain enigmatic in their potential to promote cancer development. To assess the oncogenic potential of histone missense mutations, we have gathered whole-exome sequencing data for the tumors of about 12 000 patients. Histone mutations occurred in about 16% of cancer patients, although specific cancer types showed substantially higher rates. Using genomic, structural, and biophysical analyses, we found several predominant modes of action by which histone mutations may alter function. Namely, cancer missense mutations primarily affected histone acidic patch residues and protein-binding interfaces in a cancer-specific manner and targeted interaction interfaces with specific DNA repair proteins. Consistent with this finding, we observed a high tumor mutational burden in patients with histone mutations affecting interactions with proteins involved in maintaining genome integrity. We identified potential cancer driver mutations in several histone genes, including mutations on histone H4—a highly conserved histone without previously documented driver mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8347f132028127afcb5dcde491fcf3c72af1a050" target='_blank'>
              Cancer histone mutations impact protein binding and DNA repair with possible links to genomic instability
              </a>
            </td>
          <td>
            Daniel Espiritu, Yiru Sheng, Yunhui Peng, Daria Ostroverkhova, Shuxiang Li, David Landsman, Maria J Aristizabal, Anna R. Panchenko
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cutaneous melanoma (CM), with a continuously rising incidence worldwide, represents the most aggressive type of skin cancer, and it leads to the majority of skin cancer-related deaths. Approximately 50% of CM carry the activating BRAFV600 mutation and, although BRAF inhibitors have demonstrated clinical efficacy, most patients often develop early resistance to treatment. Aberrant expression of non-coding RNAs (ncRNAs), which represent less than 2% of the entire transcriptome, has been implicated in CM development and progression. By using BRAFV600-mutant CM in vitro and in vivo models, we have recently demonstrated that the loss of Spry1 expression impairs BRAFV600-mutant CM progression. Therefore, the extensive long and small ncRNA datasets generated in this study might represent a valuable resource for the characterization of their roles in BRAFV600-mutant CM initiation and progression upon Spry1 loss, thus providing a comprehensive resource to support future studies on BRAFV600-mutant CM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c46752f6a68d20467ff6b95252e5ee4402a37" target='_blank'>
              Non-coding RNA profiling in BRAFV600E-mutant cutaneous melanoma before and after Spry1 depletion
              </a>
            </td>
          <td>
            J. Lamberti, Domenico Memoli, Barbara Montico, Francesco Silvestro, R. Guerrieri, F. Colizzi, Alessandro Weisz, A. Salvati, E. Fratta, G. Nassa
          </td>
          <td>2025-09-02</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cellular senescence is accompanied by extensive genomic reorganization, such as senescence-associated heterochromatin foci and expanded interchromatin compartments, to ultimately affect gene expression. Here, we demonstrate that chromatin structural changes in senescent cells drive significant alterations in the phase behavior and motility of paraspeckles, a type of interchromatin compartment condensate. We observe increased numbers, size, and elongation of paraspeckles harboring NONO and NEAT1_2, driven by elevated levels of those components, consistent with the micellization model of longitudinal growth rather than condensate coalescence. Enhanced paraspeckle motility is associated with HP1α-mediated heterochromatin condensation and interchromatin expansion found in cellular senescence. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03757-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c589595d2d429fd53faabd150c9bf96e091d470b" target='_blank'>
              Imbalanced chromatin distribution in cellular senescence specifies paraspeckle dynamics
              </a>
            </td>
          <td>
            Joonwoo Lee, Jinmi Choi, Jeongeun Park, Seonduk Lee, Eun-Jung Cho, Youngdae Gwon
          </td>
          <td>2025-09-02</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The circular, double-stranded DNA genomes of Human papillomaviruses (HPV) exist in a nucleosomal state throughout the infectious cycle and rely on host histone epigenetic modifications and chromatin assembly processes to promote various phases of the viral life cycle. Here, we show that the histone H3.3 chaperone HIRA and its associated complex members are recruited to HPV replication factories during the late phase of the HPV life cycle. HIRA is also recruited to HPV replication factories generated by amplification of a replicon with a minimal origin and expression of the viral replication proteins E1 and E2, demonstrating that the E1 and E2 proteins are sufficient for HIRA recruitment. Downregulation of HIRA expression reduces HPV31 DNA amplification and viral transcription in differentiated keratinocytes. Histone H3.3 that is highly phosphorylated on serine residue 31 is also enriched at sites of HPV replication and this modification links the DNA damage response to chromatin that supports rapid gene activation. We propose that deposition of histone H3.3 generates viral minichromosomes that are highly primed to support the late stages of the HPV life cycle.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a5b34a817df65cd52a7a01714509e40fb97f7ea" target='_blank'>
              Chromatin assembly by the histone chaperone HIRA facilitates Human Papillomavirus replication
              </a>
            </td>
          <td>
            Ashley N. Della Fera, Dan Chen, Alison A. McBride
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Mutations in numerous genes contribute to human cancer, with different oncogenic lesions prevalent in different cancer types. However, the malignant phenotype is simple, characterized by unrestricted cell growth, invasion, and often metastasis. One possible hypothesis explaining this dichotomy is that cancer genes regulate common targets, which then function as master regulators of essential cancer phenotypes. To identify mechanisms that drive the most fundamental feature shared by all tumors-unrestricted cell proliferation-we used a multiomic approach, which identified translation and ribosome biogenesis as common targets of major oncogenic pathways across cancer types. Proteomic analysis of tumors and functional studies of cell cultures established nucleolar and coiled-body phosphoprotein 1 as a key node, whose convergent regulation, both transcriptionally and posttranslationally, is critical for tumor cell proliferation. Our results indicate that lineage-specific oncogenic pathways regulate the same set of targets for growth control, revealing key downstream nodes that could be targeted for therapy or chemoprevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e15820af1f4a4174736d844aaa4b18b01cd6fd" target='_blank'>
              Diverse oncogenes use common mechanisms to drive growth of major forms of human cancer.
              </a>
            </td>
          <td>
            O. Kauko, Mikko Turunen, Päivi Pihlajamaa, Antti Häkkinen, R. Queiroz, M. Pääkkönen, S. Ventelä, Massimilliano Gaetani, Susanna L. Lundström, Antonio Murgia, Biswajyoti Sahu, J. Routila, Gong-Hong Wei, H. Irjala, J.L. Griffin, Kathryn S. Lilley, Teemu Kivioja, S. Hautaniemi, J. Taipale
          </td>
          <td>2025-08-22</td>
          <td>Science advances</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="ABSTRACT Herpes simplex virus type 1 (HSV-1) is a double-stranded DNA virus that infects most of the human population. We previously found that cellular topoisomerase I (TOP1) associates with the HSV-1 genome throughout infection. TOP1 relieves topological stress on DNA to enable and regulate transcription, DNA replication, and DNA repair. We hypothesized that TOP1 contributes to these key viral processes because HSV-1 does not encode a topoisomerase. We found that TOP1 inhibition with camptothecin (CPT) or β-Lapachone results in a significant reduction in viral yield. The effects of CPT are reversible and occur in a dose-dependent manner, independent of strain and cell type. CPT treatment results in a decrease in viral gene expression and DNA replication, with replication defects observed at all concentrations tested. Defects in viral gene expression are not restricted to a specific gene class. However, the timing and extent of the observed defects are dose dependent. We also demonstrate that if CPT is added to cells after the onset of viral DNA replication, subsequent rounds of replication are inhibited, indicating that the replication defect observed is not simply the result of prior steps in the infectious cycle being blocked. Together, these results demonstrate that CPT is a potent inhibitor of HSV-1 infection, supporting the hypothesis that TOP1 is active on HSV-1 genomes during infection. IMPORTANCE HSV-1 is a common human pathogen. It utilizes both viral and cellular factors to facilitate infection. We previously characterized the proteins that associate with HSV-1 DNA throughout infection. One of these proteins, which has been largely unexplored in the context of HSV-1 infection, is TOP1. TOP1 functions to relieve topological stress to regulate transcription, DNA replication, and other processes that involve DNA cleavage and unwinding. Our findings support a role for TOP1 in HSV-1 DNA replication and gene expression and highlight the potential to target TOP1 activity or interactions for antiviral therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cec9b98a8bb668803e7c5faf9951486f2e65bf8c" target='_blank'>
              Camptothecin, a topoisomerase I inhibitor, impedes productive herpes simplex virus type 1 infection
              </a>
            </td>
          <td>
            Joseph R Heath, Alison E. Lloyd, Raegen M. Kulinski, Daniel P Fromuth, Jill A. Dembowski
          </td>
          <td>2025-08-28</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is one of the most common cancers among malignant tumors, and extensive evidence suggests that the prognosis of various human cancers is associated with DNA methylation. Therefore, we analyzed potential GBM prognostic biomarkers by combining GBM methylation sequencing data and transcriptome sequencing data. Biostatistical analysis of the expression of three RNA types (long non-coding RNA, microRNA, and mRNA) from The Cancer Genome Atlas (TCGA) was performed to determine the correlation between NACAD and GBM prognosis. Additionally, we predicted and validated a competitive endogenous RNA (ceRNA) regulatory network associated with epigenetic DNA methylation modifications. The ceRNA regulatory network (DCTN1-AS1–hsa-let-7a-5p–NACAD) was found to be related to overall survival (OS) and disease-specific survival (DSS) of GBM. Furthermore, we identified NACAD as an independent prognostic factor for both GBM OS and DSS. In summary, comprehensive analysis of methylation and transcriptome data led to the identification of genes and pathways in different GBM subtypes, highlighting prognostic genes and ceRNA regulatory networks, and proposing potential candidates. Supplementary Information The online version contains supplementary material available at 10.1186/s40001-025-03020-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d242277602456fb7dfcd53ca176ff548d65ded" target='_blank'>
              Epigenetic and transcriptomic analysis revealed ceRNA regulatory network in glioblastoma multiforme
              </a>
            </td>
          <td>
            Zhenghui Li, Yanqing Li, Jenna Z Liang, Tianxiang Geng, Jing Chen, Yangyang Yuan, Yufang Sui, Wanhong Zhang, Qian Chen, Guowei Tan, Qiao Shan
          </td>
          <td>2025-10-09</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Our research investigates the spatiotemporal organization of nuclear architecture via biomolecular condensation, focusing on the role of liquid–liquid phase separation (LLPS) in gene regulation during cancer progression. We identify phosphatidylinositol 4,5 bisphosphate (PIP2) as a key nuclear lipid forming membrane less condensates with RNA, affecting transcriptional regulators and super-enhancer assembly. PIP2 shows RNA- dependent nuclear localization and interacts with RNA polymerases I and II, suggesting a structural role in transcriptionally active chromatin. Mass spectrometry revealed the RNA-dependent PIP2-associated (RDPA) nuclear proteome, enriched for intrinsically disordered regions and polybasic motifs, supporting LLPS. Among RDPA proteins, BRD4 undergoes PIP2- and RNA-dependent condensation. Using a system for inducible nuclear PIP2 depletion, we demonstrate its role in organizing transcriptional condensates and modulating super-enhancer hubs. These findings establish PIP2-driven phase separation as a key mechanism in nuclear domain organization and cancer-related gene regulation. This project is supported by the project National Institute for Cancer Research (Programme EXCELES, ID Project No. LX22NPO5102) by the European Union – Next Generation EU.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ebf08b77918307ebbd285055dc8c3a13dd93180" target='_blank'>
              SPATIO-TEMPORAL ORGANIZATION OF THE SUPER-ENHANCER STRUCTURE IN CANCER CELLS
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Transposable elements (TEs) represent a threat to genome integrity due to their proliferative capacity. Eukaryotic cells silence TEs through different molecular mechanisms, including the deposition of repressive histone marks. Previous studies have shown that TE repressive marks can spread to neighboring sequences. However, evidence for this spreading leading to nearby gene silencing remains limited. Similarly, whether TEs induce changes in the enrichment of active histone marks genome-wide, and the potential impact on gene expression have not been widely studied. Results In this work, we perform a comprehensive study of the epigenetic effects of 2235 TEs and their potential effects on nearby gene expression on Drosophila melanogaster head, gut, and ovary. While most TEs (816) induce the enrichment of the H3K9me3 repressive mark, with stronger epigenetic effects in the ovary, a substantial number (345 TEs) induce the enrichment of the H3K27ac active mark, particularly in the gut. We find that 70% of the H3K9me3 enriched TEs associated with expression changes downregulate the nearby gene, and 50% of the H3K27ac enriched TEs associated with expression changes lead to gene upregulation. These changes in expression affect specific regulatory networks only in the head. Furthermore, TE epigenetic effects on gene expression are genomic context dependent. Finally, we find that 221 TEs also affect gene expression by disrupting regions enriched for histone marks. Conclusions Overall, our results show that TEs contribute to the generation of regulatory novelty through epigenetic changes, with these epigenetic effects not restricted to gene silencing and being genomic context dependent. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03705-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75c9f4509d1e24d95361876bc4f99d2975a0de9c" target='_blank'>
              The epigenetics effects of transposable elements are genomic context dependent and not restricted to gene silencing in Drosophila
              </a>
            </td>
          <td>
            Marta Coronado-Zamora, Josefa González
          </td>
          <td>2025-08-18</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1110fcfacea74bd5588618a3f029d0c49019fbe" target='_blank'>
              Mechanisms and genomic implications of break-induced replication.
              </a>
            </td>
          <td>
            Adel Atari, Haoyang Jiang, Roger A. Greenberg
          </td>
          <td>2025-08-22</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background KAT6A-CBP (K/C) and KAT6A-P300 (K/P) fusions are recurrent genetic alterations in acute myeloid leukemia (AML) associated with poor prognosis. Despite their strong oncogenic potential, the mechanisms underlying their genomic targeting and leukemogenic function remain unclear. A major challenge has been their large size, which has impeded preclinical model development and mechanistic studies. Results We employ a domain-focused truncation strategy to generate de novo murine models of K/C and K/P fusions, which faithfully recapitulate the morphological, immunophenotypic, and transcriptomic features of KAT6A-rearranged AML. Genomic profiling reveals that KAT6A fusions preferentially localize to H3K4me2/3-marked regions, while biochemical analyses uncover that KAT6A interacts with the Nucleosome Remodeling Factor (NURF), a key H3K4me2/3 reader. Disrupting NURF-chromatin interactions via depletion or small-molecule inhibition of its subunit, Bromodomain PHD Finger Transcription Factor (BPTF), impairs K/C recruitment and disrupts MLL/COMPASS-mediated H3K4me2 deposition, defining a functional epigenetic module involving KAT6A chimeras, NURF, and MLL/COMPASS. Notably, CBP/P300 inhibition reduces histone acetylation and chromatin accessibility, further impairing the recruitment of this epigenetic module. Targeting this module via NURF or CBP/P300 inhibition demonstrates efficacy in K/C leukemia models, with enhanced therapeutic effects observed when combined. Conclusions Our study identifies a self-reinforcing epigenetic module of histone modifiers and readers in KAT6A-rearranged AML, providing mechanistic insights into the genomic targeting of KAT6A chimeras and highlighting promising combinatorial therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03743-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4b1939d769bfdab61413c545354658eef418fa8" target='_blank'>
              KAT6A chimeras form a self-reinforcing epigenetic module with NURF and MLL/COMPASS to sustain AML
              </a>
            </td>
          <td>
            Junhui Lv, Zhinang Yin, Conghui Li, Honglin Wen, Jian Ni, Peiyuan Yang, Zemin Song, Ying Xiang, Honghong Wang, Rui Lu, Li Huang, Ying Zhou, Hai-Bing Zhou, Ruijing Xiao, Pingping Fang, Kaiwei Liang
          </td>
          <td>2025-08-19</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Invasive cervical cancer (ICC) is the leading cause of cancer-related deaths among women in low-income regions, with approximately 70% of cases caused by high-risk human papillomavirus (HPV) strains—primarily HPV 16 and HPV 18. HPV infection triggers the production of viral oncoproteins E6 and E7, which interfere with the tumor suppressor proteins p53 and pRb, respectively, leading to disrupted cell cycle control and uncontrolled cellular proliferation. Moreover, only a few targeted therapies are approved for treatment.



 To address this gap, we conducted exome sequencing on 400 ICC fresh frozen samples from Guatemala, the country with one of the highest incidences of cervical cancer globally. We also performed long-read whole genome sequencing (WGS) on 29 cervical cancer cell lines with a coverage depth ranging from 50 to 100x. Additionally, ICC data from public database like TCGA was included in the analysis.



 Our lab previously identified that YAP1 oncogene amplification (11%) is associated with earlier diagnosis (12 years earlier on average), poorer survival, and a novel aggressive cervical cancer subtype, more prevalent in minority populations. Treatment with the PIK3CA inhibitor Alpelisib reduced proliferation in YAP1 amplified cervical cancer cell lines, like CaSki, SNU1000, SNU1299, QG-U, C4I and CERV-215 but had no effect on PIK3CA wildtype cells. We also observed that STK11, a tumor suppressor gene, is frequently altered in ICC. Exome sequencing data from Guatemala revealed that 32% of tumors (25% in adenocarcinoma (ASC) and 7% in squamous cell carcinoma (SCC)) have STK11 alterations, consistent with our WGS data showing alterations in 35% of cervical cancer cell lines. These mutations are associated with a 10-year earlier age of diagnosis in minority populations. Furthermore, we noticed that STK11 and YAP1 are both functionally involved in the same Hippo signaling pathway. Interestingly, we found in TCGA that 24% of STK11 wild-type cases have YAP1 amplifications while only 8% of STK11 altered cases have YAP1 amplification (p=0.009), exhibiting a co-occurrence of STK11 alterations with YAP1 amplification.



 We propose a model that HPV E7 oncoprotein and STK11 alterations activate YAP1, promoting oncogenesis. Therefore, co-occurrence of STK11 alterations and HPV infection may synergize to promote more aggressive cancers through hyper-activation of YAP1. Presently, we are validating our observations in highly expressed STK11 cervical cancer cell lines (direct RNA sequencing data) through siRNA transfection and protein expression. This suggests that evaluating YAP1 expression in STK11-deleted cell lines could uncover additional opportunities for targeted therapies and screening invasive cervical cancer cases for STK11 and YAP1 gene alterations may help identify women who are more likely to respond favorably to targeted therapies.



 Sonam Tulsyan, Emma Robinson, Hong Lou, Wen Luo, Yi Xie, Qian Yang, Isabele Rodriguez, Eduardo Gharzouzi, Enrique Alvirez, Michael Dean. Long-read whole genome sequencing identifies STK11 and YAP1 genes as novel biomarkers for treating invasive cervical cancer [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr A085.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f27666748c2b6097853c42aef3cfa545e01c9e3" target='_blank'>
              Abstract A085: Long-read whole genome sequencing identifies STK11 and YAP1 genes as novel biomarkers for treating invasive cervical cancer
              </a>
            </td>
          <td>
            Sonam Tulsyan, Emma Robinson, Hong Lou, Wen Luo, Yi Xie, Qian Yang, Isabel Rodríguez, E. Gharzouzi, E. Alvirez, Michael Dean
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumor genomic profiling has uncovered many cancer drivers whose implications in terms of tumor biology and therapeutic actionability remain understudied. Hotspot mutations in SF3B1 induce widespread transcriptomic alterations and occur across multiple cancer types. Despite this, the biological and clinical consequences of SF3B1 mutations remain elusive. Characterization of the largest SF3B1 mutant breast cancer clinical cohort to date identifies SF3B1 driver mutations in approximately 2.5% of HR+ HER2- breast cancer, with strong enrichment of K700E substitutions, substantial variation in variant allele fraction (VAF), and significantly improved overall survival due to enrichment in Luminal A disease. In vitro and in vivo studies in representative cell models suggest several of the most prevalent SF3B1 mutations have deleterious effects on cell growth, leading to selective loss of the mutation over time, providing a rationale for the low frequency and low VAF of SF3B1 mutations in breast cancer. Though all introduced hotspot mutations limit growth and are reverted to wildtype (WT) over time, mutations at position R625 have a more pronounced phenotype than K700E, providing an explanation for the clinically observed skew in mutation frequencies. RNA and DNA sequencing analyses were used to identify characteristic pathway-level transcriptomic changes in SF3B1-mutant cells and identify copy number alterations as a mechanism of both improved mutation tolerability and a means to eliminate the mutation over time. These data suggest that while SF3B1 mutations are enriched in some clinical contexts, their role in breast tumorigenesis is highly complex and dependent on secondary events that overcome their deleterious effects on cell growth and survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf1adb51d253bba215fc3013a9fc8fb3e7cd5408" target='_blank'>
              Distinct and convergent effects of SF3B1 mutations in human breast cancer.
              </a>
            </td>
          <td>
            Riley E. Bergman, Christin I Keeton, Irika R. Sinha, Jonathan P. Ling, Yu Wang, Lauren Ha, Eshani P Mehta, S. C. Reed, Ethan K Fox, Tina M. O'Grady, Yingying Yu, Stamatina Fragkogianni, Calvin Chao, Brad A. Davidson, Adam X. Miranda, S. Croessmann, Shilin Zhao, Qi Liu, Yu Shyr, W. Dalton, Paula J Hurley, B. Park
          </td>
          <td>2025-10-07</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="O6-Methylguanine-DNA methyltransferase (MGMT) acts as a genomic custodian, reversing alkylation damage to preserve DNA integrity. However, when its regulatory balance tips via promoter methylation, polymorphisms, or epigenetic silencing, MGMT can become a liability, fuelling cancer progression, treatment resistance, and poor outcomes across malignancies. This review uncovers the nuanced control of MGMT, revealing how its genetic and epigenetic shifts shape tumor behavior, therapeutic response, and risk stratification. We aim to transform molecular insights into actionable clinical strategies, reimagining MGMT as both a biomarker and therapeutic lever. We curated high-impact studies (up to 2025) from PubMed, Scopus, and Web of Science, focusing on MGMT modulation, synthetic lethality, CRISPR-based restoration, and epigenetic therapies. Emerging multi-omics and translational frameworks were prioritized. MGMT’s activity is choreographed by an intricate interplay of promoter methylation, histone marks, transcriptional regulation, and microRNA influence. These dynamics critically affect sensitivity to alkylating agents like temozolomide. Intriguingly, MGMT also engages with the immune landscape modulating response to immunotherapies. Innovations in multi-omics, single-cell analytics, and AI-based biomarker profiling are unveiling previously hidden regulatory layers. Decoding MGMT’s regulation unlocks new therapeutic frontiers. Cutting-edge strategies from CRISPR to liquid biopsy promise more personalized, resistance-proof cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/137072ec7bdae0ee7995a83d59448e3923cf6fdc" target='_blank'>
              Genetic and epigenetic landscape of O6-methylguanine-DNA methyltransferase (MGMT): implications for DNA repair and cancer therapeutics
              </a>
            </td>
          <td>
            Shishir Singh, Rajeev Nema, M. Banerjee, Atar Singh Kushwah
          </td>
          <td>2025-08-28</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The genome is compacted in the nucleus through a hierarchical chromatin organization, ranging from chromosome territories to compartments, topologically associating domains (TADs), and individual nucleosomes. Nucleosome remodeling complexes hydrolyze ATP to translocate DNA and thereby mobilize histone proteins. While nucleosome remodeling complexes have been extensively studied for their roles in regulating nucleosome positioning and accessibility, their contributions to higher-order chromatin architecture remain less well understood. Here, we investigate the roles of two key nucleosome remodelers, esBAF and INO80C, in shaping 3D genome organization in mouse embryonic stem cells. Using Hi-C, we find that loss of either remodeler has minimal effects on global compartment or TAD structures. In contrast, subcompartment organization is notably altered, suggesting that esBAF and INO80C contribute to finer-scale chromatin topology. To overcome the limited resolution of Hi-C for detecting regulatory loops, we employed promoter capture Micro-C (PCMC), which revealed that the loss of esBAF or INO80C alters a subset of promoter anchored looping interactions. Although these changes occur at distinct genomic loci for each remodeler, the affected sites are commonly enriched for bivalent chromatin regions bound by OCT4, SOX2, and NANOG (OSN), as well as BRG1 and INO80 themselves. Together, our findings reveal that esBAF and INO80C selectively influence subcompartment identity and enhancer–promoter communication at key regulatory loci, highlighting a previously underappreciated role for nucleosome remodelers in higher-order chromatin organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e95264023995f85a35edad63b59e9eef19fae0c" target='_blank'>
              esBAF and INO80C fine-tune subcompartments and differentially regulate enhancer-promoter interactions
              </a>
            </td>
          <td>
            Braulio Bonilla, Benjamin J. Patty, Snehal Sambare, Job Dekker, T. Fazzio, Sarah J. Hainer
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Cancer cells arise from the cumulative acquisition of genetic and epigenetic alterations that affect vital cellular functions. Genomic instability results from deficiencies in protective mechanisms, such as cell cycle checkpoints, DNA replication control, or DNA repair. Claspin integrates a group of crucial proteins that maintain genome integrity. It participates in key cellular events such as DNA damage checkpoint activation, DNA replication, replication stress responses, DNA repair, epigenetic memory, and apoptosis. Given its crucial functions, a role for Claspin in cancer is not a surprise. Indeed, there is a considerable body of evidence linking Claspin deregulation with cancer. For instance, over-expression of Claspin and Tim promoted the survival of cancer cells by enabling adaptation to oncogene-induced replication stress. In addition, Claspin gene (CLSPN) mutations that affect checkpoint regulation have been identified in cancer patients, suggesting that they may contribute to cancer development. Changes in Claspin expression levels may be used as a prognostic marker in several types of cancer. Finally, several therapy-resistance signaling pathways seem to converge onto Claspin’s stabilization, turning Claspin into an attractive target for chemo- and radio-sensitization. In this review, we will focus on the role of Claspin in cancer and ways in which Claspin can be exploited in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86699ba9ba454ef5c75f676effbf8547c5ac5e0c" target='_blank'>
              Claspin and Cancer: Where Are We Now?
              </a>
            </td>
          <td>
            D. Azenha, Teresa C. Martins
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Lethal prostate cancer (PCa) has passed through at least two evolutionary bottlenecks: acquisition of metastatic potential and development of castration resistance. A better understanding of how this impacts genetic heterogeneity across metastatic sites is needed to develop strategies to block metastatic spread and overcome resistance. By leveraging deep whole-exome sequencing of 93 tumors from 26 patients, we examined patterns of metastatic dissemination and clonal evolution of PCa. Phylogenetic reconstruction and mathematical modeling enabled quantification of the number of mutations and clones in the cancer ecosystem and characterization of each patient's disease as an evolutionary process. While mutations of the earliest pathogenic genetic drivers of PCa were truncal, most other likely passenger mutations, copy-number alterations, and clones arose after the cancer had spread and were confined to individual metastatic sites due to polyclonal and polyphyletic seeding. Single-tissue sequencing tended to overestimate mutation clonality and, apart from truncal drivers, underestimate mutation rates for both individual patients and cohorts. This study highlights the independent evolution of metastatic lesions, which has implications for diagnostic and targeted therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f198ff2332798cb036a19b4692ee525e77c812c" target='_blank'>
              Divergent Clonal Evolution and Early Dissemination Promote Genetic Heterogeneity of Metastases in Castration Resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Noshad Hosseini, R. Mannan, Ryan J. Rebernick, Fengyun Su, Rui Wang, Xuhong Cao, Ana Lako, Dattatreya Mellacheruvu, Jing Hu, J. Alumkal, Zachery R. Reichert, Rohit Malik, Rohit Mehra, A. Chinnaiyan, M. Cieslik
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="Neural tube defects (NTDs) are one of the most common congenital malformations. Folic acid deficiency in pregnant women increases the risk of developing NTDs; however, the underlying etiology and mechanisms remain elusive. In this study, the role of DNA double-strand breaks (DSBs) in 3D genome organization in NTDs with folate deficiency is reported. The NTD mouse model is burdened with abundant DSBs associated with the disruption of 3D genome organization. DSBs occurring in active genes lead to the stalling of RNA polymerase II (Pol II) and formation of R-loops in the 3D genome. The DSB ratios of the genomic regions negatively correlated with the distance from the transcription start sites of the gene. The DSB ratios of the proximal and distal enhancers are significantly higher and induce the displacement of loops with busy anchors. Furthermore, DSB-associated dysregulation of chromatin loops occurs in neural tube closure-associated genes that are abnormally expressed in human NTDs. Taken together, excessive DSB-associated 3D genome organization disruption within NTDs with folate deficiency contributes to the dysregulation of neural tube closure-associated genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8659be399a2187d7c9eb42445c50713cbee2a3dc" target='_blank'>
              Excessive DNA Double-Strand Breaks-Associated 3D Genome Reorganization Contributes to Neural Tube Defects with Folate Deficiency.
              </a>
            </td>
          <td>
            Ting Zhang, Lin Lin, Jianting Li, Caihua Li, Shengjun Liang, Xuemei Bai, Fang Wang, Yihua Bao, Dan Guo, Xiaochen Bo, Hao Li, Hebing Chen, Qiu Xie
          </td>
          <td>2025-09-18</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Homology-directed DNA repair (HDR) is critical for genome stability and tumor suppression. HDR is initiated by the RAD51 single-stranded (ss)DNA nucleoprotein filament which conducts the homology search and invades a homologous DNA template, creating a displacement-loop (D-loop). The RAD51 filament is assisted in these processes by several proteins. One such protein is RAD51-Associated-Protein 1 (RAD51AP1) which binds DNA and RNA and directly interacts with RAD51. Of note, RAD51AP1 overexpression is associated with poor prognosis in several different cancer types. Here, we show that RAD51AP1 activity is regulated by phosphorylation. RAD51AP1 bearing S277/282A mutations is more proficient in the stimulation of D-loop formation than wild type RAD51AP1 or phosphomimetic RAD51AP1-S277/282D. In EMSAs, RAD51AP1 with S277/282A mutations more avidly binds ssDNA, double-stranded (ds)DNA, and the nucleosome core particle than wild type RAD51AP1 or RAD51AP1-S277/282D. In cells, RAD51AP1-S277/282A confers no rescue of RAD51AP1 deficiency in toxicity tests and DNA replication assays. In contrast, RAD51AP1-S277/282D fully rescues RAD51AP1 deficiency. We provide evidence that RAD51AP1-S277 is a CDK2 target and propose a model in which RAD51AP1-S277/282 phosphorylation ensures RAD51AP1 flexibility for dynamic engagement in consecutive steps of the HDR reaction. Our results provide new mechanistic insights into RAD51AP1 regulation by a CDK.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee7032aca65f6e9576eb0613414324a154e6a31c" target='_blank'>
              Phosphoregulation of RAD51AP1 function in homology-directed repair
              </a>
            </td>
          <td>
            Neelam Sharma, Mollie E. Uhrig, Youngho Kwon, P. Sung, C. Wiese
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9490feaeec24c63eafa0bf22c93391f3883ea64" target='_blank'>
              [Exploration of ribosomal DNA copy number as an early biomarker for exposure to chemical carcinogens].
              </a>
            </td>
          <td>
            C. Ding, H. D. Xu, J. L. Lou
          </td>
          <td>2025-08-20</td>
          <td>Zhonghua lao dong wei sheng zhi ye bing za zhi = Zhonghua laodong weisheng zhiyebing zazhi = Chinese journal of industrial hygiene and occupational diseases</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="B-cell acute lymphoblastic leukemia (B-ALL) remains a major clinical challenge in hematologic oncology, characterized by a continuous evolution of molecular drivers that shape its heterogeneity across the age spectrum. Pediatric B-ALL is generally associated with high cure rates, while adult forms of the disease are often more aggressive and less responsive to treatment. This review examines the age-specific genetic and epigenetic landscapes that contribute to this disparity, revealing how the nature and timing of molecular alterations point to fundamentally different leukemogenic processes. Favorable genetic aberrations, such as ETV6::RUNX1 and hyperdiploidy, are predominant in children, whereas adults more frequently present with high-risk features, including BCR::ABL1 fusions and IKZF1 deletions. Epigenetic distinctions are similarly age-dependent, involving divergent patterns of DNA methylation, histone modifications, and non-coding RNA expression. For example, pediatric B-ALL frequently harbors mutations in epigenetic regulators like SETD2 and CREBBP, while adult B-ALL is more commonly affected by alterations in TET2 and IDH1/2. These molecular differences are not only prognostic but also mechanistic, reflecting distinct developmental trajectories and vulnerabilities. Understanding these age-driven transitions is essential for improving risk stratification and developing precision therapies tailored to the unique biology of B-ALL across the lifespan.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb81ee6e89e6609c04144954df008c19fbd5a755" target='_blank'>
              Age-Driven Genetic and Epigenetic Heterogeneity in B-ALL
              </a>
            </td>
          <td>
            Yoana Veselinova, M. Esteller, Gerardo Ferrer
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Caliban, the Drosophila ortholog of human Nuclear export mediator factor (NEMF), is a recently identified regulator of the intrinsic apoptotic signaling pathway in response to DNA damage; however, the mechanism governing its expression after DNA damage remains unclear. In this study, we demonstrated that DNA damage upregulated caliban expression concomitant with p53 activation. Over-expression of p53 upregulated the mRNA and protein levels of caliban. We characterized the core region of the caliban promoter, which exhibited enhanced activity following DNA damage or p53 activation. Further analysis of the caliban promoter revealed a p53-binding site that directly interacts with p53 in response to DNA damage. Moreover, mutation of this p53-binding site or knock-down of p53 expression abolished the DNA damage-induced increase in caliban promoter activity, confirming p53’s critical role in regulating caliban expression. Taken together, our findings indicate that caliban is a direct transcriptional target of p53 in response to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdfadba5f968970331f0d881d129c4d34407e435" target='_blank'>
              Caliban is a transcriptional target of p53 in response to DNA damage
              </a>
            </td>
          <td>
            Jiaqian Cui, Haiyan Zhang, Yan Cheng, Xiaolin Bi, Dong Li
          </td>
          <td>2025-08-28</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Topoisomerase 1 (Top1) removes transcription-related helical torsions and thus plays an important role in preventing genome instability instigated by the formation of non-canonical DNA secondary structures. The genetically tractable Saccharomyces cerevisiae model proved effective in defining the critical function of Top1 to prevent recombination and chromosomal rearrangement at G4-forming genomic loci and studying the human cancer-associated Top1 mutants through the expression of analogous yeast mutants. We previously showed that cleavage-defective Top1 mutants strongly elevate the rate of recombination at G4 DNA, which involves binding to G4 DNA and interaction with the protein nucleolin (Nsr1 in yeast). Here, we further explored the mechanism of genome instability induced by the yeast Top1Y740* mutant, analogous to the human Top1W765Stop mutant conferring resistance to CPT. We show that yTop1Y740* elevates duplications as well as recombination specifically at G4-forming sequences. Interestingly, SUMOylation of yTop1Y740*, which does not affect the G4 DNA-binding or Nsr1-interaction by this mutant, is necessary for such elevated G4-specific genome instability. Many tumors with mutations at the C-terminal residues of Top1, particularly W765, have significantly high G4-associated mutations, underscoring the importance of further investigation into how SUMOylation affects the function of these Top1 mutants at G4 DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/414ccc1f67726dc580ae95a50a7d8e620a487775" target='_blank'>
              Mutations and Recombination at G4 DNA-Forming Sequences Exacerbated by CPT-Resistant Mutant Topoisomerase 1 Is Dependent on SUMOylation
              </a>
            </td>
          <td>
            Shivani Singh, Xinji Zhu, Nayun Kim
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The Mre11 nuclease is part of the highly conserved MRX complex involved in the repair of DNA double-strand breaks (DSBs). During meiosis in budding yeast, MRX is also required for the programmed induction of DSBs by Spo11, thereby initiating homologous recombination to promote accurate chromosome segregation. Recruitment of Mre11 to meiotic DSB sites depends on Rec114-Mei4 and Mer2 (RMM), which are thought to organize the meiotic DSB machinery by a mechanism involving biomolecular condensation. Here, we explored the role of Mre11 during meiosis and its relationship to RMM condensation. We show that both Mre11 and MRX complexes form DNA-dependent, hexanediol sensitive condensates in vitro. In vivo, Mre11 assembles into DNA damage-dependent foci in vegetative cells and DSB-independent foci in meiotic cells. In vitro condensates and in vivo foci both depend on the C-terminal intrinsically-disordered region (IDR) of Mre11. Importantly, while the Mre11 IDR is dispensable for vegetative DNA repair it is essential during meiosis. The C-terminal region of Mre11 forms a short α-helix that binds a conserved region of Mer2, and mutating residues within this interface reduces Mre11 foci and DSB formation. Finally, we identified a SUMO-interacting motif within the Mre11 IDR that enhances recruitment of Mre11 during meiosis and facilitates DSB formation. Our results provide new insights into the biophysical properties of Mre11 and its role in initiating meiotic recombination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aefc58e6015e9767196fad74cb8bd730f8f99bd9" target='_blank'>
              Recruitment of Mre11 to recombination sites during meiosis
              </a>
            </td>
          <td>
            Corentin Claeys Bouuaert, Priyanka Priyadarshini, Mahesh Survi, Wael El Yazidi Mouloud, Regina Bohn, Steven Ballet, Neil Hunter, Alexander Volkov
          </td>
          <td>2025-08-19</td>
          <td>Research Square</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Multicellular life depends on the ability to activate and repress genes in a highly context-specific manner. With each cell state transition, a new transcriptional profile is established. As non-coding DNA elements, enhancers mediate their regulatory potential through the effectors they recruit. While ultimately instructed by the underlying DNA sequence, enhancer activity depends on several factors, such as transcription factor availability, chromatin state, and promoter proximity, all of which are dynamically regulated within the cell. Even when we understand the regulation of one enhancer, its genomic impact is dependent on its integration within the regulatory landscape. Thus, a full picture of enhancer dynamics can only be painted through broad, but controlled, approaches that integrate investigations into multiple levels of gene regulatory mechanisms. In this review, we will present the exit of naive pluripotency as a prime setting to do just that and contextualize how its contemporary use has been, and could be, used to reveal the intricacies of enhancer mechanistics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a198ff34681e848852fded55aab3ace3539920dc" target='_blank'>
              The exit from naive pluripotency: a platform for the study of enhancer mechanistics
              </a>
            </td>
          <td>
            Mattias Enar Jonasson, C. Buecker
          </td>
          <td>2025-08-18</td>
          <td>Biochemical Society Transactions</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Extrachromosomal DNA (ecDNA) drives the evolution of cancer cells. Its widespread presence in tumors and strong association with poor clinical outcomes make ecDNA a promising and broadly applicable therapeutic target. Recent studies have begun to unravel the mechanisms by which ecDNA promotes tumorigenesis and maintains its presence in cancer cells. These discoveries have paved the way for developing ecDNA-targeted therapies. In this Perspective, we summarize the latest advances in our understanding of the mechanism underlying both the ecDNA-induced cancer phenotype and ecDNA maintenance. We also explore potential strategies for targeting ecDNA in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55785ec96f370b7285e3fa5a23ac44e0552ecce9" target='_blank'>
              Unfolding ecDNA as a pan-cancer therapeutic target
              </a>
            </td>
          <td>
            Yang Zhang, Ruirui Chen, Haiyun Gan
          </td>
          <td>2025-10-07</td>
          <td>Medical Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Human endogenous retroviruses (HERVs), particularly the HERV-K family, are increasingly recognized for their roles in cancer biology, yet the function of ERVK3-1 (HERVK_3p21.31) in liver hepatocellular carcinoma (LIHC) remains largely unexplored. Methods We analyzed transcriptomic data from the TCGA-LIHC cohort to identify differentially expressed genes (DEGs) between high and low ERVK3-1 expression groups, followed by functional enrichment analyses (GO, KEGG, GSEA), protein-protein interaction (PPI) network construction, and hub gene identification. The immunological relevance of ERVK3-1 was assessed through TIP immune cycle analysis, single-cell RNA sequencing datasets, and correlation with immune checkpoint expression. Immunotherapy responsiveness was evaluated using TIDE and TCIA databases. Results High ERVK3-1 expression was associated with enrichment in metabolic and oxidative stress-related pathways, while low expression correlated with cell cycle and DNA replication. PPI analysis revealed mitosis-related hub genes (e.g., CCNB1, CDK1). ERVK3-1 expression promoted early immune cell recruitment but was inversely correlated with later stages of the cancer immunity cycle, including immune infiltration and T-cell killing. Single-cell data showed high ERVK3-1 expression in immunosuppressive subsets, alongside positive associations with inhibitory immune checkpoints (e.g., PD-1, CTLA-4, TIM-3). High ERVK3-1 expression also correlated with greater immune evasion and reduced immunotherapy responsiveness. Conclusions ERVK3-1 plays a multifaceted role in LIHC progression, contributing to metabolic reprogramming, immune suppression, and resistance to immunotherapy. These findings highlight ERVK3-1 as a potential prognostic biomarker and therapeutic target in liver cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12985-025-02928-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/186dc1e93c9e9bba5bd5b29915ff6bacc720805a" target='_blank'>
              Human endogenous retrovirus ERVK3-1 characterizes a metabolically active and immunosuppressive subtype of liver cancer
              </a>
            </td>
          <td>
            Xiaofen Wen, Shulv Weng, Minna Chen, Danxia Lin, Wenwu Xue, De Zeng, Jiaxin Shen
          </td>
          <td>2025-10-01</td>
          <td>Virology Journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction: Long interspersed element-1 (L1) and Alu elements are reported to regulate tumor development and progression by forming chimeric transcripts with diverse sequences. While these elements have been implicated in various cancers, the landscape of chimeric transcript formation in breast cancer remains poorly characterized.
Objective: Therefore, in this study, we comprehensively investigated the profiles of L1 and Alu chimeric transcripts in breast tumors by analyzing RNA sequences from The Cancer Genome Atlas (TCGA) database.
Methods: More than 50 billion paired-end RNA sequencing reads from 606 primary breast tumor tissues and 59 matched adjacent normal tissues from the TCGA database were aligned to L1 and Alu consensus sequences with Bowtie2 software and further analyzed.
Results: Our analysis revealed significantly elevated expression levels of L1 and Alu retrotransposons in breast tumor tissues compared to adjacent non-tumor tissues. Importantly, higher expression of L1 and Alu in breast tumors was associated with poorer 10-year overall survival, suggesting their potential as prognostic biomarkers. Further exploration demonstrated that L1 and Alu were extensively expressed in chimeric form with adjacent transcripts. A significant difference in the frequency of 651 L1-mRNA and 1,525 Alu-mRNA chimeric transcripts was observed between non-tumor and tumor tissues. Similarly, 1,009 L1-long non-coding RNA (lncRNA) and 2,575 Alu-lncRNA chimeric transcripts exhibited significantly different frequencies between non-tumor and tumor tissues. Functional enrichment analysis revealed that these differentially expressed chimeric genes played crucial roles in multiple facets of tumorigenesis, including metabolism, signal transduction, immunity, cell cycle regulation, and apoptosis.
Conclusion: Our findings demonstrate that Chimeric transcript formation with L1 and Alu retrotransposons represents a widespread phenomenon in breast cancer that impacts multiple oncogenic pathways. These results significantly expand our understanding of retrotransposon involvement in breast cancer and highlight potential novel targets for diagnostic and therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99e34ef62da2b088b93dd7d2cad99c0994dc0a0e" target='_blank'>
              Landscape of long interspersed element-1 and Alu chimeric transcripts in breast tumors
              </a>
            </td>
          <td>
            Jian Li, Yin Zhang, Yiting Zhou, Jianyou Liao, Yabin Guo
          </td>
          <td>2025-09-03</td>
          <td>Eurasian Journal of Medicine and Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Objectives Human RECQL4 and BLM helicases participate in all DNA dependent processes, including replication stress, DNA damage repair. Both helicases are overexpressed in glioblastoma (GBM), a lethal primary brain tumour, characterised by resistance to radio- and chemotherapy. BLM-depleted glioma cells exhibit senescence-associated or polypoid phenotype when exposed to temozolomide (TMZ) and olaparib (OLA), a PARP inhibitor. This study aims to investigate how RECQL4 depletion influences the response of malignant gliomas to chemotherapeutics. Methods We investigated the effect of RECQL4 depletion in glioma cells on cell growth, apoptosis, senescence and polyploidy in the response to combined TMZ and OLA treatment. We compared transcriptomes of RECQL4- and BLM-depleted LN18 and LN229 glioma cells. Drug-induced cytotoxicity, senescence-associated phenotypes, cell cycle alterations, and polyploidy were assessed using the MTT metabolic assay, β-galactosidase activity assay, and propidium iodide staining. Results RECQL4 depletion modestly affected basal glioma cell viability and proliferation, similarly to knock out of the BLM protein. Deletion of RECQL4 in glioma cells (RQ4 KO) induced profound transcriptomic alterations, dissimilar to BLM depletion. RECQL4-depleted glioma cells treated with TMZ and OLA exhibited reduced viability and increased levels of apoptosis markers. The treatment induced cell cycle arrest, however, RQ4 KO cells did not show signs of senescence phenotype or polyploidisation, when compared to BLM KO glioma cells. Interestingly, both RQ4 KO and BLM KO cells were more resistant to WP744, a doxorubicin derivative, when compared to WT LN229 glioma cells. Conclusion Our results highlight the distinct roles of RecQ helicases in a response to chemotherapeutics and support a rationale for targeting RECQL4 as a therapeutic strategy in glioblastoma. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14932-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d631e6439cbcbbf960a1d54b75f9eb6885b3c7c" target='_blank'>
              Shared and non-overlapping functions of RECQL4 and BLM helicases in chemotherapeutics-induced glioma cell responses
              </a>
            </td>
          <td>
            K. Wojnicki, B. Wojtaś, I. Ciechomska, B. Kaza, Matthew Guille, Waldemar Priebe, Bozena Kaminska
          </td>
          <td>2025-09-29</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="


 IDH-mutant gliomas are diffusely infiltrating, slow-growing brain tumors that inevitably undergo malignant progression to grade IV astrocytoma - a highly aggressive and uniformly fatal disease with dismal prognosis. Despite extensive research, the molecular mechanisms underlying this transformation remain poorly understood. While alterations in canonical driver genes such as CDKN2A, PTEN, and PDGFRA have been implicated, the broader landscape of genetic drivers involved in IDH-mutant glioma progression is largely uncharacterized. In contrast to many other malignancies, IDH-mutant gliomas acquire relatively few somatic point mutations during progression - typically in the range of 30 to 40 - suggesting that alternative mechanisms may be driving tumor evolution. Notably, these tumors exhibit widespread chromosomal instability, resulting in a heterogeneous and highly prevalent pattern of copy number alterations (CNAs). This observation raises the possibility that gene dosage changes due to CNAs play a critical role in tumor progression.



 To investigate this hypothesis, we performed integrative analysis of IDH-mutant glioma cohorts from the Mayo Clinic and TCGA datasets (~1,500 patients), identifying twelve highly recurrent CNAs significantly associated with patient outcomes. We refer to these as malignant IDH-associated CNAs (mIDH CNAs). We hypothesized that these recurrent CNAs harbor driver genes that confer selective growth advantages, thereby contributing to their recurrence and prognostic relevance. To systematically assess the functional impact of genes within these regions, we developed a novel in vivo CRISPR-based screening platform, termed CRISPR-Knock Out and Activation Linked Assay (CRISPR-KOALA). This approach utilizes stereotaxic delivery of lentiviral gRNA libraries - targeting mouse orthologs of ~1,800 genes within recurrently deleted regions and ~1,200 genes within amplified regions - into the brains of a genetically engineered mouse model of IDH-mutant glioma (rs557mut;IDHmut;P53KO;AtrxKO;Cas9GFP: Yanchus et. al Science 2022).



 Mice injected with mIDH CNA-targeting libraries developed tumors significantly more rapidly than control cohorts. Deconvolution of the CRISPR-KOALA screen identified several putative driver genes, including regulators of the NOTCH signaling pathway, lipid metabolism modulators such as THEM6, and the NMDA receptor modulator GRINA - implicating dysregulation of metabolism, immune response, and glutamatergic signaling in the malignant transformation of IDH-mutant gliomas.



 Together, these findings provide critical insights into the genetic mechanisms driving the progression of IDH-mutant gliomas and identify new candidate genes and pathways for potential therapeutic targeting.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/174bd23b8ba2dba117306ce08fc0c554d76de475" target='_blank'>
              OS09.5.A RECURRENT COPY NUMBER ALTERATIONS DRIVE IDH-MUTANT ASTROCYTOMA DEVELOPMENT AND PROGRESSION
              </a>
            </td>
          <td>
            S. Haider, K. Drucker, R. Tsai, K. Al-Zahrani, C. Chiu, T. Kollmeyer, G. Zadeh, R. Jenkins, D. Schramek
          </td>
          <td>2025-10-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) is the most common subtype of ovarian cancer (∼80%) and is one of the most lethal gynecological malignancies in the United States. HGSOC is characterized by universal mutations in the TP53 gene and defects in DNA repair pathways, contributing to high incidence of chromosomal instability (CIN). CIN refers to an ongoing process involving the gain or loss of parts of chromosomes, leading to chromosomal mis-segregation, aneuploidy, and copy number variations (CNV). CINhigh cancers display a greater dependency on KIF18A, an essential cytoskeletal motor protein for cell division, in contrast to normal somatic cells. Therefore, KIF18A represents a promising target for therapeutic interventions in CINhigh cancers. We investigated the use of a novel KIF18A inhibitor (KIF18Ai), ATX020, for selectively targeting CINhigh HGSOC cells. Our approach involved advanced techniques such as RNA sequencing (RNA-seq), siRNA-based gene silencing, cell cycle analysis, and immunofluorescence. We utilized DEPMAP and flow cytometric analysis to categorize ovarian cancer cell lines according to their aneuploidy score (AS) and ploidy to investigate their response to ATX020. Cell growth inhibition studies demonstrated a correlation between AS/ploidy levels and sensitivity to ATX020 in a panel of HGSOC cells including PEO4, OVCAR3, OVCAR8 and A2780. Assessments of cell viability, cell cycle analysis, phospho-histone H3 expression, clonogenic assays, and invasion assays indicated that the growth inhibition observed over 72 hours was primarily due to the toxic effects of ATX020 on tumorigenicity and mitotic arrest. Mechanistically, ATX020 exerts its toxic effects by inhibiting the plus-end movement of KIF18A on spindle fibers, which increases the inter-kinetochore (Inter-KT) distance and induces adverse effects such as chromosomal mis-segregation, aneuploidy, and double-strand DNA damage. We also observed that KIF18Ai-resistant cells (A2780, PEO4) showed inhibited KIF18A plus-end movement and increased inter-KT distance but did not exhibit the accompanying toxic effects. In summary, our findings indicate that ATX020 inhibits CINhigh HGSOC cells primarily by inducing mitotic arrest, causing DNA damage, and reducing tumorigenicity. This effect is facilitated by ATX020's ability to inhibit the plus-end directed movement of KIF18A along spindle fibers, which is detrimental to CINhigh cells that rely on KIF18A for normal mitotic divisions. Further mechanistic investigation is ongoing.



 Jayakumar R. Nair, Maureen M. Lynes, Laura Ghisolfi, Serena J. Silver, Jung-Min Lee. Investigating a novel KIF18A inhibitor ATX020 in chromosomally instable ovarian cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/256c1d3be3ec4d6c1fa4908dbe7af1b6e3ef12d5" target='_blank'>
              Abstract B024: Investigating a novel KIF18A inhibitor ATX020 in chromosomally instable ovarian cancer
              </a>
            </td>
          <td>
            Jayakumar R. Nair, Maureen M. Lynes, Laura Ghisolfi, Serena J Silver, Jung-Min Lee
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background: Pancreatic ductal adenocarcinoma (PDAC) is a highly heterogeneous malignancy, characterized by low tumor cellularity, a dense stromal response, and intricate cellular and molecular interactions within the tumor microenvironment (TME). Although bulk omics technologies have enhanced our understanding of the molecular landscape of PDAC, the specific contributions of non-malignant immune and stromal components to tumor progression and therapeutic response remain poorly understood. Methods: We explored genome-wide DNA methylation and transcriptomic data from the Cancer Genome Atlas Pancreatic Adenocarcinoma cohort (TCGA-PAAD) to profile the immune composition of the TME and uncover gene co-expression networks. Bioinformatic analyses included DNA methylation profiling followed by hierarchical deconvolution, epigenetic age estimation, and a weighted gene co-expression network analysis (WGCNA). Results: The unsupervised clustering of methylation profiles identified two major tumor groups, with Group 2 (n = 98) exhibiting higher tumor purity and a greater frequency of KRAS mutations compared to Group 1 (n = 87) (p < 0.0001). The hierarchical deconvolution of DNA methylation data revealed three distinct TME subtypes, termed hypo-inflamed (immune-deserted), myeloid-enriched, and lymphoid-enriched (notably T-cell predominant). These immune clusters were further supported by co-expression modules identified via WGCNA, which were enriched in immune regulatory and signaling pathways. Conclusions: This integrative epigenomic–transcriptomic analysis offers a robust framework for stratifying PDAC patients based on the tumor immune microenvironment (TIME), providing valuable insights for biomarker discovery and the development of precision immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec3422e79e1fa464a39c9ceade9afd8d95e43c98" target='_blank'>
              Deciphering the Heterogeneity of Pancreatic Cancer: DNA Methylation-Based Cell Type Deconvolution Unveils Distinct Subgroups and Immune Landscapes
              </a>
            </td>
          <td>
            Barbara Mitsuyasu Barbosa, Alexandre Todorovic Fabro, Roberto da Silva Gomes, C. Rainho
          </td>
          <td>2025-09-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="In mammals, repression of germline-specific gene expression is essential for preserving somatic cell identity and preventing disease. Germline gene silencing is often dependent on the presence of promoter 5-cytosine DNA methylation (5meC). Early mammalian development, however, is marked by a dramatic loss of 5meC levels genome-wide. Recent research has highlighted a specialized variant of the Polycomb Repressive Complex 1, PRC1.6, as a key regulator that maintains transient silencing of germline genes in this embryonic window. Eventually, PRC1.6 seems to stimulate the recruitment of de novo DNA methyltransferases (DNMTs), although the precise mechanisms remain to be fully elucidated. Evidence suggests a coordinated epigenetic relay, potentially involving direct protein interactions and shaping the local histone modification landscape. This review explores PRC1.6 as a central epigenetic hub that connects Polycomb repression, histone 3 lysine 9 (H3K9) methylation, and DNA methylation pathways. Unraveling this interplay will provide key insights into the mechanisms that maintain the critical barrier between the germline and the soma, essential not only for proper development but also for preserving somatic cell function and health throughout life.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ed478b995bf1b398b0bfe5ba03fe4e97c29208" target='_blank'>
              Epigenetic relay: Polycomb-directed DNA methylation in mammalian development
              </a>
            </td>
          <td>
            Teresa Urli, Maxim V. C. Greenberg
          </td>
          <td>2025-09-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Mammalian genomes undergo pervasive transcription in both genic and intergenic regions. Trimethylation of histone H3 lysine 4 (H3K4me3) is a deeply conserved and functionally important histone modification enriched at transcriptionally active promoters, where it facilitates RNA polymerase activity. H3K4me3 is also commonly found in intergenic regions, where its role is poorly understood. We interrogated the role of H3K4me3 at intergenic regulatory elements by using epigenetic editing to efficiently deposit H3K4me3 at intergenic loci. We found that H3K4me3 amplifies RNA polymerase activity and is actively remodeled at intergenic regions, shedding light on these important but poorly understood regions of the genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b45f64bb56f63b370548214321fa4126548d4ca8" target='_blank'>
              H3K4me3 amplifies transcription at intergenic active regulatory elements.
              </a>
            </td>
          <td>
            Haoming Yu, Yongyan Zhang, Zhicong Liao, B. W. Walters, Bluma J. Lesch
          </td>
          <td>2025-08-18</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The most common cause of death in pediatric cancer patients is a treatment-resistant tumor, compounded by metastatic spread, making surgery, radiation therapy, and chemotherapy unfeasible as curative treatment options. However, the mechanisms behind metastatic spread in pediatric tumors remain largely unexplored. We conducted whole-genome copy number profiling on 171 primary tumor and metastases samples from 17 patients with neuroblastoma, Wilms tumor, or gonadal tumors, and performed targeted deep sequencing on a subset. Phylogenetic reconstruction enabled spatiotemporal tracking of subclones. In total, 11 of 17 patients displayed at least one metastasis arising earlier, defined as occurring before the most recent common ancestor in the primary tumor. In eight patients, metastatic spread was observed several times during tumor evolution, with different subclones from the same primary tumor having metastatic capability, even colonizing the same site. Strikingly, dissemination between metastases (intermetastatic spread) occurred in eight of nine patients with metastases in at least two different sites, indicating that this is a common phenomenon in pediatric malignancy. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e24f7bd63ad2744ced5ab2ba8c285ffb7719db78" target='_blank'>
              Tracing metastatic spread in pediatric solid tumors using copy number and targeted deep sequencing.
              </a>
            </td>
          <td>
            Natalie Andersson, Michele Ferro, Caroline Jansson, Subhayan Chattopadhyay, Jenny Karlsson, David Gisselsson
          </td>
          <td>2025-09-23</td>
          <td>The Journal of pathology</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT In cellular genomes, CCCTC-binding factor (CTCF) insulators impact transcription over small distances in a one-dimensional manner and over much longer distances in a three-dimensional manner by maintaining chromatin loops. We have previously shown that the latent HSV-1 genome contains CTCF insulators that function to regulate lytic transcription of adjacent genes in a one-dimensional manner. Here, we test the hypothesis that HSV-1 CTCF insulators nucleate chromatin loops to regulate the expression of distance-separated gene regions through three-dimensional organization of viral genomes. We used 4C-seq methods to identify multiple long-range cis interactions in HSV-1 genomes that generate viral chromatin domains, including those nucleated by the viral CTCF insulator CTRL2. Deletion of the CTRL2 insulator disrupted these viral chromatin domains. Loop-nucleating interactions were quantitated with a novel approach (UMI-4C-seq) that utilizes unique molecular identifiers to label and count chromatin interactions associated with specific viewpoint primers. Cis-interaction peaks across four different viewpoints were quantified. Viral genomes lacking CTRL2 displayed more cis-interaction peaks and wider ranges of interaction lengths compared to wt virus, suggesting altered chromatin organization. Furthermore, differential looping analysis showed that viral genomes lacking CTRL2 displayed a more transcriptionally permissive chromatin environment. Thus, the CTRL2 insulator functions as a critical regulator of long-range chromatin interactions, and its deletion reshapes the viral chromatin landscape, leading to a more accessible and dynamic regulatory environment that may influence HSV-1 transcriptional programs and latency-associated chromatin states. IMPORTANCE HSV-1 is a significant lifelong human pathogen that infects 70% of adults worldwide. The latent HSV-1 genome is chromatinized and maintained in distinct chromatin structures that silence the virus, while reactivation is facilitated by transient reversal of host factors that maintain those chromatin domains. Understanding how this happens is critical for the development of novel therapeutics. It is becoming clear that CTCF insulators play a key role in the reversal that leads to reactivation. CTCF insulators are essential regulators of chromatin structure and gene expression in mammalian cells and play vital regulatory roles in transcriptional control of DNA viruses by organizing chromatin architecture during both latent and lytic stages of virus lifecycles. Here, we present the first report that latent HSV-1 genomes are organized into 3D structures to support latency yet allow the viral genome to reactivate, opening the door for future therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95a535267a81e37204f0c265ca04fb430e6f4497" target='_blank'>
              Multiple long-range cis interactions generate CTCF insulator-dependent viral chromatin domains in quiescent HSV-1 genomes
              </a>
            </td>
          <td>
            Alyssa Richman, Sophie Kogut, Terri G. Edwards, Joseph R. Boyd, Princess D. Rodriguez, Michael Mariani, Mason A. Shipley, Kayley A Manuel, Ziyun A Ye, David C. Bloom, Seth Frietze, Donna M Neumann
          </td>
          <td>2025-08-28</td>
          <td>mBio</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Hepatitis B virus (HBV) integration is a key driver of hepatocellular carcinoma (HCC) occurrence and progression; however, its oncogenic mechanisms remain incompletely understood because of limitations in detection methods and sample availability. In this study, we employed Oxford Nanopore Technologies (ONT) whole-genome sequencing and full-length transcriptome sequencing to characterize HBV integration events at the genomic and transcriptomic levels, along with their regulatory effects on structural variations (SVs) and gene expression. Functional validation was performed using dual-luciferase assays and cell-based experiments. Our findings revealed that integrated HBV sequences form long concatemers, mediating inter- and intrachromosomal recombination in the human genome. Notably, integrated HBV enhancer I (HBV-Enh I) was detected in 6 of 7 tumor tissues and was associated with aberrant gene expression. HBV integration induced oncogenic SVs, such as focal MYC amplification and NAV2 deletion, and directly modulated gene expression. Additionally, ectopic overexpression of MYOCD, driven by HBV-Enh I integration, promoted HCC cell migration and invasion. In summary, HBV integration acts as a major driver of large-scale genomic SVs and transcriptomic dysregulation, through either direct alterations in genome dosage or cis-regulatory mechanisms. HBV-Enh I is frequently integrated in HCC and might play a pivotal role in abnormal gene expression, highlighting its potential as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/428a53d8eb2aca52cd5c27a1ec4e813d0e350e58" target='_blank'>
              Long-read sequencing reveals HBV integration patterns and oncogenic impact on early-onset hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Yao Wang, Dong Yu, Yue Mei, Zhida Fu, Jian Lin, Di Wu, Yuan Yang, Hongli Yan
          </td>
          <td>2025-09-16</td>
          <td>Genome research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Non-homologous end joining (NHEJ) is a DNA repair pathway that directly ligates broken DNA ends without the need for a homologous template, whereas translesion synthesis (TLS) is a DNA damage tolerance mechanism in which specialized DNA polymerases bypass lesions on the template strand. Although both pathways play critical roles in maintaining genome integrity across organisms, they inherently introduce mutations. Here, we investigate how these two pathways contribute to spontaneous and genotoxic stress–induced genomic alterations in the yeast Yarrowia lipolytica. A NHEJ-deficient mutant (ku70) and three TLS-deficient mutants (rev1, rev3, and rad30) are subjected to mutation accumulation experiments, followed by whole-genome sequencing. Our results show that the deletion of KU70 has no significant effect on the rates of spontaneous single-nucleotide variations (SNVs), small insertions and deletions, or chromosomal rearrangements, while the deletion of REV1 and REV3 leads to significant reductions in spontaneous SNV rates. These findings indicate that TLS but not the NHEJ pathway is a major contributor to spontaneous mutagenesis in Y. lipolytica. Moreover, exposure to 0.02% methyl methanesulfonate and 80 J/m2 ultraviolet (UV) radiation resulted in 48- and 107-fold increases in SNV rates, respectively. These induced SNVs were largely dependent on DNA polymerases Rev1 and ζ, further underscoring their central roles in genotoxic stress–induced mutagenesis. We observe that DNA polymerase η can suppress C to T and C to A substitutions while promoting T to C mutations, exhibiting a dual function in regulating mutagenesis under UV treatment. Phenotypic evolution experiments reveal that TLS activity enhances the adaptive potential of Y. lipolytica under oxidative stress, underlying its broader impact on environmental fitness. Together, these findings provide new insights into the distinct roles of the NHEJ and TLS pathways in preserving genome integrity in Y. lipolytica. Key points • The NHEJ pathway has a limited role in spontaneous genomic alterations in Y. lipolytica. • DNA polymerases Rev1 and ζ contribute to most UV- and MMS-induced mutations. • The dual roles of Pol η in UV-induced mutations were revealed. • NHEJ and TLS pathways are crucial to phenotypic evolution of Y. lipolytica. Supplementary Information The online version contains supplementary material available at 10.1007/s00253-025-13575-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15d93f494460655f7859b1e3dd9cb7f63f2e1036" target='_blank'>
              The roles of NHEJ and TLS pathways in genomic alterations and phenotypic evolution in the yeast Yarrowia lipolytica
              </a>
            </td>
          <td>
            Cen Yan, Ye-Ke Wang, Yuan-Ru Xiong, Xin-Qiu Zhou, Yuan-Chun Fang, Ruo-Tian Nie, Cunqi Ye, Ke Zhang, Dao-Qiong Zheng
          </td>
          <td>2025-08-15</td>
          <td>Applied Microbiology and Biotechnology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af9a242ccc6ebd9d79ab2d3fe8d1c436d8e52055" target='_blank'>
              Enabling whole-genome DNA methylation-based classification of central nervous system tumors
              </a>
            </td>
          <td>
            Jonghoon Lee, Young Seok Ju, B.-L. Oh, S. Choi, Se Hoon Kim, Ji Won Lee, J. Phi, Y. Jung
          </td>
          <td>2025-09-21</td>
          <td>None</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Background FET (FUS, EWSR1, and TAF15) fusion oncoproteins are characteristic for several sarcomas and leukemias, including myxoid liposarcoma and Ewing sarcoma. FET oncoproteins interact with the SWI/SNF chromatin remodeling complex subtypes cBAF, PBAF, and GBAF, but their impact on SWI/SNF compositions, interactions, and downstream epigenetic effects remains elusive. Methods We employ a comprehensive immunoprecipitation and quantitative mass spectrometry approach to determine the impact of FET oncoproteins on SWI/SNF composition and their interactomes. Validation of complex composition and interaction partners is performed by glycerol gradient sedimentation assays and co-immunofluorescence analysis. Furthermore, we determine the differential chromatin accessibility and gene regulation in FET sarcomas using assay for transposase-accessible chromatin sequencing and RNA sequencing, respectively. Results Our data show that FET sarcomas have distinct SWI/SNF complex compositions, with different subunit paralogs and subtype-specific components that utilize distinct sets of interaction partners, including specific transcription factors. We show that FET oncoproteins cause no major disruption of the SWI/SNF complex composition. Instead, FUS::DDIT3-bound SWI/SNF complexes in myxoid liposarcoma cells are enriched in PBAF and GBAF components as well as most interaction partners. Conclusions These data suggest that FET oncoproteins act together with fully assembled and functional SWI/SNF complexes and recruited interaction partners. Finally, our data reveal that the SWI/SNF compositions, interactomes, and epigenetic background contribute to the tumor type in FET sarcoma. Trial registration Clinical trial number: not applicable. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s11658-025-00792-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7d390ca6f97d7ab986b2803f84e41f3c7c0db63" target='_blank'>
              FET fusion oncoproteins enrich SWI/SNF complex subtypes and interaction partners
              </a>
            </td>
          <td>
            M. Lindén, Lisa Andersson, Heba Albatrok, Vilma Canfjorden, Emma Jonasson, Kajsa Grönqvist, Daniel Sjövall, Pekka Jaako, Rossella Crescitelli, Henrik Fagman, Pierre Åman, Anders Ståhlberg
          </td>
          <td>2025-09-23</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Transcription factor E3 (TFE3) oncofusions are frequently detected in the Microphthalmia transcription factor (MiT) family translocation renal cell carcinoma (tRCC), a rare pediatric renal cancer with limited treatment options. The mechanisms by which TFE3 oncofusions promote tRCC malignancy remain inadequately defined. Here, we demonstrate that the RNA-binding capability conferred by TFE3 fusion partners drives the formation of TFE3 condensates. This further enables TFE3 oncofusions to co-condensate with RNA polymerase II (RNAPII) and other RNA-binding proteins, such as paraspeckle component 1 (PSPC1), ultimately driving the formation of transcriptional hubs to promote pro-oncogenic transcription. Dissolution of oncofusion condensates through nanobody-based chemogenetic manipulation effectively curtails tRCC cell growth both in vitro and in vivo, suggesting the therapeutic potential for targeting oncofusion condensation in tRCC. Collectively, our study establishes the causal role of RNA and RNA-binding proteins in facilitating oncofusion condensation to promote renal cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b79cdc6e21b779dea89a691271b52beca3b734a9" target='_blank'>
              RNA-mediated condensation of TFE3 oncofusions facilitates transcriptional hub formation to promote translocation renal cell carcinoma
              </a>
            </td>
          <td>
            Lei Guo, Rongjie Zhao, Yi-Tsang Lee, Junhua Huang, James Wengler, Logan Rivera, Tingting Hong, Tianlu Wang, Kunjal Rathod, Ashley Suris, Yitian Wu, Xiaoli Cai, Rui Wang, Yubin Zhou, Yun Huang
          </td>
          <td>2025-09-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tumor metastasis is a major contributor for mortality in patients with pancreatic ductal adenocarcinoma (PDAC). While circular RNAs (circRNAs) have emerged as pivotal regulators of tumor progression and metastasis, their functional roles in PDAC remain poorly understood. Through comprehensive circRNA profiling of 10 pairs of PDAC tumors and adjacent normal tissues, we identified circPRKD3 (hsa_circ_0000992) as being substantially up-regulated in malignant specimens. Functional characterization demonstrated that circPRKD3 overexpression potently enhanced cellular migration, invasion, and metastatic capacity in vitro and in vivo, whereas its knockdown produced opposite phenotypic effects. Mechanistic investigations revealed that circPRKD3 directly interacted with the oncogenic RNA-binding protein Musashi-2, protecting it from β-TRCP-mediated ubiquitination and subsequent proteasomal degradation. Clinical correlation analysis revealed a close association between elevated circPRKD3 expression and shorter survival of PDAC patients. Notably, we validated the translational potential of circPRKD3 as a liquid biopsy marker, showing that serum detection, when combined with conventional biomarkers (CA19-9, CEA, and CA125), dramatically improved diagnostic performance. These findings not only delineate a novel circRNA-mediated regulatory axis in PDAC metastasis but also identify circPRKD3 as a promising diagnostic and prognostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1441280c9bbaab596bfcee43907f7ebe16003d78" target='_blank'>
              A Circular RNA Promotes Tumor Metastasis through Stabilizing MSI2 Protein in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Zhang Li, Lanyang Gao, Yang Yang, Yue Ming, Wenrong Liu, Tingting Zhang, Zixia Ye, Fuyan Xu, Juan He, Jiao Li, Jiawei Guo, Xiaojuan Yang, Qing Zhu, Yong Peng
          </td>
          <td>2025-09-10</td>
          <td>Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Simple Summary BRG1 protein expression or its normal functions are lost in many cancers, including ~10% of lung cancer cases, due to mutation. However, how these losses contribute to the development of lung cancer is not clearly known. This study used human lung epithelial cell-derived normal, pre-cancer, and cancer cell lines to investigate the initiation and progression of lung cancer. Our results show that BRG1 loss induces epigenetic (non-genetic) and expression changes in thousands of genes genome-wide that lead to the early transformation of normal cells into pre-cancer cells. In established cancer cells, BRG1 loss and its downstream epigenetic and gene expression changes make the cancer cells become sensitive to epigenetic drugs. Taken together, these results indicate that BRG1 loss contributes to early lung cancer development and could be exploited to improve the treatment of this deadly disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23222db0933f5999fd32623b0319bdf7442135af" target='_blank'>
              BRG1 Loss Is Frequent in Lung Cancer and Transforms Lung Epithelial Cells via Transcriptional and Epigenetic Reprograming
              </a>
            </td>
          <td>
            M. Tessema, Christin M. Yingling, Loryn M. Phillips, K. Do, M. Picchi, R. Willink, Steven A. Belinsky
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA), shed by malignant tumor cells into extracellular fluid, provides valuable epigenetic information indicative of cancer status. Nipple aspirate fluid (NAF), a noninvasive liquid biopsy from at-risk women, contains nucleic acid and protein biomarkers from adjacent cancer cells, showing promise for breast cancer (BrC) detection. However, despite its potential, the application of cfDNA in NAF for BrC screening is still underexplored. Here, we report a proof-of-concept study for using cfDNA bisulfite sequencing (cfBS) to assess tumor DNA methylation signatures from NAF samples. For four healthy individuals and three BrC patients, cfBS achieved greater than 20× sequencing depth with an average coverage of 26.5× on the genome. A total of 7471 differentially methylated regions were identified, with significant hypermethylation in BrC samples compared to healthy controls. Gene set enrichment analysis indicated that the differentially methylated genes (DMGs) were significantly associated with epithelial–mesenchymal transition (EMT). By developing a novel EMT scoring metric, we found that BrC samples had more of a mesenchymal phenotype than samples from healthy individuals. CDH1, WNT2, and TRIM29 were hypermethylated near the promoter region, while COL5A2 was hypermethylated in the coding region. The DNA methylation and EMT changes were validated through The Cancer Genome Atlas Breast Invasive Carcinoma study, which confirmed that DMGs were associated with gene expression change and that our methylation-based EMT score reliably distinguished tumors from healthy controls. Our findings support the utilization of the NAF cfDNA cfBS methylation profile for noninvasive BrC screening and pave the way for enhanced early detection of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/910913725dce178fd3c9aae5a2e95c8cd7107e54" target='_blank'>
              Cell-Free DNA Bisulfite Sequencing Reveals Epithelial–Mesenchymal Transition Signatures for Breast Cancer
              </a>
            </td>
          <td>
            M. S. Jeon, Zehuan Ding, Casey Pei, Jing Li, Linglin Xie, Edward Sauter, Ke Kurt Zhang
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a4fa771302c68d0f49b58a8ac556af478b3c9db" target='_blank'>
              DUX4-rearranged B-ALL: deciphering a biological and clinical conundrum.
              </a>
            </td>
          <td>
            Jack Bakewell, Anthony V Moorman, S. Ryan
          </td>
          <td>2025-09-12</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="


 Our previous work demonstrated that oncogenic KRAS mutations in pancreatic cancer induce aberrant transcription-replication conflicts (TRCs). TRCs arise when DNA replication forks collide with transcription complexes due to loss of spatial and temporal coordination between these processes. TRC resolution mechanisms are and adaptive dependency to KRAS-mediated oncogenic stress. Here we identify a key pathway that regulates TRCs in PDAC and performs mechanistic studies towards clinical translation.



 TRC factors were identified through an siRNA screen against 200 DNA repair genes in Panc1 cells using a high-throughput proximity ligation assay (HiPLA). Knock-out and complementation experiments were performed using CRISPR-Cas9 systems. Cell proliferation was measured using incucyte. Replication stress and DNA damage were measured using multiple orthogonal methods. Protein-protein interactions were assessed by co-immunoprecipitation mass spectrometry and validated by western blot. Chromatin occupancy of key factors was determined using ChIP-seq.



 Through high-throughput proximity ligation assay (HiPLA) screening of 200 known DDR genes, we identified the BER pathway as a key mechanism for TRC resolution. Among these, Neil2 - a bifunctional glycosylase in the base excision repair (BER) pathway - emerged as one of the highest-scoring BER factors; and the only BER glycosylase involved—likely due to its transcription-coupled repair activity. Using CRISPR/Cas9-mediated knockdown, we found that loss of Neil2 led to elevated double-stranded DNA breaks (DSBs) and replication stress. Notably, this significantly impaired cell cycle progression. colony formation, and proliferation in pancreatic cancer cell lines, with minimal effects on normal pancreatic epithelial cells, highlighting Neil2’s selective oncogenic function. To elucidate the underlying mechanisms, we combined proteomic approaches—including co-immunoprecipitation and iPOND-mass spectrometry—with transcriptomic and chromatin profiling with ChIP-seq and RNAseq. Our data revealed that Neil2 plays a critical role in transcription termination, and that its dysregulation promotes RNA Polymerase II readthrough, likely contributing to R-loop accumulation, downstream TRCs, and subsequent genotoxicity. Intriguingly, deletion of Neil2’s DNA binding domain—which abolishes its DNA lesion binding capability—did not compromise its regulation of TRCs or support of cancer cell viability. This suggests that Neil2’s oncogenic role in pancreatic cancer may be DNA-binding independent.



 We uncovered a noncanonical and essential function of Neil2 in TRC regulation and oncogenic maintenance in pancreatic cancer, and we propose Neil2 as a promising therapeutic target for selective intervention in this disease.



 Fan Meng, Mustafa Raoof. Neil2 is a potential theratputic targert in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr A041.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d45ba09d3e2c9d65e7a73127234b8bec15900c3d" target='_blank'>
              Abstract A041: Neil2 is a potential theratputic targert in pancreatic cancer
              </a>
            </td>
          <td>
            Fan Meng, Mustafa Raoof
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Most of the human genome is transcribed into noncoding RNAs (ncRNAs), which although do not encode proteins, are often involved in key cellular processes. Recent technological advances have accelerated the discovery of ncRNA as well as our understanding of ncRNAs as cancer biomarkers and targets. In this “In the Limelight” special issue of FEBS Open Bio, four review articles by leading experts provide an overview of the role of ncRNAs in cancer, from their association to liver cancer to their role as oncogenes in head and neck squamous cell carcinoma (HNSCC).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39d403967dd7059c9aa239237ba085b5894f7584" target='_blank'>
              Non‐coding RNAs as key player in cancer diagnosis and treatment
              </a>
            </td>
          <td>
            Marcin Majka
          </td>
          <td>2025-09-01</td>
          <td>FEBS Open Bio</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Dps is the most abundant nucleoid-associated protein in starved Escherichia coli with ~180,000 copies per cell. Dps binds DNA and oxidises iron, facilitating survival in harsh environments. Dps-DNA complexes can form crystalline structures, leading to the proposed model that Dps reorganises the starved E. coli nucleoid into a compact liquid crystal, slowing chromosome dynamics and limiting access of other proteins to DNA. In this work, we directly tested this model using live-cell super-resolution microscopy and Hi-C analysis. We found that after 96 h of starvation, Dps compacts the nucleoid and increases short-range DNA-DNA interactions, but does not affect chromosome accessibility to large protein nanocages or small restriction enzymes. We also report that chromosome dynamics and organisation are primarily impacted by the bacterial growth phase; the effect of Dps is relatively minor. Our work clarifies the role of Dps in modulating nucleoid properties, and we propose an updated model for Dps-DNA interactions in which Dps binds, protects and compacts DNA largely without influencing chromosome access, dynamics and organisation. Additionally, this work provides a general framework for assessing the impact of nucleoid-associated proteins on key aspects of chromosome function in live cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a6930c67bb9911b075c952097a3099d7e9b19fb" target='_blank'>
              Dps binds and protects DNA in starved Escherichia coli with minimal effect on chromosome accessibility, dynamics and organisation
              </a>
            </td>
          <td>
            Lauren A. McCarthy, Lindsey E. Way, Xiaofeng Dai, Zhongqing Ren, David E. H. Fuller, Ishika Dhiman, Linaria Larkin, Jurriaan J. D. Sieben, Ilja Westerlaken, E. Abbondanzieri, Gail G Hardy, Anne S. Meyer, Xindan Wang, Julie S. Biteen
          </td>
          <td>2025-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Lysine methyltransferase 5a (KMT5a) plays a key role in the pathogenesis of many human diseases. Here, we review the diverse impacts of KMT5a activity on human cancer development and progression. First, KMT5a is the only Mammalian enzyme that specifically induces monomethylation of histone 4 (H4) on lysine 20, thus regulating chromatin organization and, in turn, the transcription of several oncogenes and tumor suppressor genes. KMT5a, by inducing H4 methylation, also critically establishes the choice between different pathways of DNA double-strand break repair, with important consequences for genomic instability and cancer origin. Finally, KMT5a also methylates lysine residues on nonhistone proteins, and KMT5a-induced methylation of key oncogenic and tumor suppressor proteins, including TP53, strongly affects cancer cell functions. Overall, KMT5a is overexpressed in a high percentage and wide variety of human cancers and has protumorigenic activity, which makes it a target for innovative therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0d2924a6f5a75832faa0ef5f5bf3e3b48c7c2e1" target='_blank'>
              Multiple functions of the lysine methyltransferase KMT5a in cancer: potential targets for innovative therapies
              </a>
            </td>
          <td>
            R. Della Monica, Michela Buonaiuto, Mariella Cuomo, Davide Costabile, Claudio Schonauer, G. Catapano, Lorenzo Chiariotti, Roberta Visconti
          </td>
          <td>2025-09-29</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Enhancer of Zeste Homolog 2 (EZH2) is a key epigenetic regulator known for its role in global gene silencing and is involved in a variety of cellular processes, including cell survival, proliferation, invasion, and self-renewal. As a core component of the Polycomb Repressive Complex 2 (PRC2), EZH2 catalyzes the trimethylation of histone H3 at lysine 27 (H3K27me3), leading to chromatin compaction and transcriptional repression. Dysregulated EZH2 expression is observed in a wide range of solid tumors and hematological malignancies and is frequently associated with increased metastatic potential and poor clinical outcomes. While EZH2 primarily mediates gene silencing through its canonical PRC2-dependent activity, it also exerts oncogenic effects via non-canonical mechanisms. In its non-canonical role, EZH2 acts independently of PRC2, interacting with other signaling molecules as a transcriptional activator or co-activator, thereby promoting the activation of oncogenic pathways. Through both canonical and non-canonical mechanisms, EZH2 significantly contributes to tumor initiation and its subsequent progression. Given its critical role in oncogenesis and cancer progression, EZH2 is under investigation as a potential biomarker for cancer diagnosis and prognosis. This review provides a comprehensive overview of EZH2’s function and oncogenic roles across human cancers. Enhanced insight into EZH2’s complex regulatory network may facilitate the development of more effective strategies to manage EZH2-driven malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac9083fda14d6cccbdbf2f40711aac3833884d4a" target='_blank'>
              EZH2 Dysregulation and Its Oncogenic Role in Human Cancers
              </a>
            </td>
          <td>
            Shiv Verma, Nikita Goyal, Suhani Goyal, Parminder Kaur, Sanjay Gupta
          </td>
          <td>2025-09-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Retroelements are repetitive sequences that comprise 42% of the human genome and are strictly regulated through various epigenetic mechanisms. Examining retroelement expression on a locus-specific level in relation to cancer can uncover distinct disease signatures. Results Using over 5000 RNA-sequencing samples, we assessed retroelement transcription across 23 tissue systems, 159 cell types, 1019 cancer cell lines, and cells isolated from various stages of embryogenesis using the specialized software tool, Telescope. In healthy individuals, 11,388 retroelements were found to be actively transcribed and dynamically regulated in a tissue- and cell type-dependent manner. Using the adult human body as a reference, we observed that 94% of cancer cell lines displayed elevated transcription of at least one cancer-specific retroelement, providing a three-fold larger reservoir of cancer biomarkers (1182) than our comparable analysis of annotated protein-coding genes (338). The precise retroelements that were transcribed following tumorigenesis were influenced by the originating location, with cancers of the blood, lungs, and soft tissue displaying the highest origin-specific activation. Moreover, nearly half of the cancer-specific retroelement loci, mostly from the HERV-H family, were found to be expressed during early embryonic development. Conclusions Our data demonstrate that elevated transcription of certain tissue-specific and embryonic retroelements can be considered as a hallmark of tumorigenesis. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00376-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e98e92e95f88b68527f2a00c19cdab4d4246b04c" target='_blank'>
              Strict retroelement regulation is frequently lost following cancer transformation and generates a promising reservoir of cancer biomarkers
              </a>
            </td>
          <td>
            Eric Russ, Sergey Iordanskiy
          </td>
          <td>2025-10-03</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA methylation inhibitors are widely used in treating myeloid malignancies, yet their precise effects on chromatin organization and nuclear architecture remain incompletely understood. Here, the integrated molecular, cellular, and biophysical approaches to investigate the impact of azacitidine (AZA) and decitabine (DEC) on chromatin structure and nuclear mechanics in AML-007 leukemia cells are presented. Confocal microscopy revealed drug-induced alterations in nuclear morphology and actin cytoskeleton organization, with DEC inducing significant nuclear enlargement and disorganization at lower concentrations (1.0 µM) compared to AZA (5.0 µM). Chromatin condensation assays demonstrated that DEC increased chromatin accessibility in a concentration-dependent manner, while AZA produced subtler effects. Optical tweezers measurements showed both agents reduced nuclear stiffness, with DEC exerting a greater impact. Spectroscopic analysis confirmed differential drug incorporation into DNA, with higher methylation loss and structural changes observed following DEC treatment. Refractive index mapping revealed chromatin decompaction, aligning with increased accessibility and nuclear softening. These findings demonstrate that DNA hypomethylating agents exert distinct, concentration-dependent effects on nuclear organization and chromatin structure, which can be quantified through molecular and biophysical readouts. This study underscores the value of integrative methods for revealing epigenetic drug effects on chromatin architecture in leukemia cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4906405a8ab8442becabd2d89c99471c5e4596eb" target='_blank'>
              Effects of DNA Methylation Inhibitors on Molecular and Structural Changes in Chromatin Organization in Leukemia Cells.
              </a>
            </td>
          <td>
            Przemysław Sareło, A. Kaczorowska, Marlena Gąsior-Głogowska, Kinga Godkowicz, Weronika Lamperska, Sławomir Drobczyński, Ewa Zioło, Anna A Licznerska, Tomasz Wróbel, Halina Podbielska, Wojciech Kałas, M. Kopaczyńska
          </td>
          <td>2025-09-29</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The strength of the repressive histone H3K27me3 signal varies across silencers. Focusing on regions with unusually strong signals—super-silencers—we show that B-cell super-silencers are initially linked to gene upregulation in development, with target genes highly expressed in stem cells. About 13% of B-cell super-silencers convert to super-enhancers in B-cell lymphoma; 22% of these recur in over half of patients. Genes like BCL6 and BACH2 tied to these conversions are downregulated faster by JQ1, a super-enhancer-disrupting anti-cancer agent. Super-silencers are enriched for B-cell cancer-associated variants—both somatic and germline—and translocation breakpoints, exceeding levels in other regulatory elements like CTCF binding sites. Over 80% of B-cell lymphoma t(3;14)(q27;q32) translocations fuse BCL6 super-silencers with enhancer-rich regions. Super-silencer repression depends on CpG content: CpG-rich elements block promoter–enhancer contacts; CpG-poor — inhibit looping. These findings highlight super-silencers’ key role in B-cell regulation and suggest their alteration may be a primary factor of B-cell carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cab550e747142b856a2c8f404a6e1c22dc14158c" target='_blank'>
              Super-silencers are crucial for development and carcinogenesis in B cells
              </a>
            </td>
          <td>
            Di Huang, H. Petrykowska, Dhaneshwar Kumar, L. Kardava, S. Moir, B. Afzali, L. Elnitski, Ivan Ovcharenko
          </td>
          <td>2025-09-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Replication stress (RS) poses a threat to genome stability and drives genomic rearrangements. The homologous recombination (HR) pathway repairs stalled replication forks (RFs) and prevents such instability. Through an E3 ubiquitin ligase screen aimed at identifying regulators of RAD51, we identified macrophage erythroblast attacher (MAEA), a core component of C-terminal to Lish (CTLH) E3 ubiquitin ligase complex, as a regulator of the HR pathway. Loss of MAEA impairs RAD51 recruitment at stalled RFs, leading to increased sensitivity to RS-inducing agents and excessive degradation of nascent DNA strands. Mechanistically, MAEA associates with and mediates the ubiquitylation of Ku80, enabling its removal from RF ends and facilitating the loading of RAD51. Notably, MAEA deficiency is associated with a developmental disorder involving microcephaly, craniofacial abnormalities, ocular defects, and heart malformations. Functional assays show that disease-linked MAEA variants (R34C, E349G, Y394D, and M396R) are defective in RS response. These findings establish MAEA as an essential factor in RF protection and genome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c499707b53080fc3870cbe7d66d1176114a45109" target='_blank'>
              Pathogenic variants in MAEA disrupt DNA replication fork stability and are associated with developmental abnormalities in humans
              </a>
            </td>
          <td>
            Elham Zeinali, Fatemeh Mashayekhi, Rabih Abou Farraj, Talah Hasanni, Marie-christine Caron, Yan Coulombe, A. Fitieh, J. N. M. Glover, Jean-Yves Masson, Ismail Hassan Ismail
          </td>
          <td>2025-08-29</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4c68bd6ad67db140a21e19e70ce02281cc63746" target='_blank'>
              Transcriptional Rewiring of BET Inhibitor Treated Ewing Sarcoma Cells Augments their Dependency on Focal Adhesion Kinase.
              </a>
            </td>
          <td>
            Shireen S Ganapathi, Emma D. Wrenn, Nicolas M Garcia, Neerja Katiyar, Stephanie I. Walter, Aya Miyaki, Jonah R. Valenti, Yuqi Kang, Marina Chan, Taranjit S. Gujral, Elizabeth R. Lawlor
          </td>
          <td>2025-09-06</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The chromatin is folded into three-dimensional (3D) structures, and aberrant 3D chromatin folding has been implicated in cancer. We performed ATAC-seq and TOP2A ChIP-seq to assess the potential effects of various anthracycline drugs on the chromatin architecture. We found that specific anthracycline variants selectively disrupt chromatin looping anchors by interfering with CTCF binding, suggesting an additional therapeutic mechanism of anthracycline drugs targeting the 3D genome. Hi-C experiments in K562 cells treated with anthracycline drugs revealed widespread disruption of 3D chromatin organization, including altered long-range regulation at the Myc locus. Furthermore, AML patients treated with anthracycline drugs exhibited changes in chromatin structures near possible looping anchors, which were associated with distinct clinical outcomes. Together, our findings indicate that anthracycline drugs function as potent and selective epigenomic modulators, with the capacity to further target the 3D genome to exert anticancer effects, highlighting their potential for personalized therapy in tumors with aberrant 3D chromatin architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3d9cc4974c393dd5c8e640f80106dc4e3c5a03a" target='_blank'>
              Targeting the 3D genome by anthracyclines for chemotherapeutic effects.
              </a>
            </td>
          <td>
            Minkang Tan, Shengnan Sun, Yuchen Liu, Andrea A. Perreault, D. Phanstiel, Liping Dou, Baoxu Pang
          </td>
          <td>2025-10-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Humans are continually exposed to a diverse array of environmental chemicals that can damage DNA and compromise genomic integrity. Among these genotoxic agents, acetaldehyde, acrolein, aristolochic acids, and vinyl chloride are particularly concerning due to their widespread presence in industrial emissions, dietary sources, and lifestyle-related exposures such as smoking and alcohol consumption. These compounds can induce structurally distinct forms of DNA damage including bulky DNA adducts, interstrand crosslinks, and other replication-blocking lesions. While canonical DNA repair pathways serve as the primary defense against such DNA damage, some lesions persist, challenging the capacity of DNA repair systems. If not efficiently repaired, DNA lesions may disrupt replication and transcription. In many cases, translesion synthesis polymerases are recruited to bypass unrepaired lesions, introducing mutations that contribute to agent-specific mutational signatures found in cancer genomes. This review systematically examines how each of these four exogenous chemicals induces DNA damage, the DNA repair pathways responsible for removing their lesions, and the role of translesion synthesis in shaping their mutational signatures. We also highlight how three-dimensional genome organization regulates lesion susceptibility and repair, contributing to variability of mutational landscapes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c10fba52c520395f5d989cae353584818e523a" target='_blank'>
              Mechanisms of DNA repair and mutagenesis induced by acetaldehyde, acrolein, aristolochic acids, and vinyl chloride.
              </a>
            </td>
          <td>
            Kyle Jones, Emmaline Dye, Nyla Gilkes, Dian Xia, Shan Jiang, Wentao Li
          </td>
          <td>2025-08-16</td>
          <td>Journal of environmental science and health. Part C, Toxicology and carcinogenesis</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Single base substitution (SBS) mutations, particularly C > T and T > C, are increased owing to unrepaired DNA replication errors in mismatch repair-deficient (MMRd) cancers. Excess CpG > TpG mutations have been reported in MMRd cancers defective in mismatch detection (dMutSα), but not in mismatch correction (dMutLα). Somatic CpG > TpG mutations conventionally result from unrepaired spontaneous deamination of 5’-methylcytosine throughout the cell cycle, causing T:G mismatches and signature SBS1. It has been proposed that MutSα detects those mismatches, prior to error correction by base excision repair (BER). However, other evidence appears inconsistent with that hypothesis: for example, MutSα is specifically expressed in S/G2 phases of the cell cycle, and defects in replicative DNA polymerase proofreading specifically cause excess CpG > TpG mutations in signature SBS10b. Methods We analysed mutation spectra and COSMIC mutation signatures in whole-genome sequencing data from 1803 colorectal cancers (164 dMutLα, 20 dMutSα) and 596 endometrial cancers (103 dMutLα, 9 dMutSα) from the UK 100,000 Genomes Project. We mapped each C > T mutation to its genomic features, including normal DNA methylation state, replication timing, transcription strand, and replication strand, to investigate the mechanism(s) by which these mutations arise. Results We confirmed that dMutSα tumours specifically had higher CpG > TpG burdens than dMutLα tumours. We could fully reconstitute the observed dMutSα CpG > TpG mutation spectrum by adding CpG > TpG mutations in proportion to their SBS1 activity to the dMutLα spectrum. However, other evidence indicated that the SBS1 excess in dMutSα cancers did not come from 5’-methylcytosine deamination alone: non-CpG C > T mutations were also increased in dMutSα cancers; and, in contrast to tumours deficient in BER, CpG > TpG mutations were biased to the leading DNA replication strand, at similar levels in dMutSα and dMutLα cancers, suggesting an origin in DNA replication. Other substitution mutations usually corrected by BER were not increased in dMutSα tumours. Conclusions There is a CpG > TpG and SBS1 excess specific to dMutSα MMRd tumours, consistent with previous reports, and we find a general increase in somatic C > T mutations. Contrary to some other studies, the similar leading replication strand bias in both dMutSα and dMutLα tumours indicates that at least some of the excess CpG > TpG mutations arise via DNA replication errors, and not primarily via the replication-independent deamination of 5’-methylcytosine. Supplementary Information The online version contains supplementary material available at 10.1186/s13073-025-01525-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2f0be2946a1331e8205341cd8d8b7fbe7da7996" target='_blank'>
              Replication-associated mechanisms contribute to an increased CpG > TpG mutation burden in mismatch repair-deficient cancers
              </a>
            </td>
          <td>
            Joseph C Ward, Ignacio Soriano, Steve Thorn, Juan Fernández-Tajes, Kitty Sherwood, Güler Gül, Joost Scheffers, A. Frangou, Ben Kinnersley, Ioannis Kafetzopoulos, Duncan Sproul, Sara Galavotti, C. Palles, Andreas J. Gruber, David N. Church, Ian Tomlinson
          </td>
          <td>2025-08-25</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0598f4e042a38ac90e8698c02adae86ab3960654" target='_blank'>
              KCTD10 is a sensor for co-directional transcription-replication conflicts.
              </a>
            </td>
          <td>
            Jake A. Kloeber, Bin Chen, Guangchao Sun, Charles S King, Zhiquan Wang, Li Wang, Zheming Wu, Shouhai Zhu, Fei Zhao, Hongran Qin, Yaobin Ouyang, Huaping Xiao, X. Tu, Jing Lu, Yanxia Jiang, K. Luo, Ping Yin, Xinyan Wu, R. Mutter, Jinzhou Huang, Zhenkun Lou
          </td>
          <td>2025-10-08</td>
          <td>Nature</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19205ad880105516a7cc4fc10706c3c6e6a6cfba" target='_blank'>
              The functions and regulation of transcription factor YBX1 in cancers.
              </a>
            </td>
          <td>
            Aixin Qiu, Yue Zhang, Yihan Dong, Dan Yue, Yang Yu
          </td>
          <td>2025-09-12</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Pancreatic ductal adenocarcinoma (PDA) is a highly aggressive disease with a 5-year survival rate of 12%. Patients are typically diagnosed at the metastatic stage due to lack of symptoms and current treatment options do not effectively control the disease. There is an urgent need to develop sensitive strategies to detect PDA at an early stage before it spreads. Characterizing circulating tumor DNA (ctDNA) in plasma is an effective approach to detect and monitor cancer. Typically, cancer genomes contain thousands of mutations, but targeted sequencing approaches only focus on pre-determined driver genes, which limits their performance in detecting ctDNA. Whole-genome sequencing (WGS) however, tracks all the tumor mutations simultaneously, and has a higher sensitivity than targeted approaches, especially in low tumor burden samples. In this study, we aim to use plasma WGS to build a liquid biopsy platform for PDA detection and characterization. To date, we have established the largest plasma WGS (20-60x) PDAC cohort which consists of over 250 baseline and longitudinal plasma samples from 205 PDA patients, with an additional 45 plasma samples from age-matched healthy individuals. Paired tumor tissue WGS and RNAseq are also available. We developed and validated a bioinformatics pipeline to detect cancer mutations in plasma and estimate plasma tumor fraction (TFx) for tumor burden. In the PDA cohort, plasma TFx increased with clinical stages (medians of 0.5% in resectable, 0.8% in locally advanced and 5% in metastatic), as did the ctDNA detection rate which was measured by whether the plasma TFx was significantly higher than controls (87%, 91% and 95%). Plasma TFx positively correlated with liver metastasis size (R=0.68, P<0.001), but not with the primary pancreatic tumor size. These results suggest that in PDA, metastases are the major site to shed ctDNA, but not primary tumors. We also found that liver metastasis size had a stronger correlation with plasma TFx than with serum CA19-9 (R=0.32, P=0.002). Baseline plasma TFx was associated with shorter overall survival (OS) in both resectable (P=0.013) and metastatic (P=0.022) cases. By analyzing the paired tumor WGS and RNAseq, we found that higher plasma TFx was associated with higher cell cycle, lower immunity and polyploid genomes in the tumors. No difference was found in plasma TFx between basal-like and classical PDA subtypes. In summary, the current study provides insights into the interplay between ctDNA dynamics, tumor biology and clinical features in PDA. Alongside with these results, we will provide a freely accessible computational platform for ctDNA genomic profiling, PDA early detection and progression monitoring using plasma.



 Yuanchang Fang, Michelle Chan-Seng-Yue, Karen Ng, Amy Zhang, Tuan Hoang, Gun Ho Jang, Sabiq Chaudhary, Eugenia Flores-Figueroa, Daniela Bevacqua, Stephanie Ramotar, Ayelet Borgida, Shawn Hutchinson, Anna Dodd, Julie Wilson, Barbara Grünwald, Robert Grant, Erica Tsang, George Zogopoulos, Jennifer Knox, Masoom Haider, Steven Gallinger, Faiyaz Notta. Plasma whole-genome sequencing to monitor pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B072.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41904b2fb7e8d9c5cced0dd565b2ced62da7f071" target='_blank'>
              Abstract B072: Plasma whole-genome sequencing to monitor pancreatic cancer
              </a>
            </td>
          <td>
            Yuanchang Fang, M. Chan-Seng-Yue, Karen Ng, Amy Zhang, Tuan Hoang, G. Jang, Sabiq Chaudhary, Eugenia Flores-Figueroa, Daniela Bevacqua, S. Ramotar, A. Borgida, S. Hutchinson, A. Dodd, Julie M. Wilson, Barbara Grünwald, Robert C. Grant, Erica S. Tsang, G. Zogopoulos, Jennifer J Knox, Masoom A. Haider, Stephen Gallinger, F. Notta
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Copy number control of DNA and centrosomes is essential for accurate genetic inheritance. DNA replication and centrosome duplication have been recognized as parallel key events for cell division. Here, we discover that the DNA replication machinery directly regulates the licensing and execution processes of centrosome duplication to prevent centrosome amplification. We find that the microcephaly protein DONSON couples DNA replication initiation with Cdc6 translocation to centrosomes. The Cdc6 signal prevents the precocious occurrence of centriole disengagement, the licensing step for centrosome duplication. During DNA replication, DONSON inhibits replisome disassembly by interacting with the CMG helicase, maintaining the intrinsic S/G2 checkpoint signal that blocks centriole-to-centrosome conversion, the execution step for centrosome duplication. Disruption of these dual signals causes precocious centrosome duplication and chromosome mis-segregation, observed in DONSON patient cells. Our results reveal that the DNA replication machinery not only duplicates genetic material but also controls the system for its accurate segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895f8bba3e9e4612c1b72098d2d19a10b12b8583" target='_blank'>
              The DNA replication machinery transmits dual signals to prevent unscheduled licensing and execution of centrosome duplication
              </a>
            </td>
          <td>
            Kyohei Matsuhashi, Kei K Ito, Kaho Nagai, Akira Sanada, Koki Watanabe, Kasuga Takumi, Atsushi Toyoda, Masamitsu Fukuyama, Shohei Yamamoto, Takumi Chinen, Grant S Stewart, Shoji Hata, Daiju Kitagawa
          </td>
          <td>2025-09-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Long interspersed nuclear element-1 (LINE-1 or L1), the only autonomously active retrotransposon in humans today, constitutes a large proportion of the genome and continues to evolve the genome and impact fundamental biological processes. L1 retrotransposition critically depends on its endonuclease and reverse transcriptase subunit open reading frame 2 protein (ORF2p), which targets genomic loci and nicks DNA using an evolutionarily distinct yet not fully understood mechanism. Our structural and biochemical analyses revealed that ORF2p is a structure-dependent endonuclease. It binds a double-stranded DNA region upstream of the nicking site and recognizes a downstream forked or flap structure for efficient DNA nicking. This discovery suggests that L1 mobilization piggybacks on chromosomal processes with noncanonical DNA structure intermediates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f0ddeb532f6d95f55ced91946685205519cc0c7" target='_blank'>
              Mechanism of DNA targeting by human LINE-1.
              </a>
            </td>
          <td>
            Wenxing Jin, Cong Yu, Yan Zhang, Changchang Cao, Tianfan Xia, Ge Song, Zhaokui Cai, Yuanchao Xue, Bing Zhu, 
          </td>
          <td>2025-10-09</td>
          <td>Science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract The mycotoxin, aflatoxin B1 (AFB1), is a potent mutagen that contaminates agricultural food supplies. After ingestion, AFB1 is oxidized into a reactive electrophile that alkylates DNA, forming bulky lesions such as the genotoxic formamidopyrimidine lesion, AFB1-Fapy dG. This lesion is mainly repaired by nucleotide excision repair (NER) in bacteria; however, in humans the picture is less clear. We report a plasmid-based host cell reactivation assay containing a site-specific AFB1-Fapy dG lesion and present evidence that this lesion is mainly repaired by transcription-coupled NER (TC-NER) in human cells. Using a combination of isogenic knockout cell lines and immortalized fibroblasts from xeroderma pigmentosum and Cockayne syndrome patients, we show that the TC-NER factors CSA, CSB, and UVSSA are required for efficient AFB1-Fapy dG repair, while the global-genome NER protein, XPC, is dispensable. Furthermore, knockout of CSB or UVSSA impairs AFB1-Fapy dG repair to a similar degree as knockout of the core NER nuclease, XPF. Our data indicate that TC-NER is the major repair pathway for AFB1-Fapy dG adducts in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97fa8068164aef8be534a5e2a67772aee6a9f8ae" target='_blank'>
              The aflatoxin B1-induced formamidopyrimidine adduct is repaired by transcription-coupled nucleotide excision repair in human cells
              </a>
            </td>
          <td>
            D. Laverty, Rachana Tomar, Sophie Erlich, Michael P. Stone, Zachary D. Nagel
          </td>
          <td>2025-09-01</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="SUMMARY DNA replication is molecularly asymmetric, due to distinct mechanisms for lagging and leading strand DNA synthesis. Whether chromatin assembly on newly replicated strands is also asymmetric remains unknown, as visualizing this short-lived state in cells is impossible. To circumvent this limitation, we combine in vitro reconstitution of the Saccharomyces cerevisiae DNA and chromatin replication machineries with replication-aware single-molecule chromatin footprinting, to study how chromatin is re-assembled on replicated DNA. Leveraging the non-destructive, single-molecule, and strand-specific nature of our data, we discover an intrinsic asymmetry in nucleosome positioning patterns and organization between lagging- and leading-strand chromatin created by the yeast replisome. This asymmetry is only partially restored upon addition of chromatin assembly factors involved in de novo histone deposition and the ATP-dependent chromatin remodeler Isw1a, implying that other regulatory factors must resolve this asymmetry in cells. In sum, our data reveal the complexity of chromatin re-establishment following DNA replication, and suggest an asymmetric chromatin assembly intermediate on each daughter chromatid. These pathways have implications for essential chromatin-templated processes such as DNA repair, transcription, and gene silencing at replication forks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a40a3157eae67e28210affbf8a47af9f74eab0a3" target='_blank'>
              The eukaryotic replisome intrinsically generates asymmetric daughter chromatin fibers
              </a>
            </td>
          <td>
            Bruna V. Eckhardt, Hannah J Richter, Inge Rondeel, Keerthi Renduchintala, Francesca Mattiroli, Vijay Ramani
          </td>
          <td>2025-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a59a21b3122614696fcbf37a9973f0709eb05a0" target='_blank'>
              Unravelling the genetics and epigenetics of the ageing tumour microenvironment in cancer.
              </a>
            </td>
          <td>
            H. Easwaran, A. Weeraratna
          </td>
          <td>2025-09-08</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="MicroRNAs (miRNAs) systematically orchestrate multiple biological processes via gene regulation in various organisms. Conventionally, miRNAs bind to untranslated regions of mRNAs to modulate gene silencing in the cytoplasm. Recent studies have revealed that a subset of miRNAs located in the nucleus can efficiently activate gene transcription by targeting enhancers, named as Nuclear Activating miRNAs (NamiRNAs). The fundamental principle of this regulatory pattern is that NamiRNAs as "wedges" open double strands of genomic DNA and provide the prerequisite for RNA polymerase II-mediated transcription. Specifically, NamiRNAs bind to their target enhancers and form the hybrids of miRNAs and single-stranded DNA, which are recognized by Argonaute 2 (AGO2) to maintain the single-stranded state. Meanwhile, AGO2 protects these miRNA-DNA hybrids from RNase H-mediated degradation to maintain their stability. Additionally, NamiRNAs induce H3K27ac enrichment at target enhancers and put them into an active state conducive to gene activation. Notably, similar to NamiRNAs, Human Identical Sequences as exogenous small RNAs as critical pathogenic factors for RNA viruses, facilitate infectious disease progression. Collectively, this review systematically elucidates the theory of NamiRNA-Enhancer-mediated Gene Activation in combination with corresponding evidence, summarizes the functions and challenges of NamiRNAs, and discusses their potential in fundamental research and clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff4fdc35f7062b8b76f4cff58088fb91115a9279" target='_blank'>
              NamiRNA-Enhancer Network: The Changing Era Challenge and Chance of miRNA.
              </a>
            </td>
          <td>
            Lu Chen, Ying Liang, Hongyan Wei, Mengxing Liu, Jin Wu, Wei Li, Wenxuan Li, Yuxiao Jin, Yinshan Li, Wei Zhao, Min Xiao, Kaicheng Zhou, Shuai Yang, Wenqiang Yu
          </td>
          <td>2025-08-26</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Nucleoside analogues are used as drugs and as labels in laboratory-based research. However, the effect of different nucleoside analogue mechanism(s) on cell sensitivity or mutagenesis is unclear. This is particularly important in cancer treatments where checkpoint proteins and DNA damage factors are often mutated. We tested six nucleoside analogues in fission yeast, Schizosaccharomyces pombe. We found that the mutations in the DNA replication checkpoint cause unique sensitivity profiles towards chemotherapeutic nucleoside analogues (gemcitabine, 5-fluorouracil, cytarabine) and the non-clinical analogue bromodeoxyuridine. Antiretroviral compounds, zidovudine and lamivudine, did not alter cell growth. We compared half-maximal inhibitory concentration (IC50) doses between checkpoint deficient yeast strains, examining culture growth and DNA mis-segregation. Intriguingly, gemcitabine and bromodeoxyuridine doses above the IC50 promoted better growth. Above each compound’s IC50 dose we saw that cells were insensitive to nucleoside analogue re-exposure, particularly in DNA replication checkpoint mutants (cds1∆, rad3∆). Thus, pairing nucleoside analogue use with personal genomics may inform drug choice, dose, and schedule. Finally, these data indicate that resistance may be predictable, informing clinical strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea9673674f99d856da45d7531d1ed654d61e6d4f" target='_blank'>
              Checkpoint-Dependent Sensitivities to Nucleoside Analogues Uncover Specific Patterns of Genomic Instability
              </a>
            </td>
          <td>
            Zainab Burhanuddin Kagalwala, Mohammed Ayan Chhipa, Zohreh Kianfard, Essam Karam, Sirasie P. Magalage, Sarah A. Sabatinos
          </td>
          <td>2025-09-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mutations reflect the net effects of myriad types of damage, replication errors, and repair mechanisms, and thus are expected to differ across cell types with distinct exposures to mutagens, division rates, and cellular programs. Yet when mutations in humans are decomposed into a set of “signatures”, one single base substitution signature, SBS5, is present across cell types and tissues, and predominates in post-mitotic neurons as well as male and female germlines [1–3]. The etiology of SBS5 is unknown. By modeling the processes by which mutations arise, we infer that SBS5 is the footprint of errors in DNA synthesis triggered by distinct types of DNA damage. Supporting this hypothesis, we find that SBS5 rates increase with signatures of endogenous and exogenous DNA damage in cancerous and non-cancerous cells and co-vary with repair rates along the genome as expected from model predictions. These analyses indicate that SBS5 captures the output of a “funnel”, through which multiple sources of damage result in a similar mutation spectrum. As we further show, SBS5 mutations arise not only from translesion synthesis but also from DNA repair, suggesting that the signature reflects the occasional, shared use of a polymerase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0289ae32bcbca4304d80ec26b288975aaa0f5e9" target='_blank'>
              Collateral mutagenesis funnels multiple sources of DNA damage into a ubiquitous mutational signature
              </a>
            </td>
          <td>
            Natanael Spisak, Marc de Manuel, Molly Przeworski
          </td>
          <td>2025-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0643ed748b04336ec828931abc9d5fbd2911609" target='_blank'>
              [Progress of scRNA-seq technology in nasopharyngeal carcinoma research].
              </a>
            </td>
          <td>
            Bin Zheng, Guanqiao Jin
          </td>
          <td>2025-09-01</td>
          <td>Lin chuang er bi yan hou tou jing wai ke za zhi = Journal of clinical otorhinolaryngology head and neck surgery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The proposed existence of a widespread “tumor microbiome” has sparked considerable interest, yet recent evidence suggests many microbial DNA signals in cancer genomes arise from contamination rather than true colonization. A September 2025 Science Translational Medicine analysis of over 5,700 cancer genomes demonstrated that most detected sequences reflected artifacts, underscoring earlier concerns about low-biomass sequencing studies. While enthusiasm for a universal tumor microbiome must be tempered, robust associations remain in specific cancers, including Helicobacter pylori in gastric carcinoma, Fusobacterium nucleatum in colorectal cancer, and human papillomavirus in cervical malignancies. This reassessment highlights the urgent need for rigorous methodological standards, such as robust negative controls, contamination-aware pipelines, and transparent reporting, to ensure reproducibility. Rather than discouragement, this represents an opportunity to refocus research on biologically plausible, clinically relevant cancer-microbe interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1b7d3da6d9e90a3a12169f914f9edbbd5cf84a" target='_blank'>
              Reassessment of microbial DNA in cancer genomes: addressing contamination controversies
              </a>
            </td>
          <td>
            Muhammad Shahid Mehmood, Fatima Hajj
          </td>
          <td>2025-10-13</td>
          <td>Annals of Medicine &amp; Surgery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA topoisomerase IIα (TOP2A) plays a key role in resolving DNA topological stress during replication, transcription, and chromosome segregation. Yatein, a biogenic precursor of deoxypodophyllotoxin possesses antitumor effects. Herein, we explored its antitumor effect in breast cancer. Yatein significantly inhibited the proliferation of breast cancer cells by MTT, colony formation, EdU assay, and cell cycle analysis. Moreover, yatein suppressed the stemness of cancer cells, causing inhibition of migration and epithelial-mesenchymal transition. Mechanism study revealed that yatein inhibited TOP2A activity and stabilized TOP2A-DNA cleavage complexes, caused DNA damage and apoptosis, suggesting its potential in the treatment of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2d219f76fc877a179547f374e85d00ebe2c5225" target='_blank'>
              Yatein-induced DNA damage of breast cancer cells by interfering with DNA topoisomerase IIα.
              </a>
            </td>
          <td>
            Yu-Yao Liu, Ya Chen, Qi-Qi Wang, Xi Zhang, Xiao-Xia Ma, Xing-De Wu, Jia Su
          </td>
          <td>2025-08-25</td>
          <td>Journal of Asian natural products research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Post-transcriptional RNA modifications, such as N6-methyladenosine (m6A) methylation and adenosine to inosine (A-to-I) editing, are critical regulators of hematopoietic stem cell (HSC) self-renewal and differentiation, yet their precise contributions to malignant transformation are not fully elucidated. In this study, we uncovered the epitranscriptomic landscape caused by knockdown of genes from the methyltransferase (METTL)-family in hematopoietic stem and progenitor cells (HSPCs). We identified both converging and distinct roles of METTL3 and METTL14, known members of the m6A writer complex, as well as orphan gene METTL13. Notably, METTL13 was uniquely upregulated by adenosine deaminase acting on RNA 1 (ADAR1) overexpression, while other METTL genes were downregulated. Knockdown of METTL13 altered the expression of multiple genes involved in oncogenic development in HSPCs. Furthermore, METTL13 was associated with a high-risk profile in pediatric T-cell acute lymphoblastic leukemia (T-ALL), and functional studies confirmed that METTL13 is required for T-ALL cell proliferation and survival both in vitro and in vivo. Collectively, our results indicate a previously unrecognized, oncogenic role for METTL13 in pre-leukemic transformation and T-ALL pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047267bf0f7cb87eac1cee31332201b93b817d49" target='_blank'>
              METTL13 Promotes Pre-Leukemic Transformation and the Development of Pediatric Leukemia
              </a>
            </td>
          <td>
            Sabina Enlund, Chae-eun Lim, Isabella Hoang, Sonali Joshi, Amanda Ramilo Amor, Cecilia Thomsson, Indranil Sinha, Shahrzad Shirazi Fard, Anna Nilsson, Ola Hermanson, Qingfei Jiang, Frida Holm
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="DNA replication initiation is orchestrated in bacteria by the replication initiator DnaA. Two models for regulation of DnaA activity in Escherichia coli have been proposed: the switch between an active and inactive form, and the titration of DnaA on the chromosome. Although proposed decades ago, experimental evidence of a titration-based control mechanism is still lacking. Here, we first identified a conserved high-density region of binding motifs near the origin of replication, an advantageous trait for titration of DnaA. We then investigated the mobility of DnaA by visualising single proteins inside single cells of wild-type and deletion mutants E. coli strains, while monitoring cellular size and DNA content. Our results indicate that the chromosome of E. coli controls the free amount of DnaA in a growth rate-dependent fashion. Moreover, they address long-standing questions on the relevance of DnaA titration in stabilising DNA replication by preventing re-initiation events during slow growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9245293eecd0e45715a401c652d843bd210c464" target='_blank'>
              The Escherichia coli replication initiator DnaA is titrated on the chromosome
              </a>
            </td>
          <td>
            Lorenzo Olivi, Stephan Köstlbacher, C. Ludwig, Mees Langendoen, Nico J. Claassens, Thijs J. G. Ettema, J. van der Oost, P. R. ten Wolde, Johannes Hohlbein, R. Staals
          </td>
          <td>2025-08-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="In a recent study published in Science , Tiebang Kang and his colleagues 1 identify ASB7 as a key negative regulator of heterochromatin maintenance through targeted degradation of the H3K9me3 methyltransferase SUV39H1. The work uncovers a cell cycle-regulated chromatin-associated ubiquitin ligase circuit that maintains epigenetic homeostasis and reveals a novel vulnerability in cancer that may be exploited therapeutically. H3K9me3 is a de ﬁ ning feature of constitutive heterochromatin, vital for transcriptional repression and genome stability. This repressive histone mark is propagated through a self-reinforcing loop, wherein HP1 binds pre-existing H3K9me3 and recruits SUV39H1 to methylate adjacent nucleosomes. While this mechanism ensures robust inheritance of silencing, unrestrained propagation could lead to widespread gene silencing and excessive chromatin compaction. How this feedback loop is constrained to maintain H3K9me3 homeostasis has remained poorly understood. To uncover regulators that limit H3K9me3 spread, authors conducted a genome-wide CRISPR-Cas9 screen using H3K9me3 immunostaining in asynchronous and S-phase – arrested cells. Among known heterochromatin regulators such as SUV39H1/2, SETDB1, and EHMT1/2, the screen identi ﬁ ed ASB7, a substrate adaptor of the CUL5 E3 ligase complex, as a top hit. ASB7 knockout led to genome-wide H3K9me3 accumulation across both repetitive and non-repetitive regions, con ﬁ rmed by CUT&RUN and immuno ﬂ uorescence. Conversely, ASB7 overexpression suppressed H3K9me3, supporting its role as a global repressor">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f123d6e18344d7fb0613cfecce34c6744d13f4e" target='_blank'>
              Fencing the genome: ubiquitin signaling restricts heterochromatin spread
              </a>
            </td>
          <td>
            Collin Bakker, Joanna Paulson, Nitika Taneja
          </td>
          <td>2025-08-21</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Recent studies have determined a close association between host microbiota and breast cancer initiation, growth and therapeutic outcomes. We previously uncovered how enterotoxigenic Bacteroides fragilis (ETBF), a gut microbe present in malignant breast tissue, aids breast cancer development and metastatic progression via activating multiple oncogenic pathways and modulating breast tumor microenvironment. Brief exposure to ETBF-secreted toxin, BFT (Bacteroides fragilis toxin), imparts long-term oncogenic, pro-stemness and pro-metastasis memory in breast cancer cells indicating the involvement of epigenetic alterations hence, we aimed to investigate the potential involvement of epigenetics in biological impact of ETBF in breast cancer. Methods RNA sequencing and Infinium methylation EPIC microarray were performed in mammary tumors developed from the BFT-exposed breast cancer cells. Alterations in the expression of tumor suppressor genes (TSGs) were assessed using RT-PCR, ICC and IHC. Demethylation agent and HDAC inhibitor were utilized to rescue the expression of BFT-mediated hypermethylated TSGs, and examine the migration/invasion potential of BFT-exposed breast cancer cells. Results EPIC methylation microarray analysis of BFT-exposed mammary tumors uncovered 64 differentially methylated significant CpG sites whose analysis led to the identification of 26 hypomethylated and 156 hypermethylated genes. Of the hypermethylated genes, also showing reduced expression in RNA-seq in BFT-exposed group, we identified five important TSGs, namely, NF2, RSK3, FAT4, DCN and DOK2. Survival analysis revealed that decreased expression of these TSGs associated with worse prognosis. While BFT exposure reduced the expression of these TSGs, treatment with Azacytidine or/and Trichostatin A resulted in rescue of the TSGs from BFT-induced repression while mitigating BFT-driven migration and invasion of breast cancer cells. Conclusions Collectively, BFT exposure epigenetically modifies the expression of TSGs and impacts migration/invasion potential of breast cancer cells, and treatment with demethylation agent(s) and HDAC inhibitors effectively diminishes the functional impact of BFT. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02111-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85dc5b1ea578546e463a62d30a1af760f0719100" target='_blank'>
              An enteropathogenic microbial toxin modulates the breast cancer epigenome resulting in concurrent silencing of tumor suppressor genes
              </a>
            </td>
          <td>
            Deepak Verma, Deeptashree Nandi, S. Parida, Archisha Saxena, Dipali Sharma
          </td>
          <td>2025-09-26</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Spontaneous mutation rates and spectra are influenced by an intricate interplay of processes including DNA replication, proofreading, and diverse DNA damage repair pathways. Although significant progress has been made in characterizing the functions of individual genes involved in these processes, their direct effects on mutation rates and spectra remain unclear. In this study, we employed a systematic gene knockout approach coupled with mutation accumulation (MA) experiments and whole-genome sequencing (WGS) to investigate the mutational landscape of Saccharomyces cerevisiae. We targeted 136 gene-deletion strains encompassing nearly all known genes associated with DNA replication and repair. Analysis of 1,081 MA lines revealed that 114 of the 136 genes significantly influenced at least one type of mutation rate. Furthermore, deletions of specific genes led to marked shifts in mutational biases and spectra, with some deletions amplifying existing biases and others reversing them entirely. In contrast, mitochondrial mutation rates were notably less affected, with no significant impact detected. Importantly, comparative analysis revealed striking similarities between yeast mutational spectrum and those observed in human cancers with the same pathway deficiencies, suggesting conserved functional roles across species. In conclusion, our findings provided critical insights into the genetic underpinnings of these signatures and underscoring the utility of yeast as a model for studying the molecular basis of cancer-associated mutational processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5323012223c8e2fdfc841a6c54bcc4ec219997da" target='_blank'>
              Comprehensive Mutational Landscape of Yeast Mutator Strains Reveals the Genetic Basis of Mutational Signatures in Cancer.
              </a>
            </td>
          <td>
            Lei Liu, Danyang Sun, Haoxuan Liu
          </td>
          <td>2025-10-06</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="RAD50 is the central protein of the MRN complex and crucial in DNA double-strand break repair. RAD50 deficiency causes a genomic instability disorder characterized by microcephaly and stunted growth. Using lentiviral constructs, we investigated whether cancer-related RAD50 missense variants can complement the delayed damage response after exposure to the chemotherapeutic agent epirubicin and/or mitotic progression in RAD50-deficient fibroblasts. Eight missense variants, all capable of forming an MRN complex, supported the DNA damage response and mitotic features to different extents, indicating these functions are separable. Three variants showed both an impaired epirubicin response and slowed cell division in the likely pathogenic range. Assessing RAD50 missense variants with distinct functional readouts may help to further elucidate their differential roles in immunodeficiency and cancer and could improve therapeutic strategies. Impact statement RAD50 has a strong impact on DNA repair and cancer therapy. Here, we analyse RAD50 missense variants at four functional levels. Some variants showed an impaired epirubicin response and mitotic progression in the pathological range, while for others these endpoints were separable. Functional heterogeneity of RAD50 variants could contribute to clinical variability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dc8ea9e9e82f5d91c26df1ede413c6f7f658286" target='_blank'>
              RAD50 missense variants differentially affect the DNA damage response and mitotic progression.
              </a>
            </td>
          <td>
            Hanna Redeker, Swantje Kebel, Lea Völkening, Anna Vatselia, Louisa Weinhold, G. Asgedom, Axel Schambach, Detlev Schindler, T. Dörk, Kristine Bousset
          </td>
          <td>2025-10-01</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is the age-related expansion of mutated hematopoietic stem cells without other hematologic abnormalities. In patients with solid tumors, CH is associated with higher mortality and may evolve to therapy-related myeloid neoplasms; however, the mechanisms by which cancer treatments promote CH dynamics remain largely unknown. Here, we analyzed 392 serial samples from a prospective cohort of breast cancer patients and showed that cytotoxic treatments (chemotherapy {+/-} radiation) led to strong therapeutic bottlenecks, resulting in significant reductions in hematopoietic allelic populations and differential clonal selection. CH clones that were positively selected and expanded through dose-dependent therapeutic bottlenecks frequently harbored mutations in TP53, PPM1D, SRCAP, DNMT3A, and YLPM1. Patients with positively selected CH during treatment had the shortest progression-free and overall survival compared to patients with unchanging or negatively selected CH across all therapies. These findings, corroborated in independent breast cancer and pan-cancer cohorts, provide strong evidence for clinical relevance of changes in CH during cancer treatment which may help identify patients at high risk for inferior outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6a553a5184b2aa0287d71bedac04551bf6a5da6" target='_blank'>
              Clonal Hematopoiesis Dynamics and Evolutionary Fitness During Cancer Treatment Impact Clinical Outcomes
              </a>
            </td>
          <td>
            Mona Arabzadeh, Yi-Han Tang, Christelle M. Colin-Leitzinger, S. Marzban, Daniel Walgenbach, Stefania Morganti, Vaidhyanathan Mahaganapathy, Erika Harper, Mingxiang Teng, Jacob Kresovich, Iman Washington, H. Parsons, Judy E. Garber, J. West, Shridar Ganesan, Hossein Khiabanian, Nancy Gillis
          </td>
          <td>2025-09-02</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Double Homeobox 4 (DUX4) is a potent transcription factor encoded by a retrogene mapped in D4Z4 repeated elements on chromosome 4q35. DUX4 has emerged as pivotal in the pathomechanisms of facioscapulohumeral muscular dystrophy (FSHD), a relatively common hereditary muscle wasting condition, although classified as a rare disease. DUX4 contributes to zygote genome activation before its expression is repressed in most somatic tissues through epigenetic mechanisms, including DNA methylation and chromatin modifications. In FSHD, inappropriate activation of DUX4 expression is driven by a complex interplay of genomic and epigenetic alterations. The ectopic presence of DUX4 in skeletal muscle cells activates genes, viral elements and pathways that are typical of very early embryonic development, disturbing cell function and ultimately contributing to muscle weakness and wasting. This review first traces the history of DUX4, from the FSHD genetic linkage studies in the early 1990s, through to identification and characterization of the DUX4 gene in 1999. We then discuss the seminal studies that showed how and why DUX4 is expressed in FSHD and the effects of this ectopic expression in muscle, notably cellular toxicity. Other pathological roles of DUX4, such as participation in cancer and viral infection, are also highlighted. Maintenance of DUX4 in the genome was explained by discovery of the function of DUX4 in zygotic genome activation to institute the totipotent cells of the embryo. Thus, we encompass the gradual transition of DUX4 over the past 25 years from being considered a pseudogene in “junk DNA” to becoming central to understanding the molecular pathogenesis of FSHD and the primary focus for FSHD therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad639b9256d2a222879537cb6ffce2084c7d6dd4" target='_blank'>
              DUX4 at 25: how it emerged from “junk DNA” to become the cause of facioscapulohumeral muscular dystrophy
              </a>
            </td>
          <td>
            Alexandra Belayew, Alberto L. Rosa, Peter S. Zammit
          </td>
          <td>2025-08-25</td>
          <td>Skeletal Muscle</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Transposable elements (TEs), comprising over one-third of the human genome, play a crucial role in its evolution, serving as a significant source of regulatory sequences. Under normal circumstances, their activity is tightly controlled by DNA methylation mechanisms; however, the effectiveness of this suppression varies substantially across tissues. The placenta, characterized by global hypomethylation, represents a unique environment where retroviruses and retrotransposons, typically silenced in somatic cells, gain the opportunity for activation. This distinct epigenetic landscape of the placenta allows transposons to participate in the regulation of genomic activity, influencing processes ranging from early embryogenesis to postnatal development. DNA hypomethylation in the placenta not only promotes TE mobilization, but also opens the possibility of using their components as independent genes and regulatory elements – promoters, enhancers, and other functional modules. These elements are involved in key aspects of placental development, including syncytiotrophoblast formation, extravillous trophoblast invasion, spiral artery remodeling, and endometrial decidualization. Importantly, TEs can serve as sources of alternative promoters for neighboring genes, and ancient mammalian transposons contain multiple transcription factor binding sites, enabling coordinated regulation of genes sharing a common function. Despite the growing interest in the role of transposable elements in placental development and function, many questions remain unanswered. In particular, the mechanisms of non-long terminal repeat (non-LTR) retrotransposon function during pregnancy remain poorly understood. A deep understanding of these processes is necessary to elucidate regulatory disorders in the placenta associated with major obstetric syndromes. This review examines the contribution of transposable elements to the functioning of the human genome, particularly their impact on gene expression, in the context of pregnancy and placental development">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fe4a3e7ac959ca8083b328f8c6eeb28a76f946c" target='_blank'>
              Transposable elements as key regulators of placental development
              </a>
            </td>
          <td>
            M. A. Zhilkina, E. Tolmacheva, S. Vasilyev
          </td>
          <td>2025-09-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract The maintenance of genomic integrity is paramount for normal cell physiology and survival as well as avoidance of carcinogenesis. Cellular DNA is periodically subjected to a myriad of exogenous and endogenous threats and requires constant monitoring to limit genomic instability. To this end, cells possess an intricate DNA damage response and repair (DDR) module comprised of different classes of protein players. The DNA helicases, ATP-dependent enzymes that unwind the DNA double helix, are one such important class of proteins, which act as a linchpin between the recognition and resolution of DNA damage via facilitating various DNA repair processes. Dysfunction or absence of DDR helicase function is implicated in several human disorders including Bloom syndrome, Werner syndrome, Rothmund–Thomson syndrome, and Fanconi anemia. Somatic helicase mutations or dysregulation of helicase function can also contribute to cancer development, progression, and chemotherapy sensitivity, making helicases a promising target class for chemotherapeutic drug intervention. In addition, recent discoveries have identified some DDR helicases in novel synthetic lethal interactions. In this critical review, we will focus on human DNA helicases that are directly or indirectly involved in DDR with special emphasis on their mechanistic actions and clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40deeb85611db34dede498af9b0ad1f2d2a9a398" target='_blank'>
              DNA repair helicases: from mechanistic understanding to therapeutic implications
              </a>
            </td>
          <td>
            Vijay Menon, Susan E. Gueble
          </td>
          <td>2025-10-07</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="BACKGROUND
Therapies for diffuse glioma induce DNA damage response (DDR), and strategies to exploit DDR defects are active areas of investigation. While global DNA methylation profiling effectively classifies gliomas into subtypes, the epigenetic and gene expression patterns of DDR genes, and their contribution to tumor classification and outcomes, have yet to be fully elucidated. Thus, dissecting the DDR epigenetics, gene expression, and single-cell heterogeneity may reveal key molecular characteristics, refine prognosis, and identify novel treatment strategies and resistance mechanisms.


METHODS
We characterized DDR epigenetics and gene expression of TCGA glioblastomas (GBM) and low-grade gliomas (LGG). Single-cell protein analysis by imaging mass cytometry (IMC) was performed on a separate cohort of 118 diffuse gliomas.


RESULTS
Analysis of TCGA cohorts revealed two DDR methylation groups that correlated with IDH mutation status and previously reported molecular groups. DDR transcription profiling further classified tumors into four groups. Those with high DDR transcription across pathways were linked to poor survival independent of IDH or MGMT status, and potentially improved prognostication beyond established biomarkers. Single-cell characterization of a separate cohort revealed intratumoral DDR diversity and identified proliferative tumor cells with high DDR protein expression across pathways that are associated with unfavorable grade and survival.


CONCLUSIONS
Tumor-level epigenetic and transcriptional DDR signatures alone can distinguish molecular-defined diagnosis and outcomes of gliomas beyond established biomarkers. A higher abundance of glioma cells with high DDR effector expression across pathways is associated with poor survival. Thus, clinical assessment of pan-DDR expression may inform prognosis and identify potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0b0fbeaa6fa3878bda82552131f1d944fc87086" target='_blank'>
              DNA damage response profile distinguishes poor-acting gliomas with shared methylome signatures.
              </a>
            </td>
          <td>
            Nalin Leelatian, Charu Singh, Richard Bouffard, R. Sundaram, Kirsten E Diggins, William Sullivan, Sateja Paradkar, Zeynep E. Erson Omay, B. Mobley, Susan E. Gueble, Juan C Vasquez, R. Bindra
          </td>
          <td>2025-08-27</td>
          <td>Neuro-oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d20ba148e7406fb91b956b71267ccb358bcdb11" target='_blank'>
              MNase stratification reveals heterogeneous 5hmC in naive B cells.
              </a>
            </td>
          <td>
            Michael Hsu, Heng-Yi Chen, Fang-Yun Lay, Qin Ma, Ben Delatte, C. J. Lio
          </td>
          <td>2025-08-28</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Genes encoding the RNA-binding proteins FUS, EWSR1, and TAF15 (FET proteins) are involved in chromosomal translocations in rare sarcomas. The encoded fusion oncoproteins typically include a DNA-binding domain and function as oncogenic transcription factors. FET-rearranged sarcomas are often aggressive malignancies affecting patients of all ages. Oncogenic fusion transcription factors are challenging to target directly, and new therapies are needed. Treatment with the small molecule SP-2509 results in reversal of the transcriptional activity of the FET fusion that causes Ewing sarcoma, EWSR1::FLI1. A similar molecule, seclidemstat (SP-2577), is currently in clinical trials for FET-rearranged sarcomas (NCT03600649), but its pharmacologic activity in FET fusion–positive sarcomas has not been demonstrated. We found potent seclidemstat cytotoxicity against both FET-rearranged and other fusion-positive sarcoma cell lines in vitro, including Ewing sarcoma, desmoplastic small round cell tumor, clear cell sarcoma, myxoid liposarcoma, and fusion-positive rhabdomyosarcoma. We also define transcriptomic effects of seclidemstat with bulk RNA sequencing. Seclidemstat recapitulated much of SP-2509 transcriptional activity in Ewing sarcoma. Widespread transcriptional changes were seen in all tested cell lines after seclidemstat treatment, regardless of the fusion protein expressed. This included reversal of FET fusion transcriptional signatures for EWSR1::WT1, EWSR1::ATF1, and EWSR1::ERG. Though novel inhibitors are unlikely to display single-agent efficacy in the clinic, these data suggest that seclidemstat remains a promising new treatment strategy for patients with FET-rearranged and other fusion-positive sarcomas. Significance: In this study, we show seclidemstat has in vitro activity in multiple rare and aggressive sarcomas caused by FET fusion proteins. With 13 RNA sequencing experiments, including multiple FET-rearranged sarcoma cell lines, this dataset is a rich resource for those studying FET-rearranged sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fb78384576a9a5b2487874950835b687b754a22" target='_blank'>
              Seclidemstat (SP-2577) Induces Transcriptomic Reprogramming and Cytotoxicity in Multiple Fusion–Positive Sarcomas
              </a>
            </td>
          <td>
            G. Rask, C. Taslim, A. Bayanjargal, Rachel D. Dreher, Matthew V. Cannon, Julia Selich-Anderson, Jesse C. Crow, Aundrietta Duncan, Emily R. Theisen
          </td>
          <td>2025-08-25</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Long noncoding RNAs (lncRNAs) are emerging as important regulators of gene expression in both normal biological systems and disease. Among them, metastasis-associated lung adenocarcinoma transcript 1 (MALAT1) has attracted considerable attention due to its high nuclear expression and evolutionary conservation. It was first identified as a biomarker for lung cancer metastasis, but since then, its involvement has been reported in a wide range of cancers. MALAT1 influences several key cancer-related processes, including cell proliferation, migration, angiogenesis, apoptosis, epithelial–mesenchymal transition, and the behavior of cancer stem cells. In this review, we take a closer look at how MALAT1 functions through different mechanisms, such as regulating RNA splicing, altering chromatin states, acting as a sponge for microRNAs, or modifying protein interactions, to drive these changes. We also discuss its growing role in shaping the immune landscape of tumors, particularly its involvement in immune evasion, inflammatory signaling, and regulation of immunogenic cell death. In addition, MALAT1 contributes to resistance against therapy, rewiring of cellular metabolism, and communication between tumor and stromal cells via exosomes. While many studies describe MALAT1 as an oncogene, others suggest it may also play a suppressive role depending on the cancer type and biological context. Here, we aim to offer a comprehensive overview of MALAT1’s diverse actions and evaluate its potential as a diagnostic marker and therapeutic target in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73cb56fb6ad82b5646d4f05c255a6ee44614bc12" target='_blank'>
              MALAT1 as a molecular driver of tumor progression, immune evasion, and resistance to therapy
              </a>
            </td>
          <td>
            A. Bitaraf, Alireza Zafarani, Pardis Jahandideh, Benyamin Hakak-Zargar, A. Haghi, Golareh Asgaritarghi, S. Babashah
          </td>
          <td>2025-10-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f43327ba689a502fd2aba63a16648713112594" target='_blank'>
              Epigenomic Heterogeneity of Non-Functional Pancreatic Neuroendocrine Tumors Uncovered by Single nucleus and Spatial ATAC Profiling.
              </a>
            </td>
          <td>
            Dejiang Wang, Xiangjun Di, Fu Gao, Gaocai Li, Liyun Lin, Shujie He, Di Zhang, Junyi Yu Jin, Yuanxin Liang, Michael Cecchini, Jill Lacy, John Kunstman, Pamela Kunz, Jing Du, Yang Liu
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Gastric cancer (GC) is the fifth-leading cause of cancer-related mortality, with a 5-year survival rate less than 20%. It develops from preneoplastic lesions to adenocarcinoma, but these early genetic alterations remain poorly understood. Therefore, we aimed to identify early genetic drivers underlying the development of preneoplastic lesions and the initiation of gastric carcinogenesis. Methods We characterized preneoplastic and early gastric adenocarcinoma using 48 samples from 16 Guatemalan patients, a country with a high incidence of GC. We sequenced a panel of 127 genes to identify early genetic drivers and possible actionable targets. Results We identified extensive genetic heterogeneity, including single nucleotide and copy number variations. After comparing our data with other studies, we identified TP53 and APC as the most mutated genes in preneoplastic lesions and early GC. Our mean tumor mutational burden was higher in diffuse (0.017 mutations/Mb) and intestinal adenocarcinomas (0.015) than in chronic gastritis (0.005), and analysis of the mutational signatures revealed several processes acting at different stages of the disease. Signatures S15 (DNA mismatch repair deficiency) and S03 (homologous recombination deficiency) were more frequent in early adenocarcinoma than in chronic gastritis, intestinal metaplasia, necrosis, tubular adenoma, and atrophy. Notably, 10 of 16 patients (62.5%) had at least one actionable mutation in their preneoplastic lesions or gastric adenocarcinomas. Conclusions We show that at the preneoplastic and earliest stages, GC is genetically heterogeneous and presents key cancer-driving mutations that may participate in neoplastic transformation and progression, with 62.5% of lesions having the potential for treatment. This study expands the limited research on early GC and highlights key opportunities for precision medicine in populations with high GC incidence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2f131a17bcf2decc26bf9525e923b46c1c5c1bd" target='_blank'>
              Genetic heterogeneity and key driver mutations in the preneoplastic and earliest stages of gastric cancer
              </a>
            </td>
          <td>
            Eduardo Alfonso Marroquín-Estrada, Héctor Martínez-Gregorio, C. Díaz-Velásquez, Fernando Ambriz-Barrera, Eduardo Emilio Córdoba-García, Laura Itzel Hernández-Romero, Miguel Ruiz De-La-Cruz, Armando Cáceres, , José Rodolfo Gil, Elizabeth Orellana, Daniel Penados, Carolina Richter, Marcelino Díaz, Felipe Vaca-Paniagua
          </td>
          <td>2025-10-09</td>
          <td>Gastroenterology Report</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="DNA-binding molecules regulate gene expression, replication, repair, and transcription, making their research crucial to molecular biology. This review investigated their architectures, processes, and impacts on cancer research and therapy to determine their biological and therapeutic potential. A thorough examination of published data on DNA-binding agents, including intercalators, groove binders, and metal complexes, focused on their chemical properties, biological activity, and therapeutic significance. Doxorubicin intercalated between base pairs to inhibit replication and transcription, whereas cisplatin produced covalent cross-links with guanine bases to cause tumor cell death. Multiple derivatives of metal complexes reduced tumor development by over 70% in leukemia models as DNA probes and therapeutics. Also reviewed, Dps proteins in *Escherichia coli* showed that their non-specific DNA binding offered up to 65% oxidative stress resistance compared to control cultures, validating DNA-protein protection as a survival strategy. In Phase II studies, amsacrine caused substantial remission in acute leukemia patients, whereas doxorubicin was more effective across many cancer types but had greater cardiotoxicity concerns. Selectivity, toxicity, and resistance limit DNA-targeting medicines, although they are successful. Chemical modifications like hydrophobic tailoring and sequence-specific binding have enhanced binding affinity and therapeutic index, yet only 10% of candidate compounds get clinical approval. Recent studies show that AI-driven design has expedited screening, lowering development costs by 30% and durations by 3–5 years. These data suggest that DNA-binding medicines have great potential in cancer, but safer, more selective, and resistance-free therapy are still needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bfaa0d838d07a9aad5a057a0a95be735af4700a" target='_blank'>
              Chemistry of DNA-binding Molecules
              </a>
            </td>
          <td>
            Evon Akram, Dina A. Najeeb, Asmaa A. Jawad, N. Bedair, A. M. Hussein, Saba R. Jaafar, Ruaa H. Ali, R. F. Shaher, M. A. Hussein, Daniah M. Hamid, Reem H. Al-Tabra, Alyaa K. Abood, Salam Mohammed
          </td>
          <td>2025-08-30</td>
          <td>Al-Salam Journal for  Medical Science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Gastric cancer (GC) remains one of the leading causes of cancer-related mortality worldwide due to its aggressive nature and high metastatic potential. Circular RNAs (circRNAs), characterized by covalently closed loop structures and high molecular stability, have emerged as important regulatory molecules in cancer biology. However, the biological functions and molecular mechanisms of most circRNAs in GC remain largely unexplored. Methods We conducted circRNA microarray profiling on three pairs of primary GC tissues and matched adjacent normal tissues to identify differentially expressed circRNAs. The expression of circ-ANKRD36C was validated by quantitative reverse transcription polymerase chain reaction (qRT-PCR) in additional clinical specimens. Functional assays were performed to assess the effects of circ-ANKRD36C knockdown on GC cell proliferation, migration, and invasion. Dual-luciferase reporter assays provided evidence of direct interaction between circ-ANKRD36C and miR-15a-5p. Clinicopathological correlations and survival outcomes were analyzed in 84 paired GC and normal tissue samples. Results Circ-ANKRD36C was significantly upregulated in GC tissues. Knockdown of circ-ANKRD36C suppressed GC cell proliferation, migration, and invasion in vitro. Dual-luciferase assays supported its direct interaction with miR-15a-5p, potentially influencing oncogenic targets such as BMI1 and CCND1. Clinically, high circ-ANKRD36C expression was associated with shorter overall survival (OS). Multivariate Cox regression analysis suggested its independent prognostic value (hazard ratio [HR] = 2.41; 95% confidence interval [CI] = 1.22–4.77; P = 0.011). Conclusion Circ-ANKRD36C is upregulated in gastric cancer and may serve as a prognostic biomarker. Functional evidence suggests it could act as a candidate regulator, but its mechanistic role requires further validation. Supplementary Information The online version contains supplementary material available at 10.1186/s12876-025-04287-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aec96a53ffb97eb8a42c6e93ec17c0d2fe31bed9" target='_blank'>
              Circular RNA profiling and functional analysis implicate circ-ANKRD36C as a potential prognostic biomarker in gastric cancer
              </a>
            </td>
          <td>
            Rongpeng Chen, Tianxing Ji, Yuanqing Chu, Feng Xiao, Zhicheng Feng, Guoqiang Wang
          </td>
          <td>2025-09-29</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Objective: Prostate cancer, a leading cause of cancer deaths in men, involves genetic alterations and DNA damage. Oxidative stress–induced DNA damage is crucial in its development, necessitating efficient DNA repair mechanisms. Base excision repair (BER), a key pathway, addresses oxidative DNA damage with enzymes like Apurinic/Apyrimidinic Endonuclease 1 (APE1). Recent findings show that altered APE1 expression disrupts BER balance, contributing to genomic instability and cancer progression. This study examines APE1 expression in prostate cancer tissues to clarify its role in cancer development and therapeutic implications. Assessing APE1 expression in prostate cancer may elucidate its role in tumor progression, therapeutic resistance, and genomic instability, offering deeper insights into the mechanistic relevance of the BER pathway in oncogenesis. Methods: A total of 50 male patients diagnosed with prostate cancer were included in this study. Tumor and adjacent normal tissue samples were collected from consecutive male patients who underwent Tru-cut prostate biopsies due to clinical suspicion of prostate cancer, and gene expression levels were analyzed using real-time polymerase chain reaction. Results: The results showed no significant difference in APE1 expression between tumor and normal tissues (P > .05). Clinical factors like smoking, age, diabetes, hypertension, PIRADS, and Gleason scores did not significantly correlate with APE1 expression. However, a statistically significant correlation was observed between PSA levels and APE1 expression (r: 0.287, P = .04). Conclusion: The authors’ findings suggest that while no substantial changes in APE1 expression were observed in prostate cancer tissues, the enzyme’s role in DNA repair remains crucial. Abnormalities in APE1 expression and function can contribute to genomic instability and cancer progression. Future studies with larger sample sizes and standardized methodologies are necessary to better understand the potential role of APE1 as a biomarker and therapeutic target in prostate cancer.Cite this article as: Rouhi V, Habibi S, Baykal Koca S, Köseoğlu H, Güven M. Investigation of apurinic/apyrimidinic endonuclease 1 expression changes in prostate cancer. Cerrahpaşa Med J 2025, 49, 0009, doi: 10.5152/cjm.2025.25009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f5a4b0fb3242b36fc3de3889e7200ec9f3f9fad" target='_blank'>
              Investigation of Apurinic/Apyrimidinic Endonuclease 1 Expression Changes in Prostate Cancer
              </a>
            </td>
          <td>
            Vahid Rouhi, Sanaz Habibi, Sevim Baykal Koca, Hikmet Köseoğlu, Mehmet Güven
          </td>
          <td>2025-08-29</td>
          <td>Cerrahpaşa Medical Journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 The complex heterogeneity of Acute lymphoblastic leukemia (ALL) often predicts poor prognosis, high morbidity, and drug resistance. Yet, the molecular drivers that initiate aggressive ALL have yet to be fully elucidated, especially in the context of the >30 subtypes of human T- and B-ALL. Here, we used a large-scale F0 transgenic screen in zebrafish to identify synergistic combinations of 65 putative oncogenes at inducing both T- and B-cell leukemia. This approach identified a new proto-oncogene SET that collaborates with both notch1a icn and mutationally activated Interleukin 7 Receptor (IL7R) to initiate aggressive leukemia in vivo. SET is a multifunctional protein that can act as a transcriptional regulator, a histone chaperone, and an inhibitor of protein phosphatase 2A (PP2A). The high expression of SET along with mutational activation of NOTCH1 and IL7R was also found in human B- and T-ALL, supporting their oncogenic roles in leukemogenesis. Our discovery that SET is capable of independently synergizing with known drivers to initiate ALL presents a unique opportunity to explore the nature of oncogene driven ALL and define the mechanisms by which it drives transformation. We next used Non-negative Matrix Factorization to unbiasedly identify six gene programs shared across the zebrafish ALL subtypes and assessed if each was predictive of outcome in human disease. One program comprised a MYC-driven transcription signature that unexpectedly predicted good outcomes in human T-ALL patients. Additional conserved programs independently stratified patients into poor outcomes and when combined with the MYC signature profile were able to better discern disease outcomes in patients, including the aggressive ETP-like and TAL1 DP-like T-ALL. Taken together, we have uncovered new roles for SET in driving ALL initiation, uncovered unexpected high MYC expression as predictive of good outcome, and defined a new combination biomarker gene signature that predicts poor overall survival in aggressive subtypes of human T-ALL. Our zebrafish-based screening approach is a powerful tool for comparative genomic studies to identify new oncogenic drivers, vulnerability pathways, and new biomarkers of aggression in ALL.



 James R Allen, Luis Antonio Corchete Sanchez, Mohamed N Bakr, Miriam Fernández-Lajarín, Alexandra Hazelwood, Nathan Ford, Anna M Lucianò, Alexandra Veloso, Alexander D Weissman, Olivia A Strom, Esther Rheinbay, David M Langenau. Zebrafish modeling predicts clinical outcomes in human acute lymphoblastic leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr B035.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79aacf42881085b0060797945bc5360d7773e354" target='_blank'>
              Abstract B035: Zebrafish modeling predicts clinical outcomes in human acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            J. R. Allen, L. A. Corchete Sanchez, M. N. Bakr, Miriam Fernández-Lajarín, Alexandra Hazelwood, Nathan Ford, Anna M Lucianò, Alexandra Veloso, Alexander D Weissman, Olivia A Strom, Esther Rheinbay, David M Langenau
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chromosomal instability (CIN) and epigenetic reprogramming are central drivers of breast cancer progression, yet the mechanisms connecting them remain elusive. We uncover a direct role of EZH2 histone methyltransferase in promoting CIN in triple-negative breast cancer (TNBC). Across breast cancers, EZH2 expression correlates with copy number alterations, and its catalytic activity is associated with increased CIN in metastasis-initiating cells. Pharmacological EZH2 inhibition suppressed CIN, revealing an unexpected vulnerability. Integrated chromatin and transcriptome profiling identified Tankyrase (TNKS), a poly(ADP-ribose) polymerase, as a direct transcriptional target of EZH2. Mechanistically, EZH2-mediated TNKS suppression disrupts CPAP (centrosomal P4.1-associated protein), driving centrosome overduplication, multipolar mitosis and exacerbated CIN. In vivo, CIN suppression is a critical mechanism underlying the anti-metastatic effects of EZH2 inhibition. These findings delineate a previously unrecognized epigenetic mechanism governing CIN and establish EZH2 inhibitors as the first therapeutic agents capable of directly suppressing CIN, underscoring the need for trials with metastasis-focused endpoints.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5010199d8cc937113c87be8a59e3c3e7e0efc720" target='_blank'>
              Epigenetic regulation of chromosomal instability by EZH2 methyltransferase.
              </a>
            </td>
          <td>
            Yang Bai, Albert S. Agustinus, Shira Yomtoubian, Cem Meydan, Dylan R McNally, Liron Yoffe, Melissa J Hubisz, Marvel Tranquille, Sneha Pramod, Christy Hong, Magdalena L Plasilova, Aakanksha R Kapoor, Arshdeep Singh, Henry G. Withers, Lukas E. Dow, A. Laughney, B. Bhinder, O. Elemento, Ari M Melnick, S. Bakhoum, Vivek Mittal
          </td>
          <td>2025-10-02</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="


 Epigenetic dysregulation drives many pediatric tumors, including neuroblastoma, an aggressive malignancy that accounts for approximately 15% of childhood cancer deaths. Drugs targeting epigenetic regulators have emerged, with FDA approvals of tazemetostat for patients with metastatic or locally advanced epithelioid sarcoma with incomplete resection and for patients with relapsed or refractory follicular lymphoma with EZH2 mutations. More recently, the menin inhibitor revumenib was FDA approved for children and adults with relapsed/refractory KMT2A-rearranged acute myeloid leukemia. Unfortunately, these drugs are generally not curative, and resistance develops by multiple mechanisms suggesting that combination therapies are needed. Our laboratory recently identified some neuroblastomas, particularly MYCN amplified, that are dependent on the SAGA complex, a transcriptional coactivator complex that regulates gene expression by modifying histones. Knockout of TADA2B, a gene essential to the histone acetyltransferase function of SAGA, or treatment with GSK699, a selective degrader of KAT2A and KAT2B, the canonical catalytic acetyltransferases of SAGA, causes cell cycle arrest and delayed progression in xenograft models of neuroblastoma. In order to nominate combination strategies with SAGA inhibitors, we used orthogonal approaches: DepMap nearest neighbor analysis and a CRISPR sensitizer screen. Utilizing the Broad Institute’s Cancer Dependency Map (DepMap), we identified EZH2, a core member of the PRC2 complex, as a co-dependency with the SAGA complex in neuroblastoma. Furthermore, we conducted a CRISPR/Cas9 screen with a focused sgRNA library (1,350 genes) to identify additional epigenetic sensitizers and drivers of resistance in neuroblastoma cells treated with GSK699. EZH2 knockout scored as sensitizing to GSK699 while knockout of BRD2, a member of the BET family of proteins and an epigenetic reader, scored as rendering resistance to GSK699. The utilization of both in vitro and preclinical studies confirmed that pharmacological inhibition of KAT2A/B using GSK699 and EZH2 using tazemetostat induced a combinatorial inhibitory effect on neuroblastoma tumor proliferation compared to single agent treatment, with marked prolongation of survival in xenograft models with combination therapy. Transcriptional analysis of tumors treated with combination therapy showed increases in epithelial to mesenchymal transition (EMT) programs and myogenesis differentiation pathways, a surprising finding under active investigation. In conclusion, our data demonstrates that inhibition of the SAGA complex in combination with EZH2 inhibition effectively suppresses the proliferation of MYCN-amplified neuroblastoma supporting further evaluation of this combination in models of neuroblastoma, including deeper mechanistic studies.



 Melinda Soeung, Kenneth Ross, Lucy Merickel, Silvi Salhotra, Audrey Taillon, Allen Basanthakumar, Kimberly Stegmaier. Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A043.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5216fb3aee2d0a8a7d9f8e5c04d43544aa31dd4b" target='_blank'>
              Abstract A043: Combination Therapeutic Targeting of Chromatin Complexes in Neuroblastoma
              </a>
            </td>
          <td>
            Melinda Soeung, Kenneth Ross, L. Merickel, S. Salhotra, A. Taillon, A. Basanthakumar, K. Stegmaier
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="CENP-C, an essential component of the kinetochore, connects centromeric chromatin to the outer kinetochore, and thereby ensures accurate chromosome segregation. Although deletion of the Mis12-binding domain (M12BD) of CENP-C does not cause developmental disorders in mice, it promotes malignant tumor progression in the two-stage DMBA/TPA-induced skin carcinogenesis model. In this study, we have demonstrated that M12BD deletion of CENP-C enhances proliferation and then promotes abnormal differentiation in DMBA/TPA-induced carcinomas in mice. To elucidate the underlying molecular mechanisms, we performed RNA sequencing and found the dysregulated expression of keratinization-related genes. Intriguingly, elevated chromosomal aneuploidy was detectable in mice with the M12BD deletion of CENP-C. Among the aneuploidies, trisomies of chromosomes 6 and 10 took place at the highest frequency. These specific chromosomal gains were accompanied by upregulation of genes involved in immune and inflammatory responses. Together, our present findings strongly suggest that M12BD of CENP-C plays a critical role in the regulation of epithelial differentiation during tumor development in mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1c39fcf56ce4b3277a5d1391cbbde0aec7e6c90" target='_blank'>
              Deletion of the Mis12C-Binding Domain of CENP-C Promotes Chromosomal Aneuploidy in Cutaneous Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Megumi Saito, Kazuhiro Okumura, Yurika Tokunaga, Sora Tanaka, Keisuke Otoyama, Yoshinori Hasegawa, Masatoshi Hara, Masakazu Hashimoto, Toshihiko Fujimori, Tatsuo Fukagawa, Yuichi Wakabayashi
          </td>
          <td>2025-10-03</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 High-grade serous ovarian cancer (HGSOC) remains the most lethal gynecologic malignancy, with a 5-year survival rate below 50%. Despite initial responsiveness to platinum-based chemotherapy and the implementation of targeted therapies such as PARP inhibitors, the majority of patients relapse and develop treatment-resistant disease. Emerging evidence suggests that the spatial organization and dynamic interactions among tumor, immune, and stromal cells within the tumor microenvironment (TME) play a critical role in therapeutic response and disease progression. However, traditional transcriptomic approaches lack the spatial context necessary to fully understand these interactions. The Xenium spatial transcriptomics (ST) platform represents an innovative technology that enables high-resolution, in situ mapping of gene expression within intact tissue architecture. By preserving the spatial relationships between cell types and molecular states, ST provides the opportunity to explore how the TME evolves under therapeutic pressure and contributes to resistance. We performed spatial transcriptomic profiling on FFPE tumor sections from three matched longitudinal pairs of HGSOC samples obtained prior to treatment and upon PARPi progression. Utilizing the Xenium 5K Plex platform from 10X Genomics, we quantified the expression of 5000 genes in over 2.5 million cells. Cell type annotation was performed via reference mapping and marker gene expression. Differential expression and gene ontology enrichment analyses were conducted to identify key transcriptional changes in tumor clusters between paired naïve and progression samples. We furthermore describe spatial cellular neighborhoods and their changes upon development of resistance. Characterization of the tumors reveals great morphological heterogeneity between patients. Comparative analysis revealed distinct shifts in the cancer cell abundance, cancer cell area and spatial arrangement of the TME between naïve and progression samples. Progression samples exhibited increased expression of, for example, hypoxia-related genes, particularly in cancer cells. Spatial mapping identified distinct cellular neighborhoods, including niches with a high density of neutrophil- or macrophage-associated transcripts, which were notably more abundant in progression samples, suggesting a role in immune evasion and tumor progression. This study demonstrates the potential of spatial transcriptomics to elucidate the architectural and transcriptional changes of the ovarian cancer microenvironment at the single-cell level. While based on early findings, our results suggest the relevance of hypoxia, stromal remodeling, and niches with distinct cellular compositions in contributing to disease progression. These data support further investigation into spatially targeted therapeutic strategies and the use of ST as a tool for microenvironment-driven biomarker discovery.



 Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera. Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr A003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cf6f443298e3305bcbf89470d2e118a90600ac1" target='_blank'>
              Abstract A003: Mapping the tumor microenvironment: Spatial Transcriptomic insights into ovarian cancer progression and resistance
              </a>
            </td>
          <td>
            Sara K. Bystrup, Kathleen J. Imbach, Sergi Cervilla García, Arola Fortian Bernabeu, Daniella Grases, Eduard Porta Pardo, Jordi Barretina, Lorena Valdivieso Almeida, Pau Guillén Sentís, Adrià Bernat Peguera
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The interplay between nuclear architecture and extracellular matrix stiffness orchestrates cell fate decisions, yet the molecular mechanisms remain poorly defined. Here, we investigate the role of Lamin A (LMNA), a nuclear structural protein whose expression correlates with tissue stiffness, in regulating cellular differentiation and fate decision. Using myoblasts and fibroblasts as models, it was observed that cells with low LMNA expression showed that higher cell deformation elevated expression of neurological genes and exhibited potential for differentiation into a neural-like fate. CUT&Tag sequencing of LMNA-knockdown cells revealed a reduction in the size of Lamin B1-associated domains, with enhanced Lamin B1 binding at muscle-related genes (Myf5 and Myf6) and diminished binding at the neural gene Nes, suggesting that changes in gene expression are associated with alterations in chromatin structure. Further analysis identified the dissolution of H3K9me2/3-labeled heterochromatin regions and their redistribution in the nucleoplasm following LMNA inhibition. Soft substrates (0.2 kPa) amplify the neural differentiation capacity in LMNA-knockout cells. Additionally, retinoic acid was shown to enhance the expression of neurologically related genes by suppressing LMNA expression. These findings reveal a novel substrate stiffness-induced mechanism by which Lamin A regulates cell fate transitions and provide a new approach for neural cell generation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/880b739a355b443815a774178c98532930794235" target='_blank'>
              Decreasing Lamin A Triggers Cell Fate Transitions through Heterochromatin-Nuclear Periphery Detethering
              </a>
            </td>
          <td>
            Lijuan Sun, Yafan Xie, Zhaoyan Zuo, Jian Liu, Jiaqi Yang, Iqra Ali, Qin Peng, Juhui Qiu
          </td>
          <td>2025-08-28</td>
          <td>Biomaterials Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Somatic cell nuclear transfer (SCNT) holds promise for animal cloning but remains limited by low efficiency and phenotypic abnormalities, often attributed to incomplete nuclear reprogramming. This study presents an integrative genomic and epigenomic analysis of cloned buffaloes and their respective donors using long-read Oxford Nanopore sequencing. Our results showed a high degree of genomic similarity between clones and donors, with most variations located in non-coding regions and structural variants (SV) distributions highly correlated at the chromosomal level. Gene and protein level overlap of SV-affected loci revealed 70.9–73.3% gene-level and 69.7–72.5% protein-level similarity. Despite this genetic similarity, DNA methylation analysis identified differentially methylated regions (DMRs), particularly in intergenic and promoter regions. Clones exhibited slightly lower CpG methylation than the donors. The DMRs in donor vs. clone comparisons indicated higher hypomethylated regions than hypermethylated regions. Functional enrichment of DMR-associated genes highlighted pathways linked to mitochondrial function, oxidative phosphorylation, and reproductive processes. Although clones showed moderate genome-wide methylation correlation with donors, key differences in methylation suggest incomplete epigenetic reprogramming. Despite these epigenetic differences, all clones were phenotypically normal and healthy into adulthood. This study offers the first comprehensive SV and methylome profile of SCNT-derived buffaloes and emphasizes the role of epigenetic mechanisms in clone development and health, providing valuable insights to enhance cloning efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ed0c9146dce33964d38cc27a7e6aa2903443aa1" target='_blank'>
              Unravelling High Nuclear Genomic Similarity and Mitochondria Linked Epigenetic Divergence in SCNT Derived Buffalo Clones via Long-Read Nanopore Genome Sequencing
              </a>
            </td>
          <td>
            Meeti Punetha, Dharmendra Kumar, Satish Kumar, Bhavya Maggo, Priya Dahiya, Pradeep Kumar, Rakesh K. Sharma, Yash Pal, P. S. Yadav
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract DNA‐binding proteins play essential roles in DNA replication, DNA repair, DNA organization, and several aspects of gene regulation. Their well‐studied structures and charge configurations aid in identifying similar functions in other proteins. Vitellogenin (Vg) is a highly conserved protein that is central to egg‐yolk formation in most animal taxa. The protein is largely viewed as a transporter, but effects on immunity and behavior are documented. Experiments in honey bees (Apis mellifera) additionally suggest a role in gene regulation. The possibility of Vg‐DNA binding has broad relevance due to Vg being phylogenetically widespread and having descendant proteins relevant to human cardiovascular health. Previously, we found that a Vg subunit can translocate to the honey bee nucleus and interact with DNA. Now, we provide a structural explanation for Vg's DNA‐binding potential by identifying conserved DNA‐binding amino acids in structural regions similar to established DNA‐binding proteins. Next, we examined how Vg‐DNA binding may elicit gene expression changes in honey bee workers, characterized by distinct changes in Vg levels over their lifetimes. Finally, we identify other nuclear proteins likely bound to the Vg‐DNA complex in honey bees. Our data suggest that Vg‐DNA binding is associated with expression changes in dozens of genes and that the Vg‐DNA complex interacts with dozens more nuclear proteins. We propose that Vg‐DNA binding can regulate several important processes in honey bee workers, including energy metabolism, behavior, and signaling. Due to the conserved nature of Vg and its descendant proteins, these functions may be present in various animals, including humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d27c066f500072bfda90d4a95c0ad8b1564af213" target='_blank'>
              Evidence for vitellogenin DNA‐binding in honey bees
              </a>
            </td>
          <td>
            Gyan Harwood, Vilde Leipart, Chris Elsik, Joseph C F Ng, F. Drabløs, G. Amdam
          </td>
          <td>2025-09-13</td>
          <td>Protein Science : A Publication of the Protein Society</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f396f45a05f69851b7610dc8198d4d5a27e35207" target='_blank'>
              Characterization of cis-regulatory elements and functional variants in colorectal cancer using epigenomics and CRISPRi screenings.
              </a>
            </td>
          <td>
            Zequn Lu, Can Chen, Heng Zhang, Bin Li, Yizhuo Liu, Jiayi Guo, Runying Xu, Ke Shi, Qianying Ma, Ming Zhang, Yimin Cai, Jinyu Huang, Hui Geng, Linyun Fan, Caibo Ning, Yanmin Li, Shuo Chen, W. Tian, Kexin Hu, Haijie Li, Xiaojun Yang, Chaoqun Huang, Yongchang Wei, Xu Zhu, Xiangpan Li, Zhen Xiong, M. Cai, Xiaoyang Wang, Shaokai Zhang, Hongda Chen, M. Dai, Kun Chen, Mingjuan Jin, Meng Jin, Ying Zhu, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2025-08-25</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The tumour suppressor protein p53 plays a central role in safeguarding genomic integrity through the regulation of DNA repair, cell cycle arrest, and apoptosis. Mutations in TP53, particularly within its DNA-binding domain, are among the most frequent genetic alterations in human cancers and are strongly associated with chemoresistance and poor prognosis. In this study, all TP53 mutations reported in the COSMIC database were systematically mapped onto all experimentally resolved TP53 three-dimensional structures available in the Protein Data Bank, supplemented with AlphaFold-predicted models to achieve full structural coverage. Mutations were classified according to their structural context—protein core, interface regions, ligand- and zinc-binding sites, and intrinsically disordered regions—and evaluated using complementary sequence- and structure-based predictive tools. The analysis revealed distinct mutational hotspots, differential distribution across structural regions, and context-dependent effects on stability and DNA-binding capacity. Notably, a subset of mutations exhibited consistent predictions of high destabilisation across all structural contexts, underscoring their potential as drivers of functional inactivation. By providing a comprehensive structural map of TP53 alterations, this work offers a valuable resource for understanding mutation-specific mechanisms of p53 dysfunction and for guiding the development of precision therapeutic strategies aimed at restoring its tumour-suppressive functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85f3fc7b1e2b5e98df635979848fcd53d6ac30ab" target='_blank'>
              Mutational Disruption of TP53: A Structural Approach to Understanding Chemoresistance
              </a>
            </td>
          <td>
            Ali F. Alsulami
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Deoxyribonucleic acid (DNA) serves as the fundamental blueprint for life, governing both the structural development and functional regulation of living organisms. The technology explores the modern understanding of DNA role in heredity and molecular genetics that highlighted key discoveries that have shaped the field. The discovery of intricate molecular mechanisms such as epigenetic modifications, non-coding RNA and the gene environment interactions has evolved beyond Mendelian genetics. The mechanism of genetic inheritance, replication and expression illustrates how traits are passed from one generation to the next generation. This review article highlights how mutation and variations in DNA sequences influence health, evolution and disease susceptibility. A significant focus is placed on revolutionary CRISPR-Cas9 Jain editing technology which has transformed genetic research and holds immense promise for future therapeutic applications. After enabling precise modifications in the genome, CRISPR opens new avenues for the treatment of genetic disorders and advancement in biotechnology. This review emphasizes how the integration of classical molecular genetics with cutting edge gene editing techniques offers a comprehensive understanding of DNA function for addressing agricultural and global health challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd606e61309a0083af92e77dc5cd4dc92727de21" target='_blank'>
              MODERN INSIGHTS OF DNA FUNCTION AND HEREDITY: A REVIEW OF MOLECULAR GENETICS AND CRISPR GENE
              </a>
            </td>
          <td>
            Mariam Fatima, Mariam Fatima¹, Muhammad Haseeb, Anwar Tarar¹, Arooba Maryam², Muhammad Azhar, Ud Din³˒⁴, Ifra Muzaffar¹, H.M. Saif, UR Rahman⁵, Rida Sattar⁶
          </td>
          <td>2025-08-21</td>
          <td>Journal of Medical &amp; Health Sciences Review</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract DNA methylation is altered in all cancers, but the mechanisms responsible for these changes are not well understood. Using data from 100 primary samples, we show that regions with intermediate methylation in normal hematopoietic cells are hotspots for stable, clonal epimutations in acute myeloid leukemia. Analysis of hematopoietic stem cell clones demonstrated that intermediate methylation at epimutation hotspots reflects random allele-specific methylation and expression at the single cell level, representing a previously unrecognized form of somatically acquired imprinting. We identified somatically imprinted regions in other tissues and show they are epimutation hotspots in other cancer types. This demonstrates that random allele-specific methylation is both a general property of normal cells, and a vulnerability that renders them susceptible to cooperating events and clonal selection during cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e7fa3ac12fa0ce8e4468a1db8da8c5facb616d2" target='_blank'>
              Intermediately Methylated Regions in Normal Cells Are Epimutation Hotspots in Cancer
              </a>
            </td>
          <td>
            Mohamed Mahgoub, Haley Abel, Nidhi Davarapalli, Heidi Struthers, E. Kotnik, Brittany Johnson, Christopher Markovic, C. Fronick, Robert Fulton, Michael P. Meers, David H. Spencer
          </td>
          <td>2025-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Tumor heterogeneity, a hallmark of cancer, frequently leads to treatment failure and relapse. However, the intricate communication between various cell types within the tumor microenvironment and their roles in tumor progression in vivo remain poorly understood. Here we establish a novel tumor heterogeneity model in the Drosophila larval eye disc epithelium and dissect the in vivo mechanisms by combining sophisticated genetics with single-cell RNA sequencing. We found that mutation of the tricellular junction protein M6 in cells surrounding RasV12 benign tumors promotes their malignant transformation. Mechanistically, early RasV12//M6−/− tumors secrete Pvf1, which activates the Pvr receptor on hemocytes, facilitating their recruitment to the tumor site. These tumor-associated hemocytes secrete the Spätzle (Spz) ligand to activate the Toll receptor within the RasV12 tumors. This enhanced activation of the Toll pathway synergizes with RasV12 to promote malignant transformation through the JNK-Hippo signaling cascade. In summary, our study elucidates the complex interplay between genetically distinct oncogenic cells and between tumors and hemocytes, highlighting how hemocytes exploit the ancient innate immune system to coordinate tumor heterogeneity and drive tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d320162041d03e1b05ee82602e945a36e317508" target='_blank'>
              Hemocytes facilitate interclonal cooperation-induced tumor malignancy by hijacking the innate immune system in Drosophila
              </a>
            </td>
          <td>
            Sihua Zhao, Yifan Guo, Xiaoyu Kuang, Xiaoqin Li, Chenxi Wu, Peng Lin, Qi Xie, Zongzhao Zhai, D. Kong, Xianjue Ma
          </td>
          <td>2025-08-22</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="BACKGROUND
Multidrug resistance remains a major obstacle in the treatment of ovarian cancer (OC) patients. Recent research has underscored the critical role of extrachromosomal circular DNA (eccDNA) in tumor initiation and progression. However, there is limited comprehensive understanding of the role eccDNA plays in tumor resistance.


OBJECTIVES
This study investigates the involvement of WWP1-eccDNA in the resistance mechanisms of OC.


MATERIAL AND METHODS
Human OC cells (SKOV3 and cisplatin-resistant SKOV3/DDP) were cultured and high-throughput sequencing was performed, leading to the identification of eccDNA in SKOV3/DDP cells. Female BALB/cA-nu nude mice with SKOV3 and SKOV3/DDP xenografts received cisplatin (5.5 mg/kg), hydroxyurea (50 mg/kg) or saline for 14 days, followed by tumor weight assessment. Digital droplet polymerase chain reaction (ddPCR) and real-time quantitative polymerase chain reaction (qPCR) were used to quantify WWP1-eccDNA, evaluating their sensitivity and accuracy. Linear DNA removal and BsmI digestion were tested to improve eccDNA detection.


RESULTS
WWP1-eccDNA was among the top upregulated eccDNA in SKOV3/DDP cells. Both cisplatin and hydroxyurea reduced tumor growth in mice, with cisplatin showing limited efficacy in resistant tumors. The ddPCR outperformed RT-qPCR in sensitivity, and linear DNA removal improved WWP1-eccDNA detection. WWP1-eccDNA levels were significantly elevated in SKOV3/DDP tumors. Treatment with cisplatin further increased its expression, whereas hydroxyurea led to a reduction in WWP1-eccDNA levels.


CONCLUSIONS
WWP1-eccDNA is critical in OC resistance, with cisplatin treatment increasing WWP1-eccDNA levels, contributing to resistance. The ddPCR proves to be a superior method for eccDNA detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ff3cc0602dcb179f8683c613e50da47a26f84f2" target='_blank'>
              Cisplatin-induced WWP1-eccDNA expression contributes to ovarian cancer resistance.
              </a>
            </td>
          <td>
            Chenyang Lu, Li HAN, Xiaojuan Guo, Ruijuan Du, Hui Zhang, K. Guo, Yun-Fei Tu, Ruifang Li
          </td>
          <td>2025-09-04</td>
          <td>Advances in clinical and experimental medicine : official organ Wroclaw Medical University</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 DNA damaging agents remain the mainstay for high grade serous ovarian cancer (HGSOC) therapy. While many HGSOCs are sensitized to DNA damaging agents via dysregulated DNA repair, those patients harboring tumors with amplification or overexpression of CCNE1 (encoding cyclin E1) respond poorly in part due to proficiency in the homologous recombination (HR) DNA repair pathway. Here, we put forth a novel paradigm that HR proficiency of CCNE1-driven HGSOCs is promoted by altered cellular metabolism. We found that alpha-ketoglutarate (aKG) is upregulated in cyclin E1-driven HGSOC cells, and suppression of aKG sensitized these cells to DNA damaging agents both in vitro and in vivo, demonstrating that aKG drives chemoresistance in these models. aKG is required for the activity of aKG-dependent dioxygenases (aKGDD). There are >60 known aKGDD, and prior work has almost exclusively focused on aKGDD-mediated histone demethylation. Through a targeted CRISPR knockout library, we discovered that Trimethyllysine Hydroxylase Epsilon (TMLHE), the first and rate-limiting enzyme in de novo carnitine synthesis, is necessary for chemoresistance to DNA damaging agents in cyclin E1-driven cells. Unexpectedly, aKG-mediated TMLHE-dependent carnitine synthesis was required for histone acetylation, while histone methylation was affected but dispensable. These data point to a previously unrecognized role for aKG in acetylation reactions. The increase in histone acetylation via aKG-dependent carnitine synthesis promoted HR-mediated repair through site-specific histone acetylation. Additionally, analysis of HGSOC patient samples demonstrated that TMLHE positively correlates with histone acetylation, and both high tumor TMLHE or high serum acetylcarnitine are associated with worse progression free survival (PFS) in HGSOC patients treated with DNA damaging agents. Pharmacological targeting of carnitine synthesis via mildronate in vivo was sufficient to decrease tumor burden >30% in combination with cisplatin. Since mildronate is a non-toxic and extensively used drug in humans, this combination shows potential applicability of inhibition of this axis in humans. These data demonstrate for the first time that HR-proficiency in cyclin E1-driven HGSOCs is mediated through a metabolic axis that directly influences histone acetylation and highlight the translational potential of targeting this axis to induce HR deficiency and sensitize these tumors to DNA damaging agents.



 Apoorva Uboveja, Nathaniel W. Snyder, Katherine M. Aird. aKG-mediated carnitine synthesis promotes chemoresistance in CCNE1-high ovarian cancers via enhancing histone acetylation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Ovarian Cancer Research; 2025 Sep 19-21; Denver, CO. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl):Abstract nr B031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7139762235894a9ff9c39c95032bcf77d46290e" target='_blank'>
              Abstract B031: aKG-mediated carnitine synthesis promotes chemoresistance in CCNE1-high ovarian cancers via enhancing histone acetylation
              </a>
            </td>
          <td>
            Apoorva Uboveja, Nathaniel W. Snyder, K. Aird
          </td>
          <td>2025-09-19</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Endometrial cancer (EC) is a common gynecologic malignancy that often exhibits molecular features such as extensive somatic copy number alterations, microsatellite instability and frequent TP53 mutations, which considerably affect the physical and mental well-being of women. The Fanconi anemia (FA) pathway is a DNA damage repair pathway involving multiple FA genes that play crucial roles in DNA damage repair as well as the maintenance of genome stability. Abnormalities in FA, such as deletions or mutations, may lead to defects in DNA damage repair, resulting in increased genomic instability and/or an abnormal cell cycle, ultimately leading to EC. This comprehensive review provides a systematic summary of EC-related FA genes, elucidates the roles of various FA genes in EC and further speculates on their related mechanisms to facilitate the development of targeted therapies that specifically target key genes, leading to a more accurate and efficient treatment for EC. The present review searched PubMed and Google Scholar for articles published in English up to June 2025 using keywords such as Fanconi anemia pathway, 22 FA genes (FANCA, FANCB, FANCC, FANCD1/BRCA2, FANCD2, FANCE, FANCF, FANCG, FANCI, FANCJ/BRIP1, FANCL, FANCM, FANCN/PALB2, FANCO/RAD51C, FANCP/SLX4, FANCQ/XPF, FANCR/RAD51, FANCS/BRCA1, FANCT/UBE2T, FANCU/XRCC2, FANCV/REV7, FANCW/RFWD3), endometrial cancer (type I: Endometrioid adenocarcinoma; Type II Uterine serous carcinoma, clear-cell carcinoma, carcinosarcoma), somatic copy number alterations, microsatellite instability, TP53 mutations, pathogenesis, genomic instability, target therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd983a8338fd34520960aa5e9a79e6e3166a39ae" target='_blank'>
              Potential role of Fanconi anemia pathway in the pathogenesis of endometrial cancer (Review)
              </a>
            </td>
          <td>
            Mengmeng Yao, Chuqi Liu, Huiyu Ping, Kaidi Meng, Xinru Li, Qingxin Li, Yuanmin Qi, Ziming Zhu, Li Zhang, Aizhong Han
          </td>
          <td>2025-08-21</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) with rearrangement of the mixed lineage leukemia gene express MLL-AF9 fusion protein, a transcription factor that impairs differentiation and drives expansion of leukemic cells. We report here that the zinc finger protein GFI1 together with the histone methyltransferase LSD1 occupies the promoter and regulates expression of the lncRNA ELDR in the MLL-r AML cell line THP-1. Forced ELDR overexpression enhanced the growth inhibition of an LSD1i/ATRA combination treatment and reduced the capacity of these cells to generate leukemia in xenografts, leading to a longer leukemia-free survival. We found that ELDR binds the clamp protein PCNA and the MCM5 helicases causing defects of DNA replication fork progression. Moreover, AML cells overexpressing ELDR showed reduced chromatin accessibility and transcription at α-satellite repeats in centromeres. In addition, ELDR RNA was detected close to MLL-AF9 at centromeres suggesting that it impedes leukemic progression preferentially of MLL-r AML by interfering with both DNA replication and centromeric transcription. Our findings reveal novel functions of the lncRNA ELDR in DNA replication and centromere biology when expressed at high levels in AML cells with MLL rearrangements. These discoveries could provide rationale for future strategies to treat MLL-r AML, which has a poor prognosis in children and adults. Delivery of the ELDR RNA could potentially be utilized as an adjunct to LSD1i/ATRA treatment or other currently used chemotherapeutic drugs to develop novel therapies for these AML subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bca6f7f7a7b4fc0402c634bb0d6e344f0b3e15b" target='_blank'>
              The lncRNA ELDR suppresses tumorigenicity of AML by interfering with DNA replication and chromatin accessibility.
              </a>
            </td>
          <td>
            K. Arman, Julie Ross, Eva-Maria Piskor, Luca Lazzari, Virginie Calderon, Gaspard Reulet, Maria Vera, Edlie St Hilaire, Hugo Wurtele, Brian T Wilhelm, T. Möröy
          </td>
          <td>2025-09-08</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Bladder cancer (BC) is a disease that predominantly affects older adults, with aging playing a critical role in its onset and progression. Age-associated phenomena, including immunosenescence and chronic inflammation, form a pro-tumor milieu, while genomic instability and epigenetic drift further increase cancer risk. The review highlights the dual role of DNA methylation in BC: global hypomethylation can activate transposable elements and oncogenes, whereas focal hypermethylation silences tumor-suppressor genes like CDKN2A, especially detrimental in older tissues that rely on these genes for senescence control. In parallel, frequent mutations in chromatin modifiers (e.g., KDM6A, KMT2D) and overexpression of histone-modifying enzymes (e.g., EZH2) alter the tumor epigenome to promote immune evasion and tumor aggressiveness. At the non-coding RNA level, dysregulated microRNAs (miRNAs) and long non-coding RNAs (lncRNAs) in BC contribute to aberrant proliferation, metastatic potential, and immune suppression, with aging-associated declines in miRNA processing further exacerbating these effects. Collectively, the accumulation of epigenetic alterations in older patients appears to facilitate both tumor progression and resistance to therapy. Looking forward, epigenetic biomarkers may improve early detection and risk stratification. Furthermore, “epigenetic therapies,” such as DNA methyltransferase inhibitors (DNMTi), EZH2 inhibitors (EZH2i), or histone deacetylases inhibitors (HDACi), hold promise to restore tumor-suppressor function and enhance immunogenicity, offering an attractive avenue for improving outcomes in older patients with BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42bc5ece634826938f00d72c674db3193d520a4b" target='_blank'>
              Epigenetic regulation of bladder cancer in the context of aging
              </a>
            </td>
          <td>
            Xuewei Liu, Guofeng Ding, Yifan Liu, Xiaoli Yan, Yan Zhao, Hailin Lv, Xiaojuan Xu
          </td>
          <td>2025-08-21</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Oral squamous cell carcinoma (OSCC) is a major health burden in most parts of the world, and pathogenesis of the disease is strongly associated with a complex combination between genetic mutations and epigenetic alterations. The current study will explore the DNA methylation phenomenon and how it affects the silencing of tumour suppressor genes and activate oncogenes in OSCC in a bid to explain the driving molecular process behind tumour development. We evaluated 50 samples of the OSCC tissues with 50 of the adjacent normal tissues, evaluated DNA methylation patterns with methylation-specific PCR and quantitative reverse-transcriptase PCR measured the expression of the DNA repair genes like BRCA1 and MLH1, among others. This finding showed different levels of DNA methylation of cancerous and normal tissues with hypermethylation causing the inactivation of important tumor suppressor genes and hypomethylation causing the activation of oncogenes. In addition, the downregulation of DNA repair genes was noted to be highly significant in OSCC samples indicating that genomic-instability may be related to epigenetic changes. These results demonstrate that aberrant DNA methylation is central to OSCC growth and progression, thus helping in the future use of methylation patterns that can be used as early detection, diagnosis, and prognostic biomarkers. Our findings support the idea that genetic, along with epigenetic, profiling is an issue of key importance toward comprehending OSCC biology and personalized therapeutic interventions. It is recommended that further confirmation be carried out to verify the clinical significance of such epigenetic markers in bigger cohorts, as well as testing such markers in guided therapies, which might eventually lead to better patient outcomes in the management of oral cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fd5e50c6888572f5cfc48d30ee386841f2ea57" target='_blank'>
              Epigenetic and genetic events of oral squamous cell carcinoma: perspective on DNA methylation, silencing of tumor suppressor gene, and activating oncogenes.
              </a>
            </td>
          <td>
            Zainab Nizar Jawad
          </td>
          <td>2025-10-07</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Replication stress (RS), if not effectively and timely addressed, could result in DNA damage in mitosis. However, the relationship between RS and other mitotic events, such as nuclear envelope (NE) breakdown and reassembly, remains poorly understood. Here we report that RS can lead to NE defect. Importantly, rather than de novo NE rupture, the defect per se is a result of nuclear envelope reassembly defect (NERD) during mitosis. Interestingly, NERD is associated with mitotic DNA damage, and repair of the damage by DNA polymerase theta (Polθ)-mediated end joining (TMEJ) ameliorates NERD. Genomic mapping of lamina associated domains (LADs) by cleavage under targets and tagmentation (CUT&Tag) identifies a population of replication stress-sensitive LADs (RESSLADs). Strikingly, a substantial portion of RESSLADs reside in the common fragile sites (CFSs). The loss of RESSLADs-NE interaction under RS might be attributed to the sustained phosphorylation of Lamin A/C at the sites of NERD. In addition, prominent NE defect is observed under multiple conditions of synthetic lethality. Altogether, these findings establish a link between genome instability and nuclear vulnerability under replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/939377138dd121885a6964852dac3b04dd8c9c53" target='_blank'>
              Mitotic DNA repair by TMEJ suppresses replication stress-induced nuclear envelope reassembly defect
              </a>
            </td>
          <td>
            Guojun Ye, Yide He, Yihui Zhang, Dongchen Li, Fuhai Liu, Yi Li, Qinglian Ge, Qiong Guo, Shuya Han, Chunyu Song, Weiping Chang, Haoyue Zhang, Qin Peng, Kun Sun, Wei-Ke Ji, Lin Deng
          </td>
          <td>2025-10-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Protein arginine methyltransferase 5 (PRMT5) has pleiotropic functions in human cells but also participates in orchestrating DNA double strand break (DSB) repair. It methylates the TIP60 histone acetyltransferase complex to facilitate recruitment to the DSB and chromatin remodeling. PRMT5 mutations affect DSB repair by homologous recombination and increase chromosomal instability. In this report we characterized genetic interactions between PRMT5 mutations and mutations in other components of DSB repair pathway. We used data deposited on the Catalogue of Somatic Mutations in Cancers (COSMIC). We found that PRMT5 makes negative genetic interactions with TIP60 and member of the 9-1-1 complex (RAD9, RAD1, HUS1) which is required for checkpoint activation. A comprehensive analysis of all cancer data deposited on COSMIC reveals very few samples with mutations in both PRMT5 and TIP60 or components of the 9-1-1 complex in samples where mutations in other DNA damage repair genes occur (e.g., MRN, checkpoint genes, etc). This suggests that when more factors of the DNA damage repair machinery are destabilized, the functions of TIP60 and 9-1-1 appear to become essential. Protein 3-D structure analysis shows that mutations affect protein-protein interactions that may destabilize 9-1-1 or TIP60 complex formation. These data highlight interesting interactions between the various genetic pathways governing DSB repair. It also reveals potential therapeutic targets. For example, inhibition of the 9-1-1 complex in a PRMT5 mutant may selectively kill the cell. Given that PRMT5 small molecule inhibitors are being developed or already deployed, these findings should inform potential applications of these drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50af5dcce0290d41d138fd6f03848b6fe42b6650" target='_blank'>
              PRMT5 genetic interactions with DNA double strand break repair genes
              </a>
            </td>
          <td>
            Hunter J. Bliss, Juliana Tron, Wesley Bush, Renee A. Bouley, Ruben C. Petreaca
          </td>
          <td>2025-10-09</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Osteosarcoma is a genomically complex tumor characterized by widespread structural rearrangements. This complexity has limited development of therapeutic strategies informed by molecular mechanisms of oncogenesis. We hypothesized that epigenetic mechanisms could drive distinct subtypes of osteosarcoma. Through analysis of chromatin accessibility, we identified an "early osteoblast-derived" (EOD) cell state characterized by upregulation of transcription factors associated with early bone development, and a "late osteoblast-derived" state (LOD), characterized by upregulation of genes involved in late bone development. We then defined core regulatory circuitries governing the underlying gene expression programs in these two cell states. Multiomic single-cell analysis indicates that these cell states co-exist in a single tumor. Finally, using a panel of patient-derived xenograft models, we identified differential drug responses dependent on these cellular states. These findings create opportunities for developing new combination therapy strategies for osteosarcoma treatment and underscore the value of defining epigenetic subtypes in highly genomically complex cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/061de65b88f0b6c7fc51a022f24db3f11873c460" target='_blank'>
              Epigenetic and transcriptional programs define osteosarcoma subtypes and establish targetable vulnerabilities.
              </a>
            </td>
          <td>
            E. López-Fuentes, Andrew S Clugston, Alex G Lee, L. Sayles, Natalie Sorensen, María V Pons Ventura, Stanley G Leung, Truc Dinh, Marcus R. Breese, E. Sweet-Cordero
          </td>
          <td>2025-10-03</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Breast cancer biology is influenced by both genetic ancestry and the environment with implications for disease outcomes. However, we lack the knowledge how non-cancerous cells within tumors are functionally altered by these factors at single-cell resolution. Here, we created a single-nucleus (sn) dataset from 33 African American, 25 Kenyan, and 24 European American women with breast cancer using combined RNA and ATAC sequencing. We successfully isolated intact, high-quality single nuclei from archival frozen breast tumor tissue using an optimized combination of enzymatic digestion and automated tissue homogenization. SnMultiome sequencing of 82 tumors was performed using the 10x Genomics platform. Following filtering, normalization, peak calling, and integration, our dataset includes a total of 292,458 nuclei. Cancerous (163,419 nuclei) and non-cancerous (129,039 nuclei) cells were distinguished based on DNA copy number. Within the microenvironment, 11 major immune, epithelial, and stromal cell types were identified. CD45+, EPCAM- cells were further analytically extracted, clustered, and manually annotated based on known marker genes, detecting 26 distinct immune subpopulations across our samples. After adjustment for potential confounders (age, body mass index, and tumor molecular subtype) in all analyses, we found that women of African descent possessed distinct, activated mesenchymal and endothelial cell subpopulations within the tumor vasculature that promote inflammation, angiogenesis, and cancer cell invasion. Women of African descent exhibited a more immune-suppressive tumor microenvironment, with elevated expression of immune checkpoint markers (PD-1, TIGIT, CTLA-4, and LAG-3), functional impairment of dendritic cells, and enrichment of regulatory and tolerogenic lymphocytic subpopulations. Further, malignant cells from African descent patients expressed a discrete set of CAR-T targets with potential implications for precision immunotherapy. Lastly, we developed three gene signatures within the microenvironment that are predictive of survival with external validation in both tumor-adjacent normal and cancerous tissues from the TCGA. Together, we uncovered molecular breast tumor characteristics of clinical significance in women of African ancestry.



 Alexandra R. Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H. Dorsey, Eytan Ruppin, Shahin Sayed, Francis Makokha, Gretchen L. Gierach, Stefan Ambs. Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy [abstract]. In: Proceedings of the 18th AACR Conference on the Science of Cancer Health Disparities; 2025 Sep 18-21; Baltimore, MD. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2025;34(9 Suppl):Abstract nr C109.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47b4c16fead2784b8ef235041a0b9c641f582e01" target='_blank'>
              Abstract C109: Single-nucleus sequencing reveals a functionally distinct breast tumor microenvironment by ancestral group with implications for disease progression and therapy
              </a>
            </td>
          <td>
            Alexandra R Harris, Huaitian Liu, Brittany D. Jenkins, Sanna Madan, Tiangen Chang, Tiffany H Dorsey, E. Ruppin, Shahin Sayed, Francis W. Makokha, Gretchen L. Gierach, S. Ambs
          </td>
          <td>2025-09-18</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a highly heterogeneous disease, with significantly higher incidence and fatality rates in the elderly. Even with recent decades of research progress in AML, the exact etiology of this deadly disease is still not fully understood, with recent advancements in sequencing technologies highlighting the role of a growing number of non-coding RNAs (ncRNAs) that are intimately associated with AML leukemogenesis. These ncRNAs have been found to have a significant role in leukemia-related cellular processes such as cell division, proliferation, and death. A few of these non-coding RNAs exhibit potential as prognostic biomarkers. The three main groups of ncRNAs that contribute unique activities, especially in cancer, are microRNAs (miRNAs), long non-coding RNAs (lncRNAs), and circular RNAs (circRNAs). Their existence or altered expression levels frequently offer vital information on the diagnosis, course of treatment, and follow-up of cancer patients. The identification of ncRNAs has opened up new avenues for the diagnosis, prognosis, and therapy of acute myeloid leukemia. In order to provide a clear understanding of the significant influence that lncRNAs have on prognostic predictions and diagnostic accuracy in AML, this review aims to provide a comprehensive and insightful understanding of how these molecules actively participate in the complex landscape of the disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f9b9b43383afdfc6c7d6a505ee096f783aeb161" target='_blank'>
              Role of Non-Coding RNAs in Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Shailendra S Maurya, Sarita Maurya, S. Chaturvedi
          </td>
          <td>2025-09-19</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d05a5f9576a9619318986d87c824d703b9e97b8" target='_blank'>
              Targeted DNA ADP-ribosylation triggers templated repair in bacteria and base mutagenesis in eukaryotes.
              </a>
            </td>
          <td>
            Darshana Gupta, Constantinos Patinios, H. V. Bassett, A. Kibe, Scott P. Collins, Charlotte Kamm, Yanyan Wang, Chengsong Zhao, Katie Vollen, C. Toussaint, Irene Calvin, G. Cullot, Eric J. Aird, Kathryn M. Polkoff, T. Nguyen-Vo, Angela Migur, Friso T Schut, Ibrahim S. Al’Abri, Tatjana Achmedov, Alessandro Del Re, Jacob E. Corn, A. Saliba, Nathan C. Crook, A. Stepanova, J. M. Alonso, Chase L. Beisel
          </td>
          <td>2025-09-04</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Introduction Ovarian Cancer remains a significant global health concern, with high mortality rates, largely due to late-stage diagnosis and limited treatment options. These extrinsic factors are driven or exacerbated by intrinsic mechanisms such as persistent activation or upregulation of Signal Transducer and Activator of Transcription 3 (STAT3). STAT3 promotes tumor growth, inhibits apoptosis, accelerates angiogenesis and metastasis, facilitates immune evasion, and contributes to chemoresistance. Consequently, STAT3 activation fosters an aggressive ovarian cancer phenotype, contributing to treatment failure, poor prognosis and low survival rates, highlighting the urgent need for novel, safe, effective and affordable STAT3-targeted therapeutic strategies. In this study, we developed a novel double-stranded DNA minicircle (mcDNA) inhibitor, designed to act as a decoy for STAT3, preventing its binding to target gene promoters. Methods Utilizing the SKOV3 ovarian cancer cell line, we evaluated the effects of our inhibitor in vitro on cell viability through MTS assay, its apoptotic and necrotic effects using flow cytometry and the expression modulation of downstream STAT3-regulated genes, assayed through RT-qPCR and Western blot analysis. Results We demonstrate that anti-STAT3 mcDNA significantly reduces the viability of SKOV3 cells at low nanomolar concentrations, while sparing the control group. The effect observed was dose-dependent. Mechanistically, anti-STAT3 mcDNA induces apoptosis and necrosis in treated cells, also revealing a certain dose-dependency, while also decreasing cell proliferation. Finally, our inhibitor significantly downregulates STAT3-dependent anti-apoptotic genes MCL1 and PIM1. Conclusion These findings suggest that anti-STAT3 mcDNA is a promising, effective and specific candidate for targeted STAT3 inhibition in SKOV3 ovarian cancer cells, warranting further validation in ovarian cancer, in vivo exploration and potential application in other types of malignancies, where STAT3 acts as an oncogenic factor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/298edc0f21db1280384ca9a0eea5b54ca57a8bc1" target='_blank'>
              Breaking the cancer code: a novel DNA minicircle to disable STAT3 in ovarian cancer cells SKOV3
              </a>
            </td>
          <td>
            Adina-Gabriela Vasilescu, Andrei-Mihai Vasilescu, Livia E. Sima, Natalia Baran, S. Szedlacsek
          </td>
          <td>2025-09-15</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Graphical Abstract Abstract Dysfunction of several WD40 family proteins causes diverse endocrine diseases. Until recently, MEP50, a WD40 protein, was considered a gene of unknown significance because no inherited disease had been linked to its function. However, genetic inactivation of MEP50 in mouse models or somatic mutations in humans drives oncogenesis in several endocrine-related cancers, including those of the prostate, breast, and uterus. In this study, we generate new knowledge through a multi-tier integration of evolutionary genomics, sequence and structural analyses, molecular mechanic calculations, and dynamic simulations of wild-type and cancer-mutated MEP50 proteins. Indeed, we find that a conserved splicing event across evolution generates an alternative MEP50 isoform, which is smaller than the canonical MEP50 and lacks the final β-sheet of the first WD40 domain, the entirety of the second WD40 domain, and the first β-sheet of the third WD40 domain. Notably, we find that this novel, short MEP50 (s-MEP50) transcript encodes a 278 amino acid protein that retains aspects of the key regulatory and interaction sites, including those critical for androgen receptor and PRMT5 binding. Finally, we analyze the mutational landscape of MEP50 in endocrine-regulated cancers and use molecular mechanic calculations and dynamic simulations to reveal that cancer-associated mutations disrupt conserved bonding networks and induce widespread structural destabilization within the WD40 domain architecture. Thus, by combining evolutionary, structural, and biophysical approaches, we advance the understanding of MEP50 genomics, providing significant mechanistic and clinically relevant insights into endocrine-regulated tissues and their cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e4d5f2f3bba5deb45827f67cca6b9cab9291043" target='_blank'>
              Genomic features of the methylosome protein MEP50 and its implications in hormone signaling and cancer
              </a>
            </td>
          <td>
            Gareth Pollin, Young-In Chi, Raul Urrutia, G. Lomberk
          </td>
          <td>2025-09-01</td>
          <td>Endocrine Connections</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Oncogenesis, tumor progression and therapy response are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions between these two systems influence tumor evolution and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC that systematically integrates genetic cancer drivers and infiltration profiles of immune cells to find pairwise combinations of drivers and TME characteristics that jointly associate with clinical outcomes. By analyzing 8500 primary tumor samples of 26 cancer types, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell levels were significantly correlated with patient survival. Subsets of tumor samples defined by some IGXs were characterized by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of immunotherapy target genes. In luminal-A breast cancer, an IGX involving MEN1 deletion combined with reduced levels of neutrophils associated with lower progression-free survival and deregulation of immune signaling pathways, as observed in two independent cancer genomics datasets. These results showcase the ability of PACIFIC to integrate complex multi-omics datasets with clinical information, enabling the identification of clinically relevant immunogenomic interactions. Such interactions provide a rich set of hypotheses for mechanistic studies and the development of biomarkers and therapeutic targets. Implications: Co-occurrence patterns of cancer drivers and TME characteristics highlight synergistic interactions with prognostic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3413ab1802170ae0154c57fdbd8827be618414" target='_blank'>
              Cancer genomic alterations and microenvironmental features encode synergistic interactions with disease outcomes.
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P Klein, Alexander T Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C L Cheng, Jigyansa Mishra, Diogo Pellegrina, K. Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2025-09-03</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC) remains a lethal malignancy with limited therapeutic advancements, underscoring the need to identify novel oncogenic drivers. This study integrates multi-omics analyses-including ChIP-seq, bulk RNA-seq, single-cell ATAC/RNA-seq, and spatial transcriptomics-to delineate enhancer-driven transcriptional networks in 59 matched LUSC tumors and adjacent normal tissues. The analyses identify 3,447 tumor-specific oncogenic enhancers (SOEs) enriched for master transcription factors (SOX2, TP63, KLF5, GRHL2) that orchestrate malignant programs. Single-cell epigenomic profiling reveals these SOEs to be exclusively active in a cancer cell-like cluster, pinpointing PTPRZ1, a receptor tyrosine phosphatase, as a top SOE-driven candidate. Functional studies demonstrate that PTPRZ1 is essential for LUSC-tumorigenesis and tumor cells proliferation and migration in vitro and in vivo. Spatial transcriptomic analysis further implicates midkine (MDK) as the ligand activating PTPRZ1 in LUSC tumor tissue. Mechanistically, PTPRZ1 mediates MDK-induced PI3K phosphorylation that is essential for LUSC tumor growth. This work illustrates enhancer addiction as a hallmark of LUSC, identifies the PTPRZ1-MDK axis as an important targetable pathway, and establishes a paradigm for dissecting epigenetic vulnerabilities in solid tumors through multi-omics integration. These findings advance precision oncology strategies for LUSC, bridging epigenomic dysregulation to therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdd6b0f8500832ade4c7417d38653f9c0588ff5" target='_blank'>
              Enhancer Reprogramming Reveals the Tumorigenic Role of PTPRZ1 in Lung Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yong-Qiang Ning, Jie Wang, Yihui Zhang, Hongrui Li, Zenan Jiang, Xinyu Su, Xiao Yang, Xiaoxuan Wu, Bing Wei, Jianming Ying, Jian Yan, Wenbin Li
          </td>
          <td>2025-09-03</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The rate of DNA synthesis is crucial for full DNA duplication. We report a key role of p21 in controlling this rate. During normal replication, p21 promotes nascent DNA synthesis alongside the DNA polymerase iota (Pol ι)/p53 complex. When p21 is down-regulated but detectable, nascent DNA tracks are longer and discontinuous and rely on primase and DNA polymerase (PrimPol). With the complete elimination of p21, nascent DNA tracks become shorter and continuous and depend on Pol kappa (κ). Endogenous p21 levels are critical for genomic stability, as both PrimPol- and Pol κ–mediated syntheses can induce chromosomal instability. The residual expression of p21 in p53-null cells influences the involvement of PrimPol or Pol κ in nascent DNA synthesis and subsequent chromosomal instability. Our results demonstrate that endogenous levels of p21 in cycling cells, insufficient for cyclin-dependent kinase inhibition, prevent genomic instability through proliferating cell nuclear antigen binding (PCNA), limiting PrimPol and Pol κ’s role in nascent DNA synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a31f81e290a07076ab9200968e4922ab77b5675a" target='_blank'>
              Endogenous p21 levels protect genomic stability by suppressing both excess and restrained nascent DNA syntheses
              </a>
            </td>
          <td>
            N. L. Calzetta, S. Mansilla, Candelaria Mares Ahlers, Agostina P. Bertolin, Lilen I. Caimi, Sofía Venerus Arbilla, Jingkun Zeng, Lisa Wiesmüller, V. Gottifredi
          </td>
          <td>2025-10-08</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) is a treatment-resistant subtype that arises through lineage plasticity, allowing tumor cells to bypass androgen receptor (AR)–targeted therapies. In a recent study, Lu et al. define a transcriptional and epigenetic hierarchy that drives this transdifferentiation. The pioneer factor FOXA2 initiates enhancer remodeling and regional DNA demethylation, while the neural lineage transcription factor NKX2-1 is required to complete the NE program. Together, these factors reorganize 3D chromatin architecture and activate lineage-specific genes through enhancer–promoter looping. Crucially, the histone acetyltransferase p300/CBP is essential cofactor in this process. Pharmacologic inhibition of p300/CBP with CCS1477 suppresses NE gene expression and impairs tumor growth in NEPC models. These findings offer a mechanistic insight of lineage plasticity and highlight p300/CBP as promising therapeutic targets. The study also raises key questions about the stability and reversibility of chromatin remodeling and sets a framework for understanding enhancer-driven plasticity in other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3331955821f5eab1873b6edce226793e7651773a" target='_blank'>
              FOXA2 and NKX2-1 Orchestrate Neuroendocrine Lineage Plasticity in Prostate Cancer
              </a>
            </td>
          <td>
            Ka-wing Fong
          </td>
          <td>2025-10-01</td>
          <td>Serican Journal of Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer development and response to treatment are evolutionary processes1,2, but characterizing evolutionary dynamics at a clinically meaningful scale has remained challenging3. Here we develop a new methodology called EVOFLUx, based on natural DNA methylation barcodes fluctuating over time4, that quantitatively infers evolutionary dynamics using only a bulk tumour methylation profile as input. We apply EVOFLUx to 1,976 well-characterized lymphoid cancer samples spanning a broad spectrum of diseases and show that initial tumour growth rate, malignancy age and epimutation rates vary by orders of magnitude across disease types. We measure that subclonal selection occurs only infrequently within bulk samples and detect occasional examples of multiple independent primary tumours. Clinically, we observe faster initial tumour growth in more aggressive disease subtypes, and that evolutionary histories are strong independent prognostic factors in two series of chronic lymphocytic leukaemia. Using EVOFLUx for phylogenetic analyses of aggressive Richter-transformed chronic lymphocytic leukaemia samples detected that the seed of the transformed clone existed decades before presentation. Orthogonal verification of EVOFLUx inferences is provided using additional genetic data, including long-read nanopore sequencing, and clinical variables. Collectively, we show how widely available, low-cost bulk DNA methylation data precisely measure cancer evolutionary dynamics, and provides new insights into cancer biology and clinical behaviour.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57090ca5bdab1ab7fa8186d6d31b9c968e9579a" target='_blank'>
              Fluctuating DNA methylation tracks cancer evolution at clinical scale
              </a>
            </td>
          <td>
            C. Gabbutt, M. Duran-Ferrer, Heather E. Grant, D. Mallo, F. Nadeu, J. Househam, N. Villamor, Madlen Müller, Simon Heath, E. Raineri, O. Krali, J. Nordlund, T. Zenz, I. Gut, E. Campo, A. López-Guillermo, J. Fitzgibbon, C. P. Barnes, D. Shibata, J. Martín-Subero, Trevor A. Graham
          </td>
          <td>2025-09-10</td>
          <td>Nature</td>
          <td>1</td>
          <td>95</td>
        </tr>

        <tr id="


 Genetic alterations in p53 (TP53) are found in most solid malignancies, including pancreatic cancer (PANC). Mutant p53 (p53mut) contributes to cancer progression and metastasis. Existing therapeutic options for p53mut cancers are ineffective and cause toxic side effects, stressing the need for better therapies. To address these clinical problems, we developed a two-drug therapeutic strategy that selectively targets p53mut cancers. This novel strategy combines a thymidine analogue (trifluorothymidine (TFT), a component of TAS102, acting as an inducer of DNA damage), and an inhibitor of poly (ADP) ribose polymerase (PARPi) that works as an amplifier of DNA damage. Incorporation of TFT into DNA provokes post-replicative repair generating single-strand DNA break intermediates. Repair of these intermediates is assisted by PARP, while inhibition of PARP increases more lethal double-strand DNA breaks. In normal p53 wild-type (WT) cells, TAS102 and PARPi activate p53-dependent G1/S checkpoint, limiting DNA damage and promoting repair. In p53mut cells with compromised G1/S checkpoint, TAS102 and PARPi cooperate as the inducer-amplifier pair to increase lethal DNA breaks and cell death. The TAS102-PARPi strategy was validated in preclinical studies. Our first-in-human phase I study with TAS102-talazoparib regimen (NCT04511039) showed that this regimen is safe with promising efficacy. The current work investigated the mechanisms underlying the response to TAS102-PARPi regimen in p53mut cancer cells. Our data show that TAS102-PARPi induces the G2/M checkpoint mediated by ATR kinase in a p53-independent manner. ATR acts through the G2-checkpoint kinases (CHK1 and WEE1) to inactivate CDK1 and stop transition to mitosis. The transcriptome analysis showed that TAS102-PARPi treatment increased expression of Claspin and inactivated PLK1-driven mitotic pathway. Claspin is critical for activation of DNA damage and replication checkpoints by facilitating ATR-CHK1 signaling, whereas PLK1 mediates inactivation of the G2-checkpoint kinases and degradation of Claspin for normal cell cycle progression and re-entry into mitosis. Thus, the PLK1-Claspin network may play a key regulatory role in G2-checkpoint in response to TAS102-PARPi by restraining premature reentry into cell cycle of cells with unrepaired DNA. The next major question was whether TAS102-PARPi regimen can cooperate with drugs disrupting G2-checkpoint. New data show that the efficacy of TAS102-PARPi therapy is greatly enhanced by a sequential application of G2-kinase WEE1 inhibitor. Our data show that this sequential triple-drug strategy works in a synthetic-lethality manner to cause massive cell death in p53mut cancers. The triple-drug strategy greatly improved tumor control in PANC patient-derived xenografts with no signs of major toxicities to normal tissues. In summary, this study provides the scientific foundation for our novel sequential triple-drug strategy that demonstrates greater efficacy against pancreatic cancer with lower toxicity.



 Andrei Bakin, Mohammed Alruwaili, Ashley Guo, Thomas Melendy, Barbara Foster, Christos Fountzilas. Effective targeting p53 mutant pancreatic cancer by novel drug-combination strategy leveraging their DNA damage liabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research—Emerging Science Driving Transformative Solutions; Boston, MA; 2025 Sep 28-Oct 1; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_3):Abstract nr B001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a93f85c005599dc813a5323d69a7be402a3898b5" target='_blank'>
              Abstract B001: Effective targeting p53 mutant pancreatic cancer by novel drug-combination strategy leveraging their DNA damage liabilities
              </a>
            </td>
          <td>
            A. Bakin, M. Alruwaili, Ashley Guo, T. Melendy, Barbara Foster, C. Fountzilas
          </td>
          <td>2025-09-28</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Excised signal circles (ESC) are circular DNA molecules generated during T- and B-cell maturation. Previously considered biologically inert, recent work by Gao et al. now show that ESCs can replicate and accumulate in healthy lymphocytes. Moreover, the authors link higher levels of ESCs to an increased risk of relapse in B-cell leukemia patients and propose that this phenomenon is due to the unique ability of ESCs to induce genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a90caaf42de342ff59afcf74bb20559cdc1183c5" target='_blank'>
              Not just a by-product: circular DNA molecules derived from V(D)J recombination are linked to worse prognosis in B-cell leukemia.
              </a>
            </td>
          <td>
            Davide Pradella, Andrea Ventura
          </td>
          <td>2025-09-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Epstein-Barr Virus (EBV) establishes life-long latent infection in >90% of adults and is a causal agent for diverse cancers and autoimmune diseases. EBV has a complex life cycle in multiple different tissue types that involve dynamic variations in viral gene expression. These gene expression changes account for the success of the virus in long-term persistence and evading host immune control, as well as its potential for driving cancer evolution and autoimmune disease. Here, we review some of the salient features of EBV gene regulation highlighting the many variations of viral transcription. We review recent advances in our understanding of the factors that bind and regulate EBV gene expression. Based on this diversity of viral transcription patterns, we propose that EBV genome consists of gene modules regulated by local promoter-proximal transcription factor combinations that are further regulated by distal regulatory interactions among the various modules that interact through architectural factors, such as CTCF and cohesion. These modules are likely to represent chromatin architectural domains, and can also interact with host chromosome domains that further regulate viral and host gene expression. We propose that this gene regulatory hierarchy provides EBV with necessary plasticity for viral persistence, as well as a strong potentiator for cancer and autoimmune disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a251ae41cd508722e48d62d5ca0f4e635efbec9f" target='_blank'>
              Transcriptional regulation of gene modules in Epstein-Barr virus.
              </a>
            </td>
          <td>
            Paul M. Lieberman
          </td>
          <td>2025-09-22</td>
          <td>Transcription</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The remarkable advances in cancer therapies significantly enhance the survival rates and longevity of cancer patients. Among childhood, adolescent, and young adult female cancer survivors, however, anti-cancer agents frequently cause primary ovarian insufficiency (POI), early menopause, and infertility, primarily due to the depletion of the ovarian reserve. Oocytes, the female germ cells, exhibit a notable susceptibility to DNA damage, given that they remain in meiotic arrest of prophase I for prolonged durations from months to years, which increases the risks of accumulating DNA damage overtime. To counteract this, a tightly controlled DNA damage response (DDR) signaling ensures that only oocytes with an intact genome progress to ovulation, fertilization, and next generations. Chemotherapeutic anti-cancer agents, including doxorubicin, cisplatin, cyclophosphamide, along with irradiation, elicit DNA damage via various mechanisms, including DNA crosslinking, single and double-strand DNA breaks, and oxidative stress. The genotoxic insults activate DDR in the oocytes, which detect and repair DNA damage or initiate apoptosis to eliminate impaired oocytes. Although several protein molecules such as DNA damage-sensing kinases, checkpoint kinases, p53 family transcription factors, and pro-apoptotic molecules, have been discovered, the precise mechanisms of DDR in determining the fate of oocytes, particularly how they differ from those in somatic cells and cancer cells, remain poorly understood. From an oncofertility perspective, the current review analyzes the molecular mechanisms of anti-cancer agent-induced DDR in oocytes and discuss knowledge gaps and urgent future research directions for preserving the ovarian reserve, fertility, and endocrine functions of young female cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b3c0ab5ecbb10104dc6e55fe017fd113001dfb6" target='_blank'>
              DNA damage response signaling in oocytes from an oncofertility perspective.
              </a>
            </td>
          <td>
            Yunman (Sonia) Song, Jiyang Zhang, Yingnan Bo, Qiang Zhang, So-Youn Kim, Shuo Xiao
          </td>
          <td>2025-09-12</td>
          <td>Biology of reproduction</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="An essential category of non-protein-coding RNA molecules, known as long noncoding RNAs (lncRNAs), are naturally occurring RNA sequences exceeding 200 nucleotides in length. Accumulating evidence has unveiled the significant involvement of lncRNAs in various cancers, where they are known to be a key factor for tumors’ statement changing, e.g., modulating proliferative, apoptotic, migratory and chemotherapy resistance in cells. In addition, lncRNAs are also regarded as potential diagnostic and prognostic tumor biomarkers. Studies have suggested that lncRNAs modulate biological processes by regulating linear RNA transcription and protein production. A recently discovered long non-coding RNA, SNHG3, is identified to have associations with cancer, which displays atypical expression patterns and functions as an oncogene in various cancers. Additionally, SNHG3 has been demonstrated to hold promise as a diagnostic biomarker and a tool for prognostic assessment in cancer patients. This review summarizes current knowledge about SNHG3’s expression, functional roles, molecular mechanisms, and implications for diagnosis and prognosis in human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ffaa9b71dfc7869fc1372cea5b3cd75f3cbc0b5" target='_blank'>
              The role of LncRNA SNHG3 in human cancers
              </a>
            </td>
          <td>
            Jinming Tang, Min Su, Yuhang Xiao, Wei Ou, Wenxiang Wang, Ren-Wang Peng, Zhining Wu
          </td>
          <td>2025-10-03</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Tumorigenesis is a multistep process involving germline and somatic alterations, also known as "drivers." Understanding how these alterations cooperate is crucial for assessing cancer risk and developing strategies to intercept malignant transformation.



 To investigate the interplay between germline pathogenic variants (GPVs) and somatic drivers in childhood cancers, and to compare tumor molecular profiles based on the origin of initiating alterations.



 We analyzed whole-genome (paired tumor-germline) and transcriptomic data from 2,397 children enrolled in the ZERO Childhood Cancer Program (2018–2025). A subset of 199 participants with autosomal dominant cancer predisposition syndromes consistent with their tumor type was selected for detailed analysis.



 GPVs were distributed across multiple cancer predisposition genes (CPGs), with TP53, NF1, SMARCB1, and DICER1 being the most prevalent. A second somatic hit in the same CPG was identified in 83% of cases (166/199), 56% resulting from loss of heterozygosity. RNA sequencing revealed additional variants (10% of cases) not detected in the DNA data. Additional somatic drivers were identified in 149 tumors (75%), 82 of which exhibited recurrent drivers for their cancer type (e.g., CDKN2A in NF1-driven malignant peripheral nerve sheath tumor). In 33% of them (27/82), no second hit was present in the CPG (e.g., NF1 GPV in low-grade glioma with somatic FGFR1 mutation), suggesting the GPV may act as a cooperating enabler. Of interest, 67 tumors harbored additional drivers atypical for their cancer type, with most (62/67) showing a second hit in the same CPG, suggesting a distinct molecular profile in this group of GPV-driven tumors (e.g. GPV in FBXW7 and Wilms tumor with somatic CDKN2A loss). In 49 cases, the only somatic event was a somatic second hit in the CPG, indicating that some GPVs may independently initiate tumorigenesis (e.g., SMARCB1, DICER1). This supports the hypothesis that GPVs can enable and promote tumour progression without the need for cooperation with other genomic drivers. Conversely, 25 cases lacked a second hit in the CPG but harbored other somatic drivers, including unexpected tumor alterations (e.g., GPV in RECQL4 and osteosarcoma without TP53 alteration), suggesting possible GPV haploinsufficiency in driving tumorigenesis through an alternative molecular pathway.



 Our study highlights the multifactorial nature of tumour initiation and promotion in GPV-driven tumorigenesis involving direct initiation and cooperation with somatic drivers. Genes like SMARCB1 can drive cancer alone, while others like NF1 need additional somatic alterations for malignancy. Notably, biallelic inactivation of CPG rarely leads directly to high-grade tumors, emphasizing gene and cancer-type specificity of this process. Future research, such as the study of induced pluripotent cancer stem cells, can help clarify the role of GPV and the sequence of multi-step events. This may guide strategies for preventing malignant transformation and enabling earlier detection.



 Noemi A Fuentes-Bolanos, Dianne Sylvester, Eliza Courtney, Chelsea Mayoh, Meera Warby, Dong Anh Khuong Quang, Paulette Barahona, Ashleigh Sullivan, Ann Altekoester, Megan Rumford, Hetal Dholaria, Marie Wong, Vaness Tyrrell, Michelle Haber, Katherine Tucker, Mark J Cowley, David S Ziegler, Mark Pinese, Loretta M S Lau. Cooperation between germline and somatic drivers in childhood cancer: insights from the ZERO cohort [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75b9cb084afe69c38e9d831c9072216bcdeed64f" target='_blank'>
              Abstract A045: Cooperation between germline and somatic drivers in childhood cancer: insights from the ZERO cohort
              </a>
            </td>
          <td>
            N. Fuentes-Bolanos, Dianne Sylvester, Eliza Courtney, Chelsea Mayoh, Meera Warby, Dong Anh Khuong Quang, P. Barahona, Ashleigh Sullivan, A. Altekoester, Megan Rumford, Hetal Dholaria, Marie Wong, Vanessa Tyrrell, Michelle Haber, Katherine Tucker, Mark J. Cowley, D. S. Ziegler, M. Pinese, L. Lau
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Abstract Background Chromophobe renal cell carcinoma (ChRCC) is the second most common non-clear cell renal cell carcinoma (ncRCC). A series of 496 ChRCC from Memorial Sloan Kettering Cancer Center demonstrated that sarcomatoid features, which were present in about 1.2% of the cases, are strongly correlated with metastatic disease and reduced overall survival. Little is known about the biological and immunological characteristics of the sarcomatoid variant of ChRCC and how it differs from its classic, non-sarcomatoid form. Methods Eight ChRCC tumor samples containing both classic and sarcomatoid components were selected. Xenium Prime 5K assay (5000 curated genes related to oncoimmunology) was used for high-resolution spatial transcriptomic profiling. Data were analyzed using Seurat and custom scripts to annotate cell types and conduct detailed phenotypic analysis. Pathway enrichment and transcription factor regulatory networks were analyzed using the SENIC package. Cell-cell communication was inferred using CellChat, and spatial proximity of immune and tumor cells (“cellular neighborhoods”) was analyzed, which model spatial gene expression patterns and intercellular distances. Results Spatial transcriptomic analyses revealed a significantly more immune-inflamed tumor microenvironment in sarcomatoid regions compared to their classic counterparts. We identified distinct spatial immune “neighborhoods” in sarcomatoid areas that were enriched in CD8+ T cells, polarized macrophages, and dendritic cells. These neighborhoods exhibited organized proximity to tumor cells expressing immune evasion markers such as PD-L1. In contrast, classic regions were relatively immune-deserted. Analysis of transcription factor activity showed upregulation of IRF1 and STAT1-related programs in the immune infiltrates of sarcomatoid regions, consistent with inflammatory activation. Importantly, comparison of matched sarcomatoid and classic regions within the same tumors enabled us to define a “sarcomatoid-enriched gene signature” composed of immune-related, pro-metastatic, and epithelial-to-mesenchymal transition (EMT) genes. This signature was validated in bulk RNA-seq data from The Cancer Genome Atlas (TCGA), where it correlated with poor prognosis in ChRCC patients. Conclusions This study presents the first spatial transcriptomic atlas of sarcomatoid ChRCC in comparison to matched non-sarcomatoid (classic) from the same patient. Profound immune microenvironmental differences were identified between tumor compartments. The sarcomatoid variant exhibits hallmarks of an immune-inflamed but potentially immune-evasive phenotype, including enriched immune infiltration, pro-inflammatory signaling, and EMT-associated transcriptional programs. These findings highlight the biological differences between sarcomatoid and classic ChRCC and uncover specific immune pathways and spatial cell-cell interactions that may be targeted in future immunotherapeutic strategies. Our work also establishes a foundational spatial map for ChRCC that offers new insight into the disease heterogeneity and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aba1786fb42a662d7a501d0a751ee46eaa895c9" target='_blank'>
              52Spatial transcriptomic mapping reveals immune-inflamed niches in sarcomatoid chromophobe renal cell carcinoma
              </a>
            </td>
          <td>
            Yan Tang, Tiegang Han, Hadi Mansour, M. Hirsch, Katrina Collins, Elizabeth P Henske
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background DNA methylation of CpG islands is altered in cancer cells. Hypermethylation of single CpG islands in the promoter regions of tumor-suppressor genes occurs already in the early stages of cancer. These methylation changes are cancer-type specific and therefore can serve as early cancer biomarker. Identifying good and reliable biomarkers is crucial for the development of diagnostic tests and their application in clinical practice and remains the most significant challenge to date. Results Here, we present a generic workflow for the discovery and design of DNA methylation-specific PCR (MSP) biomarkers using nanopore sequencing. We show that nanopore sequencing of three control and three tumor tissue samples was sufficient to predict differentially methylated regions between head and neck squamous cell carcinoma (HNSCC) and healthy control tissue samples and to design functional MSP primers. When applied to a validation cohort of 48 HNSCC and 46 control samples, four out of six designed MSP singleplex assays achieved good sensitivity and specificity with an AUC above 0.8. Conclusion Our resulting DNA methylation-based workflow demonstrates how long-read methylation data enable the design of adaptable, clinically relevant epigenetic assays, even with low coverage and small initial sample numbers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ca77d339f993655ed91f8a895c93505240e806a" target='_blank'>
              Nanopore sequencing-derived methylation biomarker prediction for methylation-specific PCR in patients with head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Daria Meyer, Anne Hennig, Anna-Bawany Hums, Orlando Guntinas‑Lichius, Martina Schmitz, Manja Marz
          </td>
          <td>2025-09-13</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f5b13afa4acec24579eb468c6dc31248a66bb3e" target='_blank'>
              Detection of clonal hematopoiesis of indeterminate potential via genome or exome sequencing profoundly underestimates disease associations
              </a>
            </td>
          <td>
            R. Corty, Y. Pershad, C. Vlasschaert, L. Luo, T. Mack, K. Amancherla, C. Robinson-Cohen, M. Savona, A. G. Bick
          </td>
          <td>2025-08-16</td>
          <td>None</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Epigenetic regulation plays a key role in the progression, proliferation, and dedifferentiation of thyroid cancer. Epigenetic control occurs at multiple levels, including DNA methylation, RNA modification, histone modification, chromatin remodeling, and chromatin accessibility. Genetic alterations in chromatin regulators are commonly observed in thyroid cancer, which includes papillary thyroid carcinoma (PTC), medullary thyroid carcinoma (MTC), anaplastic thyroid carcinoma (ATC), and follicular thyroid carcinoma (FTC). These cancers exhibit distinct characteristics in terms of genetics, biology, and clinical presentation. Therefore, we review the disease biology driven by changes in chromatin pathways in thyroid cancer. Specifically, we summarize examples of epigenetic dysregulation at each level, with mechanisms involving alterations in enzymes regulating DNA methylation, RNA modification and posttranslational modifications of histones, as well as the loss or fusion of subunits involved in chromatin remodeling and chromatin accessibility. Finally, on the basis of clinical applications, we review the current and potential future approaches for thyroid cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4b8e029502363c1309237f110ef671aea68cd77" target='_blank'>
              Epigenetic control in thyroid cancer: mechanisms and clinical perspective
              </a>
            </td>
          <td>
            Jiahui Zhang, Shengkai Zheng, Ruiwang Xie, Junsi Zhang, Xiangjin Chen, Sunwang Xu
          </td>
          <td>2025-08-17</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Menin protein is encoded by the multiple endocrine neoplasia type 1 gene, which typically serves an oncogenic role in endocrine organs. However, in mice, menin protein is a key mediator of leukemic transformation, particularly in acute myeloid leukemia (AML), and is involved in the disease process through epigenetic regulatory mechanisms. This functional paradox may be due to the unique gene expression regulatory properties of menin, which can both activate and inhibit the expression of target genes. At the molecular level, menin protein regulates the gene transcription process by interacting with multiple protein complexes and forms a complex network with multiple signaling pathways. As the core hub of transcriptional regulation, menin protein is essential for the maintenance of cellular homeostasis and its aberrant function can lead to gene expression disorders, which contributes to the development of AML. Despite the druggability challenges of several transcriptionally regulated proteins, inhibitors of menin protein have made breakthroughs in clinical development. Particularly in AML subtypes [for example, lysine methyltransferase 2A (KMT2A) rearrangement or nucleophosmin (NPM1) mutation], menin protein inhibitors have demonstrated favorable efficacy. The present study systematically reviewed biological functions of the menin protein and its application in targeted therapy for specific AML subtypes. Notably, menin inhibitors have demonstrated potential in KMT2A/NPM1-mutant leukemia, however, the off-target effects, resistance mechanisms and a lack of biomarkers due to the extensive nature of their binding interface remains to be elucidated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e18b23d4e7fbebe4140893833cf2070b2e3e6ebf" target='_blank'>
              Targeting menin in lysine methyltransferase 2A/nucleophosmin-mutated leukemia: A novel strategy from epigenetic dysregulation to clinical therapy (Review)
              </a>
            </td>
          <td>
            Junjie Bi, Hongjun Zhou
          </td>
          <td>2025-09-10</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Cancer remains a leading cause of global morbidity and mortality, underscoring the need for innovative diagnostic and prognostic biomarkers. PIWI-interacting RNAs (piRNAs) have emerged as critical regulators of gene expression and genomic stability, with increasing evidence linking their dysregulation to oncogenesis. piRNAs can modulate DNA methylation through interactions with DNA methyltransferases (DNMTs), contributing to tumor suppressor gene silencing and broader epigenetic reprogramming. However, the functional implications of piRNA-mediated methylation changes in cancer are not fully elucidated. This study investigates the circulating levels of piR-823 and piR-651, their associations with DNMT3B expression and global DNA methylation, and their correlations with clinical and molecular markers in colorectal, breast, and prostate cancers. Materials and methods A total of 300 participants, including 150 patients with histologically confirmed cancers and 150 age- and sex-matched healthy controls, were enrolled. Plasma piRNA levels were measured via qRT-PCR, while global 5-methylcytosine (5mC) and DNMT3B concentrations were quantified using ELISA. Immunohistochemistry was performed on resected tumor tissues to assess DNMT3B, Ki-67, p53, E-cadherin, vimentin, HER2, and androgen receptor (AR). Correlations were analyzed using Pearson’s test and multivariate regression models. Diagnostic performance was evaluated via Receiver Operating Characteristic (ROC) curves. Results piR-823 was significantly upregulated in colorectal and breast cancers (p < 0.001), while piR-651 was markedly downregulated in prostate cancer (p = 0.002). Cancer patients exhibited elevated levels of DNMT3B and 5mC (both p < 0.001), which correlated strongly with Ki-67 and p53 expression (r > 0.7, p < 0.001). E-cadherin expression was inversely correlated with DNMT3B and piR-823, consistent with a role in epithelial–mesenchymal transition (EMT). cfDNA levels were also significantly elevated in cancer cases (p < 0.001), suggesting potential utility as a non-invasive biomarker. Conclusion The observed associations between piRNA dysregulation, epigenetic markers, and tumor aggressiveness indicators support the potential of circulating piRNAs—particularly piR-823 and piR-651—as novel biomarkers for cancer detection and progression monitoring. These findings warrant further mechanistic studies to validate their functional roles in tumor epigenetics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec55cb867b96ef67ad7cc822f9de2fa5e24bf5e8" target='_blank'>
              Epigenetic functions and biomarker potential of PIWI-interacting RNAs in oncogenesis
              </a>
            </td>
          <td>
            Kldiashvili Ekaterina, Abiatari Ivane, Kekelia Elene, Iordanishvili Saba, Metreveli Tornike, Dumbadze Eter
          </td>
          <td>2025-09-30</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The cell-type-level epigenomic landscape of human subcutaneous adipose tissue (SAT) is not well characterized. Here, we elucidate the epigenomic landscape across SAT cell types using snm3C-seq. We find that SAT CG methylation (mCG) displays pronounced hypermethylation in myeloid cells and hypomethylation in adipocytes and adipose stem and progenitor cells, driving nearly half of the 705,063 differentially methylated regions (DMRs). Moreover, TET1 and DNMT3A are identified as plausible regulators of the cell-type-level mCG profiles. Both global mCG profiles and chromosomal compartmentalization reflect SAT cell-type lineage. Notably, adipocytes display more short-range chromosomal interactions, forming complex local 3D genomic structures that regulate transcriptional functions, including adipogenesis. Furthermore, adipocyte DMRs and A compartments are enriched for abdominal obesity genome-wide association study (GWAS) variants and polygenic risk, while myeloid A compartments are enriched for inflammation. Together, we characterize the SAT single-cell-level epigenomic landscape and link GWAS variants and partitioned polygenic risk of abdominal obesity and inflammation to the SAT epigenome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d68e6c79788b842ce0d6d08c58f642ad054ed8c2" target='_blank'>
              Single-cell DNA methylome and 3D genome atlas of human subcutaneous adipose tissue
              </a>
            </td>
          <td>
            Z. Chen, Sankha Subhra Das, Asha Kar, S. H. T. Lee, Kevin D. Abuhanna, Marcus Alvarez, Mihir G. Sukhatme, Zitian Wang, Kyla Z. Gelev, Matthew G. Heffel, Yi Zhang, Oren Avram, Elior Rahmani, S. Sankararaman, Markku Laakso, S. Heinonen, H. Peltoniemi, Eran Halperin, Kirsi H Pietiläinen, Chongyuan Luo, P. Pajukanta
          </td>
          <td>2025-08-20</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Homologous recombination (HR) is a DNA double-strand break repair pathway that facilitates genetic exchange and protects damaged replication forks during DNA synthesis. As a template-based repair process, the successful repair of a double-strand break depends on locating suitable homology from a donor DNA sequence elsewhere in the genome. In eukaryotes, Rad51 catalyzes the homology search in coordination with the ATP-dependent motor protein Rad54. The mechanism by which these two proteins regulate forces on dsDNA substrates during homology search remains unknown. Here, we have utilized single-molecule magnetic tweezers and optical trapping methods to monitor remodeling of the DNA template during the homology search. We find that the activity of Rad51 and Rad54 remodels the donor DNA substrate to control the association and dissociation of Rad51-ssDNA filaments in the absence of DNA homology. This mechanism occurs through the application of both linear (tension) and rotational (torsion) forces on the donor DNA. Finally, failure of Rad54 to act processively disrupts target selection in vivo. This study provides a basic understanding of how motorized homology search manipulates the donor DNA during the search for a suitable repair template.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/361ae663e562d2b732a14c06dc567f1a03df65e1" target='_blank'>
              The Eukaryotic homology search complex distorts donor DNA structure to probe for homology
              </a>
            </td>
          <td>
            Mitchell V. Woodhouse, Jingyi Hu, Meiling Wu, Jin Qian, James T. Inman, Michelle D. Wang, J. B. Crickard
          </td>
          <td>2025-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="for 273 TCGA colorecatal cancer subjects in a general linear model fitting a negative binomial distribution. TCGA annotations from the Covariables Univariate analysis The general linear model compared covariates This research first identifies that CXCL10 is significantly downregulated in Black colorectal cancers in two independent datatsets. We also observe that immune related pathways and cells are significantly different between the self identified Black and White Americans from The Cancer Genome Atlas. We use a combination of RT-qPCR, bulk RNA seq anal to show that CXCL10 expression is higher in the MMRdeficient CRC subtype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/652519820bbd179c6d3708acf1e63564e6b14588" target='_blank'>
              The Effect of Mismatch Repair - Deficient/ Microsatellite Instability on Black Minority Colorectal Cancer Gene Expression
              </a>
            </td>
          <td>
            Dimitri F. Joseph, Joseph F. LaComb, Andrew Fu, Ricardo E. Flores, Ellen Li, Alexandra Guillaume, Bin Chen
          </td>
          <td>2025-10-01</td>
          <td>Spartan Medical Research Journal</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="This review examines the potential of disrupting telomere maintenance in Plasmodium as a novel antimalarial strategy. Telomeres are repetitive DNA–protein structures located at chromosome termini, where they preserve genome stability and protect against degradation. Telomere maintenance is crucial for rapid growth, genome integrity, and immune evasion in Plasmodium parasites. Unlike humans, Plasmodium maintains continuous telomerase activity and uses unique telomere-binding proteins across its lifecycle. These features drive parasite virulence and antigenic variation. Emerging evidence suggests that Plasmodium telomeres harbor G-quadruplex (G4) DNA structures, which help stabilize telomeres during replication and may be good targets for small molecules to disrupt their function. Additionally, the parasite depends heavily on its telomerase catalytic subunit, PfTERT, for survival. Inhibiting PfTERT has shown promising results in blocking telomere elongation and impairing replication. Targeting this parasite-specific telomere–telomerase axis may offer a strategic means to destabilize chromosomes, weaken immune evasion, and limit parasite survival, making it a promising antimalarial approach. However, researchers must consider the risks of off-target effects in future drug designs. Though current studies are limited and remain inconclusive, we suggest that future research should investigate combining telomere-directed therapies with existing antimalarials to help overcome resistance and improve treatment outcomes. Herein, we review advances in understanding Plasmodium telomere biology, highlighting its distinct structures, critical telomere-associated proteins, and roles in pathogenesis. We further explore how selective targeting could exploit an Achilles’ heel in parasite survival, offering fresh possibilities for next-generation, parasite-specific malaria therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bb521b6b69871da352e8c028549958d3812bd32" target='_blank'>
              Plasmodium telomere maintenance: uncovering the Achilles’ heel for novel antimalarials
              </a>
            </td>
          <td>
            Theophilus N. Wakai, Dorathy Anzaku, Israel S. Afolabi
          </td>
          <td>2025-09-10</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ovarian cancer (OC) remains one of the deadliest gynecological malignancies. Immune checkpoint blockade (ICB) inhibitors efficacy in OC has been minimal, highlighting the need for a deeper understanding of the immune microenvironment in OC. Recent studies suggest that DNA methylation and transcription factors may influence the response to immunotherapy. This study aims to classify ovarian cancer into distinct immune subtypes by integrating DNA methylation and transcription factor data through comprehensive bioinformatics analysis. Using data from The Cancer Genome Atlas (TCGA), we identified twelve differentially methylated genes (DMGs) associated with transcription factors and categorized OC into two immune subtypes, C1 and C2.The C1 subtype exhibited higher levels of immune infiltration and better prognosis, characteristic of immune "hot" tumors, whereas the C2 subtype was associated with lower immune infiltration and poorer prognosis, indicative of immune "cold" tumors. A prognostic prediction model based on four key genes—KRT81, PAPPA2, FGF10, and FMO2—was developed using the least absolute shrinkage and selection operator (LASSO) and Cox regression analyses. This model effectively stratified the TCGA OC cohort into high- and low-risk groups and was validated by predicting patient survival outcomes. Additionally, drug sensitivity analysis revealed potential therapeutic targets for different risk groups, offering new avenues for precision treatment in ovarian cancer. Immunohistochemical tests confirmed the potential of KRT81 as a prognostic marker for ovarian cancer. Our findings enhance the understanding of the molecular characteristics of the OC immune microenvironment, propose novel biomarkers for prognosis, which may potentially improve the prognosis of OC. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-02630-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01d9c66278ce901978efba119b98a7523cb64b30" target='_blank'>
              DNA methylation and transcription factor-driven immune subtypes in ovarian cancer
              </a>
            </td>
          <td>
            Jingshu Hu, Mu Su, Zhijun Qin, Jiayu Li, Hengyu Wang, Kexin Chang, Guosheng He, Yan Zhang, Xiuwei Chen
          </td>
          <td>2025-08-28</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5de0d15fb6fe47895cf59141995dd4d8303baf" target='_blank'>
              Temporal genomic dynamics shape clinical trajectory in multiple myeloma.
              </a>
            </td>
          <td>
            F. Maura, Marcella Kaddoura, A. Poos, Linda B. Baughn, B. Ziccheddu, Marc-Andrea Bärtsch, Anthony Cirrincione, K. Maclachlan, M. Chojnacka, B. Diamond, M. Papadimitriou, P. Blaney, L. John, Philipp Reichert, S. Huhn, Dylan C Gagler, Yanming Zhang, Ahmet Dogan, A. Lesokhin, Faith Davies, H. Goldschmidt, Roland Fenk, Katja C. Weisel, E. Mai, N. Korde, Gareth J Morgan, S. Rajkumar, Shaji K. Kumar, S. Usmani, O. Landgren, M. Raab, Niels Weinhold
          </td>
          <td>2025-08-20</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>116</td>
        </tr>

        <tr id="Extracellular vesicles (EVs) are promising biomarkers for cancer diagnosis and prognosis due to their ability to carry specific biomolecular cargo, including DNA. However, the clinical utility of DNA methylation-based liquid biopsies using EV-DNA remains underexplored. The low quantity and relatively long length of EV-DNA complicate whole-genome methylation profiling. To address this, we develop Tn5-assisted Enzymatic Methyl-sequencing with Post-conversion Tailing (TEMPT), a bisulfite-free whole-genome profiling method for EV-DNA. TEMPT employs single-adapter Tn5 tagmentation, enzymatic conversion of unmodified cytosines, and post-conversion tailing to generate high-depth whole-genome EV-DNA methylomes. We apply TEMPT to EV-DNA from 58 gastric cancer and polyp samples, generating methylomes from sub-nanogram inputs and identifying differentially methylated regions (DMRs) that distinguish cancer from controls. We identify potential cancer biomarkers through DMR-associated genes, highlighting the roles of EVs in cellular communication. Our findings suggest that immune cells may serve as an alternative source of EV-DNA. This approach holds significant promise for advancing EV-DNA research and its applications in early disease diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad67a59c8fc16bd26c5fe5c57db4a838137c7af1" target='_blank'>
              Whole-genome methylation profiling of extracellular vesicle DNA in gastric cancer identifies intercellular communication features
              </a>
            </td>
          <td>
            Bingqian Lin, Zhenna Jiao, Shouquan Dong, Weikai Yan, Jinting Jiang, Yanfang Du, Xiaocheng Weng, Hongling Wang, Zhiyuan Hu, Yibin Liu, Xiang Zhou
          </td>
          <td>2025-08-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1cae074f29f78d9d694a20a3e12fe166bc58ebc" target='_blank'>
              RAD51-mediated homologous recombination is a pro-tumour driver pathway.
              </a>
            </td>
          <td>
            Bernard S. Lopez
          </td>
          <td>2025-09-24</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Introduction Colibactin is a small genotoxic molecule of polyketide produced by a subset of enteric bacteria including certain Escherichia coli (E. coli) harbored in the human gut microbiota. Its biosynthesis is governed by a multistep enzymatic process encoded by the polyketide synthase (pks) gene cluster. Colibactin is thought to exert its carcinogenic potential primarily through the induction of DNA interstrand crosslinks (ICLs); however, the precise mechanisms underlying its genotoxicity remain largely unresolved. In this study, we focused on ICL formation and its associated repair pathways to investigate whether colibactin-induced ICLs play a central role in the induction of chromosomal aberrations and inhibition of cell proliferation. Findings HAP1 cells deficient in FANCD2, a gene essential for ICL repair, and their wild-type counterparts were infected with colibactin producing (clb⁺) E. coli strains isolated from a Japanese colorectal cancer (CRC) patient. Following recovery culture, the frequency of micronucleated (MN) cells was assessed. The results showed that FANCD2-deficient cells exhibited a significantly higher frequency of MN cells compared to wild-type cells. Additionally, the cytotoxicity of the clb⁺ strains was evaluated using the XTT assay. FANCD2-deficient cells demonstrated higher sensitivity to the clb⁺ E. coli strains than wild-type cells. Conclusion These findings suggest that colibactin, produced by clb⁺ E. coli, can play a role in the formation of ICLs, thereby contributing significantly to the induction of chromosomal aberrations and the inhibition of human cell proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5434e698c009fe5d0b86871ed28e1115dcedc77" target='_blank'>
              DNA cross-link repair deficiency enhances human cell sensitivity to colibactin-induced genotoxicity
              </a>
            </td>
          <td>
            M. Kawanishi, Osamu Tsubohira, Ai Ueshima, Yuuta Hisatomi, Yoshimitsu Oda, Michio Sato, Noriyuki Miyoshi, M. Mutoh, Hideki Ishikawa, Keiji Wakabayashi, Takashi Yagi, Kenji Watanabe
          </td>
          <td>2025-09-22</td>
          <td>Genes and Environment</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The t(11;22) translocation encodes the EWS::FLI1 fusion oncoprotein which is the primary driver of Ewing sarcoma. EWS::FLI1 creates unique, de novo pathogenic enhancers that drive gene expression and are a central mechanism of oncogenesis. Which chromatin regulatory proteins are critical to this mechanism is understudied. Here, we perform a comparative analysis of the function of the chromatin complexes MLL3/4 and p300/CBP in EWS::FLI1-mediated gene regulation. Using EWS::FLI1 degradation models, we define a subset of EWS::FLI1-sensitive enhancers whose activity correlates with p300/CBP function. We perturb both chromatin complexes to establish that in contrast to MLL3/4, p300/CBP is a critical regulator of EWS::FLI1-driven enhancer activity and downstream gene expression. We also show that p300/CBP small-molecule inhibition decelerates tumor growth in vivo. Our work highlights the context-dependent nature of chromatin protein activity at oncogenic enhancers and reveals p300/CBP as an important regulator of Ewing sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eb04dfc69e03ddcb4aa348ab8eef9592fa55873" target='_blank'>
              p300/CBP is an essential driver of pathogenic enhancer activity and gene expression in Ewing sarcoma
              </a>
            </td>
          <td>
            Laura C Godfrey, Brandon Regalado, Sydney R Schweber, Charlie Hatton, Daniela V. Wenge, Yanhe Wen, Meaghan Boileau, Maria Wessels, Jun Qi, Christopher J Ott, K. Stegmaier, Miguel N Rivera, Scott A Armstrong
          </td>
          <td>2025-09-01</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 CDKN2A/B homozygous deletion is among the most common genetic alterations in glioblastoma (GBM), often accompanied by losses of neighbouring tumour suppressors such as MTAP and PTPRD. These co-deletions create vulnerabilities that may be exploited for synthetic lethality-based therapies. While recent network analysis has demonstrated the potential of combination therapy to target such synthetic lethal nodes, the underlying patterns of co-deletion and their downstream molecular effects remain unclear. We conducted a multi-omics analysis to systematically assess these co-deletion patterns and their functional consequences in CDKN2A/B-deleted GBM.



 Copy number variation (CNV), whole exome sequencing (WES), mRNA expression, and reverse-phase protein array (RPPA) data from TCGA-GBM (n=63) and CPTAC (n=96) were analysed. Patients with signiﬁcant CDKN2A/B losses were identiﬁed using GISTIC2.0 and assessed for co-occurring alterations in MTAP, PTPRD, and EGFR and broader 9p21 losses using Fisher’s Exact test. Downstream mRNA and protein expression changes were evaluated with Mann-Whitney U tests.



 CDKN2A/B homozygous deletion was strongly associated with MTAP loss (TCGA-GBM: 74.6%; CPTAC: 83.0%), moderate PTPRD loss (TCGA-GBM: 55.6%; CPTAC: 30.8%), and moderate EGFR ampliﬁcation (TCGA-GBM: 55.6%; CPTAC: 60.4%). Complete 9p21 loss was common (TCGA-GBM: 54.9%; CPTAC: 35.4%) but deletion proportions varied within and across cohorts. CDKN2A/B-deleted patients had signiﬁcant reductions in MTAP and PTPRD mRNA and protein expression (p < 0.05) and increases in EGFR mRNA and protein expression (p < 0.05) com- pared to non-deleted patients.



 CDKN2A/B deletion is associated with broader 9p21 loss and functional changes in key synthetic lethal nodes, supporting its potential as a therapeutic biomarker. These results justify the development of synthetic lethality- based clinical trials targeting GBM with 9p21 loss. Future work will focus on functional validation of drug combinations and clinical trial design.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbcdf237ba9e5b742a8d73e7613b54470bb9dac1" target='_blank'>
              UNDERSTANDING PATTERNS OF CO-DELETION CDKN2A/B DELETED GBM TO SUPPORT A SYNTHETIC LETHALITY APPROACH TO TREATMENT
              </a>
            </td>
          <td>
            Tanishi Moitra, Michael Castro, Matt Williams
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The presence of the PRDM16::SKI fusion gene was described, for the first time, in a T prolymphocytic leukemia (T-PLL) patient with a long indolent period and a late development treatment requiring disease. The fusion transcript was detected by RNA sequencing and validated by reverse transcriptase polymerase chain reaction and Sanger/Cycle sequencing. The chimera occurs between exon 1 of the PR/SET Domain 16 (PRDM16) gene and exon 2 of the oncogene V-Ski Avian Sarcoma Viral Oncogene Homolog (SKI) gene. The finding provides insight into the role of genetic alterations, including fusion genes, in development and progression of T-PLL and may possibly lead to the development of effective and precise targeted therapy for this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/314985a5230a419836262a0347f8d007d1be6717" target='_blank'>
              Identification of a novel PRMD16::SKI fusion gene in T-prolymphocytic leukemia
              </a>
            </td>
          <td>
            M. Brunetti, H. K. Gjelberg, H. Reikvam, F. Micci
          </td>
          <td>2025-08-26</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Cervical cancer remains the fourth most common cancer among women globally, disproportionately impacting low- and middle-income countries despite the existence of HPV vaccines. While DNA methylation has been studied extensively as a biomarker, other epigenetic mechanisms remain underexplored. This scoping review aims to report such underexplored epigenetic biomarkers linked to cervical cancer, shifting the focus beyond global nuclear DNA methylation. Literature searches were performed using Google Scholar via Publish or Perish software including studies published until January 2025. Our review focused on mitochondrial DNA, non-coding RNA, histone modifications, and repetitive elements. Mitochondrial DNA methylation has been proposed as a cervical cancer biomarker, although supporting evidence is limited. Histone modifications are more consistently reported to be involved both in cervical cancer onset and aggressiveness. Similarly, aberrant expression of lncRNAs, circRNAs, miRNAs, and piRNAs has been associated with poor prognosis. Finally, hypomethylation in repetitive elements such as LINE-1 and Alu is often observed in cervical cancer, contributing to genomic instability and tumorigenesis. Highlighting these alternative epigenetic mechanisms, our review emphasizes the importance of expanding biomarker discovery beyond the traditional nuclear DNA methylation. Understanding these mechanisms may improve early detection and personalized disease management strategies for cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a7819cfc0864d5095515e01e909ecdbee12a80f" target='_blank'>
              Epigenetic Biomarkers for Cervical Cancer Progression: A Scoping Review
              </a>
            </td>
          <td>
            Efthymios Ladoukakis, Gracia Andriamiadana, Fatema Hajizadah, Lewis G. E. James, Belinda Nedjai
          </td>
          <td>2025-09-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary DNA repair mechanisms are crucial for maintanence of DNA and cell integrity. When disrupted, they can lead to increased mutation rate and contribute to the cancerogenesis. On the other hand, cancer cells that bear mutations in genes involved in DNA repair are more prone to DNA damaging insults which opens opportunities for treatments involving synthetic lethality. microRNAs are short non-coding RNAs that regulate expression by binding to gene transcripts and inducing mRNA degradation or inhibition of translation. Here, we review the miRNA-mediated dysregulation of genes involved in DNA damage response (DDR) and DNA repair pathways in ovarian cancer (OvCa), one of the deadliest female malignancy. We also discuss miRNAs, which affect response to OvCa therapy by regulating PARP1 (Poly(ADP-Ribose) Polymerase-1), a central gene of the OvCa synthetic lethality treatments. Finally, we address the limitations of miRNAs as diagnostic biomarkers and potential targets for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd4db6c3d00cf07b80c8f608dd0b112d9dfce41" target='_blank'>
              DNA Damage and Repair in Ovarian Cancer: Focus on MicroRNAs
              </a>
            </td>
          <td>
            Katarzyna D. Arczewska, A. Piekiełko-Witkowska
          </td>
          <td>2025-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Nuclear biomolecular condensates are essential sub-compartments within the cell nucleus and play key roles in transcription and RNA processing. Bottom-up construction of nuclear architectures in synthetic settings is non-trivial but vital for understanding the mechanisms of condensates in real cellular systems. Here, we present a facile and versatile synthetic DNA protonucleus (PN) platform that facilitates localized transcription of branched RNA motifs with kissing loops (KLs) for subsequent condensation into complex condensate architectures. We identify salinity, monomer feeding, and KL-PN interactions as key parameters to control co-transcriptional condensation of these KLs into diverse artificial nuclear patterns, including single and multiple condensates, interface condensates, and biphasic condensates. Over time, KL transcripts co-condense with the PN matrix, with the final architecture determined by their interactions, which can be precisely modulated using a short DNA invader strand that outcompetes these interactions. Our findings deepen the understanding of RNA condensation in nuclear environments and provide strategies for designing functional nucleus-mimetic systems with precise architectural control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13da8ed75a9cef66af63d63bcdff443dc8cb37d3" target='_blank'>
              Constructing synthetic nuclear architectures via transcriptional condensates in a DNA protonucleus
              </a>
            </td>
          <td>
            Miao Xie, Weixiang Chen, Maria Vonk-de Roy, Andreas Walther
          </td>
          <td>2025-09-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background/Objectives: Galectin-3 (Gal-3), encoded by LGALS3, is a β-galactoside-binding lectin involved in diverse tumor-associated processes, including immune modulation, cell cycle regulation, and stress adaptation. Despite its known roles in cancer biology, the full extent of its molecular functions and prognostic relevance across tumor types remains incompletely understood. This study aimed to systematically investigate the transcriptomic impact of LGALS3 deletion and assess its clinical significance in cancer. Methods: We analyzed CRISPR-Cas9 knockout transcriptomic data from the SigCom LINCS database to characterize the consensus gene signature associated with LGALS3 loss using functional enrichment analyses. Pan-cancer survival analyses were conducted using TIMER2.0. Differential Gal-3 protein levels in ductal adenocarcinoma and normal pancreatic tissues were evaluated using the Human Protein Atlas. Finally, functional analyses were performed in pancreatic ductal adenocarcinoma (PDAC). Results: LGALS3 deletion across multiple cancer cell lines led to transcriptomic changes involving mitotic progression, stress responses, and axonal guidance pathways. High LGALS3 expression was significantly associated with worse overall survival in lower-grade glioma, PDAC, uveal melanoma, and kidney renal papillary cell carcinoma. LGALS3 knockout in YAPC cells recapitulated the pan-cancer findings, linking LGALS3 to cell morphogenesis and proliferation. Conclusions: These findings identify Galectin-3 as a key regulator of oncogenic programs and a potential prognostic biomarker in PDAC and other malignancies, with implications for therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0894155b476f9fae4b8588df639cdd5d50fcb2b" target='_blank'>
              Transcriptomic and Clinical Profiling Reveals LGALS3 as a Prognostic Oncogene in Pancreatic Cancer
              </a>
            </td>
          <td>
            Grazia Scuderi, S. Mijatović, D. Maksimović‐Ivanić, M. Di Rosa, J. Muñóz-Valle, Alexis Missael Vizcaíno-Quirarte, Gian Marco Leone, K. Mangano, P. Fagone, Ferdinando Nicoletti
          </td>
          <td>2025-10-03</td>
          <td>Genes</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Detection of cancer at early stage can significantly improve the five-year survival rate of patients. Bisulfite-based methylation detection can cause DNA damage, especially in high GC content regions which was associated with the development of cancers. Loss of aberrant methylated CpG sites in cfDNA will lead to the undetectability of certain circulating tumor DNA (ctDNA), consequently may affect the cancer detection. Our study uses enzymatic method to detect whole genome abnormal methylation regions in esophageal squamous cell carcinoma (ESCC). We also provide a pretrained neural network, hybrid of BERT and CNN (BCNN), to identify ctDNA robustly. Maximum posterior probability is utilized to estimate the fraction of ESCC-derived ctDNA in plasma for predicting the risk of ESCC cancer. Our results analysis indicated that enzyme-based whole-genome methylation sequencing retained more longer cfDNA and detected more CpG sites than bisulfite-based method in both gDNA and cfDNA. Enrichment analysis of differentially methylated regions (DMR) showed that top five pathways were associated with ESCC. Compared to traditional models, our BCNN demonstrates the best performance in identifying ctDNA (AUC = 0.970). By estimating the fraction of plasma ctDNA, our BCNN exhibits high accuracy in ESCC detection even at ultralow sequencing depths (AUC = 0.946). Specifically, in the validation cohort, when the specificity is 93.75%, 7 out of 8 early-stage ESCC (TNM Stage I) were identified as positive in our preliminary results. In conclusion, our preliminary results reinforce the idea of employing BCNN as a novel strategy for ESCC early detection in clinical practice. However, to be applied in clinical, further validation with a larger sample size is necessary. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-18278-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5af110454f8624475f32847f585947aa544c81e" target='_blank'>
              Improved circulating tumor DNA identification for detection of esophageal squamous cell carcinoma by enzymatic methyl sequencing and hybrid neural network
              </a>
            </td>
          <td>
            Jiangfeng Shen, Yuqi Ren, Ziyong Mao, Qiuxiang Hu, Yilong Wan, Jiangcun Du, Yanting Lai, Shengxiang Shao, Liuqing Zhang, Hao Wu, Jiaxi Li, Sheng Ju, Xin Tong, Jun Zhao, Lei Cao, Deyi Xiong, ChengCheng Xu, Jun-Chi Xu, Dong Jiang
          </td>
          <td>2025-09-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Chronic myeloid leukemia (CML) can be effectively treated inhibiting the disease-causing BCR::ABL1 kinase by tyrosine kinase inhibitors (TKIs). Although therapy is initially tremendously successful, resistance may occur in up to 25% of CML patients. Besides aberrations in the BCR::ABL1 kinase domain, a variety of resistance mechanisms are currently discussed, among them epigenetic reprogramming. The histone-modifying enzyme lysine methyltransferase 2D (KMT2D/MLL2) belongs to the most frequently mutated genes in cancer and is also known for its association with hereditary Kabuki syndrome. However, its role in CML is widely unknown. In the present study, we analyzed the role of the KMT2D p. (Arg191Trp) variant in imatinib-resistant CML, which was recurrently acquired in imatinib resistance in vitro. SiRNA-mediated KMT2D knockdown, but also introduction of the p. (Arg191Trp) variant into treatment-naïve K-562 cells led to impaired imatinib susceptibility visible by increased cell numbers, proliferation rates and metabolic activities under imatinib exposure (p < 0.001). The effect of KMT2D p. (Arg191Trp) could be overcome by inhibiting histone demethylation with the demethylase inhibitor LSD1. In addition, rescue of KMT2D expression in imatinib-resistant cells reinstated the response to imatinib treatment. Furthermore, gene expression analysis revealed upregulation of CCNE2 in cells harboring KMT2D p. (Arg191Trp) potentially explaining increase in cell proliferation under imatinib exposure. Overall, our findings demonstrate that the loss of the tumor suppressor KMT2D promotes TKI resistance in CML. Thus, KMT2D status could serve as an additional biomarker for TKI resistance, while restoration of its expression might be a therapeutic option to overcome this resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59b4b4f7562cd8617674ef60daedf2504f6b6932" target='_blank'>
              The role of the lysine histone methylase KMT2D in chronic myeloid leukemia
              </a>
            </td>
          <td>
            Lina Schlemminger, Inga Nagel, I. Vater, Ingolf Cascorbi, Meike Kaehler
          </td>
          <td>2025-09-16</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="With the increase in dog life expectancy, canine mammary cancer has become the most critical and frequent neoplasm in dogs. As in humans, this type of cancer has spontaneous development and is influenced by several risk factors, such as age and hormonal exposure in addition to genetic and epigenetic factors. Thus, knowing that epigenetic patterns influence a variety of biological mechanisms, such development cancer, this study aimed to correlate the methylation pattern of LINE-1 elements in healthy and neoplastic canine mammary tissue, as well as in isolated cell lines of carcinoma canine with tumoral aggressiveness degree. To this end, a retrospective study was conducted in which genomic DNA was extracted from histological sections of biopsies and cell lines using the phenol/chloroform method. To determine the percentages, the genomic DNA was pretreated with methylation-sensitive endonucleases and then amplified by qPCR with primers specific for the L1td1 gene. The results revealed that both the tumor cell lines and the canine adenoma and inflammatory carcinoma samples are hypomethylated relative to normal tissue. However, it was not possible to confirm the relationship between the percentage of LINE-1 hypomethylation and tumor aggressiveness in canine breast cancer. Therefore, the combined analysis of our results allows us to conclude that the methylation pattern of LINE-1 elements can be used as a molecular biomarker to differentiate normal canine mammary tissue from neoplastic canine mammary tissue. However, to confirm whether there is a relationship between the degree of methylation of LINE-1 elements and tumor staging, further studies with canine samples are needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da3e474a09cd5273ac4d9d3cd5402967aa34da4d" target='_blank'>
              ASSOCIATION BETWEEN LINE-1 HYPOMETHYLATION IN CANINE MAMMARY TUMOR TISSUES: IMPLICATIONS FOR DIAGNOSIS AND RESEARCH
              </a>
            </td>
          <td>
            Patrícia A. dos Santos de Carvalho Joaquim, José Guilherme Xavier, M. Lallo, Rodrigo A. Silva
          </td>
          <td>2025-09-24</td>
          <td>Revista CPAQV - Centro de Pesquisas Avançadas em Qualidade de Vida</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Meiosis is a source of genetic variation in eukaryotes. Meiosis in the eukaryotic fission yeast Schizosaccharomyces pombe leads to the formation of spores that are particularly resistant to environmental stresses. In addition to external factors, internal processes may nevertheless contribute to cellular stress and impact the genome. This study investigates the role of Pnu1 as the major meiotic nuclease in S. pombe. Transcription and cellular expression of Pnu1 are regulated upon specific phases of meiosis while its mitochondrial localization is also altered during this process. As a result, Pnu1 induces fragmentation of both genomic and mitochondrial DNA in the post meiotic phase. This sugar-non-specific endonuclease generates random double-strand breaks across the genome, an activity that appears to be mediated by direct interaction with chromatin. Given the high spore viability (∼95%) and the widespread occurrence of this phenomenon, this fragmentation appears to be physiological rather than apoptotic as observed in mammals. EndoG is the mammalian homolog of Pnu1 and is a caspases-independent apoptotic endonuclease that can allow cell survival. This study further describes the dynamics of Pnu1 action and support the conclusion that Pnu1 is a major meiotic endonuclease of S. pombe responsible for a transient post-meiotic fragmentation of cellular DNA potentially contributing to genetic variability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fdc3ebfc992761b3e2bf6cac186bc029393e2fe" target='_blank'>
              Induction of post-meiotic DNA double-strand breaks by the Pnu1 endonuclease in Schizosaccharomyces pombe.
              </a>
            </td>
          <td>
            Loïs Mourrain, Tiphanie Cavé, G. Boissonneault
          </td>
          <td>2025-09-03</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Supercoiled (Sc) circular DNA, such as plasmids, are essential in molecular biology and hold strong therapeutic potential. However, they are typically produced in Escherichia coli, resulting in bacterial methylations, unnecessary sequences, and contaminants that hinder certain applications including clinical uses. These limitations could be avoided by synthesizing plasmids entirely in vitro, but synthesizing high-purity Sc circular DNA biochemically remains a significant technical challenge. To overcome this challenge, we have developed two novel biochemical methods for in vitro synthesis of Sc circular DNA. Linear DNA with two loxP sites in the same orientation is generated by polymerase chain reaction or rolling circle amplification. Cre recombinase efficiently converts the linear DNA into relaxed circular DNA. T5 exonuclease is then used to digest unwanted linear DNA, and topoisomerases are employed to generate Sc circular DNA. Using this approach, we synthesized EGFP-FL, a 2 kb mini-circular DNA encoding essential EGFP expression elements. EGFP-FL transfected HeLa and C2C12 cells with significantly higher efficiency than its E. coli-derived counterpart. These methods enable the efficient production of Sc circular DNA from 196 bp to several kb, and in quantities from micrograms to milligrams, providing a versatile, scalable, and bacteria-free platform for basic research and therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc6c6aa3cf66d79a56fe61e937482c31c5bc93bd" target='_blank'>
              Synthesizing supercoiled circular DNA molecules in vitro
              </a>
            </td>
          <td>
            Sepideh Rezaei, Monica Moncada-Restrepo, Sophia Leng, Jeremy W. Chambers, Fenfei Leng
          </td>
          <td>2025-09-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="This study examines the contribution of dysregulated long non-coding RNAs (lncRNAs) to disease progression and prognosis in breast invasive carcinoma (BRCA) and cervical squamous cell carcinoma with endocervical adenocarcinoma (CESC) using data from The Cancer Genome Atlas (TCGA). Comparative analyses identified both shared and cancer-specific lncRNAs linked to tumorigenesis. Survival analysis using the Kaplan–Meier method highlighted several candidates with prognostic relevance. Notably, FAM83H-AS1 showed increased expression in advanced stages of BRCA and was linked with lymph node involvement and metastatic status. Its elevated expression was associated with poor survival outcomes in both BRCA and CESC cohorts. In contrast, lncRNAs such as FGF14-AS2, which were downregulated, demonstrated links to tumor-suppressive mechanisms and improved survival in BRCA patients. Subcellular localization analysis indicated predominant cytoplasmic presence of FAM83H-AS1, implying possible roles in post-transcriptional regulation. To support our in-silico observations, we carried out qRT-PCR experiments in breast cancer cell lines with varying degrees of aggressiveness, which confirmed the differential expression patterns of these lncRNAs. Overall, our findings highlight the clinical importance of lncRNA dysregulation in gynaecological cancers and point to their potential utility as biomarkers for prognosis and targeted therapy. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-16436-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96a627efdc32c519561de180942228807a9ccfd0" target='_blank'>
              Functional characterization of dysregulated LncRNAs in BRCA and CESC highlights their role in cancer prognosis
              </a>
            </td>
          <td>
            Madhur Sharma, Nidhi Chourasia, Priyanka Priyanka, D. P. Sarkar, A. Nag, Sandeep Saxena
          </td>
          <td>2025-10-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The human genome is routinely subject to endogenous (e.g. oxidative stress) and exogenous (e.g. radiation, UV) factors that induce DNA lesions. These lesions may accumulate and impair cell function or cause cancer. Hence, DNA damage repair and preservation are prerequisites for normal cellular function and maintaining genome integrity. Current research suggests that long non-coding RNAs (lncRNA) play a role in DNA damage response (DDR). lncRNAs are RNA transcripts >200 nt that do not code for proteins. After transcription, they interact with DNA, protein, mRNA, or other non-coding RNA to influence gene expression and protein function. However, there is a lack of research concerning the role of lncRNA in DDR. This study aimed to investigate and quantify the expression of DNA-damage-responsive lncRNAs in a time-course experiment. To do so, human embryonic kidney 293T (HEK293T) cells were exposed to ionizing radiation (IR) at 2 Gy to induce 60-100 double-stranded breaks (DSB) per cell. Following this exposure, RNA was collected at 4 different time points (1, 4, 8, and 24 hrs). Finally, Real-time Quantitative PCR was performed to quantify the expression of select lncRNA genes. Our findings demonstrate an association between DDR and lncRNA gene expression. We found a temporal upregulation or downregulation of lncRNA genes, specifically ENSG00000250519, ENSG00000231595, ENSG00000254338, and ENSG00000223749. The influence of DNA lesions on the expression of these lncRNA increased over 24 hrs. This finding asserts that lncRNA is an epigenetic regulator that may influence signalling networks related to DNA damage. Hence, lncRNAs may serve as biomarkers for the diagnosis, treatment, and prognosis of cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7c3f3f72f4b73fe0339171d4fca65b2b98d68e4" target='_blank'>
              Investigating DNA-damage responsive long non-coding RNA (lncRNA)
              </a>
            </td>
          <td>
            Lindsay Yu
          </td>
          <td>2025-08-28</td>
          <td>Inquiry@Queen's Undergraduate Research Conference Proceedings</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The initiation and progression of tumors are closely linked to aberrant regulation of multiple genes. Among these genes, sister chromatid cohesion acetyltransferase 2 (ESCO2) has emerged as an important target in cancer research. ESCO2 exhibits complex and heterogeneous expression patterns across various types of malignancies and tumor progression stages. Extensive studies have shown that ESCO2 primarily functions in chromosomal cohesion, significantly contributing to genomic stability maintenance. Dysregulated expression of ESCO2 in tumor cells is often correlated with malignant behaviors. Given its essential role in tumorigenesis, ESCO2 has increasingly been explored as a promising therapeutic target in recent years. This review article summarizes recent research progress regarding ESCO2 in malignant tumors, aiming to identify a novel target for their treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s40001-025-03164-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79b1b540beced4302a863f075307c9fa3cb6930a" target='_blank'>
              The expression of establishment of cohesion 1 homolog 2 (ESCO2) in tumor cells and its research progress as a therapeutic target
              </a>
            </td>
          <td>
            Chunmei Lu, Hanchi Wu, Huiyu Wang
          </td>
          <td>2025-10-06</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lymphoma is a highly heterogeneous malignancy, demanding accurate and precise diagnosis to guide the selection of the appropriate treatment for optimal outcome. Copy number aberration (CNA) has been suggested to play an important role in the occurrence and development of lymphoma and thus can be explored as biomarker to improve disease management. It is believed that CNAs in variable forms and complexities can be triggered by both exogenous (eg viral infection and ionizing radiation) and endogenous factors (eg genetic predisposition and evolutionary forces). However, conventional cytogenetic methods have limitations to detect all types of CNAs with accuracy and adequate details. The emergence of new technologies, including fluorescence in situ hybridization (FISH), chromosome microarray analysis (CMA), and especially next-generation sequencing (NGS) has made significant progress in the identification and characterization of CNAs or CNA-related genomic aberrations. Accumulating data addressing molecular insights and clinical implications have provided us more theoretical and experimental support for its clinical translation. Currently, while only limited number of CNAs or CNA-related genomic variation, such as deletion/amplification of DNA segments, have been documented in major guidelines or consensus for their clinical potential in lymphoma, more CNAs remain to be further characterized and/or discovered for their clinical relevance. Taking together, with available and upcoming evidence, CNA should play an important role as a diagnostic and prognostic biomarker while integrated with the current settings in lymphoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56be82adf67a41f68e1acf08975fb72e997dedc3" target='_blank'>
              Clinical Potential of Copy Number Aberration as a Diagnostic and Prognostic Biomarker in Lymphoma.
              </a>
            </td>
          <td>
            Xudong Zhang, Zailin Yang, Susu Yan, Minning Zhan, Shichun Tu, Weihong Ren, Yao Liu, Zun‐M. Zhu
          </td>
          <td>2025-06-01</td>
          <td>Technology in cancer research & treatment</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Small cell carcinoma of the esophagus (SCCE) is a rare and highly aggressive malignancy with limited therapeutic options and poor prognosis. The paucity of clinical specimens and lack of established experimental models have hindered a comprehensive understanding of its cellular heterogeneity and tumor microenvironment.We performed single-cell RNA sequencing on SCCE samples, and integrated them with publicly available scRNA-seq datasets from esophageal squamous cell carcinoma (ESCC), esophageal adenocarcinoma (EAC), and adjacent normal tissues (NT) from ESCC and EAC cases. An integrative transcriptomic analysis was conducted to identify cell types, infer malignant states, reconstruct differentiation trajectories, evaluate immune landscapes, and investigate fibroblast subtypes and cell–cell communication networks.SCCE tumors were characterized by a predominance of malignant epithelial cells and exhibited a profoundly immunosuppressed phenotype, with reduced immune infiltration and widespread downregulation of immune checkpoint genes. Malignant epithelial cells showed pronounced chromosomal instability and were classified into three transcriptionally distinct subtypes with divergent differentiation trajectories. The tumor microenvironment featured a complex stromal compartment, with enrichment of extracellular matrix fibroblasts (eCAFs) characterized by elevated ELF3 regulatory activity, and collagen-driven signaling predominantly mediated by inflammatory CAFs (iCAFs). SCCE also showed the most intricate cell–cell communication network among esophageal cancer subtypes.Our single-cell atlas offers a detailed view of the cellular heterogeneity and microenvironmental complexity of SCCE, highlighting its distinct tumor architecture, immune exclusion, and stromal reprogramming. These findings provide a valuable resource for understanding SCCE biology and form a basis for future mechanistic and exploratory biological investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6db8bbf656956c2fda0265e682ad202bf970eea8" target='_blank'>
              Single-cell transcriptomic analysis reveals epithelial and microenvironmental heterogeneity in small cell carcinoma of the esophagus
              </a>
            </td>
          <td>
            Xiaolei Yin, Xiaopeng Li, Lili Mi, Jiaojiao Hou, Fei Yin
          </td>
          <td>2025-10-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60e9636174b92b528395146a5a9794249738dcd9" target='_blank'>
              Excavation of Molecular Subtypes of Cervical Cancer Based on DNA Methylation Patterns.
              </a>
            </td>
          <td>
            Yiwei Zhao, Chutong Zhao, Jiyun Zhao, Yuhan Ma, Shunjin Zhang, Yujie Liu, Yuan Wang, Sijia Liu, Yunyan Zhang
          </td>
          <td>2025-09-26</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Methylation of DNA CpG domains in cellular DNA is a key mechanism of epigenetic regulation. Disruptions in the processes maintaining DNA methylation can lead to diseases like cancer. The radiation response of certain cancer cells may be affected by their DNA methylation levels, which may have consequences in their response to radiotherapy. In this work, we utilized DNA origami nanotechnology to examine whether DNA methylation impacts DNA response to ionizing radiation in solution before biological processes come into play. Our findings reveal that a protective effect is achieved with just a few methylated CpG adducts. Both low-LET (electron) and high-LET (carbon ion) irradiation show a reduced lesion count in methylated DNA, as indicated by qPCR results. AFM single-molecule observations using DNA origami nanoframes suggest fewer double-strand breaks in methylated DNA after carbon ion irradiation. This radioprotective effect may contribute to the differential radiation response of cellular DNA and should be considered when predicting and evaluating DNA radiation damage yields.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60446411a5ce1e0fa6ca09e48e25c2f668acc1a4" target='_blank'>
              CpG Methylation Protects DNA against Ionizing Radiation
              </a>
            </td>
          <td>
            Leo Sala, Tereza Zápotocká, Jana Šáchová, Václav Olšanský, David Chvatil, François Chevalier, Violaine Vizcaino, Alain Mery, J. Kočišek
          </td>
          <td>2025-08-20</td>
          <td>The Journal of Physical Chemistry. B</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Recombinases are essential enzymes in synthetic biology and biomedical research, enabling site-specific DNA modifications for applications such as gene therapy, the generation of transgenic models, and the construction of genetic circuits. Recombination efficiency depends on several factors, including intracellular recombinase concentration and the growth phase of the host cells. Although recombination is typically studied during exponential growth, the effects of stationary-phase dynamics on recombinase activity remain poorly understood. In this study, we examine how bacterial growth phase influences recombinase-mediated DNA modifications, using the serine recombinase Bxb1 as a model. We engineered a genetic system in Escherichia coli to quantify intracellular Bxb1 levels and to measure recombination efficiency across different growth phases. Our results reveal a quasi-linear relationship between recombinase concentration and recombination efficiency during exponential growth, up to a saturation point. Notably, recombination continues in the stationary phase following recombinase induction in exponential phase, despite the decline in plasmid gene expression. Cells that undergo recombination during the stationary phase show significantly higher recombination efficiencies upon re-entering exponential growth than those maintained in exponential phase throughout. These findings highlight the importance of induction timing in optimizing recombinase-based genetic modifications. Specifically, inducing recombinase expression just before the onset of stationary phase can enhance recombination efficiency while minimizing the need for high expression levels. Moreover, the observed quasi-linear relationship during exponential growth provides a framework for tuning gene expression with precision. Overall, this work offers new insights into leveraging bacterial growth dynamics to improve the design and control of synthetic genetic systems. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17024-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6df10f0c8f4691b2c66e4226a8d26ecc0ab6dc10" target='_blank'>
              Characterization of recombinase activity across cellular growth phases
              </a>
            </td>
          <td>
            M. Gonzalez-Colell, J. Macía
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6922da00f94f2dd9a584ee514f3fa6786a061103" target='_blank'>
              Genetic landscape of DNMT3A R882H reveals mechanism of aberrant oligomerization.
              </a>
            </td>
          <td>
            Emma M. Garcia, Olivia R Lavidor, Shaunak Raval, Nicholas Z. Lue, Jessica K Liang, Maeve McFadden, Derek D Hwang, Marcanthony U Zepeda, Luke W Chen, Steven A. Carr, Malvina Papanastasiou, Brian B Liau
          </td>
          <td>2025-09-25</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Despite numerous studies investigating intratumoral microorganisms and their significant roles in cancer initiation, progression, and treatment efficacy, a systematic understanding of intratumoral microorganisms remains lacking. Herein, we conducted a study using 16S rDNA data on seven types of cancer, comprising a total of 783 samples. It’s worth noting that Pseudomonas, Streptococcus, and Prevotella were found to be shared with the microbial communities of the seven cancers, suggesting that these may be associated with the occurrence and development of cancers. We anticipate establishing a foundation for related research and exploring potential methods for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80274119fe0660855f0a9a793c84d8230ca38ecb" target='_blank'>
              Pan-cancer analysis reveals intratumoral microbial diversity in multiple cancers by amplicon technology
              </a>
            </td>
          <td>
            Xudong Liu, Yuteng Yao, Aoyi Xiao, Dingyan Cao, Jingheng Zhang, Yanan Shi, Qing Zhong, Zilong He, Wenming Wu
          </td>
          <td>2025-09-02</td>
          <td>Frontiers in Cellular and Infection Microbiology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Rad51 is a pivotal DNA repair protein that plays a central role in homologous recombination repair. Its significance in tumorigenesis and cancer treatment resistance has garnered increasing attention. In recent years, studies have shown that Rad51 not only plays an important role in the biological characteristics of tumor cells but is also closely related to tumor immunity. The complexity of the tumor immune microenvironment makes the action mechanism of Rad51 more complex and affects the immune system's ability to recognize and clear tumors by influencing the interaction between tumor cells and immune cells. Although preliminary studies have suggested a potential role of Rad51 in tumor immunity, the underlying molecular mechanisms require further elucidation. Furthermore, the research on Rad51 as a therapeutic target is also continuously deepening, indicating its application potential in tumor treatment. In summary, this review aims to comprehensively analyze the role of Rad51 in tumor immunity and its challenges and opportunities in future tumor treatment research and clinical application in order to provide new ideas for clinical tumor immunotherapy. However, the current understanding of Rad51's immunomodulatory functions remains incomplete, and further preclinical and clinical studies are needed to validate its therapeutic potential, address potential off‐target effects, and elucidate tumor‐type‐specific mechanisms and microenvironmental interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00a9c3b142aaabf70111312c22aa00050983254" target='_blank'>
              Rad51 at the Crossroads: Bridging DNA Repair, Tumorigenesis, and Immune Regulation
              </a>
            </td>
          <td>
            Zhendong Qin, Mingjun Lu, Jingwei Guo, Chenyang Wang, Teng Ma
          </td>
          <td>2025-09-24</td>
          <td>Med Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Background Unrepaired DNA damage leads to the accumulation of cytosolic DNA and the activation of cGAS-STING-mediated inflammatory response, which modulates the tumor immune microenvironment and immunotherapy response. NPRL2 (NPR2 like, GATOR1 complex subunit) is located on chromosome 3p. Deep deletion of NPRL2 was identified in 10% of ccRCC, and is often mutually exclusive with loss of PBRM1, SETD2, and BAP1. However, it remains unclear whether NPRL2 loss independently influences DNA damage and anti-cancer immunity in ccRCC. Methods Both lentiviral shRNA-mediated knockdown and CRISPR/Cas9-mediated knockout were performed in human and murine RCC cell lines to suppress NPRL2 expression. The immunocompetent Renca-Balb/c murine RCC model was used to evaluate therapy response. The tumor immune microenvironment was analyzed using gene expression-based inference and immunohistochemistry (IHC). Immunofluorescence, western blotting and real-time PCR were used for molecular studies. Results Guided by our reverse phase protein array (RPPA) results, we first found that NPRL2 loss activated the ATM-CHK2 DNA damage response pathway and led to the accumulation of cytosolic DNA. The lysosome is known to play important roles in DNA repair and cytosolic DNA degradation. We further found that NPRL2 loss promoted the phosphorylation of TFEB, the master regulator of lysosome biogenesis, and thus prevented its translocation to the nucleus. Pharmacologic activation of TFEB with ML-SA1 reduced ATM phosphorylation and cytosolic DNA accumulation, and lysosomal inhibition by chloroquine or bafilomycin A1 yielded the opposite results. However, genetic inhibition of autophagy by knocking down ATG5, ATG7, BECN1, or p62 had no apparent impact. The increased cytosolic DNA led us to hypothesize that NPRL2 deficiency would consequently activate the cGAS-STING pathway. Unexpectedly, NPRL2 deficiency suppressed the cGAS-STING-mediated inflammatory response, as evidenced by dramatic decreases in TBK1 and IRF3 phosphorylation, and in downstream gene expression (eg, CCL5, CXCL10 and IFNB1). To investigate the underlying mechanism, we first analyzed the expression of the key molecules that are involved in the cGAS-STING pathway. It turned out that NPRL2 loss reduced the expression of STING at both mRNA and protein levels. By analyzing the KIRC TCGA dataset, we found that low NPRL2 expression was associated with reduced T cell infiltration and immune checkpoint (CTLA4 and PDCD1) expression, reflecting an immunologically cold tumor microenvironment, which may respond poorly to immunotherapy. To test this hypothesis, we knocked down the expression of Nprl2 in Renca cells and found that Nprl2 loss reduced the complete response rate to anti-PD1 therapy in a syngeneic tumor model. However, NPRL2/Nprl2 loss sensitized RCC cells and Renca tumors to the ATM inhibitor KU60019, while the combination of KU60019 and anti-PD1 therapy showed no evidence of synergistic effect. Conclusions These results indicate that NPRL2 loss regulates DNA damage and cytosolic DNA accumulation via lysosomal but not canonical autophagic inhibition, and on the other hand, silences cytosolic DNA response by suppressing STING expression. NPRL2 loss defines a genomically unstable but nonimmunogenic tumor phenotype associated with checkpoint inhibitor resistance in RCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af6b45a93450c0576e0305c391ec88eb6a795e2a" target='_blank'>
              56NPRL2 loss enhances DNA damage while isolating cGAS-STING pathway and conferring immunotherapy resistance
              </a>
            </td>
          <td>
            Xiande Liu, Muhammed Onel, Hongchao He, Xuesong Zhang, Yanting Zhang, Anh Hoang, Guang Peng, E. Jonasch
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies, with limited treatment options and poor prognosis. Recent advances in cancer genomic analysis enable the identification of actionable gene alterations, opening new opportunities for personalized therapy. Among these, homologous recombination DNA repair (HRR) gene alterations are associated with distinct biological behavior, favorable prognosis, and increased sensitivity to platinum-based chemotherapy. However, the prognostic impact of coexisting mutations in key driver genes—KRAS, TP53, CDKN2A, and SMAD4—within HRR-altered PDAC remains poorly understood. Methods We retrospectively analyzed PDAC patients who underwent genomic profiling testing with FoundationOne® CDx between June 2019 and December 2021 through the Center for Cancer Genomics and Advanced Therapeutics (C-CAT) database. We compared the prevalence and prognostic significance of key gene alterations between HRR-altered and HRR–wild-type (WT) tumors. Results Of 2,381 PDAC patients, 274 (11.5%) harbored HRR alterations. These patients showed significantly longer overall survival (OS) than those with HRR-WT tumors (HR = 0.66, p = 0.002). The frequencies of KRAS, TP53, and CDKN2A mutations were less frequent in HRR-altered tumors. TP53 mutation was independently associated with poorer OS across both HRR subgroups, while CDKN2A alteration was a poor prognostic factor in HRR-WT tumors. Interestingly, SMAD4 alteration was linked to improved survival in the HRR-altered group. Conclusion HRR-altered PDAC has a distinct genomic profile and is associated with a favorable prognosis. Our findings demonstrate that coexisting alterations are significant prognostic factors in both HRR-altered and HRR–wild-type tumors. These results highlight the clinical relevance of incorporating comprehensive genomic profiling into routine care to stratify patient prognosis better and inform individualized treatment strategies in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada42c90fd2263fd4d9bb1466e612ccff4b0a39f" target='_blank'>
              Prognostic impact of gene alterations via homologous recombination DNA repair gene alteration status in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Yusuke Kawanaka, Chiaki Inagaki, Masaki Okura, S. Mitani, T. Takahama, K. Yonesaka, Y. Chiba, Kazuhiko Nakagawa, H. Kawakami, H. Hayashi
          </td>
          <td>2025-08-25</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by a pronounced inter- and intra-tumoral heterogeneity, which fuels chemoresistance and contributes to high mortality rates. Previously, we have developed a branched organoid system embedded in collagen matrices that robustly recapitulates the phenotypic heterogeneity seen in both murine and human PDAC1,2. These organoids display complex, self-organized branching morphogenesis and give rise to distinct, spatially ordered tumor cell populations that reflect their underlying molecular profiles and differentiation states. Importantly, we show that the observed heterogeneity is not random but governed by defined transcriptional programs, particularly epithelial-to-mesenchymal plasticity, that drive the emergence of discrete tumor-cell states. Using integrated phenotypic and transcriptomic profiling, we map this diversity to specific biological functions in vivo, demonstrating that each organoid phenotype corresponds to a tumor-cell state with unique metastatic potential, and therapeutic vulnerabilities. Moreover, we identify dynamic, treatment-induced phenotype reprogramming events that are targetable, paving the way for rational design of state-specific therapeutic interventions. Building on our findings, we are now focusing on elucidating intra-organoid heterogeneitysuch as tip–trunk hierarchies at the single-cell level, to uncover the mechanisms of self-organization and to determine how distinct organoid phenotypes and subpopulations contribute to liver metastasis and treatment resistance. In summary, we have established a scalable and mechanistically informative organoid platform that enables in vitro modeling of PDAC heterogeneity. This system provides a framework for dissecting the tumor cell–intrinsic drivers of phenotypic plasticity and for developing phenotype-guided treatment strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47eb75a781046a1e97c826bc3797a33d46a03686" target='_blank'>
              LEVERAGING ORGANOID TECHNOLOGIES TO CHARACTERIZE DYNAMIC TUMOR CELL STATES DRIVING TREATMENT RESISTANCE IN PANCREATIC CANCER
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e980a76e192e63af260762d40719bac841d722f3" target='_blank'>
              REV1 inhibition enhances trinucleotide repeat mutagenesis.
              </a>
            </td>
          <td>
            Ava Siegel, Daniel Almstead, Naveen Kothandaraman, Jessica Reich, Erica N. Lamkin, Josh A Victor, Aarzoo Grover, Kanayo Ikeh, Hannah Koval, Andrew Crompton, Hongjun Jang, Hyejin Lee, Roxana Del Rio Guerra, Dmitry M Korzhnev, M. Kyle Hadden, Jiyong Hong, Pei Zhou, Nimrat Chatterjee
          </td>
          <td>2025-09-16</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="p53 protein is a nuclear phosphoprotein that is a critical tumor suppressor, playing a key role in regulating the cell cycle and initiating apoptosis in response to DNA damage. As a transcription factor, it also activates genes involved in DNA repair and cell cycle arrest. H2AX is a histone H2A variant, which is vital for detecting DNA double-strand breaks. When phosphorylated at Serine 139, it forms γH2AX, which recruits DNA repair proteins to damage sites. The interaction between p53 and γH2AX is central to the DNA damage response, where p53 activates repair pathways and γH2AX flags the DNA lesions. It is known that impairing γH2AX while preserving p53 activity may slow cancer progression. Towards understanding this, graphene quantum dots (GQDs) offer a promising solution for tracking γH2AX and analyzing DNA damage, where they can help visualize it by investigating how p53 contributes to DNA repair at sites marked by γH2AX. This study examines the interactions between γH2AX and p53 with three different-sized two-layered GQDs (2 × 3 nm, 5 × 6 nm, and 8 × 9 nm) using the Molecular Dynamics (MD) approach. Our analysis revealed that both proteins adsorbed strongly to the 5 × 6 nm and 8 × 9 nm GQDs, with 5 × 6 nm GQD having the highest stability, making it a key candidate for future biosensing and cancer research, whereas the 8 × 9 nm GQD has the greatest potential to denature the proteins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8ed19d2dae3c37646fedd7fca65404638a0e6b3" target='_blank'>
              Size-Dependent Interactions of γH2AX and p53 Proteins with Graphene Quantum Dots
              </a>
            </td>
          <td>
            Hassan Arif, Sachi Mehta, Isaac Macwan
          </td>
          <td>2025-08-30</td>
          <td>Micro</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Oxidative damage can introduce G>T mutations upon DNA replication. When this damage occurs ex vivo, sequenced DNA exhibits strand asymmetry, whereby sequence alignment yields G>T mismatches without corresponding C>A mismatches on the complementary strand at a given locus. Strand asymmetry is used to identify potential sequencing artefacts in somatic variant calls in cancer sequencing projects. Consistent with previous studies, we found that the strandedness of this asymmetry is frequently shared across targeted capture regions. However, while some exome sequencing datasets displayed consistent asymmetry relative to the forward reference strand, some surprisingly showed asymmetry relative to the transcription strand. Though oxidation is the principle cause of artefactual G>T mutations, we propose that the asymmetry stems from the use of single-stranded exome capture probes, as we did not find consistent asymmetry in matched whole genome sequencing. We further propose that high levels of asymmetry can be indicative of oxidation artefacts in the reported somatic variant calls of some samples. While most analysed cohorts showed low to moderate asymmetry, in one cohort (testicular germ cell tumour), approximately half of the reported G>T somatic mutations were likely to be oxidative damage artefacts, as indicated by the extent of asymmetry in mismatches and variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc0c4c9812d816e151ed2205702a319bfce4b361" target='_blank'>
              Consistent asymmetry in DNA damage artefacts across target regions in exome sequencing data
              </a>
            </td>
          <td>
            Tyler D Medina, Declan Bennett, Cathal Seoighe
          </td>
          <td>2025-07-04</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Both DNA methylation and homologous recombination (HR) are extensively studied. In bacteria, Dam methylation is the most studied DNA modification, while RecA-mediated HR is a primary mechanism to repair DNA damages including double-stranded breaks, single-stranded gaps, and stalled replication forks. While HR regulation by proteins is extensively studied, whether methylation of DNA itself directly affects the functions of RecA and HR remains unclear. Mainly by single-molecule experiments, we report that Dam methylation of single-stranded DNA (ssDNA) promotes RecA assembly, partially by reducing the effective charge of ssDNA under counterion screening. Furthermore, Dam methylation of double-stranded DNA promotes homologous pairing, joint molecule growth, and strand exchange. In cellular experiments, dam deletion impairs HR, whereas hypermethylation of the adenines in the genome enhances HR in P1 transduction assays and DNA-damage sensitivity tests without significantly upregulated HR-related genes. In addition, the preference of RecA for Dam-methylated DNA in RecA assembly and homologous pairing is conserved across divergent species covering a gram-negative bacterium Klebsiella pneumoniae, a gram-positive bacterium Bacillus subtilis, and a flowering plant Arabidopsis thaliana. Dam methylation of ssDNA increases the ATPase activity of molecular motors such as RecQ helicase that containing RecA-like domains. These findings reveal effects of DNA methylation and mechanisms regulating RecA-mediated HR and molecular motors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a366efaa4f71dbaa4fd5242b1fedea2c3fbc0149" target='_blank'>
              N6-methyladenine modification of DNA enhances RecA-mediated homologous recombination.
              </a>
            </td>
          <td>
            Xiaocong Zhao, Bin Wu, Ya-Jun Yang, Ying Li, Qi-Yuan Qiu, Liu Wang, Ya-Peng Xu, Han Gong, Lun Song, Xuejie Wang, Jing-Yao Shi, Xuefeng Chen, Shao-Ran Zhang, Qi Zong, Liang Dai, Shi-Shen Du, Yan Zhang, Wenqiang Wu, Xinghua Zhang
          </td>
          <td>2025-08-20</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The EZH2 histone methyltransferase inhibitors tazemetostat and valemetostat have recently received approval for clinical use in follicular lymphoma and adult T-cell leukemia/lymphoma, respectively. In follicular lymphoma, the expression of AID is responsible for increased mutational signatures and genomic instability. Because EZH2 inhibitors induce epigenetic and transcriptional changes in normal and lymphoma B cells, we studied whether these inhibitors could alter the pattern of AID-mediated chromosomal translocations. Here we show that treatment with EZH2 inhibitors did not significantly change AID expression or AID-dependent chromosomal translocation frequency when used as monotherapyin either CH12F3 mouse B cells or MEC-1 human B cells. In contrast, when combined with PI3Kδ inhibition, which enhances AID expression, EZH2 inhibition significantly increased the frequency of chromosomal translocations compared to either EZH2 or PI3Kδ inhibition alone both in mouse CH12F3 cells and human MEC-1 cells. EZH2 inhibition also further enhanced translocation formation in mouse B cells that were Ligase4 deficient. Mechanistically, EZH2 inhibition in B cells depletes the repressive histone modification H3K27me3 while concurrently enhancing the active histone modification H3K27ac, thereby selectively increasing transcriptional activity and facilitating chromosomal translocation formation in the presence of high AID activity or Ligase4 deficiency. These findings highlight the impact of drugs that induce epigenetic changes to influence chromosomal translocations and demonstrate the genetic safety of EZH2 inhibitors as monotherapy while highlighting the increased risk of genomic instability when used in cells prone to translocations, such as B cells with high AID levels or DNA-repair deficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7976efe4db0e0c962e5f359566e424689b7f04c0" target='_blank'>
              Epigenetic changes by EZH2 inhibition increase translocations in B cells with high AID activity or DNA repair deficiency.
              </a>
            </td>
          <td>
            Jianli Tao, Luca Alessandri, Alessandro Gasparetto, Lijuan Zhao, Xuefei Zhang, Frederick W Alt, Roberto Chiarle
          </td>
          <td>2025-08-18</td>
          <td>Blood</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="This case study describes the clinical course of a 39-year-old woman with metastatic breast cancer harboring a germline PALB2 mutation who was treated with a PARP inhibitor. She initially demonstrated a clinical benefit with reduced tumor markers and favorable imaging findings. However, disease progression occurred after eight months. Liquid biopsy-based genomic profiling identified three PALB2 reversion mutations that restored homologous recombination, leading to treatment resistance. The case illustrates both the therapeutic potential of PARP inhibitors in PALB2-mutated cancers and the emergence of resistance. It emphasizes the importance of liquid biopsy-based genomic profiling for understanding tumor evolution and guiding treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74b996b21828b366858097de357e386b35708788" target='_blank'>
              Emergence of PALB2 Reversion Mutations as a Mechanism of Resistance to Niraparib in Breast Cancer: A Case Report.
              </a>
            </td>
          <td>
            Maako Kawamura, Hiroshi Tada, Hidekazu Shirota, Miki Dobashi, Noriko Takenaga, H. Yasojima, Narumi Harada-Shoji, K. Komine, Kenichi Nakamura, Minoru Miyashita, Hisato Kawakami
          </td>
          <td>2025-10-06</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Esophageal squamous cell carcinoma (ESCC) is a highly aggressive malignancy with poor prognosis and limited treatment options. While circular RNAs (circRNAs) are frequently dysregulated in ESCC, their functional roles and molecular mechanisms in tumor progression remain largely unexplored. Methods We characterized circRNA using RT-PCR, Sanger sequencing, and fluorescent in situ hybridization. Gain- and loss-of-function studies in vitro and in vivo were performed to assess circRNA function. Molecular interactions were investigated via RNA pull-down assays coupled with mass spectrometry, electrophoretic mobility shift assays, co-immunoprecipitation, and immunoblot analysis. Clinical relevance was evaluated by RT-qPCR and immunohistochemistry in patient specimens. Results We identified circLNF, a novel circRNA derived from the long non-coding RNA FIRRE gene, which is significantly upregulated in ESCC. Functional assays demonstrated that circLNF promotes proliferative and migratory capacities in cultured cells and accelerates tumor progression and metastasis in animal models. Mechanistically, circLNF directly interacts with IGF2BP family proteins through their “CAUC” motifs, protecting them from TRIM25-mediated ubiquitination and proteasomal degradation. Importantly, circLNF expression positively correlated with IGF2BP1 protein levels in ESCC patient tissues, underscoring the clinical relevance of the circLNF-IGF2BP axis in ESCC progression. Conclusions Our findings reveal an oncogenic role of circLNF in ESCC by inhibiting TRIM25-mediated proteasomal degradation of IGF2BP proteins and highlight circLNF as a potential therapeutic target for ESCC. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02442-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fde46bb0cd6a0e4ea2cb6ea52b1582df490fa428" target='_blank'>
              A circRNA promotes esophageal squamous cell carcinoma progression by inhibiting TRIM25-mediated degradation of IGF2BP family members
              </a>
            </td>
          <td>
            Shuangyan Tan, Yue Ming, Jiawei Guo, Wenrong Liu, Hulin Ma, Yu Liu, Zhijie Xu, A-Lai Gu-Ha, Lunzhi Dai, Yi-Dan Lin, Yong Peng
          </td>
          <td>2025-10-03</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35317e7a4cee78e23641ba79f278922203278977" target='_blank'>
              A chemical-genetic interaction between PAF1 and ENL/AF9 YEATS inhibition.
              </a>
            </td>
          <td>
            Paige A. Barta, Leopold Garnar-Wortzel, T. Bishop, Rachel E Hayward, Lauren M. Hargis, James B. Shaum, H. Kwok, Brian B Liau, Benjamin F. Cravatt, Michael A. Erb
          </td>
          <td>2025-09-10</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2ab9d9615fda47177afd65cb9d942ab1bd6dc87" target='_blank'>
              Thymic epithelial cells amplify epigenetic noise to promote immune tolerance.
              </a>
            </td>
          <td>
            Noah Gamble, Jason A Caldwell, Joshua McKeever, Caroline Kaiser, Alexandra Bradu, Peyton J. Dooley, Sandy Klemm, William J. Greenleaf, Narutoshi Hibino, Aaron R. Dinner, Andrew S. Koh
          </td>
          <td>2025-08-20</td>
          <td>Nature</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background The advent of poly (ADP-ribose) polymerase inhibitors (PARPi) over the past decade has significantly altered the management of epithelial ovarian cancer (EOC). We proposed that the etiology of homologous recombination deficiency (HRD) might underlie the variable responses to PARPi observed across patient populations. Methods As part of the phase 2 study of the Chinese HRD Harmonization Project, we developed a genomic methylation sequencing (GM-seq) pipeline facilitated by the TET enzyme for the simultaneous identification of methylated modifications and genetic variations in EOC tumor samples, and compared with established DNA sequencing-based HRD assays. Results Somatic mutation and HRD scores were confounded by low tumor purity in our cohort of 98 locally advanced/advanced EOC patients. In samples with tumor purity ≥ 30% (n = 45), the GM-seq pipeline showed high consistency with DNA sequencing-based HRD assay, identifying genetic variations in homologous recombination repair (HRR) genes and HRD score with 92.6% (25/27) and 97.1% (33/34) consistency respectively, in addition to conducting methylation profiling. Moreover, different underlying mechanisms of HRD were associated with varying degrees of PARPi efficacy, with BRCA1/2 LOH group having the best efficacy (median PFS, undefined), followed by BRCA1 methylation group (median PFS, 23.4 months), and those with unknown etiology of HRD having the worst efficacy (median PFS, 8.8 months, p < 0.001). Conclusion Our findings underscore the importance of considering HRD etiology when evaluating PARPi efficacy in EOC patients. The GM-seq pipeline, represents a significant advancement in HRD detection, enabling more accurate predictions of PARPi response. Supplementary Information The online version contains supplementary material available at 10.1186/s40364-025-00843-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5213220b5e2f3e35a5d8562e34edf897da4714a0" target='_blank'>
              Comprehensive assessment of homologous recombination deficiency via simultaneous methylation and mutation analysis in epithelial ovarian cancer: implications for PARP inhibitors efficacy
              </a>
            </td>
          <td>
            Lin Dong, Huanwen Wu, Ning Li, Wenbin Li, Yan Song, Yuanyuan Xiong, Huan Yin, Huan Fang, Rongrong Chen, Xin Yi, Jie Huang, Jianming Ying
          </td>
          <td>2025-10-10</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer is the second leading cause of death worldwide, with an estimated 27.5 million new cases projected by 2040. Disruptions in cell cycle control cause DNA replication errors to accumulate during cell growth, leading to genomic instability and tumor development. Proteins that regulate cell cycle progression and checkpoint mechanisms are crucial targets for cancer therapy. NIMA-related kinases (NEKs) are a family of serine/threonine kinases involved in regulating various aspects of the cell cycle and mitotic checkpoints in humans. Among these, NEK10 is the most divergent member and has been associated with both cancer and ciliopathies, a group of disorders caused by defects in cilia structure or function. Despite its biological significance and distinctive domain architecture, the structural details of NEK10 remain largely unknown. To address this gap, we employed computational modeling techniques to predict the complete structure of the NEK10 protein. Our analysis revealed a catalytic domain flanked by two coiled-coil domains, armadillo repeats (ARM repeats), an ATP binding site, two putative ubiquitin-associated (UBA) domains, and a PEST sequence known to regulate protein degradation. Furthermore, we mapped a comprehensive interactome of NEK10, uncovering previously unreported interactions with the cancer-related proteins MAP3K1 and HSPB1. MAP3K1, a serine/threonine kinase and E3 ubiquitin ligase frequently mutated in cancers, interacts with the catalytic region of NEK10. The interaction with HSPB1, a molecular chaperone associated with poor cancer prognosis, is mediated by NEK10's ARM repeats. Our findings highlight a potential connection between NEK10, ciliogenesis, and cancer, suggesting an important role in cancer development and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84bfc7864763ec8c742e96b5b16848a5dc021a94" target='_blank'>
              In Silico Analysis of Human NEK10 Reveals Novel Domain Architecture and Protein-Protein Interactions.
              </a>
            </td>
          <td>
            A. S. Eichner, Nathaniel Zimmerman, Avdar San, Shaneen Singh
          </td>
          <td>2025-10-03</td>
          <td>Proteins</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Colorectal cancer is one of the most prevalent malignant tumors worldwide, and it has one of the highest rates of mortality and morbidity. Its emergence and progression are intricately linked to various genetic and epigenetic factors influencing colonic epithelial cells. Long non-coding RNAs function as crucial regulators of the epigenetic landscape, playing significant roles in the development, progression, invasion, metastasis, and drug resistance of various tumors, particularly colorectal cancer. Epigenetic modifications, such as DNA methylation, RNA methylation, and histone modification, facilitate heritable gene expression regulation without changing the underlying DNA sequence. Recent studies have increasingly highlighted the intricate interactions between lncRNAs and these epigenetic modifications in colorectal cancer, illuminating their participation in critical processes, including cell proliferation, apoptosis, invasion, and metastasis. This review concentrates on the relationship between long non-coding RNAs and several significant epigenetic modifications that drive colorectal cancer development, offering fresh insights for future research that aims at devising more effective treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31b080b73592dba79e60d81f2a7fb3048fe78b62" target='_blank'>
              Long non-coding RNAs promote colorectal cancer development through other epigenetic modifications
              </a>
            </td>
          <td>
            Guojuan Wang, Qingqing Ma, Naicheng Zhu, Yuwei Yan, Chen Zhong, Xinlin Hong, Yujuan Lai, Xiudan Chen, Nanxin Li, Wenyan Yu
          </td>
          <td>2025-09-28</td>
          <td>Gastroenterology Report</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In this study, we delineated the distinct transcriptomic effects of VHL mutation and chromosome 3p (chr3p) loss, revealing that chr3p loss is specifically associated with immune suppression in clear cell renal cell carcinoma (ccRCC). Furthermore, we developed driver genomic aberration (DGA) gene signatures that demonstrate superior performance in predicting both patient prognosis and treatment response compared to traditional mutation-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8edc71bb1913e06b7ed3c5e92c20979a6f59e6" target='_blank'>
              Unravelling the distinct effects of VHL mutations and chromosome 3p loss in clear cell renal cell carcinoma: Implications for prognosis and treatment
              </a>
            </td>
          <td>
            Xiang Wang, Jian-Rong Li, N. Chowdhury, Lang Wu, Cheng Chao
          </td>
          <td>2025-09-25</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) involves the expansion of hematopoietic stem cells with ageacquired mutations linked to myeloid malignancy. Advances in next-generation and single-cell sequencing, along with computational modeling, have expanded our ability to detect both common and rare CH drivers, including single-nucleotide variants and mosaic chromosomal alterations, with increasing sensitivity. While sequencing methods differ in accuracy, cost, and ability to detect low-frequency variants, they have deepened our understanding of CH biology. A growing body of evidence has identified both somatic drivers, such as variants in DNMT3A, TET2, and ASXL1, and germline genetic variants that modify CH risk, highlighting the complex interplay between inherited and acquired factors. These collective discoveries are guiding the development of targeted therapies and interventions, particularly for individuals at risk of progression to myeloid neoplasms or cardiovascular disease. Additionally, CH is emerging as a clinically relevant factor in the treatment of solid tumors, where it may influence the tumour microenvironment, treatment response and the risk of therapy-related complications. Risk stratification models are facilitating earlier identification and monitoring of high-risk individuals, enabling personalized treatment decisions. The scope of CH management continues to expand, from surveillance to intervention, with ongoing trials testing preventive strategies in high-risk populations. Emerging trial frameworks emphasize risk stratification, age-appropriateness, inclusive recruitment, decentralized trial models, and the use of traditional clinical and novel endpoints. Together, these advances reflect a shift from passive observation to proactive intervention, charting a course for early detection, precision treatment, and prevention in CH care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6faa3f8b3f8e8d6ae046148a99055b41b7b424" target='_blank'>
              CHIP ahoy: charting a decade of discovery in clonal hematopoiesis.
              </a>
            </td>
          <td>
            Casey K. Wong, Alexandra McDonald, M. Buttigieg, M. Rauh
          </td>
          <td>2025-09-04</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="There is growing interest in understanding the mechanisms underlying differences in cancer incidence among species (comparative oncology). The naked mole-rat (NMR) is often referenced as "cancer-resistant" and prior studies focused on identifying mechanisms explaining this. However, efforts to assess this in vivo have been limited. Herein, we provide evidence that the NMR presents as a novel autochthonous model of lung tumor initiation, driven by an introduction of the oncogenic Eml4-Alk fusion protein using CRISPR-mediated genome editing. Whereas in mice the inversion alone is sufficient to drive tumorigenesis, the inversion alone was insufficient to drive tumorigenesis in the NMR lung and tumor development required additional losses of the tumor suppressors p53 and pRb. Our findings suggest that the proposed "resistance" of the NMR to the development of cancer may reflect that the genetic events leading to tumor initiation are likely to be comparable to those present in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a53eef9b5cb3d97ea0c88db9955706dc37ce138" target='_blank'>
              An Autochthonous Model of Lung Cancer Identifies Requirements for Cellular Transformation in the Naked Mole-Rat.
              </a>
            </td>
          <td>
            A. Shepard, Daniel K Lester, Scott Troutman, Sany Hoxha, Walid T Khaled, E. S. Smith, Thomas J. Park, Rochelle Buffenstein, Dongliang Du, Mingxiang Teng, C. Dengler-Crish, Kenneth Y Tsai, Elsa R Flores, Andrea Ventura, Joseph L. Kissil
          </td>
          <td>2025-09-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Essential epigenetic dependencies have become evident in many cancers. Based on functional antagonism between BAF/SWI/SNF and PRC2 in SMARCB1-deficient sarcomas, we and colleagues recently completed the clinical trial of the EZH2 inhibitor tazemetostat, leading to its FDA approval. However, the principles of tumor response to epigenetic therapy in general, and tazemetostat in particular, remain unknown. First, using functional genomics of patient tumors and diverse experimental models, we define molecular mechanisms of tazemetostat resistance in SMARCB1-deficient sarcomas and rhabdoid tumors. We found distinct classes of acquired mutations that converge on the RB1/E2F axis and decouple EZH2-dependent differentiation and cell cycle control. This allows tumor cells to escape tazemetostat-induced G1 arrest despite EZH2 inhibition, and suggests a general mechanism for effective EZH2 therapy. We provide prospective biomarkers for therapy stratification, including the cell cycle-regulated PRICKLE1 gene. To circumvent tazemetostat resistance and improve response in vitro and in vivo we develop the rational combination of tazemetostat with barasertib, to target the downstream cell cycle kinase AURKB. Second, through transcriptomic inference of SMARCB1-deficient tumor cells, we predicted ATR as a target to overcome resistance to EZH2 inhibition. We found that EZH2 inhibition induces DNA damage, at least in part via upregulation of the transposase-derived PGBD5 nuclease. Finally, we establish a second rational combination strategy for increased anti-tumor efficacy between tazemetostat and the ATR inhibitor, elimusertib, by leveraging the synthetic lethal interaction with PGBD5-dependent DNA damage repair signaling via ATR. Thus, the epigenetic therapy, EZH2 inhibition, induces a targetable collateral vulnerability through activation of the PGBD5-ATR axis. This establishes a mechanistically rational combination strategy for targeting EZH2 and ATR in SMARCB1-deficient cancers based on an EZH2-PGBD5-ATR synthetic lethal axis and is immediately translatable to clinical trials for patients with these aggressive malignancies.



 Helen S Mueller. Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Discovery and Innovation in Pediatric Cancer— From Biology to Breakthrough Therapies; 2025 Sep 25-28; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2025;85(18_Suppl_2):Abstract nr A048.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e20005fa9b7768828704bae3a839702d20b1fe7d" target='_blank'>
              Abstract A048: Overcoming clinical resistance to EZH2 inhibition using rational epigenetic combination therapy
              </a>
            </td>
          <td>
            Helen S. Mueller
          </td>
          <td>2025-09-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="A cornerstone of research to improve cancer outcomes involves studies of model systems to identify causal drivers of oncogenesis, understand mechanisms leading to metastases, and develop new therapeutics. While most cancer types are represented by large cell line panels that reflect diverse neoplastic genotypes and phenotypes found in patients, prostate cancer is notable for a very limited repertoire of models that recapitulate the pathobiology of human disease. Of these, Lymph node carcinoma of the prostate (LNCaP) has served as the major resource for basic and translational studies. Here, we delineated the molecular composition of LNCaP and multiple substrains through analyses of whole genome sequences, transcriptomes, chromatin structure, AR cistromes, and functional studies. Our results determined that LNCaP exhibits substantial subclonal diversity, ongoing genomic instability and phenotype plasticity. While several oncogenic features were consistently present across strains, others were unexpectedly variable such as ETV1 expression, Y chromosome loss, a reliance on WNT and glucocorticoid receptor activity, and distinct AR alterations maintaining AR pathway activation. These results document the inherent molecular heterogeneity and ongoing genomic instability that drive diverse prostate cancer phenotypes and provide a foundation for the accurate interpretation and reproduction of research findings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/851530a0dad65b81ca7cbda1909bc73f4b9c3e44" target='_blank'>
              Multiomic assessments of LNCaP and derived cell strains reveal determinants of prostate cancer pathobiology.
              </a>
            </td>
          <td>
            Arnab Bose, Armand Bankhead Iii, Ilsa Coleman, Thomas Persse, Wanting Han, Patricia C. Galipeau, B. Hanratty, Tony L. Chu, Jared M. Lucas, Dapei Li, Rabeya Bilkis, P. Itagi, Sajida Hassan, Mallory Beightol, Minjeong Ko, R. Dumpit, Michael C. Haffner, C. Pritchard, G. Ha, Peter S Nelson
          </td>
          <td>2025-09-16</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Background Chromothripsis-driven 3p deletion and 5q amplification are early, clonal events in clear cell renal cell carcinoma (ccRCC). These lesions respectively target the epigenetic regulators SETD2 and NSD1. Paradoxically, although NSD1 is amplified on 5q, it is frequently hypermethylated and transcriptionally silenced, suggesting functional inactivation is selected for. The mechanistic implications of NSD1 and SETD2 loss in mitotic control remain poorly understood. Methods We employed a multifaceted approach integrating in vitro kinase and methyltransferase assays, mass spectrometry, and molecular modeling to investigate the regulatory relationship between NSD1, AURKA, and SETD2. CRISPR/Cas9-engineered cell lines were generated to model genetic loss of NSD1, while pharmacologic inhibitors were used to perturb NSD1 and AURKA activity. Protein-protein interactions and post-translational modifications were characterized via co-immunoprecipitation, immunoblotting, fluorescence microscopy, and quantitative mass spectrometry. Functional outcomes were assessed in cell-based systems, and the impact on tumor growth was evaluated in xenograft models examining AURKA-mediated regulation of SETD2 in vivo. Results We identified NSD1 as a methyltransferase that directly methylates AURKA, serving as a negative regulator of its kinase activity and subcellular dynamics during mitosis. Loss of NSD1, through genetic deletion or pharmacologic inhibition, led to AURKA hyperactivation, defective spindle architecture, chromosome mis-segregation, and increased micronuclei formation. Unexpectedly, we found that AURKA phosphorylates SETD2, functionally linking these two epigenetic regulators. This phosphorylation selectively regulated SETD2’s cytoskeletal activity without affecting its chromatin-associated roles. Disruption of this modification—via AURKA inhibition or mutation of the phosphorylation site—compromised mitotic fidelity and enhanced genomic instability. Importantly, phosphorylation-deficient SETD2 mutants were incapable of sustaining tumor growth in xenograft models, underscoring the oncogenic relevance of this post-translational modification. Moreover, SETD2 loss sensitized ccRCC cells to AURKA inhibition, revealing a potential therapeutic vulnerability. Conclusions Our findings reveal a novel regulatory pathway in which NSD1-mediated methylation suppresses AURKA, while AURKA phosphorylation regulates SETD2 activity on the cytoskeleton, linking these two tumor suppressors altered in early ccRCC. Disruption of this NSD1-AURKA-SETD2 axis creates a state of mitotic vulnerability, opening the door for therapeutic intervention using AURKA inhibitors in genetically defined subsets of ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0102fa4cd42475d60bdea2a4817663b752fbbed1" target='_blank'>
              43AURKA connects NSD1 and SETD2 in an epigenetic axis governing mitotic fidelity in ccRCC
              </a>
            </td>
          <td>
            R. Dere, Manga Motrapu, Ashley G Boice, Richard Han, Xiaoli Wang, Pratim Chowdhury, Sung Jung, B. V. V. Prasad
          </td>
          <td>2025-10-01</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8304854a46599698bac2a6e2d8f83da3bfb9d71b" target='_blank'>
              DNA2 enables growth by restricting recombination-restarted replication.
              </a>
            </td>
          <td>
            Jessica J R Hudson, Rowin Appanah, David Jones, Kathryn Davidson, Alice M Budden, Alina Vaitsiankova, Kok-Lung Chan, Keith W. Caldecott, Antony M. Carr, Ulrich Rass
          </td>
          <td>2025-09-03</td>
          <td>Nature</td>
          <td>0</td>
          <td>16</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [8],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>