<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-04-28 10:53:36 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>31</td>
          <td>103</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>67</td>
          <td>136</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>110</td>
          <td>144</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>326</td>
          <td>136</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>287</td>
          <td>158</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8098c87089e0ecbf0e8dc20a062afe3e12362b0b" target='_blank'>
              Oncogene Silencing via ecDNA Micronucleation
              </a>
            </td>
          <td>
            Lotte Brückner, Robin Xu, Jun Tang, Alexander Herrmann, I. Wong, Shu Zhang, Fengyu Tu, Madison Pilon, A. Kukalev, Katharina Pardon, O. Sidorova, Joshua Atta, Qinghao Yu, Davide Pradella, M. Ilić, Marco Novais-Cruz, Sarah Kaltenbach, Denise Treue, M. Giurgiu, Sergej Herzog, Ann-Sophie Hollinger, Martina Fernandez, Finnja Becker, Varvara-Rigina Louma, R. Schmargon, J. Dörr, David Gamlin, Annika Lehmann, D. Gürgen, Matthias Richter, Frank Dubois, Fabrizio Simeoni, Betheney R. Pennycook, Alastair Hamilton, Ralph K. Lindemann, Matthias Fischer, V. Bafna, Geoffrey M. Wahl, R. Koche, Howard Y. Chang, Stamatis Papathanasiou, René Medema, B. Spanjaard, Andrea Ventura, Ana Pombo, Weini Huang, Benjamin Werner, P. Mischel, A. Henssen
          </td>
          <td>2025-04-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6f03a05980baac4dbe599f109d605f3d1cea815" target='_blank'>
              EcDNA-borne PVT1 fusion stabilizes oncogenic mRNAs
              </a>
            </td>
          <td>
            Hyerim Yi, Shu Zhang, Jason Swinderman, Yanbo Wang, Vishnupriya Kanakaveti, King L. Hung, I. Wong, Suhas Srinivasan, Ellis J Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G Jones, J. Luebeck, Yanding Zhao, J. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Frances M. Liang, D. Felsher, Luke A. Gilbert, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e88cd9731c06a22174c2d15ed88fd1e8e310e0fb" target='_blank'>
              Abstract 3755: AmpliconSuite enables discovery of extrachromosomal DNA in tumor genomes
              </a>
            </td>
          <td>
            J. Luebeck, Edwin Huang, T. Liefeld, Forrest Kim, I. Wong, Bhargavi Dameracharla, Gino Prasad, Rohil Ahuja, Daniel Schreyer, Kaiyuan Zhu, Rishaan Kenkre, Tushar Agashe, Devika Torvi, Soyeon Kim, Hoon Kim, Peter Bailey, R.G.W. Verhaak, Viraj Deshpande, Michael Reich, P. Mischel, Jill P. Mesirov, V. Bafna
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a critical driver of cancer progression, contributing to tumour growth, evolution, and therapeutic resistance through oncogene amplification. Despite its significance, the replication of ecDNA remains poorly understood. In this study, we investigated the replication dynamics of ecDNA using high-resolution replication timing analysis (Repli-seq) and DNAscent, a method for measuring origin firing and replication fork movement based on ultra-long read Oxford Nanopore Sequencing, that we applied to both bulk DNA and to ecDNA isolated with FACS-based Isolation of Native ecDNA (FINE), a new method for isolating intact, chromatinised ecDNA without DNA or protein digestion. We demonstrate that ecDNA in the COLO 320DM colorectal cancer cell line exhibits largely asynchronous replication throughout the S phase, contrasting with the conserved replication timing of the corresponding normal linear chromosomal DNA in RPE-1 cells and the chromosomally reintegrated ecDNA in COLO 320HSR, which forms a homogeneously staining region. Replication origins on ecDNA are redistributed, and replication forks exhibit reduced velocity and increased stalling, particularly near the c-MYC oncogene. Under replication stress induced by hydroxyurea treatment, ecDNA replication is further compromised, leading to altered origin activation, reduced fork velocity and eventual ecDNA depletion from cells. Our findings reveal fundamental differences in the replication dynamics of ecDNA, providing insights that could inform the development of therapies targeting ecDNA-associated oncogene amplification in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc7a9c2ef16ae19f8f5b97c5088f5ea3ae1f2bd" target='_blank'>
              ecDNA replication is disorganised and vulnerable to replication stress
              </a>
            </td>
          <td>
            Jedrzej J. Jaworski, PL Pfuderer, Pawel Czyz, G. Petris, MA Boemo, Julian E. Sale
          </td>
          <td>2025-03-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9123bbc8d86d0e1ae0116fdaeb8338ea509e867c" target='_blank'>
              Abstract 3884: Uneven inheritance and functional heterogeneity derived from extrachromosomal DNA enable rapid adaptation of cancer cells
              </a>
            </td>
          <td>
            Shu Zhang, Aditi Gnanasekar, Ellis J Curtis, Jun Tang, Vishnu Shankar, I. Wong, Xiaowei Yan, Rui Li, Lu Yang, Howard Y. Chang, P. Mischel
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f629203ff4e6d119bd3d8d7b5f0a9d082c7f87e7" target='_blank'>
              Abstract 5046: Three-dimensional genome landscape of primary human cancers
              </a>
            </td>
          <td>
            Yanding Zhao, Katie E. Yost, King L. Hung, Kaiyuan Zhu, Duo Xu, Ryan Corces, Shadi Shams, Brian H Louie, Shahab Sarmashghi, Laksshman Sundaram, J. Luebeck, Stanley Clarke, A. S. Doane, Jeffrey M. Granja, Hani Choudhry, Marcin Imieliński, Andrew D Cherniack, Ekta Khurana, V. Bafna, Ina Felau, J. Zenklusen, Peter W. Laird, Christina Curtis, Cancer Genome Atlas Analysis Network, W.J. Greenleaf, Howard Y. Chang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Aneuploidy is observed as gains or losses of whole chromosomes or chromosome arms and is a common hallmark of cancer. Whereas models for the generation of aneuploidy in cancer invoke mitotic chromosome segregation errors, whole-arm losses might occur simply as a result of centromere breakage. We recently showed that elevated RNA Polymerase II (RNAPII) level over S-phase-dependent histone genes predicts rapid recurrence of human meningioma and is correlated with total whole-arm losses relative to gains. To explain this imbalance in arm losses over gains, we have proposed that histone overexpression at S-phase competes with the histone H3 variant CENP-A, resulting in centromere breaks and whole-arm losses. To test whether centromere breaks alone can drive aneuploidy, we ask whether total whole-arm aneuploids can predict outcome across different cancer types in large RNA and whole-genome sequencing databanks. We find that total whole-arm losses generally predict outcome, suggesting that centromere breakage is a major initiating factor leading to aneuploidy and the resulting changes in the selective landscape that drive most cancers. We also present evidence that centromere breakage alone is sufficient to account for whole-arm losses and gains, contrary to mitotic spindle error models for generation of aneuploidy. Our results suggest that therapeutic intervention targeting histone overexpression has the potential of reducing aneuploidy and slowing cancer progression. Significance Statement Gain or loss of whole chromosome arms following centromere breaks is frequent in cancer, but whether or not there is a common initiating event is unknown. Here we show that the total number of whole-arm losses predicts patient outcomes across cancer types, suggesting a causal relationship. This general excess of losses over gains is not predicted by mitotic error models of aneuploidy but rather suggests that centromere breaks themselves initiate whole-arm aneuploidies. Insofar as aneuploidy reshapes the selective landscapes that drive most cancers, our results have potential clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6556f74989cd6bd7e813ef9e417ef3b7531711e1" target='_blank'>
              Total whole-arm chromosome losses predict malignancy in human cancer
              </a>
            </td>
          <td>
            Ye Zheng, K. Ahmad, S. Henikoff
          </td>
          <td>2025-04-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Chromoplexy is a phenomenon of complex genome rearrangement, occurring during a single cell event and characterized by the formation of chain rearrangements affecting multiple chromosomes. Unlike other genomic rearrangements such as chromothripsis, which involves a single chromosome, chromoplexy affects several chromosomes at once, creating patterns of complex, balanced translocations, and leading to the formation of fusion genes and the simultaneous disruption of several genes. Chromoplexy was first identified in prostate cancers, but it is now observed in various cancers where gene fusions take place. The precise mechanisms behind chromoplexy remain under investigation. The occurrence of these rearrangements follows multiple double-stranded breaks that appear to occur in certain regions or during particular genome configurations (open chromatin, active transcription area), and which lead to an intricate series of inter- and intra-chromosomal translocations and deletions without significant alterations in the number of copies. Although chromoplexy is considered a very early event in oncogenesis, the phenomenon can be repeated and can constitute a mechanism of clonal tumor progression. The occurrence of chromoplexy supports the equilibrium model punctuated by tumor evolution, characterized by periods of relative stability punctuated by sudden and rapid periods of radical genomic changes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3ffdab6d59ce2f03de66a13a4efd90a8a827d6" target='_blank'>
              Chromoplexy: A Pathway to Genomic Complexity and Cancer Development
              </a>
            </td>
          <td>
            Franck Pellestor, Benjamin Ganne, Jean Baptiste Gaillard, V. Gatinois
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d61674df32f0a41cdb182a7f8115a40cdbf840a" target='_blank'>
              Abstract 3897: Investigating the prevalence of ecDNA in breast cancer
              </a>
            </td>
          <td>
            Georgia Stavrou, Chris Bailey, Jeanette Kittel, M. Jamal-Hanjani, C. Swanton, N. Kanu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Intratumour heterogeneity and phenotypic plasticity drive tumour progression and therapy resistance1,2. Oncogene dosage variation contributes to cell-state transitions and phenotypic heterogeneity3, thereby providing a substrate for somatic evolution. Nonetheless, the genetic mechanisms underlying phenotypic heterogeneity are still poorly understood. Here we show that extrachromosomal DNA (ecDNA) is a major source of high-level focal amplification in key oncogenes and a major contributor of MYC heterogeneity in pancreatic ductal adenocarcinoma (PDAC). We demonstrate that ecDNAs drive varying levels of MYC dosage, depending on their regulatory landscape, enabling cancer cells to rapidly and reversibly adapt to microenvironmental changes. In the absence of selective pressure, a high ecDNA copy number imposes a substantial fitness cost on PDAC cells. We also show that MYC dosage affects cell morphology and dependence of cancer cells on stromal niche factors. Our work provides a detailed analysis of ecDNAs in PDAC and describes a new genetic mechanism driving MYC heterogeneity in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db22625a85ea34b5a5adadae1847c161e783d66" target='_blank'>
              MYC ecDNA promotes intratumour heterogeneity and plasticity in PDAC
              </a>
            </td>
          <td>
            Elena Fiorini, A. Malinova, Daniel Schreyer, Davide Pasini, Michele Bevere, Giorgia Alessio, Diego Rosa, Sabrina D'Agosto, Luca Azzolin, Salvatore Milite, S. Andreani, F. Lupo, L. Veghini, Sonia Grimaldi, S. Pedron, Monica Castellucci, C. Nourse, R. Salvia, G. Malleo, A. Ruzzenente, A. Guglielmi, M. Milella, R. Lawlor, C. Luchini, A. Agostini, Carmine Carbone, Christian Pilarsky, A. Sottoriva, Aldo Scarpa, D. Tuveson, Peter Bailey, V. Corbo
          </td>
          <td>2025-03-12</td>
          <td>Nature</td>
          <td>1</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edc027322f110679344e9de5725b902edc43f13e" target='_blank'>
              Deregulated enhancer-promoter communication in cancer through altered nuclear architecture.
              </a>
            </td>
          <td>
            Isabelle Seufert, Claire Vargas, Sina Jasmin Wille, K. Rippe
          </td>
          <td>2025-04-12</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a97fcf511bf3ac6d39959b0d32a3431fd3b9263" target='_blank'>
              Abstract 7504: Integrated multiomic analysis to infer longitudinal patterns of ecDNA in IDH-wildtype integrated multiomic analysis to infer longitudinal patterns of ecDNA in IDH wild-type glioblastoma
              </a>
            </td>
          <td>
            Dacheng Zhao, K. Anderson, Amit Gujar, Megan L Costa, Djamel Nehar-Belaid, F. Varn, Elise T. Courtois, Paul Gabrie, Paul Robson, H. Moon, A. Gołȩbiewska, S. Paek, S. Niclou
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ddc0ae23b848c6a2bead56e11b48f6e63657e8f" target='_blank'>
              Abstract 3900: Integrating single-molecule localization microscopy and computational pipelines to elucidate ecDNA dynamics
              </a>
            </td>
          <td>
            Mesut Unal, Jessica Alsing, Gabrielle S. Dewson, Guy Nir, Kadir C Akdemir
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7bed6325a16795853f9398ff50f536009080006" target='_blank'>
              Abstract 3754: ecDNAInspector nominates high-confidence structural predictions of extrachromosomal DNA for downstream analysis
              </a>
            </td>
          <td>
            Sophia Pribus, Kathleen E. Houlahan, Lise Mangiante, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d2ac30d018ecdf847938d3f5d1a087883242d4" target='_blank'>
              Abstract 2775: High-resolution mapping of oncogenic structural changes in osteosarcoma
              </a>
            </td>
          <td>
            A. Clugston
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Multiple myeloma (MM) remains an incurable disease primarily due to the emergence of drug resistance, and the underlying mechanisms remain unclear. Extrachromosomal circular DNAs (eccDNAs) are prevalent in cancer genomes of both coding and non-coding regions. However, the role of non-coding eccDNA regions that serve as enhancers has been largely overlooked. Here, genome-wide profiling of serum eccDNAs from donors and MM patients who responded well or poorly to bortezomib-lenalidomide-dexamethasone (VRd) therapy is characterized. A high copy number of eccDNA ANKRD28 (eccANKRD28) predicts poor therapy response and prognosis but enhanced transcriptional activity. Established VRd-resistant MM cell lines exhibit a higher abundance of eccANKRD28, and CRISPR/Cas9-mediated elevation of eccANKRD28 desensitizes bortezomib and lenalidomide treatment both in vitro and in vivo. Integrated multi-omics analysis (H3K27ac ChIP-seq, scRNA-seq, scATAC-seq, CUT&Tag, et al.) identifies eccANKRD28 as an active enhancer involved in drug resistance driven by the key transcription factor, POU class 2 homeobox 2 (POU2F2). POU2F2 interacts with sequence-specific eccANKRD28 as well as RUNX1 and RUNX2 motifs to form the protein complex, which activates the promoter of oncogenes, including IRF4, JUNB, IKZF3, RUNX3, and BCL2. This study elucidates the potential transcriptional network of enhancer eccANKRD28 in MM drug resistance from a previously unrecognized epigenetic perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea6e610cf350e732841fb77da78765020fd07704" target='_blank'>
              Enhancer Extrachromosomal Circular DNA ANKRD28 Elicits Drug Resistance via POU2F2-Mediated Transcriptional Network in Multiple Myeloma.
              </a>
            </td>
          <td>
            Binzhen Chen, Jia Liu, Yaoxin Zhang, Changming Shi, Di Zhu, Guoqiang Zhang, Fei Xiao, L. Zhong, Minyue Zhang, Lai Guan Ng, Honghui Huang, Tingting Lu, Jian Hou
          </td>
          <td>2025-04-01</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c882aec893125b1655a88dcecde1b32dd7df1e2c" target='_blank'>
              Abstract 4094: Allosteric chromatin activity transition mechanisms drive epigenetic dosage compensation in essential genes triggered by oncogene activation in TRACERx
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, Xiaohong Liu, Georgia Stavrou, Nana E. Mensah, E. L. Cadieux, Özgen Deniz, M. Jamal-Hanjani, Stephan Beck, Jonas Demeulemeester, M. Tanić, C. Swanton, P. Van Loo, N. Kanu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ad9d941478590c32e5cf0763f3872cb1fd2c287" target='_blank'>
              Satellite DNA Amplification in Advanced Prostate Cancer Is Largely Independent From Euchromatic and Oncogene Amplicons.
              </a>
            </td>
          <td>
            Anja Weise, A. Ornellas, Gilda Alves, Constanze Pentzold, Jenny Holler, Melanie Wolter, Elena Jamali, Bernhard Theis, Thomas Liehr
          </td>
          <td>2025-03-17</td>
          <td>The journal of histochemistry and cytochemistry : official journal of the Histochemistry Society</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a65e388eb0715f0681bd240054e0692300312172" target='_blank'>
              Abstract 6307: ELViS : An R package for estimating copy number levels of viral genomic segments at base-resolution
              </a>
            </td>
          <td>
            Jin-Young Lee, Hyo Young Choi, J. Holt, Xiaobei Zhao, Katie Hoadley, D. N. Hayes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64a8bc0b12893b6190281e1fdb8cfc040b24528b" target='_blank'>
              Abstract LB045: Pancancer meta-analysis of the genomic features associated with tandem duplication formation in cancer
              </a>
            </td>
          <td>
            F. Menghi, Edison T. Liu
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) has been found in most types of human cancers, and ecDNA incorporating viral genomes has recently been described, specifically in human papillomavirus (HPV)-mediated oropharyngeal cancer (OPC). However, the molecular mechanisms of human-viral hybrid ecDNA (hybrid ecDNA) for carcinogenesis remains elusive. We characterize the epigenetic status of hybrid ecDNA using HPVOPC cell lines and patient-derived tumor xenografts, identifying HPV oncogenes E6/E7 in hybrid ecDNA are flanked by previously unrecognized somatic DNA enhancers and HPV L1 enhancers, with strong cis-interactions. Targeting of these enhancers by clustered regularly interspaced short palindromic repeats interference or hybrid ecDNA by bromodomain and extra-terminal inhibitor reduces E6/E7 expression, and significantly inhibites in vitro and/or in vivo growth only in ecDNA(+) models. HPV DNA in hybrid ecDNA structures are associated with previously unrecognized somatic and HPV enhancers in hybrid ecDNA that drive HPV ongogene expression and carcinogenesis, and can be targeted with ecDNA disrupting therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76981a5b85fcdfc01f05694a03b59cd41140b0ab" target='_blank'>
              Inhibition of human-HPV hybrid ecDNA enhancers reduces oncogene expression and tumor growth in oropharyngeal cancer
              </a>
            </td>
          <td>
            Takuya Nakagawa, J. Luebeck, Kaiyuan Zhu, Joshua T. Lange, Roman Sasik, Chad Phillips, S. Sadat, Sara Javadzadeh, Qian Yang, Abdula Monther, Santiago Fassardi, Allen Wang, Kersi Pestonjamasp, Brin Rosenthal, Kathleen M Fisch, P. Mischel, V. Bafna, Joseph A Califano
          </td>
          <td>2025-03-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54f93c77ea0aca0c8fd7efd0d36e03e57f00addd" target='_blank'>
              Abstract 6371: KLIPP: Precision targeting of fusions and amplified oncogenes using CRISPR
              </a>
            </td>
          <td>
            Huibin Yang, R. Hulbatte, Natalie Gratsch, Ashley Sutter, Anne Urzynicok, Mayer Cusnir, Mario Ashaka, Aadi Shiv Malhotra, Ishwarya Narayanan, Michelle Paulsen, Anna Schwendeman, Tom Wilson, Mats Ljungman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chromosome territories (CTs) are intricately organized and regulated within the nucleus. Despite remarkable advances in our understanding of genome packaging and gene expression, the interplay among CTs, pairing of parental homologous chromosomes, and genome function during development remains elusive. Here, we employ an Oligopaints-based high-resolution imaging approach to examine variable CT organization in single nuclei during the developmental process of zygotic genome activation. We reveal large-scale chromosome changes with extensive homolog pairing at the whole-chromosome level that decreases locally due to spatial variability in chromosome conformations. In the absence of one homolog copy, the dynamics of CT compaction and RNA polymerase II recruitment are supported by transcriptional changes in haploid embryos. Finally, global inhibition of transcription results in decreased CT opening and no significant impact on CT pairing levels. These findings enhance our understanding of parental genome folding and regulation, which may inform strategies for chromosome-based diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/461f9f862c0bc9f9b6c4530eae22c13287308545" target='_blank'>
              The functional organization of chromosome territories in single nuclei during zygotic genome activation
              </a>
            </td>
          <td>
            Akshada Shankar Ganesh, Taylor M. Orban, Romir Raj, Peter I. Fatzinger, Anna Johnson, Sean M. Riccard, Akhmed Zhanaidarov, Mayu Inaba, Jelena Erceg
          </td>
          <td>2025-04-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bec087600fbb41ee812fa17d65823942d9127a6b" target='_blank'>
              BTRR complex deficiency is a driver for genomic instability in Bloom syndrome
              </a>
            </td>
          <td>
            Ipek Ilgin Gönenc, Alexander Wolff, A. Busley, A. Wieland, Andrea E. Tijhuis, Christian Müller, René Wardenaar, L. Argyriou, S. Kaulfuß, Markus Räschle, D. Spierings, F. Foijer, Holger Bastians, G. Yigit, Arne Zibat, Lukas Cyganek, B. Wollnik
          </td>
          <td>2025-04-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b492f14ec0b02275c3aedf5d1fc7e017eb0ec151" target='_blank'>
              Tracking Replicating HPV Genomes in Proliferating Keratinocytes
              </a>
            </td>
          <td>
            Jonathan R. Shin, Ivan Avilov, Mario Schelhaas, Franck Gallardo, Alison A. McBride
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Recent research reveals that eukaryotic genomes form circular DNA from all parts of their genome, some large enough to carry whole genes. In organisms like yeast and in human cancers, it is often observed that extrachromosomal circular DNA (eccDNA) benefits the individual cell by providing resources for rapid cellular growth. However, our comprehension of eccDNA remains incomplete, primarily due to their transient nature. Early studies suggest they arise when DNA breaks and is subsequently repaired incorrectly. In this review, we provide an overview of the evidence for molecular mechanisms that lead to eccDNA formation in human cancers and yeast, focusing on nonhomologous end joining, alternative end joining, and homologous recombination repair pathways. Furthermore, we present hypotheses in the form of molecular eccDNA formation models and consider cellular conditions which may affect eccDNA generation. Finally, we discuss the framework for future experimental evidence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd036a4aeb41767279a21c8043d9cb8d21781a98" target='_blank'>
              Molecular mechanisms of extrachromosomal circular DNA formation
              </a>
            </td>
          <td>
            Rasmus A B Eugen-Olsen, J. M. Hariprakash, V. Oestergaard, Birgitte Regenberg
          </td>
          <td>2025-02-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Kaposi’s sarcoma-associated herpesvirus (KSHV) genome contains a terminal repeats (TR) sequence. Previous studies demonstrated that KSHV TR functions as a gene enhancer for inducible lytic gene promoters. Gene enhancers anchor bromodomain-containing protein 4 (BRD4) at specific genomic region, where BRD4 interacts flexibly with transcription-related proteins through its intrinsically disordered domain and exerts transcription regulatory function. Here, we generated recombinant KSHV with reduced TR copy numbers and studied BRD4 recruitment and its contributions to the inducible promoter activation. Reducing the TR copy numbers from 21 (TR21) to 5 (TR5) strongly attenuated viral gene expression during de novo infection and impaired reactivation. The EF1α promoter encoded in the KSHV BAC backbone also showed reduced promoter activity, suggesting a global attenuation of transcription activity within TR5 latent episomes. Isolation of reactivating cells confirmed that the reduced inducible gene transcription from TR-shortened DNA template and is mediated by decreased efficacies of BRD4 recruitment to viral gene promoters. Separating the reactivating iSLK cell population from non-responders showed that reactivatable iSLK cells harbored larger LANA nuclear bodies (NBs) compared to non-responders. The cells with larger LANA NBs, either due to prior transcription activation or TR copy number, supported KSHV reactivation more efficiently than those with smaller LANA NBs. With auxin-inducible LANA degradation, we confirmed that LANA is responsible for BRD4 occupancies on latent chromatin. Finally, with purified fluorescence-tagged proteins, we demonstrated that BRD4 is required for LANA to form liquid-liquid phase-separated dots. The inclusion of TR DNA fragments further facilitated the formation of larger BRD4-containing LLPS in the presence of LANA, similar to the “cellular enhancer dot” formed by transcription factor-DNA bindings. These results suggest that LANA binding to TR establishes an enhancer domain for infected KSHV episomes. The strength of this enhancer, regulated by TR length or transcription memory, determines the outcome of KSHV replication. Importance Gene enhancers are genomic domains that regulate frequency and duration of transcription burst at gene promoters, with BRD4 playing a critical role in their enhancer functions. KSHV latent mini-chromosome also contains an enhancer domain made with multiple copies of 801 bp identical repeat DNA fragments, terminal repeats. Here, we utilized manipulable mini-scale chromatins with convenient inducible KSHV reactivation to systematically examine the association between enhancer strength and the outcome of inducible promoter activation. This study illustrated the amount of BRD4 recruitment at the enhancer associated with frequencies of BRD4 distribution to the inducible promoters during KSHV reactivation and, therefore, KSHV lytic replication. Recruitment of BRD4 to the TR is specifically regulated by KSHV latent protein, LANA. KSHV evolves clever enhancer elements designed to be regulated by the KSHV own latent protein, LANA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41ac4dd8639934c73f419b670737fe670abf6ad3" target='_blank'>
              Studies on Gene Enhancer with KSHV mini-chromatin
              </a>
            </td>
          <td>
            Tomoki Inagaki, Ashish Kumar, Kang-Hsin Wang, Somayeh Komaki, Jonna M. Espera, Christopher S. A. Bautista, Ken-ichi Nakajima, Chie Izumiya, Y. Izumiya
          </td>
          <td>2025-03-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Chromosomal rearrangements, such as translocations, deletions, and inversions, underlie numerous genetic diseases and cancers, yet precise engineering of these rearrangements remains challenging. Here, we present a CRISPR-based homologous recombination-mediated rearrangement (HRMR) strategy that leverages homologous donor templates to align and repair broken chromosome ends. HRMR improves efficiency by approximately 80-fold compared to non-homologous end joining, achieving over 95% homologous recombination. Validated across multiple loci and cell lines, HRMR enables efficient and accurate chromosomal rearrangements. Live-cell imaging reveals that homologous donors mediate chromosome end proximity, enhancing rearrangement efficiency. Thus, HRMR provides a powerful tool for disease modeling, chromosomal biology, and therapeutic applications. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03523-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7170c44494d1fc7e8ae358052e6bacf3d4c1c009" target='_blank'>
              Recruitment and rejoining of remote double-strand DNA breaks for enhanced and precise chromosome editing
              </a>
            </td>
          <td>
            Mingyao Wang, Pengchong Fu, Ziheng Chen, Xiangnan Wang, Hanhui Ma, Xuedi Zhang, Guanjun Gao
          </td>
          <td>2025-03-11</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdd747492529d370113dee3945f74bf5ec74e07f" target='_blank'>
              High levels of DNA replication initiation factors indicate ATRi sensitivity via excessive origin firing
              </a>
            </td>
          <td>
            A. Lumeau, PL Pfuderer, S. De Angelis, JA Scarth, MA Guscott, N. Shaikh, FB Copley, H. Gerdes, Emilie L Alard, PR Cutillas, F. Mardakheh, MA Boemo, JV Forment, SE McClelland
          </td>
          <td>2025-02-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6d0de577d187a1d02372f19a6bb5a20995879b2" target='_blank'>
              Abstract 7177: Deep genomic analysis of undifferentiated sarcoma
              </a>
            </td>
          <td>
            Jef Baelen, Raphael Sciot, Ruben Cools, Jonas Demeulemeester, I. V. Bempt
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="NOTCH2NL (NOTCH2-N-terminus-like) genes arose from incomplete, recent chromosome 1 segmental duplications implicated in human brain cortical expansion. Genetic characterization of these loci and their regulation is complicated by the fact they are embedded in large, nearly identical duplications that predispose to recurrent microdeletion syndromes. Using nearly complete long-read assemblies generated from 67 human and 12 ape haploid genomes, we show independent recurrent duplication among apes with functional copies emerging in humans ∼2.1 million years ago. We distinguish NOTCH2NL paralogs present in every human haplotype (NOTCH2NLA) from copy number variable ones. We also characterize large-scale structural variation, including gene conversion, for 28% of haplotypes leading to a previously undescribed paralog, NOTCH2tv. Finally, we apply Fiber-seq and long-read transcript sequencing to human cortical neurospheres to characterize the regulatory landscape and find that the most fixed paralogs, NOTCH2 and NOTCH2NLA, harbor the greatest number of paralog-specific elements potentially driving their regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57156ab124ecd643544fdfa8cdaaa9cc74e49805" target='_blank'>
              Genetic diversity and regulatory features of human-specific NOTCH2NL duplications
              </a>
            </td>
          <td>
            Taylor Real, Prajna Hebbar, DongAhn Yoo, F. Antonacci, Ivana Pačar, M. Diekhans, Gregory J. Mikol, Oyeronke G. Popoola, Ben Mallory, Mitchell R. Vollger, Philip C. Dishuck, Xavi Guitart, Allison N. Rozanski, Katherine M. Munson, Kendra Hoekzema, Jane Ranchalis, Shane Neph, Adriana E. Sedeño-Cortés, B. Paten, Sofie R. Salama, A. Stergachis, E. Eichler
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/632a6b07fed31db9bd0bc2462cc1bb896521dfbb" target='_blank'>
              Timing and clustering co-occurring genome amplifications in cancers
              </a>
            </td>
          <td>
            Sara Cocomello, Giovanni Santacatterina, R. Bergamin, Alice Antonello, G. Caravagna
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bf8275563011d6ab6090a01c6b82a9c990741f5" target='_blank'>
              Z-DNA formation induces the totipotent-like state and primes Zscan4-dependent chromatin compartmentalization
              </a>
            </td>
          <td>
            Shireen Shajahan, Yann Loe-Mie, Marion Salmon-Legagneur, Tatiana Traboulsi, Agnès Thierry, Anne Dejean, J. Cossec
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) presents a promising target for cancer therapy; however, its spatial-temporal diversity and influence on tumor evolution and the immune microenvironment remain largely unclear. We apply computational methods to analyze ecDNA from whole-genome sequencing data of 595 urothelial carcinoma (UC) patients. We demonstrate that ecDNA drives clonal evolution through structural rearrangements during malignant transformation and recurrence of UC. This supports a model wherein tumors evolve via the selective expansion of ecDNA-bearing cells. Through multi-regional sampling of tumors, we demonstrate that ecDNA contributes to the evolution of multifocality and increased intratumoral heterogeneity. EcDNA is present in 36% of UC tumors and correlates with an immunosuppressive phenotype and poor prognosis. Single-cell RNA sequencing analyses reveal that ecDNA+ malignant cells exhibit diminished expression of major histocompatibility complex class I molecules, enabling them to evade T-cell immunity. Finally, we show that sequencing of urinary sediment-derived DNA has excellent specificity in detecting ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27eaa871f09220399d901baadcfefa6c53b148cd" target='_blank'>
              Spatial-Temporal Diversity of Extrachromosomal DNA Shapes Urothelial Carcinoma Evolution and Tumor-Immune Microenvironment.
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, Huiying Tao, Yanfen Zhu, X. Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez-Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xi Xiang, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, J. Luebeck, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-03-11</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77c67aedf7052778400f6b8258e4d803910f72b0" target='_blank'>
              Abstract 2668: Spatial-temporal diversity of extrachromosomal DNA shapes urothelial carcinoma evolution and tumor-immune microenvironment
              </a>
            </td>
          <td>
            Wei Lv, Yuchen Zeng, Conghui Li, Yuan Liang, J. Luebeck, Huiying Tao, Yanfen Zhu, X. Sui, Yue Li, Shiqi Jiang, Qingqing Gao, Elias Rodriguez Fos, Gino Prasad, Yuanmei Wang, Run Zhou, Zhe Xu, Xiaoguang Pan, Linlin Chen, Xiang Xi, Huajing Teng, Chaoyang Sun, Tianyu Qin, Wei Dong, Yongwei Li, Xun Lan, Xuesong Li, Lin Lin, Lars Bolund, Huanming Yang, R.G.W. Verhaak, B. M. Faltas, Jacob B. Hansen, Sihan Wu, P. Mischel, A. Henssen, V. Bafna, Birgitte Regenberg, Yonglun Luo, Chunhua Lin, Peng Han
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3a7d37cdffe8b6b0e4ee7704d2a0b59475d492a" target='_blank'>
              Abstract 3890: Novel miFISH single-cell approach gives new insights into chromosomal instability and evolution of colorectal cancers
              </a>
            </td>
          <td>
            Anna Roschke, Darawalee Wangsa, Nina Gutzeit, Richard Korshkov, Tim Thoerner, K. Heselmeyer-Haddad, Paul S. Meltzer
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4473ff7fede83ad4cee5cb350abfb1fe6b9f5c2d" target='_blank'>
              Abstract 7501: APOBEC3A drives chromosomal instabilityand tumor evolution in pancreatic cancer
              </a>
            </td>
          <td>
            Zhihui Zhang, Chunxu Gao, Sonja Wörmann, P. Van Loo
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Traditionally, the evolutionary perspective of cancer has been understood as gradual alterations in passenger/driver genes that lead to branching phylogeny. However, in cases of prostate adenocarcinoma and kidney renal cell carcinoma, macroevolutionary landmarks like chromoplexy and chromothripsis are frequently observed. Unfortunately, short-read sequencing techniques often miss these significant macroevolutionary changes, which involve multiple translocations and deletions at the chromosomal level. To resolve such genomic dark matters, we provided high-fidelity long-read sequencing data (78–92 Gb of ~Q30 reads) of six genitourinary tumour cell lines (one benign kidney tumour and two kidney and three prostate cancers). Based on these data, we obtained 12 high-quality, partially phased genome assemblies (Contig N50 1.85–29.01 Mb; longest contig 2.02–171.62 Mb), graph-based pan-genome variant sets (11.57 M variants including 60 K structural variants), and 5-methylcytosine sites (14.68%–27.05% of the CpG sites). We also identified several severe chromosome aberration events, which would result from chromosome break and fusion events. Our cancer genome assemblies will provide unprecedented resolution to understand cancer genome instability and chromosomal aberration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39aabbe27003a6111c9e819cce1949d58b416548" target='_blank'>
              Genome assembly resources of genitourinary cancers for chromosomal aberration at the single nucleotide level
              </a>
            </td>
          <td>
            Hyunho Han, H. Lee, Min Gyu Kim, Yoo Sub Shin, Jin Soo Chung, Jun Kim
          </td>
          <td>2025-04-01</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df9dfba50807f4d056e7d436de2c34ed7ddd425f" target='_blank'>
              Abstract 1502: Hypoxia-driven genomic instability and EcDNA formation in lung cancer: insights from TRACERx
              </a>
            </td>
          <td>
            Natasha Sharma, Chris Bailey, Corentin Richard, Jeanette Kittle, C. Swanton, M. Jamal-Hanjani, N. Kanu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Somatic mobilization of LINE-1 (L1) has been implicated in cancer etiology. We analyzed a recent TCGA data release comprised of nearly 5000 pan-cancer paired tumor-normal whole-genome sequencing (WGS) samples and ~9000 tumor RNA samples. We developed TotalReCall an improved algorithm and pipeline for detection of L1 retrotransposition (RT), finding high correlation between L1 expression and “RT burden” per sample. Furthermore, we mathematically model the dual regulatory roles of p53, where mutations in TP53 disrupt regulation of both L1 expression and retrotransposition. We found those with Li-Fraumeni Syndrome (LFS) heritable TP53 pathogenic and likely pathogenic variants bear similarly high L1 activity compared to matched cancers from patients without LFS, suggesting this population be considered in attempts to target L1 therapeutically. Due to improved sensitivity, we detect over 10 genes beyond TP53 whose mutations correlate with L1, including ATRX, suggesting other, potentially targetable, mechanisms underlying L1 regulation in cancer remain to be discovered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09c16a8e22a8e26a1832c9dc60f1f0a97f3c5163" target='_blank'>
              Pan-cancer multi-omic model of LINE-1 activity reveals locus heterogeneity of retrotransposition efficiency
              </a>
            </td>
          <td>
            Alexander Solovyov, Julie M. Behr, D. Hoyos, Eric Banks, Alexander W Drong, Bryan Thornlow, Jimmy Z Zhong, Enrique Garcia-Rivera, Wilson McKerrow, Chong Chu, Cédric Arisdakessian, Dennis M Zaller, J. Kamihara, Liyang Diao, M. Fromer, Benjamin D. Greenbaum
          </td>
          <td>2025-02-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) has potential in tumor diagnosis, particularly for improving diagnostic accuracy and early cancer detection; however, many challenges remain in its application to clinical practice. Methods We conducted a Circle‐Seq analysis on clinical samples at different stages of colorectal cancer progression to examine the dynamic changes of eccDNA during the progression of colorectal cancer. We used breakpoint-specific PCR to verify candidate eccDNAs identified by Circle‐Seq. The results were further validated using the AOM/DSS-induced colorectal cancer model. Results There was an increase in the abundance of eccDNA with the progression of colorectal cancer. The genes associated with these eccDNA molecules were primarily related to signaling pathways involved in tumor development and metastasis. Our analysis also revealed that eccDNA abundance positively correlates with gene expression, and eccDNA derived from specific genes has potential value for the early diagnosis of tumors. Conclusions This study revealed a connection between eccDNA and colorectal cancer progression and highlights the clinical potential of eccDNA for the early diagnosis of colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s10020-025-01164-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d3999071e24587252e2469121996c435197d05b" target='_blank'>
              Extrachromosomal circular DNA as a novel biomarker for the progression of colorectal cancer
              </a>
            </td>
          <td>
            Quanpeng Qiu, Yi Ding, Xiaolong Guo, Jing Han, Jiaqi Zhang, Yaping Liu, Junjun She, Yinnan Chen
          </td>
          <td>2025-04-01</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97245337195bdd219518d6c8012e2c0d61b49d80" target='_blank'>
              Abstract 3718: Characterization of chromosomal instability-related dependencies through cell-specific metric for gene dosage compensation potential
              </a>
            </td>
          <td>
            Péter Szikora, Iván Fekete, María Victoria Ruiz-Pérez, Dóra Kállai, Csilla Hegedűs, Bence Szalai, Daniel V Veres
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Introduction The recombinant human p53 adenovirus (Ad-p53) offers a promising approach for cancer therapy, yet its chromatin structure and effects on host chromatin organization and gene expression are not fully understood. Methods In this study, we employed in situ ChIA-PET to investigate the colorectal cancer cell line HCT116 with p53 knockout, comparing them to cells infected with the adenovirus-vector expressing p53. We examined alterations in chromatin interactions and gene expression following treatment with the anti-cancer drug 5-fluorouracil (5-FU). Results Our results indicate that Ad-p53 forms a specific chromatin architecture within the vector and mainly interacts with repressive or inactive regions of host chromatin, without significantly affecting the expression of associated genes. Additionally, Ad-p53 does not affect topologically associating domains (TADs) or A/B compartments in the host genome. Discussion These findings suggest that while Ad-p53 boosts p53 expression, enhancing drug sensitivity without substantially altering host HCT116 chromatin architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68b2495bcc5f798b26cfeb646dc7337f34c34784" target='_blank'>
              Chromatin structure and gene transcription of recombinant p53 adenovirus vector within host
              </a>
            </td>
          <td>
            Duo Ning, Yuqing Deng, S. Tian
          </td>
          <td>2025-02-28</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="N6-methyladenosine (m6A), the most abundant internal RNA modification in humans, regulates most aspects of RNA processing. Prostate cancer is characterized by widespread transcriptomic dysregulation; therefore, we characterized the m6A landscape of 162 localized prostate tumors with matched DNA, RNA and protein profiling. m6A abundance varied dramatically across tumors, with global patterns emerging via complex germline–somatic cooperative regulation. Individual germline polymorphisms regulated m6A abundance, cooperating with somatic mutation of cancer driver genes and m6A regulators. The resulting complex patterns were associated with prognostic clinical features and established the biomarker potential of global and locus-specific m6A patterns. Tumor hypoxia dysregulates m6A profiles, bridging prior genomic and proteomic observations. Specific m6A sites, such as those in VCAN, drive disease aggression, associating with poor outcomes, tumor growth and metastasis. m6A dysregulation is thus associated with key events in the natural history of prostate cancer: germline risk, microenvironmental dysregulation, somatic mutation and metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19695af6dbb1a44b8e5db5cde53d9277c7377300" target='_blank'>
              The landscape of N6-methyladenosine in localized primary prostate cancer
              </a>
            </td>
          <td>
            Xin Xu, Helen Zhu, Rupert Hugh-White, J. Livingstone, Stefan E. Eng, Nicole Zeltser, Yujuan Wang, Kinga Pajdzik, Sujun Chen, Kathleen E. Houlahan, Wenqin Luo, Shun Liu, Xi Xu, Minzhi Sheng, Wangyuan Guo, J. Arbet, Yuxi Song, Miranda Wang, Yong Zeng, Shiyan Wang, Guanghui Zhu, Tingxiao Gao, Wei Chen, Xinpei Ci, Wenjie Xu, Kexin Xu, M. Orain, V. Picard, H. Hovington, A. Bergeron, L. Lacombe, Bernard Têtu, Yves Fradet, Mathieu Lupien, Gong-Hong Wei, Marianne Koritzinsky, Robert G Bristow, Neil Fleshner, Xue Wu, Yang Shao, Chuan He, Alejandro Berlin, T. H. van der Kwast, Hon Leong, P. Boutros, H. H. He
          </td>
          <td>2025-03-24</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Chromosomal rearrangements involving additional material in individuals with phenotypic alterations usually result in partial trisomy, often accompanied by partial monosomy. To characterize chromosomal rearrangements and analyze genomic characteristics in the breakpoint regions in 31 patients with additional material on an autosomal chromosome. Different tests were performed to characterize these patients, including karyotyping, chromosomal microarray analysis (CMA), and fluorescent in situ hybridization (FISH). In silico analyses evaluated A/B chromosomal compartments, segmental duplications, and repetitive elements at breakpoints. The 31 rearrangements resulted in 47 copy number variations (CNVs) and a range of structural aberrations were identified, including six tandem duplications, 19 derivative chromosomes, two intrachromosomal rearrangements, one recombinant, two dicentric chromosomes, and one triplication. A deleted segment was associated with the duplication in 16 of the 19 patients with derivative chromosomes from translocation. Among the trios whose chromosome rearrangement origin could be investigated, 54,5% were de novo, 31,9% were maternally inherited, and 13,6% were paternally inherited from balanced translocations or inversion. Breakpoint analysis revealed that 22 were in the A compartment (euchromatin), 25 were in the B compartment (heterochromatin), and five were in an undefined compartment. Additionally, 14 patients had breakpoints in regions of segmental duplications and repeat elements. Our study found that a deletion accompanied by additional genetic material was present in 51.6% of the patients, uncovering the underlying genetic imbalances. Statistical analyses revealed a positive correlation between chromosome size and the occurrence of CNVs in the rearrangements. Furthermore, no preference was observed for breakpoints occurring in compartments A and B, repetitive elements, or segmental duplications. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-97077-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4680fc424cbc00a4561d2d03671779c2ded30f6" target='_blank'>
              Cytogenomic characterization of karyotypes with additional autosomal material
              </a>
            </td>
          <td>
            Priscila Soares Rodrigues, Bruna Burssed, F. Bellucco, Débora Cristina Batista Rosolen, Chong Ae Kim, M. Melaragno
          </td>
          <td>2025-04-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9feff4a41380afc1edc6d3d9d9371441107ea7a0" target='_blank'>
              VAV2 exists in extrachromosomal circular DNA and contributes Enzalutamide resistance of prostate cancer via stabilization of AR/ARv7
              </a>
            </td>
          <td>
            Qinghua Wang, Xin'an Wang, Hanchu Ye, Yicong Yao, Haopeng Li, Xin Qin, Yan Zhao, Wei Jiang, Mingming Xu, Tong Zi, Xilei Li, Xi Chen, Juan Zhou, Gang Wu
          </td>
          <td>2025-04-09</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1b5c0f4484941083873b791a9acfc4a1fd9d211" target='_blank'>
              Abstract 2847: Patterns of APOBEC3-mediated chromosomal rearrangements in solid cancers
              </a>
            </td>
          <td>
            T. Cheong, Qi Wang, Ahram Jang, Elif Karaca-Atabay, Jake June-Koo Lee, Roberto Chiarle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1dc55203f4f61b62677fd9af379d34ddca08f1" target='_blank'>
              Abstract 1479: Defining the interplay between DNAPKcs and shieldin during chromosomal break repair
              </a>
            </td>
          <td>
            Kaela Makins, Metztli Cisneros-Aguirre, F. Lopezcolorado, Jeremy M. Stark
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c127048e0d0f262eff5e6fdada1e6625367af0dd" target='_blank'>
              Abstract 5227: Extrachromosomal MYC-paralogs amplification shapes suppressive tumor microenvironment in small cell lung cancer
              </a>
            </td>
          <td>
            Jingwei Zhang, Yueqi Jin, Ence Yang, Xiao Li
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cb94d2d1b79c9a0abe29eed20fd570bda0af9d0" target='_blank'>
              Abstract 419: Chromosomal instability and oncogene dosage variations underlying KRAS G12C inhibition therapy resistance in lung cancer
              </a>
            </td>
          <td>
            Dingcheng Gao, Yingzhuo Liu, Yi Ban
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5393be1b38bee2c2ba8ad230d2f832b5bf465a65" target='_blank'>
              Abstract 2410: Copy number-based instability in breast cancer
              </a>
            </td>
          <td>
            Hannan Wong, Anna Korsakova, Andy Wu, Akila Perera, C. P. K. Patro, Shangying Chen, Shwetha Sundararaman, Vinay Warrier, Daniel G. Tenen, Jason J. Pitt
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0db92d7678fd17061debda8f9cf53b805a41693" target='_blank'>
              Abstract 7145: The pan-cancer landscape of RIT1 genetic alterations
              </a>
            </td>
          <td>
            Matteo Repetto, P. Selenica, S. Yang, B. Weigelt, A. Drilon
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce63b80e10f58d9373f4a7b7bf33e6cdea133649" target='_blank'>
              Abstract 6355: Mapping the genetic, epigenetic, and microenvironmental evolution of HER2-positive breast cancer during neoadjuvant therapy
              </a>
            </td>
          <td>
            Seongyeol Park, Eran Kotler, Zhicheng Ma, Kathleen E. Houlahan, Aziz Khan, Sophia Pribus, Christina Curtis
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98dbff58811770a1291ecf64b12b67c6f3cebb6f" target='_blank'>
              Abstract 6569: Extracellular vesicle delivery of functional extrachromosomal DNA in FGFR2-amplified cancer of unknown primary
              </a>
            </td>
          <td>
            Irene Salamon, Giulia Gallerani, G. Storci, Beatrice Fontana, Salvatore Serravalle, Francesco Valle, M. Brucale, Marco Pagano Mariano, Andrea Cavazzoni, R. Roncarati, Spartaco Santi, Massimiliano Bonafé, M. Ferracin
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a0c88288c8c7bf7b80395b40fbfcb9b8e990950" target='_blank'>
              Abstract 2834: CyclinE1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Shangwei Sun, Ling Lan, Jin Zhou, A. Bass
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471fa22c73f7c1a8083ff473dd937339ee492d0c" target='_blank'>
              Abstract A022: Decoding aneuploidy: Identifying drivers and therapeutic targets in recurrent breast cancer copy number alterations
              </a>
            </td>
          <td>
            Khalid N. Al-Zahrani, Ellen Langille, A. Obersterescu, Christopher Lowden, Katie Teng, Lauren Caldwell, David Cook, Miguel Pérez-Castro, Cynthia H Chiu, Alec Bahcheli, R. Tsai, Jacob Berman, Kin Chan, Linkang Zhang, K.W. Annie Bang, Michael Parsons, Adele Lopes, Jocelyn Nurtanto, E. I. Temel, Iosifina Fotiadou, Julien Dessapt, Hartland W. Jackson, Sean E Egan, J. Reimand, Jeffrey Wrana, Daniel Schramek
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0385c5e49d7e54e5c6f4ee5da5d2e85d080cf285" target='_blank'>
              Abstract 2848: Severus: A tool for detecting and characterizing complex structural variants in cancer using long-read sequencing
              </a>
            </td>
          <td>
            A. Keskus, A. Bryant, Tanveer Ahmad, Anton Goretsky, B. Yoo, Sergey Aganezov, Ataberk Donmez, L. Lansdon, Isabel Rodríguez, Jimin Park, Yuelin Liu, Xiwen Cui, J. Gardner, B. McNulty, Samuel Sacco, Jyoti Shetty, Yongmei Zhao, Bao Tran, G. Narzisi, Adrienne Helland, Daniel E. Cook, Pi-Chuan Chang, Alexey Kolesnikov, Andrew Carroll, Erin K. Molloy, Chengpeng Bi, Adam M. Walter, M. Gibson, I. Pushel, E. Guest, T. Pastinen, Kishwar Shafin, K. Miga, S. Malikić, Chi-Ping Day, N. Robine, Cenk Sahinalp, Michael Dean, M. Farooqi, B. Paten, M. Kolmogorov
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Background Transposons are DNA sequences able to move or copy themselves to other genomic locations leading to insertional mutagenesis. Although transposon-derived sequences account for half of the human genome, most elements are no longer transposition competent. Moreover, transposons are normally repressed through epigenetic silencing in healthy adult tissues but become derepressed in several human cancers, with high activity detected in colorectal cancer. Their impact on non-malignant and malignant tissue as well as the differences between somatic and germline retrotransposition remain poorly understood. With new sequencing technologies, including long read sequencing, we can access intricacies of retrotransposition, such as insertion sequence details and nested repeats, that have been previously challenging to characterize. Results In this study, we investigate somatic and germline retrotransposition by analyzing long read sequencing from 56 colorectal cancers and 112 uterine leiomyomas. We identified 1495 somatic insertions in colorectal samples, while striking lack of insertions was detected in uterine leiomyomas. Our findings highlight differences between somatic and germline events, such as transposon type distribution, insertion length, and target site preference. Leveraging long-read sequencing, we provide an in-depth analysis of the twin-priming phenomenon, detecting it across transposable element types that remain active in humans, including Alus. Additionally, we detect an abundance of germline transposons in repetitive DNA, along with a relationship between replication timing and insertion target site. Conclusions Our study reveals a stark contrast in somatic transposon activity between colorectal cancers and uterine leiomyomas, and highlights differences between somatic and germline transposition. This suggests potentially different conditions in malignant and non-malignant tissues, as well as in germline and somatic tissues, which could be involved in the transposition process. Long-read sequencing provided important insights into transposon behavior, allowing detailed examination of structural features such as twin priming and nested elements. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00357-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca6e6a5d11920586374ddcd1ad544515162f4af5" target='_blank'>
              Structural features of somatic and germline retrotransposition events in humans
              </a>
            </td>
          <td>
            Päivi Nummi, Tatiana Cajuso, T. Norri, Aurora Taira, Heli Kuisma, Niko Välimäki, A. Lepistö, L. Renkonen-Sinisalo, S. Koskensalo, T. Seppälä, Ari Ristimäki, Kyösti Tahkola, Anne Mattila, J. Böhm, J. Mecklin, Emma Siili, A. Pasanen, O. Heikinheimo, Ralf Bützow, A. Karhu, K. Burns, Kimmo Palin, Lauri A. Aaltonen
          </td>
          <td>2025-04-22</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Abstract The Philadelphia chromosome, the translocation between BCR and ABL genes, is seen in 95% of chronic myeloid leukemia (CML) patients. Although discovered >60 years ago, the molecular mechanism of BCR fragility is unclear. Here, we have identified several G4 DNA motifs at the BCR fragile region of CML patients. Various lines of experimentation revealed that the breakpoint regions could fold into multiple intramolecular G-quadruplex structures. The sodium bisulfite modification assay revealed single strandedness in the fragile region when present on a plasmid and human genome. Circular dichroism spectroscopy revealed the parallel G4 DNA formation, leading to polymerase arrest at the BCR breakpoints. Intracellular recombination assay revealed that DNA breakage at the BCR fragile region could join with the break generated by ISceI endonuclease. Finally, purified AID could bind and deaminate cytosines when present on single-stranded DNA generated due to G4 DNA, both in vitro and inside the cells. Therefore, our results suggest that AID binds to G4 DNA present at the BCR fragile region, resulting in the deamination of cytosines to uracil and induction of DNA breaks in one of the DNA strands, which can later get converted into a double-strand break, leading to t(9;22) chromosomal translocation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad133901fa4a2888984a520b619646b9998e103" target='_blank'>
              Formation of multiple G-quadruplexes contributes toward BCR fragility associated with chronic myelogenous leukemia
              </a>
            </td>
          <td>
            Shivangi Sharma, Elizabeth Thomas, Sumedha Dahal, Sayak Das, Shefali Kothari, U. Roy, Nitu Kumari, V. Gopalakrishnan, S. Raghavan
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f3af6173f4bd387223a85f81701f7e3b9e33817" target='_blank'>
              Abstract 1241: Deciphering heterogeneity and tumor evolution in bladder cancer using multi-omics and liquid biopsies from rapid autopsies
              </a>
            </td>
          <td>
            P. Itagi, Samantha L. Schuster, Sonali Arora, Patricia Galipeau, Thomas W Persse, F. Vakar‐López, John K. Lee, P. Grivas, R. Montgomery, Jonathan L. Wright, Hung-Ming Lam, Andrew C. Hsieh, Gavin Ha
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e60c4cd21a9e7eb8974a45714112351803792a88" target='_blank'>
              Abstract 6715: Cross-species investigation of gene copy number and cancer resistance
              </a>
            </td>
          <td>
            Morla Phan, Thompson Worden, Sarah Adamowicz, Stéphane Lair, Mauricio Seguel, Claire Jardine, Geoffrey A. Wood, Dirk Steinke
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/330a5593d326f97de03e4da3203d107ef7927e48" target='_blank'>
              Extending differential gene expression testing to handle genome aneuploidy in cancer
              </a>
            </td>
          <td>
            Katsiaryna Davydzenka, G. Caravagna, Guido Sanguinetti
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e79bd95f47abcb0c8154c2b9c5f4a7bf2825f70" target='_blank'>
              Abstract 2853: Engineered extrachromosomal oncogene amplifications promote tumorigenesis
              </a>
            </td>
          <td>
            Melissa A. Yao, Davide Pradella, Minsi Zhang, Rui Gao, Katarzyna M. Gluchowska, Ylenia Cendon Florez, Tanmay Mishra, Gaspare La Rocca, Moritz Weigl, Ziqi Jiao, Hieu H. M. Nguyen, Marta Lisi, Mateusz M. Ozimek, C. Mastroleo, Kevin Chen, Felix Grimm, J. Luebeck, Shu Zhang, Andrea Alice Zolli, Eric G Sun, Bhargavi Dameracharla, Zhengqiao Zhao, Yuri Pritykin, Carlie Siegel, Howard Y. Chang, P. Mischel, V. Bafna, C. Antonescu, Andrea Ventura
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>134</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/344c45deee3ee4d6c328b84a6e73872831228859" target='_blank'>
              Abstract B019: Cyclin E1 amplification promotes chromosomal instability and distinct vulnerabilities in gastroesophageal adenocarcinoma
              </a>
            </td>
          <td>
            Zhong Wu, Ankur Nagaraja, Chunyang Bao, Jin Zhou, A. Bass
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background The histone chaperone Holliday Junction Recognition Protein (HJURP) has been associated with multiple types of cancers, but its role in GC is not yet fully understood. Considering its functions in centromere stability and DNA repair, investigating HJURP’s role in GC may offer novel therapeutic perspectives. Methods HJURP expression was examined in a dataset comprising TCGA-STAD samples and an internal group of GC patients, utilizing RNA sequencing and Western blot techniques. Functional experiments were carried out on the AGS and HGC-27 GC cell lines. The expression levels of HJURP, MYC, and Topoisomerase II alpha (TOP2A) were assessed via quantitative real-time PCR and Western blot. Proliferation rates of the cells were determined through EdU, CCK-8, and colony formation assays. Results Compared to adjacent normal tissues, HJURP expression was notably increased in GC tissues, a finding consistent across both the TCGA-STAD database and our internal patient group. Silencing HJURP markedly reduced GC cell growth and chemoresistance. Mechanistically, HJURP enhanced MYC stability, which in turn promoted TOP2A transcription. Rescue experiments confirmed that overexpression of TOP2A alters proliferation and chemoresistance of GC cells with HJURP knockdown, indicating the dependency of this axis on MYC activity. Conclusion Our study demonstrates that HJURP is critical for promoting GC proliferation and chemoresistance through the regulation of the MYC/TOP2A transcriptional network. Targeting HJURP might offer a novel therapeutic avenue for GC, necessitating further exploration of its clinical potential. This work underscores the value of investigating histone chaperones as potential targets in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feb8f81bdc3f0ec8326d390d83a122cdb0440ebe" target='_blank'>
              HJURP modulates cell proliferation and chemoresistance via the MYC/TOP2A transcriptional axis in gastric cancer
              </a>
            </td>
          <td>
            Xu Li, Xiwen Li, Yanlin Ren, Ling Wang, Zehao Mao, Shikun Gao, Peng Ma, Junjie Chen
          </td>
          <td>2025-04-11</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6ae5fa1f276c91179bfcbf4e5cddc5ae1bea20" target='_blank'>
              Abstract 1169: FAT1 alterations contribute to chromosomal instability in cancer cells
              </a>
            </td>
          <td>
            Wei-Ting Lu, L. Zalmas, Chris Bailey, James R. M. Black, C. Martínez-Ruiz, O. Pich, F. Gimeno-Valiente, Ieva Usaite, M. Jamal-Hanjani, N. Tapon, K. Fugger, N. Mcgranahan, K. Litchfield, Nicolai J. Birkbak, Jiri Bartek, N. Kanu, C. Swanton
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac35aaec3d88c6b3cd4be700d6af5bf25aaebdef" target='_blank'>
              Drosophila SA1 expression prevents brain tumorigenesis and PARP-mediated cell elimination
              </a>
            </td>
          <td>
            Simona Totaro, Antonella Lettieri, Silvia Castiglioni, Francesco Lavezzari, C. Gervasini, Valentina Massa, Thomas Vaccari
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3c0c35a9f3d4bbbc4a89299183e6ee25dd0c9aa" target='_blank'>
              Abstract 904: Mapping the T cell-cold phenotype to the colorectal cancer genome identifies POFUT1 as a driver of tumor T cell inhibition
              </a>
            </td>
          <td>
            Calvin J. Wagner
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/391f121bccfd400a13a99c3af96bbd051ffc5650" target='_blank'>
              Cohesin mutations and chromatin changes in cancer.
              </a>
            </td>
          <td>
            Ariel D Swett, Zuzana Tothova
          </td>
          <td>2025-03-10</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9573a097c9d79cc31651e0409f2f3e1d71e62921" target='_blank'>
              Replication coordination marks the domestication of large extrachromosomal replicons in bacteria
              </a>
            </td>
          <td>
            Jakub Czarnecki, Morgan Lamberioux, O. Skovgaard, Amaury Bignaud, Najwa Taib, Théophile Niault, P. Bourhy, Julia Bos, Elvira Krakowska, D. Bartosik, Romain Koszul, M. Marbouty, Didier Mazel, Marie-Eve Val
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb8f2b1ddae01e13edf72bbcb5e1a3343151981e" target='_blank'>
              Abstract 7497: Nanopore sequencing of single-cell derived sublines provides insights into melanoma heterogeneity and evolution
              </a>
            </td>
          <td>
            Anton Goretsky, Yuelin Liu, A. Keskus, Tanveer Ahmad, S. Malikić, Glenn Merlino, Chi-Ping Day, Erin K. Molloy, S. C. Sahinalp, M. Kolmogorov
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2be5d83a44f2113bcc7ca92950bf19d6878d25c9" target='_blank'>
              Functional landscape of circular RNAs in human cancer cells
              </a>
            </td>
          <td>
            Peter Hyunwuk Her, Tiantian Li, Ziwei Huang, Xin Xu, Weining Yang, Moliang Chen, Mona Teng, Sujun Chen, Yong Zeng, Stanley Liu, Benjamin Haibe-Kains, Fraser Soares, Jie Ming, H. H. He
          </td>
          <td>2025-03-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Oncogenic gene fusions occur across a broad range of cancers and are a defining feature of some cancer types. Cancers driven by gene fusion products tend to respond well to targeted therapies, where available; thus, detection of potentially targetable oncogenic fusions is necessary to select optimal treatment. Detection methods include non-sequencing methods, such as fluorescence in situ hybridization and immunohistochemistry, and sequencing methods, such as DNA- and RNA-based next-generation sequencing (NGS). While NGS is an efficient way to analyze multiple genes of interest at once, economic and technical factors may preclude its use in routine care globally, despite several guideline recommendations. The aim of this review is to present a summary of oncogenic gene fusions, with a focus on fusions that affect tyrosine kinase signaling, and to highlight the importance of testing for oncogenic fusions. We present an overview of the identification of oncogenic gene fusions and therapies approved for the treatment of cancers harboring gene fusions, and summarize data regarding treating fusion-positive cancers with no current targeted therapies and clinical studies of fusion-positive cancers. Although treatment options may be limited for patients with rare alterations, healthcare professionals should identify patients most likely to benefit from oncogenic gene fusion testing and initiate the appropriate targeted therapy to achieve optimal treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a05e8c035d2b5c9d05b3addb1c0208226f39c0" target='_blank'>
              Oncogenic gene fusions in cancer: from biology to therapy
              </a>
            </td>
          <td>
            Stephen V Liu, Misako Nagasaka, Judith Atz, Flavio Solca, Leonhard Müllauer
          </td>
          <td>2025-04-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a02e806519d051257de949147c9dc431b4e0f1b1" target='_blank'>
              ER-tethering directs TREX1 penetration of a BAF-dependent barrier at micronuclei
              </a>
            </td>
          <td>
            Yanyang Chen, Eléonore Toufektchan, Xiaohan Luan, James H Hickling, R. Norman, P. Cifani, A. Kentsis, Wen Zhou, J. Maciejowski
          </td>
          <td>2025-04-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Ovarian cancer remains one of the most lethal gynecological malignancies owing to its high recurrence rate and chemotherapeutic resistance. MYC is a well-known proto-oncogene that is frequently amplified in ovarian cancer and has been implicated in drug resistance. Previously, we established a new promoter–reporter system combined with a CRISPR activation library to identify unknown MYC regulators, and M1AP was identified as a novel MYC regulator. However, considering the insufficient explanation for the absence of guide RNA (gRNA) of MYC, this present study explored methods to prevent the gRNA of MYC itself from binding. This study first modified the promoter–reporter vector to improve its quality, then conducted CRISPR screening and analyzed candidate genes as MYC promoter regulators using next-generation sequencing in OVSAHO ovarian cancer cells. Eighty-six genes had ≥ 1000 reads, and Pearson’s correlation coefficient analysis was performed on the cBioPortal of the Cancer Genomics database. Fourteen genes were identified as candidate MYC regulators with positive and significant correlations with MYC. Seven genes, including CYP4v2, ASPH, ANP32D, PCED1A, ABI1, FUZ, and HOOK2, demonstrated significantly higher luciferase activity than the control genes. Four genes, including ABI1, PCED1A, HOOK2, and CYP4v2, activated the MYC promoter, which showed over twofold higher activity than the control when overexpressed using a vector. In conclusion, four genes that activate MYC promoters were identified in an ovarian cancer cell line using the CRISPR library system with a modified promoter–reporter tool. These results will prove helpful in the development of novel treatment strategies for ovarian cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01644-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9220d2085ff0b29849fe6d91dbca5cdaecc789b" target='_blank'>
              Modified screening of MYC promotor region elements using the CRISPR library in ovarian cancer
              </a>
            </td>
          <td>
            Akiko Yamamoto, Yosuke Tanaka, S. Ishibashi, Masumi Ikeda, Keisuke Sugita, Masanori Ono, Hirotaka Nishi, Morito Kurata
          </td>
          <td>2025-04-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e44c370bd7521d24afccebc5dd604dc8d4b6c7b" target='_blank'>
              Abstract 2617: Uncovering genomic complexity of PAX5 internal tandem duplication using short-read and long-read sequencing
              </a>
            </td>
          <td>
            Yanling Liu, B. Ju, Li Dong, Melanie Loyd, Yuan Feng, H. Mulder, P. Kolekar, Li Fan, Hanxia Li, S. Foy, Geoffrey Neale, Patrick A. Brown, William L Carroll, M. Loh, John Easton, Xiaotu Ma
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/841194928a5faee8a089ace06821025bbd9fd89a" target='_blank'>
              Abstract LB347: Telomere-to-telomere long read whole genome sequencing provides novel insights into ovarian cancer genomics
              </a>
            </td>
          <td>
            S. Takamatsu, Jian Li, Thomas Welte, Eleonora Khlebus, Veena K Vuttaradhi, Allison L. Brodsky, B. Lawson, R. Hillman
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1acceac89bbf719cc0251731ef30332c326a163f" target='_blank'>
              Abstract 1428: Dissecting a triple-negative breast cancer super enhancer: insights into oncogenic gene regulation
              </a>
            </td>
          <td>
            Natalia Maldonado Vázquez, Nicole Zaragoza Rodríguez, Hector L. Franco
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b3cddb316bdce4c6828dd308d722612ae6e4f52" target='_blank'>
              Abstract 1070: siderGrimoire: a database of retroCNVs in human tumors and populations
              </a>
            </td>
          <td>
            T. L. Miller, Helena Beatriz da Conceição, P. A. Galante
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e3561acbee649acf0e09cb6b6caaf71d6c0db30" target='_blank'>
              Distinct mechanisms underlying extrachromosomal telomere DNA generation in ALT cancers
              </a>
            </td>
          <td>
            Junyeop Lee, Eric J. Sohn, Jina Lee, Angelo Taglialatela, Alberto Ciccia, Jaewon Min
          </td>
          <td>2025-04-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract TAQing technologies are based on the restriction enzyme-induced DNA double-strand break (DSB) formation in living cells, which results in large-scale genomic rearrangements and phenotypic alterations. Originally, the TAQing system requires heat treatments to activate restriction enzymes, which sometimes leads to cell toxicity or stress responses. Here, we developed a blue-light-controlled MagTAQing system, which induces DSBs exclusively upon blue-light exposure by assembling the split restriction enzymes via Magnet modules. Application of MagTAQing to mitotic budding yeast cells successfully triggered various genomic rearrangements upon blue-light exposure. Since this technology enables the conditional induction of genomic rearrangements in specific cells or tissues, we employed MagTAQing on meiotic yeast cells lacking the recombinase Spo11 to induce artificial DSBs. Consequently, Spo11-independent meiotic DSBs resulted in aneuploidies and nonallelic homologous recombinations between repetitive sequences such as ribosomal DNA and retrotransposons. These results suggest a pivotal role of Spo11-induced recombination in preventing chromosomal abnormality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72c560ade8948ca0b121c00661a4ad61330184b0" target='_blank'>
              Light-controlled Spo11-less meiotic DNA breaks by MagTAQing lead to chromosomal aberrations
              </a>
            </td>
          <td>
            Hideyuki Yone, Yuri Kawashima, Hayato Hirai, Arisa H. Oda, Moritoshi Sato, Hiromitsu Kono, Kunihiro Ohta
          </td>
          <td>2025-04-10</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Endemic Burkitt lymphoma (eBL) is one of the most prevalent cancer in children in sub-Saharan Africa, and while prior studies have found that Epstein-Barr virus (EBV) type and variation may alter the tumor driver genes necessary for tumor survival, the precise relationship between EBV variation and EBV-associated tumorigenesis remains unclear due to lack of scalable, cost-effective, viral whole-genome sequencing from tumor samples. This study introduces a rapid and cost-effective method of enriching, sequencing, and assembling accurate EBV genomes in BL tumor cell lines through a combination of selective whole genome amplification (sWGA) and subsequent 2-tube multiplex polymerase chain reaction along with long-read sequencing with a portable sequencer. The method was optimized across a range of parameters to yield a high percentage of EBV reads and sufficient coverage across the EBV genome except for large repeat regions. After optimization, we applied our method to sequence 18 cell lines and 3 patient tumors from fine needle biopsies and assembled them with median coverages of 99.62 and 99.68%, respectively. The assemblies showed high concordance (99.61% similarity) to available Illumina-based assemblies. The improved method and assembly pipeline will allow for better understanding of EBV variation in relation to BL and is applicable more broadly for translational research studies, especially useful for laboratories in Africa where eBL is most widespread. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-94737-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4545234794b307a82810b0886fef390a78786e91" target='_blank'>
              Comparison of nanopore with illumina whole genome assemblies of the Epstein-Barr virus in Burkitt lymphoma
              </a>
            </td>
          <td>
            Isaac E. Kim, A. Fola, Enrique Puig, T. K. Maina, Sin Ting Hui, Hongyu Ma, Kaleb Zuckerman, Eddy O. Agwati, A. Leonetti, Rebecca Crudale, Micah A. Luftig, A. M. Moormann, Cliff I. Oduor, Jeffrey A. Bailey
          </td>
          <td>2025-03-31</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background The ataxia-telangiectasia mutated (ATM) kinase phosphorylates and activates several downstream targets that are essential for DNA damage repair, cell cycle inhibition and apoptosis. Germline biallelic inactivation of the ATM gene causes ataxia-telangiectasia (A-T), and heterozygous pathogenic variant (PV) carriers are at increased risk of cancer, notably breast cancer. This study aimed to investigate whether DNA methylation profiling can be useful as a biomarker to identify tumors arising in ATM PV carriers, which may help for the management and optimal tailoring of therapies of these patients. Methods Breast tumor enriched DNA was prepared from 2 A-T patients, 27 patients carrying an ATM PV, 6 patients carrying a variant of uncertain clinical significance and 484 noncarriers enrolled in epidemiological studies conducted in France and Australia to investigate genetic and nongenetic factors involved in breast cancer susceptibility. Genome-wide DNA methylation analysis was performed using the Illumina Infinium HumanMethylation EPIC and 450K BeadChips. Correlation between promoter methylation and gene expression was assessed for 10 tumors for which transcriptomic data were available. Results We found that the ATM promoter was hypermethylated in 62% of tumors of heterozygous PV carriers compared to the mean methylation level of ATM promoter in tumors of noncarriers. Gene set enrichment analyses identified 47 biological pathways enriched in hypermethylated genes involved in neoplastic, neurodegenerative and metabolic-related pathways in tumor of PV carriers. Among the 327 differentially methylated promoters, promoters of ARHGAP40, SCGB3A1 (HIN-1), and CYBRD1 (DCYTB) were hypermethylated and associated with a lower gene expression in these tumors. Moreover, using three different deep learning algorithms (logistic regression, random forest and XGBoost), we identified a set of 27 additional biomarkers predictive of ATM status, which could be used in the future to provide evidence for or against pathogenicity in ATM variant classification strategies. Conclusions We showed that breast tumors that arise in women who carry an ATM PV display a specific genome-wide DNA methylation profile. Specifically, the methylation pattern of 27 key gene promoters was predictive of ATM PV status of the women. These genes may also represent new medical prevention and therapeutic targets for these women. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-01988-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bebfd1b3f4c2e2d684ea0f216e88d7bab9a7fec5" target='_blank'>
              Breast tumors from ATM pathogenic variant carriers display a specific genome-wide DNA methylation profile
              </a>
            </td>
          <td>
            Nicolas M Viart, Anne-Laure Renault, Séverine Eon-Marchais, Yue Jiao, L. Fuhrmann, Sophia Murat El Houdigui, Dorothée Le Gal, Eve Cavaciuti, Marie-Gabrielle Dondon, Juana Beauvallet, Virginie Raynal, D. Stoppa-Lyonnet, A. Vincent-Salomon, Nadine Andrieu, M. Southey, F. Lesueur
          </td>
          <td>2025-03-11</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="The study examines Opisthorchis viverrini (OV)-related cholangiocarcinoma (CCA), a serious malignancy common in Southeast Asia. Through multi-regional whole-exome sequencing of 52 tumor samples and 13 adjacent tissues from 13 patients, significant intratumoral heterogeneity (ITH) and inter-patient heterogeneity are shown. Chronic liver fluke infection induces a distinct mutational landscape, with 48–90% of mutations concentrated in each region of the tumor. The average mutation burden is 95 non-synonymous mutations per area, exceeding previous CCA investigations. Critical driver mutations in TP53, SMAD4, and other genes underscore their significance in pathogenesis. Mutational markers elucidate mechanisms including spontaneous deamination and impaired DNA repair. Unique mutation patterns distinguish OV-associated CCA from other variants. Chromosomal instability in patient K110 signifies aggressive tumor behavior and unfavorable prognosis. Targetable mutations such as ERBB2 underscore the possibility for personalized therapeutics. These findings underscore the necessity for personalized strategies for treatment that target both trunk and branch mutations in endemic areas. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-95142-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b48c940bd80f998f283a697250c46b3404991cb4" target='_blank'>
              Whole exome sequencing of multi-regions reveals tumor heterogeneity in Opisthorchis viverrini-associated cholangiocarcinoma
              </a>
            </td>
          <td>
            Sirinya Sitthirak, Arporn Wangwiwatsin, A. Jusakul, N. Namwat, P. Klanrit, Hasaya Dokduang, P. Sa-Ngiamwibool, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Bin Tean Teh, Luke Boulter, Yoshinori Murakami, W. Loilome
          </td>
          <td>2025-03-29</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f63a435551ab128bd453aa26cc898c619184a9dc" target='_blank'>
              Abstract 6238: Interactions between ploidy and resource availability shape clonal interference in glioblastoma
              </a>
            </td>
          <td>
            Z. Nowicka, Frederika Rentzeperis, Vural Tagal, Konstantin Maksin, Parag Katira, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Genome regulation relies on complex and dynamic interactions between DNA and proteins. Recently, powerful methods have emerged that leverage third-generation sequencing to map protein-DNA interactions genome-wide. For example, Directed Methylation with Long-read sequencing (DiMeLo-seq) enables mapping of protein-DNA interactions along long, single chromatin fibers, including in highly repetitive genomic regions. However, DiMeLo-seq involves lossy centrifugation-based wash steps that limit its applicability to many sample types. To address this, we developed DiMeLo-cito, a single-tube, wash-free protocol that maximizes the yield and quality of genomic DNA obtained for long-read sequencing. This protocol enables the interrogation of genome-wide protein binding with as few as 100,000 cells and without the requirement of a nuclear envelope, enabling confident measurement of protein-DNA interactions during mitosis. Using this protocol, we detected strong binding of CTCF to mitotic chromosomes in diploid human cells, in contrast with earlier studies in karyotypically unstable cancer cell lines, suggesting that CTCF “bookmarks” specific sites critical for maintaining genome architecture across cell divisions. By expanding the capabilities of DiMeLo-seq to a broader range of sample types, DiMeLo-cito can provide new insights into genome regulation and organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69924e2da138038fc52ccdade72f6d918fe9612d" target='_blank'>
              DiMeLo-cito: a one-tube protocol for mapping protein-DNA interactions reveals CTCF bookmarking in mitosis
              </a>
            </td>
          <td>
            Nathan Gamarra, Cy Chittenden, Kousik Sundararajan, Jacob P. Schwartz, Sofia Lundqvist, Denise Robles, Oberon Dixon-Luinenburg, Jeremy Marcus, Annie Maslan, J. M. Franklin, Aaron M. Streets, A. Straight, Nicolas Altemose
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cb48bc35dc30e8efcbd8268b117fd6382f30da8" target='_blank'>
              Abstract 2395: The role of exonized mobile elements in chimeric transcript emergence in glioblastoma
              </a>
            </td>
          <td>
            Rafael L. V. Mercuri, T. L. Miller, Filipe F. dos Santos, M. D. de Lima, Aline Rangel-Pozzo, P. A. Galante
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6879a0f12b521e3cdca31a407d6e24ccb72cd9a1" target='_blank'>
              AZD7648 (DNA-PKcs inhibitor): a two-edged sword for editing genomes.
              </a>
            </td>
          <td>
            Muhammad Waseem Sajjad, Ifrah Imran, Fatima Muzamil, R. Naqvi, Imran Amin
          </td>
          <td>2025-03-01</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e62989c3597687f0bc3f412a31ec866e11a879ff" target='_blank'>
              Characterization of extrachromosomal circular DNA associated with genomic repeat sequences in breast cancer.
              </a>
            </td>
          <td>
            Wenxiang Lu, Lingsong Yao, Ying Wang, Fuyu Li, Bingbo Zhou, Wenlong Ming, Yali Jiang, Xiaoan Liu, Yun Liu, Xiao Sun, Yan Wang, Yunfei Bai
          </td>
          <td>2025-03-26</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c0e44d4f49dc3c34ac8ec8c2f4c7c94f3fd34e" target='_blank'>
              Abstract 2854: The role of NONO in genome stability: Implications for RNA-DNA hybrid accumulation, genomic instability, and inflammatory signaling
              </a>
            </td>
          <td>
            Vikas Gaur, Sara Knowles, Weihang Chai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0fc58e0b6388ab5fc82e889af73612c384df0b6" target='_blank'>
              Abstract 5327: CDK2 inhibition produces persistent polyploid cancer cells with a survival advantage
              </a>
            </td>
          <td>
            Liliya Tyutyunyk‐Massey, Zibo Chen, Xiuxia Liu, Masanori Kawakami, Adam Harned, Yeap Ng, Brian Luke, Sameul C. Okpechi, Blessing Ogunlade, Yair Alfaro, Roberto Weigert, Kedar Narayan, Xi Liu, Ethan Dmitrovsky
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/279bd9fa8c8b49a99cb9a4bb9596137f5ef0a74d" target='_blank'>
              Sc-TUSV-Ext: Single-Cell Clonal Lineage Inference from Single Nucleotide Variants, Copy Number Alterations, and Structural Variants.
              </a>
            </td>
          <td>
            N. Bristy, Xuecong Fu, Russell Schwartz
          </td>
          <td>2025-03-06</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94444bc68fc048c7923ba3b90250650c889cfa29" target='_blank'>
              Methylglyoxal mutagenizes single-stranded DNA via Rev1-associated slippage and mispairing
              </a>
            </td>
          <td>
            Sriram Vijayraghavan, Alessandra Ruggiero, Samuel Becker, P. Mieczkowski, George S Hanna, Mark T. Hamann, Natalie Saini
          </td>
          <td>2025-03-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Chromosomal instability (CIN) is a hallmark of cancer that drives metastasis, immune evasion and treatment resistance. CIN may result from chromosome mis-segregation errors and excessive chromatin is frequently packaged in micronuclei (MN), which can be enumerated to quantify CIN. The assessment of CIN remains a predominantly manual and time-consuming task. Here, we present micronuclAI, a pipeline for automated and reliable quantification of MN of varying size and morphology in cells stained only for DNA. micronuclAI can achieve close to human-level performance on various human and murine cancer cell line datasets. The pipeline achieved a Pearson’s correlation of 0.9278 on images obtained at 10X magnification. We tested the approach in otherwise isogenic cell lines in which we genetically dialed up or down CIN rates, and on several publicly available image datasets where we achieved a Pearson’s correlation of 0.9620. Given the increasing interest in developing therapies for CIN-driven cancers, this method provides an important, scalable, and rapid approach to quantifying CIN on images that are routinely obtained for research purposes. We release a GUI-implementation for easy access and utilization of the pipeline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ad501d60562e0f7baf32d920b40de21a7a01759" target='_blank'>
              micronuclAI enables automated quantification of micronuclei for assessment of chromosomal instability
              </a>
            </td>
          <td>
            Miguel A. Ibarra-Arellano, Lindsay A. Caprio, Aroj Hada, Niklas Stotzem, Luke L Cai, Shivem B Shah, Zachary H. Walsh, Johannes C. Melms, Florian Wünneman, K. Bestak, Ibrahim Mansaray, Benjamin Izar, Denis Schapiro
          </td>
          <td>2025-03-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Tumors exhibit highly heterogeneous genomic and transcriptomic landscapes. Characterization of this intratumor heterogeneity (ITH) and its implications for the cancer therapies has been one of the focal points of computational cancer biology in the past years. However, while ITH at the level of genomic changes, such as copy number aberrations, has been well characterized, the downstream impact of ITH on the gene regulatory landscape remains underexplored. Disruptions in gene regulatory programs can lead to emergence of more aggressive or drugresistant tumor clones. In order to better understand whether the evidence for ITH at the gene regulatory level can be inferred from scRNA-seq data we have conducted a series of experiments that explore the levels of gene regulatory network (GRN) discrepancies between the clonal sub-populations of tumor cells. In particular, we quantify the extent to which GRNs inferred from different tumor clones diverge from each other in three settings: triple negative breast cancer, colorectal cancer, and a longitudinal dataset from a patient undergoing chemotherapy for breast cancer. Our analyses indicate that a substantial (15-25%) proportion of GRN edges can be attributed to clone-specific activity. Furthermore, we observe differential network patterns across outlier transcription factors implicated in tumorogenesis and tumor progression. Our findings suggest that differential network patterns between clonal subpopulations can be reliably identified, and that such patterns offer a complementary insight that is not captured by differential gene analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86413a2b4bf81ee584f265b2f474b3de585e8771" target='_blank'>
              Intratumor Heterogeneity Through the Lens of Gene Regulatory Networks
              </a>
            </td>
          <td>
            Narein Rao, Nicolae Sapoval, Hamim Zafar, L. Nakhleh
          </td>
          <td>2025-04-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e93a4683d0794daa101a332796e1ced942dd7fd5" target='_blank'>
              A Novel De Novo Exceptional Complex Chromosomal Rearrangement Involving 5 Chromosomes Resulting in Neurodevelopmental Delay and Dysmorphism
              </a>
            </td>
          <td>
            Sabri Aynacı, Sinem Kocagil, C. Yarar, Efsun Tosumoğlu, Ebru Erzurumluoğlu Gökalp, Mehmet Burak Mutlu, S. Artan
          </td>
          <td>2025-03-25</td>
          <td>Molecular Syndromology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a9bc7f4f8a4cf97f32bbe5a9d88c7eccc61afe" target='_blank'>
              Abstract 2923: Targeting DNA repair mechanisms to overcome CIC::DUX4 undifferentiated sarcomas
              </a>
            </td>
          <td>
            Zeinab Kosibaty, Cuyler Luck, Ross A. Okimoto
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c3d68234ed3b90b54b46e31e83216b5a77d6d67" target='_blank'>
              Abstract 5055: Efficient detection of chromosomal instability in diverse cancers using a targeted sequencing approach
              </a>
            </td>
          <td>
            H. Bao, Rui Liu, Haimeng Tang, Ningyou Li, Min Wu, Xunbiao Liu, Zhili Chang, Xue Wu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1437313dac6bd0a59138128e06745ba119fe182" target='_blank'>
              Does chromoanagenesis play a role in the origin of B chromosomes?
              </a>
            </td>
          <td>
            Andreas Houben, Jörg Fuchs, A. Banaei-Moghaddam, Jianyong Chen, Gihwan Kim, Taoran Liu
          </td>
          <td>2025-04-19</td>
          <td>Heredity</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The human APOBEC superfamily consists of eleven cytidine deaminase enzymes. Among them, APOBEC3 enzymes play a dual role in antiviral immunity and cancer development. APOBEC3 enzymes, including APOBEC3A (A3A) and APOBEC3B (A3B), induce mutations in viral DNA, effectively inhibiting viral replication but also promoting somatic mutations in the host genome, contributing to cancer development. A3A and A3B are linked to mutational signatures in over 50% of human cancers, with A3A being a potent mutagen. A3B, one of the first APOBEC3 enzymes linked to carcinogenesis, plays a significant role in HPV-associated cancers by driving somatic mutagenesis and tumor progression. The A3A_B deletion polymorphism results in a hybrid A3A_B gene, leading to increased A3A expression and enhanced mutagenic potential. Such polymorphism has been linked to an elevated risk of certain cancers, particularly in populations where it is more prevalent. This review explores the molecular mechanisms of APOBEC3 proteins, highlighting their dual roles in antiviral defense and tumorigenesis. We also discuss the clinical implications of genetic variants, such as the A3A_B polymorphism, mainly in HPV infection and associated cancers, providing a comprehensive understanding of their contributions to both viral restriction and cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2370c8c875291e3fc4adec0ad421696e86fb060" target='_blank'>
              APOBEC3 Proteins: From Antiviral Immunity to Oncogenic Drivers in HPV-Positive Cancers
              </a>
            </td>
          <td>
            Eliza Pizarro Castilha, Rosalba Biondo, K. P. Trugilo, Giulia Mariane Fortunato, Tim R. Fenton, Karen Brajão de Oliveira
          </td>
          <td>2025-03-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744396e4904ca408c0b73628466fecf1a1f5b26d" target='_blank'>
              Kallmann syndrome due to balanced X chromosomal pericentric inversion disrupting ANOS1.
              </a>
            </td>
          <td>
            Michihiko Aramaki, Takashi Hamajima, E. Suzuki, M. Fukami, Keiko Matsubara
          </td>
          <td>2025-04-22</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a306f36a45e59527afe8dc83f82d2317d23a26f" target='_blank'>
              Abstract A035: Functional profiling of chimeric RNA transcripts identifies new biomarkers and therapeutic targets for precision medicine in cancer
              </a>
            </td>
          <td>
            Samir Lalani, Mason Ingram, Sehajroop Gadh, Hui Li
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Introduction of uracils in specific regions within immunoglobulin genes by the activation-induced deaminase (AID) promotes mutations and double-strand breaks (DSBs). Although uracils are repaired through multiple DNA repair pathways, previous work has used mutations or DSBs as proxies for uracils and not mapped the uracils directly. We mapped uracils in the Ig heavy chain gene, Igh, in a murine cell line, CH12F3, undergoing class-switch recombination (CSR) using the uracil pull-down and sequencing technique. These cells undergo IgM-to-IgA switch upon expression of AID but do not undergo somatic hypermutation. We mapped uracils in cells defective in uracil repair and show that AID introduces high levels of uracils only in parts of switch-mu and switch-alpha regions and not in constant regions, the Igh variable region or the light chain genes. Furthermore, the peaks of uracilation match the previously determined distribution of switch junctions, which are representative of DSBs that cause isotype switching. This work confirms that AID creates uracils in both DNA strands and shows that there is a direct correlation between uracil creation and DSBs in the relevant switch regions. We evaluate proposed mechanisms of CSR in light of these findings and show that mapping uracils provides a fresh perspective on CSR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/185a8afee40ca367e856c26300572cce1c4d6efa" target='_blank'>
              Nucleotide level mapping of uracils in murine heavy chain switch regions shows correlation between uracilation and positions of switch junctions created during class-switch recombination
              </a>
            </td>
          <td>
            Rukshana Mohamad-Ramshan, Ramin Sakhtemani, Y. Butt, Michael S Lawrence, Ashok S Bhagwat
          </td>
          <td>2025-01-01</td>
          <td>Nar Molecular Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0be267d81c51ec120632413ac620f868fe9f5634" target='_blank'>
              Rapid delivery of RNase H1 enables replication fork progression under stress independently of R-loop resolution
              </a>
            </td>
          <td>
            Amandine Basille, Yea-Lih Lin, Connor Kirke, D. Cluet, Yara Bawadi, Laetitia Vachez, Virgile Dervieux, Laura Guiguettaz, Pascal Bernard, Stephan Hamperl, Philippe Pasero, Emiliano P Ricci, Vincent Vanoosthuyse
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/517df0bcb0ecef68788668311a97fb91ddb06252" target='_blank'>
              Treacle and MDC1 coordinate rDNA break repair by homologous recombination
              </a>
            </td>
          <td>
            Andrea Haenel, Johannes Leyrer, Manuel Stucki
          </td>
          <td>2025-04-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17d143808abe3792603e16e2ab02c9eaad5e2640" target='_blank'>
              Abstract 2409: Examining APOBEC3B in TCGA and ICGC breast cancer datasets reveals altered drug metabolism pathways
              </a>
            </td>
          <td>
            Joel Pardo, N. Temiz, Douglas Yee
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cf164c6f6937a8921f8d066db65a0f028f3415e" target='_blank'>
              Abstract 3891: Integrative single-cell tracking of genome evolution and tumor cell plasticity in small cell lung cancer (SCLC)
              </a>
            </td>
          <td>
            Marcel Schmiel, L. Kaiser, Julie George, Roman K. Thomas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background Chromosomal instability (CIN) is involved in about 70% of colorectal cancers (CRCs) and is associated with poor prognosis and drug resistance. From a clinical perspective, a better knowledge of these tumour’s biology will help to guide therapeutic strategies more effectively. Methods We used high-density chromosomal microarray analysis to evaluate CIN level of patient-derived organoids (PDOs) and their original mCRC tissues. We integrated the RNA-seq and mass spectrometry-based proteomics data from PDOs in a functional interaction network to identify the significantly dysregulated processes in CIN. This was followed by a proteome-wGII Pearson correlation analysis and an in silico validation of main findings using functional genomic databases and patient-tissues datasets to prioritize the high-confidence CIN features. Results By applying the weighted Genome Instability Index (wGII) to identify CIN, we classified PDOs and demonstrated a good correlation with tissues. Multi-omics analysis showed that our organoids recapitulated genomic, transcriptomic and proteomic CIN features of independent tissues cohorts. Thanks to proteotranscriptomics, we uncovered significant associations between mitochondrial metabolism and epithelial-mesenchymal transition in CIN CRC PDOs. Correlating PDOs wGII with protein abundance, we identified a subset of proteins significantly correlated with CIN. Co-localisation analysis in PDOs strengthened the putative role of IPO7 and YAP, and, through in silico analysis, we found that some of the targets give significant dependencies in cell lines with CIN compatible status. Conclusions We first demonstrated that PDO models are a faithful reflection of CIN tissues at the genetic and phenotypic level. Our new findings prioritize a subset of genes and molecular processes putatively required to cope with the burden on cellular fitness imposed by CIN and associated with disease aggressiveness. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03308-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39fada8ad025d98e9dfc80fcb5a196964bddeb3e" target='_blank'>
              Decoding chromosomal instability insights in CRC by integrating omics and patient-derived organoids
              </a>
            </td>
          <td>
            Federica Papaccio, M. Cabeza-Segura, B. Garcia-Micò, F. Gimeno-Valiente, S. Zúñiga-Trejos, V. Gambardella, M. F. Gutierrez-Bravo, C. Martínez-Ciarpaglini, P. Rentero-Garrido, Tania Fleitas, Susana Roselló, J. Carbonell-Asins, M. Huerta, D. Moro-Valdezate, D. Roda, N. Tarazona, Manuel M. Sánchez del Pino, Andrés Cervantes, Josefa Castillo
          </td>
          <td>2025-02-28</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4121bda9e26ae184324ef8e02b596fbe5bd446bc" target='_blank'>
              Abstract 1200: FET fusion oncoproteins disrupt physiologic DNA repair and create a targetable opportunity for ATR inhibitor therapies
              </a>
            </td>
          <td>
            Shruti Menon, Daniel Gracilla, Marcus R. Breese, F. D. Dela Cruz, Asmin Tulpule
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Although two-thirds of cancers arise from loss-of-function mutations in tumor suppressor genes, there are few approved targeted therapies linked to these alterations. Synthetic lethality offers a promising strategy to treat such cancers by targeting vulnerabilities unique to cancer cells with these mutations. To identify clinically relevant synthetic lethal interactions, we analyzed genome-wide CRISPR/Cas9 knock-out (KO) viability screens from the Cancer Dependency Map and evaluated their clinical relevance in patient tumors through mutual exclusivity, a pattern indicative of synthetic lethality. Indeed, we found significant enrichment of mutual exclusivity for interactions involving cancer driver genes compared to non-driver mutations. To identify therapeutic opportunities, we integrated drug sensitivity data to identify inhibitors that mimic the effects of CRISPR-mediated KO. This approach revealed potential drug repurposing opportunities, including BRD2 inhibitors for bladder cancers with ARID1A mutations and SIN3A-mutated cell lines showing sensitivity to nicotinamide phosphoribosyltransferase (NAMPT) inhibitors. However, we discovered that pharmacological inhibitors often fail to phenocopy KO of matched drug targets, with only a small fraction of drugs inducing similar effects. This discrepancy reveals fundamental differences between pharmacological and genetic perturbations, emphasizing the need for approaches that directly assess the interplay of loss-of-function mutations and drug activity in cancer models. Author Summary Synthetic lethality is an emerging approach for targeting a biological dependency in cancer cells that does not harm normal cells. This strategy is particularly valuable for targeting loss-of-function mutations in tumor suppressor genes, which are more challenging to directly target. In an effort to accelerate treatments for cancer patients, we aimed to map out these dependencies and overlap them with responses to available drugs. We discovered different outcomes when a protein is targeted by a drug versus when that same target is disrupted genetically. Thus, if a drug is to be effectively repurposed as synthetic lethal agent, feasibility studies must capture drug biology, ideally by test the drug empirically in relevant cancer models. A second notable discovery is that in vitro synthetic lethal interactions involving cancer driver genes are significantly more likely to exhibit consistent patterns, such as mutual exclusivity in human tumor samples. This is important since selection of relevant cell lines is often critical in drug development to maximize potential for translation to clinical responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4c1469ac01d0657e51e2a370ba73f2735664c7f" target='_blank'>
              Challenges and opportunities for drug repurposing in cancers based on synthetic lethality induced by tumor suppressor gene mutations
              </a>
            </td>
          <td>
            Michael Vermeulen, Andrew W. Craig, Tomas Babak
          </td>
          <td>2025-03-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c29e60f63dffafbff9d001a2d3645ac87336a0cf" target='_blank'>
              G6PC3 promotes genome maintenance and is a candidate mammary tumor suppressor.
              </a>
            </td>
          <td>
            Xin Li, Maria Rossing, Ana Moisés da Silva, Muthiah Bose, Thorkell Gudjónsson, Jan Benada, J. Thatte, Jens Vilstrup Johansen, J. Börcsök, H. van der Gulden, Ji-Ying Song, Renée Menezes, Asma Tajik, Lucía Sena, Zoltan Szallasi, Morten Frödin, J. Jonkers, F. C. Nielsen, C. S. Sørensen
          </td>
          <td>2025-04-22</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/545b6d8dbebbe17da0f379631a55f84485df7054" target='_blank'>
              The Significance of Chromosome Conformation Capture in 3D Genome
Architecture Comprehension
              </a>
            </td>
          <td>
            Emre Sefer
          </td>
          <td>2025-04-22</td>
          <td>Current Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4c41f6c6066f9f1caf2b6797a1f6cfc0b6b61d7" target='_blank'>
              Abstract 1958: Chromatin accessibility landscapes of colorectal cancer development and genome-wide cell-free DNA fragmentation
              </a>
            </td>
          <td>
            Nicholas A. Vulpescu, Zachariah H. Foda, P. Wisse, Christopher Cherry, J. Medina, Stephen B. Baylin, Kasper D. Hansen, V. Adleff, R. Fijneman, Michael A. Beer, Robert B Scharpf, G. Meijer, Beatriz Carvalho, V. Velculescu
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa24230eaf15e64077b2640e42b580f6dcef4855" target='_blank'>
              Abstract 2838: Modeling putative oncogenic variants of human topoisomerase II by genome editing of a near haploid cell line
              </a>
            </td>
          <td>
            K. C. Nitiss, Brigid Conroy, Maureen N. McCoy, J. Nitiss
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c86b0ba33764c9febe1e75cc4ff8baa0ec4e4b55" target='_blank'>
              Abstract 990: Role pf viral infection in APOBEC3 dysreguation in cancer: a single-cell gene expression profiling approach
              </a>
            </td>
          <td>
            Mohadeseh Soleimanpour-Moghadam, diako Ebrahimi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0eed4347ff4f53b8d985b0523c9458323314af82" target='_blank'>
              Abstract 5418: Targeting PKMYT1 to preferentially kill tumor cells that have CCNE1 amplifications
              </a>
            </td>
          <td>
            Samantha S. Hodge, Natalie Hill, Kayla Walzer, Ritu Kushwaha, Yihong Guan, Jeremy Fournier, Ester Fernandez-Salas
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/187ffa69b09f016709afee92a536f50201d6d4b2" target='_blank'>
              Abstract 3926: The effect of Pten haploinsufficiency on genomic and transcriptomic profiles in combined genetically-engineered mouse models for colonic neoplasia based on Apc inactivation
              </a>
            </td>
          <td>
            H. Sada, Hiroaki Niitsu, Hikaru Nakahara, Naoya Sakamoto, Hirotaka Tashiro, Hideki Ohdan, T. Hinoi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827f74e9a451d846709a409162aebd5ef66250d5" target='_blank'>
              Abstract LB468: Histological subtypes of metastatic bladder cancer have distinct tumor microenvironments and high intratumoral heterogeneity
              </a>
            </td>
          <td>
            Samantha L. Schuster, P. Itagi, Sonali Arora, Thomas W Persse, Patricia Galipeau, John K. Lee, R. Montgomery, , P. Grivas, F. Vakar‐López, Andrew C. Hsieh, Hung-Ming Lam, Gavin Ha
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background Oxaliplatin-based chemotherapy is the first-line treatment for colorectal cancer (CRC). However, oxaliplatin resistance remains a major challenge contributing to treatment failure and poor prognosis. An increased capacity for DNA damage repair is a key mechanism underlying oxaliplatin resistance. Although XPA binding protein 2 (XAB2) is implicated in various DNA damage repair mechanisms, its specific role in mediating oxaliplatin resistance remains unclear. Methods XAB2 was identified through analysis of public datasets. Western blot analysis and immunohistochemistry were performed to evaluate XAB2 expression, while survival analysis was performed to assess its clinical significance in CRC. Functional experiments were then conducted to assess the impact of XAB2 on proliferation, DNA damage repair, and oxaliplatin resistance in CRC. RNA sequencing (RNA-seq) and Chromatin immunoprecipitation-sequencing (ChIP-seq) were used to identify XAB2 target genes. Co-immunoprecipitation (Co-IP) and mass spectrometry were used to identify the proteins interacting with XAB2. Dual-luciferase reporter assays, ChIP-qPCR, Co-IP, ubiquitination site mass spectrometry, and ubiquitin assays were used to analyse the interactions and potential mechanisms involving XAB2, Annexin A2 (ANXA2), and ubiquitin-specific protease 10 (USP10). Results XAB2 was found to be expressed in CRC and was associated with poor prognosis in patients with CRC. XAB2 promoted CRC cell proliferation and enhanced oxaliplatin resistance by promoting DNA damage repair. Mechanistically, CRC cells treated with oxaliplatin exhibited increased USP10 nuclear expression. USP10 bound to XAB2 and deubiquitinated XAB2 K48-linked polyubiquitination at K593, thereby stabilising XAB2 by reducing its degradation via the ubiquitin-proteasome pathway. XAB2 upregulates ANXA2 expression at the transcriptional level by binding to the ANXA2 promoter, thereby promoting DNA damage repair, mitigating oxaliplatin-induced DNA damage, and enhancing oxaliplatin resistance. Conclusions In summary, this study demonstrates that the USP10/XAB2/ANXA2 axis promotes proliferation, DNA damage repair, and oxaliplatin resistance in CRC. These findings uncover a novel mechanism of oxaliplatin resistance in CRC and suggest potential therapeutic targets for improving the efficacy of oxaliplatin in CRC treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03357-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99a0a723b30808def36cd247f913003a04e424eb" target='_blank'>
              USP10/XAB2/ANXA2 axis promotes DNA damage repair to enhance chemoresistance to oxaliplatin in colorectal cancer
              </a>
            </td>
          <td>
            Xingwu Liu, Shaoming Zhang, Yue An, Boyang Xu, Guanyu Yan, Mingjun Sun
          </td>
          <td>2025-03-11</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract N6-methyladenosine (m6A) is the most abundant internal RNA modification in eukaryotes and plays a key role in cellular growth and development. Global changes in cellular methylated RNA and m6A-mediated transcript regulation significantly impact oncogenesis. Here, we investigate how recurrent synonymous and non-synonymous somatic mutations abolishing individual canonical methylated m6A motifs affect transcript levels and survival of patients with cancer. Moreover, we explore the effect of these mutations on creating de novo m6A motifs. To this end, we compared publicly available data on m6A sites with mutations reported in The Cancer Genome Atlas (TCGA). We find that mutations disrupting or creating m6A motifs display a low recurrence and have a negligible impact on RNA abundance. Patients with the highest number of disrupted m6A sites or newly generated m6A motifs did not generally exhibit alterations in mortality risk or outcomes. Hence, our data suggest that mutational alterations in the m6A motif landscape are unlikely to be a primary mechanism for regulating gene function across most cancer types. This may be attributed to the fact that mutations typically affect individual m6A sites, which is likely insufficient to significantly impact gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471e47695d7b9c72dc5640235adb0cf3f4ad1c59" target='_blank'>
              The role of recurrent somatic mutations that alter conserved m6A motifs in human cancer
              </a>
            </td>
          <td>
            Oliver Artz, James R White, Benoit Rousseau, G. Argilés, Mike Foote, Paul Johannet, Miteshkumar Patel, Somer Abdelfattah, Shrey Patel, Callahan Wilde, David Mieles, Luis A Diaz
          </td>
          <td>2025-04-02</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f99119d1610004635e935f3cbda8df6a5ed0a6" target='_blank'>
              Abstract 6288: Precise characterization of complex repeats in cancer genomes
              </a>
            </td>
          <td>
            Khi Pin Chua, E. Dolzhenko, Z. Kronenberg, Seiya Imoto, Seiichi Mori, M. Eberle
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction Pluripotent stem cells (PSC) are known to recurrently acquire genetic aberrations including chromosomal aneuploidy during long-term culture of which the consequences remain largely unknown. A recent study reported that xenografting of aneuploid mouse embryonic stem cells (mESC) gives rise to teratomas containing undifferentiated elements with metastatic capacity. This contrasts with benign teratomas composed of fully differentiated tissues from the three embryonic germ layers obtained by xenografting diploid mESCs. The highlighted study thus proposes that general aneuploidy may drive metastasis of PSC-derived teratomas, thus directly correlating it with malignant potential of PSCs. Main The aneuploid mESC lines used in the aforementioned study showed trisomies of chromosomes 6, 8, 11, 15 or combinations thereof. Interestingly, several of these trisomies, specifically in chromosomes 6, 8, and 11, constitute well-known chromosomal aberrations recurrently acquired in mESCs upon (long-term) in vitro culture. This is suggested to be driven by numerous (proto)oncogenes in these regions. Notably, recurrent chromosomal aberrations are also observed in human PSCs (hPSCs), mostly in chromosome 1, 12, 17, 20 and X, and have similarly been associated with the acquisition of a malignant phenotype in vitro, posing a risk to their potential clinical application. There is synteny between the chromosomes recurrently affected by such aberrations in mouse and human PSCs, namely mouse chromosome 6 and human chromosome 12 as well as mouse and human chromosome 1, suggesting a chromosome-specific phenomenon that may transcend the species barrier. Additionally, in the field of cancer, malignant (but not benign) human germ cell tumors (hGCTs), pathologically similar to the tumors derived from the aforementioned aneuploid mouse PSCs, are characterized by supernumerical copies of chromosome 12, highlighting the additional clinical relevance of these findings and the aneuploid mESC models for GCT research and treatment. Conclusions We suggest that chromosome-specific trisomies, rather than general aneuploidy, might drive teratoma metastasis upon mESC xenografting in vivo. We also observed indications of synteny between the recurrent chromosomal aberrations among human and mouse PSCs, suggesting potentially common intraspecies selection mechanisms. We finally reiterate the similarities observed between the PSC and GCT field related to chromosomal aberrations and malignancy, highlighting the relevance of these experimental models in both research fields.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9961bd4e5533b7648a3a993149a92ce0795a5ee" target='_blank'>
              Chromosome-specific aberrations, rather than general aneuploidy, may drive mouse embryonic stem cell-derived teratoma metastasis
              </a>
            </td>
          <td>
            Joaquin Montilla-Rojo, Sanne Hillenius, T. F. Eleveld, Daniela C F Salvatori, Leendert H J Looijenga
          </td>
          <td>2025-04-15</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d11e895cea0cf1e967c3e4adda2d90a1183e147" target='_blank'>
              Abstract 215: A novel framework for deciphering 3D genome structure in colorectal cancer
              </a>
            </td>
          <td>
            Amin Nooranikhojasteh, Jakob Zerbs, Ghazaleh Tavallaee, Elias Orouji
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Recent evidence indicates that FAM111B is significantly involved in the progression of various cancers. Nonetheless, the potential pan-cancer implications of FAM111B have not been systematically investigated. In this study, FAM111B’s expression and oncogenic potential were studied using TCGA and GTEx data via GEPIA2, TIMER2.0, and STRING tools. Pathway enrichment analyses with the GO, KEGG, Reactome, and WikiPathways databases were conducted to explore its role in cancer development. The results were validated via multiplex immunofluorescence assays of pancreatic cancer tissues, microarray assays of ovarian cancer tissues, and protein transcriptomics of ovarian cancer cells. The expression levels of FAM111B were elevated in most cancer types and were associated with poor prognostic outcomes. Mechanistically, FAM111B expression was positively correlated with the expression of genes involved in DNA homologous recombination repair and with the infiltration of Th2 CD4+ T cells. These observations were further substantiated in ovarian cancer cell lines and tissue specimens from pancreatic and ovarian cancers. FAM111B functions as a biomarker for the DNA repair pathway and Th2 CD4+ T-cell infiltration in human malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe1e275d0931ab0098bd9862f4488d0d9192898d" target='_blank'>
              Unveiling FAM111B: A Pan-Cancer Biomarker for DNA Repair and Immune Infiltration
              </a>
            </td>
          <td>
            Fang Wei, Wanying Li, Ting Zhou, Xianglin Yuan, Lihong Zhang
          </td>
          <td>2025-03-28</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9235dd3b879fd0da050d8e59db442b8291733b6d" target='_blank'>
              Abstract 2391: Comprehensive quantification and annotation of viral gene integration and expression across thousands of human cancer models
              </a>
            </td>
          <td>
            Sowmya Iyer, Moore van Tienen, Phillipp Trollman, Yao He, William R. Sellers, Catarina D. Campbell
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821c77d7a2a02d4160cf107b29b6628f9d6cb1ed" target='_blank'>
              EccDNA-Driven VPS41 Amplification Alleviates Genotoxic Stress via Lysosomal KAI1 Degradation.
              </a>
            </td>
          <td>
            Bin Shi, Ping Yang, Huaijin Qiao, Jinchen He, Bin Song, Hao Bai, Fengdi Jiang, Yining Zhang, Qian Li, Tao Yan, Wenling Tu, Daojiang Yu, Shuyu Zhang
          </td>
          <td>2025-04-24</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="High-risk human papillomavirus (HPV) replication requires deregulation of host DNA damage response (DDR) and inflammatory pathways. DNA topoisomerase 2β (Top2β) was previously shown to promote HPV replication, while the function of its paralog Top2α in the viral life cycle remains unknown. Elevated levels of Top2α are consistently observed in cervical intraepithelial lesions and the related carcinomas, as well as in HPV-positive cell lines. Silencing Top2α with shRNA severely suppresses HPV genome maintenance and amplification, but in a DDR-independent manner. Instead, Top2α facilitates secretion of interleukin (IL)-6 and IL-8, which are necessary for HPV replication. Mechanistically, this manipulation is regulated by toll-like receptor 4 (TLR4). Top2α binds to the TLR4 promoter to transcriptionally induce TLR4 expression. Blockade of TLR4 signaling by the specific inhibitor TAK-242 significantly reduces the secreted IL-6/IL-8 levels and HPV replication. Overall, our results reveal a novel role of Top2α to shape the inflammatory microenvironment that benefits HPV replication, making it a promising therapeutic target for HPV-associated diseases. IMPORTANCE Human papillomaviruses (HPVs) are oncogenic pathogens responsible for many HPV-related diseases such as cervical cancer. HPV replication depends on the subversion of host cellular machineries, particularly pathways governing DNA damage response (DDR) and inflammation. However, the roles of enzymes that directly introduce DNA nicks or mediate strand rejoining remain poorly characterized. Here we highlight a novel role of DNA topoisomerase 2α (Top2α) in promoting HPV genome maintenance and amplification but in a DDR-independent manner. Unlike its previously studied paralog Top2β, Top2α supports HPV replication by driving the secretion of inflammatory molecules interleukin (IL)-6 and IL-8, which are critical for the viral genome persistence and copy number expansion. To our knowledge, this is the first evidence that a DDR-associated protein manipulates cytokine secretion, rather than expression, to support HPV replication. Furthermore, blocking toll-like receptor 4 (TLR4) activities significantly reduced Top2α-dependent IL-6/IL-8 secretion and impaired HPV replication. These findings establish Top2α as a crucial player in HPV pathogenesis, with a therapeutic potential for treating HPV-associated malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9c0d0b88bb67130941343d7bf736b02de723a2d" target='_blank'>
              Topoisomerase IIα orchestrates secretion of IL-6 and IL-8 with human papillomavirus replication
              </a>
            </td>
          <td>
            Yanfei Liu, Zi Han, P. Kaminski, Chengcheng Tao, Xiaoge Li, Mengmeng Liu, Yang Li, Ying Jia, Junfen Xu, Shiyuan Hong
          </td>
          <td>2025-04-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57a62390eeb2a72ca937d2e747b4a191022a5bb6" target='_blank'>
              Abstract 2486: Decoding tumor evolution through fitness landscapes of aneuploid cells
              </a>
            </td>
          <td>
            Richard Beck, Noemi Andor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba2b2e41c178b6dbd66cbfc9ba10d448bf6866f2" target='_blank'>
              Spatial multi-omics reveals cell-type-specific nuclear compartments.
              </a>
            </td>
          <td>
            Yodai Takei, Yujing Yang, Jonathan White, Isabel N. Goronzy, Jina Yun, Meera Prasad, Lincoln J Ombelets, Simon Schindler, Prashant Bhat, Mitchell Guttman, L. Cai
          </td>
          <td>2025-04-09</td>
          <td>Nature</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Most eukaryotic cells maintain mitochondria in well-distributed, reticular networks. The size of the mitochondrial network and copy number of its genome scale with cell size. However, while the size scaling features of mitochondria and their genome are interrelated, the fitness consequences of this interdependence are not well understood. We exploit the asymmetric cell division of budding yeast to test the hypothesis that mitochondrial scaling with cell size impacts mitochondrial DNA (mtDNA) function. We find that the volume of mitochondria inherited by daughter cells affects the ability of cells to maintain functional mtDNA; daughter cells that inherit a significantly reduced volume of mitochondria have an increased frequency of losing respiratory competence. In cells with such mitochondrial inheritance defects, mtDNA integrity can be maintained by upregulating mtDNA copy number. Collectively, these data support a bet-hedging model whereby the faithful inheritance of an adequate volume of mitochondria ensures enough mtDNA copies are transmitted to daughter cells to counteract pre-existing and/or inevitable mtDNA mutations. Summary Ray et al. demonstrate that the volume of mitochondria inherited impacts mtDNA homeostasis in the model system budding yeast. They propose a model by which inheritance of an adequate mitochondrial volume results in the transmission of sufficient mtDNA copies to counteract existing and/or inevitable mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7385c093a01a8b13e3e85d1696364a442d782cc7" target='_blank'>
              The Volume of Mitochondria Inherited Impacts mtDNA Homeostasis in Budding Yeast
              </a>
            </td>
          <td>
            Michael W. Ray, WeiTing Chen, Chengzhe Duan, Guadalupe Bravo, Kyle Krueger, Erica M. Rosario, Alexis A. Jacob, L. Lackner
          </td>
          <td>2025-03-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6729d9422175815892338d2b90ca2eb101d28caa" target='_blank'>
              Continuous infiltration and evolutionary trajectory of nuclear organelle DNA inOryza
              </a>
            </td>
          <td>
            Chenbo Gong, Yicheng Huang, Mengmeng Liu, Yong Zhou, Yinwei Xu, Nahed Mohammed, Xintong Qiao, A. Zuccolo, Weibo Xie, Rod A. Wing, Jianwei Zhang, Fei Zhou, Yongjun Lin
          </td>
          <td>2025-04-25</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="A prominent feature of eukaryotic chromosomes are centromeres, which are specialized regions of repetitive DNA required for faithful chromosome segregation during cell division. In interphase cells, centromeres are non-randomly positioned in the three-dimensional space of the nucleus in a cell type-specific manner. The functional relevance and the cellular mechanisms underlying this localization are unknown, and quantitative methods to measure distribution patterns of centromeres in 3D space are needed. Here, we developed an analytical framework that combines sensitive clustering metrics and advanced modeling techniques for the quantitative analysis of centromere distributions at the single-cell level. To identify a robust quantitative measure for centromere clustering, we benchmarked six metrics for their ability to sensitively detect changes in centromere distribution patterns from high-throughput imaging data of human cells, both under normal conditions and upon experimental perturbation of centromere distribution. We found that Ripley’s K function has the highest accuracy with minimal sensitivity to variations in the number of centromeres, making it the most suitable metric for measuring centromere distributions. As a complementary approach, we also developed and validated spatial models to replicate centromere distribution patterns, and we show that a radially shifted Gaussian distribution best represents the centromere patterns seen in human cells. Our approach creates tools for the quantitative characterization of spatial centromere distributions with applications in both targeted studies of centromere organization and unbiased screening approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/174a17b1ef834044537c7e22b9436da4a482c609" target='_blank'>
              Simulation and Quantitative Analysis of Spatial Centromere Distribution Patterns
              </a>
            </td>
          <td>
            Adib Keikhosravi, Krishnendu Guin, Gianluca Pegoraro, Tom Misteli
          </td>
          <td>2025-03-25</td>
          <td>Cells</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e9fe2ca640f1cfacf9753f7fd79adbb41fae4fa" target='_blank'>
              Abstract 3952: Noncoding elements within MYCN mRNA are autonomous drivers of oncogenesis in neuroblastoma
              </a>
            </td>
          <td>
            Vishwa Patel, Lorraine-Rana E. Benhamou, Dr. John T. Powers
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Variants of uncertain significance (VUS) limit the actionability of genetic testing. A prominent example is MUTYH, a base excision repair factor associated with polyposis and colorectal cancer, which has a pathogenic variant carrier rate approaching 1 in 50 individuals in some populations. To systematically interrogate variant function in MUTYH, we coupled deep mutational scanning with a DNA repair reporter containing its lesion substrate, 8OG:A. Our variant-to-function map covers >97% of all possible MUTYH point variants (n=10,941) and achieves 100% accuracy classifying the pathogenicity of known clinical variants (n=247). Leveraging a large clinical registry, we observe significant associations with colorectal polyps and cancer, with more severely impaired missense variants conferring greater risk. We recapitulate known functional differences between pathogenic founder alleles, and highlight sites of complete missense intolerance, including residues that intercalate DNA and coordinate essential Zn2+ or Fe-S clusters. This map provides a resource to resolve the 1,032 existing missense VUS and 90 variants with conflicting interpretations in MUTYH, and demonstrates a scalable strategy to interrogate other clinically relevant DNA repair factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/123ddb79d358a7742980d1fc41247ef4a3c39e8c" target='_blank'>
              Saturation mapping of MUTYH variant effects using DNA repair reporters
              </a>
            </td>
          <td>
            Shelby L. Hemker, Ashley Marsh, Felicia Hernandez, Elena Glick, Grace Clark, Alyssa Bashir, Krystal Jiang, Jacob O. Kitzman
          </td>
          <td>2025-03-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb8c843c17a6c70005621aa61b21710d77ab8472" target='_blank'>
              Abstract SY22-03: Multifaceted mitochondria in the regulation of chromosome inheritance
              </a>
            </td>
          <td>
            M. J. Rodríguez Colman
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="RNA alternative splicing is a fundamental cellular process implicated in cancer development. Kaposi’s sarcoma-associated herpesvirus (KSHV), the etiological agent of multiple human malignancies, including Kaposi’s sarcoma (KS), remains a significant concern, particularly in AIDS patients. A CRISPR-Cas9 screening of matched primary rat mesenchymal stem cells (MM) and KSHV-transformed MM cells (KMM) identified key splicing factors involved in KSHV-induced cellular transformation. To elucidate the mechanisms by which KSHV-driven splicing reprogramming mediates cellular transformation, we performed transcriptomic sequencing, identifying 131 differential alternative splicing transcripts, with exon skipping as the predominant event. Notably, these transcripts were enriched in vascular permeability, multiple metabolic pathways and ERK1/2 signaling cascades, which play key roles in KSHV-induced oncogenesis. Further analyses of cells infected with KSHV mutants lacking latent genes including vFLIP, vCyclin and viral miRNAs, as well as cells overexpressing LANA, revealed their involvement in alternative splicing regulation. Among the identified splicing factors, FAM50A, a component of the spliceosome complex C, was found to be crucial for KSHV-mediated transformation. FAM50A knockout resulted in distinct splicing profiles in both MM and KMM cells, and significantly inhibited KSHV-driven proliferation, cellular transformation and tumorigenesis. Mechanistically, FAM50A knockout altered SHP2 splicing, promoting an isoform with enhanced enzymatic activity that led to reduced STAT3 Y705 phosphorylation in KMM cells. These findings reveal a novel paradigm in which KSHV hijacks host splicing machinery, specifically FAM50A-mediated SHP2 splicing, to sustain STAT3 activation and drive oncogenic transformation. Importance Kaposi’s sarcoma-associated herpesvirus (KSHV) causes cancers such as Kaposi’s sarcoma, particularly in AIDS patients. This study uncovers how KSHV hijacks a fundamental cellular process called RNA splicing to promote cancer development. We identified key splicing events that alter critical pathways involved in vascular permeability, metabolism, and oncogenic signaling, particularly ERK1/2 and STAT3. A specific protein, FAM50A, was found to be essential for KSHV-driven cancerous transformation. Removing FAM50A disrupted splicing, weakening cancer-promoting signals. These findings provide new insights into how viruses manipulate host cells to drive cancer and highlight RNA splicing as a potential target for future therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc1dd5dbd2f3302db90bfa70c7f9ae17b8e6ba3f" target='_blank'>
              KSHV Reprograms Host RNA Splicing via FAM50A to Activate STAT3 and Drive Oncogenic Cellular Transformation
              </a>
            </td>
          <td>
            Shenyu Sun, Karla Paniagua, Ling Ding, Xian Wang, Yufei Huang, Mario A Flores, Shou-Jiang Gao
          </td>
          <td>2025-03-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Epidermal growth factor receptor variant III (EGFRvIII) mutation is the most prevalent genetic change in glioblastoma. Abnormal DNA damage repair caused by EGFRvIII, which leads to temozolomide (TMZ) resistance, is a major cause of reduced postoperative survival in glioblastoma patients. This study aims to uncover the molecular mechanism of TMZ resistance in EGFRvIII-mutant glioblastoma.


METHODS
We constructed a CRISPR/Cas9 library to identify synthetic lethal genes for EGFRvIII-bearing cells. Abnormal epigenetic regulation of the RAD51-associated protein 1 (RAD51AP1) promoter was assessed via chromatin immunoprecipitation sequencing (ChIP-seq) and chromatin immunoprecipitation polymerase chain reaction (ChIP-PCR) analyses. In vitro and in vivo experiments were carried out to investigate the role of the RAD51AP1 gene in TMZ resistance in EGFRvIII-bearing glioblastoma.


RESULTS
The CRISPR/Cas9 library identified RAD51AP1, a synthetic lethal gene for EGFRvIII-bearing cells exposed to TMZ. ChIP-seq and ChIP-PCR analyses revealed that acetylated histone H3 lysine 27 (H3K27ac) and SRY-box transcription factor 9 (SOX9) together induced RAD51AP1 transcription in EGFRvIII cells. High expression levels of RAD51AP1, promoted formation of the RAD51-UAF1 complex to activate homologous recombination and inhibit TMZ-induced DNA damage.


CONCLUSION
The results of this study suggest that aberrant RAD51AP1 expression is a crucial mechanism by which EGFRvIII-mutant glioblastoma resists TMZ chemotherapy, laying the groundwork for future personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b5d96766fa4eddfdcc647bdcfb29efad8ef9899" target='_blank'>
              Epigenetically elevated RAD51AP1 regulates the RAD51-UAF1 complex contributing to temozolomide resistance in EGFRvIII glioblastoma.
              </a>
            </td>
          <td>
            Yuanhao Chang, Jingchen Yang, Zhuo Li, Xiaoxue Zhu, Tao Jiang, Qixue Wang, Mingchen Yu, Bo Han, Xing Liu
          </td>
          <td>2025-04-10</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6462361aae6f3cbb7d9db079e7f43e5e3ea1a7f1" target='_blank'>
              Abstract 1448: An evolutionarily conserved chromatin remodeling factor, FACT, in regulation of chromatin, gene expression and cancer
              </a>
            </td>
          <td>
            Priyanka Barman, S. Bhaumik
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a24afa888bf591284938fd750888dc7dfa3335e" target='_blank'>
              Abstract 2846: Association of cyclin E1 expression with genomic instability in ovarian cancer
              </a>
            </td>
          <td>
            Ajay Obla, Michelle Kinder, Chifei Sun, Heather Bullins, Cynthia Timmers, M. Poi, Sharon Wu, E. Missiglia, Bettina Bisig, K. Homicsko
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Mobile genetic elements evade CRISPR–Cas adaptive immunity by encoding anti-CRISPR proteins (Acrs). Acrs inactivate CRISPR–Cas systems via diverse mechanisms but generally coevolve with a narrow subset of Cas effectors that share high sequence similarity. Here, we demonstrate that AcrIIA11 inhibits Streptococcus pyogenes (Sp), Staphylococcus aureus (Sa), and Francisella novicida (Fn) Cas9s in vitro and in human cells. Single-molecule imaging reveals that AcrIIA11 hinders SaCas9 target search by reducing its diffusion on nonspecific DNA. DNA cleavage is inhibited because the AcrIIA11:SaCas9 complex binds to protospacer adjacent motif (PAM)-rich off-target sites, preventing SaCas9 from reaching its target. AcrIIA11 also greatly slows down DNA cleavage after SaCas9 reaches its target site. A negative-stain electron microscopy reconstruction of an AcrIIA11:SaCas9 RNP complex reveals that the heterodimer assembles with a 1:1 stoichiometry. Physical AcrIIA11–Cas9 interactions across type IIA and IIB Cas9s correlate with nuclease inhibition and support its broad-spectrum activity. These results add a kinetic inhibition mechanism to the phage-CRISPR arms race.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ae6d0bcafc0d2a35bdeaef7ab05074639e57301" target='_blank'>
              Mechanism of Cas9 inhibition by AcrIIA11
              </a>
            </td>
          <td>
            Kaylee E. Dillard, Hongshan Zhang, Lianne Z Dubbs, Chia-Wei Chou, Cynthia Terrace, Kamyab Javanmardi, Wantae Kim, Kevin J Forsberg, Ilya J Finkelstein
          </td>
          <td>2025-04-22</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Nitroalkenes are endogenous products generated by the metabolism of unsaturated fatty acids. They are generated under oxidative stress conditions, mediating important anti-inflammatory signaling activities through covalent modification of protein cysteine thiols. Despite being cytoprotective in benign tissue, nitroalkenes display single-agent anti-proliferative activity in breast cancer cells and sensitize them to multiple DNA-damaging agents. Initial mechanistic evidence suggested that nitroalkene anti-cancer activities are partially mediated by inhibition of homologous recombination (HR) through the recombinase RAD51 at Cys319. However, nitroalkenes are multi-target agents, and thus, it is likely that other important DNA repair targets beyond RAD51 are modified by nitroalkenes, contributing to their anti-cancer effects. We, therefore, conducted a global proteomics analysis to address this question. This analysis led to the identification of the recQ helicase RECQL4 with a nitro-alkylation at Cys1052. This modification was further confirmed by click chemistry-based chemoproteomics and determined to be DNA damage-dependent. Functional analyses demonstrated that nitroalkene modification inhibits RECQL4 ATP-dependent helicase activity and disrupts DSB end resection and downstream homology-dependent repair. Furthermore, experiments with C1052S mutant RECQL4 revealed that RECQL4 is a major mediator of nitroalkene effects on end resection, DSB formation, and repair. The evidence presented here denotes RECQL4 as an important nitroalkene target conferring DSB repair inhibition and supports further evaluation of nitroalkenes as therapeutic agents in RECQL4-amplified cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0d69c71aa177e30849ed12d6a3ba7928d0b7c50" target='_blank'>
              Chemoproteomic analysis reveals RECQL4 as a mediator of nitroalkene-dependent double-strand break repair inhibition in cancer.
              </a>
            </td>
          <td>
            Carola Neumann, Dennis Braden, Mostafa Adbel-Salam, Alp Asan, John Skoko, Thomas Conrads, Huiming Lu, Bruce Freeman, Francisco Schopfer, Ishu Saini, Jochen Kuper, Caroline Kisker, Apoorva Uboveja, N. Tangudu, Katherine Aird, Anthony Davis
          </td>
          <td>2025-03-26</td>
          <td>Research Square</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT Background The ten–eleven translocation (TET) enzyme family is a key regulator of DNA methylation, responsible for the conversion of 5‐methylcytosine to 5‐hydroxymethylcytosine to promote locus‐specific demethylation. Thus, it is not surprising that loss or attenuation of TET enzymes is implicated in genomic hypermethylation and transcriptional reprogramming that drives cancer development. Somatic mutations in TET2 are observed in the bone marrow of 5%–10% of healthy adults over 65 years of age, imparting a hematopoietic stem cell advantage and subsequent clonal hematopoiesis of indeterminate potential (CHIP), a condition which is associated with increased risk of myeloid malignancy. Somatic TET2 mutations are frequently reported in myeloid disorders, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML). Evidence suggests that TET2 mutations also affect prognosis in myeloid leukemia and other hematopoietic malignancies. However, there is a paucity of collated data on the frequency of TET2 mutations in solid human cancers. Objectives We review the published literature on TET2 mutation in human solid cancers and explore their frequency and impact on patient outcomes. Results & Conclusions Somatic TET2 mutations are reported in numerous solid human cancers, including those arising in the skin, lung and prostate. Many of the somatic TET2 mutations reported in solid cancers are recurrent, suggesting functionality. There is also evidence to suggest that somatic TET2 mutations affect prognosis in solid human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaed4855352e94dc92bd80e9eb79f7e25c4d3a21" target='_blank'>
              Landscape of TET2 Mutations: From Hematological Malignancies to Solid Tumors
              </a>
            </td>
          <td>
            Zoë L. Hawking, James M Allan
          </td>
          <td>2025-03-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8434065523456bfd26cd26f42f6a8a7a41f381b1" target='_blank'>
              Abstract 2837: Mechanisms promoting chromosomal instability following the loss of p53 and Rb
              </a>
            </td>
          <td>
            Risha Banerjee, D. Kirsch
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0362fea5e5aef827fc0c7f31abdae91baaf00292" target='_blank'>
              Abstract 5924: A Tumor-specific, methylation-based algorithm to identify MTAP gene deletions via tissue-free circulating tumor DNA
              </a>
            </td>
          <td>
            S. Gordon, Evan Warner, Geethika Yalamanchili, Jill Tsai, Scott Shell, Shile Zhang, Martina Lefterova, Justin Odegaard, Darya I. Chudova
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/255726480b1fba8b96c7a797db393035b828db59" target='_blank'>
              Abstract B022: Decipher actionable tumor suppressor defects for personalized cancer medicine
              </a>
            </td>
          <td>
            Feiyu Chen, Di Zhao
          </td>
          <td>2025-03-11</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Genome replication in eukaryotic cells is surveyed by the S-phase checkpoint, which orchestrates sequential replication origin activation to avoid exhaustion of hitherto poorly defined rate-limiting replisome components. Here, we find that excessive activation of replication origins depletes chromatin-bound PCNA and lagging strand components, thereby limiting additional PCNA loading at new origins when checkpoint control is disrupted. PAF15 (PCNA-associated factor 15) emerges as a dosage-sensitive regulator of PCNA, delineating the dynamic range of global genome duplication and defining distinct roles for PCNA on the leading and lagging strands. Through its high-affinity PIP motif and interaction within the DNA encircling channel of PCNA, PAF15 stabilizes PCNA exclusively on the lagging strand, optimizing and rate-limiting lagging strand processing. On the other hand, misregulation of PAF15—whether by overexpression or mislocalization to the leading strand—impairs replication fork progression and leads to cell death. These defects are mitigated by TIMELESS and CLASPIN, which restrain PAF15-PCNA interactions beyond the lagging strand. E2F4-mediated repression orchestrates PAF15 expression in normal and cancer cells, maintaining its optimal dosage for lagging strand-specific interactions with PCNA. Thus, the S-phase checkpoint functions in concert to restrict origin activation when lagging strand PAF15-PCNA assembly is exhausted, linking a previously concealed strand-specific rate limitation to overall replication dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe32e8db1ccfcee1501ea6b48ca4b9c67236d177" target='_blank'>
              PAF15-PCNA assembly exhaustion governs lagging strand replication and replisome integrity
              </a>
            </td>
          <td>
            Gita Chhetri, S. Badugu, Narcis-Adrian Petriman, Mikkel Bo Petersen, Ganesha P. Pitchai, Aylin Seren Güller, Jan Novotný, Barath Balarasa, Morten Frendø Ebbesen, Frederik Tibert Larsen, Tina Ravnsborg, A. K. Yadav, Anita Lunding, Hana Polasek-Sedlackova, Ole Nørregaard Jensen, Kim Ravnskjaer, Jonathan R. Brewer, Jesper G.S. Madsen, Jens S. Andersen, Kumar Somyajit
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6167c18e215886150966799c6b910032fa501470" target='_blank'>
              Abstract 2387: Characterizing HERV expression patterns across breast cancer subtypes: Implications for tumor biology and therapeutic targets
              </a>
            </td>
          <td>
            Nathália Da Roz D’Alessandre, Gabriela Der Agopian Guardia, Helena Beatriz da Conceição, Rafael Loch Batista, P. A. Favoretto Galante
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af18408df27119110ab6bd8351cf83d78fe72d56" target='_blank'>
              Accurate plasmid reconstruction from metagenomics data using assembly-alignment graphs and contrastive learning
              </a>
            </td>
          <td>
            Pau Piera Líndez, Lasse Schnell Danielsen, Iva Kovačić, Marc Pielies Avellí, J. Nesme, L. J. Jensen, J. N. Nissen, Søren J Sørensen, Simon Rasmussen
          </td>
          <td>2025-03-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc37e27440b9c33809194ed3bd279c915fb98a61" target='_blank'>
              Abstract 5408: Unraveling c-MYC role in DNA replication origin licensing through non-canonical pathways
              </a>
            </td>
          <td>
            Dalia Fleifel, Eiman Elkoumy, Arthi Adavikolanu, Sidra Qayyum, Jeanette Cook
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Prostate cancer is the most common cancer found in men, necessitating the study of genes implicated in the initiation and progression of this disease. The inhibitor of growth family member 3 (ING3) is an epigenetic regulator, whose role in prostate cancer is unknown. The aim of our study was to investigate the functional consequences of prostate-specific ablation of ING3 in mice. While we found normal prostate tissue histoarchitecture in the prostate-specific Ing3 knockout mice, increased expression of DNA-damage-associated markers suggests a role for ING3 in maintaining genomic stability. Altogether, our data show that loss of Ing3 does not lead to neoplastic transformation of the prostate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d2bd2b54e0230f2e81d06231147d56ed42056d8" target='_blank'>
              Loss of ING3 in the Prostate Leads to Activation of DNA Damage Repair Markers
              </a>
            </td>
          <td>
            Viktor Lang, Lisa Barones, Shiting Misaki Hu, Fatemeh Hashemi, Karen Bloté, Karl Riabowol, Dieter Fink
          </td>
          <td>2025-03-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA sequencing technology has undergone substantial improvements in recent years, to the extent that Third Generation Sequencing platforms are capable of massively generating long-reads. Amplicon sequencing has been among the most popular techniques due to its wide application in diverse fields of biological sciences. However, there is a lack of software specifically designed to analyse intra-individual genetic variation using amplicon long-read data. Here, we present CCS-consensuser, an end-to-end pipeline that generates consensus sequences from amplicon sequencing using high-fidelity reads produced by PacBio circular consensus sequencing (CCS). We evaluated the concordance of the results produced using CCS + CCS-consensuser and other sequencing platforms (Illumina and Sanger), as well as accuracy using a simulated dataset. This assessment showed that CCS amplicon data coupled with CCS-consensuser can produce high-quality sequences (PHRED > 30). The pipeline resulted in high proportions of identical sequence bins for real data, achieving up to 94.94% concordance with COI Sanger sequences and 92.61% with nuclear loci Illumina sequences (considering heterozygous loci), and 95.55% with a fully phased nuclear simulated dataset. Furthermore, our pipeline can be used to detect heteroplasmy in mtDNA, cross-contamination, resolve the phase of nuclear genes in diploid organisms, and conceivably for multi-copy gene systems such as rDNA. These results not only support its potential for application in studies using haploid data such as DNA barcoding, but also demonstrate its unique capacity to explore within individual haplotype variation. Therefore, our strategy shows promise for a broad range of applications in biology and medicine that have been challenging to assess using traditional techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72f603435f64e69b415cf86a3aeb1d04cc0d9089" target='_blank'>
              CCS-Consensuser: A Haplotype-Aware Consensus Generator for PacBio Amplicon Sequences.
              </a>
            </td>
          <td>
            Carlos Congrains, Forest T. Bremer, Julian R Dupuis, N. Barr, Ivonne J Garzón-Orduña, Daniel Rubinoff, C. Doorenweerd, M. Jose, Kimberley Morris, A. Kauwe, Scott Geib
          </td>
          <td>2025-04-04</td>
          <td>Molecular ecology resources</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f606536b132a87445aa5fa5b6a3753af5559e5e5" target='_blank'>
              Abstract 5074: Estimation of homologous recombination deficiency map from spatial transcriptomics
              </a>
            </td>
          <td>
            Jeongbin Park, Yuchang Seong, Dongjoo Lee, Hongyoon Choi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="N4-acetylcytidine (ac4C) is deposited on diverse RNAs by N-acetyltransferase 10 (NAT10), a protein with high biological relevance for aging and cancer. We performed a comprehensive survey of ac4C using metabolic labeling, sodium cyanoborohydride chemical treatment coupled to next-generation sequencing (NGS), and ac4C antibody–based cell and molecular biology techniques. Our analysis shows that NAT10-dependent ac4C-acetylation is robust in rRNA and specific tRNAs but low/spurious in mRNA. It also revealed an inflammatory signature and mutagenesis at transcriptionally active sites in NAT10-KO cells. This finding led us to explore the role of NAT10 in R-loops, which were recently linked to APOBEC3B-mediated mutagenesis. Our analysis showed that R-loops are ac4C-acetylated in a NAT10-dependent manner. Furthermore, NAT10 restrains the levels of R-loops at a subset of differentially expressed genes in a catalytic activity–dependent manner. Together with cellular biology data showing ac4C-modified RNA in endosomal structures, we propose that increased levels of ac4C-unmodified RNAs, likely derived from R-loops, in endosomal structures induce inflammatory responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87dd80c89b9eb6c1d31263cc293acb2bcf5cb94c" target='_blank'>
              NAT10 and N4-acetylcytidine restrain R-loop levels and related inflammatory responses
              </a>
            </td>
          <td>
            Turja K Debnath, Nathan S. Abell, Yi-Ru Li, Sravan K. Devanathan, Enrique Navedo, Blerta Xhemalçe
          </td>
          <td>2025-03-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d32199ced2607113320cf436c50b12773694e23a" target='_blank'>
              Abstract 6530: A comprehensive genomic analysis elucidates the microbial landscape of cancer
              </a>
            </td>
          <td>
            Anders B. Dohlman, Robin Mjelle, Henry Wood, I. Lee, Kevin X. Jiang, Ethan Shurberg, Alaina Shumate, Blake E. Sanders, Guanxi Qiao, Jodi Hirshman, Andrew D Cherniack, Phil Quirke, Matthew L. Meyerson
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a0c142c6c3dbc24f6010a9b80df6aa28614056f" target='_blank'>
              The Spatial Location of Chromosomes in Dividing Cells and the Relative Stability of Chromosome Spatial Structure
              </a>
            </td>
          <td>
            Chunxiao Wu, Dongyun Jiang, Tao Jiang, Luxia Xu, Qian Xu, Meng Zhao, Qin Zhu, Zhigang Guo, Jinlan Pan, Suning Chen
          </td>
          <td>2025-04-02</td>
          <td>Proceedings of Anticancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90996b84410758ba4f6bb325e48c6874e9494ef2" target='_blank'>
              Abstract 3941: CRISPR screen of human stem cell derived MYC-driven medulloblastoma tumors reveal the mitochondria as a vulnerability
              </a>
            </td>
          <td>
            Kenya Pleitez Valle, Miriam Velez, Yang Zhao, Bo Cheng, Sophia Estrada, Linwei Li, Tom O'Loughlin, Balaraman Kalyanaraman, Ming You, Luke A. Gilbert, David M. Hockenbery, D. Shackelford, Peter J. Mullen, Miller Huang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7bc3dab4afa38c6c5372774a5530ea7583082a1" target='_blank'>
              Combined Transcriptomic and Epitranscriptomic Profiling Identifies THBS1 as A Regulator of Enzalutamide Resistance in Prostate Cancer
              </a>
            </td>
          <td>
            Emmanuelle Hodara, Lisa Swartz, Aubree Mades, Daniel Bsteh, Tong Xu, S. Rhie, A. Goldkorn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d25cb925af9bcdbf8c78878896d2879472b19c22" target='_blank'>
              Use of Droplet Digital PCR for Consistent Detection of TMPRSS2:ERG Gene Fusion Transcripts Initiated In Vitro
              </a>
            </td>
          <td>
            Megan H. Check, Sarah E. Ernst, K. Sfanos
          </td>
          <td>2025-03-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0c4901cd0ac19548c13b53daf9ce1ecb7db4d98" target='_blank'>
              Abstract 1758: ATX-559, a first in class DHX9 inhibitor, and targeted therapeutic for molecularly defined tumors with genomic instability and replicative stress
              </a>
            </td>
          <td>
            Jennifer Castro, Matthew H Daniels, S. Nayak, David Brennan, Cindy Collins, , Monique Laidlaw, Jie Wu, Anugraha Raman, D. Gotur, Kevin E Knockenhauer, Shihua Yao, Simina Grigoriu, Gordon J. Lockbaum, Kate Newberry, Stephen J Blakemore, P. Boriack-Sjodin, K. Duncan, Stuart Ince, Jason A. Sager, Robert A Copeland, Serena J Silver
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f435cb93295144fbf5f94d521fa4d54709ac19f" target='_blank'>
              Abstract 4118: Decoding the noncoding: An insight into the noncoding genome of breast cancers
              </a>
            </td>
          <td>
            Arnau Cuy Saques, Anita White, Kaneez E. Rabab, Philip Scarry, Francesca Matrà, Craig Monger, Adrian P. Bracken, J. Crown, Alexander Eustace
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="While DNA:RNA hybrids contribute to multiple genomic transactions, their unscheduled formation is a recognized source of DNA lesions. Here, through a suite of systematic screens, we rather observed that a wide range of yeast mutant situations primarily triggering DNA damage actually leads to hybrid accumulation. Focusing on Okazaki fragment processing, we establish that genic hybrids can actually form as a consequence of replication-born discontinuities such as unprocessed flaps or unligated Okazaki fragments. Strikingly, such “post-lesion” DNA:RNA hybrids neither detectably contribute to genetic instability, nor disturb gene expression, as opposed to “pre-lesion” hybrids formed upon defective mRNA biogenesis, e.g., in THO complex mutants. Post-lesion hybrids similarly arise in distinct genomic instability situations, triggered by pharmacological or genetic manipulation of DNA-dependent processes, both in yeast and human cells. Altogether, our data establish that the accumulation of transcription-born DNA:RNA hybrids can occur as a consequence of various types of natural or pathological DNA lesions, yet do not necessarily aggravate their genotoxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b67011b6e27fed1e3b5196a0fb25f35fae9832be" target='_blank'>
              DNA lesions can frequently precede DNA:RNA hybrid accumulation
              </a>
            </td>
          <td>
            Raphaël M Mangione, Steven Pierce, Myriam Zheng, Robert M Martin, Coralie Goncalves, Arun Kumar, S. Scaglione, Cristiana de Sousa Morgado, Arianna Penzo, Astrid Lancrey, Robert J. D. Reid, Ophélie Lautier, P. Gaillard, Peter C Stirling, Sérgio F. de Almeida, Rodney Rothstein, Benoît Palancade
          </td>
          <td>2025-03-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background The incidence of colorectal cancer (CRC) in young adults (age of diagnosis < 50 years old) has been rapidly increasing. Although ∼20% of early-onset (EO) CRC cases are due to germline mutations, the etiology of the majority of EOCRC cases remains poorly understood. Non-genetic factors such as environmental exposure and lifestyle changes are likely to have a direct link to the increased incidence of sporadic EOCRC. We hypothesize that such factors may be observable as differences in the EOCRC epigenome, microbiome and immunome. We sought to address this by comparing differences in DNA methylation from the cohort of colorectal cancer patients in The Cancer Genome Atlas (TCGA). Further, we carefully identified intra-tumoral microbes from TCGA and two other datasets and then related the microbes to EOCRC status and deconvolved immune cell abundances. We found that DNA methylation (DNAm) age acceleration by12 years when compared with average-onset CRC (AOCRC) patients. Differentially methylated sites associated with genes are related to CREB signaling in neurons, G protein coupled receptor signaling, phagosome formation and S100 family signaling. These differences were validated in the gene expression from TCGA and a second, larger real-world dataset from the Oncology Research Information Exchange Network (ORIEN). However, no consistent differences were observed in the intra-tumor microbes between EOCRC and AOCRC. Interestingly, the most abundant microbes interacted with the immune systems differently between the EOCRC and AOCRC tumors, characterized by more, larger, positive correlations in EOCRC. These data suggest epigenetic modulation and accelerated aging may play a key role in the development of EOCRC. SIGNIFICANCE We investigated whether environmentally driven factors contribute to early-onset colorectal cancer (EOCRC). We observed accelerated epigenetic aging in EOCRC and epigenetic changes associated with chronic inflammation. Tumor immune cell abundances correlated more strongly with microbes in EOCRC than average-onset CRC. These data suggest a dysregulation of immune response in EOCRC, driving chronic inflammation and tissue aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8859cb309a43420dfdd0f2da144b37f5d23d1c73" target='_blank'>
              Epigenetic Modulation, Intra-tumoral Microbiome and Immunity in Early Onset Colorectal Cancer
              </a>
            </td>
          <td>
            Ning Jin, R. Hoyd, Ayse Selen Yilmaz, Jian Zhu, YunZhou Liu, M. J. Singh, Dennis Grencewicz, Xiaokui Mo, Matthew Kalady, Daniel Rosenberg, C. Dravillas, Eric A Singer, J. Carpten, Carlos HF Chan, Michelle L Churchman, N. Denko, Frances Di Clemente, Rebecca D Dodd, Islam Eljilany, Naomi Fei, S. Hardikar, A. Ikeguchi, A. Ma, Qin Ma, Martin D McCarter, A. Osman, Gregory Riedlinger, Lary A. Robinson, Bryan P. Schneider, Ahmad A Tarhini, G. Tinoco, Jane C Figueiredo, Yousef Zakharia, C. Ulrich, Aik Choon Tan, Daniel J. Spakowicz
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Parvoviruses are small, single-stranded DNA viruses that have evolved sophisticated mechanisms to hijack host cell machinery for replication and persistence. One critical aspect of this interaction involves the manipulation of the host’s DNA Damage Response (DDR) pathways. While the viral genome is comparatively simple, parvoviruses have developed strategies that cause significant DNA damage, activate DDR pathways, and disrupt the host cell cycle. This review will explore the impact of parvovirus infections on host genome stability, focusing on key viral species such as Adeno-Associated Virus (AAV), Minute Virus of Mice (MVM), and Human Bocavirus (HBoV), and their interactions with DDR proteins. Since parvoviruses are used as oncolytic agents and gene therapy vectors, a better understanding of cellular DDR pathways will aid in engineering potent anti-cancer agents and gene therapies for chronic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61f8c8e278af4ce203bace432e47ab666e9ddfe2" target='_blank'>
              Small Genomes, Big Disruptions: Parvoviruses and the DNA Damage Response
              </a>
            </td>
          <td>
            Rhiannon R. Abrahams, Kinjal Majumder
          </td>
          <td>2025-03-29</td>
          <td>Viruses</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb84576556f09b1b3e9082fc0a15bf2dd37e8f04" target='_blank'>
              Abstract 3343: Centrosomal and mitotic gene expression profiles across cancer types: Implications for tumor progression and survival outcomes
              </a>
            </td>
          <td>
            Christopher L. Farrell, Amy Turner, Alex Feltus, Victoria Cipollino
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c91a033b11a5d910b45df1fe7d4556a0c6c8b6e" target='_blank'>
              Abstract 5585: DNA methylation and transcription factors regulate overexpression of PVT1 in prostate cancer
              </a>
            </td>
          <td>
            Chinedum C. Udekwu, Colin Finnegan, Murtaza Barkarar, Oluwabusola Ajagbe, Rachel E. Bonacci, O. Ogunwobi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bc94f977a238b5f1df2211d9c38889ceb4f0e79" target='_blank'>
              Abstract 4039: An isogenic CRISPR screen identifies novel MYC-driven vulnerabilities
              </a>
            </td>
          <td>
            Peter Lin, Corey Lourenco, Jennifer Cruickshank, Luis Palomero, Jenna E. van Leeuwen, Amy H Y Tong, Katherine Chan, S. E. Ghamrasni, M. Pujana, D. Cescon, Jason Moffat, Linda Z. Penn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa86b709f959a0cab24a9ee68d51374b7177f4bf" target='_blank'>
              Germline-Somatic Interactions in BRCA-Associated Cancers: Unique Molecular Profiles and Clinical Outcomes Linking ATM to TP53 Synthetic Essentiality.
              </a>
            </td>
          <td>
            Ali T. Arafa, Siddhartha Yadav, C. H. Marshall, Elizabeth Mauer, Minxuan Huang, Binyam Yilma, Y. van der Pol, Stamatina Fragkogianni, Emily A. Teslow, Samuel Kellen, Ella Boytim, Christine Luo, Megan Ludwig, Weijie Zhang, Arockia Jayaraj, Deborah K Armstrong, William B Isaacs, J. Drake, Hai Dang Nguyen, R. S. Huang, Calvin Chao, Emil Lou, S. Dehm, F. Couch, Justin H. Hwang, E. Antonarakis
          </td>
          <td>2025-02-28</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54bb9c5a7aaba38fdef52e5c2b089ee886ff2b49" target='_blank'>
              Seasonal brain regeneration and chromosome instability are linked to selection on DNA repair in Sorex araneus
              </a>
            </td>
          <td>
            William R. Thomas, Tanya M. Lama, Diana Moreno Santillán, Marta Farré, Cecilia Baldoni, Linelle Abueg, Jennifer R. Balacco, O. Fedrigo, G. Formenti, Nivesh Jain, J. Mountcastle, Tatiana Tilley, Alan Tracey, David A. Ray, D. Dechmann, Dominik von Elverfeldt, John Nieland, Angelique P. Corthals, Erich D. Jarvis, Liliana M. Dávalos
          </td>
          <td>2025-04-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Eukaryotic somatic cells undergo continuous telomere shortening because of end-replication problems. Approximately 10%~15% of human cancers rely on alternative lengthening of telomeres (ALT) to overcome telomere shortening. ALT cells are characterized by persistent telomere DNA replication stress and rely on recombination-based DNA repair pathways for telomere elongation. The Bloom syndrome (BLM) helicase is a member of the RecQ family, which has been implicated as a key regulator of the ALT mechanism as it is required for either telomere length maintenance or telomere clustering in ALT-associated promyelocytic leukemia bodies (APBs). Here, we summarize recent evidence detailing the role of BLM in the activation and maintenance of ALT. We propose that the role of BLM-dependent recombination and its interacting proteins remains a crucial question for future research in dissecting the molecular mechanisms of ALT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecfa6b604141f203a1f4a0229bf2941af19cc8c" target='_blank'>
              Multiple functions of the ALT favorite helicase, BLM
              </a>
            </td>
          <td>
            Shun Chang, Jiang Tan, Ren Bao, Yanduo Zhang, Jinkai Tong, Tongxin Jia, Jing Liu, Juhua Dan, Shuting Jia
          </td>
          <td>2025-03-01</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01f5cfa8e398963dd5157942872b501bf3ce27d8" target='_blank'>
              Abstract 442: Dual KAT6/7 inhibition disrupts epigenetic programs that promote tumor evolution and adaptive drug resistance
              </a>
            </td>
          <td>
            Manav Gupta, Pranav Gupta, Scott R. Tyler, N. Ghaddar, Steven L. Spivak, Zabrisky Roland, Katelyn N. Lukas, I. Shabalin, Jonathon S. Ryan, Carl Schultz, Daniel M. Walden, Santosh Keshipeddy, Muzaffar Alam, Daniel L. Severance, Anjali Bisaria, Kelly S. Tego, Parker Y. Jameson, Mason J. Appel, Brittany Cruzan, J. Jain, Michael E. Dalziel, Sunjay Sethi, Rebeca M. Choy, Richard Zang, Linghu Xin, Yuchen Bai, P. Teriete, Diana M. Muñoz, Christian R. Frey, Claire L. Neilan, J. Taygerly, Jacob Burch-Konda, P. Barsanti, , Brian T. Jones
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Transposable elements are abundant in host genomes but are generally considered to be confined to the cell in which they are expressed, with the notable exception of endogenous retroviruses. Here, we identify a group of LTR retrotransposons that infect the germline from somatic cells within the Drosophila ovary, despite lacking the fusogenic Envelope protein typically required for retroviral entry. Instead, these elements encode a short transmembrane protein, sORF2, with structural features reminiscent of viral cell-cell fusogens. Through genetics, imaging, and electron microscopy, we show that sORF2 localizes to invasive somatic protrusions, enabling the direct transfer of retrotransposon capsids into the oocyte. Remarkably, sORF2-like proteins are widespread among insect retrotransposons and also occur in piscine nackednaviruses and avian picornaviruses. These findings reveal a noncanonical, Envelope-independent transmission mechanism shared by retrotransposons and non-enveloped viruses, offering important insights into host-pathogen evolution and soma-germline interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81230deb71ae111c0bc4f1081eb1e2d5be4c7d01" target='_blank'>
              Direct cell-to-cell transmission of retrotransposons
              </a>
            </td>
          <td>
            Maya Voichek, Andreas Bernhard, Maria Novatchkova, Dominik Handler, Paul Möseneder, Baptiste Rafanel, P. Duchek, Kirsten A. Senti, Julius Brennecke
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89052d3ed2d82ce2288e348ab845b3ae781f2935" target='_blank'>
              Abstract LB265: The spatial landscape of cancer hallmarks reveals patterns of tumor ecological dynamics and drug sensitivity
              </a>
            </td>
          <td>
            E. Porta-Pardo, Mustafa Sibai, Sergi Cervilla, D. Grases, M. Esteller
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Functional characterization of genetic variants has the potential to advance the field of precision medicine by enhancing the efficacy of current therapies and accelerating the development of new approaches to combat genetic diseases. MUTYH is a DNA repair enzyme that recognizes and repairs oxidatively damaged guanines [8-oxoguanine (8-oxoG)] mispaired with adenines (8-oxoG·A). While some mutations in the MUTYH gene are associated with colorectal cancer, most MUTYH variants identified in sequencing databases are classified as variants of uncertain significance. Convoluting clinical classification is the absence of data directly comparing homozygous versus heterozygous MUTYH mutations. In this study, we present the first effort to functionally characterize MUTYH variants using precision genome editing to generate heterozygous and homozygous isogenic cell lines. Using a MUTYH-specific lesion reporter in which we site-specifically incorporate an 8-oxoG·A lesion in a fluorescent protein gene, we measure endogenous MUTYH enzymatic activity and classify them as pathogenic or benign. Further, we modify this reporter to incorporate the MUTYH repair intermediate (8-oxoG across from an abasic site) and validate it with co-immunoprecipitation experiments to demonstrate its ability to characterize the mechanism by which MUTYH mutants are defective at DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e7c0e491d2ef8489374e03b9d4bf701b388bba" target='_blank'>
              Precision genome editing and in-cell measurements of oxidative DNA damage repair enable functional and mechanistic characterization of cancer-associated MUTYH variants
              </a>
            </td>
          <td>
            Carlos A Vasquez, Nicola R B Osgood, Marcanthony U Zepeda, Dominika K Sandel, Quinn T. Cowan, Malalage N. Peiris, Daniel J Donoghue, Alexis C. Komor
          </td>
          <td>2025-03-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/960a7979050b70a2d24fbb180b4e757ef27292f4" target='_blank'>
              Abstract LB436: VRN16: A novel PKMYT1 kinase selective inhibitor with wide therapeutic window
              </a>
            </td>
          <td>
            Kwangwoo Hwang, Seohyun Jo, Ho-Soo Lee, Youngyi Lee, Donghyun Park, Songyee Han, Kyungah Seo, Jiseon Choi, Soochan Kim, Jiyeon Kim, Dong-hyuk Seo, Younho Lee, Yikyung Ko, Hyunjong Kwak, Ye Ji Son, Jeonghun Lee, Soyeon Je, Hong-ryul Jung, Daekwon Kim, Sunghwan Kim, Jaeyoung Ahn
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f27f0e3c94f86f6d8c0491e1cda5afb641525f8e" target='_blank'>
              Abstract 471: CRISPR-Cas9 as a novel cancer gene therapy approach against pancreatic cancers
              </a>
            </td>
          <td>
            S. Teh, Akhil Kotwal, Kirsten Bowland, Alexis Bennett, E. Halper-Stromberg, L. Morsberger, Sarah Wheelan, Nicholas J. Roberts, Chien-Fu Hung, M. Goldstein, , J. Eshleman
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Abstract Archaea, often thriving in extreme habitats, are believed to have evolved efficient DNA repair pathways to cope with constant insults to their genomes. However, how these organisms repair DNA double-strand breaks (DSBs), the most lethal DNA lesions, remains unclear. Here, we show that replicative primase consisting of the catalytic subunit PriS and the noncatalytic subunits PriL and PriX from the hyperthermophilic archaeon Saccharolobus islandicus is involved in DSB repair. We show that the overproduction or knockdown of PriL increases or decreases, respectively, the rate of survival and mutation frequency of S. islandicus cells following treatment with a DNA damaging agent. The increase in mutation is attributed primarily to an increase in small insertions or deletions. Further, overproduction of PriL enhances the repair of CRISPR-generated DSBs in vivo. These results are consistent with the extraordinary ability of PriSL to promote annealing between DNA strands sharing microhomology in addition to the activity of the heterodimer in terminal transfer and primer extension. The primase-mediated DSB repair is cell-cycle dependent since PriL is barely detectable during the S/G2 transition. Our data demonstrate that replicative primase is involved in DSB repair through microhomology-mediated end joining in Archaea.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41c802b392e2aa09bc0c77d38347ce453e14aa3" target='_blank'>
              Archaeal replicative primase mediates DNA double-strand break repair
              </a>
            </td>
          <td>
            Daijiang Xiong, Zhimeng Li, Wen Qi, Shaoying Wang, Junkai Huang, Ningning Zhang, Zhenfeng Zhang, Li Huang
          </td>
          <td>2025-04-22</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f499ed27ee797b62ddf192d5f361132033db2e" target='_blank'>
              Abstract 5080: Investigation of exome-wide tumor heterogeneity on colorectal tissue-based single cells
              </a>
            </td>
          <td>
            Nikolett Szakállas, Alexandra Kalmár, B. Barták, Béla Molnár, István Takács
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37141413d3e7032baebf85246c91ff6528d18e4f" target='_blank'>
              Abstract 7033: Unraveling Kiaa1549-Braf driven brain tumors: From modeling to therapy
              </a>
            </td>
          <td>
            Ylenia Cendon Florez, Ram Kannan, Matthias A. Karrajannis, Darlene Dela Cruz, Udi Segal, Danilo Maddalo, Scott Foster, R. Benezra, Andrea Ventura
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Topoisomerase III-beta (Top3b) reduces nucleic acid torsional stress and intertwining generated during RNA and DNA metabolism while protecting the genome from pathological R-loops, which otherwise result in DNA breakage and genome instability. By studying Top3b knockout mice (Top3b-KO), we find that the loss of Top3b accelerates the development of spontaneous lymphoid tumors arising in spleens and lymph nodes, the organs with prominent Top3b expression. Aging Top3b-KO mice also display splenomegaly and systemic immune alterations including neutrophilia and lymphopenia suggestive of chronic inflammation. At the molecular level, Top3b deficiency causes genome-wide R-loop accumulation in splenocytes as measured by CUT&Tag sequencing. Increased R-loops is associated with genomic DNA breaks and activation of immune signaling pathways including the IL-6 signaling, interleukin-7 signaling and cGAS-STING. Moreover, knocking-out Top3b promotes the rapid development of syngeneic EL4 T-cell lymphomas. In conclusion, our work implies that, in addition to its role in preserving the nervous system, Top3b protects from tumorigenesis and immune dysregulations. Significance Topoisomerase III-beta (TOP3B) is the only dual RNA-DNA topoisomerase. Its inactivation has previously been shown to be critical for neurodevelopmental disorders in humans and R-loop suppression in cell line models. Here, we demonstrate that loss of Top3b accelerates spontaneous lymphomagenesis and promotes syngeneic lymphoma growth in association with innate immune defects and R-loop accumulation in Top3b-KO mice. These findings provide evidence for a previously unappreciated role of Top3b as a tumor suppressor gene and immune regulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/670788860a0acb20f381f0101f5e0bb53a8bd0df" target='_blank'>
              Topoisomerase IIIβ protects from tumorigenesis and immune dysregulation
              </a>
            </td>
          <td>
            Rasel Al Mahmud, S. Baechler, A. Dhall, Sourav Saha, Hongliang Zhang, Shuling Zhang, Min-Jung Lee, Nahoko Sato, S. Rastogi, Suresh Kumar, Muhammad S. Alam, L. Saha, Beverly A. Mock, Valentina M Factor, Y. Pommier
          </td>
          <td>2025-03-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The mouse genome is transcribed at different rates in both directions from the newly formed genome after fertilization. During embryonic genomic activation (EGA/ZGA), the first RNA metabolism creates heterogeneity between blastomeres. Indeed, ZGA-dependent maternal RNA degradation is crucial to regulate gene expression and enable the initiation and acquisition of full developmental competence. Subsequently, from the new genome, in addition to mRNAs, a wide range of regulatory ncRNAs are also transcribed. Regulatory ncRNAs (non-coding RNAs) have profoundly influenced fields ranging from developmental biology to RNA-mediated non-Mendelian inheritance, exhibiting sequence-specific functions. To date, the database cataloging ncRNA is not exhaustive, but their high sequence diversity, length and low expression level can vary within the same genome depending on environmental conditions, making understanding their functions often ambiguous. Indeed, during transcription control, cellular RNA content varies continuously. This phenomenon is observed in genetically identical organisms studied—bacteria, flies, plants and mammals—due to changes in transcription rates, and therefore, it impacts cellular memory. Importantly, experimental data regarding the simple modification of RNAs levels by microinjection into fertilized mouse eggs suggest that they certainly play a driving role in establishing and transmitting newly formed expression information. The idea here is that, even in a stable genome, transcripts can vary rapidly and significantly in response to environmental changes, initiated by transcriptional variations in the genome, thus altering cellular memory.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a768d936b723e7adb7dcc685d117648dc4b4eb5c" target='_blank'>
              RNA-Mediated Non-Mendelian Inheritance in Mice: The Power of Memory
              </a>
            </td>
          <td>
            M. Rassoulzadegan
          </td>
          <td>2025-04-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Inaccurate chromosome segregation can lead to the formation of aneuploid cells that harbor an imbalanced complement of chromosomes. Several fungal species are not only able to tolerate the detrimental effects of aneuploidy but can use it to adapt to environmental pressures. The fungal pathobiont Candida albicans frequently acquires supernumerary chromosomes that enable growth in the presence of antifungal drugs or in specific host niches, yet the transcriptional changes associated with aneuploidy are not globally defined. Here, a karyotypically diverse set of C. albicans strains revealed that expression generally correlated with gene copy number regardless of the strain karyotype. Unexpectedly, aneuploid strains shared a characteristic transcriptional profile that was distinct from a generalized environmental stress response previously defined in aneuploid yeast cells. This aneuploid transcriptional response led to altered growth and oxidative balances relative to euploid control strains. The increased expression of reactive oxygen species (ROS) mitigating enzymes in aneuploid cells reduced the levels of ROS but caused an acute sensitivity to both internal and external sources of oxidative stress. Taken together, our work demonstrates common transcriptional and phenotypic features of aneuploid C. albicans cells with consequences for infection of different host niches and susceptibility to environmental stimuli.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2e3ff6416e6e4c37bdd4902bf219f1aee84a69b" target='_blank'>
              Aneuploidy confers a unique transcriptional and phenotypic profile to Candida albicans
              </a>
            </td>
          <td>
            Anna I Mackey, Robert J. Fillinger, P. S. Hendricks, Gregory J. Thomson, Christina A. Cuomo, Richard J. Bennett, Matthew Z. Anderson
          </td>
          <td>2025-04-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96bb67ca8a1fde17024b6bb4d2cb9ffe3c6efe6e" target='_blank'>
              Activation of DCTN1-RET Fusion Through Coiled-coil Domain as a Potential Target for RET Inhibitors.
              </a>
            </td>
          <td>
            Kohei Hayashi, K. Ishida, Masanori Kato, Shuichi Ohkubo, Yoshihiro Uto
          </td>
          <td>2025-03-28</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Metastasis is a leading cause of colorectal cancer (CRC)-related mortality, yet its molecular mechanisms remain poorly understood. Long noncoding RNAs (lncRNAs) have emerged as critical regulators of CRC metastasis, but their specific roles are not fully elucidated. This study identifies and characterizes a novel lncRNA LINC02167 as a critical regulator of CRC metastasis. Methods LINC02167 expression was analyzed in CRC tissues via real-time quantitative polymerase chain reaction and fluorescence in situ hybridization. Functional assays evaluated its role in CRC cell migration, invasion, and metastasis in vitro and in vivo. Mechanistic exploration involves a combination of techniques, including RNA sequencing, mass spectrometry, RNA pull-down, RNA immunoprecipitation, chromatin immunoprecipitation, luciferase reporter assays, RNA stability assays, and bioinformatics analysis, to uncover the molecular interactions and pathways regulated by LINC02167. Results LINC02167 is markedly upregulated in CRC tissues and strongly correlates with advanced clinical features and poor prognosis. Functional analyses reveal that LINC02167 enhances CRC cell migration and invasion in vitro and promotes metastasis in vivo. Mechanistically, LINC02167 serves as a molecular scaffold, forming a complex with YBX1 and ILF3 to facilitate YBX1 binding to NSUN2-mediated m5C modification sites on KSR1 mRNA, thereby stabilizing KSR1 mRNA and activating the ERK/MAPK signaling pathway to drive CRC metastasis. Additionally, MYC-driven transcriptional activation leads to the upregulation of LINC02167 in CRC. Conclusions This study uncovers a novel mechanism through which LINC02167 promotes the ERK/MAPK pathway and CRC metastasis via m5C modification, underscoring its potential as a promising therapeutic target for metastatic CRC treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13046-025-03368-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c87a1229126a6672350e24e97d2e6f03e765a6b" target='_blank'>
              LINC02167 stabilizes KSR1 mRNA in an m5C-dependent manner to regulate the ERK/MAPK signaling pathway and promotes colorectal cancer metastasis
              </a>
            </td>
          <td>
            Junwen Qi, Tao Jiang, Bowen Liu, Qihang Hu, Junnan Chen, Ning Ma, Yixin Xu, Hu Song, Jun Song
          </td>
          <td>2025-04-15</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faa65e1fd6a4ddcb57e0eaf994498d400b2eb5d2" target='_blank'>
              Tissue-specific mutagenesis from endogenous guanine damage is suppressed by Polκ and DNA repair
              </a>
            </td>
          <td>
            Yang Jiang, M. Przybilla, Linda Bakker, Foster C. Jacobs, Dylan Mckeon, Roxanne van der Sluijs, Juliette Wezenbeek, Joeri van Strien, Jeroen Willems, Alex E E Verkennis, Jamie Barnett, Adrian Baez Ortega, Federico Abascal, Peter W. Villalta, P. Knipscheer, I. Martincorena, S. Balbo, Juan Garaycoechea
          </td>
          <td>2025-03-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63affd7e1896b137912e86c03b4cbb1a8fc8724f" target='_blank'>
              Abstract 6243: Mathematical modeling of cell-free DNA fragmentation to map chromatin landscapes in health and cancer
              </a>
            </td>
          <td>
            Brian Sun Liu, Mengran Zhang, Takeshi Sugio, M. Esfahani
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d1e49024118358c3779c136165092b1f37168ca" target='_blank'>
              Single-cell parallel analysis of DNA damage and transcriptome reveals selective genome vulnerability.
              </a>
            </td>
          <td>
            Dongsheng Bai, Zhenkun Cao, Nivedita Attada, Jinghui Song, Chenxu Zhu
          </td>
          <td>2025-03-24</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3d0eb8bb7d998c887793731d6c54bbc2c2eb824" target='_blank'>
              Abstract 2983: BUB1B is an inactive kinase required for survival of chromosomally unstable cells
              </a>
            </td>
          <td>
            Nazario Bosco, R. Depetris, Jasmina Prabhakara, Akanksha Verma, Jared L. Johnson, Lewis C Cantley, Scott Drutman, Shinsan M. Su, Rumin Zhang, Derek A. Cogan, Christina H Eng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e4f1b205820b2198713ca41956bb0fc4b67c197" target='_blank'>
              Abstract 208: Epigenomic characterization of keratinocyte carcinomas in xeroderma pigmentosum
              </a>
            </td>
          <td>
            Bea Riebesehl, Lea S. Beensen, Felix Bormann, Wilfried Roth, Carlos F. Menck, Thomas G. Hofmann, F. Lyko, M. R. Paredes
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Background: KMT2A rearrangements occur in ~10% of acute myeloid leukemia (AML) cases and are critical for classification, risk stratification, and use of targeted therapy. However, insertions involving the KMT2A gene can evade detection using chromosomal analysis and/or fluorescence in situ hybridization (FISH). Methods: We present a case of a 22-year-old woman with acute monoblastic leukemia harboring a cryptic KMT2A::AFDN fusion identified by RNA sequencing. Initial FISH showed a 3′ KMT2A deletion, while conventional karyotyping and the automated bioinformatic pipeline for optical genome mapping (OGM) did not identify the canonical translocation. Results: To resolve these discrepancies, metaphase KMT2A FISH (break-apart fusion probe) was performed to assess whether KMT2A was translocated to another chromosome. However, the results did not support this possibility. As the fusion signal remained on the normal chromosome 11, with the 5′ KMT2A signal localized to the derivative chromosome 11. A subsequent manual review of the OGM data revealed a cryptic ~300 kb insertion of AFDN into the 3′ region of KMT2A, reconciling the discrepancies between chromosomal analysis, FISH, and RNA fusion results. Conclusions: This case highlights the importance of integrating multiple testing modalities with expert review when there is a discrepancy. Our findings emphasize the need for a comprehensive approach to genomic assessment to enhance diagnostic accuracy and guide therapeutic decision-making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f9b71c31c191c38bbe63bd8eb6331d6c11375f1" target='_blank'>
              Cryptic KMT2A::AFDN Fusion Due to AFDN Insertion into KMT2A in a Patient with Acute Monoblastic Leukemia
              </a>
            </td>
          <td>
            Qing Wei, Gokce A. Toruner, B. Thakral, Keyur P. Patel, N. Pemmaraju, Sa A. Wang, R. Kanagal-Shamanna, Guilin Tang, G. Issa, S. Loghavi, L. J. Medeiros, Courtney D. DiNardo
          </td>
          <td>2025-03-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e324f756104548defdd734daed3dcc1c70d2889" target='_blank'>
              Abstract 1743: QLS1209, A highly potent and selective PKMYT1 inhibitor demonstrates activity in tumors with CCNE1Amp and/ or FBXW7LoF mutations
              </a>
            </td>
          <td>
            Yuxing Zhang, Jiasheng Fu, Rongyan Wang, Jianping Chen, Changliang He, Junguo Hao, Xinrui Shao, Ling Li, Hua Qin, Guangyue Lei, Ping Chen, Dong Yang, Xinghua Cheng, Qingmei Zheng, Zheng Wang, Guqin Shi, Wenyuan Qian, Daqing Sun, Weimei Sun, Weikang Tao
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="To maintain cell fitness, deleterious genetic alterations are buffered by compensatory changes in additional genes. In cancer, buffering processes could be targeted by synthetic lethality. However, despite the large-scale identification of synthetic lethal effects in preclinical models, evidence that these operate clinically is limited. This impedes the application of synthetic lethal approaches. By integrating molecular profiling data from >9,000 cancers with synthetic lethal screens, we show that transcriptomic buffering of tumor suppressor gene (TSG) loss by hyperexpression of synthetic lethal partners is a common phenomenon, extending to multiple TSGs and histotypes. Transcriptomic buffering is also notable in cancers that phenocopy TSG loss, such as BRCAness cancers, where expression of BRCA1/2 synthetic lethal genes correlates with clinical outcome. Synthetic lethal genes that exhibit transcriptomic buffering also represent more robust synthetic lethal effects. These observations have implications for understanding how tumor cells tolerate TSG loss, in part explain transcriptomic architectures in cancer and provide insight into target selection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29a52e19b56a0e26f2474ec866322bdccd6f6854" target='_blank'>
              The transcriptomic architecture of common cancers reflects synthetic lethal interactions
              </a>
            </td>
          <td>
            S. Haider, R. Brough, Santiago Madera, Jacopo Iacovacci, Aditi Gulati, Andrew J. Wicks, J. Alexander, S. Pettitt, Andrew N J Tutt, C. J. Lord
          </td>
          <td>2025-03-03</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/197e6d30a9ae12cbdea22ce3dfece89aab97864e" target='_blank'>
              Abstract 2491: Conditional generation of mutational profiles in cancer
              </a>
            </td>
          <td>
            Andy Wu, Hannan Wong, Aakansha Narain, Vedant Sandhu, Jason J. Pitt
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3da42eb5cba4f866717a7ca7075985f83d3c3fb" target='_blank'>
              Review Article
‘An Overview Genetics of Inherited Neurological and Non-Neurological Disorders in India’
              </a>
            </td>
          <td>
            Male D. N., Amle V. S., Ade R. P., Vyvahare S. S., Pawar V. K.
          </td>
          <td>2025-04-05</td>
          <td>International Journal of Advanced Research in Science, Communication and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d9f71c97b02d4a1d072a400c199093fb87572c0" target='_blank'>
              Abstract 3458: Antagonism between tumors and lymphocytes drives coevolutionary processes throughout metastasis
              </a>
            </td>
          <td>
            Cort B Breuer, Marcos Labrado, Nathan E. Reticker-Flynn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background The occurrence and development of colorectal cancer (CRC) is an incredibly long process that involves continuous changes in the tumor microenvironment. These constant changes may ultimately result in genetic alterations and changes in the metabolic processes of some symbiotic bacteria as a way to adapt to the changing environment. Patients with CRC exhibit an altered abundance of Bacteroides fragilis (B. fragilis) as indicated by several studies. To better understand the genomic characteristics and virulence spectrum of B. fragilis strains in tumor tissues, B. fragilis strains were isolated from tumor and paracancerous tissues of CRC patients. Methods The isolates were identified using 16 S rRNA sequencing, morphological analysis, physiological and biochemical characterization and PCR, and they were then subjected to whole genome sequencing (WGS) analysis. Results A strain of B. fragilis enterotoxin (BFT) bft1-producing ZY0302 and a non-enterotoxin-producing B. fragilis ZY0804 were isolated from cancerous and paraneoplastic tissues, respectively. Analysis based on the core and nonessential genes showed that the genomic profiles of the isolates, ZY0302 and ZY0804, differed from those of B. fragilis from other tissue sources. This core and the co-evolution of non-essential genes may be the result of their adaptation to fluctuations in the tumor microenvironment and enhancing their survival. In addition, the ZY0302 and ZY0804 genomes underwent extensive horizontal gene transfer and varying degrees of genomic rearrangements, inversions, insertions, and deletion events, which may favor the enhancement of bacteria’s ability to adapt to environmental changes. For instance, the virulence factors, such as the capsular biosynthesis gene clusters and components of the type IV secretion system, acquired through horizontal gene transfer, may facilitated B. fragilis in evading immune responses and managing oxidative stress. Moreover, our analysis revealed that multiple virulence factors identified in the isolates were mainly involved in bacterial adhesion and colonization, oxidative stress, iron acquisition, and immune evasion. This observation is worth noting given that enzymes such as neuraminidase, lipase, hemolysin, protease, and phosphatase, along with genes responsible for LPS biosynthesis, which are recognized for their association with the virulence of B. fragilis, were prevalent among the isolates. Conclusions In summary, it is our assertion that the alterations observed in both core and nonessential genes of B. fragilis, which have been isolated from tissues of colorectal cancer patients, along with significant instances of horizontal gene transfer to the genome, are likely intended to enhance adaptation to the evolving conditions of the tumor microenvironment. This study may provide new insights into the interaction between B. fragilis and the CRC microenvironment. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11421-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c824eb8f15816e1643c36807ead38caf17a553" target='_blank'>
              Genomic alterations in Bacteroides fragilis favor adaptation in colorectal cancer microenvironment
              </a>
            </td>
          <td>
            Hao Yang, Yu Gan, Shenghai Jiang, Xianchang Zhu, Yang Xia, Dengmei Gong, Xianrang Xie, Yao Gong, Yi Zhang, Qian Lei, Maijian Wang, Jida Li
          </td>
          <td>2025-03-18</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b3723b406875364ef9c3c49f307934927dcf437" target='_blank'>
              Whole-arm chromosome rearrangements and telomeric signal displays: current cytogenetic updates on the examples of mammalian karyotype integrative variation.
              </a>
            </td>
          <td>
            N. S. Bulatova, A. Graphodatsky, Victor Spangenberg, Svetlana V. Pavlova
          </td>
          <td>2025-03-29</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e4f3702444f966febda408aa09a277d286c200" target='_blank'>
              Abstract 1409: Reprogramming the alternative polyadenylation of an oncogenic transcription factor as a therapeutic strategy for prostate cancer
              </a>
            </td>
          <td>
            Giulia Pagani, Chiara Casirati, Stefania Fornezza, Roberta Cacioppo, P. Gandellini
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53622534a6ad4bf417cb989b871bcc51c0303f4" target='_blank'>
              Abstract 1420: Long-read sequencing of single-stranded DNA of colorectal cancer patients
              </a>
            </td>
          <td>
            Béla Molnár, K. Rada, Nikolett Szakállas, B. Barták, Alexandra Kalmár, István Takács
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a92d215150e22b00d817410c07ac0e4856717a42" target='_blank'>
              Multidimensional OMICs reveal ARID1A orchestrated control of DNA damage, splicing, and cell cycle in normal-like and malignant urothelial cells.
              </a>
            </td>
          <td>
            Rebecca M Schlösser, Florian Krumbach, Eyleen Corrales, G. Andrieux, Christian Preisinger, Franziska Liss, A. Golzmann, M. Boerries, Kerstin Becker, Ruth Knüchel, Stefan Garczyk, Bernhard Lüscher
          </td>
          <td>2025-04-01</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccd321a5ebdb5267d14caeaf74b23869909c920a" target='_blank'>
              Restoration of tumor suppressor protein with enhanced activity using engineered tRNAs to induce tumor regression
              </a>
            </td>
          <td>
            Bangyan Kong, Zhetao Zheng, Zeyuan Mi, Zhiyang Dou, Yuelin Yang, Yuxuan Shen, Lian-Hua Dong, Jingru Lv, Ying Li, Qing Liang, Songlin Tian, Jiayu Wang, Hua Qiao, Qing Xia
          </td>
          <td>2025-03-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88e879216882ba777a52dac02a65cb65b0e331d4" target='_blank'>
              Abstract 5361: Clinical implications of The Cancer Genome Atlas molecular classification system in esophagogastric cancer
              </a>
            </td>
          <td>
            H. Walch, Raktim Borpatragohain, Justin Jee, W. Chatila, Christopher Fong, S. Maron, Geoffrey Y Ku, D. Ilson, Yelena Y. Janjigian, , Pari Shah, Dan Coit, M. Bains, Valerie W. Rusch, B. Park, M. Bott, Katherine D. Gray, David R. Jones, Michael F. Berger, N. Schultz, V. Strong, D. Molena, S. Sihag
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06e5c087c7af55ddb110acf4864affc68be63a67" target='_blank'>
              Abstract 4065: Unveiling drivers of esophageal adenocarcinoma with CRISPR pooled screen and single-cell RNA-sequencing
              </a>
            </td>
          <td>
            Ebtihal H. Mustafa, Julia V. Milne, Ka Meng Wu, Katherine Papastratos, N. Thio, Wayne Phillips, Nicholas Clemons
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d3123c40ae6a85b19d981bbd4cd36ea0d8081af" target='_blank'>
              Abstract 7181: Gene enhancers as novel rationale therapeutic targets for impeding colorectal cancer progression
              </a>
            </td>
          <td>
            F. Genua, Xinxin Shao, E. Finnegan, Christina Caragine, Sarah L. Maguire, Paulien Van Minsel, G. Kirwan, Bernard Thienpont, Neville Sanjana, Weipeng Wang, Sudipto Das
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbcd5ab9fc14637f7ac25a5d726846fbbce055ae" target='_blank'>
              Abstract 2396: Gene fusion and variant-aware isoform detection with functional prediction from long-read RNA sequencing
              </a>
            </td>
          <td>
            Colette Felton, Andrea Galvez, L. Sayles, Christopher Vollmers, Alejandro Sweet-Cordero, Angela N. Brooks
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae14464f2ce77e42ffad40f3c6898b5621cece3e" target='_blank'>
              PLK4: Master Regulator of Centriole Duplication and Its Therapeutic Potential.
              </a>
            </td>
          <td>
            Muhammad Hamzah, Franz Meitinger, Midori Ohta
          </td>
          <td>2025-04-21</td>
          <td>Cytoskeleton</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Ovarian cancer is the deadliest gynecological cancer globally, with epithelial ovarian cancer (EOC) comprising up to 90% of cases. A molecular characterization linking the histological subtypes with tumor grade in EOC has been suggested. Variations in genetic biomarkers such as BRCA1/2, MSH2, MLH1/6, BRIP1, and RAD51C/D have been studied in EOC. In addition, molecular characteristics, including DNA methylation and RNA transcription, are being explored as potential new biomarkers for the diagnosis and prognosis of this type of neoplasia. The present review focused on the role of DNA methylation and non-coding RNA expression in the development of ovarian carcinomas and their association with diagnosis, prognosis, and the resistance of cancer cells to radiotherapy and chemotherapy. The present review considered the transition from the DNA structure to the RNA expression in ovarian carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e068de948072497de4cda02de43b3f132918d12" target='_blank'>
              Role of DNA methylation and non‑coding RNAs expression in pathogenesis, detection, prognosis, and therapy‑resistant ovarian carcinoma (Review)
              </a>
            </td>
          <td>
            Victor M Del Castillo Falconi, Jenny Godinez Rodriguez, Verónica Fragoso‑Ontiveros, Laura Contreras-Espinosa, Abraham Pedroza‑Torres, José Díaz‑chávez, Luis A Herrera
          </td>
          <td>2025-04-01</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="KRAS is a proto-oncogene that contains activating mutations in up to 30% of tumors. Many conventional therapies inhibit both cancerous and normal cells, which may cause toxicity. Thus, programmable mutant-selective targeted inhibitors are needed. Peptide nucleic acids (PNAs) incorporate base sequences analogous to DNA, with modified peptide backbones instead of ribose-phosphate backbones, allowing PNAs to hybridize with DNA with high avidity to suppress transcription. Here, we developed KRAS G12D-selective PNA oligomers with novel cell-penetrating flanking regions. Fluorescein-labeled PNA oligomers displayed high uptake rates in cells and nuclei. Exposure to PNA-delivery peptide conjugates resulted in repression of KRAS G12D mRNA and protein expression within 2 hours and lasting up to 48 hours. Varying cell-penetrating peptide (CPP) compositions and lengths of complementary KRAS sequences were tested using dose-response cell viability assays. These experiments identified configurations that were effective at selectively preventing growth of on-target KRAS G12D cells, while relatively sparing off-target KRAS G12C cells. Electrophoretic mobility shift assays demonstrated in vitro binding and selectivity for KRAS G12D DNA sequences. CPP-PNA-G12D-1 was effective against a panel of pancreatic ductal adenocarcinoma cell lines and patient-derived xenografts in vivo. These results show promise for an enhanced PNA-delivery peptide conjugate strategy as both a tool for studying tumors driven by oncogenic point mutations and as a potential therapeutic strategy to selectively target mutant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c1932e0ee0ee85a7870c225677d456ad8a7a109" target='_blank'>
              Selective targeting of oncogenic KRAS G12D using peptide nucleic acid oligomers attached to cell-penetrating peptides
              </a>
            </td>
          <td>
            Jayati Mondal, Dennis Lam, Mary E. Gerritsen, T. Brotz, Jodi G. Kennedy, Bruce Rehlaender, Arthur J. Ross, Daniel E. Levy, Christopher A. Bonagura, W. Lanzilotta, Frank McCormick, Jeffrey H. Rothman, Andrew L. Wolfe
          </td>
          <td>2025-04-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Many antibiotics indirectly generate reactive oxygen species (ROS) that can damage bacterial genomes. Oxidised nucleobases become genotoxic when they are targeted for repair through excision, generating a single-strand discontinuity that can be converted to a double-strand break (DSB) by an oncoming replication fork. Because the genomic location of nucleobase oxidation is stochastic, investigating the fate of DNA replication machinery (replisome) at single-strand discontinuities has been limited. Here we have addressed this issue by expressing Cas9 nickases in Bacillus subtilis to create site specific single-strand discontinuities in a bacterial chromosome. We find that nicks in either leading or lagging strand arrest bacterial replication fork progression and generate a DSB that requires repair using homologous recombination to allow replication restart. These discoveries provoke reassessment of the fundamental mechanism of bacterial homologous recombination and provide insights to the development of alternative antimicrobials by identifying a specific pathway that can potentiate ROS-dependent bacterial killing. HIGHLIGHTS Single-strand discontinuities inactivate the bacterial replisome Nicks on the leading or lagging strand template differentially affect fate of the helicase Double-strand breaks are repaired via homologous recombination and elicit the replication restart pathway Resection often generates a DNA strand with a 5′-end that must be processed to ensure productive helicase reloading">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f733169863ada127f78f4c42d1d0de7e45edaadc" target='_blank'>
              Rescuing bacterial genome replication: essential functions to repair a double-strand break and restart DNA synthesis
              </a>
            </td>
          <td>
            Charles Winterhalter, Stepan Fenyk, H. Murray
          </td>
          <td>2025-03-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc2065faaa8c8b01822bceb610976fd9fb9507b1" target='_blank'>
              Mechanism of DNA replication fork breakage and PARP1 hyperactivation during replication catastrophe.
              </a>
            </td>
          <td>
            Pedro Ortega, Elodie Bournique, Junyi Li, Ambrocio Sanchez, Gisselle Santiago, Brooke R. Harris, Josefine Striepen, J. Maciejowski, Abby M. Green, Rémi Buisson
          </td>
          <td>2025-04-18</td>
          <td>Science advances</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The importance of genomic structural variants (SVs) is well-appreciated, but much less is known about their mutational properties than of single nucleotide variants (SNVs) and short indels. The reason is simple: the longer the mutation, the less likely it will be covered by a single sequencing read, thus the harder it is to map unambiguously to a unique genomic location. Here we report SV mutation rate estimates from six mutation accumulation (MA) lines from two strains of C. elegans (N2 and PB306) using long-read (PacBio) sequencing. The inferred SV mutation rate ∼1/10 the SNV rate and ∼1/4 the short indel rate. We identified 40 mutations, and removed 52 false positives (FP) by manual inspection of each SV call. Excluding one atypical line (5 mutations, 35 FPs), the signal (inferred mutant) to noise (FP) ratio is approximately 2:1. False negative rates were determined by simulating variants in the reference genome, and observing ’recall’. Recall rate ranges from >90% for short indels and declines as SV length increases. Small deletions have nearly the same recall rate as small insertions (∼100bp), but deletions have higher recall rates than insertions as size increases. The reported SV mutation rate is likely an underestimate. A quarter of identified SV mutations occur in SV hotspots that harbor pre-existing low complexity repeat variation. By comparison of the spectrum of spontaneous SVs to wild isolates, we infer that natural selection is not only efficient at removing SVs in exons, but also removes roughly half of SVs in intergenic regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/145fe9e1ff1d8adbb4d54b58e524b442b04f4a5a" target='_blank'>
              High rate of mutation and efficient removal by selection of structural variants from natural populations of Caenorhabditis elegans
              </a>
            </td>
          <td>
            A. Saxena, Charles F. Baer
          </td>
          <td>2025-03-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Purpose Small cell carcinoma of the ovary, hypercalcemic type (SCCOHT) is a rare and lethal cancer lacking effective treatment. Its genomic mutations and tumor microenvironment need further exploration. Methods We performed whole-exome sequencing or gene panel test to explore the SMARCA4 mutation spectrum in SCCOHT (15 samples). Single-cell RNA sequencing was conducted on one primary lesion with matched normal ovarian tissue and one recurrent lesion to investigate the intra-tumoral heterogeneity and immune microenvironment. Multiplex immunofluorescence staining validated T cell infiltration and PD-1 expression. Results 13/15 (86.7%) patients harbored SMARCA4 mutations. The loss of heterozygosity (LOH) occurred in 10/15 (66.7%) patients. Cancer cells and immune cells were observed in SCCOHT tumors. Cancer cells were further divided into seven subtypes and one from recurrent lesion exhibited the highest stemness accompanied by high expression of genes related to cell mitosis (AURKB, CHEK2, CCNB1, WEE1), DNA repair (BRCA1, RAD51) and epigenetic (EZH2, DNMT1). Immune cells mainly included macrophages and T cells. Lipid-associated tumor-associated macrophages (TAMs) was mainly in primary lesion while inflammatory cytokine-enriched TAMs in recurrent lesion. CD4+/ CD8+ T cell infiltration was observed in SCCOHT tumor and a certain proportion of T cells expressed PD-1. Conclusions SCCOHT exhibits universal SMARCA4 LOH and significant intra-tumoral heterogeneity, suggesting potential therapeutic targets, including CHEK2, CCNB1, and WEE1. Exhausted T cells and distinct TAM subsets infiltrate tumors. Targeting macrophage polarization or cytokine signaling may also be promising. These findings provide insights for developing novel therapies to improve outcomes in SCCOHT. Clinical trial number Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01649-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdba27a2b17d1b242308fa85ab562553932c973d" target='_blank'>
              Single-cell RNA sequencing reveals the intra-tumoral heterogeneity and immune microenvironment of small cell carcinoma of the ovary, hypercalcemic type
              </a>
            </td>
          <td>
            Yijun Gao, Kewei Zheng, Haowen Tan, Mingyi Kang, Bingjian Lu, Ling Chen, Jing Xu, Chong Lu, Ranran Chai, Congjian Xu, Yu Kang
          </td>
          <td>2025-04-09</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e18a54b51e51772ed2be11a7f9c312e5504aa79" target='_blank'>
              Abstract 6127: Genomic instability regulates STING mediated anti-tumor immunity through distinctive cytosolic DNA structures
              </a>
            </td>
          <td>
            Shayla R. Mosley, Natalie Lapa, Courtney Mowat, Kristi Baker
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771e1b2442ed827fe07315db2d2daebd9063c39b" target='_blank'>
              Abstract LB048: Functional characterization of genetic regulators driving melanoma drug resistance via Single-Cell high-content CRISPR screening
              </a>
            </td>
          <td>
            Zihan Xu, Ziyu Lu, A. Ugurbil, Abdulraouf Abdulraouf, Jianxiang Zhang, Wei Zhou, Junyue Cao
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7839d82620913bd93e3b37227d99428d183e100a" target='_blank'>
              Abstract 4413: Combined transcriptomic and epitranscriptomic profiling identifies THBS1 as a regulator of enzalutamide resistance in prostate cancer
              </a>
            </td>
          <td>
            Emmanuelle Hodara, Lisa Swartz, Aubree Mades, Tong Xu, S. Rhie, A. Goldkorn
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e993964715fda888d384887c1b0e033eb189929c" target='_blank'>
              Abstract LB308: Rewiring of methylation during embryogenesis can seed cancer decades later
              </a>
            </td>
          <td>
            Lori D. Kregar, N. Williams, Joe Lee, Michael Spencer-Chapman, O. Nikolaienko, E. Mitchell, Liv B Gansmo, P. Lønning, Elisa Laurenti, Lucy Yates, S. Knappskog, J. Nangalia, Peter J Campbell
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e93f87b45a480b89d7a05bed3b25376e4e282650" target='_blank'>
              Abstract LB304: NTHL1-D239Y variant enhances tumorigenesis in colorectal cancer through cancer initiation and chronic inflammation
              </a>
            </td>
          <td>
            Tarek Masannat, F. Ghishan, P. Kiela, J. Sweasy
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0f99bba46d4c798400f4f104ff216c0c2a15d1" target='_blank'>
              Abstract 4294: SMCHD1 is a synthetic lethal strategy for cancers with a cancer-testis antigen gene expression signature
              </a>
            </td>
          <td>
            Victor Quereda, Youyou Zhang, Diana Klimaszewska, Stefano Tonin, Theodore Groth, Yanhua Rao, Michael T. McCabe, Benjamin Schwartz, Markus Queisser, Rachel Buxton, Tom Berrodin, Simon Peace
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e843a00844275db4fbe08c936ffad2c5534db0e" target='_blank'>
              Abstract 99: HMGB1deficiency occurs in a broad range of human cancers and is often associated with unfavorable tumor phenotype
              </a>
            </td>
          <td>
            Viktoria Chirico, Hena Sharifi, M. Tsourlakis, Şeyma Büyücek, Clara Marie von Bargen, Katharina Möller, F. Lutz, D. Dum, C. Hube-Magg, E. Bady, Lisa Hornsteiner, Georgia Makrypidi-Fraune, Piero Caneve, M. Lennartz, Morton Freytag, S. Rico, S. Kind, V. Reiswich, E. Burandt, T. Clauditz, P. Lebok, C. Fraune, T. Krech, S. Minner, A. Marx, W. Wilczak, R. Simon, G. Sauter, S. Steurer, K. Jansen
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Abstract Oxford-Nanopore PromethION sequencing is a PCR-free method that retains epigenetic markers and provides direct quantitative information about DNA methylation. Using this long-read sequencing technology, we successfully assembled 5 myxozoan genomes free from discernible host DNA contamination, surpassing previous studies in both quality and completeness. Genome assembly revealed DNA methylation patterns within myxozoan genomes, particularly in GC-rich regions within gene bodies. The findings not only refute the notion of myxozoans lacking DNA methylation capability but also offer a new perspective on gene regulation in these parasites. The high-quality genome assemblies lay a solid foundation for future research on myxozoans, including new strategies to control these commercially significant fish pathogens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00d951e1f9c2c5cdc9c33694af88484b900b44b" target='_blank'>
              Long-read metagenomic sequencing negates inferred loss of cytosine methylation in Myxosporea (Cnidaria: Myxozoa)
              </a>
            </td>
          <td>
            Antonio Starčević, Rayline T. A. Figueredo, J. Naldoni, L. L. Corrêa, Beth Okamura, Edson A. Adriano, Paul F Long
          </td>
          <td>2025-01-06</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/195e5c9d321296053fe8059a5c1ae5b1f465a2aa" target='_blank'>
              Abstract 5054: Understanding the genetic heterogeneity within clonal spheroids and its interaction with the tumor microenvironment
              </a>
            </td>
          <td>
            Anshu Yadav, Rupesh Chaturvedi, Rana P. Singh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third most common cancer and remains a significant challenge due to high rates of drug resistance and limited therapeutic options. Circular RNAs (circRNAs) are increasingly recognized for their roles in CRC initiation, progression, and drug resistance. However, no circRNA-based therapies have yet entered clinical development, underscoring the need for comprehensive detection and mechanistic studies of circRNAs in CRC. Here, we identified and characterized a circular RNA, circ_0001766 (hsa_circ_0001766), through microarray analysis of CRC tissues. Our results showed that circ_0001766 is downregulated in CRC tissues and closely associated with patient survival and metastasis. Functional experiments demonstrated that circ_0001766 inhibits CRC cell proliferation, migration and invasion both in-vitro and in-vivo. Mechanistically, hypoxia downregulates Quaking (QKI), an RNA-binding protein essential for the biogenesis of circ_0001766 by binding to introns 1 and 3 of PDIA4 pre-mRNA. Reduced QKI expression under hypoxic conditions leads to decreased circ_0001766 levels in CRC. Circ_0001766 acts as a competitive endogenous RNA, sponging miR-1203 to prevent the degradation of PPP1R3C mRNA. Loss of circ_0001766 results in decreased PPP1R3C expression, leading to the activation of mTOR signaling and increased phosphorylation of Myc, which promotes CRC progression and rapamycin resistance. Our study reveals that overexpression of circ_0001766 or PPP1R3C in CRC cells inhibits the mTOR and Myc pathway, thereby resensitizing cells to rapamycin. The combination of circ_0001766 or PPP1R3C with rapamycin markedly inhibits CRC cell proliferation and induces apoptosis by reducing rapamycin-induced Myc phosphorylation. In summary, our study elucidates a critical circ_0001766/miR-1203/PPP1R3C axis that modulates CRC progression and rapamycin resistance. Our findings highlight circ_0001766 as a promising therapeutic target in CRC, providing a new avenue for enhancing the efficacy of existing treatments and overcoming drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b74672a547f1af8a48e1db65d0a8f5bc5c6388dd" target='_blank'>
              QKI-induced circ_0001766 inhibits colorectal cancer progression and rapamycin resistance by miR-1203/PPP1R3C/mTOR/Myc axis
              </a>
            </td>
          <td>
            Yulai Zhou, Yan Gao, Yinghui Peng, Changjing Cai, Ying Han, Yihong Chen, Gongping Deng, Yanhong Ouyang, Hong Shen, Shan Zeng, Yangfeng Du, Zemin Xiao
          </td>
          <td>2025-04-23</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/771756b2729241a36b5c4bbcbf190204361bdbc6" target='_blank'>
              Abstract 1427: Characterization of epigenetic regulation of MYC in acute myeloid leukemia
              </a>
            </td>
          <td>
            W. S. Streitfeld, William M. Yashar, Itallia V. Pacentine, , Samantha J. Worme, Gurkan G. Yardimci, Andrew C. Adey, Theodore P. Braun
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e04034cb839d11700bfc365e9345aa1519c5848" target='_blank'>
              Abstract 6712: Epigenetic reactivation of a tumor suppressor program in leukemia
              </a>
            </td>
          <td>
            Eric Wang, Cuijuan Han, Alexander Calderon, I. Aifantis, A. Tsirigos, Abimbo Lawal, S. H. Rajendran, Sadik Karma, Hussein A. Abbas, M. Y. Yassouf, Zhiping Zhang, Pedro Miura
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94b4705110ed5255e541024e217bd1b38f97bdc5" target='_blank'>
              Abstract 2802: Circular RNA variants of theNUMB gene modulate the oncogenicity of oral squamous cell carcinoma
              </a>
            </td>
          <td>
            Chung-Hsien Chou, Pin-Yu Chen, Shu-Chun Lin, Kuo-Wei Chang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5d3048f7fc1ab2dec5d8c000f11dda761b0959e" target='_blank'>
              Abstract SY04-01: Mapping the functional microproteome in cancer
              </a>
            </td>
          <td>
            John R. Prensner
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Simple Summary Ductal adenocarcinoma of the pancreas is a cancer with exceedingly dismal prognosis as less than 5% of all patients survive more than 5 years. Therefore, targeted molecular therapies are urgently needed. MTAP deficiency is currently one of the most promising molecular targets for tumor therapies. MTAP deficiency—often caused by homozygous 9p21 deletion—is easily detectable by immunohistochemistry because it is characterized by a complete MTAP expression loss. It makes tumor cells more susceptible to PRMT5- and MAT2A-inhibiting drugs as well as to pemetrexed (antifolate therapy). However, the success of targeted therapies critically depends on the extent of heterogeneity of the targeted tumor cell trait in clinical tumors. In this study, intratumoral heterogeneity of MTAP deficiency was thus assessed in more than 200 ductal adenocarcinomas of the pancreas in a tissue microarray format and whole-section analysis was carried out by immunohistochemistry and fluorescence in situ hybridization. MTAP deficiency was found in almost 40% of all cancers, was always caused by homozygous deletion, and was rarely heterogenous in TMA and large-section analyses (less than 10%). Pancreatic adenocarcinomas may thus represent an ideal cancer type for studying new drugs targeting MTAP-deficient cancer cells in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa497701c09c398746a0513a639855972ce4cb6a" target='_blank'>
              Deficiency of MTAP Is Frequent and Mostly Homogeneous in Pancreatic Ductal Adenocarcinomas
              </a>
            </td>
          <td>
            N. Gorbokon, Katharina Teljuk, V. Reiswich, M. Lennartz, S. Minner, R. Simon, G. Sauter, W. Wilczak, T. Clauditz, Nina Schraps, T. Hackert, Faik G. Uzunoglu, M. Kluth, L. Bubendorf, Matthias Matter, F. Viehweger, Morton Freytag, Frank Jacobsen, Katharina Möller, S. Steurer
          </td>
          <td>2025-04-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8876255590dea7ef4a31a07f23a3590537b155f1" target='_blank'>
              Immunogenomic cancer evolution: A framework to understand cancer immunosuppression.
              </a>
            </td>
          <td>
            S. Kumagai, Yusaku Momoi, H. Nishikawa
          </td>
          <td>2025-03-28</td>
          <td>Science immunology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f70cae7350af93e8189b0a48977a508877000d4" target='_blank'>
              Abstract SY27-02: Exploring the significance of somatic mtDNA mutations in the context of acute leukemia
              </a>
            </td>
          <td>
            M. Kundu
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/879969a6652e4d8acc441456b7bcb05f2b8a551c" target='_blank'>
              Abstract 3345: Application of cGAS-immunopositive micronuclei as an indicator of tumor aggressiveness in liposarcoma
              </a>
            </td>
          <td>
            Kiyota Kato, Toru Motoi, Yu Naito, Masumi Ogawa, Chihiro Onda, Hideo Arai, Shin-ichiro Horiguchi, Toshihide Hirai, T. Hishima
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Oscillatory p53 expression occurs in individual cells responding to DNA breaks. While the majority of cells exhibit the same qualitative response, quantitative features of the oscillations (e.g., amplitude or period) can be highly variable between cells, generating heterogeneity in downstream cell fate responses. Since heterogeneity can be detrimental to therapies based on DNA damage, methods to induce synchronization of p53 oscillations across cells in a population have the potential to generate more predictable responses to DNA-damaging treatments. Using mathematical modeling and time-lapse microscopy, we demonstrated that p53 oscillations can be synchronized through the phenomenon of phase resetting. Surprisingly, p53 oscillations were synchronized over a wider range of damage-induction frequencies than predicted computationally. Recapitulating the range of synchronizing frequencies required, non-intuitively, a less robust oscillator. We showed that p53 phase resetting altered the expression of downstream targets responsible for cell fate depending on target mRNA stability. This study demonstrates that p53 oscillations can be phase reset and highlights the potential of driving p53 dynamics to reduce cellular variability and synchronize cell fate responses to DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/372ff4f36befbd056cd96855dd2d6817d3ff38d6" target='_blank'>
              Pulsed stimuli enable p53 phase resetting to synchronize single cells and modulate cell fate
              </a>
            </td>
          <td>
            Harish Venkatachalapathy, Samuel Dallon, Zhilin Yang, Samira M. Azarin, Casim A. Sarkar, Eric Batchelor
          </td>
          <td>2025-03-03</td>
          <td>Molecular Systems Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b995a18ffa49845c7801e75f621e56fe9ecdff48" target='_blank'>
              Probability-based sequence comparison finds the oldest ever nuclear mitochondrial DNA segments in mammalian genomes
              </a>
            </td>
          <td>
            Muyao Huang, Martin C. Frith
          </td>
          <td>2025-03-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Mosaic chromosomal alterations of the autosomes (aut-mCAs) are large structural somatic mutations which cause clonal hematopoiesis and increase cancer risk. Here, we detected aut-mCAs in 1,011,269 participants across four biobanks. Through integrative analysis of the minimum critical region and inherited genetic variation, we found that proto-oncogenes exclusively drive chromosomal gains, tumor suppressors drive losses, and copy-neutral events can be driven by either. We identified three novel inherited risk loci in CHI3L2, HLA class II, and TERT that modulate aut-mCA risk and ten novel aut-mCA-specific loci. We found specific aut-mCAs are associated with cardiovascular, cerebrovascular, or kidney disease incidence. High-risk aut-mCAs were associated with elevated plasma protein levels of therapeutically actionable targets: NPM1, PARP1, and TACI. Participants with multiple high-risk features such as high clonal fraction, more than one aut-mCA, and abnormal red cell morphology had a 50% cumulative incidence of blood count abnormalities over 2 years. Leveraging inherited variation, we causally established aut-mCAs as premalignant lesions for chronic lymphocytic leukemia. Together, our findings provide a framework integrating somatic mosaicism, germline genetics, and clinical phenotypes to identify individuals who could benefit from preventative interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f202aec9ed88d0660af9bd306eaa9b009384d1c" target='_blank'>
              Genetic drivers and clinical consequences of mosaic chromosomal alterations in 1 million individuals
              </a>
            </td>
          <td>
            K. Zhao, Y. Pershad, H. Poisner, X. Ma, K. Quade, C. Vlasschaert, T. Mack, N. K. Khankari, K. von Beck, J. Brogan, A. Kishtagari, R. Corty, Y. Li, Y. Xu, A. Reiner, P. Scheet, P. Auer, A. G. Bick
          </td>
          <td>2025-03-06</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Single-stranded DNA (ssDNA) gaps impact genome stability and PARP inhibitor (PARPi) sensitivity, especially in BRCA1/2-deficient tumors. Using single-molecule DNA fiber analysis, electron microscopy, and biochemical methods, we found that MRN, CtIP, EXO1, and DNA2-WRN/BLM resect ssDNA gaps through a mechanism different from their actions at DNA ends. MRN resects ssDNA gaps in the 3'-to-5' direction using its pCtIP-stimulated exonuclease activity. Unlike at DNA ends, MRN does not use its endonucleolytic activity to cleave the 5'-terminated strand flanking the gap or the ssDNA. EXO1 and DNA2-WRN/BLM specifically resect the 5' end of the gap independent of MRN-CtIP. This resection process alters ssDNA gap repair kinetics in BRCA1-proficient and -deficient cells. In BRCA1-deficient cells treated with PARPis, excessive resection results in larger ssDNA gaps, hindering their repair and leading to DNA breaks in subsequent cell cycle stages due to ssDNA gaps colliding with DNA replication forks. These findings broaden our understanding of the role of human nucleases in DNA metabolism and have significant implications for defining the mechanisms driving PARPi sensitivity in BRCA-deficient tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbd5bfadd87ec60e42f11dd8435521e451769fad" target='_blank'>
              MRN-CtIP, EXO1, and DNA2-WRN/BLM act bidirectionally to process DNA gaps in PARPi-treated cells without strand cleavage.
              </a>
            </td>
          <td>
            Isabelle M Seppa, Ilaria Ceppi, Mithila Tennakoon, Giordano Reginato, Jessica Jackson, Celia D Rouault, Sumedha Agashe, Vladislav O Sviderskiy, Mangsi Limbu, Erica Lantelme, A. Meroni, Stefan Braunshier, Damiano Borrello, Priyanka Verma, Petr Cejka, A. Vindigni
          </td>
          <td>2025-03-24</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/822f11adca9e71b6e0ff40e6fc00f85ee3a4109f" target='_blank'>
              Aneuploidy and ploidy variation conditioned by the B chromosome of maize
              </a>
            </td>
          <td>
            Patrice S. Albert, Hua Yang, Zhi Gao, Cassidy DeVore, J. Birchler
          </td>
          <td>2025-04-25</td>
          <td>Heredity</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca7ecdbac1d09bcfe33d7f9c228aa944ed8e3ef5" target='_blank'>
              Abstract 6484: Single-cell transcriptomic analysis of hepatocellular carcinoma: elucidating intratumoral heterogeneity and immune microenvironment crosstalk
              </a>
            </td>
          <td>
            Qingyang Zhang, Y. Tsui, Vanilla Xin Zhang, Daniel Wai-Hung Ho, Irene Oi-Lin Ng
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea10556f7e26587dbcdc0c9ad631944d5c7f3c43" target='_blank'>
              Abstract 3824: Investigating the role of allelic imbalance in pancreatic cancer aggressiveness, survival, and therapeutic resistance
              </a>
            </td>
          <td>
            Enrico Gurreri, Li Zhang, Hania Khan, Lisa Maria Mustachio, Ningping Feng, T. Heffernan, Joseph R Marszalek, Basturk Olca, David Klimstra, Stefano Sioletic, A. Sgambato, G. Tortora, Giulio F. Draetta, Luigi Perelli, G. Genovese
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3680f1af9ff6a4b7bc7625b9703b156e9e23eacd" target='_blank'>
              Abstract 2719: Decoding role of KMT5C-H4K20me3 in EGFRi resistance in non-small cell lung cancer
              </a>
            </td>
          <td>
            Jihye Son, Alexandra Glaws, Christina Davidson, Ching-Hua Shih, Paula M. Vertino, A. Kasinski
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5281b0b530161da0728f26a1aa6d170b1c6d1d79" target='_blank'>
              Abstract 4891: Genome-first approach identifies shelterin complex gene variants and associated cancers
              </a>
            </td>
          <td>
            Akanksha Nagarkar, Hasset T. Nurelegne, Jung Kim, K. D. de Andrade, Sharon A. Savage
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aee6939711394df481ecca9ad3d57b55fd487283" target='_blank'>
              Abstract SY14-03: Exploiting dysregulated ribosomal homeostasis in chromosome 9p21.3 deleted cancers and microsatellite unstable cancers
              </a>
            </td>
          <td>
            Patricia Borck, Isabella Boyle, Kristina Jankovic, Nolan R. Bick, Kyla Foster, Anthony Lau, Lucy Parker-Burns, Daniel A Lubicki, Ashir A. Borah, Nicholas J Lofaso, Sohani Das Sharma, Riya V Kishen, Joshua M Dempster, Francisca Vazquez, Edmond M Chan
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdbc9b66da684de5689ef08f5586218feacc4935" target='_blank'>
              Germline loss-of-function mutations in PPP2R1B in patients with a personal and family history of cancer.
              </a>
            </td>
          <td>
            Sahar Mazhar, Caitlin M. O’Connor, Alexis Harold, Amanda Dowdican, P. Ulintz, Erika N Hanson, Yuping Zhang, Michelle F. Jacobs, Sofia D. Merajver, Mark W. Jackson, Anthony Scott, A. Sieuwerts, A. Chinnaiyan, G. Narla
          </td>
          <td>2025-04-03</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>167</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b41aaa58f8e3f0c715544a400d743c926b588c9a" target='_blank'>
              Abstract 5412: Developing combination therapies for telomerase-overexpressing cancers
              </a>
            </td>
          <td>
            Vincent Maranda, Frederick S. Vizeacoumar, Yue Zhang, Liliia Kyrylenko, A. Freywald, Franco J. Vizeacoumar
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="We have recently demonstrated that genetic inactivation of EWSR1 : : FLI1 by CRISPR/Cas9, successfully blocks cell proliferation in a cell model of Ewing sarcoma. However, CRISPR/Cas9-mediated gene editing can exhibit off-target effects, and thus, precise regulation of Cas9 expression in target cells is essential to develop gene-editing strategies to inactivate EWSR1 : : FLI1 in Ewing sarcoma cells. In this study, we demonstrate that Cas9 can be specifically expressed in Ewing sarcoma cells when located downstream a promoter consisting of GGAA repeats and a consensus TATA box (GGAAprom). Under these conditions, Cas9 is selectively expressed in Ewing sarcoma cells that express EWSR1 : : FLI1 oncoproteins, but not in cells expressing wild-type FLI1. Consequently, Ewing sarcoma cells infected with GGAAprom>Cas9 and a specific gRNA designed to inactivate EWSR1 : : FLI1, showed successful EWSR1 : : FLI1 inactivation and the subsequent blockade of cell proliferation. Notably, GGAAprom>Cas9 can be efficiently delivered to Ewing sarcoma cells via adenoviral vectors both in vitro and in vivo, highlighting the potential of this approach for Ewing sarcoma treatment. Our results demonstrate that the CRISPR/Cas9 machinery is safe and specific for Ewing sarcoma cells when driven under a GGAAprom, paving the way for the development of cancer gene therapies based on the selective expression of genes with therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ba0d95a65d0d8225a5257acbb9853ce632bbe70" target='_blank'>
              Targeted inactivation of EWSR1 : : FLI1 gene in Ewing sarcoma via CRISPR/Cas9 driven by an Ewing-specific GGAA promoter
              </a>
            </td>
          <td>
            Saint T Cervera, Selene Martínez, María Iranzo-Martínez, L. Notario, R. M. Melero-Fernández de Mera, Javier Alonso
          </td>
          <td>2025-03-15</td>
          <td>Cancer Gene Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="ABSTRACT Gene fusions represent important oncogenic driver mutations resulting in aberrant cellular signaling. In up to 17% of all solid tumors at least one gene fusion can be identified. Precision therapy targeting fusion gene signaling has demonstrated effective clinical benefit. Advancements in clinically relevant next-generation sequencing and bioinformatic techniques have enabled expansion of therapeutic opportunity to subpopulations of patients with fusion gene expression. Clinically, tyrosine inhibitors have shown efficacy in treating fusion gene expressing cancers. Fusion genes are also clonal mutations, meaning it is a personal cancer target involving all cancer cells of that patient, not just a subpopulation of cancer cells within the cancer mass. Thus, both fusion signal disruption and immune signal targeting are effective therapeutic directions. This review discusses fusion gene targeting, therapeutic resistance, and molecular biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a39d77dcb6dc4121a7fa7e989f8a588d3f3ba74" target='_blank'>
              Clinical characterization and therapeutic targeting of fusion genes in oncology
              </a>
            </td>
          <td>
            Susan Morand, Lauren Rager, Daniel Craig, Alexander Nemunaitis, Khalil Choucair, Donald Rao, L. Stanbery, Richard C Phinney, Adam Walter, Maurizio Ghisoli, John Nemunaitis
          </td>
          <td>2025-03-24</td>
          <td>Future Oncology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Introduction: Leukocyte adhesion deficiency type 1 (LAD1) is a severe inborn error of immunity caused by mutations in the ITGB2 gene, which encodes the beta-2 integrin subunit (CD18). These mutations lead to the absence or deficiency of CD18/CD11a, b, and c heterodimers, crucial for leukocyte adhesion and immune function. CRISPR-Cas9 Gene editing technology represents a promising approach for correcting these genomic defects restore the stable expression of CD18 and reverse the disease. Methods: We developed a CRISPR-Cas9-based gene correction strategy using Jurkat cells and patient-derived lymphoblastoid cell lines as surrogates for hematopoietic progenitor cells. Three candidate gRNAs were first predicted in silico using CRISPOR and experimentally tested in wild-type ITGB2-expressing Jurkat cells to identify the gRNA with the highest genomic DNA cleavage efficiency. The most efficient gRNA was then paired with espCas9 and used alongside five homology-directed repair templates (HDRs) (single-stranded donor oligonucleotides, ssODNs) to repair ITGB2 defects in patient-derived lymphoblastoid cell lines. CD18 expression levels in edited cells were quantified via flow cytometry, and whole-genome sequencing (WGS) was conducted to assess off-target effects and insertion accuracy. Results: Among the three candidate gRNAs, 2-rev gRNA exhibited the highest genomic cleavage rate in Jurkat cells. Using this gRNA with espCas9 and HDR-2, we achieved a 23% restoration of CD18 expression in LAD1 patient-derived cells, a level sufficient to change the disease course from severe to moderate. Whole-genome sequencing confirmed the absence of off-target mutations or undesired DNA insertions, demonstrating high specificity and precision in gene correction. Discussion: This CRISPR-Cas9-based method provides a precise and effective approach for correcting ITGB2 mutations in LAD1 patients. The high-fidelity gene editing process, validated through WGS, supports its potential for future applications in CD34+ hematopoietic stem cell therapies. The approach can be further optimized for clinical translation, offering a path toward a stable and long-term cure for LAD1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af64b4b32aef04c80bbc3b2f45c925d4b04fb25b" target='_blank'>
              Lymphoblastoid and Jurkat cell lines are useful surrogate in developing a CRISPR-Cas9 method to correct leukocyte adhesion deficiency genomic defect
              </a>
            </td>
          <td>
            Ahmad R. Ramadan, Noureddine Ben Khalaf, Khaled Trabelsi, H. Bakheit, I. Ben‐Mustapha, M. Barbouche, M. Fathallah
          </td>
          <td>2025-03-21</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a59583f3996a4cd3fc1170e30a4a01eaa9a2e62" target='_blank'>
              Abstract 372: Loss of RB1 sensitizes TP53-mutated cancer cells to WEE1 inhibition by azenosertib
              </a>
            </td>
          <td>
            Daehwan Kim, O. Harismendy, Catherine Lee, Erika Cabrera, Danielle Jandial, Doris S Kim, M.R. Lackner, Jianhui Ma
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e9bae0a2ea1cdb031459bcffccc70e1301dd19c" target='_blank'>
              DNA Fragment Fusion and Nucleic Acid Detection by Fusion Recombinase-Aided Amplification.
              </a>
            </td>
          <td>
            Xingxing Xiao, Xi Yang, Kexin Xu, Fuyuan Huang, Yan Zhang, Yelin Jiang, Yangbin Shi, Qinghong Zhou, Luying Wang, Jiahai Lu, Zongliang Gao, Yongliang Lou
          </td>
          <td>2025-03-19</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f68977168d050d5c5f7c3fa27b15c83e3b09b0a" target='_blank'>
              Abstract 1011: Transposable element patterns across genetic ancestries in prostate cancer patients
              </a>
            </td>
          <td>
            Nicholas Korvink, Lucas Ueta, Hannah Park, Vinay Kumar, Farah Rahmatpanah
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b17c8f4c66aab3898b741804dfbd546dbdd92394" target='_blank'>
              Abstract 6825: Novel therapeutic strategies targeting MECOM rearranged AMLs discovered by unbiased drug screen
              </a>
            </td>
          <td>
            Christine E. Birdwell, W. Fiskus, Christopher P. Mill, T. Kadia, Naval G Daver, C. Dinardo, K. Sasaki, John A. Davis, Kaberi Das, Hanxi Hou, Surbhi Sharma, R. K. Shamanna, Michele Ceribelli, K. Bhalla
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="Objective This study examines the cellular heterogeneity of epithelial cells within pancreatic ductal adenocarcinoma (PDAC) and their contributions to tumor progression, metastasis, and immunosuppressive interactions using single-cell RNA sequencing. Methods Single-cell RNA-sequencing data from two datasets (GSE154778 and GSE158356) were integrated using the Harmony algorithm, followed by quality control, clustering, and differential gene expression analysis. Distinct subpopulations of epithelial cells were identified, and their gene expression profiles were analyzed. To assess the malignancy of these subpopulations, single-cell copy number variation (CNV) analysis and trajectory analysis were conducted. Additionally, intercellular communication was examined using the CellChat platform. Results The analysis revealed pronounced heterogeneity among PDAC epithelial cells, with specific subpopulations exhibiting distinct roles in tumor proliferation, extracellular matrix remodeling, and metastatic dissemination. Subpopulations 4 and 6 were characterized by increased CNV levels and a more malignant phenotype, suggesting an enhanced capacity for metastasis. Single-cell trajectory analysis, along with CellChat, mapped the temporal evolution of epithelial cells, identifying key regulatory genes such as DCBLD2 and JUN. A prognostic model incorporating five key genes, including KLF6, was developed and demonstrated strong predictive accuracy for patient outcomes. Notably, KLF6 emerged as a critical prognostic marker associated with immune modulation, particularly through interactions with M2 macrophages. Conclusion The study highlights the pronounced heterogeneity of epithelial cells in PDAC and their distinct contributions to tumor progression, metastasis, and immune modulation. Through single-cell transcriptomic and CNV analyses, we identified epithelial subpopulations with varying malignant potentials and distinct interactions with the tumor microenvironment. Among these, KLF6 emerged as a key regulator associated with immune modulation and metastasis. Our findings emphasize the significance of epithelial cell heterogeneity in shaping pancreatic cancer progression. These insights provide a foundation for future investigations into novel prognostic markers and therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1186/s12957-025-03793-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c02e096347df4014f9175b24f19fbd3cae4aa9d0" target='_blank'>
              Deciphering the heterogeneity of epithelial cells in pancreatic ductal adenocarcinoma: implications for metastasis and immune evasion
              </a>
            </td>
          <td>
            Jie Liu, Hui Li, Xiuyun Lin, Jiani Xiong, Guangfeng Wu, Lingyan Ding, Bin Lin
          </td>
          <td>2025-04-16</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66047aa9da0269c010edbf7b80afbec9051b4fd1" target='_blank'>
              Pan-cancer analysis reveals PRRT4 is a potential prognostic factor of AML.
              </a>
            </td>
          <td>
            Wenqiong Xiang, Fangjun Chen, Hao Zhou, Guilin Ren, Guangliang Qiang, Li Wang
          </td>
          <td>2025-04-25</td>
          <td>Hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ced01904c6ac2b8e8cc6e89a2ccb440437dbae71" target='_blank'>
              Abstract 2729: Investigating the interplay between RNA methyltransferase PCIF1 and DNA hypomethylating drug decitabine in TP53-mutated acute myeloid leukemia
              </a>
            </td>
          <td>
            Li-Yang Shiau, Po-Shen Ko, Kao-Jung Chang, Shih-Hwa Chiou
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce10ef6d68c6903f3dcfc0aa9cd8d7cbbe833212" target='_blank'>
              Abstract 2992: Uncovering metabolic dysregulation and therapeutic targets in aneuploid models of squamous cell carcinoma
              </a>
            </td>
          <td>
            Nadja Zhakula, Sejal Jain, Laura Byron, Joan J. Castellano, Matthew L. Meyerson, Alison M. Taylor
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Recent research has shifted the focus from the genetic code of DNA to its structural variations, which significantly impact cancer, genetic diseases, and gene regulation. Structural changes, such as the transition from B-DNA to A-DNA, influence DNA stability and flexibility and are affected by factors like DNA methylation and sugar puckering. This study is the first to investigate the relationship between DNA conformational changes and lifespan in two rodent species. The analysis focused on long-lived Nannospalax xanthodon and shorter-lived Rattus rattus, utilizing infrared spectroscopy and principal component analysis (PCA) to examine liver DNA. Results indicated that transition from B-form to A- and Z-forms were more prevalent in N. xanthodon than in R. rattus. However, the dominant DNA conformations in both species are in B-form. Additionally, N-type sugar puckers (C3-endo conformation), associated with these DNA forms, were more prominent in N. xanthodon. In contrast, S-type sugar puckers (C2-endo conformation), characteristic of B-DNA, were found at lower levels in N. xanthodon. Furthermore, the variations in methylation-specific structural modifications of nucleobases were quantitatively assessed among these species. The study proposes a significant connection between the long lifespan of N. xanthodon, which live underground, and their unique DNA structure, offering insights into how different DNA forms, as well as the conformations of their backbone and sugar-base components, may affect longevity, highlighting potential research avenues regarding the biomolecular aspects of aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90e194ada8d97ff0f6ac7375a7dc3c14a580e955" target='_blank'>
              Structural features of DNA and their potential contribution to blind mole rat (Nannospalax xanthodon) longevity
              </a>
            </td>
          <td>
            Gamzenur Sönmez, Tuba Yağcı Gurbanov
          </td>
          <td>2025-03-25</td>
          <td>Biogerontology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) poses a formidable clinical challenge due to multidrug resistance (MDR) caused by tumor heterogeneity. To elucidate the intricate mechanisms underlying HGSOC heterogeneity, we conducted a comprehensive analysis of five single-cell transcriptomes and eight spatial transcriptomes derived from eight HGSOC patients. This study provides a comprehensive view of tumor heterogeneity across the spectrum of gene expression, copy number variation (CNV), and single-cell profiles. Our CNV analysis revealed intratumor heterogeneity by identifying distinct tumor clones, illuminating their evolutionary trajectories and spatial relationships. We further explored the homogeneity and heterogeneity of CNV across tumors to pinpoint the origin of heterogeneity. At the cellular level, single-cell RNA sequencing (scRNA seq) analysis identified three meta-programs that delineate the functional profile of tumor cells. The communication networks between tumor cell clusters exhibited unique patterns associated with the meta-programs governing these clusters. Notably, the ligand-receptor pair MDK - NCL emerged as a highly enriched interaction in tumor cell communication. To probe the functional significance of this interaction, we induced NCL overexpression in the SOVK3 cell line and observed enhanced tumor cell proliferation. These findings indicate that the MDK - NCL interaction plays a crucial role in promoting HGSOC tumor growth and may represent a promising therapeutic target. In conclusion, this study comprehensively unravels the multifaceted nature of HGSOC heterogeneity, providing potential therapeutic strategies for this challenging malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/375bc93ef6f0dd0954afbd4a3ef7ab6a5339dc00" target='_blank'>
              Deciphering ovarian cancer heterogeneity through spatial transcriptomics, single-cell profiling, and copy number variations
              </a>
            </td>
          <td>
            Songyun Li, Zhuo Wang, Hsien-da Huang
          </td>
          <td>2025-03-04</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d3f608a4fe82676c1f869676898474a93427b4d" target='_blank'>
              Abstract LB449: InTraSeq unveils transcriptomic and proteomic landscapes of Th17 differentiation
              </a>
            </td>
          <td>
            Majd M. Ariss
          </td>
          <td>2025-04-25</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2bb3598a34b80763bb5456df92bb300a45ad0cc" target='_blank'>
              Abstract 1166: Lurbinectedin synergizes with selinexor by inhibiting DNA damage repair and enhancing immune cell infiltration in neuroendocrine lung and prostate tumors
              </a>
            </td>
          <td>
            E. Redin, P. Manoj, Juan Qiu, Barbara de Mello, Hong Zhong, Suleyman Vural, Y. Zhan, Rodrigo Romero, N. Socci, R. Koche, Charles Sawyers, E. de Stanchina, Charles M. Rudin, Á. Quintanal-Villalonga
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/779b836948ef731bffffb3b6e6c60be9d313ae8e" target='_blank'>
              Abstract 6523: A multi-omics analysis correlating intratumoral microbial abundance with methylation patterns in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Tianyi Chen, Annie N. Do, Suravi Bajaj, Erin Jang, Rishi Yalamarty, Pranava Gande, Kevin Zhang, Ellie Sang, W. T. Li, J. Wang-Rodriguez, W. Ongkeko
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="CRISPR-associated (Cas) transposases (CAST) are RNA-guided systems capable of programmable integration of large segments of DNA without creating double-strand breaks. Engineered Cascade CAST function in human cells but are challenging to deploy due to the complexity of the targeting components. Unlike Cascade, which require three Cas proteins, type V-K CAST require a single Cas12k effector for targeting. Here, we show that compact type V-K CAST from uncultivated microbes are repurposable for programmable DNA integration into the genome of human cells. Engineering for nuclear localization and function enables integration of a therapeutically relevant transgene at a safe-harbor site in multiple human cell types. Notably, off-targets are rare events reproducibly found in specific genomic regions. These CAST advancements are expected to accelerate applications of genome editing to therapeutic development, biotechnology, and synthetic biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729b5eccbfaf3e212cbd8bcab377e318bdda9fa9" target='_blank'>
              Integration of therapeutic cargo into the human genome with programmable type V-K CAST
              </a>
            </td>
          <td>
            Jason Liu, Daniela S Aliaga Goltsman, Lisa M. Alexander, Khak Khak Khayi, Jennifer H Hong, Drew T Dunham, Christine A Romano, Morayma M. Temoche-Diaz, Shailaja Chadha, Rodrigo Fregoso Ocampo, Jennifer Oki-O'Connell, Owen P Janson, K. Turcios, Liliana Gonzalez-Osorio, Jared Muysson, Jenat Rahman, Sarah M Laperriere, Audra E. Devoto, C. Castelle, Cristina N. Butterfield, Gregory J Cost, Christopher T. Brown, Brian C. Thomas
          </td>
          <td>2025-03-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a30e97ab73c3d9f10f09249e3cecf21c25bbdf3" target='_blank'>
              Establishment of SHON Gene Knockout Breast Cancer Cell Line using CRISPR-Cas9 Technology
              </a>
            </td>
          <td>
            Alyssa Clemas
          </td>
          <td>2025-03-26</td>
          <td>Rangahau Aranga: AUT Graduate Review</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="For nearly three decades, interspecies somatic cell nuclear transfer (iSCNT) has been explored as a potential tool for cloning, regenerative medicine, and wildlife conservation. However, developmental inefficiencies remain a major challenge, largely due to persistent barriers in nucleocytoplasmic transport, mitonuclear communication, and epigenome crosstalk. This review synthesized peer-reviewed English articles from PubMed, Web of Science, and Scopus, spanning nearly three decades, using relevant keywords to explore the molecular mechanisms underlying iSCNT inefficiencies and potential improvement strategies. We highlight recent findings deepening the understanding of interspecies barriers in iSCNT, emphasizing their interconnected complexities, including the following: (1) nucleocytoplasmic incompatibility may disrupt nuclear pore complex (NPC) assembly and maturation, impairing the nuclear transport of essential transcription factors (TFs), embryonic genome activation (EGA), and nuclear reprogramming; (2) mitonuclear incompatibility could lead to nuclear and mitochondrial DNA (nDNA-mtDNA) mismatches, affecting electron transport chain (ETC) assembly, oxidative phosphorylation, and energy metabolism; (3) these interrelated incompatibilities can further influence epigenetic regulation, potentially leading to incomplete epigenetic reprogramming in iSCNT embryos. Addressing these challenges requires a multifaceted, species-specific approach that balances multiple incompatibilities rather than isolating a single factor. Gaining insight into the molecular interactions between the donor nucleus and recipient cytoplast, coupled with optimizing strategies tailored to specific pairings, could significantly enhance iSCNT efficiency, ultimately transforming experimental breakthroughs into real-world applications in reproductive biotechnology, regenerative medicine, and species conservation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2095defd86819f42b7aa13e7b28c1855e90cdf2a" target='_blank'>
              The Complexities of Interspecies Somatic Cell Nuclear Transfer: From Biological and Molecular Insights to Future Perspectives
              </a>
            </td>
          <td>
            Peachanika Pankammoon, Marvin Bryan Segundo Salinas, C. Thitaram, A. Sathanawongs
          </td>
          <td>2025-04-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea1a8d6967192526fb7f6ecd11f9a23af479e446" target='_blank'>
              Abstract 7499: Driver mutations propel subclonal evolution and spatial patterns of tumor growth in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Evit R. John, Pooja Shukla, Anish Jain, Alexandria Lau, T. Lesluyes, Huw Ogilvie, M. Tarabichi, J. Roszik, Yinyin Yuan, P. Van Loo, Lawrence Kwong, Yun Shin Chun
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Background Gastric cancer is a highly aggressive malignancy characterized by a complex tumor microenvironment (TME). Cancer-associated fibroblasts (CAFs), which are a key component of the TME, exhibit significant heterogeneity and play crucial roles in tumor progression. Therefore, a comprehensive understanding of CAFs is essential for developing novel therapeutic strategies for gastric cancer. Methods This study investigates the characteristics and functional information of CAF subtypes and explores the intercellular communication between CAFs and malignant epithelial cells (ECs) in gastric cancer by analyzing single-cell sequencing data from 24 gastric cancer samples. CellChat was employed to map intercellular communication, and Seurat was used to integrate single-cell sequencing data with spatial transcriptome data to reconstruct a comprehensive single-cell spatial map. The spatial relationship between apCAFs and cancer cells was analyzed using multicolor immunohistochemistry. Results Cells were categorized into nine distinct categories, revealing a positive correlation between the proportions of epithelial cells (ECs) and fibroblasts. Furthermore, six fibroblast subpopulations were identified: inflammatory (iCAFs), pericytes, matrix (mCAFs), antigen-presenting (apCAFs), smooth muscle cells (SMCs), and proliferative CAFs (pCAFs). Each of these subpopulations was linked to various biological processes and immune responses. Malignant ECs exhibited heightened intercellular communication, particularly with CAF subpopulations, through specific ligand-receptor interactions. High-density regions of CAF subpopulations displayed spatial exclusivity, with pericytes serving as a source for iCAFs, mCAFs, and apCAFs. Notably, malignant ECs and apCAFs showed increased interactions, with certain ligand-receptor pairs potentially impacting the prognosis of gastric cancer. Multiplex immunohistochemistry (mIHC) confirmed the close spatial proximity of apCAFs to cancer cells in gastric cancer. Conclusion Our study provided a comprehensive characterization of CAF heterogeneity in gastric cancer and revealed the intricate intercellular networks within the TME. The identified CAF subpopulations and their interactions with malignant cells could serve as potential therapeutic targets. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06376-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24163c4c446c34923a67d83b5c03139aa6a3cb4" target='_blank'>
              Single-cell RNA sequencing and spatial transcriptomics reveal the heterogeneity and intercellular communication of cancer-associated fibroblasts in gastric cancer
              </a>
            </td>
          <td>
            Xijie Zhang, Bo Ren, Bo Liu, Rui Wang, Sen Li, Yuzhou Zhao, Wence Zhou
          </td>
          <td>2025-03-18</td>
          <td>Journal of Translational Medicine</td>
          <td>2</td>
          <td>4</td>
        </tr>

        <tr id="Tandem repeats play an important role in centromere structure, subtelomeric regions, DNA methylation, recombination and the regulation of gene activity. Analysis of their distribution in genomes offers a potential means for predicting putative centromere locations, which continues to be a challenge for genome annotation. Here we present RepeatOBserver (https://github.com/celphin/RepeatOBserverV1), a new tool for visualising repeat patterns and identifying putative centromere locations, using a Fourier transform of DNA walks. RepeatOBserver can identify and visualise a broad range of perfect and imperfect repeats (3-5000 bp long) in genome assemblies without any a priori knowledge of repeat sequences or the need for optimising parameters. RepeatOBserver heatmaps can distinguish between tandem and retrotransposon repeats. We analysed 159 chromosomes with experimentally-verified centromere positions from 12 plant and animal species. We find that 93% of experimentally-verified tandem repeat centromeres occur in regions of low sequence diversity and 97% of retrotransposon centromeres occur in regions with a high abundance of repeat lengths. Depending on the centromere type predicted by the heatmaps, putative centromere locations can be predicted using either a genomic Shannon diversity index or a repeat abundance sum. RepeatOBserver can also locate other regions of interest including potential neocentromeres and gene copy variation. Split and inverted tandem repeats at inversion boundaries suggest that chromosomal inversions or mis-assemblies can also be located. RepeatOBserver is a flexible tool for comprehensive characterisation of repeat patterns that can be used to visualise and identify a variety of regions of interest in genome assemblies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b7a0c655a4ba01e7821226bff9c91a6b3f8818" target='_blank'>
              RepeatOBserver: Tandem Repeat Visualisation and Putative Centromere Detection.
              </a>
            </td>
          <td>
            Cassandra Elphinstone, Rob Elphinstone, Marco Todesco, Loren H. Rieseberg
          </td>
          <td>2025-03-04</td>
          <td>Molecular ecology resources</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/719b971bcc934d40b7d1d6ebe27c7858cb375380" target='_blank'>
              Chromatin as a Coevolutionary Graph: Modeling the Interplay of Replication with Chromatin Dynamics
              </a>
            </td>
          <td>
            Sevastianos Korsak, Krzysztof Banecki, Karolina Buka, Piotr J. Górski, Dariusz Plewczynski
          </td>
          <td>2025-04-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c9e1b40d6dd2d79567e3b2693bbf168417cd0f7" target='_blank'>
              Pan-cancer analysis unveils the role and mechanisms of neddylation modifications in tumorigenesis.
              </a>
            </td>
          <td>
            Qianhua Hu, Xiang Li, Ping Wang, Ying Xie
          </td>
          <td>2025-03-19</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e1d772bceb37d658c289fadfd2de9d117e82a8a" target='_blank'>
              Abstract 4103: MLL4/COMPASS dysfunction in cancer and treatment
              </a>
            </td>
          <td>
            Zibo Zhao, Sarah Gold, Yuki Aoi, K. Meghani, Luke St John, Rukkia Liaqat, Carolyn Moloney, Yanni Yu, Jun Qian, I. Ben-Sahra, Rintaro Hashizume, D. Mahalingam, Joshua Meeks, A. Shilatifard
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11550165fed57fd5f16146624c3f1bdaea238042" target='_blank'>
              Abstract 4252: Genetic and pharmacological disruption of the HBS1L/PELO complex impairs mRNA homeostasis and leads to in vivo tumor regressions in FOCAD-deleted cancers
              </a>
            </td>
          <td>
            Lauren Martires, M. Tonini, Hongxiang Zhang, Minjie Zhang, Chengyin Min, Charlotte B Pratt, Katherine Lazarides, Frank J. Bruzzese, D. Whittington, Patrick McCarren, T. Teng, Madhavi Bandi, Shanchuan Zhao, Stephen Paik, Eden N. Gordon, Shangtao Liu, Alvin Lu, Maria Dam Ferdinez, Binzheng Shen, Hannah Stowe, Yi Yu, Samuel R. Meier, Ashley H Choi, Lauren Grove, Alan Huang, Alice Tsai, W. Mallender, Wenhai Zhang, Jannik N Andersen, Edward Wu, Hilary E. Nicholson
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/029a236a2721717e13973bdc5e5b49afdd737065" target='_blank'>
              Abstract 2624: Aneuploidy induction mechanism in Ewing sarcoma
              </a>
            </td>
          <td>
            Priyani H. Hapugaswatta, M. Azuma
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b914af171018318ea435ca066b2219faeffe1da3" target='_blank'>
              Abstract 212: Disruption of the menin-MLL complex impedes HCC maintenance and increases sensitivity to NF-Y loss
              </a>
            </td>
          <td>
            Margarita M. Dzama, Peyton C. Kuhlers, Jesse R. Raab
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2daedc5d36f880d0e2fc581751bb206d1ca1d20" target='_blank'>
              Recurrent spontaneous miscarriages from sperm after ABVD chemotherapy in a patient with Hodgkin's lymphoma: sperm DNA and methylation profiling.
              </a>
            </td>
          <td>
            G. Lecuyer, A. Rolland, A. Neyroud, Bertrand Evrard, Nathan Alary, Clemence Genthon, Nathalie Dejucq-Rainsford, Célia Ravel, Jessika Moreau, N. Moinard, Mohamed Hadi Mohamed Abdelhamid, Christophe Klopp, Louis Bujan, Frédéric Chalmel
          </td>
          <td>2025-04-15</td>
          <td>Asian journal of andrology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ffb5861f3091086b8e6d39be2fc47951f44015b" target='_blank'>
              Cohesins: Crossroad Between Cornelia de Lange Spectrum and Cancer Predisposition.
              </a>
            </td>
          <td>
            Laura Rigotti, Stefano Rebellato, Antonella Lettieri, Silvia Castiglioni, M. Mariani, Simona Totaro, C. Saitta, C. Gervasini, G. Fazio, Valentina Massa, Giovanni Cazzaniga, Angelo Selicorni
          </td>
          <td>2025-04-05</td>
          <td>American journal of medical genetics. Part A</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18fc5411928817db2e4ae82f8b06dabe1ec683eb" target='_blank'>
              Abstract 2672: Eco-evolutionary profiling of the UV-driven clonal dynamics and tumor initiation
              </a>
            </td>
          <td>
            S. Avdieiev, L. Tordesillas, Karol Prieto, O. Chiang, Zhihua Chen, Nihir Patel, Sofia Cordero, L. S. Simoes, Y. A. Chen, Noemi Andor, Robert A. Gatenby, Elsa R. Flores, Christopher J. Whelan, Joel S. Brown, Kenneth Y. Tsai
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f55512a8c15d7d2448765df1044f7457324d44a" target='_blank'>
              Abstract 2488: Employing RNA base editors to engineer personalized tumor-specific neoantigens
              </a>
            </td>
          <td>
            Rona Merdler-Rabinowicz, Netanel Landesman, E. Ruppin, E. Levanon
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Simple Summary Telomerase plays a critical role in maintaining the telomere length, thereby enabling tumor cells to escape senescence and acquire unlimited proliferative potential, a hallmark of cancer. Increasing evidence suggests that TERT, the catalytic subunit of telomerase, exerts additional biological functions that promote cancer progression independently of its role in telomere maintenance. These extratelomeric functions involve the regulation of signaling pathways that are critical for cell survival and proliferation, establishing feed-forward loops that drive cancer cell growth, resistance to apoptosis, and disease progression. This study underscores the importance of TERT’s non-canonical functions in B-cell malignancies and highlights the potential of targeting these functions as an innovative therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fa0c56c50991697be5ea164c0309587220e3483" target='_blank'>
              Beyond Telomeres: Unveiling the Extratelomeric Functions of TERT in B-Cell Malignancies
              </a>
            </td>
          <td>
            S. Giunco, M. R. Petrara, Stefano Indraccolo, Vincenzo Ciminale, Anita De Rossi
          </td>
          <td>2025-03-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8ac2ba6fe27ed44bff66e7ee62b13701829278a" target='_blank'>
              Abstract 6437: Mutational patterns in normal tissue hold the key to understanding the cellular origins of cancer
              </a>
            </td>
          <td>
            Kirsten Kübler, R. Karlić, Mendy Miller, C. Stewart, William D. Foulkes, P. Polak, Peter F. Arndt, G. Getz
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd9f59a39a075b237d1c5fa6c39932e43dffde05" target='_blank'>
              Long term latency of highly mutated cells in normal mouse skin is reversed by exposure to tumor promoters and chronic tissue damage.
              </a>
            </td>
          <td>
            Y. R. Li, E. Kandyba, Kyle Halliwill, R. Delrosario, Quan Tran, Nora Bayani, Di Wu, Olga K. Mirzoeva, Melissa Q Reeves, S. M. A. Islam, Laura Riva, E. Bergstrom, Kavya Achanta, John DiGiovanni, Ludmil B. Alexandrov, Allan Balmain
          </td>
          <td>2025-03-13</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42b078825f058bd38a50c194336988df8e0a2a93" target='_blank'>
              Enhancer hijacking discovery in acute myeloid leukemia by pyjacker identifies MNX1 activation via deletion 7q.
              </a>
            </td>
          <td>
            Etienne Sollier, Anna Riedel, U. Toprak, Justyna A. Wierzbinska, D. Weichenhan, J. P. Schmid, M. Hakobyan, Aurore Touzart, E. Jahn, B. Vick, Fiona Brown-Burke, Katherine Kelly, Simge Kelekçi, Anastasija Pejkovska, Ashish Goyal, Marion Bähr, Kersten Breuer, Mei-Ju May Chen, Maria Llamazares-Prada, M. Hartmann, M. Schönung, Nadia Correia, A. Trumpp, Yomn Abdullah, Ursula Klingmüller, S. S. Mughal, B. Brors, Frank Westermann, Elias Ulrich, Robert J. Autry, Matthias Schlesner, S. Vosberg, Tobias Herold, P. Greif, Dietmar Pfeifer, Michael Lübbert, Thomas Fischer, F. Heidel, Gebhard Gebhard, W. Walter, Torsten Haferlach, Ann-Kathrin Eisfeld, Krzysztof Mrózek, D. Nicolet, L. Bullinger, L. Smeenk, Claudia A. J. Erpelinck-Verschueren, Roger Mulet-Lazaro, Ruud Delwel, Aurélie Ernst, Michael Scherer, P. Lutsik, I. Jeremias, Konstanze Döhner, H. Döhner, D. Lipka, C. Plass
          </td>
          <td>2025-03-31</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/187ef3faa0e6c95091fda1daef8c22d574a2979f" target='_blank'>
              Training your genome
              </a>
            </td>
          <td>
            Alan Herbert
          </td>
          <td>2025-04-08</td>
          <td>Open Access Government</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb9bbc590d0b38c9e40c5b483743c16479ab91ce" target='_blank'>
              Abstract 5090: Bioinformatics analysis of the role of asymmetric self renewal associated genes in the cancer progression
              </a>
            </td>
          <td>
            Seokyoung Hwang, Minju Lee, Jayhyun Cho, Minkyu Kim, Sang Kook Lee, Minsoo Noh
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b17b9e4fc19871fa1320841a02f8c8ec183453e" target='_blank'>
              Abstract 5170: A xenograft mouse model of virus-positive Merkel cell carcinoma directly links germ cell theory to somatic cancer
              </a>
            </td>
          <td>
            Sara Contente, Sarah Rahman, Wendy Yang
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Ubiquitin-Specific Protease 39 (USP39) has been implicated in numerous malignancies, however, its pathogenic mechanisms and impact on the tumor immune microenvironment (TIME) remain incompletely characterized. Based on The Cancer Genome Atlas (TCGA) and the Genotype-Tissue Expression (GTEx) databases, we investigated the diagnostic and prognostic values of USP39 across various cancer types. Additionally, we examined the correlation between USP39 expression and immune-related gene signature, immune cell infiltration pattern, tumor microsatellite instability (MSI), and tumor mutation burden (TMB). This study specifically focused on exploring the clinical relevance and molecular functions of USP39 in pancreatic adenocarcinoma (PAAD), with particularly emphasis on its role in shaping the TIME and modulating responses to immunotherapy. The results demonstrated that evaluated USP39 expression significantly correlated with advanced tumor stage and unfavorable clinical outcomes across multiple cancer types, most notably in PAAD. Functional enrichment analysis indicated that USP39 potentially promotes tumor progression through multiple oncogenic signaling cascades. In vitro experimental validation confirmed that USP39 knockdown inhibited migration and proliferation of pancreatic cancer cells while inducing apoptosis. Additionally, we identified significant positive correlations between USP39 expression and immune checkpoint molecules, particularly prominent in PAAD. Furthermore, we observed associations between USP39 expression and TMB in 16 cancer types and MSI in 11 cancer types, suggesting that heightened USP39 expression may enhance responsiveness to immunotherapeutic interventions. Collectively, our findings establish USP39 as a valuable immune-related biomarker with both diagnostic and prognostic utility across multiple cancer types, especially PAAD, underscoring its potential as a promising therapeutic target for cancer immunotherapy. Clinical trial number Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14096-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da88d72c5b2fdc180fcf555322747ee1904f03a7" target='_blank'>
              Identification of USP39 as a prognostic and predictive biomarker for determining the response to immunotherapy in pancreatic cancer
              </a>
            </td>
          <td>
            Jiahui Yuan, Beibei Xu, Yongcheng Su, Pingping Zhang, Xianbin Zhang, Peng Gong
          </td>
          <td>2025-04-22</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/675c67dd0133028e8c6f69ff63a5bc578aace569" target='_blank'>
              TOPO-seq reveals DNA topology-induced off-target activity by Cas9 and base editors.
              </a>
            </td>
          <td>
            Min Duan, Pan Gao, Yi-Zhou Zhang, Yu-Long Hu, Lei Zhou, Zhong-Chen Xu, Houyuan Qiu, Xiao-Han Tong, Rui-Jin Ji, Xinlin Lei, Hao Yin, Cun-Lan Guo, Ying Zhang
          </td>
          <td>2025-04-02</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05d2093be8b42fa3a464257730acab22244b4f95" target='_blank'>
              Abstract 3658: LINE1 ORF1P is a druggable target for early onset colorectal cancer and cancer prevention
              </a>
            </td>
          <td>
            Dan Zhao, Yifei Wang, P. Neeli, Yong Li
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf9b3ebe106350ec8d1fdf90b3379079a41b4ae0" target='_blank'>
              Abstract 7159: Characterizing the role of a KDM3C germline polymorphism in chemoradiotherapy outcomes for rectal and head and neck cancers
              </a>
            </td>
          <td>
            A. Hasan, Pragya Priyadarshini, E. Demidova, P. Czyzewicz, Takahiko Murayama, Shreya M Shah, Mark Andrake, Yan Zhou, Roland M. Dunbrack, Israel Cañadas, Thomas J Galloway, M. Einarson, Barbara Burtness, E. Golemis, J. E. Meyer, S. Arora
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdb7d8e81c3f345d4ada707194f030cd48f14acb" target='_blank'>
              The malignant signature gene of cancer-associated fibroblasts serves as a potential prognostic biomarker for colon adenocarcinoma patients
              </a>
            </td>
          <td>
            Hao Zhang, Zirui Zhuang, Li Hong, Ruipeng Wang, Jinjing Xu, Youyuan Tang
          </td>
          <td>2025-04-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is a clinically and genetically heterogenous disease. Recent next-generation sequencing (NGS) studies have uncovered numerous low-frequency mutated genes in CLL patients, with SAMHD1 emerging as a candidate driver gene. However, the biological and clinical implications of SAMHD1 mutations remain unclear. Using CRISPR/Cas9, we generated CLL models to investigate the impact of SAMHD1 deficiency on pathogenesis and explore therapeutic strategies. Moreover, we performed NGS in treatment-naïve CLL patients to characterize SAMHD1 mutations and employed RNA-sequencing to evaluate their clinical significance. Our study shows that SAMHD1 inactivation impairs the DNA damage response by reducing homologous recombination efficiency through BRCA1 and RAD51 dysregulation. Importantly, SAMHD1 colocalizes with BRCA1 at DNA damage sites in CLL cells. This research also identifies that SAMHD1-mutated cells are more sensitive to PARP inhibition. Clinically, SAMHD1 dysfunction negatively impacts clinical outcome of CLL cases: SAMHD1 mutations reduce failure-free survival (median 46 vs 57 months, p = 0.033), while low SAMHD1 expression associates with shorter time to first treatment (median 47 vs 77 months; p = 0.00073). Overall, this study elucidates that SAMHD1 dysfunction compromises DNA damage response mechanisms, potentially contributing to unfavorable clinical outcomes in CLL, and proposes PARP-inhibitors as a potential therapeutic approach for SAMHD1-mutated CLL cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10778333b9af70279760bf116919d89a182e85ca" target='_blank'>
              SAMHD1 dysfunction impairs DNA damage response and increases sensitivity to PARP inhibition in chronic lymphocytic leukemia
              </a>
            </td>
          <td>
            Alberto Rodríguez-Sánchez, M. Quijada-Álamo, C. Pérez-Carretero, A. B. Herrero, Andrés Arroyo-Barea, J. Dávila-Valls, Araceli Rubio, A. Garcia de Coca, R. Benito-Sánchez, Ana E. Rodríguez-Vicente, J. Hernández-Rivas, M. Hernández-Sánchez
          </td>
          <td>2025-03-26</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b73933fff90d843d07b4b99040d504c9196ab75d" target='_blank'>
              Abstract 1452: DVL1 regulates global transcriptomic changes and associates with the transcription factor ETS1
              </a>
            </td>
          <td>
            Dalia Martinez-Marin, Monica Sharma, Jenna C van Wunnik, Flávia Sardela de Miranda, G. Boligala, Ella C. Jull, Grace C Stroman, Rachel L. Babcock, Kevin Pruitt
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Objectives/Goals: Large-scale tumor sequencing efforts have led to annotations of novel cancer hotspot mutations that may underlie driver or cooperative function. We have sought to define the molecular consequences of such hotspots associated with pediatric DICER1 syndrome cancers, with the ultimate goal of revealing novel targets that may inform new standards of care. Methods/Study Population: We have performed genomic analysis to identify tumor types (in TCGA and MSK-IMPACT patient data) for which mutations in the Dicer1 gene (encoding Dicer protein) emerge as the dominant signature of driver function. As Dicer is a critical RNA processing factor responsible for the generation of microRNAs, which are posttranscriptional gene regulatory molecules, we have modeled these mutations in human embryonic stem cells in order to study the direct effects on miRNAs and their target genes in an isogenic background. In addition to providing the required setting for unambiguous attribution of function to specific mutations, clonal human ES cells offer an opportunity for modeling of both developmental and cancer requirements associated with altered Dicer function. Results/Anticipated Results: Through generation of genomics and functional datasets from matched genotypes in Dicer mutated human ES cells, we have identified specific alterations in miRNAs and their effects on target genes. Unexpectedly, we found direct evidence for both loss of function and gain of function attributable to Dicer mutations. In addition, through integrated analysis of genomic data from tumor sequencing datasets and our human ES cell models, we have identified potential miRNA and target gene alterations that underlie tumorigenic potential, nominating gene candidates for targeted therapy in DICER1 syndrome. Direct mouse modeling of such candidate gene targets has revealed evidence for driver function of identified miRNA and their targets. Discussion/Significance of Impact: DICER1 syndrome cancers comprise a wide variety of rare pediatric tumor types. Presently, we still lack an effective standard of care. Furthermore, the previous lack of molecular profiling precluded targeted therapy opportunities. Our precise knock-in modeling of Dicer hotspots and deep profiling of relevant tumors now provide candidate targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/597fce8f39319f3d39a09a21bf59ed3021208522" target='_blank'>
              518 Modeling of cancer mutations found in pediatric DICER1 syndrome informs novel therapeutic targeting strategies
              </a>
            </td>
          <td>
            David Jee, Seungjae Lee, Dapeng Yang, Robert W. Rickert, Renfu Shang, Danwei Huangfu, Eric C. Lai
          </td>
          <td>2025-03-26</td>
          <td>Journal of Clinical and Translational Science</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Norcantharidin (NCTD), a bioactive compound derived from traditional Chinese medicine, has demonstrated promising anticancer activity against multiple malignancies, particularly hepatocellular carcinoma (HCC). However, its epigenetic regulatory mechanisms and associated transcriptional consequences remain poorly characterized. Methods In this study, we integrated biochemical assays with a panel of cellular analyses assessing cell viability, proliferation, colony formation, and migratory capacity to investigate NCTD’s therapeutic potential in HCC progression. Potential molecular targets of NCTD were systematically identified through integrated network pharmacology approaches. Chromatin immunoprecipitation quantitative PCR (ChIP-qPCR) was performed to quantify H3K27me3 enrichment level at the TOP2A locus in NCTD-treated HCC cells. Molecular docking simulations were employed to examine structural interactions between NCTD and EZH2 (enhancer of zeste homolog 2), while co-immunoprecipitation assays were further conducted to validate protein-protein interactions between EZH2 and protein phosphatase 1 (PP1). Results We identified topoisomerase IIα (TOP2A) as a critical molecular target mediating NCTD’s anti-HCC effects. Functional characterization revealed that NCTD significantly attenuated HCC cell proliferation and induced G2/M phase cell cycle arrest through disruption of the TOP2A-p53 signaling axis. Mechanistic investigations demonstrated that NCTD epigenetically suppresses TOP2A transcription via PRC2 (Polycomb Repressive Complex 2)-mediated deposition of the repressive histone mark H3K27me3 at the TOP2A promoter. Structural biology analyses confirmed direct binding of NCTD to EZH2 protein, consequently impairing PP1-mediated dephosphorylation and enhancing PRC2 complex stability. Conclusion Our findings establish that NCTD exerts anticancer effects in HCC through epigenetic silencing of TOP2A. This work not only elucidates a novel pharmacoepigenetic mechanism underlying NCTD’s antitumor activity but also provides translational rationale for developing PRC2-targeted therapeutic strategies in HCC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fe675c585b18a10405ecafd6e4610e88106014a" target='_blank'>
              Norcantharidin inhibits TOP2A expression via H3K27me3 mediated epigenetic regulation to alleviate the progression of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Ruibing Wu, H. Yuan, Yuehua Wang, Xianggang Gou, Wanhua Hou, Zhongzheng Zhou, Xinran Wang, Xiuling Deng, C. Wang, Haisheng Wang, Jia Yan
          </td>
          <td>2025-04-03</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b8af7271900611d5ca420224052b7797bd5bdd2" target='_blank'>
              Abstract 4331: Collaboration between p63 and ZNF148 in regulating squamous cell carcinoma progression
              </a>
            </td>
          <td>
            Artem Smirnov, E. Candi
          </td>
          <td>2025-04-21</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Background Colon cancer is a prevalent malignancy that significantly threatens human health. In recent years, mRNA cancer vaccines have demonstrated considerable potential and distinct advantages in colon cancer treatment. Thus, This study identifies CUL7, ENO2, and MPP2 as potential antigens for colon cancer mRNA vaccines. Through multi-omics analysis, we classify COAD into three immune subtypes (C1-C3) with distinct molecular and clinical features. Methods Data from TCGA and GEO databases were analyzed using bioinformatics tools. Prognostic indices were calculated with GEPIA2, and TIMER assessed antigen-presenting cell infiltration. Survival analysis was performed using Kaplan-Meier curves and Cox proportional hazards models. Immune subtypes were classified via non-negative matrix factorization (NMF) clustering, with k=3 determined by cophenetic correlation (0.92) and silhouette width (average = 0.85). Drug sensitivity, immune cell infiltration, and gene set variation were analyzed using R packages such as “pRRophetic,” CIBERSORT, and GSVA. Functional enrichment analysis was performed with GO, KEGG, and GSEA. Experimental validation included immunohistochemistry and RT-PCR to confirm gene expression. Results Analysis of TCGA-COAD data revealed copy number variants in 16,354 genes, with CUL7, ENO2, and MPP2 showing significant antigen-presenting cell infiltration and associations with overall survival (OS) and relapse-free survival (RFS). Based on molecular mechanisms, cellular features, and clinical characteristics, colon cancer was categorized into three immune subtypes (C1, C2, and C3) distinct from Thorsson’s pan-cancer subtypes (C1-C6) in pathway enrichment, with the C2 subtype exhibited significantly longer overall survival (OS) than C1 and C3 (median OS: C2 = 68 months vs. C1 = 42 months, C3 = 37 months; log-rank P < 0.001). The distribution of these immune subtypes showed disparities in immune patterns, and a correlation between key components and immune cells was observed. Prognostic correlation analysis indicated that the gray and turquoise modules were closely linked to colorectal cancer prognosis. Additionally, RT-PCR confirmed the association of CUL7, ENO2, and MPP2 expression levels with colon cancer. Conclusions CUL7, ENO2, and MPP2 were identified as potential antigens for colon cancer mRNA vaccines, with MPP2 showing particular immunological relevance. This study provides a foundation for mRNA vaccine development and patient stratification for vaccination in colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1a45b842cb272622dcc3d38dbf71c40a5757ce3" target='_blank'>
              Immunophenotyping of colon cancer for identification of potential antigens for colon cancer vaccines
              </a>
            </td>
          <td>
            Xuan Wang, Jingjiang Lai, Dawei Wang, Keyi Wei, Jing Yang
          </td>
          <td>2025-04-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/155bc347b38f90da8edc9334b8d5ad220e6809aa" target='_blank'>
              CRTC2 Forms Co-condensates with YTHDF2 that Enhance Translational Efficiency of m6A-Modified mRNAs to Drive Hepatocarcinogenesis and Lenvatinib Resistance.
              </a>
            </td>
          <td>
            Meixi Wang, Fangdi Zou, Shengxin Wang, Yichen Yang, Cong Xia, Lu Chen, Ben Liu, Lian Li, Mulin Jun Li, Haixin Li, Weijie Song, Ruifang Niu, Zhi-Yong Yuan, Jie Yang, Xiangchun Li, Kexin Chen, Zhiqiang Wu, Zeyun Mi
          </td>
          <td>2025-03-04</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Head and neck squamous cell carcinoma (HNSCC) often exhibits poor response rates to immune checkpoint inhibitor (ICI) therapies, largely owing to the intricate composition and spatial organization of immune cells within the tumor-immune microenvironment (TIME). The diversity of immune cell populations, their spatial relationships, and dynamic interactions significantly influence the immunosuppressive nature of the TIME, thereby limiting the efficacy of immunotherapy. To address these challenges and enhance the therapeutic potential of ICIs in HNSCC, a comprehensive analysis of the TIME is essential. Spatial transcriptomics (ST), a cutting-edge technology, enables high-resolution mapping of gene expression within the spatial context of the tumor, providing critical insights into the functional roles and interactions of immune cells in the TIME. This review highlights the importance of ST in uncovering the complexities of the TIME in HNSCC and proposes strategies for leveraging these insights to develop more effective immunotherapeutic approaches. By integrating spatial and molecular information, this review aims to pave the way for personalized and precision-based treatments in HNSCC, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef3b95effc04f0811082363795fb3dc9666b0f8" target='_blank'>
              Mapping immunotherapy potential: spatial transcriptomics in the unraveling of tumor-immune microenvironments in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Seo-Won Choi, Jeong Heon Kim, Jisu Hong, Minsu Kwon
          </td>
          <td>2025-04-08</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [5],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>