<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2026-02-23 10:03:43 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>48</td>
          <td>107</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>97</td>
          <td>141</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E. Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>129</td>
          <td>146</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>419</td>
          <td>141</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>352</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a circular, double-stranded DNA molecule distinct from chromosomal DNA, primarily found in cancer cells as a subtype of extrachromosomal circular DNA (eccDNA). Unlike eccDNAs found in normal cells, cancer-associated ecDNAs are large, clonal, and carry complete oncogenes. These ecDNAs are increasingly recognized as crucial drivers of cancer pathogenesis, contributing significantly to the evolution of tumor heterogeneity and the acquisition of therapeutic resistance through mechanisms such as high-level gene amplification and altered gene regulation. Historically, the understanding of these mobile genetic elements in cancer has been limited. This review synthesizes current knowledge on ecDNA’s structural and functional features, formation mechanisms and roles in cancer initiation, progression and therapeutic resistance. Moreover, we summarize five emerging therapeutic approaches that target ecDNA to inform future cancer research and precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb27a6296222e1300898cc02edbec7c0adb86fa2" target='_blank'>
              Extrachromosomal DNA and cancer: function, formation, and clinical implications
              </a>
            </td>
          <td>
            Yucai Wu, Rui Rui, Tai Tian, Jiaying Zheng, Shi-ming He, Liqun Zhou, Xuesong Li, Yanqing Gong
          </td>
          <td>2026-01-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e78908da394712537095f77c8b8136eb7ecf216" target='_blank'>
              Evolution of oncogene amplification across 86,000 cancer cell genomes
              </a>
            </td>
          <td>
            J. Lee, Sohrab Salehi, Matthew A. Myers, , Melissa A. Yao, Duaa H. Al-Rawi, , Eric G. Sun, Kerstin Thol, Seongmin Choi, Eliyahu Havasov, Asher Preska Steinberg, Michelle Wu, Nicole Rusk, Caitlin Timmons, Cheryl Zi Jin Phua, Stephen Martis, Neeman Mohibullah, P. Manoj, E. Redin, Á. Quintanal-Villalonga, P. Razavi, Sam Aparicio, Natasha Rekhtman, Viviane Tabar, Mark M. Awad, H. Yu, K. Yu, Andrea Ventura, Andrew Mcpherson, C. Rudin, Sohrab P. Shah
          </td>
          <td>2026-02-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6b22a73c1f46495b3ff8ba940a6c1a1abbc7b2c" target='_blank'>
              scAmp analyzes focal gene amplifications at single-cell resolution
              </a>
            </td>
          <td>
            Matthew G. Jones, Natasha E. Weiser, King L. Hung, Xiaowei Yan, Sangya Agarwal, J. Luebeck, Aditi Gnanasekar, B. Howitt, Ellis J Curtis, Kevin Yu, John C. Rose, Katerina Kraft, Valeh Valiollah Pour Amiri, Leena Satpathy, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2026-02-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Segmental copy-number gains are major contributors to human genetic variation and disease, but how these alterations arise remains incompletely understood. Here, based on the analyses of both experimental evolution and human disease genomes, we describe a general mechanism of segmental copy-number gain from a rearrangement process termed ‘breakage–replication/fusion’. The hallmark genomic feature of breakage–replication/fusion is adjacent parallel breakpoints: two or more rearrangement breakpoints derived from replication of a single ancestral DNA end. We show that adjacent parallel breakpoints are a widespread feature of DNA duplications in human disease genomes and experimental models of chromothripsis. In addition to adjacent parallel breakpoints, breakage–replication/fusion also explains two other patterns of complex rearrangements with unclear provenance: chains of short (≤1 kb) insertions and high-level amplification consisting of inverted segments. Together, these findings revise the mechanistic model for chromothripsis and provide a new conceptual framework for understanding the origin of segmental DNA duplication during genome evolution. This paper introduces breakage–replication/fusion, a genomic rearrangement process underpinning three patterns of copy-number gains found in cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/943c5640094a2cf860368d879c898eeb29d9cb6e" target='_blank'>
              A breakage–replication/fusion process explains complex rearrangements and segmental DNA amplification
              </a>
            </td>
          <td>
            Cheng-Zhong Zhang, Carlos Mendez-Dorantes, Kathleen H. Burns, David Pellman
          </td>
          <td>2026-01-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Canine osteosarcoma (OS) is an aggressive bone cancer with remarkable similarity to human OS. Despite its value as a comparative oncology model, the drivers of genomic instability remain poorly defined, hindering cross-species insights into disease progression. We applied long-read PacBio HiFi whole genome sequencing to matched tumor and blood samples from four purebred dogs to generate high-resolution maps of somatic structural variants (SVs). Tumor genomes showed extensive rearrangement complexity, with satellite repeat regions, particularly SAT1_CF, enriched near somatic SV breakpoints and occurring within genomic contexts marked by focal hypomethylation. We also identified multiple extrachromosomal DNA (ecDNA) elements carrying oncogenes indicating ecDNA-mediated oncogene amplification. Together, these data provide an integrated genomic and epigenomic view of canine OS, implicating satellite repeats as recurrent substrates for rearrangement and ecDNA as a prominent mode of genome remodeling. This work advances our understanding of OS genome dynamics and establishes a comparative framework for investigating repeat-driven instability and oncogene amplification across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5300af19b1916e9abe3db71ff1fa34ffceb5e24d" target='_blank'>
              Satellite DNA fragility accompanies complex genome rearrangements and ecDNA oncogene amplification in canine osteosarcomas
              </a>
            </td>
          <td>
            F. Yilmaz, Wonyoung Kong, Sabriya A. Syed, Francis H. O’Neill, Jody T. Lombardi, P. K. Ng, Ching C. Lau, Charles Lee
          </td>
          <td>2025-12-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Genomic alterations are a hallmark of cancer, and extrachromosomal DNA (ecDNA) has emerged as a key source of oncogene selection, tumor growth, and drug resistance. The intratumor heterogeneity and clonal selection of ecDNA is, however, poorly understood. In this study, we pursue a computational approach that leverages allelic imbalance and outlier expression from standard single-cell RNA sequencing (scRNA-seq) to deconvolve the tumor heterogeneity of ecDNA at the single-cell level (ecSingle). Using this approach, we identify oncogene-carrying ecDNAs in tumor samples at the single-cell level, which we validate using genome sequencing. Moreover, we show the superiority of using single-molecule long-read sequencing in resolving ecDNA. ecDNAs displayed extensive intratumor heterogeneity, including subclonal oncogene-carrying ecDNA in primary tumor cells that segregate with distinct transcriptional cell states. Importantly, we show that a rare ecDNA+ clone in the primary tumor can expand to form dominant clones in relapse tumors. Our study introduces a novel approach to studying ecDNA at the single-cell level, enabling both clonal evolution and transcription cell state analysis. We apply this approach to cancer samples to gain deeper insights into the role of ecDNA in intratumor heterogeneity and cellular plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7df14db8ad5c3596e9b413772fed54a722b4bb2b" target='_blank'>
              Resolving clonal evolution and selection of extrachromosomal DNA at single-cell resolution
              </a>
            </td>
          <td>
            J. Hendriksen, A. Locallo, B. Schlotmann, Francisco Germán Rodríguez González, Jane Skjøth-Rasmussen, C. Yde, D. Nørøxe, H. S. Poulsen, U. Lassen, J. Weischenfeldt
          </td>
          <td>2026-01-29</td>
          <td>Genome Biology</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/794649b9036f93693e77ad613d6e85889d3eff72" target='_blank'>
              Targeting extrachromosomal DNA in human cancers.
              </a>
            </td>
          <td>
            I. Wong, Hyerim Yi, Bruno Melillo, Benjamin F. Cravatt, Howard Y. Chang, P. Mischel
          </td>
          <td>2026-02-03</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="


 Extrachromosomal DNA (ecDNA) amplifications are key drivers of human cancers. Here, we show that ecDNAs are major platforms for generating and amplifying oncogene fusion transcripts across diverse cancer types. By integrating analysis of whole genome and transcriptome sequences from 1,825 tumor samples and cancer cell lines of a wide variety of tissue types, we reveal that ecDNAs have the highest rate of oncogene fusion events of any copy number alteration. Focusing on the most common ecDNA fusion hotspot, we find that fusion of the 5' end of the long noncoding RNA gene, PVT1–with exon 1 joined to diverse 3' partners–confers increased RNA stability, and enhances MYC-dependent transcription and cancer cell survival. These results demonstrate that ecDNA fosters genome instability and frequent oncogene fusion formation under strong selection in cancer.



 Shu Zhang, Hyerim Yi, Jason Swinderman, Yanbo Wang, Vishnupriya Kanakaviti, King L. Hung, Ivy Tsz-Lo. Wong, Suhas Srinivasan, Ellis J. Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G. Jones, Jens Luebeck, Chris Bailey, Yanding Zhao, Julia A. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Kabir S. Mahajan, Frances Liang, Mariam Jamal-Hanjani, Dean W. Felsher, Luke A. Gilbert, Vineet Bafna, Paul S. Mischel, Howard Y. Chang. EcDNA-derived structural variants: A common pathway to oncogenic fusion transcript amplification in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A007.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b68513e53c43f7bee512369c94fed2f8b73ac179" target='_blank'>
              Abstract A007: EcDNA-derived structural variants: A common pathway to oncogenic fusion transcript amplification in cancer
              </a>
            </td>
          <td>
            Shu Zhang, Hyerim Yi, J. Swinderman, Yanbo Wang, Vishnupriya Kanakaviti, King L. Hung, I. Wong, Suhas Srinivasan, Ellis J Curtis, Aarohi Bhargava-Shah, Rui Li, Matthew G. Jones, J. Luebeck, Chris Bailey, Yanding Zhao, J. Belk, Katerina Kraft, Quanming Shi, Xiaowei Yan, Simon K. Pritchard, Kabir S. Mahajan, Frances Liang, M. Jamal-Hanjani, D. Felsher, Luke A. Gilbert, V. Bafna, P. Mischel, Howard Y. Chang
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="The human genome is not confined to chromosomes. Extrachromosomal circular DNA—eccDNA and its megabase-scale counterpart ecDNA—is a persistent, transcriptionally active, and mutationally accelerated compartment present in normal and neoplastic cells. Yet every stage of genomic analysis, from DNA extraction to variant annotation, is engineered for linear, diploid templates. This review argues that circular DNA mutations are not rare events; they are systematically erased by methodologies that mistake the map for the territory. We synthesize evidence that circular DNA exhibits a mutational spectrum fundamentally distinct from chromosomal DNA: microhomology-driven indels, complex structural rearrangements within single molecules, and interchromosomal chimeric fusions that generate novel oncogenic reading frames. Its mutation rate is accelerated by rolling-circle replication, suppressed homologous recombination, and torsional stress—quantified as μ_eff = μ_bp × CN × f_rep, exceeding chromosomal rates by two orders of magnitude. Experimental isolation and detection methods each carry specific biases that distort mutational discovery. Computational reconstruction requires graph-based assembly and copy-number-normalized burden metrics. Validation by inverse PCR, mutation-specific FISH, and single-cell sequencing reveals that ecDNA mutations exhibit non-Mendelian inheritance yet drive transient transcriptional dominance. Major barriers include the absence of a circular reference genome, indistinguishable replication errors, and zero clinical standardization. Nevertheless, ecDNA amplifications occur in 14% of cancers, confer prognosis independent of chromosomal status, and serve as privileged sites for resistance mutations. eccDNA burden tracks cellular age and genotoxic stress. Emerging therapies targeting ecDNA segregation or transcription are plausible but lack specificity. The central conclusion is not that circular DNA matters—it is that the genome is not what we thought it was.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/829f7f2d91e8a0a5e3b416bb5b2bd0074f23ae03" target='_blank'>
              The Hidden Mutational Landscape of Extrachromosomal Circular DNA
              </a>
            </td>
          <td>
            Lali Latsabidze
          </td>
          <td>2026-02-12</td>
          <td>Longevity Horizon</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is an aggressive malignancy associated with single-nucleotide variants in VHL, PBRM1, and SETD2. However, the role of structural variation (SV)—which have broader genomic impacts—and 3D genome architecture in ccRCC remains inadequately understood. Here, we reported a comprehensive molecular characterization. Through multi-omics analysis, we identify novel SV-associated oncogenic targets and reveal multidimensional 3D genome reorganization during ccRCC progression. We elucidate the dynamic interplay between SV and 3D chromatin architecture, demonstrating how structural rearrangements drive oncogenic dysregulation through 3D genome reorganization. Notably, an unrecognized pathogenic enhancer hijacking event is discovered and experimentally validated, leading to constitutive activation of the proto-oncogene SEMA5B. Furthermore, we developed a machine learning-based prognostic framework using enhancer hijacking signatures. Collectively, this work establishes a valuable resource for ccRCC research by elucidating how SVs and 3D genome reorganization collectively drive oncogenesis, and translates these findings into a clinically applicable prognostic tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f8e6235c583e0191c138b2862bedc511ab3f80" target='_blank'>
              Structural variation drives enhancer hijacking via 3D genome disruption in ccRCC
              </a>
            </td>
          <td>
            Yu Dong, Wenjiao Xia, Zitong Yang, Liangliang Ren, Hongru Wang, Yiyang Zhou, Qinchen Li, Zhi Chen, Zhinan Xia, Yichun Zheng, Feifan Wang, Ning He, Bing Cheng, Dongmei Ma, Wei Shao, Wei Guo, Shuwen Wang, Ziqiao Liu, Junxiao Shen, Yiming Qi, Xuke Gong, Juan Jin, Bo Xie, Guixin Zhu, Cheng Zhang
          </td>
          <td>2026-01-06</td>
          <td>NPJ Digital Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chromosome 3p (chr3p) is frequently deleted in multiple cancers, indicating the presence of shared tumor suppressors. Analysis of genomic alterations in 33 different cancer types implicates the deletion or deleterious mutations of SET-domain-containing 2 (SETD2) at chr3p21 in significantly facilitating the formation of isochromosomes, consisting of two identical mirror-imaged arms, thereby promoting genomic instability conducive to large-scale chromosomal rearrangements and rapid cancer genome evolution. Fracturing of dicentric isochromosomes during cell division is pervasive and follows the dynamic fragmentation pattern of solids under impulse. Across cancers, isochromosomes form most frequently on chr8 to amplify the MYC-containing q-arm and chr17 to delete the TP53-containing p-arm. In the most aggressive uveal melanoma (UVM) subtype, chr3 deletion also includes MITF, a critical melanocyte differentiation and survival factor, and co-occurs with chr8q amplification. We demonstrate that MITF is a master transcriptional regulator of GNAQ/GNA11, mutated in 90% of UVM patients, and the associated synthetic-lethal genes identified by recent CRISPR screening studies. MITF maintains MAPK and calcium homeostasis in UVM, and its deletion is thus accidental, creating an early crisis during oncogenesis. We further show that MITF, MYC, and GNAQ/GNA11 form coupled regulatory feedback loops in the melanocyte lineage, and MITF deletion in UVM creates acute dependency on MYC-mediated rescue via chr8q amplification, often as a consequence of isochromosome formation. The discovered feedback loops predict both overall and relapse-free patient survival within the most aggressive UVM subtype, explain sensitivity to therapeutic gene perturbations, and inform effective combinatorial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f31378489089eb1239e625480e1522a21f193cf" target='_blank'>
              Chromosome 3p deletion leads to extensive genomic alterations in diverse cancers and confers synthetic lethality in uveal melanoma
              </a>
            </td>
          <td>
            Mitchell C. Cutler, P. B. Howland, M. Hejna, Jun S. Song
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are circular DNA fragments frequently found in human cancers, where they amplify oncogenes, drive tumor heterogeneity, and promote therapy resistance and poor prognosis. Despite their prevalence, how ecDNAs interact with the immune system remains poorly understood. Here, we show that the cytosolic DNA sensor cGAS detects ecDNA fragments in the cytoplasm and activates the innate immune response. cGAS and STING are frequently silenced in ecDNA+ tumors through promoter hypermethylation. Restoring cGAS or STING in human and murine ecDNA+ cancer cells reactivates innate immune signaling and selectively suppresses ecDNA+ tumor growth in an immunocompetent mouse model. Using two ecDNA biogenesis models, we show that the cGAS-STING pathway restricts de novo ecDNA formation. Together, our findings identify innate immune sensing as a natural barrier to ecDNA-driven oncogenesis and establish cGAS-STING reactivation as a therapeutic strategy for ecDNA+ cancers. Highlights The cGAS-STING pathway is frequently silenced in ecDNA+ tumors Restoration of cGAS in ecDNA+ cells activates innate immune responses cGAS expression suppresses ecDNA+ tumor growth in vivo The cGAS-STING pathway restricts de novo ecDNA biogenesis">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/663b6039e119f140e6b2d7e0ab0731409522972e" target='_blank'>
              Innate immune sensing via the cGAS-STING pathway restricts extrachromosomal DNA–driven tumorigenesis
              </a>
            </td>
          <td>
            Tuo Li, Qing-Lin Yang, Kailiang Qiao, Anli Zhang, Chenglong Sun, Huocong Huang, P. Mischel, Sihan Wu, Zhijian J. Chen
          </td>
          <td>2026-01-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="TFE3 translocation renal cell carcinoma (tRCC), an aggressive kidney cancer driven by TFE3 gene fusions, is frequently misdiagnosed owing to morphologic overlap with other kidney cancer subtypes. Conventional liquid biopsy assays that detect tumor DNA via somatic mutations or copy number alterations are unsuitable for tRCC since it often lacks recurrent genetic alterations and because fusion breakpoints are highly variable between patients. We reasoned that epigenomic profiling could more effectively detect tRCC because the driver fusion constitutes an oncogenic transcription factor that alters gene regulation. By defining a TFE3-driven epigenomic signature in tRCC cell lines and detecting it in patient plasma using ChIP-seq, we distinguished tRCC from clear-cell RCC (AUC = 0.86) and samples of individuals without evidence of cancer (AUC = 0.92) at low tumor fractions (<1%). This work establishes a framework for noninvasive epigenomic detection, diagnosis, and monitoring of tRCC, with implications for other mutationally quiet, fusion-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181d702745222ebeaadeb7b01c3a274dc1851478" target='_blank'>
              Cell-free DNA epigenomic profiling enables noninvasive detection and monitoring of translocation renal cell carcinoma
              </a>
            </td>
          <td>
            S. Garinet, Karl Semaan, Jiao Li, Ze Zhang, Prathyusha Konda, A. Sadagopan, J. Canniff, Noa Phillips, K. Klega, Medha Pandey, H. Savignano, Matthew P Davidsohn, Kevin Lyons, Alessandro Medda, Prateek Khanna, M. Achom, B. Fortunato, Rashad Nawfal, Razane El Hajj Chehade, J. O’Toole, J. Horst, D. Freeman, Rachel Trowbridge, C. Chau, William D. Figg, Jacob E. Berchuck, Brian D. Crompton, J-H. Seo, T. Choueiri, M. Freedman, S. Baca, S. Viswanathan
          </td>
          <td>2026-02-02</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac603fef64d1ae37e58513b3a4b599d750808d42" target='_blank'>
              XAI-ecDNA: A Proof-of-Concept Framework for Explainable Multi-Modal Extrachromosomal DNA Detection Using Synthetic FISH and Genomics
              </a>
            </td>
          <td>
            Asim Manzoor, Minahal Amin
          </td>
          <td>2026-01-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c21032e04b711532f2c1af652faef7052c5a06e" target='_blank'>
              PanCNV-Explorer: Deciphering copy number alterations across human cancers
              </a>
            </td>
          <td>
            Nadine S. Kurz, Kevin Kornrumpf, Anna-Rosa Krüger, Jürgen Dönitz
          </td>
          <td>2026-01-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63674f265b1ac623907383e45f60b5724b528b86" target='_blank'>
              Recurrent Escape from Osimertinib-Induced Senescence Promotes Genomic Instability Associated with Therapeutic Resistance
              </a>
            </td>
          <td>
            H. McDaid, Nazia S Jamil, Qualia Hooker, Nadjet Cornejal, Dean Hosgood
          </td>
          <td>2026-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de5dab2b2a49c5a9b6f6dd158d618f0ef49f45bd" target='_blank'>
              Cell Ploidy Modulates Nucleolar Size via Dosage of Nucleolar Organizing Regions
              </a>
            </td>
          <td>
            Tianhao Bi, Varuni Rastogi, Maged Zeineldin, , Yi Dong, Mulin Li, Tatianna C. Larman
          </td>
          <td>2026-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bce9b91a05b2c305ecf72c3bf59327984c56dba6" target='_blank'>
              Functional inactivation of the telomerase chaperone TCAB1 primes cells for the activation of ALT in osteosarcoma
              </a>
            </td>
          <td>
            Joshua Keegan, Sydney Sorbello, Joakin O. Mori, Shugo Muratani, I. Leshchiner, Christopher M. Heaphy, R. L. Flynn
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Eukaryotic genomes are abundant in satellite DNA (satDNA): large blocks of tandemly-repeated sequences that accumulate in heterochromatic genome regions. SatDNAs are dynamic in their genomic location and abundance across species. Some satDNAs overlap essential genome regions such as centromeres and telomeres, but even pericentromeric satDNA can have effects on phenotypes, raising questions about their functional significance. However, it remains unclear whether these effects depend on satDNA sequence, copy number, higher-order structural organization, or genomic context. The highly repetitive nature of satDNA arrays has long hindered detailed genomic and genetic analyses. Recent advances in long-read sequencing now facilitate both the detailed characterization of satDNA structure and the development of more targeted approaches to genetic analysis. Here we present a sequential CRISPR/Cas9-based strategy to make mutations in satDNA arrays and demonstrate its utility using an autosomal pericentromeric satDNA in Drosophila melanogaster called Responder (Rsp). Rsp is the target of a sperm-killing male meiotic driver, Segregation Distorter (SD), where sensitivity to sperm killing positively correlates with Rsp copy number. Using our CRISPR/Cas9 approach, we generated an allelic series of Rsp deletion and expansion variants in two genetic backgrounds and examined their effects on spermatogenesis. Our approach produced precise satDNA variants efficiently, with minimal detectable off-target effects. The resulting mutations affect sensitivity to SD that scale with Rsp copy number. This work establishes a new framework for experimentally dissecting satDNA function and provides insights into the evolutionary and functional roles of satDNA in genome organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f2bac6b9fbac768b087852174f053e8622b039f" target='_blank'>
              Targeted editing of pericentromeric satellite DNA alters sensitivity to meiotic drive
              </a>
            </td>
          <td>
            D. Eickbush, J. Rahmat, M. Lindsay, J. Bowers, N. Fuda, A. Larracuente
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Mutations in epigenetic regulators are common in bladder cancer, yet their impact on therapeutic responses remains unclear. Here, we identify that loss-of-function mutations in KDM6A, a histone demethylase altered in about 26% of advanced bladder cancers, are associated with poor survival after cisplatin chemotherapy, whereas they correlate with improved outcomes with anti-PD-1 therapy. Using CRISPR-Cas9-engineered murine and human bladder cancer models, we show that KDM6A deficiency increases formation of extrachromosomal circular DNA carrying chemoresistance loci, promoting cisplatin resistance. In parallel, KDM6A loss impairs DNA repair and rewires tumor metabolism, reducing glycolysis and lactate output. This metabolic shift diminishes histone lactylation in regulatory T cells, suppressing immunoregulatory genes and limiting expansion of PD-1hi regulatory T cells. Collectively, our findings establish KDM6A mutation as a key regulator of therapeutic responses, providing a foundation for its use in guiding precision therapy in advanced bladder cancer. Genes encoding key epigenetic regulators, including Lysine Demethylase 6A (KDM6A), are frequently mutated in bladder cancer. Here, the authors show that loss of KDM6A promotes formation of extrachromosomal circular DNA (eccDNA), genomic instability, and metabolic reprogramming, driving resistance to cisplatin chemotherapy while simultaneously enhancing sensitivity to immune checkpoint inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9139b0ede19d5b7c3ea40131744831c88cd696ed" target='_blank'>
              Loss of KDM6A-mediated genomic instability and metabolic reprogramming regulates response to therapeutic perturbations in bladder cancer
              </a>
            </td>
          <td>
            Pratishtha Singh, R. D’Rozario, Bidisha Chakraborty, Swadhin Meher, D. Raychaudhuri, Aminah J. Tannir, Yang Li, Anurag Majumdar, Jessalyn Hawkins, Yun Xiong, Philip L. Lorenzi, Padmanee Sharma, K. Akdemir, P. Pilié, Abhinav K. Jain, Byron H Lee, Sangeeta Goswami
          </td>
          <td>2026-01-07</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="Simple Summary Long interspersed element-1 sequences are normally silenced in healthy cells but frequently reactivated in cancer, where they contribute to genomic instability and transcriptional dysregulation. However, the determinants of their transcriptional heterogeneity in non-small cell lung cancer remain poorly defined. In this study, we systematically characterized locus-specific LINE-1 transcript expression across a large panel of non-small cell lung cancer cell lines to determine how genomic context shapes retrotransposon activity. We show that LINE-1 transcription is dominated by evolutionarily young elements and follows conserved chromosomal patterns across tumor subtypes, with the highest activity observed in lung squamous cell carcinoma. Importantly, LINE-1 expression is anchored by a small number of conserved genomic hotspot loci that remain active across tumor backgrounds and are located near cancer-relevant genes. These findings provide a genome-scale framework for understanding retrotransposon regulation in cancer and highlight conserved LINE-1 loci as potential markers of genome instability and regulatory vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a76a2793974308abd948a4e791ee9224ae6f4f7f" target='_blank'>
              LINE-1 Transcript Heterogeneity in Non-Small Cell Lung Cancers Is Driven by Host Genomic Context and Conserved Functional Hotspots
              </a>
            </td>
          <td>
            Yingshan Wang, Kenneth S. Ramos
          </td>
          <td>2026-01-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Elevated levels of extrachromosomal DNAs (ecDNAs) are associated with poor prognoses of many cancer types. These large circular DNAs typically harbour oncogenes and regulatory elements which, together with high levels of ecDNA transcription, confer a growth advantage to cancer cells. Replication of ecDNAs, followed by their unequal distribution at mitosis, further promotes rapid cancer evolution. In contrast to ecDNAs, the role of circular DNA by-products from V(D)J recombination in cancer development has largely been overlooked. Developing lymphocytes generate millions of excised signal circles (ESCs) each day through gene rearrangement at the immunoglobulin and T-cell receptor loci. Despite their similar size to ecDNAs, ESCs were long assumed to be inert and lost during cell division. However, it is now known that ESCs potently trigger genome instability when complexed with recombinase proteins. Not only this, but new data show that just like ecDNAs, ESCs replicate and persist, with high levels strongly correlating with poor prognosis of B-cell precursor acute lymphoblastic leukaemia (BCP-ALL). Despite these striking similarities, the properties of ESCs and ecDNAs are seldom linked. Here, we provide the first comparative review of ecDNAs and ESCs, and highlight the reasons why these molecules are more closely related than once assumed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec100c92fb04b0426668a2656e80f3ac4a05ad94" target='_blank'>
              The newfound relationship between extrachromosomal DNAs and excised signal circles.
              </a>
            </td>
          <td>
            Dylan Casey, Zeqian Gao, J. Boyes
          </td>
          <td>2026-01-03</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Introduction Cancer cell proliferation occurs within the context of persistent genomic instability. In this review, we propose the RS–CCD–CIN axis as a systems-level framework in which replication stress (RS), cell cycle deregulation (CCD) and chromosomal instability (CIN) form an interdependent triad that shapes tumour evolution. This axis represents a constrained metastable state in which genomic instability is tolerated and buffered. The objective of this review is to synthesize the current understanding of how the RS–CCD–CIN axis contributes to tumour heterogeneity, adaptability and therapy response. Discussion Evidence indicates that RS, CCD and CIN operate as a dynamic, interconnected network rather than as independent processes. Replication stress induces DNA damage and mutagenesis, while partial checkpoint disruption permits cells with unresolved lesions to proliferate. Chromosomal instability generates both structural and numerical alterations, contributing to intratumoural heterogeneity. Together, these processes facilitate adaptation to environmental and therapeutic pressures. Extrachromosomal DNA, micronuclei formation and cytosolic DNA signalling, including the cGAS–STING pathway, connect genomic instability to adaptive responses and immune modulation. Single-cell and spatial profiling reveal temporal and spatial variability in RS, CCD and CIN states, highlighting the limitations of static biomarkers. Therapeutically, targeting individual components often yields limited durability, whereas approaches that simultaneously perturb multiple aspects of the RS–CCD–CIN axis may improve clinical outcomes. Conclusions This review highlights the RS–CCD–CIN axis as a fragile and metastable architecture that supports cancer evolution, while also being susceptible to collapse. A deeper understanding of this interconnected framework may inform the development of therapeutic strategies and enhance the management of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/294582a14e94664c9f90d67cf0b0db4402660bea" target='_blank'>
              A system-level metastable model of cancer evolution: integrating replication stress, cell cycle deregulation and chromosomal instability
              </a>
            </td>
          <td>
            Dongjing Ma, Jinbin Wang, Y. M. Chen, Ling Yao, Qianya Wei, Jianjun Wu
          </td>
          <td>2026-01-28</td>
          <td>Annals of Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Despite the many advances in single cell genomics, detecting structural rearrangements in single cells, particularly error-free sister-chromatid exchanges, remains challenging. Here we describe sci-L3-Strand-seq, a combinatorial indexing method with linear amplification for DNA template strand sequencing that cost-effectively scales to millions of single cells, as a platform for mapping mitotic crossover (CO) and resulting genome instability events. We provide a computational framework to fully leverage the throughput, as well as the relatively sparse but multifaceted genotype information within each cell that includes strandedness, digital counting of copy numbers, and haplotype-aware chromosome segmentation, to systematically distinguish seven possible types of mitotic CO outcomes. We showcase the power of sci-L3-Strand-seq by quantifying the rates of error-free and mutational COs in thousands of cells, enabling us to explore enrichment patterns of genomic and epigenomic features. The throughput of sci-L3-Strand-seq also gave us the ability to measure subtle phenotypes, opening the door for future large mutational screens. Furthermore, mapping clonal lineages provided insights into the temporal order of certain genome instability events, showcasing the potential to dissect cancer evolution. Altogether, we show the wide applicability of sci-L3-Strand-seq to the study of DNA repair and structural variations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74a3596ca3c9b197876145ec4548ef0ebb22dbce" target='_blank'>
              High-throughput mapping of spontaneous mitotic crossover and genome instability events with sci-L3-Strand-seq
              </a>
            </td>
          <td>
            Peter Chovanec, Trevor Ridgley, Yi Yin
          </td>
          <td>2026-02-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74e725a85a73539cf9bc922a83eb56481b485413" target='_blank'>
              Pan-cancer silencer transcription atlas reveals leukemia silencer hijacking FOXP1 to enhance MYC oncogene
              </a>
            </td>
          <td>
            M. Guo, Liutao Chen, Shengcheng Deng, Jingkai Zhang, Zhijie Hu, Weiwen Wang, Songyang Zhou, Yuanyan Xiong
          </td>
          <td>2026-01-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/450603a806f69ebb4194f3b7b3ab19ed0a5b5513" target='_blank'>
              A holocentric pangenome links karyotype evolution to meiotic recombination
              </a>
            </td>
          <td>
            Meng Zhang, Stefan Steckenborn, Marco Castellani, Laia Marín-Gual, A. S. Câmara, Laura A. Robledillo, Letícia Maria-Parteka, N. Sargheini, Magdalena Marek, Eduardo Chacón-Madrigal, Andrés Mauricio Gatica Arias, L. Felix, William W. Thomas, B. Huettel, A. Pedrosa‐Harand, John T. Lovell, André L. L. Vanzela, Aurora Ruiz-Herrera, A. Marques
          </td>
          <td>2026-01-21</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="Human artificial chromosomes (HACs) are inherited through cell divisions alongside natural chromosomes, serving as tools for interrogating chromosomal elements and as vectors for large genetic cargoes (1–5). Despite recent progress (6–8), large (i.e., multiple Mb) HACs have not been reported. Further, HAC formation via epigenetic seeding of centromeric chromatin currently requires prior engineering of recipient human cells (6–8), hampering their potential deployment in many useful cell types. Here, we designed, built, and delivered to human cells a 2 Mb HAC construct that is ∼3 times larger than the prior generation. We also report a robust epigenetic centromere seeding approach that initiates immediately upon delivery to the human cell cytoplasm and bypasses genetic engineering of target cells. The HACs are then faithfully inherited in the absence of selection. Thus, formation of functional centromeric chromatin in the same cell cycle of HAC delivery drives high efficiency HAC formation. Teaser A system for efficient protein-only centromere seeding permits visualizing the first step of HAC formation">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/755b604be820a14cd765634bf7256cd13a7ee21b" target='_blank'>
              Protein-only centromeric chromatin assembly streamlines human artificial chromosome formation
              </a>
            </td>
          <td>
            Gabriel J. Birchak, P. K. Allu, Prakriti Kashyap, G. K. Anbalagan, Shu-Cheng Chuang, Glennis A. Logsdon, Daniel G. Gibson, John I. Glass, Ben E. Black
          </td>
          <td>2026-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/638cde0ab64b6b068ce716defb37f6b2cb4ab7b9" target='_blank'>
              Unraveling the molecular mechanisms underlying spontaneous multipolar mitosis through CIN-seq
              </a>
            </td>
          <td>
            Pin-Rui Su, Ting-Chun Chou, M. T. López-Cascales, Junfeng Hunag, Tsai-Ying Chen, Sjoukje van Wieren, Changwei Li, Cecile Beerens, Li You, J. Storteboom, M. Chien
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Transposable elements (TEs) are a major source of genomic variability, yet their relation with other mutational processes and DNA repair in cancers remains poorly understood. Here we combined deep sequencing approaches (RNAseq, whole exome sequencing, and targeted TE-flank sequencing) with computational analyses to investigate the transcriptional activity of active human L1 and Alu elements across 526 experimental and 2488 TCGA cancer samples. By quantifying somatic TE insertions in 40 experimental pairs of cancer and matched normal tissues, we found that TE insertional activity (roughly 20 insertions per sample for each class of TEs) correlates with L1 transcription, is increased in cancers and has substantial intersample variability. TE insertions also correlated with activation of non-homologous end joining, mismatch and nucleotide excision repair pathways, and with transcription of TERT and APOBEC3B genes. Based on highly correlated genes, we created an expression signature reflecting TE insertional activity (AUC 0.819-0.903). On larger experimental and literature tumor cohorts, the signature strongly correlated with the activation levels of most of DNA repair pathways except those leading to ATM checkpoint activation and cell cycle arrest. It was also associated with many genome instability markers (chimeric genes, tumor mutation burden, gene copy number variation, loss of heterozygosity), but showed reduced values in cancers with microsatellite instability. Finally, the signature was associated with worse overall survival in pancreatic cancer (HR 5.9) and lesser effects in stomach, lung, and cervical cancers. These results shed light on the interplay of TE activities, DNA repair, and genome instability in human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18b44c54459c606eb131e85fc4b773aef30ba060" target='_blank'>
              Insertional activity of human Alu and L1 retrotransposons is associated with DNA repair pathways and genome instability in cancer.
              </a>
            </td>
          <td>
            M. Suntsova, Aleksander Modestov, Elizaveta Rabushko, Nikolai Komarov, Ivan Gaziev, Anastasia Novosadskaya, Anastasiya Barysionak, G. Zakharova, N. Shaban, Anna Khristichenko, Anna A. Emelianova, Marianna A. Zolotovskaia, E. Poddubskaya, A. Seryakov, Anton Buzdin
          </td>
          <td>2026-02-07</td>
          <td>Computers in biology and medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Phenotypic plasticity is a prominent cancer feature that contributes to metastatic potential and resistance to therapy across multiple cancer types. Cancer cell state transitions have been attributed to transcriptional programs, such as the AP1/TEAD-regulated gene network driving the mesenchymal-like (MES) phenotype. In addition, during dissemination, tumor cells are subjected to variable loads of physical mechanical pressure and constriction across transited tissue, which are thought to impact nuclear molecular crowding. How the interplay between mechanical pressure, global 3D nuclear architecture and transcriptional programs contributes to MES identity and metastatic adaptation remains unclear. Using cutaneous melanoma as a model for early dissemination, we integrate in vitro and in vivo epigenomic profiling with nanoscale imaging of cell lines and patient samples to investigate chromatin organization features underlying the MES phenotype. We find that in MES cells, CTCF is relocated from domain boundaries to regulatory regions of EMT-like genes, leading to reduced insulation, extended topological associated domains (TADs) and increased inter-domain contacts, and de novo formation of chromatin hubs. This conformational rewiring, along with loss of heterochromatin, supports nuclear deformability during invasion and dissemination. Conversely, physical constriction of melanocytic cells induces MES-like chromatin features —including CTCF repositioning and heterochromatin loss— and promotes metastasis in vivo. Similarly, pharmacological inhibition of the heterochromatin mark H3K9me3 triggers MES characteristics and increases invasiveness. These results demonstrate that metastatic competency involves both epigenetic and structural nuclear reprogramming, enabling shifts in gene networks and physical adaptability. Our findings reveal mechanistic links between nuclear architecture and aggressive tumor behavior, identifying potential biomarkers and therapeutic targets to intercept metastatic progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92c8bf70d1df18ff53fcb785d7213684e545662a" target='_blank'>
              Chromatin architecture and physical constriction cooperate in phenotype switching and cancer cell dissemination
              </a>
            </td>
          <td>
            Pietro Berico, Cody L Dunton, L. Almassalha, Amanda Flores-Yanke, Karla I Medina, Nicolas Acosta, Tara Muijlwijk, Catherine Do, Soobeom Lee, S. Edmiston, D. Corcoran, Allison Reiner, Caroline E. Kostrzewa, Kathy Dorsey, Milad Ibrahim, Ronglai Shen, Nancy E. Thomas, Amanda W. Lund, Ata S. Moshiri, Iman Osman, I. Aifantis, Jane A. Skok, Vadim Backman, Eva Hernando
          </td>
          <td>2026-02-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="In a recent study published in Nature, Cosenza et al. 1 introduced MAGIC (machine-learning-assisted genomics and imaging convergence), a versatile platform that photolabels cells with micronuclei and other nuclear atypia, enabling direct quanti ﬁ cation of de novo chromosomal abnormality (CA) formation and ongoing chromosomal instability (CIN) at single-cell resolution. Using this framework, the authors show how spontaneous and induced lesions from pre-mitotic DNA damage and mitotic errors drive distinct CIN trajectories, with TP53 inactivation markedly amplifying CA burden and complexity, highlighting how CIN emergence and propagation prior to clonal expansion may inform tumour initiation, progression and the identi ﬁ cation of actionable biomarkers and therapeutic vulnerabilities. CIN is the ongoing process by which cells acquire new CAs through repeated chromosome missegregation and structural rearrangements, altering chromosome number and structure across divisions (Fig. 1a). CAs are pervasive across cancers and typically affect more genes than substitutions or small insertion/ deletion alterations, fuelling clonal diversi ﬁ cation and tumour adaptation. 2 – 5 Accurate chromosome segregation and genome integrity rely on coordinated processes such as DNA replication and repair, telomere maintenance, spindle assembly and mitotic exit. 2 When these are compromised and cells bypass senescence and apoptosis, concurrent pre-mitotic and mitotic defects drive ongoing CIN. 2,5 However, most CIN">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0c5eb4ff0382eff914e4d728cb5754b2a5b73b7" target='_blank'>
              It’s a kind of MAGIC: uncovering the origins of chromosomal instability
              </a>
            </td>
          <td>
            D. Gómez-Peregrina, César Serrano
          </td>
          <td>2026-02-05</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e01dc9e9620e1364bb485bdf2ea0135eedca2399" target='_blank'>
              Chromosomal instability and chromosome 17p loss drive convergent NDE1 synthetic lethality in metastatic cancer cells
              </a>
            </td>
          <td>
            Caroline M. Teddy, Jacob P. Hoj, Julia D Caci, Min Lu, Shane T. Killarney, Tal Ben-Yishay, U. Ben-David, Kris C. Wood
          </td>
          <td>2026-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6dd597d404f6624bed3a43a3ecd045b834d7d8b" target='_blank'>
              Integration of an anellovirus genome in the SKNO-1 acute myeloid leukemia cell line
              </a>
            </td>
          <td>
            Na Cui, S. Goya, E. Piliper, Alexander L. Greninger
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Hepatocellular carcinoma (HCC) is the most common primary liver malignancy worldwide and is associated with a high mortality rate. The unique genetic features and structural properties of extrachromosomal circular DNA (eccDNA) offer a promising approach for early cancer diagnosis.


OBJECTIVE
This study aims to investigate the potential of eccDNA as a diagnostic marker by identifying its distinguishing characteristics using a two-dimensional cell culture model.


METHODS
EccDNA was isolated from the supernatant of HepG2 and THLE-2 cell cultures. Long-read sequencing and bioinformatic analysis were employed to detect and characterize eccDNA. The size distribution, chromosomal origin, and genomic annotation of eccDNA from both cell types were compared. PCR was performed to validate DNA fragments at the junctions of the closed-circular forms.


RESULTS
The total number of cell-free eccDNA derived from HepG2 cells averaged 2,669,673 bases, encompassing 2,542 sequences, while THLE-2 cells averaged 857,718 bases with 975 sequences. Size distribution analysis revealed that most cell-free eccDNA from both cell types ranged from 351 to 600 bp. Bioinformatic analysis revealed that approximately 80% of eccDNAs corresponded to gene coding regions. Ten eccDNAs were the most frequently detected in HepG2 cells, with four eccDNAs harboring CDC27P11, RAC1P3, LOC112268123, and LOC124902279 genes successfully validated and uniquely detected in HepG2 cells, suggesting their potential as biomarkers for HCC diagnosis.


CONCLUSION
Distinct eccDNA types were identified in HepG2 cells, which may serve as promising biomarkers for HCC diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2857a0fc338d1571175ba6c4c210b5ca1dafdfd9" target='_blank'>
              Identification of Extrachromosomal Circular DNA Isolated from Cell Culture Supernatant and Its Potential Applications for Hepatocellular Carcinoma Diagnosis.
              </a>
            </td>
          <td>
            Suchitraporn Sukthaworn, Suchanuch Ondee, Opas Choksupmanee, Suparat Taengchaiyaphum, Prasert Yodsawat, Ponsit Sathapondecha
          </td>
          <td>2026-01-21</td>
          <td>Asian Pacific journal of cancer prevention : APJCP</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has emerged as a pivotal contributor to cancer progression, facilitating oncogene amplification, dysregulated gene expression, and tumor heterogeneity. Despite its significance in cancer, the interplay between eccDNA and key epigenetic regulators such as EZH2 remains largely unexplored. In this study, we systematically investigate the correlation between Tazemetostat, a highly selective EZH2 inhibitor, and alterations in the eccDNA landscape and transcriptional programs in ovarian cancer. Through integrated profiling using Circle-seq and RNA sequencing, we demonstrate that EZH2 inhibition is associated with markedly reprogrammed eccDNA dynamics. Furthermore, multi-omics integration identified that 67 genes exhibited concordant changes in both eccDNA abundance and transcript expression. Subsequent analyses also pinpointed 11 genes as putative effectors of drug response. Notably, spatial single-cell transcriptomics identified SFRP1 as the most consistently reactivated tumor suppressor across eccDNA, bulk expression, and spatial datasets, based on predefined statistical and biological criteria, by Tazemetostat. Moreover, SFRP1 was one of the genes that varied the most within cancer-associated fibroblast populations, exhibiting distinct spatial expression patterns. Taken together, this study establishes the first potential evidence that EZH2 inhibition may reprogram eccDNA dynamics to potentially restore SFRP1 tumor suppressor expression in ovarian cancer. By integrating multi-omics and spatial single-cell transcriptomics, we uncovered a novel epigenetic–eccDNA axis that may contribute to oncogenic plasticity and therapeutic resistance. This could result in a paradigm shift in targeting eccDNA-driven malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6e68c77ce796a11b7676543278dbc14c7ad88f7" target='_blank'>
              EZH2 Inhibition Restores Tumor Suppressor SFRP1 Activity by Reprogramming Extrachromosomal Circular DNA Dynamics in Ovarian Cancer
              </a>
            </td>
          <td>
            Tao Han, Qingya Yan, Yaqi Zhang, Yu Gan, Kaifan Li, Liping Guan, Changqin Jing, Ciqing Yang, Pengfei Li, Bo Gao, Xiang Zhou, Qian Hao
          </td>
          <td>2026-02-15</td>
          <td>Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Recent advances in genomic technologies have greatly enhanced our understanding of genotype–phenotype relationships and improved diagnostic of genetic diseases. However, the dissection of complex structural variants remains challenging due to the limitations of current methods in resolving their breakpoint and interpreting phenotypes involving multiple disrupted genes. In this study, we demonstrate how an integrative approach—combining cytological, genomic, and transcriptomic methods—enables the detection, structural and functional characterization of a complex structural variants affecting the MBD5, USP34, and XPO1 genes. Our findings underscore the utility of the Exo-C, a modified chromosome conformation capture technique, in resolving complex rearrangements. We also report, for the first time, a composite neurodevelopmental phenotype resulting from the combined effects of MBD5-associated intellectual disability and 2p15p16.1 microdeletion syndromes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c987c3067a72472e130e25f49fdb579eae34de80" target='_blank'>
              Zooming into rearranged genome: applying pipeline of cytological, genomic, and transcriptomic methods for structural variant interpretation
              </a>
            </td>
          <td>
            M. Gridina, T. Lagunov, P. Belokopytova, N. Torgunakov, A. Nurislamov, D. Yurchenko, Zhanna G Markova, T. Markova, Yana Stepanchuk, Galina Koksharova, Pavel Orlov, Anna Subbotovskaia, O. Ryzhkova, Nadezhda V. Shilova, Veniamin Fishman
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/308ec9e744b73dd3ecd51f7b2385a3a46667c2cb" target='_blank'>
              Pangenome analysis reveals the evolutionary dynamics of repeat-based holocentromeres
              </a>
            </td>
          <td>
            Piotr Włodzimierz, Estela Perez-Roman, A. S. Câmara, Laura A. Robledillo, Gokilavani Thangavel, Meng Zhang, Jacob González Isa, L. M. Parteka, Marco Castellani, B. Huettel, André L. L. Vanzela, Ian R. Henderson, Alexandros Bousios, A. Marques
          </td>
          <td>2026-01-18</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4322917e9d3faaba1bbb348dcf524837265ecd27" target='_blank'>
              Genomic and Evolutionary Determinants of Two-hit Frequencies in Tumor Suppressor Genes
              </a>
            </td>
          <td>
            Nivedita Mukherjee, Radhakrishnan Sabarinathan
          </td>
          <td>2026-02-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4a41cb33c7197e06376843089b1b62d0d1ef0e9" target='_blank'>
              Intrachromosomal insertion as a diagnostic challenge: a hidden structural rearrangement causing recurrent duplication and deletion
              </a>
            </td>
          <td>
            Rie Kawamura, Yui Shichiri, Hideki Suzuki, Yuri Murase, Yuki Naru, T. Hara, Ayana Tsuboi, Hanae Satano, Eiji Sugihara, Hiroki Kurahashi
          </td>
          <td>2026-01-25</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1248ad827da125e00e0f24cf372ada5d8bfee2" target='_blank'>
              DNA replication errors drive genome-wide small inverted triplication dynamics
              </a>
            </td>
          <td>
            Yi Lei, Yu Zhou, Haitao Sun, Hang Yuan, X. Pei, Jessica D. Hess, Yao Yan, Zunsong Hu, Mian Zhou, Zhaohui Gu, Li Zheng, Xiwei Wu, Binghui Shen
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Neuroblastoma RAS viral oncogene homolog (NRAS) mutations occur in 20-30% of cutaneous melanomas, defining a distinct molecular subtype. While NRAS mutations drive unique gene expression programs, locus-specific transcriptional regulation has not been systematically explored. This study characterized coordinated transcriptional upregulation in NRAS-mutant melanoma, focusing on chromosome 1p13.2. RNA-seq and mutation data for 472 melanoma samples from The Cancer Genome Atlas Skin Cutaneous Melanoma were analyzed. Differential expression, coexpression assessment, and copy number variation (CNV) correlation were performed for 45 protein-coding genes within 1p13.2. Findings were validated in MSK-IMPACT 2021 (n = 696). In NRAS-mutant melanomas, 56% of 1p13.2 genes were upregulated (false discovery rate < 0.05) versus 12% genome-wide (P < 0.001), including TRIM33, AMPD1, GNAI3, and SLC25A24. Upregulation was NRAS-specific: 84% showed higher expression versus BRAF-mutant tumors. Six of the 10 most NRAS-correlated genes localized to 1p13.2. CNV gains showed dose-dependent effects (gains: 1.8-3.2 fold; amplifications: 3.5-5.4 fold), with strong correlations (AMPD1 ρ = 0.68, GNAI3 ρ = 0.62, TRIM33 ρ = 0.58; P < 0.001). MSK-IMPACT validation confirmed 1p13.2 upregulation. As these findings are based on transcriptomic and copy number analyses, additional protein-level and functional studies across independent cohorts are required to confirm biological significance. NRAS-mutant melanomas harbor a coordinated transcriptional program within 1p13.2, driven by CNV gains. This locus contains genes with potential druggability, offering new avenues for combinatorial targeting alongside mitogen-activated protein kinase pathway inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb61b6e16a4770e24198eb7ffe671bddbc211884" target='_blank'>
              Chromosome 1p13.2 gene cluster upregulation in neuroblastoma RAS viral oncogene homolog pathogenic variant melanoma: a potential biomarker axis.
              </a>
            </td>
          <td>
            Sophie M. Bilik, Olivia M Burke, S. Elman
          </td>
          <td>2026-01-29</td>
          <td>Melanoma research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="CTNNB1, the gene encoding β-catenin, is a frequent target for oncogenic mutations activating the canonical Wnt signaling pathway, typically through missense mutations within a degron hotspot motif in exon 3. Here, we combine saturation genome editing with a fluorescent reporter assay to quantify signaling phenotypes for all 342 possible missense mutations in the mutation hotspot. Our data define the genetic requirements for β-catenin degron function, refine the consensus motif for substrate recognition by β-TRCP and reveal diverse levels of signal activation among known driver mutations. Tumorigenesis in different human tissues involves selection for CTNNB1 mutations spanning distinct ranges of predicted activity. In hepatocellular carcinoma, mutation effect scores distinguish two tumor subclasses with different levels of β-catenin signaling, and weaker mutations predict greater immune cell infiltration in the tumor microenvironment. Our work provides a resource to understand mutational diversity within a pan-cancer mutation hotspot, with potential implications for targeted therapy. Saturation genome editing of a cancer mutation hotspot in CTNNB1, the gene encoding β-catenin, reveals a gradient of effects of missense mutations on Wnt signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0951b8c87bfe861a396eafd5a171689184ab2592" target='_blank'>
              Mutational scanning reveals oncogenic CTNNB1 mutations have diverse effects on signaling
              </a>
            </td>
          <td>
            Anagha Krishna, A. Meynert, K. S. Dolt, Martijn J. E. Kelder, A. Mesropian, Ailith Ewing, C.M. Brouwers, J. Claassens, M. Linssen, Shahida Sheraz, G. Taylor, Philippe Gautier, Anna Ferrer-Vaquer, Graeme R Grimes, Hannes Becher, Ryan Silk, A. Gris-Oliver, R. Pinyol, C. Semple, T. Kendall, Tom Bird, A. Hadjantonakis, Joseph A. Marsh, J. Llovet, P. Hohenstein, Andrew J Wood, Derya D. Ozdemir
          </td>
          <td>2026-02-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Significance Replication stress (RS) is a major driver of genome instability in cancer. We discovered that the CHAMP1 complex, a set of proteins that assembles heterochromatin, plays a critical role in relieving RS. By depositing repressive histone marks, the complex stabilizes replication forks and prevents their degradation. Disruption of CHAMP1 complex creates unique vulnerabilities in cancer cells, including those cells with the (ALT) mechanism of telomere maintenance or with CCNE1 amplification. Our findings reveal a mechanism by which heterochromatin structure can safeguard DNA replication and highlight CHAMP1 as a potential therapeutic target for tumors with high levels of RS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8461ad6cad0781da64419f6c03309084a55c23fc" target='_blank'>
              CHAMP1 complex promotes heterochromatin assembly and reduces replication stress
              </a>
            </td>
          <td>
            Feng Li, Amira Elbakry, Felix Y. Zhou, Tianpeng Zhang, Ramya Ravindranathan, Huy Nguyen, Aleem Syed, Lifang Sun, S. Mukkavalli, Roger A. Greenberg, Alan D. D’Andrea
          </td>
          <td>2026-01-02</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Significance The classic Luria–Delbrück experiment showed how mutations arise in single-copy bacterial chromosomes. Yet many genetic systems, including plasmids and the genomes of mitochondria, plastids, and cancer cells, exist in multiple copies. In such systems, polyploidy generates opposing forces: Extra copies increase the supply of new mutations, while random segregation slows their fixation. Therefore, it remains unclear whether polyploidy promotes or constrains evolution. Using plasmids as a case study, we combine theory, modeling, experiments, and a bioinformatic extension of the Luria–Delbrück framework to show that mutation rates increase logarithmically with copy number. Our results settle a long-standing debate, demonstrating that plasmids are hotspots of bacterial evolution and revealing the rules by which polyploidy sets the tempo of evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecbf44e01949a84dc5dd9b45780af8eafce6a78d" target='_blank'>
              Plasmid mutation rates scale with copy number
              </a>
            </td>
          <td>
            Paula Ramiro-Martínez, Ignacio de Quinto, Laura Jaraba-Soto, V. Lanza, Cristina Herencias-Rodríguez, Adrián González Casanova, R. Peña-Miller, J. Rodríguez-Beltrán
          </td>
          <td>2026-01-22</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract Changes in the copy number of large genomic regions, termed copy number variations (CNVs), contribute to important phenotypes. CNVs are readily identified using conventional approaches when present in a large fraction of the cell population. However, CNVs in only a few genomes are often overlooked but important; if beneficial, a de novo CNV that arises in a single genome can expand during selection to create a population of cells with novel characteristics. While single cell methods for studying de novo CNVs are increasing, we continue to lack information about CNV dynamics in rapidly evolving microbial populations. Here, we investigated de novo CNVs in the genome of the Plasmodium parasite that causes human malaria. The highly AT-rich Plasmodium falciparum genome readily accumulates CNVs that facilitate rapid adaptation. We employed low-input genomics and specialized computational tools to evaluate the impact of sub-lethal stress on the de novo CNV rate. We observed a significant increase in genome-wide de novo CNVs following treatment with an antimalarial compound that inhibits replication. De novo CNVs encompassed genes from various cellular pathways participating in human infection. This snapshot of CNV dynamics emphasizes the connection between replication stress, DNA repair, and CNV generation in this important microbial pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cc90f4b9d9a1317ece4f68d2ac843d4fc5a32e4" target='_blank'>
              Replication stress increases de novo CNVs across the malaria parasite genome
              </a>
            </td>
          <td>
            Noah Brown, A. Luniewski, Xuanxuan Yu, Michelle Warthan, Shiwei Liu, Julia Zulawinska, Syed Ahmad, Nadia Prasad, Molly Congdon, Webster Santos, Feifei Xiao, J. Guler
          </td>
          <td>2026-01-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/566c090308e26a25e5654df687dc08aebd75d6cb" target='_blank'>
              EZH2 Inhibition Reshapes 3D Chromatin Architecture to Induce Immunogenic Phenotype in Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Sana Parveen, Ramesh Adhinaveni, Kun Fang, L. Choppavarapu, Meijun Du, Gustavo Leone, N. De Sarkar, Victor X. Jin, Hui-Zi Chen
          </td>
          <td>2026-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Highlights What are the main findings? The reduction in the nuclear ESCRT CHMP7 drives telomeric aberrations. The reduction in the nuclear-envelope-assembly-related factors LEM2 and BAF1 drives telomeric aberrations. What are the implications of the main findings? The ESCRT machinery is interlinked with telomere organization. Fragilization of telomeres can be an effect of mutations in ESCRT-encoding genes. Abstract During open mitosis, reassembly of the nuclear envelope requires the coordinated recruitment of the ESCRT machinery, initiated by the chromatin-associated factor BAF1 and the nuclear-envelope-associated factor LEM2. Because telomeres are enriched at the reforming envelope, we investigated whether ESCRT factors contribute to telomere integrity. Reduction in the pivotal nuclear ESCRT factor CHMP7 caused DNA damage, heterochromatin disorganization, and telomere defects, including sister telomere associations and telomere free ends. Extending this analysis, we found that additional ESCRT components, including TSG101, VPS28, CHMP4B, and the ESCRT-associated factor AKTIP/Ft1, also contribute to telomere integrity, although with different strengths. Genetic interaction analyses suggest that CHMP7 converges in a common pathway with CHMP4B and AKTIP/Ft1, while it functions in parallel routes to TNKS1, a telomere-specific regulator of the shelterin TRF1. More genetic analyses indicated that BAF1 and LEM2 contribute to safeguarding of telomeres during nuclear envelope reassembly. Because defects in nuclear envelope dynamics and chromatin–membrane coupling are hallmarks of disorders associated with nuclear deformation and fragility, including aging and cancer, our findings contribute a new angle into these conditions and suggest potential targets for selectively modulating telomere maintenance pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9538b39736d1f85f26227cc54ae42097f17929f" target='_blank'>
              Reduced CHMP7 Expression Compromises Telomere Integrity in Mammalian Cells
              </a>
            </td>
          <td>
            Romina Burla, Mattia La Torre, Klizia Maccaroni, S. Tacconi, L. Dini, Isabella Saggio
          </td>
          <td>2026-01-29</td>
          <td>Cells</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Neuronal cell adhesion molecule (NRCAM) has been implicated in glioma biology through splicing alterations reported by prior studies. However, the relative contributions of genomic and epigenetic mechanisms to NRCAM dysregulation in gliomas remain unclear. Methods We analyzed data from The Cancer Genome Atlas (TCGA) Lower-Grade Glioma (LGG) cohort using UCSC Xena and cBioPortal. Copy number variation (CNV), DNA methylation (Illumina HumanMethylation450 BeadChip arrays), and mutation profiles were assessed for NRCAM. Kaplan–Meier survival analyses were performed with Xena, stratifying patients by copy number status and methylation state. Correlations between mutation burden and fraction of genome altered (FGA) were evaluated using Spearman and Pearson methods. Results Somatic mutations in NRCAM were rare in the TCGA-LGG cohort. In contrast, CNV and methylation changes were frequent and clinically relevant. Copy number gains at the NRCAM locus were associated with significantly shorter overall survival, while higher methylation of NRCAM correlated with improved survival outcomes. NRCAM mutation count did not show a linear correlation with FGA, suggesting these alterations are largely independent of overall genomic instability. The findings highlight copy number imbalance and epigenetic regulation as predominant mechanisms of NRCAM dysregulation. Conclusion NRCAM is recurrently dysregulated in lower-grade gliomas through CNVs and DNA methylation, both of which stratify patient survival. Together with previously reported splicing deregulation, these data suggest that NRCAM functions as a multilayered regulator of glioma progression. NRCAM methylation may represent a potential prognostic biomarker, while therapeutic modulation of NRCAM warrants further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a2630cdb2c0d21a56b05bf8738d1e26c2abc4bc" target='_blank'>
              Multi-Layered Dysregulation of NRCAM in Lower-Grade Gliomas: Insights From TCGA Copy Number and Epigenetic Analyses
              </a>
            </td>
          <td>
            S. Lehrer, P. Rheinstein
          </td>
          <td>2026-01-01</td>
          <td>Brain Tumor Research and Treatment</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Osteosarcoma (OS) exhibits substantial genomic complexity and inter-patient heterogeneity, necessitating longitudinal, patient-matched analyses to understand acquired features of tumor evolution. However, most published OS data is limited to primary tumor samples, limiting insight into patient-specific resistance mechanisms. To address this, we characterized the genomic landscape of paired primary and metastatic tumor samples from dogs with spontaneous OS. Whole-genome and single-cell RNA sequencing reveal mutation and gene expression profiles that are predominantly organized by patient identity. Mutational burden and pathway alterations such as those involving PI3K, NOTCH, TP53, MAPK, RAS and epigenetic regulation differ between primary and metastatic samples. Variants present in tumor tissue are readily detectable in paired cfDNA samples, demonstrating the utility of this assay for identifying tumor-specific alterations associated with treatment resistance. Analysis of bulk RNA-seq data to estimate cell-type composition shows greater immune cell representation in metastases, underscoring the importance of immune signaling pathways in OS. These findings exemplify the presence of patient-specific alterations in genomic architecture over the course of tumor progression, linking CNV amplification, pathway reprogramming, and immune evasion in metastatic OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d967dfc80d383c8de7c87cc500998e454e27b0e7" target='_blank'>
              Multi-omic Longitudinal Analysis of Canine Osteosarcoma Identifies Inter-Patient Heterogeneity and Immune Enrichment in Metastatic Lesions
              </a>
            </td>
          <td>
            Christopher Husted, Kerstin Seidel, Tanya T. Karagiannis, Cornelia Peterson, Kate Megquier, Diane Genereux, Jillian M. Richmond, Elinor K. Karlsson, Dave Shulman, Brian Crompton, Cheryl A London, Heather L. Gardner
          </td>
          <td>2026-01-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Genomic instability is a hallmark of cancer and contributes to disease progression through various mechanisms. DNA copy number variation (CNV) is one of the most significant consequences causing a phenomenon called aneuploidy. Aneuploidy is characterized by an imbalanced genome and is associated with lethal progression in prostate cancer. Currently, sequencing-based CNV analysis is widely used to identify cancer-specific aneuploidy that contribute to tumorigenesis, immune evasion, and disease progression. Performing low-pass whole genome sequencing (LP-WGS) on DNA extracted from bulk tissues or cell cultures, is a convenient approach to use for detecting aneuploidy and focal CNVs. However, conventional bulk LP-WGS methods average genomic signals across thousands of cells, often overlooking tumor heterogeneity and small populations with distinct CNV patterns. To address this limitation, we optimized a microfluid nanowell-based system to analyze aneuploidy patterns at a single-cell level using disassociated tissues and isolated nuclei from prostate samples. This pipeline is compatible with both preclinical models and clinical specimens and can be integrated with other multi-omics platforms to generate multimodal datasets for basic and translational research. By leveraging high-throughput single-cell sequencing technologies, our approach enables direct DNA CNV measurement with high sensitivity at a single-cell resolution. Ultimately, this approach aims to improve the resolution of CNV detection, enhance molecular stratification and enable multimodal data integration for better understanding of clinical prostate cancer samples.



 Chloe Springer, Faith Kim, Kathryn Echandía-Monroe, Aditi Shirke, Gustavo Guitierrez-Cruz, Madeline Wong, Kun-Lin Ho, Duanduan Ma, Elise G. DeArment, Amina Ali, William D. Figg, Gregory Chesnut, Matthew G. Vander Heiden, Xiaofeng A. Su. Characterizing aneuploid tumor heterogeneity in prostate cancer through single-cell DNA sequencing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A060.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6642b028922ed7c8904563bae6731f81ce2d7a38" target='_blank'>
              Abstract A060: Characterizing aneuploid tumor heterogeneity in prostate cancer through single-cell DNA sequencing
              </a>
            </td>
          <td>
            Chloe Springer, Faith Kim, Kathryn Echandía-Monroe, Aditi Shirke, Gustavo Guitierrez-Cruz, Madeline M. Wong, Kun-Lin Ho, Duanduan Ma, Elise G. DeArment, Amina Ali, W. Figg, Gregory T. Chesnut, M. V. Vander Heiden, Xiaofeng A. Su
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Background/Aim Lipomas are benign tumors of adipocytic origin. The most frequent chromosome rearrangement in lipomas involves the high-mobility group AT-hook 2 gene (HMGA2), which maps to 12q14. We investigated two lipomas showing rearrangements of chromosomal bands 8q22 and 12q14 in their karyotypes, with the aim of identifying the gene products affected by these aberrations. Materials and Methods The two lipomas were selected because their karyotype showed an 8;12-rearrangement with the very same breakpoint position. The cases were investigated using RNA sequencing, reverse transcription polymerase chain reaction (RT-PCR), Sanger sequencing techniques, Fluorescence in situ hybridization (FISH), and array comparative genomic hybridization (aCGH). Results RNA sequencing and RT-PCR of the two lipomas showed the presence of the HMGA2::CIBAR1-DT chimera. Conclusions The HMGA2::CIBAR1-DT fusion, identified here for the first time, is a recurrent transcript in lipomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e35b7cf2f12e853b8a43f5d8374c50a16fbddd11" target='_blank'>
              Identification of the HMGA2::CIBAR1-DT fusion transcript in two lipomas with chromosomal rearrangements involving chromosomes 8 and 12
              </a>
            </td>
          <td>
            M. Brunetti, K. Andersen, I. Lobmaier, F. Micci
          </td>
          <td>2026-01-06</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c352c29f84fbf8771435e1fc6bcfeba36df1f644" target='_blank'>
              BAP1 loss impairs Non-Homologous End Joining DNA repair promoting genomic instability
              </a>
            </td>
          <td>
            Sabrina Caporali, Carla Schuy, Melanie Rall-Scharpf, Alessio Butera, Lisa Wiesmüller, Ivano Amelio
          </td>
          <td>2026-02-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract DNA is most often found in its canonical B-form double-helical structure, but can also adopt alternative conformations, known as non-B DNA structures. Numerous non-B structures have been characterized, including G-quadruplexes, i-motifs, Z-DNA, hairpins, cruciforms, slipped structures, R-loops, and H-DNA. Non-B DNA motifs are enriched in functional regions, including near transcription start and end sites, topologically associated domains, and replication origins, suggesting their importance in gene regulation, genome organization, and replication. However, these structures are intrinsically prone to error-generating processing, leading to genomic instability and hence have been implicated in the development of human diseases. Here, we discuss recent advances in understanding the biological roles of non-B DNA structures and their contribution to genomic instability in somatic and germline contexts. We highlight how they promote replication stress, transcription stalling, and DNA breaks, resulting in the formation of mutational hotspots. Emerging technologies have enabled the detailed mapping of previously challenging repetitive regions that harbor potential non-B DNA-forming sequences, and are poised to unravel additional contributions in human disease and evolution. Furthermore, we explore the dual role of non-B DNA as a driver of genetic variation that facilitates evolutionary adaptation and as a source of mutations that contribute to tissue dysfunction and aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f81eab064c5f9cbe45e223c9b4a2bde11b74f6f7" target='_blank'>
              Non-B DNA structures and their contributions to genetic diversity, aging, and disease
              </a>
            </td>
          <td>
            Eleftherios Bochalis, Irene Dereki, Guliang Wang, Argyro Sgourou, Karen M. Vasquez, IIias Georgakopoulos-Soares
          </td>
          <td>2026-02-05</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Whole-genome doubling (WGD) is a frequent yet poorly understood genomic event linked to adverse clinical outcomes.
 Multiple distinct mechanisms can lead to tetraploidization and aneuploidy, including defects in the mitotic checkpoint, chromosome cohesion, and disruptions of microtubules, spindle assembly, or centromere function. In addition, cell-cell fusion events can also result in polyploidization. Polyploid cells often exhibit chromosomal instability, which contributes to increased intratumoral heterogeneity and accelerates the evolution of the cancer genome. Although chromosomal instability and the presence of tetraploid or hyperdiploid cancer cells have been linked to treatment resistance and enhanced phenotypic diversity, the molecular mechanisms driving these associations remain poorly understood.
 We began by assessing the frequency of WGD in breast cancer across tumor subtypes and stages of disease progression. To do this, we performed WGD calling or reanalyzed publicly available genomic sequencing datasets with annotated WGD status. Analysis of three datasets revealed that WGD occurs more frequently in HER2+ and triple-negative breast cancers (TNBC) compared to hormone receptor-positive tumors. Furthermore, WGD frequency was consistently higher in distant metastases than in primary tumors.
 We then established through homofusion several murine mammary tumor models of WGD of TNBC to investigate how WGD drives tumor evolution. We observed that WGD increases transcriptomic and epigenetic heterogeneity and identified the survivin inhibitor YM155 as a selective suppressor of WGD+ tumors.
 Importantly, WGD promoted immune evasion by enabling escape from CD8+ T cell-mediated immune responses, rendering WGD+ tumors more sensitive to anti-PD-L1 therapy. Through single-cell profiling, we found that WGD+ cancer cells exhibit impaired antigen presentation, driven in part by a reduced response to IFN-γ and epigenetic silencing of MHC class I transcriptional regulators via the PRC2 complex.
 Treatment with a PRC2 inhibitor preferentially inhibited WGD+ tumor growth, restored antigen presentation, and enhanced CD8+ T cell infiltration. These findings suggest that combined inhibition of PRC2 and PD-1 represents a promising therapeutic approach for WGD+ breast cancers.



 Whole genome doubling, Breast Cancer, single cell profiling, immune escape



 P. Foidart, Z. Li, X. Can, M. Seehawer, E. Rojas-Jimenez, P. Baldominos, J. Nishida, T. Bui, B. Diciaccio, S. Parvin, M. Goyette, P. Yan, X. Qiu, R. Li, Y. Jiang, Y. Xie, X. Huang, L. E. Stevens, P. Cejas, L. Mangiante, C. I. Sotomayor Vivas, K. Houlahan, T. Scales, I. S. Harris, C. Curtis, A. G. Letai, H. W. Long, J. Agudo, K. Polyak. Whole-genome doubling promotes immune evasion in TNBC by epigenetically silencing antigen presentation pathways through PRC2 activity [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2025; 2025 Dec 9-12; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2026;32(4 Suppl):Abstract nr PS2-11-27.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08753d581f8856e6e0eafe884e8eb564b13b727d" target='_blank'>
              Abstract PS2-11-27: Whole-genome doubling promotes immune evasion in TNBC by epigenetically silencing antigen presentation pathways through PRC2 activity
              </a>
            </td>
          <td>
            P. Foidart, , X. Can, M. Seehawer, E. Rojas-Jimenez, P. Baldominos, J. Nishida, T. Bui, B. Diciaccio, S. Parvin, M. Goyette, P. Yan, , R. Li, Y. Jiang, Y. Xie, X. Huang, L. Stevens, P. Cejas, L. Mangiante, C. Vivas, K. Houlahan, T. Scales, I. Harris, C. Curtis, A. Letai, H. W. Long, J. Agudo, K. Polyak
          </td>
          <td>2026-02-17</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Extrachromosomal circular DNA has emerged as a pivotal factor in tumor biology, contributing to intratumor heterogeneity, oncogene amplification, and tumor evolution. Despite its relevance, the presence and molecular characteristics of plasma-derived cell-free extrachromosomal circular DNA (eccDNA) in cancer patients remain insufficiently explored. In this study, we profiled plasma-derived cell-free eccDNA from a multi-cancer cohort consisting of 413 cancer patients and 239 healthy individuals. We analyzed eccDNA fragment size distributions using a patient-derived xenograft (PDX) mouse model and identified fragment size features to distinguish tumor-derived eccDNA from non-tumor-derived eccDNA. We further performed in silico fragment size selection and developed a gene-based annotation method to characterize the gene content carried by cell-free eccDNA across different cancer types. By utilizing these cell-free eccDNA signatures, we developed ScanTecc (screening cancer types with cell-free eccDNA), a machine learning-based approach for cancer detection and tissue-of-origin classification. The classification performance of ScanTecc was further evaluated at the individual sample level using multiple machine learning classifiers, including adaptive boosting and logistic regression. We observed a significantly higher abundance and longer fragment lengths of eccDNA in cancer patients’ plasma. Analysis of the PDX mouse model revealed a distinct fragment size threshold of approximately 1,000 base pairs that effectively differentiates tumor-derived eccDNA from non-tumor-derived counterparts. Following fragment size selection, gene-based annotation of large-sized eccDNA revealed cancer type–specific enrichment of tumor-associated genes. Leveraging these features, ScanTecc achieved an overall AUC of 0.92 for distinguishing cancer patients from healthy individuals, with consistently high performance across disease stages, including stage I (AUC = 0.92) and stage IV (AUC = 0.93). ScanTecc also enabled accurate tissue-of-origin classification and achieved an overall AUC of 0.79 in identifying specific cancer types, with AUC values ranging from 0.70 for gastric cancer to 0.81 for ovarian cancer. Our study establishes a multi-cancer plasma cell-free eccDNA landscape and introduces a non-invasive cancer screening framework based on cell-free eccDNA features, highlighting the potential of plasma cell-free eccDNA for early cancer detection and tumor classification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd2c300c02b9f286a9e46bf7c8db5ce03368c9f" target='_blank'>
              Detection of primary cancer types via fragment size selection in circulating cell-free extrachromosomal circular DNA
              </a>
            </td>
          <td>
            Jingwen Fang, Songwen Luo, Shouzhen Li, Yehong Xu, Jing Wang, Benjie Shan, Mingjun Hu, Qiaoni Yu, Wen Zhang, Ke Liu, Yunying Shao, Jiaxuan Yang, Youyang Zhou, Guangtao Xu, Xinfeng Yao, Ruoming Sun, Mengyuan Zhang, Kun Li, Xihai Xu, Yongliang Zhang, Zhihong Zhang, Xinghua Han, Yueyin Pan, Chuang Guo, Kun Qu
          </td>
          <td>2026-01-06</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The use of programmable nucleases has transformed genome editing and functional genomics. Clustered regularly interspaced palindromic repeats (CRISPR)/CRISPR-associated 9 (Cas9) was developed such that targeted genomic lesions [usually DNA double-stranded breaks (DSBs)] could be introduced in vivo with ease and precision. In the presence of homology donors, these lesions facilitate high-efficiency precise genome editing (PGE) via homology-directed repair (HDR) pathways. Because DSBs can lead to genomic instability, however, a large amount of effort has been invested in methodologies (e.g., base editors) that only require nicking the chromosomal DNA on one strand. Indeed, we have demonstrated in human cells that oligodeoxynucleotide (ODN)-mediated PGE using nickase variants of Cas9 can proceed by at least two HDR subpathways termed synthesis-dependent strand annealing (SDSA) and single-stranded DNA incorporation (ssDI). Which pathway is utilized is determined by which chromosomal strand (sense or anti-sense/Watson or Crick) is nicked and by the strandedness (sense or anti-sense/Watson or Crick) of the donor ODN. While the mechanism of mammalian SDSA is moderately well understood, that of ssDI is not. To gain genetic insight into ssDI, we carried out a genome-wide CRISPR knockout screen to identify those genes which, when absent, enable increased ssDI. This screen identified the protein lysine methyl transferase (PKMT) Su(var)3–9, enhancer-of-zeste and trithorax (SET) domain bifurcated histone lysine methyltransferase 1 (SETDB1):activating transcription factor 7-interacting protein (ATF7IP) heterodimer and the downstream human silencing hub (HUSH) complex as strong negative regulators of ssDI. Consistent with their well-known biological effects, the negative regulation of ssDI by SETDB1/ATF7IP and HUSH was specific for transgenic reporters and for a HUSH-regulated single-copy gene, but was not observed at other (non-HUSH regulated) single-copy endogenous loci. In toto, these experiments underscore the profound impact that chromatin modifiers - and by extension, chromatin structure – have on PGE outcomes. Specifically, we have identified SETDB1/ATF7IP and the HUSH complex as major negative regulators of the HDR subpathway, ssDI, when the target is a transgene. These experiments are a proof-of-principle that chromatin can act as a potent barrier to genetic recombination and they strongly support the feasibility of extending similar chromatin modulating strategies to enhance PGE efficiency at endogenous single-copy loci.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827654ec916ba3c373ade5ee57ce9ecca2279a3e" target='_blank'>
              SETDB1/ATF7IP regulate the precise genome engineering of HUSH-regulated genes
              </a>
            </td>
          <td>
            B. Ruis, Henry Ward, Chad L. Myers, A. Bielinsky, Eric A. Hendrickson
          </td>
          <td>2026-02-09</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Interferons (IFNs) are key cytokines that drive immune responses against infections and cancer, yet few therapies have successfully leveraged IFN signaling for cancer treatment. Long noncoding RNAs (lncRNAs) are emerging as promising therapeutic candidates, but their roles in immune modulation remain largely unexplored. Here, we functionally characterize a breast cancer-associated lncRNA, BRRIAR, which primes the IFN signaling pathway in specific cancer contexts and represents a potential therapeutic strategy for estrogen receptor-positive (ER+) breast cancer. BRRIAR expression and subcellular localization were examined using qPCR, in situ hybridization, single-cell RNA sequencing and spatial transcriptomics. BRRIAR target genes were identified through CRISPR interference, chromatin interaction assays and ChIP sequencing. Mechanistic studies in ER + breast cancer cells included CRISPR-Cas9 genome-wide screens, RNA sequencing, RNA pull-down followed by mass spectrometry, proliferation assays and Western blotting. The therapeutic potential of BRRIAR was evaluated via intratumoral delivery of lipid nanoparticle-encapsulated BRRIAR in ER + breast cancer xenograft models. Immune activation was assessed using flow cytometry and cytokine profiling of human peripheral blood mononuclear cells (PBMCs). We demonstrate that BRRIAR is a key target gene at the 3p26 breast cancer risk region. Primarily expressed in ER + breast tumors, BRRIAR acts both in cis and in trans. Nuclear BRRIAR regulates BHLHE40 expression in cis through chromatin interactions, while cytoplasmic BRRIAR binds in trans to the pattern recognition receptor RIG-I, priming IFN signaling. Overexpression of BRRIAR RNA triggers RIG-I signaling, inducing IFN responses, drives rapid, dose-dependent apoptosis of ER + breast cancer cells in vitro and in vivo, and promotes immune activation in human PBMCs. These findings establish lncRNAs as key regulators of tumor immunity and uncover a critical link between genetic risk, lncRNAs, cancer immunosurveillance and breast cancer development, positioning BRRIAR as a promising lncRNA-based RIG-I activator for ER + breast cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/133eefbee1af54653a522f0634b27dc9045c572f" target='_blank'>
              BRRIAR lncRNA alters breast cancer risk by modulating interferon signaling in cis and in trans
              </a>
            </td>
          <td>
            H. Sivakumaran, Sneha R. Nair, Mainá Bitar, Xue Lu, Lu Wang, Ji Liu, Deshapriya S. Karunarathne, P. Prakrithi, Sebastien Jacquelin, Isela Sarahi Rivera, Kristine M Hillman, Susanne Kaufmann, Rebekah Ziegman, Wei Shi, Sarah Alexandrou, C. E. Caldon, R. Veedu, Quan H. Nguyen, Jonathan Beesley, Michelle N. Wykes, J. French, S. Edwards
          </td>
          <td>2026-01-07</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cb53da812dbac416c29e7f7225ad2ab6decfdb9" target='_blank'>
              Exploring the Space of Tumor Phylogenies Consistent with Single-Cell Whole-Genome Sequencing Data
              </a>
            </td>
          <td>
            Samin Rahman Khan, P. Sashittal
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Colorectal cancer (CRC) develops through evolutionary processes involving genomic alterations, epigenetic regulation, and microenvironmental interactions. While traditionally explained by the stepwise accumulation of driver mutations, contemporary evidence supports a ‘Big Bang’ model in which many early-arising clones expand simultaneously to establish extensive heterogeneity. We reviewed recent studies employing spatially resolved multi-omic sequencing of tumour glands combined with computational modelling. These approaches enable high-resolution reconstruction of clonal architecture, transcriptional states, and chromatin accessibility. Findings show that although early clonal mutations shape tumour expansion, gene expression variability can be independent of genetic ancestry and instead reflects phenotypic plasticity driven by microenvironmental cues. Epigenomic analyses identified recurrent somatic chromatin accessibility alterations in promotors and enhancers of oncogenic pathways, frequently in the absence of DNA mutations, suggesting alternative mechanisms of gene regulation. Immune-focused studies demonstrated that early silencing of antigen-presenting genes and loss of neoantigens facilitate immune escape despite active surveillance. CRC is shaped by an interplay of genome, epigenome, and immune evolution, with non-genetic mechanisms and tumour plasticity emerging as important drivers of progression and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c00d0ede4db6fdd44f5a7c3b138a4161c7d8d0f7" target='_blank'>
              Advances in Colorectal Cancer Cell Biology and Clonal Evolution
              </a>
            </td>
          <td>
            Sopozme Toghey, Elizabeth J. Harvey-Jones, Jonathan D. Towler, Charlotte J. H. Hafkamp, Irene Y. Chong
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Hepatoblastoma (HB) is a paediatric liver malignancy arising from hepatic precursor cells, with >90% of cases harbouring a mutation in exon 3 of CTNNB1. We present a fully genetically characterised HB tumour organoid (tumoroid) biobank, which allows for in vitro studies of disease progression and clonal dynamics in vitro. We established a biobank of 14 tumoroid lines from 9 different patients. Tumours and tumoroids were characterised by whole genome sequencing (WGS) and histology, revealing strong concordance in cell morphology and β-catenin staining. In tumour—tumoroid pairs, identical pathogenic CTNNB1 variants were found, alongside shared copy number alterations (CNAs) and mutations. Variant allele frequency (VAF) was consistently higher in tumoroids, indicating increased tumour purity in vitro. In addition to CTNNB1, we frequently observed ARID1A alterations (single-nucleotide variants [SNVs] or CNAs in 56% of patients), and MYC gains as described previously. In paired pre- and post-treatment samples, we observed a clear increase in mutational load, attributed to a chemotherapy signature. Notably, from one patient, we analysed 4 tumour samples (3 post-treatment) with 4 matching tumoroid lines, all carrying a novel BCL6 mutation and loss of ARID1A. Mutational profiles varied across samples from different locations, suggesting intratumoral heterogeneity and clonal selection during tumoroid derivation. Taken together, this biobank allows detailed analysis of HB tumour biology, including treatment-induced progression and clonal dynamics across temporally and spatially distinct samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af5f1ebd1dcbe4bda8108f789b036cfe1c91f713" target='_blank'>
              A Fully Annotated Hepatoblastoma Tumoroid Biobank Details Treatment-Induced Evolution and Clonal Dynamics in Paediatric Cancer Patients
              </a>
            </td>
          <td>
            Gijs J. F. van Son, F. Ringnalda, M. V. van Roosmalen, Thomas A. Kluiver, Quinty Hansen, E. Duiker, Marius C. van den Heuvel, V. E. de Meijer, R. D. de Kleine, R. R. de Krijger, J. Zsiros, Weng Chuan Peng, R. van Boxtel, M. van de Wetering, Karin Sanders, Hans Clevers
          </td>
          <td>2026-01-18</td>
          <td>Organoids</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Whole-genome duplication (WGD) fuels tumor evolution and therapy resistance, yet the molecular mechanisms governing the switch from the canonical mitotic cell cycle to the endoreplication cycle remain unclear. Here, we combine single-cell proteomics, manifold learning, and live-cell imaging to map the intersection of the mitotic and endoreplication cycles in breast cancer cells exposed to genotoxic agents. We identify two distinct routes to WGD driven by distinct p21 dynamics. High p21 induction induces G2 exit and endocycling, whereas insufficient p21 permits mitotic entry followed by slippage and endomitosis. This therapy-induced switch acts as a facultative stress response, generating drug-resistant polyploid populations that propagate genomic instability through replication stress and the generation of replication-competent micronuclei. Both paths to WGD converge on a common polyploid G0 state dependent on cyclin D1:CDK4/6 activity to complete the transition to the endoreplication cycle, revealing a shared vulnerability. Sequential treatment with genotoxic agents followed by CDK4/6 inhibitors preserves the cytotoxic efficacy of DNA-damaging drugs while simultaneously blocking entry into the endoreplication cycle and WGD-driven evolutionary rescue. These findings reveal the molecular rules governing the switch from the mitotic to endoreplication cycle and highlight the potential of WGD-blocking drugs as adjuvant therapies to inhibit drug resistance and suppress tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64dace37f687cf3f6f5d90e43b0621daaa3dd0b0" target='_blank'>
              Distinct p21 dynamics drive alternative routes to whole-genome duplication through a common CDK4/6-dependent polyploid G0 state
              </a>
            </td>
          <td>
            Yian Yang, Anuraag Bukkuri, Dante Poe, Janet McLaughlin, Sihan Hao, Daniel D. Brown, Katarzyna M. Kedziora, Wayne Stallaert
          </td>
          <td>2026-01-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Transposable elements (TEs) occupy nearly half of the human genome and play diverse biological roles. Despite their abundance, the extent to which TEs contribute to three-dimensional (3D) genome structure remains unclear. To investigate this, we developed te_hic, a computational pipeline that integrates genomic data with chromatin interactions. Our analysis reveals that TE sequences are responsible for 3D genome structure in interphase nuclei. This phenomenon is mediated by the recruitment of specific transcription factors to TEs, which enables both the formation and disruption of chromatin contacts. We computationally identified known contact-forming proteins (CTCF, RAD21, SMC3) and breaking proteins (RNF2), as well as novel candidate contact-formers (SMARCA4, MAFK). Knockdown of the predicted contact-formers SMARCA4 and MAFK decreased contacts and loops at and between TEs. Notably, SMARCA4 knockdown selectively reduced short-range contacts, highlighting its role in maintaining 3D genome structure mediated by TE binding. Overall, our findings demonstrate that TEs are crucial determinants of 3D genome organization in mammalian cells. Key findings TEs determine 48% of the 3D genome structure alone, and 70% of heterotypic contacts. A/B compartments, TADs, and loop signatures can be retrieved using TE-mapped reads only. TFs can be divided into contact formers and breakers at TEs. SMARCA4 and MAFK build chromatin contacts between TE sequences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df3ef8e203c378a8d1a7856db6aaa8c3b98aa293" target='_blank'>
              Dissecting the contribution of transposable elements to interphase chromosome structure
              </a>
            </td>
          <td>
            Liyang Shi, Zhen Xiao, I. A. Babarinde, Xiuling Fu, Gang Ma, Zhouqi Huang, Li Sun, Xuemeng Zhou, Jiangping He, Alexander V. Strunnikov, A. P. Hutchins
          </td>
          <td>2024-10-27</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Outcomes for patients with esophageal squamous cell carcinoma (ESCC) remain poor, partly due to treatment resistance, particularly to DNA-damaging therapies. Poly(ADP‑ribose) polymerase 1 (PARP1) plays a critical role in repairing single‑strand DNA breaks (SSBs). Unrepaired SSBs can be converted into double‑strand breaks (DSBs) during DNA replication, potentially leading to cell death. Genomic amplification of the distal portion of chromosome 3q (3q26-q29) is a frequent copy‑number alteration in ESCC, which harbors genes encoding several oncoproteins. However, whether long noncoding RNAs (lncRNAs) from this region contribute to ESCC pathogenesis and treatment resistance remains poorly understood. In situ hybridization and qPCR were used to assess RNA expression. Protein PARylation was evaluated by immunoprecipitation followed by Western blotting. Cellular phenotypes were quantified using the cell counting kit-8, Annexin V/Propidium iodide staining, and clonogenic assays. DNA damage was monitored by immunofluorescence staining for phosphorylated histone H2AX (γH2AX) and p53-binding protein 1 (53BP1) and by comet assays. RNA-protein interactions were assessed through RNA pulldown coupled with mass spectrometry and RNA immunoprecipitation. Chromatin fractionation and detergent pre-extraction immunofluorescence were conducted to examine PARP1 chromatin association. ESCC growth and responses to treatments were evaluated using xenograft models. The lncRNA LINC00885, hereafter referred to as PARylator, was the most upregulated lncRNA encoded within the 3q26-q29 amplicon in ESCC. PARylator was predominantly nuclear and interacted with PARP1. Knockdown of PARylator increased γH2AX and 53BP1 foci and comet tail moment, triggered apoptosis, reduced clonogenicity, sensitized ESCC cells to cisplatin and ionizing radiation. In vivo, PARylator knockdown impaired tumor growth and increased cisplatin sensitivity in ESCC xenografts. Mechanistically, PARylator promoted PARP1 recruitment to chromatin and catalytic activation, thereby increasing PARP1 auto-PARylation and enhancing the PARylation of X-ray repair cross-complementing 1 (XRCC1). PARylator was further upregulated in response to DNA damage. The DNA damage-responsive, 3q26-q29 amplicon-encoded lncRNA PARylator promotes PARP1‑mediated PARylation and SSB repair, thereby limiting DSB accumulation and supporting ESCC cell survival and resistance to DNA-damaging therapies. Targeting PARylator, alone or in combination with DNA-damaging agents, may represent a novel avenue for ESCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ecafccdcfd377c547198645eff6618f1001f7c3" target='_blank'>
              The lncRNA PARylator promotes PARP1 activation and resistance to DNA‑damaging therapy in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Teng Fei Qi, Yadong Liu, Jinkun Yang, Yi Meng Yue, Man Man Han, Hongtao Liu, Jing Li, Min Liu, Yuwei Zhang, Jia Kou, Wen Jin Li, Xiaoying Liu, Ting La, Tao Liu, Song Chen, Xu Dong Zhang, Shundong Cang, Liu Teng, Tianli Fan
          </td>
          <td>2025-12-31</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Enhancers are distal cis-regulatory elements that orchestrate spatiotemporal gene expression patterns in response to developmental cues and environmental stimuli. Genetic and epigenetic alterations in enhancers are associated with the initiation and progression of human diseases, including cancers. Over the past few decades, accumulating evidence has revealed that a class of nascent RNA transcripts, known as enhancer RNAs (eRNAs), is broadly transcribed from active enhancers. These eRNA species contribute to complex and dynamic gene regulatory networks under both physiological and pathological conditions through diverse mechanisms. Notably, dysregulated eRNA expression has been reported across various cancer types and is often correlated with patient survival outcomes. Consequently, eRNAs are emerging as promising biomarkers and therapeutic targets for cancer treatment. This review provides a comprehensive summary of the current understanding of eRNAs and their mechanisms of action in gene regulation. We discuss the critical roles of eRNAs in both health and disease and highlight their diagnostic and prognostic value, as well as their therapeutic potential in cancer. Additionally, we review current strategies for targeting RNA transcripts, including eRNAs, and discuss the major challenges in developing eRNA-targeted therapies. Finally, we propose future directions for advancing eRNA-based interventions in the treatment of human diseases, including cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9397eec4991d568fddc2093214f80298c06bfee4" target='_blank'>
              Deciphering the regulatory landscape of enhancer RNAs in health and disease
              </a>
            </td>
          <td>
            Qian Wang, P. ten Dijke, Chuannan Fan
          </td>
          <td>2026-01-29</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0842575f97c19bbdfdf55ac42444d8e91ee3215" target='_blank'>
              Intrinsic DNA sequence determinants and tissue-specific regulation of human replication origins
              </a>
            </td>
          <td>
            Marcell Veiner, Marina Salvadores, Iván Galván-Femenía, Fran Supek
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Wellcome Trust scientists have shown that “mutational signatures” in specific nucleotide contexts accumulate in genomes of mammalian tissues, providing clues to underlying causes of specific signatures. Analysis of cancer genomes has identified more than 50 single‐base substitution (SBS) signatures, with SBS1, SBS5, and SBS40 linked to aging and present in normal tissues. SBS1 results from cytosine demethylation, whereas SBS5 and SBS40 arise from unknown endogenous mechanisms. We hypothesized that loss of fragile‐site genes drives these two signatures. FHIT, located at FRA3B, is frequently deleted in cancers, and Fhit‐deficient mouse tissues exhibit a mutation profile resembling human SBS5. Data mining of cancer genome sequences showed that FHIT/Fra3B was the gene loss most significantly correlated with human SBS5 mutations, likely representing the endogenous molecular process determining SBS5 mutations in soma, germ cells, and cancers, and inversely associated with lifespan in mammals. Exome sequences of Fhit knockout tissues revealed ~1000 SBS1 and ~4000 SBS5 mutations per sample, compared to markedly fewer in wild‐type controls. Using SigProfilerAssignment, similar analyses performed with Fhit ko samples where Wwox is also frequently lost identified SBS40c. These findings implicate fragile‐site gene inactivation as a major source of clock‐like mutational signatures, contributing to lifelong mutation accumulation and potentially influencing aging, evolution, and speciation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b576f2a56d83e49e328a800a1ee42f5ac37825f" target='_blank'>
              Endogenous Processes Underlying Clock‐Like Mutational Signatures
              </a>
            </td>
          <td>
            T. Druck, Rami I. Aqeilan, C. Marcelo Aldaz, N. Zanesi, Kay Huebner
          </td>
          <td>2026-01-01</td>
          <td>Genes, Chromosomes & Cancer</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Retrotransposons are genomic parasites frequently reactivated in cancers, where their mobility can cause genetic alterations. However, it remains unclear whether their gene products contribute to cancer beyond mutagenesis. Here, we uncover a chromatin-associated function of RNAs from Long Interspersed Element-1 (LINE-1), the only autonomous retrotransposon in the human genome. Subcellular-resolved transcriptomics revealed that LINE-1 RNAs are primarily nascent transcripts produced by full-length, cell-type-specific genomic copies of evolutionarily young subfamilies. Using a long-read chromosome conformation assay, we identified a class of highly interactive LINE-1 loci required for gene expression across cancer subtypes, revealing an unexpected regulatory role for LINE-1 locus transcription in oncogenic gene control. LINE-1 RNA depletion disrupted LINE-1-centric chromatin interactions and downregulated associated genes, whereas genomic insertion of an inducible LINE-1 generated de novo chromatin interactions in a transcription-dependent manner. Therefore, beyond their mutagenic potential, retrotransposons also regulate cancer gene expression by nucleating chromatin architecture through their transcriptional activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e9e6bb366cb35c566b3052102db69c89368901c" target='_blank'>
              LINE-1 Locus Transcription Nucleates Oncogenic Chromatin Architecture.
              </a>
            </td>
          <td>
            Michael Lee, Yuannyu Zhang, Jun Yi Stanley Lim, Tao Dai, James S Ye, Margaret B Cervantes, Varun Sondhi, Sisi Zheng, Yoon Jung Kim, Brandon Chen, R. DeBerardinis, Jian Xu
          </td>
          <td>2026-01-05</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Micronuclei (MN) are small extranuclear chromosomal fragments that arise from genomic instability and serve as established biomarkers for genotoxicity and disease susceptibility. Once considered only markers of disease, they are now recognized as active, causative drivers of disease progression, driven by emerging evidence of epigenetic regulation within these structures. The micronucleus assay is recognized as a cost‐effective and minimally invasive method for monitoring genotoxicity resulting from both chronic and early exposure to environmental factors such as arsenic and lead, as well as from genetic instability associated with cancer progression. This review critically examines the expanding role of MN beyond traditional cytogenetic endpoints, with particular emphasis on recent insights into their epigenetic landscape. Mass spectrometry‐based studies have demonstrated that MN possess distinct histone posttranslational modification signatures compared to primary nuclei, including alterations in H3K27ac, H3K9ac, and H3K18ac. These modifications affect chromatin structure, gene expression, and DNA repair mechanisms. In the context of xenobiotic exposures, MN‐associated epigenetic changes may function as early indicators of disease progression. Additionally, rupture of the MN envelope can activate innate immune responses through the cGAS–STING pathway or result in chromothripsis, both of which contribute to cancer progression. The concept of “Micronuclear Epigenetics” is highlighted, with its potential application in high‐throughput diagnostic platforms, particularly liquid biopsy, discussed. This approach may enhance early detection and risk stratification in exposure‐induced toxicity and diseases such as cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a963e7666be290f5a7f2a006e9788007c4d9584f" target='_blank'>
              Potential of Micronuclear Epigenetic Signatures in Analyses of Toxicity and Genomic Instability
              </a>
            </td>
          <td>
            Somnath Paul, Ankita Das, A. Bandyopadhyay, Ashok K. Giri
          </td>
          <td>2026-01-27</td>
          <td>Cell Biology International</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1504ceb179d5285f33902944e8fdc793d756f87c" target='_blank'>
              Extrachromosomal DNA drives molecular and clinical heterogeneity in hepatocellular carcinoma: a multi-omics analysis and prognostic model development.
              </a>
            </td>
          <td>
            Kai Huang, Guangquan Zhang, Shuai Hu, Yongfei He, Hang Zhai, Junming Xu, Jicai Wang, Shengjie Hong, Fenfang Wu, Xianjie Shi
          </td>
          <td>2026-02-03</td>
          <td>Human genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT Human papillomavirus (HPV) and Merkel cell polyomavirus (MCPyV) are DNA tumor viruses that cause human cancer. The mechanisms by which HPV and MCPyV oncoproteins induce genomic instability are not well defined. This minireview discusses the influence of these oncoproteins on the repertoire of proteins at replicating DNA, known as the host replisome, and discusses how new technologies like isolation of proteins on nascent DNA (iPOND) can drive the discovery of viral dysregulation of the host replisome to enhance our understanding of viral oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d034cfb38775ad20a94958e31b3237066d26d7c8" target='_blank'>
              Viruses and the host replisome: discovering oncogenic mechanisms of small DNA tumor viruses
              </a>
            </td>
          <td>
            Christopher D Collins, Matthew Stefely, Kavi Mehta, Megan E. Spurgeon
          </td>
          <td>2026-01-07</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Fusion genes are pivotal drivers of tumorigenesis, often generating oncogenic chimeric RNAs and fusion circular RNAs. However, the mechanisms by which these transcripts synergistically contribute to cancer progression remain poorly understood. Here, we identified a lung cancer-specific chimeric RNA KANSL1-ARL17A (chKANSARL) and its circular variant fusion circular RNA KANSL1-ARL17 A (F-circKA), both derived from the fusion gene KANSARL. Functional assays revealed that overexpression of either chKANSARL or F-circKA significantly enhanced lung cancer cell proliferation, migration, and invasion, while their knockdown suppressed these malignant phenotypes. In vivo experiments demonstrated that chKANSARL overexpression accelerated tumor growth in immunodeficient mice. Notably, coexpression experiments uncovered a synergistic regulatory interaction between F-circKA and chKANSARL, amplifying oncogenic effects. Mechanistically, miRNA sequencing and dual-luciferase assays revealed that F-circKA acts as a molecular sponge for miR-6860, thereby derepressing chKANSARL expression. Rescue experiments further validated this regulatory axis, wherein miR-6860 inhibition reversed the tumor-suppressive effects of F-circKA knockdown. Collectively, our study identifies and characterizes a novel F-circKA/miR-6860/chKANSARL regulatory axis, revealing how dual transcriptional outputs from the KANSARL fusion gene can synergistically drive lung cancer progression. These findings highlight a previously unrecognized layer of cooperative regulation between linear and circular fusion RNAs in oncogenesis and provide a new framework for understanding fusion gene-mediated tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261c7a04a77f3f0847188162738372118fe5132c" target='_blank'>
              The KANSL1-ARL17A fusion gene generates oncogenic chKANSARL and F-circKA RNAs that synergistically drive lung cancer progression via a novel F-circKA/miR-6860/chKANSARL axis
              </a>
            </td>
          <td>
            Xinchao Guan, Tao Liu, Sili Chen, Junwei Zhang, Huanliang Huang, Dexiong Chen, Qiaoyuan Yang
          </td>
          <td>2026-01-01</td>
          <td>The Journal of Biological Chemistry</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7f9dd83e428e2d26b0def31173f1786046f888c" target='_blank'>
              CHD4 epigenetically coordinates genomic instability and immunosuppression to drive pan-cancer progression and confer HDAC inhibitor sensitivity
              </a>
            </td>
          <td>
            Guangxu Fu, Yong Tao, Keyi Feng, Yuxing Chen, Wen Zhang, , Guoda Hu, Yunsheng Ou
          </td>
          <td>2026-02-13</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) was characterized by a highly complex genome, with structural variations (SVs) playing a significant role in its development. In this study, we employed Oxford Nanopore Technology long‐read sequencing in paired tumor and adjacent normal liver tissues from 74 Chinese HCC patients to thoroughly characterize the landscape of somatic SVs. Our analysis revealed that somatic SVs were more prevalent in hepatitis B virus (HBV)‐related HCC, with chromosome 1 emerging as a major hotspot, and several members of the chromosome 1 open reading frame (C1orf) family genes expression level exhibited significant age‐related difference. Notably, HBV‐related HCC cases exhibited a higher frequency of deletions, particularly among younger ones (≤ 35 years old). In addition, we observed an increased burden of HBV integration events in younger ones. Remarkably, the divergent‐paired related homeobox (DPRX) loci was identified as a novel gene for HBV integration in younger patients. Together, these findings delineated the somatic SV landscape in HCC and underscored age‐associated HBV‐related genomic alterations as key pathological features of hepatocarcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83cdc214e4351424b89dd91edf26352c6acad965" target='_blank'>
              Landscape of Somatic and Age‐Related Pathogenic Structural Variations in Hepatocellular Carcinoma Revealed by Long‐Read Sequencing
              </a>
            </td>
          <td>
            Zhewen Wei, Yinghao Cao, Hongchao Liu, Mei Liu, Bolun Zhang, Jianming Ying, Jianqiang Cai, Xinyu Bi, Jian‐jun Zhao, Jianguo Zhou, Zhi-yu Li, Zhen Huang, Jianmei Liu, Xueyan Lv, Zhiwen Luo, Zhicheng Wei, Xiaoshi Zhang, Yi Yang, Yiqiao Deng, Yanjiang Yin, Jinghua Chen, Junbo Liang, Xiaoyue Wang, Ye‐fan Zhang, Hong Zhao
          </td>
          <td>2026-01-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/173cc4555c30056ffb452d93c67137c372b260db" target='_blank'>
              Gene duplication is associated with gene diversification and potential neofunctionalization in lung cancer evolution.
              </a>
            </td>
          <td>
            Paul Ashford, A. Frankell, Zofia Piszka, Camilla S. M. Pang, M. Abbasian, Maise Al Bakir, M. Jamal-Hanjani, N. Mcgranahan, C. Swanton, Christine A. Orengo
          </td>
          <td>2026-02-19</td>
          <td>Genome research</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42d6ede17df58e0c555e75494899b3a936c6c0ed" target='_blank'>
              Fully T2T pedigree assemblies reveal genetic stability and epigenetic plasticity of human centromeres across inheritance and cell-fate transitions
              </a>
            </td>
          <td>
            Shihua Dong, Xiaoyun Xing, Monika Cechova, Hailey Loucks, Selvamani Vijayalingam, Amber Neilson, M. Sentmanat, Juan Macias, Tianjie Liu, Z. Dong, Benpeng Miao, Wenjin Zhang, Chad Tomlinson, Heather Schmidt, Edward A. Belter, Ming Hu, Xiaoxia Cui, N. Stitziel, K. Miga, Ting Wang
          </td>
          <td>2026-02-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369c5e33b0cad07e2b10ea8763123bd8af3e00d3" target='_blank'>
              TERT drives liver tumorigenesis beyond telomere elongation
              </a>
            </td>
          <td>
            Laura Braud, Julien Vernerey, Arnaud Guille, Pierre Cordier, Clémence Ginet, Tom Egger, Manuel Bernabe, D. Churikov, Quentin Da Costa, Aïda Meghraoui, C. Desdouets, Li Gu, F. Bertucci, Christophe Lachaud, Vincent Géli
          </td>
          <td>2026-02-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a896c0ab1e3409be8572616d4493f40acb05363d" target='_blank'>
              Haplotype-aware segmentation with HapASeg increases accuracy of detecting homolog-specific somatic copy number alterations.
              </a>
            </td>
          <td>
            Oliver Priebe, Ron Solan, Conor Messer, Claudia Chu, Julian Hess, Gad Getz
          </td>
          <td>2026-02-14</td>
          <td>Genome biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc2d84fd2fd2721449315ec11065338aff77d0ad" target='_blank'>
              MtDNA heteroplasmy controls tumor immune reprogramming through mitochondrial translation
              </a>
            </td>
          <td>
            Ning Liu, Guoshi Chai, Qunyu Lv, Xiaoling Deng, Jiatong Chen, Qian Zhang, Leping Zhang, Xu Liu, Ruolan Tang, Zhe Xu, Shang Cai, Min Jiang
          </td>
          <td>2026-02-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Z-DNA/Z-RNA is an alternative left-handed nucleic acid conformation with established and emerging roles in gene regulation, immunity, and genome instability. However, its occurrence dynamics and lineage specificity across the tree of life have not yet been fully characterized. Utilizing the recently developed and improved Z-DNA searching tool, ZSeeker, we analyzed 281,139 complete organismal genomes, including multiple Telomere-to-Telomere genome assemblies, and generated genome-wide Z-nucleic acid maps, examined their topography, and compared them to dinucleotide-preserving controls. Cellular genomes featured pervasive Z-DNA enrichment relative to expectation, with enrichments of ∼1.5 and ∼1.7-fold in Bacteria and Archaea and ∼3-fold in Eukaryota. In contrast, Viruses exhibited large differences between lineages, with modest enrichment in several DNA viral groups and pronounced depletion across RNA clades, most notably Influenza A/B strains. We built a LASSO regression model trained on non-Influenza viruses (cross-validated R² ≈ 0.73), which identified GC content, genome type, and host type as the leading predictors for Z-nucleic acid density, yet it significantly over-predicted Z-RNA density in Influenza A/B. More than 99% of assemblies exceeded the +2 SD threshold, and a “typical Influenza” genome was predicted at 2.76 bp/kb compared to ∼0.016 bp/kb observed (a ∼170-fold overestimation based on chance alone). Together, these results reveal domain- and lineage-specific regimes: cellular genomes are enriched for Z-DNA consistent with regulatory roles, whereas influenza viruses appear to have undergone strong, lineage-specific depletion of Z-RNA-forming sequences, likely reflecting evolutionary pressure tied to host sensing pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d5326854d4ddc954dec631bbab0b254c15320a" target='_blank'>
              Characterization of Z-DNA dynamics across the tree of life
              </a>
            </td>
          <td>
            Georgios Megalovasilis, Eleftherios Bochalis, Michail Patsakis, Dionysios V. Chartoumpekis, Guliang Wang, Karen M. Vasquez, IIias Georgakopoulos-Soares
          </td>
          <td>2025-12-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9dbefd676963f04b85639797c520690ca798555" target='_blank'>
              Human and viral whole genome sequencing identify HPV and APOBEC as oncogenic drivers in sinonasal squamous cell carcinoma
              </a>
            </td>
          <td>
            H. B. Chong, M. E. Bryan, M. Lin, W. C. Faquin, L. J. Mirabello, S. K. Mishra, J. Lewis, M. S. Lawrence, D. Faden
          </td>
          <td>2026-02-09</td>
          <td>None</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f6c40c7ea1ba98d656117fae44f7bccd796a048" target='_blank'>
              Incomplete developmental silencing of cancer-testis antigen BORIS in humanized mouse model promotes cancer susceptibility
              </a>
            </td>
          <td>
            Emma Price, E. Pugacheva, D. N. Bhatt, Yon Ji, Sheila Yeboah, Arielle Scott, D. Loukinov, Victor V. Lobanenkov
          </td>
          <td>2025-12-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Genomic instability is a hallmark of cancer, encompassing both sequence and structural alterations that drive tumor evolution and heterogeneity. The APOBEC3 family of deoxycytidine deaminases has emerged as a major source of mutagenic activity in cancers. R-loops are RNA-DNA hybrids and structural barriers that interfere with replication and transcription. Among the APOBEC3 family, APOBEC3C (A3C) is particularly worthy of attention for its upregulation, driving the DNA replication stress tolerance in response to replication stress-inducing drug gemcitabine. However, the molecular mechanisms of gemcitabine resistance and regulatory circuitries mediated by A3C remain largely unknown, especially in checkpoint-deficient tumors. Initially, we screened that A3C was a putative transcriptional target of p53, and p53-deficient H1299 cells harboring A3C elicited a chemoresistant phenotype upon gemcitabine treatment both in vitro and in vivo. A3C expression enhanced Chk1-dependent S-phase checkpoint activation, thus slowing down replication fork progression and facilitating DNA repair. Pull-down assay and proteomic analysis identified that A3C had a specific interaction with the RNA helicase DDX5, which coordinately played critical roles in R-loop resolution. In contrast to A3C, DDX5 expression attenuated Chk1-dependent S-phase checkpoint activation. Knockdown of DDX5 in A3C-proficient H1299 cells attenuated gemcitabine-induced Chk1 activation and enhanced the therapeutic index of gemcitabine by promoting R-loop accumulation. Therefore, we conclude that A3C/DDX5/R-loop complex may impair the sensitivity of gemcitabine by modulating Chk1 dynamics and DNA replication/damage response machinery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3c0c1bd150bb81f122f34c4b4eaf7affdf65ea8" target='_blank'>
              APOBEC3C coordinates DDX5 in R-loop resolution and dynamic control of Chk1-mediated stress-responsive circuitry as a prerequisite for gemcitabine resistance in p53-deficient cells
              </a>
            </td>
          <td>
            Li Tao, Yang Zhao, Zhuangzhaung Jiang, Shujing Kong, Yanlin Ding, Tengyang Ni, Weimin Wang, Yanqing Liu
          </td>
          <td>2026-01-07</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65f0e034fc2d48b2d8aadef360dfbb3df87d941e" target='_blank'>
              N6-methyldeoxyadenosine is a transgenerational epigenetic mark protecting DNA from deletion
              </a>
            </td>
          <td>
            Xiaodong Li, Sarah Allen, Li-Yang Lyu, Sebastian Bechara, Cristina Engeroff, Alan Hendrick, M. Nowacki
          </td>
          <td>2026-02-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Abstract Background DDB1 and CUL4‐associated factor 7 (DCAF7) is a WD‐repeat adaptor that recruits substrates to the CUL4DDB1 ubiquitinligase complex, but its pan‐cancer relevance and mechanistic contribution to tumor progression remain unclear. Methods Multi‐omics datasets (genomic, transcriptomic, epigenomic, proteomic and single‐cell) from 33 tumor types were integrated to define DCAF7 expression, regulation, and clinical significance. Somatic alterations and copy‐number variation were analysed across cohorts, and promoter methylation and RNA modification signatures were interrogated. Immune associations were assessed by computational deconvolution and checkpoint‐gene profiling. Pathway and network analyses were performed to infer DCAF7‐linked programmes. Mechanistic and functional validation was conducted in hepatocellular carcinoma (LIHC) cell lines (HepG2, Huh7) using DCAF7 perturbation and pharmacologic Wnt inhibition. Results DCAF7 was overexpressed in most cancers, consistent with copy‐number gain, focal promoter hypomethylation and putative m6A‐linked post‐transcriptional regulation, whereas hypermethylation at two CpG loci predicted poor prognosis in LIHC. DCAF7 alterations, predominantly amplifications, were associated with shorter overall survival in LIHC and positively correlated with DCAF7 mRNA abundance across cohorts. Immunogenomic analyses linked high DCAF7 to CD4+ T‐cell enrichment, broad upregulation of checkpoint genes (PD‐1/PD‐L1, CTLA‐4, TIGIT), and increased tumour mutational burden, microsatellite instability and neoantigen load, suggesting an immune‐evasive phenotype. Network and enrichment analyses converged on canonical Wnt/β‐catenin, Hippo and cell‐cycle programs. In vitro, DCAF7 promoted LIHC cell proliferation and migration by stabilising β‐catenin via increased inhibitory Ser9 phosphorylation of GSK‐3β, thereby inducing c‐Myc and cyclin D1; DCAF7 knockdown or the Wnt inhibitor XAV939 attenuated these effects. Drug‐response modelling further predicted increased sensitivity of DCAF7‐high tumours to 17‐AAG, docetaxel and alsterpaullone. Conclusions DCAF7 is frequently activated by genetic and epigenetic mechanisms across cancers, associates with an immunotherapy‐relevant tumour immune milieu, and drives Wnt/β‐catenindependent malignant phenotypes in LIHC. These findings support DCAF7 as a prognostic biomarker and a candidate therapeutic target, particularly for stratified intervention in LIHC. Key points DCAF7 is up‐regulated in various tumours and correlates with poor prognosis, particularly in LIHC. High DCAF7 expression is linked to CD4+ T cell infiltration, up‐regulation of immune checkpoint genes and increased tumour mutational burden, suggesting a role in tumour immune escape. DCAF7 stabilises β‐catenin by enhancing GSK‐3β Ser9 phosphorylation, thereby driving c‐Myc/cyclin D1 expression and contributing to proliferation and migration in LIHC. DCAF7‐high tumours demonstrate therapeutic vulnerability to 17‐AAG, docetaxel and CDK/GSK‐3 inhibitor, revealing potential targeted treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b20940e080f4082f8c5b31c396f8b45f801709c0" target='_blank'>
              Pan‐cancer multi‐omics reveals DCAF7 as an immune‐modulating prognostic driver and Wnt/β‐catenin activator in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Ruina Luan, Hanbin Lin, Xin Zhao, Jianpeng Li, Maohe Chen, Shiping Luo, Xinjian Lin
          </td>
          <td>2025-12-31</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e782f632b1dfd9799e057f85fe078312d95ae7e" target='_blank'>
              Evidence that extra copies of chromosome 1q play a role in the early phases of pancreatic neoplasia.
              </a>
            </td>
          <td>
            C. Douville, Jeeun Parksong, Marco dal Molin, Sarah Graham, Patricia T. Greipp, R. Knudson, S. Curtis, Yuxuan Wang, L. Dobbyn, M. Popoli, Janine Ptak, Natalie Silliman, K. Romans, Christine A. Iacobuzio-Donahue, Alvin P Makoohon-Moore, A. Lennon, Michael Goggins, R. Hruban, Ashley L. Kiemen, Chetan Bettegowda, K. Kinzler, N. Papadopoulos, L. D. Wood, B. Vogelstein
          </td>
          <td>2026-02-20</td>
          <td>Science advances</td>
          <td>0</td>
          <td>229</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6532d2e7cd4f584634ccf28fbab489accbcdd3b5" target='_blank'>
              DNA repeats: origins, conservation, and role in human disease
              </a>
            </td>
          <td>
            A. K. Rana
          </td>
          <td>2026-01-06</td>
          <td>Genome Instability & Disease</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21cdf0e2b16662a5abcf179b0e48052eb8ef7f91" target='_blank'>
              Ribosomal DNA copy number variation shapes human physiology and disease risk
              </a>
            </td>
          <td>
            A. Raj, J. Brown, N. Thayer, M. Hotz, I. Lam, N. Fong, E. P. Sorokin, M. Thanaj, D. Rothschild, J. Pritchard, M. Barna, D. Hendrickson
          </td>
          <td>2026-01-13</td>
          <td>None</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f2b7d842c50e19f140b3dba7c07392f64035b86" target='_blank'>
              Single-nucleus multi-omics delineates distinct epigenetic programs associated with tumor progression in lung adenocarcinoma.
              </a>
            </td>
          <td>
            Yeji Kim, Huiram Kang, Y. Hwang, Kyung Soo Kim, S. Moon, Jeonghan Kim, Seung Joon Kim, Hae-Ock Lee
          </td>
          <td>2026-02-11</td>
          <td>Clinical epigenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Circular DNA molecules—mitochondrial DNA (mtDNA) and extrachromosomal circular DNA (ecDNA)—are fundamental genetic elements whose structural integrity is essential for cellular homeostasis. Pathological alterations in their length, caused by deletions or duplications, are directly implicated in a broad spectrum of human diseases, from inherited mitochondrial disorders to aggressive cancers. This review provides a comprehensive methodological guide for detecting these specific structural changes. We systematically evaluate classical techniques (long-range PCR, Southern blotting, electron microscopy) that offer direct visualization and validation, and modern high-throughput approaches (short- and long-read sequencing) that enable genome-wide discovery. Special emphasis is placed on quantitative clinical tools like digital droplet PCR (ddPCR) for ultrasensitive monitoring. By presenting a comparative framework, we guide the selection of optimal methods based on resolution, throughput, and clinical applicability. We conclude that an integrated, multi-method strategy is indispensable for robust analysis, positioning the precise detection of circular DNA length alterations as a cornerstone of emerging precision diagnostics and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9367ea411656b4af8c5e9c55df093cb7809d8cc8" target='_blank'>
              The Circular DNA Size Code
              </a>
            </td>
          <td>
            Zurab Abashidze
          </td>
          <td>2026-01-31</td>
          <td>Longevity Horizon</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chromosomal translocations are prevalent genetic events across multiple pediatric cancers, notably in CNS tumors, solid tumors, and leukemias. For decades, Fusion oncoproteins resulting from chromosomal translocations have been proposed as a hallmark of cancers, some of which can drive the process of cancers as the initial event of the disease. In addition, studies have shown that some tumor cells become addicted to the activity of fusion proteins, and cell death occurs when the fusion proteins are depleted. These researches suggest that fusion oncoproteins are one of the most promising targets for cancer treatment. Although fusion proteins are already recognized as critical oncogenic drivers, increasing evidence suggests that they can also form positive feedback loops with other proteins. In cancer patients, positive feedback loops have been shown to activate various oncogenic signals to drive tumor development, and influencing tumor cells’ sensitivity to different therapies. Therefore, these loops not only amplify the functions of the fusion proteins but also render single-agent targeting of the fusion protein insufficient to suppress tumor growth, highlighting the therapeutic potential of combination strategies in treating fusion-positive tumors. This review highlights the oncogenic roles of fusion protein-driven positive feedback loops in tumor initiation and progression, outline the molecular mechanisms underlying their formation and function, and summarize emerging therapeutic strategies targeting these circuits, offering new insights into the treatment of fusion-positive cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43211d0c7c2e449b055a88bf1178c869727d1a0b" target='_blank'>
              Fusion oncoproteins orchestrate tumorigenesis and sustain malignant progression via a positive feedback mechanism
              </a>
            </td>
          <td>
            Wenwen Ying, Xiaomin Wang, Jiayi Yu, Jinhu Wang, Qiao-Chu He, Bo Yang, Yifan Chen, Meidan Ying
          </td>
          <td>2026-01-03</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Dysplastic nodules (DN) are precursors to cirrhosis‐associated malignancy, and the HBV DNA integration in the human genome plays a critical role in tumorigenesis. However, the precise relationship between DN and HBV integration remains unclear. We performed HBV‐capture sequencing on 19 cirrhosis patients with HBV infection (DN, 10; RN, 9), out of which 10 subjects (DN, 9; RN, 1) underwent RNA sequencing. We performed HBV‐capture sequencing on 19 cirrhosis patients with HBV infection (DN, 10; RN, 9), out of which 10 subjects (DN, 9; RN, 1) underwent RNA sequencing. 1936 and 1450 HBV integration sites were identified in the DN and RN samples, respectively. The number of HBV integration sites in DN correlated with nodule size. Breakpoints in HBV genome were concentrated within 100 bps towards the 5' or 3' end of involved HBV genes. Furthermore, integration numbers also positively correlated with number of point mutations in DN samples. We identified 53 and 29 recurrent genes containing HBV integrations in >=2 samples in DN and RN samples, respectively. Higher clonality was observed for HBV integrations in recurrent genes than other HBV‐integrated genes. The HBV integrations in recurrent genes were predominantly located in intron regions. Notably, among those recurrently HBV‐integrated genes in DN samples, SCHIP1, ZDHHC14, YPEL2, RABGAP1L, and SOX5 displayed significant expression alterations. Moreover, clinical indicators revealed significant prolongation of prothrombin time in two DN patients with HBV integrations in SCHIP1 and ZDHHC14. As a new insight regarding HBV integrations in DN stage, our findings suggest a possible role of HBV integration in the transformation of DN to early‐stage liver cancer by affecting the expression of key genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef5e9a714b1a7dd80b4097dd4b95245da6b9365c" target='_blank'>
              Analysis of HBV Integration Reveals New Insights of Oncogenic Mechanism in Dysplastic Liver Nodule
              </a>
            </td>
          <td>
            Xi Zeng, Hui Liu, Zheqi Xu, Xinjie Rao, Yuyouye Wang, Fang Peng, Wei Dong, Ziying Wang, Zhenguang Wang, Xing Gu, Fuchen Liu, Guoliang Li, Weiping Zhou, Linghao Zhao
          </td>
          <td>2026-01-01</td>
          <td>Journal of Medical Virology</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="SUMMARY Plant nuclear genomes contain a variable, though typically minor, fraction of DNA sequences of plastid origin known as NUPTs. Unlike the massive transfer of DNA and genes from the proto‐organelle genome to the nucleus that occurred during the endosymbiotic event that gave rise to plastids, the formation of NUPTs is an ongoing process that does not imply concomitant DNA loss. Although NUPTs are generally considered to be potentially deleterious insertions that are continuously generated and rapidly eliminated at near‐constant turnover rates, accumulating evidence reveals alternative evolutionary trajectories. In this review, we discuss recent findings that highlight the episodic formation of NUPTs, their subsequent proliferation, and their eventual long‐term fixation within the nuclear genome. We also explore their non‐random spatial association with specific genomic elements. NUPTs show preferential overlap with specific superfamilies of transposable elements, which may facilitate their proliferation and dispersal throughout the nuclear genome. Regarding protein‐coding genes, the contribution of NUPTs varies among species. In contrast, NUPTs are found to be consistently enriched among certain classes of non‐coding RNA genes, notably rRNA, tRNA, and specific regulatory RNA families, suggesting that they are involved in the evolution of gene regulation and translational machinery. Overall, these findings underscore the unexpected complexity of the mechanisms underlying NUPT formation and support the idea that they are a significant source of genome variation and evolutionary innovation. Further research is necessary to fully elucidate the mechanisms underlying NUPT formation, as well as to determine their potential adaptive significance in plant genome evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72824b12a89eeb048c0f674beb8dd4ae9c4603a5" target='_blank'>
              Nuclear DNA of plastid origin (NUPTs), neglected driver of genome variation and evolutionary innovation
              </a>
            </td>
          <td>
            Lorenzo Carretero-Paulet, Juan Pablo Marczuk-Rojas, Aaron Gálvez-Salido
          </td>
          <td>2026-02-01</td>
          <td>The Plant Journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/774d717fb446d87b8fca81f2c265377058a6eee1" target='_blank'>
              On-target mutations confer resistance to WRN helicase inhibitors in Microsatellite Unstable Cancer Cells
              </a>
            </td>
          <td>
            G. Picco, Yanhua Rao, Angham Al Saedi, Samantha J. Walker, S. Bhosle, Yang Lee, Sara F. Vieira, Maria Garcia-Casado, Gilberto Valdes Garcia, Kieron May, F. Sassi, Inigo Barrio Hernandez, Mamta Sharma, Cansu Dinçer, Theo Bell, Anastasia Kavasakali, Ray Shenje, Cuthbert Martyr, Edward J. Brnardic, Nanhua Deng, Hristina Grigorova Dimitrova, Emre Karakoc, Sandeep Sundara Rajan, Nicole Chan, Emma Hitch, Katrina McCarten, Camille Fourneaux, Zoe Hewitson, H. Lightfoot, S. Barthorpe, James P. Phelan, Phil Landis, Brian T. Jones, Diana M. Muñoz, Jay Prakash, P. Barsanti, J. Taygerly, Michael P. DeMartino, Emanuel Gonçalves, A. Bertotti, L. Trusolino, Michael A. White, Geeta Sharma, M. Coelho, J. Houseley, Benjamin Schwartz, Mathew J. Garnett
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>67</td>
        </tr>

        <tr id="Eukaryotic genomes contain numerous transposable elements (TEs), whose dysregulation threatens genome stability and may contribute to cancer. Pancreatic adenocarcinoma (PAAD) is among the deadliest cancers, marked by abundant stroma that obscures tumor-specific molecular signals, complicating bulk-tissue analyses. Here, using 71 patient-derived PAAD organoids, we show that TE activities may potentially promote tumorigenesis and provide a source of novel immunotherapeutic targets. We identify 16 new TE-derived transcripts fused with 15 known oncogenes, exhibiting potential oncogenic function and prognostic value. Notably, LTR7-PLAAT4, present in 29% of tumors, encodes a protein variant transcriptionally regulated by FOXM1 binding to the LTR7 promoter. LTR7-PLAAT4 isoform 2 is associated with increased cholesterol ester accumulation and lipid droplet formation mediated through BSCL2 coexpression, potentially fostering tumor progression. On the immunogenic front, HLA-I immunopeptidomics of AsPC-1 cells and DAC13 organoids identify over 11,000 peptides respectively. Althought mutation-derived neoantigens are rare, several peptides are originated from TE-chimeric transcripts, including four predicted by TEprof2. The peptide FLIQHLPLV, detected in 27% of organoids, exhibits robust immunogenicity, validated by T2 binding, mass spectrometry and ELISPOT assays with HLA-genotyped PBMCs. Together, these findings suggest that TE activities may contribute to PAAD progression and diversify its immunopeptidome, providing new opportunities for molecular subtyping and potential immunotherapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/112bba488753be082c706c6e45c781f5f0d2fb01" target='_blank'>
              Transcriptomic landscape of transposable elements reveals LTR7-PLAAT4 as a potential oncogene and therapeutic target in pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Meilong Shi, Chuanqi Teng, Shan Zhang, Xiaobo He, Lingyun Xu, Fengxian Han, Rongqi Wen, Ganjun Yu, Jingwen Liu, Yang Feng, Yanfeng Wu, Yan Ren, G. Jin, Jing Li
          </td>
          <td>2026-01-08</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Cancer development is driven by somatic evolution and clonal selection. However, traditional selective pressure analysis methods have treated all sites within a gene equally, such a gene-level model oversimplifies the complexity of cancer evolution. In this study, we introduced CN/CS-calculator, a novel site-specific method that can capture selective pressures acting across different gene sites. By deciphering the interplay between the selection pattern and the function of a gene in oncogenesis, CN/CS-calculator uncovers a unique class of mini-driver genes, which exhibit weak positive selection, with certain critical sites providing context-dependent promoter effects on the fitness of cancer subclones while others are constrained by evolutionary conservation. Our method emphasizes the importance of site-specific analysis in uncovering how subtle evolutionary forces shape cancer biology. The refined understanding offers new insights into the mechanisms of cancer heterogeneity and molecular evolution, with potential implications for advancing therapeutic strategies and prognostic assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f1b33838c929960cc9e2d6ebf3f9c1d9939a9da" target='_blank'>
              Identification of cancer mini-drivers by deciphering selective landscape in the cancer genome
              </a>
            </td>
          <td>
            Xunuo Zhu, Wenyi Zhao, Siqi Wang, Jingwen Yang, Jingqi Zhou, Binbin Zhou, Ji Cao, Bo Yang, Zhan Zhou, Xun Gu
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="(1) Background: The transcription factor p73 exists in multiple isoforms with divergent functions in cancer. While DNp73 promotes stemness, epithelial–mesenchymal transition (EMT), and metastasis, the tumor-suppressive isoform TAp73 can also switch to promoting cancer progression. How isoforms sharing the same DNA-binding domain produce divergent outcomes remains unclear. (2) Methods: Here, we performed CUT&RUN in combination with JASPAR, transcriptomics, proteomics, patient survival and gene expression data to map genome-wide and promoter-associated DNA-binding and coregulatory transcription factor (coTF) profiles of TAp73α and DNp73β in melanoma cells. (3) Results: Systematic screening for motif enrichment in cancer hallmark gene sets revealed TAp73- and DNp73-specific coTF repertoires with distinct functions. We identified a coregulator signature for EMT genes enriched for both isoforms that has tumor context-dependent effects on survival and correlates with unfavorable patient prognosis. Of these EMT-associated coTFs, PATZ1 was validated as a novel direct interactor of DNp73β. (4) Conclusions: Our results provide a comprehensive reference map of p73 isoform-specific binding and coregulator recruitment and establish a workflow to model their influence on cancer reprogramming with implications for AI-based individualized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51d7357286e4d775a6aeae941e92117c88a7c586" target='_blank'>
              Integrative Multimodal Profiling of TAp73 and DNp73 Reveals Isoform-Specific Transcriptomic Coregulator Landscapes in Cancer Programs
              </a>
            </td>
          <td>
            Steffen Möller, A. Spitschak, Nico Murr, Brigitte M Pützer
          </td>
          <td>2025-12-31</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Short tandem repeats (STRs), or microsatellites, are highly mutable genomic elements that modulate gene regulations and are implicated in a range of human diseases. However, detecting mosaic STR mutations at single-cell resolution remains challenging due to both technical and biological complexities. To address this, we developed BayesMonSTR, a robust algorithm that enables accurate detection of mosaic STR mutations. Using this tool in single-cell analysis of human tissues, we reveal an accumulation of longer mosaic STR insertions and deletions (indels) in aging mitotic and post-mitotic cells. Strikingly, prefrontal cortex (PFC) neurons accumulate a higher burden of STR mutations than B cells or lung epithelium, with aged neurons exhibiting a particularly pronounced increase in longer STR deletions. These mutations are enriched at transcription start sites (TSSs) and active enhancers of highly expressed genes. Our work establishes a foundation for genome-wide, hypothesis-free discovery of disease-associated mosaic STR mutations and reveals a previously unexplored landscape of mosaic STR variation in development and aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd1d43260f2ab61bb3b1461c097bc6e22ec5bab5" target='_blank'>
              Unravelling genome-wide mosaic microsatellite mutations at single-cell resolution
              </a>
            </td>
          <td>
            Chunyi Wang, Wenxuan Fan, Weixiang Wang, Yonghe Xia, Jinhong Lu, Xiaoyu Ma, Jichuan Yu, Yunchao Zheng, Yan Luo, Wenlong Li, Qing Yang, Meizhen Lin, Huan Liu, Yangning Lan, Chengyu Li, Xiaodong Liu, Danyang He, Shang Cai, Xiuyan Yu, Dan Zhou, M. Kellis, Xushen Xiong, Qi Xie, Yanmei Dou
          </td>
          <td>2026-02-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Hepatitis B virus (HBV) covalently closed circular DNA (cccDNA) serves as a stable nuclear template for viral transcription, yet the mechanisms controlling its chromatin accessibility remain poorly understood. Here, we performed single-cell assay for transposase-accessible chromatin sequencing (ATAC-seq) in HBV-infected human hepatocytes to profile cccDNA chromatin states. We identified discrete closed regions within cccDNA, while a subset of cells exhibited open chromatin at these loci, accompanied by elevated CEP290 (centrosomal protein 290) and several additional genes. Functional studies using CRISPR/Cas9-mediated knockdown or CRISPR-based transcriptional activation demonstrated that CEP290 positively regulates cccDNA and total HBV DNA levels. Combining CEP290 knockdown with entecavir or HBV-targeting siRNA further reduced cccDNA, although complete elimination was not achieved. Bulk ATAC-seq demonstrated that CEP290 influences chromatin accessibility at selected cccDNA regions and affects host genomic loci. Taken together, our findings suggest that CEP290, along with other gene candidates identified from regions of sporadically opened cccDNA chromatin, may influence HBV cccDNA regulation and viral replication. These results illustrate that single-cell chromatin profiling can reveal host factors potentially involved in HBV persistence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca036b835cd87ff7350ede73d6a532ce9e7da800" target='_blank'>
              CEP290 is associated with chromatin accessibility of hepatitis B virus cccDNA.
              </a>
            </td>
          <td>
            Sadahiro Iwabuchi, Yiang-Yi Li, Takeharu Sakamoto, Tadashi Imafuku, Kazuhisa Murai, Atsushi Tajima, Masao Honda, Shinichi Hashimoto
          </td>
          <td>2026-02-01</td>
          <td>Journal of molecular biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="To investigates Flap endonuclease 1 (FEN1) as a prognostic and immune-related biomarker for different cancer types by combining multi-omics analyses and experimental validation. We determined the expression, mutational profiles and clinical implications of FEN1 in 33 cancer types based on multi-omics data from The Cancer Genome Atlas (TCGA), Genotype-Tissue Expression (GTEx) and Clinical Proteomic Tumor Analysis Consortium (CPTAC). Functional assays such as FEN1 knockout in liver hepatocellular carcinoma (LIHC) cell lines (Huh-7 and HCCLM3) were conducted to investigate the effect of FEN1 on proliferation, migration, cell cycle progression, and DNA damage repair. Our study has found that FEN1 was overexpressed in 17 cancers, including breast cancer (BRCA), LIHC, lung adenocarcinoma (LUAD). FEN1 overexpression was significantly associated with tumor stage, poor prognosis and diagnosis. Mechanistically, FEN1 promotes the malignant cell phenotypes by promoting cell cycle progression and the DNA damage repair process. External dataset analyses verified FEN1 upregulation in liver cancer tissues, with a positive correlation with poor survival. In vitro and in vivo experiments revealed that FEN1 knockdown can inhibit the proliferation, migration, cell cycle progression, and DNA repair capacity of HCC cells and tumors. In conclusion, pan-cancer studies revealed that FEN1 is overexpressed and positively associated with adverse clinicopathological characteristics in various cancer types. This indicates its potential role as a wide-range biomarker. Furthermore, FEN1 is associated with immune microenvironment regulation, and in some cases, may be associated with immunotherapy response. In LIHC models, experimental validation revealed that FEN1 stimulates tumorigenesis by stimulating proliferation, migration, and genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7474f848decbc0e7dd032bf72f7e503b26f0af8d" target='_blank'>
              Flap endonuclease 1 drives tumorigenesis across cancers and is a master regulator of proliferation and genomic instability in liver cancer.
              </a>
            </td>
          <td>
            Fang Li, Xu Ji, Yuzhe Zhang, Aoran Liu, Ruipeng Li, Xiaoli Peng, Yan Wang, Jipeng Mei, Yuan Yang, Yanliang Wang, Lina Wu, Bengang Wang, Ye Zhang
          </td>
          <td>2026-02-11</td>
          <td>International journal of biological macromolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22bc12167aa754b1c4de77e9b51370ab0ba0cf6c" target='_blank'>
              Mitotic errors as triggers of cell death and inflammation
              </a>
            </td>
          <td>
            D. Rizzotto, Christian Zierhut, A. Villunger
          </td>
          <td>2026-01-01</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Dysplastic nodules (DN) are precursors to cirrhosis‐associated malignancy, and the HBV DNA integration in the human genome plays a critical role in tumorigenesis. However, the precise relationship between DN and HBV integration remains unclear. We performed HBV‐capture sequencing on 19 cirrhosis patients with HBV infection (DN, 10; RN, 9), out of which 10 subjects (DN, 9; RN, 1) underwent RNA sequencing. In total, 1936 and 1450 HBV integration sites were identified in the DN and RN samples, respectively. The number of HBV integration sites in DN correlated with nodule size. Breakpoints in HBV genome were concentrated within 100 bps toward the 5′ or 3′ end of involved HBV genes. Furthermore, integration numbers also positively correlated with number of point mutations in DN samples. We identified 53 and 29 recurrent genes containing HBV integrations in ≥ 2 samples in DN and RN samples, respectively. Higher clonality was observed for HBV integrations in recurrent genes than other HBV‐integrated genes. The HBV integrations in recurrent genes were predominantly located in intron regions. Notably, among those recurrently HBV‐integrated genes in DN samples, SCHIP1, ZDHHC14, YPEL2, RABGAP1L, and SOX5 displayed significant expression alterations. Moreover, clinical indicators revealed significant prolongation of prothrombin time in two DN patients with HBV integrations in SCHIP1 and ZDHHC14. As a new insight regarding HBV integrations in DN stage, our findings suggest a possible role of HBV integration in the transformation of DN to early‐stage liver cancer by affecting the expression of key genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8808ccbc83696379caab37a3ec557df0b07f831b" target='_blank'>
              Analysis of Hepatitis B Virus Integration Reveals New Insights of Oncogenic Mechanism in Dysplastic Liver Nodule
              </a>
            </td>
          <td>
            Xi Zeng, Hui Liu, Zheqi Xu, Xinjie Rao, Yuyouye Wang, Fang Peng, Wei Dong, Ziying Wang, Zhenguang Wang, Xing Gu, Fuchen Liu, Guoliang Li, Weiping Zhou, Linghao Zhao
          </td>
          <td>2026-01-01</td>
          <td>Journal of Medical Virology</td>
          <td>1</td>
          <td>6</td>
        </tr>

        <tr id="Epstein–Barr virus (EBV) is an endemic herpesvirus implicated in autoimmunity, cancer and neurological disorders. Although primary infection is often subclinical, persistent EBV infection can drive immune dysregulation and long-term complications. Despite the ubiquity of infection, the determinants of EBV persistence following primary exposure remain poorly understood, although human genetic variation partially contributes to this phenotypic spectrum1, 2–3. Here we demonstrate that existing whole genome sequencing (WGS) data of human populations can be used to quantify persistent EBV DNA. Using WGS and health record data from the UK Biobank (n = 490,560) and All of Us (n = 245,394), we uncover reproducible associations between blood-derived EBV DNA quantifications and respiratory, autoimmune, neurological and cardiovascular diseases. We evaluate genetic determinants of persistent EBV DNA via genome association studies, revealing heritability enrichment in immune-associated regulatory regions and protein-altering variants in 148 genes. Single-cell and pathway level analyses of these loci implicate variable antigen processing as a primary determinant of EBV DNA persistence. Further, relevant gene programs were enriched in B cells and antigen-presenting cells, consistent with their roles in viral reservoir and clearance. Human leukocyte antigen genotyping and predicted viral epitope presentation affinities implicate major histocompatibility complex class II variation as a key modulator of EBV persistence. Together, our analyses demonstrate how re-analysis of human population-scale WGS data can elucidate the genetic architecture of viral DNA persistence, a framework generalizable to the broader human virome4. Population-scale WGS reveals genetic determinants of persistent EBV DNA, linking immune regulation—especially antigen processing and MHC class II variation—to EBV persistence and heterogeneous disease associations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/505426b60f1eb9d87e7f038fc4c71105366741ae" target='_blank'>
              Population-scale sequencing resolves determinants of persistent EBV DNA
              </a>
            </td>
          <td>
            Sherry S Nyeo, Erin M. Cumming, O. Burren, Meghana S. Pagadala, Jacob C. Gutierrez, Thahmina Ali, Laura C. Kida, Yifan Chen, Hoyin Chu, Fengyuan Hu, Xueqing Zoe Zou, Benjamin Hollis, M. Fabre, S. MacArthur, Quanli Wang, Leif S. Ludwig, Kushal K. Dey, S. Petrovski, R. Dhindsa, Caleb A. Lareau
          </td>
          <td>2026-01-28</td>
          <td>Nature</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="


 Genomic copy number alterations cause aneuploidy, a hallmark of cancer characterized by imbalanced chromosome numbers. Aneuploidy has been shown to negatively associate with cancer immune signaling, but these mechanisms remain understudied. Of note, high levels of aneuploidy are strongly correlated with lethal progression and poor patient outcomes in prostate cancer. Our recent work identified chromosome 8q (chr8q) gain, one of the most frequent aneuploidy events in prostate tumors, as a key driver of disease progression, in part through the cohesin RAD21 gene located on the chr8q24 region. In both prostate cell line and organoid models, we demonstrated that increased RAD21 expression accelerates oncogenesis and aggressive tumor proliferation by mitigating cellular stress during early-stage tumorigenesis. Therefore, we hypothesize that elevated RAD21 levels enable cancer cells to evade immune surveillance by alleviating oncogenic stress and DNA damage. In this study, we analyzed publicly available genomic datasets and discovered that higher RAD21 expression negatively correlated with natural killer (NK) cell infiltration in prostate tumors. To further test our hypothesis, we modulated RAD21 levels in isogenic prostate cell culture models. Using an NK cell co-culture assay, we observed that cells with increased RAD21 expression were significantly less susceptible to NK-mediated cytotoxicity and exhibited greater survivorship when co-cultured with NK cells. In addition, we found that cellular levels of various pro-inflammatory cytokines, including those that stimulate NK cell activation, were significantly downregulated when RAD21 was overexpressed. These findings are consistent with our observation that tumors with high RAD21 levels are linked to reduced NK cell infiltration in prostate cancer cases. Our results suggest that amplification of RAD21, commonly associated with chr8q gains, facilitates immune evasion in cancer cells, thereby promoting early-stage oncogenesis. Targeting RAD21-driven immune evasion may reveal novel therapeutic strategies to restore immune surveillance mechanisms in prostate cancer.



 Elise G. DeArment, Faith Kim, Ruoxi W. Wang, Kate Lu, John R. Lozada, Christine Luo, Andrew Elliott4, Nicholas A. Zorko3, Justin H. Hwang3, David Takeda5, Xiaofeng A. Su. Elevated RAD21 levels promote immune evasion in prostatic malignancies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A032.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ba3556a59b07de1fb3550ddaa2e2a0eb7e783af" target='_blank'>
              Abstract A032: Elevated
 RAD21
 levels promote immune evasion in prostatic malignancies
              </a>
            </td>
          <td>
            Elise G. DeArment, Faith Kim, Ruoxi W. Wang, Kate Lu, John R Lozada, Christine Luo, A. Elliott, Nicholas A. Zorko, Justin H. Hwang, David Takeda, Xiaofeng A. Su
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The epigenetic model for the development of cancer is based on the concept that cancer develops due to overall hypomethylation of the genome with concomitant hypermethylation of promoters of oncogenes. Recent research unravels the epigenetic mechanisms of the malignant transformation of the cell and it contributes to the development of more efficient therapies.

Methods. Detailed investigation of recent literature on PubMed using the keywords DNA repair, epigenetics

Conclusions. It is known that a prominent mechanism of the surveillance of the integrity of the genome is the DNA repair system that protects the cell from deleterious insults by the recruitment of the DNA Damage Response (DDR). Mutations of the genes of this complex system are associated with immunodeficiencies and a predisposition to cancer. Recently, defects of the epigenetic DNA repair sequential regulation are described in cancer tissues. The epigenetic dysfunction involves changes in the methylation patterns of promoters and genes, aberrant histone modifications and the expression of variants, deregulation of histone readers and miRNA processing which will be analyzed in the review.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a012ab5b95d13c2f716ead45fcd5505e77df30ec" target='_blank'>
              Epigenetics in DNA Repair Mechanism and their Relation to Cancer
              </a>
            </td>
          <td>
            Salavoura Aikaterini
          </td>
          <td>2026-01-25</td>
          <td>Journal of Human and Clinical Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Hepatocellular carcinoma (HCC) is characterized by marked intratumoral heterogeneity and poor clinical outcomes. Dysregulated ribosome biogenesis has emerged as a fundamental hallmark of tumor initiation and progression; however, the specific molecular drivers linking this machinery to HCC pathogenesis remain largely undefined. Methods: By integrating multi-omics data from the TCGA and ICGC cohorts, FBLL1 was identified as a key prognostic candidate gene. Its cellular and spatial distribution was analyzed using single-cell RNA sequencing and spatial transcriptomics. Its biological functions in vitro and in vivo were validated through functional experiments, including lentivirus-mediated ectopic expression and siRNA-mediated gene knockdown. Finally, its molecular mechanism was elucidated through transcriptomic analysis and Western blotting. Results: FBLL1 was significantly upregulated in HCC and correlated with poor patient survival. Spatial and single-cell analyses showed that FBLL1 expression was preferentially enriched in malignant hepatocytes within the tumor region. Functionally, knockdown FBLL1 could inhibit the proliferation and clonogenic capacity of HCC cells, while overexpression FBLL1 in non-tumorigenic hepatocytes could promote the tumorigenic phenotype in xenograft models. Transcriptomic analysis indicated that FBLL1 overexpression was associated with the synergistic upregulation of c-Myc and multiple EGFR ligands, as well as decreased expression of hepatocyte functional markers. Consistently, modulation of FBLL1 expression affected the activity of the EGFR–MAPK signaling pathway. Conclusions: Our study identifies FBLL1 as a previously unrecognized regulator associated with malignant state transition in HCC. Rather than acting as a direct regulator of core signaling components, FBLL1 is associated with ligand-dependent activation of the EGFR–MAPK pathway in conjunction with c-Myc upregulation. These findings indicate that FBLL1 represents a promising therapeutic target for disrupting oncogenic signaling programs in liver cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73452fadba8b76804fe5e945db231c49dcf6ea8b" target='_blank'>
              Integrated Multi-Omics and Spatial Transcriptomics Identify FBLL1 as a Malignant Transformation Driver in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Junye Xie, Shujun Guo, Yujie Xiao, Yibo Zhang, An Hong, Xiaojia Chen
          </td>
          <td>2026-01-27</td>
          <td>Cells</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 The minor spliceosome (MiS) is a specialized RNA-processing machinery upregulated in cancer to promote expression of oncogenic genes. Here, we identify its catalytic component, U6atac snRNA, as a druggable vulnerability in prostate and breast cancers. U6atac knockdown triggers R-loop–mediated DNA damage while impairing DNA repair by downregulating key factors such as BRCA1, PARP1, TP53BP1, and CHK1/2, disabling both homologous recombination and non-homologous end joining. This dual effect sensitizes tumors to PARP inhibitors, cisplatin, and radiation, independent of BRCA status. Moreover, we uncover an adaptive resistance mechanism driven by extracellular vesicles enriched in U6atac, which amplify MiS activity and facilitate therapy escape; a process reversed by U6atac depletion. Across multiple in vitro and in vivo models, MiS targeting demonstrates tumor-selective activity with minimal toxicity. These findings position U6atac as a central regulator of genome stability and establish MiS targeting as a promising approach to potentiate genotoxic therapy and overcome resistance.



 Anke Katharina. Augspach, Mark Rubin, Paola Francica, Sven Rottenberg, Rahul Kanadia. Targeting Minor Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B003.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c71eaaa6cc7e28ae4d142a474ab9e0350163c6b" target='_blank'>
              Abstract B003: Targeting Minor Splicing Disrupts DNA Repair and Overcomes Therapy Resistance in Prostate and Breast Cancer
              </a>
            </td>
          <td>
            Anke Augspach, M. Rubin, P. Francica, Sven Rottenberg, Rahul Kanadia
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="MYC genomic amplification and pathway activation is associated with aggressive behavior and poor prognosis in osteosarcoma (OS). However, a discordance exists between copy-number gains and transcriptional output from MYC in OS, and defining these mechanisms is critical to understand and intercept persistent MYC signaling. Here, we showed that cytoplasmic mRNA (poly(A)) sustains MYC activation in OS. Multi-omics profiling and single-cell transcriptomics identified TENT5A, a non-canonical RNA-binding poly(A) polymerase, as selectively upregulated in MYC-activated tumors and enriched in proliferative, stem-like populations. Biochemical and genetic evidence demonstrated that TENT5A directly bound MYC mRNA via its PAP/OAS1 domain, extended its poly(A) tail, and stabilized the transcript, thereby reinforcing MYC-driven stemness and chemoresistance. Gain- and loss-of-function assays, orthotopic xenografts, and patient-derived organoids confirmed that elevated TENT5A enhanced tumor-initiating capacity and reduced chemotherapy sensitivity. Pharmacologic inhibition of TENT5A disrupted MYC mRNA stabilization, shortened poly(A) tails, and reversed chemoresistance in preclinical models. These findings delineate a post-transcriptional RNA-stabilization pathway that reconciles the disconnect between MYC genetic alterations and transcriptional activity and nominate the RNA-binding protein TENT5A as a therapeutically tractable target in OS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73d515c1b7f0b534d63c1002ea77732498dd44f1" target='_blank'>
              TENT5A Maintains MYC mRNA Stability to Enhance Osteosarcoma Stemness.
              </a>
            </td>
          <td>
            Yining Tao, Qi Zhang, Haoyu Wang, Binghui Yang, Haoran Mu, Kaiyuan Liu, Weisong Zhao, Xiyu Yang, Bowen Zhao, D. Zuo, Liu Yang, Zhengdong Cai, Zongyi Wang, Hongsheng Wang, Yingqi Hua, Wei Sun
          </td>
          <td>2026-01-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Analytical challenges in detecting sequencing-unknown extrachromosomal circular DNA (eccDNA)─a molecule with critical roles in tumorigenesis and therapeutic potential─primarily arise from interference by long linear genomic and mitochondrial DNA. Here, we for the first time introduce a selective real-time rolling circle amplification (sRT-RCA) method that overcomes this limitation by combining RecBCD and PacI dual-enzyme digestion to remove interfering nucleic acids, thus permitting direct Phi29 polymerase-mediated RT-RCA without the need for conventional physical purification step. This approach achieves unprecedented sensitivity (2 pg, ∼4 × 10-19 mol circular DNA molecules) within a 104-fold higher background of genomic DNA. Furthermore, the established method, utilizing a simplified direct-lysis workflow, is capable of detecting eccDNA from as few as 125 cells. By this approach, we estimated that eccDNA averages 540 copies (8kb plasmid equivalents) in certain cancer cells (K562). Clinically, eccDNA was detected in 3/10 early stage and 6/10 advanced-stage patients. These findings indicate that eccDNA is widespread in cancer patients and demonstrate the potential utility of our method in clinical applications and translational research, thereby bridging a key technical gap and accelerating the translation of eccDNA-based diagnostics and personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad0f1ea2bf7309e407f53aa8233632e2a1ae6a7f" target='_blank'>
              Ultrasensitive and Rapid Screening of Tumor-Associated Extrachromosomal Circular DNA by Selective Real-Time Rolling Circle Amplification.
              </a>
            </td>
          <td>
            Yiran Liu, Gang Li, Jing Zheng, Qing Li, Wei-Min Tong, Weiyi Lai, Hailin Wang
          </td>
          <td>2026-02-03</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Significance Our studies show the levels of type I topoisomerases are increased in human papillomaviruses (HPV)-positive cells, leading to enhanced levels of DNA breaks and activation of DNA repair pathways. Furthermore, knockdown of either TOP1α or TOP3β impaired viral replication and transcription. Importantly, these two topoisomerases act by distinct mechanisms to regulate viral functions. Elucidating the mechanisms by which topoisomerases regulate viral gene expression and replication offers a promising avenue for targeted therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/843f6cbaaa32b6f9a8d2bc8fb897a0a89ef42241" target='_blank'>
              Differential roles of type I topoisomerases in regulating HPV pathogenesis
              </a>
            </td>
          <td>
            A. Vats, Conor W Templeton, Lou Laimins
          </td>
          <td>2026-01-02</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41b9f167411ef3927697ca406826ae6128e4e410" target='_blank'>
              Hemizygous loss of helicases promotes genomic instability and cancer development.
              </a>
            </td>
          <td>
            Karolin Voßgröne, F. Favero, Krushanka Kashyap, F. G. Rodríguez-González, A. Olsen, Xin Li, Balca R. Mardin, J. Weischenfeldt, Claus S. Sørensen
          </td>
          <td>2026-02-20</td>
          <td>Science advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The total DNA of an organism is called the genome. DNA contains all the instructions for how to build an organism and how that organism will function. Some DNA sequences are repeated thousands and thousands of times throughout the genome. Repeats lined up one after another are called satellite DNA. The number of copies of a given satellite DNA sequence can rapidly change, and differ among individuals. Although satellite DNAs were once considered useless, researchers are continuously discovering their important roles in various organisms. Satellite DNAs are essential for keeping organisms functioning properly. They help cells divide and keep the genome integrity. They can affect behavior, health, and help the organism to overcome stressful conditions. In these ways, satellite DNAs increase Earth’s biodiversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e93be9ccc179d27f89aeee0b54094657efcfefe" target='_blank'>
              Mysteries of Repetitive DNA Segments
              </a>
            </td>
          <td>
            Eva Šatović-Vukšić, Miroslav Plohl
          </td>
          <td>2025-12-29</td>
          <td>Frontiers for Young Minds</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="

 Circular RNAs (circRNAs), a unique class of endogenous noncoding RNAs characterized by their covalently closed loop structure, have emerged as pivotal players in cancer biology due to their exceptional stability, tissue‐specific expression, and diverse regulatory functions. This comprehensive review delves into the multifaceted roles of circRNAs as epigenetic modulators in cancer pathogenesis, systematically exploring their biogenesis via back‐splicing—regulated by
 cis
 ‐elements and
 trans
 ‐factors—and their general functions, including directly recruiting chromatin‐modifying enzymes or DNA methyltransferases or by sequestering noncoding RNAs that feed back to the chromatin, thereby establishing heritable and DNA sequence–invariant expression states. A distinctive feature of this work is its focused examination of how circRNAs interface with key epigenetic regulators, such as histone modifiers, DNA methyltransferases, and chromatin remodeling complexes, thereby influencing gene expression and driving tumor initiation, metastasis, and drug resistance. The review further integrates recent insights from single‐cell sequencing and discusses stoichiometric and structural nuances that define authentic miRNA sponge functions, setting it apart from previous summaries. By synthesizing current evidence and highlighting both mechanistic depth and clinical relevance—such as the potential of circRNAs as biomarkers and therapeutic targets—this article not only advances our understanding of circRNA‐mediated epigenetic dysregulation in cancer but also outlines future research directions and technological challenges, underscoring its significance in bridging basic molecular insights with translational oncology.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df145e1c14bbc9d33031580b823f8b55632795bd" target='_blank'>
              Circular RNAs: Epigenetic Puppeteers Pulling the Strings of Cancer Pathogenesis
              </a>
            </td>
          <td>
            Yimao Wu, Zichang Chen, Junying Lee, Gökhan Zengin, Mengyao Li
          </td>
          <td>2026-01-14</td>
          <td>Medicine Bulletin</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Patients with von Hippel-Lindau (VHL) disease develop multiple distinct kidney cysts and clear cell renal cell carcinomas (ccRCCs) throughout their lifetime. Although cysts are typically considered precursors of ccRCC, no molecular evidence of cysts to tumor progression was provided to date, due to the lack of an accurate molecular characterization of these lesions. We performed a comprehensive molecular characterization of four kidney cysts and six ccRCCs obtained from the same VHL-disease patient, thus all sharing the same genetic germline and host environment. We combined whole genome sequencing (WGS) and RNA sequencing (RNA-seq) profiling with pathological examinations. Cysts and tumors exhibited distinct transcriptomic profiles. Tumors showed increased glycolysis and response to hypoxia, while cysts were associated with enrichment in extracellular matrix organization and inflammation and displayed elevated expression of nephron distal portion markers. Deconvolution analysis further revealed different stromal and immune microenvironments with cysts enriched in myofibroblasts and plasma cells signatures whereas tumors were associated with tumor-associated macrophages (TAMs) and tumor vasculature signatures. Genomically, most cysts and tumors were clonally independent, harboring distinct somatic single-nucleotide variants. Despite sharing some somatic mutational signatures, tumors and cysts exhibited divergent somatic copy number alterations. Only tumors displayed chromosome 3p loss, resulting in VHL loss of heterozygosity, without other driver mutations. VHL patients develop multiple kidney cysts and solid tumors throughout their lifetime and their clinical management is challenging. This study presents the first in-depth molecular analysis of kidney cysts and ccRCC in a single VHL patient. We observed distinct molecular profiles between cysts and tumors, suggesting independent origins. While preliminary, these findings challenge the assumption that cysts always serve as precursors of ccRCC in VHL disease and underscore the need for larger studies to improve surveillance and management of renal lesions in patients with VHL disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89a5cddad02651b1247d054c02f906a3863c26a3" target='_blank'>
              Distinct genomic, microenvironmental, and nephron signatures in VHL kidney cysts and tumors
              </a>
            </td>
          <td>
            I. Rowe, F. Corea, G. Pipitone, G. Scotti, Roberta Lucianò, M. Sant'angelo, N. Tenace, I. Franco, Annamaria Ferrara, Claudio Doglioni, M. Ponzoni, F. de Cobelli, F. Montorsi, U. Capitanio, Paola Carrera, A. Larcher, Andrea Salonia, Federico Daniela Francesco Giorgia Maria Grazia Annalisa Ra Belladelli Canibus Cei Guazzarotti Patricelli Russ, F. Belladelli, D. Canibus, F. Cei, G. Guazzarotti, M. Patricelli, A. Russo, Chiara Re, L. Salerno
          </td>
          <td>2026-01-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53bd31d7ad6a7db4d77a69357d29f94ed64bdd6e" target='_blank'>
              DNA polymerase kappa stabilized by Ptbp2 interacts with MRE11 and promotes genomic instability in leukemia
              </a>
            </td>
          <td>
            Shristi Lama, Bibhudev Barik, Sajitha Is, Tannistha Sarkar, Sayantan Chanda, Monalisa Behera, Payel Guha, Subhankar Priyadarshi Behera, S. Biswas, Sonali Mohapatra, G. Biswas, Soumen Chakraborty
          </td>
          <td>2026-02-10</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdbf7c97f85876b986624ce952ff6a0df44fa34d" target='_blank'>
              A genomic and epigenomic view of human centromeres.
              </a>
            </td>
          <td>
            Kate E Jaggi, Savannah J. Hoyt, Rachel J. O’Neill, B. Sullivan
          </td>
          <td>2026-01-06</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="


 BRCA1 is a large protein that regulates many cellular tumor suppressive processes and maintains genome integrity, at least in part through DNA repair and euploidy maintenance. Interestingly, multiple studies suggest that the function of BRCA1 and/or the DNA damage repair mechanisms it regulates are altered in a substantial subset of breast cancers (BCs). Recently, we deciphered the molecular basis for a new process of BRCA1-driven, small RNA-mediated repair of R-loop-associated DNA damage. These previously undescribed, 30- 35 nt long small RNAs, called sdRNA (single-stranded, DNA damage-associated small RNA), are synthesized by BRCA1/RNAi-containing complexes. sdRNAs catalyze the repair of ssDNA breaks by promoting the formation of an atypical DNA repair complex composed of PALB2 (a bona fide BC suppressor gene product) and RAD52 (recently shown to be involved in R-loop processing at double-strand breaks, and also associated with BC). We sought to determine the levels and role of these sdRNAs in breast tumor formation using human specimens and patient-derived models of breast tissue.



 A total of 30 normal breast samples and 96 BC samples were collected. To ensure a sufficient amount of tissue available for RNA sequencing, some samples were expanded into organoids, which are three-dimensional tissue cultures that mimic the characteristics of the parent tissues. Whole-genome small RNA sequencing was used to determine the expression profiles of sdRNAs in BCs across all major subtypes. Abundance profiling and computational modeling methods were used to identify key BC-associated sdRNAs. Key sdRNAs, highly expressed in BCs, were overexpressed in organoid culture to determine the impact of sdRNAs on the growth of normal breast cells.



 BC samples expressed very high levels of multiple sdRNAs compared to normal breast cells, as determined by focused RNA measurements (qRT-PCR) and by small RNA sequencing. sdRNAs were particularly increased in triple-negative BCs, compared to ER+ BCs, including sdRNAs associated with genes that play roles in DNA damage and repair, cell growth, survival, proliferation, and metastasis. We were able to identify a select few of these differentially expressed sdRNAs in an independent published study on circulating small RNAs in urine and blood in BC patients that underwent neoadjuvant therapy and surgery. The circulating sdRNAs were identified as differentially expressed throughout the study timeline. A dataset of small RNAs in BCs, specifically targeting the detection of sdRNAs, was generated across all major subtypes and linked to clinical annotations to identify sdRNAs for further evaluation in vitro. From this, 770 key BC-associated sdRNAs were identified, including for example sdRNAs associated with the AKT1 and SOD1 genes. Conditions were successfully optimized to transfect sdRNAs into breast organoids in 3-dimensional culture with high efficiency. After select sdRNAs were transfected in normal breast organoids, organoid growth rates were observed to be higher compared to non-specific control RNAs.



 BC samples have higher levels of multiple sdRNAs compared to normal breast cells. Our initial studies have identified specific sdRNAs associated with BC subtypes that increase the growth of normal breast tissue, consistent with a potential role in breast tumor development. Our data further suggest that sdRNAs highly expressed in BCs can be detected in liquid biopsies, and future studies will assess whether they can be used as a biomarker of therapy efficiency. Ongoing efforts are focused on further assessing the roles of sdRNAs in cancer initiation and exploring the expression of BC-associated sdRNAs in other cancer types.



 D. K. DeConti, J. Z. Yu, S. Warhadpande, F. Abderazzaq, A. Desbuleux, E. Hatchi, J. M. Rosenbluth. Global expression of small DNA damage-associated RNAs (sdRNAs) in breast cancers [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2025; 2025 Dec 9-12; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2026;32(4 Suppl):Abstract nr PS2-12-09.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bee74904a1dc1bc3f19576ad31cce65636e3dae" target='_blank'>
              Abstract PS2-12-09: Global expression of small DNA damage-associated RNAs (sdRNAs) in breast cancers
              </a>
            </td>
          <td>
            D. DeConti, J. Yu, S. Warhadpande, Fieda Abderazzaq, A. Desbuleux, E. Hatchi, J. Rosenbluth
          </td>
          <td>2026-02-17</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89078a30fc642f15d7607009ddb82791df527ad7" target='_blank'>
              Analysis of genetic mutation characteristics in 40 Chinese patients with hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Shaoshao Xu, Lei-Ting Li, Dongchang Yang, Tao Liu, Binyu Liu, Qingqing Xun, Yanrong Liu
          </td>
          <td>2026-02-09</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Autoimmune disorders (AID) are complex, multifactorial, and pervasive human health conditions that affect people globally. Nearly all AID are characterized by heightened inflammation and a breakdown of immune regulation, resulting in a loss of tissue homeostasis. Additionally, the hyper-inflammatory environment observed in AID is ideal for triggering neoplasia, resulting in various forms of cancer and further complicating disease treatment. Within the dysregulated immune microenvironment underlying AID, there is abundant DNA damage and genome instability. We posit that recurrent cycles of inflammation and DNA damage can result in stochastically elevated levels of somatic mutations in autoimmune disorders, and contribute to downstream malignancies. Understanding AID-associated DNA damage and mutagenicity would illuminate mechanisms that would subsequently inform disease prognosis, pathology, and progression. We explore this theme in our review by highlighting studies linking AID with genome instability and mutations, and carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fc6f49f8355a19b9e8cefc2dd834d5b14d2315b" target='_blank'>
              Genome Instability and Somatic Mutagenesis in Autoimmune Diseases
              </a>
            </td>
          <td>
            Sriram Vijayraghavan, Natalie Saini
          </td>
          <td>2026-02-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97ff32e487ddd3e5870b709d9c8a8e204635bef1" target='_blank'>
              Single-nucleus DNA sequencing delves into the varied genomic evolution of pancreatic cancer
              </a>
            </td>
          <td>
            Haochen Zhang, Christine A. Iacobuzio-Donahue
          </td>
          <td>2026-02-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Simple Summary In many types of cancer, cells double their entire set of chromosomes, a state known as tetraploidy. This change can accelerate the cancer’s growth, spread, and resistance to drugs. We wanted to understand how this chromosome doubling changes the cell’s mitochondria. We compared cells with a normal versus a doubled set of chromosomes from human cancers, yeast, and fungi. We found that in all cases, cells with doubled chromosomes were larger and contained more mitochondria, which were also more active. By identifying this shared trait, our research points to these adapted mitochondria as a potential new target for treating cancers with unstable chromosomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9e2f77cdf8945e8a66c23348025539c8ac24aa7" target='_blank'>
              Mitochondrial Adaptations Underlying Tetraploidization in Human Cancer, Fungal, and Yeast Models
              </a>
            </td>
          <td>
            Mohamed Jemaà, Ameni Bedoui, Nihel Ammous, Ali Gargouri, Mohamed Guerfali
          </td>
          <td>2026-01-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Whole genome doubling (WGD) is a frequent event in cancer evolution associated with chromosomal instability, metastasis, and poor prognosis. While the genomic consequences of WGD are well documented, non-genetic alterations that accompany WGD, such as changes to cell and nuclear size, may also play an important role in tetraploid (4N) cancer cell physiology. Here, we showed that cell and nuclear volume do not always scale with DNA content after WGD in cancer cells, resulting in 4N cells that differ in size. Small size was associated with enhanced cell fitness, mitotic fidelity, and tumorigenicity in 4N cancer cells and with poor patient survival in WGD-positive human cancers. Overall, these results suggest that cell and nuclear size may contribute to the tumorigenic potential of 4N cancer cells and could be an important prognostic marker in human tumors that undergo WGD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e3ce0e5eeb35d45bde98218a3b940e6aa2d9ace" target='_blank'>
              Cell and Nuclear Size are Associated with Chromosomal Instability and Tumorigenicity in Cancer Cells that Undergo Whole Genome Doubling.
              </a>
            </td>
          <td>
            Mathew Bloomfield, Sydney M Huth, Daniella S McCausland, Ron Saad, Nazia Bano, Tran N Chau, Megan L Sweet, N. Baudoin, Andrew McCaffrey, Kai Fluet, Eva M. Schmelz, U. Ben-David, Daniela Cimini
          </td>
          <td>2026-01-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Essential meiotic structure‐specific endonuclease 1 (EME1) is integral to the maintenance of genomic stability in various cancers. However, its biological role and expression profile of this molecule in nasopharyngeal carcinoma (NPC) remain to be explored. In this study, we found that EME1 was overexpressed in NPC specimens compared to adjacent noncancerous tissues and was correlated with poorer overall survival outcomes. Furthermore, EME1 knockdown significantly inhibited the proliferation and migration of NPC cells in vitro and in vivo, with a corresponding sensitization to either camptothecin (CPT) or olaparib, evidenced by a further suppression of proliferation upon drug treatment. Notably, silencing EME1 significantly increased the sensitivity of NPC cell lines to CPT by enhancing ATM‐CHEK2 phosphorylation and inducing nuclear abnormalities. Collectively, our findings suggest that combining EME1 modulation with agents such as CPT or olaparib could be an effective treatment strategy for NPC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64d4b0d132ea96aa28bec17ad81c4b1af8056bcb" target='_blank'>
              Targeting EME1 Increases the Sensitivity of Camptothecin in Nasopharyngeal Carcinoma Cells
              </a>
            </td>
          <td>
            Xizhen Jiang, Falian Liang, Zhirui Lin, Fang Yang, Dong-ping Chen, Gengde Hong, Jinquan Liu, Wenjing Yin, Mengyao Wang, Bin Qi
          </td>
          <td>2026-01-14</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a highly aggressive cancer of the biliary tract, distinguished by significant intratumoral heterogeneity (ITH), which contributes to therapy resistance and unfavorable clinical outcomes. Traditional genome profiling has revealed recurring driver changes in CCA; yet, genomic data alone fails to elucidate functional pathway activation, adaptive signaling, and the diverse treatment responses reported among tumor locations and disease subtypes. This review analyses the use of integrated sequencing technologies, proteogenomics, and phosphoproteomics to systematically characterize intratumoral heterogeneity in cholangiocarcinoma and convert molecular diversity into therapeutically applicable discoveries. We present evidence that the combination of genomic sequencing and mass spectrometry–based proteomics facilitates the direct correlation of genetic mutations with protein expression, post-translational modifications, and signaling system activity. Phosphoproteomic profiling specifically offers functional insights into kinase-driven networks that dictate tumor aggressiveness, therapeutic susceptibility, and adaptive resistance mechanisms, which cannot be anticipated only from DNA-level analysis. We propose that integrating proteogenomic and phosphoproteomic analyses into diagnostic and therapeutic assessments can enhance molecular classification, reveal subtype- and region-specific therapeutic dependencies, and guide rational combination treatment strategies, based on recent extensive proteogenomic studies and functional proteomic investigations in CCA. Pathway-level analysis of intratumoral heterogeneity provides a framework for selecting targeted medicines, predicting resistance, and informing personalized treatment strategies in CCA. The combination of sequencing, proteogenomics, and phosphoproteomics is essential for advancing precision oncology in cholangiocarcinoma. The implementation of this multi-layered analytical approach may better patient classification, refine therapy choices, and eventually improve clinical outcomes for individuals with this particular heterogeneous cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a96c2c6b9428d020026e895b0e1d0644f1ab872b" target='_blank'>
              Integrative Sequencing and Proteogenomic Approaches to Intratumoral Heterogeneity in Cholangiocarcinoma: Implications for Precision Diagnosis and Therapy
              </a>
            </td>
          <td>
            Sirinya Sitthirak, Arporn Wangwiwatsin, Apinya Jusakul, N. Namwat, Poramate Klanrit, Sitthirug Roytrakul, Hasaya Dokduang, Thitinat Duangchan, Yanisa Rattanapan, Attapol Titapun, Apiwat Jareanrat, Vasin Thanasukarn, Natcha Khuntikeo, Teh Bin Tean, Luke Boulter, Yoshinori Murakami, W. Loilome
          </td>
          <td>2026-01-07</td>
          <td>Medical Sciences</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Clonal hematopoiesis of indeterminate potential (CHIP) is a precursor condition characterized by the expansion of blood cell clones harboring somatic mutations originating in hematopoietic stem cells (HSCs). Since individuals with CHIP face a high risk of developing myeloid malignancies, targeting CHIP clones could provide a viable strategy for leukemia prevention. Despite its clinical significance, the mechanisms underlying CHIP predisposition and progression remain poorly understood. Recent genome wide association studies (GWAS) have identified several non-coding genetic loci that are strongly associated with CHIP; however, their underlying mechanisms still remain unknown. We hypothesize that risk variants in these non-coding loci modulate enhancer elements active in HSCs. To test this, we selected 1,374 non-coding variants from 51 loci associated for CHIP risk in the UK Biobank and screened them for regulatory activity using a Massively Parallel Reporter Assay (MPRA). We performed our lentiviral MPRA screen in MUTZ-3 cells, a human hematopoietic cell line relevant to HSCs, which express CD34 surface marker and are dependent on HSC-specific transcription factors. Using a MPRA library of ∼73,000 constructs in CD34+ fraction of MUTZ-3 cells, we identified 87 variants representing 32 GWAS loci. We used targeted genome editing to demonstrate endogenous enhancer activity across 3 MPRA variants that affect the transcription of NKD2, FLT3, and MSI2. Our functional studies on MSI2 indicate that presence of higher levels of MSI2 mediated by CHIP risk allele enhances the clonal expansion of TET2 knockout hematopoietic stem and progenitor cells, providing a mechanistic link whereby non-coding genetic variants can influence the expansion of mutant CHIP clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/103a0a6127003db01ce70e503d0d9d03c9b83f1d" target='_blank'>
              Systematic functional dissection of germline noncoding risk variants impacting clonal hematopoiesis
              </a>
            </td>
          <td>
            Trieu Nguyen, Jessica Jeejan, Takeshi Iwasaki, S. Kales, Joyeeta Chakraborty, Chie Yanase, A. Shekhar, Dominika Kwasniak, Aditi Hegde, Richard Voit, Kristy Stengel, Keisuke Ito, Ryan Tewhey, Satish K. Nandakumar
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abstract Somatic mutations, key alterations in cancer development, exert differential effects across tissues and biological layers, such as transcriptomes, proteomes, and post-translational modifications (PTMs). Although previous pan-cancer studies have characterized the molecular landscape of cancer, the effects of individual somatic mutations across different tissues remain insufficiently explored. Here, we developed Panorama to evaluate the oncogenic potential of single somatic mutations across all cancer types. We collected cancer proteogenomics or multiomics data from over 10 000 individuals across 19 cancer types. Based on five evaluation criteria, we assessed whether a specific mutation affects the abundance of a particular gene’s transcriptome, proteome, or phosphoproteome; the tumor microenvironment; specific RNA- or protein-based signaling pathways; and outlier-level overexpression of PTMs, aiding in potential drug target identification. By leveraging five oncogenic metrics, Panorama quantifies the oncogenic potential of individual somatic mutations and provides a framework for identifying driver mutations by incorporating their downstream effects. With Panorama, researchers can integrate cancer proteogenomics data, providing a comprehensive approach that enhances our understanding of single somatic mutations in specific tissues. Finally, Panorama was developed as a web-based database to ensure easy access for researchers and is freely available at http://139.150.65.64:8080/or https://github.com/prosium/panorama.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3988e7434c0ad5c1c94f629876b4c62c68b08126" target='_blank'>
              Panorama: a database for the oncogenic evaluation of somatic mutations in pan-cancer
              </a>
            </td>
          <td>
            Seung-Jin Park, Seon-Young Kim
          </td>
          <td>2026-01-15</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e23018893423ab572d0573b2072c315fcd456326" target='_blank'>
              OncoBERT: Context-Aware Modeling of Somatic Mutations for Precision Oncology
              </a>
            </td>
          <td>
            Sushant Patkar, Noam Auslander, Stephanie A. Harmon, P. Choyke, B. Turkbey
          </td>
          <td>2026-02-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="DEAD-box (DDX) RNA helicases are essential regulators of RNA metabolism and gene expression. Among them, DDX10 remains poorly characterized despite growing evidence supporting its involvement in human diseases. This review provides a comprehensive analysis of DDX10, from its structural and functional features to its emerging roles in solid tumors and hematologic malignancies. We discuss how DDX10, through its conserved domains, contributes to pre-rRNA processing, ribosome biogenesis, and cell proliferation, and explore potential links between DDX10 and processes such as liquid–liquid phase separation (LLPS) and epigenetic regulation, which may underlie its roles in cancer cell plasticity and stress response. We argue that the dysregulation of these fundamental cellular processes positions DDX10 as a focal point where aberrant RNA metabolism and altered molecular condensates converge to disrupt transcriptional homeostasis and drive oncogenic transformation. Aberrant DDX10 expression is a recurrent feature across multiple cancers, where it promotes tumor progression, therapy resistance, and poor prognosis. Moreover, DDX10 participates in oncogenic fusion events, most notably the NUP98::DDX10 fusion identified in a subset of acute myeloid leukemias, which drives leukemogenesis by disrupting transcriptional regulation and cellular differentiation. Given its tumor-associated expression and diverse biological functions, DDX10 is increasingly recognized as a potential diagnostic biomarker and a promising target for therapeutic strategies. By consolidating current knowledge under this unifying framework, this review highlights the multifaceted roles of DDX10 in cancer biology, advocating further research into its molecular functions and translational potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6180292fbacfd0407c4b7281fb3b5202eef78597" target='_blank'>
              DDX10 RNA Helicase: Structure, Function, and Oncogenic Roles Across Solid and Hematologic Tumors
              </a>
            </td>
          <td>
            Giorgia Isinelli, Genny Scacci, Arianna Capocchia, C. Emiliani, Cristina Mecucci, R. la Starza, Danika Di Giacomo
          </td>
          <td>2026-01-27</td>
          <td>Genes</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Pheochromocytomas and paragangliomas (PPGLs) are rare neuroendocrine tumors primarily involving the adrenal medulla and its associated paraganglia, with heterogeneous clinical behavior and complex molecular drivers. This study aimed to characterize DNA methylation and gene expression patterns in PPGLs to understand the molecular differences between tumor subtypes and malignancy. We performed an integrative analysis of DNA methylation (Illumina EPIC 850K) and gene expression profiles (Affymetrix microarrays) in 24 PPGLs, comparing these with The Cancer Genome Atlas (TCGA) data, to delineate cluster- and malignancy-specific epigenetic patterns. Comparison between pseudohypoxic Cluster I and kinase-signaling Cluster II tumors revealed 13 differentially methylated CpG sites, with a specific CpG within DSCAML1 showing hypermethylation in Cluster II accompanied by increased expression, suggesting context-dependent gene body methylation effects. Benign versus malignant comparisons identified 101 differentially methylated CpGs, including hypermethylated CpG in BAIAP2L1 and hypomethylated CpG in SHANK1 in malignant tumors. Pathway enrichment of differentially methylated genes revealed alterations in Notch signaling, adherens junctions, cytoskeletal regulation, and intracellular transport. Gene expression analysis demonstrated partial overlap between clusters, with malignant tumors exhibiting distinct transcriptional profiles involving RNA processing, metabolism, and adhesion pathways. Correlation between methylation and expression was generally limited, emphasizing that methylation-dependent gene regulation is a locus-specific and context-dependent regulation. These findings illustrate a complex interplay between epigenetic modifications and transcriptional programs in PPGLs, enhancing our understanding of molecular heterogeneity and tumor classification, and identifying candidate biomarkers and therapeutic targets for malignant progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/abb446f2ee7aba49b9c00aa8f44910b6b716df36" target='_blank'>
              Integrative Epigenomic and Transcriptomic Profiling Define Malignancy- and Cluster-Specific Signatures in Pheochromocytomas and Paragangliomas
              </a>
            </td>
          <td>
            Mouna Tabebi, M. Łysiak, Oliver Gimm, P. Söderkvist
          </td>
          <td>2026-01-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Viral infection plays a significant role in the development and evolution of many cancers. It is estimated that between 13% to 18% of all new cancer cases are caused by infection. It is known that viral infection can induce transcriptional changes in cancer, including altering the expression of a seven-membered DNA editing enzyme family known as APOBEC3 or A3. These cytosine deaminases have been implicated in infectious diseases and cancer for their ability to induce cytidine to uridine (C>U) mutations in both viral and host DNA. Indeed, A3-associated mutational signatures SBS2 and SBS13 are present in more than 70% of cancer types and around 50% of all cancer genomes. However, little is known about the interplay between viral infection and A3 dysregulation in cancer. Accurate profiling of A3 expressions and their dysregulations caused by viral infections in cancer has historically been challenging using bulk RNA-seq. To address this, we leverage existing single-cell RNA-seq (scRNA-seq) data sets and our custom single cell bioinformatics pipeline, which uses multiple machine learning models for cell type annotation and cancer cell risk prediction, statistical methods for detection of mutational signatures at the single cell resolution, and custom k-mer based viral reference alignment for viral read detection in order to identify the source of A3 and viral reads across individual cell types within the breast cancer tumor microenvironment. Our studies using this pipeline have provided unparalleled resolution within the tumor microenvironment required to address questions about the association between viral presence and A3 dysregulation in various cancers. Previous analysis of head and neck tumors (GEO: GSE173468) allowed our lab to identify the presence of HPV16 viral reads, which were associated with a significant increase in A3B expression in infected basal cells of the tumor. Our recent analysis of breast cancer tumors (GEO: GSE16529 and GSE243526) has given us critical insights into cell type-specific A3 expression dynamics, indicating that the source of A3A expression was solely found coming from macrophage cells in the tumor and that high-risk cancer cells had primarily high expression of A3B and lacked A3A expression. In this study, we analyzed publicly available scRNA-seq datasets from two normal tissues (NTs), three primary tumors (PTs), and three tamoxifen-treated recurrent tumors (RTs) (GEO: GSE240112) in order to investigate the association between the presence of viral reads, A3 expression, and mutational signatures SBS2/13. We are currently optimizing our pipeline to further improve our viral read identification and association with specific cell types within each tumor microenvironment. This work is supported by the Texas Biomedical Research Institute’s Postdoctoral Forum Fellowship Grant.



 J. D. Lehle, M. Soleimanpour, N. Haghjoo, D. Ebrahimi. Single cell mutational fingerprints and the viral culprits: uncovering APOBEC3 dysregulation in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2025; 2025 Dec 9-12; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2026;32(4 Suppl):Abstract nr PS2-12-19.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00e05140208148a719b032d68c10a3889c3c1cca" target='_blank'>
              Abstract PS2-12-19: Single cell mutational fingerprints and the viral culprits: uncovering APOBEC3 dysregulation in breast cancer
              </a>
            </td>
          <td>
            J. Lehle, M. Soleimanpour, N. Haghjoo, D. Ebrahimi
          </td>
          <td>2026-02-17</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c2f5392a871706eceb6c7ad5228a2907ac09eeb" target='_blank'>
              Association of copy number alterations with the immune transcriptomic landscape in cancer
              </a>
            </td>
          <td>
            S. Loipfinger, A. Bhattacharya, C. G. Urzúa-Traslaviña, M. V. van Vugt, M. de Bruyn, Rudolf S. Fehrmann
          </td>
          <td>2026-01-17</td>
          <td>npj Systems Biology and Applications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29746864ab4e71ce2bf45dd37a7db9f0d5365e74" target='_blank'>
              Spatial patterns of APOBEC mutagenesis in the tumour microenvironment of Asian breast cancer
              </a>
            </td>
          <td>
            Zi-Ching Tan, Yang Wu, Zhen Wei Neo, Mai Chan Lau, S. Teo, J. P. S. Yeong, S. Chang, Jia-Wern Pan
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="


 Capicua (CIC) rearranged sarcomas are a rare but lethal group of soft-tissue sarcomas most commonly found in pediatric and adolescence and young adults (AYA). The most well-characterized CIC-rearrangement is the prototypical CIC::DUX4 fusion oncoprotein, which currently has no effective therapies and is associated with dismal clinical outcomes. Using rare patient-derived CIC::DUX4 cells we recently identified a molecular dependence on the CCNE/CDK2 complex in CIC::DUX4 sarcoma, whereby the CIC::DUX4 oncoprotein transcriptionally upregulates CCNE1 to drive sarcoma growth and survival. Mechanistically, increased CCNE1expression through transcriptional upregulation compromises the G1/S transition to enhance DNA replication stress and genomic instability. Cancer cells adapt to these high replication stress states through increased dependence on cell-cycle checkpoints that enable replication stalling and DNA repair. Consistent with this, we recently defined the G2/M checkpoint kinase, WEE1 as a key regulator of mitotic entry and survival in CIC::DUX4 sarcomas. WEE1 inhibition releases the G2/M break and allows premature mitotic entry and subsequent death through irreversible DNA damage and mitotic catastrophe. We will discuss strategies to pharmacologically exploit this CCNE1 mediated high replication stress state through combinatorial drug approaches that augment replication stress and/or co-target the G2/M checkpoint in CIC-rearranged cancers.



 Ross A. Okimoto, Padmini Bisoyi, Rovingaile Kriska Ponce, Nicholas Thomas. Targeting replication stress and cell-cycle checkpoints to overcome CIC-rearranged sarcomas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr IA006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/232b2d519a6b8c3679dd821434455deea7a95897" target='_blank'>
              Abstract IA006: Targeting replication stress and cell-cycle checkpoints to overcome CIC-rearranged sarcomas
              </a>
            </td>
          <td>
            Ross A. Okimoto, Padmini Bisoyi, R. Ponce, Nicholas J. Thomas
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Ewing sarcoma refers to a family of bone tumors primarily characterized by various fusion gene types, with EWSR1-FLI1 being the most common. These fusion genes lead to the development of tumors growth over time. While most fusion gene detection tools rely on RNA-based data, few utilize genomic data. Existing methodologies face challenges in sensitivity and accuracy of detection. To improve genomic understanding, a novel de novo assembly-based analysis tool, DenovoFusion, was developed. DenovoFusion is a specialized tool designed to detect DNA-level chimeric events and address current limitations in accurate breakpoint identification. The complex fusion detection pipeline includes contig assembly, alignment, and realignment steps that can be customized to the user’s needs, focusing on identifying potential fusion breakpoints. After validation using simulated datasets and 100 Ewing sarcoma DNA sequencing datasets from a study at the Curie Institute in Paris, this approach was compared with the approaches of related tools such as HMFtools, Genefuse and FACTERA. In the analysis of 100 samples, DenovoFusion demonstrated superior accuracy and a false-positive rate of zero when compared to FACTERA. It also performed comparably to methods that utilize predefined fusion lists. By examining ETS family-related fusions in all samples, known fusions were validated, and rare breakpoint variants, including novel ones within the intronic region between EWSR1 exon 11 and FLI1 exon 6, were discovered. The novel assembly-based approach for fusion gene detection improves accuracy in short-read sequencing data, offers a customizable research platform, and shows promise for future applications with long-read sequencing technologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b26deee12d41fa9815f9a87758dae0ea7a1b0034" target='_blank'>
              A de novo assembly based fusion gene detection concept based on DNA-seq data of 100 Ewing sarcoma cases
              </a>
            </td>
          <td>
            Xinwei Zhao, Marc Hotfilder, Eberhard Korsching
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The human genomic triad refers to the presence in every human somatic cell of a small, maternally inherited mitochondrial genome, as well as two nuclear genomes, inherited from the father and mother. Men carry distinct sex chromosomes, Y and X, in their two genomes. Women also have two distinct nuclear genomes with X sex chromosomes, one inherited from the mother and one from the father. Currently, the state-of-the-art in genome research, including forensic applications, is the sequencing and assembly of the diploid genome by haplotypes (two genomes). This is crucial, primarily for DNA phenotyping, which is a valuable complement to other approaches used for DNA identification in criminal investigations. Moreover, the role of DNA phenotyping after the start of universal DNA population registration (which will happen sooner or later) will diminish over time, but will not disappear completely, as the remains of people who have not yet undergone such DNA registration may still be present. The ethical issues raised in connection with both DNA registration and DNA phenotyping are largely imaginary.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36148c17214e356d391370566f85696093ca0bf1" target='_blank'>
              THE TRIAD OF HUMAN GENOMES AND GENOMIC REGISTRATION OF THE ENTIRE POPULATION
              </a>
            </td>
          <td>
            A. Chemeris, R. Garafutdinov, A. Khaliullina, Rushan Galyautdinov, F. Aminev
          </td>
          <td>2025-12-30</td>
          <td>Bulletin of the Institute of Law of the Bashkir State University</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="CRISPR-Cas9 has become a popular genome editing tool for biomedical research and drug development due to its capability to enable precise correction or integration of genetic mutations in the genome. However, precise genome editing competency varies dramatically between cell types depending on their capabilities for DNA damage. In this proof-of-concept study, we took the example of HepG2 and MCF7 to show that omics profiling identifies bottlenecks that are associated with poor precise knock-in (KI) efficiency in hard-to-engineer cells. These bottlenecks include previously described factors such as the predominance of non-homologous end joining (NHEJ) repair and impaired homologous recombination (HR) capability, but also reveals apoptotic priming status of the cells as a limiting factor. Upon further comparative analysis between HepG2 and MCF7 cells, we pinpointed and validated the proliferating cell nuclear antigen (PCNA) as a target to overexpress to enhance precise KI efficiency in MCF7. Overall, we describe how employing a multi-omics approach to characterize cell models of interest can facilitate an in-depth understanding of their editability molecular signature, empowering us to manipulate the activity of key pathways for precise editing, and therefore increase efficiency of desired editing outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a93f75f6b65fd744e6a6a08cd46d5295f2abfb1" target='_blank'>
              Omics-aided design genome editing strategy for challenging human immortalized cell models
              </a>
            </td>
          <td>
            Patricia Mendoza-Garcia, Benjamin Keith, Markus Nordberg, Ella Quist, Cristina Ferrás, Ghaith M. Hamza, Ramy Elgendy, Stephanie Ashenden, Jordi Chi, Natalie R. van Zuydam, Neil Hattersley, Xiang Zhang
          </td>
          <td>2026-02-12</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/213457505483d2de87291e80a391cdd8fe7c3867" target='_blank'>
              Linking somatic mutations in cancer to the electronic properties of DNA
              </a>
            </td>
          <td>
            Benoît de Witte, Cyril Karamaoun, Pauline Hermans, Maxime Tarabichi, F. Pucci, Marianne Rooman
          </td>
          <td>2026-01-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e04b464c0e1b167410299c1bc664ca1e5464bb8" target='_blank'>
              cellSTAAR: incorporating single-cell-sequencing-based functional data to boost power in rare variant association testing of noncoding regions
              </a>
            </td>
          <td>
            Eric Van Buren, Yi Zhang, Xihao Li, M. Selvaraj, Zilin Li, Hufeng Zhou, Nicholette D. Palmer, Donna K. Arnett, J. Blangero, Eric Boerwinkle, B. Cade, Jenna C. Carlson, April P. Carson, Yii-DerIda Chen, J. Curran, R. Duggirala, Myriam Fornage, Nora Franceschini, M. Graff, C. Gu, Xiuqing Guo, Jiang He, Nancy Heard-Cosa, Lifang Hou, Yi-Jen Hung, Rita R. Kalyani, Sharon L. R. Kardia, Eimear E. Kenny, C. Kooperberg, B. Kral, Leslie Lange, Daniel Levy, Changwei Li, Simin Liu, D. Lloyd-Jones, R. Loos, A. Manichaikul, L. W. Martin, Rasika A Mathias, R. Minster, B. Mitchell, J. Mychaleckyj, T. Naseri, Kari E. North, Jeffry R. O'Connell, James A. Perry, P. Peyser, B. Psaty, L. Raffield, Ramachandran S. Vasan, S. Redline, Alexander P. Reiner, Stephen S. Rich, Jennifer A. Smith, Brian W. Spitzer, Hua Tang, Kent D. Taylor, Russell Tracy, S. Viali, Lisa R. Yanek, Wei Zhao, , Namiko Abe, Gonçalo R. Abecasis, F. Aguet, Christine Albert, L. Almasy, Alvaro Alonso, Seth Ament, Peter Anderson, Pramod Anugu, Deborah Applebaum-Bowden, K. Ardlie, D. Arking, A. Ashley-Koch, Stella Aslibekyan, T. Assimes, Paul L. Auer, D. Avramopoulos, N. Ayas, Adithya Balasubramanian, John Barnard, Kathleen Barnes, R. G. Barr, E. Barron-Casella, L. Barwick, Terri L Beaty, Gerald Beck, Diane M. Becker, Lewis C. Becker, Rebecca L. Beer, Amber Beitelshees, Emelia Benjamin, Takis Benos, Marcos Bezerra, Larry Bielak, Joshua C. Bis, Thomas Blackwell, Nathan Blue, Don Bowden, Russell P Bowler, Jennifer Brody, U. Broeckel, J. Broome, Deborah L. Brown, Karen L. Bunting, Esteban G. Burchard, Carlos Bustamante, Erin J. Buth, J. Cardwell, V. Carey, J. Carrier, C. Carty, Richard Casaburi, Juan P. Casas Romero, James M. Casella, Peter Castaldi, M. Chaffin, Christy Chang, Yi-Cheng Chang, Daniel I. Chasman, S. Chavan, Bo-Juen Chen, Wei-Min Chen, Michael Cho, S. Choi, Lee-Ming Chuang, M. Chung, Ren-Hua Chung, Clary Clish, S. Comhair, M. Conomos, Elaine Cornell, Adolfo Correa, Carolyn Crandall, J. Crapo, L. Cupples, J. Curtis, B. Custer, Coleen Damcott, D. Darbar, S. David, Colleen P. Davis, M. Daya, Michael R. DeBaun, Dawn Demeo, R. Deka, Scott Devine, H. Dinh, H. Doddapaneni, Qing Duan, Shannon Dugan-Perez, J. P. Durda, S. Dutcher, Charles B. Eaton, L. Ekunwe, Adel Boueiz, Patrick T Ellinor, L. Emery, S. Erzurum, Charles Farber, J. Farek, T. Fingerlin, M. Flickinger, C. Frazar, Mao Fu, Stephanie M. Fullerton, L. Fulton, Stacey Gabriel, Weiniu Gan, Shanshan Gao, Yan Gao, Margery L S Gass, Heather Geiger, B. Gelb, M. Geraci, S. Germer, R. Gerszten, Auyon Ghosh, Richard Gibbs, Chris Gignoux, M. Gladwin, David E. Glahn, S. Gogarten, Da-Wei Gong, H. Goring, S. Graw, Kathryn J. Gray, Daniel Grine, Colin Gross, Yue Guan, Namrata Gupta, Jeffery Haessler, Michael N. Hall, Yi Han, Patrick J. Hanly, Daniel N Harris, Nicola L. Hawley, Benjamin D. Heavner, S. Heckbert, Ryan Hernandez, David M. Herrington, Craig Hersh, Bertha Hidalgo, James E Hixson, Brian Hobbs, John Hokanson, Elliott Hong, Karin F. Hoth, C. Hsiung, Jianhong Hu, Yi-Jen Hung, H. Huston, C. Hwu, M. Irvin, Rebecca D. Jackson, D. Jain, C. Jaquish, J. Johnsen, Andrew Johnson, Craig W Johnson, R. Johnston, Kimberly Jones, H. M. Kang, R. Kaplan, Shannon Kelly, Michael D. Kessler, Alyna T Khan, Ziad M. Khan, Wonji Kim, J. Kimoff, G. Kinney, Barbara A. Konkle, Holly Kramer, Christoph Lange, Ethan Lange, Cathy C. Laurie, C. Laurie, M. LeBoff, J. Lefaive, Jiwon Lee, Sandra Lee, Wen-Jane Lee, David Levine, Joshua Lewis, Xiaohui Li, Yun Li, Henry J. Lin, Honghuang Lin, Yongmei Liu, Yu Liu, S. Lubitz, K. Lunetta, James Luo, U. Magalang, Michael P. Mahaney, B. Make, Alisa K Manning, J. Manson, Melissa A Marton, S. Mathai, Susanne May, P. McArdle, Merry-Lynn N. McDonald, Sean K. McFarland, S. McGarvey, Daniel J. McGoldrick, Caitlin P. McHugh, B. McNeil, Hao Mei, J. Meigs, Vipin Menon, L. Mestroni, G. Metcalf, Deborah Meyers, Emmanuel Mignot, Julie Mikulla, N. Min, Mollie A Minear, M. Moll, Z. Momin, M. Montasser, Courtney G. Montgomery, Donna M Muzny, Girish N. Nadkarni, Rakhi Naik, Sergei Nekhai, Sarah C. Nelson, Bonnie Neltner, C. Nessner, Deborah Nickerson, Osuji Nkechinyere, T. O’Connor, H. Ochs-Balcom, Geoffrey O. Okwuonu, Allan Pack, David T. Paik, Nicholette D. Palmer, J. Pankow, G. Papanicolaou, Cora Parker, J. Peralta, Marco V. Perez, U. Peters, Lawrence S. Phillips, Jacob Pleiness, Toni Pollin, Wendy S. Post, Julia Powers Becker, M. Boorgula, Michael H. Preuss, P. Qasba, D. Qiao, Zhaohui Qin, N. Rafaels, M. Rajendran, D. Rao, Laura Rasmussen-Torvik, A. Ratan, Robert M Reed, C. Reeves, E. Regan, M. Reupena, Kenneth M. Rice, Rébecca Robillard, N. Robine, Dan Roden, C. Roselli, I. Ruczinski, Alexi Runnels, Pamela H. Russell, S. Ruuska, Kathleen A. Ryan, E. Sabino, D. Saleheen, S. Salimi, Sejal Salvi, Steven Salzberg, Kevin Sandow, Vijay G. Sankaran, J. Santibanez, K. Schwander, D. Schwartz, F. Sciurba, C. Seidman, J. Seidman, Vivien Sheehan, S. Sherman, Amol C. Shetty, A. Shetty, Wayne Hui-Heng Sheu, M. B. Shoemaker, Brian H. Silver, Edwin Silverman, Robert Skomro, A. V. Smith, Josh Smith, Nicholas L. Smith, Tanja Smith, S. Smoller, Beverly M. Snively, M. Snyder, Tamar Sofer, N. Sotoodehnia, A. Stilp, G. Storm, E. Streeten, J. Su, Y. Sung, Jody M Sylvia, Adam A. Szpiro, F. Sériès, D. Taliun, M. Taub, Matthew Taylor, Simeon Taylor, M. Telen, Timothy A. Thornton, M. Threlkeld, Lesley E. Tinker, D. Tirschwell, Sarah Tishkoff, Hemant K. Tiwari, Catherine Tong, Michael Y. Tsai, Dhananjay Vaidya, D. Berg, Peter Vandehaar, S. Vrieze, Tarik Walker, R. Wallace, A. Walts, Fei Fei Wang, Heming Wang, Jiongming Wang, Karol Watson, Jenn Watt, D. Weeks, J. Weinstock, B. Weir, Scott T. Weiss, L. Weng, Jennifer Wessel, Cristen J. Willer, Kayleen Williams, L. K. Williams, Scott Williams, Carla Wilson, James G. Wilson, L. Winterkorn, Q. Wong, Baojun Wu, Joseph Wu, Huichun Xu, Lisa R. Yanek, Ivana V. Yang, Ketian Yu, S. Zekavat, Yingze Zhang, Snow Xueyan Zhao, Xiaofeng Zhu, Elad Ziv, Michael C. Zody, S. Zoellner, Mariza de Andrade, Paul de Vries, L. Fuentes, Jerome L. Rotter, G. Peloso, P. Natarajan, Xihong Lin
          </td>
          <td>2025-12-31</td>
          <td>Nature Methods</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="ABSTRACT The single-stranded DNA deaminase APOBEC3B (A3B) is capable of potently restricting the replication of a range of viruses, including retroviruses (cDNA) and herpesviruses (genomic DNA). However, these and likely other DNA virus families have evolved host species-specific counter-defenses that are equally potent and serve to protect viral DNA from restriction. Although high-risk human papillomavirus (HPV) infection triggers A3B upregulation, potentially as part of an antiviral response, the impact of this restriction factor on papillomavirus replication and pathogenesis has yet to be assessed. To study human A3B antiviral function in the absence of a species-specific counter-defense, here, we ask whether human A3B is capable of restricting Mus musculus papillomavirus (MmuPV1) in cellulo and in vivo. First, we created human A3B and catalytic mutant A3B-E255A expressing FVB/N mice. Second, MmuPV1 gene expression and replication were quantified in primary keratinocytes from these animals and, surprisingly, enzymatically active human A3B caused no measurable impairment in viral transcript or DNA accumulation. Third, A3B, catalytic mutant A3B-E255A, and nontransgenic FVB/N animals were infected with MmuPV1, and similar pathologies were found, regardless of the A3B functionality. Thus, despite likely never being exposed to human A3B during evolution, MmuPV1 appears to be unaffected by this potent, primate-specific antiviral factor. These results suggest that MmuPV1 and perhaps papillomaviruses more broadly possess a conserved mechanism to efficiently escape restriction by human A3B and related DNA deaminases. IMPORTANCE Human papillomaviruses (HPVs) are nearly ubiquitous, and persistent infection with high-risk types causes approximately 5% of cancers worldwide. Although HPV vaccination is effective for preventing infection, insufficient global coverage and a rising incidence of HPV-associated malignancies, such as oropharyngeal carcinoma, highlight the need to understand innate virus clearance mechanisms. APOBEC3 enzymes are a central component of the mammalian innate immune system and are hypothesized to restrict papillomavirus infection, particularly between species. Here, we establish mice that express the human antiviral enzyme APOBEC3B (A3B). Surprisingly, we find that human A3B is incapable of blocking the replication of a murine papillomavirus (Mus musculus papillomavirus 1, MmuPV1) in relevant primary cells from these animals or in infected tissues in vivo. These findings highlight the complexity of teasing apart host-pathogen interactions and suggest that papillomaviruses may have a general mechanism for escaping restriction by antiviral enzymes, such as A3B. Human papillomaviruses (HPVs) are nearly ubiquitous, and persistent infection with high-risk types causes approximately 5% of cancers worldwide. Although HPV vaccination is effective for preventing infection, insufficient global coverage and a rising incidence of HPV-associated malignancies, such as oropharyngeal carcinoma, highlight the need to understand innate virus clearance mechanisms. APOBEC3 enzymes are a central component of the mammalian innate immune system and are hypothesized to restrict papillomavirus infection, particularly between species. Here, we establish mice that express the human antiviral enzyme APOBEC3B (A3B). Surprisingly, we find that human A3B is incapable of blocking the replication of a murine papillomavirus (Mus musculus papillomavirus 1, MmuPV1) in relevant primary cells from these animals or in infected tissues in vivo. These findings highlight the complexity of teasing apart host-pathogen interactions and suggest that papillomaviruses may have a general mechanism for escaping restriction by antiviral enzymes, such as A3B.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/352aeb9cedcc877ce026a88960367a20f3937484" target='_blank'>
              Mus musculus papillomavirus MmuPV1 resists restriction by human APOBEC3B
              </a>
            </td>
          <td>
            Xingyu Liu, Andrea Bilger, Denis L. Lee, P. Argyris, Jiarui Chen, Ella T Ward-Shaw, Emilia Barreto Duran, Yu-Hsiu T. Lin, Cameron Durfee, S. H. Chun, Mahmoud Ibrahim, Joshua Proehl, Allen J. York, Paul F. Lambert, R. Harris
          </td>
          <td>2026-02-11</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="


 Ewing sarcoma is driven by the EWSR1–FLI1 fusion oncoprotein, a transcription factor created by chromosomal translocation in otherwise low-mutation tumors. EWSR1–FLI1 recruits chromatin-remodeling complexes, opens chromatin, and establishes broad H3K27ac/H3K4me1 domains. Uniquely, it binds normally silent GGAA microsatellites dispersed across the genome and converts them into de novo enhancers, elevating transcription of downstream targets. We hypothesize that EWSR1–FLI1 engagement of GGAA repeats globally destabilizes chromatin and enforces a differentiation block, and that direct inhibition of its DNA binding will restore transcriptional control. To test this, we established a propidium iodide (PI) uptake assay after exposure to the chromatin-damaging agent CBL0137 as a surrogate for chromatin accessibility, benchmarking ES against non-ES tumor and non-tumor cells, and in parallel engineered a DNA-binding–dependent reporter to screen small molecules that disrupt fusion–DNA interactions. ES cells showed greater PI uptake following CBL0137 than controls, consistent with globally relaxed chromatin. The reporter screen identified candidate compounds that decreased fusion-dependent activity in a dose-responsive manner; orthogonal luciferase assays supported potential on-target inhibition, and viability assays revealed selective toxicity in ES relative to fusion-negative sarcomas. These findings support a model in which fusion oncoproteins promote chromatin relaxation and establish a complementary screening platform that nominates direct fusion inhibitors, providing mechanistic insight and therapeutic starting points for pediatric sarcomas.



 Ayah Salameh. GGAA Repeat Binding by EWSR1/FLI1 Drives Chromatin Destabilization: A Mechanistic and Drug Discovery Platform [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A010.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4c2a09d1dd63318d72cdbf406bfb1ca4ece465c" target='_blank'>
              Abstract A010: GGAA Repeat Binding by EWSR1/FLI1 Drives Chromatin Destabilization: A Mechanistic and Drug Discovery Platform
              </a>
            </td>
          <td>
            Ayah Salameh
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a21b610301b7b55b77a33ac6317a40f28dcdb985" target='_blank'>
              A gastric microbial chromatin remodeler drives gastric cancer progression and immune evasion by reprogramming the host epigenome
              </a>
            </td>
          <td>
            Xiaoshan Xie, Yue Wei, Zhikai Zheng, Jiaying Zheng, Xijie Chen, Jiarui Wang, Ning Ma, Xiaoling Huang, Peng Zhang, Boyu Zhang, Hanyong Cai, Li Ma, Lishi Xiao, Qingxin Liu, Wenyu Wang, S. Nomura, Shi Chen, Xiangqi Meng, Mong-Hong Lee
          </td>
          <td>2025-12-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27e64bdfdb14f37106c0cabb353f42df99006bef" target='_blank'>
              Ancient eukaryotic immunity through genome editing of viral sequences
              </a>
            </td>
          <td>
            Lisa Mettrop, A. Lipzen, Gilles Mirambeau, K. Barry, Igor V. Grigoriev, G. Piganeau, Marc Krasovec
          </td>
          <td>2026-01-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="High mobility group A1 (HMGA1), a non-histone chromatin structural protein encoded by the HMGA1 gene, plays a critical role in cancer. Recent studies have increasingly focused on its functions in genomic stability and cell death, revealing its involvement in tumorigenesis, cancer progression, and chemotherapy resistance. Consequently, inhibiting HMGA1 represents a promising strategy for developing novel cancer therapies. This review summarizes the cellular and molecular functions of HMGA1 in regulating genomic integrity and cell death in cancer. Furthermore, we discuss current HMGA1-targeting strategies, with emphasis on approaches leveraging its structural and functional characteristics, aiming to provide new insights for future research on HMGA1-targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9dfc0cc9a26137eed4ac1b69fa1e9407df182b1" target='_blank'>
              The role of HMGA1 in genome stability: Implications in human cancer
              </a>
            </td>
          <td>
            Xin-Yuan Lei, Kai-yue He, Yong-Ping Jian, Z. Xu
          </td>
          <td>2026-01-26</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background & objectives Genetic instability is frequent in tumour cells and might occur due to an imbalance of homologous recombination (HR). HR is a crucial mechanism of DNA double-strand break (DSB) repair that depends on the formation and resolution of Holliday junctions for genomic stability maintenance. The SMC6 complex with SMC5 is involved in DSB repair. We sought to investigate the association between SMC6 expression, genomic instability, and prognosis of breast cancer. Methods This was an observational retrospective cohort study. We assessed SMC6 expression and copy number variation (CNV) data measured by qRT-PCR and whole-genome comparative genomic hybridization in 33 women with breast cancer who are non-carriers of BRCA1/BRCA2 mutations. According to nuclear staining, the SMC6 protein expression was evaluated on a tissue microarrayer containing 481 samples classified as SMC6low (negative/weak) or SMC6high (moderate/strong). Results SMC6low tumours tend to show higher CNV. SMC6high group presented poorer disease-free survival than the SMC6low group (P=0.050), mainly for the luminal subtype (P=0.005). SMC6low/ERpos were protective biomarkers for recurrence. Interpretation & Conclusions There is a possible association between SMC6 expression and relapse of breast cancer, also suggesting that SMC6 abnormal expression may indicate tumour genetic instability in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cee362a99ed17056295307f3fac61c772083810" target='_blank'>
              SMC6 expression & outcome of breast cancer
              </a>
            </td>
          <td>
            F. Mangone, A. Krepischi, A. C. Pavanelli, Pedro Henrique Fernandes Gatti, D. Carraro, M. Nagai
          </td>
          <td>2025-11-01</td>
          <td>The Indian Journal of Medical Research</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Despite its critical role in tumor evolution, a detailed quantitative understanding of the evolutionary dynamics of aneuploidy remains elusive. Here we introduce ALFA-K (Adaptive Local Fitness landscapes for Aneuploid Karyotypes), a method that infers chromosome-level karyotype fitness landscapes from longitudinal single-cell data. ALFA-K estimates fitness of thousands of karyotypes closely related to observed populations, enabling robust prediction of emergent karyotypes not yet experimentally detected. We validate ALFA-K’s performance using synthetic data from an agent-based model and empirical data from in vitro and in vivo passaged cell lines. Analysis of fitted landscapes suggests several key insights: (1) Whole genome doubling facilitates aneuploidy evolution by narrowing the spectrum of deleterious copy-number changes; (2) Environmental context and cisplatin treatment significantly modulate the fitness impact of these changes; (3) Fitness effects of copy-number changes depend on parental karyotype; and (4) Chromosome mis-segregation rates strongly influence the predominant karyotypes in evolving populations. The evolutionary dynamics of aneuploidy in solid tumors are challenging to study. Here the authors introduce a method, ALFA-K, which estimates karyotype fitness and predicts emergent karyotypes before experimental detection, and test its performance on synthetic and empirical data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e9f0c1aa74fa8dcd40ddabd793fc67f7d7cd6be" target='_blank'>
              ALFA-K: Local adaptive mapping of karyotype fitness landscapes
              </a>
            </td>
          <td>
            Richard J Beck, Tao Li, Noemi Andor
          </td>
          <td>2025-12-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Transposable elements (TEs) are mobile DNA sequences capable of self-replication (especially retrotransposons) within the genome, which may lead to various forms of DNA damage. The introduction of this review encompasses the diverse classes and subclasses of TEs, particularly emphasizing the most active TEs present in the human genome. An analysis of the retrotransposition process of TEs is presented, illustrating how this mechanism can result in DNA damage and gene rearrangements. Furthermore, the review meticulously examines the implications of TE insertions on gene expression and genomic organization, which may contribute to the development of various diseases, including cancer. The relationship between TE activation and the aging process is also explored, with an emphasis on that epigenetic modifications associated with aging can lead to the derepression of TEs, thereby promoting genomic instability and inflammation. These factors may play a significant role in the pathogenesis of age-related diseases, such as cancer, cardiovascular disorders, and neurodegenerative conditions. Finally, the review considers potential therapeutic approaches aimed at targeting TE activity to alleviate the impacts of aging and associated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1552ca333c184d411cd1fdc06105d2ca4b0a5435" target='_blank'>
              The role of transposable elements activity in genomic instability and their relationship to aging process.
              </a>
            </td>
          <td>
            Jingran Hu, Tian Mao, Kainan Huang, Shangzhi Yang, Wen-xian Yu, Jiacheng Huang, Shiqi Jin, Chuanyu Sun, Zeyidan Jiapaer, Xianli Wang
          </td>
          <td>2026-01-18</td>
          <td>Cell cycle</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddfc626bda7c6db9e371ab841d5c95cbd7dff747" target='_blank'>
              RNAseq analysis reveals the recurrent loss of heterozygosity in lung cancer and associated transcription patterns
              </a>
            </td>
          <td>
            Ruslan Gumerov, Wangzhen He, Phong Luong, Filippo Dall’Olio, Y. Vassetzky, Anna Schwager
          </td>
          <td>2026-01-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Genomic instability is a hallmark of nearly all human cancers, yet the genetic regulators that govern this instability, their functional roles, and their therapeutic potential remain incompletely understood in breast cancer. In this study, we conducted a comprehensive analysis of the expression profiles of genomic instability-maintaining genes (GIMGs) in breast cancer and revealed that elevated expression of GIMGs was associated with adverse clinical outcomes and an altered tumor immune microenvironment. Through further analysis, we identified a core set of GIMGs and pinpointed RPGRIP1L as a critical driver of genomic instability in breast cancer. Notably, RPGRIP1L expression was significantly upregulated in breast cancer tissues and strongly correlated with poor patient prognosis. Functional studies demonstrated that RPGRIP1L knockdown reduced genomic instability in tumor cells, as evidenced by decreased Phosphorylated Histone H2AX (γH2AX) expression, and suppressed tumor growth in mouse models. Additionally, high RPGRIP1L expression was linked to elevated expression of immune checkpoint Programmed Death Ligand 1(PD-L1) in human breast cancer samples. Mechanistically, we found that RPGRIP1L promoted the activation of the Hedgehog signaling pathway, which in turn drived tumor proliferation and upregulated PD-L1 expression. Collectively, these findings highlight RPGRIP1L as a key genomic instability-maintaining gene in human breast cancer, offering critical insights into the molecular mechanisms underlying disease progression. Furthermore, targeting RPGRIP1L may represent a promising therapeutic strategy for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c08de5eb400d357e336428d7f7a366a3a038cb3" target='_blank'>
              Identification of RPGRIP1L as an instability-maintaining gene to drive tumor growth and PD-L1 expression via Hedgehog signaling in breast cancer
              </a>
            </td>
          <td>
            Meng Lv, Yan’e Liu, Weihong Cao, Yongmei Wang, Qi Wang, Xue-Qiang Gao, Baowei Peng, Hai-bo Wang, Yan Mao
          </td>
          <td>2025-12-30</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e961f589165d98aa8aff64f0539527c2012d419" target='_blank'>
              Pooled single-cell screen in colorectal cancer defines transcriptional modules linked to oncogenes.
              </a>
            </td>
          <td>
            Viola Hollek, Francisca Böhning, Catalina Florez Vargas, A. Sieber, M. Morkel, Nils Blüthgen
          </td>
          <td>2026-01-19</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59b02f1b354e2c642c1280c585ed8336d631e18" target='_blank'>
              ZMYM3 S464: a potential phospho-regulatory hub in epigenetic remodeling and oncogenesis
              </a>
            </td>
          <td>
            Apoorva Pai, Althaf Mahin, Samseera Ummar, Athira Perunelly Gopalakrishnan, Prathik Basthikoppa Shivamurthy, A. Rajeev, Rajesh Raju
          </td>
          <td>2026-01-06</td>
          <td>Molecular Genetics and Genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76b560eb7504f0651c24b57c1e03e1ecc5ce721e" target='_blank'>
              DNAJB6 as an immuno-oncogenic hub in liver hepatocellular carcinoma: multi-omic profiling reveals prognostic significance and therapeutic vulnerability
              </a>
            </td>
          <td>
            Yiting Deng, Jin Li, Peng Wang, Xuelian Luo, Hongxia Xu, Yu Xiao, Zhenchuan Wang, Yifei Lu, Ying Chen, Feng Ge
          </td>
          <td>2026-01-21</td>
          <td>Journal of Molecular Histology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Hereditary paragangliomas (PGLs) caused by germline SDHD pathogenic variants (PVs) exhibit a parent‐of‐origin effect, with tumors arising almost exclusively when the PV is inherited from the paternal allele. The Hensen model proposes that a cluster of maternally expressed tumor suppressor genes (TSGs) on chromosome 11p15.5 may play a crucial role in SDHD‐related PGL tumorigenesis, wherein somatic loss of maternal 11p and wild‐type SDHD allele, in conjunction with a paternally inherited SDHD PV, triggers tumor development. To systematically localize and identify the most crucial maternal‐expressed TSGs within 11p15.5, we developed a novel single nucleotide variant (SNV)‐oriented, capture‐based targeted enrichment approach followed by next‐generation sequencing (NGS) to enable high‐resolution loss‐of‐heterozygosity (LOH) analysis. Among 13 SDHD‐related PGLs and 23 non‐SDHD‐related PGLs, a somatic loss of 11p15.5‐15.4 was detected in 92% and 47%, respectively, a significant difference (p = 0.0035). Parental genotype analysis confirmed that the lost chromosome was of maternal origin. In our studies, 12/13 SDHD‐related tumors demonstrated complete loss of the maternal 11p15.5‐15.4 region, preventing localization of a specific driver TSG. Only one exceptional SDHD‐related tumor retained this region, warranting further investigation into the mechanism underlying parent‐of‐origin tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27b365a662db25feb9efe53f1684d83f7862183" target='_blank'>
              A Novel Targeted Sequence for Chromosome 11p15.5 Maternal Loss in SDHD‐Related Paragangliomas
              </a>
            </td>
          <td>
            Huei-Pin Lai, Chen-Hui Lee, Hsuan Hu, An-Ko Chung, Pei-Lung Chen, Wei-Shiung Yang, Wan-Chen Wu
          </td>
          <td>2026-01-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ffb82464a4840ef8616fc6853b84c7285b6e602" target='_blank'>
              Functional footprints of homologous recombination deficiency in prostate cancer revealed by ctDNA fragmentation and transcription factor accessibility.
              </a>
            </td>
          <td>
            Georgios Vlachos, T. Moser, Isaac Lazzeri, Matthias J. Moser, Lisa Glawitch, Emil Bauernhofer, Anna Eberhard, C. Beichler, Hanieh Sadeghi, J. Blatterer, Stefan Kühberger, N. Monsberger, A. Terbuch, Karl Kashofer, J. Geigl, T. Bauernhofer, E. Heitzer
          </td>
          <td>2026-01-09</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b53bd609ebfc34297e879a3902b3d7a47ccfcb00" target='_blank'>
              Phosphoproteome landscape of ARID1A and its implications in DNA damage response and breast cancer pathogenesis
              </a>
            </td>
          <td>
            Apoorva Kumar, Leona Dcunha, Althaf Mahin, Prathik Basthikoppa Shivamurthy, Suhail Subair, Athira Perunelly Gopalakrishnan, A. Rajeev, Rajesh Raju
          </td>
          <td>2026-01-22</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Human T-lymphotropic virus type 1 (HTLV-1), the first human retrovirus identified, is linked to adult T-cell leukemia and HTLV-1-associated myelopathy/tropical spastic paraparesis. However, its post-transcriptional regulation remains poorly understood. Here, we used Oxford Nanopore direct RNA sequencing to profile the HTLV-1 transcriptome and epitranscriptome in MT2 cells. We identified 23 transcript isoforms, encompassing canonical and novel splice variants. Polyadenylation analysis revealed a predominant poly(A) tail length of around 50–100 nucleotides with transcript-specific variations. Distinct RNA modifications, including pseudouridine, N6-methyladenosine, and 5-methylcytidine, were enriched near the 3′ end and varied among transcript classes, with generally lower modification ratios in viral transcripts. These findings provide a more comprehensive map of HTLV-1 RNA splicing, polyadenylation, and modifications in MT2 cells, offering new insights into viral gene regulation and pathogenic mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a1e4aff36600c0fd3356ace5c0413dcc761d3f3" target='_blank'>
              Transcriptomic and Epitranscriptomic Landscape of Integrated HTLV-1 in MT2 Cells
              </a>
            </td>
          <td>
            Shuanglong Wei, Bohan Zhang, Jingwan Han, Hanping Li, Yongjian Liu, Lei Jia, Jingyun Li, Xiaotian Huang, Lin Li
          </td>
          <td>2025-12-30</td>
          <td>Viruses</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="DNA G-quadruplexes (G4s) spontaneously arise in guanine-rich regions such as telomeres. Although G4s regulate essential biological process, their persistence can interfere with DNA metabolism and induce replication stress (RS). Helicases such as BLM and FANCJ play key roles in resolving G4 structures to maintain genome stability. Our investigation delves into how BLM and FANCJ respond to telomeric RS induced by the telomeric G4 ligand, RHPS4. To examine their individual and combined functions, we employed CRISPR/Cas9 to deplete BLM and siRNA to silence FANCJ in U251MG glioblastoma cells. Our results revealed a functional interplay between the two helicases: RHPS4 treatment led to increased FANCJ protein levels and telomeric recruitment in both wild-type and BLM-/- cells. Notably, FANCJ depletion resulted in heightened sensitivity to RHPS4, accompanied by increased telomere dysfunction and DNA damage, particularly in the absence of BLM. These findings underscore a compensatory role for FANCJ in maintaining telomere integrity under RS and highlight its potential as a therapeutic target for enhancing the efficacy of G4-stabilizing agents in cancer treatment. Functional dissection of G-quadruplex–induced replication stress uncovers complementary telomeric roles of BLM and FANCJ helicases and identifies FANCJ loss as a vulnerability that increases telomere damage and sensitivity to G4-stabilizing agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebc7441c3fa58b798b9b453f4dd284ea4b01715a" target='_blank'>
              BLM and FANCJ role in the response to G-quadruplex-dependent telomeric replicative stress
              </a>
            </td>
          <td>
            Ludovica Bonanni, D. Muoio, Elise Fouquerel, Agnese Perelli, Erica Salvati, Francesco Berardinelli, A. Antoccia
          </td>
          <td>2026-01-06</td>
          <td>Communications Biology</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b544cab2c7f5ac6e8301325b2bf082f8d3ca21" target='_blank'>
              TRMT112 drives a tumor growth and metastasis-promoting program in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Amr R. Elhamamsy, Brandon J. Metge, M. H. Elbahoty, Bhavya Papineni, Heba M. Alsheikh, Dongquan Chen, Rajeev S. Samant, L. Shevde
          </td>
          <td>2026-01-08</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Human papillomavirus (HPV) infection is a leading cause of cervical cancer and a significant contributor to anogenital and oropharyngeal malignancies worldwide. While the oncogenic functions of HPV oncoproteins E6 and E7 in disrupting nuclear tumor suppressor pathways are well established, their influence on mitochondrial biology has only recently emerged as a critical facet of HPV-driven carcinogenesis. This review synthesizes current evidence on the qualitative and quantitative alterations of mitochondrial DNA (mtDNA) and their functional consequences in HPV-associated cancers. We discuss how E6 and E7 modulate mitochondrial dynamics, bioenergetics, and redox balance, contributing to metabolic reprogramming, resistance to apoptosis, and adaptation to tumor microenvironmental stress. We also examine the clinical significance of mtDNA mutations, deletions, and copy number variations as potential biomarkers for diagnosis, prognosis, and therapy response. Advances in multi-omics approaches, high-throughput sequencing, and patient-derived organoid models have accelerated the exploration of mitochondria as therapeutic targets. Integrating mitochondrial profiling into HPV-related cancer research holds promise for identifying novel metabolic vulnerabilities and guiding the development of mitochondria-directed treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25dca4a25a42e375c179e79296851a7aeadff46b" target='_blank'>
              Mitochondrial DNA Alterations in HPV-Related Cancers: Emerging Insights and Future Directions
              </a>
            </td>
          <td>
            M. O. Cakir, Melis Selek, Guldide Kayhan, Betül Yılmaz, M. Ozdogan, G. H. Ashrafi
          </td>
          <td>2026-02-02</td>
          <td>DNA</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Retrotransposon mobilization in germline cells enables the rewriting of genetic information to drive genome innovation, species evolution, and adaptation through the generation of de novo mutations. However, uncontrolled mobilization can cause DNA breaks and genome instability, often leading to sterility. How retrotransposon mobilization that can be retained for genome evolution persists despite negative outcomes of retrotransposon activity remains poorly understood. Here, we used Drosophila spermatogenesis as a model to investigate retrotransposon mobilization dynamics. Although many retrotransposon families are transcriptionally active, we found that the LTR retrotransposon nomad completes the full mobilization cascade (including mRNA export, protein translation, and reverse transcription) to produce double-stranded DNA (dsDNA) the most efficiently. Strikingly, despite successfully generating dsDNA, nomad rarely achieves genomic reintegration. Instead, its newly synthesized DNA predominantly forms extrachromosomal circular DNA (ecDNA). These findings show that retrotransposon-derived DNA largely remains as ecDNA. This could prevent widespread genomic integration during spermatogenesis, potentially preserving genome stability with the presence of limited retrotransposon activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9420ae83c1dc149dca66d3f18d2a4cfd87487588" target='_blank'>
              Retrotransposon activation during spermatogenesis achieves massive ecDNA biogenesis but rare integration.
              </a>
            </td>
          <td>
            Lauren Tracy, Yao Chen, Z. Zhang
          </td>
          <td>2026-01-22</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="5-Methylcytosine (mC) is a key regulatory element of mammalian genomes, and plays important roles in development and disease. mC is predominantly written onto CpG dyads by DNA methyltransferases, and can be further oxidized by ten-eleven translocation dioxygenases (TETs) to 5-hydroxymethyl-, 5-formyl-, and 5-carboxylcytosine. This process results in different symmetric and asymmetric combinations of cytosine forms across the two strands of CpGs, each of which represents a unique physicochemical signature in the major groove of DNA. A comprehensive understanding of the individual functions of oxidized mC modifications can therefore only be achieved by considering both strands of CpG dyads. Here, we provide a brief overview of the current state of knowledge on the sequencing and mapping of individual CpG dyad states, their influence on the intrinsic properties of DNA, and their interactions with chromatin proteins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ce94d7f0a338919de147b40cf1b27947501ec1c" target='_blank'>
              The role of cytosine modification symmetry in mammalian epigenome regulation
              </a>
            </td>
          <td>
            Zeyneb Vildan Cakil, Lena Engelhard, Daniel Summerer
          </td>
          <td>2025-12-29</td>
          <td>Chemical Science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Oncofusions, cancer-causing chimeric proteins formed by chromosomal rearrangements, are key drivers in approximately 16% of all cancer cases. These fusion proteins are associated with a wide range of malignancies, including varieties of leukemia, lung cancer, and sarcoma. Thousands of unique oncofusions have been identified in recent years, yet most of these fusion proteins lack mechanistic annotation. Many oncofusions dysregulate chromatin architecture in complex ways that are not yet fully elucidated. Investigating these chromatin alterations is both labor-intensive and costly, limiting the feasibility of large-scale studies. This lack of mechanistic insight has hindered the development of targeted therapies, with only a handful of efficacious small-molecule inhibitors directed toward oncofusions currently in clinical trials or approved for patient use. To address this critical gap, we adapted our recently developed PROD-ATAC method into a high-throughput, scalable platform to systematically elucidate the effects of dozens of candidate inhibitory small molecules on the chromatin dysregulation profiles induced by a pooled library of diverse oncofusions. In the PROD-ATAC scheme, a pooled protein-coding variant library is expressed in a single disease-agnostic cell line, and scATAC-seq (single-cell assay for transposase accessible chromatin) catalogs each variant’s effect on the chromatin landscape. We applied PROD-ATAC to simultaneously assay the chromatin regulatory impact of >40 small molecule inhibitors against a panel of tens of oncofusions (including the majority of patient-identified fusion sequences causal for Ewing sarcoma) in a single assay, generating genome-wide chromatin accessibility datasets representative of >1,000 unique drug - fusion interactions. In this manner, we characterized several small molecules as previously unidentified inhibitors of multiple oncofusions. Certain inhibitors were found to abrogate the chromatin dysregulation caused by an oncofusion either globally or in a targeted cell signaling pathway-specific manner. By profiling multiple rare fusions causal for the same cancers, we identified separate instances of homogenous and heterogenous modes of chromatin dysregulation associated with distinct fusion proteins. In this work, PROD-ATAC chromatin accessibility readouts coupled the identification of novel oncofusion inhibitors with the characterization of each inhibitor’s impact on the epigenetic landscape of oncofusion-expressing cells.



 James E. Corban, Max Frenkel, Srivatsan Raman. High-throughput characterization of inhibitor impacts on chromatin dysregulation induced by oncofusions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A024.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ff99b547f5b85ca4c657ff5fd20f9c13db3f972" target='_blank'>
              Abstract A024: High-throughput characterization of inhibitor impacts on chromatin dysregulation induced by oncofusions
              </a>
            </td>
          <td>
            James E. Corban, Max Frenkel, Srivatsan Raman
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ba610dfa98762d93c525d86aa944670017fdef6" target='_blank'>
              Nanopore sequencing reveals hidden landscape of short L1 transductions in colorectal cancer.
              </a>
            </td>
          <td>
            Päivi Nummi, Aurora Taira, Janne Ravantti, T. Norri, Niko Välimäki, A. Lepistö, L. Renkonen-Sinisalo, S. Koskensalo, Toni T. Seppälä, A. Ristimäki, Kyösti Tahkola, Anne Mattila, Jan Böhm, J. Mecklin, Emma Siili, A. Pasanen, O. Heikinheimo, Ralf Bützow, A. Karhu, Lauri A. Aaltonen, Kimmo Palin, Tatiana Cajuso
          </td>
          <td>2026-02-12</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="The NUDT family comprises evolutionarily conserved enzymes that hydrolyze diverse substrates, including nucleoside phosphates, inositol polyphosphates, and RNA caps. Contrary to earlier perspectives focusing primarily on genome protection, compelling evidence now indicates that the majority of NUDT function as pro-tumorigenic factors. Cancer’s complex landscape, characterized by uncontrolled proliferation, evasion of apoptosis, metabolic reprogramming (like the Warburg effect), and genomic instability, creates an environment where NUDT exert significant influence. Key NUDT members, such as MTH1 (NUDT1), NUDT5, NUDT15, and NUDT22, are frequently overexpressed in cancers and actively promote tumor survival and progression. They achieve this not only by “sanitizing” the nucleotide pool to maintain genomic stability in cancer cells (hydrolyzing damaged nucleotides/caps), but also by dysregulating critical signaling pathways. The relationship between NUDT and cancer is multifaceted, involving intricate roles in nucleotide metabolism, redox homeostasis, and DNA repair. This functional diversity underscores their potential as therapeutic targets. Pharmacological inhibition of specific NUDT, particularly MTH1 and NUDT5, is an active area of research. Such inhibition aims to exploit cancer cell vulnerabilities by increasing the accumulation of damaged nucleotides and enhancing susceptibility to DNA-damaging agents (e.g., chemotherapy, radiotherapy) or PARP inhibitors, offering promising avenues for novel combination therapies. This review comprehensively overviews the mechanisms, diverse functions, and pathophysiological roles of NUDT in cancer biology, critically evaluating their therapeutic potential and the challenges in targeting them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32bb55be44a45d931e8f7256b168a3f66129eaa7" target='_blank'>
              The multifaceted regulatory roles of Nudix hydrolases in cancer and their therapeutic potential
              </a>
            </td>
          <td>
            Jianguo Pan, Jiaxi Yang, Kewei Wang
          </td>
          <td>2026-01-14</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Diversity-Generating Retroelements (DGRs) are specialized genetic systems typically harnessed in nature to evolve new molecular recognition. This mechanism, known as mutagenic retrohoming, relies on an error-prone reverse transcriptase (bRT) that introduces errors at template adenines, followed by the incorporation of the resulting mutagenized complementary DNA (cDNA) into a homologous target gene. Although widely distributed, DGRs are conspicuously absent from key bacterial models, limiting our understanding of their functionality in these hosts and their potential as engineering tools. Here, we demonstrate the ‘plug-and-play’ nature of the Bordetella phage BPP-1 DGR by successfully reconstituting the mutagenic retrohoming mechanism in Escherichia coli. Using high-throughput tools available in this tractable bacterium, we identified key regulatory factors that allowed us to enhance DGR efficiency over 1000-fold. Systematic analysis defines how sequence context governs bRT’s fidelity, uncovering a distinct error profile for the AAC motifs prevalent in natural DGR templates. This intrinsic bias prioritizes the sampling of residues essential for antigen recognition, effectively focusing the evolutionary search within the most productive regions of sequence space. Furthermore, a transposon sequencing screen identified the single-stranded DNA exonuclease ExoI as an inhibitor of DGR activity. While removing ExoI enhanced activity by more than ten-fold, we found that its nuclease activity was dispensable for this inhibition, suggesting a non-catalytic mechanism. Finally, a genome-scale survey highlighted enhanced DGR efficiency at targets located near the replication origin and oriented outwardly from it. This bias is clearly linked to replication directionality, suggesting that certain aspects of DNA replication cycles promote mutagenic retrohoming. Collectively, our work reveals previously unappreciated mechanistic features of DGRs and establishes this reconstituted system as a powerful platform for targeted gene diversification and clarifying the molecular mechanism of mutagenic retrohoming.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3056c5dd9dbb15cccee0988bdd3d94253ddb1908" target='_blank'>
              High-throughput analyses of a reconstituted diversity-generating retroelement identify intrinsic and extrinsic determinants of diversification
              </a>
            </td>
          <td>
            Irem Unlu, Marina K. Smiley, Vladimir Potapov, Yoan Renoux-Martin, Zhi-Yi Sun, Hoong Chuin Lim
          </td>
          <td>2026-02-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f88af5bb8874c2112f831be5ec9601df23ed25e" target='_blank'>
              MER57E3 transposable elements regulate gene expression in a human cell model of neural development.
              </a>
            </td>
          <td>
            Michelle Almeida da Paz, Umut Yildiz, Minyoung Kim, Víctor Campos-Fornés, Marina Pinkasz, Thomas Dahlet, K. Noh, Leila Taher
          </td>
          <td>2026-01-31</td>
          <td>Genome biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Clinical genomic sequencing has revealed that prostate cancer (PCa) progression is associated with changes to chromosome number and structure, termed chromosomal instability (or CIN). While mutations in common driver genes are rare in primary PCa, CIN is detected in >90% of cases, suggesting that CIN may be a driver of the disease. However, the underlying mechanisms that cause CIN in PCa are poorly understood. Cells are susceptible to CIN during mitosis, when the duplicated genome must be equally segregated into two daughter cells. Normally, two centrosomes (cytoplasmic, non-membranous organelles) nucleate and organize microtubules to guide assembly of the mitotic spindle, thus ensuring the fidelity of cell division. However, we recently discovered that cells within primary PCa lack centrosomes, the first cancer type to show this phenomenon. Furthermore, we showed that centrosome elimination in non-tumorigenic cells generates CIN which can transform these cells, causing them to form xenograft tumors in mice. Therefore, we sought to understand the underlying mechanisms that drive the centrosome loss observed in PCa. The prostate is a naturally hypoxic organ, averaging around 4% oxygen tension in the healthy organ which declines as men age. Additionally, tumor hypoxia is strongly correlated with high CIN, particularly in prostate cancers. However, there are few mechanistic links between hypoxia and CIN. Therefore, we hypothesized that hypoxia could cause centrosome loss and, thus, predispose cells to CIN. Indeed, we found that immortalized prostate RWPE-1 cells progressively lost centrosomes when exposed to 1% O2 for as short as 6 hours in vitro. Using ultrastructure expansion microscopy (U-ExM), we find centrosomes are replaced by non-functional microtubule husks devoid of centrosome proteins which we call ‘remnants’. These phenomena also occur in PC-3 and LNCaP prostate cancer lines. Furthermore, we observe centrosome loss specifically in the hypoxic regions of xenograft tumors. Using a combination of RNAseq and chemical inhibitors, we determined that centrosome elimination sits at the crossroads of 3 transcriptional programs: (1) centrosome elimination occurs in basal cells and basal-like cancers that express p63; (2) cells must be confluent with active Hippo pathway; and (3) hypoxia induces a HIF-independent transcriptional response. When these conditions are met, cells transcriptionally downregulate 4 key centrosome proteins (Cep63, Cep152, Cep192, and Cep215), resulting in the depletion of a protective shell of pericentriolar material (PCM) and subsequent centrosome elimination. By overexpressing the mitotic kinase Polo-like kinase 1 (PLK1), we can strengthen the PCM and prevent elimination. Finally, we identify PCa cells containing remnants within prostatectomy tissue samples. This work is significant as it identifies the first examples and mechanisms of centrosome elimination in human somatic and cancer cells, provides a mechanistic link between hypoxia and CIN, and identifies remnants as a cellular indicator for CIN predisposition.



 John M. Ryniawec, Natalya K. Seppanen, Gregory C. Rogers, Anne E. Cress. Hypoxia-induced centrosome elimination as a driver of chromosomal instability in prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr B065.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0fec5efccdcc580bc977b0a1bb739d683cd475" target='_blank'>
              Abstract B065: Hypoxia-induced centrosome elimination as a driver of chromosomal instability in prostate cancer
              </a>
            </td>
          <td>
            John M. Ryniawec, Natalya K. Seppanen, Gregory C Rogers, A. Cress
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="


 ZFTA-RELA (ZR) is the most recurrent genetic alteration seen in pediatric supratentorial ependymoma (EPN) and is sufficient to initiate tumors in mice. Despite the oncogenic potential of ZR, it is observed nearly exclusively in childhood EPN, with tumors located distinctly in the supratentorial region of the central nervous system (CNS). We hypothesized that specific chromatin modules accessible during brain development would render distinct cell lineage programs at direct risk of transformation by ZR.



 To this end, we performed combined single nucleus ATAC and RNA (snMultiome) sequencing of the developing mouse forebrain, as compared to ZR-driven mouse and human EPN.



 We demonstrate that specific developmental lineage programs present in transient progenitor cells and regulated by PLAG/L family transcription factors (TF) are at risk of neoplastic transformation. Binding of this chromatin network by ZR or other PLAG/L family motif targeting fusion oncoproteins lead to persistent chromatin accessibility at oncogenic loci and oncogene expression. Cross-species analysis of mouse and human ZR EPN reveals significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling progenitor-like or radial glial-like cells that establish a putative tumor cell hierarchy. In vivo lineage tracing studies reveal neoplastic clones that aggressively dominate tumor growth and establish the entire EPN cellular hierarchy.



 These findings unravel developmental epigenomic states critical for fusion oncoprotein-driven transformation and elucidate how these states continue to shape tumor progression.



 Alisha Kardian, Hua Sun, Stephen Mack. Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr LT004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/885b82368395c60d63823ec56b4ae77477004d30" target='_blank'>
              Abstract LT004: Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development
              </a>
            </td>
          <td>
            A. Kardian, Hua Sun, S. Mack
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a27e17b141d07dfd3357f3ee906c1f3a2792dd3c" target='_blank'>
              SigDyn: single-cell mutational signature dynamics
              </a>
            </td>
          <td>
            S. Costa, Joana P. Gonçalves
          </td>
          <td>2026-01-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction The APOBEC3 family of cytidine deaminases induces somatic mutations that are highly prevalent in cancers, but the functional consequences remain largely unknown. Methods To determine these consequences, we exposed MCF7 tumorigenic breast epithelial cells to APOBEC3A, APOBEC3B or APOBEC3H Haplotype I. Results Comparative analysis between cells pre and post ‐APOBEC3 exposure revealed fewer deamination‐dependent γH2AX foci post‐APOBEC3 exposure, despite maintaining APOBEC3 protein expression. In a mouse xenograft model, high expressing, but not low expressing APOBEC3A‐exposed cells caused increased tumor progression. In contrast, high expressing, but not low expressing APOBEC3B‐exposed cells decreased tumor size. APOBEC3H Haplotype I‐exposed cells stochastically increased tumor progression independent of expression levels. Consistent with tumor data, RNA‐seq showed upregulation of tumor enhancing pathways only in cells that enhanced tumor progression. Discussion The results indicate that in a breast cancer xenograft model, APOBEC3A and APOBEC3H Haplotype I are more likely to contribute to enhanced tumor progression than APOBEC3B.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab8cfe2468bc6ac6d9c493b9b25b352fc47d1037" target='_blank'>
              Differential effects on tumor progression by APOBEC3A, APOBEC3B, and APOBEC3H Haplotype I in a breast cancer mouse xenograft model
              </a>
            </td>
          <td>
            Milaid Granadillo Rodríguez, Lai Wong, Arzhang Shayeganmehr, Diogo Pellegrina, F. Vizeacoumar, Franco J. Vizeacoumar, Mohamed Helmy, L. Chelico
          </td>
          <td>2026-01-28</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Somatic structural variations (SVs) are common in cancer. Although a small fraction of SVs in breast and ovarian cancers can be attributed to homologous recombination deficiency, the underlying molecular mechanisms for the vast majority of somatic SVs remain unclear. Here, we focus on the roles of transcription and DNA replication collisions in genomic instability in cancer. Such collisions are unavoidable in cells since both transcription and replication use the same DNA as template. We hypothesized that transcription replication collisions (TRCs), if not properly repaired, would lead to collapsed replication forks and result in SVs. To this end, we studied somatic SVs in 6193 high-coverage whole-genome sequenced primary and metastatic tumors from three independent pan-cancer cohorts. A total of 13 conserved SV signatures, representing independent molecular mechanisms, were deconvoluted from these cohorts using non-negative matrix factorization approach. We detected replicated-strand bias, the expected footprint of transcription-replication collision, in large tandem duplications (TDs) across multiple cohorts. This bias was only observed in expressed genes, consistent with TRCs depending on transcription activity. Large TDs were abundant in female-specific (breast, ovarian and uterus), upper gastric-intestinal tract, and prostate cancers. They were associated with CDK12 mutations and worse patient survival. CDK12 is a cyclin-dependent kinase, a key regulator of transcription elongation and termination. Deleting or suppressing CDK12 using CRISPR-Cas9 in prostate cell lines increased RNA:DNA hybrids (R-loops), promoted TRCs, and ultimately led to large TDs. Finally, we found that cells lacking CDK12 were sensitive to WEE1, CHK1 and ATR inhibitors. In summary, our data suggest that large TDs in cancer form due to impaired TRC repair and can be used as a biomarker for prognosis and treatment.



 Lixing Yang. A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A078.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a771b9c6597f1781d70e2b6d5df9ff54196c4d85" target='_blank'>
              Abstract A078: Transcription and DNA replication collisions lead to large tandem duplications and expose targetable therapeutic vulnerabilities in CDK12-mutant prostate cancer
              </a>
            </td>
          <td>
            Lixing Yang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9b6108bbfc8477081a30cd15de6db71d5ddd4a1" target='_blank'>
              Exploiting synthetic lethality in epithelial ovarian cancer: multi-dimensional approaches beyond DNA damage repair.
              </a>
            </td>
          <td>
            Mandana Bigdeli, Élizabeth Tremblay, Diane Provencher, A. Mes-Masson, Francis Rodier
          </td>
          <td>2026-02-10</td>
          <td>Molecular cancer</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b31122c56712ad14285c46f7cc089d3bb75901b" target='_blank'>
              Replication timing uncovers a two-compartment nuclear architecture of interphase euchromatin.
              </a>
            </td>
          <td>
            H. Akram, E. Wear, L. Mickelson-Young, Z. Turpin, L. Hanley-Bowdoin, W. F. Thompson, L. Concia, H. Bass
          </td>
          <td>2026-02-20</td>
          <td>The Plant cell</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="The field of epitranscriptomics discovered N6-methyladenosine (m6A) modifications, which function as fundamental elements that control RNA metabolism properties that powerfully affect cancer biology. This review examines the way m6A modifications shape RNA stability while regulating translation, together with their eraser and reader proteins. We demonstrate that m6A modifications guide oncogene and tumor suppressor transcript outcomes, which promote tumor growth, metastasis, and therapeutic resistance. The regulatory function of m6A depends significantly on its relationship with ncRNAs that mainly include miRNAs, lncRNAs, and circRNAs. The review examines the effects of m6A on ncRNA production, stability, export, and degradation, as well as the regulation of m6A protein expression by ncRNAs, highlighting intricate reciprocal feedback loops that drive cancer progression. The interplay between m6A RNA modifications and ncRNAs provides emerging evidence on how they collectively influence the tumor microenvironment, modulate immune system responses, and contribute to resistance. Harnessing ncRNA-m6A interactions for managing drug resistance offers promising therapeutic avenues. However, advancing our understanding of the context-specific roles of m6A modifications and translating these insights into clinical applications remains a significant challenge. This review synthesizes recent findings on ncRNA-m6A crosstalk to lay the groundwork for developing epitranscriptomic strategies in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0e0eb5d331194e0274cc28e14543827ba3e7f5f" target='_blank'>
              The epitranscriptome meets non-coding RNA: m6A-mediated regulation in oncogenesis and therapy
              </a>
            </td>
          <td>
            Prasanna Srinivasan Ramalingam, M. Rejili, Faouzi Haouala, Md Sadique Hussain, Yumna Khan, Mudasir Maqbool, Janaki Ramaiah Mekala, S. Arumugam
          </td>
          <td>2026-01-13</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="A fundamental challenge in studying therapy resistance is understanding whether it results from pre-existing cellular states (“priming”) or drug-induced changes (“adaptation”). While lineage barcoding enables retrospective analysis of cells before and after treatment, current methods struggle to efficiently capture rare lineages in single-cell RNA sequencing (scRNA-seq) or isolate multiple specific lineages simultaneously for functional study. To overcome these limitations, we developed CloneSweeper, a multiplexed lineage tracking platform that pools enrichment libraries to isolate or enrich multiple rare lineages. CloneSweeper utilizes a dual-function barcode expressed as both a Cas9 gRNA for live-cell sorting and a 3’ UTR transcript for high-recovery detection in 10x Genomics scRNA-seq. We applied CloneSweeper to a model of BRAF V600E melanoma, where we identified that resistance to targeted therapy emerges from a polyclonal population of rare, pre-existing lineages. By simultaneously targeting and enriching 21 distinct rare lineages prior to treatment, we defined a heritable, primed state characterized by de-differentiation and elevated mesenchymal markers. We demonstrate that these primed cells are not quiescent but instead exhibit upregulated inflammatory and stress response signaling, specifically via the AP-1 and NF-κB1 pathways. CloneSweeper thus provides a robust framework for dissecting the molecular mechanisms of rare biological phenomena through simultaneous, multiplexed lineage isolation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cfe1b1a36a95588a97f5cc3024b429ba771c00" target='_blank'>
              Multiplexed enrichment and tracking of lineages with CloneSweeper
              </a>
            </td>
          <td>
            Robert J. Vander Velde, Raymond W. S. Ng, Christopher Coté, Sydney M. Shaffer
          </td>
          <td>2026-01-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The tumor suppressor RAD51D is essential for homologous recombination (HR). Pathogenic variants in RAD51D are associated with breast and ovarian cancers. However, most clinical missense variants are of unknown significance. We performed a multiplex assay of variant effect to test 6,888 RAD51D coding variants for loss-of-function. The resulting variant-to-function map perfectly separates known pathogenic and benign variants and is validated by orthogonal HR and biochemical assays across 70 clinical variants. Our screen shows that variants in the DNA-binding or ATPase core most severely compromise HR, and we identify the RAD51D-RAD51C interface within the BCDX2 complex as essential for regulating its ATPase activity. We hypothesize that, paradoxically, the primary function of RAD51D is to slow the ATPase activity of BCDX2, thereby allowing sufficient time and space for RAD51 filament assembly. Together, we identify hotspots of deleterious RAD51D variants and uncover the mechanisms by which variants compromise its biochemical functions. Highlights Used a multiplexed assay of functional effect (MAVE) to assess the functionality via olaparib sensitivity of 6,888 RAD51D coding variants, which can be used for variant classification Provided cellular functional analysis for 70 clinically-identified breast and ovarian cancer RAD51D variants Identified key regions and enzymatic activities of RAD51D critical for its function in the BCDX2 and the X3CDX2 complexes Determined mechanism of RAD51D-mediated regulation of BCDX2 ATPase activity">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b147f7b191823354d4eee4bfd6f57b8ab44617ab" target='_blank'>
              High-throughput mapping of 6,888 RAD51D variants identifies distinct biochemical functions needed for homologous recombination and olaparib response
              </a>
            </td>
          <td>
            Kristie E. Darrah, Shelby L. Hemker, Yashpal Rawal, Noah J. Goff, Phoebe S. Parker, Gayatri Ganesan, Caleb M. Stratton, Katherine Oppenheimer, Ella Roberts, Elena Glick, Nicole Banks, Arjun Kumar, Silvia Casadei, Matthew W. Snyder, Katherine L Nathanson, S. Domchek, Lea M. Starita, Shaun K. Olsen, Patrick Sung, J. Kitzman, Kara A. Bernstein
          </td>
          <td>2026-01-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>106</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a19852969bd81baff3364e7db6ef026b54ff18d9" target='_blank'>
              Circulating extrachromosomal circular DNA as a prognostic biomarker for colorectal cancer.
              </a>
            </td>
          <td>
            Xuanmei Luo, Jian Cui, Jinxin Shi, Gaoyuan Sun, Lili Zhang, Yayu Li, Yingyu Guo, Lu Kuai, Tianhan Sun, Qi Luo, Jiahui Cai, Q. An, Wei Zhang, Fei Xiao, Gang Zhao
          </td>
          <td>2026-02-07</td>
          <td>Cell communication and signaling : CCS</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2770d616654ff2fea428335622c1756448d7d587" target='_blank'>
              APE1 Coordinates Its Disordered Region and Metal Cofactors to Drive Genome Surveillance
              </a>
            </td>
          <td>
            Donghun Lee, Subin Kim, Gyeongpil Jo, Juwon Kim, Jungmin Yoo, Jejoong Yoo, Ja Yil Lee, Gwangrog Lee
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Breast cancer is the most commonly diagnosed type of cancer worldwide in women. Estrogen receptor alpha (ERα), a well-studied transcription factor, is the major oncogene in over 70% of breast cancer cases. The majority of earlier studies have primarily focused on ERα transcriptional activity that promotes tumor progression. We discovered that ERα is a non-canonical RNA-binding protein that associates with over 1,000 mRNAs, mainly through their 3’-untranslated regions (UTRs), in breast cancer cells. Interestingly, recent studies have shown that a broad spectrum of transcription factors (TFs), including ERα, directly bind RNA; yet, how this association directly orchestrates gene expression remains largely unexplored. This led us to ask an outstanding question about the functional role of oncogenic ERα on RNA metabolism in breast cancer. Since the 3’-UTR is a molecular hub where cis and trans-regulatory elements converge to determine mRNA fate, we hypothesized that oncogenic ERα regulates the stability of a set of mRNAs by binding to its 3’-UTR, which in turn affects the downstream gene expression program that promotes breast cancer progression. To address this question, we performed SLAM-seq, a genome-wide approach for characterizing mRNA stability in MCF7 WT cells and MCF7 cells harboring an ER⍺ RNA binding domain mutation (ER⍺ RBDM). We found that the stability of a subset of ERα mRNA targets is decreased upon loss of ERα RNA-binding activity. We then focused on a critical oncogenic mRNA, NSUN2, an enzyme that catalyzes the 5-methylcytosine (m5C) modification on mRNA, which is an ERα target. NSUN2 is a m5C writer protein, a well-studied RNA methyltransferase enzyme, and is significantly upregulated in breast cancer. Interestingly, survival analysis shows that high NSUN2 correlates with poor prognosis in ER-positive breast cancer patients, but not in ER-negative patients. We show that ER⍺ WT strongly binds to the 3’UTR of NSUN2 mRNA, whereas the ER⍺ RNA binding mutant (ER⍺ RBDM) doesn’t bind to NSUN2 3’UTR. Strikingly, we observed that the loss of ERα RNA binding destabilizes NSUN2 mRNA and protein levels. Furthermore, we found that global RNA m5C methylation is significantly downregulated in breast cancer cells expressing the ERα RBD. Notably, ERα RBDM cells are impaired in forming tumors in vivo, and this phenotype is rescued by overexpressing NSUN2. This suggests the existence of ERα-mediated epitranscriptomic marks via NSUN2 in breast cancer progression.



 N. Mohan, A. Dabrowska, V. Subramanyam, I. Liu, D. Kuzuoglu-Öztürk, D. Ruggero. Oncogenic er alpha as an rna-binding protein controls mrna stability and epitranscriptome of breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2025; 2025 Dec 9-12; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2026;32(4 Suppl):Abstract nr PS2-11-26.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac9211fe0f94e6ea6753b3dab513d5e54983725d" target='_blank'>
              Abstract PS2-11-26: Oncogenic er alpha as an rna-binding protein controls mrna stability and epitranscriptome of breast cancer
              </a>
            </td>
          <td>
            N. Mohan, A. Dąbrowska, V. Subramanyam, I. Liu, D. Kuzuoğlu-Öztürk, D. Ruggero
          </td>
          <td>2026-02-17</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="High-grade serous ovarian carcinoma (HGSOC) is characterised by profound genomic instability and limited durable responses to standard therapy, leading to poor prognosis. The use of next-generation sequencing technologies has improved understanding of its molecular landscape, revealing consistent Tumour Protein p53 (TP53) mutations, homologous recombination defects, pathway alterations, and epigenetic dysregulation. Such genomic profiling now underpins the classification criteria between the ovarian cancer subtypes described by the Cancer Genome Atlas. Widespread chromosomal instability and pathogenic variants in multiple genes distinguish HGSOC from other subtypes of ovarian cancer and, further, from low-grade serous ovarian cancer. Importantly, the new-found understanding of the genomic landscape of HGSOC guides the use of platinum-based chemotherapies and Poly(ADP-ribose) Polymerase (PARP) inhibitors, with homologous recombination deficiency emerging as a cancer vulnerability that enhances treatment response. A combined multi-omics approach integrates transcriptomics, proteomics, metabolomics, and epigenomics to further the understanding of the characteristics, therapeutic targets and treatment resistance within HGSOC. Despite these advances, major challenges persist, including intratumoural heterogeneity and the poor diversity of genomic datasets. Artificial Intelligence (AI) technology, Clustered regularly interspaced short palindromic repeats (CRISPR)-based gene editing, neoantigen-guided immunotherapy and ovarian cancer vaccination indicate a promising future for genomics-guided interventions and support the integration of genomics within multi-omic approaches to improve HGSOC outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63375259d1ee4ee2c082f84da6b555791abc91c8" target='_blank'>
              High-Grade Serous Ovarian Carcinoma in the Genomics Era: Current Applications, Challenges and Future Directions
              </a>
            </td>
          <td>
            M. Lewis, C. Caricato, Hannah Roberts, Subhasheenee Ganesan, Nadia Amel Seksaf, E. Maniati, Michail Sideris
          </td>
          <td>2026-02-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="DNA-protein crosslinks (DPCs) are highly toxic DNA lesions that arise both from normal cellular metabolism and as an intended consequence of cancer chemotherapy. Key anticancer agents, including topoisomerase poisons and PARP inhibitors, exert their therapeutic effects by trapping enzymes on DNA, converting them into toxic barriers that block replication. To counteract this threat, cells have evolved specialized mechanisms to detect and remove DPCs. This review explores the molecular mechanisms by which these therapies trap proteins on DNA and the multi-layered defense systems cells use to resolve them-ranging from enzymatic degradation to mechanical extraction. We further examine how these processes are modulated by the cell cycle and chromatin landscape. Importantly, we highlight emerging evidence that alterations in DPC repair pathways are frequent in cancer and serve as critical determinants of treatment response. Ultimately, this review integrates mechanistic insights with clinical data to highlight how exploiting DPC repair defects can overcome drug resistance and guide the development of rational, synthetic lethal combination therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7911590556e8eee40c77772227dc1932d925b21a" target='_blank'>
              Cellular strategies for repairing trapped protein-DNA complexes
              </a>
            </td>
          <td>
            Maria Sideridou, Doukissa Ioanna Machli, Dora Lontra, Theodoros Rampias
          </td>
          <td>2026-02-10</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Sarcomas, malignant neoplasms of mesenchymal origin, comprise more than 100 entities and numerous histologic subtypes. Their relative rarity compared with carcinomas, interpatient diversity, and pronounced intratumor heterogeneity (ITH) make sarcomas clinically challenging. Yet, they also provide excellent models for studying the molecular and cellular underpinnings of ITH and how it translates into diverse clinical outcomes. New single-cell and spatial omics technologies have revealed the complex architecture of sarcomas with unprecedented resolution. In this review, we use five representative entities - two genomically complex (osteosarcoma, dedifferentiated liposarcoma) and three fusion-driven, genomically "quieter" (Ewing sarcoma, synovial sarcoma, alveolar rhabdomyosarcoma) sarcomas - to showcase recent advances in the field of ITH and the temporal evolution of sarcomas. Specifically, we review how clonal and subclonal molecular alterations, fluctuations in oncogene activity and other forms of cellular plasticity, as well as the formation of complex intratumor ecosystems, drive sarcoma progression and shape therapy resistance. We also provide an overview of recent advances in modeling ITH in sarcomas, as well as in evaluating functional diversity and assessing patient-specific treatment responses. Finally, we outline future directions in ITH research and its clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d649eb6814167f528f5b4d6a4f56100c96ce1e38" target='_blank'>
              Intratumor Heterogeneity and Clonal Evolution in Sarcomas: Molecular, Cellular, and Functional Drivers of Therapy Resistance.
              </a>
            </td>
          <td>
            T. Grünewald, Ina Oehme, Johanna Wagner, S. Fröhling
          </td>
          <td>2026-02-01</td>
          <td>Seminars in cancer biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Fusion oncogenes drive malignant transformation by altering transcriptional programs, disrupting cellular homeostasis, and frequently rewiring protein condensate networks. Emerging evidence suggests that aberrant condensate formation is a key mechanism through which these fusion proteins exert oncogenic control. Specifically, by amplifying transcriptional output, sequestering cofactors, and enforcing pathological gene expression states. Despite their genetic validation as oncogenic drivers, many fusion proteins remain pharmacologically intractable due to the absence of catalytic domains or well-defined binding pockets. To overcome this, we developed an integrated discovery platform that directly assesses the biophysical and functional properties of fusion protein-driven condensates and enables identification of small molecules that disrupt these oncogenic assemblies. Using protein language models and in silico predictors of phase separation, we systematically mapped condensate-forming potential across recurrent oncogenic fusions catalogued in TCGA. This analysis revealed that most fusion proteins (including EML4-ALK and EWS-FLI1) harbor extensive intrinsically disordered regions (IDRs) that promote abnormal condensation and transcriptional dysregulation. We further identified disease-specific variants that alter condensate formation relative to wild-type proteins, providing mechanistic insight into oncogenic gain- and loss-of-function events. To experimentally validate and therapeutically exploit these findings, we established PhaseScan™, a high-throughput droplet microfluidics screening platform capable of quantitatively profiling condensate dynamics and screening thousands of compounds against reconstituted fusion condensates to identify therapeutically relevant small molecule inhibitors. Across multiple fusion proteins, PhaseScan™ identified small molecules that selectively disrupt aberrant condensates while sparing physiological assemblies such as nuclear speckles and stress granules. Using EML4-ALK as a representative model, we demonstrated that compound-mediated condensate dissolution disrupts oncogenic ALK signaling and reverses oncogenic phenotypes in cell-based systems. Our results highlight condensate biology as a generalizable vulnerability across fusion-driven cancers and demonstrate the feasibility of discovering small molecules against fusion proteins traditionally considered undruggable. By integrating AI-driven predictive modeling, and on-target molecular high-throughput screening, our platform provides a roadmap for rationally identifying and drugging fusion protein condensates. This approach establishes condensate-targeted therapeutics as a novel class of precision medicines for patients with fusion-positive malignancies.



 Andrew Seeber, William E. Arter, Seema Qamar, Cinzia Sgambato, Amal Alex, Kadi Saar, Julia Doh, Martin Kulander, Tuomas Knowles, Shilpi Arora. Targeting aberrant condensate formation in fusion-positive cancers with an integrated discovery platform [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr PR004.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee0c93f691d35862b44375fdee303e25769c39e0" target='_blank'>
              Abstract PR004: Targeting aberrant condensate formation in fusion-positive cancers with an integrated discovery platform
              </a>
            </td>
          <td>
            A. Seeber, W. Arter, S. Qamar, Cinzia Sgambato, Amal Alex, K. Saar, Julia Doh, M. Kulander, Tuomas Knowles, Shilpi Arora
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Fusion-positive rhabdomyosarcoma (FP-RMS), an aggressive pediatric soft tissue cancer, is driven by the oncogenic fusion transcription factors PAX3::FOXO1 (P3F) or PAX7::FOXO1. In a subset of FP-RMS cases, amplification of the MIR17HG locus leads to overexpression of the oncogenic miR-17–92 cluster. However, miR-17–92 is also upregulated in a subset of FP-RMS tumors lacking this amplification, suggesting the involvement of alternative regulatory mechanisms. Here, we show that P3F and MYCN cooperatively activate MIR17HG transcription in FP-RMS. CRISPR/Cas9-mediated knockout of P3F or MYCN in FP-RMS cell lines significantly reduced miR-17–92 expression. In a human myoblast model with inducible P3F expression, MYCN alone induced modest MIR17HG transcription, whereas co-expression with P3F led to robust activation of the cluster and acquisition of oncogenic phenotypes. Chromatin immunoprecipitation sequencing (ChIP-seq) revealed a P3F binding site located 1.84 Mb upstream of the MIR17HG promoter. CRISPR-mediated deletion of this site markedly reduced MIR17HG expression and impaired transformation in the myoblast system. Functional inhibition of mature miR-17–92 members using miRNA sponges also suppressed proliferation and transformation in FP-RMS cells. In vitro studies showed that miR-17–92 is necessary but not sufficient for transformation, highlighting its role as a key downstream effector of P3F and MYCN. Together, these findings define a cooperative transcriptional axis where P3F and MYCN activate MIR17HG through a distal DNA binding element, thereby revealing a novel therapeutic vulnerability in FP-RMS.



 Shabir Zargar, Pawam Raut, Hana Kim, Rachel Hoffman, Benjamin Z. Stanton, Frederic G. Barr. Oncogenic MIR17HG Expression Is Transcriptionally Regulated by PAX3::FOXO1 and MYCN in Fusion-Positive Rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B029.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2cec2c5fd391552299369bfa86ed2e2816c5ee0" target='_blank'>
              Abstract B029: Oncogenic MIR17HG Expression Is Transcriptionally Regulated by PAX3::FOXO1 and MYCN in Fusion-Positive Rhabdomyosarcoma
              </a>
            </td>
          <td>
            Shabir Zargar, Pawam Raut, Hana Kim, Rachel A. Hoffman, Benjamin Z. Stanton, Frederic G. Barr
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) in plasma consists of short DNA fragments resulting from a non-random fragmentation process, with distinct fragmentomic characteristics that are related with their cellular origins. Here, we report that somatic variant signatures in cfDNA markedly differ between non-cancerous controls and cancer patients, indicating that tumor-associated signals are retained in these variants. Surprisingly, even in controls, cfDNA molecules harboring somatic variants exhibit cancer-like fragmentomic characteristics, such as reduced size, decreased DNA methylation, and altered end motif usages and distributions in the nucleosome structure. Further investigations suggest that such cancer-like traits are associated with somatic variants derived from clonal hematopoiesis. Importantly, these somatic variants-associated fragmentomic aberrations are more pronounced in cancer patients, enabling cancer diagnosis. In a large pan-cancer cohort, we utilize AI to integrate genomic, fragmentomic, and epigenomic features to develop diagnostic models named FreeSV and FreeSV+. Leveraging somatic variant-associated features alone, the FreeSV model achieved area under the ROC curves (AUCs) between 0.81-0.92 across cancer types; however, when genomewide features are also included, the AUCs of FreeSV+ model substantially increased to 0.93-0.99 across cancer types, highlighting the significance of integrative genomic and fragmentomic analyses in cfDNA for cancer liquid biopsy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07d931b70db2d8495e7d9b12e6b1f59f47b890a1" target='_blank'>
              Cancer-Like Fragmentomic Characteristics of Somatic Variants in Cell-Free DNA.
              </a>
            </td>
          <td>
            Zhenyu Zhang, Yunyun An, Mengqi Yang, Yuqi Pan, Xiaoyi Liu, Fanglei Gong, Huizhen Lin, Bianbian Tang, Yunxia Bai, Xin Zhao, Yu Zhao, Changzheng Du, Kun Sun
          </td>
          <td>2026-01-22</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Long non-coding RNA (lncRNA) DLEU1 has been implicated in tumorigenesis, yet its mechanistic role in gastric cancer (GC) remains elusive. We investigated the epigenetic regulation and oncogenic function of DLEU1 in GC through chromatin immunoprecipitation, RNA-protein interaction assays, and functional analyses in organoids and xenograft models. The molecular mechanisms underlying DLEU1-mediated DNA repair and metabolic adaptation were elucidated using western blotting, quantitative RT-PCR, and luciferase reporter assays. DLEU1 was significantly upregulated in GC, driven by H3K27 acetylation and H3K4 methylation. Mechanistically, DLEU1 promoted DNA repair by facilitating ASCC2 nuclear translocation and its interaction with ALKBH3, thereby stabilizing E2F1 mRNA. In turn, E2F1 directly activated G6PD transcription, leading to enhanced NADPH production, redox homeostasis, and glucose metabolism. Functionally, co-targeting DLEU1 and ASCC2 synergized with G6PD inhibition, significantly impairing GC cells viability and tumor growth. Our findings establish DLEU1 as a key oncogenic lncRNA in GC, orchestrating DNA repair, redox balance, and metabolic adaptation via the ASCC2-ALKBH3-E2F1-G6PD axis. Targeting this pathway may provide a promising therapeutic strategy for overcome GC chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d35a8302c655b7d3e300cd1133d4afc96839a284" target='_blank'>
              Histone modification-regulated LncRNA DLEU1 interacts with ASCC2/ALKBH3 complex to drive DNA repair, antioxidant homeostasis and glucose metabolism in gastric cancer
              </a>
            </td>
          <td>
            Xiaoyang Zhang, Xin Wang, Qi Wang, Xu Wang, Hui Sun, Yingxue Liu, C. Tan, S. Ni, Weiwei Weng, Meng Zhang, Lei Wang, Dan Huang, Jie Chen, Xiaoyu Wang, Lu Gan, Mierxiati Abudurexiti, Wenfeng Wang, Jinjia Chang, Weiqi Sheng, Midie Xu
          </td>
          <td>2026-01-02</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background Lung cancer remains the leading cause of cancer−related mortality. Single−cell RNA sequencing (scRNA−seq) enables high−resolution mapping of tumor microenvironment, but how malignant cell states connect to patient outcomes and therapeutic vulnerabilities is not fully understood. Methods We analyzed scRNA−seq profiles from 13 primary lung tumors and matched normal lung tissues (48,149 cells). Malignant versus non−malignant epithelial cells were defined by inferCNV, co−expression programs resolved with Hotspot, transcription factor regulons inferred by pySCENIC, and differentiation potential estimated by CytoTRACE and diffusion pseudotime. Key malignant programs were cross−checked in an independent LUAD scRNA−seq cohort (GSE131907). Program signatures were projected into TCGA−LUAD to build an eight−gene LASSO−Cox risk model, which was further validated in two external NSCLC/LUAD cohorts (GSE30219 and GSE31210). Immune contexture was inferred by CIBERSORT and TIDE, drug sensitivities predicted with pRRophetic, and the function of FAM189A2 tested by knockdown and overexpression in A549 and NCI−H23 cells. Results We delineated 12 major cell populations and six malignant epithelial subclusters, including an early GPRC5A+ subpopulation with high developmental potential. Malignant programs corresponding to cell−cycle and inflammatory states (Modules 8, 14 and 16) were associated with worse overall survival, whereas Module 15, preferentially active in the GPRC5A+ state, was linked to better outcomes. An eight−gene signature (LAMA5, ACTB, B4GALT1, KLF5, KRT18, FAM189A2, SLC34A2, S100A11) robustly stratified patients into high− and low−risk groups across TCGA−LUAD and two validation cohorts. High−risk tumors displayed an immune−enriched yet matrix−restrained microenvironment with higher CD8+ T cells, whereas low−risk tumors showed greater predicted sensitivity to Aurora kinase, IGF−1R, mTOR and TGF−β inhibition. FAM189A2 was down−regulated in tumors, higher expression predicted better survival, and bidirectional perturbation in A549 and H23 cells demonstrated that loss of FAM189A2 enhanced, while overexpression attenuated, migration and invasion in vitro. Conclusions Integrating single−cell and bulk transcriptomes links malignant epithelial state programs to prognosis, yields a practical eight−gene risk model validated in multiple LUAD cohorts, and nominates FAM189A2 as a putative tumor suppressor and potential biomarker in lung cancer. These findings suggest testable strategies for risk stratification and therapy selection that warrant prospective evaluation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9149373826a7f4a30a6edb092024413c1a5945d" target='_blank'>
              Integration of single cell and bulk transcriptomic analyses identifies FAM189A2 as a key prognostic gene in lung cancer
              </a>
            </td>
          <td>
            Peng Guo, Mengqi Xiang, Jing Chen, Huachuan Zhang
          </td>
          <td>2026-01-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Breast cancer is one of the most severe global health challenges, with its incidence continuing to rise. The development of this malignancy is associated with numerous risk factors; however, its primary etiology often remains unclear. Research on the potential association between breast cancer and carcinogenic human papillomavirus (HPV) has been conducted for three decades, yet a definitive consensus has not been reached. The development of prophylactic cervical cancer vaccines has greatly heightened the urgency of this issue: if a causal relationship can be established, it would open a feasible avenue for preventing this common cancer. This review aims to systematically integrate relevant evidence and provide a critical commentary on the association between HPV and breast cancer. We analyzed evidence from 34 studies on HPV DNA detection in breast cancer cells. We discuss the prevalence and genotypes of HPV detected in breast cancer across different geographical regions, the state of the viral genome (integrated vs. episomal), viral load, and potential mechanisms for HPV-associated breast carcinogenesis—particularly its role in inducing genomic instability via activation of APOBEC3B. Additionally, we explore the histopathological and clinical characteristics of HPV-positive breast cancer. The synthesized evidence suggests that high-risk HPV types are present in a subset of breast cancers worldwide, but with lower viral loads compared to cervical cancer, indicating a potentially different mode of action. The association between HPV and breast cancer warrants further rigorous investigation to clarify its clinical and preventive implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91c5a2843a4cdacf97e3a7acb855e5ff458bbd2b" target='_blank'>
              HPV, APOBEC3B, and the origins of breast cancer: a narrative review and perspectives on novel mechanisms
              </a>
            </td>
          <td>
            Zhiyong Liu, Ran Chen
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 The CBFA2T3–GLIS2 (C/G) fusion defines an exceptionally aggressive, epigenetically driven subtype of pediatric AML (pAML) confined to infants (<3 years). Arising from a pericentric inversion of chromosome 16, it is the most frequent lesion in acute megakaryoblastic leukemia (AMKL; ∼30%) and also occurs in non-AMKL subtypes, indicating lineage-independent pathogenicity. Clinically, C/G+ leukemia shows relapse rates > 90% and < 15% five-year survival, making it among the most treatment-refractory pediatric cancers. Lacking cooperating mutations, C/G acts as a singular oncogenic driver that reprograms the enhancer landscape, yet how it sustains epigenetic programs enforcing leukemic identity and chemoresistance remains unknown. We integrated multi-omic and functional epigenetic profiling across patient samples, C/G+ AML lines, and a developmentally faithful cord-blood CD34+ HSPC co-culture model. DNA methylation (DNAm) was mapped by meEM-seq, chromatin states by CUT&RUN and ATAC-seq, and transcription by RNA-seq. Functional studies included DNMT3B knockout, gRNA-resistant rescue, and inducible dCas9–TET1 demethylation, complemented by promoter-capture Hi-C and C/G+ xenograft validation. C/G+ pAML exhibited a promoter-biased hypermethylation signature (Δβ ≥ 0.2 vs NBM), with ∼50% of altered CpGs at promoters. The fusion bound the DNMT3B promoter, inducing its transcription without global 5mC gain but driving focal hypermethylation at active promoters linked via long-range enhancer loops. These hypermethylated promoters paradoxically stabilized expression of CRE-residing genes (BMP2, MED12, ITGB2) enriched for EMT, KRAS, heme-metabolism, and apoptotic pathways. In the fetal C/G–UCB-HSPC model, these signatures were faithfully recapitulated, confirming that the fusion alone reprograms the fetal hematopoietic epigenome. Motif analysis identified E2F2 and E2F4 as transcription factors stabilizing methylated yet transcriptionally active CREs. DNMT3B loss reduced locus-specific 5mC and target-gene expression but upregulated DNMT1 and UHRF1, suggesting a compensatory maintenance shunt that preserves DNAm at apoptotic loci and raises the apoptotic threshold. Targeted demethylation of PMAIP1 (NOXA) restored its transcription and apoptosis, confirming that DNAm stabilizes but constrains inducibility of pro-apoptotic genes. In vivo, DNMT3B-deficient xenografts regained Venetoclax sensitivity and extended survival, phenocopying DNMT inhibition (DNMTi). This study identifies C/G as an epigenetic architect converting DNAm from a repressive to a stabilizing signal, locking leukemic enhancers into an active yet apoptosis-resistant state. By defining the C/G–DNMT3B–DNMT1/UHRF1 axis and transcription-factor-mediated maintenance of methylated CREs, we reveal a unified mechanism linking developmental origin, transcriptional persistence, and therapy resistance in pAML. These findings position aberrant DNAm as a druggable epigenetic vulnerability and provide the mechanistic rationale for DNMTi + BCL-2 blockade in high-risk, treatment-refractory pAML.



 Samrat Roy Choudhury, Arundhati Chavan, Rhonda E. Ries, Giselle Almeida Gonzalez, Soheil Meshinchi, Jason Farrar. CBFA2T3–GLIS2 fusion reprograms enhancer-linked DNA methylation to enforce apoptotic resistance and defines an epigenetic vulnerability in pediatric AML [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr PR006.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e01e458a3959338b14b992ffa07b3d02aa22cbfe" target='_blank'>
              Abstract PR006: CBFA2T3–GLIS2 fusion reprograms enhancer-linked DNA methylation to enforce apoptotic resistance and defines an epigenetic vulnerability in pediatric AML
              </a>
            </td>
          <td>
            S. R. Choudhury, Arundhati Chavan, R. Ries, Giselle Almeida Gonzalez, S. Meshinchi, Jason Farrar
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9433706dc45908c524cf9721057f38db3f2b4ec4" target='_blank'>
              Oncogenic p53 induces mitotic errors in lung cancer cells by recopying DNA replication forks conferring targetable proliferation advantage.
              </a>
            </td>
          <td>
            Shilpa Singh, Lilia Gheghiani, Brandon Velasco, R. Frum, Steven R Grossman, Brad Windle, S. Deb, S. Deb
          </td>
          <td>2026-01-26</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract The development of biomarkers for population screening, early cancer detection, monitoring, and recurrence surveillance offers substantial potential to improve patient outcomes and save lives. Nullomers are short k-mers that are absent from a human genome, and neomers are the subset of nullomers that emerge recurrently due to somatic mutations during cancer development. Here, we have developed neomerDB, a database that encompasses a catalogue of neomers across cancer types and organs. We examined 10 000 whole exome sequencing and 2658 whole genome sequencing tumour-matched samples and identified the set of neomers associated with each cancer type and organ. We also analysed 76 215 whole genomes and 730 947 whole exomes of individuals from diverse ancestries, from which we removed nullomers and neomers that can arise due to germline variants in the population. Finally, we conducted a case study demonstrating that neomers can be utilized to detect glioblastoma from liquid biopsy samples (n = 38), utilizing cell-free DNA and cell-free RNA, achieving a Receiver Operating Characteristic - Area Under the Curve score of 0.98 and a precision-recall score of 0.99. neomerDB is a user-friendly database that enables advanced searches, provides interactive visualizations, and download options for neomer biomarkers. neomerDB is publicly available at https://neomerDB.com/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d61e31009fa3725049e1e6c9f385fb155746c12" target='_blank'>
              neomerDB: a comprehensive database of neomer biomarkers in cancer
              </a>
            </td>
          <td>
            Kimonas Provatas, Candace S. Y. Chan, Ioannis Kerasiotis, Eleftherios Bochalis, Akshatha Nayak, Brad E Zacharia, Georgios A. Pavlopoulos, Wei Li, IIias Georgakopoulos-Soares
          </td>
          <td>2026-01-15</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Epigenetic alterations, particularly DNA methylation, and dysregulation of the tumor immune microenvironment (TIME) are increasingly recognized as critical factors in esophageal squamous cell carcinoma (ESCC) pathogenesis. Understanding the dynamic interplay between DNA methylation changes and TIME evolution during ESCC progression remains essential. We established a 4-nitroquinoline 1-oxide (4NQO)-induced ESCC mouse model, capturing distinct pathological stages: normal esophageal epithelium (Normal), esophageal simple hyperplasia (ESSH), intraepithelial neoplasia (IEN), and ESCC. Genome-wide DNA methylation profiling was performed using the Infinium Mouse Methylation BeadChip (285 K), coupled with transcriptome analysis via bulk RNA sequencing (RNA-seq). Immunohistochemistry (IHC) for Cd45 (leukocyte common antigen) validated immune cell infiltration. DNA methylation profiling revealed progressive genome-wide hypomethylation during ESCC development, with hypomethylated probes significantly enriched in immune response pathways. Notably, ESSH exhibited a methylation profile similar to IEN and ESCC. RNA-seq identified escalating numbers of differentially expressed genes (DEGs). Immune deconvolution analysis and IHC of Cd45 demonstrated dynamic changes in TIME composition from ESSH onwards. Furthermore, dynamic expression clustering identified an innate immune response-related gene cluster highly expressed in ESSH. Integrative analysis yielded 495 methylated regulatory genes, significantly enriched in leukocyte cell-cell adhesion and T cell activation pathways (e.g., Ptprc/Cd45, Il12rb1, Tox), with peak activity in ESSH. These findings highlight ESSH as a critical window where epigenetically driven immune changes facilitate ESCC progression. Targeting these early epigenetic-immune interactions may offer a novel strategy for ESCC early detection and combination therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42eab2688ceba2ad3f606cde51c1958eb7bd27fd" target='_blank'>
              Decoding epigenetic and transcriptional landscapes: DNA methylome-transcriptome integration reveals novel drivers in 4NQO-Induced esophageal squamous cell carcinoma mouse model
              </a>
            </td>
          <td>
            Yanli Qian, Runhua Lin, Zhiwei Liu, Xinxin Zhang, Dongming Lin, Min Su
          </td>
          <td>2025-12-27</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="DNA damage repair is essential for maintaining genomic integrity, thereby preventing diseases like cancer. Traditionally, radioresistance has been linked to the ability of cells to repair DNA double-strand breaks (DSBs) accurately. Recent research emphasizes the critical role of spatial chromatin organization and its dynamic reorganization in regulating repair and gene expression. In this study, we have employed single-molecule localization microscopy (SMLM) and Python-based mathematical methods of statistics and topology to locally analyze the spatial organization of γH2AX and 53BP1 foci in 15N-ion irradiated normal human dermal fibroblasts (NHDF) and highly radioresistant U87 glioblastoma cells over extended post-irradiation (PI) periods with nanoscale resolution. Our findings reveal that U87 cancer cells fail to regulate chromatin changes at DSB sites during and after repair. Specifically, Ripley's statistics and cluster analysis showed that both NHDF and U87 cells exhibit smaller, denser, and better separated γH2AX nano-foci surrounded by 53BP1 nano-foci. Mathematical topology approaches, including persistent homology, revealed that γH2AX nano-foci (clusters) have lower topological similarity compared to the more conserved 53BP1 nano-foci during the 24-hour repair period. These findings support the non-random, functional spatial organization of DSB repair (nano)foci and demonstrate its preservation in cancer cells. However, principal component analysis of persistent images showed that γH2AX- and 53BP1 nano-foci in normal fibroblasts exhibit stable, closed cycles, while U87 cells display chaotic, open shifts in topology in the 2D latent space. Combined with DSB repair kinetics measurements, this observation indicates that although U87 cells rejoin DSBs similar to normal cells, they experience more pronounced, dysregulated chromatin alterations during repair, ultimately failing to restore it to its pre-irradiation state. These alterations correlate with topologically more variable DSB sites, slower (more challenging) repair focus formation but faster repair once foci are established, compared to NHDF cells. More disorganized repair and persistent topological alterations likely contribute to genetic instability of cancer cells after irradiation and the development of radioresistant clones, posing challenges for effective radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a8908731964b69b6a1849f2467536142d35d4aa" target='_blank'>
              Nanoscale topology of γH2AX and 53BP1 foci in U87 cancer cells and normal NHDF after high-LET radiation-induced DSB repair.
              </a>
            </td>
          <td>
            Kim A Küntzelmann, Laura R Pardo, Myriam Schäfer, Jonas Weidner, I. Falková, Jiří Toufar, Lucie Toufarová, F. Bestvater, Michael Hausmann, Martin Falk
          </td>
          <td>2026-02-04</td>
          <td>Nanoscale</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Microsporidia, as opportunistic parasitic pathogens, constitute a formidable threat to human health. Although the regulatory circuitry of the nucleus-targeted effector EnP1 remains highly intricate and only partially characterized, our study identifies histone H2A as a novel binding partner of EnP1. Furthermore, we demonstrate that both EnP1 overexpression and microsporidia infection induce monoubiquitination of H2A (H2Aub) through downregulation of BAP1 expression. Subsequent mechanistic analyses revealed that elevated H2Aub levels positively correlate with enhanced microsporidian proliferation, whereas attenuation of H2Aub markedly suppresses pathogen expansion. Furthermore, EnP1 orchestrates the enrichment of H2Aub at the SLC7A11 promoter, driving its transcriptional upregulation. Collectively, these findings underscore that EnP1 modulates the ferroptosis state of host cells through H2Aub-mediated epigenetic reprogramming, ultimately facilitating pathogen propagation. This study endeavors to elucidate the critical survival strategies of microsporidia within host cells mediated by EnP1 and to unravel the multifaceted interplay between these pathogens and their hosts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99fb7fd0d1568a55a164eeac099af6d3c083a6ab" target='_blank'>
              EnP1 exploits H2Aub-dependent epigenetic reprogramming to promote microsporidia proliferation in host cells
              </a>
            </td>
          <td>
            Jingyu Guan, Yongliang Wang, Ming Fu, Liyuan Tang, Musa Makongoro Sabi, Huimin Zhu, Hua Cong, Chunxue Zhou, Huaiyu Zhou, Hongnan Qu, Bing Han
          </td>
          <td>2026-01-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Chromosomal instability (CIN), impaired telomere biology, and aberrant DNA methylation are implicated in colorectal cancer (CRC) development. Tracking these alterations from precancerous lesions through tumours to metastases may reveal biomarkers of CRC initiation and progression. Tissue samples from 44 patients with either high-grade colorectal dysplasia (HGA; n = 13) or advanced metastatic CRC (n = 31) were analyzed. CIN was assessed in all patients using either low-coverage whole-exome sequencing or microarray-based comparative genomic hybridization. In a subset of patients, genome-wide CpG methylation profiling (n = 19) and telomere length measurements (n = 15) were performed. CIN was detected in 85% of HGA patients, spanning focal CNVs in MALAT1 (46%) to recurrent alterations on chromosomes 11, 13, and 20, with PTK6 being the most frequently amplified (61%). CIN was comparable between primary tumours and synchronous metastases but was significantly elevated in metachronous cases. DEK was amplified in all metastases but the aberration was absent in primaries, irrespective of tissue chronicity. Methylation profiling distinguished HGA from adjacent non-dysplastic mucosa (9,859 differentially methylated CpGs) and unrelated tumour tissues (17,638 CpGs), whereas primary tumours and metastases differed at only five CpG sites. Both primary tumours and metastases appeared epigenetically younger than colonic mucosa. Metastases exhibited significantly shorter telomeres than both primary tumours (p = 0.019) and colonic mucosa (p = 0.001). The amplification of PTK6 may serve as an early biomarker detectable at the HGA stage, while DEK amplification appears crucial for metastatic progression and may represent a therapeutic target. Further validation is needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eac4e7ee54e068b1f7e50d381ee4cd9677cd3421" target='_blank'>
              Patterns of Chromosomal Instability and Epigenetic Alterations in Colorectal Cancer Progression: From High-Grade Dysplasia to Liver Metastases.
              </a>
            </td>
          <td>
            Kateřina Šašková, M. Landfors, V. Hlaváč, K. Šeborová, Jan Brůha, Katerina Honkova, K. Tomášová, Saba Selvi, Jana Drabova, Fernanda S Hackenhaar, Torbjörn K Nilsson, P. Vodička, Anna Valíčková, Václav Liška, Sanja A. Farkas, S. Degerman, Michal Kroupa
          </td>
          <td>2026-02-13</td>
          <td>Mutagenesis</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b40c40c3869325d70e55d95b98e9c465cbc80c" target='_blank'>
              MORC2 promotes acute myeloid leukemia by silencing LINE-1 retrotransposon
              </a>
            </td>
          <td>
            Shuai Wang, Fuqiang Wang, Zixuan Li, Nana Wang, Zhidong Gu, Zhimin Gu
          </td>
          <td>2026-02-21</td>
          <td>Functional & Integrative Genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f073f4531f143af0477555e22095e869e7b9e27" target='_blank'>
              The CeRNA role of HOTAIR: sponging MiRs to promote chemoresistance
              </a>
            </td>
          <td>
            Negarsadat Nademi, Pelin Ozfiliz-Kilbas
          </td>
          <td>2026-01-22</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor cell plasticity drives metastasis and therapy resistance, yet its regulation by oncoprotein dosage dynamics remains poorly understood. In Ewing sarcoma (EwS), variations in EWS::FLI1 (EF) fusion oncoprotein activity have been associated with epithelial-mesenchymal plasticity (EMP). Using degron technology, we precisely modulated endogenous EF in EwS cells and linked phenotypic states to distinct oncoprotein dosages. Strikingly, modest EF depletion promoted a pro-metastatic phenotype that diminished upon near-complete EF loss, revealing a paradoxical effect of submaximal EF inhibition. Nascent RNA-sequencing uncovered distinct gene clusters with heterogenous transcriptional responses to graded EF loss. Genes most sensitive to subtle EF depletion harbored GGAA microsatellites within EF-bound enhancers, while chromatin profiling uncovered candidate cofactors regulating EF-repressed EMP programs. Transient EF depletion followed by rapid restoration, modelling oncoprotein fluctuations, caused persistent dysregulation of genes functionally linked to enhanced extravasation and metastatic burden in preclinical models. This study highlights the therapeutic challenge of incomplete EF elimination, serving a paradigm in which oncoprotein dosage dynamics act as non-genetic drivers of disease progression and reveal novel vulnerabilities of advanced disease. The partial and transient suppression of the EF oncoprotein was found to paradoxically promote Ewing sarcoma metastatic potential through persistent transcriptional dysregulation, establishing the oncoprotein dosage dynamics as a key regulator of metastatic plasticity and therapeutic vulnerability. Graded EF depletion using dTAGV-1 elicited a non-linear phenotypic response, with modest EF loss being sufficient to induce a pro-metastatic state. Integration of acute transcriptional responses with chromatin profiling led to the identification of EF co-regulators. EF restoration following transient depletion resulted in lasting dysregulation of pro-metastasis-related genes and emergence of hybrid epithelial-mesenchymal states, thereby enhancing extravasation and lung metastasis in vivo. Partial or transient EF inhibition may promote disease progression and worsen clinical outcomes, highlighting that therapeutic strategies targeting EF must achieve complete and sustained suppression. Graded EF depletion using dTAGV-1 elicited a non-linear phenotypic response, with modest EF loss being sufficient to induce a pro-metastatic state. Integration of acute transcriptional responses with chromatin profiling led to the identification of EF co-regulators. EF restoration following transient depletion resulted in lasting dysregulation of pro-metastasis-related genes and emergence of hybrid epithelial-mesenchymal states, thereby enhancing extravasation and lung metastasis in vivo. Partial or transient EF inhibition may promote disease progression and worsen clinical outcomes, highlighting that therapeutic strategies targeting EF must achieve complete and sustained suppression. The partial and transient suppression of the EF oncoprotein was found to paradoxically promote Ewing sarcoma metastatic potential through persistent transcriptional dysregulation, establishing the oncoprotein dosage dynamics as a key regulator of metastatic plasticity and therapeutic vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5051c5027b542c8e38511f6f4e8d71524906c31f" target='_blank'>
              Modelling EWS::FLI1 protein fluctuations reveal determinants of tumor plasticity in Ewing sarcoma
              </a>
            </td>
          <td>
            Veveeyan Suresh, Christoph Hafemeister, Andri Konstantinou, S. Grissenberger, C. Sturtzel, M. Zylka, F. Cidre‐Aranaz, A. Wenninger-Weinzierl, Karla Queiroz, D. Kurek, Martin Distel, Anna C. Obenauf, T. Grünewald, F. Halbritter, H. Kovar, Valerie Fock
          </td>
          <td>2026-01-03</td>
          <td>EMBO Molecular Medicine</td>
          <td>2</td>
          <td>32</td>
        </tr>

        <tr id="Sex chromosome aneuploidies (SCAs) are collectively common genetic disorders that impact diverse body systems. The molecular mechanisms by which an extra or missing sex chromosome increases clinical risk are not fully understood, but they likely involve imbalances in expression and regulation of dosage-sensitive genes. There has been a recent surge in transcriptomic and epigenomic studies on genomic effects of SCAs - making it an opportune time to: (i) map existing knowledge of SCA impacts on gene expression and regulation; (ii) resolve consensus findings on SCA dosage-sensitive genes; and (iii) define gaps and high priority areas for future research. Following the Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines (protocol registered on the international Prospective Register of Systematic Reviews PROSPERO CRD42024473984), we searched nine databases and screened the titles and abstracts of 2421 records, thoroughly evaluated 161 full-text articles, and identified 57 eligible studies for abstraction of methodological features and results. Our review spans 18 years of research and encompasses samples from 930 individuals with SCAs and 2192 euploidic controls. The recent acceleration in publication rates outstrips that for biomedical research as a whole. Studies have only recently started to diversify away from the most studied SCAs (47,XXY and 45,X karyotypes), tissue types (blood-derived, gonadal) and measurement methods (transcriptomic analysis by RNAseq). We identify a core set of dosage-sensitive genes that are recurrently impacted by SCAs across multiple tissues. These genes concentrate in 3 protein-protein interaction networks that are predominantly enriched for chromatin remodelling, and represent candidate drivers of downstream phenotypes. This systematic review of SCA impacts on the human genome helps to target the future research efforts that are now needed to (i) address existing knowledge gaps by diversifying the karyotypes, tissues and genomic features analyzed, and (ii) test the causal role for recurrently dysregulated genes. Meeting these goals would provide a molecular foundation to drive both basic and clinical understanding of sex chromosome influences on human phenotypic variation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2f04563ca279e23a5b7a1d63c20549007f5b86a" target='_blank'>
              Sex chromosome aneuploidy impacts on human gene expression and regulation: a systematic review
              </a>
            </td>
          <td>
            Marcela Legue, Melanie Staszewski, Maya Mastronardo, Gisela Butera, Aleksandra Dakic, Armin Raznahan
          </td>
          <td>2025-12-30</td>
          <td>Molecular Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Gene fusions are critical oncogenic drivers and therapeutic targets in diverse cancers. Long-read ribonucleic acid sequencing (RNA-seq) offers an unprecedented opportunity to resolve the full-length structure of fusion isoforms, but its high intrinsic error rates pose significant challenges to the precise identification of true fusion events. Here, we developed GFSeeker, an innovative splicing-graph-based computational framework for accurate gene fusion detection from long-read RNA-seq. GFSeeker employs a unique pipeline based on a splicing graph reference and a dual re-alignment validation to effectively overcome data noise from high error rates. Benchmarking across simulated, non-tumor, and cancer cell line datasets demonstrated GFSeeker’s state-of-the-art performance, achieving 6%–15% higher F1 score compared to existing methods. Notably, GFSeeker successfully identified the known fusion event, MATN2–POP1, in the MCF-7 cancer cell line, missed by other tools, highlighting its superior sensitivity in resolving complex fusion events. These results validate GFSeeker as a powerful and reliable tool for gene fusion discovery, heralding its significant potential to advance cancer research and precision diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86fa854d23dc7ec4d6c96653cc3d0b39ac6f3a32" target='_blank'>
              GFSeeker: a splicing-graph-based approach for accurate gene fusion detection from long-read RNA sequencing data
              </a>
            </td>
          <td>
            Bingyan Wang, Heng Hu, Runtian Gao, Guohua Wang, Tao Jiang
          </td>
          <td>2026-01-01</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The senescence-associated secretory phenotype (SASP) exerts dual roles in tumor suppression and progression, yet how it is regulated in head and neck squamous cell carcinoma (HNSCC) remains unclear. Here, we identify LIM homeobox 1 (LHX1) as a key transcriptional suppressor of STING, whose downregulation enables evasion of SASP-mediated tumor surveillance. Notably, high LHX1 expression correlated with poor prognosis in HNSCC patients. Mechanistically, LHX1, in complex with LDB1, directly bound to the STING promoter to mediate transcriptional repression via the deposition of the repressive histone mark H3K9me3, thereby blocking SASP activation. Depletion of LHX1 restored STING-dependent SASP and impaired cancer stem cell self-renewal. Therapeutic disruption of the LHX1-LDB1 complex using engineered peptides re-activated STING signaling, induced SASP, and significantly suppressed tumor growth. In this study, we employed human and mouse-derived HNSCC cell lines, xenograft models, and clinical samples to assess the functional relevance of LHX1 in regulating SASP and tumor progression. Our findings reveal LHX1 as a master transcriptional repressor of STING-mediated senescence and highlight the therapeutic potential of targeting the LHX1-LDB1 axis to restore tumor-suppressive SASP in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91c542eb07fdf0c718c2bb9764fab625fc0a071c" target='_blank'>
              Targeting the LHX1-LDB1 Complex Restores STING-dependent Senescence Surveillance and Inhibits Head and Neck Cancer Progression
              </a>
            </td>
          <td>
            Mingshu Long, Yang Chen, Ruixue Du, Jiejie Yang, Jingjing Wang, Yunqing Sun, Xuezhang Tian, Shaowei Wang, Yunhong Zhong, Weilian Liang, Junjie Zhang, Z. Shang
          </td>
          <td>2026-01-01</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Ectonucleotide pyrophosphatase/phosphodiesterase 1 (ENPP1), 1st identified in breast cancer and subsequently in multiple other cancer types, is an innate immune checkpoint regulator that recently emerged as a promising biomarker and therapeutic target. Homologous recombination deficiency (HRD) has gained clinical relevance with therapeutic vulnerability, particularly in breast and ovarian cancers. Despite the increasing significance of ENPP1 and HRD in cancer biology and treatment, their potential relationships have not yet been comprehensively investigated. We analyzed the relationship between ENPP1 expression and HRD score across the Cancer Genome Atlas pan-cancer and individual tumor types using the Pearson and Spearman correlations. To account for heterogeneity, pan-cancer samples were clustered using linear regression into 3 groups based on Bayesian Information Criterion. Differential expression, functional enrichment, and survival analyses were performed for these clusters at both the pan-cancer and representative tumor type levels. Although the pan-cancer relationship between ENPP1 expression and HRD score was heterogeneous, significant correlations were observed in 11 tumor types. Linear regression-based clustering resolved this heterogeneity into 3 functionally and clinically distinct groups: Cluster 1 was characterized by proliferation programs; Cluster 2 by extracellular matrix remodeling, differentiation, and immune response; and Cluster 3, by metabolic reprogramming. Clinically, Cluster 3 was associated with better survival than Clusters 1 and 2 in a pan-cancer analysis (P < .0001). At the individual tumor type level, these global cluster features were further modified in tissue-specific contexts, reflecting local microenvironment adaptation. Significant survival differences were observed in patients with adrenocortical carcinoma, chromophobe renal cell carcinoma, low grade glioma, and mesothelioma, further underscoring the tissue-specific modification of global cluster features. Our comprehensive pan-cancer analysis revealed the intrinsic heterogeneity of ENPP1 expression and HRD score, which may arise from complex and dynamic interactions with diverse cancer hallmarks, including proliferation, extracellular matrix remodeling, immune response, and metabolic reprogramming, and can be generalized into 3 clusters with distinct molecular and clinical characteristics. At the individual tumor type level, these global cluster features were further modified to adapt to a tissue-specific microenvironment, manifesting distinct tissue-specific patterns. Collectively, these findings provide a foundation for refining biomarker-driven precision medicine strategies for diverse tumor types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad738ec3c685dfb5e7e6594798ea773356192816" target='_blank'>
              Interaction between ENPP1 and homologous recombination deficiency defines distinct pan-cancer signatures: A retrospective observational study
              </a>
            </td>
          <td>
            Yong Min Kim, Sung‐Hak Lee, Woong Na, Il Ju Lee, Mihye Kwon, Oyeon Jo, Young Soo Song
          </td>
          <td>2026-01-02</td>
          <td>Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Ecotropic viral integration site 1 (EVI1), encoded by the EVI1 gene on chromosome 3q26.2, is a dual-domain zinc finger transcription factor that functions as a potent proto-oncogene in a wide spectrum of hematological malignancies. Under normal physiological conditions, its expression is tightly regulated and restricted primarily to hematopoietic stem cells and specific embryonic tissues. However, aberrant overexpression of EVI1 is a hallmark of aggressive myeloid leukemias, including acute myeloid leukemia (AML), myelodysplastic syndromes (MDS), and the blast crisis of chronic myeloid leukemia (CML). The oncogenic activation of EVI1 occurs through diverse genetic mechanisms, most notably chromosomal rearrangements involving the 3q26 locus, such as inv(3)(q21q26.2) and t(3;3)(q21;q26.2), which juxtapose the EVI1 gene with potent enhancers like that of GATA2. Other mechanisms include the formation of oncogenic fusion genes (e.g., AML1-EVI1, ETV6-EVI1), enhancer hijacking, and retroviral insertional mutagenesis. Once overexpressed, EVI1 drives leukemogenesis through multifaceted molecular actions. It acts as a master transcriptional regulator, profoundly disrupting normal hematopoietic differentiation by repressing key lineage-specific transcription factors like RUNX1 and interfering with cytokine-induced maturation. Concurrently, EVI1 promotes cell survival and proliferation by modulating critical signaling pathways, including the potent inhibition of the tumor-suppressive TGF-β pathway and the activation of the pro-survival PI3K/AKT/mTOR cascade via PTEN suppression. EVI1 also cooperates with a multitude of other oncogenic lesions, such as MLL rearrangements, AML1 mutations, and activated RAS signaling, to accelerate disease progression. Clinically, EVI1 overexpression is one of the most robust independent indicators of poor prognosis, associated with therapy resistance and reduced overall survival. This review provides a detailed discussion of the mechanisms underlying EVI1’s activation, its complex molecular functions in hematopoietic transformation, and its profound clinical implications in hematological malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dad0221eebaacefbada4381fc35dd15a2fbadc0" target='_blank'>
              The oncogenic role of ecotropic viral integration site 1 in hematological malignancies: mechanisms of activation and leukemogenesis
              </a>
            </td>
          <td>
            Yuchao Hao, Jing Liu, Jiacheng Lou, Jinsong Yan
          </td>
          <td>2026-02-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: Head and neck squamous cell carcinoma remains a highly aggressive malignancy with limited predictive biomarkers for prognosis and radiotherapy response. Increasing evidence indicates that pseudogenes are functionally active regulators of cancer biology, yet their clinical relevance in HNSCC is poorly defined. Methods: Using transcriptomic and clinical data from The Cancer Genome Atlas, we analyzed the expression and clinical significance of two pseudogenes, ANXA2P2 and PA2G4P4, in HNSCC. Associations with clinicopathological features, HPV status, tumor subtypes, survival, genomic instability, radiotherapy response, and immune landscape were assessed using bioinformatic tools. Results: Both pseudogenes were significantly upregulated in HNSCC compared to normal tissues. Higher expression levels correlated with adverse clinicopathological features, increased tumor proliferation and wound-healing capacity, and unfavorable TCGA molecular subtypes. High ANXA2P2 and PA2G4P4 expression was associated with reduced overall survival, while their combined low-expression signature identified patients with significantly improved overall and disease-free survival. Notably, lower expression of both pseudogenes was observed in patients responding to radiotherapy, whereas higher expression was linked to genomic instability parameters and enrichment of oncogenic pathways, including MYC, PI3K/AKT/mTOR, cell cycle regulation, and DNA repair. ANXA2P2 expression differed significantly by HPV status, showing reduced levels in HPV-positive tumors. Furthermore, pseudogene expression stratified distinct immune profiles, including immune subtypes, stromal and immune scores, and specific immune cell populations. Conclusions: ANXA2P2 and PA2G4P4 are clinically relevant pseudogenes associated with tumor aggressiveness, immune modulation, and radiotherapy response in HNSCC. These findings support their potential utility as prognostic and predictive biomarkers and provide a rationale for further functional validation in experimental models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33872a456524f0aadce25f6d9d2640840e11a5a0" target='_blank'>
              ANXA2P2 and PA2G4P4 Pseudogenes Are Associated with the Response to Ionizing Radiation and Could Be Used as Potential Biomarkers: In Silico Study
              </a>
            </td>
          <td>
            Tomasz Kolenda, Piotr Białas, Kacper Kamiński, Maria Dziuba, Małgorzata Czernecka, Aleksandra Leszczyńska, Kacper Guglas, Joanna Kozłowska-Masłoń, Paulina Potter, Klaudia Dudek, Nina Grzejda, Karina Tylkowska, Anna Zapłata, Marlena Janiczek-Polewska, Paulina Gieremek, Katarzyna Regulska, Patrycja Mantaj, Anna Florczak-Substyk, Anna Przybyła, U. Kazimierczak, E. Leporowska, Zefiryn Cybulski, Beata J Stanisz, Anna Teresiak
          </td>
          <td>2026-01-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="


 Clonal hematopoiesis (CH) is exemplified by the expansion of genomically altered hematopoietic stem and progenitor cells that may give rise to altered immune repertoires. Driven largely by age and chronic inflammation, CH has been historically linked to myeloid malignancies. Although emerging evidence points to a modest prevalence of CH within solid tumors, the impact of these naturally occurring alterations on anti-tumor immunity and therapeutic responses remains less defined. Here, we employed somatic mutation calling algorithms to detect single nucleotide variants (SNVs) in both epithelial and immune compartments within tumors. Our preliminary analyses of single-cell RNA-sequencing (scRNA-seq) of 32 primary, therapy-naïve prostate tumors encompassing 11 patients identified conventional mutations representative of both CH and prostate cancer. Within the tumor-epithelial compartment, we identified recurrent FOXA1 class 1C mutations in 27% (3/11), alongside NCOR1 mutations in 18% (2/11), IDH1 in 9% (1/11), and SPOP in 9% (1/11) of patients. Of note, these SNVs were detected across multiple samples for the same patient. Interestingly, we also detected germline HOXB13 G84E variants, associated with increased risk for prostate cancer, in 2 cases at a high clonality within epithelial cells (>75%). CH alterations were identified in 27% (3/11) of patients based on the detection of exonic, nonsynonymous SNVs within myeloid cells. Specifically, these variants affected SF3B2 (variant allele frequency; VAF = 0.30), UBE2A (VAF = 0.30), and KMT2C (VAF = 0.43) that are predicted deleterious across multiple classifier algorithms. Patients with detectable CH alterations displayed immunologically hotter tumor microenvironments, with CH+ patients having significantly higher proportions of tumor-infiltrating monocytes (16.22% vs 7.13%, Fisher’s exact test p<0.0001), macrophages (6.49% vs 2.88%, p<0.0001), CD8+ T cells (10.35% vs 3.33%, p<0.0001), CD4+ T cells (2.81% vs 0.74%, p<0.0001), and B cells (1.81% vs 0.59%, p<0.0001), and lower fractions of epithelial cells (50.94% vs 69.81%, p<0.0001). In summary, we leveraged de novo mutational algorithms to identify somatic alterations using scRNA-seq. To our knowledge, our study is the first to detect CH alterations in prostate cancer at the single-cell resolution. Further work will integrate our mutational signatures with transcriptomics to elucidate the impact of both cancer- and immune-intrinsic alterations on anti-tumor immunity. Additionally, we will incorporate mitochondrial DNA-based lineage tracing algorithms on matched peripheral blood, primary, and metastatic tumor samples to deconvolute the spatiotemporal regulation of naturally occurring CH alterations across cancer progression. We envision our findings will illuminate novel drivers of immune function and regulation within prostate cancer that may be exploited for future immunomodulatory therapies.



 John R. Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang. Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A040.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03587209005d8227c89ae44e47e8117ddd350a30" target='_blank'>
              Abstract A040: Interrogating the Genomic Co-Evolution of the Immune System and Prostate Cancer Along the Time and Treatment Continuum
              </a>
            </td>
          <td>
            John R Lozada, Christine Luo, Devin A. Schmeck, Jeffrey S. Miller, Akash Patnaik, Emmanuel S. Antonarakis, Frank Cichocki, Justin H. Hwang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 Fibrolamellar Carcinoma (FLC) is a liver cancer of adolescents and young adults driven by the fusion of DNAJB1, a heat shock protein and PRKACA, the catalytic subunit of protein kinase A (PKA). It is characterized by increased PKA activity leading to widespread changes. A profile of FLC at genomics, transcriptomics, methylomics and proteomics in a multi-omic study that studied 1,412 tumor-normal paired liver samples from patients with FLC, Hepatoblastoma, Intrahepatic Cholangiocarcinoma, and Hepatocellular Carcinoma, yielded the FLC Data Base (FLC-DB). To facilitate the visualization of the multiomics, we developed a user-friendly web interface for the FLC-DB. It uses functions from the R-library OmicsKit and enables interactive exploration for non-bioinformaticians to retrieve information and generate publication-ready plots. It comprises 3 main tabs. First, Documentation, which describes the samples and datasets, the transcriptomic FLC Signature, and a reduced set of 35 genes with strong and recurrent upregulation across all samples, selected for clinical purposes. Second, the Transcriptomics tab, where users can interactively explore the distribution of gene counts across liver cancers, comparing normal samples against tumors and tumor subtypes. This has the option to directly perform Differential Expression Analysis using subsets that can be dynamically defined by the user in the web interface. Third, the Genomics tab, which allows exploration of Single Nucleotide Variants (SNV) from Whole-Genome and Whole-Exome Sequencing (WGS/WES) and download of the SNVs annotations and perform downstream analyses. This allows exploring questions such as: Is a transcript consistently upregulated across FLC studies? Are there recurrent SNVs in FLC? What gene signatures are shared between FLC and HBL? The FLC-DB web interface facilitates exploration of complex multi-omics data, broadening its access to non-bioinformatic researchers, patients and the community. It is currently available at https://fibrolamellardatabase.org/.



 Daniel F. Guevara-Diaz, David Requena, Sanford M. Simon. The fibrolamellar carcinoma database [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B031.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6039ebc7493e6eddced2c3191b76583f0aea8b24" target='_blank'>
              Abstract B031: The fibrolamellar carcinoma database
              </a>
            </td>
          <td>
            Daniel F. Guevara-Diaz, David Requena, Sanford M. Simon
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer cell characteristics are determined by gene expression, influenced by genomic, epigenetic, and transcriptional modifications. Genomic rearrangements and transcriptional splicing can result in the formation of fusion genes. BCR-ABL1 is an established fusion gene employed as a biomarker in leukemia. A single gene can amalgamate with several other genes and may impact cellular fate. Ethnicity-specific variants of fusion genes have been identified, such as the TMPRSS2-ERG variation observed in prostate malignancies among African-American, Caucasian, and Japanese populations in research studies. Next-generation sequencing has provided a new method for predicting genomic and transcriptomic changes. We aim to identify fusion genes in the Indian population using cancer samples to enhance diagnostic outcomes. This study performed a meta-analysis of tumor-specific RNA sequencing data for liver, tongue, and ovarian cancers, which are available online. It identified known fusion genes, including TRO-MAGED2, KRT14-S100A9, RNASE10-CD38, ACTN4-ACTN1, RGPD1-RANBP2, CTSC-RAB38, C15orf57-CBX3, AMBRA1-CKAP5, ATP2B3-ATP2B4, CNKSR3-IPCEF1, E2F4-RPL14, and MZT2A-MZT2B, along with 101 novel fusion genes. Novel fusion genes GABRP_SCGB3A2 and WWOX_FUT1 were identified in all three tumor tissues. GABRP acts as a tumor inducer, whereas SCGB3A2 functions as a tumor suppressor. WWOX2 serves as a tumor suppressor, whereas FUT1 functions as a promoter of malignancy. The interplay between tumor inducers and suppressors may serve as a survival mechanism for cancer cells, a subject that has received limited research attention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4406b3ded6f3886093045dfec09d59265a20dd7e" target='_blank'>
              RNAseq-based meta-analyses revealed tumor suppressor-inducer fusion events in liver, oral, and ovarian cancer in the Indian population: a cancer cell surviving mechanism.
              </a>
            </td>
          <td>
            Rahul Yadav, H. Khan, P. Singh, Pramod Kumar, Dinesh Kumar Singhal
          </td>
          <td>2026-02-09</td>
          <td>Nucleosides, nucleotides & nucleic acids</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Dysregulated mRNA translation is often observed in cancer and is an emerging mechanism of oncogene expression and modulation of immune response in cancer. Pancreatic adenocarcinoma (PDAC) is a highly aggressive and challenging form of cancer with a 5-year survival rate of 13% that urgently requires novel therapeutic approaches. PDAC is characterized by low mutation rates, an immunologically suppressive environment, and resistance to immunotherapy, highlighting the need for new insights into the mechanisms of immune recognition and the discovery of tumor-specific antigens. Our work addresses the unmet need to discover the neoantigens and improve the treatment modalities in PDAC. Our and other’s work have previously established that RNA helicase EIF4A (Eukaryotic Initiation Factor 4A) dependent mRNA translation contribute to oncogenic programs and modulate immune response, offering potential therapeutic targets to inhibit PDAC growth. In this study we have mapped the translational landscape and mRNA Translation Start Site (RTSS) in PDAC cells following EIF4A inhibitor CR-1-31B treatment. CR-1-31B suppress cap-dependent mRNA translation while preserving and upregulating the mRNA translation of HLA Class I proteins including HLA-A/B/C (Human Leukocyte Antigen) and B2M (b-2 microglobulin). Next, we establish that CR-1-31B promotes immune checkpoint blockade protein PD-L1 expression. Interestingly while EIF4A inhibition results in downregulation of cap-dependent mRNA translation we observed a significant increase in the translation products arising from the usage of alternate RTSS in PDAC cells. Further we show that these non-canonical translation products can generate immunogenic peptides that can be presented by HLA Class I and elicit T cell activation in vitro. These neoantigens are not dependent on the somatic mutational status of cancer and hence, are particularly interesting for solid tumors with low somatic mutational burden such as PDAC. Our key findings show that targeting EIF4A using CR-1-31B converts PDAC tumors from an immunologically “cold” tumor to an immunologically “hot” tumor in a multifaceted fashion, by 1) promoting non-canonical translation products driven neoantigens, 2) enhancing protein production of HLA Class I proteins, and 3) upregulating expression of PD-L1, and 4) further sensitizing CR-1-31B treated tumors to anti-PD-L1 immune checkpoint blockade therapy. Our study establishes the contributions of each of these aspects on promoting anti-tumor immunity with the objective of developing a new treatment modality for PDAC.


 Kamini Singh. EIF4A Blockade Drives Neoantigen Generation and Immune Remodeling in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Immuno-Oncology Conference (AACR IO): Discovery and Innovation in Cancer Immunology: Revolutionizing Treatment through Immunotherapy; 2026 Feb 18-21; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Immunol Res 2026;14(2 Suppl):Abstract nr B047.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c284fd46f67974298984960aab16198b6341e97" target='_blank'>
              Abstract B047: EIF4A Blockade Drives Neoantigen Generation and Immune Remodeling in Pancreatic Cancer
              </a>
            </td>
          <td>
            Kamini Singh
          </td>
          <td>2026-02-18</td>
          <td>Cancer Immunology Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63700b1580a28e6e919137fb8581425a7fd29e67" target='_blank'>
              Digging deeper into cancer resistance in the extraordinarily long-lived naked mole-rat
              </a>
            </td>
          <td>
            Tanvi T Patel, Vince G Amoroso, Kaitlyn N. Lewis Hardell, Vikram Narayan, Martha A. Delaney, Rochelle Buffenstein
          </td>
          <td>2026-01-27</td>
          <td>Mammalian Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Familial cancers are caused by inherited mutations in specific genes that regulate cell growth, division, and repair. Approximately 5–10% of all cancer cases have a hereditary component, where germline mutations in certain genes increase an individual’s susceptibility to developing cancer. Two major categories of genes are involved in cancer development: tumour suppressor genes and oncogenes. Both play critical roles in regulating normal cell behaviour, and when mutated, they can contribute to uncontrolled cell proliferation and tumour formation. In addition to genetic mutations, epigenetic alterations also play a significant role in familial cancer. Epigenetics refers to changes in gene expression due to DNA methylation, histone modifications, and the dysregulation of non-coding RNAs without alter the underlying DNA sequence. Familial cancer syndromes follow various inheritance patterns, including autosomal dominant, autosomal recessive, X-linked, and mitochondrial inheritance, each with distinct characteristics. Identifying genetic mutations associated with familial cancers is a cornerstone of genetic counselling, which helps individuals and families navigate the complex intersection of genetics, cancer risk, and prevention. Early identification of mutations enables personalized strategies for risk reduction, early detection, and, when applicable, targeted treatment options, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69f296f3ae0afec7c6382eb8b634a83d98dd9138" target='_blank'>
              Genetic Basis of Familial Cancer Risk: A Narrative Review
              </a>
            </td>
          <td>
            Eman Fares Sabik
          </td>
          <td>2026-01-13</td>
          <td>DNA</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15ea2c9c7605b557b8e4ec203f1280961cf9800c" target='_blank'>
              Exploring the oncogenic mechanism of plasmacytoma variant translocation 1 (PVT1) gene in solid cancers; emphasis on microRNA regulation pathways
              </a>
            </td>
          <td>
            Mojtaba Zehtabi, Shirin Azizidoost, Abdolah Mousavi Salehi, Mahrokh Abouali Gale Dari, Mohammad Amin Zargar, Maryam Khombi Shooshtari, Krzysztof Data, Dominika Domagała, Julia Niebora, Małgorzata Józkowiak, P. Chmielewski, Aleksandra Partyńska, Aleksandra Górska, A. Bryja, M. Kulus, Adam Kamiński, Teresa Wysocka, Marek Spaczyński, Piotr Dzięgiel, P. Mozdziak, Maryam Farzaneh, B. Kempisty
          </td>
          <td>2026-01-29</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c61b70c406aecf91abbf9e8d3611bdefdb5b3ab6" target='_blank'>
              Dependency Map correlation analysis reveals WDR89 as a genome maintenance factor
              </a>
            </td>
          <td>
            Sean W. Minaker, Gian L. Negri, G. Morin, Peter C. Stirling
          </td>
          <td>2026-01-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Colorectal cancer ranks third in global incidence and second in cancer mortality. Patient-derived models are irreplaceable for studying tumor biology. We established a human epithelial cell line from a rectal adenocarcinoma overexpressing cancer stem cell marker ALDH1A1, and we investigated the effect of ALDH1A1 knockout on tumor cell traits. The cell line and its CRISPR-Cas9 ALDH1A1 knockouts were characterized by genomic and cytogenetic methods (CNV, WES, RNAseq, karyotype), in vitro (proliferation, response to chemotherapy, migration, invasion, apoptosis), and in vivo methods. We identified the landscape of somatic mutations and copy number alterations in the original tumor and the derived cell line. Genetic attenuation of ALDH1A1 was characterized by an increase in migratory potential and extensive metastatic ability, accompanied by reduced growth of subcutaneous xenografts and alterations in gene expression associated with inhibited proliferation and promoted invasion and metastasis, ultimately resulting in dysregulation of the Wnt signaling pathway. Increased metastatic potential was also confirmed in HT-29 cells after ALDH1A1 genetic attenuation. CRISPR-Cas9-mediated editing led to functional, cellular, and molecular changes confirming the role of ALDH1A1 in colorectal cancer carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b72d554a35d6005b3610cd9e7621ca3ab6650f1a" target='_blank'>
              Genetic attenuation of ALDH1A1 increases metastatic potential and aggressiveness in colorectal cancer.
              </a>
            </td>
          <td>
            M. Poturnajova, Z. Kozovska, Ondrej Pos, Kristína Pavlov, S. Gulati, Peter Makovický, Kristina Jakic, M. Burikova, Eva Sedlackova, Barbora Svitková, S. Tyčiaková, V. Bystrý, Nicolas Blavet, Boris Tichý, Matej Hrnciar, J. Budiš, Miroslav Tomas, Peter Dubovan, Georgina Kolnikova, Veronika Repaska, Nikoleta Mojzesova, E. Zomborská, D. Pindak, Michal Mego, T. Szemeš, M. Matúšková
          </td>
          <td>2026-02-03</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="As a tumor suppressor gene, CCDC6 encodes a coiled-coil domain-containing protein that is ubiquitously expressed and involved in crucial cellular processes such as DNA damage response and apoptosis, although its precise mechanisms remain elusive. Initially identified as part of a fusion gene, CCDC6 can form fusion genes with a variety of proto-oncogenes, including both kinase- and non-kinase-coding genes, thereby facilitating oncogenesis. Alterations in CCDC6 expression across various cancers underscore its intricate role and potential influence on the efficacy of anticancer therapies. Recent findings have demonstrated that CCDC6 can undergo liquid-liquid phase separation (LLPS) and facilitate the LLPS of its associated fusion proteins, providing new perspectives on its functional characterization and potential therapeutic implications in related diseases. We present a comprehensive overview of CCDC6, encompassing its protein characteristics and physiological and genomic aspects. Furthermore, we explored the association between CCDC6 alterations and carcinogenesis, as well as their implications for therapeutic interventions. The objective of this review is to furnish the medicinal community with current information and valuable insights pertaining to diseases associated with CCDC6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872145069c4ef81fa1ace0d713f50e945a6721cc" target='_blank'>
              The multiple roles of the disordered protein CCDC6 in cancer development.
              </a>
            </td>
          <td>
            Guifeng Wei, Yiji Chen, Yichao Kong, Donglai Li, Yang Wang, Ting Qiu, Xiabin Chen
          </td>
          <td>2026-01-24</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Pathogenic variants in EXOSC3, a noncatalytic subunit of the RNA exosome, cause pontocerebellar hypoplasia type 1B (PCH1B), yet substantial variability in disease severity is observed among individuals carrying different EXOSC3 alleles. The molecular mechanisms of RNA exosome dysfunction in individuals carrying EXOSC3 p.G191 variants remains unresolved. To address this, we generated CRISPR/Cas9-engineered human cell models harboring EXOSC3 p.G191 variants and performed integrated transcriptomic, proteomic, and computational structural analyses. EXOSC3 p.G191 variants produced extensive, allele- and dosage-dependent alterations in gene expression and splicing, with heterozygous variants causing broad but attenuated disruption relative to homozygous EXOSC3 G191D/G191D cells. All EXOSC3 G191 variants promoted increased skipping of exon 3 in EXOSC3 transcripts, generating a short isoform predicted to encode an unstable proteoform. Molecular dynamics and λ-dynamics simulations predicted substantial thermodynamic destabilization of all EXOSC3 G191 variant proteoforms, consistent with reduced protein abundance and thermal stability measured by global proteomics and PISA. At the protein complex level, EXOSC3 p.G191 variants were associated with coordinated decreases in all RNA exosome core subunits and the exonuclease EXOSC10, consistent with destabilization of RNA exosome assembly and orphan protein decay. In contrast, the catalytic exonuclease DIS3 and multiple independent RNA processing pathways were upregulated, indicating compensatory recalibration of RNA metabolism. Together, these findings link variant-induced alternative splicing, RNA exosome complex destabilization, and adaptive network responses to phenotypic variability in EXOSC3 p.G191-associated disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6712e609a96286c4c86826dc037712751fee05b4" target='_blank'>
              EXOSC3 G191 Variants Trigger System-Wide Recalibration of RNA Processing Machinery
              </a>
            </td>
          <td>
            H. S. Wijeratne, Avery M. Runnebohm, Monica P. Barron, Ariana J. Hampton, Rudong Li, S. Justice, Sophia H. Lenceski, James D. Rooney, Whitney R. Smith-Kinnaman, G.S. Chang, E. Doud, Jungsu Kim, Stephane Pelletier, Yunlong Liu, Jonah Z. Vilseck, A. Mosley
          </td>
          <td>2026-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The post-genomic era has transformed medical genetics, raising renewed debate over the role of medical cytogenetics in clinical practice. High-throughput sequencing and chromosomal microarray technologies now dominate cancer diagnostics, prenatal testing, and rare disease evaluation by enabling rapid detection of gene-level variation, often leading to the perception that cytogenetics is obsolete. However, this view overlooks the unique and complementary strengths of cytogenetic analysis. Although the relationship between cytogenetics and current NGS technologies can be compared to that between forests and trees versus leaves—both of which are necessary for clinical diagnosis—cytogenetic methods uniquely enable direct in situ visualization of chromosomes, allowing detection of large-scale structural and numerical genome alterations at the level of individual cells and cell populations. These system-level features that are frequently invisible or difficult to interpret using sequencing-based approaches alone yet are critical in disease contexts where genome architecture itself carries biological and clinical significance beyond individual genes. This article, therefore, advances a new perspective based on Genome Architecture Theory: that karyotype-level information organizes gene-level function and that many previous gene-centric genetic concepts require reexamination within a unified framework of clinical genomics. Rather than being replaced, cytogenetics is increasingly integrated with sequencing within a unified framework of clinical genomics that combines high-resolution molecular detail with system-level insight into genome organization. Reassessing the role of cytogenetics, therefore, has important implications for medical education, diagnostic strategy, and healthcare policy, as cytogenetics provides the appropriate platform for understanding system-level inheritance through karyotype coding and for advancing molecular medicine from a genome systems perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f55a3cccc1e3b9d9b3c2e2530a657ae5830cf723" target='_blank'>
              Conversation with Future Clinical Cytogeneticists: The New Frontiers
              </a>
            </td>
          <td>
            Jing Christine Ye, Rishi Chowdhury, Henry H. Heng
          </td>
          <td>2026-02-12</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background
Enhancer of zeste homolog 2 (EZH2) is frequently upregulated in prostate cancer (PCa) and further increased in castration-resistant prostate cancer (CRPC), a lethal state characterized by profound immune dysfunction. However, EZH2-associated immune programs in bulk cohorts and their corresponding cell-type-specific features in single-cell RNA sequencing (scRNA-seq) data have not been systematically delineated in advanced PCa.


Methods
We integrated bulk RNA-seq and scRNA-seq to map EZH2-associated transcriptional and immune features in PCa. In bulk cohorts (TCGA-PRAD and an independent metastatic CRPC cohort), we quantified EZH2 expression, clinical outcomes, and immune-signature enrichment. Immune-modulated differentially expressed genes (IMDEGs) were defined by intersecting EZH2-associated differential expression with correlations to Treg/TAM-related signature scores, and were used for NMF-based immune subtyping and penalized Cox modeling with validation. In scRNA-seq cohorts (GSE264573 and an independent CRPC cohort), malignant epithelial cells were inferred by copy-number alteration profiles, EZH2^high versus EZH2^low malignant programs were characterized, T-cell subsets were quantified, and tumor-Treg communication was inferred using CellPhoneDB as hypothesis-generating predictions. For perturbation, the EZH2 inhibitor tazemetostat was evaluated in the CRPC-relevant C42 cell line with H3K27me3 readouts and transcriptomic profiling, with key changes validated by RT-qPCR.


Results
Across bulk cohorts, higher EZH2 expression was associated with adverse clinical outcomes and increased enrichment of immunosuppressive signatures, including Treg- and TAM-related programs. IMDEG-based NMF subtyping identified patient groups with distinct immune states, and an IMDEG-derived risk score stratified prognosis. Single-cell profiling revealed elevated EZH2 in CRPC malignant cells and Tregs; EZH2^high malignant cells exhibited a proliferative transcriptional state accompanied by reduced expression of immune-related programs. Predicted tumor-Treg interaction patterns were stronger in CRPC and positively associated with EZH2 expression. In C42 cells, tazemetostat reduced H3K27me3 and induced coordinated transcriptional changes, including upregulation of immune- and inflammation-associated genes such as TIMP3, PLCG2, and SOCS3, validated by RT-qPCR.


Conclusion
This multi-layer integrative analysis suggests that EZH2 is associated with proliferative malignant states and immunosuppressive microenvironment features in advanced PCa, including Treg-linked crosstalk. Transcriptomic profiling following EZH2 inhibition supports modulation of these programs by EZH2-targeted perturbation, while functional and causal mechanisms warrant further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e76f24369aa98ded1ada86e15969bb34cda77e5" target='_blank'>
              Integrative bulk and single-cell transcriptomics link EZH2 to immunosuppressive programs and tumor-Treg crosstalk in castration-resistant prostate cancer.
              </a>
            </td>
          <td>
            Xing Xiong, Jianhu Xie, Ping Dai, Chao Cheng, Jiani Liu, Guixiao Huang
          </td>
          <td>2026-02-03</td>
          <td>Frontiers in immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Stochastic and combinatorial expression of clustered Protocadherin (Pcdh) genes generates extraordinary cell-surface protein diversity, which provides individual neurons with a unique cell surface 'barcode' that functions in neuronal self/non-self discrimination. Recent advances in understanding the mechanisms by which individual neurons randomly express Pcdh genes have revealed critical insights into the function of the cohesin complex, its unloader WAPL, and the insulator protein CTCF in neural self/non-self discrimination. The unique genomic architecture of the Pcdh locus - where nearly 60 tandemly organized promoters compete for a handful of shared enhancers over nearly 1 million base pairs of DNA - have provided novel insights into how enhancers and promoters communicate, and how these interactions vary with genomic distances and between distinct cell-types. These studies highlight the importance of investigating cohesin and its DNA loop extrusion activity in mice, where the relative stoichiometries of the cohesin complex and its regulators, and thus their activities, differ among neural cell-types, and where molecular mechanisms can be directly linked to cellular physiology and brain development. Here, we review recent findings and discuss how the Pcdh gene cluster has become a new paradigm for the study of the molecular functions of cohesin, its role in the regulation of gene expression, and the implications regarding neuropsychiatric and neurodevelopmental diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45a3e362d4d0b3aa4e6bf98b115b3395456af302" target='_blank'>
              The role of cohesin and DNA loop extrusion in the generation of single neuron identity.
              </a>
            </td>
          <td>
            Lea Kiefer, Tom Maniatis, Daniele Canzio
          </td>
          <td>2026-02-17</td>
          <td>Current opinion in genetics & development</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Merkel cell polyomavirus (MCPyV) has been identified as the causative agent of Merkel cell carcinoma, and its non-coding control region (NCCR) has been demonstrated to play a critical role in regulating viral transcription. While NCCR variants exist, their comparative impact on bidirectional promoter activity remains poorly characterized. The present study conducted an in vitro evaluation of bidirectional transcription levels of five major MCPyV NCCR types (I, IIa-1, IIa-2, IIb, IIc). The NCCRs were subsequently cloned into a bidirectional reporter vector, which expresses green (EGFP, early) and red (RFP, late) fluorescent proteins. Subsequent to transfection into HEK293 cells, promoter activity was quantitatively analyzed via fluorescence imaging and flow cytometry. Bioinformatic analysis revealed high sequence similarity (> 94%) among the five NCCRs and predicted conserved transcription factor binding sites. The results indicated that all the variants exhibited stronger late promoter activity compared with the early promoter activity (p < 0.01). These observations are in alignment with the established biology of MCPyV. However, no statistically significant differences in the early/late transcription ratio or overall fluorescence intensity were observed between the different NCCR types under these conditions. These findings suggest that the core promoter function is conserved among these major NCCR variants in this model system. This study provides a foundational comparison of MCPyV NCCR activity, highlighting the need for further investigation in more physiologically relevant models to understand how NCCR diversity may influence viral pathogenesis in vivo. Moreover, incorporating models of viral genome integration is essential to understand mechanism of MCPyV carcinogenesis and viral-host interaction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/287fc6d54ee53d0d2c193c6a1dedcea18ce71955" target='_blank'>
              In vitro evaluation of bidirectional transcription levels of five types of non-coding control regions of Merkel cell polyomavirus
              </a>
            </td>
          <td>
            Han Mo, Xiaotong Qi, Xuan Wu, Liang Lu, Yong Ai, Xiaohua Tao, Xianfeng Zhou
          </td>
          <td>2025-12-31</td>
          <td>Virology Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Dysregulation of repair pathways, specifically the base excision repair (BER) axis, is frequently observed in cancer. This dysfunction is associated with accelerated tumor growth and reduced patient survival, suggesting that BER components may serve as potential targets for cancer therapy. This study focused on the expression of Uracil-DNA glycosylase (UNG), a key enzyme in BER, in colorectal tumor tissues and adjacent normal tissues. We systematically explored the clinical significance, biological functions, and molecular mechanisms of UNG in colorectal cancer (CRC). Our findings revealed that UNG was significantly upregulated in CRC, and its higher expression correlated with poorer patient outcomes. Functional assays demonstrated that silencing UNG inhibited cell proliferation, reduced migration and invasion capabilities, and induced apoptosis along with S/G2 phase cell cycle arrest. Mechanistically, UNG appears to interact with the AMPK, AKT/mTOR, and ERK signaling pathways. These results suggested that UNG exhibited oncogenic potential in CRC by regulating malignant behaviors and oncogenic signaling cascades, positioned it as a potential prognostic biomarker and therapeutic target for CRC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40071783d4dcf5a7a90c5996af36927f2f827940" target='_blank'>
              Silencing Uracil-DNA glycosylase inhibits colorectal cancer progression
              </a>
            </td>
          <td>
            Jingjing Yang, Li-zhou Shi, Yong-wei Hu, Hua Chen, Qing-hua Wang, Wei Han
          </td>
          <td>2026-01-04</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8bb1ad1f1afa1b592836b21a838884cdc277c82" target='_blank'>
              Targeting EZH2 Oncogenic Splicing: Decoding the Regulatory Network and Antisense Correction
              </a>
            </td>
          <td>
            Md Rafikul Islam, Preeti Nagar, Naomi McNaughton, Shabiha Afroj Heeamoni, Md Mahbub Hasan, Shila Kandel, Panagiotis Tsakiroglou, W. Dalton, Omar Abdel-Wahab, A. Krainer, M. A. Rahman
          </td>
          <td>2026-01-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6da8c58eace175b556f1165210d11f554c8a306d" target='_blank'>
              Regulatory Mechanisms of Maternal Imprinting at the Dlk1-Dio3 Domain
              </a>
            </td>
          <td>
            Ariella Weinberg-Shukron, Frances L Dearden, Ana Moreno-Barriga, Lok Ting Nick New, Marco Müller, Rebecca C. Rancourt, C. Edwards, A. Ferguson-Smith
          </td>
          <td>2026-01-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Tumor evolution involves genetic, transcriptional, and phenotypic alterations that shape cancer cell behavior and interactions with the microenvironment. While single-cell technologies have advanced our understanding of this process, spatial dynamics remain incompletely characterized. Here, whole-exome sequencing (WES), imaging mass cytometry (IMC), and spatial transcriptomics (ST) were integrated to study molecular evolution and immune responses in two lung adenocarcinoma (LUAD) mouse models: a genetically engineered model (129S4/Sv-KrasLSL-G12D, termed 129S4 K) and a carcinogen-induced precancer model (129S4 U). Compared to 129S4 K, 129S4 U tumors exhibited higher mutational, neoantigen but lower copy number variation (CNV) burdens at matched developmental timepoints, consistent with findings of higher mutational burden in human smoking-related LUAD than nonsmoking LUAD. We profiled over 1.4 million spatial single cells from 284 IMC regions of interest and 51,531 spatial transcriptomic spots from 156 lesions across 141 mice. Macrophage abundance increased with tumor progression, while CD8 T-cell and B-cell densities declined in late-stage LUAD. 129S4 U showed greater immune infiltration in both tumor and adjacent normal tissue, higher T-cell cytotoxicity signature score in line with its higher mutational and neoantigen burdens. LUAD progression was marked by early morphological shifts and late-stage changes in cell states and interactions. These data define spatial and genetic landscapes of LUAD development and provide a framework for investigating immune evolution and therapeutic strategies in early carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a0e7ad7c0953d4cab7fb1f9dcb251447c108ead" target='_blank'>
              Spatial Profiling Reveals Distinct Molecular and Immune Evolution of Mouse Lung Adenocarcinoma Precancers with or Without Carcinogen Exposure.
              </a>
            </td>
          <td>
            Bo Zhu, M. Aminu, Pingjun Chen, Jian-Rong Li, Chuanpeng Dong, Chenyang Li, Yanhua Tian, Shaowen Lu, Hong Chen, Chenxi Ma, Xin Hu, Jie-Tong Ye, Andrew Liu, Beibei Huang, Frank R. Rojas, Parra Cuentas Edwin Roger, Ou Shi, M. Nilsson, A. Poteete, Khaja B Khan, Wei Lu, Luisa M. Solis Soto, Junya Fujimoto, C. Haymaker, Ignacio I. Wistuba, Zhubo Wei, Linghua Wang, Don L. Gibbons, Ken Chen, Alexandre Reuben, Jason M Schenke, J. Heymach, Chao Cheng, Jia Wu, Jianjun Zhang
          </td>
          <td>2026-01-25</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="High-risk neuroblastoma (NB) remains one of the most difficult pediatric cancers to treat, particularly in patients with MYCN amplification, where survival outcomes have remained relatively stagnant despite aggressive multimodal therapy. In this project, I focused on understanding how MYCN amplification alters tumor biology at the transcriptomic level and whether these changes reveal actionable therapeutic vulnerabilities.
Using RNA-seq data from the TARGET-NBL cohort, I performed a differential expression analysis comparing MYCN-amplified and non-amplified high-risk tumors. I identified 1,842 significantly dysregulated genes, including a consistent 3.4-fold overexpression of ODC1 in MYCN-amplified samples. I then integrated this disease signature with LINCS L1000 perturbation profiles to identify FDA-approved drugs capable of reversing the observed expression pattern. This analysis highlighted Eflornithine (DFMO) as a strong candidate targeting polyamine metabolism. To better understand why such computational findings rarely translate into clinical use, I also conducted a structured survey of pediatric oncology professionals to assess real-world translational barriers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c09aaf83b7cc79af8709ef3204a330817c3d5a59" target='_blank'>
              Integrative Bioinformatics Analysis of Dyseregulated Pathways in Neuroblastoma to Identify Repurposable FDA-Approved Drug Candidates
              </a>
            </td>
          <td>
            Korounganba Rajkumar
          </td>
          <td>2026-02-05</td>
          <td>International Journal For Multidisciplinary Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d41e3dfbd14bf43361d23b6ad2843724145ac656" target='_blank'>
              Synthetic lethality and DNA damage response targeting in cancer stem cells: a comprehensive review.
              </a>
            </td>
          <td>
            Usamah Sayed, Waleed K. Abdulsahib, Ihsan Khudhair Jasim, H. Malathi, Priya Priyadarshini Nayak, D. Alex Anand, Gunjan Mukherjee, Aashna Sinha, Sadridin Eshkaraev
          </td>
          <td>2026-02-13</td>
          <td>Discover oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Brain tumors represent some of the most formidable challenges in neuro-oncology due to their aggressive clinical course, resistance to therapy, and profound molecular heterogeneity. Among the emerging regulatory elements reshaping our understanding of tumor biology are long non-coding RNAs (lncRNAs), a diverse class of RNA transcripts that modulate gene expression and cellular behavior without encoding proteins. This review provides an in-depth and integrative examination of the biogenesis, regulatory mechanisms, and functional roles of lncRNAs in brain tumor development and progression. We systematically explore both canonical and non-canonical pathways of lncRNA biogenesis, detailing how these influence structural specificity and molecular interactions. This review synthesized evidence retrieved from PubMed/MEDLINE, Scopus, and Web of Science, covering publications from January 2010 to June 2025. This analysis highlights key gaps, such as context-dependent therapeutic effects that limit translational applicability. A major focus is placed on the interplay between lncRNAs and core oncogenic signaling pathways, including Phosphoinositide 3-kinase (PI3K)/serine/threonine kinase (AKT), Signal Transducer and Activator of Transcription 3 (STAT3), Wingless/Int-1 (Wnt)/β-catenin, and Transforming Growth Factor-Beta (TGF-β), which drive malignant transformation, invasion, stemness, and therapeutic resistance in gliomas. Furthermore, we dissect the molecular functions of lncRNAs as epigenetic regulators, competitive endogenous RNAs (ceRNAs), and structural scaffolds, and discuss their contribution to the dynamic tumor microenvironment. By synthesizing the latest findings, this review underscores the academic and translational importance of targeting lncRNA-associated networks. It also highlights emerging therapeutic approaches, such as antisense oligonucleotides, RNA interference, CRISPR-Cas systems, and natural lncRNA-modulating compounds, which collectively represent a promising frontier in precision medicine for brain tumors. This work offers a critical framework for future research and therapeutic innovation in the lncRNA landscape of neuro-oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/068dee3e72b42160a1ffe0ae42698471932faad9" target='_blank'>
              Unraveling the Function of lncRNAs in Gliomas: Interaction With Signaling Pathways and Therapeutic Opportunities.
              </a>
            </td>
          <td>
            A. Doghish, Heba R. Ghaiad, Nourhan Elfar, N. H. El Said, Abdullah F. Radwan, Mai A Abd-Elmawla, Hend H. Mohamed, Osama A. Mohammed, Hanan A. Rizk
          </td>
          <td>2026-02-18</td>
          <td>Journal of biochemical and molecular toxicology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Simple Summary Transposable elements (TEs) are mobile genetic elements that are present in great numbers in the majority of eukaryotic genomes. They are major drivers of genome evolution as they can facilitate chromosome rearrangements, provide mechanisms for genomic shuffling, and contribute to genome expansion, thereby altering genome architecture. Additionally, TEs can also modify gene expression by disrupting regulatory sequences, impairing genes, and promoting the emergence of new sequences. Despite their abundance, identifying TEs is challenging and time-consuming due to their extreme diversity in DNA sequence. Many TE families are ancient, and most of their sequences have become inactive due to accumulated mutations and fragmentation. Consequently, different copies of the same TE can differ greatly from each other, which makes identifying decayed copies particularly difficult. In this work, we employ transformer architecture for TE classification. TEclass2 is an integrated classifier that rapidly predicts TE orders and superfamilies using models built on this advanced machine learning approach. The software is available through a web interface, allowing users to classify sequences into sixteen superfamilies according to the Wicker classification system, or alternatively, users can download the source code to train and build custom classification models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bf277b82c7226a65ad3cdcbe7a255c99eb31947" target='_blank'>
              Transformer-Based Classification of Transposable Element Consensus Sequences with TEclass2
              </a>
            </td>
          <td>
            Lucas Bickmann, Matias Rodriguez, Xiaoyi Jiang, Wojciech Makałowski
          </td>
          <td>2025-12-29</td>
          <td>Biology</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d11f8627925c6b14efde1341627645217602b269" target='_blank'>
              Molecular background of Philadelphia chromosome dependent enhancement of cellular growth and tyrosine kinase inhibitor sensitivity
              </a>
            </td>
          <td>
            Md Faruq Hossain, L. Hagenau, Lars R. Jensen, J. Rhode, Thomas Sura, M. G. Salazar, Ana Tzvetkova, Corinna Jensen, S. Edwards, Heiko Dunkel, Stefan Simm, Josefine Radke, A. W. Kuss
          </td>
          <td>2026-02-19</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="By generating 194 epigenomic and transcriptomic datasets from 57 human tissue samples using H3K27ac and HIF2α chromatin immunoprecipitation sequencing (ChIP-seq), assay for transposase-accessible chromatin using sequencing (ATAC-seq), and RNA sequencing, we provide a comprehensive, integrated characterization of clear cell renal cell carcinoma (ccRCC) across normal, tumor, and metastatic states. Our analyses provide several insights into ccRCC biology. First, we demonstrate significant reprogramming of enhancer and HIF2α cistromes as well as chromatin accessibility during the normal-to-tumor transition, whereas localized and metastatic tumors show minimal epigenomic differences. Second, we show reactivation of kidney-specific developmental pathways driving malignancy. Third, we perform a cistrome-wide association study in ccRCC, validating five established RCC risk loci and identifying six novel loci, including a locus at 12q24 linked to SCARB1 that was functionally validated. These datasets provide new perspectives on the role of developmental pathways in ccRCC tumorigenesis, insights into epigenetic mechanisms of ccRCC heritability, and a comprehensive epigenomic atlas for the research community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b37eefc44af89f924062f3fabaa4f6f9cb92f8ba" target='_blank'>
              Generation of a comprehensive epigenomic atlas in clear cell renal cell carcinoma informs kidney cancer progression and heritability.
              </a>
            </td>
          <td>
            Sarah Abou Alaiwi, E. Adib, Ziwei Zhang, S. Spisák, A. Prósz, Paulo Cordeiro, T. El Zarif, Z. Sztupinszki, Amin H. Nassar, , Rianna Campbell, J. Steinharter, Mitchell Machiela, Diptavo Dutta, S. Chanock, M. Purdue, M. Pomerantz, S. Signoretti, Zoltán Szállási, A. Gusev, S. Baca, T. Choueiri, M. Freedman
          </td>
          <td>2026-02-01</td>
          <td>Cell reports</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="Genomic alterations affecting components of the fibroblast growth factor (FGF) signaling axis can trigger aberrant pathway activation and tumor development. Genomic truncation of the FGF receptor 2 (FGFR2) exon 18 (E18) disrupts the FGFR2 carboxy-terminal tail (C-tail), acting as a potent driver alteration across multiple tumor types. Here, we analyzed human oncogenomic datasets to reveal that E18 truncations are similarly prevalent in FGFR3, an FGFR2 paralog. FGFR3 E18 truncations primarily occurred due to rearrangements (REs) that involve transforming acidic coiled-coil containing protein 3 (TACC3), resulting in FGFR3ΔE18-TACC3 gene fusions. In contrast to E18-truncated FGFR2, functional in vitro and in vivo examination of Fgfr3 variants demonstrated that the truncation of Fgfr3 E18 is insufficient to promote oncogenic activity in cell lines or in the lungs and mammary glands of mice. Only the combination of an Fgfr3 E18 truncation with a RE partner gene that encodes a receptor-dimerizing domain resulted in the development of tumors, which were sensitive to FGFR inhibition. Overall, these findings suggest that patients with cancers that are positive for rearranged FGFR3 resulting in E18 truncation and a fusion to dimerizing partners should be considered for FGFR-targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94a9640747833b06de97dd43802396e30e71e136" target='_blank'>
              C-terminal Truncation and Fusion Partner Determine Oncogenicity of FGFR3.
              </a>
            </td>
          <td>
            Julia Yemelyanenko, J. Bhin, Eline van der Burg, Anne Paulien Drenth, Jessica K. Lee, C. Lutz, Lea Dörner, E. Wientjens, S. Klarenbeek, Ji-Ying Song, H. Moon, S. Annunziato, Natalie Proost, Bjørn Siteur, J. S. Ross, M. van de Ven, O. van Tellingen, Shridar Ganesan, L. Wessels, D. Zingg, Jos Jonkers
          </td>
          <td>2025-12-29</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 Fusion oncoproteins generated by chromosomal translocations are defining molecular drivers of several pediatric cancers, functioning in part by reprogramming gene expression and disrupting cellular differentiation. Ewing sarcoma, an aggressive cancer of bone and soft tissue, exemplifies this paradigm and is driven in the majority of cases by the oncogenic fusion protein EWS::FLI1, formed by joining the low-complexity domain (LCD) of the RNA-binding protein EWS to the DNA-binding domain (DBD) of the transcription factor FLI1. EWS::FLI1 alters transcriptional and RNA processing programs and assembles into aberrant biomolecular condensates, yet the molecular basis for these fusion-driven assemblies remains poorly understood. Here, we integrate NMR spectroscopy, mutagenesis, biophysical and cellular assays, microscopy, and all-atom molecular dynamics simulations to dissect how sequence features and domain interactions shape the condensate properties of EWS and EWS::FLI1. We find that the EWS LCD undergoes extensive self-association mediated by tyrosine-rich motifs that generate compact intramolecular conformations and robust multivalent networks driving phase separation. The RNA-binding domains of EWS engage the LCD through transient electrostatic and aromatic interactions, modulating condensate dynamics, whereas the FLI1 DBD of the fusion protein directly interacts with the EWS LCD via flexible “wings”. Incorporation of the FLI1 DBD markedly alters the material properties of condensates, producing emergent biophysical behaviors distinct from either parental protein. These altered condensate properties possibly contribute to the cellular dysregulation and toxicity associated with EWS::FLI1, in part through mis-localization and aberrant interactions with native EWS that disrupt normal nuclear organization and RNA processing. Together, these findings define a molecular framework in which weak, multivalent, and chemically specific interactions give rise to the emergent properties of the EWS::FLI1 fusion protein. By linking sequence-encoded features, interdomain crosstalk, and condensate material states, we propose a mechanistic description of how fusion with ETS-family DBDs reprograms the physical behavior of EWS assemblies. These emergent condensate properties provide a potential unifying description of how EWS::FLI1 perturbs transcriptional regulation, RNA metabolism, and nuclear organization in pediatric Ewing sarcoma.



 David S. Libich, Erich J. Sohn, Kandarp A. Sojitra, Aiola Stoja, Courtney N. Johnson, Jeetain Mittal, Alex A J. Bishop. Emergent condensate behaviors of the EWS::FLI1 fusion protein in Ewing sarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B005.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e6d38099a1e2be11dc3ccd83fb0db5c31397244f" target='_blank'>
              Abstract B005: Emergent condensate behaviors of the EWS::FLI1 fusion protein in Ewing sarcoma
              </a>
            </td>
          <td>
            David S. Libich, Erich J. Sohn, Kandarp A. Sojitra, Aiola P. Stoja, Courtney N. Johnson, J. Mittal, Alex A J. Bishop
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/683398225f12a992fd68b945ba84e877fc29b6de" target='_blank'>
              Interpreting cancer genetics through a two-step "evolutionary cascade hypothesis": bridging neutral and selective perspectives.
              </a>
            </td>
          <td>
            A. Ottaiano, M. Santorsola, Francesco Sabbatino, Roberto Sirica, Francesco Caraglia, Anna Ceccarelli, Vincenza Granata, Ines Simeone, S. Zappavigna, M. Berretta, Giovanni Savarese, Michele Caraglia
          </td>
          <td>2026-02-13</td>
          <td>Journal of translational medicine</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Aneuploidy is prevalent across human cancers, yet its specific contributions to tumor evolution remain poorly understood. In this issue, Cross, Nowinski, Cresswell, Mossner, and colleagues present a longitudinal analysis of 755 samples from 167 patients with colorectal cancer, uncovering the temporal dynamics of karyotype evolution in this tumor type. See related article by Cross et al., p. 218.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8c1772974604d4bdebde97d913a898acbd365b8" target='_blank'>
              Holding the Line: Negative Selection Maintains the Karyotype in Metastatic Colorectal Cancer.
              </a>
            </td>
          <td>
            Hajime Okada, Rachel Slutsky, U. Ben-David
          </td>
          <td>2026-02-06</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2879e795e7efcebd514454af1fc1a736f45a29d8" target='_blank'>
              Somatic and germline mutational processes across the tree of life
              </a>
            </td>
          <td>
            Sangjin Lee, Yichen Wang, Haynes Heaton, Emily Mitchell, M. Maddison, Liam M. Crowley, Patrick Adkins, N. Mieszkowska, Mark Blaxter, R. Durbin, P. Campbell
          </td>
          <td>2025-12-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Oncogenic mutations of SF3B1 are common in myeloid cancers, chronic lymphocytic leukemia (CLL) and select solid tumors. Their mechanistic basis for promoting oncogenesis has been investigated in detail, with the stereotyped missplicing of mRNA protein coding sequences most intensively studied. These changes, in genes such as MAP3K7, BRD9, and ABCB7, typically lead to loss-of-function, thus contributing to cancer pathogenesis.Here we systematically analyzed the impact of mutant SF3B1 on non-coding regions of mRNA transcripts across disease types, in both cell lines and primary patient specimens. This identified numerous novel and highly reproducible splicing alterations in such regions. Studies of one target gene, DCAF16, revealed multiple complex mutation-induced alterations in its 5' and 3' untranslated regions (5', 3' UTRs). Remarkably, these were mechanistically associated with increased DCAF16 protein levels in SF3B1 mutant cells, representing the first time that oncogenic SF3B1 has been shown to increase levels of a target protein in a gain-of-function manner. DCAF16 is a substrate recognition adapter for the DDB1/CUL4 E3 ubiquitin ligase complex. Novel protein degrader small molecules which co-opt DCAF16 to degrade BRD4 as a neosubstrate demonstrated preferential selectivity for SF3B1 mutant cancers and CLL primary patient specimens due to increased DCAF16 protein levels. In turn, this reveals the therapeutic relevance of mutant SF3B1 dysregulation of transcript untranslated regions and uncovers a novel strategy for the treatment of these important neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc898e0fdc805e68eb2bae19245eee2675983440" target='_blank'>
              Oncogenic SF3B1 mutations alter the splicing of mRNA noncoding regions to induce a novel therapeutic vulnerability.
              </a>
            </td>
          <td>
            Michal Sekrecki, Agata Sekrecka, Rohan Lattupally, Kenny Le, Xiaokang Jin, Chen Mozes, Brendan G. Dwyer, Zhe Zhuang, Bryan A. Romero, J. Pineda, Xunhong Cao, Linh Nguyen, Vicky W Chen, Crystal Zhou, Jared A. Wallace, Kailee L Tanaka, C. Tiwari, Austin Gabel, Won Jun Kim, Robert F Stanley, Salima Benbarche, Jaspreet Thind, Abhinaya Muruganandham, T. Y. Zhang, Peter L Greenberg, J. Gotlib, G. Mannis, William Shomali, G. Salmasi, Calvin J Kuo, T. Shanafelt, Irtisha Singh, Daichi Inoue, Finn K. Hansen, N. Gray, Capucine Van Rechem, B. Fakhri, Xiaoyu Zhang, Sydney X. Lu
          </td>
          <td>2026-01-26</td>
          <td>Blood</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="


 Prostate cancer (PCa) is the second leading cause of cancer-related deaths among men in the U.S. Overexpression of PVT1 exon 9 has been shown to transform non-tumorigenic prostate epithelial cells into aggressive tumor-forming cells. We have recently identified a non-canonical promoter within PVT1 that is associated with PVT1 exon 9 expression and hypothesize that epigenetic factors like DNA methylation and transcription factors regulate its overexpression in prostate cancer, given that gene expression is modulated by epigenetic alterations. Using enzymatic methyl-sequencing (EM-Seq), we profiled tumor and adjacent normal prostate tissues. At a differential methylation threshold of ≥±5% and q ≤0.05, we identified 1050 differentially methylated cytosines (DMCs), with 108 hypomethylated and 80 hypermethylated sites in tumor tissues. Regional tiling analysis (1kb windows) also revealed 172 differentially methylated regions (DMRs), including 42 hypomethylated and 16 hypermethylated DMRs in tumor tissue. PVT1 exon 9-associated promoter was visibly hypomethylated in tumors relative to adjacent normal tissues. In vitro, 5-azacytidine (a DNA methyltransferase inhibitor) treatment induced significant upregulation of PVT1 exons 4A and 9 in WPE1-NA22, DU145, PC-3 and MDA-PCa-2b cell lines versus the non-tumorigenic RWPE-1 cell line. Computational transcription factor analysis revealed CDX1 binds to the newly-identified non-canonical promoter, while CRISPR-mediated base editing of its signal sequence reduced PVT1 exon 9 expression in PVT1 exon 9 overexpressing cells. In addition, CDX1 expression was also found to have a positive correlation with high grade PCa. These findings indicate that PVT1 exon 9 overexpression and oncogenicity in PCa is driven by promoter hypomethylation and CDX1 regulation.



 Chinedum C. Udekwu, Siti Khaula. Nurazizah, Seidu Adams, Bonnacci Rachel, E.Oluwabunmi Olapade-Olaopa, Olorunseun O. Ogunwobi. The oncogenic role of PVT1 exon 9 overexpression in prostate Cancer is mediated by DNA methylation and CDX1 [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A065.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9a9e20af1415b799b87aab15045ca553659c6c2" target='_blank'>
              Abstract A065: The oncogenic role of PVT1 exon 9 overexpression in prostate Cancer is mediated by DNA methylation and CDX1
              </a>
            </td>
          <td>
            Chinedum C. Udekwu, Siti Khaula. Nurazizah, Seidu Adams, Bonnacci Rachel, E. Oluwabunmi Olapade-Olaopa, O. Ogunwobi
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="


 Aberrant histone modifications and chromatin structure alter gene expression that drive phenotypic and functional behavior of aggressive prostate cancer. Technological limitations have constrained comprehensive analyses of epigenomic alterations to less relevant immortalized cell lines and pre-clinical models, as many current epigenomic assay techniques require hundreds of thousands of cells, thus limiting interrogation of clinical samples. To facilitate rare cell epigenetic analysis in patient derived cancer organoids (PDCOs) and circulating tumor cells (CTCs) from patients with prostate cancer, we miniaturized the Cleavage Under Targets and Tagmentation (CUT&Tag) assay with an Exclusive Liquid Repellency (ELR)-enabled “Lossless Altered Histone Modification Analysis System” (LAHMAS) device. CUT&Tag was developed to investigate protein-DNA interactions with high resolution, lower background and decreased cellular input requirements relative to ChIP but performance is less reliable for low inputs, making analysis of rare cell samples difficult for cell number <5000.



 Organoids were grown from primary tumor samples from 3 patient undergoing prostatectomy. There were a total of 5 loci collected. These samples were digested to single cells prior to assessment in the Cut&Tag assay with an antibody to H3K27Me3 or control IgG. Each sample was paired with RNAseq to assess gene expression.



 Performing CUT&Tag on the LAHMAS device increases the signal to for very low input samples, enabling the investigation of chromatin dynamics in rare cell sample types, including PDCOs and CTCs. We observed significantly higher FRiP scores using the microscale method compared to macroscale method (average 30.3% vs 18.2%, p<0.0001) facilitating accurate chromatin profiling of low input and rare cell samples. H3K27Me3 peaks are conserved from 3000 down to 50 nuclei in cell line models on the LAHMAS device. We are evaluated H3K27Me3 histone modifications from PDCOs grown from prostatectomy samples from 3 patiens using the LAHMAS device. We obtained high quality results from these samples, obtaining yields (LNCaP: 0.147-5.15 [median: 1.13] vs. PDCO: 0.144-5.36 [0.445]), and MACS2 FRiP scores (LNCaP: 1.79-29.91 [median: 4.91] vs. PDCO: 0.73-10.21 [5.76]) similar to cell line sample. We have also leveraged the LAHMAS device to evaluate circulating tumor cells from a patient with prostate adenocarcinoma progressing on chemotherapy. This patient demonstrated high expression of ASCL1 (0 tpm) and low expression KLK3 (16237.6 tpm) by RNAseq. Concordantly, we detected H3K27Me3 peaks at the promoter of ASCL1, consistent with the repression of transcription.



 The pairing of this LAHMAS microfluidic device with a highly sensitive CUT&Tag assay has enabled the investigation of chromatin dynamics in rare cells such as PDCOs and CTCs to evaluate the contribution of these molecular alterations in tumorigenesis and treatment resistance.



 Zachary J. Kauffman, Kevin Koesser, Kyle T. Helzer, Marina N. Sharifi, Erika Heninger, Emma E. Recchia, David Kosoff, David F. Jarrard, Shuang G. Zhao, Jamie M. Sperger, David J. Beebe, Joshua M. Lang. Lossless altered histone modification analysis system to investigate patient derived cancer organoids and circulating tumor cells from patients with prostate cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A030.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9c2af5b98402fcb05077456f840101469749964" target='_blank'>
              Abstract A030: Lossless altered histone modification analysis system to investigate patient derived cancer organoids and circulating tumor cells from patients with prostate cancer
              </a>
            </td>
          <td>
            Zachary J Kauffman, Kevin Koesser, K. Helzer, M. Sharifi, E. Heninger, Emma E Recchia, D. Kosoff, David F. Jarrard, Shuang G. Zhao, J. Sperger, David J. Beebe, Joshua M Lang
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7af8a77b3ff4fd7949033f0b161d8f776f7bbd9" target='_blank'>
              Epigenetic regulation by CBX7 in glioblastoma: molecular mechanisms and translational perspectives
              </a>
            </td>
          <td>
            Fatima Fayyaz, Gregorio Favale, Lucia Capasso, Rosaria Casalino, Donato Mele, Giulia Verrilli, Mariarosaria Conte, V. Carafa, A. Nebbioso, Lucia Altucci
          </td>
          <td>2025-12-29</td>
          <td>Epigenetics Communications</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6268adfd325a8d97365a3887d57675323be7812d" target='_blank'>
              Integrated analysis of extrachromosomal circular DNA and transcriptome reveals associations between microbial profiles and Immunoinflammatory markers in hypospadias
              </a>
            </td>
          <td>
            Zhilin Yang, Jiaying Yu, Xiaoning Hong, Jianchun Yin, Tiejun Zhang, Changze Song, Chunxiao He, Shuang Mao, Huisheng Yuan, Peng Han, Jiang Li, Xi Xiang, Shoulin Li, Zilong Wang
          </td>
          <td>2026-02-16</td>
          <td>World Journal of Urology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db9ee68261862574cb9b7d1551d82bf740d99cfd" target='_blank'>
              Elevated mutation in haploid yeast driven by translesion synthesis
              </a>
            </td>
          <td>
            Jacob Fredette-Roman, Denise R. Smith, Sanad B. Omari, Nathaniel P. Sharp
          </td>
          <td>2026-01-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ANLN is a highly conserved actin-binding protein that plays a critical role in cytokinesis by interacting with key cytoskeletal components such as actin, myosin, and RhoA. Increasing evidence shows that ANLN is aberrantly overexpressed in various cancers, including lung, breast, and liver cancers, and that its elevated expression is associated with enhanced tumor cell proliferation, migration, and invasion. Because of its central involvement in tumorigenesis and disease progression, ANLN has emerged as a promising prognostic biomarker and potential therapeutic target. Recent studies have demonstrated that ANLN contributes to resistance to chemotherapy, targeted therapy, and immunotherapy through multiple molecular mechanisms. This review provides a systematic overview of the physiological functions of ANLN, its roles in cancer initiation and progression, and its regulatory mechanisms in treatment resistance, offering biological insights into precision oncology and potential strategies for overcoming therapeutic resistance in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27a40af0d42ab2b0397c3e4bc1fc5ecc54b07af4" target='_blank'>
              The role of ANLN in malignant tumors: pathogenesis, treatment resistance and targeted strategies
              </a>
            </td>
          <td>
            Yixuan Wang, Yang Xiao, Jielin Yu, YuanHua Zou, Zhe Wang, Xianghong Yang
          </td>
          <td>2026-01-07</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Clinical divergence between patients harboring CIC-rearrangements is frequently observed. For example, the prototypical CIC::DUX4 fusion associates with soft tissue tumors while CIC::NUTM1 fusions typically localize to the central nervous system (brain/spinal cord). The basis for these differences is poorly understood due to a lack of molecular tools. To address this need, we generated patient-informed, synthetic coding sequences for CIC::NUTM1, CIC::LEUTX, and ATXN1::DUX4 and validated them in structure-function studies and in genetic zebrafish models. We found that CIC::NUTM1 drives a transcriptional program distinct from that of CIC::DUX4 due to a C-terminal NUTM1 functional domain, CIC::LEUTX weakly activates CIC target genes through LEUTX transactivation sequences, and ATXN1::DUX4 upregulates CIC target genes via the ATXN1 AXH domain. Our findings indicate that the CIC fusion binding partner may alter overall fusion oncoprotein activity. Implications: These first-generation synthetic tools illuminate partner gene-specific mechanistic biology while providing an unprecedented resource to study CIC-family fusions beyond CIC::DUX4 and allow for the dissection of this rare subgroup of cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2ad85fdf7e4bbaff65e1119c97ad2343b464e58" target='_blank'>
              Modeling of Capicua Family Fusion Oncoprotein-Driven Cancers Reveals Gene-Specific Functionality.
              </a>
            </td>
          <td>
            Cuyler Luck, Yongfeng Luo, E. Vasileva, Kyle A. Jacobs, Julia Riad, Christopher D Macaraig, R. Ponce, James F. Amatruda, Ross A. Okimoto
          </td>
          <td>2026-01-23</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2da359f1145632202e29954441b9ab9dc80c306" target='_blank'>
              KIF20A inhibits TRIM21-dependent ubiquitination of DHX9 to boost SOX2 stability, enhancing OSCC stemness and ferroptosis resistance
              </a>
            </td>
          <td>
            Ziyun Zhang, Yi Li, Jingjiang Hu, Xingjie Tang, Zhanpeng Li, Xiaoyan Sun, Dade Feng, Yixin Yao, Chao Mao, Yongguang Tao, Li Xie, Huaiqing Luo, Yufei Li, Xing Yu, Xiaoning Peng, Li Cong, Yiqun Jiang
          </td>
          <td>2026-02-11</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db19d348df7947bfaa58517d856ab10f519b71ee" target='_blank'>
              Epigenomic landscape of nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            Saleh Suleiman Silmi Almohammadin, R. B. S. M. N. Mydin, R. Bahar, Denisha Sivasamy, Muhamad Yusri Musa
          </td>
          <td>2025-12-26</td>
          <td>Medical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Primary hepatic angiosarcoma is a very rare and extremely aggressive malignancy that has hadfew therapeutic options and has been closely linked to aristolochic acid which is a strongenvironmental carcinogen commonly found in some herbal medicines. The hallmark mutationalsignature of aristolochic acid is the observed A: T↔ T: A transversion at context-specifictrinucleotide motifs, and this pattern can be used to discriminatively identify AA-induced tumorswith genome-wide analyses. This signature can act as a molecular signature that has the potentialto explain the etiology of cancers, can aid retrospective exposure assessment, as well ascontribute to early diagnosis. Coincidently, the molecular profiling has highlighted newbiomarkers, serine/threonine kinase p53 pathways, aberrations of ERCC1 DNA repair enzyme,and up-regulation of endothelial cell markers. which have collectively enhanced diagnosis andstratification outcomes. These markers have been tested as potential predictive targets andprovide the basis of precision approaches that encompass PARP inhibitors that target DNA-repairdefects, targeted kinase inhibitors, and monoclonal antibodies against angiogenic pathways.Combination therapies that include the targeted with conventional chemotherapy have beenshown to be more effective. Even more, the development of innovative technologies, includingliquid biopsy, single-cell sequencing, and artificial intelligence-mediated searches in mutationalsignatures, further expand personalized medicine of Primary hepatic angiosarcoma and allow anearlier manifestation and precision intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63b9f27b6bcd4ee7287251279079994596ebf2bc" target='_blank'>
              Uncovering Aristolochic Acid–Driven Mutational Signatures and Novel Biomarkers inPrimary Hepatic Angiosarcoma for Targeted Therapeutic Development
              </a>
            </td>
          <td>
            Ramzan Mohammad, Jasvir Kaur, Ruhit Ashraf, Shafkat Hussain Malik, Pursuing B. Pharmacy
          </td>
          <td>2026-01-10</td>
          <td>Pan-African Journal of Health and Psychological Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da7a9e1a9e3edf4edb1c29327ece5ed4b0de7ab1" target='_blank'>
              Colorectal cancer risk variants in the 11q13.4 locus are associated with variable POLD3 transcript expression which may promote DNA damage and telomere shortening in colorectal cancer cells
              </a>
            </td>
          <td>
            E. Clarkson, N. DeLeoN, S. Aldulaimi, J. Fernández-Tajes, T. Roberts, I. Tomlinson, A. Lewis
          </td>
          <td>2025-12-27</td>
          <td>None</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Oncofetal reprogramming has recently emerged as a critical concept in translational cancer research, particularly for its role in driving therapeutic resistance across a variety of malignancies. This biological process refers to a pattern of gene expression that is restricted to embryogenesis, but becomes expressed again in a subpopulation of cancer cells. These genes are typically suppressed after embryogenesis, and their aberrant re-expression in tumors endows cancer cells with stem-like properties and enhanced adaptability. The goal of this review is the following: (i) comprehensively examine the multifaceted nature of oncofetal reprogramming; (ii) elucidate its underlying molecular mechanisms, including its regulators and effectors; and (iii) evaluate its consequences for the therapeutic response in different cancer types. We comprehensively integrate the latest findings from colorectal, breast, lung, liver, and other cancers to provide a detailed understanding of how oncofetal programs interfere with tumor response to treatment. Among the candidates, YAP1 and AP-1 have emerged as central transcriptional drivers of this reprogramming process, especially in colorectal and breast cancers. We also explore the distinct expression patterns of oncofetal genes across different tumor types and how these patterns correlate with treatment outcomes and patient survival. Lastly, we propose a dual-targeting therapeutic strategy that simultaneously targets both cancer stem cells and oncofetal-reprogrammed populations as a more effective approach to overcome resistance and limit recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a262b7681553fb103b5d82c8b95c6fdfada4bf5" target='_blank'>
              Oncofetal Reprogramming: A New Frontier in Cancer Therapy Resistance
              </a>
            </td>
          <td>
            Anh L Nguyen, Molly Lausten, Bruce M. Boman
          </td>
          <td>2026-01-29</td>
          <td>International Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Rhabdomyosarcoma (RMS) is an aggressive pediatric soft tissue cancer, with approximately 350 cases per year in the US. The highly aggressive tumor subtype, Fusion-Positive Rhabdomyosarcoma (FP-RMS) is characterized by chromosomal translocations yielding oncogenic fusion transcription factors (TFs). The most common RMS fusion (PAX3::FOXO1) forms an in-frame chimera of the DNA-binding domain of PAX3 with the activator domain of FOXO1. We hypothesize that understanding the precise mechanisms of PAX3::FOXO1’s chromatin engagement and nucleosome invasion would enable precision therapy to target these mechanisms to ultimately improve patient outcomes. Our studies are providing evidence that PAX3::FOXO1 has pioneer activity, including nucleosome engagement, and an ability to recognize multiple chromatin states including both open and repressive chromatin regions. We are investigating the biochemical regulatory functions of PAX3::FOXO1 binding to specific DNA motifs embedded within chromatin or nucleosomes, developing synthetic epigenetics to investigate motifs from our genomics-scale studies. We are interested in the preservation of naturally occurring post-translational modifications from the mammalian cellular context, and have sought to build a highly versatile system for investigation of TF/nucleosome targeting from cell extracts. In our studies, we are finding new evidence for PAX3::FOXO1’s targeting of homeodomain and paired domain DNA motifs in free DNA and also in a nucleosome context. We are validating these findings through parallel approaches including coupling electrophoretic mobility shift assays (EMSAs) with westerns and antibody-targeting of complexes within EMSA experiments. We are finding that PAX3::FOXO1’s binding to high-complexity motifs, conjoining paired domain and homeodomain sequences, occurs during nucleosome invasion. Through iterative comparisons, custom unmodified mono-nucleosomes containing three major classes of DNA binding motifs reveal PAX3::FOXO1’s binding to these nucleosomes without a strong motif preference within the categories examined in our series. These findings support a mechanistic hypothesis where PAX3::FOXO1 may invade nucleosomes, altering their local positioning prior to generation of chromatin accessibility. Additional investigations are underway to unravel these mechanisms further and probe the contexts for motif specificity or generality in nucleosome engagement, and to understand conformational changes in the target nucleosomes. The mechanisms we are uncovering will contribute to the search for new therapeutic vulnerabilities as our community progresses toward targeted therapies for FP-RMS.



 Chamithi Karunanayake, Hayden Statmore, Ehsan Akbari, Alexi Tallan, Frederic Barr, Michael G. Poirier, Benjamin Stanton. Biochemical mechanisms of PAX3::FOXO1 chromatin invasion in Rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B016.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dbbbd914885e1df8de9eefd492cd42d843d809e" target='_blank'>
              Abstract B016: Biochemical mechanisms of PAX3::FOXO1 chromatin invasion in Rhabdomyosarcoma
              </a>
            </td>
          <td>
            Chamithi Karunanayake, Hayden Statmore, Ehsan Akbari, Alexi Tallan, Frederic G. Barr, Michael G. Poirier, Benjamin Z. Stanton
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Aging is closely linked to epigenetic remodeling, with DNA methylation (DNAm) emerging as a robust biomarker for estimating epigenetic age (EA) and quantifying senescence. Dysregulation of aging‐associated DNAm has been implicated in diverse pathologies, including acute myeloid leukemia (AML). However, the effect of these epigenetic alterations in diseases and the underlying mechanism remains largely uncharacterized. Using causality‐enriched epigenetic clocks, we identified that adaptive DNAm dynamics are sensitive to short‐term therapeutic intervention in treating AML and may exhibit adaptive effects linked to better health outcomes. Subsequently, integrative genomic analysis showed significant associations between epigenetic aging and recurrent AML driver mutated genes, particularly DNMT3A and IDH2. The elevated adaptive aging associates with improved overall survival in cytogenetically normal AML harboring these mutations, highlighting its prognostic value in specific genomic contexts. Mechanistic analysis demonstrated that differentially methylated CpG sites in mutated gene‐specific AML subtypes are enriched at polycomb repressive complex 2 (PRC2) targets. These findings link mutation‐specific epigenetic aging, PRC2‐mediated methylation dynamics, and AML pathogenesis, offering insights into how aging‐related epigenetic dysregulation fosters malignant transformation. This study shows that AdaptAge can help reveal AML‑related DNAm dynamics when combined with genetic stratification, suggesting a path toward future biomarker development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64cddd44f88f3fd69ce34359b187ca201d02ef77" target='_blank'>
              PRC2‐Related Epigenetic Age Acceleration in Acute Myeloid Leukemia with DNMT3A and IDH2 Mutations
              </a>
            </td>
          <td>
            Zhengyi Yan, Luowei Yuan, Jinxing Wang, Shen Gu, Yong-Xing Lei
          </td>
          <td>2026-01-01</td>
          <td>Advanced Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Lung cancer remains a leading cause of cancer-related mortality owing to its aggressiveness and pronounced heterogeneity. Cancer stem cells (CSCs) have been implicated in tumor progression and therapeutic resistance; however, their transcriptional programs and spatial organization in non-small cell lung cancer (NSCLC) are not fully characterized. Here, we integrated single-cell RNA sequencing with spatial transcriptomics to systematically define CSC-associated phenotype in NSCLC. Weighted gene co-expression network analysis identified a 127-gene CSC signature enriched for stemness-related pathways, with DSG2 emerging as a dominant marker. Elevated DSG2 expression was associated with chemotherapy resistance and predicted response to EGFR tyrosine kinase inhibitors based on independent clinical proteomic datasets. Spatial mapping revealed preferential enrichment of DSG2+CSCs at tumor margins, where they co-localized with FAP+myofibroblasts (myCAFs). Multiplex immunofluorescence demonstrated that FAP+myCAFs expressed MMP9 and MMP12, consistent with a microenvironment supportive of epithelial-mesenchymal transition and CSC maintenance. Functional co-culture assays showed that myCAFs enhanced CSC-associated phenotypes in DSG2^high tumor cells in an MMP-dependent manner. Collectively, these findings delineate a spatially organized DSG2+CSC-myCAF niche that contributes to therapeutic resistance in NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a5313526bb6f0160b56c7fabe77fa7494a3d29e" target='_blank'>
              DSG2+ Cancer Stem Cells Co-Located With FAP+ Myofibroblasts in the Tumor Boundary That Determines the Efficacy of Immunotherapy in Non-Small Cell Lung Cancer.
              </a>
            </td>
          <td>
            G. Fan, Le Tang, Tongji Xie, Mengwei Yang, Lin Li, Sheng Yang, P. Xing, Xiaohong Han, Yuankai Shi
          </td>
          <td>2026-02-15</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fef918e1d4b64862104d2dc24b6aa7569489a94" target='_blank'>
              Genomic profiling implicates candidate genes and mutagenic pathways driving lung cancer recurrence
              </a>
            </td>
          <td>
            L. Luhari, A. Valter, A. Bahcheli, K. C. Cheng, M. Bayati, A. Ustav, A. Velthut-Meikas, K. Oselin, J. Reimand
          </td>
          <td>2026-01-01</td>
          <td>None</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="A large number of reports have suggested that the cancer is associated with the gene mutations and chromosomal aberrations, and thus it has been a focus of research. Today new insights are proposed by analyzing relevant literature, suggesting that the cancer is not caused by the gene mutations and chromosomal aberrations, but by the regulatory dysfunction of gene expression resulted from the nuclear damage, among which, the gene mutations and chromosomal aberrations are only an accompany phenomena also resulting from the nuclear damage. The gene mutations generally do not cause diseases, but if they do, they mainly cause the single-gene diseases. The chromosomal aberrations usually refer to the abnormalities in chromosome structure and number, often leading to cell death or inducing various chromosomal diseases. Some chronic refractory diseases, such as the cancer, Alzheimer's disease, degenerative diseases, are probably not caused by the gene mutations and chromosomal abnormalities, but rather by the abnormal regulation of gene expression and the dysfunction of DNA transcription-protein synthesis resulted from the nuclear damage. These diseases are all the diseases of nuclear dysfunction and collapse of differentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4826bbb55a85cfdd2750ad0771ebe3b7d95aef3d" target='_blank'>
              Differences and Pathogenicity Between the Gene Mutation and Chromosomal Aberration
              </a>
            </td>
          <td>
            Maojin Li
          </td>
          <td>2025-12-29</td>
          <td>Medical Research Frontiers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Ewing sarcoma is the second most common childhood bone cancer. In approximately 85% of cases, the tumor is caused by expression of the fusion oncoprotein EWSR1::FLI1. EWSR1::FLI1 is generated from a chromosome translocation event between chromosomes 11 and 22. This combines the low complexity domain (LCD) of EWSR1 with the DNA-binding domain (DBD) of FLI1. Previous genomic studies have shown that EWSR1::FLI1 is a potent genome regulator at GGAA microsatellites. However, due to the technical difficulty in purifying the full-length EWSR1::FLI1, there has been a lack of biochemical studies characterizing the properties of the fusion protein. In this study, we successfully purified the full-length EWSR1::FLI1 and conducted a series of in vitro biochemical and fluorescence-based experiments to investigate its interaction with DNA and nucleosomes. We found that truncated EWSR1::FLI1 shows pioneering activities, and an alpha-helix in the FLI1 DBD further facilitates its binding to nucleosomes. Surprisingly, we observed that the DNA-binding sequence preference differs between full-length and truncated proteins. This suggests that the EWSR1 LCD might also be involved in regulating sequence-specific DNA binding. Our findings provide new insights into how EWSR1::FLI1 binds to DNA and nucleosomes as a pioneer factor to alter genome regulation in Ewing sarcoma.



 Ruo-Wen Chen, John E. Tokarsky, Andrea K. Byrum, Megann A. Boone, Michael G. Poirier, Emily R. Theisen. Full-length EWSR1::FLI1 reveals low-complexity domain regulation of DNA and nucleosome binding [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B019.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33e8c926b6591529a86791c98a3602fff2c11724" target='_blank'>
              Abstract B019: Full-length EWSR1::FLI1 reveals low-complexity domain regulation of DNA and nucleosome binding
              </a>
            </td>
          <td>
            Ruo-Wen Chen, Jack Tokarsky, Andrea K. Byrum, M. Boone, Michael G. Poirier, Emily R. Theisen
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6f559d73997500b8aa91fc53cd9abd2c2f32134" target='_blank'>
              YBX3 overexpression in mesothelioma drives aberrant cell proliferation
              </a>
            </td>
          <td>
            Angela Rubio, Robert F. Harvey, A. Craxton, Mark Southwood, C. Ficken, Catarina Franco, Lajos Kalmár, Goretti Muñoz, Ian R Powley, Stephan Kamrad, Rui Guan, David Fernandez-Antoran, K. R. Patil, John Le Quesne, Marion MacFarlane, Anne E Willis
          </td>
          <td>2026-02-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 Epigenetic reprogramming is a major contributor to therapy resistance in prostate cancer (PCa). Indeed, chromatin accessibility profiling classified PCa into distinct subtypes, predictive of clinical outcome. Histone modifications influence nucleosome positioning, ultimately dictating chromatin compactness. The histone methyltransferase G9a, frequently upregulated in advanced PCa, catalyzes a repressive histone modification (di-methylation of histone H3 lysine 9) that signals for gene transcriptional repression. CM-272 is a reversible and highly selective inhibitor against G9a activity, and we have previously shown in cell line models of PCa that it depicted strong anti-tumoral effects. However, fundamental aspects of altered transcriptomic programs remain unexplored. Here, we validated, in vivo, the CM-272 effects in advanced PCa, and explored the molecular mechanisms underscoring G9a inhibition-mediated effects. Specifically, the PCa patient-derived xenograft model BM18 was used to assess the mode of action of CM-272 in vivo. Moreover, RNA sequencing of tumor tissues was performed to further explore the underlying gene expression signature. Finally, chromatin immunoprecipitation followed by qPCR was used to investigate the underlying functional consequence of epigenetic targeting. Our results showed that CM-272 significantly delayed BM18 tumor growth, specifically impairing tumor cell G9a activity and proliferation. Transcriptomic profiling revealed 119 significantly upregulated and 99 downregulated genes in response to G9a inhibition. Notably, by integrating our gene expression dataset with published chromatin accessibility data, an increase in the androgen receptor (AR) signature and a concomitant decrease in the neuroendocrine (NE) features were observed in CM-272-treated samples. Additionally, pathway enrichment and master regulator analysis pinpointed both the androgen receptor and TP53 networks as significantly upregulated in treated tissues. In line with the reduction of the NE score, the MYC/MYCN were the most significantly downregulated transcription factors. Moreover, under the androgen pathway, upregulation of AZGP1, STK39, and DPP4 was observed. At the promoter level, a significant reduction in H3K9me2 levels was detected in treated samples, allowing, in turn, an increase in AR binding. Additionally, upon treatment, higher p53 binding was detected at the promoter of the p53-repressed genes AURKA, CDC25B, and CD44, whose expression was downregulated. Overall, targeting G9a with CM-272 might hold promising therapeutic potential for the management of advanced disease by modulating relevant tumoral transcriptional programs associated with core vulnerabilities in PCa.



 Filipa Moreira-Silva, Mark Zimmerli, Panagiotis Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio. G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Innovations in Prostate Cancer Research and Treatment; 2026 Jan 20-22; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(2_Suppl):Abstract nr A045.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5911786a7c980e0c86fa5660d05b013ec96ea309" target='_blank'>
              Abstract A045: G9a blockade uncovers therapeutic vulnerabilities in advanced prostate cancer, leading to tumor growth delay
              </a>
            </td>
          <td>
            Filipa Moreira-Silva, Mark Zimmerli, P. Chouvardas, Beat Roth, Carmen Jerónimo, Marianna Kruithof-de Julio
          </td>
          <td>2026-01-20</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="This study aims to characterize the mutational profiles of the TP53 and CYP17A1 genes in Senegalese patients with colorectal cancer (CRC) in order to better understand the local molecular mechanisms of tumorigenesis. The analysis included 24 patients with CRC and 24 healthy controls. The sequences of exon 4 of TP53 and the promoter region at exon 1 of CYP17A1 were concatenated, then subjected to genetic structuring, historical demography, and phylogenetic relationship analyses using DnaSP (v5.10), MEGA (v7.014), and Arlequin (v3.1). The results reveal significant genetic differentiation between cancerous and healthy tissues (FST = 0.113, p = 0.009), as well as greater genetic diversity within tumors. Neutrality tests (Tajima's D and Fu's FS) indicate recent demographic expansion in the tumor population, with an excess of rare variants. The multimodal distribution of mismatches and the haplotype network confirm this evolutionary dynamic, marked by the emergence of haplotypes specific to cancerous tissues. These data suggest that the accumulation of mutations in TP53 and CYP17A1 contributes to genetic heterogeneity and CRC progression in the Senegalese population, opening up prospects for targeted therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff0b969d6ec5ed2bb17722c362958a0e34367f77" target='_blank'>
              Evolutionary Signature and Genetic Structure of Concatenated TP53-CYP17A1 Genes in Colorectal Cancer in Senegal
              </a>
            </td>
          <td>
            Anna Ndong, Bineta Kénémé, M. Sembène
          </td>
          <td>2026-01-10</td>
          <td>European Journal of Applied Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Comparative transcriptome analyses of gamma herpesviruses have been revolutionized by long‐read sequencing and integrative multi‐omics approaches, enabling unprecedented resolution of viral gene expression. Cross‐species studies of Kaposi's sarcoma‐associated herpesvirus (KSHV), Epstein–Barr virus (EBV), and murine gamma herpesvirus 68 (MHV68) have uncovered a vast diversity of transcript isoforms, including alternative 5′/3′ UTR variants, alternative splice isoforms, and transcripts encoding truncated or extended coding sequences. These efforts revealed novel transcript isoforms harboring partial open reading frames, entirely new ORFs, and an expanding repertoire of viral non‐coding RNAs. Additional regulatory mechanisms, such as promoter dependence on viral replication factors, selective polyadenylation site usage, frameshifting via alternative splicing, and extensive readthrough transcription, further illustrate the complex strategies governing gamma herpesvirus transcriptomes. This review synthesizes these discoveries, highlighting transcriptional strategies employed by gamma herpesviruses for persistence through different phases of replication, while outlining how integrative transcriptomics has reshaped our understanding of herpesvirus gene regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8150269b600bc18ee3d2193456ba4418ddf8a3de" target='_blank'>
              Cross‐Species Insights Into Gamma Herpesvirus Transcriptomes: Long‐Read and Multi‐Omics Perspectives
              </a>
            </td>
          <td>
            Ritu Shekhar, Sarah McMahon, S. Tibbetts, Erik K. Flemington, Rolf Renne
          </td>
          <td>2026-01-01</td>
          <td>Journal of Medical Virology</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 Ewing sarcoma is a pediatric bone and soft tissue cancer thought to arise from mesenchymal stem cells (MSCs). It is characterized by fusion of the RNA-binding protein EWS to an ETS family transcription factor, most often FLI1. EWS::FLI1 induces changes in gene expression that have been well-studied and have been linked to its ability to bind to and alter chromatin state at GGAA repeat sequences within the genome. In addition to these effects, EWS::FLI1 alters mRNA splicing; however, the mechanism of this activity remains unclear.



 We curated a sarcoma-focused transcriptome from published long-read RNA sequencing of Ewing sarcoma, osteosarcoma, and synovial sarcoma patient tumors and matched adjacent normal tissue (215M reads). We supplemented this with published long-read RNA sequencing data from relevant normal cell types, osteosarcoma cell lines, and Ewing sarcoma lines with or without EWS::FLI1 knockdown (105M reads). We used this transcriptome as a reference to evaluate expression and splicing from short-read RNA-seq of Ewing sarcoma cell lines, MSCs, and MSC-based models of Ewing sarcoma and established Ewing-associated expression and splicing signatures.



 We generated scores based on these signatures in published knockdown-rescue experiments in which endogenous EWS::FLI1 was silenced by shRNA and replaced with exogenous shRNA-resistant mutants. Expression and splicing scores were highly correlated within these datasets (r2 = 0.89), suggesting that similar regions of the fusion protein might be required for both its functions. In contrast, the scores had limited correlation within an internal dataset of RNA-binding protein perturbations (r2 = 0.02). We next examined the effects of pharmacological agents thought to alter EWS::FLI1 activity. Interestingly, we found that lurbinectedin which disrupts EWS::FLI1’s DNA-binding could partially revert both expression and splicing scores. In contrast, YK-4-279 which disrupts the fusion’s protein-protein interactions and MS0621 which reverts a Ewing-associated chromatin signature had limited effects on these scores. To further interrogate the potential role of EWS::FLI1 DNA-binding in splicing regulation, we silenced EWS::FLI1 in TC-32 cells and replaced it with either the wildtype fusion or a DNA-binding null mutant. Unlike the wildtype construct, the DNA-binding null mutant could not rescue expression or splicing. Finally, we examined the effects of the previously published KRAB-ZFA7 repressor which silences chromatin at GGAA repeats while sparing single GGAA sites and found that it could revert both scores.



 Together, these data establish a potential connection between EWS::FLI1 activity at GGAA repeats and its effects on mRNA splicing. Additionally, we find that lurbinectedin, an FDA-approved agent currently in a phase 1/2 trial for Ewing sarcoma, suppresses both the splicing and expression effects of EWS::FLI1. Emerging compounds designed to target EWS::FLI1 DNA-binding directly will likely have similar effects.



 David V. Allegakoen, Jeffrey A. Toretsky. EWS::FLI1 activity at GGAA repeats is essential for maintenance of Ewing-like splicing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr B013.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6934f37ed326d496d296a4806fccea6f323a4f63" target='_blank'>
              Abstract B013: EWS::FLI1 activity at GGAA repeats is essential for maintenance of Ewing-like splicing
              </a>
            </td>
          <td>
            D. Allegakoen, J. Toretsky
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26ac9c72d0a2bb774d37193df575cf880cc8777d" target='_blank'>
              Methylome and transcriptome mapping reveal miniscule DNA methyltransferase regulons in Salmonella enterica serovar Typhimurium
              </a>
            </td>
          <td>
            A. Ershova, Charles H. Howard, K. Hokamp, Andrew D. S. Cameron, Carsten Kröger
          </td>
          <td>2026-01-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7927789f4e0744dcfe5a0ffaabff6993c035def0" target='_blank'>
              Intratumor microbiota Delftialacustiris correlated with METTL3 to control development of papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Kai Qiu, Qingji Xie, Deye Zeng, Yu Huang, Ting Lin
          </td>
          <td>2026-01-29</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee5f3ed947fc38c6b718d5944ddaa9a13ea9180f" target='_blank'>
              ATRX loss couples genome instability at a G-rich repeat to dysregulation of human alpha-globin expression.
              </a>
            </td>
          <td>
            Yuqi Shen, Kinam Gupta, Sue Mei Tan-Wong, Sean Wen, Chris Fisher, Liezel Tamon, Nicholas J Proudfoot, R. Gibbons, D. Higgs
          </td>
          <td>2026-02-14</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/468c034edcdec5179082358a5f9944277909facb" target='_blank'>
              Identification of DNA methylation based prognostic subtype and signature in epithelial ovarian cancer
              </a>
            </td>
          <td>
            Jingjing Zhou, Yuxin Chen, Wei Liu, Chao Wang, Lei Liu, Wei Geng, Linqing Chai, Dong Yang, H. Dai, Haixin Li, Yanrui Zhao, Kexin Chen, Hong Zheng, Lian Li
          </td>
          <td>2025-12-27</td>
          <td>Human Genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Introduction: Endometrial cancer is the most common gynecological neoplasm, with its origins closely linked to genetic alterations. This study highlights mutations in genes involved in DNA repair, especially the mismatch repair system and the BRCA1/2 genes, as central elements in the development of the disease, its molecular profile, and response to treatments. Understanding these genetic impacts is crucial to improving diagnostic methods and driving the development of personalized therapeutic approaches.
Methodology: A literature review was conducted, searching for articles in the PubMed and Scopus databases. Inclusion criteria included molecular, clinical, and epidemiological studies that established a correlation between mutations in DNA repair genes and endometrial cancer, published in the last ten years. The selected studies were analyzed to gather information on the prevalence of mutations in the mismatch repair system and BRCA1/2 genes, the frequency of microsatellite instability, the clinical and molecular profiles of the tumors, and the impact of these alterations on the response to therapies, especially immunotherapy.
Results: The results of the review indicate that mismatch repair deficiency, caused by germline mutations or epigenetic methylation, represents a relevant molecular event in approximately 20 to 30% of endometrial cancer cases. This deficiency is associated with microsatellite instability and a hypermutated phenotype, characterized by a high mutational burden. This condition makes the tumor more immunogenic, favoring a positive response to immunotherapy with immune checkpoint inhibitors. Furthermore, germline mutations in the MMR genes are linked to Lynch Syndrome, which significantly increases the risk of developing endometrial cancer in young women. Mutations in the BRCA1/2 genes have also been identified, especially in the more aggressive serous subtype, suggesting a role in homologous recombination deficiency and tumor progression.
Conclusion: Genetic alterations in genes responsible for DNA repair play a key role in the molecular biology of endometrial cancer. They not only determine specific molecular subtypes of the disease but also serve as valuable indicators for diagnosis, prognostic assessment, and patient selection for targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5c325770f41a4ab3814fbe13355802187ffd82c" target='_blank'>
              Factors in Genetic Alterations in DNA Repair Pathways and Their Effects on Endometrial Cancer
              </a>
            </td>
          <td>
            Mari Uyeda, Yussef Zenun Messias Gonçalves, Gabriel Maluf
          </td>
          <td>2026-02-10</td>
          <td>International Healthcare Review (online)</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="


 Ewing sarcoma (ES) is an aggressive pediatric malignancy driven by the EWS/FLI1 oncofusion, which reprograms enhancers, enforces transcriptional reprogramming, and imposes high levels of replication stress. Multimodal therapy yields ∼70% survival in localized ES, but <30% in relapsed/metastatic disease, underscoring the need for new targets. As part of the Pediatric Dependency Map Accelerator (PedDep), a collaboration between St. Jude Children’s Research Hospital, Dana-Farber Cancer Institute, and the Broad Institute, we analyzed CRISPR/Cas9 screens across 27 ES models to prioritize tumor-selective dependencies, identifying the DEAD-box helicase DDX19A as a top specific dependency in EWS/FLI-driven tumors. DDX19A is a conserved RNA helicase with roles in mRNA export and RNA remodeling, but its role in cancer remains unexplored. Using CRISPR knockout and degron-based systems, we validated DDX19A as essential for ES survival. Acute DDX19A loss caused rapid cytotoxicity, with near-complete cell death by 96 hours. Mechanistic studies showed within 24 hours there is a global increase in chromatin accessibility (ATACseq), gain of poised enhancers (H3K27ac-ChIPseq), along with a widespread transcriptional upregulation (RNAseq). These changes are consistent with a background already marked by elevated R-loops and replication stress driven by EWS/FLI1. DDX19A depletion further increased R-loop accumulation, triggering DNA damage, checkpoint activation, G2/M arrest by 72 hours and apoptosis. These data suggest that DDX19A acts as a transcriptional gatekeeper, constraining oncogenic overload and maintaining transcriptional homeostasis. Loss of this function destabilizes ES cells, producing transcriptional catastrophe and cell death. Importantly, DDX19A represents a druggable helicase, opening translational opportunities. In collaboration with the Cupido laboratory in Chemical Biology and Therapeutics at St. Jude, we are testing first-in-class small-molecule inhibitors of DDX19A in vitro and in ES xenograft models in vivo to assess efficacy. Ongoing studies focus on mapping EWS/FLI1 occupancy relative to DDX19A-dependent chromatin changes, dissecting cell death and replication stress pathways, and evaluating DDX19A inhibitors as single agents and in combination with DNA damage response modulators. In conclusion, our findings establish DDX19A as a selective and druggable vulnerability in ES. Targeting this transcriptional gatekeeper highlights a novel way to exploit transcriptional balance in cancer, leading to a new paradigm of therapeutic transcriptional overload in transcription factor-driven malignancies such as ES.



 Saikat Chakraborty, Victoria Akingbehin, Daniel Estevez. Prado, John Harper, Abigail Fish2, Tommaso Cupido, Lillian M. Guenther. Exploiting the dependency on DDX19A to induce transcriptional catastrophe in Ewing sarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr LT001.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e93447ff178770f7646a5ded74e9cd093bfeda60" target='_blank'>
              Abstract LT001: Exploiting the dependency on DDX19A to induce transcriptional catastrophe in Ewing sarcoma
              </a>
            </td>
          <td>
            S. Chakraborty, Victoria Akingbehin, Daniel Estevez. Prado, John Harper, Abigail Fish2, Tommaso Cupido, Lilly Guenther
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Whilst most mutations cells accumulate are neutral, some can alter cellular functions leading to positive selection and clonal expansions. In inflammatory bowel disease (IBD), epithelial injury-repair cycles raise mutation rates and favour clones with growth advantages. Previous studies [1-3] showed that clones with IL-17 signalling pathway mutations, including PIGR, NFKBIZ and ZC3H12A, recurrently expand in IBD despite being uncommon in colorectal cancer. Organoid work suggested that NFKBIZ mutations provide resistance to IL-17A-induced apoptosis and loss of PIGR was hypothesised to worsen inflammation by promoting dysbiosis. Thus, a positive feedback loop could be established, in which mutant clones drive tissue damage while expanding further. We aim to improve our understanding of how clonal expansions and driver genes influence disease biology in IBD. To achieve this, we performed paired DNA-RNA sequencing of individual human colonic crypts to characterise the transcriptional consequences of driver mutations.



 Multiomics laser-microdissection of single colonic crypts was possible with an improved low-input protocol. Fresh biopsies were collected during colonoscopies from inflamed and uninflamed colon regions. We obtained over 200 new biopsies from 25 patients which, combined with prior data, enabled selection analysis across 1500+ crypts from 50+ patients. We generated 700+ single crypts with paired DNA-RNA data, enabling differential expression analysis (DGE) between mutant and wild-type crypts.



 Our work confirms elevated mutation rates and characteristic mutational signatures associated with azathioprine treatment. Beyond identifying PIGR, NFKBIZ and ZC3H12A as strongly selected genes, consistent with prior work, we show that NFKBIZ is also highly prevalent in individuals of European ancestry. We also expand the catalogue of putative driver genes, particularly highlighting IL13RA1, ELF3 and MED12. DGE analysis suggests mutations in some genes e.g., NFKBIZ, may protect crypts from inflammatory effects, while others remain influenced by inflammation, potentially expanding primarily through increased crypt fission. Crypts with cancer-related mutations display elevated inflammatory signalling and epithelial-to-mesenchymal transition features compared to crypts with IBD-characteristic mutations e.g., ZC3H12A. These findings reveal how distinct driver mutations shape epithelial behaviour in IBD.



 Overall, paired DNA and RNA sequencing of single colonic crypts enabled us to broaden the catalogue of driver genes implicated in IBD and characterise their transcriptomic consequences in human tissue for the first time. Thus, improving our understanding of how mutations shape the epithelial landscape in IBD.
 References:
 1.Olafsson S, McIntyre RE, Coorens T, Butler T, Jung H, Robinson PS, Lee-Six H, Sanders MA, Arestang K, Dawson C, Tripathi M, Strongili K, Hooks Y, Stratton MR, Parkes M, Martincorena I, Raine T, Campbell PJ, Anderson CA. Somatic Evolution in Non-neoplastic IBD-Affected Colon. Cell. 2020 Aug 6;182(3):672-684.e11. doi: 10.1016/j.cell.2020.06.036. Epub 2020 Jul 21. PMID: 32697969; PMCID: PMC7427325.
 2.Kakiuchi N, Yoshida K, Uchino M, Kihara T, Akaki K, Inoue Y, Kawada K, Nagayama S, Yokoyama A, Yamamoto S, Matsuura M, Horimatsu T, Hirano T, Goto N, Takeuchi Y, Ochi Y, Shiozawa Y, Kogure Y, Watatani Y, Fujii Y, Kim SK, Kon A, Kataoka K, Yoshizato T, Nakagawa MM, Yoda A, Nanya Y, Makishima H, Shiraishi Y, Chiba K, Tanaka H, Sanada M, Sugihara E, Sato TA, Maruyama T, Miyoshi H, Taketo MM, Oishi J, Inagaki R, Ueda Y, Okamoto S, Okajima H, Sakai Y, Sakurai T, Haga H, Hirota S, Ikeuchi H, Nakase H, Marusawa H, Chiba T, Takeuchi O, Miyano S, Seno H, Ogawa S. Frequent mutations that converge on the NFKBIZ pathway in ulcerative colitis. Nature. 2020 Jan;577(7789):260-265. doi: 10.1038/s41586-019-1856-1. Epub 2019 Dec 18. PMID: 31853061.
 3.Nanki K, Fujii M, Shimokawa M, Matano M, Nishikori S, Date S, Takano A, Toshimitsu K, Ohta Y, Takahashi S, Sugimoto S, Ishimaru K, Kawasaki K, Nagai Y, Ishii R, Yoshida K, Sasaki N, Hibi T, Ishihara S, Kanai T, Sato T. Somatic inflammatory gene mutations in human ulcerative colitis epithelium. Nature. 2020 Jan;577(7789):254-259. doi: 10.1038/s41586-019-1844-5. Epub 2019 Dec 18. PMID: 31853059.
 4.EllisP,MooreL,SandersMA,ButlerTM,BrunnerSF,Lee-SixH,OsborneR,FarrB,CoorensTHH,LawsonARJ,CaganA,Stratton MR, Martincorena I, Campbell PJ. Reliable detection of somatic mutations in solid tissues by laser-capture microdissection and low-input DNA sequencing. Nat Protoc. 2021 Feb;16(2):841-871. doi: 10.1038/s41596-020-00437-6. Epub 2020 Dec 14. PMID: 33318691.
 Conflict of interest:
 Torra I Benach, Maria: No conflict of interest
 Marshall, Henry: No conflict of interest
 Jung, Hyunchul: No conflict of interest
 Abascal, Federico: No conflict of interest
 Lawson, Andrew: No conflict of interest
 Hu, May: No conflict of interest
 Goodwin, Scott: No conflict of interest
 Garri, Wendy: No conflict of interest
 Harris, Bradley: Personal Fees: I have received honoraria for a presentation at a BridgeBio board meeting.
 Garcia Salinas, Isaac: No conflict of interest
 Jarman, Georgee: No conflict of interest
 El Garwany, Omar: No conflict of interest
 Hooks, Yvette: No conflict of interest
 Sanghvi, Rashesh: No conflict of interest
 O’Neill, Laura: No conflict of interest
 Yogakanthi, Saiumaeswar: No conflict of interest
 Brezina, Biljana: No conflict of interest
 Chappel, Lia: No conflict of interest
 Cotobal Martin, Cristina: No conflict of interest
 Rahbari, Raheleh: No conflict of interest
 Raine, Timothy: Grant: Abbvie, Takeda Personal Fees: TR has received research/educational grants and/or speaker/consultation fees from Abbvie, Alfasigma, Arena, Aslan, AstraZeneca, Boehringer-Ingelheim, BMS, Celgene, Domain Therapeutics, Eli Lilly, Ferring, Galapagos, Gilead, GSK, Heptares, LabGenius, Janssen, MonteRosa, Mylan, MSD, Novartis, Numab, Pfizer, Roche, Sandoz, Scientia, Takeda, UCB and XAP therapeutics
 Anderson, Carl: Personal Fees: Carl Anderson (CAA) has received consultancy or speaker fees from Genomics plc, BridgeBio Ltd and Glaxo Smith Kline.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c73cadd26922fc11658f33d34a414918b714ee81" target='_blank'>
              DOP109 Investigating the role of driver mutations in Ulcerative Colitis through paired DNA and RNA sequencing of single colonic crypts
              </a>
            </td>
          <td>
            M. Torra, I. Benach, H. Marshall, H. Jung, F. Abascal, A. Lawson, M. Hu, S. Goodwin, W. Garri, B. Harris, I. G. Salinas, G. Jarman, O. El Garwany, Y. Hooks, R. Sanghvi, L. O'Neill, S. Yogakanthi, B. Brezina, L. Chappel, C. Martin, R. Rahbari, T. Raine, C. Anderson
          </td>
          <td>2026-01-01</td>
          <td>Journal of Crohn’s and Colitis</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Mutant p53(Mtp53) not only loses its canonical tumor-suppressive functions but also acquires oncogenic gain-of-function properties, positioning it as a central orchestrator in reshaping the tumor immune microenvironment. This review systematically delineates how Mtp53 actively establishes and sustains an immunosuppressive niche through multiple interconnected mechanisms, including chronic inflammation, immune cell dysfunction, reprogramming of cancer-associated fibroblasts, metabolic dysregulation, epigenetic hijacking, and potentially aberrant liquid–liquid phase separation, thereby promoting immune evasion and therapeutic resistance. We integrate current evidence to propose a conceptual “metabolism–epigenetics–immunity” axis: Mtp53-driven metabolic reprogramming—such as accumulation of lactate or α-ketoglutarate—can modulate chromatin modifications and immune gene expression. Notably, the full
 in vivo
 causal chain of this axis remains unestablished; existing support derives primarily from stepwise experimental data and strong correlations. The immunological impact of Mtp53 is highly context-dependent, shaped by co-mutations and tissue origin. In TP53/KRAS co-mutant non-small cell lung cancer (NSCLC), Mtp53 enhances tumor immunogenicity and improves response to immune checkpoint inhibitors (ICIs); conversely, in immunologically “cold” tumors—such as triple-negative breast cancer, pancreatic ductal adenocarcinoma, and colorectal cancer—it promotes T-cell exhaustion or myeloid suppression, reflecting marked cancer-type heterogeneity. Therapeutic approaches include Mtp53 reactivators (e.g., APR-246, PC14586), degraders, synthetic lethal strategies, and neoantigen vaccines. Although APR-246 showed efficacy in a phase II trial (NCT03072043), it failed to improve survival in phase III (NCT03745716) due to lack of TP53 mutation stratification. Its combination with pembrolizumab (NCT04383938) demonstrated acceptable safety (immune-related adverse events in ∼12%) but limited efficacy, underscoring the need for biomarker-guided, precision-based combinations. Thus, a multidimensional biomarker platform is urgently needed—one integrating TP53 mutation subtypes (e.g., R175H
 vs
 . nonsense mutations), dynamic ctDNA monitoring (VAF ≥ 0.01%), tumor immune microenvironment (TIME) features (e.g., TILs, MDSCs), and spatial multi-omics—to enable precise molecular stratification and personalized intervention in Mtp53-driven cancers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e3f2c235e49b5522a1b8c8e57fe450edd58eb9f" target='_blank'>
              The impact of p53 mutation on tumor immune evasion: mechanistic insights and clinical implications
              </a>
            </td>
          <td>
            Luo Liang, Weidong Wang
          </td>
          <td>2026-02-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Accurate detection of gene fusions is critical for the diagnosis and management of sarcomas. However, occasional false-negative results may occur due to factors such as low tumor cellularity, limited assay coverage of relevant genomic regions, or mis-annotation by bioinformatic pipelines. We recently encountered a case in which our clinical SARCP fusion detection assay failed to identify an HMGA2::NCOR2 (exon 3–exon 16) fusion.



 The SARCP assay is a targeted, amplicon-based RNA-seq next-generation sequencing (NGS) assay designed to detect gene fusions across 138 genes, including HMGA2. Although NCOR2 is not among the targeted genes, the assay is capable of detecting fusions when only one gene partner is represented.



 Suspicion of a false-negative result arose when the pathologist noted that the tumor’s histologic features—keratin-positive, giant cell-rich soft tissue—were consistent with a subtype previously associated with the HMGA2::NCOR2 fusion. This prompted a review of the bioinformatic output, which had erroneously reported an HMGA2::MAML2 fusion. The misannotation was traced to sequence homology involving a CAA/CAG repeat at the 5′ end of NCOR2 exon 16, likely resulting from DNA polymerase slippage and misalignment in low-complexity, partially complementary repetitive regions. This erroneously lead to the target genomic sequence getting mapped to MAML2 exon 2 as opposed to the NCOR2 exon 16.



 This case highlights how bioinformatic mis-annotation can lead to occasional false-negative fusion calls. It underscores the importance of manual review in cases with strong clinical or pathologic suspicion and calls attention to the need for fusion-calling algorithms that better handle repetitive and homopolymeric sequences.



 Ganesh P. Pujari, Mitch L. Klebig, Kurt S. Willkom, Montha Chhunn, Stefan W. Nelson, Neiladri Saha, Kevin C. Halling. Mis-Annotation of an HMGA2::NCOR2 Fusion in FFPE Sarcoma Samples: A Case Illustrating Bioinformatic Limitations in Fusion Detection [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A012.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b640522fda2138d045848d46734cd774a873c147" target='_blank'>
              Abstract A012: Mis-Annotation of an HMGA2::NCOR2 Fusion in FFPE Sarcoma Samples: A Case Illustrating Bioinformatic Limitations in Fusion Detection
              </a>
            </td>
          <td>
            Ganesh P. Pujari, Mitch L. Klebig, Kurt S. Willkom, Montha Chhunn, Stefan W. Nelson, Neiladri Saha, K.C. Halling
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f687787e5f5da10c05fbeb66dd1e619139451c70" target='_blank'>
              Multiomics reveals the mechanism of KNG1 in promoting tumor progression in adamantinomatous craniopharyngioma
              </a>
            </td>
          <td>
            Lin Zhou, Zhicheng Xu, C. Fang, Hui Ren, C. Yang, Zhigang Wang, Tao Hong, Jie Wu
          </td>
          <td>2026-01-01</td>
          <td>Journal of Neuro-Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Enhancer of Zeste Homolog 2 (EZH2) is the enzymatic subunit of the Polycomb Repressive Complex 2 (PRC2). It catalyzes H3K27 methylation for epigenetic silencing of tumor suppressors and critically drives prostate cancer (PCa) progression. However, inhibitors of EZH2 catalytic function (EZH2i), such as EPZ-6438, showed limited efficacy in PCa. Here, we designed and developed a series of VHL-based proteolysis-targeting chimera (PROTAC) degraders of EZH2 using EPZ-6438 as a ligand and identified PROTAC-6272 as a lead compound. PROTAC-6272 effectively degraded EZH2 and other PRC2 subunits across diverse PCa cell lines. However, PROTAC-6272 and other similar EZH2i-based PROTACs were consistently unable to decrease androgen receptor (AR), a gene that is directly activated by solo EZH2. Mechanistically, EZH2 PROTACs failed to degrade EZH2 coactivators, such as p300, due to their inability to engage EZH2 outside of the PRC2 complex. Nevertheless, PROTAC-6272 exhibited anti-proliferative activities superior to EPZ-6438 in some PCa models, wherein it induced p21 expression and cellular senescence by disrupting a methylation-independent PRC2 function. In summary, while EZH2i-based PROTACs failed to target the PRC2-independent functions of EZH2, they confer added benefits over EPZ-6438 by abolishing a polycomb-dependent but methylation-independent function of EZH2, offering therapeutic advantages in some PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bf1f78e5f0c3905edc37f25bad7b551a5037e88" target='_blank'>
              EZH2 PROTACs outperform catalytic inhibitors in prostate cancer by targeting a methylation-independent function of PRC2
              </a>
            </td>
          <td>
            Wanqing Xie, Qi Chu, Lourdes T. Brea, Guihua Zeng, Yuan Wang, Xiaodong Lu, Mohan Zheng, Corinne R Ley, Zhiquan Lei, Hongshun Shi, Joshua L. Zhu, Lihu Gong, M. C. Martin, Xianglin Shi, Galina A Gritsina, A. Grigorescu, Hana Chandonnet, Xin Liu, Jonathan C. Zhao, G. Schiltz, Jindan Yu
          </td>
          <td>2026-01-07</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Transposable Elements (TEs) represent a class of mobile genomic sequences, which may seriously disrupt gene regulation and can contribute to tumorigenesis. Yet, their role in NSCLC has remained unexplored to a great degree. Therefore, an integrated transcriptomic and Transposable Element (TE) analysis was performed to investigate TE-driven gene dysregulation in NSCLC. Hierarchical clustering of differentially expressed TE revealed significant over-representation of LTR1A1 and HERVL18-int in the cancer samples, with notably high expression of LINE and ERV members, especially HERVL-int, L1MC5, and L1M5. The intersection of TE expression with differentially expressed genes revealed several TE-associated genes involved in cell cycle regulation, genomic stability, and tumor progression. Fusion transcript analysis highlighted unique cancer-specific events, offering insights into TE-mediated transcriptomic alterations. Molecular docking of TE-associated proteins, HMMR, and PBK suggested potential interactions that may influence oncogenic pathways. Collectively, our findings uncover novel TE-driven mechanisms of gene dysregulation in NSCLC and highlight specific TEs and associated genes as potential diagnostic markers and therapeutic targets, offering a framework for future experimental studies to explore their mechanistic and clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a971dca83ca242097742334d02952f660161515" target='_blank'>
              Unraveling novel transposable elements (TEs)-driven gene dysregulation in non-small cell lung cancer (NSCLC) by integrated transcriptomic and TEs analysis.
              </a>
            </td>
          <td>
            Sahadevan Shrinidhi, R. Sagaya Jansi, A. Khusro
          </td>
          <td>2026-02-01</td>
          <td>Computational biology and chemistry</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The loss of chromosome 3p and the inactivation of the tumor suppressor gene von Hippel-Lindau (VHL) were identified in clear cell renal cell carcinomas (ccRCC) over three decades ago. Since then, mutations in genes for the three chromatin modulators, polybromo 1 (PBRM1), SET domain-containing 2 (SETD2), and BRCA1-associated protein-1 (BAP1), have been recognized as common in ccRCC. Although these genomic alterations are central to understanding ccRCC’s development, other deregulated cellular processes are also prominent in these tumors. Metabolic reprogramming is a key hallmark of this disease, characterized by various changes linked to the stabilization of hypoxia-inducible factors (HIF), including increased aerobic glycolysis, elevated lipid levels, and glutamine dependence for cell survival. Additionally, HIF-α stabilization plays a crucial role in regulating the immune system, thereby enhancing CD8+ T lymphocyte cytotoxicity. Immune checkpoint inhibitors (ICI) are now used as first-line treatments to target the often highly infiltrated tumor microenvironment of ccRCC. However, the effectiveness of ICI varies and is difficult to predict. Although emerging studies are beginning to provide insight, evidence suggests roles for PBRM1, SETD2, and BAP1 in metabolic regulation and in shaping the tumor immune microenvironment in ccRCC. Here, we review recent advances in this field and examine their impact on the management of ccRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74d8124d8b083fe78630c316b5f74781f77ea65c" target='_blank'>
              The fall of the genome protectors triad: PBRM1, SETD2, and BAP1’s impact on metabolism and immunity in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Mathieu Johnson, Sandra Turcotte
          </td>
          <td>2026-01-12</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Aging imposes significant influence in alteration of organ structure, function, and susceptibility to disease. Long non-coding RNAs (lncRNAs) are frequently dysregulated in the aging kidney, however, the key lncRNAs and associated mechanism involved in regulating kidney aging remains poorly investigated. Herein, we performed unbiased whole transcriptome sequencing on the kidney tissue samples from young and aging mice. The results of differentially expressed lncRNAs among two groups revealed that Gm44981, a 1943 bp lncRNA, was down regulated in the aging kidney. Fluorescence in situ hybridization (FISH) assay showed that Gm44981 was mainly located in the nucleus of glomerular mesangial cells (MCs). Functional experiments showed that overexpression of Gm44981 in MC cells significantly promoted cell proliferation capacity. Specifically, overexpression of Gm44981 was associated with increased H3K27me3 level and enhanced enrichment of EZH2 at the Cdkn1a promoter region, which correlated with the suppression of Cdkn1a expression and attenuated MCs senescence. Together, our findings highlighted the crucial roles of lncRNA Gm44981 in regulation of glomerular MCs senescence and provided novel targets for aging therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57da2eda293b7f74f3300a978c676b511c518879" target='_blank'>
              LncRNA Gm44981 modulates EZH2–H3K27me3–p21 axis to suppress mesangial cell senescence and kidney aging
              </a>
            </td>
          <td>
            Jie Li, Limin Wei, Fanfan Gao, Lu Zeng, Hongli Jiang, Lei Chen
          </td>
          <td>2026-02-11</td>
          <td>Renal Failure</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="In all cells, hexameric helicases drive the unwinding of parental chromosomal DNA at replication forks to provide the single-stranded DNA templates required by replicative DNA polymerases. DNA unwinding proceeds via a steric exclusion mechanism in which the helicase encircles and translocates along one DNA strand while sterically excluding the opposite strand from its central channel. The details of how hexameric helicases translocate on single-stranded DNA remain incompletely understood and likely vary among species, as structural and mechanistic features-such as motor domain architecture and translocation polarity-shape helicase function. Recent high-resolution cryo-EM structures of the eukaryotic CMG (Cdc45-MCM-GINS) helicase, including complexes stalled at leading-strand G-quadruplexes, reveal two predominant DNA-bound conformations: planar and spiral. These structures show that different subsets of MCM subunits alternately engage the leading-strand template, defining intermediates of a nonrotary, hand-over-hand translocation mechanism. This mode of translocation differs from the sequential rotary hand-over-hand mechanism proposed for bacterial hexameric helicases, instead resembling that of other ring-shaped ATPase motors and can be described as a variant of the helical inchworm model. The evolution of this mechanism may reflect CMG's specialized role as a replisome organizer, enabling it to coordinate accessory factors and optimize replication fork progression. Together, these findings highlight the mechanistic diversity and evolutionary adaptability of hexameric helicases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b39a99042bbff405717d35a7afa256356bd265f4" target='_blank'>
              DNA translocation by the CMG helicase: the helical inchworm model.
              </a>
            </td>
          <td>
            S. Batra, Benjamin Allwein, Y. Lucia Wang, R. Hite, Dirk Remus
          </td>
          <td>2026-02-01</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The paradigm of the Hallmarks of Cancer, updated by Douglas Hanahan in 2022, represents one of the most influential syntheses for understanding the functional capabilities that sustain neoplastic transformation. However, its traditional interpretation, often reductionist and fragmentary, does not capture the non-linear, emergent, and adaptive dynamics of tumor behavior. This review proposes a reinterpretation of the hallmarks through the lens of complexity theory, conceptualizing colorectal cancer (CRC) as a self-organizing, open system operating far from equilibrium. Using an integrative conceptual approach, we map the ten classical hallmarks and the new dimensions proposed in 2022 (phenotypic plasticity, non-mutational epigenetic reprogramming, polymorphic microbiomes, and senescence) onto the fundamental properties of complex systems: nonlinearity, emergence, feedback, openness, and historical dependence. We argue that CRC should not be understood as a simple sum of molecular alterations but as a dynamic network of interactions among cells, tissues, and microenvironments where global organization emerges from local rules. This systems-based perspective provides a conceptual foundation for translational models and integrative methodologies in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0199a76d1009c249f4505d66d97a40ec8df69d8" target='_blank'>
              Colorectal cancer as a complex adaptive system: integrating the hallmarks of cancer with complexity theory.
              </a>
            </td>
          <td>
            Luis R. Basbus
          </td>
          <td>2026-02-03</td>
          <td>Frontiers in oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e05ed224c6b5426e50bd95626e52d43390fadb" target='_blank'>
              The Mutational Epidemiology of Childhood Cancer.
              </a>
            </td>
          <td>
            Logan G Spector, Cassandra Clark, Zhanni Lu, Nathan Anderson, Erin L Marcotte, A. D. de Smith
          </td>
          <td>2025-12-30</td>
          <td>Clinical chemistry</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="NUTM1-rearrangement (NUTM1-r) defines a significant subset of B-cell acute lymphoblastic leukemia (B-ALL), particularly in infants lacking KMT2A-rearrangements (KMT2A-r), yet its underlying molecular characteristics remain poorly understood. Here, we establish that NUTM1-r leukemia is a discrete entity characterized by a unique transcriptional and epigenetic landscape, notably featuring global DNA hypomethylation, irrespective of the 5' fusion partner. Functional interrogation of NUTM1 fusions reveals a dual oncogenic role: they drive commitment towards the B-lymphoid lineage while concurrently conferring potent leukemic stem cell properties. Strikingly, expression of a representative fusion, BRD9-NUTM1, is sufficient to induce serially-transplantable prepro-B-like leukemia in vivo, faithfully recapitulating the key molecular and immunophenotypic features of human NUTM1-r B-ALL. Mechanistically, NUTM1 fusions establish an aberrant chromatin state, marked by global enhancement of H3K27 acetylation and the creation of distinctive open chromatin regions that co-opt both B-lineage and stemness-related transcriptional programs, including those involving NF-κB and posterior HoxA genes. In stark contrast to resistant KMT2A-r leukemias, NUTM1-r leukemic cells exhibit a profound sensitivity to chemotherapy. This vulnerability is mechanistically linked to the leukemia's dependence on active transcription. Our findings delineate the unique molecular profile of NUTM1-r leukemias, revealing specific vulnerabilities that rationalize their favorable clinical outcomes and suggest opportunities for modified therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec942964a1f8b552f79cec056ff910b4f7724fe5" target='_blank'>
              Comprehensive Molecular and Functional Analysis of NUTM1-Rearranged Leukemia.
              </a>
            </td>
          <td>
            Koutarou Nishimura, Tomoya Isobe, Tsukasa Shigehiro, Masaki Nomura, Weijia Zang, Muran Xiao, Wenjuan Liao, Yui Koike, Akira Nishimura, A. Sato-Otsubo, Hiromi Yamazaki, Hiromi Ito, Shinri Okada, Naomi Matsumoto, Wataru Saika, Yifan Zhang, Yumi Aoyama, Chihiro Hasegawa, Takaya Yamasaki, Yasuo Kubota, Kentaro Oki, N. Kiyokawa, G. Nagae, Kenichi Yoshida, Y. Nannya, Hiroo Ueno, Shiro Fukuda, K. Tatsuno, S. Tsutsumi, Yusuke Shiozawa, Y. Shiraishi, K. Chiba, Hiroko Tanaka, Mariko Eguchi, Yuki Arakawa, K. Koh, Takao Deguchi, Daisuke Tomizawa, T. Miyamura, Eiichi Ishii, Shuki Mizutani, Satoru Miyano, Hiroyuki Aburatani, Seishi Ogawa, A. Takaori-Kondo, Akihiko Yokoyama, Omar Abdel-Wahab, Patricia Ernst, Junko Takita, Tomokatsu Ikawa, Masatoshi Takagi, Daichi Inoue
          </td>
          <td>2025-12-29</td>
          <td>Blood</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c301551b95c4181f479a557a4624db3c9a192361" target='_blank'>
              LSD1 maintains oocyte quality by epigenetically repressing CAMK2β to control granulosa cell-oocyte communication
              </a>
            </td>
          <td>
            Jixian Zhang, Yuanyuan Yu, Zijian Zhu, Heng Zhang, Zhu Hu, Zhiyi Sheng, Meina He, Tuo Zhang
          </td>
          <td>2026-01-28</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Fusion-positive rhabdomyosarcoma (FP RMS) is a pediatric soft tissue cancer associated with the skeletal muscle lineage and defined by the 2;13 and 1;13 chromosomal translocations. These chromosomal translocations break within the PAX3 or PAX7 and FOXO1 genes and generate a PAX::FOXO1 fusion gene, either PAX3::FOXO1 or PAX7::FOXO1. Our previous studies showed that a human myoblast model of FP RMS depends on both MYCN and PAX3::FOXO1 for oncogenic transformation. The Myc family genes MYC and MYCN are frequently amplified or overexpressed in various cancers and play a critical role in the regulation of oncogenic properties. This study aims to investigate the expression of MYC and MYCN in FP RMS cell lines and their role in FP RMS growth and oncogenic transformation.



 The protein expression of MYC, MYCN, and PAX3::FOXO1 (or PAX7::FOXO1) was evaluated by Western blotting in FP RMS cell lines. The CRISPR-Cas9 system was applied to knock down these genes in the FP RMS cell lines, and then population growth, colony formation, and focus formation were used to determine growth and oncogenic transformation.



 The FP RMS cell lines were classified into three groups based on the protein expression profile of MYC and MYCN: MYC-dominant, MYCN-dominant, and MYC/MYCN-codominant. As a positive control, knockdown of the specific PAX::FOXO1 gene in each cell line markedly attenuated the growth and oncogenic transformation in all three subsets of FP RMS cell lines. In MYC-dominant CW9019 cells, MYCN knockdown had a small effect on growth and oncogenicity while MYC knockdown significantly diminished these parameters, and double knockdown of MYC and MYCN produced an even stronger inhibitory effect. In contrast, knockdown of MYCN in MYCN-dominant RH5 cells substantially reduced growth and transformation while MYC knockdown had no effect in these activities and double knockdown of MYC and MYCN produced effects comparable to MYCN knockdown. Finally, knockdown of either MYC or MYCN only partially inhibited growth and oncogenic behavior in MYC/MYCN-codominant FP RMS cell lines including RH30, RH41, and RH28, whereas double knockdown of MYC and MYCN generated a stronger inhibitory effect. These findings demonstrate that MYC and MYCN are functionally redundant in most FP RMS cell lines in terms of oncogenic behavior, and MYC or MYCN are functionally dominant in a smaller subset of FP RMS lines. Moreover, expression of MYC and MYCN contributes to the growth and oncogenic transformation in FP RMS cell lines.



 The Myc family proteins MYC and MYCN are differentially expressed in FP RMS cell lines and are considered as critical factors for regulating the growth and oncogenicity in these cell lines.



 PAWAN K. RAUT, Bishwanath Chatterjee, Hana Kim, Shabir Zargar, Frederic G. Barr. Expression of Myc family proteins contributes to the growth and oncogenic transformation of fusion-positive rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Fusion-Positive Cancer: From Discovery to Therapy; 2026 Jan 13-15; Philadelphia PA. Philadelphia (PA): AACR; Cancer Res 2026;86(1_Suppl):Abstract nr A009.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e376fdd909f7610a37fc7b5208ef08fd46ac7fca" target='_blank'>
              Abstract A009: Expression of Myc family proteins contributes to the growth and oncogenic transformation of fusion-positive rhabdomyosarcoma
              </a>
            </td>
          <td>
            P. K. Raut, Bishwanath Chatterjee, Hana Kim, Shabir Zargar, Frederic G. Barr
          </td>
          <td>2026-01-13</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="PVT1 lncRNA can regulate multi-gene expression through diverse mechanisms, one of which is through binding interactions with mRNAs. Our previous study highlighted its regulatory role in pan-cancer systems through predicted interactions with select mRNAs that are significantly differentially expressed across 15 cancer types. The structural basis of these interactions are yet to be propounded. Here, in order to identify and compare secondary structural features that may mediate PVT1 binding to select cancer-relevant mRNAs, and to determine evidence of evolutionary conservation, if any, we adopted the secondary structure folding information to identify key intermolecular interactions mediating the binding of PVT1 to specific regions, 5'UTR, coding region, and 3'UTR, of these select mRNAs, which may influence the translation process. Forming stable secondary structures, using both Watson-Crick and non-Watson-Crick base pairs, the flexibility of PVT1 lncRNA in interacting with these varied molecules at specific locations is deduced at the secondary structure level. To demonstrate the possible presence of conserved structural elements in PVT1 secondary structure generated based on 7SL non-coding RNA seed sequences, covariation analysis identified 10 significantly co-varying base pairs, suggesting structural conservation. The location of the start point of these lncRNA-mRNA interactions is majorly in the open loop regions. A-U nucleotides in the loops are observed to be higher in number than G-C nucleotides in PVT1 secondary structure. This may initiate multiple base-pairing interactions with other macromolecules more readily, owing to a lesser strength of the hydrogen bonding interactions between A-U base pairs. In the case of these mRNAs, comparatively speaking, there is a variability in the number of purines in the loop regions in their respective secondary structures. Since GC content correlates with the stability of mRNA secondary structures, our analysis shows that even though there is a variable sequence length, some of these mRNAs may demonstrate a higher stability of their specific secondary structures based on a higher GC content. Further, in order to potentially correlate with high protein expression, the distal segment of CDS and the 3'UTR regions of mRNAs require the presence of increased secondary structure. In our analysis, we found the same underlying pattern in a few of our select mRNA molecules. Exploration of the sequence and structural details of these lncRNA-mRNA interactions led us to an insight on a probable mechanism of a single PVT1 molecule being able to bind multiple mRNAs simultaneously or sequentially, in a spatio-temporal manner. Our research also seeks to further elucidate the contribution of bases and intermolecular interactions in the formation of these complexes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4b918075da273a0c0699b34025158629ff3799" target='_blank'>
              Structural basis of long non-coding RNA PVT1 interactions with select mRNAs universal in pan-cancer system: A computational study
              </a>
            </td>
          <td>
            Euphinia Tiberius Kharsyiemiong, Bhupal Haribhakta Raghunandan, Seema Mishra
          </td>
          <td>2026-01-01</td>
          <td>Computational biology and chemistry</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background
Hepatocellular carcinoma (HCC) is characterized by profound transcriptomic dysregulation, yet the mechanism(s) by which DNA methylation is coordinated with chromatin modifications to regulate alternative splicing during tumorigenesis remains poorly understood.


Methods
Using prospectively paired multi-omics data obtained from MASLD-HCC patients and coupled with a premalignant MASLD cohort we have uncovered a previously unrecognized gene-regulatory axis centered on TACC3 isoform switching.


Results
In the non-tumoral context, the TACC3-201 isoform directly engages the histone acetyltransferase KAT2A to coordinate the regulation of NOTCH4 signaling. In HCC, this regulatory axis is disrupted whereby FOXM1 overrides DNMT1-mediated methylation, upregulating TACC3, and decoupling TACC3 from the KAT2A-associated NOTCH4 co-expression module. This rewiring is licensing tumor-specific cell-cycle progression and epigenetic plasticity. Thus, FOXM1 reshapes the TACC3-KAT2A interaction, while DNMT1 drives context-dependent DNA methylation, activating the CDK1-inhibitory kinase PKMYT1.


Conclusion
We uncovered TACC3-KAT2A as an emerging regulatory axis caused by alternative splicing in HCC and propose FOXM1-driven TACC3 inhibition to synergistically disrupt mitotic fidelity and transcriptional regulation, potentially offering new therapeutic avenues for HCC with reduced toxicity to the normal liver.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf7a722bb3ee72e1c3c1c11d826cf0475a9c45b6" target='_blank'>
              FOXM1 influences DNA methylation to augment TACC3 alternative splicing directed by KAT2A in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Li Zhao, Jesper B. Andersen
          </td>
          <td>2026-01-27</td>
          <td>Clinical and molecular hepatology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41a4d8f7cfbb24e2436a8634a78d5520a6081b4c" target='_blank'>
              Coordinated DNA methyltransferase 3A and methyltransferase-like 7A activity reprograms the tumor microenvironment through discoidin domain receptor 1 signaling.
              </a>
            </td>
          <td>
            Zhengyang Bai, Dan Yang, Jiayi Li, Yaobang Liu, Bin Lian, Jinping Li
          </td>
          <td>2026-02-16</td>
          <td>Cancer biology & medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="


 Identification of genetic alterations in homologous recombination deficiency (HRD) and DNA damage response (DDR) genes in breast cancer (BC) is key for prognostication and potential selection for targeted therapies, such as PARPi. While HRD/DDR somatic oncogenic alterations are generally associated with ductal BCs, their roles in invasive lobular carcinoma (ILC) remain unclear. Here, we sought to investigate the spectrum of HRD/DDR somatic genetic alterations in ILC and their histologic features of these BCs.



 We interrogated 1,076 ILC previously subjected to paired tumor-normal targeted sequencing (MSK-IMPACT) for oncogenic somatic genetic alterations in 22 HRD/DDR genes. Loss of heterozygosity (LOH) was assessed using FACETS, and mutational signatures using SigMA. HRD status was evaluated using an algorithm optimized for MSK-IMPACT. Histologic assessment was performed as per the WHO. ER/HER2 status was determined per ASCO/CAP guidelines, and follow-up data retrieved from medical records.



 We identified 53 ILC, including 14 primary ILCs and 39 recurrent/metastatic ILCs harboring somatic oncogenic alterations in HRD/DDR genes, with BRCA2 (38%) and ATM (23%) being the most frequently affected. Other recurrently altered (≥2 cases) HRD/DDR genes were CHEK2, PALB2, BRCA1, BARD1, RAD50, RAD51C, NBN, RECQL and ABRAXAS1. LOH analysis revealed that 53% of HRD/DDR genetic alterations in ILCs were bi-allelic, including 75% of BRCA2 and 70% of ATM cases. Most (71%) primary ILCs were of pleomorphic type or exhibited pleomorphic features (Pleo-ILC), histologic grade 2/3 (93%) and ER+/HER2- (93%). Of the metastatic ILCs, 59% were pleomorphic, and most were ER+/HER2- (74%), followed by HER2+ (18%). Median follow-up for the HRD/DDR-mutant primary ILC cohort was 72 months (range 21-150); 6 patients (43%) had no evidence of disease, 5 were deceased (36%), and 3 were alive with disease (21%). Median follow-up time for the HRD/DDR-mutant metastatic ILC cohort was 84 months (range 19-288); 62% of patients were deceased. ILCs with biallelic inactivation of BRCA2 were classified as HRD by our algorithm in 60% of cases, and half of them showed a dominant HRD mutational signature, whereas the majority of those with ATM biallelic alterations (83%) were classified as non-HRD and exhibited either dominant APOBEC or clock signature (60%). The most frequent non-HRD/DDR affected genes in the cohort were CDH1 (76%), PIK3CA (48%), and FOXA1 (21%).



 Our findings show that, although at a lower frequency than that reported for ductal BC, DNA repair deficiency including HRD/DDR defects can occur in a subset of ILC. Somatic alterations affecting HRD/DDR genes in ILC are often biallelic, with BRCA2 and ATM being the most altered genes. While these tumors are phenotypically diverse, they often display pleomorphic histology and appear to be associated with more aggressive clinical behavior. These findings highlight the potential clinical relevance of HRD/DDR profiling in ILC, which may help inform treatment strategies.



 C. J. Schwartz, D. Muldoon, H. Dopeso, A. Gazzo, P. Terraf, M. Repetto, B. Weigelt, D. Mandelker, L. Norton, H. Y. Wen, H. Zhang, D. Ross, B. Edi, A. Mamtani, S. Shen, C. Bandlamudi, F. Pareja. Homologous Recombination Deficiency and DNA-Damage Response in Breast Invasive Lobular Carcinoma [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2025; 2025 Dec 9-12; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2026;32(4 Suppl):Abstract nr PS4-03-18.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80fc8cb5589ff91ec4481f5aa06189a04b64cc5" target='_blank'>
              Abstract PS4-03-18: Homologous Recombination Deficiency and DNA-Damage Response in Breast Invasive Lobular Carcinoma
              </a>
            </td>
          <td>
            C. Schwartz, D. Muldoon, H. Dopeso, A. Gazzo, P. Terraf, M. Repetto, B. Weigelt, D. Mandelker, L. Norton, , H. Zhang, D. Ross, B. Edi, A. Mamtani, S. Shen, C. Bandlamudi, F. Pareja
          </td>
          <td>2026-02-17</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>92</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80c94f22bd474e7a698c571178ecbf4e10acbded" target='_blank'>
              Z-DNA formation regulates the totipotent-like state and primes Zscan4-dependent chromatin compartmentalization.
              </a>
            </td>
          <td>
            Shireen Shajahan, Yann Loe-Mie, Laure Asselin, Marion Salmon-Legagneur, Tatiana Traboulsi, Agnès Thierry, Anne Dejean, J. Cossec
          </td>
          <td>2026-02-17</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Telomerase is known as a very specific marker of embryonic cells. It is responsible for telomere elongation (bypassing the end-replication problem) and thus supports normal cell division during tissue and organ development. But it is generally absent or very low in most normal adult somatic cells. However, its overexpression in adulthood (due to secondary expression and activity restoration) is commonly known to be associated with cancer. Apart from its canonical function (associated with telomere length restoration), it also carries out various other roles. Its non-canonical activity covers mitochondrial and epigenetic processes. Consequently, it contributes to the cell response to stress and chemotherapeutic drug treatment. A more detailed understanding of these phenomena offers the opportunity to identify new pathways and targets that may serve as critical factors in breast cancer diagnostics and therapy. In this article, we summarize the latest reports on the discovery of telomerase’s nature, including its canonical and non-canonical roles. The manuscript highlights how these mechanisms contribute to tumorigenesis, therapy resistance, and the survival of cancer cells. Understanding these multifaceted mechanisms behind hTERT’s role in (breast) cancer progression and therapy resistance is crucial for developing more effective therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5301bfbbaf0a4c2684d1a5d04cf7184a1b01801a" target='_blank'>
              Telomeric Ends and Telomerase—Canonical and Non-Canonical Roles in Breast Cancer Tumorigenesis and Therapy Resistance
              </a>
            </td>
          <td>
            Magdalena Kostrzewa, Julia Niedzielska, Wiktoria Mieczkowska, Maja Hoffmann, Julia Rypińska, Adrian Kowalczyk, Magdalena Stachowiak, B. Rubiś
          </td>
          <td>2026-01-30</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Aneuploidy is present in about 90% human solid tumors. Certain tumors remain addicted to aneuploidy. Paradoxically, artificially induced aneuploidy in normal cells elicits cellular stresses and decreases cell fitness. How aneuploidy initially emerges during tumorigenesis is thus unclear. Using human embryonic stem cells (ESCs) as a model, we show that aneuploid ESCs form immature teratomas with an enrichment of mesenchymal tissues. Specifically, chromosome 8 (chr8) gain, a prevalent form of aneuploidy in human cancers, promotes the expansion of mesenchymal stem cells (MSCs) in teratomas and MSC proliferation in vitro. We further show that human embryonal rhabdomyosarcomas, tumors of mesenchymal origin, lack dominant driver mutations but frequently harbor chr8 gain. Our study suggests a plausible mechanism for aneuploidy emergence during tumorigenesis, links specific aneuploidy to the mesenchymal lineage, and paves the way for identifying vulnerabilities of aneuploid MSCs, which may have important roles in tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edd9fc8f9ca1f0297a88c0d451a67c13ef908528" target='_blank'>
              Aneuploidy of chromosome 8 promotes the mesenchymal lineage during cell fate transitions.
              </a>
            </td>
          <td>
            Cai Liang, Guanchen Li, Qiuqin Zhang, Xinlei Wang, Yu Liu, Wenjuan Yin, Ting Tao, Jinhu Wang, Haishan Gao, Shutao Qi, Hongtao Yu
          </td>
          <td>2026-02-20</td>
          <td>Science advances</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="ABSTRACT Small DNA tumor viruses such as polyomaviruses have evolved persistent, in some cases lifelong infections despite their compact genomes and host immune pressure. This review synthesizes historical and recent insights into the mechanisms underlying polyomavirus persistence and shedding, including dynamic host cell cycle regulation, viral non-coding control region modulation, and viral microRNA-mediated repression. We highlight modes of shedding consistent with concurrent latent/lytic and smoldering infections, discuss emerging evidence of reversible latency, and identify unresolved questions in viral-host interplay. Understanding these strategies is critical for managing viral reactivation and disease in immunocompromised patients and exemplifies the remarkable evolutionary success of polyomaviruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4cbf37e5d9b69dc6b4a023237fb48855d94d952" target='_blank'>
              mGem: Deciphering how polyomaviruses coexist with their hosts for a lifetime
              </a>
            </td>
          <td>
            James M. Pipas, Christopher S. Sullivan
          </td>
          <td>2026-01-05</td>
          <td>mBio</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Deinococcus radiodurans is an extremely radioresistant bacterium that has captured the interest of researchers in pursuit of innovations in radioresistant technology and potential applications for bioremediation. The following paper provides an overview of the bacterium’s unique biological mechanisms that enable it to maintain such resilience to ionizing radiation. Genetic redundancies and genomic copies play a considerable role in withstanding radiation-induced mutations and DNA damage. Similarly, antioxidant systems, including both enzymatic and non-enzymatic ones, counteract the effects of radiation by removing radiation-induced reactive oxygen species that harm the bacterium’s proteome and genome. Lastly, highly efficient DNA repair mechanisms ensure a speedy recovery from DNA damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8de85ccb80d750a749a5ea83961684c96bf549bd" target='_blank'>
              The Biological Survival Mechanisms Behind the World’s Most Radioresistant Bacterium
              </a>
            </td>
          <td>
            Brandon Cho
          </td>
          <td>2026-01-15</td>
          <td>Scholarly Review Journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Motivation Single cell RNAseq (scRNAseq) is an ideal tool to characterize the heterogeneity within the tumor microenvironment, however, accurate identification of tumor cells can be a challenge. Reference-based methods can be inaccurate, if reference datasets are even available. Current purpose-built methods can be inaccurate, particularly with highly heterogeneous tumor types. Improved methods are needed. We explored the use of genetic variants to distinguish tumor from normal cells within scRNAseq data. Results We characterized the limitations inherent to calling variants from scRNAseq data, quantifying how data sparsity precludes genetic distance calculation between single cells. As a novel workaround, we pooled data from transcriptionally similar cell clusters to call high quality variants and then calculated pairwise differences between cell populations and performed hierarchical clustering. We quantified confidence in genetic divergence between tumor and normal cell populations using bootstrapping. We performed extensive validation to assess accurate identification of tumor cells using ground-truth datasets. Application of our method to human scRNAseq samples highlighted the utility of our approach and revealed how mutational burden influences successful tumor cell identification. Improved cell type assignment in scRNAseq data will facilitate analysis of tumor samples and, in turn, accelerate our understanding of the mechanisms underlying tumor progression and reveal potential biological vulnerabilities that can be exploited to develop improved treatment options. Availability and implementation Our method is publicly available as an R package: SCANBIT (Single Cell Altered Nucleotide Based Inference of Tumor) https://github.com/kidcancerlab/scanBit.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9bcdfb11e48000089460046e024824475367747" target='_blank'>
              SCANBIT facilitates identification of tumor cell populations in scRNAseq data using pseudobulked SNV calls
              </a>
            </td>
          <td>
            MV Cannon, MJ Gust, AC Gross, M. Cam, JB Reinecke, L. Jimenez Garcia, CH Strawser, L. Ryan, M. Sammons, CZ Zhang, R. Roberts
          </td>
          <td>2026-01-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ea2744fede5f7a4975fd041ac239356c320c431" target='_blank'>
              Large-scale mapping of environmental-genetic interactions illustrates the dynamic nature of cell-cycle and DNA repair regulation.
              </a>
            </td>
          <td>
            Benjamin W. Herken, Garrett T Wong, Anna Mäkiniemi, E. Lundberg, Thomas M. Norman, Luke A. Gilbert
          </td>
          <td>2026-02-19</td>
          <td>Molecular cell</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="AREAS COVERED
we discuss WRN as a caretaker of the genome that can function as a tumor suppressor or tumor enabler, and review progress on targeting functional WRN in microsatellite-unstable (MSI) cancers. We discuss mechanisms and distinct properties of synthetic lethality between the loss of WRN and the MSI-high tumor status, and derive lessons that can be used to identify or improve upon other cases of WRN synthetic lethality. We highlight the properties of a new generation of WRN helicase inhibitors that may confer increased lethality to MSI cancer cells while sparing normal tissues. We also discuss targeting mutant WRN in Werner syndrome and in cancer.


EXPERT OPINION
Significant gaps remain in understanding how WRN loss promotes pathogenesis. Filling these conceptual and informational gaps will allow better prediction of tissue-specific adverse effects of WRN helicase inhibition or WRN loss. We highlight important, unanswered questions in WRN biology and newly emerging research and translational opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7eb2ac748841065c2a53ce43595a163ef1c66b9" target='_blank'>
              The Werner syndrome RECQ helicase as a therapeutic target: new insights.
              </a>
            </td>
          <td>
            Julia M Sidorova, Raymond J. Monnat
          </td>
          <td>2026-01-20</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Long-read whole-transcriptome sequencing (WTS) has the potential to precisely characterize fusion oncogenes that drive leukemia and other cancers. While there are a variety of general-purpose fusion detection algorithms that use modern long-read sequencing data, they show poor sensitivity for precision diagnostics in B-ALL and do not robustly assess technical and analytical parameters (ex. sequencing depth) to reliably detect fusion transcripts. FUSILLI (FUSions In Leukemia Long-read sequencing Investigator) is a novel long-read fusion detection algorithm, with a focus on targeted genomic subtyping in B-ALL. FUSILLI was evaluated against extant methods using nanopore WTS from 51 pediatric B-ALL samples sequenced at high depth and 68 at low depth (mean of 11.2M and 1.4M reads, respectively). In the high-depth cohort, FUSILLI demonstrated increased sensitivity (0.81) compared to FusionSeeker, JAFFAL, and LongGF (0.63, 0.76, and 0.70, respectively), while maintaining high specificity (0.92). At lower sequencing depth, FUSILLI showed correspondingly lower sensitivity (0.27) but still outperformed the other fusion callers (sensitivities ranging from 0.09 to 0.16). Computational down sampling suggests that ten million reads is sufficient to sensitively detect B-ALL-relevant fusions using this approach. FUSILLI detects B-ALL fusions with high sensitivity at modest sequencing depth, supporting implementation of nanopore WTS as a low-cost and globally accessible sequencing-based molecular diagnostic platform for pediatric B-ALL and other fusion-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/884c4daa4f67a077c5abf5eb62ebb345ad0e753d" target='_blank'>
              Long-read whole-transcriptome sequencing and selective gene panel profiling enable sensitive detection of fusion oncogenes in pediatric B-cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            John Lin, Kofi B. Opoku, M. Litzow, E. Paietta, Ching-Hon Pui, S. Jeha, K. Roberts, C. Mullighan, T. Alexander, Jeremy R. Wang
          </td>
          <td>2026-02-01</td>
          <td>The Journal of molecular diagnostics : JMD</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="The conventional role of tRNA as a pivotal molecule in protein synthesis is well-established. However, recent studies have unveiled a previously underappreciated function of tRNA chemical modifications in tumor biology, particularly in tumor initiation, progression, and metastasis. Gynecological malignancies represent a significant threat to women's health. Despite advances in medical technology, gynecological malignancies remain difficult to manage, partly due to limited treatment options. Delayed diagnosis and high tumor heterogeneity further contribute to poor prognosis. This review focuses on tRNA modifications in gynecological cancers, including ovarian, cervical, and endometrial cancers. It systematically elucidates the expression profiles and regulatory mechanisms of various tRNA modifications. Additionally, it explores their correlation with malignant characteristics-such as enhanced tumor cell proliferation, drug resistance, invasion, and metastasis. Furthermore, it explores the translational potential of these modifications as biomarkers, therapeutic targets, and prognostic indicators, and the potential links in the TME, metabolic reprogramming, and Immunotherapeutic Response, offering a novel perspective for the precise diagnosis and management of gynecological tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f920589c9a5fbdd60166d2ddf11d9fb94072912" target='_blank'>
              Application progress of tRNA modification in gynecological tumors: A review.
              </a>
            </td>
          <td>
            Pengfei Wang, Xiaofei Chen, L. Xue, Yapeng Huang, Xiaoyu Wang, Weidong Ji, Hu Li
          </td>
          <td>2026-02-01</td>
          <td>International journal of biological macromolecules</td>
          <td>0</td>
          <td>4</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024', '2025', '2026'],
    y: [0, 2, 13],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>