<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-09-15 09:29:27 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>37</td>
          <td>104</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>78</td>
          <td>138</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>114</td>
          <td>145</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>356</td>
          <td>138</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>313</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) amplification enhances intercellular oncogene dosage variability and accelerates tumor evolution by violating foundational principles of genetic inheritance through its asymmetric mitotic segregation. Spotlighting high-risk neuroblastoma, we demonstrate how ecDNA amplification undermines the clinical efficacy of current therapies in cancers with extrachromosomal MYCN amplification. Integrating theoretical models of oncogene copy number-dependent fitness with single-cell ecDNA quantification and phenotype analyses, we reveal that ecDNA copy-number heterogeneity drives phenotypic diversity and determines treatment sensitivity through mechanisms unattainable by chromosomal oncogene amplification. We demonstrate that ecDNA copy number directly influences cell fate decisions in cancer cell lines, patient-derived xenografts, and primary neuroblastomas, illustrating how extrachromosomal oncogene dosage-driven phenotypic diversity offers a strong evolutionary advantage under therapeutic pressure. Furthermore, we identify senescent cells with reduced ecDNA copy numbers as a source of treatment resistance in neuroblastomas and outline a strategy for their targeted elimination to improve the treatment of MYCN-amplified cancers.


SIGNIFICANCE
ecDNA-driven tumor genome evolution provides a major challenge to curative cancer therapies. We demonstrate that ecDNA copy-number dynamics drives treatment resistance by promoting oncogene dosage-dependent phenotypic heterogeneity in MYCN-amplified cancers. Exploiting phenotype-specific vulnerabilities of ecDNA cells, therefore, presents a powerful strategy to overcome treatment resistance. See related article by Korsah, p. XX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a317a5c4269a3d4d0174cf78851658866c2ba8" target='_blank'>
              Extrachromosomal DNA-Driven Oncogene Dosage Heterogeneity Promotes Rapid Adaptation to Therapy in MYCN-Amplified Cancers.
              </a>
            </td>
          <td>
            Giulia Montuori, Fengyu Tu, Di Qin, R. Schmargon, Elias Rodriguez-Fos, K. Helmsauer, Hui Hui, Susmita Mandal, Karin Purshouse, Lara Fankhänel, Bartolomeo Bosco, B. Spanjaard, H. Seyboldt, Laura Grunewald, M. Schmitt, D. Gürgen, Viktoria Buck, M. Rosenfeldt, Frank P B Dubois, S. Schallenberg, Annika Lehmann, J. Theißen, S. Taschner-Mandl, Arend Koch, P. Hundsdoerfer, A. Künkele, Angelika Eggert, Matthias Fischer, G. Gargiulo, Teresa G Krieger, Lukas Chavez, Fabian Coscia, Benjamin Werner, Weini Huang, A. Henssen, J. Dörr
          </td>
          <td>2025-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA), a major focal oncogene amplification mode found across cancer, has recently regained attention as an emerging cancer hallmark, with a pervasive presence across cancers. With technical advancements such as high-coverage sequencing and live-cell genome imaging, we can now investigate the behaviors and functions of ecDNA. However, we still lack an understanding of how to eliminate ecDNA. We observed depletion of cells containing ecDNA during lentiviral but not transposon-based transduction, whereas we sought to investigate the mechanism of ecDNA behavior. This discovery may provide critical information on utilizing a lentiviral system in emerging ecDNA research. Additionally, this observation suggests specific sensitivities for cells with ecDNA. Significance: ecDNA is an essential factor in cancer progression. We found that a group of cancer cells with ecDNA is selectively depleted after lentiviral infection. This finding provides promise for ecDNA-specific targeting, suggests the need for caution in using lentivirus, and offers alternative ways to study ecDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/acb47b6253f4e8e1db726d8195373534a90d452c" target='_blank'>
              Selective Depletion of Cancer Cells with Extrachromosomal DNA via Lentiviral Infection
              </a>
            </td>
          <td>
            Eunhee Yi, Amit D Gujar, Dacheng Zhao, Kentaro Suina, Xue Jin, Katharina Pardon, Qinghao Yu, Larisa Kagermazova, Emmanuel E Korsah, Noah A Dusseau, J. Boeke, A. Henssen, R.G.W. Verhaak
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="Circular extrachromosomal DNA (ecDNA) is a common form of oncogene amplification in aggressive cancers. The frequency and diversity of ecDNA has been catalogued in adult and some childhood cancers; however, its role in most pediatric cancers is not well-understood. To address this gap, we accessed large pediatric cancer genomics data repositories and identified ecDNA algorithmically using cloud computing. This retrospective cohort comprises 2,968 patients covering all major childhood solid tumor types. Aggressive tumor types had particularly high incidences of ecDNA. Pediatric patients whose tumors harbored extrachromosomal DNA had poorer five-year overall survival than children whose tumors contain only chromosomal amplifications. We catalogue known and potentially novel oncogenes recurrently amplified on ecDNA and show that ecDNA often evolves during disease progression. These results highlight patient populations that could potentially benefit from future ecDNA-directed therapies. To facilitate discovery, an interactive catalogue of ecDNA in childhood cancer is available at https://ccdi-ecdna.org/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8b9576b9fdef51f927edfe8dbc69e8e7f1f4c7f" target='_blank'>
              Extrachromosomal DNA associates with poor survival across a broad spectrum of childhood solid tumors
              </a>
            </td>
          <td>
            Owen S. Chapman, S. Sridhar, Eugene Yui-Ching Chow, Rishaan Kenkre, Jonathan Kirkland, Aditi Dutta, Shanqing Wang, Wenshu Zhang, Miguel Brown, J. Luebeck, Yan Yuen Lo, Elias Rodriguez-Fos, A. Henssen, K. Okonechnikov, David R. Ghasemi, K. Pajtler, Daisuke Kawauchi, V. Bafna, Megan Paul, K. Yip, Jill P. Mesirov, Lukas Chavez
          </td>
          <td>2025-07-23</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) is a critical driver of cancer progression, contributing to tumour growth, evolution, and therapeutic resistance through oncogene amplification. Despite its significance, the replication of ecDNA remains poorly understood. In this study, we investigated the replication dynamics of ecDNA using high-resolution replication timing analysis (Repli-seq) and DNAscent, a method for measuring origin firing and replication fork movement, that we applied to both bulk DNA and to ecDNA isolated with FINE (Fluorescence-activated cell sorting-based Isolation of Native ecDNA), a new method for isolating, chromatinized ecDNA without DNA or protein digestion. We demonstrate that ecDNA in the COLO 320DM colorectal cancer cell line exhibits largely asynchronous replication throughout the S phase, contrasting with the conserved replication timing of the corresponding chromosomal DNA in RPE-1 cells and the chromosomally reintegrated ecDNA in COLO 320HSR. Replication origins on ecDNA are redistributed, and replication forks exhibit reduced velocity and increased stalling. Under replication stress induced by hydroxyurea treatment, ecDNA replication is further compromised, leading to altered origin activation, reduced fork velocity and eventual ecDNA depletion from cells. Our findings reveal fundamental differences in the replication dynamics of ecDNA, providing insights that could inform the development of therapies targeting ecDNA-associated oncogene amplification in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec71fc330fef12833948b0057870ba85688ca5b1" target='_blank'>
              ecDNA replication is disorganized and vulnerable to replication stress
              </a>
            </td>
          <td>
            Jedrzej J Jaworski, Pauline L Pfuderer, Pawel Czyz, Gianluca Petris, M. Boemo, Julian E. Sale
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT Extrachromosomal circular DNA (eccDNA) is an emerging class of genetic material that exists outside of the chromosomal genome. These circular DNA molecules are gaining increasing attention as important biomarkers in various cancers because of their roles in gene amplification, genetic heterogeneity, and drug resistance. In this review, we explore in depth the impacts of eccDNAs on cancer biology, their potential to predict treatment sensitivity and resistance, and their involvement in the development of new anticancer therapies. eccDNAs can be used as biomarkers for various tumor types to help diagnose and predict prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1f8c896f37f0d86135daabcae627eb40e4e88c" target='_blank'>
              Extrachromosomal Circular DNA as a Cancer Biomarker: From Diagnosis to Treatment
              </a>
            </td>
          <td>
            Hexin Li, Jiahui Cai, Gang Zhao, Lihui Zou
          </td>
          <td>2025-09-08</td>
          <td>Cancer Innovation</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/127ba85daf25e695241e6cc43a8f791c28684c9d" target='_blank'>
              DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution
              </a>
            </td>
          <td>
            F. Gimeno-Valiente, C. Castignani, E. Larose Cadieux, Nana E. Mensah, Xiaohong Liu, Kezhong Chen, O. Chervova, Takahiro Karasaki, C.E. Weeden, Corentin Richard, Siqi Lai, Carlos Martínez-Ruiz, E. Lim, A. Frankell, T. Watkins, Georgia Stavrou, Ieva Usaite, , Daniele Marinelli, S. Saghafinia, G. Wilson, P. Dhami, H. Vaikkinen, Jonathan Steif, S. Veeriah, R. Hynds, Martin Hirst, C. Hiley, A. Feber, Özgen Deniz, M. Jamal-Hanjani, N. Mcgranahan, Stephan Beck, J. Demeulemeester, M. Tanić, C. Swanton, P. Van Loo, N. Kanu
          </td>
          <td>2025-09-01</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Abstract Large chromosomal copy number gains are ubiquitous throughout cancer types. However, which genes drive their selective advantage is not well established, and therefore they are hardly utilized in clinical practice. Our analysis of copy number patterns in pan-cancer datasets suggests that the selective advantage of copy number gains is largely driven by known oncogenes. Analysis of CRISPR screening data identifies a list of 101 genes that are likely to mediate the effect of these gains, which is highly enriched in annotated oncogenes but also contains genes that have not been implicated in cancer so far. Moreover, we show that specific gains are associated with drug sensitivity or resistance, with a strong enrichment of gains of oncogenes with increased sensitivity to inhibitors targeting these specific genes. Finally, we provide examples where gains can function as relevant clinical biomarkers for diagnosis and treatment. Thus, large copy number gains exert their selective advantage through known and novel oncogenes, and their systematic analysis could advance precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecd5fd07460798ed6e6a37e20560052b55679a6c" target='_blank'>
              Low-amplitude copy number gains shape cancer through known and novel oncogenes with associated therapeutic vulnerabilities
              </a>
            </td>
          <td>
            T. F. Eleveld, B. Ylstra, Leendert H J Looijenga
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76cfc80059f8767909d0c8bd930a6c33c0803bbb" target='_blank'>
              Preservation and Clonal Behavior of Extrachromosomal DNA in Patient-Derived Xenograft Models of Childhood Cancers.
              </a>
            </td>
          <td>
            Rishaan Kenkre, Jon D. Larson, Owen S. Chapman, J. Luebeck, Yan Yuen Lo, Megan Paul, Wenshu Zhang, V. Bafna, Robert J Wechsler-Reya, Lukas Chavez
          </td>
          <td>2025-08-19</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs), which replicate and segregate in a non-Mendelian manner, serve as vectors for accelerated tumor evolution. By integrating chromatin accessibility, whole-genome sequencing, and Hi-C-based genome topology data from a cohort of metastatic Castration-Resistant Prostate Cancer (mCRPC) cases, we show that epigenetically activated repeat DNA, amplified in ecDNAs, drive oncogene overexpression. Specifically, we identify a subgroup of mCRPCs (20%) characterized by clusters of accessible LINE1 repeat DNA elements flanking the androgen receptor (AR) gene. These LINE1 elements are co-amplified with AR and provide binding sites for prostate-lineage transcription factors, including AR, FOXA1 and HOXB13. Accessible LINE1 elements establish novel 3D chromatin interactions with the AR gene, forging a new regulatory plexus driving AR overexpression and confers resistance to androgen signaling inhibitors. Our findings indicate how tumor evolution is driven by the convergence of genetic and epigenetic alterations on repeat DNA, activating and amplifying them to allow oncogene overexpression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e6c5dce2ec927f5cff069554fa04e5223fd82a5" target='_blank'>
              Genetic and Epigenetic Reprogramming of Transposable Elements Drives ecDNA-Mediated Metastatic Prostate Cancer
              </a>
            </td>
          <td>
            Lisanne Mout, Thaidy Moreno-Rodriguez, Giacomo Grillo, Ankita Nand, Tina Keshavarzian, S. Bahl, Komaldeep Kang, M. Bootsma, Emma Minnee, Stanley Zhou, Kathleen H. Burns, E. Corey, Peter Nelson, S. Dehm, Shuang G Zhao, W. Zwart, Felix Y. Feng, David A Quigley, M. Lupien
          </td>
          <td>2025-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Oncogenes amplified on extrachromosomal DNA (ecDNA) contribute to treatment resistance and poor survival across cancers. Currently, the spatiotemporal evolution of ecDNA remains poorly understood. In this study, we integrate computational modeling with samples from 94 treatment-naive human glioblastomas (GBM) to investigate the spatiotemporal evolution of ecDNA. We observe oncogene-specific patterns of ecDNA spatial heterogeneity, emerging from random ecDNA segregation and differing fitness advantages. Unlike PDGFRA-ecDNAs, EGFR-ecDNAs often accumulate prior to clonal expansions, conferring strong fitness advantages and reaching high abundances. In corroboration, we observe pretumor ecDNA accumulation in vivo in genetically engineered mouse neural stem cells. Variant and wild-type EGFR-ecDNAs often coexist in GBM. Those variant EGFR-ecDNAs, most commonly EGFRvIII-ecDNA, always derive from preexisting wild-type EGFR-ecDNAs, occur early, and reach high abundance. Our results suggest that the ecDNA oncogenic makeup determines unique evolutionary trajectories. New concepts such as ecDNA clonality and heteroplasmy require a refined evolutionary interpretation of genomic data in a large subset of GBMs.


SIGNIFICANCE
We study spatial patterns of ecDNA-amplified oncogenes and their evolutionary properties in human GBM, revealing an ecDNA landscape and ecDNA oncogene-specific evolutionary histories. ecDNA accumulation can precede clonal expansion, facilitating the emergence of EGFR oncogenic variants, reframing our interpretation of genomic data in a large subset of GBMs. See related article by Korsah et al., p. XX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a02add441c27bf4e8e365b8edcace16e00d2cda" target='_blank'>
              Extrachromosomal DNA-Driven Oncogene Spatial Heterogeneity and Evolution in Glioblastoma.
              </a>
            </td>
          <td>
            I. Noorani, Magnus Haughey, J. Luebeck, Andrew Rowan, E. Grönroos, Francesco Terenzi, I. Wong, Davide Pradella, Marta Lisi, Jeanette Kittel, Natasha Sharma, Chris Bailey, C. Weeden, Donald M Bell, Eric Joo, Vittorio Barbè, Matthew G. Jones, King L. Hung, E. Nye, M. Green, Lucy Meader, Emma Norton, Mark Fabian, N. Kanu, M. Jamal-Hanjani, T. Santarius, Andrea Ventura, James A R Nicoll, D. Boche, Howard Y. Chang, V. Bafna, Weini Huang, P. Mischel, C. Swanton, Benjamin Werner
          </td>
          <td>2025-09-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="DNA abnormalities characterized by cytogenetic imaging at the single cell resolution, i.e. karyotyping, have long served as cancer diagnostic and prognostic biomarkers. To enable in-silico karyotyping using unpaired whole-genome sequencing data, we developed Seq2Karyotype (S2K), a tool that fits karyotype models with clonality estimation based on read-depth and allelic imbalance in a bulk sample and supports visualization-guided refinement. Analysis on 19 adult and pediatric cancer cell lines revealed unexpected intratumoral heterogeneity involving multiple copy number variation (CNV) states including whole-genome duplication, which were validated by imaging and single-cell omics profiling. Analyses on patient samples showed high concordance with clinical cytogenetic reports for acute myeloid leukemia, and revealed evolutionary trajectories from multi-region metastatic neuroblastomas implicating reversion. These findings highlight extensive and dynamic intratumoral heterogeneity contributed by CNV in both cell line models and patient samples, which may inform future research on tumor evolution under selective pressure such as drug exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0572920d9d208e3e04940659c463f83c4e0642eb" target='_blank'>
              Seq2Karyotype (S2K): A Method for in-silico Karyotyping Using Single-Sample Whole-Genome Sequencing Data
              </a>
            </td>
          <td>
            Limeng Pu, Karol A. Szlachta, Virginia Valentine, Meiling Jin, Sivaraman Natarajan, Robert Greenhalgh, Nadezhda V. Terekhanova, Jian Wang, Daniel K. Putnam, Li Dong, Lingxiang Jiang, Soumya Tumbath, Xiumei Huang, Xiaotu Ma, Thomas Look, Marcin Wlodarski, Lu Wang, Steven Burden, J. Easton, Xiang Chen, Jinghui Zhang
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69d250508121e21820a44d64995e2f038263f7bb" target='_blank'>
              Clone copy number diversity is linked to survival in lung cancer.
              </a>
            </td>
          <td>
            Piotr Pawlik, Kristiana Grigoriadis, Abigail Bunkum, Helena Coggan, A. Frankell, Carlos Martínez-Ruiz, Takahiro Karasaki, A. Huebner, Andrew Rowan, Jasmin Fisher, A. Hackshaw, C. Swanton, S. Zaccaria, N. Mcgranahan
          </td>
          <td>2025-08-13</td>
          <td>Nature</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Multiple metrics for read-level DNA methylation pattern analysis have provided new insights into DNA methylation modifications. However, the performance of these metrics and their relationship with DNA methylation ratios in identifying biologically meaningful regions have remained unclear. Here, we systematically benchmarked five read-level DNA methylation metrics using whole-genome bisulfite sequencing data from 59 individuals across six healthy tissue types and six tumor types. We found that DNA methylation concurrence (MCR) effectively captured tissue-specific features independent of the DNA methylation ratios. Regions that exhibited decreased MCR (MCDRs) in tumors were significantly enriched in promoter and intergenic regions and strongly overlapped with tumor-gained chromatin accessibility sites. The further analysis of histone modifications, including H3K4me3, H3K27ac, and H3K9ac, confirmed that MCDRs marked active gene regulatory elements. Motif enrichment analysis revealed a strong preference for CTCF binding within MCDRs. Additionally, 3D genome analysis supported a model in which MCDRs, independent of DNA methylation ratios, contribute to active gene regulation by facilitating CTCF binding and long-range chromatin interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e01ae2e6ef4d74e2fae8d606fc1cf846f2e7bba" target='_blank'>
              DNA Methylation Concurrence, Independent of DNA Methylation Ratios, Is Associated with Chromatin Accessibility and 3D Genome Architecture
              </a>
            </td>
          <td>
            Guian Zhang, Yixian Yang, Dan Cui, Jia Li
          </td>
          <td>2025-07-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 In tumors, clonal populations of malignant cells evolve a diverse repertoire of copy number variants (CNVs), often resulting in profound intra-tumor heterogeneity (ITH). ITH is associated with poor prognosis, cancer progression, increased risk of therapy resistance, metastasis and relapse. Traditionally, researchers have studied clonal heterogeneity and tumor evolution using bulk-tissue assays, however, these techniques provide little or no spatial information and have low sensitivity for focal/low copy-number CNVs, particularly when they are subclonal or rare. DNA FISH is the gold standard for detecting CNVs at cellular resolution, but it is limited to one gene per tissue section. To overcome the above limitations, we developed multiplexed-CNV FISH, a spatial omics assay that detects gain and loss of DNA loci at single-gene (∼10kb) and single-cell resolution in human tissues. Remarkably, CNV-FISH analysis of CRC tumor sections identified small and subclonal CNVs not detected by existing methods with resolution of CNV spanning whole chromosome arms. Our results indicate that CNV-FISH can reveal unexpected and previously undetected CN heterogeneity within tumors, which could potentially underlie treatment resistance and eventual metastasis of cancer cells.



 Mei Suen KONG, Hui Ting Grace YEO, Jiamin TOH, Lin Li, Jonathan Shaozhong Aow, Joanito Irwan IGNASIUS, Jeeranan BOONRUANGKAN, Jen Yi WONG, Nirmala ARUL RAYAN, Lavanya MUTHUKRISHNAN, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon LIM, Bee Huat Iain TAN, Kok Hao CHEN, Shyam PRABHAKAR. Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P06.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae7336ba20d4eb282822f6335d4e3b64784f2e7c" target='_blank'>
              Abstract P06: Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer
              </a>
            </td>
          <td>
            Mei Suen KONG, Hui Ting Grace YEO, Jiamin Toh, Lin Li, Jonathan Aow, Joanito Irwan IGNASIUS, Jeeranan Boonruangkan, Jen Yi WONG, N. A. Rayan, Lavanya Muthukrishnan, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon Lim, Bee Huat Iain TAN, Kok Hao CHEN, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT Tumors of infectious origin globally represent 13%. Oncogenic DNA viruses such as human papillomavirus (HPV), hepatitis B virus (HBV), and Epstein-Barr virus (EBV) are responsible for approximately 60% of these tumors. These oncoviruses are extensively studied to understand their role in cancer development, particularly through viral genome integration into the host DNA. Retroviruses require integration mediated by viral integrase for persistence, whereas DNA oncoviruses do not need integration for replication; instead, integration occurs incidentally. This process often targets fragile sites in the human genome, causing structural rearrangements that disrupt genes, activate proto-oncogenes, and increase genomic instability, all contributing to tumorigenesis. Integration near promoter regions and active genes is closely linked to carcinogenesis, highlighting its importance in developing diagnostic and therapeutic strategies. This review summarizes viral integration’s role in oncogenesis, mechanisms of integration, and methods to study this process, focusing on DNA tumor viruses such as HBV, EBV, HPV, and Merkel cell polyomavirus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ef1eb5cd84b08745894e948edf6d9875f648ea7" target='_blank'>
              Genome integration of human DNA oncoviruses
              </a>
            </td>
          <td>
            Zuzana Vojtěchová, R. Tachezy
          </td>
          <td>2025-07-23</td>
          <td>Journal of Virology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a common feature of human cancers and is linked to tumour progression, drug resistance, and metastasis1–6. Here we examine the impact of WGD on somatic evolution and immune evasion at single-cell resolution in patient tumours. Using single-cell whole-genome sequencing, we analysed 70 high-grade serous ovarian cancer samples from 41 patients (30,260 tumour genomes) and observed near-ubiquitous evidence that WGD is an ongoing mutational process. WGD was associated with increased cell–cell diversity and higher rates of chromosomal missegregation and consequent micronucleation. We developed a mutation-based WGD timing method called doubleTime to delineate specific modes by which WGD can drive tumour evolution, including early fixation followed by considerable diversification, multiple parallel WGD events on a pre-existing background of copy-number diversity, and evolutionarily late WGD in small clones and individual cells. Furthermore, using matched single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signalling and cGAS-STING pathway activation result from ongoing chromosomal instability, but this is restricted to predominantly diploid tumours (WGD-low). By contrast, predominantly WGD tumours (WGD-high), despite increased missegregation, exhibited cell-cycle dysregulation, STING1 repression, and immunosuppressive phenotypic states. Together, these findings establish WGD as an ongoing mutational process that promotes evolvability and dysregulated immunity in high-grade serous ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2a0c8aae8bd565763175795d01ffdf375af883" target='_blank'>
              Ongoing genome doubling shapes evolvability and immunity in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Duaa H Al-Rawi, Matthew Zatzman, A. Weiner, Samuel Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nicholas Ceglia, Danguolė Norkūnaitė, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, Ginger J Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, R. Grisham, Ying L Liu, A. Viale, Nicole Rusk, Yulia Lakhman, Lora H. Ellenson, S. Tavaré, Sam Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2025-07-16</td>
          <td>Nature</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97e0ce2d3aba4ba6a29f1b308846500c4a3d7ec7" target='_blank'>
              Beyond inversions and deletions: the evolutionary and functional insights from translocations, fissions, and fusions in animal genomes.
              </a>
            </td>
          <td>
            Célian Diblasi, M. Saitou
          </td>
          <td>2025-08-01</td>
          <td>Heredity</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chromosome-derived circular DNAs have different characters from linear DNAs and RNAs, which exert distinctive functions, and participate in a widespread range of physiological and pathological processes. Although, the composition, structure and genome-wide frequency of circular DNAs have not yet been illuminated systemically, circular DNAs are identified to act as unanticipated major sources of somatic rearrangements, which is an important genomic feature in tumorigenesis and represents a multihit and ongoing mutagenic process. And, this process remarkably contributes to oncogenic remodeling and oncogenes amplification through chimeric circularization and reintegration of circular DNAs into the linear genome, which is closely associated with high cancer mortality and morbidity. It has been found that circular DNAs-stimulated oncogenes amplification and overexpression implicate in diverse cytopathological processes during cancer progression, including cell proliferation, apoptosis, autophagy, epithelial-mesenchymal transition (EMT), extracellular matrix (ECM) stiffness and angiogenesis. Herein, we comprehensively introduced circular DNAs’ formation and characteristics, and emphatically elaborated circular DNAs-induced oncogenes amplification and overexpression and their functional mechanisms regulating cancer behaviors. Lastly, some mysteries in oncogenic circular DNAs’ study were introduced, and the perspectives were also discussed in cancer diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/825b11b400d7213811736a403636f24a5a529f78" target='_blank'>
              Circular DNAs Participate in Tumorigenesis and Its Mechanisms
              </a>
            </td>
          <td>
            Yuan Tan, Faqing Tang
          </td>
          <td>2025-07-30</td>
          <td>Clinical Medicine and Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains a lethal malignancy with poor prognosis. We investigated intratumoral deoxyribonucleic acid methylation heterogeneity by analyzing 44 tumor samples and 5 normal samples from 6 cases of PDAC by using high-resolution methylation arrays. Two distinct methylation profiles were identified: T1, which is similar to normal pancreatic tissue and is associated with well-differentiated histology, and T2, which is significantly different from normal tissue and is linked to poorly differentiated morphology and squamous features. Validation using The Cancer Genome Atlas (TCGA) confirmed these profiles and revealed the association of T2 with shorter disease-free survival (p = 0.04). Differentially methylated region analysis identified the substantial hypomethylation of transcription regulation genes in T2 profiles (false discovery rate [FDR] q < 0.001). Gene set enrichment analysis with TCGA gene expression data demonstrated the upregulation of DNA repair and MYC target genes in T2 samples (FDR q < 0.001). Phylogenetic analysis with our multi-sampling dataset suggested an evolutionary trajectory from T1 to T2 profiles coinciding with aggressive phenotypes and increased genomic instability. Cases exhibited varying degrees of intratumoral heterogeneity from distinctly separated clusters to minimal differences. This comprehensive characterization of the epigenetic landscape of PDAC provides insights into tumor evolution and heterogeneity with potential implications for patient stratification and the development of epigenetic-based diagnostic and therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-13919-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd12eab03975f1f1da59d2699d262a5bc0933404" target='_blank'>
              Epigenetic evolution and clinicopathological implications of distinct DNA methylation profiles in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            K. Kitahama, Yu-Jui Ho, Kaishi Satomi, Takahiro Shibayama, Kiyotaka Nagahama, K. Ohtsuka, Hiroaki Ohnishi, Yoshihiro Sakamoto, Junji Shibahara, Akimasa Hayashi
          </td>
          <td>2025-08-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Streptococcus pneumoniae (pneumococcus) is a genetically diverse opportunistic bacterial pathogen that expresses two phase-variable loci encoding restriction-modification systems. Comparisons of two genetically-distinct pairs of epigenetically-distinct variants, each distinguished by a stabilised arrangement of one of these phase-variable loci, found the consequent changes in genome-wide DNA methylation patterns were associated with differential expression of mobile genetic elements (MGEs). This relationship was hypothesised to be mediated through changes in xenogenic silencing (XS) or nucleoid organisation. Therefore the chromosomal conformation of both variants of each pair were characterised using Illumina Hi-C, and Nanopore Pore-C, sequencing. Both methods concurred that the organisation of the pneumococcal chromosome was dominated by small-scale structures, with most pairwise interactions between loci <25 kb apart. Neither found substantial evidence for higher-order structure or XS in the pneumococcal genome, with more complex contact patterns only evident around the replication origin. Comparisons between the variants identified phage-related chromosomal islands (PRCIs) as the foci of differential contact densities between the variants. This was driven by copy number variation, resulting from variable excision and replication of the episomal PRCIs. However, the methods were discordant in their identification of the variant in which the PRCI was more actively replicating in both pairs. Validatory experiments demonstrated that the prevalence of circular PRCIs was not determined by DNA modification, but instead varied stochastically between colonies in both backgrounds, and was metastable during vegetative growth. PRCI excision was inducible by mitomycin C, but independent of the presence of a prophage. Yet transcriptional activation of these elements was affected by both signals, indicating transcription and replication are separately regulated. Therefore pneumococcal MGEs do not appear to be subject to XS, resulting in heterogeneity being generated within these bacterial populations through the frequent local disruption of chromosome conformation resulting from the stochastic excision and reintegration of episomal elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e055c4fe6d0e438363a006251436d6f88a77020" target='_blank'>
              Pore-C sequencing identifies episome-driven chromosome conformation perturbations differentiating pneumococcal epigenetic variants
              </a>
            </td>
          <td>
            Tze Yee Lim, Samuel T. Horsfield, C. Troman, Stephen D. Bentley, M. Kwun, N. Croucher
          </td>
          <td>2025-08-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are linked to cancer progression and poor prognosis, offering valuable genetic insights into tumors. Accurate detection of genomic alterations in CTCs is essential for improving cancer diagnosis and treatment. To address this, we develop Uniform Chromosome Conformation Capture (Uni-C), a method for profiling 3D chromatin architecture and genomic alterations at the single-cell level. Using Uni-C, we analyze CTCs from pancreatic cancer patient-derived xenograft (PDX) and spontaneous tumor mouse models. In the PDX model, integrating data from seven CTCs captures 88.7% of SNPs and INDELs, and 75.0% of structural variants present in tumor tissue. These findings indicate that variants detected in CTCs reflect tumor genomic features. Notably, we observe chromatin conformation differences between mitotic and interphase CTCs, suggesting potential markers of cell vitality. In the spontaneous tumor model, we identify driver gene mutations in CTCs and predict neoantigens, advancing early cancer detection and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcdb93a8b0e33fd2f466feefe41fc91b323a6623" target='_blank'>
              Identification of multiple genomic alterations and prediction of neoantigens from circulating tumor cells at the single-cell level
              </a>
            </td>
          <td>
            Xiaochen Gao, Xinyu Li, Weize Xu, Ming Jiao, Yu Guo, Jiajia Wang, Weihao Wang, Jiling Feng, Qianqian Guo, Chengchao Wu, Taiyu Zhang, Yuqin Yang, Da Lin
          </td>
          <td>2025-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Acral melanoma (AM) is an aggressive melanoma subtype with limited therapeutic options and poor outcomes. In non-European descent and admixed populations, like those residing in Latin America, AM accounts for a significant proportion of cutaneous melanoma cases. Here, we performed comprehensive genomic and functional profiling of AM from a uniquely diverse Brazilian cohort. Whole-exome and transcriptome sequencing revealed low mutation burden and predominance of copy number alterations, including high-amplitude focal amplifications termed hailstorms. These hailstorms frequently affected chromosomes 11, 5 and 22 and key oncogenes such as CCND1, GAB2, CDK4, and TERT. The presence of hailstorms in the long arms of chromosomes 11 and 22 was associated with higher focal copy number burden and loss of DNA damage response genes (ATM, CHEK1), suggesting a permissive genomic environment driving structural instability. To explore the unique genomic context of AM, we established a comprehensive collection of patient-derived xenograft (AM-PDX) models that faithfully retain the histopathological and genomic features of the original tumours. Functional exploration of AM-specific vulnerabilities through pharmacological and CRISPR/Cas9 knockout screenings identified strong sensitivity to targeting MAPK, CDK4/6, MDM2, and WEE1 pathways. Notably, the pan-RAS(ON) inhibitor RMC-7977 effectively reduced viability in NRAS-, KRAS-, and KIT-mutant AM cell lines. Finally, CRISPR screens revealed AM-enriched dependencies, including CRKL and SF3B4, highlighting previously unrecognized vulnerabilities. Our findings emphasize the distinct biology of AM compared to other subtypes of melanoma, provide a valuable resource of models reflective of Latin American ancestry, and identify potential drivers and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0407eee77f76a83d31f10dc4bc4ede91f52ba5a" target='_blank'>
              Modelling Acral Melanoma in Admixed Brazilians Uncovers Genomic Drivers and Targetable Pathways
              </a>
            </td>
          <td>
            Annie Cristhine Moraes Sousa-Squiavinato, Sara Santos Bernardes, Flavia C. Aguiar, Antonio C. Facciolo, Martín del Castillo Velasco Herrera, J. R. C. Wong-Ramirez, P. Basurto-Lozada, Aretha Brito Nobre, Geethanjali Annamalai, Rebecca Martins Cadimo do Nascimento, Jacqueline M Boccacino, Rafaela Fagundes, Pedro Sodré do R. Barros, Mariana de Moraes Pitombo, R. Olvera-León, Larissa Satiko Alcantara Sekimoto Matsuyama, J. Billington, I. Vermes, Irving Simonin-Wilmer, D. G. Carvalho, João Pedro Cavalcante Simoes, Priscila Valverde Fernandes, Luiz Fernando Nunes, Andreia Cristina de Melo, Jadivan Leite de Oliveira, M. Herlyn, Andrew E. Aplin, C. D. Robles-Espinoza, David J. Adams, P. Possik
          </td>
          <td>2025-08-12</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>117</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are frequently observed in tumors, yet their role in pediatric cancers remains poorly understood. The heteroplasmic nature of mtDNA—where mutant and wild-type mtDNA coexist—complicates efforts to define its contribution to disease progression. In this study, bulk whole-genome sequencing of 637 matched tumor-normal samples from the Pediatric Cancer Genome Project revealed an enrichment of functionally impactful mtDNA variants in specific pediatric leukemia subtypes. Collectively, the results from single-cell sequencing of five diagnostic leukemia samples demonstrated that somatic mtDNA mutations can arise early in leukemogenesis and undergo positive selection during disease progression, achieving intermediate heteroplasmy—a “sweet spot” that balances mitochondrial dysfunction with cellular fitness. Network-based systems biology analyses link specific heteroplasmic mtDNA mutations to metabolic reprogramming and therapy resistance. We reveal somatic mtDNA mutations as a potential source of functional heterogeneity and cellular diversity among leukemic cells, influencing their fitness and shaping disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b147587976da235e9ad9ba432d45c69ceee07d90" target='_blank'>
              Somatic mtDNA mutations at intermediate levels of heteroplasmy are a source of functional heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, Liang Ding, , Robert J. Autry, B. Sejdiu, Q. Pan, M. Franco, Wenan Chen, Huiyun Wu, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, S. Brady, Jinghui Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, Jiyang Yu, Gang Wu, S. Pounds, M. Kundu
          </td>
          <td>2025-09-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Genome instability is a fundamental feature and hallmark of cancer, associated with aggressiveness, drug resistance and poor prognosis. RAI2 was initially identified as a novel metastasis suppressor protein specifically associated with the presence of disseminated tumour cells in the bone marrow of breast cancer patients, but its molecular function is largely unknown. Methods We analysed the consequences of RAI2 depletion on gene expression and genomic stability in luminal breast cancer cell lines, performed cytotoxicity profiling using a library of pharmacologically active compounds, and characterized a potential function of the RAI2 protein in the DNA damage response. We performed in silico validation in different breast cancer datasets. Results Analysis of clinical samples revealed that in primary breast tumours, low RAI2 gene expression is significantly associated with genomically unstable tumours and poor prognosis. RAI2 depletion in breast cancer cell lines resulted in loss of mitotic fidelity characterized by prolonged mitosis with increased chromosome segregation errors and micronuclei formation. Drug screening revealed increased sensitivity of RAI2-depleted breast cancer cells to topoisomerase I and Aurora A inhibitors. We also found that genotoxic stress induces the RAI2 protein, which has an affinity for and colocalises with poly-(ADP-ribose). We validated the association of RAI2 gene expression with DNA repair capacity in clinical samples. Conclusions Our findings support, for the first time, a functional role of RAI2 in the maintenance of genomic stability. Understanding the underlying the molecular mechanism could help to improve patient diagnosis and treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02085-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d5723a488a418c3d94e5cf45e2172b0ac676d31" target='_blank'>
              Retinoic acid-induced 2 deficiency impairs genomic stability in breast cancer
              </a>
            </td>
          <td>
            Lena Boettcher, Sarah Greimeier, Kerstin Borgmann, Shabbir S. Mughal, Bernhard Ellinger, K. Bartkowiak, B. Zobiak, Antonio V. Failla, Pascal Steffen, Ellen Claus, K. Besler, Christopher Buccitelli, Violetta Schaaf, A. Ozga, Simona Parretta, Svenja Schneegans, W. Mansour, Jan O. Korbel, Hartmut Schlueter, B. Brors, Klaus Pantel, H. Wikman, S. Werner
          </td>
          <td>2025-07-22</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="DNA methylation regulation involves multi-layered chromatin interactions that require remodeling proteins like the helicase, lymphoid-specific (HELLS). Here, we generate HELLS and DNA methyltransferase 3A and B (DNMT3A/B) knockout human pluripotent stem cells and report telomere-to-telomere maps of whole genome bisulfite sequencing data combined with ATAC-sequencing. Disrupting HELLS induces a global loss of DNA methylation that is distinct from the DNMTs, in particular over peri/centromeric satellite repeats as defined in the telomere-to-telomere genome assembly. However, HELLS appears dispensable for local enhancer remodeling and the potential to differentiate into the three embryonic germ layers. Taken together, our results further clarify the genomic targets and role of HELLS in human cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03681-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/def329d62128136166a3e0287e80d3095f66e512" target='_blank'>
              HELLS is required for maintaining proper DNA modification at human satellite repeats
              </a>
            </td>
          <td>
            Philine Guckelberger, Leah Haut, Rosaria Tornisiello, H. Kretzmer, Alexander Meissner
          </td>
          <td>2025-07-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Whole-genome analyses have revealed that large-scale structural variations (SVs) such as whole-genome duplication (WGD) occur early in the development of many cancers. However, the diversity of chromosomal abnormalities within tumors before and after WGD remains poorly understood. Here, we analyzed various types of Japanese tumor genomes via whole-genome sequencing and examined the diversity of WGD by focusing on large SVs at the chromosomal level. WGD was detected in 52% of cases, while the frequency of chromothripsis (CT) was 20%. Although aneuploidy via deletion of chromosome arms was common in many cancers, in rare ovarian cancers, all chromosomes were near-haploidy before WGD. Minor allele analysis revealed that many non-mutated ohnolog genes drifted down chromosome arms after WGD and returned to normal ploidy, but only 17p, including TP53, which is also an ohnolog, underwent loss of heterozygosity due to arm deletion before WGD in most cancers. TP53 mutations were frequently detected in WGD and CT-positive tumors, and these SVs strongly correlated with homologous recombination deficiency scores. Furthermore, these tumors had many mutations that continued to generate neoantigens and resulted in worse survival outcomes. Diversity analysis of tumors with WGD will provide a new perspective on structural abnormalities in tumor genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53236a139dc9b0a444537137348625f035c225eb" target='_blank'>
              Allelic Imbalance and Chromothripsis Lead to Diversity in Japanese Tumor Genomes With Whole-Genome Duplication.
              </a>
            </td>
          <td>
            K. Hatakeyama, Takeshi Nagashima, Sumiko Ohnami, S. Ohnami, Koji Maruyama, K. Ohshima, Y. Shimoda, A. Naruoka, H. Kenmotsu, K. Urakami, Y. Akiyama, Ken Yamaguchi
          </td>
          <td>2025-08-06</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Functional inactivation of tumor suppressor genes drives cancer initiation, progression, and treatment responses. Most tumor suppressor genes are inactivated through 1 of 2 well-characterized mechanisms: DNA-level mutations, such as point mutations or deletions, and promoter DNA hypermethylation. Here, we report a distinct third mechanism of tumor suppressor inactivation based on alterations to the histone rather than DNA code. We demonstrated that PAX2 is an endometrial tumor suppressor recurrently inactivated by a distinct epigenetic reprogramming event in more than 80% of human endometrial cancers. Integrative transcriptomic, epigenomic, 3D genomic, and machine learning analyses showed that PAX2 transcriptional downregulation is associated with replacement of open/active chromatin features (H3K27ac/H3K4me3) with inaccessible/repressive chromatin features (H3K27me3) in a framework dictated by 3D genome organization. The spread of the repressive H3K27me3 signal resembled a pearl necklace, with its length modulated by cohesin loops, thereby preventing transcriptional dysregulation of neighboring genes. This mechanism, involving the loss of a promoter-proximal superenhancer, was shown to underlie transcriptional silencing of PAX2 in human endometrial cancers. Mouse and human preclinical models established PAX2 as a potent endometrial tumor suppressor. Functionally, PAX2 loss promoted endometrial carcinogenesis by rewiring the transcriptional landscape via global enhancer reprogramming. The discovery that most endometrial cancers originate from a recurring epigenetic alteration carries profound implications for their diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5897c9bf68c93e4fb62e86f78e7178d27155eec3" target='_blank'>
              A distinct mechanism of epigenetic reprogramming silences PAX2 and initiates endometrial carcinogenesis
              </a>
            </td>
          <td>
            Subhransu S. Sahoo, Susmita G Ramanand, Ileana C. Cuevas, Yunpeng Gao, Sora Lee, Ahmed Abbas, Xunzhi Zhang, Ashwani Kumar, Prasad R. Koduru, Sambit Roy, Russell R. Broaddus, Victoria L Bae-Jump, Andrew B. Gladden, Jayanthi Lea, Elena Lucas, Chao Xing, Akio Kobayashi, Ram-Shankar Mani, Diego H. Castrillon
          </td>
          <td>2025-08-15</td>
          <td>The Journal of Clinical Investigation</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background Human endogenous retroviruses (HERVs) harbor accessory proteins that influence cellular processes and have been linked to a wide variety of diseases, including cancer. This study investigates locus-specific HERV expression and its association with gene dysregulation in hepatocellular carcinoma (HCC), a highly prevalent and deadly form of liver cancer worldwide. Methods We analyzed RNASeq data from 424 HCC samples from The Cancer Genome Atlas (TCGA), which comprised 371 tumor and 50 matched normal tissues from a total of 371 hepatocellular carcinoma participants. We employed Telescope to identify and quantify HERV expression across the total RNA sequencing data. Results The majority of differentially expressed HERVs exhibited reduced expression in tumor tissue (166 downregulated vs. 50 upregulated), suggesting a potential functional role of HERV expression patterns in shaping the pathophysiological landscape of HCC. Specifically, the suppression of HERV-H family members, which are known to regulate cellular differentiation, may contribute to tumor dedifferentiation, increased plasticity, and enhanced metastatic potential. This loss of differentiation control and increased adaptability may play a critical role in driving the progression of liver cancer. Discussion Our study highlights a significant association of HERV expression with HCC, highlighting the differential regulation of specific HERV families in tumor tissue. For example, HERVH and ERVLE families showed consistent downregulation in tumor samples, while HERVE and HERV9 were more commonly upregulated. These shifts may reflect underlying changes in transcriptional regulation or chromatin structure between normal and malignant tissues. Rather than indicating a singular functional role, the observed expression patterns likely reflect a multifaceted relationship between HERVs and tumor biology. Further studies will be needed to determine whether these expression differences contribute to, or result from, tumor progression and to explore their potential as biomarkers or therapeutic targets. Clinical trial number Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00367-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06da0045d446b88205ce15609079cd44c93959a4" target='_blank'>
              Locus-specific HERV expression associated with hepatocellular carcinoma
              </a>
            </td>
          <td>
            Hui Wen, M. Pérez‐Losada, Lopa Mishra, Keith A. Crandall
          </td>
          <td>2025-07-16</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Tardigrades are microscopic animals that can survive exceptional levels of ionizing radiation or desiccation – DNA-damaging conditions that would kill most animals. Irradiation or radiomimetic drug treatment of the tardigrade Hypsibius exemplaris can induce remarkably high expression levels of DNA repair genes, primarily those in the base excision repair and nonhomologous end joining pathways. How tardigrades can repair widespread DNA damage without producing frequent, large-scale chromosome structural abnormalities, like chromosome translocations and fusions, is unknown. Here, we report the results of examining chromosome and nuclear architecture throughout the cell cycle in early embryos of H. exemplaris. We found that H. exemplaris chromosomes are maintained in an individualized form throughout the cell cycle. We were surprised to also find that each chromosome is housed in a fully or partially separate lamin-lined compartment, instead of all chromosomes being housed in a single, nearly spherical nuclear lamina and envelope. Our results reveal unusual chromosomal and nuclear organization in a tardigrade. We speculate that these unexpected features might limit chromosomal rearrangements during DNA damage repair in extreme conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59b0e24503d3afcd35a7b4c4c2e06e9d2060436" target='_blank'>
              Chromosomes remain individualized through interphase in embryos of the tardigrade Hypsibius exemplaris
              </a>
            </td>
          <td>
            Lillian D. Papell, Adriana N Coke, Bailey N. de Jesus, Clayton J. Harry, Pu Zhang, Bob Goldstein
          </td>
          <td>2025-07-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumors (ATRTs) are aggressive central nervous system tumors mainly affecting young children. Extensive molecular characterization based on gene expression and DNA methylation patterns has solidly established three major ATRT subtypes (MYC, SHH and TYR), which show distinct clinical features, setting the basis for more effective, targeted treatment regimens. Transcriptional activity of transposable elements (TEs), like LINE1s and LTRs, is tightly linked with human cancers as a direct consequence of lifting epigenetic repression over TEs. The sole recurrent biallelic loss-of-function mutation in SMARCB1 in ATRTs, a core component of the SWI/SNF chromatin remodeling complex, raises the question of how TE transcription contributes to ATRT development. Here, we comprehensively investigate the transcriptional profiles of 1.9M LINE1 and LTR elements across ATRT subtypes in primary human samples. We find TE transcription profiles allow sample stratification into ATRT subtypes. The TE activity signature in the ATRT-MYC subtype is unique, setting these tumors apart from SHH and TYR ATRTs. More specifically, ATRT-MYC show broadly reduced transcript levels of LINE1 and ERVL-MaLR subfamilies. ATRT-MYC also displayed significantly less LTR and LINE1 loci with bidirectional promoter activity. Furthermore, we identify 849 differentially transcribed TEs in primary samples, which are predictive towards established ATRT-SHH and -MYC cell line models. In summary, including TE transcription profiles into the molecular characterization of ATRTs might reveal new tumor vulnerabilities leading to novel therapeutic interventions, such as immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02078-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c60e5327059223ad1194aac3ab94faeb7d65828" target='_blank'>
              Expression of LTR and LINE1 transposable elements defines atypical teratoid/rhabdoid tumor subtypes
              </a>
            </td>
          <td>
            M.V. Hamann, Shweta Godbole, Maisha Adiba, Sabrina M. Leddy, Jelena Navolić, Ghazaleh Tabatabai, Daniel J Merk, Ulrike C. Lange, J. Neumann
          </td>
          <td>2025-07-22</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53f0d40328372c75f3f03439d0b2ad59f2263c92" target='_blank'>
              Transcriptional regulation of Ligase IV by an intronic regulatory element directs thymocyte development.
              </a>
            </td>
          <td>
            Matthew D Estrada, Christopher J Gebhardt, Mariam A Salem, Christina N Rau, Kruthika Sharma, Rebecca A Glynn, Craig H Bassing, Eugene M Oltz, Patrick L. Collins
          </td>
          <td>2025-09-05</td>
          <td>Genes and immunity</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="KRAS is mutationally activated in 45–50% of colorectal cancer (CRC) cases, and while KRAS-targeted therapies have shown some clinical promise, upfront and acquired resistance limit their efficacy. To explore the acute response and mechanisms underlying KRAS inhibitor resistance, we used targeted exome sequencing and single-cell spatial transcriptomics to analyze patient-matched pre-treatment, on-treatment, and progression biopsies from patients treated with combined KRASG12C and EGFR inhibition. Acquired genetic events were identified in most patients at progression but were often subclonal and coexisted with transcriptional adaptive states. Mesenchymal, YAP, and fetal-like transcriptional signatures predominated in resistant tumors, while tumor cell-intrinsic inflammatory programs were induced in the early treatment phase. Single-cell spatial analysis revealed significant intratumoral heterogeneity, with diverse adaptive states predominating in different zones of individual tumors. Using human and murine organoid models, we show that these drug-induced inflammatory programs are cancer-cell autonomous and precede the emergence of regenerative fetal-like programs associated with drug resistance. We uncover TBK1 as a promising target to abrogate the early inflammatory adaptive phase and enhance responses to KRAS inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75a7967138a21f862a2f0477898f16663439bb11" target='_blank'>
              Concurrent genetic and non-genetic resistance mechanisms to KRAS inhibition in CRC
              </a>
            </td>
          <td>
            Salvador Alonso, Kevan Chu, Marie J Parsons, Elizabeth Granowsky, H. Gunasinghe, J. Shia, R. Yaeger, Lukas E. Dow
          </td>
          <td>2025-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Genome instability is a hallmark of cancer, often driven by mutations and altered expression of genome maintenance factors involved in DNA replication and repair. Rap1 Interacting Factor 1 (RIF1) plays a crucial role in genome stability and is implicated in cancer pathogenesis. Cells express two RIF1 splice variants, RIF1-Long and RIF1-Short, which differ in their ability to protect cells from DNA replication stress. Here, we investigate differential expression and splicing of RIF1 in cancer cell lines following replication stress and in patients using matched normal and tumour data from The Cancer Genome Atlas (TCGA). Overall RIF1 expression is altered in several cancer types, with increased transcript levels in colon and lung cancers. RIF1 also exhibits distinct splicing patterns, particularly in specific breast cancer subtypes. In Luminal A (LumA), Luminal B (LumB), and HER2-enriched breast cancers (HER2E), RIF1 Exon 31 tends to be excluded, favouring RIF1-Short expression and correlating with poorer clinical outcomes. These breast cancer subtypes also tend to exclude other short exons, suggesting length-dependent splicing dysregulation. Basal breast cancer also shows exon exclusion, but unlike other subtypes, it shows no short-exon bias. Surprisingly, however, in basal breast cancer, RIF1 Exon 31 is not consistently excluded, which may impact prognosis since RIF1-Long protects against replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ca94768efdf45848f937fd47caaaca2fe19dbc3" target='_blank'>
              Dysregulated Alternative Splicing in Breast Cancer Subtypes of RIF1 and Other Transcripts
              </a>
            </td>
          <td>
            Emma Parker, Laura Akintche, Alexandra Pyatnitskaya, S. Hiraga, A. Donaldson
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The identification of CENPA, CENPB, and CENPC by Earnshaw and Rothfield 40 years ago has revealed the remarkable diversity and complexity of centromeres and confirmed most seed plants and animals have centromeres comprised of complex satellite arrays. The rapid evolution of centromeres and positive selection on CENPA and CENPC led to the centromere drive model, in which competition between tandem satellite arrays of differing size and centromere strength for inclusion in the egg of animals or megaspore of seed plants during female meiosis drives rapid evolution of centromeres and kinetochore proteins. Here we review recent work showing that non-B-form DNA structures in satellite centromeres make them sites of frequent replication fork stalling, and that repair of collapsed forks by break-induced replication rather than unequal sister chromatid exchange is likely the primary mode of satellite expansion and contraction, providing the variation in satellite copy number that is the raw material of centromere drive. Centromere breaks at replication, rather than errors at mitosis, can account for most centromere misdivisions that underlie aneuploidies in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65fb737fe39c4474778577289e08e6983374bf35" target='_blank'>
              Centromeres drive and take a break
              </a>
            </td>
          <td>
            P. Talbert, S. Henikoff
          </td>
          <td>2025-08-04</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Significance Modified cytosine bases are essential components of mammalian genomes, but how they are distributed in double-stranded DNA remains unclear. Here, we report a method to simultaneously sequence the two most abundant modified bases—5-methylcytosine and 5-hydroxymethylcytosine (hmC)—across both strands of genomic DNA. Applying this method to DNA from mouse embryonic stem cells, we find that at CpG sites, hmC forms different combinations with cytosine variants across the DNA double helix, exhibiting unique distributions at enhancers and gene bodies, and distinct relationships with transcription. Our findings demonstrate that the double-stranded context of hmC is a key feature of the epigenome, revealing a further layer of regulatory information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8822428faae43abf3590008b224a007bfe559580" target='_blank'>
              SCoTCH-seq reveals that 5-hydroxymethylcytosine encodes regulatory information across DNA strands
              </a>
            </td>
          <td>
            Jack S Hardwick, Somdutta Dhir, Angie Kirchner, A. Simeone, Sean M. Flynn, James M. Edgerton, Rafael de Cesaris Araujo Tavares, Isabel Esain-Garcia, D. Tannahill, Pauline Y. Golder, Jack M Monahan, Walraj S. Gosal, Shankar Balasubramanian
          </td>
          <td>2025-07-31</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Background: Genomic instability, a hallmark of cancer, leads to copy number variations disrupting gene dosage balance and contributing to tumor progression. One of the most affected oncogenes is MYC, whose overexpression is tightly regulated to avoid cytotoxicity. In aneuploid cancer cells, gene dosage compensation mechanisms involving microRNAs (miRNAs) from the miR-17/92 cluster contribute in regulating MYC expression. Targeting this miRNA-mediated compensation system represents a promising therapeutic strategy leading to an uncontrolled and lethal MYC overexpression. Results: Synthetic miRNA sponges targeting miR-17, miR-19a, and miR-20a, key regulators of MYC dosage compensation, were designed and validated. Breast cancer cells (MCF7) with stable exogenous MYC overexpression were used to assess the impact of sponge constructs on MYC regulation. Quantitative RT-PCR revealed a significant reduction in miRNA expression and a corresponding increase in endogenous MYC levels upon sponge treatment. Functional assays in multiple colorectal cancer cell lines with varying MYC copy numbers demonstrated a time-dependent increase in cell death following sponge transfection. Cytotoxic effects increased with MYC copy number, confirming a correlation between gene dosage sensitivity and therapeutic response. Conclusions: Our findings demonstrate that miRNA sponges targeting the miR-17/92 cluster can effectively disrupt MYC dosage compensation, leading to selective cytotoxicity in MYC-amplified cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd723751e515fe0f5023dc6afe839f1035b1fac" target='_blank'>
              A Synthetic Sponge System Against miRNAs of the miR-17/92 Cluster Targets Transcriptional MYC Dosage Compensation in Aneuploid Cancer
              </a>
            </td>
          <td>
            Diana M. Bravo-Estupiñan, Carsten Geiß, Jorge L. Arias-Arias, Mariela Montaño-Samaniego, Ricardo Chinchilla-Monge, Christian Marín-Müller, Steve Quirós-Barrantes, Anne Régnier-Vigouroux, Miguel Ibáñez-Hernández, R. Mora-Rodríguez
          </td>
          <td>2025-09-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="DNA carries genetic information, ensuring the stable transmission of genetic material through generations. However, DNA sequences can be constantly rewritten, allowing evolution and adaptation to occur at both the cellular and species levels. To facilitate this dynamic process, the genome is enriched with mobile genetic elements that can move within DNA sequences. While transposable elements are the classic example of mobile DNA, we propose that extrachromosomal circular DNA (ecDNA) represents another class of mobile genetic elements. This class can also introduce a new layer of genome dynamics, significantly shaping host biology. This review traces the historical discoveries and conceptual evolution of ecDNA, categorizing its diverse forms across organisms and highlighting its life cycle from biogenesis to maintenance and clearance. We discuss ecDNA's pivotal roles in physiological processes, including development, stress adaptation, and species evolution, and emphasize its pathological significance in cancer progression, drug resistance, and viral infections. Finally, we explore therapeutic strategies targeting ecDNA biology, offering a framework for translational advances in oncology and antiviral treatment. Together, these insights position ecDNA as a critical force in shaping genome dynamics and as a promising target for future biomedical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a2c075da6c2a7912686c8ef608ee9b31391b746" target='_blank'>
              Extrachromosomal Circular DNA: A Mobile Genetic Element Shaping Host Biology.
              </a>
            </td>
          <td>
            Shun Yao, Oliver W. Chung, Ling Wang, Z. Zhang
          </td>
          <td>2025-08-05</td>
          <td>Annual review of cell and developmental biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Microhomology-mediated tandem duplication (TD) is a distinct mutational signature in BRCA1-deficient tumors. While several mechanisms have been proposed for its generation, much understanding comes from repeat-based reporters, which may not represent the region of the human genome lacking neighboring repeats. To address this limitation, we developed a repeat-less TD reporter and a PCR-based endogenous site-specific TD assay to examine TDs induced by replication-coupled single-ended DNA double-strand breaks (seDSBs) in Brca1-deficient mouse embryonic stem cells. While TDs induced by seDSBs were detectable in wild-type cells, they were significantly elevated in Brca1-deficient cells, independent of classical non-homologous end joining. Some of these TDs arose from allelic DNA synthesis followed by a shift to microhomology-mediated non-allelic sister chromatid recombination, suggesting increased premature termination of allelic DNA synthesis, a late step of allelic break-induced replication (aBIR), in Brca1-deficient cells. Disrupting RAD51 loading reduced TDs in wild-type cells but not in Brca1-deficient cells, indicating both RAD51-dependent and -independent TD formation. Additionally, RAD54 and BRCA1–BARD1 suppressed TDs in a RAD51-dependent manner, with the effect of BARD1 partially mediated through RAD51 interaction. These findings together implicate late steps of aBIR in TD suppression and provide novel insights into the mechanisms underlying BRCA1-linked TD formation in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7edf834c059a1df782121d4a36a790a684d7b6d4" target='_blank'>
              Late steps of allelic break-induced replication suppress tandem duplication associated with BRCA1 deficiency
              </a>
            </td>
          <td>
            Zhi-cheng Huang, Yili Feng, Qian Liu, Ruo-Dan Chen, Si-Cheng Liu, Meng Wang, A. Xie
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="R-loop is a special DNA–RNA hybrid genomic structure. Since its identification, the molecular mechanisms of physiological processes such as class switch recombination have been revealed, uncontrolled regulation of which has become the underlying cause of many diseases. With the development of molecular biology techniques, R-loops found at different sites and formed via different mechanisms have been discovered. These non-classical R-loops participate in various cellular activities via different mechanisms. A set of sophisticated regulatory mechanisms are required to control the number of various R-loops in vivo. The levels of non-classical R-loops in tumor cells differ significantly from those in normal cells; in addition, the regulatory mechanisms for establishing R-loop homeostasis differ, which may be utilized to develop breakthrough therapies for various tumors. In this review, we summarize the current state of knowledge regarding non-classical R-loops in tumor cells, the mechanisms that promote or inhibit this structure, the effects of this structure on tumor cells, and the possible therapeutic targets. By systematically elucidating the pathogenic mechanisms of atypical R-loops, we can achieve precision targeting of tumors and revolutionize clinical precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/676d0195748900abca38448fb2ef5985171fe4bf" target='_blank'>
              Atypical R-loops in cancer: decoding molecular chaos for therapeutic gain
              </a>
            </td>
          <td>
            Yuan Sun, Sheng Wang, N. Ge, Jintao Guo, Guoxin Wang, Fan Yang, Siyu Sun
          </td>
          <td>2025-08-14</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The DNA-incorporating nucleoside analogs azacytidine (AZA) and decitabine (DEC) have clinical efficacy in blood cancers, yet the precise mechanism by which these agents kill cancer cells has remained unresolved – specifically, whether their anti-tumor activity arises from conventional DNA damage or DNA hypomethylation via DNA methyltransferase 1 (DNMT1) inhibition. This incomplete mechanistic understanding has limited their broader therapeutic application, particularly in solid tumors, where early clinical trials showed limited efficacy. Here, through the assessment of drug sensitivity in over 600 human cancer models and comparison to a non-DNA-damaging DNMT1 inhibitor (GSK-3685032), we establish DNA hypomethylation, rather than DNA damage, as the primary killing mechanism of AZA and DEC across diverse cancer types. In further support of an epigenetic killing mechanism, CRISPR drug modifier screens identified a core set of chromatin regulators, most notably the histone deubiquitinase USP48, as AZA and DEC protective factors. We show that USP48 is recruited to newly hypomethylated CpG islands and deubiquitinates non-canonical histones, establishing USP48 as a key molecular link between the two components of epigenetic gene regulation: DNA methylation and chromatin modification. Furthermore, loss of USP48, which occurs naturally through biallelic deletions in human cancers, sensitized both hematologic and solid tumors to DNMT1 inhibition in vitro and in vivo. Our findings elucidate the epigenetic mechanism of action of AZA and DEC and identify a homeostatic link between DNA methylation and chromatin state, revealing new therapeutic opportunities for DNMT1 inhibitors in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4849479727b73a833dd0ff467a2ebd5a79ba5e8" target='_blank'>
              Discovery of chromatin-based determinants of azacytidine and decitabine anti-cancer activity
              </a>
            </td>
          <td>
            Rishi V. Puram, Qiangzong Yin, YuhJong Liu, Justine C. Rutter, Daniel Bondeson, Maria C. Saberi, Lisa Miller, Michael Du, Khanh Nguyen, Donovan L. Batzli, Hilina B. Woldemichael, Christian Taeger, Anna Goldstein, Ming Y. Chu, Qi Guo, D. Mani, Michael Naumann, M. Ronan, M. Rees, Blanche C. Ip, Mustafa Kocak, Mikołaj Słabicki, John G Doench, Jennifer A. Roth, Steven A. Carr, N. Udeshi, Jingyi Wu, Todd R. Golub
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Abstract DNA–protein crosslinks (DPCs) form following exposure to various alkylating agents, including environmental carcinogens, cancer chemotherapeutics, and reactive aldehydes. If not repaired, DPCs can interfere with key biological processes such as transcription and replication and activate programmed cell death. A growing body of evidence implicates nucleotide excision repair (NER), homologous recombination, and other mechanisms in the removal of DPCs. However, the effects of genomic context on DPC formation and removal have not been comprehensively addressed. Using a combination of next-generation sequencing and DPC enrichment via protein precipitation, we show that DPCs induced following exposure to formaldehyde are non-randomly distributed across the human genome, based on chromatin state. The data further show that the efficiency of DPC removal correlates with transcription at loci transcribed by RNA polymerase II. Data presented herein indicate that efficient removal of chromosomal DPCs requires both the Cockayne syndrome group B gene as well as “downstream” TC-NER factor xeroderma pigmentosum group A gene. In contrast, loci transcribed by RNA polymerase I showed no evidence of transcription-coupled DPC removal. Taken together, our results indicate that complex interactions between chromatin organization, transcriptional activity, and numerous DNA repair pathways dictate genomic patterns of DPC formation and removal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/138482f47c7713ca6d5e082edda14419c0390855" target='_blank'>
              Genome-wide mapping of formaldehyde-induced DNA–protein crosslinks reveals unique patterns of formation and transcription-coupled removal in mammalian cells
              </a>
            </td>
          <td>
            Duha Alshareef, Charlie T Nguyen, Kayla N Tucker, Micah D. Gearhart, N. Tretyakova, Colin Campbell
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Comprehensive epigenomic studies in multiple myeloma (MM) that unravel the connections between major epigenetic regulators, their intertwined collaboration and the potential of combinatorial targeting remain limited. Utilizing ChIP-seq, ATAC-seq, RNA-seq, and DNA methylation (DNAme) data, we generated whole-genome chromatin annotations from normal plasma cells and MM patients, revealing epigenomic re-configuration affecting downstream genes involved in tumour growth and survival. Primary MM samples showed global DNA hypomethylation but site-specific hypermethylation was observed at transcription start sites, promoters, and enhancers. Moreover, increased deposition of H3K27me3 was observed in clinically relevant functional chromatin clusters. Combined EZH2 and DNMTs inhibition resulted in extensive epigenomic alterations activating apoptosis and cell cycle genes, leading to increased G2/M arrest and apoptosis in MM cell lines. Our findings provide novel insights into the role of epigenetic gene silencing in MM tumorigenesis and the interplay between the Polycomb repressive complex 2 and DNAme. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17093-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1dfe4ebbad2d35b5cf5e512e8911342a3522d08" target='_blank'>
              Combinatorial DNMTs and EZH2 inhibition reprograms the H3K27me3 and DNAme-mediated onco-epigenome to suppress multiple myeloma proliferation
              </a>
            </td>
          <td>
            Alba Atienza Párraga, P. Nylund, K. Diamanti, Berta Garrido-Zabala, Stefania Iliana Tziola, Louella Vasquez, P. Pyl, D. Raykova, Aron T. Skaftason, Anqi Ma, Jian Jin, J. Martín-Subero, Fredrik Öberg, Elke De Bruyne, Jan Komorowski, Helena Jernberg Wiklund, A. Kalushkova
          </td>
          <td>2025-08-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="While mitogenic signaling is known to regulate cell-cycle entry during the G1 phase, its function in the G2 phase remains elusive. Here we show that mitogenic signaling controls whether G2-arrested cells proceed through mitosis or undergo whole-genome duplication. Although mitogenic signaling is not required for the G2/M transition under normal conditions, it modulates E2F transcriptional activity via c-Myc. When G2 arrest occurs due to CDK4/6 and CDK2 suppression, E2F activity levels determine the status of APC/C inactivation and the CDK2-Rb feedback loop. Upon release from G2 arrest, cells maintaining APC/C inactivation promptly induce CDK2 activation and FoxM1 phosphorylation, driving mitotic entry. Conversely, APC/C reactivation degrades cyclin A and abolishes the CDK2-Rb loop, necessitating CDK4/6 activation for cell-cycle re-entry. This regulatory mechanism mirrors the G1-phase process, resulting in whole-genome duplication. In cancer cells, this process promotes genome instability and oncogene amplification, contributing to aggressive behavior. These findings reveal a previously unrecognized mitogen-dependent checkpoint that governs cell fate in the G2 phase.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8c796e5108bc1d588171ecee8be6809857c2d14" target='_blank'>
              E2F activity determines mitosis versus whole-genome duplication in G2-arrested cells
              </a>
            </td>
          <td>
            Kibum Kim, Jessica Armand, Sungsoo Kim, Hee Won Yang
          </td>
          <td>2025-07-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Oncomicrobes are estimated to cause 15% of cancers worldwide. When cancer whole-genome sequencing (WGS) data are collected, the microbes present are also sequenced, allowing the investigation of potential etiological and clinical associations. Interrogating the microbial community for 8908 patients encompassing 22 cancer types from the Genomics England WGS dataset revealed that only colorectal tumors exhibited unmistakably distinct microbial communities that can reliably be used to distinguish anatomical site [positive predictive value (PPV) = 0.95]. This pattern was validated in two independent datasets. Potential clinical relevance uncovered by our analyses included accurate detection of alphapapillomaviruses [human papillomavirus (HPV)] in oral cancers, when compared with current clinical standards, and the detection of rare, highly pathogenic viruses such as human T-lymphotropic virus-1. Biomarker investigations demonstrated statistically significant associations (P < 0.05) between a subset of anaerobic bacteria and survival in certain subtypes of sarcoma. Our results contradict previous claims that each cancer type has a distinct microbiological signature but highlight the potential value of microbial analysis for certain cancers as WGS of tumor samples becomes common in the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82e28dcd99723ccc30fe571b2d59df15e8ee150f" target='_blank'>
              The landscape of microbial associations in human cancer.
              </a>
            </td>
          <td>
            Abraham Gihawi, H.M. Wood, Jeremy Clark, Justin O'Grady, Rosalind A. Eeles, David C Wedge, G. M. Jakobsdottir, G. Magiorkinis, A. Schache, Liam Masterson, Matt Lechner, Tim R. Fenton, Terence M Jones, Adrienne M Flanagan, Solange De Noon, Alex Rubinsteyn, Rachel Hurst, Colin S Cooper, Daniel S. Brewer
          </td>
          <td>2025-09-03</td>
          <td>Science translational medicine</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="T cell receptor (TCR) diversity, essential for the recognition of a wide array of antigens, is generated through V(D)J recombination. The Tcra and Tcrd genes reside within a shared genomic locus, with Tcrd rearrangement occurring first in the double-negative (DN) stage during thymocyte development. Elucidating the regulatory mechanisms governing Tcrd rearrangement is therefore crucial for understanding the developmental coordination of both Tcrd and Tcra rearrangements. Chromatin architecture, orchestrated by CTCF-cohesin complexes and their binding sites, plays a fundamental role in regulating V(D)J recombination of antigen receptor genes. In this study, we report that EACBE, a CTCF binding element (CBE) located downstream of the Tcra-Tcrd locus, regulates Tcrd rearrangement. EACBE promotes the usage of proximal Vδ gene segments by facilitating spatial proximity between the Tcrd recombination centre and these Vδ elements. Notably, EACBE counteracts the insulating effects of INTs, two CBEs that demarcate the proximal V region from the Dδ-Jδ-Cδ cluster, thereby enabling effective chromatin extrusion. Furthermore, EACBE indirectly shapes the Tcra repertoire through its influence on Tcrd rearrangement. These findings reveal a novel regulatory axis involving special chromatin configuration and highlight distinct roles for specific CTCF binding sites in modulating antigen receptor gene assembly.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39969305cd184c6ac930cc7afb8573a26cbe9e2b" target='_blank'>
              The insulator EACBE regulates V(D)J recombination of Tcrd gene by modulating chromatin organization
              </a>
            </td>
          <td>
            Yongchang Zhu, Ranran Dai, Hao Zhao, Junwei Luo, Keyi Li, Wei Xue, Litao Qin, Hongyuan Pan, Shixiu Liao, Bingtao Hao
          </td>
          <td>2025-07-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fa29e8cd8f6fdd768b529f92b9275ea04ce76dc" target='_blank'>
              Genetic landscape of cancer: mechanisms, key genes, and therapeutic implications.
              </a>
            </td>
          <td>
            Arun Karnwal, Joydeep Dutta, Aqueel-Ur-Rehman, Abdel Rahman Mohammad Said Al-Tawaha, Natalia Nesterova
          </td>
          <td>2025-08-17</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>1</td>
          <td>10</td>
        </tr>

        <tr id="Chromatin, the spatial organizer of genomic DNA, is hierarchically folded into higher-order structures to facilitate DNA compaction, enabling genome surveillance. Understanding the organization and function of the three-dimensional (3D) genome is critical to profile chromatin accessibility and functional interactions that govern gene regulation across multiple biological processes, including aging and one of its hallmarks, cellular senescence. Cellular senescence constitutes a defensive stress response to various intrinsic and extrinsic stimuli, preserving cellular and organismal homeostasis through a generally irreversible cell cycle arrest. In this review article we discuss epigenetic alterations occurring to DNA and chromatin that drive and fuel the onset of this complex phenomenon. As such, we describe major large-scale chromatin events, including the formation of higher-order chromatin structures and the 3D spatial alterations of the genome that occur during senescence. We also discuss global heterochromatin loss, deficiencies in nuclear lamins, the depletion of core histones and their modifications, as well as the epigenetic regulation of the senescence-associated secretory phenotype (SASP), all of which serve key roles in the epigenome of senescent cells. To clearly demonstrate the significance of epigenetic modifications, data from a computational meta-analysis are presented, aiming to further underpin key epigenetic mechanisms occurring in senescent cells. Last, we highlight promising epigenetic modulators implemented in therapeutic strategies for senescent cell detection and elimination, possibly leading to significant clinical advances against various age-related diseases as well as the delay and prevention of the aging onset.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fd36c31f599c330b53d9bff8eb9f2a73d497006" target='_blank'>
              Interrogating the regulatory epigenome of cellular senescence
              </a>
            </td>
          <td>
            Dimitris-Foivos Thanos, Orestis A. Ntintas, Emmanouil Athanasiadis, A. Papaspyropoulos, R. Petty, V. Gorgoulis
          </td>
          <td>2025-08-31</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Abstract The murine endogenous retrovirus MERVL is dynamically activated in a small population of in vitro cultured mouse embryonic stem cells (mESCs) exhibiting totipotent-like features. Yet, the relationship between MERVL activation and cell fate decisions of mESCs is incompletely understood. Through a genome-wide knockout screen, we discovered that MERVL activity is intrinsically linked to DNA damage response pathways. Loss of Ints7, a backbone subunit of the Integrator complex, increased DNA damage and triggered MERVL expression. Mechanistically, Ints7 depletion induced phosphorylation of Kap1, increased chromatin accessibility at MERVL loci, and activated the p53–Dux axis to drive MERVL transcription. Intriguingly, DNA damage-induced MERVL resurgence followed the cleavage of caspase-3, often accompanying a process known as anastasis—cell survival after transient apoptotic signaling. Collectively, our study uncovered that MERVL activation in mESCs is integrated into the cellular circuit for decision-making in response to DNA damage, suggesting that sublethal caspase activation can influence the developmental potential of stem cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9647f46def2d9561823487f197db2f3a1327b93a" target='_blank'>
              Ints7 deficiency activates DNA damage response to elicit resurgence of endogenous retrovirus MERVL and anastasis of embryonic stem cells
              </a>
            </td>
          <td>
            Yunfan Shen, Li Li, Huiling Ni, Hui Li, Mingrui Xu, Xiaoyang Tan, Zhangjie Li, Pishun Li, Fang Chen, Song Mao, Gongping Sun, Kai Yuan
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract DNA double-strand breaks (DSBs) represent critical events in genome integrity, arising from both endogenous cellular processes and exogenous factors. These breaks are implicated in various genomic aberrations and chromosomal rearrangements, leading to cancers and genetic disorders. Common and rare fragile sites, containing repetitive elements and non-B DNA structures, are particularly prone to breakage under replication stress, which play a pivotal role in cancer development and genetic diseases. Accurate quantification of DNA breaks in the context of repetitive sequences such as microsatellites or non-B DNA structures is technically challenging. We have been comparing four different methods to reliably quantify DSBs in repetitive DNA, namely Southern blot, DSB-PCR, real-time DSB-qPCR, and digital PCR (dPCR). We show here that dPCR offers enhanced sensitivity and specificity compared to other methods. This provides significant applications for future disease diagnosis, understanding molecular mechanisms generating chromosomal breakage and for the development of gene therapies for microsatellite expansion disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccab30f2a498f797c04595f2b9078ba3ca2f3603" target='_blank'>
              Fast and accurate quantification of double-strand breaks in microsatellites by digital PCR
              </a>
            </td>
          <td>
            Cécile Palao, Adèle Kovacs, M. Teixeira, G. Richard
          </td>
          <td>2025-08-09</td>
          <td>Biology Methods & Protocols</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Simple Summary This study examined unclassified chromosomal abnormalities (UnCA) in patients with Hodgkin’s lymphoma (HL) before, during, and after anticancer treatment. HL is a rare cancer affecting 2–4 people per 100,000 individuals annually. Although treatment is highly effective, survivors face an increased risk of developing other cancers. In this study, we analyzed blood samples from HL patients and survivors at different stages of treatment, focusing on changes in their chromosomes that do not fit standard categories. These changes include alterations in chromosomes and nuclei. We found that UnCA levels were higher in HL patients than in healthy individuals at all stages; the general UnCA frequency decreased during treatment but increased again one year after treatment, at the expense of two specific UnCAs, free chromatin and micronuclei clusters. Given their persistence and association with genomic instability, these UnCAs might serve as early indicators of secondary cancers in HL survivors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb5a19d761a4166c2c377dcfbbe9dbb9f00192f1" target='_blank'>
              Unclassified Chromosomal Abnormalities as an Indicator of Genomic Damage in Survivors of Hodgkin’s Lymphoma
              </a>
            </td>
          <td>
            S. Ramos, B. Molina, María Del Pilar Navarrete-Meneses, D. Cervantes-Barragan, Valentín Lozano, S. Frias
          </td>
          <td>2025-07-23</td>
          <td>Cancers</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Small round cell sarcomas (SRCS) are highly aggressive tumors in soft tissues and bone of mostly children and young adults. Despite being different in many aspects, including genetics, possible cell-of-origin, and pathology, patients with any of these entities all receive the same therapeutic regimen. Although several pre-clinical models of Ewing sarcoma have been established, such as cell lines and patient-derived tumor xenografts, few models exist for other SRCS. Here, we describe a pediatric SRCS tumor organoid (tumoroid) biobank containing long-term tumoroid cultures with different translocations, including EWSR1::FLI1, EWSR1::ERG, CIC::DUX4, and BCOR-rearrangements. Using histology, whole genome sequencing and RNA sequencing, we demonstrate that these tumoroids retain histological characteristics, known marker gene expression and chromosomal rearangements of their matching patient tumors. In addition, we compare mutation clusters in the tumoroids across patient-matched longitudinal samples, which shows that cellular heterogeneity is maintained. Drug screening on the tumoroid models unveils entity-specific drug sensitivity to various cytotoxic compounds and targeted compounds, including MCL-1 inhibitors for CIC::DUX4 sarcomas. Taken together, this newly established SRCS patient-derived tumoroid biobank represents a promising source of material for future basic cancer research and drug screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d54a3ce373321b31d8cc83feaca65eb77c05c889" target='_blank'>
              Small round cell sarcoma tumoroid biobank reveals CIC::DUX4 sarcoma vulnerability to MCL-1 inhibition
              </a>
            </td>
          <td>
            F. Ringnalda, Gijs J. F. van Son, Laurens H. G. Verweij, Seok-Young Kim, Vicky Amo-Addae, U. E. Flucke, L. Hiemcke-Jiwa, Karin P S Langenberg, Jos A. M. Bramer, L. Heimans, M. A. van de Sande, W. V. van Houdt, Max M. van Noesel, H. Kerstens, Marcel Santoso, Georg Seifert, O. Delattre, Katia Scotlandi, B. Geoerger, J. H. Merks, J. Molenaar, R. van Boxtel, Marc van de Wetering, K. Sanders, H. Clevers
          </td>
          <td>2025-08-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background/Aim Cell lines serve as valuable in vitro models to study altered cellular signaling pathways, to identify mutations in key oncogenic genes, and to test potential antitumor drugs. The Jurkat cell line, for example, has provided important information about various signaling pathways in lymphoblastic leukemia, establishing most of what is currently known about T-cell receptor (TCR) signaling. However, many aspects of the genome modification of this cell line have not yet been analyzed. To identify genes of potential biological and clinical relevance in acute T-lymphoblastic leukemia (T-ALL), we performed an array comparative genomic hybridization (aCGH) approach on the widely used Jurkat cell line and examined the association of the detected copy number alterations (CNAs) with cancer hallmarks and T-ALL pathogenesis. Materials and Methods Cells were harvested by using trypsin/EDTA from culture flasks to extract genomic DNA. aCGH experiments were performed on an Agilent microarray platform using the SurePrint G3 Cancer CGH + SNP Microarray 4×180 K. Functional enrichment analysis of all CNAs was performed with the R package g:Profiler2. The association of these alterations with key cancer hallmarks was analyzed using the Cancer Hallmarks web-tool. Results Our analysis revealed several novel CNAs, including losses at 5p15.2, 6q27, 10q22.2, 14q11.2, 18q11.2-q12.1, and Xp22.33, as well as gains at 2p11.2, 7p21.2, 7q21.2 and 18p11.32. Genes within these regions were associated with important oncogenic pathways, including ‘sustained proliferative signaling’, ‘tumor suppressor evasion’, and ‘angiogenesis promotion’. Conclusion These findings suggest that Jurkat cells may serve as a valuable model for identifying new targets for cancer research. Further studies are required to confirm the phenotypic implications of these variants, which may open new avenues for exploring the functional impact of these alterations and their potential role in the development of therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7da88c0ba8ab646537d572f0801ee5d18aa14a5e" target='_blank'>
              High-Resolution aCGH Analysis of the Jurkat Cell Line: Copy Number Alterations and Their Association With Cancer Hallmarks
              </a>
            </td>
          <td>
            Layssa Mota DA COSTA, Wallax Augusto SILVA FERREIRA, C. B. Machado, Carolina Koury NASSAR AMORIM, Ana Virgínia VAN DEN BERG, M. O. de Moraes Filho, M. E. D. de Moraes, Lucas Eduardo Botelho de Souza, A. Khayat, E. H. C. de Oliveira, C. A. Moreira-Nunes
          </td>
          <td>2025-08-29</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cee7ec76a900e6d0548a03ab74a2803cf82d9035" target='_blank'>
              Genomic Divergence Between Matched Primary and Metastatic Tumors Across Cancer Types: A Pan-Cancer Analysis of 5,692 Samples
              </a>
            </td>
          <td>
            Y. Ergun
          </td>
          <td>2025-08-26</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Oncogenic p53 mutations (Onc-p53) are frequent in lung and many other solid tumors often associated with chromosome aberrations. Why cells with Onc-p53 develop chromosomal aberrations and whether the abnormalities contribute to tumor growth remain elusive. Evidence in this communication demonstrate for the first time that replication stress induced by Onc-p53 triggers re-copying of DNA replication forks, which generates replication intermediates that cause persistent mitotic aberration and DNA segregation errors. Replication intermediates from re-copied replication forks induced by Onc-p53 activate ATM signaling, which stabilizes Onc-p53, reinforces its ability to upregulate replication factors for sustaining replication stress, thus generating a feedforward cycle accelerating tumor formation. In agreement with this observation our time lapse video microscopy show in real time that persistent mitotic aberration and DNA segregation errors induced by Onc-p53 confer selective growth advantage. Accordingly, human lung tumors with Onc-p53 show selection of cells with mitotic aberration during serial passages. Knock down of active replication forks reduces re-copied fork generation by Onc-p53 and specifically induces apoptotic death of lung cancer cells expressing Onc-p53 in xenograft lung tumors synergistically in cooperation with inhibitors of ATM activation, deselecting cells with Onc-p53 with mitotic errors. This communication reveals a novel mechanism which interconnects replication stress induced by Onc-p53 to its stabilization and ability to generate chromosomal aberration in lung cancer cells that both accelerate tumor growth and serve as a targetable therapeutic vulnerability. These findings will be extremely valuable for tumor-specific treatment of a high percentage of cancer patients with p53 mutation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b77d562f308b7a32a1524622e9f1f8039e4924f9" target='_blank'>
              Oncogenic p53 induces mitotic errors in lung cancer cells by recopying DNA replication forks conferring targetable proliferation advantage
              </a>
            </td>
          <td>
            S. Deb, Shilpa Singh, Lilia Gheghiani, R. Frum, Steven Grossman, Brad Windle, S. Deb
          </td>
          <td>2025-08-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Cancerous transcriptome alterations in carcinoma cells could be originated from either genetic copy number changes or epigenetic reprogramming. Ovarian cancer (OV) is the most malignant gynecologic tumor, known for high aneuploidy with robust copy number alterations. However, low aneuploidy ovarian tumors are also frequently found, indicating an essential contribution of epigenetic factors during tumorigenesis and cancer development. Chromatin remodeling modulates the transcriptome epigenetically in a variety of cancer types, but its role in OV is still unclear. Hence, we investigated a cohort of 102 OV patients, analyzed transcriptomic and clinical data from public databases, and performed cellular experiments. We found that RUVBL2, a subunit of the INO80 complex, functions as the key oncogenic chromatin remodeler in OV. RUVBL2 is upregulated in tumors, particularly in low-aneuploidy cases, and is associated with poor prognosis. RUVBL2 drives nucleosome dynamics and elevates chromatin accessibility selectively at promoter regions. The landscape of RUVBL2-dependent modulation of chromatin accessibility and the transcriptome exhibits activation of various transcription factors, especially the AP-1 family, and upregulation of a series of key genes, including CDKN3, MYBL2, and ZNF144, resulting in mediation of cell cycle and Hippo signaling pathway to promote DNA synthesis and cell proliferation. Hence, RUVBL2-dependent chromatin remodeling plays a key role in oncogenic reprogramming of the transcriptome in OV. These findings provide novel insights into the molecular etiology of OV and disclose potential biomarkers and drug targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2407b3becd0c884fdbbc4e31bad36bc2c5c367" target='_blank'>
              Profiling of RUVBL2-Induced Transcriptome Alterations Highlights a Critical Role for Chromatin Remodeling in Ovarian Cancer.
              </a>
            </td>
          <td>
            Renhao Xue, Yingjie Wang, Xiaomei Luo, Hao Zhang, Dongcheng Guan, Shuo Shi, Yu Wang
          </td>
          <td>2025-07-01</td>
          <td>BioFactors</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Merkel cell carcinoma (MCC) is an aggressive disease with poor survival outcomes and increasing incidence. There is a clear and present need for enhanced understanding of cellular mechanisms of tumorigenesis, validation of robust genetic signatures predictive of aggressive disease, and novel informatics tools to simplify analysis of Merkel cell polyomavirus (MCPyV)-host genome interactions. Genomic DNA was harvested from 54 MCC tumors for exome sequencing and in-depth genetic profiling of a 226-gene panel. We further developed a robust informatics package (MCPyViewer) optimized for MCPyV integration site analysis with graphical output to simplify usability for end-users. Finally, we assessed the prognostic impact of specific genetic signatures on MCC-specific survival in our cohort. Our study included 54 patients (n = 44 MCPyV-positive), 11 (20.4 %) of which had died of MCC at last follow-up. Human genes altered at high frequency included LRP1B (n = 10, 18.5 %), FAT1 (n = 9, 16.7 %), KMT2D (n = 9, 16.7 %), and RB1 (n = 7, 13.0 %). In 36 of 44 (81.8 %) MCPyV-positive tumors, we identified viral integration into the human genome with a median of two events per tumor. In six tumors, MCPyV integrated into COSMIC Tier 1 or Tier 2 cancer-related human genes. Implications: A combined genomics score incorporating tumor mutational burden and copy number variation was strongly prognostic of MCC-specific survival controlling for lymph node metastases and tumor MCPyV status, thus, our study adds critical understanding to prognostic markers and tumorigenic mechanisms in MCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ef7b14dad4673e2d7fa9c8cccb49e7531dcb5de" target='_blank'>
              Genomic Signatures of Poor Prognosis in Merkel Cell Carcinoma: A Single-Institution Prospective Study.
              </a>
            </td>
          <td>
            Joshua D Smith, A. Bhangale, Wenjin Gu, C. Brummel, E. Gensterblum-Miller, Scott McLean, Paul W. Harms, K. Harms, C. Bichakjian, M. Spector, R. E. Mills, J. C. Brenner
          </td>
          <td>2025-07-21</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="CNS embryonal tumors with PLAGL amplification (ET, PLAGL) are a recently described tumor type marked by amplification of one of the PLAG family genes, PLAGL1 or PLAGL2. Separately, a supratentorial, ependymoma-like CNS tumor type with PLAG family alteration, namely PLAGL1 fusion, was also reported (NET_PLAGL1). Here, we use DNA methylation profiling in combination with copy number, RNA-seq, and histological analysis to characterize and classify a novel group of CNS embryonal tumors harboring PLAG1 gene fusions (n=12). Through our screening, we identified a subset of CNS tumors (n=12) epigenetically distinct from other known CNS tumor types, but clustering close to the PLAGL1- and PLAGL2-amplified ET, PLAGL subtypes in our t-SNE analysis. Copy number profiles indicated putative PLAG1 fusions, which were confirmed in 9/12 tumors (not determined in 3/12). Different 5’ fusion partners (ASAP1, ADGRG1, TMEM68, TCF4, CHD7, NCALD, HNRNPK, LOC105378102) were identified that upregulate wild-type PLAG1 through promoter hijacking. Expression analysis shows upregulation of PLAG1 as well as IGF2, DLK1, Desmin, CYP2W1, and RET, which are also robustly expressed in PLAGL1/2-amplified tumors. Patient characteristics, survival data, and clinical/imaging analysis show additional similarities to PLAGL1/2-amplified tumors. Median age at diagnosis was 5 years, tumors were located throughout the neuroaxis, and original histological diagnoses were heterogeneous. The tumors demonstrated morphologic heterogeneity, with most composed of densely cellular areas of primitive small blue cells, alongside focal regions showing clear cell morphology, microcystic changes, and ependymoma-like perivascular pseudorosettes. Applied treatment regimens were also heterogeneous, but some favorable responses to therapy were observed. In summary, we describe a third subtype of PLAG family-altered pediatric CNS embryonal tumor characterized by PLAG1 gene fusion, which leads to upregulation of PLAG1 and downstream genes. We therefore propose to rename ET, PLAGL to ET, PLAG (CNS embryonal tumor with PLAG family gene alteration) together with a specification of the respective subtype. Supplementary Information The online version contains supplementary material available at 10.1007/s00401-025-02917-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e48e98c986a7d6a88f7b4564b94e380215cf7f5b" target='_blank'>
              PLAG1 fusions define a third subtype of CNS embryonal tumor with PLAG family gene alteration
              </a>
            </td>
          <td>
            Michaela-Kristina Keck, M. Al-Hussaini, N. Amayiri, Akosua Adoma Boakye Yiadom, Gabriel Chamyan, E. Cheesman, Cécile Faure-Conter, Miguel Garcia-Ariza, Guillaume Gauchotte, M. Hasselblatt, Mette Jorgensen, J. Kilday, Gabriela Lamas, C. Lavarino, Marilyn M Li, Fabiana Lubieniecki, O. Maher, D. Meyronet, Jan Mueller, Mariarita Santi, U. Schüller, A. L. Seidinger, M. Sill, S. Sudhakar, María Tallón García, A. Tauziède‐Espariat, Pascale Varlet, A. Vasiljevic, A. Wittmann, Andreas von Deimling, D. A. Solomon, F. Sahm, A. Tietze, K. von Hoff, P. Sievers, David T W Jones
          </td>
          <td>2025-08-02</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Cancer genomes frequently carry APOBEC (apolipoprotein B mRNA editing catalytic polypeptide-like)-associated DNA mutations, suggesting APOBEC enzymes as innate mutagens during cancer initiation and evolution. However, the pure mutagenic impacts of the specific enzymes among this family remain unclear in human normal cell lineages. Here, we investigated the comparative mutagenic activities of APOBEC3A and APOBEC3B, through whole-genome sequencing of human normal gastric organoid lines carrying doxycycline-inducible APOBEC expression cassettes. Our findings demonstrated that transcriptional upregulation of APOBEC3A led to the acquisition of a massive number of genomic mutations in just a few cell cycles. By contrast, despite clear deaminase activity and DNA damage, APOBEC3B upregulation did not generate a significant increase in mutations in the gastric epithelium. APOBEC3B-associated mutagenesis remained minimal even in the context of TP53 inactivation. Further analysis of the mutational landscape following APOBEC3A upregulation revealed a detailed spectrum of APOBEC3A-associated mutations, including indels, primarily 1 bp deletions, clustered mutations, and evidence of selective pressures acting on cells carrying the mutations. Our observations provide a clear foundation for understanding the mutational impact of APOBEC enzymes in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ca9c435f5c5a8910902a03b815334d801814d13" target='_blank'>
              APOBEC3A drives deaminase mutagenesis in human gastric epithelium.
              </a>
            </td>
          <td>
            Yohan An, Ji-Hyun Lee, Joonoh Lim, J. Youk, Seongyeol Park, Ji-Hyung Park, Kijong Yi, Taewoo Kim, Chang Hyun Nam, Won Hee Lee, Soo A Oh, Yoo Jin Bae, Thomas M. Klompstra, Haeun Lee, Jinju Han, Junehwak Lee, Jung Woo Park, Jie-Hyun Kim, Hyunki Kim, Hugo Snippert, B. Koo, Young Seok Ju
          </td>
          <td>2025-08-26</td>
          <td>Genome research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) plays a significant role in cancer development. However, our understanding of its role in normal cells is limited. This study aimed to explore the roles of eccDNA in the differentiation of human bone marrow mesenchymal stem cells (BMSCs). Methods Through circular DNA sequencing (Circle-seq) and computational analysis, we documented eccDNAs in human BMSCs and their differentiation into osteoblasts, adipocytes, and chondrocytes. Next, the assay for transposase-accessible chromatin with high throughput sequencing (ATAC-seq) and RNA sequencing (RNA-seq) data were integrated with Circle-seq data. The roles of phosphatidylinositol 4-kinase alpha (PI4KA) and eccDNA as functional enhancers in human BMSC differentiation were assessed in vitro. Results Our results demonstrate that eccDNA is common in human BMSCs. In the differentiated groups including osteoblasts, adipocytes and chondrocytes, eccDNA-encoded genes presented higher expression compared to the background. We found eccDNA displayed enhanced chromatin accessibility in human BMSCs, which contribute to increased transcription of genes, such as runt-related transcription factor 2 (RUNX2), a key transcription factor for BMSC osteogenesis. We further found PI4KA, an eccDNA-encoded gene, promoted osteogenic differentiation of human BMSCs via the PI3K/AKT/mTOR pathway. Finally, we demonstrated eccDNA as functional enhancers to regulate BMSC differentiation in a dose-dependent manner. Conclusions Our study provides the evidence of eccDNA regulating BMSC differentiation and highlights the roles of eccDNA as transcription template and enhancer in normal cells, which will facilitate future research and clinical applications. Supplementary Information The online version contains supplementary material available at 10.1186/s13287-025-04516-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9298ef8d097b3fec00680e4438db26b85d21b6" target='_blank'>
              Extrachromosomal circular DNAs in the differentiation of human bone marrow mesenchymal stem cells
              </a>
            </td>
          <td>
            Yuxi Gu, Yidan Song, Shuhua Wang, Jun Liu
          </td>
          <td>2025-07-18</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Repetitive DNA sequences are abundant in genomes and can adopt alternative DNA structures (i.e., non-B DNA). One such structure, Z-DNA, has been shown to stimulate genetic instability in a variety of organisms, including human cells and mice. Z-DNA-forming sequences are enriched at mutation hotspots in human cancer genomes, implicating them in cancer etiology. Aging is a known risk factor for the development of cancer, and genetic instability is a hallmark of both aging and cancer. However, how aging affects the mutagenic potential of Z-DNA has not yet been investigated. Here, we explored the effects of aging on the mutagenic processing of Z-DNA using a transgenic mouse model. Surprisingly, Z-DNA-induced mutations decreased or remained unchanged with increasing age. Cleavage of Z-DNA was unaffected with increasing age, suggesting that downstream repair processing, such as double-strand break repair processes, may be involved in the age-related changes in Z-DNA-induced mutagenesis in mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d857eba11e4fc492ce4ba5d13294865a2445fdd6" target='_blank'>
              Effects of Aging on Z-DNA-Induced Genetic Instability In Vivo
              </a>
            </td>
          <td>
            Tonia T. Li, Alexandra M D'Amico, Laura A. Christensen, Karen M. Vasquez
          </td>
          <td>2025-08-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 Somatic mutations are desirable targets for selective cancer elimination, yet most are found within noncoding regions. We have adapted CRISPR-Cas9 as a novel, cancer-specific killing strategy by targeting the subset of somatic mutations that create protospacer adjacent motifs (PAMs), which have evolutionally allowed bacterial cells to distinguish between self and non-self DNA for Cas9-induced double strand breaks (DSBs). We hypothesized that somatic 5’-NGG-3’ PAMs arising from base substitutions (e.g. NCG→NGG) could serve as CRISPR-Cas9 targets for selective cancer elimination through the induction of multiple DSBs. Whole genome sequencing (WGS) of paired tumor-normal (T-N) samples from three PC patients were put through PAMfinder (software for somatic PAM discovery). On average, 417 somatic PAMs per PC case were found (>99% of them in non-exons) and 97% of the selected subset were Sanger confirmed. We then analysed T-N samples from ICGC which revealed medians of 455 somatic PAMs per tumor in two PC cohorts (N=174 combined), 2790 in lung (N=29), and 3236 in esophageal (N=388) cancer cohorts. We performed γH2AX staining on PC cell lines transduced with 4-9 sgRNAs designed from our PAM discovery approach, and observed more foci with increased number of sgRNAs in the target cell line but not in the non-target cell line. Deep sequencing and WGS on clones transduced with cancer-specific sgRNAs also demonstrated on-target activity in the target cell line but not in either the patient’s normal cells or an irrelevant cancer cell line. No off-target events were detected. To test selective toxicity in vitro and in vivo, we transduced 4-9 sgRNAs into PC co-cultures and found 69-99% selective cell death of three target cell lines through flow cytometry. Finally, using mouse xenograft models, 92% weight reduction (N=10, P=0.0004) of tumors transduced with a pool of 9 cancer-specific sgRNAs was achieved compared to tumors transduced with non-targeting sgRNAs (N=8).



 Selina Shiqing K. Teh, Kirsten Bowland, Alexis Bennett, Eitan Halper-Stromberg, Akhil Kotwal, Hong Liang, Sarah Wheelan, Chien-Fu Hung, Michael Goldstein, Nicholas J. Roberts, and James R. Eshleman. CRISPR-Cas9 as a Novel Cancer Gene Therapy Approach against Pancreatic Cancers [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr LT02.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b50e15f1763d831356c9974d95d4708f8806a3e" target='_blank'>
              Abstract LT02: CRISPR-Cas9 as a Novel Cancer Gene Therapy Approach against Pancreatic Cancers
              </a>
            </td>
          <td>
            S. Teh, Kirsten Bowland, Alexis Bennett, Eitan Halper-Stromberg, Akhil Kotwal, Hong Liang, Sarah Wheelan, Chien-Fu Hung, M. Goldstein, Nicholas J. Roberts, James R. Eshleman
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Ovarian high-grade serous carcinoma (HGSC) is characterized by pervasive genomic instability and high inter- and intra-tumor heterogeneity. Approximately half of HGSC tumors harbor homologous recombination deficiency (HRD), rendering them vulnerable to PARP inhibitors and platinum-based chemotherapy. In contrast, patients lacking HRD (HR-proficient, HRP) generally respond poorly to the current therapies. To overcome heterogeneity and identify relevant HGSC subtypes, we characterized the genomic landscape of 640 tumors from 243 patients using whole-genome sequencing. Our chromosomal instability (CIN) signature-based analysis characterized the structural variation landscape and revealed five HGSC subtypes, validated in an independent dataset. Two HRD subtypes, associated with BRCA1- or BRCA2-driven alterations, demonstrated favorable treatment responses. Strikingly, three HRP subtypes emerged, marked by unique structural alterations and gene expression patterns, tumor microenvironment interactions, and different chemotherapy responses. Finally, organoid experiments showed subtype-specific sensitivity to CHK1 inhibition, suggesting prexasertib as a potential targeted treatment for most currently untreatable HRP patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57e62a189c25cb112512c18bbcd2dda397ed67ef" target='_blank'>
              Decoding the Genomic and Functional Landscape of Emerging Subtypes in Ovarian Cancer.
              </a>
            </td>
          <td>
            Giulia Micoli, K. Lavikka, Yilin Li, A. Pirttikoski, D. Afenteva, Wojciech Senkowski, G. Marchi, Anna Vaharautio, T. Muranen, Titta Joutsiniemi, S. Hietanen, A. Virtanen, K. Wennerberg, J. Hynninen, J. Oikkonen, S. Hautaniemi
          </td>
          <td>2025-07-31</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Gastric cancer (GC) is classified into four molecular subtypes according to the Epstein-Barr virus-positive status, microsatellite instability, genomic stability and chromosomal instability (CIN). The CIN subtype is characterized by a high frequency of gene amplifications in receptor tyrosine kinases (RTKs) and a poor prognosis. In addition, the CIN subtype often exhibits intratumoral heterogeneity and indicates insensitivity to targeted drugs. Elucidating the molecular mechanisms of CIN in GC is therapeutically crucial; however, the molecular mechanisms involved are not yet fully understood. The kinetochore-associated 1 (KNTC1) gene encodes kinetochore-associated protein 1 (KNTC1), a major component of the outer kinetochore. The downregulation of KNTC1 causes a high frequency of lagging chromosomes and consequent aneuploidy and CIN in Drosophila and Caenorhabditis elegans. However, the association between KNTC1 and CIN in GC has not yet been clarified. Therefore, the present study investigated the role of KNTC1 in GC CIN. It was found that GC cell lines with a high frequency of lagging chromosomes had a low KNTC1 mRNA expression. Notably, KNTC1 knockdown increased the frequency of lagging chromosomes in GC cell lines. In particular, GC cell lines with the amplification of RTK genes exhibited a significant increase in the frequency of lagging chromosomes. On the whole, the findings of the present study suggest that the suppression of KNTC1 expression may contribute to CIN in GC and may be involved in the generation of intratumoral genetic heterogeneity in GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa4f34e8d31ebf5a0db6caf271a09a3caf4a8164" target='_blank'>
              Downregulation of kinetochore-associated 1 gene increases lagging chromosomes and contributes to chromosomal instability in gastric cancer cells
              </a>
            </td>
          <td>
            Daiki Ohsaki, Kazuki Kanayama
          </td>
          <td>2025-08-04</td>
          <td>Medicine International</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Tumors represent dynamically evolving populations of mutant cells, and many advances have been made in understanding the biology of their progression. However, there are key unresolved questions about the conditions that support a cell’s initial transformation, which cannot be easily captured in patient populations and are instead modeled using transgenic cellular or animal systems. Results Here, we use extensive patient atlas data to define common features of the tumor DNA methylation landscape as they compare to healthy human cells and apply this benchmark to evaluate 21 engineered human and mouse models for their ability to reproduce these patterns. Notably, we find that genetically induced cellular transformation rarely recapitulates the widespread de novo methylation of Polycomb regulated promoter sequences as found in clinical samples, but can trigger global changes in DNA methylation levels that are consistent with extensive proliferation in vitro. Conclusions Our results raise pertinent questions about the relationship between genetic and epigenetic aspects of tumorigenesis as well as provide an important molecular reference for evaluating existing and emerging tumor models. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03650-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b654b38672fde0e6d98d2fd32826d3930c1b68a1" target='_blank'>
              Direct genetic transformation bypasses tumor-associated DNA methylation alterations
              </a>
            </td>
          <td>
            Sara Hetzel, Eran Hodis, Elena Torlai Triglia, Alexander Kovacsovics, Kathleen Steinmann, A. Gnirke, Meiying Cui, Daniel McQuaid, Raha Weigert, Georg Pohl, M. Muzumdar, Serge Leyvraz, Ulrich Keilholz, M. Yaspo, Aviv Regev, H. Kretzmer, Zachary D. Smith, Alexander Meissner
          </td>
          <td>2025-07-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Background/Objectives: Optical genome mapping (OGM) has recently emerged as a new technology in the clinical cytogenomics laboratories. This methodology has the ability to detect balanced and unbalanced structural rearrangements using ultra-high molecular weight DNA. This article discusses the uses of this new technology in both constitutional and somatic settings, its advantages as well as opportunity for improvements. Methods: We reviewed the medical and scientific literature for methodology and current clinical uses of OGM. Results: OGM is a recent addition to the methods used in cytogenomics laboratories and can detect a wide range of structural and copy number variations across a plethora of diseases. Conclusions: Clinical cytogenomics is an important laboratory specialty for which various technologies have been validated over the last several decades to improve detection of copy number and structural variations and their association to human disease. OGM has proven to be a powerful tool in the arsenal of clinical laboratories and provides a unified workflow for the detection of chromosomal aberrations across a wide range of diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/415f99beee1f8b4d54a0090a8bf5feba2107610f" target='_blank'>
              Optical Genome Mapping: A New Tool for Cytogenomic Analysis
              </a>
            </td>
          <td>
            Brynn Levy, Rachel D. Burnside, Y. Akkari
          </td>
          <td>2025-07-31</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The majority of life-threatening cancers arise from epithelial tissues. These epithelial cancers include cutaneous squamous cell carcinoma (cSCC), the second-most common cancer. cSCC is highly invasive and accounts for an estimated 15,000 deaths each year. We identified SRCAP, a chromatin remodeler that regulates the chromatin occupancy of the histone H2A variant H2A.Z, as a frequently mutated gene in cSCC. Analysis of cSCC mutations in epithelial cancers identified a hotspot truncating mutation in SRCAP, which removes 42% of the protein sequences after amino acid 1879. While SRCAP mutations have been previously connected to the pathogenesis of Floating-Harbor syndrome (FHS), these typically occur downstream, with a hotspot mutation leading to protein truncation after amino acid 2444. We found that expressing the SRCAP-1879 truncation in an HRas-CDK4-driven cSCC model was sufficient to increase proliferation, impair terminal differentiation, and accelerate invasion. Mechanistically, the expression of SRCAP-1879 in primary human keratinocytes was sufficient to dysregulate genes crucial for carcinogenesis (e.g., proliferation, differentiation, and motility) without altering H2A.Z occupancy. In particular, the expression of SRCAP-1879 truncation led to strong induction of the matrix metalloproteinase MMP9 expression level, accompanied by increased keratinocyte cell motility, which was sensitive to matrix metalloprotease inhibition. In contrast, the expression of the SRCAP-FHS truncation did not increase but instead reduced cell motility as well as the expression of MMP9. Taken together, our findings identify a previously under-characterized role of the SRCAP-1879 truncating mutation in promoting multiple aspects of epithelial cancer progression, including invasion, distinct from the well-recognized roles of SRCAP mutations in FHS pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22b5e5845afc4ca4f1754b4e5d0d5637c1e90c8a" target='_blank'>
              A dominant SRCAP truncating mutation promotes squamous cell carcinoma progression
              </a>
            </td>
          <td>
            S. Droll, Elena I. O. Dewar, Celia Xue, Maxwell C Levine, Benny J Zhang, Xiaomin Bao
          </td>
          <td>2025-08-26</td>
          <td>Oncogenesis</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Base excision repair, initiated by DNA glycosylases, preserves genomic integrity by removing damaged bases. DNA glycosylases Ogg1 and Mutyh were shown to alter the hippocampal transcriptome independently of DNA damage repair. However, the role of DNA glycosylases as transcriptional regulators and their involvement in cognition remain unclear. Here, we combine transcriptomic and epigenomic analyses of the hippocampus in mice deficient in DNA glycosylases. We report that the combined deficiency of Ogg1 and Mutyh impairs spatial long-term memory. Mechanistically, Ogg1 and Mutyh modulate DNA methylation of polycomb repressive complex 2 (PRC2) target genes. PRC2 occupancy and associated histone post-translational modifications depend on Ogg1 and Mutyh in neurons and glia. These epigenetic changes correlate with cell-type specific differences in gene expression of PRC2 targets. Finally, human genetic data link DNA glycosylase-regulated genes to cognitive function, highlighting a novel role for Ogg1 and Mutyh in modulating the epigenome to control transcriptional responses relevant for brain-related diseases. Graphical Abstract Created in BioRender. Scheffler, K. (2025). https://BioRender.com/jwscej4. Supplementary Information The online version contains supplementary material available at 10.1007/s00018-025-05730-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/788c473f610d3e71e5a7b1644717d7c2a8e6b1a6" target='_blank'>
              DNA glycosylases Ogg1 and Mutyh influence gene expression of PRC2 targets associated with cognition
              </a>
            </td>
          <td>
            Andreas Abentung, Teri Sakshaug, Rabina Dumaru, N. Liabakk, Mingyi Yang, Junbai Wang, Magnar Bjørås, Katja Scheffler
          </td>
          <td>2025-08-08</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background Understanding the role of cancer hotspot mutations is essential for unraveling mechanisms of tumorigenesis and identifying therapeutic vulnerabilities. Correcting cancer mutations with base editing is a novel, yet promising approach for investigating the biology of driver mutations. Results Here, we present a versatile platform to investigate the functional impact of cancer hotspot mutations through adenine base editing in combination with transcriptomic profiling. Using this approach, we correct TP53 hotspot mutations in cancer cell lines derived from diverse tissues, followed by mRNA sequencing to evaluate transcriptional changes. Remarkably, correcting these mutations not only reveals the dependency on mutant allele expression but also restores highly conserved tumor-suppressive transcriptional programs, irrespective of tissue origin or co-occurring mutations, highlighting a shared p53-dependent regulatory network. Our findings demonstrate the utility of this base editing platform to systematically interrogate the functional consequences of cancer-associated mutations and their downstream effects on gene expression. Conclusions This work establishes a robust framework for studying the transcriptional dynamics of cancer hotspot mutations and sheds light on the conserved biological processes reinstated by p53 correction, offering potential avenues for future targeted therapies. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03667-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d441d58b01f4910f8c55a860db9d7cbefe24b5" target='_blank'>
              A base editing platform for the correction of cancer driver mutations unmasks conserved p53 transcription programs
              </a>
            </td>
          <td>
            Pascal Wang, Rituparno Sen, Frank Buchholz, Shady Sayed
          </td>
          <td>2025-07-22</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7238ca759d86a62210ebf70bb830d0369e9ff71e" target='_blank'>
              Excised DNA circles from V(D)J recombination promote relapsed leukaemia.
              </a>
            </td>
          <td>
            Zeqian Gao, James N. F. Scott, Matthew P Edwards, Dylan Casey, Xiaoling Wang, Andrew D Gillen, S. Ryan, Lisa J Russell, Anthony V Moorman, Ruth de Tute, Catherine Cargo, Anthony M Ford, D.R. Westhead, J. Boyes
          </td>
          <td>2025-08-06</td>
          <td>Nature</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="


 DNA repair and autophagy are distinct biological processes vital for cell survival. Although autophagy helps maintain genome stability, there is no evidence of its direct role in the repair of DNA lesions. We discovered that lysosomes process Topoisomerase 1-cleavage complexes (TOP1cc) DNA lesions in human cancer cell. TOP1 resolves DNA topological stress ahead of DNA replication and transcription. If not promptly repaired, TOP1cc hinder the progression of DNA replication and transcription fostering genomic instability. Thus, stabilising TOP1cc with TOP1 poisons, such as Camptothecin or its analogue, is highly effective and widely used in cancer therapy. Proteolysis of TOP1cc by the proteasome is described at excessively high doses of CPT that are not relevant in a clinical setting, calling into question the relevance of the proteasome role in TOP1cc repair for cancer therapy and patient response. Here we described that repair of TOP1cc induced by clinically relevant dose of Camptothecin depends on selective autophagy. Proteomics, sequencing, biochemistry techniques and live imaging all demonstrate the lysosomal uptake of TOP1cc upon induction of replication stress. Selective degradation of TOP1cc by autophagy directs DNA damage repair and cell survival. TOP1cc are exported from the nucleus to lysosomes through transient alteration of the nuclear envelope and independent of the proteasome. Mechanistically, the autophagy receptor TEX264 acts as a TOP1cc sensor at DNA replication forks, triggering TOP1cc processing by the p97 ATPase and mediating the delivery of TOP1cc to lysosomes in an MRE11 nuclease and ATR kinase-dependent manner. We found an evolutionarily conserved role for selective autophagy in DNA damage repair that enables cell survival, protects genome stability, and is clinically relevant for colorectal cancer patients.



 Pauline Lascaux, Gwendoline Hoslett, Sara Tribble, Ivan Antičević, Cecile Otten, Ignacio Torrecilla, Yichen Zhao, Wei Song, Cristiano Peron, Giulio Deangeli, Enric Domingo, James Bancroft, Loïc Carrique, Errin Johnson, Alvin Wei Tian Ng, Joanne Ngeow, Nuno Raimundo, Tim Maughan, Marta Popović, Ira Milošević, and Kristijan Ramadan. TEX264 DRIVES SELECTIVE AUTOPHAGY OF DNA LESIONS TO PROMOTE DNA REPAIR AND CELL SURVIVAL [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P11.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ada9e8b0a0e36cefc3f3e65806469a480324003" target='_blank'>
              Abstract P11: TEX264 DRIVES SELECTIVE AUTOPHAGY OF DNA LESIONS TO PROMOTE DNA REPAIR AND CELL SURVIVAL
              </a>
            </td>
          <td>
            Pauline Lascaux, Gwendoline Hoslett, Sara Tribble, Ivan Anticevic, Cecile Otten, Ignacio Torrecilla, Yichen Zhao, Wei Song, Cristiano Peron, Giulio Deangeli, Enric Domingo, James Bancroft, L. Carrique, Errin Johnson, Alvin Wei Tian Ng, Joanne Ngeow, Nuno Raimundo, Tim Maughan, Marta Popović, Ira Milošević, Kristijan Ramadan
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a type of hematological cancer that arises from uncontrolled proliferation of plasma cells. In addition to frequent genetic mutations, malignant plasma cells are characterized by alterations to the epigenome. Myeloma cells display a genome-wide loss of DNA methylation and a corresponding increase in ‘active’ chromatin modifications. The dramatic epigenetic remodeling that occurs in cancer genomes is associated with loss of silencing at transposable elements, which can impact genome regulation. Through paired epigenome and transcriptome profiling of patient derived MM samples, we have found that loss of DNA methylation in MM genomes results in the formation of partially methylated domains that are variable across patients. This loss of DNA methylation coincides with the expression of hundreds of transcripts driven by LINE-1 (L1) retrotransposons that are epigenetically silenced in normal cells. MM samples can be stratified based on L1 activity with distinct gene expression signatures. The high L1 samples are characterized by a more proliferative, less differentiated state as well as inhibition of interferon and genome defense pathways. Several L1 promoters generate chimeric transcripts with adjacent oncogenes. We further find that KRAB-zinc finger proteins (KZFPs) that are responsible for the epigenetic silencing of L1s have abnormally low abundance in MM samples with high L1 activity. These results indicate that cell proliferation in MM is associated with a loss of KZFP expression and activation of L1 elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e8c94d95f4ddee7048f28a27ac43f9a3d704355" target='_blank'>
              Methylation variability and LINE-1 activation in multiple myeloma
              </a>
            </td>
          <td>
            Qianhui Wan, Amy Leung, Mahek Vinod Bhandari, Hiroyuki Kato, Joo Y. Song, Dustin Schones
          </td>
          <td>2025-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract G-quadruplexes (G4s) are prevalent at promoters and superenhancers (SEs), exclude nucleosomes, and recruit transcription factors. This study sought to determine whether the nucleosome exclusion affects the recruitment of the SE marker BRD4, which typically binds to acetylated histones and facilitates SE–promoter contacts via the phase separation-dependent mechanism. Analyses of the available whole-genome data revealed that SEs with the highest G4 density were depleted of nucleosomes but not of BRD4. This led us to test the possibility of histone-independent BRD4 maintenance at G4-rich SEs. A typical SE G4 destabilized a nearby nucleosome in vitro and, unlike B-DNA, bound weakly to BRD4 bromodomains. Similar to an acetylated nucleosome, the G4 promoted phase separation in BRD4 solutions. This effect was not altered by the histone competitor JQ1. However, it was attenuated by two known G4 ligands, suggesting that they could disrupt SE–promoter communication in cells. Consistently, these ligands downregulated several genes regulated by G4-rich SE-contacting promoters more efficiently than they did SE-independent genes. Our findings underscore the significance of G4-rich SEs as transcriptional regulators and provide new insights into their organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ca6b9e0a4bd9850a35faa9e07c7255af9d5b35c" target='_blank'>
              G-quadruplexes as potential traps for superenhancer marker BRD4: ligand-sensitive binding and co-separation in vitro
              </a>
            </td>
          <td>
            Iuliia Pavlova, Olga M. Ivanova, Mikhail S Iudin, Anastasiya V Surdina, Nikolay A Barinov, M. E. Bogomiakova, Sergey D Oreshkov, Z. Shenkarev, V. Severov, Dmitriy V Klinov, Victoria O Shender, A. Bogomazova, M. Lagarkova, A. Varizhuk, Vladimir B Tsvetkov
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Overexpression of the proto-oncogene MYC occurs in >70% of cancers and is especially prevalent in breast cancer. Myc partners with transcription factor Max to bind to the E-box DNA response element. By patterning our frankenproteins on the basic region/helix-loop-helix/leucine zipper motif of Max, we designed MEF and MEF/C93 to bind to the E-box. In bacterial one-hybrid assays and quantitative electrophoretic mobility shift assay, both proteins bound specifically to the E-box with high sequence-specificity and affinity (Kd = 8 nM) rivaling native transcription factors. Quantitative PCR revealed that MEF and MEF/C93 selectively downregulated Myc target genes in Myc-dependent MDA-MB-231 breast cancer cells, but not in Myc-independent MCF-7 cells. Fluorescence colocalization demonstrated transport into cell nuclei. Our proteins displayed IC50 values of 1-2 μM in cell viability assays in MDA-MB-231, compared with ∼25 μM in MCF-7. These results demonstrate the specificity of targeting Myc-dependent cancer cells and mark significant progress toward protein-based therapies aimed at Myc-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8cc39bbdc2499f24c52dd2589dd7f515be4ffe6" target='_blank'>
              Designer Frankenproteins That Halt the Proliferation of Myc-Driven Cancer Cells.
              </a>
            </td>
          <td>
            Maryam Ali, Raneem Akel, Francine He, Micheline Piquette-Miller, Jumi A Shin
          </td>
          <td>2025-09-03</td>
          <td>Journal of medicinal chemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Structural maintenance of chromosomes (SMC) complexes organize genomes by extruding DNA loops, while replisomes duplicate entire chromosomes. These essential molecular machines must collide frequently in every cell cycle, yet how such collisions are resolved in vivo remains poorly understood. Taking advantage of the ability to load SMC complexes at defined sites in the Bacillus subtilis genome, we engineered head-on and head-to-tail collisions between SMC complexes and the replisome. Replisome progression was monitored by genome-wide marker frequency analysis, and SMC translocation was monitored by time-resolved ChIP-seq and Hi-C. We found that SMC complexes do not impede replisome progression. By contrast, replisomes restrict SMC translocation regardless of collision orientations. Combining experimental data with simulations, we determined that SMC complexes are blocked by the replisome and then released from the chromosome. Occasionally, SMC complexes can bypass the replisome and continue translocating. Our findings establish that the replisome is a barrier to SMC-mediated DNA-loop extrusion in vivo, with implications for processes such as chromosome segregation, DNA repair, and gene regulation that require dynamic chromosome organization in all organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/686025c7ec346de8dd0c0dabfe2ed3d2aaa2a6a1" target='_blank'>
              Replisomes restrict SMC translocation in vivo
              </a>
            </td>
          <td>
            Qinchan Liao, Hugo B. Brandão, Zhongqing Ren, Xindan Wang
          </td>
          <td>2025-08-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background Despite the comprehensive advancement in the field of cancer therapeutics, there remains an urgent need to identify new pathophysiological mechanisms that can be targeted in isolation or in combination with existing therapeutic regimens. The epithelial-to-mesenchymal transitions (EMT) induced by hypoxia, cytokines, and growth factors involves acquisition of invasive and migratory properties by cancer cells. Epigenetic alterations of DNA methylations and/or histone modifications cause substantial transcriptomic reprogramming in cancer cells during EMT and metastasis, which can be therapeutically targeted by a thorough understanding of the mutual interactions among the epigenetic processes. Previously, the mammalian DNA methyltransferases (DNMTs) have been shown to possess redox- and Ca++- dependent active DNA 5mC demethylation activities in addition to the cytosine methylation activity. Methods In this study, we have carried out experiments using a range of molecular, cellular, and genome editing approaches including cell culturing, CRISPR/Cas9-editing, si- or sh-RNA-mediated knockdown, quantitative RT-PCR, western blotting, ChIP-qPCR, Na-bisulfite sequencing, EMT and lung colonization assays in conjunction with DNA methylome and DNMT3A ChIP-Seq analyses, Results We found that active DNA demethylation activity of DNMT3A is essential for hypoxia-induced EMT of the SW480 colon cancer cells, its global genomic DNA demethylation, and promoter DNA demethylation/transcriptional activation of EMT-associated genes including TWIST1 and SNAIL1. DNMT3A also regulates hypoxia-induced HIF-1α binding to and transcriptional activation of the TWIST1 promoter as well as genome-wide DNA demethylation and EMT of breast cancer and liver cancer cells. Mechanistic analysis supports a regulatory model where hypoxia-induced H3K36me3 mark recruits DNMT3A to demethylate CpG in the hypoxia-responsive element (HRE), thereby facilitating HIF-1α binding and activation of the promoters of EMT genes. Conclusions Altogether, this study has provided the first demonstration of a physiological function of the active DNA demethylation activity of the DNMTs. Equally important, our findings have revealed a missing link between the HIF-1α pathway and the O2-sensing KDM pathway both of which are known to be essential for a wide set of normal and disease-associated cellular processes. Finally, the active DNA demethylation activity of DNMT3A has now emerged as a new potential target for therapeutic development to prevent EMT and metastasis of cancer cells. Clinical trial number Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s11658-025-00775-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/754e95e75763fae58f7d2af9f820e9cabdaae0fe" target='_blank'>
              Hypoxia-induced genome-wide DNA demethylation by DNMT3A and EMT of cancer cells
              </a>
            </td>
          <td>
            Biswanath Chatterjee, P. Majumder, Chun-Chang Chen, Jing-Ping Wang, P. Su, Hung-Cheng Lai, Ching-Chen Liu, Hsin-Nan Lin, Chen-Hsin A Yu, Hanna S Yuan, Che-Kun James Shen
          </td>
          <td>2025-08-05</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e39cce89c2554835e6bd04e09986a82ce912ced" target='_blank'>
              The CIC::DUX4 oncoprotein maintains DNA integrity through direct regulation of the catalytic subunit of DNA polymerase epsilon (POLE).
              </a>
            </td>
          <td>
            Zeinab Kosibaty, Cuyler Luck, Ross A. Okimoto
          </td>
          <td>2025-08-04</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Telomere homeostasis is pivotal in various biological processes including ontogeny, reproduction, physiological aging, and the onset of numerous diseases such as tumors. In human stem cells and approximately 85% of tumor cells, telomerase formed by TERT and TERC RNA complex is responsible for elongating telomeres. However, the intricate and precise regulatory mechanisms governing telomerase remain largely elusive. Methods and results We developed a genome-wide trimolecular fluorescence complementation (TriFC) screen to identify TERC RNA-interacting proteins and found ZC3H15 (Zinc finger CCCH domain-containing protein 15) to interact with telomerase. ZC3H15 interacts with TERT via its N-terminal domain in an RNA-dependent manner. The proximity labeling technique PhastID revealed that ZC3H15 associates with proteins involved in regulation of ribonucleoprotein (RNP) complex biogenesis, snRNP assembly and RNA localization. Deletion of ZC3H15 upregulated telomerase activity but interestingly resulted in shortened telomeres and induced senescence in HTC75 cells, suggesting an unknown mechanism in regulating telomere length. Notably, we found ZC3H15 to associate with GEMs nuclear bodies, and its deletion led to the spatiotemporal fusion of GEMs and Cajal bodies, resulting in the sequestration of telomerase within Cajal bodies and a reduction in telomerase recruitment to telomeres during the S phase. Consistent with these findings, ZC3H15 ablation accumulated TERC precursor RNA. Conclusions These observations provide valuable insights into the molecular mechanisms by which ZC3H15 regulates telomerase dynamics and cellular senescence. ZC3H15 may represent a new target for cancer treatment and anti-aging therapies. Supplementary Information The online version contains supplementary material available at 10.1186/s13578-025-01449-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df824506b64a6082d42aacda64e880366cc87c4" target='_blank'>
              Disruption of ZC3H15 compromises telomere length maintenance by entrapping telomerase within cajal bodies
              </a>
            </td>
          <td>
            Chuanlei Wang, Wei Chen, Ruofei Li, Yue Yang, Jiali Wu, Yuyang Tian, Zibin He, Song Lin, Xining Wang, Jianxi Zhu, Wenbin Ma, Songyang Zhou, Yan Huang
          </td>
          <td>2025-07-22</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Glioblastoma, the most common primary malignant brain tumor, has a median survival of less than two years. This is due in part to a subpopulation of cells called glioblastoma stem cells (GSCs), which drive tumor recurrence. Transposable elements (TEs) are expressed at higher levels in cancer stem cells, enhancing the oncogenic potential and plasticity of cells through changes in gene expression, fusion transcript generation, and genomic rearrangement. Results Leveraging a large previously published dataset, we investigated the expression of TEs in bulk RNA sequencing data from 42 GSCs to identify subpopulations defined by their TE expression profile. Using telescope, a locus-specific approach to quantifying TE expression, we identified 858 TE loci that were expressed and defined two groups of GSCs using a consensus clustering approach. These TE-driven clusters displayed significant differences in both transcription factor (TF) and gene expression, with one group significantly enriched for a mesenchymal signature based on Gene Set Enrichment Analysis. Next, we extracted the locations and sequences of the TE regulatory domains and elucidated TF binding motifs within the TE sequences. This showed that the SOX11 consensus motif was enriched in the 5’ untranslated region of differentially expressed long interspersed nuclear elements (LINE). SOX11, a known inducer of LINE expression, was significantly under-expressed in the mesenchymal GSC cluster, which correlated with the concurrent decreased expression of LINE transcripts. These loci also overlapped with the enhancer elements of genes that were significantly downregulated, suggesting a potential link between TF binding to TE regulatory regions and gene expression. Conclusions Although further mechanistic studies are required, the identified link between TE location, TE and TF expression, and corresponding gene expression suggests that TEs may play a regulatory role in GSC transcription regulation. The current findings highlight the need for further investigation into the role of TEs in defining the gene regulatory and expression landscapes of GSCs. Future studies in this area could have therapeutic implications, given that glioblastoma recurrence may be driven by these cells. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00370-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ceb4559d6649970b8ef9337621d841fa0e46f7e4" target='_blank'>
              Transposable element dynamics in glioblastoma stem cells: insights from locus-specific quantification
              </a>
            </td>
          <td>
            Mattia Pizzagalli, Yusuke Suita, Owen P. Leary, Nikos Tapinos
          </td>
          <td>2025-09-02</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract R-loops or DNA–RNA hybrids are prominent nucleic acid structures that commonly arise during transcription. These structures play important biological functions, such as regulating gene expression and DNA repair. However, when unresolved by nucleic acid processing factors, pathological R-loops can be harmful and lead to genome instability. N6-Methyladenosine (m6A), the most prevalent modification in messenger RNA, has been recently identified to be crucial for regulating R-loop balance and maintaining genome stability. Strikingly, m6A-modified R-loop formation can have opposing consequences, either stabilization or resolution, depending on the biological context. In this review, we discuss the current knowledge of the regulatory roles of m6A on R-loops across various processes, including gene transcription, DNA repair, and centromere and telomere stability. Additionally, we explore other m6A-mediated processes, such as nascent transcription and chromatin landscape, that potentially affect R-loop dynamics. Finally, we discuss the current limitations and future directions of studying the m6A–R-loop axis, as well as the opportunities to target this pathway as a potential therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5d8ee286e956882ce3a5149e41ece752c38e0d" target='_blank'>
              m6A modification in R-loop homeostasis: a potential target for cancer therapeutics
              </a>
            </td>
          <td>
            Minh-Anh Vu, Manuela Spagnuolo, Chun-Long Chen
          </td>
          <td>2025-08-11</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metastasis remains a major cause of cancer mortality. This study, expanding upon previous findings in the MMTV-PyMT model, investigated four independent mouse models, representing luminal (MMTV-PyMT, MMTV-Myc), HER2-amplified (MMTV-Her2), and triple-negative (C3(1)TAg) breast cancer subtypes. Consistent with previous results, limited evidence for metastasis-associated somatic point mutations was found for all models. We also found that oncogenic drivers significantly influenced the number and size of metastasis-specific copy number variations (MSCNVs), but common driver-independent MSCNVs were rare. Furthermore, analyzing a cohort with varying genetic backgrounds while maintaining a constant oncogenic driver (PyMT) revealed that genetic background profoundly impacts MSCNVs. Transcriptome analysis demonstrated that oncogenic drivers strongly shaped metastasis-specific gene expression (MSGE), with each driver exhibiting distinct expression profiles. In contrast, MSGE in the PyMT-F1 cohort was more variable across strains. Despite the diversity of MSCNV and MSGE, functional analysis revealed that both mechanisms converge on the modulation of key cellular processes, including immune responses, metabolism, and extracellular matrix interactions. These findings emphasize the complex interplay between oncogenic drivers and genetic background in shaping the genomic and transcriptional landscapes of metastatic lesions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddbb3089644af163030736f3789ed79a1b202fc7" target='_blank'>
              Genetic background and oncogenic driver determines the genomic evolution and transcriptomics of mammary tumor metastasis
              </a>
            </td>
          <td>
            Christina R Ross, Karol Szczepanek, Jack Sanford, Gabriel A. Needle, Tinghu Qiu, Larry Pearce, Maxwell P Lee, Kent W. Hunter
          </td>
          <td>2025-08-14</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Satellite DNA is essential for chromosome stability and gene regulation, yet its specific roles in early human embryogenesis remain poorly defined. Here, we integrated the complete human genome reference (T2T-CHM13) with RNA-seq data to investigate the expression and regulation of the satellite DNA element D20S16 across key stages of human embryonic development. We identified 20 distinct D20S16 tandem repeat clusters, but found that only two, both located on chromosome 20, were actively transcribed during early embryogenesis. Expression of D20S16 was high in early developmental stages, significantly declining thereafter. Comparative analysis revealed minimal expression of D20S16 in macaque embryos, correlating with fewer and shorter repeat units. Beyond embryogenesis, D20S16 also exhibited notably high expression levels in breast cancer and testicular tissues, suggesting additional biological roles. Furthermore, we investigated the evolutionary distribution of D20S16 across primates and other mammals. Our findings highlight the potential regulatory functions of satellite DNA in human development, pointing to the importance of specific chromosomal contexts for transcriptional activation. This study enhances our understanding of satellite DNA’s functional and evolutionary significance, laying the groundwork for future research into its roles in development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3615f227ececbc4cca8de8d71737eff9957c4e38" target='_blank'>
              Unveiling unique expression patterns of D20S16 satellite DNA in human embryonic development
              </a>
            </td>
          <td>
            Yajie Hu, Kenji Mizuguchi, Kosuke Hashimoto
          </td>
          <td>2025-07-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Epigenetic traits are persistent cellular and organismal properties that do not result from changes in DNA sequence. One such property involves transmission of chromosomes, which entails the formation of highly specialized chromatin structures, the kinetochores, on selected chromosomal regions, called centromeres. Centromere function is essential and centromeres are determined epigenetically by the deposition of a variant histone H3 CENP-A (CENH3 in plants). Either reduced or ectopic function alone leads to genome instability, decreased fitness, aneuploid syndromes, and cancer. At times, however, centromeres malfunction in an apparently programmed mode. This is exemplified by a peculiar centromeric syndrome involving selective elimination of a chromosome set, which can affect a wide range of organisms, including plants. Over half a century ago, plant geneticists described this syndrome in interspecific crosses of barley. Building on their work, we examine the growing understanding of how CENH3 function can be modified to affect epigenetic regulation of centromeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024266b37112d5916123a1fab820a516c26c7be4" target='_blank'>
              Pioneers of chromosome elimination
              </a>
            </td>
          <td>
            Luca Comai, M. P. A. Marimuthu
          </td>
          <td>2025-07-29</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="R-loops, chromatin structures containing DNA-RNA hybrids with displaced single-stranded DNA, play crucial roles in various cellular processes. Their formation is influenced by factors such as DNA topology, RNA stability, and the presence of GC-rich regions. However, excessive or uncontrolled R-loop accumulation can threaten genomic stability, leading to DNA damage, particularly double-strand breaks. To preserve genome integrity, cells have developed mechanisms to regulate R-loop formation and resolution. Dysregulation of these processes is linked to several diseases, including cancer and neurodegenerative disorders. In this review, we will explore the dynamics of R-loop formation and resolution and how they are detected, their roles in DNA damage and repair, and how their dysregulation may lead to immune responses and disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3728d9fd65e0fa0dfc3340b92946a313355d5d1" target='_blank'>
              The paradox of R-loops: guardians of the genome or drivers of disease?
              </a>
            </td>
          <td>
            Shachinthaka D Dissanayaka Mudiyanselage, Phillip Wulfridge, K. Sarma
          </td>
          <td>2025-08-06</td>
          <td>Genome research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708353dcfdc8ca7c4e2b434fced2427c746e282e" target='_blank'>
              Tracing the evolution of single-cell 3D genomes in Kras-driven cancers.
              </a>
            </td>
          <td>
            Miao Liu, Shengyan Jin, Sherry S Agabiti, Tyler B Jensen, Tianqi Yang, Jonathan S. D. Radda, Christian F. Ruiz, Gabriel Baldissera, Moein Rajaei, Fangyong Li, Jeffrey P. Townsend, M. Muzumdar, Siyuan Wang
          </td>
          <td>2025-08-18</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Esophageal carcinoma (ESCA) is a common tumor in the digestive system, resulting in approximately 300,000 death worldwide every year. ALDH18A1, a potential RNA binding protein (RBP), is significantly overexpressed as in ESCA, while its functions and mechanism is unclear. In this study, we silenced ALDH18A1 in KYSE150 cells using small interfering RNA (siALDH18A1), and evaluated its effect on cell characteristics. Finally, we conducted whole transcriptome sequencing (RNA-seq) and RNA immunoprecipitation (iRIP-seq) experiments to explore the downstream targets of ALDH18A1, and verified their change patterns using quantitative polymerase chain reaction. We found that ALDH18A1 exhibited a high expression level in ESCA patients using online database. The proliferation, invasion, and migration abilities of KYSE150 cells were significantly inhibited in siALDH18A1 samples. Through the investigation of RNA-seq data, we found that ALDH18A1 globally regulates the expression and alternative splicing profiles of KYSE150 cells by identifying 437 differentially expressed genes (DEGs) and 1071 ALDH18A1-regulated AS events (RASEs), respectively. The functions of RASE genes (RASGs) were enriched in pathways such as DNA repair and gene expression. In addition, through the integration analysis of iRIP-seq data and DEGs, we found that ALDH18A1 bound to and regulated the expression of 59 DEGs, including LAMB3 and IL1B, which were verified by RT-qPCR experiment. In addition, we performed an overlap analysis between the genes bound by ALDH18A1 and RASGs and found 339 bound RASGs. The functional enrichment of these genes revealed pathways such as DNA repair, cell proliferation, and DNA damage response. Among them, we found significant alternative splicing events of MTA1, which were confirmed by PCR experiments. In summary, our study comprehensively explored the functions and downstream targets of ALDH18A1 in ESCA cells, highlighting the molecular mechanism of ALDH18A1 on the carcinogenic characteristics of ESCA. The identified molecular targets, especially DNA repair-related genes, can serve as potential therapeutic targets for future ESCA treatment. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-08006-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4696a54f3a85524e5b23e6278282dcbdfa70bd9a" target='_blank'>
              ALDH18A1 has carcinogenic functions and regulates alternative splicing events of DNA repair-related genes in esophageal carcinoma cells
              </a>
            </td>
          <td>
            Yongkang Wang, Maimaiti Yisireyili, Kelimu Abudureyimu, Atsushi Enomoto
          </td>
          <td>2025-08-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Epigenetic regulation is critical to B cell development, guiding gene expression via DNA methylation, histone modifications, chromatin remodeling, and noncoding RNAs. In mature B cell neoplasms, particularly diffuse large B cell lymphoma (DLBCL), follicular lymphoma (FL), and chronic lymphocytic leukemia (CLL), these mechanisms are frequently disrupted. Recurrent mutations in key epigenetic regulators such as EZH2, KMT2D, CREBBP, and TET2 lead to altered chromatin states, repression of tumor suppressor genes, and enhanced oncogenic signaling. Dysregulation of specific microRNAs (e.g., miR-155, miR-21) further contributes to pathogenesis and therapeutic resistance. In DLBCL, hypermethylation of SMAD1 and CREBBP mutations are associated with immune evasion and chemoresistance. In FL, EZH2 gain-of-function and KMT2D loss-of-function mutations alter germinal center B cell programming, while in CLL, DNA hypomethylation patterns reflect the cell of origin and correlate with clinical outcome. Targeted therapies such as the EZH2 inhibitor tazemetostat have demonstrated efficacy in EZH2-mutant FL, while HDAC and BET inhibitors show variable responses across B cell malignancies. The limitations of current epigenetic therapies reflect the complexity of targeting epigenetic dysregulation rather than therapeutic futility. These challenges nonetheless highlight the relevance of epigenetic alterations as biomarkers and therapeutic targets, with potential to improve the management of mature B cell neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a068788c20e74398875dae829a5f27eeefbe604a" target='_blank'>
              Unraveling the Epigenetic Landscape of Mature B Cell Neoplasia: Mechanisms, Biomarkers, and Therapeutic Opportunities
              </a>
            </td>
          <td>
            Nawar Maher, F. Maiellaro, J. Ghanej, S. Rasi, R. Moia, G. Gaidano
          </td>
          <td>2025-08-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract Methylglyoxal (MG) is a highly reactive aldehyde that is produced endogenously during metabolism, and from exogenous sources like sugary food and cigarette smoke. Unless detoxified by glyoxalases, MG can readily react with DNA and proteins, generating characteristic glycation-derived lesions. As a result, MG exposure has been linked to a variety of human diseases, including cancers. Prior studies show that MG preferentially makes adducts on guanine residues, causing DNA damage. However, in vivo, how such events impact genome-wide MG mutagenicity is poorly understood. Such information is essential to comprehend the true contribution of MG to genome instability and global mutational burden. In the present study, we show that MG can robustly mutagenize single-stranded DNA in the yeast genome, within a guanine-centered mutable motif. We demonstrate that genome-wide MG mutagenesis is greatly elevated in the absence of the glyoxalase Glo1, and abrogated in the presence of the aldehyde quencher aminoguanidine. Importantly, we uncovered strand slippage and mispairing as the predominant mechanism for generation of all MG-associated mutations, and demonstrate that the translesion polymerase Rev1 is a key player in this pathway. Finally, we find that the primary MG-associated mutation is enriched in a variety of sequenced tumor datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2ac197fe47da2319b761beabd3304e807a2cc0f" target='_blank'>
              Methylglyoxal mutagenizes single-stranded DNA via Rev1-associated slippage and mispairing
              </a>
            </td>
          <td>
            Sriram Vijayraghavan, Alessandra Ruggiero, Samuel Becker, P. Mieczkowski, George S. Hanna, Mark T. Hamann, Natalie Saini
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28b36c19d01be738ee1237dc98c176d6789c1e47" target='_blank'>
              Chromatin-associated circRNA ciCRLF3(2) regulates cell differentiation blockage via activating non-homologous end joining-based DNA repair.
              </a>
            </td>
          <td>
            Ke-Jia Pu, Xiao-Tong Chen, Shun-Xin Zhu, Yan An, Xin-Yi Feng, Heng-Jing Huang, Cheng-Lin Zhou, Mei-Ying Ye, Yun-Chun Wei, Yi-Xuan Ma, Chen Fang, Nan Zhang, Dan Wang, Bin-Rong Han, Jun-Yi Lian, Tian-Qi Chen, Yumeng Sun, Yue-Qin Chen, Wentao Wang
          </td>
          <td>2025-09-04</td>
          <td>Cell death and differentiation</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Ribophorin 1 (RPN1), a key component of the oligosaccharyltransferase complex, is implicated in tumor progression through glycosylation-mediated pathways, yet its pan-cancer roles remain unexplored. This study presents a comprehensive multi-omics analysis of RPN1 across 33 cancers such as sarcoma (SARC), integrating genomic, transcriptomic, and proteomic data from TCGA, GTEx, and CPTAC. RPN1 was significantly overexpressed in 14 malignancies and correlated with advanced tumor stages and poor prognosis in glioblastoma (GBM), lower-grade glioma, SARC and hepatocellular carcinoma, validated in an independent glioma cohort (n = 151). Genomically, RPN1 amplification linked to homologous recombination deficiency and elevated tumor mutational burden, suggesting a role in genomic instability. Critically, multiplex immunofluorescence demonstrates RPN1 overexpression colocalizes with CD206+ M2 macrophages in tumor microenvironments, while in vitro coculture experiments confirm RPN1-dependent microglial recruitment and M2 polarization. RPN1 expression negatively correlates with CD8+ T cell infiltration and predicts resistance to chemotherapy (GBM, ovarian cancer) and immunotherapy (GBM, esophageal carcinoma), though it associates with PD-1 inhibitor sensitivity in bladder cancer. Functional validation shows RPN1 knockdown suppresses proliferation, migration, and invasion in GBM cells. Pathway enrichment connects RPN1 to endoplasmic reticulum stress, glycosylation, DNA repair, and immune checkpoint regulation. These findings position RPN1 as a multimodal oncogenic driver promoting genomic instability, immunosuppressive microenvironment remodeling, and context-dependent therapeutic vulnerabilities across cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fad4cdee9a5624b27cf2ed12f95c27e148fc7b5" target='_blank'>
              RPN1 Is Associated With Immunosuppression in Pan-Cancer and Affects the Malignant Phenotype of Tumor.
              </a>
            </td>
          <td>
            Shaoru Lin, Changwu Wu, Qing Liu, Fushu Luo, Jun Tan
          </td>
          <td>2025-08-26</td>
          <td>FASEB journal : official publication of the Federation of American Societies for Experimental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Small-cell lung cancer (SCLC), an aggressive neuroendocrine tumor strongly associated with exposure to tobacco carcinogens, is characterized by early dissemination and dismal prognosis with a five-year overall survival of less than 7%. High-frequency gain-of-function mutations in oncogenes are rarely reported, and intratumor heterogeneity (ITH) remains to be determined in SCLC. Here, via multiomics analyses of 314 SCLCs, we found that the ASCL1+/MKI67+ and ASCL1+/CRIP2+ clusters accounted for 74.38% of the 190,313 SCLC cancer cells from 39 patients, with the ASCL1+SOX1+ stem-like cell cluster across SCLC subtypes. The major histocompatibility complex (MHC) class I molecules were expressed at low levels in six and high levels in five cancer cell clusters and were inversely associated with the KI67 expression level. Abnormal splicing of mRNAs was a feature of SCLC, with focal adhesion kinase (FAK) splicing variants identified in 119 (77.3%) of 154 patients. FAK variants exhibited elevated kinase activity, were associated with the worst prognosis, and were sensitive to FAK inhibitors in patient-derived organoids and xenograft models. Eleven high-frequency mutations were identified in addition to TP53 and RB1, and smoking status and tumor stage did not affect microbiota variance in SCLC. Taken together, our data further revealed the complicated ITH and discovered that FAK splicing variants represent high-frequency gain-of-function alterations in oncogene in SCLC and potential therapeutic targets for this recalcitrant cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3bde80c19d7166cce5785be8ae3a4a5e971fbbd" target='_blank'>
              Characterization of the extrinsic and intrinsic signatures and therapeutic vulnerability of small cell lung cancers
              </a>
            </td>
          <td>
            Gui-zhen Wang, Zheng Wang, , Yun Tan, Wen-Zhao Zhong, Guo-Gui Sun, Yu-Tao Liu, B. Pan, Chen Huang, Di Wang, Beibei Sun, Dong-Ni Chen, Bin Zhang, , Sheng Li, , Si-Chong Han, Fu-ying Yang, Xue-Yan Shi, Xiao-Liang Jie, , , , Li Zhang, Mingkun Li, Na Wang, Jin-Song Liu, Ying Dong, Manli Wang, Yan Wang, Chang-Li Wang, , , Jian-Ming Ying, Chong Chen, Yunchao Huang, , Yuan-Yuan Zhang, Yan Yu, 
          </td>
          <td>2025-09-10</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Genome-wide association studies (GWAS) relevant to osteoporosis have identified hundreds of loci; however, understanding how these variants influence the phenotype is complicated because most reside in non-coding DNA sequence that serves as transcriptional enhancers and repressors. To advance knowledge on these regulatory elements in osteoclasts (OCs), we performed Micro-C analysis, which informs on the genome topology of these cells and integrated the results with transcriptome and GWAS data to further define loci linked to BMD. Using blood cells isolated from 4 healthy participants aged 31-61 yr, we cultured OC in vitro and generated a Micro-C chromatin conformation capture dataset. We characterized chromatin loops (CLs) in OC from among more than 69 million chromatin interactions identified in the genome. Of the CL identified in OC, >16 000 were unique compared to precursor cells. When sentinel single nucleotide polymorphisms from osteoporosis and bone-related GWAS and those in linkage disequilibrium at r2 > 0.6 were mapped to CL for OC, 12 588 of these variants were observed within chromatin contact regions. Notable in differential gene ontology enrichment analyses of the topology data for OC and precursors were pathways regulating pluripotency of stem cells, Wnt signaling, nucleotide-binding oligomerization domain (NOD)-like receptor signaling and chemokine signaling. These data, in combination with other 3D genome architecture and epigenetic data (eg, histone modifications and chromatin accessibility), will be useful in modeling to predict genome-wide, which enhancers regulate which genes in OC. This data will therefore also be informative for resolving GWAS hits. In conclusion, we have generated a high-resolution genome topology dataset for human OC and have used this to identify CLs relevant to studies of the genetics of osteoporosis. This data will serve as a powerful resource to inform future functional studies of OC biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1db75c088b9f9ba0973d277c93aec7ad4cfeffd1" target='_blank'>
              Genome topology analysis and transcriptomics of human osteoclasts reveals enhancer–promoter interactions at loci for bone traits and diseases
              </a>
            </td>
          <td>
            Scott G Wilson, Purdey J Campbell, Dhanya Sooraj, Kassandra M. Leatherbarrow, B. Mullin, Suzanne J Brown, K. Zhu, S. Mullin, Bryan K Ward, Jordan Zhang, Jonathon Torchia, F. Dudbridge, Jiake Xu, N. Pavlos, David Chandler, John P Walsh
          </td>
          <td>2025-07-16</td>
          <td>JBMR Plus</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Repeated oncogenic mutations and polyclonal proliferation are evident in cancers. However, little is known about the polyclonal principles governing the systemic cancerous lineage during immunotherapy. Here, we examine a unique autopsy case of metastatic urothelial carcinoma that exhibits different treatment responses to anti-PD-1 therapy at each tumor site. By performing in-depth analyses of different multiregional bulk tumor masses, we reveal that subsets of subclones acquire potential driver mutations under treatment selection pressure. Spatial transcriptomics analysis reveals that subclones resistant to immunotherapy form distinct immunosuppressive environments consistent with their habitats. Furthermore, different cancer hallmarks are identified in each of the subclones that expand under immunotherapy at single-cell level; for example, one subclone is more proliferative, and another is more stem-cell-like. In summary, this study provides an overall picture of the polyclonal competition and changes in the immune microenvironment that are related to resistance to immunotherapy in patients with malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f357f9c7e57b9e95e8ee48a6378f3d690f3275bd" target='_blank'>
              Clonal diversity shapes the tumour microenvironment leading to distinct immunotherapy responses in metastatic urothelial carcinoma
              </a>
            </td>
          <td>
            Takashi Kamatani, K. Umeda, T. Iwasawa, Fuyuki Miya, K. Matsumoto, S. Mikami, Kensuke Hara, Masayuki Shimoda, Yutaka Suzuki, Jo Nishino, Mamoru Kato, Kazuhiro Kakimi, Nobuyuki Tanaka, M. Oya, T. Tsunoda
          </td>
          <td>2025-08-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is one of the most prevalent adult leukemias, derived from mature B-cells and exhibiting a highly heterogeneous disease course. Standard cytogenetic analysis of CLL includes FISH and karyotyping. However, conventional chromosome analysis of cancer specimens is often constrained by low chromosomal resolution, and FISH analysis is limited by the number of probes that can be applied. This study highlights the application of optical genome mapping (OGM), a high-resolution cytogenomic tool that visualizes ultra-long, sequence-labeled DNA molecules, to uncover the structural complexity of the cancer genome and assess the clinical relevance of chromothripsis in CLL. Comprehensive cytogenetic analysis was conducted on a 43-year-old male diagnosed with chronic lymphocytic leukemia. Karyotyping revealed a complex rearrangement: 46,XY,der(3)t(3;13)(p2?3;q14.3),der(4)t(?3;4)(p23;p16),add(11)(p13),del(13)(q14)[12]/46,sl,del(11)(q2?2.2q23.3)[6]. FISH analysis further identified the loss of ATM and a partial deletion of the D13S319 locus. OGM analysis performed on bone marrow revealed a complex genotype including chromothripsis of chromosome 13, and structural rearrangements involving chromosomes 3, 4, and 11. Additionally, multiple intrachromosomal translocations and interstitial microdeletions of chromosome 13 were identified. The resolution of these aberrations has been significantly enhanced with examples including: ogm[GRCh38] t(3;13)(p26.3;q33.1)(2,706,645~2,721,113;103,142,901~103,154,241][VAF0.45], ogm[GRCh38] t(4;13)(p15.31;q32.1)(20,869,721~20,907,265;96,617,837~96,630,317)[VAF0.42],. In conclusion, OGM revealed the intricate structural alterations of the cancer genome. The high resolution provided by OGM could facilitate the discovery of oncogenic mechanisms, novel fusion genes, prognostic markers, and potential therapeutic targets. OGM serves as a powerful tool for revisiting CLL disease classification by offering deeper insights into complex genomic rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4656aef9b3e1a0de1b2f5258ea921c03f0e3137" target='_blank'>
              Case Report: Unraveling complex genomic alterations in a case of chronic lymphocytic leukemia using optical genome mapping
              </a>
            </td>
          <td>
            Leila Youssefian, Trilochan Sahoo, Jia-Chi Wang
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Alternative lengthening of telomeres (ALT) is a telomerase-independent telomere maintenance mechanism observed in 15% of human cancers. A hallmark of ALT cancers is the presence of C-circles, circular single-stranded DNAs (ssDNAs) enriched with cytosine-rich telomere (C-rich, CCCTAA) sequences. G-circles, containing guanosine-rich telomere (G-rich, GGGTTA) ssDNAs, also exist but are much less abundant. Recent studies indicate that excessive displacement of Okazaki fragments during lagging-strand synthesis is a unique feature of ALT telomeres and responsible for generating C-circles/C-rich ssDNAs. However, the distinct characteristics of C-circles compared to G-circles remain unclear. Here, we demonstrate that co-deficiency of the DNA translocases SMARCAL1 and FANCM leads to abundant generation of G-circle/G-rich ssDNAs. These G-rich ssDNAs mainly exist in linear form, ranging in size from 500 to 3000 nucleotides, which differs significantly from the structure and size of C-circle/C-rich ssDNAs. Mechanistically, both C-rich and G-rich ssDNAs originate from BLM/POLD-mediated excessive strand displacement; however, they differ in their origins and initiation mechanisms. Specifically, C-rich ssDNAs arise from lagging daughter strands initiated by the CST complex, whereas G-rich ssDNAs originate from leading daughter strands through RAD51-dependent G-strand synthesis. Our findings propose two distinct mechanisms for generating two different extrachromosomal telomere DNAs, C- and G-circles, during ALT-mediated telomere elongation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12e4731f01013d1812909194114a8c4a4ce7f91c" target='_blank'>
              Distinct mechanisms underlying extrachromosomal telomere DNA generation in ALT cancers
              </a>
            </td>
          <td>
            Junyeop Lee, Eric J. Sohn, Jina Lee, Ashley Y Cheng, Angelo Taglialatela, Alberto Ciccia, Jaewon Min
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Noncoding RNAs play pivotal roles in tumorigenesis and cancer progression. Recent evidence has identified vault RNAs (vtRNAs) as critical regulators of cellular homeostasis. The human genome encodes four vtRNA paralogs, which are differentially expressed in cancer tissues and contribute to tumor development. The best studied vtRNA1-1 is involved in regulating apoptosis resistance, autophagy, lysosomal biogenesis, and drug resistance. Here, we present the first comprehensive characterization of vtRNA1-2 using a knockout hepatocellular carcinoma (HCC) cell line. Loss of vtRNA1-2 impaired cancer cell viability and proliferation by modulating mitogen-activated protein kinase signaling. Additionally, vtRNA1-2-deficient cells exhibited reduced motility and a decreased invasive potential. Unlike vtRNA1-1, vtRNA1-2 did not influence autophagy or lysosomal activity. Instead, vtRNA1-2 is implicated in the regulation of angiogenesis, a key process in tumor progression. VTRNA1-2-promoter hypomethylation is correlated with chromatin accessibility in liver cancer samples and we uncovered an association between promoter methylation and key patient clinical conditions as registered in the TCGA metadata. These findings highlight a distinct oncogenic role for vtRNA1-2 in HCC and suggest that it may serve as a potential therapeutic target. Our study underscores the functional divergence among vtRNA paralogs, supporting the concept that each exerts unique biological effects rather than acting as redundant molecular entities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a7d53d484fb924c98ee71e879af767b3e2a99b9" target='_blank'>
              Regulation of angiogenesis and cancer cell proliferation by human vault RNA1-2
              </a>
            </td>
          <td>
            Stefano Gallo, Anastasiia Suspitsyna, D. Sánchez-Taltavull, R. Fort, M. A. Duhagon, Deborah Stroka, Norbert Polacek
          </td>
          <td>2025-08-27</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Herpesviruses are ubiquitous pathogens that cause a wide range of disease. Upon nuclear entry, their genomes associate with histones and chromatin modifying enzymes that regulate the progression of viral transcription and outcome of infection. While the composition and modification of viral chromatin has been extensively studied on bulk populations of infected cells by chromatin immunoprecipitation, this key regulatory process remains poorly defined at single-genome resolution. Here we use high-resolution quantitative imaging to investigate the spatial proximity of canonical and variant histones at individual Herpes Simplex Virus 1 (HSV-1) genomes within the first 90 minutes of infection. We identify significant population heterogeneity in the stable enrichment and spatial proximity of canonical histones (H2A, H2B, H3.1) at viral DNA (vDNA) relative to established promyelocytic leukaemia nuclear body (PML-NB) host factors that are actively recruited to viral genomes upon nuclear entry. We show the replication-independent histone H3.3/H4 chaperone Daxx to cooperate with PML to mediate the enrichment and spatial localization of variant histone H3.3 at vDNA and limit the rate of HSV-1 genome decompaction. This host response is counteracted by the viral ubiquitin ligase ICP0, which degrades PML to disperse Daxx and variant histone H3.3 from vDNA to stimulate the progression of viral immediate-early (IE) transcription, genome expansion, and onset of HSV-1 replication. Our data support a model of intermediate and sequential histone assembly initiated by Daxx that limits the rate of HSV-1 genome decompaction independently of the stable enrichment of histones H2A and H2B at vDNA required to facilitate canonical nucleosome assembly. We identify HSV-1 genome decompaction upon nuclear infection to play a key role in the initiation and functional outcome of HSV-1 lytic infection, findings pertinent to the transcriptional regulation of many nuclear replicating herpesvirus pathogens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/617d8ae459ea016f8dca2b2644f05eeab5725c80" target='_blank'>
              Daxx mediated histone H3.3 deposition on HSV-1 DNA restricts genome decompaction and the progression of immediate-early transcription
              </a>
            </td>
          <td>
            Ashley P. E. Roberts, Anne Orr, V. Iliev, Lauren Orr, S. McFarlane, Zhousiyu Yang, I. Epifano, C. Loney, Mila Collados Rodriguez, Anna. R Cliffe, Kristen L Conn, Chris Boutell
          </td>
          <td>2025-08-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="SUMMARY Ovarian high-grade serous carcinoma (HGSC) is characterized by extensive intra-peritoneal dissemination and tumor heterogeneity. In the metastatic cascade, tumors utilize several transcriptional programs to translocate and survive in distant tissues. Here, we analyzed multi-modal, real-world data from 350 tumor samples across 160 patients with HGSC to identify transcriptional programs that drive intra-peritoneal metastasis and heterogeneity. We identified nine transcriptional programs, including those regulating epithelial-mesenchymal transition and immune modulation and cytoskeletal reorganization, which shape distinct metastatic trajectories to solid and ascitic environments and are associated to treatment response. Our results reveal pronounced intra-patient transcriptional heterogeneity, which in some cases surpassed inter-patient heterogeneity, highlighting the importance of multi-site sampling for accurate prognostication and combinatorial treatments in HGSC. Our extensive characterization offers novel insights into intra-peritoneal metastasis with significant prognostic implications, reveals histomorphological biomarkers for patient stratification and paves the way for innovative therapeutic strategies aimed at impairing cancer cell adaptability and limiting metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32c6dc18825437bb12a5a64bda09f91c2c6d329" target='_blank'>
              Multi-modal characterization of transcriptional programs that drive metastatic cascades to solid sites and ascites in ovarian cancer
              </a>
            </td>
          <td>
            Kaiyang Zhang, Essi Kahelin, G. Marchi, Oskari Lehtonen, S. Salloum, K. Lavikka, Yilin Li, Felix Dietlein, A. Lahtinen, J. Oikkonen, S. Hietanen, J. Hynninen, Antti Häkkinen, A. Virtanen, S. Hautaniemi
          </td>
          <td>2025-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Abstract Chain-terminating nucleoside analogs (CTNAs) are frequently used as antiviral and anticancer agents. CTNAs are incorporated into the end of nascent DNA during DNA synthesis and inhibit subsequent DNA polymerization, thereby restricting the proliferation of viruses and cancer cells. Alovudine, a thymidine analog, exerts chain-termination effects and is used as an antiviral drug. However, the mechanisms underlying cellular tolerance to alovudine have not been fully elucidated. Here, we show that flap endonuclease-1 (Fen1) and p53 binding protein-1 (53BP1) counteractively mitigate the chain-terminating effect of alovudine. We found that the cells deficient in Fen1 (FEN1−/−) showed stronger chain-terminating effects of alovudine with augmented DNA damage than wild-type cells, leading to their alovudine hypersensitivity. Moreover, we found that the Okazaki fragment maturation was critically slowed in FEN1−/− cells when cells were challenged with alovudine. Remarkably, the loss of 53BP1 rescued these phenotypes. We found that 53BP1 formed sub-nuclear foci upon alovudine treatment and these 53BP1 foci were critically increased in FEN1−/− cells, indicating that Fen1 suppresses alovudine-mediated toxic 53BP1 subnuclear foci thereby promoting Okazaki fragment maturation and suppressing DNA damage. In this study, we uncovered the previously unappreciated role of Fen1 in the suppression of toxic action of 53BP1 at alovudine-incorporated Okazaki fragment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4219df598b7bcfd3d493fa90ebbd81e6530ccd18" target='_blank'>
              The flap endonuclease-1 promotes cellular tolerance to a chain-terminating nucleoside analog, alovudine, by counteracting the toxic effect of 53BP1
              </a>
            </td>
          <td>
            Md. Bayejid Hosen, Ryotaro Kawasumi, Kouji Hirota
          </td>
          <td>2025-07-18</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="DNA polymerase epsilon (POLe) is the leading strand replicative polymerase. POLe mutations located primarily in the proofreading domain cause replication errors and increase mutation burden in cancer cells. Consequently, POLe has been classified as a cancer driver gene. Certain POLe frameshift mutations that affect the proofreading domain are purified in cancer cells, but point mutations in other domains have also been reported. Here we use an artificial intelligence algorithm to determine what other mutations co-occur with POLe mutations in colorectal cancers. We partitioned POLe mutations into driver, passenger, and WT (no mutation), then assessed mutations in other genes in these three groups. We found that a driver POLe mutation is not likely to associate with driver mutations in other genes. Thus, driver mutations in colorectal cancers appear to purify in a manner that is independent of POLe. Mutations that affect POLe function do not necessarily increase the frequency of driver mutations in other genes. Structural analysis shows that many POLe driver mutations affect coordination of the Mg2+ ion in the active site. Our data show that the accumulation of colorectal cancer mutations is driven by complex factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ef4c4fe770c28fab273ae3b812783f54ae78686" target='_blank'>
              Comprehensive Analysis of Human Colorectal Cancers Harboring Polymerase Epsilon Mutations
              </a>
            </td>
          <td>
            Louis M. Gibson, Phanithan Konda, Hunter J. Bliss, Devi D. Nelakurti, Golrokh Mirzaei, Renee A. Bouley, Jing J. Wang, Ruben C. Petreaca
          </td>
          <td>2025-07-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Cytosine DNA methylation patterns vary widely across eukaryotes, with its ancestral roles being understood to have included both transposable element (TE) silencing and host gene regulation. To further explore these claims, in this study, we reevaluate the evolutionary origins of DNA methyltransferases and characterize the roles of cytosine methylation on underexplored lineages, including the amoebozoan Acanthamoeba castellanii, the glaucophyte Cyanophora paradoxa, and the heterolobosean Naegleria gruberi. Our analysis of DNA methyltransferase evolution reveals a rich ancestral eukaryotic repertoire, with several eukaryotic lineages likely subsequently acquiring enzymes through lateral gene transfer (LGT). In the three species examined, DNA methylation is enriched on young TEs and silenced genes, suggesting an ancestral repressive function, without the transcription-linked gene body methylation of plants and animals. Consistent with this link with silencing, methylated genomic regions co-localize with heterochromatin marks, including H3K9me3 and H3K27me3. Notably, the closest homologs of many of the silenced, methylated genes in diverse eukaryotes belong to viruses, including giant viruses. Given the widespread occurrence of this pattern across diverse eukaryotic groups, we propose that cytosine methylation was a silencing mechanism originally acquired from bacterial donors, which was used to mitigate the expression of both transposable and viral elements, and that this function may persist in creating a permissive atmosphere for LGT in diverse eukaryotic lineages. These findings further highlight the importance of epigenetic information to annotate eukaryotic genomes, as it helps delimit potentially adaptive LGTs from silenced parasitic elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48579c32d97033a0fd54ac2bedb8cb3bebb700ed" target='_blank'>
              Repressive Cytosine Methylation is a Marker of Viral Gene Transfer Across Divergent Eukaryotes
              </a>
            </td>
          <td>
            Luke A. Sarre, Giselle Azucena Gastellou Peralta, Pedro Romero Charria, Vladimir Ovchinnikov, Alex de Mendoza
          </td>
          <td>2025-07-25</td>
          <td>Molecular Biology and Evolution</td>
          <td>1</td>
          <td>22</td>
        </tr>

        <tr id="Cutaneous melanoma (CM), with a continuously rising incidence worldwide, represents the most aggressive type of skin cancer, and it leads to the majority of skin cancer-related deaths. Approximately 50% of CM carry the activating BRAFV600 mutation and, although BRAF inhibitors have demonstrated clinical efficacy, most patients often develop early resistance to treatment. Aberrant expression of non-coding RNAs (ncRNAs), which represent less than 2% of the entire transcriptome, has been implicated in CM development and progression. By using BRAFV600-mutant CM in vitro and in vivo models, we have recently demonstrated that the loss of Spry1 expression impairs BRAFV600-mutant CM progression. Therefore, the extensive long and small ncRNA datasets generated in this study might represent a valuable resource for the characterization of their roles in BRAFV600-mutant CM initiation and progression upon Spry1 loss, thus providing a comprehensive resource to support future studies on BRAFV600-mutant CM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/710c46752f6a68d20467ff6b95252e5ee4402a37" target='_blank'>
              Non-coding RNA profiling in BRAFV600E-mutant cutaneous melanoma before and after Spry1 depletion
              </a>
            </td>
          <td>
            J. Lamberti, Domenico Memoli, Barbara Montico, Francesco Silvestro, R. Guerrieri, F. Colizzi, Alessandro Weisz, A. Salvati, E. Fratta, G. Nassa
          </td>
          <td>2025-09-02</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cellular senescence is accompanied by extensive genomic reorganization, such as senescence-associated heterochromatin foci and expanded interchromatin compartments, to ultimately affect gene expression. Here, we demonstrate that chromatin structural changes in senescent cells drive significant alterations in the phase behavior and motility of paraspeckles, a type of interchromatin compartment condensate. We observe increased numbers, size, and elongation of paraspeckles harboring NONO and NEAT1_2, driven by elevated levels of those components, consistent with the micellization model of longitudinal growth rather than condensate coalescence. Enhanced paraspeckle motility is associated with HP1α-mediated heterochromatin condensation and interchromatin expansion found in cellular senescence. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03757-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c589595d2d429fd53faabd150c9bf96e091d470b" target='_blank'>
              Imbalanced chromatin distribution in cellular senescence specifies paraspeckle dynamics
              </a>
            </td>
          <td>
            Joonwoo Lee, Jinmi Choi, Jeongeun Park, Seonduk Lee, Eun-Jung Cho, Youngdae Gwon
          </td>
          <td>2025-09-02</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Mutations in numerous genes contribute to human cancer, with different oncogenic lesions prevalent in different cancer types. However, the malignant phenotype is simple, characterized by unrestricted cell growth, invasion, and often metastasis. One possible hypothesis explaining this dichotomy is that cancer genes regulate common targets, which then function as master regulators of essential cancer phenotypes. To identify mechanisms that drive the most fundamental feature shared by all tumors-unrestricted cell proliferation-we used a multiomic approach, which identified translation and ribosome biogenesis as common targets of major oncogenic pathways across cancer types. Proteomic analysis of tumors and functional studies of cell cultures established nucleolar and coiled-body phosphoprotein 1 as a key node, whose convergent regulation, both transcriptionally and posttranslationally, is critical for tumor cell proliferation. Our results indicate that lineage-specific oncogenic pathways regulate the same set of targets for growth control, revealing key downstream nodes that could be targeted for therapy or chemoprevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e15820af1f4a4174736d844aaa4b18b01cd6fd" target='_blank'>
              Diverse oncogenes use common mechanisms to drive growth of major forms of human cancer.
              </a>
            </td>
          <td>
            O. Kauko, Mikko Turunen, Päivi Pihlajamaa, Antti Häkkinen, R. Queiroz, M. Pääkkönen, S. Ventelä, Massimilliano Gaetani, Susanna L. Lundström, Antonio Murgia, Biswajyoti Sahu, J. Routila, Gong-Hong Wei, H. Irjala, J.L. Griffin, Kathryn S. Lilley, Teemu Kivioja, S. Hautaniemi, J. Taipale
          </td>
          <td>2025-08-22</td>
          <td>Science advances</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3da656da84cf3e9dc1144dda4829e9c83aef2891" target='_blank'>
              CHAMP1 Complex Promotes Heterochromatin Assembly and Reduces Replication Stress.
              </a>
            </td>
          <td>
            Feng Li, Amira Elbakry, Felix Y Zhou, Tianpeng Zhang, Ramya Ravindranathan, Huy Nguyen, Aleem Syed, Lifang Sun, S. Mukkavalli, Roger A. Greenberg, Alan D. D’Andrea
          </td>
          <td>2025-08-05</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Transposable elements (TEs) represent a threat to genome integrity due to their proliferative capacity. Eukaryotic cells silence TEs through different molecular mechanisms, including the deposition of repressive histone marks. Previous studies have shown that TE repressive marks can spread to neighboring sequences. However, evidence for this spreading leading to nearby gene silencing remains limited. Similarly, whether TEs induce changes in the enrichment of active histone marks genome-wide, and the potential impact on gene expression have not been widely studied. Results In this work, we perform a comprehensive study of the epigenetic effects of 2235 TEs and their potential effects on nearby gene expression on Drosophila melanogaster head, gut, and ovary. While most TEs (816) induce the enrichment of the H3K9me3 repressive mark, with stronger epigenetic effects in the ovary, a substantial number (345 TEs) induce the enrichment of the H3K27ac active mark, particularly in the gut. We find that 70% of the H3K9me3 enriched TEs associated with expression changes downregulate the nearby gene, and 50% of the H3K27ac enriched TEs associated with expression changes lead to gene upregulation. These changes in expression affect specific regulatory networks only in the head. Furthermore, TE epigenetic effects on gene expression are genomic context dependent. Finally, we find that 221 TEs also affect gene expression by disrupting regions enriched for histone marks. Conclusions Overall, our results show that TEs contribute to the generation of regulatory novelty through epigenetic changes, with these epigenetic effects not restricted to gene silencing and being genomic context dependent. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03705-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75c9f4509d1e24d95361876bc4f99d2975a0de9c" target='_blank'>
              The epigenetics effects of transposable elements are genomic context dependent and not restricted to gene silencing in Drosophila
              </a>
            </td>
          <td>
            Marta Coronado-Zamora, Josefa González
          </td>
          <td>2025-08-18</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1110fcfacea74bd5588618a3f029d0c49019fbe" target='_blank'>
              Mechanisms and genomic implications of break-induced replication.
              </a>
            </td>
          <td>
            Adel Atari, Haoyang Jiang, Roger A. Greenberg
          </td>
          <td>2025-08-22</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The mechanisms through which mutations in splicing factor genes drive clonal hematopoiesis (CH) and myeloid malignancies, and their close association with advanced age, remain poorly understood. Here we show that telomere maintenance plays an important role in this phenomenon. First, by studying 454,098 UK Biobank participants, we find that, unlike most CH subtypes, splicing-factor-mutant CH is more common in those with shorter genetically predicted telomeres, as is CH with mutations in PPM1D and the TERT gene promoter. We go on to show that telomere attrition becomes an instrument for clonal selection in advanced age, with splicing factor mutations ‘rescuing’ HSCs from critical telomere shortening. Our findings expose the lifelong influence of telomere maintenance on hematopoiesis and identify a potential shared mechanism through which different splicing factor mutations drive leukemogenesis. Understanding the mechanistic basis of these observations can open new therapeutic avenues against splicing-factor-mutant CH and hematological or other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4518f74e6ec990a2a9612e62e836380371ad0a19" target='_blank'>
              Telomere attrition becomes an instrument for clonal selection in aging hematopoiesis and leukemogenesis
              </a>
            </td>
          <td>
            Matthew A. McLoughlin, Sruthi Cheloor Kovilakam, William G. Dunn, Muxin Gu, Jake Tobin, Y. Pershad, N. Williams, Daniel Leongamornlert, Kevin Dawson, Laura Bond, Ludovica Marando, Sean Wen, Rachael Wilson, Giampiero Valenzano, Vasiliki Symeonidou, Justyna Rak, Aristi Damaskou, M. Gozdecka, Xiaoxuan Liu, Clea Bárcena, Josep Nomdedeu, Paul Costeas, Ioannis D. Dimitriou, E. Fiorillo, V. Orrù, José Guilherme de Almeida, Thomas McKerrell, Matthew Cullen, I. Mohorianu, T. Foukaneli, Alan J. Warren, Chi Wong, George Follows, A. Godfrey, Emma J Gudgin, Francesco Cucca, Eoin McKinney, E. J. Baxter, M. Gerstung, J. Mitchell, D. Wiseman, Alexander G. Bick, M. Fabre, Pedro M Quiros, J. Nangalia, Siddhartha Kar, George S. Vassiliou
          </td>
          <td>2025-08-28</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="The HOXA gene locus coordinates body patterning, hematopoiesis, and differentiation. While studying blood phenotype-associated variation within the HOXA locus, we identified a genetic variant, rs17437411, associated with globally reduced blood counts, protection from blood cancers, and variation in anthropometric phenotypes. We find that this variant disrupts the activity of a previously unstudied antisense long non-coding RNA (lncRNA) located between HOXA7 and HOXA9, which we have named HOTSCRAMBL. The HOTSCRAMBL variant disrupts lncRNA function and reduces human hematopoietic stem cell (HSC) self-renewal. Mechanistically, HOTSCRAMBL enables appropriate expression and splicing of HOXA genes in HSCs, most notably HOXA9, in an SRSF2-dependent manner. Given the critical role of HOXA gene expression in some blood cancers, we also demonstrate that HOTSCRAMBL variation or deletion compromises HOXA-dependent acute myeloid leukemias. Collectively, we show how insights from human genetic variation can uncover critical regulatory processes required for effective developmental gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/854c6d558bc4abf3b6a29b21c9a18aa5b2bee350" target='_blank'>
              Genetic variation reveals a homeotic long noncoding RNA that modulates human hematopoietic stem cells
              </a>
            </td>
          <td>
            Peng Lyu, Gaurav Agarwal, Chun-Jie Guo, Tianyi Ye, Chen Weng, Mateusz Antoszewski, Samantha Joubran, Alexis Caulier, Michael Poeschla, V. G. Sankaran
          </td>
          <td>2025-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Lethal prostate cancer (PCa) has passed through at least two evolutionary bottlenecks: acquisition of metastatic potential and development of castration resistance. A better understanding of how this impacts genetic heterogeneity across metastatic sites is needed to develop strategies to block metastatic spread and overcome resistance. By leveraging deep whole-exome sequencing of 93 tumors from 26 patients, we examined patterns of metastatic dissemination and clonal evolution of PCa. Phylogenetic reconstruction and mathematical modeling enabled quantification of the number of mutations and clones in the cancer ecosystem and characterization of each patient's disease as an evolutionary process. While mutations of the earliest pathogenic genetic drivers of PCa were truncal, most other likely passenger mutations, copy-number alterations, and clones arose after the cancer had spread and were confined to individual metastatic sites due to polyclonal and polyphyletic seeding. Single-tissue sequencing tended to overestimate mutation clonality and, apart from truncal drivers, underestimate mutation rates for both individual patients and cohorts. This study highlights the independent evolution of metastatic lesions, which has implications for diagnostic and targeted therapy strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f198ff2332798cb036a19b4692ee525e77c812c" target='_blank'>
              Divergent Clonal Evolution and Early Dissemination Promote Genetic Heterogeneity of Metastases in Castration Resistant Prostate Cancer.
              </a>
            </td>
          <td>
            Noshad Hosseini, R. Mannan, Ryan J. Rebernick, Fengyun Su, Rui Wang, Xuhong Cao, Ana Lako, Dattatreya Mellacheruvu, Jing Hu, J. Alumkal, Zachery R Reichert, Rohit Malik, Rohit Mehra, A. Chinnaiyan, M. Cieslik
          </td>
          <td>2025-08-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>169</td>
        </tr>

        <tr id="Background Cancer diagnosis using cell-free DNA (cfDNA) has the potential to improve treatment and survival but has several technical limitations. Methods In this study, we developed a prediction model based on neomers, DNA sequences 13–17 nucleotides in length that are predominantly absent from the genomes of healthy individuals and are created by tumor-associated mutations. Results We show that neomer-based classifiers can accurately detect cancer, including early stages, and distinguish subtypes and features. Analysis of 2577 cancer genomes from 21 cancer types shows that neomers can distinguish tumor types with higher accuracy than state-of-the-art methods. Generation and analysis of 465 cfDNA whole-genome sequences demonstrates that neomers can precisely detect lung and ovarian cancer, including early stages, with an area under the curve ranging from 0.89 to 0.94. By testing various promoters or over 9000 candidate enhancer sequences with massively parallel reporter assays, we show that neomers can identify cancer-associated mutations that alter regulatory activity. Conclusions Combined, our results identify a sensitive, specific, and simple cancer diagnostic tool that can also identify cancer-associated mutations in gene regulatory elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43a304be17c87fe26db851c5f7942035d92028e5" target='_blank'>
              Leveraging sequences missing from the human genome to diagnose cancer
              </a>
            </td>
          <td>
            Ilias Georgakopoulos-Soares, Ofer Yizhar-Barnea, I. Mouratidis, Candace S Y Chan, Michail Patsakis, Akshatha Nayak, Rachael Bradley, Mayank Mahajan, Jasmine Sims, Dianne Laboy Cintrón, Ryder L. Easterlin, Julia S Kim, Emmalyn Chen, Geovanni Pineda, Guillermo E. Parada, J. Witte, C. Maher, Felix Y. Feng, Ioannis A. Vathiotis, Nikolaos K. Syrigos, E. Panagiotou, Andriani Charpidou, Konstantinos N. Syrigos, Jocelyn S Chapman, M. Kvale, M. Hemberg, N. Ahituv
          </td>
          <td>2025-08-21</td>
          <td>Communications Medicine</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Hepatoblastoma is the most common pediatric liver cancer, with the fastest rising incidence among childhood malignancies. Early genomic studies revealed that hepatoblastoma has the lowest mutational burden of any human cancer, however, recent advances in single-cell RNA-seq, multiomics, spatial transcriptomics, and functional genomics screenings have revealed substantial complexity. Diverse cellular subpopulations, divergent WNT signaling, key developmental pathways, and intricate interactions between the tumor cells and tumor immune microenvironment (TME) collectively shape tumor heterogeneity, disease progression, therapeutic responses, and genetic dependencies. Mosaic embryonic loss of heterozygosity (LOH) at chromosome 11p15.5 may be a hepatoblastoma-initiating event, as clonal expansion of 11p15.5 LOH occurs in adjacent normal liver tissue. A cholangiocyte-like subpopulation expresses FGF19, in a SOX4-dependent, paracrine manner, to drive the proliferation of neighboring embryonal hepatoblastoma cells. WNT-signaling dependent MDK promotes the immunosuppressive TME, which impairs immune cell infiltration. The TME may also be driven by islands of erythroblasts, which influence treatment resistance. Plasticity driven by changes in chromatin accessibility enables differentiation transition between hepatocytic and liver progenitor cell types, which is associated with treatment resistance. Here, we review recent findings in pediatric hepatoblastoma cells, tumor-associated cell types, and genetic dependencies that will serve to advance hepatoblastoma therapy. Clinical trial number: Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02405-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/361b9ebd3fa588c3f28e25ba36c91c45f4b07fb6" target='_blank'>
              From preneoplastic lesion to heterogenous tumor: recent insights into hepatoblastoma biology and therapeutic opportunities
              </a>
            </td>
          <td>
            Jun Yang, A. Davidoff, Andrew J. Murphy
          </td>
          <td>2025-07-19</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a07119c6c29989e52a1007b94a87a49c0954c86a" target='_blank'>
              Phase separation of ERCC6L2-CtIP regulates the extent of DNA end resection.
              </a>
            </td>
          <td>
            Yixin Yin, Jinlong Lin, Xiaoxia Cai, Runcong Nie, Jiliang Lin, Shuting Li, Zhicheng Xiang, Yihong Ling, Yiyang Zhang, Jie Zhou, Jiewei Chen, Wenping Lin, Jinling Duan, Xueyi Zheng, Dan Xie, Muyan Cai
          </td>
          <td>2025-09-05</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Ovarian cancer (OV) has the highest mortality rate among gynecological cancers and shows varied responses to chemotherapy combined with PARP inhibitors based on homologous recombination deficiency (HRD) subtypes. Methods This study enrolled 143 Chinese OV patients to determine the HRD score grouping threshold using genomic features, dividing patients into HRD-high and HRD-low groups. Multi-omics sequencing was conducted on 70 patients receiving adjuvant chemotherapy with PARP inhibitors. Results In this study, TP53 mutations enriched in the HRD-high group, while ARID1A, PIK3CA, and PTEN mutations were more common in the HRD-low group. HRD-high patients exhibited stronger immune activation, including elevated STAT1 expression, HLA signatures, and increased M1 macrophage infiltration, correlating with better prognosis. Additionally, peripheral blood analysis revealed higher bMSI and maxVAF levels in HRD-low patients compared to HRD-high patients, suggesting ctDNA as a potential tool for dynamic monitoring post-treatment. Conclusions This study identified distinct molecular and immune profiles between HRD subgroups in Chinese ovarian cancer patients. Patients with HRD-high and STAT1 expression ≥ 74 suggests PARPi benefit. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00806-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5abcfeae4983ac959a97accf690879bf81c76cc2" target='_blank'>
              Genomic and transcriptomic data reveal molecular differences between homologous recombination deficiency subgroups in Chinese ovarian cancer patients
              </a>
            </td>
          <td>
            Hongxia Wang, Wenhong Zhao, Wenhao Zhou, Na Wang, Yijie Li, Kaiyun Qin, Jingde Jia, Jiaqian Wang, Congcong Song, Yu Yu, Fenghua Zhang, Xu Cui, Lanlan Zhao, Haitao Luo, Zhengmao Zhang
          </td>
          <td>2025-08-31</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Tumor mutational burden (TMB) has emerged as a potential surrogate for neoantigen load and an indicator of immune checkpoint (IC)-blockade response; however, its precise significance in breast cancer (BC) is not fully understood. Here, we comprehensively characterized the genomic repertoire of BCs with a TMB ≥ 10 mut/Mb (TMB-high [n = 527]) to identify putative predictors of importance. The predominant mutational signature was apolipoprotein B mRNA-editing enzyme catalytic polypeptide (APOBEC) in 64.7% of tumors. TMB-high BCs were enriched in KMT2C, ARID1A, PTEN, NF1, and RB1 alterations, which are associated with APOBEC mutagenesis. Further identified were loss-of-function ARID1A and PTEN alterations, which are linked to immune cell exclusion. ESR1 p.E380Q prevailed among all ESR1 hotspot mutations, supporting APOBEC-mediated effects. Finally, mutations in DNA damage response and repair genes were seen at a higher frequency than in non-TMB-high BCs. These findings provide justification for exploring combined pharmacologic inhibition to improve IC-based efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e8980a3a4147f92cf6718bb89169af5da92bd71" target='_blank'>
              Genomic characterization of tumor mutational burden-high breast carcinomas
              </a>
            </td>
          <td>
            Theodore Vougiouklakis, Chad M. Vanderbilt, Satshil Rana, A. Mohanty, F. Pareja, E. Brogi, Christopher J. Schwartz, M. Arcila, Marc Ladanyi, Hanna Y Wen, D. Ross
          </td>
          <td>2025-08-08</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Throughout the last century, aneuploidy has been cemented as a hallmark of cancer. Although the association of aneuploidy with tumorigenesis has been well established, the role of these genetic imbalances in tumor formation has only recently begun to be elucidated. Advancements in genomics have revealed the complexity and context dependence of the effect of aneuploidy on cancer growth, while developments in genetic editing have allowed for proper modeling of specific aneuploidies. In this review, we discuss the key factors to consider when studying the role of aneuploidy in cancer and the tools that are available to do so. We then highlight recent studies that establish phenotypic contributions of aneuploidy to tumorigenicity. In particular, we highlight how general aneuploidy and chromosomal instability affect the tumor microenvironment and how specific chromosomal alterations, including the loss of chromosome 9p and the gain of chromosomes 8q and 1q, influence tumor behavior and therapeutic responses. Finally, we emphasize the potential of targeting aneuploidy-induced vulnerabilities to improve cancer treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b1804dd6a9ac26725f995589c82459ba79d112" target='_blank'>
              Modeling and targeting general and chromosome-specific aneuploidy in cancer.
              </a>
            </td>
          <td>
            Aleah Goldberg, Maria Trifas, Teresa Davoli
          </td>
          <td>2025-07-28</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Transposable elements (TEs) have hijacked cellular machineries to replicate and spread throughout host genomes. TEs now make up a significant portion of eukaryotic genomes and play notable roles in genomic evolution, driving both speciation and providing raw material for genetic innovation. Barbara McClintock's pioneering work on these “jumping genes” laid the foundation for modern TE research; however, her paradigm‐shifting theories in which TEs act as “controlling elements” were initially rejected due to the long‐held belief that TEs were “junk” or parasitic DNA elements. Historically, the highly repetitive nature of TEs made it challenging to both identify and investigate functions. However, recent advances in genomics have greatly accelerated our understanding of TEs. Despite their potential to cause insertional mutagenesis and disease, many transposable elements have been co‐opted by host genomes to contribute to gene regulation and development. In contrast to protein‐coding genes that typically begin their journey as DNA, are transcribed into RNA, and reach their ultimate functional form as proteins, TEs can function as cis‐regulatory DNA, functional RNA, and in rare cases, domesticated proteins and fusion events between TE and host genes. Driven by rapidly advancing technologies, the roles of TEs in both development and disease are being uncovered faster than ever, making current and future work an exciting continuation of Barbara McClintock's groundbreaking legacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/831a0252c52d2b92a8d6f28251facae6b2cf9eff" target='_blank'>
              From Junk DNA to Genomic Treasure: Impacts of Transposable Element DNA, RNA, and Protein in Mammalian Development and Disease
              </a>
            </td>
          <td>
            Ten D. Li, Katelyn Toohill, A. Modzelewski
          </td>
          <td>2025-07-01</td>
          <td>Wiley Interdisciplinary Reviews. RNA</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Maintaining cellular ploidy is critical for normal physiological processes, although gains in ploidy are frequently observed during development, tissue regeneration, and metabolism, and potentially contribute to aneuploidy, thereby promoting tumor evolution. Although numerous computational tools have been developed to estimate cellular ploidy from whole-genome sequencing (WGS) data at bulk or single-cell resolution, to the knowledge, no systematic comparison of their performance has been conducted. Here, a benchmarking study is presented of 11 methods for bulk WGS and 8 methods for single-cell WGS data, utilizing both experimental and simulated datasets derived from diploid cells mixed with aneuploid or polyploid cells. For bulk WGS tools, their performance is evaluated in estimating tumor purity and ploidy, as well as the influence of preprocessing steps, somatic mutation callers, purity, sequencing platforms, and depths. It is found that PURPLE outperforms other methods when tumor purity exceeded 30%, regardless of sequencing coverage or platform. However, all existing tools performed poorly applied to euploid samples or long-read sequencing data. For single-cell WGS tools, their ploidy detection accuracy is assessed, and SeCNV is identified as the top-performing method. These findings provide valuable guidance for future research on ploidy analysis and ongoing improvements in computational tools for single-cell sequencing data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffa9982ba9f6208840a8a8f142151d3325d86992" target='_blank'>
              Benchmarking Ploidy Estimation Methods for Bulk and Single-Cell Whole Genome Sequencing.
              </a>
            </td>
          <td>
            Yawei Song, Zilv Mei, Qijie Zheng, Qingqing Yuan, Yu Liang, Jiaqi Gao, Lang Zhou, Shuheng Wu, Wei Wu
          </td>
          <td>2025-09-08</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="ABSTRACT Background In various malignancies, the co‐occurrence of BRAF V600E and TERT promoter mutations (C228T and C250T) has been associated with tumor aggressiveness and poor prognosis. However, the presence of these TERT promoter mutations in Langerhans cell histiocytosis (LCH) remains unexplored. Methods We investigated the prevalence of TERT promoter C228T and C250T mutations in 40 formalin‐fixed, paraffin‐embedded (FFPE) LCH samples positive for BRAF V600E. A nested PCR approach followed by Sanger sequencing, complemented by NGS, was used for mutation screening. Results The variant allele frequency (VAF) of BRAF V600E in the analyzed samples ranged from 0.1% to 33.6%, with a median of 17%, and 33 of 40 successfully amplified LCH samples did not harbor TERT C228T or C250T mutations. Conclusion Our findings indicate that LCH BRAF V600E‐positive samples lack the TERT promoter C228T and C250T hotspot mutations. This suggests that TERT promoter mutations may not play a significant role in LCH pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/901b319d68cfec71eec7b38f8210df81809dc882" target='_blank'>
              TERT Promoter C228T and C250T Hotspot Mutations Are Absent in BRAF V600E‐Positive Langerhans Cell Histiocytosis
              </a>
            </td>
          <td>
            Abdou Malik Da Silva, Z. Hélias‐Rodzewicz, I. Ungureanu, J. Emile
          </td>
          <td>2025-07-29</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The impact of ionizing radiation (IR) with induction of various DNA damage is based not only on genetic but also on epigenetic effects. Epigenetic modifications determine the chromatin structure and DNA accessibility, thereby regulating cellular functions through the expression of individual genes or entire groups of genes. However, the influence of DNA repair processes on the restoration of local chromatin structures and global nuclear architectures is still insufficiently understood. In multicellular organisms, epigenetic mechanisms control diverse cellular functions of specific cell types through precise temporal and spatial regulation of gene expression and silencing. How altered epigenetic mechanisms regulate the pathophysiological function of cells, tissues, and ultimately entire organs following IR exposure remains to be investigated in detail. Radiation-induced epigenetic processes are particularly critical for immature cell populations such as tissue-specific stem and progenitor cells during development and differentiation of organ tissues. Genome-wide patterns of DNA and histone modifications are established cell types—specifically during the development and differentiation of organ tissues but can also be fundamentally altered in adult organism by stress responses, such as radiation-induced DNA damage. Following IR exposure, epigenetic factors are not always fully restored to their original state, resulting in epigenetic dysfunction that causes cells to lose their original identity and function. Moreover, severe radiation-induced DNA damage can induce premature senescence of cells in complex tissues, which ultimately leads to signs of aging and age-related diseases such as cancer. In this work, we provide an overview of the most important epigenetic changes following IR exposure and their pathophysiological significance for the development of acute and chronic radiation reactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78d105ae28e3df5bffe779131184bc03d9d06867" target='_blank'>
              Role of Ionizing Radiation in Shaping the Complex Multi-Layered Epigenome
              </a>
            </td>
          <td>
            C. Rübe, M. A. A. Al-razaq, Carola Meier, Markus Hecht, Christian Rübe
          </td>
          <td>2025-08-08</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Epstein–Barr virus, the first identified human DNA tumour virus, is detectable in more than 90% of nasopharyngeal carcinoma patients in endemic regions. The 3D chromosome conformation analysis reveals that virus‒host chromatin interactions induce the spatial reorganisation of loops and compartments, resulting in “enhancer infestation” and switch of “H3K27 bivalency” at EBV-interacting regions. Through this mechanism, EBV hijacks KDM5B, a gatekeeper of genome stability, and its binding targets, leading to aberrant activation of an EBVIR-enhancer-KDM5B signature. Cancer cells with this signature present increased MYC activation, DNA damage responses, and epigenetic plasticity of epithelial-immune dual features with metastatic potential. Our multicentre multiomics study confirms that this signature is the prerequisite for chromosome instability and can be utilised as a risk factor for distant metastasis. This study highlights a mechanism in which latent viral episomes can alter the host high-order epigenotype, promoting transcriptional rewiring and metastasis in NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d24b257eddf1aac8b8f7ed9ed3a3ecfd24278af6" target='_blank'>
              Virus-human chromatin interactions reorganise 3D genome and hijack KDM5B for promoting metastasis in nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            D. Chung, Zhaozheng Hou, Ying Wang, Kazi Anisha Islam, Songran Liu, Jiayan Liu, Larry Ka-Yue Chow, Yuki Yuk-Wun Wong, Cyrus Paak-Ting Chak, Yingpei Zhang, Lanqi Gong, Ziyang Qi, K. Cheng, Zhuoyou Yu, Ping Feng, Zilu Huang, R. Ngan, Xin-Yuan Guan, W. Ng, Zhonghua Liu, A. Tsang, D. Kwong, Ann Wing-Mui Lee, V. H. Lee, Honglin Chen, Yunfei Xia, Wei Dai
          </td>
          <td>2025-08-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Introduction Retrotransposons (REs) constitute nearly half of the genome and include long terminal repeat (LTR) elements, Long INterspersed Elements (LINE), and Short INterspersed Elements (SINE). REs are typically silenced in somatic tissues via DNA methylation but can be reactivated through DNA hypomethylation, potentially impacting gene regulation. Here, we investigate genome-scale profiles of RE methylation in high-grade serous ovarian carcinoma (HGSOC) and associations with survival among Black women. Methods Methylation levels of LTR, LINE-1, and Alu (type of SINE) in 200 HGSOC tumors were predicted using a random forest approach and clustered using multiple consensus algorithms. Associations between RE methylation clusters and survival were evaluated using Cox proportional hazard regression, adjusting for age, stage, and debulking status. We performed sensitivity analyses restricted to women with late-stage disease and with adjustment for BRCA1/BRCA2 mutations. Results Two RE methylation clusters were identified. Cluster 1 exhibited a more hypomethylated RE profile (“Active”), while Cluster 2 was more hypermethylated (“Repressed”). No statistically significant differences in patient or clinical characteristics were observed between clusters. Compared to the Active Cluster, the Repressed Cluster was associated with an increased risk of mortality (HR = 2.41; 95% CI 1.04–5.59) and had a lower proportion of T cells. This association was consistent in sensitivity analyses. Conclusion A more hypermethylated RE profile was linked to worse survival among Black women with HGSOC, highlighting the potential of RE methylation as a prognostic biomarker. Further research is needed to understand the underlying biological mechanisms and their implications in ovarian cancer biology and treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01942-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bc642af0cdc56a04adbb30045ba5a19d537e41" target='_blank'>
              Retrotransposon methylation profiles and survival in Black women with high-grade serous ovarian carcinoma
              </a>
            </td>
          <td>
            C. Colin-Leitzinger, Katherine A Lawson-Michod, Courtney E Johnson, Irma M Vlasac, Sean Yoder, Tania Mesa, Dana Roeber, Chad D Huff, M. Hildebrandt, Kristen Haller, Anthony J Alberg, E. Bandera, Melissa Bondy, Michele L Cote, Theresa A Hastert, Edward S Peters, Paul D Terry, Andrew B. Lawson, Andy Berchuck, Brooke L Fridley, J. Chern, J. Doherty, Jeffrey R Marks, J. Schildkraut, Brock C. Christensen, Lucas A Salas, Lauren C. Peres
          </td>
          <td>2025-07-30</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Genomic instability and loss of proteostasis are two of the primary Hallmarks of Aging. Although these hallmarks are well-defined in the literature, the mechanisms that drive genomic instability and loss of proteostasis as cells age are still largely unknown. Using budding yeast replicative lifespan as a model for aging in actively dividing cells, we identified nuclear proteins that were depleted in the earliest stages of aging. We found that many age-depleted proteins were involved in ribosome biogenesis, specifically in ribosome processing, or in maintenance of chromatin stability. We focused on topoisomerase I (Top1) as a novel age-depleted nuclear protein and found that its depletion in the early stages of aging was not a result of transcriptional changes or changes in protein turnover. Despite the stark depletion of Top1 in early aging, rescue of this age-dependent depletion was actually harmful to replicative lifespan. We found that Top1, when overexpressed, disrupts the stoichiometry of the RENT complex by pulling Sir2 away from the ribosomal DNA (rDNA), a phenotype which is further enhanced when the overexpressed Top1 is catalytically dead. Loss of Sir2 from the rDNA via the overexpression of catalytically dead Top1 decreases RNA Pol II silencing of a reporter gene inside or adjacent to the rDNA, consistent with the lifespan defect. Finally, we found that the catalytic activity of Top1 plays an important role in the establishment of rDNA silencing, raising the possibility that rDNA secondary structure/DNA topology is important for RNA Pol I-dependent spreading of silent chromatin across the rDNA locus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cb1885993b368b41abc338c8a39ec0f62ce5c9c" target='_blank'>
              Age-dependent topoisomerase I depletion alters recruitment of rDNA silencing complexes
              </a>
            </td>
          <td>
            Lindsey N. Power, Natalia Zawrotna, M. Dinda, Abigail E. Weir, Bishal P. Paudel, Oshil Ghimire, Karolina Kisiel, Christopher T. Letai, Kevin A. Janes, Jeffrey S Smith
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Introduction Ovarian cancer (OC) is one of the leading causes of cancer-related death in women worldwide. Treatment with PARP-inhibitors has significantly improved survival in patients with high-grade serous cancer (HGSC) bearing BRCA1/2 mutations (~22% of the cases), and/or homologous recombination deficiency (HRD, ~50%). Unfortunately, limited therapeutic alternatives are available for BRCA1/2 wild type/HR proficient HGSC patients, who usually exhibit resistance to standard treatments and poor prognosis. Methods Herein, we present the results of a comprehensive genomic profiling (CGP) analysis using the Oncomine Comprehensive Assay® (OCA) Plus in a consecutive retrospective cohort of 102 HGSC patients characterized in our institution. Results Genomic instability, measured by Genomic Instability Metric (GIM) >16, was found in 40% of the cases and was significantly associated with BRCA1/2 mutations (p=0.009), with a better prognosis in terms of recurrence-free survival (p=0.01). CCNE1 amplification was observed in 29% of cases and was negatively correlated with BRCA1/2 mutations (p=0.001), without any association with GIM, supporting CCNE1 as a strong and independent driver of tumorigenesis. Additionally, CCNE1 amplification was validated with fluorescent in situ hybridization (FISH), supporting the analytical robustness of NGS data (rho=0.93), and investigated by immunohistochemistry (IHC), revealing that CCNE1 protein overexpression was observed in the absence of gene amplification in 45% of cases. Discussion Our real-world study supports the clinical utility of the GIM metric and the analytical validity of CCNE1 amplification, a new promising biomarker for personalizing treatment in HR proficient HGSC patients. The discordance between CCNE1 amplification and protein expression raises intriguing questions about the mechanisms of CCNE1-driven tumorigenesis and warrants further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef09810cfae4b459a30bafb300e6ffa23d0d9a9a" target='_blank'>
              Genomic instability and CCNE1 amplification as emerging biomarkers for stratifying high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            E. Conca, Daniele Lorenzini, E. Minna, Luca Agnelli, Matteo Duca, Marco Gentili, Beatrice Bodini, M. Polignano, Mara Mantiero, Silvia Damian, A. Devecchi, Gianpaolo Dagrada, Rita Carminati, Alice Ardore, Francesca Barbetta, Nathalia Brito Da Chuna, Andrea Guerrizio, A. Busico, I. Capone, Alberta Piccolo, E. Tamborini, Federica Perrone, Massimo Milione, Biagio Paolini, A. Vingiani, Francesco Raspagliesi, F. de Braud, Giancarlo Pruneri
          </td>
          <td>2025-08-06</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Summary Intratumoral heterogeneity, originating from genetic, epigenetic, and phenotypic cellular diversity, is pervasive in cancer. As these heterogeneous states employ diverse mechanisms to promote tumor progression, metastasis, and therapy resistance, emergence of cancer heterogeneity is one of the most significant barriers to curative treatment. Here we leverage deep learning approaches to develop a high-throughput image-analysis paradigm with subcellular resolution that quantifies and predicts colorectal cancer (CRC) patient outcome based on tissue architecture and nuclear morphology. We further combine this approach with spatial transcriptomics, multiplex immunohistochemistry, and patient-derived organoids to uncover a dynamic, co-evolutionary relationship between tumor architecture and cell states. We identify clinically relevant architectural interfaces in CRC tissue that diversify cellular identities by favoring distinct cancer stem cell states and thus promote evolution of tumor heterogeneity. Specifically, tissue fragmentation and associated compressive forces promote loss of classic stem cell signature and acquisition of fetal/regenerative stem cell states, which initially emerges as a hybrid state with features of epithelial-to-mesenchyme (EMT) transition. Additional tumor stroma communication then diversifies these states into distinct stem cell and EMT states at the invasive margins of tumors. Reciprocally, Wnt signaling state of tumor cells tunes their responsiveness to tissue architecture. Collectively, this work uncovers a feedback loop between cell states and tissue architecture that drives cancer heterogeneity and cell state diversification. Machine learning-based analyses harness this co-dependency, independently of mutational status or cancer stage, to predict patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e8d83b8caf4ac682c803318ebb577d7836c261e" target='_blank'>
              Colorectal cancer heterogeneity co-evolves with tumor architecture to determine disease outcome
              </a>
            </td>
          <td>
            Fabien Bertillot, Solène Hervé, Nelly Drobjazko, Noemi Kedei, Mona El Gendi, Maria O Hernandez, Kai Kruse, E. Erkan, Mirjam I. Binner, Karolina Punovuori, T. Pellinen, M. Ahtiainen, Jan Böhm, J. Mecklin, Caj Haglund, Jaana Hagström, Mike Nikolaev, N. Gjorevski, Matthias Lütolf, Toni T Seppälä, Sara A. Wickström, Yekaterina A. Miroshnikova
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="ABSTRACT Objective To illustrate a new barcoded orthotopic patient‐derived xenograft (PDX) mouse model where one can investigate phenotypic effects of single‐cell level gene manipulation in a pooled format. To address some concerns of current PDX mouse models of head and neck squamous cell carcinoma (HNSCC): (1) genomic evolution with passage by generating high‐purity cancer cells, which can also be utilized for other downstream applications, including cell culture‐based studies, and (2) cost‐effectiveness of current PDX models. Methods Two‐millimeter tumor cubes from nine patients were implanted into immunodeficient mouse flanks subcutaneously. Purified tumor cells were obtained from subcutaneous xenografts. Various numbers of purified tumor cells were then injected into the lingual tissue of immunodeficient mice, and the lowest amount of cells needed to achieve a 100% orthotopic engraftment rate were identified. Clonal stability was tested using a lentiviral barcoding system. The orthotopic PDXs' genetic landscapes were characterized using whole exome sequencing. Results This approach yielded an overall engraftment rate of 88.9%. The purification process increased cancer cell purity from 34% to 92%. Lingual injection of 100,000 purified tumor cells achieved a 100% orthotopic engraftment rate from purified subcutaneous PDX tumor cells while maintaining clonal and genetic stability. Conclusion Our study presents a barcoded orthotopic patient‐derived xenograft model for head and neck squamous cell carcinoma with clonal stability. This model provides a way to study phenotypic effects of single cell level gene manipulation in a pooled format. The method can be adapted for in vitro work as well.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/770f65963b9ed4389a6e68906eb989805afcd015" target='_blank'>
              Barcoded Orthotopic Patient‐Derived Head & Neck Squamous Cell Carcinoma Model Demonstrating Clonal Stability and Maintenance of Cancer Driver Mutational Landscape
              </a>
            </td>
          <td>
            Peiran Zhou, Claire B Mills, Zhao Ming Dong, Brittany R Barber, Slobodan Beronja
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Microscopic examination of biopsy tissues remains essential for cancer diagnosis, despite advancements in sequencing technologies. Alterations in nuclear size or the nuclear-to-cytoplasmic ratio are hallmark features of cancer cells and often correlate with disease progression. However, the mechanisms underlying nuclear size abnormalities and their impact on tumor progression remain unclear. In this study, we demonstrate that nuclear hypertrophy occurs in response to enhanced DNA replication stress, a key characteristic of cancer cells. Increased actin polymerization within the nucleus appears to be the primary mechanism driving nuclear hypertrophy downstream of the ATR-CHEK1 pathway. Replication stress-induced nuclear hypertrophy alters transcriptomic profiles and chromatin topology, while reducing the migratory and metastatic capacity of cancer cells. In addition, nuclear hypertrophy in cancer cells is associated with increased infiltration of antitumor immune cells. Our findings suggest that cell-autonomous effects of nuclear hypertrophy do not promote cellular fitness or aggressive characteristics in cancer cells. This may explain why cells with nuclear hypertrophy are not positively selected and persist as a subpopulation during tumor progression and metastasis. Furthermore, the link between replication stress and nuclear hypertrophy provides insights into why enlarged nuclei are consistently observed in advanced-stage cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8dd09203e34a2e72f651d176a77deed8567aab11" target='_blank'>
              Replication stress-induced nuclear hypertrophy alters chromatin topology and impacts cancer cell fitness.
              </a>
            </td>
          <td>
            Changgon Kim, Semyeong Hong, Soo Hyeon Ma, Joochan Lee, Hee So, Jun Young Kim, Eunbie Shin, Kippeum Lee, Sohee Choi, Juyeon Park, YongKeun Park, You-Me Kim, Ji Hun Kim, Joon Kim
          </td>
          <td>2025-09-10</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Non-homologous end joining (NHEJ) is a DNA repair pathway that directly ligates broken DNA ends without the need for a homologous template, whereas translesion synthesis (TLS) is a DNA damage tolerance mechanism in which specialized DNA polymerases bypass lesions on the template strand. Although both pathways play critical roles in maintaining genome integrity across organisms, they inherently introduce mutations. Here, we investigate how these two pathways contribute to spontaneous and genotoxic stress–induced genomic alterations in the yeast Yarrowia lipolytica. A NHEJ-deficient mutant (ku70) and three TLS-deficient mutants (rev1, rev3, and rad30) are subjected to mutation accumulation experiments, followed by whole-genome sequencing. Our results show that the deletion of KU70 has no significant effect on the rates of spontaneous single-nucleotide variations (SNVs), small insertions and deletions, or chromosomal rearrangements, while the deletion of REV1 and REV3 leads to significant reductions in spontaneous SNV rates. These findings indicate that TLS but not the NHEJ pathway is a major contributor to spontaneous mutagenesis in Y. lipolytica. Moreover, exposure to 0.02% methyl methanesulfonate and 80 J/m2 ultraviolet (UV) radiation resulted in 48- and 107-fold increases in SNV rates, respectively. These induced SNVs were largely dependent on DNA polymerases Rev1 and ζ, further underscoring their central roles in genotoxic stress–induced mutagenesis. We observe that DNA polymerase η can suppress C to T and C to A substitutions while promoting T to C mutations, exhibiting a dual function in regulating mutagenesis under UV treatment. Phenotypic evolution experiments reveal that TLS activity enhances the adaptive potential of Y. lipolytica under oxidative stress, underlying its broader impact on environmental fitness. Together, these findings provide new insights into the distinct roles of the NHEJ and TLS pathways in preserving genome integrity in Y. lipolytica. Key points • The NHEJ pathway has a limited role in spontaneous genomic alterations in Y. lipolytica. • DNA polymerases Rev1 and ζ contribute to most UV- and MMS-induced mutations. • The dual roles of Pol η in UV-induced mutations were revealed. • NHEJ and TLS pathways are crucial to phenotypic evolution of Y. lipolytica. Supplementary Information The online version contains supplementary material available at 10.1007/s00253-025-13575-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15d93f494460655f7859b1e3dd9cb7f63f2e1036" target='_blank'>
              The roles of NHEJ and TLS pathways in genomic alterations and phenotypic evolution in the yeast Yarrowia lipolytica
              </a>
            </td>
          <td>
            Cen Yan, Ye-Ke Wang, Yuan-Ru Xiong, Xin-Qiu Zhou, Yuan-Chun Fang, Ruo-Tian Nie, Cunqi Ye, Ke Zhang, Dao-Qiong Zheng
          </td>
          <td>2025-08-15</td>
          <td>Applied Microbiology and Biotechnology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="mRNA decapping enzyme 2 (DCP2) is a key regulator of mRNA degradation, influencing RNA metabolism and gene expression. While implicated in various diseases, its role in Hepatocellular carcinoma (HCC) remains unclear. This study explores DCP2 as a potential biomarker for HCC through bioinformatics and experimental analyses. Public databases, including The Cancer Genome Atlas (TCGA) and GEPIA2 online database, revealed significant DCP2 upregulation in HCC tissues, correlating with Tumor-Node-Metastasis (TNM) staging, histological grading, platelet-to-lymphocyte ratio (PLR), neutrophil-to-lymphocyte ratio (NLR), and alpha-fetoprotein (AFP) levels. Prognostic analysis confirmed high DCP2 expression as an indicator of poor overall survival (OS). Functional enrichment analyses linked DCP2 to cancer-associated pathways, including PI3K/Akt, Hippo signaling, cholesterol metabolism, and oxidative phosphorylation. Immune infiltration analysis showed significant correlations with tumor-associated macrophages (TAMs) and regulatory T cells (Tregs), suggesting a role in immune modulation. Experimental validation via western blot and immunohistochemistry (IHC) confirmed elevated DCP2 expression in HCC cell lines and tissues. These findings suggest that DCP2 may be involved in HCC progression by influencing mRNA degradation, cancer-related signaling, and the immune microenvironment. Although DCP2 shows potential as a diagnostic and prognostic biomarker, as well as a therapeutic target, further functional studies—such as gene knockdown and overexpression experiments—are needed to confirm its mechanistic role in the development of hepatocellular carcinoma (HCC). Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10308-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea65acb3c02d509d0fb354d65ba55dd81bb1cdf9" target='_blank'>
              mRNA decapping enzyme2 DCP2 expression correlates with progression and prognosis of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Tianyi Huang, Rui Wang, Cuihua Lu, Jiaying Zhang
          </td>
          <td>2025-08-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Endometrial carcinoma (EC), the most common gynecologic cancer type in developed countries, encompasses four molecular subtypes (POLEmut, MMRd, p53abn, and NSMP) that have prognostic values and guide treatment decisions. Additionally, dual loss of ARID1A and ARID1B (referred to as ARID1A/B) characterizes a significant portion of dedifferentiated/undifferentiated EC (DD/UDEC), a rare but highly aggressive subtype of EC. To advance the translational research for ECs, we analyzed the genomic features of a panel of 39 EC cell lines, leading to the identification of cell lines representing each of these EC molecular subtype. Histologic and immunohistochemical analyses of xenografted tumors from these cell lines confirmed their resemblance of cognate primary EC molecular subtypes. Further investigation of the publicly available genome-wide CRISPR screen data for EC cell lines identified multiple specific genetic dependencies in MMRd, p53abn, and ARID1A/B-dual deficient EC cell lines. Particularly, ARID1A/B-dual deficient DD/UDEC cells selectively rely on mitochondrial oxidative phosphorylation in vitro and in vivo. Therefore, through molecular subtyping of EC cell lines and subsequent characterization of molecular subtype-specific genetic dependencies, our study provides a framework that guides the utility of the EC cell line models for accelerating translational research in EC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e1f3b25336fe4feba408a071575d0aa7bd9be8e" target='_blank'>
              Molecular subtyping of endometrial carcinoma cell lines uncovers subtype-specific targetable vulnerabilities
              </a>
            </td>
          <td>
            Eunice Li, Rebecca Ho, Ran Tao, Yannes Wai Yan Choi, Chae Young Shin, Shary Chen, Bengul Gokbayrak, J. Senz, Betty Yao, Liam Johnston, Spencer D Martin, Eric Yang, Mark S Carey, Bryan T. Hennessy, D. Huntsman, Ramon I Klein Geltink, Lynn Hoang, Yemin Wang
          </td>
          <td>2025-07-24</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Abstract Diversification and selective propagation are the main driving forces of evolution, resulting in the emergence of organisms possessing various strategies. Here, we conducted in vitro evolution of the replication origin (oriC) under the pressure of an AT-rich mini-chromosome amplification using a reconstituted Escherichia coli replication-cycle reaction (RCR). Using next-generation sequencing, we identified that these evolved oriCs contain select mutations within the duplex unwinding element (DUE) and DnaA-binding site (DnaA box) regions, which are crucial for replication initiation. Real-time detection of RCR amplification (real-time RCR) revealed that the DUE mutations, which decreased the GC content, along with the introduction of a specific A/T sequence in DUE-M and a consecutive KAK (K = T or G) motif in DUE-R, enhanced the RCR amplification efficiency compared to the wild-type oriC (oriCwt). A competitive amplification assay also elucidated that the DnaA box mutations confer a competitive advantage over coexistent oriCwt. Although these DUE and DnaA box mutations were selected through the same amplification reaction, they exhibited distinct competitive amplification strategies. The DUE mutant represents a faster propagation strategy (r-strategy), while the DnaA box mutant represents an adaptive strategy with a competitive advantage (K-strategy), representing the r/K-selection theory in molecular evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76d8adc8e642b9b4fca6ba9da0b2f823fd1341a1" target='_blank'>
              Adaptive evolution of oriC through in vitro propagation of a mini-chromosome in RCR
              </a>
            </td>
          <td>
            Shota Suzuki, M. Su’etsugu
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Intratumor heterogeneity (ITH) is associated with anti-tumoral immune response and with the efficiency of cancer immunotherapy, yet overcoming ITH remains a significant challenge. Notably, cellular MYC (c-MYC) has been shown to be a pivotal orchestrator of this ITH progression. Here, we develop a c-MYC-based sensing circuit (cMSC) that is activated exclusively by aberrant c-MYC levels, along with an exosome-based cell-to-cell (CtC) system that augments communication among tumor cells, effectively targeting all cells in tumors circumventing the limitations imposed by ITH. Further expression of multifunctional immunostimulatory agents in these cMSC-reprogrammed cancer cells remodels the tumor microenvironment, enhancing selective T-cell-mediated oncolysis. Our cMSC/CtC platform specifically senses aberrant c-MYC expression and subsequently triggers a robust cancer immunotherapeutic response. These findings offer a promising avenue for targeting cancers via precisely sensing c-MYC, overcoming the limitations of ITH.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd1994e5ac75ac89b2d1f1610daef6efdae5bcc0" target='_blank'>
              Reprogramming the tumor microenvironment with c-MYC-based gene circuit platform to enhance specific cancer immunotherapy
              </a>
            </td>
          <td>
            Hengji Zhan, Hongjin Wang, Bolin Pan, Junlin Lu, Kanghua Xiao, Jiajian Lai, Zehua Chen, Kaiwen Jie, Siting Chen, Hong Li, Tianxin Lin, Xu Chen
          </td>
          <td>2025-08-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Ubiquitin carboxyl terminal hydrolase L3 (UCHL3) is a deubiquitinase belonging to UCH protease family. Previous studies have demonstrated that abnormal expression of UCHL3 is correlated with the development of human diseases, especially in tumors. Notably, UCHL3 exhibits contradictory biological functions across different cancer types. Initial studies identified UCHL3 as a tumor suppressor in prostate cancer. However, emerging evidence suggests that UCHL3 serves as an oncogene in other cancers. Several UCHL3 inhibitors have shown a promising anti-tumor effect in vitro and in vivo. UCHL3 has also been linked to DNA damage repair, which is essential for genome stability. Overexpression of UCHL3 enhances DNA damage repair induced resistance to chemotherapy or radiotherapy in certain tumor types. This review aims to summarize the promoting role of UCHL3 in DNA damage repair and its dual, paradoxical roles in tumor progression, along with the associated mechanisms, and to provide insights into the use of UCHL3 as a therapeutic target for overcoming DNA damage repair-mediated resistance to chemotherapy and radiotherapy, as well as for treating tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce3d5f1b68a091ec87b927bf44542627cac87c63" target='_blank'>
              UCHL3: a crucial deubiquitinase in DNA damage repair and tumor progression
              </a>
            </td>
          <td>
            Jiahao Liu, Shulu Hu, Junqi Xiao, Jumei Zhou
          </td>
          <td>2025-07-21</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Current cancer chemotherapy uses etoposide to induce DNA double-strand breaks for cell killing, which can cause fluctuations in stemness promoters like OCT4A, SOX-2, or NANOG in relation to senescence promoters p21Cip1, p27, or p16ink4A, and mitigate anti-cancer therapy. These fluctuations take place over several days and are associated with changes in the organization of euchromatin and heterochromatin. Here, we show how the induction of double-strand breaks by etoposide led to the compaction of euchromatin and de-condensation of heterochromatin on day 3. This was accompanied by a co-localization of euchromatin and heterochromatin marks as detected by Single-Molecule Localization Microscopy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a3f66309c268ad41eb3fdc8156d8889c120362" target='_blank'>
              Chemotherapy (Etoposide)-Induced Intermingling of Heterochromatin and Euchromatin Compartments in Senescent PA-1 Embryonal Carcinoma Cells
              </a>
            </td>
          <td>
            Marc Bayer, Jaroslava Zajakina, Myriam Schäfer, K. Salmina, Fēlikss Rūmnieks, J. Jansons, F. Bestvater, R. Kurg, J. Erenpreisa, Michael Hausmann
          </td>
          <td>2025-07-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="ABSTRACT RNA epigenetics, also referred to as epitranscriptomics, has emerged as a critical regulatory layer in cancer biology, extending beyond the scope of traditional DNA and histone modifications. It encompasses a series of dynamic posttranscriptional processes—including RNA biosynthesis, splicing, transport, stability, degradation, translation, and chemical modifications—orchestrated by RNA‐binding proteins (RBPs) and noncoding RNAs (ncRNAs). Collectively, these mechanisms influence mRNA fate, shape transcriptional output, and reprogram the tumor microenvironment. Importantly, both coding RNA and ncRNA are themselves subjected to epigenetic regulation, forming intricate feedback loops that contribute to oncogenesis, immune evasion, metastasis, and therapeutic resistance. In this review, we systematically synthesize the current understanding of RNA metabolism and RNA epigenetic modifications during tumor progression, with a particular focus on the roles of RBPs and RNA modifications. Furthermore, we highlight recent advances in RNA‐based therapeutic strategies, including mRNA vaccines, antisense oligonucleotides, siRNAs, and circRNA scaffolds. These innovative approaches offer promising avenues for targeting transcriptionally active yet genomically “undruggable” cancer drivers. Together, our synthesis provides a comprehensive framework for understanding RNA epigenetics in tumor biology and lays the groundwork for precision oncology guided by transcriptome plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de7fa009ea26201e23b24a2c7b7bd45eb23b5831" target='_blank'>
              RNA Epigenetics in Cancer: Current Knowledge and Therapeutic Implications
              </a>
            </td>
          <td>
            Shanhe Huang, Zonglin Li, Weilong Lin, Ruihui Xie, Hai Huang
          </td>
          <td>2025-08-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Human endogenous retroviruses (HERVs) are remnants of ancient viral infections and comprise 6–8% of the human genome. Their biological functions in cancer remain poorly understood, especially in glioblastoma, the most common and deadly primary brain cancer in adults. Prior studies on HERV expression in glioblastoma have yielded conflicting results. Here, we employed orthogonal computational pipelines to address these limitations. Results Locus-specific analysis revealed marked heterogeneity of transcription among HERVs within the same clade. We found that individual HERV proviruses are more than twice as likely to be underexpressed in glioblastoma than overexpressed and that most differentially expressed HERVs are exonized within transcripts. These HERVs are enriched in the 3’-terminal exons of transcripts, associated with alternative polyadenylation and contribute conserved polyadenylation signals. We identified HERV expression patterns associated with glioblastoma subtypes and anatomic features. We also identified three proviruses or proviral fragments of particular interest including one associated with survival and one exonized within a currently unannotated cancer-specific transcript. Among the most recently integrated clade of HERVs, excluding solo-LTRs, only the HML-2 provirus at 1q22 is overexpressed in glioblastoma. We report previously undescribed transcripts incorporating this provirus that may encode several proteins. Conclusions This study represents the first systematic exploration of the heterogeneity of HERV expression between anatomic features of any cancer. It shows that exaptation of HERV polyadenylation signals and HERV-associated APA are widespread in the human transcriptome and identifies critical structural information regarding the HML-2 1q22 provirus transcript, which has been the focus of several recent analyses. Our findings underscore the importance of locus-specific and anatomic feature-specific HERV analysis and suggest structural and functional roles for HERVs in glioblastoma-associated transcripts. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00365-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae712fda8b06db59cbe0784641111444d4db678e" target='_blank'>
              In-depth analysis of endogenous retrovirus expression in glioblastoma
              </a>
            </td>
          <td>
            Daniel A. Murimi-Worstell, Irene B. Murimi-Worstell, Farrah M. Roy, A. Burn, Michael Freeman, Ralph B. Puchalski, John M. Coffin
          </td>
          <td>2025-07-16</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Quiescent cells (Q) are seemingly inactive, developmentally arrested cells, whose universal characteristic is the ability to promptly reenter the cell cycle upon sensing of external cues. Q cells are responsive to the environment and flexible enough to adapt to available resources. In budding yeast, quiescent nuclear features are drastically distinct from those observed in nutrient replete conditions: the nuclear volume is reduced, the telomeres relocate from the nuclear periphery to the center of the nucleus into a hypercluster, chromatin is found in a compacted, hypoacetylated state, and transcription is globally shutdown. Yet, Q cells can restart transcription within minutes of refeeding. Here, we follow the global decrease of transcription in sorted, developing Q populations, and its reactivation upon release. We find that transcription and telomere clustering dynamics in and out of quiescence are independent events. We report a genome-wide redistribution of the transcription machinery as cells progress into quiescence. Although most genes are shut down, 3% of coding genes remain active. Furthermore, RNAPII accumulates at one third of gene promoters. The corresponding genes are highly enriched among those showing a high level of transcription and high frequency of expression in individual cells, shortly after cells are refed, as monitored by single-cell RNA-seq. Our results point toward a role for quiescent-specific RNAPII distribution to ensure a rapid and robust transcriptional response upon return to growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79310aef16945d91c92b6960ca1816e65764f91e" target='_blank'>
              Intergenic accumulation of RNAPII maintains the potential for swift transcriptional restart upon release from quiescence.
              </a>
            </td>
          <td>
            Manuela Baquero Pérez, Gertjan Laenen, Isabelle Loïodice, Mickael Garnier, Ugo Szachnowski, Antonin Morillon, Myriam Ruault, Angela Taddei
          </td>
          <td>2025-08-12</td>
          <td>Genome research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background
 Breast cancer (BC) remains the second leading cause of cancer-related mortality among women worldwide. Liquid biopsy based on circulating tumor DNA (ctDNA) offers a promising noninvasive approach for early detection; however, differentiating malignant tumors from benign abnormalities remains a significant challenge. Results Here, we developed a multimodal approach to analyze cfDNA methylation and fragmentomic patterns in 273 BC patients, 108 individuals with benign breast conditions, and 134 healthy controls. Genome-wide analyses revealed distinct cfDNA copy number alterations and cytosine-enriched cleavage sites in BC patients. Targeted sequencing further revealed unique methylation patterns, including hypermethylation in GPR126, KLF3, and TLR10 and hypomethylation in TOP1 and MAFB. Our machine-learning model achieved an AUC of 0.90, with 93.6% specificity and 62.1–66.3% sensitivity for stage I–II cancers. In symptomatic populations, sensitivities were 50.0%, 68.2%, and 64.7% for BI-RADS categories 3, 4, and 5, respectively, with 96.1% specificity. Conclusions These findings underscore the potential of cfDNA biomarkers to enhance BC detection and reduce the rate of unnecessary biopsies. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02371-z.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18aacd3f9fbc0c3fe9a757e1b6d9cf892dfdded8" target='_blank'>
              Multimodal analysis of cell-free DNA enhances differentiation of early-stage breast cancer from benign lesions and healthy individuals
              </a>
            </td>
          <td>
            Thi Tuong Vi Van, T. Tran, Thi Hue Hanh Nguyen, Van Thien Chi Nguyen, Dac Ho Vo, Giang Thi Huong Nguyen, Trong-Hieu Nguyen, Kim Sang To, Anh Luan Nguyen, Cao Hong An Tran, T. Jasmine, Thi Loan Vo, Thi Huong Thoang Nai, T. Tran, My Hoang Truong, Ngan Chau Tran, Thi Loc Le, Thi Hong Nhung Nguyen, Ngoc Hieu Tu, Thanh Son Tran, Bao Toan Le, Van Phong Tang, Pham Thanh Nhan Nguyen, Khac Tien Nguyen, Van Chien Ho, Xuan Vinh Nguyen, Nhu Nhat Tan Doan, Thi Trang Tran, Thi Minh Thu Tran, V. Tran, M. Le, Thi Luyen Vu, B. Tieu, Huu-Phuc Nguyen, Luu Hong Dang Nguyen, Ngoc Minh Phan, Thi Van Phan, T. Do, Thi Huyen Dao, Hung-Sang Tang, Duy Nguyen, Hoa Giang, Minh-Duy Phan, H. Nguyen, Duc Hieu Vo, Levis Tran
          </td>
          <td>2025-08-20</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="R-loops are three-stranded nucleic acid structures consisting of a DNA/RNA hybrid and a displaced single-stranded DNA. Once considered rare byproducts of transcription, R-loops are now recognized as important regulators of various nuclear processes. In particular, evidence indicates a role for R-loops in regulating DNA methylation dynamics. R-loops have been shown to promote active DNA demethylation—the enzymatic reversal of 5-methylcytosine (5mC) back into cytosine—by recruiting associated proteins, providing an attractive targeting mechanism. Nevertheless, many important aspects of this process, including whether the associated proteins bind to and function on R-loops, remain to be substantiated. In this study, we demonstrate for the first time that thymine DNA glycosylase (TDG), a key enzyme in the active DNA demethylation pathway, binds tightly to R-loops in vitro and can excise DNA demethylation intermediates 5-formylcytosine (5fC) and 5-carboxycytosine (5caC) from DNA in DNA/RNA hybrid duplexes. We also show that R-loops guide the strand-specific activity of TDG at CpG sites, potentially explaining the asymmetric distribution of 5fC/5caC at gene promoters. Furthermore, we provide important mechanistic insights into base excision on DNA/RNA hybrid duplexes using 19F NMR. Finally, our findings suggest that TDG–R-loop interactions may be widespread in human cells. Collectively, our results provide strong evidence that R-loops play a critical role in DNA demethylation and support a mechanism in which 5fC/5caC are directly removed from DNA/RNA hybrids in cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f720c25376cfceef4fd5beb6c69c7fd9ad1d0f50" target='_blank'>
              Thymine DNA Glycosylase Binds to R-Loops and Excises 5-Formyl and 5-Carboxyl Cytosine from DNA/RNA Hybrids
              </a>
            </td>
          <td>
            Baiyu Zhu, L. Pidugu, Mary E. Cook, Xinyu Y Nie, E. A. P. T. Amarasekara, Jerome S. Menet, A. Drohat, Jonathan T. Sczepanski
          </td>
          <td>2025-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Replication stress (RS) poses a threat to genome stability and drives genomic rearrangements. The homologous recombination (HR) pathway repairs stalled replication forks (RFs) and prevents such instability. Through an E3 ubiquitin ligase screen aimed at identifying regulators of RAD51, we identified macrophage erythroblast attacher (MAEA), a core component of C-terminal to Lish (CTLH) E3 ubiquitin ligase complex, as a regulator of the HR pathway. Loss of MAEA impairs RAD51 recruitment at stalled RFs, leading to increased sensitivity to RS-inducing agents and excessive degradation of nascent DNA strands. Mechanistically, MAEA associates with and mediates the ubiquitylation of Ku80, enabling its removal from RF ends and facilitating the loading of RAD51. Notably, MAEA deficiency is associated with a developmental disorder involving microcephaly, craniofacial abnormalities, ocular defects, and heart malformations. Functional assays show that disease-linked MAEA variants (R34C, E349G, Y394D, and M396R) are defective in RS response. These findings establish MAEA as an essential factor in RF protection and genome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c499707b53080fc3870cbe7d66d1176114a45109" target='_blank'>
              Pathogenic variants in MAEA disrupt DNA replication fork stability and are associated with developmental abnormalities in humans
              </a>
            </td>
          <td>
            Elham Zeinali, Fatemeh Mashayekhi, Rabih Abou Farraj, Talah Hasanni, Marie-christine Caron, Yan Coulombe, A. Fitieh, J. N. M. Glover, Jean-Yves Masson, Ismail Hassan Ismail
          </td>
          <td>2025-08-29</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="ABSTRACT Cell survival after irradiation depends on the cell cycle at the time of exposure. This has been thought to be due to cell cycle-dependent nuclear DNA damage repair mechanisms. Here, we show the relationships between the exposed dose, the cell cycle phase at the time of exposure and changes in mitochondrial DNA copy numbers (mtDNAcn) after irradiation. We used a fluorescent ubiquitylation-based cell cycle indicator (FUCCI), which allows visualization of the cell cycle, and confirmed cell cycle synchronization in human cervical HeLa cells. In synchronous HeLa-FUCCI cells, the mtDNAcn changed with the progression of the cell cycle. Also, G1 phase-synchronized cells showed a dose-dependent increase of mtDNAcn at 48 h after X-ray exposure, whereas G2 cells showed a dose-dependent increase at 24 h. In addition, S phase-synchronized cells showed a dose-dependent increase at 24 and 48 h after irradiation. These results showed the cell cycle- and dose-dependent effects on mtDNAcn after irradiation, which might shed light on the emerging role of mitochondrial genome and in cell survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d01df049eba0f6144ec33798db72137448ce9b4" target='_blank'>
              Cell cycle- and dose-dependent effects on mitochondrial DNA copy number variation following irradiation
              </a>
            </td>
          <td>
            Ryosuke Seino, Kai Nishikubo, Hisanori Fukunaga
          </td>
          <td>2025-07-18</td>
          <td>Journal of Cell Science</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Humans are continually exposed to a diverse array of environmental chemicals that can damage DNA and compromise genomic integrity. Among these genotoxic agents, acetaldehyde, acrolein, aristolochic acids, and vinyl chloride are particularly concerning due to their widespread presence in industrial emissions, dietary sources, and lifestyle-related exposures such as smoking and alcohol consumption. These compounds can induce structurally distinct forms of DNA damage including bulky DNA adducts, interstrand crosslinks, and other replication-blocking lesions. While canonical DNA repair pathways serve as the primary defense against such DNA damage, some lesions persist, challenging the capacity of DNA repair systems. If not efficiently repaired, DNA lesions may disrupt replication and transcription. In many cases, translesion synthesis polymerases are recruited to bypass unrepaired lesions, introducing mutations that contribute to agent-specific mutational signatures found in cancer genomes. This review systematically examines how each of these four exogenous chemicals induces DNA damage, the DNA repair pathways responsible for removing their lesions, and the role of translesion synthesis in shaping their mutational signatures. We also highlight how three-dimensional genome organization regulates lesion susceptibility and repair, contributing to variability of mutational landscapes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c10fba52c520395f5d989cae353584818e523a" target='_blank'>
              Mechanisms of DNA repair and mutagenesis induced by acetaldehyde, acrolein, aristolochic acids, and vinyl chloride.
              </a>
            </td>
          <td>
            Kyle Jones, Emmaline Dye, Nyla Gilkes, Dian Xia, Shan Jiang, Wentao Li
          </td>
          <td>2025-08-16</td>
          <td>Journal of environmental science and health. Part C, Toxicology and carcinogenesis</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary In acute lymphoblastic leukemia (ALL), loss of heterozygosity (LOH), including copy-neutral LOH, occurs alongside classic chromosomal aberrations. The aim of this study is to identify and analyze patterns of LOH, assess their frequency, and evaluate their association with clinical characteristics and early treatment response during the induction phase of the ALL protocol. The study included 853 children with ALL: 120 patients with B-ALL LOH+ and 58 patients with T-ALL LOH+. LOH was analyzed using CytoScan HD SNP microarrays. Multiple correspondence analysis and hierarchical clustering identified three clusters based on genetic and clinical profiles. In B-ALL, two clusters showed poor prognosis and treatment response, while Cluster 2—with CDKN2A duplication and rare LOH—had favorable outcomes. In T-ALL, Cluster 1 had a good prognosis despite CDKN2A LOH; Cluster 2 showed aggressive features with biallelic deletions; Cluster 3 lacked LOH of the CDKN2A gene and was characterized by a genetically stable profile. In B-ALL, whole-chromosome LOH, especially of chromosome 9, was frequent, unlike in T-ALL. LOH was less common in chromosomes affected by trisomy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d14e9a0230d506db7f58337160f9eee4169d81f6" target='_blank'>
              Loss of Heterozygosity in Pediatric Acute Lymphoblastic Leukemia and Its Prognostic Impact: A Retrospective Study
              </a>
            </td>
          <td>
            Borys Styka, G. Ręka, Aleksandra Ozygała, M. Janiszewska, Magdalena Stelmach, Paulina Skowera, Z. Urbańska, Monika Lejman
          </td>
          <td>2025-07-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a59a21b3122614696fcbf37a9973f0709eb05a0" target='_blank'>
              Unravelling the genetics and epigenetics of the ageing tumour microenvironment in cancer.
              </a>
            </td>
          <td>
            H. Easwaran, A. Weeraratna
          </td>
          <td>2025-09-08</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 NDC80 is pivotal in cell division, particularly in regulating the G2/M transition and mitotic progression. Recent studies have demonstrated that NDC80 is significantly overexpressed in multiple solid tumors. Further analysis has suggested that its high expression is significantly associated with an elevated pathological grade, increased metastatic risk, and shortened overall survival in patients with cancer. However, its precise role in pan-cancer development, progression, and prognosis remains unclear.



 We conducted a multi-omics analysis of NDC80 using genomic, transcriptomic, and proteomic data from 33 cancer types in The Cancer Genome Atlas, Clinical Proteomic Tumor Analysis Consortium, Genotype-Tissue Expression, and Human Protein Atlas.



 The results demonstrated frequent NDC80 mutations across multiple malignancies and significantly elevated expression in tumor tissues compared with that in their normal counterparts, correlating with worse overall and disease-free survival. Moreover, NDC80 expression was strongly associated with oncogenic pathways, key protein regulators, cellular components, myeloid-derived suppressor cell infiltration, ESTIMATE scores, and cancer-related signaling networks.



 These findings underscore the potential of NDC80 as a prognostic biomarker and therapeutic target for cancer treatment.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/babdc68f18a78e0f774f50e2e30eff000da42137" target='_blank'>
              Integrative human pan-cancer analysis of NDC80
              </a>
            </td>
          <td>
            Zhengchun Liu, Jianing Wang, Peizhen Geng, Yuan Yuan, Tianwei Zhang, Shuai Wang, Yang Wang, Yihui Liu
          </td>
          <td>2025-07-21</td>
          <td>Oncology and Translational Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Overview 
Background 
The CEP55 encoded product is a centrosome protein.Participate in the development of the cranioskeletal system，Establish protein localizationand intermediate separation。Acting on the upstream or internal division of mitotic cells。CEP55 can affect the prognosis of various tumor types through multiple pathways。We evaluated the role of CEP55 using pan cancer data from the Cancer Genome Atlas (TCGA) and Genotyping Tissue Expression (GTEx) projects. 
  
Method 
The potential role of CEP55 in different tumors was explored based on the Cancer Genome Atlas (TCGA) and Genotyping Tissue Expression (GTEx) datasets.CEP55 related expression differences, tumor mutation burden (TMB), microsatellite instability (MSI), MATH, clinical staging and gene expression, immune cell analysis (xCELL, CIBERSORT, Timer), gene expression prognosis, etc.Our research findings indicate that CEP55 affects the prognosis of cancer patients, and its role varies among different types of cancer。In some cases, elevated CEP55 gene expression is detrimental to survival, while the opposite is true in other tumor types. 
  
 Result 
CEP55 在各种肿瘤组织和癌症邻近的正常组织中表达存在差异。CEP55 与 TMB 和 MIS 等常见免疫治疗靶点相关，开发与 CEP55 相关的药物，具有通过 TMB 和 MSI 免疫靶点治疗相关肿瘤的潜力。CEP55 与肿瘤的临床分期相关，可作为临床肿瘤不同阶段调整治疗方案的依据。CEP55 在各种免疫细胞分析中显示出显著的相关性，表明 CEP55 广泛影响肿瘤免疫细胞，进而可能影响各种肿瘤的免疫调节，并可能参与多种肿瘤免疫微环境的调节，有可能作为潜在的免疫治疗靶点。CEP55 可能是一个潜在的新治疗靶点。 
  
结论 
CEP55 的表达在不同肿瘤之间有所不同，并且与各种肿瘤的临床分期相关。它可能参与多种肿瘤的免疫调节，并可能作为潜在的治疗靶点。">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67d8e40912555132587f4a0683eb384c8c0b07be" target='_blank'>
              Analysis of CEP55 pan cancer immune and prognostic based on TCGA and GTEx databases
              </a>
            </td>
          <td>
            Xiulian Xu, Pan Wang, Qijun Lv, Hailin Gan, Yiming Huang, Zhiheng Liu
          </td>
          <td>2025-08-26</td>
          <td>Psycho-Oncologie</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Replication timing (RT), the temporal order of DNA replication during S phase, influences regional mutation rates, yet the mechanistic basis for RT-associated mutagenesis remains incompletely defined. To identify drivers of RT-dependent mutation biases, we analyzed whole-genome sequencing data from cells with disruptions in DNA replication/repair genes or exposed to mutagenic compounds. Mutation distributions between early- and late-replicating regions were compared using bootstrapping and statistical modeling. We identified 14 genes that exhibit differential effects in early- or late-replicating regions, encompassing multiple DNA repair pathways, including mismatch repair (MLH1, MSH2, MSH6, PMS1, and PMS2), trans-lesion DNA synthesis (REV1) and double-strand break repair (DCLRE1A and PRKDC), DNA polymerases (POLB, POLE3, and POLE4), and other genes central to genomic instability (PARP1 and TP53). Similar analyses of mutagenic compounds revealed 19 compounds with differential effects on replication timing. These results establish replication timing as a critical modulator of mutagenesis, with distinct DNA repair pathways and exogenous agents exhibiting replication timing-specific effects on genomic instability. Our systematic bioinformatics approach identifies new DNA repair genes and mutagens that exhibit differential activity during the S phase. These findings pave the way for further investigation of factors that contribute to genome instability during cancer transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13a232208698bfbf694bd7e822cd1f4798018e9a" target='_blank'>
              Unveiling Replication Timing-Dependent Mutational Biases: Mechanistic Insights from Gene Knockouts and Genotoxins Exposures
              </a>
            </td>
          <td>
            Hadas Gross-Samuels, A. Koren, Itamar Simon
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Endogenous retroviruses (ERVs) are remnants of retroviral infections that have become stably integrated into host germline genomes. Far beyond passive genomic elements, ERVs actively shape host evolution through complex mechanisms involving genetic innovation, immune modulation, and species adaptation. This review provides a comprehensive synthesis of ERV biology, highlighting recent advances in their classification, amplification mechanisms, and epigenetic silencing. Particular emphasis is placed on the cross-talk between ERVs and exogenous retroviruses (XRVs), demonstrating how receptor competition, recombination, and immune evasion contribute to virus-host co-evolution. We explore ERVs as molecular markers for phylogenetic reconstruction, with case studies such as Koala retrovirus (KoRV) and HERV-K illustrating regional transmission dynamics and co-opted immune functions. Additionally, we discuss the functional domestication of ERVs into regulatory elements, non-coding RNAs, and envelope-derived fusion proteins that influence gene expression, antiviral defense, and placental development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bff7f347985d489007229fb4a5b8a3cd06f14b32" target='_blank'>
              Endogenous Retroviruses in Host-Virus Coevolution: From Genomic Domestication to Functional Innovation
              </a>
            </td>
          <td>
            Ruqi Jiang, Jingjun Zhou, Yue Liu, Guanjin Zhou, Dongdong Fan, Lixin Xiang, Ye Chen, Jianzhong Shao
          </td>
          <td>2025-08-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="X-chromosome inactivation (XCI) is a crucial mechanism of dosage compensation in female mammals ensuring that genes from only one X chromosome are expressed, initiated through expression of the long noncoding RNA Xist. Recent evidence underscores the significance of molecular crowding—most likely via liquid–liquid phase separation (LLPS)—in forming Xist RNA-driven condensates critical for establishing and sustaining the silenced state. By integrating existing knowledge and emerging ideas, we provide a comprehensive perspective on the molecular underpinnings of XCI and outline how manipulation of LLPS-based mechanisms offers new avenues for novel therapeutic approaches. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03666-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8719ab7b9e423b53c745911745943aa3173aa818" target='_blank'>
              Xist condensates: perspectives for therapeutic intervention
              </a>
            </td>
          <td>
            Irene Perotti, Laura Broglia, G. G. Tartaglia, Andrea Cerase
          </td>
          <td>2025-07-21</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Amplified centrosome number causes genomic instability, most severely through division into more than two aneuploid daughter cells (multipolar mitosis). Several mechanisms that suppress multipolar division have been uncovered, yet mechanisms that favor viable multipolar division are poorly understood. To uncover factors that promote viability in cells with frequent centrosome amplification and multipolar division, we conducted an unbiased Drosophila genetic screen. In 642 mutagenized lines, we exploited the ability of intestinal papillar cells to form and function despite multipolar divisions. Our top hit is an unnamed gene, CG3168. We name this gene synaptic vesicle glycoprotein 2, reflecting homology to human Synaptic Vesicle Glycoprotein 2 (SV2) proteins. GFP-tagged SV2 localizes to the plasma membrane. In cells with amplified centrosomes, SV2 positions membrane-adjacent centrosomes, which prevents severe errors in chromosome alignment and segregation. Our results uncover membrane-based multipolar division regulation and reveal a novel vulnerability in cells with common cancer properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/604d54b8cce75407680ad35bfbf26bb3a7a65d49" target='_blank'>
              Synaptic vesicle glycoprotein 2 enables viable aneuploidy following centrosome amplification.
              </a>
            </td>
          <td>
            Jane E Blackmer, Erin A. Jezuit, Archan Chakraborty, Satya N Yalamanchi, Ruth A. Montague, Erin E. Dickert, Nora G Peterson, William Outlaw, Donald T. Fox
          </td>
          <td>2025-07-31</td>
          <td>Genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc32b216cd93b5ffb3fc4906eb7ae996906a5999" target='_blank'>
              POFUT1 as a cancer biomarker: insights into its oncogenic mechanisms and clinical relevance.
              </a>
            </td>
          <td>
            Oumaima Mazour, Sébastien Legardinier, B. Badaoui, A. Germot
          </td>
          <td>2025-08-07</td>
          <td>Journal of molecular medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Post-transcriptional RNA modifications, such as N6-methyladenosine (m6A) methylation and adenosine to inosine (A-to-I) editing, are critical regulators of hematopoietic stem cell (HSC) self-renewal and differentiation, yet their precise contributions to malignant transformation are not fully elucidated. In this study, we uncovered the epitranscriptomic landscape caused by knockdown of genes from the methyltransferase (METTL)-family in hematopoietic stem and progenitor cells (HSPCs). We identified both converging and distinct roles of METTL3 and METTL14, known members of the m6A writer complex, as well as orphan gene METTL13. Notably, METTL13 was uniquely upregulated by adenosine deaminase acting on RNA 1 (ADAR1) overexpression, while other METTL genes were downregulated. Knockdown of METTL13 altered the expression of multiple genes involved in oncogenic development in HSPCs. Furthermore, METTL13 was associated with a high-risk profile in pediatric T-cell acute lymphoblastic leukemia (T-ALL), and functional studies confirmed that METTL13 is required for T-ALL cell proliferation and survival both in vitro and in vivo. Collectively, our results indicate a previously unrecognized, oncogenic role for METTL13 in pre-leukemic transformation and T-ALL pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/047267bf0f7cb87eac1cee31332201b93b817d49" target='_blank'>
              METTL13 Promotes Pre-Leukemic Transformation and the Development of Pediatric Leukemia
              </a>
            </td>
          <td>
            Sabina Enlund, , Isabella Hoang, Sonali Joshi, Amanda Ramilo Amor, Cecilia Thomsson, Indranil Sinha, Shahrzad Shirazi Fard, Anna Nilsson, Ola Hermanson, Qingfei Jiang, Frida Holm
          </td>
          <td>2025-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="DNA transactions introduce torsional constraints that pose an inherent risk to genome integrity. While topoisomerase 1 (TOP1) activity is essential for DNA supercoil removal, the aberrant stabilization of TOP1:DNA cleavage complexes (TOP1ccs) can result in cytotoxic DNA lesions. What protects genomic hot spots of topological stress from excessive TOP1cc accumulation remains unknown. Here, we identify chromatin context as an essential means to coordinate TOP1cc resolution. Through its ability to bind poly(ADP-ribose) (PAR), the histone variant macroH2A1.1 facilitates TOP1cc repair factor recruitment and lesion turnover, thereby preventing DNA damage in response to transcription-associated topological stress. The alternatively spliced macroH2A1.2 isoform is unable to bind PAR or protect from TOP1ccs. Impaired macroH2A1.1 splicing, a frequent cancer feature, was predictive of increased sensitivity to TOP1 poisons in a pharmaco-genomic screen in breast cancer cells, and macroH2A1.1 inactivation mirrored this effect. We propose macroH2A1 alternative splicing as an epigenetic modulator of TOP1-associated genome maintenance and a potential cancer vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfdec4fbd7a5287d669fa0b66871ebd9fb8a9c98" target='_blank'>
              Epigenetic control of topoisomerase 1 activity presents a cancer vulnerability
              </a>
            </td>
          <td>
            Tae-Hee Lee, Colina X Qiao, Vladislav Kuzin, Yuepeng Shi, Marina Farkas, Zhiyan Zhao, Vijayalalitha Ramanarayanan, Tongyu Wu, Tianyi Guan, Xianzhen Zhou, David Corujo, M. Buschbeck, Laura Baranello, Philipp Oberdoerffer
          </td>
          <td>2025-08-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="DNA replication initiation is orchestrated in bacteria by the replication initiator DnaA. Two models for regulation of DnaA activity in Escherichia coli have been proposed: the switch between an active and inactive form, and the titration of DnaA on the chromosome. Although proposed decades ago, experimental evidence of a titration-based control mechanism is still lacking. Here, we first identified a conserved high-density region of binding motifs near the origin of replication, an advantageous trait for titration of DnaA. We then investigated the mobility of DnaA by visualising single proteins inside single cells of wild-type and deletion mutants E. coli strains, while monitoring cellular size and DNA content. Our results indicate that the chromosome of E. coli controls the free amount of DnaA in a growth rate-dependent fashion. Moreover, they address long-standing questions on the relevance of DnaA titration in stabilising DNA replication by preventing re-initiation events during slow growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9245293eecd0e45715a401c652d843bd210c464" target='_blank'>
              The Escherichia coli replication initiator DnaA is titrated on the chromosome
              </a>
            </td>
          <td>
            Lorenzo Olivi, Stephan Köstlbacher, C. Ludwig, Mees Langendoen, Nico J. Claassens, Thijs J. G. Ettema, J. van der Oost, P. R. ten Wolde, Johannes Hohlbein, R. Staals
          </td>
          <td>2025-08-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Amplification of human chromosome 20q has been reported as the most frequently recurring genetic abnormality associated with large scale changes in mRNA and protein levels in sporadic colorectal carcinomas. While some studies have found 20q amplification to be consistent between primary and metastatic samples from the same patient with a role in the development of metastasis and worse patient prognosis, others have reported association with improved overall survival for a subset of these patients with colorectal cancer (CRC). To fine map the Minimal Common Regions (MCRs) of amplification on chromosome 20q and identify the candidate genes playing roles in progression of the disease, microarray comparative genomic hybridization analyses of two in vitro cultured CRC liver metastasis cell line model systems was utilized. Microarray expression analysis led to the identification of a candidate gene signature comprising of four genes, BMP7, DNMT3B, UBE2C and YWHAB, residing in the MCRs that were over expressed in CRC cells. By validating our results in a training set of 23 adenocarcinomas (tumors) and five adenomas (polyps) using reverse transcription-quantitative PCR, as well as analyses of two larger colorectal cancer test data sets derived from 195 The Cancer Genome Atlas and 182 MD Anderson Cancer Center patients with colorectal adenocarcinoma patients, this gene signature was ascertained to be associated with lymph node spread and/or distant metastasis (P<0.05). Previously reported functional studies of the gene signature indicated their involvement in inflammatory and immune response pathways driving CRC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19ea2107086249d394733ab4fbf6c82be7552a16" target='_blank'>
              Chromosome 20q gene signature associated with colorectal cancer progression
              </a>
            </td>
          <td>
            Jennifer Carter Jones, A. Hegde, Yu-Jing Huang, G. Manyam, Vibhuti Srivastava, J. Song, Yulan Cheng, Ralf Krahe, Warapen Treekitkarnmongkol, Stephen J. Meltzer, S. Kopetz, Stanley R Hamilton, Hiroshi Katayama, Subrata Sen
          </td>
          <td>2025-07-18</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Tumor evolution is shaped by selective pressures imposed by physiological factors as the tumor naturally progresses to colonize local and distant tissues, as well as by therapy. However, the distinction between these two types of pressures and their impact on tumor evolution remain elusive, mainly, due to extensive intra-tumor heterogeneity. To disentangle the effects of these selective pressures, we analyze data from diverse cohorts of patients, of both treated and untreated cancers. We find that, despite the wide variation across patients, the selection strength on tumor genomes in individual patients is stable and largely unaffected by tumor progression in the primary settings, with some cancer-specific signatures detectable in the progression to metastases. However, we identify a nearly universal shift toward neutral evolution in tumors that resist treatment and demonstrate that this regime is associated with worse prognosis. We validate these findings on both published and original datasets. We suggest that monitoring the selection regime during cancer treatment can assist clinical decision-making in many cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95ee20056cad563b0d086b8e70065c4daacf4f" target='_blank'>
              Genome-level selection in tumors as a universal marker of resistance to therapy
              </a>
            </td>
          <td>
            Erez Persi, P. Sudalagunta, Yuri I. Wolf, Rafael R. Canevarolo, Mehdi Damaghi, K. Shain, A. Silva, Eugene V. Koonin
          </td>
          <td>2025-07-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is the age-related expansion of mutated hematopoietic stem cells without other hematologic abnormalities. In patients with solid tumors, CH is associated with higher mortality and may evolve to therapy-related myeloid neoplasms; however, the mechanisms by which cancer treatments promote CH dynamics remain largely unknown. Here, we analyzed 392 serial samples from a prospective cohort of breast cancer patients and showed that cytotoxic treatments (chemotherapy {+/-} radiation) led to strong therapeutic bottlenecks, resulting in significant reductions in hematopoietic allelic populations and differential clonal selection. CH clones that were positively selected and expanded through dose-dependent therapeutic bottlenecks frequently harbored mutations in TP53, PPM1D, SRCAP, DNMT3A, and YLPM1. Patients with positively selected CH during treatment had the shortest progression-free and overall survival compared to patients with unchanging or negatively selected CH across all therapies. These findings, corroborated in independent breast cancer and pan-cancer cohorts, provide strong evidence for clinical relevance of changes in CH during cancer treatment which may help identify patients at high risk for inferior outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6a553a5184b2aa0287d71bedac04551bf6a5da6" target='_blank'>
              Clonal Hematopoiesis Dynamics and Evolutionary Fitness During Cancer Treatment Impact Clinical Outcomes
              </a>
            </td>
          <td>
            M. Arabzadeh, Y.-H. Tang, C. Colin-Leitzinger, S. Marzban, D. Walgenbach, S. Morganti, V. Mahaganapathy, E. Harper, M. Teng, J. K. Kresovich, I. Washington, H. A. Parsons, J. E. Garber, J. West, S. Ganesan, H. Khiabanian, N. Gillis
          </td>
          <td>2025-09-02</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Metastable epialleles (MEs) are genomic loci at which epigenetic marks are established stochastically during early embryonic development and maintained during subsequent differentiation and throughout life, leading to stable epigenetic and phenotypic variation among genetically identical individuals. Although MEs were first described in mice over 20 years ago, the extent of epigenetic metastability in the mouse genome remains unknown. We present the first unbiased genome-wide screen for MEs in mice. Using deep whole-genome bisulfite sequencing across tissues derived from the three embryonic germ layers in isogenic C57BL/6J mice, we identified only 29 MEs, precisely localizing them and documenting their rarity. Consistent with recent findings, we found no effects of maternal dietary methyl donor supplementation on ME methylation in the offspring, challenging previous assertions that MEs generally exhibit developmental plasticity. Most but not all MEs are associated with intracisternal A-particle (IAP) elements, tending to localize to the 5′ end of the IAP. Additionally, we discovered autosomal regions at which systemic interindividual variation in DNA methylation is associated with sex, providing insights into sex-associated epigenetic development that apparently precedes sexual differentiation. Our findings indicate that expression of transcription factors, including CCCTC-binding factor (CTCF) and specific KRAB zinc finger proteins during early embryonic development, plays a key role in orchestrating stochastic establishment and/or maintenance of DNA methylation at metastable transposable elements. Overall, these findings advance our understanding of the genomic determinants of epigenetic metastability and suggest that interindividual epigenetic variation at MEs is unlikely to be a major determinant of phenotypic variation among isogenic mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708a43f6a2e4e91d381b16e3a28eb5d04dfb8092" target='_blank'>
              Mouse metastable epialleles are extremely rare
              </a>
            </td>
          <td>
            Chathura J Gunasekara, U. Maduranga, Taylor Zhang, Jonathan N Wells, Maria S. Baker, Eleonora Laritsky, Yumei Li, C. Coarfa, Yi Zhu, R. Waterland
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="DNA-binding molecules regulate gene expression, replication, repair, and transcription, making their research crucial to molecular biology. This review investigated their architectures, processes, and impacts on cancer research and therapy to determine their biological and therapeutic potential. A thorough examination of published data on DNA-binding agents, including intercalators, groove binders, and metal complexes, focused on their chemical properties, biological activity, and therapeutic significance. Doxorubicin intercalated between base pairs to inhibit replication and transcription, whereas cisplatin produced covalent cross-links with guanine bases to cause tumor cell death. Multiple derivatives of metal complexes reduced tumor development by over 70% in leukemia models as DNA probes and therapeutics. Also reviewed, Dps proteins in *Escherichia coli* showed that their non-specific DNA binding offered up to 65% oxidative stress resistance compared to control cultures, validating DNA-protein protection as a survival strategy. In Phase II studies, amsacrine caused substantial remission in acute leukemia patients, whereas doxorubicin was more effective across many cancer types but had greater cardiotoxicity concerns. Selectivity, toxicity, and resistance limit DNA-targeting medicines, although they are successful. Chemical modifications like hydrophobic tailoring and sequence-specific binding have enhanced binding affinity and therapeutic index, yet only 10% of candidate compounds get clinical approval. Recent studies show that AI-driven design has expedited screening, lowering development costs by 30% and durations by 3–5 years. These data suggest that DNA-binding medicines have great potential in cancer, but safer, more selective, and resistance-free therapy are still needed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bfaa0d838d07a9aad5a057a0a95be735af4700a" target='_blank'>
              Chemistry of DNA-binding Molecules
              </a>
            </td>
          <td>
            Evon Akram, Dina A. Najeeb, Asmaa A. Jawad, N. H. Bedair, A. M. Hussein, Saba R. Jaafar, Ruaa H. Ali, R. F. Shaher, Marwa A. Hussein, Daniah M. Hamid, Reem H. Al-Tabra, Alyaa K. Abood, Salam Mohammed
          </td>
          <td>2025-08-30</td>
          <td>Al-Salam Journal for  Medical Science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer progression is often accompanied by epigenetic silencing of tumor-suppressor microRNAs such asmiR-200c, a key regulator of epithelial-to-mesenchymal transition (EMT) and metastasis. Given the reversible nature of DNA methylation, we employed a CRISPR/dCas9-TET1 system to target the miR-200c promoter and restore its expression in MCF-7 and MDA-MB-231 breast cancer cell lines. Two gRNAs were designed to flank CpG-rich regions of the miR-200c promoter, and their individual or combined delivery enabled site-specific demethylation. Co-transfection with both gRNAs resulted in a synergistic increase in miR-200c expression, likely due to expanded coverage of dCas9-TET1 recruitment. This upregulation led to the downregulation of key EMT-related transcription factors ZEB1, ZEB2, and the oncogene KRAS, as well as increased E-cadherin expression in MDA-MB-231 cells. However, E-cadherin changes in MCF-7 cells were minimal, highlighting the complex and context-dependent nature of epigenetic regulation. Functional assays further confirmed the anti-tumorigenic effects of miR-200c restoration, with reduced cell viability and increased apoptosis, effects more pronounced in MDA-MB-231 cells, which initially exhibited higher miR-200c promoter methylation. Collectively, our findings demonstrate that CRISPR/dCas9-TET1-mediated epigenetic editing effectively reactivates miR-200c, reverses EMT-associated gene expression, and impairs tumor cell aggressiveness, supporting its potential as a targeted therapeutic strategy in breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e592b60e9ed7df393f38257baac9b31bbd24eb" target='_blank'>
              CRISPR/dCas9-TET1–mediated epigenetic editing reactivates miR-200c in breast cancer cells
              </a>
            </td>
          <td>
            Mahyar Zahraei, Yasamin Azimi, Morteza Karimipour, Fatemeh Rahimi-Jamnani, Vahideh Valizadeh, Masoumeh Azizi
          </td>
          <td>2025-08-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background/Objectives: DNA-damaging agents can contribute to genetic instability, and such agents are often used in cancer chemotherapeutic regimens due to their cytotoxicity. Thus, understanding the mechanisms involved in DNA damage processing can not only enhance our knowledge of basic DNA repair mechanisms but may also be used to develop improved chemotherapeutic strategies to treat cancer. The high-mobility group box protein 1 (HMGB1) is a known nucleotide excision repair (NER) cofactor, and its family member HMGB3 has been implicated in chemoresistance in ovarian cancer. Here, we aim to understand the potential role(s) of HMGB3 in processing DNA damage. Methods: A potential role in NER was investigated using HMGB3 knockout human cell lines in response to UV damage. Subsequently, potential roles in DNA interstrand crosslink (ICL) and DNA double-strand break (DSB) repair were investigated using mutagenesis assays, metaphase spreads, foci formation, a variety of DNA repair assays, and TagSeq analyses in human cells. Results: Interestingly, unlike HMGB1, HMGB3 does not appear to play a role in NER. We found evidence to suggest that HMGB3 is involved in the processing of both DSBs and ICLs in human cells. Conclusions: These novel results elucidate a role for HMGB3 in DNA damage repair and, surprisingly, also indicate a distinct role of HMGB3 in DNA damage repair from that of HMGB1. These findings advance our understanding of the role of HMGB3 in chemotherapeutic drug resistance and as a target for new chemotherapeutic strategies in the treatment of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1261da2765ba41543d61e2f0da2291c8e870115b" target='_blank'>
              High-Mobility Group Box Protein 3 (HMGB3) Facilitates DNA Interstrand Crosslink Processing and Double-Strand Break Repair in Human Cells
              </a>
            </td>
          <td>
            Jillian Dangerfield, Anirban Mukherjee, W. Reh, Anna M. Battenhouse, Karen M. Vasquez
          </td>
          <td>2025-09-04</td>
          <td>Genes</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Germline mutations in the DNA repair helicase XPD can cause the diseases xeroderma pigmentosum (XP) and trichothiodystrophy (TTD). XP patients bear an increased risk of skin cancer including melanoma. This is not observed for TTD patients despite DNA repair defects. To examine whether TTD cells harbor features counteracting tumorigenesis, we developed a TTD melanoma cell model containing the XPD variant R722W. Intriguingly, TTD melanoma cells exhibited reduced proliferation and an increased signature of the melanocyte lineage factor MITF, along with a strong basal upregulation of REDD2, an inhibitor of the mTOR/S6K/4EBP1-dependent messenger RNA (mRNA) translation machinery. REDD2 levels were partially driven by MITF and contributed to reduced melanoma proliferation. In a TTD model for melanocytes—the progenitor cells of melanoma—the MITF gene signature was also increased, but here without affecting REDD2 expression. However, ribosomal protein synthesis was reduced particularly in R722W melanocytes after UV stress, indicating a compromised mRNA translation machinery. Impaired translation was also demonstrated for the TTD XPD variant A725P, but not for an XP variant. Concludingly, the impaired translation and reduced fitness observed in TTD melanocytes and melanoma cells, particularly after UV stress, offer a possible explanation why TTD patients do not develop melanomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6262e92871597f9c2f4cf51c63e9ae2bb27bcd01" target='_blank'>
              Anti-tumorigenic properties by trichothiodystrophy mutations in melanocytic cells
              </a>
            </td>
          <td>
            Rupesh Paudel, Lena F Sorger, Anita Hufnagel, Mira Pasemann, Tamsanqa Hove, André Marquardt, Susanne Kneitz, Andreas Schlosser, Caroline Kisker, Jochen Kuper, Svenja Meierjohann
          </td>
          <td>2025-08-27</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="PLXNA3 is a transmembrane protein essential for axon guidance and synapse formation, gaining attention in cancer research for its roles in cell migration, invasion, and signal transduction. However, its biological significance in breast cancer remains underexplored. Here, bulk RNA sequencing and survival analysis revealed that elevated PLXNA3 levels were correlated with increased malignancy and poorer survival in breast cancer. Single-cell RNA sequencing demonstrated that PLXNA3 was predominantly enriched in tumor cell clusters within the tumor immune microenvironment, rather than in immune cell clusters. In vitro experiments further demonstrated that knockdown and overexpression of PLXNA3 significantly altered the proliferative, invasive, and migratory capacities of breast cancer cells. Additionally, the overexpression of PLXNA3 was negatively correlated with immune activation status and served as a predictor of poor response to anti-PD1 immunotherapy. Our findings suggested that high expression of PLXNA3 is associated with poor prognosis in breast cancer and plays a crucial role in cancer immunity, making it a promising novel target for intervention in breast cancer. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-13843-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f126a6deb0547875700a586994249ce5224fe17" target='_blank'>
              Comprehensive analysis of the PLXNA3 gene on prognosis and immune characteristics in breast cancer
              </a>
            </td>
          <td>
            Wen Sun, Yifan Xu, Qi Qi, Weiya Zhang, H. Pan, Guangshun Sun, Shui Wang, Wenbin Zhou
          </td>
          <td>2025-08-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Viral oncolysis is considered a promising cancer treatment method because of its good tolerability and durable anti-tumor effects. Compared with other oncolytic viruses, Newcastle disease virus (NDV) has some distinct advantages. As an RNA virus, NDV does not recombine with the host genome, making it safer compared with DNA viruses and retroviruses; NDV can induce syncytium formation, allowing the virus to spread among cells without exposure to host neutralizing antibodies; and its genome adheres to the hexamer genetic code rule (genome length as a multiple of six nucleotides), ensuring accurate replication, low recombination rates, and high genetic stability. Although wild-type NDV has a killing effect on various tumor cells, its oncolytic effect and working mechanism are diverse, increasing the complexity of generating engineered oncolytic viruses with NDV. This study aims to employ whole-genome CRISPR-Cas9 knockout screening and RNA sequencing to identify putative key regulatory factors involved in the interaction between NDV and human colon cancer HCT116 cells and map their global interaction networks. The results suggests that NDV infection disrupts cellular homeostasis, thereby exerting oncolytic effects by inhibiting cell metabolism and proliferation. Meanwhile, the antiviral immune response triggered by NDV infection, along with the activation of anti-apoptotic signaling pathways, may be responsible for the limited oncolytic efficacy of NDV against HCT116 cells. These findings not only enhance our understanding of the oncolytic mechanism of NDV against colonic carcinoma but also provide potential strategies and targets for the development of NDV-based engineered oncolytic viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e05426a39d80cc8e94e4737d115ccac397145186" target='_blank'>
              Genome-Wide Screening Reveals the Oncolytic Mechanism of Newcastle Disease Virus in a Human Colonic Carcinoma Cell Line
              </a>
            </td>
          <td>
            Yu Zhang, Shufeng Feng, Gaohang Yi, Shujun Jin, Yongxin Zhu, Xiaoxiao Liu, Jinsong Zhou, Hai Li
          </td>
          <td>2025-07-25</td>
          <td>Viruses</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Objective: Prostate cancer, a leading cause of cancer deaths in men, involves genetic alterations and DNA damage. Oxidative stress–induced DNA damage is crucial in its development, necessitating efficient DNA repair mechanisms. Base excision repair (BER), a key pathway, addresses oxidative DNA damage with enzymes like Apurinic/Apyrimidinic Endonuclease 1 (APE1). Recent findings show that altered APE1 expression disrupts BER balance, contributing to genomic instability and cancer progression. This study examines APE1 expression in prostate cancer tissues to clarify its role in cancer development and therapeutic implications. Assessing APE1 expression in prostate cancer may elucidate its role in tumor progression, therapeutic resistance, and genomic instability, offering deeper insights into the mechanistic relevance of the BER pathway in oncogenesis. Methods: A total of 50 male patients diagnosed with prostate cancer were included in this study. Tumor and adjacent normal tissue samples were collected from consecutive male patients who underwent Tru-cut prostate biopsies due to clinical suspicion of prostate cancer, and gene expression levels were analyzed using real-time polymerase chain reaction. Results: The results showed no significant difference in APE1 expression between tumor and normal tissues (P > .05). Clinical factors like smoking, age, diabetes, hypertension, PIRADS, and Gleason scores did not significantly correlate with APE1 expression. However, a statistically significant correlation was observed between PSA levels and APE1 expression (r: 0.287, P = .04). Conclusion: The authors’ findings suggest that while no substantial changes in APE1 expression were observed in prostate cancer tissues, the enzyme’s role in DNA repair remains crucial. Abnormalities in APE1 expression and function can contribute to genomic instability and cancer progression. Future studies with larger sample sizes and standardized methodologies are necessary to better understand the potential role of APE1 as a biomarker and therapeutic target in prostate cancer.Cite this article as: Rouhi V, Habibi S, Baykal Koca S, Köseoğlu H, Güven M. Investigation of apurinic/apyrimidinic endonuclease 1 expression changes in prostate cancer. Cerrahpaşa Med J 2025, 49, 0009, doi: 10.5152/cjm.2025.25009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f5a4b0fb3242b36fc3de3889e7200ec9f3f9fad" target='_blank'>
              Investigation of Apurinic/Apyrimidinic Endonuclease 1 Expression Changes in Prostate Cancer
              </a>
            </td>
          <td>
            Vahid Rouhi, Sanaz Habibi, Sevim Baykal Koca, Hikmet Köseoğlu, Mehmet Güven
          </td>
          <td>2025-08-29</td>
          <td>Cerrahpaşa Medical Journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/813fcceb7d9e7dd4a1d1ee8db0535b3e857d7b20" target='_blank'>
              Unraveling the Potential of ctDNA in Precision Medicine for Breast Cancer.
              </a>
            </td>
          <td>
            Juscelino Carvalho de Azevedo Junior, Fernanda Jardim da Silva, Anna Carolina Lima Rodrigues, Stefanie Braga Maia de Sousa, Jéssica Manoelli Costa da Silva, Iago Barroso Ramos, Bárbara do Nascimento Borges, Vanessa Morais Freitas, D. Calcagno
          </td>
          <td>2025-08-07</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Peroxisome proliferator-activated receptor γ (PPARγ), which is expressed in a variety of malignancies, governs biological functions through transcriptional programs. Defining the molecular mechanisms governing the selection of canonical versus non-canonical PPARγ binding sequences may provide the opportunity to design regulators with distinct functions and side effects. Acetylation at K268/293 in mouse Pparγ2 participates in the regulation of adipose tissue differentiation, and the conserved lysine residues (K154/155) in mouse Pparγ1 governs lipogenesis in breast cancer cells. Herein, the PPARγ1 acetylated residues K154/155 were shown to be essential for oncogenic ErbB2 driven breast cancer growth and mammary tumor stem cell expansion in vivo. The induction of transcriptional modules governing growth factor signaling, lipogenesis, cellular apoptosis, and stem cell expansion were dependent upon K154/155. The acetylation status of the K154/155 residues determined the selection of genome-wide DNA binding sites, altering the selection from canonical to non-canonical (C/EBP) DNA sequence-specific binding. The gene signature reflecting the acetylation-dependent genomic occupancy in lipogenesis provided predictive value in survival outcomes of ErbB2+ breast cancer. The Pparγ1 acetylation site is critical for ErbB2-induced breast cancer tumor growth and may represent a relevant target for therapeutic coextinction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72bf36b7f21c4b94702b8969aa76f9a853c780b1" target='_blank'>
              PPARγ acetylation governs mammary adenocarcinoma tumor growth via acetylated residues that determine DNA sequence-specific binding
              </a>
            </td>
          <td>
            Lifeng Tian, X. Jiao, Chenguang Wang, Danni Li, A. Ertel, Joanna Achinger-Kawecka, S. Addya, Raymond E Soccio, Eric R. Chen, Balázs Győrffy, Gabriele Di Sante, Zhijiu Zhong, Haidar Alkhafaji, Nina Entcheva, Elyssa M. Campbell, Peter A McCue, Andrew V. Kossenkov, Rita Pancsa, Peter Tompa, Susan J. Clark, Richard G Pestell
          </td>
          <td>2025-07-26</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Collecting duct carcinoma (CDC) is a rare but aggressive form of renal cell carcinoma (RCC) that has limited understanding and an undefined systemic therapeutic regimen. Herein, we conducted a comprehensive proteogenomic analysis of CDC tumors and normal adjacent tissues to elucidate the biology of the disease. CDC exhibited high heterogeneity in tumor mutational burden, and enhanced ribosome biogenesis was the most striking malignant feature of CDC, even compared with other common kidney carcinomas. Genomic data indicated that UTP6 and HN1 amplification on chromosome 17q were associated with the activations of ribosome biogenesis and cell migration, respectively, which were relevant to tumor proliferation and metastasis. Proteomic-based classification identified 3 clusters, among which, tumors overexpressing ribosome biogenesis signaling (GP1) clustered into the most aggressive subtype, while tumors with increased energy metabolism (GP3) exhibited significant sensitivity to anti-vascular endothelial growth factor agents. Immune subtyping revealed a complex immune landscape of CDC. Additionally, increased RPF2, contributing to ribosome production, was validated to be associated with malignant phenotypes, and targeting RPF2 could exert an anti-oncogenic role by disrupting ribosome biogenesis and perturbing the MDM2–p53 interaction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12e1fc6365b169a4b11d9331cdcff3ef36a173f1" target='_blank'>
              Proteogenomic Analysis Identifies Clinically Relevant Subgroups of Collecting Duct Carcinoma
              </a>
            </td>
          <td>
            Yuanyuan Qu, Xiaoru Pei, Jinwen Feng, Xin Yan, Linhui Zhang, Jun Wang, Xin Yao, Jiasheng Bian, Yu Gan, H. Gan, Xuewen Jiang, Ping Yang, Maoping Cai, Liqing Li, Xinqiang Wu, Weiwei Jing, Chao Zhang, Jianyuan Zhao, Hailiang Zhang, Guohai Shi, Xiang Zhou, Dingwei Ye, Chen Ding
          </td>
          <td>2025-08-06</td>
          <td>Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The XPO1 (exportin 1) gene encodes exportin 1 protein responsible for transporting proteins and RNA from the nucleus to the cytoplasm. It has been used as a biomarker for lymphoma detection. XPO1E571K mutation has been frequently observed and identified as a good prognostic indicator for lymphoma patients. The detection of a target molecule released by lymphoma cells into blood circulation (cell-free circulating tumor DNA, cfDNA) is a better method than tissue biopsy. However, cfDNA concentration in blood circulation is very low in cancer patients. Therefore, a precise and sensitive method is needed. In this study, cfDNA was extracted, and then the XPO1 gene was detected and amplified using conventional PCR. Sanger sequencing was employed to verify the DNA sequences. FAST-COLD-PCR was developed to detect XPO1E571K gene mutation using a CFX96 Touch Real-Time PCR System. The optimal critical temperature (Tc) was 73.3 °C, allowing selective amplification of XPO1E571K mutant DNA while wild-type XPO1 could not be amplified. XPO1E571K gene mutation can be detected by this method with high specificity and sensitivity in lymphoma patients. This approach facilitates rapid and straightforward detection in a timely manner after the diagnosis. Accordingly, the optimized FAST-COLD-PCR conditions can be used as a prototype for XPO1E571K mutant detection in lymphoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2337549ca06913503c667aac38d66d4d170ff79" target='_blank'>
              Detection of XPO1E571K Gene Mutation from Cell-Free DNA in Blood Circulation of Lymphoma Patients by FAST-COLD PCR
              </a>
            </td>
          <td>
            Suwit Duangmano, Natsima Viriyaadhammaa, Pinyaphat Khamphikham, Nutjeera Intasai, A. Tantiworawit, Teerada Daroontum, S. Chiampanichayakul, S. Anuchapreeda
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Targeting the DNA damage response (DDR) exhibits potent efficacy in inducing immune activation and enhancing patient prognosis. However, the benefits of DDR regulation are not universally observed across all patients, owing to the intricate compensatory mechanisms operative in certain cancers. There still exists a gap in the function of activated DDR protein in esophageal squamous cell carcinoma (ESCC). Here, we demonstrate that increased expression of DDR genes contributes to the progression of esophageal squamous cell carcinoma and suppresses the tumor immune microenvironment. Notably, the abundant presence of the DDR protein KIN in ESCC tissues facilitates efficient DNA damage clearance and promotes escape from apoptosis. Depletion of KIN significantly inhibited proliferation and induced DNA damage accumulation in ESCC cells. Mechanistically, KIN functions to support the recruitment of the R-loop regulator DHX9 to R-loop sites, thereby addressing DNA damage associated R-loops. Intriguingly, the depletion of KIN activates the STING pathway via NFκB signaling, which is induced by the accumulation of R-loops, ultimately initiating an innate immune response. Depletion of KIN improved the immune microenvironment and the effect of immune therapy in mouse model. Collectively, our findings identify KIN as a novel R-loop binding protein that facilitates the recruitment of the R-loop resolution complex and suppresses tumor-intrinsic innate immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e3c03a2576fc82b57363fed2fb18edfc9c71a5" target='_blank'>
              DNA/RNA-binding protein KIN17 supports esophageal cancer progression via resolving noncanonical STING activation induced by R-loop
              </a>
            </td>
          <td>
            Zichao Wei, N. Zhao, Lu Kuang, J. Cong, Sujuan Zheng, Yi Li, Zhihua Liu
          </td>
          <td>2025-08-15</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05361b43fcc5f75fe89674ef423d0f325e486737" target='_blank'>
              Characterizing neurofibromin 1-altered breast cancer through genomic, functional, and clinical analyses.
              </a>
            </td>
          <td>
            Xueyan Mao, Kang Ma, Yunjie Wang, Cheng Long, Xuejing Tan, Xi Chen, Bo Chen
          </td>
          <td>2025-08-14</td>
          <td>Virchows Archiv : an international journal of pathology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="A substantial amount of castration-resistant prostate cancer (CRPC) progresses into a neuroendocrine (NE) subtype, known as NEPC, which is associated with poor clinical outcomes. Here we report distinct three-dimensional chromatin architectures between NEPC and CRPC tumors, which were recapitulated by isogenic cell lines undergoing NE transformation (NET). Mechanistically, pioneer factors such as FOXA2 initiate binding at NE enhancers to mediate regional DNA demethylation and induce neural transcription factor (TF) NKX2-1 expression. NKX2-1 preferentially binds gene promoters and interacts with enhancer-bound FOXA2 through chromatin looping. NKX2-1 is highly expressed in NEPC and indispensable for NET of prostate cancer. NKX2-1/FOXA2 further recruits p300/CBP to activate NE enhancers, and pharmacological inhibition of p300/CBP effectively blunts NE gene expression and abolishes NEPC tumor growth. Taken together, our study reports a hierarchical network of TFs governed by NKX2-1 in critically regulating chromatin remodeling and driving luminal-to-NE transformation and suggests promising therapeutic approaches to mitigate NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f79a3ac68a2814bb628761a75464adfb40b777a" target='_blank'>
              NKX2-1 drives neuroendocrine transdifferentiation of prostate cancer via epigenetic and 3D chromatin remodeling
              </a>
            </td>
          <td>
            Xiaodong Lu, Viriya Keo, Irina Cheng, Wanqing Xie, Galina A Gritsina, Juan Wang, Lina Lu, Cheng-Kai Shiau, Yueying He, Qiushi Jin, Peng Jin, Martin G Sanda, Victor Corces, Nicolas Altemose, Ruli Gao, Jonathan C Zhao, Jindan Yu
          </td>
          <td>2025-07-21</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Radiotherapy is a mainstay of treatment for head and neck squamous cell carcinoma (HNSCC), either definitive or adjuvant to surgery. Biological factors known to affect radiation response are hypoxia and DNA repair capacity, but several lines of evidence indicate that other genes and pathways in the tumor cells might be involved that have not been elucidated. Here, we report the results of a genome-wide CRISPR-Cas9 functional genomics screen in HNSCC cells to identify radiosensitizing genes. Remarkably, microtubule organizing genes were identified with CLASP1 as most unexpected radiosensitizing hit. Clonogenic assay confirmed the radiosensitizing effect of CLASP1 knockout. Functional analysis showed that CLASP1 knockout has major impact during S-phase, and resulted in mitotic cells with broken chromosomes and cell death. CLASP1 and possibly the microtubule machinery in broader sense seem involved in protection of HNSCC cells against radiation–induced DNA damage. This newly identified mechanism provides an outlook for novel treatment approaches in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/284fd55d39146101abef95bbef149eda9c32d030" target='_blank'>
              An unexpected role of CLASP1 in radiation response and S-phase regulation of head and neck cancer cells
              </a>
            </td>
          <td>
            R. D. de Roest, Marijke Buijze, Myrthe Veth, Klaas de Lint, G. Pai, M. Rooimans, Rob M. F. Wolthuis, A. Brink, J. Poell, R. Brakenhoff
          </td>
          <td>2025-08-06</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Background SPOCK1, a matricellular glycoprotein, has been implicated in tumor progression, metastasis, and the tumor immune microenvironment, yet its specific roles in breast cancer (BRCA) remain unclear. This study aimed to systematically explore the expression pattern, prognostic significance, mutation landscape, immune association, and spatial localization of SPOCK1 in breast cancer through integrated multi-omics analyses. Methods Transcriptomic, genomic, and clinical data from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) were utilized. Bulk RNA sequencing and single-cell RNA sequencing (scRNA-seq) analyses were conducted, including functional enrichment, immune infiltration assessments, mutation profiling, and transcription factor activity analysis. Multiplex immunohistochemistry (mIHC) was performed to validate the spatial distribution of SPOCK1+ cancer-associated fibroblasts (CAFs) within the tumor microenvironment. Statistical analyses were performed using R and GraphPad Prism. Results SPOCK1 was broadly overexpressed in multiple cancer types and significantly associated with poor prognosis in BRCA. High SPOCK1 expression correlated with immune checkpoint activation, enhanced immune infiltration, and enriched metastasis-related pathways such as epithelial–mesenchymal transition (EMT) and TGF-β signaling. Single-cell analysis identified CAFs as the primary cell population expressing SPOCK1, with spatial mIHC confirming their close proximity to tumor cells. Furthermore, SPOCK1-high CAFs exhibited stronger intercellular communications with malignant cells via collagen, fibronectin, and IGFBP signaling pathways, alongside distinct transcription factor and metabolic profiles. In breast cancer CAF cell lines with knockdown of ANXA2 we found that the expression of both SPOCK1 and IGF1 was reduced. Conclusion SPOCK1 serves as a critical regulator of breast cancer progression, influencing tumor metastasis and reshaping the immune microenvironment via CAF-mediated mechanisms. These findings suggest that targeting SPOCK1+ CAFs could offer new therapeutic opportunities for breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd6ec4d3a2efd20e6cfad0b54c715da9bed1f6fb" target='_blank'>
              SPOCK1 promotes the progression of breast cancer by modulating cancer-associated fibroblasts and exerts a synergistic effect with ANXA2
              </a>
            </td>
          <td>
            Yuan Jie, Xin Fei, Meng Fan
          </td>
          <td>2025-08-05</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is a lethal brain tumor with limited therapies. NUF2, a kinetochore protein involved in cell cycle regulation, shows oncogenic potential in various cancers; however, its role in GBM pathogenesis remains unclear. In this study, we investigated NUF2’s function and mechanisms in GBM and developed an MRI-based machine learning model to predict its expression non-invasively, and evaluated its potential as a therapeutic target and prognostic biomarker. Functional assays (proliferation, colony formation, migration, and invasion) and cell cycle analysis were conducted using NUF2-knockdown U87/U251 cells. Western blotting was performed to assess the expression levels of β-catenin and MMP-9. Bioinformatic analyses included pathway enrichment, immune infiltration, and single-cell subtype characterization. Using preoperative T1CE Magnetic Resonance Imaging sequences from 61 patients, we extracted 1037 radiomic features and developed a predictive model using Least Absolute Shrinkage and Selection Operator regression for feature selection and random forest algorithms for classification with rigorous cross-validation. NUF2 overexpression in GBM tissues and cells was correlated with poor survival (p < 0.01). Knockdown of NUF2 significantly suppressed malignant phenotypes (p < 0.05), induced G0/G1 arrest (p < 0.01), and increased sensitivity to TMZ treatment via the β-catenin/MMP9 pathway. The radiomic model achieved superior NUF2 prediction (AUC = 0.897) using six optimized features. Key features demonstrated associations with MGMT methylation and 1p/19q co-deletion, serving as independent prognostic markers. NUF2 drives GBM progression through β-catenin/MMP9 activation, establishing its dual role as a therapeutic target and a prognostic biomarker. The developed radiogenomic model enables precise non-invasive NUF2 evaluation, thereby advancing personalized GBM management. This study highlights the translational value of integrating molecular biology with artificial intelligence in neuro-oncology. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-15721-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2af44d79456ec646874f72650101c19e902b458" target='_blank'>
              Uncovering novel functions of NUF2 in glioblastoma and MRI-based expression prediction
              </a>
            </td>
          <td>
            Rong-de Zhong, Yun-sheng Liu, Qian Li, Zeng-wei Kou, Fan-fan Chen, Heng Wang, Na Zhang, Han Tang, Yuan Zhang, Guo-dong Huang
          </td>
          <td>2025-09-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Transcription factors (TFs) play a crucial role in tumorigenesis by driving oncogene expression in key signaling pathways. However, their small size and flat surfaces make them challenging targets for small-molecule inhibitors, while macromolecular therapies struggle to cross the cell membrane. Modulating TF activity at the genetic level offers a promising alternative. Antisense oligonucleotides (ASOs), which regulate protein expression by targeting mRNA, have emerged as effective therapeutics for previously undruggable proteins, including TFs. Over the past two decades, ASO therapeutics have advanced significantly, demonstrating long-lasting efficacy by promoting mRNA degradation. c-Myc, a key regulator of oncogene expression, drives cancer cell growth and proliferation but remains undruggable due to its nuclear localization and dynamic structure. In this study, we utilized our ASO development platform to design ASOs targeting c-Myc. Our sequence optimization algorithm achieved high accuracy, with one of three designed ASOs successfully silencing c-Myc. Ex vivo validation showed that ASO3 inhibited A549 cell growth with an IC50 of 152.5 nM. At the molecular level, ASO3 significantly reduced both c-Myc mRNA and protein expression. Functional assays, including trypan blue exclusion assay and CCK-8, confirmed that ASO3 decreased cell viability, suppressed proliferation, and induced apoptosis. These findings highlight ASO3's therapeutic potential and support further investigation as an anti-cancer agent targeting c-Myc.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b33b3079149d5ba644ab8a2a4958404ec130feef" target='_blank'>
              Targeting c-Myc with antisense oligonucleotides to induce apoptosis in tumor cells.
              </a>
            </td>
          <td>
            Yuemei Ye, Yanhui Wang, Zhaoyun Zong, Shiyu Chen
          </td>
          <td>2025-08-12</td>
          <td>Nucleosides, nucleotides & nucleic acids</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Initially, pseudogenes were considered to be “junk DNA”, and their biological role was unclear. However, some of the pseudogenes are engaged in the process of cancerogenesis and perform essential functions in competing for endogenous ribonucleic acid (ceRNA) networks and competing for RNA binding proteins (RBPs). They either positively or negatively regulate gene expression and act as suppressive and oncogenic transcripts. In this review, we look at some of the pseudogenes that play a role in the epithelial-to-mesenchymal transition (EMT) process and the maintenance of cancer-initiating cells (CIC), which are essential in understanding cancer development and progression metastasis and resistance to commonly used therapies. The group of discussed pseudogenes consists of CHIAP2, PTENP1, SUMO1P3, NANOGP8, OCT4-PG1/4, or HMGA1-P6, which are connected with different molecular pathways. Moreover, we discussed pseudogenes as potential diagnostic molecules that can be used as a new class of biomarkers. This potential usage may be valuable for oncology and personalized medicine in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8fe0d4350cd2131303d74c06087d56d8dfd818f" target='_blank'>
              Pseudogenes in the carcinogenesis: epithelial-to-mesenchymal transition process and cancer initiating cells
              </a>
            </td>
          <td>
            Tomasz Kolenda, Piotr Chałaj, Aleksandra Cichowicz, Aleksandra Trojańska, Andrzej Bałoniak, Malwina Kwaśniewska, Magda Odrobińska, Kacper Guglas, Joanna Kozłowska-Masłoń, Paulina Gieremek, Paulina Poter, Marlena Janiczek-Polewska, Anna Florczak-Substyk, Anna Przybyła, Patrycja Mantaj, Katarzyna Regulska, Beata J Stanisz, Zefiryn Cybulski, U. Kazimierczak
          </td>
          <td>2025-08-08</td>
          <td>Reports of Practical Oncology and Radiotherapy</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Emerging evidence has demonstrated that pseudouridylation regulates mRNA translation and gene expression, yet its molecular characteristics in hepatocellular carcinoma (HCC) remain unknown. Using public databases, we developed pseudouridylation-related molecular subtype and risk score model to assess HCC patient prognosis and disclose their clinical feature, molecular mechanism and immune landscape. Furthermore, quantitative polymerase chain reaction (qPCR) was performed to verify the expression of RDM1, CDCA3 and FLVCR1. Specifically, functional enrichment analysis revealed pseudouridylation-related genes (PRGs) predominantly regulate transcriptional and translational regulation. Prognostic PRGs stratified HCC into two distinct subtypes, the cluster 1 had a poor prognosis and was characterized by high alpha fetoprotein level, poor differentiation, advanced tumor stage, large tumor size, frequent TP53 mutation, up-regulation of cell cycle-and mitosis-associated genes, which was similar to the aggressive proliferation subtype of HCC. In contrast, the cluster 2 exhibited good prognosis and increased infiltration of immune cells, resembling the non-proliferation subtype of HCC, and suggesting its potential responsiveness to immunotherapy. Survival analysis discovered that the risk score model served as an independent prognostic factor, with high-risk group exhibiting significantly shorter overall survival and recurrence-free survival than low-risk group. Notably, receiver operating characteristic analysis revealed that the risk model had a powerful predictive performance for 1-and 3-year survival (AUC=0.806). In addition, functional enrichment analysis suggested that upregulated genes of high-risk group displayed an enrichment of cell cycle progression, mitotic division, and some oncogenic signaling pathways (PLK1, FOXM1, and p53 signaling pathways). qPCR experiment confirmed the significant overexpression of RDM1, CDCA3, and FLVCR1 in HCC tissues, being consistent with public database analysis. In conclusion, pseudouridylation related-molecular subtype and risk model may effectively predict the prognosis and therapeutic response of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e3d571cd2491ccee2926479bbc9813ac0fda913" target='_blank'>
              Pseudouridylation-Related Genes Predict Prognosis and Therapeutic Response in Hepatocellular Carcinoma Patients
              </a>
            </td>
          <td>
            Chenlu Lan, Donghua Gao, Yong Wei, Huasheng Huang, Xianwei Lv, Xin Zhou, Weijuan Qin, Xiwen Liao, Guangzhi Zhu, Tao Peng
          </td>
          <td>2025-08-16</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Summary/Abstract Glioblastoma stem cell (GSC) cultures are initiated from glioblastoma (GBM) surgical resection tissue. They can capture and propagate key GBM primary tumor molecular and cellular features. We have deeply characterized four isocitrate dehydrogenase (IDH)-expressing (or IDH+) GSC cultures from unrelated adults to serve as cellular models for the majority of adult primary GBM. We demonstrate that GSC cultures can be continuously propagated in defined, serum-free media and 5% oxygen without requiring specialized growth substrates; have well-defined genomic and mtDNA variants and gene/protein expression profiles; and highly reproducible dose-survival curves when treated with the GBM standard-of-care therapies of ionizing radiation (IR) and temozolomide (TMZ). We also illustrate how expressed lentiviral barcodes, mtDNA variants and single cell gene expression profiling can be used to define and track cellular heterogeneity over 40 days after IR treatment. These well-characterized IDH+ GSC cultures can support many high throughput in vitro assay formats, including xenograft, organoid and other GBM disease modeling protocols. They should prove a useful resource to better understand GBM biology, and to identify new and more effective GBM therapies and treatment regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b852883dfd3891f1795ac6382d7546d2b8963f5d" target='_blank'>
              Cellular heterogeneity and therapeutic response profiling of human IDH+ glioma stem cell cultures
              </a>
            </td>
          <td>
            N. Chambwe, Scott R. Kennedy, Brendan F. Kohrn, Pavlo Lazarchuk, Mario Leutert, Guangrong Qin, Bahar Tercan, M. Sanchez-Contreras, Weiliang Tang, Jerome J. Graber, Patrick J. Paddison, Judit Villén, Ilya Shmulevich, Raymond J Monnat
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f396f45a05f69851b7610dc8198d4d5a27e35207" target='_blank'>
              Characterization of cis-regulatory elements and functional variants in colorectal cancer using epigenomics and CRISPRi screenings.
              </a>
            </td>
          <td>
            Zequn Lu, Can Chen, Heng Zhang, Bin Li, Yizhuo Liu, Jiayi Guo, Runying Xu, Ke Shi, Qianying Ma, Ming Zhang, Yimin Cai, Jinyu Huang, Hui Geng, Linyun Fan, Caibo Ning, Yanmin Li, Shuo Chen, W. Tian, Kexin Hu, Haijie Li, Xiaojun Yang, Chaoqun Huang, Yongchang Wei, Xu Zhu, Xiangpan Li, Zhen Xiong, M. Cai, Xiaoyang Wang, Shaokai Zhang, Hongda Chen, M. Dai, Kun Chen, Mingjuan Jin, Meng Jin, Ying Zhu, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2025-08-25</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Mitotic bookmarking, the retention of regulatory proteins and lncRNAs on chromatin during mitosis, epigenetically sustains competency for phenotype-specific gene expression in progeny cells. Gene expression is predominantly suppressed during mitosis. Bookmarking provides the guidance for the resumption of gene expression in progeny cells that is obligatory for physiological control of lineage commitment, specialized cell structure and phenotypic function. While regulatory continuity is supported by the persistence of genome-associated regulatory complexes, altered bookmarking mediates plasticity for responsiveness to physiological cues. Bookmarking fidelity ensures genome integrity and controls expression of tumor suppressors and proto-oncogenes. Cancer-compromised aberrations in bookmarking results in transcriptional dysregulation and the initiation of tumor-associated processes. Keywords: Mitotic Bookmarking, Mitosis, Epigenetic Control, Gene Expression, Cell Division.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1d78052a96f4610f52d1801e1448869b1e34100" target='_blank'>
              Mitotic Bookmarking Provides Epigenetic Persistence or Plasticity for Biological Control and Cancer.
              </a>
            </td>
          <td>
            Andrew J. Fritz, Emory Pacht, R. Toor, Janine Warren, Haley W Greenyer, Jackson R Del Porto, Abigail G Person, Sadie J Korzec, Georgiy Zotkin, Jessica L Heath, Prachi N Ghule, J. Gordon, Andre J. van Wijnen, Seth E Frietze, Karen C. Glass, Jane Lian, Jane L. Stein, Gary S. Stein
          </td>
          <td>2025-07-16</td>
          <td>Biochemistry and cell biology = Biochimie et biologie cellulaire</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background: Pancreatic ductal adenocarcinoma (PDAC) is a highly heterogeneous malignancy, characterized by low tumor cellularity, a dense stromal response, and intricate cellular and molecular interactions within the tumor microenvironment (TME). Although bulk omics technologies have enhanced our understanding of the molecular landscape of PDAC, the specific contributions of non-malignant immune and stromal components to tumor progression and therapeutic response remain poorly understood. Methods: We explored genome-wide DNA methylation and transcriptomic data from the Cancer Genome Atlas Pancreatic Adenocarcinoma cohort (TCGA-PAAD) to profile the immune composition of the TME and uncover gene co-expression networks. Bioinformatic analyses included DNA methylation profiling followed by hierarchical deconvolution, epigenetic age estimation, and a weighted gene co-expression network analysis (WGCNA). Results: The unsupervised clustering of methylation profiles identified two major tumor groups, with Group 2 (n = 98) exhibiting higher tumor purity and a greater frequency of KRAS mutations compared to Group 1 (n = 87) (p < 0.0001). The hierarchical deconvolution of DNA methylation data revealed three distinct TME subtypes, termed hypo-inflamed (immune-deserted), myeloid-enriched, and lymphoid-enriched (notably T-cell predominant). These immune clusters were further supported by co-expression modules identified via WGCNA, which were enriched in immune regulatory and signaling pathways. Conclusions: This integrative epigenomic–transcriptomic analysis offers a robust framework for stratifying PDAC patients based on the tumor immune microenvironment (TIME), providing valuable insights for biomarker discovery and the development of precision immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec3422e79e1fa464a39c9ceade9afd8d95e43c98" target='_blank'>
              Deciphering the Heterogeneity of Pancreatic Cancer: DNA Methylation-Based Cell Type Deconvolution Unveils Distinct Subgroups and Immune Landscapes
              </a>
            </td>
          <td>
            Barbara Mitsuyasu Barbosa, Alexandre Todorovic Fabro, Roberto da Silva Gomes, C. Rainho
          </td>
          <td>2025-09-05</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Dysregulated expression of circular RNAs (circRNAs) has been implicated in the initiation and progression of various diseases, including cancer. In this study, we investigated the role of hsa_circ_0005571 in modulating the biological characteristics of colorectal cancer (CRC) cells. Initially, the circRNA expression profiles of CRC tissues and corresponding normal tissues were analyzed using bioinformatics to identify differentially expressed circRNAs. Subsequently, quantitative real-time PCR (qRT‒PCR) was used to validate the expression levels of hsa_circ_0005571 in both CRC tissues and cell lines. In vitro assays, including the Cell Counting Kit-8 (CCK-8), colony formation, 5-ethynyl-2′-deoxyuridine (EdU) incorporation, wound healing, and Transwell invasion assays, were employed to assess the effects of hsa_circ_0005571 on CRC cell proliferation and invasion. Additionally, tumorigenic potential was examined through in vivo tumorigenesis experiments in nude mice. Bioinformatic predictions and experimental evidence indicated that hsa_circ_0005571 may function as a sponge for miR-520f-3p, which was found to be downregulated in CRC cells. Moreover, LRP8 was predicted and confirmed as a direct downstream target of miR-520f-3p, with both the mRNA and protein levels of LRP8 significantly elevated in CRC cells. Overall, our results suggest that hsa_circ_0005571 enhances CRC proliferation and invasion by modulating the miR-520f-3p/LRP8 axis. Furthermore, elevated hsa_circ_0005571 expression in CRC tissue samples and cell lines relative to that in normal controls was positively correlated with metastasis (p = 0.029) and advanced clinical stage (p = 0.006). Survival analyses revealed that CRC patients with high levels of hsa_circ_0005571 expression had significantly lower overall survival rates than did those with low expression (p < 0.05). Functional assays confirmed that hsa_circ_0005571 facilitates CRC cell proliferation and migration, whereas its knockdown results in reduced migration, proliferation, and tumorigenic potential in vivo. Finally, Western blot analyses demonstrated that LRP8 expression varied in accordance with the levels of hsa_circ_0005571 and miR-520f-3p, further confirming the involvement of the hsa_circ_0005571/miR-520f-3p/LRP8 regulatory axis in CRC. Our study suggested that hsa_circRNA_0005571 promotes proliferation and migration through the miR-520f-3p/LRP8 axis in CRC. Consequently, hsa_circ_0005571 may represent a promising novel gene target for the diagnosis and treatment of CRC. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-16450-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da3bdc932b4f1529cee6d8b185f8814c7089e79a" target='_blank'>
              Hsa_circ_0005571 promotes the proliferation and invasion of colorectal cancer cells
              </a>
            </td>
          <td>
            Haipeng Ge, Jiaxuan Zhang, Ran Tao, Li Ding, Song Jiang, Jiale Shen, L. Bian, Jun-Jiao Qin
          </td>
          <td>2025-08-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d320162041d03e1b05ee82602e945a36e317508" target='_blank'>
              Hemocytes facilitate interclonal cooperation-induced tumor malignancy by hijacking the innate immune system in Drosophila.
              </a>
            </td>
          <td>
            Sihua Zhao, Yifan Guo, Xiaoyu Kuang, Xiaoqin Li, Chenxi Wu, Peng Lin, Qi Xie, Zongzhao Zhai, D. Kong, Xianjue Ma
          </td>
          <td>2025-08-22</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) with rearrangement of the mixed lineage leukemia gene express MLL-AF9 fusion protein, a transcription factor that impairs differentiation and drives expansion of leukemic cells. We report here that the zinc finger protein GFI1 together with the histone methyltransferase LSD1 occupies the promoter and regulates expression of the lncRNA ELDR in the MLL-r AML cell line THP-1. Forced ELDR overexpression enhanced the growth inhibition of an LSD1i/ATRA combination treatment and reduced the capacity of these cells to generate leukemia in xenografts, leading to a longer leukemia-free survival. We found that ELDR binds the clamp protein PCNA and the MCM5 helicases causing defects of DNA replication fork progression. Moreover, AML cells overexpressing ELDR showed reduced chromatin accessibility and transcription at α-satellite repeats in centromeres. In addition, ELDR RNA was detected close to MLL-AF9 at centromeres suggesting that it impedes leukemic progression preferentially of MLL-r AML by interfering with both DNA replication and centromeric transcription. Our findings reveal novel functions of the lncRNA ELDR in DNA replication and centromere biology when expressed at high levels in AML cells with MLL rearrangements. These discoveries could provide rationale for future strategies to treat MLL-r AML, which has a poor prognosis in children and adults. Delivery of the ELDR RNA could potentially be utilized as an adjunct to LSD1i/ATRA treatment or other currently used chemotherapeutic drugs to develop novel therapies for these AML subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bca6f7f7a7b4fc0402c634bb0d6e344f0b3e15b" target='_blank'>
              The lncRNA ELDR suppresses tumorigenicity of AML by interfering with DNA replication and chromatin accessibility.
              </a>
            </td>
          <td>
            K. Arman, Julie Ross, Eva-Maria Piskor, Luca Lazzari, Virginie Calderon, Gaspard Reulet, Maria Vera, Edlie St Hilaire, Hugo Wurtele, Brian T Wilhelm, T. Möröy
          </td>
          <td>2025-09-08</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Survival strategy of bacteria is expanded by extrachromosomal elements (ECEs). However, their genetic diversity and functional roles for adaptability are largely unknown. Here, we discover a novel family of intracellular ECEs using 56 saliva samples by developing an efficient microbial DNA extraction method coupled with long-read metagenomics assembly. Even though this ECE family was not hitherto identified, our global prevalence analysis using 476 salivary metagenomic datasets elucidates that these ECEs reside in 74% of the population. These ECEs, which we named, “Inocles”, are giant plasmid-like circular genomic elements of 395 kb in length, including Streptococcus as a host bacterium. Inocles encode a series of genes that contribute to intracellular stress tolerance, such as oxidative stress and DNA damage, and cell wall biosynthesis and modification involved in the interactions with oral epithelial cells. Moreover, Inocles exhibit significant positive correlations with immune cells and proteins responding to microbial infection in peripheral blood. Intriguingly, we examine and find their marked reductions among 68 patients of head and neck cancers and colorectal cancers, suggesting its potential usage for a novel biomarker of gastrointestinal cancers. Our results suggest that Inocles potentially boost the adaptive capacity of host bacteria against various stressors in the oral environment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93ef6b2491714273e3a5acd966b2b147c6dd3e3d" target='_blank'>
              Giant extrachromosomal element “Inocle” potentially expands the adaptive capacity of the human oral microbiome
              </a>
            </td>
          <td>
            Yuya Kiguchi, Nagisa Hamamoto, Y. Kashima, L. Runtuwene, A. Ishizaka, Yuta Kuze, T. Enokida, N. Tanaka, Makato Tahara, S. Kageyama, T. Fujisawa, Riu Yamashita, A. Kanai, Josef S. B. Tuda, Taketoshi Mizutani, Yutaka Suzuki
          </td>
          <td>2025-08-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="RNAs adopt heterogenous folded structures that are essential for function and thus play critical roles in cellular biology. An example of this is the ribosome, a complex, three-dimensionally folded macromolecular machine that coordinates protein synthesis. Advances in RNA biochemistry, structural and molecular biology, and bioinformatics have unveiled other non-coding RNAs whose functions are dictated by their structure. It is not surprising that aberrantly folded RNA structures enable disease. 
R-loops are nucleic acid structures encompassing an RNA-DNA hybrid and a displaced single-stranded DNA (ssDNA) (Hegazy et al., 2019). Once thought to be uncommon transcriptional byproducts, but reclassified as prevalent and functionally significant in a variety of eukaryotic and prokaryotic species. During transcription, R-loops naturally form when the developing RNA hybridizes with the DNA template strands; displacing the non-template strand and creating a three-stranded structure. RNA loops are fundamental secondary structural elements that play crucial roles in RNA function and interaction within cells. Recent developments in immunoprecipitation techniques (e.g., DRIP-seq) and high-throughput sequencing have made it possible to profile R-loops throughout the genome, demonstrating their occurrence in both coding and non-coding areas. The RNA loops are involved in a wide range of biological processes spanning genome regulation, physiology and repair. Mounting evidence also supports R-loop deregulation as a frequent, initiating, event during the development of several human pathologies, such as cancer and neurological disorder. Several applications of RNAloop in biomedical research, clinical and diagnostics underscores their application potential. With this background, we update the current available literature nd review diverse aspects of RNAloops traversing from their roles in cell biology and physiology, genome regulation and instability, the cellular mechanisms and methods of detection. Finally, we in detail curate literature of their roles in Human diseases with emphasis on neurodegenerative diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c3db695265d37192e791c7dff9f5cd030549c5d" target='_blank'>
              RNA-Loops (DNA/RNA Hybrids) Mechanisms, Genomic and Cellular Roles-A Review with Emphasis on Neuropsychiatric Diseases
              </a>
            </td>
          <td>
            Kiran Kumar H.B, Mahalakshmi B.R, Priya M.D., Rajesha S, R. H. S, Latha K, Ranjini. P, Shankar J, K. R. Kini, S.Chandra Nayak, Shailasree Sekhar
          </td>
          <td>2025-08-07</td>
          <td>Journal of Applied Life Sciences International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9a4403306942197c6791291eb7e939f11dbad7" target='_blank'>
              Transposable elements in health and disease: Molecular basis and clinical implications.
              </a>
            </td>
          <td>
            Yaqiang Hong, Nian Liu
          </td>
          <td>2025-08-15</td>
          <td>Chinese medical journal</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The bacterial cytoplasm is characterized by a distinctive membrane-less organelle, the nucleoid, which harbors chromosomal DNA. We investigate the effects of dynamic processes associated with transcription and translation on the structure of this organelle, using coarse-grained molecular dynamics (MD) simulations implemented with out-of-equilibrium reactions. Our model captures the scale of the entire cell and incorporates a reaction-diffusion system for ribosomes and polyribosomes, combining their out-of-equilibrium dynamics with excluded volume interactions with DNA. Our findings demonstrate that out-of-equilibrium reactions increase the size of the nucleoid. In addition, we show that the nucleoid size increase is proportional to transcriptional activity. Our model reproduces the time-dependent change in nucleoid size observed in rifampicin treatment experiments, where the pool of polyribosomes is depleted. Furthermore, we find these active processes are essential for complete sister chromosome separation and correct nucleoid positioning within the cell. Overall, our study reveals the effects of the central dogma processes on the internal organization and localization of bacterial nucleoids.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5869c17c0b2f03dcafbf482f5c2ecb59859979e3" target='_blank'>
              Effects of central dogma processes on the compaction and segregation of bacterial nucleoids
              </a>
            </td>
          <td>
            Mu-Hung Chang, M. Lavrentovich, Jan-Michael Y. Carrillo, Jaan Männik
          </td>
          <td>2025-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d05a5f9576a9619318986d87c824d703b9e97b8" target='_blank'>
              Targeted DNA ADP-ribosylation triggers templated repair in bacteria and base mutagenesis in eukaryotes.
              </a>
            </td>
          <td>
            Darshana Gupta, Constantinos Patinios, H. V. Bassett, A. Kibe, Scott P. Collins, Charlotte Kamm, Yanyan Wang, Chengsong Zhao, Katie Vollen, C. Toussaint, Irene Calvin, G. Cullot, Eric J. Aird, Kathryn M. Polkoff, T. Nguyen-Vo, Angela Migur, Friso T Schut, Ibrahim S. Al’Abri, Tatjana Achmedov, Alessandro Del Re, Jacob E. Corn, A. Saliba, Nathan C. Crook, A. Stepanova, J. M. Alonso, Chase L. Beisel
          </td>
          <td>2025-09-04</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Background DNA changes induced by stress may be stable through cell division and passed to subsequent generations. Plant improvement programs require that individuals develop a new heritable trait. The action of mutagens applied in classical mutagenesis is known from their transgenerational inheritance, observed as mutations. DNA methylation is an epigenetic mechanism that generates heritable phenotypic variation by influencing gene expression and modifying DNA accessibility to mutagens. Fagopyrum esculentum (common buckwheat) and F. tataricum (Tartary buckwheat) are essential for food production and valuable for medical purposes. Results In this study, we aimed to investigate the transgenerational changes in the genome and epigenome of two Fagopyrum species, by analysing 1st and 3rd generations, followed by treatment with gamma rays. The genomic instability, observed as DNA fragmentation, micronuclei, and changes in cell cycle profile, was observed in 1st generation, whereas 3rd generation shows recovery in DNA stability. Gamma-ray stress caused alterations in the pattern and level of DNA methylation in the roots of both analysed generations. The response to gamma rays depends on the species – in 1st generation, a higher increase in DNA methylation level in F. esculentum and only slight changes in F. tataricum were observed. The DNA methylation level in the 3rd generation changed only in F. esculentum. Conclusions Gamma ray is a stress factor that affects the pattern and level of DNA methylation in F. esculentum and F. tataricum. Significant differences were observed between control and 3rd generation only in F. esculentum. Differences in DNA methylation following gamma irradiation between the analysed Fagopyrum species may result from various forms of pollination in the analysed species. By using gamma ray-induced mutation, we got a highly stable mutant of F. tataricum, which may result in a self-pollinating trait. Understanding the changes in DNA methylation following mutagenic treatment can be essential in a breeding programme. Supplementary Information The online version contains supplementary material available at 10.1186/s12870-025-07033-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16c669739a40e63380b2299a72c170061851584f" target='_blank'>
              Transgenerational genetic and epigenetic changes induced by gamma-ray in Fagopyrum species
              </a>
            </td>
          <td>
            Katarzyna Sala-Cholewa, Magdalena Rojek-Jelonek, Jolanta Kwasniewska
          </td>
          <td>2025-07-31</td>
          <td>BMC Plant Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Neoplastic transformation of mammalian cells involves intricate interactions between genetic, epigenetic and architecture modifications of the nucleus. Neuroblastoma is a malignant pediatric tumor with high biological and clinical heterogeneity representing a challenging model of study. We aimed to explore the changes in genome architecture and epigenetics being associated with neuroblastoma malignancy. We employed the neuroblastoma cell line SKNBE2 overexpressing the ncRNA NDM29 to differentiate from highly malignant into neuron-like cells. By 3D confocal microscopy, we explored the nuclear architecture (volume, elongation, compactness, and chromatin density). Using super-resolution microscopy (STED) and histone H3 immunolabelling we assessed the epigenetic rearrangement, and by enzyme-linked immunoassay the global DNA methylation. Then we assessed the mRNA expression of the main epigenetic modifying enzymes by quantitative PCR, and the expression of NF-κB-regulated genes by cDNA microarray. Compared to malignant NB cells, the NDM29-overexpressing cells, assuming a neuron-like phenotype, exhibited smaller and more elongated nuclei, redistribution of H3K9-acetylated and -methylated chromatin domains and DNA hypermethylation. In line with these results, in neuron-like cells the acetyltransferase KAT2A and the DNA methyltransferase DNMT1 were up-regulated, while most of NF-κB-regulated genes were down-regulated. Our findings reveal modifications of the nuclear structure and epigenome during neuroblastoma retro-transformation induced by NDM29 overexpression, with impacts on gene expression. These results offer potential insights into better understanding the mechanism of neuroblastoma malignancy in terms of chromatin rearrangements, opening exciting prospects for prognostic and therapeutic approaches with a focus on the nuclear level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53128c07de4d9b019036fd21bd73a5a365d261f0" target='_blank'>
              Epigenome and three-dimensional genome architecture remodeling during NDM29-mediated retro-transformation of neuroblastoma cells
              </a>
            </td>
          <td>
            Francesca Baldini, Aldo Pagano, L. Zeaiter, P. Bianchini, Hawraa Zbeeb, A. Diaspro, Laura Vergani
          </td>
          <td>2025-07-31</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Oncogenesis, tumor progression and therapy response are shaped by somatic alterations in the cancer genome and features of the tumor immune microenvironment (TME). How interactions between these two systems influence tumor evolution and clinical outcomes remains incompletely understood. To address this challenge, we developed the multi-omics analysis framework PACIFIC that systematically integrates genetic cancer drivers and infiltration profiles of immune cells to find pairwise combinations of drivers and TME characteristics that jointly associate with clinical outcomes. By analyzing 8500 primary tumor samples of 26 cancer types, we report 34 immunogenomic interactions (IGXs) in 13 cancer types in which context-specific combinations of genomic alterations and immune cell levels were significantly correlated with patient survival. Subsets of tumor samples defined by some IGXs were characterized by tumor-intrinsic and microenvironmental metrics of immunogenicity and differential expression of immunotherapy target genes. In luminal-A breast cancer, an IGX involving MEN1 deletion combined with reduced levels of neutrophils associated with lower progression-free survival and deregulation of immune signaling pathways, as observed in two independent cancer genomics datasets. These results showcase the ability of PACIFIC to integrate complex multi-omics datasets with clinical information, enabling the identification of clinically relevant immunogenomic interactions. Such interactions provide a rich set of hypotheses for mechanistic studies and the development of biomarkers and therapeutic targets. Implications: Co-occurrence patterns of cancer drivers and TME characteristics highlight synergistic interactions with prognostic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b3413ab1802170ae0154c57fdbd8827be618414" target='_blank'>
              Cancer genomic alterations and microenvironmental features encode synergistic interactions with disease outcomes.
              </a>
            </td>
          <td>
            Masroor Bayati, Zoe P Klein, Alexander T Bahcheli, Mykhaylo Slobodyanyuk, Jeffrey To, Kevin C L Cheng, Jigyansa Mishra, Diogo Pellegrina, K. Guevara-Hoyer, Chris McIntosh, Mamatha Bhat, J. Reimand
          </td>
          <td>2025-09-03</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Lung squamous cell carcinoma (LUSC) remains a lethal malignancy with limited therapeutic advancements, underscoring the need to identify novel oncogenic drivers. This study integrates multi-omics analyses-including ChIP-seq, bulk RNA-seq, single-cell ATAC/RNA-seq, and spatial transcriptomics-to delineate enhancer-driven transcriptional networks in 59 matched LUSC tumors and adjacent normal tissues. The analyses identify 3,447 tumor-specific oncogenic enhancers (SOEs) enriched for master transcription factors (SOX2, TP63, KLF5, GRHL2) that orchestrate malignant programs. Single-cell epigenomic profiling reveals these SOEs to be exclusively active in a cancer cell-like cluster, pinpointing PTPRZ1, a receptor tyrosine phosphatase, as a top SOE-driven candidate. Functional studies demonstrate that PTPRZ1 is essential for LUSC-tumorigenesis and tumor cells proliferation and migration in vitro and in vivo. Spatial transcriptomic analysis further implicates midkine (MDK) as the ligand activating PTPRZ1 in LUSC tumor tissue. Mechanistically, PTPRZ1 mediates MDK-induced PI3K phosphorylation that is essential for LUSC tumor growth. This work illustrates enhancer addiction as a hallmark of LUSC, identifies the PTPRZ1-MDK axis as an important targetable pathway, and establishes a paradigm for dissecting epigenetic vulnerabilities in solid tumors through multi-omics integration. These findings advance precision oncology strategies for LUSC, bridging epigenomic dysregulation to therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdd6b0f8500832ade4c7417d38653f9c0588ff5" target='_blank'>
              Enhancer Reprogramming Reveals the Tumorigenic Role of PTPRZ1 in Lung Squamous Cell Carcinoma.
              </a>
            </td>
          <td>
            Yong-Qiang Ning, Jie Wang, Yihui Zhang, Hongrui Li, Zenan Jiang, Xinyu Su, Xiao Yang, Xiaoxuan Wu, Bing Wei, Jianming Ying, Jian Yan, Wenbin Li
          </td>
          <td>2025-09-03</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01fb40ddbc348d6bdf921a2ddd2696429a42111c" target='_blank'>
              Energy Deficiency-Induced ATG4B Nuclear Translocation Inhibits PRMT1-Mediated DNA Repair and Promotes Leukemia Progression.
              </a>
            </td>
          <td>
            Zhenkun Wang, Xianli Zhang, Yuanyuan Zhou, Shuyi Lin, Yuting Fu, Haiping Liu, Qiongdan Gao, YanHong Xiao, Zhao Yin, Shuang Liu, Kexiu Huang, Suqi Deng, Juan Du, Hui Zeng, Jia Wang, Jinping Zheng, Qinghua Zhou, Tianfeng Chen, Xiaoling Gao, Zhenyu Ju, Bo Liu
          </td>
          <td>2025-08-11</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The type II topoisomerases, gyrase and topoisomerase IV, are essential enzymes in nearly all bacteria and are the targets of fluoroquinolones, which are some of the most widely prescribed broad-spectrum antibacterials in clinical use. As part of their catalytic cycle, gyrase and topoisomerase IV transiently cleave DNA in a sequence-dependent manner. However, it is unclear whether this sequence-dependence is species-specific. Therefore, using our recently developed SHAN-seq method, we mapped and compared cleavage sites for type II topoisomerases from three different pathogenic bacterial species, Escherichia coli, Bacillus anthracis, and Mycobacterium tuberculosis in the presence of the fluoroquinolone, ciprofloxacin. We found that the enzymes have substantially different DNA cleavage specificities that vary between gyrase and topoisomerase IV, across species, with supercoil chirality, and in response to ciprofloxacin. Our results demonstrate that bacterial species fine-tune the DNA cleavage specificity of their type II topoisomerases. This finding suggests that cleavage specificity may play important physiological roles and, in turn, may affect the susceptibility of bacteria to fluoroquinolone antibacterials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e082902e5461cf61bfe338f2489bf47116a6cee" target='_blank'>
              Bacterial type II topoisomerases cleave DNA in a species-specific manner
              </a>
            </td>
          <td>
            Ian L. Morgan, Jeffrey Y Jian, N. Osheroff, K. Neuman
          </td>
          <td>2025-07-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Alkyladenine DNA glycosylase (AAG) is a critical enzyme in the base excision repair (BER) pathway, responsible for removing a broad spectrum of alkylated DNA lesions. While AAG maintains genomic stability, dysregulated activity has been implicated in cancer development, drug resistance, and neurodegenerative diseases. This review synthesizes the current knowledge on AAG’s structure, catalytic mechanism, and polymorphic variants, highlighting their potential roles in disease pathogenesis. A comprehensive bioinformatics analysis of over 370 AAG single-nucleotide polymorphisms (SNPs) is presented, identifying ~40% as high-risk variants likely to impair enzymatic function. Notably, 151 SNPs were predicted to be damaging by multiple algorithms, including substitutions at catalytic residues and non-conserved sites with unknown functional consequences. Analysis of cancer databases (COSMIC, cBioPortal, NCBI) revealed 93 tumor-associated AAG variants, with 18 classified as high-impact mutations. This work underscores the need for mechanistic studies of AAG variants using structural biology, cellular models, and clinical correlation analyses. Deciphering AAG’s polymorphic landscape may unlock personalized strategies for cancer prevention and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f35650f59a2a5d69d14473a41169b47ec06f0170" target='_blank'>
              The Link Between Human Alkyladenine DNA Glycosylase and Cancer Development
              </a>
            </td>
          <td>
            O. A. Kladova, A. A. Kuznetsova
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Newcastle disease virus (NDV), as an avian pathogen, can infect a broad spectrum of cell lines in vitro. However, noncarcinoma cell lines possessing nonsusceptibility to NDV are rare. Here, we isolated primary mouse embryonic fibroblasts (MEFs), which are nonsusceptible to NDV. MEF-derived cells were generated by passaging the cells over fifty times to achieve spontaneous immortalization. Two of the resulting cell lines were named SLM-21 and MEF50. Karyotype analysis revealed that SLM-21 has a near-tetraploid karyotype and that MEF50 shows a near-tetraploid and near-hexaploid chimeric karyotype. NDV exerted a significant cytopathic effect on MEF50, and substantial viral replication was observed. In contrast, NDV did not have a significant effect on SLM-21, indicating that SLM-21 was a nonsusceptible cell line to NDV, while MEF50 was a susceptible cell line. The NDV authentic sialic acid (SA) receptors SA 2,3-Gal and SA 2,6-Gal were expressed in SLM-21. Transcriptomic analysis revealed that the non-susceptibility of SLM-21 may be related to its broadly activated antiviral pathways and fine-tuned regulation of the cell cycle and DNA damage. This study provides a basic cell platform for exploring viral susceptibility and pathogenesis as well as host–virus interactions during NDV infection. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-14404-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09200173ad224d2368f7c735d3ba4913547a95de" target='_blank'>
              A noncarcinoma mouse cell line is nonsusceptible to Newcastle disease virus established by spontaneous immortalization
              </a>
            </td>
          <td>
            Hanlue Li, Zijie Zhou, Chuanyu Chang, Weiwei Shi, Yuzhen Li, Qiaolin Wei, Fanxing Meng, Jian-yuan Tian, Qiqi Hao, Jinming Liu, Haijin Liu, Zengqi Yang, Sa Xiao
          </td>
          <td>2025-08-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Retinoblastoma is the most common intraocular malignant tumor in childhood. Although current treatments offer a high survival rate, treatment toxicity, tumor relapse, and treatment resistance require a deeper understanding of the disease mechanisms to develop adapted therapies. Microscopically, this tumor is characterized by different states of differentiation and proliferation, ranging from poorly differentiated to well-differentiated retinoblastoma. Retinocytoma, on the other hand, is a benign non-proliferative tumor. Recent next-generation multi-omics analyses classified retinoblastoma tumors in subtypes 1 or 2, subtype 2 presenting a later age of onset, more genetic alterations, and higher metastatic potential. In parallel, several single-cell transcriptomics studies demonstrated intratumoral heterogeneity. However, mapping the different cell populations directly on the tumor and comparing histological features and molecular subtypes remains an unmet need. Methods Spatial transcriptomics was used to characterize a primary enucleated case with two histologically distinct areas. The whole transcriptomic profile of sixteen regions of interest, covering the two differentiation patterns of the tumor and the non-tumoral retina, was analyzed. Results The clustering of the regions of interest based on whole transcriptome data correlated with the histological description: cluster 1 (6 regions of interest) corresponded to highly differentiated areas and cluster 2 (6 regions of interest) to poorly differentiated areas. They showed enrichment for phototransduction and proliferation respectively, confirmed by immunohistochemistry for markers of these pathways. The publicly available molecular signatures of the two retinoblastoma subtypes categorized our regions of interest into two groups, which correlated perfectly with histological observation and transcriptomic profiles. Further analysis of the expression of specific senescence markers in the well-differentiated area did not support the diagnosis of retinocytoma as it did not confirm the expected up-regulation seen in this tumor type. Conclusions This study demonstrates a strong correlation between histological observation and molecular profiling, representing the first mapping of retinoblastoma composed of adjacent molecular subtypes 1 and 2. It also highlights the diagnostic ambiguity between retinocytoma and well-differentiated non-proliferative subtype 1 retinoblastoma and emphasizes the need for specific biomarkers to differentiate these two tumor types. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14814-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/599058c9510122a1b04d9c9b60c21f33e8817b4e" target='_blank'>
              Spatial transcriptomics of retinoblastoma: a visual window on intra-patient heterogeneity
              </a>
            </td>
          <td>
            A. Moulin, J. Thevenet, L. Mazzeo, S. Tissot, C. Stathopoulos, F. L. Munier, A. Berger
          </td>
          <td>2025-09-02</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="ABSTRACT Background Gastric cancer (GC) is the fourth leading cause of cancer‐related death globally. Tumor profiling has revealed actionable gene alterations that guide treatment strategies and enhance survival. Among Hispanics living in Puerto Rico (PRH), GC ranks among the top 10 causes of cancer‐related death. However, the genetic mutational landscape of GC tumors from PRH remains unexplored. This study aimed to identify the most prevalent genetic alterations in GC tumors among PRH. Methods We examined tumor mutational profiles of GC from 106 PRH between 2015 and 2022 (provided by CARIS Life Sciences and the Precision Oncology Alliance). Next‐generation sequencing data were available for 85 cases, which were categorized as hypermutated (≥ 10 mutations/megabase) or non‐hypermutated (< 10 mutations/megabase). Results Among the non‐hypermutated cases, the most frequently mutated genes were TP53 (56.9%), CDH1 (29.2%), ARID1A (27.4%), and KMT2D (25.7%). Compared to TCGA, a majority non‐Hispanic cohort, PRH had significantly higher mutational frequencies in driver genes in both intestinal type (TP53, CBLB, and MYH11) and diffuse type (CDH1, ARID1A, and KMT2D) GC. Discussion Intestinal‐type GC in PR aligns with the chromosomal instability (CIN) molecular classification, showing a higher frequency of TP53 mutations than TCGA, potentially indicating more aggressive tumor biology and poorer prognosis. Diffuse‐type GC showed higher CDH1 mutations, correlating with the genomically stable (GS) classification, characterized by fewer chromosomal changes but significant genetic alterations, including those in ARID1A and KMT2D. Conclusions This study shows that the unique genetic landscape of GC tumors in this group may lead to more aggressive cases and affect treatment responses, contributing to higher mortality rates. The higher mutation rates in biomarkers related to prognosis and therapy suggest that further research might uncover additional susceptibility variants. This underscores the importance of including Hispanics in genomic studies to better understand the genetic pathways associated with the risk and progression of GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/863fb64c7ea6f314845067da6d3c3ed255479243" target='_blank'>
              Actionable Genes and Carcinogenic Pathways for Gastric Cancer in Latinos
              </a>
            </td>
          <td>
            Ingrid M. Montes-Rodríguez, Hilmaris Centeno-Girona, Sol V Pérez-Mártir, Noridza Rivera, Marcia Cruz-Correa
          </td>
          <td>2025-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="


 Nasopharyngeal Carcinoma (NPC) is a highly metastatic malignancy, often diagnosed at advanced stages with poor prognosis and survival rates. It is strongly associated with Epstein-Barr virus (EBV) exposure and Chinese/Asian ethnic backgrounds, leading to approximately 300 new cases annually in Singapore. EBV plays a critical role in NPC development, with both its latent and lytic phases contributing to tumorigenesis. The virus precisely controls its methylation pattern to regulate latent gene expression and lytic reactivation. In contrast, disruptions in DNA modification patterns in humans are frequently observed in tumorigenesis, suggesting transcriptional dysregulation. Currently, few studies have explored limited loci in human and EBV epigenomes, with most only profiling population-level average methylation status in NPC. These studies cannot distinguish between 5-hydroxymethylcytosine (5hmC) and 5-methylcytosine (5mC), which have opposing regulatory roles. Moreover, existing methods fail to access repetitive regions, which comprise significant portions of both the EBV and human genomes. To address these gaps, we generated whole-genome, single-molecule, single-base resolution profiles of 5mC and 5hmC modifications in EBV and human DNA from different NPC tumor samples and cell lines using direct Nanopore ultralong-read sequencing. This approach offers unique insights into the complexities of EBV and human DNA modifications during NPC development. We discovered that distinct EBV strains cause contrasting 5mC changes in both human and EBV epigenomes, with some inducing hypermethylation and others hypomethylation. While hypomethylation was previously linked to APOBEC activity, our findings suggest that EBV strain-specific sequences play a significant role. Notably, in EBV epigenome, 5mC differences are concentrated in repetitive regions, whereas 5hmC profiles are consistent across strains, with persistent 5hmC modifications at specific loci. Our study provides valuable epigenome knowledge to the field of NPC research. Further data and detailed mechanistic studies are crucial to deepening our understanding of disease onset, progression, and advancing early detection and therapeutic strategies.



 Li Zhe, Yao Fei, Sia Yee Yen, Melvin Chua Lee Kiang, Joshua Tay, Liu Jianjun. Towards Profiling DNA Modifications of Virus and Human Host DNA in Epstein-Barr Virus (EBV)-positive Nasopharyngeal Carcinoma (NPC) with Nanopore Ultra-Long Reads Sequencing [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P52.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e731f932485dff94f0e6e21fa72516a251e45df3" target='_blank'>
              Abstract P52: Towards Profiling DNA Modifications of Virus and Human Host DNA in Epstein-Barr Virus (EBV)-positive Nasopharyngeal Carcinoma (NPC) with Nanopore Ultra-Long Reads Sequencing
              </a>
            </td>
          <td>
            Li Zhe, Yao Fei, Sia Yee Yen, Melvin Chua Lee Kiang, Joshua Tay, Jianjun Liu
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Multiple Myeloma (MM) is a malignant plasma cell dyscrasia that progresses through the consecutive asymptomatic, often undiagnosed, precancerous stages of Monoclonal Gammopathy of Undetermined Significance (MGUS) and Asymptomatic Multiple Myeloma (SMM). MM is characterized by low survival rates, severe complications and drug resistance; therefore, understanding the molecular mechanisms of progression is crucial. This study aims to detect genetic mutations, both germline and somatic, that contribute to disease progression and drive tumorigenesis at the final stage of MM, using samples from patients presenting MGUS or SMM, and newly diagnosed MM patients. Methods: Mutations were identified through a fully computational pipeline, implemented in a Linux and RStudio environment, applied to each patient sequence, obtained through single-cell RNA-sequencing (scRNA-seq), separately. Structural and functional mutation types were identified by stage, along with the affected genes. The analysis included quality control, removal of the Unique Molecular Identifiers (UMIs), trimming, genome mapping and result visualization. Results: The findings revealed frequent germline and somatic mutations, with distinct structural and functional patterns across disease stages. Mutations in key genes were identified, pointing to molecules that may play a central role in carcinogenesis and disease progression. Notable examples include the HLA-A, HLA-B and HLA-C genes, as well as the KIF, EP400 and KDM gene families, with the first four already confirmed. Comparative analysis between the stages highlighted molecular transition events from one stage to another. Emphasis was given to novel genes discovered in newly diagnosed MM patients, that might contribute to the tumorigenesis that takes place. Conclusions: This study contributes to the understanding of the genetic basis of plasma cell dyscrasias and the transition events between the stages, offering insights that could aid in early detection and diagnosis, guide the development of personalized therapeutic strategies, and improve the understanding of mechanisms responsible for resistance to existing therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db8c2ca600e5f8d213e747b7e35b473639690b95" target='_blank'>
              Genomic Insights into Tumorigenesis in Newly Diagnosed Multiple Myeloma
              </a>
            </td>
          <td>
            Marina Kyriakou, C. Papaloukas
          </td>
          <td>2025-08-23</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73c8f319f11687277fd179aa2df80ba7f003d32d" target='_blank'>
              Loss of the Y chromosome drives cancer metabolic reprogramming.
              </a>
            </td>
          <td>
            Boopathi Subramaniyan, Bipul R. Acharya, Prince Anand, Saravana Kumar Kailasam Mani, Saswat Mohapatra, Xingyu Chen, Fangyuan Qu, Yesai S Fstkchyan, Corazon Gutierrez, Hany A Abdel-Hafiz, James C. Costello, Dan Theodorescu
          </td>
          <td>2025-08-31</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="ABSTRACT Prostate cancer (PCa) is an extremely heterogeneous cancer and is highly prevalent in the older male population. Since intra‐tumour heterogeneity (ITH) commonly results in PCa chemotherapy resistance and recurrence, it is critical to explore its effects on tumour behaviour. Prognostic genes related to ITH were identified, and a signature was constructed using Cox regression analyses and multiple machine learning algorithms. Single‐cell RNA sequencing data extracted from PCa and CRPC samples were analysed via sub‐clustering, pseudotime, cell communication and drug sensitivity approaches to elucidate their function. The oncogenic potential of hub genes was confirmed by immunohistochemistry and cell proliferation assays. An 11‐gene signature underlying a prostate cancer meta‐program (PCMP) was generated by selecting an optimal combination of machine learning methods. Survival assays and multivariate Cox regression analyses conducted in multiple cohorts revealed the superior prognostic value of the PCMP signature. Functional enrichment analyses indicated that it dysregulates the cell cycle. Using trajectory and cell–cell communication analyses, we illustrated that PCMP genes exert oncogenic effects by enhancing the proliferation and oxidative phosphorylation of epithelial cells. Intra‐cellular assays also demonstrated that CENPA and CKS1B had promising malignant potential. In summary, our research not only establishes the association between the PCMP signature and reveals its malignant characteristics, but also deepens our understanding of the mechanisms underlying PCa progression and ITH. It holds promise for the development of targeted therapeutic interventions, thereby offering clinical benefits to patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/045ffd0cd32bcb5869978af9b2509598525fb2d8" target='_blank'>
              Single‐Cell and Bulk RNA Sequencing Highlights Intra‐Tumoral Heterogeneity and Malignant Progression Mechanisms in Prostate Cancer
              </a>
            </td>
          <td>
            Junchao Wu, Ziqi Chen, Wentian Wu, Jiaxuan Qin, Rongfang Zhong, Jialin Meng, Yu Yin, Peng Guo, Song Fan
          </td>
          <td>2025-08-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0641382c381145c3c4ec79eaca66ce9a1d4b8df0" target='_blank'>
              Telomere Crisis Shapes Cancer Evolution.
              </a>
            </td>
          <td>
            Joe Nassour, J. Karlseder
          </td>
          <td>2025-08-11</td>
          <td>Cold Spring Harbor perspectives in biology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Background Gastric cancer (GC) ranks as the fifth most prevalent malignancy and the third leading cause of cancer-related mortality worldwide, with complex pathogenesis driven by genetic and epigenetic alterations. While genetic contributors to GC have been extensively studied, the functional role of N6-methyladenosine (m6A) RNA methylation—the most abundant eukaryotic RNA modification—in gastric carcinogenesis remains insufficiently characterized. This study aimed to investigate transcriptome-wide m6A methylation dysregulation and its mechanistic implications in GC progression. Methods Methylated RNA immunoprecipitation sequencing (MeRIP-seq) was performed to map m6A epitranscriptomes in paired GC and adjacent normal tissues. Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analyses were conducted on 253 hypomethylated mRNAs to delineate the biological pathways associated with m6A dysregulation. The transcriptomic profiles were analyzed using RNA-seq, while a retrospective cohort analysis (n = 58) evaluated the correlations between IGF2BP3 expression and the clinicopathological characteristics of patients with GC. Results MeRIP-seq analysis demonstrated transcriptome-wide m6A hypomethylation in GC tissues, identifying 271 significantly reduced peaks (p < 0.01). Functional annotation revealed enrichment of hypomethylated transcripts in metabolic pathways and transcriptional dysregulation. Notably, m6A-related genes exhibited widespread activation in GC, with IGF2BP3 showing the most pronounced upregulation (106-fold increase, p < 0.0001). Clinically, elevated IGF2BP3 expression significantly correlated with lymph node involvement (p = 0.016) and advanced TNM staging (p = 0.028). Conclusion This study establishes m6A methylation dysregulation as a critical mechanism in GC pathogenesis and identifies IGF2BP3 as both a potential therapeutic target and a prognostic biomarker in GC. Supplementary Information The online version contains supplementary material available at 10.1186/s40246-025-00802-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91a9f3e1f9eee919cc6ee02a44855814e67eb1ac" target='_blank'>
              Clinical and prognostic significance of m6A hypomethylation and IGF2BP3 overexpression in gastric cancer: an integrated epigenomic-transcriptomic analysis
              </a>
            </td>
          <td>
            Xiang-yun Hu, Zhe Wang, Youwei Kou, Yujing Huang
          </td>
          <td>2025-08-22</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1353c24c2d18ad56030f170e19dffc05bf916315" target='_blank'>
              HPV16 recruitment of SMARCAL1 to viral and host replication forks is required for the viral life cycle.
              </a>
            </td>
          <td>
            C. James, Aya H. Youssef, Jenny D Roe, Floriana Cappiello, F. Aiello, Benedetta Perdichizzi, Rachel L. Lewis, Austin J. Witt, A. T. Prabhakar, Xu Wang, Molly L Bristol, P. Pichierri, I. Morgan
          </td>
          <td>2025-07-22</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Pleomorphic giant cell adenocarcinoma (PGCA) of the prostate is a rare, aggressive variant characterized by multinucleated giant cells, sarcomatoid features, and resistance to conventional therapies. Despite its recognition in the WHO 2016 guidelines, the molecular drivers and clinicopathological correlates of PGCA remain poorly characterized. This study presents the first integrative clinicogenomic profiling of PGCA, revealing a novel prognostic gene signature with direct implications for diagnosis and treatment. Methods We conducted comprehensive clinicopathological and genomic analyses of a treatment-refractory PGCA case using histology, immunohistochemistry (IHC), whole-exome sequencing (WES), clonal evolution modeling, and multicohort validation. IHC assessed key prostate cancer markers (AR, AMACR, KLK3, PTEN, NKX3-1, VIM), while WES compared somatic alterations in PGCA, adjacent adenocarcinoma, and stromal tissue. Public datasets (prostate_dkfz_2018, prad_tcga, prad_mcspc_mskcc_2020) were used for external validation. Results PGCA displayed profound pleomorphism, necrosis, and complete loss of luminal markers (AR/AMACR/KLK3), along with strong vimentin (VIM) expression, consistent with epithelial–mesenchymal transition. WES revealed PGCA-specific mutations enriched in cell-cycle and inflammatory response pathways, distinct from metabolic alterations in the adjacent adenocarcinoma. Clonal evolution analysis showed divergent progression from a shared ancestral clone. Importantly, mutations in ADAMTS7, CDH1, DRD5, MGAT5, and TP53 emerged as a robust five-gene signature predictive of biochemical recurrence, metastasis, and poor survival, validated across multiple independent cohorts. Conclusion Our study provides the first molecular roadmap of prostatic PGCA to date, establishing a novel five-gene prognostic signature and revealing fundamental insights into its pathogenesis through divergent evolution from conventional adenocarcinoma. These insights offer new opportunities for precise diagnosis, prognostic stratification, and targeted therapeutic strategies for this lethal prostate cancer variant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3fd48864bfffc9bb6aed51ef499e504db68fa6c" target='_blank'>
              Clonal divergence and genomic meltdown in prostatic pleomorphic giant cell adenocarcinoma
              </a>
            </td>
          <td>
            Xiaoshi Ma, Kun Chen, Jing Zhang, Liming Liu, Jiping Luo, Kaipeng Huang, Hongying Zhang, Danni Liu, Jizhou Gou, Changyin Feng, Xia Zhao, Wanying Li, Lipeng Chen, Li Yin, Xianlin Meng, Zhiqiang Cheng
          </td>
          <td>2025-08-12</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract While the majority of viruses falls within a handful of well-defined realms, many novel viruses and virus-like genetic elements remain to be discovered. Given the abundance and diversity of small, circular DNA genomes in animal gut viromes, we hypothesized that viruses of similar size might have been overlooked in other environments. Re-analyzing existing ocean virome datasets for small viruses, we discover a novel group of circular mobile genetic elements of ∼4.6kb in size, likely with double-stranded DNA, which we term Charybdis elements. Charybdis elements form a singular group based on cohesive gene content, encode around 10 hypothetical genes but apparently contain no virus, transposon or plasmid associated hallmark proteins. They are detected exclusively in marine datasets ranging from surface water to hydrothermal vents and sediments, from the Arctic to the Antarctic. Virome datasets separating virus-like particles by density gradients show that Charybdis elements accumulate in the fraction representing extracellular vesicles and small, tail-less viruses that are often associated with lipid membranes. One specific clade of Charybdis elements is shown to integrate into the genomes of the common marine group II archaeon candidate order Poseidoniales, the putative host of these elements. Analysis of thousands genomes shows a common genome architecture consisting of two modules: An accessory module containing a diverse range methyltransferases and other genes likely involved in host interactions, and a more conserved (putative) structural module. The structural module encodes proteins that are confidently predicted to form striking tri- or pentameric structures. These structures show stark resemblance and in some cases homology to protrusion or tail-elements on the capsids of viruses from the realms Varidnaviria and Duplodnaviria . Given these traits, Charybdis elements likely represent a hitherto undiscovered group of virus or virus-satellites, with at least one protein of a novel double-Greek-key fold serving as a potential capsid element. However, since their hosts remain unculturable, the true nature of these ubiquitous marine elements remains as of yet unresolvable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88d6ae6d60b7d2c07e786f432ef49127c628ff31" target='_blank'>
              A novel mobile genetic element with virus-like characteristics is widespread in the world’s oceans
              </a>
            </td>
          <td>
            P. Kirchberger, Antoni Luque
          </td>
          <td>2025-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="As the second most deadly cancerous disease worldwide, colorectal cancer (CRC) stands in the center of scientific interest in hope to develop novel approaches for precise diagnostics and prognosis determination. Metastatic disease remains the main cause of CRC mortality. To investigate the underlying genetic differences between CRC patients with synchronous and metachronous liver metastases, we performed whole-exome sequencing of 210 patient samples using formalin-fixed paraffin-embedded samples from primary tumors and the paired liver metastatic tissue. The analyses included types and levels of mutations and copy number variation. APC and TP53 were the most commonly mutated genes in all samples with differing frequency between primary CRC (both 50%) and its metachronous metastasis (both 64%). While MPDZ gene mutations were restricted to primary tumors that developed metachronous metastases only, mutations in VCAN, MTCL1, MDN1, SHROOM2, SPEG, and GLI2 were more prevalent in primary tumors giving rise to synchronous metastases. FBN1 mutations were unique to synchronous liver metastatic tissue. Analysis of genetic interactions revealed different associations between mutated genes in patients with tumors of different chronicity, including driver genes such as TP53, which was associated with APC in synchronous patients, while in primary tumors with metachronous metastases it co-occurs with mutations in NBPF11 and PRAMEF15, respectively. The results suggest that distinct tumor progression pathways account for different chronicity outcomes further affecting patients' survival. However, larger studies are needed incorporating transcriptomic and epigenomic data to shed further light on the mechanistic chain from mutations to downstream gene expression regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d84510875a6bc07b90f6ac9563e83dc402040c5" target='_blank'>
              Genetic differences between primary colorectal cancer and its paired synchronous and metachronous metastases.
              </a>
            </td>
          <td>
            Marie Rajtmajerová, Filip Ambrożkiewicz, V. Hlaváč, A. Trailin, L. Cervenkova, Jan Brůha, S. Šůsová, Pavel Soucek, P. Vodicka, L. Vodickova, O. Kubeček, Stanislav Filip, Venkata R. Mallela, Wenjing Ye, F. Zitricky, Václav Liška, Kari Hemminki
          </td>
          <td>2025-08-30</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Background Endometrial cancer (EC), a type of uterine cancer, is witnessing a global increase in incidence. Despite advancement in diagnosis and treatment, metastatic or recurrent EC often exhibits a poor prognosis, necessitating novel therapeutic strategies. DEAD-box helicase 17 (DDX17) is implicated in several cancers. Our study aimed to uncover the biological function and molecular mechanism of DDX17 in EC. Methods EC and matched adjacent normal tissues from 80 patients were analyzed; DDX17 mRNA/protein expression was quantified via RT-qPCR and immunoblotting in clinical specimens and cell lines (HEC-1A, HEC-1B, Ishikawa), with functional assays (proliferation/migration/invasion) performed following DDX17 overexpression in vitro, while xenograft modeling in BALB/c nude mice enabled in vivo validation through immunofluorescence and immunohistochemical staining; mechanistic studies employed RNA immunoprecipitation (RIP-PCR), m6A-specific RNA immunoprecipitation (MeRIP-PCR), and protein interaction analyses. Results DDX17 was significantly downregulated in EC tissues/cells, correlating with poor prognosis in clinical cohorts. Overexpression of DDX17 suppressed tumorigenesis both in vitro and in vivo through PI3K/AKT pathway inactivation. METTL3-mediated m6A modification stabilized DDX17 mRNA, with IGF2BP2 specifically recognizing m6A-modified transcripts. Critically, METTL3 ablation reversed DDX17 stabilization and abolished its tumor-suppressive effects, while PI3K inhibition (LY294002) phenocopied METTL3 restoration in rescuing DDX17 deficiency-induced oncogenicity. Conclusion METTL3-mediated m6A modification stabilizes DDX17 to suppress EC cell proliferation, migration, and invasion through an IGF2BP2-dependent mechanism by inactivating the PI3K/AKT pathway. Supplementary Information The online version contains supplementary material available at 10.1007/s10238-025-01817-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc74064b9143aa7c166a780986e41cb0e801d2f4" target='_blank'>
              METTL3 stabilizes DDX17 mRNA via IGF2BP2-mediated m6A modification to suppress endometrial cancer progression
              </a>
            </td>
          <td>
            Yuan Zhang, Mengjun Liu, Jing Liu, Jing Zhang
          </td>
          <td>2025-08-07</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Nuclear architecture and chromosome folding are often speculated to influence genome replication. In yeasts, centromeres cluster close to the spindle pole body and telomeres position at the nuclear periphery. This ‘Rabl configuration’ spatially segregates the most early and late replicating parts of the genome, centromeres and telomeres, respectively, suggesting that origin position along the centromere–telomere axis may influence origin activity. Here, we investigated DNA replication in a wild-type, 16-chromosome Saccharomyces cerevisiae strain and in its single-chromosome counterpart engineered by chromosome fusion and elimination of all but two telomeres and one centromere, which strongly affects genome folding and abrogates the Rabl conformation. Using nanopore sequencing-based methods, we found that the DNA replication program of both strains was virtually indistinguishable, with the exception of origin inactivation next to deleted centromeres and changes in origin efficiency and fork direction at chromosome fusions, as anticipated from the known origin-regulation properties of centromeres and telomeres. Only a handful of replication changes, mostly due to local origin repression, were observed elsewhere. Fork speed was also unaffected except at deleted centromeres. In conclusion, the DNA replication program of budding yeast is remarkably resilient to perturbations of chromosome folding and loss of the Rabl conformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e28f5bc4a9bd5e110c57ffd165964c5f1a57708" target='_blank'>
              Replication program of a single-chromosome budding yeast strain
              </a>
            </td>
          <td>
            Jade Pellet, Laurent Lacroix, Bertrand Theulot, Emma Simonin Chavignier, Alan Tourancheau, F. Proux, Sylvie Hermann-Le Denmat, G. Millot, Benoît Le Tallec, O. Hyrien
          </td>
          <td>2025-08-11</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43797c7834d20e5589904d0168347625fb0cad6e" target='_blank'>
              Unraveling cellular dynamic changes in tumor evolution induced by long-term low dose-rate radiation.
              </a>
            </td>
          <td>
            Wanshi Li, Weiwei Pei, Yiwei Wang, Jing Nie, Guang Hu, Yongsheng Zhang, Zhifei Cao, Zaozao Chen, Keyan Miao, Yuqi Chen, Likai Shi, Yingchu Dai, , Hailong Pei, C. Ye, Yasser F Ali, Wentao Hu, Wenying Yan, Guangming Zhou
          </td>
          <td>2025-09-05</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="ABSTRACT Diffuse large B‐cell lymphoma (DLBCL) is the most common hematological malignancy. More than half of DLBCL patients achieve long‐term remission after treatment, but a third relapse after conventional Rituximab (R)‐based chemotherapy regimens, such as CHOP (cyclophosphamide, doxorubicin, vincristine and prednisone). Cancer cells are exposed to chronic replication stress, which impedes the duplication of their genome. Functional DNA repair pathways are therefore important for the survival of cancer cells. This dependence can be exploited therapeutically to hamper repair of the intrinsic DNA damage occurring during replication or to exacerbate DNA damage induced by chemotherapy. Using CRISPR‐Cas9 screening, we identified CHEK1, WEE1, ATR and RAD51 DNA repair factors as essential genes in DLBCL cells. According to these results, we investigated the effect of small molecules targeting DNA replication stress and DNA repair mechanisms, alone or in combination with the R‐CHOP genotoxic agents, cyclophosphamide and doxorubicin. Applying a standard threshold of 2 SDs below the IC50 of the genotoxic agent alone, a total of 3 synthetic lethal combinations have been identified including cyclophosphamide with CHK1/2 inhibitor, cyclophosphamide and ATR inhibitor and doxorubicin with DNAPK inhibitor. Co‐treatment with these molecules led to cell death, DNA damage induction and cell cycle arrest in DLBCL cells more efficiently than genotoxic agents alone. These data have been validated using primary DLBCL cells from patients. Our results open new perspectives for therapeutic approaches exploiting the synthetic lethality of genotoxic agents with DNA repair inhibitors to improve the therapeutic outcome of patients with DLBCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61ea8280a7e78b4aa4b06c11d4f0a4eaa409f0ab" target='_blank'>
              Synthetic Lethal Combinations of DNA Repair Inhibitors and Genotoxic Agents to Target High‐Risk Diffuse Large B Cell Lymphoma
              </a>
            </td>
          <td>
            Sara Ovejero, Julie Devin, Laura Alibert, Camille Soun, Yea-Lih Lin, Laure Dutrieux, M. Abouladze, Elvira Garcia de Paco, Ouissem Karmous Gadacha, A. Constantinou, Guillaume Cartron, C. Herbaux, O. Elemento, Philippe Pasero, Sandrine Roulland, J. Moreaux, C. Bret
          </td>
          <td>2025-08-23</td>
          <td>Hematological Oncology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Dysfunction of several WD40 family proteins causes diverse endocrine diseases. Until recently, MEP50, a WD40 protein, was considered a Gene of Unknown Significance (GUS) because no inherited diseases had been linked to its function. However, genetic inactivation of MEP50 in mouse models or somatic mutations in humans drive oncogenesis in several endocrine-related cancers, including those of the prostate, breast, and uterus. In this study, we generate new knowledge through a multi-tier integration of evolutionary genomics, sequence and structural analyses, molecular mechanics calculations, and dynamic simulations of wild-type and cancer-mutated MEP50 proteins. Indeed, we find that a conserved splicing event across evolution generates an alternative MEP50 isoform, which is smaller than the canonical MEP50 and lacks the final β-sheet of the first WD40 domain, the entirety of the second WD40 domain, and the first β-sheet of the third WD40 domain. Notably, we find that this novel, Short MEP50 (s-MEP50) transcript encodes a 278 amino acid protein that retains aspects of the key regulatory and interaction sites, including those critical for androgen receptor and PRMT5 binding. Finally, we analyze the mutational landscape of MEP50 in endocrine-regulated cancers and use molecular mechanics calculations and dynamic simulations to reveal that cancer-associated mutations disrupt conserved bonding networks and induce widespread structural destabilization within the WD40 domain architecture. Thus, by combining evolutionary, structural, and biophysical approaches, we advance the understanding of MEP50 genomics, providing significant mechanistic and clinically relevant insights into endocrine-regulated tissues and their cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e4d5f2f3bba5deb45827f67cca6b9cab9291043" target='_blank'>
              Genomic Features of the methylosome protein MEP50 and its implication in Hormone Signaling and Cancer.
              </a>
            </td>
          <td>
            Gareth Pollin, Young-In Chi, Raul Urrutia, G. Lomberk
          </td>
          <td>2025-09-08</td>
          <td>Endocrine connections</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Aim This study aims to explore the dynamic tumor microenvironment of hepatocellular carcinoma (HCC) through deep transcriptomic analysis and to identify key regulatory genes, among which MRE11 was further validated for its immunomodulatory and prognostic significance. Methods We performed Summary-data-based Mendelian Randomization (SMR) analysis to identify genes causally associated with HCC and intersected these with DNA damage repair (DDR) genes, leading to the identification of MRE11. A comprehensive evaluation of MRE11 expression in HCC was conducted using transcriptomic data analysis. We collected data from 92 HCC patient samples and validated MRE11 expression differences in HCC tissues through qPCR, immunohistochemistry, and Western blotting. Publicly available single-cell RNA sequencing (scRNA-seq) data and spatial transcriptomics were utilized to explore MRE11’s dynamic mechanisms in the tumor microenvironment (TME) of both primary and post-immunotherapy cases. We also screened for differentially expressed genes and constructed a robust HCC prognosis model using 101 machine-learning algorithms. Results Our results demonstrated that high MRE11 expression is strongly associated with poor prognosis in HCC. In the primary TME, MRE11 regulates immune responses, facilitating immune evasion. Single-cell analysis revealed significant tumor heterogeneity in MRE11 high-expression groups, particularly in macrophages and malignant cells, where MRE11 regulates immune evasion and tumor progression via the cGAS-STING pathway and HGF-MET axis. Under immunotherapy, high MRE11 expression facilitated epithelial-mesenchymal transition (EMT) and extensive remodeling of the TME. Furthermore, MRE11 dynamically enhanced macrophage regulation, exhibiting immunosuppressive and tumor-invasive features. Finally, our prognostic model exhibited strong predictive accuracy across multiple datasets. Conclusion High MRE11 expression is crucial in regulating the immune microenvironment in HCC, fostering immune evasion and driving tumor progression. MRE11 emerges as a promising biomarker for HCC diagnosis and a potential target for personalized immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03931-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60428c69be5bed6fbcd8373c293bad782a86b9ab" target='_blank'>
              Elucidating the dynamic tumor microenvironment through deep transcriptomic analysis and therapeutic implication of MRE11 expression patterns in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Ruiqiu Chen, Chaohui Xiao, Zizheng Wang, Guineng Zeng, Shaoming Song, Gong Zhang, Lin Zhu, Penghui Yang, Rong Liu
          </td>
          <td>2025-08-21</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="SUMMARY The ring-shaped sliding clamp PCNA enables DNA polymerases to perform processive DNA synthesis during replication and repair. The loading of PCNA onto DNA is catalyzed by the ATPase clamp loader RFC. Using a single-molecule platform to visualize the dynamic interplay between PCNA and RFC on DNA, we unexpectedly discovered that RFC continues to associate with PCNA after loading, contrary to the conventional view. Functionally, this clamp-loader/clamp complex is required for processive DNA synthesis by polymerase δ (Polδ), as the PCNA-Polδ assembly is inherently unstable. This architectural role of RFC is dependent on the BRCT domain of Rfc1, and mutation of its DNA-binding residues causes sensitivity to DNA damage in vivo. We further showed the FEN1 flap endonuclease can also stabilize the PCNA-Polδ interaction and mediate robust synthesis. Overall, our work revealed that, beyond their canonical enzymatic functions, PCNA-binding proteins harbor non-catalytic functions essential for DNA replication and genome maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00baaea7b1484df608f6f240f34fa8c252f35c53" target='_blank'>
              A non-catalytic role for RFC in PCNA-mediated processive DNA synthesis
              </a>
            </td>
          <td>
            Gabriella N. L. Chua, Emily C. Beckwitt, Victoria Miller-Browne, O. Yurieva, Dan Zhang, Bryce Katch, John W Watters, Kaitlin Abrantes, Ryogo Funabiki, Xiaolan Zhao, Michael E. O’Donnell, Shixin Liu
          </td>
          <td>2025-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Fanconi anemia (FA) is a rare genetic disorder caused by defects in the repair of DNA interstrand crosslinks (ICLs)—highly toxic lesions that impede essential processes like DNA replication and transcription, leading to severe genome instability. Clinically, FA presents with a broad spectrum of symptoms, including progressive bone marrow failure, congenital abnormalities, and an elevated predisposition to various malignancies, particularly acute myeloid leukemia and squamous cell carcinomas. This review provides a comprehensive overview of both the endogenous and exogenous sources of ICLs and the DNA repair pathways responsible for their resolution, with a primary focus on the FA pathway. We also discuss the tumorigenic consequences of FA pathway deficiencies, highlighting the molecular mechanisms that contribute to the heightened cancer risk observed in FA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6689c7c75058ca5686e353d84e63237862e17744" target='_blank'>
              Comprehensive review on Fanconi anemia: insights into DNA interstrand cross-links, repair pathways, and associated tumors
              </a>
            </td>
          <td>
            Chenyan Fang, Zhoujun Zhu, Jun Cao, Jun Huang, Yipeng Xu
          </td>
          <td>2025-07-30</td>
          <td>Orphanet Journal of Rare Diseases</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed5d78cd0349a405a7f55287ad35ae506ecd433c" target='_blank'>
              HBV DNA integration gene CCDC91 is oncogenic and a potential therapeutic target for hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Mengge Li, Shusheng Wu, H. Luo, J. Niu, Yuan Fang, Wen-ju Chen, Lulu Cao, Yifu He
          </td>
          <td>2025-07-20</td>
          <td>Communications biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d635136f6062b087a4cb1e8318c15fddd1b00672" target='_blank'>
              Rewiring DNA repair with PARP-based chemical inducers of proximity.
              </a>
            </td>
          <td>
            Bryce da Camara, Eric M. Bilotta, Erin F Broderick, Ashwini Premashankar, Paige A. Barta, Trever R. Carter, Lauren M. Hargis, D. Durocher, Michael A. Erb
          </td>
          <td>2025-07-31</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Background Pancreatic cancer (PC) has been one of the most severe digestive system malignant tumor with poor prognosis that threatens human health. Chemotherapy is essential for patients with advanced PC, but unfortunately the curative effect is limited by chemoresistance. CircTNPO3, a recently discovered circular RNA (circRNA), has been indicated to be associated with multi-types of tumors. However, the function and mechanism of circTNPO3 in regulating PC malignant phenotype and chemoresistance still remain obscure. Methods qRT-PCR and ISH were used to analyze circTNPO3 expression in PC cells and pathological specimens. The subcellular localization of circTNPO3 was visualized through nucleoplasmic RNA separation and FISH assays. The effect of cicTNPO3 on PC cell proliferation, migration and invasion was assessed using EdU, colony formation, wound healing and Transwell assays respectively. Cell apoptosis was detected using ELISA, AO/EB, Hoechst 33342 and flow cytometry assays. The binding potential between circTNPO3, miR-188-5p and CDCA3 was verified by Ago2-RIP, RNA pull down and dual-luciferase reporter assays. The relationship between CDCA3, TRAF2 and NF-κB-p65 was analyzed using Pearson correlation, and the expression was detected using immunoblotting. The nucleus translocation of p65 was evaluated using IF assay. The effect of circTNPO3 on PC growth and metastasis was analyzed using subcutaneous and lung metastatic tumor models in vitro. Deoxyelephantopin, a small molecule extract from traditional Chinese medicine, was applied to evaluate the potential of circTNPO3 as therapeutic target. Results CircTNPO3 was aberrantly highly expressed in PC cells and tissues, and negatively associated with patient prognosis and gemcitabine chemotherapy sensitivity. Functionally, silencing circTNPO3 attenuated the malignant phenotypes and chemoresistance of PC in vitro and in vivo, conversely, facilitated by circTNPO3 overexpression. Mechanically, cytoplasmic circTNPO3 functioned as a sponge of miR-188-5p, and partially alleviated the effect of miR-188-5p on downstream molecules, which further upregulate the CDCA3 and TRAF2 expression and NF-κB activity, finally promoted PC progression and chemoresistance. More innovatively, the potential of circTNPO3 as a novel diagnostic biomarker and therapeutic target for PC was primarily validated in present study. Conclusion CircTNPO3 acted as an oncogenic and chemoresistant gene in PC, mechanically through targeting miR-188-5p and regulating CDCA3, TRAF2 and NF-κB signaling pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f1d26cb630aae0db6e3d479639c5ab170cfe5a5" target='_blank'>
              The functional roles of deoxyelephantopin potential target circTNPO3 in regulating pancreatic cancer malignant phenotype and gemcitabine chemoresistance via miR-188-5p/CDCA3/TRAF2-mediated remodeling of NF-κB signaling pathway
              </a>
            </td>
          <td>
            D. Ji, Jia Liu, Yanhui Zhang, Li Hou, Haonan Feng, Xue-sha Xing, Deming Guan, Tiangang Cui, Yi Xu, Gang Tan
          </td>
          <td>2025-07-31</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57090ca5bdab1ab7fa8186d6d31b9c968e9579a" target='_blank'>
              Fluctuating DNA methylation tracks cancer evolution at clinical scale.
              </a>
            </td>
          <td>
            C. Gabbutt, M. Duran-Ferrer, Heather E. Grant, D. Mallo, F. Nadeu, J. Househam, N. Villamor, , Simon Heath, E. Raineri, O. Krali, J. Nordlund, T. Zenz, I. Gut, E. Campo, A. López-Guillermo, J. Fitzgibbon, C. P. Barnes, D. Shibata, J. Martín-Subero, Trevor A. Graham
          </td>
          <td>2025-09-10</td>
          <td>Nature</td>
          <td>1</td>
          <td>95</td>
        </tr>

        <tr id="BACKGROUND
Colorectal cancer (CRC), a predominant cancer type globally, is one of the most common cancers worldwide. Immhibune-checkpoint initors (ICIs) have robust efficacy in the treatment of patients with metastatic DNA mismatch repair (dMMR) CRC, however, some of the patients among them demonstrate resistance to immunotherapy, the underlying molecular mechanisms remain elusive.


METHODS
Using single-cell sequencing data, we identified and annotated cell populations, systematically comparing cell proportions and gene expression changes across different groups. Gene set enrichment analysis (GSEA) was conducted to identify significantly enriched signaling pathways in T cell subpopulations. Furthermore, we assessed the impact of transcriptional dysregulation in TSTR and γδ_T cells on tumor immunotherapy responses and drug resistance. The potential role of T cell subset interactions in modulating drug resistance and sensitivity was investigated through comprehensive cell communication analysis. Lastly, clinical samples from 190 CRC patients were collected, with paired adjacent normal tissues. Immunohistochemistry (IHC) was performed using antibodies against FOS and KLRB1.


RESULTS
T cells were identified as the predominant cell population influencing immunotherapy outcomes. T cell subsets exhibited distinct functional characteristics, with notable differences in their distribution between resistant and sensitive tumor groups. Specifically, exhausted T cells (Tex), GZMK + T, TSTR, regulatory T cells (Treg), and γδ_T cells were associated with therapeutic resistance. For Tex and GZMK + T cells, resistance was correlated with activation of antigen processing and presentation pathways, whereas oxidative stress pathways were downregulated. In contrast, γδ_T cells in the sensitive group exhibited activation of protein folding pathways, which may contribute to anti-tumor immune responses. Transcriptional network dysregulation in TSTR and γδ_T cells was observed in the drug-resistant group. Cell-to-cell communication analysis showed stronger interactions among T cell subpopulations, with changes in key signaling pathways linked to treatment resistance. Additionally, downregulation of the CD69-KLRB1 signaling pathway was identified as a potential mechanism of drug resistance in CRC. Lastly, high expression of FOS was significantly associated with a worse prognosis, whereas high expression of KLRB1 predicted improved clinical outcomes, including prolonged overall survival and progression-free survival, and emerged as an independent prognostic factor for CRC patients.


CONCLUSION
This study highlights the pivotal role of T cell subsets in patients with metastatic dMMR CRC who resistance to anti-PD-1 therapy, revealing that transcriptional dysregulation and impaired cell communication networks are central mechanisms underlying drug resistance. Notably, KLRB1 has been identified as a promising biomarker for immunotherapy response in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3475e66480ee040f7a2e535d7ffc47aa52446ec" target='_blank'>
              Single-cell transcriptome analysis reveals regulatory programs associated with tumor resistance during immunotherapy in colorectal cancer.
              </a>
            </td>
          <td>
            Yan Chen, Tao Liu, Guangtao Min, Cong Wang, Dali Xi, Lei Jiang
          </td>
          <td>2025-09-10</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Polyploid giant cancer cells (PGCCs) represent a distinct subpopulation of tumor cells characterized by enlarged or multiple nuclei and aneuploidy. PGCCs are products of genomic instability, possessing cancer stem cell properties and exhibiting significant resistance to radiotherapy and chemotherapy. They can generate highly invasive daughter cells through asymmetric division, exhibiting epithelial-mesenchymal transition characteristics, and facilitating tumor recurrence and metastasis. In vivo, PGCCs with daughter cells in tumor tissue can migrate and infiltrate into the forefront stroma to form tumor budding, which are closely related to solid tumor recurrence, metastasis, and drug resistance. Studies have shown that inhibiting sphingolipid enzyme acid ceramidase or regulating autophagy can reduce the production of PGCCs with daughter cells. Under appropriate induction conditions, PGCCs with daughter cells can be induced to differentiate into benign tissues such as adipocytes, chondrocytes, and osteocytes, inhibiting their malignant proliferation and invasive destruction. This study reviewed the recent research developments regarding PGCCs, mainly explored the endogenous mechanisms of PGCCs formation and their malignant phenotype, as well as the process of tumor budding formation in vivo and potential therapeutic strategies targeting PGCCs. The main novelty of this study lies in exploring the translation of PGCCs basic research into the clinical pathological prognostic role of tumor budding, which can reveal the potential mechanism of PGCCs/tumor budding formation at the molecular level, providing theoretical basis for prognosis assessment, monitoring of recurrence and metastasis risks, as well as improving drug resistance and targeted therapy in cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49a3661c8ef6e9fb0fa2a5fd9bf5fffcb0f097f8" target='_blank'>
              Polyploid giant cancer cells and tumor budding: translation from basic research to clinical application
              </a>
            </td>
          <td>
            Peng Huang, Rong Wu, Zhimou Yang, Yuwei Li, Fei Fei, Yongjun Yu
          </td>
          <td>2025-07-16</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA N6-methyladenine (6mA) is an emerging epigenetic mark in the mammalian genome. ALKBH1 preferentially exhibits 6mA demethylase activity for single-stranded DNA (ssDNA) or bubbled/bulged DNA, but not for double-stranded DNA (dsDNA). Nevertheless, ALKBH1 significantly decreases the cellular 6mA level in genomic DNA, whose prevailing DNA conformation in living mammalian cells is dsDNA. Therefore, the demethylase activity of ALKBH1 toward 6mA in genomic DNA, especially dsDNA, remains largely debated. Here, we found that YTHDF3 increases the 6mA demethylase activity of ALKBH1 in genomic DNA with different conformations, including dsDNA. Compared with ALKBH1, YTHDF3 preferentially recognizes and binds to 6mA-modified DNA with different conformations. YTHDF3 recognizes 6mA in genomic DNA, and binds ALKBH1 to recruit it to sites near 6mA in genomic DNA, thereby facilitating the ALKBH1-mediated removal of 6mA in genomic dsDNA. In summary, YTHDF3 is a novel genomic DNA reader and guides ALKBH1 to remove 6mA in human genomic DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9589673dbf4b0dcfb9624f20bcd04da6bc6ed357" target='_blank'>
              YTHDF3 recognizes DNA N6-methyladenine and recruits ALKBH1 for 6mA removal from genomic DNA
              </a>
            </td>
          <td>
            Xin-Hui Chen, Zi-Lu Wang, Jincui Yang, Min Chen, Si-Yi Zhao, Kun-Xiong Guo, Xuelong Zheng, Zhengwei Zhao, Xiaoqiang Chen, Jing Li, Minmin Zhang, Ling Ran, Huifang Zhu, Xiaofeng Gu, Guang-Rong Yan
          </td>
          <td>2025-07-25</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT DNA replication is regulated by factors that promote or inhibit initiation. In Bacillus subtilis, YabA is a negative regulator of replication initiation, while the newly identified kinase CcrZ is a positive regulator. The consequences of under-initiation or over-initiation of replication on genome stability remain unclear. In this work, we measure the origin-to-terminus ratio as a proxy for replication initiation activity. We show that ΔccrZ and several ccrZ alleles under-initiate DNA replication, while ablation of yabA or overproduction of CcrZ leads to over-initiation. We find that cells under-initiating DNA replication have few incidents of replication fork stress as determined by the low frequency formation of RecA-GFP foci compared with wild type. In contrast, cells over-initiating replication show levels of RecA-GFP foci formation analogous to cells directly challenged with DNA-damaging agents. We show that cells under-initiating and over-initiating DNA replication are both sensitive to mitomycin C, demonstrating that changes in replication initiation frequency cause an increase in sensitivity to genotoxic stress. With these results, we propose that cells under-initiating DNA replication are sensitive to DNA damage due to asynchronous DNA replication, leading to inefficient homologous recombination. In cells over-initiating replication, we propose that an increase in the number of replication forks leads to replication fork stress, which is further exacerbated by chromosomal DNA damage. Together, our study shows that DNA replication initiation frequency must be tightly controlled because changes in initiation influence replication fork fate and the capacity of cells to efficiently repair damage to their genetic material. IMPORTANCE The regulation of DNA replication is fundamental to cell growth and cell cycle control. In eukaryotes, under-initiation or over-initiation leads to genome instability. In bacteria, it is unclear how changes in replication initiation frequency impact DNA replication status and genome integrity. We show that tight regulation of DNA replication initiation is critical for maintaining genome integrity. Cells over-initiating or under-initiating DNA replication are sensitive to DNA damage. Furthermore, cells over-initiating DNA replication experience replication fork stress at levels that phenocopy those observed in cells challenged with DNA damage from mitomycin C. Our results establish the critical importance of properly regulating DNA replication initiation frequency because an imbalance in initiation results in replication fork perturbations, deficiencies in DNA repair, and genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4a27c31f5c5a118965016651f3b5fe82634d9fb" target='_blank'>
              DNA replication initiation timing is important for maintaining genome integrity
              </a>
            </td>
          <td>
            Tristan T Reed, Abigail H Kendal, Katherine J. Wozniak, Lyle A. Simmons
          </td>
          <td>2025-07-21</td>
          <td>Journal of Bacteriology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id=" As a class of endogenous non‑coding RNAs, circular RNAs (circRNAs) are produced by an event known as back‑splicing. Although circRNAs were initially considered to be the product of abnormal splicing events, increasing evidence has shown their vital role in the development of various diseases, especially malignant tumors. Tumor metastasis is the leading cause of tumor deterioration and cancer‑associated mortality, and involves biological changes to malignant cells. A number of circRNAs can mediate tumor progression, especially tumor metastasis. Furthermore, due to their unique structural features, circRNAs are highly stable in body fluids, and have temporal and tissue specificity, making them potentially ideal non‑invasive biomarkers for sensitive monitoring of tumor changes. The present review focuses on circRNAs associated with tumor metastasis and discusses their functional mechanisms. Furthermore, the current review summarizes advances regarding circRNAs as biomarkers and detection strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bede549f2f5e269cd821afda2d7ab2c5424c7b04" target='_blank'>
              Circular RNAs: Fundamental mechanisms in tumor metastasis and detection strategies (Review).
              </a>
            </td>
          <td>
            Xiangjun Dong, Yinwei Che, Yuzhuo Jiao, Hao Dong, Qingchao Ren, Huashan Sun, Tao Zhao
          </td>
          <td>2025-08-28</td>
          <td>Molecular medicine reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The human genome is routinely subject to endogenous (e.g. oxidative stress) and exogenous (e.g. radiation, UV) factors that induce DNA lesions. These lesions may accumulate and impair cell function or cause cancer. Hence, DNA damage repair and preservation are prerequisites for normal cellular function and maintaining genome integrity. Current research suggests that long non-coding RNAs (lncRNA) play a role in DNA damage response (DDR). lncRNAs are RNA transcripts >200 nt that do not code for proteins. After transcription, they interact with DNA, protein, mRNA, or other non-coding RNA to influence gene expression and protein function. However, there is a lack of research concerning the role of lncRNA in DDR. This study aimed to investigate and quantify the expression of DNA-damage-responsive lncRNAs in a time-course experiment. To do so, human embryonic kidney 293T (HEK293T) cells were exposed to ionizing radiation (IR) at 2 Gy to induce 60-100 double-stranded breaks (DSB) per cell. Following this exposure, RNA was collected at 4 different time points (1, 4, 8, and 24 hrs). Finally, Real-time Quantitative PCR was performed to quantify the expression of select lncRNA genes. Our findings demonstrate an association between DDR and lncRNA gene expression. We found a temporal upregulation or downregulation of lncRNA genes, specifically ENSG00000250519, ENSG00000231595, ENSG00000254338, and ENSG00000223749. The influence of DNA lesions on the expression of these lncRNA increased over 24 hrs. This finding asserts that lncRNA is an epigenetic regulator that may influence signalling networks related to DNA damage. Hence, lncRNAs may serve as biomarkers for the diagnosis, treatment, and prognosis of cancer and other diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7c3f3f72f4b73fe0339171d4fca65b2b98d68e4" target='_blank'>
              Investigating DNA-damage responsive long non-coding RNA (lncRNA)
              </a>
            </td>
          <td>
            Lindsay Yu
          </td>
          <td>2025-08-28</td>
          <td>Inquiry@Queen's Undergraduate Research Conference Proceedings</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Hepatoblastoma (HB) is the most common pediatric liver malignancy. However, its cellular origin and molecular drivers remain poorly defined. Using single-nuclear RNA sequencing (snRNA-seq), we identified a proliferative, hepatocyte-derived tumor cell population (cycling HepT) enriched for Enhancer of Zeste Homolog 2 (EZH2) expression, particularly in the aggressive embryonal subtype. Integrative genomic and transcriptomic profiling confirmed EZH2 overexpression. Disruption of the PRC2 complex was evident through mislocalization and reduced expression of SUZ12, a core component. EZH2 overexpression correlated with upregulation of mitotic regulators such as AURKB and Ki67 in human HB gene expression analysis as compared to background liver. Targeted sequencing identified variants of uncertain significance in EZH2 and SUZ12 in 11 of 11 patient tumors. Pharmacologic inhibition of EZH2 with EPZ-6438 reduced proliferation and sensitized HB cells to cisplatin through gene regulation, potentially modulating platinum accumulation both in vitro and in vivo. In summary, EZH2 promotes HB progression through epigenetic silencing and noncanonical signaling pathways. These findings support EZH2’s contribution to HB pathogenesis, therefore identifying it as a novel therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b140db08052390107c208976008407b428e0ef2" target='_blank'>
              Oncogenic Role of Aberrant EZH2 in Hepatoblastoma
              </a>
            </td>
          <td>
            Kathryn Glaser, Lara Berklite, Brian T Hickner, Priyanka Rao, R. Patel, Andrew A. Badachhape, Somak Roy, Emily J Schepers, Nikolai Timchenko, James I Geller, S. Ranganathan, Greg Tiao, Bruce J Aronow, Takanori Takebe, Sarah E. Woodfield, Sanjeev A. Vasudevan, E. DePasquale, Alex J Bondoc
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Approximately 30% of human cancers carry various RAS mutations, including KRAS, NRAS, and HRAS. Among these mutations, KRAS is the most prevalent isoform detected in lung cancer. While several small molecular inhibitors targeting specifically KRASG12C have been developed and tested clinically, alternative approaches are still necessary due to expected drug resistance. In this study, we present evidence that the loss of DDX3X significantly delays tumor progression in various KRAS-driven lung cancer models. Inhibition of DDX3X disrupts cysteine and glutathione metabolism, thereby inducing ferroptosis in lung cancer cells. This effect is primarily mediated by the downregulation of Cystathionine-β-synthase (CBS), the rate-limiting enzyme in cysteine generation. Mechanistically, DDX3X directly binds to the transcription factor JUND, which mediates the transcriptional regulation of METTL16, a key N6-methyladenosine methyltransferase, and subsequently regulates m6A modification and translation of CBS transcripts. This cascade induces hypermethylation and high expression of CBS, consequently triggering cysteine production and maintaining antioxidative homeostasis, which is essential for the survival of KRAS-driven lung cancer cells. Finally, we demonstrate that a newly developed DDX3X PROTAC degrader J10 efficiently delays lung cancer progression with multiple advantages compared to DDX3X small molecular inhibitor RK-33 and limited side effects. These findings unveil the potential of DDX3X as a valuable target for adjuvant therapies in managing KRAS-driven lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bf00a29097f2bb628af25496cb6f0ce5c7d29b5" target='_blank'>
              Targeting DDX3X suppresses progression of KRAS-driven lung cancer by disrupting antioxidative homeostasis and inducing ferroptosis
              </a>
            </td>
          <td>
            Meijuan Dian, Liang Yun, Qingyu Meng, Songwen Lin, Ming Ji, Ying Zhou, Wenqian Liu, Zhuoying Yang, Yayan Zhao, Gaoyuan Li, Jianjun Jiang, WeiChao Hao, Zhijie Chen, Zehao Zhou, Ruihao Zhang, Tianyuan Liu, Yujing He, Tianbao Yan, Haofei Wang, Shane J.F. Cronin, Josef M. Penninger, K. Cai, Shuan Rao
          </td>
          <td>2025-08-30</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f33d799c8af989b9135dc6d795da0cb937e5a6fb" target='_blank'>
              Targeting MCM10 disrupts cancer stemness and counteracts sorafenib resistance in hepatocellular carcinoma.
              </a>
            </td>
          <td>
            Ziyun Zhang, Lu Liang, Yi Li, Ye Fan, Yao Liu, Zhifei He, Xiumin Wu, Li Cong, Yiqun Jiang, Tao Wan
          </td>
          <td>2025-08-01</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Recombinases are essential enzymes in synthetic biology and biomedical research, enabling site-specific DNA modifications for applications such as gene therapy, the generation of transgenic models, and the construction of genetic circuits. Recombination efficiency depends on several factors, including intracellular recombinase concentration and the growth phase of the host cells. Although recombination is typically studied during exponential growth, the effects of stationary-phase dynamics on recombinase activity remain poorly understood. In this study, we examine how bacterial growth phase influences recombinase-mediated DNA modifications, using the serine recombinase Bxb1 as a model. We engineered a genetic system in Escherichia coli to quantify intracellular Bxb1 levels and to measure recombination efficiency across different growth phases. Our results reveal a quasi-linear relationship between recombinase concentration and recombination efficiency during exponential growth, up to a saturation point. Notably, recombination continues in the stationary phase following recombinase induction in exponential phase, despite the decline in plasmid gene expression. Cells that undergo recombination during the stationary phase show significantly higher recombination efficiencies upon re-entering exponential growth than those maintained in exponential phase throughout. These findings highlight the importance of induction timing in optimizing recombinase-based genetic modifications. Specifically, inducing recombinase expression just before the onset of stationary phase can enhance recombination efficiency while minimizing the need for high expression levels. Moreover, the observed quasi-linear relationship during exponential growth provides a framework for tuning gene expression with precision. Overall, this work offers new insights into leveraging bacterial growth dynamics to improve the design and control of synthetic genetic systems. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-17024-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6df10f0c8f4691b2c66e4226a8d26ecc0ab6dc10" target='_blank'>
              Characterization of recombinase activity across cellular growth phases
              </a>
            </td>
          <td>
            M. Gonzalez-Colell, J. Macía
          </td>
          <td>2025-08-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) is a common characteristic of human cancers, which occurs most frequently in ovarian and breast cancers. The unique genetic vulnerabilities, coupled with the synthetic lethality effect of Poly-(ADP-ribose) polymerase (PARP) inhibitors (PARPi), provide a promising opportunity for targeting HRD cancers. However, only a few HRD patients benefit from PARPi monotherapy, and it is therefore imperative to explore effective combination therapies for HRD tumors. Growing evidence has underscored the distinct tumor microenvironment (TME) landscape of HRD cancers. Immune activation and immune suppression co-exist in a dynamic balance during the development of HRD cancers. At the late stage, however, negative immune regulation predominates, resulting in the formation of an immunosuppressive microenvironment. This intricate network reprograms cancer biology in multiple aspects and serves as a potential target for cancer treatment. In this review, we briefly outline the current HRD tests and genetic characteristics of HRD cancers, focusing on breast, ovarian, pancreatic, and prostate cancers. We then summarize the interactions and crosstalk between immune cells and cancer cells, as well as various signaling pathways within the TME of HRD cancers. Additionally, we highlight recent advances in combining PARP inhibitors with immunotherapies in preclinical models and clinical trials of HRD cancers. This review provides valuable insights and perspectives into the distinct landscape of TME in HRD cancers, and offers a rationale for expanding the application of this combined therapeutic approach to a broader range of HRD cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/befa4ba7c7bbd20ccb4cc8148dd06bbf9c57c776" target='_blank'>
              The distinct landscape of tumor immune microenvironment in homologous recombination deficient cancers
              </a>
            </td>
          <td>
            Qiuyang Xu, Yuanjia Wen, Taiyuan Huang, Huayi Li, Xingzhe Liu, Shen-nan Shi, Wenjian Gong, Gordon B. Mills, Ding Ma, Qing-cui Gao, Yong Fang
          </td>
          <td>2025-08-20</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Lymphoid neoplasms (LNs) are heterogeneous malignancies arising from lymphoid cells, displaying diverse clinical and molecular features. Although LNs are collectively frequent, individual subtypes are rare, posing challenges for genetic association studies. Indeed, genome-wide association studies (GWAS) explained only a fraction of the heritability. Shared genetic susceptibility and overlapping risk factors suggest a partially common etiology across subtypes. We employed a multi-trait GWAS strategy to improve discovery power by leveraging pleiotropy among LN subtypes. We defined LN phenoclusters based on cell of origin, somatic mutation profiles, and approved therapeutic agents. Using data from three large cohorts—the UK Biobank, Million Veteran Program, and FinnGen—we analyzed 31,937 LN cases and 1.2 million controls across 8 individual subtypes and 7 phenoclusters. We replicated the novel associations in two independent cohorts (All of Us and the Prostate, Lung, Colorectal, and Ovarian Cancer Screening Trial) with 2892 LN cases and 165,791 controls. We identified 76 genome-wide significant loci for individual subtypes or subtype clusters, including 20 novel associations. We identified the subtypes contributing to each locus, putative candidate causal variants, and genes underlying the associations, and found enrichment of specific cell types, biological processes, and drugs associated with LN risk genes. Overall, this study identified new LN genetic risk loci and candidate genes, providing insights that may inform novel therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d8ac41083338a114514b280145d44117043358f" target='_blank'>
              Clustering of lymphoid neoplasms by cell of origin, somatic mutation and drug usage profiles: a multi-trait genome-wide association study
              </a>
            </td>
          <td>
            Murat Güler, F. Canzian
          </td>
          <td>2025-08-29</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) ranks as the fifth most prevalent malignancy worldwide. Disruptions in copper homeostasis adversely affect liver function. Cuproptosis, a recently defined form of regulated cell death triggered by intracellular copper accumulation, disrupts the tricarboxylic acid cycle and mitochondrial respiration. However, the specific roles and mechanisms of cuproptosis-related genes (CRGs) in HCC pathogenesis remain incompletely understood. Materials and methods We systematically evaluated the expression of 10 CRGs in HCC tissues versus adjacent normal tissues. Bioinformatics analyses included Gene Ontology (GO), Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment, and Gene Set Enrichment Analysis (GSEA) were performed. Immune infiltration levels within the tumor microenvironment were assessed. The prognostic significance of CDKN2A was evaluated using Kaplan-Meier (KM) survival analysis and univariate/multivariate Cox proportional hazards regression. CDKN2A protein expression was validated using immunohistochemistry (IHC). Results CDKN2A was significantly overexpressed in HCC compared to normal tissues. Bioinformatics analyses implicated CDKN2A in DNA replication, organelle fission, and cell cycle checkpoint signaling. Immune-related analysis revealed that high CDKN2A expression correlated positively with dendritic cell (DC) and Th2 cell infiltration, but negatively with CD8 + T cell and natural killer (NK) cell infiltration. KM analysis demonstrated that high CDKN2A expression predicted significantly shorter overall survival in HCC patients. Univariate and multivariate Cox regression identified CDKN2A as an independent prognostic risk factor. Conclusions This study demonstrates that CDKN2A is significantly overexpressed in HCC and plays a role in immune microenvironment modulation. CDKN2A serves as a promising independent prognostic biomarker for HCC, associated with poorer patient survival. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03496-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e16599e64b7443664631899640a1dd84fe24a22" target='_blank'>
              Systematic analysis identifies CDKN2A as a prognostic biomarker for hepatocellular carcinoma
              </a>
            </td>
          <td>
            Shilin He, Yanling Zhang, Junjie Xu, Xiao Liang
          </td>
          <td>2025-08-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="

Cancer remains one of the most complex diseases globally, driven by genetic and epigenetic
alterations that disrupt normal cellular processes. This complexity is further amplified by
mechanisms such as uncontrolled cell proliferation, inhibition of apoptosis, angiogenesis, immune
evasion, and metabolic reprogramming, all of which contribute to tumor growth and metastasis.
While advancements in diagnostic technologies, including next-generation sequencing and liquid
biopsies, have facilitated early detection and the development of personalized therapeutic strategies,
traditional treatments such as chemotherapy, radiotherapy, and immunotherapy continue to
face significant limitations, including drug resistance, adverse side effects, and tumor heterogeneity.
Recent studies have emphasized the potential of bacteriophages in cancer treatment. As viruses
that specifically target bacteria, phages offer unique advantages due to their genetic modifiability,
roles in immunotherapy, and capabilities in targeted drug delivery. They can reshape the tumor microenvironment,
enhance immune responses, and deliver therapeutic agents directly to cancer
cells. Moreover, phages hold promise in addressing antibiotic resistance in cancer patients and improving
the efficacy of traditional therapies through combination strategies. However, challenges
remain regarding the optimization of phage stability, bioavailability, and tumor-specific targeting.
Research and clinical trials on the therapeutic application of phages are critical to unlocking their
full potential as innovative approaches to overcome the limitations of current cancer treatments.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f176f476b0e17d22a339146ab7610aed3ab80193" target='_blank'>
              Bacteriophages in Cancer Treatment
              </a>
            </td>
          <td>
            Mustafa Ustundag, Berrin Ustundag
          </td>
          <td>2025-08-22</td>
          <td>Current Cancer Therapy Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal malignancies, with limited treatment options and poor prognosis. Recent advances in cancer genomic analysis enable the identification of actionable gene alterations, opening new opportunities for personalized therapy. Among these, homologous recombination DNA repair (HRR) gene alterations are associated with distinct biological behavior, favorable prognosis, and increased sensitivity to platinum-based chemotherapy. However, the prognostic impact of coexisting mutations in key driver genes—KRAS, TP53, CDKN2A, and SMAD4—within HRR-altered PDAC remains poorly understood. Methods We retrospectively analyzed PDAC patients who underwent genomic profiling testing with FoundationOne® CDx between June 2019 and December 2021 through the Center for Cancer Genomics and Advanced Therapeutics (C-CAT) database. We compared the prevalence and prognostic significance of key gene alterations between HRR-altered and HRR–wild-type (WT) tumors. Results Of 2,381 PDAC patients, 274 (11.5%) harbored HRR alterations. These patients showed significantly longer overall survival (OS) than those with HRR-WT tumors (HR = 0.66, p = 0.002). The frequencies of KRAS, TP53, and CDKN2A mutations were less frequent in HRR-altered tumors. TP53 mutation was independently associated with poorer OS across both HRR subgroups, while CDKN2A alteration was a poor prognostic factor in HRR-WT tumors. Interestingly, SMAD4 alteration was linked to improved survival in the HRR-altered group. Conclusion HRR-altered PDAC has a distinct genomic profile and is associated with a favorable prognosis. Our findings demonstrate that coexisting alterations are significant prognostic factors in both HRR-altered and HRR–wild-type tumors. These results highlight the clinical relevance of incorporating comprehensive genomic profiling into routine care to stratify patient prognosis better and inform individualized treatment strategies in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ada42c90fd2263fd4d9bb1466e612ccff4b0a39f" target='_blank'>
              Prognostic impact of gene alterations via homologous recombination DNA repair gene alteration status in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            Yusuke Kawanaka, Chiaki Inagaki, Masaki Okura, S. Mitani, T. Takahama, K. Yonesaka, Y. Chiba, Kazuhiko Nakagawa, Hisato Kawakami, H. Hayashi
          </td>
          <td>2025-08-25</td>
          <td>Frontiers in Medicine</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="In Chronic Lymphocytic Leukemia (CLL), t(14;19)(q32;q13), leading to the overexpression of BCL3, is found in ∼1% of cases and is associated with an aggressive disease. In this study, leveraging a large CLL patient cohort collected thanks to an international collaboration, we investigate for the first time the circular transcriptome (circRNAome) associated with the rare t(14;19), in comparison with CLL without t(14;19) and B cells of age-matched healthy donors. We described the circRNAs commonly dysregulated in CLL, including circCSNK1G3 and circEXOC6B(3–5), which were depleted, and circZNF609 and circLPAR3, which were overexpressed in malignant cells. Of importance, we disclosed the circRNA signature of CLL with t(14;19), formed by circRNAs with expression significantly altered specifically in link with this lesion, ectopically expressed like circCDK14(3–4), circCORO1C, circCLEC2D, and circEMB, or downregulated like circCEP70(3–6). Several of these molecules were previously shown to be dysregulated or play a role in cancer, whereas most of the signature circRNAs deserve further investigation. CLL patients with high circCORO1C and circCLEC2D expression had significantly worse clinical outcomes, with shorter time to first treatment and overall survival. This study disclosed new molecular features of the aggressive CLL subtype with t(14;19). Supplementary Information The online version contains supplementary material available at 10.1186/s13045-025-01725-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/042a6909cf8dd587b1601b4163aa8ab8452f9369" target='_blank'>
              Circular RNA signature of aggressive CLL with t(14;19)(q32;q13). An ERIC study
              </a>
            </td>
          <td>
            Eleonora Roncaglia, Enrico Gaffo, G. Calabretto, M. Fürstenau, K. A. Rogers, P. Baliakas, Chenghua Cui, Cecelia R Miller, C. Haferlach, K. Plevová, D. Oscier, Z. Davis, F. Nguyen-Khac, G. Rigolin, A. Athanasiadou, Fanny Baran-Marszak, Alberto Valiente, M. Terol, P. Abrisqueta, B. Espinet, A. Puiggros, A. Martines, L. Bonaldi, Francesca R Mauro, L. Scarfò, T. Chatzikonstantinou, E. Tausch, K. Kreuzer, A. Kater, F. Bosch, M. Doubek, P. Panagiotidis, O. Kalashnikova, F. Frezzato, V. Ruocco, Silvia Orsi, Kimia Salek, Roberto Merlo, Alberto Caregari, Ilias Glogovitis, A. Cellini, F. Angotzi, Andrea Serafin, Shuhua Yi, B. Eichhorst, J. Woyach, A. Cuneo, P. Ghia, K. Stamatopoulos, L. Trentin, A. Visentin, Stefania Bortoluzzi
          </td>
          <td>2025-07-22</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is characterized by a pronounced inter- and intra-tumoral heterogeneity, which fuels chemoresistance and contributes to high mortality rates. Previously, we have developed a branched organoid system embedded in collagen matrices that robustly recapitulates the phenotypic heterogeneity seen in both murine and human PDAC1,2. These organoids display complex, self-organized branching morphogenesis and give rise to distinct, spatially ordered tumor cell populations that reflect their underlying molecular profiles and differentiation states. Importantly, we show that the observed heterogeneity is not random but governed by defined transcriptional programs, particularly epithelial-to-mesenchymal plasticity, that drive the emergence of discrete tumor-cell states. Using integrated phenotypic and transcriptomic profiling, we map this diversity to specific biological functions in vivo, demonstrating that each organoid phenotype corresponds to a tumor-cell state with unique metastatic potential, and therapeutic vulnerabilities. Moreover, we identify dynamic, treatment-induced phenotype reprogramming events that are targetable, paving the way for rational design of state-specific therapeutic interventions. Building on our findings, we are now focusing on elucidating intra-organoid heterogeneitysuch as tip–trunk hierarchies at the single-cell level, to uncover the mechanisms of self-organization and to determine how distinct organoid phenotypes and subpopulations contribute to liver metastasis and treatment resistance. In summary, we have established a scalable and mechanistically informative organoid platform that enables in vitro modeling of PDAC heterogeneity. This system provides a framework for dissecting the tumor cell–intrinsic drivers of phenotypic plasticity and for developing phenotype-guided treatment strategies aimed at overcoming resistance and improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47eb75a781046a1e97c826bc3797a33d46a03686" target='_blank'>
              LEVERAGING ORGANOID TECHNOLOGIES TO CHARACTERIZE DYNAMIC TUMOR CELL STATES DRIVING TREATMENT RESISTANCE IN PANCREATIC CANCER
              </a>
            </td>
          <td>

          </td>
          <td>2025-08-21</td>
          <td>European Journal of Histochemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract BACKGROUND A major barrier in our understanding of glioblastoma (GBM) is the difficulty in tracking tumour development in real-time. We adopted a novel approach that combines the principles of classic cellular barcoding with CRISPR/Cas-9 technology and single-cell RNA sequencing known as continuous lineage tracing to apply a phylogenetic approach to studying tumour development. METHODS Patient derived glioma initiating cell lines were engineered with expressed DNA barcodes with CRISPR/Cas-9 targets and subsequently implanted to create an intracranial xenograft model. Tumors were sent for single cell RNA sequencing; clonal relationships were surmised through identification of expressed barcodes, and cells were characterized by their transcriptional profiles. Phylogenetic lineage trees were created utilizing lineage reconstructive algorithms to define cell fitness and expansion. RESULTS Our work has revealed a significant amount of intra-clonal cell state heterogeneity, suggesting that tumour cells engage in phenotype switching prior to therapeutic intervention. We defined a consistent transcriptional pattern for tumour engraftment and in vivo clonal advantage. Phylogenetic lineage trees allowed us to define gene signatures of both cell fitness and expansion, which correlate strongly with published neural-mesenchymal and developmental-injury response phenotypes. GBMs exist along a transcriptional gradient between undifferentiated but “high-fit” cells and terminally differentiated, “low-fit” cells: cells with highly fitness appear to represent cells undergoing cell state transition. CONCLUSION We successfully employed a novel lineage tracing technique in GBM, creating a powerful tool for real-time tracing of tumour growth through the analysis of a unique set of highly detailed transcriptional data with associated clonal and phylogenetic relationships.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51825ee39f0efc909dc234d3cc75f48f8c3aa8f6" target='_blank'>
              56 SINGLE CELL CRISPR/CAS-9 LINEAGE TRACING REVEALS EVOLUTIONARY FITNESS AND EXPANSION AXES IN GLIOBLASTOMA
              </a>
            </td>
          <td>
            A. Ajisebutu, F. Fan, Chloe Gui, Y. Ellenbogen, A. Landry, Jeff Liu, Q. Wei, G. Zadeh
          </td>
          <td>2025-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="RNA methylation modifications are widespread in eukaryotes and prokaryotes, with N6-methyladenosine (m6A) methylation being the most prevalent internal modification in eukaryotic mRNA and having become a prominent focus of tumor research in recent years. Up to now, substantial evidence has suggested that the dysregulated RNA demethylase ALKBH5 can interact with m6A reader proteins to modulate a wide range of mRNA biological progress, including mRNA shearing, export, metabolism, and stability, ultimately influencing tumorigenesis and development. To deeply understand the regulatory roles of ALKBH5 and reader proteins in tumor progression, this review aims to summarize the structures of ALKBH5 and reader proteins, as well as their cooperative regulatory mechanisms that affect the occurrence and development of tumors originating from different systems. Furthermore, the potential applications of targeting ALKBH5 and reader proteins in antitumor drug development are summarized, hoping to provide a strong basis for advancing antineoplastic research in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ba5a13a08e9d17690fa603d6d63f1ba2a51ac6e" target='_blank'>
              Interactions between ALKBH5 and reader proteins in tumors: functions and molecular mechanisms
              </a>
            </td>
          <td>
            Jiahui Ou, Bingchen Liu, Yi Yu, Yingchun He, Yuyu Gao, Lingli Chen, Xia Chen, Huai Tao
          </td>
          <td>2025-09-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Both DNA methylation and homologous recombination (HR) are extensively studied. In bacteria, Dam methylation is the most studied DNA modification, while RecA-mediated HR is a primary mechanism to repair DNA damages including double-stranded breaks, single-stranded gaps, and stalled replication forks. While HR regulation by proteins is extensively studied, whether methylation of DNA itself directly affects the functions of RecA and HR remains unclear. Mainly by single-molecule experiments, we report that Dam methylation of single-stranded DNA (ssDNA) promotes RecA assembly, partially by reducing the effective charge of ssDNA under counterion screening. Furthermore, Dam methylation of double-stranded DNA promotes homologous pairing, joint molecule growth, and strand exchange. In cellular experiments, dam deletion impairs HR, whereas hypermethylation of the adenines in the genome enhances HR in P1 transduction assays and DNA-damage sensitivity tests without significantly upregulated HR-related genes. In addition, the preference of RecA for Dam-methylated DNA in RecA assembly and homologous pairing is conserved across divergent species covering a gram-negative bacterium Klebsiella pneumoniae, a gram-positive bacterium Bacillus subtilis, and a flowering plant Arabidopsis thaliana. Dam methylation of ssDNA increases the ATPase activity of molecular motors such as RecQ helicase that containing RecA-like domains. These findings reveal effects of DNA methylation and mechanisms regulating RecA-mediated HR and molecular motors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a366efaa4f71dbaa4fd5242b1fedea2c3fbc0149" target='_blank'>
              N6-methyladenine modification of DNA enhances RecA-mediated homologous recombination.
              </a>
            </td>
          <td>
            Xiaocong Zhao, Bin Wu, Ya-Jun Yang, Ying Li, Qi-Yuan Qiu, Liu Wang, Ya-Peng Xu, Han Gong, Lun Song, Xuejie Wang, Jing-Yao Shi, Xuefeng Chen, Shao-Ran Zhang, Qi Zong, Liang Dai, Shi-Shen Du, Yan Zhang, Wenqiang Wu, Xinghua Zhang
          </td>
          <td>2025-08-20</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="In most solid tumors, hypoxia constitutes a defining microenvironmental feature that reprograms malignant cells into a highly metastatic state by driving cellular plasticity and exacerbating chromosomal instability (CIN). However, the mechanisms by which cancer cells concurrently co-opt these elements of hypoxic adaptation to promote metastasis remains poorly understood. Here, we report that hypoxia promotes metastasis by suppressing the JmjC-containing histone lysine demethylase Kdm8. CRISPR/Cas9-mediated targeting of Kdm8 in a Kras;Trp53-driven mouse model of pancreatic ductal adenocarcinoma (PDA) robustly rewires the malignant cell transcriptomic programs, leading to a profound loss of the epithelial morphology and widespread metastatic disease. In PDA patients, a high KDM8-induced gene signature is associated with reduced metastatic burden and better survival in advanced disease. Notably, Kdm8 suppression in normoxia recapitulates key aspects of the global epigenetic and transcriptomic rewiring, mitotic spindle defects, and CIN induced by hypoxia. Moreover, disruption of Kdm8’s demethylase activity phenocopies Kdm8 loss, whereas expression of hypermorphic Kdm8 variants resistant to hypoxic suppression markedly reduces metastasis beyond the levels achieved by the wildtype protein. Through the suppression of Kdm8 demethylase function, hypoxia unleashes a potent metastatic program by simultaneously advancing cellular plasticity and CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7ead3df1c0ef44aa86f0839ae810b8a1fead329" target='_blank'>
              Hypoxia-induced metastatic heterogeneity in pancreatic cancer
              </a>
            </td>
          <td>
            Pradeep Moon Gunasekaran, Qianqian Wang, Yoke-Chen Chang, Polina Guseva, Rajika Chauhan, Alexander Kley, Gene Lee, Siddharth Ghosh Roy, Yousef Masoudpoor, Arthur Roberts, Kelly Watkins Walton, Lucyann Franciosa, Shafiq Bhat, Emmanuel Zachariah, Kishan Patel, Zhongren Zhou, Wenjin Chen, J. Ni, S. Gu, Cristina Montagna, Shin-Heng Chiou
          </td>
          <td>2025-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="PAX3-FOXO1, an oncogenic transcription factor, drives a particularly aggressive subtype of rhabdomyosarcoma (RMS) by enforcing gene expression programs that support malignant cell states. Here, we showed that PAX3-FOXO1+ RMS cells exhibit altered pyrimidine metabolism and increased dependence on enzymes involved in de novo pyrimidine synthesis, including dihydrofolate reductase (DHFR). Consequently, PAX3-FOXO1+ cells displayed increased sensitivity to inhibition of DHFR by the chemotherapeutic drug methotrexate, and this dependence was rescued by provision of pyrimidine nucleotides. Methotrexate treatment mimicked the metabolic and transcriptional impact of PAX3-FOXO1 silencing, reducing expression of genes related to PAX3-FOXO1-driven malignant cell states. Accordingly, methotrexate treatment slowed the growth of multiple PAX3-FOXO1+ tumor xenograft models but not the fusion-negative counterparts. Taken together, these data demonstrate that PAX3-FOXO1 induces cell states characterized by altered pyrimidine dependence and nominate methotrexate as an addition to the current therapeutic arsenal for treatment of these malignant pediatric tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ba839c9a2b458336e1dd8e4d0285ec2394738d6" target='_blank'>
              PAX3-FOXO1 Drives Targetable Cell State-Dependent Metabolic Vulnerabilities in Rhabdomyosarcoma.
              </a>
            </td>
          <td>
            Katrina I. Paras, Julia S. Brunner, Angela M Montero, Jacob A Boyer, Benjamin T. Jackson, Sangita Chakraborty, Abigail Xie, Kristina Guillan, Armaan Siddiquee, Lourdes Pajuelo Torres, Joshua D. Rabinowitz, A. Kung, Daoqi You, F. D. Dela Cruz, Lydia W. S. Finley
          </td>
          <td>2025-09-05</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fa3ab0bece749f3dd05030992350c29b3fbf920" target='_blank'>
              Targeting transcription in neuroblastoma: focus on the Core regulatory circuit.
              </a>
            </td>
          <td>
            O. Kuchur, S. S. Pogodaeva, Victoria I Zhdankina, Anna V Scherbakova, Alexander A. Shtil, N. V. Antipova
          </td>
          <td>2025-08-11</td>
          <td>Expert opinion on therapeutic targets</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="CRISPR−Cas12a proteins are RNA-guided endonucleases classified as type V−A effectors that function similarly to Cas9, but possess distinct biochemical features. Previous studies have reported that compared to Cas9, Cas12a exhibits reduced off-target activity, yet the mechanistic origin of this high specificity remains unclear. In this study, we used single-molecule fluorescence assays to investigate the kinetic basis for the reduced off-target effects of Cas12a. Introducing double mismatches at various positions within the target DNA enabled systematic analysis of the off-target effects on individual reaction steps in the Cas12a-mediated DNA cleavage reaction: seeding, stable R–loop formation, and DNA cleavage. Our results show that mismatches within a 17 bp PAM–proximal seed region significantly impair stable R–loop formation and subsequent cleavage, whereas mismatches in the PAM–distal region exert minimal or negligible effects. These results suggest that the low off-target tolerance of Cas12a and the resulting high on-target selectivity arise from the high sensitivity of the R–loop formation rate to DNA mismatches in the PAM–proximal region, which strongly correlates with cleavage efficiency. This work establishes R–loop formation as a conformational checkpoint for specific target cleavage, and provides a mechanistic framework to improve the fidelity of genome editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a7a4e4e5b839479c01ff628ac76a74b1991e57e" target='_blank'>
              Kinetic basis for Cas12a off-target discrimination
              </a>
            </td>
          <td>
            Yuyoung Kim, You Hee Choi, Minji Kim, Y. Jang, Sanghwa Lee
          </td>
          <td>2025-07-28</td>
          <td>BMB Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer is the origin of a novel tissue that attracts resources, spreads beyond boundaries, avoids normal controls, and escapes immunity. How does a novel tissue arise? The puzzle is that two seemingly different processes appear to be the primary driving force. On the one hand, overwhelming evidence links (epi)genetic driver mutations to the origin and progression of tumors. Common oncogenic mutations such as KRAS accelerate cell division, and common knockouts of tumor suppressors such as TP53 abrogate cell death or checks on cell division. On the other hand, cancerous tissues create complex traits that require intricate changes in cells and multiple interactions between different cell types. Such novelty often arises by hijacking the developmental plasticity that normally creates the diverse cells and tissues of our bodies from a single original zygotic cell. How can we reconcile the simple genetic changes in carcinogenesis with the complex developmental plasticity that creates novel tissues? This perspective advocates a new model. (Epi)genetic mutations release developmental plasticity. That developmental plasticity creates novel cellular interactions and complex tissues. Initially, novel traits created by developmental plasticity may not be stably heritable, thus subsequent (epi)genetic changes must stabilize the phenotypic novelty. Recent studies show how classic oncogenic and tumor suppressor driver mutations, such as KRAS and TP53, may primarily act in early carcinogenesis as broad releasers of developmental plasticity rather than as stimulators of cell division or knockout of limitations on cellular clonal expansion. In the new model, genetics releases, plasticity creates, and genetics stabilizes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0abe05e544bd1630685c7bbe6b925595c19894a8" target='_blank'>
              How cancer arises: Genetics releases, plasticity creates, genetics stabilizes
              </a>
            </td>
          <td>
            Steven A. Frank
          </td>
          <td>2025-07-31</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f9d2ade8ce124c86164da260d86f13c994b3a19" target='_blank'>
              KEAP1 mutations activate the NRF2 pathway to drive cell growth and migration, and attenuate drug response in thyroid cancer.
              </a>
            </td>
          <td>
            Nicholas E Bambach, J. Ricarte-Filho, E. R. Reichenberger, Christian Hernandez-Padilla, K. Hinkle, A. Isaza, Andrew J. Bauer, A. Franco
          </td>
          <td>2025-08-13</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c38bcffc5c385d9f9befa33ae4e8c24ff19f54" target='_blank'>
              Synthesis of Oligonucleotides Containing a Dna Distorting Interstrand Crosslink Produced by Mitomycins.
              </a>
            </td>
          <td>
            Sarah Marks, Gabriel Martinez, Michael Adamov, Christopher Alley, Bianca Davila, Ana G Petrovic, R. Abzalimov, Padmanava Pradhan, Marta Concheiro-Guisan, Elise Champeil
          </td>
          <td>2025-08-14</td>
          <td>Chembiochem : a European journal of chemical biology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Familial adenomatous polyposis (FAP) is an autosomal dominant disorder characterized by germline mutations in the adenomatous polyposis coli (APC) gene. This leads to numerous colorectal adenomas and a high risk of colorectal cancer (CRC). Our stem cell-derived colon organoid model revealed that a heterozygous APC mutation is sufficient to induce colorectal cancer formation. We found a link between APC mutation type, organoid maturation and FAP severity. Here, we show that severe germline mutations in hESCs employ diverse mechanisms of carcinogenesis. FAP1-hESCs expressing a truncated 332-amino acid protein exhibited a hyperactivated mTOR pathway, including PTEN inactivation and increased S6K1 and eIF4E activation. This affected oncogenic c-Myc expression and contributed to apoptosis resistance. Rapamycin treatment restored differentiation potential in FAP1 organoids but not FAP2 organoids, which expressed a larger truncated protein without mTOR pathway activation. Our in vitro colon organoids system findings were validated in human patients. Notably, a colon from a FAP1 patient exhibited high expression of mTOR pathway proteins. These findings highlight the potential of rapamycin for personalized therapy in FAP patients with distinct mTOR-mediated APC mutations. Our colon organoid model is valuable for studying CRC and developing new diagnostic, preventive, and therapeutic approaches to prevent or delay tumorigenesis in FAP patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64d1c7c2702ad998db554a2e2b3a91b1217f0fb1" target='_blank'>
              Rapamycin rescues APC-mutated colon organoid differentiation
              </a>
            </td>
          <td>
            Aline Habib, Rose Mamistvalov, D. Ben-Yosef
          </td>
          <td>2025-07-23</td>
          <td>Cancer Gene Therapy</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Homologous recombination (HR) repairs double-stranded DNA breaks (DSBs) by generating single-stranded DNA (ssDNA), which is initially coated by Replication Protein A (Rpa). Rad51, a recombinase, catalyzes strand invasion but binds ssDNA with lower affinity than Rpa, necessitating mediator proteins like Rad52 (yeast) or BRCA2 (humans) for Rad51 loading. The mechanisms of this exchange remain unclear. We show that Saccharomyces cerevisiae Rad52 uses its disordered C-terminus to sort polydisperse Rad51 into discrete monomers. Using fluorescent-Rad51 and single-molecule optical tweezers, we visualize Rad52-mediated Rad51 filament formation on Rpa-coated ssDNA, preferentially at ssDNA–dsDNA junctions. Deleting the C-terminus of Rad52 disrupts Rad51 sorting and loading. Addition of the Rad51 paralog Rad55–Rad57 enhances Rad51 binding by ~60%. Despite structural differences, Rad52 and BRCA2 share conserved functional features. We propose a unified “Sort, Stack & Extend” (SSE) mechanism by which mediator proteins and paralogs coordinate Rad51 filament assembly during HR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0aab8e82f8ff1dfbfa0fb9286395cafcf5df0b9" target='_blank'>
              Mechanism of Rad51 filament formation by Rad52 and Rad55-Rad57 in homologous recombination
              </a>
            </td>
          <td>
            Jaigeeth Deveryshetty, Ayush Mistry, Sushil Pangeni, Mohamed Ghoneim, Monica Tokmina-Lukaszewska, Steven K Gore, Jie Liu, Vikas Kaushik, Simrithaa Karunakaran, Angela Taddei, W. Heyer, T. Ha, B. Bothner, Edwin Antony
          </td>
          <td>2025-07-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="


 Gliomas are classified as one of the most lethal cancers owing to their invasive and aggressive nature. Numerous genetic and epigenetic changes have been studied in the development and progression of GBM. Dysregulation of epigenetic function may bring about profound changes in gene expression to facilitate glioma formation and development. SETD2, a key epigenetic regulator is frequently mutated in various cancers; SETD2 regulates m6A mRNA methylation (epitranscriptome) via H3K36 trimethylation. The m6A RNA methylation modification regulates various biological functions like gene regulation, cell self–renewal and maintenance, cell differentiation, invasion, and tumorigenesis. The RNA modifications are executed by a set of complex proteins collectively known as writers, readers and erasers (RNA modifiers). These modifiers work along with the molecular cues laid down by SETD2 via H3K36 trimethylation. The positive correlation between SETD2 expression and m6A RNA modifiers highlights its direct involvement in epitranscriptomics. Glioma, a highly malignant grade IV brain tumor, is characterized by limited therapeutic options and poor clinical outcomes. However, the specific role of SETD2 in glioma and its impact on m6A RNA methylation remain inadequately defined. In this study, we investigated the expression of m6A RNA methylation regulators, the molecular pathways contributing to tumor progression, and the outcomes associated with SETD2-mediated m6A modifications in glioma. Our in vitro analyses demonstrated that knocking down SETD2 in glioma cell lines resulted in reduced oncogenic properties, such as decreased cell proliferation and invasiveness. This effect was accompanied by a global reduction in m6A methylation levels within the transcriptome, which was closely linked to the downregulation of key m6A writers, METTL3 and METTL14. These findings suggest that SETD2 plays a crucial role in maintaining m6A methylation, and its disruption can drive glioma progression through altered epitranscriptomic landscapes, offering new insights into molecular oncogenesis and potential therapeutic targets.



 Subhadra Kumari, Srinivasan M. THE ROLE OF SETD2 IN THE MOLECULAR ONCOGENESIS OF GLIOMA THROUGH m6A RNA MODIFICATION [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P75.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33a10b3b7631ab056fcfe5ee35c047e63418abb6" target='_blank'>
              Abstract P75: THE ROLE OF SETD2 IN THE MOLECULAR ONCOGENESIS OF GLIOMA THROUGH m6A RNA MODIFICATION
              </a>
            </td>
          <td>
            Subhadra Kumari, M. Srinivasan
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) involves the expansion of hematopoietic stem cells with ageacquired mutations linked to myeloid malignancy. Advances in next-generation and single-cell sequencing, along with computational modeling, have expanded our ability to detect both common and rare CH drivers, including single-nucleotide variants and mosaic chromosomal alterations, with increasing sensitivity. While sequencing methods differ in accuracy, cost, and ability to detect low-frequency variants, they have deepened our understanding of CH biology. A growing body of evidence has identified both somatic drivers, such as variants in DNMT3A, TET2, and ASXL1, and germline genetic variants that modify CH risk, highlighting the complex interplay between inherited and acquired factors. These collective discoveries are guiding the development of targeted therapies and interventions, particularly for individuals at risk of progression to myeloid neoplasms or cardiovascular disease. Additionally, CH is emerging as a clinically relevant factor in the treatment of solid tumors, where it may influence the tumour microenvironment, treatment response and the risk of therapy-related complications. Risk stratification models are facilitating earlier identification and monitoring of high-risk individuals, enabling personalized treatment decisions. The scope of CH management continues to expand, from surveillance to intervention, with ongoing trials testing preventive strategies in high-risk populations. Emerging trial frameworks emphasize risk stratification, age-appropriateness, inclusive recruitment, decentralized trial models, and the use of traditional clinical and novel endpoints. Together, these advances reflect a shift from passive observation to proactive intervention, charting a course for early detection, precision treatment, and prevention in CH care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6faa3f8b3f8e8d6ae046148a99055b41b7b424" target='_blank'>
              CHIP ahoy: charting a decade of discovery in clonal hematopoiesis.
              </a>
            </td>
          <td>
            Casey K. Wong, Alexandra McDonald, M. Buttigieg, M. Rauh
          </td>
          <td>2025-09-04</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/018e4ec4dd8ef81f5b174f65fbe205bcc992d85a" target='_blank'>
              The Impact of Polyploid Giant Cancer Cells: The Root of Stress Resilience.
              </a>
            </td>
          <td>
            Yuta Ogawa, Lydia Fisher, Tomonori Matsumoto
          </td>
          <td>2025-09-08</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8304854a46599698bac2a6e2d8f83da3bfb9d71b" target='_blank'>
              DNA2 enables growth by restricting recombination-restarted replication.
              </a>
            </td>
          <td>
            Jessica J R Hudson, Rowin Appanah, David Jones, Kathryn Davidson, Alice M Budden, Alina Vaitsiankova, Kok-Lung Chan, Keith W. Caldecott, Antony M. Carr, Ulrich Rass
          </td>
          <td>2025-09-03</td>
          <td>Nature</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="BACKGROUND
Mediator complex subunit 10 (MED10) serves as a critical regulator of eukaryotic gene expression by facilitating RNA polymerase II activity. Our investigation aims to characterize MED10's functional contributions and underlying molecular pathways in hepatocellular carcinoma (HCC) development.


METHODS
MED10 expression patterns in HCC and their correlation with clinicopathological parameters and patient outcomes were examined using bioinformatics databases and immunohistochemistry. Subsequently, we systematically investigated the biological functions of MED10 in the malignant progression of HCC through comprehensive in vitro experiments, including assessments of cell migration (transwell and wound healing assays), proliferative capacity (cell counting kit-8, colony formation, and 5-Ethynyl-2'-deoxyuridine assays), and cell cycle progression (flow cytometry analysis). Furthermore, we elucidated the underlying molecular mechanisms using real-time quantitative PCR (RT-qPCR), western blotting, immunofluorescence staining, and public database analyses. Furthermore, an in vivo subcutaneous xenograft model was employed to validate MED10's impact on tumor growth.


RESULTS
The results revealed a marked increase in MED10 expression levels within HCC tissues, showing a strong association with unfavorable clinical outcomes. Mechanistically, MED10 induced the epithelial-mesenchymal transition (EMT) and enhanced HCC cell migration. Moreover, MED10 overexpression drives HCC cell cycle progression and proliferation by activating rapidly accelerated fibrosarcoma 1 (RAF1), a process potentially mediated through the mitogen-activated protein kinase (MEK)/extracellular signal-regulated kinase (ERK)/cellular myelocytomatosis oncogene (c-Myc) signaling axis.


CONCLUSION
MED10 promotes HCC cell migration and EMT but, more importantly, also drives cell cycle progression and proliferation via RAF1 activation, and is related to the MEK/ERK/c-Myc axis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83554affa0ce8872e89c5a91b5147c43bb5338b7" target='_blank'>
              MED10 as a Novel Oncogenic Driver in HCC: Promoting Cell Cycle Progression and Proliferation Through RAF1 Activation.
              </a>
            </td>
          <td>
            Junhao Liu, Yongxue Lv, Kejun Liu, Zhengquan Li, Bendong Chen, Yang Bu
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Benzo[a]pyrene diol-epoxide (BPDE)-induced DNA adducts contribute to the disproportionate mutagenesis of codon 12 in the KRAS gene, driven by preferential DNA damage and impaired repair. Codon susceptibility, however, extends beyond oncogenic hotspots, suggesting that BPDE lesions may serve as biomarkers of individual DNA repair capacity and cancer risk. While the genotoxic effects of tobacco smoke are well characterised, their influence on DNA repair remains underexplored.Here, we modelled BPDE-adducted KRAS sequences at codons 12 and 14, which have been suggested to exhibit differential repair rates, to assess local helical distortion and its impact on nucleotide excision repair (NER).We show that BPDE adduction at codon 12 induces distinct DNA distortion compared to codon 14, appearing closer to the canonical DNA structure and therefore potentially evading DNA repair, resulting in altered Rad4 binding and compromised lesion recognition.Our findings link the mutational hotspot at KRAS codon 12 to impaired NER and highlight the critical role of local sequence context in repair efficiency. These results provide new insights into the interplay between sequence-dependent DNA structure and repair, with implications for mutation accumulation and cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/deb2f8fc596d96cb0f40c21d8e92383d7aff1334" target='_blank'>
              Local sequence context at KRAS codons modulates DNA repair efficiency: insights from molecular dynamics simulations
              </a>
            </td>
          <td>
            James Davies, Georgina Menzies
          </td>
          <td>2025-09-03</td>
          <td>Frontiers in Molecular Biosciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) remains a global health burden, with disproportionately high mortality in China’s Guangxi region, where endemic Clonorchis sinensis (C. sinensis) infection coincides with elevated HCC incidence. Preliminary single-cell sequencing revealed marked overexpression of cluster of differentiation 24 (CD24) in HCC tissues, suggesting its potential pathological role. This study aims to elucidate the oncogenic mechanisms of C. sinensis excretory-secretory products (CsESPs) and their link to CD24-mediated HCC progression. Methods We employed an integrated clinical and experimental approach. First, clinical cohort analysis assessed CD24 expression in C. sinensis-associated HCC cases. Multiplatform bioinformatics validation (GEPIA/UALCAN/TIMER) evaluated CD24’s prognostic significance and immune microenvironment modulation. Functional studies (quantitative polymerase chain reaction (qPCR), Western blotting, CCK-8 assays, flow cytometry) examined CsESPs’ effects on CD24 expression, cell proliferation, and apoptosis. Mechanistic investigations (chromatin immunoprecipitation, dual-luciferase reporter assays) identified E2F1-mediated transcriptional activation of CD24. siRNA-mediated CD24 knockdown validated its role in CsESPs-driven oncogenesis. Additionally, the expression of immune checkpoint (CTLA-4, LAG-3) was assessed in the co-cultures of peripheral blood mononuclear cells (PBMCs)–HCC cells. Results Clinical cohort analysis confirmed significant CD24 upregulation in HCC, particularly in C. sinensis-infected cases. Bioinformatic analyses linked high CD24 expression to poor prognosis and immune microenvironment alterations. Functional assays demonstrated that CsESPs enhance CD24 expression, promoting proliferation and inhibiting apoptosis. Mechanistically, E2F1 directly binds to CD24 promoter, driving its transcription upon CsESPs exposure. CD24 silencing reversed CsESPs-induced oncogenic effects. Furthermore, CsESPs upregulated immune checkpoints (CTLA-4, LAG-3) in the co-cultures of PBMC–HCC cells, an effect reversed by CD24 knockdown. Conclusions Our findings establish a novel parasitic carcinogenesis paradigm wherein C. sinensis promotes HCC development through E2F1-mediated transcriptional activation of CD24, simultaneously identifying prognostic biomarkers and therapeutic targets while suggesting combinatory immunotherapy strategies for parasite-associated HCC. Graphical Abstract Supplementary Information The online version contains supplementary material available at 10.1186/s13071-025-06979-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c44a225526ec9b8357a014fc963659daa1b7f9c" target='_blank'>
              Clonorchis sinensis-driven hepatocarcinogenesis via E2F1-CD24 transcriptional axis: mechanistic and therapeutic implications
              </a>
            </td>
          <td>
            Wen-Min Lu, Jin Yan, Zhao-Ji Liu, Yong Wu, Qian-Ru Cui, Ji Feng, Yu Chen, Guang-Zhi Zhu, Tao Peng, Jing Zhou, Guo-Dong Lu
          </td>
          <td>2025-08-19</td>
          <td>Parasites & Vectors</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Human topoisomerase II alpha (TOP2α) resolves DNA intertwines between sister chromatids during mitosis. How cohesin, an SMC complex that holds sister chromatids, affects TOP2α decatenation is unclear. To addres this, we developed a quadruple-trap optical tweezers assay to create DNA braids and study TOP2α decatenation at the single-molecule level in real-time. We show that TOP2α resolves both single and multiple braids but becomes inefficient at forces exceeding 28 pN. TOP2α is sensitive to DNA geometry, exhibiting a chiral preference for right-handed braid crossings, and requires loading directly at DNA crossovers to act. Pre-loading TOP2α onto individual DNA strands before braid formation, in the presence of ATP, prevents decatenation. Finally, we show that human cohesin, but not condensin I, binds stably to DNA braids and blocks TOP2α activity. Our study provides novel insights into the role of substrate accessibility in regulating TOP2α‘s activity and highlights cohesin as a barrier to decatenation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7fc5b97ef55e14d8e05557d238a795889fb56aa" target='_blank'>
              Substrate accessibility regulation of human TopIIa decatenation by cohesin
              </a>
            </td>
          <td>
            E. Cutts, Sanjana Saravanan, Paul Girvan, Benjamin Ambrose, Gemma L M Fisher, David S. Rueda, Luis Aragon
          </td>
          <td>2025-08-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="CRISPR/Cas9 gene-editing technology has revolutionized biomedical research by enabling precise, efficient, and cost-effective genome manipulation. This paper focuses on the application of CRISPR/Cas9 in cancer research, with an emphasis on triple-negative breast cancer (TNBC), a particularly aggressive and treatment-resistant subtype of breast cancer that lacks expression of estrogen, progesterone, and HER2 receptors. Breast cancer is the most commonly diagnosed cancer among women worldwide, and TNBC represents one of its most challenging forms due to the absence of effective targeted therapies. While hormone receptor-positive and HER2-positive subtypes benefit from endocrine and targeted treatments, TNBC patients are often limited to chemotherapy, which is associated with high toxicity and limited efficacy. This review synthesizes recent CRISPR-based studies that identify oncogenes and tumor suppressors relevant to TNBC, explores advanced delivery systems such as non-viral nanoparticles, and evaluates the therapeutic promise of this technology. Despite current limitations—including off-target effects, delivery challenges, and ethical concerns—CRISPR/Cas9 offers a promising path toward developing personalized, gene-targeted treatments for TNBC and other hard-to-treat cancers.

Key words: CRISPR/Cas9, gene editing, breast cancer, triple negative breast cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c17f19abfb2d0fe3b181a39613fc90c46bf2f21" target='_blank'>
              Review of Current Applications of CRISPR Gene Editing Techniques in Breast Cancer Therapeutics Development
              </a>
            </td>
          <td>
            Yichen Zhang
          </td>
          <td>2025-08-01</td>
          <td>International Journal of Health Sciences and Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSOC) is the most lethal gynecological malignancy. Therapeutic options remain limited for patients lacking predictive biomarkers, particularly those with BRCA wild-type tumors or those who have acquired resistance to both PARP inhibitors and platinum-based chemotherapy. Replication stress, TP53 mutations, and genomic instability characterize HGSOC. The cellular response to replication stress is primarily mediated by checkpoint kinases; however, this mechanism is frequently impaired in tumor cells. Consequently, cancer cells become increasingly dependent on the replication stress response (RSR) pathway for survival, and susceptible to therapies targeting the ATR-CHK1-WEE1 axis—a key regulator of genomic integrity. Inhibition of these checkpoint kinases can disrupt cell cycle control, inducing mitotic catastrophe and subsequent cancer cell death. Another defining feature of HGSOC is its immunosuppressive tumor microenvironment (TME), which has limited the efficacy of immune checkpoint inhibitors. Emerging evidence suggests that inhibition of the RSR pathway may not only exploit intrinsic tumor vulnerabilities but also modulate the TME to enhance anti-tumor immune responses. This provides rationale for combination approaches integrating RSR pathway inhibitors with innovative immune checkpoint blockade (ICB). This review examines the mechanistic rationale and therapeutic potential of such combinations, drawing on both preclinical and clinical data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/faa92875288685395bcc326c756f9f278710481b" target='_blank'>
              Interplay of replication stress response and immune microenvironment in high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Laura Venegas, Stephanie Lheureux
          </td>
          <td>2025-09-05</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Stem cells are essential for tissue maintenance, repair, and regeneration, yet their dysregulation gives rise to cancer stem cells (CSCs), which drive tumor progression, metastasis, and therapy resistance. Despite extensive research on stemness and oncogenesis, a critical gap remains in our understanding of how the transcriptomic landscapes of normal somatic stem cells (SSCs) diverge from those of CSCs to enable malignancy. This review synthesizes current knowledge of the key signaling pathways (Wnt, Notch, Hedgehog, TGF-β), transcription factors (Oct4, Sox2, Nanog, c-Myc, YAP/TAZ), and epigenetic mechanisms (chromatin remodeling, DNA methylation, microRNA regulation) that govern stemness in SSCs and are hijacked or dysregulated in CSCs. We highlight how context-specific modulation of these pathways distinguishes physiological regeneration from tumorigenesis. Importantly, we discuss the role of epithelial–mesenchymal transition (EMT), cellular plasticity, and microenvironmental cues in reprogramming and maintaining CSC phenotypes. By integrating transcriptomic and epigenetic insights across cancer biology and regenerative medicine, this review provides a framework for identifying vulnerabilities specific to CSCs while still preserving normal stem cell function. Understanding these distinctions is essential for the development of targeted therapies that minimize damage to healthy tissues and advance precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fb9e90607e3b0ca792b0cb08f2440605b878350" target='_blank'>
              Transcriptomic Comparisons of Somatic and Cancer Stem Cells
              </a>
            </td>
          <td>
            Austin Drysch, Arun Ahuja, Dillan Prasad, Rishi Jain, Sharbel Romanos, A. Alwakeal, Christopher S. Ahuja
          </td>
          <td>2025-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Despite the critical role of DNA methylation, clinical implementations harnessing its promise have not been described in acute myeloid leukemia. Utilizing DNA methylation from 3314 leukemia patient samples across 11 harmonized cohorts, we describe the Acute Leukemia Methylome Atlas, which includes robust models capable of accurately predicting AML subtypes. A genome-wide prognostic model as well as a targeted panel of 38 CpGs significantly predict five-year survival in our pediatric and adult test cohorts. To accelerate rapid clinical utility, we develop a specimen-to-result protocol that uses long-read nanopore sequencing and machine learning to characterize patients’ whole genomes and epigenomes. Clinical validation on patient samples confirms high concordance between epigenomic signatures and genomic lesions, though uniquely rare karyotypes remained challenging due to limited available training data. These results unveil the potential for increased affordability, speed, and accuracy for patients in need of complex molecular diagnosis and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081a93b57ec5afcf494724d930e4432c5308fda4" target='_blank'>
              Epigenomic diagnosis and prognosis of Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Francisco Marchi, Vivek M Shastri, Richard J Marrero, Nam Nguyen, Antonella Öttl, Ann-Kathrin Schade, Marieke Landwehr, O. Krali, J. Nordlund, Matin Ghavami, Fernando Sckaff, Vikash K. Mansinghka, Xueyuan Cao, William Slayton, Petr Starostik, Christopher Cogle, Raul C Ribeiro, J. Rubnitz, J. Klco, Abdelrahman A. Elsayed, A. Gamis, Timothy J. Triche, R. Ries, E. A. Kolb, R. Aplenc, T. Alonzo, S. Pounds, S. Meshinchi, J. Lamba
          </td>
          <td>2025-07-29</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>86</td>
        </tr>

        <tr id="Abstract Replication stress, a hallmark of cancer cells, is detected by checkpoint mechanisms that trigger a range of cellular responses. Among these, the preservation of replication fork integrity is crucial for ensuring survival in the presence of DNA damage. In budding yeast checkpoint mutants, DNA damage leads to irreversible replication fork arrest and subsequent cell death, though the underlying mechanism remains unclear. Our study reveals that several DNA processing factors, including Rad51, the Rad5 HIRAN and helicase domains, and the catalytic activity of Mus81, contribute to this lethality. Nevertheless, their roles are masked by their essential functions in cell survival after damage removal. Additionally, we show that these factors, along with Exo1, drive the gradual degradation of nascent DNA at replication forks upon DNA damage. Notably, this degradation can be mitigated by expression of human PrimPol, which is absent in yeast. These findings suggest that the essential role of S-phase checkpoints upon DNA damage is to safeguard stalled replication forks from aberrant processing, thereby preserving nascent DNA integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06a85a0f3ac89e36eb9aea37ff07e545bcefac9e" target='_blank'>
              S-phase checkpoint protects from aberrant replication fork processing and degradation
              </a>
            </td>
          <td>
            Iván Núñez-Martín, L. Drury, María I Martínez-Jiménez, Luis Blanco, J. Diffley, A. Aguilera, Belén Gómez-González
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Nucleoporin 98 rearrangements (NUP98r) are recurrent in myeloid neoplasms and are subtype-defining for acute myeloid leukemia (AML) in the World Health Organization Classification 5th edition (WHO5) and the International Consensus Classification (ICC). Identification of NUP98r is essential given frequency of treatment resistance and possibility of sensitivity to targeted therapies. However, NUP98r is often cryptic on karyotype and has over 40 described partners. Therefore, it is underdiagnosed in the absence of dedicated testing that is not routine at many practices, e.g. RNA-based next generation sequencing (NGS), NUP98 break-apart FISH, or RT-PCR for specific NUP98 fusions. Historically, AML with NUP98r has received the most attention in pediatric AML, where its incidence is highest, but has been increasingly characterized in adult AML. By contrast, the incidence and behavior of NUP98 fusions in myelodysplastic syndromes (MDS) is less understood and based predominantly on case reports. In this study, we describe our adult institutional experience with a clinically validated anchored multiplex PCR RNA-based targeted NGS assay, explore strategies for rational use of specific testing for NUP98r including a proof-of-principle based on WT1 and FLT3-ITD mutational status, and integrate our results with a review of the literature. In total, we identified 3 MDS and 15 AML patients with NUP98r as the genetic driver, including two novel fusion partners (FGF14 and LAMC3), thus highlighting the utility of NGS testing to detect NUP98 fusions. Recognition of NUP98r in myeloid neoplasms is crucial for accurate diagnosis and prognosis, with significant implications for therapy or enrollment in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa097a5f4c5088e97ab7df0262f894ba23236049" target='_blank'>
              Strategies for identifying NUP98 rearrangements in adult myeloid neoplasms.
              </a>
            </td>
          <td>
            Lisa D Yuen, Robert P Hasserjian, Amir T Fathi, M. Luskin, Eric S. Winer, Paola Dal Cin, Annette S. Kim, R. C. Lindsley, Harrison K. Tsai, Valentina Nardi
          </td>
          <td>2025-08-14</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Objective Patients diagnosed with metastatic pheochromocytoma/paraganglioma (PCC/PGL) have limited treatment options. In some cases, peptide receptor radionuclide therapy (PRRT) is followed by an eruption of metastases, possibly originating from tumor cells with a radioresistant phenotype. However, the underlying mechanisms of radioresistance in PCC/PGL remain largely unknown and appropriate models are missing. Methods Two genetically modified mouse pheochromocytoma (MPC) cell lines, one positive and one negative for hypoxia-inducible factor 2α expression (MPC+HIF-2α and MPC+EV [empty vector], respectively), were X-ray-conditioned through fractionated irradiation at sublethal doses. Two procedures were tested: one allowed for recovery between each irradiation step (recIR), while the other demanded daily irradiation (dayIR). Changes in cell morphology, growth rates, and DNA repair (γH2AX immunostaining) were characterized in response to irradiation. Results We generated two MPC+HIF-2α- and two MPC+EV-derived cell lines that tolerate irradiations with X-rays at dose fractions of 2 Gy per day without significant growth inhibition. All recIR-and dayIR-conditioned cell lines showed increased DNA repair capacity. Morphological changes toward stronger clustering and slower growth were more pronounced in dayIR-conditioned than in recIR-conditioned cell lines. X-ray-conditioned MPC+HIF-2α cells showed the highest increase in resistance to X-ray-treatment with dose fractions up to 5 Gy per day. Conclusion The herein established X-ray-conditioned MPC cell lines represent PCC/PGL models with a radioresistant phenotype. Further investigations on the radiation-induced genetic responses of these cell lines, their corresponding tumor spheroids, and tumor allografts in mice will help to elucidate the underlying mechanisms of acquired radioresistance and radionuclide therapy-induced metastatic eruption in PCC/PGL. Lastly, the suitability of advanced PRRT and complementary treatments can be tested to improve theranostic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42539527ca697e5d2588e236622b3f72185855c8" target='_blank'>
              Radioresistant mouse pheochromocytoma cell lines
              </a>
            </td>
          <td>
            S. Lemm, Marcel Gebhardt, Thomas Groß, Susan Richter, M. Ullrich, Jens Pietzsch
          </td>
          <td>2025-07-30</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10f3ac7d8f1f9723125587e98e6c28a4657680b4" target='_blank'>
              Integrated control of cancer stemness by σ1 receptor in advanced prostate cancer.
              </a>
            </td>
          <td>
            G. Civenni, G. Sandrini, Jessica Merulla, C. Musumeci, Elisa Federici, Arianna Vallegra, A. Kokanovic, Simone Mosole, D. Shinde, Elisa Sorrenti, Alyssa J J Paganoni, Martina Marchetti, Riccardo Valzelli, D. Albino, Matteo Pecoraro, Andrea Rinaldi, Marco Bolis, Roger Geiger, Tobias Winge, Catharina Holtschulte, E. Laurini, S. Pricl, G. M. Carbone, Bernhard Wünsch, C. Catapano
          </td>
          <td>2025-09-02</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c7a100e23150d73084b13fa87d7c5ad45a0fbaf" target='_blank'>
              The multifaceted role of KIF15 in cancer progression and therapy.
              </a>
            </td>
          <td>
            Sarah Qutayba Badraldin, Karar H. Alfarttoosi, Hayder Naji Sameer, A. K. Bishoyi, Subbulakshmi Ganesan, Aman Shankhyan, Subhashree Ray, Ahmed Yaseen, Zainab H. Athab, Mohaned Adil
          </td>
          <td>2025-08-16</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Activation of the DNA damage checkpoint upon genotoxin treatment induces a multitude of cellular changes to cope with genome stress. After prolonged genotoxin treatment, the checkpoint can be downregulated to allow cell cycle and growth resumption. In yeast, downregulation of the DNA damage checkpoint requires the Srs2 DNA helicase, which removes the ssDNA binding complex replication protein A (RPA) and the associated Mec1 checkpoint kinase from DNA, thus dampening Mec1-mediated checkpoint. However, it is unclear whether the ‘anti-checkpoint’ role of Srs2 is temporally and spatially regulated to allow timely checkpoint termination while preventing superfluous RPA removal. Here we address this question by examining regulatory elements of Srs2, such as its phosphorylation, sumoylation, and protein-interaction sites. Our genetic analyses and checkpoint level assessment suggest that the RPA countering role of Srs2 is promoted by Srs2 binding to proliferating cell nuclear antigen (PCNA), which recruits Srs2 to a subset of ssDNA containing regions. RPA antagonism is further fostered by Srs2 sumoylation, which we found depends on the Srs2-PCNA interaction and Mec1, and peaks after Mec1 activity reaches maximal levels. These data support a model in which Srs2 recruitment to PCNA adjacent to ssDNA-RPA filaments, followed by Mec1-dependent sumoylation, modulates RPA-mediated checkpoint signaling, while Srs2 action is limited at ssDNA regions lacking proximal PCNA, thereby favoring RPA-mediated ssDNA protection and repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3cc5ba6f7dbf7cf8edfcba66210c75993616a9c" target='_blank'>
              Srs2 binding to proliferating cell nuclear antigen (PCNA) and its sumoylation contribute to replication protein A (RPA) antagonism during the DNA damage response
              </a>
            </td>
          <td>
            Jiayi Fan, Nalini Dhingra, Tammy Yang, Vicki Yang, Xiaolan Zhao
          </td>
          <td>2025-08-01</td>
          <td>eLife</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7fc879badbc6a58189ea836408c6e92780a41f0" target='_blank'>
              Childhood Cancer Predisposition and Evolutionary Constraints: Novel lessons from Germline Genomes from 1,127 Children with Cancer.
              </a>
            </td>
          <td>
            U. Stoltze, T. V. O. Hansen, J. Foss-Skiftesvik, A. Byrjalsen, K. Henriksen, Adrian Otamendi Laspiur, A. Gerdes, S. Ostrowski, Erik Sørensen, Mads Bak, C. Lautrup, K. Grønskov, Elena Papaleo, H. Hasle, T. Mikkelsen, P. Wehner, A. Saksager, M. K. Andersen, Mimi Kjærsgaard, T. D. Hjortshøj, J. H. Frederiksen, David Scheie, T. Olsen, R. Tuckuviene, Marianne Olsen, Z. Tümer, R. Mathiasen, J. Brok, A. Sehested, Bram L Gorissen, Simon Rasmussen, K. Karczewski, Lisa Lyngsie Hjalgrim, K. Wadt, Kjeld Schmiegelow
          </td>
          <td>2025-08-15</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="RAD51C is a tumor suppressor gene with over 285 variants of unknown significance (VUS) found in primary ovarian tumors. RAD51C is a paralog of the recombinase RAD51, and it forms complexes with other paralogs to regulate RAD51 activity. We screened 27 ovarian cancer-derived RAD51C VUS to identify those that affect the assembly of functional tetrameric RAD51B-C-D-XRCC2 (BCDX2) complex. With yeast 3-hybrid and biochemical analyses, we identify a mutation cluster of the RAD51C Walker B region affecting protein interactions with other RAD51 paralogs. By further analyzing these variants for homologous recombination (HR), replication fork regression, DNA binding and ATPase activity, and RAD51 filament formation, we identified separation-of-function alleles that uncouple RAD51C distinct enzymatic activities with HR and replication. Thus, our analysis of RAD51C identifies additional VUS with functional defects, which will aid in pathogenicity classification and inform future strategies to treat individuals harboring RAD51C loss-of-function alleles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bffbfb833ccd0e556ecb0a9a1063bc4252e91613" target='_blank'>
              Comprehensive RAD51C ovarian cancer variant analysis uncouples homologous recombination and replicative functions
              </a>
            </td>
          <td>
            Hayley L. Rein, Yashpal Rawal, Anna L Palovcak-Lightbourn, Gayatri S Ganesan, Phoebe S Parker, Reagan Russell, Kristie E Darrah, Mohammad Afsar, Meghan R Sullivan, Sarah R. Hengel, M. Radke, P. Opresko, J. Yanowitz, E. Greene, Jung-Min Lee, S. Domchek, Elizabeth M. Swisher, Shaun K. Olsen, Patrick Sung, Kara A. Bernstein
          </td>
          <td>2025-07-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Precise, monoallelic expression of imprinted genes is governed by cis regulatory elements called imprinting control regions (ICRs) and enhancer-promoter (E-P) interactions shaped by local chromatin architecture. The Igf2/H19 locus employs allele-specific CTCF binding at the ICR to instruct enhancer accessibility to maternal H19 and paternal Igf2 promoters. Here, we investigate the CTCF-bound centrally conserved domain (CCD), intergenic to H19 and Igf2, and an adjacent widely expressed lncRNA. Using transgenic mice, deletion alleles reinforced CCD as a neonatal muscle-specific repressor of maternal Igf2. However, deletion of the abutting lncRNA did not affect Igf2 levels. Unexpectedly, in adult skeletal muscle where Igf2 is normally repressed, absence of CCD resulted in remarkable, high-level activation of Igf2 from both parental alleles. Through multimodal chromatin analyses, we identified a conserved putative adult skeletal muscle enhancer (PaSME) insulated between chromatin domains at ICR and CCD. We propose that removal of CCD allows PaSME to drive robust abnormal Igf2 activation on both alleles in adult skeletal muscle. Thus, we uncover CCD as a developmental biallelic muscle-specific repressor, adding a new layer of architectural regulation to the extensively studied Igf2/H19 locus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3725abb4c970419b7932a2b1cf59a02ed186a3dd" target='_blank'>
              Igf2 adult-specific skeletal muscle enhancer activity revealed in mice with intergenic CTCF boundary deletion
              </a>
            </td>
          <td>
            J. L. Thorvaldsen, Aimee M. Juan, Yemin Lan, Christopher Krapp, Marisa S. Bartolomei
          </td>
          <td>2025-08-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Base excision repair (BER) is a DNA repair pathway responsible for protecting the genome against modified nucleotides. DNA polymerase β (Pol β) participates in this process by removing the remnants of a damaged nucleotide and filling in the resulting gap. Pol β is overexpressed in some cancers and is synthetic lethal in cells deficient in BRCA1/2, providing additional impetus for identifying inhibitors of this enzyme. We report noncovalent Pol β inhibitors that are nonnative nucleotides. The inhibitors were identified via a combination of structural and biochemical analysis, as well as serendipity, from an initial library of covalent inhibitor candidates in which diversity was introduced sequentially at the C3'- and C5-positions of pyrimidine nucleotides. The molecules are among the most potent Pol β inhibitors (Ki ≤ 70 nM) of the enzyme's polymerase and lyase activities. Kinetic analyses reveal that the molecules inhibit Pol β noncompetitively. Fluorescence anisotropy and kinetic experiments reveal that the more potent inhibitor binds in the lyase domain and does not prevent DNA binding. Neither the more potent noncompetitive inhibitor nor a neutral protide exhibits cytotoxic synergism with the DNA damaging agent methyl methanesulfonate in HeLa cells. Cell permeability experiments suggest that micromolar levels of the more potent noncompetitive inhibitor and corresponding protide are taken up by HeLa cells following 24 h incubation (25 μM). However, based upon a comparison with other molecules, it is possible that they are membrane bound. The molecules identified could be useful tools in biochemical studies and provide a starting point for creating new Pol β inhibitors that function in cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38f4dcc329d5c6e0100f120af6c1e368b06b918e" target='_blank'>
              Noncompetitive Inhibition of DNA Polymerase β by a Nonnative Nucleotide.
              </a>
            </td>
          <td>
            A. H. Howlader, Xuanhe Jiang, Zehui Zhou, Marc M Greenberg
          </td>
          <td>2025-09-07</td>
          <td>The Journal of organic chemistry</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mitotic DNA synthesis (MiDAS) serves to complete the replication of genomic loci that are not fully replicated in S phase in response to replication stress. Previous studies suggest that MiDAS might proceed via break-induced DNA replication, a sub-pathway of homologous recombination repair activated at broken or collapsed replication forks. We set out to define whether DNA double strand break end-resection factors play a role in MiDAS. Here, we show that several core end-resection factors, including MRE11, CtIP and BRCA1 are essential for MiDAS. In addition, while loss of WRN or DNA2 impairs MiDAS, there is no requirement for other known end-resection factors such as EXO1 and BLM. Moreover, both the exonuclease and the helicase activities of WRN contribute to MiDAS. Because oncogene-induced replication stress is common in cancers, targeting of WRN or other factors required for MiDAS could facilitate the development of targeted cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4914c3ccc0499e2ad21835d0e0d8803ffd6d2ef" target='_blank'>
              DNA double-strand break end resection factors and WRN facilitate mitotic DNA synthesis in human cells
              </a>
            </td>
          <td>
            Szymon A. Barwacz, Katrine Lundgaard, Wei Wu, Philipp H. Richter, Liqun Ren, Rahul Bhowmick, Marisa M. Gonçalves Dinis, Masato T. Kanemaki, Ying Liu
          </td>
          <td>2025-08-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="


 Meningiomas are the only primary tumors that undergo hyperostosis in the tumor microenvironment. Clinical observations suggest TRAF7short somatic variants (SSVs) are enriched in skull-base meningiomas, and that SSVs targeting the TRAF7 WD40-domain may be associated with meningioma-induced hyperostosis (MIH). Here, we integrate single-cell RNA sequencing and lineage tracing data with mechanistic and functional studies across re- gionally distinct primary meningioma cell cultures. We ﬁnd that TRAF7 WD40-domain mutations drive menin- gioma cell transdifferentiation via PRRX1 into an osteoblast-like state that is associated with mineralization in vitro and MIH in vivo.



 Single-cell RNA sequencing,mitochondrial alteration enrichment from single-cell transcriptomes to establish re- latedness (MAESTER) lineage tracing, RNA velocity and pseudotime analyses were used to deﬁne cell types, cell states, and evolutionary trajectories from 8 human samples. Regionally distinct primary meningioma cell cul- tures (n=6) from 3 meningiomas with MIH were derived for mechanistic and functional studies. Targeted next- generation DNA sequencing, QPCR, immunoﬂuorescence, cell proliferation, immunoprecipitation with western blot, and mineralization assays with alizarin red stains were used to deﬁne how TRAF7 WD40-domain mutations inﬂuence protein function and cell state.



 Single-cell transcriptomics and lineage tracing identiﬁed a differentiation event from meningioma origin clus- ter to an osteoblast-like meningioma cell cluster that was enriched for bone-associated genes, driven by genes associated with PRRX1. Primary meningioma cell cultures were validated using DNA methylation proﬁling and SSTR2A expression. Only cell cultures from intraosseous regions expressed RUNX2, SP7, or ALPL, and under- went mineralization in vitro. Suppression of mutant TRAF7 using siRNAs blocked in vitro mineralization, and media from meningioma cell cultures from intraosseous regions induced mineralization of TRAF7 wildtype meningioma cell cultures in vitro.



 These data demonstrate that TRAF7 WD40-domain mutations drive meningioma cell differentiation and intra- tumor hyperostosis via expression of PRRX1, and shed light on fundamental mechanisms underlying lamellar bone formation.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/190662e8dee7fcce899ad86ef23836987311c4dd" target='_blank'>
              TRAF7 WD40-DOMAIN MUTATIONS DRIVE MENINGIOMA TRANSDIFFERENTIATION AND INTRATUMOR HYPEROSTOSIS
              </a>
            </td>
          <td>
            C. Eaton, E. Goldschmidt, Ayush Aggarwal, Zhenhong Chen, David R Raleigh
          </td>
          <td>2025-09-01</td>
          <td>Neuro-Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Nasopharyngeal carcinoma (NPC) is an epithelial malignancy closely associated with Epstein-Barr virus (EBV) infection. Although patients with early-stage NPC can achieve a high cure rate through radiotherapy, recurrence and distant metastasis remain the primary causes of treatment failure in patients with advanced-stage NPC. Circular RNA (circRNA) is a class of covalently closed non-coding RNAs involved in multiple aspects of tumor biology. Recent evidence has shown that certain circRNAs can encode functional peptides, which participate in the regulation of tumor-related signaling pathways. In NPC, circRNAs have been implicated in the modulation of signaling pathways, including NF-κB and JAK/STAT, both of which are activated in the EBV-infected microenvironment. Furthermore, frequently mutated genes in NPC, such as TNF receptor-associated factor 3 and cylindromatosis lysine 63 deubiquitinase, are known regulators of the NF-κB pathway, suggesting a potential link between genetic alterations and circRNA-related mechanisms. This article systematically reviews the biological mechanisms of circRNA-encoded peptides, summarizes the expression and function of circRNA in NPC and focuses on discussing the potential roles of circRNA-encoded peptides in tumor microenvironment regulation, immune escape and clinical application prospects. By integrating existing research results, this article aims to provide a new perspective and theoretical basis for the in-depth exploration of circRNA-encoded peptides in the field of NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/026498398de031d10f9b113d1987dfe09fb352cd" target='_blank'>
              Functional mechanisms of circular RNA-encoded peptides and future research strategies and directions in nasopharyngeal carcinoma (Review)
              </a>
            </td>
          <td>
            Wei-hua Xu, Zhichao Ma, Wei Gong, Shengmiao Fu, Xinping Chen
          </td>
          <td>2025-08-18</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The activating signal co-integrator 1 complex subunit 3 (ASCC3), the largest subunit of ASCC, is one of two Ski2-like helicases with duplicated helicase cassettes encoded by the human genome. ASCC3 has been implicated in transcriptional regulation, alkylation damage repair, and ribosome quality control. In addition, published proteomics studies suggest that ASCC3 is associated with stalled forks. However, little is known about its role in replication stress. Here we report that ASCC3 is recruited to stalled forks by its binding partner ASCC2, whose recruitment to stalled forks is dependent upon both its ubiquitin binding activity and polyubiquitylation of PCNA at K164 that is catalyzed by SHPRH, HLTF, and RFWD3. In response to replication stress. ASCC3 unwinds DNA, generating both parental and nascent ssDNA. This unwinding activity is required to promote fork reversal. We demonstrate that ASCC3 also interacts with RPA and stimulates RPA accumulation on ssDNA upon replication stress, promoting ATR activation. Furthermore, ASCC3 functions to antagonize RAD51-mediated recombination and prevents accumulation of chromosome breaks/gaps and mis-segregation in response to replication stress. Our work underscores a previously uncharacterized but critical role of ASCC3 in controlling multiple replication stress responses to maintain genomic stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ab7f91fd278f6d682cb1ebb5121a136d61cd073" target='_blank'>
              The Ski2 helicase ASCC3 unwinds DNA upon fork stalling to control replication stress responses
              </a>
            </td>
          <td>
            Shi-Quan Cui, Nicole L. Batenburg, Yan Coulombe, Aruna Arumugam, John R. Walker, Sadaf Valeh Sheida, A. Bielinsky, Markus C. Wahl, Jean-Yves Masson, Xu-Dong Zhu
          </td>
          <td>2025-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Polymerase epsilon mutant (POLE-mut) endometrial cancers (EC) are characterized by a near 100% disease-specific survival rate, even when treated by surgery alone. This spectacular survival, combined with the ultramutated genome and high level of neoantigens in these tumors, indicates a substantial degree of immune control in preventing disease spread and recurrence. Although these features are intriguing, the immune infiltration of POLE-mut EC has predominantly been confined to immunohistochemistry studies. Here, we used state of the art single-cell RNA and TCR sequencing to characterize the immune landscape of POLE-mutant ECs. Moreover, we uniquely analyzed patient blood samples taken two to eight years after curative treatment to assess formation of long-term immune memory in circulation. We identified specialized tumor-infiltrating myeloid subsets at different stages of maturation, an array of lymphocytes ranging from immature to cytotoxic and adaptive natural killer (NK) as well as tumor-reactive exhausted and effector T cells, contributing to a highly inflammatory anti-tumor response. Remarkably, our analysis of blood samples taken years after curative treatment uncovered the presence of tumor-reactive T cell clones that matched the primary tumor. This indicates the formation of systemic long-term memory immune responses in POLE-mut EC survivors. Our study highlights the distinctive immunogenicity of POLE-mut EC and identifies key features associated with persistent anti-tumor immunity that may contribute to prolonged, relapse-free survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0198b259b8d5d685a1ba64c108955b1306a25438" target='_blank'>
              Spatiotemporal immune landscape and long-term immune memory in POLE-mutant endometrial cancer at the single-cell level.
              </a>
            </td>
          <td>
            K. Brummel, M. Requesens, N. van Rooij, H. Workel, F. Eggink, A. Plat, R. Wardenaar, D. Spierings, F. Foijer, David N. Church, Joost Bart, H. Nijman, M. de Bruyn
          </td>
          <td>2025-08-29</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Insufficient infiltration of cytotoxic T cells into solid tumors remains a critical obstacle in cancer immunotherapy. Despite extensive efforts to comprehend the mechanisms governing this limited infiltration, few studies have focused on the evolution of T cell motility behavior after co-culture. In this study, we combined quantitative cell trajectory analysis, computational modeling, and bulk/single-cell RNA sequencing to systematically characterize the impact of cell interactions. We reveal that in a 2.5D co-culture system with multiple cancer-cell clusters, cancer-specific T cells exhibit increased directional persistence, which facilitates their efficient searching of cancer-cell clusters. Additionally, these T cells form prolonged interactions with cancer cells, which is the most crucial factor for their accumulation on cancer-cell clusters. Furthermore, post-interaction, a cancer-cell subpopulation displays immunosuppressive traits, reducing T cell attractant expression, and undergoing epithelial-to-mesenchymal transition. These findings offer valuable insights into improving immunotherapy efficacy and tackling T cell infiltration challenges in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f0a380330210bd455d4b1fb482cbeaabdb64b1d" target='_blank'>
              Deciphering antigen-specific T cell navigation tactics and cancer immune evasion in co-cultures
              </a>
            </td>
          <td>
            Xinyue Li, Taoli Jin, Lisha Wang, Senbin Wu, Yerong Liu, Ming Li, Weijing Han, Xuefei Li
          </td>
          <td>2025-07-31</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87bd4da898162ddcade022e38bbb9fc6e885ebf4" target='_blank'>
              Intratumoral Microbe Correlated with Expression of DNA Methylation Genes in Hepatocellular Carcinoma.
              </a>
            </td>
          <td>
            Wei Chen, Xiang Zhang, Minhui Chi, Qi Zheng
          </td>
          <td>2025-07-25</td>
          <td>Digestive diseases and sciences</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="


 Control of cell division is tightly regulated in eukaryotic cells, and dysfunction in cell cycle checkpoints is a key hallmark of malignant transformation that promotes a fitness advantage over non-cancer cells. One of the most critical mechanisms of cell cycle regulation is via the cyclin dependent kinases (CDKs), which connect resource availability sensing and growth signaling with cell division and transcription elongation processes. Novel combination therapy approaches to co-target cell cycle and transcriptional CDKs may improve cancer-specific targeting of CDK dysfunction. In the current study, we assessed the effectiveness of fadraciclib, a new CDK2/9 inhibitor, for the treatment of advanced colorectal cancer (CRC).



 A panel of eighteen CRC patient-derived organoids (PDOs) were used to assess the efficacy of fadraciclib. Efficacy was further validated in patient-derived xenografts (PDXs). CDK2/9 target inhibition, cell cycle arrest, and cell killing mechanisms were investigated using western blotting, flow cytometry, and immunofluorescence staining, respectively.



 CRC PDOs exhibited greater sensitivity to fadraciclib compared to chemotherapy and palbociclib. This efficacy was validated in vivo using three matched patient-derived xenografts (PDXs), showing significant tumor growth inhibition with fadraciclib compared to vehicle (p<0.05) and no serious adverse effects. Fadraciclib induced G2/M cell cycle arrest, leading to multipolar mitosis and anaphase catastrophe.



 Our results using patient-derived models suggest that fadraciclib is a promising therapy for advanced CRC by inhibiting CDK2/9, which affects critical pathways in cell cycle regulation and transcription.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f24e0b1ceb93e513f1201f67a1ccdb5200644cb" target='_blank'>
              Dual targeting of conserved cell cycle and transcription programs in advanced colorectal cancer by fadraciclib
              </a>
            </td>
          <td>
            Mohammad Zokaasadi, Wylie K Watlington, Divya L Dayanidhi, John B Mantyh, Gabrielle Rupprecht, Shannon J McCall, David G Blake, Jason A. Somarelli, David S Hsu
          </td>
          <td>2025-08-08</td>
          <td>Evolution, Medicine, and Public Health</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Irrational antibiotic use contributes to the development of antibiotic resistance in bacteria, which is a major cause of healthcare-associated infections globally. Molecular research has shown that multiple resistance frequently develops from the uptake of pre-existing resistance genes, which are subsequently intensified under selective pressures. Resistant genes spread and are acquired through mobile genetic elements which are essential for facilitating horizontal gene transfer. MGEs have been identified as carriers of genetic material and are a significant player in evolutionary processes. These include insertion sequences, transposons, integrative and conjugative elements, plasmids, and genomic islands, all of which can transfer between and within DNA molecules. With an emphasis on pathogenic bacteria, this review highlights the salient features of the MGEs that contribute to the development and spread of antibiotic resistance. MGEs carry non-essential genes, including AMR and virulence genes, which can enhance the adaptability and fitness of their bacterial hosts. These elements employ evolutionary strategies to facilitate their replication and dissemination, thus enabling survival without positive selection for the harboring of beneficial genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77b64cdf50075b00fb810e3ab3030d74dcc1372a" target='_blank'>
              Unraveling the role of mobile genetic elements in antibiotic resistance transmission and defense strategies in bacteria
              </a>
            </td>
          <td>
            Ranjith Kumavath, Puja Gupta, Eswar Rao Tatta, Mahima S Mohan, Simi Asma Salim, S. Busi
          </td>
          <td>2025-08-08</td>
          <td>Frontiers in Systems Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Somatic mutations in KRAS and TP53 are among the most common genetic alterations in pancreatic ductal adenocarcinoma (PDAC). Advances in PCR-based technologies now enable the detection of these mutations in tumor tissue and cell-free DNA (cfDNA), providing a minimally invasive approach to assess tumor burden. However, in resectable PDAC, circulating tumor DNA (ctDNA) may represent less than 0.1% of total cfDNA, requiring highly sensitive detection methods. The aim of our study was to assess two PCR-based assays—competitive allele-specific PCR (castPCR) and digital PCR (dPCR)—for detecting selected somatic mutations in tumor tissue, cfDNA, and extracellular vesicle-associated DNA (EV-DNA) from plasma. Matched primary tumor and preoperative plasma samples were collected from 50 patients undergoing upfront resection for PDAC. CastPCR was used for detecting selected KRAS, TP53, SMAD4, and CDKN2A mutations in tumor DNA. Additionally, dPCR was used to analyze KRAS and TP53 mutations in tumor DNA as well as cfDNA and EV-DNA. The concordance between both platforms was 71.4% for KRAS p.G12D and 58.3% for the analysis of TP53 p.R273H mutations in tumor tissue. However, dPCR detected these mutations in an additional 28.6% and 39.6% of samples, respectively. In cfDNA, dPCR identified KRAS p.G12D in 10.2% and TP53 p.R273H in 2.0% of samples. Mutation detection in EV-DNA was limited by low DNA yield. Both platforms proved effective for tumor DNA analysis, with dPCR offering greater sensitivity. Somatic mutation detection from liquid biopsy using dPCR further supports its potential utility in the preoperative setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee844d3f665f29711ab6ed753d0498189b82feba" target='_blank'>
              Somatic Mutation Detection in Tumor Tissue and Matched Cell-Free DNA Using PCR-Based Methods in Pancreatic Cancer Patients Undergoing Upfront Resection
              </a>
            </td>
          <td>
            Hana Zavrtanik Čarni, D. Badovinac, Tanja Blagus, K. Goričar, Branislava Ranković, Alenka Matjašič, A. Zupan, A. Tomažič, Vita Dolžan
          </td>
          <td>2025-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) often exhibits microvascular invasion (MVI), a feature with unclear mechanisms. Therefore, it is crucial to resolve its related cellular populations and molecular networks using single-cell analysis. Methods Both single-cell RNA sequencing (scRNA-seq) and RNA-seq data for HCC were obtained from public databases. ScRNA-seq data were clustered and annotated using Seurat. DAVID, CellChat, and Monocle 2 were used for scRNA-seq functional enrichment analysis, intercellular communication analysis, and cell trajectory analysis, respectively. We further assessed key gene expression in HCC cell lines and examined their effects on cell functions using CCK-8, scratch, and transwell assays. Results The HCC ecosystem comprising myofibroblasts (MFs), hepatocytes, proliferative hepatocytes, endothelial cells, dendritic cells, proliferative NK/T cells, plasma B cells, and macrophages was revealed. MFs showed the greatest difference between MVI-absent and MVI-present patients and were subdivided into five clusters. Key genes for angiogenesis are overexpressed in MF2 cells and enriched in the pathways of angiogenesis, cell migration, cell proliferation, and signal transduction. Pseudotime analysis revealed MF2 cells from MVI-present patients clustered at the terminal state and positively correlated with angiogenesis. CAMK2N1 in the markers of MF2 cells was significantly associated with advanced M stage and poor prognosis. Further cellular assays showed that CAMK2N1 expression was downregulated in HCC cells, and its knockdown increased the proliferation, migration, and invasion levels of cancer cells. Conclusion This study highlighted the role and potential mechanism of MFs in promoting MVI formation and provides a potential marker for HCC prognosis among MF markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaf4f625c933dabd650b6102c659f0dd38853188" target='_blank'>
              Deciphering the role of myofibroblasts in microvascular invasion of hepatocellular carcinoma
              </a>
            </td>
          <td>
            Xiang Liu, Jiangyang Pan, Lijia Wang, Qiao Xie, Tianbo Fan, Qi Wang
          </td>
          <td>2025-08-29</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genes partaking in DNA damage repair (DDR) pathways exert a key role in the progression of glioblastoma (GBM) and treatment resistance. Essential DDR genes, including XRCC, ERCC, and MGMT, are mutated in GBM, with single nucleotide variants (SNVs) clinically linked to the. However, the structural consequences and biophysical alterations to protein dynamics linked to RCC/MGMT SNVs in GBM are not well-studied. We analyzed GBM-related SNVs in XRCC2, XRCC3, ERCC2, and MGMT. Structural Alphafold models were built. Eight molecular dynamics simulation (MDS) runs for up to 100 ns were carried out via an all-atom forcefield. Structural analyses, such as H-bond, RMSD, RMSF, gyration, and SASA were evaluated. Moreover, protein-protein interactions (PPIs) and mRNA expression levels were assessed by STRING/Cytoscape. The SNVs, XRCC2 (Arg188His), XRCC3 (Thr241Met), ERCC2 (Lys751Gln), and MGMT (Leu84Phe), were associated with notable structural changes. MDS study revealed increased flexibility, compactness, as well as alterations in H-bonding and solvent exposure in mutants of several DDR pathway genes compared to wild-type RCC/MGMT proteins. The modified regional flexibility for all genes was identified by RMSF computation. Pathway analysis showed XRCC3 as the most connected hub within the DDR network. Upregulation of DDR genes, in particular XRCC2/MGMT, correlated with a poorer clinical outlook. A statistically meaningful and direct positive correlation of ERCC2 and MGMT was shown (p < 0.05). The present investigation indicated the significance of GBM-associated DDR gene SNVs in protein dynamics and GBM pathogenesis. This study suggests notable changes in the biophysical structural parameters of mutated ERCC2 and XRCC3, whereas changes in certain parameters of mutant XRCC2/MGMT were less prominent.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebf6c678732893a843d84db9665e563dea77ae7f" target='_blank'>
              Single-nucleotide polymorphisms in DNA repair genes ERCC, XRCC, and MGMT and implications in glioblastoma: a pathway analysis and structural dynamics study.
              </a>
            </td>
          <td>
            Yashmin Afshar, Aida Naseri, Amirmohammad Amoozadehsamakoosh, Alma Naseri, Melika Abrishami, Niloufar Yazdanpanah, Kiarash Saleki, N. Rezaei
          </td>
          <td>2025-07-28</td>
          <td>Journal of biomolecular structure & dynamics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="ABSTRACT Background and Aim Historical genetic sequencing of specific cancer variants has been superseded by comprehensive genomic profiling (CGP). This narrative review aimed to capture current international evidence on the clinical utility of CGP for cancer prevention, detection and treatment. Materials and Methods A literature search of three databases was performed to identify key studies on the frequency of germline and somatic variants in adult cancers and the extent to which they inform diagnosis, management and outcome. Findings were inductively mapped and narratively synthesised. Results Consolidated results from 95 original research papers showed that pathogenic germline (familial) variants are found in ~10% of adults with cancer, of whom 53%–61% are offered germline genotype‐directed treatment. Importantly, 50% of germline carriers would not have satisfied the eligibility criteria for genetic testing and/or reported a negative family history. Actionable somatic variants occur in 27%–88% of cases, which markedly impact the diagnosis for cancers of unknown primary. Matched treatments were identified for 31%–48% of cancer patients, of whom 33%–45% received it. Response and survival rates were better in individuals receiving matched therapies compared to those receiving standard of care or unmatched therapies. Trials show that circulating tumour DNA (ctDNA) assays are feasible and sensitive. The relatively non‐invasive ctDNA sample collection is appealing for cancers with inaccessible or unknown primary sites, and serial monitoring of residual disease and/or treatment response. Conclusions As matched therapies are underutilised due to declining patient condition and fewer prior therapies predicting better response rates, research is needed on the suitability of cancer genomic profiling as a frontline test.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fedb0d01f281e00aec9cacae6c6620746559f77" target='_blank'>
              Utility of Germline, Somatic and ctDNA Testing in Adults With Cancer
              </a>
            </td>
          <td>
            E. DeBortoli, Ella McGahan, Tatiane Yanes, Jennifer Berkman, L. Aoude, A. K. Smit, Akira Gokoolparsadh, Azure Hermes, Lyndsay Newett, Mackenzie Bourke, Susan Hanson, Helen Hughes, Oliver Hofmann, I. Goranitis, Rebekah McWhirter, Vivienne Milch, Julia Steinberg, A. McInerney-Leo
          </td>
          <td>2025-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="BACKGROUND/AIM
Perihilar cholangiocarcinoma (pCCA) is an aggressive bile duct cancer with a poor prognosis (10-40% 5-year survival after resection), that is mostly found in Thailand and has limited prognostic models. Copy neutral loss of heterozygosity (cnLOH) in various cancers affects their progression leading to poorer overall survival. This study aimed to explore cnLOH patterns in individual pCCA patients in order to develop a predictive model for prognostic biomarkers.


MATERIALS AND METHODS
DNA was extracted from 14 frozen tissues and chromosome microarray (CMA) was performed to analyze cnLOH.


RESULTS
Analysis of cnLOH revealed a median of 14 events per sample. The most frequent cnLOH was observed on chromosome 1q, followed by high occurrences on chromosomes 2, 3p, 5q, 6q, 7q, 11q, and 12q. Importantly, we identified several specific cnLOH regions (2p23.2, 2p23.3, and 6p11.1) where alterations were significantly associated with decreased overall survival (p<0.05).


CONCLUSION
These results provide evidence that cnLOH events may contribute to disease progression and influence overall survival in patients with pCCA, indicating a potential role in the underlying molecular mechanisms of this malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f556c6ff78653a53627b1719e756fbd40ecdbda" target='_blank'>
              Discovery of a Copy-neutral Loss of Heterozygosity Pattern in Perihilar Cholangiocarcinoma With Potential Prognostic Value.
              </a>
            </td>
          <td>
            Sutheemon Techa-ay, Sasithorn Watcharadetwittaya, Raksawan Deenonpoe, Kitti Intuyod, A. Techasen, Poramate Klanrit, W. Loilome, P. Sa-Ngiamwibool, Malinee Thanee
          </td>
          <td>2025-08-28</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Genomic antigen loss is a recurring mechanism of resistance to chimeric antigen receptor T-cell (CAR-T) and T-cell engagers (TCE) in relapsed/refractory multiple myeloma (RRMM). Yet, it remains unclear whether these events are acquired under treatment or merely selected from pre-existing, undetectable clones. By leveraging chemotherapy mutational signatures as temporal barcodes within whole genome sequencing data, we could time genomic antigen escape in 4 out of 11 RRMM patients. In all cases, the biallelic loss was driven by genomic events acquired after exposure to BCMA- and GPRC5D-targeted CAR-T/TCE, and not present at baseline. Longitudinal digital PCR analysis corroborated that resistance mutations were undetectable at therapy initiation but emerged preceding relapse. Among 752 newly diagnosed patients only 2.7% and 9% had monoallelic inactivation of TNFRSF17 and GPRC5D, respectively, with no biallelic loss. Our findings suggest limited utility of mutational screening prior to CAR-T/TCE, while underscoring the importance of dynamic surveillance during therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/481837333886a5830de7147acdc462180a9ceabf" target='_blank'>
              Timing genomic antigen loss in multiple myeloma treated with T-cell redirecting immunotherapies.
              </a>
            </td>
          <td>
            M. Papadimitriou, Sungwoo Ahn, B. Diamond, Holly Lee, J. McIntyre, M. Truger, Michael A Durante, B. Ziccheddu, Katalin Osz, O. Landgren, L. Rasche, Nizar J. Bahlis, Paola Neri, F. Maura
          </td>
          <td>2025-09-04</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>34</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [8],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>