<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2025-08-11 09:34:46 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>37</td>
          <td>104</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>77</td>
          <td>138</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, J. Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>114</td>
          <td>145</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) amplification promotes intratumoral genetic heterogeneity and accelerated tumor evolution1–3; however, its frequency and clinical impact are unclear. Using computational analysis of whole-genome sequencing data from 3,212 cancer patients, we show that ecDNA amplification frequently occurs in most cancer types but not in blood or normal tissue. Oncogenes were highly enriched on amplified ecDNA, and the most common recurrent oncogene amplifications arose on ecDNA. EcDNA amplifications resulted in higher levels of oncogene transcription compared to copy number-matched linear DNA, coupled with enhanced chromatin accessibility, and more frequently resulted in transcript fusions. Patients whose cancers carried ecDNA had significantly shorter survival, even when controlled for tissue type, than patients whose cancers were not driven by ecDNA-based oncogene amplification. The results presented here demonstrate that ecDNA-based oncogene amplification is common in cancer, is different from chromosomal amplification and drives poor outcome for patients across many cancer types. A pan-cancer analysis finds that extrachromosomal DNA is pervasive and associated with oncogene amplification and poor patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, J. Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>345</td>
          <td>138</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Focal chromosomal amplification contributes to the initiation of cancer by mediating overexpression of oncogenes1–3, and to the development of cancer therapy resistance by increasing the expression of genes whose action diminishes the efficacy of anti-cancer drugs. Here we used whole-genome sequencing of clonal cell isolates that developed chemotherapeutic resistance to show that chromothripsis is a major driver of circular extrachromosomal DNA (ecDNA) amplification (also known as double minutes) through mechanisms that depend on poly(ADP-ribose) polymerases (PARP) and the catalytic subunit of DNA-dependent protein kinase (DNA-PKcs). Longitudinal analyses revealed that a further increase in drug tolerance is achieved by structural evolution of ecDNAs through additional rounds of chromothripsis. In situ Hi-C sequencing showed that ecDNAs preferentially tether near chromosome ends, where they re-integrate when DNA damage is present. Intrachromosomal amplifications that formed initially under low-level drug selection underwent continuing breakage–fusion–bridge cycles, generating amplicons more than 100 megabases in length that became trapped within interphase bridges and then shattered, thereby producing micronuclei whose encapsulated ecDNAs are substrates for chromothripsis. We identified similar genome rearrangement profiles linked to localized gene amplification in human cancers with acquired drug resistance or oncogene amplifications. We propose that chromothripsis is a primary mechanism that accelerates genomic DNA rearrangement and amplification into ecDNA and enables rapid acquisition of tolerance to altered growth conditions. Chromothripsis—a process during which chromosomes are ‘shattered’—drives the evolution of gene amplification and subsequent drug resistance in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>303</td>
          <td>159</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) amplification enhances intercellular oncogene dosage variability and accelerates tumor evolution by violating foundational principles of genetic inheritance through its asymmetric mitotic segregation. Spotlighting high-risk neuroblastoma, we demonstrate how ecDNA amplification undermines the clinical efficacy of current therapies in cancers with extrachromosomal MYCN amplification. Integrating theoretical models of oncogene copy number-dependent fitness with single-cell ecDNA quantification and phenotype analyses, we reveal that ecDNA copy-number heterogeneity drives phenotypic diversity and determines treatment sensitivity through mechanisms unattainable by chromosomal oncogene amplification. We demonstrate that ecDNA copy number directly influences cell fate decisions in cancer cell lines, patient-derived xenografts, and primary neuroblastomas, illustrating how extrachromosomal oncogene dosage-driven phenotypic diversity offers a strong evolutionary advantage under therapeutic pressure. Furthermore, we identify senescent cells with reduced ecDNA copy numbers as a source of treatment resistance in neuroblastomas and outline a strategy for their targeted elimination to improve the treatment of MYCN-amplified cancers.


SIGNIFICANCE
ecDNA-driven tumor genome evolution provides a major challenge to curative cancer therapies. We demonstrate that ecDNA copy-number dynamics drives treatment resistance by promoting oncogene dosage-dependent phenotypic heterogeneity in MYCN-amplified cancers. Exploiting phenotype-specific vulnerabilities of ecDNA cells, therefore, presents a powerful strategy to overcome treatment resistance. See related article by Korsah, p. XX.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23a317a5c4269a3d4d0174cf78851658866c2ba8" target='_blank'>
              Extrachromosomal DNA-Driven Oncogene Dosage Heterogeneity Promotes Rapid Adaptation to Therapy in MYCN-Amplified Cancers.
              </a>
            </td>
          <td>
            Giulia Montuori, Fengyu Tu, Di Qin, R. Schmargon, Elias Rodriguez-Fos, K. Helmsauer, Hui Hui, Susmita Mandal, Karin Purshouse, Lara Fankhänel, Bartolomeo Bosco, B. Spanjaard, H. Seyboldt, Laura Grunewald, M. Schmitt, D. Gürgen, Viktoria Buck, M. Rosenfeldt, Frank P B Dubois, S. Schallenberg, Annika Lehmann, J. Theißen, S. Taschner-Mandl, Arend Koch, P. Hundsdoerfer, A. Künkele, Angelika Eggert, Matthias Fischer, G. Gargiulo, Teresa G Krieger, Lukas Chavez, Fabian Coscia, Benjamin Werner, Weini Huang, A. Henssen, J. Dörr
          </td>
          <td>2025-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Circular extrachromosomal DNA (ecDNA) is a common form of oncogene amplification in aggressive cancers. The frequency and diversity of ecDNA has been catalogued in adult and some childhood cancers; however, its role in most pediatric cancers is not well-understood. To address this gap, we accessed large pediatric cancer genomics data repositories and identified ecDNA algorithmically using cloud computing. This retrospective cohort comprises 2,968 patients covering all major childhood solid tumor types. Aggressive tumor types had particularly high incidences of ecDNA. Pediatric patients whose tumors harbored extrachromosomal DNA had poorer five-year overall survival than children whose tumors contain only chromosomal amplifications. We catalogue known and potentially novel oncogenes recurrently amplified on ecDNA and show that ecDNA often evolves during disease progression. These results highlight patient populations that could potentially benefit from future ecDNA-directed therapies. To facilitate discovery, an interactive catalogue of ecDNA in childhood cancer is available at https://ccdi-ecdna.org/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8b9576b9fdef51f927edfe8dbc69e8e7f1f4c7f" target='_blank'>
              Extrachromosomal DNA associates with poor survival across a broad spectrum of childhood solid tumors
              </a>
            </td>
          <td>
            Owen S. Chapman, S. Sridhar, Eugene Yui-Ching Chow, Rishaan Kenkre, Jonathan Kirkland, Aditi Dutta, Shanqing Wang, Wenshu Zhang, Miguel Brown, J. Luebeck, Yan Yuen Lo, Elias Rodriguez-Fos, A. Henssen, K. Okonechnikov, David R. Ghasemi, K. Pajtler, Daisuke Kawauchi, V. Bafna, Megan Paul, Kevin Yip, Jill P. Mesirov, Lukas Chavez
          </td>
          <td>2025-07-23</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Abstract Extrachromosomal DNA (ecDNA) is a critical driver of cancer progression, contributing to tumour growth, evolution, and therapeutic resistance through oncogene amplification. Despite its significance, the replication of ecDNA remains poorly understood. In this study, we investigated the replication dynamics of ecDNA using high-resolution replication timing analysis (Repli-seq) and DNAscent, a method for measuring origin firing and replication fork movement, that we applied to both bulk DNA and to ecDNA isolated with FINE (Fluorescence-activated cell sorting-based Isolation of Native ecDNA), a new method for isolating, chromatinized ecDNA without DNA or protein digestion. We demonstrate that ecDNA in the COLO 320DM colorectal cancer cell line exhibits largely asynchronous replication throughout the S phase, contrasting with the conserved replication timing of the corresponding chromosomal DNA in RPE-1 cells and the chromosomally reintegrated ecDNA in COLO 320HSR. Replication origins on ecDNA are redistributed, and replication forks exhibit reduced velocity and increased stalling. Under replication stress induced by hydroxyurea treatment, ecDNA replication is further compromised, leading to altered origin activation, reduced fork velocity and eventual ecDNA depletion from cells. Our findings reveal fundamental differences in the replication dynamics of ecDNA, providing insights that could inform the development of therapies targeting ecDNA-associated oncogene amplification in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec71fc330fef12833948b0057870ba85688ca5b1" target='_blank'>
              ecDNA replication is disorganized and vulnerable to replication stress
              </a>
            </td>
          <td>
            Jedrzej J Jaworski, Pauline L Pfuderer, Pawel Czyz, Gianluca Petris, M. Boemo, Julian E. Sale
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Oncogene amplification is a key driver of cancer pathogenesis and is often mediated by extrachromosomal DNA (ecDNA). EcDNA amplifications are associated with increased pathogenicity of cancer and poorer outcomes for patients. EcDNA can be detected accurately using fluorescence in situ hybridization (FISH) when cells are arrested in metaphase. However, the majority of cancer cells are non-mitotic and must be analyzed in interphase, where it is difficult to discern extrachromosomal amplifications from chromosomal amplifications. Thus, there is a need for methods that accurately predict oncogene amplification status from interphase cells. Here, we present interSeg, a deep learning-based tool to cytogenetically determine the amplification status as EC-amp, HSR-amp, or not amplified from interphase FISH images. We trained and validated interSeg on 652 images (40,446 nuclei). Tests on 215 cultured cell and tissue model images (9,733 nuclei) showed 89% and 97% accuracy at the nuclear and sample levels, respectively. The neuroblastoma patient tissue hold-out set (67 samples and 1,937 nuclei) also revealed 97% accuracy at the sample level in detecting the presence of focal amplification. In experimentally and computationally mixed images, interSeg accurately predicted the level of heterogeneity. The results showcase interSeg as an important method for analyzing oncogene amplifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a539ca1a35b8049ad46a136d79392bb325111564" target='_blank'>
              Accurate Prediction of ecDNA in Interphase Cancer Cells using Deep Neural Networks
              </a>
            </td>
          <td>
            Utkrisht Rajkumar, Gino Prasad, Ellis J Curtis, I. Wong, Xiaowei Yan, Shu Zhang, Lotte Brückner, Kristen Turner, Julie Wiese, Justin Wahl, Sihan Wu, J. Theißen, Matthias Fischer, Howard Y. Chang, A. Henssen, P. Mischel, V. Bafna
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6865d140b6544d7dfc0dca54fb01a54fdf212f1a" target='_blank'>
              Mitotic BLM functions are required to maintain genomic stability
              </a>
            </td>
          <td>
            Tamara Eleanore Hamann, A. Wieland, Andrea Tirincsi, Kruno Vukušić, Farbod Mohseni, René Wardenaar, Marialucrezia Losito, Ipek Ilgin Gönenc, B. Wollnik, F. Foijer, I. Tolic, Zuzana Storchová, M. Räschle
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Abstract Large chromosomal copy number gains are ubiquitous throughout cancer types. However, which genes drive their selective advantage is not well established, and therefore they are hardly utilized in clinical practice. Our analysis of copy number patterns in pan-cancer datasets suggests that the selective advantage of copy number gains is largely driven by known oncogenes. Analysis of CRISPR screening data identifies a list of 101 genes that are likely to mediate the effect of these gains, which is highly enriched in annotated oncogenes but also contains genes that have not been implicated in cancer so far. Moreover, we show that specific gains are associated with drug sensitivity or resistance, with a strong enrichment of gains of oncogenes with increased sensitivity to inhibitors targeting these specific genes. Finally, we provide examples where gains can function as relevant clinical biomarkers for diagnosis and treatment. Thus, large copy number gains exert their selective advantage through known and novel oncogenes, and their systematic analysis could advance precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecd5fd07460798ed6e6a37e20560052b55679a6c" target='_blank'>
              Low-amplitude copy number gains shape cancer through known and novel oncogenes with associated therapeutic vulnerabilities
              </a>
            </td>
          <td>
            T. F. Eleveld, B. Ylstra, Leendert H J Looijenga
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ABSTRACT Aims & methods DNA methylation and genomic instability are critical drivers of cancer initiation and malignant progression. However, the roles of methylation aberrations and genomic instability in malignant progression have not been thoroughly investigated in intrahepatic cholangiocarcinoma (ICC). To address this, we identified differentially methylated regions (DMRs) and somatic copy number alterations (SCNAs) from 341 ICC samples across various stages. Results Our findings revealed that stages IAIB, II, IIIA, and IIIB exhibited comparable methylation changes, whereas stage IV ICC showed a pronounced accumulation of stage-specific methylation alterations. Leveraging these findings, we developed a classification model that effectively distinguished stage IV ICC from earlier stages with high accuracy using 15 DMRs. Furthermore, stage IV ICC exhibited slightly higher genomic instability, including an elevated aneuploidy score and a greater proportion of focal amplifications. We also observed a positive correlation between SCNA burden and DNA methylation entropy in the promoter, gene body, and CpG island regions, with the gene body of MDM2 serving as a notable example. Conclusions These findings highlight the potential of DNA methylation as a biomarker for metastasis diagnosis and the interplay between local genomic instability and aberrant methylation, emphasizing their synergistic roles in driving the evolutionary trajectory of ICC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d995c84a0cfeeb8c871fd5bd4ce5a040e83eca26" target='_blank'>
              Methylation aberrations and genomic instability synergistically drive the evolution of intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Guanghao Li, Youhuang Bai, Feng Tao, Tingting Hu, Ting Wang, Yong Zeng, Deqiang Sun
          </td>
          <td>2025-06-17</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) of chromosomal origin is commonly present in all eukaryotic organisms and tissue tested so far. EccDNA populations exhibit immense diversity and a characteristically low degree of overlap between samples, suggesting low inherence of eccDNA between cells or a deficiency the methods by which eccDNA is detected. This study revisits the Circle-seq approach for enrichment of eccDNA to address if these limitations, hypothesizing that experimental procedures significantly contribute to the observed low eccDNA overlap. We optimized the protocol by reducing the time. Linear DNA is digested by increasing exonuclease V activity. We employed CRISPR-Cas9 for mitochondrial linearization, which proved superior to restriction enzymes. A key finding is the critical role of random hexamer primer concentration and genomic DNA input in Rolling Circle Amplification (RCA) for generating high-quality long amplicons from eccDNA (concatemeric tandem copy, CTC), essential for confident de novo eccDNA construction from long-read sequencing data. Lower primer concentrations substantially increased the percentage of CTC-derived eccDNA and improved the overlap of identified eccDNAs in technical replicates. Applying this revisited approach to human myeloma and breast cancer cell lines, as well as xenograft models, demonstrated that the optimized conditions enhanced the overlap of detected eccDNA up to over 50% overlap which substantially improved over previous studies (less than 1%). Moreover, Additionally, the oncogenic signature of eccDNAs can be identified across all replicates. These findings provide guidelines for developing standardized procedures for eccDNA profiling, advancing our understanding of eccDNA biology and its potential clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/842e0482ffc6f8bbb0e4561726b21f49e5e47e0b" target='_blank'>
              An Enhancement of Extrachromosomal Circular DNA Enrichment and Amplification to Address the Extremely Low Overlap Between Replicates
              </a>
            </td>
          <td>
            Charles M. Burnham, Alongkorn Kurilung, Visanu Wanchai, Birgritte Regenberg, J. Delgado-Calle, Alexei G. Basnakian, I. Nookaew
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Increases in the copy number of large genomic regions, termed amplifications, are an important adaptive strategy for many organisms. Numerous amplifications across the AT-rich Plasmodium falciparum genome contribute directly to drug resistance or impact the fitness of this protozoan parasite. During the characterization of malaria parasites selected with a dihydroorotate dehydrogenase (DHODH) inhibitor that targets pyrimidine biosynthesis, we detected increased copies of a genomic region that encompassed 3 genes (~ 5 kb) including GTP cyclohydrolase I (GCH1 amplicon). While amplification of this gene is reported in antifolate-resistant parasites, GCH1 amplicons had not previously been implicated in DHODH inhibitor resistance. Results Here, we explored the expansion of the GCH1 locus in this family of parasite lines using long-read sequencing and single-read visualization. We directly quantified higher numbers of tandem GCH1 amplicons in selected parasite lines (up to 9 GCH1 amplicons) compared to parental P. falciparum parasites (strictly 3 GCH1 amplicons). Because each read represents DNA from an individual genome, we were able to appreciate hidden variation within a single parasite line (3, to 5, to 7 amplicons) that was not reflected in other DNA-based analysis methods. While all GCH1 amplicons shared a consistent structure, expansions arose in precise 2-unit steps within selected lines. We found conserved AT-rich sequences at amplicon boundaries, which is consistent with the Plasmodium model of CNV formation. Parasite lines with expanded GCH1 also had DHODH amplicons on a separate chromosome. When we evaluated prior DHODH inhibitor selections, we observed that GCH1 amplification was not required for resistance; however, selection outcomes suggest that pre-existing GCH1 amplicons may support amplification at the DHODH locus. Conclusions We identified previously undetected heterogeneity in gene copy number by viewing long pieces of DNA from individual genomes. This approach was possible due to the amplicon’s tandem orientation and relatively small size that can be spanned by a single long ONT read. The positive association between DHODH and GCH1 copy number, combined with the metabolic connection between P. falciparum pyrimidine and folate biosynthesis, justifies further investigation into the adaptive evolution of these two genomic loci. Supplementary Information The online version contains supplementary material available at 10.1186/s12864-025-11859-5.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df91538345e9b090469612002190dc6883763df6" target='_blank'>
              Direct long-read visualization reveals hidden variation in GCH1 gene copy number and precise expansion steps
              </a>
            </td>
          <td>
            Shiwei Liu, Julia Zulawinska, Emily R. Ebel, A. Luniewski, Charles Danis, Mary Lewis Simpson, Jane Kim, Nnenna Ene, Thomas Braukmann, Molly Congdon, Webster Santos, Ellen Yeh, J. Guler
          </td>
          <td>2025-07-17</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10436da9e9bcdf67b6e4cfcf17b6e58df4672a34" target='_blank'>
              Spatio-temporal chromosomal arrangements by late-replicating heterochromatin
              </a>
            </td>
          <td>
            Satyadev Polisetty, Shuvadip Dutta, Rakesh Netha Vadnala, D. Notani, R. Padinhateeri, K. Sanyal
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="


 In tumors, clonal populations of malignant cells evolve a diverse repertoire of copy number variants (CNVs), often resulting in profound intra-tumor heterogeneity (ITH). ITH is associated with poor prognosis, cancer progression, increased risk of therapy resistance, metastasis and relapse. Traditionally, researchers have studied clonal heterogeneity and tumor evolution using bulk-tissue assays, however, these techniques provide little or no spatial information and have low sensitivity for focal/low copy-number CNVs, particularly when they are subclonal or rare. DNA FISH is the gold standard for detecting CNVs at cellular resolution, but it is limited to one gene per tissue section. To overcome the above limitations, we developed multiplexed-CNV FISH, a spatial omics assay that detects gain and loss of DNA loci at single-gene (∼10kb) and single-cell resolution in human tissues. Remarkably, CNV-FISH analysis of CRC tumor sections identified small and subclonal CNVs not detected by existing methods with resolution of CNV spanning whole chromosome arms. Our results indicate that CNV-FISH can reveal unexpected and previously undetected CN heterogeneity within tumors, which could potentially underlie treatment resistance and eventual metastasis of cancer cells.



 Mei Suen KONG, Hui Ting Grace YEO, Jiamin TOH, Lin Li, Jonathan Shaozhong Aow, Joanito Irwan IGNASIUS, Jeeranan BOONRUANGKAN, Jen Yi WONG, Nirmala ARUL RAYAN, Lavanya MUTHUKRISHNAN, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon LIM, Bee Huat Iain TAN, Kok Hao CHEN, Shyam PRABHAKAR. Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P06.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae7336ba20d4eb282822f6335d4e3b64784f2e7c" target='_blank'>
              Abstract P06: Cellular-resolution Multiplexed-CNV FISH Captures Spatial Copy Number Heterogeneity and Clonal Architecture of Colorectal Cancer
              </a>
            </td>
          <td>
            Mei Suen KONG, Hui Ting Grace YEO, Jiamin Toh, Lin Li, Jonathan Aow, Joanito Irwan IGNASIUS, Jeeranan Boonruangkan, Jen Yi WONG, N. A. Rayan, Lavanya Muthukrishnan, Dominique Camat MACALINAO, Wei Qiang LEOW, Kiat Hon Lim, Bee Huat Iain TAN, Kok Hao CHEN, Shyam Prabhakar
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2a0c8aae8bd565763175795d01ffdf375af883" target='_blank'>
              Ongoing genome doubling shapes evolvability and immunity in ovarian cancer.
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Duaa H Al-Rawi, Matthew Zatzman, A. Weiner, Samuel Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nicholas Ceglia, Danguolė Norkūnaitė, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, Ginger J Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, R. Grisham, Ying L Liu, A. Viale, Nicole Rusk, Yulia Lakhman, Lora H. Ellenson, S. Tavaré, Sam Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2025-07-16</td>
          <td>Nature</td>
          <td>0</td>
          <td>93</td>
        </tr>

        <tr id="Chromosome-derived circular DNAs have different characters from linear DNAs and RNAs, which exert distinctive functions, and participate in a widespread range of physiological and pathological processes. Although, the composition, structure and genome-wide frequency of circular DNAs have not yet been illuminated systemically, circular DNAs are identified to act as unanticipated major sources of somatic rearrangements, which is an important genomic feature in tumorigenesis and represents a multihit and ongoing mutagenic process. And, this process remarkably contributes to oncogenic remodeling and oncogenes amplification through chimeric circularization and reintegration of circular DNAs into the linear genome, which is closely associated with high cancer mortality and morbidity. It has been found that circular DNAs-stimulated oncogenes amplification and overexpression implicate in diverse cytopathological processes during cancer progression, including cell proliferation, apoptosis, autophagy, epithelial-mesenchymal transition (EMT), extracellular matrix (ECM) stiffness and angiogenesis. Herein, we comprehensively introduced circular DNAs’ formation and characteristics, and emphatically elaborated circular DNAs-induced oncogenes amplification and overexpression and their functional mechanisms regulating cancer behaviors. Lastly, some mysteries in oncogenic circular DNAs’ study were introduced, and the perspectives were also discussed in cancer diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/825b11b400d7213811736a403636f24a5a529f78" target='_blank'>
              Circular DNAs Participate in Tumorigenesis and Its Mechanisms
              </a>
            </td>
          <td>
            Yuan Tan, Faqing Tang
          </td>
          <td>2025-07-30</td>
          <td>Clinical Medicine and Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9eeecebde718f44128a5af1add184f4e02b1d49" target='_blank'>
              NuMA promotes constitutive heterochromatin compaction by stabilizing linker histone H1 on chromatin
              </a>
            </td>
          <td>
            Yao Wang, Wenxue Zhao, Jiahao Niu, Cuifang Liu, Xiaotian Wang, Weihong Yuan, Shanshan Ai, Wolfgang Baumeister, Guohong Li, Aibin He, Peng Xu, Cheng Li, Yujie Sun
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) remains a lethal malignancy with poor prognosis. We investigated intratumoral deoxyribonucleic acid methylation heterogeneity by analyzing 44 tumor samples and 5 normal samples from 6 cases of PDAC by using high-resolution methylation arrays. Two distinct methylation profiles were identified: T1, which is similar to normal pancreatic tissue and is associated with well-differentiated histology, and T2, which is significantly different from normal tissue and is linked to poorly differentiated morphology and squamous features. Validation using The Cancer Genome Atlas (TCGA) confirmed these profiles and revealed the association of T2 with shorter disease-free survival (p = 0.04). Differentially methylated region analysis identified the substantial hypomethylation of transcription regulation genes in T2 profiles (false discovery rate [FDR] q < 0.001). Gene set enrichment analysis with TCGA gene expression data demonstrated the upregulation of DNA repair and MYC target genes in T2 samples (FDR q < 0.001). Phylogenetic analysis with our multi-sampling dataset suggested an evolutionary trajectory from T1 to T2 profiles coinciding with aggressive phenotypes and increased genomic instability. Cases exhibited varying degrees of intratumoral heterogeneity from distinctly separated clusters to minimal differences. This comprehensive characterization of the epigenetic landscape of PDAC provides insights into tumor evolution and heterogeneity with potential implications for patient stratification and the development of epigenetic-based diagnostic and therapeutic strategies. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-13919-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd12eab03975f1f1da59d2699d262a5bc0933404" target='_blank'>
              Epigenetic evolution and clinicopathological implications of distinct DNA methylation profiles in pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            K. Kitahama, Yu-Jui Ho, Kaishi Satomi, Takahiro Shibayama, Kiyotaka Nagahama, K. Ohtsuka, Hiroaki Ohnishi, Yoshihiro Sakamoto, Junji Shibahara, Akimasa Hayashi
          </td>
          <td>2025-08-06</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Introduction Balanced chromosomal abnormalities (BCAs) are structural variations that can underlie a wide spectrum of neurodevelopmental disorders, often remaining undetected by conventional diagnostic approaches. Whole-genome sequencing (WGS) allows for base-pair resolution of structural variants across the entire genome, making it a powerful tool to detect cryptic chromosomal rearrangements and refine breakpoint mapping. RNA sequencing (RNA-Seq), by enabling the detection of gene expression changes and fusion transcripts, provides complementary functional insights into the consequences of genomic alterations. This study integrated WGS and RNA-Seq to precisely characterize the breakpoints and assess the functional impact of de novo BCAs in two unsolved cases of Neurodevelopmental Disorders. Materials and methods Short read WGS was used to identify the chromosomal breakpoints and gene disruptions caused by BCAs. RNA-Seq on blood RNA was employed to detect differential gene expression and potential fusion transcripts of disrupted genes. Results In the first case, the inversion inv(8) (p11.2q13) disrupted two genes at the breakpoints, namely, CHD7 and SLC20A2. These genes are in opposite orientations, and the inversion realigned them in the same direction, generating two novel fusion genes. Disruption of CHD7 confirmed the suspected diagnosis of CHARGE syndrome. The interruption of SLC20A2, commonly associated with neurological symptoms, prompted further clinical evaluation. RNA-Seq identified in-frame fusion transcripts from the chimeric genes in the blood, suggesting a potential deleterious phenotypic effect. In the second case, WGS revealed a balanced translocation t(17; 22) (q25; q13) that disrupted EP300 at 22q25, confirming Rubinstein-Taybi syndrome. The concurrent disruption of RBFOX3 at 17q13 suggested additional neurological implications, particularly related to epilepsy. Transcriptomic analysis demonstrated the monoallelic and significantly reduced expression of EP300. Conclusion These findings highlight the crucial role of WGS in identifying disease-associated BCAs and underscore the complementary value of RNA-Seq in assessing their functional consequences. This integrated approach enhanced diagnostic accuracy and clinical management, paving the way for more comprehensive and personalized care in these two patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d8f54f85ebf927d3f8b7d5f4a6236e7f717fa51" target='_blank'>
              Integrating genome and transcriptome analysis to decipher balanced structural variants in unsolved cases of neurodevelopmental disorders
              </a>
            </td>
          <td>
            S. Mellone, Alice Spano, Denise Vurchio, Giulia Borgonovi, Alessandro Ugonotti, Giulia Paglino, Alba Bianco, Sara Ronzani, Maurizio Sciancalepore, Flavia Prodam, Amanda Papa, Maurizio Viri, Umberto Dianzani, Mara Giordano
          </td>
          <td>2025-07-07</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Circulating tumor cells (CTCs) are linked to cancer progression and poor prognosis, offering valuable genetic insights into tumors. Accurate detection of genomic alterations in CTCs is essential for improving cancer diagnosis and treatment. To address this, we develop Uniform Chromosome Conformation Capture (Uni-C), a method for profiling 3D chromatin architecture and genomic alterations at the single-cell level. Using Uni-C, we analyze CTCs from pancreatic cancer patient-derived xenograft (PDX) and spontaneous tumor mouse models. In the PDX model, integrating data from seven CTCs captures 88.7% of SNPs and INDELs, and 75.0% of structural variants present in tumor tissue. These findings indicate that variants detected in CTCs reflect tumor genomic features. Notably, we observe chromatin conformation differences between mitotic and interphase CTCs, suggesting potential markers of cell vitality. In the spontaneous tumor model, we identify driver gene mutations in CTCs and predict neoantigens, advancing early cancer detection and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcdb93a8b0e33fd2f466feefe41fc91b323a6623" target='_blank'>
              Identification of multiple genomic alterations and prediction of neoantigens from circulating tumor cells at the single-cell level
              </a>
            </td>
          <td>
            Xiaochen Gao, Xinyu Li, Weize Xu, Ming Jiao, Yu Guo, Jiajia Wang, Weihao Wang, Jiling Feng, Qianqian Guo, Chengchao Wu, Taiyu Zhang, Yuqin Yang, Da Lin
          </td>
          <td>2025-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Objectives: To investigate the clinical diagnostic and prognostic value of extrachromosomal circular DNA (eccDNA) in breast cancer, eccDNA profiles were constructed for 81 breast cancer tumor tissues and 33 adjacent non-tumor tissues. Methods: The distribution characteristics of eccDNA across functional genomic elements and repetitive sequences were systematically analyzed. Furthermore, a diagnostic model for differentiating malignant and normal breast tissues, as well as a prognostic prediction model, was developed using a random forest algorithm. Results: EccDNA in breast cancer tissues harbor a higher proportion of functional elements and repetitive sequences, with their annotated genes significantly enriched in tumor- and immune-related pathways. However, no significant differences in eccDNA features were observed across breast cancer subtypes or pathological stages. In the validation cohort, the eccDNA-based diagnostic model achieved an AUC of 0.83, with repetitive elements and enhancer-associated features contributing the most to diagnostic performance. The prognostic model achieved an AUC of 0.78, with repetitive element annotations also showing strong prognostic relevance. Conclusions: These findings highlight the promising potential of eccDNA in the development of precision diagnostics and prognostic systems for breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f033fc855abaceeac39bacf8c5fcd1d56a705c9" target='_blank'>
              A Study on the Diagnostic and Prognostic Value of Extrachromosomal Circular DNA in Breast Cancer
              </a>
            </td>
          <td>
            Fuyu Li, Wenxiang Lu, Lingsong Yao, Yunfei Bai
          </td>
          <td>2025-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="DNA damage from routine cellular processes or exogenous insults can have a lasting impact on gene regulation beyond genetic mutations. The prevailing paradigm for the consequences of DNA damage repair revolves around restoration of the original genetic sequence, but long-term changes in chromatin configuration, gene expression and DNA modifications have not been analyzed. We introduced numerous, simultaneous Cas9-mediated DNA double strand breaks (DSBs) at defined locations in human glioblastoma cells and tracked both non-genetic and genetic alterations over time. Megabase-scale genomic alterations that endured two weeks after the initial damage were detected, involving a shift from transiently increased intra-TAD interactions to persistent long range cis and trans contacts, alterations in gene-expression and associated large structural variations. These findings reveal that widespread DNA damage, such as chemotherapy or radiotherapy, can trigger long-term genetic and non-genetic modifications which alter cellular function and may impact tumor outcome and the emergence of resistant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23f7337991d50d13c4c6add597a9dcc86990c583" target='_blank'>
              Persistent chromatin alterations and gene expression reprogramming follow widespread DNA damage in glioblastoma
              </a>
            </td>
          <td>
            A. Modrek, Ken Chandradoss, Catherine Do, R. Ezhilarasan, Theodore Sakellaropoulos, Jerome Karp, Yingwen Ding, Ze-Yan Zhang, Melanie Graciani, Giulia Cova, Jennifer E. Phillips-Cremins, Erik P. Sulman, Jane A. Skok
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Tumor evolution is one of the major mechanisms responsible for acquiring therapy-resistant and more aggressive cancer clones. Whether the tumor microenvironment through immune-mediated mechanisms might promote the development of more aggressive cancer types is crucial for the identification of additional therapeutic opportunities. Here, we identify a subset of tumor-associated neutrophils, defined as tumor-associated neutrophil precursors (PreNeu). These PreNeu are enriched in highly proliferative hormone-dependent breast cancers and impair DNA repair capacity. Mechanistically, succinate secreted by tumor-associated PreNeu inhibits homologous recombination, promoting error-prone DNA repair through non-homologous end-joining regulated by PARP-1. Consequently, breast cancer cells acquire genomic instability promoting tumor editing and progression. Selective inhibition of these pathways induces increased tumor cell killing in vitro and in vivo. Tumor-associated PreNeu score correlates with copy number alterations in highly proliferative hormone-dependent tumors from breast cancer patients. Treatment with PARP-1 inhibitors counteract the pro-tumoral effect of these neutrophils and synergize with endocrine therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05cc16bf8fdaf7c258d80a42bf69d6348e6e3dfa" target='_blank'>
              Tumor-associated neutrophil precursors impair homologous DNA repair and promote sensitivity to PARP inhibition
              </a>
            </td>
          <td>
            Siddhartha Mukherjee, Cindy Garda, Letizia Boffa, Angela Rita Elia, Matteo Massara, Maria Teresa Balia, Daniela Brina, Simone Mosole, Anna Campagnari, G. Cassanmagnago, Andrea Rinaldi, Giacomo Lazzaroni, D. Jarrossay, Diego Morone, Ilaria Ceppi, Riccardo De Sillo, Isabella Giacomini, Ilaria Craparotta, L. Di Rito, S. Barry, Endre Laczko, Sebastian Streb, F. Meani, Simona di Lascio, Nancy Hynes, E. Lugli, Simone Puccio, Stephen-John Sammut, Ulrike Perriard, Yves Harder, Lorenzo Rossi, M. Gasparri, Marco Bolis, Petr Cejka, Arianna Calcinotto
          </td>
          <td>2025-07-30</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Mammalian genomes are subdivided into euchromatic A compartments that contain mostly active chromatin, and inactive, heterochromatic B compartments. However, it is not well understood how A and B genome compartments are established and maintained. Here we study SMCHD1, an SMC-like protein best known for its role in X chromosome inactivation, in human male myoblasts. SMCHD1 colocalizes with Lamin B1 and the heterochromatin mark H3K9me3. Loss of SMCHD1 leads to extensive heterochromatin and Lamin B1 depletion at the nuclear lamina, acquisition of active chromatin states and increased DNA methylation along chromosomes. In absence of SMCHD1, long range intra-chromosomal contacts between B compartments are lost while many new TADs and loops are formed. Inactivation of SMCHD1 promotes numerous B to A compartment transitions accompanied by activation of silenced genes. The data suggests that SMCHD1 functions as an anchor for heterochromatin domains at the nuclear lamina ensuring that these domains are poorly accessible to DNA methyltransferases and to epigenome modification enzymes that typically operate in active chromatin. Thus, the properties of SMCHD1 in heterochromatin maintenance extend well beyond its role in X chromosome inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80d576a82dc3af4516675e7e1ac94325c2cc7c05" target='_blank'>
              SMCHD1 maintains heterochromatin, genome compartments and epigenome landscape in human myoblasts
              </a>
            </td>
          <td>
            Zhijun Huang, Wei Cui, Ishara Ratnayake, Kristin Gallik, Lorna Cohen, Rabi Tawil, Gerd P Pfeifer
          </td>
          <td>2025-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Copy number variants (CNVs) are duplications and deletions of DNA sequences that contribute to genetic variation between individuals and underlie rapid adaptive evolution. Increases in gene copy number can confer a strong selective advantage in some circumstances, but CNVs can also incur fitness costs. Prior research has shown that experimental evolution of Saccharomyces cerevisiae in nutrient-limited chemostats recurrently selects for amplifications of nutrient transporter genes, including GAP1, MEP2, and PUT4 in glutamine, ammonium and proline limited conditions, respectively. However, the fate of these CNVs upon return of the organism to a non-selective environment, is unknown. To investigate the fitness costs and stability of CNVs upon reversal of the original selection pressure, we studied 15 unique CNV lineages that had been selected in different nitrogen-limited chemostats, including both segmental amplifications and whole-chromosome aneuploidies. We studied the stability of CNVs using a fluorescent CNV reporter system during propagation in nutrient-rich media using serial dilution of batch cultures for 110-220 generations. We found that only one-third of CNV lineages repeatedly underwent loss of CNVs and reversion to a single-copy genotype. All lineages containing aneuploidies showed rapid reversion dynamics, whereas lineages containing segmental amplifications were remarkably stable – only one of 11 strains reverted to a single copy. Pairwise competitive fitness assays revealed strong fitness defects associated with reverting CNVs in the nutrient-rich environment but minimal fitness defects in lineages that did not revert. Reversion of CNVs led to increased fitness. Using simulation-based inference to estimate reversion rates and fitness effects we find that negative selection is the primary driver of CNV loss although aneuploid reversion rates are high with estimates of 10 −5 to 10 −3 . Whole-genome sequencing of clones isolated from evolved populations revealed that reversion of both aneuploids and a segmental amplification left no evidence of prior existence of the CNV, making the genomes of revertants indistinguishable from the single-copy ancestor. Our findings provide novel evidence of the low fitness cost and high stability of most segmental CNVs upon removal of the selection pressure, and that gene amplifications with large fitness costs are readily reversible, highlighting the significance of CNVs for genome evolution as well as rapid and reversible adaptation to transient selection pressures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/478ede766edac8c6526ad5e7502f78feea49ddaf" target='_blank'>
              Segmental copy number amplifications are stable in the absence of selection
              </a>
            </td>
          <td>
            Titir De, Nadav Ben Nun, P. Spealman, Ina Suresh, Grace Avecilla, Farah Abdul-Rahman, Yoav Ram, David Gresham
          </td>
          <td>2025-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The DNA-incorporating nucleoside analogs azacytidine (AZA) and decitabine (DEC) have clinical efficacy in blood cancers, yet the precise mechanism by which these agents kill cancer cells has remained unresolved -- specifically, whether their anti-tumor activity arises from conventional DNA damage or DNA hypomethylation via DNA methyltransferase 1 (DNMT1) inhibition. This incomplete mechanistic understanding has limited their broader therapeutic application, particularly in solid tumors, where early clinical trials showed limited efficacy. Here, through the assessment of drug sensitivity in over 600 human cancer models and comparison to a non-DNA-damaging DNMT1 inhibitor (GSK-3685032), we establish DNA hypomethylation, rather than DNA damage, as the primary killing mechanism of AZA and DEC across diverse cancer types. In further support of an epigenetic killing mechanism, CRISPR drug modifier screens identified a core set of chromatin regulators, most notably the histone deubiquitinase USP48, as AZA and DEC protective factors. We show that USP48 is recruited to newly hypomethylated CpG islands and deubiquitinates non-canonical histones, establishing USP48 as a key molecular link between the two components of epigenetic gene regulation: DNA methylation and chromatin modification. Furthermore, loss of USP48, which occurs naturally through biallelic deletions in human cancers, sensitized both hematologic and solid tumors to DNMT1 inhibition in vitro and in vivo. Our findings elucidate the epigenetic mechanism of action of AZA and DEC and identify a homeostatic link between DNA methylation and chromatin state, revealing new therapeutic opportunities for DNMT1 inhibitors in solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a981e247fc2d88e8e208dd64bc975ec0543e96de" target='_blank'>
              Discovery of chromatin-based determinants of azacytidine and decitabine anti-cancer activity
              </a>
            </td>
          <td>
            Rishi V. Puram, Qiangzong Yin, YuhJong Liu, Justine C. Rutter, Daniel Bondeson, Maria C. Saberi, Lisa Miller, Michael Du, Khanh Nguyen, Donovan L. Batzli, Hilina B. Woldemichael, Christian Taeger, Anna Goldstein, Ming Y. Chu, Qi Guo, D. Mani, Michael Naumann, M. Ronan, M. Rees, Blanche C. Ip, Mustafa Kocak, Mikołaj Słabicki, John G Doench, Jennifer A. Roth, Steven A. Carr, N. Udeshi, Jingyi Wu, Todd R. Golub
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1231b220c32fa68e474d1bf7533e08b4effde6bb" target='_blank'>
              Cell enlargement causes mitotic errors and aneuploidy in cells that evade senescence after CDK4/6 inhibition
              </a>
            </td>
          <td>
            Aanchal U. Pareri, Marialucrezia Losito, F. Foijer, AT Saurin
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d14e9bed5ce451868baa1e549e50257d339b27d" target='_blank'>
              Transposable elements hitchhike on Starships across fungal genomes
              </a>
            </td>
          <td>
            Hanne Griem-Krey, Julia de Fraga Sant’Ana, Ursula Oggenfuss, Y. P. Calegari-Alves, Ana Luiza Marques, Markus Berger, L. Santi, W. O. Beys-da-Silva, M. Habig
          </td>
          <td>2025-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Whole-genome analyses have revealed that large-scale structural variations (SVs) such as whole-genome duplication (WGD) occur early in the development of many cancers. However, the diversity of chromosomal abnormalities within tumors before and after WGD remains poorly understood. Here, we analyzed various types of Japanese tumor genomes via whole-genome sequencing and examined the diversity of WGD by focusing on large SVs at the chromosomal level. WGD was detected in 52% of cases, while the frequency of chromothripsis (CT) was 20%. Although aneuploidy via deletion of chromosome arms was common in many cancers, in rare ovarian cancers, all chromosomes were near-haploidy before WGD. Minor allele analysis revealed that many non-mutated ohnolog genes drifted down chromosome arms after WGD and returned to normal ploidy, but only 17p, including TP53, which is also an ohnolog, underwent loss of heterozygosity due to arm deletion before WGD in most cancers. TP53 mutations were frequently detected in WGD and CT-positive tumors, and these SVs strongly correlated with homologous recombination deficiency scores. Furthermore, these tumors had many mutations that continued to generate neoantigens and resulted in worse survival outcomes. Diversity analysis of tumors with WGD will provide a new perspective on structural abnormalities in tumor genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53236a139dc9b0a444537137348625f035c225eb" target='_blank'>
              Allelic Imbalance and Chromothripsis Lead to Diversity in Japanese Tumor Genomes With Whole-Genome Duplication.
              </a>
            </td>
          <td>
            K. Hatakeyama, Takeshi Nagashima, Sumiko Ohnami, S. Ohnami, Koji Maruyama, K. Ohshima, Y. Shimoda, A. Naruoka, H. Kenmotsu, K. Urakami, Y. Akiyama, Ken Yamaguchi
          </td>
          <td>2025-08-06</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Polymerase theta (POLθ) inhibitors were developed to overcome resistance to PARP inhibitor treatment in homologous recombination (HR) deficient cancer. Biomarkers identifying PARP inhibitor-resistant cancer cases that specifically rely on POLθ activity for cancer cell survival will help to identify the sensitive patient population. From the TCGA RNAseq data, we determined POLθ expression levels, and from whole-exome and whole-genome sequencing data, we determined the number of POLθ-associated mutational signatures in solid tumors with various HR deficiency status. We found that POLθ expression levels did not differ significantly between HR-proficient and HR-deficient cancers. However, POLθ expression correlated strongly with proliferation-associated gene expression signatures and was predominantly observed in the S and G2/M phases of the cell cycle. POLθ-associated mutational signatures are correlated with POLθ expression levels only in BRCA2-deficient cancers. POLθ expression level and POLθ-associated mutational signatures may be indicative of POLθ inhibitor sensitivity in BRCA2-deficient tumors, but are unlikely to be informative in other cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3498ae751ba9f700cb39c5f192432c94acf2d269" target='_blank'>
              Polymerase theta expression is correlated with proliferative capacity but not with DNA repair deficiency status in solid tumors
              </a>
            </td>
          <td>
            Z. Sztupinszki, Regina Fiam, O. Pipek, M. Dióssy, J. Börcsök, Aurél Prósz, I. Csabai, Zoltan Szallasi
          </td>
          <td>2025-06-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eec7c3cc61289a3954a3ddd8b996ed6f9c80850e" target='_blank'>
              The adaptive molecular landscape of reprogrammed telomeric sequences
              </a>
            </td>
          <td>
            Melania D’Angiolo, Benjamin P. Barré, Sakshi Khaiwal, J. Muenzner, Johan Hallin, Matteo De Chiara, Nicolò Tellini, J. Warringer, M. Ralser, Eric Gilson, Gianni Liti
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Objective Repetitive DNA comprises the majority of nuclear genomes in eukaryotes and is critical for genome stability, gene regulation and evolutionary innovation. Yet, most genomic surveys focus on lowcopy regions, leaving repeats underexplored. Here, we assess how repetitive elements shape genomic diversity and rapid radiation in Dyckia (Bromeliaceae) by characterizing the abundance, composition and variability of major repeat families using lowcoverage whole-genome sequencing and the RepeatExplorer2 pipeline. Results description A substantial proportion of Dyckia genome consists of repetitive DNA, reaching ~ 71% in D. densiflora, D. elata, and D. consimilis. Notable interspecific and intraspecific variation was observed, with D. consimilis ranging from 50.3 to 69.1%. Satellite DNA, though present in all species, varied in abundance (0.1–4.7%), indicating heterochromatin flexibility. Ty3/Gypsy and Ty1/Copia LTR retrotransposons dominate the repeatome, with lineage-specific expansions of Tekay and Ogre elements. Despite general conservation, population-level differences in repeat composition suggest a role in genome restructuring and phenotypic plasticity. These patterns point to repeat dynamics as a key driver of genome evolution, taxonomic complexity, and ecological adaptability in Dyckia. Supplementary Information The online version contains supplementary material available at 10.1186/s13104-025-07359-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3a9ee02793b5181e16f379a04487e1d2a8906d6" target='_blank'>
              Genomic characterization of repetitive DNA and transposable elements in Dyckia (Pitcairnioideae) species
              </a>
            </td>
          <td>
            João Victor da Silva Rabelo-Araujo, A. F. Francisconi, Zirlane Portugal da Costa, Caroline Bertocco Garcia, Otavio Batista de Castro Ribeiro, Ana Cristina Silva Amoroso Anastacio, Maria Imaculada Zucchi
          </td>
          <td>2025-07-09</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The DNA repair protein RAD18 activates ‘Y-family’ Trans-Lesion Synthesis (TLS) DNA polymerases that are DNA damage-tolerant and potentially error-prone. RAD18 is also frequently overexpressed and pathologically activated in cancer cells. However, the extent to which RAD18 shapes cancer genomes and impacts tumorigenesis is unclear. Therefore, we tested the effect of Rad18 status on chemically-induced and oncogene-driven tumorigenesis. In a chemically-induced oral carcinogenesis model, acute (2-16 days) 4NQO-treatment induces expression of Rad18 and TLS polymerase mRNAs in mouse oral epithelial cells prior to emergence of oral squamous cell carcinomas (OSCCs). Chronic (8 week) 4NQO-treatment leads to onset of oral tumors that is accelerated in Rad18-/- mice when compared with Rad18+/+ animals. Analysis of OSCC genomes reveals increased levels of G(C)>T(A) transversions in Rad18-/- tumors when compared with Rad18+/+. Therefore, Rad18 promotes error-free bypass of 4NQO-induced DNA lesions and suppresses 4NQO-induced oral carcinogenesis. In a KrasG12D-induced lung carcinogenesis model, Rad18-deficiency did not affect rates or incidence of oncogene-induced lung tumors or mutations. Taken together, we demonstrate that Rad18 has context-specific tumor-suppressive activity. Given the prevalence of 4NQO-like environmental exposures, RAD18 is highly likely to shape human cancer genomes and perhaps influence other aspects of the tumorigenic process.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/318ddf85ddf5e7011f464ee8a8d2e2b51144fcdd" target='_blank'>
              DIFFERENTIAL ROLES OF RAD18 IN REPRESSING CARCINOGEN- AND ONCOGENE-DRIVEN MUTAGENESIS IN VIVO
              </a>
            </td>
          <td>
            J. R. Anand, Bethany Wagner, Jitong Lou, Qisheng Gu, Yang Yang, Gaith N Droby, Di Wu, Akankshya Jena, Jialiu Xie, Yuri Fedoriw, Bernard E Weissman, Scott E. Williams, Cyrus Vaziri
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71c01f9fcd4978a181214c5cc7b38fe087c3cef1" target='_blank'>
              ZCCHC4 Promotes Translation of Replication-dependent Histone mRNAs by Recruiting Cytoplasmic eIF3 complex
              </a>
            </td>
          <td>
            Ruiqi Wang, Xiaoyan Shi, Yangyi Zhang, Yuci Wang, Yanlan Cao, Rui Feng, Chen Chen, Yingchun Zhang, Hao Chen, Honghui Ma
          </td>
          <td>2025-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background Human endogenous retroviruses (HERVs) harbor accessory proteins that influence cellular processes and have been linked to a wide variety of diseases, including cancer. This study investigates locus-specific HERV expression and its association with gene dysregulation in hepatocellular carcinoma (HCC), a highly prevalent and deadly form of liver cancer worldwide. Methods We analyzed RNASeq data from 424 HCC samples from The Cancer Genome Atlas (TCGA), which comprised 371 tumor and 50 matched normal tissues from a total of 371 hepatocellular carcinoma participants. We employed Telescope to identify and quantify HERV expression across the total RNA sequencing data. Results The majority of differentially expressed HERVs exhibited reduced expression in tumor tissue (166 downregulated vs. 50 upregulated), suggesting a potential functional role of HERV expression patterns in shaping the pathophysiological landscape of HCC. Specifically, the suppression of HERV-H family members, which are known to regulate cellular differentiation, may contribute to tumor dedifferentiation, increased plasticity, and enhanced metastatic potential. This loss of differentiation control and increased adaptability may play a critical role in driving the progression of liver cancer. Discussion Our study highlights a significant association of HERV expression with HCC, highlighting the differential regulation of specific HERV families in tumor tissue. For example, HERVH and ERVLE families showed consistent downregulation in tumor samples, while HERVE and HERV9 were more commonly upregulated. These shifts may reflect underlying changes in transcriptional regulation or chromatin structure between normal and malignant tissues. Rather than indicating a singular functional role, the observed expression patterns likely reflect a multifaceted relationship between HERVs and tumor biology. Further studies will be needed to determine whether these expression differences contribute to, or result from, tumor progression and to explore their potential as biomarkers or therapeutic targets. Clinical trial number Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s13100-025-00367-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06da0045d446b88205ce15609079cd44c93959a4" target='_blank'>
              Locus-specific HERV expression associated with hepatocellular carcinoma
              </a>
            </td>
          <td>
            Hui Wen, M. Pérez‐Losada, Lopa Mishra, Keith A. Crandall
          </td>
          <td>2025-07-16</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="The spatial organization of the genome is essential for its functions, including gene expression and chromosome segregation. Phase separation and loop extrusion have been proposed to underlie compartments and topologically associating domains, however, whether the fold of genomic DNA inside the nucleus is consistent with such mechanisms has been difficult to establish in situ. Here, we present a 3D DNA-tracing workflow that resolves genome architecture in single structurally well-preserved cells with nanometre resolution. Our findings reveal that genomic DNA generally behaves as a flexible random coil at the 100-kb scale. At CTCF sites however, we find Cohesin-dependent loops in a subset of cells, in variable conformations from the kilobase to megabase scale. The 3D-folds we measured in hundreds of single cells allowed us to formulate a computational model that explains how sparse and dynamic loops in single cells underlie the appearance of compact topological domains measured in cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72d5b9c23558a964615d529fe8d8e6ae55020cca" target='_blank'>
              Nanoscale 3D DNA tracing in non-denatured cells resolves the Cohesin-dependent loop architecture of the genome in situ
              </a>
            </td>
          <td>
            K. S. Beckwith, Ø. Ødegård-Fougner, N. R. Morero, C. Barton, F. Schueder, W. Tang, S. Alexander, Jan-Michael Peters, R. Jungmann, E. Birney, J. Ellenberg
          </td>
          <td>2025-07-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Mapping the spatial organization of DNA-level somatic copy number changes in tumors can provide insight to understanding higher-level molecular and cellular processes that drive pathogenesis. We describe an integrated framework of spatial transcriptomics, tumor/normal DNA sequencing, and bulk RNA sequencing to identify shared and distinct characteristics of an initial cohort of eleven gliomas of varied pathology and a replication cohort of six high-grade glioblastomas. We identify focally amplified extrachromosomal DNA (ecDNA) in four of the eleven initial gliomas, with subclonal tumor heterogeneity in two EGFR-amplified grade IV glioblastomas. In a TP53-mutated glioblastoma, we detect a subclone with EGFR amplification on ecDNA coupled to chromosome 17 loss of heterozygosity. To validate subclonal somatic aneuploidy and copy number alterations associated with ecDNA double minutes, we examine the replication cohort, identifying MDM2/MDM4 ecDNA subclones in two glioblastomas. The spatial heterogeneity of EGFR and p53 inactivation underscores the role of ecDNA in enabling rapid oncogene amplification and enhancing tumor adaptability under selective pressure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d48c8db15868df338dc48b4c0bf5130f5e0f200b" target='_blank'>
              Resolving spatial subclonal genomic heterogeneity of loss of heterozygosity and extrachromosomal DNA in gliomas
              </a>
            </td>
          <td>
            Michelle G. Webb, Frances Chow, Carmel G. McCullough, Bohan Zhang, John J. Y. Lee, Rania Bassiouni, Norman E. Garrett, Kyle Hurth, J. Carpten, Gabriel Zada, David W. Craig
          </td>
          <td>2025-06-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="ABSTRACT Tardigrades are microscopic animals that can survive exceptional levels of ionizing radiation or desiccation – DNA-damaging conditions that would kill most animals. Irradiation or radiomimetic drug treatment of the tardigrade Hypsibius exemplaris can induce remarkably high expression levels of DNA repair genes, primarily those in the base excision repair and nonhomologous end joining pathways. How tardigrades can repair widespread DNA damage without producing frequent, large-scale chromosome structural abnormalities, like chromosome translocations and fusions, is unknown. Here, we report the results of examining chromosome and nuclear architecture throughout the cell cycle in early embryos of H. exemplaris . We found that H. exemplaris chromosomes are maintained in an individualized form throughout the cell cycle. We were surprised to also find that each chromosome is housed in a fully or partially separate lamin-lined compartment, instead of all chromosomes being housed in a single, nearly spherical nuclear lamina and envelope. Our results reveal unusual chromosomal and nuclear organization in a tardigrade. We speculate that these unexpected features might limit chromosomal rearrangements during DNA damage repair in extreme conditions. SIGNIFICANCE STATEMENT We have investigated unusual chromosome organization in an organism that survives extremely DNA-damaging environments, a tardigrade, through early embryonic cell cycles. Chromosomes in fixed and stained embryos appeared more condensed through interphase than is typical for animal cells. Chromosomes remained individualized, in fully or partially separate lamin-lined compartments, through interphase. The results reveal a unique nuclear and chromosomal organization in tardigrades, which we speculate might contribute to limiting chromosomal structure abnormalities under DNA damaging conditions in nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b59b0e24503d3afcd35a7b4c4c2e06e9d2060436" target='_blank'>
              Chromosomes remain individualized through interphase in embryos of the tardigrade Hypsibius exemplaris
              </a>
            </td>
          <td>
            Lillian D. Papell, Adriana N Coke, Bailey N. de Jesus, Clayton J. Harry, Pu Zhang, Bob Goldstein
          </td>
          <td>2025-07-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Extrachromosomal-circular DNA (eccDNA) are circular-DNA elements that are implicated in cellular processes such as genomic diversification and instability, along with other genomic mechanisms. However, their structure and chromatinization state is poorly understood. Here we identified a 358-bp circular micro-eccDNA molecule derived from the human Titin gene, which was selected for this study based on its size and sequence characteristics, and used atomistic molecular dynamics simulations to study its interaction with nucleosomes and corresponding topological state. We show the presence or absence of bound nucleosomes provides a topological switch between intact and denatured DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/779c13815a53f4713be7d07b72908d9198ac23e6" target='_blank'>
              Nucleosome-mediated conformational switches in micro-eccDNAs
              </a>
            </td>
          <td>
            Sarah Harris, Victor Velasco Berrelleza, William Sandel, T. Sutthibutpong, Victoria N. Parikh, Euan A. Ashley, Andrew Fire, S. Levene, Thomas C Bishop, M. Shoura
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Atypical teratoid rhabdoid tumors (ATRTs) are aggressive central nervous system tumors mainly affecting young children. Extensive molecular characterization based on gene expression and DNA methylation patterns has solidly established three major ATRT subtypes (MYC, SHH and TYR), which show distinct clinical features, setting the basis for more effective, targeted treatment regimens. Transcriptional activity of transposable elements (TEs), like LINE1s and LTRs, is tightly linked with human cancers as a direct consequence of lifting epigenetic repression over TEs. The sole recurrent biallelic loss-of-function mutation in SMARCB1 in ATRTs, a core component of the SWI/SNF chromatin remodeling complex, raises the question of how TE transcription contributes to ATRT development. Here, we comprehensively investigate the transcriptional profiles of 1.9M LINE1 and LTR elements across ATRT subtypes in primary human samples. We find TE transcription profiles allow sample stratification into ATRT subtypes. The TE activity signature in the ATRT-MYC subtype is unique, setting these tumors apart from SHH and TYR ATRTs. More specifically, ATRT-MYC show broadly reduced transcript levels of LINE1 and ERVL-MaLR subfamilies. ATRT-MYC also displayed significantly less LTR and LINE1 loci with bidirectional promoter activity. Furthermore, we identify 849 differentially transcribed TEs in primary samples, which are predictive towards established ATRT-SHH and -MYC cell line models. In summary, including TE transcription profiles into the molecular characterization of ATRTs might reveal new tumor vulnerabilities leading to novel therapeutic interventions, such as immunotherapy. Supplementary Information The online version contains supplementary material available at 10.1186/s40478-025-02078-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c60e5327059223ad1194aac3ab94faeb7d65828" target='_blank'>
              Expression of LTR and LINE1 transposable elements defines atypical teratoid/rhabdoid tumor subtypes
              </a>
            </td>
          <td>
            M.V. Hamann, Shweta Godbole, Maisha Adiba, Sabrina M. Leddy, Jelena Navolić, Ghazaleh Tabatabai, Daniel J Merk, Ulrike C. Lange, J. Neumann
          </td>
          <td>2025-07-22</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/929755ae2e9de92ac1b7d8df8a259bc5e4e63f92" target='_blank'>
              Hicberg: Reconstruction of contact signals from repeated elements
              </a>
            </td>
          <td>
            Sébastien Gradit, Samuel Ortion, Pauline Larrous, Maëlys Delouis, Romain Koszul, A. Cournac
          </td>
          <td>2025-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/314188a34bfab3927bff3ed290d5c9df78c0fa8b" target='_blank'>
              ChAHP Silences SINE Retrotransposons by Inhibiting TFIIIB Recruitment
              </a>
            </td>
          <td>
            Jakob Schnabl-Baumgartner, Fabio Mohn, Michaela Schwaiger, Josip Ahel, Jennifer Steiner, Yukiko Shimada, Sirisha Aluri, M. Bühler
          </td>
          <td>2025-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The identification of CENPA, CENPB, and CENPC by Earnshaw and Rothfield 40 years ago has revealed the remarkable diversity and complexity of centromeres and confirmed most seed plants and animals have centromeres comprised of complex satellite arrays. The rapid evolution of centromeres and positive selection on CENPA and CENPC led to the centromere drive model, in which competition between tandem satellite arrays of differing size and centromere strength for inclusion in the egg of animals or megaspore of seed plants during female meiosis drives rapid evolution of centromeres and kinetochore proteins. Here we review recent work showing that non-B-form DNA structures in satellite centromeres make them sites of frequent replication fork stalling, and that repair of collapsed forks by break-induced replication rather than unequal sister chromatid exchange is likely the primary mode of satellite expansion and contraction, providing the variation in satellite copy number that is the raw material of centromere drive. Centromere breaks at replication, rather than errors at mitosis, can account for most centromere misdivisions that underlie aneuploidies in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65fb737fe39c4474778577289e08e6983374bf35" target='_blank'>
              Centromeres drive and take a break
              </a>
            </td>
          <td>
            P. Talbert, S. Henikoff
          </td>
          <td>2025-08-04</td>
          <td>Chromosome Research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Tumors of infectious origin globally represent 13%. Oncogenic DNA viruses such as human papillomavirus (HPV), hepatitis B virus (HBV), and Epstein-Barr virus (EBV) are responsible for approximately 60% of these tumors. These oncoviruses are extensively studied to understand their role in cancer development, particularly through viral genome integration into the host DNA. Retroviruses require integration mediated by viral integrase for persistence, whereas DNA oncoviruses do not need integration for replication; instead, integration occurs incidentally. This process often targets fragile sites in the human genome, causing structural rearrangements that disrupt genes, activate proto-oncogenes, and increase genomic instability, all contributing to tumorigenesis. Integration near promoter regions and active genes is closely linked to carcinogenesis, highlighting its importance in developing diagnostic and therapeutic strategies. This review summarizes viral integration's role in oncogenesis, mechanisms of integration, and methods to study this process, focusing on DNA tumor viruses such as HBV, EBV, HPV, and Merkel cell polyomavirus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ef1eb5cd84b08745894e948edf6d9875f648ea7" target='_blank'>
              Genome integration of human DNA oncoviruses.
              </a>
            </td>
          <td>
            Zuzana Vojtěchová, R. Tachezy
          </td>
          <td>2025-07-23</td>
          <td>Journal of virology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="DNA carries genetic information, ensuring the stable transmission of genetic material through generations. However, DNA sequences can be constantly rewritten, allowing evolution and adaptation to occur at both the cellular and species levels. To facilitate this dynamic process, the genome is enriched with mobile genetic elements that can move within DNA sequences. While transposable elements are the classic example of mobile DNA, we propose that extrachromosomal circular DNA (ecDNA) represents another class of mobile genetic elements. This class can also introduce a new layer of genome dynamics, significantly shaping host biology. This review traces the historical discoveries and conceptual evolution of ecDNA, categorizing its diverse forms across organisms and highlighting its life cycle from biogenesis to maintenance and clearance. We discuss ecDNA's pivotal roles in physiological processes, including development, stress adaptation, and species evolution, and emphasize its pathological significance in cancer progression, drug resistance, and viral infections. Finally, we explore therapeutic strategies targeting ecDNA biology, offering a framework for translational advances in oncology and antiviral treatment. Together, these insights position ecDNA as a critical force in shaping genome dynamics and as a promising target for future biomedical interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a2c075da6c2a7912686c8ef608ee9b31391b746" target='_blank'>
              Extrachromosomal Circular DNA: A Mobile Genetic Element Shaping Host Biology.
              </a>
            </td>
          <td>
            Shun Yao, Oliver W. Chung, Ling Wang, Z. Zhang
          </td>
          <td>2025-08-05</td>
          <td>Annual review of cell and developmental biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Microhomology-mediated tandem duplication (TD) is a distinct mutational signature in BRCA1-deficient tumors. While several mechanisms have been proposed for its generation, much understanding comes from repeat-based reporters, which may not represent the region of the human genome lacking neighboring repeats. To address this limitation, we developed a repeat-less TD reporter and a PCR-based endogenous site-specific TD assay to examine TDs induced by replication-coupled single-ended DNA double-strand breaks (seDSBs) in Brca1-deficient mouse embryonic stem cells. While TDs induced by seDSBs were detectable in wild-type cells, they were significantly elevated in Brca1-deficient cells, independent of classical non-homologous end joining. Some of these TDs arose from allelic DNA synthesis followed by a shift to microhomology-mediated non-allelic sister chromatid recombination, suggesting increased premature termination of allelic DNA synthesis, a late step of allelic break-induced replication (aBIR), in Brca1-deficient cells. Disrupting RAD51 loading reduced TDs in wild-type cells but not in Brca1-deficient cells, indicating both RAD51-dependent and -independent TD formation. Additionally, RAD54 and BRCA1–BARD1 suppressed TDs in a RAD51-dependent manner, with the effect of BARD1 effect partially mediated through RAD51 interaction. These findings together implicate late steps of aBIR in TD suppression and provide novel insights into the mechanisms underlying BRCA1-linked TD formation in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7edf834c059a1df782121d4a36a790a684d7b6d4" target='_blank'>
              Late steps of allelic break-induced replication suppress tandem duplication associated with BRCA1 deficiency
              </a>
            </td>
          <td>
            Zhi-cheng Huang, , Qian Liu, Ruo-Dan Chen, Si-Cheng Liu, Meng Wang, A. Xie
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Polarity genes form intracellular protein complexes that establish epithelial cell polarity and homeostasis for various normal cellular functions. Partitioning Defective 6B (PARD6B), a polarity gene, functions as a scaffold node for the Par protein complex. However, its contribution to colon cancer is not well understood. In this study, we showed that PARD6B regulates tumor progression in colorectal cancer (CRC). PARD6B is located on chromosome 20, which is frequently amplified in CRC, and its expression in CRC correlates with DNA copy number amplification, including enhancer regions. Immunohistochemistry and single-cell analyses also showed that PARD6B expression was significantly higher in cancer cells. Furthermore, unlike other polarity gene groups comprising the Par complex, PARD6B mRNA expression was the only independent poor prognostic factor. In vitro and in vivo experiments revealed that PARD6B positively regulates cell proliferation and cell cycle progression. In silico analysis also showed that PARD6B expression positively regulated MYC expression, a pathway believed to be associated. Additional in silico and in vitro analyses supported the hypothesis that PARD6B regulates miR-34c, which directly targets and represses MYC expression. Pan-cancer analysis indicated that PARD6B is highly expressed in gastrointestinal tumors, including CRC, and that high PARD6B mRNA expression is a poor prognostic factor in other cancer types. In summary, highly expressed PARD6B can promote CRC growth by upregulating MYC expression while suppressing miR-34c expression, making PARD6B a potential prognostic biomarker and therapeutic target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e805e1cc21e6cdb6e4ec857af7492b495f17f7a1" target='_blank'>
              Polarity Gene PARD6B Promotes Tumor Growth of Colorectal Cancer via Increasing MYC Expression.
              </a>
            </td>
          <td>
            Kosuke Hirose, Takaaki Masuda, Hajime Otsu, Taro Tobo, Kiyotaka Hosoda, Tadashi Abe, Yusuke Nakano, M. Hashimoto, Takanari Tatsumi, Tomohiko Ikehara, Takashi Ofuchi, Shinsaku Itoyama, Yuki Ando, Yasuo Tsuda, Yusuke Yonemura, K. Sugimachi, E. Oki, T. Yoshizumi, K. Mimori
          </td>
          <td>2025-06-18</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>85</td>
        </tr>

        <tr id="CIC::DUX4 translocation-positive sarcomas (CDS) are rare, highly proliferative tumors associated with chemotherapy resistance. Without a clear understanding of the molecular pathways driving CDS, no effective treatment regimens have been developed. Here, we identify protection against DNA damage through upregulation of POLE and DNA repair pathways specific to CDS. These data may explain the high proliferation rate and chemotherapy resistance of these tumors and suggest targeted treatment strategies for CDS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9c1daa0e52dfd54fcd1b1c744a6ff9aaea6aa92" target='_blank'>
              Upregulation of POLE and proficient DNA repair are features of CIC::DUX4 sarcomas
              </a>
            </td>
          <td>
            Nathan D. Seligson, Anjali Paragji, Feha Shahalam, Binyam Yilma, James L Chen
          </td>
          <td>2025-06-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract DNA–protein crosslinks (DPCs) form following exposure to various alkylating agents, including environmental carcinogens, cancer chemotherapeutics, and reactive aldehydes. If not repaired, DPCs can interfere with key biological processes such as transcription and replication and activate programmed cell death. A growing body of evidence implicates nucleotide excision repair (NER), homologous recombination, and other mechanisms in the removal of DPCs. However, the effects of genomic context on DPC formation and removal have not been comprehensively addressed. Using a combination of next-generation sequencing and DPC enrichment via protein precipitation, we show that DPCs induced following exposure to formaldehyde are non-randomly distributed across the human genome, based on chromatin state. The data further show that the efficiency of DPC removal correlates with transcription at loci transcribed by RNA polymerase II. Data presented herein indicate that efficient removal of chromosomal DPCs requires both the Cockayne syndrome group B gene as well as “downstream” TC-NER factor xeroderma pigmentosum group A gene. In contrast, loci transcribed by RNA polymerase I showed no evidence of transcription-coupled DPC removal. Taken together, our results indicate that complex interactions between chromatin organization, transcriptional activity, and numerous DNA repair pathways dictate genomic patterns of DPC formation and removal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/138482f47c7713ca6d5e082edda14419c0390855" target='_blank'>
              Genome-wide mapping of formaldehyde-induced DNA–protein crosslinks reveals unique patterns of formation and transcription-coupled removal in mammalian cells
              </a>
            </td>
          <td>
            Duha Alshareef, Charlie T Nguyen, Kayla N Tucker, Micah D. Gearhart, Natalia Y Tretyakova, Colin Campbell
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Jumping translocations (JTs) are rare chromosomal abnormalities that play a crucial role in the pathogenesis of various cancer types. These rearrangements, especially those involving chromosome 1q, are frequently associated with tumor progression, therapeutic resistance, and poor prognosis. One gene of particular interest, human Jumping Translocation Breakpoint (JTB), has been identified at the site of translocation breakpoints and exhibits complex, context-dependent roles in cancer biology. JTB protein functions as a pivotal regulator in mitosis, chromosomal segregation, apoptosis, and cellular metabolism. It is functionally linked with the chromosomal passenger complex (CPC) and is implicated in processes such as epithelial–mesenchymal transition (EMT), immune evasion, and therapy resistance, especially in breast and prostate cancers. Advances in genomic, transcriptomic, and proteomic research have highlighted the significant potential of JTB as a diagnostic biomarker and a target for therapeutic interventions. This review underscores the dual role of JTB as both a tumor suppressor and oncogene, depending on the cellular context, and advocates for its continued investigation at the genomic, transcriptomic, and proteomic levels. Understanding JTB’s multifaceted contributions to tumor biology may pave the way for novel biomarkers and targeted treatments in cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48960a3d042990da8ae30d21a39837a5c48dded4" target='_blank'>
              From Jumping Gene to Cancer: Revisiting the Role of JTB Protein
              </a>
            </td>
          <td>
            Taniya M Jayaweera, Madhuri Jayathirtha, Krishan Weraduwage, Petra Kraus, C. Darie, A. Neagu
          </td>
          <td>2025-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract Multiple myeloma (MM) is a type of hematological cancer that arises from uncontrolled proliferation of plasma cells. In addition to frequent genetic mutations, malignant plasma cells are characterized by alterations to the epigenome. Myeloma cells display a genome-wide loss of DNA methylation and a corresponding increase in ‘active’ chromatin modifications. The dramatic epigenetic remodeling that occurs in cancer genomes is associated with loss of silencing at transposable elements, which can impact genome regulation. Through paired epigenome and transcriptome profiling of patient derived MM samples, we have found that loss of DNA methylation in MM genomes results in the formation of partially methylated domains that are variable across patients. This loss of DNA methylation coincides with the expression of hundreds of transcripts driven by LINE-1 (L1) retrotransposons that are epigenetically silenced in normal cells. MM samples can be stratified based on L1 activity with distinct gene expression signatures. The high L1 samples are characterized by a more proliferative, less differentiated state as well as inhibition of interferon and genome defense pathways. Several L1 promoters generate chimeric transcripts with adjacent oncogenes. We further find that KRAB-zinc finger proteins (KZFPs) that are responsible for the epigenetic silencing of L1s have abnormally low abundance in MM samples with high L1 activity. These results indicate that cell proliferation in MM is associated with a loss of KZFP expression and activation of L1 elements. Significance Epigenome and transcriptional profiling of patient-derived multiple myeloma samples reveals heterogeneous loss of DNA methylation, reduction of KZFP expression and LINE-1 activation as a clinical indicator of progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e8c94d95f4ddee7048f28a27ac43f9a3d704355" target='_blank'>
              Methylation variability and LINE-1 activation in multiple myeloma
              </a>
            </td>
          <td>
            Qianhui Wan, Amy Leung, Mahek Vinod Bhandari, Hiroyuki Kato, Joo Y. Song, Dustin Schones
          </td>
          <td>2025-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc6c0eb75e7f5a0febc3a26c4b5b85dcca6192f2" target='_blank'>
              Validation of the MAGIC genomic signature on RNAseq in soft-tissue sarcomas
              </a>
            </td>
          <td>
            A. Benhaddou, G. Pérot, P. Rochaix, T. Valentin, G. Ferron, F. Chibon
          </td>
          <td>2025-07-21</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="T cell receptor (TCR) diversity, essential for the recognition of a wide array of antigens, is generated through V(D)J recombination. The Tcra and Tcrd genes reside within a shared genomic locus, with Tcrd rearrangement occurring first in the double-negative (DN) stage during thymocyte development. Elucidating the regulatory mechanisms governing Tcrd rearrangement is therefore crucial for understanding the developmental coordination of both Tcrd and Tcra rearrangements. Chromatin architecture, orchestrated by CTCF-cohesin complexes and their binding sites, plays a fundamental role in regulating V(D)J recombination of antigen receptor genes. In this study, we report that EACBE, a CTCF binding element (CBE) located downstream of the Tcra-Tcrd locus, regulates Tcrd rearrangement. EACBE promotes the usage of proximal Vδ gene segments by facilitating spatial proximity between the Tcrd recombination centre and these Vδ elements. Notably, EACBE counteracts the insulating effects of INTs, two CBEs that demarcate the proximal V region from the Dδ-Jδ-Cδ cluster, thereby enabling effective chromatin extrusion. Furthermore, EACBE indirectly shapes the Tcra repertoire through its influence on Tcrd rearrangement. These findings reveal a novel regulatory axis involving special chromatin configuration and highlight distinct roles for specific CTCF binding sites in modulating antigen receptor gene assembly.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39969305cd184c6ac930cc7afb8573a26cbe9e2b" target='_blank'>
              The insulator EACBE regulates V(D)J recombination of Tcrd gene by modulating chromatin organization
              </a>
            </td>
          <td>
            Yongchang Zhu, Ranran Dai, Hao Zhao, Junwei Luo, Keyi Li, Wei Xue, Litao Qin, Hongyuan Pan, Shixiu Liao, Bingtao Hao
          </td>
          <td>2025-07-17</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The capacity of a plasmid to express genes is constrained by its length and copy number. However, the interplay between these parameters and their constraints on plasmid evolution have remained elusive due to the absence of comprehensive quantitative analyses. Here, we present ‘Pseudoalignment and Probabilistic Iterative Read Assignment’ (pseuPIRA), a computational method that overcomes previous computational bottlenecks, enabling rapid and accurate determination of plasmid copy numbers at large scale. We apply pseuPIRA to all microbial genomes in the NCBI RefSeq database with linked short-read sequencing data (4644 bacterial and archaeal genomes including 12,006 plasmids). The analysis reveals three scaling laws of plasmids: first, an inverse power-law correlation between plasmid copy number and plasmid length; second, a positive linear correlation between protein-coding genes and plasmid length; and third, a positive correlation between metabolic genes per plasmid and plasmid length, particularly for large plasmids. These scaling laws imply fundamental constraints on plasmid evolution and functional organization, indicating that as plasmids increase in length, they converge toward chromosomal characteristics in copy number and functional content.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96d0a945dacbeb8801a3cb57a4c7f7eb8870d06a" target='_blank'>
              Scaling laws of bacterial and archaeal plasmids
              </a>
            </td>
          <td>
            Rohan Maddamsetti, Irida Shyti, Maggie L. Wilson, Hye-in Son, Yasa Baig, Zhengqing Zhou, Jia Lu, Lingchong You
          </td>
          <td>2025-07-02</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="Agrobacterium-mediated plant transformation, which enables the delivery of DNA using transfer DNA (T-DNA) binary vectors, is an essential technique in plant research. T-DNAs randomly integrate into the host genome, and multiple T-DNAs can sometimes integrate during a single transformation, necessitating the development of tools to elucidate the T-DNA insertion sites (TISs). Here, we present T-DNAreader, which identifies TISs from RNA sequencing data with high precision, sensitivity, and speed, outperforming existing tools. T-DNAreader enables the efficient and reliable identification of TISs within transcribed regions and standardizes the characterization of T-DNA-containing transgenic plants, which can be expanded to various organisms containing T-DNA. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03655-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae9a9abbf154662461d09ad2dd90fa2197ff32ea" target='_blank'>
              T-DNAreader: fast and precise identification of T-DNA insertion sites in plant genomes using RNA sequencing data
              </a>
            </td>
          <td>
            H. Lee, Pil Joon Seo
          </td>
          <td>2025-07-10</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="


 Transposable elements (TEs) are dynamic repetitive regions constituting 50% of the genome. In adult cancers, TEs contribute to carcinogenesis through structural rearrangements that disrupt tumor suppressors and by onco-exaptation, which drives aberrant oncogene expression. TEs are characterized in adult malignancies, with epithelial cancers harbouring significantly more new insertions. However, the role of TEs in pediatric cancer is unknown. Many pediatric cancers arise from predisposition, such as Li-Fraumeni Syndrome, which is caused by germline TP53 (gTP53) variants. TP53 suppresses TEs by binding L1 elements to inhibit transcription and by inducing apoptosis in cells with high retrotransposition. Moreover, adult cancers with somatic TP53 variants harbour significantly more TEs than wildtype tumors. To investigate the pediatric context, we conducted a pan-cancer analysis of germline and tumor TE landscapes in pediatric patients with (n = 66 germline; n = 40 tumor) and without (n = 465 germline; n = 276 tumor) gTP53 variants. Germline TEs were identified using MELT, xTEA, and INSurVeyor, while tumor TEs were called with xTEA and TotalReCall. TEs were consolidated across individuals using JASMINE and annotated with AnnotSV. We excluded 21% of germline and 2% of tumor TEs classified as common, defined as present in over 3% of gnomAD or pediatric non-cancer controls (n=189). Overall, 11% of pediatric tumors harbored at least one L1 insertion, with a median of 0 insertions per tumor, substantially lower than the 21% reported in adult cancers using the same TE callers (Solovyov et al). Frequency in germline or tumor TEs did not vary significantly by tumor type, age of onset, sex, prior treatment, or disease state. Germline and tumor TEs were found in 35 and 147 cancer genes respectively. Tumor TEs significantly affected many cancer pathways, including those involved in mitosis. While no structural variants (SVs) involved TEs at both breakpoints, 171 SVs in six samples were associated with a TE at one breakpoint. Importantly, both germline and tumor TE burden did not differ between gTP53 carriers and wildtype individuals, nor did the number of evolutionarily young or full-length elements. Pathway enrichment analyses revealed no significant differences in TE-affected pathways between gTP53 and wildtype groups. However, a gradient-boosted tree model trained on TE distribution across genomic windows predicted gTP53 status with an AUPRC of 0.77, suggesting that gTP53 variants shape the germline TE landscape at a positional level rather than through global frequency changes. In summary, pediatric cancers exhibit a quiescent TE landscape, with low insertion rates across tumor types. gTP53 variants do not alter the germline or tumor TE frequency,but influence the location of germline insertions. TE analysis in pediatric cancer and gTP53 carriers will enhance our understanding of tumorigenesis, informing future treatment approaches.



 Brianne Laverty, Shilpa Yadahalli, Safa Majeed, Ashby Kissoondoyal, Alexander Solovyov, Scott Davidson, Yisu Li, Mehdi Layeghifard, Adam Shlien, Vallijah Subasri, David Malkin. Pediatric pan-cancer characterization of transposable elements and their modulation by germline TP53 variants [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A039.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ba8aeceb59adfe162dadfd953074aed2a705a85" target='_blank'>
              Abstract A039: Pediatric pan-cancer characterization of transposable elements and their modulation by germline TP53 variants
              </a>
            </td>
          <td>
            B. Laverty, Shilpa Yadahalli, Safa Majeed, Ashby Kissoondoyal, Alexander Solovyov, Scott Davidson, Yisu Li, M. Layeghifard, A. Shlien, Vallijah Subasri, D. Malkin
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 APOBEC3 (A3) enzymes play a pivotal role in mutagenesis across various cancer types. These enzymes, which are single-stranded DNA cytosine deaminases, convert cytosine to uracil (C-to-U) as part of innate antiviral immune responses. Aberrant expression of A3 enzymes in several cancers leads to DNA damage, mutagenesis, and genomic instability. In breast cancer, in particular, APOBEC3B (A3B) is the major endogenous mutator that contributes to tumor evolution and resistance to treatment (1–4). DNA C-to-U deamination events result in characteristic C-to-T transitions and C-to-G transversions, and genomic uracils can also be processed into single- or double-stranded DNA breaks and larger chromosomal aberrations. Consequently, tumors with elevated A3B levels endure chronic genotoxic stress and may exhibit increased sensitivity to inhibitors targeting specific DNA repair pathways. Previous research from our laboratory identified DNA uracil glycosylase 2 (UNG2), a critical initiator of the base excision repair (BER) pathway, as a synthetic lethal partner with A3B (5). Genetic disruption of UNG2, combined with high A3B expression, led to cell death (5). This finding suggests that other DNA repair proteins could potentially serve as synthetic lethal partners with A3B under conditions of elevated A3B-induced DNA damage. To test this hypothesis, we used pharmacological and genetic inhibition of DNA damage repair proteins in isogenic cell line models for A3B. Specifically, we quantified cancer cell viability following treatment with various commercially available DNA damage response inhibitors, in the presence or absence of A3B. Additionally, we performed CRISPR screens using a guide RNA library targeting 237 DNA damage repair and response genes in a doxycycline-inducible TREX-293-A3Bi-eGFP cell line. Cells expressing or lacking A3B were harvested for DNA extraction and sequencing at different time points. By comparing guide RNA abundance between doxycycline-treated cells and untreated cells, we identified dropout guides that disrupt genes, creating potential synthetic lethal combinations with A3B. The results from both pharmacological and genetic inhibition approaches will be presented.
 Selected References:
 1. Burns, M. B. et al. APOBEC3B is an enzymatic source of mutation in breast cancer. Nature 494, 366–370 (2013).
 2. Law, E. K. et al. The DNA cytosine deaminase APOBEC3B promotes tamoxifen resistance in ER-positive breast cancer. Sci Adv 2, e1601737 (2016).
 3. Bertucci, F. et al. Genomic characterization of metastatic breast cancers. Nature 569, 560–564 (2019).
 4. Venkatesan, S. et al. Induction of APOBEC3 exacerbates DNA replication stress and chromosomal instability in early breast and lung cancer evolution. Cancer Discov (2021) doi:10.1158/2159-8290.CD-20-0725.
 5. Serebrenik, A. A. et al. The deaminase APOBEC3B triggers the death of cells lacking uracil DNA glycosylase. Proc Natl Acad Sci USA 116, 22158–22163 (2019).
 Citation Format: Bojana Stefanovska, Benjamin Troness, Kevin Lin, Chad Myers, Reuben S. Harris. APOBEC3B And DNA Damage Repair As Synthetic Lethal Pairs [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86e021a93ee84b96a73fc8c1738f94674803feaf" target='_blank'>
              Abstract P5-06-26: APOBEC3B And DNA Damage Repair As Synthetic Lethal Pairs
              </a>
            </td>
          <td>
            B. Stefanovska, Benjamin Troness, Kevin Lin, Chad Myers, R. Harris
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Influenza A virus (IAV) is a highly diverse pathogen with genetic variability primarily driven by mutation and reassortment. Using next-generation sequencing (NGS), we characterised defective viral genomes (DVGs) generated during the serial passaging of influenza A/Puerto Rico/8/1934 (H1N1) virus in embryonated chicken eggs. Deletions were the most abundant DVG type, predominantly accumulating in the polymerase-encoding segments. Notably, we identified and validated a novel class of multisegment DVGs arising from intersegment recombination events, providing evidence that the IAV RNA polymerase can detach from one genomic template and resume synthesis on another. Multisegment recombination primarily involved segments 1–3 but also occurred between other segment pairings. In specific lineages, certain multisegment DVGs reached high frequencies and persisted through multiple passages, suggesting they are not transient by-products of recombination but may possess features that support stable maintenance. Furthermore, multisegment DVGs were shown to be encapsidated within virions, similar to deletion DVGs. The observation of recombination between segments with limited sequence homology underscores the potential for complex recombination to expand IAV genetic diversity. These findings suggest recombination-driven DVGs represent a previously underappreciated mechanism in influenza virus evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/703aa1fba00eaf6f6b2b6c3aad9de3c0760d9a3b" target='_blank'>
              Intersegment Recombination During Influenza A Virus Replication Gives Rise to a Novel Class of Defective Viral Genomes
              </a>
            </td>
          <td>
            Soraya Anisi, George Noble, Rory Williams, Jack Hales, Hannah E. Bridgewater, Andrew Easton, William Collier, Phillip Gould
          </td>
          <td>2025-06-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Motivation The wide range of cellular complexity created by multicellular organisms is due in large part to the intricate and synergistic interplay of regulatory complexes throughout the eukaryotic genome. These regulatory elements “enhance” specific gene programs and have been shown to operate in diverse networks that are distinct across cell states of the same organism. Attempts to characterize and predict enhancers have typically focused on leveraging information-dense DNA sequence in parallel with epigenomic assays. We examined the viability of enhancer prediction using only a minimal set of epigenomic datasets without direct DNA information. Results We demonstrate that chromatin datasets are sufficient to identify enhancers genome-wide with high accuracy. By training networks leveraging data from multiple cell types simultaneously, we generated a cell-type invariant enhancer prediction platform that utilized only the patterns of protein binding for inference. We also showed the utility of swarm-based adversarial attacks [adversarial particle swarm optimization (APSO)] to deconvolute trained genomic neural networks for the first time. Critically, unlike saliency mapping or other game-theory based approaches, APSO is completely network-architecture independent and can be applied to any prediction engine to derive the features that drive inference. Availability and implementation All software and code for data downloading, processing, enhancer inference, eXplainable AI (XAI), and complete figure generation are publicly available on GitHub at https://github.com/EpiGenomicsCode/ChromEnhancer and Zenodo at https://doi.org/10.5281/zenodo.15652797.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5515d8f4201165cbbdeefdd1affb6ab4930eae1" target='_blank'>
              Adversarial attack of sequence-free enhancer prediction identifies chromatin architecture
              </a>
            </td>
          <td>
            Jamil Gafur, Olivia W Lang, William Lai
          </td>
          <td>2025-06-24</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d8f77ef0357a3406cb091f33b54c917e8f5049d" target='_blank'>
              The role of plasmid copy number and mutation rate in evolutionary outcomes.
              </a>
            </td>
          <td>
            Alexander (Olek) Pisera, Chang C. Liu
          </td>
          <td>2025-07-14</td>
          <td>Nature ecology & evolution</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background The selection of replication origins is a defining characteristic of DNA replication in eukaryotes, yet its mechanism in humans has not been well defined. Potential DNA replication origins are licensed through the recruitment of a pair of minichromosome maintenance complexes (MCMs). In yeast, a subset of MCMs is selected for initiation by the SLD3–SLD7 firing factors during G1. Like in yeast, excessive numbers of MCM complexes are loaded onto chromatin during G1 in human cells, but it is unclear how MCMs are selected for firing. Results We examine genomic binding locations for TICRR/TRESLIN and MTBP, the human orthologs for the yeast replication initiation factors Sld3 and Sld7. TRESLIN and MTBP binding patterns are detectable in a G1 synchronized population and have a higher binding signal compared to asynchronously cycling cells. MTBP binds to early-mid replicating regions in an early S population. Our data suggest MTBP is dependent on TRESLIN for proper association with chromatin during G1 but not S phase. We show that TRESLIN and MTBP binding during G1 does not require origins licensed with loaded MCMs. Conclusions We provide evidence for a chromatin binding mechanism of TRESLIN-MTBP during G1 that is dependent on TRESLIN and does not require interactions with licensed origins. MTBP binding location and footprint during S phase differs from that seen in G1, implicating two separate modes of binding. These data highlight binding mechanisms for DNA replication initiation factors in human cells that have diverged from those shown in yeast, suggesting differences in origin selection. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03665-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b74245a8a7e7194eb9255132a18f436ef65a462" target='_blank'>
              Cell cycle-dependent TICRR/TRESLIN and MTBP chromatin binding mechanisms and patterns
              </a>
            </td>
          <td>
            Tyler D. Noble, Courtney G. Sansam, Kimberlie A. Wittig, Blanka Majchrzycka, Christopher L. Sansam
          </td>
          <td>2025-07-07</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Extrachromosomal circular DNA (eccDNA) plays a significant role in cancer development. However, our understanding of its role in normal cells is limited. This study aimed to explore the roles of eccDNA in the differentiation of human bone marrow mesenchymal stem cells (BMSCs). Methods Through circular DNA sequencing (Circle-seq) and computational analysis, we documented eccDNAs in human BMSCs and their differentiation into osteoblasts, adipocytes, and chondrocytes. Next, the assay for transposase-accessible chromatin with high throughput sequencing (ATAC-seq) and RNA sequencing (RNA-seq) data were integrated with Circle-seq data. The roles of phosphatidylinositol 4-kinase alpha (PI4KA) and eccDNA as functional enhancers in human BMSC differentiation were assessed in vitro. Results Our results demonstrate that eccDNA is common in human BMSCs. In the differentiated groups including osteoblasts, adipocytes and chondrocytes, eccDNA-encoded genes presented higher expression compared to the background. We found eccDNA displayed enhanced chromatin accessibility in human BMSCs, which contribute to increased transcription of genes, such as runt-related transcription factor 2 (RUNX2), a key transcription factor for BMSC osteogenesis. We further found PI4KA, an eccDNA-encoded gene, promoted osteogenic differentiation of human BMSCs via the PI3K/AKT/mTOR pathway. Finally, we demonstrated eccDNA as functional enhancers to regulate BMSC differentiation in a dose-dependent manner. Conclusions Our study provides the evidence of eccDNA regulating BMSC differentiation and highlights the roles of eccDNA as transcription template and enhancer in normal cells, which will facilitate future research and clinical applications. Supplementary Information The online version contains supplementary material available at 10.1186/s13287-025-04516-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a9298ef8d097b3fec00680e4438db26b85d21b6" target='_blank'>
              Extrachromosomal circular DNAs in the differentiation of human bone marrow mesenchymal stem cells
              </a>
            </td>
          <td>
            Yuxi Gu, Yidan Song, Shuhua Wang, Jun Liu
          </td>
          <td>2025-07-18</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc739b29aa041a82d43e91798c3e41caf831aa6b" target='_blank'>
              Enhancer Poising Enables Pathogenic Gene Activation by Noncoding Variants
              </a>
            </td>
          <td>
            Ethan W. Hollingsworth, Zhuoxin Chen, Cindy X. Chen, Sandra H. Jacinto, Taryn A Liu, Evgeny Z. Kvon
          </td>
          <td>2025-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83adb5995a58165fece89d2aea822be3362a5535" target='_blank'>
              Epigenetic Subtypes of High-Grade T1 Bladder Cancer Reveal Intra-Tumor Heterogeneity and Distinct Interactions with Tumor Microenvironment
              </a>
            </td>
          <td>
            J. Bellmunt, Yingtian Xie, N. Juanpere, M. Munoz, Shweta Kukreja, Sonsoles Liria Veiga, Rong Li, Xintao Qiu, Yijia Jiang, Alba Font-Tello, Marie Nunez Duarte, Ilana B. Epstein, Silvia Hernández-Llodrà, Marta Lorenzo, Silvia Menendez, T. Choueiri, Myles Brown, H. Long, Paloma Cejas
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 Background: Large-scale genomic profiling has cataloged the prevalence of single base substitution (SBS) mutational signatures associated with the activity of Apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) cytidine deaminases in breast cancer (BC). These mutational signatures are enriched in metastatic BC (mBC) compared to early tumors indicating an association with poor prognosis and a potential function in therapy resistance and disease progression. We sought out to investigate whether APOBEC3 mutational signatures can serve as biomarkers for poor treatment outcomes and if APOBEC3 mutagenesis-driven genomic instability can induce therapy resistance in mBC. Methods: We analyzed SBS mutational signatures in 3,880 BC samples with paired tumor-normal sequencing by the MSK-IMPACT assay using the SigMA algorithm. We utilized the detailed clinical annotation to assess the clinical characteristics of APOBEC3-dominant tumors including survival analyses on endocrine and targeted therapies. We generated cellular models of BC to investigate the molecular drivers of APOBEC3 mutagenesis and its function in promoting therapeutic resistance. We performed whole genome sequencing (WGS) of BC models and paired primary/metastatic patient samples to identify broader genomic alterations mediated by APOBEC3 activity. Results: Building on published results, we found that APOBEC3 mutational signatures were highly prevalent in all subtypes of BC and enriched in metastatic hormone receptor-positive (HR+) and triple-negative breast cancers (TNBC) compared to unmatched primary tumors (p < 0.0001 for HR+/HER2-, p < 0.01 for HR+/HER2+ and TNBC). APOBEC3 mutational signatures were independently associated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor combination therapy in patients with HR+ mBC (HR 1.5, 95% CI 1.2 - 1.8, p < 0.001). Expression of APOBEC3A (A3A) and APOBEC3B (A3B) enzymes generated APOBEC3-associated alterations including single nucleotide variants, copy number alterations (CNAs), and clustered mutations in a deamination-dependent manner, and promoted resistance acquisition of estrogen receptor-positive (ER+) BC cells to agents including an ER degrader and CDK4/6 inhibitors. In HER2+ cells, endogenous A3A-driven APOBEC3 activity was also necessary for faster resistance development against anti-HER2 therapies. WGS analyses of resistant models identified CNA events such as loss of heterozygosity in chromosome 13 exclusively in APOBEC3-positive cells. Upon exposure to the CDK4/6 inhibitor abemaciclib, these cells acquired APOBEC3-context truncating mutations in RB1 tumor suppressor gene, a well-characterized mechanism of resistance. Detailed analyses of WGS of five paired patient samples also highlighted acquired resistance-linked alterations such as PIK3CA E54XK mutations in APOBEC3-dominant tumors, which were corroborated in paired pre/post-treatment samples in our clinical cohort. The acquisition of APOBEC3-context alterations in APOBEC3-dominant samples highlights the causality of APOBEC3 mutagenesis in driving resistance-promoting changes. Lastly, comparison of mutational signatures in the paired cohort demonstrated pre-existence of APOBEC3 signatures in 75% of pre-treatment samples that became APOBEC3-dominant post-treatment illustrating that APOBEC3 mutagenesis can be active during early stages of BC. Conclusions: Our work reveals that APOBEC3 mutational signatures predict poor treatment outcomes of HR+ mBC. We demonstrate that APOBEC3 mutagenesis, primarily through the enzymatic activity of A3A and A3B, drives resistance to endocrine and targeted therapies by causing APOBEC3-context resistance-associated changes. We further show that the presence of APOBEC3 mutagenesis can be detected before therapy exposure and may therefore represent a valuable biomarker and therapeutic target.
 Citation Format: Avantika Gupta, Andrea Gazzo, Pier Selenica, Anton Safonov, Fresia Pareja, Edaise M. da Silva, David N. Brown, Yingjie Zhu, Juber Patel, Juan Blanco-Heredia, Bojana Stefanovska, Michael A. Carpenter, Xin Pei, Denise Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, Britta Weigelt, Nadeem Riaz, Simon N. Powell, Pedram Razavi, Reuben S. Harris, Jorge S. Reis-Filho, Antonio Marra, Sarat Chandarlapaty. APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr GS2-11.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b57625ce2cff9cbb61ff4f9d159b42b1e5eb578" target='_blank'>
              Abstract GS2-11: APOBEC3 mutagenesis induces resistance-promoting genomic alterations in breast cancer
              </a>
            </td>
          <td>
            Avantika Gupta, Andrea M Gazzo, P. Selenica, A. Safonov, F. Pareja, E. D. da Silva, David N Brown, Yingjie Zhu, Juber Patel, J. Blanco-Heredia, B. Stefanovska, Michael A. Carpenter, Xin Pei, D. Frosina, Achim A. Jungbluth, Marc Ladanyi, Giuseppe Curigliano, B. Weigelt, N. Riaz, Simon Powell, P. Razavi, R. Harris, J. Reis-Filho, Antonio Marra, S. Chandarlapaty
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Genetic lesions drive cancer development and progression, and understanding their origins will reveal the mechanisms of carcinogenesis. We showed that murine FAM72A promotes mutagenic DNA repair during antibody maturation by acting as a substrate adaptor of the CTLHMKLN1 E3 ligase to induce the proteasome degradation of Uracil DNA glycosylase 2 (UNG2), a pivotal enzyme of the base excision repair. In humans, the FAM72 gene has expanded to include four paralogues named FAM72A-D. Bioinformatic studies suggested that the human FAM72 genes are overexpressed in a broad range of cancers. However, the functional roles of FAM72A-D in human biology and cancer are unknown. Here, we show that FAM72 family members are minimally expressed in most healthy tissues except for thymus, and that FAM72A, B and D are overexpressed in primary tumorigenic tissues. Human FAM72 expression inversely correlates with UNG2 protein level in human cell lines and primary tumorigenic tissues suggesting that human FAM72 promotes UNG2 degradation. However, only FAM72A is able to bind to and induce UNG2 degradation in human cells. Our results suggest that the ability of FAM72A to induce UNG2 degradation contributes to neoplasia in a variety of cancer types by promoting mutagenic repair of genomic dUs. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-07723-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/369de028cb26608e44c32340a1371447a2c75ba9" target='_blank'>
              FAM72A promotes UNG2 degradation and mutagenesis in human cancer cells
              </a>
            </td>
          <td>
            Yuqing Feng, P. Barbulescu, Chetan K Chana, Melika Shirdarreh, Hong Yang, Lizhen Wu, Sami Mamand, Mohammad Kashem, Amin Zia, Ming Han, Julissa Tsao, Trevor Pugh, David Cescon, David G. Schatz, Frank Sicheri, Alberto Martin, R. Pezo
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) encompasses tumors arising in the colon (CC) and rectum (RC), often treated as a single disease despite emerging evidence of biological divergence. Understanding the molecular differences between CC and RC is critical for improving diagnosis, prognosis, and therapeutic strategies. Methods We performed an integrated genomic and transcriptomic analysis of CC and RC data from The Cancer Genome Atlas (TCGA) to investigate their degree of similarity and observed that these tumors present distinct molecular profiles, which suggest an evolution through divergent pathways. Comparative analyses included copy number alterations (CNAs), somatic mutations, driver gene prediction, differential gene expression, pathway enrichment, and survival analysis. Results Chromosomal analyses revealed that 43% of focal and 77% of large-scale CNAs were specific of CC, while 10.5% and 57% were specific of RC with 8% of mutant genes unique to CC and 0.18% to RC. CC and RC presented distinct profiles of gene mutations, with CC showing significantly higher tumor mutational burden (0.51 muts/Mb vs 0.28 muts/Mb in RC). Distinct mutational signatures were identified, with CC characterized by a higher frequency of PIK3CA, BRAF, and DNAH1 mutations, while RC showed enrichment for TP53 and NRAS mutations. Importantly, analysis of predicted non-canonical driver genes identified ACVR1B, LTBP4, SETD1A as CC-specific drivers and C4BPA, EHD1 as RC-specific drivers, underscoring divergent oncogenic mechanisms. However, the most substantial divergence was observed in transcriptomic profiling, with 56% and 33% of DEGs (in CC and RC, respectively) that were tumor-type specific. Notably, RC tumors segregated into two distinct transcriptional subtypes (Cluster 1 and Cluster 2), with Cluster 1 showed a more heterogeneous Consensus Molecular Subtypes (CMS) distribution, while Cluster 2 enriched in CMS4 (mesenchymal) and CMS3 (metabolic) consensus molecular subtypes. Accordingly, Gene Set Enrichment Analysis revealed CC-specific upregulation of Wnt, MYC, and mTOR signaling pathways, and RC-specific enrichment of GPCR and neuronal development pathways. On the other hand, pseudogene expression was significantly higher in CC, suggesting differential mechanisms of transcriptional dysregulation. Finally, we identified an RC-specific multigene survival signature as a prognostic model involving upregulation of C2CD4B, HSPD1P1, LINC01356, CBX3P9, GATA2-AS1 and downregulation of ATP5F1EP2, HSP90AB3P and SNRPFP1. Conclusions Collectively, our findings provide robust molecular evidence that CC and RC follow divergent oncogenic pathways, emphasizing the need for site-specific biomarker development and therapeutic targeting in colorectal cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12967-025-06908-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab7228806c5ae698ac668178d1fa7e354b1b537" target='_blank'>
              Distinctive chromosomal, mutational and transcriptional profiling in colon versus rectal cancers
              </a>
            </td>
          <td>
            Maria Teresa De Angelis, Antonia Rizzuto, Angela Amaddeo, Carlo Sagnelli, Niccolò Vono, Michela Reda, Valentina Lise, Luca Parrillo, Carmela de Marco, Donatella Malanga, Gianluca Santamaria, Giuseppe Viglietto
          </td>
          <td>2025-08-06</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199bc35f7ab947054a7f2e2f4d2df4bd560fe16b" target='_blank'>
              Conserved cell state dynamics reveal targetable resistance patterns in ovarian high-grade serous carcinoma
              </a>
            </td>
          <td>
            A. Pirttikoski, Laura Gall-Mas, Wojciech Senkowski, David Fontaneda-Arenas, Matias Marin Falco, E. Erkan, J. Hynninen, K. Wennerberg, Anna Vähärautio
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Ovarian high-grade serous carcinoma (HGSC) is characterized by pervasive genomic instability and high inter- and intra-tumor heterogeneity. Approximately half of HGSC tumors harbor homologous recombination deficiency (HRD), rendering them vulnerable to PARP inhibitors and platinum-based chemotherapy. In contrast, patients lacking HRD (HR-proficient, HRP) generally respond poorly to the current therapies. To overcome heterogeneity and identify relevant HGSC subtypes, we characterized the genomic landscape of 640 tumors from 243 patients using whole-genome sequencing. Our chromosomal instability (CIN) signature-based analysis characterized the structural variation landscape and revealed five HGSC subtypes, validated in an independent dataset. Two HRD subtypes, associated with BRCA1- or BRCA2-driven alterations, demonstrated favorable treatment responses. Strikingly, three HRP subtypes emerged, marked by unique structural alterations and gene expression patterns, tumor microenvironment interactions, and different chemotherapy responses. Finally, organoid experiments showed subtype-specific sensitivity to CHK1 inhibition, suggesting prexasertib as a potential targeted treatment for most currently untreatable HRP patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57e62a189c25cb112512c18bbcd2dda397ed67ef" target='_blank'>
              Decoding the Genomic and Functional Landscape of Emerging Subtypes in Ovarian Cancer.
              </a>
            </td>
          <td>
            Giulia Micoli, K. Lavikka, Yilin Li, A. Pirttikoski, D. Afenteva, Wojciech Senkowski, G. Marchi, Anna Vaharautio, T. Muranen, Titta Joutsiniemi, S. Hietanen, A. Virtanen, K. Wennerberg, J. Hynninen, J. Oikkonen, S. Hautaniemi
          </td>
          <td>2025-07-31</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="PARP inhibitors (PARPi), recently introduced for treating metastatic castration-resistant prostate cancer (mCRPC), have heightened interest in molecular profiling for pathogenic aberrations in homologous recombination DNA repair (HRR) genes in all mCRPC patients. Liquid biopsy offers a viable alternative to archival tumor tissue for genetic analysis. In this study, we assessed the feasibility and utility of combining mutational panel sequencing with shallow whole genome sequencing (sWGS) to refine HRR status determination from plasma in prostate cancer (PCa) patients. The mutational profile of 16 HRR genes was assessed in 63 PCa patients: 28.6% of patients harbored putative pathogenic variants in HRR-related genes. A HRR-mutant status was defined for 10 patients (15.8%). Through the integration of sWGS data, plasma samples non-informative about somatic alterations were identified, and germline/somatic origin of HRR mutations was defined. Matched tumor tissue was available for 41 patients, with an 85.7% concordance rate between plasma and tissue mutational analyses. Additionally, we explored the copy number variation (CNV) profile using sWGS and it was found concordant with the literature PCa profiles. Our findings demonstrated that ctDNA analysis through liquid biopsy is a reliable alternative to tissue-based methods for identifying SNVs and CNVs. However, concordance was affected by ctDNA levels in plasma and clonal hematopoiesis. The data highlight the utility of integrating sWGS with targeted mutation analysis for comprehensive molecular profiling of PCa patients. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05384-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad00381f62d746252d19dc78da101336888d0016" target='_blank'>
              Mutational and low-coverage whole genome sequencing identifies actionable DNA repair alterations in prostate cancer plasma DNA
              </a>
            </td>
          <td>
            A. Virga, Milena Urbini, Maurizio Polano, E. Petracci, G. Tedaldi, G. Gurioli, G. Marisi, D. Angeli, Andrea Ambrosini-Spaltro, Giovanni De Luca, Ilaria Cangini, Valentina Zampiga, Giovanna Cenacchi, Giuseppe Toffoli, C. Casadei, M. C. Cursano, V. Conteduca, U. de Giorgi, Paola Ulivi
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="ABSTRACT Background Epigenetic regulatory genes (epiRG) are pivotal in the epigenetic regulation of the human genome, primarily through DNA and histone modifications. These genes are frequently mutated in human cancers, particularly bladder cancer (BC). However, the functional impact of epiRG mutations on patient outcomes remains poorly understood. Methods In this study, we developed gene signatures for the most frequent genomic aberrations of epiRG using The Cancer Genome Atlas Bladder Carcinoma (TCGA‐BLCA) dataset and validated these signatures with independent tumor expression profiles for prognostic relevance. Furthermore, we evaluated the role of these signature scores in the immune system within the tumor microenvironment (TME). Finally, we assessed the correlation between epiRG and global DNA methylation. Results Our results indicated that the inferred aberration‐specific signature scores were more predictive of patient stratification than the genomic aberrations. Notably, certain signature scores were significantly associated with patient progression, whereas others correlated with the tumor immune microenvironment via interactions with the immune system. Patients with mutations had high signature scores in CREBBP‐mut and EP300‐mut, which revealed poor overall survival. Conversely, KDM6A‐mut signatures showed an opposite trend, with low scores linking to favorable prognosis through enhanced immune activity. Also, other epiRG signature scores were strongly correlated with the immune system in TME and successfully predicted patients who responded to immunotherapy. Global methylation analysis revealed that high signature scores of KDM6A‐mut are associated with hypomethylation. Conclusions These findings collectively establish epiRG signature scores as powerful biomarkers that integrate genomic, epigenetic, and immune microenvironment features for improved prognostic prediction in bladder cancer. This integrative approach not only advances our understanding of epigenetic mechanisms in BC but also offers potential for developing innovative prognostic tools and therapeutic strategies tailored to personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21c006331fc69988a65728ddc6ae59549e19f926" target='_blank'>
              Integrative Analysis Reveals the Prognostic Effects of Epigenetic Regulators in Bladder Cancer
              </a>
            </td>
          <td>
            Venugopalareddy Mekala, Yupei Lin, Xiang Wang, Naail Chowdhury, Jianrong Li, Chao Cheng
          </td>
          <td>2025-07-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Aurantiochytrium limacinum holds great promise for producing sustainable single-cell oil as an alternative to fish oil. However, research into its complex biological and biochemical characteristics and efforts toward strain improvement have been hampered by insufficient genetic tools. Until now, genetic transformations of A. limacinum have relied solely on chromosome integration, which is inefficient and prone to insertional mutagenesis and other issues related to genetically modified organisms (GMOs). This paper describes the first centromeric plasmid for A. limacinum. Amplification of this shuttle vector by E. coli enables direct delivery into A. limacinum via electroporation, where it undergoes stable replication and segregation into daughter cells. The key to the stable plasmid maintenance lies in a 500 bp segment derived from chromosome 24 of Phaeodactylum tricornutum. While this segment does not significantly enhance the efficiency of vector transformation, it enables the replication and maintenance of the shuttle vector in the host cell as closed circular DNA. The plasmid from three transformants demonstrates a high segregation efficiency of 96.8 ± 0.3% (n = 3), even in the absence of antibiotic selection. This novel centromeric plasmid considerably enhances the flexibility of genetic manipulations and gene expression in A. limacinum, opening new avenues for its study and industrial application. Key points • First centromeric plasmid developed for genetic transformation in A. limacinum. • The novel plasmid enhances flexibility in genetic manipulation and gene expression. • The plasmid achieves 96.8 ± 0.3% (n = 3) segregation efficiency without antibiotic selection. Supplementary Information The online version contains supplementary material available at 10.1007/s00253-025-13527-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e1a22ec96ed65caa34d28f985f7648b9334af03" target='_blank'>
              Novel centromeric plasmid for stable extrachromosomal gene expression in Aurantiochytrium limacinum
              </a>
            </td>
          <td>
            P. P. Renta, Cian-Huei Syu, Ta-Yu Huang, Yi-Ting Chang, Yu-Feng Liang, Ssu-Ting Chen, Po-Wei Weng, Ming-Chen Hsu, Keng-Hung Lin, Tsu-tung Liu, Anna C-C Jang, Che-Chia Tsao, Han-Jia Lin, Hung-Yun Lin, Yi-Min Chen
          </td>
          <td>2025-07-04</td>
          <td>Applied Microbiology and Biotechnology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Tumors represent dynamically evolving populations of mutant cells, and many advances have been made in understanding the biology of their progression. However, there are key unresolved questions about the conditions that support a cell’s initial transformation, which cannot be easily captured in patient populations and are instead modeled using transgenic cellular or animal systems. Results Here, we use extensive patient atlas data to define common features of the tumor DNA methylation landscape as they compare to healthy human cells and apply this benchmark to evaluate 21 engineered human and mouse models for their ability to reproduce these patterns. Notably, we find that genetically induced cellular transformation rarely recapitulates the widespread de novo methylation of Polycomb regulated promoter sequences as found in clinical samples, but can trigger global changes in DNA methylation levels that are consistent with extensive proliferation in vitro. Conclusions Our results raise pertinent questions about the relationship between genetic and epigenetic aspects of tumorigenesis as well as provide an important molecular reference for evaluating existing and emerging tumor models. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03650-2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b654b38672fde0e6d98d2fd32826d3930c1b68a1" target='_blank'>
              Direct genetic transformation bypasses tumor-associated DNA methylation alterations
              </a>
            </td>
          <td>
            Sara Hetzel, Eran Hodis, Elena Torlai Triglia, Alexander Kovacsovics, Kathleen Steinmann, A. Gnirke, Meiying Cui, Daniel McQuaid, Raha Weigert, Georg Pohl, M. Muzumdar, Serge Leyvraz, Ulrich Keilholz, M. Yaspo, Aviv Regev, H. Kretzmer, Zachary D. Smith, Alexander Meissner
          </td>
          <td>2025-07-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="The identification of recurrent oncogenic drivers has enabled targeted therapeutic strategies for subsets of non-small cell lung carcinoma (NSCLC) and other malignancies. Oncogenic fusions involving the RAS-activating guanine exchange factor (GEF) RASGRF1 are reported in multiple tumors, but their prevalence and genetic heterogeneity remain undefined. Here, we query RNA-seq data from a real-world database of diverse human malignancies and identify 40 tumors with rearrangements involving RASGRF1 or the related RASGRF2 predicted to generate chimeric proteins. Half of these fusions occur in NSCLC, pancreatic cancer, and melanoma and are enriched in tumors without other established driver alterations. A subset of RASGRF fusions contains transmembrane partners, and membrane localization enhances RAS activation and transforming activity. Loss of N-terminal PH1 and DH domains in RASGRF fusions also promotes transformation. Although some fusions lack the PH1 but not the DH domain, our functional assays indicate that loss of the PH1 domain alone is insufficient to drive cellular transformation. Our findings provide insights about the tissue distribution, structural diversity, and oncogenic mechanisms of RASGRF fusions. As cell models driven by these fusions are sensitive to MAPK pathway inhibition, oncogenic RASGRF fusions may represent a therapeutic target in rare molecular subsets of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bb9a75bf2438a8363d43341f77587497989fb9b" target='_blank'>
              Genetic and oncogenic features of RASGRF fusions
              </a>
            </td>
          <td>
            Sreya Das, Daniel S Lenchner, Ellen Jaeger, Lisa Hunihan, Dana F DeSantis, Stamatina Fragkogianni, Karyn Ronski, Frederick H Wilson
          </td>
          <td>2025-07-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Neuroblastoma is a heterogeneous disease where patient stratification is critical for prognosis and treatment decisions and where it recently has been suggested that the presence of telomere maintenance mechanisms (TMM) should be considered in risk stratification. We investigated the utility of DNA methylation-based classification for neuroblastoma diagnostics by analysing 303 tumours samples from two cohorts. We show that of the total number of cases, an average of 90% of the samples classified as neuroblastoma, while 66% also achieved confident classification into the three NB subclasses: “MYCN-type”, “ALT/TERT TMM positive” and “TMM negative”. The tumours classified as MYCN-type showed genomic amplification of MYCN (MNA); however, some MYCN-type cases lacked evident MNA, suggesting that epigenetic states might be influenced by other factors such as activating ALK mutations. Survival analysis indicated similar poor survival probabilities for patients classified as TMM positive or MYCN type, distinct from the inferior survival of TMM-negative cases. All cases, with complementary genomic data available, associated with TMM positivity also presented features associated with telomere lengthening mechanisms, including TERT or ATRX alteration. However, some tumours positive for these features, especially TERT rearrangement, classified as MYCN type rather than TMM positive, indicating that MNA and other mechanisms introduce a methylation pattern that supersede or overlap with pattern imposed by TERT. Chromosomal copy number alterations (CNAs) characteristic of methylation subclasses were identified, including 1p deletion and 17q gain in MYCN type and combinations of 11q loss, 3p loss, 7q gain, and 17q gain in TMM-positive cases, highlighting the potential of the methylation arrays to replace SNP arrays for prognostic genomic assessments. Our study demonstrates that DNA methylation-based classification stratifies neuroblastoma into clinically relevant subgroups, aiding diagnostic and prognostic decisions, although discrepancy between genomic features and methylation classification does occur. The interplay between genomic alterations and methylation patterns could give clues into the discrepancy and underscores the complexity of neuroblastoma biology and the need for further research and validation of clinical outcomes of the patients in the respective subclasses. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01936-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43f713d2afc90db793e049e383b6f7ea1cd34f79" target='_blank'>
              DNA methylation profiling for molecular classification of neuroblastoma
              </a>
            </td>
          <td>
            Maja Löfgren, A. Djos, Shiva Rezaei, Medha Suman, P. Kogner, Tommy Martinsson, S. Fransson, Helena Carén
          </td>
          <td>2025-07-27</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Abstract The multitude of DNA lesion types, and the nuclear dynamic context in which they occur, presents a challenge for genome integrity maintenance as this requires the engagement of different DNA repair pathways. Specific “repair controllers” that facilitate DNA repair pathway crosstalk between double-strand break (DSB) repair and base excision repair (BER) and that regulate BER protein engagement at lesion sites have yet to be identified. Here, we find that DNA polymerase β (Polβ), crucial for BER, is ubiquitylated in a BER complex-dependent manner by TRIP12, an E3 ligase that partners with UBR5 to restrain DSB repair signaling. Furthermore, we find that TRIP12, but not UBR5, controls cellular levels and chromatin loading of Polβ. Required for Polβ foci formation, TRIP12 influences Polβ involvement after radiation-induced DNA damage, a process regulated by TRIP12-mediated ubiquitylation of Polβ. Notably, excessive TRIP12-mediated engagement of Polβ affects DSB formation and radiation sensitivity, underscoring its role in promoting precedence for BER over DSB repair. The herein discovered function of TRIP12, in the governance of Polβ-directed BER, supports a role for TRIP12 in assuring BER lesion removal at complex DSB sites to optimize DSB repair at the nexus of DNA repair pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a976cd6688072a28607e4b25adebc5f7fdda3bbf" target='_blank'>
              TRIP12’s role in the governance of DNA polymerase β involvement in DNA damage response and repair
              </a>
            </td>
          <td>
            B. Inanc, Qingming Fang, Wynand P. Roos, Joel F. Andrews, Xuemei Zeng, Jennifer Clark, Jianfeng Li, Nupur B. Dey, Md Ibrahim, Peter Sykora, Zhongxun Yu, Charlotte R Pearson, Andrea Braganza, Marcel Verheij, Jos Jonkers, Nathan A. Yates, C. Vens, Robert W. Sobol
          </td>
          <td>2025-06-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Background Understanding the role of cancer hotspot mutations is essential for unraveling mechanisms of tumorigenesis and identifying therapeutic vulnerabilities. Correcting cancer mutations with base editing is a novel, yet promising approach for investigating the biology of driver mutations. Results Here, we present a versatile platform to investigate the functional impact of cancer hotspot mutations through adenine base editing in combination with transcriptomic profiling. Using this approach, we correct TP53 hotspot mutations in cancer cell lines derived from diverse tissues, followed by mRNA sequencing to evaluate transcriptional changes. Remarkably, correcting these mutations not only reveals the dependency on mutant allele expression but also restores highly conserved tumor-suppressive transcriptional programs, irrespective of tissue origin or co-occurring mutations, highlighting a shared p53-dependent regulatory network. Our findings demonstrate the utility of this base editing platform to systematically interrogate the functional consequences of cancer-associated mutations and their downstream effects on gene expression. Conclusions This work establishes a robust framework for studying the transcriptional dynamics of cancer hotspot mutations and sheds light on the conserved biological processes reinstated by p53 correction, offering potential avenues for future targeted therapies. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03667-7.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d441d58b01f4910f8c55a860db9d7cbefe24b5" target='_blank'>
              A base editing platform for the correction of cancer driver mutations unmasks conserved p53 transcription programs
              </a>
            </td>
          <td>
            Pascal Wang, Rituparno Sen, Frank Buchholz, Shady Sayed
          </td>
          <td>2025-07-22</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7238ca759d86a62210ebf70bb830d0369e9ff71e" target='_blank'>
              Excised DNA circles from V(D)J recombination promote relapsed leukaemia.
              </a>
            </td>
          <td>
            Zeqian Gao, James N. F. Scott, Matthew P Edwards, Dylan Casey, Xiaoling Wang, Andrew D Gillen, S. Ryan, Lisa J Russell, Anthony V Moorman, Ruth de Tute, Catherine Cargo, Anthony M Ford, D.R. Westhead, J. Boyes
          </td>
          <td>2025-08-06</td>
          <td>Nature</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background. Medulloblastoma (MB) prognosis and response to therapy depend largely on genetic changes in tumor cells. Many genes and chromosomal abnormalities have been identified as prognostic factors, including amplification of MYC oncogenes, gains in 1q and 17q, deletions in 10q and 21p, or isochromosomes 17 (i(17)(q10)). The frequency of these abnormalities varies greatly between ethnic populations, but the frequency of specific abnormalities, such as MYCC and MYCN amplification, 17q gain, and deletions, in the Russian population is unknown. Objective: The aim is to study the frequency of MYCC and MYCN amplifications, 17q gain, and 17p deletion and determine their prognostic value in Russian patients with MB. Methods. This study was performed on MB cells obtained from 18 patients (12 boys and 6 girls, aged between 3 months and 17 years, with a median age of 6.5 years). Determination of cytogenetic aberrations was carried out using FISH assays with MYCC-SO, MYCN-SO, and MYCN-SG/cen2 probes, as well as cen7/p53 dual color probes and PML/RARα dual color probes (Abbott Molecular, USA). One-way ANOVA and Fisher’s F-test were used to compare the two groups. The differences were considered significant when p < 0.05. Results. In 77.7% of patients (14/18), the classical type of MB was present; in 16.7% (3/18), desmoplastic type; and in 5.6% (1/18), nodular desmoplasic types of neoplasms. Amplification of MYC genes was detected in 22.2% of Russian patients (n = 4 out of 18). Patients with MYC amplification had the worst overall survival (OS: 0% vs. 68%, p = 0.0004). Changes on the 17th chromosome were found in 58.3% of patients. Deletion of 17p occurred in 23.1%, and gain of 17q occurred in 46.2%. There were no significant differences in OS, clinical signs, or the presence of additional 17q material or 17p deletion among patients with MB. Conclusions: Amplification of the MYC gene is a predictor of poor overall survival to therapy and a high risk of metastatic relapse. This allows us to more accurately stratify patients into risk groups in order to determine the intensity and duration of therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5f5e5b1eac08421c304a6da77fd1d4ce4707e64" target='_blank'>
              The Prognostic Value of Amplification of the MYCC and MYCN Oncogenes in Russian Patients with Medulloblastoma
              </a>
            </td>
          <td>
            A. Chernov, Ekaterina Batotsyrenova, Sergey N Zheregelya, S. Pyurveev, V. Kashuro, Dmitry Ivanov, Elvira Galimova
          </td>
          <td>2025-07-27</td>
          <td>Diseases</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 DNA repair and autophagy are distinct biological processes vital for cell survival. Although autophagy helps maintain genome stability, there is no evidence of its direct role in the repair of DNA lesions. We discovered that lysosomes process Topoisomerase 1-cleavage complexes (TOP1cc) DNA lesions in human cancer cell. TOP1 resolves DNA topological stress ahead of DNA replication and transcription. If not promptly repaired, TOP1cc hinder the progression of DNA replication and transcription fostering genomic instability. Thus, stabilising TOP1cc with TOP1 poisons, such as Camptothecin or its analogue, is highly effective and widely used in cancer therapy. Proteolysis of TOP1cc by the proteasome is described at excessively high doses of CPT that are not relevant in a clinical setting, calling into question the relevance of the proteasome role in TOP1cc repair for cancer therapy and patient response. Here we described that repair of TOP1cc induced by clinically relevant dose of Camptothecin depends on selective autophagy. Proteomics, sequencing, biochemistry techniques and live imaging all demonstrate the lysosomal uptake of TOP1cc upon induction of replication stress. Selective degradation of TOP1cc by autophagy directs DNA damage repair and cell survival. TOP1cc are exported from the nucleus to lysosomes through transient alteration of the nuclear envelope and independent of the proteasome. Mechanistically, the autophagy receptor TEX264 acts as a TOP1cc sensor at DNA replication forks, triggering TOP1cc processing by the p97 ATPase and mediating the delivery of TOP1cc to lysosomes in an MRE11 nuclease and ATR kinase-dependent manner. We found an evolutionarily conserved role for selective autophagy in DNA damage repair that enables cell survival, protects genome stability, and is clinically relevant for colorectal cancer patients.



 Pauline Lascaux, Gwendoline Hoslett, Sara Tribble, Ivan Antičević, Cecile Otten, Ignacio Torrecilla, Yichen Zhao, Wei Song, Cristiano Peron, Giulio Deangeli, Enric Domingo, James Bancroft, Loïc Carrique, Errin Johnson, Alvin Wei Tian Ng, Joanne Ngeow, Nuno Raimundo, Tim Maughan, Marta Popović, Ira Milošević, and Kristijan Ramadan. TEX264 DRIVES SELECTIVE AUTOPHAGY OF DNA LESIONS TO PROMOTE DNA REPAIR AND CELL SURVIVAL [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P11.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ada9e8b0a0e36cefc3f3e65806469a480324003" target='_blank'>
              Abstract P11: TEX264 DRIVES SELECTIVE AUTOPHAGY OF DNA LESIONS TO PROMOTE DNA REPAIR AND CELL SURVIVAL
              </a>
            </td>
          <td>
            Pauline Lascaux, Gwendoline Hoslett, Sara Tribble, Ivan Anticevic, Cecile Otten, Ignacio Torrecilla, Yichen Zhao, Wei Song, Cristiano Peron, Giulio Deangeli, Enric Domingo, James Bancroft, Loïc Carrique, Errin Johnson, Alvin Wei Tian Ng, Joanne Ngeow, Nuno Raimundo, Tim Maughan, Marta Popović, Ira Milošević, Kristijan Ramadan
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract G-quadruplexes (G4s) are prevalent at promoters and superenhancers (SEs), exclude nucleosomes, and recruit transcription factors. This study sought to determine whether the nucleosome exclusion affects the recruitment of the SE marker BRD4, which typically binds to acetylated histones and facilitates SE–promoter contacts via the phase separation-dependent mechanism. Analyses of the available whole-genome data revealed that SEs with the highest G4 density were depleted of nucleosomes but not of BRD4. This led us to test the possibility of histone-independent BRD4 maintenance at G4-rich SEs. A typical SE G4 destabilized a nearby nucleosome in vitro and, unlike B-DNA, bound weakly to BRD4 bromodomains. Similar to an acetylated nucleosome, the G4 promoted phase separation in BRD4 solutions. This effect was not altered by the histone competitor JQ1. However, it was attenuated by two known G4 ligands, suggesting that they could disrupt SE–promoter communication in cells. Consistently, these ligands downregulated several genes regulated by G4-rich SE-contacting promoters more efficiently than they did SE-independent genes. Our findings underscore the significance of G4-rich SEs as transcriptional regulators and provide new insights into their organization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ca6b9e0a4bd9850a35faa9e07c7255af9d5b35c" target='_blank'>
              G-quadruplexes as potential traps for superenhancer marker BRD4: ligand-sensitive binding and co-separation in vitro
              </a>
            </td>
          <td>
            Iuliia Pavlova, Olga M. Ivanova, Mikhail S Iudin, Anastasiya V Surdina, Nikolay A Barinov, M. E. Bogomiakova, Sergey D Oreshkov, Z. Shenkarev, V. Severov, Dmitriy V Klinov, Victoria O Shender, A. Bogomazova, M. Lagarkova, A. Varizhuk, Vladimir B Tsvetkov
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Endogenized giant viruses are emerging as major contributors to the genome evolution of microbial eukaryotes, with both degraded and fully functional latent viruses being found integrated in diverse lineages. The mechanisms that determine the fate of viral integrants are poorly understood, however. Acanthamoeba is a unicellular eukaryote known for undergoing lateral gene transfer (LGT) with viruses. Here we have leveraged chromosome-scale assemblies of two strains of Acanthamoeba, Neff and C3, to investigate the genomic mechanisms that mediate the fate of viral integrations in eukaryotic genomes. Results Viral integrations in the C3 and Neff genomes are largely non-overlapping and disproportionately found in sub-telomeric regions. Multiple partial copies of these insertions are found throughout the Neff genome, but they are not expressed, do not obviously encode functions associated with their own mobility, and are colonized by host mobile elements. Viral regions are hypermethylated and highly condensed, suggesting that the expression of recently acquired viral DNA is suppressed in heterochromatic regions. Conclusions We propose a model for the trajectory of viral sequences in Acanthamoeba: (i) integration of DNA from giant viruses, (ii) epigenetic suppression of the viral DNAs, allowing them to persist in the genome, and (iii) deterioration of viral genomes by point mutation, mobile element colonization, and intra- and inter-chromosomal recombination. Viral integrations in Acanthamoeba spp. are transient and may not have long-lasting effects on the fitness of the amoeba. Our work highlights the importance of host genome dynamics and epigenetic silencing for understanding the evolution of endogenized viral elements. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02280-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7f54d3754da37f780fd27a13a0f10078e856d51" target='_blank'>
              Epigenetic silencing and genome dynamics determine the fate of giant virus endogenizations in Acanthamoeba
              </a>
            </td>
          <td>
            Cédric Blais, Morgan J. Colp, Luke A. Sarre, Alex de Mendoza, John M. Archibald
          </td>
          <td>2025-07-01</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The discovery of oncogenes and tumor suppressor genes provided important insights into the molecular pathogenesis of cancer but also revealed some contradictions in the prevailing somatic mutation theory. The evidence that noncoding RNAs (ncRNAs) form an elaborate network that regulates the flow of genetic information in eukaryotic cells offers an explanation for the inconsistencies. ncRNAs comprise a wide variety of molecules that interact with one another as well as with other RNAs, DNA, and proteins, over whose activities they exert a regulatory influence. The outcome of the dynamic interactions of the cell’s biomolecules is the emergence of higher-order states of equilibrium, called attractor states, which correspond to the gene-expression configurations of distinct cell types. Attractor states are relatively stable systems, but they are susceptible to perturbation by a disturbing force, such as mutation. Mutations that disrupt the ncRNA network can enable the cell to undergo a state transition towards a potentially neoplastic one. This is the crux of tumorigenesis. An inquiry into the architecture of the ncRNA network and its role in tumorigenesis is required to complement our knowledge of the well-known cancer genes as well as serve as a guide in the design of new anticancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/751cbcc8f9af147a3e904b8dfcf8de5075149f58" target='_blank'>
              The Primary Role of Noncoding RNA in the Pathogenesis of Cancer
              </a>
            </td>
          <td>
            Amil Shah
          </td>
          <td>2025-06-30</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Structural maintenance of chromosomes (SMC) complexes organize genomes by extruding DNA loops, while replisomes duplicate entire chromosomes. These essential molecular machines must collide frequently in every cell cycle, yet how such collisions are resolved in vivo remains poorly understood. Taking advantage of the ability to load SMC complexes at defined sites in the Bacillus subtilis genome, we engineered head-on and head-to-tail collisions between SMC complexes and the replisome. Replisome progression was monitored by genome-wide marker frequency analysis, and SMC translocation was monitored by time-resolved ChIP-seq and Hi-C. We found that SMC complexes do not impede replisome progression. By contrast, replisomes restrict SMC translocation regardless of collision orientations. Combining experimental data with simulations, we determined that SMC complexes are blocked by the replisome and then released from the chromosome. Occasionally, SMC complexes can bypass the replisome and continue translocating. Our findings establish that the replisome is a barrier to SMC-mediated DNA-loop extrusion in vivo, with implications for processes such as chromosome segregation, DNA repair, and gene regulation that require dynamic chromosome organization in all organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/686025c7ec346de8dd0c0dabfe2ed3d2aaa2a6a1" target='_blank'>
              Replisomes restrict SMC translocation in vivo
              </a>
            </td>
          <td>
            Qinchan Liao, Hugo B. Brandão, Zhongqing Ren, Xindan Wang
          </td>
          <td>2025-08-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background DNA methylation plays a crucial role in cancer development and progression and has been linked to genetically and clinically distinct tumor classes, including IDH-mutated and IDH-wildtype adult-type diffuse gliomas. Here, we identify a CpG-island methylator phenotype (CIMP) that characterizes the receptor tyrosine kinase 2 (RTK2) subtype of IDH-wildtype glioblastoma. Results This RTK2-CIMP affects genomic locations and cell functions distinct from those of IDH mutation-associated IDH-CIMP and suppresses the expression of its target genes. The RTK2-CIMP-region chromatin is characterized by a combination of repressive and activating marks, including polycomb-associated H3K27me3 and enhancer-associated H3K4me1, consistent with DNA methylation-mediated silencing of genes with bivalent-state promoters in neural progenitor cells. Functionally, RTK2-CIMP affects neuronal lineage genes and is significantly associated with astrocyte-like glioblastoma, suggesting that RTK2-CIMP is an epigenetic signature of the astrocyte-like cell state. Furthermore, we demonstrate that RTK2-CIMP can be induced by genetic manipulation in glioblastoma cells. Conclusions Our results suggest that RTK2-CIMP is a key contributor to cell-state plasticity in glioblastoma. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03670-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94bee397b61a1f7011617f82ad34a983c326fca3" target='_blank'>
              A new IDH-independent hypermethylation phenotype is associated with astrocyte-like cell state in glioblastoma
              </a>
            </td>
          <td>
            Ana Luisa Costa, Daria Doncevic, Yonghe Wu, Lin Yang, Ka-Hou Man, Anna-Sophie Spreng, Hannah Winter, M. Fletcher, B. Radlwimmer, Carl Herrmann
          </td>
          <td>2025-07-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e39cce89c2554835e6bd04e09986a82ce912ced" target='_blank'>
              The CIC::DUX4 oncoprotein maintains DNA integrity through direct regulation of the catalytic subunit of DNA polymerase epsilon (POLE).
              </a>
            </td>
          <td>
            Zeinab Kosibaty, Cuyler Luck, Ross A. Okimoto
          </td>
          <td>2025-08-04</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Despite being a consolidated practice in modern cancer genomics, mutational signature analysis poses various challenges. First, determining the number of signatures is a complex task and depends on heuristics. Second, several signatures lack a clear etiology, raising concerns about whether they result from computational artifacts or actual mutagenic processes. Last, approaches for signature assignment are highly influenced by the catalogue of signatures used for the analysis. To overcome these limitations, we introduce RESOLVE (Robust EStimation Of mutationaL signatures Via rEgularization), a framework designed for the efficient extraction, assignment, and confidence estimation of mutational signatures. RESOLVE enables the stratification of cancer genomes according to the active mutational processes, providing insights into those that are consistently shared among various cancer types. We applied RESOLVE to 20 000 samples from adult and pediatric cancers, providing a comprehensive characterization of subtypes associated with specific mutational processes and alterations in driver genes. Our analysis demonstrates that RESOLVE accurately fits observed mutations with a smaller set of signatures compared to existing catalogues, suggesting the existence of a limited number of dominant mutational processes in cancer genomes. Clustering analysis revealed distinct patient groups characterized by specific signatures, with significant associations between certain signatures and patient prognosis. Additionally, we identified strong associations between signatures and driver gene mutations, offering insights into cancer subtype mechanisms and evolution. Our findings highlight the efficiency of RESOLVE and its potential impact on personalized cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f33c3f703237609301199e20410a7b8d03b3cefb" target='_blank'>
              Comprehensive analysis of mutational processes across 20 000 adult and pediatric tumors
              </a>
            </td>
          <td>
            M. Villa, Federica Malighetti, Luca De Sano, Alberto Maria Villa, N. Cordani, A. Aroldi, M. Antoniotti, G. Caravagna, Alex Graudenzi, R. Piazza, L. Mologni, Daniele Ramazzotti
          </td>
          <td>2025-07-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Oncogene expression can cause replication stress (RS), leading to DNA double-strand breaks (DSB) that require repair through pathways such as homologous recombination, non-homologous end-joining, and microhomology-mediated end-joining (MMEJ). Cyclin D1 (encoded by CCND1) is a well-known oncoprotein overexpressed in cancer; however, its role in RS is unknown. Using mantle cell lymphoma (MCL) as a naturally occurring model of cyclin D1 overexpression, we examined its impact on RS and DSB-repair mechanisms. Cyclin D1 overexpression elevated RS, increased DNA damage, especially during mitosis, and caused specific upregulation of MMEJ. Furthermore, cyclin D1 activates the polymerase theta (POLQ) transcription by binding its promoter loci, driving POLΘ-mediated MMEJ that is essential to withstand cyclin D1-induced RS. Moreover, concurrent ATM deficiency further intensifies RS, enhances POLQ expression and heightens reliance on MMEJ mediated DNA damage repair. Consequently, inhibition of POLΘ in cyclin D1-overexpressed settings further exacerbates RS, causing single-strand DNA gap accumulations and chromosomal instability, ultimately leading to apoptosis, an effect amplified in ATM-deficient cells. Targeting MMEJ via POLΘ inhibition is, therefore, an effective strategy in the context of cyclin D1 overexpression and ATM deficiency and may provide a unique therapeutic approach for treating MCL and other malignancies characterized by similar alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e92152fc5a144bdda47d831e771bd37fa31380d" target='_blank'>
              Cyclin D1 overexpression induces replication stress and microhomology-mediated end-joining dependence in mantle cell lymphoma.
              </a>
            </td>
          <td>
            J. Abeykoon, S. Asada, G. Zhu, Y. Hirohashi, L. Moreau, Divya R Iyer, S. Mukkavalli, K. Parmar, G. Zambrano, Lige Jiang, Dongni Yi, M. Manske, K. Gwin, Rebecca L. King, J. Cerhan, Xiaosheng Wu, Zhenkun Lou, G. Shapiro, T. Witzig, A. D'Andrea
          </td>
          <td>2025-07-03</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="The DNA double-strand breaks (DSBs) generated by exogenous and endogenous factors are repaired by two pathways: homologous recombination (HR) and non-homologous end-joining (NHEJ). These two pathways compete for DSB repair, and the choice of pathway depends on the context of the DNA lesion, the stage of the cell cycle, and the ploidy in the yeast Saccharomyces cerevisiae. However, the mechanistic details of the DSB repair pathway choice and its consequences for S. cerevisiae genome stability remain unclear. Here, we present PCR-based and cell-based assays as well as data analysis methods to quantitatively measure the efficiency of HR and NHEJ at DSBs in S. cerevisiae. An intermolecular recombination assay between plasmid and chromosomal DNA involving G-quadruplex DNA and a “suicide-deletion” assay have been utilized to evaluate the efficiency of HR and NHEJ, respectively. These streamlined protocols and optimized growth conditions can be used to identify the NHEJ- and HR-deficient S. cerevisiae mutant strains. Key features • Optimized protocol for intermolecular recombination involving G-quadruplex-forming DNA sequences derived from recombination hotspots in S. cerevisiae. • Optimized protocol to quantify the efficiency of NHEJ in S. cerevisiae. • Quantitative assessment of HR and NHEJ efficiency and data validation. This protocol is used in: eLife (2024), DOI: 10.7554/eLife.96933.3">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1641acc658df2d6862f6dc5893be4745d0b3fb7" target='_blank'>
              Assessing the Efficiency of Double-Strand Break Repair Mediated by Homologous Recombination and Non-homologous End-Joining Pathways in Saccharomyces cerevisiae
              </a>
            </td>
          <td>
            S. Badugu, Kshitiza M Dhyani, K. Muniyappa
          </td>
          <td>2025-07-05</td>
          <td>Bio-protocol</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Acremonium chrysogenum is a notable filamentous fungus recognized for its essential contribution to the pharmaceutical sector through the biosynthesis of cephalosporin C (CPC). CPC functions as a key intermediate in the biosynthesis of β-lactam antibiotics, which are employed to combat bacterial infections. This study successfully generated a telomere-to-telomere (T2T) chromosome-scale genome sequence for A. chrysogenum, combining BGI short reads, PacBio HiFi long reads, and Hi-C technology. This genome sequence contained eight complete chromosomes (29.00 Mb) and a circular mitochondrial genome (27.27 kb), featuring an N50 length of 3.87 Mb. Repetitive elements accounted for 9.65% of genomic content, and a total of 7,745 genes involved in protein coding were annotated. This well-assembled reference genome of A. chrysogenum serves as an important foundation for elucidating the biosynthetic pathway of cephalosporin C and for molecular breeding. Furthermore, it offers valuable insights into chromosome organization, genome evolution, and regulatory mechanisms, facilitating future advancements in antibiotic research and fungal biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12587d2631e390bb270a6373edb55bfcb1c7374e" target='_blank'>
              The telomere-to-telomere chromosome-scale genome assembly of Acremonium chrysogenum
              </a>
            </td>
          <td>
            Chao Han, Yiping Zhang, Hengyu Liang, Minliang Chen, Jiahuang Li, Zichun Hua
          </td>
          <td>2025-08-07</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Plasmids constitute a key tool in synthetic biology, providing a versatile framework for various research and industrial ventures. An essential determinant of plasmid functionality is its copy number, impacting both protein production rates and host cell metabolic burden. However, currently there is no model that can computationally predict the plasmid copy number from the sequence of its origin of replication (ORI). We present a novel software solution tailored to simplify plasmid copy number design, poised to redefine plasmid engineering workflows. At the heart of our tool lies a comprehensive machine learning model, informed by numerous features extracted from the ORI. This computational model emphasizes the importance of promoter strength and the RNA folding dynamics of regulatory elements within the ORI. Additionally, we detail a robust protocol for the efficient manipulation of plasmid ORIs used to validate our model's predictive capabilities. This innovation represents a paradigm shift in plasmid-centric methodologies, offering unprecedented avenues for advancement in synthetic biology research and industrial applications. The software is available at: https://pcn-gradient.vercel.app/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef47a6451c0062151b76e5642c026f3faf686c38" target='_blank'>
              Deciphering plasmid replication dynamics: a computational approach to predict ColE1-like plasmid copy number.
              </a>
            </td>
          <td>
            Rinat Saban, Ofri Tfilin, David Haggiag, Yam Tawachi, Matan Arbel, T. Tuller
          </td>
          <td>2025-07-01</td>
          <td>Journal of the Royal Society, Interface</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Epigenetic traits are persistent cellular and organismal properties that do not result from changes in DNA sequence. One such property involves transmission of chromosomes, which entails the formation of highly specialized chromatin structures, the kinetochores, on selected chromosomal regions, called centromeres. Centromere function is essential and centromeres are determined epigenetically by the deposition of a variant histone H3 CENP-A (CENH3 in plants). Either reduced or ectopic function alone leads to genome instability, decreased fitness, aneuploid syndromes, and cancer. At times, however, centromeres malfunction in an apparently programmed mode. This is exemplified by a peculiar centromeric syndrome involving selective elimination of a chromosome set, which can affect a wide range of organisms, including plants. Over half a century ago, plant geneticists described this syndrome in interspecific crosses of barley. Building on their work, we examine the growing understanding of how CENH3 function can be modified to affect epigenetic regulation of centromeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024266b37112d5916123a1fab820a516c26c7be4" target='_blank'>
              Pioneers of chromosome elimination
              </a>
            </td>
          <td>
            Luca Comai, M. P. A. Marimuthu
          </td>
          <td>2025-07-29</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genetic alterations alone cannot account for the diverse phenotypes of cancer cells. Even cancers with the same driver mutation show significant transcriptional heterogeneity and varied responses to therapy. However, the mechanisms underpinning this heterogeneity remain under-explored. Here, we find that novel enhancer usage is a common feature in acute lymphoblastic leukemia (ALL). In particular, KMT2A::AFF1 ALL, an aggressive leukemia with a poor prognosis and a low mutational burden, exhibits substantial transcriptional heterogeneity between individuals. Using single cell multiome analysis and extensive chromatin profiling, we reveal that much transcriptional heterogeneity in KMT2A::AFF1 ALL is driven by novel enhancer usage. By generating high resolution Micro Capture-C data in primary patient samples, we identify patient-specific enhancer activity at key oncogenes such as MEIS1 and RUNX2, driving high levels of expression of both oncogenes in a patient-specific manner. Overall, our data show that enhancer heterogeneity is highly prevalent in KMT2A::AFF1 ALL and may be a mechanism that drives transcriptional heterogeneity in cancer more generally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d523b069d65585980e3b6635ddde12c81e15186" target='_blank'>
              Enhancer heterogeneity in acute lymphoblastic leukemia drives differential gene expression in patients.
              </a>
            </td>
          <td>
            Alastair L. Smith, N. Denny, Catherine Chahrour, Kim Sharp, Marta Arachi, A. M. Dopico-Fernandez, N. Elliott, Joe R Harman, Thomas Jackson, H. Geng, Owen Smith, Jonathan Bond, Irene Roberts, Ronald W W Stam, Nicholas T Crump, James O. J. Davies, Anindita Roy, Thomas A. Milne
          </td>
          <td>2025-07-29</td>
          <td>Blood</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="R-loops, chromatin structures containing DNA-RNA hybrids with displaced single-stranded DNA, play crucial roles in various cellular processes. Their formation is influenced by factors such as DNA topology, RNA stability, and the presence of GC-rich regions. However, excessive or uncontrolled R-loop accumulation can threaten genomic stability, leading to DNA damage, particularly double-strand breaks. To preserve genome integrity, cells have developed mechanisms to regulate R-loop formation and resolution. Dysregulation of these processes is linked to several diseases, including cancer and neurodegenerative disorders. In this review, we will explore the dynamics of R-loop formation and resolution and how they are detected, their roles in DNA damage and repair, and how their dysregulation may lead to immune responses and disease pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3728d9fd65e0fa0dfc3340b92946a313355d5d1" target='_blank'>
              The paradox of R-loops: guardians of the genome or drivers of disease?
              </a>
            </td>
          <td>
            Shachinthaka D Dissanayaka Mudiyanselage, Phillip Wulfridge, K. Sarma
          </td>
          <td>2025-08-06</td>
          <td>Genome research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Homologous recombination is a high-fidelity DNA repair mechanism essential for maintaining genome stability. Impairment of this pathway, often due to BRCA1 or BRCA2 inactivation, leads to homologous recombination deficiency (HRD), forcing cells to rely on error-prone mechanisms for repairing DNA double-strand breaks, such as nonhomologous or microhomology-mediated end joining. HRD is a clinically important biomarker, particularly in breast and ovarian cancers, as it predicts responsiveness to platinum-based chemotherapies and PARP inhibitors. However, current tests in the clinical setting, mostly based on targeted panel sequencing, lack specificity and lead to a substantial number of false positives. In contrast, whole-genome sequencing, despite its high accuracy, remains largely confined to research because of high costs and logistical constraints. In this issue of Cancer Research, Abbasi and colleagues present HRProfiler, a machine learning-based tool that accurately detects HRD using whole-exome sequencing (WES) data, which is increasingly accessible in clinical oncology. Notably, it demonstrates improved sensitivity in the WES setting compared with existing tools, such as HRDetect and SigMA. As WES continues to gain traction, HRProfiler offers a promising step toward democratizing HRD detection and enabling more precise, genomics-guided treatment strategies. This article is part of a special series: Driving Cancer Discoveries with Computational Research, Data Science, and Machine Learning/AI. See related article by Abbasi et al., p. 2504.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab81bfb225ffd3e5881d30b8dbaccdbb884d86b5" target='_blank'>
              Leveraging Whole-Exome Sequencing and Mutational Signatures to Detect Homologous Recombination Deficiency in Cancer.
              </a>
            </td>
          <td>
            Joonoh Lim, Young Seok Ju
          </td>
          <td>2025-07-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2222f4aa3745d3827861310375ad2ef6308f3da" target='_blank'>
              Mapping genetic modifiers of epimutation rates reveals a punctuated-equilibrium model of CG methylome evolution
              </a>
            </td>
          <td>
            Zhilin Zhang, Wilma Wanney, Yangyang Xu, J. Zicola, Angela M. Hancock, Robert J. Schmitz, F. Johannes
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has emerged as a novel biomarker for cancer detection due to its tumor-specific amplification and stable structure in circulation. However, its clinical application is hindered by extremely low abundance in biofluids and the lack of robust detection techniques. To address this, we screened for tumor-associated eccDNA biomarkers and developed NPCC (Nested PCR-CRISPR/Cas12a), a novel method combining nested PCR for ultrasensitive amplification with CRISPR/Cas12a for sequence-specific detection. The assay employs two rounds of junction-specific PCR to enrich eccDNA, followed by CRISPR/Cas12a-mediated cleavage guided by target-specific crRNA. Validation using synthetic circular DNA standards demonstrated a limit of detection (LoD) of 10-6 fM, representing a >100-fold improvement over conventional PCR, with no cross-reactivity to linear or genomic DNA fragments. In plasma samples from 88 cancer patients, NPCC successfully detected multiple tumor-specific eccDNAs, including the hepatocellular carcinoma marker eccDNA-HCC-1 (AUC = 0.8977). NPCC overcomes key technical barriers in liquid biopsy, offering a cost-effective, highly sensitive, and specific platform for noninvasive cancer diagnostics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aeca77128a740c764bbb2a48a28ef00f3cfa9edd" target='_blank'>
              Ultrasensitive eccDNA Detection for Tumor Diagnostics by Using CRISPR/Cas12a-Coupled Nested PCR.
              </a>
            </td>
          <td>
            Gege Liu, Junlu Wu, Bin Yang, J. Kong, Yaran Li, Chen Chen, Jiayi Hu, Zujun Sun, Yanqiang Hou, Ruixin Sun, Dong Li
          </td>
          <td>2025-07-31</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The Mre11 nuclease, part of the conserved MRX complex involved in the repair of DNA double-strand breaks (DSBs), is also essential to initiate meiotic recombination in budding yeast by promoting Spo11-induced DSBs. Recruitment of Mre11 to meiotic DSB sites depends on Rec114-Mei4 and Mer2 (RMM) that organize the meiotic DSB machinery by a mechanism involving biomolecular condensation. Here, we explored the role of Mre11 during meiosis and its relationship to RMM condensation. We show that both Mre11 and MRX complexes form DNA-dependent, hexanediol sensitive condensates in vitro. In vivo, Mre11 assembles into DNA damage-dependent foci in vegetative cells and DSB-independent foci in meiotic cells. In vitro condensates and in vivo foci both depend on the C-terminal intrinsically-disordered region (IDR) of Mre11. Importantly, while the Mre11 IDR is dispensable for vegetative DNA repair it is essential during meiosis. The C-terminus of Mre11 forms a short α-helix that binds a conserved region of Mer2, and mutating residues within this interface reduces Mre11 foci and DSB formation. Finally, we identified a SUMO-interacting motif within the Mre11 IDR that enhances recruitment of Mre11 during meiosis and facilitates DSB formation. This work identifies multiple mechanisms that collaborate to recruit Mre11 during meiosis to initiate recombination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d416cf934fcb6f095caa1612b3bfc936b858edeb" target='_blank'>
              Recruitment of Mre11 to recombination sites during meiosis
              </a>
            </td>
          <td>
            Priyanka Priyadarshini, Mahesh Survi, Wael El Yazidi Mouloud, Regina Bohn, Steven Ballet, Neil Hunter, Alexander N Volkov, Corentin Claeys Bouuaert
          </td>
          <td>2025-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Epigenetics and genome science have become central to current molecular biology research. Among the key mechanisms ensuring genomic integrity is the silencing of transposable elements in germline cells, a process essential for fertility in both sexes. A pivotal component of this silencing machinery involves PIWI-interacting RNAs (piRNAs), a distinct class of small non-coding RNAs that regulate gene expression and suppress transposable elements at both the transcriptional and post-transcriptional levels. piRNAs function in concert with PIWI proteins, whose expression is critical for proper oogenesis, spermatogenesis, and early zygote development. Disruptions in piRNA or PIWI protein pathways not only impair germline function but also contribute to genome instability, unchecked cell proliferation, and aberrant epigenetic modifications, hallmarks of tumorigenesis. Emerging evidence links the dysregulation of the piRNA/PIWI axis to the development and progression of various cancers, including lung and colorectal carcinomas. This review highlights the fundamental roles of piRNAs and PIWI proteins in reproductive biology and their increasingly recognized relevance in cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e264ddb82fab7df947d4ae7f4949ecb2a0652c26" target='_blank'>
              piRNA-Mediated Maintenance of Genome Stability in Gametogenesis and Cancer
              </a>
            </td>
          <td>
            Martyna Zawalska, Maciej Tarnowski
          </td>
          <td>2025-06-20</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Methylglyoxal (MG) is a highly reactive aldehyde that is produced endogenously during metabolism, and from exogenous sources like sugary food and cigarette smoke. Unless detoxified by glyoxalases, MG can readily react with DNA and proteins, generating characteristic glycation-derived lesions. As a result, MG exposure has been linked to a variety of human diseases, including cancers. Prior studies show that MG preferentially makes adducts on guanine residues, causing DNA damage. However, in vivo, how such events impact genome-wide MG mutagenicity is poorly understood. Such information is essential to comprehend the true contribution of MG to genome instability and global mutational burden. In the present study, we show that MG can robustly mutagenize single-stranded DNA in the yeast genome, within a guanine-centered mutable motif. We demonstrate that genome-wide MG mutagenesis is greatly elevated in the absence of the glyoxalase Glo1, and abrogated in the presence of the aldehyde quencher aminoguanidine. Importantly, we uncovered strand slippage and mispairing as the predominant mechanism for generation of all MG-associated mutations, and demonstrate that the translesion polymerase Rev1 is a key player in this pathway. Finally, we find that the primary MG-associated mutation is enriched in a variety of sequenced tumor datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2ac197fe47da2319b761beabd3304e807a2cc0f" target='_blank'>
              Methylglyoxal mutagenizes single-stranded DNA via Rev1-associated slippage and mispairing
              </a>
            </td>
          <td>
            Sriram Vijayraghavan, Alessandra Ruggiero, Samuel Becker, P. Mieczkowski, George S Hanna, Mark T. Hamann, Natalie Saini
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Castration-resistant prostate cancer is a heterogeneous disease with variable phenotypes commonly observed in later stages of the disease. These include cases that retain expression of luminal markers and those that lose hormone dependence and acquire neuroendocrine features. While there are distinct transcriptomic and epigenomic differences between castration-resistant adenocarcinoma and neuroendocrine prostate cancer, the extent of overlap and degree of diversity across tumor metastases in individual patients has not been fully characterized. Here we perform combined DNA methylation, RNA-sequencing, H3K27ac, and H3K27me3 profiling across metastatic lesions from patients with CRPC/NEPC. Integrative analyses identify DNA methylation-driven gene links based on location (H3K27ac, H3K27me3, promoters, gene bodies) pointing to mechanisms underlying dysregulation of genes involved in tumor lineage (ASCL1, AR) and therapeutic targets (PSMA, DLL3, STEAP1, B7-H3). Overall, these data highlight how integration of DNA methylation with RNA-sequencing and histone marks can inform intraindividual epigenetic heterogeneity and identify putative mechanisms driving transcriptional reprogramming in castration-resistant prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdf4a71accfefcd4f60da4f3b6e81d8837cae193" target='_blank'>
              Intraindividual epigenetic heterogeneity underlying phenotypic subtypes of advanced prostate cancer
              </a>
            </td>
          <td>
            Kei Mizuno, Sheng-Yu Ku, V. B. Venkadakrishnan, Martin K. Bakht, M. Sigouros, Joanna Chan, A. Trigos, Jordan H. Driskill, J. Manohar, Abigail King, Adam G. Presser, Min Jin Kim, Alok K. Tewari, H. Long, David Quigley, T. Choueiri, Steven P. Balk, Sarah J. Hill, J. Mosquera, David Einstein, Shahneen Sandhu, M. Taplin, H. Beltran
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Genome instability is a hallmark of cancer, often driven by mutations and altered expression of genome maintenance factors involved in DNA replication and repair. Rap1 Interacting Factor 1 (RIF1) plays a crucial role in genome stability and is implicated in cancer pathogenesis. Cells express two RIF1 splice variants, RIF1-Long and RIF1-Short, which differ in their ability to protect cells from DNA replication stress. Here, we investigate differential expression and splicing of RIF1 in cancer cell lines following replication stress and in patients using matched normal and tumour data from The Cancer Genome Atlas (TCGA). Overall RIF1 expression is altered in several cancer types, with increased transcript levels in colon and lung cancers. RIF1 also exhibits distinct splicing patterns, particularly in specific breast cancer subtypes. In Luminal A (LumA), Luminal B (LumB), and HER2-enriched breast cancers (HER2E), RIF1 Exon 31 tends to be excluded, favouring RIF1-Short expression and correlating with poorer clinical outcomes. These breast cancer subtypes also tend to exclude other short exons, suggesting length-dependent splicing dysregulation. Basal breast cancer also shows exon exclusion, but unlike other subtypes, it shows no short-exon bias. Surprisingly, however, in basal breast cancer, RIF1 Exon 31 is not consistently excluded, which may impact prognosis since RIF1-Long protects against replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ca94768efdf45848f937fd47caaaca2fe19dbc3" target='_blank'>
              Dysregulated Alternative Splicing in Breast Cancer Subtypes of RIF1 and Other Transcripts
              </a>
            </td>
          <td>
            Emma Parker, Laura Akintche, Alexandra Pyatnitskaya, S. Hiraga, A. Donaldson
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ddc1079a033fb614c3439063ef9f1c82d7614ef4" target='_blank'>
              Phage-Encoded TelN Inhibits Mre11-Rad50 to Protect Hairpin Telomeres
              </a>
            </td>
          <td>
            Maya Houmel, Nicolas Pellaton, Anna Anchimiuk, Stephan Gruber
          </td>
          <td>2025-06-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 The apolipoprotein B mRNA editing enzyme, catalytic polypeptide- like 3 (APOBEC3 or A3) family of cytosine deaminases has been implicated in some of the most prevalent mutational signatures in cancer. A3-associated mutational signatures have been identified in more than 70% of cancer types and around 50% of all cancer genomes, with prominence in breast cancer as well as other cancer types. Many tumors are hypermutated by C-to-T/G mutations induced by A3 within TCW (W:T,A) motifs, accounting for many driver mutations in genes such as PIK3CA, ERBB2, and PPP2R1A. The genome and transcriptome of tumor tissue biopsies contain a wealth of information about these A3-associated signatures. However, quantification of A3 expression changes in tumor cells is confounded by the ubiquitous expression of these enzymes in infiltrating immune cells. To overcome this we will utilize single-cell RNA-seq analysis across a wide range of publicly available breast cancer datasets in order to follow the relative changes of A3 expression within individual cell types. Results from this analysis will give our lab insights into to changes in A3 expression in tumor cells which will be helpful in unveiling the complex relationships among breast cancer cells, stromal elements, and immune cells, facilitating the development of personalized therapies and prognostic markers, while also shedding light on breast cancer heterogeneity and potential vulnerabilities.
 Citation Format: Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi. Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P2-06-27.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4bd3dd6ddb39a7b246a7fddf688cff30066edc6" target='_blank'>
              Abstract P2-06-27: Mapping APOBEC Profiles in Breast Cancer Microenvironments: Bridging Bulk and Single-Cell RNA-seq Data
              </a>
            </td>
          <td>
            Jake Lehle, Mohadeseh Soleimanpour, Diako Ebrahimi
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Invasive lobular carcinoma (ILC) is a common subtype of breast cancer that is defined in part by genetic loss of CDH1 caused by mutation or deletion, leading to loss of cell adhesion protein E-cadherin in >90% of ILC. Genetic loss of CDH1 is an early event in ILC oncogenesis, yet the mechanisms by which CDH1/E-cadherin acts as a tumor suppressor are not well understood. To study how early CDH1 loss drives ILC oncogenesis, we used a series of non-transformed human mammary epithelial cell (HMEC) models to target CDH1/E-cadherin, inhibiting extracellular E-cadherin signaling using antibodies versus modeling genetic CDH1 loss using siRNA or knockout via CRISPR/Cas9. Through transcriptome analyses across four HMEC models, we found that the mode of E-cadherin loss or suppression is critical for the subsequent phenotype. Antibody-mediated inhibition of cell-cell contacts induced gene signatures of epithelial-mesenchymal transition (EMT), consistent with the role of E-cadherin suppression during the EMT process. Conversely, genetic CDH1 loss – as in ILC oncogenesis – repressed EMT signatures, and instead remodeled gene expression toward a luminal epithelial phenotype. Using single cell transcriptomics and flow cytometry analyses of cell lineage markers, we found that genetic loss of CDH1 reprogrammed cells to a luminal progenitor-like phenotype. By isolating luminal versus basal cells prior to CDH1 knockout, we found that CDH1 loss led to remodeling of lineage identity in both populations, converging on a new lineage homeostasis with a luminal progenitor-like phenotype. Consistent with increased progenitor features, CDH1 loss enhanced proliferative capacity over the finite lifespan of the HMECs, highlighting a feature of early CDH1 loss that may contribute to clonal advantage during tumor initiation. Our findings support that inhibition of E-cadherin results in different transcriptional response compared to CDH1 loss, with the latter driving a transcriptional and phenotypic state characteristic of a luminal progenitor-like population, which offers new insight into early events in ILC oncogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b55a904a9de6859d0ee1985b86b2f0ef97b6b1ec" target='_blank'>
              CDH1 loss remodels gene expression and lineage identity in human mammary epithelial cells
              </a>
            </td>
          <td>
            Maggie Musick, Chinasa A Ufondu, Carmen E. Rowland, Joseph L. Sottnik, Madeleine T. Shackleford, Camryn S. Nesiba, J. Ostrander, Matthew J. Sikora
          </td>
          <td>2025-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Nonhomologous end joining (NHEJ) is required for repairing DNA double strand breaks (DSBs) generated by the RAG endonuclease during lymphocyte antigen receptor gene assembly by V(D)J recombination. The ataxia telangiectasia–mutated (ATM) and DNA-dependent protein kinase catalytic subunit (DNA-PKcs) kinases regulate functionally redundant pathways required for NHEJ. Here, we report that loss of the senataxin helicase leads to a strong defect in RAG DSB repair upon inactivation of DNA-PKcs. The NHEJ function of senataxin is redundant with the RECQL5 helicase and the HLTF translocase and is epistatic with ATM. Co-inactivation of ATM, RECQL5, and HLTF results in an NHEJ defect similar to that from the combined deficiency of DNA-PKcs and senataxin or losing senataxin, RECQL5, and HLTF. These data suggest that ATM and DNA-PKcs regulate the functions of senataxin and RECQL5/HLTF, respectively, to provide redundant support for NHEJ.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19bdfb0688308baf4bbac48b67309551e1f66348" target='_blank'>
              Senataxin and DNA-PKcs redundantly promote non-homologous end joining repair of DNA double strand breaks during V(D)J recombination
              </a>
            </td>
          <td>
            Bo-Ruei Chen, Thu Pham, Lance D. Reynolds, Nghi Dang, Yanfeng Zhang, Kimberly Manalang, Gabriel Matos-Rodrigues, J. R. Neidigk, A. Nussenzweig, Jessica K Tyler, Barry P Sleckman
          </td>
          <td>2025-06-20</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea81a914a4a5b1e0240796b0fe52e36431943de7" target='_blank'>
              Genetic barcoding uncovers the clonal makeup of solid and liquid biopsies and their ability to capture intra-tumoral heterogeneity
              </a>
            </td>
          <td>
            A. Serrano, Tom S. Weber, J. Berthelet, Sarah Ftouni, Farrah El-Saafin, Verena C. Wimmer, K. Rogers, Elgene Lim, Emmanuelle Charaffe-Jauffret, Christophe Ginestier, David Williams, Frédéric Hollande, B. Yeo, Sarah-Jane Dawson, S. Naik, D. Merino
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eca7843e2f4903653396c94ffa8fe3fdaa395232" target='_blank'>
              ZNF280A links DNA double-strand break repair to human 22q11.2 distal deletion syndrome.
              </a>
            </td>
          <td>
            Thomas L Clarke, Hyo Min Cho, Ilaria Ceppi, Boya Gao, Tribhuwan Yadav, Giorgia G Silveira, Ruben Boon, Barbara Martinez-Pastor, Nana Yaa A Amoh, Belen Machin, Tiziano Bernasocchi, Dua Ashfaq, Josefina Mendez, Zeeba Kamaliyan, José Del Río Pantoja, Giuliana Sardi Rogines, Blaine T Crowley, D. McGinn, V. Giunta, Oanh Tran, E. Zackai, Li Lan, Lee Zou, B. S. Emanuel, Donna M. McDonald-McGinn, Petr Cejka, Raul Mostoslavsky
          </td>
          <td>2025-06-01</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b7bd3ebe84ec1189f788307a3b3b752befef2ae" target='_blank'>
              Integrated Single-Cell and Spatial Transcriptomic Analysis Reveals an Aging-Associated Fibroblast Subtype Linked to Tumor Progression in Human Skin
              </a>
            </td>
          <td>
            Haibin Wu, Danping Pan
          </td>
          <td>2025-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA topoisomerases are essential for maintaining DNA topology, gene expression and the accurate transmission of genetic information. Mitochondria possess circular DNA (mtDNA), which, unlike nuclear chromosomes, lacks protective histones and exists in nucleoprotein complexes called nucleoids, which are vital for mtDNA stability. Although the mitochondrial genome encodes essential genes involved in ATP production via oxidative phosphorylation, it does not encode crucial mtDNA maintenance genes and depends entirely on nuclear-encoded proteins for mtDNA maintenance. These include nuclear-encoded topoisomerases (i.e. Top1mt, Top2α, Top2β and Top3α), which alleviate topological stress during mtDNA transcription and replication, and mitochondrial transcription factor A (TFAM), are crucial for ensuring proper nucleoid structure and mtDNA packaging. Furthermore, tyrosyl-DNA phosphodiesterase 1 and 2 (TDP1 and TDP2) participate in the repair of mtDNA damage associated with trapped topoisomerase-mtDNA complexes, which can compromise mtDNA integrity and contribute to neurodegeneration, cancer and premature aging. Drugs that stabilize these protein-DNA adducts (PDAs) to induce mtDNA damage and mitochondrial dysfunction are promising new strategies for cancer therapy. This Review explores the essential roles of mitochondrial topoisomerases, overviews mechanisms involved in mtDNA repair and discusses how mitochondrial fission and mitophagy are employed as a survival strategy for clearing damaged mtDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc13f830961da01cf0f5401d6373dc48a97645fa" target='_blank'>
              Mitochondrial topoisomerases, nucleoid architecture and mtDNA repair in human disease.
              </a>
            </td>
          <td>
            Sangheeta Bhattacharjee, B. Das
          </td>
          <td>2025-07-01</td>
          <td>Journal of cell science</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is highly resistant to chemotherapy. However, PDAC with germline BRCA mutations, which lead to homologous recombination (HR) deficiency (HRD), demonstrated an increased sensitivity to platinum-based chemotherapy regimens. This increased chemosensitivity was also seen in PDACs with germline or somatic mutations in the DNA double-strand damage response (DDR) genes beyond canonical HR genes such as BRCA1, BRCA2, and PALB2. However, there are no consensus methods to determine HRD status; and neither is there a well-defined list of HR-DDR genes. In addition, how HRD and/or HR-DDR gene mutation status impacts the tumor immune microenvironment including tumor mutation burden (TMB), neoantigen load, T cell receptor (TCR) repertoire, and effector T cell infiltration is unknown. Thus, in this study, we developed a new method to categorize PDACs into HRD-positive and HRD-negative subgroups by using results from whole exome sequencing, whole genome sequencing, or both into consideration. We classified a cohort of 89 PDACs into HRD-positive (n = 18) and HRD-negative (n = 69) tumors. HR-DDR gene variants were identified more frequently in HRD-positive PDACs than HRD-negative PDACs, with RAD51B, BRCA2 and ATM alterations most frequently identified in HRD-positive PDACs. Notably, TMB and neoantigen load was significantly higher in HRD-positive PDACs compared to HRD-negative tumors. Interestingly, HRD-positive PDACs, PDACs with high tumor mutational burden, and PDAC with high neoantigen load were all associated with lower CD8 + T lymphocyte infiltration and T cell clonal diversity, suggesting a mechanism of resistance to immune checkpoint inhibitors (ICIs). Therefore, this study suggests that treatments to enhance effector T cell infiltration and T cell clonal diversity may overcome resistance to ICI-based immunotherapy in HRD-positive PDACs. Supplementary Information The online version contains supplementary material available at 10.1186/s40164-025-00673-0.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d76f00cebbe1db7a6bb20d7d6bd25ec9f1ccbba1" target='_blank'>
              Homologous recombination-DNA damage response defects increase TMB and neoantigen load, but not effector T cell density and clonal diversity in pancreatic cancer
              </a>
            </td>
          <td>
            Mengyue Lei, Jessica Gai, T. McPhaul, Huijuan Luo, Penghui Lin, Dongbin Liu, Michael Pishvaian, Nicholas J Roberts, Kui Wu, Jin He, Lei Zheng
          </td>
          <td>2025-06-18</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Senescence has been shown to contribute to the progression of aging related diseases including degenerative disc disease (DDD). However, the mechanisms regulating senescence in the intervertebral disc (IVD) and other tissues/diseases remain poorly understood. Recently, in a CRISPRa genome-wide screen, our lab identified a previously uncharacterized zinc finger protein, ZNF865 (BLST), that regulates a wide array of genes related to protein processing, cell senescence and DNA damage repair. Here, we demonstrate that ZNF865 expression is correlated with age and disease state in human patient IVD samples and mouse IVD. Utilizing CRISPR-guided gene modulation, we show that ZNF865 is necessary for healthy cell function and is a critical protein in regulating senescence and DNA damage in intervertebral disc cells, with implications for a wide range of tissues and organs. We also demonstrate that downregulation of ZNF865 induces senescence and upregulation mitigates senescence and DNA damage in human nucleus pulposus (NP) cells. Importantly, upregulation of ZNF865 shifts the chromatin landscape and gene expression profile of human degenerative NP cells towards a healthy cell phenotype. Collectively, our findings establish ZNF865 as a novel modulator of genome stability and senescence and as a potential therapeutic target for mediating senescence/DNA damage in senescence related diseases and disorders. Summary Degenerative disc disease (DDD) is a major contributor to chronic low back pain, a leading cause of disability globally1–3. Cellular senescence has emerged as a key driver of disc degeneration4,5, characterized by cell-cycle arrest and the secretion of pro-inflammatory and matrix-degrading factors collectively termed the senescence-associated secretory phenotype (SASP). While the pathological role of senescent cells in musculoskeletal aging is increasingly recognized6–8, the upstream molecular regulators remain poorly understood. Here we identify a previously uncharacterized zinc finger protein, ZNF865, as a novel regulator of senescence and genomic stability in human nucleus pulposus (NP) cells. CRISPRi-mediated downregulation of ZNF865 in healthy NP cells induced senescence, increased expression of p16 and p21, and led to increases in DNA damage. Conversely, upregulation of ZNF865 in degenerative NP cells restored proliferation, suppressed senescence markers, reduced DNA damage, significantly diminished SASP factor secretion and restored transcriptomic and epigenetic profiles to a healthy phenotype. This study represents the first functional characterization of ZNF865 and establishes it as an important regulator for senescence in disc cells. These findings highlight ZNF865 as a promising therapeutic target for mitigating senescence-driven pathologies in DDD and potentially other age-related disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a94b93bbdecb6b043f4125dc44dd985694b019b6" target='_blank'>
              ZNF865 (BLST) Regulates Human Cell Senescence and DNA Damage
              </a>
            </td>
          <td>
            Christian Lewis, H. Levis, Jonah Holbrook, Jacob T. Polaski, Timothy D. Jacobsen, S. Gullbrand, B. Diekman, J. Iatridis, Jason Gertz, Brandon Lawrence, Robert D Bowles
          </td>
          <td>2025-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b0586d8fafa9a29f6a58bf48055edaf80d268f3" target='_blank'>
              RIPOR2 promotes multinucleation of melanoma cells downstream of the RAS/ERK oncogenic pathway
              </a>
            </td>
          <td>
            Axelle Wilmerding, Aurélie Richard, Nicolas Macagno, E. Hirsinger, Tarek Gharsalli, Léa Bellenger, N. Naouar, Caroline Gaudy, Stéphanie Mallet, N. Degardin, Lauranne Bouteille, Nathalie Caruso, D. Duprez, Y. Graba, S. Medjkane, H. Etchevers, Marie-Claire Delfini
          </td>
          <td>2025-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Abstract Transposable elements (TE), also called transposons, are repetitive DNA sequences making up half of the human genome. Initially, TEs were described as “junk DNA” because they lack specific function. However, they have been recognized for their ability to replicate and integrate into different genomic locations; this “jumping gene” activity results in genomic instability, variation of the chromosomal architecture, and transcriptional dysregulation, all of which represent major hallmarks of cancer. In this respect, the involvement of TE in tumorigenesis was extensively studied and their role as diagnostic and therapeutic tools in cancer is now well-established. Transposons’ products, including TE-derived cancer antigens, transcripts, and associated epigenetic modifications, mainly hypomethylation, were found to be promising biomarkers in several types of cancer ensuring early disease detection and prognosis. In addition, TE are currently used to design innovative approaches, with transposon-based systems, namely, Sleeping Beauty and PiggyBac, enabling precise genomic modifications and novel strategies for cancer therapy. Therefore, the aim of this review is to provide an overview on the dual application of TE as diagnostic and therapeutic tools in cancer, paving the way to improved clinical outcomes. Clinical trial number Not applicable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4636ade80b58aaf2a75f12fb6871d3d8ab2244c5" target='_blank'>
              Exploring transposable elements: new horizons in cancer diagnostics and therapeutics
              </a>
            </td>
          <td>
            Ahlam Chaaban, Reem Sleem, Jana Santina, Mohamad Rima, José-Noel Ibrahim
          </td>
          <td>2025-07-12</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Hepatoblastoma is the most common pediatric liver cancer, with the fastest rising incidence among childhood malignancies. Early genomic studies revealed that hepatoblastoma has the lowest mutational burden of any human cancer, however, recent advances in single-cell RNA-seq, multiomics, spatial transcriptomics, and functional genomics screenings have revealed substantial complexity. Diverse cellular subpopulations, divergent WNT signaling, key developmental pathways, and intricate interactions between the tumor cells and tumor immune microenvironment (TME) collectively shape tumor heterogeneity, disease progression, therapeutic responses, and genetic dependencies. Mosaic embryonic loss of heterozygosity (LOH) at chromosome 11p15.5 may be a hepatoblastoma-initiating event, as clonal expansion of 11p15.5 LOH occurs in adjacent normal liver tissue. A cholangiocyte-like subpopulation expresses FGF19, in a SOX4-dependent, paracrine manner, to drive the proliferation of neighboring embryonal hepatoblastoma cells. WNT-signaling dependent MDK promotes the immunosuppressive TME, which impairs immune cell infiltration. The TME may also be driven by islands of erythroblasts, which influence treatment resistance. Plasticity driven by changes in chromatin accessibility enables differentiation transition between hepatocytic and liver progenitor cell types, which is associated with treatment resistance. Here, we review recent findings in pediatric hepatoblastoma cells, tumor-associated cell types, and genetic dependencies that will serve to advance hepatoblastoma therapy. Clinical trial number: Not applicable. Supplementary Information The online version contains supplementary material available at 10.1186/s12943-025-02405-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/361b9ebd3fa588c3f28e25ba36c91c45f4b07fb6" target='_blank'>
              From preneoplastic lesion to heterogenous tumor: recent insights into hepatoblastoma biology and therapeutic opportunities
              </a>
            </td>
          <td>
            Jun Yang, A. Davidoff, Andrew J. Murphy
          </td>
          <td>2025-07-19</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="DNA polymerase delta (Pol δ) is a cornerstone of genomic stability, orchestrating DNA replication and repair through its catalytic subunit, POLD1. This subunit’s 3’–5’ exonuclease domain proofreads replication errors, ensuring fidelity. However, POLD1 mutations—particularly in this domain—disrupt this function, triggering genomic instability and a hypermutated state in cancers. This review delves into the multifaceted roles of POLD1 mutations, spotlighting their contributions to tumorigenesis and immunotherapy responses. Beyond their established link to syndromes like polymerase proofreading-associated polyposis (PPAP), these mutations unexpectedly enhance tumor immunogenicity in microsatellite-stable (MSS) tumors, previously considered largely resistant to immune checkpoint inhibitors (ICIs). By elevating tumor mutation burden and generating unique mutational signatures (e.g., SBS10d), POLD1 mutations sensitize MSS tumors to ICIs, challenging the dominance of microsatellite instability (MSI) as an immunotherapy predictor. Integrating structural insights, molecular mechanisms, and clinical data, this review positions POLD1 mutations as both a driver of cancer progression and a promising biomarker, redefining therapeutic possibilities in precision oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02d2442be2c75ad66ffe5186a1b712852b468974" target='_blank'>
              Beyond proofreading: POLD1 mutations as dynamic orchestrators of genomic instability and immune evasion in cancer
              </a>
            </td>
          <td>
            Huiqing Chen, Jiayu Wei, Qi Tang, Guohui Li, Yajing Zhou, Zhen Zhu
          </td>
          <td>2025-06-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT DNA methylation is a common epigenetic modification that maintains the integrity of the DNA sequence while profoundly influencing gene expression and phenotypic variation. Aberrant DNA methylation has been associated with the onset and progression of diseases, including cancer, metabolic disorders, and neurodevelopmental disorders. Recent advancements in detection technology led to a gradual increase in the exploration of DNA methylation as a valuable biomarker for cancer diagnosis and therapy. Single-base resolution has been achieved for whole-genome methylation analyses through second-generation sequencing technology, significantly enhancing detection efficiency. Additionally, PCR-based methods offer simple and feasible solutions for methylation analysis. In this review, we discuss various methods for detecting DNA methylation, focusing on bisulfite conversion-based techniques, methylation-sensitive restriction enzyme methods, enzyme conversion-based methods, third-generation sequencing approaches, and artificial intelligence. Furthermore, we briefly summarize the methylation biomarkers used for tumor diagnosis and the corresponding sample types employed. We believe that this information provides valuable insights for selecting and optimizing DNA methylation analysis tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecc2bff17d09231bf27c4fb389c540837b1a009a" target='_blank'>
              Advancements in DNA methylation technologies and their application in cancer diagnosis
              </a>
            </td>
          <td>
            Yang Yang, Xiaosha Wen, Li Wang
          </td>
          <td>2025-07-28</td>
          <td>Epigenetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The majority of cancer cells experience replication stress, which ultimately causes them to enter mitosis with underreplicated DNA. To alleviate the consequences of replication stress, cells utilize a mechanism known as MiDAS that functions to complete synthesis of underreplicated DNA in early mitosis. This process is considered an Achilles heel for highly replicative cancers. In this study, we show that human TopBP1 localizes to sites of underreplicated DNA marked by FANCD2 and promotes MiDAS through recruitment of the nuclease scaffold protein SLX4. Additionally, we demonstrate that the recruitment of SLX4 to TopBP1 foci in mitosis depends on TopBP1-K704, SLX4-T1260, and several SUMO-interaction motifs in SLX4. Lastly, we show that the recruitment of SLX4 to TopBP1 foci in mitosis is important to prevent transmission of DNA damage to daughter cells. Based on this, we hypothesize that targeting the TopBP1-SLX4 interaction in mitosis may be a potential strategy for anti-cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d03ab5b01bb44008997c89ac819106d74fa906d" target='_blank'>
              TopBP1 coordinates DNA repair synthesis in mitosis via recruitment of the nuclease scaffold SLX4
              </a>
            </td>
          <td>
            Jonas Bagge, Kamilla Vandsø Petersen, Sinem N Karakus, Thorbjørn M Nielsen, Johanne Rask, Christian R Brøgger, Jonas Jensen, Meliti Skouteri, Antony M. Carr, I. Hendriks, V. Oestergaard, Michael Lisby
          </td>
          <td>2025-07-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Tumor mutational burden (TMB) has emerged as a potential surrogate for neoantigen load and an indicator of immune checkpoint (IC)-blockade response; however, its precise significance in breast cancer (BC) is not fully understood. Here, we comprehensively characterized the genomic repertoire of BCs with a TMB ≥ 10 mut/Mb (TMB-high [n = 527]) to identify putative predictors of importance. The predominant mutational signature was apolipoprotein B mRNA-editing enzyme catalytic polypeptide (APOBEC) in 64.7% of tumors. TMB-high BCs were enriched in KMT2C, ARID1A, PTEN, NF1, and RB1 alterations, which are associated with APOBEC mutagenesis. Further identified were loss-of-function ARID1A and PTEN alterations, which are linked to immune cell exclusion. ESR1 p.E380Q prevailed among all ESR1 hotspot mutations, supporting APOBEC-mediated effects. Finally, mutations in DNA damage response and repair genes were seen at a higher frequency than in non-TMB-high BCs. These findings provide justification for exploring combined pharmacologic inhibition to improve IC-based efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e8980a3a4147f92cf6718bb89169af5da92bd71" target='_blank'>
              Genomic characterization of tumor mutational burden-high breast carcinomas
              </a>
            </td>
          <td>
            Theodore Vougiouklakis, Chad M. Vanderbilt, Satshil Rana, A. Mohanty, F. Pareja, E. Brogi, Christopher J. Schwartz, M. Arcila, Marc Ladanyi, Hanna Y Wen, D. Ross
          </td>
          <td>2025-08-08</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Many essential functions of organisms are encoded in highly repetitive genomic regions, including histones involved in DNA packaging, centromeres that are core components of chromosome segregation, ribosomal RNA comprising the protein translation machinery, telomeres that ensure chromosome integrity, piRNA clusters encoding host defenses against selfish elements, and virtually the entire Y Chromosome. These regions, formed by highly similar tandem arrays, pose significant challenges for experimental and computational studies, impeding sequence-level descriptions essential for understanding genetic variation. Here, we report the assembly and variation analysis of such repetitive regions in Drosophila melanogaster, offering significant improvements to the existing community reference assembly. Our work successfully recovers previously elusive segments, including complete reconstructions of the histone locus and the pericentric heterochromatin of the X Chromosome, spanning the Stellate locus to the distal flank of the rDNA cluster. To infer structural changes in these regions where alignments are often not practicable, we introduce landmark anchors based on unique variants that are putatively orthologous. These regions display considerable structural variation between different D. melanogaster strains, exhibiting differences in copy number and organization of homologous repeat units between haplotypes. In the histone cluster, although we observe minimal genetic exchange indicative of meiotic crossing over, the variation patterns suggest mechanisms such as unequal sister chromatid exchange. We also examine the prevalence and scale of concerted evolution in the histone and Stellate clusters and discuss the mechanisms underlying these observed patterns.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/624ae86af4f411108a369c23810a20e4ee6dde43" target='_blank'>
              Genetic variation in recalcitrant repetitive regions of the Drosophila melanogaster genome.
              </a>
            </td>
          <td>
            Harsh G. Shukla, Mahul Chakraborty, J. Emerson
          </td>
          <td>2025-07-16</td>
          <td>Genome research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Throughout the last century, aneuploidy has been cemented as a hallmark of cancer. Although the association of aneuploidy with tumorigenesis has been well established, the role of these genetic imbalances in tumor formation has only recently begun to be elucidated. Advancements in genomics have revealed the complexity and context dependence of the effect of aneuploidy on cancer growth, while developments in genetic editing have allowed for proper modeling of specific aneuploidies. In this review, we discuss the key factors to consider when studying the role of aneuploidy in cancer and the tools that are available to do so. We then highlight recent studies that establish phenotypic contributions of aneuploidy to tumorigenicity. In particular, we highlight how general aneuploidy and chromosomal instability affect the tumor microenvironment and how specific chromosomal alterations, including the loss of chromosome 9p and the gain of chromosomes 8q and 1q, influence tumor behavior and therapeutic responses. Finally, we emphasize the potential of targeting aneuploidy-induced vulnerabilities to improve cancer treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37b1804dd6a9ac26725f995589c82459ba79d112" target='_blank'>
              Modeling and targeting general and chromosome-specific aneuploidy in cancer.
              </a>
            </td>
          <td>
            Aleah Goldberg, Maria Trifas, Teresa Davoli
          </td>
          <td>2025-07-28</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Tumor relapse remains a significant obstacle to successful therapy. Preclinical animal models that accurately reflect tumor relapse in patients are urgently needed. Here, we employed a dual recombinase-mediated genetic system to genetically trace and ablate proliferating cells in a polyomavirus middle T antigen (PyMT)-induced spontaneous murine breast cancer model. This system enabled the acute ablation of cells that had undergone proliferation within a defined time window, resulting in a drastic tumor shrinkage, followed by a gradual tumor relapse due to the presence of residual low-cycling cells. We then applied single-cell RNA sequencing (scRNA-seq) to unbiasedly compare the tumor ecosystems of the primary and relapsed PyMT tumors. Compared with the primary tumors, the relapsed tumors exhibited a higher proportion of cancer stem cells and pro-tumor γδ T cells, as well as co-expression of Spp1 and Vegfa in multiple myeloid cell populations – features that predict poor therapeutic response and unfavorable outcomes in human breast cancer patients. Collectively, this proliferation tracing and ablation model emulates chemotherapies that preferentially eliminate proliferating cancer cells, serving as a robust tool and a valuable resource for testing novel therapeutic strategies in relapsed tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95fdfebff6c6c3226be581651d876e8c2d584753" target='_blank'>
              Modeling tumor relapse using proliferation tracing and ablation transgenic mouse
              </a>
            </td>
          <td>
            Chuang Zhao, Xin-nan Zheng, Han-Ying Huang, Lin Tian
          </td>
          <td>2025-07-17</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Rhabdomyosarcoma (RMS) contributes to 3% of all childhood cancers with roughly 400-500 cases diagnosed each year in the United States. The World Health Organization classifies rhabdomyosarcoma into four histological subtypes which include alveolar, embryonal, spindle-cell and pleomorphic. The primary genetic drivers in a subset of alveolar and spindle-cell histological subtypes are gene fusions. This review explores the fusion oncogenes identified in RMS such as PAX-and NCOA2-based fusions, along with discussing studies defining fusion oncogene biology and tumorigenic mechanisms. Focus areas include data around transformation events and progression along with dysregulated biological processes. Furthermore, we summarize model systems, ranging from cell to animal models, that have been implemented to study fusion oncogenes identified in RMS. With the constant identification of novel fusion oncogenes, this review also emphasizes the need for genetically characterizing RMS tumors and rapidly developing new model systems. These models are critical to study fusion oncogene activity and to delineate key regulatory players and potential therapeutic targets that suppress tumorigenesis. The identification of RMS fusion oncogenes and integration with animal and cell culture models will help identify conserved molecular targets, optimize therapeutic approaches, and ultimately improve clinical outcomes for children with RMS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b23d84102b5431b0bc6bd415779fd94bf54425c" target='_blank'>
              Fusion oncogenes in rhabdomyosarcoma: model systems, mechanisms of tumorigenesis, and therapeutic implications
              </a>
            </td>
          <td>
            Chinmay S. Sankhe, Lisa Hall, Genevieve C. Kendall
          </td>
          <td>2025-06-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genomic instability underlies various diseases, including cancer. This instability arises from defects in critical cellular processes, particularly those involved in DNA damage repair. Therefore, a detailed understanding of these repair mechanisms is essential for developing strategies to prevent or diagnose such diseases. The MRN complex, composed of MRE11, NBS1, and RAD50, is among the earliest elements involved in detecting DNA damage. Upon detecting DNA breaks, this complex triggers a cascade of signaling events that regulate both cell cycle arrest and DNA repair. These signaling pathways are tightly controlled by various post-translational modifications, notably ubiquitination. Although several ubiquitin ligases have been implicated in different stages of the DNA damage response, our knowledge remains limited. In this study, we reveal that βTrCP, a substrate-recognizing subunit of the SCF (SKP1/CUL1/F-box protein) ubiquitin ligase, interacts in vivo with the proteins of the MRN complex. These interactions occur in normally proliferating cells and are dependent on the GSK3 kinase. Moreover, we show that βTrCP enhances the recruitment of the MRN complex to chromatin through MRE11, thereby promoting the efficient DNA damage repair. Hence, alterations in βTrCP function affecting MRN dynamics could have severe consequences for the cell homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47da9e36f36ee547f6e7e42f2ff12a54a26f4687" target='_blank'>
              βTrCP facilitates MRN complex localization on chromatin to enhance DNA repair
              </a>
            </td>
          <td>
            Alejandro Belmonte-Fernández, Joaquín Herrero-Ruíz, Carmen Sáez, M. Japón, M. Mora-Santos, F. Romero
          </td>
          <td>2025-07-11</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Epstein–Barr virus, the first identified human DNA tumour virus, is detectable in more than 90% of nasopharyngeal carcinoma patients in endemic regions. The 3D chromosome conformation analysis reveals that virus‒host chromatin interactions induce the spatial reorganisation of loops and compartments, resulting in “enhancer infestation” and switch of “H3K27 bivalency” at EBV-interacting regions. Through this mechanism, EBV hijacks KDM5B, a gatekeeper of genome stability, and its binding targets, leading to aberrant activation of an EBVIR-enhancer-KDM5B signature. Cancer cells with this signature present increased MYC activation, DNA damage responses, and epigenetic plasticity of epithelial-immune dual features with metastatic potential. Our multicentre multiomics study confirms that this signature is the prerequisite for chromosome instability and can be utilised as a risk factor for distant metastasis. This study highlights a mechanism in which latent viral episomes can alter the host high-order epigenotype, promoting transcriptional rewiring and metastasis in NPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d24b257eddf1aac8b8f7ed9ed3a3ecfd24278af6" target='_blank'>
              Virus-human chromatin interactions reorganise 3D genome and hijack KDM5B for promoting metastasis in nasopharyngeal carcinoma
              </a>
            </td>
          <td>
            D. Chung, Zhaozheng Hou, Ying Wang, Kazi Anisha Islam, Songran Liu, Jiayan Liu, Larry Ka-Yue Chow, Yuki Yuk-Wun Wong, Cyrus Paak-Ting Chak, Yingpei Zhang, Lanqi Gong, Ziyang Qi, , Zhuoyou Yu, Ping Feng, Zilu Huang, R. Ngan, Xinyuan Guan, W. Ng, Zhonghua Liu, A. Tsang, D. Kwong, Ann Wing-Mui Lee, V. H. Lee, Honglin Chen, Yunfei Xia, Wei Dai
          </td>
          <td>2025-08-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3dcbc0047fe628bc7d0d1f2c63478bc9274ca9ad" target='_blank'>
              Punctuated mutagenesis promotes multi-step evolutionary adaptation in human cancers
              </a>
            </td>
          <td>
            Christopher J Graser, Wenbo Wu, Cole Christini, Mia Petljak, Franziska Michor
          </td>
          <td>2025-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eee44887a88c522cc74478c5f67d3f1813be9b0" target='_blank'>
              DPYSL3B is a regulator of chemoresistance via DNA repair and metabolic reprogramming in prostate cancer
              </a>
            </td>
          <td>
            R. Kaarijärvi, H. Kaljunen, Nella Itkonen, Kseniia Bureiko, K. Ketola
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background The human genome contains approximately 3,200 near full-length autonomous human endogenous retroviral (HERV) genomes distributed across the 23 chromosomes. These autonomous HERV proviral genomes include long terminal repeats (LTRs) capable of promoting RNA transcription. In quiescent cells, most HERV loci remain transcriptionally silent. However, environmental changes, such as epigenetic remodeling of chromatin, can activate these silenced loci. Methods To study HERV reactivation, we previously analyzed autonomous HERV expression patterns in monocytes isolated from peripheral blood mononuclear cells (PBMCs) identified in single-cell RNA sequencing (scRNA-seq) databases using the Azimuth application. We developed a Window-based HERV Alignment (WHA) method, which analyzes aligned DNA sequences using sequential, non-overlapping windows of defined lengths. Samples were scored as positive (>= 9 good/usable windows) or negative (<= 8 good/usable windows). Results Using WHA, we established a control set from 31 normal individuals, with fewer than 8 windows at selected HERV loci. We analyzed scRNA-seq data from three studies of hospitalized COVID-19 patients and found distinct HERV expression patterns in monocytes. Unique patterns were also found in patients with influenza, Dengue virus, or sepsis. We next examined HERV expression at early (<7 days) and late (>14 days) timepoints post COVID-19 recovery and detected HERV loci in both groups. Analyzing 12 patients with post-acute sequelae of COVID-19 (PASC), we identified three HERV loci expressed in all patients. Some loci showed amplified numbers of good/usable windows, indicating longer transcripts and greater sequence depth. The most amplified locus was located within an intron of JAKMIP2, which, along with neighboring host genes, also showed increased transcription. Conclusion Previous studies have shown that viral infections, including COVID-19, influenza, and Dengue virus, as well as sepsis, can induce innate immune memory in monocytes through epigenetic remodeling of hematopoietic stem and myeloid precursor cells. The identification of co-amplified HERV loci and neighboring host gene transcripts in monocytes from PASC patients suggests expansion of epigenetically remodeled myeloid progenitors. The identification of these HERV-host gene patterns provides a foundation needed to understand the clinical features of patients with PASC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3890c566bb6e206114c5472e39818cdcd77102fc" target='_blank'>
              Amplification of select autonomous HERV loci and surrounding host gene transcription in monocytes from patients with post-acute sequelae of COVID-19
              </a>
            </td>
          <td>
            Hyunmin Koo, Casey D. Morrow
          </td>
          <td>2025-06-26</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Genomic instability is a hallmark of cancer, enabling the generation of mutations and gross chromosomal rearrangements to drive neoplastic cell transformation and oncogenesis. The BRCA1-BARD1 protein complex acts to eliminate highly toxic DNA double-strand breaks, to ensure the faithful propagation of our genetic blueprint and to suppress cancer development. BRCA1 is a well-described tumor suppressor protein associated with hereditary breast and ovarian cancers as well as sporadic breast cancers, with loss or mutation of BRCA1 leading to triple negative breast cancer and poor patient prognosis. The BRCA1-BARD1 complex promotes homologous recombination (HR), which is the major pathway for the accurate repair of double-strand breaks. However, there is little information regarding the intricate roles fulfilled by BRCA1-BARD1 in this process, or how loss of specific BRCA1-BARD1 functions leads to tumorigenesis. BRCA1 has been previously reported to physically interact with both DNA and RAD51, key factors in HR, but the contributions of the interaction attributes to DNA damage repair remain unknown. Here, we delineate major sites of DNA and RAD51 binding in BRCA1 and use a combination of biochemical and NMR methods to identify the specific residues mediating interactions with these ligands. This has allowed us to develop mutations to ablate BRCA1’s ability to interact with these substrates without affecting BRCA1-BARD1’s interaction with other key DNA repair substrates. Using these BRCA1 mutants impaired for either DNA or RAD51 binding, we have interrogated the contributions of these interaction attributes to BRCA1-BARD1’s function by comparing the activity of WT to mutant BRCA1-BARD1 in biochemical assays to reconstitute various steps of HR. We have found that both DNA and RAD51 binding are indispensable for BRCA1-BARD1’s ability to promote RAD51-mediated D-loop formation, thus helping to delineate the mechanism by which BRCA1 promotes HR. Our studies provide the foundation to determine the functional consequences of cancer mutations in BRCA1-BARD1 and for the development of therapeutic strategies to target HR-deficient tumors.
 Citation Format: Angela Jasper, Hoang Dinh, Cody M. Rogers, Sameer Salunkhe, Hardeep Kaur, Antoine Baudin, David S. Libich, Patrick Sung. The Mechanistic Role of BRCA1 DNA and RAD51 Binding in DNA Double-Strand Break Repair [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-06.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19be3dd24b8cbf64d48385ebe0e058a4d636d0fd" target='_blank'>
              Abstract P5-06-06: The Mechanistic Role of BRCA1 DNA and RAD51 Binding in DNA Double-Strand Break Repair
              </a>
            </td>
          <td>
            Angela M Jasper, Hoang H Dinh, Cody M Rogers, Sameer Salunkhe, Hardeep Kaur, Antoine Baudin, David S. Libich, Patrick Sung
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Colon cancer is a highly aggressive solid tumor. Because most studies have focused on the intrinsic oncogenic pathways of tumors, we focused on the relationship between DNA methylation genes in the tumor immune microenvironment and colon cancer prognosis. We download RNA-seq data from the TCGA dataset. DNA methylation genes were scored using the GSVA method, and this score was subjected to GSEA, GO and KEGG enrichment analysis. Then, in vitro experiments examined the effect of silencing LARS2 on the proliferation and apoptosis of HCT116 cells. We obtained 2635 genes, 144 were obtained after single factor screening, and 8 genes were obtained through random forest dimensionality reduction. They are LARS2, TEX2, BRIP1, QSOX2, HOOK1, COX19, NEK4 and STXBP4. Survival analysis indicated that patients with high Riskscore had poor prognosis. There were significant differences in the expression of DNA methylation genes between the two groups with high and low Riskscore. The immune checkpoints of different Riskscore groups include Antigen present, Ligand, Receptor, Co-inhibitor, Co-stimulator, Other, and Cell adhesion. There were significant differences in the expression of genes and DNA integrity checkpoint, histone deacetylation, mitotic DNA damage checkpoint, TGF-beta signaling pathway, Platinum drug resistance and Protein processing in endoplasmic reticlum pathway at the level of Antigen present. Silencing of LARS2 decreased the proliferative capacity and increased the apoptosis rate of HCT116. Our findings suggest that LARS2 methylation could affects colon cancer development. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10669-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/146fe9925f0f237535e6dcfdafa1d3f52c10c07d" target='_blank'>
              LARS2 DNA methylation predicts the prognosis in colon cancer
              </a>
            </td>
          <td>
            Yingzhou Fu, Wei Xie, Qifan Ren, Jiao Wu, Ying Long, Ying Liu
          </td>
          <td>2025-07-22</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pediatric brain tumors, the most devastating cancers affecting children, are believed to originate from neural stem/progenitor cells in developing brain. In precise timing and specific regions during the brain development, chromatin deregulation plays crucial roles in redirecting normal neuronal differentiation pathways toward tumorigenesis. Indeed, epigenomic abnormalities are thought to be more important for brain tumor formation especially in children than adults, as pediatric brain tumors generally exhibit fewer genetic mutations compared to adult brain tumors. Given the small number of mutations, targeting such limited alterations involved in cancer epigenomes is expected to be more effective in pediatric brain tumors. The mechanisms of cancer epigenomes include mutation or dysregulation of chromatin remodelers, histone modifiers, histones themselves, and DNA methylation enzymes. Furthermore, genomic rearrangements and/or higher-order chromatin topology also contribute to these epigenomic mechanisms. These mechanisms are commonly observed in various types of pediatric brain tumors. However, alterations in chromatin regulatory factors differ across tumor types, reflecting the unique epigenetic landscapes shaped by their tumor origins. Accordingly, clarifying their functional similarities and differences across tumor types could offer valuable insights for finding new therapeutic strategies. Thus, this review article focuses on elucidating how pediatric brain tumors arise from epigenomic deregulation and what epigenetic molecules or mechanisms could serve as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a334571a5c8d012851148672d620a490ae61353" target='_blank'>
              Epigenetic modifications and their roles in pediatric brain tumor formation: emerging insights from chromatin dysregulation
              </a>
            </td>
          <td>
            Kento Kawata, Owen S. Chapman, Satoshi Narumi, Daisuke Kawauchi
          </td>
          <td>2025-06-17</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5484870331a7d0dee6989fa1ddf5ea8e0b01c6cb" target='_blank'>
              Functional genomics and tumor microenvironment analysis reveal prognostic biological subtypes in Mantle cell lymphoma
              </a>
            </td>
          <td>
            Sunandini Sharma, Roshia Ali, Alyssa Bouska, Dylan Jochum, Meghana Kesireddy, Simeon Mahov, Joseph Lownik, Weiwei Zhang, Waseem G Lone, Mahfuza Afroz Soma, Alicia Gamboa, Vaishnavi Devarkonda, Dalia Elgamal, Atqiya Fariha, Adnan Mansoor, Douglas Stewart, Peter Martin, Brian K. Link, R. Advani, Paul M. Barr, A. Goy, A. Mehta, M. Kamdar, D. Stephens, Veronika Bachanova, L. Smith, Ryan D Morin, P. Pararajalingam, M. Lunning, Kai Fu, Dennis Wiesenberger, Wing C. Chan, J. Khoury, T. Greiner, J. Vose, Akil Merchant, Chengfeng Bi, J. Iqbal
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea1387277252c00fad4b66b55a8349b13921b432" target='_blank'>
              MitoPerturb-Seq identifies common and gene-specific single-cell responses to mitochondrial DNA depletion and heteroplasmy
              </a>
            </td>
          <td>
            Stephen P. Burr, Kathryn Auckland, Angelos Glynos, Abhilesh S. Dhawanjewar, Wei Wei, Cameron Ryall, Antony M. Hynes-Allen, Malwina Prater, Matylda Sczaniecka-Clift, Julien Prudent, Patrick F. Chinnery, Jelle van den Ameele
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="ABSTRACT 2’-deoxyuridine (dU) is a common DNA lesion resulting from cytosine deamination and from dUMP incorporation by DNA polymerases, both of which are prevalent in cancer. The primary mechanism that repairs dU lesions in genomic DNA is base excision repair initiated by Uracil-DNA Glycosylase 1 (UNG1). We generated Ung knockout mouse B16 melanoma cells to investigate the consequences of UNG deficiency in a well-characterized, immunoproficient, syngeneic mouse cancer model. We show that UNG-deficient (ΔUNG) B16 tumors have altered growth kinetics in vivo and that their delayed growth is T-cell dependent. Immune profiling revealed reduced CD8 + T cell infiltration but augmented CD4 + Th1 responses in ΔUNG tumors. In vitro , ΔUNG tumor cells exhibit strongly suppressed cell-intrinsic type-I interferon, type-II interferon, and inflammatory signaling gene expression signatures as well as altered cytokine and chemokine secretion. In vivo , ΔUNG tumors exhibit a modified inflammatory cytokine and chemokine milieu. Furthermore, ΔUNG tumor cells have altered sensitivity to exogenous interferons in vitro , with increased sensitivity to IFN-γ but decreased sensitivity to IFN-α/β. Collectively, our data show that tumor cell-specific UNG deficiency results in an altered tumor microenvironment in vivo and provide proof-of-concept data for the use of UNG inhibitors to modulate inflammatory pathways in tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bc26c13b6ac7757d4f5f6a8fbbc2a735a2ecafa" target='_blank'>
              Uracil-DNA glycosylase deficiency is associated with repressed tumor cell-intrinsic inflammatory signaling and altered sensitivity to exogenous interferons
              </a>
            </td>
          <td>
            F. Vendetti, Pinakin Pandya, Carina R. Sclafani, Reyna E. Jones, Daniel Ivanov, Robert W. Sobol, Christopher J. Bakkenist
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1683b65b8227a142ec0662e619dfd830c76c770" target='_blank'>
              Major effect mutations drive DNA methylation variation after colonization of a novel habitat
              </a>
            </td>
          <td>
            Johan Zicola, Emmanuel Tergemina, Mehmet Göktay, Célia Neto, Robert J. Schmitz, Angela M. Hancock
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Retroviral transduction of cancer cell lines has been used to find genes related to therapy resistance. Characterization of virus integration sites (VIS) pinpointing these genes has been cumbersome. This study defines a sequencing-based protocol to rapidly characterize genomic loci near VIS. The protocol selectively amplifies VIS-genome junctions using the retroviral vector neomycin (NEO) gene, Genomic Walker Adapter approach, linker-mediated NEO-PCR (LM-NEO-PCR) or biotinylated NEO-capture, followed by long terminal repeats PCR (LTR-PCR). LTR-genome junctions were sequenced (NGS), reads mapped, quantified, and linked to genes. The protocol was tested on DNA from single clones holding twenty reported VIS loci and on multiplex and diluted clone DNA samples. Our VIS-NGS protocol enriched significantly (p < 0.02) more loci at high reads coverage in samples with VIS compared to negative controls. The protocol found seventeen reported VIS loci (85%) in single clone DNAs, of which fifteen loci (88%) were also detected in multiplex samples. Six loci were evaluated for all dilutions, with three loci detected at lowest 1% clone proportion. The protocol can be conducted in two weeks and successfully found almost all VIS loci in single VIS clones and detected half of the evaluated loci at low clone proportion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc457805a586e034f6d53bc20d3aa71d5bdfc7ad" target='_blank'>
              Novel protocol for mapping virus integration sites in genes involved in therapy resistance
              </a>
            </td>
          <td>
            M. Massimino, Zhangzan Huang, J. Helmijr, Camilla Muritti, Anna Uijtdewillegen, P. Vigneri, John W M Martens, Maurice P H M Jansen
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Gene regulation often entails a cooperative dynamic interplay among several protein molecules and several distinct DNA segments. Intersegment transfer of the Fis protein stimulates DNA inversion during DNA recombination. Individual DNA segments have been found to facilitate the dissociation of Fis proteins already bound to DNA and also allow for the transfer of the Fis between segments. Here, we use the hybrid coarse-grained AWSEM/3SPN.2C model to simulate the Fis protein intersegment transfer and explore its mechanism. We show that entropic effects within the Fis protein-DNA complex dictate the transfer pathway through specific structural configurations, involving specific grooves and consequent orientation constraints on the DNA segments. Multiple copies of the Fis protein facilitate intersegment transfer, explaining how changes in protein-DNA stoichiometry and concentration influence how the Fis-DNA complex architecture is established. This orientational dependence indicates that the assembly of the Fis-DNA complex mimics the interlocking of screws, functioning as a molecular machine that may couple to DNA supercoiling and torsional stress in the DNA generated by motor proteins, thus offering a potential regulatory mechanism for chromosomal organization and gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5a31cc6e4951420f062570d943b327e0539b5ca" target='_blank'>
              Intersegment Transfer and the Dynamical Architecture of Fis Protein-DNA Multimer Complexes.
              </a>
            </td>
          <td>
            Xun Chen, Shikai Jin, Cheng-Han Liu, Yaakov Levy, Min-Yeh Tsai, Peter G. Wolynes
          </td>
          <td>2025-08-05</td>
          <td>Journal of the American Chemical Society</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Endometrial carcinoma (EC), the most common gynecologic cancer type in developed countries, encompasses four molecular subtypes (POLEmut, MMRd, p53abn, and NSMP) that have prognostic values and guide treatment decisions. Additionally, dual loss of ARID1A and ARID1B (referred to as ARID1A/B) characterizes a significant portion of dedifferentiated/undifferentiated EC (DD/UDEC), a rare but highly aggressive subtype of EC. To advance the translational research for ECs, we analyzed the genomic features of a panel of 39 EC cell lines, leading to the identification of cell lines representing each of these EC molecular subtype. Histologic and immunohistochemical analyses of xenografted tumors from these cell lines confirmed their resemblance of cognate primary EC molecular subtypes. Further investigation of the publicly available genome-wide CRISPR screen data for EC cell lines identified multiple specific genetic dependencies in MMRd, p53abn, and ARID1A/B-dual deficient EC cell lines. Particularly, ARID1A/B-dual deficient DD/UDEC cells selectively rely on mitochondrial oxidative phosphorylation in vitro and in vivo. Therefore, through molecular subtyping of EC cell lines and subsequent characterization of molecular subtype-specific genetic dependencies, our study provides a framework that guides the utility of the EC cell line models for accelerating translational research in EC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e1f3b25336fe4feba408a071575d0aa7bd9be8e" target='_blank'>
              Molecular subtyping of endometrial carcinoma cell lines uncovers subtype-specific targetable vulnerabilities
              </a>
            </td>
          <td>
            Eunice Li, Rebecca Ho, Ran Tao, Yannes Wai Yan Choi, Chae Young Shin, Shary Chen, Bengul Gokbayrak, J. Senz, Betty Yao, Liam Johnston, Spencer D Martin, Eric Yang, Mark S Carey, Bryan T. Hennessy, D. Huntsman, Ramon I Klein Geltink, Lynn Hoang, Yemin Wang
          </td>
          <td>2025-07-24</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Background: Often, neoadjuvant therapy, which relies on the induction of double-strand breaks (DSBs), is used prior to surgery to shrink tumors by inducing cancer cell apoptosis. However, recent studies have suggested that this treatment may also induce a fluctuating state between senescence and stemness in PA-1 embryonal carcinoma cells, potentially affecting therapeutic outcomes. Thus, the respective epigenetic pathways are up or downregulated over a time period of days. These fluctuations go hand in hand with changes in spatial DNA organization. Methods: By means of Single-Molecule Localization Microscopy in combination with mathematical evaluation tools for pointillist data sets, we investigated the organization of euchromatin and heterochromatin at the nanoscale on the third and fifth day after etoposide treatment. Results: Using fluorescently labeled antibodies against H3K9me3 (heterochromatin tri-methylation sites) and H3K4me3 (euchromatin tri-methylation sites), we found that the induction of DSBs led to the de-condensation of heterochromatin and compaction of euchromatin, with a peak effect on day 3 after the treatment. On day 3, we also observed the co-localization of euchromatin and heterochromatin, which have marks that usually occur in exclusive low-overlapping network-like compartments. The evaluation of the SMLM data using topological tools (persistent homology and persistent imaging) and principal component analysis, as well as the confocal microscopy analysis of H3K9me3- and H3K4me3-stained PA-1 cells, supported the findings that distinct shifts in euchromatin and heterochromatin organization took place in a subpopulation of these cells during the days after the treatment. Furthermore, by means of flow cytometry, it was shown that the rearrangements in chromatin organization coincided with the simultaneous upregulation of the stemness promotors OCT4A and SOX2 and senescence promotors p21Cip1 and p27. Conclusions: Our findings suggest potential applications to improve cancer therapy by inhibiting chromatin remodeling and preventing therapy-induced senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a3f66309c268ad41eb3fdc8156d8889c120362" target='_blank'>
              Chemotherapy (Etoposide)-Induced Intermingling of Heterochromatin and Euchromatin Compartments in Senescent PA-1 Embryonal Carcinoma Cells
              </a>
            </td>
          <td>
            Marc Bayer, Jaroslava Zajakina, Myriam Schäfer, K. Salmina, Fēlikss Rūmnieks, J. Jansons, F. Bestvater, R. Kurg, J. Erenpreisa, Michael Hausmann
          </td>
          <td>2025-07-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="XPO1 is a nuclear export receptor that is essential for cell survival. Previous genomic analyses have identified recurrent XPO1 hotspot mutations in cancer. Here, we conducted a large-scale genomic analysis of 217,570 cancer patients to identify and characterize XPO1 variants from real-world patient tumors. XPO1 harbored a R749Q mutation in various solid tumors, with a clear enrichment in endometrial and colorectal cancers, and XPO1R749Q mutations significantly co-occurred with POLE mutations. Analysis of isogenic colon cancer cell lines revealed that XPO1R749Q localized more in the cytoplasm than wildtype XPO1, with enhanced export of a large number of proteins. Structural modeling of XPO1R749Q suggested an increase in RanGTP affinity, which is consistent with enhanced protein export. A compound library screen using over 200 FDA-approved anticancer drugs indicated a general trend towards chemoresistance, specifically to topoisomerase I inhibition, in XPO1R749Q mutant cells. Mechanistically, XPO1R749Q mutant cells exhibited enhanced DNA damage response via RPA phosphorylation in response to topoisomerase I inhibition. Combining XPO1 and topoisomerase I inhibitors reduced DNA damage-induced RPA phosphorylation and mediated synergistic antitumor effects in cells harboring the XPO1R749Q mutation. Finally, the combination of selinexor and irinotecan overcame chemotherapeutic resistance in xenograft mouse models, prolonging survival. These findings suggest that XPO1 alterations in cancer are selected for in POLE mutant tumors and may confer resistance to DNA-damaging chemotherapies, which have implications for patients with tumors bearing XPO1R749Q and for XPO1 inhibitor development in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e52bf0adf008e92bc55a03de218e8fe730ea2f" target='_blank'>
              XPO1R749Q Mutations Co-occur with POLE Mutations in Cancer and can be Targeted to Overcome Chemoresistance.
              </a>
            </td>
          <td>
            T. Totiger, Wannasiri Chiraphapphaiboon, Y. Baca, S. Chaudhry, R. Notti, S. Montoya, M. Chojnacka, Gabriel Gaidosh, Jumana Afaghani, Maurizio Affer, Chris Armstrong, Paul M Kavanaugh, Efe Karaca, Jenna Zabroski, Michael Lewis, Alyssa Maye, Jacob Jahn, R. Soni, Daniel Bilbao, Phil Walker, Andrew Elliott, E. Lou, Wafik S. El-Deiry, Josean Rodriguez, Hai Dang Nguyen, Justin Taylor
          </td>
          <td>2025-06-18</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="ABSTRACT RNA epigenetics, also referred to as epitranscriptomics, has emerged as a critical regulatory layer in cancer biology, extending beyond the scope of traditional DNA and histone modifications. It encompasses a series of dynamic posttranscriptional processes—including RNA biosynthesis, splicing, transport, stability, degradation, translation, and chemical modifications—orchestrated by RNA‐binding proteins (RBPs) and noncoding RNAs (ncRNAs). Collectively, these mechanisms influence mRNA fate, shape transcriptional output, and reprogram the tumor microenvironment. Importantly, both coding RNA and ncRNA are themselves subjected to epigenetic regulation, forming intricate feedback loops that contribute to oncogenesis, immune evasion, metastasis, and therapeutic resistance. In this review, we systematically synthesize the current understanding of RNA metabolism and RNA epigenetic modifications during tumor progression, with a particular focus on the roles of RBPs and RNA modifications. Furthermore, we highlight recent advances in RNA‐based therapeutic strategies, including mRNA vaccines, antisense oligonucleotides, siRNAs, and circRNA scaffolds. These innovative approaches offer promising avenues for targeting transcriptionally active yet genomically “undruggable” cancer drivers. Together, our synthesis provides a comprehensive framework for understanding RNA epigenetics in tumor biology and lays the groundwork for precision oncology guided by transcriptome plasticity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de7fa009ea26201e23b24a2c7b7bd45eb23b5831" target='_blank'>
              RNA Epigenetics in Cancer: Current Knowledge and Therapeutic Implications
              </a>
            </td>
          <td>
            Shanhe Huang, Zonglin Li, Weilong Lin, Ruihui Xie, Hai Huang
          </td>
          <td>2025-08-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Extensively studied over the past four decades, the TP53 gene has emerged as a pivotal watchman in cellular defense and a key factor in cancer biology. TP53 is the most frequently mutated gene in human malignancies, 50% of which carry alterations to it. Initially, the functions of p53 were thought to be restricted to cell‐cycle arrest and apoptosis. With time, however, a growing number of functions have been discovered, illustrating p53's role as a master switch between any cellular stress and cellular or multicellular responses that contribute to its anti‐tumor activity. Indeed, the peculiar landscape of TP53 mutations and its high heterogeneity are linked both to the structure of the protein and its ubiquitous function in regulating cellular homeostasis. Mutations in p53 are associated with poor response to therapy and shorter survival in most cancer types, and the diagnosis of p53 mutations is currently used to improve case management in some types of leukemia and lymphoma. Although TP53 has been defined as a tumor suppressor gene, overexpressed mutated p53 variants found in human tumors are defined as dominant oncogenes with a potential gain of function, which makes the gene a very attractive target for developing new cancer treatments. Beyond its role in cancer, our review also highlights TP53's significance in non‐neoplastic conditions, such as bone marrow failure syndromes and certain developmental disorders, where chronic p53 activation plays a crucial role in cellular stress responses, demonstrating its broader biological importance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f422279b37d3fc7b3177be1ae5e2e34940dcc8f7" target='_blank'>
              The TP53 tumor suppressor gene: From molecular biology to clinical investigations
              </a>
            </td>
          <td>
            Panagotis Baliakas, Thierry Soussi
          </td>
          <td>2025-06-16</td>
          <td>Journal of Internal Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8503e81194e8e95ebd9edbe7edc4c0fd9d960853" target='_blank'>
              Mechanisms of resistance to VHL loss-induced genetic and pharmacological vulnerabilities
              </a>
            </td>
          <td>
            Jianfeng Ge, Shoko Hirosue, Saroor A. A. Patel, L. Wesolowski, Anna Dyas, Cissy Yong, Leticia Castillon, Sanne de Haan, J. Drost, Grant D. Stewart, Anna C Obenauf, D. Muñoz-Espín, Sakari Vanharanta
          </td>
          <td>2025-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background: The long-term goal of chronic hepatitis B research is a functional cure (HBsAg seroclearance). Although currently used nucleos(t)ide analogs can efficiently inhibit viral replication, they do not reduce viral RNAs or proteins produced from covalently closed circular DNA (cccDNA), and rarely achieve a functional cure. To overcome this situation, revealing the mode of the existence of cccDNA is required, including identifying the interreacting proteins with cccDNA. Here, we aimed to identify novel proteins that interact with cccDNA. Methods: Using an in vitro HBV infection model and a sequence-specific proximity labelling method consisting of dead Cas9 and biotin identification (BioID2), we comprehensively determined proteins that possibly interact with cccDNA. After identifying the candidate proteins, the HBV RNA transcription levels were examined by knocking out the associated genes. Results:﻿ We identified ABHD14B as a protein that interacts with cccDNA and inhibits HBV RNA transcription from cccDNA. ABHD14B decreases the acetylation levels of histone proteins that control the transcription levels of HBV RNA in cccDNA. Moreover, ABHD14B interacts with TFII-I, which binds directly to cccDNA in a sequence-dependent manner. These results suggest that the host protein, ABHD14B, is recruited to cccDNA via the TFII-I protein, inhibiting HBV RNA transcription from cccDNA by deacetylating cccDNA histones. Conclusions: ABHD14B was newly identified as a suppressor of HBV RNA transcription from cccDNA, which may improve our understanding of the mode of existence of cccDNA, providing a basis for development of a functional cure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7bd95fe07e8a95469056b65d704b3979e1b6c0b9" target='_blank'>
              CRISPR-mediated proximity labeling unveils ABHD14B as a host factor to regulate HBV cccDNA transcriptional activity
              </a>
            </td>
          <td>
            Kazuma Sekiba, Nozomi Miyake, Yu Miyakawa, Chikako Shibata, Takahiro Seimiya, T. Kishikawa, Mitsuhiro Fujishiro, Motoyuki Otsuka
          </td>
          <td>2025-07-14</td>
          <td>Hepatology Communications</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Gene expression is finely controlled by the abundance and activation status of transcription factors and their regulators, as well as by a number of reversible modifications of DNA and histones that are commonly referred to as epigenetic marks. Such alterations (i.e., methylation, acetylation, and ubiquitination) are catalyzed by an array of dedicated enzymes with antagonistic activity, including methyltransferases and demethylases, acetyltransferases and deacetylases, as well as ubiquitin ligases and deubiquitinating enzymes. The epigenetic control of transcription is critical not only for embryonic and postembryonic development but also for the preservation of homeostasis in all adult tissues. In line with this notion, epigenetic defects have been associated with a variety of human disorders, including (but not limited to) congenital conditions as well as multiple hematological and solid tumors. Here, we provide an in-depth discussion of the impact of epigenetic alterations on cancer stemness, i.e., the ability of a small population of poorly differentiated malignant cells to (1) self-renew while generating a more differentiated progeny, and (2) exhibit superior tumor initiating/repopulating potential along with exceptional plasticity and improved resistance to environmental and therapy-elicited stress. Moreover, we critically evaluate the potential and limitations of targeting epigenetic modifiers as a means to eradicate cancer stem cells for therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3477193515567caf9d7adf26496dfc00e067cba2" target='_blank'>
              Epigenetic regulation of cancer stemness
              </a>
            </td>
          <td>
            C. Galassi, G. Manic, Manel Esteller, Lorenzo Galluzzi, Ilio Vitale
          </td>
          <td>2025-08-01</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="DNA exists biologically as a highly dynamic macromolecular complex subject to myriad chemical modifications that alter its physiological interpretation, yet most sequencing technologies only measure Watson-Crick base pairing interactions. Third-generation sequencing technologies can directly detect novel and modified bases, yet the difficulty and cost of training these techniques for each novel base has so far limited this potential. Here, we present a method based on barcoded split-pool synthesis to generate reference standard oligonucleotides allowing novel base sequencing. Using novel base detection, we perform multidimensional sequencing to retrieve information, both physiologically stored and experimentally encoded, from DNA, allowing us to characterize the preferential replication of deleterious mitochondrial genome mutations, the infection dynamics of a host-pathogen model, and the effect of chemotherapy on cancer cell DNA at the single molecule level. The low cost and experimental simplicity of this method make this approach widely accessible to the research community, enabling complex experimental interrogation across the biological sciences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72de8476da20b409721bf544eb2f184f300e0091" target='_blank'>
              Multidimensional third-generation sequencing of modified DNA bases allows interrogation of complex biological systems
              </a>
            </td>
          <td>
            Serena S David, Brendan A Pacheco, Kensei Kishimoto, Sam Vantine, Kai Hu, Haibo Liu, Diana L Davis, Hoang Tran, Benjamin F Sallis, Levi Ali, Cole M. Haynes, Beth A McCormick, L. Zhu, William A Flavahan
          </td>
          <td>2025-07-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="DEAD-box helicase 41 (DDX41) is implicated in germline (GL)-predisposed myeloid neoplasms, where pathogenic GL variants often lead to disease following the acquisition of a somatic variant in trans, most commonly p.R525H. However, the precise molecular mechanisms by which DDX41 variants contribute to the pathogenesis of myeloid neoplasms remain poorly understood, partly due to challenges in establishing cellular and animal models that faithfully recapitulate the human disease phenotype. This limitation highlights the necessity of directly analyzing primary human disease cells. In this case report, conducted to pursue this objective, we implemented single-cell RNA sequencing integrated with genotyping at the p.R525 locus in a myelodysplastic neoplasm (MDS) harboring both germline and somatic DDX41 variants, leveraging highly efficient Terminator-Assisted Solid-phase cDNA amplification and sequencing. We found that acquiring p.R525H induced G2/M cell cycle arrest selectively in colony-forming unit-erythroid cells, accompanied by R-loop accumulation, which impaired erythropoiesis through DNA damage. In hematopoietic stem and myeloid progenitor populations, gene expression profiles were largely similar between p.R525H-positive and -negative cells. However, ligand-receptor interaction and transcriptional regulation analyses suggested a non-cell-autonomous influence from p.R525H-expressing cells on GL variant-only cells. This interaction drove convergence toward a shared expression profile, highlighting an intricate interplay shaping the patient’s MDS phenotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43fbd448cf6560b9e09e5f237fa9deac7aa303c9" target='_blank'>
              Genotype-integrated single-cell transcriptome analysis reveals the role of DDX41 pR525H in a patient with myelodysplastic neoplasms
              </a>
            </td>
          <td>
            Hirotaka Matsui, Akiko Nagamachi, Minori Koizumi, Rei Kudo, Masahiko Ajiro, H. Harada, Yuka Harada, Shigeyuki Shichino, Akihide Yoshimi
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Structural variations (SVs) represent genomic variations that involve breakage and rejoining of DNA segments. SVs can alter normal gene dosage, lead to rearrangements of genes and regulatory elements within a topologically associated domain, and potentially contribute to physical traits, genomic disorders, or complex traits. Recent advances in sequencing technologies and bioinformatics have greatly improved SV detection and interpretation at unprecedented resolution and scale. Despite these advances, the functional impact of SVs, the underlying SV mechanism(s) contributing to complex traits, and the technical challenges associated with SV detection and annotation remain active areas of research. This review aims to provide an overview of structural variations, their mutagenesis mechanisms, and their detection in the genomics era, focusing on the biological significance, methodologies, and future directions in the field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f005d90812c5790ee3488e2503f389fb8c3ab57b" target='_blank'>
              Structural Variants: Mechanisms, Mapping, and Interpretation in Human Genetics
              </a>
            </td>
          <td>
            Shruti Pande, Moez Dawood, Christopher M. Grochowski
          </td>
          <td>2025-07-29</td>
          <td>Genes</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e7f45bb9128c765dc6171cc504564b0c6834197" target='_blank'>
              SMCHD1 compacts DNA directly in an ATP-regulated manner
              </a>
            </td>
          <td>
            Joel Ng, Xiaodan Zhao, Xuyao Liu, Aghil Soman, Lars Nordenskiöld, Jie Yan, Shifeng Xue
          </td>
          <td>2025-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is the most common renal malignancy. However, the combined clinical and biological scores commonly used to predict the clinical outcome are imperfect and need improvement. The main goal of our study was to assess the effect of mtDNA genetics on the prognosis of ccRCC patients and to explore morphologic correlation. Mitochondrial DNA copy number (mtDNAcn) variation between tumor and paired matched healthy kidney tissue was assessed by real-time quantitative PCR and expressed as a ratio in 105 patients. According to this median ratio, the cohort was divided into two groups: “LOW” (n = 53) and “HIGH” (n = 52). Cancer-Specific Survival (CSS) and Disease-Free Survival were assessed in each group. The tumor samples were classified into two subtypes (Clear or Eosinophilic cells) according to the cytoplasmic morphology. CSS was significantly reduced in the “HIGH” than in the “LOW” group with respective 5-year survival rates: 78.7% (CI 95: 64.8–95.5) and 95.5% (CI 95 87.1–100.0) (Hazard Ratio: 7.4 (CI 95: 1.9–29.9, p = 0.027*) in multivariate analysis, including pathological classification, tumor size, International Society of Urological Pathology grade, lymphovascular invasion, dedifferentiated pattern, necrosis and adjuvant therapy. Next-generation sequencing of mtDNA was performed on 14 tumors and matched healthy kidney tissue. No hotspot mutation or redundant large deletion was found. None of the variants or large deletions identified had an impact on prognosis. MtDNAcn variation in tumor relative to normal kidney appears as an independent prognostic factor in ccRCC, which was also associated with eosinophilic morphology. MtDNA content could be considered an additional prognostic factor, in combination with other predictive parameters. Furthermore, these results underline the importance of the role of mitochondria in ccRCC and the need for further functional studies to understand the pathophysiological mechanisms better and consider therapies targeting mitochondrial metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288c14d8ba3e745b3421617281d655151662ced6" target='_blank'>
              MtDNA copy number enrichment is associated with poor prognosis and eosinophilic morphology in clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            Sarah Bellal, C. Rolley, Jérémy Richard, Nolwenn Bounaix, Vincent Le Corre, Marie-Christine Copin, Odile Blanchet, Pierre Bigot, Vincent Procaccio, C. Bris
          </td>
          <td>2025-07-23</td>
          <td>Pathology and Oncology Research</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b94cf63206b221ede960c719c7b5d998ebadf2b" target='_blank'>
              Specific TP53 mutations impair the recruitment of 53BP1 to DNA double-strand breaks underlying the mechanism of radioresistance.
              </a>
            </td>
          <td>
            Paolo Fagherazzi, L. Stixová, E. Bártová
          </td>
          <td>2025-07-14</td>
          <td>European biophysics journal : EBJ</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The nucleotide excision repair (NER) mechanism is responsible for removing bulky DNA damage, such as pyrimidine dimers induced by ultraviolet (UV) light. The NER pathway excises the damaged strand through incisions at the 5’ and 3’ ends of the damage, with the 5’ incision catalyzed by the XPF-ERCC1 endonuclease complex. Here, we identify an XPF ortholog in Trypanosoma brucei, the causative agent of human African trypanosomiasis (sleeping sickness). XPF-deficient parasites exhibit hypersensitivity to UV irradiation and a slower rate of DNA damage repair. Consistent with its role in DNA repair, XPF localizes to the nucleus, associating with nucleoplasmic and nucleolar regions. Additionally, we demonstrate that TbXPF protects against intra- and inter-strand crosslinks induced by cisplatin and mitomycin C, respectively. The presence of a functional NER pathway in trypanosomes suggests that these organisms are susceptible to replication- and transcription-blocking DNA damage in vivo. Under genotoxic stress, genome stability and parasite survival may heavily rely on DNA repair systems such as NER which, for this reason, could be an effective target for chemotherapeutic interventions. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-08659-y.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbdd4ff3d6a3681f1e698b78ba5f4c131040ae39" target='_blank'>
              Role of the nucleotide excision repair endonuclease XPF in the kinetoplastid parasite Trypanosoma brucei
              </a>
            </td>
          <td>
            Claudia Gomez-Liñan, María Sáez-Maldonado, Laura Montosa-Hidalgo, L. Ruiz‐Pérez, Dolores González-Pacanowska, Antonio E. Vidal
          </td>
          <td>2025-07-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="STUDY QUESTION
Are segmental aneuploidies identified in human embryos more likely to occur within known fragile sites of the genome?


SUMMARY ANSWER
Segmental breaks in the autosomes of human preimplantation embryos occur more frequently in known fragile areas of the genome.


WHAT IS KNOWN ALREADY
Fragile sites represent specific loci in the genome characterized by inhibition of DNA synthesis when exposed to known inhibitors and are particularly sensitive to replication stress and instability.


STUDY DESIGN, SIZE, DURATION
This was a retrospective analysis of single nucleotide polymorphism (SNP) array-based preimplantation genetic testing data from biopsies performed on 2066 human blastocysts in 98 assisted reproduction laboratories around the world from September 2019 to January 2023.


PARTICIPANTS/MATERIALS, SETTING, METHODS
This multicenter study included eligible patients undergoing IVF with preimplantation genetic testing (PGT), in which at least one embryo was diagnosed with a segmental aneuploidy. The mean maternal age was 36.4 years (SD 4.1), ranging from 25 to 44 years. These samples were processed on high-density SNP arrays. Chromosome level copy number and B allele frequency (BAF) plots from these embryos were used to determine segmental aneuploidy breakpoints. Known fragile sites catalogued by the HumCFS database were used for correlation analyses.


MAIN RESULTS AND THE ROLE OF CHANCE
Overall, a side-by-side pairing of observed breakpoints and known fragile sites demonstrated a strong concordance (r = 0.81, 95% CI [0.6, 0.92]). A chi-square test for independence for stratified groups showed a highly significant correlation between all observed breakpoints and known fragile sites (597 expected vs. 848 observed; P < 0.001) and for telomeric breaks alone (521 expected vs. 784 observed; P < 0.001). Observed interstitial breaks alone were not correlated to expected breakpoints (75 expected vs. 64 observed; P > 0.05).


LIMITATIONS, REASONS FOR CAUTION
These findings should be interpreted with caution, as limitations in genomic resolution may bias detection and classification of smaller segmental aneuploidies. Additionally, this study touched upon the distribution of meiotic to mitotic breakpoints in human blastocysts as they relate to known fragile sites. Since meiotic aneuploidies increase with advanced maternal age and many IVF patients undergoing PGT-A testing fall in this category, a sampling bias should be considered for this specific metric.


WIDER IMPLICATIONS OF THE FINDINGS
Demonstrating that segmental aneuploidies significantly correlate with known fragile sites highly susceptible to replication stress offers insight into the origin of subchromosomal imbalances and hints at the influence of stressors on reproductive success.


STUDY FUNDING/COMPETING INTEREST(S)
This study received no external funding and was fully supported by the participating authors and their affiliated institutions. The authors declare no conflicts of interest related to this study.


TRIAL REGISTRATION NUMBER
N/A.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14f488ce17d9309a0718d17d57c1fee173909b44" target='_blank'>
              Blastocyst segmental aneuploidy breakpoints are highly correlated with human genome fragile sites.
              </a>
            </td>
          <td>
            Elaine de Quadros, Jia Xu, Nathan Treff, Diego Marin, Arielle Freedman, C. Milevski, Kathleen Miller, Minglei Bian
          </td>
          <td>2025-07-30</td>
          <td>Human reproduction</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8da4d10721badd9a65f0ec25fedb84c3d3183aaf" target='_blank'>
              Direct functional HOXA9/DNA-binding competitors versus epigenetic inhibitors of HOXA9 expression on cell proliferation, death and differentiation processes in the model of MLL-rearranged acute myeloid leukemia.
              </a>
            </td>
          <td>
            Julie Vrevin, Mélanie Lambert, Marine Andrique, Nathalie Jouy, M. David-Cordonnier
          </td>
          <td>2025-08-04</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract The MYC and RAS oncogenes were found early on to cooperate in the transformation of nonimmortalized primary cells, and in tumor development in transgenic mouse models. MYC prevents RAS-induced senescence. Moreover, tumor regression resulting from the suppression of MYC expression is associated with cellular senescence. How MYC prevents RAS-induced senescence and why suppression of MYC in tumors causes senescence remains to be elucidated. Here, we show that MYC interacts with the Pol β DNA polymerase and stimulates its enzymatic activities to accelerate the repair of oxidative DNA damage by the base excision repair (BER) pathway, thereby enabling RAS-driven cancer cells to avoid senescence and continue to proliferate despite producing excess levels of reactive oxygen species. The carboxy-terminal domain of MYC, which is needed for heterodimerization with its MAX partner and DNA binding, is not required for the DNA repair activity of MYC. Ectopic expression of MYCΔCTD accelerates DNA repair, protects against RAS-induced senescence, and cooperates with RAS in the transformation of primary cells, whereas MYC mutants inactive in DNA repair exhibit weaker or no activity in these assays. These results demonstrate that the function of MYC in BER plays an important role in the MYC–RAS cooperation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c13c7980326ac55cbce4b685dfa42090cd38be7" target='_blank'>
              The function of MYC in base excision repair protects against RAS-induced senescence
              </a>
            </td>
          <td>
            C. C. F. Faraco, Wanting Zhu, Anne-Marie Fortier, Z. Ramdzan, Elise Vickridge, Virakbott Ong, Félix Goudreau, Billel Djerir, Jérôme Bellemare, Shujing Gu, Huanyi Zhang, Brittany L Carroll, Shreya S Krishnan, L. Leduy, Alba Guarné, Morag Park, Alexandre Maréchal, Alain Nepveu
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Multicellular organisms arise from a single genome template in the zygote, necessitating the cells of the developing embryo to up- and downregulate specific genes to establish and maintain their identity. This template is maintained, propagated, and interpreted as chromatin, a polymer of nucleic acids and associated structural and regulatory proteins. Recent genome-wide surveys documented a wealth of disease-associated mutations in chromatin factors, indicating their fundamental significance and potential for therapeutic targeting. However, chromatin factors exist in a complex balance, with a single deficiency often leading to pleiotropic downstream effects. Here, we review the mechanisms of chromatin regulation and partitioning, highlighting examples of how these processes are altered in human diseases. We argue that loss of chromatin fidelity, both locally at specific genes and regulatory elements, and globally at the megabase-scale, contributes to many pathological states and may thus represent an intriguing target for corrective interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70b3c6ccb1965f22efefabec61c021483fcae985" target='_blank'>
              Aere Perennius: How Chromatin Fidelity is Maintained and Lost in Disease.
              </a>
            </td>
          <td>
            Dustin Fetch, Amina Jumamyradova, Cameron Chapa, Yong Ge, Mansour Mohamadzadeh, Alexey A Soshnev
          </td>
          <td>2025-07-22</td>
          <td>NAR Molecular Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are distinct nucleic acid secondary structures formed by guanine-rich sequences in both DNA and RNA. These structures readily form and fulfill diverse biological functions. The structural diversity of G4s is influenced by several factors, including their strand orientation, glycosidic bond angles, and loop configurations. G4s are widely distributed in functionally significant genomic regions, including telomeres, promoter regions, exons, 5' untranslated region (5' UTR), intron region, and 3' untranslated region (3' UTR). G4s are implicated in critical biological processes, including telomere elongation, DNA replication, DNA damage repair, transcription, translation, and epigenetic regulation. This overview offers a comprehensive analysis of the determinants of G4 structure and their impact on associated biological processes. Briefly, it describes the effects of G4s on cancers, viruses, and other pathogenic substances. This overview aims to contribute new ideas for the regulation of related mechanisms and their potential impact on the treatment strategies of related diseases. This article is categorized under: RNA Structure and Dynamics > RNA Structure, Dynamics and Chemistry RNA Structure and Dynamics > Influence of RNA Structure in Biological Systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0eb5827dbe52a8e0e2bc9b47ded6461cd148b5b" target='_blank'>
              G-Quadruplexes in Gene Regulation and Cellular Function.
              </a>
            </td>
          <td>
            Qi Wen, Lijin Guo, Farhad Bordbar, Qinghua Nie
          </td>
          <td>2025-07-01</td>
          <td>Wiley interdisciplinary reviews. RNA</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background The mammalian Tumor Susceptibility Gene 101 (TSG101) encodes a protein with diverse functions that control the proliferation and survival of cells, but its role in malignant transformation and cancer development has remained enigmatic. Methods To study the pro-tumorigenic functions of TSG101, we developed a bi-transgenic mouse model that expresses exogenous TSG101 along with a luciferase reporter in a ligand-controlled manner in the mammary gland epithelium. We performed a comprehensive histopathologic, biochemical, and molecular characterization of ductal hyperplasia and mammary tumors. Unsupervised hierarchical clustering based on 1,723 intrinsic genes of ten TSG101-overexpressing cancers alongside 251 tissue samples representing 31 reference mammary tumor models and normal mammary glands was conducted. Results Females overexpressing TSG101 develop ductal hyperplasia, adenomyoepitheliomas, and palpable adenosquamous carcinomas at an average latency of approximately ten months. These metaplastic mammary tumors are comprised of transforming basal and luminal epithelial cells. Using a GFP reporter strain to monitor the transgene activation at the single-cell level, we determined that the epithelial heterogeneity within transforming ducts and ensuing carcinomas originated from the luminal epithelium. At the molecular level, TSG101-induced mammary tumors are triple-negative and exhibit gene expression signatures of Wnt and inflammatory cytokine signaling, which are key regulators of epithelial cell fate. The ligand-controlled downregulation of exogenous TSG101 in established carcinomas led to tumor regression. We demonstrated that the TSG101-mediated activation of PI3K/AKT signaling, as well as upregulation of Cyclin D1 and MDM2, are dependent on the perpetual expression of the TSG101 oncoprotein. Conclusions The collective findings of this study provide in vivo evidence that TSG101 possesses pro-tumorigenic properties that extend to cancer progression and maintenance, suggesting that this protein could be a rational molecular target to prevent and treat a subset of mammary tumors. Supplementary Information The online version contains supplementary material available at 10.1186/s13058-025-02007-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca345fd462ac7f96e7ae408ea2e3598885c80b06" target='_blank'>
              Overexpression of TSG101 causes the development of adenosquamous mammary carcinoma
              </a>
            </td>
          <td>
            Rayane Dennaoui, Patrick D. Rädler, Madison N. Wicker, Kerry Vistisen, R. Ferraiuolo, A. Triplett, Hridaya Shrestha, Tessa A. Liner, Karoline C. Manthey, Hallgeir Rui, R. Cardiff, Teresa M. Gunn, C. Perou, Kay-Uwe Wagner
          </td>
          <td>2025-07-07</td>
          <td>Breast Cancer Research : BCR</td>
          <td>1</td>
          <td>164</td>
        </tr>

        <tr id="Cyclin dependent kinases 4 and 6 inhibitors have brought great improvements in the treatment of luminal breast cancer, but resistance is a major clinical hurdle. Multiple biomarkers of resistance have been proposed, but none is currently utilized in clinical practice. By performing single-cell RNA sequencing of seven palbociclib-naïve luminal breast cancer cell lines and palbociclib-resistant derivatives, we show that established biomarkers and pathways related to CDK4/6i resistance present marked intra- and inter- cell-line heterogeneity. Transcriptional features of resistance could be already observed in naïve cells correlating with levels of sensitivity (IC50) to palbociclib. Resistant derivatives showed transcriptional clusters that significantly varied for proliferative, estrogen response signatures or MYC targets. This marked heterogeneity was validated in the FELINE trial where, compared to the sensitive ones, ribociclib-resistant tumors developed higher clonal diversity at genetic level and showed greater trascriptional variability for genes associated with resistance. A potential signature of resistance inferred from the cell-line models, positively enriched for MYC targets and negatively enriched for estrogen response markers, was probed on the FELINE trial, separating sensitive from resistant tumors and revealing higher heterogeneity in resistant versus sensitive cells. These data suggest that heterogeneity for CDK4/6 inhibitors resistant markers might facilitate the development of resistance and challenge the validation of clinical biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab65930181b447a2afea51540e3bba84721ca231" target='_blank'>
              Single-cell transcriptomics reveals biomarker heterogeneity linked to CDK4/6 Inhibitor resistance in breast cancer cell lines
              </a>
            </td>
          <td>
            I. Migliaccio, M. Bonechi, Dario Romagnoli, Giulia Boccalini, F. Galardi, C. Guarducci, A. Nardone, R. Schiff, L. Biganzoli, L. Malorni, M. Benelli
          </td>
          <td>2025-07-31</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Background In-solution hybridization enrichment of genetic markers is a method of choice in paleogenomic studies, where the DNA of interest is generally heavily fragmented and contaminated with environmental DNA, and where the retrieval of genetic data comparable between individuals is challenging. Here, we benchmark the commercial “Twist Ancient DNA” reagent from Twist Biosciences using sequencing libraries from ancient human samples of diverse demographic origin with low to high endogenous DNA content (0.1–44%). For each library, we tested one and two rounds of enrichment and assessed performance compared to deep shotgun sequencing. Results We find that the “Twist Ancient DNA” assay provides robust enrichment of approximately 1.2M target SNPs without introducing allelic bias that may interfere with downstream population genetics analyses. Additionally, we show that pooling up to 4 sequencing libraries and performing two rounds of enrichment is both reliable and cost-effective for libraries with less than 27% endogenous DNA content. Above 38% endogenous content, a maximum of one round of enrichment is recommended for cost-effectiveness and to preserve library complexity. Conclusions In conclusion, we provide researchers in the field of human paleogenomics with a comprehensive understanding of the strengths and limitations of different sequencing and enrichment strategies, and our results offer practical guidance for optimizing experimental protocols. Supplementary Information The online version contains supplementary material available at 10.1186/s13059-025-03622-6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c477c495125e3cd66f843708e054b0141348cbd8" target='_blank'>
              Optimized in-solution enrichment of over a million ancient human SNPs
              </a>
            </td>
          <td>
            Roberta Davidson, Xavier Roca-Rada, Shyamsundar Ravishankar, Leonard Taufik, Christian Haarkötter, Evelyn Collen, Matthew P. Williams, Peter Webb, M. I. Mahmud, Erlin Novita, Idje Djami, Gludhug A. Purnomo, Cristina Santos, A. Malgosa, Linda R. Manzanilla, Ana Maria Silva, Sofia Tereso, Vítor Matos, Pedro C. Carvalho, Teresa Fernandes, A. Maurer, João C. Teixeira, Ray Tobler, Lars Fehren-Schmitz, B. Llamas
          </td>
          <td>2025-07-03</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The cellular networks that maintain genome stability encompass numerous pathways involved in all aspects of nucleic acid metabolism. Through bioinformatic analysis, we identified the Zinc Finger CCCH-Type Containing 4 protein (ZC3H4), a suppressor of noncoding RNA (ncRNA) production, as a pivotal player in this system. Experimentally, ZC3H4 deficiency led to increased DNA damage, abnormal mitosis, and cellular senescence. Biochemical analysis and super-resolution microscopy revealed that the loss of ZC3H4 increased replication stress (RS)—a major driver of genome instability—by inducing a hypertranscription state that promoted R loop formation and transcription-replication conflicts (TRCs), both of which drive RS. Further bioinformatic analysis demonstrated that ZC3H4 preferentially binds to genomic regions prone to TRCs and R loops, where it suppresses ncRNA bursts, functioning as part of the Restrictor complex. Our findings identify ZC3H4 as a crucial factor in maintaining genome integrity, strategically positioned at the critical intersection of DNA and RNA synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2eceaa09e7e61059a9c40891a86a83f0e1a6b53" target='_blank'>
              ZC3H4 safeguards genome integrity by preventing transcription-replication conflicts at noncoding RNA loci
              </a>
            </td>
          <td>
            Yann Frey, Liana J Goehring, M. Haj, G. Róna, Carel Fijen, Michele Pagano, Tony T Huang, Eli Rothenberg, Y. Ziv, Yosef Shiloh
          </td>
          <td>2025-06-18</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Ewing sarcoma is a rare type of cancer arising from bone and soft tissues mainly affecting children and young adults. Treatments include intensive chemotherapy, surgery and radiotherapy, however more than 30% of patients die from the disease. Direct drug targeting of EWS-FLI1 remains a significant challenge, therefore new approaches are urgently required.Analysis of PRMT1 and PRMT5 transcript expression using the R2 platform focusing on the Filion dataset of sarcomas that includes Ewing sarcoma patients alongside other fusion-positive sarcomas and breast and lung cancer datasets. Immunoblotting across a panel of Ewing sarcoma cell lines detected PRMT1 and PRMT5 expression and associated activity. Cell viability assay after PRMT inhibition, with and without olaparib, were conducted by trypan blue exclusion and MTT assay. DNA damage was detected by immunofluorescence staining for markers of DNA damage (γH2AX) and double-strand breaks (53BP1).We show that the expression and activity of the arginine methyltransferases PRMT1 and PRMT5 are elevated in Ewing sarcoma and that inhibition of PRMT1 or PRMT5 with pre-clinical inhibitors leads to growth arrest and apoptosis that is dependent on the expression of the driver oncogene EWSR1::FLI1. Mechanically, we show that PRMT1 and PRMT5 inhibitors promote DNA damage, and that PRMT5 inhibitors synergise with the PARP inhibitor olaparib to induce elevated DNA damage and reduced cell viability.Our study implies that PRMT1/PRMT5 are important mediators of EWSR1::FLI1 oncogenicity and that drug targeting PRMT1/PRMT5 in combination with DNA damaging chemotherapies could be an effective therapeutic strategy for the treatment of ES patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd386d567ff3eb3e36e6bd679d51d53f791750a7" target='_blank'>
              Arginine methylation regulates Ewing sarcoma cell viability in a EWSR1::FLI1 dependent manner and provides a therapeutic opportunity
              </a>
            </td>
          <td>
            Ciara Ward, Charles Brockwell, Gavin S. McNee, Emily Orton, Emily N. P. Prowse, S. Gatz, Clare C Davies
          </td>
          <td>2025-08-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Heritable gene activity variations that do not alter the underlying DNA sequence are known as epigenetic modifications. Mutations that disrupt genome structure and function are key drivers of oncogenesis. In addition to genetic mutations that cause direct disruptions in the DNA sequence, epigenetic changes can affect gene expression, which helps the development of cancerous traits. Changes in DNA methylation and histone mark patterns are the main drivers of these epigenetic modifications, as they encourage the growth and spread of tumors. In this review, we explore the substantial implications of epigenetic control on tumor genesis, metastatic behaviour, metabolic pathways, and the tumor microenvironment, delving into the intricacies of this intricate regulation. We pay special attention to the dysregulation at every stage of epigenetic modulation, which includes, but is not limited to, abnormalities in the enzymes that modify histones and methylate DNA, subunit loss or fusions in chromatin remodeling complexes, and disruptions in higher-order chromatin structure. We also highlight the development of inhibitors of medications targeted at epigenetic enzymes and summarize the abnormal roles of enzymes in DNA methylation, histone acetylation, and histone methylation during tumour progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97d006bb544ee4859c5808a78f45579e1cfd4a19" target='_blank'>
              Epigenetic regulators in cancer therapy and progression
              </a>
            </td>
          <td>
            Hany E. Marei
          </td>
          <td>2025-06-28</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction V(D)J recombination, initiated by recombination-activating gene (RAG) endonucleases, is a crucial process for the generation of diversified antigen receptors of T and B lymphocytes but regarded dispensable for innate natural killer (NK) lymphocytes lacking clonotypic receptors. Methods To explore the impact of potential rearrangements on NK cell maturation, RAG-fate mapping reporter human induced pluripotent stem cell (iPSC) lines were generated by introduction of RSS-invEGFP constructs into the AAVS1 locus using CRISPR/Cas9 and differentiated into NK cells in vitro. Results GFP expression was observed in up to 14% of mature NK cells characterized by a CD45dim CD56dimCD57+NKG2C+/−KIR+/− phenotype and unproductive genetic rearrangements in the IGH locus. Advanced maturation was further revealed by transcriptomic studies using RNA sequencing. Despite their strong effector function, DNA damage response and survival to ionizing radiation were compromised. Discussion These findings suggest a role of RAG expression in NK cell ontogeny supporting the development of a terminally differentiated effector population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6697344fa4f70950fad5843a2ad7ef88eb6f129" target='_blank'>
              RAG recombinase expression discriminates the development of natural killer cells
              </a>
            </td>
          <td>
            Jasmin Sprissler, U. Pannicke, E. Rump, H. Schrezenmeier, N. Casadei, M. Pogoda, Laurence Kuhlburger, M. B. Oquendo, Stefan Czemmel, K. Debatin, Miriam Erlacher, Klaus Schwarz, Kerstin Felgentreff
          </td>
          <td>2025-07-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa7409bece8a61b55b1fe618c705451021becc29" target='_blank'>
              Tissue-Specific Activation of Microhomology-Mediated End Joining with Age Reflects Dynamic Rewiring of DNA Repair in Rats
              </a>
            </td>
          <td>
            Diksha Rathore
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="ABSTRACT Maintaining genome integrity is essential for the proper functioning and development of organisms. An intriguing aspect is that neocentromeres can form at non-centromeric sites. CENP-A, a key epigenetic marker of centromeres, is often mislocalized to ectopic sites in cancers when overexpressed. Its deposition on centromeres relies on transcription of centromeric non-coding RNAs. Subsequently, ectopic CENP-A is frequently found at transcriptionally active and chromosome breakpoint regions. We previously engineered a stable ectopic CENP-A site on a naïve chromosome by overexpressing PCAT2, a non-centromeric oncogenic lncRNA that recruits CENP-A to its transcribing locus. We tracked cells with this transgene to analyze the longevity of ectopic CENP-A. We discovered that this induced epigenetic memory was lost due to suppression by epigenetic silencing mechanisms, restoring CENP-A to previous levels. These findings suggest that cells have mechanisms to prevent neocentromere formation at ectopic sites by suppressing transcription unless selective pressure favors it">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aec7b7b4ef76af3d3bde1c36b2a65ab677cc6c45" target='_blank'>
              Oncogenic lncRNA transgene transcription modulates epigenetic memory at a naïve chromosomal locus
              </a>
            </td>
          <td>
            Sweta Sikder, Songjoon Baek, Yamini Dalal, Ganesan Arunkumar
          </td>
          <td>2025-07-31</td>
          <td>Nucleus</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Amplified centrosome number causes genomic instability, most severely through division into more than two aneuploid daughter cells (multipolar mitosis). Several mechanisms that suppress multipolar division have been uncovered, yet mechanisms that favor viable multipolar division are poorly understood. To uncover factors that promote viability in cells with frequent centrosome amplification and multipolar division, we conducted an unbiased Drosophila genetic screen. In 642 mutagenized lines, we exploited the ability of intestinal papillar cells to form and function despite multipolar divisions. Our top hit is an unnamed gene, CG3168. We name this gene synaptic vesicle glycoprotein 2, reflecting homology to human Synaptic Vesicle Glycoprotein 2 (SV2) proteins. GFP-tagged SV2 localizes to the plasma membrane. In cells with amplified centrosomes, SV2 positions membrane-adjacent centrosomes, which prevents severe errors in chromosome alignment and segregation. Our results uncover membrane-based multipolar division regulation and reveal a novel vulnerability in cells with common cancer properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/604d54b8cce75407680ad35bfbf26bb3a7a65d49" target='_blank'>
              Synaptic vesicle glycoprotein 2 enables viable aneuploidy following centrosome amplification.
              </a>
            </td>
          <td>
            Jane E Blackmer, Erin A. Jezuit, Archan Chakraborty, Satya N Yalamanchi, Ruth A. Montague, Erin E. Dickert, Nora G Peterson, William Outlaw, Donald T. Fox
          </td>
          <td>2025-07-31</td>
          <td>Genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Transposable elements (TEs) drive major genome size and structural variations, yet evolutionary forces affecting their accumulation and removal remain unclear. Classical models predict that higher recombination rates lead to more efficient purifying selection, such as TE removal. However, in the painted lady butterfly (Vanessa cardui), smaller chromosomes harbor denser TE content than larger ones despite higher recombination rates. This unexpected pattern raises questions about whether similar trends occur across other Lepidoptera species and what evolutionary forces are behind this pattern. Across ten species spanning ten lepidopteran families, we investigated the relationship between chromosome size and TE organization using comparative genomics. We observed that smaller chromosomes consistently have higher TE densities in all the investigated species. Chromosome size had positive correlations with average inter-TE distance for both young (<5% divergence) and old TEs (5% to 10% divergence). However, the ratio of these distances (young/old TEs) was negatively correlated with chromosome size in eight of ten species, with two showing no statistically significant correlation, suggesting that smaller chromosomes have higher removal rates of sequence between TEs, potentially due to nonallelic homologous recombination, causing the loss of unique sequences between nonallelic homologs. Population genomics analyses showed inconsistent correlations between chromosome size and genetic diversity or selection coefficients between Danaus plexippus and Spodoptera frugiperda, ruling out the efficiency of purifying selection or selective constraint as the main driver. Taken together, we demonstrate that Lepidoptera has a unique genomic feature of denser TEs in smaller chromosomes, with nonallelic homologous recombination as a potential driving force.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/483c8b5fa0abe1f5104617a2081bb976e2f8c58d" target='_blank'>
              Lepidopteran Genomes Have Denser Transposable Elements in Smaller Chromosomes, Likely Driven by Non-allelic Homologous Recombination
              </a>
            </td>
          <td>
            Hyerin An, Kiwoong Nam
          </td>
          <td>2025-07-12</td>
          <td>Genome Biology and Evolution</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Amplification of human chromosome 20q has been reported as the most frequently recurring genetic abnormality associated with large scale changes in mRNA and protein levels in sporadic colorectal carcinomas. While some studies have found 20q amplification to be consistent between primary and metastatic samples from the same patient with a role in the development of metastasis and worse patient prognosis, others have reported association with improved overall survival for a subset of these patients with colorectal cancer (CRC). To fine map the Minimal Common Regions (MCRs) of amplification on chromosome 20q and identify the candidate genes playing roles in progression of the disease, microarray comparative genomic hybridization analyses of two in vitro cultured CRC liver metastasis cell line model systems was utilized. Microarray expression analysis led to the identification of a candidate gene signature comprising of four genes, BMP7, DNMT3B, UBE2C and YWHAB, residing in the MCRs that were over expressed in CRC cells. By validating our results in a training set of 23 adenocarcinomas (tumors) and five adenomas (polyps) using reverse transcription-quantitative PCR, as well as analyses of two larger colorectal cancer test data sets derived from 195 The Cancer Genome Atlas and 182 MD Anderson Cancer Center patients with colorectal adenocarcinoma patients, this gene signature was ascertained to be associated with lymph node spread and/or distant metastasis (P<0.05). Previously reported functional studies of the gene signature indicated their involvement in inflammatory and immune response pathways driving CRC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19ea2107086249d394733ab4fbf6c82be7552a16" target='_blank'>
              Chromosome 20q gene signature associated with colorectal cancer progression
              </a>
            </td>
          <td>
            Jennifer Carter Jones, A. Hegde, Yu-Jing Huang, G. Manyam, Vibhuti Srivastava, J. Song, Yulan Cheng, Ralf Krahe, Warapen Treekitkarnmongkol, Stephen J. Meltzer, S. Kopetz, Stanley R Hamilton, Hiroshi Katayama, Subrata Sen
          </td>
          <td>2025-07-18</td>
          <td>Oncology Reports</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Tumor evolution is shaped by selective pressures imposed by physiological factors as the tumor naturally progresses to colonize local and distant tissues, as well as by therapy. However, the distinction between these two types of pressures and their impact on tumor evolution remain elusive, mainly, due to extensive intra-tumor heterogeneity. To disentangle the effects of these selective pressures, we analyze data from diverse cohorts of patients, of both treated and untreated cancers. We find that, despite the wide variation across patients, the selection strength on tumor genomes in individual patients is stable and largely unaffected by tumor progression in the primary settings, with some cancer-specific signatures detectable in the progression to metastases. However, we identify a nearly universal shift toward neutral evolution in tumors that resist treatment and demonstrate that this regime is associated with worse prognosis. We validate these findings on both published and original datasets. We suggest that monitoring the selection regime during cancer treatment can assist clinical decision-making in many cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a95ee20056cad563b0d086b8e70065c4daacf4f" target='_blank'>
              Genome-level selection in tumors as a universal marker of resistance to therapy
              </a>
            </td>
          <td>
            Erez Persi, P. Sudalagunta, Yuri I. Wolf, Rafael R. Canevarolo, Mehdi Damaghi, K. Shain, A. Silva, Eugene V. Koonin
          </td>
          <td>2025-07-16</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="DNA sequencing technologies are widely used to study tumor evolution within a cancer patient. However, analyses require various computational methods, including those to infer clone sequences (genotypes of cancer cell populations), clone frequencies within each tumor sample, clone phylogeny, mutational tree, dynamics of mutational signatures, and metastatic cell migration events. Therefore, we developed GenoPath, a streamlined pipeline of existing tools to perform tumor evolution analysis. We also developed and added tools to visualize results to assist interpretation and derive biological insights. We have illustrated GenoPath’s utility through a case study of tumor evolution using metastatic prostate cancer data. By reducing computational barriers, GenoPath broadens access to tumor evolution analysis. The software is available at https://github.com/SayakaMiura/GP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e039aed15ffae5155d36f38b8ec85975e0960dc5" target='_blank'>
              GenoPath: a pipeline to infer tumor clone composition, mutational history, and metastatic cell migration events from tumor DNA sequencing data
              </a>
            </td>
          <td>
            Ryan M. Tobin, Shikha Singh, Sudhir Kumar, Sayaka Miura
          </td>
          <td>2025-07-02</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Metastable epialleles (MEs) are genomic loci at which epigenetic marks are established stochastically during early embryonic development and maintained during subsequent differentiation and throughout life, leading to stable epigenetic and phenotypic variation among genetically identical individuals. Although MEs were first described in mice over 20 years ago, the extent of epigenetic metastability in the mouse genome remains unknown. We present the first unbiased genome-wide screen for MEs in mice. Using deep whole-genome bisulfite sequencing across tissues derived from the three embryonic germ layers in isogenic C57BL/6J mice, we identified only 29 MEs, precisely localizing them and documenting their rarity. Consistent with recent findings, we found no effects of maternal dietary methyl donor supplementation on ME methylation in the offspring, challenging previous assertions that MEs generally exhibit developmental plasticity. Most but not all MEs are associated with intracisternal A-particle (IAP) elements, tending to localize to the 5′ end of the IAP. Additionally, we discovered autosomal regions at which systemic interindividual variation in DNA methylation is associated with sex, providing insights into sex-associated epigenetic development that apparently precedes sexual differentiation. Our findings indicate that expression of transcription factors, including CCCTC-binding factor (CTCF) and specific KRAB zinc finger proteins during early embryonic development, plays a key role in orchestrating stochastic establishment and/or maintenance of DNA methylation at metastable transposable elements. Overall, these findings advance our understanding of the genomic determinants of epigenetic metastability and suggest that interindividual epigenetic variation at MEs is unlikely to be a major determinant of phenotypic variation among isogenic mice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/708a43f6a2e4e91d381b16e3a28eb5d04dfb8092" target='_blank'>
              Mouse metastable epialleles are extremely rare
              </a>
            </td>
          <td>
            Chathura J Gunasekara, U. Maduranga, Taylor Zhang, Jonathan N Wells, Maria S. Baker, Eleonora Laritsky, Yumei Li, C. Coarfa, Yi Zhu, R. Waterland
          </td>
          <td>2025-07-19</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Maintaining genomic stability is essential for reducing the risk of carcinogenesis. Homologous recombination (HR) is a high-fidelity DNA repair mechanism that addresses double-strand breaks and interstrand crosslinks. The present review examined two key components of HR: RAD51, the eukaryotic recombinase and PALB2, a scaffolding protein. Their structural and functional roles are explored in the context of breast and ovarian cancer. RAD51 facilitates homology search and strand invasion, while PALB2 links BRCA1 and BRCA2, stabilizing RAD51 filaments. Mutations in these genes compromise HR, increasing susceptibility to various cancers and impacting treatment efficacy by impairing DNA repair. The present review discussed the clinical implications of RAD51 and PALB2 mutations, focusing on risk stratification, PARP inhibitor efficacy and emerging therapies. Additionally, it highlighted the potential of RAD51 and PALB2 as biomarkers and therapeutic targets, contributing to advances in personalized cancer management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4cbc5968373fbc15750cedb90b63387fd5376bb" target='_blank'>
              RAD51 and PALB2 in precision oncology: Clinical implications for HRD associated breast and ovarian cancers (Review)
              </a>
            </td>
          <td>
            M. A. Kausar, K. Alshammari, Fahaad S Alenazi, Sadaf Anwar, Amany Mohammed Khalifa, Tarig A. N. Ginawi, Abdulaziz Asiri, M. Najm, S. Rabbani, Mohamed El-Tanani, Saumyatika Gantayat
          </td>
          <td>2025-07-03</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Objective. This study describes mutations of genes that stimulate and regulate cell growth, programmed cell death, DNA repair, and cell growth suppression in a boy with osteosarcoma. Case Report. We report a case of bone sarcoma in a 9-year-old boy with possible familial predisposition. In our patient, only a subset of tumor cells expressed the ATRX protein, which is known to control the expression of several genome regions. The function of the p53 protein, which acts as a transcription factor that regulates the DNA damage repair response, cell cycle progression, and apoptosis pathways, is lost in 40-50% of malignant cells. Retinoblastoma was positive in the predominant subset of tumor cells. Deletion is found on chromosome 9, cytoband 9p21.3, where the genes for CDKN2A and CDKN2B are located. Neoplastic cells were SATB2-positive in a substantial subset, with nuclear staining. The SATB2 protein is a DNA-binding protein involved in transcriptional regulation and chromatin remodeling. Chromosomal losses of 8p and 19q11-q13.43 were also found. These regions contain several tumor suppressor genes, including NKX3.1, whose reduced expression correlates with 8p loss in high-grade tumors. Although there was no known cancer syndrome in the family, the maternal grandfather had a similar tumor requiring amputation. Conclusion. Chromosomal instability is a hallmark of osteosarcoma and is characterized by heterogeneous and extensive genetic complexity. Various numerical and structural genomic rearrangements have been described in cancer cells. However, there is little consistent genetic change to understand the etiopathogenesis of this aggressive tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec206c4dcb70e35f0aed0eba180bbda6edcf22d" target='_blank'>
              Genetic Analysis of Osteosarcoma Cells in a 9-year-old Boy: Genes Involved in Cell Cycle Control
              </a>
            </td>
          <td>
            Dragan Jovanovic, Alwajih Tariq, Sara Dlugos
          </td>
          <td>2025-04-01</td>
          <td>Acta Medica Academica</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="INTRODUCTION
Long noncoding RNAs are essential regulators in numerous biological processes and have been linked to various diseases, including cancer. Despite their initial classification as transcriptional byproducts, lncRNAs have been shown to modulate chromatin structure, transcription, RNA processing, protein translation, and intranuclear transport. LINC-PINT, a lncRNA induced by P53, is particularly noteworthy for its role in tumor suppression across multiple cancers.


METHODS
By utilizing the PubMed database and applying inclusion criteria based on relevance, literature quality, and data availability, we conducted a comprehensive analysis of 128 studies to provide an overview of the functions of LINC-PINT and its mechanisms of action in cancers.


RESULTS
LINC-PINT was confirmed to function as a tumor suppressor factor in many cancers, such as triple-negative breast cancer, non-small cell lung cancer, gastric cancer, glioma, melanoma, osteosarcoma, laryngeal squamous cell carcinoma, esophageal cancer, colorectal cancer, nasopharyngeal carcinoma, retinoblastoma, ovarian cancer, thyroid cancer, hepatocellular carcinoma, and pancreatic cancer by promoting apoptosis and senescence, inhibiting proliferation, migration, invasion, drug resistance, cell stemness, EMT, radioresistance, and DNA damage repair.


DISCUSSION
LINC-PINT serves as a tumor suppressor with its ability to sponge miRNAs, regulate epigenetic modulation, DNA damage repair, etc. Despite the promising findings, the complex and tissue-specific functions of LINC-PINT, along with the need for further clinical validation, underscore the importance of continued research to fully understand its mechanisms and potential as a therapeutic target.


CONCLUSION
LINC-PINT is a potential target in cancer progression and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c614cdfeae321548caa4f0f7d2f981f35ae5a41" target='_blank'>
              LINC-PINT: A Distinctive Long Non-Coding RNA Functioning as a Potential Suppressor in Tumorigenesis.
              </a>
            </td>
          <td>
            Jiayi Li, Yining Pan, Songqiang Li, Cheng Chen, Chengfu Yuan
          </td>
          <td>2025-07-22</td>
          <td>Current topics in medicinal chemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) represent one of the most severe forms of genomic damage. Although substantial progress has been made in elucidating general patterns associated with DSBs, the influence of 3D chromatin structure on DSB formation remains underexplored, particularly concerning its spatial configuration. Here, the ChromInSight framework is introduced. Using standardized datasets,Hi-DSB is developed and deployed in ChromInSight, a genome-wide DSB prediction model based on graph contrastive learning (GCL), and applied advanced interpretability techniques to identify DSB-associated genomic patterns. The findings reveal that the spatial cluster-scene between hub nodes and DSB sites is predominantly shaped by the 3D conformation of chromatin, rather than by linear genomic distance. This phenomenon is validated at both the Loop and topologically associating domain (TAD) levels and proposed a "spatial isolation - damage containment" hypothesis, which illustrates the genome strategy for managing damage. These findings support the role of 3D genome architecture in genomic instability. Consequently, the framework provides a powerful tool for investigating the intricate relationship between chromatin structure and genomic stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31e5b0b135f6e23dfe9751ad20057b548c477ad6" target='_blank'>
              ChromInSight: Revealing DNA Double-Strand Breaks Through Chromatin Structural Insights With an Interpretable Graph Neural Network Framework.
              </a>
            </td>
          <td>
            Kang Xu, Zongyuan Yu, Canzhuang Sun, Conglin Gou, Jiangyue Zhu, Jun Wang, Xiaochen Bo, Guoxian Yu, Hao Li, Hebing Chen
          </td>
          <td>2025-06-30</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="This review examines the increasingly prominent role of mechanics within cancer formation and progression. The extremely varied and contradictory genetic landscape of cancer is in stark contrast to the seemingly universal mechanical characteristics of cancer cells and their tumour microenvironment, and mechanics may be a principal unifying trait of this disease. The tight regulation of innate cell mechanical properties raises the possibility that destabilisation of the cell drives tumour formation in an attempt to restore cell mechanical homeostasis. With losses in cell stiffness more pronounced at the cell nucleus, we hypothesise that destabilisation occurs within the nucleus, likely within the nucleosome. Beyond the mechanical properties of the cell, this compromise to the chromatin structure holds significant repercussions for both genetic and epigenetic regulation, providing scope for significant genetic dysregulation and mutation. However, the nature of such genetic events will be dependent upon the region of mechanical destabilisation; thus, introducing greater variability and heterogeneity to genetic changes. We conclude with the hypothesis that cancer has a mechanical genesis, in which cell nuclear destabilisation functions as the enabling hallmark of cancer. It is theorised that both genetic and structural dysfunction stem from this nuclear destabilisation, driving disease pathology and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/440a5c35e068d9fe1560a5f26a4055f7a6f2a62d" target='_blank'>
              Nuclear destabilisation - a possible genesis of cancer?
              </a>
            </td>
          <td>
            Daniel D Scott, Francesco Bettariga, Marco Ventin, Chris Bishop, Britta Stordal
          </td>
          <td>2025-07-14</td>
          <td>Biological reviews of the Cambridge Philosophical Society</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="PLXNA3 is a transmembrane protein essential for axon guidance and synapse formation, gaining attention in cancer research for its roles in cell migration, invasion, and signal transduction. However, its biological significance in breast cancer remains underexplored. Here, bulk RNA sequencing and survival analysis revealed that elevated PLXNA3 levels were correlated with increased malignancy and poorer survival in breast cancer. Single-cell RNA sequencing demonstrated that PLXNA3 was predominantly enriched in tumor cell clusters within the tumor immune microenvironment, rather than in immune cell clusters. In vitro experiments further demonstrated that knockdown and overexpression of PLXNA3 significantly altered the proliferative, invasive, and migratory capacities of breast cancer cells. Additionally, the overexpression of PLXNA3 was negatively correlated with immune activation status and served as a predictor of poor response to anti-PD1 immunotherapy. Our findings suggested that high expression of PLXNA3 is associated with poor prognosis in breast cancer and plays a crucial role in cancer immunity, making it a promising novel target for intervention in breast cancer. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-13843-1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f126a6deb0547875700a586994249ce5224fe17" target='_blank'>
              Comprehensive analysis of the PLXNA3 gene on prognosis and immune characteristics in breast cancer
              </a>
            </td>
          <td>
            Wen Sun, Yifan Xu, Qi Qi, Weiya Zhang, H. Pan, Guangshun Sun, Shui Wang, Wenbin Zhou
          </td>
          <td>2025-08-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="


 The substantial improvement in childhood cancer survival rates has been primarily achieved through therapy intensification, often at the cost of life-long adverse effects. Therapies can be associated with characteristic patterns of mutations found in the tumor genome, known as mutational signatures. These signatures, therefore, can act as markers of resistance and late effects of therapy. We employed a large-scale analysis of 611 whole-genome sequenced pediatric tumors with harmonized therapy data, integrating de novo mutational signature extraction with ensemble machine learning (ML) models trained on comprehensive genomic features (single- and double-base substitutions, indels, structural variants, and copy number alterations) to identify therapy-associated patterns. De novo extraction across single-nucleotide variants, indels, structural variants, and copy number alterations revealed 95 distinct mutational signatures, including 31 novel ones. Platinum therapy was associated with the highest mutagenic burden, with canonical signatures (e.g., SBS31, SBS35, DBS5) detected in 41% of exposed tumors. By aligning with treatment timelines, we defined a median latency of 91 days and a burden threshold of ∼1500 mutations for platinum signatures to emerge—appearing in all osteosarcomas (most common primary bone cancer) and two-thirds of neuroblastomas (most common extracranial solid tumor in children). Despite these insights, the reasons why some tumors do not exhibit these signatures, and how this relates to their response to therapy, remain poorly understood. To uncover subtler therapy-associated signals beyond known COSMIC signatures, we trained ensemble ML models on various types of genomic alterations. Our ML models achieved high predictive performance (F1-score = 0.89) in classifying platinum-exposed tumors. Importantly, predictive accuracy persisted even after masking features corresponding to canonical platinum signatures, indicating the presence of non-canonical mutational patterns. Feature analysis identified biologically plausible non-canonical genomic features enriched in platinum-exposed tumors across pediatric cancers, suggesting previously unannotated mutational footprints of therapy. Notably, ML analysis revealed distinct non-canonical genomic features associated with anthracyclines, and select antimetabolites and alkylating agents, shedding light on therapies previously lacking characterized mutational signatures. These novel genomic signals were also validated across independent pediatric and adult cancer cohorts. This work highlights the power of ML to detect therapy-induced mutational patterns beyond known mutational signatures. As cancer remains the leading cause of disease-related death among children in many countries, this integrated ML-driven approach offers a powerful strategy for improved therapy monitoring, early resistance prediction, and ultimately, the development of more personalized and effective treatment strategies (e.g., stratification and de-escalation) for children with cancer.



 Mehdi Layeghifard, Marcos Díaz-Gay, Pedro L. Ballester, Elli Papaemmanuil, Mark Cowley, Anita Villani, Ludmil B. Alexandrov, Adam Shlien. Unmasking hidden mutational footprints of therapy in pediatric tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr A038.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6789b44a7ab2f8906895c50474f02a3bfc36f14" target='_blank'>
              Abstract A038: Unmasking hidden mutational footprints of therapy in pediatric tumors
              </a>
            </td>
          <td>
            M. Layeghifard, M. Díaz-Gay, Pedro L. Ballester, E. Papaemmanuil, Mark J. Cowley, A. Villani, Ludmil B. Alexandrov, A. Shlien
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="Evolution and outgrowth of drug-resistant cancer cells is a common cause of treatment failure. Patients with leukemic cutaneous T-cell lymphoma (L-CTCL) have a poor prognosis due to development of drug-resistance and severe bacterial infections. Here, we show that most L-CTCL patients harbor multiple genetically distinct subclones that express an identical clonal antigen receptor but display distinct phenotypes and functional properties. These co-existing malignant subclones exhibit differences in tissue homing, metabolism, and cytokine expression, and respond differently to extrinsic factors like Staphylococcus (S.)aureus and cancer drugs. Indeed, while S. aureus toxins selectively enhance activation and proliferation of certain subclones, these responsive subclones are also the most intrinsically sensitive to cancer drugs when the stimuli are removed. Consequently, although divergent evolution of malignant subclones drives aggressiveness, adaptability and drug-resistance, by removing extrinsic stimuli and mapping malignant subclones, we can expose inherent vulnerabilities that can be exploited in the treatment of these cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9c75ce389008c94fe5703552a6c93e0ffef4c7" target='_blank'>
              Divergent Evolution of Malignant Subclones Maintains a Balance Between Induced Aggressiveness and Intrinsic Drug Resistance in T Cell Cancer.
              </a>
            </td>
          <td>
            T. B. Buus, C. K. Vadivel, Maria Gluud, Martin R J Namini, Ziao Zeng, S. Hedebo, Menghong Yin, A. Willerslev-Olsen, Emil M H Pallesen, Lang Yan, E. Blümel, E. U. Ewing, S. Ahmad, Lara P Sorrosal, Carsten Geisler, C. Bonefeld, A. Woetmann, M. Andersen, T. Mustelin, Claus Johansen, Marion Wobser, M. Kamstrup, E. Guenova, Jürgen C. Becker, S. Koralov, R. Bech, N. Ødum
          </td>
          <td>2025-06-16</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/813fcceb7d9e7dd4a1d1ee8db0535b3e857d7b20" target='_blank'>
              Unraveling the Potential of ctDNA in Precision Medicine for Breast Cancer.
              </a>
            </td>
          <td>
            Juscelino Carvalho de Azevedo Junior, Fernanda Jardim da Silva, Anna Carolina Lima Rodrigues, Stefanie Braga Maia de Sousa, Jéssica Manoelli Costa da Silva, Iago Barroso Ramos, Bárbara do Nascimento Borges, Vanessa Morais Freitas, D. Calcagno
          </td>
          <td>2025-08-07</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Effective identification of oncogenic mutations is essential for diagnosis, forecasting resistance, and metastasis in remission. It is required for an optimal drug regimen. We develop a framework to discover mutations that co-exist in different oncoproteins, and those that are excluded, likely encoding oncogene-induced senescence. For the first, mapping the proteins onto pathways assist combinatorial drug selections and help detect metastases. The second provides the molecular basis for synthetic lethality, to date investigated at the genome level. Our pan-cancer profiles of ~60,000 tumor sequences, detect 3424 co-existing tumor-specific mutations. Mapping them onto pathways indicates that they preferentially promote specific primary tumors. We uncover metastatic mutations and provide metastatic breast-cancer markers. This work not only clarifies the mechanistic basis of intratumor mutational diversity but usefully reveals markers for metastasis in patients' genomes and introduces a novel computational framework for detecting metastasis based on tumor mutational profiles. Mapping the mutations onto pathways provides an invaluable metastasis-targeting resource, guiding drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd8dbee9272fd01bd4b589963df2977d50caa0e8" target='_blank'>
              Mutations in tumor signaling, metastases, and synthetic lethality establish distinct patterns.
              </a>
            </td>
          <td>
            B. R. Yavuz, Uğur Şahin, Hyunbum Jang, Ruth Nussinov, Nurcan Tunçbağ
          </td>
          <td>2025-08-04</td>
          <td>PLoS computational biology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 A crucial bulwark of genome stability is the DNA repair system. The homologous recombination (HR) pathway is essential for the repair of the most harmful DNA lesions, like DNA double-strand breaks. Deficiencies in this pathway result in genomic instability and contribute to tumorigenesis. BRCA1 and BRCA2 (Breast Cancer type 1 and 2 susceptibility proteins) play critical roles in HR. Germline mutations in the BRCA1 and BRCA2 genes are responsible for nearly 50% of all familial breast and ovarian cancers. BRCA1/2-deficiencies coerce tumor cells to depend on alternative DNA repair pathways. Recent studies have demonstrated that inactivation of RAD52 protein induces lethality (synthetic lethality) in BRCA1/2-deficient cancer cells, without affecting normal cells. By exploiting this synthetic lethality relationship, targeting RAD52 with inhibitors may provide an exceptionally selective method for eliminating cancer cells. Focusing on DNA repair proteins involved in synthetically lethal relationships in cancer cells has become an important strategy in the development of specific cancer therapeutics. Our laboratory has developed small molecule inhibitors targeting RAD52. We have successfully created small molecule inhibitors of RAD52 (RAD52i) and have shown their ability to inhibit the biochemical activities RAD52 including DNA annealing and DNA strand exchange. Using the GFP-based recombination assays, we show that small molecule inhibitors of RAD52 disrupt the homologous DNA repair (HDR) and single strand annealing (SSA) repair pathways in human cells. Our compounds exhibited a notable reduction in cell viability in BRCA deficient cells. Furthermore, our small molecule RAD52i exhibited considerable efficacy in a mouse xenograft model cells markedly suppressing growth of human BRCA1-mutated cancer cells. We aim to develop small molecule RAD52i as novel cancer therapeutics.
 Citation Format: Shiva Ostadrahimi, Matthew J. Rossi, Sarah F. DiDomenico, Saiful Amin, Alexander V. Mazin. TARGETING BRCA-DEFICIENT CANCER CELLS WITH SMALL MOLECULE RAD52 INHIBITORS [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P4-12-25.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745e4e002b1607f75e1773159229e8749f23eb29" target='_blank'>
              Abstract P4-12-25: TARGETING BRCA-DEFICIENT CANCER CELLS WITH SMALL MOLECULE RAD52 INHIBITORS
              </a>
            </td>
          <td>
            Shiva Ostadrahimi, Matthew J Rossi, Sarah F DiDomenico, Saiful Amin, Alexander Mazin
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Radiotherapy is a mainstay of treatment for head and neck squamous cell carcinoma (HNSCC), either definitive or adjuvant to surgery. Biological factors known to affect radiation response are hypoxia and DNA repair capacity, but several lines of evidence indicate that other genes and pathways in the tumor cells might be involved that have not been elucidated. Here, we report the results of a genome-wide CRISPR-Cas9 functional genomics screen in HNSCC cells to identify radiosensitizing genes. Remarkably, microtubule organizing genes were identified with CLASP1 as most unexpected radiosensitizing hit. Clonogenic assay confirmed the radiosensitizing effect of CLASP1 knockout. Functional analysis showed that CLASP1 knockout has major impact during S-phase, and resulted in mitotic cells with broken chromosomes and cell death. CLASP1 and possibly the microtubule machinery in broader sense seem involved in protection of HNSCC cells against radiation–induced DNA damage. This newly identified mechanism provides an outlook for novel treatment approaches in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/284fd55d39146101abef95bbef149eda9c32d030" target='_blank'>
              An unexpected role of CLASP1 in radiation response and S-phase regulation of head and neck cancer cells
              </a>
            </td>
          <td>
            R. D. de Roest, Marijke Buijze, Myrthe Veth, Klaas de Lint, G. Pai, M. Rooimans, Rob M. F. Wolthuis, A. Brink, J. Poell, R. Brakenhoff
          </td>
          <td>2025-08-06</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Centriole duplication is a vital process for cellular organisation and function, underpinning essential activities such as cell division, microtubule organisation and ciliogenesis. This review summarises the latest research on the mechanisms and regulatory pathways that control this process, focusing on important proteins such as polo-like kinase 4 (PLK4), SCL/TAL1 interrupting locus (STIL) and spindle assembly abnormal protein 6 (SAS-6). This study examines the complex steps involved in semi-conservative duplication, from initiation in the G1–S phase to the maturation of centrioles during the cell cycle. Additionally, we will explore the consequences of dysregulated centriole duplication. Dysregulation of this process can lead to centrosome amplification and subsequent chromosomal instability. These factors are implicated in several cancers and developmental disorders. By integrating recent study findings, this review emphasises the importance of centriole duplication in maintaining cellular homeostasis and its potential as a therapeutic target in disease contexts. The presented findings aim to provide a fundamental understanding that may inform future research directions and clinical interventions related to centriole biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/232fe9bbe42f0847ce312206baced030552a1c9f" target='_blank'>
              Centriole Duplication at the Crossroads of Cell Cycle Control and Oncogenesis
              </a>
            </td>
          <td>
            Claude Prigent
          </td>
          <td>2025-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b5bfa4068fc673194fc62b48dc4a50c589763e7" target='_blank'>
              Ataxin-2 preserves oocyte genome integrity by promoting timely premeiotic DNA replication
              </a>
            </td>
          <td>
            V. L. Monteiro, Crystal Yu, Camilla Roselli, B. Bakthavachalu, Mani Ramaswami, T. Hurd
          </td>
          <td>2025-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background NSUN6 is a regulator of tRNA methylation that partially exists in the Golgi and centrosomes. It catalyzes the methylation of tRNACys and tRNAThr at the C72 site, which affects tRNA generation. However, its expression, prognosis, and immune invasion in pan-cancer have not been studied. This extensive research tapped into the TCGA database, gathering 33 carefully paired sets of cancer and nearby normal tissues, covering a broad spectrum of 33 different cancer types. Moreover, this study explored how NSUN6 expression relates to the presence of immune and stromal cells in the tumor’s intricate immune environment. To further understand NSUN6’s role, a detailed Gene Set Enrichment Analysis (GSEA) was conducted, revealing its enrichment patterns in various carcinoma subtypes and hinting at its involvement in tumor growth. Analysis of NSUN6 expression patterns revealed a significant increase in eight cancer types, including lung adenocarcinoma and pancreatic cancer (P < 0.001). Additionally, a reduction was observed in six cancer types, such as cholangiocarcinoma and thyroid cancer (P < 0.001). Survival analysis indicated that individuals with lung adenocarcinoma, pancreatic cancer, and other cancers expressing high levels of NSUN6 had improved prognosis. The association between NSUN6 expression and patient outcomes was evident, including overall survival, disease-specific survival, disease-free interval, and progression-free interval. Furthermore, individuals with high levels of NSUN6 expression showed significantly longer survival times than those with low levels (P < 0.05). The involvement of NSUN6 in immune infiltration was evident, GSEA showed a significant correlation between NSUN6 expression and the five most significant enrichment pathways in different tumors. The NSUN6 gene functions as an initial indicator for diagnosis in renal clear cell carcinoma, pancreatic cancer, lung adenocarcinoma, and low-grade brain glioma. There is an association between its elevated expression and the predictive outcome, as well as the immune system’s infiltration, in 33 varieties of cancer. This study is limited by its reliance on in silico analyses without experimental validation. Additionally, the use of publicly available datasets may introduce variability due to differences in data sources and platforms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba0727a8de06b6d8752b41beb8058ddf29b34aed" target='_blank'>
              NSUN6 expression correlates with prognosis and immune infiltration in human cancers based on pan-cancer analysis
              </a>
            </td>
          <td>
            Jie Gao, Yao Xu, Chan Tian, Qiuhong Zhang, Ming Zhang, H. Shan, Jie Shi, Zongjuan Ming, Shuanying Yang, Xia Yang
          </td>
          <td>2025-07-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3cf9b1f7da5af7fa436901f6e280557bb0b8010" target='_blank'>
              An M13 phagemid toolbox for engineering tuneable DNA communication in bacterial consortia
              </a>
            </td>
          <td>
            Abhinav Pujar, Anchita Sharma, Hadi Jbara, Manish Kushwaha
          </td>
          <td>2025-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma accounts for 90% of primary liver malignancies and ranks as the fourth leading cause of global cancer mortality. DNA methylation drives HCC pathogenesis by orchestrating tumor suppressor silencing and oncogenic pathway activation, fundamentally governing tumor heterogeneity and therapeutic resistance. While alterations in white blood cell counts have emerged as independent prognostic indicators in HCC progression, their causal relationship with tumor epigenetics remains unclear. Given that HCC tumors actively secrete cytokines and remodel systemic immunity through epigenetic mechanisms, peripheral white blood cell dynamics serve as a compelling surrogate for studying tumor-driven immune dysregulation and identifying methylation-based therapeutic targets. Methods We integrated DNA methylation profiles from The Cancer Genome Atlas-Liver Hepatocellular Carcinoma (TCGA-LIHC) with summary statistics from six genome-wide association studies of white blood cell counts. Using these data, we applied a two-sample Mendelian Randomization (MR) framework to systematically investigate the causal effects of HCC-related CpG sites on white blood cell counts. The robustness of our findings was established through stringent criteria, comprehensive sensitivity analyses, and Bayesian colocalization, followed by functional validation with TCGA data and a transcriptome-wide association study (TWAS). Results Our analysis identified 26 HCC-specific CpG sites with causal effects on white blood cell counts. A multi-step validation pipeline integrating Bayesian colocalization and tumor-specific expression analysis robustly confirmed three core mediator genes: BTN3A2, TRIM27, and S100A12. TWAS provided independent, gene-level validation, revealing BTN3A2 as a powerful regulator of lymphocyte and neutrophil counts, TRIM27 as a key immunometabolic checkpoint, and S100A12 as a modulator of innate immunity. Conclusion This study establishes epigenome-immune crosstalk in HCC by identifying BTN3A2, S100A12 and TRIM27 as central methylation-immunoregulatory hubs. The discovered methylation-immune axes provide mechanistic insights for developing combination therapies targeting epigenetic checkpoints to reverse immunosuppressive microenvironments in advanced HCC. Supplementary Information The online version contains supplementary material available at 10.1186/s12885-025-14693-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/282ecf09d34731818d82e7690045faf501ada4b4" target='_blank'>
              Decoding the epigenetic-immune nexus in hepatocellular carcinoma: a Mendelian randomization study reveals BTN3A2, S100A12 and TRIM27 as white blood cell regulators
              </a>
            </td>
          <td>
            Yang Qiu, Haiyan Zhang, Xiao Yu, Jielu Pan, Dinghong Xiao, Hongyu Miao, Ruiqing Wang, Nan Shen, Peimin Pu, Yinling Zhou, Zhidong Liu, Lin He, Lianjun Xing
          </td>
          <td>2025-08-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The chromosomal passenger complex (CPC), comprising Survivin, Aurora B kinase, INCENP, and Borealin, is classically known for its essential functions during mitosis. However, recent findings expand the CPC's role beyond cell division, uncovering novel functions in replication stress response and genome stability maintenance. In our recent study by Falke et al. [1], we demonstrate that CPC components, particularly Survivin and Aurora B, contribute to the protection of stalled replication forks and facilitate translesion synthesis (TLS) - a specialized DNA damage tolerance mechanism. This regulation involves a functional interaction with proliferating cell nuclear antigen (PCNA), a central coordinator of DNA replication and repair. The work not only reshapes our understanding of CPC biology but also reveals a mechanistic link how tumor cells may exploit CPC activity to maintain replication under genotoxic stress. These findings open new avenues for targeting CPC-PCNA interactions in cancers characterized by elevated replication stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27440a8fa21c0d8a560b047855971e8e0f1ea441" target='_blank'>
              From mitosis to mutagenesis: Chromosomal passenger proteins at the crossroads of replication stress and cancer resilience
              </a>
            </td>
          <td>
            S. Knauer, Roland H Stauber
          </td>
          <td>2025-07-07</td>
          <td>Cell Signaling</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Systematic discovery of transcription factor (TF) landscapes in low-input samples and at single cell level is a major challenge in the fields of molecular biology, genetics, and epigenetics. Here, we present cleavage under Dynamic targets and Tagmentation (DynaTag), enabling robust mapping of TF-DNA interactions using a physiological salt solution during sample preparation. DynaTag uncovers occupancy alterations for 15 TFs in stem cell and cancer tissue models. We highlight changes in TF-DNA binding for NANOG, MYC, and OCT4, during stem-cell differentiation, at both bulk and single-cell resolutions. DynaTag surpasses CUT&RUN and ChIP-seq in signal-to-background ratio and resolution. Furthermore, using tumours of a small cell lung cancer model derived from a single female donor, DynaTag reveals increased chromatin occupancy of FOXA1, MYC, and the mutant p53 R248Q at enriched gene pathways (e.g. epithelial-mesenchymal transition), following chemotherapy treatment. Collectively, we believe that DynaTag represents a significant technological advancement, facilitating precise characterization of TF landscapes across diverse biological systems and complex models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0a8af886e5fec85aa2e58ed1b6a6ad04cf228fa" target='_blank'>
              DynaTag for efficient mapping of transcription factors in low-input samples and at single-cell resolution
              </a>
            </td>
          <td>
            Pascal Hunold, Giulia Pizzolato, Nadia Heramvand, L. Kaiser, Giulia Barbiera, Olivia van Ray, Roman K Thomas, Julie George, M. Peifer, Robert Hänsel-Hertsch
          </td>
          <td>2025-07-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Specialized DNA polymerases facilitate various cellular processes. Despite extensive research, the mutagenic effects of these error-prone enzymes on genomes are not fully understood. Here we show that Pol IV promotes genomic instability in Pseudomonas aeruginosa by misincorporating oxidized guanine nucleotides. This activity led to a distinctive mutational signature, characterized by A-to-C transversions occurring preferentially at AT sites flanked by a 5’G and/or 3’C. Furthermore, Pol IV preferentially targeted pathogenicity genes located at specific chromosomal locations near the replication termination region and rRNA-encoding operons. Half of the mutation events catalyzed by Pol IV impaired gene function. This can be attributed to the bias of Pol IV for mutating codons with its preferred sequence contexts, leading to substitutions to unreactive alanine and glycine residues. Remarkably, mutation signatures identified for Pol IV were found in clinical isolate genomes of P. aeruginosa, providing compelling evidence for its role in genetic diversification during pathogen adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54f7c02761e12b8b73eebc31b5beda14fbe3ac0c" target='_blank'>
              The low-fidelity DNA Pol IV accelerates evolution of pathogenicity genes in Pseudomonas aeruginosa
              </a>
            </td>
          <td>
            Sofía D. Castell, Consuelo M. Fernandez, Ignacio N. Tumas, L. M. Margara, María C. Miserendino, Danilo Ceschin, R. Pezza, Mariela R. Monti
          </td>
          <td>2025-08-02</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Mitotic bookmarking, the retention of regulatory proteins and lncRNAs on chromatin during mitosis, epigenetically sustains competency for phenotype-specific gene expression in progeny cells. Gene expression is predominantly suppressed during mitosis. Bookmarking provides the guidance for the resumption of gene expression in progeny cells that is obligatory for physiological control of lineage commitment, specialized cell structure and phenotypic function. While regulatory continuity is supported by the persistence of genome-associated regulatory complexes, altered bookmarking mediates plasticity for responsiveness to physiological cues. Bookmarking fidelity ensures genome integrity and controls expression of tumor suppressors and proto-oncogenes. Cancer-compromised aberrations in bookmarking results in transcriptional dysregulation and the initiation of tumor-associated processes. Keywords: Mitotic Bookmarking, Mitosis, Epigenetic Control, Gene Expression, Cell Division.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1d78052a96f4610f52d1801e1448869b1e34100" target='_blank'>
              Mitotic Bookmarking Provides Epigenetic Persistence or Plasticity for Biological Control and Cancer.
              </a>
            </td>
          <td>
            Andrew J. Fritz, Emory Pacht, R. Toor, Janine Warren, Haley W Greenyer, Jackson R Del Porto, Abigail G Person, Sadie J Korzec, Georgiy Zotkin, Jessica L Heath, Prachi N Ghule, J. Gordon, Andre J. van Wijnen, Seth E Frietze, Karen C. Glass, Jane Lian, Jane L. Stein, Gary S. Stein
          </td>
          <td>2025-07-16</td>
          <td>Biochemistry and cell biology = Biochimie et biologie cellulaire</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Cytosine DNA methylation patterns vary widely across eukaryotes, with its ancestral roles being understood to have included both transposable element silencing and host gene regulation. To further explore these claims, in this study, we reevaluate the evolutionary origins of DNA methyltransferases and characterise the roles of cytosine methylation on underexplored lineages, including the amoebozoan Acanthamoeba castellanii, the glaucophyte Cyanophora paradoxa, and the heterolobosean Naegleria gruberi. Our analysis of DNA methyltransferase evolution reveals a rich ancestral eukaryotic repertoire, with several eukaryotic lineages likely subsequently acquiring enzymes through lateral gene transfer (LGT). In the three species examined, DNA methylation is enriched on young transposable elements and silenced genes, suggesting an ancestral repressive function, without the transcription-linked gene body methylation of plants and animals. Consistent with this link with silencing, methylated genomic regions co-localise with heterochromatin marks, including H3K9me3 and H3K27me3. Notably, the closest homologues of many of the silenced, methylated genes in diverse eukaryotes belong to viruses, including giant viruses. Given the widespread occurrence of this pattern across diverse eukaryotic groups, we propose that cytosine methylation was a silencing mechanism originally acquired from bacterial donors which was used to mitigate the expression of both transposable and viral elements, and that this function may persist in creating a permissive atmosphere for LGT in diverse eukaryotic lineages. These findings further highlight the importance of epigenetic information to annotate eukaryotic genomes, as it helps delimit potentially adaptive LGTs from silenced parasitic elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48579c32d97033a0fd54ac2bedb8cb3bebb700ed" target='_blank'>
              Repressive cytosine methylation is a marker of viral gene transfer across divergent eukaryotes.
              </a>
            </td>
          <td>
            Luke A. Sarre, Giselle Azucena Gastellou Peralta, Pedro Romero Charria, Vladimir Ovchinnikov, Alex de Mendoza
          </td>
          <td>2025-07-25</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="During carcinogenesis, cells must acquire a telomere maintenance mechanism in order to avoid telomere shortening-induced replicative senescence. While most tumors activate telomerase, a minority of them employ a recombinational mechanism called Alternative Lengthening of Telomeres (ALT). One of the most investigated features is the association between ALT and ATRX mutations, since this has been shown to be the gene with the highest rate of mutations among ALT tumors. However, most of these studies, and in particular, mechanistic studies in vitro, have been carried out on mesenchymal tumors (sarcomas). In the present study, using genomic and expression data from the DepMap portal, we identified several non-mesenchymal ALT cell lines, and we compared the incidence of ATRX and other gene mutations between ALT cell lines of different origins (mesenchymal, neural, epithelial, hematopoietic). We confirmed that ATRX is frequently mutated in mesenchymal and neural ALT cell lines but not in epithelial ones. Our results showed that mutations of ATRX or other proteins involved in the maintenance of telomere integrity are needed for ALT activation in all cell types, and ATRX is preferentially mutated in mesenchymal ALT cells. Besides a more precise interpretation of the role of ATRX loss in ALT establishment, we proposed a model in which mutation of this gene impairs differentiation in mesenchymal and neural cells (but not in epithelial ones). Therefore, we explained the high incidence of ATRX mutations in mesenchymal and neural tumors with the fact that they both trigger ALT and impair differentiation, thus promoting two steps at once in the process of carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f454c9a5bcf29e99c79d4426a0a12fbbf069c0ed" target='_blank'>
              Alternative Lengthening of Telomeres: The Need for ATRX Mutations Is Lineage-Dependent
              </a>
            </td>
          <td>
            I. Udroiu, Antonella Sgura
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most prevalent and deadly neoplasms globally; this fact puts emphasis on the need for accurate molecular biomarkers for early detection and accurate prognosis. Circular RNAs (circRNAs) have recently emerged as very promising cancer biomarkers. In this study, we thoroughly examined whether the expression levels of circular transcripts of the protein arginine methyltransferase 1 (PRMT1) gene can predict the prognosis of patients diagnosed with colorectal adenocarcinoma, the most frequent type of CRC. Hence, a highly sensitive quantitative PCR (qPCR) assay was developed and applied to quantify circ-PRMT1 expression in cDNAs from 210 primary colorectal adenocarcinoma tissue specimens and 86 paired normal colorectal mucosae. Extensive biostatistical analysis was then performed to assess the potential prognostic power of circ-PRMT1. Significant overexpression of this molecule was observed in colorectal adenocarcinoma tissue samples in contrast to their non-cancerous counterparts. Moreover, higher circ-PRMT1 expression was correlated with poorer disease-free survival (DFS) and worse overall survival (OS) in colorectal adenocarcinoma patients. Interestingly, multivariate Cox regression analysis revealed that the prognostic value of the expression of this circRNA does not depend on other established prognostic factors included in the prognostic model. Furthermore, the stratification of patients based on TNM staging revealed that higher circ-PRMT1 levels were significantly related to shorter DFS and OS intervals, particularly in patients with colorectal adenocarcinoma of TNM stage II or III. In summary, this original research study provides evidence that circ-PRMT1 overexpression represents a promising molecular biomarker of poor prognosis in colorectal adenocarcinoma, not depending on other established prognostic factors such as TNM staging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b06ea5f530ca9a15a72069b8a7542753ca04c1f" target='_blank'>
              Overexpression of Circular PRMT1 Transcripts in Colorectal Adenocarcinoma Predicts Recurrence and Poor Overall Survival
              </a>
            </td>
          <td>
            P. Kokoropoulos, S. Christodoulou, P. Tsiakanikas, P. Vassiliu, C. Kontos, Nikolaos Arkadopoulos
          </td>
          <td>2025-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has been described in several eukaryotic species and has been shown to impact phenomena as diverse as cancer and herbicide tolerance. EccDNA is thought to arise mainly through transposable element (TE) mobilization. Because studies based on short-read sequencing cannot efficiently identify full-length eccDNA forms generated from TEs, we employed the CIDER-Seq pipeline based on long-read sequencing, to obtain full-length eccDNAs from Arabidopsis. The generated eccDNA datasets identified centromeric/pericentromeric regions as hotspots of eccDNAs with several eccDNA molecules originating from Helitron and LTR TEs. To investigate the role of epigenetic marks on TE-derived eccDNA biogenesis, we studied Arabidopsis methylation mutants dcl3, rdr6, ros1, and ddm1. Contrasting the TE-suppression previously reported in the hypermethylated ros1 mutants, we identified activation of TEs in ros1, specifically of LTR/Gypsy TEs. An enrichment of LTR/Copia elements was identified in actively dividing calli and the shoot apical meristem (SAM). Uncharacterized “variable TEs” with high eccDNA and expression were identified in the SAM, including ATCOPIA58. Together, our study reveals the genomic origins of eccDNAs and delineates the link between epigenetic regulation, transposon mobilization, and eccDNA biogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a8fe16883b4dfe42fe6b49221fc2e6a347cfdc2" target='_blank'>
              A comprehensive atlas of full-length Arabidopsis eccDNA populations identifies their genomic origins and epigenetic regulation
              </a>
            </td>
          <td>
            S. S. Zaidi, Sara Shakir, Hanne De Kort, Devang Mehta, V. Nguyen, Ruben Gutzat, Hervé Vanderschuren
          </td>
          <td>2025-07-01</td>
          <td>PLOS Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Although the significance of intratumoral heterogeneity (ITH) has been widely acknowledged in various cancers, its role in pancreatic cancer (PC) remains underexplored and warrants further investigation. Methods Pancreatic cancer transcriptomic data were acquired from the TCGA and GEO databases. The DEPTH2 algorithm, in combination with differential expression analysis, was used to identify genes associated with intratumoral heterogeneity (ITH). We applied univariate Cox regression analysis and multiple machine learning techniques to establish a reliable prognostic model. Patients were then stratified according to their ITH scores, and differences between subgroups were examined through pathway enrichment analysis, immune cell infiltration profiling, and drug response prediction. Furthermore, we conducted subcellular localization and differential analysis of ITH using single-cell data, followed by cell-cell communication analysis to explore interaction relationships and identify key pathways. Results Patients exhibiting lower intratumoral heterogeneity (ITH) levels demonstrated poorer clinical outcomes. The constructed 11-gene signature successfully differentiated individuals into high- and low-risk categories with significant survival differences. Immune profiling revealed notable differences in immune cell composition between the two groups, with patients in the high ITH cohort exhibiting enhanced immune activation. Drug sensitivity analysis indicated a differential response to therapies, with high-risk patients more resistant to certain drugs. Single-cell RNA sequencing identified a greater ITH score in epithelial cells, highlighting key interactions, particularly involving Galectin signaling pathways. Conclusion Our results highlight intratumoral heterogeneity’s prognostic and therapeutic relevance in pancreatic cancer, suggesting its potential utility in guiding individualized treatment approaches. Supplementary Information The online version contains supplementary material available at 10.1007/s12672-025-03080-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c15cd4c65cc7acb2163700609aefaa2af8b33ee" target='_blank'>
              Integrating bulk and single-cell RNA sequencing reveals intratumor heterogeneity phenotypes and immune infiltration in pancreatic cancer
              </a>
            </td>
          <td>
            Li Zhao, Yao Huang, Jiang Li, Rui Xie, Kaiming Li
          </td>
          <td>2025-07-13</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9589673dbf4b0dcfb9624f20bcd04da6bc6ed357" target='_blank'>
              YTHDF3 recognizes DNA N6-methyladenine and recruits ALKBH1 for 6mA removal from genomic DNA.
              </a>
            </td>
          <td>
            Xin-Hui Chen, Zi-Lu Wang, Jincui Yang, Min Chen, Si-Yi Zhao, Kun-Xiong Guo, Xuelong Zheng, Zhengwei Zhao, Xiaoqiang Chen, Jing Li, Minmin Zhang, Ling Ran, Huifang Zhu, Xiaofeng Gu, Guang-Rong Yan
          </td>
          <td>2025-07-25</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 The single-stranded DNA cytosine deaminase enzyme, APOBEC3B (A3B), has been implicated as a mutational driver in multiple human cancers1,2. Breast cancer in particular shows high levels of A3B expression and positive associations with poor clinical outcomes3. Recent studies have also demonstrated that human A3B expression in mice is carcinogenic with significantly shortened life expectancies4. The only genetic factor in breast cancer thus far associated with A3B expression and APOBEC signature single base substitutions is p53 mutation5. Therefore, here we test the hypothesis in vivo that human A3B expression will show pathlogical synergy in the form of accelerated mammary tumor penetrance upon combination with p53 haploinsufficiency due to a heterozygous deletion mutation spanning exons 2-10. Custom and commercially available animals were subjected to standard breeding practices to generate experimental (A3B, p53Delta exons 2-10/+) and control groups (p53Deltanexons 2-10/+). Surprisingly, in contrast to A3B-accelerated tumor development on a wildtype genetic background, A3B had no effect on the rates of tumor development in p53 heterozygous animals (median 13.5 months). The observed genetic epistasis was not due to a lack of A3B expression or activity, as evidenced by strong IHC positivity and a clear acculumlation of APOBEC signature single base substitution mutation in tumors. We conclude that A3B-accelerated tumor development requires full p53 function, and that a haploinsufficiency in p53 enables tumor cells (or their precursors) to better tolerate DNA damage lesions induced by A3B.
 Selected References:
 1. Petljak, M. et al. Addressing the benefits of inhibiting APOBEC3-dependent mutagenesis in cancer. Nat Genet 54, 1599-1608 (2022).
 2. Butler, K. & Banday, A. R. APOBEC3-mediated mutagenesis in cancer: causes, clinical significance and therapeutic potential. J Hematol Oncol 16, 31 (2023).
 3. Roelofs, P. A. et al. Clinical implications of APOBEC3-mediated mutagenesis in breast cancer. Clin Cancer Res 29, 1658-1669 (2023).
 4. Durfee, C. et al. Human APOBEC3B promotes tumor development in vivo including signature mutations and metastases. Cell Rep Med 4, 101211 (2023).
 5. Burns, M. B. et al. APOBEC3B is an enzymatic source of mutation in breast cancer. Nature 494, 366–370 (2013).\
 Citation Format: Joshua Proehl, Cameron Durfee, Yuan Zhao, Nuri Alpay Temiz, Reuben Harris. p53 Haploinsufficiency Epistatically Masks A3B Tumor Phenotype in vivo [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-06-26.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8de4891dd8d8ec2c399682371385c3645385994" target='_blank'>
              Abstract P3-06-26: p53 Haploinsufficiency Epistatically Masks A3B Tumor Phenotype in vivo
              </a>
            </td>
          <td>
            Joshua Proehl, Cameron Durfee, Yuan Zhao, N. A. Temiz, R. Harris
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Abstract The bacterial cytoplasm is characterized by a distinctive membrane-less organelle, the nucleoid, which harbors chromosomal DNA. We investigate the effects of dynamic processes associated with transcription and translation on the structure of this organelle, using coarsegrained molecular dynamics (MD) simulations implemented with out-of-equilibrium reactions. Our model captures the scale of the entire cell and incorporates a reaction-diffusion system for ribosomes and polyribosomes, combining their out-of-equilibrium dynamics with excluded volume interactions with DNA. Our findings demonstrate that out-of-equilibrium reactions increase the size of the nucleoid. In addition, we show that the nucleoid size increase is proportional to transcriptional activity. Our model reproduces the time-dependent change in nucleoid size observed in rifampicin treatment experiments, where the pool of polyribosomes is depleted. Furthermore, we find these active processes are essential for complete sister chromosome separation and correct nucleoid positioning within the cell. Overall, our study reveals the effects of the central dogma processes on the internal organization and localization of bacterial nucleoids. Significance Understanding how bacteria organize their chromosomes is fundamental to cell biology. Through our coarse-grained molecular dynamics simulations incorporating out-of-equilibrium processes of transcription and translation, we are able to capture the effects of these central dogma processes on DNA organization and demonstrate that these active biological processes expand the nucleoid and facilitate the separation of daughter chromosomes. Our simulations are compared to experimental measurements and highlight the impact of the out-of-equilibrium conditions of the living cell. These findings point out the crucial interplay between physical forces and biological activity in cellular organization, suggesting that cellular structure depends on non-equilibrium processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5869c17c0b2f03dcafbf482f5c2ecb59859979e3" target='_blank'>
              Effects of central dogma processes on the compaction and segregation of bacterial nucleoids
              </a>
            </td>
          <td>
            Mu-Hung Chang, Maxim O. Lavrentovich, Jan-Michael Y. Carrillo, Jaan Männik
          </td>
          <td>2025-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31620c8deab30d15f98e2b8cf7eeeade98239959" target='_blank'>
              LINC01410 in cancer: a comprehensive review of its oncogenic role, regulatory mechanisms, and clinical implications.
              </a>
            </td>
          <td>
            Saman Morovat, Pejman Morovat, Ashkan Kalantary-Charvadeh, Marzieh Mojbafan, S. Teimourian
          </td>
          <td>2025-07-12</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background Glioblastoma (GBM) originates from neuroepithelial tissue and is one of the most common intracranial malignant tumors in adults, with high recurrence rate and poor prognosis. In recent years, SOX9 has been reported to play an important role in many diseases and cancers, and is a promising target, but it has been rarely reported in GBM. Methods RNA sequencing data of GBM were obtained from the Cancer Genome Atlas (TCGA) database and the Genotype-Tissue Expression (GTEx) database for analysis of SOX9 expression and differentially expressed genes (DEGs). Moreover, functional enrichment analysis of GBM-related DEGs was performed by GO/KEGG, GSEA, and protein-protein interaction (PPI) network. Additionally, the clinical significance of SOX9 in GBM was assessed by Kaplan-Meier Cox regression and prognostic model. What’s more, we analyzed SOX9-related immune cell infiltration and expression of immune checkpoints in GBM. The incorporated studies were analyzed using the R package. Results SOX9 was highly expressed in a range of malignant tumor tissues, including GBM. Surprisingly, high SOX9 expression was remarkably associated with better prognosis in the lymphoid invasion subgroups in a sample of 478 cases (P < 0.05). Totally, 126 differentially significant genes (DSGs) were identified between high- and low- expression group, of which 29 genes were upregulated and 97 genes were downregulated. Furthermore, high expression of SOX9 was an independent prognostic factor for IDH (isocitrate dehydrogenase)-mutant in Cox regression analysis. Screening was performed by LASSO coefficients to select non-zero variables that satisfied the coefficients of lambda. min, and four genes were screened out. OR4K2 and IDH status were prognostic factors associated with THCA in multifactorial COX regression analysis. SOX9, OR4K2 and IDH status were included in the nomogram prognostic model. Correlation analysis indicated SOX9 expression was correlated with immune cell infiltration and expression of immune checkpoints in GBM. Conclusion SOX9 was identified as a diagnostic and prognostic biomarker in glioblastoma, particularly in IDH-mutant cases. Its expression was closely correlated with immune infiltration and checkpoint expression, indicating its involvement in the immunosuppressive tumor microenvironment. SOX9-based gene signatures further supported a robust nomogram model, underscoring its potential as a therapeutic and prognostic target in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a2a8c97c4df9bf9651c0f0af4a4efc671535b22" target='_blank'>
              High expression of SOX9 is a diagnostic and prognostic indicator of glioma
              </a>
            </td>
          <td>
            Libo Xu, Zhenhao Wang, Mao Li, Qingsong Li
          </td>
          <td>2025-07-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7dc9684ae5c68a44233b20b2227e17304b0f29d3" target='_blank'>
              Evolutionary insights into provirus-encoded CRISPR-Cas systems in halophilic archaea
              </a>
            </td>
          <td>
            Doron Naki, U. Gophna
          </td>
          <td>2025-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction Pancreatic cancer is a highly malignant tumor of the digestive system with a dismal prognosis. Despite advances in diagnosis and treatment, overall survival remains extremely low. Early diagnostic markers and an improved understanding of tumor-microenvironment interactions are essential for developing more effective therapies. Methods We analyzed 74 single-cell RNA sequencing (scRNA-seq) samples, performing unsupervised clustering and marker-gene expression profiling to define major cell types. Large-scale chromosomal copy-number variation (CNV) analysis distinguished malignant from non-malignant ductal cells. Non-negative matrix factorization (NMF) identified stage-associated gene modules, which were integrated with TCGA bulk-RNA data and machine-learning feature selection to pinpoint candidate prognostic genes. Two independent cohorts were used for validation. Regulatory network inference (pySCENIC) and ligand–receptor interaction analysis (CellPhoneDB) explored cross-talk between malignant cells and macrophages. Finally, in vitro knockdown of CTSV assessed its functional role in pancreatic cancer (PAC) cell proliferation and migration. Results Three prognosis-related genes—ANLN, NT5E, and CTSV—were selected based on their strong association with clinical stage and validated in external datasets. High expression of these genes correlated with poorer overall survival and an increased infiltration of M0 macrophages. CellPhoneDB predicted significant interactions between high-expression malignant ductal cells and M0 macrophages via CXCL14–CXCR4 and IL1RAP–PTPRF axes, with SPI1 identified as an upstream regulator of IL1RAP. In vitro CTSV knockdown significantly inhibited PAC cell proliferation and migration. Discussion Our integrative single-cell and bulk-RNA workflow identifies ANLN, NT5E, and CTSV as novel prognostic biomarkers in pancreatic cancer and highlights a pro-tumorigenic interaction between malignant ductal cells and macrophages. Targeting CTSV or disrupting CXCL14–CXCR4 and IL1RAP–PTPRF signaling may offer new therapeutic avenues for PAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d50ce5676a35dfc57b1b078e8894ee25f419175d" target='_blank'>
              Integration of single-cell RNA and bulk RNA sequencing revealed malignant ductal cell heterogeneity and prognosis signatures in pancreatic cancer
              </a>
            </td>
          <td>
            Haiyang Du, Gao Si, Jiqing Si, Xuejie Song, Fuchun Si
          </td>
          <td>2025-07-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer cells utilize codon-biased translation to fuel tumorigenesis and drug resistance, but underlying mechanisms remain poorly understood. Here, we show ALKBH1 is overexpressed in acute myeloid leukemia (AML) and essential for leukemia stem/initiating cell (LSC/LIC) self-renewal and AML development/maintenance, whereas dispensable for normal hematopoiesis. ALKBH1 enhances mitochondrial assembly/function and oxidative phosphorylation (OXPHOS), crucial for AML survival/proliferation and resistance to venetoclax, a potent BCL2 inhibitor and widely-used first-line targeted therapy for AML in clinic. Mechanistically, ALKBH1 catalyzes 5-formylcytosine (f5C) at tRNA wobble positions, reprograming decoding and facilitating codon-biased translation, a mechanism we term "Epitranslatomic Midas touch", which in turn drives leukemogenesis and drug resistance by promoting synthesis of key oncogenic proteins like WDR43. Targeting ALKBH1, particularly together with venetoclax, exhibited potent anti-leukemia efficacy in preclinical models with favorable safety profiles. Collectively, our findings elucidate ALKBH1's pivotal role in codon-biased translation and tumorigenesis, and propose a novel therapeutic strategy for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a3e11cf9a76aba0c7266c1034a7a9bfa25e9f8a" target='_blank'>
              ALKBH1 drives tumorigenesis and drug resistance via tRNA decoding reprogramming and codon-biased translation.
              </a>
            </td>
          <td>
            Chao Shen, Yuan Che, Keren Zhou, Kitty Wang, Wei Li, Diyuan Xue, Tong Wu, Lu Yang, Meiling Chen, Yue Sheng, Chengwan Zhang, S. Robinson, Huiying Chen, Lillian Sau, Zhenhua Chen, M. Wunderlich, Li Han, Tingting Tang, Y. Qing, Dong Wu, Miao Sun, Keith Leung, Katarzyna Dąbrowska, P. Pirrotte, Yueh-Chiang Hu, Lucy Y Ghoda, Bin Zhang, J. Mulloy, Minjie Wei, Guido Marcucci, Chuan He, Xiaolan Deng, Jianjun Chen
          </td>
          <td>2025-08-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Background DNA changes induced by stress may be stable through cell division and passed to subsequent generations. Plant improvement programs require that individuals develop a new heritable trait. The action of mutagens applied in classical mutagenesis is known from their transgenerational inheritance, observed as mutations. DNA methylation is an epigenetic mechanism that generates heritable phenotypic variation by influencing gene expression and modifying DNA accessibility to mutagens. Fagopyrum esculentum (common buckwheat) and F. tataricum (Tartary buckwheat) are essential for food production and valuable for medical purposes. Results In this study, we aimed to investigate the transgenerational changes in the genome and epigenome of two Fagopyrum species, by analysing 1st and 3rd generations, followed by treatment with gamma rays. The genomic instability, observed as DNA fragmentation, micronuclei, and changes in cell cycle profile, was observed in 1st generation, whereas 3rd generation shows recovery in DNA stability. Gamma-ray stress caused alterations in the pattern and level of DNA methylation in the roots of both analysed generations. The response to gamma rays depends on the species – in 1st generation, a higher increase in DNA methylation level in F. esculentum and only slight changes in F. tataricum were observed. The DNA methylation level in the 3rd generation changed only in F. esculentum. Conclusions Gamma ray is a stress factor that affects the pattern and level of DNA methylation in F. esculentum and F. tataricum. Significant differences were observed between control and 3rd generation only in F. esculentum. Differences in DNA methylation following gamma irradiation between the analysed Fagopyrum species may result from various forms of pollination in the analysed species. By using gamma ray-induced mutation, we got a highly stable mutant of F. tataricum, which may result in a self-pollinating trait. Understanding the changes in DNA methylation following mutagenic treatment can be essential in a breeding programme. Supplementary Information The online version contains supplementary material available at 10.1186/s12870-025-07033-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16c669739a40e63380b2299a72c170061851584f" target='_blank'>
              Transgenerational genetic and epigenetic changes induced by gamma-ray in Fagopyrum species
              </a>
            </td>
          <td>
            Katarzyna Sala-Cholewa, Magdalena Rojek-Jelonek, Jolanta Kwasniewska
          </td>
          <td>2025-07-31</td>
          <td>BMC Plant Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Neoplastic transformation of mammalian cells involves intricate interactions between genetic, epigenetic and architecture modifications of the nucleus. Neuroblastoma is a malignant pediatric tumor with high biological and clinical heterogeneity representing a challenging model of study. We aimed to explore the changes in genome architecture and epigenetics being associated with neuroblastoma malignancy. We employed the neuroblastoma cell line SKNBE2 overexpressing the ncRNA NDM29 to differentiate from highly malignant into neuron-like cells. By 3D confocal microscopy, we explored the nuclear architecture (volume, elongation, compactness, and chromatin density). Using super-resolution microscopy (STED) and histone H3 immunolabelling we assessed the epigenetic rearrangement, and by enzyme-linked immunoassay the global DNA methylation. Then we assessed the mRNA expression of the main epigenetic modifying enzymes by quantitative PCR, and the expression of NF-κB-regulated genes by cDNA microarray. Compared to malignant NB cells, the NDM29-overexpressing cells, assuming a neuron-like phenotype, exhibited smaller and more elongated nuclei, redistribution of H3K9-acetylated and -methylated chromatin domains and DNA hypermethylation. In line with these results, in neuron-like cells the acetyltransferase KAT2A and the DNA methyltransferase DNMT1 were up-regulated, while most of NF-κB-regulated genes were down-regulated. Our findings reveal modifications of the nuclear structure and epigenome during neuroblastoma retro-transformation induced by NDM29 overexpression, with impacts on gene expression. These results offer potential insights into better understanding the mechanism of neuroblastoma malignancy in terms of chromatin rearrangements, opening exciting prospects for prognostic and therapeutic approaches with a focus on the nuclear level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53128c07de4d9b019036fd21bd73a5a365d261f0" target='_blank'>
              Epigenome and three-dimensional genome architecture remodeling during NDM29-mediated retro-transformation of neuroblastoma cells
              </a>
            </td>
          <td>
            Francesca Baldini, Aldo Pagano, L. Zeaiter, P. Bianchini, Hawraa Zbeeb, A. Diaspro, Laura Vergani
          </td>
          <td>2025-07-31</td>
          <td>PLOS One</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d7803de2661e62cd9d27e86eb2d0ae2e148e6e4" target='_blank'>
              Chromatin Control of EBV Infection and Latency.
              </a>
            </td>
          <td>
            P. Lieberman, I. Tempera
          </td>
          <td>2025-07-13</td>
          <td>Current topics in microbiology and immunology</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Clonal expansions of hematopoietic cells carrying mosaic chromosomal alterations (mCAs) are commonly detectable in elderly individuals. Here, we studied 43,617 autosomal mCAs that we ascertained in 484,081 UK Biobank participants using new, high-resolution computational methods to analyze blood-derived whole-genome sequencing data. Shorter mCAs ([≤]1 Mb) clustered at 53 genomic hotspots (34 previously undetected), several of which implicated chromosomal fragile sites as a recurrent source of somatic deletions. Chronic lymphocytic leukemia (CLL)-associated deletions at 13q14 were detectable in 1% of 65- to 70-year-old individuals -- a five-fold higher rate than previously observed -- suggesting opportunities for incorporating this mosaic mutation in clinical screening and in genetic association studies. Rare protein-coding variants in 38 genes associated (p<1.2x10-5; FDR<0.01) with clonal expansions of copy-neutral loss of heterozygosity (CN-LOH) mutations that modified the allelic dosages of these variants, implicating genes in DNA damage response, cell cycle, cytokine signaling, and protein ubiquitination pathways as targets of clonal selection via CN-LOH-induced allelic substitution. For several genes with proliferation-increasing functions, CN-LOH mutations produced revertant mosaicism, removing deleterious inherited variants. Polygenic effects of common inherited variants within CN-LOH mutations also contributed modestly to clonal selection. These results show that our blood genomes are constantly evolving as we age and often accrue mCAs in predictable ways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d082ef6098c2dc18ea4d90e8ab3334b1e46a5ee" target='_blank'>
              Patterns and drivers of 43,617 mosaic chromosomal alterations in blood
              </a>
            </td>
          <td>
            D. Tang, N. Kamitaki, R. E. Mukamel, S. Rubinacci, P.-R. Loh
          </td>
          <td>2025-07-30</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 The loss of chromosome 16q is the single most common genomic event in BC, occurring in over 50% of tumors. Despite this, the literature is focused almost exclusively on the importance of focal mutations that show a maximum frequency of ∼35-40%. We have modeled 16q loss by deleted this region in mammary epithelium of genetically modified mice. This event was sufficient to induce mammary tumor formation. This finding establishes for the first time that 16q is a BC-suppressing chromosome arm. We have gone on to identify two genes on 16q that can promote tumor formation when one copy is lost (Cbfb and Ankrd11, each in cooperation with Pik3caHR). We have also identified mutations that cooperate with 16q syntenic sequence loss in spontaneous tumors from our 16q mouse model (c-Met gene amplification and p53 mutation) as well as in a Sleeping Beauty transposon-based screen (identifying mutations that enhance Ras and Rho signaling). Finally, we have used transcriptional profiling and proteomics to compare matching mammary tumors with/without 16q-syntenic sequence loss, which revealed that hemizygous deletion of this region suppressed keratinocyte-like differentiation in mammary tumors.
 Citation Format: Sean Egan, Idil Eda Temel, YeJi An, Katelyn Kozma, Amanda Loch, Wei Wang. 16q is a breast cancer suppressor arm [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P3-04-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30e57f36d665bf1dd94705e6477b5680fa560c1a" target='_blank'>
              Abstract P3-04-17: 16q is a breast cancer suppressor arm
              </a>
            </td>
          <td>
            Sean E Egan, Idil Eda Temel, Yeji An, Katelyn J Kozma, Amanda J. Loch, Wei Wang
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Extrachromosomal and mobile genetic elements, including plasmids, and accessory chromosomes, are prevalent in all life domains. Elements integrated into the host chromosome replicate and segregate via the host life cycle. In contrast, the persistence of autonomously replicating elements relies on their ability to remain within the host population. Here, we compare the evolutionary advantage of different persistence strategies found in prokaryotic plasmids. Through intracellular competitions between plasmid genotypes, we find that the combination of active partitioning during cell division with a toxin-antitoxin (TA) system for post-segregational killing increases plasmid fitness more than either strategy alone. Mathematical modeling of long-term plasmid evolution, calibrated with empirical plasmid loss dynamics, further supports these findings. A survey of enterobacterial genomes indicates that partitioning and TA systems are core features of large plasmids. Indeed, we confirm the presence of a previously unrecognized type I TA system in conjugative IncX3 plasmids, which serve as important vectors of antibiotic resistance in human pathogens. These findings suggest that large plasmids – including conjugative and mobilizable types – encode TA systems, some of which have yet to be identified. The combination of TA and partitioning systems emerges as the most effective strategy for the evolutionary success of low-copy extrachromosomal elements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ddba67c058e73d4520b91af55962bc53698843f" target='_blank'>
              The combination of active partitioning and toxin-antitoxin systems is most advantageous for low-copy plasmid fitness
              </a>
            </td>
          <td>
            Johannes Effe, M. Santer, Yiqing Wang, Theresa E. Feenstra, Nils F. Hülter, Tal Dagan
          </td>
          <td>2025-08-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="The induced “recombinogenic situation” in hematopoietic stem cells and the activation of the cell’s reparative systems create the basis for recombination events between fragments of extracellular double-stranded DNA delivered into the cell and chromosomal DNA or other forms of the reparative-recombination process. In mouse and rat model organisms as well as in human bone marrow cells, changes in the amount of telomeric DNA in hematopoietic stem cells were assessed as an indicator of repair and recombination events that have occurred. In all experiments performed, recombinant human angiogenin was used as a comparison factor. Dot blot hybridization showed that in the colony cells obtained from the bone marrow cells of the model organisms as well as from human bone marrow cells treated with a double-stranded DNA preparation, there was a significant increase in the amount of telomeric DNA. Amplification of telomeric DNA in colony cells is not associated with contamination of the original DNA preparation with which the bone marrow cells were treated. Treatment of bone marrow cells with DNA that does not carry telomeric sequences (AluI PCR fragment) does not lead to an increase in the amount of telomeric DNA in the cells of grown colonies. This suggests the participation in the amplification of telomeric DNA of an extrachromosomal DNA template carrying telomeric DNA. It has been established that treatment of bone marrow cells with angiogenin also leads to an increase in telomeric DNA in colony cells. A comparison of the type of colonies with the intensity of hybridization (i. e. the amount of telomeric DNA in the sample) suggests that the increase in the amount of detectable telomeric DNA following treatment with angiogenin and hDNAgr has a fundamentally different origin. Western blot analysis and real-time PCR revealed that the increase in the amount of telomeric DNA following treatment of bone marrow cells with a double-stranded DNA preparation does not correlate with the activity of endogenous/exogenous telomerase. For angiogenin, it has been shown that an increase in the amount of telomeric DNA may be the result of activation of endogenous telomerase activity. A principle has been developed for the amplification of a new genetic trait that came into hematopoietic stem cells with extracellular double-stranded DNA material and was fixed in the recipient genome or was transitively present in the cell as new genetic information.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11f0c77ec133a09bb58f64518224081f769056e8" target='_blank'>
              Concept of natural genome reconstruction. Part 3. Analysis of changes in the amount of telomeric DNA in colony cells as a new amplified feature that arose during the processing of hematopoietic bone marrow stem cells
              </a>
            </td>
          <td>
            V. Ruzanova, S. Oshikhmina, G. S. Ritter, E. Dolgova, S. Kirikovich, E. Levites, Y. R. Efremov, T. V. Karamysheva, A. Bogomolov, M.I. Meschaninova, A. L. Mamaev, O. Taranov, S. V. Sidorov, S. D. Nikonov, O. Leplina, A. Ostanin, E. Chernykh, N. A. Kolchanov, A. Proskurina, S. S. Bogachev
          </td>
          <td>2025-07-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Mobile genetic elements (e.g. transposons and introns) are genomic parasites, DNA sequences that can proliferate within host genomes. In prokaryotes, these elements are minimized by purifying selection and host-cell defenses. In eukaryotes, however, these parasitic sequences may comprise most of the genome. This extreme imbalance in eukaryotes is maintained by a metabolically quiescent germline through which the genomic parasites are transmitted to the next generation via gametes and spread throughout the population via sexual reproduction. Some eukaryotes avoid the threat posed by genomic parasites by eliminating them from somatic cells, whereas others retain the parasites in somatic cells and attempt to suppress them by epigenetic silencing mechanisms. Here, we review the evidence for the ongoing competition between host and parasite for genome occupancy. We conclude that defenses against mobile genetic elements vary greatly among organisms, and this variation accounts for the enormous range in genome size among organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58c5828d52742180cad7b0c29c98a298b55b8bda" target='_blank'>
              The biological and evolutionary consequences of competition between DNA sequences that benefit the cell and DNA sequences that benefit themselves
              </a>
            </td>
          <td>
            A. J. Bendich, Scott O. Rogers
          </td>
          <td>2025-07-08</td>
          <td>Nucleic Acids Research</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="Replication timing (RT), the temporal order of DNA replication during S phase, influences regional mutation rates, yet the mechanistic basis for RT-associated mutagenesis remains incompletely defined. To identify drivers of RT-dependent mutation biases, we analyzed whole-genome sequencing data from cells with disruptions in DNA replication/repair genes or exposed to mutagenic compounds. Mutation distributions between early- and late-replicating regions were compared using bootstrapping and statistical modeling. We identified 14 genes that exhibit differential effects in early- or late-replicating regions, encompassing multiple DNA repair pathways, including mismatch repair (MLH1, MSH2, MSH6, PMS1, and PMS2), trans-lesion DNA synthesis (REV1) and double-strand break repair (DCLRE1A and PRKDC), DNA polymerases (POLB, POLE3, and POLE4), and other genes central to genomic instability (PARP1 and TP53). Similar analyses of mutagenic compounds revealed 19 compounds with differential effects on replication timing. These results establish replication timing as a critical modulator of mutagenesis, with distinct DNA repair pathways and exogenous agents exhibiting replication timing-specific effects on genomic instability. Our systematic bioinformatics approach identifies new DNA repair genes and mutagens that exhibit differential activity during the S phase. These findings pave the way for further investigation of factors that contribute to genome instability during cancer transformation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13a232208698bfbf694bd7e822cd1f4798018e9a" target='_blank'>
              Unveiling Replication Timing-Dependent Mutational Biases: Mechanistic Insights from Gene Knockouts and Genotoxins Exposures
              </a>
            </td>
          <td>
            Hadas Gross-Samuels, A. Koren, Itamar Simon
          </td>
          <td>2025-07-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8fda3215ef96d1797670c5063bd5db4df44e918" target='_blank'>
              Genetic variants in oncogenic miRNA and 3' untranslated region of tumor suppressor genes: emerging insight into cancer genetics.
              </a>
            </td>
          <td>
            Anita Choudhary, Anil Kumar, A. Munshi
          </td>
          <td>2025-06-30</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Cancer progression and development are influenced by a complex interplay between inherited (germline) and acquired (somatic) mutations. Current cancer genomic research in Middle Eastern populations predominantly utilizes targeted panels to examine predefined genes, potentially overlooking a broader spectrum of genomic contributions to cancer predisposition. Addressing this gap, this study adopts an unbiased approach using whole exome sequencing (WES) data to identify both high- and low-penetrance genetic variants within the United Arab Emirates (UAE) population. This investigation features a case-control matching analysis comprising 62 patients diagnosed with various cancer types and 142 unrelated healthy controls. The results showed a potential association between cancer predisposition and variants within. The results demonstrate an association between cancer predisposition and variants within C-terminal Binding Protein 2 (CTBP2), DNA Polymerase Theta (POLQ) and Tektin 4 (TEKT4). Gene set enrichment analysis showed enriched pathways associated with cancer-related biological processes such as DNA repair and depleted pathways related to translation, cellular metabolic process, and mitochondrial functions. This study highlights that the distinctive genetic composition of underrepresented groups influences the penetrance of pathogenic variants that could contribute to hereditary cancer risk in ways that diverge from patterns observed in more extensively researched cohorts. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-05296-3.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/972fb28f89280ac4ce146e45e2b00c9a74cb002a" target='_blank'>
              Pan-Cancer Exome-wide analysis of germline mutational patterns and pathways
              </a>
            </td>
          <td>
            Halima Alnaqbi, Michael Olbrich, Noora Zayed, Mira Mousa, S. Azzam, S. Tirmazy, Habiba S. Alsafar
          </td>
          <td>2025-07-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The DNA helicase HELQ is involved in homologous recombination repair, interstrand cross‐link repair, and replication stress response. Its functional defects are associated with infertility and abnormal gametogenesis. However, the specific mechanisms of HELQ in the development of germ cells remain to be elucidated. Here, we uncovered that HELQ deficiency led to proliferation defects of primordial germ cells (PGCs) in mouse embryos, thus compromising the establishment of reproductive reserve. Mechanistically, we found that HELQ interacted with the H3K9me3 demethylase KDM4B, and the absence of HELQ led to a marked increase in both total and chromatin‐bound protein levels of KDM4B, resulting in reduced H3K9me3 levels in the region of the retrotransposon LINE‐1, which triggered its high expression and subsequently caused DNA damage accumulation. Moreover, the developmental defects of HELQ‐deficient PGCs were alleviated by inhibition of retrotransposition. These results indicate that HELQ maintains the genome stability of PGCs by repressing LINE‐1 expression. Our study reveals a critical role of HELQ in PGC development and provides new insights into reproductive disorders caused by defects in DNA damage response factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6d6e1277ec183328a48e290753878371fe1fc44" target='_blank'>
              HELQ Maintains Genome Stability of Primordial Germ Cells by Inhibiting LINE‐1 Expression
              </a>
            </td>
          <td>
            Lili Cao, Jiayi Ren, Zhaojie Kong, Mengchun Hu, Yaxuan Zhang, Zi-Jiang Chen, Yingying Qin, Yajuan Yang, Shidou Zhao
          </td>
          <td>2025-06-21</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Genetic progression models of cancer continue to determine the roadmap of carcinogenesis, although the sequence of genetic events is inferred rather than empirically determined through longitudinal analyses. Here, we present a unique longitudinal study of oral leukoplakia lesions that transformed into carcinoma. Lesions were followed from initial diagnosis through to malignant transformation. For each lesion, biopsies at baseline, the carcinoma, and all available intermediate biopsies were studied for the presence of dysplasia, genomic copy number aberrations, and mutations in selected head and neck cancer genes. Using this information, the phylogenetic history of all carcinomas was reconstructed within the context of applied interventions. In total, 71 biopsies of 21 lesions were studied. The median time to malignant transformation was 60 months. Oral carcinogenesis emerged as a multi-(sub)clonal process. Treatment interventions appeared to impact clonal selection, although lesions always remained after excision. Notably, the lesions demonstrated different routes of progression. A canonical pattern of progression was observed, characterized by longitudinal accumulation of abnormal morphology and genetic changes. However, some lesions followed an alternative, stable pattern of oncogenic progression, with no apparent increase in morphological changes or accumulation of genetic aberrations. This latter pattern appeared to be associated with the development of previously identified 'copy number quiet' head and neck tumors. This study provides a novel perspective on the temporal evolution of oral leukoplakia and the different routes to cancer progression and highlights the key role of multiclonal field cancerization in lesion recurrence and cancer development. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95ac89f69d77b6dd7fcbbebb782a481d2786ada6" target='_blank'>
              New insights into the genetic progression of cancer through longitudinal analysis of oral lesions.
              </a>
            </td>
          <td>
            L. Wils, J. Poell, A. Brink, Laura A N Peferoen, I. Evren, E. Brouns, Jan G. de Visscher, Erik H van der Meij, Elisabeth Bloemena, R. Brakenhoff
          </td>
          <td>2025-07-29</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Transposons are mobile genetic elements capable of moving within the genome. Leveraging this property—particularly the cut-and-paste mechanism of DNA transposons—has enabled the development of technologies for inserting exogenous DNA fragments into host genomes. While targeted integration is a key goal for therapeutic applications, this review highlights the value of their intrinsic randomness. By combining the ability to freely design the DNA cargo with the stochastic nature of transposon integration, it becomes possible to generate highly sensitive reporter cells. These can be used to efficiently identify functional markers, uncover novel signaling pathways, and establish innovative platforms for drug screening. As more subfamilies of transposons become available for research use, their complementary biases may enhance the coverage and diversity of genome-wide screening approaches. Although inherently unpredictable, this strategy embraces randomness as a strength, and we propose that it holds great promise for driving new advances in biology, cellular engineering, and medical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4d806cc8a8f4473c66fef98ec788d4bd5b59391" target='_blank'>
              Random Insertion Reporter Gimmicks Powered by Cut-and-Paste DNA Transposons
              </a>
            </td>
          <td>
            Yamato Kasahara, Kentaro Semba, Shinya Watanabe, K. Ishikawa
          </td>
          <td>2025-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is an aggressive subtype of breast cancer with few effective targeted therapies. Taxanes and other microtubule-targeting agents (MTAs) are frontline chemotherapies for TNBC; however, the molecular pathways that cause TNBC taxane sensitivity are largely unknown, preventing selection of taxane-responsive patients and development of more selective therapeutic strategies. In this study, we identified tumor-selective vulnerabilities in TNBC harboring inactivation of the tumor suppressor PTPN12 by integrating proteogenomic characterization and synthetic lethality screening. We discovered that PTPN12 inactivation drives mitotic defects through aberrant hyperactivation of the ubiquitin ligase complex APCFZR1, a critical regulator of the cell cycle. Consistent with the mitotic stress caused by PTPN12 inactivation in TNBC cell lines, tumors harboring loss of PTPN12 exhibit heightened sensitivity to taxane chemotherapy. Collectively, this data suggests that PTPN12 inactivation may drive chromosomal instability and favorable MTA response in TNBC; two prominent features of the disease with unclear mechanistic etiology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13c548af869e91466a22ceb7887ea1b6fdb37dc0" target='_blank'>
              Integrative proteogenomics and forward genetics reveals a novel mitotic vulnerability in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Nicholas J Neill, S. Satpathy, Karsten Krug, Jitendra K Meena, Nivetha Ramesh Babu, Cheyenne Calderon, Desmon Reed, Marcus J Weber, L. Dobrolecki, Alaina Lewis, Christina Sallas, M. Anurag, Kimberly R Holloway, Chen Huang, S. Vasaikar, Maria F Cardenas, Beom-Jun Kim, Doug W. Chan, Shayan C. Avanessian, S. Tyagi, Mayra Orellana, Sufeng Mao, Heyuan Li, Fade Gong, S. Kurley, K. Meerbrey, Calla M Olson, Amritha Nair, Tingting Sun, Hsiang-Ching Chung, Elizabeth A. Bowling, Jarey H Wang, Pengju Zhang, Peng Xiao, Duxiao Yang, F. Stossi, Mei-Yin C Polley, Alexander B. Saltzman, F. Mundt, D. Mani, Michael A Gillette, S. Hilsenbeck, George Miles, Carolina Gutierrez, C. K. Osborne, Charles Y. Lin, N. Gray, Jinpeng Sun, David A. Wheeler, C. Perou, A. Malovannaya, Michael T. Lewis, Bing Zhang, Matthew J Ellis, Steven A. Carr, Thomas F Westbrook
          </td>
          <td>2025-08-11</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="Background/Objectives: Krüppel-like factor 4 (KLF4) emerged as an epigenetically regulated gene in a variety of settings, including cell reprogramming and malignant cell proliferation. The aim of the present manuscript is to explore the relationship described in recent years between circular RNAs, miRNAs, and KLF4. These have been shown to be involved in cancers having diverse histological origins, including some of the most prevalent and deadly tumors for the human population. Expression and protein levels of this transcription factor correlate with invasiveness and prognosis in a context- and tissue-specific fashion. Methods: The literature was obtained through two main PubMed queries. The first is “miRNA and KLF4 and cancer” and is limited to the last 5 years. The second is “circRNA and KLF4”, which yielded publications between 2013 and 2024. The oncological publications were selected. Results: A number of circRNA/miRNA axes that regulate the downstream transcription factor KLF4 emerged in the last few years. circRNAs act as sponges for miRNAs and synergize with KLF4, which can function as either a tumor promoter or suppressor in different tumors. Conclusions: The axes represented by circRNA/miRNA/KLF4 emerged as a new layer of epigenetic regulation. These RNA-based modulators explain the complex regulation of this transcription factor and open the way to new therapeutic targeting possibilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9a3e7b131d3c048a24a47a22d4b39bf8b794eef" target='_blank'>
              circRNA/miRNA Networks Regulate KLF4 in Tumor Development
              </a>
            </td>
          <td>
            R. Frazzi, E. Farnetti, D. Nicoli
          </td>
          <td>2025-07-29</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="High-risk human papillomaviruses (HR HPVs) are responsible for around 5% of the world's cancer burden. Activation and interaction with the host DNA damage response (DDR) promotes the HPV16 life cycle. This study demonstrates a crucial interaction between HPV16 and SMARCAL1, a protein involved in the stabilization of stalled DNA replication forks. SMARCAL1 can complex with E2, is recruited to E1-E2 replicating DNA, and SMARCAL1 knockdown reduces the fidelity of E1-E2 mediated DNA replication in C33a cells but does not alter replication levels. SMARCAL1 is recruited to the HPV16 genome in HPV16-immortalized foreskin keratinocytes (HFK+HPV16), and in situ protein interaction with nascent DNA replication forks (SIRF) assays demonstrated that SMARCAL1 is hyper-recruited to host replication forks in HFK+HPV16 cells. Using COMET, DNA fiber, and cell growth assays it was determined that knockdown of SMARCAL1 increased DNA damage and impaired replication fork progression in HFK+HPV16 cells, ultimately resulting in growth arrest. The viral genome integrates following SMARCAL1 knockdown in HFK+HPV16 cells. Therefore, SMARCAL1 facilitates host and viral DNA replication in HFK+HPV16 cells. Overall, the results demonstrate that HPV16 promotes SMARCAL1 recruitment to viral and host replication forks and is an essential factor for the HPV16 life cycle. The results expand our understanding of DDR proteins that regulate the HPV16 life cycle, and suggest that inhibition of SMARCAL1 function represents a novel anti-viral strategy for the treatment and prevention of HPV infections.


Importance
HPV16 is responsible for the majority of HPV+ cancers, contributing to 54% of cervical cancers and ∼90% of HPV+HNSCC. Integration of viral genomes into host DNA can promote cervical cancer progression and correlates with poor prognosis in HPV-associated HNSCC, where around 70% of HPV+ cancers contain episomal viral genomes. Developing effective antiviral therapies requires a deeper understanding of the interplay between viral replication and host DNA damage response (DDR) pathways. This report demonstrates that SMARCAL1 is essential for HPV16 replication and keratinocyte proliferation and that its depletion leads to replication stress, DNA damage, and viral genome integration. This work underscores the delicate balance between viral exploitation of the host DDR and the risk of genome instability. These insights contribute to the broader understanding of HPV pathogenesis and may inform the development of therapeutic strategies targeting viral replication to prevent disease progression and improve clinical outcomes in HPV-associated cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1353c24c2d18ad56030f170e19dffc05bf916315" target='_blank'>
              HPV16 recruitment of SMARCAL1 to viral and host replication forks is required for the viral life cycle.
              </a>
            </td>
          <td>
            C. James, Aya H. Youssef, Jenny D Roe, Floriana Cappiello, F. Aiello, Benedetta Perdichizzi, Rachel L. Lewis, Austin J. Witt, A. T. Prabhakar, Xu Wang, Molly L Bristol, P. Pichierri, I. Morgan
          </td>
          <td>2025-07-22</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Understanding how the assembly of nascent mRNA into a ribonucleoprotein (mRNP) influences R-loop homeostasis is crucial for gaining insight into the cellular mechanisms that prevent genome instability. Here, we identify three RNA-binding proteins, Rie1, Rim4 and She2, whose expression levels are important to limit R-loop accumulation and, thus, to prevent DNA damage. Interestingly, Rim4 and She2 are overrepresented in CBP80-containing mRNPs formed in the absence of THO. In addition, we found that an excess of the RNA exosome component Dis3 impairs its function, promoting R-loops, particularly from non-coding RNAs, which cause genomic instability. Our results indicate that changes in the availability of different RBPs or RNAs, causes R-loop-mediated DNA damage in the cell. These results may help to understand the mechanism that promotes cancer, as several RBPs are overexpressed in different types of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58254c8464c859b1341c1f452949c65b62663dc9" target='_blank'>
              Cellular imbalance of specific RNA-binding proteins associates with harmful R-loops
              </a>
            </td>
          <td>
            José Antonio Mérida-Cerro, Guillaume Chevreux, Benoît Palancade, Ana G. Rondón, A. Aguilera
          </td>
          <td>2025-07-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="ABSTRACT Background Mutational landscape is prognostic in colorectal cancer (CRC). Rat sarcoma (RAS) oncogenes, such as KRAS and NRAS, with driver mutations, portend poor survival outcomes, whereas pathologic mutations in HRAS are extremely rare, and their prognostic value remains uncertain. Methods This retrospective study analyzed the Oncology Research Information Exchange Network (ORIEN) alliance tumor RNA‐Seq data in Stages II and III CRC to investigate the association between RAS gene expression and survival outcomes. Results High transcript levels of HRAS were associated with superior overall survival (OS). The high HRAS‐associated OS benefit was most pronounced in patients with right‐sided primary expressing low KRAS transcript levels in the absence of pathologic KRAS mutations. Conclusions Contrary to the notion that RAS family genes are proto‐oncogenic, this study demonstrates that high HRAS transcript levels are associated with superior OS in Stages II and III CRC. The potential of HRAS as a prognostic biomarker should be explored further.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6201baf2d0af4b663c5c54e1af500df1e4f27566" target='_blank'>
              Proto‐Oncogene HRAS Transcript Level and Overall Survival in Stages II and III Colorectal Cancer
              </a>
            </td>
          <td>
            Donghyun Kim, Saima Sharif, Juan Antonio Raygoza Garay, Avanish S. Bhakta, P. Boland, M. Cavnar, Michelle L. Churchman, H. Hatoum, Lyen C. Huang, Joseph Kim, Richard Kim, Robert W Lentz, Sarbajit Mukherjee, Mary T. O'Donnell, Benjamin Quartey, M. Reilley, Robert J. Rounbehler, B. Salhia, Bryan P. Schneider, C. Chan
          </td>
          <td>2025-07-31</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Long noncoding RNAs (lncRNAs), defined as noncoding transcripts exceeding 200 nucleotides without protein-coding potential, have been increasingly recognized for encoding polipeptides or proteinsimplicated in the regulation of malignant tumor initiation, progression, and prognosis. Among them, small nucleolar RNA host gene 29 (SNHG29) has attracted increasing attention because of its aberrant expression across diverse malignancies. This report reviews the latest research regarding its expression patterns, regulatory functions, and involved signaling pathways. This review synthesizes current insights into the contributions of SNHG29 to oncogenic processes, including cell proliferation, migration, epithelial‒mesenchymal transition, chemoresistance, and immunosuppression. Furthermore, evidence indicates that SNHG29 functions as a competing endogenous RNA (ceRNA), modulating tumor progression via the miRNA‒mRNA axis and signaling pathways such as the ATR‒Chk1, PI3K‒AKT, and Wnt‒β-catenin pathways. Cumulative data position SNHG29 as a promising diagnostic and prognostic biomarker, as well as a potential molecular target for therapeutic intervention in various malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef7951689b147bbb46276371501a41cc5d89fba" target='_blank'>
              The role of long noncoding RNA SNHG29 in malignant tumors
              </a>
            </td>
          <td>
            Tingcheng Bu, Wendong Gu, Yingjie Shao
          </td>
          <td>2025-07-15</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f72e0c1deac040357dc9e7ae4b906709cffda79a" target='_blank'>
              Methylation dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Adriana V.A. Fonseca, Christopher Boniface, Caroline J. Watson, Samuel Hackett, C. Gabbutt, Akemi D. Ramos-Yamasaki, Amanda B. Tan, Jose Montoya, R. Strogantsev, Lars L. P. Hanssen, Yuexuan Zhang, Sophia Apostolidou, A. Gentry-Maharaj, Sadik C Esener, Paul T Spellman, Trevor Graham, U. Menon, Hisham Mohammed, J. Blundell
          </td>
          <td>2025-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad1b298ef1c59f8a47db65e3714d33449ef55095" target='_blank'>
              Genetically encoded control of in vitro transcription-translation coupled DNA replication
              </a>
            </td>
          <td>
            Sebastian Barthel, Maximilian Hoffmann-Becking, Islomjon G Karimov, Tobias J. Erb
          </td>
          <td>2025-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cervical cancer is one of the most common malignant tumors among women worldwide. Its primary etiology is closely associated with human papillomavirus infection, which poses a serious threat to the health of women. N6-methyladenosine (m6A) modifications notably affect the biological characteristics of tumor cells, such as their proliferation, metastasis and chemoresistance, by regulating the stability, translation and degradation of RNA. It also serves an important regulatory role in the pathogenesis of cervical cancer. The present review details the mechanisms underlying m6A modification in cervical cancer and analyzes its impact on tumor progression. Moreover, it explores the potential clinical applications of m6A modification as a biomarker and therapeutic target to provide new insights and evidence regarding the early diagnosis and individualized treatment of patients with cervical cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b66d595728a32d4a2ecd97272d112fd5969eeebc" target='_blank'>
              N6-methyladenosine in cervical carcinogenesis: Mechanistic insights and therapeutic perspectives (Review)
              </a>
            </td>
          <td>
            Man Xu, Fanglei Yang, Huan Chen, Feiyun Jiang
          </td>
          <td>2025-07-14</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Although recent genetic studies have identified numerous genetic alterations in diffuse large B-cell lymphoma (DLBCL), their biological relevance remains elusive. Here we performed in vivo CRISPR loss-of-function screening targeting 86 genes recurrently altered in DLBCL to examine oncogenicity of sgRNA-targeted genes, association between genotype and lineage, occurrence of second-hit alterations, and cooperability among sgRNA-targeted genes and second-hit alterations. Transplantation of the CRISPR library-transduced hematopoietic stem/progenitor cells induces various hematologic malignancies, including B-lymphomas in mice. Enrichment analysis of sgRNA-targeted genes demonstrates significant overrepresentation of Kmt2d, Pax5, and Trp53 in B-lymphomas. Whole-exome sequencing identifies recurrent second-hit driver alterations, showing significant enrichment of Trp53 alterations in sgKmt2d-targeted B-lymphomas. Importantly, KMT2D and TP53 mutations are found to be the most prevalent co-occurring combination in human DLBCL, which is more prominent in relapsed/refractory DLBCL. Moreover, this combination confers significantly worse prognosis independent of clinical factors. Transcriptomic sequencing identifies overexpression of Yap1, the Hippo pathway component, in double sgKmt2d-targeted/Trp53-altered B-lymphomas. Furthermore, chromatin accessibility analysis demonstrates enrichment of TEAD1 binding motifs in regions that gained accessibility and increased expression of their nearest genes in these B-lymphomas. Most importantly, genetic and pharmacological inhibition of YAP1 suppresses in vitro cell proliferation and in vivo tumor growth of a human KMT2D/TP53-altered DLBCL cell line and prolongs survival of mice transplanted with double sgKmt2d-targeted/Trp53-altered B-lymphoma cells. Our findings demonstrate the utility of in vivo CRISPR screening to integrate human cancer genomics with mouse modeling, and highlight the functional interplay between KMT2D and TP53 aberrations, providing insights into therapeutic strategies in DLBCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87541f86c750b412ce2794d97282b0873c8733c8" target='_blank'>
              In vivo CRISPR screening reveals cooperation of KMT2D and TP53 deficiencies in B-cell lymphomagenesis.
              </a>
            </td>
          <td>
            Kentaro Yamaguchi, J. Koya, Kota Mizuno, Yosuke Mizukami, Kota Yoshifuji, Y. Saito, Mariko Tabata, S. Shingaki, Mitsuhiro Yuasa, Yuta Ito, Kazutaka Nakashima, K. Dreval, R. D. Morin, K. Chiba, Ai Okada, Yuichi Shiraishi, Koichi Murakami, Yasunori Kogure, Koichi Ohshima, Keisuke Kataoka
          </td>
          <td>2025-07-17</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="While reproduction in Leishmania is primarily clonal, genomic analyses of natural isolates provide evidence for hybridization within and between species. Genetic exchange has been formally demonstrated via the generation of hybrids in the laboratory. Experimentally, genetic exchange in Leishmania is nonobligatory, relatively rare, and naturally confined to life cycle stages present in the sandfly midgut. Per whole genome sequencing, allele inheritance is Mendelian and involves meiosis-like recombination of the nuclear genome. Deletion of meiosis- and plasmogamy-related genes reveals their requirement for successful hybridization. Mitochondrial DNA inheritance appears uniparental for maxicircle kinetoplast DNA (kDNA) but biparental for minicircle kDNA. To account for the current absence of identified haploid gametes and for the hybridization of aneuploid genomes, alternative modes of genetic exchange have been proposed. Future studies will need to confirm the existence of gametes, explore the conditions promoting their development, and exploit the generation of sexual recombinants to map genes controlling important traits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/751571e5638d5dba2ad37af65cdd9973a9ab8f21" target='_blank'>
              Genetic Exchange in Leishmania: Understanding the Cryptic Sexual Cycle.
              </a>
            </td>
          <td>
            C. M. Catta-Preta, David Sacks
          </td>
          <td>2025-06-17</td>
          <td>Annual review of microbiology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Bladder cancer has been considered as one of the most common urinary malignancies. Growing evidence has indicated that Centromere Protein F (CENPF) is a promising molecular biomarker for many human malignant tumors. However, the role of CENPF in bladder cancer (BC) proliferation, migration, invasion, cell cycle and apoptosis thereof remain unclear. In the present study, high-throughput sequencing technology and bioinformatics analysis were conducted to identify mRNAs profiles in 10 pairs of bladder cancer tissues and adjacent noncancerous tissues. CENPF was overexpression in bladder cancer tissues, and higher in MIBC than NMIBC. We investigated the expression of CENPF in 30 other pairs bladder cancers tissues, and the results were in accordance with the sequencing results. Furthermore, Immunohistochemical staining, showed that strong intensity of CENPF in BC tissues than normal tissues. Increased staining of CENPF was detected of tumor cells in MIBC compared with NMIBC. This suggests that CENPF might be highly expressed in aggressive and invasive tumor cells. Subsequently, in vitro functional experiments also demonstrated that the siRNA interference of CENPF expression significantly weakened the proliferation, migration, invasion and apoptosis of BC cells, and the cells were arrested in the G2/S phase in Cell cycle. Moreover, functional enrichment analyses, lncRNA-miRNA-mRNA, and protein-protein interaction networks revealed that CENPF was potentially involved in carcinogenesis and evolution of bladder cancer. Taken together, these results demonstrated that CENPF may serve as a potential biomarker of tumor occurrence, progression, and even prognosis for bladder cancer. However, further research is needed to be further clarified the pathway mechanisms of CENPF in bladder cancer. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10677-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcea81a86745e041752c09be80ee21f9ca4c7d66" target='_blank'>
              CENPF overexpression in bladder cancer cells enhances proliferation, migration, invasion, and apoptosis
              </a>
            </td>
          <td>
            Ruixiang Song, Xin Chen, Zhensheng Zhang, Huiqing Wang, Jinshan Xu, Shuxiong Zeng, Wentao Zhang, Zhaoyang Sheng, Xudong Yao
          </td>
          <td>2025-07-14</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Small Ubiquitin-like Modifier 1 Pseudogene 3 (SUMO1P3) is a novel long non-coding RNA (lncRNA) located at the 1q23.2 locus of the human chromosome. Recent evidence indicates that SUMO1P3 is aberrantly upregulated in nine types of human cancer and functions as an oncogene. Elevated SUMO1P3 expression is strongly associated with unfavorable clinicopathological features and poor prognosis in eight cancer types. Mechanistically, SUMO1P3 functions as a miRNA sponge, an epigenetic regulator, and directly interacting with proteins. It activates key signaling pathways, such as the Wnt/β-catenin and AKT pathways, and regulates Epithelial- Mesenchymal Transition (EMT), which facilitates cancer progression and therapy resistance. Due to its diverse functional roles, SUMO1P3 emerges as a promising diagnostic and prognostic biomarker, as well as a potential therapeutic target in precision oncology. This review provides a comprehensive summary of current research on SUMO1P3, highlighting its regulatory mechanisms, biological functions, and clinical significance in cancer biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d4f3e3fbd6a4fd472988ea5e4c6612a35b0592f" target='_blank'>
              The Vital Role of Long Non-Coding RNA SUMO1P3 in the Regulation of Human Cancer: Current Perspectives and Future Challenges.
              </a>
            </td>
          <td>
            Jingjie Yang, Yuzhang Wei, Chengran Gao, Zihang Wang, Yulong Liu, Haodong He, Hao Zhou, Guihua Liao, Gang Zhou, Chengfu Yuan
          </td>
          <td>2025-07-29</td>
          <td>Current topics in medicinal chemistry</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Circular RNA (circRNA), a class of evolutionarily conserved, structurally stable, and tissue-specifically expressed noncoding RNA, is increasingly recognized as a key regulator of diverse biological processes and disease pathogenesis, including cancer. While the roles of circRNAs in tumorigenesis are well-documented, their involvement in the tumorigenesis induced by environmental chemical carcinogens (ECCs) remains relatively unexplored. Recent studies have identified aberrant expressions of specific circRNAs during ECC exposure-related carcinogenesis, suggesting their critical regulatory functions. Given their unique structure and broad regulatory roles, circRNAs exhibit great potential as diagnostic, therapeutic, and prognostic biomarkers for ECC exposure-associated cancers. This review summarizes the characteristics and functions of circRNAs, as well as the potential regulatory mechanisms in ECC exposure-induced cancer and the dysregulations of circRNAs caused by ECCs. We highlight the complexity and heterogeneity of circRNA regulatory networks, emphasizing the need for integrated and dynamic investigations to fully elucidate the underlying mechanisms. Future research efforts should prioritize biomarker studies to facilitate the prevention, early detection, and effective treatment of ECC exposure-associated cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/760c4eaf87c1e364f138dd2080aea7e51f03d039" target='_blank'>
              Circular RNAs: A Potential Regulator in Environmental Chemical Carcinogenesis.
              </a>
            </td>
          <td>
            Huijun Huang, Yiqi Zhou, Yiyan Huang, Jiaxin Wang, Shiyi Ouyang, Meiqi Lan, Lieyang Fan, Yun Zhou
          </td>
          <td>2025-07-17</td>
          <td>Chemical research in toxicology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3115319084e3008e2ffcee186c2141b84b2e97e1" target='_blank'>
              Assembly and delivery of large DNA via chromosome elimination in yeast.
              </a>
            </td>
          <td>
            Yuan Ma, Yao Xiong, Jiayu Xu, Hui Xu, Zong-Heng Fu, Guang-Rong Zhao, Yi Wu, Yingjin Yuan
          </td>
          <td>2025-07-10</td>
          <td>Nature protocols</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Recent studies have revealed that polyclonality—where multiple distinct subclones cooperate during early tumor development—is a critical feature of tumor evolution, as demonstrated by Sadien et al. and Lu et al. in Nature (October 2024). These findings show that early polyclonal interactions can overcome fitness barriers, ultimately transitioning to monoclonality as dominant clones emerge. Understanding and targeting these interclonal dynamics offers new therapeutic opportunities. In this perspective, we outline how computational modeling and artificial intelligence (AI) tools can provide deeper insights into tumor polyclonality and identify actionable therapeutic strategies. By applying ligand–receptor interaction analysis, clonal trajectory reconstruction, network and pathway modeling, and spatial analysis, researchers can prioritize communication hubs, evolutionary bottlenecks, and microenvironmental niches that sustain tumor progression. These approaches, when integrated with experimental validation, offer a translational pathway from foundational discoveries to personalized cancer treatments aimed at disrupting cooperative subclonal ecosystems and preventing malignant progression. We commend the recent Nature publications, “Polyclonality overcomes fitness barriers in Apc-driven tumorigenesis” by Sadien et al. [1] and “Polyclonal-to-monoclonal transition in colorectal precancerous evolution” by Lu et al. [2], both featured on 2024 October 30. These groundbreaking studies employed distinct lineage tracing methods to investigate the origins and evolutionary dynamics of colorectal and intestinal tumorigenesis. Despite their different approaches, both studies reached convergent conclusions: Polyclonality plays a pivotal role in the early stages of tumor development, providing critical insights into how diverse cellular populations collaborate to overcome fitness barriers and drive tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca6580f8163812203e6792c0b0ae89c7c42827a3" target='_blank'>
              Artificial Intelligence-Powered Insights into Polyclonality and Tumor Evolution
              </a>
            </td>
          <td>
            Hong Zhao, T. Ideker, Stephen T C Wong
          </td>
          <td>2025-06-13</td>
          <td>Research</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="Introduction Retrotransposons (REs) constitute nearly half of the genome and include long terminal repeat (LTR) elements, Long INterspersed Elements (LINE), and Short INterspersed Elements (SINE). REs are typically silenced in somatic tissues via DNA methylation but can be reactivated through DNA hypomethylation, potentially impacting gene regulation. Here, we investigate genome-scale profiles of RE methylation in high-grade serous ovarian carcinoma (HGSOC) and associations with survival among Black women. Methods Methylation levels of LTR, LINE-1, and Alu (type of SINE) in 200 HGSOC tumors were predicted using a random forest approach and clustered using multiple consensus algorithms. Associations between RE methylation clusters and survival were evaluated using Cox proportional hazard regression, adjusting for age, stage, and debulking status. We performed sensitivity analyses restricted to women with late-stage disease and with adjustment for BRCA1/BRCA2 mutations. Results Two RE methylation clusters were identified. Cluster 1 exhibited a more hypomethylated RE profile (“Active”), while Cluster 2 was more hypermethylated (“Repressed”). No statistically significant differences in patient or clinical characteristics were observed between clusters. Compared to the Active Cluster, the Repressed Cluster was associated with an increased risk of mortality (HR = 2.41; 95% CI 1.04–5.59) and had a lower proportion of T cells. This association was consistent in sensitivity analyses. Conclusion A more hypermethylated RE profile was linked to worse survival among Black women with HGSOC, highlighting the potential of RE methylation as a prognostic biomarker. Further research is needed to understand the underlying biological mechanisms and their implications in ovarian cancer biology and treatment. Supplementary Information The online version contains supplementary material available at 10.1186/s13148-025-01942-9.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bc642af0cdc56a04adbb30045ba5a19d537e41" target='_blank'>
              Retrotransposon methylation profiles and survival in Black women with high-grade serous ovarian carcinoma
              </a>
            </td>
          <td>
            Christelle M. Colin-Leitzinger, Katherine A Lawson-Michod, Courtney E Johnson, Irma M Vlasac, Sean Yoder, Tania Mesa, Dana Roeber, Chad Huff, M. Hildebrandt, Kristen Haller, Anthony J Alberg, E. Bandera, Melissa Bondy, Michele L Cote, Theresa A Hastert, Edward S Peters, Paul D Terry, Andrew B. Lawson, Andy Berchuck, Brooke L Fridley, J. Chern, J. Doherty, Jeffrey R Marks, J. Schildkraut, Brock C. Christensen, Lucas A Salas, Lauren C. Peres
          </td>
          <td>2025-07-30</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Brain tumors are one of the most severe types of malignant tumors and glioma accounts for ~80% of malignant brain tumors. The current treatment methods for glioma are limited and patients with glioma often experience relapse following treatment, which leads to a poor prognosis for these patients. Therefore, novel therapeutic targets and methods urgently need to be explored. The present review screened studies that mainly focused on the epigenetic regulation of small guanosine triphosphate (GTP)ase in glioma. These small GTPases participate in most cellular biological processes, including differentiation, proliferation, cell migration, apoptosis, vesicle and organelle dynamics and transport, nuclear dynamics and cytoskeleton regulation. Due to the diversity and importance of the biological functions of small GTPases, an increasing number of studies have focused on them; however, the incidence of changes in the gene structure of small GTPases is considered to be low in glioma. Several studies have shown that the abnormal expression of genes encoding small GTPases is often influenced by epigenetic regulation in glioma. Epigenetic regulation is a dynamic and reversible process, which implies that the reversal of abnormal epigenetic modifications is a potential treatment strategy for glioma. These previous studies, which are summarized in the present review, not only provide new therapeutic targets and prognostic markers, but also provide information regarding the treatment of glioma. The current review may provide valuable insights for future research and promote the clinical translation of relevant research results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44b0933c82de7174662d27dd036b6327ca1cf162" target='_blank'>
              The role of abnormal epigenetic regulation of small GTPases in glioma (Review)
              </a>
            </td>
          <td>
            Mingyang Zhang, Yimin Huang, Qiang Zhang, Xiaoyan Zhang, Lumei Kang, Jianguo Wang
          </td>
          <td>2025-07-02</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chemical inducers of proximity (CIPs) can elicit durable-and often neomorphic-biological effects through the formation of a ternary complex, even at low equilibrium occupancy of their targets. This "event-driven" pharmacology is exemplified by CIPs that promote targeted protein degradation, but other applications remain underexplored. We developed a generalizable strategy to discover event-driven CIPs by tracking the cellular effects of heterobifunctional small molecules alongside quantitative measures of intracellular target engagement. Using this approach, we discovered PCIP-1, which inhibits DNA repair by recruiting BET proteins to PARP2. Unlike conventional PARP inhibitors, PCIP-1 activity is observed at low equilibrium occupancy of PARP1/2 and without inhibition of PARP-catalyzed PARylation, yet it retains synthetic lethality in cancer cells with homologous recombination deficiencies. PARP1 knockout, which confers resistance to conventional PARP drugs, increases sensitivity to PCIP-1, offering a potential new mechanism to overcome clinical resistance. Through these studies, we demonstrate that DNA repair can be rewired by CIPs and introduce a new form of event-driven pharmacology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d635136f6062b087a4cb1e8318c15fddd1b00672" target='_blank'>
              Rewiring DNA repair with PARP-based chemical inducers of proximity.
              </a>
            </td>
          <td>
            Bryce da Camara, Eric M. Bilotta, Erin F Broderick, Ashwini Premashankar, Paige A. Barta, Trever R. Carter, Lauren M. Hargis, D. Durocher, Michael A. Erb
          </td>
          <td>2025-07-31</td>
          <td>bioRxiv : the preprint server for biology</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is the third most prevalent malignancy and the second leading cause of cancer-related mortality worldwide. The pathogenesis of CRC primarily stems from the gradual accumulation of genetic mutations, which drive oncogene (e.g., KRAS) activation and tumor suppressor gene (e.g., TP53) inactivation. Loss of genetic stability facilitates the conversion of proto-oncogenes into active oncogenes and the functional impairment of tumor suppressors, collectively propelling CRC progression. The tumor suppressor protein p53, a transcription factor, induces cell cycle arrest, apoptosis, and DNA damage repair under cellular stress, and prevents cancer development by regulating various cellular responses. However, in CRC pathogenesis, TP53 mutations (detected in ~ 74% of cases) subvert these protective mechanisms through dual mechanisms: (i) dominant-negative suppression of wild-type p53 (wt-p53) function, and (ii) acquisition of neomorphic pro-tumorigenic activities, termed gain-of-function (GOF) mutations. New evidence from laboratory and clinical trials shows that some new therapeutic strategies have the potential to treat CRC by reactivating and restoring p53 function, depleting p53 mutants, or targeting p53 with immunotherapy. In this review, we summarize the function of p53 and characterize its mutation in CRC, emphasizing the influence of p53 mutation in the pathogenesis of CRC. In addition, we also describe the current therapeutic strategies for targeting p53 mutants in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/366c1fd84b915fa1e7d9335c6be5c8ce5ddaee30" target='_blank'>
              p53 in colorectal cancer: from a master player to a privileged therapy target
              </a>
            </td>
          <td>
            Sicheng Yan, Fuyuan Zhan, Yuxuan He, Yuehong Zhu, Zhihong Ma
          </td>
          <td>2025-06-19</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Multiple myeloma (MM) is closely associated with abnormal DNA repair and genome instability. The bone marrow microenvironment, particularly myeloma associated macrophages (MΦs) is critical to the progression of MM. However, there is limited understanding on the role of MΦs in DNA repair in MM. Here, we found that MΦs regulated DNA repair in MM cells by the CXCL5/8-CXCR2 axis. By promoting non-homologous end joining rather than homology-directed repair, MΦs increased the probability of chromosomal translocations in MM cells. Furthermore, clinical data confirmed that MΦs are closely associated to the increased genetic variations of MM patients' primary cells. The study elucidates a mechanism by which MΦs regulates DNA repair in MM in the microenvironment and provides a potentially new target to counter MM progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7463e4f2bfa77ad622fca89d072c6a09f3dba6b9" target='_blank'>
              Macrophages promote aberrant DNA repair in multiple myeloma via the CXCL5/8-CXCR2 axis.
              </a>
            </td>
          <td>
            Mengmeng Dong, D. He, Jinna Zhang, Haimeng Yan, Haoguang Chen, E. Zhang, Yili Feng, Jingsong He, Xi Huang, Guoqiao Chen, Xiuna Sun, F. Cheng, Huiyao Gu, Huanping Wang, Anyong Xie, Zhen Cai, Cai Lab
          </td>
          <td>2025-06-26</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Previously, we identified that mutant p53 expression in cancer cells promotes engulfment of neighbouring cancer cells to form cell-in-cell (CIC) structures. This process gave mutant p53 cells an advantage in tumour formation in mouse xenograft experiments. TP53 can be found mutated at nearly every amino acid in cancers and mutant p53 expression is associated with various GOF (Gain-of-function) processes, including cancer cell invasion, metastasis, stemness and drug resistance. In the current manuscript, we identified SH3BGRL (Src homology 3 binding glutamate rich protein like) as a mutant p53-regulated gene and investigated to what extent SH3BGRL expression and cell engulfment are responsible for mutant p53-dependent anchorage-independent growth and chemoresistance. We demonstrate that mutant p53 expression drives cell engulfment in which the mutant p53 host cell moves in the direction of the target internal cell to form CIC structures. This is therefore more reminiscent of cell engulfment rather than cell entosis, in which cells invade into host cells. Using NGS (Next Generation Sequencing), we identified novel target genes of mutant p53 and demonstrate that cell engulfment requires SH3BGRL expression. We generated mutant p53 and p53 KO cell lines that stably overexpressed SH3BGRL and determined that SH3BGRL promotes etoposide resistance in mutant p53 cells and anchorage-independent growth independent of mutant p53 expression. Through FACS sorting of pure cell engulfing (CIC) populations, we could also show that engulfing cells have an enhanced etoposide resistance. These data suggest that SH3BGRL and cell engulfment are required for certain GOFs of mutant p53.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba6ff758bccb84d82f6118096ce3f5daf639f6e" target='_blank'>
              Mutant p53 induces SH3BGRL expression to promote cell engulfment
              </a>
            </td>
          <td>
            Lobsang Dolma, Mary I Patterson, Antonia Banyard, C. Hall, Steven J Bell, Wolfgang Breitwieser, Sudhakar Sahoo, John Weightman, Maria Pazos Gil, Garry Ashton, C. Behan, Nicola Fullard, Lewis D Williams, P. Muller
          </td>
          <td>2025-07-01</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="X chromosome inactivation (XCI) is crucial for balancing X-linked gene dosage in female cells by randomly silencing one X chromosome during early embryogenesis. However, accurately classifying cells based on the parental origin of the inactivated X chromosome in single-cell samples remains challenging. Here we present FemXpress, a computational tool leveraging X-linked single nucleotide polymorphisms (SNPs) to group cells based on the origin of the inactivated X chromosome in female single-cell RNA sequencing (scRNA-Seq) data. FemXpress performs robustly on both simulated and real datasets, without requiring parental genomic information, and can also identify genes that escape XCI. Applying FemXpress to single-cell RNA-Seq data from multiple tissues of a cynomolgus monkey, we reveal heterogeneity in XCI origin across organs and cell types. In each organ, we identify candidate XCI-escaping genes, and within each cell type, we observe gene expression differences associated with XCI origin, potentially contributing to phenotypic variability. Furthermore, FemXpress demonstrated strong performance in phasing XCI in scRNA-Seq datasets from embryos and colon tumors. In summary, FemXpress provides a powerful approach for XCI status analysis, offering new insights into XCI dynamics at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/478a669d81b3cf05029932ee293479ee1f9bcebc" target='_blank'>
              FemXpress: Systematic Analysis of X Chromosome Inactivation Heterogeneity in Female Single-Cell RNA-Seq Samples.
              </a>
            </td>
          <td>
            Xin Wang, Yingke Ma, Fan Li, Wentao Cui, Tianshi Pan, Siqi Wang, Sinan Ma, Qingtong Shan, Chao Liu, Yukai Wang, Ying Zhang, Yuanchun Zhou, Wei Li, Pengfei Wang, Qi Zhou, Guihai Feng
          </td>
          <td>2025-06-29</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Eukaryotic Argonautes (eAgos) have traditionally been characterized by their ability to utilize RNA guides to identify RNA targets, thereby engaging in post-transcriptional gene silencing pathways. While some eAgos have been demonstrated to use DNA guides for RNA cleavage, the ability of eAgos to cleave DNA targets remains unclear. In this study, we characterized CsAgo, an eAgo protein derived from thermophilic eukaryote Chaetomiumsp. MPI-CAGE-AT-0009, demonstrating a novel ability to cleave both DNA and RNA targets in vitro. Guided by short single-stranded DNA (ssDNA) or RNA, CsAgo exhibits robust RNA cleavage activity at 20–90°C in vitro. CsAgo can effectively cleave ssDNA guided by RNA guides at 20–50°C in vitro. Notably, CsAgo can utilize DNA guides to effectively cleave ssDNA, plasmid double-stranded DNA (dsDNA), and linear dsDNA at ≥80°C in vitro. Based on its ability to cleave dsDNA at high temperatures, CsAgo demonstrates versatility and efficacy in simplifying routine cloning workflows. Additionally, we have developed a CsAgo-based nucleic acid detection method based on a Pyrococcus furiosus Ago-mediated nucleic acid detection method, which exhibits a high sensitivity of six copies/reaction. These results suggest that eAgos include that, in theory, can be utilized for potential DNA-targeting applications. This not only enhances our understanding of eAgos but also expands the toolkit for DNA manipulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b83907ec94e09a230d08b65521fe7a733061ab1d" target='_blank'>
              In vitro programmable DNA cleavage by a eukaryotic Argonaute
              </a>
            </td>
          <td>
            Guangbo Yan, Xia Li, Longyu Wang, Fei Wang, Siyuan Lv, Xiaolan Yu, Zhongchen Li, Jiakai Cui, Baotong Sun, Jia Hou, Wenqiang Li, Xiaochen Xie, Shuai Zhao, Zhiwei Zhang, Wanping Chen, Yanhong Peng, Yang Liu, Lixin Ma
          </td>
          <td>2025-06-20</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background SUGT1 (Suppressor of the G2 allele of SKP1) and FH (fumarate hydratase) have recently garnered significant attention from the research community. SUGT1 functions as a molecular chaperone, regulating the stability and activity of various proteins, while FH is a key enzyme in the tricarboxylic acid cycle, catalyzing the reversible conversion of fumarate to malate. Existing literature has established their essential roles in signaling, tumorigenesis, and cancer progression. However, their functions and mechanisms in ovarian cancer (OC) remain poorly understood. Results We found that high SUGT1 expression is associated with a more advanced FIGO stage in OC. SUGT1 knockdown significantly inhibits OC cell proliferation and metastasis, while its overexpression has the opposite oncogenic effect. Mechanistically, we revealed that SUGT1 promotes FH protein degradation via the ubiquitin-proteasome pathway. Moreover, FH knockdown partly reversed the inhibitory effects of SUGT1 knockdown on tumor cell proliferation, migration, and proteins of phosphorylated PI3K/AKT and Vimentin. In summary, We demonstrated that SUGT1 exerts oncogenic functions in OC by regulating FH stability. Conclusions Our study is the first to provide experimental evidence elucidating the SUGT1-FH relation and its role in OC progression, offering potential significance for clinical diagnosis and therapy. Supplementary Information The online version contains supplementary material available at 10.1186/s13048-025-01744-w.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/467d1c72a28cbccbc2152c3f132424b0a5db294b" target='_blank'>
              Mechanism by which SUGT1 downregulates FH to promote proliferation and migration in serous ovarian cancer
              </a>
            </td>
          <td>
            Tianli Mu, Bo Ren, Ziteng Kuang, Runze He, Bingjie Rui, Ye Yang, Yuxi Liu, Danbo Geng, Yuci Zhang, Min Wang
          </td>
          <td>2025-07-29</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Chromosome segregation is essential for cellular proliferation. Unlike eukaryotes, bacteria lack cytoskeleton-based machinery to segregate their chromosomal DNA (nucleoid). The bacterial ParABS system segregates the duplicated chromosomal regions near the origin of replication. However, this function does not explain how bacterial cells partition the rest (bulk) of the chromosomal material. Furthermore, some bacteria, including Escherichia coli, lack a ParABS system. Yet, E. coli faithfully segregates nucleoids across various growth rates. Here, we provide theoretical and experimental evidence that polysome production during chromosomal gene expression helps compact, split, segregate, and position nucleoids in E. coli through nonequilibrium dynamics that depend on polysome synthesis, degradation (through mRNA decay), and exclusion from the DNA meshwork. These dynamics inherently couple chromosome segregation to biomass growth across nutritional conditions. Halting chromosomal gene expression and thus polysome production immediately stops sister nucleoid migration, while ensuing polysome depletion gradually reverses nucleoid segregation. Redirecting gene expression away from the chromosome and toward plasmids causes ectopic polysome accumulations that are sufficient to drive aberrant nucleoid dynamics. Cell width enlargement experiments suggest that limiting the exchange of polysomes across DNA-free regions ensures nucleoid segregation along the cell length. Our findings suggest a self-organizing mechanism for coupling nucleoid compaction and segregation to cell growth without the apparent requirement of regulatory molecules.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b38632c47e3d185d80d1fdc81e020f23db296ddb" target='_blank'>
              Nonequilibrium polysome dynamics promote chromosome segregation and its coupling to cell growth in Escherichia coli
              </a>
            </td>
          <td>
            Alexandros Papagiannakis, Q. Yu, Sander K. Govers, Wei-Hsiang Lin, N. Wingreen, C. Jacobs-Wagner
          </td>
          <td>2025-06-24</td>
          <td>eLife</td>
          <td>0</td>
          <td>79</td>
        </tr>

        <tr id="High-grade serous ovarian cancer (HGSC) accounts for more than 200,000 deaths each year. Despite recent advances in treating HGSC with neoadjuvant chemotherapy, the majority of patients ultimately develop chemotherapy resistance. HGSC is characterized by TP53 mutations and widespread copy number alterations and occurs frequently in the setting of deleterious germline BRCA1/2 variations, but many cases lack putative driver mutations. Here, we performed whole-exome, whole-genome, and whole transcriptome sequencing along with mass spectrometry to characterize the molecular landscape of HGSC in 22 paired samples obtained before and after neoadjuvant chemotherapy. Responsiveness to chemotherapy was determined for each patient. Evidence at the DNA, RNA, and protein level revealed numerous defects in cell-cell and cell-matrix interactions as well as disruption of cell polarity and cytoskeletal regulation in HGSC, indicating that defects in epithelial integrity were present in the majority of HGSC patients. Non-responsive HGSC harbored subclones with putative survival mutations. Additionally, ineffective nonsense mediated decay resulted in the persistence of transcripts with frameshift mutations that were translated into aberrant proteins detectable in HGSC samples. Together, these findings suggest that HGSC may arise through defects in maintenance of epithelial integrity that lead to shedding of malignant cells throughout the peritoneum, and the presence of resistant subclones prior to chemotherapy may decrease the chemosensitivity of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/896a702c0b394e4d17c91e5aa33566545b16c4dc" target='_blank'>
              Multi-omic Characterization of Pre- and Post-neoadjuvant Chemotherapy Treated Ovarian Cancer Reveals Mediators of Tumorigenesis and Chemotherapy Response.
              </a>
            </td>
          <td>
            Mark C Valentine, Mary M Mullen, E. Kotnik, Matthew A Powell, Premal H Thaker, C. McCourt, L. Kuroki, Andrea R Hagemann, Rebecca C Arend, A. Holehouse, R. Mitra, D. Mutch, Katherine C Fuh
          </td>
          <td>2025-07-22</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Hepatoblastoma (HB) is the most common pediatric liver malignancy. However, its cellular origin and molecular drivers remain poorly defined. Using single-nuclear RNA sequencing (snRNA-seq), we identified a proliferative, hepatocyte-derived tumor cell population (cycling HepT) enriched for Enhancer of Zeste Homolog 2 (EZH2) expression, particularly in the aggressive embryonal subtype. Integrative genomic and transcriptomic profiling confirmed EZH2 overexpression. Disruption of the PRC2 complex was evident through mislocalization and reduced expression of SUZ12, a core component. EZH2 overexpression correlated with upregulation of mitotic regulators such as AURKB and Ki67 in human HB gene expression analysis as compared to background liver. Targeted sequencing identified variants of uncertain significance in EZH2 and SUZ12 in 11 of 11 patient tumors. Pharmacologic inhibition of EZH2 with EPZ-6438 reduced proliferation and sensitized HB cells to cisplatin through gene regulation, potentially modulating platinum accumulation both in vitro and in vivo. In summary, EZH2 promotes HB progression through epigenetic silencing and noncanonical signaling pathways. These findings support EZH2’s contribution to HB pathogenesis, therefore identifying it as a novel therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b140db08052390107c208976008407b428e0ef2" target='_blank'>
              Oncogenic Role of Aberrant EZH2 in Hepatoblastoma
              </a>
            </td>
          <td>
            Kathryn Glaser, Lara Berklite, Brian T Hickner, Priyanka Rao, R. Patel, Andrew A. Badachhape, Somak Roy, Emily J Schepers, Nikolai Timchenko, James I Geller, S. Ranganathan, Greg Tiao, Bruce J Aronow, Takanori Takebe, Sarah E. Woodfield, Sanjeev A. Vasudevan, Erica AK DePasquale, Alex J Bondoc
          </td>
          <td>2025-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d0b84286795af8fada727049222f23cfa6cf9a8" target='_blank'>
              The evolution of high-order genome architecture revealed from 1,000 species
              </a>
            </td>
          <td>
            Yizhuo Che, Stephen J. Bush, Hui Lin, Xiaofei Yang, Qi Xie, Yuchun Liu, Deyu Meng, Kai Ye
          </td>
          <td>2025-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Circular RNAs (circRNAs) are a class of covalently closed non-coding RNAs that regulate the progression of multiple cancers. Recent studies have revealed the presence of several post-transcriptional modifications such as N6-methyladenosine (m6A), N1-methyladenosine (m1A), 5-methylcytosine (m5C), N4-acetylcytidine (ac4C), and N7-methylguanosine (m7G) on circRNAs. This review synthesizes the results of articles retrieved through systematic searches of PubMed and Web of science databases, mainly focusing on circRNA modifications in cancers. These modifications affect the biogenesis, metabolism, stability, and function of circRNAs in cancers, highlighting the critical roles of circRNAs modifications in cancers. Moreover, circRNAs crosstalk with epigenetic modifications of mRNA across various cancers, offering new perspectives for cancer therapy. Innovations in detection techniques, such as anti-modifying antibodies and mass spectrometry, have improved the identification and sensitivity of epigenetically modified circRNAs. Emerging technologies including artificial intelligence (AI)-based bioinformatics algorithms will accelerate the development of RNA epigenetic modifications-based precision therapies. Herein, we summarize a range of epigenetically modified circRNAs and their future research directions. We hope to develop clinical protocols that targeting circRNAs modification for the treatment of refractory malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edebcdf6176f17b4538381fc1ab3dca2e4573291" target='_blank'>
              Dynamic modifications of circular RNAs drive oncogenesis.
              </a>
            </td>
          <td>
            Jiaojiao Dai, Zhe Wang, Xiaoxun Cheng, Zhengze Hu, Jinghan Hua
          </td>
          <td>2025-07-15</td>
          <td>Epigenomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The mechanisms of cancer are discussed by analyzing the characteristics of the functional state and biological behavior of the abnormal nuclear cells. The abnormal nuclear cells with abnormal nuclear structure and function are a kind of sick cell or functional defect cells having existed in human body for a long time. The abnormal nuclear cells are resulted from the nuclear damage caused by the radiation, viruses and various carcinogenic compounds. Some of genes in human body are expressed, some are not expressed for life. The expressional genes are functional genes, the genes never expressed for life in human body are dormant genes or sealed genes. The nuclear damages destroy cell state of differentiation, affect gene expressional regulation and change gene expressional profiling, resulting in loss of expression of the functional genes and reactivation of the sealed genes; which finally leads to cancer, aging and other chronic refractory diseases. The cancer is not resulted from the genetic mutations or chromosomal aberrations, but rather the reactivation of genes involved in proliferation due to the nuclear damage. The biological characteristics of the cancer cells, such as the shedding and metastasis, immune tolerance, uncontrolled, loss of contact inhibition function and so on, all originate from the nuclear aberrant cells. The nuclear damage can trigger the genes that drive mitosis, leading to cancer. Thus, re-sealing the several genes that trigger the proliferation may completely prevent or cure cancers.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd09554e196fbeabd1e2fd9b39e2eabe92efdf78" target='_blank'>
              Cancers Are Derived from the Disruption of Cell Differentiation
              </a>
            </td>
          <td>
            Maojin Li
          </td>
          <td>2025-06-23</td>
          <td>Cancer Research Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Cell-free DNA (cfDNA) is a promising biomarker and carries information about physiological and pathological states. Nucleosome and transcription factor footprints have been extensively studied based on double-stranded cfDNA sequencing; however, protein-DNA interaction footprints in single-stranded cfDNA fragments remain undercharacterized.


METHODS
We used a single-stranded library preparation and sequencing method to investigate a new characteristic of cfDNA-strand specificity, based on the depth differences between the 2 DNA strands. The plasma cfDNA of 34 healthy individuals was analyzed to evaluate strand-specific bias on chromosomes and the mitochondrial genome. Then, the cfDNA samples were compared between 17 males and 17 females, as well as between 14 lung cancer patients and 18 noncancer controls.


RESULTS
Short cfDNA (<160 bases) had higher strand-specific bias than long fragments. CfDNA strand-specific-biased regions were more enriched in chromosomes with higher gene densities and enriched in promoters, exons, 5' untranslated region (UTR), 3' UTR, and CpG islands. Genomic regions with higher CpG densities and lower methylation levels were more likely to generate strand-specific bias. For the mitochondrial DNA (mtDNA), the cfDNA strand-specific bias was pronounced in the noncoding region, exhibiting the footprints of mtDNA replication and transcription. Significant molecular sex differences could be revealed on cfDNA at the gene level, and distinguished features of lung cancer patients were also discovered.


CONCLUSIONS
As a proof-of-concept, we demonstrated the existence of a cfDNA strand-specific characteristic, which reflects footprint signatures including transcription and methylation. This study provides insights into the biological properties of cfDNA and a potential novel role as biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18fe1ad62eb2836556b498c0ff8038957069e336" target='_blank'>
              Cell-Free DNA Comprises the Strand-Specific Characteristic Associated with Transcription and Methylation.
              </a>
            </td>
          <td>
            Changlin Yang, Rijing Ou, Guodan Zeng, Wei Chen, Yeqin Wang, Xinxin Wang, Lingguo Li, Jinjin Xu, Wenwen Zhou, Haiqiang Zhang, Jianhua Yin, Xin Jin, Yan Zhang
          </td>
          <td>2025-06-16</td>
          <td>Clinical chemistry</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Based on the current markers, numerous targeted therapies have been put forward for clinical application; however, treatment resistance and recurrence still remain the main causes of breast cancer-related mortality. In addition, breast cancer exhibits significant heterogeneity, and patients with apparently similar tumor subtypes exhibit variable responses to identical drug treatments. Therefore, accurate prediction of breast cancer progression and personalized treatment plans will maximize patient benefit by avoiding overtreatment and undertreatment. Methods In this study, our emphasis was placed on exploring the function of DnaJ heat shock protein family member C24 (DNAJC24, also known as DPH4) in breast cancer through bioinformatic analysis by using public databases and clinicopathological samples. We performed in vitro functional assays to explore the biological roles of DNAJC24. Results Bioinformatics analysis of TCGA data demonstrated significantly lower DNAJC24 expression in breast cancer tissues compared to adjacent paracancer tissues (p < 0.001). Immunohistochemistry showed low DNAJC24 expression in 72.4% (210/290) of breast cancer tissues versus 50.8% (134/264) of adjacent paracancer tissues. DNAJC24 expression decreased progressively with advanced clinical stages (p < 0.05), was lowest in HER2-enriched subtype, and highest in Luminal-A subtype. Overexpression of DNAJC24 in breast cancer cells significantly reduced colony formation (p < 0.05), proliferation rates, chemotaxis (p < 0.05), invasion (p < 0.05), and migration abilities (p < 0.05) compared to controls. Conversely, DNAJC24 knockdown in cancer cells produced opposite effects, significantly enhancing chemotaxis, invasion, and migration (all p < 0.05). Conclusions Lower DNAJC24 expression is strongly associated with breast cancer malignancy, advanced clinical stages, aggressive molecular subtypes, and poorer prognosis. Functionally, DNAJC24 inhibits proliferation, invasion, and migration of breast cancer cells, underscoring its potential as a prognostic biomarker in breast cancer. Supplementary Information The online version contains supplementary material available at 10.1186/s12935-025-03918-4.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ae13b122cfebf6782756284de29558bce6e8ff6" target='_blank'>
              DNAJC24 suppresses breast cancer malignancy and serves as a prognostic biomarker
              </a>
            </td>
          <td>
            W. Meng, Wei Liu, Yulong Yang, Xiao-rui Wang, Linlin Zhan, Yi Luo, Liwei Chen, Yu Wang, Guangtao Li, Yehui Shi, Yuchao He, Zhongsheng Tong, Hua Guo
          </td>
          <td>2025-07-25</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Ovarian cancer ranks as the eighth most common malignancy in women globally. Early detection remains a critical challenge for improving ovarian cancer diagnosis and treatment outcomes. Our prior study demonstrated that ARHGAP40 downregulation in basal cell carcinoma is attributed to CpG island hypermethylation. However, the expression and methylation status of ARHGAP40 in ovarian cancer remain unexplored. Here, we investigated ARHGAP40 protein expression in normal fallopian tubes, ovarian benign tumors, borderline tumors, low-grade serous carcinoma (LGSC) and high-grade serous carcinomas (HGSC). Methylation analysis of the ARHGAP40 was performed using bisulfite sequencing PCR (BSP). MethyLight assays were developed to detect methylated circulating tumor DNA (ctDNA) fragments of ARHGAP40. IHC results revealed absent or weak ARHGAP40 protein expression in 93.8% (30/32) of HGSC, 11.1% (2/18) of borderline tumors, and in 14.3% (1/7) of LGSC cases. ARHGAP40 protein expression was robust expressed in all normal fallopian tubes (15/15) and benign ovarian tumors (8/8). CpG island hypermethylation in the ARHGAP40 promoter showed a strong inverse correlation with protein expression (P < 0.001). Furthermore, methylated ctDNA for ARHGAP40 was detected in 80.0% (4/5) of HGSC patients’ plasma, but not in benign tumor (0/3) and healthy controls (0/15). Our findings suggest that promoter hypermethylation may be a mechanism underlying ARHGAP40 silencing in HGSC. Detection of methylated ARHGAP40 ctDNA may serve as a noninvasive biomarker for early diagnosis and monitoring of HGSC. While our findings suggest the potential of methylated ARHGAP40 as an early diagnostic biomarker, the small sample size warrants validation in larger cohorts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a13bc1c0523b1d5d86e6d21e3c7e8792952655" target='_blank'>
              Methylated ARHGAP40 DNA as a potential biomarker for early diagnosis in high-grade ovarian serous cancer
              </a>
            </td>
          <td>
            Jiaxin Chai, Dongni Leng, Shuwei Guo, Rusong Zhang, Jiandong Wang, Yun Gu, Qiu Rao
          </td>
          <td>2025-07-02</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Renal cell carcinoma (RCC) represents the most common solid renal malignancy, accounting for 90% of kidney cancers and 3% of all malignancies. Clear cell RCC (ccRCC) is the predominant histologic subtype, comprising roughly 80% of cases. Despite a 5-year survival rate of 75%, up to 30% of patients with localized ccRCC eventually develop recurrence or distant metastases. Mitotic spindle organizing protein 2A (MZT2A), also known as FAM128A, GCP8, or MOZART2A, is located on chromosome 2q21.1 and encodes a 158-amino acid protein first described by Hutchins et al. in 2010. To date, limited studies have linked MZT2A to tumorigenesis, primarily in lung and gastric cancers. The tumor microenvironment (TME), particularly tumor-infiltrating immune cells, plays a critical role in cancer progression and prognosis. Using paired tumor and normal samples from TCGA, we assessed MZT2A expression across multiple cancer types. Collectively, these findings suggest that MZT2A is involved in immune cell infiltration and the tumor microenvironment, may promote KIRC progression, and is overexpressed in cases with poor outcomes, supporting its potential as a diagnostic and prognostic biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d43bd634f019908094dd923cc5fef3d5a13d2f5" target='_blank'>
              Integrated transcriptomic analysis identifies MZT2A as a prognostic and immune-related biomarker in KIRC.
              </a>
            </td>
          <td>
            Yunjing Zhang, Zekun Xin, Xinnuo Li, Yanmei Chen
          </td>
          <td>2025-07-15</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="


 Soft tissue sarcomas (STS) comprise ∼7% of pediatric cancers, with over half classified as rhabdomyosarcoma (RMS), equating to approximately 4.5 cases per 1 million children. Germline mutations in the TP53 tumor suppressor gene (MutTP53), most notably in Li-Fraumeni Syndrome (LFS), represent the strongest heritable risk factor for RMS and are associated with poorer overall survival. Despite this, a transcriptomic signature specific to LFS-associated RMS (LFS-RMS) remains undefined. To address this, we performed droplet-based single-cell RNA sequencing (scRNA-seq; 10x Genomics) on RMS tumors and matched healthy muscle tissue from Trp53 R172H/WT mice, harboring a mutation analogous to the human TP53 R175H hotspot. Tumors were collected at endpoint from male (n = 2) and female (n = 3) mice. Cell types were identified through differential gene expression (DGE) analysis and annotated via consensus across machine learning-based tools. Trajectory analysis using Monocle3 revealed lineage transitions, while Generalized Additive Models (GAMs) identified genes associated with the progression from non-malignant to cancer-associated states. We observed consistent alterations in cell subtype proportions between tumor and healthy tissue, including a significant reduction in the M1/M2 macrophage proportion. Trajectory analysis identified key transcripts driving the transition from muscle-associated to cancer-associated fibroblasts and myogenic cells. Pathways enriched in GO analysis suggest that during the transition to cancer-associated phenotypes, cells in LFS-RMS adopt gene programs that reduce susceptibility to immune cell invasion in the tumor microenvironment. Our findings provide the first single-cell map of LFS-RMS, revealing dynamic transcriptional states and biomarkers of cells states associated with tumorigenesis. These insights pave the way for improved therapeutic approaches tailored to the complex cellular heterogeneity of LFS tumors. Further, these findings may broadly enhance RMS diagnostics and precision oncology approaches.



 Ashby Kissoondoyal, Paula R. Quaglietta, Brianne Laverty, Safa Majeed Grant, David Malkin. Identifying single-cell transcriptomic signatures and changes underlying the evolution of fusion-negative rhabdomyosarcoma in Li-Fraumeni Syndrome mice [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Artificial Intelligence and Machine Learning; 2025 Jul 10-12; Montreal, QC, Canada. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(13_Suppl):Abstract nr B042.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f326946c46ad1fff7d92ca7092d9a3464822119" target='_blank'>
              Abstract B042: Identifying single-cell transcriptomic signatures and changes underlying the evolution of fusion-negative rhabdomyosarcoma in Li-Fraumeni Syndrome mice
              </a>
            </td>
          <td>
            Ashby Kissoondoyal, Paula R. Quaglietta, B. Laverty, Safa Majeed Grant, D. Malkin
          </td>
          <td>2025-07-10</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Many pathogenic fungi display 'two-speed genome', with the fast-evolving genomic compartments enriched with repetitive sequences, particularly the transposons, which have been shown to drive the variation of pathogenicity-associated genes. Supernumerary chromosomes (SCs) are known to facilitate genomic variation in fungal pathogens, but their specific role in such processes remains understudied. In this study, we assessed the transferability of SCs between asexual Magnaporthe oryzae strains during co-culture and co-infection, and investigated their role in genome reconstruction through experimental evolution assays. We found that SCs could be horizontally transferred between M. oryzae strains and revealed frequent structural variations facilitated by SCs, including deletions, duplications, translocations, and SC-core chromosome recombinations during and after horizontal transfer. Remarkably, all observed intra- and inter-chromosome rearrangements were confined to core chromosome ends and SCs, indicating a robust role of SCs in facilitating genetic exchange within fast-evolving genomic compartments. Additionally, SC carrying the avirulence gene AvrPikE modulates M. oryzae virulence against Pikh rice through horizontal transfer, loss of whole SC, and segmental deletions. Our findings establish SCs as critical players in shaping the diversity and dynamics of the pathogenic fungal genomes, highlighting them as a cradle for the variation of pathogenicity-associated genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30fc7ba98841d16bfd287154023ca3410a86f26c" target='_blank'>
              Experimental insights into genome reconstruction driven by horizontal transfer of supernumerary chromosomes in Magnaporthe oryzae.
              </a>
            </td>
          <td>
            Zhenyu Fang, Yuyong Li, Jianqiang Huang, Meilian Chen, Xiali Chen, Xueting Mo, Zhenhui Zhong, Xiuxiu Li, Guodong Lu, Guifang Lin, Zonghua Wang, Huakun Zheng
          </td>
          <td>2025-08-03</td>
          <td>The New phytologist</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Ewing sarcoma is a relatively rare aggressive tumor of bones and soft tissues characterized by specific chromosomal translocations involving genes and transcription factors of the FET and ETS families. A number of studies have demonstrated the presence of molecular genetic events preceding these specific rearrangements.The article presents a clinical observation of the treatment of a 24-year-old patient with Ewing sarcoma of the X rib and CHEK2-associated cancer predisposition syndrome with an oncologically burdened family history, including Ewing sarcoma in a first-degree relative. Specific translocation of the EWSR1 gene (22q12) was identified, characteristic of tumors of the Ewing sarcoma/PNET (primitive neuroectodermal tumors) family, as well as 59 variants of different functional significance in suppressor genes and driver genes was identified in the course of a comprehensive molecular genetic study using fluorescence in situ hybridization (FISH) and next-generation sequencing (NGS) with a targeted custom panel including 415 genes involved in carcinogenesis. The patient underwent combined treatment in the volume of neoadjuvant polychemotherapy, surgical stage (extirpation of the X rib with plastic surgery) and subsequent adjuvant polychemotherapy. During dynamic observation for the period 2022–2025, there were no signs of progression and metastatic process. Molecular genetic profiling in Ewing sarcoma has identified markers that may act as risk modifiers for disease development and progression, determine sensitivity/resistance to standard treatment methods, and serve as potential targets for personalized treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89b306fbc45bbb4cc8b3926afaa4d96e92f8aed8" target='_blank'>
              Ewing sarcoma and CHEK2-related tumor predisposition syndrome (case report and review)
              </a>
            </td>
          <td>
            L. N. Lyubchenko, K. M. Chernavina, A. I. Senderovich, A. Kolomeytseva, M. Sedova, V. V. Chernyshov, A. Fedenko
          </td>
          <td>2025-07-25</td>
          <td>Bone and soft tissue sarcomas, tumors of the skin</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Cancer is a leading cause of death worldwide, and researchers are continuously working to uncover its drivers and new therapeutic targets. In this review, we explain the role of the oncogenic protein CREPT that is overexpressed in the vast majority of cancers, leading to poor overall survival. CREPT ablation has shown sustained tumor regression in vitro and in mouse models. CREPT activation mediates cancer progression by enhancing the cell cycle by promoting chromatin loop formation. Thus, selectively targeting CREPT expression may provide alternative strategies for inhibiting abnormal cell division, metastasis, and tumor growth, and may improve the efficacy of the currently available therapies in combination, and will be key to reversing cancerous growth in patients with CREPT-driven cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c17bae866b8a91dd27eb2fc5ec68639a8ba51279" target='_blank'>
              Disrupting Cell Cycle Machinery: CREPT Is an Emerging Target in Cancer Therapy
              </a>
            </td>
          <td>
            Umar Farooq, Jun Li, Zhijie Chang
          </td>
          <td>2025-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background Non-coding RNAs (ncRNAs) in human cells constitute a substantial portion of the transcriptome but do not lead to protein synthesis. Among them, long non-coding RNAs (lncRNAs, > 200 nucleotides long) are fascinating in their ability to orchestrate critical cellular functions that govern cell development, differentiation, and metabolism. Therefore, the dysregulation of lncRNAs has been linked with several diseases, chiefly cancers. Results We focused here on colorectal cancer (CRC), the second-highest cause of mortalities related to cancer worldwide, and more particularly on three lncRNAs, i.e., LINC01589, MELTF-AS1, and UXT-AS1, known to be dysregulated in CRC. We identified a vulnerability in these lncRNAs that could be exploited from a therapeutic point of view: a part of their sequence folds into a secondary structure referred to as G-quadruplex (G4), which is suspected to play active roles in the lncRNA functions. We demonstrate here that these sequences do fold into G4s both in vitro and in CRC cells, and that these G4s can be modulated using PhpC, a prototype molecule for destabilizing G4s. Conclusion We describe an innovative anticancer strategy that fully abides by the rules of chemical biology. We indeed modulate the formation of G4s in cells using ad hoc molecular tools in the aim of disturbing the homeostasis and inner functioning of lncRNAs. By exploiting cellular outcomes, we infer how this pharmacomodulation affects CRC biology and, beyond this, the fate of CRC cells owing to the flawed repertoire of correction and/or compensatory mechanisms in cancer cells. Supplementary Information The online version contains supplementary material available at 10.1186/s12915-025-02322-8.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/754c36f086e107d8742747ebe134ba178418b705" target='_blank'>
              Pharmacomodulation of G-quadruplexes in long non-coding RNAs dysregulated in colorectal cancer
              </a>
            </td>
          <td>
            Shubham Sharma, Jérémie Mitteaux, A. Pipier, Marc Pirrotta, Marie-José Penouilh, D. Monchaud, Bhaskar Datta
          </td>
          <td>2025-08-08</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The original 100.3 Mb reference genome for Caenorhabditis elegans, generated from the wild-type laboratory strain N2, has been crucial for analysis of C. elegans since 1998 and has been considered complete since 2005. Unexpectedly, this long-standing reference was shown to be incomplete in 2019 by a genome assembly from the N2-derived strain VC2010. Moreover, genetically divergent versions of N2 have arisen over decades of research and hindered reproducibility of C. elegans genetics and genomics. Here we provide a 106.4 Mb gap-free, telomere-to-telomere genome assembly of C. elegans, generated from CGC1, an isogenic derivative of the N2 strain. We use improved long-read sequencing and manual assembly of 43 recalcitrant genomic regions to overcome deficiencies of prior N2 and VC2010 assemblies and to assemble tandem repeat loci, including a 772 kb sequence for the 45S rRNA genes. Although many differences from earlier assemblies come from repeat regions, unique additions to the genome are also found. Of 19,972 protein-coding genes in the N2 assembly, 19,790 (99.1%) encode products that are unchanged in the CGC1 assembly. The CGC1 assembly also may encode 183 new protein-coding and 163 new ncRNA genes. CGC1 thus provides both a completely defined reference genome and corresponding isogenic wild-type strain for C. elegans, allowing unique opportunities for model and systems biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c89374937e846af8ac1bd9833e5b0b9344446ab" target='_blank'>
              CGC1, a new reference genome for Caenorhabditis elegans
              </a>
            </td>
          <td>
            Kazuki Ichikawa, M. Shoura, Karen L. Artiles, Dae-Eun Jeong, Chie Owa, Haruka Kobayashi, Yoshihiko Suzuki, Manami Kanamori, Yu Toyoshima, Yuichi Iino, A. Rougvie, Lamia Wahba, Andrew Fire, Erich M. Schwarz, Shinichi Morishita
          </td>
          <td>2025-07-15</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Circular RNAs (circRNAs) are covalently closed RNA molecules generated through a non-canonical splicing event known as back-splicing. This particular class of non-coding RNAs has attracted growing interest due to its evolutionary conservation across eukaryotes, high expression in the central nervous system, and frequent dysregulation in various pathological conditions, including cancer. Traditionally, circRNAs have been characterised by their ability to function as microRNA (miRNA) and protein sponges. However, recent discoveries from multiple research groups have uncovered a novel and potentially transformative mechanism of action: the direct interaction of circRNAs with messenger RNAs (mRNAs) to regulate their fate. These interactions can influence mRNA stability and translation, revealing a new layer of post-transcriptional gene regulation. In this review, we present and analyse the latest evidence supporting the emerging role of circRNAs in diverse biological contexts. We highlight the growing body of research demonstrating circRNA-mRNA interactions as a functional regulatory mechanism and explore their involvement in key physiological and pathophysiological processes. Understanding this novel mechanism expands our knowledge of RNA-based regulation and opens new opportunities for therapeutic strategies targeting circRNA-mRNA networks in human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58d4d3f0032738d3daf69daef3e7b0d31248f1c4" target='_blank'>
              Direct circRNA-mRNA Binding Controls mRNA Fate: A New Mechanism for circRNAs
              </a>
            </td>
          <td>
            Raffaele Garraffo, Manuel Beltran Nebot
          </td>
          <td>2025-07-18</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer is an enigma, and different types and subtypes add to the complexity of this disease. Innumerable molecules play important roles in glial pathogenesis. One such molecular component discovered recently and originating from the non-coding region of the human genome is the long non-coding RNA, or lncRNA. Multiple lncRNAs have been shown to play a role in tumor development in different cancer types. Although underappreciated as a therapeutic component, emerging studies have highlighted their crucial contributions to intercellular signaling. Studies suggest that lncRNAs play multiple, diverse roles in cancer cells. Their contribution to disease pathogenesis seems contrasting because the same lncRNA facilitating tumorigenesis in a particular cancer type may play a completely antagonistic role in another. In this review, we discuss multiple lncRNAs and the intracellular targets that are pertinent to glioma pathology, the sensitive expression levels, their impact on the disease progression, the pathways they affect, and their consideration as potential biomarkers for glioma. We further discuss multiple lncRNAs in the most aggressive glioma subtype, glioblastoma. Overall, we provide comprehensive information about key lncRNAs in glioma pathogenesis which could be of future fundamental and therapeutic relevance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f473a5d748d1ada307293cc0b8a9bbbf79222c54" target='_blank'>
              Long non-coding RNA as modulators of brain pathology: insights into cellular signaling diversity
              </a>
            </td>
          <td>
            Megha Chaudhary, Shreya Maity, Naveen Soni, Shweta Dongre, Ashwani Tiwari, Bhawana Bissa
          </td>
          <td>2025-07-24</td>
          <td>Academia Molecular Biology and Genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite significant successes, immunotherapy continues to face numerous challenges in practical applications. CD82 plays an important role in the tumor microenvironment (TME) across various cancers. However, the prognostic value of CD82 in immunotherapy and its role in the immune response remain unclear. The expression levels of CD82 in healthy human tissues were assessed using the human protein atlas (HPA) database tool. The association among CD82 expression levels, copy number variations (CNV), and methylation was explored using the gene set cancer analysis (GSCA) database. Timer tumor immune estimation resource (TIMER) database is used for immune checkpoint and immune infiltration. The LinkedOmincs database is used for enrichment analysis. The statistical inquiry was performed through the R program (version 4.4.1). The result has shown that CD82 mRNA levels were significantly downregulated in most tumor types. Abnormal CD82 mRNA expression was associated with CNV and methylation changes in tumor tissues. Furthermore, several immune checkpoint genes related to CD82 influenced prognosis and were linked to increased immune cell infiltration across various cancer forms, including liver hepatocellular carcinoma (LIHC). The prognosis of the patients under multiple cancer forms, particularly LIHC, was remarkably influenced by CD82, which plays a vital role in modulating immune infiltration. CD82 has the potential to serve as both a prognostic indicator and an immunological biomarker in human cancer. CD82 expression is frequently downregulated across various cancer associated with poor prognosis, influenced by CNV and DNA methylation alterations. Its abnormal expression correlates with changes in immune checkpoint gene activity and enhanced immune cell infiltration, notably in liver. These associations suggest that CD82 plays a significant role in shaping the tumor immune microenvironment and affects patient prognosis. Consequently, CD82 holds promise as both a prognostic biomarker and an immunomodulatory target in cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e6b1867f8cfd33352d9c4d7c23852fa5423b5ed" target='_blank'>
              Prognostic and immunological significance of CD82 in cancers with focus on hepatocellular carcinoma: A bioinformatics approach
              </a>
            </td>
          <td>
            Sikandar Ali, Shiyu Liu, Jiali Chen, Jawad Ullah, Guoyu Mou, Xiaohua Huang
          </td>
          <td>2025-07-25</td>
          <td>Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 DNA methylation, an essential epigenetic modification, plays a critical role in establishing and maintaining cellular identities by regulating gene expression. In cancer, aberrant DNA methylation patterns contribute to tumor development and progression, yet the cell-type-specific methylation alterations within the tumor microenvironment (TME) remain poorly understood. In breast cancer, a highly heterogeneous malignancy, endothelial cells within the TME (TECs) are key players in tumor angiogenesis and progression. Methylation changes detected in bulk tumor analyses, however, reflect signals from multiple cell types present in the TME, making it challenging to pinpoint cell-type-specific epigenetic changes. Recently, methods have been developed to using DNA methylation to deconvolute cell type proportions which can in turn be used to attribute methylation alterations to specific cell types in the TME. A reference-based cell-type deconvolution algorithm, the Hierarchical Tumor Immune Microenvironment Epigenetic Deconvolution (HiTIMED) method, allows accurate quantification of cell type proportions within the TME. We accessed DNA methylation array data from 609 breast tumors and 331 nontumor normal breast tissues from the Gene Expression Omnibus (GEO) and the Genotype-Tissue Expression (GTEx) portal. To these DNA methylation profiles, we applied HiTIMED and measured tumor and non-tumor cell proportions. Then, applying a statistical interaction model called CellDMC we leveraged the cell type proportion data to identify differential methylation specific to cell types in the breast cancer TME, including TECs and tumor cells. We identified 1897 tumor-cell-specific and 180 TEC-specific differentially methylated cytosines (DMCs) in cancer compared to normal breast tissue (FDR ≤ 1e-10). Importantly, in the cell-type unadjusted differential analysis, tumors exhibited hypermethylation compared to normal tissues; however, in the tumor-cell-specific analysis, tumor cells were hypomethylated relative to normal samples. By applying these advanced techniques to breast cancer samples, our study unveils cell-type-specific DNA methylation changes that are pivotal in driving the pro-tumorigenic environment. These findings not only enhance our understanding of the molecular mechanisms underlying breast cancer progression but also highlight potential therapeutic targets within the TME. The integration of HiTIMED and CellDMC represents a significant methodological advancement in cancer epigenetics, offering a more precise and insightful approach to studying the complex interplay between different cell types within the TME.
 Citation Format: Barbara Barbara, Lucas A. Salas, Brock C. Christensen. Tumor and endothelial cell-specific methylation alterations identified in breast cancer with epigenetic deconvolution [abstract]. In: Proceedings of the San Antonio Breast Cancer Symposium 2024; 2024 Dec 10-13; San Antonio, TX. Philadelphia (PA): AACR; Clin Cancer Res 2025;31(12 Suppl):Abstract nr P5-06-29.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77ae964d40b279e042709ea22a740c1da93f0fc" target='_blank'>
              Abstract P5-06-29: Tumor and endothelial cell-specific methylation alterations identified in breast cancer with epigenetic deconvolution
              </a>
            </td>
          <td>
            Barbara Barbara, Lucas A Salas, Brock C. Christensen
          </td>
          <td>2025-06-13</td>
          <td>Clinical Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Müller glia (MG) are retinal resident cells with diverse functions, including reprograming and regeneration in certain species. While the mammalian retina possesses molecular mechanisms for MG dedifferentiation and neuronal differentiation, it fails to generate neural progenitors in vivo. We previously proposed that an epigenetic barrier, driven by DNA methylation, may prevent complete MG reprograming in response to damage. DNA demethylases, such as Ten-Eleven Translocation (TET) and Growth Arrest and DNA Damage-Inducible Protein 45 (GADD45) families, are induced by damage and play roles in cellular reprograming and regeneration. Tet enzymes are regulated by metabolic factors (iron, vitamins A and C) and respond to environmental stimuli. Here, we investigated DNA demethylation as a regulatory signal for MG dedifferentiation and neuronal differentiation in response to N-methyl-D-aspartate (NMDA), a neurotransmitter used to model retinal neurodegeneration. Using mouse primary cultures and antibodies against methylated (5mC, MeCP2) and unmethylated (5 hmC, H3K4Me3) DNA, we analyzed MG epigenetic changes under control, vitamin-supplemented, and NMDA-stimulated conditions. We also assessed DNA demethylase expression and key reprograming genes (Ascl1, Lin28, Nestin). Vitamin A and C increased 5hmC levels but did not upregulate Tet enzymes or reprograming genes. In contrast, NMDA increased Tet3 and reprograming gene expression. Tet3 knockdown led to a rapid 5mC increase and impaired NMDA-induced upregulation of Ascl1, Lin28, and Nestin, suggesting its critical role in MG dedifferentiation. Conversely, Tet3 overexpression induced morphological changes and early neuronal marker expression. These findings identify Tet3 as a key epigenetic regulator of MG reprograming and a potential target for retinal regeneration strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52ab362578948bd169fde4a8815b59e370d39ecc" target='_blank'>
              Tet3-mediated DNA demethylation is essential for maintaining the dedifferentiation capacity of mammalian Müller glia
              </a>
            </td>
          <td>
            Erick J. Martinez-Colin, Ivonne Lezama, Lenin D. Ochoa-de la Paz, Mónica Lamas
          </td>
          <td>2025-07-17</td>
          <td>Frontiers in Molecular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="An R-loop is a complex nucleic acid structure consisting of an RNA–DNA hybrid and an associated single-stranded DNA. This structure plays an important role in many biological processes, including gene regulation, DNA replication, and maintenance of genome stability. Under physiological conditions, the R-loop is in a dynamic balance of generation and decomposition, which is a complex process involving multiple mechanisms of its formation, clearance, and regulation. Emerging evidence indicates that R-loops can drive inflammation and participate in immune processes. This review summarizes immune dysregulation resulting from impaired R-loop clearance and analyzes recent advances in computational methods for R-loop regulation, aiming to identify new avenues for developing R-loop-targeted immunotherapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29bce6b2f2435a1bf918ec22e6cdf3b596fdd7f0" target='_blank'>
              Unraveling R-loops: The hidden drivers of inflammation and immune dysregulation
              </a>
            </td>
          <td>
            Fan Zhang, Dingyun Wang, Guodong Zhao, Dingmao Wang
          </td>
          <td>2025-06-13</td>
          <td>Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background/Objectives: Increasing evidence suggests that lncRNAs are core regulators in the field of tumor progression, with context-specific functions in oncogenic tumorigenesis. LINC01133, a lncRNA that has been identified as both an oncogene and a tumor suppressor, remains largely unexplored in terms of its molecular mechanisms. The purpose of this study was to conduct an in silico analysis, incorporating literature research on various cancer types, to investigate the structural and functional duality of LINC01133. This analysis aimed to identify pathways influenced by LINC01133 and evaluate its mechanism of action as a potential therapeutic target and diagnostic biomarker. Methods: In silico analyses and a narrative review of the literature were performed to predict conserved structural elements, functional internal loops, and overall conservation of the LINC01133 sequence among different vertebrate organisms, summarizing the empirical evidence regarding its roles as a tumor suppressor and tumor-promoting roles in various types of tumors. Results: LINC01133 harbors the evolutionarily conserved structural regions that might allow for binding to relevant driver signaling pathways, substantiating its specific functionality. Its action extends beyond classical tumor mechanisms, affecting proliferation, migration, invasion, and epigenetic pathways in various types of tumors, as indicated by the in silico results and narrative review of the literature we present here. Clinical outcome associations pointed to its potential as a biomarker. Conclusions: The dual character of LINC01133 in tumor biology further demonstrates its prospective therapeutic value, but complete elucidation of its mechanisms of action requires further investigation. This study establishes LINC01133 as a multifaceted lncRNA, supporting context-specific strategies in targeting its pathways, and calls for expanded research to harness its full potential in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f66180b93b3358b4aae4204b9c68eb1c73a7d7f" target='_blank'>
              The Good, the Bad, or Both? Unveiling the Molecular Functions of LINC01133 in Tumors
              </a>
            </td>
          <td>
            Leandro Teodoro Júnior, M.C. Sogayar
          </td>
          <td>2025-07-30</td>
          <td>Non-Coding RNA</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="To what extent overdominance might contribute to the maintenance of genetic diversity within genomes is still an ongoing research question. Pseudo-overdominance created by the complementation of deleterious alleles in heterozygotes has recently become a subject of particular interest. Simulations and theory suggest that pseudo-overdominance may occur in low recombining regions. Here, we conduct a comprehensive investigation of large low-recombining (LLR) regions in cultivated populations of pearl millet, an outcrossing diploid African cereal. We examine seven large regions ranging from 5 to 88 Mb and six of them are pericentromeric. These LLR regions exhibit an excess of heterozygotes, a distinctive hallmark of overdominance. They display a tendency toward a higher diversity and a larger ratio of non-synonymous and deleterious variants. We conduct a more in-depth study of the largest 88 Mb region, identified on chromosome 3. Interestingly, haplotypes of this region have been introgressed from wild relatives. Using long read sequencing, we confirm their strong divergence and the presence of inversions across one of them. One of the haplotypes seems to be highly deleterious in the homozygous state. A total of 17% of the cultivated pearl millet genome exhibit a local population structure suggestive of overdominance or possibly pseudo-overdominance. Our empirical results contribute to the accumulation of knowledge, which will enhance our understanding of the potential role of overdominance or pseudo-overdominance in maintaining genetic diversity, particularly in low recombining regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8563879e9297d9a2f3d59a5ca6775ee20a33deef" target='_blank'>
              Interplay between large low-recombining regions and pseudo-overdominance in a plant genome
              </a>
            </td>
          <td>
            Marine Salson, Maud Duranton, Stella Huynh, C. Mariac, Christine Tranchant-Dubreuil, Julie Orjuela, Phillipe Cubry, Anne-Céline Thuillet, C. Burgarella, Miguel de Navascués, Leïla Zekraouï, M. Couderc, Sandrine Arribat, Nathalie Rodde, A. Barnaud, A. Faye, N. Kane, Y. Vigouroux, C. Berthouly-Salazar
          </td>
          <td>2025-07-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="


 Gliomas are classified as one of the most lethal cancers owing to their invasive and aggressive nature. Numerous genetic and epigenetic changes have been studied in the development and progression of GBM. Dysregulation of epigenetic function may bring about profound changes in gene expression to facilitate glioma formation and development. SETD2, a key epigenetic regulator is frequently mutated in various cancers; SETD2 regulates m6A mRNA methylation (epitranscriptome) via H3K36 trimethylation. The m6A RNA methylation modification regulates various biological functions like gene regulation, cell self–renewal and maintenance, cell differentiation, invasion, and tumorigenesis. The RNA modifications are executed by a set of complex proteins collectively known as writers, readers and erasers (RNA modifiers). These modifiers work along with the molecular cues laid down by SETD2 via H3K36 trimethylation. The positive correlation between SETD2 expression and m6A RNA modifiers highlights its direct involvement in epitranscriptomics. Glioma, a highly malignant grade IV brain tumor, is characterized by limited therapeutic options and poor clinical outcomes. However, the specific role of SETD2 in glioma and its impact on m6A RNA methylation remain inadequately defined. In this study, we investigated the expression of m6A RNA methylation regulators, the molecular pathways contributing to tumor progression, and the outcomes associated with SETD2-mediated m6A modifications in glioma. Our in vitro analyses demonstrated that knocking down SETD2 in glioma cell lines resulted in reduced oncogenic properties, such as decreased cell proliferation and invasiveness. This effect was accompanied by a global reduction in m6A methylation levels within the transcriptome, which was closely linked to the downregulation of key m6A writers, METTL3 and METTL14. These findings suggest that SETD2 plays a crucial role in maintaining m6A methylation, and its disruption can drive glioma progression through altered epitranscriptomic landscapes, offering new insights into molecular oncogenesis and potential therapeutic targets.



 Subhadra Kumari, Srinivasan M. THE ROLE OF SETD2 IN THE MOLECULAR ONCOGENESIS OF GLIOMA THROUGH m6A RNA MODIFICATION [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P75.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33a10b3b7631ab056fcfe5ee35c047e63418abb6" target='_blank'>
              Abstract P75: THE ROLE OF SETD2 IN THE MOLECULAR ONCOGENESIS OF GLIOMA THROUGH m6A RNA MODIFICATION
              </a>
            </td>
          <td>
            Subhadra Kumari, M. Srinivasan
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract The Asian icefish, Protosalanx chinensis, has undergone extensive colonization in various waters across China for decades due to its ecological and physiological significance as well as its economic importance in the fishery resource. Here, we decoded a telomereto-telomere (T2T) genome for P. chinensis combining PacBio HiFi long reads and ultra-long ONT (nanopore) reads and Hi-C data. The telomere was identified in both ends of the contig/chromosome. The expanded gene associated with circadian entrainment suggests that P. chinensis may exhibit a high sensitivity to photoperiod. The contracted genes’ immune-related families and DNA repair associated with positive selection in P. chinensis suggested the selection pressure during adaptive evolution. The population genetic analysis reported the genetic diversity and genomic footprints in 231 individuals from 7 different locations. The introduced highest alkalinity population (HRCL) exhibited higher values of inbreeding coefficients and clustered different from other groups suggested local environmental adaptation. Thus, the T2T genome and genetic variation can be valuable resources for genomic footprints in P. chinensis, shedding light on its evolution, comparative genomics, and the genetic differences between natural and introduced populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff0e41e198fbe29dbf276b6596922e3840811763" target='_blank'>
              Telomere-to-telomere genome and resequencing of 231 individuals reveal evolution, genomic footprints in Asian icefish, Protosalanx chinensis
              </a>
            </td>
          <td>
            Yanfeng Zhou, Chenhe Wang, Binhu Wang, Dongpo Xu, Xizhao Zhang, You Ge, Shulun Jiang, Fujiang Tang, Chunhai Chen, Xuemei Li, Jianbo Jian, Yang You
          </td>
          <td>2025-01-06</td>
          <td>GigaScience</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Mucosal melanoma (MM) is a deadly cancer derived from mucosal melanocytes. To test the consequences of MM genetics, we develop a zebrafish model in which all melanocytes experience CCND1 expression and loss of PTEN and TP53. Surprisingly, melanoma only develops from melanocytes lining internal organs, analogous to the location of patient MM. We find that zebrafish MMs have a unique chromatin landscape from cutaneous melanomas. Internal melanocytes are labeled using a MM-specific transcriptional enhancer. Normal zebrafish internal melanocytes share a gene expression signature with MMs. Patient and zebrafish MMs show increased migratory neural crest and decreased antigen presentation gene expression, consistent with the increased metastatic behavior and decreased immunotherapy sensitivity of MM. Our work suggests that the cell state of the originating melanocyte influences the behavior of derived melanomas. Our animal model phenotypically and transcriptionally mimics patient tumors, allowing this model to be used for MM therapeutic discovery. As this is a non-MAPK driven genetically engineered model of melanoma, our work also has implications for the 15% of cutaneous melanoma patients who lack MAPK-driving mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99321e90c4b2d4aa26110c1e8b75f84421ebcd50" target='_blank'>
              Specific oncogene activation of the cell of origin in mucosal melanoma
              </a>
            </td>
          <td>
            Swathy Babu, Jiajia Chen, Chloé S. Baron, Kaiwen Sun, Emily J. Robitschek, Alicia M. McConnell, Constance Wu, Aikaterini Dedeilia, Moshe Sade-Feldman, Rodsy Modhurima, Michael P. Manos, Kevin Y. Chen, Anna M. Cox, Calvin G. Ludwig, M. Kellis, Elizabeth I Buchbinder, N. Hacohen, Jie-Ying Yang, Genevieve M Boland, Brian J. Abraham, David Liu, Leonard I. Zon, Megan L. Insco
          </td>
          <td>2025-07-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>113</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bcdcfe442326883ec1ff1d1b7880290f079cb82" target='_blank'>
              Germline genetic variation impacts clonal hematopoiesis landscape and progression to malignancy.
              </a>
            </td>
          <td>
            Jie Liu, Duc Tran, Liying Xue, B. Wiley, C. Vlasschaert, Caroline J. Watson, Hamish A J MacGregor, X. Zong, I. Chan, Indraniel Das, M. Uddin, A. Niroula, Gabriel Griffin, Benjamin L Ebert, T. Mack, Y. Pershad, B. Sharber, Michael F. Berger, A. Zehir, R. Ptashkin, Ross L. Levine, E. Papaemmanuil, V. Joseph, Teng Gao, Y. Kemel, D. Mandelker, Konrad H Stopsack, Paul D. P. Pharoah, Semanti Mukherjee, Li Ding, Yinyin Cao, MJ Walter, Jamie R Blundell, Nilanjan Chatterjee, K. Offit, Lucy A. Godley, Daniel C. Link, Z. Stadler, Alexander G. Bick, P. Natarajan, Kelly L Bolton
          </td>
          <td>2025-07-15</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>131</td>
        </tr>

        <tr id="The AP-1 transcription factor FOSL1, also known as Fra-1, is a crucial oncoprotein that plays an important role in human tumor progression and metastasis and has thus emerged as a promising therapeutic target. FOSL1 regulates the expression of a large protein-coding gene network, and this molecular mechanism can promote the progression of tumors. Interestingly, recent studies have shown that FOSL1 can also achieve the same protumor effect by regulating certain noncoding RNAs (ncRNAs). However, more studies have shown that ncRNAs can regulate the expression and activity of FOSL1, thereby affecting the occurrence and development of tumors, which indicates that ncRNAs can be regulators of FOSL1 in cancer. In this review, we first provide a comprehensive overview of the expression and function of FOSL1 and ncRNAs in tumors and then focus on the mutual regulatory relationship between ncRNAs and FOSL1, as well as their regulatory effects on and mechanisms of tumor progression. In addition, we further explored the potential clinical applications of the FOSL1-ncRNA system in cancer treatment, providing a theoretical basis for the study of FOSL1 and/or ncRNA-related molecular markers or targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f413c89530da64009a6941f40cf33c862cd2843" target='_blank'>
              Noncoding RNAs as regulators of FOSL1 in cancer
              </a>
            </td>
          <td>
            Xiaochang Wang, Li Wang, Shoushi Wang, Jinjie Zhang, Xueyang Wang, Ting Zhang, Linwei Li, Jin Wei, Yi Zhao, Zhixia Zhou
          </td>
          <td>2025-08-01</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Plasmodium parasites, the causative agents of malaria, undergo crucial developments within the mosquito vector, initiated by the formation of male and female gametes. Male gametogenesis involves three rapid rounds of mitosis without nuclear or cell division, followed by a single round of DNA segregation and nuclear division during gamete budding. How the cell organizes the segregation of eight genomes from a single octoploid nucleus into eight haploid gametes is currently unknown. Here we discovered an atypical Arp2/3 complex in Plasmodium important for DNA segregation during male gametogenesis. Unlike the canonical Arp2/3 complex found in other eukaryotes, Plasmodium Arp2/3 localizes to endomitotic spindles and interacts with a kinetochore-associated protein. Disruption of Arp2/3 subunits or actin polymerization interferes with kinetochore–spindle association, causes the formation of subhaploid gametes, and blocks transmission. Our work identified an evolutionary divergent Arp2/3 complex in malaria parasites, provides insights into gametogenesis, and reveals potential targets for transmission-blocking interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a0eb8cd0287d7e4de3c5f68362eedb92788d67" target='_blank'>
              An atypical Arp2/3 complex is required for Plasmodium DNA segregation and malaria transmission
              </a>
            </td>
          <td>
            Franziska Hentzschel, David Jewanski, Yvonne Sokolowski, Pratika Agarwal, Anna Kraeft, Kolja Hildenbrand, Lilian P. Dorner, Mirko Singer, Matthias Marti, Friedrich Frischknecht
          </td>
          <td>2025-06-13</td>
          <td>Nature Microbiology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Triple negative breast cancers (TNBCs) are a heterogenous subcategory of breast cancers which have significantly worse survival outcomes compared to other breast cancer subtypes. However, limited advances have been made towards a targeted TNBC therapeutic and traditional chemotherapies remain the frontline therapy. Therefore, this study aimed to examine the therapeutic potential of a novel TNBC target, Barrier-to-Autointegration Factor 1 (Banf1), including determining Banf1 expression and cellular localisation in non-malignant breast cells and a panel of TNBC cell lines. Bioinformatic analysis of patient samples demonstrated that Banf1 is overexpressed in all breast cancer stages and subtypes. Banf1 depletion inhibited proliferation and induced mitotic arrest in TNBC cells via loss of nuclear envelope integrity and aberrant nuclear morphology, inducing TNBC tumour cell-specific cell death. These findings highlight the significant overexpression and functional involvement of Banf1 in TNBC progression and suggests that it may have potential as a novel anti-cancer target, supporting further investigation. Supplementary Information The online version contains supplementary material available at 10.1038/s41598-025-10346-x.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecd1f5909e830afa31cad48c33f6b5d6c0d1eeae" target='_blank'>
              The inner nuclear membrane protein, Banf1, has an essential role in triple negative breast cancer cell proliferation and survival
              </a>
            </td>
          <td>
            M. Rose, J. Burgess, Chee Man Cheong, Iain Richard, Amila Suraweera, M. Adams, Pascal H. G. Duijf, Kenneth J. O’Byrne, Derek J. Richard, E. Bolderson
          </td>
          <td>2025-07-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="In meiosis, ploidy reduction is driven by a complex series of DNA breakage and recombination events between homologous chromosomes, orchestrated by meiotic HORMA domain proteins (HORMADs). Meiotic HORMADs possess a central chromatin binding region (CBR) whose architecture varies across eukaryotic groups. Here, we determine high-resolution crystal structures of the meiotic HORMAD CBR from two diverged aquatic holozoa, Schistosoma mansoni and Patiria miniata, which reveal tightly-associated PHD and winged helix-turn-helix (wHTH) domains. We show that PHD–wHTH CBRs bind duplex DNA through their wHTH domains, and identify key residues that disrupt this interaction. Combining experimental and predicted structures, we show that the CBRs’ PHD domains likely interact with the tail of histone H3, and may discriminate between unmethylated and trimethylated H3 lysine 4. Finally, we show that holozoa Hop1 CBRs bind nucleosomes in vitro in a bipartite manner involving both the PHD and wHTH domains. Our data reveal how meiotic HORMADs with PHD–wHTH CBRs can bind chromatin and potentially discriminate between chromatin states to drive meiotic recombination to specific chromosomal regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a79b34bae8b1dbcf266da5a700548580b6368ded" target='_blank'>
              Bipartite chromatin recognition by Hop1 from two diverged holozoa
              </a>
            </td>
          <td>
            Alyssa A. Rodriguez, Alessandro E. Cirulli, Katie Chau, Justin Nguyen, Qiaozhen Ye, Kevin D. Corbett
          </td>
          <td>2025-06-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/687af5811b08e28febe794844c74138cc2e493e9" target='_blank'>
              The PARP1-EXD2 axis orchestrates R-loop resolution to safeguard genome stability.
              </a>
            </td>
          <td>
            Zhaoshuang Li, Yu Liu, Yuanhui Liu, Yiting Zhang, M. Huen, Huasong Lu, Zhigang Zhang, Jianwei Zhou, Dong Fang, Ting Liu, Jun Huang
          </td>
          <td>2025-06-27</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="BACKGROUND
Efferocytosis (ER) plays a crucial role in the programmed clearance of dead cells, a process that is mediated by phagocytic immune cells. However, further exploration is needed to determine the full extent of its impact on the progression of pancreatic ductal adenocarcinoma (PDAC), particularly through interactions among tumor cells, stromal cells, and immune cells within the tumor microenvironment (TME).


METHODOLOGY AND RESULTS
In this study, we comprehensively analyzed the Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) database, as well as additional databases from multiple bioinformatics websites, utilizing 167 ER features derived from the integration of single-cell RNA sequencing (scRNA-seq) and bulk transcriptomic data. A set of 14 ER-associated prognostic signatures, referred to as the "14-gene panel" genes, was identified based on overall survival (OS)/disease-free survival (DFS) data, Pearson correlation coefficients, and multivariate Cox regression analyses. The model pathways enriched by the four-gene combination represented by "LEAF" and the 14-gene combination represented by the "14-gene panel" presented a high degree of similarity, including among the adhesion, mitotic, G2/M checkpoint, and epithelial‒mesenchymal transition (EMT) signaling pathways. Least absolute shrinkage and selection operator (LASSO) regression was subsequently employed to construct an ER risk scoring system using deep learning, based on the following formula: LGALS3, EMP1, ASPH, and FNDC3B, collectively termed the "LEAF" panel. Additionally, random survival forest (RSF) algorithms facilitated the identification of a key panel of genes, designated "LEAP" genes, including LGALS3, EREG, ASPH, and PLS3; three of which genes (ASPH, LGALS3, and EREG) were identified as key factors influencing the behaviors of PDAC tumors, tumor-associated stroma, and macrophages. Finally, we utilized experimental methods, including Boyden chamber analyses, immunohistochemical staining, and cell cycle analyses, to demonstrate that interference with ASPH suppresses the malignant properties of tumors, including proliferation and migration. Multiplex immunofluorescence staining was employed to identify EREG as highly relevant to the M2 macrophage subpopulation.


CONCLUSION
Our findings underscore the importance of considering a novel prognostic signature comprising 14 ER genes in the context of the TME when investigating the biology of PDAC. Future studies may explore how modulating these interactions could lead to novel therapeutic opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59c81897cb1c769e789b52358a323bc255ac9a02" target='_blank'>
              A Novel 14-Gene Panel Associated With Efferocytosis for Predicting Pancreatic Cancer Prognosis Through Bulk and Single-Cell Databases.
              </a>
            </td>
          <td>
            Yaheng Wu, Lin Zhao, Dingyan Yi, Zhihua Tian, Bin Dong, Chunxiang Ye, Jingtao Liu, Huachong Ma, Wei Zhao
          </td>
          <td>2025-07-30</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) has recently become a focus of cardiovascular disease (CVD) research as it is possibly involved in disease mechanisms and may function as a biomarker. The literature suggests that eccDNAs may modify gene expression patterns and contribute to the pathogenesis of diseases such as pulmonary arterial hypertension (PAH). Many eccDNAs have been reported to be upregulated in PAH patients with promisingly high diagnostic sensitivity and specificity values. The advent of eccDNA detection from plasma samples using high‐throughput sequencing technologies has opened new avenues for non‐invasive diagnosis. In this review, we focus on the role of eccDNA in CVDs, concentrating on its prospects for use as a biomarker of disease development and stage. However, further studies should be conducted to explicate the functional aspects of eccDNAs in cardiovascular health and disease. This work may eventually lead to the development of novel treatment options and improved clinical outcomes for CVD patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/903ec3642b6c2ce94d9db06e20223312c1b439d8" target='_blank'>
              Extrachromosomal Circular DNA: Emerging Insights Into Cardiovascular Disease Mechanisms and Biomarker Potential
              </a>
            </td>
          <td>
            Saim Mahmood Khan, Jawairya Muhammad Hussain, Surraiya Riaz Mahmood Khan, Bushra Nadeem, Taniyat Naseem, Hammad Jawed, Aina Lashari
          </td>
          <td>2025-06-29</td>
          <td>iLABMED</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Innate immune cells constitute the majority of the tumor microenvironment (TME), where they mediate both natural anti-tumor immunity and immunotherapy responses. While single-cell T- and B-cell receptor sequencing has provided fundamental insights into the clonal dynamics of human adaptive immunity, the lack of appropriate tools has precluded similar analysis of innate immune cells. Here, we describe a method that leverages somatic mitochondrial DNA (mtDNA) mutations to reconstruct clonal lineage relationships between single cells across cell types in native human tissues. We jointly sequenced single-cell transposase-accessible chromatin and mtDNA to profile n =124,958 cells from matched tumor, non-involved lung tissue (NILT), and peripheral blood of early-stage non-small cell lung cancer (NSCLC) patients, as well as n =93,757 cells from matched tumor and peripheral blood of ovarian cancer patients. Single-cell concomitant profiling of lineage and cell states of thousands of immune cells resolved clonality across cell types, tissue sites, and malignancies. Clonal tracing of innate immune cells demonstrates that TME-resident myeloid subsets, including macrophages and type 3 dendritic cells (DC3), are clonally linked to both circulating and tissue-infiltrating monocytes. Further, we identify distinct DC-biased and macrophage-biased myeloid clones, enriched in the tumor and NILT, respectively, and find that their circulating monocyte precursors exhibit distinct epigenetic profiles, suggesting that myeloid differentiation fate may be predetermined before TME infiltration. These results delineate the clonal pathways of intratumoral myeloid cell recruitment and differentiation in human cancer and suggest that remodeling of the tumor myeloid compartment may be peripherally programmed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ba9df6c66740c3cece720fa35456bef19fb17f8" target='_blank'>
              Clonal lineage tracing of innate immune cells in human cancer
              </a>
            </td>
          <td>
            Vincent Liu, Katalin D. Sandor, Patrick K. Yan, Max Miao, Yajie Yin, Robert R. Stickels, , Kamir J. Hiam-Galvez, Jacob Gutierrez, Wenxi Zhang, Sairaj Sajjath, Raeline Valbuena, Steven Wang, Bence Dániel, Leif S. Ludwig, Brooke E Howitt, Caleb A. Lareau, Ansu Satpathy
          </td>
          <td>2025-07-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df35593218f0e76cafa59bd11175b1d4c35d9c92" target='_blank'>
              Activated PLK1 promotes the migration and invasion of polyploid giant cancer cells with daughter cells via β-catenin/STAT3 signaling pathway.
              </a>
            </td>
          <td>
            Xinyue Zhou, Yi Ning, Xiaorui Wang, Jing Xu, Yandi Liu, Mingqing Zhang, Shiwu Zhang
          </td>
          <td>2025-07-28</td>
          <td>Human cell</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="All germlines—including those of humans, protists, cancer, and metazoans—are capable of proliferating through asymmetric cell division, giving rise to committed stem cells. Their common evolutionarily roots trace back to the hypoxic germline of the last common ancestor of amoebozoans, metazoans, and fungi, referred to as the Ur-germline. Consequently, all modern germlines that produce stem cells—including the cancer stemgermline—retain physiological characteristics of this ancestral Ur-germline. Stress, particularly hyperoxic germline conditions, can irreversibly damage the DNA repair genes in the stemgermline, leading to genome dysfunction, mitotic arrest (senescence), and loss of key functions such as stemness and asymmetric cell division. In most cases, genomically compromised senescent cells in humans and animals undergo apoptotic senescence and are eliminated. However, a minority of dysfunctional cells undergo restorative senescence, initiating a process of unicellularization and genome reconstruction. During unicellularization, genes associated with multicellular functions are downregulated, while ancient unicellular stemgermline genes are upregulated. Cells exiting restorative senescence follow a genome repair program via characteristic unicellular mechanisms, such as hyperpolyploidization and depolyploidization. This process gives rise to progenitor cells that establish a cancer stemgermline and a unicellular cancer cell system. Briefly, restorative senescence escapers restore genomic architecture, function, and molecular integrity using evolutionary mechanisms inherited from the Ur-germline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e904557b06689862f566644873a1f68fbe6aba" target='_blank'>
              Non-genetic mechanisms in cancer evolution: senescence, unicellularization, and cycles of stemness recovery
              </a>
            </td>
          <td>
            V. Niculescu
          </td>
          <td>2025-06-25</td>
          <td>Academia Molecular Biology and Genomics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Despite the critical role of DNA methylation, clinical implementations harnessing its promise have not been described in acute myeloid leukemia. Utilizing DNA methylation from 3314 leukemia patient samples across 11 harmonized cohorts, we describe the Acute Leukemia Methylome Atlas, which includes robust models capable of accurately predicting AML subtypes. A genome-wide prognostic model as well as a targeted panel of 38 CpGs significantly predict five-year survival in our pediatric and adult test cohorts. To accelerate rapid clinical utility, we develop a specimen-to-result protocol that uses long-read nanopore sequencing and machine learning to characterize patients’ whole genomes and epigenomes. Clinical validation on patient samples confirms high concordance between epigenomic signatures and genomic lesions, though uniquely rare karyotypes remained challenging due to limited available training data. These results unveil the potential for increased affordability, speed, and accuracy for patients in need of complex molecular diagnosis and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081a93b57ec5afcf494724d930e4432c5308fda4" target='_blank'>
              Epigenomic diagnosis and prognosis of Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Francisco Marchi, Vivek M Shastri, Richard J Marrero, Nam Nguyen, Antonella Öttl, Ann-Kathrin Schade, Marieke Landwehr, O. Krali, J. Nordlund, Matin Ghavami, Fernando Sckaff, Vikash K. Mansinghka, Xueyuan Cao, William Slayton, Petr Starostik, Christopher Cogle, Raul C Ribeiro, J. Rubnitz, J. Klco, Abdelrahman A. Elsayed, A. Gamis, Timothy J. Triche, R. Ries, E. A. Kolb, R. Aplenc, T. Alonzo, S. Pounds, S. Meshinchi, J. Lamba
          </td>
          <td>2025-07-29</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Cancer remains a formidable global health burden owing to its persistently high incidence and mortality, highlighting the need to elucidate its underlying biological mechanisms. Such insights are crucial for refining diagnostic precision and therapeutic efficacy. Notably, human tumors represent highly heterogeneous ecosystems, comprising a dynamic interplay between malignant cells and non-malignant components, such as immune infiltrates and stromal elements. This cellular heterogeneity constitutes a major obstacle in tumor research. Recent technological advancements have enabled the development and application of novel tools for investigating tumor heterogeneity. In this context, single-cell sequencing technologies emerged as transformative tools for dissecting tumor architecture at a cellular resolution, offering unprecedented insights into the cellular diversity and molecular underpinnings of cancer. In particular, single-cell RNA sequencing (scRNA-seq) has been widely used to analyze the tumor microenvironment, particularly its immune components, thereby providing a valuable framework for understanding tumor-immune system interactions. However, the intrinsic heterogeneity of tumor cells has received comparatively less attention, with limited studies focusing on these malignant populations. Comprehensive profiling of tumor cells has become increasingly feasible as scRNA-seq technologies continue to evolve, offering higher throughput, increased cell capture efficiency, and more advanced analytical pipelines. Despite these advancements, current applications of scRNA-seq remain primarily focused on immune and stromal cell populations. Therefore, a paradigm shift is warranted, redirecting the investigative focus toward tumor cells and gaining deeper insights into tumorigenesis and therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45c3598e7911b47f7291bb4a9f0c4b83bd05a9bc" target='_blank'>
              Exploration and Application of Malignant Cell Heterogeneity Analysis with Single-Cell Transcriptome Sequencing Technology.
              </a>
            </td>
          <td>
            Wen Zhang, Tianliang Liu, Ting Liu, Yu Zhang, Zhenguo Zhao, Xinxin Zhang, Xiaoyang Li, Jiasheng Zhong, Zhicheng Li, Shifu Chen, Libin Xu
          </td>
          <td>2025-07-25</td>
          <td>Aging and disease</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e559b4aa95ac215d2b0dba18f35a9867916e3c3" target='_blank'>
              ADH5/ALDH2 dehydrogenases and DNA polymerase theta protect normal and malignant hematopoietic cells from formaldehyde challenge: therapeutic implications.
              </a>
            </td>
          <td>
            Jessica Atkins, Anna-Mariya Kukuyan, Monika M. Toma, M. Drzewiecka, Umeshkumar Vekariya, Adam Karami, M. Nieborowska-Skorska, R. Nejati, E. Hadzijusufovic, P. Valent, Tomasz Stokłosa, Tomasz Sliwinski, Mariusz A. Wasik, Tomasz Skorski
          </td>
          <td>2025-07-10</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="High-risk neuroblastomas exhibit a high degree of intratumoral heterogeneity. Single-cell RNA sequencing has greatly improved our understanding of these tumors, but the method lacks cellular tissue context and spatial information about local signaling dynamics. To address this, we profiled untreated and chemotherapy-treated high-risk neuroblastomas from archived, formalin-fixed, paraffin-embedded (FFPE) tissues from two patients using spatial transcriptomics. We confirmed the transcriptional and cellular heterogeneous nature of the neuroblastoma microenvironment and identified several unique spatial niches and patterns. In one of the treated tumors, a spatially constrained cluster of undifferentiated and 11p-gained cancer cells was identified, surrounded by a rim of macrophages. A signaling interaction between the chemokine CCL18 and its receptor PITPNM3 was predicted between these cells. In the other tumor, we identified a stromal cluster with high transcriptional similarity to the adrenal cortex. These adrenocortical-like cells expressed several oncogenic ligand-encoding genes (e.g. ALKAL2 and NRTN), which were predicted to communicate with neighboring cancer cells that expressed the corresponding receptors (e.g. ALK, RET). Several of these interactions were further validated experimentally and were shown to be clinically relevant. Collectively, our spatial analysis identifies multiple previously unrecognized signaling axes that may offer novel therapeutic options in neuroblastoma. © 2025 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dce7c8d7d28d1cfa3f77ec4f74044ee737c8a1da" target='_blank'>
              Spatial transcriptomics exploration of the primary neuroblastoma microenvironment in archived FFPE samples unveils novel paracrine interactions.
              </a>
            </td>
          <td>
            J. Siaw, Peter Merseburger, Marcus Borenäs, Caroline Jansson, Jenny Karlsson, A. Claeys, E. Jennische, D. Lind, David Gisselsson Nord, R. Palmer, Jimmy Van den Eynden
          </td>
          <td>2025-08-08</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Systems that perform continuous hypermutation of designated genes without compromising the integrity of the host genome can substantially accelerate the evolution of new or enhanced protein functions. We describe an orthogonal DNA replication system in Escherichia coli based on the controlled expression of the replisome of bacteriophage T7 (T7-ORACLE). The system replicates circular plasmids that enable high transformation efficiencies and seamless integration into standard molecular biology workflows. Engineering of T7 DNA polymerase yielded variant proteins with mutation rates of 1.7 × 10-5 substitutions per base in vivo-100,000-fold above the genomic mutation rate. We demonstrated continuous evolution using the T7 replisome by expanding the substrate scope of TEM-1 β-lactamase and increasing activity 5000-fold against clinically relevant monobactam and cephalosporin antibiotics in less than 1 week.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed28e398210a42a2a56f735d7fe9f761f9cf0a6" target='_blank'>
              An orthogonal T7 replisome for continuous hypermutation and accelerated evolution in E. coli.
              </a>
            </td>
          <td>
            Christian S Diercks, Philipp Sondermann, Cynthia Rong, Thomas G Gillis, Yahui Ban, Celine Wang, David A. Dik, Peter G. Schultz
          </td>
          <td>2025-08-07</td>
          <td>Science</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Geminiviridae is the largest group of single-stranded (ss) DNA viruses, infecting a wide range of plant species and posing a significant threat to agriculture. Hence, we used molecular dynamics simulations to explore the structural impacts and the stability of Ageratum yellow vein virus (AYVV) capsids in the presence and absence of DNA. The findings show that dimers, trimers, pentamers, and higher multimers are remarkably stable when bound to DNA during the course of the 300 ns simulation. On the contrary, the oligomers demonstrate significant instability and disintegration in the absence of DNA. While DNA-free oligomers retained structural integrity, the quaternary association broke down, particularly in pentameric and dimeric units. While previous experimental studies have demonstrated the stability of geminiviral pentamers, our observations suggest that their resilience is likely linked to their association with DNA. Remarkably, the dimers lacking the bound DNA also display an atypical variation in curvature over time. Consequently, we propose that any trigger that loosens the association of the genome with the oligomers would be a prerequisite for the intersubunit contacts in the capsid to break and release the genome. We hypothesize that genomic DNA may interact with the pentamers to initiate capsid assembly, which may pave the way for the dimers and trimers to be integrated. The natural flexibility of the dimers could render it easier to provide the appropriate curvature and closure to the swiftly forming capsid. Thus, techniques that target DNA-protein interactions in geminiviruses could be the key to their effective management in fields.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67020310361e4b5049d66b37a061235a0fb67d42" target='_blank'>
              Elucidating the role of genome in geminiviral capsid dynamics.
              </a>
            </td>
          <td>
            Yagavi Ravindran, Kothai Thiruvengadam, L. S. Baddireddi, Sangita Venkataraman
          </td>
          <td>2025-06-29</td>
          <td>Journal of biomolecular structure & dynamics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="


 Tumor tissues exhibit cell-specific genetic variations and gene expression profiles critical to disease diagnosis, progression, and treatment outcomes. Single cell sequencing offers detailed insights into the heterogeneity of cancer cells which is pivotal for developing personalized treatment strategies. However, current high-throughput methods primarily detect gene expression levels, lacking information on genetic variants such as mutations or gene fusions. Understanding tumor responses to therapy requires linking cell-type specific mutations with gene expression profiles to understand cancer development and progression.



 We developed FocuSCOPE, a high-throughput single cell sequencing solution that simultaneously measures gene expression and detects genetic variantion from individual cells. Single cell partitioning is performed using SCOPE-chip®, a portable microwell chip the size of a microscopy slide. The technology utilizes barcoding microbeads conjugated with oligos to efficiently capture mRNA and target sequences. The optimized chemistry facilitates amplification and library construction of both mRNA and target regions, including point mutations, fusion genes, and non-polyadenylated RNAs such as viral RNAs.



 NB4, CCRF, and K562 cell lines were mixed in equal proportions. Libraries were prepared using the FocuSCOPE Blood Cancer Panel. Experiments demonstrated the specific detection of KRAS and TP53 mutations in mixed cell lines (NB4, CCRF, and K562) using the FocuSCOPE Blood Cancer Panel. The NB4 cell line contained KRAS (A18D) and TP53 (R248Q) mutations, CCRF contained KRAS (G12D) and TP53 (R248Q, R175H), and K562 contained only the BCR-ABL1 fusion gene. Additionally, FocuSCOPE effectively captured PML-RARA and BCR-ABL1 fusion genes with high sensitivity.



 FocuSCOPE enables comprehensive analysis of gene expression profiles and targeted genetic sequences, capturing driver mutations, fusion genes, and non-polyA sequences such as viral sequences with high sensitivity, thus advancing our understanding of tumor biology and therapeutic responses. This makes it an invaluable tool for personalized medicine, guiding therapeutic decisions and improving patient outcomes.



 Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick. FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia [abstract]. In: Proceedings of Frontiers in Cancer Science 2024; 2024 Nov 13-15; Singapore. Philadelphia (PA): AACR; Cancer Res 2025;85(15_Suppl):Abstract nr P17.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fafbd9d94e3c2b4f9a27806a0df8f1e19c248cc6" target='_blank'>
              Abstract P17: FocuSCOPE: A Multi-omics Solution to Simultaneously Analyse Genetic Variants and Transcriptome in Single Cells in Leukemia
              </a>
            </td>
          <td>
            Ankhita Nair, Rachael Lee, Sujatha Narayanankutty, Yingting Wang, Jonathan Scolnick
          </td>
          <td>2025-08-01</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Epigenetics and chromatin biology have been two distinct research fields for many years. There is a close relationship between epigenetics and chromatin biochemistry, and the dynamic changes of chromatin serve as an important basis for epigenetic regulation. The relationship between epigenetics and chromatin is reflected in the dynamic changes and structural adjustments of chromatin. In this article, we focus on previously underappreciated chromatin remodeling proteins. This review summarizes the biological significance of chromatin remodeling complexes in malignancies, including the physiological role of chromatin remodeling complexes in normal cells and the dysregulation of chromatin remodeling complexes in malignant tumors with genetic, epigenetic, and gene expression alterations. It is important to explore the impact of chromatin remodeling complexes on cancer therapy, as predictors of drug response, targets of drug response, and contributors to therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7307ea466a4a0493cfc44f580cad60747699034c" target='_blank'>
              Chromatin and epigenetic regulation in malignant tumors: A comprehensive review.
              </a>
            </td>
          <td>
            Xiaoman Zhang, Dequan Liu, Xiaochi Chen, Tian Li, Guangzhen Wu
          </td>
          <td>2025-08-04</td>
          <td>Annals of the New York Academy of Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Viral oncolysis is considered a promising cancer treatment method because of its good tolerability and durable anti-tumor effects. Compared with other oncolytic viruses, Newcastle disease virus (NDV) has some distinct advantages. As an RNA virus, NDV does not recombine with the host genome, making it safer compared with DNA viruses and retroviruses; NDV can induce syncytium formation, allowing the virus to spread among cells without exposure to host neutralizing antibodies; and its genome adheres to the hexamer genetic code rule (genome length as a multiple of six nucleotides), ensuring accurate replication, low recombination rates, and high genetic stability. Although wild-type NDV has a killing effect on various tumor cells, its oncolytic effect and working mechanism are diverse, increasing the complexity of generating engineered oncolytic viruses with NDV. This study aims to employ whole-genome CRISPR-Cas9 knockout screening and RNA sequencing to identify putative key regulatory factors involved in the interaction between NDV and human colon cancer HCT116 cells and map their global interaction networks. The results suggests that NDV infection disrupts cellular homeostasis, thereby exerting oncolytic effects by inhibiting cell metabolism and proliferation. Meanwhile, the antiviral immune response triggered by NDV infection, along with the activation of anti-apoptotic signaling pathways, may be responsible for the limited oncolytic efficacy of NDV against HCT116 cells. These findings not only enhance our understanding of the oncolytic mechanism of NDV against colonic carcinoma but also provide potential strategies and targets for the development of NDV-based engineered oncolytic viruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e05426a39d80cc8e94e4737d115ccac397145186" target='_blank'>
              Genome-Wide Screening Reveals the Oncolytic Mechanism of Newcastle Disease Virus in a Human Colonic Carcinoma Cell Line
              </a>
            </td>
          <td>
            Yu Zhang, Shufeng Feng, Gaohang Yi, Shujun Jin, Yongxin Zhu, Xiaoxiao Liu, Jinsong Zhou, Hai Li
          </td>
          <td>2025-07-25</td>
          <td>Viruses</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The CRISPR/Cas system has attracted increasing attention in accurate nucleic acid detection. Herein, we reported a CRISPR/Cas12a-chemiluminescence cascaded bioassay (CCCB) for the amplification-free and sensitive detection of human papillomavirus type 16 (HPV-16) and parvovirus B19 (PB-19). A magnetic bead (MB)-linking single-stranded DNA (LssDNA)-alkaline phosphatase (ALP) complex was constructed as the core component of the bioassay. During the detection process, the single-stranded target DNA was captured and enriched by LssDNA and then activated the trans-cleavage activity of Cas12a. Due to the Cas12a-mediated cleavage of LssDNA, ALP was released from the MB, subsequently catalyzing the substrate to generate a chemiluminescence (CL) signal. Given the cascade combination of CRISPR/Cas12a with the CL technique, the limits of detection for HPV-16 and PB-19 DNA were determined as 0.14 pM and 0.37 pM, respectively, and the whole detection could be completed within 60 min. The practicality and reliability of the platform were validated through target-spiked clinical specimens, and the recovery rate was 93.4–103.5%. This dual-amplification strategy—operating without target pre-amplification—featured high specificity, low contamination risk, facile preparation, and robust stability. It provides a novel approach for sensitive nucleic acid detection, with the potential for rapid extension to the diagnosis of various infectious diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59899f53ef027d63c2b6a1e8d9eb261cb093dae8" target='_blank'>
              CRISPR/Cas12a-Chemiluminescence Cascaded Bioassay for Amplification-Free and Sensitive Detection of Nucleic Acids
              </a>
            </td>
          <td>
            Xiaotian Guan, Peizheng Wang, Yi Wang, Shuqing Sun
          </td>
          <td>2025-07-24</td>
          <td>Biosensors</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Conventional wisdom holds that nuclear oncogenes and tumor suppressors initiate malignant transformation. However, mounting research suggests that mitochondrial dysfunction—rooted in the unique evolutionary history and genetic autonomy of mitochondria—may serve as a more fundamental driver of oncogenesis. This paper proposes a “mitochondria-first” hypothesis of cancer, emphasizing the pivotal role of mitochondrial DNA (mtDNA) mutations, metabolic reprogramming, and immune evasion. By examining the evolutionary conflict between host and mitochondria, evaluating high mtDNA mutation rates, and highlighting the disruptive potential of mitochondrial transfer to immune cells, we outline robust mechanisms through which mitochondria could ignite cancer development. We also discuss emerging diagnostic and therapeutic approaches that target mitochondrial integrity, offering a potential paradigm shift in oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a09759d1fd0b99bd9f5921041ad32e37eb247e60" target='_blank'>
              Are Mitochondria the True Origin of Cancer? A Hypothesis-Driven Perspective
              </a>
            </td>
          <td>
            Sergio Da Silva
          </td>
          <td>2025-07-01</td>
          <td>Onco</td>
          <td>0</td>
          <td>5</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2025'],
    y: [4],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>